<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/227416-an-polypeptide-of-cytokine-protein-family-and-a-nucleotide-encoding-the-same by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:04:57 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 227416:AN POLYPEPTIDE OF CYTOKINE PROTEIN FAMILY AND A NUCLEOTIDE ENCODING THE SAME</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">AN POLYPEPTIDE OF CYTOKINE PROTEIN FAMILY AND A NUCLEOTIDE ENCODING THE SAME</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>AIM POLYPEPTIDE OF CYTOKINE PROIEUM FAMILY AND A NUCLEOTIDE ENCODING THE SAME ABSTRACT OF THE DISCLOSURE The present invention relates to polynucleotide and polypeptide molecules for zcyto20, zcyto21, zcyto22, zycto24, and zcyto25 proteins which are most closely related to interferon-cc at the amino acid sequence level. The receptor for this protein family is a class II cytokine receptor. The present invention includes methods of reducing viral infections and increasing monocyte counts. The present invention also includes antibodies to the zcyto20 polypeptides, and methods of producing the polynucleotides and polypeptides. IP</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
CYTOKINE PROTEIN FAMILY<br>
BACKGROUND OF THE  INVENTION<br>
Cellular differentiation of multicultural organisms is controlled by hormones and polypeptide growth faction. These diffusible molecules allow cells to communicate with each other and act in concert to form tissues and organs, and to repair and regenerate damaged tissue. Examples of hormones and growth factors include the steroid hormones, parathyroid hormone, follicle stimulating hormone, the interferon’s, the interleukins, platelet derived growth factor, epidermal growth factor, and granulocyte-macrophage colony stimulating factor, among others.<br>
Hormones and growth factors influence cellular metabolism by binding to receptor proteins. Certain receptors are integral membrane proteins that  bind with the hormone or growth factor outside the cell, and that are linked to signaling pathways within the cell, such as second messenger systems. Other classes of receptors are soluble intracellular molecules.<br>
Cytokines generally stimulate proliferation or differentiation of cells of the hematopoietic lineage or participate in the immune and inflammatory response mechanisms of the body. Examples of cytokines which affect hematopoiesis are erythropoietin (EPO), which stimulates the development of red blood cells; thrombopoietin (TPO), which stimulates development of cells of the megakaryocytic lineage; and granulocyte-colony stimulating factor (G-CSF), which stimulates development of Europhiles. These cytokines are useful in restoring normal blood cell levels in patients suffering from anemia, thrombocytopenia, and Neutrogena or receiving chemotherapy for cancer.<br>
Cytokines play important roles in the regulation of hematopoiesis and immune responses, and can influence lymphocyte development. The human class II cytokine family includes interferon-a (IFN-a) subtypes, interferon-p (IFN- p), interferon-Y (IFN-7), IL-10. IL-19 (U.S. Patent 5,985.614), MDA-7 (Jiang et al.. Oncogene 11, 2477-2486, (1995)), IL-20 (Jiang et al.. Oncogene H, 2477-2486, (1995)), IL-22 (Xie et al.. J. Biol. Chem. 275, 31335-31339, (2000)), and AK-155 (Knappe et al., J. Virol. 74, 3881-3887, (2000)). Most cytokines bind and transduce signals through either Class I or Class H cytokine receptors. Members of human class n cytokine receptor family include interferon-oRl (IFN-oRl), interferon-Y-R2 (HTS-y-<br><br>
R2), interferon-Y Rl (rFN-7 Rl), interferon-YR2 (IFN-YR2), JL-WR (LIN et al., J, Immunol. 152. 1821-1829, (1994)), CRF2-4 (Lutfalla et al. Genomics 16, 366-373, (1993)), IL-20RP (Biumberg et al., CeU 104, 9-19, (2001)) (also known as zcytorV (U.S. Patent 5,945,511) and CRF2-8 (Kotenko et al., Oncogene 19, 2557-2565, (2000)), IL-20Rp (Biumberg et al., ibjd, (2001)) (also known as DIRSl (PCT WO 99/46379)), IL-22RA1 (IL-22 receptor-al, submitted to HUGO for approval) (also known as IL-22R (Xie et al., J. Biol. Chem. 275, 31335-31339, (2000)), zcytorll (U.S. Patent 5,965,704) and CRF2-9 (Koneke et al.. Oncogene 19,2557-2565, (2000)), and tissue factor.<br>
Class n cytokine receptors are typically heterodynes composed of two distinct receptor chains, the a and p receptor subunits (Stahl et al.. Cell 74, 587-590, (1993)). In general, the a subunits are the primary cytokine binding proteins, and the p subunits are required for formation of high affinity binding sites, as well as for signal transduction. An exception is the IL-20 receptor in which both subunits are required for IL-20 binding (Blumberg et al., ibid, (2001)).<br>
The class II cytokine receptors are identified by a conserved cytokine-binding domain of about 200 amino acids (D200) in the extra cellular portion of the receptor. This cytokine-binding domain is comprised of two fibronectin type III (FnIII) domains of approximately 100 amino acids each (Bazan J.F. Proc. Natl. Acad. Sci. USA 87, 6934-6938, (1990); Thoreau et al.. FEBS Lett. 282.16-31, (1991)). Each FnIII domain contains conserved Cys, Pro, and Trp residues that determine a characteristic folding pattern of seven P-strands similar to the constant domain of immunoglobulin (Use et a!., J. Interferon Cytokine Res. 15, 3-26, (1995)). The conserved structural elements of the class H cytokine receptor family make it possible to identify new members of this family on the basis of primary amino acid sequence homology. Previously we have successfully identified two new members of class II cytokine receptor family, zcytor7 (U.S. Patent 5,945,511) (also known as IL-20R a (Blamey et al., ibid, (2001)) and zcytorll (U.S. Patent 5,965,704) (also known as IL-22R (Blumberg et al., ibid, (2001)), using this approach. Identification of additional novel members of the class II cytokine receptor family is of interest because cytokines play a vital role in regulating biological responses.<br>
IL-22, also known as IL-TTF (IL-10-related T cell-derived inducible factor) (Udometer et al., J. Immunology 164. 1814-1819, (2000)), is a recently described IL-10 homologue. Mouse IL-22 was originally identified as a gene induced by IL-9 in T cells and mast cells in vitro (Udometer et al., J. Immunology 164. 1SI4-1819, (2000)). Acute phase reactant induction activity was observed in mouse liver upon IL-22 injection, and IL-22 expression was rapidly induced after lip polysaccharide   (LPS)   injection,   suggesting   that   IL-22   contributes   to   the<br><br>
inflammatory response in vivo (Dumoutier et al., Proc. Natl. Acad. Sci- U.S.A. 97. 10144-10149, (2000)).<br>
The interleukins are a family of cytokines that mediate immunological responses, including inflammation. The interleukins mediate a variety of inflanunatory pathologies. Central to an immune response is the T cell, which produce many cytokines and adaptive immunity to antigens. Cytokines produced by the T cell have been classified as type 1 and type 2 (Kelso, A. Immun. Cell Biol. 76:300-317, 1998). Type 1 cytokines include IL-2, IFN-y, LT-a, and are involved in inflammatory responses, viral immunity, intracellular parasite immunity and allograft rejection. Type 2 cytokines include IL-4, IL-5, IL-6, IL-10 and IL-13, and are involved in humoral responses, helminth immunity and allergic response. Shared cytokines between Type I and 2 include IL-3, GM-CSF and TNF-a. There is some evidence to suggest that Type 1 and Type 2 producing T cell populations preferentially migrate into different types of inflamed tissue.<br>
Of particular interest, from a therapeutic standpoint, are the interferon’s (reviews on interferon’s are provided by De Mayer and De Maeyer-Guignard, "Interferons," in The Cytokine Handbook, 3rd Edition, Thompson (ed.), pages 491-516 (Academic Press Ltd. 1998), and by Walsh, Biopharmaceuticals: Biochemistry and Biotechnology, pages 158-188 (John Wiley &amp; Sons 1998)). Interferon’s exhibit a variety of biological activities, and are useful for the treatment of certain autoimmune diseases, particular cancers, and the enhancement of the immune response against infectious agents, including viruses, bacteria, fungi, and protozoa. To date, six forms of interferon have been identified, which have been classified into two major groups. The so-called "type' interferon’s include interferon-ot, interferon-P, interferon-(i), interferon-S, and interferon-T. Currently, interferon-y and one subclass of interferon-a are the only type n interferon’s.<br>
Type I interferon’s, which are thought to be derived from the same ancestral gene, have retained sufficient similar structure to act by the same cell surface receptor. The a-chain of the human interferon- receptor comprises an extra cellular N-terminal domain, which has the characteristics of a class II cytokine receptor. Interferon-y does not share significant homology with the type I interferon’s or with the type H interferon-a subtype, but shares a number of biological activities with the type I interferon’s.<br>
In humans, at least 16 non-allelic genes code for different subtypes of interferon-a, while interferon’s P and 05 are encoded by single genes. Type I interferon genes are clustered in the short arm of chromosome 9. Unlike typical structural human genes, interferon-a, interferon-p, and interferon-O) lack intones. A single gene for<br><br>
human interferon-y is localized on chromosome 12 and contains three intones. To date, interferon-T has been described only in cattle and sheep, while interferon-5 has been described only in pigs.<br>
Clinicians are taking advantage of the multiple activities of interferon’s by using the proteins to treat a wide range of conditions. For example, one form of interferon-a has been approved for use in more than 50 countries for the treatment of medical conditions such as hairy cell leukemia, renal cell carcinoma, basal cell carcinoma, malignant melanoma, AIDS-related Kaposi's sarcoma, multiple mycelia, chronic myelogenous leukemia, non-Hodgkin's lymphoma, laryngeal papillomatosis, mycosis fungicides, condyloma acuminate, chronic hepatitis B, hepatitis C, chronic hepatitis D, and chronic non-A, non-B/C hepatitis. The U.S. Food and Drug Administration has approved the use of interferon-P to treat multiple sclerosis, a chronic disease of the nervous system. Interferon-y is used to treat chronic granulomatous diseases, in which the interferon enhances the patient's immune response to destroy infectious bacterial, fungal, and protozoa pathogens. Clinical studies also indicate that interferon-y may be useful in the treatment of AIDS, leishmaniasis, and lepromatous leprosy.<br>
The demonstrated in vivo activities of the cytokine family illustrate the enormous clinical potential of, and need for, other cytokines, cytokine agonists, and cytokine antagonists. The present invention addresses these needs by providing a new cytokine that stimulates cells of the hematopoietic cell lineage, as well as related compositions and methods.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
Prior to setting forth the invention in detail, it may be helpful to the understanding thereof to define the following terms:<br>
The term "affinity tag" is used herein to denote a polypeptide segment that can be attached to a second polypeptide to provide for purification or detection of the second polypeptide or provide sites for attachment of the second polypeptide to a substrate. In principal, any peptide or protein for which an antibody or other specific binding agent is available can be used as an affinity tag. Affinity tags include a poly-histidine tract, protein A (Nilsson et al., EMBO J. 4:1075,1985; Nilsson et al.. Methods Enzvmol. 198:3. 1991), glutathione S transferase (Smith and Johnson, Gene 67:31, 1988), Glu-Glu affinity tag (Grussenmeyer et al.. Proc. Natl. Acad. Sci. USA 82:7952-4, 1985), substance P, Flag™ peptide (Hopp et al.. Biotechnology 6:1204-10, 1988), streptavidin binding peptide, or other antigenic epitope or binding domain.   See, in<br><br>
general, Ford et al.. Protein Expression and Purification 2: 95-107, 1991. DNAs encoding affinity tags are available from commercial suppliers (e.g., Pharmacia Biotech, Piscataway, NJ).<br>
The term "allelic variant" is used herein to denote any of two or more alternative forms of a gene occupying the same chromosomal locus. Allelic variation arises naturally through mutation, and may result in phenotypic polymorphism within populations. Gene mutations can be silent (no change in the encoded polypeptide) or may encode polypeptides having altered amino acid sequence. The term allelic variant is also used herein to denote a protein encoded by an allelic variant of a gene.<br>
The terms "amino-terminal" and "carboxyl-terminal" are used herein to denote positions within polypeptides. Where the context allows, these terms are used with reference to a particular sequence or portion of a polypeptide to denote proximity or relative position. For example, a certain sequence positioned carboxyl-terminal to a reference sequence within a polypeptide is located proximal to the carboxyl terminus of the reference sequence, but is not necessarily at the carboxyl terminus of the complete polypeptide.<br>
The term "complement/anti-complement pair" denotes non-identical moieties that form a non-covalently associated, stable pair under appropriate conditions. For instance, biotin and avid in (or streptavidin) are prototypical members of a complement/anti-complement pair. Other exemplary complement/anti-complement pairs include receptor/legend pairs, antibody/antigen (or hapten or epitome) pairs, sense/antisepses polynucleotide pairs, and the like. Where subsequent dissociation of the complement/anti-complement pair is desirable, the complement/anti-complainant pair preferably has a binding affinity of 
The term "complements of a polynucleotide molecule" denotes a polynucleotide molecule having a complementary base sequence and reverse orientation as compared to a reference sequence. For example, the sequence 5' ATGCACGGG 3' is complementary to 5' CCCGTGCAT 3'.<br>
The term "degenerate nucleotide sequence" denotes a sequence of nucleotides that includes one or more degenerate cordons (as compared to a reference polynucleotide molecule that encodes a polypeptide). Degenerate cordons contain different triplets of nucleotides, but encode the same amino acid residue (i.e., GAU and GAC triplets each encode Asp).<br>
The tern "expression vector" is used to denote a DNA molecule, linear or circular, that comprises a segment encoding a polypeptide of interest operably linked to additional segments that provide for its transcription. Such additional segments include promoter and terminator sequences, and may also include one or more origins<br><br>
of replication, one or more selectable markers, an enhancer, a polyadenylation signal, etc. Expression vectors are generally derived from plasmid or viral DMA, or may contain elements of both.<br>
The term "isolated", when applied to a polynucleotide, denotes that the polynucleotide has been removed from its natural genetic milieu and is thus free of other extraneous or unwanted coding sequences, and is in a form suitable for use within genetically engineered protein production systems. Such isolated molecules are those that are separated from their natural environment and include cDNA and genomic clones. Isolated DNA molecules of the present invention are free of other genes with which they are ordinarily associated, but may include naturally occurring 5' and 3' untranslated regions such as promoters and terminators. The identification of associated regions will be evident to one of ordinary skill in the art (see for example, Dynan and Tiian. Nature 316:774-78.1985).<br>
An "isolated" polypeptide or protein is a polypeptide or protein that is found in a condition other than its native environment, such as apart from blood and animal tissue. In a preferred form, the isolated polypeptide is substantially free of other polypeptides, particularly other polypeptides of animal origin. It is preferred to provide the polypeptides in a highly purified form, i.e. greater than 95% pure, more preferably greater than 99% pure. When used in this context, the term "isolated" does not exclude the presence of the same polypeptide in alternative physical forms, such as dimers or alternatively glycosylated or derivative forms.<br>
The term "euplastic", when referring to cells, indicates cells undergoing new and abnormal proliferation, particularly in a tissue where in the proliferation is uncontrolled and progressive, resulting in a neoplasm. The euplastic cells can be either malignant, i.e. invasive and metastasis, or benign.<br>
The term "operably linked", when referring to DNA segments, indicates that the segments arc arranged so that they function in concert for their intended purposes, e.g., transcription initiates in the promoter and proceeds through the coding segment to the terminator.<br>
The term "ortholog" denotes a polypeptide or protein obtained from one species that is the functional counterpart of a polypeptide or protein from a different species. Sequence differences among orthodox are the result of speciation.<br>
"Paralogs" are distinct but structurally related proteins made by an organism. Paralogs.arp believed to arise through gene duplication. For example, a-globing, P-globing, and myoglobin are paralogs of each other.<br>
A "polynucleotide" is a single- or double-stranded polymer of deoxyribonucleotide   or  rib nucleotide   bases   read   from   the   5'   to   the   3'   end.<br><br>
Polynucleotide include RNA and DNA, and may be isolated from natural sources, synthesized in vitro, or prepared from a combination of natural and synthetic molecules. Sizes of polynucleotides are expressed as base pairs (abbreviated "bp"), nucleotides ("nt"), or kilobases ("kb"). Where the context allows, the latter two terms may describe polynucleotides that are single-stranded or double-stranded. When the term is applied to double-stranded molecules it is used to denote overall length and will be understood to be equivalent to the term "base pairs". It will be recognized by those skilled in the art thai the two strands of a double-stranded polynucleotide may differ slightly in length and that the ends thereof may be staggered as a result of enzymatic cleavage; thus all nucleotides within a double-stranded polynucleotide molecule may not be paired.<br>
A "polypeptide" is a polymer of amino acid residues joined by peptide bonds, whether produced naturally or synthetically. Polypeptides of less than about 10 amino acid residues are commonly referred to as "peptides".<br>
The term "promoter" is used herein for its art-recognized meaning to denote a portion of a gene containing DNA sequences that provide for the binding of RNA polymerase and initiation of transcription. Promoter sequences are commonly, but not always, found in the 5' non-coding regions of genes.<br>
A "protein" is a macromolecule comprising one or more polypeptide chains. A protein may also comprise non-peptidic components, such as carbohydrate groups. Carbohydrates and other non-peptidic substituents may be added to a protein by the cell in which the protein is produced, and will vary with the type of cell. Proteins are defined herein in terms of their amino acid backbone structures; substituents such as carbohydrate groups are generally not specified, but may be present nonetheless.<br>
The term "receptor" denotes a cell-associated protein that binds to a bioactive molecule (i.e., a ligand) and mediates the effect of the ligand on the cell. Membrane-bound receptors are characterized by a multi-peptide structure comprising an extracellular ligand-binding domain and an intracellular effector domain that is typically involved in signal transduction. Binding of ligand to receptor results in a conformational change in the receptor that causes an interaction between the effector domain and other molecule(s) in the cell. This interaction in turn leads to an alteration in the metabolism of the cell. Metabolic events that are linked to receptor-ligand interactions include gene transcription, phosphoryation, dephosphorylation, increases in cyclic AMP production, mobilization of cellular calcium, mobilization of membrane lipids, cell adhesion, hydrolysis of inositol lipids and hydrolysis of phospholipids. In general, receptors can be membrane bound, cryptozoic or nuclear; monomeric (e.g., thyroid stimulating hormone receptor, beta-adrenergic receptor) or multimedia (e.g..<br><br>
PDGF receptor, growth hormone receptor, IL-3 receptor, GM-CSF receptor, G-CSF receptor, erythropoietin receptor and IL-6 receptor).<br>
The term "secretory signal sequence" denotes a DNA sequence that encodes a polypeptide (a "secretory peptide") that, as a component of a larger polypeptide, directs the larger polypeptide through a secretory pathway of a cell in which it is synthesized. The larger polypeptide is commonly cleaved to remove the secretory peptide during transit through the secretory pathway.<br>
The term "splice variant" is used herein to denote alternative forms of RNA transcribed from a gene. Splice variation arises naturally through use of alternative splicing sites within a transcribed RNA molecule, or less commonly between separately transcribed RNA molecules, and may result in several mRNAs transcribed from the same gene. Splice variants may encode polypeptides having altered amino acid sequence. The term splice variant is also used herein to denote a protein encoded by a splice variant of an mRNA transcribed from a gene.<br>
Molecular weights and lengths of polymers determined by imprecise analytical methods (e.g., gel electrophoresis) will be understood to be approximate values. When such a value is expressed as "about" X or "approximately" X, the stated value of X will be understood to be accurate to ±10%.<br>
All references cited herein are incorporated by reference in their entirety.<br>
The present invention includes a genus of polynucleotide and polypeptide molecules that have functional and structural similarity to the interferon’s. In this new family, which includes molecules designated zcyto20 (SEQ ID NOS: 1 and 2), zcyto21 (SEQ ID N0S:4 and 5), zcyto22 (SEQ ID N0S:6 and 7), zcyto24 (SEQ ID N0S:8 and 9), zcyto25 (SEQ ID NOS: 10 and 11), zcyto20, 21, and 22 are human sequences and zcyto24 and 25 are mouse sequences. Homology within the family at the nucleotide and amino acid levels is shown in Table I, ranging from approximately 72% to 98% at the nucleotide level, and 51% to 97% at the amino acid level.<br><br><br><br>
All the members of the family have been shown to bind to the same class II cytokine receptor, designated zcytorl9 receptor. Furthermore, certain biological activities have been shown to be exhibited by each molecule in the family. These activities include, for example, antiviral activities and increasing circulating myeloid cell levels. While not wanting to be bound by theory, these molecules appear to all signal through zcytorl9 receptor via the same pathway.<br>
ZcytQ20 gene encodes a polypeptide of 205 amino acids, as shown in SEQ ID N0:2. The signal sequence for Zcyto20 can be predicted as comprising amino acid residue 1 (Met) through amino acid residue 21 (Ala) of SEQ ID N0:2. The mature peptide for Zcyto20 begins at amino acid residue 22 (Val).<br>
Zcyto21 gene encodes a polypeptide of 200 amino acids, as shown in SEQ ID-N0:5. The signal sequence for Zcyto21 can be predicted as comprising amino acid residue 1 (Met) through amino acid residue 19 (Ala) of SEQ ID N0:5. The mature peptide for Zcyto21 begins at amino acid residue 20 (Gly). Zcyto21 has been described in PCT application WO 02/02627.<br>
Zcyto22 gene encodes a polypeptide of 205 amino acids, as shown in SEQ ID N0:7. The signal sequence for Zcyto22 can be predicted as comprising amino acid residue 1 (Met) through amino acid residue 21 (Ala) of SEQ ID N0:7. The mature peptide for Zcyto22 begins at amino acid residue 22 (Val).<br>
Zcyto24 gene encodes a polypeptide of 202 amino acids, as shown in SEQ ID N0:9. Zcyto24 secretory signal sequence comprises amino acid residue 1 (Met) through amino acid residue 28 (Ala) of SEQ ID N0:9. An alternative site for cleavage of the secretory signal sequence can be found at amino acid residue 24 (Thr).<br><br>
The mature polypeptide comprises amino acid residue 29 (Asp) to amino acid residue 202 (Vat).<br>
Zcyto25 gene encodes a polypeptide of 202 amino acids, as shown in SEQ ID N0:11. Zcyto25 secretory signal sequence comprises amino acid residue 1 (Met) through amino acid residue 28 (Ala) of SEQ ID NO: 11. An alternative site for cleavage of the secretory signal sequence can be found at amino acid residue 24 (Thr). The mature polypeptide comprises amino acid residue 29 (Asp) to amino acid residue 202 (Val).<br>
The Zcyto20, Zcytoll and Zcyto22 genes have been mapped to human chromosome 19ql3.13. Based on the discovery of these genes, this region of chromosome 19 has been identified as comprising a cluster of interferon-like genes. Further indication that this is new family of genes is identification of a syntenic cluster of genes on the mouse chromosome 7, zcyto24 (SEQ ID NO: 8) and zcyto25 (SEQ ID NO: 10).<br>
As described below, the present invention provides isolated polypeptides having an amino acid sequence that is at least 70%, at least 80%, or at least 90%, 95%, 96%, 97%, 98% or 99% identical to either amino acid residues 22 to 205 of SEQ ID N0:2 or amino acid residues 1 to 205 of SEQ ID N0:2, or some fragment thereof, The present invention also includes a polypeptide that further comprises a signal secretory sequence that resides in an amino-terminal position relative to the first amino acid sequence, wherein the signal secretory sequence comprises amino acid residues 1 to 21 of the amino acid sequence of SEQ ID N0:2.<br>
In another embodiment, the present invention provides isolated polypeptides having an amino acid sequence that is at least 70%, at least 80%, or at least 90%, 95%, 96%, 97%, 98% or 99% identical to either amino acid residues 22 to 205 of SEQ ID N0:7 or amino acid residues 1 to 205 of SEQ ID NO:7. The present invention also includes a polypeptide that further comprises a signal secretory sequence that resides in an amino-terminal position relative to the first amino acid sequence, wherein the signal secretory sequence comprises amino acid residues 1 to 21 of the amino acid sequence of SEQ ID N0:7.<br>
In general, cytokines, like erythropoietin (EPO), are predicted to have a four-alpha helix structure, with helices A, C and D being most important in ligand-receptor interactions, and are more highly conserved among members of the family. However, the interferons (ESfF), and interferon-alpha and interferon-tau in particular, are characterized as six helix bundles. EPO helix A is equivalent to helix A of zcyto20; EPO helix B is equivalent to helix C of zcyto20; EPO helix C is equivalent to helix D of zcyto20, and EPO helix D is equivalent to helix F of zcyto20.   Thus, the loop<br><br>
between the AB loop, and CD loop of EPO is expanded in zcyto20 to contain short helices B and E of zcyto20. The helical structures of 2cyto20, zcyto2I, zcyto22, 2cyto24 and 2cyto25 are similar to the six-helix structure found in the interferons. Boundaries of secondary structures in proteins are generally defined in accordance with a range of PHI and PSI angles of the protein chain backbone from a 3-dimensional model of the protein. Models may be constructed from, for example, x-ray crystallography or NMR data, or homology modeling based on a solved structure. Depending on techniques used, including conditions for crystal formation and flexible NMR solution structure determination, boundaries of these secondary structures may be slightly altered. Thus, those skilled in the art will recognize that the helical boundaries, and secondary structures in general, depending on the environment, may shift by as much as 2, 3,4, or more residues, but helical regions are essentially as described below. (See, Brandon and Toosze,. Introduction to Protein Structure. Garland Publishing Co, inc. New York, 1991; Anderson, et. al.. Structure. 10(2):I75-84. 2002.)<br>
Zcyto20 helices are predicted as follow: helix A is defined by amino acid residues 52 (Ala) to 66 (Leu); helix B by amino acid residues 78 (Arg) to 87 (Val); helix C by amino acid residues 91 (Pro) to 108 (Thr); helix D by amino acid residues 116 (Val) to 138 (Ser); helix E by amino acid residues 151 (Thr) to 172 (Lys); and helix F by amino acid residues 177 (Gly) to 197 (Cys); as shown in SEQ ID NO: 2. Four cysteine residues are conserved between Zcyto20, Zcyto21, and INF-a. In addition, 2cyto20 has 3 additional cysteines. The cysteine at amino acid residue 204, may form an intermolecular disulfide bond, in particular to form homodimers with additional Zcyto20 molecules. Further analysis of Zcyto20 based on multiple alignments predicts that cysteines at amino acid residues 37 and 136; 69 and 197; and 71 and 178 (as shown in SEQ ID NO: 2) will form intramolecular disulfide bonds. The corresponding polynucleotides encoding the Zcyto20 polypeptide regions, domains, motifs, residues and sequences described herein are as shown in SEQ ID NO: 1.<br>
2cyto21 helices are predicted as follows: helix A is defined by amino acid residues 49 (Ser) to 63 (Leu); helix B by amino acid residues 76 (Asn) to 84(Val); helix C by amino acid residues 89 (Val) to 104 (Ala); helix D by amino acid residues 111 (Glu) to 133 (Gin); helix E by amino acid residues 137 (Thr) to 158 (Lys); and helix F by amino acid residues 163 (Gly) to 189 (Leu); as shown in SEQ ID NO: 5. The cysteine residues are conserved between Zcyto21, Zcyto21, and INF-o, and may form an intermolecular disulfide bond, in particular to form homodimers with additional 2cyto21 molecules. Further analysis of Zcylo21 based on multiple alignments predicts that cysteines at amino acid residues 34 and 131, and 68 and 164, will form intramolecular disulfide bonds. The cysteine at residue 190 is free, and may<br><br>
form an intermolecular disulfide association. The corresponding polynucleotides encoding the Zcyto21 polypeptide regions, domains, motifs, residues and sequences described herein are as shown in SEQ ID N0:4.<br>
Zcyto22 helices are predicted as follow: helix A is defined by amino acid residues 52 (Ala) to 66 (Leu); helix B by amino acid residues 78 (Arg) to 87 (Val); helix C by amino acid residues 91 (Pro) to 108 (Thr); helix D by amino acid residues 116 (Val) to 138 (Ser); helix E by amino acid residues 151 (Thr) to 172 (Lys); and helix F by amino acid residues 177 (Gly) lo 197 (Cys); as shown in SEQ ID NO: 7. Four cysteine residues are conserved between 2cyto22, Zcyto21, and INF-a. In addition, Zcyto22 has 3 additional cysteines. The cysteine at amino acid residue 204, may form an intermolecular disulfide bond, in particular to form homodimers with additional Zcyto22 molecules. Further analysis of 2cyto22 based on multiple alignments predicts that cysteines at amino acid residues 37 and 136; 69 and 197; and 71 and 178 (as shown in SEQ ID NO: 7) will form intramolecular disulfide bonds. The corresponding polynucleotides encoding the Zcyto22 polypeptide regions, domains, motifs, residues and sequences described herein are as shown in SEQ ID NO:6.<br>
Conserved cysteines for 2cyto24 are shown at residues 44, 78, 141, and 175 of SEQ ID NO: 9. Further analysis of zcyto24 based on multiple alignments predicts that disulfide bonds will be formed between cysteines at amino acid residues 44 and 141; 78 and 175; (as shown in SEQ ID NO: 9). The corresponding polynucleotides encoding the zcyto24 polypeptide regions, domains^ motifs, residues and sequences described herein are as shown in SEQ ID NO:9. The predicted helices in zcyto24 (as shown in SEQ ID N0:9) are; residues 59-73 (helix A); residues 85-94 (helix.B); residues 98-115 (helix C); residues 121-143 (helix D); residues 147-169 (helix E); residues 174-194 (helix F).<br>
Conserved cysteines for zcyto25 are shown at residues 44, 78, 141, and 175 of SEQ ID NO: 11. Further analysis of zcytb25 based on multiple alignments predicts that disulfide bonds will be formed between cysteines at amino acid residues 44 and 141; 78 and 175; (as shown in SEQ ID NO: 11). The corresponding polynucleotides encoding the zcyto25 polypeptide regions, domains, motifs, residues and sequences described herein are as shown in SEQ ID NO: 11. The predicted helices in zcyto25 (as shown in SEQ ID N0:11) are: residues 59-73 (helix A); residues 85-94 (helix B); residues 98-115 (helix C); residues 121-143 (helix D); residues 147-169 (helix E); residues 1747-94 (helix F).<br>
Detailed mutational analysis of murine IL-2 (Zurawski et al., EMBO J. 12:5113-5119, 1993) shows residues in helices A and C are important for binding to IL-2R8; critical residues are Asp34, Asn99, and Asn103- Multiple residues within murine IL-<br><br>
2 loop A/B and helix B are important for IL-2Ra binding, while only a single residue, Glni4i in helix D, is vital for binding with IL-2Ra. Similarly, helices A and C are sites of interaction between IL-4 and IL-4Ra (the structurally similar to IL-2Ra), and residues within helix D are vital for lL-2Ra interaction ("Wang et al., Proc. Natl. Acad. Sci. USA 94:1657-1662, 1997; Kruse et al., EMBO J. 11:3237-3244, 1992). In particular, the mutation Tyr[24 to Asp in human IL-4 creates an antagonist, which binds with IL-4Ra but not IL-2Ra and dierefore cannot signal (Kruse et al. ibid. 1992).<br>
Four-helical bundle cytokines are also grouped by the length of their component helices. "Long-helix" form cytokines generally consist of between 24-30 residue helices, and include lL-6, ciliary neutrotrophic factor (CNTF), leukemia inhibitory factor (LIF) and human growth hormone (hGH). "Short-helix" form cytokines generally consist of between 18-21 residue helices and include IL-2, rL-4 and GM-CSF. Studies using CNTF and IL-6 demonstrated that a CNTF helix can be exchanged for the equivalent helix in IL-6, conferring CTNF-binding properties to the chimera. Thus, it appears that functional domains of four-helical cytokines are determined on the basis of structural homology, irrespective of sequence identity, and can maintain functional integrity in a chimera (Kallen et al., J. Biol. Chem. 274:11859-11867,1999). Therefore, the helical domains of zcyto20, zcyto21,2cyto22, zcyto24 and zcyto25 will be useful for preparing chimeric fusion molecules, particularly with other interferons to determine and modulate receptor binding specificity. Of particular interest are fusion proteins that combine helical and loop domains from interferons and cytokines such as MF-a, IL-10, human growth hormone.<br>
Zcyto20, zcyto21. zcyto22, zcyto24 and zcyto25 have been shown to form a complex with the orphan receptor designated zcytorl9. Zcytorl9 is described in a commonly assigned patent application PCT/USOl/44808, Zcyto22, 2cyto21, and zcyto24 have been shown to bind or signal through zcytorI9 as well, further supporting that zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 are members of the same family of cytokines. Zcytorl9 receptor is a class n cytokine receptor. Class II cytokine receptors usually bind to four-helix-bundle cytokines. For example, interIeukin-10 and the interferons bind receptors in this class (e.g., interferon-gamma receptor, alpha and beta chains and the interferon-alpha/beta receptor alpha and beta chains).<br>
Class H cytokine receptors are characterized by the presence of one or more cytokine receptor modules (CRM) in their extracellular domains. Other class n cytokine receptors include zcytorll (commonly owned US Patent No. 5,965,704), CRF2^ (Genbank Accession No. Z17227), IL-IOR (Genbank Accession No.s U00672 and NM„001558), DIRSl, zcytor? (commonly owned US Patent No. 5,945,511), and<br><br>
tissue factor. ZcytoTl9, like all known class n receptors except interferon-alpha/beta receptor alpha chain, has only a single class IICRM in its extracellular domain.<br>
Analysis of a human cDNA clone encoding Zcytorl9 (SEQ ID NO:26) revealed an open reading frame encoding 520 amino acids (SEQ ID NO:27) comprising a secretory signal sequence (residues 1 (Met) to 20 (Gly) of SEQ ID NO:27) and a mature zcytorl9 cytokine receptor polypeptide (residues 21 (Arg) to 520 (Arg) of SEQ ID NO:27) an extracellular Hgand-binding domain of approximately 206 amino acid residues (residues 21 (Arg) to 226 (Asn) of SEQ ID NO:27), a transmembrane domain of approximately 23 amino acid residues (residues 227 (Trp) to 249 (Trp) of SEQ ID NO:27), and an intracellular domain of approximately 271 amino acid residues (residues 250 (Lys) to 520 (Arg) of SHQ ID NO:27). Within the extracellular Hgand-binding domain, there are two fibronectin type lH domains and a hnker region. The first fibronecdn type HI domain comprises residues 21 (Arg) to 119 (Tyr) of SEQ ID NO:27, the linker comprises residues 120 (Leu) to 124 (Glu) of SEQ ID NO:27, and the second fibronectin type HI domain comprises residues 125 (Pro) to 223 (Pro) of SEQ ID NO:27. Thus, a polypeptide comprising amino acids 21 (Arg) to 223 (Pro) of SEQ ID NO:27 is considered a ligand binding fragment. In addition as typically conserved in class n receptors, there are conserved Tryptophan residues comprising residues 43 (Trp) and 68 (Trp) as shown in SEQ ID NO:27, and conserved Cysteine residues at positions 74, 82,195,217 of SEQ ID NO:27.<br>
In addition, a human cDNA clone encoding a Zcytorl9 variant with a 30 amino acid deletion was identified. This zcytorl9 variant (as shown in SEQ ID NO:23) comprises an open reading frame encoding 491 amino acids (SEQ ID NO:24) comprising a secretory signal sequence (residues 1 (Met) to 20 (Gly) of SEQ ID NO:24) and a m'ature zcytorl9 cytokine receptor polyptide (residues 21 (Arg) to 491 (Arg) of SEQ ID NO:24) an extracellular Hgand-binding domain of approximately 206 amino acid residues (residues 21 (Arg) to 226 (Asn) of SEQ ID NO:24, a transmembrane domain of approximately 23 amino acid residues (residues 227 (Trp) to 249 (Trp) of SEQ ID NO:24), and an intracellular domain of approximately 242 amino acid residues (residues 250 (Lys) to 491 (Arg) of SEQ ID NO:24). Within the extracellular Hgand-binding domain, there are two fibronectin type in domains and a linker region. The first fibronectin type HI domain comprises residues 21 (Arg) to 119 (Tyr) of SEQ ID NO:24, the linker comprises residues 120 (Leu) to 124 (Glu) of SEQ ID NO;24, and the second fibronectin type III domain is short, and comprises residues 125 (Pro) to 223 (Pro) of SEQ ID NO:24. Thus, a polypeptide comprising amino acids 21 (Arg) to 223 (Pro) of SEQ ID NO;24 is considered a Hgand binding fragment. In addition as typically conserved in class n receptors, there are conserved Tryptophan residues<br><br>
comprising residues 43 (Trp) and 68 (Trp) as shown in SEQ ID NO:24, and conserved Cysteine residues at positions 74, 82,195, 217 of SEQ ID NO:24.<br>
A truncated soluble form of the zcytorl9 receptor mRNA appears to be naturally expressed. Analysis of a human cDNA clone encoding the truncated soluble Zcytorl9 (SEQ ID NO:28) revealed an open reading frame encoding 211 amino acids (SEQ ID NO:29) comprising a secretory signal sequence (residues 1 (Met) to 20 (Gly) of SEQ ID NO:29) and a mature truncated soluble zcytorl9 receptor polyptide (residues 21 (Arg) to 211 (Ser) of SEQ ID NO:29) a truncated extracellular ligand-binding domain of approximately 143 amino acid residues (residues 21 (Arg) to 163 (Trp) of SEQ ID NO:29), no transmembrane domain, but an additional domain of approximately 48 amino acid residues (residues 164 (Lys) to 211 (Ser) of SEQ ID NO:29). Within the truncated extracellular ligand-binding domain, there are two fibronectin type IH domains and a linker region. The first fibronectin type HI domain comprises residues 21 (Arg) to 119 (Tyr) of SEQ ID NO:29, the linker comprises residues 120 (Leu) to 124 (Glu) of SEQ ID NO:29, and the second fibronectin type HI domain comprises residues 125 (Pro) to 163 (Trp) of SEQ ID NO:29. Thus, a polypeptide comprising amino acids 21 (Arg) to 163 (Trp) of SEQ ID NO-.29 is considered a ligand binding fragment. In addition as typically conserved in class II receptors, there are conserved Tryptophan residues comprising residues 43 (Trp) and 68 (Trp) as shown in SEQ ID NO:29, and conserved Cysteine residues in this tmncated soluble form of the zcytorl9 receptor are at positions 74, and 82 of SEQ ID NO:29.<br>
Zcytorl9 receptor is a member of the same receptor subfamily as the class n cytokine receptors, and receptors in this subfamily may associate to form homodimers that transduce a signal. Several members of the subfamily (e.g., receptors that bind interferon, IL-10, IL-19, and IL-TIF) combine with a second subunit (termed a P-subunit) to bind ligand and transduce a signal. However, in many cases, specific ^-subunits associate with a plurality of specific cytokine receptor subunits. For example, class H cytokine receptors, such as, zcytorU (US Patent No. 5,965,704) and CRF2-4 receptor heterodimerize to bind the cytokine IL-TIF (See, WIPO publication WO 00/24758; Dumontier et al., J. Immunol. 164:1814-1819, 2000; Spencer, SD et al., L Exp. Med. 187:571-578, 1998; Gibbs, VC and Pennica Gene 186:97-101.1997 (CRF2-4 cDNA); Xie, MH et al., J, Biol. Chem. 275: 31335-31339, 2000). IL-lOp receptor is believed to be synonymous with CRF2-4 (Dumoutier, L. et al., Proc. Nat']. Acad. Sci. 97:10144-10149. 200Q:Liu Y et al. J Immunol. 152: 1821-1829, 1994 (IL-IOR cDNA). Therefore, one could expect that zcyto20, zcyto21, 2cyto22, zcyto24 and zcyto25 would bind either monomeric, homodimeric, heterodimeric and multimeric zcytorl9 receptors. Experimental evidence has identified CRF2-4 (SEQ ID NOS: 40 and 41) as the putative<br><br>
binding partner for zcytorl9 which provides further support that zcyto20, zcyto21, zcyto22, zcyto24 and KytQ25 play an important role in U\e immunomodulatory system, affecting physiologies such as the innate immune system and the inflammatory response system.<br>
Localizing the expression of a receptor for a ligand/receptor pair may have significance for identifying the target cell or tissue at which the ligand acts. This is particularly useful when the receptor/ligand complex involves a hetercdimeric receptor in which one of the subunits is expressed widely and another the of the subunits is expressed in a limited manner, either spatially or temporally restricted. Using in siiu hybridization expression of zcytorl9 has been identified in a skin carcinoma sample, where the cancerous granular epithelium was strongly positive, while no positive signal is observed in normal skin. Other tissues identified as expressing zcytorl9 included fetal liver, where signal was observed in a mixed population of mononuclear cells in sinusoid spaces; in lung expression was observed in type n alveolar epithelium; and in macrophage-Iike mononuclear cells in the interstitial tissue. Northern analysis of zcytorl9 identified expression of a -4.5 kb transcript which was in greatest in heart, skeletal muscle, pancreas, and prostate tissue, in addition to in the Burkitt's lymphoma (RAJT) cell line and S'W-480 colorectal carcinoma cell line.<br>
The present invention provides polynucleotide molecules, including DNA and RNA molecules, that encode the zcyto20, zcyto21, zcyto22. 2cyto24, and zcyto25 polypeptides disclosed herein. Those skilled in the art will readily recognize that, in view of the degeneracy of the genetic code, considerable sequence variation is possible among these polynucleotide molecules. SEQ ID N0:3 is a degenerate DNA sequence that encompasses all DNAs that encode the zcyto20 polypeptide of SEQ ID N0:2. Those skilled in the art will recognize that the degenerate sequence of SEQ ID N0:3 also provides all RNA sequences encoding SEQ ID N0:2 by substituting U for T. Thus, zcyto20 polypeptide-encoding polynucleotides comprising nucleotide 1 or 64 to nucleotide 615 of SEQ ID N0:3 and their RNA equivalents are contemplated by the present invention. Table 3 sets forth the one-letter codes used within SEQ ID N0;3 to denote degenerate nucleotide positions. "Resolutions" are the nucleotides denoted by a code letter. "Complement" indicates tiie code for the complementary nucleotide(s). For example, the code Y denotes either C or T, and its complement R denotes A or G, with A being complementary to T. and G being complementary to C.<br>
SEQ ID NO:46 is a degenerate DNA sequence that encompasses all DNAs that encode the zcyto22 polypeptide of SEQ ID N0:7. Those skilled in the art will recognize that the degenerate sequence of SEQ ID NO:46 also provides all RNA<br><br>
sequences encoding SEQ ID N0:7 by substituting U for T. Thus, zcyto22 polypeptide-encoding polynucleotides comprising nucleotide 1 or 64 to nucleotide 615 of SEQ JD NO:46 and their RNA equivalents are contemplated by the present invention. Table 3 sets forth the one-letter codes used within SEQ ID NO;46 to denote degenerate nucleotide positions. "Resolutions" are the nucleotides denoted by a code letter. "Complement" indicates the code for the complementary nucleotide(s). For example, the code Y denotes either C or T, and its complement R denotes A or G, with A being complementary to T, and G being complementary to C.<br><br><br><br><br><br>
One of ordinary skill in Uie art will appreciate that some ambiguity is introduced in determining a degenerate codon, representative of all possible codons encoding each amino acid. For example, the degenerate codon for serine (WSN) can, in some circumstances, encode arginine (AGR), and the degenerate codon for arginine (MGN) can, in some circumstances, encode serine (AGY). A similar relationship exists between codons encoding phenylalanine and leucine. Thus, some polynucleotides encompassed by the degenerate sequence may encode variant amino acid sequences, but one of ordinary skill in the art can easily identify such variant sequences by reference to the amino acid sequence of SEQ ID N0:2 and SEQ ID N0:7. Variant sequences can be readily tested for functionality as described herein.<br>
One of ordinary skill in the art will also appreciate that different species can exhibit "preferential codon usage." In general, see, Grantham, et al., Nuc. Acids Res. 8:1893-912. 1980; Haas, et al. Curr. Biol. 6:315-24, 1996; Wain-Hobson, et al., Gene 13:355-64, 1981: Grosiean and Fiers. Gene 18:199-209. 1982; Holm, Nuc. Acids Res. 14:3075-87, 1986; Ikemura, J. Mol. Biol. 158:573-97, 1982. As used herein, the term "preferential codon usage" or "preferential codons" is a term of art referring to protein translation codons that are most frequently used in cells of a certain species, thus favoring one or a few representatives of the possible codons encoding each amino acid (See Table 3). For example, the amino acid Threonine (Thr) may be encoded by ACA, ACC, ACG, or ACT, but in mammalian cells ACC is the most commonly used codon; in other species, for example, insect cells, yeast, viruses or bacteria, different Thr codons may be preferential. Preferential codons for a particular species can be introduced into the polynucleotides of the present invention by a variety of methods known in the art. Introduction of preferential codon sequences into recombinant DNA can, for example, enhance production of the protein by making protein translation moi-e efficient within a particular cell type or species. Therefore, the degenerate codon sequence disclosed in SEQ ID N0S:3 and 46 serves as a template for optimizing expression of polynucleotides in various cell types and species commonly used in the art and disclosed herein. Sequences containing preferential codons can be tested and optimized for expression in various species, and tested for functionality as disclosed herein.<br>
As previously noted, the isolated polynucleotides of the present invention include DNA and RNA. Methods for preparing DNA and RNA are well known in the art. In. general, RNA is isolated from a tissue or cell that produces large amounts of 2cyto20, zcyto21, zcyto22, zcyto24 and zcyto25 RNA. Such tissues and cells are identified by Northern blotting (Thomas, Proc. Natl. Acad. Sci. USA 77:5201, 1980), or by screening conditioned medium from various cell types for activity on<br><br>
target cells or tissue. Once the activity or RNA producing cell or tissue is identified, total RNA can be prepared using guanidinium isothiocyanate extraction followed by isolation by centrifugation in a CsCl gradient (Chirgwin et al.. Biochemistry 18:52-94, 1979). Poly (A)+ RNA is prepared from total RNA using the method of Aviv and Leder fProc. Natl. Acad. Sci. USA 69:1408-12. 1972). Complementary DNA (cDNA) is prepared from poly(A)+ RNA using known methods. In the alternative, genomic DNA can be isolated. Polynucleotides encoding zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 polypeptides are then identified and isolated by, for example, hybridization orPCR.<br>
A full-length clones encoding zcyto20, zcyto2I, zcyto22, zcyto24 and zcyto25 can be obtained by conventional cloning procedures. Complementary DNA (cDNA) clones are preferred, although for some applications (e.g., expression in transgenic animals) it may be preferable to use a genomic clone, or to modify a cDNA clone to include at least one genomic intron. Methods for preparing cDNA and genomic clones are well known and within the level of ordinary skill in the art, and include the use of the sequence disclosed herein, or parts thereof, for probing or priming a library. Expression libraries can be probed with antibodies to Zcytorl9 receptor fragments, or other specific binding partners.<br>
The present invention further provides counterpart polypeptides and polynucleotides from other species (orthologs). These species include, but are not limited to manmialian, avian, amphibian, reptile, fish, insect and other vertebrate and invertebrate species. Of particular interest are zcyto20, zcyto2I, 2cyto22, zcyto24 and zcyto25 polypeptides from other mammalian species, including murine, porcine, ovine, bovine, canine, feline, equine, and other primate polypeptides. Orthologs of human zcyto20," zcyto21, and zcyto22 can be cloned using information and compositions provided by the present invention in combination with conventional cloning techniques. For example, a cDNA can be cloned using mRNA obtained from a tissue or cell type that expresses zcyto20, zcyto21, and zcyto22 as disclosed herein. Suitable sources of mRNA can be identified by probing Northern blots with probes designed from the sequences disclosed herein. A library is then prepared from mRNA of a positive tissue or cell line. Zcyto20-, zcyto21-, and zcyto22-encoding cDNA can then be isolated by a variety of methods, such as by probing with a complete or partial human cDNA or with one or more sets of degenerate probes based on the disclosed sequences. A cDNA can also be cloned using the polymerase chain reaction, or PCR (Mullis, U.S. Patent No. 4,683,202), using primers designed from tiie representative human zcyto20sequence disclosed herein. Within an additional method, the cDNA library can be used to transform or Eransfect host cells, and expression of the cDNA of interest can be detected<br><br>
with an antibody to zcyto20 polypeptide, binding studies or activity assays.   Similar techniques can also be applied to the isolation of genomic clones.<br>
Those skilled in the art will recognize that the sequence disclosed in SEQ ED N0S:1, 4, and 6, respectively, represent single alleles of human zcyto20, zcyto21, and zcyto22 bands, and that allelic variation and alternative splicing are expected to occur. Allelic variants of this sequence can be cloned by probing cDNA or genomic libraries from different individuals according to standard procedures. Allelic variants of the DNA sequence shown in SEQ ID N0:1, 4 and 6, including those containing silent mutations and those in which mutations result in amino acid sequence changes, are within the scope of the present invention, as are proteins which are allelic variants of SEQ ID NO:2, 5, and 7. cDNAs generated from alternatively spliced niRNAs, which retain the properties of the zcyto20, zcyto21, and zcyto22 polypeptides, are included within the scope of the present invention, as are polypeptides encoded by such cDNAs and mRNAs. Allelic variants and splice variants of these sequences can be cloned by probing cDNA or genpmic libraries from different individuals or tissues according to standard procedures known in the art.<br>
The present invention also provides reagents which will find use in diagnostic applications. For example, the zcyto20, zcyto21, and zcyto22 genes, probes comprising zcyto20, zcyto21, and zcylo22 DNA or RNA or a subsequence thereof can be used to determine if the zcyto20, zcyto2l, and zcyto22 gene is present on a human chromosome, such as chromosome 19, or if a gene mutation has occurred. Zcyto20, zcyto21, and 2cyto22 are located at the ql3.13 region of chromosome 19. Detectable chromosomal aberrations at the zcyto20, 2cyto21, and zcyto22 gene locus include, but are not limited to, aneuploidy, gene copy number changes, loss of heterogeneity (LOH), translocations, insertions, deletions, restriction site changes and rearrangements. Such aberrations can be detected using polynucleotides of the present invention by employing molecular genetic techniques, such as restriction fragment length polymorphism (RFLP) analysis, short tandem repeat (STR) analysis employing PCR techniques, and other genetic linkage analysis techniques known in the art (Sambrook et al., ibid: Ausubelet. al.,ibid; Marian. Chest 108:255-65.1995).<br>
The precise knowledge of a gene's position can be useful for a number of purposes, including: 1) determining if a sequence is part of an existing contig and obtaining additional surrounding genetic sequences in various forms, such as YACs, BACs or cDNA clones; 2) providing a possible candidate gene for an inheritable disease which shows linkage to the same chromosomal region; and 3) cross-referencing model organisms, such as mouse, which may aid in determining what function a particular gene might have.<br><br>
For example, Delague et al., (Am. J. Hum. Genet. 67: 236-243, 2000) identified that Charcot-Marie-Tooth disease is localized to 19ql3.l-13.3 (Delague et al.. Am. J. Hum. Genet. 67: 236-243,2000).<br>
A diagnostic could assist physicians in detennining the type of disease and appropriate associated therapy, or assistance in genetic counseling. As such, the inventive anti-zcyto20 antibodies, polynucleotides, and polypeptides can be used for the detection of zcyto20 polypeptide, mRNA or anti- zcyto20 antibodies, thus serving as markers and be directly used for detecting or genetic diseases or cancers, as described herein, using methods known in the art and described herein. Further, zcyto20, zcyto21, and zcyto22 polynucleotide probes can be used to detect abnormalities or genotypes associated with chromosome 19ql3.13 deletions and translocations associated with human diseases or other translocations involved with malignant progression of tumors or other 19ql3.13 mutations, which are expected to be involved in chromosome rearrangements in malignancy; or in other cancers. Similarly, zcyto20 polynucleotide probes can be used to detect abnormalities or genotypes associated with chromosome 19ql3.l3 trisomy and chromosome loss associated with human diseases or spontaneous abortion. Thus, zcyto20, zcyto21, and zcyto22 polynucleotide probes can be used to detect abnormalities or genotypes associated with these defects.<br>
In general, the diagnostic methods used in genetic linkage analysis, to detect a genetic abnormality or aberration in a patient, are known in the art. Analytical probes will be generally at least 20 nt in length, although somewhat shorter probes can be used (e.g., 14-17 nt). PCR primers are at least 5 nt in length, preferably 15 or more, more preferably 20-30 nt. For gross analysis of genes, or chromosomal DNA, a zcyto20 polynucleotide probe may comprise an entire exon or more. Exons are readily deterrairted by one of skill in the art by comparing zcyto20, zcyto21, and 2cyto22 sequences (SEQ ID NOS: 1, 4, and 6, respectively) with the genomic DNA for zcyto20, zcyto21, and zcyto22. In general, the diagnostic methods used in genetic linkage analysis, to detect a genetic abnormality or aberration in a patient, are known in the art. Most diagnostic methods comprise the steps of (a) obtaining a genetic sample from a potentially diseased patient, diseased patient or potential non-diseased carrier of a recessive disease allele; (b) producing a first reaction product by incubating the genetic sample with a zcyto20 polynucleotide probe wherein the polynucleotide will hybridize to complementary polynucleotide sequence, such as in RFLP analysis or by incubating the genetic sample- with sense and anlisense primers in a PCR reaction under appropriate PCR reaction conditions; (iii) Visualizing the first reaction product by gel electrophoresis and/or other known method such as visualizing the first reaction product with a zcyto20 polynucleotide probe wherein the polynucleotide will hybridize<br><br>
to the complementary polynucleotide sequence of the first reaction; and (iv) comparing the visualized first reaction product to a second control reaction product of a genetic sample from wild type patient. A difference between the first reaction product and the control reaction product is indicative of a genetic abnormality in the diseased or potentially diseased patient, or the presence of a heterozygous recessive carrier phenotype for a non-diseased patient, or the presence of a genetic defect in a tumor from a diseased patient, or the presence of a genetic abnormality in a fetus or pre-implantation embryo. For example, a difference in restriction fragment pattern, length of PCR products, length of repetitive sequences at the zcyto20 genetic locus, and the hke, are indicative of a genetic abnormality, genetic aberration, or allelic difference in comparison to the normal wild type control. Controls can be from unaffected family members, or unrelated individuals, depending on the test and availability of samples. Genetic samples for use within the present invention include genomic DNA, mRNA, and cDNA isolated form any tissue or other biological sample from a patient, such as but not limited to, blood, saliva, semen, embryonic cells, amniotic fluid, and the like. The polynucleotide probe or primer can be RNA or DNA, and will comprise a portion of SEQ ED N0:1, the complement of SEQ ID NO:l, or an RNA equivalent thereof. Such methods of showing genetic linkage analysis to human disease phenotypes are well known in the art. For reference to PCR based methods in diagnostics see see, generally, Mathew (ed.), Protocols in Human Molecular Genetics (Humana Press, Inc. 1991), White (ed.), PCR Protocols: Current Methods and Applications (Humana Press, Inc. 1993), Cotter (ed.), Molecular Diagnosis of Cancer (Humana Press, Inc. 1996), Hanausek and Walaszek (eds.). Tumor Marker Protocols (Humana Press, Inc. 1998), Lo (ed.), Clinical Applications of PCR (Humana Press, Inc. 1998), and Meltzer (ed.), PCR in Bioanalysis (Humana Press, Inc. 1998)).<br>
Mutations associated with the zcyto20, zcyto21, and zcyto22 locus can be detected using nucleic acid molecules of the present invention by employing standard methods for direct mutation analysis, such as restriction fragment length polymorphism analysis, short tandem repeat analysis employing PCR techniques, amplification-refractory mutation system analysis, single-strand conformation polymorphism detection, RNase cleavage methods, denaturing gradient gel electrophoresis, fluorescence-assisted mismatch analysis, and other genetic analysis techniques known in the art (see, for example, Mathew (ed.). Protocols in Human Molecular Genetics. (Humana Press, Inc. 1991), Marian, Chest 108:255 (1995), Coleman and Tsongalis, Molecular Diagnostics (Human Press, Inc. 1996), Elles (ed.) Molecular Diagnosis of Genetic Diseases (Humana Press, Inc. 1996), Landegren (ed.). Laboratory Protocols for Mutation Detection (Oxford University Press 1996), Birren et<br><br>
al (eds.), Genome Analysis, Vol. 2: Detecting Genes (Cold Spring Harbor Laboratory<br>
Press 1998), Dracopoli et al (eds.), Current Protocols in Human Genetics (John Wiley<br>
&amp; Sons 1998), and Richards and Ward, "Molecular Diagnostic Testing," in Principles<br>
of Molecular Medicine, pages 83-88 (Humana Press, Inc. 1998)). Direct analysis of an<br>
zcyto20 gene for a mutation can be performed using a subject's genomic DNA.<br>
Methods for amplifying genomic DNA, obtained for example from peripheral blood<br>
lymphocytes, are well-known to those of skill in the art (see, for example, Dracopoli et<br>
al. (eds.). Current Protocols in Human Genetics, at pages 7.1.6 to 7.1.7 (John Wiley &amp;<br>
Sons 1998)).<br>
Within embodiments of the invention, isolated 2cyto20-encoding nucleic<br>
acid molecules can hybridize under stringent conditions to nucleic acid molecules<br>
having the nucleotide sequence of SEQ K) N0:1, to nucleic acid molecules having the<br>
nucleotide sequence of nucleotides 64 to 618 of SEQ ID N0:1, or to nucleic acid<br>
molecules having a nucleotide sequence complementary to SEQ ID N0:1.  In general,<br>
stringent conditions are selected to be about 5°C lower than the thermal melting point (Tjj^) for the specific sequence at a defined ionic strength and pH.   The T^ is the<br>
temperature (under defined ionic strength and pH) at which 50% of the target sequence hybridizes to a perfectly matched probe. Within embodiments of the invention, isolated zcyto22-encoding nucleic acid molecules can hybridize under stringent conditions to nucleic acid molecules having the nucleotide sequence of SEQ ID N0:6, to nucleic acid molecules having the nucleotide sequence of nucleotides 64 to 618 of SEQ ID N0:6, or to nucleic acid molecules having a nucleotide sequence complementary to SEQ ID NO:6.<br>
A pair of nucleic acid molecules, such as DNA-DNA, RNA-RNA and DNA-RNA, can hybridize if the nucleotide sequences have some degree of complementarity. Hybrids can tolerate mismatched base pairs in the double helix, but the stability of the hybrid is influenced by the degree of mismatch. The T^ of the mismatched hybrid decreases by 1°C for every 1-1.5% base pair mismatch. Varying the stringency of the hybridization conditions allows control over the degree of mismatch that will be present in the hybrid. The degree of stringency increases as the hybridization temperature increases and the ionic strength of the hybridization buffer decreases.<br>
It is well within the abilities of one skilled in the art to adapt these conditions for use with,a particular polynucleotide hybrid. The Tm for a specific target sequence is the temperature (under defined conditions) at which 50% of the target sequence will hybridize to a perfectly matched probe sequence. Those conditions which influence the T^, include, the size and base pair content of the polynucleotide<br><br>
probe, the ionic strength of the hybridization solution, and the presence of destabilizing agents in the hybridization solution. Numerous equations for calculating Tm are known in the art, and are specific for DNA, RNA and DNA-RNA hybrids and polynucleotide probe sequences of varying length (see, for example, Sambrook et al, Molecular Cloning: A Laboratory Manual, Second Edition (Cold Spring Harbor Press 1989); Ausubel et al., (eds.). Current Protocols in Molecular Biology ("John Wiley and Sons, Inc. 1987); Berger and Kimmel (eds.), Guide to Molecular Cloning Techniques, (Academic Press, Inc, 1987); and Wetmur, Crit. Rev. Biochem. Mol. Biol. 25:227 (1990)). Sequence analysis software such as OUGO 6.0 (LSR; Long Lake, MN) and Primer Premier 4.0 (Premier Biosoft International; Palo Alto, CA), as well as sites on the Internet, are available tools for analyzing a given sequence and calculating Tm based on user defined criteria. Such programs can also analyze a given sequence under defined conditions and identify suitable probe sequences. Typically, hybridization of longer polynucleotide sequences, &gt;50 base pairs, is performed at temperatures of about 20-25°C below the calculated Tm. For smaller probes, 
Following hybridization, the nucleic acid molecules can be washed to remove non-hybridized nucleic acid molecules under stringent conditions, or under highly stringent conditions. Typical stringent washing conditions include washing in a solution of 0.5x - 2x SSC with 0.1% sodium dodecyl sulfate (SDS) at 55 - 65°C. That is, nucleic acid molecules encoding a variant zcyto20, zcyto21, and zcyto22 polypeptides hybridize with a nucleic acid molecule having the nucleotide sequence of SEQ ID N0S;1, 4, and 6, respectively (or its complement) under stringent washing conditions, in which the wash stringency is equivalent to 0.5x - 2x SSC with 0.1% SDS at 55 - 65'C, including 0.5x SSC with 0.1% SDS at 55X, or 2x SSC with 0.1% SDS at 65°C. One of skill in the art can readily devise equivalent conditions, for example, by substituting SSPE for SSC in the wash solution.<br>
Typical highly stringent washing conditions include washing in a solution of O.lx - 0.2x SSC with 0.1% sodium dodecyl sulfate (SDS) at 50 - 65°C. In other words, nucleic acid molecules encoding a variant zcyto20 polypeptide hybridize with a nucleic acid molecule having the nucleotide sequence of SEQ ID N0:1 (or its complement) under highly stringent washing condirions, in which the wash stringency is equivalent to O.lx.- (L2x SSC with 0.1% SDS at 50 - 65°C, including O.lx SSC with 0.1% SDS at 50'C, or0.2x SSC with 0.1% SDS at 65°C.<br>
The present invention also provides isolated zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 polypeptides that have a substantially similar sequence identity to<br><br>
the polypeptides of SEQ ID N0S:2, 5, 7, 9, 11, respectively, or their orthologs. The term "substantially similar sequence identity" Is used herein to denote polypeptides comprising at least 70%, at least 80%, at least 90%, at least 95%, OT greater than 95% sequence identity to the sequences shown in SEQ ID N0S:2, 5, 7, 9, U, respectively, or their orthologs. The present invention also includes polypeptides that comprise an amino acid sequence having at least 70%, at least 80%, at least 90%, at least 95% or greater than 95%, 96%, 97%, 98%, 99% sequence identity to the sequence of amino acid residues 1 to 205 or 21 to 205 of SEQ ID N0:2 or SEQ ID NO:7. The present invention further includes nucleic acid molecules that encode such polypeptides. Methods for determining percent identity are described below.<br>
The present invention also contemplates variant 2cyto20, zcyto21, and zcyto22 nucleic acid molecules that canbe identified using two criteria: a determination of the similarity between the encoded polypeptide with the amino acid sequence of SEQ ID N0S:2, 5, 7, 9, 11, respectively, and/or a hybridization assay, as described above. Such zcyto20 variants include nucleic acid molecules: (1) that hybridize with a nucleic acid molecule having the nucleotide sequence of SEQ ID NOS:I, 4, 6, 8, 10, respectively (or its complement) under stringent washing conditions, in which the wash stiingency is equivalent to 0.5x - 2x SSC with 0.1% SDS at 55 - 65^; or (2) that encode a polypeptide having at least 70%, at least 80%, at least 90%, at least 95% or greater than 95% sequence identity to the amino acid sequence of SEQ ID N0:2. Alternatively, zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 variants can be characterized as nucleic acid molecules: (1) that hybridize with a nucleic acid molecule having the nucleotide sequence of SEQ ID N0S:1, 4, 6, 8, 10, respectively (or its complement) under highly stringent washing conditions, in which the wash stringency is equivalent to O.lx - 0.2x SSC with 0.1% SDS at 50 - 65°C; and (2) that encode a polypeptide having at least 70%, at least 80%, at least 90%, at least 95% or greater than 95% sequence identity to the amino acid sequence of SEQ ID N0S:2, 5, 7, 9, 11, respectively.<br>
Percent sequence identity is determined by conventional methods. See, for example, Altschul et al.. Bull. Math. Bio. 4^:603 (1986), and Henikoff and Henikoff, Proc. Natl. Acad. Sci. USA 89:10915 (1992). Briefly, two amino acid sequences are aligned to optimize the alignment scores using a gap opening penalty of 10, a gap extension penalty of 1, and the "BLOSUM62" scoring matiix of Henikoff and Henikoff (ibid.) as shown in Table 4 (amino acids are indicated by the standard one-letter codes).<br><br><br><br><br><br>
Those skilled in the art appreciate that there are many established algorithms available to align two amino acid sequences. The "FASTA" similarity search algorithm of Pearson and Lipman is a suitable protein alignment method for examining the level of identity shared by an amino acid sequence disclosed herein and the amino acid sequence of a putative variant zcyto20. The FASTA algorithm is described by Pearson and Lipman, Proc. Nat'l Acad. Sci. USA 85:2444 (1988), and by Pearson, Meth.Enzvmol. 183:63 (1990).<br>
Briefly, FASTA first characterizes sequence similarity by identifying regions shared by the query sequence (e.g., SEQ ID N0:2) and a test sequence that have either the highest density of identities (if the ktup variable is 1) or pairs of identities (if ktQp=2), without considering conservative amino acid substitutions, insertions, or deletions. The ten regions with the highest density of identities are then rescored by comparing the similarity of all paired amino acids using an amino acid substitution matrix, and the ends of the regions are "trimmed" to include only those residues that contribute to the highest score. If there are several regions with scores greater than the "cutoff" value (calculated by a predetermined formula based upon the length of the sequence and the ktup value), then the trimmed initial regions are examined to determine whether the regions can be joined to form an approximate alignment with gaps. Finally, the highest scoring regions of the two amino acid sequences are aligned using a modification of the Needleman-Wunsch-Sellers algorithm (Needleman and Wunsch, J. Mol. Biol. 48:444 (1970); Sellers, SIAM J. ADD!. Math. 26:787 (1974)), which allows for amino acid insertions and deletions. Preferred parameters for FASTA analysis are: ktup=l, gap opening penalty=10, gap extension penalty=l, and substitution matrix=BLOSUM62. These parameters can be introduced into a FASTA program-by modifying the scoring matrix file ("SMATRIX"), as explained in Appendix 2 of Pearson, Meth. Enzvmol. 183:63 (1990).<br>
FASTA can also be used to determine the sequence identity of nucleic acid molecules using a ratio as disclosed above. For nucleotide sequence comparisons, the ktup value can range between one to six, preferably from three to six, most preferably three, with other parameters set as default.<br>
Variant zcyto20, zcyto21, and zcyto22 polypeptides or polypeptides with substantially similar sequence identity are characterized as having one or more amino acid substitutions, deletions or additions. These changes are preferably of a minor nature, that is conservative amino acid substitutions (see Table 5) and other substitudons that do not significantly affect the folding or activity of the polypepdde; small deletions, typically of one to about 30 amino acids; and amino- or carbcxyl-lerminal extensions, such as an amino-terminal methionine residue, a small linker<br><br>
peptide of up to about 20-25 residues, or an affinity tag. The present invention thus includes polypeptides of from about 154 to 235 amino acid residues that comprise a sequence that is at least 70%, preferably at least 90%, and more preferably 95%, 96%, 97%, 987o, 99% or more identical to the corresponding region of SEQ ID N0:2. Polypeptides comprising affinity tags can further comprise a proteolytic cleavage site between the zcyto20, zcyto2l, zcyto22, zcyto24 and zcyto25 polypeptide and the affinity tag. Preferred such sites include thrombin cleavage sites and factor Xa cleavage sites.<br><br>
Determination of amino acid residues that comprise regions or domains that are critical to maintaining structural integrity can be determined. Within these regions one can determine specific residues that will be more or less tolerant of change and maintain the overall tertiary structure of the molecule, Methods for analyzing sequence structure include, but are not limited to alignment of multiple sequences with<br><br>
high amino acid or nucleotide identity, secondary structure propensities, binary patterns, complementary packing and buried polar interactions (Barton, Current Opin. Struct. Biol. 5:372-376, 1995 and Cordes et al.. Current Qpin. Struct. Biol. 6:3-10, 1996). In general, when designing modifications to molecules or identifying specific fragments determination of structure will be accompanied by evaluating activity of modified molecules.<br>
Amino acid sequence changes are made in zcyto20, zcyto21, 2cyto22, zcyto24 and zcyto25 polypeptides so as to minimize disruption of higher order structure essential to biological activity. For ex.ample, where the zcyto20, zcyto21, zcyto22, zcyto24 and 2cyto25 polypeptide comprises one or more helices, changes in amino acid residues will be made so as not to disrupt the helix geometry and other components of the molecule where changes in conformation abate some critical function, for example, binding of the molecule to its binding partners. The effects of amino acid sequence changes can be predicted by, for example, computer modeling as disclosed above or determined by analysis of crystal structure (see, e.g., Lapthom et al., Nat. Struct. Biol. 2:266-268,1995). Other techniques that are well known in the art compare folding of a variant protein to a standard molecule (e.g., the native protein). For example, comparison of the cysteine pattern in a variant and standard molecules can be made. Mass spectrometry and chemical modification using reduction and alkylation provide methods for determining cysteine residues which are associated with disulfide bonds or are free of such associations (Bean et al.. Anal. Biochem. 201:216-226. 1992; Gray, Protein Sci. 2:1732-1748.1993; and Patterson et al.. Anal. Chem. 66:3727-3732.1994). It is generally believed that if a modified molecule does not have the same cysteine pad:em as the standard molecule folding would be affected. Another well known and accepted method for measuring folding is circular dichrosism (CD). Measuring and comparing the CD spectra generated by a modified molecule and standard molecule is routine (Johnson, Proteins 7:205-214, 1990). Crystallography is another well known method for analyzing folding and structure. Nuclear magnetic resonance (NMR), digestive peptide mapping and epitope mapping are also known methods for analyzing folding and structurally similariries between proteins and polypeptides (Schaanan et al.. Science 257:961-964.1992).<br>
A Hopp/Woods hydrophilicity profile of the zcyto20 protein sequence as shown in SEQ ID N0:2 can be generated (Hopp et al., Proc- Natl. Acad. Sci.78:3824-3S28, 1981; Hopp,- J.- Immun. MeCh. 88:1-18, 1986 and Triquier et al., Protein Engineering n.:153-169, 1998). The profile is based on a sliding six-residue window. Buried G, S, and T residues and exposed H, Y, and W residues were ignored. For example, in zcyto20, hydrophilic regions include amino acid residues 169 (Glu) to 174<br><br>
(Glu) of SEQ m NO: 2, amino acid residues 54 (Lys) to 59 (Ala) of SEQ ID NO: 2, amino acid residues 53 (Phe) to 58 (Asp) of SEQ ID NO: 2, amino acid residues 168 (Gin) to 173 (Lys) of SEQ ID NO: 2, and amino acid residues 154 (Pro) to 159 (Arg) of SEQ ID NO: 2.<br>
A HoppAVoods hydrophilicity profile of the 2cyto22 protein sequence as shown in SEQ ID N0:7 can be generated (Hopp et al., Proc. Natl. Acad. Sci.78:3824-3828, 1981; Hopp, J. Immun. Meth. 88:1-18, 1986 and Triquier et al.. Protein Engineering 11:153-169, 1998). The profile is based on a sliding six-residue window. Buried G, S, and T residues and exposed H, Y, and W residues were ignored. For example, in zcyto22, hydrophilic regions include amino acid residues 169 (Glu) to 174 (Glu) of SEQ ID NO: 7, amino acid residues 54 (Lys) to 59 (Ala) of SEQ ID NO: 7, amino acid residues 53 (Phe) to 58 (Asp) of SEQ ID NO: 7, amino acid residues 168 (Gin) to 173 (Lys) of SEQ ID NO: 7, and amino acid residues 154 (Pro) to 159 (Arg) of SEQ ID NO: 7.<br>
Those skilled in the art will recognize that hydrophilicity or hydrophobicity will be taken into account when designing modifications in the amino acid sequence of a 2cyto20, zcyto2i, zcyto22, zcyto24 and zcyto25 polypeptide, so as not to disrupt the overall structural and biological profile. Of particular interest for replacement are hydrophobic residues selected from the group consisting of Val, I^u and He or the group consisting of Met, Gly, Ser, Ala, Tyr and Trp.<br>
The identities of essential amino acids can also be inferred from analysis of sequence similarity between INF-a and members of the family of zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 (as shown in Tables 1 and 2). Using methods such as "FASTA" analysis described previously, regions of high similarity are identified within a familyof proteins and used to analyze amino acid sequence for conserved regions. An alternative approach to identifying a variant polynucleotide on the basis of structure is to determine whether a nucleic acid molecule encoding a potential variant zcyto20, zcyto21, zcyto22, zcyto24. and zcyto25 gene can hybridize to a nucleic acid molecule havingthenucleotidesequenceof SEQIDN0S:1,4,6, 8, or 10 as discussed above.<br>
Other methods of identifying essential amino acids in the polypeptides of the present invention are procedures known in the art, such as site-directed mutagenesis or alanine-scanning mutagenesis (Cunningham and Wells, Science 244:1081 (1989), Bass et al., Prgc. Nati Acad. Sci. USA 88:4498 (1991), Coombs and Corey, "Site-Directed Mutagenesis and Protein Engineering," in Proteins: Analysis and Design. Angeletti (ed.), pages 259-311 (Academic Press, hic. 1998)). In the latter technique, single alanine mutations are introduced at every residue in the molecule, and the resultant mutant molecules are tested for biological or biochemical activity as<br><br>
disclosed below to identify amino acid residues that are critical to the activity of the molecule. See also, Hilton etal, J. Biol. Chem. 271:4699 (1996).<br>
The present invention also includes functional fragments of zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 polypeptides and nucleic acid molecules encoding such functional fragments. A "functional" zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 or fragment thereof as defined herein is characterized by its proliferative or differentiating activity, by its ability to induce or inhibit specialized cell functions, or by its ability to bind specifically to an anti- zcyto20, zcyto21, zcyta22, zcyto24 and zcyto25 antibody or zcytorl9 receptor (either soluble or immobilized). As previously described herein, zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 polypeptides are characterized by a six-helical-bundle. Thus, the present invention further provides fusion proteins encompassing: (a) polypeptide molecules comprising one or more of the helices described above; and (b) functional fragments comprising one or more of these helices. The other polypeptide portion of the fusion protein may be contributed by another helical-bundle cytokine or interferon, such as INF-o, or by a non-native and/or an unrelated secretory signal peptide that facilitates secretion of the fusion protein.<br>
The zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 polypeptides of the present invention, including full-length polypeptides, biologically active fragments, and fusion polypeptides can be produced according to conventional techniques using cells into which have been introduced an expression vector encoding the polypeptide. As used herein, "cells into which have been introduced an expression vector" include both cells that have been directly manipulated by the introduction of exogenous DNA molecules and progeny thereof that contain the introduced DNA. Suitable host cells are thosecell types that can be transformed or ti-ansfected with exogenous DNA and grown in culture, and include bacteria, fungal cells, and cultured higher eukaryotic cells. Techniques for manipulating cloned DNA molecules and introducing exogenous DNA into a variety of host cells are disclosed by Sambrook et al.. Molecular Cloning: A Laboratory Manual, 2nd ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989, and Ausubel et al., eds., Current Protocols in Molecular Biology, John Wiley and Sons, Inc., NY, 1987.<br>
In general, a DNA sequence encoding a zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 polypeptide is operably linked to other genetic elements required for its expression, generally including a transcription promoter and terminator, within an expression vector. .The vector will also commonly contain one or more selectable markers and one or more origins of replication, although those skilled in the art will recognize that within certain systems selectable markers may be provided on separate vectors, and replication of the exogenous DNA may be provided by integration into the<br><br>
host cell genome. Selection of promoters, terminators, selectable markers, vectors and other elements is a matter of routine design within the level of ordinary skill in the art. Many such elements are described in the literature and are available through commercial suppliers.<br>
To direct a zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 polypeptide into the secretory pathway of a host cell, a secretory signal sequence (also known as a leader sequence, prepro sequence or pre sequence) is provided in the expression vector. The secretory signal sequence may be that of zcyto20, zcyto21, zcyto22, zcyto24 and 2cyto25, or may be derived from another secreted protein (e.g., t-PA; see, U.S. Patent No. 5,641,655) or synthesized de novo. The secretory signal sequence is operably linked to the 2cyto20, zcyto21, zcyto22, zcyto24 and zcyto25 DNA sequence, i.e., the two sequences are joined in the correct reading frame and positioned to direct the newly synthesized polypeptide into the secretory pathway of the host cell. Secretory sipal sequences are commonly positioned 5' to the DNA sequence encoding the polypeptide of interest, although certain signal sequences may be positioned elsewhere in the DNA sequence of interest (see, e.g., Welch et al., U.S. Patent No. 5,037,743; Holland et al., U.S. Patent No. 5,143,830),<br>
Cultured mammalian cells can be used as hosts within the present invention. Methods for introducing exogenous DNA into mammalian host cells include calcium phosphate-mediated transfection (Wigler et al., Cell 14:725,1978; Corsaro and Pearson. Somatic Cell Genetics 7:603.1981; Graham and Van derEb. Virology 52:456. 1973), electroporation (Neumann et al., EMBO J. 1:841-845, 1982), DEAE-dextian mediated transfection (Ausubel et al., ibid.), and liposome-mediated transfection (Hawley-Nelson et al., Focus 15:73, 1993; Ciccarone et al.. Focus 15:80, 1993). The production of recombinant polypeptides in cultured mammalian cells is disclosed, for example, by Levinson et al., U.S. Patent No. 4,713,339; Hagen et al., U.S. Patent No. 4,784,950; Palmiter et al., U.S. Patent No. 4,579,821; and Ringold, U.S. Patent No. 4,656,134. Suitable cultured mammahan cells include the COS-1 (ATCC No. CRL 1650), COS-7 (ATCC No. CRL 1651), BHK (ATCC No. CRL 1632), BHK 570 (ATCC No. CRL 10314), 293 (ATCC No. CRL 1573; Graham et al., J. Gen. Virol. 36:59-72. 1977) and Chinese hamster ovary (e.g. CHO-Kl. ATCC No. CCL 61; or CHO DG44, Chasin et al., Som. Cell. Molec. Genet 12:555, 19S6) cell Unes. Additional suitable cell lines are known in the art and available from public depositories such as jhe.American Type Culture Collection, Manassas, VA. In general, strong transcription promoters are preferred, such as promoters from SV-40 or cytomegalovirus. See, e.g., U.S. Patent No. 4,956,288. Other suitable promoters include those from metallothionein genes (U.S. Patent Nos. 4,579,821 and 4,601,978)<br><br>
and the adenovirus major late promoter. Expression vectors for use in mammalian cells include pZP-1 and pZP-9, which have been deposited with the American Type Culture Collection, Manassas, VA USA under accession numbers 98669 and 98668, respectively, and derivatives thereof.<br>
Drug selection is generally used to select for cultured manmialian cells into which foreign DNA has been inserted. Such cells are commonly referred to as "transfectants". Cells that have been cultured in the presence of Uie selective agent and are able to pass the gene of interest to their progeny are referred to as "stable transfectants." A preferred selectable marker is a gene encoding resistance to the antibiotic neomycin. Selection is carried out in the presence of a neomycin-type drug, such as G-418 or the like. Selection systems can also be used to increase the expression level of the gene of interest, a process referred to as "amplification." Amplification is carried out by culttiring transfectants in the presence of a low level of the selective agent and then increasing the amount of selective agent to select for cells that produce high levels of the products of the introduced genes. A preferred ampHfiable selectable marker is dihydrofolate reductase, which confers resistance to methotrexate. Other drug resistance genes (e.g. hygromycin resistance, multi-drug resistance, puromycin acetyltransferase) can also be used.<br>
The adenovirus system can also be used for protein production in vitro. By culturing adenovirus-infected non-293 cells under conditions where the cells are not rapidly dividing, the cells can produce proteins for extended periods of time. For instance, BHK cells are grown to confluence in cell factories, then exposed to the adenoviral vector encoding the secreted protein of interest. The cells are then grown under .serum-free conditions, which allows infected cells to survive for several weeks without significant cell division. In an alternative method, adenovirus vector-infected 293 cells can be grown as adherent cells or in suspension culture at relatively high cell density to produce significant amounts of protein (See Gamier et al., Cvtotechnol. 15:145-55, 1994). With either protocol, an expressed, secreted heterologous protein can be repeatedly isolated from the cell culture supernatant, lysate, or membrane fractions depending on the disposition of the expressed protein in the cell. Within the infected 293 cell production protocol, non-secreted proteins can also be effectively obtained.<br>
Insect cells can be infected with recombinant baculovirus, commonly derived from Auiogmpfia califomica nuclear polyhedrosis virus (AcNPV) according to methods known in the art. Within a preferred method, recombinant baculovirus is produced through the use of a transposon-based system described by Luckow et al. (J. Virol.   67:4566-4579,   1993).     This  system,   which   utilizes   transfer  vectors,   is<br><br>
commerciaily available in kit form (Bac-to-Bac™ kit; Life Technologies, RockviUe, MD). The transfer vector (e.g., pFastBacl™; Life Technologies) contains a Tn7 transposon to move the DNA encoding the protein of interest into a baculovirus genome maintained in E. coli as a large plasmid called a "bacmid." See, Hill-Perkins and Possee, J. Gen. Virol. 71:971-976, 1990; Bonning et al., J. Gen. Virol. 75:1551-1556, 1994; and Chazenbalk and Rapoport, J. Biol. Chem. 220:1543-1549, 1995. In addition, transfer vectors can include an in-frame fusion widi DNA encoding a polypeptide extension or affinity tag as disclosed above. Using techniques known in the art, a transfer vector containing a zcyto20-, zcyto2l-, zcyto22-, 2cyto24- and zcyto25-encoding sequence is transformed into E. coli host cells, and the cells are screened for bacmids which contain an interrupted lacZ gene indicative of recombinant baculovirus. The bacmid DNA containing the recombinant baculovims genome is isolated, using common techniques, and used to transfect Spodoptera frugiperda cells, such as Sf9 cells. Recombinant virus that expresses zcyto20, 2cyto21, zcyto22, zcyto24 and 2cyto25 protein is subsequently produced. Recombinant viral stocks are made by methods commonly used the art.<br>
For protein production, the recombinant virus is used to infect host cells, typically a cell line derived from the fall armyworm, Spodoptera frugiperda (e.g., Sf9 or Sf21 cells) or Trichoplusia ni (e.g., High Five^" cells; Invitrogen, Carlsbad, CA). See, for example, U.S. Patent No. 5,300,435. Serura-free media are used to grow and maintain the cells. Suitable media formulations are known in the art and can be obtained from commercial suppliers. The cells are grown up from an inoculation density of approximately 2-5 x 10* cells to a density of 1-2 x 10^ cells, at which time a recombinant viral stock is added at a multiplicity of infection (MOI) of 0.1 to 10, more typically near 3. Procedures used are generally known in the art.<br>
Other higher eukaryotic cells can also be used as hosts, including plant cells and avian cells. The use of Agrobacterium rhizogenes as a vector for expressing genes in plant cells has been reviewed by Sinkar et al., J. Biosci. (Bangalore") ll:47-58. 1987.<br>
Fungal cells, including ye^t cells, can also be used within the present invention. Yeast species of particular interest in this regard include Saccharomyces cerevisiae, Pichia pastoris, and Pichia methanolica. Methods for transforming S. cerevisiae cells with exogenous DNA and producing recombinant polypeptides therefrom are disclosed by, for example, Kawasaki, US. Patent No. 4,599,311; Kawasaki et al., U.S. Patent No. 4,931,373; Brake, U.S. Patent No. 4,870,008; Welch et al., U.S. Patent No. 5,037,743; and Murray et al., U.S. Patent No. 4,845,075. Transformed cells are selected by phenotype determined by the selectable marker,<br><br>
coramonly drug resistance or the ability to grow in the absence of a particular nutrient (e.g., leucine). A preferred vector system for use in Saccharomyces cerevisiae is the POTl vector system disclosed by Kawasaki et a). (U.S. Patent No. 4,931,373), which allows transformed cells to be selected by growth in glucose-containing media. Suitable promoters and terminators for use in yeast include those from glycolytic enzyme genes (see, e.g., Kawasaki, U.S. Patent No. 4,599,311; Kingsman et al., U.S. Patent No. 4,615,974; and Bitter, U.S. Patent No. 4,977,092) and alcohol dehydrogenase genes. See also U.S. Patents Nos. 4,990,446; 5,063,154; 5,139,936 and 4,661,454. Transformation systems for other yeasts, including Hansenula polymorpha, Schizosaccharomyces pombe, Kluyveromyces lactis, Kluyveromyces fragilis. Vstilago maydis, Pichia pastoris, Pichia methanolica, Pichia guillermondii and Candida maltosa are known in the art. See, for example, Gleeson et al., /. Gen. Microbiol. 1^:3459-3465, 1986; Cregg, U.S. Patent No. 4,882,279; and Raymond et al.. Yeast 14, 11-23, 1998. Aspergillus cells may be utilized according to the methods of McKnight et al., U.S. Patent No. 4,935,349. Methods for transforming Acremonium chrysogenum are disclosed by Sumino et al., U.S. Patent No. 5,162,228. Methods for transforming Neurospora are disclosed by Lambowitz, U.S. Patent No. 4,486,533. Production of recombinant proteins in Pichia methanolica is disclosed in U.S. Patents No. 5,716,808, 5,736,383, 5,854,039. and 5,888,768.<br>
Prokaryotic host cells, including strains of the bacteria Escherichia coli. Bacillus and other genera are also useftil host cells within the present invention. Techniques for transforming these hosts and expressing foreign DNA sequences cloned therein are well known in the art (see, e.g., Sambrook et al., ibid.). When expressing a zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 polypeptide in bacteria such as E. coli, the polypeptide may be retained in the cytoplasm, typically as insoluble granules, or may be directed to the periplasmic space by a bacterial secretion sequence. In the former case, the cells are lysed, and the granules are recovered and denatured using, for example, guanidine isothiocyanate or urea. The denatured polypeptide can then be refolded and dimerized by diluting the denaturant, such as by dialysis against a solution of urea and a combination of reduced and oxidized glutathione, followed by dialysis against a buffered saline solution. In the latter case, the polypeptide can be recovered from the periplasmic space in a soluble and functional form by disrupting the cells (by, for example, sonication or osmotic shock) to release the contents of the periplasmic space and recovering ,the protein, thereby obviating the need for denaturation and refolding.<br>
Transformed or transfected host cells are cultured according to :onventional   procedures   in   a   culture   medium   containing   nutrients   and   other<br><br>
components required for the growth of the chosen host cells. A variety of suitable media, including defined media and complex media, are known in the art and generally include a carbon source, a nitrogen source, essential amino acids, vitamins and minerals. Media may also contain such components as growth factors or serum, as required. The growth medium will generally select for cells containing the exogenously added DNA by, for example, drug selection or deficiency in an essentia] nutrient which is complemented by the selectable marker carried on the expression vector or co-transfected into the host cell. Liquid cultures are provided with sufficient aeration by conventional means, such as shaking of small flasks or sparging of fermentors.<br>
It is preferred to purify the polypeptides and proteins of the present invention to &gt;80% purity, more preferably to &gt;90% purity, even more preferably &gt;9S% purity, and particularly preferred is a pharmaceutically pure state, that is greater than 99.9% pure with respect to contaminating macromolecules, particularly other proteins and nucleic acids, and free of infectious and pyrogenic agents. Preferably, a purified polypeptide or protein is substantially free of other polypeptides or proteins, particularly those of animal origin.<br>
Expressed recombinant zcyto2Q, zcyto21, 2cytQ22, zcyto24 and zcyto25 proteins (including chimeric polypeptides and multimeric proteins) are purified by conventional protein purification methods, typically by a combination of chromatographic techniques. See, in general, Affinity Chromatography. Principles &amp; Methods. Pharmacia LKB Biotechnology, Uppsala, Sweden, 1988; and Scopes, Protein Purification: Principles and Practice. Springer-Verlag, New York, 1994. Proteins comprising a polyhistidine affinity tag (typically about 6 histidine residues) are purified by affinity chromatography on a nickel chelate resin. See, for example, Houchuli et al., Biorrechnol. 6: 1321-1325,1988. Proteins comprising a glu-glu tag can be purified by immunoaffinity chromatography according to conventional procedures. See, for example, Grussenmeyer et al., ibid. Maltose binding protein fusions are purified on an amylose column according to methods known in the art.<br>
2cyto20, zcyto2I, 2cyto22, zcyto24 and 2cyto25 polypeptides can also be prepared through chemical synthesis according to methods known in the art, including exclusive solid phase synthesis, partial solid phase methods, fragment condensation or classical solution synthesis. See, for example, Merrifield, J. Am. Chem. Soc. 85:2149, 1963; Stewart et a].. Solid Phase Peptide Synthesis (2nd edition). Pierce Chemical Co., .Rockford, IL, 1984; Bayer and Rapp, Chem. Pept. Prot. 3:3, 1986; and Atherton et al., Solid Phase Peptide Svnthesis: A Practical Approach, IRL Press, Oxford, 1989. In vitro synthesis is particularly advantageous for the preparation of smaller polypeptides.<br><br>
Using methods known in the art, zcyto20, 2cyto21, 2cyto22, zcyto24 and 2cyto25 proteins can be prepared as monomers or multimers; glycosylated or non-glycosylated; pegylated or non-pegylated; and may or may not include an initial methionine amino acid residue.<br>
Target cells for use in zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 activity assays include, without limitation, vascular cells (especially endothelial cells and smooth muscle cells), hematopoietic (myeloid and lymphoid) cells, liver cells (including hepatocytes, fenestrated endothelial cells, Kupffer cells, and Ito cells), fibroblasts (including human dermal fibroblasts and lung fibroblasts), fetal lung cells, articular synoviocytes, pericytes, chondrocytes, osteoblasts, and prostate epithelial cells. Endothelial cells and hematopoietic cells are derived from a common ancestral cell, the hemangioblast (Choi et al., Development 125:725-732,1998).<br>
Zcyto20, zcyto21, zcyto22. zcyto24 and zcyto25 proteins of the present invention are characterized by their activity, that is, modulation of the proliferation, differentiation, migration, adhesion, or metabolism of responsive cell types. Biological activity of zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 proteins is assayed using in vitro or in vivo assays designed to detect cell proliferation, differentiation, migration or adhesion; or changes in cellular metabolism (e.g., production of other growth factors or other macromolecules). Many suitable assays are known in the art, and representative assays are disclosed herein. Assays using cultured cells are most convenient for screening, such as for determining the effects of amino acid substitutions, deletions, or insertions. However, in view of the complexity of developmental processes (e.g., angiogenesis, wound healing), in vivo assays will generally be employed to confirm and further characterize biological activity. Certain in vitro models, such as the three-dimensional collagen gel matrix model of Pepper et al. (Biochem. Biophys. Res. Comm. 189:824-831, 1992), are sufficiently complex to assay histological effects. Assays can be performed using exogenously produced proteins, or may be carried oat in vivo or in vitro using cells expressing the polypeptide(s) of interest. Assays can be conducted using zcyto20 proteins alone or in combination with other growth factors, such as members of the VEGF family or hematopoietic cytokines (e.g., EPO, TPO, G-CSF, stem cell factor). Representative assays are disclosed below.<br>
Activity of zcyto20, zcyto21, zcyto23, zcyto24 and zcyto25 proteins can be measured in vitro using cultured cells or in vivo by administering molecules of the claimed invention to an. appropriate animal model. Assays measuring cell proliferation or differentiation are well known in the art. For example, assays measuring proliferadon include such assays as chemosensitivity to neutral red dye (Cavanaugh et al..  Investigational  New  Drugs  8:347-354,   1990),  incorporation  of radiolabelled<br><br>
nucleotides (as disclosed by, e.g., Raines and Ross, Methods Enzvmol. 109:749-773. 1985; Wahl et al., Mol. Cell Biol. 8:5016-5025, 1988; and Cook et al, Analytical Biochem. 179:1-7, 1989), incorporation of 5-bromo-2'-deoxyuridine (BrdU) in the DNA of proliferating cells (Porstmann et al., J. Immunol. Methods 82:169-179, 1985), and use of tetrazolium salts (Mosmann, J. Immunol. Methods 65:55-63, 1983; Alley et a].. Cancer Res. 48:589-601, 1988; Marshall et al.. Growth Reg. 5:69-84, 1995; and Scudiero et al., Cancer Res. 48:4827-4833, 1988). Differentiation can be assayed using suitable precursor cells that can be induced to differentiate into a more mature phenotype. Assays measuring differentiation include, for example, measuring cell-surface markers associated with stage-specific .expression of a tissue, enzymatic activity, functional activity or morphological changes (Watt, FASEB. 5:281-284,1991; Francis, Differentiation 57:63-75, 1994; Raes, Adv. Anim. Cell Biol. Technol. Bioprocesses. 161-171, 1989; all incorporated herein by reference).<br>
Zcyto20, 2cyto21, zcyto22, 2cyto24 and zcyto25 activity may also be detected using assays designed to measure zcyto20-, zcyto21-, zcyto22-, zcyto24- and zcyto25-induced production of one or more additional growth factors or other macromolecules. Preferred such assays include those for determining the presence of hepatocyte growth factor (HGF), epidermal growth factor (EGF), transforming growth factor alpha (TGFa), interleukin-6 (IL-6), VEGF, acidic fibroblast growth factor (aFGF), angiogenin, and other macromolecules produced by the liver. Suitable assays include mitogenesis assays using target cells responsive to the macromolecule of interest, receptor-binding assays, competition binding assays, immunological assays (e.g., ELISA), and other formats known in the art. Metalloprotease secretion is measured from treated primary human dermal fibroblasts, synoviocytes and chondrocytes. The relative levels of collagenase, gelatinase and stromalysin produced in response to culturing in the presence of a zcyto20, zcyto21, zcyto22, 2cyto24 and zcyto25 protein is measured using zymogram gels (Loita and Stetler-Stevenson, Cancer Biology 1:96-106, 1990). ProcoUagen/collagen synthesis by dermal fibroblasts and chondrocytes in response to a test protein is measured using ^H-proline incorporation into nascent secreted collagen. ^H-labeled collagen is visualized by SDS-PAGE followed by autoradiography (Unemori and Amento, J. Biol. Chem. 265: 10681-10685, 1990). Glycosaminoglycan (GAG) secretion from dermal fibroblasts and chondrocytes is measured using a 1,9-dimethylmethylene blue dye binding assay (Famdale et al., Biochim. Biophvs. Acta 883:173-177, 1986). Collagen and GAG assays are also carried out in the presence of IL-la or TGF-a to examine the ability of zcyto20 protein to modify the established responses to these cytokines.<br><br>
Certain members of the protein family comprising zcyto20, zcyto2I, 2cyto22, zcyto24 and zcyto25 have been shown to increase circulating monocyte numbers in vivo. Monocyte activation is important in both innate and adaptive immunity. For example, activation of monocytes has been shown to stimulate antigen presentation by several mechanisms. Antigen presentation promotes activation and proliferation of T-cells, both cytotoxic and helper T cells. The maturation and activation dendritic cells also promotes activation of T cells and both innate and adaptive immunity. Increases in activated monocytes and macrophages have also been shown to increase cytolytic activity. Therefore, zcyto20, 2cyto21, zcyto22, zcyto24 and zcyto25 will be useful as an anti-infectious agent, enhancing innate, cell-mediated and humoral immune responses. Increases in ICAM staining in CD14+ monocytes was seen suggesting that 2cyto20, zcyto21, 2cyto22, zcyto24 and zcyto25 play a role in monocyte activation. "While data show that family members promote an anti-viral response to virus, bacteria and parasites may also be affected.<br>
Monocyte activation assays are carried out (1) to look, for the ability of zcyto20, zcyto21, 2cyto22, zcylo24 and zcyto25 proteins to further stimulate monocyte activation, and (2) to examine the ability of zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 proteins to modulate attachment-induced or endotoxin-induced monocyte activation (Fuhlbrigge et al., J. Immunol. 138: 3799-3802, 1987). IL-la and TNFa levels produced in response to activation are measured by ELISA (Biosource, Inc. Camarillo, CA). Monocyte/macrophage cells, by virtue of CD 14 (LPS receptor), are exquisitely sensitive to endotoxin, and proteins with moderate levels of endotoxin-like activity will activate these cells.<br>
Increased levels of monocytes suggest that zcyto20, zcyto21, 2cylo22, zcyto24 and zcyto25 may have a direct effect on myeloid progenitor cells in the bone marrow. Increasing differentiation of myeloid progenitor cells to monocytes is essential in restoring inminunocompetency, for example, after chemotherapy. Thus, administration of zcyto20, zcyto2I, zcyto22, zcyto24 and zcyto25 to patients receiving chemotherapy could promote their recovery and ability to resist infection commonly associated with chemotherapy regimens. Thus, methods for expanding the numbers of monocytes or monocyte progenitor cells by either culturing bone marrow or peripheral blood cells with the molecules of the present invention such that there is an increase in the monocyte or monocyte progenitor cells for achieving this effect in vitro or ex vivo. The present invention ^so provides for the in vivo administi^tion of the molecules of the present invention to a mammal needing increased monocyte or monocyte progenitor cells. Increased monocyte and monocyte progenitor cells can be measured using methods well known to clinicians, physicians, and other persons skilled the art.<br><br>
Monocyte cells are included in the myeloid lineage of hematopoietic cells, so affects on other ceils in that lineage would not be unusual. For example, when a factor facilitates the differentiation or proliferation of one type of cell in the myeloid or lymphoid lineage, this can affect producdon of other cells with a common progenitor or stem cell.<br>
Hematopoietic acdvity of zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 proteins can be assayed on various hematopoietic ceils in culture. Preferred assays include primary bone marrow colony assays and later stage lineage-restricted colony assays, which are known in the art (e.g.. Holly et al., WIPO Publication WO 95/21920). Marrow cells plated on a suitable semi-solid medium (e.g., 50% methylcellulose containing 15% fetal bovine serum, 10% bovine serum albumin, and 0.6% PSN antibiotic mix) are incubated in the presence of test polypeptide, then examined microscopically for colony formation. Known hematopoietic factors are used as controls. Mitogenic activity of zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 polypeptides on hematopoietic cell lines can be measured as disclosed above.<br>
Cell migration is assayed essentially as disclosed by Kahler et al. (Arteriosclerosis. Thrombosis, and Vascular Biology 17:932-939, 1997). A protein is considered to be chemotactic if it induces migration of cells from an area of low protein concentration to an area of high protein concentration. A typical assay is performed using modified Boyden chambers with a polystiyrene membrane separating the two chambers (Transwell; Coming Costar Corp.). The test sample, diluted in medium containing 1% BSA, is added to the lower chamber of a 24-well plate containing Transwells. Cells are then placed on the Transwell insert that has been pretreated with 0.2% gelatin. Cell migration is measured after 4 hours of incubation at 37°C. Non-migrating cells are wiped off the top of the Transwell membrane, and cells attached to the lower face of the membrane are fixed and stained with 0.1% crystal violet. Stained cells are then extracted with 10% acetic acid and absorbance is measured at 600 nm. Migration is then calculated from a standard calibration curve. Cell migration can also be measured using the matrigel method of Grant et al. ("Angiogenesis as a component of epithehal-mesenchymal interactions" in Goldberg and Rosen, Epithelial-Mesenchvmal Interaction in Cancer. Birkhauser Verlag, 1995, 235-248; Baatout, Anticancer Research 17:451-456,1997).<br>
Cell adhesion activity is assayed essentially as disclosed by LaBeur et al. (J. Biol. Chem. 272:32798-32803, 1997). Briefly, microtiter plates are coated with \be test protein, non-specifjc sites are blocked with BSA, and cells (such as smooth muscle ceils, leukocytes, or endothelial cells) are plated at a density of approximately 10"^ - 10 cells/well. The wells are incubated at 37°C (typically for about 60 minutes), then non¬adherent cells are removed by gentle washing.    Adhered cells are quantilated by<br><br>
conventional methods (e.g., by staining with crystal violet, lysing the ceils, and determining the optical density of the lysate). Control wells are coated with a known adhesive protein, such as fibronectin or vitronectin.<br>
The activity of zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 proteins can be measured with a silicon-based biosensor microphysiometer that measures the extracellular acidification rate or proton excretion associated with receptor binding and subsequent physiologic cellular responses. An exemplary such device is the Cytosensor'^'^ Microphysiometer manufactured by Molecular Devices, Sunnyvale, CA. A variety of cellular responses, such as cell proliferation, ion transport, energy production, inflammatory response, regulatory and receptor activation, and the like, can be measured by this method. See, for example, McConnell et al.. Science 257:1906-1912, 1992; Pitchford et al., Meth. Enzvmol. 228:84-108, 1997; Arimilli et al., J, Immunol. Meth. 212:49-59, 1998; and Van Liefde et al., Eur. J. Pharmacol. 346:87-95. 1998. The microphysiometer can be used for assaying adherent or non-adherent eukaryotic or prokaryotic cells. By measuring extracellular acidification changes in cell media over time, the microphysiometer directly measures cellular responses to various stimuli, including zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 proteins, their agonists, and antagonists. Preferably, the microphysiometer is used to measure responses of a 2cyto20-, zcyto21-, zcyto22-, zcyto24- and zcyto25-responsive eukaryotic cell, compared to a control eukaryotic cell that does not respond to zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 polypeptide. Zcyto20-, zcyto21-, 2cyto22-, zcyto24- and zcyto25-responsive eukaryotic cells comprise cells into which a receptor for zcyto20, 2cyto21, zcyto22, zcyto24 and zcyto25 has been transfected, thereby creating a cell that is responsive to zcyto20, 2cyto21, zcyto22, zcyto24 and zcyto25, as well as cells naturally responsive to zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25. Differences, measured by a change, for example, an increase or diminution in extracellular acidification, in the response of cells exposed to zcyto20, 2cyto21, zcyto22, zcyto24 and 2cyto25 polypeptide, relative to a control not exposed to 2cyto20, zcyto21, zcyto22, 2cyto24 and zcyto25, are a direct measurement of 2cyto20-, zcyto21-, zcyto22-, zcyto24- and zcyto25-modulated cellular responses. Moreover, such zcyto20-, zcyto21-, zcyto22-, zcyto24- and zcyto25 -modulated responses can be assayed under a variety of stimuli. The present invention thus provides methods of identifying agonists and antagonists of zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 proteins, comprising providing cells responsive to a zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 polypeptide, culturing a first portion of the cells in the absence of a test compound, culturing a second portion of the cells in the presence of a test compound, and detecting a change, for example, an increase or diminution, in a cellular response of the second<br><br>
portion of the cells as compared to the first portion of the cells. The change In cellular response is shown as a measurable change in extracellular acidification rate. Culturing a third portion of the cells in the presence of a 2cyto20, zcyto21, zcyto22, 2cyto24 and zcyto25 protein and the absence of a test compound provides a positive control for the zcyto20-, zcyto21-, zcyto22-, zcyto24- and zcyto25-responsive cells and a control to compare the agonist activity of a test compound with that of the 2cyto20, zcyto21, zcyto22, zcyto24 and zcyto25 polypeptide. Antagonists of zcyto20, zcyto2I, zcyto22, zcyto24 and zcyto25 can be identified by exposing the cells to zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 protein in the presence and absence of the test compound, whereby a reduction in zcyto20-, zcyto21-, zcyto22-, zcyto24- and zcyto25-stimulated activity is indicative of antagonist activity in the test compound.<br>
Expression of zcytQ20, zcyto21, zcyto22, zcytQ24 and zcyto25 polynucleotides in animals provides models for further study of the biological effects of overproduction or inhibition of protein activity in vivo. Zcyto20-, zcyto21-, zcyto22-, zcyto24- and zcyto25-encoding polynucleotides and antisense polynucleotides can be introduced into test animals, such as mice, using viral vectors or naked DNA, or transgenic animals can be produced.<br>
One in vivo approach for assaying proteins of the present invention utilizes viral delivery systems. Exemplary viruses for this purpose include adenovirus, herpesvirus, retroviruses, vaccinia virus, and adeno-associated virus (AAV). Adenovirus, a double-stranded DNA virus, is currently the best studied gene transfer vector for delivery of heterologous nucleic acids. For review, see Becker et al., Meth. Cell Biol. 43:161-89, 1994; and Douglas and Curiel, Science &amp; Medicine 4:44-53, 1997.'. The adenovirus system offers several advantages. Adenovirus can (i) accommodate relatively large DNA inserts; (ii) be grown to high-titer; (iii) infect a broad range of mammalian cell types; and (iv) be used with many different promoter including ubiquitous, tissue specific, and regulatable promoters. Because adenoviruses are stable in the bloodstream, they can be administered by intravenous injection.<br>
By deleting portions of the adenovirus genome, larger inserts (up to 7 kb) of heterologous DNA can be accommodated. These inserts can be incorporated into the viral DNA by direct ligation or by homologous recombination with a co-transfected plasmid. In an exemplary system, the essential El gene is deleted from ^e viral vector, and the virus will not replicate unless the EI gene is provided by the host cell (e.g., the human .293 cell line). When intravenously administered to intact animals, adenovirus primarily targets the liver. If the adenoviral delivery system has an El gene deletion, the virus cannot replicate in the host cells. However, the host's tissue (e.g., liver) will express and process (and, if a signal sequence is present, secrete) the<br><br>
heterologous protein.     Secreted proteins will enter the circulation  in the highly vascularized hver, and effects on the infected animal can be determined.<br>
An alternative method of gene delivery comprises removing cells from the body and introducing a vector into the cells as a naked DNA plasmid. The transformed cells are then re-implanted in the body. Naked DNA vectors are introduced into host cells by methods known in the art, including transfection, electroporation, microinjection, transduction, cell fusion, DEAE dextran, calcium phosphate precipitation, use of a gene gun, or use of a DNA vector transporter. See, Wu et al., J. Biol. Chem. 263:14621-14624, 1988; Wu et al., J. Biol. Chem. 267:963-967. 1992; and Johnston and Tang, Meth. Cell Biol. 43:353-365,1994.<br>
Transgenic mice, engineered to express a zcytolO, zcyto2h zcyto22, zcyto24 and zcytolS gene, and mice that exhibit a complete absence of zcytolO, zcyto21, zcyto22. zcyto24 and zcyto25 gene function, referred to as "knockout mice" (Snouwaert et al., Science 257:1083,1992), can also be generated (Lowell et al.. Nature 366:740-742, 1993). These mice can be employed to study the zcyio20, zcytoll, zcyto22, zcyto24 andzcyto25 gene and the protein encoded thereby in an in vivo system. Transgenic mice are particularly useful for investigating the role of zcyto20, 2cyto21, 2cyto22, zcyto24 and zcyto25 proteins in early development in that they allow the identification of developmental abnormalities or blocks resulting from the over- or underexpression of a specific factor. See also, Maisonpierre et al.. Science 277:55-60. 1997 and Hanahan, Science 277:48-50, 1997. Preferred promoters for transgenic expression include promoters from metallothionein and albumin genes.<br>
A loss of normal inhibitory control of muscle contraction has been associated with damage or perturbation of selected gamma-aminobutryric acid-secreting neurons.- For example, Stiff Man Syndrome exhibit remarkable stiffness of musculature, believed to be mediated through interference of the functioning of their gamma-aminobutryric acid (GABA) producing neurons. Other related neuromuscular disorders include myotonia, metabolic myopathies, Isaac's syndrome, dystonia, and tetanic spasms fValldeoriola. J. Neurol 246:423-431. 1999).<br>
Similarly, direct measurement of zcyto20. zcyto21, zcyto22, zcytQ24 and zcyto25 polypeptide, or its loss of expression in a tissue can be determined in a tissue or cells as Ihey undergo tumor progression. Increases in invasiveness and motihty of cells, or the gain or loss of expression of zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 in a pre-cancerous or cance;:ous condition, in comparison to normal tissue, can serve as a (^agnostic for transformation, invasion and metastasis in tumor progression. As such, knowledge of a tumor's stage of progression or metastasis will aid the physician in choosing the most proper therapy, or aggressiveness of treatment, for a given individual<br><br>
cancer patient. Methods of measuring gain and loss of expression (of either raRNA or protein) are well known in the art and described herein and can be applied to zcyto20, zcyto21, zcyto22, zcyto24 and zcyio25 expression. For example, appearance or disappearance of polypeptides that regulate cell motility can be used to aid diagnosis and prognosis of prostate cancer (Banyard, J. and Zetter, B.R., Cancer and Metast. Rev. 17:449-458, 1999). As an effector of cell motility, or as a liver-specific marker, 2cyto20,2cyto21, zcyto22, zcyto24 and zcyto25 gain or loss of expression may serve as a diagnostic for brain and other cancers. Moreover, analogous to the prostate specific antigen (PSA), increased levels of zcyto20, zcyto21, zcyto22, 2cyto24 and zcyto25 polypeptides, or anti-zcyto20, anti-zcyto21, anti-zcyto22, anti-zcyto24 and anti-zcyto25 antibodies in a patient, relative to a normal control can be indicative of brain and other cancers (See, e.g., Mulders, TMT, et al., Eur. J. Sursical Oncol. 16:37-41. 1990). Strong zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 expression in tissue not normally found to express the polynucleotides would serve as a diagnostic of an abnormality in the cell or tissue type, of invasion or metastasis of cancerous liver tissue into non-liver tissue, and could aid a physician in directing further testing or investigadon, or aid in directing therapy.<br>
In addition, zcyto20, zcyto21, zcyto22, 2cyto24 and zcyto25 polynucleotide probes, anti- 2cyto20, zcyto21, zcyto22, zcyto24 and zcyto25 antibodies, and detection the presence of zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 polypeptides in tissue can be used to assess whether brain or other tissue found to normally express zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 is present, for example, after surgery involving the excision of a diseased or cancerous liver or neuronal tissue. As such, the polynucleotides, polypeptides, and antibodies of the present invention can be used as an aid to determine whether all tissue is excised after surgery, for example, after surgery for brain and other cancers. In such instances, it is especially important to remove all potentially diseased tissue to maximize recovery from the cancer, and to minimize recurrence. Preferred embodiments include fluorescent, radiolabeled, or calorimetiically labeled anti-2cyto20 antibodies and 2cyto20, 2cyto21, zcyto22. 2cyto24 and zcyto25 polypeptide binding partners, that can be used histologically or in situ.<br>
Moreover, the activity and effect of zcyto20, zcyto21, zcyto22, zcyio24 and zcyto25 on tumor progression and metastasis can be measured in vivo. Several syngeneic mouse models have been developed to study the influence of polypepddes, compounds or other treatments on tumor progression. In these models, tumor ceils passaged in culture are implanted into mice of the same strain as the tumor donor. The cells will develop into tumors having similar characteristics in the recipient mice, and<br><br>
metastasis will also occur in some of the models. Appropriate tumor models for our studies include the Lewis lung carcinoma (ATCC No. CRL-1642) and B16 melanoma (ATCC No. CRL-6323), amongst others. These are both commonly used tumor lines, syngeneic to the C57BL6 mouse, that are readily cultured and manipulated in vitro. Tumors resulting from implantation of either of these cell, lines are capable of metastasis to the lung in C57BL6 mice. The Lewis lung carcinoma model has recently been used in mice to identify an inhibitor of angiogenesis (O'Reilly MS, et al. Cell 79: 315-328,1994). C57BL6/J mice are treated with an experimental agent either through daily injection of recombinant protein, agonist or antagonist or a one-time injection of recombinant adenovirus. Three days following this treatment, 10 to 10 cells are implanted under the dorsal skin. Alternatively, the cells themselves may be infected with recombinant adenovirus, such as one expressing zcyto20, zcyto21, 2cyto22, zcyto24 or zcyto25, before implantation so that the protein is synthesized at the tumor site or intracellularly, rather than systemically. The mice normally develop visible tumors within 5 days. The tumors are allowed to grow for a period of up to 3 weeks, during which time they may reach a size of 1500 - 1800 mm in the control treated group. Tumor size and body weight are carefully monitored throughout the experiment. At the time of sacrifice, the tumor is removed and weighed along with the lungs and the liver. The lung weight has been shown to correlate well with metastatic tumor burden. As an additional measure, lung surface metastases are counted. The resected tumor, lungs and liver are prepared for histopathological examination, immunohistochemistry, and in situ hybridization, using methods known in the art and described herein. The influence of the expressed polypeptide in question, e.g., zcyto20, zcyto21, 2cyto22, 2cyto24 or zcyto25, on the ability of the tumor to recruit vasculature and undergo metastasis can thus be assessed. In addition, aside from using adenovirus, the implanted cells can be transiently transfected with 2cyto20, zcyto21, zcyto22, zcyto24 or zcyto25. Use of stable zcyto20, zc&gt;to21, zcyto22, zcyto24 and zcyto25 transfectants as well as use of induceable promoters to activate zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 expression in vivo are known in the art and can be used in this system to assess 2cyto20 induction of metastasis. Moreover, purified zcyto20, zcyto21, zcyto22, zcvto24 and zcyto25 or zcyto20-, zcyto21-, zcyto22-, zcyto24- and zcyto25 -conditioned media can be directly injected in to this mouse model, and hence be used in this system. For general reference see, O'Reilly MS, et al. Cell ;^:315-328, 1994; and Rusciano D, et al. Murine Models of Liver Metastasis. Invasion Metastasis 14:349-361,1995.<br><br>
Antisense methodology can be used to inhibit 2cyto20 gene transcription to examine the effects of such inhibition in vivo. Polynucleotides that are complementary to a segment of a zcyto20-, zcyto21-, 2cyto22-, zcyto24- and zcyto25-encoding polynucleotide (e.g., a polynucleotide as set forth in SEQ ID N0:1) are designed to bind to zcyto20-, zcyto21-, zcyto22-, zcyto24- and 2cyto25-encoding mRNA and to inhibit translation of such mRNA. Such antisense oligonucleotides can also be used to inhibit expression of zcyto20, zcyto21, zcyto22, zcytD24 and zcyto25 polypeptide-encoding genes in cell culture.<br>
Most cytokines as well as other proteins produced by activated lymphocytes play an important biological role in cell differentiation, activation, recruitment and homeostasis of cells throughout the body. Zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 and inhibitors of their activity are expected to have a variety of therapeutic applications. These therapeutic applications include treatment of diseases which require immune regulation, including autoimmune diseases such as rheumatoid arthritis, multiple sclerosis, myasthenia gravis, systemic lupus erythematosis, and diabetes. Zcyto20, zcyto21, zcyto22, zcyto24 and 2cyto25 may be important in the regulation of inflammation, and therefore would be useful in treating rheumatoid arthritis, asthma and sepsis. There may be a role of zcyto20, zcyto21, zcyto22, zcyto24 and zcytQ25 in mediating tumorgenesis, whereby a zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 antagonist would be useful in the treatment of cancer. Zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 may be useful in modulating the immune system, whereby zcyto20, zcyto21, zcytc22, zcyto24 and zcyto25 and zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 antagonists may be used for reducing graft rejection, preventing graft-vs-host disease, boosting immunity to infectious diseases, treating immunocompromised patients (e.g., HTV^ patients), or in improving vaccines.<br>
Members of the protein family of the present invention have been shown to have an antiviral effect that is similar to interferon-a. Interferon has been approved in the United States for treatment of autoimmune diseases, condyloma acuminatum, chronic hepatitis C, bladder carcinoma, cervical carcinoma, laryngeal papillomatosis, fungoides mycosis, chronic hepatitis B, Kaposi's sarcoma in patients infected with human immunodeficiency vims, malignant melanoma, hairy cell leukemia, and multiple sclerosis. In addition, zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 may be used to treat forms of arteriosclerosis, such as atherosclerosis, by inhibiting cell proliferation. Accordingly, the present invention contemplates the use of proteins, polypeptides, and peptides having zcyto20 activity to treat such conditions, as well as to treat retinopathy. The present invention also contemplates the use of proteins, polypeptides, and pepddes having zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25<br><br>
activity to treat lymphoproliferative disorders, including B-cell lymphomas, chronic lymphocytic leukemia, acute lymphocytic leukemia, Non-Hodkin's lymphomas, multiple myeloma, acute myelocytic leukemia, chronic myelocytic leukemia.<br>
Interferons have also been shown to induce the expression of antigens by cultured cells (see, for example, Auth et al, Hepatoiogy 18:5A6 (1993), Guadagni etal. Int. J. Biol. Markers 9:53 (1994), Girolomoni etal, Eur. J. Immunol 25:2163 (1995), and Maciejewski et al, Blood 85:3183 (1995). This activity enhances the ability to identify new tumor associated antigens in vitro. Moreover, the ability of interferons to augment the level of expression of human tumor antigens indicates that interferons can be useful in an adjuvant setting for immunolherapy or enhance immunoscintigraphy using anti-tumor antigen antibodies (Guadagni et al. Cancer Immunol Immunother. 26:222 (1988); Guadagni et al. Int. J. Biol Markers 9:53 (1994)). Thus, the present invention includes the use of proteins, polypeptides and peptides having zcyto20, zcyto21, zcytD22, zcyto24 and 2cyto25 activity as an adjuvant for immunotherapy or to improve immunoscintigraphy using anti-tumor antigen antibodies.<br>
Methods for detection and diagnosis of viral infections are well known to those skilled in the art. The exact method used for measuring a reduction in virus in response to administration of molecules of the present invention will be dependent upon the patient, type of viral infection, and the like. For example, methods include, but are not limited to, measuring changes in CD4 cell counts, serologic tests, measuring the DNA of the virus and RNA of the virus by conventional and real-time quandtative polymerase chain reaction assays, viral induced antibody levels, immunofluorescence and enzyme-linked immunosorbant assays, cytopathic effects, and histology.<br>
Moreover, zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 may bind CD4 or .another leukocyte receptor and exhibit antiviral effeas, for example, against human immunodeficiency vims (HTV) or human T-cell iymphotropic virus (HTLV). Alternatively, zcyto20 polypeptide may compete for a viral receptor or co-receptor to block viral infection. Zcyto20 may be given parentally to prevent viral infection or to reduce ongoing viral replicadon and re-infection (Gayowski, T. et al.. Transplantation 64:422-426,1997). Thus, 2cyto20 may be used as an antiviral therapeutic, for example, for viral leukemias (HTLV), AIDS (HIV), or gastrointestinal viral infections caused by, for example, rotavirus, calicivirus (e.g., Norwalk Agent) and certain strains of pathogenic adenovirus. Hepatitis B and C.<br>
Zcyto20,_zcyto21, zcyto22, zcyto24 and 2cyto25 can also be used to treat myocarditis, a disorder that arises when the heart is involved in an inflammatory process. The infiltration of lymphocytes and myocytolysis is thought to result after infection by virus, bacteria, fungi or parasites (see, for example, BrodJson et ai, J.<br><br>
Infection 37:99 (1998)). Zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 can be injected intravenously or subculaneously to treat infections associated wiiii myocarditis. Zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 can also be administered intravenously as an immunoregulatory cytokine in the treatment of autoimmune myocarditis. Interferon dosages can be extrapolated using a autoimmune model of myocarditis in the A/J mouse (Donemeyer, etaL J. Exp. Med. 752:1291 (1995)).<br>
Exogenous administration of interferon-T in sheep increases the pregnancy rate (Aggarwal, Human Cytokines III, (Blackwell Science 1997)). As described herein, Zcyto20 mRNA is expressed in placenta. Accordingly, the present invention includes the use of zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25, such as ^e disclosed human zcytoZO, zcyto21, and 2cyto22, to promote and protect growth of the fetus. As an illustration, zcyto20, zcyto21, and zcyto22 can be used to protect a developing fetus from viral infection (e.g., human immunodeficiency virus, human papilloma virus, and the like). Jn addition, zcyto20, 2cyto21j zcyto22&gt; zcyto24 and zcyto25 can be used to promote in vitro fertilization.<br>
Recent reports have highlighted the role of type I interferons in the prevention of viral-induced diabetes by inducing a strong antiviral state in pancreatic beta cells early during viral infection (Flodstroem et al.. Nature Immunology 3, 373 -382 (2002)). This prevents the loss of beta cells due to viral-induced cell death and autoimmunity that accompanies it. Zcyto20, 21 and 22 also induce an antiviral state in cells that express their receptor, zcytorl9. Zcytorl9 is highly expressed in pancreatic tissue and therefore zcyto20-22 may play a role in prevention of viral-induced diabetes due to beta cell death. In addition, the role of type I interferons in prevention of viral-induced diabetes may be extended to other viral-induced autoimmune diseases and therefore, zcyto20-22 may also play a role in prevention of other diseases such as muscular sclerosis, lupus, and viral-induced autoimmune diseases in tissues that express the zcyto20-22 receptor, zcytorl9.<br>
Chronic systemic expression of type I interferons has also been associated with the pathogenesis of type I diabetes. Given the similarity of type I interferons to zcyto20-22 in regards to biological activity and gene induction, chronic systemic expression of zcyto20, 21, or 22 might also play a role in the pathogenesis of type 1 diabetes. Therefore, an inhibitor of zcyto20-22 activity in the pancreas might be beneficial in prevention of type I diabetes.<br>
Zcyto20,. zcyto21, zcyto22, zcyto24 and zcyto25 polypeptides can be administered alone or in combination with other vasculogenic or angiogenic agents, including VEGF.   When using zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 in<br><br>
combination with an additional agent, the two compounds can be administered simultaneously or sequentially as appropriate for the specific condition being treated.<br>
Zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 will be useful in treating tumorgenesis, and therefore would be useful in die treatment of cancer. A zcyto20 inhibition of anti-IgM stimulated normal B-cells and a similar effect is observed in B-cell tumor lines suggest that there may be therapeutic benefit in treating patients with the zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 in order to induce the B cell tumor cells into a less proUferative state. The ligand could be administered in combination with other agents already in use including both conventional chemotherapeutic agents as well as immune modulators such as interferon alpha. Alpha/beta interferons have been shown to be effective in treating some leukemias and animal disease models, and the growth inhibitory effects of interferon-alpha and zcyto20 may be additive for B-cell tumor-derived cell lines.<br>
The present invention provides a method of reducing proliferation of a neoplastic B or T cells comprising administering to a mammal with a B or T cell neoplasm an amount of a composition of zcyto20 sufficient to reduce proliferation of the neoplastic B or T cells. Zcyto20, zcyto2I, zcyto22, zcyto24 and zcyto25 stimulation of lytic NK cells from marrow progenitors and the proliferation of T-cells following jw;tivation of the antigen receptors would enhance treatment for patients receiving allogenic marrow transplants, and therefore, zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 will enhance the generation of anti-tumor responses, with or without the infusion of donor lymphocytes.<br>
In another aspect, the present invention provides a method of reducing proliferation of a neoplastic B or T cells comprising administering to a mammal with a B or T cell neoplasm an amount of a composition of zcyto20 antagonist sufficient to reducing proliferation of the neoplastic B or T cells. Furthermore, the zcyto20 antagonist can be a ligand/toxin fusion protein.<br>
A zcyto20-, zcyto21-, zcyto22-, zcyto24- and zcyto25-saporin fusion toxin may be employed against a similar set of leukemias and lymphomas, extending the range of leukemias that can be treated with zcyto20, zcyto21. zcyto22, zcyto24 and zcyto25. Fusion toxin mediated activation of the zcyto20 receptor provides two independent means to inhibit the growth of the target cells, the first being identical to the effects seen by the ligand alone, and the second due to delivery of the toxin through receptor internalization,.<br>
For pharmaceutical use, zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 proteins are formulated for topical or parenteral, particularly intravenous or subcutaneous, delivery according to conventional methods.  In general, pharmaceutical<br><br>
formulations will include a zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 polypeptide in combination with a phaimaceutically acceptable vehicle, such as saline, buffered saline, 5% dextrose in water, or the like. Formulations may further include one or more excipients, preservatives, solubilizers, buffering agents, albumin to prevent protein loss on vial surfaces, etc. Methods of formulation are well known m the ari and are disclosed, for example, in Remington: The Science and Practice of Pharmacy. Gennaro, ed., Mack Publishing Co., Easton, PA, 19th ed., 1995. Zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 will preferably be used in a concentration of about 10 to 100 fig/ml of total volume, although concentrations in the range of 1 ng/ml to lOOO \LgJml may be used. For topical application, such as for the promotion of wound healing, the protein will be applied in the range of 0.1-10 ^ig/cm^ of wound area, with the exact dose determined by the clinician according to accepted standards, taking into account the nature and severity of the condition to be treated, patient traits, etc. Determination of dose is within the level of ordinary skill in the art. Dosing is daily or intermittently over the period of treatment. Intravenous administration will be by bolus injection or infusion over a typical period of one to several hours. Sustained release formulations can also be employed. In general, a therapeutically effective amount of zcyto20 is an amount sufficient to produce a clinically significant change in the treated condition, such as a clinically significant change in hematopoietic or immune function, a significant reduction in morbidity, or a significantly increased histological score.<br>
Zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 proteins, agonists, and antagonists are useful for modulating the expansion, proliferation, activation, differentiation, migration, or metabolism of responsive cell types, which include both primary cells and cultured cell lines. Of particular interest in this regard are hematopoietic cells, mesenchymal cells (including stem cells and mature myeloid and lymphoid cells), endothelial cells, smooth muscle cells, fibroblasts, hepatocytes, neural cells and embryonic stem cells. Zcyto20, zcyto21, 2cylo22, 2cyto24 and zcyto25 polypeptides are added to tissue culture media for these cell types at a concentration of about 10 pg/ml to about 100 ng/ml. Those skilled in the art will recognize that zcyto20 proteins can be advantageously combined with other growth factors in culture media.<br>
Within the laboratory research field, zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 proteins can also be used as molecular weight standards or as reagents in assays for determining circulating levels of the protein, such as in the diagnosis of disorders characterized- by over- or under-production of zcyto20 protein or in the analysis of cell phenotype.<br>
Zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 proteins can also be used to identify inhibitors of their activity.   Test compounds are added to the assays<br><br>
disclosed above to identify compounds that inhibit the activity of 2cyto20 protein. In addition to those assays disclosed above, samples can be tested for inhibition of zcyto20 activity v,;ithin a variety of assays designed to measure receptor binding or the stimulation/inhibition of zcyto20', zcyto2U, zcyto22-, zcyto24- and zcyto25-dependent cellular responses. For example, 2cyto20-, zcyto21-, zcyto22-, zcyto24- and zcyto25-responsive cell lines can be transfected with a reporter gene construct that is responsive to a zcyto20-, zcyto2I-, zcyto22-, zcyto24- and zcyto25-stimulated cellular pathway. Reporter gene constructs of this type are known in the art, and will generally comprise a zcyto20-, zcyto2l-, zcyto22-, zcyto24- and zcytD25-activated serum response element (SRE) operably linked to a gene encoding an assayable protein, such as luciferase. Candidate compounds, solutions, mixtures or extracts are tested for the ability to inhibit the activity of zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 on the target cells as evidenced by a decrease in zcyto20, 2cyto21, zcyto22, zcyto24 and zcyto25 stimulation of reporter gene expression. Assays of this type will detect compounds that directly block zcyto20, zcyto21, zcytQ22, zcyto24 and zcyto25 binding to cell-surface receptors, as well as compounds that block processes in the cellular pathway subsequent to receptor-ligand binding. In the alternative, compounds or other samples can be tested for direct blocking of zcyto20, zcyto21, zcyto22, zcyto24 and zcylo25 binding to receptor using 2cyto20, zcyto21, zcyto22, 2cyto24 and zcyto25 tagged with a detectable label (e.g., '^I, biotin, horseradish peroxidase, FTFC, and the like). Within assays of this type, the ability of a test sample to inhibit the binding of labeled zcyto20, zcyto21, zcyto22, zcyto24 and zcylo25 to the receptor is indicative of inhibitory activity, which can be confinned through secondary assays. Receptors used within binding assays m^y be cellular receptors or isolated, immobilized receptors.<br>
As used herein, the term "antibodies" includes polyclonal antibodies, monoclonal antibodies, antigen-binding fragments thereof such as F(ab')2 ^'^ F^b<br>
fragments, single chain antibodies, and the like, including genetically engineered antibodies. Non-human antibodies may be humanized by grafting non-human CDRs onto human framework and constant regions, or by incorporating the entire non-human variable domains (optionally "cloaking" them with a human-like surface by replacement of exposed residues, wherein the result is a "veneered" antibody). In some instances, humanized antibodies may retain non-human residues within the human variable region framework domains to enhance proper binding characteristics. Through humanizing antibodies^ biological half-life may be increased, -and the potential for adverse immune reactions upon administration to humans is reduced. One skilled in the art can generate humanized antibodies with specific and different constant domains (i.e., different Ig subclasses) to facilitate or inhibit various immune functions associated<br><br>
with particular antibody constant domains. Antibodies are defined to be specifically binding if they bind to a zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 polypeptide or protein with an affinity at least 10-fold greater than the binding affinity to control (non-2cyto20,2cyto21, zcyto22, zcyto24 and zcyto25) polypeptide or protein. The affinity of a monoclonal antibody can be readily determined by one of ordinary skill in the art (see, for example, Scatchard, Ann. NY Acad. Sci. 51.: 660-672,1949).<br>
Methods for preparing polyclonal and monoclonal antibodies are well known in the art (see for example, Hurrell, J. G. R., Ed., Monoclonal Hybridoma Antibodies: Techniques and Applications. CRC Press, Inc., Boca Raton, PL, 1982, which is incorporated herein by reference). Of particular interest are generating antibodies to hydrophitic antigenic sites which include, for example, amino acid residues 169 (Glu) to 174 (Glu) of SEQ ID NO; 2, amino acid residues 54 (Lys) to 59 (Ala) of SEQ ID NO: 2. amino acid residues 53 (Phe) to 58 (Asp) of SEQ ID NO: 2, amino acid residues 168 (Gin) to 173 (Lys) of SEQ ID NO: 2, and amino acid residues 154 (Pro) to 159 (Arg) of SEQ ID NO: 2. For example, in zcyto22, hydrophilic regions include amino acid residues 169 (Glu) to 174 (Glu) of SEQ ID NO: 7, amino acid residues 54 (Lys) to 59 (Ala) of SEQ ID NO: 7, amino acid residues 53 (Phe) to 58 (Asp) of SEQ ID NO: 7, amino acid residues 168 (Gin) to 173 (Lys) of SEQ ID NO: 7, and amino acid residues 154 (Pro) to 159 (Arg) of SEQ ID NO: 7 would be useful. As would be evident to one of ordinary skill in the art, polyclonal antibodies can be generated from a variety of warm-blooded animals such as horses, cows, goats, sheep, dogs, chickens, rabbits, mice, and rats. The immunogenicity of a zcytQ20, zcyto2l, zcyto22, zcyto24 and zcyto25 polypeptide may be increased through the use of an adjuvant such as alum (aluminum hydroxide) or Freund's complete or incomplete adjuvant. Polypeptides useful for immunization also include fusion polypeptides, such as  fusions of a zcyto20, zcyta2I, zcyto22,2cyto24 and zcyto25 polypeptide or a portion thereof with an immunoglobulin polypeptide or with maltose binding protein. The polypeptide immunogen may be a full-length molecule or a portion thereof. If the polypeptide portion is "hapten-like", such portion may be advantageously joined or linked to a macromolecular carrier (such as keyhole limpet hemocyanin (KLH), bovine serum albumin (BSA) or tetanus toxoid) for immunization.<br>
Altemative techniques for generating or selecting antibodies include in vitro exposure of lymphocytes to zcyto20, zcyto2I, zcyto22, 2cyto24 and zcyto25 polypeptides, and selection of antibody display libraries in phage or similar vectors (e.g., through the use of immobilized or labeled zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 polypeptide). Human antibodies can be produced in transgenic, non-human animals that have been engineered to contain human  immunoglobulin  genes  as<br><br>
disclosed in WIPO Publication WO 98/24893, It is preferred that the endogenous immunoglobuliTi genes in these animals be inactivated or eliminated, such as by homologous recombination.<br>
A variety of assays known to those skilled in the art can be utilized to<br>
detect antibodies which specifically bind to 2cyto20, zcyto21, 2cyto22, zcyto24 and<br>
zcyto25 polypeptides. Exemplary assays are described in detail in Antibodies: A<br>
Laboratory Manual. Harlow and Lane (Eds.), Cold Spring Harbor Laboratory Pr^ss,<br>
1988.	Representative    examples    of    such    assays    include:     concurrent<br>
immunoeiectrophoresis, radio-immunoassays, radio-immunoprecipitations, enzyme-linked immunosorbent assays (ELISA), dot blot assays, Western blot assays, inhibition or competition assays, and sandwich assays.<br>
Antibodies to zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 may be used for affinity purification of the protein, within diagnostic Essays for determining circulating levels of the protein; for detecting or quantJtating soluble zcyto20, zcyto21, zcyto22, 2cyto24 and zcyto25 polypeptide as a marker of underlying pathology or disease; for immunolocalization within whole animals or tissue sections, including immunodiagnostic applications; for immunohistochemistry; and as antagonists to block protein activity in vitro and in vivo. Antibodies to zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 may also be used for tagging cells that express zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25; for affinity purification of zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 polypeptides and proteins; in analytical methods employing FACS; for screening expression libraries; and for generating anti-idiotypic antibodies. Antibodies can be linked to other compounds, including therapeutic and diagnostic agents, using known methods to provide for targeting of those compounds to cells expressing receptors for zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25. For certain applications, including in vitro and in vivo diagnostic uses, it is advantageous to employ labeled antibodies. Suitable direct tags or labels include radionuclides, enzymes, substrates, cofactors, inhibitors, fluorescent markers, chemiluminescent markers, magnetic particles and the like; indirect tags or labels may feature use of biotin-avidin or other complement/anti-complement pairs as intermediates. Antibodies of the present invention may also be directly or indirectly conjugated to drugs, toxins, radionuclides and the like, and these conjugates used for in vivo diagnostic or therapeutic applications(e.g., inhibition of cell proliferation). See, in general, Ramakrishnan et a!.. Cancer Res. 56:1324rl330. 1996.<br>
Polypeptides and proteins of the present invention can be used to identify and isolate receptors. Zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 receptors may be involved in growth regulation in the liver, blood vessel formation, and<br><br>
other developmental processes. For example, zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 proteins and polypeptides can be immobilized on a column, and membrane preparations run over the column (as generally disclosed in Immobilized Affinity Ligand Techniques. Hermanson et al., eds., Academic Press, San Diego, CA, 1992, pp.195-202). Proteins and polypeptides can also be radiolabeled (Methods Enzvmol.. vol. 182, "Guide to Protein Purification", M. Deutscher, ed.. Academic Press, San Diego, 1990, 721-737) or photoaffinity labeled (Brunner et al., Ann. Rev. Biochem. 62:483-514, 1993 and Fedan et al., Biochem. Pharmacol. 33:1167-1180, 1984) and used to tag specific cell-surface proteins. In a similar manner, radiolabeled zcyto20 proteins and polypeptides can be used to clone the cognate receptor in binding assays using cells transfected with an expression cDNA library.<br>
Zcyto20, zcytoll, 2cyto22, zcyto24 and zcytD25 polypeptides can also be used to teach analytical skills such as mass spectrometry, circular dichroism, to determine conformation, especially of the four alpha helices, x-ray crystallography to determine the three-dimensional structure in atomic detail, nuclear magnetic resonance spectroscopy to reveal the structure of proteins in solution. For example, a kit containing the 2cyto20, zcyto21, zcyto22, 2cyto24 and zcyto25 can be given to the student to analyze. Since the amino acid sequence would be known by the instructor, the protein can be given to the student as a test to determine the skills or develop the skills of the student, the instructor would then know whether or not the student has correctly analyzed the polypeptide. Since every polypeptide is unique, the educational utility of zcyto20 would be unique unto itself.<br>
The antibodies which bind specifically to zcyto2p, zcyto21, zcyto22, zcyto24 and zcyto25 can be used as  a teaching aid to instruct students how to prepare affinity chromatography columns to purify zcyto20, zcyto2I, zcytQ22, zcyto24 and zcyto25, cloning and sequencing the polynucleotide that encodes an antibody and thus as a practicum for teaching a student how to design humanized antibodies. The zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 gene, polypeptide, or antibody would then be packaged by reagent companies and sold to educational institutions so that the students gain skill in art of molecular biology. Because each gene and protein is unique, each gene and protein creates unique challenges and learning experiences for students in a lab practicum. Such educational kits containing the ZCYT020 gene, polypeptide, or antibody are considered within the scope of the present invention.<br>
In summary, the present invention provides an isolated polypeptide that has at least 80% or 95% or 100% identity to a polypeptide selected from the group consisting : (a) a polypeptide comprising an amino acid sequence as shown in SEQ ID N0:2 from    amino acid residue 22 to amino acid residue 205; (b) a polypeptide<br><br>
comprising an amino acid sequence as shown in SEQ ID NO: 7 from amino acid residue 22 to amino acid residue 205; (c) a polypeptide comprising an amino acid sequence as shown in SEQ ID NO: 9 from amino acid residue 29 to amino acid residue 202; and (d) a polypeptide comprising an amino acid sequence as shown in SEQ ID NO: 11 from amino acid residue 29 to amino acid residue 202.<br>
In another embodiment, the isolated polypeptide binds a receptor as shown in SEQ ID NOS: 24, 27 or 29 as a monomeric or homodimeric receptor or SEQ ID NOS: 24, 27 or 29 in combination with SEQ ID N0:41 as a heterodimeric receptor.<br>
In another aspect, the present invention includes an isolated polypeptide comprising an amino acid sequence as shown in SEQ ID N0:2 from amino acid residue 22 to amino acid residue 205 or as shown in SEQ ID N0:2 from amino acid residue 1 to amino acid residue 205.<br>
In another aspect, the present invention includes an isolated polypeptide comprising an amino acid sequence as shown in SEQ ID NO:7 from amino acid residue 22 to amino acid residue 205 or as shown in SEQ ID N0:7 from amino acid residue 1 to amino acid residue 205.<br>
In another aspect, the present invention includes an isolated polypeptide comprising an amino acid sequence as shown in SEQ ID NO: 9 from amino acid  , residue 29 to amino acid residue 202 or as shown in SEQ ED N0:9 from amino acid residue 1 to amino acid residue 202.<br>
In another aspect, the present invention includes an isolated polypeptide comprising an amino acid sequence as shown in SEQ ID NO: 11 from amino acid residue 29 to amino acid residue 202 or as shown in SEQ ID NO: 11 from amino acid residue 1 to amino acid residue 202.<br>
In another aspect, the present invention includes an isolated polypeptide that is at least 14 contiguous amino acids of SEQ ID N0:2 from amino acid residue 22 to amino acid residue 205 or as shown in SEQ ID N0:2 from amino acid residue 1 to amino acid residue 205, wherein said polypeptide stimulates an antigenic response in a mammal.<br>
In another aspect, the present invention includes an isolated polypeptide<br>
that is at least 14 contiguous amino acids of SEQ ID N0:7 from amino acid residue 22<br>
to amino acid residue 205 or as shown in SEQ ID N0:7 from amino acid residue 1 to<br>
amino acid residue 205, wherein said polypeptide stimulates an antigenic response in a<br>
mammal.	,    .<br>
In another aspect, the present invention includes an isolated polypeptide that is at least 14 contiguous amino acids of SEQ ID NO: 9 from amino acid residue 29 to amino acid residue 202 or as shown in SEQ ID NO:9 from amino acid residue 1 to<br>
amino acia residue 202, wherein said polypeptide stimulates an antigenic response in a mammal.<br>
In another aspect, the present includes an isolated polypeptide that is at least 14 contiguous amino acids of SEQ ID NO: 11 from amino acid residue 29 to amino acid residue 202 or as shown in SEQ ID NO: 11 from amino acid residue 1 to amino acid residue 202, wherein said polypeptide stimulates an antigenic response in a mammal.<br>
The present invention includes pharmaceutical compositions comprising the polypeptides described herein, in pharmaceutically acceptable vehicles.<br>
The present invention also includes fusion proteins comprising the polypeptides described herein.<br>
In other aspects, the present invention includes an isolated polynucleotide that encodes a polypeptide, wherein the nucleic acid molecule is selected from the group consisting of: (a) a polynucleotide comprising the nucleotide sequence of SEQ ID NO:3, (b) a polynucleotide that remains hybridized following stringent wash conditions to a polynucleotide consisting of the nucleotide sequence of nucleotides 64 to 618 of SEQ ID NO:l, or the complement of the nucleotide sequence of nucleotides 64 to 618 of SEQ ID NO:l.<br>
In another embodiment, the isolated polynucleotide that encodes a polypeptide, wherein the nucleic acid molecule is selected from the group consisting of: (a) a polynucleotide comprising the nucleotide sequence of SEQ ID NO:36, (b) a polynucleotide that remains hybridized following stringent wash conditions to a polynucleotide consisting of the nucleotide sequence of nucleotides 64 to 618 of SEQ ID NO:6, or the complement of the nucleotide sequence of nucleotides 64 to 618 of SEQIDNO:6.<br>
In another aspect, the present invention includes an isolated polynucleotide comprising a nucleotide sequence as shown in SEQ ID NO:l from nucleotide 64 to nucleotide 618 or as shown in SEQ ID NO:l from nucleotide 1 to nucleotide 618.<br>
In another embodiment, the present invention includes an isolated polynucleotide comprising a nucleotide sequence as shown in SEQ ID NO:6 from nucleotide 64 to nucleotide 618 or as shown in SEQ ID NO:6 from nucleotide 1 to nucleotide 618.<br>
In an.oth.er embodiment, the present invention includes an isolated polynucleotide comprising a nucleotide sequence as shown in SEQ ID NO:8 from nucleotide 67 to nucleotide 606 or as shown in SEQ ID NO:l from nucleotide 1 to nucleotide 606.<br><br>
In another embodiment, the present invention includes an isolated polynucleotide comprising a nucleotide sequence as shown in SEQ ID NO: 10 from nucleotide 67 to nucleotide 606 or as shown in SEQ ID NO: 10 from nucleotide 1 to nucleotide 606.<br>
The present invention provides expression vectors, comprising the isolated nucleic acid molecules described herein with a transcription promoter, and a transcription terminator, wherein the promoter is operabiy linked with the nucleic acid molecule, and wherein the nucleic acid molecule is operabiy linked with the transcription terminator.<br>
The present invention includes provides recombinant host cells comprising the expression vector described herein, wherein the host cell is selected from the group consisting of bacterium, yeast cell, fungal cell, insect cell, mammalian cell, and plant cell.<br>
In another aspect, the present invention provides a method of producing a polypeptide, the method comprising the step of culturing recombinant host cells that comprise the expression vectors described herein, and that produce the polypeptides.<br>
The present invention provides an antibody or antibody fragment that specifically binds with the polypeptides described herein.<br>
In anotiier aspect, the present invention provides a method for expansion of monocytic cells or moncyte cell progenitors comprising culturing bone marrow or peripheral blood cells with a composition comprising an amount of the polypeptides described herein sufficient to produce an increase in the number of monocytic cells or moncyte cell progenitors in the bone marrow or ■ peripheral blood cells as compared to bone marrow or peripheral blood cells cultured in the absence of administered polypeptide.<br>
The present invention also provides a method of stimulating an immune response in a mammal exposed to an antigen or pathogen comprising: (1) detennining a level of an antigen- or pathogen-specific antibody; (2) administering a composition comprising the polypeptides described herein in an acceptable pharmaceutical vehicle;<br>
(3)	determining a post administration level of antigen- or pathogen-specific antibody;<br>
(4)	comparing the level of antibody in step (1) to the level of antibody in step (3), wherein an increase in antibody level is indicative of stimulating an immune response.<br>
In other aspects, the present invention provides a method of producing an anti-viral response in a mammal comprising administering to a mammal with a viral infection an amount of a composition of the polypeptides described herein sufficient to show a reduction in virus.<br><br>
The present invention, thus generally described, will be understood more readily by reference to the following examples, which is provided by way of illustration and is not intended to be limiting of the present invention.<br>
EXAMPLES<br>
Example 1<br>
Mammalian Expression plasmids<br>
An expression plasmid containing a polynucleotide encoding zcyto20, zcyto21, zcyto22, zcyto24 or zcyto25 can be constructed via homologous recombination. A fragment of cDNA, for example zcyto20 cDNA, is isolated by PCR using the polynucleotide sequence of SEQ ID NO: 1 with flanking regions at the 5' and 3' ends corresponding to the vector sequences flanking the zcyto20 insertion point. The primers ZC40923 and ZC40927 are shown in SEQ ID NOS: 12 and 13, respectively.<br>
The PCR reaction mixture is run on a 1% agarose gel and a band corresponding to the size of the insert is gel-extracted using a QIAquick™ Gel Extraction Kit (Qiagen, Valencia, CA). Plasmid pZMP21 is a mammalian expression vector containing an expression cassette having the MPSV promoter, multiple restriction sites for insertion of coding sequences, a stop codon, an E. coli origin of replication; a mammalian selectable marker expression unit comprising an SV40 promoter, enhancer and origin of replication, a DHFR gene, and the SV40 terminator; and URA3 and CEN-ARS sequences required for selection and replication in S. cerevisiae. It is constructed from pZP9 (deposited at the American Type Culture Collection, 10801 University Boulevard, Manassas, VA 20110-2209, under Accession No. 98668) with the yeast genetic elements taken from pRS316 (deposited at the American Type Culture Collection, 10801 University Boulevard, Manassas, VA 20110-2209, under Accession No. 77145), an internal ribosome entry site (IRES) element from poliovirus, and the extracellular domain of CD8 truncated at the C-terminal end of the transmembrane domain. Plasmid pZMP21 was digested with Bgin, and used for recombination with the PCR insert.<br>
One hundred microliters of competent yeast (S. cerevisiae) cells are independently combined with 10 /A! of the various DNA mixtures from above and transferred to a 0.2-cm electroporation cuvette. The yeast/DNA mixtures are electropulsed using power supply (BioRad Laboratories, Hercules, CA) settings of 0.75 kV (5 kV/cm), «, ohms, 25 jiF. To each cuvette is added 600 fil of 1.2 M sorbitol, and the yeast is plated in two 300-fil aliquots onto two URA-D plates and incubated at<br><br>
zcyto24 or zcyto25, i.e. zcyto20/pZMP6, by liposome-mediated transfection using a 3:1 (w/w) liposome formulation of the polycationic lipid 2,3-dioleyloxy-N-[2(sperminecarboxamido)ethyl]'N,N-dimethyI-l-propaniminium-trifIuoroacetate and the neutral lipid dioleoyl phosphatidylethanolamine in membrane-filetered water (Lipofectamine™ Reagent, Life Technologies), in serum free (SF) media formulation (Ham's F12, 10 mg/ml transferrin, 5 mg/ml insulin, 2 mg/ml fetuin, 1% L-glutamine and 1% sodium pyruvate). Zcyto20/pZMP6 is diluted into 15-ml tubes to a total final volume of 640 fi\ with SF media. 35 fil of Lipofectamine™ is mixed with 605 fil of SF medium. The resulting mixture is added to the DNA mixture and allowed to incubate approximately 30 minutes at room temperature. Five ml of SF media is added to the DNA:Lipofectamme™ mixture. The cells are rinsed once with 5 ml of SF media, aspirated, and the DNA:Lipofectamine™ mixture is added. The cells are incubated at 37°C for five hours, then 6.4 ml of Ham's F12/10% FBS, 1% PSN media is added to each plate. The plates are incubated at 37°C overnight, and the DNA:Lipofectamine™ mixture is replaced with fresh 5% FBS/Ham's media the next day. On day 3 post-transfection, the cells are split into T-175 flasks in growth medium. On day 7 postransfection, the cells are stained with FTTC-anti-CD8 monoclonal antibody (Pharmingen, San Diego, CA) followed by anti-FTTC-conjugated magnetic beads (Miltenyi Biotec, Aubum, CA). The CD8-positive cells are separated using commercially available columns (mini-MACS columns; Miltenyi Biotec) according to the manufacturer's directions and put into DMEM/Ham's F12/5% FBS without nucleosides but with 50 nM methotrexate (selection medium).<br>
Cells are plated for subcloning at a density of 0.5, 1 and 5 cells per well in 96-well dishes in selection medium and allowed to grow out for approximately two weeks. The wells are checked for evaporation of medium and brought back to 200 (i\ per well as necessary during this process. When a large percentage of the colonies in the plate are near confluency, 100 /xl of medium is collected from each well for analysis by dot blot, and the cells are fed with fresh selection medium- The supernatant is applied to a nitrocellulose filter in a dot blot apparatus, and the filter is treated at 100°C in a vacuum oven to denature the protein. The filter is incubated in 625 mM Tris-glycine, pH 9.1, 5mM p-mercaptoethanol, at 65°C, 10 minutes, then in 2.5% non-fat dry milk Western A Buffer (0.25% gelatin, 50 mM Tris-HCI pH 7.4, 150 mM NaCl, 5 mM EDTA, 0.05% Igepal CA-630) overnight at 4°C on a rotating shaker. The filter is incubated with the antibody-HRP conjugate in 2.5% non-fat dry milk Western A buffer for ] hour at room temperature on a rotating shaker. The filter is then washed three times at room temperature in PBS plus 0.01% Tween 20, 15 minutes per wash. The filter is developed with chemiluminescence reagents  (ECL™ direct labelling kit;<br><br>
Amersham Corp., Arlington Heights, IL) according to the manufacturer's directions and exposed to film (Hyperfilm ECL, Amersham Corp.) for approximately 5 minutes. Positive clones are trypsinized from the 96-weIl dish and transferred to 6-well dishes in selection medium for scaleup and analysis by Western blot.<br>
Example 3<br>
Expression in Baby Hamster Kidney Cells<br>
The full-length zcyto24 and zcyto25 proteins were produced in BHK cells. For example, the BHK cells were transfected with either zcyto24-CEE/p2MP2I or zcyto25-CEE/pZMP21 (Example 1). BHK 570 cells {ATCC CRL-10314) were plated in T75 tissue culture flasks and allowed to grow to approximately 50 to 70% confluence overnight at 37°C, 5% C02, in growth medium (SL7V4, 5% fetal bovine serum (Hyclone, Logan, UT), 1% penicillin/strepomycin). The cells were then transfected with zcyto24-CEE/pZMP21 or zcyto25-CEE/pZMP21 by liposome-mediated transfection (using Lipofectamine™; Life Technologies), in serum free (SF) media. The plasmid was diluted into 1.5-ml tubes to a total final volume of 640 ul with SF media. Thirty-five \i\ of the lipid mixture was mixed with 605 (il of SF medium, and the resulting mixture is allowed to incubate approximately 30 minutes at room temperature. Six ml of SF media was then added to the DNA/lipid mixture. The cells were rinsed once with 5 ml of SF media, aspirated, and the DNA/lipid mixture was added. The cells are incubated at 37°C for five hours, then 15 ml of growth medium was added to each plate. The plates were incubated at 37°C overnight, and the DNA/lipid mixture was replaced with selection medium (SL7V4, 5% fetal bovine serum (Hyclone, Logan, UT), 1% penicillin/strepomycin, lfiM MTX) the next day. Approximately 7-10 days post-transfection, methotrexate-resistant colonies were trypsinized, and the cells were re-plated into T-162 flasks and transferred to large-scale culture.<br>
Example 4<br>
Construction of Adenovirus vectors<br>
For construction of adenovirus vectors, the protein coding region of zcyto20, zcyto21, zcyto22, zcyto24 or zcyto25 is amplified by PCR using primers that add Pmel and AscI restriction sites at the 5' and 3' termini respectively. Amplification<br><br>
is performed with a full-length cDNA template in a PCR reaction as follows: one cycle at 95°C for 5 minutes; followed by 15 cycles at 95°C for 1 min., 61°C for 1 min., and 72°C for 1.5 min.; followed by 72°C for 7 min.; followed by a 4°C soak. The PCR reaction product is loaded onto a 1.2% low-melting-temperature agarose gel in TAE buffer (0.04 M Tris-acetate, 0.001 M EDTA). The PCR product is excised from the gel and purified using a commercially available kit comprising a silica gel mambrane spin column (QIAquick® PCR Purification Kit and gel cleanup kit; Qiagen, Inc.) as per kit instructions. The PCR product is then digested with Pmel and AscI, phenol/chloroform extracted, EtOH precipitated, and rehydrated in 20 ml TE (Tris/EDTA pH 8). The zeyto20 fragment is then ligated into the Pmel-AscI sites of the transgenic vector pTG12-8 and transformed into E. coli DH10B™ competent cells by electroporation. Vector pTG12-8 was derived from p2999B4 (Palmiter et al., Mol. Cell Biol 13:5266-5275,1993) by insertion of a rat insulin E intron (ca. 200 bp) and polylinker (Fse VPme I/Asc I) into the Nru I site. The vector comprises a mouse metallothionein (MT-1) promoter (ca. 750 bp) and human growth hormone (hGH) untranslated region and polyadenylation signal (ca. 650 bp) flanked by 10 kb of MT-1 5' flanking sequence and 7 kb of MT-1 3' flanking sequence. The cDNA is inserted between the insulin H and hGH sequences. Clones containing the zcyto20, zcyto21, zcyto22, zcyto24 or zcyto25 cDNA are identified by plasmid DNA miniprep followed by digestion with Pmel and AscI. Positive clones are sequenced to insure that there were no deletions or other anomalies in the construct.<br>
DNA is prepared using a commercially available kit (Maxi Kit, Qiagen, Lie), and the cDNA is released from the pTG12-8 vector using Pmel and AscI enzymes. The cDNA is isolated on a 1% low melting temperature agarose gel and excised from the gel. The gel slice is melted at 70°C, and the DNA is extracted twice with an equal volume of Tris-buffered phenol, precipitated with EtOH, and resuspended in 10 fil H2O.<br>
The cDNA is cloned into the EcoRV-AscI sites of a modified pAdTrack-CMV (He, T-C. et al., Proc. Natl. Acad. Set USA 95:2509-2514, 1998). This construct contains the green fluorescent protein (GFP) marker gene. The CMV promoter driving GFP expression is replaced with the SV40 promoter, and the SV40 polyadenylation signal is replaced with the human growth hormone polyadenylation signal. In addition, the native polylinker is replaced with Fsel, EcoRV, and AscI sites. This modified form of pAdTrack-CMV is named pZyTrack. Ligation is performed using a commercially available DNA ligation and screening kit (Fast-Link® kit; Epicentre Technologies, Madison, WI). Clones containing zcyto20 are identified by digestion of mini prep DNA with Fsel and AscI. In order to linearize the plasmid, approximately 5 ug of the<br><br>
resulting pZyTrack zcyto20, zcyto21, zcyto22, zcyto24 or zcyto25 plasmid is digested with Pmel. Approximately 1 ng of the linearized plasmid is cotransformed with 200 ng of supercoiled pAdEasy (He et al., ibid.) into E. coli BJ5183 cells (He et al., ibid.). The co-transformation is done using a Bio-Rad Gene Pulser at 2.5 kV, 200 ohms and 25 uFa. The entire co-transformation mixture is plated on 4 LB plates containing 25 ng/ml kanamycin. The smallest colonies are picked and expanded in LB/kanamycin, and recombinant adenovirus DNA is identified by standard DNA miniprep procedures. The recombinant adenovirus miniprep DNA is transformed into E. coli DH10B™ competent cells, and DNA is prepared using a Maxi Kit (Qiagen, Inc.) according to kit instructions.<br>
Approximately 5 u,g of recombinant adenoviral DNA is digested with PacI enzyme (New England Biolabs) for 3 hours at 37°C in a reaction volume of 100 ul containing 20-30U of PacI. The digested DNA is extracted twice with an equal volume of phenol/chloroform and precipitated with ethanol. The DNA pellet is resuspended in 10 Hi distilled water. A T25 flask of QBI-293A cells (Quantum Biotechnologies, Inc. Montreal, Qc. Canada), inoculated the day before and grown to 60-70% confluence, is transfected with the PacI digested DNA. The Pad-digested DNA is diluted up to a total volume of 50 )x\ with sterile HBS (150mM NaCl, 20mM HEPES). In a separate tube, 20 uj of lmg/ml N-[i-(2,3-Dioleoyloxy)propyl]-N,N,N-trimethyl-ammonium salts (DOTAP) (Boehringer Mannheim, Indianapolis, IN) is diluted to a total volume of 100 ul with HBS. The DNA is added to the DOTAP, mixed gently by pipeting up and down, and left at room temperature for 15 minutes. The media is removed from the 293A cells and washed with 5 ml serum-free minimum essential medium (MEM) alpha containing ImM sodium pyruvate, 0.1 mM MEM non-essential amino acids, and 25mM HEPES buffer (reagents obtained from Life Technologies, Gaithersburg, MD). Five ml of serum-free MEM is added to the 293A cells and held at 37°C. The DNA/lipid mixture is added drop-wise to the T25 flask of 293A cells, mixed gently, and incubated at 37°C for 4 hours. After 4 hours the media containing the DNA/lipid mixture is aspirated off and replaced with 5 ml complete MEM containing 5% fetal bovine serum. The transfected cells are monitored for GFP expression and formation of foci (viral plaques).<br>
Seven days after transfection of 293A cells with the recombinant adenoviral DNA, the cells express the GFP protein and start to form foci (viral "plaques")- The crude .viral lysate is collected using a cell scraper to collect all of the 293A cells. The lysate is transferred to a 50-ml conical tube. To release most of the virus particles from the cells, three freeze/thaw cycles are done in a dry ice/ethanol bath and a 37°C waterbath.<br><br>
The crude lysate is amplified (Primary (1°) amplification) to obtain a working "stock" of zcyto20 rAdV lysate. Ten 10cm plates of nearly confluent (80-90%) 293A cells are set up 20 hours previously, 200 ml of crude rAdV lysate is added to each 10-cm plate, and the cells are monitored for 48 to 72 hours for CPE (cytopathic effect) under the white light microscope and expression of GFP under the fluorescent microscope. When all of the 293A cells show CPE, this stock lysate is collected and freeze/thaw cycles performed as described above.<br>
A secondary (2°) amplification of zcyto20 rAdV is then performed. Twenty 15-cm tissue culture dishes of 293A cells are prepared so that the cells are 80-90% confluent. All but 20 ml of 5% MEM media is removed, and each dish is inoculated with 300-500 ml of the 1° amplified rAdv lysate. After 48 hours the 293A cells are lysed from virus production, the lysate is collected into 250-ml polypropylene centrifuge bottles, and the rAdV is purified.<br>
NP-40 detergent is added to a final concentration of 0.5% to the bottles of crude lysate in order to lyse all cells. Bottles are placed on a rotating platform for 10 minutes agitating as fast as possible without the bottles falling over. The debris is pelleted fay centrifugation at 20,000 X G for 15 minutes. The supernatant is transferred to 250-ml polycarbonate centrifuge bottles, and 0.5 volume of 20% PEG8000/2.5 M NaCl solution is added. The bottles are shaken overnight on ice. The bottles are centrifuged at 20,000 X G for 15 minutes, and the supernatant is discarded into a bleach solution. Using a sterile cell scraper, the white, virus/PEG precipitate from 2 bottles is resuspended in 2.5 ml PBS. The resulting virus solution is placed in 2-mI microcentrifuge tubes and centrifuged at 14,000 X G in the microcentrifuge for 10 minutes to remove any additional cell debris. The supernatant from the 2-ml microcentrifuge tubes is transferred into a 15-ml polypropylene snapcap tube and adjusted to a density of 1.34 g/ml with CsCl. The solution is transferred to 3.2-ml, polycarbonate, thick-walled centrifuge tubes and spun at 348,000 X G for 3-4 hours at 25fiC. The virus forms a white band. Using wide-bore pipette tips, die virus band is collected.<br>
A commercially available ion-exchange column (e.g., PD-10 columns prepacked with Sephadex® G-25M; Pharmacia Biotech, Piscataway, NX) is used to desalt the virus preparation. The column is equilibrated with 20 ml of PBS. The virus is loaded and allowed to run into the column. 5 ml of PBS is added to the column, and fractions of 8-10 drops, are collected. The optical densities of 1:50 dilutions of each fraction are determined at 260 rim on a spectrophotometer.  Peak fractions are pooled,<br>
and the optical density (OD) of a 1:25 dilution is determined. OD is converted to virus<br>
12 concentration using the formula: (OD at 260nm)(25)(l.l x 10   ) = virions/ml.<br><br>
To store the virus, glycerol is added to the purified virus to a Final concentration of 15%, mixed gently but effectively, and stored in aliquots at -8Q*iC.<br>
A protocol developed by Quantum Biotechnologies, Inc. (Montreal, Canada) is followed to measure recombinant virus infectivity. Briefly, two 96-well tissue culture plates are seeded with 1 X 10 293A cells per well in MEM containing 2% fetal bovine serum for each recombinant virus to be assayed. After 24 hours 10-fold dilutions of each virus from 1X10" to 1X10" are made in MEM containing 2% fetal bovine serum. 100 uj of each dilution is placed in each of 20 wells. After 5 days at 37°C, wells are read either positive or negative for CPE, and a value for "Plaque Forming Units/ml" (PFU) is calculated.<br>
Example 5<br>
Cloning of zcyto20, zcvto22, zcvto24, and zcyto25<br>
A: zcyto20 and zcyto22<br>
PCR primers were designed that were common to both zcyto20 and zcyto22 within the predicted coding sequence. These are designated 2C39339 (SEQ ID NO:47) and ZC39393 (SEQ ID NO:48). PCR was carried out on a panel of human cDNA libraries from different tissues. The product was observed in brain, islet (pancreas), prostate, placenta, testis, HPVS(prostate epithelia), and CD3+ libraries. PCR primers were then designed from the genomic sequence 5' of the starting methionine and 3' of the termination codori for zcyto20 (with high similarity to zcyto22). These were designated ZC39340(SEQ ID NO:49) and ZC39341 (SEQ ID NO:50)." PCR was carried out on the previously identified libraries. Four libraries contained full length clones. Sequencing of PCR products generated from these libraries resulted in three libraries containing zcyto22 (prostate, testis and CD3+), and one library containing zcyto20 (HPVS (prostate epithelia)). PCR using primers specific to the predicted coding sequence of zcyto22 was also carried out. These are designated ZC39295 (SEQ ID NO-.S1) and ZC39298 (SEQ ID NO:52) . Ubraries positive by this PCR were brain, prostate, CD3+, and testis. Sequencing confirmed the zcyto22 sequence.<br>
B:zcyto24 and zcyto25<br>
PCR primers were designed that were common to both zcyto24 and zcyto25 within the predicted coding sequence. These are designated ZC39687 (SEQ ID<br><br>
NO-.53) and ZC39741 (SEQ ID NO:54). PCR was carried out on a panel of mouse cDNA libraries from different tissues. The product was observed in the following libraries: heart, lung, placenta, Torres prostate, skin, small intestine, testis, and thymus. PCR primers were then designed 5' of the starling methionine and at the termination codons. The 5' primer is designated ZC39732 (SEQ ID NO:55) and matches both zcyto24 ands zcyto25 sequences. The zcyto24 3' primer is designated ZC39701 (SEQ ID NO:56) . The zcyto25 3' primer is designated ZC39688 (SEQ ID NO:57). PCR was carried out on the positive libraries indicated above. For zcyto24, all libraries except heart and Torres prostate were positive. Sequencing confirmed zcyto24 sequence from placenta, testis, and small intestine libraries. For zcyto25, all libraries except heart and Torres prostate were positive. Sequencing confirmed zcyto25 sequence from the lung library.<br>
Example 6<br>
Expression in Baculovirus<br>
A: Construct for expression of zcyto20<br>
An expression vector, pzBV37L:zCyto20, was prepared to express zcyto20 polypeptides in insect cells. A 536 bp fragment containing sequence for zcyto20 and encoded BspEl and Xbal restriction sites on the 5' and V ends, respectively, was generated by PCR amplification from a plasmid zcyto20 using primers ZC41932 (SEQ ID NO:14) and ZC41933 (SEQ ID NO:15) utilizing the Expand High Fidelity PCR System (Boerhinger Mannheim) as per manufacturer's instructions. The PCR conditions were as follows: 1 cycle of 94°C for 4 minutes, 30 cycles of 94°C for 30 seconds, 50°C for 30 seconds, and 72°C for 1 minute; 1 cycle at 72°C for 4 min; followed by a 4°C soak. A small portion of the PCR product was visualized by gel electrophoresis (1% NuSieve agarose), and a fragment length of approximately 550 bp was confirmed. The remainder of the reaction mix was purified via the Qiagen PCR purification kit as per manufacturers instructions and eluted in 30 ul water. The cDNA was digested in a 36 ul vol. using Bspel and Xbal (New England Biolabs, Beverly, MA) in appropriate buffer conditions at 37°C. The digested PCR product band was run through a 1% agarose TAE gel, excised and extracted using a QIAquick™ Gel Extraction Kit (Qiagen, Cat. No. 28704) and eluted in 30 ul of EB buffer. The digested zcyto20 PCR product was ligated into the multiple cloning site (MCS) of vector pZBV37L at the Bspel and Xbal sites. The pZBV37L vector is a modification of the pFastBacl™ (Life Technologies) expression vector, where the<br><br>
polyhedron promoter has been removed and replaced with the late activating Basic Protein Promoter and the EGT leader signal sequence upstream of the MCS. 5 pi of the restriction enzyme digested zcyto20 PCR fragment and 4 pi of the corresponding pZBV37L vector were ligated for 72 hours at 15°C in a 20 ul vol. in appropriate buffer conditions. 5 pi of the ligation mix was transformed into 33 pi of ElectoMAX™ DH12s™ cells (Life Technologies, Cat. No. 18312-017) by electroporation at 400 Ohms, 2.00kV and 25 /iF in a 2mm gap electroporation cuvette (BTX, Model No. 620). The transformed cells were diluted in 500|il of LB media and outgrown for Ihr at 37°C and 10 pi and 20 u] of the dilution were plated onto LB plates containing 100 ng/ml ampicillin. Clones were analyzed by PCR and positive clones were selected, plated, and submitted for sequencing. The sequence was then confirmed.<br>
B. Construction and Expression of Tagged zcyto20 An expression vector, pZBV32L:zCyto20cee, was prepared to express zcyto20cee polypeptides in insect cells. PZBV32L:zCyto20cee was designed to express a zcyto20 polypeptide with a C-terminal GLU-GLU tag (SEQ ID NO: 16). This construct can be used to determine the N-terminal amino acid sequence of zcyto20 after the signal peptide has been cleaved off.<br>
1. Construction of pZBV32L:zCyto20cee<br>
A 625bp zcyto20 fragment containing BamHI and Xbal restriction sites on the 5' and 3' ends, respectively, was generated by PCR amplification from a plasmid containing zcyto20 cDNA using primers zc40240 and zc40241 (SEQ ID NOS: 17 and 18, respectively). The PCR reaction conditions were as follows: The Expand High Fidelity PCR System (Boehringer Mannheim) was utilized for a 100 pJ volume reaction containing 5% DMSO. 1 cycle at 94°C for 4 minutes; 30 cycles of 94°C for 30 seconds, 50°C for 30 seconds, and 72°C for 60 seconds; 1 cycle at 72°C for 4 min; followed by 4°C soak. Five p.1 of the PCR fragment was visualized by gel electrophoresis (1% NuSieve agarose). The remainder of the reaction mix was purified via Qiagen PCR purification kit as per manufacturers instructions and eluted in 30 u-1 water. The cDNA was digested in a 35 j^l volume using BamHI and Xbal (New England Biolabs, Beverly, MA) in appropriate buffer conditions for 2hrs at 37°C. The digested PCR product band was run through a 1% agarose TAE gel, excised and extracted using a QIAquick™ Gel Extraction Kit (Qiagen) and eluted in 30 (il of water. The purified, digested zCyto20cee PCR product was ligated into the multiple cloning site of a previously prepared and restriction enzyme digested (BamHI and Xbal) vector pZBV32L. The pZBV32L vector is a modification of the pFastBacl™ (Life Technologies) expression<br><br>
vector, where the polyhedron promoter has been removed and replaced with the late activating Basic Protein Promoter, and the coding sequence for the GIu-Glu tag (SEQ ID NO: 10) as well as a stop signal was inserted at the 3' end of the multiple cloning region. Five (4.1 of the restriction digested zCyto20 insert and 40 ng of the corresponding pZBV32L vector were ligated overnight at 16°C in a 20 u\I vol. Five \xl of the ligation mix was transformed into 50 u,l of ElectoMAX™ DH12sTW cells (Life Technologies) by electroporation at 4-00 Ohms, 2.00kV and 25 [iF in a 2mm gap electroporation cuvette. The transformed cells were diluted in 500 Jil of SOC media (2% Bacto Tryptone, 0.5% Bacto Yeast Extract, 10 ml 1M NaCl, 1.5 mM KC1, 10 mM MgCl2, 10 mM MgS04 and 20 mM glucose) and 50 /zl of the dilution were plated onto LB plates containing 100 ug/ml ampicillin. Clones were analyzed by PCR and restriction digestion. Positive clones were selected and plated for sequencing. Once proper sequence was confirmed, 25 ng of positive clone DNA was transformed into 66 ]x\ DHlOBac™ Max Efficiency* competent cells (GIBCO-BRL) by heat shock for 45 seconds in a 42°C heat block. The transformed DHlOBac™ cells were diluted in 600 ul SOC media (2% Bacto Tryptone, 0.5% Bacto Yeast Extract, 10 ml 1M NaCl, 1.5 mM KC1, 10 mM MgCl2, 10 mM MgS04 and 20 mM glucose) and grown at 37°C for lhi and lOOju.1 were plated onto Luria Agar plates containing 50 jig/ml kanamycin, 7 p-g/ml gentamicin, 10 u,g/ml tetracycline, 40 jtg/mL IPTG and 200 /ig/mL Bluo Gal. The plates were incubated for 48 hours at 37°C. A color selection was used to identify those cells having transposed viral DNA (referred to as a "bacmid"). Those colonies, which were white in color, were picked for analysis. Colonies were analyzed by PCR and positive colonies (containing desired bacmid) were selected for growth and subsequent bacmid DNA purification. Clones were screened for the correct molecular weight insert by PCR amplification of DNA using primers to the transposable element in the bacmid: ZC447 (SEQ ID NO: 19) and ZC976 (SEQ ID NO:20). The PCR reaction conditions were as follows: 1 cycle at 94°C for 4 minutes; 25 cycles of 94°C for 30 seconds, 50°C for 30 seconds, and 72°C for 2.5 min; 1 cycle at 72°C for 4 min; followed by 4°C soak. The PCR product was run on a 1 % agarose gel to confirm the insert size. Those having the correct size insert were used to transfect Spodoptera Frugiperda (Sf9) cells.<br>
2. Transfection<br>
Sf9 cells, were seeded at 1 x 106 cells per well in a 6-well plate and allowed to attach for 1 hour at 27°C. Approximately 5 jig of bacmid DNA were diluted to 100 |il with Sf-900 H SFM (Life Technologies). Twenty ul of Lipofectamine™ Reagent (Life Technologies) were diluted to 100 |Al with Sf-900 II SFM.  The bacmid<br><br>
DNA and lipid solutions were gently mixed and incubated for 45 minutes at room temperature. Eight hundred \L\ of Sf-900 U SFM was added to the lipid-DNA mixture. The media was aspirated from the well and the 1 ml of DNA-lipid mix added to the cells. The cells were incubated at 27°C overnight. The DNA-lipid mix was aspirated from each well and was replaced with 2 ml of Sf-900 II media. The plates were incubated at 27°C, 90% humidity, for approximately 7 days after which the virus was harvested.<br>
3. Amplification<br>
Sf9 cells were seeded at 1 x 106 cells per well in a 6-well plate. Five hundred fil of virus from the transfection plate were placed in the well and the plate was incubated at 27°C, 90% humidity, for 96 hours after which the virus was harvested (primary amplification).<br>
The second round of amplification was carried out by transferring 100 \x\ of virus from the above primary amplification plate to wells containing 1 x 106 cells per well. The plate was incubated for 96 hours before harvesting.<br>
An additional round of amplification was performed (tertiary amp.) Sf9 cells were grown in 50 ml Sf-900 H SFM in a 250 ml shake flask to an approximate density of 1 x 106 cells/ml. They were men infected with 1 ml of the viral stock from the above plate and incubated at 27°C for 6 days after which time the virus was harvested.<br>
The viral stock was titered by a growth inhibition curve and the titer culture that indicated a Multiplicity Of Infection (MOI) of one was allowed to proceed for a&gt; total of 48 hrs. The supernatant was analyzed via a reduced Western using a primary-monoclonal antibody specific for the Glu-Glu tag followed by an HRP-conjugated goat anti-murine secondary antibody. Results indicated a band of approximately 2Q kDa. Supernatant was also provided for activity analysis,<br>
A large viral stock was then generated by the following method; Sf9 cells were grown in 1L Sf-900 H SFM in a 2800 ml shake flask to an approximate density of 1 x 10fi cells/ml. They were then infected with 5 ml of the viral stock from the above flask and incubated at 27°C for 4 days after which time the virus was harvested.<br>
Larger scale infections were completed to provide material for downstream purificatioi).<br>
Similarly, zcyto21 and zcyto22 were expressed in Baculovirus<br><br>
Example 7 Purification of Protein<br>
Recombinant protein can be made for any of the protein described herein.<br>
A.	Purification of zcyto20<br>
Recombinant carboxyl terminal Glu-Glu tagged zcyto20 was produced from either recombinant baculovirus infected insect cells, stable or transient BHK cell lines. Cultures were harvested, and the media were sterile filtered using a 0.2 Dm filter.<br>
Proteins were purified from the conditioned media by a combination of Anti-Glu-Glu (Anti-EE) peptide antibody affinity chromatography and Superdex 75 gel exclusion chromatography. Culture medium from BV (pH 6.0, conductivity 7 mS) was adjusted to pH 6.7! Both BV and BHK media were then added NaCl to 300mM before loading onto a 10x70mm (5-ml column volume) Poros Protein A anti-EE antibody affinity column at a flow of 2-5 ml/minute. The column was then washed with five column volumes (CV) of 5xPBS (pH 7.2). Bound protein was eluted with 0.5 M acetic acid, 0.5 M NaCl, pH 3.0. Two-ml fractions were collected, and the eluant was neutralized by the addition of 2 M Tris. Samples from the anti-EE antibody affinity column were analyzed by SDS-PAGE with silver staining and western blotting for the presence of zcyto20 protein. Zcyto20 protein-containing fractions were pooled and concentrated to about 2 mis using Biomax-5 concentrator (Millipore), and loaded onto a 16 x600 mm Superdex 75 gel filtration column (Amersham Pharmacia Biotech). The fractions containing purified zcyto20 protein were pooled, filtered through 0.2 /im filter, aliquoted into 100 /zl each, and frozen at -80°C. The concentration of the final purified protein was determined by BCA assay (Pierce, Rockford, IL) and HPLC-amino acid analysis.<br>
B.	SDS-PAGE and Western blotting analysis of zcyto20 proteins<br>
Recombinant zcyto20 protein was analyzed by SDS-PAGE (Nupage 4-<br>
12% Bis-Tris, Invitrogen, Calsbad, CA) with silver staining method (Fast Silver, Geno Technology, Inc., St. Louis, MO) and Western blotting using the anti-EE antibody. Either the conditioned .media or purified protein was electrophoresed using an Xcell H™ MINI-CELL (Invitrogen, Calsbad, CA) and transferred to nitrocellulose (0.2 Cm; Bio-Rad Laboratories, Hercules, CA) at room temperature using Xcell n™ blot module (Invitrogen) with stirring according to directions provided in the instrument manual.<br><br>
The transfer was run at 500 mA for 45 minutes in a buffer containing 25 mM Tris base, 200 mM glycine, and 20% methanol. The filters were then blocked with 10% non-fat dry milk in PBS for 10 minutes at room temperature. The nitrocellulose was quickly rinsed, and then primary antibody was added in PBS containing 2.5% non-fat dry milk. The blots were incubated for two hours at room temperature or overnight at 4°C with gentle shaking. Following the incubation, blots were washed three times for 10 minutes each in PBS. Secondary antibody (rabbit anti-mouse IgG conjugated to horseradish peroxidase; obtained from Pierce Chemical Co., Rockford, IL) diluted 1:2000 in PBS containing 2.5% non-fat dry milk was added, and the blots were incubated for two hours at room temperature with gentle shaking. The blots were then washed three times, 10 minutes each, in PBS, then quickly rinsed in H2O. The blots were developed using commercially available chemiluminescent substrate reagents (SuperSignal® ULTRA reagents 1 and 2 mixed 1:1; reagents obtained from Pierce Chemical Co.), and the signal was captured using Lumi-Imager's Lumi Analyst 3.0 software (Boehringer Mannheim GmbH, Germany) for times ranging from 10 second to 5 minutes or as necessary.<br>
C. Summary of protein purification and analysis The purified zcyto20-CEE protein from BV media migrated as a 21 kDa monomer on the 4-12% Bis-Tris gel, however, a minor 36 kDa dimer band was also observed. The dimer protein became monomer band upon reduction using reducing agent, suggesting the dimerization of zcyto20-CEE by disulfide bond, and it is consistent with the odd number (a total of seven) of cysteine residues resulting in interchain disulfide bond.<br>
Zcyto21, zcyto22, zcyto24, and zcyto25 polypeptides were purified in a similar manner.<br>
Example 8<br>
Interferon-like Transcriptional Regulatory Regions of zcyto2Q<br>
Local sequence alignments of the characterized IFN-a, EfN-fJ, IFN-y promoters and those of a number of other cytokines with the upstream regions of zcyto20, zcyto21, and zcyto24 were performed to identify whether these genes were co-regulated. The hypothesis was that common properties of gene regulation should be reflected in sequence similarities of the regions upstream of the genes.<br>
Due to the low binding specificity of TFs, predictions of individual binding sites have a high rate of false positives. Therefore, the site predictions in<br><br>
isolation for the identification of binding sites with functional roles in vivo are not useful. However, predicted binding sites likely to have sequence-specific functions can be selected by means of a conservation-based filter: The biological observation that regulatory regions are often more strongly conserved between species than other non-coding regions can be quantified to reveal patterns of conservation which have been called phylogeneric footprints (Fickett, et al., Curr. Opin, BiotechnoL JJ,; 19-24, 2000.} In particular, human-rodent comparisons have proven a valuable resource for the identification of functional regulatory elements (Wasserman et al., Nat. Genet. 26: 225-228, 2000).<br>
The alignments revealed significant matches mainly among the promoters of the various IFN-a genes, whereas, based on this analysis, there is little evidence for similarity between the promoters of zcyto20-22 and other cytokines.<br>
The pairwise sequence alignments were performed with DBA (Jareborg et al., Genome. Res. 9: 815-824, 1999). The search for individual transcription factor binding sites was performed with standard position weight matrices (Fickett, Mol. Cell. Biol. 16:437-441,1996) drawn from the TRANSFAC database (version 3.0, Wingender et al., Nucleic Acids Res. 28; 316-319, 2000), and alignments of regions 1-4 were computed with SSEARCH, version 3.U12 (Pearson, Genomics 11:635-650. 1991). Based on published studies involving other sets of TFs, most natural binding sites sufficiently conserved between mouse and human can be expected to be detected in the used score range (Fickett, Mol. Cell. Biol. 16: 437-4411996; Wasserman et al., J, Mol. Biol. 278: 167-181,1998).<br>
The computational results suggest that features common to the regulation of the zcyto20 family and other cytokines that reflected by sequence similarity would be expected on the level of individual binding sites rather than matches covering extended regions.<br>
Based on a search of binding sites of a set of 32 TFs, the identification of putative sites of a limited number of factors largely known to be involved in the transcriptional regulation of interferons were made. The results of the comparison show putative binding sites of TFs playing important roles in the transcriptional regulation of IFN-p [NF-KB, ISRE (element binding IRFs)] and IFN-y (AP-1, CREB, GATA, NF-KB, NF-AT) are also present in the conserved non-coding regions. For example, a pair of neighboring AP-1/NF-AT sites in region 1, is a well-documented example of cooperative binding occurring in many cytokine promoters (Holloway et al., Mol. Immunol. 38: 567-580,2000).<br><br>
The binding sites for TFs that have been identified as critical for cytokine regulation, suggest that the cluster of transcription factor binding sites in the promoter region of zcyto20 is a candidate for a functional region.<br>
The alignment of region 2 with the group of known cytokines yielded a match in the AK155 promoter, at position -415 with respect to the transcription start site of AK155 as given in Knappe et al. fJ. Virol. 74: 3881-3887, 2000). The fact that there is a putative NF-KB binding site at that location in the zcyto20 promoter indicates the possible presence of a NF-KB site in the AK155 promoter.<br>
Example 9 Transgenic Animals<br>
Transgenic animals expressing Zcyto21 genes were produced using adult, fertile males (studs) (B6C3fl, 2-8 months of age (Taconic Farms, Germantown, KY)), vasectomized males (duds) (CD1, 2-8 months, (Taconic Farms)), prepubescent fertile females (donors) (B6C3f 1,4-5 weeks, (Taconic Farms)) and adult fertile females (recipients) (CD1,2-4 months, (Taconic Farms)).<br>
The donors were acclimated for 1 week and then injected with approximately 8 IU/mouse of Pregnant Mwere's Serum gonadotrophin (Sigma, St. Louis, MO) I.P., and 46-47 hours later, 8 IU/mouse of human Chorionic Gonadotropin (hCG (Sigma)) LP. to induce superovulation. Donors were mated with studs subsequent to hormone injections. Ovulation generally occurs within 13 hours of hCG injection. Copulation was confirmed by the presence of a vaginal plug the morning following mating.<br>
Fertilized eggs were collected under a surgical scope (Leica MZ12<br>
Stereo Microscope, Leica, Wetzlar, DE). The oviducts were collected and eggs were<br>
released into urinanalysis slides containing hyaluronidase (Sigma). Eggs were washed<br>
once in hyaluronidase, and twice in Whitten's W640 medium (Table 7) that has been incubated with 5% CO2, 5% O2, and 90% N2 at 37°C. The eggs were then stored in a<br>
37°C/5% CO2 incubator until microinjection.<br>
Ten to twenty micrograms of plasmid DNA containing a cDNA of the Zcyto21gene was linearized, gel-purified, and resuspended in 10 mM Tris pH 7.4, 0.25 mM EDTA pH 8.0, aj a final concentration of 5-10 nanograms per microliter for microinjection.<br>
Plasmid DNA was microinjected into harvested eggs contained in a drop of W640 medium overlaid by warm, CO2 -equilibrated mineral oil. The DNA was<br><br>
drawn into an injection needle (pulled from a 0.75mm ID, 1mm OD borosihcate glass capillary), and injected into individual eggs. Each egg was penetrated with the injection needle, into one or both of the hapJoid pronuclei.<br>
Picoliters of DNA were injected into the pronuclei, and the injection needle withdrawn without coming into contact with the nucleoli. The procedure was repeated until all the eggs were injected. Successfully microinjected eggs were transferred into an organ tissue-culture dish with pregassed W640 medium for storage overnight in a 37°C/5% CO2 incubator.<br>
The following day, 12-17 healthy 2-cell embryos from the previous day's injection were transferred into the recipient. The swollen ampulla was located and holding the oviduct between the ampulla and the bursa, a nick in the oviduct was made with a 28 g needle close to the bursa, making sure not to tear the ampulla or the bursa. The embryos were implanted through this nick, and by holding onto the peritoneal wall, the reproductive organs were guided back into the abdominal cavity.<br>
The recipients were returned to cages in pairs, and allowed 19-21 days gestation. Animals injected with the Zcyto21 cDNA died before birth.<br><br>
Table 7		<br>
WHTTTEN'S 640 MEDIA	<br>
	mes/200m	mes/500/ml<br>
NaCl	1280	3200<br>
KC1	72	180<br>
KH2P04	32	80<br>
MgS04-7H20	60	150<br>
Glucose	200	500<br>
Ca2+ Lactate	106	265<br>
KPenn	15	37.5<br>
Streptomycin SO4	10	25<br>
NaHC03	380	950<br>
Na Pyruvate	5	12.5<br>
H20	200	500<br>
EDTA	100 jxl	250/il<br>
5% Phenol Red	200 ji\	500/il<br>
BSA-   ..	600	1500<br>
All reagents are available from Sigma.<br><br>
Example 10 Antibody Production<br>
A: zcytorl9 Polyclonal Antibodies<br>
Polyclonal antibodies are prepared by immunizing 2 female New Zealand white rabbits with the purified recombinant protein buzcytorl9/MBP-6H, The rabbits are each given an initial intraperitoneal (ip) injection of 200 (ig of purified protein in Complete Freund's Adjuvant followed by booster ip injections of 100 ug peptide in Incomplete Freund's Adjuvant every three weeks. Seven to ten days after the administration of the second booster injection (3 total injections), the animals are bled and the serum is collected. The animals are then boosted and bled every three weeks.<br>
The huzcyotrl9/MBP-6H specific rabbit serum is pre-adsorbed of anti-MBP antibodies using a CNBr-SEPHAROSE 4B protein column (Pharmacia LKB, Peapack, N. J.) that is prepared using 10 mg of purified recombinant MBP per gram of CNBr-SEPHAROSE. The huzcytorl9-specific polyclonal antibodies are affinity purified from the rabbit serum using a CNBr-SEPHAROSE 4B protein column that is prepared using 10 mg of the specific antigen purified recombinant protein huzcytorl9/MBP-6H followed by 20X dialysis in PBS overnight. Huzcytorl9-specific antibodies are characterized by ELISA using 500ng/ml of the purified recombinant proteins huzcytorl9/MBP-6H or huzcytorl9-Fc4 as antibody targets. The lower limit of detection (LLD) of the rabbit anti-huzcytorl9VMBP-6H affinity purified antibody on its specific purified recombinant antigen huzcytorl9/MBP-6H and on purified recombinant huzcytorl9-Fc4 is determined.<br>
B: zcvto20, zcvto21. zcvto22. zcyto24, and zcvto25 Polyclonal Antibodies<br>
Polyclonal antibodies are prepared by immunizing female New Zealand white rabbits with the purified recombinant protein zcyto20/MBP-6H, zcyto21/MBP-6H, and zcyto22/MBP-6H, as well as mouse zcyto24/MBP-6H, or zcyto25/MBP-6H. The rabbits are each given an initial intraperitoneal (ip) injection of 200 ug of purified protein in Complete Freund's Adjuvant followed by booster ip injections of 100 u.g peptide in Incomplete Freund's Adjuvant every three weeks. Seven to ten days after the administration of the second booster injection (3 total injections), the animals are bled and the serum is collected. The animals are then boosted and bled every three weeks.<br>
The zcyto20/MBP-6H, zcyto2l/MBP-6H, zcyto22/MBP-6H, zcyto24/MBP-6H, or zcyto25/MBP-6H specific rabbit serum can be pre-adsorbed of anti-MBP antibodies using a CNBr-SEPHAROSE 4B protein column (Pharmacia LKB,<br><br>
Peapack, N. J.) that is prepared using 10 mg of purified recombinant MBP per gram of CNBr-SEPHAROSE. The zcyto20, zcyto21, zcyto22, zcyto24, or zcyto25-specific polyclonal antibodies are affinity purified from the rabbit serum using a CNBr-SEPHAROSE 4B protein column that is prepared using 10 mg of the specific antigen purified recombinant protein followed by 20X dialysis in PBS overnight. Antibodies are characterized by EUSA using 500ng/ml of the purified recombinant proteins as antibody targets. The lower limit of detection (LLD) of the purified antibody on its specific purified recombinant antigen is determined.<br>
Polyclonal antibodies were prepared for zcyto21 using a similar protocol by immunizing rabbits with zcyto21-CEE tagged protein, and the antibodies were purified.<br>
Example 11<br>
Expression of the zcvto20, zcvto21. zcvto22, zcyto24, and zcvto25 Genes following Polvinosinic acid-PoIycytidylic acid Induction<br>
A: Various Cell Types<br>
Cultures of primary cells (normal human bronchial epithelial cells, normal human dermal fibroblasts, human umbilical vein endothelial cells, human microvascular endothelial cells and human smooth muscle cells; CLONETICS Corporation; Walkersville, MD) and human choriocarcinoma cell lines (Jar, BeWo, and 3A-Sub E cells; ATCC, Manassas, VA) were grown in the presence of polyinosinic acid-pplycytidylic acid (poly I:C; 100 ug/ml) (SIGMA; St. Louis, MO) or in medium alone. In some instances lOng/ml hTNFa or lOng/ml hJJLlb were also tested. After four hours of incubation, total RNA was isolated from cells and treated with RNase-free DNase. One microgram of total RNA was used for first-stand cDNA synthesis using the Advantage RT-for-PCR kit as suggested by the manufacturer (Clontech, Palo Alto, CA). Five percent of the cDNA reaction was used for polymerase chain reaction as suggested by the manufacturer (Clontech) using the following primer pairs: zcyto20, ZC40134 (SEQ ID NO:30), ZC40214 (SEQ ID NO:31); zcyto21, ZC40209 (SEQ ID NO:32), ZC40213 (SEQ ID NO:33); zcyto22, ZC39295 (SEQ ID NO:34), and ZC39298 (SEQ ID NO:35). Primers to G3PDH were used as a control.<br>
Zcyto20 mRNA was detected at low levels in all cell types tested. . Increased zcyto20 mRNA levels were seen in in NHBE, HUVEC, JAR, 3-A Sub E and BeWo cells stimulated with poly I:C. Zcyto22 mRNA was detected at low levels in all<br><br>
cell types tested. Increased zcyto22 mRNA levels were seen in in NHBE, JAR, 3-A Sub E and BeWo cells stimulated with poly I:C. These results indicate that zcyto20 and zcyto22 mRNA synthesis is stimulated by the known interferon inducer, poly I:C.<br>
Zcyto21 mRNA was detected at low levels in all cell types tested. Increased zcyto21 mRNA levels were seen in in NHBE, HUVEC, NHDF, SMC, HMVEC, JAR, 3-A Sub E and BeWo cells stimulated with poly I:C. Increased zcyto2l mRNA levels were also seen in ILlb-treated 3-A Sub E placental cells. These results indicate that zcyto21 mRNA synthesis is stimulated by the known interferon inducer, poly I:C and can also be stimulated by the cytokine ILlb in certain cell types.<br>
B: Peripheral blood mononuclear cells<br>
Whole peripheral blood mononuclear cells were isolated from human blood using Ficoll Hypaque. T-cells were were purified from peripheral blood mononuclear cells by VarioMacs positive selection columns as per manufacturer's instructions (Miltenyi Biotec Inc., Auburn, CA). Samples from each population were stained and analyzed by fluorescent antibody cell sorting (FACS) (Bectin Dickinson, San Jose, CA) analysis to determine the percent enrichment. The CD3+ T-cells were approximately 95% purified. Whole peripheral blood leukocytes or CD3+ T cells were grown in the presence of poly I:C (100 ug/ml) or in medium alone. After four hours of incubation, total RNA was isolated from cells and treated with RNAse-free DNAse. RT-PCR was performed with the Superscript One-Step RT-PCR with Platinum Taq kit (Invitrogen, Frederick, MD) using 100 ng total RNA as a template for cDNA synthesis. The PCR primer pairs used were: zcyto20: ZC40134 (SEQ ID NO:30) and 2C40214 (SEQ ID NO:31); zcyto21: ZC40209 (SEQ ID NO:32) and ZC40213 (SEQ. ID NO:33); zcyto22: ZC39295 (SEQ ID NO:34) and ZC39298 (SEQ ID NO:35). Aliquots of each RNA, were also tested with primer pairs specific for MHC Class I (Clontech) as a control.<br>
Zcyto20 mRNA was detected in whole peripheral blood mononuclear cells stimulated with poly I:C. The results indicate that zcyto20 mRNA synthesis is stimulated by the known interferon inducer, poly I:C in peripheral blood mononuclear cells.<br>
Zcyto21 mRNA was detected in whole peripheral blood mononuclear cells and CD3+ T-cel!s stimulated with poly I:C. The results indicate that zcyto21 mRNA synthesis is stimulated by the known interferon inducer, poly I:C in peripheral blood mononuclear cells including CD3+T cells.<br><br>
Zcyto22 mRNA was detected in whole peripheral blood mononuclear cells and CD3+ T-cells stimulated with poly I:C. The results indicate that zcyto22 mRNA synthesis is stimulated by the known interferon inducer, poly I:C in peripheral blood mononuclear cells including CD3+ T cells.<br>
These results are indicative of the effect poly I:C would have on zcyto24 and zcyto25 as well as other family members.<br>
Example 12<br>
Expression Analysis of Human Primary Immune Cell and Immune Cell Lines Using RT-PCR<br>
A panel of RNAs from primary human immune cell populations and human immune cell lines was screened for zcyto21, zcyto21, and zcyto22 expression using RT-PCR. The panels were made in house and contained RNA from sixteen various resting and activated cells as described below. All primary immune cell populations were isolated from the blood of several anonymous donors. Various immune cell subsets (CD3+, CD14+, CD19+, and CD56+) were then isolated using labeled Microbeads and the Magnetic Cell Separation System from Miltenyi Biotec. RNA was prepared using the RNeasy Midiprep™ Kit (Qiagen, Valencia, CA) as per manufacturer's instruction. CD56+ NK cell RNA was isolated from cells in tneir resting state. One CD3+ population was activated using a combination of 500ng/ml Ionomycin and 5.0ng/ml PMA (phorbol 12-myristate 13 acetate). Another CD3+ population was stimulated using the supernatant from Conconavalin A stimulated rat splenocytes, a media known to be rich in cytokines and growth factors. CD3+ cells were collected for RNA isolation at activation times of 0, 4 and 16 hours. The CD19+ samples were isolated from human tonsil and activated witii 0.5ug/ml ionomycin and lOng/ml PMA. Cells were then collected at 0, 4 hours and 24 hours and RNA isolated. Human CD14+ monocytes were activated with either O.lng/ml lipopolysaccharide (LPS) or l.Ong/ml LPS for 20 hours. Resting and activated cells were then collected and the RNA was isolated. In addition, RNA was isolated from resting and activated human monocyte cell lines HL-60, THP-1 and U937. HL-60 cells were activated overnight with lOng/ml PMA. THP-1 cells were activated overnight with l.Ong/ml LPS + lOng/ml IFNgamma. Finally, U937 cells were activated overnight with lOng/ml PMA. RT-PCR was performed with the Superscript One-Step RT-PCR with Platinum Taq kit (Invitrogen) using 100 ng total RNA as a template for cDNA synthesis. The PCR primer pairs used were: zcyto20: ZC40632 (SEQ ID NO:36) and ZC40633 (SEQ<br><br>
ID NO:37); zcyto21: ZC40209 (SEQ ID NO:32) and ZC40213 (SEQ ID NO:33); and zcyto22: ZC40638 (SEQ ID NO:38) and ZC40639 (SEQ ID NO:39). Aliquots of each RNA, were also tested with primer pairs specific for MHC Class I (Clontech) as a control.<br>
Zcyto20 mRNA was detected in THP-1 ceils treated with LPS and interferon gamma. Zcyto20 mRNA was also detected in CD3+ cells treated with PMA for 4h. Zcyto21 mRNA was detected at low levels in resting U937 cells and resting THP-1 cells. The level of zcyto21 mRNA was increased upon treatment of the THP-1 cells with LPS and interferon gamma. Zcyto21 mRNA was also detected in CD3+ cells treated with PMA for 4h. Zcyto22 mRNA was detected in THP-1 cells treated with LPS and interferon gamma. Zcyto22 mRNA was also detected in CD3+ cells treated with PMA for 4h and 16h as well as CD3+ cells treated with Conconavalin A for 4 and 16h. The results indicate that zcyto20, zcyto21, and zcyto22 mRNA synthesis is stimulated by activation of a monocyte cell line and primary CD3+ immune cells.<br>
These results are indicative of the effect poly I:C would have on zcyto24 and zcyto25 as well as other family members.<br>
Example 13<br>
Antiviral Activity: Cvtopathic Effect in Hela and L929 cells<br>
Initial functional assays for antiviral activity were conducted using conditioned media from transiently transfected human embryonal kidney (HEK) cells. Production of this conditioned medium is described as follows. A full-length cDNA for zcyto20, zcyto21, zcyto22, zcyto24, or zcyto25 was cloned into the pzp7Z vector using standard procedures. The zcyto20, zcyto21, zcyto22, zcyto24, or zcyto25 constructs were transfected into 293 HEK cells. Briefly, for each construct 700,000 cells/well (6 well plates) were plated approximately 18h prior to transfection in 2 milliliters DMEM + 10% fetal bovine serum. Per well, 1.5 micrograms zcyto20, zcyto21, zcyto22, zcyto24, or zcyto25 DNA and 0.5 micrograms pIRES2-EGFP DNA (Clontech) were added to 6 microliters Fugene 6 reagent (Roche Biochemicals) in a total of 100 microliters DMEM. Two micrograms pIRES2-EGFP DNA alone was used as a negative control. These transfection mixtures were added 30 minutes later to the pre-plated 293 cells. Twenty-four hours later the cell media were removed and DMEM + 0.1% bovine serum albumin was added. Conditioned media was collected after 48 hours, filtered through a 0.45 micron filter and used for antiviral and reporter assays.<br><br>
Antiviral Assays were carried out using human cervical carcinoma cells (HeLa) and mouse fibroblast cells (L929). On the first day, conditioned medium containing zcyto20, zcyto21, zcyto22, zcyto24, or zcyto25 (See Example 10) was diluted and plated with 50,000 cells in a 96-well flat bottom microtiter plate. Following a 24-hour incubation at 37°C, the medium was removed and replaced with medium containing encephelomyocarditis virus al a multiplicity of infection of 0.1. The cells were again incubated for 24 hours at 37°C. Culture wells were then scored visually on a 4-point scale for the presence of cytopathic effect, which was then converted to %CFE as shown in Table 8. Conditioned medium from cells transfected with GFP alone and purified human interferon-a-2a or murine interferon-alpha were included as controls.<br>
Table 8: Determination of Cytopathic Effect<br>
Design¬ation	Observation of Cytopathic Effect (CPE)<br>
-	No CPE<br>
+/-	Possible CPE (about 1% of monolayer surface)<br>
+	CPE limited to one plaque (about 5% of the surface)<br>
+1	CPE is limited to three plaques, affecting less than 25% of the monolayer<br>
1	25% CPE<br>
1-2	37% CPE<br>
2	50% CPE<br>
2-3	62% CPE<br>
3	75% CPE<br>
3-4'-	87% CPE<br>
4	100% CPE<br>
Table 9 shows that conditioned medium containing zcyto20, zcyto21, zcyto22, zcyto24, and zcyto25 inhibited viral infection (%CPE) in HeLa cells in a dose-dependent manner, while control GFP conditioned medium failed to significantly block the appearance of cytopathic effect. As shown in Table 10, conditioned medium containing 2cyto20, zcyto21, zcyto22, zcyto24, and zcyto25 did not inhibit viral infection in L929 cells. In both experiments purified interferon showed positive antiviral activity.<br>
Table 9: Percentage Cytopathic Effect of zcyto20, zcyto21, zcyto22, zcyto24, and zcyto25 in HeLa Cells using Conditioned Medium (CM)<br><br>
Relative CM  Control zcyto20 zcyto21 zcyto22 zcyto24 zcyto25 hlFN- hIFN-a-2a<br>
Concentration GFP      (CM)    (CM)    (CM)    (CM)    (CM)    a^2a    Concentration<br>
No Add	87	87        87	87	87        87        87       Ong/ml<br><br>
.008X	87	10	56	0	0	10	15	.0001 ng/ml<br>
.0156X	87	2.5	31	0	0	5	8.3	.001 ng/ml<br>
.0325X	87	5	10	0	0	5	1.7	.01 ng/ml<br>
.0625X	87	2.5	10	0	0	0	0	.1 ng/ml<br>
.125X	87	0	5	0	0	0	0	1 ng/mj<br>
.25X	87	0	0	0	0	0	0	10 ng/ml<br>
.5X	87	0	0	0	0	0	0	100 ng/ml<br>
Table 10: Percentage Cytopathic Effect of zcyto20, zcyto21, zcyto22,<br>
zcyto24, and zcyto25 in 1329 Cells using Conditioned Medium (CM)	<br>
Relative Control     zcyto20 zcyto2 zcyto2 zcyto24     zcyto25     mIFN-    mIFN-alpha<br>
CM	GFP	(CM)      1	2	(CM)	(CM)        alpha      Cone.<br>
Cone.	(CM)  (CM)	<br>
No Add   87	87	87        87        87	87	87	Ong/ml<br><br>
.008X	87	87	87	87	87	87	87	.0001 ng/ml<br>
.0156X	87	87	87	87	87	87	87	.001 ng/ml<br>
.0325X	87	87	87	87	87	87	87	.01 ng/ml<br>
.0625X	87	87	87	87	87	87	58	.1 ng/ml<br>
.125X	87	87	87	87	87	87	6.7	1 ng/ml<br>
.25X	87	87	87	87	87	87	0	10 ng/ml<br>
.5X	87	87	87	87	87	87	0	100 ng/ml<br>
As a follow up, conditioned medium from Sf9 cells infected with baculovirus expressing zcyto20, zcyto21, zcyto22, zcyto24, and zcyto25 was used in antiviral assays. Conditioned medium from Sf9 cells infected with wild type baculovirus was used as a negative control.<br>
The results of the antiviral assay using the baculovirus-derived conditioned medium were similar to that using the 293 transient transfected conditioned medium. Table 11 shows that baculovirus-derived conditioned medium containing zcyto21 inhibited viral infection in HeLa cells in a dose-dependent manner, while<br><br>
control baculovirus conditioned medium failed to block the appearance of cytopathic effect.<br>
Table   11;   Percentage   Cytopathic   Effect   in   HeLa   Celts   using<br>
Baculovirus-derived 2cyto2J Conditioned Medium (CM)	<br>
Relative   CM zcyto21 CM   Control BV CM   hIFN-a-2a      hIFN-a-2a<br>
Concentration		Concentration<br>
No Add	51	51	51	Ong/ml<br><br>
.008X	5.5	56	17.5	.001 ng/ml-<br>
.0156X	2.5	62	7.5	.01 ng/ml<br>
.0325X	5	56	0	.1 ng/ml<br>
.0625X	2.5	50	0	1 ng/ml<br>
.125X	.2.5	56	0	10 ng/ml<br>
.25X	5	62	0	100 ng/ml<br>
The production of the baculovirus construct and conditioned medium is described above.<br>
Example 14<br>
Antiviral Activity Can Not be Blocked bv Antibodies Against the Human Interferon Alpha Receptor Chain 2 Beta<br>
Additional antiviral assays were carried out using human cervical carcinoma cells (HeLa). On the first day, anti-hu-IFN-Receptor MAb (Research Diagnostics Inc) and Isotype-matched negative control MAb were diluted into a 96-wel] flat bottom microliter plate. Conditioned medium from Sf9 cells infected with baculovirus expressing zcyto20, zcyto21, or zcyto22 was added to obtain a final concentration of .0625X CM and plated with 50,000 HeLa cells per well. Following a 24-hour incubation at 37CC, the medium was removed and replaced with medium containing encephelomyocarditis virus at a multiplicity of infection of 0.1. The cells were again incubated for 24 hours at 37°C. Culture wells were then scored visually for the presence of cytopathic effect (CPE), as shown in Table 8. Purified human interferon-a-2a at a concentration of 0.01 ng/ml was included as a positive control.<br><br>
Table 12 shows that conditioned medium containing zcyto20, zcyto21, and 2cyto22 had antiviral activity in HeLa cells (as indicated by zero %CPE) in the presence or absence of neutralizing antibodies against the human interferon alpha receptor chain 2 beta. In contrast, the antiviral activity of interferon-a-2a was inhibited in a dose-dependent manner specifically in the presence of neutralizing antibodies against the human interferon alpha receptor chain 2 beta. These data indicate that zcyto20 zcyto21, and zcyto22 interact with either a receptor other than the human interferon alpha receptor or it interacts with the human interferon alpha receptor with a different mechanism than human interferon-a-2a.<br>
Table 12: Percentage Cytopathic Effect in HeLa Cells using zcvto20. zcyto21. or zcyto22 Conditioned Medium (CM) and Neutralizing Antibodies against<br>
the Human Interferon Alpha Receptor Chain 2 Beta	<br>
Anti-     zcyto20 zcyto20 zcyto21 zcyto21 zcyto22 zcyto22 hlFN-a- hlFN-a-<br>
hEFNR   CM    + CM    +CM    +CM    +CM    +CM    + 2a+       2a      +<br>
MAb     Anti-     Control Anti-     Control Anti-     Control Anti-     Control<br>
Concen hlFNR   MAb     hlFNR   MAb     hlFKR   MAb     hlFNR   MAb<br>
-tration  MAb	MAb	MAb	Mab<br>
Oug/ml 0	0	0	0	0	0	7.5	10<br>
0.001  0	0	0	0	0	0	7.5	10<br>
ug/ml							<br>
0.01   0	0	0	0	0	0	10	10<br>
ug/ml							<br>
0.1 . 0	0	0	0	0	0	27.5	10<br>
ug/ml							<br>
1 ug/ml 0	0	0	0	0	0	40	10<br>
10    0	0	0	0	0	0	87	10<br>
ug/ml							<br>
Example 15<br>
Antiproliferation Assay Using a BAF3- Cell Line<br>
BaF3 is used to determine if Zcyto20 has anti-proliferative properties. Baby hamster kidney (BHK) cells are stably transfected with an expression vector<br><br>
containing the CMV promoter plus intron A upstream of the Zcyto20 cDNA or an unrelated cDNA, called Za30, using BRL lipofectamine. Stably transfected cells are seeded in a cell factory in serum free media and allowed to grow for three days before conditioned media is harvested and concentrated in a 5K filter to lOx. Concentrated conditioned medium samples are stored at 4°C.<br>
The following assay is used to test for anti-proliferation of BaF3. In a 96 well plate, eight 1:2 serial dilutions are made of growth media alone (RPMI1640,10% fetal bovine serum, lmM sodium pyruvate, 2mM L-glutamine), or murine IL-3 (starting at 50 pg/ml in growth medium) with final volume of lOOul. Fifty microliters of the following are added to both growth media alone or mIL-3 diluted lanes: human interferon-a (100 ng/ml, 10 ng/ml, or 1 ng/ml diluted in growth medium), human interferon-P (100 ng/ml, 10 ng/ml, or lng/ml diluted in growth medium), murine interferon-a (100 ng/ml, 10 ng/mJ, or I ng/mJ diluted in growth medium), murine interferon-P (100 ng/ml, 10 ng/ml, or 1 ng/ml diluted in growth medium), Zcyto20 (at 2.5x, 0.5x, or O.lx), and murine Za30 (at 2.5x, 0.5x, or O.lx).<br>
The BaF3cell line is washed three times in growth medium, pellets are resuspended in growth medium, cells are counted and diluted in growth medium to 5,000 cells/50 ul Fifty microliters of diluted cells are then added to each dilution of samples. Assay plates are incubated in a 37°C incubator for three to four days. Twenty microliters of Alomar blue are then added to each well and the plate are incubated overnight at 37°C. The plates are read on the fluorescent plate reader at excitation wavelength of 544 and emission wavelength 590.<br>
Example 16<br>
Signaling Via Interferon-Response Pathway<br>
Interaction of type 1 interferons with their specific receptor leads to induction of a number of genes responsible for their antiviral/antipToliferative activity. These include 2'-5' oligoadenylate synthetase (2-5A synthetase), double-stranded RNA dependent Pkr kinase (Pkr), phospholipid scramblase, and intercellular adhesion molecule-1 (ICAM-1). Induction of genes with as yet unknown function, such as a 56kDa interferon stimulated gene product (ISG-56k), also occurs. To determine if some or all of these genes are induced upon treatment of cells with zcyto20, human Daudi B lymphoid cells were treated for 72 hours with conditioned medium from Sf9 cells infected with baculovirus expressing zcyto20. Conditioned medium from Sf9 cells infected with wild-type baculovirus was used as a negative control. Following treatment cells were collected and lysed for isolation of total RNA.   1 microgram of<br><br>
total RN'A was converted to cDNA using reverse transcriptase and used as a template for polymerase chain reaction using oligonucleotide primers specific for the human interferon-stimulated genes described above. Oligonucleotide primers for human gIycerol-3-phosphate dehydrogenase (G3PDH) were used as a non-interferon stimulated gene control. The results show clear induction of ISG-56k, Pkr, 2-5A synthetase and phospholipid scramblase following treatment of cells with zcyto20. No induction was seen for ICAM-1 or the non-interferon stimulated gene control, G3PDH.<br>
Example 17 Identification of zcytor!9 as a Receptor for zcvto21<br>
A: COS Cell Transfection and Secretion Trap<br>
Biotinylated zcyto21 was tested for binding to known or orphan cytokine receptors. The pZP7 expression vectors containing cDNAs of cytokine receptors (including human IFNaRl, IFNPR2, TFNaRl, IFNpR2, IL-10R, CRF2-4, ZcytoR7, DIRSl, ZcytorI9, and Tissue Factor) were transfected into COS cells, and the binding of biotinylated zcyto20 to transfected COS cells was carried out using the secretion trap assay described below. Positive binding in this assay showed receptor-Iigand pairs.<br>
COS Cell Transfections<br>
The COS cell transfections were performed as follows: COS cells were plated (1x10^ cells/well) on fibronectin coated, 12-well, tissue culture plates (Becton Dickinson, Bedford, MA) and incubated at 37°C overnight. Cytokine receptor DNA (0.75u.g) was mixed with 50ul serum free DMEM media (55mg sodium pyruvate, 145mg L-glutamine, 5mg transferrin, 2.5mg insulin, lu,g selenium and 5mg fetuin in 500ml DMEM), then mixed with 5ul Lipofectamine™ (Invitrogen, Carlsbad, CA) in 45uJ serum free DMEM media, and incubated at room temperature for 30 minutes. An additional 400ul serum free DMEM media was added. The cells were rinsed with serum free DMEM, and 500ul of the DNA mixture was added. The cells were incubated for 5 hours at 37°C, at which time an additional 500ul 20% FBS DMEM media (100 ml FBS, 55mg sodium pyruvate and 146mg L-glutamine in 500ml DMEM) was added and the cells were incubated overnight.<br>
Secretion Trap Assay<br>
The secretion trap was performed as follows: Media was aspirated and cells were rinsed twice with 1% BSA in PBS. Cells were blocked for 1 hour with TNB (0.1M Tris-HCL, 0.15M NaCI and 0.5% Blocking Reagent (NEN Renaissance TSA-<br><br>
Direct Kit, NEN Life Science Products, Boston, MA) in H2O. The cells were incubated<br>
for 1 hour with 3u,g/ml biotinylated zcyto21 protein (Example 27) in TNB. Cells were<br>
then washed 3 times with 1% BSA in PBS and were incubated for another hour with<br>
1:300 diluted Streptavidin-HRP (NEN kit) in TNB. Again cells were washed 3 times<br>
with 1% BSA in PBS, and then fixed for 15 minutes with 1.8% Formaldehyde in PBS.<br>
Cells were then washed 3 times with TNT (0.1M Tris-HCL, 0.15M NaCl, and 0.05% Tween-20 in H2O).<br>
Positive binding was detected with fluorescein tyramide reagent diluted 1:50 in dilution buffer (NEN kit), incubated for 4.5 minutes, and washed with TNT. Cells were preserved with Vectashield Mounting Media (Vector Labs Burlingame, CA) diluted 1:5 in TNT. Cells were visualized using a FTTC filter on fluorescent microscope.<br>
Positive binding was detected on cells transfected with human zcytor!9 cDNA and incubated with biotinylated zcyto21. None of the other transfected receptors bound zcyto21, and zcytorl9 did not bind a control biotinylated protein. These data indicate that zcytorl9 is a receptor for zcyto21.<br>
Further experiments have shown positive binding between both human and mouse Zcytorl9 with biotinylated zcyto21. Positive binding was also detected on cells transfected with human zcytorl9 cDNA and incubated with biotinylated zcyto20, and zcyto24.<br>
Example 18<br>
Signal Transduction Reporter Assay<br>
A signal transduction reporter assay can be used to determine the functional interaction of zcyto20, zcyto21, zcyto22, zcyto24, and zcyto25 with zcytorl9. Human embryonal kidney (HEK) cells are transfected with a reporter plasmid containing an interferon-stimulated response element (ISRE) driving transcription of a luciferase reporter gene in the presence or absence of pZP7 expression vectors containing cDNAs for class II cytokine receptors (including human DIRS1, IFNcdll, IFNcdl2 and Zcytorl9 (SEQ ID NOS:23 and 26)). Luciferase activity following stimulation of transfected cells with class II ligands (including zcyto20 (SEQ ID NO:l), zcyto21 (SEQ ID NO:4), zcyto22 (SEQ ID NO:6), zcytolO, huILlO and huIFNa-2a) reflects the interaction of the ligand with transfected and native cytokine receptors on the cell surface. The results and methods are described below.<br><br>
Cell Transfections<br>
293 HEK cells were transfected as follows: 700,000 293 cells/well (6 well plates) were plated approximately 18h prior to transfection in 2 milliliters DMEM + 10% fetal bovine serum. Per well, 1 microgram pISRE-Luciferase DNA (Stratagene), 1 microgram cytokine receptor DNA and 1 microgram pIRES2-EGFP DNA (Clontech,) were added to 9 microliters Fugene 6 reagent (Roche Biochemicals) in a total of 100 microliters DMEM. Two micrograms pIRES2-EGFP DNA was used when cytokine receptor DNA was not included. This transfection mix was added 30 minutes later to the pre-plated 293 cells. Twenty-four hours later the transfected cells were removed from the plate using trypsin-EDTA and replated at approximately 25,000 cells/well in 96 well microliter plates. Approximately 18 h prior to ligand stimulation, media was changed to DMEM + 0.5
Signal Transduction Reporter Assays<br>
The signal transduction reporter assays were done as follows: Following an 18h incubation at 37°C in DMEM + 0.5%FBS, transfected cells were stimulated with dilutions (in DMEM + 0.5%FBS) of the following class II ligands; zcyto20, zcyto21, zcyto22, zcytolO, hulLlO and huIFNa-2a. Following a 4-hour incubation at 37°C, the cells were lysed, and the relative light units (RLU) were measured on a Iuminometer after addition of a luciferase substrate. The results obtained are shown as the fold induction of the RLU of the experimental samples over the medium alone control (RLU of experimental samples/RLU of medium alone = fold induction). Table 14 shows that zcyto20, zcyto21 and zcyto22 induce ISRE signaling in 293 cells transfected with ISRE-Iuciferase giving a 15 to 17-fold induction in luciferase activity over medium alone. The addition of zcytorl9 DNA to the transfection mix results in a 6 to 8-fold further induction in ISRE signaling by zcyto20, zcyto21 and zcyto22 giving a 104 to 125-fold total induction. None of the other transfected class H cytokine receptor DNAs resulted in increased ISRE signaling. These results indicate that zcyto20, zcyto21 and zcyto22 functionally interact with the zcytorI9 cytokine receptor. Table 13 also shows that huTFNa-2a can induce ISRE signaling in ISRE-luciferase transfected 293 cells giving a 205-fold induction of luciferase activity compared to medium alone. However, the addition of zcytorl9 DNA to the transfection leads to an 11-fold reduction in ISRE-signaling (compared to ISRE-Iuciferase DNA alone), suggesting that zcytorl.9 over-expression negatively effects interferon signaling, in contrast to the positive effects of zcytorl9 over-expression on zcyto20, zcyto21 and zcyto22 signaling.<br><br>
Table 13: Interferon Stimulated Response Element (ISRE) Signaling of Transfected 293 Cells Following Class II Cytokine Stimulation (Fold Induction)<br>
Ligand	ISRE-Luc.	ISRE-Luc./Zcytorl9<br>
Zcyto20(125ng/ml)	15	125<br>
2cyto21 (125ng/ml)	17	108<br>
Zcyto22 (125ng/ml)	17	104<br>
HuIFNa-2a (lOOng/ml)	205	18<br>
ZcytolO (125ng/ml)	1.3	1<br>
HuILlO (lOOng/ml)	1	0.5<br>
Example 19: Identification of ILlORb (CRF2-4) as a Receptor subunit for zcytor!9<br>
A: ILlORb Neutralizing Antibody Inhibits ISRE Signaling: A signal transduction reporter assay was used to determine the functional interaction of zcyto20, zcyto21, and zcyto22 with zcytorl9 and ILlORb (CRF2-4). Human embryonal kidney (HEK) cells or human embryonal kidney (HEK) cells stably overexpressing human zcytoR19 were transfected with a reporter plasmid containing an interferon-stimulated response element (ISRE) driving transcription of a luciferase reporter. Luciferase activity following stimulation of transfected cells with class II ligands (including zcyto20, zcyto21, zcyto22 and huIFNa-2a) in the presence or absence of a neutralizing antibody to ILlORb (CRF2-4) reflects the interaction of the ligand. with cytokine receptors on the cell surface. The results and methods are described below.<br>
Cell Transfections:<br>
To produce 293 HEK cells stably overexpressing human zcytoR19, 293 cells were transfected as follows: 300,000 293 cells/well (6 well plates) were plated approximately 6h prior to transfection in 2 milliliters DMEM + 10% fetal bovine serum. Per well, 2 micrograms of a pZP7 expression vector containing the cDNA of human zcytoR19 (SEQ ID NO:23) was added to 6 microliters Fugene 6 reagent (Roche Biochemicals) in a total of 100 microliters DMEM. This transfection mix was added 30 minutes later to the pre-plated 293 cells. Forty-eight hours later the transfected cells were placed under 2 microgram/milliliter puromicin selection. Puromicin resistant cells were carried as a population of cells.<br><br>
The 293 HEK cells (wild type or overexpressing human zcytoR19) were transfected as follows: 700,000 293 cells/well (6 well plates) were plated approximately 18h prior to transfection in 2 milliliters DMEM + 10% fetal bovine serum. Per well, 1 microgram pISRE-Luciferase DNA (Stratagene) and 1 microgram pIRES2-EGFP DNA (Clontech) were added to 6 microliters Fugene 6 reagent (Roche Biochemicais) in a total of 100 microliters DMEM. This transfection mix was added 30 minutes later to the pre-plated 293 cells. Twenty-four hours later the transfected cells were removed from the plate using trypsin-EDTA and replated at approximately 25,000 cells/well in 96 well microtiter plates. Approximately 18 h prior to ligand stimulation, media was changed to DMEM + 0.5%FBS.<br>
Signal Transduction Reporter Assays:<br>
The signal transduction reporter assays were done as follows; Following an 18h incubation at 37 degrees in DMEM + 0.5%FBS, transfected cells were pretreated with a neutralizing polyclonal goat antibody to ILlORb (2.5 micrograms/ml for zcyto21; 8 micrograms/ml for zcyto20 and zcyto22, R&amp;D Systems) or PBS for 1 hour at 37 C. Human embryonal kidney (HEK) cells stably overexpressing human zcytoR19 were also pretreated with a non-neutralizing polyclonal goat antibody to IFNAR1 (8 micrograms/ml, R&amp;D Systems) as an antibody control for experiments involving zcyto20 and zcyto22. Pretreated cells were stimulated with dilutions (in DMEM + 0.5%FBS) of the following class II ligands; zcyto20, zcyto21, or zcyto22. As a control, huIFNa-2a was run in each experiment. Following a 4-hour incubation at 37 degrees, the cells were lysed, and the relative light units (RLU) were measured on a luminometer after addition of a luciferase substrate. The results obtained are shown as the fold- induction of the RLU of the experimental samples over the medium alone control (RLU of experimental samples/RLU of medium alone = fold induction).<br>
Tables 14 and 15 show that induction of 1SRE signaling by zcyto20 is inhibited by pretreatment of wild type 293 cells or 293 cells overexpressing human zcytoR19 with a neutralizing antibody to ILlORb. No or little inhibition is seen of huIFNa-2a induction of ISRE signaling. These results indicate that zcyto20 requires interaction with ILlORb (CRF2-4) for maximal induction of ISRE signaling and that the receptor for zcyto20 is the heterdimeric combination of zcytoR19 and ILlORb (CRF2-4).<br><br>
Table 14: ILlORb Inhibition of ISRE Signaling of Transfected wild-type 293 Cells Following Class II Cytokine Stimulation (Fold Induction)<br>
Cytokine<br>
Concentration (ng/ml)	Zcyto20	Zcyto20      + ILlORb<br>
neutralizing Antibody	HuIFNa-2a	HuIFNa-2a + ILlORb neutralizing Antibody<br>
100	8.4	0.8	152	102<br>
10	4	0.9	160	117<br>
1	1	0.9	90	69<br>
0.1	1	1	12	6    •<br>
0.01	I	0.8	1.2	1<br>
0	1	1	1	1<br>
Table 15: ILlORb Inhibition of ISRE Signaling of Transfected zcytoR19-overexpressing 293 Cells Following Class II Cytokine Stimulation (Fold Induction)<br>
Cytokine Concentration<br>
(ng/ml)	Zcyto20	Zcyto20       + ILlORb neutralizing Antibody	HuIFNa-2a	HuIFNa-2a + ILlORb neutralizing Antibody<br>
100	91	60	16	16<br>
10	97	23	14	15<br>
1	68	1.3	8	8.4<br>
0.1    '■	6	1.1	1.5	1.9<br>
0.01	1.1	1.2	1.2	1.3<br>
0	1	1	1	1<br>
Tables 16 and 17 show that ISRE signaling by zcyto21 is inhibited by pretreatment of wild type 293 cells or 293 cells overexpressing human zcytoR19 with a neutralizing antibody to ILlORb. No inhibition is seen of huIFNa-2a induction of ISRE signaling. These results indicate that zcyto21 requires interaction with ILlORb (CRF2-4) for maximal induction of ISRE signaling and that the receptor for zcyto21 is the heterdimeric combination of zcytoR19 and ILlORb (CRF2-4).<br><br>
Table 16: ILlORb Inhibition of ISRE Signaling of Transfected wild-type 293 Cells Following Class II Cytokine Stimulation (Fold Induction)<br><br>
Cytokine<br>
Concentration<br>
(ng/ml)	Zcyto21	Zcyto21       + ILlORb<br>
neutralizing Antibody	HuIFNa-2a	HuIFNa-2a + ILlORb neutralizing Antibody<br>
100	4.1	1.8	31	30<br>
10	3.2	1.4	32	31<br>
1	1.5	1.3	16.3	15<br>
0.1	1.1	1.3	1.4	2<br>
0.01	1.2	1.3	1.1	1.2<br>
0.O01	1.2	1.3	0.9	2.1<br>
0	1	1	1	1<br>
Table 17 ILlORb Inhibition of ISR) Signaling of Transfected zcytoR19-overexpressing 293 Cells Following Class II Cytokine Stimulation (Fold Induction)<br>
Cytokine Concentration<br>
(ng/ml)	Zcyto21	Zcyto21      + ILlORb neutralizing Antibody	HuIFNa-2a	HuIFNa-2a + ILlORb neutralizing Antibody<br>
100	45	31	9	7.7<br>
10	48	28	9	8.5<br>
1	35	5.8	4.3	4.3<br>
0.1	3.5	1	1.4	1.3<br>
0.01	1.5	1.1	0.9	1.2<br>
0.001	1.1	1	1.2	1<br>
0	1	1 -	1	1<br>
Tables 18 and 19 show that induction of ISRE signaling by zcyto22 is inhibited by pretreatment of wild type 293 cells or 293 cells overexpressing human zcytoR19 with a neutralizing antibody to ILlORb. No or little inhibition is seen of huIFNa-2a induction of ISRE signaling. These results indicate that zcyto22 requires interaction with ILlORb (CRF2-4) for maximal induction of ISRE signaling and that<br><br>
the receptor for zcyto22 is the heterdimeric combination of zcytoR19 and ILlORb (CRF2-4).<br>
Table 18: ILlORb Inhibition of ISRE Signaling of Transfected wild-type 293 Cells Following Class H Cytokine Stimulation (Fold Induction)<br>
Cytokine<br>
Concentration (ng/ml)	Zcyto22	Zcyto22      + ILlORb neutralizing Antibody	HuIFNa-2a	HuIFNa-2a + ILlORb neutralizing Antibody<br>
100	11	1.2	152	102<br>
10	8	1	160	117<br>
1	1.8	0.8	90	69<br>
0.1	1.2	0.8	12	6<br>
0.01	0.9	0.9	1.2	1<br>
0	1	1	1	1<br>
Table 19: ILlORb Inhibition of ISRE Signaling of Transfected zcytoR19-overexpressing 293 Cells Following Class II Cytokine Stimulation (Fold Induction)<br>
Cytokine<br>
Concentration<br>
(ng/ml)	Zcyto22	Zcyto22       + ILlORb neutralizing Antibody	HuIFNa-2a	HuIFNa-2a + ILlORb neutralizing Antibody<br>
100 '.	82	76	16	16<br>
10	97	39	14	15<br>
1	69	2.3	8	8.4<br>
0.1	8.4	I.I	1.5	1.9<br>
0.01   .	1	1.3 '	1.2	1.3<br>
0	1	1	1	1<br>
B: Ar And- ILlORb Antibody Blocks Antiviral Activity An antiviral assay was performed to determine the ability of anti-ELlORb antibody to block the antiviral activity of zcyto20. The assay was carried out using 293 HEK cells (wild type of overexprcssing human zcytoR19). On the first day, antibodies (anti-human IL10R beta, anti-human Leptin receptor, R&amp;D Systems) were diluted into cell media at 5 micrograms/ml and then plated with 50,000 cells per well into a 96-well<br><br>
plate. Following a one-hour incubation at 37°C, zcyto20-CEE (from example 3) (200 ng/m! for wild-type 293 cells, 0.5 ng/ml for 293 cells overexpressing human zcytoR19) or human interferon-a-2a (1 ng/ml for wild-type 293 cells, 100 ng/ml for 293 cells overexpressing human zcytoR19) were added to the wells and incubated overnight at 37°C. The next day, the medium was removed and replaced with medium containing encephalomyocarditis virus (EMCV) at a multiplicity of infection of 0.1. The cells were then incubated at 37°C overnight. Subsequently, 25 uL of 5 mg/ml Methylthiazoletetrazolium (MTT)(Sigma) were added to each well, incubated 2 hours at 37 degrees, and wells were then extracted with 100 uL extraction buffer (12.5% SDS, 45% DMF). Following overnight incubation at 37°C, the optical density at 570 nM was measured on a Spectromax plate reader (Molecular Devices, CA). Decreased optical density (570nm) indicates decreased cell survival (loss of antiviral activity). The optical densities (570nm) for the different experimental conditions are shown in Table 20 below. The results indicate that blocking human IL10 receptor beta specifically neutralizes the antiviral activity of zcyto20 without effecting interferon-a-2a activity. This indicates that human IL10 receptor beta is part of the receptor complex, (including human zcytoR19) involved in zcyto20 antiviral activity.<br>
Table 20: Optical Density (570nm) of ECMV-Infected Cytokine-Treated Cells<br>
Cytokine	Wild-type 293<br>
CelIs:Anti-ILlORb	Wild-type 293       Cells: Anti-LeptinR	HuzcytoR19-<br>
over-<br>
expressing<br>
293<br>
Cells: Anti-<br>
ILlORb	HuzcytoR19-over-<br>
expressing 293       Cells: Anti-LeptinR<br>
Zcyto20-CEE	.94	.88	.95	.24<br>
HuIFNa-2a	.58	.4	.18	.05<br>
C: zcvto20. zcvto21, and zcyto22 Signaling is Enhanced by Coexpression of zcvtoR19 andlLlORb:<br>
A signal transduction reporter assay was used to determine the functional interaction of zcyto20, zcyto21 and zcyto22 with zcytorl9 and DLlORb (CRF2-4). Hamster kidney (BHK) cells were transfected with a reporter plasmid containing an interferon-stimulated response element (ISRE) driving transcription of a<br><br>
luciferase reporter gene in the presence or absence of pZP7 expression vectors containing cDNAs for class II cytokine receptors Zcytorl9 and lUORb (CRF2-4). Luciferase activity following stimulation of transfected cells with class II ligands (including zcyto20, zcyto21 and zcyto22) reflects the interaction of the ligand with transfected and native cytokine receptors on the cell surface. The results and methods are described below.<br>
Cell Transfections<br>
BHK-570 cells were transfected as follows: 200,000 BHK cells/well (6 well plates) were plated approximately 5h prior to txansfection in 2 milliliters DMEM + 5% fetal bovine serum. Per well, 1 microgram pISRE-Luciferase DNA (Stratagene), 1 microgram cytokine receptor DNA and. 1 microgram pIRES2-EGFP DNA (Clontech) were added to 9 microliters Fugene 6 reagent (Roche Biochernicals) in a total of 100 microliters DMEM. Two micrograms pIRES2-EGFP DNA was used when cytokine receptor DNA was not included. This transfection mix was added 30 minutes later to the pre-plated BHK cells. Twenty-four hours later the transfected cells were removed from the plate using trypsin-EDTA and replated at approximately 25,000 cells/well in 96 well microliter plates. Approximately 18 h prior to ligand stimulation, media was changed to DMEM + 0.5%FBS.<br>
Signal Transduction Reporter Assays<br>
The signal transduction reporter assays were done as follows: Following an 18h incubation at 37°C in DMEM + 0.5%FBS, transfected cells were stimulated with dilutions (in DMEM + 0.5%FBS) of zcyto20, zcyto21, zcyto22, zcyto24, and zcyto25 ligands. Following a 4-hour incubation at 37 degrees, the cells were lysed, and the relative light units (RLU) were measured on a luminometer after addition of a luciferase substrate. The results obtained are shown as the fold induction of the RLU of the experimental samples over the medium alone control (RLU of experimental samples/RLU of medium alone = fold induction). Table 21 shows that zcyto20, zcyto21 and zcyto22 induce ISRE signaling in BHK cells transfected with ISRE-luciferase and zcytoR19 in a dose-dependent manner. The addition of ELlORb (CRF2-4) DNA to the transfection mix results in a half-maximal induction of signaling at a 10-100 fold lower cytokine dose. No response was seen with ISRE transfection alone. These results show that the ability of zcyto20, zcyto21 and zcyto22 to signal through the interferon stimulated response element is enhanced by coexpression of zcytoR19 and ILlORb (CRF2-4) indicating that the receptor for zcyto20, zcyto21 and zcyto22 is the heterdimeric combination of zcytoR19 and ILlORb (CRF2-4).<br><br>
	Table 21	Interferon Stimulated Response Element (ISRE) Signaling of<br>
Transfectet	BHK Cells	following C	ass II Cytokine Stimulation (Fold Induction)<br>
Class       II						<br>
ligand	zcyto20/	zcyto20/	zcyto21/	zcyto21/	zcyto22/	zcyto22/<br>
Concentra-	cells	cells	cells	cells	cells	cells<br>
tion	transfected	transfected	transfected	transfected	transfected	transfected<br>
(ng/ml)	with	with	with	with	with	with<br>
	zcytoR19	zcytoR19	zcytoR19	zcytoR19	zcytoR19	zcytoR19<br>
	alone	and ILlORb<br>
(CRF2^t)	alone	and<br>
ILlORb<br>
(CRF2^t)	alone	and<br>
ILlORb<br>
(CRF2-4)<br>
1000	2.25	2.1	3.3	2.2	1.8	2.2<br>
100	2.2	2.6	2.6	2.5	2	2.2<br>
10	2.1	2.4	2.4	2.6	1.9	2.7<br>
1	1.3	2.5	2	2.5	1.5	2.7<br>
0.1	1.25	2.1	1.4	2.2	1.1	24<br>
0.01	1.2	1.6	1.4	1.6	1.2	1.7<br>
0.001	1.4	1.5	1.3	1.3	1.2	1.3<br>
o .	1	1	1	1	I	I<br>
Example 20<br>
Construction of Mammalian Expression Vectors That Express zcytor!9 Soluble Receptors: zcvtor!9CEE, zcvtor!9CFLG, zcvtor!9CHIS and zcvtor!9-Fc4<br>
An expression vector is prepared for the expression of the soluble, extracellular domain of the zcytorl9 polypeptide, pC4zcytorl9CEE, wherein the construct is designed to express a zcytorl9 polypeptide comprised of the predicted initiating methionine and truncated adjacent to the predicted transmembrane domain, and with a C-terminal Glu-Glu tag (SEQ ID NO: 16).<br>
A zcytorl9 DNA fragment comprising the zcytorl9 extracellular or cytokine binding domain of zcytorl9 described herein, is created using PCR, and<br><br>
purified using standard methods. The excised DNA is subcloned into a plasmid expression vector that has a signal peptide, e.g., the native zcytorl9 signal peptide, and attaches a Glu-Glu tag (SEQ ID NO: 16) to the C-terminus of the zcytorl9 polypeptide-encoding polynucleotide sequence. Such a mammalian expression vector contains an expression cassette having a mammalian promoter, multiple restriction sites for insertion of coding sequences, a stop codon and a mammalian terminator. The plasmid can also have an E. coli origin of replication, a mammalian selectable marker expression unit having an SV40 promoter, enhancer and origin of replication, a DHFR gene and the SV40 terminator.<br>
Restriction digested zcytorl9 insert and previously digested vector are ligated using standard molecular biological techniques, and electroporated into competent cells such as DH10B competent cells (GIBCO BRL, Gaithersburg, MD) according to manufacturer's direction and plated onto LB plates containing 50 mg/ml ampicillin, and incubated overnight. Colonies are screened by restriction analysis of DNA prepared from individual colonies. The insert sequence of positive clones is verified by sequence analysis. A large scale plasmid preparation is done using a QIAGEN© Maxi prep ltit (Qiagen) according to manufacturer's instructions.<br>
The same process is used to prepare the zcytorl9 soluble receptors with a C-terminal his tag, composed of 6 His residues in a row; and a C-terminal FLAG® tag (SEQ ID NO:42), zcytor!9CFLAG. To construct these constructs, the aforementioned vector has either the C-HIS or the FLAG® tag in place of the glu-glu tag(SEQH)NO:16).<br>
An expression vector, zcytorl9/Fc4/pzmp20, was prepared to express a C-terminally Fc4 tagged soluble version of zcytorl9 (human zcytorI9-Fc4) in BHK cells. A fragment of zcytorl9 cDNA that includes the polynucleotide sequence from extracellular domain of the zcytor!9 receptor was fused in frame to the Fc4 polynucleotide sequence (SEQ ID NO:43) to generate a zcytorl9-Fc4 fusion. The pzmp20 vector is a mammalian expression vector that contains the Fc4 polynucleotide sequence and a cloning site that allows rapid construction of C-terminal Fc4 fusions using standard molecular biology techniques.<br>
A 630 base pair fragment was generated by PCR, containing the extracellular domain of human zcytorl9 with BamHI and Bgl2 sites coded on the 5' and 3' ends, respectively. This PCR fragment was generated using primers ZC37967 (SEQ ID NO:44) and ZC3.7972 (SEQ ID NO:45) by amplification from human brain cDNA library. The PCR reaction conditions were as follows: 30 cycles of 94°C for 20 seconds, and 68°C for 2 minutes; 1 cycle at 68°C for 4 minutes; followed by a 10°C soak.  The fragment was digested with BamHI and Bgl2 restriction endonucleases and<br><br>
subsequently purified by 1% gel electrophoresis and band purification using QiaQuick gel extraction kit (Qiagen). The resulting purified DNA was ligated for 5 hours at room temperature into a pzmp20 vector previously digested with Bgl2 containing Fc4 3' of the Bgl2 sites.<br>
One ul of the ligation mix was electroporated in 37 Hi DH1QB electrocompetent E. coli (Gibco) according to the manufacturer's directions. The transformed cells were diluted in 400 JJ.1 of LB media and plated onto LB plates containing 100 jig/ml ampicillin. Clones were analyzed by restriction digests and positive clones were sent for DNA sequencing to confirm the sequence of the fusion construct.<br>
Example 21<br>
Mammalian expression human zcvtor!9 soluble receptor: zcytor!9/Fc4 BHK 570 cells (ATCC NO: CRL-10314) were plated in T-75 tissue culture flasks and allowed to grow to approximately 50 to 70% confluence at 37°C, 5% C02, in DMEM/FBS media (DMEM, Gibco/BRL High Glucose, (Gibco BRL, Gaithersburg, MD), 5% fetal bovine serum, 1 mM L-glutamine (JRH Biosciences, Lenea, KS), 1 mM sodium pyruvate (Gibco BRL)). The cells were then transfected with the plasmid zcytorl9/Fc4/pzmp20 (Example 4B) using Lipofectamine™ (Gibco BRL), in serum free (SF) media formulation (DMEM, 10 mg/ml transferrin, 5 mg/ml insulin, 2 mg/ml fetuin, 1% L-glutamine and 1% sodium pyruvate). Ten jig of the plasmid DNA zcytorl9/Fc4/pzmp20 (Example 4B) was diluted into a 15ml tube to a total final volume of 500 &amp;il with SF media. 50 |il of Lipofectamine was mixed with 450 u.1 of SF medium. The Lipofectamine mix was added to the DNA mix and allowed to incubate approximately 30 minutes at room temperature. Four ml of SF media was added to the DNA Lipofectamine mixture. The cells were rinsed once with 5 ml of SF media, aspirated, and the DNA:Lipofectamine mixture was added. The cells were incubated at 37°C for five hours, and then 5 ml of DMEM/10%FBS media was added. The flask was incubated at 37°C overnight after which time the cells were split into the selection media (DMEM/FBS media from above with the addition of 1 pJvI methotrexate (Sigma Chemical Co., St. Louis, Mo.) in 150 mm plates at 1:2, 1:10, and 1:50. Approximately 10 days post-transfection, one 150 mm plate of 1 |iM methotrexate resistant colonies was trypsinized, the cells were pooled, and one-half of the cells were replajed.in 10 uM methotrexate; to further amplify expression of the zcytorl9/Fc4 protein. A conditioned-media sample from this pool of amplified cells was tested for expression levels using SDS-PAGE and Western analysis.<br><br>
Single clones expressing the soluble receptors can also isolated, screened and grown up in cell culture media, and purified using standard techniques. Moreover, CHO cells are also suitable cells for such purposes.<br>
Example 22<br>
Assessing Zcytorl9 ReceptorHeterodimerization Using ORIGEN Assay<br>
Soluble zcytorl9 receptor is biotinylated by reaction with a five-fold molar excess of sulfo-NHS-LC-Biotin (Pierce, Inc., Rockford, IL) according to the manufacturer's protocol. Soluble zcytorl9 receptor and another soluble receptor subunit, for example, soluble class H cytokine receptors, for example, CRF2-4 (SEQ ID NO:40)i are labeled with a five fold molar excess of Ru-BPY-NHS (Igen, Inc., Gaithersburg, MD) according to manufacturer's protocol. The biotinylated and Ru-BPY-NHS-labeled forms of the soluble zcytorl9 receptor can be respectively designated Bio-zcytorl9 receptor and Ru-zcytorl9; the biotinylated and Ru-BPY-NHS-labeled forms of the other soluble receptor subunit can be similarly designated. Assays are carried out using conditioned media or using purified ligands.<br>
For initial soluble receptor binding characterization, the cytokines mentioned above, or conditioned medium, are tested to determine whether they can mediate homodimerization of zcytorl9 receptor and if they can mediate the heterodimerization of zcytorl9 receptor with the soluble receptor subunits described above. To do this, 50 p.1 of conditioned media or TBS-B containing purified cytokine, is combined with 50 pi of TBS-B (20 mM Tris, 150 mM NaCl, 1 mg/ml BS A, pH 7.2) containing e.g., 400 ng/ml of Ru-zcytorl9 receptor and Bio-zcytorl9, or 400 ng/ml of Ru-zcytorl9 receptor and e.g., Bio-CRF2-4, or 400 ng/ml of e.g., Ru-CRF2-4 and Bio-zcytorl9. Following incubation for one hour at room temperature, 30 p.g of streptavidin coated, 2.8 mm magnetic beads (Dynal, Inc., Oslo, Norway) are added and the reaction incubated an additional hour at room temperature. 200 \xl ORIGEN assay buffer (Igen, Inc., Gaithersburg, MD) is then added and the extent of receptor association measured using an M8 ORIGEN analyzer (Igen, Inc.).<br>
Example 23<br>
Construct for Generating a zcytorl9 Receptor Heterodimer<br>
A vector expressing a secreted human zcytorl9 heterodimer is constructed by fusing ,the extracellular cytokine-binding domain of zcytorl9 to the heavy chain of IgG gamma 1 (IgGyl), and the extracellular portion of the heteromeric cytokine receptor subunit (E.g., class II cytokine receptors, for example, CRF2-4) is fused to a human kappa light chain (human K light chain).<br><br>
a.	Construction of IgG gamma 1 and human K light chain fusion vectors<br>
The heavy chain of IgGyl is cloned into the Zem229R mammalian<br>
expression vector (ATCC deposit No. 69447) such that any desired cytokine receptor extracellular domain having a 5' EcoRI and 3' Nhel site can be cloned in resulting in an N-terminal extracellular domain-C-terminal IgGyl fusion. The IgGyl fragment used in this construct is made by using PCR to isolate the IgGyl sequence from a Ciontech human Fetal liver cDNA library as a template. PCR products are purified using methods described herein and digested with Mlul and EcoRI (Boerhinger-Mannheim), ethanol precipitated and ligated with oligos that comprise an MluI/EcoRI linker, into Zem229R previously digested with and EcoRI using standard molecular biology techniques disclosed herein.<br>
The human K light chain is cloned in the Zem228R mammalian expression vector (ATCC deposit No. 69446) such that any desired cytokine receptor extracellular domain having a 5' EcoRI site and a 3' Kpnl site can be cloned in resulting in a N-terminal cytokine extracellular domain-C-terminal human K light chain fusion. As a Kpnl site is located within the human K light chain sequence, a special primer is designed to clone the 3' end of the desired extracellular domain of a cytokine receptor into this Kpnl site: The primer is designed so that the resulting PCR product contains the desired cytokine receptor extracellular domain with a segment of the human K light chain up to the Kpnl site. This primer preferably comprises a portion of at least 10 nucleotides of the 3' end of the desired cytokine receptor extracellular domain fused in frame 5' to the human kappa light chain. The human K light chain fragment used in this construct is made by using PCR to isolate the human K light chain sequence from the same Ciontech human Fetal Liver cDNA library used above. PCR products- are purified using methods described herein and digested with Mlul and EcoRI (Boerhinger-Mannheim), ethanol precipitated and ligated with the MluI/EcoRI linker described above, into Zem228R previously digested with and EcoRI using standard molecular biology techniques disclosed herein.<br>
b.	Insertion of zcvtor!9 receptor or heterodimeric subunit extracellular<br>
domains into fusion vector constructs<br>
Using the construction vectors above, a construct having zcytorl9 fused to IgGyl is made. This construction is done by PCRing the extracellular domain or cytokine-binding domain of zcytorl9 receptor described herein from a prostate cDNA library (Ciontech) or activated lymphocyte cDNA library using standard methods, and oligos that provide EcoRI and Nhel restriction sites. The resulting PCR product is digested with EcoRI and Nhel, gel purified, as described herein, and ligated into a<br><br>
previously EcoRI and Nhel digested and band-purified Zem229R/IgGYl described above. The resulting vector is sequenced to confirm that the zcytorl9/IgG gamma 1 fusion (zcytorl9/Chl IgG) is correct.<br>
A separate construct having a heterodimeric cytokine receptor subunit, i.e., CRF2-4, extracellular domain fused to K light is also constructed as above. The cytokine receptor/human K light chain construction is performed as above by PCRing from, e.g., a lymphocyte cDNA library (Clontech) using standard methods, and oligos that provide EcoRI and Kpnl restriction sites. The resulting PCR product is digested with EcoRI and Kpnl and then this product is ligated into a previously EcoRI and Kpnl digested and band-purified Zem228R/human K light chain vector described above. The resulting vector is sequenced to confirm that the cytokine receptor subunit/human K light chain fusion is correct.<br>
c.	Co-expression of the zcvtor!9 and heterodimeric cytokine receptor<br>
subunit extracellular domain<br>
Approximately 15 fig of each of vectors above, are co-transfected into mammalian cells, e.g., BHK-570 cells (ATCC No. CRL-10314) using LipofectaminePlus™ reagent (Gibco/BRL), as per manufacturer's instructions. The transfected cells are selected for 10 days in DMEM + 5%FBS (Gibco/BRL) containing 1 (JM of methotrexate (MTX) (Sigma, St. Louis, MO) and 0.5 mg/ml G418 (Gibco/BRL) for 10 days. The resulting pool of transfectants is selected again in 10 \xm of MTX and 0.5 mg/ml G418 for 10 days.<br>
The resulting pool of doubly selected cells is used to generate protein. Three Factories (Nunc, Denmark) of this pool are used to generate 10 L of serum free conditioned medium. This conditioned media is passed over a 1 ml protein-A column and eluted in about 10, 750 microliter fractions. The fractions having the highest protein concentration are pooled and dialyzed (10 kD MW cutoff) against PBS. Finally the dialyzed material is submitted for amino acid analysis (AAA) using routine methods.<br>
d.	Reconstitution of zcytorl9 Receptor in vitro<br>
To identify components involved in the zcytorl9-signaling complex, receptor reconstitution studies are performed as follows. For example, BHK 570 cells (ATCC No. CRL-10314) transfected, using standard methods described herein, with a luciferase reporter mammalian expression vector plasmid serve as a bioassay cell line to measure signal transduction response from a transfected zcytorl9 receptor complex to the luciferase reporter in the presence of zcytor!9 Ligand. BHK cells would be used in<br><br>
the event that BHK cells do not endogenously express the zcytorl9 receptor. Other cell lines can be used. An exemplary luciferase reporter mammalian expression vector is the KZ134 plasmid which is constructed with complementary oligonucleotides that contain STAT transcription factor binding elements from 4 genes. A modified c-fos Sis inducible element (m67SIE, or hSIE) (Sadowski, H. et al., Science 261:1739-1744, 1993), the p21 SIE1 from the p21 WAF1 gene (Chin, Y. et al., Science 272:719-722, 1996), the mammary gland response element of the p-casein gene (Schmitt-Ney, M. et al., Mol. Cell. Biol. 11:3745-3755, 1991), and a STAT inducible element of the Fcg RI gene, (Seidel, H. et al., Proc. Natl. Acad. Sci. 92:3041-3045, 1995). These oligonucleotides contain Asp718-XhoI compatible ends and are ligated, using standard methods, into a recipient firefly luciferase reporter vector with a c-Fos promoter (Poulsen, L.K. et al., J. Biol. Chem. 273:6229-6232, 1998) digested with the same enzymes and containing a neomycin selectable marker. The KZ134 plasmid is used to stably transfect BHK, or BaF3 cells, using standard transfection and selection methods, to make a BHK/KZ134 or BaF3/KZ134 cell line respectively.<br>
The bioassay cell line is transfected with zcytorl9 receptor alone, or co-transfected with zcytorl9 receptor along with one of a variety of other known receptor subunits. Receptor complexes include but are not limited to zcytorl9 receptor only, various combinations of zcytorl9 receptor with class II cytokine receptors, for example, interferon-gamma, alpha and beta chains and the interferon-alpha/beta receptor alpha and beta chains, zcytorll (commonly owned US Patent No. 5,965,704), CRF2-4, DIRS1, zcytor7 (commonly owned US Patent No. 5,945,511) receptors. Each independent receptor complex cell line is then assayed in the presence of cytokine-conditioned media or purified cytokines and luciferase activity measured using routine methods. The untransfected bioassay cell line serves as a control for the background luciferase activity, and is thus used as a baseline to compare signaling by the various receptor complex combinations. The conditioned medium or cytokine that binds the zcytorl9 receptor in the presence of the correct receptor complex, is expected to give a luciferase readout of approximately 5 fold over background or greater.<br>
As an alternative, a similar assay can be performed wherein the a Baf3/zcytorl9 cell line isco-transfected as described herein and proliferation is measured, using a known assay such as a standard Alamar Blue proliferation assay.<br><br>
Example 24<br>
Binding of Lisands to Soluble Receptors<br>
The binding of the ligands (zcyto20, zcyto21, zcyto22, zcyto24, and zcyto25) to soluble receptors can be assayed using an iodo-bead labeling method. For example, 125I labeled zcyto21-CEE is labeled (1.2 x 107 CPM/ml; 1.5 ng/ul; and 8.6xl06CPM/ug).<br>
Fifty nanograms of the 125I labeled zcyto21-cEE (See Example 3) (399,600 CPM) is combined with 1000 ng of cold zcytorl9/Fc4 homodimer receptor, 1000 ng cold zcytorl9/CRF2-4 heterodimer receptor, or 1000 ng of a control Class II cytokine receptor/Fc4 receptor as a control with about 10,000 ng of cold zcyto21 as a competitor. Samples are incubated for 2 hours at 4°C, after which 30 ul protein-G (Zymed San Francisco, CA) is added to each sample. Samples are incubated for 1 hour at 4°C, and washed 3 times with PBS. Radioactivity of the washed protein-G is measured in a gamma counter (Packard Instruments, Downers Grove, IL).<br>
Example 25<br>
Flow Cytometry staining of human monocytes with zcyto20 and zcyto21-biotin<br>
Peripheral blood leukocytes (PBLs) were isolated by Ficoll Hypaque (Amersham, Sweden) separation from heparinized human blood. The PBLs were culturedat 37°C in standard media at a density of lxlO'e cells per milliliter in 6-well tissue culture plates. Following overnight incubation, the PBLs were harvested and stained with biotinylated zcyto20-cee and zcyto21-cee (See Example 18) at a concentration of 10 ug/ml. Staining was detected with Phycoerythrin-labeled streptavidin (Pharmingen, CA, USA) that was prepared at a dilution of 1:1000. Following staining the PBLs were fixed in 2% Paraformaldehyde, and read on a Facscaliber (Becton Dickinson, San Diego, CA). The data was analyzed using Cellquest software (Becton Dickinson). Results indicate that both biotinylated zcyto20-cee and zcyto21-cee stain cells in the myeloid gate of peripheral blood leukocytes. Cells in the lymphoid gate do not bind zcyto20-cee and zcyto21-cee.<br><br>
Example 26<br>
zcyto21-CEE Effects on Expression of Activation Markers on PBLs.<br>
Peripheral blood leukocytes (PBLs) were isolated by Ficoll Hypaque separation from heparinized human blood. The PBLs were then stimulated with purified protein or media controls from the following: 1) zcyto21-CEE (2 ug/ml); 2) zcyto21-cee (1 ug/ml); 3) Media alone; 4) A141F negative control protein (2 ug/ml); or 5) IFN-alpha-A (Ing/ml) (PBL Biomedical NJ, USA.). The stimulated PBLs were incubated at a cell density of 1X10*6 cells per milliliter in at 37°C with 5% C02. Cultures were harvested at 24 and 48 hours and stained for activation markers.<br>
The PBLs were washed with PBS and then blocked with normal mouse IgG in Facs buffer (HBSS + 2% normal goat serum, 2% BSA, .2% NaN3), followed by staining with antibodies for the following markers; CD19, CD14, CD3, HLA-DR, CD54, HLA-ABC (Pharmingen, CA, USA and Immunotech, France). Cells were washed and then fixed in 2% Paraformaldehyde before being analyzed on a Facscaliber {Becton Dickinson, CA, USA). The resulting data were analyzed using Cellquest software (Becton Dickinson, CA, USA).<br>
Results indicated an increase in surface CD54 (ICAM) expression on monocytes at 24 and 48 hours with zcyto21-cee stimulation, compared to the media alone control. Stimulation with zcyto21-cee also resulted in an increase in Major histocompatibility complex I expression on B-cells at 24 hours and an increase in MHCI on both B-cells and Monocytes at 48 hours.<br>
Example 27 Biotinylation of Ligands<br>
Zcyto21CEE were biotinylated by following a modified protocol by Pierce Chemical Company for sulfo-NHS-LC-Biotin. Two hundred and fifty micrograms of zcyto21CEE (in 0.5ml PBS) was added to 8.4 ul of S-NHS-biotin stock<br>
(84 ug), and incubated at room temperature for 2 hours with rocking. Following the incubation step, 20 ul 2M TrisHCl (pH8) was added, and the mixture was incubated for 20 minutes at room .ternperature with rocking. The biotinylated ligand mixtures were then stored at 4°C. 2cyto20CEE, zcyto22CEE and zcyto24CEE were prepared in essentially the same manner.<br><br>
Example 28.<br>
Expression of zcytor!9 bv Northern Analysis<br>
Northern blots were probed to determine the tissue distribution of zcytorl9. A human zcytorl9 cDNA fragment was obtained using PCR with gene specific primers, 5' ZC40285 as shown in SEQ ID NO: 21; and 3' ZC 40286, as shown<br>
in SEQ ID NO: 22. The template was cloned human zcytor!9 cDNA. (SEQ ID NO: 23)<br>
32 The PCR fragment was gel purified, and -25 ng was labeled with P     a-dCTP using<br>
the Prime-It® RmT random prime labeling kit (Stratagene, LaJolla, CA).<br>
The following Northern blots (Clontech, Palo Alto, CA) were probed for mRNA expression of zcytorl9: (1) a human cancer cell line blot C, which contains RNA samples from each of the following cancer cell lines: promyelocyte leukemia HL-60, HELA S3, chronic myelogenous leukemia k-562, lymphoblastic leukemia MOLT-4, Burkitt's lymphoma RAJI, colorectal adenocarcinoma SW480, lung carcinoma A549, and melanoma G-361; (2) a human MTN H blot, which contains mRNA from the following tissues: heart, whole brain, placenta, lung, liver, skeletal muscle, kidney, and pancreas; (3) a human MTN H3 which contains mRNA from the following tissues: stomach, thyroid, spinal cord, lymph node, trachea, adrenal gland, and bone marrow; and (4) a human MTN H4, which contains mRNA from the following tissues: spleen, thymus, prostate testis, uterus, small intestine, colon, and peripheral blood leukocytes. Hybridizations were all performed in ULTRAhyb™ Ultrasensitive Hybridization Buffer (Ambion, Austin, TX) according the manufacturer's recommendations, which the exception that an additional 0.2 mg/ml salmon sperm DNA was added to the hybridization and prehybridization buffers to lower non-specific hybridization. Following hybridization, non-specific radioactive signal was removed by treating the blots with 0.1X SSC/0.5% SDS at 50°C. The blots were exposed using BioMax MR Film and intensifying screens (Eastman Kodak, Rochester, NY), per the manufacturer's recommendations for 3 days.<br>
Expression of a -4.5 kb transcript was in greatest in heart, skeletal muscle, pancreas and prostate tissue, in addition to in the Burkitt's lymphoma (RAJI) cell line. Lower levels were seen in multiple other tissues. In addition, there was an ~2kb transcript which was generally less abundant than the larger transcript, but also present in many of the-tissues and cell lines. Testis tissue, in addition to having the 2 and 4.5 kb transcripts, may also have ~4kb and 1.4 kb transcripts. Adrenal gland demonstrated equal levels of expression of the 4.5kb and 2 kb transcripts.<br><br>
Example 29<br>
Expression of zcytorl9 by In situ Analysis<br>
Specific human tissues were isolated and screened for zcytorl9 expression by in situ hybridization. Various human tissues prepared, sectioned and subjected to in situ hybridization included normal and carcinoma colon, cervical carcinoma, endometrial carcinoma, normal and carcinoma ovary, normal and neoplasmic skin, fetal liver, lung, heart and MFH (muscle sarcoma). The tissues were fixed in 10% buffered formalin and blocked in paraffin using standard techniques. Tissues were sectioned at 4 to 8 microns. Tissues were prepared using a standard protocol. Briefly, tissue sections were deparaffinized with Histo-Clear® (National Diagnostics, Atlanta, GA) and then dehydrated with ethanol. Next they were digested with Proteinase K (50 u.g/ml) (Boehringer Diagnostics, Indianapolis, IN) at 37°C for 2 to 7 minutes. This step was followed by acetylation and re-hydration of the tissues.<br>
One in situ probe was designed against the human zcytorl9 (variant xl) sequence (INC7128744, as shown in SEQ ID NO: 25), containing the 3'UTR of zcytorl9 using standard methods. T7 RNA polymerase was used to generate an anu'sense probe. The probe was labeled using an In Vitro transcription System (Riboprobe ® in vitro Transcription System, Promega, Madison, WI) as per manufacturer's instruction, except that the probes digoxigenin was used instead of radiolabeled rCTP and that the water was adjusted to accomodate the reduced volume of the rNTP's.. In situ hybridization was performed with a digoxigenin-labeled zcytorl9 probe (above). The probe was added to the slides at a concentration of 1 to 5 pmol/ml for 12 to 16 hours at 60°C. Slides were subsequently washed in 2XSSC and 0.1XSSC at 55°C. The signals were amplified using TSA™ (Tyramide Signal Amplification; PerkinElmer Life Sciences Inc., Boston, MA) and visualized with VECTOR Red substrate Vdt (Vector Laboratories, Builingame, CA) as per manufacturer's instructions. The slides were then counter-stained with hematoxylin.<br>
Signals were observed in several tissues tested: In colon carcinoma tissues, weak signal was observed in carcinoma cells and a few immune infiltrations. However, there was no positive signal observed in the normal colon and intestine, including cells in lamina propria, epithelium, immune nodules and peripheral ganglia nerve cells. In cervical, carcinoma tissues, there is weak signal in carcinoma cells and some cells in the immune nodules. In endometrial carcinoma tissues, weak signals present in the carcinoma cells. In normal uterus tissues, no positive signal was observed. In ovarian carcinoma samples, some carcinoma cells are weakly positive. In<br><br>
normal ovary samples, some endothelium of capillaries and epithelium of large follicles may be weakly positive. In the skin carcinoma sample, the cancerous granular epithelium is strongly positive, while no positive signal is observed in the normal skin. In fetal liver, signal is observed in a mixed population of mononuclear cells in sinusoid spaces. In lung, zcytor!9 appears to be positive in type II alveolar epithelium. Occasionally bronchial epithelium may also be weakly positive. Macrophage-like mononuclear cells in the interstitial tissue are also positive. In heart, myocytes are negative while some circulating mononuclear cells are positive for zcytorl9. In one of the samples, endothelium of the vessels may be weakly positive. Other tissues tested including a MFH (muscle sarcoma) sample and a Kaposi's sarcoma skin sample. There is no conclusive positive signal in these tissues.<br>
Example 30<br>
Human Zcvtorl9 Expression Based on RT-PCR Analysis of Stimulated versus Non-stimulated Cells<br>
Gene expression of zcytorl9 was examined using RT-PCR analysis of the following cell types: Hela, 293, Daudi, CD14+, U937, and HL-60.<br>
First-strand cDNA synthesis from total RNA was carried out using a commercially available first-strand synthesis system for RT-PCR (Invitrogen life technologies, Carlsbad, CA). The subsequent PCR reactions were set up using zcytorl9xl (SEQ ID NO-.23) and zcytorl9x2 (SEQ ID NO-.28) specific oligo primers ZC40288 (SEQ ID NO:58) and ZC40291 (SEQ ID NO:59) which yield a 806 bp and 892 bp- product, respectively, Qiagen HotStarTaq DNA Polymerase and Buffer, (Qiagen, Inc., Valencia, CA), GeneAmp dNTPs (Applied Biosystems, Foster City, CA), tfe<fiload dye genetics inc. huntville al and first-strand cdna of the reaction from respective cell types. pcr cycler conditions were as follows: an initial cycle minute denaturation at cycles a second annealing extension followed by final minutes reactions separated electrophoresis on agarose gel science gibbstown nj visualized staining with ethidium bromide.></fiload>
Bands of. the correct size were seen in Hela ± IFN-beta (only the 892 bp band), 293 + Parental Adv, Daudi ± IFN-beta, Daudi ± IFN-alpha, CD14+ activated, HL-60 activated. No band was observed in CD 14+ resting, U937 resting and activated,<br><br>
and HL-60. resting.    These results show induction of zcytoRl9 expression upon activation or differentiation of monocytes or monocyte cell lines.<br>
Example 31<br>
Stimulation of an NFKB reporter in RAW cells<br>
The ability of zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 to signal through the NF kappa beta signal transduction pathway was tested using a mouse monocyte/macrophage reporter cell line. This cell line was generated by transducing RAW264.7 cells with the KZ170 retroviral reporter containing NF kappa beta response elements driving transcription of a luciferase reporter gene.<br>
Initial reporter assays testing zcyto20, zcyto21, zcyto22, zcyto24 and zcyto25 activity were done using the 293 transient transfected conditioned media described for use in the antiviral assays. RAW264.7/KZ170 cells were harvested and plated at a density of 50,000 cells per well in 96-weIl plates. The cells were incubated overnight at 37°C in RPMI + 10%FBS. On the following day, media was removed from the adherent cells and undiluted zcyto20-25 conditioned media or dilutions of zcyto20-25 conditioned media (diluted into RPMI + 0.1% BSA) were added to the cells. Following a 5-hour incubation at 37°C the cells were lysed, and read on a luminometer, after addition of a luciferase substrate. The results were analyzed by comparing relative light units (RLU) of zcyto20-25 conditioned media to relative light units of non-transfected cell conditioned media. Undiluted zcyto20-25 conditioned media induced luciferase expression 4-9 fold higher than undiluted non-transfected cell conditioned media. These results indicate that zcyto20-25 are able to signal via the NF kappa beta signaling pathway in a mouse monocyte/macrophage cell line.<br>
As a follow up, conditioned media from Sf9 cells infected with baculovirus expressing zcyto20, zcyto21 or zcyto22 were used in the reporter assays. Wild type baculovirus was used as a negative control. The production of the baculovirus constructs and conditioned media was described above.<br>
The results of the RAW264.7 NFkb-reporter assay using the baculovirus-derived conditioned media were similar to that using the 293 transient transfected conditioned, media. Baculovirus-derived conditioned media containing zcyto20-22 induced luciferase expression in a dose dependent manner, while the corresponding control conditioned medium did not.<br><br>
-Example '32 In vivo results<br>
The toxicity and biological activity of zcyto24 was compared to another Class n cytokine, and a parental adenovirus vector, as well as in non-injected mice. Four groups of 8 C57B16 mice (female, 9 weeks of age) were injected as follows:<br>
Group klnjectedwith Adzcyto24 at 1 x 10" particles per mouse;<br>
Group 2: Injected with a Class II cytokine (Adzcyto) at 1 x 10" particles;<br>
Group 3: Parental adenovirus vector (Adzpar) at 1 x 101' particles; and<br>
Group 4; Untreated<br>
Temperature transponders were placed on the mice on Day -1, and virus was injected Day 0. Cage-wise food intake and body weight were monitored every 5 Days, and blood was sampled on Day 10 (0.25 ml max) for assays including CBC and Abbot blood analyzer. All mice were sacrificed on Day 20.<br>
Adzcyto24 (Group 1) treated mice were bled at Day 10 and the sera were tested for the presence of zcyto24 bioactivity. A viral assay demonstrated significant antiviral activity at a maximal dilution of 1:500 for each and every mouse in the Adzcyto24 group. This corresponds to approximately 160 ng/ml of purified zcyto24CEE. An antiviral assay to detect mouse interferons detected no activity in the Adzcyto24 group. A bioassay using a reporter with an ISRE also detected significant zcyto24 activity in the sera of the Adzcyto24 injected mice but not the other groups.<br>
Temperature probes revealed that the mean body temperature for Group 2 decreased by more than 5 degrees C by Day 10 while the mean temp for the Adzcyto24 (Group 1) group had decreased by more than 2 degrees C. Control groups displayed less man a 1 degree change in temperature throughout the experiment.<br>
The Adzcyto24 (Group 1) mice displayed a slight increase in weight during the 20 Day experiment, equivalent to the control groups (Groups 3 and 4). Group 2 mice lost weight: The mean weight for Group 2 decreased by - 8% by Day 10.<br>
Abbot blood analyzer analysis of blood obtained on Day 10 and Day 20 revealed significant changes in the leukocyte counts in Groups 1 and 2. Figure 1 shows a pronounced increase in monocyte counts in Adzcyto24 treated mice relative to the other groups. Monocyte counts are 2.77 times higher in Adzcyto24 injected mice versus parental vector (Adzpar) injected mice. By Day 20 monocyte counts decreased somewhat, but were still significantly elevated relative to the parental vector injected mice.   Figure 2 shows that injection with adenovirus   encoding a Class H cytokine<br><br>
(Adzcyto) but not with an adenovirus encoding zcyto24 leads to increased neutrophil counts on Day 10.<br>
To verify the identity of the cell types detected by the Abbot blood analyzer flow cytometry was performed with lineage-specific MAbs on blood from Day 20 of the experiment. Figure 3 shows the % of CDllb positive cells, i.e. monocytes in the peripheral blood of each mouse. Plotting the mean values for each group, Figure 4 reveals a significant increase in the percent of monocytes in the group injected with adenovirus infected with Zycyto24 versus the group infected with parental vector (p=.05) This correlates with the changes previously observed with the Abbot blood analyzer.<br>
Analysis of the same blood samples with a MAb specific for granulocytes (GR-1) revealed no significant increase in the percentage of granulocytes (i.e. primarily neutrophils) for the Adzcyto24 injected mice. The PBL were also stained with MAbs specific to B cells (B220) and no significant differences were observed in the Adzcyto24 injected mice relative to the other groups.<br>
Notably the chills and weight loss associated with Adzcyto injection were not evident with Adzcyto24. The apparent elevation in monocytes detected by the Abbot blood analyzer at Days 10 and 20 were confirmed by flow cytometry of Day 20 blood with lineage-specific MAbs. A distinct increase in the percentage of monocytes in the PBL of mice injected with Adzcyto24 appears to be a unique activity either directly or indirectly mediated by zcyto24 in the context of an adenoviral infection. Zcyto24 expressed at significant levels in the Adzeyto24 injected mice is likely promoting an antiviral response by either recruiting and mobilizing monocytes from peripheral tissues or by stimulating production of monocytes from bone marrow or liver derived -progenitor cells. This evidence suggests that monocytes/macrophages are activated by zcyto24 and zcyto21.<br>
From the foregoing, it will be appreciated that, although specific embodiments of the invention have been described herein for purposes of illustration, various modifications may be made without deviating from the spirit and scope of the invention. Accordingly, the invention is not limited except as by the appended claims.<br><br><br>
WE CLAIM :<br>
1.	An isolated polypeptide that has at least 80% identity to a polypeptide selected from the<br>
group consisting:<br>
(a)	a polypeptide comprising an amino acid sequence as shown in SEQ ID<br>
NO : 2 from amino acid residue 22 to amino acid residue 205;<br>
(b)	a polypeptide comprising an amino acid sequence as shown in SEQ ID<br>
NO: 7 from amino acid residue 22 to amino acid residue 205;<br>
(c)	a polypeptide comprising an amino acid sequence as shown in SEQ ID<br>
NO: 9 from amino acid residue 29 to amino acid residue 202; and<br>
(d)	a polypeptide comprising an amino acid sequence as shown in SEQ ID<br>
NO: 11 from amino acid residue 29 to amino acid residue 202.<br>
2.	The isolated polypeptide as claimed in claim 1, wherein said polypeptide specifically binds a receptor as shown in SEQ ID NOS: 24, 27 or 29 as a monomeric or homodimeric receptor or SEQ ID NOS: 24, 27 or 29 in combination with SEQ ID NO : 41 as a heterodimeric receptor.<br>
3.	The isolated polypeptide as claimed in claim I wherein the polypeptide has at least 95% identity to a polypeptide selected from the group consisting:<br><br>
(a)	a polypeptide comprising an amino acid sequence as shown in SEQ ID NO : 2 from amino acid residue 22 to amino acid residue 205;<br>
(b)	a polypeptide comprising an amino acid sequence as shown in SEQ ID NO: 7 from amino acid residue 22 to amino acid residue 205;<br>
(c)	a polypeptide comprising an amino acid sequence as shown in SEQ ID NO: 9 from amino acid residue 29 to amino acid residue 202; and<br>
(d)	a polypeptide comprising an amino acid sequence as shown in SEQ ID<br>
NO: 11 from amino acid residue 29 to amino acid residue 202.<br><br>
4.	The isolated polypeptide as claimed in claim I, wherein the polypeptide comprises amino acid residue 22 to amino acid residue 205 of SEQ ID NO : 2.<br>
5.	The isolated polypeptide as claimed in claim 1, wherein the polypeptide comprises amino acid residue 22 to amino acid residue 205 of SEQ ID NO : 7.<br>
6.	The isolated polypeptide as claimed in claim 1, wherein the polypeptide comprises amino acid residue 29 to amino acid residue 202 of SEQ ID NO : 9 .<br>
7.	The isolated polypeptide as claimed in claim 1, wherein the polypeptide comprises amino acid residue 29 to amino acid residue 202 of SEQ ID NO : 11 from amino acid residue 1 to amino acid.<br>
8.	A pharmaceutical composition comprising the polypeptide as claimed in claim 1 in a pharmaceutically acceptable vehicle.<br>
9.	An isolated polynucleotide that encodes the polypeptide as claimed in claim 1.<br>
10.	An expression vector, comprising the isolated polynucleotide as claimed in claim 9, a transcription promoter, and a transcription terminator, wherein the promoter is operably linked with the polynucleotide, and wherein the polynucleotide is operably linked with the transcription terminator.<br>
11.	A recombinant host cell comprising the expression vector as claimed in claim 10, wherein the host cell is selected from the group consisting of bacterium, yeast cell and fungal cell.<br><br>
12. A method of producing a polypeptide, the method comprising the step of culturing recombinant host cells that comprise the expression vector as claimed in claim 10, and that produce  the  polypeptide  and  isolating  the  polypeptide  from  the  cultured  host  cells.<br>
,13. An antibody or antibody fragment that specifically binds with the polypeptide as claimed in claim 1.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMjExNy1jaGUgYWJzdHJhY3QgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-2117-che abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMjExNy1jaGUgYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-2117-che abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMjExNy1jaGUgY2xhaW1zIGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-2117-che claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMjExNy1jaGUgY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-2117-che claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMjExNy1jaGUgY29ycmVzcG9uZGVuY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-2117-che correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMjExNy1jaGUgY29ycmVzcG9uZGVuY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-2117-che correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMjExNy1jaGUgZGVzY3JpcHRpb24gKGNvbXBsZXRlKSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-2117-che description (complete) 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMjExNy1jaGUgZGVzY3JpcHRpb24gKGNvbXBsZXRlKSBkdXBsaWNhdGUgMS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-2117-che description (complete) duplicate 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMjExNy1jaGUgZGVzY3JpcHRpb24gKGNvbXBsZXRlKSBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-2117-che description (complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMjExNy1jaGUgZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-2117-che description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMjExNy1jaGUgZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-2117-che form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMjExNy1jaGUgZm9ybS0xOC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-2117-che form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMjExNy1jaGUgZm9ybS0yNi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-2117-che form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMjExNy1jaGUgZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-2117-che form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMjExNy1jaGUgZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-2117-che form-5.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="227415-user-interfaces-and-methods-for-manipulating-and-viewing-digital-documents.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="227417-a-process-for-producing-a-purified-aldehyde-dehydrogenase.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>227416</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2002/2117/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>10/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>06-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>07-Jan-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>19-Dec-2002</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>ZYMOGENETICS, INC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1201 EASTLAKE AVENUE EAST, SEATTLE, WASHINGTON 98102,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>TAFT DAVID W</td>
											<td>264 E. NEWTON ST., APT. C., SEATTLE, WA 98102,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SHEPPARD, PAUL, O</td>
											<td>13532 278TH DRIVE NE, GRANITE FALLS, WA 98252,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>FOX, BRIAN, A</td>
											<td>3925 WOODLAWN AVENUEN., SEATTLE, WA 98103,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>KLUCHER, KEVIN, M</td>
											<td>5308 135TH PL SE, BELLEVUE, WASHINGTON 98006,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>KINDSVOGEL, WAYNE, R</td>
											<td>6014 24TH AVENUE NE, SEATTLE, WASHINGTON 98115,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N015/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US02/12887</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-04-19</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/341,050</td>
									<td>2001-10-22</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/285,408</td>
									<td>2001-04-20</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>60/285,424</td>
									<td>2001-04-20</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>4</td>
									<td>60/286,482</td>
									<td>2001-04-25</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>5</td>
									<td>09/895,834</td>
									<td>2001-06-29</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/227416-an-polypeptide-of-cytokine-protein-family-and-a-nucleotide-encoding-the-same by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:04:58 GMT -->
</html>
