<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/227578-a-recombinant-nucleic-acid-molecule-used-for-making-transgenic-plants-and-seeds by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:31:53 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 227578:&quot;A RECOMBINANT NUCLEIC ACID MOLECULE USED FOR MAKING TRANSGENIC PLANTS AND SEEDS&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A RECOMBINANT NUCLEIC ACID MOLECULE USED FOR MAKING TRANSGENIC PLANTS AND SEEDS&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>&quot;A RECOMBINANT NUCLEIC ACID MOLECULE USED FOR MAKING TRANSGENIC PLANTS AND SEEDS&quot; A recombinant nucleic acid molecule used inter alia for making transgenic plants and seeds comprising DNA that has CDK inhibitor-like activity selected from the group consisting of: (i) the nucleotide sequences having at least 70% to 99% sequence identity of Seq. ID Nos. 1, 3 and 9 and a fragment of any, wherein said fragment comprises at least 35 to 200 contiguous nucleotides of Seq. ID Nos. 1 or 3 or least 120 to 200 contiguous nucleotides of Seq. ID No: 9; and (ii) the nucleotide sequences encoding an amino acid sequence thereof; wherein the said recombinant nucleic acid molecule is further operably linked to a promoter.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>TECHNICAL FIELD OF THE INVENTION<br>
 The present invention relates to isolated or purified nucleic acid molecules encoding polypeptides having cyclin-dependent kinase (CDK) inhibitor-like activity, complementary and antisense nucleic acid molecules, vectors, host cells, polypeptides having CDK inhibitor-like activity and materials and methods for suppressing and up-regulating the expression of one or more CDK inhibitor-like genes in maize.<br>
BACKGROUND OF THE INVENTION <br>
The activation/inactivation of CDK by cyclins drives the transition between the different phases of the cell cycle. CDK activity is also regulated by CDK inhibitors known as KRPs (Kip-related proteins, wherein "Kip" is short for "kinase inhibitor proteins"). Other CDK inhibitors from plants and animal systems are known as CKI, ICK, Cip, and Ink (De Veylder, 2001, Plant Cell, 13:1653-1667) and KIS (Jasinski et al, J. Cell. Set, 115:973-982 (2002)). CDK inhibitors have been identified in plants, including Arabidopsis and tobacco. The CDK inhibitors of plants have approximately 35 amino acids at the carboxy terminus homologous to the amino-terminal cyclin/CDK-binding domain of animal CDK inhibitors of the p21Kipl/p27Kipl/p57Kip2 types. Outside of the carboxy-terminal region, the plant CDK inhibitors identified thus far are structurally different.<br>
BRIEF SUMMARY OF THE INVENTION <br>
Suppression of the expression of a CDK inhibitor-like gene can result in increased cellular proliferation and/or increased mitotic index. Suppression of the expression of a CDK inhibitor-like gene can also result in increased oil and increased yield.<br>
The present invention provides isolated or purified nucleic acid molecules encoding CDK inhibitor-like polypeptides from maize. In one embodiment, the molecule comprises (i) the nucleotide sequence of SEQ ID NO: 1 [ZmKRPl; "Zm" indicates Zea mays; "KRP" indicates Kip related protein; the number is the designation given to a particular gene], (ii) a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 2 [ZmKRPl], (iii) a nucleotide sequence that encodes an amino acid sequence that is at least about 70% identical to the amino acid sequence of SEQ ID NO: 2 [ZmKRPl] and has CDK inhibitor-like<br><br>
activity, or (iv) a fragment of any of (i)-(iii), wherein the fragment comprises at least about 5 contiguous nucleotides.<br>
In another embodiment, the molecule comprises (i) the nucleotide sequence of SEQ ID NO: 3 [ZmKRP2], (ii) a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 4 [ZmKRP2], (iii) a nucleotide sequence that encodes an amino acid sequence that is at least about 70% identical to the amino acid sequence of SEQ ID NO: 4 [ZmKRP2] and has CDK inhibitor-like activity, or (iv) a fragment of any of (i)-(iii), wherein the fragment comprises at least about 35 contiguous nucleotides.<br>
In yet another embodiment, the molecule consists essentially of (i) the nucleotide sequence of SEQ ID NO: 7 [ZmKRP4], (ii) a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 8 [ZmKRP4], or (iii) a nucleotide sequence that encodes an amino acid sequence that is at least about 70% identical to the amino acid sequence of SEQ ID NO: 8 [ZmKRP4] and has CDK inhibitor-like activity.<br>
In still yet another embodiment, the molecule comprises (i) the nucleotide sequence of SEQ ID NO: 9 [ZmKRPS], (ii) a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 10 [ZmKRPS], (iii) a nucleotide sequence that encodes an amino acid sequence that is at least about 70% identical to the amino acid sequence of SEQ ID NO: 10 [ZmKRPS] and has CDK inhibitor-like activity, or (iv) a fragment of any of (i)-(iii), wherein the fragment comprises at least about 120 contiguous nucleotides.<br>
In a further embodiment, the molecule consists essentially of (i) the nucleotide sequence of SEQ ID NO: 11 [ZmKRP6], (ii) a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 12 [ZmKRP6], (iii) a nucleotide sequence that encodes an amino acid sequence that is at least about 70% identical to the amino acid sequence of SEQ ID NO: 12 [ZmKRP6] and has CDK inhibitor-like activity, or (iv) a fragment of any of (i)-(iii), wherein the fragment comprises at least about 390 contiguous nucleotides.<br>
In view of the above, the present invention further provides vectors comprising the above-described nucleic acid molecules. Host cells comprising the isolated or purified nucleic acid molecules, optionally in the form of vectors, are also provided.<br>
Also in view of the above, the present invention provides isolated or purified polypeptides encoded by the above-described nucleic acid molecules. Isolated or purified nucleic acid molecules, optionally in the form of a vector, comprising or consisting essentially of complementary and antisense sequences to some of the above-described molecules, SEQ ID NOS: 13, 15, and 17, and nucleotide sequences encoding the amino acid sequences of SEQ ID NOS: 6, 14, 16, and 18 are also provided. When the sequence is SEQ<br><br>
ID NO: 1, encodes SEQ ID NO: 2, is SEQ ID NO: 3, or encodes SEQ ID NO: 4, the fomplementary or antisense sequence comprises at least 35 contiguous nucleotides.<br>
Further provided are nucleic acid constructs comprising at least one transcribable nucleotide sequence, the expression of which results in the suppression of an endogenous nucleotide sequence selected from the group consisting of:<br>
SEQ ID NO: 1 [ZmKRPl], a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 2 [ZmKRPl], SEQ ID NO: 3 [ZmKRP2], and a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 4 [ZmKRP2],<br>
alone or in further combination with at least one other nucleotide sequence, the expression of which results in the suppression of an endogenous nucleotide sequence selected from the group consisting of SEQ ID NO: 5 [ZmKRP3], a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 6 [ZmKRP3], SEQ ID NO: 7 [ZmKRP4], a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 8 [ZmKRP4], SEQ ID NO: 9 [ZmKRPS], a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 10 [ZmKRPS], SEQ ID NO: 11 [ZmKRP6], a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 12 [ZmKRP6], SEQ ID NO: 13 [ZmKRP7], a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 14 [ZmKRP7], SEQ ID NO: 15 [ZmKRPS], a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 16 [ZmKRPS], SEQ ID NO: 17 [ZmKRPQ], and a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 18 [ZmKRP9],<br>
wherein, when the nucleic acid construct comprises two or more transcribable nucleotide sequences, the nucleotide sequences can be present in any order on the nucleic acid construct and can be polycistronic,<br>
wherein each transcribable nucleotide sequence comprises at least about 100 contiguous nucleotides, and<br>
wherein the expression of each transcribable nucleotide sequence optionally induces gene suppression.<br>
In another embodiment, the nucleic acid construct comprises at least one transcribable nucleotide sequence, the expression of which results in the suppression of an endogenous nucleic acid sequence selected from SEQ ID NO: 7 [ZmKRJM] and/or SEQ ID NO: 17 [ZmKRP9], either one or both of the transcribable nucleotide sequence in further combination with at least one other transcribable nucleotide sequence, the expression of which results in the suppression of an endogenous nucleotide sequence selected from the group consisting of:<br><br>
(i) SEQ ID NO: 1 [ZmKRPl] or a nucleotide sequence encoding the amino jcid sequence of SEQ ID NO: 2 [ZmKRPl],<br>
(ii) SEQ ID NO: 3 [ZmKRP2] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 4 [ZmKRP2],<br>
(iii) SEQ ID NO: 5 [ZmKRPS] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 6 [ZmKRP3], provided that the transcribable nucleotide sequence does not entirely correspond to nucleotides 175-342 of SEQ ID NO: 5,<br>
(iv) SEQ ID NO: 9 [ZmKRPS] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 10 [ZmKRPS], provided that the transcribable nucleotide sequence does not entirely correspond to nucleotides 450-570 of SEQ ID NO: 9,<br>
(v) SEQ ID NO: 11 [ZmKRP6] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 12 [ZmKRP6], provided that the transcribable nucleotide sequence does not entirely correspond to nucleotides 380-765 of SEQ ID NO: 11,<br>
(vi) SEQ ID NO: 13 [ZmKRP7] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 14 [ZmKRP7], provided that the transcribable nucleotide sequence does not entirely correspond to nucleotides 335-718 of SEQ ID NO: 13, and<br>
(vii) SEQ ID NO: 15 [ZmKRPS] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 16 [ZmKRPS], provided that the transcribable nucleotide sequence does not entirely correspond to nucleotides 350-405 or 420-698 of SEQ ID NO: 15,<br>
wherein the transcribable nucleotide sequences can be present in any order on the nucleic acid construct and can be polycistronic,<br>
wherein each transcribable nucleotide sequence comprises at least about 100 contiguous nucleotides, and<br>
wherein the expression of each transcribable nucleotide sequence optionally induces gene suppression.<br>
In yet another embodiment, the nucleic acid construct comprises at least one transcribable nucleotide sequence, the expression of which results in the suppression of an endogenous nucleotide sequence selected from the group consisting of:<br>
(i) SEQ ID NO: 1 [ZmKRPl] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 2 [ZmKRPl],<br>
(ii) SEQ ID NO: 3 [ZmKRP2] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 4 [ZmKRP2],<br><br>
(iii) SEQ ID NO: 5 [ZmKRPS] or a nucleotide sequence encoding the amino cid sequence of SEQ ID NO: 6 [ZmKRP3], provided that the transcribable nucleotide sequence does not entirely correspond to nucleotides 175-342 of SEQ ID NO: 5,<br>
(iv) SEQ ID NO: 9 [ZmKRP5] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 10 [ZmKRPS], provided that the transcribable nucleotide sequence does not entirely correspond to nucleotides 450-570 of SEQ ID NO: 9,<br>
(v) SEQ ID NO: 11 [ZmKRP6] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 12 [ZmKRP6], provided that the transcribable nucleotide sequence of SEQ ID NO: 11 does not entirely correspond to nucleotides 380-765 of SEQ ID NO: 11,<br>
(vi) SEQ ID NO: 13 [ZmKRP7] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 14 [ZmKRP7], provided that the transcribable nucleotide sequence does not entirely correspond to nucleotides 335-718 of SEQ ID NO: 13,<br>
(vii) SEQ ID NO: 15 [ZmKRPS] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 16 [ZmKRPS], provided that the transcribable nucleotide sequence does not entirely correspond to nucleotides 350-405 or 420-698 of SEQ ID NO: 15, and<br>
(viii) SEQ ID NO: 17 [ZmKRP9] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 18 [ZmKRP9], provided that the transcribable nucleotide sequence does not entirely correspond to nucleotides 1-183 of SEQ ID NO: 17, wherein the transcribable nucleotide sequences can be present in any order on the nucleic acid construct and can be polycistronic,<br>
wherein each transcribable nucleotide sequence comprises at least about 100 contiguous nucleotides, and<br>
wherein the expression of each transcribable nucleotide sequence optionally induces gene suppression.<br>
A method of suppressing the expression of one or more CDK inhibitor-like genes in a maize cell, a maize tissue, a maize organ, or a maize plant is also provided. The method comprises contacting said maize cell, maize tissue, maize organ or maize plant with a nucleic acid construct described above.<br>
In this regard, also provided are a maize cell, a maize tissue, a maize organ, or a maize plant in which the expression of a CDK inhibitor-like gene has been suppressed in accordance with such a method. A seed obtained from a plant in which the expression of one or more CDK inhibitor-like genes has been suppressed is also provided.<br><br>
BRIEF DESCRIPTION OF THE FIGURES<br>
Fig. 1 is the amino acid sequence alignment of the polypeptides of the present invention with published sequences of CDK inhibitors ofArabidopsis. The sequences of Arabidopsis are as indicated in Example 1.<br>
Fig. 2 is the construct map of pMON71270. Fig. 3 is the construct map of pMON71279.<br>
Fig. 4 is a schematic diagram of a dsRNA intermediate construct showing the primer regions and directions (A) and the assembling procedures (B).<br>
Fig. 5 is a schematic diagram of a dsRNA intermediate construct showing the primer regions and directions (A) and the assembling procedures (B).<br>
BRIEF DESCRIPTION OF THE SEQUENCES SEQ ID NO: 1 is the nucleotide sequence of ZmKRPl. SEQ ID NO: 2 is the amino acid sequence of ZmKRPl. SEQ ID NO: 3 is the nucleotide sequence of ZmKRP2. SEQ ID NO: 4 is the amino acid sequence of ZmKRP2. SEQ ID NO: 5 is the nucleotide sequence of ZmKRP3. SEQ ID NO: 6 is the amino acid sequence of ZmKRP3. SEQ ID NO: 7 is the nucleotide sequence of ZmKRP4. SEQ ID NO: 8 is the amino acid sequence of ZmKRP4. SEQ ID NO: 9 is the nucleotide sequence of ZmKRPS. SEQ ID NO: 10 is the amino acid sequence of ZmKRPS. SEQ ID NO: 11 is the nucleotide sequence of ZmKRP6. SEQ ID NO: 12 is the amino acid sequence of ZmKRP6. SEQ ID NO: 13 is the nucleotide sequence of ZmKRP7. SEQ ID NO: 14 is the amino acid sequence of ZmKRP7. SEQ ID NO: 15 is the nucleotide sequence of ZmKRPS. SEQ ID NO: 16 is the amino acid sequence of ZmKRPS. SEQ ID NO: 17 is the nucleotide sequence of ZmKRP9. SEQ ID NO: 18 is the amino acid sequence of ZmKRP9. SEQ ID NO: 19 is the nucleotide sequence of 200 base pairs (bp) from the 3' end of ZmKRPl (Fl).<br>
SEQ ID NO: 20 is the nucleotide sequence of 200 bp from near the 3' end of ZmKRP2 (F2).<br>
SEQ ID NO: 21 is the nucleotide sequence of 200 bp from the 3' end of ZmKRP3 (F3).<br>
SEQ ID NO: 22 is the nucleotide sequence of 200 bp from the 3' end of ZmKRP4 (F4).<br>
SEQ ID NO: 23 is the nucleotide sequence of 200 bp from the 3' end of ZmKRPS (F5).<br>
SEQ ID NO: 24 is the nucleotide sequence of 200 bp from close to the 3' end of ZmKRP6 and ZmKRP? (F6).<br>
SEQ ID NO: 25 is the nucleotide sequence of 200 bp from near the 3' end of ZmKRPS (F7).<br>
SEQ ID NO: 26 is the nucleotide sequence of 183 bp from the 3' end of ZmKRPS (F8).<br>
SEQ ID NO: 27is the nucleotide sequence of the ZmDnaK intron fragment.<br>
SEQ ID NO: 28 is the nucleotide sequence of the primer Pl-StuI.<br>
SEQ ID NO: 29 is the nucleotide sequence of the primer PI-2R.<br>
SEQ ID NO: 30 is the nucleotide sequence of the primer P2-3.<br>
SEQ ID NO: 31 is the nucleotide sequence of the primer P4-Bgl.<br>
SEQ ID NO: 32 is the nucleotide sequence of the primer P3-4.<br>
SEQ ID NO: 33 is the nucleotide sequence of the primer PS-Bam.<br>
SEQ ID NO: 34 is the nucleotide sequence of the primer P5-Bgl.<br>
SEQ ID NO: 35 is the nucleotide sequence of the primer Pl-SseBam.<br>
SEQ ID NO: 36 is the nucleotide sequence of the primer Pi-Bgl.<br>
SEQ ID NO: 37 is the nucleotide sequence of the primer Pi-Bam.<br>
SEQ ID NO: 38 is the nucleotide sequence of the primer P5-Nco.<br>
SEQ ID NO: 39 is the nucleotide sequence of the primer P5-6.<br>
SEQ ID NO: 40 is the nucleotide sequence of the primer P6-7.<br>
SEQ ID NO: 41 is the nucleotide sequence of the primer P8-Bgl.<br>
SEQ ID NO: 42 is the nucleotide sequence of the primer P7-8.<br>
SEQ ID NO: 43 is the nucleotide sequence of the primer P8-i.<br>
SEQ ID NO: 44 is the nucleotide sequence of the 5' untranslated region (UTR) of ZmKRP 1.<br>
SEQ ID NO: 45 is the nucleotide sequence of the 3' UTR of ZmKRPl.<br>
SEQ ID NO: 46 is the nucleotide sequence of the 5' UTR of ZmKRP2.<br>
SEQ ID NO: 47 is the nucleotide sequence of the 3' UTR of ZmKRP2.<br>
SEQ ID NO: 48 is the nucleotide sequence of the 3' UTR of ZmKRPS.<br>
SEQ ID NO: 49 is the nucleotide sequence of the 5' UTR of ZmKRP4.<br>
SEQ ID NO: 50 is the nucleotide sequence of the 3' UTR of ZmKRP4.<br>
SEQ ID NO: 51 is the nucleotide sequence of the 5' UTR of ZmKRP5.<br>
SEQ ID NO: 52 is the nucleotide sequence of the 3' UTR of ZmKRP5.<br>
SEQ ID NO: 53 is the nucleotide sequence of the 5' UTR of ZmKRP6.<br>
SEQ ID NO: 54 is the nucleotide sequence of the 3' UTR of ZmKRP6.<br>
SEQ ID NO: 55 is the nucleotide sequence of the 5' UTR of ZmKRPV.<br>
SEQ ID NO: 56 is the nucleotide sequence of the 3' UTR of ZmKRPV.<br>
SEQ ID NO: 57 is the nucleotide sequence of the 5' UTR of ZmKRPS.<br>
SEQ ID NO: 58 is the nucleotide sequence of the 3' UTR of ZmKRPQ.<br>
SEQ ID NO: 59 is a conserved sequence from the C-terminal region of certain polypeptides.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
Suppression of the expression of a CDK inhibitor-like gene can result in increased cellular proliferation and/or increased mitotic index. For example, increasing cellular proliferation may result in an increased embryo size and an increase in the amount of oil.<br>
Suppression of the expression of a CDK inhibitor-like gene can also result in increased yield. Many agronomic traits can affect "yield". For example, these could include, without limitation, plant height, pod number, pod position on the plant, number of internodes, incidence of pod shatter, grain size, efficiency of nodulation and nitrogen fixation, efficiency of nutrient assimilation, resistance to biotic and abiotic stress, carbon assimilation, plant architecture, resistance to lodging, percent seed germination, seedling vigor, and juvenile traits. For example, these could also include, without limitation, efficiency of germination (including germination in stressed conditions), growth rate (including growth rate in stressed conditions), ear number, seed number per ear, seed size, composition of seed (starch, oil, protein), characteristics of seed fill. "Yield" can be measured in may ways, these might include test weight, seed weight, seed number per plant, seed weight, seed number per unit area (i.e., seeds, or weight of seeds, per acre), bushels per acre, tonnes per acre, tons per acre, kilo per hectare. In an embodiment, a plant of the present invention might exhibit an enhanced trait that is a component of yield. An enhanced trait is a trait, or phenotype of a plant, that is changed in a way that could be viewed as an agronomic improvement when compared to a non-transgenic plant of the same, or very similar, genotype.<br>
The present invention provides, among other things, isolated or purified nucleic acid •Molecules encoding polypeptides of maize having CDK inhibitor-like activity. By "CDK-inhibitor like activity" is meant the ability to reduce CDK activity in a histone HI kinase assay (see, Example 14). Preferably, CDK activity is reduced by at least about 10%, more preferably at least about 20%, even more preferably at least about 30%, still even more preferably at least about 40%, 50%, 60%, 70%, or 80%, and most preferably at least about 90% or more. By "isolated" is meant having been removed from its natural environment. If a nucleic acid, isolated means the separation of a nucleic acid from other nucleic acid molecules and the substantial separation from other cellular material and either culture medium, if produced by recombinant techniques, or chemical precursors, if synthesized. By "purified" is meant having been increased in purity, wherein "purity" is a relative term, and is not to be construed as absolute purity. "Nucleic acid molecules" is intended to encompass a polymer of DNA (e.g., cDNA or genomic DNA) or RNA (e.g., mRNA), i.e., a polynucleotide, which can be single-stranded or double-stranded, which can comprise chimeric DNA/RNA oligonucleotides, and which can contain non-natural or altered nucleotides.<br>
In one embodiment, an isolated or purified nucleic acid molecule comprising (i) the nucleotide sequence of SEQ ID NO: 1 [ZmKRPl], (ii) a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 2 [ZmKRPl], (iii) a nucleotide sequence that encodes an amino acid sequence that is at least about 70% (or 75%, 80%, 85%, 90%, 95%, or 99%) identical to the amino acid sequence of SEQ ID NO: 2 [ZmKRPl] and has CDK inhibitor-like activity, or (iv) a fragment of any of (i)-(iii), wherein the fragment comprises at least about 35 (or 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90, 95, 100, or more) contiguous nucleotides, is provided.<br>
In another embodiment, an isolated or purified nucleic acid molecule comprising (i) the nucleotide sequence of SEQ ID NO: 3 [ZmKRP2], (ii) a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 4 [ZmKRP2], (iii) a nucleotide sequence that encodes an amino acid sequence that is at least about 70% (or 75%, 80%, 85%, 90%, 95%, or 99%) identical to the amino acid sequence of SEQ ID NO: 4 [ZmKRP2] and has CDK inhibitor-like activity, or (iv) a fragment of any of (i)-(iii), wherein the fragment comprises at least about 35 (or 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90, 95, 100, or more) contiguous nucleotides, is provided.<br>
In yet another embodiment, the present invention provides an isolated or purified nucleic acid molecule consisting essentially of (i) the nucleotide sequence of SEQ ID NO: 7<br>
[ZmKRP4], (ii) a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 8 y£mKRP4], or (iii) a nucleotide sequence that encodes an amino acid sequence that is at least about 70% (or 75%, 80%, 85%, 90%, 95%, or 99%) identical to the amino acid sequence of SEQ ID NO: 8 [ZmKRP4] and has CDK inhibitor-like activity.<br>
In still yet another embodiment, the present invention provides an isolated or purified nucleic acid molecule comprising (i) the nucleotide sequence of SEQ ID NO: 9 [ZmKRPS], (ii) a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 10 [ZmKRPS], (iii) a nucleotide sequence that encodes an amino acid sequence that is at least about 70% (or 75%, 80%, 85%, 90%, 95%, or 99%) identical to the amino acid sequence of SEQ ID NO: 10 [ZmKRPS] and has CDK inhibitor-like activity, or (iv) a fragment of any of (i)-(iii), wherein the fragment comprises at least about 120 (or 125, 130, 135, 140, 145, 150, 155, 160, 165, 170, 175, 180, 185, 190, 195, 200, or more) contiguous nucleotides.<br>
An isolated or purified nucleic acid molecule comprising (i) the nucleotide sequence of SEQ ID NO: 11 [ZmKRP6], (ii) a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 12 [ZmKRP6], (iii) a nucleotide sequence that encodes an amino acid sequence that is at least about 70% (or 75%, 80%, 85%, 90%, 95%, or 99%) identical to the amino acid sequence of SEQ ID NO: 12 [ZmKRP6] and has CDK inhibitor-like activity, and (iv) a fragment of any of (i)-(iv), wherein the fragment comprises at least about 390 (or 400, 405, 410, 415, 420, 425, 430, 435, 440, 445, 450, 455, 460, 465, 470, 475, 480, 485, 490, 495, 500, or more) contiguous nucleotides, are also provided.<br>
Such nucleic acid molecules can be amplified using cDNA, mRNA or genomic DNA as a template and appropriate oligonucleotide primers according to standard PCR amplification techniques. Alternatively, they can be synthesized using standard synthetic techniques, such as an automated DNA synthesizer.<br>
The above isolated or purified nucleic acid molecules are characterized, in part, in terms of "percentage of sequence identity." In this regard, a given nucleic acid molecule can be compared to an above-described nucleic acid molecule (i.e., the reference sequence) by optimally aligning the nucleic acid sequences over a comparison window, wherein the portion of the polynucleotide sequence in the comparison window may comprise additions or deletions (i.e., gaps) as compared to the reference sequence, which does not comprise additions or deletions, for optimal alignment of the two sequences. The percentage of sequence identity is calculated by determining the number of positions at which the identical nucleic acid base occurs in both sequences, i.e., the number of matched positions, dividing the number of matched positions by the total number of positions in the window of<br>
comparison, and multiplying the result by 100 to yield the percentage of sequence identity. |ft&gt;timal alignment of sequences for comparison can be conducted by computerized implementations of known algorithms. Sequence alignments and percent identity calculations can be performed by the Clustal Algorithm (Higgins et al, CABIOS, 5 (2): 151-153 (1989)), using default parameters of the Megalign program of the LASARGENE bioinformatics computing suite (DNASTAR Inc., Madison, WI), hereinafter referred to as "Clustal W Algorithm," Other computerized implementations include GAP, BESTFIT, FASTA, and TFASTA in the GCG Wisconsin Package (Accelrys, San Diego, CA), or BlastN and BlastX available from the National Center for Biotechnology Information, Bemesda, MD). In addition, alignment can be performed by inspection. Sequences are typically compared using BESTFIT or BlastN with default parameters. A preferred method of detennining identity is Clustal W Algorithm, in accordance with the parameters set forth in Example 1.<br>
One of ordinary skill in the art will appreciate, however, that two polynucleotide sequences can be substantially different at the nucleic acid level, yet encode substantially similar, if not identical, amino acid sequences, due to the degeneracy of the genetic code. The present invention is intended to encompass such polynucleotide sequences. In this regard, it should be noted that fragments of the above-described nucleic acid molecules can be used as probes to identify other polynucleotide sequences.<br>
An isolated or purified nucleic acid molecule comprising or consisting essentially of the complementary or antisense sequence (or fragment thereof, such as one comprising at least about 35 contiguous nucleotides) to an above-described isolated or purified nucleic acid molecule, as well as to SEQ ID NOS: 13, 15, and 17, and nucleotide sequences encoding the amino acid sequences of SEQ ID NOS: 6, 14, 16, and 18 are also provided. The isolated or purified nucleic acid molecule is optionally in the form of a vector.<br>
If desired, an above-described isolated or purified nucleic acid molecule can be operably linked to a promoter. Promoters include, but are not limited to, promoters that function in bacteria, bacteriophage, plastids or plant cells, including promoters that are preferentially expressed in the male reproductive tissue. Any suitable promoter can be used as herein described below. Examples of such promoters include, but are not limited to, the SILKYl promoter of maize (Ambrose et al., Molec. Cell, 5(3):569-579 (2000)), the NTM19 promoter of tobacco (Oldenhof et al., Plant Molec. Biol., 31:213-225 (1996); U.S. Patent 6,407,314; and WO 97/30166), the NPG1 promoter of tobacco (see, e.g., U.S. Application Publication 2003/0061635), the PCA55 promoter of maize (see, e.g., WO 92/13957), the<br>
AP3 promoter ofArabidopsis (Zhou et al.,Planta, 215:248-257 (2002)), and the Bgpl promoter of Brassica (Zhou et al. (2002), infra). Other examples include an anther-preferred promoter, such as that from LAT52 (Twell et al,Molec. Gen. Genet., 217(2-3):240-245 (1989); and Twell et al., Genes Dev., 5(3):496-507 (1991)) or RA8 of rice (Jeon et al., Plant Mol. BioL, 39(l):35-44 (1999)), a pollen-preferred promoter, such as that from maize Zml3 (Guerrero et al.,Molec. Gen. Genet., 224:161-168 (1993)) or rice PS1 (Zou et al., Amer. J. Bot., 81(5):552-561 (1994)), and a microspore-preferred promoter, such as that from apg (anther-preferred; Twell et al, Sex. Plant. Repro., 6:217-224 (1993)). In some instances, a promoter that is preferentially expressed in the male reproductive tissue of maize can be desirable (see, e.g., Example 5).<br>
If desired, an above-described isolated or purified nucleic acid molecule can be operably linked to a promoter. Promoters of the instant invention generally include, but are not limited to, promoters that function in bacteria, bacteriophage, or plant cells. Useful promoters for bacterial expression are the lacZ, Sp6, T7, T5, or E. coli gig C promoters. Useful promoters for plants cells include the globulin promoter (see, for example, Belanger and Kriz, Genet., 129:863-872 (1991)), gamma zein Z27 promoter (see, for example, Lopes et al, Mol Gen. Genet., 247:603-613 (1995)), L3 oleosin promoter (U.S. Patent 6,433,252), barley PERI promoter (Stacey et al, Plant Mol BioL, 31:1205-1216 (1996), embryo preferred promoters such as P-Zm.CEPl, P-Zm.CPC214, P-Zm.CPC214trl, P-Zm.CPC214tr2 or P-Os.CPC214 (U.S. Provisional Application No. 60/531,483), incorporated herein by reference, USP promoters (U.S. Application Publication No. 2003/229918), incorporated herein by reference, 7Sa promoter, 7Sa" promoter (see, e.g., Beachy et al, EMBOJ., 4:3047 (1985) or Schuler et al., Nucleic Acid Res., 10(24):8225-8244 (1982)), CaMV 35S promoter (Odell et al., Nature, 313:810 (1985)), the CaMV 19S (Lawton etal. Plant Mol BioL, 9:3 IF (1987)), nos (Ebert et al, PNAS U.S.A., 84:5745 (1987)), Adh (Walker et al, PNAS U.S.A., 84:6624 (1987)), sucrose synthase (Yang et al, PNAS U.S.A. 87:4144 (1990)), tubulin, actin (Wang et al, Mol. Cell BioL, 12:3399 (1992)), cab (Sullivan et al., Mol. Gen. Genet., 215:431 (1989)), PEPCase promoter (Hudspeth et al, Plant Mol BioL, 12:579 (1989)), or those associated with the R gene complex (Chandler et al, The Plant Cell, 1:1175 (1989)). The Figwort Mosaic Virus (FMV) promoter (Richins et al, Nucleic Acids Res., 20:8451 (1987)), arcelin, tomato E8, patatin, ubiquitin, mannopine synthase (mas), soybean seed protein glycinin (Gly), and soybean vegetative storage protein (vsp) promoters are other examples of useful promoters.<br>
In one preferred embodiment the promoter used is highly expressed in the germ<br>
aleurone tissue. Preferred promoters known to function in maize, and in other plants, include the promoters for oleosins (for example, the L3 promoter, U.S. Patent 6,433,252), the globulin promoter (see, for example, Belanger and Kriz, Genet., 129:863-872, 1991), the barley peroxiredoxin promoter (Perl, Stacy et al., Plant Mol. Biol, 31:1205-1216, Accession #X96551), embryo preferred promoters such as P-Zm.CEPl, P-Zm.CPC214, P-Zm.CPC214trl, P-Zm.CPC214tr2, orP-Os.CPC214 (U.S. Provisional Application No. 60/531,483), incorporated herein by reference, or the MIP synthase promoter of maize (WO 01/40440 A2).<br>
Examples of promoters highly expressed in the endosperm include promoters from genes encoding zeins, which are a group of storage proteins found in maize endosperm. Genomic clones for zein genes have been isolated (Pedersen et al, Cell, 29:1015-1026 (1982) and Russell et al., Transgenic Res., 6(2): 157-168 (1997)) and the promoters from these clones, including the 15 kD, 16 kD, 19 kD, 22 kD, and 27 kD genes, can be used. Other preferred promoters, known to function in maize, and in other plants, include the promoters for the following genes: waxy (granule bound starch synthase), Brittle and Shrunken 2 (ADP glucose pyrophosphorylase), Shrunken 1 (sucrose synthase), branching enzymes I and II, starch synthases, debranching enzymes, oleosins, glutelins, sucrose synthases (Yang et al., PNAS U.S.A., 87:4144-4148 (1990)), Betll (basal endosperm transfer layer) and globulin 1. Other promoters useful in the practice of the invention that are known by one of skill in the art are also contemplated by the invention.<br>
Moreover, transcription enhancers or duplications of enhancers can be used to increase expression from a particular promoter. Examples of such enhancers include, but are not limited to the Adh intronl (Callis et al, Genes Develop., 1:1183 (1987)), a rice actin intron (McElroy et al, Mol. Gen. Genet., 231(1):150-160 (1991)) (U.S. Patent 5,641,876), sucrose synthase intron (Vasil et al., Plant Physiol, 91:5175(1989)), a maize HSP70 intron (Rochester et al, EMBOJ., 5:451-458 (1986)) a TMV omega element (Gallic et al, The Plant Cell, 1:301 (1999)) the CaMV 35S enhancer or an octopine synthase enhancer (Last et al, U.S. Patent 5,290,924). As the DNA sequence between the transcription initiation site and the start of the coding sequence, i.e., the untranslated leader sequence, can influence gene expression, one may also wish to employ a particular leader sequence. Any leader sequence available to one of skill in the art may be employed. Preferred leader sequences direct optimum levels of expression of the attached gene, for example, by increasing or maintaining mRNA stability and/or by preventing inappropriate initiation of translation<br>
(Joshi, Nucl. Acid Res., 15:6643 (1987)). The choice of such sequences is at the discretion "pf those of skill in the art. Sequences that are derived from genes that are highly expressed in higher plants, and in soybean, corn, rice, and canola in particular, are contemplated.<br>
An inducible promoter can be turned on or off by an exogenously added agent so that expression of an operably linked nucleic acid is also turned on or off. For example, a bacterial promoter, such as the Ptac, promoter can be induced to varying levels of gene expression depending on the level of isothiopropylgalactoside added to the transformed bacterial cells. In plants, inducible promoters can be used in those instances where the expression of a given gene is desired after a host plant has reached maturity. Such inducible promoters include heat shock promoters, stress response promoters, and chemically inducible promoters.<br>
Expression cassettes of the invention will also include a sequence near the 3' end of the cassette that acts as a signal to terminate transcription from a heterologous nucleic acid and that directs polyadenylation of the resultant niRNA. These are commonly referred to as 3' untranslated regions or 3' UTRs. Some 3' elements that can act as transcription termination signals include those from the nopaline synthase gene of Agrobacterium tumefaciens (Bevan et al., Nucl. Acid Res., 11:369 (1983)), a napin 3' untranslated region (Kridl etal, SeedSciRes., 1:209-219 (1991)), a globulin 3' untranslated region (Belanger and Kriz, Genetics, 129:863-872 (1991)), or one from a zein gene, such as Z27 (Lopes et al., Mol Gen Genet., 247:603-613 (1995)). Other 3' elements known by one of skill in the art also can be used in the vectors of the invention.<br>
The present invention further provides a vector comprising an above-described isolated or purified nucleic acid molecule. A nucleic acid molecule as described above can be cloned into any suitable vector and can be used to transform or transfect any suitable host. The selection of vectors and methods to construct them are commonly known to persons of ordinary skill in the art and are described in general technical references (see, in general, "Recombinant DNA Part D," Methods in Enzymology, Vol. 153, Wu and Grossman, eds., Academic Press (1987)). Desirably, the vector comprises regulatory sequences, such as transcription and translation initiation and termination codons, which are specific to the type of host (e.g., bacterium, fungus, or plant) into which the vector is to be introduced, as appropriate and taking into consideration whether the vector is DNA or RNA.<br>
Constructs of vectors, which are circular or linear, can be prepared to contain an entire nucleic acid sequence as described above or a portion thereof ligated to a replication system functional in a prokaryotic or eukaryotic host cell. Replication systems can be<br>
derived from ColEl, 2 mjj, plasmid, X phage, fl filamentous phage, Agrobacterium species f.g., Ag. tumefaciens and Ag. rhizogenes), and the like.<br>
In addition to the replication system and the inserted nucleic acid, the construct can include one or more marker genes, which allow for selection of transformed or transfected hosts. Marker genes include biocide resistance, e.g., resistance to antibiotics, heavy metals, herbicides; etc., complementation in an auxotrophic host to provide prototrophy, and the like.<br>
Suitable vectors include those designed for propagation or expression or both. A preferred cloning vector is selected from the group consisting of the pUC series, the pBluescript series (Stratagene, LaJolla, CA), the pET series (Novagen, Madison, WI), the pGEX series (Pharmacia Biotech, Uppsala, Sweden), and the pEX series (Clonetech, Palo Alto, CA). Bacteriophage vectors, such as XGT10, A.GT11, XZapII (Stratagene), A, EMBL4, and X NM1149, also can be used. Examples of plant expression vectors include pBHOl, pBI101.2, pBI101.3, pBI121, and pBIN19 (Clonetech, Palo Alto, CA). Examples of yeast expression vectors include pYES2.1, pYES2, and other pYES derivatives (Invitrogen, Carlsbad, CA).<br>
A plant expression vector can comprise a native or nonnative promoter operably linked to an above-described nucleic acid molecule. The selection of promoters, e.g., strong, weak, inducible, tissue-specific (i.e., specifically or preferentially expressed in a tissue), organ-specific (i.e., specifically or preferentially expressed in an organ) and developmental-specific (i.e., specifically or preferentially expressed during a particular stage(s) of development), is within the skill in the art. Similarly, the combining of a nucleic acid molecule as described above with a promoter is also within the skill in the art (see, e.g., Sambrook et al, Molecular Cloning: A Laboratory Manual, 2nd ed., Cold Spring Harbor Laboratory, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY (1989)).<br>
If it is desired to up-regulate the expression of a CDK inhibitor-like gene, i.e., to increase the expression of a CDK inhibitor-like gene by any means, such as by about twofold, ten-fold, 100-fold, or more at the mRNA level or protein level, it is preferred to do so by introducing a gene encoding a CDK inhibitor-like polypeptide as provided herein. The gene is preferably introduced by way of a vector. It is preferred that a lot of inhibitor is expressed in the tissue of interest. Multiple extra copies of the gene can be introduced into the plant cell, plant tissue, plant organ, or plant or a vector comprising a strong promoter, such as promoter, which is preferentially or specifically expressed in male reproductive tissue, is introduced into the plant cell, plant tissue, plant organ, or plant such that the gene is<br>
expressed at a higher rate, thereby generating more mRNA, which, in turn, is translated into<br>
t ""<br>
IfcRre of the encoded protein.<br>
As used herein "suppression" or "suppressing" means any of the well-known methods for suppressing a transcript or a protein from a gene including post-transcriptional gene suppression and transcriptional suppression. Post-transcriptional gene suppression is mediated by transcribed RNA having homology to a gene targeted for suppression. The RNA transcribed from the suppressing transgene preferably has a double stranded component to effect what is called RNA interference (RNAi). Transcriptional suppression is mediated by a transcribed double-stranded RNA having homology to promoter DNA sequence to effect what is called promoter /ra/«-suppression.<br>
More particularly, post-transcriptional gene suppression by double-stranded RNA can result from plant transformation with anti-sense DNA constructs as disclosed by Shewmaker et al. (U.S. Patents 5,107,065 and 5,759,829), from plant transformation with a sense-oriented DNA construct as disclosed by Jorgensen et al. (U.S. Patents 5,283,184 and 5,231,020), or with an RNAi construct as disclosed by Redenbaugh et al. (Safety Assessment of Genetically Engineered Flavr Savr™ Tomato, CRC Press, Inc., 1992), by Goldbach et al. (EP 0426195 Al,  1991), by Sijen et al. (The Plant Cell, 8:2277-2294, 1996), by Waterhouse et al.<br>
(WO 99/53050), by Graham et al. (WO 99/49029), by Lowe et al. (U.S. Application Publication No. 2003/0175965 Al), by Fillatti (U.S. Patent Application No. 10/465,800), by Plaetinck et al. (U.S. Application Publication No. 2003/0061626 Al), by Liu et al. (U.S. Patent 6,326,193), by Agrawal et al. (WO 94/01550), by Werner et al. (WO 98/05770), by Oeller (U.S. Application Publication No. 2002/0048814 Al), by Gutterson et al. (U.S. Application Publication No. 2003/0018993 Al), and by Glassman et al. (U.S. Application Publication No. 2003/0036197 Al). All of the above-described patents, applications, and international publications disclosing materials and methods for post-transcriptional gene suppression in plants are incorporated herein by reference.<br>
Transcriptional suppression such as promoter /raws-suppression can be effected by expressing a DNA construct comprising a promoter operably linked to inverted repeats of promoter DNA for a target gene. Constructs useful for such gene suppression mediated by promoter /ra/«-suppression are disclosed by Mette et al., The EMBO Journal, 18(1):241-148, 1999 and Mette et al., The EMBO Journal, 19(19):5194-5201, 2000, both of which are incorporated herein by reference.<br>
A preferred method of post-transcriptional gene suppression in plants employs either b"kse-oriented or antisense-oriented, transcribed RNA which is stabilized, e.g., with a terminal hairpin structure. A preferred DNA construct for effecting post-transcriptional gene suppression is transcribed to a segment of antisense oriented RNA having homology to a gene targeted for suppression, where the antisense RNA segment is followed at the 3' end by a contiguous, complementary, shorter segment of RNA in the sense orientation. The use of self-stabilized antisense RNA oligonucleotides in plants is disclosed in WO 94/01550 (Agrawal et al.). See also WO 98/05770 (Werner et al.} where the antisense RNA is stabilized by hairpin forming repeats of poly(CG) nucleotides. See also U.S. Application Publication No. 2002/0048814 Al (Oeller) where sense or antisense RNA is stabilized by a poly(T)-poly(A) tail. See also U.S. Application Publication No. 2003/0018993 Al (Gutterson et al.) where sense or antisense RNA is stabilized by an inverted repeat of a subsequence of an NOS gene. See also U.S. Application Publication No. 2003/0036197 Al (Glassman et al.) where RNA having homology to a target is stabilized by 2 complementary RNA regions. All of the above-described patents, applications, and international publications disclosing materials and methods for employing stabilized RNA and its use in gene suppression in plants are incorporated herein by reference.<br>
Transcriptional suppression such as promoter fr-a/w-suppression can be effected by expressing a DNA construct comprising a promoter operably linked to inverted repeats of promoter DNA for a target gene. Constructs useful for such gene suppression mediated by promoter rra/w-suppression are disclosed by Mette et al., The EMBO Journal, 18(1 ):241 -148, 1999 and Mette et al., The EMBO Journal, 19(19):5194-5201, 2000, both of which are incorporated herein by reference.<br>
Still yet another method is the use of a dominant negative mutant. For example, a dominant negative mutant of a polypeptide having CDK inhibitor-like activity as described herein can be generated by completely or partially deleting the C-terminal coding sequence, in particular all or part of the C-terminal coding sequence that is highly conserved among the polypeptides described herein. The resulting mutant can be operably linked to a promoter, such as an embryo-specific promoter from maize, for example, and cloned into a vector for introduction into a corn plant or part thereof. See, e.g., Jasinski et al., Plant Physiol, 130:1871-1882(2002)).<br>
Ribozymes also have been reported to have use as a means to inhibit expression of endogenous plant genes (see, e.g., Merlo et al., Plant Cell, 10(10): 1603-1622 (1998)). It is possible to design ribozymes that specifically pair with virtually any target RNA and cleave<br>
the phosphodiester backbone at a specific location, thereby functionally inactivating the target RNA. In carrying out this cleavage, the ribozyme is not itself altered and is, thus, capable of recycling and cleaving other molecules, making it a true enzyme. The inclusion of ribozyme sequences within antisense RNAs confers RNA-cleaving activity upon them, thereby increasing the activity of the constructs. The design and use of target RNA-specific ribozymes is described in Haseloff et al, Nature, 334:585-591 (1988). Preferably, the ribozyme comprises at least about 20 continuous nucleotides complementary to the target sequence on each side of the active site of the ribozyme.<br>
Alternatively, reverse genetics systems, which are well-known in the art, can be used to generate and isolate suppressed or null mutants. One such system, the Trait Utility System for Corn, i.e., TUSC, is based on successful systems from other organisms (Ballinger et al., PNAS U.S.A., 86:9402-9406 (1989); Kaiser et al., PNAS U.S.A. 87:1686-1690 (1990); and Rushforth et al., Mol. Cell. Biol, 13:902-910 (1993)). The central feature of the system is to identify MM transposon insertions within a DNA sequence of interest in anticipation that at least some of these insertion alleles will be mutants. To develop the system in corn, DNA was collected from a large population of Mutator transposon stocks that were then self-pollinated to produce F2 seed. To find MM transposon insertions within a specified DNA sequence, the collection of DNA samples is screened via PCR using a gene-specific primer and a primer that anneals to the inverted repeats of MM transposons. A PCR product is expected only when the template DNA comes from a plant that contains a MM transposon insertion within the target gene. Once such a DNA sample is identified, F2 seed from the corresponding plant is screened for a transposon insertion allele. Transposon insertion mutations of the anl gene have been obtained via the TUSC procedure (Bensen et al., Plant Cell, 7:75-84 (1995)). This system is applicable to other plant species, at times modified as necessary in accordance with knowledge and skill in the art.<br>
T-DNA insertional mutagenesis can be used to generate insertional mutations in one of the above-mentioned genes so as to affect adversely the expression of a given gene. T-DNA tagged lines of plants can be screened using PCR. For example, a primer can be designed for one end of the T-DNA and another primer can be designed for the gene of interest and both primers can be used in PCR. If no PCR product is obtained, then there is no insertion in the gene of interest. In contrast, if a PCR product is obtained, then there is an insertion in the gene of interest. Insertional mutations, however, often generate null alleles, which can be lethal. Alternatively, if there is more than one gene that encodes for a given<br><br>
enzyme, a mutation in one of the genes may not result in decreased expression of the IKzyme encoded by the gene.<br>
Another alternative method to decrease expression of a given gene is to use a compound that inhibits expression of one of the above-mentioned genes or that inhibits the activity of the protein encoded by one of the above-mentioned genes. In this regard, x-ray or gamma radiation can be used as can chemical mutagens, such as ethyl methyl sulfonate (EMS) or dimethyl butyric acid (DMB).<br>
In addition to the above, gene replacement technology can be used to increase or decrease expression of a given gene. Gene replacement technology is based upon homologous recombination (see, Schnable etal, Curr. Opinions Plant Biol, 1:123 (1998)). The nucleic acid of the enzyme of interest can be manipulated by mutagenesis (e.g., insertions, deletions, duplications, or replacements) to either increase or decrease enzymatic function. The altered sequence can be introduced into the genome to replace the existing, e.g., wild-type, gene via homologous recombination (Puchta and Hohn, Trends Plant Sci., 1:340 (1996); Kempin et ai, Nature, 389:802 (1997)).<br>
The activity of a given CDK inhibitor-like polypeptide can be measured in vitro. For example, the ability of a given CDK inhibitor-like polypeptide to prevent phosphorylation of HI histones by CDK can be measured (see, e.g., Example 14).<br>
Whether or not a given CDK inhibitor-like polypeptide affects the expression of other genes can be determined using a transgenic plant, for example. As microarray chip technology becomes established and available (DeRisi et al, Science, 278:680-686 (1997)), the effect of a given genetic alteration on the expression of all identified genes of maize can be determined using microarray chips. Metabolic radiotracer studies can be performed to measure the generation of different product pools in vivo. In such studies, radioactively labeled precursors are provided to intact tissues and the radioactive label is monitored as the precursor is metabolized. By comparing wild-type plants and plants that have reduced activities of one of the CDK inhibitor-like polypeptide genes, the effect of the reduction in a given CDK inhibitor-like polypeptide can be determined.<br>
In view of the above, the present invention provides a host cell comprising an above-described isolated or purified nucleic acid molecule, optionally in the form of a vector. Suitable hosts include bacteria, yeast and plant cells, including E. coli, B. subtilis, A. tumefaciens, S. cerevisiae, and N. crassa. E. coli hosts include TB-1, TG-2, DH5ot, XL-Blue MRF' (Stratagene, Austin, TX), SA2821, Y1090, and TG02. Plant cells include maize cells.<br>
Also in view of the above, the present invention provides an isolated or purified |f olypeptide encoded by an above-described isolated or purified nucleic acid molecule. The polypeptide preferably comprises an amino end and a carboxyl end. The polypeptide can comprise D-amino acids, L-amino acids, or a mixture of D- and L-amino acids. The D-form of the amino acids, however, is particularly preferred, since a protein comprised of D-amino acids is expected to have a greater retention of its biological activity in vivo, given that the D-amino acids are not recognized by naturally occurring proteases.<br>
Alterations of the native amino acid sequence to produce variant polypeptides can be done by a variety of means known to those ordinarily skilled in the art. For instance, amino acid substitutions can be conveniently introduced into the polypeptides at the time of synthesis. Alternatively, site-specific mutations can be introduced by ligating into an expression vector a synthesized oligonucleotide comprising the modified site. Alternately, oligonucleotide-directed, site-specific mutagenesis procedures can be used, such as disclosed in Walder et al, Gene, 42:133 (1986); Bauer et al, Gene, 37:73 (1985); and U.S. Patents 4,518,584 and 4,737,462.<br>
It is within the skill of the ordinary artisan to select synthetic and naturally-occurring amino acids that effect conservative or neutral substitutions for any particular naturally-occurring amino acids. The ordinarily skilled artisan desirably will consider the context in which any particular amino acid substitution is made, in addition to considering the hydrophobicity or polarity of the side-chain, the general size of the side chain and the pK value of side-chains with acidic or basic character under physiological conditions. For example, lysine, arginine, and histidine are often suitably substituted for each other, and more often arginine and histidine. As is known in the art, this is because all three amino acids have basic side chains, whereas the pK value for the side-chains of lysine and arginine are much closer to each other (about 10 and 12) than to histidine (about 6). Similarly, glycine, alanine, valine, leucine, and isoleucine are often suitably substituted for each other, with the proviso that glycine is frequently not suitably substituted for the other members of the group. This is because each of these amino acids is relatively hydrophobic when incorporated into a polypeptide, but glycine's lack of an a-carbon allows the phi and psi angles of rotation (around the a-carbon) so much conformational freedom that glycinyl residues can trigger changes in conformation or secondary structure that do not often occur when the other amino acids are substituted for each other. Other groups of amino acids frequently suitably substituted for each other include, but are not limited to, the group consisting of glutamic and aspartic acids; the group consisting of phenylalanine, tyrosine,<br>
and tryptophan; and the group consisting of serine, threonine, and optionally, tyrosine. Additionally, the ordinarily skilled artisan can readily group synthetic amino acids with naturally-occurring amino acids.<br>
If desired, the polypeptides can be modified, for instance, by glycosylation, amidation, carboxylation, or phosphorylation, or by the creation of acid addition salts, amides, esters, in particular C-terminal esters, and N-acyl derivatives of the polypeptides of the invention. The polypeptides also can be modified to create protein derivatives by forming covalent or noncovalent complexes with other moieties in accordance with methods known in the art. Covalently-bound complexes can be prepared by linking the chemical moieties to functional groups on the side chains of amino acids comprising the polypeptides, or at the N- or C-terminus. Desirably, such modifications and conjugations do not adversely affect the activity of the polypeptides (and variants thereof). While such modifications and conjugations can have greater or lesser activity, the activity desirably is not negated and is characteristic of the unaltered polypeptide.<br>
The polypeptides (and fragments, variants, and fusion proteins) can be prepared by any of a number of conventional techniques. The polypeptide can be isolated or purified from a naturally occurring source or from a recombinant source. For instance, in the case of recombinant proteins, a DNA fragment encoding a desired protein can be subcloned into an appropriate vector using well-known molecular genetic techniques (see, e.g., Maniatis et al., Molecular Cloning: A Laboratory Manual. 2nd ed. (Cold Spring Harbor Laboratory, 1989) and other references cited herein under "EXAMPLES"). The fragment can be transcribed and the protein subsequently translated in vitro. Commercially available kits also can be employed (e.g., such as manufactured byClontech, Palo Alto, CA; Amersham Life Sciences, Inc., Arlington Heights, IL; InVitrogen, San Diego, CA, and the like). The polymerase chain reaction optionally can be employed in the manipulation of nucleic acids.<br>
Such polypeptides also can be synthesized using an automated peptide synthesizer in accordance with methods known in the art. Alternately, the polypeptide (and fragments, variants, and fusion proteins) can be synthesized using standard peptide synthesizing techniques well-known to those of ordinary skill in the art (e.g., as summarized in Bodanszky, Principles of Peptide Synthesis. (Springer-Verlag, Heidelberg: 1984)). In particular, the polypeptide can be synthesized using the procedure of solid-phase synthesis (see, e.g., Merrifield, J. Am. Chem. Soc., 85:2149-54 (1963); Barany et al, Int. J. Peptide Protein Res., 30:705-739 (1987); and U.S. Patent 5,424,398). If desired, this can be done using an automated peptide synthesizer. Removal of the t-butyloxycarbonyl (t-BOC) or<br>
9-fluorenylmethyloxycarbonyl (Fmoc) amino acid blocking groups and separation of the protein from the resin can be accomplished by, for example, acid treatment at reduced temperature. The polypeptide-containing mixture then can be extracted, for instance, with diethyl ether, to remove non-peptidic organic compounds, and the synthesized protein can be extracted from the resin powder (e.g., with about 25% w/v acetic acid). Following the synthesis of the polypeptide, further purification (e.g., using HPLC) optionally can be done in order to eliminate any incomplete proteins, polypeptides, peptides, or free amino acids. Amino acid and/or HPLC analysis can be performed on the synthesized polypeptide to validate its identity. For other applications according to the invention, it may be preferable to produce the polypeptide as part of a larger fusion protein, either by chemical conjugation, or through genetic means, such as are known to those ordinarily skilled in the art. In this regard, the present invention also provides a fusion protein comprising the isolated or purified polypeptide (or fragment thereof) or variant thereof and one or more other polypeptides/protein(s) having any desired properties or effector functions.<br>
The nucleic acid molecules, vectors, and polypeptides can be used in agricultural methods and various screening assays (see, e.g., WO 02/28893 and U.S. Patent 6,215,048). For example, a nucleic acid molecule can be used to express, e.g., via a vector, a CDK inhibitor-like polypeptide in a host cell, to detect mRNA encoding a CDK inhibitor-like polypeptide in a biological sample, to detect a genetic alteration in a gene encoding a CDK inhibitor-like polypeptide, such as via a Southern blot, to suppress a CDK inhibitor-like polypeptide, or to up-regulate a CDK inhibitor-like polypeptide so as to render a plant male-sterile or a dwarf, or to inhibit the formation of certain organs, for example. The polypeptides can be used to compensate for deficiencies in CDK inhibitor-like polypeptides or for the presence of mutated CDK inhibitor-like polypeptides having reduced or no activity in a maize plant, or to treat excessive levels of substrates, whether direct or indirect, for CDK inhibitor-like polypeptides in a maize plant. Alternatively, the polypeptides can be used to screen agents for the ability to modulate their activity.<br>
In view of the above, the present invention provides various nucleic acid constructs, which can be used to suppress a CDK inhibitor-like polypeptide. In one embodiment, the nucleic acid construct comprises at least one transcribable nucleotide sequence, the expression of which results in the suppression of an endogenous nucleotide sequence selected from the group consisting of:<br>
SEQ ID NO: 1 [ZmKRPl], a nucleotide sequence encoding the amino acid sequence pf SEQ ID NO: 2 [ZmKRPl], SEQ ID NO: 3 [ZmKRP2], and a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 4 [ZmKRP2],<br>
alone or in further combination with at least one other transcribable nucleotide sequence, the expression of which results in the suppression of an endogenous nucleotide sequence selected from the group consisting of SEQ ID NO: 5 [ZmKRPS], a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 6 [ZmKRP3], SEQ ID NO: 7 [ZmKRP4], a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 8 [ZmKRP4], SEQ ID NO: 9 [ZmKRPS], a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 10 [ZmKRPS], SEQ ID NO: 11 [ZmKRP6], a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 12 [ZmKRP6], SEQ ID NO: 13 [ZmKRP7], a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 14 [ZmKRPV], SEQ ID NO: 15 [ZmKRPS], a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 16 [ZmKRPS], SEQ ID NO: 17 [ZmKRP9], and a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 18 [ZmKRP9],<br>
wherein, when the nucleic acid construct comprises two or more transcribable nucleotide sequences, the nucleotide sequences can be present in any order on the nucleic acid construct and can be polycistronic, wherein each transcribable nucleotide sequence comprises at least about 100 (or 125, 150, 175, 200, 225, 250, 275, 300, 325, 375, 400, 425, 450, 500, 750, 1,000, 1,500, 2,000, or more) contiguous nucleotides, and wherein the expression of each transcribable nucleotide sequence optionally induces gene suppression. By "optionally" is meant that each transcribable nucleotide sequence, independent of any and all other nucleotide sequences present on the construct, either induces gene suppression or does not induce gene suppression.<br>
Alternatively, the nucleic acid construct comprises transcribable nucleotide sequences, the expression of which results in the suppression of endogenous nucleotide sequences:<br>
(i) SEQ ID NO: 1 [ZmKRPl] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 2 [ZmKRPl],<br>
(ii) SEQ ID NO: 3 [ZmKRP2] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 4 [ZmKRP2],<br>
(iii) SEQ ID NO: 5 [ZmKRP3] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 6 [ZmKRPS],<br>
(iv) SEQ ID NO: 7 [ZmKRP4] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 8 [ZmKRP4],<br>
(v) SEQ ID NO: 9 [ZmKRPS] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 10 [ZmKRPS],<br>
(vi) SEQ ID NO: 11 [ZmKRP6] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 12 [ZmKRP6],<br>
(vii) SEQ ID NO: 13 [ZmKRP7] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 14 [ZmKRP7],<br>
(viii) SEQ ID NO: 15 [ZmKRPS] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 16 [ZmKRPS], and<br>
(ix) SEQ ID NO: 17 [ZmKRP9] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 18 [ZmKRP9].<br>
In another embodiment, the nucleic acid construct comprises at least one transcribable nucleotide sequence, the expression of which results in the suppression of an endogenous nucleotide sequence of SEQ ID NO: 7 [ZmKRP4] and/or SEQ ID NO: 17 [ZmKRP9], either one or both in further combination with at least one other transcribable nucleotide sequence, the expression of which results in the suppression of an endogenous nucleotide sequence selected from the group consisting of:<br>
(i) SEQ ID NO: 1 [ZmKRPl] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 2 [ZmKRP2],<br>
(ii) SEQ ID NO: 3 [ZmKRP2] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 4 [ZmKRP2],<br>
(iii) SEQ ID NO: 5 [ZmKRPS] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 6 [ZmKRP3], provided that the transcribable nucleotide sequence does not entirely correspond to nucleotides 175-342 of SEQ ID NO: 5,<br>
(iv) SEQ ID NO: 9 [ZmKRPS] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 10 [ZmKRPS], provided that the transcribable nucleotide sequence does not entirely correspond to nucleotides 450-570 of SEQ ID NO: 9,<br>
(v) SEQ ID NO: 11 [ZmKRP6] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 12 [ZmKRP6], provided that the transcribable nucleotide sequence does not entirely correspond to nucleotides 380-765 of SEQ ID NO: 11,<br>
(vi) SEQ ID NO: 13 [ZmKRP7] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 14 [ZmKRP7], provided that the transcribable nucleotide sequence does not entirely correspond to nucleotides 335-718 of SEQ ID NO: 13, and<br>
(vii) SEQ ID NO: 15 [ZmKRPS] or a nucleotide sequence encoding the amino acid -equence of SEQ ID NO: 16 [ZmKRPS], provided that the transcribable nucleotide sequence does not entirely correspond to nucleotides 350-405 or 420-698 of SEQ ID NO: 15,<br>
wherein the transcribable nucleotide sequences can be present in any order on the nucleic acid construct and can be polycistronic, wherein each transcribable nucleotide sequence comprises at least about 100 (or 125, 150, 175, 200, 225, 250, 275, 300, 325, 350, 375, 400, 425, 450, 475, 500, 750, 1,000, 1,500, 2,000, or more) contiguous nucleotides, and wherein the expression of each transcribable nucleotide sequence optionally induces gene suppression. As used herein, gene suppression refers to the reduction of gene expression through various techniques, including, but not limited to, sense suppression, antisense, dsRNA, and ribozyme technologies.<br>
In yet another embodiment, the nucleic acid construct comprises at least one transcribable nucleotide sequence, the expression of which results in the suppression of an endogenous nucleotide sequence selected from the group consisting of:<br>
(i) SEQ ID NO: 1 [ZmKRPl] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 2 [ZmKRPl],<br>
(ii) SEQ ID NO: 3 [ZmKRP2] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 4 [ZmKRP2],<br>
(iii) SEQ ID NO: 5 [ZmKRP3] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 6 [ZmKRPS], provided that the transcribable nucleotide sequence does not entirely correspond to nucleotides 175-342 of SEQ ID NO: 5,<br>
(iv) SEQ ID NO: 9 [ZmKRP5] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 10 [ZmKRP5], provided that the transcribable nucleotide sequence does not entirely correspond to nucleotides 450-570 of SEQ ID NO: 9,<br>
(v) SEQ ID NO: 11 [ZmKRP6] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 12 [ZmKRP6], provided that the transcribable nucleotide sequence of SEQ ID NO: 11 does not entirely correspond to nucleotides 380-765 of SEQ ID NO: 11,<br>
(vi) SEQ ID NO: 13 [ZmKRP7] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 14 [ZmKRP7], provided that the transcribable nucleotide sequence does not entirely correspond to nucleotides 335-718 of SEQ ID NO: 13,<br>
(vii) SEQ ID NO: 15 [ZmKRPS] or a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 16 [ZmKRPS], provided that the transcribable nucleotide sequence does not entirely correspond to nucleotides 350-405 or 420-698 of SEQ ID NO: 15, and<br>
(viii) SEQ ID NO: 17 [ZmKRP9] or a nucleotide sequence encoding the amino acid -equence of SEQ ID NO: 18 [ZmKRP9], provided that the transcribable nucleotide sequence does not entirely correspond to nucleotides 1-183 of SEQ ID NO: 17,<br>
wherein the transcribable nucleotide sequences can be present in any order on the nucleic acid construct and can be polycistronic, wherein each transcribable nucleotide sequence comprises at least about 100 (or 125, 150, 175, 200, 225, 250, 275, 300, 325, 350, 375, 400, 425, 450, 500, 750, 1,000, 1,500, 2,000, or more) contiguous nucleotides, and wherein the expression of each transcribable nucleotide sequence optionally induces gene silencing. By "does not entirely correspond" means that at least one, preferably at least about three nucleotides correspond to nucleotides outside of the designated ranges of nucleotides.<br>
With respect to the above nucleic acid constructs, that at least one transcribable nucleotide sequence, the expression of which results in suppression, can be from a noncoding sequence. For example, the noncoding sequence can be an intron, a nucleotide sequence from a promoter region, a nucleotide sequence from a 5' untranslated region, a nucleotide sequence from a 3' untranslated region, or a fragment of any of the foregoing. Examples of such sequences include those of SEQ ID NOS: 44-58.<br>
The above constructs are exemplary and are not intended to be limiting. In the suppression constructs, for example, ZmKRPl or ZmKRP2 can be inhibited alone or together. In this regard, ZmKRPl or ZmKRP2 can be inhibited alone or together, in further combination with any one of ZmKRP3, ZmKRP4, ZmKRPS, ZmKRPG, ZmKRP7, ZmKRPS, and ZmKRP9. For the sake of convenience, the abbreviation "ZmKRP" has been deleted from the following lists of combinations. For example, 1, 2 and 3; 1, 2 and 4; 1, 2 and 5; 1, 2 and 6; 1, 2 and 7; 1, 2 and 8; 1, 2 and 9; 1 and 3; 1 and 4; 1 and 5; 1 and 6; 1 and 7; 1 and 8; 1 and 9; 2 and 3; 2 and 4; 2 and 5; 2 and 6; 2 and 7; 2 and 8; 2 and 9; 1, 3 and 4; 1, 3 and 5; 1, 3 and 6; 1, 3 and 7; 1, 3 and 8; 1, 3 and 9; 2, 3 and 4; 2, 3 and 5; 2, 3 and 6; 2, 3 and 7; 2, 3 and 8; 2, 3 and 9; 1, 4 and 5; 1, 4 and 6; 1, 4 and 7; 1, 4 and 8; 1, 4 and 9; 1, 5 and 6; 1, 5 and 7; 1, 5 and 8; 1, 5 and 9; 2, 4 and 5; 2, 4 and 6; 2, 4 and 7; 2, 4 and 8; 2, 4 and 9; 1, 5 and 6; 1, 5 and 7; 1, 5 and 8; 1, 5 and 9; 2, 5 and 6; 2, 5 and 7; 2, 5 and 8; 2, 5 and 9; 1, 6 and 7; 1, 6 and 8; 1, 6 and 9; 2, 6 and 7; 2, 6 and 8; 2, 6 and 9; 1, 7 and 8; 1, 7 and 9; 2, 7 and 8; 2, 7 and 9; 1, 8 and 9; 2, 8 and 9; 1, 3, 4, and 5; 1, 3, 4, and 6; 1, 3, 4, and 7; 1, 3, 4, and 8; 1, 3, 4, and 9; 2, 3, 4, and 5; 2, 3, 4, and 6; 2, 3, 4, and 7; 2, 3, 4, and 8; 2, 3, 4, and 9; 1, 4, 5, and 6; 1, 4, 5, and 7; 1, 4, 5, and 8; 1, 4, 5, and 9; 2, 4, 5, and 7; 2, 4, 5, and 8; 2, 4, 5, and 9; 1, 5, 6, and 7; 1, 5, 6, and 8; 1, 5, 6, and 9; 2, 5, 6, and 7; 2, 5, 6, and 8; 2, 5, 6, and 9;<br>
1,	6, 7, and 8; 1, 6, 7, and 9; 2, 6, 7, and 8; 2, 6, 7, and 9; 1, 7, 8, and 9; 2, 7, 8, and 9; 1, 3, 4,<br>
^ and 6; 1, 3, 4, 5, and 7; 1, 3, 4, 5, and 8; 1, 3, 4, 5, and 9; 2, 3, 4, 5, and 6; 2, 3, 4, 5, and 7;<br>
2,	3, 4, 5, and 8; 2, 3, 4, 5, and 9; 1, 3, 4, 5, 6, and 7; 1, 3, 4, 5, 6, and 8; 1, 3, 4, 5, 6, and 9;<br>
2, 3, 4, 5, 6, and 7; 2, 3, 4, 5, 6, and 8; 2, 3, 4, 5, 6, and 9; 1, 3, 4, 5, 6, 7, and 8; 1, 3, 4, 5, 6,<br><br>
7,	and 9; 2, 3, 4, 5, 6, 7, and 8; 2, 3, 4, 5, 6, 7, and 9; 1, 3, 4, 5, 6, 7, 8, and 9; 2, 3, 4, 5, 6, 7,<br>
8,	and 9; 1, 2, 3, and 4; 1, 2, 3, and 5; 1, 2, 3, and 6; 1, 2, 3, and 7; 1, 2, 3, and 8; 1, 2, 3, and<br>
9; 1, 2, 4, and 5; 1, 2, 4, and 6; 1, 2, 4, and 7; 1, 2, 4, and 8; 1, 2, 4, and 9; 1, 2, 5, and 6; 1, 2,<br>
5, and 7; 1, 2, 5, and 8; 1, 2, 5, and 9; 1, 2, 6, and 7; 1, 2, 6, and 8; 1, 2, 6, and 9; 1, 2, 7, and<br>
8; 1, 2, 7, and 9; 1, 2, 8, and 9; 1, 2, 3, 4, and 5; 1, 2, 3, 4, and 6; 1, 2, 3, 4, and 7; 1, 2, 3, 4,<br>
and 8; 1, 2, 3, 4, and 9; 1, 2, 4, 5, and 6; 1, 2, 4, 5, and 7; 1, 2, 4, 5, and 8; 1, 2, 4, 5, and 9; 1,<br>
2, 5, 6, and 7; 1, 2, 5, 6, and 8; 1, 2, 5, 6, and 9; 1,2, 6, 7, and 8; 1, 2, 6, 7, and 9; 1, 2, 7, 8,<br>
and 9; 1, 2, 3, 4, 5, and 6; 1, 2, 3, 4, 5, and 7; 1, 2, 3, 4, 5, and 8; 1, 2, 3, 4, 5, and 9; 1, 2, 3,<br><br>
4,	5, 6, and 7; 1, 2, 3, 4, 5, 6, and 8; 1, 2, 3, 4, 5, 6, and 9; 1, 2, 3, 4, 5, 6, 7, and 8; 1, 2, 3, 4,<br>
5,	6, 7, and 9; or 1, 2, 3, 4, 5, 6, 7, 8, and 9 can be inhibited. Furthermore, the constructs can<br>
comprise the nucleotide sequences in any order. The nucleotide sequences can also vary in<br>
length.<br>
In the suppression constructs, ZmKRP4 or ZmKRP9 can be inhibited alone or together. In this regard, ZmKRP4 or ZmKRP9 can be inhibited alone or together, in further combination with any one of ZmKRPl, ZmKRP2, ZmKRP3, ZmKRPS, ZmKRP6, ZmKRP7, and ZmKRPS. For example, 4 and 1; 4 and 2; 4 and 3; 4 and 5; 4 and 6; 4 and 7; 4 and 8; 4, 1 and 2; 4, 1 and 3; 4, 1 and 5; 4, 1 and 6; 4, 1 and 7; 4, 1 and 8; 4, 1, 2 and 3; 4,<br>
1,	2, and 5; 4, 1, 2, and 6; 4, 1, 2, and 7; 4, 1, 2, and 8; 4, 1, 2, 3, and 5; 4, 1, 2, 3, and 6; 4, 1,<br>
2,	3, and 7; 4, 1, 2, 3, and 8; 4, 1, 2, 3, 5, and 6; 4, 1, 2, 3, 5, and 7; 4, 1, 2, 3, 5, and 8; 4, 1,<br>
2, 3, 5, 6, and 7; 4, 1, 2, 3, 5, 6, and 8; 4, 1, 2, 3, 5, 6, 7, and 8; 4, 2 and 3; 4, 2 and 5; 4, 2<br>
and 6; 4, 2 and 7; 4, 2 and 8; 4, 3 and 5; 4, 3 and 6; 4, 3 and 7; 4, 3 and 8; 4, 5 and 6; 4, 5 and<br>
7; 4, 5 and 8; 4, 6 and 7; 4, 6 and 8; 4, 7 and 8; 4, 2, 3, and 5; 4, 2, 3, and 6; 4, 2, 3, and 7; 4,<br>
2, 3, and 8; 4, 3, 5, and 6; 4, 3, 5, and 7; 4, 3, 5, and 8; 4, 3, 6, and 7; 4, 3, 6, and 8; 4, 5, 6,<br>
and 7; 4, 5, 6, and 8; 4, 6, 7, and 8; 4, 2, 3, 5, and 6; 4, 2, 3, 5, and 7; 4, 2, 3, 5, and 8; 4, 2, 3,<br>
5, 6, and 7; 4, 2, 3, 5, 6, and 8; 4, 2, 3, 5, 6, 7, and 8; 9 and 1; 9 and 2; 9 and 3; 9 and 5; 9 and<br>
6; 9 and 7; 9 and 8; 9, 1 and 2; 9, 1 and 3; 9, 1 and 5; 9, 1 and 6; 9, 1 and 7; 9, 1 and 8; 9, 1,<br>
2, and 3; 9, 1, 2, and 5; 9, 1, 2, and 6; 9, 1, 2, and 7; 9, 1, 2, and 8; 9, 1, 2, 3, and 5; 9, 1, 2, 3,<br>
and 6; 9, 1, 2, 3, and 7; 9, 1, 2, 3, and 8; 9, 1, 2, 3, 5, and 6; 9, 1, 2, 3, 5, and 7; 9, 1, 2, 3, 5,<br>
and 8; 9, 1, 2, 3, 5, 6, and 7; 9, 1, 2, 3, 5, 6, and 8; 9, 1, 2, 3, 5, 6, 7, and 8; 9, 2 and 3; 9, 2<br>
and 5; 9, 2 and 6; 9, 2 and 7; 9, 2 and 8; 9, 3 and 5; 9, 3 and 6; 9, 3 and 7; 9, 3 and 8; 9, 5 and<br>
6; 9, 5, and 7; 9, 5, and 8; 9, 6 and 7; 9, 6 and 8; 9, 7 and 8; 9, 2, 3, and 5; 9, 2, 3, and 6; 9, 2, h and 7; 9, 2, 3, and 8; 9, 3, 5, and 6; 9, 3, 5, and 7; 9, 3, 5, and 8; 9, 3, 6, and 7; 9, 3, 6, and 8; 9, 5, 6, and 7; 9, 5, 6, and 8; 9, 6, 7, and 8; 9, 2, 3, 5, and 6; 9, 2, 3, 5, and 7; 9, 2, 3, 5, and 8; 9, 2, 3, 5, 6, and 7; 9, 2, 3, 5, 6, and 8; 9, 2, 3, 5, 6, 7, and 8; 4, 9 and 1; 4, 9 and 2; 4, 9 and 3; 4, 9 and 5; 4, 9 and 6; 4, 9 and 7; 4, 9 and 8; 4, 9, 1, and 2; 4, 9, 1, and 3; 4, 9, 1, and 5; 4, 9, 1, and 6; 4, 9, 1, and 7; 4, 9, 1, and 8; 4, 9,1, 2, and 3; 4, 9, 1, 2, and 5; 4, 9, 1, 2, and 6; 4, 9, 1, 2, and 7; 4, 9, 1, 2, and 8; 4, 9, 1, 2, 3, and 5; 4, 9, 1, 2, 3, and 6; 4, 9, 1, 2, 3, and 7; 4, 9, 1, 2, 3, and 8; 4, 9, 1, 2, 3, 5, and 6; 4, 9, 1, 2, 3, 5, and 7; 4, 9, 1, 2, 3, 5, and 8; 4, 9, 1, 2, 3, 5, 6, and 7; 4, 9, 1, 2, 3, 5, 6, and 8; 4, 9, 1, 2, 3, 5, 6, 7, and 8; 4, 9, 2, and 3; 4, 9, 2, and 5; 4, 9, 2, and 6; 4, 9, 2, and 7; 4, 9, 2, and 8; 4, 9, 3, and 5; 4, 9, 3, and 6; 4, 9, 3, and 7;<br>
4,	9, 3, and 8; 4, 9, 5, and 6; 4, 9, 5, and 7; 4, 9, 5, and 8; 4, 9, 6, and 7; 4, 9, 6, and 8; 4, 9, 7,<br>
and 8; 4, 9, 2, 3, and 5; 4, 9, 2, 3, and 6; 4, 9, 2, 3, and 7; 4, 9, 2, 3, and 8; 4, 9, 3, 5, and 6; 4,<br>
9, 3, 5, and 7; 4, 9, 3, 5, and 8; 4, 9, 3, 6, and 7; 4, 9, 3, 6, and 8; 4, 9, 5, 6, and 7; 4, 9, 5, 6,<br>
and 8; 4, 9, 6, 7, and 8; 4, 9, 2, 3, 5, and 6; 4, 9, 2, 3, 5, and 7; 4, 9, 2, 3, 5, and 8; 4, 9, 2, 3,<br>
5,	6, and 7; 4, 9, 2, 3, 5, 6, and 8; or 4, 9, 2, 3, 5, 6, 7, and 8 can be inhibited. Furthermore,<br>
the constructs can comprise the nucleotide sequences in any order. The nucleotide<br>
sequences can also vary in length.<br>
Any of the above constructs can further comprise a promoter, which is preferentially or specifically expressed in the germ and/or aleurone of a kernel of maize and the promoter is operably linked to at least one nucleotide sequence. The promoter can be an oleosin promoter, such as L3, a globulin promoter, or a Perl promoter, for example. In the above suppression constructs, the at least one transcribable nucleotide sequence, the expression of which results in suppression, may be a noncoding sequence. The noncoding sequence can be an intron, a nucleotide sequence from a promoter region, a nucleotide sequence from a 5' untranslated region, a nucleotide sequence from a 3' untranslated region, or a fragment of any of the foregoing. Examples of such sequences include those of SEQ ID NOS: 44-58.<br>
The above constructs can be made in accordance with methods known in the art (see, in general, "Recombinant DNA Part D," Methods in Enzymology, Vol. 153, Wu and Grossman, eds., Academic Press (1987) and the references cited herein under "EXAMPLES"). Suitable methods of construction are exemplified in the Examples set forth herein.<br>
Techniques for contacting a plant cell, a plant tissue, a plant organ or a plant with a nucleic acid construct, such as a vector, so that the nucleic acid construct is taken up by a plant cell, alone or as part of a plant tissue, a plant organ, or a plant, and expressed therein<br>
are known in the art. Such methods involve plant tissue culture techniques, for example, jtierein, "contacting" is intended to mean that the cell, tissue, organ, or plant is brought into contact with the nucleic acid construct or vector in such a manner that the vector enters the cell and is expressed therein.<br>
The plant cell, plant tissue, plant organ, or plant can be contacted with the vector by any suitable means as known in the art. Preferably, a transgenic plant expressing the desired protein is to be produced. Various methods for the introduction of a desired polynucleotide sequence encoding the desired protein into plant cells include, but are not limited to: (1) physical methods such as microinjection (Capecchi, Cell, 22(2):479-488 (1980)), electroporation (Fromm etal, PNAS U.S.A., 82(17):5824-5828 (1985); U.S. Patent 5,384,253) and microprojectile mediated delivery (biolistics or gene gun technology) (Christou et al, Bio/Technology, 9:957 (1991); Fynan et al., PNAS U.S.A. 90(24):! 1478-11482 (1993)); (2) virus mediated delivery methods (Clapp, Clin. Perinatal., 20(1):155-168 (1993); Lu et al.,J. Exp. Med., 178(6):2089-2096 (1993); Eglitis and Anderson, Biotechniques, 6(7):608-614 (1988); and (3) Agrobacterium-mediated transformation methods; (Fraley et al., PNAS U.S.A., 80:4803 (1983).<br>
The most commonly used methods for transformation of plant cells are the Agrobacterium-mediated DNA transfer process) and the biolistics or microprojectile bombardment mediated process (i.e., the gene gun). Agrobacterium-mediated transformation is achieved through the use of a genetically engineered soil bacterium belonging to the genus Agrobacterium. Several Agrobacterium species mediate the transfer of a specific DNA known as "T-DNA," which can be genetically engineered to carry any desired piece of DNA into many plant species. The major events marking the process of T-DNA mediated pathogenesis are: induction of virulence genes, processing and transfer of T-DNA. This process is the subject of many reviews (Ream, Ann. Rev. Phytopathol, 27:583-618 (1989); Howard and Citovsky, Bioassays, 12:103-108 (1990); Kado, Crit. Rev. Plant Scl, 10:1-32 (1991); Zambryski, Annual Rev. Plant Physiol. Plant Mol. Biol., 43:465-490 (1992); Gelvin, In Transgenic Plants, Kung and Wu, eds., Academic Press, San Diego, pp. 49-87 (1993); Binns and Howitz, In Bacterial Pathogenesis of Plants and Animals (Dang, ed.). Berlin: Springer Verlag, pp. 119-138 (1994); Hooykaas and Beijersbergen, Ann. Rev. Phytopathol., 32:157-179 (1994); Lessl and Lanka, Cell, 77:321-324 (1994); and Zupan and Zambryski, Annual Rev. Phytopathol., 27:583-618 (1995)).<br>
Agrobacterium-mediated genetic transformation of plants involves several steps. The first step, in which the virulent Agrobacterium and plant cells are first brought into<br>
contact with each other, is generally called "inoculation." The Agrobacterium-contaming ^lution is then removed from contact with the explant by draining or aspiration. Following the inoculation, the Agrobacterium and plant cells/tissues are permitted to be grown together for a period of several hours to several days or more under conditions suitable for growth<br>
and<br>
T-DNA transfer. This step is termed "co-culture." Following co-culture and T-DNA delivery, the plant cells are treated with bactericidal or bacteriostatic agents to kill the Agrobacterium remaining in contact with the explant and/or in the vessel containing the explant. If this is done in the absence of any selective agents to promote preferential growth of transgenic versus non-transgenic plant cells, then this is typically referred to as the "delay" step. If done in the presence of selective pressure favoring transgenic plant cells, then it is referred to as a "selection" step. When a "delay" is used, it is typically followed by one or more "selection" steps. Both the "delay" and "selection" steps typically include bactericidal or bacteriostatic agents to kill any remaining Agrobacterium cells because the growth of Agrobacterium cells is undesirable after the infection (inoculation and co-culture) process.<br>
A number of wild-type and disarmed strains of Agrobacterium tumefaciens and Agrobacterium rhizogenes harboring Ti or Ri plasmids can be used for gene transfer into plants. The Agrobacterium hosts contain disarmed Ti and Ri plasmids that do not contain the oncogenes that cause tumorigenesis or rhizogenesis, respectively, which are used as the vectors and contain the genes of interest that are subsequently introduced into plants. Preferred strains include, but are not limited to, Agrobacterium tumefaciens strain C58, a nopaline-type strain that is used to mediate the transfer of DNA into a plant cell, octopine-type strains such as LBA4404 or succinamopine-type strains, e.g., EHA101 or EHA105. The nucleic acid molecule, prepared as a DNA composition in vitro, is introduced into a suitable host such as E. coli and mated into the Agrobacterium, or directly transformed into competent Agrobacterium. These techniques are well-known to those of skill in the art.<br>
The Agrobacterium can be prepared either by inoculating a liquid such as Luria Burtani (LB) media directly from a glycerol stock or streaking the Agrobacterium onto a solidified media from a glycerol stock, allowing the bacteria to grow under the appropriate selective conditions, generally from about 26°C - 30°C, or about 28°C, and taking a single colony or a small loop of Agrobacterium from the plate and inoculating a liquid culture medium containing the selective agents. Those of skill in the art are familiar with procedures for growth and suitable culture conditions for Agrobacterium as well as<br>
subsequent inoculation procedures. The density of the Agrobacterium culture used for<br>
J:oculation and the ratio of Agrobacterium cells to explant can vary from one system to the<br>
WP<br>
next, and, therefore, optimization of these parameters for any trans formation method is<br>
expected.<br>
Typically, an Agrobacterium culture is inoculated from a streaked plate or glycerol stock and is grown overnight and the bacterial cells are washed and resuspended in a culture medium suitable for inoculation of the explant.<br>
With respect to microprojectile bombardment (U.S. Patents 5,550,318; 5,538,880; 5,610,042; and WO 95/06128; each of which is specifically incorporated herein by reference in its entirety), particles are coated with nucleic acids and delivered into cells by a propelling force. Exemplary particles include those comprised of tungsten, platinum, and preferably, gold. It is contemplated that, in some instances, DNA precipitation onto metal particles would not be necessary for DNA delivery to a recipient cell using microprojectile bombardment. However, it is contemplated that particles can contain DNA, rather than be coated with DNA. Hence, it is proposed that DNA-coated particles can increase the level of DNA delivery via particle bombardment but are not, in and of themselves, necessary.<br>
For the bombardment, cells in suspension are concentrated on filters or solid culture medium. Alternatively, immature embryos or other target cells may be arranged on solid culture medium. The cells to be bombarded are positioned at an appropriate distance below the macroprojectile stopping plate.<br>
An illustrative embodiment of a method for delivering DNA into plant cells by acceleration is the Biolistics Particle Delivery System (BioRad, Hercules, CA), which can be used to propel particles coated with DNA or cells through a screen, such as a stainless steel or Nytex screen, onto a filter surface covered with monocot plant cells cultured in suspension. The screen disperses the particles so that they are not delivered to the recipient cells in large aggregates. It is believed that a screen intervening between the projectile apparatus and the cells to be bombarded reduces the size of projectiles aggregate and may contribute to a higher frequency of transformation by reducing the damage inflicted on the recipient cells by projectiles that are too large.<br>
Microprojectile bombardment techniques are widely applicable, and can be used to transform virtually any plant species. Examples of species that have been transformed by microprojectile bombardment include monocot species, such as maize (WO 95/06128), barley, wheat (U.S. Patent 5,563,055, specifically incorporated herein by reference in its entirety), rice, oat, rye, sugarcane, and sorghum; as well as a number of dicots including<br>
tobacco, soybean (U.S. Patent 5,322,783, specifically incorporated herein by reference in its ^tirety), sunflower, peanut, cotton, tomato, and legumes in general (U.S. Patent 5,563,055, specifically incorporated herein by reference in its entirety).<br>
To select or score for transformed plant cells regardless of transformation methodology, the DNA introduced into the cell contains a gene that functions in a regenerable plant tissue to produce a compound that confers upon the plant tissue resistance to an otherwise toxic compound. Genes of interest for use as a selectable, screenable, or scorable marker include, but are not limited to, GUS, green fluorescent protein (GFP), luciferase (LUX), antibiotic or herbicide tolerance genes. Examples of antibiotic resistance genes include the penicillins, kanamycin (and neomycin, G418, and bleomycin); methotrexate (and trimethoprim); chloramphenicol; kanamycin and tetracycline.<br>
Particularly preferred selectable marker genes for use in the present invention include genes that confer resistance to compounds such as antibiotics, like kanamycin (nptll), hygromycin B (aph IV), and gentamycin (aac3 and aacC4) (Dekeyser et al., Plant Physio!., 90:217-223 (1989)), and herbicides, like glyphosate (Della-Cioppa et al, Bio/Technology, 5:579-584 (1987)). Other selection devices also can be implemented including, but not limited to, tolerance to phosphinothricin, bialaphos, and positive-selection mechanisms (Joersbo et al., Mol. Breed., 4:111-117 (1998)) and are considered within the scope of the present invention.<br>
Transformed plant cells, which are derived by any of the above transformation techniques, can be cultured to regenerate a whole plant, which possesses the desired transformed phenotype. Any suitable plant culture medium can be used. Examples of suitable media would include but are not limited to MS-based media (Murashige and Skoog, Physiol. Plant, 15:473-497, 1962) or N6-based media (Chu et al, Scientia Sinica, 18:659, 1975) supplemented with additional plant growth regulators including but not limited to auxins, cytokinins, ABA, and gibberellins. Those of skill in the art are familiar with the variety of tissue culture media, which when supplemented appropriately, support plant tissue growth and development and are suitable for plant transformation and regeneration. These tissue culture media can either be purchased as a commercial preparation, or custom prepared and modified. Those of skill in the art are aware that media and media supplements such as nutrients and growth regulators for use in transfonnation and regeneration and other culture conditions such as light intensity during incubation, pH, and incubation temperatures that can be optimized for the particular variety of interest.<br>
One of ordinary skill will appreciate that, after an expression cassette is stably incorporated in transgenic plants and confirmed to be operable, it can be introduced into other plants by sexual crossing. Any of a number of standard breeding techniques can be used, depending upon the species to be crossed.<br>
In view of the foregoing, the present invention provides a method of suppressing the expression of one or more CDK inhibitor-like genes in a maize cell, a maize tissue, a maize organ, or a maize plant. The method comprises contacting said maize cell, maize tissue, maize organ, or maize plant with a nucleic acid construct as described above.<br>
Thus, the present invention further provides a maize cell, a maize tissue, a maize organ, or a maize plant in which the expression of a CDK inhibitor-like gene has been suppressed in accordance with such a method. Also provided is a seed obtained from such a plant and the oil and meal obtained from such a seed.<br>
The present invention also provides a container of over about 10,000, more preferably about 20,000, and even more preferably about 40,000 seeds where over about 10%, more preferably about 25%, more preferably about 50%, and even more preferably about 75% or more preferably about 90% of the seeds are seeds derived from a plant of the present invention.<br>
The present invention also provides a container of over about 10 kg, more preferably about 25 kg, and even more preferably about 50 kg seeds where over about 10%, more preferably about 25%, more preferably about 50%, and even more preferably about 75% or more preferably about 90% of the seeds are seeds derived from a plant of the present invention.<br>
Any of the plants or parts thereof of the present invention may be processed to produce a feed, meal, protein, or oil preparation. A particularly preferred plant part for this purpose is a seed. In a preferred embodiment the feed, meal, protein, or oil preparation is designed for ruminant animals. Methods to produce feed, meal, protein, and oil preparations are known in the art. See, for example, U.S. Patents 4,957,748; 5,100,679; 5,219,596; 5,936,069; 6,005,076; 6,146,669; and 6,156,227. In a preferred embodiment, the protein preparation is a high protein preparation. Such a high protein preparation preferably has a protein content of greater than 5% w/v, more preferably 10% w/v, and even more preferably 15%w/v.<br>
In a further embodiment, meal of the present invention may be blended with other meals. In a preferred embodiment, the meal produced from plants of the present invention or generated by a method of the present invention constitutes greater than about 0.5%, about<br>
1%, about 5%, about 10%, about 25%, about 50%, about 75%, or about 90% by volume or Ifieight of the meal component of any product. In another embodiment, the meal preparation may be blended and can constitute greater than about 10%, about 25%, about 35%, about 50%, or about 75% of the blend by volume.<br>
The present invention further provides a method of up-regulating the expression of one or more CDK inhibitor-like genes in a maize cell, a maize tissue, a maize organ, or a maize plant. The method comprises contacting a maize cell, tissue, organ, or plant with at least one nucleic acid construct comprising at least one nucleotide sequence selected from the group consisting of SEQ ID NO: 1 [ZmKRPl], a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 2 [ZmKRPl], SEQ ID NO: 3 [ZmKRP2], a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 4 [ZmKRP2], SEQ ID NO: 7 [ZmKRP4], a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 8 [ZmKRP4], SEQ ID NO: 9 [ZmKRPS], a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 10 [ZmKRP5], SEQ ID NO: 11 [ZmKRP6], and a nucleotide sequence encoding the amino acid sequence of SEQ ID NO: 12 [ZmKRP6], wherein, when a cell, tissue, or organ is contacted with the nucleic acid construct, the method further comprises generating a plant from the cell, tissue, or organ, whereupon the expression of one or more CDK inhibitor-like genes is up-regulated in a maize cell, tissue, organ, or plant. The method can comprise contacting the cell, tissue, organ, or plant with more than one nucleic acid construct and each nucleic acid construct comprises a different nucleotide sequence. The nucleic acid can comprise a promoter that is preferentially expressed in the male reproductive tissue of maize, wherein the promoter is operably linked to the at least one nucleotide sequence. An example of such a promoter is the SILKY1 promoter of maize and others as described herein above. Preferential expression in the male reproductive tissue of maize can result in the generation of a male-sterile maize plant.<br>
EXAMPLES<br>
The present invention is described further in the context of the following examples. These examples serve to illustrate further the present invention and are not intended to limit the scope of the invention.<br>
Example 1<br>
This example describes the identification of CDK inhibitors from maize.<br>
Initial tBLASTn searches (Altschul et al., Nuc. Acid Res., 25:3389-3402 (1997)) of Monsanto's proprietary EST database were performed using publicly available sequences of<br>
Arabidopsis CDK inhibitor-like genes AtKRPl ("At" indicates Arabidopsis thaliana; KRP1 ^ as defined above; GenBank #AAC49698), AtKRP2 (GenBank #CAB76424), AtKRPS (GenBank #CAC41617), AtKRP4 (GenBank #CAC41618), AtKRPS (GenBank #CAC41619), AtKRP6 (GenBank #CAC41620), and AtKRP? (GenBank #CAC41621). Additionally, the following conserved sequence from the C-terminal region of the above polypeptides was used in a tBLASTn search:<br>
(PTTAEIEDFFSEAEEQQQKQFIEKYNFDIVNDEPLEGRYEWVKLKP) [SEQ ID NO: 59].<br>
Resulting from the BLAST searches, seven maize sequence clusters were identified as having homology to the above-mentioned Arabidopsis CDK inhibitor-like genes. As used herein, "sequence clusters" refer to a group of EST sequences with overlapping regions of homology. Representative clones from each cluster were full-length double-strand sequenced and were used to BLAST search against Monsanto proprietary EST databases. An additional 2 clusters were identified from this second BLAST search and the representative clones were full-length double-strand sequenced. DNA sequence analysis indicated the presence of at least 9 different CDK inhibitor-like clusters in maize. These 9 clusters and their representative clones (named ZmKRPl through ZmKRP9) are summarized in Table 1.<br>
Table 1. Maize CDK inhibitor-like clusters and representative EST clones and ORF SEQ ID NO:<br><br>
(Table Removed)<br>
Coding regions and deduced amino acid sequences of the representative clones of these 9 clusters are shown in the sequence listing as SEQ ID NOs: 1-18. Full-length open reading frames (ORFs) were identified for ZmKRPl, 2, 4, 5, and 6, while partial ORFs were identified for ZmKRP3, 7, 8, and 9.<br>
Protein sequence alignments indicated that maize KRPs share small conserved regions with Arabidopsis ICK/KRPs, especially at the C-terminal regions (Figure 1). Additionally, some maize KRPs had high sequence identity to each other. For example, ZmKRP6 and ZmKRP7 were 90% identical, while ZmKRPl and ZmKRP2 were 84% identical. Other ZmKRP members shared only 16-50% identity.<br>
Sequence alignments and percent identity calculations were performed by the Clustal W Algorithm.<br>
Example 2<br>
This example describes the identification of CDK inhibitors targeted for suppression in corn embryos.<br>
The Monsanto proprietary com cDNA database was queried using DNA sequences from the maize KRP coding regions identified in Example 1 (SEQ ID NOs: 1, 3, 5, 7, 9, 11, 13, 15, and 17) using the CDK BLASTn protocol. This was done to determine which<br><br>
sequences were represented in cDNA libraries from developing kernel/embryo tissues, as ^ell as various non-kernel tissues. cDNA sequences corresponding to ZmKRPl, ZmKRP2, ZmKRP4, ZmKRP6, ZmKRPT, and ZmKRP9 (SEQ ID NOs: 1, 2, 4, 6, 7, and 9) were identified in kernel or embryo cDNA libraries. cDNA sequences corresponding to ZmKRP3, ZmKRP5, and ZmKRPS were found only in libraries generated from non-kernel libraries. These results indicated that at least the group of genes represented by the sequences found in the kernel or embryo cDNA libraries needed to be targeted for suppression. However, failure to identify cDNA sequences corresponding to a given gene in kernel libraries does not preclude the possibility of kernel expression, as a sequence may not be represented in a given library. Because all 9 genes may be expressed in the embryo, all were targeted for suppression using a dsRNA suppression construct.<br>
Example 3<br>
This example describes the construction of vectors for the tissue-specific suppression of the ZmKRP genes and the transformation of corn plants with these vectors.<br>
To suppress the 9 maize CDK inhibitor-like genes identified in Example 1, a dsRNA strategy was used. To build the dsRNA construct to suppress all 9 ZmKRP genes (ZmKRPl-9), 200 base pair fragments (F) were selected from the 3' end region of each gene (Fl to F7, SEQ ID NOs: 19 to 25), except for ZmKRP9, where 183 base pairs were selected (SEQ ID NO: 26) and connected in a polycistronic fashion. Since ZmKRP6 and ZmKRP? are identical in this 3' 200 base pair region, one fragment was used to target suppression of both genes (F6). To make the final corn expression construct, two intermediate constructs were first generated. The construction of the 2 intermediate dsRNA constructs is shown schematically in Figures 4 and 5.<br>
Each intermediate dsRNA construct contained 4 of the 3'-end fragments in both the sense and antisense orientation separated by an intervening intron region. The first intermediate construct contained the Fl through F4 sequences, and the second contained sequences F5 through F8. The two stem-loop structures, with respective promoters and 3' UTRs, were combined in one corn expression vector (Figure 2) and used for corn transformation using the Agrobacterium transformation method described in Example 10.<br>
To build the intermediate dsRNA construct containing the Fl to F4 sequences, the sense connections of Fl to F4, antisense connections of Fl to F4, and ZmDnaK (gene encoding for HSP70) intron fragment (SEQ ID NO: 27) (see e.g., U.S. Patent 5,424,412; Rochester et al, EMBOJ., 5:451-458 (1986)) were individually generated by PCR and<br>
assembled using a BamHI/Bglll directional cloning method (Figure 4). The PCR methods followed standard protocols set forth by the manufacturer (Roche Applied Science, Indianapolis, IN), and adding 5% DMSO, unless otherwise noted. The DMSO was added due to the high GC content of corn DNA.<br>
The Fl fragment was PCR amplified from the DNA of the ZmKRPl EST clone using primers Pl-Stui (SEQ ID NO: 28) and P1-2R (SEQ ID NO: 29). The primers are set forth in Table 2 and shown in Figure 5. The Fl fragment was then purified using standard methodology well-known in the art. The Fl-2 fragment was generated using PCR amplification by using the purified Fl PCR product mixed with an equivalent amount of the DNA of the ZmKRP2 EST clone as template, and Pl-Stui (SEQ ID NO: 28) and P2-3 (SEQ ID NO: 30) as primers. The PCR product Fl-2 was gel-purified using standard methodology. Similarly, F3 and F4 were PCR connected using primers P4-Bgl (SEQ ID NO: 31), P3-4 (SEQ ID NO: 32), and P5-Bam (SEQ ID NO: 33) resulting in a PCR product F3-4. The Fl-2 and F3-4 products were subsequently PCR connected using primers Pl-Stui, P2-3, and PS-Bam (SEQ ID NO: 33). The resulting PCR product, Fl-4, was purified and cloned into a vector using the Topo-TA cloning system (Invitrogen, Carlsbad, CA). The resulting clone, F14B3-sense, was full-length double-strand sequenced. The Fl-4 fragment was then PCR amplified from the F14B3-sense clone using primers P5-Bgl (SEQ ID NO: 34) and Pl-SseBam (SEQ ID NO: 35) to generate the antisense fragment. The resulting PCR product was again cloned using the Topo-TA cloning system to form the F14B3-antisense clone. This clone was confirmed by sequencing using standard methodology.<br>
Table 2. Primers used in construction of ZmKRP dsRNA vector<br><br>
(Table Removed)<br>
The ZmDnaK-intron (see e.g., U.S. Patent 5,424,412) was PCR amplified from the plant expression vector pMON70091 which contains a Z27 zein promoter (Lopes et al.,Mol Gen. Genet., 247:603-613 (1995) and ZmDnaK intron using primers Pi-Bgl (SEQ ID NO: 36) and Pi-Bam (SEQ ID NO: 37). The PCR product, DnaK-1, was cloned using the Topo-TA cloning system. This clone was confirmed by full-length double-strand sequencing. The Fl-4 sense, ZmDnaK intron, and Fl-4 antisense fragments were assembled in a step-wise fashion. Firstly, the ZmDnaK intron was excised from the DnaK-1 construct described above using Bglll and BamHI. This intron was then ligated to the BamHI site of F14B3-sense construct, such that the Bglll site of ZmDnaK-intron/BamHI-Bglll ligated to the BamHI site (introduced by primer P5-Bam) at the 3' end of Fl-4. This ligation reaction yielded the construct F14In2. Secondly, the Fl-4 fragment was excised from clone F14B3-antisense with Bglll and BamHI, and the Fl-4/BglII-BamHI fragment was ligated to the BamHI site of F14In2, such that the Bglll site of Fl-4/ Bglll-BamHI ligated to the BamHI site (introduced by primer Pi-Bam) at the 3' end of the ZmDnaK intron. This ligation reaction yielded the construct F142cl.<br>
Finally, the whole DNA segment containing the Fl-4 sense; ZmDnaK intron; Fl-4 antisense fragment was excised from clone F142cl using Stui and Sse8387I and cloned into pMON80611. pMON80611 is a binary vector for Agrobacterium transformation of plants. It contains left and right borders for T-DNA transfer, a plant selectable marker cassette consisting of enhanced 35s promoter: :glyphosate resistance marker gene::nos 3' UTR (U.S.<br>
Patent 5,627,061) and plant expression cassette sequences which include an oleosin L3 yromoter (U.S. Patent 6,433,252) and a maize globulin 3' UTR (see, for example, Belanger and Kriz, Genet., 129:863-872, (1991)) with cloning sites available for expression of a gene of interest. The resulting expression cassette was named WWZHEN03.0024.<br>
The intermediate dsRNA construct containing the F5 to F8 fragments was then generated. The sense connections of F5 to F8, antisense connections of F5 to F8, and ZmDnaK intron (SEQ ID NO: 27) were individually generated by PCR and assembled using either PCR or the BamHI/BgHI directional cloning method as described above and shown in Figure 5. Primers used in those PCR connections were listed in Table 2 and Figure 5. The assembled segment comprising F5-8 sense; ZmDnaK intron; F5-8 antisense was then cloned to vector pMON80611 to generate the second expression cassette WWZHEN03.0023. To combine WWZHEN03.0023 and WWZHEN03.0024, the 4528 bp AscI-Fsel segment from WWZHEN03.0023 that comprising F5-8 sense; ZmDnaK intron; F5-8 antisense was gel purified (Qiagen Inc., Valencia, CA) and ligated to WWZHEN03.0024 digested with MM and Fsel to generate the final corn transformation construct pMON71270 (Figure 2).<br>
A Perl promoter-driven ZmKRP suppression construct was made from the two intermediate dsRNA contructs WWZHEN03.0023 and WWZHDN03.0024 (above). First, the segment comprising Fl-4 sense; ZmDnaK intron; Fl-4 antisense was excised from WWZHDN03.0024 by Sfil and Sse8387I digestion, and then cloned to the Sfil and Sse8387I sites of vector pMON68290 which contained Perl to generate WWZHEN03.0041. Similarly, the segment comprising F5-8 sense; ZmDnaK intron; F5-8 antisense was excised from WWZHEN03.0023 by Sfil and Sse8387I digestion, and then cloned to the Sfil and Sse8387I sites of vector pMON68290 which contained Perl to generate WWZHEN03.0042. To combine WWZHEN03.0041 and WWZHEN03.0042, the 4481 bp AscI segment from WWZHEN03.0042, comprising F5-8 sense; ZmDnaK intron; F5-8 antisense, was gel purified (Qiagen Inc., Valencia, CA) and ligated to WWZHEN03.0041 digested with Mlul to generate the final corn transformation construct pMON71279 (Figure 3).<br>
Example 4<br>
This example sets forth the construction of plant transformation vectors for the suppression of one or more ZmKRP genes in corn.<br>
Additional seed-specific suppression constructs are generated using alternative size fragments, ranging from 50 base pairs to the full-length gene, and either alone or in various combinations. By way of example, a dsRNA construct using a 200 base pair fragment of<br>
ZmKRPl operably linked to the com L3 (oleosin) promoter is generated. The Fl fragment |^id the ZmDnaK-intron are generated as described above in Example 3. The two fragments are ligated to generate an intermediate Fl; ZmDnaK-intron fragment. An aliquot of isolated and purified Fl fragment is then PCR amplified to generate Fl-anti sense. The Fl-antisense is then ligated to the intermediate Fl; ZmDnaK-intron fragment to generate a dsRNA construct Fl; ZmDnaK-intron; Fl-antisense. The construct is then cloned into a vector downstream of the corn L3 (oleosin) promoter. Alternatively the construct is cloned into a vector downstream of a promoter such as P-Zm.CEPl, P-Zm.CPC214, P-Zm.CPC214trl, P-Zm.CPC214tr2, P-Os.CPC214, and P-Hv.PERl as well as other embryo-preferred promoters known to those skilled in the art. Similar constructs are generated with ZmKRP2, ZmKRP3, ZmKRP4, ZmKRPS, ZmKRP6, ZmKRP7, ZmKRPS, and ZmKRP9. Transgenic corn plants are produced. Transgenic seed is analyzed from these constructs to identify alterations in germ mass, kernel mass, cellular proliferation, whole kernel oil levels, whole kernel protein levels (% dry weight), whole kernel amino acid composition, free amino acid levels, and micronutrient composition, as set forth in the examples that follow. The ZmKRP gene fragments used in dsRNA constructs, which result in alterations in any of the abovementioned properties, are then combined, depending on the desired phenotype of the transformed plant, in a single polycistronic dsRNA construct, which is then transformed into corn using Agrobacterium transformation; transgenic plants are generated as described in Example 10.<br>
Example 5<br>
This example describes how the expression of endogenous CDK inhibitor-like genes in a plant can be suppressed by using antisense technology.<br>
The level of ZmKRPl generated in a cell of a corn plant can be decreased by introducing into the genome a transgene that expresses an antisense RNA of the ZmKRPl polypeptide. For example, the ZmKRPl cDNA, or a fragment thereof at least 100 bp in length, is cloned into the plant expression vector pMON80611 in the antisense orientation, which drives expression under control of the L3 promoter and contains the globulin 3' UTR (Belanger and Kriz, Genet., 129:863-872 (1991)). The resulting vector is transformed into the genome of a corn plant as described in Example 10. Multiple copies of the antisense gene can be introduced into a genome. Other CDK inhibitor-like genes, for instance ZmKRP2, can be made into antisense constructs and expressed separately or in combinations, as described above for ZmKRPl.<br><br>
Example 6<br>
This example describes how the expression of endogenous CDK inhibitor-like genes in a plant can be suppressed by using ribozyme technology.<br>
The level of ZmKRPl mRNA generated in a cell of a corn plant can be decreased by introducing into the genome a transgene that expresses a ribozyme targeted against an mRNA coding for a ZmKRPl polypeptide. The ribozyme construct is generated by cloning at least 24 bp of ZmKRPl cDNA with a ribozyme sequence added adjacent to the target AUG of the endogenous gene (Merlo et al., Plant Cell, 10(10): 1603-1622 (1998)) into pMON80611, which drives expression from the L3 promoter and contains the globulin 3' UTR (Belanger and Kriz, Genet., 129:863-872 (1991)). The resulting construct is transformed into a com plant as described in Example 10. Multiple copies of the ribozyme-containing ZmKRP genes can be introduced into a genome as described in Example 10. Ribozyme-containing genes targeted against a combination of other ZmKRP polypeptides (as described herein above) can be introduced into a genome as described above for ZmKRPl, such that multiple ZmKRP genes are simultaneously suppressed.<br>
Example 7<br>
This example describes how the expression of endogenous CDK inhibitor genes in a plant can be suppressed by using homologous recombination technology.<br>
The level of ZmKRPl generated in a cell of a corn plant can be decreased by a gene replacement method via homologous recombination. In this method, the endogenous ZmKRPl gene is replaced by a mutagenized ZmKRPl gene. The mutagenized gene codes for a ZmKRPl that is less efficient, or completely deficient, in inhibition of CDK than the one encoded by the endogenous, replaced gene. The ZmKRPl-producing gene is mutagenized by one or more nucleotide deletions, insertions, duplications or replacements. The mutagenized gene is fused to a selectable marker gene and transformed into a cell of a corn plant as described in Example 10. Homologous recombination events that may result in gene replacement are selected on the basis of the selectable marker gene (Puchta et al., Trends Plant ScL, 1:340 (1996); Kempin et al, Nature, 389:802 (1997)). Gene replacements are confirmed by Southern blot analysis or PCR and DNA sequencing.<br>
In the same manner as described above, other CDK inhibitor genes can be suppressed, independently or in combinations.<br>
Example 8<br>
This example describes how the expression of endogenous CDK inhibitor genes in a plant can be suppressed by using sense suppression technology.<br>
The level of ZmKRP 1 polypeptide generated in a cell of a plant is decreased by sense suppression. For example, the cDNA coding for an ZmKRP 1 polypeptide is fused to upstream (5') such as Perl, and/or downstream (3') transcriptional or translational regulatory sequences and the chimeric gene is cloned into an appropriate transformation vector that carries a selectable marker gene and the vector is transformed into a cell of a corn plant as described in Example 10. Transformants are selected on the basis of the presence of a marker gene (for instance glyphosate resistance). Transformants are confirmed by Southern blot analysis of the DNA from putative transformants. Most of the transgenic organisms that will be derived from such experimentations will express the transgene without affecting endogenous expression. However, in a few cases, both of the transgene and the endogenous gene will be suppressed. To identify these sense-suppressed plants, RNA or protein extracts from at least 60 transgenic plants will be analyzed for the presence of the ZmKRP mRNA or polypeptide, respectively. Other CDK inhibitor-like genes can be used, alone or in various combinations.<br>
Example 9<br>
This example sets forth one method of generating male sterile maize plants by expressing maize CDK inhibitors in male reproductive tissue.<br>
The com SILKY1 promoter is operably linked to ZmKRP 1, ZmKRP2, ZmKRP4, ZmKRPS, and ZmKRP6 in separate expression constructs containing the nos 3' UTR (Bevan et al, Nucl. Acid Res., 11:369 (1983)) (other possibilities exist) and a glyphosate resistance selectable marker. (See, U.S. Patent 5,627,061). One or more of the resulting expression cassettes is/are transformed into corn using the Agrobacterium transformation protocol set forth in Example 10 and transgenic plants are generated.<br>
Male sterility is determined by performing pollen germination assays (Walden, D.B. (1994), In vitro Pollen Germination (pp 723-724), The Maize Handbook, Freeling, M. and Walbot, V. eds, Springer-Verlag, New York, Inc.), if pollen is present. Additionally, pollen viability is assayed by the ability of the transgenic pollen to fertilize ovules of non-transgenic plants and to produce viable seed harboring the transgene. Sterile transgenic plants are maintained by fertilizing the transgenic plants with non-transgenic donor pollen.<br>
Example 10<br>
This example describes the transformation of maize plants with constructs containing maize CDK inhibitor-like sequences.<br>
The transformation vectors described above are used to transform maize plants using the following procedure. Maize plants are grown in a greenhouse under standard practices. Controlled pollinations are made. Agrobacterium ABI containing a vector in glycerol stock is streaked out on solid LB medium supplemented with antibiotics kanamycin (50mg/L), spectinomycin (50 mg/L), streptomycin (50 mg/L), and chloramphenicol (25 mg/L) and incubated at 28°C for 2 days. Two days before Agrobacterium inoculation of the maize immature embryos, one colony or a small loop of Agrobacterium from the Agrobacterium plate is picked up and inoculated into 25 mL of liquid LB medium supplemented with 100 mg/L of spectinomycin and 50 mg/L of kanamycin in a 250-mL flask. The flask is placed on a shaker at approximately 150 rpm and 26°C overnight. The Agrobacterium culture is then diluted (1 to 5) in the same liquid medium and put back on the shaker. Several hours later, one day before inoculation, the Agrobacterium cells are spun down at 3500 rpm for 15 min. The bacterium cell pellet is re-suspended in induction broth with 200 I^M of acetosyringone and 50 mg/L spectinomycin and 25 mg/L kanamycin and the cell density was adjusted to 0.2 at O.D.660- The bacterium cell culture (50 mL in each 250-mL flask) is then put back on the shaker and grown overnight. On the morning of inoculation day, the bacterium cells are spun down and washed with liquid 1/2MS VI medium (Table 3) supplemented with 200 u,M of acetosyringone. After one more spinning, the bacterium cell pellet is re-suspended in 1A MS PL medium (Table 3) with 200 uM of acetosyringone (Table 3) and the cell density was adjusted to 1.0 at O.DSGO for inoculation.<br>
Reagents are commercially available and can be purchased from a number of suppliers (see, for example, Sigma Chemical Co., St. Louis, MO).<br>
Table 3.Media used '.<br><br>
(Table Removed)<br>
. Media '/•&gt; MSVI and !/2 MSPL were used as liquid. Co-culture medium was solidified with 5.5 mg/1 low EEO agarose. All other media were solidified with 7 g/1 Phytagar for NPTII selection and with 3 g/1 phytagel for glyphosate selection.<br>
An elite corn line (LH244) is used for transformation in connection with this invention. Ears containing immature embryos are harvested approximately 10 days after pollination and kept refrigerated at 4°C until use (up to 5 days post-harvest). The preferred embryo size for this method of transformation is -1.0-2.0 mm. This size is usually achieved 10 days after pollination inside the green house with the growth conditions of an average temperature of 87°F, day length of 14 hours with supplemental lighting supplied by GE 1000 Watt High Pressure Sodium lamps.<br>
Immature embryos are isolated from surface sterilized ears and directly dropped into the prepared Agrobacterium cell suspension in 1.5-mL microcentrifuge tube. The isolation lasts continuously for 15 min. The tube is then set aside for 5 min, which made the inoculation time for individual embryos from 5 to 20 min. After Agrobacterium cell suspension is removed using a fine tipped sterile transfer pipette, the immature embryos are transferred onto the co-culture medium (Table 3). The embryos are placed on the medium with the scutellum side facing up. The embryos are cultured in a dark incubator (23°C) for approximately 24 h.<br><br>
The embryos are then transferred onto a modified MS medium (MSW50, TableS) Implemented with 0.1 or 0.25 mM glyphosate and 250 mg/L carbenicillin to inhibit Agrobacterium in Petri dishes (100 mm x 25 mm). The cultures are incubated in a dark culture room at 27°C for 2-3 weeks. All the callus pieces are then transferred individually onto the first regeneration medium (MS/6BA, Table 3) supplemented with the same levels of glyphosate. The cultures are grown on this medium and in a culture room with 16-h light/Sri dark photoperiod and 27°C for 5-7 days. They are then transferred onto the second regeneration medium (MSOD, Table 3) in petri dish (100 mm x 25 mm) for approximately 2 weeks. All the callus pieces with regenerating shoots and living tissue are transferred onto the same medium contained in phytatrays for shoots to grow further before being moved to soil. It takes 2-4 weeks. The regeneration media (MS6BA and MSOD) are all supplemented with 250 mg/L carbenicillin and 0.1 or 0.25 mM glyphosate.<br>
These developing plantlets are then transferred to soil, hardened off in a growth chamber at 27°C, 80% humidity, and low light intensity for approximately 1 week, and then transferred to a greenhouse and grown under standard greenhouse conditions.<br>
The greenhouse-grown plants are then analyzed for expression levels as well as oil and protein levels.<br>
Example 11<br>
This example describes expression analysis of transgenic plants.<br>
RNA expression analysis is used to verify misexpression or suppression of CDK inhibitors in transgenic plants. As used in this application, "misexpression" means spatial or temporal expression other than endogenous expression, e.g. overexpression and ectopic expression. RNA is extracted by a standard protocol (Wadsworth et al., Analytical Biochemistry, 172(l):279-283 (1988)). To determine the misexpression or suppression of ZmKRP genes in corn embryo tissues, TaqMan analysis is performed using the TaqMan One-Step RT-PCR Master Mix Reagents Kit and Protocol (#4310299 rev. C) (Applied Biosystems, Foster City CA). cDNA is synthesized (SMART PCR cDNA Synthesis Kit, #K1052-1, Clonetech Laboratories, Inc.) and standard PCR using gene-specific primers and agarose-gel electrophoresis to visualize PCR products.<br>
Example 12<br>
This example provides the analytical procedures to determine oil and protein content, mass differences, amino acid composition, free amino acid levels, and micronutrient content of transgenic maize plants.<br>
Oil levels (on a mass basis and as a percent of tissue weight) of first generation<br>
corn kernels and dissected germ and endosperm are determined by low-resolution H nuclear magnetic resonance (NMR) (Tiwari et al, JAOCS, 51:104-109 (1974); or Rubel, JAOCS, 71:1057-1062 (1994)), whereby NMR relaxation times of single kernel samples are measured, and oil levels are calculated based on regression analysis using a standard curve generated from analysis of corn kernels with varying oil levels as determined gravimetrically following accelerated solvent extraction. One-way analysis of variance and the Student's T-test (JMP, version 4.04, SAS Institute Inc., Cary, NC, USA) are performed to identify significant differences between transgenic and non-transgenic kernels as determined by transgene-specific PCR.<br>
Oil levels and protein levels in second generation seed are determined by NIT spectroscopy, whereby NIT spectra of pooled seed samples harvested from individual plants are measured, and oil and protein levels are calculated based on regression analysis using a standard curve generated from analysis of corn kernels with varying oil or protein levels, as determined gravimetrically following accelerated solvent extraction or elemental (%N) analysis, respectively. One-way analysis of variance and the Student's T-test are performed to identify significant differences in oil (% kernel weight) and protein (% kernel weight) between seed from marker positive and marker negative plants.<br>
The levels of free amino acids are analyzed from each of the transgenic events using the following procedure. Seeds from each of the transgenic plants are crushed individually into a fine powder and approximately 50 mg of the resulting powder is transferred to a pre-weighed centrifuge tube. The exact sample weight is recorded and 1.0 ml of 5% trichloroacetic acid is added to each sample tube. The samples are mixed at room temperature by vortex and then centrifuged for 15 minutes at 14,000 rpm on an Eppendorf microcentrifuge (Model 5415C, Brinkmann Instrument, Westbury, NY). An aliquot of the supernatant is removed and analyzed by HPLC (Agilent 1100) using the procedure set forth in Agilent Technical Publication "Amino Acid Analysis Using the Zorbax Eclipse-AAA Columns and the Agilent 1100 HPLC," March 17, 2000.<br>
Quantitative determination of total amino acids from corn is performed by the following method. Kernels are ground and approximately 60 mg of the resulting meal is acid-hydrolyzed using 6 N HC1 under reflux at 100°C for 24 hrs. Samples are dried and reconstituted in 0.1 N HC1 followed by precolumn derivatization with a-phthalaldehyde (OPAO for HPLC analysis. The amino acids are separated by a reverse-phase Zorbax Eclipse XDB-C18 HPLC column on an Agilent 1100 HPLC (Agilent, Palo Alto, CA). The<br>
amino acids are detected by fluorescence. Cysteine, proline, asparagine, glutamine, and tryptophan are not included in this amino acid screen (Henderson et al., "Rapid, Accurate, Sensitive and Reproducible HPLC Analysis of Amino acids, Amino Acid Analysis Using Zorbax Eclipse-AAA Columns and the Agilent 1100 HPLC," Agilent Publication (2000); see, also, "Measurement of Acid-Stable Amino Acids," AACC Method 07-01 (American Association of Cereal Chemists, Approved Methods, 9th edition (LCCC# 95-75308)). Total tryptophan is measured in corn kernels using an alkaline hydrolysis method as described (Approved Methods of the American Association of Cereal Chemists -10th edition, AACC ed, (2000) 07-20 Measurement of Tryptophan - Alakline Hydrolysis).<br>
Tocopherol and tocotrienol levels in seeds are assayed by methods well-known in the art. Briefly, 10 nig of seed tissue are added to 1 g of microbeads (Biospec Product Inc, Barlesville, OK) in a sterile microfuge tube to which 500 ul 1% pyrogallol (Sigma Chemical Co., St. Louis, MO)/ethanol have been added. The mixture is shaken for 3 minutes in a mini Beadbeater (Biospec) on "fast" speed, then filtered through a 0.2 |^m filter into an autosampler tube. The filtered extracts are analyzed by HPLC using a Zorbax silica HPLC column<br>
(4.6 mmx250 mm) with a fluorescent detection, an excitation at 290 nm, an emission at 336 nm, and bandpass and slits. Solvent composition and running conditions are as listed below with solvent A as hexane and solvent B as methyl-t-butyl ether. The injection volume is 20 (4,1, the flow rate is 1.5 ml/minute and the run time is 12 minutes at 40°C. The solvent gradient is 90% solvent A, 10% solvent B for 10 minutes; 25% solvent A, 75% solvent B for 11 minutes; and 90% solvent A, 10% solvent B for 12 minutes. Tocopherol standards in 1% pyrogallol/ethanol are run for comparison (a-tocopherol, y-tocopherol, pMocopherol, 5-tocopherol, and tocopherol (tocol)). Standard curves for alpha, beta, delta, and gamma tocopherol are calculated using Chemstation software (Hewlett Packard). Tocotrienol standards in 1% pyrogallol/ethanol are run for comparison (a-tocotrienol, y- tocotrienol, P- tocotrienol, 8- tocotrienol). Standard curves for a-, (3-, 8-, and y-tocotrienol are calculated using Chemstation software (Hewlett Packard).<br>
Carotenoid levels within transgenic corn kernels are determined by a standard protocol (Craft, Meth. Enzymol, 213:185-205 (1992)). Plastiquinols and phylloquinones are determined by standard protocols (Threlfall et al., Methods in Enzymology, XVIII, part C, 369-396 (1971); and Ramadan et al, Eur. Food Res. Technol, 214(6):521-527 (2002)).<br>
Results from events containing pMON71270 are shown in Table 4.<br>
(Table Removed)<br>
Examples of increased oil and organ size are demonstrated in several events. The milligrams oil per germ was increased in two independent events, ZMJS86578 (p=0.05) and ZM_S86487 (p=0.1). The milligrams of oil per kernel (p=0.05) and kernel oil as a percentage of dry wt (p=0,01) were both increased in event ZM_S86481. Germ oil as a percentage of dry wt was increased in three independent events, ZM_S 86521 (p=0,l), ZM_S86548 (p=0.01) and ZM_S86578 (p=0.05). Germ wt (gram dry wt) (p=0.1) and germ wt as a percentage of total kernel wt (p=0.1) were increased in event ZM_S86487.<br>
Results from events containing pMON71279 are shown in Table 5. <br>
(Table Removed)<br>
 Examples of increased oil and organ size are demonstrated in several events. Germ wt (g dry wt) was increased in three independent events, ZMJS105183 (p=0.01), ZM_S105166 (p=0.05) and ZM_S106973 (p=0.01). The milligrams of oil per germ was increased in three independent events, ZM_S105183 (p=0.01), ZM_S105185 (p=0.05), and ZM_S106979 (p=0.01). Germ wt as a percentage of total kernel wt was increased in two independent events, ZM_S 105183 (p=0.1) and ZM_S 106940 (p=0.1). The milligrams of oil per kernel was increased in two independent events, ZM_S105166 (p=0.1) and ZM_S106979 (p=0.05). Kernel oil as a percentage of dry wt was increased in event ZM_S106985 (p=0.1). Kernel wt (g dry wt) was increased in two independent events ZM_S105166 (p=0.05) and ZM_S 106979 (p=0.01).<br>
Results show that seed harboring the pMON71270, and other constructs described in Examples 3 and 4, have at least one of the following phenotypes: increased germ mass (as a percent of kernel weight), increased whole kernel oil levels (as a percent of kernel weight), improved protein quality (increased lysine, tryptophan, threonine, isoleucine and methionine, for example), free amino acids (up or down) or increased micronutrient content, including tocopherols, tocotrienols, carotenoids, plastiquinols and phylloquinones.<br>
Example 13<br>
This example describes the histological analysis of tissues from transgenic maize<br>
plants.<br>
Mature or developing kernels of fresh or frozen corn are sliced sagitally. Tissue slices of 1 mm or thinner are stained for 1 to 2 min in a solution of 0.01% aniline blue C.I. 42755 (Baker, Phillipsburg, N.J.) in 0.1M phosphate buffer, pH 8.0. Samples are rinsed in 0.1 M phosphate buffer, pH 8.0, and examined with either of a standard fluorescent microscope using a narrow violet cube Nikon V-4 (EX 380-420nm, DM 430 nm, BA 450 nm) or an ultraviolet light source (EX330-380, DM400, BA420). Alternatively, samples can be visualized using a confocal microscope using an Argon-ion laser (emission wavelength 488 nm and 514 nm). Aniline blue stains the cell walls and nuclei of plants (Smith and McCully, Stain Technology, 53(2):79-85 (1978)). Cell size is determined by using an outlining tool in the Nikon EZ C-l software. Approximately 25 to 50 cells are outlined and the area the cells occupy is recorded and the average area per cell is calculated.<br>
Histological analysis of events containing pMON71270 yielded 3 events that demonstrated a significant increase in cell number as characterized by more cells per unit area in the embryo (1.3x, 1.7x, 2.4x, p=0.05) with no reduction in embryo mass.<br>
Histological analysis of events containing pMON71279 gave 3 events with wild-type cell number per unit area (l.lx, 0.97x, 0.96x, p=0.05), but due to increased embryo mass, yielded an increase in the total number of cells per embryo.<br>
To determine mitotic indices, corn tissues are fixed, stained with a fluorescent probe, cleared and imaged using fluorescent microscopy or confocal scanning laser microscopy (CLSM). Mitotic index is determined by counting the number of cells in an area that are in prometaphase, metaphase or anaphase and reported as a percentage. For example, corn tissue can be fixed in formalin, propinionic acid and ethanol, stained with propidium iodide and cleared with xylene (Running et al., Confocal Microscopy of the Shoot Apex, Methods in Cell Biology, 49:217-229 (1995)).<br>
Example 14<br>
This example describes how to measure CDK inhibitor activity.<br>
Protein extracts from transgenic and non-transgenic plant tissues are enriched for CDK using p!3Sucl resin (Oncogene Sciences, Cambridge, MA), and histone HI kinase assays are performed (Wang et al., Plant J., 15:501-510 (1998)). Transgenic plant tissues<br>
with increased CDK inhibitor activity have reduced kinase activity, whereas transgenic plant ^issues with reduced CDK inhibitor activity have increased kinase activity.<br>
The following references are specifically incorporated herein by reference: Birren etal., Genome Analysis: A Laboratory Manual Series, Volume 1, Analyzing DNA, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY (1997), Birren et al., Genome Analysis: A Laboratory Manual Series, Volume 2, Detecting Genes, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY (1998), Birren et al., Genome Analysis: A Laboratory Manual Series, Volume 3, Cloning Systems, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY (1999), Birren et al., Genome Analysis: A Laboratory Manual Series, Volume 4, Mapping Genomes, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY (1999), Harlow et al., Antibodies: A Laboratory Manual, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY (1988),<br>
Harlow et al., Using Antibodies: A Laboratory Manual, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY (1999),<br>
Sambrook et al., Molecular Cloning: A Laboratory Manual, 2nd edition, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY (1989),<br>
Ausubel et al., Current Protocols in Molecular Biology, John Wiley &amp; Sons, Inc., NY (1999), and<br>
Croy, Plant Molecular Biology Labfase, BIOS Scientific Publications, Ltd. (UK) and Blackwell Scientific Publications (UK) (1993).<br>
All references, including publications, patent applications, and patents, cited herein are hereby incorporated by reference to the same extent as if each reference were individually and specifically indicated to be incorporated by reference and were set forth in its entirety herein.<br>
While this invention has been described with an emphasis upon preferred embodiments, it will be apparent to those of ordinary skill in the art that variations in the preferred embodiments can be prepared and used and that the invention can be practiced otherwise than as specifically described herein. The present invention is intended to include such variations and alternative practices. Accordingly, this invention includes all modifications encompassed within the spirit and scope of the invention as defined by the following claims.<br>
The use of the terms "a" and "an" and "the" and similar referents in the context of describing the invention (especially in the context of the following claims) are to be construed to cover both the singular and the plural, unless otherwise indicated herein or<br>
clearly contradicted by context. The terms "comprising," "having," "including," and ^'containing" are to be construed as open-ended terms (i.e., meaning "including, but not limited to,") unless otherwise noted. Recitation of ranges of values herein are merely intended to serve as a shorthand method of referring individually to each separate value falling within the range, unless otherwise indicated herein, and each separate value is incorporated into the specification as if it were individually recited herein. All methods described herein can be performed in any suitable order unless otherwise indicated herein or otherwise clearly contradicted by context. The use of any and all examples, or exemplary language (e.g., "such as") provided herein, is intended merely to better illuminate the invention and does not pose a limitation on the scope of the invention unless otherwise claimed. No language in the specification should be construed as indicating any non-claimed element as essential to the practice of the invention.<br>
Preferred embodiments of this invention are described herein, including the best mode known to the inventors for carrying out the invention. Variations of those preferred embodiments may become apparent to those of ordinary skill in the art upon reading the foregoing description. The inventors expect skilled artisans to employ such variations as appropriate, and the inventors intend for the invention to be practiced otherwise than as specifically described herein. Accordingly, this invention includes all modifications and equivalents of the subject matter recited in the claims appended hereto as permitted by applicable law. Moreover, any combination of the above-described elements in all possible variations thereof is encompassed by the invention unless otherwise indicated herein or otherwise clearly contradicted by context.<br><br><br><br><br><br><br><br>
WE CLAIM:<br>
1.	A recombinant nucleic acid molecule used inter alia for making<br>
transgenic plants and seeds comprising DNA that has CDK inhibitor-like<br>
activity selected from the group consisting of:<br>
(i) the nucleotide sequences having at least 70% to 99% sequence identity of Seq. ID Nos. 1, 3 and 9 and a fragment of any, wherein said fragment comprises at least 35 to 200 contiguous nucleotides of Seq. ID Nos. 1 or 3 or least 120 to 200 contiguous nucleotides of Seq. ID No: 9; and<br>
(ii) the nucleotide sequences encoding an amino acid sequence thereof; wherein the said recombinant nucleic acid molecule is further operably linked to a promoter.<br>
2.	The recombinant nucleic acid molecule as claimed in claim 1, wherein the said nucleotide sequences encoding an amino acid sequence is at least 70% to 99% identical to the amino acid sequences selected from the group consisting of SEQ ID Nos.: 2, 4 and 10.<br>
3.	The recombinant nucleic acid molecule as claimed in claim 1, wherein the promoter is the SILKY1 promoter of maize, the NPGl promoter of tobacco, the AP3 promoter of Arahidopsis, the Bgpl promoter of Brassica, an anther-preferred promoter, a pollen-preferred promoter, or a microspore-preferred promoter.<br>
4.	A nucleic acid molecule comprising the complementary or antisense sequence to the said recombinant nucleic acid molecule and a polypeptide encoded by the recombinant nucleic acid molecule as claimed in claims 1 to 3.<br><br>
5. A vector comprising the recombinant nucleic acid as claimed in any one of claims 1 to 4.<br><br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LURFTE5QLTIwMDYtQWJzdHJhY3QtKDA2LTA4LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">296-DELNP-2006-Abstract-(06-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LURFTE5QLTIwMDYtQWJzdHJhY3QtKDEzLTAxLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">296-DELNP-2006-Abstract-(13-01-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LURFTE5QLTIwMDYtQWJzdHJhY3QtKDI5LTEyLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">296-DELNP-2006-Abstract-(29-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LWRlbG5wLTIwMDYtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">296-delnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LURFTE5QLTIwMDYtQ2xhaW1zKDEzLTAxLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">296-DELNP-2006-Claims(13-01-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LURFTE5QLTIwMDYtQ2xhaW1zLSgwNi0wOC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">296-DELNP-2006-Claims-(06-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LURFTE5QLTIwMDYtQ2xhaW1zLSgyOS0xMi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">296-DELNP-2006-Claims-(29-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LWRlbG5wLTIwMDYtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">296-delnp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LWRlbG5wLTIwMDYtY29tcGxldGUgc3BlY2lmaWNhdGlvbiAoZ3JhbnRlZCkucGRm" target="_blank" style="word-wrap:break-word;">296-delnp-2006-complete specification (granted).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LURFTE5QLTIwMDYtQ29ycmVzcG9uZGVuY2UtT3RoZXJzLSgwNi0wOC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">296-DELNP-2006-Correspondence-Others-(06-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LURFTE5QLTIwMDYtQ29ycmVzcG9uZGVuY2UtT3RoZXJzLSgyOS0xMi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">296-DELNP-2006-Correspondence-Others-(29-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LWRlbG5wLTIwMDYtY29ycmVzcG9uZGVuY2Utb3RoZXJzLTEucGRm" target="_blank" style="word-wrap:break-word;">296-delnp-2006-correspondence-others-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LWRlbG5wLTIwMDYtY29ycmVzcG9uZGVuY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">296-delnp-2006-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LURFTE5QLTIwMDYtRGVzY3JpcHRpb24gKENvbXBsZXRlKS0oMDYtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">296-DELNP-2006-Description (Complete)-(06-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LWRlbG5wLTIwMDYtZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">296-delnp-2006-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LURFTE5QLTIwMDYtRHJhd2luZ3MtKDA2LTA4LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">296-DELNP-2006-Drawings-(06-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LWRlbG5wLTIwMDYtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">296-delnp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LURFTE5QLTIwMDYtRm9ybS0xLSgwNi0wOC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">296-DELNP-2006-Form-1-(06-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LURFTE5QLTIwMDYtRm9ybS0xLSgyOS0xMi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">296-DELNP-2006-Form-1-(29-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LWRlbG5wLTIwMDYtZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">296-delnp-2006-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LWRlbG5wLTIwMDYtZm9ybS0xOC5wZGY=" target="_blank" style="word-wrap:break-word;">296-delnp-2006-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LURFTE5QLTIwMDYtRm9ybS0yLSgwNi0wOC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">296-DELNP-2006-Form-2-(06-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LURFTE5QLTIwMDYtRm9ybS0yLSgyOS0xMi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">296-DELNP-2006-Form-2-(29-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LWRlbG5wLTIwMDYtZm9ybS0yLnBkZg==" target="_blank" style="word-wrap:break-word;">296-delnp-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LURFTE5QLTIwMDYtRm9ybS0zLSgwNi0wOC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">296-DELNP-2006-Form-3-(06-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LWRlbG5wLTIwMDYtZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">296-delnp-2006-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LWRlbG5wLTIwMDYtZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">296-delnp-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LURFTE5QLTIwMDYtR1BBLSgwNi0wOC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">296-DELNP-2006-GPA-(06-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LWRlbG5wLTIwMDYtZ3BhLnBkZg==" target="_blank" style="word-wrap:break-word;">296-delnp-2006-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LWRlbG5wLTIwMDYtcGN0LTEwMS5wZGY=" target="_blank" style="word-wrap:break-word;">296-delnp-2006-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LWRlbG5wLTIwMDYtcGN0LTIxMC5wZGY=" target="_blank" style="word-wrap:break-word;">296-delnp-2006-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LWRlbG5wLTIwMDYtcGN0LTIyMC5wZGY=" target="_blank" style="word-wrap:break-word;">296-delnp-2006-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LWRlbG5wLTIwMDYtcGN0LTIzNy5wZGY=" target="_blank" style="word-wrap:break-word;">296-delnp-2006-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LWRlbG5wLTIwMDYtcGN0LTMwNC5wZGY=" target="_blank" style="word-wrap:break-word;">296-delnp-2006-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LWRlbG5wLTIwMDYtcGN0LTQwMS5wZGY=" target="_blank" style="word-wrap:break-word;">296-delnp-2006-pct-401.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LWRlbG5wLTIwMDYtcGN0LTQwMi5wZGY=" target="_blank" style="word-wrap:break-word;">296-delnp-2006-pct-402.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LWRlbG5wLTIwMDYtcGN0LTQwOS5wZGY=" target="_blank" style="word-wrap:break-word;">296-delnp-2006-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LWRlbG5wLTIwMDYtcGN0LTQxNi5wZGY=" target="_blank" style="word-wrap:break-word;">296-delnp-2006-pct-416.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LURFTE5QLTIwMDYtUGV0aXRpb24tMTM3LSgwNi0wOC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">296-DELNP-2006-Petition-137-(06-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2LURFTE5QLTIwMDYtUGV0aXRpb24tMTM4LSgwNi0wOC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">296-DELNP-2006-Petition-138-(06-08-2008).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="227577-a-bioactive-cationic-amphiphile-for-intracellular-delivery-of-therapeutic-molecules-and-a-process-for-the-preparation-thereof.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="227579-a-composition-of-spores-of-nonpathogenic-bacteria-of-the-bacillus-genus.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>227578</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>296/DELNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>05/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>30-Jan-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>13-Jan-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>17-Jan-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MONSANTO TECHNOLOGY, LLC.,</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>800 NORTH LINDBERGH BLVD., ST. LOUIS, MO 63167, UNITED STATES OF AMERICA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>WEI ZHENG</td>
											<td>800 NORTH LINDBERGH BLVD, ST. LOUIS, MO 63167, U.S.A.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>BETH SAVIDGE</td>
											<td>800 NORTH LINDBERGH BLVD., ST, LOUIS, MO 63167, U.S.A.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/29</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2004/022758</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-07-14</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/486, 935</td>
									<td>2003-07-14</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>10/890, 629</td>
									<td>2004-07-14</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/227578-a-recombinant-nucleic-acid-molecule-used-for-making-transgenic-plants-and-seeds by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:31:54 GMT -->
</html>
