<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/227543-covalent-tethering-of-functional-groups-to-proteins by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:26:03 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 227543:COVALENT TETHERING OF FUNCTIONAL GROUPS TO PROTEINS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">COVALENT TETHERING OF FUNCTIONAL GROUPS TO PROTEINS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A mutant hydrolase optionally fused to a protein of interest is provided. The mutant hydrolase is capable of forming a bond with a substrate for the corresponding nonmutant (wild-type) hydrolase which is more stable than the bond formed between the wild-type hydrolase and the substrate. Substrates for hydrolases comprising one or more functional groups are also provided, as well as methods of using the mutant hydrolase and the substrates of the invention. Also provided a fusion protein capable of forming a stable bond with a substrate and cells which express the fusion protein.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>COVALENT TETHERING OF FUNCTIONAL GROUPS TO PROTEINS<br>
Cross-Reference to Related Applications<br>
This application claims the benefit of the filing date of U.S. application<br>
Serial No. 60/444,094, filed January 31,2003, and U.S. application Serial No.<br>
60/474,659, filed May 30, 2003, under 35 U.S.C. ยง 119(e), and incorporates<br>
those applications by reference herein.<br>
Field of the Invention<br>
This invention relates to the field of biochemical assays and reagents.<br>
More specifically, this invention relates to mutant proteins covalently linked<br>
(tethered) to one or more functional groups and to methods for their use.<br>
Background of the Invention<br>
The specific detection of molecules is a keystone in understanding the<br>
role of that molecule in the cell. Labels, e.g., those that are covalently linked to<br>
a molecule of interest, permit the ready detection of that molecule in a complex<br>
mixture. The label may be one that is added by chemical synthesis in vitro or<br>
attached in vivo, e.g., via recombinant techniques. For instance, the attachment<br>
of fluorescent or other labels onto proteins has traditionally been accomplished<br>
by in vitro chemical modification after protein purification (Hermanson, 1996).<br>
For in vivo attachment of a label, green fluorescent protein (GFP) from the<br>
jellyfish Aequorea victoria can be genetically fused with many host proteins to<br>
produce fluorescent chimeras in situ (Tsien, 1998; Chalfie et al., 1998).<br>
However, while GFP-based indicators are currently employed in a variety of<br>
assays, e.g., measuring pH (Kneen et al., 1998; Llopis et al., 1998; Miesenbb'ck<br>
et al., 1998), Ca2+ (Miyawaki et al., 1997; Rosomer et al., 1997), and membrane<br>
potential (Siegel et al., 1997), the fluorescence of intrinsically labeled proteins<br>
such as GFP is limited by the properties of protein structure, e.g., a limited range<br>
of fluorescent colors and relatively low intrinsic brightness (Cubitt et al., 1995;<br>
Ormo'etal., 1996).<br>
To address the deficiencies of GFP labeling in situ, Griffen et al. (1998)<br>
synthesized a tight-binding pair of molecular components: a small receptor<br>
domain composed of as few as six natural amino acids and a small (
dalton), synthetic ligand that could be linked to various spectroscopic probes or<br>
crosslinks. The receptor domain included four cysteines at the i, i + 1, i + 4, and<br>
/ + 5 positions of an a helix and the ligand was 4',5'-bis(l,3,2-dithioarsolan-2-<br>
yl)fluorescein (FLASH). Griffen et al. disclose that the ligand had relatively few<br>
binding sites in nontransfected mammalian cells, was membrane-permeant and<br>
was nonfluorescent until it bound with high affinity and specificity to a<br>
tetracysteine domain in a recombinant protein, resulting in cells being<br>
fluorescently labeled ('TLASH" labeled) with a nanomolar or lower dissociation<br>
constant. However, with respect to background binding in cells, Stroffekova et<br>
al. (2001) disclose that FLASH-EDTa binds non-specifically to endogenous<br>
cysteine-rich proteins. Furthermore, labeling proteins by FLASH is limited by<br>
the range of fluorophores that may be used.<br>
Receptor-mediated targeting methods use genetically encoded targeting<br>
sequences to localize fluorophores to virtually any cellular site, provided that the<br>
targeted protein is able to fold properly. For example, Farinas et al. (1999)<br>
disclose that cDNA transfection was used to target a single-chain antibody (sFv)<br>
to a specified site in a cell. Farinas et al. disclose that conjugates of a hapten (4-<br>
ethoxymethylene-2-phenyl-2-oxazolin-5-one, phOx) and a fluorescent probe<br>
(e.g., BODIPY Fl, tetramethylrhodamine, and fluorescein) were bound with high<br>
affinity (about 5 nM) to the subcellular site for the sFv in living Chinese hamster<br>
ovary cells, indicating that the targeted antibody functioned as a high affinity<br>
receptor for the cell-permeable hapten-fluorophore conjugates. Nevertheless,<br>
functional sFv expression may be relatively poor in reducing environments.<br>
Thus, what is needed is an improved method to label a desired protein.<br>
Summary of the Invention<br>
The invention provides methods, compositions and kits for tethering<br>
(linking), e.g., via a covalent or otherwise stable bond, one or more functional<br>
groups to a protein of the invention or to a fusion protein (chimera) which<br>
includes a protein of the invention. A protein of the invention is structurally<br>
related to a wild-type (native) hydrolase but comprises at least one amino acid<br>
substitution relative to the corresponding wild-type hydrolase and binds a<br>
substrate of the corresponding wild-type hydrolase but lacks or has reduced<br>
catalytic activity relative to the corresponding wild-type hydrolase (which<br>
mutant protein is referred to herein as a mutant hydrolase). The aforementioned<br>
tethering occurs, for instance, in solution or suspension, in a cell, on a solid<br>
support or at solution/surface interfaces, by employing a substrate for a<br>
hydrolase which includes a reactive group and which has been modified to<br>
include one or more functional groups. As used herein, a "substrate" includes a<br>
substrate having a reactive group and optionally one or more functional groups.<br>
A substrate which includes one or more functional groups is generally referred to<br>
herein as a substrate of the invention. As used herein, a "functional group" is a<br>
molecule which is detectable or is capable of detection (e.g., a chromophore,<br>
fluorophore or luminophore), or can be bound or attached to a second molecule<br>
(e.g., biotin, hapten, or a cross-linking group) or includes one or more amino<br>
acids, e.g., a peptide or polypeptide including an antibody or receptor, one or<br>
more nucleotides, lipids including lipid bilayers, a solid support, e.g., a<br>
sedimental particle, and the like. A functional group may have more than one<br>
property such as being capable of detection and being bound to another<br>
molecule. As used herein a "reactive group" is the minimum number of atoms in<br>
a substrate which are specifically recognized by a particular wild-type or mutant<br>
hydrolase of the invention. The interaction of a reactive group in a substrate and<br>
a wild-type hydrolase results in a product and the regeneration of the wild-type<br>
hydrolase. A substrate, e.g., a substrate of the invention, may also optionally<br>
include a linker, e.g., a cleavable linker.<br>
A substrate useful in the invention is one which is specifically bound by<br>
a mutant hydrolase, and preferably results in a bond formed with an amino acid,<br>
e.g., the reactive residue, of the mutant hydrolase which bond is more stable than<br>
the bond formed between the substrate and the corresponding amino acid of the<br>
wild-type hydrolase. While the mutant hydrolase specifically binds substrates<br>
which may be specifically bound by the corresponding wild-type hydrolase, no<br>
product or substantially less product, e.g., 2-, 10-, 100-, or 1000-fold less, is<br>
formed from the interaction between the mutant hydrolase and the substrate<br>
under conditions which result in product formation by a reaction between the<br>
corresponding wild-type hydrolase and substrate. The lack of, or reduced<br>
amounts of, product formation by the mutant hydrolase is due to at least one<br>
substitution in the mutant hydrolase, which substitution results in the mutant<br>
hydrolase forming a bond with the substrate which is more stable than the bond<br>
formed between the corresponding wild-type hydrolase and the substrate.<br>
Preferably, the bond formed between a mutant hydrolase and a substrate of the<br>
invention has a half-life (i.e., t^) that is at least 2-fold, and more preferably at<br>
least 4- or even 10-fold, and up to 100-, 1000- or 10,000-fold, greater than the t&gt;A<br>
of the bond formed between a corresponding wild-type hydrolase and the<br>
substrate under conditions which result in product formation by the<br>
corresponding wild-type hydrolase. Preferably, the bond formed between the<br>
mutant hydrolase and the substrate has a t^ of at least 30 minutes and preferably<br>
at least 4 hours, and up to at least 10 hours, and is resistant to disruption by<br>
washing, protein denaturants, and/or high temperatures, e.g., the bond is stable to<br>
boiling in SDS.<br>
In one embodiment, the substrate is a substrate for a dehalogenase, e.g., a<br>
haloalkane dehalogenase or a dehalogenase that cleaves carbon-halogen bonds in<br>
an aliphatic or aromatic halogenated substrate, such as a substrate for<br>
Rhodococcus, Staphylococcus, Pseudomonas, Burkholderia, Agrobacterium or<br>
Xanthobacter dehalogenase, of a substrate for a serine beta-lactamase. In one<br>
embodiment, a substrate of the invention optionally includes a linker which<br>
physically separates one or more functional groups from the reactive group in<br>
the substrate. For instance, for some mutant hydrolases, i.e., those with deep<br>
catalytic pockets, a substrate of the invention can include a linker of sufficient<br>
length and structure so that the one or more functional groups of the substrate of<br>
the invention do not disturb the 3-D structure of the hydrolase (wild-type or<br>
mutant). For example, one example of a substrate of the invention for a<br>
dehalogenase includes a reactive group such as (CtfekoX where X is a halide<br>
and a functional group such as tetramethyh-hodamine (TAMRA), e.g., TAMRACi4H24O4-<br>
Cl.<br>
In one embodiment, a linker is preferably 12 to 30 atoms in length. The<br>
linker may not always be present in a substrate of the invention, however, in<br>
some embodiments, the physical separation of the reactive group and the<br>
functional group may be needed so that the reactive group can interact with the<br>
reactive residue in the mutant hydrolase to form a covalent bond. Preferably,<br>
when present, the linker does not substantially alter, e.g., impair, the specificity<br>
or reactivity of a substrate having the linker with the wild-type or mutant<br>
hydrolase relative to the specificity or reactivity of a corresponding substrate<br>
which lacks the linker with the wild-type or mutant hydrolase. Further, the<br>
presence of the linker preferably does not substantially alter, e.g., impair, one or<br>
more properties, e.g., the function, of the functional group.<br>
Thus, the invention provides a compound of formula (I): R-linker-A-X,<br>
wherein R is one or more functional groups, wherein the linker is a multiatom<br>
straight or branched chain including C, N, S, or O, wherein A-X is a substrate for<br>
a dehalogenase, and wherein X is a halogen. In one embodiment, an alkylhalide<br>
is covalently attached to a linker, L, which is a group or groups that covalently<br>
attach one or more functional groups to form a substrate for a dehalogenase. As<br>
described herein, a mutant dehalogenase, DhaA.H272F, was bound to substrates<br>
for DhaA which included 5-(and 6-) carboxy fluorescein (FAM), e.g., FAM-<br>
4-Cl, TAMRA, e.g., TAMRA-Cn^CU-Cl, andbiotin, e.g., biotin-<br>
Cl, and there was no significant quenching effect of this binding on<br>
FAM or TAMRA fluorescence or on biotin binding to streptavidin. As also<br>
described herein, a mutant dehalogenase, e.g., DhaA.D106C and DhaA.D106E<br>
as well as DhaA.D106C:H272F and DhaA.D106E:H272F, bound FAMCi4H2404-<br>
Cl and/or TAMRA-Ci4H24O4-Cl. In one embodiment, the substrate is<br>
R-(CH2)2O(CH2)2O(CH2)2<xch2 wherein r is a functional group. to></xch2>
prepare such a substrate, a functional group may be reacted with a molecule such<br>
as NH(CH2)20(CH2)20(CH2)20(CH2)6C1.<br>
In one embodiment, substrates of the invention are permeable to the<br>
plasma membranes of cells. For instance, as described herein the plasma<br>
membranes of prokaryotic (E. coif) and eukaryotic (CHO-K1) cells were<br>
permeable to TAMRA-Ci4H2404-Cl and biotin-Cigl^CVCl and, these<br>
substrates were rapidly and efficiently loaded into and washed out of cells in the<br>
absence of a mutant hydrolase. hi the presence of a mutant hydrolase, at least a<br>
portion of the substrate was prevented from being washed out of the cells. Thus,<br>
the bound portion of the substrate can serve as a marker or as a means to capture<br>
the mutant hydrolase or a fusion thereof.<br>
The invention further provides methods for preparing a substrate for a<br>
hydrolase which substrate is modified to include one or more functional groups.<br>
Exemplary functional groups for use in the invention include, but are not limited<br>
to, an amino acid, protein, e.g., enzyme, antibody or other immunogenic protein,<br>
a radionuclide, a nucleic acid molecule, a drug, a lipid, biotin, avidin,<br>
streptavidin, a magnetic bead, a solid support, an electron opaque molecule,<br>
chromophore, MRI contrast agent, a dye, e.g., a xanthene dye, a calcium<br>
sensitive dye, e.g., l-[2-amino-5-(2,7-dichloro-6-hydroxy-3-oxy-9-xanthenyl)-<br>
phenoxy]-2-(2'-amino-5'-methylphenoxy)ethane-N,N,N',N'-tetraaceticacid<br>
(Fluo-3), a sodium sensitive dye, e.g., 1,3-benzenedicarboxylic acid, 4,4'-<br>
[ 1,4,10,13 -tetraoxa-7,16-diazacyclooctadecane-7,16-diylbis(5-methoxy-6,2-<br>
benzofurandiyl)]bis (PBFI), a NO sensitive dye, e.g., 4-amino-5-methylamino-<br>
2',7'-difluorescein, or other fluorophore. hi one embodiment, the functional<br>
group is an immunogenic molecule, i.e., one which is bound by antibodies<br>
specific for that molecule. In one embodiment, the functional group is not a<br>
radionuclide.<br>
The invention also includes a mutant hydrolase which comprises at least<br>
one amino acid substitution relative to a corresponding wild-type hydrolase,<br>
which substitutions) renders the mutant hydrolase capable of forming a bond,<br>
e.g., a covalent bond with a substrate for the corresponding hydrolase, e.g., a<br>
substrate of the invention, which is more stable than the bond formed between a<br>
corresponding wild-type hydrolase and the substrate.<br>
In one embodiment, the mutant hydrolase of the invention comprises at<br>
least one ammo acid substitution hi a residue which, in the wild-type hydrolase,<br>
is associated with activating a water molecule, e.g., a residue in a catalytic triad<br>
or an auxiliary residue, wherein the activated water molecule cleaves the bond<br>
formed between a catalytic residue in the wild-type hydrolase and a substrate of<br>
the hydrolase. As used herein, an "auxiliary residue" is a residue which alters<br>
the activity of another residue, e.g., it enhances the activity of a residue that<br>
activates a water molecule. Residues which activate water within the scope of<br>
the invention include but are not limited to those involved in acid-base catalysis,<br>
6<br>
for instance, histidine, aspartic acid and glutamic acid. In another embodiment,<br>
the mutant hydrolase of the invention comprises at least one amino acid<br>
substitution in a residue which, in the wild-type hydrolase, forms an ester<br>
intermediate by nucleophilic attack of a substrate for the hydrolase.<br>
For example, wild-type dehalogenase DhaA cleaves carbon-halogen<br>
bonds in halogenated hydrocarbons (HaloC3-HaloCio). The catalytic center of<br>
DhaA is a classic catalytic triad including a nucleophile, an acid and a histidine<br>
residue. The amino acids in the triad are located deep inside the catalytic pocket<br>
of DhaA (about 10 A long and about 20 A2 'in cross section). The halogen atom<br>
in a halogenated substrate for DhaA, for instance, the chlorine atom of a Clalkane<br>
substrate, is positioned in close proximity to the catalytic center of DhaA.<br>
DhaA binds the substrate, likely forms an ES complex, and an ester intermediate<br>
is formed by nucleophilic attack of the substrate by Aspl06 (the numbering is<br>
based on the protein sequence of DhaA) of DhaA (Figure 1). His272 of DhaA<br>
then activates water and the activated water hydrolyzes the intermediate,<br>
releasing product from the catalytic center. As described herein, mutant DhaAs,<br>
e.g., a DhaA.H272F mutant, which likely retains the 3-D structure based on a<br>
computer modeling study and basic physico-chemical characteristics of wildtype<br>
DhaA (DhaA.WT), were not capable of hydrolyzing one or more substrates<br>
of the wild-type enzyme, e.g., for Cl-alkanes, releasing the corresponding<br>
alcohol released by the wild-type enzyme. As further described herein, mutant<br>
serine beta-lactamases, e.g., a blaZ.E166D mutant, a blaZ.N170Q mutant and a<br>
blaZ.El 66D:N170Q mutant, were not capable of hydrolyzing one or more<br>
substrates of a wild-type serine beta-lactamase.<br>
Thus, in one embodiment of the invention, a mutant hydrolase is a<br>
mutant dehalogenase comprising at least one amino acid substitution in a residue<br>
which, in the wild-type dehalogenase, is associated with activating a water<br>
molecule, e.g., a residue in a catalytic triad or an auxiliary residue, wherein the<br>
activated water molecule cleaves the bond formed between a catalytic residue in<br>
the wild-type dehalogenase and a substrate of the dehalogenase. In one<br>
embodiment, at least one substitution is in a residue corresponding to residue<br>
272 in DhaA from Rhodococcus rhodochrous. A "corresponding residue" is a<br>
residue which has the same activity (function) in one wild-type protein relative<br>
to a reference wild-type protein and optionally is in the same relative position<br>
when the primary sequences of the two proteins are aligned. For example, a<br>
residue which forms part of a catalytic triad and activates a water molecule in<br>
one enzyme may be residue 272 in that enzyme, which residue 272 corresponds<br>
to residue 73 in another enzyme, wherein residue 73 forms part of a catalytic<br>
triad and activates a water molecule. Thus, in one embodiment, a mutant<br>
dehalogenase of the invention has a phenylalanine residue at a position<br>
corresponding to residue 272 in DhaA from Rhodococcus rhodochrous. In<br>
another embodiment of the invention, a mutant hydrolase is a mutant<br>
dehalogenase comprising at least one amino acid substitution in a residue<br>
corresponding to residue 106 in DhaA from Rhodococcus rhodochrous. For<br>
example, a mutant dehalogenase of the invention has a cysteine or a glutamate<br>
residue at a position corresponding to residue 106 in DhaA from Rhodococcus<br>
rhodochrous. In a further embodiment, the mutant hydrolase is a mutant<br>
dehalogenase comprising at least two amino acid substitutions, one in a residue<br>
corresponding to residue 106 and one in a residue corresponding to residue 272<br>
in DhaA from Rhodococcus rhodochrous. In yet a further embodiment, the<br>
mutant hydrolase is a mutant serine beta-lactamase comprising at least one<br>
amino acid substitution in a residue corresponding to residue 166 or residue 170<br>
in a serine beta-lactamase of Staphylococcus aureus PCI.<br>
The mutant hydrolase may be a fusion protein, e.g., a fusion protein<br>
expressed from a recombinant DNA which encodes the mutant hydrolase and at<br>
least one protein of interest or a fusion protein formed by chemical synthesis.<br>
For instance, the fusion protein may comprise a mutant hydrolase and an enzyme<br>
of interest, e.g., luciferase, RNasin or RNase, and/or a channel protein, a<br>
receptor, a membrane protein, a cytosolic protein, a nuclear protein, a structural<br>
protein, a phosphoprotein, a kinase, a signaling protein, a metabolic protein, a<br>
mitochondria! protein, a receptor associated protein, a fluorescent protein, an<br>
enzyme substrate, a transcription factor, a transporter protein and/or a targeting<br>
sequence, e.g., a myristilation sequence, a mitochondrial localization sequence,<br>
or a nuclear localization sequence, that directs the mutant hydrolase, for<br>
example, a fusion protein, to a particular location. The protein of interest may be<br>
fused to the N-terminus or the C-terminus of the mutant hydrolase. In one<br>
embodiment, the fusion protein comprises a protein of interest at the N-terminus,<br>
and another protein, e.g., a different protein, at the C-terminus, of the mutant<br>
hydrolase. For example, the protein of interest may be a fluorescent protein or<br>
an antibody. Optionally, the proteins in the fusion are separated by a connector<br>
sequence, e.g., preferably one having at least 2 amino acid residues, such as one<br>
having 13 to 17 amino acid residues. The presence of a connector sequence in a<br>
fusion protein of the invention does not substantially alter the function of either<br>
protein in the fusion relative to the function of each individual protein. Thus, for<br>
a fusion of a mutant dehalogenase and Renilla luciferase, the presence of a<br>
connector sequence does not substantially alter the stability of the bond formed<br>
between the mutant dehalogenase and a substrate therefor or the activity of the<br>
luciferase. For any particular combination of proteins in a fusion, a wide variety<br>
of connector sequences may be employed. In one embodiment, the connector<br>
sequence is a sequence recognized by an enzyme, e.g., a cleavable sequence.<br>
For instance, the connector sequence may be one recognized by a caspase, e.g.,<br>
DEVD (SEQ ID NO:64), or is a photocleavable sequence.<br>
In one embodiment, the fusion protein may comprise a protein of interest<br>
at the N-terminus and, preferably, a different protein of interest at the C-terminus<br>
of the mutant hydrolase. As described herein, fusions of a mutant DhaA with<br>
GST (at the N-terminus), a Flag sequence (at the C-terminus) and Renilla<br>
luciferase (at the N-terminus or C-terminus) had no detectable effect on bond<br>
formation between the mutant DhaA and a substrate for wild-type DhaA which<br>
includes a functional group. Moreover, a fusion of a Flag sequence and<br>
DhaA.H272F could be attached to a solid support via a streptavidin-biotin-<br>
CigH32O4-DhaA.H272F bridge (an SFlag-ELISA experiment). Further, a fusion<br>
of Renilla luciferase (RXuc) and DhaA.H272F could be attached to Magnesilโข<br>
particles coated with a substrate for wild-type DhaA which includes a functional<br>
group. In addition, the attached fusion comprising R.Luc was shown to be<br>
enzymatically active.<br>
Exemplary proteins of interest include, but are not limited to, an<br>
imrmmogenic protein, fluorescent protein, selectable marker protein, membrane<br>
protein, cytosolic protein, nuclear protein, structural protein, enzyme, e.g.,<br>
RNase, enzyme substrate, receptor protein, transporter protein, transcription<br>
factor, channel protein, e.g., ion channel protein, phospho-protein, kinase,<br>
signaling protein, metabolic protein, mitochondria! protein, receptor associated<br>
protein, nucleic acid binding protein, extracellular matrix protein, secreted<br>
protein, receptor ligand, serum protein, or a protein with reactive cysteines.<br>
The invention also includes compositions and kits comprising a substrate<br>
for a hydrolase which includes a linker, a substrate for a hydrolase which<br>
includes one or more functional groups*and optionally a linker, a linker which<br>
includes one or more functional groups, a substrate for a hydrolase which lacks<br>
one or more functional groups and optionally includes a linker, a linker, or a<br>
mutant hydrolase, or any combination thereof. For example, the invention<br>
includes a solid support comprising a substrate of the invention, a kit comprising<br>
a substrate of the invention, a kit comprising a vector encoding a dehalogenase<br>
of the invention, or a kit comprising a vector encoding a serine beta-lactamase of<br>
the invention.<br>
Also provided is an isolated nucleic acid molecule (polynucleotide)<br>
comprising a nucleic acid sequence encoding a hydrolase. hi one embodiment,<br>
the isolated nucleic acid molecule comprises a nucleic acid sequence which is<br>
optimized for expression in at least one selected host. Optimized sequences<br>
include sequences which are codon optimized, i.e., codons which are employed<br>
more frequently in one organism relative to another organism, e.g., a distantly<br>
related organism, as well as modifications to add or modify Kozak sequences<br>
and/or introns, and/or to remove undesirable sequences, for instance, potential<br>
transcription factor binding sites. In one embodiment, the polynucleotide<br>
includes a nucleic acid sequence encoding a dehalogenase, which nucleic acid<br>
sequence is optimized for expression is a selected host cell. In one embodiment,<br>
the optimized polynucleotide no longer hybridizes to the corresponding nonoptimized<br>
sequence, e.g., does not hybridize to the non-optimized sequence<br>
under medium or high stringency conditions. In another embodiment, the<br>
polynucleotide has less than 90%, e.g., less than 80%, nucleic acid sequence<br>
identity to the corresponding non-optimized sequence and optionally encodes a<br>
polypeptide having at least 80%, e.g., at least 85%, 90% or more, ammo acid<br>
sequence identity with the polypeptide encoded by the non-optimized sequence.<br>
Constructs, e.g., expression cassettes, and vectors comprising the isolated<br>
nucleic acid molecule, as well as kits comprising the isolated nucleic acid<br>
molecule, construct or vector are also provided.<br>
Further provided is a method of expressing a mutant hydrolase of the<br>
invention. The method comprises introducing to a host cell a recombinant<br>
nucleic acid molecule encoding a mutant hydrolase of the invention so as to<br>
express the mutant hydrolase. In one embodiment, the mutant hydrolase may be<br>
isolated from the cell. The mutant hydrolase may be expressed transiently or<br>
stably, constitutively or under tissue-specific or drug-regulated promoters, and<br>
the like. Also provided is an isolated host cell comprising a recombinant nucleic<br>
acid molecule encoding a mutant hydrolase of the invention.<br>
In one embodiment, the invention provides a method to detect or<br>
determine the presence or amount of a mutant hydrolase. The method includes<br>
contacting a mutant hydrolase with a hydrolase substrate which comprises one or<br>
more functional groups. The mutant hydrolase comprises at least one amino acid<br>
substitution relative to a corresponding wild-type hydrolase, wherein the at least<br>
one amino acid substitution results in the mutant hydrolase forming a bond with<br>
the substrate which is more stable than the bond formed between the<br>
corresponding wild-type hydrolase and the substrate, and wherein the at least<br>
one amino acid substitution in the mutant hydrolase is a substitution at an amino<br>
acid residue in the corresponding wild-type hydrolase that is associated with<br>
activating a water molecule which cleaves the bond formed between the<br>
corresponding wild-type hydrolase and the substrate or at an amino acid residue<br>
in the corresponding wild-type hydrolase that forms an ester intermediate with<br>
the substrate. The presence or amount of me functional group is detected or<br>
determined, thereby detecting or determining the presence or amount of the<br>
mutant hydrolase. In one embodiment, the mutant hydrolase is in or on the<br>
surface of a cell. In another embodiment, the mutant hydrolase is in a cell lysate.<br>
Also provided are methods of using a mutant hydrolase and a substrate<br>
for a corresponding hydrolase which includes one or more functional groups,<br>
e.g., to isolate a molecule or to detect or determine the presence or amount of,<br>
location, e.g., intracellular, subcellular or extracellular location, or movement of<br>
certain molecules in cells. In one embodiment, a method to isolate a molecule of<br>
interest in a sample is provided. The method includes contacting a sample with a<br>
fusion protein comprising a mutant hydrolase and a protein which binds a<br>
molecule of interest with a hydrolase substrate which comprises one or more<br>
functional groups. The mutant hydrolase comprises at least one arnino acid<br>
substitution relative to a corresponding wild-type hydrolase, wherein the at least<br>
one amino acid substitution results in the mutant hydrolase forming a bond with<br>
the substrate which is more stable than the bond formed between the<br>
corresponding wild-type hydrolase and the substrate, and wherein the at least<br>
one amino acid substitution in the mutant hydrolase is a substitution at an amino<br>
acid residue in the corresponding wild-type hydrolase that is associated with<br>
activating a water molecule which cleaves the bond formed between the<br>
corresponding wild-type hydrolase and the substrate or at an amino acid residue<br>
in the corresponding wild-type hydrolase that forms an ester intermediate with<br>
the substrate. In one embodiment, at least one functional group is a solid support<br>
or a molecule which binds to a solid support. In one embodiment, the sample<br>
contains intact cells while in another embodiment, the sample is a cell lysate or<br>
subcellular fraction. Then the molecule of interest is isolated.<br>
For example, the invention includes method to isolate a protein of<br>
interest. The method includes contacting a fusion protein comprising a mutant<br>
hydrolase and a protein of interest with a hydrolase substrate which comprises at<br>
least one functional group. The mutant hydrolase comprises at least one amino<br>
acid substitution relative to a corresponding wild-type hydrolase, wherein the at<br>
least one amino acid substitution results in the mutant hydrolase forming a bond<br>
with the substrate which is more stable than the bond formed between the wildtype<br>
hydrolase and the substrate, and wherein the at least one amino acid<br>
substitution in the mutant hydrolase is a substitution at an amino acid residue in<br>
the wild-type hydrolase that is associated with activating a water molecule which<br>
cleaves a bond formed between the wild-type hydrolase and the substrate or at an<br>
amino acid residue in the wild-type hydrolase that forms an ester intermediate<br>
with the substrate, hi one embodiment, at least one functional group is a solid<br>
support or a molecule which binds to a solid support Then the protein of<br>
interest is isolated.<br>
In another embodiment, the invention includes a method to identify an<br>
agent that alters the interaction of a protein of interest with a molecule suspected<br>
of interacting with the protein of interest. The method includes contacting at<br>
least one agent with the molecule suspected of interacting with the protein of<br>
interest, a fusion protein comprising mutant hydrolase and the protein of interest,<br>
and a hydrolase substrate which comprises one or more functional groups. The<br>
mutant hydrolase comprises at least one amino acid substitution relative to a<br>
corresponding wild-type hydrolase, wherein the at least one amino acid<br>
substitution results in the mutant hydrolase forming a bond with the substrate<br>
which is more stable than the bond formed between the corresponding wild-type<br>
hydrolase and the substrate, and wherein the at least one amino acid substitution<br>
in the mutant hydrolase is a substitution at an amino acid residue in the<br>
corresponding wild-type hydrolase that is associated with activating a water<br>
molecule which cleaves a bond formed between the corresponding wild-type<br>
hydrolase and the substrate at an amino acid residue in the wild-type hydrolase<br>
that forms an ester intermediate with the substrate. In one embodiment at least<br>
one functional group is a solid support or a molecule which binds to a solid<br>
support. Then it is determined whether the agent alters the interaction between<br>
the protein of interest and the molecule suspected of interacting with the protein<br>
of interest.<br>
Moreover, a substrate of the invention bound to a solid support or a<br>
mutant hydrolase bound to a solid support may be used to generate protein<br>
arrays, cell arrays, vesicle/organelle arrays and cell membrane arrays.<br>
The invention thus provides methods to monitor the expression, location<br>
and/or movement (trafficking) of proteins in a cell as well as to monitor changes<br>
in microenvironments within a cell. In one embodiment, the use of a mutant<br>
hydrolase and a substrate of the invention permits functional analysis of proteins,<br>
e.g., ion channels. In another embodiment, the use of two pairs of a mutant<br>
hydrolase/substrate permits multiplexing, simultaneous detection, and FRET- or<br>
BRET-based assays. For example, mutant dehalogenases with substitutions at<br>
different residues of a catalytic triad may each preferentially bind certain<br>
substrates of the invention but not others or a mutant dehalogenase and a mutant<br>
beta-lactamase may be employed with their respective substrates, thus permitting<br>
multiplexing. Other applications include capturing the stable complex which<br>
results from contacting the mutant hydrolase with a corresponding substrate of<br>
the invention, on a solid substrate for analytical or industrial purposes (e.g., to<br>
study kinetic parameters of the tethered enzyme, to generate enzyme<br>
chains/arrays, to metabolize industrial components, and the like), to detect<br>
protein-protein interactions, to determine the effect of different<br>
compounds/drugs on an interaction between a fusion protein comprising a<br>
protein of interest and a mutant hydrolase with other molecules, to isolate or<br>
purify molecules which bind to a protein of interest fused to the mutant<br>
hydrolase, or to isolate or purify cells, organelles or fragments thereof. For<br>
example, a protein of interest may be fused to a mutant hydrolase and then<br>
linked to a solid support via the specific interaction of a functional group which<br>
is a ligand for an acceptor group and is present in a substrate of the invention,<br>
with an acceptor group present on the solid support. Such a substrate may be<br>
contacted with the fusion protein prior to contact with the solid support,<br>
contacted with the solid support prior to contact with the fusion protein, or<br>
simultaneously contacted with the fusion protein and the solid support. Such a<br>
system permits the resulting complex to be employed to detect or isolate<br>
molecules which bind to the protein of interest. The binding molecule may be a<br>
protein, e.g., a fusion of the binding protein and a functional group, e.g., GFP,<br>
luciferase, an antibody, e.g., one conjugated to horseradish peroxidase (HRP),<br>
alkaline phosphatase (AP) or a fluorophore.<br>
To isolate, sort or purify cells, the mutant hydrolase may be expressed on<br>
the outside surface of cells (e.g., via a fusion with a plasma membrane protein).<br>
To isolate, purify or separate organelles, the mutant hydrolase is expressed on<br>
the cytosolic surface of the organelle of interest. In another embodiment, to<br>
create an optimal platform for growing different cells, the mutant hydrolase is<br>
fused with an extracellular matrix component or an outer membrane protein and<br>
tethered to a three-dimensional cell culture or a platform for tissue engineering.<br>
As an example, primary neurons or embryonic stem cells may be grown on the<br>
platform to form a feeder layer.<br>
Other applications include detecting or labeling cells. Thus, the use of a<br>
mutant hydrolase and a corresponding substrate of the invention permits the<br>
detection of cells, for instance, to detect cell migration in vitro or in vivo after<br>
implantation or injection into animals (e.g., angiogenesis/chemotaxis assays,<br>
migration of implanted neurons, normal, malignant, or recombinantly modified<br>
cells implanted/injected into animals, and the like), and live cell imaging<br>
followed by immunocytochemistry. In another embodiment, the invention<br>
provides a method to label newly synthesized proteins. For example, cells<br>
comprising a vector which expresses a mutant hydrolase of the invention or a<br>
fusion thereof, are contacted with a substrate for the hydrolase which lacks a<br>
functional group. Cells are then contacted with an agent, e.g., an inducer of gene<br>
expression, and a substrate for the hydrolase which contains one or more<br>
functional groups. The presence, amount or location of the mutant hydrolase or<br>
fusion thereof is thenidetected or determined. The presence, amount or location<br>
of the mutant hydrolase or fusion thereof is due to newly synthesized mutant<br>
hydrolase or a fusion thereof. Alternatively, cells comprising a vector which<br>
expresses a mutant hydrolase of the invention or a fusion thereof, are contacted<br>
with a substrate for the hydrolase having a functional group, e.g., a green<br>
fluorophore, then contacted with an agent and a substrate havinga different<br>
functional group, e.g., a red fluorophore. In one embodiment, the mutant<br>
hydrolase is fused to a membrane localization signal and so can be employed to<br>
monitor events in or near the membrane.<br>
Accordingly, the invention provides a method to label a cell. The<br>
method includes contacting a cell comprising a mutant hydrolase with a<br>
hydrolase substrate which comprises one or more functional groups. The mutant<br>
hydrolase comprises at least one amino acid substitution relative to a<br>
corresponding wild-type hydrolase, wherein the at least one amino acid<br>
substitution results in the mutant hydrolase forming a bond with the substrate<br>
which is more stable than the bond formed between the corresponding wild-type<br>
hydrolase and the substrate, and wherein the at least one amino acid substitution<br>
in the mutant hydrolase is a substitution at an amino acid residue in the<br>
corresponding wild-type hydrolase that is associated with activating a water<br>
molecule which cleaves a bond formed between the corresponding wild-type<br>
hydrolase and the substrate or at an amino acid residue in the corresponding<br>
wild-type hydrolase that forms an ester intermediate with the substrate. Then the<br>
presence or amount of the functional group is detected or determined.<br>
Cells expressing selectable marker proteins, such as ones encoding<br>
resistance to neomycin, hygromycin, or puromycin, are used to stably transform<br>
cells with foreign DNA. It may be desirable to observe which cells contain<br>
selectable marker proteins as well as fluorescently labeled molecules. For<br>
instance, it may be preferable to label the selectable marker protein with a<br>
fluorescent molecule that is added exogenously to living cells. By this method,<br>
the selectable marker protein becomes visible when only when needed by<br>
addition of the fluorophore, and the fluorescence will subsequently be lost when<br>
selectable marker proteins are naturally regenerated through cellular metabolism.<br>
Thus, in one embodiment, the invention provides a method for labeling a cell<br>
which expresses a selectable marker protein. The method includes providing a<br>
cell comprising an expression cassette comprising a nucleic acid sequence<br>
encoding a fusion protein. The fusion protein comprises a selectable marker<br>
protein, e.g., one which confers resistance to at least one antibiotic, and a second<br>
protein that is capable of stably and optionally irreversibly binding a substrate or<br>
a portion thereof which includes an optically detectable molecule. For instance,<br>
the protein may be an alkyl transferase which irreversibly transfers an alkyl<br>
group and an optically detectable molecule from a substrate to itself, thereby<br>
labeling the alkyl transferase, e.g., an alkyl transferase such as O6-alkylguanine<br>
DNA alkyltransferase. Exemplary proteins useful in this embodiment of the<br>
invention include, but are not limited to, alkyl transferases, peptidyl glycinealpha-<br>
amidating monoxygenases, type I topoisomerases, hydrolases, e.g., serine<br>
and epoxide hydrolases as well as the mutant hydrolases described herein,<br>
aminotransferases, cytochrome P4SO monooxygenases, acetyl transferases,<br>
decarboxylases, oxidases, e.g., monoamine oxidases, reductases, e.g.,<br>
ribonucleotide reductase, synthetases, e.g., cyclic ADP ribose synthetase or<br>
thymidylate synthetase, dehydrogenases, e.g., aldehyde dehydrogenase,<br>
syntheses, e.g., nitric oxide synthase (NOS), lactamases, cystathionine gammalyases,<br>
peptidases, e.g., carboxypeptidase A, aromatase, proteases, e.g., serine<br>
protease, xylanases, glucosidases, mannosidases, and demethylases and other<br>
proteins, including wild-type proteins, which form an irreversible or otherwise<br>
stable bond with one or more substrates, e.g., enzymes which are capable of<br>
mechanism-based inactivation. Thus, in this embodiment, a stable bond, i.e.,<br>
one which is formed between a substrate and a wild-type or mutant enzyme, has<br>
a t&gt;A of at least 30 minutes and preferably at least 4 hours, and up to at least 10<br>
hours, and is resistant to disruption by washing, protein denaturants, and/or high<br>
temperatures, e.g., the bond is stable to boiling in SDS.<br>
The cell which expresses the fusion protein is contacted with the<br>
substrate so as to label the cell. In one embodiment, the cell is fixed prior to<br>
contact with the substrate. In another embodiment, the substrate and fixative are<br>
contacted with the cell at the same time. In yet another embodiment, the fixative<br>
is added to the cell after the cell is contacted with the substrate. In one<br>
embodiment, the fusion protein forms an ester bond with the substrate. In<br>
another embodiment, the fusion protein forms a thioester bond with the<br>
substrate. Also provided is a fusion gene encoding the fusion protein, and a cell<br>
which expresses the fusion protein.<br>
When performing image analysis on a cell, it may be desirable to fix the<br>
cell with a preservative (fixative) such as paraformaldehyde, acetone or<br>
methanol which generally maintains most features of cellular structure. Such<br>
fixed cells are then often analyzed by adding fluorescent stains or fluorescently<br>
labeled antibodies to reveal specific structures within the cells. Another method<br>
to fluorescently label cells is to express a fluorescent protein, e.g., GFP, in cells<br>
prior to fixation. Unfortunately, the efficient fluorescence of these proteins is<br>
dependent on protein structure, which can be disrupted by preservatives, thus<br>
decreasing the efficiency of imaging in those cells.<br>
Accordingly, the invention provides a method for labeling a cell with a<br>
functional group, e.g., fluorophore. The method includes providing a cell which<br>
expresses a mutant hydrolase of the invention or a fusion thereof, and contacting<br>
the cell with a hydrolase substrate which includes at least one functional group.<br>
In one embodiment, the cell is fixed prior to contact with the substrate. In<br>
another embodiment, the substrate and fixative are contacted with the cell at the<br>
same time. In yet another embodiment, the fixative is added to the cell after the<br>
cell is contacted with the substrate. Then the presence or location of the mutant<br>
hydrolase, or fusion thereof, in the cell is detected or determined. In one<br>
embodiment, the mutant hydrolase forms an ester bond with the substrate, while<br>
in another embodiment, the mutant hydrolase forms a thioester bond with the<br>
substrate.<br>
The invention also provides processes and intermediates disclosed herein<br>
that are useful for preparing compounds, compositions, nucleic acids, proteins,<br>
or other materials of the invention.<br>
Brief Detciiptioa of the Figures<br>
Figure 1 is a schematic of a reaction in the catalytic triad of Rhodococcus<br>
rhodochrous dehalogenase with an alkylhalide substrate.<br>
Figure 2 shows a three-dimensional model of a wild-type DhaA<br>
Rhodococcus rhodochrous dehalogenase and four mutant DhaAs (H283Q, G, A<br>
or F). A cyan ribbon is a 3-D model of the DhaA.WT based on the crystal<br>
structure of this protein (Newman et al., 1999) (panel A). The purple ribbon is a<br>
3-D model of the H272Q, H272G and K272A mutants (panel A), or a 3-D model<br>
of the H272F mutant (panel B). Three-dimensional models were generated by<br>
calculating a Molecular Probability Density Function followed by several<br>
optimization steps including Restrained Stimulated Annealing Molecular<br>
Dynamics (MD) scheme. 3-D modeling was done on Silicon Graphics<br>
computer-station using software Insightll (USA).<br>
Figure 3 shows the purification of wild-type and mutant DhaA proteins.<br>
GST-DhaA.WT-Flag (odd numbered lanes) and GST-DhaA.H272F-Flag (even<br>
numbered lanes) fusion proteins were found to be soluble and efficiently purified<br>
on GSS-Sepharose 4FF (lanes 3 and 4-crude E. coli supernatant; lanes 5 and 6-<br>
washes; lanes 7 through 10-purified proteins). Treatment of the fusion proteins<br>
with Factor Xa led to the formation of two proteins, GST and DhaA (WT or<br>
mutant; lanes 11 and 12, respectively). Moreover, GST was efficiently removed<br>
on GSS-Sepharose 4FF (WT or mutant; lanes 13 and 14, respectively). All<br>
proteins had the predicted molecular weight.<br>
Figure 4 illustrates the hydrolysis of 1-Cl-butane by wild-type DhaA and<br>
mutant DhaAs.<br>
Figure 5 shows precipitation of DhaA. WT and DhaA.H272F/A/G/ Q<br>
mutants with various concentrations of (NEU^SO* Lanes 1,5, and 9,0 %<br>
(NH4)2SO4; lanes 2,6, and 10,10 % (NH4)2SO4; lanes 3,7, and 11,10-45 %<br>
 (NH4)2SO4; and lanes 4, 8, and 12,45-70 % (NHOaSC^. Panel A: lanes 1-4,<br>
DhaA.WT; lanes 5-8, DhaA.H272G; and lanes 9- 12, DhaA.H272Q. Panel B:<br>
lanes 1-^, DhaA.WT; lanes 5-8, DhaA.H272F; and lanes 9-12, DhaA.H272A.<br>
Figure 6 depicts the substrate specificity of wild-type DhaA. Using a<br>
phenol red-based assay (Essg), the initial rate of the reaction was determined<br>
during the first 60 seconds after enzyme addition by four 15 second readings.<br>
Figure 7 shows substrates for DhaA which include a functional group<br>
(e.g., 5-(and 6-)-carboxyfluorescein (FAM), Anth (anthracene) or biotin) and a<br>
linker.<br>
Figure 8 A shows a HPLC separation of products of FAM-Ci4H24C&gt;4-Cl<br>
hydrolysis by wild-type DhaA.<br>
Figure 8B shows a HPLC analysis of product (as a percent of substrate)<br>
produced by wild-type DhaA hydrolysis of FAM-CuI^CVCl over time.<br>
Figure 9 shows SDS-PAGE analysis of the binding of wild-type DhaA<br>
(lanes 1, 3, and 5 in panel A and lanes 1-8 in panel B) and mutant DhaA<br>
(DhaA.H272F); (lanes 2,4, and 6 in panel A and lanes 9-14 in panel B), to<br>
TAMRA-Ci4H24O4-Cl (lanes 1 and 2 in panel A); ROX-Ci4H24O4-Cl (lanes 3<br>
and 4 in panel A); FAM-Ci4H24O4-Cl (lanes 5 and 6 in panel A); or biotin-<br>
Ci8H32O4.Cl (panel B). The concentration of biotin-Cigl^CVCl in panel B as: 0<br>
|j,M (lanes 1 and 8), 125 \M (lanes 2 and 9) 25 uM (lanes 3 and 10), 5 jiM (lanes<br>
4 and 11), 1 uM (lanes 5 and 12), 0.2 uM (lanes 6 and 13), and 0.04 uM (lanes 7<br>
and 14).<br>
Figure 10 illustrates that pretreatment of a mutant DhaA with a substrate,<br>
biotin-CigH&amp;CU-Cl, blocks binding of another substrate. DhaAWT-lanes 1 and<br>
2; DhaA.H272 mutants: F, lanes 3 and 4; G, lanes 5 and 6; A, lanes 7 and 8; and<br>
Q, lanes 9 and 10. Samples 2,4,6,8, and 10 were pretreated with biotin-<br>
C18H3204-C1.<br>
Figure 11 shows MALDI-TOF analysis of enzyme substrate complexes.<br>
Mass spectra of GST-DhaA.WT or GST-DhaA.H272F incubated with FAMC14H2404-<br>
C1.<br>
Figure 12 illustrates SDS-PAGE analysis of the binding properties of<br>
DhaA mutants with substitutions at residue 106, and DhaA mutants with<br>
substitutions at residue 106 and residue 272, to TAMRA-Ci4H24C&gt;4-Cl. 2 ug of<br>
protein and 25 \M TAMRA-Ci^foCU-Cl in 32 ul were incubated for one hour<br>
at room temperature. 10 ul of each reaction was loaded per lane. Lane 1 -<br>
DhaA.D106C; lane 2-DhaA.D106C: H272F; lane 3-DhaA.D106E; lane 4-<br>
DhaA.D106E:H272F; lane 5-DhaA.D106Q; lane 6-DhaA.D106Q:H272F; lane<br>
7-DhaA.WT; and lane 8-DhaA.H272F.. The gel was imaged with a 570 nm<br>
filter.<br>
Figure 13 depicts analysis ofRenilla luciferase activity in samples having<br>
a fusion of luciferase and a mutant DhaA tethered to a solid support (a<br>
streptavidin coated plate). Capture of the fusion was accomplished using a<br>
substrate of DhaA (i.e., biotin-CigHszCVCl). No activity was found in fractions<br>
with a fusion ofRenilla luciferase and wild-type DhaA.<br>
Figure 14 shows SDS-PAGE analysis of two-fold serial dilutions of E.<br>
coli expressing either wild-type DhaA (DhaA.WT-Flag, lanes 1-4 of each panel)<br>
or mutant DhaA.H272F (DhaA.H272F-Flag, lanes 5-7 of each panel) treated<br>
with biotin-CigH32O4-Cl (panel A) or TAMRA-CuKfoCVCl (panel B) in vivo.<br>
Arrows mark proteins with Mr corresponding to Mr of DhaA-Flag.<br>
Figure 15 shows the binding of TAMRA-Ci2H24O4-Cl to eukaryotic cell<br>
proteins in vivo. Two-fold serial dilutions of proteins from CHO-K1 cells<br>
expressing either DhaA.WT-Flag (lanes 1-4) or DhaA.H272F-Flag (lanes 5-8)<br>
were treated with TAMRA-CutfoCU-Cl. Arrows mark proteins with Mr<br>
corresponding to Mr of DhaA-Flag.<br>
Figure 16 illustrates the permeability of TAMRA-Ci2H24O4-Cl to CHOKl<br>
cells. CHO-K1 cells (A, bright field image) were treated with TAMRACi2H2g04-<br>
Cl (25 uM, for 5 minutes at 37ยฐC) and quickly washed with PBS (panel<br>
B). Panel C shows the cells after the washing procedure.<br>
Figure 17 shows images of cells transfected with GFP-connector-<br>
DhaA.WT-FlagorGFP-connector-DhaA.H272F-Flag. CHO-K1 cells were<br>
transfected with DNA coding GFP-connector-DhaA.WT-Flag (panels A-C) or<br>
GFP-connector-DhaA.H272F-Flag (panels D-F) and treated with TAMRACi2H2gO4-<br>
Cl. Panels A, D-bright field; panels B, E-GFP filter set; and panels C,<br>
F-TAMRA filter set.<br>
Figure 18 shows Western blot analysis of proteins from cells transfected<br>
with GFP-connector-DhaA.WT-Flag (lanes 1-4) or GFP-connector-<br>
DhaA.H272F-Flag (lanes 5-8). CHO-K1 cells were transfected with either GFPconnector-<br>
DhaA.WT-Flag or GFP-connector-DhaA.H272F-Flag and then treated<br>
with TAMRA-Ci4H24O4-Cl (25 uM) for 0,5,15 or 60 minutes, washed with<br>
PBS (4 x 1.0 ml), and collected in SDS-sample buffer. The samples were<br>
resolved on SDS-PAGE, and analyzed on a fluoroimager. Lanes 1-4, GFPconnector-<br>
DhaA.WT-Flag treated for 0,5,15, or 60 minutes, respectively.<br>
Lanes 5-8, GFP-connector-DhaA.H272F-Flag treated for 0,5,15,60 minutes,<br>
respectively. Arrows mark proteins with Mr corresponding to Mr of GFPconnector-<br>
DhaA.H272F-Flag.<br>
Figure 19 illustrates the toxicity of selected substrates (panel A, TAMRA<br>
and panel B, ROX) for CHO-K1 cells.<br>
Figure 20 illustrates a reaction scheme for a serine beta-lactamase. The<br>
reaction begins with the formation of a precovalent encounter complex (Figure<br>
19A), and moves through a high-energy acylation tetrahedral intermediate<br>
(Figure 19B) to form a transiently stable acyl-enzyme intermediate, forming an<br>
ester through the catalytic residue Ser70 (Figure 19C). Subsequently, the acylenzyme<br>
is attacked by hydrolytic water (Figure 19D) to form a high-energy<br>
deacylation intermediate (Figure 19E) (Minasov et al., 2002), which collapses to<br>
form the hydrolyzed product (Figure 19F). The product is then expelled,<br>
regenerating free enzyme.<br>
Figure 21 shows hydrolysis of FAP by GST-blaZ over time.<br>
Figure 22 shows the binding of bocellin to fusions of GST and<br>
blaZ.E166D, blaZ.N170Q or blaZ.E166D:N170Q. Lane 1-dye/no blaZ; lane 2-<br>
blaZ.WT; lane 3- blaZ.E166D; lane 4-blaZ.N170Q; and lane 5-<br>
WaZ.E166D:N170Q.<br>
Figure 23 shows the binding of CCF2 to fusions of GST and<br>
blaZ.E166D, blaZ.N170Q or blaZ.E166D:N170Q. Lane 1-dye/no blaZ; lane 2-<br>
GST-blaZ.WT; lane 3-GST-blaZ.E166D; lane 4-GST-blaZ.N170Q; and lane 5-<br>
GST- blaZ.E!66D:N170Q.<br>
Figure 24 provides fluorescence and DIG images of living CHO-K1 cells<br>
transfected with a construct encoding GFP-connector-DhaA.H272F-NLS3 and<br>
stained with TAMRA-Ci4H24O4-Cl. TAMRA filter-top left; GFP filter-top<br>
right; "A" and "B" overlaid-bottom left; overlaid image "C" and DIG image of<br>
the cell-bottom right. NLS3 = tandem repeat of a nuclear localization sequence<br>
from SV40 T antigen.<br>
Figure 25 shows fluorescence images of living CHO-K1 cells transfected<br>
with a construct encoding GFP-p-arrestin2 (left) and a construct encoding<br>
DhaA.H272F-p-arrestin2 and stained with TAMRA-Ci^fe-A (right).<br>
Figure 26 shows an SDS-PAGE analysis of DhaA expression in E. coli.<br>
Lanes: 1, Molecular weight standards; 2, Wild-type DhaA crude lysate; 3, Wildtype<br>
DhaA cell-free lysate; 4, DhaA.H272F crude lysate; 5, DhaA.H272F cellfree<br>
lysate; 6, vector control crude lysate; 7, vector control cell-free lysate; 8,<br>
DhaA.E130Q Cl mutant crude lysate; 9, DhaA.E130Q Cl mutant cell-free<br>
lysate; 10, DhaA.E130L A5 mutant crude lysate; 11, DhaA.E130L A5 mutant<br>
cell-free lysate; 12, DhaA.E130A A12 mutant crude lysate; 13, DhaA.E130A<br>
A12 mutant cell-free lysate; 14, Molecular weight standards. The arrow indicates<br>
the location of the DhaA protein, -s, lysate before centrifugation; +s, lysate after<br>
centrifugation.<br>
Figure 27 shows an immunoblot analysis of DhaA containing lysates.<br>
Lanes: 1, Wild-type DhaA crude lysate; 2, Wild-type DhaA cell-free lysate; 3,<br>
DhaA.H272F crude lysate; 4, DhaA.H272F cell-free lysate; 5, vector control<br>
crude lysate; 6, vector control cell-free lysate; 7, Molecular weight standards; 8,<br>
DhaA.E130Q Cl mutant crude lysate; 9, DhaA.E130Q Cl mutant cell-free<br>
lysate; 10, DhaA.E130L A5 mutant crude lysate; 11, DhaA.E130L A5 mutant<br>
cell-free lysate; 12, DhaA.EBOA A12 mutant crude lysate; 13, DhaA.ElSOA<br>
A12 mutant cell-free lysate; 14, Molecular weight standards. The arrow indicates<br>
the location of the DhaA protein.<br>
Figure 28 provides fluoroimage analysis of in vitro covalent alkylenzyme<br>
formation. Lanes: 1, Fluorescent molecular weight standards; 2, DhaA<br>
wild-type; 3, DhaA.H272F mutant; 4, DhaA- (vector only control); 5,<br>
DhaA.E130Q mutant; 6, DhaA.E130L mutant; 7, DhaA.E130A mutant. The<br>
arrow indicates the location of the fluorescent enzvme-alkyi covalent<br>
intermediate.<br>
Figure 29 provides fluoroimage analysis of covalent alkyl-enzyme<br>
formation in whole cells. Lanes: 1, Fluorescent molecular weight standards; 2,<br>
DhaA wild-type; 3, DhaA.H272F mutant; 4, DhaA- (vector only control); 5,<br>
DhaA.E130Q mutant; 6, DhaA.E130L mutant; 7, DhaA.E130A mutant; 8,<br>
Fluorescent molecular weight standards. The arrow indicates the location of the<br>
fluorescent enzyrne-alkyi covalent intermediate.<br>
Figures 30 A-B show Western blot analyses of DhaA-Flag captured on<br>
streptavidin (SA) coated beads. CHO-K1 cells transiently expressing<br>
DhaA.H272F-Flag were treated with (A) or without (B) biotin-CigH32O4-Cl (25<br>
uM, 0.1% DMSO, 60 minutes, 37ยฐC). Excess biotin-CigHsaCU-Cl was washed<br>
out, cells were lysed, and 10 ul of cell lysate was incubated with 5 ul of SAcoated<br>
beads (Pierce) for 60 minutes at room temperature (RT). Cell lysates<br>
(lane 1), proteins which were not bound to beads (lane 2), and proteins which<br>
were bound to beads (lane 3) were resolved on SDS-PAGE, transferred to<br>
nitrocellulose membrane, and probed with anti-Flag antibody (Sigma).<br>
Figures 30 C-D illustrate analyses of hR.Luc-DhaA captured on SA<br>
coated beads. CHO-K1 cells transiently expressing hR.Luc-connector-<br>
DhaA.H272F-Flag were treated with or without biotin-CigHsjCU-Cl (25 uM,<br>
0.1% DMSO, 60 minutes, 37ยฐC). Cells were lysed, and 10 ul of cell lysate was<br>
incubated with 5 ul of SA-coated beads (Pierce) for 60 minutes at room<br>
temperature. Unbound material was washed out, and hR.Luc activity determined<br>
using Promega's "Renilla Luciferase Assay System" (C) or captured hR.Luc<br>
analyzed by Western blot (D). C) Column 1, cells treated with biotin-CigHaaCv<br>
Cl, and excess biotin-CigHsaCU-Cl washed out; column 2, untreated cells; and<br>
column 3, cells treated withbiotin-CigHsjO^Cl without washing out excess<br>
biotin-CigHsaCVCl. D) Cell lysate (lane 1), proteins which were not bound to<br>
beads (lane 2), and proteins which were bound to beads (lane 3) were resolved<br>
on SDS-PAGE, transferred to nitrocellulose membrane, and probed with anti-<br>
R.Luc antibody (Chemicon).<br>
Detailed Description<br>
Definitions<br>
A "nucleophile" is a molecule which donates electrons.<br>
A "selectable marker protein" encodes an enzymatic activity that confers<br>
to a cell the ability to grow in medium lacking what would otherwise be an<br>
essential nutrient (e.g., the TRP1 gene hi yeast cells) or in a medium with an<br>
antibiotic or other drug, i.e., the expression of the gene encoding the selectable<br>
marker protein in a cell confers resistance to an antibiotic or drug to that cell<br>
relative to a corresponding cell without the gene. When a host cell must express<br>
a selectable marker to grow hi selective medium, the marker is said to be a<br>
positive selectable marker (e.g., antibiotic resistance genes which confer the<br>
ability to grow in the presence of the appropriate antibiotic). Selectable markers<br>
can also be used to select against host cells containing a particular gene (e.g., the<br>
sacB gene which, if expressed, kills the bacterial host cells grown in medium<br>
containing 5% sucrose); selectable markers used in this manner are referred to as<br>
negative selectable markers or counter-selectable markers. Common selectable<br>
marker gene sequences include those for resistance to antibiotics such as<br>
ampicillin, tetracycline, kanamycin, puromycin, bleomycin, streptomycin,<br>
hygromycin, neomycin, Zeocinโข, and the like. Selectable auxotrophic gene<br>
sequences include, for example, hisD, which allows growth in histidine free<br>
media in the presence of histidinol. Suitable selectable marker genes include a<br>
bleomycin-resistance gene, a metallothionein gene, a hygromycin Bphosphotransferase<br>
gene, the AURI gene, an adenosine deaminase gene, an<br>
aminoglycoside phosphotransferase gene, a dihydrofolate reductase gene, a<br>
thymidine kinase gene, a xanthine-guanine phosphoribosyltransferase gene, and<br>
the like.<br>
A "nucleic acid", as used herein, is a covalently linked sequence of<br>
nucleotides in which the 3' position of the pentose of one nucleotide is joined by<br>
a phosphodiester group to the 5' position of the pentose of the next, and in which<br>
the nucleotide residues (bases) are linked in specific sequence, i.e., a linear order<br>
of nucleotides. A "polynucleotide", as used herein, is a nucleic acid containing a<br>
sequence mat is greater than about 100 nucleotides in length. An<br>
"oligonucleotide" or "primer", as used herein, is a short polynucleotide or a<br>
portion of a polynucleotide. The term "oligonucleotide" or "oligo" as used<br>
herein is defined as a molecule comprised of 2 or more deoxyribonucleotides or<br>
ribonucleotides, preferably more than 3, and usually more than 10, but less than<br>
250, preferably less than 200, deoxyribonucleotides or ribonucleotides. The<br>
oligonucleotide may be generated in any manner, including chemical synthesis,<br>
DNA replication, amplification, e.g., polymerase chain reaction (PCR), reverse<br>
transcription (RT), or a combination thereof. A "primer" is an oligonucleotide<br>
which is capable of acting as a point of initiation for nucleic acid synthesis when<br>
placed under conditions hi which primer extension is initiated. A primer is<br>
selected to have on its 3' end a region that is substantially complementary to a<br>
specific sequence of the target (template). A primer must be sufficiently<br>
complementary to hybridize with a target for primer elongation to occur. A<br>
primer sequence need not reflect the exact sequence of the target. For example, a<br>
non-complementary nucleotide fragment may be attached to the 5' end of the<br>
primer, with the remainder of the primer sequence being substantially<br>
complementary to the target. Non-complementary bases or longer sequences can<br>
be interspersed into the primer provided that the primer sequence has sufficient<br>
complementarity with the sequence of the target to hybridize and thereby form a<br>
complex for synthesis of the extension product of the primer. Primers matching<br>
or complementary to a gene sequence may be used in amplification reactions,<br>
RT-PCR and the like.<br>
Nucleic acid molecules are said to have a "S'-terminus" (5' end) and a<br>
"3'-terminus" (3' end) because nucleic acid phosphodiester linkages occur to the<br>
5' carbon and 3' carbon of the pentose ring of the substituent mononucleotides.<br>
The end of a polynucleotide at which a new linkage would be to a 5' carbon is its<br>
5' terminal nucleotide. The end of a polynucleotide at which a new linkage<br>
would be to a 3' carbon is its 3' terminal nucleotide. A terminal nucleotide, as<br>
used herein, is the nucleotide at the end position of the 3'- or 5'-terminus.<br>
DNA molecules are said to have "5' ends" and "3' ends" because<br>
mononucleotides are reacted to make oHgonucleotides in a manner such that the<br>
5' phosphate of one mononucleotide pentose ring is attached to the 3' oxygen of<br>
its neighbor in one direction via a phosphodiester linkage. Therefore, an end of<br>
an oHgonucleotides referred to as the "5' end" if its 5' phosphate is not linked to<br>
the 3' oxygen of a mononucleotide pentose ring and as the "3; end" if its 3'<br>
oxygen is not linked to a 5' phosphate of a subsequent mononucleotide pentose<br>
ring.<br>
As used herein, a nucleic acid sequence, even if internal to a larger<br>
oligonucleotide or polynucleotide, also may be said to have 5' and 3' ends. In<br>
either a linear or circular DNA molecule, discrete elements are referred to as<br>
being "upstream" or 5' of the "downstream" or 3' elements. This terminology<br>
reflects the fact that transcription proceeds in a 5' to 3' fashion along the DNA<br>
strand. Typically, promoter and enhancer elements that direct transcription of a<br>
linked gene (e.g., open reading frame or coding region) are generally located 5'<br>
or upstream of the coding region. However, enhancer elements can exert their<br>
effect even when located 3' of the promoter element and the coding region.<br>
Transcription termination and polyadenylation signals are located 3' or<br>
downstream of the coding region.<br>
The term "codon" as used herein, is a basic genetic coding unit,<br>
consisting of a sequence of three nucleotides that specify a particular amino acid<br>
to be incorporation into a polypeptide chain, or a start or stop signal. The term<br>
"coding region" when used in reference to structural gene refers to the nucleotide<br>
sequences that encode the amino acids found in the nascent polypeptide as a<br>
result of translation of a mRNA molecule. Typically, the coding region is<br>
bounded on the 5' side by the nucleotide triplet "ATG" which encodes the<br>
initiator methionine and on the 3' side by a stop codon (e.g., TAA, TAG, TGA).<br>
In some cases the coding region is also known to initiate by a nucleotide triplet<br>
"TTG".<br>
As used herein, the terms "isolated and/or purified" refer to in vitro<br>
preparation, isolation and/or purification of a nucleic acid molecule, a<br>
polypeptide, peptide or protein, so that it is not associated with in vivo<br>
substances. Thus, the term "isolated" when used in relation to a nucleic acid, as<br>
in "isolated oligonucleotide" or "isolated polynucleotide" refers to a nucleic acid<br>
sequence that is identified and separated from at least one contaminant with<br>
which it is ordinarily associated in its source. An isolated nucleic acid is present<br>
in a form or setting that is different from that in which it is found in nature. In<br>
contrast, non-isolated nucleic acids (e.g., DNA and RNA) are found in the state<br>
they exist in nature. For example, a given DNA sequence (e.g., a gene) is found<br>
on the host cell chromosome in proximity to neighboring genes; RNA sequences<br>
(e.g., a specific mRNA sequence encoding a specific protein), are found in the<br>
cell as a mixture with numerous other mRNAs that encode a multitude of<br>
proteins. Hence, with respect to an "isolated nucleic acid molecule", which<br>
includes a polynucleotide of genomic, cDNA, or synthetic origin or some<br>
combination thereof, the "isolated nucleic acid molecule" (1) is not associated<br>
with all or a portion of a polynucleotide in which the "isolated nucleic acid<br>
molecule" is found in nature, (2) is operably linked to a polynucleotide which it<br>
is not linked to in nature, or (3) does not occur in nature as part of a larger<br>
sequence. The isolated nucleic acid molecule may be present in single-stranded<br>
or double-stranded form. When a nucleic acid molecule is to be utilized to<br>
express a protein, the nucleic acid contains at a minimum, the sense or coding<br>
strand (i.e., the nucleic acid may be single-stranded), but may contain both the<br>
sense and anti-sense strands (i.e., the nucleic acid may be double-stranded).<br>
The term "wild-type" as used herein, refers to a gene or gene product that<br>
has the characteristics of that gene or gene product isolated from a naturally<br>
occurring source. A wild-type gene is that which is most frequently observed in<br>
a population and is thus arbitrarily designated the "wild-type" form of the gene.<br>
In contrast, the term "mutant" refers to a gene or gene product that displays<br>
modifications in sequence and/or functional properties (i.e., altered<br>
characteristics) when compared to the wild-type gene or gene product. It is<br>
noted that naturally-occurring mutants can be isolated; these are identified by the<br>
fact that they have altered characteristics when compared to the wild-type gene<br>
or gene product.<br>
The term "recombinant DNA molecule" means a hybrid DNA sequence<br>
comprising at least two nucleotide sequences not normally found together in<br>
nature. The term "vector" is used in reference to nucleic acid molecules<br>
into which fragments of DNA may be inserted or cloned and can be used to<br>
transfer DNA segments) into a cell and capable of replication in a cell. Vectors<br>
may be derived from plasmids, baoteriophages, viruses, cosmids, and the like.<br>
The terms "recombinant vector", "expression vector" or "construct" as<br>
used herein refer to DNA or RNA sequences containing a desired coding<br>
sequence and appropriate DNA or RNA sequences necessary for the expression<br>
of the operably linked coding sequence in a particular host organism.<br>
Prokaryotic expression vectors include a promoter, a ribosome binding site, an<br>
origin of replication for autonomous replication in a host cell and possibly other<br>
sequences, e.g. an optional operator sequence, optional restriction enzyme sites.<br>
A promoter is defined as a DNA sequence that directs RNA polymerase to bind<br>
to DNA and to initiate RNA synthesis. Eukaryotic expression vectors include a<br>
promoter, optionally a polyadenylation signal and optionally an enhancer<br>
sequence.<br>
A polynucleotide having a nucleotide sequence "encoding a peptide,<br>
protein or polypeptide" means a nucleic acid sequence comprising the coding<br>
region of a gene, or a fragment thereof which encodes a gene product having<br>
substantially the same activity as the corresponding full-length peptide, protein<br>
or polypeptide. The coding region may be present in either a cDNA, genomic<br>
DNA or RNA form. When present in a DNA form, the oligonucleotide may be<br>
single-stranded (i.e., the sense strand) or double-stranded. Suitable control<br>
elements such as enhancers/promoters, splice junctions, polyadenylation signals,<br>
etc. may be placed in close proximity to the coding region of the gene if needed<br>
to permit proper initiation of transcription and/or correct processing of the<br>
primary RNA transcript. Alternatively, the coding region utilized in the<br>
expression vectors of the present invention may contain endogenous<br>
enhancers/promoters, splice junctions, intervening sequences, polyadenylation<br>
signals, etc. In further embodiments, the coding region may contain a<br>
combination of both endogenous and exogenous control elements.<br>
The term "transcription regulatory element" or "transcription regulatory<br>
sequence" refers to a genetic element or sequence that controls some aspect of<br>
the expression of nucleic acid sequences). For example, a promoter is a<br>
regulatory element that facilitates the initiation of transcription of an operably<br>
linked coding region. Other regulatory elements include, but are not limited to,<br>
transcription factor binding sites, splicing signals, polyadenylation signals,<br>
termination signals and enhancer elements.<br>
Transcriptional control signals in eukaryotes comprise "promoter" and<br>
"enhancer" elements. Promoters and enhancers consist of short arrays of DNA<br>
sequences that interact specifically with cellular proteins involved in<br>
transcription. Promoter and enhancer elements have been isolated from a variety<br>
of eukaryotic sources including genes in yeast, insect and mammalian cells.<br>
Promoter and enhancer elements have also been isolated from viruses and<br><br>
analogous control elements, such as promoters, are also found in prokaryotes.<br>
The selection of a particular promoter and enhancer depends on the cell type<br>
used to express the protein of interest. Some eukaryotic promoters and<br>
enhancers have a broad host range while others are functional in a limited subset<br>
of cell types. For example, the SV40 early gene enhancer is very active in a<br>
wide variety of cell types from many mammalian species and has been widely<br>
used for the expression of proteins in mammalian cells. Two other examples of<br>
promoter/enhancer elements active in a broad range of mammalian cell types are<br>
those from the human elongation factor 1 gene (Uetsuki et al., 1989; Kim et al.,<br>
1990; and Mizushima and Nagata, 1990) and the long terminal repeats of the<br>
Rous sarcoma virus (Gorman et al., 1982); and the human cytomegalovirus<br>
(Boshart et al., 1985).<br>
The term "promoter/enhancer" denotes a segment of DNA containing<br>
sequences capable of providing both promoter and enhancer functions (i.e., the<br>
functions provided by a promoter element and an enhancer element as described<br>
above). For example, the long terminal repeats of retroviruses contain both<br>
promoter and enhancer functions. The enhancer/promoter may be "endogenous"<br>
or "exogenous" or "heterologous." An "endogenous" enhancer/promoter is one<br>
that is naturally linked with a given gene in the genome. An "exogenous" or<br>
"heterologous" enhancer/promoter is one that is placed in juxtaposition to a gene<br>
by means of genetic manipulation (i.e., molecular biological techniques) such<br>
that transcription of the gene is directed by the linked enhancer/promoter.<br>
The presence of "splicing signals" on an expression vector often results<br>
in higher levels of expression of the recombinant transcript in eukaryotic host<br>
cells. Splicing signals mediate the removal of introns from the primary RNA<br>
transcript and consist of a splice donor and acceptor site (Sambrook et al., 1989).<br>
A commonly used splice donor and acceptor site is the splice junction from the<br>
16SRNAofSV40.<br>
Efficient expression of recombinant DNA sequences in eukaryotic cells<br>
requires expression of signals directing the efficient termination and<br>
polyadenylation of the resulting transcript. Transcription termination signals are<br>
generally found downstream of the polyadenylation signal and are a few hundred<br>
nucleotides in length. The term "poly(A) site" or "poly(A) sequence" as used<br><br>
herein denotes a DNA sequence which directs both the termination and<br>
polyadenylation of the nascent RNA transcript. Efficient polyadenylation of the<br>
recombinant transcript is desirable, as transcripts lacking a poly(A) tail are<br>
unstable and are rapidly degraded. The poly(A) signal utilized in an expression<br>
vector may be "heterologous" or "endogenous." An endogenous poly(A) signal<br>
is one that is found naturally at the 3' end of the coding region of a given gene in<br>
the genome. A heterologous poly(A) signal is one which has been isolated from<br>
one gene and positioned 3' to another gene. A commonly used heterologous<br>
poly(A) signal is the SV40 poly(A) signal. The SV40 poly(A) signal is<br>
contained on a 237 bp BamH. IIBel I restriction fragment and directs both<br>
termination and polyadenylation (Sambrook et al., 1989).<br>
Eukaryotic expression vectors may also contain "viral replicons "or "viral<br>
origins of replication." Viral replicons are viral DNA sequences which allow for<br>
the extrachromosomal replication of a vector in a host cell expressing the<br>
appropriate replication factors. Vectors containing either the S V40 or polyoma<br>
virus origin of replication replicate to high copy number (up to 104 copies/cell)<br>
in cells that express the appropriate viral T antigen. In contrast, vectors<br>
containing the replicons from bovine papillomavirus or Epstein-Barr virus<br>
replicate extrachromosomally at low copy number (about 100 copies/cell).<br>
The term "in vitro" refers to an artificial environment and to processes or<br>
reactions that occur within an artificial environment. In vitro environments<br>
include, but are not limited to, test tubes and cell lysates. The term "in situ"<br>
refers to cell culture. The term "in vivo" refers to the natural environment (e.g.,<br>
an animal or a cell) and to processes or reaction that occur within a natural<br>
environment.<br>
The term "expression system" refers to any assay or system for<br>
determining (e.g., detecting) the expression of a gene of interest. Those skilled<br>
in the field of molecular biology will understand that any of a wide variety of<br>
expression systems may be used. A wide range of suitable mammalian cells are<br>
available from a wide range of sources (e.g., the American Type Culture<br>
Collection, Rockland, MD). The method of transformation or transfection and<br>
the choice of expression vehicle will depend on the host system selected.<br>
Transformation and transfection methods are described, e.g., in Sambrook et al.,<br>
1989. Expression systems include in vitro gene expression assays where a gene<br>
of interest (e.g., a reporter gene) is linked to a regulatory sequence and the<br>
expression of the gene is monitored following treatment with an agent that<br>
inhibits or induces expression of the gene. Detection of gene expression can be<br>
through any suitable means including, but not limited to, detection of expressed<br>
mRNA or protein (e.g., a detectable product of a reporter gene) or through a<br>
detectable change in the phenotype of a cell expressing the gene of interest.<br>
Expression systems may also comprise assays where a cleavage event or other<br>
nucleic acid or cellular change is detected.<br>
The term "gene" refers to a DNA sequence that comprises coding<br>
sequences and optionally control sequences necessary for the production of a<br>
polypeptide from the DNA sequence. The polypeptide can be encoded by a fulllength<br>
coding sequence or by any portion of the coding sequence so long as the<br>
portion encodes a gene product with substantially the same activity as the fulllength<br>
polypeptide.<br>
Nucleic acids are known to contain different types of mutations. A<br>
"point" mutation refers to an alteration in the sequence of a nucleotide at a single<br>
base position from the wild-type sequence. Mutations may also refer to insertion<br>
or deletion of one or more bases, so that the nucleic acid sequence differs from a<br>
reference, e.g., a wild-type, sequence.<br>
As used herein, the terms "hybridize" and "hybridization" refer to the<br>
annealing of a complementary sequence to the target nucleic acid, i.e., the ability<br>
of two polymers of nucleic acid (polynucleotides) containing complementary<br>
sequences to anneal through base pairing. The terms "annealed" and<br>
"hybridized" are used interchangeably throughout, and are intended to<br>
encompass any specific and reproducible interaction between a complementary<br>
sequence and a target nucleic acid, including binding of regions having only<br>
partial complementarity. Certain bases not commonly found in natural nucleic<br>
acids may be included in the nucleic acids of the present invention and include,<br>
for example, inosine and 7-deazaguanine. Those skilled in the art of nucleic acid<br>
technology can determine duplex stability empirically considering a number of<br>
variables including, for example, the length of the complementary sequence,<br>
base composition and sequence of the oligonucleotide, ionic strength and<br>
incidence of mismatched base pairs. The stability of a nucleic acid duplex is<br>
measured by the melting temperature, or "Tm". The Tm of a particular nucleic<br>
acid duplex under specified conditions is the temperature at which on average<br>
half of the base pairs have disassociated.<br>
The term "stringency" is used in reference to the conditions of<br>
temperature, ionic strength, and the presence of other compounds, under which<br>
nucleic acid hybridizations are conducted. With "high stringency" conditions,<br>
nucleic acid base pairing will occur only between nucleic acid fragments that<br>
have a high frequency of complementary base sequences. Thus, conditions of<br>
"medium" or "low" stringency are often required when it is desired that nucleic<br>
acids which are not completely complementary to one another be hybridized or<br>
annealed together. The art knows well that numerous equivalent conditions can<br>
be employed to comprise medium or low stringency conditions. The choice of<br>
hybridization conditions is generally evident to one skilled in the art and is<br>
usually guided by the purpose of the hybridization, the type of hybridization<br>
(DNA-DNA or DNA-RNA), and the level of desired relatedness between the<br>
sequences (e.g., Sambrook et al., 1989; Nucleic Acid Hybridization, A Practical<br>
Approach, IRL Press, Washington D.C., 1985, for a general discussion of the<br>
methods).<br>
The stability of nucleic acid duplexes is known to decrease with an<br>
increased number of mismatched bases, and further to be decreased to a greater<br>
or lesser degree depending on the relative positions of mismatches in the hybrid<br>
duplexes. Thus, the stringency of hybridization can be used to maximize or<br>
minimize stability of such duplexes. Hybridization stringency can be altered by:<br>
adjusting the temperature of hybridization; adjusting the percentage of helix<br>
destabilizing agents, such as formamide, in the hybridization mix; and adjusting<br>
the temperature and/or salt concentration of the wash solutions. For filter<br>
hybridizations, the final stringency of hybridizations often is determined by the<br>
salt concentration and/or temperature used for the post-hybridization washes.<br>
"High stringency conditions" when used in reference to nucleic acid<br>
hybridization include conditions equivalent to binding or hybridization at 42 ยฐC<br>
in a solution consisting of 5X SSPE (43.8 g/1 NaCl, 6.9 g/1 NaH2PO4 H2O and<br>
1.85 g/1 EDTA, pH adjusted to 7.4 with NaOH), 0.5% SDS, 5X Denhardt's<br>
reagent and 100 ug/ml denatured salmon sperm DNA followed by washing in a<br>
solution comprising 0.1X SSPE, 1.0% SDS at 42 ยฐC when a probe of about 500<br>
nucleotides in length is employed.<br>
"Medium stringency conditions" when used in reference to nucleic acid<br>
hybridization include conditions equivalent to binding or hybridization at 42 ยฐC<br>
in a solution consisting of 5X SSPE (43.8 g/l NaCl, 6.9 g/1 NaH2PO4 H2O and<br>
1.85 g/1 EDTA, pH adjusted to 7.4 with NaOH), 0.5% SDS, 5X Denhardt's<br>
reagent and 100 fig/ml denatured salmon sperm DNA followed by washing in a<br>
solution comprising l.OX SSPE, 1.0% SDS at 42 ยฐC when a probe of about 500<br>
nucleotides in length is employed.<br>
"Low stringency conditions" include conditions equivalent to binding or<br>
hybridization at 42ยฐC in a solution consisting of 5X SSPE (43.8 g/1 NaCl, 6.9 g/1<br>
NaH2PO4 H2O and 1.85 g/1 EDTA, pH adjusted to 7.4 with NaOH), 0.1% SDS,<br>
5X Denhardt's reagent [SOX Denhardt's contains per 500 ml: 5 g Ficoll (Type<br>
400, Pharmacia), 5 g BSA (Fraction V; Sigma)] and 100 g/ml denatured salmon<br>
sperm DNA followed by washing in a solution comprising 5X SSPE, 0.1% SDS<br>
at 42 ยฐC when a probe of about 500 nucleotides in length is employed.<br>
By "peptide", "protein" and "polypeptide" is meant any chain of amino<br>
acids, regardless of length or post-translational modification (e.g., glycosylation<br>
or phosphorylation). Unless otherwise specified, the terms are interchangeable.<br>
The nucleic acid molecules of the invention encode a variant (mutant) of a<br>
naturally-occurring (wild-type) protein or fragment thereof which has<br>
substantially the same activity as the full length mutant protein. Preferably, such<br>
a mutant protein has an amino acid sequence that is at least 85%, preferably<br>
90%, and most preferably 95% or 99%, identical to the amino acid sequence of a<br>
corresponding wild-type protein.<br>
Polypeptide molecules are said to have an "amino terminus"<br>
(N-terminus) and a "carboxy terminus" (C-terminus) because peptide linkages<br>
occur between the backbone amino group of a first amino acid residue and the<br>
backbone carboxyl group of a second amino acid residue. The terms<br>
"N-terminal" and "C-terminal" in reference to polypeptide sequences refer to<br>
regions of polypeptides including portions of the N-terminal and C-terminal<br>
regions of the polypeptide, respectively. A sequence that includes a portion of<br>
the N-terminal region of polypeptide includes amino acids predominantly from<br>
the N-terminal half of the polypeptide chain, but is not limited to such<br>
sequences. For example, an N-terminal sequence may include an interior portion<br>
of the polypeptide sequence including bases from both the N-terminal and<br>
C-terminal halves of the polypeptide. The same applies to C-terminal regions.<br>
N-terminal and C-terminal regions may, but need not, include the amino acid<br>
defining the ultimate N-terminus and C-terminus of the polypeptide,<br>
respectively.<br>
The term "isolated" when used in relation to a polypeptide, as in "isolated<br>
protein" or "isolated poiypeptide" refers to a polypeptide that is identified and<br>
separated from at least one contaminant with which it is ordinarily associated in<br>
its source. Thus, an isolated polypeptide (1) is not associated with proteins<br>
found in nature, (2) is free of other proteins from the same source, e.g., free of<br>
human proteins, (3) is expressed by a cell from a different species, or (4) does<br>
not occur in nature. In contrast, non-isolated polypeptides (e.g., proteins and<br>
enzymes) are found in the state they exist in nature. The .terms "isolated<br>
polypeptide", "isolated peptide" or "isolated protein" include a polypeptide,<br>
peptide or protein encoded by cDNA or recombinant RNA including one of<br>
synthetic origin, or some combination thereof.<br>
The term "recombinant protein" or "recombinant polypeptide" as used<br>
herein refers to a protein molecule expressed from a recombinant DNA<br>
molecule. In contrast, the term "native protein" is used herein to indicate a<br>
protein isolated from a naturally occurring (i.e., a nonrecombinant) source.<br>
Molecular biological techniques may be used to produce a recombinant form of<br>
a protein with identical properties as compared to the native form of the protein.<br>
The term "fusion polypeptide" as used herein refers to a chimeric protein<br>
containing a protein of interest (e.g., luciferase, an affinity tag or a targeting<br>
sequence) joined to a different protein, e.g., a mutant hydrolase.<br>
As used herein, the term "antibody" refers to a protein having one or<br>
more polypeptides substantially encoded by immunoglobulin genes or fragments<br>
of immunoglobulin genes. The recognized immunoglobulin genes include the<br>
kappa, lambda, alpha, gamma, delta, epsilon and mu constant region genes, as<br>
well as the myriad of immunoglobulin variable region genes. Light chains are<br>
classified as either kappa or lambda. Heavy chains are classified as gamma, mu,<br>
alpha, delta, or epsilon, which in turn define the immunoglobulin classes, IgG,<br>
IgM, IgA, IgD and IgE, respectively.<br>
The basic immunoglobulin (antibody) structural unit is known to<br>
comprise a tetramer. Each tetramer is composed of two identical pairs of<br>
polypeptide chains, each pah: having one "light" (about 25 kD) and one "heavy"<br>
chain (about 50-70 kD). The N-terminus of each chain defines a variable region<br>
of about 100 to 110 or more amino acids primarily responsible for antigen<br>
recognition. The terms variable light chain (Vz) and variable heavy chain (V/y)<br>
refer to these light and heavy chains respectively.<br>
Antibodies may exist as intact immunoglobulins, or as modifications hi a<br>
variety of forms including, for example, FabFcz, Fab, Fv, Fd, (Fab')2ยป an Fv<br>
fragment containing only the light and heavy chain variable regions, a Fab or<br>
(Fab)'2 fragment containing the variable regions and parts of the constant<br>
regions, a single-chain antibody, e.g., scFv, CDR-grafted antibodies and the like.<br>
The heavy and light chain of a Fv may be derived from the same antibody or<br>
different antibodies thereby producing a chimeric Fv region. The antibody may<br>
be of animal (especially mouse or rat) or human origin or may be chimeric or<br>
humanized. As used herein the term "antibody" includes these various forms.<br>
The terms "cell," "cell line," "host cell," as used herein, are used<br>
interchangeably, 'and all such designations include progeny or potential progeny<br>
of these designations. By "transformed cell" is meant a cell into which (or into<br>
an ancestor of which) has been introduced a nucleic acid molecule of the<br>
invention. Optionally, a nucleic acid molecule of the invention may be<br>
introduced into a suitable cell line so as to create a stably transfected cell line<br>
capable of producing the protein or polypeptide encoded by the nucleic acid<br>
molecule. Vectors, cells, and methods for constructing such cell lines are well<br>
known in the art. The words "transformants" or "transformed cells" include the<br>
primary transformed cells derived from the originally transformed cell without<br>
regard to the number of transfers. All progeny may not be precisely identical in<br>
DNA content, due to deliberate or inadvertent mutations. Nonetheless, mutant<br>
progeny that have the same functionality as screened for in the originally<br>
transformed cell are included hi the definition of transformants.<br>
The term "homology" refers to a degree of complementarity. There may<br>
be partial homology or complete homology (i.e., identity). Homology is often<br>
measured using sequence analysis software (e.g., Sequence Analysis Software<br>
Package of the Genetics Computer Group. University of Wisconsin<br>
Biotechnology Center. 1710 University Avenue. Madison, WI 53705). Such<br>
software matches similar sequences by assigning degrees of homology to various<br>
substitutions, deletions, insertions, and other modifications. Conservative<br>
substitutions typically include substitutions within the following groups:<br>
glvcine, alanine; valine, isoleucine, leucine; aspartic acid, glutamic acid,<br>
asparagine, glutamine; serine, threonine; lysine, arginine; and phenylalanine,<br>
tyrosine.<br>
The term "purified" or "to purify" means the result of any process that<br>
removes some of a contaminant from the component of interest, such as a<br>
protein or nucleic acid. The percent of a purified component is thereby<br>
increased in the sample.<br>
The term "operably linked" as used herein refer to the linkage of nucleic<br>
acid sequences in such a manner that a nucleic acid molecule capable of<br>
directing the transcription of a given gene and/or the synthesis of a desired<br>
protein molecule is produced. The term also refers to the Linkage of sequences<br>
encoding amino acids in such a manner that a functional (e.g., enzymatically<br>
active, capable of binding to a binding partner, capable of inhibiting, etc.)<br>
protein or polypeptide, or a precursor thereof, e.g., the pre- or prepro-form of the<br>
protein or polypeptide, is produced.<br>
All amino acid residues identified herein are in the natural<br>
L-configuration. In keeping with standard polypeptide nomenclature,<br>
abbreviations for amino acid residues are as shown in the following Table of<br>
Correspondence.<br>
TABLE OF CORRESPONDENCE<br>
1-Letter 3-Letter AMINO ACID<br>
Y Tyr L-tyrosine<br>
G Gly L-glycine<br>
F Phe L-phenylalanine<br><br>
M Met L-methionine<br>
A Ala L-alanine<br>
S Ser L-serine<br>
I He L-isoleucine<br>
L Leu L-leucine<br>
T Thr L-threonine<br>
V Val L-valine<br>
P Pro L-proline '<br>
K Lys L-lysine<br>
H His L-histidine<br>
Q Gin L-glutamine<br>
E Glu L-glutamic acid<br>
W Trp L-tryptophan<br>
R Arg L-arginine<br>
D Asp L-aspartic acid<br>
N Asn L-asparagine<br>
C Cys L-cysteine<br>
As used herein, the term "poly-histidine tract" or (His tag) refers to a<br>
molecule comprising two to ten histidine residues, e.g., a poly-histidine tract of<br>
five to ten residues. A poly-histidine tract allows the affinity purification of a<br>
covalently linked molecule on an immobilized metal, e.g., nickel, zinc, cobalt or<br>
copper, chelate column or through an interaction with another molecule (e.g., an<br>
antibody reactive with the His tag).<br>
As used herein, "pure" means an object species is the predominant<br>
species present (i.e., on a molar basis it is more abundant than any other<br>
individual species in the composition), and preferably a substantially purified<br>
fraction is a composition wherein the object species comprises at least about 50<br>
percent (on a molar basis) of all macromolecular species present. Generally, a<br>
"substantially pure" composition will comprise more than about 80 percent of all<br>
macromolecular species present in the composition, more preferably more than<br>
about 85%, about 90%, about 95%, and about 99%. Most preferably, the object<br>
species is purified to essential homogeneity (contaminant species cannot be<br>
detected in the composition by conventional detection methods) wherein the<br>
composition consists essentially of a single macromolecular species.<br>
I. Mutant Hvdrolases and Fusions ^hereof<br>
Mutant hydrolases within the scope of the invention include but are not<br>
limited to those prepared via recombinant techniques, e.g., site-directed<br>
mutagenesis or recursive rnutagenesis, and comprise one or more amino acid<br>
substitutions which render the mutant hydrolase capable of forming a stable, e.g.,<br>
covalent, bond with a substrate, such as a substrate modified to contain one or<br>
more functional groups, for a corresponding nonmutant (wild-type) hydrolase.<br>
Hydrolases within the scope of the invention include, but are not limited to,<br>
peptidases, esterases (e.g., cholesterol esterase), glycosidases (e.g.,<br>
glucosamylase), phosphatases (e.g., alkaline phosphatase) and the like. For<br>
instance, hydrolases include, but are not limited to, enzymes acting on ester<br>
bonds such as carboxylic ester hydrolases, thiolester hydrolases, phosphoric<br>
monoester hydrolases, phosphoric diester hydrolases, triphosphoric monoester<br>
hydrolases, sulfuric ester hydrolases, diphosphoric monoester hydrolases,<br>
phosphoric triester hydrolases, exodeoxyribonucleases producing 5'-<br>
phosphomonoesters, exoribonucleases producing 5'-phosphomonoesters,<br>
exoribonucleases producing 3'-phosphomonoesters, exonucleases active with<br>
either ribo- or deoxyribonucleic acid, exonucleases active with either ribo- or<br>
deoxyribonucleic acid, endodeoxyribonucleases producing 5'-<br>
phosphomonoesters, endodeoxyribonucleases producing other than 5'-<br>
phosphomonoesters, site-specific endodeoxyribonucleases specific for altered<br>
bases, endoribonucleases producing 5'-phosphomonoesters, endoribonucleases<br>
producing other than S'-phosphomonoesters, endoribonucleases active with<br>
either ribo- or deoxyribonucleic, endoribonucleases active with either ribo- or<br>
deoxyribonucleic glycosylases; glycosidases, e.g., enzymes hydrolyzing O- and<br>
S-glycosyl, and hydrolyzing N-glycosyl compounds; acting on ether bonds such<br>
as trialkylsulfonium hydrolases or ether hydrolases; enzymes acting on peptide<br>
bonds (peptide hydrolases) such as aminopeptidases, dipeptidases, dipeptidylpeptidases<br>
and tripeptidyl-peptidases, peptidyl-dipeptidases, serine-type<br>
carboxypeptidases, metallocarboxypeptidases, cysteine-type carboxypeptidases,<br>
omega peptidases, serine endopeptidases, cysteine endopeptidases, aspartic<br>
endopeptidases, metalloendopeptidases, threonine endopeptidases, and<br>
endopeptidases of unknown catalytic mechanism; enzymes acting on carbonnitrogen<br>
bonds, other than peptide bonds, such as those in linear amides, in<br>
cyclic amides, in linear amidinesj in cyclic amidines, in nitriles, or other<br>
compounds; enzymes acting on acid anhydrides such as those in phosphorouscontaining<br>
anhydrides and in sulfonyl-containing anhydrides; enzymes acting on<br>
acid anhydrides (catalyzing transmembrane movement); enzymes acting on acid<br>
anhydrides or involved in cellular and subcellular movement; enzymes acting on<br>
carbon-carbon bonds (e.g., in ketonic substances); enzymes acting on halide<br>
bonds (e.g., in C-halide compounds), enzymes acting on phosphorus-nitrogen<br>
bonds; enzymes acting on sulfur-nitrogen bonds; enzymes acting on carbonphosphorus<br>
bonds; and enzymes acting on sulfur-sulfur bonds. Exemplary<br>
hydrolases acting on halide bonds include, but are not limited to, alkylhalidase,<br>
2-haloacid dehalogenase, haloacetate dehalogenase, thyroxine deiodinase,<br>
haloalkane dehalogenase, 4-chlorobenzoate dehalogenase, 4-chlorobenzoyl-CoA<br>
dehalogenase, and atrazine chlorohydrolase. Exemplary hydrolases that act on<br>
carbon-nitrogen bonds in cyclic amides include, but are not limited to,<br>
barbiturase, dihydropyrimidinase, dihydroorotase, carboxymethylhydantoinase,<br>
allantoinase, p-lactamase, imidazolonepropionase, 5-oxoprolinase (ATPhydrolysing),<br>
creatininase, L-lysine-lactamase, 6-aminohexanoate-cyclic-dimer<br>
hydrolase, 2,5-dioxopiperazine hydrolase, W-methylhydantoinase (ATPhydrolysing),<br>
cyanuric acid amidohydrolase, maleimide hydrolase. "Betalactamase"<br>
as used herein includes Class A, Class C and Class D beta-lactamases<br>
as well as D-ala carboxypeptidase/transpeptidase, esterase EstB, penicillin<br>
binding protein 2X, penicillin binding protein 5, and D-amino peptidase.<br>
Preferably, the beta-lactamase is a serine beta-lactamase, e.g., one having a<br>
catalytic serine residue at a position corresponding to residue 70 in the serine<br>
beta-lactamase of S. aureus PCI, and a glutamic acid residue at a position<br>
corresponding to residue 166 in the serine beta-lactamase of ยฃ aureus PCI,<br>
optionally having a lysine residue at a position corresponding to residue 73, and<br>
also optionally having a lysine residue at a position corresponding to residue<br>
234, in the beta-lactamase of S. aureus PCI.<br>
In one embodiment, the mutant hydrolase is a haloalkane dehalogenase,<br>
e.g., such as those found in Gram-negative (Keuning et al., 1985) and Grampositive<br>
haloalkane-utilizing bacteria (Keuning et al., 1985; Yokota et al., 1987;<br>
Scholtz et al., 1987; Sallis et al., 1990). Haloalkane dehalogenases, including<br>
DMA SomXanthobacter autotrophicus GJ10 (Janssen et al., 1988,1989) and<br>
DhaA from Rhodococcus rhodochrous, are enzymes which catalyze hydrolytic<br>
dehalogenation of corresponding hydrocarbons. Halogenated aliphatic<br>
hydrocarbons subject to conversion include C2-Cio saturated aliphatic<br>
hydrocarbons which have one or more halogen groups attached, wherein at least<br>
two of the halogens are on adjacent carbon atoms. Such aliphatic hydrocarbons<br>
include volatile chlorinated aliphatic (VGA) hydrocarbons. VGA's include, for<br>
example, aliphatic hydrocarbons such as dichloroethane, 1,2-dichloro-propane,<br>
1,2-dichlorobutane and 1,2,3-trichloropropane. The term "halogenated<br>
hydrocarbon" as used herein means a halogenated aliphatic hydrocarbon. As<br>
used herein the term "halogen" includes chlorine, bromine, iodine, fluorine,<br>
astatine and the like. A preferred halogen is chlorine.<br>
As described herein, the invention includes a fusion protein comprising a<br>
mutant hydrolase and arnino acid sequences for a protein of interest, e.g.,<br>
sequences for a marker protein or affinity tag, e.g., luciferase, GFP, or a<br>
polyhistidine sequence, a nucleic acid binding protein, an extracellular matrix<br>
protein, a secreted protein, a receptor ligand, a serum protein, an immunogenic<br>
protein, a fluorescent protein, a protein with reactive cysteines, a receptor<br>
protein, e.g., NMDA receptor, a channel protein, e.g., a sodium-, potassium- or a<br>
calcium-sensitive channel protein including a HERG channel protein, or a<br>
transporter protein, e.g., EA AT 1-4 glutamate transporter, as well as targeting<br>
signals, e.g., a plastid targeting signal, a nuclear localization signal or a<br>
myristilation sequence.<br>
II. Optimized Hvdrolase Sequences, and Vectors and Host Cells Encoding<br>
the Hvdrolase<br>
A nucleic acid molecule comprising a nucleic acid sequence encoding a<br>
hydrolase or a fusion thereof is optionally optimized for expression in a<br>
particular host cell and also optionally operably linked to transcription regulatory<br>
sequences, e.g., one or more enhancers, a promoter, a transcription termination<br>
sequence or a combination thereof, to form an expression cassette.<br>
In one embodiment, a nucleic acid sequence encoding a hydrolase or a<br>
fusion thereof is optimized by replacing codons in a wild-type or mutant<br>
hydrolase sequence with codons which are preferentially employed in a<br>
particular (selected) cell. Preferred codons have a relatively high codon usage<br>
frequency in a selected cell, and preferably their introduction results in the<br>
introduction of relatively few transcription factor binding sites for transcription<br>
factors present in the selected host cell, and relatively few other undesirable<br>
structural attributes. Thus, the optimized nucleic acid product has an improved<br>
level of expression due to improved codon usage frequency, and a reduced risk<br>
of inappropriate transcriptional behavior due to a reduced number of undesirable<br>
transcription regulatory sequences.<br>
An isolated and optimized nucleic acid molecule of the invention may<br>
have a codon composition that differs from that of the corresponding wild-type<br>
nucleic acid sequence at more than 30%, 35%, 40% or more than 45%, e.g.,<br>
50%, 55%, 60% or more of the codons. Preferred codons for use in the<br>
invention are those which are employed more frequently than at least one other<br>
codon for the same amino acid in a particular organism and, more preferably, are<br>
also not low-usage codons in that organism and are not low-usage codons in the<br>
organism used to clone or screen for the expression of the nucleic acid molecule.<br>
Moreover, preferred codons for certain amino acids (i.e., those amino acids that<br>
have three or more codons), may include two or more codons that are employed<br>
more frequently than the other (non-preferred) codon(s). The presence of<br>
codons in the nucleic acid molecule that are employed more frequently in one<br>
organism than in another organism results hi a nucleic acid molecule which,<br>
when introduced into the cells of the organism that employs those codons more<br>
frequently, is expressed in those cells at a level that is greater than the expression<br>
of the wild-type or parent nucleic acid sequence in those cells.<br>
In one embodiment of the invention, the codons that are different are<br>
those employed more frequently in a mammal, while in another embodiment the<br>
codons that are different are those employed more frequently in a plant.<br>
Preferred codons for different organisms are known to the art, e.g., see<br>
www.ka2fusft.or.jp./codon/. A particular type of mammal, e.g., a human, may<br>
have a different set of preferred codons than another type of mammal. Likewise,<br>
a particular type of plant may have a different set of preferred codons than<br>
another type of plant. In one embodiment of the invention, the majority of the<br>
codons that differ are ones that are preferred codons in a desired host cell.<br>
Preferred codons for organisms including mammals (e.g., humans) and plants are<br>
known to the art (e.g., Wada et al., 1990; Ausubel et al., 1997). For example,<br>
preferred human codons include, but are not limited to, CGC (Arg), CTG (Leu),<br>
TCT (Ser), AGC (Ser), ACC (Thr), CCA (Pro), CCT (Pro), GCC (Ala), GGC<br>
(Gly), GTG (Val), ATC (De), ATT (He), AAG (Lys), AAC (Asn), CAG (Gin),<br>
CAC (His), GAG (Glu), GAC (Asp), TAC (Tyr), TGC (Cys) and TTC (Phe)<br>
(Wada et al., 1990). Thus, in one embodiment, synthetic nucleic acid molecules<br>
of the invention have a codon composition which differs from a wild type<br>
nucleic acid sequence by having an increased number of the preferred human<br>
codons, e.g., CGC, CTG, TCT, AGC, ACC, CCA, CCT, GCC, GGC, GTG,<br>
ATC, ATT, AAG, AAC, CAG, CAC, GAG, GAC, TAC, TGC, TTC, or any<br>
combination thereof. For example, the nucleic acid molecule of the invention<br>
may have an increased number of CTG or TTG leucine-encoding codons, GTG<br>
or GTC valine-encoding codons, GGC or GOT glycine-encoding codons, ATC<br>
or ATT isoleucine-encoding codons, CCA or CCT proline-encoding codons,<br>
CGC or CGT arginine-encoding codons, AGC or TCT serine-encoding codons,<br>
ACC or ACT threonine-encoding codon, GCC or GCT alanine-encoding codons,<br>
or any combination thereof, relative to the wild-type nucleic acid sequence, hi<br>
another embodiment, preferred C. elegans codons include, but are not limited, to<br>
UUC (Phe), UUU (Phe), CUU (Leu), UUG (Leu), AUU (He), GUU (Val), GUG<br>
(Val), UCA (Ser), UCU (Ser), CCA (Pro), ACA (Thr), ACU (Thr), GCU (Ala),<br>
GCA (Ala), UAU (Tyr), CAU (His), CAA (Gin), AAU (Asn), AAA (Lys), GAU<br>
(Asp), GAA (Glu), UGU (Cys), AGA (Arg), CGA (Arg), CGU (Arg), GGA<br>
(Gly), or any combination thereof. In yet another embodiment, preferred<br>
Drosophilia codons include, but are not limited to, UUC (Phe), CUG (Leu),<br>
CUC (Leu), AUC (He), AUU (He), GUG (Val), GUC (Val), AGC (Ser), UCC<br>
(Ser), CCC (Pro), CCG (Pro), ACC (Thr), ACG (Thr), GCC (Ala), GCU (Ala),<br>
UAC (Tyr), CAC (His), CAG (Gin), AAC (Asn), AAG (Lys), GAU (Asp), GAG<br>
 (Glu), UGC (Cys), COG (Arg), GGC (Gly), GGA (gly), or any combination<br>
thereof. Preferred yeast codons include but are not limited to UUU (Phe), UUG<br>
(Leu), UUA (Leu), CCU (Leu), AUU (He), GUU (Val), UCU (Ser), UCA (Ser),<br>
CCA (Pro), CCU (Pro), ACU (Thr), ACA (Thr), GCU (Ala), GCA (Ala), UAU<br>
(Tyr), UAC (Tyr), CAU (His), CAA (Gin), AAU (Asn), AAC (Asn), AAA<br>
(Lys), AAG (Lys), GAU (Asp), GAA (Glu), GAG (Glu), UGU (Cys), CGU<br>
(Tip), AGA (Arg), CGU (Axg), GGU (Gly), GGA (Gly), or any combination<br>
thereof. Similarly, nucleic acid molecules having an increased number of<br>
codons that are employed more frequently in plants, have a codon composition<br>
which differs from a wild-type or parent nucleic acid sequence by having an<br>
increased number of the plant codons including, but not limited to, CGC (Arg),<br>
CTT (Leu), TCT (Ser), TCC (Ser), ACC (Thr), CCA (Pro), CCT (Pro), GCT<br>
(Ser), GGA (Gly), GTG (Val), ATC (lie), ATT (He), AAG (Lys), AAC (Asn),<br>
CAA (Gin), CAC (His), GAG (Glu), GAC (Asp), TAG (Tyr), TGC (Cys), TTC<br>
(Phe), or any combination thereof (Murray et al., 1989). Preferred codons may<br>
differ for different types of plants (Wada et al., 1990).<br>
In one embodiment, an optimized nucleic acid sequence encoding a<br>
hydrolase or fusion thereof has less than 100%, e.g., less man 90% or less than<br>
80%, nucleic acid sequence identity relative to a non-optimized nucleic acid<br>
sequence encoding a corresponding hydrolase or fusion thereof. For instance,<br>
an optimized nucleic acid sequence encoding DhaA has less than about 80%<br>
nucleic acid sequence identity relative to non-optimized (wild-type) nucleic acid<br>
sequence encoding a corresponding DhaA, and the DhaA encoded by the<br>
optimized nucleic acid sequence optionally has at least 85% amino acid<br>
sequence identity to a corresponding wild-type DhaA. In one embodiment, the<br>
activity of a DhaA encoded by the optimized nucleic acid sequence is at least<br>
10%, e.g., 50% or more, of the activity of a DhaA encoded by the non-optimized<br>
sequence, e.g., a mutant DhaA encoded by the optimized nucleic acid sequence<br>
binds a substrate with substantially the same efficiency, i.e., at least 50%, 80%,<br>
100% or more, as the mutant DhaA encoded by the non-optimized nucleic acid<br>
sequence binds the same substrate.<br>
The nucleic acid molecule or expression cassette may be introduced to a<br>
vector, e.g., a plasmid or viral vector, which optionally includes a selectable<br>
marker gene, and the vector introduced to a cell of interest, for example, a<br>
prokaryotic cell such as E. coli, Streptomyces spp., Bacillus spp., Staphylococcus<br>
spp. and the like, as well as eukaryotic cells including a plant (dicot or monocot),<br>
fungus, yeast, e.g., Pichia, Saccharomyces or Schizosaccharomyces, or<br>
mammalian cell. Preferred mammalian cells include bovine, caprine, ovine,<br>
canine, feline, non-human primate, e.g., simian, and human cells. Preferred<br>
mammalian cell lines include, but are not limited to, CHO, COS, 293, Hela, CV-<br>
1, SH-SY5Y (human neuroblastoma cells), HEK293, and NIH3T3 cells.<br>
The expression of the encoded mutant hydrolase may be controlled by<br>
any promoter capable of expression in prokaryotic cells or eukaryotic cells.<br>
Preferred prokaryotic promoters include, but are not limited to, SP6, T7, T5, tac,<br>
bla, trp, gal, lac or maltose promoters. Preferred eukaryotic promoters include,<br>
but are not limited to, constitutive promoters, e.g., viral promoters such as CMV,<br>
SV40 and RSV promoters, as well as regulatable promoters, e.g., an inducible or<br>
repressible promoter such as the tet promoter, the hsp70 promoter and a<br>
synthetic promoter regulated by CRE. Preferred vectors for bacterial expression<br>
include pGEX-5X-3, and for eukaryotic expression include pCIneo-CMV.<br>
The nucleic acid molecule, expression cassette and/or vector of the<br>
invention may be introduced to a cell by any method including, but not limited<br>
to, calcium-mediated transformation, electroporation, microinjection,<br>
lipofection, particle bombardment and the like.<br>
III. Functional Groups<br>
Functional groups useful in the substrates and methods of the invention<br>
are molecules that are detectable or capable of detection. A functional group<br>
within the scope of the invention is capable of being covalently linked to one<br>
reactive substituent of a Wfunctional linker or a substrate for a hydrolase, and, as<br>
part of a substrate of the invention, has substantially the same activity as a<br>
functional group which is not linked to a substrate found in nature and is capable<br>
of forming a stable complex with a mutant hydrolase. Functional groups thus<br>
have one or more properties that facilitate detection, and optionally the isolation,<br>
of stable complexes between a substrate having that functional group and a<br>
mutant hydrolase. For instance, functional groups include those with a<br>
characteristic electromagnetic spectral property such as emission or absorbance,<br>
magnetism, electron spin resonance, electrical capacitance, dielectric constant or<br>
electrical conductivity as well as fiinctional groups which are ferromagnetic,<br>
paramagnetic, diamagnetic, luminescent, electrochemiluminescent, fluorescent,<br>
phosphorescent, chromatic, antigenic, or have a distinctive mass. A functional<br>
group includes, but is not limited to, a nucleic acid molecule, i.e., DNA or RNA,<br>
e.g., an oligonucleotide or nucleotide, a protein, e.g., a luminescent protein, a<br>
peptide, for instance, an epitope recognized by a ligand, e.g., biotin or ^<br>
streptavidin, a hapten, an amino acid, a lipid, a lipid bilayer, a solid support, a<br>
fluorophore, a chromophore, a reporter molecule, a radionuclide, an electron<br>
opaque molecule, a MRI contrast agent, e.g., manganese, gadolinium (III) or<br>
iron-oxide particles, and the like. Methods to detect a particular functional<br>
group are known to the art For example, a nucleic acid molecule can be<br>
detected by hybridization, amplification, binding to a nucleic acid binding<br>
protein specific for the nucleic acid molecule, enzymatic assays (e.g., if the<br>
nucleic acid molecule is a ribozyme), or, if the nucleic acid molecule itself<br>
comprises a molecule which is detectable or capable of detection, for instance, a<br>
radiolabel or biotin, it can be detected by an assay suitable for that molecule.<br>
Exemplary functional groups include haptens, e.g., molecules useful to<br>
enhance immunogenicity such as keyhole limpet hemacyanin (KLH), cleavable<br>
labels, for instance, photocleavable biotin, and fluorescent labels, e.g., Nhydroxysuccinimide<br>
(NHS) modified coumarin and succinimide or<br>
sulfonosuccinimide modified BODIPY (which can be detected by UV and/or<br>
visible excited fluorescence detection), rhodamine, e.g., Rl 10, rhodols, CRG6,<br>
Texas Methyl Red (TAMRA), Rox5, FAM, or fluoroscein, coumarin derivatives,<br>
e.g., 7 aminocoumarin, and 7-hydroxycoumarin, 2-amino-4-methoxynapthalene,<br>
1-hydroxypyrene, resorufin, phenalenones or benzphenalenones (U.S. Patent No.<br>
4,812,409), acridinones (U.S. Patent No. 4,810,636), anthracenes, and<br>
derivatives of a- and p-napthol, fluorinated xanthene derivatives including<br>
fluorinated fluoresceins and rhodols (e.g., U.S. Patent No. 6,162,931), and<br>
bioluminescent molecules, e.g., luciferase or GFP. A fluorescent (or<br>
bioluminescent) functional group linked to a mutant hydrolase by virtue of being<br>
linked to a substrate for a corresponding wild-type hydrolase, may be used to<br>
sense changes in a system, like phosphorylation, in real time. Moreover, a<br>
fluorescent molecule, such as a chemosensor of metal ions, e.g., a 9-<br>
carbonylanthracene modified glycyl-histidyl-lysine (GHK) for Cu2+, in a<br>
substrate of the invention may be employed to label proteins which bind the<br>
substrate. A bioluminescent or fluorescent functional group such as BODIPY,<br>
rhodamine green, GFP, or infrared dyes, also finds use as a functional group and<br>
may, for instance, be employed in interaction studies, e.g., using BRET, FRET,<br>
LRET or electrophoresis.<br>
Another class of functional group is a molecule that selectively interacts<br>
with molecules containing acceptor groups (an "affinity" molecule). Thus, a<br>
substrate for a hydrolase which includes an affinity molecule can facilitate the<br>
separation of complexes having such a substrate and a mutant hydrolase, because<br>
of the selective interaction of the affinity molecule with another molecule, e.g.,<br>
an acceptor molecule, that may be biological or non-biological in origin. For<br>
example, the specific molecule with which the affinity molecule interacts<br>
(referred to as the acceptor molecule) could be a small organic molecule, a<br>
chemical group such as a sulfhydryl group (-SH) or a large biomolecule such as<br>
an antibody or other naturally occurring ligand for the affinity molecule. The<br>
binding is normally chemical in nature and may involve the formation of<br>
covalent or non-covalent bonds or interactions such as ionic or hydrogen<br>
bonding. The acceptor molecule might be free hi solution or itself bound to a<br>
solid or semi-solid surface, a polymer matrix, or reside on the surface of a solid<br>
or semi-solid substrate. The interaction may also be triggered by an external<br>
agent such as light, temperature, pressure or the addition of a chemical or<br>
biological molecule that acts as a catalyst. The detection and/or separation of the<br>
complex from the reaction mixture occurs because of the interaction, normally a<br>
type of binding, between the affinity molecule and the acceptor molecule.<br>
Examples of affinity molecules include molecules such as immunogenic<br>
molecules, e.g., epitopes of proteins, peptides, carbohydrates or lipids, i.e., any<br>
molecule which is useful to prepare antibodies specific for that molecule; biotin,<br>
avidin, streptavidin, and derivatives thereof; metal binding molecules; and<br>
fragments and combinations of these molecules. Exemplary affinity molecules<br>
include His5 (HHHHH) (SEQ ID NO:19), HisX6 (HHHHHH) (SEQ ID NO:20),<br>
C-myc (EQKLISEEDL) (SEQ ID NO:21), Flag (DYKDDDDK) (SEQ ID<br>
N0:22), SteptTag (WSHPQFEK) (SEQ ID NO:23), HA Tag (YPYDWDYA)<br>
(SEQ ID NO:24), thioredoxin, cellulose binding domain, chitin binding domain,<br>
S-peptide, T7 peptide, calmodulin binding peptide, C-end KNA tag, metal<br>
binding domains, metal binding reactive groups, amino acid reactive groups,<br>
inteins, biotin, streptavidin, and maltose binding protein. For example, a<br>
substrate for a hydrolase which includes biotin is contacted with a mutant<br>
hydrolase. The presence of the biotin in a complex between the mutant<br>
hydrolase and the substrate permits selective binding of the complex to avidin<br>
molecules, e.g., streptavidin molecules coated onto a surface, e.g., beads,<br>
microwells, nitrocellulose and the like. Suitable surfaces include resins for<br>
chromatographic separation, plastics such as tissue culture surfaces or binding<br>
plates, microtiter dishes and beads, ceramics and glasses, particles including<br>
magnetic particles, polymers and other matrices. The treated surface is washed<br>
with, for example, phosphate buffered saline (PBS), to remove molecules that<br>
lack biotin and the biotin-containing complexes isolated. In some case these<br>
materials maybe part of biomolecular sensing devices such as optical fibers,<br>
chemfets, and plasmon detectors.<br>
Another example of an affinity molecule is dansyllysine. Antibodies<br>
which interact with the dansyl ring are commercially available (Sigma Chemical;<br>
St. Louis, MO) or can be prepared using known protocols such as described in<br>
Antibodies: A Laboratory Manual (Harlow and Lane, 1988). For example, the<br>
anti-dansyl antibody is immobilized onto the packing material of a<br>
chromatographic column. This method, affinity column chromatography,<br>
accomplishes separation by causing the complex between a mutant hydrolase<br>
and a substrate of the invention to be retained on the column due to its<br>
interaction with the immobilized antibody, while other molecules pass through<br>
the column. The complex may then be released by disrupting the antibodyantigen<br>
interaction. Specific chromatographic column materials such as ionexchange<br>
or affinity Sepharose, Sephacryl, Sephadex and other chromatography<br>
resins are commercially available (Sigma Chemical; St. Louis, MO; Pharmacia<br>
Biotech; Piscataway, N.J.). Dansyllysine may conveniently be detected because<br>
of its fluorescent properties.<br>
When employing an antibody as an acceptor molecule, separation can<br>
also be performed through other biochemical separation methods such as<br>
immunoprecipitation and immobilization of antibodies on filters or other<br>
surfaces such as beads, plates or resins. For example, complexes of a mutant<br>
hydrolase and a substrate of the invention may be isolated by coating magnetic<br>
beads with an affinity molecule-specific or a hydrolase-specific antibody. Beads<br>
are oftentimes separated from the mixture using magnetic fields.<br>
Another class of functional molecules includes molecules detectable<br>
using electromagnetic radiation and includes but is not limited to xanthene<br>
fluorophores, dansyl fluorophores, coumarins and coumarin derivatives,<br>
fluorescent acridinium moieties, benzopyrene based fluorophores, as well as 7-<br>
nitrobenz-2-oxa-l,3-diazole, and 3-N-(7-nitrobenz-2-oxa-l,3-diazol-4-yl)-2,3-<br>
diamino-propionic acid. Preferably, the fluorescent molecule has a high<br>
quantum yield of fluorescence at a wavelength different from native amino acids<br>
and more preferably has high quantum yield of fluorescence that can be excited<br>
in the visible, or in both the UV and visible, portion of the spectrum. Upon<br>
excitation at a preselected wavelength, the molecule is detectable at low<br>
concentrations either visually or using conventional fluorescence detection<br>
methods. Electrochemiluminescent molecules such as ruthenium chelates and its<br>
derivatives or nitroxide amino acids and their derivatives are detectable at<br>
femtomolar ranges and below.<br>
In addition to fluorescent molecules, a variety of molecules with physical<br>
properties based on the interaction and response of the molecule to<br>
electromagnetic fields and radiation can be used to detect complexes between a<br>
mutant hydrolase and a substrate of the invention. These properties include<br>
absorption in the UV, visible and infrared regions of the electromagnetic<br>
spectrum, presence of chromophores which are Raman active, and can be further<br>
enhanced by resonance Raman spectroscopy, electron spin resonance activity<br>
and nuclear magnetic resonances and molecular mass, e.g., via a mass<br>
spectrometer.<br>
Methods to detect and/or isolate complexes having affinity molecules<br>
include chromatographic techniques including gel filtration, fast-pressure or<br>
high-pressure liquid chromatography, reverse-phase chromatography, affinity<br>
chromatography and ion exchange chromatography. Other methods of protein<br>
separation are also useful for detection and subsequent isolation of complexes<br>
between a mutant hydrolase and a substrate of the invention, for example,<br>
electrophoresis, isoelectric focusing and mass spectrometry.<br>
IV. Linkers<br>
The term "linker", which is also identified by the symbol 'L1, refers to a<br>
group or groups that covalently attach one or more functional groups to a<br>
substrate which includes a reactive group or to a reactive group. A linker, as<br>
used herein, is not a single covalent bond. The structure of the linker is not<br>
crucial, provided it yields a substrate that can be bound by its target enzyme. In<br>
one embodiment, the linker can be a divalent group that separates a functional<br>
group (R) and the reactive group by about 5 angstroms to about 1000 angstroms,<br>
inclusive, in length. Other suitable linkers include linkers that separate R and the<br>
reactive group by about 5 angstroms to about 100 angstroms, as well as linkers<br>
that separate R and the substrate by about 5 angstroms to about SO angstroms, by<br>
about 5 angstroms to about 25 angstroms, by about 5 angstroms to about 500<br>
angstroms, or by about 30 angstroms to about 100 angstroms.<br>
In one embodiment the linker is an amino acid.<br>
In another embodiment, the linker is a peptide.<br>
In another embodiment, the linker is a divalent branched or unbranched<br>
carbon chain comprising from about 2 to about 30 carbon atoms, which chain<br>
optionally includes one or more (e.g., 1,2,3, or 4) double or triple bonds, and<br>
which chain is optionally substituted with one or more (e.g., 2,3, or 4) hydroxy<br>
or oxo (=O) groups, wherein one or more (e.g., 1,2,3, or 4) of the carbon atoms<br>
in the chain is optionally replaced with a non-peroxide -O-, -S- or -NH-.<br>
In another embodiment, the linker is a divalent group of the formula -WF-<br>
W- wherein F is (Ci-C3o)alkyl, (C2-C3o)alkenyl, (C2-C3o)alkynyl, (C3-<br>
C8)cycloalkyl, or (C6-Cio)aryl, wherein W is -N(Q)C(=O)-, -C(=O)N(O&gt;,<br>
-OC(=O)-, -C(=O)O-, -O-, -S-, -S(O)-, -S(O)2-, -N(Q)-, -C(=0)-, or a direct<br>
bond; wherein each Q is independently H or (Ci-Cfi)alkyl<br>
In another embodiment, the linker is a divalent branched or unbranched<br>
carbon chain comprising from about 2 to about 30 carbon atoms, which chain<br>
optionally includes one or more (e.g., 1,2,3, or 4) double or triple bonds, and<br>
which chain is optionally substituted with one or more (e.g., 2, 3, or 4) hydroxy<br>
or oxo (=O) groups.<br>
In another embodiment, the linker is a divalent branched or unbranched<br>
carbon chain comprising from about 2 to about 30 carbon atoms, which chain<br>
optionally includes one or more (e.g., 1 , 2, 3, or 4) double or triple bonds.<br>
In another embodiment, the linker is a divalent branched or unbranched<br>
carbon chain comprising from about 2 to about 30 carbon atoms.<br>
In another embodiment, the linker is a divalent branched or unbranched<br>
carbon chain comprising from about 2 to about 20 carbon atoms, which chain<br>
optionally includes one or more (e.g., 1, 2, 3, or 4) double or triple bonds, and<br>
which chain is optionally substituted with one or more (e.g., 2, 3, or 4) hydroxy<br>
or oxo (=O) groups.<br>
In another embodiment, the linker is a divalent branched or unbranched<br>
carbon chain comprising from about 2 to about 20 carbon atoms, which chain<br>
optionally includes one or more (e.g., 1, 2, 3, or 4) double or triple bonds.<br>
In another embodiment, the linker is a divalent branched or unbranched<br>
carbon chain comprising from about 2 to about 20 carbon atoms.<br>
In another embodiment, the linker is -(CH2CH2O)-i.io.<br>
In another embodiment, the linker is -C(-O)NH(CH2)3S<br>
-C(=0)NH(CH2)5C(=0)NH(CH2)-;-CH2OC(=O)NH(CH2)20(CH2)20(CH2)-;<br>
-(CH2)4C(=0)NH(CH2)20(CH2)20(CH2)3-;<br>
-C(=0)NH(CH2)5C(=0)NH(CH2)20(CH2)20(CH2)3-;<br>
Specifically, (Ci-C3o)alkyl can be methyl, ethyl, propyl, isopropyl, butyl,<br>
iso-butyl, sec-butyl, pentyl, 3-pentyl, hexyl, heptyl, octyl, nonyl, or decyl; (C3-<br>
Cg)cycloalkyl can be cyclopropyl, cyclobutyl, cyclopentyl, or cyclohexyl;(C2-<br>
CsoJalkenyl can be vinyl, allyl, 1-propenyl, 2-propenyl, 1 -butenyl, 2-butenyl, 3-<br>
butenyl, 1,-pentenyl, 2-pentenyl, 3-pentenyl, 4-pentenyl, 1- hexenyl, 2-hexenyl,<br>
3-hexenyl, 4-hexenyl, 5-hexenyl, heptenyl, octenyl, nonenyl, or decenyl; (C2-<br>
C30)alkynyl can be ethynyl, 1-propynyl, 2-propynyl, 1-butynyl, 2-butynyl, 3-<br>
butynyl, 1-pentynyl, 2-pentynyl, 3-pentynyl, 4-pentynyl, 1- hexynyl, 2-hexynyl,<br>
3-hexynyl, 4-hexynyl, 5-hexynyl, heptynyl, octynyl, nonynyl, or decynyl; and<br>
(Ce-CioJaryl can be phenyl, indenyl, or naphthyl.<br>
The term "amino acid," when used with reference to a linker, comprises<br>
the residues of the natural amino acids (e.g., Ala, Arg, Asn, Asp, Cys, Glu, Gin,<br>
Gly, His, Hyl, Hyp, He, Leu, Lys, Met, Phe, Pro, Ser, Thr, Trp, Tyr, and Val) in<br>
D or L form, as well as unnatural amino acids (e.g., phosphoserine,<br>
phosphothreonine, phosphotyrosine, hydroxyproline, gamma-carboxyglutamate;<br>
hippuric acid, octahydroindole-2-carboxylic acid, statine,<br>
1,2,3,4,-tetrahydroisoquinoline-3-carboxylic acid, penicillamine, ornithine,<br>
citruline, a-methyl-alanine, para-benzoylphenylalanine, phenylglycine,<br>
propargylglycine, sarcosine, and tert-butylglycine). The term also includes<br>
natural and unnatural amino acids bearing a conventional amino protecting<br>
group (e.g., acetyl or benzyloxycarbonyl), as well as natural and unnatural amino<br>
acids protected at the carboxy terminus (e.g. as a (Ci-C6)alkyl, phenyl or benzyl<br>
ester or amide). Other suitable amino and carboxy protecting groups are known<br>
to those skilled in the art (see for example, Greene, Protecting Groups In<br>
Organic Synthesis; Wiley: New York, 1981, and references cited therein). An<br>
amino acid can be linked to another molecule through the carboxy terminus, the<br>
amino terminus, or through any other convenient point of attachment, such as,<br>
for example, through the sulfur of cysteine.<br>
The term "peptide" when used with reference to a linker, describes a<br>
sequence of 2 to 25 amino acids (e.g. as defined hereinabove) or peptidyl<br>
residues. The sequence may be linear or cyclic. For example, a cyclic peptide<br>
can be prepared or may result from the formation of disulfide bridges between<br>
two cysteine residues in a sequence. A peptide can be linked to another<br>
molecule through the carboxy terminus, the amino terminus, or through any<br>
other convenient point of attachment, such as, for example, through the sulfur of<br>
a cysteine. Preferably a peptide comprises 3 to 25, or 5 to 21 amino acids.<br>
Peptide derivatives can be prepared as disclosed in U.S. Patent Numbers<br>
4,612,302; 4,853,371; and 4,684,620. Peptide sequences specifically recited<br>
herein are written with the amino terminus on the left and the carboxy terminus<br>
on the right.<br>
In one embodiment, a substrate of the invention for a dehalogenase<br>
which has a linker has the formula (I):<br>
Rโlinker-A-X (I)<br>
wherein R is one or more functional groups (such as a fluorophore, biotin,<br>
luminophore, or a fluorogenic or luminogenic molecule, or is a solid support,<br>
including microspheres, membranes, glass beads, and the like), wherein the<br>
linker is<br>
a multiatom straight or branched'chain including C, N, S, or O, wherein A-X is a<br>
substrate for a dehalogenase, and wherein X is a halogen. In one embodiment,<br>
A-X is a haloaliphatic or haloaromatic substrate for a dehalogenase. In one<br>
embodiment, the linker is a divalent branched or unbranched carbon chain<br>
comprising from about 12 to about 30 carbon atoms, which chain optionally<br>
includes one or more (e.g., 1,2,3, or 4) double or triple bonds, and which chain<br>
is optionally substituted with one or more (e.g., 2,3, or 4) hydroxy or oxo (=O)<br>
groups, wherein one or more (e.g., 1,2,3, or 4) of the carbon atoms in the chain<br>
is optionally replaced with a non-peroxide -O-, -S- or -NH-. In one embodiment,<br>
A is CHaCHj or CKfeCHbCHj. In one embodiment, a linker in a substrate for a<br>
dehalogenase such as a Rhodococcus dehalogenase, is a multiatom straight or<br>
branched chain including C, N, S, or O, and preferably 11-30 atoms when the<br>
functional group R includes an aromatic ring system or is a solid support.<br>
In another embodiment, a substrate of the invention for a dehalogenase<br>
which has a linker has formula (II):<br>
Rโlinker-CH2-CH2-CH2-X (II)<br>
where X is a halogen, preferably chloride. In one embodiment, R is one or more<br>
functional groups, such as a fluorophore, biotin, luminophore, or a fluorogenic or<br>
luminogenic molecule, or is a solid support, including microspheres,<br>
membranes, glass beads, and the like. When R is a radiolabel, or a small<br>
detectable atom such as a spectroscopically active isotope, the linker can be 0-30<br>
atoms.<br>
V. Svnthsscs for Exemplary Substrates<br>
[2-(2-Hydroxy-ethoxy)-ethyl]-carbamic acid anthracen-9-ylmethyI ester. To<br>
a stirring slurry of 9-anthracenemethanol (10 g, 48 mmol) and 4-nitrophenyl<br>
chloroformate (13.6 g, 67.5 mmol) in 200 ml CHjCk was added triethylamine<br>
 (6.7 ml, 0.19 mol). The resulting gold colored solution was allowed to stir 16<br>
hrs at room temperature. At this point, 2-(2-aminoethoxy)ethanol (14.4 ml,<br>
0.144 mol) was added and stirring continued for another 24 hours. The CH2C12<br>
reaction mixture was then washed with a 2% sodium hydroxide (w/w) solution<br>
until no/&gt;-nitrophenol was observed in the organic layer. The dichloromethane<br>
was dried with sodium sulfate, filtered, and evaporated under reduced pressure.<br>
The crude product was further purified by column chromatography on<br>
silica gel 60, progressively eluting with 1% to 3% methanol in dichloromethane.<br>
7.6 g (58% yield) of a yellow solid was isolated: 'H NMR (CDC13) 8 8.38 (s, H-<br>
10), 8.28 (d, H-l, 8), 7.94 (d, H-4,5), 7.44 (m, H-2,3,6, 7), 6.06 (s, Qfc-anth),<br>
5.47 (t, exchangeable, NH), 3.53 (bs, Qfc-OH) 3.33 (m, three -GHz-)- Mass<br>
spectrum, m/e Calcd for QjoHaaNCV: 340.15. Found: 340.23. Calcd for<br>
C20H2iNNaO/: 340.15. Found: 340.23.<br>
a compound of formula in<br>
{2-[2-(6-Chloro-hexyloxy)-ethoxyJ-ethyl}-carbamlc acid anthracen-9-<br>
ylmethyl ester. A 100 ml round bottom flask was charged with [2-(2-Hydroxyethoxy)-<br>
ethyl]-carbanu'c acid anthracen-9-ylmethyl ester (1.12 g, 3mmol) and<br>
fresh sodium hydride, 60% dispersion in mineral oil (360 mg, 9mmol) under<br>
inert atmosphere. 20 ml anhydrous THF was added and the reaction allowed to<br>
stir for 30 minutes. The flask is then cooled to between -10 and -20ยฐC by<br>
means of an ice/Nad bath. When the temperature is reached l-chloro-6-<br>
lodohexane (1 ml, 6 mmol) is added via syringe. The reaction is maintained at<br>
ice/NaCI temperature for 2 hours, then slowly allowed to warm to room<br>
temperature overnight. At this point silica gel 60 is co-absorbed onto the<br>
reaction mixture with loss of solvent under reduced pressure. Silica gel<br>
chromatography takes place initially with heptane as eluent, followed by 10%,<br>
20%, and 25% ethyl acetate. A total of 0.57 g (41% yield) of product is isolated<br>
from appropriate fractions: 1H NMR (CDC13) 8 8.48 (s, H-10), 8.38 (d, H-l, 8),<br>
8.01 (d, H-4, 5), 7.52 (dt, H-2,3,6,7), 6.13 (s, Cfiz-anth), 5.29 (bs,<br>
exchangeable, NH), 3.74 (m, 4H), 3.55-3.15 (m, 8H), 1.84 (m, 4H), 1.61 (m,<br>
1H), 1.43 (m, 1H), 1.25 (m, 2H). Mass spectrum, m/e Calcd for<br>
C26H32C1NO4H2O: 475.21(100%), 476.22(29.6%). Found: 475.21,476.52.<br>
a compound of formula IV<br>
2-[2-(6-chlorohexyloxy)-ยซthoxy]-ethyI-ammonium trifluoro-acetate. To {2-<br>
[2-(6-Chloro-hexyloxy)-ethoxy]-ethyl}-carbamic acid anthracen-9-yhnethyl<br>
ester (0.56 g, 1.2 mmol) dissolved in 4 ml dichloromethane was added 2 drops of<br>
anisole. The reaction mixture is cooled by means of an ice/Nad bath. After 10<br>
minutes trifluoroacetic acid (2 ml) is added. The reaction mixture turns dark<br>
brown upon addition and is allowed to stir for 30 minutes. All volatiles are<br>
removed under reduced atmosphere. The residue is re-dissolved in CHiClj and<br>
washed twice with water. The aqueous fractions are frozen and lyophilized<br>
overnight. An oily residue remains and is dissolved in anhydrous DMF to be<br>
used as a stock solution in further reactions. Mass spectrum, m/e Calcd for<br>
Ci0H23ClN02<br>
+: 224.14(100%), 226.14(32%). Found: 224.2, 226.2.<br>
a compound of formula V<br>
General methodology for reporter group conjugation to 2-[2-(6-chlorohexyloxy)-<br>
ethoxy]-ethylamuie. To one equivalent of the succinimidyl ester of<br>
the reporter group in DMF is added 3 equivalence of 2-[2-(6-chlorohexyloxy)-<br>
ethoxyj-ethyl-ammonium trifluoro-acetate stock solution, followed by<br>
diisopropylethylamine. The reaction is stirred from 8 to 16 hours at room<br>
temperature. Purification is accomplished by preparative scale HPLC or silica<br>
gel chromatography.<br>
N-{2-[2-(6-Chlorohexyloxy)-ethoxy]-ethyl}-fluoresceiJi-5-amide. The title<br>
compound was prepared using the above methodology. Purification was<br>
accomplished using preparative scale HPLC. Mass spectrum, m/e Calcd for<br>
': 580.17(100%), 581.18(32%). Found: 580.18, 581.31.<br>
a compound of formula VI<br>
N-{2-[2-(6-ChlorohexyloJcy)-ethoxy]-ethyl}-biotin-amlde. The title compound<br>
was prepared using the above methodology. Purification was accomplished<br>
using silica gel chromatography (2% to 5% methanol in dichlororaethane). Mass<br>
spectrum, m/e Calcd for C2oH37ClN3O4S+: 450.22(100%), 452.22(32%). Found:<br>
449.95,451.89.<br>
a compound of formula VII<br>
N-{2-[2-(6-Chlorohexyloxy)-ethoxy]-ethyl}-tetramethyIrhodamlne-5-(and-<br>
6)-amide. The titie compound was prepared using the above methodology.<br>
Purification was accomplished using preparative scale HPLC. Separation of<br>
structural isomers was realised. Mass spectrum, m/e Calcd for Css^ClNsOe*:<br>
636.28(100%), 637.29(39.8%), 638.28(32.4%). Found: 636.14,637.15, 638.14.<br>
V-CI<br>
a compound of formula VIII a compound of formula IX<br>
N-{2-[2-(6-Chlorohexyloxy)-ethoxy]-ethyl}-rhodamine R110-5-(and -6)-<br>
amide. The title compound was prepared using the above methodology.<br>
Purification was accomplished using preparative scale HPLC. Separation of<br>
structural isomers was realized. Mass spectrum, m/e Calcd for CsiHssClNsO<br>
580.2(100%), 581.2(35.6%), 582.2(32,4%). Found: 580.4, 581.4, 582.2.<br>
a compound of formula X a compound of formula XI<br>
6-({4-[4,4difluoro-5-(thIophen-2-ylH-bora-3a-4a-diaza-s-lndacene-3-<br>
yl]phenoxy}-acetylamino)-hexanoic add {2-[2-(6-chlorohexyIoxy)-ethoxy]-<br>
ethyl}-amide. The title compound was prepared using the above methodology.<br>
Purification was accomplished using silica gel chromatography (3% to 5%<br>
methanol in dichloromethane). Mass spectrum, m/e Calcd for<br>
C37H47BC1F2N4O5S+: 743.3(100%). Found: 743.4.<br>
a compound of formula XII<br>
6-({4-[4,4difluoro-5-(thlophen-2-yl)-4-bora-3a-4a-diaza-s-indacene-3-<br>
yllsryryloxy}-acetylamino)-hexanolc acid {2-[2-(6-chIorohexyloxy)-ethoxyJethyl}-<br>
amide. The title compound was prepared using the above methodology.<br>
Purification was accomplished using silica gel chromatography (3% methanol in<br>
dichloromethane). Mass spectrum, m/e Calcd for C39H48BClF2N4NaO5S*:<br>
791.3(100%). Found: 7.91.3.<br>
a compound of formula XIII<br>
Triethylammoniuin3-[5-[2-(4-tert-Butyl-7-dlethylamIno-chroinen-2-<br>
yUdene)-ethyl!dene]-3-(5-{2-[2-(6-chlorohexyloxy)-ethoxy]-ethylcarbamoyl}-<br>
pentyl)-2,4,6-trioxo-tetrahydro-pyrimidIn-l-yl]-propane-l-suIfonicacid<br>
anion. The title compound was prepared using the above methodology.<br>
Purification was accomplished using preparative scale HPLC. Mass spectrum,<br>
m/e Calcd for C42H62ClN4OioS": 849.4(100%), 850.4(48.8%), 851.4(36.4%).<br>
Found: 849.6, 850.5,851.5.<br>
a compound of formula XIV<br>
2-tert-Butyl-4-{3-[l-(5-{2-[2-(6-chloroJiexyloxy)-ethoxy]-ethylcarbamoyl}-<br>
pentyl)-3,3-dlmethyl-5-sulfo-l,3-dlhydro-lndol-2-ylidene]-propenyl}-7-<br>
diethylamino-chromenyllum chloride. The title compound was prepared using<br>
58<br>
the above methodology. Purification was accomplished using preparative scale<br>
HPLC. Mass spectrum, m/e Calcd for C46H67C1N3O7S": 840.4(100%),<br>
841.4(54.4%). Found: 840,5, 841.5.<br>
a compound of formula XV<br>
N-{2-[2-(6-ChJoroheiyloxy)-ethoxy]-ethyl}-3-{4-[5-(4-dimethylaminophenyI)-<br>
oxazol-2-yl]-benzenesulfonylamino}-propionamide. The title<br>
compound was prepared using the above methodology. Purification was<br>
accomplished using preparative scale HPLC. Mass spectrum, m/e Calcd for<br>
C3oH4oClN4O6S': 619.2(100%), 620.2(35%). Found: 619.5,620.7.<br>
a compound of formula XVI<br>
N-{2-[2-(6-Chlorohexyloxy)-ethoxy]-ยซthyl}-9'-<br>
chloro8emlnaphthofluorescein-5-(and -6)-amlde. The title compound was<br>
prepared using the above methodology. Purification was accomplished using<br>
preparative scale HPLC. Separation of structural isomers was realized. Mass<br>
spectrum, m/e Calcd for C3sH34Cl2N<y:></y:>
667.17(39.8%). Found: 666.46,668.44,667.51.<br>
a compound of formula XVII<br>
XVIII<br>
a compound of formula<br>
N-{2-(2-((ยป-Clilorohexyloxy)-ethoxy]-etliyl}-<br>
seminaphthodimethylrhodamine-5-(and -6)-amide. The title compound was<br>
prepared using the above methodology. Purification was accomplished using<br>
preparative scale HPLC. Mass spectrum, m/e Calcd for CaTHsgCn^Oy":<br>
657.24(100%), 658.24(42%), 659.23(32%). Found: 657.46, 658.47, 659.45.<br>
a compound of formula XDC a compound of formula XX<br>
6-(3&gt;,6'-dipivaloylfluorescein-5-(and-6)-carboxamido) hexanoic acid {2-[2-<br>
(6-chIorohexyloxy)-ethoxy]-ethyl}-amIde. To a 100 ml round bottom flask<br>
containing 6-(3',6'-dipivaloylfluorescein-5-(and-6)-carboxamido) hexanoic acid<br>
succinimidyl ester (0.195g, 0.26 mmol) was added 2-[2-(6-chlorohexyloxy)-<br>
ethoxyl-ethylamine (~ 0.44 mmol) in 25 ml Et2O, followed by 2 ml of pyridine.<br>
The reaction mixture was allowed to stir overnight. After evaporation under<br>
reduced pressure, the residue was subjected to silica gel 60 column<br>
chromatography, progressively using 2% to 5% methanol in dichloromethane as<br>
eluent. The appropriate fractions were collected and dried under vacuum (0.186<br>
g, 0.216 mmol, and 84% yield). Mass spectrum, m/e Calcd for C47H6oClN2Ou+:<br>
863.39(100%), 864.39(54.4%), 865.39(34.6%). Found: 862.94, 864.07, 864.94.<br>
a compound of formula XXI a compound of formula-XXII<br>
6-(fluorescein-5-(and-6)-carboxamido) hexanoic acid {2-[2-(6-<br>
chlorohexyloxy)-ethoxy]-ethyl}-amide. 6-(3',6'-dipivaloylfluorescein-5-(and-<br>
6)-carboxamido) hexanoic acid {2-[2-(6-chlorohexyloxy)-ethoxy]-ethyl}-amide<br>
(0.186 g, 0.216 mmol) was dissolved in 5 ml methanol and 0.5 ml 2M sodium<br>
carbonate(aq) added. The reaction mixture was stirred for 16 hours, then<br>
filtered. Purification was accomplished using preparative scale HPLC.<br>
Separation of structural isomers was realized. Mass spectrum, m/e Calcd for<br>
C37H44C1N2<v found:></v>
a compound of formula XXIII a compound of formula XXIV<br>
{2-[2-(4-ChIorobutoxy)-ethoxy]-ethyl}-carbainIcacidanthracen-9-ylmethyl<br>
ester. A 50 ml round bottom flask was charged with [2-(2-Hydroxyethoxy)-<br>
ethylj-carbamic acid anthracen-9-ylmethyl ester (0.25 g, 0.74 mmol) and fresh<br>
sodium hydride, 60% dispersion in mineral oil (150 mg, 3.75 mmol) under inert<br>
atmosphere. 10 ml anhydrous THF was added and the reaction allowed to stir<br>
for 5 minutes. After this point, l-chloro-4-Iodobutane (180 \il, 1.5 mmol) is<br>
added via syringe. The reaction is stirred at room temperature for 24 hours.<br>
Silica gel 60 is co-absorbed onto the reaction mixture with loss of solvent under<br>
reduced pressure. Silica gel column chromatography takes place initially with<br>
heptane as eluent, followed by 10%, 20%, and 30%* ethyl acetate. A total of 0.1<br>
g (32% yield) of product is isolated from appropriate fractions: 1H NMR<br>
(CDC13) 8 8.50 (s, H-10), 8.40 (d, H-l, 8), 8.03 (d, H-4, 5), 7.53 (dt, H-2,3, 6,<br>
7), 6.15 (s, CHa-anth), 5.19 (m, exchangeable, Ng), 3.93-3.32 (m, 12H) 1.69-<br>
1.25 (m, 4H). Mass spectrum, m/e Calcd for C24H28C1NO4 H2O: 447.18<br>
(100.0%), 448.18 (27.1%). Found: 447.17,448.41.<br>
62<br>
a compound of formula XXV<br>
2-(2-{2-[2-(2-Chloroethoxy)-ethoxyj-ethoxy}-ethyl)-isoindole-l,3-dione. 2-(2-<br>
{2-[2-(2-Hydroxy-ethoxy)-ethoxy]-ethoxy}-ethyl)-isoindole-l ,3-dione (0.5 g,<br>
1.55 mmol) was prepared by the method of Nielsen, J. and Janda, K.D.<br>
(Methods: A Companion to Methods in Enzymology 6,361-371 (1994)). To<br>
this reagent was added polystyrene-supported triphenylphosphine about 3 mmol<br>
P/g (0.67 g, 2 mmol) and 6 ml carbon tetrachloride, into a 25 ml round bottom<br>
fitted with a reflux condenser. The reaction set-up was sparged with argon then<br>
heated to reflux for 2 hours. Upon cooling, more polystyrene-supported<br>
triphenylphosphine (0.1 g, 0.3 mmol) was added and the reaction refluxed for an<br>
additional one hour. The cooled solution was filtered and the resin washed with<br>
additional carbon tetrachloride. Evaporation of solvent yielded 0.4 g (75.5%<br>
yield) of pure title compound: *H NMR (CDC13) 8 7.82 (dd, 2 H), 7.69 (dd, 2H),<br>
3.88 (t, 2H), 3.71 (q, 4 H), 3.63-3.56 (m, 12H). Mass spectrum, m/e Calcd for<br>
Ci6H2iClN05<br>
+: 342.11 (100.0%), 344.11 (32.0%). Found: 341.65,343.64.<br>
a compound of formula XXVI<br>
2-[2-(2-{2-[2-(2-Chloroethoxy)-ethoxy]-ethoxy}-ethoxy)-ethyl]-isoindole-l,3-<br>
dione. The title compound was prepared according to the previous example in<br>
89% yield: 1H NMR (CDC13) 5 7.77 (dd, 2 H), 8 7.64 (dd, 2H), 3.83 (t, 2H), 3.67<br>
(m, 4 H), 3.60-3.52 (m, 14H). Mass spectrum, m/e Calcd for<br>
386.14 (100.0%), 388.13 (32.0%). Found: 385.88, 387.83.<br>
a compound of formula XXVII<br>
2-{2-[2-(2-{2-[2-(2-Chloroethoxy)-ethoxy]-ethoxy}-ethoxy)-ethoxy]-ethyl}-<br>
isoindole-l,3-dione. The title compound was prepared according to the<br>
synthesis of 2-(2- {2-[2-(2-Chloro-ethoxy)-ethoxy]-ethoxy} -ethyl)-isoindole-<br>
1,3-dione in 92% yield: 1H NMR (CDC13) 6 7.84 (dd, 2 H), 7.71 (dd, 2H), 3.90<br>
(t, 2H), 3.74 (q, 4 H), 3.67-3.58 (m, 18H). Mass spectrum, m/e Calcd for<br>
C2oH29ClNO7<br>
+: 430.16 (100.0%). Found: 429.85.<br>
a compound of formula XXVIII<br>
VI. Exemplary Methods of Use<br>
The invention provides methods to monitor the expression, location<br>
and/or trafficking of molecules in a cell, as well as to monitor changes in<br>
microenvironments within a cell. In one embodiment, a mutant hydrolase and a<br>
corresponding substrate which includes a functional group are employed to label<br>
a cell, e.g., a cell in an organism or cell culture, or a cellular component. For<br>
instance, cells are contacted with a vector encoding the mutant hydrolase, such<br>
as one encoding a fusion between the mutant hydrolase and a nuclear<br>
localization signal. The expression of the vector in the cell may be transient or<br>
stable. Then the cell is contacted with a substrate of the invention recognized by<br>
the mutant hydrolase. Alternatively, cells are concurrently contacted with the<br>
vector and the substrate. Then the presence or location of the functional group<br>
of the substrate in the cell, a lysate thereof, or a subcellular fraction thereof, is<br>
detected or determined.<br>
The substrates of the invention are preferably soluble in an aqueous or<br>
mostly aqueous solution, including water and aqueous solutions having a pH<br>
greater than or equal to about 6. Stock solutions of substrates of the invention,<br>
however, may be dissolved in organic solvent before diluting into aqueous<br>
solution or buffer. Preferred organic solvents are aprotic polar solvents such as<br>
DMSO, DMF, N-methylpyrrolidone, acetone, acetonitrile, dioxane,<br>
tetrahydrofuran and other nonhydroxylic, completely water-miscible solvents. In<br>
general, the amount of substrate of the invention employed is the minimum<br>
amount required to detect the presence of the functional group in the sample<br>
comprising a mutant hydrolase or a fusion thereof, within a reasonable time,<br>
with minimal background or undesirable labeling. The exact concentration of a<br>
substrate of the invention and a corresponding mutant hydrolase to be used is<br>
dependent upon the experimental conditions and the desired results. The<br>
concentration of a substrate of the invention typically ranges from nanomolar to<br>
micromolar. The required concentration for the substrate of the invention with a<br>
corresponding mutant hydrolase is determined by systematic variation in<br>
substrate until satisfactory labeling is accomplished. The starting ranges are<br>
readily determined from methods known in the art.<br>
In one embodiment, a substrate which includes a functional group with<br>
optical properties is employed with a mutant hydrolase to label a sample. Such a<br>
substrate is combined with the sample of interest comprising the mutant<br>
hydrolase for a period of time sufficient for the mutant hydrolase to bind the<br>
substrate, after which the sample is illuminated at a wavelength selected to elicit<br>
the optical response of the functional group. Optionally, the sample is washed to<br>
remove residual, excess or unbound substrate. In one embodiment, the labeling<br>
is used to determine a specified characteristic of the sample by further<br>
comparing the optical response with a standard or expected response. For<br>
example, the mutant hydrolase bound substrate is used to monitor specific<br>
components of the sample with respect to their spatial and temporal distribution<br>
in the sample. Alternatively, the mutant hydrolase bound substrate is employed<br>
to determine or detect the presence or quantity of a certain molecule. In another<br>
embodiment, the mutant hydrolase bound substrate is used to analyze the sample<br>
for the presence of a molecule that responds specifically to the functional group.<br>
A detectable optical response means a change in, or occurrence of, a<br>
parameter in a test system that is capable of being perceived, either by direct<br>
observation or instrumentally. Such detectable responses include the change in,<br>
or appearance of, color, fluorescence, reflectance, chemiluminescence, light<br>
polarization, light scattering, or x-ray scattering. Typically the detectable<br>
response is a change in fluorescence, such as a change in the intensity, excitation<br>
or emission wavelength distribution of fluorescence, fluorescence lifetime,<br>
fluorescence polarization, or a combination thereof. The detectable optical<br>
response may occur throughout the sample comprising a mutant hydrolase or a<br>
fusion thereof or in a localized portion of the sample comprising a mutant<br>
hydrolase or a fusion thereof. Comparison of the degree of optical response with<br>
a standard or expected response can be used to determine whether and to what<br>
degree the sample comprising a mutant hydrolase or a fusion thereof possesses a<br>
given characteristic.<br>
In another embodiment, the functional group is a ligand for an acceptor<br>
molecule. Typically, where the substrate comprises a functional group that is a<br>
member of a specific binding pair (a ligand), the complementary member (the<br>
acceptor) is immobilized on a solid or semi-solid surface, such as a polymer,<br>
polymeric membrane or polymeric particle (such as a polymeric bead).<br>
Representative specific binding pairs include biotin and avidin (or streptavidin or<br>
anti-biotin), IgG and protein A or protein G, drug and drug receptor, toxin and<br>
toxin receptor, carbohydrate and lectin or carbohydrate receptor, peptide and<br>
peptide receptor, protein and protein receptor, enzyme substrate and enzyme,<br>
sense DNA or RNA and antisense (complementary) DNA or RNA, hormone and<br>
hormone receptor, and ion and chelator. Ligands for which naturally occurring<br>
receptors exist include natural and synthetic proteins, including avidin and<br>
streptavidin, antibodies, enzymes, and hormones; nucleotides and natural ori<br>
synthetic oligonucleotides, including primers for RNA and single- and doublestranded<br>
DNA; lipids; polysaccharides and carbohydrates; and a variety of drugs,<br>
including therapeutic drugs and drugs of abuse and pesticides. Where the<br>
functional group is a chelator of calcium, sodium, magnesium, potassium, or<br>
another biologically important metal ion, the substrate comprising such a<br>
functional group functions as an indicator of the ion. Alternatively, such a<br>
substrate may act as a pH indicator. Preferably, the detectable optical response of<br>
the ion indicator is a change in fluorescence.<br>
The sample comprising a mutant hydrolase or a fusion thereof is typically<br>
labeled by passive means, i.e., by incubation with the substrate. However, any<br>
method of introducing the substrate into the sample comprising a mutant<br>
hydrolase or a fusion thereof, such as microinjection of a substrate into a cell or<br>
organelle, can be used to introduce the substrate into the sample comprising a<br>
mutant hydrolase or a fusion thereof. The substrates of the present invention are<br>
generally non-toxic to living cells and other biological components, within the<br>
concentrations of use.<br>
The sample comprising a mutant hydrolase or a fusion thereof can be<br>
observed immediately after contact with a substrate of the invention. The sample<br>
comprising a mutant hydrolase or a fusion thereof is optionally combined with<br>
other solutions in the course of labeling, including wash solutions,<br>
permeabilization and/or fixation solutions, and other solutions containing<br>
additional detection reagents. Washing following contact with the substrate<br>
generally improves the detection of the optical response due to the decrease in<br>
non-specific background after washing. Satisfactory visualization is possible<br>
without washing by using lower labeling concentrations. A number of fixatives<br>
and fixation conditions are known in the art, including formaldehyde,<br>
paraformaldehyde, formalin, glutaraldehyde, cold methanol and 3:1<br>
methanol: acetic acid. Fixation is typically used to preserve cellular morphology<br>
and to reduce biohazards when working with pathogenic samples. Selected<br>
embodiments of the substrates are well retained in cells. Fixation is optionally<br>
followed or accompanied by permeabilization, such as with acetone, ethanol,<br>
DMSO or various detergents, to allow bulky substrates of the invention, to cross<br>
cell membranes, according to methods generally known in the art. Optionally,<br>
the use of a substrate may be combined with the use of an additional detection<br>
reagent that produces a detectable response due to the presence of a specific cell<br>
component, intracellular substance, or cellular condition, in a sample comprising<br>
a mutant hydrolase or a fusion thereof. Where the additional detection reagent<br>
has spectral properties that differ from those of the substrate, multi-color<br>
applications are possible.<br>
At any time after or during contact with the substrate comprising a<br>
functional group with optical properties, the sample comprising a mutant<br>
hydrolase or a fusion thereof is illuminated with a wavelength of light that<br>
results in a detectable optical response, and observed with a means for detecting<br>
the optical response. While some substrates are detectable colorimetrically, using<br>
ambient light, other substrates are detected by the fluorescence properties of the<br>
parent fluorophore. Upon illumination, such as by an ultraviolet or visible<br>
wavelength emission lamp, an arc lamp, a laser, or even sunlight or ordinary<br>
room light, the substrates, including substrates bound to the complementary<br>
specific binding pair member, display intense visible absorption as well as<br>
fluorescence emission. Selected equipment that is useful for illuminating the<br>
substrates of the invention includes, but is not limited to, hand-held ultraviolet<br>
lamps, mercury arc lamps, xenon lamps, argon lasers, laser diodes, and YAG<br>
lasers. These illumination sources are optionally integrated into laser scanners,<br>
fluorescence microplate readers, standard or mini fluorometers, or<br>
chromatographic detectors. This colorimetric absorbance or fluorescence<br>
emission is optionally detected by visual inspection, or by use of any of the<br>
following devices: CCD cameras, video cameras, photographic film, laser<br>
scanning devices, fluorometers, photodiodes, quantum counters, epifluorescence<br>
microscopes, scanning microscopes, flow cytometers, fluorescence microplate<br>
readers, or by means for amplifying the signal such as photomultiplier tubes.<br>
Where the sample comprising a mutant hydrolase or a fusion thereof is examined<br>
using a flow cytometer, a fluorescence microscope or a fluorometer, the<br>
instrument is optionally used to distinguish and discriminate between the<br>
substrate comprising a functional group which is a fluorophore and a second<br>
fluorophore with detectably different optical properties, typically by<br>
distinguishing the fluorescence response of the substrate from that of the second<br>
fluorophore. Where the sample comprising a mutant hydrolase or a fusion<br>
thereof is examined using a flow cytometer, examination of the sample<br>
comprising a mutant hydrolase or a fusion thereof optionally includes isolation<br>
of particles within the sample comprising a mutant hydrolase or a fusion thereof<br>
based on the fluorescence response of the substrate by using a sorting device.<br>
In one embodiment, intracellular movements may be monitored using a<br>
fusion of the mutant hydrolase of the invention. For example, beta-arrestin is a<br>
regulator of G-protein coupled receptors, that moves from the cytoplasm to the<br>
cell membrane when it is activated. A cell containing a fusion of a mutant<br>
hydrolase and beta-arrestin and a substrate of the invention allows the detection<br>
of the movement of beta-arrestin from the cytoplasm to the cell membrane as it<br>
associates with activated G-protein coupled receptors.<br>
In another embodiment, FRET may be employed with a fusion of the<br>
mutant<br>
hydrolase and a fluorescent protein, e.g., GFP, or a fusion with a protein that<br>
binds fluorescent molecules, e.g., O-alkylguanine-DNA alkyltransferase (AGT)<br>
(Keppler et al., 2003). Alternatively, a fusion of a mutant hydrolase and a<br>
protein of interest and a second fusion of a fluorescent protein and a molecule<br>
suspected of interacting with the protein of interest may be employed to study<br>
the interaction of the protein of interest with the molecule, e.g., using FRET.<br>
One cell may contain the fusion of a mutant hydrolase and a protein of interest<br>
while another cell may contain the second fusion of a fluorescent protein and a<br>
molecule suspected of interacting with the protein of interest. A population with<br>
those two cells may be contacted with a substrate and an agent, e.g., a drug, after<br>
which the cells are monitored to detect the effect of agent administration on the<br>
two populations.<br>
In yet another embodiment, the mutant hydrolase is fused to a fluorescent<br>
protein. The fusion protein can thus be detected in cells by detecting the<br>
fluorescent protein or by contacting the cells with a substrate of the invention<br>
and detecting the functional group in the substrate. The detection of the<br>
fluorescent protein may be conducted before the detection of the functional<br>
group. Alternatively, the detection of the functional group may be conducted<br>
before the detection of the fluorescent protein. Moreover, those cells can be<br>
contacted with additional substrates, e.g., those having a different functional<br>
group, and the different functional group in the cell detected, which functional<br>
group is covalentiy linked to mutant hydrolase not previously bound by the first<br>
substrate.<br>
In yet another embodiment, a fusion of a mutant hydrolase and a<br>
transcription factor may be employed to monitor activation of transcription<br>
activation pathways. For example, a fusion of a mutant hydrolase to a<br>
transcription factor present in the cytoplasm in an inactive form but which is<br>
translocated to the nucleus upon activation (e.g., NF kappa Beta) can monitor<br>
transcription activation pathways.<br>
In another embodiment, biotin is employed as a functional group in a<br>
substrate and the fusion includes a mutant hydrolase fused to a protein of interest<br>
suspected of interacting with another molecule, e.g., a protein, in a cell. The use<br>
of such reagents permits the capture of the other molecule which interacts in the<br>
cell with the protein fused to the mutant hydrolase, thereby identifying and/or<br>
capturing (isolating) the interacting molecule(s).<br>
In one embodiment, the mutant hydrolase is fused to a protein that is<br>
secreted. Using that fusion and a substrate of the invention, the secreted protein<br>
may be detected and/or monitored. Similarly, when the mutant hydrolase is<br>
fused to a membrane protein that is transported between different vesicular<br>
compartments, in the presence of the substrate, protein processing within these<br>
compartments can be detected. In yet another embodiment, when the mutant<br>
hydrolase is fused to an ion channel or transport protein, or a protein that is<br>
closely associated with the channel or transport protein, the movement of ions<br>
across cell or organelle membranes can be monitored in the presence of a<br>
substrate of the invention which contains an ion sensitive fluorophore. Likewise,<br>
when the mutant hydrolase is fused to proteins associated with vesicals or<br>
cytoskeleton, in the presense of the substrate, transport of proteins or vesicals<br>
along cytoskeletal structures can be readily detected.<br>
In another embodiment, the functional group is a drug or toxin. By<br>
combining a substrate with such a functional group with a fusion of a mutant<br>
hydrolase and a targeting molecule such as an antibody, e.g., one which binds to<br>
an antigen associated with specific tumor cells, a drug or toxin can be targeted<br>
within a cell or within an animal. Alternatively, the functional group may be a<br>
fluorophore which, when present in a substrate and combined with a fusion of a<br>
mutant hydrolase and a targeting molecule such as a single chain antibody, the<br>
targeting molecule is labeled, e.g., a labeled antibody for in vitro applications<br>
such as an ELISA.<br>
In yet another embodiment, when fused to a protein expressed on the cell<br>
surface, a mutant hydrolase on the cell surface, when combined with a substrate<br>
of the invention, e.g., one which contains a fluorophore, maybe employed to<br>
monitor cell migration (e.g., cancer cell migration) in vivo or in vitro. In one<br>
embodiment, the substrate of the invention is one that has low or no permeability<br>
to the cell membrane. Alternatively, such a system can be used to monitor the<br>
effect of different agents, e.g., drugs, on different pools of cells. In yet another<br>
embodiment, the mutant hydrolase is fused to a HERG channel. Cells<br>
expressing such a fusion, in the presence of a substrate of the invention which<br>
includes a K+-sensitive fluorophore, may be employed to monitor the activity of<br>
the HERG channel, e.g., to monitor drug-toxicity.<br>
hi another embodiment, the substrate of the invention includes a<br>
functional group useful to monitor for hydrophobic regions, e.g., Nile Red, in a<br>
cell or organism.<br>
Thus, the mutant hydrolases and substrates of the invention are useful in<br>
a wide variety of assays, e.g., phage display, panning, ELISA, Western blot,<br>
fluorometric microvolume assay technology (FMAT), and cell and subcellular<br>
staining.<br>
The invention will be further described by the following non-limiting<br>
examples.<br>
Example I<br>
General Methodolpgies<br>
Unless defined otherwise, all technical and scientific terms used herein<br>
have the same meaning as commonly understood by one of ordinary skill in the<br>
field of molecular biology and cellular signaling and modeling. Generally, the<br>
nomenclature used herein and the laboratory procedures in spectroscopy, drug<br>
discovery, cell culture, molecular genetics, plastic manufacture, polymer<br>
chemistry, diagnostics, amino acid and nucleic acid chemistry, and alkane<br>
chemistry described below are those well known and commonly employed in the<br>
art. Standard techniques are typically used for preparation of plastics, signal<br>
detection, recombinant nucleic acid methods, polynucleotide synthesis, and<br>
microbial culture and transformation (e.g., electroporation, lipofection).<br>
The techniques and procedures are generally performed according to<br>
conventional methods in the art and various general references (see generally,<br>
Sambrook et. al. Molecular Cloning: A laboratory manual, 2d ed. (1989) Cold<br>
Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., and Lakowicz, J.R.<br>
Principles of Fluorescence Spectroscopy, New York: Plenum Press (1983) for<br>
fluorescent techniques, which are incorporated herein by reference) and which<br>
are provided throughout this document. Standard techniques are used for<br>
chemical synthesis, chemical analysis, and biological assays.<br>
Materials<br>
All oligonucleotides were synthesized, purified and sequenced by<br>
Promega Corporation (Madison, WI) or the University of Iowa DNA Facility<br>
(Iowa City, Iowa). Restriction enzymes and DNA modifying enzymes were<br>
obtained from Promega Corporation (Madison, WI), New England Biolabs, Inc.<br>
(Beverly, MA) or Stratagene Cloning Systems (La Jolla, CA), and were used<br>
according to the manufacturer's protocols. Competent E. coli JM109 were<br>
provided by Promega Corporation or purchased from Stratagene Cloning<br>
Systems. Small-scale plasmid DNA isolations were done using the Qiagen<br>
Plasmid Mini Kit (Qiagen Inc., Chatsworth, CA). DNA legations were<br>
performed with pre-tested reagent kits purchased from Stratagene Cloning<br>
Systems. DNA fragments were purified with QIAquick Gel Extraction Kits or<br>
QIAquick PCR purification Kits purchased from Qiagen Inc.<br>
The vectors used for generating DhaA mutants and their fusions were as<br>
follows: pET21 (Invitrogen, Carlsbad, CA), pRL-null (Promega, Madison, WI),<br>
pGEX-5x-3 (Amersham Biosciences; Piscataway, NJ), and EGFP and DsRED2<br>
(both from CLONTECH, Palo Alto, CA),.<br>
SDS-polyacrylamide gels and associated buffers and stains, as well as<br>
electroblot transfer buffers, were obtained from BioWhittaker Molecular<br>
Applications (Rockland, ME). Protein molecular weight standards were<br>
purchased from Invitrogen.<br>
Sigma-Aldrich was the source of Anti FlagR monoclonal antibody<br>
antibodies (anti FLAGR M2 monoclonal antibody (mouse) (F3165)), Anti<br>
FLAGR M2 HRP Conjugate and Anti FLAGR M2 FITC conjugate (A8592 and<br>
F4049, respectively). Chemicon (Temecula, CA) was the source of monoclonal<br>
anti-Renilla luciferase antibody (MAB4410). Promega Corp. was the source of<br>
HRP-conjugated goat anti-mouse IgG and HRP-conjugated streptavidin (W4021<br>
and G714, respectively).<br>
1-Cl-butane, 1-Cl-hexane, 1-Cl-octane, 1-Cl-decane, 1-Cl-butanol, 1-C1-<br>
hexanol, 1-Cl-octanol, and 1-Cl-decanol were obtained from Aldrich or from<br>
Fluka (USA). All salts, monobasic potassium phosphate, dibasic potassium<br>
phosphate, imidazole, HEPES, sodium EDTA, ammonium sulfate, and Tris free<br>
base were from Fisher (Biotech Grade).<br>
Glutathione Sepharose 4 FF, glutathione, MonoQ and Sephadex G-25<br>
prepackaged columns were from Amersham Biosciences.<br>
Luria-Broth ("LB") was provided by Promega Corporation.<br>
Methods<br>
PCR reactions. DNA amplification was performed using standard<br>
polymerase chain reaction buffers supplied by Promega Corp. Typically, 50 ul<br>
reactions included Ix concentration of the manufacturer's supplied buffer, 1.5<br>
mM MgCl2,125 yM dATP, 125 uM dCTP, 125 uM dGTP, 125 uM dTTP, 0.10-<br>
1.0 uM forward and reverse primers, 5 U AmpliTaqยฎ DNA Polymerase and 
ng target DNA. Unless otherwise indicated, the thermal profile for amplification<br>
of DNA was 35 cycles of 0.5 minutes at 94ยฐC; 1 minute at 55ยฐC; and 1 minute at<br>
72ยฐC.<br>
DNA sequencing. All clones were confirmed by DNA sequencing using<br>
the dideoxy-terminal cycle-sequencing method (Sanger et al., 1977) and a<br>
Perkin-Elmer Model 310 DNA sequencer. (Foster City, CA).<br>
SDS-PAGE. Proteins were solubilized in a sample buffer (1% SDS, 10%<br>
glycerol, and 1.0 mM p-mercaptoethanol, pH 6.8; Promega Corporation), boiled<br>
for 5 minutes and resolved on SDS-PAGE (4-20% gradient gels; BioWhittaker<br>
Molecular Applications). Gels were stained with Coomassie Blue (Promega<br>
Corp.) for Western blot analysis or were analyzed on a fluoroimager (Hitachi,<br>
Japan) at an Em/Em appropriate for each fluorophore evaluated.<br>
Western blot analysis. Electrophoretic transfer of proteins to a<br>
nitrocellulose membrane (0.2 um, Scheicher &amp; Schuell, Germany) was carried<br>
out in 25 mM Tris base/188 mM glycine (pH 8.3), 20% (v/v) methanol for 2.0<br>
hours with a constant current of 80 mA (at 4ยฐC) in Xcell II Blot module<br>
(Invitrogen). The membranes were rinsed with TBST buffer (10 mM Tris-HCl,<br>
150 mM NaCl, pH 7.6, containing 0.05% Tween 20) and incubated in blocking<br>
solution (3% dry milk or 1% BSA in TBST buffer) for 30 minutes at room<br>
temperature or overnight at 4ยฐC. Then membranes were washed with 50 ml of<br>
TBST buffer and incubated with anti-FLAGR monoclonal antibody M2 (dilution<br>
1:5,000), wA\-Renilla luciferase monoclonal antibody (dilution 1:5,000), or<br>
HRP-conjugated streptavidin (dilution 1:10,000) for 45 minutes at room<br>
temperature. Then the membranes were washed with TBST buffer (50 ml, 5<br>
minutes, 3 times). The membranes that had been probed with antibody were<br>
then incubated with HRP-conjugated donkey anti-mouse IgG (30 minutes, room<br>
temperature) and then the washing procedure was repeated. The proteins were<br>
visualized by the enhanced chemiluminescence (ECL) system (Pharmacia-<br>
Amersham) according to the manufacturer's instructions. Levels of proteins<br>
were quantified using computer-assisted densitometry.<br>
Protein concentration. Protein was measured by the microtiter protocol<br>
of the Pierce BCA Protein assay (Pierce, Rockford, IL) using bovine serum<br>
albumin (BSA) as a standard.<br>
Statistic analysis. Data were expressed as mean +/- S.E.M. values from<br>
experiments performed in quadruplicate, representative of at least 3 independent<br>
experiments with similar results. Statistical significance was assessed by the<br>
student's t test and considered significant when/? 
Bacterial cells. The initial stock of Dh5a cells containing pET-3a with<br>
Rhodococcus rodochorus (DhaA) was kindly provided by Dr. Clifford J.<br>
Unkefer (Los Alamos National Laboratory, Los Alamos, NM) (Schindler et al.,<br>
1999; Newman et al., 1999). Bacteria were cultured in LB using a premixed<br>
reagent provided by Promega Corp. Freezer stocks of E. coli BL21 (XDE3)<br>
pET3a (stored in 10% glycerol, -80ยฐC) were used to inoculate Luria-Bertani agar<br>
plates supplemented with ampicillin (50 fig/ml) (Sambrook et al., 1989). Single<br>
colonies were selected and used to inoculate two 10 ml cultures of Luria-Bertani<br>
medium containing 50 ng/ml ampicillin. The cells were cultured for 8 hours at<br>
37ยฐC with shaking (220 rpm), after which time 2 ml was used to inoculate each<br>
of two 50 ml of Luria-Bertani medium containing 50 |J.g/ml ampicillin, which<br>
were grown overnight at 37ยฐC with shaking. Ten milliliters of this culture was<br>
used to inoculate each of two 0.5 L Luria-Bertani medium with ampicillin.<br>
When the A^o of the culture reached 0.6, isopropyl-1-thio-p-Dgalactopyranoside<br>
(IPTG) was added to a final concentration of 0.5 mM, and<br>
cultures were maintained for an additional 4 hours at 30ยฐC with shaking. The<br>
cells were then harvested by centrifugation and washed with 10 mM Tris-SO4,1<br>
mM EDTA, pH 7.5. The cell pellets were stored at -70ยฐC prior to cell lysis.<br>
Mammalian cells. CHO-K1 cells (ATCC-CCL61) were cultured in a 1:1<br>
mixture of Ham's F12 nutrients and Dulbecco's modified minimal essential<br>
medium supplemented with 10% fetal bovine serum (FBS), 100 U/ml penicillin,<br>
and 100 mg/ml streptomycin, in an atmosphere of 95% air and 5% COj at 37ยฐC.<br>
Rat hippocampal (El 8) primary neurons were isolated as described<br>
below. Briefly, fragments of embryonic (El 8) rat hippocampus in Hibernateโข<br>
E media (GIBCO, Invitrogen, Carlsbad, CA), obtained from Dr. Brewer<br>
(Southern Illinois University), were dissociated and plated on poly-D-lysin<br>
coated (0.28 mg/cm2; Sigma) glass/plastic-ware and cultured in serum-free<br>
Neurobasalโข media with B27 supplement (NB27, GIBCO). All media were<br>
changed every 2-3 days.<br>
Transfecu'on. To study transient expression of different proteins, cells<br>
were plated in 35 mm culture dishes or 24 well plates. At about 80-90%<br>
confluency, the cells were exposed to a mixture of lipofectamine/DNA/antibiotic<br>
free media according to the manufacturer's (GIBCO) instructions. The<br>
following day, media was replaced with fresh media and cells were allowed to<br>
grow for various periods of time.<br>
Fluorescence. Fluorescence in cells in 96 well plates was measured on<br>
fluorescent plate reader CytoFluorll (Beckman) at an Eex/Eem appropriate for<br>
particular fluorophores (e.g., Em/Em for TAMRA is 540/575 nm).<br>
Example II<br>
A DJiaA-Basect Tethering System<br>
A. Wild-Type and Mutant DhaA Proteins and Fusions Thereof<br>
A halo-alkane dehydrogenase from Rhodococcus rhodochrous is a<br>
product of the DhaA gene (MW about 33 kDa). This enzyme cleaves carbonhalogen<br>
bonds in aliphatic and aromatic halogenated compounds, e.g., HaloCs-<br>
HaloCio. The catalytic center of DhaA is a typical "catalytic triad", comprising a<br>
nucleophile, an acid and a histidine residue. It is likely that substrate binds to<br>
DhaA to form an E'S complex, after which nucleophilic attack by Asp 106 forms<br>
an ester intermediate, His272 then activates HzO that hydrolyzes the<br>
intermediate, releasing product from the catalytic center. To determine whether<br>
a point mutation of the catalytic His272 residue impairs enzymatic activity of the<br>
enzyme so as to enable covalent tethering of a functional group (FG) to this<br>
protein, mutant DhaAs were prepared.<br>
Materials and Methods<br>
To prepare mutant DhaA vectors, Promega's in vitro mutagenesis kit<br>
which is based on four primer overlap-extension method was employed (Ho et t<br>
al., 1989) to produce DhaA.H272 to F, A, G, or H mutations. The external<br>
primers were oligonucleotides 5'-<br>
GCTTCACTTGTCGTCATCGTCCTTGTAGTCA-3' (SEQ ID NO:1) and 5'-<br>
GCTTCACTTGTCGTCATCGTCCTTGTAGTCA-3' (SEQ ID NO:2), and the<br>
internal mutagenic primers were as follows: H272F (5;-<br>
CCGGGATTGHCTACCTCCAGGAAGA03'), SEQ ID NO:3), H272A (5'-<br>
CCGGGATTGGยฃยฃTACCTCCAGGAAGAC-3'; SEQ ID NO:4), H272G (5'-<br>
CCGGGATTGCAG.TACCTCCAGGAAGAC-3'; SEQ ID NO:5), and H272Q<br>
(5'-CCGGGATTGGIiCTACCTCCAGGAAGAC-3'; SEQ ID NO:6) (the<br>
mutated codons are underlined). The mutated dehalogenase genes were<br>
subcloned into the pET-3a vector. For overexpression of mutant dehalogenases,<br>
the pET-3a vector was transformed into competent E. coli BL21 (DE3). The<br>
DhaA sequence in clones was confirmed by DNA sequencing.<br>
GST-DhaA (WT or H272F/A/G/H mutants) fusion cassettes were<br>
constructed by cloning the appropriate DhaA coding regions into SaWNotl sites<br>
of pGEXSxS vector. Two primers (5'-<br>
ACGCGTCGACGCCGCCATGTCAGAAATCGGTACAGGC-3' and 5'-<br>
ATAAGAATGCGGCCGCTCAAGCGCTTCAACCGGTGAGTGCGGGGAGC<br>
CAGCGCGC-3'; SEQ ID N0s:7 and 8, respectively) were designed to add a<br>
Sail site and a Kozak consensus sequence to the 5' coding regions of DhaA, to<br>
add aNotl, EcoR47Ul, sndAgel restriction site and stop codons to the 3' coding<br>
region of DhaA, and to amplify a 897 bp fragment from a DhaA (WT or mutant)<br>
template. The resulting fragments were inserted into the SaWNotl site of pGEX-<br>
\.<br>
5X-3, a vector containing a glutathione S-transferase (GST) gene, a sequence<br>
encoding a Factor Xa cleavage site, and multiple cloning sites (MCS) followed<br>
by a stop codon.<br>
A Flag coding sequence was then inserted into the Agel/EcoR47lU<br>
restriction sites of the pGEX5X-3 vector. In frame with the six nucleotide Agel<br>
site is a sequence for an 11 amino acid peptide, the final octapeptide of which<br>
corresponds to the Flag peptide (Kodak Imaging Systems, Rochester, NY). Two<br>
complementary oligonucleotides (5'-<br>
CCGGTGACTACAAGGACGATGACGACAAGTGAAGC-3', sense, SEQ ID<br>
NO:9, and 5'-GCTTCACTTGTCGTCATCGTCCTTGTAGTCA-3', antisense,<br>
SEQ ID NO: 10) coding the Flag peptide (Kodak Imaging Systems, Rochester,<br>
NY) were annealed. The annealed DNA had an Agel site at the 5' end and an<br>
EcoR47Hl at the 3' end. The annealed DNA was digested with^gel and<br>
EcoR47lll and then subcloned into the GST-DhaA.WT or GST-DhaA.H272F<br>
mutant constructs at the Agel and EcoR471E sites. All gene fusion constructs<br>
were confirmed by DNA sequencing.<br>
To generate GST-DhaA fusion proteins, enzyme expression was induced<br>
by the addition of isopropyl-b-D-thiogalactopyranoside (at a final concentration<br>
of 0.5 mM) when the culture reached an optical density of 0.6 at 600 nm. The<br>
cells were harvested in Buffer A (10 mM Tris-SCU, 1 mM EDTA, 1 mM pmercaptoethanol,<br>
and 10 % glycerol, pH 7.5), and disrupted by sonication using<br>
a Vibra Cellโข sonicator (Sonics &amp; Materials, Danbury, CT, USA). Cell debris<br>
was removed by centrifugation at 19,800 x g for 1 hour. The crude extract was<br>
further purified on a GSS-Sepharose 4 fast flow column (Amersham<br>
Biosciences; Piscataway, NJ) according to the manufacturer's instructions. The<br>
elution fractions containing GST-DhaA fusion protein were pooled, dialyzed<br>
against a 10 mM Tris-SCU buffer (containing 20 mM Na2SO4 and 1 mM EDTANa2)<br>
overnight at 4ยฐC, and stored at<br>
-20ยฐC until use. To generate DhaA (WT or mutant), GST was cleaved from the<br>
fusion proteins with Factor Xa, and the products purified on GSS-Sepharose 4<br>
(Amersham Biosciences; Piscataway, NJ) according to the manufacturer's<br>
instructions. Homogeneity of the proteins was verified by SDS-PAGE. In some<br>
experiments, the cell free extract was fractionated using 45-70% saturated<br>
ammonium sulfate as described by Newman et al. (1999).<br>
Results<br>
Figure 3 shows robust, IPTG inducible production of GST-DhaA.WTFlag<br>
(lane 1) and GST-DhaA.H272F-Flag (lane 2) fusion proteins. Moreover,<br>
the proteins were soluble and could be efficiently purified on GSS-Sepharose<br>
4FF (lanes 5-10, odd numbered lanes correspond to GST-DhaA.WT-Flag and<br>
even numbered lanes correspond to GST-DhaA.H272F-Flag). Treatment of the<br>
fusion proteins with Factor Xa led to the formation of two proteins GST and<br>
DhaA (WT or mutant, lanes 11 and 12, respectively), and GST was efficiently<br>
removed on GSS-Sepharose 4FF (WT or mutant, lanes 13 and 14, respectively).<br>
In addition, all proteins had the predicted molecular weight.<br>
B. Mutation of ff272 H?piif? Ability of DhaA to Hvdrolyze Cl-Alkanes.<br>
Inability of an enzyme to release product of the enzymatic reaction into<br>
surrounding media is essential for the tethering system. This inability can be<br>
detected by significant reduction of the hydrolytic activity of the enzyme.<br>
To study the effect of a point mutation on the activity of DhaA (WT or<br>
mutant) hydrolysis of Cl-alkanes, apH-indicator dye system as described by<br>
Holloway et al. (1998) was employed.<br>
Materials and Methods<br>
The reaction buffer for a pH-indicator dye system consisted of 1 mM<br>
HEPES-SO4 (pH 8.2), 20 mM Na2SO4, and 1 mM EDTA. Phenol red was added<br>
to a final concentration 25 ng/ml. The halogenated compounds were added to<br>
apparent concentrations that could insure that the dissolved fraction of the<br>
substrate was sufficient for the maximum velocity of the dehalogenation<br>
reaction. The substrate-buffer solution was vigorously mixed for 30 seconds by<br>
vortexing, capped to prevent significant evaporation of the substrate and used<br>
within 1-2 hours. Prior to each kinetic determination, the phenol red was titrated<br>
with a standardized solution of HC1 to provide an apparent extinction coefficient.<br>
The steady-state kinetic constants for DhaA were determined at 558 nm at room<br>
temperature on a Beckman Du640 spectrophotometer (Beckman Coulter,<br>
Fullerton, CA). Kinetic constants were calculated from initial rates using the<br>
computer program SigmaPlot. One unit of enzyme activity is defined as the<br>
amount required to dehaiogenate 1.0 mM of substrate/minute under the specific<br>
conditions.<br>
Results<br>
As shown in Figure 4, using 0.1 mg/ml of enzyme and 10 mM substrate<br>
at pH 7.0-8.2, no catalytic activity was found with any of four mutants. Under<br>
these conditions, the wild-type enzyme had an activity with 1-Cl-butane of 5<br>
units/mg of protein. Thus, the activity of the mutants was reduced by at least<br>
700-fold.<br>
Aliquots of the supernatant obtained from E. coli expressing DhaA (WT<br>
or one of the mutants) were treated with increasing concentrations of (NH4)2SO4.<br>
The proteins were exposed to each (NH^SC^ concentration for 2 hours (4ยฐC),<br>
pelleted by centrinigation, dialyzed overnight against buffer A, and resolved on<br>
SDS-PAGE.<br>
As shown in Figure 5, a major fraction of DhaA. WT and the<br>
DhaA.H272F mutant was precipitated by 45-70% of (NH4)2SO4. No<br>
precipitation of these proteins was observed at low (NH4)2SO4 concentrations,<br>
to contrast, the DhaA.H272Q, DhaA.H272G and DhaA.H272A mutants could be<br>
precipitated by 10% (NH^SOj. This is a strong indication of the significant<br>
change of the physico-chemical characteristics of the DhaA.H272Q,<br>
DhaA.H272G and DhaA.H272A mutants. At the same time, the DhaA.H272F<br>
mutation had no significant effect on these parameters. These data are in good<br>
agreement with results of computer modeling of the effect of mutations on the 3-<br>
D structure of DhaA, indicating that among all tested mutants, only the<br>
DhaA.H272F mutation had no significant effect on the predicted 3-dimensional<br>
model (see Figure 2). Based on these results, DhaA.H272F was chosen for<br>
further experiments.<br>
To form a covalent adduct, the chlorine atom of Cl-alkane is likely<br>
positioned in close proximity to the catalytic amino acids of DhaA (WT or<br>
mutant) (Figure 2). The crystal structure of DhaA (Newman et al., 1999)<br>
indicates that these amino acids are located deep inside of the catalytic pocket of<br>
DhaA (approximately 10 A long and about 20 A2 in cross section). To permit<br>
entry of the reactive group in a substrate for DhaA which includes a functional<br>
group into the catalytic pocket of DhaA, a linker was designed to connect the Clcontaining<br>
substrate with a functional group so that the functional group is<br>
located outside of the catalytic pocket, i.e., so as not to disturb/destroy the 3-D<br>
structure of DhaA.<br>
To determine if DhaA is capable of hydrolyzing Cl-alkanes with a long<br>
hydrophobic carbon chain, DhaA.WT was contacted with various Cl-alkane<br>
alcohols. As shown in Figure 6, DhaA.WT can hydrolyze 1-Cl-alkane alcohols<br>
with 4-10 carbon atoms. Moreover, Ihe initial rate of hydrolysis (IRH) of Clalkanes<br>
had an inverse relationship to the length of a carbon chain, although poor<br>
solubility of long-chain Cl-alkanes in aqueous buffers may affect the efficiency<br>
of the enzyme-substrate interaction. Indeed, as shown in Figure 6, the IRH of 1-<br>
Cl-alkane-10-decanol is much higher than the IRH of 1-Cl-decane. More<br>
importantly, these data indicate that DhaA can hydrolyze Cl-alkanes containing<br>
relatively polar groups (e.g., HO-group).<br>
FAM-modified Cl-alkanes with linkers of different length and/or<br>
hydrophobicity were prepared (Figure 7). DhaA.WT efficiently hydrolyzed Clalkanes<br>
with a relatively bulky functional group (FAM) if the linker was 12 or<br>
more atoms long. No activity of DhaA.H272F/A/G/Q mutants was detected<br>
with any of the tested Cl-alkanes (data not shown). In addition, modification of<br>
the (CHz)6 region adjacent to the Cl-atom led to a significant reduction of the<br>
IRH of the 14-atom linker by DhaA.WT. Nevertheless, if the length and<br>
structure of the linker is compatible with the catalytic site of a hydrolase, the<br>
presence of a linker in a substrate of the invention has substantially no effect on<br>
the reaction.<br>
Some of the samples were analyzed on an automated HPLC (Hewlett-<br>
Packard Model 1050) system. A DAD detector was set to record UV-visible<br>
spectra over the 200-600 ran range. Fluorescence was detected at an Em/Eem<br>
equal 480/520 ran and 540/575 nm for FAM- and TAMRA-modified substrates,<br>
respectively. Ethanol extracts of Cl-alkanes or products of Cl-alkane hydrolysis<br>
were analyzed using analytical reverse phase Cu column (Adsorbosphere HS,<br>
5\i, 150 x 4.6 mm; Hewlett-Packard, Clifton, NJ) with a linear gradient of 10<br>
mM ammonium acetate (pH 7.0):ACN (acetom'trile) from 25:75 to 1:99 (v/v)<br>
applied over 30 minutes at 1.0 ml/minute. Quantisation of the separated<br>
compounds was based on the integrated surface of the collected peaks.<br>
Figure 8A shows the complete separation of the substrate and the product<br>
of the reaction. Figure 8B indicates that wild-type DhaA very efficiently<br>
hydrolyzed FAM-CnH^CVCl. Similar results were obtained when TAMRACi4H24O4-<br>
Cl or ROX.5-Ci4H24O4-Cl were used as substrates (data not shown).<br>
Taken together these data confirm the results of the pH-indicator dye-based<br>
assay showing complete inactivation of DhaA by the DhaA.H272F mutation.<br>
i<br>
C. Covalent Tethering of Functional Groups to DhaA Mutants In Vitro<br>
Materials and Methods<br>
MALDI analysis of proteins was performed at the University of<br>
Wisconsin Biotechnology Center using a matrix assisted laser<br>
desorption/ionization time-of-life (MALDI-TOF) mass spectrometer Bruker<br>
Biflex III (Bruker, USA.). To prepare samples, 100 ug of purified DhaA (WT or<br>
H272F mutant) or GST-DhaA (WT or H272F mutant) fusion protein (purified to<br>
about 90% homogeneity) in 200 nl of buffer (1 mM HEPES-SO4 (pH 7.4), 20<br>
mM NaaSO4, and 1 mM EDTA) were incubated with or without substrate<br>
(FAM-Ci4H24O4-Cl, at 1.0 mM, final concentration) for 15 minutes at room<br>
temperature. Then the reaction mixtures were dialyzed against 20 mM<br>
CHaCOONHU (pH 7.0) overnight at 4ยฐC and M/Z values of the proteins and<br>
protein-substrate complexes determined.<br>
Oligonucleotides employed to prepare DhaA.D106 mutants include for<br>
DhaA.D106C:<br>
S'-CTTGGGTTTGGAAGAGGTCGTCCTGGTCATCCACIQCTGGGGC-S1<br>
(SEQIDNO:13)and5'-<br>
TGAGCCCCAQยฃAGTGGATGACCAGGACGACCTCTTCCAAACC-3'(SEQ<br>
ID NO: 14);<br>
forDhaA.D106Q:<br>
S'-CTTGGGTTTGGAAGAGGTCGTCCTGGTCATCCACCAGTGGGGC-S1<br>
(SEQ ID NO:34) and 5'-<br>
TGAGCCCCAG1QGTGGATGACCAGGACGACCTCTTCCAAACC-3'(SEQ<br>
ID NO:35);<br>
forDhaA.D106E:<br>
S'-CTTGGQTTTGGAAGAGGTCGTCCTGGTCATCCACGAATGGGGC-S1<br>
(SEQ ID NO:52) and 5'-<br>
TGAGCCCCAHยฃGTGGATGACCAGGACGACCTCTTCCAAACC-3'(SEQ<br>
ID NO:53); and<br>
forDhaA.D106Y:<br>
5'-CTTGGGTTTGGAAGAGGTCGTCCTGGTCATCCAClACTGGGGC-3'<br>
(SEQIDNO^andS1-<br>
TGAGCCCCAGJ^GTGGATGACCAGGACGACCTCTTCCAAACC-3'(SEQ<br>
ID NO:55). The annealed oligonucleotides contained a Styl site at the 5' end and<br>
the Blpl site at the 3' end. The annealed oligonucleotides were digested with Styl<br>
and Blpl and subcloned into GST-DhaA.WT or GST-DhaA.H272F at Styl and<br>
Blpl sites. All mutants were confirmed by DNA sequencing.<br>
Results<br>
To confirm that DhaA.H272 mutants were capable of binding Cl-alkanes<br>
with functional groups, these mutants or their GST-fusions, as well as the<br>
corresponding wild-type proteins or fusions, were contacted with FAMCi4H24O4-<br>
Cl, TAMRA-Ci4H24O4-Cl, ROX.5-Ci4H24O4-Cl, orbiotin-CuH32O4-<br>
Cl for 15 minutes at room temperature. Then the proteins were resolved on<br>
SDS-PAGE. The gels containing proteins were incubated with FAM-Ci4H24O4-<br>
Cl, TAMRA-Ci4H24O4-Cl, or ROX.5-Ci4H24O4-Cl and were analyzed by<br>
fluoroimager (Hitachi, Japan) at an Eex/Eem appropriate for each fluorophore.<br>
Gels containing proteins incubated with biotin-CigH32O4-Cl were transferred to a<br>
nitrocellulose membrane and probed with HRP conjugated streptavidin.<br>
As shown in Figure 9, TAMRA-Ci4H24O4-Cl (lanes 1 and 2 in panel A),<br>
FAM-Ci^-^-Cl (lanes 3 and 4 in panel A), and ROX.5-C14H24O4-C1 (lanes 5<br>
and 6 in panel A) bound to DhaA.H272F (lanes 2,4 and 6 in panel A) but not to<br>
DhaA.WT (lanes 1,3 and 5 in panel A). Biotin-CigH34O4-Cl bound to<br>
DhaA.H272F (lanes 9-14 in panel B) but not to DhaA.WT (lanes 1-8 in panel<br>
B). Moreover, the binding of biotJn-CigH34O4-Cl to DhaA.H272F (lanes 9-14 in<br>
panel B) was dose dependent and could be detected at 0.2 pM. Further, the bond<br>
between substrates and DhaA.H272F was very strong, since boiling with SDS<br>
did not break the bond.<br>
All tested DhaA.H272 mutants, i.e. H272F/G/A/Q, bound to TAMRACi4-<br>
Cl (Figure 10). Further, the DhaA.H272 mutants bind the substrates in a<br>
highly specific manner, since pretreatment of the mutants with one of the<br>
substrates (biotin-Qgl^CVCl) completely blocked the binding of another<br>
substrate (TAMRA-Ci^^-Cl) (Figure 10).<br>
To determine the nature of the bond between Cl-alkanes and the<br>
DhaA.H272F mutant (or the GST-DhaA.H272F mutant fusion protein), these<br>
proteins were incubated with and without FAM-Ci^-A-Cl, and analyzed by<br>
MALDI. As shown in Figure 11, the bond between mutant DhaA.H272F and<br>
FAM-Ci4H24C&gt;4-Cl is strong. Moreover, the analysis of the E*S complex<br>
indicated the covalent nature of the bond between the substrate (e.g., FAMCi4H24O4-<br>
Cl) and DhaA.H272F. The MALDI-TOF analysis also confirms that<br>
the substrate/protein adduct is formed in a 1:1 relationship.<br>
DhaA mutants at another residue in the catalytic triad, residue 106, were<br>
prepared. The residue at position 106 in wild-type DhaA is D, one of the known<br>
nucleophilic amino acid residues. D at residue 106 in DhaA was substituted<br>
with nucleophilic ammo acid residues other than D, e.g., C, Y and E, which may<br>
form a bond with a substrate which is more stable than the bond formed between<br>
wild-type DhaA and the substrate. In particular, cysteine is a known nucleopnile<br>
in cysteine-based enzymes, and those enzymes are not known to activate water.<br>
A control mutant, DhaA.D106Q, single mutants DhaA.D106C,<br>
DhaA.D106Y, and DhaA.D106E, as well as double mutants<br>
DhaA.D106C:H272F, DhaA.D106E:H272F, DhaA.D106Q:H272F, and<br>
DhaA.D106Y:H272F were analyzed for binding to TAMRA-Cnl^CVCl<br>
(Figure 12). As shown in Figure 12, TAMRA-CM^CVCl bound to<br>
DhaA.D106C, DhaA.D106C:H272F, DhaA.D106E, and DhaA.H272F. Thus,<br>
the bond formed between TAMRA-CH^CVCl and cysteine or glutamate at<br>
residue 106 in a mutant DhaA is stable relative to the bond formed between<br>
TAMRA-Ci4H24O4-Cl and wild-type DhaA. Other substitutions at position 106<br>
alone or in combination with substitutions at other residues in DhaA may yield<br>
similar results. Further, certain substitutions at position 106 alone or in<br>
combination with substitutions at other residues in DhaA may result in a mutant<br>
DhaA that forms a bond with only certain substrates.<br>
Example HI<br>
Tethering of Luciferase to a Solid Support via a<br>
Mutan{ phaA and a Substrate of the Invention<br>
Materials and Methods<br>
phRLuc-linker-DhaA.WT-Flag and phRLuc-linker-DhaA.H272F-Flag<br>
fusion cassettes were constructed by cloning the phRLuc coding region into the<br>
NheVSall sites of the pCIneo vector which contains a myristic acid attachment<br>
peptide coding sequence (MAS). Two primers (5'-<br>
GCTTCACTTGTCGTCATCGTCCTTOTAGTCA-3'; SEQ ID NO:11) and (5'-<br>
GCTTCACTTGTCGTCATCGTCCTTGTAGTCA-3'; SEQ ID NO: 12) were<br>
designed to add Nhel and Sail sites to the 5' and 3' coding regions, respectively,<br>
of phRLuc and to amplify a 900 bp fragment from a phRLuc template (pGL3<br>
vector, Promega). Then, a myristic acid attachment peptide coding sequence<br>
was excised with Nhel and Sail restriction enzymes and the amplified fragment<br>
containing phRLuc was inserted into the NheUSall restriction sites of<br>
pCIneo.DhaA.(WT or H272F)-Flag vector. The sequence of each construct was<br>
confirmed by DNA sequencing. Promega's TNTยฎ T7Quick system was then<br>
used to generate fusion proteins in vitro.<br>
Results<br>
To demonstrate tethering of proteins to a solid support via DhaA.H272FCl-<br>
alkane bridge, vectors encoding a fusion protein ofRenilla luciferase (hRLuc,<br>
N-terminus of the fusion), a protein connector (17 amino acids, see Table I), and<br>
DhaA (WT or H272F mutant) were prepared. The Flag epitope was then fused<br>
to the C-terminus of DhaA.<br>
Table I<br>
(Table Removed)<br>
atcgaaggtcgtgggatccccaggaatt<br>
cccgggtcgacgccgcc (SEQ ID<br>
NO:26)<br>
tccggatcaagcttgggcgacgaggtgg<br>
acggcgggccctctagagccacc<br>
(SEQIDNO:28)<br>
accggttccggatcaagcttgcggtacc<br>
gcgggccctctagagcc (SEQ ID<br>
NO:30)<br>
tccggatcaagcttgcggtaccgcgggc<br>
cctctagagccgtcgacgccgcc<br>
(SEQIDNO:32)<br>
Accggt<br>
Peotide<br>
Connector<br>
iegrgiprnsrvdaa<br>
(SEQIDNO:27)<br>
sgsslgdevdggpsrat<br>
(SEQIDNO:29)<br>
tgsgsslryrgpsra<br>
(SEQIDNO:31)<br>
sgsslryrgpsravdaa<br>
(SEQIDNO:33)<br>
Tg<br>
SDS-PAGE followed by Western blot analysis showed that the proteins<br>
had their predicted molecular weights and were recognized by anti-R.Luc and<br>
anti-Flag* M2 antibodies. In addition, all fusion proteins had Renilla luciferase<br>
activity (as determined by Promega's Renilla Luciferase Assay System in PBS<br>
pH 7.4 buffer).<br>
Tethering of proteins to a solid support via a DhaA.H272F-Cl-alkane<br>
bridge was shown by using biotin-CigHaaCVCl as a substrate and streptavidin<br>
(SA)-coated 96 well plates (Pierce, USA) as solid support. Translated proteins<br>
were contacted with biotin-CigHszCVCl substrate at 25 pM (final concentration),<br>
for 60 minutes at room temperature. Unbound biotin-CiglfoCU-Cl was removed<br>
by gel-filtration on Sephadex O-25 prepackaged columns (Amersham<br>
Biosciences). Collected fractions of R.Luc-connector-DhaA fusions were placed<br>
in SA-coated 96-well plate for 1 hour at room temperature, unbound proteins<br>
were washed out and luciferase activity was measured.<br>
Figure 13A shows Renilla luciferase activity captured on the plate.<br>
Analysis of these data indicated that only the fusion containing the mutant DhaA<br>
was captured. The efficiency of capturing was very high (more than 50% of<br>
Renilla luciferase activity added to the plate was captured). In contrast, the<br>
efficiency of capturing of fusions containing wild-type DhaA as well as Renilla<br>
luciferase was negligibly small (
DhaA.H272F with a non-biotinylated substrate (TAMRA-Ci4H24O4-Cl)<br>
decreased the efficiency of capturing by about 80%. Further, there was no effect<br>
of pretreatment with a nonbiotinylated substrate on the capturing of the R.Lucconnector-<br>
DhaA.WT oiRenilla luciferase.<br>
Taken together, these data demonstrate that active enzymes (e.g., Renilla<br>
luciferase) can be tethered to a solid support that forms part of a substrate of the<br>
invention (Cl-alkane-DhaA.H272F-bridge), and retain enzymatic activity.<br>
Example IV<br>
Mutant DhaA and Substrate System In Vivo<br>
A. Covalent Tethering of Functional Groups to DhaA Mutants In Vivo: in<br>
Prokaryotes and Eukarvotes<br>
Materials and Methods<br>
To study the binding of a substrate of the invention to a mutant hydrolase<br>
expressed in prokaryotes, E. coli cells BL21 (^DE3) pLys65 were transformed<br>
with pOEX-5X-3.DhaA.WT-Flag or pQEX-5X-3.DhaA.H272F-Flag, grown in<br>
liquid culture, and induced with IPTG. Either TAMRA-Ci4H24O4-Cl or biotin-<br>
CigthzCVCl was added to the induced cells (final concentration, 25 uM). After<br>
1 hour, cells were harvested, washed with cold PBS (pH 7.3), disrupted by<br>
sonication, and fractionated by centrifugation at 19,800 x g for 1 hour. Soluble<br>
fractions were subjected to SDS-PAGE. Gels with proteins isolated from cells<br>
treated with TAMRA-Ci4H24O4-Cl were analyzed on a fluoroimager, while<br>
proteins from cells treated with biotm-CigHszCVCl were transferred to a<br>
nitrocellulose membrane and probed with HRP-conjugated streptavidin.<br>
To study the binding of TAMRA-Cu^CVCl in mammalian cells,<br>
DhaA.WT-Flag and DhaA.H272F-Flag coding regions were excised from<br>
pGEX-5X-3.DhaA.WT-Flag or pGEX-5X-3.DhaA.H272F-Flag, respectively,<br>
gel purified, and inserted into SaWNotl restriction sites of pCIneo.CMV vector<br>
(Promega). The constructs were confirmed by DNA sequencing.<br>
CHO-K1 cells were plated in 24 well plates (Labsystems) and transfected<br>
with a pCIneo-CMV.DhaA.WT-Flag or pCIneo-CMV.DhaA.H272F-Flag vector.<br>
Twenty-four hours later, media was replaced with fresh media containing 25 uM<br>
TAMRA-C14H24O4-C1 and the cells were placed into a CO2 incubator for 60<br>
minutes. Following this incubation, media was removed, cells were quickly<br>
washed with PBS (pH 7.4; four consecutive washes: 1.0 ml/cm2; 5 seconds each)<br>
and the cells were solubilized in a sample buffer (1% SDS, 10% glycerol, and<br>
the like; 250 ul/well). Proteins (10 nl/lane) were resolved on SDS-PAGE (4-<br>
20% gradient gels) and the binding of the TAMRA-Ci4H24O4-Cl was detected by<br>
a fluoroimager (Hitachi, Japan) at Eex/Eam equal 540/575 nm.<br>
Results<br>
Figures 14A and B show the binding of biotin-CigHtaCVCl (A) and<br>
TAMRA-Ci2H24O4-Cl (B) to E. coli proteins in vivo. The low molecular band<br>
on Figure 14A is an E. coli protein recognizable by HRP-SA, while the<br>
fluorescence detected in the bottom part of Panel B was fluorescence of free<br>
TAMRA-Ci2H24O4-Cl. Figure 15 shows the binding of TAMRA-Ci2H24O4-Cl to<br>
eukaryotic cell proteins in vivo.<br>
Analysis of Figure 14 and Figure 15 showed that the DhaA.H272F-Flag<br>
mutant but not DhaA.WT-Flag binds TAMRA-Cu^CVCl or biotin-QgEfoCV<br>
Cl in vivo. Moreover, the bond between DhaA.H272F-Flag and the substrate<br>
was very strong (probably covalent), since boiling with SDS followed by SDSPAGE<br>
did not disrupt the bond between the mutant enzyme and the substrate.<br>
B. Permeability of Cell Membrane to Substrates of the Invention<br>
Materials and Methods<br>
CHO-K1 Cells (ATCC-CCL61) were cultured in a 1:1 mixture of Ham's<br>
F12 nutrients and Dulbecco's modified minimal essential medium supplemented<br>
with 10% fetal bovine serum (FBS), 100 U/ml penicillin, and 100 mg/ml<br>
streptomycin, in an atmosphere of 95% air and 5% CO2 at 37ยฐC.<br>
To study uptake of different substrates, cells were plated in LT-II<br>
chambers (Nunc) or 96 well plates (Labsystems) at a density of 30,000 cells/cm2.<br>
The following day, media was replaced with media containing different<br>
concentrations of the substrates and cells were placed back in a CO2 incubator<br>
for 2, 5 or 15 minutes. At the end of the incubation, media containing substrate<br>
was removed and cells were quickly washed with PBS (pH 7.4; four consecutive<br>
washes: 1.0 ml/cm2; 5 seconds each). Fresh media was then added to cells, and<br>
the cells were returned to the CO2 incubator at 37ยฐC. The level of fluorescence<br>
in cells in 96 well plates was measured on fluorescent plate reader CytoFluor II<br>
(Beckman) at Em/Em equal 480/520 nm and 540/575 nm for FAM- and<br>
TAMRA-modified substrates, respectively. Fluorescent images of the cells were<br>
taken on inverted epifluorescent microscope Axiovert-100 (Carl Zeiss) with<br>
filter sets appropriate for detection of FITC and TAMRA.<br>
Results<br>
As shown in Figure 16, CHO-K1 cells treated with TAMRA-Ci^KfoCV<br>
Cl (25 uM, 5 minutes at 37ยฐC) could be quickly and efficiently loaded with<br>
TAMRA-Ci^HjgCVCL Image analysis indicated that the fluorescent dye crossed<br>
the cell membrane. Figure 16 also shows that TAMRA-Ci4H28O4-Cl could be<br>
efficiently washed out of the cells. Taken together these data indicate that the<br>
plasma membrane of CHO-K1 cells is permeable to TAMRA-CnHagCU-Cl.<br>
In contrast, FAM-Ci4H24O4-Cl did not cross the plasma membrane of<br>
CHO-K1 cells, even when cells were pretreated with FAM-Ci4H24O4-Cl at high<br>
concentrations (i.e., 100 uM) and for much longer periods of time (60 minutes)<br>
(data not shown). Thus, the different permeabilities of the cell plasma<br>
membrane for various substrates of the invention, e.g., TAMRA-Ci4H24O4-Cl<br>
and FAM-Ci4H24O4-Cl, provides a unique opportunity to label proteins<br>
expressed on the cell surface and proteins expressed inside the cell with different<br>
fluorophores, thereby allowing biplexing.<br>
Example V<br>
DhaA-based Tethering for Cell Imaging In Vivo<br>
A. Colocalization of GFP and TAMRA-C^H^O^-Cl in Living Mammalian Cells<br>
Materials and Methods<br>
A GFP-connector-DhaA fusion cassette was constructed by replacing the<br>
Renilla luciferase coding region in Packard's vector coding GFP-DEVD-Rluc(h)<br>
(Packard #6310066) with DhaA.WT-Flag or DhaA.H272F-Flag coding regions.<br>
Two primers (5'- GGAATGGGCCCTCTAGAQCGACGATGTCA -3'; SEQ ID<br>
NO:15, and 5'- CAOTCAOTCACGATGGATCCGCTC AA -3'; SEQ ID<br>
NO: 16) were designed to add Apal and Bamffl sites (underlined) to the 5' and 3'<br>
coding regions of DhaA, respectively, and to amplify a 980 bp fragment from a<br>
pGEX-5X-3.DhaA.WT-Flag or pGEX-5X-3.DhaA.H272F-Flag template. The<br>
R.Luc coding region was excised with^pal and BamHl restriction enzymes.<br>
Then the 980 bp fragment containing DhaA was inserted into the ApaUBamHl<br>
site of the GFP-DEVD-Rluc(h) coding vector. The sequence of the gene fusion<br>
constructs was confirmed by DNA sequencing.<br>
Cells transiently expressing GFP-connector-DhaA.WT-Flag or GFPconnector-<br>
DhaA.H272F-Flag fusion proteins were plated in LT-II chambers<br>
(Nunc) at a density of 30,000 cells/cm2. The next day, media was replaced with<br>
fresh media containing 25 uM of TAMRA-CHH24O4-C1 and the cells were<br>
placed back into in a CQj incubator for 60 minutes. At the end of the<br>
incubation, media containing substrates was removed, cells were quickly washed<br>
with PBS (pH 7.4; four consecutive washes: 1.0 ml/cm2; 5 seconds each) and<br>
new media was added to the cells. The cells were placed back into in a COj<br>
incubator and after 60 minutes the cells were quickly washed with PBS (pH 7.4;<br>
four consecutive washes: 1.0 ml/cm2; 5 seconds each). Fluorescent images of<br>
the cells were taken on inverted epifluorescent microscope Axiovert-100 (Carl<br>
Zeiss) with filter sets appropriate for detection of GFP and TAMRA.<br>
Results<br>
As shown by the images in Figure 17, cells transfected with either GFPconnector-<br>
DhaA.WT-Flag or GFP-connector-DhaA.H272F-Flag showed robust<br>
expression of the protein(s) with light emitting characteristics of GFP. Analysis<br>
of the images of the same cells taken with a TAMRA-filter set showed that cells<br>
expressing GFP-connector-DhaA.WT-F.ag were dark and could not be<br>
distinguished from cells that do not express this fusion protein. In contrast, cells<br>
expressing GFP-connector-DhaA.H272F-Flag were very bright and<br>
unmistakably recognizable.<br>
Western blot analysis of proteins isolated from CHO-K1 cells transfected<br>
with GFP-connector-DhaA.WT-Flag or GFP-connector-DhaA.H272F-Flag<br>
vectors showed that these cells expressed proteins that were recognized by an<br>
anti-Flag antibody and had the predicted molecular weight for the fusion proteins<br>
(data not shown). A fluoroscan of the SDS-PAGE gel with these proteins<br>
showed strong/covalent binding of TAMRA to GFP-connector-DhaA.H272FFlag<br>
and no binding to GFP-connector-DhaA.WT-Flag (Figure 18).<br>
B. Fusion Partners of DhaA in DhaA.WT-Flag and DhaA.H272F-Flag are<br>
Fractional<br>
To determine whether fusion of two proteins leads to the loss of the<br>
activity of one or both proteins, several JDhaA-based fusion proteins (see Table<br>
II) with DhaA at the C- or N-terminus of the fusion and a connector sequence,<br>
e.g., one having 13 to 17 amino acids, between the two proteins, were prepared.<br>
The data showed that the functional activity of both proteins in the fusion was<br>
preserved.<br>
Table II<br>
(Table Removed)<br>
To study the toxicity of Cl-alkanes, CHO-K1 cells were plated in 96 well<br>
plates to a density of 5,000 cells per well. The next day, media was replaced<br>
with fresh media containing 0-100 uM concentrations of Cl-alkanes and the cells<br>
were placed back into a COz incubator for different periods of time. Viability of<br>
the cells was measured with CellTiter-Gloโข Luminescence Cell Viability Assay<br>
(Promega) according to the manufacturer's protocol. Generally, 100 jj.1 of<br>
CellTiter-Gloโข reagent was added directly to the cells and the luminescence<br>
was recorded at 10 minutes using a DYNEX MLX microtiter plate luminometer.<br>
In some experiments, in order to prevent fluorescence/luminescence<br>
interference, the media containing fluorescent Cl-alkanes was removed and the<br>
cells were quickly washed with PBS (pH 7.4; four consecutive washes: 1.0<br>
ml/cm2; 5 seconds each) before addition of CellTiter-Gloโข reagent. Control<br>
experiments indicated that this procedure had no effect on the sensitivity or<br>
accuracy of the CellTiter-Gloโข assay.<br>
Results<br>
As shown in Figure 19, TAMRA-Ci/iEfoiCU-Cl showed no toxicity on<br>
CHO-K1 cells even after a 4 hour treatment at a 100 uM concentration the (the<br>
highest concentration tested). After a 24 hour treatment, no toxicity was<br>
detected at concentrations of 6.25 uM (the "maximum non-toxic<br>
concentration"). At concentrations &gt; 6.25 jiM, the relative luminescence in<br>
CHO-K1 cells was reduced in a dose-dependent manner with an ICjo of about<br>
100 uM. No toxicity of biotin-CuHMCU-Cl was observed even after 24 hours of<br>
treatment at 100 uM. In contrast, ROXS-Cnl^CVCl had a pronounced toxic<br>
effect as a reduction of the RLU in CHO-K1 cells could be detected after a 1<br>
hour treatment. The ICso value of this effect was about 75 jaM with no apparent<br>
ATP reduction at a 25 pM concentration. The ICso value of ROX5-Ci4H24O4-Cl<br>
toxicity and the "maximum non-toxic concentration" of ROXS-Cut^CU-Cl<br>
decreased in a time-dependent manner reaching 12.5 |iM and 6.25 uM,<br>
respectively.<br>
D. Detection of DhaA.D106C in CHO cells contacted with TAMRA- or DiAc-<br>
FAM-containing substrates and a fixative<br>
CHO cells (ATCC, passage 4) were seeded into 8-well chamber slides<br>
(German coverglass system) at low density in DMEM:F12 media (Gibco)<br>
containing 10% FBS and 1 mM glutamine (growth media) without antibiotics.<br>
Two days later, cells were inspected using an inverted phase microscope. Two<br>
visual criteria were confirmed before applying the transfection reagents: 1) the<br>
level of cellular confluence per chamber was approximately 60-80%, and 2)<br>
&gt;90% of the cells were adherent and showed a flattened morphology. The media<br>
was replaced with 150 ul of fresh pre-warmed growth media and cells were<br>
incubated for approximately 1 hour.<br>
Cells were transfected using the Transit TKO system (Miris). The TKO<br>
lipid was diluted by adding 7 ul of lipid per 100 ul of serum-free DMEM:F12<br>
media, and then 1.2 ug of transfection-grade DhaA.D106C DNA was added per<br>
100 ul of lipid containing media. The mixture was incubated at room<br>
temperature for 15 minutes, and then 25 ul aliquots were transferred into<br>
individual culture chambers (0.3 ug DNA). Cells were returned to the incubator<br>
for 5-6 hours, washed two times with growth media, 300 ul of fresh growth<br>
media was added, and then cells were incubated for an additional 24 hours.<br>
Transfected or non-transfected control cells were incubated with 12.5 uM<br>
TAMRA-Ci4H24O4-Cl or 12.5 uM DiAc-FAM-Ci4H24O4-Cl in 10%<br>
FBS/DMEM for 30 minutes at 37ยฐC and 5%CO2. Cells were washed with warm<br>
growth media three times, 300 ul fresh growth media was added, and then cells<br>
were incubated for 1 hour.<br>
Growth media was replaced with warm PBS and live cells were<br>
visualized using a Zeiss Axiovert 100 inverted microscope equipped with a<br>
rhodamine filter set (Exciter filter= 540, Emission filter= 560LP) and a<br>
fluorescein filter set (Exciter filter= 490, Emission filter= 520), and a Spot CCD<br>
camera. Images were captured with exposure times of 0.15-0.60 seconds at gain<br>
settings of 4 or 16.<br>
Discreet and specifically labeled transfected cells were evident in both<br>
TAMRA-Ci4H24O4-Cl and DiAoFAM-Ci4H24O4-Cl labeled cells. The majority<br>
of cells were non-transfected cells and they did not retain the label.<br>
The PBS was removed and cells were fixed with 3.7%<br>
paraformaldehyde/0.1% Triton in PBS for 15 minutes. The fixative was<br>
removed, PBS was added, and a second set of images was captured for both<br>
TAMRA-Ci4H24O4-Cl and DiAc-FAM-Ci4H24O4-Cl labeled cells.<br>
The PBS was replaced with 50% methanol in PBS and cells were<br>
incubated for 15 minutes, followed by a 15 minute incubation in 95% methanol.<br>
A third set of images was captured and then an equal volume mixture of<br>
methanol and acetone was applied to the cells and incubated for 15 minutes. The<br>
media was replaced with PBS and a fourth set of images was collected.<br>
Results suggested that the binding of the substrates to the DhaA.D106C<br>
mutant was stable following fixation with paraformaldehyde and subsequent<br>
processing of fixed cell samples in methanol and acetone. Furthermore, the<br>
brightness of the TAMRA or FAM fluorescence was unchanged under these<br>
conditions.<br>
Example VI<br>
Mutant Beta-Lactamase fblaZVbased Tethering<br>
The serine-p-lactamases, enzymes that confer bacterial resistance to Plactam<br>
antibiotic, likely use the hydroxyl group of a serine residue (Ser70 in<br>
the class A consensus numbering scheme of Ambler et al. (1991)) to degrade a<br>
wide range of p-lactam compounds. The reaction begins with the formation of<br>
a precovalent encounter complex (Figure 20A), and moves through a highenergy<br>
acylation tetrahedral intermediate (Figure 20B) to form a transiently<br>
stable acyl-enzyme intermediate, forming an ester through the catalytic residue<br>
Ser70 (Figure 20C). Subsequently, the acyl-enzyme is attacked by hydrolytic<br>
water (Figure 20D) to form a high-energy deacylation intermediate (Figure<br>
20E) (Minasov et al., 2002), which collapses to form the hydrolyzed product<br>
(Figure 20F). The product is then expelled, regenerating free enzyme. As in<br>
serine proteases, this mechanism requires a catalytic base to activate the serine<br>
nucleophile to attack the amide bond of the substrate and, following formation<br>
of the acyl-enzyme intermediate, to activate the hydrolytic water for attack on<br>
the ester center of the adduct.<br>
A. Mutant p-Lactamase and Fusions Thereof<br>
Materials and Methods<br>
The plasmid pTS32 harboring Staphylococcus aureus PCI blaZ gene<br>
(Zawadzke et al., 1995) was kindly provided by Dr. O. Herzberg (University of<br>
Maryland Biotechnology Institute). The blaZ gene has the following sequence:<br>
AGCTTACTAT GCCATTATTA ATAACTTAGC CATTTCAACA<br>
CCTTCTTTCA AATATTTATAATAAACTATT GACACCGATA<br>
TTACAATTGT AATATTATTG ATTTATAAAA<br>
ATTACAACTGTAATATCGGA GGGnTATTT TGAAAAAGTT<br>
AATATTTTTA ATTGTAATTG CTTTAGTTTTAAGTGCATGT<br>
AATTCAAACA GTTCACATGC CAAAGAGTTA AATGATTTAG<br>
AAAAAAAATATAATGCTCAT ATTGGTGTTT ATGCTTTAGA<br>
TACTAAAAGT GGTAAGGAAG TAAAATTTAATTCAGATAAG<br>
AGATTTGCCT ATGCTTCAAC TTCAAAAGCG ATAAATAGTG<br>
CTATnTGTTAGAACAAGTA CCTTATAATA AGTTAAATAA<br>
AAAAGTACAT ATTAACAAAG ATGATATAGTTGCTTATTCT<br>
CCTATTTTAG AAAAATATGT AGGAAAAGAT ATCACTTTAA<br>
AAGCACTTATTGAGGCTTCA ATGACATATA GTGATAATAC<br>
AGCAAACAAT AAAATTATAA AAGAAATCGGTGGAATCAAA<br>
AAAGTTAAAC AACGTCTAAA AGAACTAGGA GATAAAGTAA<br>
CAAATCCAGTTAGATATGAG ATAGAATTAA ATTACTATTC<br>
ACCAAAGAGC AAAAAAGATA CTTCAACACCTGCTGCCTTC<br>
GGTAAGACCC TTAATAAACT TATCGCCAAT GGAAAATTAA<br>
GCAAAGAAAACAAAAAATTC TTACTTGATT TAATGTTAAA<br>
TAATAAAAGC GGAGATACTT TAATTAAAGACGGTGTTCCA<br>
AAAGACTATA AGGTTGCTGA TAAAAGTGGT CAAGCAATAA<br>
CATATGCTTCTAGAAATGAT GTTGCTTTTG TTTATCCTAA<br>
GGGCCAATCT GAACCTATTG TTTTAGTCATTTTTACGAAT<br>
AAAGACAATA AAAGTGATAA GCCAAATGAT AAGTTGATAA<br>
GTGAAACCGCCAAOAGTGTA ATGAAGGAAT TTTAATATTC<br>
TAAATGCATA ATAAATACTG ATAACATCTTATATTTTGTA<br>
TTATATTTTG TATTATCGTT GAC (SEQ ID NO:36).<br>
GST-WoZ (WT and E166D, N170Q, or E166D:N170Q mutants) fusion<br>
cassettes were constructed by introducing point mutations into the blaZ gene and<br>
cloning the blaZ coding regions into SaWAgel sites of pGEXSx3 vector. The<br>
internal mutagenic primers were as follows: E166D (5'-<br>
CCAGTTAGATATGACATAGAATTAAATTACTATTCACC-3', SEQ ID<br>
NO:56;5'-GGTGAATAGTAATTTAATTCTATGTCATATCTAACTGG-3',<br>
SEQ ID NO:57); N170Q (5'-<br>
CCAGTTAGATATGAGATAGAATTACAGTACTATTCACC-3', SEQ ID<br>
NO:58; and 5'-GGTGAATAGTACTGTAATTCTATCTCATATCTAACTGG-<br>
3', SEQ ID NO:59); and E166D:N170Q<br>
(5'CCAGTTAGATATGACATAGAATTACAGTACTATTCACC-3'; SEQ ID<br>
NO:60 and 5'-GGTGAATAGTACTGTAATTCTATGTCATATCTAACTGG-3;<br>
SEQ ID NO:61). Two external primers (5'-<br>
CAACAGGTCGACGCCGCCATGAAAGAGTTAAATGATTTAG-3', SEQ ID<br>
NO:62; and 5'-GTAGTCACCGGTAAATTCCTTCATTACACTCTTGGC-3',<br>
SEQ ID NO:63) were designed to add N-termind Sail site and a Kozak sequence<br>
to the 5' coding region, add an Agel site to the 3' coding regions ofblaZ, and to<br>
amplify a 806 bp fragment from a WaZ.WT template. The resulting fragment<br>
was inserted into the Sali/Agel site of the vector pGEX-5X-3 containing a<br>
glutathione S-transferase (GST) gene, a sequence coding a Factor Xa cleavage<br>
site, and multiple cloning sites (MCS) followed by a sequence coding for Flag<br>
and stop codons. These gene fusion constructs were confirmed by DNA<br>
sequencing.<br>
The GST-WaZ (WT or mutants) fusion proteins were overexpressed in<br>
competent E. coll BL21 (X DBS) cells and purified essentially as described for<br>
DhaA and GST-DhaA fusion proteins (except the potassium phosphate buffer<br>
(0.1 M, pH 6.8) was used instead of Buffer A). Homogeneity of the proteins<br>
was verified by SDS-PAGE.<br>
The chromogenic substrate 6-p-[(Furylacryloyl)amido]penicillanic acid<br>
triethylamine salt (FAP) was purchased from Calbiochem (La Jolla, CA).<br>
Hydrolysis of FAP was monitored by loss of adsorbance at 344 nm (deltaE =<br>
1330 M"1 cm"1) on a Beckman Du640 spectrophotometer (Beckman Coulter,<br>
Fullerton, CA). All assays were performed at 25ยฐC in 0.1 M potassium<br>
phosphate buffer at pH 6.8.<br>
In CCF2, the cephalosporin core links a 7-hydroxycoumarin to a<br>
fluorescein. In the intact molecule, excitation of the coumarin (Eex- 409 nm)<br>
results in FRET to the fluorescein, which emits green light (Eem- 520 nm).<br>
Cleavage of CCF2 by p-lactamase results in spatial separation of the two dyes,<br>
disrupting FRET such that excitation of coumarin now gives rise to blue<br>
fluorescence (Bex- 447 nm). CCF2 was purchased from Aurora Biosciences<br>
Corporation (San Diego, CA). Reduction of the FRET signal and an increase in<br>
blue fluorescence were measured on Fluorescence Multi-well Plate Reader<br>
CytoFluorll (PerSeptive Biosystems, Framingham, MA, USA).<br>
Results<br>
All p-lactamases, including p-lactamase from Staphylococcus aureus<br>
PCI, hydrolyze p-lactams of different chemical structure. The efficiency of<br>
hydrolysis depends on the type of the enzyme and chemical structure of the<br>
substrate. Penicillin is considered to be a preferred substrate for p-lactamase<br>
from Staphylococcus aureus PCI.<br>
The effect of point mutation(s) on the ability of P-lactamase to hydrolyze<br>
penicillins was studied as described in Zawadzke et al. (1995). As shown in<br>
Figure 20, a GST-p-lactamase PCI fusion protein efficiently hydrolyzed FAP.<br>
Hydrolysis of FAP by blaZ.E\66D, WaZ.N170Q or WaZ.E166D:N170Q blaZ<br>
mutants could not be detected even after 60 minutes of co-incubation.<br>
Therefore, these mutations lead to significant inactivation ofblaZ.<br>
To show that W0Z.E166D, WaZ.N170Q, or WaZ.E166D:N170Q mutants<br>
bind p-lactams, and therefore different functional groups could be tethered to<br>
these proteins via p-lactams, GST fusions of these mutants were incubated with<br>
BOCELLINโข FL, a fluorescent penicillin (Molecular Probes Inc., Eugene, OR).<br>
Proteins were resolved on SDS-PAGE and analyzed on fluoroimager (Hitachi,<br>
Japan) at an Em/Eon appropriate for the particular fluorophore. The data in<br>
Figure 22 show that all blaZ mutants bind bocellin. Moreover, the bond<br>
between blaZ mutants and fluorescent substrates was very strong, and probably<br>
covalent, since boiling with SDS followed by SDS-PAGE did not disrupt the<br>
bond. Also, the binding efficiency of double mutant WaZ.E166D:N170Q<br>
(judged by the strength of the fluorescent signal of protein-bound fluorophore)<br>
was much higher than binding efficiency of either of the single mutants, and the<br>
binding efficiency of WaZ.N170Q was higher than binding efficiency of<br>
blaZ.E\66D. These data, in combination with current understanding of the role<br>
of the individual amino acids in hydrolysis of beta-lactams, show that additional<br>
mutations (e.g., a mutation of an auxiliary amino acid) can improve efficiency of<br>
tethering of functional groups to a mutated protein.<br>
The effect of point mutation(s) on the ability of P-lactamase to hydrolyze<br>
cephalosporins was also studied using CCF2, a FRET-based substrate described<br>
by Zlokarnik et al. (1998). As shown in Figure 23, the GST-p-lactamase PCI<br>
fusion protein efficiently hydrolyzed CCF2 (lane 2). Single point mutations<br>
(i.e., E166D or N170Q) reduced the ability of the fusion proteins to hydrolyze<br>
CCF2 (lanes 3 and 4). The replacement of two amino acids<br>
(WaZ.El66D:N170Q mutants, lane 5) had an even more pronounced effect on<br>
the CCF2 hydrolysis. However, all blaZ mutants were capable of hydrolyzing<br>
CCF2.<br>
Thus, an amino acid substitution at position 166 or 170, e.g., Glul66Asp<br>
or AsnlTOGly enables the mutant beta-lactamase to trap a substrate and therefore<br>
tether the functional group of the substrate to the mutant beta-lactamase via a<br>
stable, e.g., covalent, bond. Moreover, mutation of an amino acid that has an<br>
auxiliary effect on EbO activation increased the efficiency of tethering.<br>
Example VII<br>
Targeting of DhaA.H272F to the Nucleus and Cvtosol of Living Cells<br>
Materials and Methods<br>
A GFP-connector-DhaA.H272F-NLS3 fusion cassette was constructed<br>
by inserting a sequence encoding NLS3 (three tandem repeats of the Nuclear<br>
Localization Sequence (NLS) from simian virus large T-antigen) into the<br>
Agel/BamtU sites of a pCIneo.GFP-connector-DhaA.H272F-Flag vector. Two<br>
complementary oligonucleotides (5'-<br>
CCGGTGATCCAAAAAAGAAGAGAAAGGTAGATCCAAAAAAGAAGAG<br>
AAAGGTAGATCCAAAAAAGAAGAGAAAGGTATGAG -3', sense, SEQ ID<br>
NO:37, and 5'-<br>
GATCCTCATACCTTrCTCTTCTITITrGG^<br>
TGGATCTACCirrCTCTTCTTTTTTGGATCA -3', antisense, SEQ ID<br>
NO: 3 8) coding for the NLS3 peptide, were annealed. The annealed DNA had an<br>
AgeJ site at 5' end and a BamHl site at the 3' end. The annealed DNA was<br>
subcloned into the GFP-connector-DhaA.H272F-Flag construct at the<br>
Agel/Bamtil sites. The sequence of the gene fusion construct was confirmed by<br>
DNA sequencing.<br>
A DhaA.H272F-p-arrestin2 fusion cassette was constructed by replacing<br>
the pGFP2 coding region in Packard's vector encoding GFP2-p-arrestin2<br>
(Packard #6310176-1F1) with the DhaA.H272F-Flag coding region. Two<br>
primers (5'-ATTATGCTGAGTGATATCCC-3f; SEQ ID NO:39, and 5'-<br>
CTCGGTACCAAGCTCCTTGTAGTCA-31; SEQ ID NO:40) were designed to<br>
add a Kpnl site to the 3' coding region ofDhaA, and to amplify a 930 bp<br>
fragment from a pGEX5X-3.DhaA.H272F-Flag template. The pGFP2 coding<br>
region was excised with Nhel and Kpnl restriction enzymes, then the 930 bp<br>
fragment containing encoding DhaA.H272F was inserted into the Nhel and Kpnl<br>
sites of the GFP2-{i-arrestin2 coding vector. The sequence of the fusion<br>
construct was confirmed by DNA sequencing.<br>
CHO-K1 or 3T3 cells transiently expressing GFP-connector-<br>
DhaA.H272F-NLS3, GFP2-|J-arrestin2 or DhaA.H272F-p-arrestin2 fusion<br>
proteins were plated in LT-II chambers (Nunc) at a density of 30,000 cells/cm2.<br>
The next day, media was replaced with fresh media containing 25 |iM of<br>
TAMRA-CnHbCVCl and the cells were placed back into &amp;CO2 incubator for<br>
60 minutes. At the end of the incubation, substrate media was removed, cells<br>
were quickly washed with PBS (pH 7.4; four consecutive washes: 1 .0 ml/cm2; 5<br>
seconds each), and new media was added to the cells. The cells were placed<br>
back into a CQj incubator and after 60 minutes the cells were quickly washed<br>
with PBS (pH 7.4; 1.0 ml/cm2). Fluorescent images of the cells were taken on<br>
confocal microscope Pascal- 5 (Carl Zeiss) with filter sets appropriate for the<br>
detection of GFP and TAMRA.<br>
Results<br>
As shown by the images in Figure 24, GFP and TAMRA were colocalized<br>
in the cell nucleus of cells expression GFP-connector-DhaA.H272FNLS3<br>
and contacted with TAMRA-Ci4H24O4-Cl.<br>
As shown by the images in Figure 25, GFP-p-arrestin2 expressing cells<br>
have a typical p-arrestin2 cytosolic localization. A fluoroscan of the SDS-PAGE<br>
gel of DhaA.H272F-p-arrestin2 showed strong binding of a TAMRA containing<br>
DhaA substrate to cells expressing DhaA.H272F-p-arrestin2.<br>
Site-Directe4 Mutagenesis of DhaA Catalytic Residue 130<br>
Haloalkane dehalogenases use a three-step mechanism for cleavage of<br>
the carbon-halogen bond. This reaction is catalyzed by a triad of amino acid<br>
residues composed of a nucleopbile, base and acid which, for the haloalkane<br>
dehalogenase fiomXanthobacter autotrophicus (DhlA), are residues Asp 124,<br>
His289 and Asp260, respectively (Franken et al., 1991), and in Rhodococcus<br>
dehalogenase enzyme (DhaA), Aspl06, His272 and GlulSO (Newman et al.,<br>
1999).<br>
Unlike the haloalkane dehalogenase nucleophile and base residues, the<br>
role of the third member of the catalytic triad is not yet fully understood. The<br>
catalytic acid is hydrogen bonded to the catalytic His residue and may assist the<br>
His residue in its function by increasing the basicity of nitrogen in the imidazole<br>
ring. Krooshof et al. (1997), using site-directed mutagenesis to study the role of<br>
the DhlA catalytic acid Asp260, demonstrated that a D260N mutant was<br>
catalytically inactive. Furthermore, this residue apparently had an important<br>
structural role since the mutant protein accumulated mainly in inclusion bodies.<br>
The haloalkane dehalogenase from Sphinogomonas paucimobilis (LinB) is the<br>
enzyme involved in y-hexachlorocyclohexane degradation (Nagata et al., 1997).<br>
Hynkova et al., (1999) replaced the putative catalytic residue (Glu-132) of the<br>
LinB with glutamine (Q) residue. However, no activity was observed for the<br>
E132Q mutant even at very high substrate conpentrations.<br>
To examine the role of the DhaA catalytic triad acid GlulSO in protein<br>
production and on the ability of the mutant protein to form covalent alkylenzyme<br>
intermediates with a fluorescent-labeled haloalkane substrate, sitedirected<br>
mutagenesis was employed to replace the DhaA glutamate (E) residue at<br>
position 130 with glutamine, leucine and alanine.<br>
Materials and Methods<br>
Strains and plasmids. Ultracompetent E. coli XL10 Gold (Stratagene;<br>
Tet1 &amp;(mcrA)183 h(mcrCB-hsdSMR-mrr)173 endAJ supE44 thi-1 recAl gyrA96<br>
relAl lac Hte [F proAB lacI*Z&amp;M15 Tn/0 (Tot1) Amy Cam1]) was used to as a<br>
host in transformation of site-directed mutagenesis reactions. E. coli strain<br>
JM109 (e!4-(McrA-) recAl endAJ gyrA96 thi-1 hsdR17(rK.- mK+) supE44<br>
relAl k(lac-proAB) [F' traD36proAB lacPZAMlSJ) was used as the host for<br>
gene expression and whole cell enzyme labeling studies. A GST-DhaA-FLAG<br>
gene fusion cloned into plasmid pGEX5X3, designated<br>
pGEX5X3DhaAWT.FLAG, was used as the starting template for El 30<br>
mutagenesis. A mutant plasmid containing a H272F mutation in DhaA,<br>
designated pGEX5X3DhaAH272F-FLAG, was used as a positive control in<br>
labeling studies and the cloning vector pGEX5X3 was used as a negative<br>
control.<br>
Site-directed mutagenesis of the DhaA El 30 residue. The sequence of<br>
the oligonucleotides used for mutagenesis is shown below. The underlined<br>
nucleotides indicate the position of the altered codons. The oligonucleotides<br>
were synthesized by Integrated DNA Technologies (Coralville, IA) at the 100<br>
runole scale and modified by phosphorylation at the 5' end.<br>
DhaAElSOQ 5'<br>
CAAAGGTATTGCATGTATGยฃAQTTCATCCGGCCTATCCCG 3' (SEQ ID<br>
NO:41)<br>
DhaAE130L 5'<br>
GTCAAAGGTATTGCATGTATGCieTTCATCCGGCCTATCCCGACS1<br>
(SEQIDNO:42)<br>
DhaA E130A 5' AGGTATTGCATGTATGGยฃSTTCATCCGGCCTATCCC 3'<br>
(SEQ ID NO:43)<br>
Site-directed mutagenesis was performed using the QuikChange Multi kit<br>
according to the manufacturer's instructions (Stratagene, La Jolla, CA). The<br>
mutagenesis reactions were introduced into competent E. coli XL10 Gold cells<br>
and transtbrmants were selected on LB agar plates containing ampicillin (100<br>
Hg/mL). Plasmid DNA isolated from individual transformants was initially<br>
screened for the loss of an EcoRl site due to replacement of the glutamate codon<br>
(GAAttc). Clones suspected of containing the desired codon change from each<br>
reaction were selected and subjected to DNA sequence analysis (SeqWright,<br>
Houston, TX). The primer used to confirm the sequence of the mutants in the<br>
pGEX5X3 vector was as follows: 5' GGGCTGGCAAGCCACGTTTGGTG 3'<br>
(SEQIDNO:44).<br>
DhaA mutant analysis. The three DhaA El30 substitution mutants were<br>
compared to the following constructs: Wild-type DhaA, DhaA.H272F, and a<br>
DhaA negative control (pGEX5X3 vector only). Overnight cultures of each<br>
clone were grown in 2 mL of LB containing ampicillin (100 jig/mL) by shaking<br>
at 30ยฐC. The overnight cultures were diluted 1:50 into a sterile flask containing<br>
50 mL fresh LB medium and ampicillin (100 ug/mL). The cultures were<br>
incubated with shaking at 25ยฐC to minimize the production of insoluble protein<br>
species. When the cultures reached mid-log phase (ODeoo^-fi), IPTG (0.1 mM)<br>
was added and the cultures were incubated with shaking at 25ยฐC for an<br>
additional 22 hours. For labeling of whole cells with a tetramethylrhodamine<br>
(TAMRA) haloalkane conjugated substrate, the cell density of each culture was<br>
adjusted to ODeoo^l prior to adding substrate to a concentration of 15 nM. The<br>
cells were incubated with gentle agitation at 4ยฐC for approximately 18 hours.<br>
Following incubation, 20 ul of cells from each labeling reaction was added to 6<br>
ul of 4X SDS loading dye and the samples were boiled for about 3 minutes prior<br>
to being loaded onto a 4-20% acrylamide gel (Tris glycine). For in vitro labeling<br>
studies, crude lysates of IPTG induced cultures were prepared by collecting 3<br>
mL of cells (ODeoo-l) and resuspending the resulting pellet in 75 uL PBS.<br>
Following a freeze/thaw step, 225 uL of IX Cell Culture Lysis Reagent<br>
(Promega Corp., Madison, WI) containing 1.25 mg/mL lysozyme was added to<br>
facilitate lysis of the cells. A 20 uL sample of each lysate was combined with 25<br>
uL of IX PBS. The TAMRA labeled haloalkane substrate was added to a final<br>
concentration of 25 uM. The labeling reactions were incubated at room<br>
temperature for 2 hours. A 25 ul sample of each labeling reaction was added to<br>
6 fil of 4X SDS loading dye and the samples were boiled for about 3 minutes<br>
prior to being loaded onto a 4-20% acrylamide gel (Tris glycine). The gels were<br>
imaged using a Fluorlmager SI instrument (Amersham Biosciences, Piscataway,<br>
NJ) set to detect emission at 570 nm.<br>
Cell-free lysates were generated by centrifugation of crude lysates for 15<br>
minutes at 14,000 RPM. Protein production was monitored by SDS-PAGE and<br>
Western blot analysis. Proteins transferred to a PVDF membrane were<br>
incubated with an anti-FLAGR antibody conjugated with alkaline phosphatase<br>
(AP) (Sigma, St. Louis, MO). The blot was developed with the Western Blue<br>
stabilized substrate for alkaline phosphatase (Promega Corp., Madison, WI).<br>
Results<br>
The role of the DhaA catalytic acid hi the hydrolysis of the alkyl-enzyme<br>
intermediate was probed by site-directed mutagenesis. The DhaA codon E130<br>
was replaced with a codon for glutamine (Q), leucine (L) or alanine (A), as these<br>
substitutions would likely be least disruptive to the structure of the enzyme.<br>
Following mutagenesis, restriction endonuclease screening and DNA sequence<br>
analysis was used to verify the desired codon changes. Sequence verified<br>
DhaA.EBOQ, DhaA.E130L and DhaA.E130A clones, designated Cl, A5 and<br>
A12, respectively, were chosen for further analysis. The E130 mutants were<br>
analyzed for protein expression and for their ability to form a covalent alkylenzyme<br>
intermediate with a TAMRA labeled haloalkane substrate. The three<br>
E130 gene variants were over-expressed in E. coli JM109 cells following<br>
induction with IPTG. SDS-PAGE analysis of crude cell lysates showed that<br>
cultures expressing the wild-type and mutant dhaA genes accumulated protein to<br>
approximately the same level (Figure 26; lanes 2,4,6, 8,10, and 12).<br>
Furthermore, the DhaA protein that was produced by the wild-type and H272F<br>
constructs was for the most part soluble since the amount of protein did not<br>
change appreciably after centrifugation (Figure 26; lanes 3 and 5). The abundant<br>
22 kDa protein bands present in the vector only lanes (Figure 26; lanes 6 and 7)<br>
represented the GST protein. These results, however, are in stark contrast to the<br>
DhaA.E130Q, DhaA.E130L and DhaA.E130A mutants that appeared to<br>
accumulate predominantly insoluble DhaA protein. This conclusion is based on<br>
the observation that after centrifugation, there was a significant loss in the<br>
amount of DhaA protein present in cell-free lysates (Figure 26; lanes 9,11, and<br>
13). Nevertheless, a protein band that comigrates with DhaA was clearly<br>
observed in each DhaA.ElSO mutant lanes after centrifugation (+s) suggesting<br>
the presence of soluble enzyme. Western analysis was, therefore, used to<br>
determine if the protein bands observed in the DhaA.ElSO mutants following<br>
centrifugation represented soluble DhaA material. The immunoblot shown in<br>
Figure 27 confirmed the presence of soluble DhaA protein in each of the<br>
DhaA.E130 mutant cell-free lysates (lanes 9,11, and 13).<br>
The DhaA.ElSO mutants were also examined for their ability to generate<br>
an alkyl-enzyme covalent intermediate. Crude lysates prepared from IPTG<br>
induced cultures of the various constructs were incubated in the presence of the<br>
TAMRA labeled substrate. Figure 28 showed that the DhaA.H272F mutant<br>
(lane 3) was very efficient at producing this intermediate. No such product<br>
could be detected with either the WT DhaA or negative control lysates. Upon<br>
initial examination, the DhaA.ElSO mutants did not appear to produce detectable<br>
levels of the covalent product. However, upon closer inspection of the<br>
fluoroimage extremely faint bands were observed that could potentially represent<br>
minute amounts of the covalent intermediate (Figure 28; lanes 5-7). Based on<br>
these results, the ability of whole cells to generate a covalent, fluorescent alkylenzyme<br>
intermediate was investigated.<br>
Figure 29 shows the results of an in vivo labeling experiment comparing<br>
each of the DhaA.ElSO mutants with positive (DhaA.H272F mutant) and<br>
negative (DhaA-) controls. As expected, the DhaA.H272F mutant was capable<br>
of generating a covalent alkyl-enzyme intermediate as evidenced by the single<br>
fluorescent band near the molecular weight predicted for the GST-DhaA-Flag<br>
fusion (Figure 29, lane 3). As previously observed with the in vitro labeling<br>
results, no such product could be detected with either the wild-type or negative<br>
control cultures (Figure 29, lanes 2 and 3) but very faint fluorescent bands<br>
migrating at the correct position were again detected with all three DhaA.ElSO<br>
substituted mutants (Figure 29, lanes 5-7). These results point to the possibility<br>
that the DhaA.ElSOQ, L and A mutants have the ability to trap covalent alkylenzyme<br>
intermediates. The efficiency of this reaction, however, appears to<br>
proceed at a dramatically reduced rate compared to the DhaA.H272F mutant<br>
enzyme.<br>
The results of this mutagenesis study suggest that the DhaA catalytic acid<br>
residue DhaA.ElSO plays an important structural role in the correct folding of<br>
the enzyme. The DhaA protein was clearly sensitive to substitutions at this<br>
amino acid position as evidenced by the presence of largely insoluble protein<br>
complexes in the DhaA.ElSOQ, DhaA.ElSOL and DhaA.ElSOA crude lysates.<br>
Nevertheless, based on SDS-PAGE and immunoblot analyses, a significant<br>
quantity of soluble DhaA protein was detected in the cell-free lysates of all three<br>
DhaA.El 30 mutants.<br>
Example IX<br>
Capturing of DhaA.H272F-Flag and DhaA.H272F-Flag Renilla Luciferase<br>
Fusion Proteins Expressed in Living Mammalian Cells<br>
Materials and Methods<br>
CHO-K1 cells were plated in 24 well plates (Labsystems) at a density of<br>
30,000 cells/cm2 and transfected with a pCIneo.DhaA.WT-Flag or<br>
pCIneo.hRLuc-connector-DhaA.H272F-Flag vector. Twenty-four hours later,<br>
media was replaced with fresh media containing 25 uM biotin-CigHsiO/t-Cl and<br>
0.1% DMSO, or 0.1% DMSO alone, and the cells were placed in a COa<br>
incubator for 60 minutes. At the end of the incubation, the media was removed,<br>
cells were quickly washed with PBS (pH 7.4; four consecutive washes; 1.0<br>
ml/cm2; 5 seconds each) and new media was added to the cells. In some<br>
experiments, the media was not changed. The cells were placed back in a CCh<br>
incubator.<br>
After 60 minutes, media was removed, and the cells were collected in<br>
PBS (pH=7.4,200 ul/well, RT) containing protease inhibitors (Sigma #P8340).<br>
The cells were lysed by trituriation through a needle (IM1 23GTW). Then, cell<br>
lysates were incubated with MagnaBind Streptavidin coated beads (Pierce<br>
#21344) according to the manufacturer's protocol. Briefly, cell lysates were<br>
incubated with beads for 60 minutes at room temperature (RT) using a rotating<br>
disk. Unbound material was collected; beads were washed with PBS (3 x 500<br>
ul, pH=7.4, RT) and resuspended in SDS-sample buffer (for SDS-PAGE<br>
analysis) or PBS (pH=7.4, for determination of R.Luc activity). Proteins were<br>
resolved on SDS-PAGE, transferred to a nitrocellulose membrane, analyzed with<br>
anti-Flag-Ab or anti-R.Luc-Ab, and bound antibody detected by an enhanced<br>
chemiluminescence (ECL) system (Pharmacia-Amersham). Activity of hR.Luc<br>
bound to beads was determined using Promega's "Renilla Luciferase Assay<br>
System" according to the manufacturer's protocol.<br>
Results<br>
Capturing of proteins expressed in living cells allows for analysis of<br>
those proteins with a variety of analytic methods/techniques. A number of<br>
capturing tools are available although most of those tools require generation of a<br>
highly specific antibody or genetically fusing a protein of interest with specific<br>
tag peptides/proteins (Jarvik and Telmer, 1998; Ragaut et al., 1999). However,<br>
those tags have only limited use for live cell imaging. To capture DhaA.H272F<br>
and functional proteins fused to DhaA.H272F, SA-coated beads were used<br>
(Savage et al., 1992).<br>
Biotin-CigHazCVCl was efficiently hydrolyzed by wild-type DhaA, and<br>
covalently bound to DhaA.H272F and DhaA.H272F fusion proteins in vitro and<br>
in vivo. Moreover, binding was observed both in E. coli and in mammalian<br>
cells. Control experiments indicated that about 80% of the DhaA.H272F-Flag<br>
protein expressed in CHO-K1 cells was labeled after a 60 minute treatment.<br>
CHO-K1 cells transiently expressing DhaA.H272F-Flag were treated<br>
with biotin-CuHaaCVCl. Biotin-CigHsaCVCl treated cells were lysed and cell<br>
lysates were incubated with SA-coated beads. Binding of DhaA.H272F to beads<br>
was analyzed by Western blot using anti-Flag11 antibody. As shown in Figure<br>
30D, DhaA.H272F-Flag capturing was not detected in the absence of biotin-<br>
CigH32O4-Cl treatment. At the same time, more than 50% of the DhaA.H272FFlag<br>
expressed in cells was captured on SA-coated beads if the cells were treated<br>
To show the capturing of functionally active proteins fused to<br>
DhaA.H272F-Flag, cells were transfected with a vector encoding hR.Lucconnector-<br>
DhaA.H272F-Flag, and the luciferase activity captured on the beads<br>
measured. As shown in Figure 30C, significant luciferase activity was detected<br>
on beads incubated with a lysate of Motin-CigHazO^Cl treated cells. At the<br>
same time, no luciferase activity was detected on beads incubated with a lysate<br>
from cells that were not treated with biotin-CigHsaCVCl. Moreover, no hR.Luc<br>
activity was detected on beads incubated with lysate from the cells treated with<br>
biotin-CigHsaCU-Cl when free biotin-CigHsaCVCl was not washed out.<br>
Taken together, these data show that functionally active protein (hRXuc)<br>
fused to the DhaA.H272F can be efficiently captured using biotin-CigHsaCVCl<br>
and SA-coated beads. The capture is biotin-dependent, and can be competed-off<br>
by excess of biotin-CigHsaO^Cl. As a significant inhibitory effect of the beads<br>
on the hR.Luc activity was observed (data not shown), SDS-PAGE and Western<br>
blot analysis with anti-R.Luc antibody were used to estimate the efficiency of<br>
capture of hR.Luc-connector-DhaA.H272F-Flag fusion protein. As shown in<br>
Figure 30D, more than 50 % of hR.Luc-connector-DhaA.H272F-Flag fusion<br>
protein can be captured in biotin-dependent manner. This is in good agreement<br>
with the capturing efficiency of DhaA.H272F-Flag (see Figure 30A).<br>
Example ft<br>
Optimized DhaA Gene<br>
DhaA General Sequence Design<br>
A synthetic DAoA.H272F gene was prepared which had a human codon<br>
bias, low CG content, selected restriction enzyme recognition sites and a reduced<br>
number of transcription regulatory sites. Relative to the amino sequence<br>
encoded by a wild-type DhaA gene which lacks a signal sequence (SEQ ID<br>
NO:51), and/or to Z)/iaA.H272F, the amino acid sequence of a codon-optimized<br>
DhaA gene and flanking sequences included: 1) a Gly inserted at position 2, due<br>
to introduction of an improved Kozak sequence (GCCACCATGG; SEQ ED<br>
N0:45) and a BamHI site (thus the H272F active site mutation in DhaA mutants<br>
with the Gly insertion is at position 273); 2) a A292G substitution due to<br>
introduction of a SmallXmallAval site which, in the DhaA. mutant with the Gly<br>
insertion, is at position 293; 3) the addition of Ala-Gly at the C-terminus due to<br>
introduction of a Nael (JVgoMIV) site; 4) the addition ofNhel, Pvull, EcoRV and<br>
Ncol sites in the 5' flanking sequence; 5) the addition of NNNN in the 5' flanking<br>
sequence to eliminate search algorithm errors at the end and to maintain the<br>
ORF1 (i.e., NNN-NGC-TAG-CCA-GCT-GGC-GAT-ATC-GCC-ACC-ATGGGA;<br>
SEQ ID NO:46); 6) at the 3' end a Notl site, the addition of NNNN to<br>
eliminate search algorithm errors at the end, aPocI site with ORF Leu-Ile-Lys,<br>
and two stop codons, at least one of which is a TAA (i.e.,<br>
TAATAGTTAATTAAGTAAGCGGCCGCNNNN; SEQ ID NO:47).<br>
SEQ ID NO:51 has the following sequence:<br>
atgtcagaaatcggtacaggcttccccttcgacccccattatgtggaagtcctgggcgagcgtat<br>
gcactacgtcgatgttggaccgcgggatggcacgcctgtgctgttcctgcacggtaacccgacct<br>
cgtcctacctgtggcgcaacatcatcccgcatgtagcaccgagtcatcggtgcattgctccagac<br>
ctgatcgggatgggaaaatcggaoaaaccagacctcgattatttcttcgacgaccacgtccgcta<br>
cctcgatgccttcatcgaagccttgggtttggaagaggtcgtcctggtcatccacgactggggct<br>
cagctctcggattccactgggccaagcgcaatccggaacgggtcaaaggtattgcatgtatggaa<br>
ttcatccggcctatccogacgtgggacgaatggccggaattcgcccgtgagaccttccaggcctt<br>
ccggaccgccgacgtcggccgagagttgatcatcgatcagaacgctttcatcgagggtgcgct.ee<br>
cgaaatgcgtcgtccgtccgcttacggaggtcgagatggaccactatcgcgagcccttcctcaag<br>
cctgttgaccgagagccactgtggcgattccccaacgagctgcccatcgccggtgagcccgcgaa<br>
catcgtcgcgctcgtcgaggcatacatgaactggctgcaccagtcacctgtcccgaagttgttgt<br>
tctggggcacacccggcgtactgatccccccggccgaagccgcgagacttgccgaaagcctcccc<br>
aactgcaagacagtggacatcggcccgggattgcactacctccaggaagacaacccggaccttat<br>
cggcagtgagatcgcgcgctggctccccgcactctag<br>
Codon Selection<br>
Codon usage data was obtained from the Codon Usage Database<br>
(http://www.kazusa.or.jp/codonA. which is based on: GenBank Release 131.0 of<br>
15 August 2002 (See, Nakamura et al., 2000). Codon usage tables were<br>
downloaded for HS: Homo sapiens [gbpri] 50,031 CDS's (21,930,294 codons);<br>
MM: Mus musculus [gbrod] 23,113 CDS's (10,345,401 codons); EC:<br>
Escherichia coli [gbbct] 11,985 CDS's (3,688,954 codons); and EC K12:<br>
Escherichia coli K12 [gbbct] 4,291 CDS's (1,363,716 codons). HS and MM<br>
were compared and found to be closely similar, thus the HS table was used. EC<br>
and EC K12 were compared and found to be closely similar, therefore the EC<br>
K12 table was employed.<br>
The overall strategy for selecting codons was to adapt codon usage for<br>
optimal expression in mammalian cells while avoiding low-usage E. coli codons.<br>
One "best" codon was selected for each amino acid and used to back-translate<br>
the desired protein sequence to yield a starting gene sequence. Another selection<br>
criteria was to avoid high usage frequency HS codons which contain CG<br>
dinucleotides, as methylation of CG has been implicated in transcriptional gene<br>
regulation and can cause down-regulation of gene expression in stable cell lines.<br>
Thus, all codons containing CG (8 human codons) and TA (4 human codons,<br>
except for Tyr codons) were excluded. Codons ending in C were also avoided as<br>
they might form a CG with a downstream codon. Of the remaining codons,<br>
those with highest usage in HS were selected, unless a codon with a slightly<br>
lower usage had substantially higher usage in E. coli.<br>
DhaA Gene Sequences<br>
To generate a starting DhaA. sequence, codon usage tables in Vector NTI<br>
8.0 (Informax) were employed. The DhaA.v2.1 protein sequence (SEQ ID<br>
NO:48) was back translated to create a starting gene sequence, hD/wA.v2.1-0,<br>
and flanking regions were then added, as described above, to create hD/iaA.v2.1 -<br>
OF (SEQ ID NO:49).<br>
DhaA.v2.1:<br>
MGSEIGTGFPFDPHYVEVLGERMHYVDVGPRDGTPVLFLHGNPTSSYLW<br>
RNIEPHVAPSHRCIAPDLIGMGKSDKPDLDYFFDDHVRYLDAFIEALGLEE<br>
WLVfflDWGSALGFHWAKRNPERVKGIACMEFIRPIPTWDEWPEFARET<br>
FQAFRTADVGRELIIDQNAFIEGALPKCWRPLTEVEMDHYREPFLKPVD<br>
REPLWRFPNELPIAGEPANIVALVEAYMNWLHQSPWKLLFWGTPGVLIP<br>
PAEAARLAESLPNCKTVDIGPGLFYLQEDNPDLIGSEIARWLPGLAG<br>
(SEQ ID NO:48)<br>
hDhaA.v2.1-OF:<br>
NNNNGCTAGCCAGCTGGCGATATCGCCACCATGGGATCCGAGATTGG<br>
GACAGGGTTTCCTTTTGATCCTCATTATGTGGAGGTGCTGGGGGAGA<br>
GAATGCATTATGTGGATGTGGGGCCTAGAGATGGGACACCTGTGCTG<br>
TTTCTGCATGGGAATCCTACATCTTCTTATCTGTGGAGAAATATTATT<br>
CCTCATGTGGCTCCTTCTCATAGATGTATTGCTCCTGATCTGATTGGG<br>
ATGGGGAAGTCTGATAAGCCTGATCTGGATTATTTTTTTGATGATCAT<br>
GTGAGATATCTGGATGCTTTTATTGAGGCTCTGGGGCTGGAGGAGGT<br>
GGTGCTGGTGATTCATGATTGGGGGTCTGCTCTGGGGTTTCATTGGGC<br>
TAAGAGAAATCCTGAGAGAGTGAAGGGGATTGCTTGTATGGAGTTTA<br>
TTAGACCTATTCCTACATGGGATGAGTGGCCTGAGTTTGCTAGAGAG<br>
ACATTTCAGGCTTTTAGAACAGCTGATGTGGGGAGAGAGCTGATTAT<br>
TGATCAGAATGCTTTTATTGAGGGGGCTCTGCCTAAGTGTGTGGTGA<br>
GACCTCTGACAGAGGTGGAGATGGATCATTATAGAGAGCCTTTTCTG<br>
AAGCCTGTGGATAGAGAGCCTCTGTGGAGATTTCCTAATGAGCTGCC<br>
TATTGCTGGGGAGCCTGCTAATATTGTGGCTCTGGTGGAGGCTTATAT<br>
GAATTGGCTGCATCAGTCTCCTGTGCCTAAGCTGCTGTTTTGGGGGAC<br>
ACCTGGGGTGCTGATTCCTCCTGCTGAGGCTGCTAGACTGGCTGAGTC<br>
TCTGCCTAATTGTAAGACAGTGGATATTGGGCCTGGGCTGTTTTATCT<br>
GCAGGAGGATAATCCTGATCTGATTGGGTCTGAGATTGCTAGATGGC<br>
TGCCCGGGCTGGCCGGCTAATAGTTAATTAAGTAAGCGGCCGCNNlSrN<br>
(SEQ ID NO:49)<br>
Further Optimization<br>
Programs and databases used for identification and removal of sequence<br>
motifs were from Genomatix Software GmbH (Munich, Germany,<br>
http://www.genomatix.4e'&gt;: GEMS Launcher Release 3.5.1 (April 2003),<br>
Matlnspector professional Release 6.1 (January 2003), Matrix Family Library<br>
Ver 3.1.1 (April 2003, including 318 vertebrate matrices in 128 families),<br>
Modellnspector professional Release 4.8 (October 2002), Model Library Ver 3.1<br>
(March 2003,226 modules), SequenceShaper tool, and User Defined Matrices.<br>
The sequence motifs to be removed from starting gene sequences in order of<br>
priority were restriction enzyme recognition sequences listed below;<br>
transcription factor binding sequences including promoter modules (i.e., 2<br>
transcription fector binding sites with defined orientation) with a default score or<br>
greater, and vertebrate transcription factor binding sequences with a minimum<br>
score of =0.75/matrix^ptimized; eukaryotic transcription regulatory sites<br>
including a Kozak sequence, splice donor/acceptor sequences, polyA addition<br>
sequences; and prokaryotic transcription regulatory sequences including E. coli<br>
promoters and E. coli RBS if less than 20 bp upstream of a Met codon.<br>
User-defined Matrices<br>
Subset DhaA<br>
Format: Matrix name (core similarity threshold/matrix similarity<br>
threshold): U$AatII (0.75/1.00), U$BamHI (0.75/1.00), U$BglI (0.75/1.00),<br>
USBglll (0.75/1.00), USBsal (0.75/1.00), U$BsmAI (0.75/1.00), U$BsmBI<br>
(0.75/1.00), USBstEII (0.75/1.00), USBstXI (0.75/1.00), U$Csp45I (0.75/1.00),<br>
U$CspI (0.75/1.00), USDral (0.75/1.00), U$EC-P-10 (1.00/Optimized), U$ECP-<br>
35 (l.OO/Optimized), USEC-Prom (1.00/Optimized), U$EC-RBS (0.75/1.00),<br>
USEcoRI (0.75/1.00), USEcoRV (0.75/1.00), USHindlll (0.75/1.00), USKozak<br>
(0.75/Optimized), U$KpnI (0.75/1.00), USMluI (0.75/1.00), U$NaeI (0.75/1.00),<br>
USNcoI (0.75/1.00), USNdel (0.75/1.00), U$NheI (0.75/1.00), U$NotI<br>
(0.75/1.00), USNsil (0.75/1.00), U$PacI (0.75/1.00), U$PflMI (0.75/1.00),<br>
U$PmeI (0.75/1.00), USPolyAsig (0.75/1.00), USPstI (0.75/1.00), USPvuII<br>
(0.75/1.00), USSacI (0.75/1.00), U$SacH (0.75/1.00), U$SalI (0.75/1.00), USSfil<br>
(0.75/1.00), USSgfl (0.75/1.00), U$SmaI (0.75/1.00), USSnaBI (0.75/1.00),<br>
USSpel (0.75/1.00), U$Splice-A (0.75/Optimized), U$Splice-D<br>
 (0.75/Optimized), USXbal (0.75/1.00), USXcml (0.75/1.00), USXhoI<br>
(0.75/1.00), and ALL vertebrates.lib.<br>
Subset DhaA-EQ<br>
Without K coli specific sequences: USAatll (0.75/1.00), U$BamHI<br>
(0.75/1.00), USBgll (0.75/1.00), USBglll (0.75/1.00), U$BsaI (0.75/1.00),<br>
USBsmAI (0.75/1.00), USBsmBI (0.75/1.00), USBstEII (0.75/1.00), USBstXI<br>
(0.75/1.00), U$Csp45I (0.75/1.00), USCspI (0.75/1.00), USDral (0.75/1.00),<br>
USEcoRI (0.75/1.00), U$EcoRV (0.75/1.00), USHindlll (0.75/1.00), U$Kozak<br>
(0.75/Optimized), USKpnl (0.75/1.00), U$MluI (0.75/1.00), USNael (0.75/1.00),<br>
USNcoI (0.75/1.00), U$NdeI (0.75/1.00), U$NheI (0.75/1.00), U$NotI<br>
(0.75/1.00), USNsil (0.75/1.00), U$PacI (0.75/1.00), USPflMI (0.75/1.00),<br>
USPmel (0.75/1.00), USPolyAsig (0.75/1.00), U$PstI (0.75/1.00), USPvuII<br>
(0.75/1.00), USSacI (0.75/1.00), U$SacH (0.75/1.00), U$SalI (0.75/1.00), USSfil<br>
(0.75/1.00), U$Sgfl (0.75/1.00), USSmal (0.75/1.00), USSnaBI (0.75/1.00),<br>
USSpel (0.75/1.00), U$Splice-A (0.75/Optimized), U$Splice-D<br>
(0.75/Optimized), USXbal (0.75/1.00), USXcml (0.75/1.00), USXhoI<br>
(0.75/1.00), and ALL vertebrates.lib.<br>
Strategy fop Removal of Sequence Motifs<br>
The undesired sequence motifs specified above were removed from the<br>
starting gene sequence by selecting alternate codons that allowed retention of the<br>
specified protein and flanking sequences. Alternate codons were selected in a<br>
way to conform to the overall codon selection strategy as much as possible.<br>
A. General Steps<br>
- Identify undesired sequence matches with Matlnspector using matrix family<br>
subset "DhaA" or "DhaA-EC" and with Modellnspector using default<br>
settings.<br>
- Identify possible replacement codons to remove undesired sequence matches<br>
with SequenceShaper (keep ORF).<br>
Incorporate all changes into a new version of the synthetic gene sequence<br>
and re-analyze with Matlnspector and Modellnspector.<br>
B. Specific Steps<br>
- Remove undesired sequence matches using subset "DhaA-EC" and<br>
SequenceShaper default remaining thresholds (0.70/Opt-0.20).<br>
- For sequence matches that cannot be removed with this approach use lower<br>
SequenceShaper remaining thresholds (e.g., 0.70/Opt-0.05).<br>
- For sequence matches that still cannot be removed, try different<br>
combinations of manually chosen replacement codons (especially if more<br>
than 3 base changes might be needed). If that introduces new sequence<br>
matches, try to remove those using the steps above (a different starting<br>
sequence sometimes allows a different removal solution).<br>
- Use subset "DhaA" to check whether problematic E. coli sequences motifs<br>
were introduced, and if so try to remove them using an analogous approach<br>
to that described above for non E. coli sequences.<br>
Use an analogous strategy for the flanking (non-open reading frame) sequences.<br>
C. Identification and Removal of Putative CpG Islands<br>
Software used: EMBOSS CpGPlot / CpGReport<br>
http://www.ebi.ac.uk/emooss/op8plot/index.htmD (see, Gardiner-Garden et al.,<br>
1987).<br>
Parameters: default (modified): Window: 100; Step: 1; Obs/Exp: 0.6;<br>
MinPC: 50; Length: 100; Reverse: no; Complement: no. After the removal of<br>
undesked sequence motifs, the gene sequence was checked for putative CpG<br>
islands of at least 100 bases using the software described above. If CpG islands<br>
were identified, they were removed by selecting, at some of the CG dinucleotide<br>
positions, alternate codons that allowed retention of the specified<br>
protein and flanking sequences, but did not introduce new undesired sequence<br>
motifs.<br>
D. Restriction Sites<br>
A unique Murii/Mfel (C'AATTG) site was introduced to allow removal of<br>
the C-terminal 34 amino acids, including a putative myristylation site (GSEIAR)<br>
near the C-terminus. Another unique site, a Nrul site, was introduced to allow<br>
removal of the C-terminal 80-100 amino acids.<br>
hDhaA.v2.1-6F (FINAL, with flanking sequences)<br>
NNNNGCTAGCCAGCTGGCgcgGATATCGCCACCATGQGATCCGAGATT<br>
GGGACAGGGTTcCCTTTTGATCCTCAcTATGTtGAaGTGCTGGGgGAaAG<br>
AATGCAcTAcGTGGATGTGGGGCCTAGAGATGGGACcCCaGTGCTGTTc<br>
CTcCAcGGGAAcCCTACATCTagcTAcCTGTGGAGaAAtATTATaCCTCAT<br>
GTtGCTCCTagtCATAGgTGcATTGCTCCTGATCTGATcGGGATGGGGAA<br>
GTCTGATAAGCCTGActtaGAcTAcTTTTTTGATGAtCATGTtcGATActTGG<br>
ATGCTTTcATTGAGGCTCTGGGGCTGGAGGAGGTGGTGCTGGTGATaC<br>
AcGAcTGGGGGTCTGCTCTGGGGTTTCAcTGGGCTAAaAGgAATCCgGA<br>
GAGAGTGAAGGGGATTGCTTGcATGGAgTTTATTcGACCTATTCCTACt<br>
TGGGAtGAaTGGCCaGAGTTTGCcAGAGAGACATTTCAaGCcTTTAGAA<br>
CtGCcGATGTGGGcAGgGAGCTGATTATaGAcCAGAATGCTTTcATcGAG<br>
GGGGCTCTGCCTAAaTGTGTaGTcAGACCTCTcACtGAaGTaGAGATGGA<br>
cCATTATAGAGAGCCcTTTCTGAAGCCTGTGGATcGcGAGCCTCTGTGG<br>
AGgTTtCCaAATGAGCTGCCTATTGCTGGGGAGCCTGCTAATATTGTGG<br>
CTCTGGTGGAaGCcTATATGAAcTGGCTGCATCAGagTCCaGTGCCcAAG<br>
CTaCTcTTTTGGGGGACtCCgGGaGTtCTGATTCCTCCTGCcGAGGCTGCT<br>
AGACTGGCTGAaTCcCTGCCcAAtTGTAAGACcGTGGAcATcGGcCCtGGg<br>
CTGTTTTAcCTcCAaGAGGAcAAcCCTGATCTcATcGGGTCTGAGATcGCa<br>
cGgTGGCTGCCCGGGCTGGCCGGCTAATAGTTAATTAAGTAgGCGGCC<br>
GCNNNN (SEQ ID NO:50)<br>
A comparison of the nucleic acid sequence identity of different DhaA<br>
genes (without flanking sequences) is shown in Table III.<br>
Table III<br>
(Table Removed)<br>
"Giy added at position 2, H272F, A292G, Ala-Gly added to C-terminus<br>
bcodon optimized<br>
The GC content of different DhaA genes (without flanking sequences) is<br>
provided in Table IV,<br>
112<br>
Table IV<br>
(Table Removed)<br>
Vertebrate transcription factor binding sequence families (core similarity:<br>
0.75/matrix similarity: opt) and promoter modules (default parameters:<br>
optimized threshold or 80% of maximum score) found in different DhaA genes<br>
are shown in Table V.<br>
Gene name<br>
(Table Removed)<br>
Note: 3 bp insertion before EcoRV in hDhaA.v.2.1 -OF and in hDhaA.v2.1-6F to<br>
remove 5' binding sequence matches in 3' flanking region.<br>
The remaining transcription factor binding sequence matches in<br>
hDhaA.v2.1-6F included<br>
in the 5' flanking region: Family: VSNEUR (NeuroD, Beta2, HLH<br>
domain), best match: DNA binding site for NEUROD1 (BETA-2 / E47 dimer)<br>
(MEDLINE 9108015): in the open reading frame: Family: VSGATA (GATA<br>
binding factors), best match: GATA-binding factor 1 (MEDLINE 94085373V<br>
Family: VSPCAT (Promoter CCAAT binding factors), best match: cellular and<br>
viral CCAAT box, (MEDLINE 9Q23Q2991 Family: VSRXRF (RXR<br>
heterodimer binding sites), best match: Farnesoid X - activated receptor<br>
(RXR/FXR dimer) (MEDLINE 11792716): and in the 3' flanking region:<br>
Family: VSHNFl (Hepatic Nuclear Factor 1), best match: Hepatic nuclear factor<br>
1 (MEDLINE 95194383). Family: VSBRNF (Brn POU domain factors), best<br>
match: POU transcription factor Brn-3 (MEDLINE 91113081 Family: VSRBIT<br>
 (Regulator of B-Cell IgH transcription), best match: Bright, B cell regulator of<br>
IgH transcription (MEDLINE 96127903). Family: VSCREB (Camp-Responsive<br>
Element Binding proteins), best match: E4BP4, bZEP domain, transcriptional<br>
represser (MEDLINE 22118224), Family: VSHOMS (Homeodomain subfamily<br>
S8), best match: Binding site for S8 type homeodomains (MEDLINE<br>
94Q5_i59_3J, Family: VSNKXH (NKX/DLX - Homeodomain sites), best match:<br>
DLX-1, -2, and -5 binding sites (MEDLINE 11798166). Family: VSTBPF (Tata-<br>
Binding Protein Factor), best match: Avian C-type LTR TATA box (MEDLINE<br>
6322120). and Family: VSNKXH (NKX/DLX - Homeodomain sites), best<br>
match: Prostate-specific homeodomain protein NKX3.1 (MEDLINE 10871372).<br>
The other sequence motifs remaining in hDhaA.v2.1-6F in the open<br>
reading frame were for an E. coli RBS (AAGG) l ib upstream of a Met codon<br>
which was not removed due to retain the protein sequence (Lys-Gly: AA(A/G)-<br>
GGN), and a BsmAl restriction site (GTCTC) which was riot removed due to<br>
introduction of transcription factor binding site sequences.<br>
The putative CpG islands in the coding sequence for each of the DhaA<br>
genes was analyzed as in EMBOSS CpGPlot/CpGReport with default<br>
parameters, and the results are shown in Table VI.<br>
Table VI<br>
(Table Removed)<br>
References<br>
Ambler et al., Biochem. J.. 27&amp;4710 (1991).<br>
Ausubel et al., Current Protocols in Molecular Biology. Vol. Ill, A. 1(3-<br>
4), Supplement 38 (1997).<br>
Chalfie, M. and Kain, S. R., eds., GFP: GrecQ Fluorescent Protein<br>
Strategies and Applications (Wiley, New York, 1998).<br>
Cubitt et al., Trends Biochem. Sci.. 20j448 (1995).<br>
Eu and Andrade, Luminescence, 1&amp;57-63 (2001).<br>
Farinas et al., J. Biol. Chem.. 224:7603 (1999).<br>
Franken et al., EMBO J.. 10:1297 (1991).<br>
Gardiner-Garden et al., J. Mol. Biol.. 19jS_:261 (1987).<br>
Griffin et al., Science. 211:269 (1998).<br>
Hermanson, Bioconiugate Techniques. Academic Press, San Diego, CA<br>
(1996).<br>
Ho et al., Gene. 22:51 (1989).<br>
Holloway et al., J. Microbiol. Methods. 3.2:31 (1998).<br>
Hynkova et al., FEBS Lett.. 446:177 (1999).<br>
Janssen et al., Bur. J. Biochem.. 171:67 (1988).<br>
Janssen et al., J. Bactenol. 121:6791 (1989).<br>
Jarvik and Telmer, Ann. Rev. Genet.. 32:601-618 (1998).<br>
Keppler et al., Nature Biotechnology. 21:86 (2003).<br>
Keuning et al., J. Bactenol.. 163:635 (1985).<br>
Kneen et al., Biophvs. J.. 24:1591 (1998).<br>
Krooshof et al., Biochemistry. 3&amp;9571 (1997).<br>
Kulakova et al., Microbiology. 141:109 (1997).<br>
Lakowicz, J.R. Principles of Fluorescence Spectroscopy, New York:<br>
Plenum Press (1983).<br>
Llopis et al., Proc. Natl. Acad. Sci. USA. g5_:6803 (1998).<br>
MiesenbSck et al., Nature. 3_9_4:192 (1998).<br>
Minasov et al., J. Am. Chem. Soc.. 124:5333 (2002).<br>
Miyawaki et al., Nature. 3J5&amp;882 (1967).<br>
Nagata et al., Appl. Environ. Microbiol.. 61:3707 (1997).<br>
Nakamura et al., Nucl. Acids. Res.. 28:292 (2000).<br>
Newman et al., Biochemistry. 38,16105 (1999).<br>
Ormo' et al., Science. 223:1392 (1996).<br>
Pries et al., J. Biol. Chem.. 27J:10405 (1995).<br>
Ragaut et al., Nat. BiotechnoL 17:1030-1032 (1999).<br>
Rosomer et al., J. Biol. Chem.. 222:13270 (1997).<br>
Sallis et al., J. Gen. Microbiol.. JJ&amp; 115 (1990).<br>
Sambrook et al., Cold Spring Harbor Laboratory Press, Cold Spring<br>
Harbor, N.Y. 1989.<br>
Sanger et al., Proc. Natl. Acad. Sci. U.S.A.. 74:5463 (1977).<br>
Savage et al., Avidin-Biotin Chemistry: A Handbook (Pierce Chemical<br>
Company, Rockford, IL) (1992).<br>
Schindler, Biochemistry. 28:5772 (1999).<br>
Scholtz et al., J. BacterioL. lยฃg:5016 (1987).<br>
Silverman, Mechanism-based enzyme in activation, in Methods<br>
Enzvmologv. 249_:240 (1995).<br>
Stroffekova et al., Bur. J. PhvsioL 442:859 (2001).<br>
Tsien, Ann. Rev. Biochem.. 62:509 (1998).<br>
Yokota et al., J. BacterioL. 132:4049 (1987).<br>
Zawadzke et al., Protein Engineering. 8:1275 (1995).<br>
Zlokarnik et al., Science. 279:84 (1998).<br>
All publications, patents and patent applications are incorporated herein<br>
by reference. While in the foregoing specification this invention has been<br>
described in relation to certain preferred embodiments thereof, and many details<br>
have been set forth for purposes of illustration, it will be apparent to those skilled<br>
in the art that the invention is susceptible to additional embodiments and that<br>
certain of the details described herein may be varied considerably without<br>
departing from the basic principles of the invention.<br><br><br><br><br><br><br><br>
WE CLAIM:<br>
1.	A compound of formula (I): R-A-X, wherein R is a functional group as herein defined, A is a linker, which is a branched or unbranched carbon chain comprising from 2 to 30 carbon atoms, the said chain optionally includes one or more double or triple bonds, and is optionally substituted with one or more hydroxy or oxo (=0) groups, wherein one or more of the carbon atoms in the chain are optionally replaced with a non-peroxide --O--, --S-- or --NH--, the linker-A separates R and X by at least 11 atoms, A is (CH2)n and n=2-10, A-X is a substrate for dehalogenase, and X is CI.<br>
2.	A compound as claimed in claim 1 wherein a compound of formula (I): R -A-X, wherein A is a linker, which is a branched or unbranched carbon chain comprising from 2 to 30 carbon atoms, the said chain optionally includes one or more double or triple bonds, and if substituted includes one or more hydroxyl groups, wherein one or more of the carbon atoms in the chain is optionally replaced with a non-peroxide --O--, --S-- or --NH--, the linker-A separates R and X by at least 11 atoms, A is (CH2)n and n=2-10, A-X is a substrate for dehalogenase, and X is CI, and R is a functional group comprising biotin, a solid support or a xanthene, a coumarin, a chromene, an indole, an isoindole, an oxazole, a BOD1PY, an imidazole, a pyrimidine, a thiophene, a pyrene, a benzopyrene, a benzofuran, a phenalenone, an acridinone, a resorufm, a naphthalene, an anthracene, an acridinium, an a-naphthol, a ร-naphthol and a dansyl.<br><br>
3.	A compound as claimed in claim 1 or 2 wherein the protein dehalogenase is Rhodococcus dehalogenase.<br>
4.	A compound as claimed in claim 1 or 2 wherein the linker comprises 3 to 30 atoms.<br><br><br>
A compound as claimed in claim 1 or 2 wherein the linker has 11 to 30 atoms.<br>
A compound as claimed in claim 1 or 2 is N-{2-[2-(6-Chlorohexyloxy)-ethoxy] -ethyl} -biotin-amide.<br>
A compound as claimed in claim 1 or 2 wherein R is separated from A-X by up to 100 angstroms.<br>
A compound as claimed in claim 1 or 2 wherein R is separated from A-X by up to 500 angstroms.<br>
A compound as claimed in claim 1 or 2 wherein the carbon chain comprises (CH2CH20)y and y=2-8.<br>
A compound as claimed in claim 1 wherein R is biotin, a solid support or a fluorophore.<br>
A compound as claimed in claim 10 wherein the fluorophore is a xanthene, a coumarin, a chromene, an indole, an isoindole, an oxazole, a BODIPY, an imidazole, a pyrimidine, a thiophene, a pyrene, a benzopyrene, a benzofuran, a phenalenone, an acridinone, a resorufin, a naphthalene, an anthracene, an acridinium, an a-naphthol, a P-naphthol and a dansyl.<br>
A compound as  claimed in claim 11 wherein the xanthene is a fluorescein, a rhodamine or a rhodol.<br>
A compound as   claimed  in claim 12 wherein the fluorescein comprises 5- (or 6-) carboxy fluorescein.<br>
A compound as claimed in claim 1 or 2 wherein n=4-10.<br><br><br>
A compound as claimed in claim 1 is preferably of formula (II):<br>
(Formula Removed) <br><br>
A method for preparing a compound of the formula R-A-X comprising coupling a compound of formula R-Y with a compound of formula Z-A-X, wherein Y and Z are groups that can react to link R to A-X, wherein A is a linker, which is a branched or unbranched carbon chain comprising from 2 to 30 carbon atoms, which chain optionally includes one or more double or triple bonds, and which chain is optionally substituted with one or more hydroxy or oxo (=0) groups, wherein one or more of the carbon atoms in the chain is optionally replaced with a non-peroxide --O--, -- S-- or --NH--, wherein the linker-A separates R and X by at least 11 atoms, wherein A is (CH2)n and n=2-10, wherein X is CI, and A-X is a substrate for a dehalogenase, wherein R is a functional group with a carboxy group that is capable of being coupled through the carboxy group to the linker-A, and R-Y is an activated ester of R and wherein Z is an amine suitable to react with the activated ester to form an amide bond.<br>
A method for preparing a compound of the formula R-A-X wherein A is a linker, which comprises an amide bond comprising coupling a corresponding activated ester with a corresponding amine to provide the compound of formula R-A-X, wherein R is a functional group, the said linker -A is a branched or unbranched carbon chain comprising from 2 to 30 carbon atoms, which chain optionally includes one or more double or triple bonds, and which chain is optionally substituted with one or more hydroxy or oxo (=0) groups, wherein one or more of the carbon atoms in the chain is optionally replaced with a non-peroxide --O--, --S-- or --NH--, wherein A is (CH2)n and n=2-10, A-X is a substrate for a dehalogenase, and X is CI.<br><br>
18.	A method for preparing a compound of the formula R-A-X wherein a is a<br>
linker comprises an amide bond comprising coupling a corresponding activated<br>
ester with a corresponding amine to provide the compound of formula R-A-X,<br>
wherein R is a functional group, the linker- A is a branched or unbranched<br>
carbon chain comprising from 2 to 30 carbon atoms, which chain optionally<br>
includes one or more double or triple bonds, and which chain if substituted<br>
includes one or more hydroxyl groups, wherein one or more of the carbon<br>
atoms in the chain is optionally replaced with a non-peroxide --O--, --S-- or --<br>
NHโ, wherein A is (CH2)n and n=2-10, A--X is a substrate for a dehalogenase<br>
and X is a halogen.<br>
19.	A compound of formula (I): R-A-X, wherein R, A, X, has the meaning as<br>
herein before described, substantially such as herein described.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1kZWxucC0yMDA1LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">3867-delnp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1kZWxucC0yMDA1LWFzc2lnbm1lbnQucGRm" target="_blank" style="word-wrap:break-word;">3867-delnp-2005-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1ERUxOUC0yMDA1LUNsYWltcy0oMTktMDMtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">3867-DELNP-2005-Claims-(19-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1kZWxucC0yMDA1LWNsYWltcy0yOC0wOS0yMDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">3867-delnp-2005-claims-28-09-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1kZWxucC0yMDA1LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">3867-delnp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1ERUxOUC0yMDA1LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTktMDMtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">3867-DELNP-2005-Correspondence-Others-(19-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1ERUxOUC0yMDA1LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjMtMDktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">3867-DELNP-2005-Correspondence-Others-(23-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1kZWxucC0yMDA1LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">3867-delnp-2005-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1ERUxOUC0yMDA1LURlc2NyaXB0aW9uIChDb21wbGV0ZSktKDE5LTAzLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">3867-DELNP-2005-Description (Complete)-(19-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1ERUxOUC0yMDA1LURlc2NyaXB0aW9uIChDb21wbGV0ZSktKDIzLTA5LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">3867-DELNP-2005-Description (Complete)-(23-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1kZWxucC0yMDA1LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">3867-delnp-2005-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1ERUxOUC0yMDA1LURyYXdpbmdzLSgxOS0wMy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">3867-DELNP-2005-Drawings-(19-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1kZWxucC0yMDA1LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">3867-delnp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1ERUxOUC0yMDA1LUZvcm0tMS0oMjMtMDktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">3867-DELNP-2005-Form-1-(23-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1kZWxucC0yMDA1LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">3867-delnp-2005-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1kZWxucC0yMDA1LWZvcm0tMTMtKDI5LTA5LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">3867-delnp-2005-form-13-(29-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1kZWxucC0yMDA1LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">3867-delnp-2005-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1ERUxOUC0yMDA1LUZvcm0tMi0oMTktMDMtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">3867-DELNP-2005-Form-2-(19-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1kZWxucC0yMDA1LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">3867-delnp-2005-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1ERUxOUC0yMDA1LUZvcm0tMy0oMTktMDMtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">3867-DELNP-2005-Form-3-(19-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1kZWxucC0yMDA1LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">3867-delnp-2005-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1kZWxucC0yMDA1LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">3867-delnp-2005-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1kZWxucC0yMDA1LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">3867-delnp-2005-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1kZWxucC0yMDA1LXBjdC0xMDEucGRm" target="_blank" style="word-wrap:break-word;">3867-delnp-2005-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1kZWxucC0yMDA1LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">3867-delnp-2005-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1kZWxucC0yMDA1LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">3867-delnp-2005-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1kZWxucC0yMDA1LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">3867-delnp-2005-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1kZWxucC0yMDA1LXBjdC00MDEucGRm" target="_blank" style="word-wrap:break-word;">3867-delnp-2005-pct-401.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg2Ny1kZWxucC0yMDA1LXBjdC00MDIucGRm" target="_blank" style="word-wrap:break-word;">3867-delnp-2005-pct-402.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhdC5qcGc=" target="_blank" style="word-wrap:break-word;">abstrat.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="227542-fungicides.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="227544-a-feed-structure-for-adjusting-the-phase-difference-between-at-least-two-signal-components.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>227543</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3867/DELNP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>05/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>30-Jan-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>13-Jan-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>30-Aug-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>Promega Corporation</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>2800 WOODS HOLLOW ROAD, MADISON WISCONSIN 53711, USA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>KLAUBERT, DIETER</td>
											<td>250 BLUE SKY DRIVE ARROYO GRANDE, CALIFORNIA 93420 USA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>MCDOUGALL, MARK</td>
											<td>595 HERITAGE LANE ARROYO GRANDE, CALIFORNIA 93420 USA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>ZIMPRICH, CHAD</td>
											<td>909 VIRGIN LAKE DRIVE STOUGHTON, WISCONSIN 53589 USA</td>
										</tr>
										<tr>
											<td>4</td>
											<td>WOOD, KEITH V</td>
											<td>8380 SWAN ROAD, MOUNT HOREB, WISCONSIN, 53572, USA</td>
										</tr>
										<tr>
											<td>5</td>
											<td>LOS, GEORGYI V.</td>
											<td>3035 HARTWICKE DRIVE MADISON WISCONSIN 53711 USA</td>
										</tr>
										<tr>
											<td>6</td>
											<td>BULLEIT, ROBERT F</td>
											<td>210 JENNA DRIVE VERONA, WISCONSIN 53593</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2004/002607</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-01-30</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/444,094</td>
									<td>2003-01-31</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/474,094</td>
									<td>2003-05-30</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/227543-covalent-tethering-of-functional-groups-to-proteins by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:26:04 GMT -->
</html>
