<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/227522-antigenic-peptide-libraries by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:22:34 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 227522:&quot;ANTIGENIC PEPTIDE LIBRARIES&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;ANTIGENIC PEPTIDE LIBRARIES&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method for designing antigenic peptide libraries accounts for naturally occurring and potential variability in a group of protein sequences from a variable pathogen. The peptide libraries can elicit an immune response against a range of pathogen variants.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>ANTIGENIC PEPTIDES<br>
CLAIM OF PRIORITY<br>
This application claims priority to Russian Patent Application Serial No. 2002126396, filed on September 27, 2002, the entire contents of which are hereby incorporated by reference.<br>
FEDERALLY SPONSORED RESEARCH OR DEVELOPMENT<br>
The U.S. Government may have certain rights in this invention pursuant to Department of Health and Human Services Grant Nos. ISTC #2450 and BTEP #34.<br>
TECHNICAL FIELD<br>
This invention relates to antigenic peptides.<br>
BACKGROUND<br>
A number of pathogens important to human health are variable pathogens, such as HIV-1, HIV-2, hepatitis B and C, influenza, dengue types 1-4 viruses, malaria, and tuberculosis. These and other variable pathogens present a challenge to vaccine development, due to the extremely high genetic variability of the pathogen. In particular, the number of people infected with HIV throughout the world is growing rapidly, especially in countries with poor health care resources. The development of a highly efficient vaccine could help restrict the propagation of this epidemic. One of the main obstacles to developing such a vaccine is the extremely high genetic diversity of HIV. The genetic diversity is observed in various regions of the world in different subtypes, within a single subtype and even, in some cases, within an individual. This variability results from the high rate of mutations in viral proteins occurring during virus replication. See, for example, Coffin, J.M. Science 1995, 267:483-489; Delwart et al., J. Virol. 1994 68:6672-6683; Markham R.B. et al. Proc. Natl. Acad. Sci. U.S.A. 1998, 95:12568-12573; and McCutchan, F.E. et al. AIDS 1996 10 Suppl 3:S 13-S20, each of which is incorporated by reference in its entirety.<br>
It has been shown in animal models that simultaneous exposure to multiple antigenic determinants often induces immune responses to each of them (Thomson, S.A. et al. Proc. Natl. Acad. Sci. U.S.A. 1995, 92:5845-5849; Thomson, S.A. et al. J. Virol. 1998, 72: 2246-<br>
2252; and Woodberry, T. et al. J. Virol. 1999, 73:5320-5325, each of which is incorporated by reference in its entirety). This indicates that multiple antigenic determinants are suitable as components of a single vaccine construct.<br>
SUMMARY<br>
The protective component of an immune response to a variable pathogen is often directed against a variable region of a protein. If, on the other hand, the protective response was directed against a conserved region of a protein, pathogen variability would not present a problem to immunological control of infections and vaccine prophylaxis. However, an immune response against conserved regions of variable pathogens often proves to be nonprotective. A peptide antigen based on a variable region can induce a protective response, but the variability of these regions is so great that it is impractical to cover the variability by addition of individual peptide sequences. A potential vaccine against a variable pathogen should include a variety of antigens, to produce an immune response to the variants of the pathogen to which an individual may become exposed.<br>
Vaccine prophylaxis or therapy against variable infectious agents requires constructing a vaccine capable of inducing wide-range humoral and cellular immune responses to antigen variable regions. A challenge for vaccine prophylaxis and therapy against variable infectious agents is the development of a vaccine capable of inducing an immune response against variable or hypervariable regions of their protein components.<br>
A library of chimeric peptide antigens containing one or more potentially protective epitopes and vaccine constructs containing such antigens can produce a protective immune response. In particular, chimeric peptide libraries are able to induce a protective immune response including both humoral and cellular responses to HIV-1, HIV-2, hepatitis C and B viruses, influenza virus, dengue types 1-4 viruses, malaria, tuberculosis, or other variable    pathogens.<br>
Chimeric peptide libraries can be designed to represent naturally occurring or potential variants of protein epitopes. Peptide libraries developed in this manner can produce a broad immune cross-response and can be applied for constructing vaccines, Pharmaceuticals, and diagnostic kits. The design methods can be applied to a broad range of antigenic determinants of an infectious agent. Different methods can be applied to one infectious agent or one of its proteins or to a single epitope of the protein.<br>
Immunogenic sets of chimeric peptides, referred to as variable chimeric pep tide libraries (VCPLs), represent the naturally occurring and potential variability of antigenically active regions. Variable chimeric peptide libraries are sets of homologous peptides variable at one or more positions. They are designed to mimic the genetic diversity of variable protective determinants of an infectious agent. Such VCPLs can induce production of a wide range of antibodies and cytotoxic T-lymphocytes (CTLs) with a joint specificity that covers the diversity of antigenic variants of the variable infectious agent.<br>
In one aspect, a method of manufacturing a family of antigenic peptides includes locating a plurality of variable positions in a region of a pathogen protein, choosing a peptide sequence of the pathogen protein including the plurality of variable positions, selecting one or more substitute amino acid residues for one of the variable positions based on antigenic similarity to amino acid residues naturally occurring at the variable position of the pathogen protein, and preparing a family of antigenic peptides based on the peptide sequence and including the substitute amino acid residues.<br>
In another aspect, a method of designing a family of peptide sequences includes locating a plurality of variable positions in a region of a pathogen protein, choosing a peptide sequence of the pathogen protein including the plurality of variable positions, and selecting one or more substitute amino acid residues for one of the variable positions of the peptide based on antigenic similarity to amino acid residues naturally occurring at the variable position of the pathogen protein, thereby forming a family of peptide sequences.<br>
Selecting can include determining the antigenic similarity using an antigenic similarity matrix. A frequency can be assigned to each substitute amino acid residue in the family of antigenic peptides. Preparing can include weighting the substitute amino acid residues in the family of antigenic peptides based on the assigned frequency. Assigning can include considering the frequency with which the variations naturally occur. The pathogen protein can include a hypervariable region. The pathogen protein can be associated with a virus. The virus can be HIV, hepatitis B virus or hepatitis C virus, or an influenza. The pathogen protein can be HIV gp120. The region can be the V1, V2, V3, V4, or V5 regions. The pathogen protein can be associated with a malaria pathogen, a dengue pathogen, or a tuberculosis pathogen.<br>
The family of antigenic peptides can include members, such that the members taken together have antigenic similarity to each naturally occurring sequence of the region of the<br>
pathogen protein. The family of antigenic peptides can include members, such that the members taken together have antigenic similarity to a non-naturally occurring sequence of the region of the pathogen protein. The method can include identifying peptide sequences of the family, the identified peptide sequences being representative of the sequence diversity of the entire family. Fewer than 500 sequences can be identified as being representative of the sequence diversity of the entire family. Identifying can include calculating a distance between peptide sequences of the family. Calculating a distance can include using an antigenic similarity matrix.<br>
The method can include determining an antigenic similarity between a peptide of the family and a region of a human protein. The method can include removing a peptide from the family of antigenic peptides before preparing the family if the determined antigenic similarity between the peptide of the family and the region of a human protein exceeds a predetermined threshold.<br>
Preparing the family of antigenic peptides can include chemical synthesis of the family of peptides. The chemical synthesis can includes combinatorial synthesis, whereby the peptides are formed as a mixture of different sequences. The separate peptides can be mixed. The chemical synthesis can include parallel synthesis, whereby each peptide is formed separately from other peptides. Preparing the family of antigenic peptides can include expression of the family of peptides by a host organism.<br>
In another aspect, a composition includes a family of antigenic peptides having amino acid sequences having antigenic similarity to amino acid sequences of a variable region of a pathogen protein, wherein each antigenic peptide in the family has at least one amino acid position that varies relative to other antigenic peptides in the family.<br>
One amino acid residue can occur more frequently than another in the position that varies. The family can include greater than 150 or greater than 1,000 mutually unique antigenic peptides. The family can include fewer than 100,000 or fewer than 50,000 mutually unique antigenic peptides. The family can include between 1,000 and 50,000 mutually unique antigenic peptides. The family of antigenic peptides can include sequences having antigenic similarity to sequences from a subtype of HIV. The subtype can be subtype A, subtype B, subtype C, subtype D, subtype F, subtype G, a recombinant subtype, a subtype of HIV group N, a subtype of HIV group O, or combinations thereof. At least two members<br>
of the family of antigenic peptides can be mixed together. The family of antigenic peptides can be separated according to sequence. The family can include a multiple antigenic peptide.<br>
In another aspect, a method of eliciting an immune response in a subject includes administering to the subject a composition including a family of antigenic peptides having amino acid sequences having antigenic similarity to amino acid sequences of a variable region of a pathogen protein of a pathogen.<br>
Antigenic similarity can be determined using an antigenic similarity matrix. The composition can be administered to a subject prior to infection by the pathogen. The family of antigenic peptides can have amino acid sequences having antigenic similarity to amino acid sequences of a subtype of the pathogen protein. The family of antigenic peptides can have amino acid sequences having antigenic similarity to amino acid sequences from more than one subtype of the pathogen protein. The composition is administered to a subject infected by the pathogen. The subject can be infected by a subtype of the pathogen. The family of antigenic peptides can have amino acid sequences having antigenic similarity to amino acid sequences from the subtype by which the subject is infected.<br>
In another aspect, a method of diagnosing infection includes contacting a sample with a family of peptides having amino acid sequences having antigenic similarity to amino acid sequences of a variable region of a pathogen protein, wherein each peptide in the family has at least one amino acid position that varies relative to other peptides in the family.<br>
The family of peptides can be antigenically similar to a pathogen subtype, or to more than one pathogen subtype. The family of peptides can be immobilized on a substrate before contacting. The method can include determining if the sample includes antibodies that bind specifically to the family of peptides.<br>
In general, antigenic peptide libraries reflect the existing and potential diversity of a variable region of a pathogen protein. The peptide libraries can be used in prophylactic, therapeutic, and diagnostic applications. The full antigenic diversity of a pathogen epitope can be represented by the library. A library designed to reflect the diversity of all variants of a pathogen can be a broad-spectrum vaccine or diagnostic. A library designed to reflect the diversity one or more subtypes of pathogen can be a subtype-specific vaccine or diagnostic. Peptide sequences that are highly similar to sequences found in human proteins and are thus likely to induce an autoimmune reaction can be removed from the library. A library can be designed for any variable pathogen protein of known sequence. Examples of variable<br>
pathogens include HIV-1, HIV-2, hepatitis B and C, influenza, dengue types 1-4 viruses, malaria, and tuberculosis.<br>
The details of one or more embodiments are set forth in the description below. Other features and advantages will be apparent from the description and from the claims.<br>
DETAILED DESCRIPTION<br>
Much effort has been devoted to development of a vaccine against HIV in particular, but without success to date. To be efficient against the broad range of HIV-1 isolates, potential vaccines should contain immunogens of main isolates representing the entire range of HIV-1 isolates. It is necessary to identify these immunogens and improve their immunogenic properties (McMichael, A.J., and Hanke, T, Nature Medicine 2003, 9:874-880; Baltimore, D. and Heilman, C. Sci. Am. 1998, 279: 98-103; and van der Groen, G. et al. AIDS Res. Hum. Retroviruses 1998, 14 Suppl. 3: S211-S221, each of which is incorporated by reference in its entirety).<br>
Therefore, the resulting vaccine construct should contain key protective B- and T-cell epitopes specific to the broad range of HIV-1 isolates, combined with modern delivery means and adjuvants. Activation of both humoral and cellular branches of the immune system is required (Berzofsky, J.A., and Berkower, I.J., AIDS 1995, 9 Suppl. A: S143-S157, which is incorporated by reference in its entirety). The local immune response on mucous membranes and circulating neutralizing antibodies are considered to play key roles in preventing HIV infection, whereas virus-specific CTLs are likely to eliminate infected cells. See, for example, Lehner, T. et al. Science 1992, 258:1365-1369; and Goulder, P.J. and Walker, B.D. Nature Med. 1999, 5:1233-1235, each of which is incorporated by reference in its entirety.<br>
The amino acid sequence of the envelope glycoprotein (also called env or gp120) of HIV is highly variable between independent isolates and between sequential isolates from a single infected individual. The amino acid variability in gp120 is concentrated in specific variable regions, numbered VI-V5. The most variable regions often contain neutralizing epitopes so that the virus partially evades the host's immune response and establishes a persistent infection. The principal neutralizing determinant of gp120 is located in the third hypervariable region (V3).<br>
A number of candidate peptide vaccines against HIV-1 have been developed with the recognition of the V3 region as the principal neutralizing determinant in mind. For example,<br>
a method of rational design of a polyvalent subunit vaccine against HIV included determination of neutralizing epitopes of the V2 and V3 domains of HIV gp120 isolates from various areas of the world. See, for example, U.S. Patent No. 6,042,836, which is incorporated by reference in its entirety. Another approach to synthetic immunogenic HIV-1 peptides included designing a tandem synthetic peptide containing the sequences of T- and B-cell epitopes and the sequence of the V3 loop from various HIV isolates. See, for example, U.S. Patent No. 5,817,754, which is incorporated by reference in its entirety. However, the high genetic variability of the V3 domain results in a variety of sequences, which cannot be simultaneously introduced into a vaccine construct.<br>
For the same reasons as in the case of vaccines, it is extremely difficult to diagnose antibodies against hypervariable regions of variable infectious agents. These antibodies are the earliest and dominant in the immune response. An example is antibodies against the V3 domain of HIV-1 gp!20. See, for example, Baltimore, D. and Heilman, C. Sci. Am. 1998, 279: 98-103, Berzofsky, J.A., and Berkower, I.J., AIDS 1995, 9 Suppl. A: S143-S157, and Carlos, M.P. et al., AIDS Res. Hum. Retroviruses 2000, 16:153-161, each of which is incorporated by reference in its entirety.<br>
The diagnostics of such antibodies against the V3 hypervariable domain of gp120 depends on the genotypes of the antigens involved and the genotype of the infectious agent. It can require using multiple sequences corresponding to a broad range of HIV-1 variants. Otherwise, there is a high risk of missing a rare or modified variant of the virus.<br>
A peptide-based vaccine can include a library of peptides. The peptides can be similar to a variable region of a pathogen protein. In particular, the variable region can include a known antigenic determinant. The peptide sequences in the library are chosen to reflect the naturally occurring and potential variability of the variable region.<br>
Chimeric peptide libraries that mimic the genetic diversity of variable protein regions of a pathogen can induce a protective immune response to a broad spectrum of pathogen variants. To design the chimeric peptide library, a theoretical analysis of protein sequence information is carried out. The theoretical analysis results in a library design that can mimic all known and potential variants of the variable protein region. The library can be a subtype-specific library, mimicking only those variants belonging to a particular subtype of the pathogen, or a broad spectrum library, mimicking variants belonging to more than one<br>
subtype. The peptides of the library can bind antibodies that are specific to its various antigenic variants.<br>
In general, the design of an antigenic chimeric peptide library begins by inspecting sequences of a protein from a variable pathogen. A variable region of the protein is identified. The variable region can be a known antigenic determinant. A peptide fragment within the protein sequence including at least one variable position is selected to be the basis of the chimeric peptide library. The fragment includes one or more positions having naturally sequence variations. The peptide fragment can include non-varying positions. The fragment can be 10-50 residues in length, such as 15-45 residues or 20-40 residues in length. For each variable position in the peptide, the residues that occur in that position and the frequency with which each occurs are identified. The choice of residues at each variable position in the library is determined by considerations of antigenic similarity to the residues naturally occurring at that position. The frequency of each residue at a variable position is assigned according to the frequency that it or its corresponding antigenically similar residue occurs naturally. Selection of residues and frequencies is described in more detail below. The residue selection and frequency assignment can be automated, i.e. carried out by a computer program.<br>
The number of sequences in the library (the size of the library) is the product of the number of possible residues at each position in the peptide. The size of the library can include, for example, more than 200,000,000 sequences, fewer than 200,000,000 sequences, fewer than 500,000 sequences, fewer than 200,000 sequences, fewer than 100,000 sequences, fewer than 50,000 sequences, more than 5,000 sequences, more than 10,000 sequences, or more than 20,000 sequences. The size of the library can be reduced by eliminating the residue that occurs with the lowest frequency until a desired size is reached.<br>
Immunogenic sets of chimeric peptides, (variable chimeric peptide libraries or VCPLs) when taken together, represent the naturally occurring and potential variability of antigenically active regions. Variable chimeric peptide libraries are sets of homologous peptides variable at one or more positions. They are designed to mimic the genetic diversity of main variable protective determinants of an infectious agent. Such VCPLs can induce a wide range of antibodies and cytotoxic T-lymphocytes (CTLs), having a  joint specificity that covers the diversity of antigenic variants of the infectious agent. Thus, VCPLs induce<br>
humoral and cellular immune responses against the diversity of currently existing variants of an infectious agent and against potential variants which may arise in the future.<br>
Libraries of variable chimeric peptides can be constructed so that the conserved positions of these peptides represent amino acids common for all sequences, and the heterogeneous diversity of the variable positions is mimicked by one or several amino acids. The mimicking amino acids can be chimeric, in other words, they mimic the general heterogeneity of variable positions but need not occur at corresponding positions among the known and potential diversity of natural sequences.<br>
The method of designing VCPLs rests upon comprehensive investigation of the whole body of structural evidence on the organization of immunodominant protein epitopes and on the induction of a protective immune response to the target pathogen. The library design begins with collection and analysis of amino acid sequences from the target pathogen, and estimation of the genetic variability and epidemiological significance of pathogen subtypes. The amino acid sequences are searched for conserved, variable, and hypervariable regions that contribute to the development of a protective immune response against the pathogen.<br>
The antigenic properties of the peptide library can be controlled by selecting the pathogen protein sequences that the library is designed from. For example, a library mimicking a particular subtype of pathogen can be designed by selecting only sequences derived from that subtype. In general, the sequences within any one subtype or sub-subtype are more similar to each other than to sequences from other subtypes throughout their genomes. Group M is the main group of viruses in the HIV-1 global pandemic, and it contains multiple subtypes. Group O is the "outlier" group, and group N is a very distinctive form of the virus that is Non-M, Non-O. The HIV-1 subtypes from the M group of HIV-1, for example, are phylogenetically associated groups or clades of HIV-1 sequences, and are labeled Al, A2, B, C, D, Fl, F2, G, H, J and K. In addition, recombinant HIV-1 sequences are possible, where the viral genome includes regions of sequence from more than one subtype. See, for example, the HIV Sequence Database, at hiv-web.lanl.gov. These subtypes represent different lineages of HIV, and have some geographical associations. A library mimicking a particular subtype or recombinant form can be useful, for example, in cases where particular subtypes are found predominantly in particular geographic locations.<br>
The number of potential evolutionary variants of the sequence of an antigenically active region can be enormous. For example, the potential antigenic variability of the V3 region of HIV-1 gp!20 is as high as 1025-1030 variants. Once the complete variability at each position in the peptide is determined from the naturally occurring sequences, the variability of the desired VCPL can be reduced by recognizing the most significant amino acids that determine the genotypic diversity of the antigenic variants. The rates of occurrence of various amino acids at variable positions of an immunogenic epitope are determined with regard to their relative occurrence in the alignment of the sequences of the region at issue.<br>
The size of the VCPL can be decreased by recognizing the most antigenically significant amino acids at each variable position, and combining them in groups of variable composition with one or more amino acids representing the groups of variable composition. The amino acids representing groups of variable composition are determined on the basis of antigenic similarity.<br>
Antigenic similarity is a quantitative measure of how readily one amino acid residue can substitute for another in a protein-protein interaction, such as, for example, an antibody-antigen interaction. Two residues are very antigenically similar if substituting one for the other in a particular epitope has a small effect on the antigenic behavior of that epitope. Conversely, if the substitution has a large effect, then the two residues have a low antigenic similarity. In general, residues that have similar physical and chemical characteristics will have a high antigenic similarity.<br>
Antigenic similarity can be expressed in a matrix, called an antigenic similarity matrix (ASM). The construction of a matrix is described in, for example, Maksyutov, A.Z., Eroshkin, A.M., and Kulichkov, V.A, Mol. Biol. 1987, 21:30-47, which is incorporated by reference in its entirety. The matrix can account for the affinity of two residues, that is, how frequently two residues are in contact in a globular protein, as determined by examination of 3D structures. See, for example, Warme, P.K., and Morgan, R.S., J. Mol. Biol. 1978, 118: 289-304, which is incorporated by reference in its entirety. The matrix is also adjusted for the frequency with which an amino acid occurs in the hypervariable regions of immunoglobulins. See, for example, Bourgarit, J., J. Ann. Immunol. 1980, 13ID: 267-287, which is incorporated by reference in its entirety.<br>
The values of an initial matrix can be transformed by the formula:<br>
(Formula Removed)<br>
for each pair of amino acids i and j, and Rij and Rij• are elements of the initial matrix, thus producing a symmetric matrix. To differentiate the effects of coinciding amino acids and increase the weight of the residues that occur frequently in antigenic epitopes, the diagonal elements can be adjusted according to the following formula:<br>
(Formula Removed)<br>
where hi is the hydrophilicity of the amino acid (Hopp, T.P., and Woods, K.R., Proc. Natl. Acad. Sci. USA 1981, 78:3824-3828, which is incorporated by reference in its entirety), Sj is the relative frequency of the amino acid in antigenic determinants of proteins (Maksyutov, A.Z. and Zagrebelnaya, E.S., Comp. Applic. Biosci. 1993, 9:291-297, which is incorporated by reference in its entirety), and c is 6.6, the minimum value of hi + sj, introduced to insure a positive sum (hi + Si +c). Finally, to satisfy the local alignment program SIM, the quantity 75 is subtracted from each element in the matrix. One such matrix is presented in Table 1. The ASM of Table 1 is aimed at the search for locally similar regions exhibiting close structural features in respect to interaction between an antibody or another binding ligand and the binding site (see, for example, Maksyutov A. Z. et al., Molecular Biology 2002, 36: 346-358, which is incorporated by reference in its entirety).<br>
Qualitatively, a matrix value for a pair of amino acids indicates that they are good substitutes for one another, and a low number indicates that they are poor substitutes. For example, valine and leucine have a high antigenic similarity according to the matrix (matrix value of 9), but valine and aspartic acid do not (matrix value of-59), as would be expected from chemical considerations.<br>
Table 1<br>
(Table Removed)<br>
At each position in the peptide, the matrix value for each pair of residues occurring naturally is looked up. A pair having a positive matrix value will be represented in the library by the residue that occurs more frequently in the natural sequences, and its frequency in the library will be the sum of frequencies for each residue that it represents. The remaining residues are combined in subgroups, and for each subgroup, the matrix is searched for a residue that has a positive matrix value with each residue in the subgroup. If such a residue is found, it is assigned a frequency equal to the sum of frequencies of residues in the subgroup. This residue can be one that does not naturally occur in that position. If a naturally occurring residue does not have a positive matrix value with any other naturally occurring residue at that position, and cannot be represented by another residue not naturally occurring at that position, it will represent itself in the library.<br>
As an example of how the matrix can be used to choose residues for a particular position in a peptide library, consider a variable position in where the following six different residues occur naturally: A (5%), D (20%), E (35%), M (22%), V (10%), and Y (8%). The matrix value for each pair of naturally occurring residues is then looked up:AD	-41AE	-49AM	-7AV	-1AY	-25DE	4DM	-7<br>
(Table Removed)<br>
Only one pair, DE, has a positive matrix value. That pair of residues will be represented by the more frequent naturally occurring residue, E, and will have a frequency equal to the sum of the frequency of the residues it represents (20+35= 55%). The remaining residues, A, M, V and Y, are divided into subgroups and the matrix is searched for other residues that are antigenically similar to these. The subgroup of residues A, M, and V each have a positive matrix value with L:<br>
AL 3 ML 4 VL 9<br>
Thus, in the library, A, M, and V will be represented by L, which is antigenically similar to all three. The frequency of L in the library is the sum of the frequencies of A, M and V: 5+10+22 = 37%. Finally, since Y is not antigenically similar to another residue, Y represents itself in the library with a frequency of 8%. Table 2 summarizes how the residues from natural sequences are represented in the library for this example.<br>
Table 2<br><br>
(Table Removed)<br>
Theoretical analysis of the potential cross-reactivity of antibodies raised against peptide libraries shows that in order to induce an immune cross-response sufficiently broad<br>
to cover the whole set of antigenic variants of a variable infectious agent would require a library containing tens of thousands of homologous peptides. A smaller library having hundreds of homologous peptides could not induce a sufficiently broad immune response to existing and potential variants of a variable infectious agent. However, increasing the size of the library also increases the danger of that the library will elicit an autoimmune response. The analysis was carried out by methods for estimation of the cross-reactivity of antibodies against homologous antigens. See, for example, Maksyutov A.Z., Voprosy Virusologii 1989, 3: 283-288; Maksyutov A.Z., Eroshkin A.M., and Kulichkov V.A., Molekularnaya Biologiya 1987, 21: 39-47; Maksyutov, A. Z., and Zagrebelnaya, E. S., Computer Applications in the Biosciences 1993, 9: 291-297; and Maksyutov, A. Z., and Zagrebelnyi S.N., Molekularnaya Biologiya 1993, 27: 980-991, each of which is incorporated by reference in its entirety.<br>
Typically during the natural infection of an individual and selection of antigenic variants of an infectious agent there is too little time for formation of a sufficient amount of immune effectors capable of cross-reaction with normal human proteins to induce an autoimmune response. Induction of autoimmune processes usually takes a long time, and often requires the constant presence of the sensitizing antigen. Hence, the probability of autosensitization in an individual as a result of local similarity between variants of the hypervariable regions of the variable infectious agent and a human protein may be insignificant. On the other hand, the risk of induction of autoimmune conditions is increased by repeated use of a peptide vaccine for prophylaxis or therapy of an infectious disease. Therefore, during the design of polyepitope vaccine constructs, it should be taken into account that the long persistence of "time-conservative" variants of variable regions of the variable infectious agent can bring about autoimmune conditions, particularly with a vaccine construct containing adjuvants for increasing the immune response. Thus, the design of vaccine constructs on the basis of VCPLs requires comprehensive theoretical consideration of the local similarity between VCPL peptides and the whole set of human proteins.<br>
It can be advantageous to screen the sequences in a library for their potential to generate an autoimmune response. If a peptide in the library is highly similar to a peptide present in a human protein, an immune response could be generated against the human protein, with deleterious health effects for the person. A measure of the likelihood that a peptide sequence might generate an autoimmune response can be calculated by determining<br>
the antigenic similarity of the peptide sequence in the library to sequences found in human proteins.<br>
At the final stage of the method of designing a VCPL, the local similarity between the VCPL peptides and the whole set of human proteins can be analyzed. Candidate vaccines on the based on VCPLs can be screened for potentially autoimmunogenic determinants resulting from local similarity of VCPL peptides to human proteins. These potentially autoimmunogenic determinants can be removed from the library.<br>
The epitopes to be modeled in a VCPL can be chosen so as not to be autoimmunogenic. The method of identifying potentially immunopathogenic regions in proteins of an infectious agent on the basis of recognition of regions displaying close local similarity to human proteins provides a key to rational design of safe polyepitope vaccines against pathogenic microorganisms, because it allows elimination of epitopes with an immunopathogenic potential. In particular, antigenic determinants displaying similarity to human proteins that occur often and/or support important physiological functions of the body should be eliminated.<br>
To search for local similarity between a library sequence and human proteins, the SIM program can be used (Huang, X. and Miller, W., Adv. Appl. Math., 1991, 12, 337-357, which is incorporated by reference in its entirety). The sequences of human proteins are available in public databases, for instance SWISS-PROT (rl. 41, FEB-2003) and SP-TrEMBL (rl. 23, MAR-2003). The comparison can be carried out using the version of SIM available on the European Molecular Biology Laboratory (EMBL) server. The following parameters of the SIM program can be used: gap-open penalty 100, gap-extension penalty 20.<br>
The local similarity search may produce a large number of hits, many of which have a small degree of similarity between a peptide sequence from the library and a region of a human protein. It can be useful to filter the results of the similarity search to identify most relevant peptide-protein similarities. The filtering can be performed as follows. The twenty commonly occurring amino acids are divided into intersecting groups with physicochemical similarities: group 1 - Lys, Arg; group 2 - Asp, Glu; group 3 - Asn, Gin; group 4 - Ser, Thr; group 5 - Ala, He, Leu, Val; group 6 - Ala, Gly; group 7 - Leu, Phe; group 8 - Leu, Met, Val; and group 9 - Phe, Trp, Tyr.<br>
For each individual hit of local similarity between a library peptide and a human protein, the following values are calculated:<br>
index of similarity (Score determined by the SIM program); the percentage of matching amino acids (Match %); the number of matching amino acids (M); the number of mismatching amino acids (MM); the number of gaps (Gap); the number of conservative substitutions (Ccons) (that is, the number of substitutions within groups 1, 2, 3 &amp; 4, 5-9); the number of substitutions of charged amino acids (His, Lys, Arg, Asp, Glu) by one of opposite charge (C+_); the number of non-conservative substitutions or deletions of charged amino acids (C±); the number of non-conservative substitutions or deletions of polar amino acids (Asn, Gin, Ser, Thr) (Cpolar); and the number of non-conservative substitutions or deletions of aromatic amino acids (Phe, Trp, Tyr) (Carom)-<br>
Then an empirical algorithm tests if the following conditions are met:<br>
l)Index of similarity not less than 320 (Score &gt; 320).<br>
2)Number of matching amino acids in a given region not less than 6 (M &gt; 6).<br>
3)Percent of matches not less than 50% (Match % &gt; 50%).<br>
4)Number of matching amino acids and conservative substitutions in a given region not less than 8: M + Ccons ^ 8. However, if within a homologous region there is a subregion which comprises a string of 8 amino acids in a row, which are either conservative or matching then this region is taken as a whole regardless of the percent of match (Match %).<br>
5)(MM - CCons + Gap)/(M + Ccons) 
6)1.5*C+. + C± + 0.5*(Cpoiar + Carom) 
If all the above criteria are met, then the nature and/or function of the human protein can be considered. If the library sequence is similar to a sequence that occurs frequently in human proteins, or is similar to a sequence in a protein having an important physiological function, that sequence can be removed from the library.<br>
In certain circumstances, it can be desirable for the library to have a small number of sequences, such as fewer than 1,000 sequences, fewer than 500 sequences, or fewer than 250 sequences. The library is representative of the full antigenic diversity found in the naturally occurring sequences. Such a representative chimeric peptide library (RCPL) can be created by selecting sequences from a large library (e.g., a VCPL) such that the selected sequences are representative of the full antigenic diversity found in the naturally occurring sequences<br>
and non-natural sequences generated from the large library. The non-natural sequences can reflect potential sequences, for example, by including in a single sequence variants at different positions that have not been found together naturally.<br>
Because an RCPL has a smaller number of peptides than a VCPL, it is inherently less likely to induce an autoimmune response. The sequences chosen for the RCPL can be biased towards certain residues at certain positions, analogous to the residue frequencies in a VCPL. An RCPL can also be biased toward particular discrete sequences. In other words, if an RCPL includes 100 distinct peptide sequences, those peptides can be combined in any arbitrary ratio. Despite the much smaller number of sequences, the antigenic diversity of an RCPL can be as great as that of VCPL from which the RCPL is created. In this way, a mixture of a relatively small number of chemical species (i.e. peptide sequences) can produce an equally broad immune response as a mixture of a very large number of chemical species.<br>
RCPLs are sets of homologous peptides evenly covering the diverse variants of hypervariable regions of infectious agents. RCPLs are characterized by smaller numbers of peptides in the libraries (compared to VCPLs), which evenly cover the set of antigenic variants of hypervariable regions of variable infectious agents, including potential variants. RCPLs are designed to mimic the antigenic diversity of main variable protective determinants of an infectious agent. RCPLs allow production of a wide range of antibodies and cytotoxic T-lymphocytes (CTLs), whose combined specificity covers the set of antigenic variants of the variable infectious agent. Thus, RCPLs can induce a humoral and cellular response against the diversity of current and potential variants of variable infectious agents.<br>
To select the specified number, N, of peptides in the RCPL, a large number (e.g. &gt;1,000, &gt;10,000, &gt;100,000, &gt;1,000,000, or &gt;10,000,000) of peptides sequences are chosen at random from the whole diversity of variants of the mimicked hypervariable antigenically active region (e.g., a VCPL), accounting for the rates of occurrence of amino acids at particular positions of the mimicked region. The set of randomly chosen peptide sequences is arranged in the order of the relative rates of their occurrence in the whole diversity of the variants of the mimicked hypervariable antigenic region. The first 37-100% of the sequences from the set of randomly chosen peptides are used. The percentage used is determined by empirical considerations.<br>
The peptide sequence having amino acids with the greatest rates of occurrence at each position is taken as the first sequence of the RCPL. Alternatively, any peptide of the set of<br>
variants of the mimicked hypervariable antigenic region may be used as the first, or reference, sequence.<br>
Each subsequent peptide sequence added to the RCPL is chosen from the set of randomly chosen peptides such that the distance between the new sequence and all sequences previously added to the library is maximal or approaches the maximum. The distance between two peptides, RIJ, can be determined with the use of the antigenic similarity matrix (see, e.g., Table 1).RIJ is calculated according to:<br>
(Formula Removed)<br>
where 4 is the klh residue of peptide i,jk is the klh residue of peptide j, and / is the number of amino acid residues in each peptide of the library. The function Fcan be described by an antigenic similarity matrix:<br>
(Formula Removed)<br>
where the value of ASM(IK,JK is the matrix value for the pair of residues represented by jk and jk.<br>
For each peptide n from the randomly chosen peptides, the sum of distances Sn between the first N peptides selected (i.e., those already in the RCPL) and peptide n is calculated according to the formula:<br>
(Formula Removed)<br>
where Rin is calculated as desibed above. The peptide n that has the greatest value of Sn is added to the library. The selection of sequences from the large set of randomly chosen sequences continues until the RCPL has the desired number of sequences.<br>
The function F can alternatively be used simply to count the number of substitutions between two peptides. In this case, the value of F(ik,jK) is 1 when iK equals;ik (i.e. both peptides have the same residue at position k), and F(ik,jk) is 0 when ik and.jk are different. When selected in this way, the sequences of the RCPL should evenly cover the whole set of variants of the mimicked hypervariable antigenically active region.<br>
Each peptide included in the RCPL can be screened for autoimmunogenic properties as described above. If a peptide sequence is determined to have the potential to induce an autoimmune response, it can be excluded from the RCPL. Another candidate, virtually identical in the required properties, is instead chosen as the next peptide in the RCPL. The<br>
choice algorithm is designed so that the each new peptide added to the RCPL is chosen from a number of sequences of equal value (i.e. of equal distance from the previous peptide sequence in the library).<br>
A sufficient number of peptides chosen as described here can evenly cover the whole diversity of the antigenic variants of hypervariable epitopes of variable infectious agents. The greater is the number of peptides making up an RCPL, the "denser" the immune response that it can induce. Because the algorithm of choosing N peptides is incremental, the first N/2 peptides also evenly cover the whole diversity of the variants, although less densely.<br>
Libraries of representative chimeric peptides can be constructed as candidate vaccines. Taken together, the peptide sequences in the library cover the whole diversity of variants of hypervariable antigenically active regions of a variable infectious agent. The sequences are designed to mimic the genetic diversity of main variable protective determinants of infectious agents. Also, the mimicking peptides can be chimeric, because, although they mimic the heterogeneous set of variants, the sequences in the RCPL actually occur in few, if any, of the known and potential variants of epitope sequences.<br>
In general, peptide libraries can be synthesized manually or by using an automated synthesizer as described, e.g., in Lebl M. and Krchnak V. (1997) Solid-Phase Peptide Synthesis, Methods in Enzymology, 289, Academic Press, Minneapolis, Minnesota, which is incorporated by reference in its entirety. The synthesis can be a. combinatorial synthesis, in which mixtures of amino acid are coupled to the growing peptide, and a mixture of peptide sequences is formed. In a combinatorial synthesis, amino acids are added to a coupling reaction in proportions dictated by the desired ratio of residues at each position in the peptide, adjusted for the relative coupling rate for each amino acid. Alternatively, the library can be produced by parallel synthesis, in which each sequence is synthesized separately, and the library can be a group of individually pure peptide sequences. Such a library can be converted to a mixture by mixing the pure peptides.<br>
Automated combinatorial synthesis can be preferable when a large library is to be produced. Split synthesis can be preferable when a small library of individual sequences (e.g. an RCPL) is to be produced.<br>
The peptide libraries can be part of a multiple antigenic peptide (MAP). A multiple antigenic peptide includes a branched core from which several peptides extend. The branched core can be based on branching repeats of lysine. The MAP can include for<br>
example 4 or 8 peptide chains. The MAP can include a hydrophobic group, such as an alkyl chain derived from palmitic acid. The hydrophobic group can facilitate assembly of the MAP with liposomes or virus-like particles.<br>
The peptides can include the 20 common naturally occurring L-amino acids. The peptides can optionally include corresponding D-amino acids, or mixtures of D- and L-amino acids. Non-natural amino acids can likewise be included in the peptides.<br>
The peptide library can be produced by recombinant methods. A DNA construct encoding the peptides of the library can be introduced into a host organism capable of protein expression from the construct. Suitable host organisms can include bacteria such as E. coli, Lactobacillus, and Bifida, and plants, such as tobacco.<br>
The peptide libraries can be used in diagnostic applications, for example to determine the presence of antibodies, or to determine CTL function. For example, the libraries may be used an antigens for an enzyme-linked immunosorbent assay (ELIS A) to determine the presence of antibodies in a subject sample. In a solid-phase assay such as an ELISA, the peptides of the library are immobilized on a substrate (for example, in a polystyrene well). The substrate is then exposed to a test sample (e.g., serum from a subject) under conditions that allow specific peptide-antibody complex formation. Any unbound antibodies are then washed away. Antibodies associated with the substrate by virtue of a specific peptide-antibody interaction are then detected. Detection can include exposing the antibodies to a secondary antibody specific for immunoglobulins from the subject. The secondary antibody can be radiolabelled, or coupled to an enzyme such as horseradish peroxidase or alkaline phosphatase, that catalyzes the formation of a colored product.<br>
The diagnostic peptide library can be a broad spectrum diagnostic reagent, or a subtype specific reagent. If the library was designed to be antigenically similar to all known subtypes of a pathogen, then it can be used to detect that a subject has been infected by the pathogen, regardless of what subtype the subject is infected with. If instead the library was designed to be antigenically similar to only one subtype, then it can be used to detect if a subject was infected by the same subtype. A group of subtype specific libraries can be used together to determine which of several subtypes a subject was infected by.<br>
Vaccines based on antigenic peptides are known. See, for example, U.S. Patent Nos. 4,601,903, 4,599,230, and 4,599,231, each of which is incorporated by reference in its entirety. Vaccines may be prepared as injectables, as liquid solutions or emulsions. The<br>
peptides may be mixed with pharmaceutically acceptable excipients that are compatible with the peptides. Excipients may include water, saline, dextrose, glycerol, ethanol, and combinations thereof. The vaccine may further contain auxiliary substances such as wetting or emulsifying agents, pH buffering agents, or adjuvants to enhance the effectiveness of the vaccines. Methods of achieving adjuvant effect for the vaccine include the use of agents such as aluminum hydroxide or phosphate (alum), commonly used as 0.05 to 0.1 percent solution in phosphate buffered saline. Vaccines may be administered parenterally, by injection subcutaneously or intramuscularly. Alternatively, other modes of administration including suppositories and oral formulations may be desirable. For suppositories, binders and carriers may include, for example, polyalkylene glycols or triglycerides. Oral formulations may include normally employed incipients such as, for example, pharmaceutical grades of saccharine, cellulose and magnesium carbonate. These compositions take the form of solutions, suspensions, tablets, pills, capsules, sustained release formulations or powders.<br>
The vaccines are administered in a manner compatible with the dosage formulation, and in such amount as is therapeutically effective, protective and immunogenic. The quantity to be administered depends on the subject to be treated, including, for example, the capacity of the individual's immune system to synthesize antibodies, and to produce a cell-mediated immune response. Precise amounts of active ingredient required to be administered depend on the judgment of the practitioner. However, suitable dosage ranges are readily determinable by one skilled in the art and may be of the order of micrograms of the peptides. Suitable regimes for initial administration and booster doses are also variable, but may include an initial administration followed by subsequent administrations. The dosage of the vaccine may also depend on the route of administration and will vary according to the size of the host.<br>
Nucleic acid molecules encoding the peptides of the library may also be used directly for immunization by administration of the nucleic acid molecules directly, for example by injection, or by first constructing a live vector, such as Salmonella, BCG, adenovirus, poxvirus, vaccinia or poliovirus, and administering the vector.<br>
The vaccine can be a prophylactic vaccine that induces a protective immune response in a subject, before the subject is exposed to the pathogen. In general, a prophylactic vaccine includes a broad-spectrum peptide library. In some cases, a prophylactic vaccine can includea subtype-specific peptide library, for instance, when the subject to be vaccinated is likely to be exposed to only one subtype of the pathogen. The vaccine can be a therapeutic vaccine, designed to induce or enhance an immune response in a subject that is infected by the pathogen. If the subject is known to be infected with a particular subtype of pathogen, the therapeutic vaccine can be one designed to induce an immune response against that subtype.<br>
Examples<br>
Peptide libraries were designed based on each of the variable regions V1-V5 of HIV gp120. The sequences used to generate the libraries were from HIV subtype A, B, C, D, F, G, or sequences from all these subtypes were used together. A peptide library generated using sequences from only one subtype can be used as a subtype-specific vaccine, whereas a library generated from multiple subtypes can be used as a broad spectrum vaccine. Each library was initially designed to include fewer than 200,000,000 sequences. From the initial library, smaller libraries were designed to include fewer than 500,000 sequences and fewer than 50,000 sequences. Smaller libraries are designed by removing infrequently occurring residues from the larger library. In Tables 3-37 below, the identities and relative frequencies of residues are given. For example, in Table 3, residue X11 can be S, N or G in the largest library; S or N in the next smaller library, and S in the smallest library. The numbers represent the relative frequencies of each residue at that position. In the largest library, position X11 is occupied by S, N or G in an 80:13:7 ratio; in the next smaller library, X11 is occupied by S or N in an 80:13 ratio; and in the smallest library, every peptide has S at X11.<br>
Chimeric Peptide Libraries - V1region of HIV gp120<br>
Peptide libraries based on the V1 region of HIV gp120 from subtypes A, B, C, D, F, and G were designed. The libraries had the structure:<br>
C-T-X1-X2-X3-X4-X5-X6-X7-X8-X9-X10-X11-X12-X13-X14-X15-X16-X17-E-X18-K-N where X'-X18 are defined according to Table 3.<br><br>
Table 3<br>
(Table Removed)<br>
Pep tide libraries based on the V1 region of HIV gp120 from subtype A were designed. The libraries had the structure:<br>
C-T-X1-X2-X3-X4-X5-X6-X7-X8-X9-X10-X11-X12-X13-X14-X15-X16-E-X17-K-N where X'-X17 are defined according to Table 4.<br>
Table 4<br>
(Table Removed)<br>
Peptide libraries based on the V1 region of HIV gp120 from subtype B were designed. The libraries had the structure:<br>
C-T-X1-X2-X3-N-X4-X5-X6-X7-X8-X9-X10-X11-X12-X13-X14-X15-E-X16-X17-E-X18-K-N where X'-X18 are defined according to Table 5.<br>
Table 5<br>
(Table Removed)<br>
Peptide libraries based on the VI region of HIV gp!20 from subtype C were designed. The libraries had the structure:<br>
C-X1-X2-X3-X4-X5-X6-X7-X8-X9-X10-X11-X12-X13-X14-X15-X16-X17-E-X18-K-N where X'-X18 are defined according to Table 6.<br>
Table 6<br>
(Table Removed)<br>
Peptide libraries based on the V1 region of HIV gp120 from subtype D were designed. The libraries had the structure:<br>
C-X1-X2-X3-X4-X5-X6-X7-T-X8-X9-X10-X11-X12-X13-X14-X15-X16-E-X17-K-N<br>
where X1-X17   are defined according to Table 7. Table 7<br>
(Table Removed)<br>
Peptide libraries based on the V1 region of HIV gp120 from subtype F were designed. The libraries had the structure:<br>
C-X1-X2-X3-X4-X5-X6-T-X7-X8-X9-X10-X11-X12-X13-T-L-K-X14-X15-X16-X17-X18-X19-X20-N whereX1-X20 are defined according to Table 8.<br>
Table 8<br>
(Table Removed)<br>
Peptide libraries based on the VI region of HIV gp120 from subtype G were designed. The libraries had the structure:<br>
C-T-X1-X2-X3-X4-X5-X6-X7-X8-X9-X10-X11-X12-X13-X14-X15-X16-X17-N<br><br><br>
where X1 -X17   are defined according to Table 9.<br>
Table 9<br>
(Table Removed)<br>
Chimeric Peptide Libraries - V2 region of HIV gp120<br>
Pep tide libraries based on the V2 region of HIV gp120 from subtypes A, B, C, D, F, and G were designed. The libraries had the structure:<br>
S-X1-N-X2-T-T-X3-X4-R-X5-K-X6-X7-X8-X9-X10-X11-L-F-Y-X12-L-D-V-V-X13-I-X14-X15-X16-X17-X18-<br>
X19-Y-X20-L-X21-X22-C where X'-X22 are defined according to Table 10.<br>
Table 10<br>
(Table Removed)<br>
Peptide libraries based on the V2 region of HIV gp120 from subtype A were designed. The libraries had the structure:<br>
S-X1-X2-X3-T-T-X4-L-R-D-K-K-X5-X6-V-X7-X8-L-F-Y-R-L-D-V-V-X9-I-X10-X11-X12-X13-X14-X15-X16-X17-X18-Y-R-L-I-X19-C<br>
where X'-X19 are defined according to Table 11.<br>
Table 11<br>
(Table Removed)<br>
Pep tide libraries based on the V2 region of HIV gp120 from subtype B were designed. The libraries had the structure:<br>
S-F-X1-I-T-T-X2-X3-X4-X5-K-X6-X7-X8-X9-X10-A-X11-F-X12-X13-L-D-V-V-X14-I-X15-X16-X17-X18-T-<br>
X19-Y-X20-L-X21-X22-C where X'-X22 are defined according to Table 12.<br>
Table 12<br>
(Table Removed)<br>
Peptide libraries based on the V2 region of HIV gp120 from subtype C were designed. The libraries had the structure:<br>
S-F-N-X1-T-T-E-L-R-D-K-X2-X3-X4-X5-X6-A-L-F-Y-R-X7-D-I-V-X8-L-X9-X10-X11-X12-X13-X14-Y-X15-L-I-X16-C where X'-X16 are defined according to Table 13.<br>
Table 13<br>
(Table Removed)<br>
Peptide libraries based on the V2 region of HIV gp!20 from subtype D were designed. The libraries had the structure:<br>
S-F-N-I-T-T-X1-V-X2-D-K-X3-X4-X5-X6-X7-A-L-X8-Y-R-X9-D-V-V-X10-X11-X12-X13-X14-X15-X16-<br>
X17-X18-Y-R-L-I-X19-C where X'-X19 are defined according to Table 14.<br>
Table 14<br>
(Table Removed)Peptide libraries based on the V2 region of HIV gp120 from subtype F were designed. The libraries had the structure:<br>
S-F-N-X1-T-X2-E-V-R-D-K-X3-X4-K-X5-X6-A-L-F-Y-R-L-D-I-X7-X8-I-X9-X10-X11-X12-X13-X14-X15-Y-R-L-X16-X17-X18<br>
where X'-X18 are defined according to Table 15. Table 15<br>
(Table Removed)<br>
Peptide libraries based on the V2 region of HIV gp120 from subtype G were designed. The libraries had the structure:<br>
S-F-N-X1-T-X2-X3-X4-X5-D-K-X6-K-X7-E-Y-A-L-F-Y-R-X8-D-V-V-X9-I-X10-X11-X12-X13-X14-X15-Y-<br>
Xlb-L-X17-X18-C<br>
where X'-X18 are defined according to Table 16. Table 16<br>
(Table Removed)Chimeric Peptide Libraries - V3 region of HIV gp120<br>
Peptide libraries based on the V3 region of HIV gp120 from subtypes A, B, C, D, F, nd G were designed. The libraries had the structure:<br>
X1-C-X2-R-P-X3-N-N-T-R-X4-X5-X6-X7-X8-G-X9-G-X10-X11-X12-X13-X14-T-G-X15- I-X16-G-X17- I-R -X18-A-X19-C-X20 where X'-X20 are defined according to Table 17. Table 17<br>
(Table Removed)<br>
Peptide libraries based on the V3 region of HIV gp120 from subtype A were designed. The libraries had the structure:<br>
X1-C-X2-R-P-X3-N-N-X4-R-X5-X6-V-X7-I-G-P-G-X8-X9-F-X10-X11-X12-X13-<br>
X14-I-X15-G-X16-I-R-X17-A-X18-C-X19<br>
where X'-X19 are defined according to Table 18. A dash "-" indicates no residue at that position. For example, if X12 is represeented by a dash, then no residue occupies X12, and the residue of X1' is bound directly to the residue of X  .<br>
Table 18<br>
(Table Removed)<br>
Peptide libraries based on the V3 region of HIV gp120 from subtype B were designed. The libraries had the structure:<br>
X1-C-X2-R-P-X3-N-N-T-R-K-X4-I-X5-X6-G-X7-G-X8-X9-X10-X11-X12-T-X13-X14-I-X15-G-X16-I-R-X17-A-X18-C-X19 where X'-X19 are defined according to Table 19. Table 19<br>
(Table Removed)<br>
Peptide libraries based on the V3 region of HIV gp120 from subtype C were designed. The library had the structure:X1-C-X2-R-P-X3-N-N-T-R-X4-X5-X6-X7-I-G-P-G-Q-X8-F-X9-X10-T-X11-X12-I-X13-G-X14-I-R-X15-A-X16-C-X17<br>
r!7<br>
where X -X   are defined according to Table 20. Table 20<br>
(Table Removed)<br>
Pep tide libraries based on the V3 region of HIV gp120 from subtype D were designed. The libraries had the structure:<br>
X1-C-X2-R-P-X3-X4-X5-X6-R-X7-X8-X9-X10-I-G-X11-G-X12-X13-X14-X15-X16-T-X17-X18-G-X19-I-X20-X21-A-X22-C-X23 where X'-X23 are defined according to Table 21. Table 21<br>
(Table Removed)<br>
Pep tide libraries based on the V3 region of HIV gp120 from subtype F were designed. The libraries had the structure:<br>
X1-C-T-R-P-X2-N-N-X3-R-K-X4-I-X5-L-G-P-G-X6-X7-X8-X9-X10-X11-X12-X13-I-X14-G-X15-I-R-X16-A-X17-C-X18 where X'-X18 are defined according to Table 22. Table 22<br>
(Table Removed)<br>
Pep tide libraries based on the V3 region of HIV gp120 from subtype G were designed. The libraries had the structure:<br>
X1-C-X2-R-P-X3-N-N-T-R-K-S-X4-X5-X6-G-X7-X8-X9-X10-X11-X12-X13-T-X14-<br>
I-X15-G-X16-I-R-X17-A-X18-C-X19 where X'-X19 are defined according to Table 23.Table 23<br>
(Table Removed)<br>
Chimeric Peptide Libraries - V4 region of HIV gp120<br>
Pep tide libraries based on the V4 region of HIV gp120 from subtypes A, B, C, D, F and G were designed. The libraries had the structure:<br>
C-N-T-T-X1-L-F-N-S-T-W-X2-X3-X4-X5-W-X6-T-X7-X8-X9-X10-X11-X12-X13-<br>
V14  v15 V16 V17 I  V18 I    V19 f*<br>
x -x -A -x -i-A -L-A -LI where X'-X19 are defined according to Table 24.<br>
Table 24<br>
(Table Removed)<br>
Peptide libraries based on the V4 region of HIV gp!20 from subtype A were designed. The libraries had the structure:<br>
C-X1-T-S-X2-L-F-N-X3-T-W-X4-X5-X6-X7-X8-X9-X10-X11-T-X12-X13-X14-X15-X16-I-X17-L-X18-C<br>
where X'-X18 are defined according to Table 25. Table 25<br>
(Table Removed)Peptide libraries based on the V4 region of HIV gp!20 from subtype B were designed. The libraries had the structure:<br>
C-N-T-T-X1-L-F-N-S-T-W-X2-X3-X4-X5-X6-X7-T-X8-X9-X10-X11-X12-X13-X14-G-X15-X16-X17-I-X18-L-X19-C where X'-X19 are defined according to Table 26. Table 26<br>
(Table Removed)<br>
Peptide libraries based on the V4 region of HIV gp!20 from subtype C were designed. The libraries had the structure:<br>
C-N-T-S-X1-L-F-N-X2-T-Y-X3-X4-X5-X6-X7-X8-X9-X10-X11-X12-X13-X14-X15-I-<br>
X16-L-X17-C where X'-X17 are defined according to Table 27.<br>
Table 27<br>
(Table Removed)<br>
Peptide libraries based on the V4 region of HIV gp!20 from subtype D were designed. The libraries had the structure:<br>
C-X1-T-S-X2-L-F-N-X3-X4-X5-X6-X7-X8-X9-X10-X11-X12-X13-X14-I-X15-C<br>
r!5<br>
where X -X   are defined according to Table 28 Table 28<br>
(Table Removed)<br>
Peptide libraries based on the V4 region of HIV gp120 from subtype F were designed. The libraries had the structure:<br>
X1-X2-T-X3-X4-L-F-X5-X6-X7-X8-X9-X10-X11-X12-X13-I-X14-L-X15-C where X'-X15 are defined according to Table 29.<br>
Table 29<br>
(Table Removed)<br>
Pep tide libraries based on the V4 region of HIV gp!20 from subtype G were designed. The libraries had the structure:<br>
C-N-T-S-X1-L-F-X2-X3-X4-X5-X6-X7-X8-X9-X10-X11-X12-X13-X14-I-X15-L-X16-C where X'-X16 are defined according to Table 30.<br>
Table 30<br>
(Table Removed)<br>
Chimeric Peptide Libraries - V5 region of HIV gp!20<br>
Pep tide libraries based on the V5 region of HIV gp!20 from subtypes A, B, C, D, F, and G were designed. The libraries had the structure:<br>
X1-X2-X3-X4-X5-X6-E-X7-F-R-P-X8 where X'-X8 are defined according to Table 31. Table 31<br>
(Table Removed)<br>
Peptide libraries based on the V5 region of HIV gp!20 from subtype A were designed. The libraries had the structure:<br>
X1-X2-X3-X4-X5-X6-X7-X8-X9-R-P-X10 where X'-X10 are defined according to Table 32.Table 32<br>
(Table Removed)<br>
Pep tide libraries based on the V5 region of HIV gp!20 from subtype B were designed. The libraries had the structure:<br>
X1-X2-X3-X4-X5-X6-E-X7-F-R-P-G where X'-X7 are defined according to Table 33.<br>
Table 33<br>
(Table Removed)<br>
Peptide libraries based on the V5 region of HIV gp!20 from subtype C were designed. The libraries had the structure: X1-X2-X3-X4-X5-E-X6-X7-X8-X9-X10<br>
,10<br>
where X -X   are defined according to Table 34. <br>
Table 34<br><br>
(Table Removed)<br>
Peptide libraries based on the V5 region of HIV gp!20 from subtype D were designed. The libraries had the structure:<br>
X1-X2-X3-X4-X5-X6-X7-E-X8-X9-R-X10-X11 where X'-X11 are defined according to Table 35.<br>
Table 35<br>
(Table Removed)<br>
Peptide libraries based on the V5 region of HIV gp!20 from subtype F were designed. The libraries had the structure:<br>
X1-X2-X3-X4-X5-X6-X7-F-R-P-X8 where X'-X8 are defined according to Table 36.<br>
Table 36<br>
(Table Removed)<br>
Peptide libraries based on the V5 region of HIV gp120 from subtype G were designed. The libraries had the structure:<br>
X1-X2-X3-X4-X5-X6-X7-X8-X9-X10-X11-X12 where X'-X12 are defined according to Table 37.<br>
Table 37<br>
(Table Removed)<br>
Synthesis and Antigenic Properties of a VCPL<br>
A peptide library mimicking the naturally occurring diversity of sequences in the main hypervariable V3 region of the HIV-1 (subtype B) gp120 envelope protein was designed. The identity and frequency of the amino acids in the library were chosen based on sequence alignments and antigenie similarity. HIV-1 sequences are available from the HIV Sequence Database (hiv-web.lanl.gov). The peptide library had the structure:<br>
N-N-N-T-R-K-X1-I-X2-X3-X4-X5-G-X6-X7-X8-Y-X9-T-G-X10-I-I-G-X11-I-R-Q where X1- X11 were defined according to Table 38.<br>
Table 38<br><br>
(Table Removed)<br>
The peptides of the library were synthesized on Applied Biosystems 430A and Vega Coupler C250 automatic synthesizers with the use of the Boc/Bzl strategy of peptide chain elongation. A styrene copolymer with 1% divinylbenzene was used as the polymer support, and a phenacylamidomethyl (PAM) group served as the anchor group (Sparrow, J.T. J. Org. Chem. 1976, 41:1350-1353, which is incorporated by reference in its entirety). Condensation was done with 1-hydroxybenzotriazole esters prepared directly before use by mixing a 3-fold molar excess relative to the polymers of the Boc product of amino acid, 1-hydroxybenzotriazole and diisopropylcarbodiimide. The completeness of the condensation was monitored (Kaiser, E. et al. Anal. Biochem. 1970, 34:395-398, which is incorporated by reference in its entirety). In the event of a positive ninhydrin test after neutralization of the peptidyl polymer, condensation was repeated. If the test was negative, then the temporary Boc protective groups were removed with undiluted trifluoroacetic acid, and neutralization was performed by adding N,N'-diisopropylethylamine directly to the reaction mixture (Schnolzer, M. et al. Int. J. Pept. Prot. Res. 1992, 40: 180-183, which is incorporated by reference in its entirety). After removal of the Boc groups the following amino acid residue was attached. Permanent protective groups used for side chains were: 2-chlorobenzyloxycarbonyl groups for lysine, dichlorobenzyl for tyrosine, the corresponding benzyl esters for aspartic acid, glutamic acid, serine and threonine, formyl for tryptophan, tosyl for arginine, and methionine was introduced in the form of the corresponding sulfoxide. Upon completion of assembly of the protective polypeptide chain, the end products were deblocked with a one-time cleavage from the polymer with the aid of anhydrous hydrogen fluoride in the presence of scavengers (Tarn, J.P. et al. J. Am. Chem. Soc. 1983, 105: 6442-6444, which is incorporated by reference in its entirety).<br>
During the synthesis of peptide libraries a mixture of protected amino acid residues was added in the condensation stage in variable positions relative to the peptidyl polymer in a certain quantitative relationship, which provided a given incorporation of different amino acid residues to the growing polypeptide chain. Because different amino acids have different rates of amino group acylation, a number of model compounds were synthesized and reaction rate coefficients were derived. The Boc products of the amino acids which were introduced into the mixture, were used in ratios adjusted for the reaction rate coefficients, to provide peptide products having the desired ratios (as shown in Table 38) in the variable positions.<br>
The peptide library was purified by gel permeation chromatography using Sepharose G-50 and 50% acetic acid as the eluent. The synthesized compounds were characterized by analytical single-photon high-performance liquid chromatography (Gilson chromatograph, France; Xterra RP18 column, 125 A, 3.9 x 150 mm, 5 ^im, flow rate 1 mL/min, eluent 0.1% trifluoroacetic acid, 10-40% and 20-40% acetonitrile gradient in 16 min for pentarphin and cyclopentarphin, respectively), and by amino acid analysis (hydrolysis 6N HC1, 24 hours, 110 °C; LKB amino acid analyzer, Alpha Plus, Sweden). The purity of the end products according to analytic HPLC was not less than 95%. The amino acid composition of peptide hydrolysates (6N HC1, 120 °C, 24 hours, LKB 4151 analyzer, Alpha Plus) corresponded to the theoretical.<br>
BALB/c line mice ages 5-6 weeks were immunized 3 times intraperitoneally with the peptide library diluted in a phosphate buffer solution with Complete Freund's Adjuvant (CFA). The mice were administered 0.2 mL peptide solution in a dose of 50 fig/head. The second and third immunization were conducted 2 and 4 weeks after the first immunization, respectively. A control group of animals were administered a peptide solution mimicking the antigenic portion of protein E of the tick encephalitis virus (peptide sequence: KRDQSDRGWGNHAGLFGKGSIVT) according to the same scheme and in the same doses. Ten days after the last immunization a blood sample was taken from the retroorbital vein of the animals for determination of the titer of specific antibodies in the serum.<br>
The following antigens were immobilized in separate wells of microtiter plates for immunoenzyme analysis (Medpolimer, Moscow): the chimeric-peptide library, and each of 15 synthetic peptides representing the amino acid sequences of the V3 region of gp!20 from different HIV-1 subtypes. The 15 synthetic peptides represent the consensus sequences of<br>
appropriate subtypes. Where several peptides are present from one subtype, they represent different variants.<br>
The antigens were dissolved in 0.05 M carbonate-bicarbonate buffer solution, pH 9.6 to a final concentration of 10 µg/mL, placed in each well of the microtiter plate in 0.1 mL amounts and incubated overnight at a temperature of 37 °C. The wells were washed 3 times with phosphate buffer solution with Tween 20, and non-specific binding was blocked with a 0.5% solution of bovine serum albumin, fraction 5 (Sigma). Mouse blood sera samples were serially diluted two-fold in a Tris-HCl buffer solution, and placed in 0.1 mL amounts into the wells with immobilized antigen and incubated at 37 °C for 30 min. After incubation the wells were washed 5 times with a phosphate buffer solution. The antigen-antibody complexes were determined by adding 0.1 mL quantities of horseradish peroxidase-conjugated goat antibodies against mouse IgG (Sigma) in phosphate buffer solution. Following incubation at 37 °C for 30 min, a substrate mixture containing orthophenylene diamine was added. The enzyme reaction was halted by the addition of a sulfuric acid solution, and the optical density at 492 nm was measured using a Multiscan EX microplate photometer. The results of titer determination of the antigen-specific antibodies in the mouse blood sera are presented in Table 39.<br>
Immunization of mice with the chimeric-peptide library caused formation of serum antibodies specific both for the chimeric-peptide library itself (the immunizing antigens) and for peptides forming antigenic determinants in the V3 region of some known HIV-1 subtypes. In particular, the antibody liters for subtype B peptides were higher than for other subtypes, as would be expected for a peptide library design to mimic the diversity of subtype B and not other subtypes. The chimeric peptide library was immunogenic, and induced formation of antibodies that interact with antigenic determinants of a broad spectrum of HIV -1 variants.<br>
Table 39<br><br>
(Table Removed)<br>
The peptide library described above was tested as a diagnostic reagent in solid-phase immunoenzyme analysis (i.e., ELIS A) of serum samples from 50 HIV-infected patients and 30 healthy donors for the purpose of determining specific antibodies. Preliminarily sera of HIV-infected patients were analyzed with the aid of a number of commercial test systems. In addition, the peptide library was assessed for its ability to detect antibodies to HIV-1 with the aid of standard sera panels anti-HIV-1 series 010 and series 007.<br>
The chimeric-peptide library was dissolved in 0.05 M carbonate-bicarbonate buffer solution, pH 9.6 to a final concentration of 1.5 micrograms/mL. The solutions were added in 0.1 mL quantities to the wells of the microtiter plates and incubated at room temperature overnight, to immobilize the peptides in the wells.  Then the wells were washed 3-5 times with a solution of FSB-T (phosphate-salt buffer solution containing Tween 20). Henceforth the method completely corresponds to the instructions for the use of the immunoenzyme test system for detecting antibodies to HIV-1 produced by DGUEPP "Vektor-BiAlgam" (Kol'tsovo, Novosibirsk Oblast).<br>
Blood sera samples were diluted 20 fold in an FSB-T solution containing 0.2% casein and were placed in wells of the microtiter plate in 0.1 mL amounts and incubated at 37 °C for 30 min, to bind antibodies to the immobilized peptides. Then the wells were washed 5-7<br>
times with FSB-T solution. To the wells were added 0.1 mL amounts of a monoclonal antibody to IgG conjugated to horseradish peroxidase in a dilution of 1:20 in FSB-T containing 0.2% casein. The microtiter plate was incubated at 37 °C for 30 min, then washed 5-7 times with an FSB-T solution. Next, 0.1 mL amounts of a solution of substrate mixture of 0.05% TMB and 0.005% hydrogen peroxide in 0.05 M phosphate-citrate buffer solution, pH 5.0 was added to the wells. The microtiter plate was kept at room temperature (18-22° C) for 15-20 min in the dark. The reaction was stopped by addition of 0.05 mL amounts of 2 M sulfuric acid solution to the wells. Calculation of the results is accomplished by measurement of the optical density of the samples on a Multiscan-type automatic spectrophotometer at a wavelength of 450 nm.<br>
As a control for ruling out non-specific results in setting up the reaction a substrate mixture (BCP + substrate) is used, along with a control conjugate (BCP + conjugate + substrate), and a control serum which does not contain antibodies to HIV according to data from testing in Russian and foreign third generation screening test systems.<br>
The test sera are judged to have antibodies to HFV-1 if the values of the optical density (OD) of the corresponding investigative solution in the wells with the investigative sera exceed the critical level ODcrit, calculated according to the formula:<br>
ODcrit = OD(C')mean + 0.2 <br>
where OD(C')mean is the mean optical density for negative control sera.<br>
The results of the study of the anti-HFV-1 standard panel sera series 010 and 007 with the use of the chimeric-peptide library showed an 81% sensitivity and 93% specificity. In addition, a study was undertaken of the effectiveness of the chimeric-peptide library for detection of specific antibodies to HIV-1 in blood sera samples with the use of 50 sera from HFV-infected patients and 30 sera from healthy donors. The results of the tests showed more than 80% detection of positive samples containing specific antibodies to HIV-1 antigens on the basis of results with commercial third generation test systems.<br>
To test the immunogenicity of the chimeric peptide library in different antigen forms, the library was synthesized as linear peptides, a multiple antigen peptide of 4 linear peptides (MAP4), or multiple antigen peptide of 8 linear peptides (MAPS). Rabbits were immunized subcutaneously with 300 micrograms of the constructions described above mixed with either complete Freund's adjuvant (first immunization) or incomplete Freund's adjuvant (second and third immunizations) at 12 day intervals. At day 12 after the third immunization, the<br>
rabbits were bled and all blood sera were tested by ELISA against the linear, MAP4 and MAPS antigens. Table 40 illustrates that immunization of rabbits with the HIV-1 V3 library in different antigen forms induces a strong humoral immune response to the highly variable V3 loop sequences from chimeric peptide library.<br>
Table 40<br>
(Table Removed)<br>
The immunogenicity of the MAP4 chimeric peptide library in different formulations was tested. The library was formulated in Freund's adjuvant, artificial virus like particles (VLP) or in phosphate buffered solution (PBS). In addition, the MAP4 construction was modified with palmitic acid (MAP4-P) and formulated in liposomes or microemulsion. The liposomes included 100 micrograms of MAP4-P and 250 micrograms of dsRNA, and the microemulsion included 100 micrograms of MAP4-P and 100 micrograms of dsRNA.<br>
The VLPs contain a DNA molecule covered with polypeptides carrying the MAP4 peptide library. The target polypeptides are exposed on the surface of the particle and are attached to DNA via spermidine-polygluquine conjugates. The dimensions of the particle will allow the target antigens to conjugate on the surface in copy numbers ranging from one to several hundred. The core of the VLP can contain DNA segments as large as 10,000 bp. See, for example, Lebedev, L. R. et al. (2000) Molecular genetics, microbiology and virology <russian lebedev l. r. et al. molecular biology and biotechnology sizov a. each of which is incorporated by reference in its entirety.></russian>
The liposomes forms were prepared with the MAP4-Plibrary. The palmitic acid modification increases the lipophilicity of the peptides. The liposomal compositions of the peptide were made by solubilization-injection. The size of the bulk of liposomes was within 200 nm.<br>
The microemulsions were also made with the MAP4-P peptide construct and contained (in 1 mL) 100 micrograms of peptide, 250 micrograms of ridostin, and 800<br>
microgams of the commercial (ICN) cationic amphiphile dimethyldioctadecyl ammonium bromide (DDAB).<br>
Mice (five mice in each group) were immunized intraperitoneally with the constructions described above three times at 12 days intervals. The data in Table 41 indicate the high diversity of individual immune response in conventional mice, but nevertheless strong humoral immune response was detected for all systems after 3rd immunization, except for the peptide library solution in PBS. The antigen formulation of MAP4-P in liposomes induced high antibody levels after the second immunization.<br>
Table 41<br><br>
(Table Removed)<br>
To determine the polarization of the immune response of mice immunized with the MAP4 library in different delivery systems described above, on day 10 after the 3rd immunization, splenocytes were distinguished. The splenocytes were stimulated in vitro with the library of linear peptides. A number of splenocytes produced IFN-gamma or IL-4 cytokines, markers of Thl or Th2 immune response, respectively, as measured by ELISPOT. The number of cells that produced one of two cytokines was noted, after stimulation or without stimulation (Table 42). The results presented in Table 42 show that immunization of mice with a solution of MAP4 peptide library with Freund's Adjuvant does not result in a major difference (significance level 0.05) in the number of stimulated cells from the control measurement (without stimulation), which confirms the absence of induction of the T-helper<br>
response with immunization by antigens in a mixture with Freund's Adjuvant. The highest number of cells producing IFN-gamma after stimulation in vitro using the linear peptide library was noted in mice immunized with a microemulsion containing the MAP4 library. The immunization using VLP resulted in the formation of a less pronounced immune response of the Thl type. Comparison of the number of cells producing IL-4 and IFN-gamma reveals that immunization of mice with the MAP4 library in VLPs results in a balanced T-helper response, i.e. the number of cells producing IFN-gamma and IL-4 does not significantly differ. On the other hand, immunization of mice with the same antigens in a microemulsion results in a significant predominance of the Thl type immune response after stimulation with the library of linear peptides. Thus, the above example shows the possibility of inducing a strong T-helper response using the MAP4 library of chimeric peptides.<br>
Table 42<br>
(Table Removed)<br>
VLP-artufucual virus-like particlesFreund's adjuvant/incomplete Freund's adjuvant<br>
An RCPL having 30 peptides, each 28 residues in length, that models the antigenic diversity of HIV-1 subtype B variants was synthesized in the form of MAP4. The antigenic<br>
properties of the RCPL was compared to a VCPL which also modeled the antigenic diversity of HIV-1 subtype B variants.<br>
RCPL sequences (a period '.' indicates that the residue unchanged from the first sequence listed):<br>
(Table Removed)<br>
A rabbit was immunized 3 times with the MAP4 VCPL and CFA (first immunization) or IFA (second immunization). The antibody titer of the sera was tested by ELISA using either the immunizing antigen, or the MAP4 RCPL as the immobilized peptide. The titer of rabbit serum in ELISA with MAP4 VCPL on solid phase was 1:204,800, and with MAP4 RCPL on solid phase was 1:102,400.<br>
A number of embodiments have been described. Nevertheless, it will be understood that various modifications may be made. For example, chimeric peptide libraries can be designed to mimic the antigenic diversity of other HIV proteins, or of proteins from other variable pathogens. Accordingly, other embodiments are within the scope of the following claims.<br><br><br><br>
WHAT IS CLAIMED IS:<br>
1.	A method of manufacturing a family of antigenic peptides comprising:<br>
locating a plurality of variable positions in a region of a pathogen protein;<br>
choosing a peptide sequence of the pathogen protein including the plurality of<br>
variable positions;<br>
selecting one or more substitute amino acid residues for one of the variable positions based on antigenic similarity to amino acid residues naturally occurring at the variable position of the pathogen protein; and<br>
preparing a family of antigenic peptides based on the peptide sequence and including the substitute amino acid residues.<br>
2.	The method of claim 1, wherein selecting includes determining the antigenic<br>
similarity using an antigenic similarity matrix.<br>
3.	The method of claim 1, further comprising assigning a frequency to each<br>
substitute amino acid residue in the family of antigenic peptides.<br>
4.	The method of claim 3, wherein preparing includes weighting the substitute<br>
amino acid residues in the family of antigenic peptides based on the assigned frequency.<br>
5.	The method of claim 3, wherein assigning further includes considering the<br>
frequency with which the variations naturally occur.<br>
6.	The method of claim 1, wherein the pathogen protein includes a hypervariable<br>
region.<br>
7.	The method of claim 1, wherein the pathogen protein is associated with a<br>
virus.<br>
8.	The method of claim 7, wherein the virus is HIV.<br>
9.	The method of claim 1, wherein the pathogen protein is HIV gp!20.The method of claim 9, wherein the region is selected from the group<br>
consisting of the VI, V2, V3, V4, and V5 regions.<br>
10.	The method of claim 7, wherein the virus is hepatitis B virus or hepatitis C<br>
virus.<br>
11.	The method of claim 7, wherein the virus is an influenza virus.<br>
12.	The method of claim 7, wherein the virus a dengue virus.<br>
13.	The method of claim 1, wherein the pathogen protein is associated with a<br>
malaria pathogen.<br>
14.	The method of claim 1, wherein the pathogen protein is associated with a<br>
tuberculosis pathogen.<br>
15.	The method of claim 1, further comprising determining an antigenic similarity<br>
between a peptide of the family and a region of a human protein.<br>
16.	The method of claim 16, further comprising removing a peptide from the<br>
family of antigenic peptides before preparing the family if the determined antigenic similarity<br>
between the peptide of the family and the region of a human protein exceeds a predetermined<br>
threshold.<br>
17.	The method of claim 1, wherein the family of antigenic peptides includes<br>
members, such that the members taken together have antigenic similarity to each naturally<br>
occurring sequence of the region of the pathogen protein.<br>
18.	The method of claim 1, wherein the family of antigenic peptides includes<br>
members, such that the members taken together have antigenic similarity to a non-naturally<br>
occurring sequence of the region of the pathogen protein.<br>
19.	The method of claim 1, further comprising identifying peptide sequences of<br>
the family, the identified peptide sequences being representative of the sequence diversity of<br>
the entire family.The method of claim 20, wherein fewer than 500 sequences are identified.<br>
21.	The method of claim 20, wherein identifying includes calculating a distance<br>
between peptide sequences of the family.<br>
22.	The method of claim 22, wherein calculating a distance includes using an<br>
antigenic similarity matrix.<br>
23.	The method of claim 1, wherein preparing the family of antigenic peptides<br>
includes chemical synthesis of the family of peptides.<br>
24.	The method of claim 24, wherein the chemical synthesis includes<br>
combinatorial synthesis, whereby the peptides are formed as a mixture of different<br>
sequences.<br>
25.	The method of claim 24, wherein the chemical synthesis includes parallel<br>
synthesis, whereby each peptide is formed separately from other peptides.<br>
26.	The method of claim 26, further comprising mixing the separate peptides.<br>
27.	The method of claim 1, wherein preparing the family of antigenic peptides<br>
includes expression of the family of peptides by a host organism.<br>
28.	A composition comprising a family of antigenic peptides having amino acid<br>
sequences having antigenic similarity to amino acid sequences of a variable region of a<br>
pathogen protein, wherein each antigenic peptide in the family has at least one amino acid<br>
position that varies relative to other antigenic peptides in the family.<br>
29.	The composition of claim 29, wherein one amino acid residue occurs more<br>
frequently than another in the position that varies.<br>
30.	The composition of claim 29, wherein the family includes greater than 150<br>
mutually unique antigenic peptides.<br>
31.	The composition of claim 29, wherein the family includes greater than 1,000<br>
mutually unique antigenic peptides.<br>
33.	The composition of claim 29, wherein the family includes fewer than 100,000<br>
mutually unique antigenic peptides.<br>
34.	The composition of claim 29, wherein the family includes fewer than 50,000<br>
mutually unique antigenic peptides.<br>
35.	The composition of claim 29, wherein the family includes between 1,000 and<br>
50,000 mutually unique antigenic peptides.<br>
36.	The composition of claim 29, wherein the pathogen protein is HIV gp!20.<br>
37.	The composition of claim 36, wherein the variable region is selected from the<br>
group consisting of the VI region, the V2 region, the V3 region, the V4 region and the V5<br>
region.<br>
38.	The composition of claim 37, wherein the family of antigenic peptides<br>
includes sequences having antigenic similarity to sequences from a subtype of HIV.<br>
39.	The composition of claim 38, wherein the subtype is selected from the group<br>
consisting of subtype A, subtype B, subtype C, subtype D, subtype F, subtype G, a<br>
recombinant subtype, a subtype of HIV group N, a subtype of HIV group O, and<br>
combinations thereof.<br>
40.	The composition of claim 29, wherein at least two members of the family of<br>
antigenic peptides are mixed together.<br>
41.	The composition of claim 29, wherein the family of antigenic peptides are<br>
separated according to sequence.<br>
42.	The composition of claim 29, wherein the family includes a multiple antigenic<br>
pep tide.<br>
43.	A peptide library comprising a family of peptides including the fragment:<br>
-N-N-T-R-X4-X5-X6-X7-X8-G-X9-G-X10-X11-X12-X13-X14-T-G-X15-I-X16-G-X17-I-R-<br>
wherein each X4-X17 is a fragment zero, one, two or three amino acid residues in length.<br>
44.	The peptide library of claim 43, wherein the family has antigenic similarity to<br>
the V3 region of HIV gp!20.<br>
45.	The peptide library of claim 44, wherein the family has antigenic similarity to<br>
the V3 region of HIV gp 120 of HIV subtypes A, B, C, D, F, and G.<br>
46.	The peptide library of claim 43, wherein the family of peptides have the<br>
formula:<br>
X1-C-X2-R-P-X3-N-N-T-R-X4-X5-X6-X7-X8-G-X9-G-X10-X11-X12-X13-X14-T-G-X15-I-X16-G-X17-I-R-X18-A-X19-C-X20 wherein each X'-X20 is a fragment zero, one, two or three amino acid residues in length.<br>
47.	The peptide library of claim 46, wherein for each peptide of the family, X1<br>
independently is N, V, T or E.<br>
48.	The peptide library of claim 46, wherein for each peptide of the family, X2<br>
independently is T or I.<br>
49.	The peptide library of claim 46, wherein for each peptide of the family, X3<br>
independently is N, S, G or Y.<br>
50.	The peptide library of claim 46, wherein for each peptide of the family, X4<br>
independently is K, Q or T.<br>
51.	The peptide library of claim 46, wherein for each peptide of the family, X5<br>
independently is S, G or R.<br>
52.	The peptide library of claim 46, wherein for each peptide of the family, X6<br>
independently is I, T or M.<br>
53.	The peptide library of claim 46, wherein for each peptide of the family, X7<br>
independently is H, R, P, S or N.<br>
54.	The peptide library of claim 46, wherein for each peptide of the family, X8<br>
independently is I or M.<br>
55.	The peptide library of claim 46, wherein for each peptide of the family, X9<br>
independently is P or L.<br>
56.	The peptide library of claim 46, wherein for each peptide of the family, X10<br>
independently is R or Q.<br>
57.	The peptide library of claim 46, wherein for each peptide of the family, X11<br>
independently is A, T or V.<br>
58.	The peptide library of claim 46, wherein for each peptide of the family, X12<br>
independently is F or W.<br>
59.	The peptide library of claim 46, wherein for each peptide of the family, X13<br>
independently is Y, F or H.<br>
60.	The peptide library of claim 46, wherein for each peptide of the family, X14<br>
independently is A or T.<br>
61.	The peptide library of claim 46, wherein for each peptide of the family, X15<br>
independently is D, Q, A or R.<br>
62.	The peptide library of claim 46, wherein for each peptide of the family, X16<br>
independently is I or T.<br>
63.	The peptide library of claim 46, wherein for each peptide of the family, X17<br>
independently is D or N.<br>
64.	The peptide library of claim 46, wherein for each peptide of the family, X18<br>
independently is Q or K.<br>
65.	The peptide library of claim 46, wherein for each peptide of the family, X19<br>
independently is H or Y.<br>
66.	The peptide library of claim 46, wherein for each peptide of the family, X20<br>
independently is N or T.<br>
67.       The peptide library of claim 46, wherein for each peptide of the family, X1 independently is N, V, T or E; X2 independently is T or I; X3 independently is N, S, G or Y; X4 independently is K, Q or T; X5 independently is S, G or R; X6 independently is I, T or M; X7 independently is H, R, P, S or N; X8 independently is I or M; X9 independently is P or L; X10 independently is R or Q; X11 independently is A, T or V; X12 independently is F or W; X13 independently is Y, F or H; X14 independently is A or T; X15 independently is D, Q, A or R; X16 independently is I or T; X17 independently is D or N; X18 independently is Q or K; X19 20 •<br>
independently is H or Y; and X   independently is N or T.<br>
68.	The peptide library of claim 46, wherein at least two members of the family of<br>
peptides are mixed together.<br>
69.	The peptide library of claim 46, wherein the family of peptides are separated<br>
according to sequence.<br>
70.	The peptide library of claim 46, wherein the family includes greater than 150<br>
mutually unique peptide sequences.<br>
71.	The peptide library of claim 46, wherein the family includes fewer than<br>
100,000 mutually unique peptide sequences.<br>
72.	The peptide library of claim 46, wherein the family includes fewer than 500<br>
mutually unique peptide sequences, the sequences being representative of the entire sequence<br>
diversity available.<br>
73.	A peptide library comprising a family of peptides including the fragment:<br>
-N-N-X4-R-X5-X6-V-X7-I-G-P-G-X8-X9-F-X10-X11-X12-X13-X14-I-X15-G-X16-I-R-<br>
wherein each X4-X16 is a fragment zero, one, two or three amino acid residues in length.<br>
74.	The peptide library of claim 73, wherein the family has antigenic similarity to<br>
the V3 region of HIV gp!20.<br>
75.	The peptide library of claim 74, wherein the family has antigenic similarity to<br>
the V3 region of HIV gp!20 of HIV subtype A.<br>
76.	The peptide library of claim 73, wherein the family of peptides have the<br>
formula:<br>
X1-C-X2-R-P-X3-N-N-X4-R-X5-X6-V-X7-I-G-P-G-X8-X9-F-X10-X11-X12-X13-X14-I-X15-G-X16-I-R-X17-A-X18-C-X19 wherein each X'-X19 is a fragment zero, one, two or three amino acid residues in length.<br>
77.	The peptide library of claim 76, wherein for each peptide of the family, X1<br>
independently is N, T, or D.<br>
78.	The peptide library of claim 76, wherein for each peptide of the family, X2<br>
independently is T or I.<br>
79.	The peptide library of claim 76, wherein for each peptide of the family, X3<br>
independently is N, G, or S.<br>
80.	The peptide library of claim 76, wherein for each peptide of the family, X4<br>
independently is T or K.<br>
81.	The peptide library of claim 76, wherein for each peptide of the family, X5<br>
independently is K, T, or Q.<br>
82.	The peptide library of claim 76, wherein for each peptide of the family, X6<br>
independently is S, G, R or N.<br>
83.	The peptide library of claim 76, wherein for each peptide of the family, X7<br>
independently is R or H.<br>
84.	The peptide library of claim 76, wherein for each peptide of the family, X8<br>
independently is Q or R.<br>
85.	The peptide library of claim 76, wherein for each peptide of the family, X9<br>
independently is A, T, or V.<br>
86.	The peptide library of claim 76, wherein for each peptide of the family, X10<br>
independently is Y, F, or H.<br>
87.	The peptide library of claim 76, wherein for each peptide of the family, X11<br>
independently is A or T.<br>
88.	The peptide library of claim 76, wherein for each peptide of the family, X12<br>
independently is T, R, A, or no residue.<br>
89.	The peptide library of claim 76, wherein for each peptide of the family, X13<br>
independently is G, D, or no residue.<br>
90.	The peptide library of claim 76, wherein for each peptide of the family, X14<br>
independently is D, A, N, R, or V.<br>
91.	The peptide library of claim 76, wherein for each peptide of the family, X15<br>
independently is I or T.<br>
92.	The peptide library of claim 76, wherein for each peptide of the family, X16<br>
independently is D or N.<br>
93.	The peptide library of claim 76, wherein for each peptide of the family, X17<br>
independently is Q, K, or E.<br>
94.	The peptide library of claim 76, wherein for each peptide of the family, X18<br>
independently is H or Y.<br>
95.	The peptide library of claim 76, wherein for each peptide of the family, X19<br>
independently is N, T, or E.<br>
96.	The peptide library of claim 76, wherein for each peptide of the family X1<br>
independently is N, T, or D; X2 independently is T or I; X3 independently is N, G, or S; X4<br>
independently is T or K; X5 independently is K, T, or Q; X6 independently is S, G, R or N;<br>
X7 independently is R or H; X8 independently is Q or R; X9 independently is A, T, or V; X10<br>
independently is Y, F, or H; X11 independently is A or T; X12 independently is T, R, A, or no<br>
residue; X13 independently is G, D, or no residue; X14 independently is D, A, N, R, or V; X15<br>
independently is I or T; X16 independently is D or N; X17 independently is Q, K, or E; X18<br>
independently is H or Y; and X19 independently is N, T, or E.<br>
97.	The peptide library of claim 76, wherein at least two members of the family of<br>
peptides are mixed together.<br>
98.	The peptide library of claim 76, wherein the family of peptides are separated<br>
according to sequence.<br>
99.	The peptide library of claim 76, wherein the family includes greater than 150<br>
mutually unique peptide sequences.<br>
100.	The peptide library of claim 76, wherein the family includes fewer than<br>
100,000 mutually unique peptide sequences.<br>
101.	The peptide library of claim 76, wherein the family includes fewer than 500<br>
mutually unique peptide sequences, the sequences being representative of the entire sequence<br>
diversity available.<br>
102.	A peptide library comprising a family of peptides including the fragment:<br>
-N-N-T-R-K-X4-I-X5-X6-G-X7-G-X8-X9-X10-X11-X12-T-X13-X14-I-X15-G-X16-I-R-<br>
wherein each X4-X16 is a fragment zero, one, two or three amino acid residues in length.<br>
103.	The peptide library of claim 102, wherein the family has antigenic similarity<br>
to the V3 region of HIV gp!20.<br>
104.	The peptide library of claim 103, wherein the family has antigenic similarity<br>
to the V3 region of HIV gp!20 of HIV subtype B.<br>
105.	The peptide library of claim 102, wherein the family of peptides have the<br>
formula:<br>
X1-C-X2-R-P-X3-N-N-T-R-K-X4-I-X5-X6-G-X7-G-X8-X9-X10-X11-X12-T-X13-X14-I-X15-G-X16-I-R-X17-A-X18-C-X19 wherein each X'-X19 is a fragment zero, one, two or three amino acid residues in length.<br>
106.	The peptide library of claim 105, wherein for each peptide of the family, X1<br>
independently is N, T, or H.The peptide library of claim 105, wherein for each peptide of the family, X2<br>
independently is T or I.<br>
107.	The peptide library of claim 105, wherein for each peptide of the family, X3<br>
independently is N, S, or G.<br>
108.	The peptide library of claim 105, wherein for each peptide of the family, X4<br>
independently is S, G, or R.<br>
109.	The peptide library of claim 105, wherein for each peptide of the family, X5<br>
independently is H, P, N, T or Y.<br>
110.	The peptide library of claim 105, wherein for each peptide of the family, X6<br>
independently is I or M.<br>
111.	The peptide library of claim 105, wherein for each peptide of the family, X7<br>
independently is P, L, or W.<br>
112.	The peptide library of claim 105, wherein for each peptide of the family, X8<br>
independently is R, Q, G or S.<br>
113.	The peptide library of claim 105, wherein for each peptide of the family, X9<br>
independently is A, V or T.<br>
114.	The peptide library of claim 105, wherein for each peptide of the family, X10<br>
independently is F, W, or V.<br>
115.	The peptide library of claim 105, wherein for each peptide of the family, X11<br>
independently is Y, F or H.<br>
116.	The peptide library of claim 105, wherein for each peptide of the family, X12<br>
independently is T or A.<br>
117.	The peptide library of claim 105, wherein for each peptide of the family, X13<br>
independently is G, E or R.The peptide library of claim 105, wherein for each peptide of the family, X14<br>
independently is E, Q, R or G.<br>
119.	The peptide library of claim 105, wherein for each peptide of the family, X15<br>
independently is I or T.<br>
120.	The peptide library of claim 105, wherein for each peptide of the family, X16<br>
independently is D or N.<br>
121.	The peptide library of claim 105, wherein for each peptide of the family, X17<br>
independently is Q or K.<br>
122.	The peptide library of claim 105, wherein for each peptide of the family, X18<br>
independently is H or Y.<br>
123.	The peptide library of claim 105, wherein for each peptide of the family, X19<br>
independently is N or T.<br>
124.	The peptide library of claim 105, wherein for each peptide of the family, X1<br>
independently is N, T or H; X2 independently is T or I; X3 indepeor S; X9 independently is A, V<br>
or T; X10 indndently is N, S, or G; X4<br>
independently is S, G, or R; X5 independently is H, P, N, T or Y; X6 independently is I or M;<br>
X7 independently is P, L, or W; X8 independently is R, Q, G ependently is F, W, or V; X11 independently is Y, F or H; X12 independently is T<br>
or A; X13 independently is G, E or R; X14 independently is E, Q, R or G; X15 independently is<br>
I or T; X16 independently is D or N; X17 independently is Q or K; X18 independently is H or<br>
Y; and X19 independently is N or T.<br>
125.	The peptide library of claim 105, wherein at least two members of the family<br>
of peptides are mixed together.<br>
126.	The peptide library of claim 105, wherein the family of peptides are separated<br>
according to sequence.<br>
127.	The peptide library of claim 105, wherein the family includes greater than 150<br>
mutually unique peptide sequencesThe peptide library of claim 105, wherein the family includes fewer than<br>
100,000 mutually unique peptide sequences.<br>
129.	The peptide library of claim 105, wherein the family includes fewer than 500<br>
mutually unique peptide sequences, the sequences being representative of the entire sequence<br>
diversity available.<br>
131.	A peptide library comprising a family of peptides including the fragment:<br>
-N-N-T-R-X4-X5-X6-X7-I-G-P-G-Q-X8-F-X9-X10-T-X11-X12-I-X13-G-XU-I-R-<br>
wherein each X4-X14 is a fragment zero, one, two or three amino acid residues in length.<br>
132.	The peptide library of claim 131, wherein the family has antigenic similarity<br>
to the V3 region of HIV gp!20.<br>
133.	The peptide library of claim 132, wherein the family has antigenic similarity<br>
to the V3 region of HIV gp!20 of HIV subtype C.<br>
134.	The peptide library of claim 131, wherein the family of peptides have the<br>
formula:<br>
X1-C-X2-R-P-X3-N-N-T-R-X4-X5-X6-X7-I-G-P-G-Q-X8-F-X9-X10-T-X11-X12-I-X13-G-X14-I-R-X15-A-X16-C-X17 wherein each X'-X17 is a fragment zero, one, two or three amino acid residues in length.<br>
135.	The peptide library of claim 134, wherein for each peptide of the family, X1<br>
independently is V, N, E, T, M or R.<br>
136.	The peptide library of claim 134, wherein for each peptide of the family, X2<br>
independently is T, I, or A.<br>
137.	The peptide library of claim 134, wherein for each peptide of the family, X3<br>
independently is N, G, S, or H.<br>
138.	The peptide library of claim 134, wherein for each peptide of the family, X4<br>
independently is K, E or Q.<br>
139.	The peptide library of claim 134, wherein for each pep tide of the family, X5<br>
independently is S or G.<br>
140.	The peptide library of claim 134, wherein for each peptide of the family, X<br>
independently is I or M.<br>
141.	The peptide library of claim 134, wherein for each peptide of the family, X<br>
independently is R or G.<br>
Q<br>
142.	The peptide library of claim 134, wherein for each peptide of the family, X<br>
independently is T, A, or V.<br>
143.	The peptide library of claim 134, wherein for each peptide of the family, X9<br>
independently is Y or F.<br>
144.	The peptide library of claim 134, wherein for each peptide of the family, X10<br>
independently is A or T.<br>
145.	The peptide library of claim 134, wherein for each peptide of the family, X11<br>
independently is G, N, D, K or T.<br>
146.	The peptide library of claim 134, wherein for each peptide of the family, X12<br>
independently is D, G, N, S, or I.<br>
147.	The peptide library of claim 134, wherein for each peptide of the family, X13<br>
independently is I or T.<br>
148.	The peptide library of claim 134, wherein for each peptide of the family, X14<br>
independently is D or N.<br>
149.	The peptide library of claim 134, wherein for each peptide of the family, X15<br>
independently is Q, K, E, L or H.<br>
150.	The peptide library of claim 134, wherein for each peptide of the family, X16<br>
independently is H, Y, or N.<br>
151.	The peptide library of claim 134, wherein for each peptide of the family, X17<br>
independently is N, T, I, D or H.<br>
152.	The peptide library of claim 134, wherein for each peptide of the family, X1<br>
independently is V, N, E, T, M or R; X2 independently is T, I, or A; X3 independently is N,<br>
G, S, or H; X4 independently is K, E or Q; X5 independently is S or G; X6 independently is I<br>
or M; X7 independently is R or G; X8 independently is T, A, or V; X9 independently is Y or<br>
F; X10 independently is A or T; X11 independently is G, N, D, K or T; X12 independently is<br>
D, G, N, S, or I; X13 independently is I or T; X14 independently is D or N; X15 independently<br>
is Q, K, E, L or H; X16 independently is H, Y, or N; and X17 independently is N, T, I, D or H.<br>
153.	The peptide library of claim 134, wherein at least two members of the family<br>
of peptides are mixed together.<br>
154.	The peptide library of claim 134, wherein the family of peptides are separated<br>
according to sequence.<br>
155.	The peptide library of claim 134, wherein the family includes greater than 150<br>
mutually unique peptide sequences.<br>
156.	The peptide library of claim 134, wherein the family includes fewer than<br>
100,000 mutually unique peptide sequences.<br>
157.	The peptide library of claim 134, wherein the family includes fewer than 500<br>
mutually unique peptide sequences, the sequences being representative of the entire sequence<br>
diversity available.<br>
158.	A peptide library comprising a family of peptides including the fragment:<br>
-R-P-X3-X4-X5-X6-R-X7-X8-X9-X10-I-G-X11-G-X12-X13-X14-X15-X16-T-X17-X18-G-X19-I-<br>
wherein each X3-X19 is a fragment zero, one, two or three amino acid residues in length.<br>
159.	The peptide library of claim 158, wherein the family has antigenic similarity<br>
to the V3 region of HIV gp!20.<br>
160.	The peptide library of claim 159, wherein the family has antigenie similarity<br>
to the V3 region of HIV gp!20 of subtype D.<br>
161.	The peptide library of claim 158, wherein the family of peptides have the<br>
formula:<br>
X1-C-X2-R-P-X3-X4-X5-X6-R-X7-X8-X9-X10-I-G-X11-G-X12-X13-X14-X15-X16-T-X17-X18-G-X19-I-X20-X21-A-X22-C-X23<br>
wherein each X'-X23 is a fragment zero, one, two or three amino acid residues in length.<br>
162.	The peptide library of claim 161, wherein for each peptide of the family, X1<br>
independently is N or T.<br>
163.	The peptide library of claim 161, wherein for each peptide of the family, X2<br>
independently is T or I.<br>
164.	The peptide library of claim 161, wherein for each peptide of the family, X3<br>
independently is Y or N.<br>
165.	The peptide library of claim 161, wherein for each peptide of the family, X4<br>
independently is N or K.<br>
166.	The peptide library of claim 161, wherein for each peptide of the family, X5<br>
independently is N or K.<br>
167.	The peptide library of claim 161, wherein for each peptide of the family, X6<br>
independently is T, I or K.<br>
168.	The peptide library of claim 161, wherein for each peptide of the family, X7<br>
independently is Q or R.<br>
169.	The peptide library of claim 161, wherein for each peptide of the family, X8<br>
independently is S, R or G.<br>
170.	The peptide library of claim 161, wherein for each peptide of the family, X9<br>
independently is T or I.<br>
171.	The peptide library of claim 161, wherein for each peptide of the family, X10<br>
independently is H, P, S or R.<br>
172.	The peptide library of claim 161, wherein for each peptide of the family, X11<br>
independently is P, L, S or Q.<br>
173.	The peptide library of claim 161, wherein for each peptide of the family, X12<br>
independently is Q or R.<br>
174.	The peptide library of claim 161, wherein for each peptide of the family, X13<br>
independently is A or T.<br>
175.	The peptide library of claim 161, wherein for each peptide of the family, X14<br>
independently is L or Y.<br>
176.	The peptide library of claim 161, wherein for each peptide of the family, X15<br>
independently is Y or F.<br>
177.	The peptide library of claim 161, wherein for each peptide of the family, X16<br>
independently is T or A.<br>
178.	The peptide library of claim 161, wherein for each peptide of the family, X17<br>
independently is I or K.<br>
I O<br>
179.	The peptide library of claim 161, wherein for each peptide of the family, X18<br>
independently is I, T or K.<br>
180.	The peptide library of claim 161, wherein for each peptide of the family, X19<br>
independently is D or N.<br>
181.	The peptide library of claim 161, wherein for each peptide of the family, X20<br>
independently is R or G.<br>
182.	The peptide library of claim 161, wherein for each peptide of the family, X21<br>
independently is Q or K.<br>
183.	The peptide library of claim 161, wherein for each peptide of the family, X22<br>
independently is H or Y.<br>
184.	The peptide library of claim 161, wherein for each peptide of the family, X23<br>
independently is N, T or K.<br>
185.	The peptide library of claim 161, wherein for each peptide of the family, X1<br>
independently is N or T; X2 independently is T or I; X3 independently is Y or N; X4<br>
independently is N or K; X5 independently is N or K; X6 independently is T, I or K; X7<br>
independently is Q or R; X8 independently is S, R or G; X9 independently is T or I; X10<br>
independently is H, P, S or R; X11 independently is P, L, S or Q; X12 independently is Q or R;<br>
X13 independently is A or T; X14 independently is L or Y; X15 independently is Y or F; X16<br>
independently is T or A; X17 independently is I or K; X18 independently is I, T or K; X19<br>
independently is D or N; X20 independently is R or G; X21 independently is Q or K; X22<br>
independently is H or Y; and X23 independently is N, T or K.<br>
186.	The peptide library of claim 161, wherein at least two members of the family<br>
of peptides are mixed together.<br>
187.	The peptide library of claim 161, wherein the family of peptides are separated<br>
according to sequence.<br>
188.	The peptide library of claim 161, wherein the family includes greater than 150<br>
mutually unique peptide sequences.<br>
189.	The peptide library of claim 161, wherein the family includes fewer than<br>
100,000 mutually unique peptide sequences.<br>
190.	The peptide library of claim 161, wherein the family includes fewer than 500<br>
mutually unique peptide sequences, the sequences being representative of the entire sequence<br>
diversity available.<br>
191.	A peptide library comprising a family of peptides including the fragment:<br>
-N-N-X3-R-K-X4-I-X5-L-G-P-G-X6-X7-X8-X9-X10-X11-X12-X13-I-X14-G-X15-I-R-<br>
wherein each X3-X15 is a fragment zero, one, two or three amino acid residues in length.The peptide library of claim 191, wherein the family has antigenic similarity<br>
the V3 region of HIV gp!20.<br>
192.	The peptide library of claim 192, wherein the family has antigenic similarity<br>
the V3 region of HIV gp 120 of HIV subtype F.<br>
193.	The peptide library of claim 191, wherein the family of peptides have the<br>
formula:<br>
X1-C-T-R-P-X2-N-N-X3-R-K-X4-I-X5-L-G-P-G-X6-X7-X8-X9-X10-X11-X12-X13-I-X14-G-X15-I-R-X16-A-X17-C-X18 wherein each X'-X18 is a fragment zero, one, two or three amino acid residues in length.<br>
195.	The peptide library of claim 194, wherein for each peptide of the family, X1<br>
independently is N or T.<br>
196.	The peptide library of claim 194, wherein for each peptide of the family, X2<br>
independently is N or S.<br>
197.	The peptide library of claim 194, wherein for each peptide of the family, X<br>
independently is T or I.<br>
198.	The peptide library of claim 194, wherein for each peptide of the family, X4<br>
independently is S, G, or R.<br>
199.	The peptide library of claim 194, wherein for each peptide of the family, Xs<br>
independently is H, R, Q, P, Y, or S.<br>
200.	The peptide library of claim 194, wherein for each peptide of the family, X6<br>
independently is Q, R or H.<br>
201.	The peptide library of claim 194, wherein for each peptide of the family, X7<br>
independently is A, V or T.<br>
Q<br>
202.	The peptide library of claim 194, wherein for each peptide of the family, X<br>
independently is F or I.<br>
203.	The peptide library of claim 194, wherein for each peptide of the family, X9<br>
independently is Y, H or F.<br>
204.	The peptide library of claim 194, wherein for each peptide of the family, X10<br>
independently is A or T.<br>
205.	The peptide library of claim 194, wherein for each peptide of the family, X11<br>
independently is T or A.<br>
206.	The peptide library of claim 194, wherein for each peptide of the family, X12<br>
independently is G, D or S.<br>
207.	The peptide library of claim 194, wherein for each peptide of the family, X13<br>
independently is D, A, or N.<br>
208.	The peptide library of claim 194, wherein for each peptide of the family, X14<br>
independently is I or T.<br>
209.	The peptide library of claim 194, wherein for each peptide of the family, X15<br>
independently is D or N.<br>
210.	The peptide library of claim 194, wherein for each peptide of the family, X16<br>
independently is K or Q.<br>
211.	The peptide library of claim 194, wherein for each peptide of the family, X17<br>
independently is H or Y.<br>
212.	The peptide library of claim 194, wherein for each peptide of the family, X18<br>
independently is N, I, D, or K.<br>
213.	The peptide library of claim 194, wherein for each peptide of the family, X1<br>
independently is N or T; X2 independently is N or S; X3 independently is T or I; X4<br>
independently is S, G, or R; X5 independently is H, R, Q, P, Y, or S; X6 independently is Q,<br>
R or H; X7 independently is A, V or T; X8 independently is F or I; X9 independently is Y, H<br>
or F; X10 independently is A or T; X1' independently is T or A; X12 independently is G, D or<br>
S; X13 independently is D, A, or N; X14 independently is I or T; X15 independently is D or N; X16 independently is K or Q; X17 independently is H or Y; and X18 independently is N, I, D, K or X.<br>
214.	The peptide library of claim 194, wherein at least two members of the family<br>
of peptides are mixed together.<br>
215.	The peptide library of claim 194, wherein the family of peptides are separated<br>
according to sequence.<br>
216.	The peptide library of claim 194, wherein the family includes greater than 150<br>
mutually unique peptide sequences.<br>
217.	The peptide library of claim 194, wherein the family includes fewer than<br>
100,000 mutually unique peptide sequences.<br>
218.	The peptide library of claim 194, wherein the family includes fewer than 500<br>
mutually unique peptide sequences, the sequences being representative of the entire sequence<br>
diversity available.<br>
219.	A peptide library comprising a family of peptides including the fragment:<br>
-N-N-T-R-K-S-X4-X5-X6-G-X7-X8-X9-X10-X11-X12-X13-T-X14-I-X15-G-X16-I-R-<br>
wherein each X4-X16 is a fragment zero, one, two or three amino acid residues in length.<br>
220.	The peptide library of claim 219, wherein the family has antigenic similarity<br>
to the V3 region of HIV gp!20.<br>
221.	The peptide library of claim 220, wherein the family has antigenic similarity<br>
to the V3 region of HIV gp!20 of HIV subtype G.<br>
222.	The peptide library of claim 219, wherein the family of peptides have the<br>
formula:<br>
X1-C-X2-R-P-X3-N-N-T-R-K-S-X4-X5-X6-G-X7-X8-X9-X10-X11-X12-X13-T-X14-I-X15-G-X16-I-R-X17-A-X18-C-X19 wherein each X'-X19 is a fragment zero, one, two or three amino acid residues in length.<br>
223.	The peptide library of claim 222, wherein for each peptide of the family, X1<br>
independently is N, I, T, M or R.<br>
224.	The peptide library of claim 222, wherein for each peptide of the family, X2<br>
independently is T or I.<br>
225.	The peptide library of claim 222, wherein for each peptide of the family, X3<br>
independently is N or S.<br>
226.	The peptide library of claim 222, wherein for each peptide of the family, X4<br>
independently is I or K.<br>
227.	The peptide library of claim 222, wherein for each peptide of the family, X5<br>
independently is T, R, H, P or N.<br>
228.	The peptide library of claim 222, wherein for each peptide of the family, X6<br>
independently is F or I.<br>
229.	The peptide library of claim 222, wherein for each peptide of the family, X7<br>
independently is P, T, or L.<br>
230.	The peptide library of claim 222, wherein for each peptide of the family, X8<br>
independently is G or R.<br>
231.	The peptide library of claim 222, wherein for each peptide of the family, X9<br>
independently is Q, R, H or G.<br>
232.	The peptide library of claim 222, wherein for each peptide of the family, X10<br>
independently is A, T or V.<br>
233.	The peptide library of claim 222, wherein for each peptide of the family, X11<br>
independently is F, I or S.<br>
234.	The peptide library of claim 222, wherein for each peptide of the family, X12<br>
independently is Y or H.The peptide library of claim 222, wherein for each peptide of the family, X13<br>
independently is A or T.<br>
235.	The peptide library of claim 222, wherein for each peptide of the family, X14<br>
independently is G, D or S.<br>
236.	The peptide library of claim 222, wherein for each peptide of the family, X15<br>
independently is I or T.<br>
237.	The peptide library of claim 222, wherein for each peptide of the family, X16<br>
independently is D or N.<br>
238.	The peptide library of claim 222, wherein for each peptide of the family, X17<br>
independently is Q, L, K or P.<br>
239.	The peptide library of claim 222, wherein for each peptide of the family, X18<br>
independently is H, Y or F.<br>
240.	The peptide library of claim 222, wherein for each peptide of the family, X19<br>
independently is N, K or T.<br>
241.	The peptide library of claim 222, wherein for each peptide of the family, X1<br>
independently is N, I, T, M or R; X2 independently is T or I; X3 independently is N or S; X4<br>
independently is I or K; X5 independently is T, R, H, P or N; X6 independently is F or I; X7<br>
independently is P, T, or L; X8 independently is G or R; X9 independently is Q, R, H or G;<br>
X10 independently is A, TorV;Xu independently is F, lor S; X12 independently is YorH;<br>
X13 independently is A or T; X14 independently is G, D or S; X15 independently is I or T; X16<br>
independently is D or N; X17 independently is Q, L, K or P; X18 independently is H, Y or F;<br>
X19 independently is N, K or T.<br>
242.	The peptide library of claim 222, wherein at least two members of the family<br>
of peptides are mixed together.<br>
243.	The peptide library of claim 222, wherein the family of peptides are separated<br>
according to sequence.<br>
245.	The peptide library of claim 222, wherein the family includes greater than 150<br>
mutually unique peptide sequences.<br>
246.	The peptide library of claim 222, wherein the family includes fewer than<br>
100,000 mutually unique peptide sequences.<br>
247.	The peptide library of claim 222, wherein the family includes fewer than 500<br>
mutually unique peptide sequences, the sequences being representative of the entire sequence<br>
diversity available.<br>
248.	A method of eliciting an immune response in a subject comprising<br>
administering to the subject a composition including a family of antigenic peptides having<br>
amino acid sequences having antigenic similarity to amino acid sequences of a variable<br>
region of a pathogen protein of a pathogen.<br>
249.	The method of claim 248, wherein the antigenic similarity is determined using<br>
an antigenic similarity matrix.<br>
250.	The method of claim 248, wherein the composition is administered to a<br>
subject prior to infection by the pathogen.<br>
251.	The method of claim 250, wherein the family of antigenic peptides has amino<br>
acid sequences having antigenic similarity to amino acid sequences of a subtype of the<br>
pathogen protein.<br>
252.	The method of claim 250, wherein the family of antigenic peptides has amino<br>
acid sequences having antigenic similarity to amino acid sequences from more than one<br>
subtype of the pathogen protein.<br>
253.	The method of claim 248, wherein the composition is administered to a<br>
subject infected by the pathogen.<br>
254.	The method of claim 253, wherein the subject is infected by a subtype of the<br>
pathogen.255.	The method of claim 254, wherein the family of antigenic peptides has amino<br>
acid sequences having antigenic similarity to amino acid sequences from the subtype by<br>
which the subject is infected.<br>
256.	A method of designing a family of peptide sequences, comprising:<br>
locating a plurality of variable positions in a region of a pathogen protein;<br>
choosing a peptide sequence of the pathogen protein including the variable positions;<br>
and<br>
selecting one or more substitute amino acid residues for one of the variable positions of the peptide based on antigenic similarity to amino acid residues naturally occurring at the variable position of the pathogen protein, thereby forming a family of peptide sequences.<br>
257.	The method of claim 256, wherein selecting includes determining the<br>
antigenic similarity using an antigenic similarity matrix.<br>
258.	The method of claim 256, further comprising assigning a frequency to each<br>
substitute amino acid residue in the family of antigenic peptide sequences.<br>
259.	The method of claim 258, wherein assigning further includes considering the<br>
frequency with which the variations naturally occur.<br>
260.	The method of claim 256, further comprising determining an antigenic<br>
similarity between a peptide of the family and a region of a human protein.<br>
261.	The method of claim 260, further comprising removing a peptide sequence<br>
from the family of peptide sequences if the calculated antigenic similarity between the<br>
peptide of the family and the region of a human protein exceeds a predetermined threshold.<br>
262.	The method of claim 256, wherein the family of peptide sequences includes<br>
members, such that the members taken together have antigenic similarity to each naturally<br>
occurring sequence of the region of the pathogen protein.The method of claim 256, wherein the family of antigenic peptides includes<br>
members, such that the members taken together have antigenic similarity to a non-naturally<br>
occurring sequence of the region of the pathogen protein.<br>
263.	The method of claim 256, further comprising identifying peptide sequences of<br>
the family, the identified peptide sequences being representative of the sequence diversity of<br>
the entire family.<br>
264.	The method of claim 264, wherein fewer than 500 sequences are identified.<br>
265.	The method of claim 264, wherein identifying includes calculating a distance<br>
between peptide sequences of the family.<br>
266.	The method of claim 266, wherein calculating a distance includes using an<br>
antigenic similarity matrix.<br>
267.	The method of claim 256, further comprising identifying peptide sequences of<br>
the family having antigenic similarity to a subtype of the pathogen protein.<br>
268.	The method of claim 256, wherein the pathogen protein includes a<br>
hypervariable region.<br>
269.	The method of claim 256, wherein the pathogen is a virus.<br>
270.	The method of claim 270, wherein the virus is HIV.<br>
271.	The method of claim 271, wherein the pathogen protein is gp!20.<br>
272.	The method of claim 272, wherein the region is selected from the group<br>
consisting of the VI region, the V2 region, the V3 region, the V4 region, and the V5 region.<br>
273.	A method of diagnosing infection comprising contacting a sample with a<br>
family of peptides having amino acid sequences having antigenic similarity to amino acid<br>
sequences of a variable region of a pathogen protein, wherein each peptide in the family has<br>
at least one amino acid position that varies relative to other peptides in the family.<br>
275.	The method of claim 274, wherein an ti genie similarity is determined using an<br>
antigenic similarity matrix.<br>
276.	The method of claim 274, wherein the family of peptides is antigenically<br>
similar to a pathogen subtype.<br>
277.	The method of claim 274, wherein the family of peptides is antigenically<br>
similar to more than one pathogen subtype.<br><br>
278.	' The method of claim 274, further comprising immobilizing the family of<br>
peptides on a substrate before contacting.<br>
279.	The method of claim 274, further comprising determining if the sample<br>
includes antibodies that bind specifically to the family of peptides.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM4OC1kZWxucC0yMDA1LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1388-delnp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM4OC1kZWxucC0yMDA1LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1388-delnp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM4OC1kZWxucC0yMDA1LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1388-delnp-2005-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM4OC1kZWxucC0yMDA1LWNvcnJlc3BvbmRlbmNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">1388-delnp-2005-correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM4OC1kZWxucC0yMDA1LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">1388-delnp-2005-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM4OC1kZWxucC0yMDA1LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">1388-delnp-2005-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM4OC1kZWxucC0yMDA1LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">1388-delnp-2005-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM4OC1kZWxucC0yMDA1LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">1388-delnp-2005-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM4OC1kZWxucC0yMDA1LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">1388-delnp-2005-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM4OC1kZWxucC0yMDA1LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">1388-delnp-2005-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM4OC1kZWxucC0yMDA1LXBjdC0xMDEucGRm" target="_blank" style="word-wrap:break-word;">1388-delnp-2005-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM4OC1kZWxucC0yMDA1LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">1388-delnp-2005-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM4OC1kZWxucC0yMDA1LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">1388-delnp-2005-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM4OC1kZWxucC0yMDA1LXBjdC0zMDEucGRm" target="_blank" style="word-wrap:break-word;">1388-delnp-2005-pct-301.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM4OC1kZWxucC0yMDA1LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">1388-delnp-2005-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM4OC1kZWxucC0yMDA1LXBjdC0zMDgucGRm" target="_blank" style="word-wrap:break-word;">1388-delnp-2005-pct-308.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM4OC1kZWxucC0yMDA1LXBjdC0zMzIucGRm" target="_blank" style="word-wrap:break-word;">1388-delnp-2005-pct-332.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM4OC1kZWxucC0yMDA1LXBjdC00MDkucGRm" target="_blank" style="word-wrap:break-word;">1388-delnp-2005-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM4OC1kZWxucC0yMDA1LXBjdC00MTYucGRm" target="_blank" style="word-wrap:break-word;">1388-delnp-2005-pct-416.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="227521-a-process-for-recovery-of-ammonium-sulphate-from-vanadium-pentoxide-plant-effluent.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="227523-a-protective-relay-for-monitoring-a-power-distribution-network.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>227522</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1388/DELNP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>05/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>30-Jan-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>12-Jan-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>06-Apr-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>STATE RESEARCH CENTER OF VIROLOGY AND BIOTECHNOLOGY &quot;VECTOR&quot;</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>PGT KOLTSOVO, NOVOSIBIRSKY R-N, NOVOSIBIRSKAYA OBL.,630559, RUSSIA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>AMIR ZAKIEVICH MAKSYUTOV</td>
											<td>POS. KOLTSOVO, 12-27 NOVOSIBIRSKAYA OBL., 630559, RUSSIA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>ALEXANDER BORISOVICH RYZHIKOV</td>
											<td>POS. KOLTSOVO, 24-5, NOVOSIBIRSKAYA OBL., 630559, RUSSIA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>ALEXANDER ALEXANDROVICH KOLOBOV</td>
											<td>PRIMORSKOE SHOSSE, 265-102, SESTRORETSK, 188640, RUSSIA</td>
										</tr>
										<tr>
											<td>4</td>
											<td>ZAKI AMIROVICH MAKSYUTOV</td>
											<td>POS. KOLTSOVO, 12-27, NOVOSIBIRSKAYA OBL., 630559, RUSSIA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 1/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/RU2003/000421</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-09-25</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>2002126396</td>
									<td>2002-10-03</td>
								    <td>Russia</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/227522-antigenic-peptide-libraries by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:22:35 GMT -->
</html>
