<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/227722-method-for-non-diagnostic-analysis-of-interfering-rna by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:55:32 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 227722:&quot;METHOD FOR NON-DIAGNOSTIC ANALYSIS OF INTERFERING RNA&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;METHOD FOR NON-DIAGNOSTIC ANALYSIS OF INTERFERING RNA&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>&quot;METHOD FOR NON-DIAGNOSTIC ANALYSIS OF INTERFERING RNA&quot; The present invention relates to methods for the detection and characterization of interfering RNAs such as micro RNAs (miRNAs) and small interfering RNAs (siRNAs) and other short nucleic acid molecules. More particularly, the present invention relates to improved methods for the detection and quantitation of ineterfering RNA expression. The present invention further provides for the detection of variants and types of miRNAs and siRNAs.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The present Application claims priority to U.S. Provisional Application Serial Number 60/434,518, filed.December 18, 2002, and U.S. Application Serial Number 60/443,814, filed January 30, 2003.<br>
FIELD OF THE INVENTION<br>
The present invention relates to a method for non-diagnostic analysis of interfering<br>
RNA. This invention relates to compositions and methods for the detection and characterization of interfering RNAs such as micro RNAs (miRNAs) and small<br>
interfering RNAs (siRNAs) and other short nucleic acid molecules. More particularly,<br>
the present invention relates to improved methods for the detection and quantitation of<br>
interfering RNA. expression. The present invention further provides for the detection of<br>
variants and types of miRNAs and siRNAs.<br>
BACKGROUND OF THE INVENTION<br>
MicroRNAs (miRNAs) are a new class of noncoding RNAs, which are encoded as short inverted repeats in the genomes of invertebrates and vertebrates (Ambros, (2001) Cell 107, 823-826; Moss (2002) Curr. Biol. 12, R138-R140). miRNAs are modulators of target mRNA translation and stability, although most target mRNAs remain to be identified. miRNAs sequence-specifically control translation of target mRNAs by binding to sites of antisense complementarity in 3' untranslated regions (UTRs) (Ambros, supra; Moss, supra; Lagos-Qumtana et al., (2001) Science 294, 853-858; Lau et al., (2001) Science 294, 858-862; Lee et al., (2001) Science 294, 862-864).<br>
Several miRNAs, such as let-7 RNA, miR-1, miR-34, miR-60, and miR-87, are highly conserved between invertebrates and vertebrates, implicating that they may recognize multiple sites and/or multiple targets of presumably conserved function (Lagos-Quintana et al.. supra; Lau et al., supra; Lee et al., supra; Pasquinelli et al., (2000) Nature 408:86). The small temporal RNAs (stRNAs) lm-4 and let-7 represent a subclass of miRNAs identified by genetic analysis m Caenorhabditis eiegans, which are developmentally regulated and themselves control developmental programs, such as timing of neuronal rewiring, Dauer larva formation, vulva formation, and the terminal differentiation of hypoderrnal cells.<br><br><br>
miRNAs are typically excised from 60- to 70-nucleotide foldback RNA precursor structures, which are sometimes detected at the onset of miRNA precursor expression (Grishok et al., (2001) Cell 106,23-34; Hutvagner et al. (2001) Science 93,834-838; Ketting et al., (2001) Genes Dev. 75,2654-2659) or during expression of very abundant miRNAs (Lagos-Quintana et al., supra; Lau et al., supra; Lee et al., supra). Generally, only one of the strands of the hairpin precursor molecule is excised and accumulates, presumably because it is protected by associated proteins from RNA degradation. These putative proteins may mediate the translational suppression. The miRNA precursor processing reaction requires Dicer RNase IE and Argonaute family members (Grishok et al., supra; Hutvagner et al., supra; Ketting et al., supra).<br>
In addition to their impact on gene expression, these small RNAs, often in the range of 21-22 nucleotides, may find utility in areas of therapeutics and drug discovery, e.g. as drug targets or as pharmaceutical agents. Thus, in some circumstances, it may be important to know approximately how much of each miRNA exists in cells. In some cases, it may further be important to compare levels of miRNA in different tissue types or before and after application of a stimulus, e.g. a chemical or physical intervention. Because related siRNAs and miRNAs may be present in low amounts in cells, it is desirable that methods of detection be both sensitive and specific. Moreover, for certain applications, it may be beneficial to identify methods suitable for high throughput screening, e.g. homogeneous methods, multiplexed methods, or those suitable to highly parallel automated manipulation and limited temperature changes.<br>
Although miRNAs play important roles in the regulation of gene expression, effective techniques for the detection and quantitation of miRNA expression are lacking. To date, the principal methods used for quantitation of miRNAs are based on gel electrophoresis. The miRNAs are detected either by Northern blotting or by the presence of radioactive RNase-resistant duplexes. Northern blotting and chip hybridization methods have relatively low analytical sensitivity (Krichevsky et al. 2003), so microgram quantities of RNA are needed for analyses; moreover, transfer of small RNAs to filters can introduce problems with reproducibility of quantitation and is not typically amendable to high-throughput. Moreover, detection methods based on RNase resistance require highly radioactive probes. Further, assays based solely on probe hybridization<br><br>
may not provide adequate discrimination between isotypes closely related in sequence. Alternative approaches involve cloning the miRNAs and then sequencing the inserts. While this approach maybe suitable for discriminating single-base differences between closely related miRNA species, it is time consuming and laborious.<br>
Like miRNAs, small interfering RNAs (siRNAs) are small RNA molecules involved in cell defense, e.g. against viral RNA, via a response termed RNA interference (RNAi) (Cullen, BJEL, Nature Immunology, 3: 597-599 (2002)). One class of siRNAs is produced through the action of the Dicer enzyme and RNA-induced silencing .complex (RISC) protein complex as part of the RNAi response to the presence of double stranded RNA in cells (Khvorova, A. et al., Cell 115: 209-216 (2003)). Another class of siRNAs is synthetic and encompasses short duplexes, usually 21-23 nt with characteristic dinucleotide overhangs (Elbashir, S.M. et al., EMBO J. 20: 6877-6888 (2001)) introduced directly into cells via transfection or expression from an introduced vector (Paul, C.P. et al., Nature Biotechnology 20: 505-508 (2002), US Patent Application Publication No. 2003/0148519A1, herein incorporated by reference in its entirety for all purposes). In some cases, siRNAs appear to persist as defined sequences, making them analogous in function and composition to miRNAs (Elbashir, S.M. et al., supra). What is needed are efficient and accurate methods of detecting and quantitating miRNA and siRNA levels.<br>
SUMMARY OF THE INVENTION<br>
The present invention relates to compositions and methods for the detection and characterization of small nucleic acids, such as interfering RNAs and other short nucleic acid molecules. More particularly, the present invention relates to unproved methods for the detection and quantitation of interfering RNA expression.<br>
For example, the present invention provides a method, comprising: hybridizing at least one nucleic acid (e.g., that contains sequence that is not complementary to the interfering RNA) to a interfering RNA target to generate a detection structure and detecting the detection structure. In some embodiments, the interfering RNA target is an miRNA. In other embodiments, the interfering RNA target is an siRNA. Li some embodiments, the siRNA is double stranded.<br><br>
In some embodiments, the detection structure comprises an invasive cleavage structure. For example, in some embodiments, the nucleic acid comprises first and second oligonucleotides configured to form an invasive cleavage structure in combination with the miRNA. In some embodiments, the nucleic acid comprises a first oligonucleotide configured to form an invasive cleavage structure in combination with said miRNA. In some embodiments, the first oligonucleotide comprises a 5' portion and a 3' portion, wherein said 3' portion is configured to hybridize to said target sequence, and wherein said 5' portion is configured to not hybridize to said target sequence. In some embodiments employing a second oligonucleotide, the second oligonucleotide comprises a 5' portion and a 3' portion, wherein said 5' portion is configured to hybridize to said target sequence, and wherein said 3' portion is configured to not hybridize to said target sequence. In some embodiments, the detecting step comprises use of an INVADER assay.<br>
In some embodiments, the detection structure comprises a circular oligonucleotide hybridized to said small RNA to generate a circular detection structure. In some embodiments, the detecting step comprises use of a rolling circle replication assay.<br>
In some embodiments, the detecting step(s) comprises use of a detection assay including, but not limited to, sequencing assays, polymerase chain reaction assays, hybridization assays, hybridization assays employing a probe complementary to a mutation, microarray assays, bead array assays, primer extension assays, enzyme mismatch cleavage assays, branched hybridization assays, NASBA assays, molecular beacon assays, cycling probe assays, ligase chain reaction assays, invasive cleavage structure assays, ARMS assays, and sandwich hybridization assays. In some preferred embodiments, the detecting step is carried out in cell lysate.<br>
In some embodiments, the methods of the present invention comprise detecting a second nucleic acid target, hi some preferred embodiments, the second nucleic acid target is RNA. In some particularly preferred embodiments, the second nucleic acid target is U6 RNA or GAPDH mRNA.<br>
In some embodiments, the nucleic acid used to form the detection structure comprises a template with one or more sites sufficiently complementary to the small RNA so as to allow the RNA to hybridize to the template and be extended in an extension<br><br>
reaction. In some embodiments, the extension reaction is a polymerase chain reaction wherein one or more RNAs are used as primers in the polymerase chain reaction. In some such embodiments, a single type of RNA binds to two locations on the template to provide the polymerase chain reaction primers. In other embodiments, two or more RNAs are used as primers. In such embodiments, the detection of an amplification product signifies the presence of the two or more RNAs in the sample (i.e., an miRNA multiplex assay). Similar methods may be employed in a ligase chain reaction where the miRNAs are used as the ligated oligonucleotide(s).<br>
In some embodiments, the method comprises detection of a plurality of miRNAs. In some such embodiments, the plurality of miRNAs comprises polymorphisms of the same miRNA. In other embodiments, the plurality of miRNAs comprises different miRNAs (e.g., Let-7, miR-1, miR-ld, miR-135, nu'R-15, miR-16, miR-124a, or miR125b).<br>
The present invention also provides kits for conducting any of the above methods. For example, in some embodiments, the present invention provides kits comprising a nucleic acid configured for forming a detection structure when hybridized to an RNA target sequence. In some embodiments, the kits are configured to detect an miRNA. In some preferred embodiments, kits are configured to detect a Let-7, miR-1, miR-135, miR-15, miR-16, miR-lb, miR-124a, or miR125b miRNA. In some preferred embodiments, kits are configured to co-detect a second RNA target with an miRNA target.<br>
DESCRIPTION OF THE DRAWINGS<br>
Figure 1 shows a schematic diagram of INVADER oligonucleotides, probe oligonucleotides and FRET cassettes for detecting two different alleles (e.g., differing by a single nucleotide) in a single reaction.<br>
Figure 2 shows an exemplary detection structure utilized in some embodiments of the present invention.<br>
Figure 3 shows a second exemplary detection structure utilized in some embodiments of the present invention.<br><br>
,   Figure 4 shows a third exemplary detection structure used in some embodiments of the present inventioa<br>
Figure 5 shows exemplary oligonucleotides for use with the present invention. Bases of the target miRNA are underlined in lower case. DNA residues in probe or INVADER oligonucleotides are in regular type. Lower case type indicates 2'-O-methyl residues.<br>
Figure 6 shows the results of temperature optimization experiments for let-7.<br>
Figure 7 shows the results of temperature optimization experiments for let-7.<br>
Figure 8 shows the results of limit of detection (LOD) experiments for let-7.<br>
Figure 9 shows the results of cross reactivity experiments using let-7 miRNA.<br>
Figure 10 shows the results of LOD experiments for miR-1.<br>
Figure 11 shows the results of CUBA VASE enzymes IX and Xn comparisons using let-7 miRNA.<br>
Figure 12 shows a partial sequence alignment of U6 RNA sequences from various organisms.<br>
Figure 13 shows the results of temperature optimization experiments for mir-135.<br>
Figure 14 shows the results of LOD experiments for mir-135.<br>
Figure 15 contains a graphical representation of average counts obtained for the detection of let-7 in cell lysates.<br>
Figure 16 shows the results of miRNA and mRNA in cell lysates with and without RNAse A treatment<br>
Figure 17 shows results of invasive cleavage assays comparing the effects of including full-length vs. shortened ARRESTOR oligonucleotides.<br>
Figure 18 shows the results of temperature optimization experiments using the assay designs described in Figure 16.<br>
Figure 19 shows the results of temperature optimization experiments using the 10-mer probe and 12-mer INVADER oligonucleotide designs.<br>
Figure 20 shows the results of experiments to compare the LODs of two alternative oligonucleotide designs.<br>
Figure 21 shows results comparing the effects of substituting 2'-deoxy residues for some or all of the 2'-O-methyl residues in the probe and INVADER oligonucleotides.<br><br>
Figure 22 shows the results of invasive cleavage assays to detect miR-124a.<br>
Figure 23 shows the results of experiments to detect five different miRNA species in total RNA isolated from 20 different tissue types.<br>
Figure 24 shows results of experiments testing the effect on miRNA detection of altering probe and oligonucleotide length.<br>
Figure 25 shows exemplary invasive cleavage oligonucleotide designs for detection of an siRNA. Lower case residues indicate 2'-O-methyl.<br>
DEFINITIONS<br>
To facilitate an understanding of the present invention, a number of terms and phrases are defined below:<br>
As used herein, the term "miRNA" refers to micro RNA. As used herein, the term "miRNA target sequence" refers to a miRNA that is to be detected (e.g., in the presence of other nucleic acids). In some embodiments, a miRNA target sequence is a variant of a miRNA.<br>
As used herein, the term "RNA detection structure" refers to a structure formed by hybridizing a nucleic acid (e.g:, an oligonucleotide) to an RNA target, e.g., an miRNA or siRNA. In some embodiments, the nucleic acid is a single nucleic acid (e.g., a larger nucleic acid with a small region (or regions) of homology to the miRNA). In other embodiments, the nucleic acid comprises two nucleic acids (e.g., that hybridize to the miRNA to form a hairpin (e.g., single or double hairpin) structure). In preferred embodiments, miRNA detection structures are capable of detection using known nucleic acid detection methods, including, but not limited to, those disclosed herein.<br>
In some embodiments, RNA detection structures are further modified following the hybridization step. For example, in some embodiments, the nucleic acid that is hybridized to the RNA provides a template for extension of the RNA by a nucleic acid polymerase. The RNA is hybridized to the nucleic acid and is then extended using the polymerase. In other embodiments, the nucleic acid serves as a template for the hybridization and ligation of additional nucleic acids to the RNA.<br>
The term "siRNAs" refers to short interfering RNAs. In some embodiments, siRNAs comprise a duplex, or double-stranded region, where each strand of the double-<br><br>
stranded region is about 18 to 25 nucleotides long; the double-stranded region can be as short as 16, and as long as 29, base pairs long, where the length is determined by the antisense strand. Often siRNAs contain from about two to four unpaired nucleotides at the 3' end of each strand. SiRNAs appear to function as key intermediates in triggering RNA interference in invertebrates and in vertebrates, and in triggering sequence-specific RNA degradation during posttranscriptional gene silencing in plants. At least one strand of the duplex or double-stranded region of a siRNA is substantially homologous to or substantially complementary to a target RNA molecule. The strand complementary to a target RNA molecule is the "antisense" strand; the strand homologous to the target RNA molecule is the "sense" strand and is also complementary to the siRNA antisense strand. One strand of the double stranded region need not be the exact length of the opposite strand' thus, one strand may have at least one fewer nucleotides than the opposite complementary strand, resulting in a "bubble" or at least one unmatched base in the opposite strand. One strand of the double-stranded region need not be exactly complementary to the opposite strand; thus, the strand, preferably the sense strand, may have at least one mismatched base pair.<br>
SiRNAs may also contain additional sequences; non-limiting examples of such sequences include linking sequences, or loops, which connect the two strands of the duplex region. This form of siRNAs may be referred to "si-like RNA", "short hairpin siRNA" where the short refers to the duplex region of the siRNA, or "hairpin siRNA". Additional non-limiting examples of additional sequences present in siRNAs include stem and other folded structures. The additional sequences may or may not have known functions; non-limiting examples of such functions include increasing stability of an siRNA molecule, or providing a cellular destination signal.<br>
As used herein, the terms "subject" and "patient" refer to any organisms including plants, microorganisms and animals (e.g., mammals such as dogs, cats, livestock, and humans).<br>
As used herein, the term "INVADER assay reagents" or "invasive cleavage assay reagents" refers to one or more reagents for detecting target sequences, said reagents comprising oligonucleotides capable of forming an invasive cleavage structure in the presence of the target sequence. Li some embodiments, the INVADER assay reagents<br><br>
further comprise an agent for detecting the presence of an invasive cleavage structure (e.g., a cleavage agent). la some embodiments, the oligonucleotides comprise first and second oligonucleotides, said first oligonucleotide comprising a 5' portion complementary to a first region of the target nucleic acid and said second oligonucleotide comprising a 3' portion and a 5' portion, said 5' portion complementary to a second region of the target nucleic acid downstream of and contiguous to the first portion. In some embodiments, the 3' portion of the second oligonucleotide comprises a 3' terminal nucleotide not complementary to the target nucleic acid. In preferred embodiments, the 3' portion of the second oligonucleotide consists of a single nucleotide not complementary to the target nucleic acid. In some embodiments, the first and second oligonucleotides are covalentiy coupled to one another (e.g., through a linker).<br>
la some embodiments, the INVADER assay reagents further comprise a solid support. For example, in some embodiments, the one or more oligonucleotides of the assay reagents (e.g., first and/or second oligonucleotide, whether bridging or non-bridging) is attached to the solid support In some embodiments, the INVADER assay reagents further comprise a buffer solution. In some preferred embodiments, the buffer<br>
solution comprises a source of divalent cations (e.g., Mn2+ and/or Mg^4" ions). Individual ingredients (e.g., oligonucleotides, enzymes, buffers, target nucleic acids) that collectively make up INVADER assay reagents are termed "INVADER assay reagent components."<br>
In some embodiments, the INVADER assay reagents further comprise a third oligonucleotide complementary to a third portion of the target nucleic acid upstream of the first portion of the first target nucleic acid. In yet other embodiments, the INVADER assay reagents further comprise a target nucleic acid. In some embodiments, the INVADER assay reagents further comprise a second target nucleic acid. In yet other embodiments, the INVADER assay reagents further comprise a third oligonucleotide comprising a 5' portion complementary to a first region of the second target nucleic acid. In some specific embodiments, the 3' portion of the third oligonucleotide is covalently linked to the second target nucleic acid. In other specific embodiments, the second target nucleic acid further comprises a 5' portion, wherein the 5" portion of the second target<br><br>
nucleic acid is the third oligonucleotide. In still other embodiments, the INVADER assay reagents further comprise an ARRESTOR molecule (e.g., ARRESTOR oligonucleotide).<br>
The inclusion of 2' O-methylated ARRESTOR oligonucleotides, which are base-paired fully to each probe's target-specific region and partially to its 5'-flap region, sequesters uncleaved probes and prevents X-structure formation in the secondary reaction, as described in Eis et al, Nature Biotechnology, 19:673-676 (2001), herein incorporated by reference in its entirety for all purposes.<br>
In some preferred embodiments, the INVADER assay reagents further comprise reagents for detecting a nucleic acid cleavage product. In some embodiments, one or more oligonucleotides in the INVADER assay reagents comprise a label. In some preferred embodiments, said first oligonucleotide comprises a label. In other preferred embodiments, said third oligonucleotide comprises a label. In particularly preferred embodiments, the reagents comprise a first and/or a third oligonucleotide labeled with moieties that produce a fluorescence resonance energy transfer (FRET) effect.<br>
In some embodiments one or more the INVADER assay reagents may be provided in a predispensed format (i.e., premeasured for use in a step of the procedure without re-measurement or re-dispensing). In some embodiments, selected INVADER assay reagent components are mixed and predispensed together. In preferred embodiments, predispensed assay reagent components are predispensed and are provided in a reaction vessel (including but not limited to a reaction tube or a well, as in, e.g., a microtiter plate). In particularly preferred embodiments, predispensed INVADER assay reagent components are dried down (e.g., desiccated or lyophilized) in a reaction vessel.<br>
In some embodiments, the INVADER assay reagents are provided as a kit. As used herein, the term "kit" refers to any delivery system for delivering materials. In the context of reaction assays, such delivery systems include systems that allow for the storage, transport, or delivery of reaction reagents (e.g., oligonucleotides, enzymes, etc. in the appropriate containers) and/or supporting materials (e.g., buffers, written instructions for performing the assay etc.) from one location to another. For example, kits include one or more enclosures (e.g., boxes) containing the relevant reaction reagents and/or supporting materials. As used herein, the term "fragmented kit" refers to delivery systems comprising two or more separate containers that each contains a subportion of<br><br>
the total kit components. The containers may be delivered to the intended recipient together or separately. For example, a first container may contain an enzyme for use in an assay, while a second container contains oligonucleotides. The term "fragmented kit" is intended to encompass kits containing Analyte specific reagents (ASR's) regulated under section 520(e) of the Federal Food, Drug, and Cosmetic Act, but are not limited thereto. Indeed, any delivery system comprising two or more separate containers that each contains a subportion of the total kit components are included in the term "fragmented kit." In contrast, a "combined kit" refers to a delivery system containing all of the components of a reaction assay in a single container (e.g., in a single box housing each of the desired components). The term "kit" includes both fragmented and combined kits.<br>
In some embodiments, the present invention provides INVADER assay reagent kits comprising one or more of the components necessary for practicing the present invention. For example, the present invention provides kits for storing or delivering the enzymes and/or the reaction components necessary to practice an INVADER assay. The kit may include any and all components necessary or desired for assays including, but not limited to, the reagents themselves, buffers, control reagents (e.g., tissue samples, positive and negative control target oligonucleotides, etc.), solid supports, labels, written and/or pictorial instructions and product information, inhibitors, labeling and/or detection reagents, package environmental controls (e.g., ice, desiccants, etc.), and the like, hi some embodiments, the kits provide a sub-set of the required components, wherein it is expected that the user will supply the remaining components, hi some embodiments, the kits comprise two or more separate containers wherein each container houses a subset of the components to be delivered. For example, a first container (e.g., box) may contain an enzyme (e.g., structure specific cleavage enzyme in a suitable storage buffer and container), while a second box may contain oligonucleotides (e.g., INVADER oligonucleotides, probe oligonucleotides, control target oligonucleotides, etc.).<br>
The term "label" as used herein refers to any atom or molecule that can be used to provide a detectable (preferably quantifiable) effect, and that can be attached to a nucleic acid or protein. Labels include, but are not limited to, dyes; radiolabels such as 32P; binding moieties such as biotin; haptens such as digoxgenin; luminogenic,<br><br>
phosphorescent or fluorogenic moieties; mass tags; and fluorescent dyes alone or in combination with moieties that can suppress or shift emission spectra by fluorescence resonance energy transfer (FRET). Labels may provide signals detectable by fluorescence, radioactivity, colorimetry, gravimetry, X-ray diffraction or absorption, magnetism, enzymatic activity, characteristics of mass or behavior affected by mass (e.g., MALDI time-of-flight mass spectrometry; fluorescence polarization), and the like. A label may be a charged moiety (positive or negative charge) or alternatively, may be charge neutral. Labels can include or consist of nucleic acid or protein sequence, so long as the sequence comprising the label is detectable.<br>
As used herein, the term "distinct" in reference to signals refers to signals that can be differentiated one from another, e.g., by spectral properties such as fluorescence emission wavelength, color, absorbance, mass, size, fluorescence polarization properties, charge, etc., or by capability of interaction with another moiety, such as with a chemical reagent, an enzyme, an antibody, etc.<br>
As used herein, the terms "complementary" or "complementarity" are used hi reference to polynucleotides (i.e., a sequence of nucleotides such as an oligonucleotide or a target nucleic acid) related by the base-pairing rules. For example, for the sequence " 5'-A-G-T-3'," is complementary to the sequence " 3'-T-C-A-5'." Complementarity may be "partial," in which only some of the nucleic acids' bases are matched according to the base pairing rules. Or, there may be "complete" or "total" complementarity between the nucleic acids. The degree of complementarity between nucleic acid strands has significant effects on the efficiency and strength of hybridization between nucleic acid strands. This is of particular importance in amplification reactions, as well as detection methods that depend upon binding between nucleic acids. Either term may also be used in reference to individual nucleotides, especially within the context of polynucleotides. For example, a particular nucleotide within an oligonucleotide may be noted for its complementarity, or lackjhereof, to a nucleotide within another nucleic acid strand, in contrast or comparison to the complementarity between the rest of the oligonucleotide and the nucleic acid strand.<br><br>
The term "homology" and "homologous" refers to a degree of identity. There may be partial homology or complete homology. A partially homologous sequence is one that is less than 100% identical to another sequence.<br>
As used herein, the term "hybridization" is used in reference to the pairing of complementary nucleic acids. Hybridization and the strength of hybridization (i.e.t the strength of the association between the nucleic acids) is influenced by such factors as the degree of complementary between the nucleic acids, stringency of the conditions involved, and the Tm of the formed hybrid. "Hybridization" methods involve the annealing of one nucleic acid to another, complementary nucleic acid, i.e., a nucleic acid having a complementary nucleotide sequence. The ability of two polymers of nucleic acid containing complementary sequences to find each other and anneal through base pairing interaction is a well-recognized phenomenon. The initial observations of the "hybridization" process by Marmur and Lane, Proc. Natl. Acad. Sci. USA 46:453 (1960) and Doty et al., Proc. Natl. Acad. Sci. USA 46:461 (1960) have been followed by the refinement of this process into an essential, tool of modern biology.<br>
The complement of a nucleic acid sequence as used herein refers to an oligonucleotide which, when aligned with the nucleic acid sequence such that the 5' end of one sequence is paired with the 3' end of the other, is in "antiparallel association." Certain bases not commonly found in natural nucleic acids may be included in the nucleic acids of the present invention and include, for example, inosine and 7-deazaguanine. Complementarity need not be perfect; stable duplexes may contain mismatched base pairs or unmatched bases. Those skilled in the art of nucleic acid technology can determine duplex stability empirically considering a number of variables including, for example, the length of the oligonucleotide, base composition and sequence of the oligonucleotide, ionic strength and incidence of mismatched base pairs.<br>
As used herein, the term "Tm" is used in reference to the "melting temperature." The melting temperature is the temperature at which a population of double-stranded nucleic acid molecules becomes half dissociated into single strands. Several equations for calculating the Tm of nucleic acids are well known in the art. As indicated by standard references, a simple estimate of the Tm value may be calculated by the equation: Tm = 81.5 + 0.41(% G + C), when a nucleic acid is in aqueous solution at 1 M NaCl (see<br><br>
e.g., Anderson and Young, Quantitative Filter Hybridization, in Nucleic Acid Hybridization (1985). Other references (e.g., Allawi and SantaLucia, Biochemistry 36: 10581-94 (1997) include more sophisticated computations which take structural and environmental, as well as sequence characteristics into account for the calculation of Tm.<br>
The term "gene" refers to a DNA sequence that comprises control and coding sequences necessary for the production of an KNA having a non-coding function (e.g., a ribosomal or transfer RNA), a polypeptide or a precursor. The RNA or polypeptide can be encoded by a full length coding sequence or by any portion of the coding sequence so long as the desired activity or function is retained.<br>
The term "wild-type" refers to a gene or a gene product that has the characteristics of that gene or gene product when isolated from a naturally occurring source. A wild-type gene is that which is most frequently observed in a population and is thus arbitrarily designated the "normal" or "wild-type" form of the gene. In contrast, the term "modified," "mutant," or "polymorphic" refers to a gene or gene product that displays modifications hi sequence and or functional properties (i.e., altered characteristics) when compared to the wild-type gene or gene product. It is noted that naturally-occurring mutants can be isolated; these are identified by the fact that they have altered characteristics when compared to the wild-type gene or gene product.<br>
The term "recombinant DNA vector" as used herein refers to DNA sequences containing a desired heterologous sequence. For example, although the term is not limited to the use of expressed sequences or sequences that encode an expression product, in some embodiments, the heterologous sequence is a coding sequence and appropriate DNA sequences necessary for either the replication of the coding sequence in a host organism, or the expression of the operably linked coding sequence in a particular host organism. DNA sequences necessary for expression in prokaryotes include a promoter, optionally an operator sequence, a ribosome binding site and possibly other sequences. Eukaryotic cells are known to utilize promoters, polyadenlyation signals and enhancers.<br>
The term "oligonucleotide" as used herein is defined as a molecule comprising two or more deoxyribonucleotides or ribonucleotides, preferably at least 5 nucleotides, more preferably at least about 10-15 nucleotides and more preferably at least about 15 to 30 nucleotides. The exact size will depend on many factors, which in turn depend on the<br><br>
ultimate function or use of the oligonucleotide. The oligonucleotide may be generated in any manner, including chemical synthesis, DNA replication, reverse transcription, PCR, or a combination thereof.<br>
Because mononucleotides are reacted to make oligonucleotides in a manner such that the 5' phosphate of one mononucleotide pentose ring is attached to the 3' oxygen of its neighbor in one direction via a phosphodiester linkage, an end of an oligonucleotide is referred to as the "51 end" if its 5' phosphate is not linked to the 3' oxygen of a mononucleotide pentose ring and as the "3' end" if its 3' oxygen is not linked to a 5' phosphate of a subsequent mononucleotide pentose ring. As used herein, a nucleic acid sequence, even if internal to a larger oligonucleotide, also may be said to have 5' and 3' ends. A first region along a nucleic acid strand is said to be upstream of another region if the 3' end of the first region is before the 5' end of the second region when moving along a strand of nucleic acid in a 5' to 3' direction.<br>
When two different, non-overlapping oligonucleotides anneal to different regions of the same linear complementary nucleic acid sequence, and the 3' end of one oligonucleotide points towards the 5' end of the other, the former may be called the "upstream" oligonucleotide and the latter the "downstream" oligonucleotide. Similarly, when two overlapping oligonucleotides are hybridized to the same linear complementary nucleic acid sequence, with the first oligonucleotide positioned such that its 5' end is upstream of the 5* end of the second oligonucleotide, and the 3' end of the first oligonucleotide is upstream of the 3* end of the second oligonucleotide, the first oligonucleotide may be called the "upstream" oligonucleotide and the second oligonucleotide may be called the "downstream" oligonucleotide.<br>
The term "primer" refers to an oligonucleotide that is capable of acting as a point of initiation of synthesis when placed under conditions in which primer extension is initiated. An oligonucleotide "primer" may occur naturally, as in a purified restriction digest or may be produced synthetically.<br>
A primer is selected to be "substantially complementary to a strand of specific sequence of the template. A primer must be sufficiently complementary to hybridize with a template strand for primer elongation to occur. A primer sequence need not reflect the exact sequence of the template. For example, a non-complementary nucleotide<br><br>
fragment may be attached to the 5' end of the primer, with the remainder of the primer sequence being substantially complementary to the strand. Non-complementary bases or longer sequences can be interspersed into the primer, provided that the primer sequence has sufficient complementarity with the sequence of the template to hybridize and thereby form a template primer complex for synthesis of the extension product of the primer.<br>
The term "cleavage structure" as used herein, refers to a structure that is formed by the interaction of at least one probe oligonucleotide and a target nucleic acid, forming a structure comprising a duplex, the resulting structure being cleavable by a cleavage means, including but not limited to an enzyme. The cleavage structure is a substrate for specific cleavage by the cleavage means in contrast to a nucleic acid molecule that is a substrate for non-specific cleavage by agents such as phosphodiesterases, which cleave nucleic acid molecules without regard to secondary structure (i.e., no formation of a duplexed structure is required).<br>
The term "cleavage means" or "cleavage agent" as used herein refers to any means that is capable of cleaving a cleavage structure, including but not limited to enzymes. "Structure-specific nucleases" or "structure-specific enzymes" are enzymes that recognize specific secondary structures in a nucleic molecule and cleave these structures. The cleavage means of the invention cleave a nucleic acid molecule in response to the formation of cleavage structures; it is not necessary that the cleavage means cleave the cleavage structure at any particular location within the cleavage structure.<br>
The cleavage means may include nuclease activity provided from a variety of sources including the CLEAVASE enzymes (Third Wave Technologies, Madison, WI), the FEN-1 endonucleases (including RAD2 and XPG proteins), Taq DNA polymerase and E. coli DNA polymerase I. The cleavage means may include enzymes having 5' nuclease activity (e.g., Taq DNA polymerase (DNAP), E. coli DNA polymerase I). The cleavage means may also include modified DNA polymerases having 5' nuclease activity but lacking synthetic activity. Examples of cleavage means suitable for use in the methods and kits of the present invention are provided in U.S. Patent Nos. 5,614,402;<br><br>
5,795,763; 5,843,669; PCX Appln. Nos WO 98/23774; WO 02/070755A2; and WO0190337A2, each of which is herein incorporated by reference it their entireties.<br>
The term "thermostable" when used in reference to an enzyme, such as a 5' nuclease, indicates that the enzyme is functional or active (i.e., can perform catalysis) at an elevated temperature, i.e., at about 55°C or higher (e.g., including, but not limited to, 60°C, 65°C, 70°C, 75°C, 80°C, 85°C or 90°C).<br>
The term "cleavage products" as used herein, refers to products generated by the reaction of a cleavage means with a cleavage structure (i.e., the treatment of a cleavage structure with a cleavage means).<br>
The term "non-target cleavage product" refers to a product of a cleavage reaction that is not derived from the target nucleic acid. As discussed above, in some of the methods of the present invention, cleavage of the cleavage structure generally occurs within the probe oligonucleotide. The fragments of the probe oligonucleotide generated by this target nucleic acid-dependent cleavage are "non-target cleavage products."<br>
The term "probe oligonucleotide", in the context of an INVADER assay reaction, refers to an oligonucleotide that interacts with a target nucleic acid to form a cleavage structure in the presence or absence of an INVADER oligonucleotide. When annealed to the target nucleic acid, the probe oligonucleotide and target form a cleavage structure and cleavage occurs within the probe oligonucleotide.<br>
The term "INVADER oligonucleotide" refers to an oligonucleotide that hybridizes to a target nucleic acid at a location near the region of hybridization between a probe and the target nucleic acid, wherein the INVADER oligonucleotide comprises a portion (e.g., a chemical moiety, or nucleotide-whether complementary to that target or not) that overlaps with the region of hybridization between the probe and target. In some embodiments, the INVADER oligonucleotide contains sequences at its 3' end that are substantially the same as sequences located at the 5" end of a probe oligonucleotide.<br>
The term "ARRESTOR molecule" refers to an agent added to or included in an invasive cleavage reaction in order to stop one or more reaction components from participating in a subsequent action or reaction. This may be done by sequestering or inactivating some reaction component (e.g., by binding or base-pairing a nucleic acid component, or by binding to a protein component). The term "ARRESTOR<br><br>
oligonucleotide" refers to an oligonucleotide included in an invasive cleavage reaction in order to stop or arrest one or more aspects of any reaction (e.g., the first reaction and/or any subsequent reactions or actions; it is not intended that the AKRESTOR oligonucleotide be limited to any particular reaction or reaction step). This may be done by sequestering some reaction component (e.g., base-pairing to another nucleic acid, or binding to a protein component). However, it is not intended that the term be so limited as to just situations in which a reaction component is sequestered.<br>
The term "cassette" as used herein refers to an oligonucleotide or combination of oligonucleotides configured to generate a detectable signal in response to cleavage of a probe oligonucleotide in an INVADER assay. In preferred embodiments, the cassette hybridizes to a non-target cleavage product from cleavage of the probe oligonucleotide to form a second invasive cleavage structure, such that the cassette can then be cleaved.<br>
Secondary cleavage reactions in some preferred embodiments of the present invention include the use of FRET cassettes. Such molecules provide both a secondary target (Secondary Reaction Target or SRT) and a FRET labeled cleavable sequence, allowing homogeneous detection (i.e., without product separation or other manipulation after the reaction) of the sequential invasive cleavage reaction. Other preferred embodiments use a secondary reaction system in which the FRET probe and synthetic target are provided as separate oligonucleotides. The cleaved 5'-flaps from a primary reaction act as invasive oligonucleotides in a secondary reaction, in which they bind to the appropriate secondary-reaction target (SRT).<br>
In some embodiments, the cassette is a single oligonucleotide comprising a hairpin portion (i.e., a region wherein one portion of the cassette oligonucleotide hybridizes to a second portion of the same oligonucleotide under reaction conditions, to form a duplex). In other embodiments, a cassette comprises at least two oligonucleotides comprising complementary portions that can form a duplex under reaction conditions. In preferred embodiments, the cassette comprises a label. In particularly preferred embodiments, cassette comprises labeled moieties that produce a fluorescence resonance energy transfer (FRET) effect.<br>
The term "substantially single-stranded" when used in reference to a nucleic acid substrate means that the substrate molecule exists primarily as a single strand of nucleic<br><br>
acid in contrast to a double-stranded substrate which exists as two strands of nucleic acid which are held together by inter-strand base pairing interactions.<br>
As used herein, the phrase "non-amplified oligonucleotide detection assay" refers to a detection assay configured to detect the presence or absence of a particular target sequence (e.g., miKNA, SNP, repeat sequence, etc.) that has not been amplified (e.g., by PCR), without creating copies of the target sequence. A "non-amplified oligonucleotide detection assay may, for example, amplify a signal used to indicate the presence or absence of a particular target sequence or polymorphism within a target sequence, so long as the target sequence is not copied.<br>
As used herein, the phrase "non-amplifying oligonucleotide detection assay" refers to a detection assay configured to detect the presence or absence of a target sequence (e.g., miRNA, SNP, repeat sequence, etc.), without creating copies of the target sequence. A "non-amplifying oligonucleotide detection assay" may, for example, amplify a signal used to indicate the presence or absence of a particular target sequence or polymorphism in a target sequence, so long as the target sequence is not copied.<br>
The term "sequence variation" as used herein refers to differences in nucleic acid sequence between two nucleic acids. For example, a wild-type structural gene and a mutant form of this wild-type structural gene may vary in sequence by the presence of single base substitutions and/or deletions or insertions of one or more nucleotides. These two forms of the structural gene are said to vary in sequence from one another. A second mutant form of the structural gene may exist This second mutant form is said to vary in sequence from bom the wild-type gene and the first mutant form of the gene.<br>
The term "liberating" as used herein refers to the release of a nucleic acid fragment from a larger nucleic acid fragment, such as an oligonucleotide, by the action of, for example, a 5' nuclease such that the released fragment is no longer covalently attached to the remainder of the oligonucleotide.<br>
The term "Km" as used herein refers to the Michaelis-Menten constant for an enzyme and is defined as the concentration of the specific substrate at which a given enzyme yields one-half its maximum velocity in an enzyme catalyzed reaction.<br>
The term "nucleotide analog" as used herein refers to modified or non-naturally occurring nucleotides including but not limited to analogs that have altered stacking<br><br>
interactions such as 7-deazapurines (i.e., 7-deaza-dATP and 7-deaza-dGTP); base analogs with alternative hydrogen bonding configurations (e.g., such as Iso-C and Iso-G and other non-standard base pairs described in U.S. Patent No. 6,001,983 to S. Benner and herein incorporated by reference); non-hydrogen bonding analogs (e.g., non-polar, aromatic nucleoside analogs such as 2,4-difluorotoluene, described by B.A. Schweitzer and E.T. Kool, J. Org. Chem., 1994, 59,7238-7242, B.A. Schweitzer and E.T. Kool, J. Am. Chem. Soc., 1995,117,1863-1872; each of which is herein incorporated by reference); "universal" bases such as 5-nitroindole and 3-nitropyrrole; and universal purines and pyrimidines (such as "K" and "P" nucleotides, respectively, P. Kong, et al, Nucleic Acids Res., 1989,17,10373-10383, P. Kong et al, Nucleic Acids Res., 1992, 20,5149-5152). Nucleotide analogs include nucleotides having modification on the sugar moiety, such as dideoxy nucleotides and 2-O-mettiyl nucleotides. Nucleotide analogs include modified forms of deoxyribonucleotides as well as ribonucleotides.<br>
The term "polymorphic locus" is a locus present in a population that shows variation between members of the population (e.g., the most common allele has a frequency of less than 0.95). In contrast, a "monomorphic locus" is a genetic locus at little or no variations seen between members of the population (generally taken to be a locus at which the most common allele exceeds a frequency of 0.95 in the gene pool of the population).<br>
The term "microorganism" as used herein means an organism too small to be observed with the unaided eye and includes, but is not limited to bacteria, virus, protozoans, fungi, and ciliates.<br>
The term "microbial gene sequences" refers to gene sequences derived from a microorganism.<br>
The term "bacteria" refers to any bacterial species including eubacterial and archaebacterial species.<br>
The term "virus" refers to obligate, ultramicroscopic, intracellular parasites incapable of autonomous replication (i.e., replication requires the use of the host cell's machinery).<br>
The term "sample" in the present specification and claims is used in its broadest sense. On the one hand it is meant to include a specimen or culture (e.g., microbiological<br><br>
cultures). On the other hand, it is meant to include both biological and environmental samples. A sample may include a specimen of synthetic origin.<br>
Biological samples may be animal, including human, fluid, solid (e.g., stool) or tissue, as well as liquid and solid food and feed products and ingredients such as dairy items, vegetables, meat and meat by-products, and waste. Biological samples may be obtained from all of the various families of domestic animals, as well as feral or wild animals, including, but not limited to, such animals as ungulates, bear, fish, lagomorphs, rodents, etc.<br>
Environmental samples include environmental material such as surface matter, soil, water and industrial samples, as well as samples obtained from food and dairy processing instruments, apparatus, equipment, utensils, disposable and non-disposable items. These examples are not to be construed as limiting the sample types applicable to the present invention.<br>
The term "source of target nucleic acid" refers to any sample that contains nucleic acids (RNA (e.g., mIRNA) or DNA). Particularly preferred sources of target nucleic acids are biological samples including, but not limited to blood, saliva, cerebral spinal fluid, pleura! fluid, milk, lymph, sputum and semen.<br>
An oligonucleotide is said to be present hi "excess" relative to another oligonucleotide (or target nucleic acid sequence) if that oligonucleotide is present at a higher molar concentration that the other oligonucleotide (or target nucleic acid sequence). When an oligonucleotide such as a probe oligonucleotide is present in a cleavage reaction in excess relative to the concentration of the complementary target nucleic acid sequence, the reaction may be used to indicate the amount of the target nucleic acid present. Typically, when present in excess, the probe oligonucleotide will be present hi at least a 100-fold molar excess; typically at least 1 pmole of each probe oligonucleotide would be used when the target nucleic acid sequence was present at about 10 finoles or less.<br>
A sample "suspected of containing" a first and a second target nucleic acid may contain either, both or neither target nucleic acid molecule.<br>
The term "reactant" is used herein in its broadest sense. The reactant can comprise, for example, an enzymatic reactant, a chemical reactant or light (e.g.,<br><br>
ultraviolet light, particularly short wavelength ultraviolet light is known to break oligonucleotide chains). Any agent capable of reacting with an oligonucleotide to either shorten (e.g., cleave) or elongate (he oligonucleotide is encompassed within the term "reactant."<br>
As used herein, the term "purified" or "to purity1 refers to the removal of contaminants from a sample. For example, in some embodiments, recombinant CLEAVASE nucleases are expressed in bacterial host cells and the nucleases are purified by the removal of host cell proteins; the percent of these recombinant nucleases is thereby increased in the sample.<br>
As used herein the term "portion" when in reference to a protein (as in "a portion of a given protein") refers to fragments of that protein. The fragments may range in size from four ammo acid residues to the entire amino acid sequence minus one ammo acid (e.g., 4, 5,6,..., n-1).<br>
The term "nucleic acid sequence" as used herein refers to an oligonucleotide, nucleotide or polynucleotide, and fragments or portions thereof, and to DNA or RNA of genomic or synthetic origin, which may be single or double stranded, and represent the sense or antisense strand. Similarly, "amino acid sequence" as used herein refers to peptide or protein sequence.<br>
. As used herein, the terms "purified" or "substantially purified" refer to molecules, either nucleic or amino acid sequences, that are removed from their natural environment, isolated or separated, and are at least 60% free, preferably 75% free, and most preferably 90% free from other components with which they are naturally associated. An "isolated polynucleotide" or "isolated oligonucleotide" is therefore a substantially purified polynucleotide.<br>
The term "continuous strand of nucleic acid" as used herein is means a strand of nucleic acid that has a continuous, covalently linked, backbone structure, without nicks or other disruptions. The disposition of the base portion of each nucleotide, whether base-paired, single-stranded or mismatched, is not an element in the definition of a continuous strand. The backbone of the continuous strand is not limited to the ribose-phosphate or deoxyribose-phosphate compositions that are found in naturally occurring, unmodified nucleic acids. A nucleic acid of the present invention may<br><br>
comprise modifications in the structure of the backbone, including but not limited to phosphorothioate residues, phosphonate residues, 2' substituted ribose residues (e.g., 2'-O-methyl ribose) and alternative sugar (e.g., arabinose) containing residues.<br>
The term "continuous duplex" as used herein refers to a region of double stranded nucleic acid in which there is no disruption in the progression of basepairs within the duplex (i.e., the base pairs along the duplex are not distorted to accommodate a gap, bulge or mismatch with the confines of the region of continuous duplex). As used herein the term refers only to the arrangement of the basepairs within the duplex, without implication of continuity in the backbone portion of the nucleic acid strand. Duplex nucleic acids with uninterrupted basepairing, but with nicks in one or both strands are within the definition of a continuous duplex.<br>
The term "duplex" refers to the state of nucleic acids in which the base portions of the nucleotides on one strand are bound through hydrogen bonding the their complementary bases arrayed on a second strand. The condition of being in a duplex form reflects on the state of the bases of a nucleic acid. By virtue of base pairing, the strands of nucleic acid also generally assume the tertiary structure of a double helix, having a major and a minor groove. The assumption of the helical form is implicit in the act of becoming duplexed.<br>
The term "template" refers to a strand of nucleic acid on which a complementary copy is built from nucleoside triphosphates through the activity of a template-dependent nucleic acid polymerase. Within a duplex the template strand is, by convention, depicted and described as the "bottom" strand. Similarly, the non-template strand is often depicted and described as the "top" strand.<br>
DESCRIPTION OF THE INVENTION<br>
The present invention relates to compositions and methods for the detection and characterization of micro RNAs (miRNAs) or other short nucleic acid molecules, e.g. siRNAs, The present invention provides improved methods of detecting, characterizing and quantitating miRNA expression. In some embodiments, the present invention provides methods of detecting miRNA expression comprising adding a nucleic acid to a miRNA to aid in detection. The resulting "miRNA detection structure" is then detected<br><br>
using any suitable method including, but not limited to, those disclosed herein. While the following description focuses on the detection and quantisation of miRNAs, it should be understood that the invention also finds use with other short nucleic acid molecules (e.g., DNA and RNA of less than, for example, 50,40,30, or 20 nucleotides in length).<br>
I.        Formation of a miRNA Detection Structure<br>
In some embodiments, the present invention provides methods of generating miRNA detection structures to aid in the detection of miRNAs. miRNAs are small in size (approximately 21 nucleotides) and are thus difficult to detect using standardized hybridization methods. In some embodiments, the methods of the present invention comprise adding a nucleic acid molecule to an miRNA (e.g., via hybridization, extension, or ligation) to generate a detection structure. Such detection structures can then be detected using any suitable method.<br>
In one particular embodiment, the detection structure described in Figure 2 is generated for detection of miRNAs. In this embodiment, two oligonucleotides are annealed to the miRNA to form a double loop or "dumbbell" like structure. The dumbbell structure creates a larger region of double-stranded nucleic acid by extending the ends of the miRNA with a double-stranded region of oligonucleotide. In some embodiments, each end of the miRNA is extended between 2 and 5 nucleotides. In some embodiments, the ends of the oligonucleotides comprise additional nucleic acid sequences that do not hybridize to the miRNA. In some embodiments, these additional sequences form invasive cleavage structures (e.g.t INVADER assay invasive cleavage structures). In some embodiments, invasive cleavage structures are detected by the INVADER assay (See e,g., below description).<br>
In other embodiments, the detection structure described in Figure 3 is generated for the detection of miRNAs. In this embodiment, one oligonucleotide is annealed to the miRNA to generate an arched structure. The miRNA brings the ends of the oligonucleotide together with greater efficiency than in ±e absence of the miRNA. In some embodiments, the ends of the oligonucleotide comprise additional sequences that extend beyond the miRNA and do not hybridize to the miRNA. In some embodiments, these additional sequences form invasive cleavage structures (e.g., INVADER assay<br><br>
invasive cleavage structures). In some embodiments, invasive cleavage structures are detected by the INVADER assay (See e.g., below description). In other embodiments, following cleavage of an INVADER assay invasive cleavage structure, the resulting ends are ligated to form a circular structure. In other embodiments, one oligonucleotide is hybridized to a miRNA such that the ends of the oligonucleotide are brought in close proximity (e.g., hybridized to adjacent nucleotides of the miRNA) and are then ligated.<br>
In still further embodiments, the detection structures described in Figures 16 and 17 are generated. In this embodiment, either probe or INVADER oligonucleotides are extended to create a single hairpin loop or "half dumbbell" structure. In some embodiments, the ends of the oligonucleotides comprise additional nucleic acid sequences that do not hybridize to the miRNA. In some embodiments, these additional sequences form invasive cleavage structures (e.g., INVADER assay invasive cleavage structures). In some embodiments, invasive cleavage structures are detected by the INVADER assay (See e.g., below description).<br>
In other embodiments, these additional sequences are complementary to additional oligonucleotides added to reaction mixtures to stabilize a cleavage structure, e.g. an INVADER assay invasive cleavage structure (Figure 4).<br>
In some embodiments, circular structures generated as described above are detected using a rolling circle replication assay (See e.g., below description of rolling circle replication).<br>
In still further embodiments, detection structures are generated from long oligonucleotides (e.g., greater than 50,100,1000 or more nucleotides) with short region(s) of homology to miRNAs. One or more miRNAs are hybridized to the oligonucleotides to generate detection structures. In some embodiments, these detection structures are detected by extension of miRNAs (e.g., via ligation or polymerization reactions such as RT-PCR). In some embodiments, these detection structures are further detected by hybridization to oligonucleotides conjugated to solid supports, such as microspheres, or other surfaces or structures.<br>
In some embodiments, oligonucleotides used to form detection structures comprise one or more nucleotide analogs. For example, in some embodiments, 2'-O-methyl nucleotides are utilized. The present invention is not limited to a particular<br><br>
mechanism. Indeed, an understanding of the mechanism is not necessary to practice the present invention. Nonetheless, it is contemplated that the presence of 2'-O-methyl bases increases the stability of the hybridized detection structure and aids in further detection methods.<br>
H.       Detection of interfering RNAs<br>
hi some embodiments, the present invention provides methods of detecting miRNAs. The present invention is not limited to a particular detection assay. Any suitable method maybe utilized including, but not limited to, those disclosed herein.<br>
In some preferred embodiments of the present invention, miRNA detection methods are quantitative. The present invention is not limited to a particular mechanism. Indeed, an understanding of the mechanism is not necessary to practice the present invention. Nonetheless, it is contemplated that levels of a particular miRNA in the body are associated with a level of gene expression from their cognate genes. The present invention thus provides methods of correlated miRNAs with gene expression of particular genes (e.g., genes involved in disease states or metabolism). For example, in some embodiments, the methods of the present invention are utilized to determine the presence of abnormal (e.g., high or low) levels of a particular miRNA or to determine the effect of an intervention (e.g., drug) on miRNA expression. In other embodiments, heterologous miRNAs (e.g., from expression vectors, transgenic constructs, transfection, etc.) are detected to characterize the efficiency of miRNA expression systems.<br>
In some embodiments, the present invention provides methods of detecting a particular miRNA (e.g., a miRNA such as mir-1 or mir-135). hi other embodiments, the methods of the present invention are used to distinguish between variants (e.g., polymorphisms or mutations) in a particular miRNA. In still further embodiments, the present invention provides methods of lysing cells to be tested for the presence of miRNAs.<br>
A.       INVADER Assay<br>
hi some embodiments, the INVADER assay is used for the detection of miRNAs. hi some embodiments, the INVADER assay comprises forming a nucleic acid cleavage<br><br>
structure that is dependent upon the presence of a target nucleic acid and cleaving the nucleic acid cleavage structure so as to release distinctive cleavage products. 5' nuclease activity, for example, is used to cleave the target-dependent cleavage structure and the resulting cleavage products or the cleavage of the cleavage structure is indicative of the presence of specific target nucleic acid sequences in the sample. When one or two (or more) strands of nucleic acid, or oligonucleotides, both hybridize to a target nucleic acid strand such that they form an overlapping invasive cleavage structure, as described below, invasive cleavage can occur. Through the interaction of a cleavage agent (e.g., a 5' nuclease) and the upstream oligonucleotide, the cleavage agent can be made to cleave the downstream oligonucleotide at an internal site hi such a way that a distinctive fragment is produced. Such embodiments have been termed the INVADER assay (Third Wave Technologies) and are described hi U.S. Patent Nos. 5,846,717,5,985,557, 5,994,069, 6,001,567,6,090,543,6,348,314, and 6,458,535, WO 97/27214 WO 98/42873, Lyamichev et al., Nat. Biotech., 17:292 (1999), Hall et al, PNAS, USA, 97:8272 (2000), each of which is herein incorporated by reference in its entirety for all purposes).<br>
The INVADER assay detects hybridization of probes to a target by enzymatic cleavage of specific structures by structure specific enzymes (See, INVADER assays, Third Wave Technologies; See e.g., U.S. Patent Nos. 5,846,717; 6,090,543; 6,001,567; 5,985,557; 5,994,069; 6,090,543; 6,348,314; 6,458,535; U.S. Patent App. Nos. 20030186238 (Ser. No. 10/084839); 20030104378A1 (Ser. No. 09/864636); Lyamichev et al., Nat Biotech., 17:292 (1999), Hall et al., PNAS, USA, 97:8272 (2000), WO97/27214 and WO98/42873, each of which is herein incorporated by reference in its entirety for all purposes).<br>
The INVADER assay detects specific DNA and RNA sequences by using structure-specific enzymes (e.g. FEN endonucleases) to cleave a complex formed by the hybridization of overlapping oligonucleotide probes (See, e.g. Figure 1). Elevated temperature and an excess of one of the probes enable multiple probes to be cleaved for each target sequence present without temperature cycling. In some embodiments, these cleaved probes then direct cleavage of a second labeled probe. The secondary probe oligonucleotide can be 5'-end labeled with fluorescein that is quenched by an internal<br><br>
dye. Upon cleavage, the de-quenched fluorescein labeled product may be detected using a standard fluorescence plate reader.<br>
The INVADER assay detects specific sequences, mutations, and SNPs in unamplified, as well as amplified, RNA and DNA, including genomic DNA. In the embodiments shown schematically in Figure 1, the INVADER assay uses two cascading steps (a primary and a secondary reaction) both to generate and then to amplify the target-specific signal. For convenience, the alleles in the following discussion are described as wild-type (WT) and mutant (MT), even though this terminology does not apply to all genetic variations. In the primary reaction (Figure 1, panel A), the WT primary probe and the INVADER oligonucleotide hybridize hi tandem to the target nucleic acid to form an overlapping structure. An unpaired "flap" is included on the 5' end of the WT primary probe. A structure-specific enzyme (e.g. the CLEAVASE enzyme, Third Wave Technologies) recognizes the overlap and cleaves off the unpaired flap, releasing it as a target-specific product. In the secondary reaction, this cleaved product serves as an INVADER oligonucleotide on the WT fluorescence resonance energy transfer (WT-FRET) probe to again create the structure recognized by the structure specific enzyme (panel A). When the two dyes on a single FRET probe are separated by cleavage (indicated by the arrow in Figure 1), a detectable fluorescent signal above background fluorescence is produced. Consequently, cleavage of this second structure results in an increase in fluorescence, indicating the presence of the WT allele (or mutant allele if the assay is configured for the mutant allele to generate the detectable signal). In some embodiments, FRET probes having different labels (e.g. resolvable by difference in emission or excitation wavelengths, or resolvable by time-resolved fluorescence detection) are provided for each allele or locus to be detected, such that the different alleles or loci can be detected hi a single reaction. In such embodiments, the primary probe sets and the different FRET probes may be combined in a single assay, allowing comparison of the signals from each allele or locus in the same sample.<br>
If the primary probe oligonucleotide and the target nucleotide sequence do not match perfectly at the cleavage site (e.g., as with the MT primary probe and the WT target, Figure 1, panel B), the overlapped structure does not form and cleavage is   . suppressed. The structure specific enzyme (e.g., CLEAVASE VDI enzyme, Third Wave<br><br>
Technologies) used cleaves the overlapped structure more efficiently (e.g. at least 340-fold) than the non-overlapping structure, allowing excellent discrimination of the alleles.<br>
The probes turn over without temperature cycling to produce many signals per target (i.e., linear signal amplification). Similarly, each target-specific product can enable the cleavage of many FRET probes.<br>
The primary INVADER assay reaction is directed against the target DNA (or RNA) being detected. The target DNA is the limiting component in the first invasive cleavage, since the INVADER and primary probe are supplied in molar excess. In the second invasive cleavage, it is the released flap that is limiting. When these two cleavage reactions are performed sequentially, the fluorescence signal from the composite reaction accumulates linearly with respect to the target DNA amount.<br>
In certain embodiments, the INVADER assay, or other nucleotide detection assays, are performed with accessible site-designed oligonucleotides and/or bridging oligonucleotides. Such methods, procedures and compositions are described in U.S. Pat. 6,194,149,6,358,691, 6355,437, U.S. Pat. Application Ser. No. 09/882,945, and PCT Applications WO9850403, and WO0198537, all of which are specifically incorporated by reference in their entireties.<br>
In some preferred embodiments,'the exposing of the sample to the oligonucleotides and the agent comprises exposing the sample to the oligonucleotides and the agent under conditions wherein an invasive cleavage structure is formed between said target sequence and said oligonucleotides if said target sequence is present in said sample, wherein said invasive cleavage structure is cleaved by said cleavage agent to form a cleavage product.<br>
In some embodiments, the target sequence comprises a first region and a second region, the second region downstream of and contiguous to the first region, and the oligonucleotides comprise first and second oligonucleotides, wherein at least a portion of the first oligonucleotide is completely complementary to the first portion of the target sequence and wherein the second oligonucleotide comprises a 3" portion and a 5' portion, wherein the 5' portion is completely complementary to the second portion of the target nucleic acid.<br><br>
In some preferred embodiments, the exposing of the sample to the oligonucleotides and the agent comprises exposing the sample to the oligonucleotides and the agent under conditions wherein an invasive cleavage structure is formed between the target sequence and the oligonucleotides if the target sequence is present in the sample, wherein the invasive cleavage structure is cleaved by the cleavage agent to form a cleavage product.<br>
In some preferred embodiments, the exposing of the sample to the oligonucleotides and the agent comprises exposing the sample to the oligonucleotides and the agent under conditions wherein an invasive cleavage structure is formed between said target sequence and said oligonucleotides if said target sequence is present in said sample, wherein said invasive cleavage structure is cleaved by said cleavage agent to form a cleavage product.<br>
In some particularly preferred embodiments, the target sequence comprises a first region and a second region, said second region downstream of and contiguous to said first region, and said oligonucleotides comprise first and second oligonucleotides, wherein at least a portion of said first oligonucleotide is completely complementary to said first portion of said target sequence and wherein said second oligonucleotide comprises a 3' portion and a 5' portion, wherein said 5' portion is completely complementary to said second portion of said target nucleic acid.<br>
In certain embodiments, the present invention provides kits for assaying a pooled sample (e.g., a pooled blood sample or pooled cell lysates) using INVADER detection reagents (e.g. primary probe, INVADER probe, and FRET cassette). In preferred embodiments, the kit further comprises instructions on how to perform the INVADER assay, and hi some embodiments, how to apply the INVADER detection assay to pooled samples from many individuals, or to "pooled" samples from many cells (e.g., from a biopsy sample) from a single subject.<br>
The present invention further provides assays in which the target nucleic acid is reused or recycled during multiple rounds of hybridization with oligonucleotide probes and cleavage of the probes without the need to use temperature cycling (i.e., for periodic denaturation of target nucleic acid strands) or nucleic acid synthesis (i.e., for the polymerization-based displacement of target or probe nucleic acid strands). When a<br><br>
cleavage reaction is run under conditions in which the probes are continuously replaced on the target strand (e.g. through probe-probe displacement or through an equilibrium between probe/target association and disassociation, or through a combination comprising these mechanisms, (Reynaldo et al., J. Mol. Biol. 97: 511-520 (2000)), multiple probes can hybridize to the same target, allowing multiple cleavages, and the generation of multiple cleavage products.<br>
The INVADER Assay Reaction:<br>
In preferred embodiments of the INVADER DNA assay, two oligonucleotides (a discriminatory Primary Probe and an INVADER Oligo) hybridize in tandem to the target DNA to form an overlapping structure. The 5'-end of the Primary Probe includes a 5 -flap that does not hybridize to the target DNA (Figure 1). The 3'-nucleotide of the bound INVADER oligonucleotide overlaps the Primary Probe, but need not hybridize to the target DNA. The CLEAVASE enzyme recognizes this overlapping structure and cleaves off the unpaired S'-flap of the Primary Probe, releasing it as a target-specific product. The Primary Probe is designed to have a melting temperature close to the reaction temperature. Thus, under the isothermal assay conditions, Primary Probes, which are provided in excess, cycle on the target DNA. This allows for multiple rounds of Primary Probe cleavage for each target DNA, and amplification of the number of released 5-fiaps.<br>
In the secondary reaction, each released 5'-flap can serve as an INVADER oligonucleotide on a fluorescence resonance energy transfer (FRET) Cassette to create another overlapping structure that is recognized and cleaved by the CLEAVASE enzyme (Figure 1). When the FRET Cassette is cleaved, the fluorophore (F) and quencher (Q) are separated, generating detectable fluorescence signal. Similar to the initial reaction, the released 5'-flap and the FRET Cassette cycle, resulting in amplified fluorescence signal. The initial and secondary reactions run concurrently in the same well.<br>
The biplex format of the INVADER DNA Assay enables simultaneous detection of two DNA sequences in a single well. Most often, this involves detection of two variants of a particular polymorphism (e.g., in a rm'RNA). The biplex format uses two different discriminatory Primary Probes, each with a unique 5'-flap, and two different FRET Cassettes, each with a spectrally distinct fluorophore. By design, the released 5'-<br><br>
flaps will bind only to their respective FRET Cassettes to generate a target-specific signal.<br>
In some embodiments, the present invention provides kits comprising one or more of the components necessary for practicing the present invention. For example, the present invention provides kits for storing or delivering the enzymes of the present invention and/or the reaction components necessary to practice a cleavage assay (eg., the INVADER assay). By way of example, and not intending to limit the kits of the present invention to any particular configuration or combination of components, the following section describes one embodiment of a kit for practicing the present invention:<br>
In some embodiments, the kits of the present invention provide the following reagents:<br>
CLEAVASE enzyme	Primary Probe Oligos<br>
DNA Reaction Buffer 1	INVADER Oligo<br>
FRET Cassette 1 (e.g., F) FRET Cassette 2 (e.g., R) Mutant DNA controls Wild type DNA controls "No Target" Blank control<br>
In other embodiments, the kits of the present invention are configured for direct detection<br>
of RNA. These kits may provide the following reagents:<br>
CLEAVASE enzyme	Primary Probe<br>
oligonucleotides<br>
DNA Reaction Buffer 1	INVADER Oligo<br>
FRET Probe I (e.g.,F) FRET Probe 2 (e.g.,R) Secondary Reaction Target 1 Secondary Reaction Target 2 ARRESTOR oligonucleotide  <br>
ARRESTOR oligonucleotide<br>
2<br>
Mutant DNA controls<br>
Wild type DNA controls<br>
"No Target" Blank control<br>
An additional consideration has to do with undesired effects resulting from particular combinations of oligonucleotides in a single reaction. One such effect is target-independent generation of background signal. Certain oligonucleotides in combination with others may generate signal in the INVADER assay in the absence of the particular target being detected. Separation of these oligonucleotide combinations into different pools can be used to alleviate this effect. Similarly, certain oligonucleotide combinations can artificially repress signal generation from a desired target Again, separation of these combinations into different pools can alleviate this effect.<br>
The designs of the probes sets (e.g., the oligonucleotides and/or their sequences) are adapted for use in miRNA detection assays using the guidelines for reaction design and optimization provided herein (See e.g., the Experimental Section). For example, in some embodiments, the reaction temperature is reduced (e.g., to 50-60°C) to account for the smaller region of hybridization.<br>
In some embodiments, a kit of the present invention provides a list of additional components (e.g., reagents, supplies, and/or equipment) to be supplied by a user in order to perform the methods of the invention. For example, and without intending to limit such additional components lists to any particular components, one embodiment of such a list comprises the following:<br>
»    Clear CHELLOUT-14 liquid wax (MJ Research) or RNase-free, optical grade mineral oil (Sigma, Cat. No. M-5904)<br>
•	96-well polypropylene microplate (MJ Research, Cat. No. MSP-9601)<br>
•	Sterile 1.5-ml or 2.0-ml microcentrifuge tubes<br>
•	Sterile, DNase/RNase free disposable aerosol barrier pipet tips<br>
»    Multichannel pipets (0.5-10^1,2.5-20^1)<br>
•	Thermal cycler or other heat source (e.g., lab oven or heating block).<br><br>
•	Miscellaneous laboratory equipment (tube racks, micropipetors, multichannel pipet,<br>
microcentrifuge, vortex mixer).<br>
•	Fluorescence microplate reader (a preferred plate reader is top-reading and equipped<br>
with light filters have the following characteristics:<br>
Excitation	Emission<br>
(Wavelength/Bandwidth)	(Wavelength/Bandwidth)<br>
485 ma/20 ran	530 nm/25 nm<br>
560 nm/20 nm	620 nm/40 nm<br>
In some embodiments, a kit of the present invention provides a list of optional components (e.g., reagents, supplies, and/or equipment) to be supplied by a user to facilitate performance of the methods of the invention. For example, and without intending to limit such optional components lists to any particular components, one embodiment of such a list comprises the following:<br>
•	Sterile 8-tube strip or microplate (optional)<br>
•	Disposable plastic trough (optional)<br>
•	Plate sealing tape (optional)<br>
In some embodiments, a kit of the present invention provides a list of required components to be supplied by a user to facilitate performance of the methods of the invention for which multiple alternatives are acceptable (e.g. sample preparation kits). For example, and without intending to limit such optional components lists to any particular components, one embodiment of such a list comprises the following:<br>
•	QIAGENQIAAMP Blood Kit<br>
•	Centra Systems PUREGENE Kit<br>
•	Centra Systems GENERATION Products<br>
In some embodiments of a kit, detailed protocols are provided. In preferred embodiments, protocols for the assembly of INVADER assay reactions (e.g., formulations and preferred procedures for making reaction mixtures) are provided. In<br><br>
particularly preferred embodiments, protocols for assembly of reaction mixtures include computational or graphical aids to reduce risk of error in the performance of the methods of the present invention (e.g., tables to facilitate calculation of volumes of reagents needed for multiple reactions, and plate-layout guides to assist in configuring multi-well assay plates to contain numerous assay reactions).<br>
In some embodiments, supplementary documentation, such as protocols for ancillary procedures, e.g., for the preparation of additional reagents, or for preparation of samples for use in the methods of the present invention, are provided. In preferred<br>
«•<br>
embodiments, supplementary documentation includes guidelines and lists of precautions provided to facilitate successful use of the methods and kits by unskilled or inexperienced users. In particularly preferred embodiments, supplementary documentation includes a troubleshooting guide, e.g., a guide describing possible problems that maybe encountered by users, and providing suggested solutions or corrections to intended to aid the user in resolving or avoiding such problems.<br>
In preferred embodiments, samples are diluted to concentrations that correspond to a 10-jd addition per reaction. The concentration of a 100-ng sample should be 15ng/nl.<br>
B.       Rolling Circle Replication<br>
In other embodiments, rolling circle replication methods (Amersham Biosciences, Piscataway, NJ) are utilized for detection of miRNA detection structures (See e.g., U.S. Patents 6,344,329;-6,143,495; 6,316,229; 6,210,884,6,183,960 and 6,235,502; each of which is herein incorporated by reference). In some embodiments, rolling circle replication is used to detect circular miRNA detection structures generated from the annealing of the ends of a single oligonucleotide annealed to a miRNA. In some embodiments, the ends of the oligonucleotide hybridize to the miRNA with no overlap. This oligonucleotide can be ligated in the presence or absence of miRNA. However, the ligation reaction is more efficient in the presence of the miRNA. hi such embodiments, the level of circular molecules detected over time is compared to a control reaction lacking miRNA.<br><br>
In other embodiments, the ends of the oligonucleoti.de hybridize to the miRNA with overlapping ends to generate an invasive cleavage structure. Such structures are cleaved prior to ligation, thus improving the specificity of the generation of the circular detection structure.<br>
Rolling circle amplification (RCA) involves replication of circular single-stranded DNA molecules. In RCA, a rolling circle replication primer hybridizes to circular nucleic acid molecules followed by rolling circle replication of the nucleic acid molecules using a strand-displacing DNA polymerase. Amplification takes place during rolling circle replication in a single reaction cycle. Rolling circle replication results in large DNA molecules containing tandem repeats of the nucleic acid sequence. This DNA molecule is referred to as a tandem sequence DNA (TS-DNA).<br>
In some embodiments, ligation-mediated rolling circle amplification (LM-RCA), which involves a ligation operation prior to replication, is utilized. In the ligation operation, an probe hybridizes to its cognate target nucleic acid sequence, if present, followed by ligation of the ends of the hybridized probe to form a covalently closed, single-stranded nucleic acid. After ligation, a rolling circle replication primer hybridizes to probe molecules followed by rolling circle replication of the circular molecules using a strand-displacing DNA polymerase. Generally, LM-RCA comprises mixing an open circle probe with a target sample, resulting in an probe-target sample mixture, and incubating the probe-target sample mixture under conditions promoting hybridization between the open circle probe and a target sequence, mixing ligase with the probe-target sample mixture, resulting in a ligation mixture, and incubating the ligation mixture under conditions promoting ligation of the open circle probe to form an amplification target circle (ATC), mixing a rolling circle replication primer (RCRP) with the ligation mixture, resulting in a primer-ATC mixture, and incubating the primer-ATC mixture under conditions that promote hybridization between the amplification target circle and the rolling circle replication primer, mixing DNA polymerase with the primer-ATC mixture, resulting in a polymerase-ATC mixture, and incubating the polymerase-ATC mixture under conditions promoting replication of the amplification target circle, where replication of the amplification target circle results in formation of tandem sequence DNA (TS-DNA).<br><br>
C.       Additional Detection Methods<br>
The present invention is not limited to INVADER assay or rolling circle assay detection. Any method that allows for the detection of miRNA detection structures may be utilized. Exemplary, non-limiting detection assay that find use in the methods of the present invention are described below.<br>
1.        Hybridization Assays<br>
In some embodiments of the present invention, detection structures are detected using a hybridization assay. La a hybridization assay, the presence of absence of a given nucleic acid sequence is determined based on the ability of the DNA from the sample to hybridize to a complementary DNA molecule (e.g., a oligonucleotide probe). A variety of hybridization assays using a variety of technologies for hybridization and detection are available. A description of a selection of assays is provided below.<br>
a.	Direct Detection of Hybridization<br>
In some embodiments, hybridization of a probe to the sequence of interest (e.g., a SNP or mutation) is detected directly by visualizing a bound probe (e.g., a Northern or Southern assay, See e.g., Ausabel et al. (eds.), Current Protocols in Molecular Biology, John Wiley &amp; Sons, NY [1991]). In a these assays, genomic DNA (Southern) or RNA (Northern) is isolated from a subject. The DNA or RNA is then cleaved with a series of restriction enzymes that cleave infrequently in the genome and not near any of the markers being assayed. The DNA or RNA is then separated (e.g., on an agarose gel) and transferred to a membrane. A labeled (e.g., by incorporating a radionucleotide) probe or probes specific for the SNP or mutation being detected is allowed to contact the membrane under a condition of low, medium, or high stringency conditions. Unbound probe is removed and the presence of binding is detected by visualizing the labeled probe.<br>
b.	Detection of Hybridization Using "DNA Chip" Assays<br><br>
In some embodiments of the present invention, variant sequences are detected using a DNA chip hybridization assay. In this assay, a series of oligonucleotide probes are affixed to a solid support. The oligonucleotide probes are designed to be unique to a given target sequence (e.g., miKNA target sequence). The DNA sample of interest is contacted with the DNA "chip" and hybridization is detected.<br>
In some embodiments, the DNA chip assay is a GeneChip (Afrymetrix, Santa Clara, CA; See e.g., U.S. Patent Nos. 6,045,996; 5,925,525; and 5,858,659; each of which is herein incorporated by reference) assay. The GeneChip technology uses miniaturized, high-density arrays of oligonucleotide probes affixed to a "chip." Probe arrays are manufactured by Affymetrix's light-directed chemical synthesis process, which combines solid-phase chemical synthesis with photolithographic fabrication techniques employed in the semiconductor industry. Using a series of photolithographic masks to define chip exposure sites, followed by specific chemical synthesis steps, the process constructs high-density arrays of oligonucleotides, with each probe in a predefined position in the array. Multiple probe arrays are synthesized simultaneously on a large glass wafer. The wafers are then diced, and individual probe arrays are packaged hi injection-molded plastic cartridges, which protect them from the environment and serve as chambers for hybridization.<br>
The nucleic acid to be analyzed is isolated, amplified by PCR, and labeled with a fluorescent reporter group. The labeled DNA is then incubated with the array using a fluidics station. The array is then inserted into the scanner, where patterns of hybridization are detected. The hybridization data are collected as light emitted from the fluorescent reporter groups already incorporated into the target, which is bound to the probe array. Probes that perfectly match the target generally produce stronger signals than those that have mismatches. Since the sequence and position of each probe on the array are known, by complementarity, the identity of the target nucleic acid applied to the probe array can be determined.<br>
In other embodiments, a DNA microchip containing electronically captured probes (Nanogen, San Diego, CA) is utilized (See e.g., U.S. Patent Nos. 6,017,696; 6,068,818; and 6,051,380; each of which are herein incorporated by reference). Through the use of microelectronics, Nanogen's technology enables the active movement and<br><br>
concentration of charged molecules to and from designated test sites on its semiconductor microchip. DNA capture probes unique to a given target sequence are electronically placed at, or "addressed" to, specific sites on the microchip. Since DNA has a strong negative charge, it can be electronically moved to an area of positive charge.<br>
First, a test site or a row of test sites on the microchip is electronically activated with a positive charge. Next, a solution containing the DNA probes is introduced onto the microchip. The negatively charged probes rapidly move to the positively charged sites, where they concentrate and are chemically bound to a site on the microchip. The microchip is then washed and another solution of distinct DNA probes is added until the array of specifically bound DNA probes is complete.<br>
A test sample is then analyzed for the presence of target sequences by determining which of the DNA capture probes hybridize, with target sequences. An electronic charge is also used to move and concentrate target molecules to one or more test sites on the microchip. The electronic concentration of sample DNA at each test site promotes rapid hybridization of sample DNA with complementary capture probes (hybridization may occur hi minutes). To remove any unbound or nonspecifically bound DNA from each site, the polarity or charge of the site is reversed to negative, thereby forcing any unbound or nonspecifically bound DNA back into solution away from the capture probes. A laser-based fluorescence scanner is used to detect binding.<br>
In still further embodiments, an array technology based upon the segregation of fluids on a flat surface (chip) by differences hi surface tension (ProtoGene, Palo Alto, CA) is utilized (See e.g;,U.S. Patent Nos. 6,001,311; 5,985,551; and 5,474,796; each of which is herein incorporated by reference). Protogene's technology is based on the fact that fluids can be segregated on a flat surface by differences in surface tension that have been imparted by chemical coatings. Once so segregated, oligonucleotide probes are synthesized directly on the chip by ink-jet printing of reagents. The array with its reaction sites defined by surface tension is mounted on a X/Y translation stage under a set of four piezoelectric nozzles, one for each of the four standard DNA bases. The translation stage moves along each of the rows of the array and the appropriate reagent is delivered to each of the reaction sites. For example, the A amidite is delivered only to the sites where amidite A is to be coupled during that synthesis step and so on. Common<br><br>
reagents and washes are delivered by flooding the entire surface and then removing them by spinning.<br>
DNA probes unique for the target sequence (e.g., miRNA target sequence) of interest are affixed to the chip using Protogene's technology. The chip is then contacted with the PCR-amplified genes of interest. Following hybridization, unbound DNA is removed and hybridization is detected using any suitable method (e.g., by fluorescence de-quenching of an incorporated fluorescent group).<br>
In yet other embodiments, a "bead array" is used for the detection of polymorphisms (Hlumina, San Diego, CA; See e.g., PCT Publications WO 99/67641 and WO 00/39587, each of which is herein incorporated by reference). Hlumina uses a BEAD ARRAY technology that combines fiber optic bundles and beads that self-assemble into an array. Each fiber optic bundle contains thousands to millions of individual fibers depending on the diameter of the bundle. The beads are coated with an oligonucleotide specific for the detection of a given SNP or mutation. Batches of beads are combined to form a pool specific to the array. To perform an assay, the BEAD ARRAY is contacted with a prepared subject sample (e.g., nucleic acid sample). Hybridization is detected using any suitable method.<br>
c.        Enzymatic Detection of Hybridization<br>
In some embodiments of the present invention, hybridization is detected by enzymatic cleavage of specific structures.<br>
In some embodiments, hybridization of a bound probe is detected using a TaqMan assay (PE Biosystems, Foster City, CA; See e.g., U.S. Patent Nos. 5,962,233 and 5,538,848, each of which is herein incorporated by reference). The assay is performed during a PCR reaction. The TaqMan assay exploits the 5'-3' exonuclease activity of the AMPLITAQ GOLD DNA polymerase. A probe, specific for a given allele or mutation, is included in the PCR reaction. The probe consists of an oligonucleotide with a 5'-reporter dye (e.g., a fluorescent dye) and a 3'-quencher dye. During PCR, if the probe is bound to its target, the 5'-3' nucleolytic activity of the AMPLITAQ GOLD polymerase cleaves the probe between the reporter and the quencher dye. The separation of the<br><br>
reporter dye from the quencher dye results in an increase of fluorescence. The signal accumulates with each cycle of PCR and can be monitored with a fluorimeter.<br>
In still further embodiments, polymorphisms are detected using the SNP-IT primer extension assay (Orchid Biosciences, Princeton, NJ; See e.g., U.S. Patent Nos. 5,952,174 and 5,919,626, each of which is herein incorporated by reference). In this assay, SNPs are identified by using a specially synthesized DNA primer and a DNA polymerase to selectively extend the DNA chain by one base at the suspected SNP location. DNA in the region of interest is amplified and denatured. Polymerase reactions are then performed using miniaturized systems called microfluidics. Detection is accomplished by adding a label to the nucleotide suspected of being at the target sequence location. Incorporation of the label into the DNA can be detected by any suitable method (e.g., if the nucleotide contains a biotin label, detection is via a fluorescently labeled antibody specific for biotin).<br>
2.        Other Detection Assays<br>
Additional detection assays useful "m the detection of mIRNA detection structures include, but are not limited to, enzyme mismatch cleavage methods (e.g., Variagenics, U.S. Pat. Nos. 6,110,684, 5,958,692, 5,851,770, herein incorporated by reference hi their entireties); polymerase chain reaction; branched hybridization methods (e.g., Chiron, U.S. Pat. Nos. 5,849,481,5,710,264,5,124,246, and 5,624,802, herein incorporated by reference hi their entireties); NASBA (e.g.t U.S. Pat. No. 5,409,818, herein incorporated by reference hi its entirety); molecular beacon technology (e.g., U.S. Pat. No. 6,150,097, herein incorporated by reference in its entirety); E-sensor technology (Motorola, U.S. Pat. Nos. 6,248,229,6,221,583,6,013,170, and 6,063,573, herein incorporated by reference in their entireties); cycling probe technology (e.g., U.S. Pat Nos. 5,403,711, 5,011,769, and 5,660,988, herein incorporated by reference in their entireties); Dade Behring signal amplification methods (e.g., U.S. Pat. Nos. 6,121,001,6,110,677,5,914,230,5,882,867, and 5,792,614, herein incorporated by reference in their entireties); ligase chain reaction (Barnay Proc. Natl. Acad. Sci USA 88,189-93 (1991)); and sandwich hybridization methods (e.g., U.S. Pat. No. 5,288,609, herein incorporated by reference in its entirety).<br><br>
EXPERIMENTAL<br>
The following examples are provided in order to demonstrate and further illustrate certain preferred embodiments and aspects of the present invention and are not to be construed as limiting the scope thereof.<br>
Example 1 Materials and Methods<br>
•	The following final concentrations (unless noted) were used for all reactions:<br>
Probe = 1 uM<br>
INVADER=luM<br>
ARRESTOR = 2.67uM CLEAVASE XII enzyme = 30 ng<br>
All synthetic miRNA oligonucleotides were purchased from Dharmacon and gel purified on 20% denaturing acrylamide. Synthetic miRNAs were used to determine temperature optima (see below) and LOD.<br>
•	INVADER, probe, and ARRESTOR oligonucleotides were synthesized either by<br>
Integrated DNA Technologies (IDT) or Third Wave Technologies and purified on<br>
20% denaturing acrylamide, unless otherwise indicated.<br>
•	The following 2.5 X primary reaction buffer was used (unless otherwise noted) for all<br>
reactions:<br>
25mMMOPSpH7.5 62.5mMKCl 0.125% Tween 20 0.125%NonidetNP40 62.5 mM MgSO4 5% PEG<br>
•	Unless otherwise noted, all reactions were overlaid with 10 ul mineral oil prior to the<br>
first thermal incubation.<br>
•	Unless otherwise noted, synthetic miRNAs contained a 5'OH. Experiments<br>
v-<br>
comparing detection of 5' phosphorylated vs. unphosphorylated synthetic miRNA<br><br>
targets indicated that there was no significant difference in the ability of the INVADER assay to detect these two different types of synthetic molecules.<br>
Example 2 Temperature optimization experiments for let-7 and mir-1<br>
The oligonucleotide design for let-7 is shown in Figure 5. The oligonucleotide design for mir-1 is shown in Figure 5. The following primary mixes were made and incubated at 50°C ± 10°C in a 96 well plate for 30 minutes. In addition, a no target master mix was prepared (addition of HjO in place of RNA). All reactions were covered with mineral oil to prevent evaporation.<br><br>
After completion of the primary reaction, 5 u.1 of the following secondary reaction mix were added and the reaction was then reaction incubated at 60°C for 10 -15 minutes.(Table Removed)<br><br><br>
	er completion of the reaction, the plate was read in a C YTOFLUOR 4000 fluorescence microplate reader using an excitation wavelength of 485 nm and emission wavelength of 530 nm. Results are shown in Figures 6 and 7. Stacking of the 5'-end of the INVADER oligonucleotide to the 3'-end of the miRNA is enhanced when the 3'-end of the INVADER oligonucleotide is 2'-O-methylated. In addition, 2'-O-methylation of the 5'-end of the INVADER oligonucleotide increases the reaction temperature. Extending the 2'-0-methylated bases of the INVADER oligonucleotide so that they base pair with the first two bases of the miRNA (SEQ ID NO: 8 (1496-96-02) vs. SEQ ID NO: 23 (1496-96-03) in design SEQ ID NO: 9 (1496-96-01R) of let-7a) increases the temperature optimum of the described reaction but does not enhance the detection.<br>
Example 3 LOD experiments for let-7 and miR-1<br>
After determining the optimal reaction temperature for each set of probe and INVADER oligonucleotides and determining the best working design (from the temperature optimization net signal), the following experiment was set up to determine the LOD of the design using synthetic RNA. The following reaction mix was aliquoted into a 96-well plate (see plate setup below) with each well containing:<br><br>
(Table Removed)<br>
2.5 ul of the following miRNA concentations were added in triplicates or quadruplicates using the following setup:<br><br>
The plate was overlayed with mineral oil (10 ul) and incubated at 50°C for 2 hrs. After completion of the primary reaction, 5 ul of the following was added to each well and the plates were incubated at 60°C for 1.5 hrs. The plate was read using the settings described above (see Example 2).<br><br>
(Table Removed)he LOD for let-7 and mir-1 was next tested on human RNA samples. The protocol described above was utilized. 50 - 100 ng of tissue specific total human RNA samples (Clonetech, Palo Alto, CA) was used.  Results are shown in Figures 8 and 10. Using total RNA the let-7a INVADER assay detects the same tissue expression profile as<br>
seen before for let-7a expression levels depending on the source of tissue (Pasquinelli et al, 408:86 [2000]).<br>
Example 4<br>
Cross reactivity experiments for Iet-7 a,c,e, and f This Example describes an analysis of the cross reactivity of probe and/or INVADER oligonucleotides directed against one sub-type of let-7 for another sub-type. The protocol for synthetic let-7a miRNA setup described in Example 3 was utilized. Figure 5 shows the oligonucleotide designs. The following plate setup was used:<br><br>
(Table Removed)he results are shown in Figure 9. For let-7a design, cross reactivity is maximum when the miRNA is of the same length with a one base change away from the cleavage site. In other words, mismatches at the INVADER oligonucleotide/miRNA hybridizing regions result hi high cross reactivity when the mismatch is furthest from the cleavage site (let-7c). Cross reactivity is the lowest when base changes are opposite (or close to) the cleavage site. For let-7a, the worst cross reactivity is with let-7c, which results in 25% of the signal. This Example demonstrates that the INVADER assay is able to differentiate between very similar miRNAs.<br>
Example 5 CLEAVASE IX enzyme vs CLEAVASE <br>
This Example describes the optimization of CLEAVASE enzymes for use in miRNA assays. The protocol for temperature optimization described above was utilized. Either 20 ng of the CLEAVASE DC enzyme (Third Wave Technologies, Madison, WI) or 30 ng of the CLEAVASE XII enzyme (Third Wave Technologies, Madison, WI) was used. The following buffer was used for the CLEAVASE DC enzyme:<br>
2.5 X primary reaction buffer. 25 mM MOPS pH 7.5,250 mM KC1,0.125% Tween 20, 0.125% Nonidet NP40,31.25 mM MgSO4,10% PEG. 7.5 X secondary reaction buffer: 87.5 mM MgSO4<br>
The following buffer was used for the CLEAVASE XII enzyme:<br>
2.5 X primary reaction buffer 25 mM MOPS pH 7.5,62.5 mM KC1,0.125% Tween 20, 0.125% Nonidet NP40,62.5 mM MgSO4,5% PEG. 7.5 X secondary reaction buffer: HjO<br>
The LOD experimental protocol was used with either the CLEAVASE DC or XH enzymes. The LOD was determined for both enzymes. The results are shown in Figure 11.<br>
Signal increased linearly with increasing amounts of the let-7 miRNA when assayed with either the CLEAVASE DC enzyme or the CLEAVASE XH enzyme. However, R2 values were greater in the CLEAVASE XII enzyme, indicating greater linearity. Moreover, the LOD was lower with the CLEAVASE XII enzyme. The net signal for the detection of 2.5 amoles was 20 counts with the CLEAVASE DC enzyme and 66.75 with the CLEAVASE XD enzyme.<br>
Example 6<br>
raiR-135, GAPDH and U6 RNA A. Design of oligonucleotides to detect miR135<br>
This example describes assay design and LOD determination for miRJSf A miR135. Experiments were performed as described in Examples 2 and 3 for miR-1. The<br><br>
oligonucleotide designs are described in Figure 5. Each of the designs (A-D) utilizes different INVADER and probe oligonucleotides for the detection of mir-135 miRNA. Results of the temperature optimization experiments comparing perfonnance of all of the designs are shown in Figure 13. Design D gave the highest signal. Results of LOD experiments using assay design D are shown in Figure 14.<br>
Figure 14A presents the raw counts generated from four replicate assays at each of the indicated target concentrations. The average counts obtained with each target concentration are indicated as are the net signal and fold-over-zero (FOZ). The limit of detection of the miR-135 target in this experiment was 164 zmoles, equivalent to 98,743 molecules. Figure 14B contains a graphical representation of the average counts obtained at each concentration and indicates that the INVADER assay is linear throughout much of the concentration range tested.<br>
B.	Design of oligonucleotides to detect GAPDH and U6 RNA<br>
In some circumstances, it may be desirable to co-detect, e.g. in a biplex assay, an RNA generally present in all cells at constant levels along with one or more miRNA species, which may be expressed in a tissue-specific manner. INVADER assays were therefore designed to two distinct RNAs generally found in all cell types: human glyceraldehydes-3-phosphate dehydrogenase (hGAPDH) and U6 RNA.<br>
In the case of hGAPDH, the following oligonucleotides have been used in biplex miRNA detection assays: INVADER oligonucleotide (SEQ ID NO: 41); probe (SEQ ID NO: 42); ARRESTOR oligonucleotide (SEQ ID NO: 43); SRT oligonucleotide (SEQ ID N0:49), FRET oligonucleotide (red dye) (SEQ ID NO: 48).<br>
In the case of U6, sequence alignments of the U6 RNAs of 8 diverse species from<br>
C.	elegans' to mouse to arabidopsis to humans to identify a region suitable for the design<br>
of a "universal" INVADER assay. The alignment is shown in Figure 12; the<br>
. oligonucleotide sequences created to detect this sequence are SEQ ED NOs: 93-95.<br>
Initial experiments carried out with these oligonucleotides on cell lysates using SEQ ID NOs: 45-47 demonstrated that signal from U6 reactions reached saturation well before miRNA signal, possibly owing to large quantities of U6 RNA in cells. Therefore, titration reactions were carried out to determine whether diluting the probe and INVADER oligonucleotide concentrations would render this probe set suitable for use in<br><br>
biplex miRNA detection assays with INVADER and probe final concentrations ranging from 1 uM to 12.5 nM.  Final concentrations of the INVADER and probe oligonucleotides between 12.5-50 nM were suitable for biplex miRNA detection for miR-Id and let-7a. ARRESTOR, SRT, and FRET probe concentrations were as described in the previous examples. Further experiments demonstrate that detection of U6 RNA with the "universal" U6 RNA oligonucleotides (SEQ ID NOs: 93-95) is comparable to detection with SEQ ID NOs: 45-47.<br>
Example 7<br>
Detection of let-7, GAPDH, and U6 RNA in cell lysates A. Detection of let 7a in cell lysates<br>
This example describes detection of the let-7 miRNA directly in total cell RNA as well as in uninduced fibroblast cells from a human osteosarcoma cell line, MG63 (Third Wave Technologies, Madison, WI; catalog number CRL-1427). Total cell RNA was extracted using TRIZOL (Gibco-BRL), as previously described (Chomczynski et al., Anal. Biochem. 162:156-156 (1987)), and cell lysates were prepared as described by Eis et al., Nature Biotechnology, 19: 673-6 (2001); both publications are herein incorporated by reference.<br>
Reactions were set up as follows. Aliquots of 5 ul of either cell lysate, synthetic miRNA target in lysis buffer (Eis et al., Nature Biotechnology, 19:673-6 (2001)) at the indicated concentrations, or 5 ul of 20 ng/ul of tRNA (for the no target controls) were pipetted into the appropriate wells of a microtiter plate. A primary reaction master mix was made for 96 reactions containing the following reagents.<br><br>
Aliquots of 5 jil of the primary reaction master mix were added to the wells containing the appropriate target or control. The plate was overlayed with mineral oil (10 pi) and incubated at 53°C for 2 hrs. After completion of the primary reaction, 5 ul of the following was added to each well, and the plates were incubated at 60°C for 1.5 hrs. The plate was read using the settings described above (see Example 2).<br><br>
(Table Removed)All targets were assayed in quadruplicate. The average counts obtained for the different numbers of cells assayed for both total RNA'and cell lysates were plotted in Figure 15. A standard curve obtained from INVADER assays on known quantities of synthetic let-7a miRNA was used to extrapolate the let-7a copy number per cell. The number of cells from which cell lysates were generated was determined during the seeding procedure prior to cell lysis as described in Eis et a/., Nature Biotechnology, 19: 673-6 (2001), herein incorporated by reference. In this experiment, the limit of detection in cell lysates was reached in the total RNA extracts obtained from 156 cells. B. Lysis in absence of Mg"1"1"<br>
An alternative lysis procedure was developed as follows. It had been noted that when the above lysis procedure was used, long mRNAs, i.e. from GAPDH, were not being detected in the quantities expected. Experiments were carried out to examine the effect of Mg"1"1" on extraction of RNA in lysates. Extracts lysed in the presence or absence of MgClz were compared to total cell RNA extracts prepared using TRIZOL as described above in this example.<br>
Hela cells (7.5 X 106 cells) were suspended in 100 ul of a solution of 10 mM MOPS buffer, pH 7.5, with 100 mM KC1. Aliquots of 10 ul were added to separate tubes and lysed with 100 ul of two different lysis buffers prepared as follows:<br><br>
(Table Removed)ll tubes were then incubated at 80°C for 15 minutes to lyse the cells, and then centrifuged to pellet debris. Aliquots of 5 ul of the various lysates were added to INVADER reactions as follows.<br>
Primary INVADER reactions were as described above for let-7a; PI 'oligonucleotide mixes were also made for GAPDH (SEQ ID NOs: 41-43) and for U6 (SEQ ID NOs: 45-47 at 50 nM final concentration).<br><br>
(Table Removed) Primary INVADER reactions were as described above for let-7a; PI oligonucleotide mixes were also made for GAPDH (SEQ ID NOs: 41-43) and for U6 (SEQ ID NOs: 45-<br>
47 at 50 nM final concentration).<br>
Primary reaction mixtures were incubated at 49°C for 1 hour. Aliquots of the following secondary reaction mixture were then added:<br><br><br>
(Table Removed)econdary reaction mixture included SRTs (SEQ ID NO: 22 for let-7; SEQ ID NO: 49 for GAPDH and U6) target, FRET oligonucleotides (SEQ ID NO: 21 for let-7; SEQ ID NO: 48 for GAPDH and U6), arresters (SEQ ID NO:7 for let-7, SEQ ID NO:43 for GAPDH, and SEQ ID NO: 47 for U6) at the concentrations indicated in Example 7A.<br>
Secondary reactions were run at 60°C for 1 hour. Reactions were read on a CYTOFLUOR microplate reader as described in Example 2. The results are presented in Figure 16 and indicate that presence of the GAPDH signal is dependent on the absence of Mg-H- from the lysis buffer, whereas U6 RNA signal remains relatively constant regardless of the presence of Mg++. Additional experiments confirmed that all RNAs were detectable in total cell RNA at levels comparable to those obtained from lysis in the absence of Mg-H-.<br>
Example 8 Alternative INVADER assay designs for detection of various miRNAs<br>
A. Alternative designs for detection of let-7 A<br>
This example describes the creation and testing of alternative oligonucleotide designs for detection of the let-7a miRNA. In one series of experiments, a set of alternative designs was created in which the target specific regions of both the INVADER oligonucleotide and the probe oligonucleotide were eleven nucleotides long. A second set of designs was created in which the target specific regions of the probe oligonucleotides were 10 nucleotides long and the target specific regions of the INVADER oligonucleotides were 12 nucleotides long.<br>
1. Oligonucleotide designs<br>
a. 11-mer probe and INVADER oligonucleotide designs<br>
Figure 5 shows sets of alternative oligonucleotide designs for detection of the let-7a miKNA in which the target specific regions of both the probe and INVADER oligonucleotides are 11 nucleotides long. SEQ ID NOs: 50-51 provide a design in which both the INVADER and probe oligonucleotides are linear. SEQ ID NO: 6 contains a probe oligonucleotide that forms a stem-loop structure); SEQ ID NO: 5, an INVADER oligonucleotide that forms a stem-loop structure); SEQ ID NOs: 5-6, both probe and INVADER oligonucleotides with stem-loops.<br>
b. 10-mer probe and 12-mer INVADER oligonucleotide designs<br>
Figure 5 shows a set of alternative oligonucleotide designs for detection of the let-7a miRNA in which the target specific regions of the probe comprise 10 nucleotides, and those of the INVADER oligonucleotides, 12 nucleotides. SEQ ID NOs: 52-53 provide a design in which both the INVADER and probe oligonucleotides are linear. SEQ ID NO: 2 contains a probe oligonucleotide that forms a stem-loop structure); SEQ ID NO: 1, an INVADER oligonucleotide that forms a stem-loop structure).<br>
2.  Temperature optimization profiles of alternative oligonucleotide designs for detection of the Iet-7a miRNA<br>
Temperature optimization experiments were carried out as follows. A master mix was made for 24 reactions. Each reaction contained the following:<br><br>
(Table Removed)<br>
Various combinations of probe and INVADER oligonucleotides were used in this experiment as indicated in Figures 16-17.<br>
Secondary reaction mixes were as described in Example 3 for let-7. Where appropriate, ARRESTOR sequences were made to compliment the entire loop and target specific regions of the probe and extending 6 bases toward the 5' end of the probe.<br>
In the case of the 11-mer temperature optimization experiment, the primary reactions were run at 50 ± 9°C for 1 hour followed by a 15 minute secondary reaction at 60°C as described in Example 2. As for the 10-mer probe, with the 12-mer INVADER oligo, the primary reactions were run at 50 ± 9°C for 15 minutes followed by a 15 minute secondary reaction at 60°C.<br>
Results for the designs in which the target specific portions of the INVADER and probe oligonucleotides were 11 nucleotides long are presented in Figure 18. Figure ISA shows the temperature optimization profiles of each design. Figure 18B shows the normalized maximum performance of each design, including the optimum temperature for each. Results for the designs in which the target specific portion of the probe oligonucleotide was 10 bases and that of the INVADER oligonucleotide was 12 are presented in Figure 19.. Figure 19 A shows the temperature optimization profiles, and Figure 19 B, the normalized maximum performance of each design.<br>
Examination of these results suggests that which design results in maximum performance varies depending on both reaction conditions and the relative stability of the miRNA-oligonucleotide hybrid fonned. For example, when the target specific regions of both oligonucleotides are 11 bases long, the probe target specific region has a predicted Tm of 49°C and that of the INVADER, of 37°C. In this case, stabilization of the INVADER oligonucleotide-miRNA interaction confers improved assay performance on this design. However, for the let-7a designs in which probes were 10-mers and INVADER oligonucleotides, 12-mers,, the target specific regions of the two oligonucleotides have approximately equivalent Tms. In this case, the design in which both oligonucleotides are looped works best.<br>
3.  LOD of Iet-7a using two alternative designs<br>
Experiments were set up as described in Example 3 to compare the LOD of the double loop design and the single loop design, in which the INVADER oligonucleotide forms a stem-loop structure.<br>
Reactions to determine LOD were run in quadruplicate. Reaction mixtures contained the following reagents (final concentrations):<br><br>
(Table Removed)liquots of 5 ul of miRNA were added to the wells containing the reaction mixtures at the final concentrations indicated in Figure 20. Primary reactions were run for 1.5 hours at the optimal temperatures for each designed as determined in Example 8B (50°C for the looped INVADER oligonucleotide design and 53°C for the double loop design). The secondary reactions were set up as described in Examples 2 and 3 and run forlhourat60°C.<br>
The results in Figure 20 show net signal produced as a function of moles of miRNA. The linear ranges of the plots indicated that more signal was produced from a given amount of miRNA using the INVADER loop design than from the double loop design. Similarly, an examination of the table in Figure 20 indicates that the fold-over-zero values at each miRNA level are greater for the single loop design. Both designs resulted in sufficient FOZ at the lowest concentrations tested, 2.68 X 10"20 moles, or 26.8 zeptomoles, equivalent to approximately 16,000 molecules.<br>
4.   Full length vs. Shortened ARRESTOR Oligonucleotides<br>
Experiments were conducted to evaluate the relative performance of full-length ARRESTOR molecules, e.g. as shown in Figures 4 and 12, in which the ARRESTOR molecules extend at their 5' ends around the loop, throughout the length of the miRNA-specific region of the probe and into the 5' flap region vs. shortened ARRESTOR molecules that are complementary only to the miRNA-specific region of the probe and part of the 5' flap but do not extend into the loop region or beyond. Reactions were set up as follows to detect synthetic let-7a miRNA:<br><br>
(Table Removed)liquots of 6 ul of the primary reaction mix were added to the appropriate wells of a microtiter plate followed by aliquots of 4 ul of syntheticlet-7a miRNA or 4 pi of 10 ng/ul tRNA in dHfoO at the final concentrations indicated in the table below. Primary INVADER reactions were incubated at 53°C for 1.5 hours.<br>
Aliquots of secondary reaction mixtures were added as follows:<br><br>
(Table Removed)<br>
Secondary reactions were incubated at 60°C for 1.5 hours. Microtiter plates were read as described in Example 2. The results were as shown in Figure 17.<br>
These results indicate that there is no significant different in signal generation or limit of detection when full-length or shortened AKRESTOR oligonucleotides complementary to the miRNA-specific portion of the probe are used in the secondary INVADER reaction.<br>
B. Alternative designs using linear probe and INVADER oligonucleotides<br>
Alternative designs were tested in which both the probe and INVADER oligonucleotides contain a universal sequence, and neither oligonucleotide forms a hairpin. A schematic of the design is presented in Figure 4. The universal sequence is present on the 5' end of the INVADER oligonucleotide and on the 3' end of the probe oligo. A short, complementary "capture" oligonucleotide is added and is comprised of 2'-O-methyl residues, allowing it to promote co-axial stacking in the presence of the miRNA (e.g. SEQ ID NO: 60). Designs were created for both miR-15 (SEQ ID NOs: 58-59 and 61) and mir-135 (SEQ ED NOs: 63-65). Initial designs, while leading to high nonspecific background signal in the absence of miRNA target, nonetheless indicate that it is feasible to detect miKNAs with such universal capture oligonucleotides.<br>
Example 9 Effect of 2'-O-methyIation of nucleotide residues in the loops<br>
This example describes experiments aimed at assessing the effect of substituting 2'-deoxy residues for some or all of the 2'-O-methyl residues incorporated in the probe and INVADER oligonucleotides used for detecting miRNAs. All of the designs presented hi the preceding examples include 2'-O-methyl residues in the loop regions as described in Example 2. Experiments were conducted to test the effect of substituting 2' deoxy residues for some or all of the 2'-O-methyl residues in the INVADER and probe oligonucleotides designed to detect the let-7a miRNA.<br>
Figure 5 shows the modified let-7a designs. SEQ ID NOs: 5-6 contain 2'-O-methyl residues as described in Example 2. The design in SEQ ID NOs: 73-74 contain 2'deoxy residues at all positions; and those in SEQ ED NOs: 75-76,2'-O-methyl residues in the portions of the stems adjacent to the target.<br>
INVADER reactions were set up to compare the signal generation and temperature optima of the three different designs. Reactions were as described in the LOD experiments in Example 8 and included 100 pM synthetic miRNA, 1 uM probe, and 10 uM INVADER oligonucleotide. Primary reactions were run for 15 minutes at the temperatures indicated; secondary reactions were run for 5 minutes at 60°C.<br>
The results of the INVADER assays are shown in Figure 21 and indicate that the design in which the stem loop structures are comprised of 2'-0-methyl residues yields the most signal, followed by the design in which the bases adjacent to the target are comprised of 2'-0-methyl residues. The oligonucleotides comprised entirely of 2'-deoxy residues generated the lowest levels of signal.<br>
A further set of experiments was designed to test additional design variations as follows: probe and INVADER oligonucleotides with shorter hairpins, probe and INVADER oligonucleotides with more stable loops or, alternatively with shorter loops, probe and INVADER oligonucleotides with only three 2'-O-methyl residues. Primary reactions were set up to test detection of miR-15 as follows.<br>
The following probe/INVADER oligonucleotide combinations were tested.<br><br>
Primary reaction mixes were made as follows.<br><br><br><br>
Aliquots of 5 ul of Primary reaction mix were added to 5 ul of synthetic miR-15 RNA at the following final amounts: 0, 0.1 amole, 0.33 amole, 1.09 amole. Primary reactions were incubated at 52.5°C for 2 hours.<br>
Secondary reaction mixes were made as follows.<br><br>
Aliquots of 5 ul were added and the reactions incubated at 60°C for 45 minutes. The results, in relative fluorescent units (RFUs) are presented below.<br><br><br>
(Table Removed) <br><br>
These results suggest that the designs in which the probe oligonucleotide contained a shortened hairpin and a highly stable tetra-loop comprised of 2'-O-methyl residues hi combination with the original INVADER oligonucleotide design (2'-O-methyl residues, TTTT loop, long hairpin) may generate a somewhat higher FOZ value. Otherwise, none of the alternative design oligonucleotide sets offered any improvement over the original designs. It is noteworthy that the combination of an all-DNA INVADER oligonucleotide with the original chimeric probe oligonucleotide gave FOZ values approximately equivalent to those obtained with both chimeric probe and INVADER oligonucleotides. hi some applications, substitution of an all DNA INVADER oligonucleotide may be desirable to reduce oligonucleotide synthesis costs and may be made without sacrificing limit of detection.<br>
Further experiments demonstrated that it is possible to compensate for sub-optimal signal generation with particular oligonucleotide sets by adding more RNA (e.g. lysate, purified total RNA, synthetic miRNA) to the reaction. Similarly, additional experiments hi which various oligonucleotides (i.e. probe, INVADER, ARRESTOR, or various combinations thereof) were gel purified as described in Example 1 indicated that standard gel purification of all three types of oligonucleotides gives maximal signal. It is possible to achieve signal levels approximately equal to the maximal levels with gel<br>
purified probes if the other oligonucleotides, i.e. the INVADER and ARRESTOR oligonucleotides, are desalted following synthesis.<br>
Example 10 Detection of miRNA expression in total RNA from multiple tissue types<br>
This example describes experiments carried out to test the suitability of the INVADER assay to detect different miRNA species in total RNA extracted from diverse tissue types. In order to evaluate tissue specific gene expression, temperature optima and LODs were first determined for each design.<br>
1.	INVADER and probe oligonucleotide designs<br>
INVADER assay oligonucleotides were designed to detect the miR-15, miR-16, and miR-125b miRNA species. The designs for these assays are presented in Figure 5. The designs for let-7a and miR-135 are described in Examples 2 and 6, respectively.<br>
2.	Determination of temperature optima and LODs<br>
Temperature optimization experiments were conducted for each of these oligonucleotide sets as described in Example 8. Each primary reaction included 1 nM of the targeted miRNA and was carried out for 15 minutes at temperatures ranging from 50± 9°C. Secondary reactions were as described in Example 2 and were run for 1 to 1.5 hours at 60°C. Optimum temperatures were as follows:<br><br>
(Table Removed)nce the temperature optima were obtained, LODs were determined for each miRNA species as described in Example 8. All LODs were     30 zeptomoles.<br>
3. Gene expression profiling<br>
Gene expression profiling was carried out on total RNA extracted from 20 different tissue types. Total RNA was purchased from Clontech (Palo Alto, CA, catalog number K4008-1, Human Total RNA Master Panel H). For let-7a, 50 ng of total RNA was tested in each reaction; for the other miRNA species, 100 ng of total RNA was tested. All reactions were set up as described in Example 8; primary reactions were run at the temperature optima for 1.5 hours; secondary reactions were as described above. The gene expression profiles for each miRNA species are presented in Figure 23. These results indicate that the INVADER assay can be used to examine miRNA expression in different tissue types. These data further suggest that let-7a and miR-125b are expressed in a wide variety of tissues; the other miRNA species appear to be more specific to a limited number of tissue types.<br>
Example 11 Effects of variable oligonucleotide length on INVADER assay detection of miRNA<br>
This example describes the impact of alterations in probe and INVADER oligonucleotide length on detection of the let-7a 22-nt miRNA. In particular, these experiments compare detection of an miRNA that forms perfect stacking interactions between the ends of the probe and INVADER oligonucleotides to detection of an miRNA that forms both 5* and 3' overlaps as well as to one that results in a single nucleotide gap at both the 5'and 3'ends.<br>
Figure 24 shows the results of analyzing three different types of designs. SEQ ID NOs: 5-6 shows a perfect stack between the 22-nt target and the flanking ends of the looped probe and INVADER oligonucleotides. In SEQ ID NOs: 83-84, both the probe and INVADER oligonucleotides are extended by a single base, resulting in both 5' and 3' overlaps. In SEQ ID NOs: 85-86, both the probe and INVADER oligonucleotides are shortened by a single base, relative to the designs in SEQ ID NOs: 5-6, resulting in a single nucleotide gap at both ends.<br>
INVADER assays were set up to test the performance of these oligonucleotide sets for detection of synthetic let-7a miRNAs. Reactions were carried out as described in Example 8 and included 100 pM synthetic let-7a miRNA, 1 nM probe and 10<br>
INVADER oligonucleotide. Primary reactions were run for 15 minutes at 53 °C; secondary reactions, for 5 minutes at 60°C, as described in Example 2. The results are presented in Figure 24.<br>
These data indicate that in this experiment, a single nucleotide overlap at both ends of the miRNA target resulted in an approximately 30% decrease in signal generation as well as a reduction of 2°C in optimal temperature. A one nucleotide gap at both ends of the target, however, did not reduce signal generation, though it did reduce the optimal reaction temperature by 5°C.<br>
Example 12 Discrimination of miRNA from precursor RNA and from encoding DNA<br>
Experiments were carried out to determine whether the INVADER miRNA assay discriminated the miRNA target itself from both its precursor RNA and from the DNA encoding the miRNA.<br>
A.	Precursor cross-reactivity test<br>
Precursor let-7 RNA (SEQ ID NO: 87) was transcribed in vitro and analyzed by capillary electrophoresis to determine whether it contained any fragments that might mimic the let-7a miRNA. The shortest contaminating fragment was estimated to be approximately 45 nt. LOD reactions were run essentially as described in Example 3 at precursor or synthetic 5' P let-7a mi-RNA concentrations as indicated in the table below. PI mixes contained 10 nM probe SEQ ID NO: 6 and 100 uM INVADER oligonucleotide SEQ ID NO: 5. Primary reactions were run at 53 °C for 1 hour; secondary reactions were run at 58 °C for 1 hour with secondary reaction mixes essentially as described in Example 3 (FRET probe SEQ ID NO: 21, SRT SEQ ID NO: 22, and ARRESTOR SEQ ID NO: 7). The results of this experiment indicated that this miRNA assay is approximately 4% cross reactive vs. the precursor RNA.<br>
B.	Discrimination of RNA vs. DNA signal<br>
Reactions were run to detect let-7a miRNA in cell lysates as described in Example 7. Prior to detection with the INVADER assay, aliquots 1 pi of 8 ng/p.1 RNAse A (Qiagen, Inc.) were added to 80 pi of cell lysate and incubated at 37°C for 2.25 hours. The RNAse A treated samples failed to generate any signal above background, indicating<br>
that signal generated in assays lacking RNAse A arises from detection of the miRNA target and not the encoding DNA (Figure 16). Further experiments were carried out in which RNAse A was added either prior to the primary reaction or prior to the secondary reaction. When RNAse A was added prior to the primary reaction, no signal was generated, consistent with the previous results. When RNAse A was added subsequent to the primary reaction, no loss of signal was observed, further indicating that the signal being detected is due to RNA and that there is no adverse effect of RNAse on other reaction components, e.g. the CLEAVASE enzyme.<br>
Example 13 Detection of a dual form miRNA<br>
Oligonucleotide designs were created for miR-124a. These oligonucleotides can be used to detect two naturally occurring miRNAs—one 21 nt in length and the other, 22 nt.<br>
Temperature optimization reactions were set up, essentially as described in Example 3, using 1 nM of synthetic miRNA target, 25 primary reaction and a 15 minute secondary reaction. The oligonucleotides used in these reactions are listed in Figure 5 (SEQ ID NOs: 90-92). Temperature profiles for the two different length miRNA targets are shown in Fig. 22 and indicate lhat the same oligonucleotide designs can be used to detect both targets.<br>
Example 14 Oligonucleotide designs for detection of an siRNA<br>
Approaches similar to those described in the preceding examples may similarly be used to detect siRNAs. Figure 25 illustrates two alternative INVADER assay designs for detection of a p-actin siRNA. This siRNA is described in Harborth, J. et al., Journal of Cell Science, 114:4557-4565 (2001). One design is presented for each the sense and antisense strands; exemplary oligonucleotides for detecting this siRNA are listed in Figure 26, SEQ ID Nos: 101-106.<br>
All publications and patents mentioned in the above specification are herein incorporated by reference. Various modifications and variations of the described method and system of the invention will be apparent to those skilled in the art without departing<br>
from the scope and spirit of the invention. Although the invention has been described in connection with specific preferred embodiments, it should be understood that the invention as claimed should not be unduly limited to such specific embodiments. Indeed, various modifications of the described modes for carrying out the invention that are obvious to those skilled in molecular biology, genetics, or related fields are intended to be within the scope of the following claims.<br><br><br><br><br><br>
We Claim:<br>
1.	A method for non-diagnostic analysis of interfering RNA in a sample,<br>
comprising:<br>
(a)	hybridizing an interfering RNA target to at least one nucleic acid of<br>
the kind such as herein described, wherein said nucleic acid<br>
contains a first region that is complementary to at least a portion of<br>
said interfering RNA target, and a second region that is not<br>
complementary to said interfering RNA target, wherein said second<br>
region of said nucleic acid forms a hairpin loop to form a half<br>
dumbbell structure when said nucleic acid is hybridized to said<br>
interfering RNA, to generate a detection structure; and<br>
(b)	detecting said detection structure in a manner known per se;<br>
wherein said  interfering  RNA is selected from the group consisting of<br>
microRNA and siRNA.<br>
2.	The method as claimed in claim 1, preferably comprising reacting said detection structure with at least one of a structure-specific nuclease or a DNA polymerase to form a modified detection structure, wherein said reacting comprises cleaving and/or extending at least one of said nucleic acid or said interfering RNA in said detection structure.<br>
3.	The method as claimed in claim 1, wherein said detecting comprises quantitating said interfering RNA.<br>
4.	The method as claimed in claim 1, wherein said detecting comprises forming an invasive cleavage structure, cleaving said invasive cleavage structure, and detecting the cleavage of said invasive cleavage structure.<br>
5.	The method as claimed   in claim 4, wherein said detection structure<br><br>
comprises first and second oligonucleotides configured to form an invasive cleavage structure in combination with said miRNA, wherein formation of an invasive cleavage structure with said microRNA requires both of said first and said second oligonucleotides.<br>
6.	The method as claimed in claim 4, wherein said detection structure comprises a first oligonucleotide configured to form an invasive cleavage structure in combination with said miRNA, wherein said first oligonucleotide and said microRNA in combination are sufficient to form an invasive cleavage structure.<br>
7.	The method as claimed in claim 5, preferably comprising a second nucleic acid, wherein said second nucleic acid contains a first region that is complementary to a portion of said interfering RNA target that is not complementary to said first nucleic acid, and a second region that is not complementary to said interfering RNA target, wherein said second region of said second nucleic acid forms a hairpin loop to form a half dumbbell structure when said nucleic acid is hybridized to said interfering RNA.<br>
8.	The method as claimed in claim 1, wherein said detecting comprises use of an invasive cleavage assay.<br>
9.	The method as claimed in claim 1, wherein said detecting comprises use of<br>
a detection assay selected from the group consisting of a sequencing<br>
assay(s), a polymerase chain reaction assay(s), a hybridization assay(s), a<br>
hybridization assay(s) employing a probe complementary to a mutation, a<br>
microarray assay(s), a bead array assay(s), a primer extension assay(s),<br>
an enzyme mismatch cleavage assay(s), a branched hybridization<br>
assay(s), a NASBA assay(s), a molecular beacon assay(s), a cycling probe<br>
assay(s), a ligase    chain reaction assay(s), an invasive cleavage structure<br><br>
assay(s), an ARMS assay(s) and a sandwich hybridization assay(s).<br>
10.	The method as claimed in claim 1, wherein said sample comprises a cell lysate.<br>
11.	The method as claimed in claim 1, wherein a plurality of different miRNAs are detected.<br>
12.	The method as claimed in claim 11, wherein said plurality of miRNAs comprise a first miRNA and a second miRNA that is said first miRNA having a polymorphism.<br>
13.	The method as claimed in claim 1, preferably comprising detecting a second nucleic acid target.<br>
14.	The method as claimed in claim 13, wherein said second nucleic acid target is RNA.<br>
15.	The method as claimed in claim 14, wherein said second nucleic acid target is selected from the group consisting of U6 and GAPDH.<br>
16.	The method as claimed in claim 1, wherein said interfering RNA is miRNA is selected from the group consisting of Let-7, miR-1, miR-135, miR-15, miR-16, miR125b, miR-1d, and miR124a.<br>
17.	A kit for carrying out the method as claimed in any of the preceding claims, for analyzing an interfering RNA in a sample, said kit comprising a nucleic acid configured for forming a detection structure when hybridized to a microRNA or siRNA target, wherein the nucleic acid comprises a first<br><br>
region that is complementary to at least a portion of said microRNA or siRNA target, and a second region that is not complementary to said microRNA or siRNA target, wherein said second region of said nucleic acid forms a hairpin loop to form a half dumbbell structure when said nucleic acid is hybridized to said microRNA or to said siRNA, to form a detection structure.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1kZWxucC0yMDA1LWFic3RyYWN0LSgwNy0xMC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">3157-delnp-2005-abstract-(07-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1ERUxOUC0yMDA1LUFic3RyYWN0LSgyOS0wOS0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">3157-DELNP-2005-Abstract-(29-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1kZWxucC0yMDA1LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">3157-delnp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1kZWxucC0yMDA1LWNsYWltcy0oMDctMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">3157-delnp-2005-claims-(07-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1ERUxOUC0yMDA1LUNsYWltcy0oMjktMDktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">3157-DELNP-2005-Claims-(29-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1kZWxucC0yMDA1LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">3157-delnp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1kZWxucC0yMDA1LWNvbXBsZXRlIHNwZWNpZmljYXRpb24oZ3JhbnRlZCkucGRm" target="_blank" style="word-wrap:break-word;">3157-delnp-2005-complete specification(granted).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1kZWxucC0yMDA1LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTYtMDMtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3157-delnp-2005-Correspondence Others-(16-03-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1kZWxucC0yMDA1LWNvcnJlc3BvbmRlbmNlLW90aGVycy0oMDctMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">3157-delnp-2005-correspondence-others-(07-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1ERUxOUC0yMDA1LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjktMDktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">3157-DELNP-2005-Correspondence-Others-(29-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1ERUxOUC0yMDA1LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMzAtMDktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">3157-DELNP-2005-Correspondence-Others-(30-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1kZWxucC0yMDA1LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">3157-delnp-2005-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1kZWxucC0yMDA1LWRlc2NyaXB0aW9uIChjb21wbGV0ZSktKDA3LTEwLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">3157-delnp-2005-description (complete)-(07-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1ERUxOUC0yMDA1LURlc2NyaXB0aW9uIChDb21wbGV0ZSktKDI5LTA5LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">3157-DELNP-2005-Description (Complete)-(29-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1kZWxucC0yMDA1LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">3157-delnp-2005-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1kZWxucC0yMDA1LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">3157-delnp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1kZWxucC0yMDA1LWZvcm0tMS0oMDctMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">3157-delnp-2005-form-1-(07-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1kZWxucC0yMDA1LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">3157-delnp-2005-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1kZWxucC0yMDA1LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">3157-delnp-2005-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1kZWxucC0yMDA1LWZvcm0tMi0oMDctMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">3157-delnp-2005-form-2-(07-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1ERUxOUC0yMDA1LUZvcm0tMi0oMjktMDktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">3157-DELNP-2005-Form-2-(29-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1kZWxucC0yMDA1LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">3157-delnp-2005-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1ERUxOUC0yMDA1LUZvcm0tMy0oMjktMDktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">3157-DELNP-2005-Form-3-(29-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1kZWxucC0yMDA1LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">3157-delnp-2005-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1kZWxucC0yMDA1LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">3157-delnp-2005-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1kZWxucC0yMDA1LUdQQS0oMTYtMDMtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3157-delnp-2005-GPA-(16-03-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1ERUxOUC0yMDA1LUdQQS0oMjktMDktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">3157-DELNP-2005-GPA-(29-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1kZWxucC0yMDA1LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">3157-delnp-2005-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1ERUxOUC0yMDA1LVBldGl0aW9uLTEzNy0oMzAtMDktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">3157-DELNP-2005-Petition-137-(30-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1ERUxOUC0yMDA1LVBldGl0aW9uLTEzOC0oMzAtMDktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">3157-DELNP-2005-Petition-138-(30-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzE1Ny1kZWxucC0yMDA1LXBldGl0dGlvbiAxMzgucGRm" target="_blank" style="word-wrap:break-word;">3157-delnp-2005-petittion 138.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="227721-process-for-the-preparation-of-1-2-dichloroethane-free-crystals-of-zonisamide-and-the-highly-pure-crystals-of-zonisamide.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="227723-dental-care-herbal-formulation-and-its-preparation-thereof.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>227722</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3157/DELNP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>05/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>30-Jan-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>19-Jan-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>15-Jul-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>THIRD WAVE TECHNOLOGIES, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>502 SOUTH ROSA ROAD, MADISON, WISCONSIN 53719-1256, U.S.A.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>LYAMICHEV, VICTOR</td>
											<td>2523 CARRIEDALE COURT, MADISON, WISCONSIN 53711, U.S.A.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>ALLAWI, HATIM T.</td>
											<td>1138 N. HIGH POINT ROAD, #110, MADISON, WISCONSIN 53717, U.S.A.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>NERI, BRUCE P.</td>
											<td>5714 KILKENNY PLACE, MADISON, WISCONSIN 53711, U.S.A.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>OLSON-MUNOZ, MARIL YN</td>
											<td>5878 TREE LINE DRIVE, MADISON, WISCONSIN 53711, U.S.A.</td>
										</tr>
										<tr>
											<td>5</td>
											<td>CHEHAK, LUANNE</td>
											<td>3211 VOLD COURT, APARTMENT 8, JAESVILLE, WISCONSIN 54536, U.S.A.</td>
										</tr>
										<tr>
											<td>6</td>
											<td>OLSON, SARAH M.</td>
											<td>2968 THINNES STREET, CROSS PLAINS, WISCONSIN 53528, U.S.A.</td>
										</tr>
										<tr>
											<td>7</td>
											<td>DAHLBERG, JAMES E.</td>
											<td>1119 MERRILL SPRINGS ROAD, MADISON, WISCONSIN 53705, U.S.A.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/11</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2003/041549</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-12-18</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/434,518</td>
									<td>2002-12-18</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/227722-method-for-non-diagnostic-analysis-of-interfering-rna by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:55:33 GMT -->
</html>
