<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/227868-a-novel-antibiotic-free-selection-system by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:17:53 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 227868:&quot;A NOVEL, ANTIBIOTIC-FREE SELECTION SYSTEM&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A NOVEL, ANTIBIOTIC-FREE SELECTION SYSTEM&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>This invention relates to a novel, antibiotic-free selection system comprising a bacterial cell of the kind such as herein described and a mutated or non-mutated vector carrying an araD gene, or a catalytically active fragment thereof added as a selection marker; said bacterial cell being deficient of araD gene or having combined deletions of the kind such as herein described.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>This invention relates to a novel, antibiotic-free selection system<br>
Field of the invention<br>
The present invention relates to a novel selection system, which is based on the use of an araD gene, a mutated form of an araD gene, a complementary sequence thereof, or a catalytically active fragment thereof as a selection marker and to the use of a bacterial strain deficient of the araD gene. The present invention further relates to novel vectors containing an araD gene, a mutated form of an araD gene, a complementary sequence thereof, or a catalytically active fragment thereof and to novel bacterial strains deficient of an araD gene. The present invention additionally relates to a method of selecting the cells transformed with a plasmid, which contains the gene of interest.<br>
Background of the invention<br>
An essential requirement for effective genetic engineering of bacteria and other cells propagated in cell cultures is the capacity to select the cells with a specific genotypic alteration. The most common selection strategy in recombinant DNA technology is to included selection marker in the cloning vector or plasmid. A selection marker can be a cloned gene or a DNA sequence, which allows the separation of the host cells containing the selection marker from those not containing it. The selection marker together with a suitable selection medium maintains the cloning vector in the cells. Otherwise, since the replication of plasmids is an energetic burden for the bacterial host, in a growing culture the bacteria, which have lost the plasmid, would have a growth advantage over the cells with the plasmid.<br>
For most purposes, an antibiotic resistance gene is a commonly used selection marker. However, for the production of recombinant therapeutics, where the goal is to generate a product, such as a DNA vaccine, in high yield for administration in patients, the use of antibiotic resistance genes presents problems: the spread of antibiotic resistant pathogens is a serious worldwide problem [Levy, S. B., J. Antimicrob. Chemother. 49 (2002) 25-30]. Therefore the antibiotic resistance genes cannot have extensive use in the pharmaceutical industry, and for instance, according to the regulations of the U.S. Food and Drug Administration, no antibiotic resistance genes are allowed in experimental DNA vaccines entering the third phase.<br>
Alternatively,   antibiotic-free  selection   systems   have   been   suggested. Such antibiotic-free selection systems include bacterial toxin-antitoxin<br><br><br>
systems [Engelberg-Kulka, H. and Glaser, G., Annu Rev Microbiol 53 (1999) 43-70], genes responsible for resistance against heavy metals, such as tellurium [Silver, S. and Phung, L. T., Annu Rev Microbiol 50 (1996) 753-789], and systems, in which the plasmid encodes a gene complementing a host auxotro-phy [Wang, M.D., etal., J. Bacteriol. 169 (1987) 5610-5614].<br>
US Patent Application 2000/0014476 A1 generally discloses, inter alia, the use of a non-antibiotic selection marker, which may be a gene whose product is necessary for the metabolism of the cell under certain culturing conditions, such as a catabolism gene, which makes it possible for the cell to assimilate a certain substance present in the culture medium (specific carbon or nitrogen source) etc. No specific examples of such suitable genes are given. This approach is not necessarily applicable for commercial production, since the deletion an essential component, such as an amino acid or a carbon source, from the growth medium reduces the yield, which is not desirable. Additionally, the manipulation of the growth medium in terms of omitting an essential nutritient may considerably increase the cost of the growth medium, since commercially available nutritient mixtures must be replaced by individual nutritients.<br>
For commercial therapeutic purposes it would be of advantage to use a gene, which is not essential for the growth of the host but whose manipulation still affects the growth in selected circumstances. Additionally, in view of the therapeutic use, it would be of advantage to use a gene, whose deletion leads to accumulation of compounds, which are toxic to the host cell but not toxic to mammalians, including humans. Also it would be of advantage to use smaller genes, which in turn would allow the construction of smaller plas-mids for which the energy consumption for replication is smaller and thus the growth rate of bacterial culture and plasmid yield are improved.<br>
Short description of the invention<br>
The object of the present invention is to provide a novel antibiotic-free selection system, which avoids the problems of previously disclosed selection systems for use in the production of recombinant therapeutic products.<br>
Another object of the invention is to provide a novel antibiotic-free selection system, which can be safely used in the production of recombinant therapeutic products in terms of the environment and the patient safety.<br>
A further object of the invention is to provide a novel antibiotic-free selection system, which can be cost-effectively used in the production of re-combinant therapeutic products using standard growth mediums.<br>
A still further object of the invention is to provide a novel antibiotic-free selection system, which provides an increased growth rate and improved yield.<br>
Yet another object of the present invention is to provide a novel vector containing a selection marker, which is non-toxic to the environment and to humans and which is capable of a long-term maintenance in the host.<br>
Yet another object of the present invention is to provide a novel host cell containing a gene defect, which is not hazardous to the environment.<br>
Still another object of the present invention is to provide a method for selection of cells carrying a gene of interest for the production of recombi-nant therapeutic products.<br>
It was surprisingly found that the objects of the present invention are met by the use of the araD gene, a mutated form of an araD gene, a complementary sequence thereof, or a catalytically active fragment thereof as a selection marker and the use of a specific bacterial host deficient of the araD gene.<br>
Accordingly, the present invention provides a novel selection system comprising a bacterial cell deficient of an araD gene into which a vector carrying an araD gene, a complementary sequence thereof, or a catalytically active fragment thereof has been added as a selection marker. One embodiment of the present invention relates to a selection system wherein the araD gene is the araD gene or the L-ribulose-5-phosphate 4-epimerase (EC 5.1.3.4.). Another embodiment of the present invention relates to a selection system wherein the araD gene is mutated.<br>
The present invention further provides novel vectors, which contain an araD gene, a mutated form of an araD gene, a complementary sequence thereof, or a catalytically active fragment thereof as a selection marker.<br>
The present invention further provides novel bacterial strains, which are deficient of the araD gene.<br>
The present invention further provides a method of selecting the cells transformed with a plasmid, which contains 1) the araD gene, a mutated form of an araD gene, a complementary sequence thereof, or a catalytically active fragment thereof as a selection marker and 2) the gene of interest, the<br><br>
method comprising inserting said plasmid into the araD deficient host cell and growing the cells in a growth medium containing arabinose.<br>
Drawings<br>
Figure 1 shows the use of arabinose as a carbon source by the E coll cells (Lin, 1987).<br>
Figure 2 shows the map of S6wtd1EGFP. The coding sequences for the dIEGFP, E2 and kanamycin resistance marker aminogiycoside-3'-0-phosphotransferase (kana) are indicated by arrows. Additional features are indicated by solid boxes: 10E2BS - ten BPV E2 binding sites with high affinity; CMV-tk - human cytomegalovirus immediately early promoter and HSV Th gene leader sequence; intron - rabbit beta-globin gene intron with optimized SD and SA sites; tkpa - HSV Tk gene poiyadenylation signal; RSV LTR -Rous sarcoma virus long terminal repeat; bgh pA - bovine growth hormone gene poiyadenylation signal; pUCori - bacterial origin of replication derived from the pUC18 plasmid.<br>
Figure 3 shows the map of S6wtd1EGFP/fana/araD1. The coding sequences for the dIEGFP, E2, kanamycin resistance marker aminoglycoside-S'-O-phosphotransferase   (kana)   and   L-ribulose-5-phosphate  4-epimerase (araD) are indicated by arrows. Additional features are indicated by solid boxes: 10E2BS - ten BPV E2 binding sites with high affinity; CMV-tk - human cytomegalovirus immediately early promoter and HSV Th gene leader sequence; intron - rabbit beta-globin gene intron with optimized SD and SA sites; tkpa - HSV Tk gene poiyadenylation signal; RSV LTR - Rous sarcoma virus long terminal repeat; bgh pA - bovine growth hormone gene poiyadenylation signal; pUCori - bacterial origin of replication derived from the pUC18 plasmid. Figure 4 shows the map of S6wtd1EGFP/cana/araD2. The coding sequences for the dIEGFP, E2, kanamycin resistance marker aminogiycoside-3-O-phosphotransferase   (kana)   and   L-ribulose-5-phosphate   4-epimerase (araD) are indicated by arrows. Additional features are indicated by solid boxes: 10E2BS -ten BPV E2 binding sites with high affinity; CMV-tk - human cytomegalovirus immediately early promoter and HSV Th gene leader sequence; intron - rabbit beta-globin gene intron with optimized SD and SA sites; tkpa - HSV Tk gene poiyadenylation signal; RSV LTR - Rous sarcoma virus long terminal repeat; bgh pA - bovine growth hormone gene poiyadenylation signal; pUCori - bacterial origin of replication derived from the pUC18 plasmid.<br><br>
Figure 5 shows the map of S6wtd1EGFP/araD1. The coding sequences for the dIEGFP, E2 and L-ribulose-5-phosphate 4-epimerase (araD) are indicated by arrows. Additional features are indicated by solid boxes: 10E2BS - ten BPV E2 binding sites with high affinity; CMV-tk - human cy-tomegalovirus immediately early promoter and HSV Th gene leader sequence; intron - rabbit beta-globin gene intron with optimized SD and SA sites; tkpa -HSV Tk gene polyadenylation signal; RSV LTR - Rous sarcoma virus long terminal repeat; bgh pA - bovine growth hormone gene polyadenylation signal; pUCori - bacterial origin of replication derived from the pUC18 plasmid.<br>
Figure 6 shows the map of S6wtd1EGFP/araD2. The coding sequences for the dIEGFP, E2 and L-ribulose-5-phosphate 4-epimerase (araD) are indicated by arrows. Additional features are indicated by solid boxes: 10E2BS - ten BPV E2 binding sites with high affinity; CMV-tk - human cy-tomegalovirus immediately early promoter and HSV Th gene leader sequence; intron - rabbit beta-globin gene intron with optimized SD and SA sites; tkpa -HSV Tk gene polyadenylation signal; RSV LTR - Rous sarcoma virus long terminal repeat; bgh pA - bovine growth hormone gene polyadenylation signal; pUCori - bacterial origin of replication derived from the pUC18 plasmid.<br>
Figure 7A and 7B shows the electrophoretic analysis of the plasmid DMA of the S6wtd1EGFP/araD1 (7A) and S6wtd1 EGFP/araD2 (7B) extracted from the E. coli strain AG1 delta araD grown in different media.<br>
Figure 8 shows the restriction pattern analysis of the plasmid DNA of the S6wtdiEGFP/araD1 and S6wtd1EGFP/araD2 extracted from the E. coli strain AGIdelt∆araD<br>
Figure 9 shows the electrophoretic analysis of the S6wtd1EGFP/araD2 in stability assay.<br>
Figure 10A and 10B shows the restriction pattern analysis of the S6wtd1EGFP/araD2 in stability assay.<br>
Figure 11 shows the growth parameters of fed-batch fermentation of AGI∆araD S6wtd1EGFP/araD2 measured and registered during fermentation. The abbreviations are as follows: sPump = feeding speed; pO2 = the oxygen concentration; Temp = growth temperature; mys = desired growth rate; OD = optical density at 600nm.<br>
Figure 12 shows the scheme of lysis and purification of AGI∆araD S6wtd1EGFP/araD2.<br>
Figure 13 shows the araD locus sequence of clone #13.<br><br>
Figure 14 shows the map of plasmid p3hCG.<br>
Figure 15 shows the map of plasmid paraDMgB.<br>
Figure 16 shows the map of plasmid p3araD1hCG.<br>
Figure 17 shows the map of plasmid p3araD2hCG.<br>
Figure 18 shows the results of the analysis of L-arabinose sensitivity of E. co// strains with disrupted araD.<br>
Figure 19 shows the results of the analysis of the L-arabinose sensitivity in M9 and yeast extract medium with different glucose and arabinose concentrations.<br>
Figure 20 shows the map of plasmid p2 MG C #11.<br>
Figure 21 shows the map of plasmid paraD MG C #145.<br>
Figure 22 shows the E. co// genomic fragment containing the sgbE gene.<br>
Figure 23 shows the E. co// genomic fragment containing ulaF gene.<br>
Detailed description of the invention<br>
The present invention is based on an effort to find an alternative, antibiotic-free selection system, which could be used in the production of recom-binant therapeutic products to be administered in vivo, especially in the production of DNA vaccines. Surprisingly it was found that the araD gene involved in the pentose phosphate pathway of both prokaryotic and eukaryotic organisms, such as mammalians including humans, can be successfully used as a selection marker in an auxotrophic host cell for the plasmid. The use of the auxotrophy has the advantage of not involving a use or generation of toxic substances that could later contaminate the plasmid preparation.<br>
An efficient selection system has been constructed on the basis of araD/araC genes [Ariza, R. R., et a/., Carcinogenesis 14 (1993) 303-305]. However, this selection system has been used in the studies on the mechanisms of mutagenesis but not used before as a selection marker for plasmid maintenance. Ariza et a/, used a strain where the araC gene contains a termination codon and the araD gene is inactivated. A product of the supF gene, which codes for a suppressor tRNA, was introduced on the plasmid. In the presence of active suppressor tRNA, enzymatically active product from araC was produced causing cell growth arrest (because araD was inactive). This system allows to study the suppression of mutations by supF tRNA: in case sup? is inactivated by mutation, the cells can grow on arabinose. Therefore, this selection system is based on araC gene and not on araD gene. araD was<br><br>
not introduced into a plasmid, nor was the system designed or characterized for plasmid production purposes.<br>
The araD gene codes for an enzyme which is responsible for epim-erization of ribulose-5-phosphate to xylulose-5-phosphate (Fig. 1) and therefore allows the use arabinose in the pentose phosphate pathway [Engelsberg, E., et a/., J. Bacteriol. 84: (1962) 137-146]. If araD is inactivated, ribulose-5-phosphate accumulates in the bacterial cell leading to growth arrest.<br>
If the chromosomal copy of araD is inactivated in the host cell and an intact copy of the araD gene, a mutated form of the araD gene, a complementary sequence thereof, or a catalytically active fragment thereof is inserted into the plasmid, the growth advantage of the plasmid-containing cells in medium containing L-arabinose is achieved as a result from two effects. First, the plasmid-containing cells can use arabinose as a carbon source, and second, the toxic ribulose-5-phosphate does not accumulate. This allows the use of rich growth media supplemented with arabinose. In rich media the E. coli cells grow fast and the plasmid yield is high. Inexpensive standard components of the bacterial growth media, such as yeast extract, can be used as an amino acid source. The traces of ribulose-5-phosphate that theoretically could contaminate the plasmid preparation are not a problem, when the preparation is administered in vivo, as ribulose-5-phosphate can be efficiently metabolized by human cells and is not toxic.<br>
The use of mutated form of the araD gene offers particular advantages. Selection systems of the invention comprising a bacterial cell deficient of an araD gene into which a vector carrying a mutated form of the araD gene as a selection marker produce an optimal concentration of the araD gene product L-ribulose-5-phosphate 4-epimerase to afford rapid uninhibited growth of the bacteria. Similar advantaged are obtained by the use selection systems containing a vector carrying an intact araD gene but comprising deletions or mutations elsewhere in the araD gene locus.<br>
The selection system of the invention comprises 1) a vector carrying an araD gene, a mutated form of the araD gene, a complementary sequence thereof, or a catalytically active fragment thereof as a selection marker and 2) a specific bacterial strain deficient of the araD gene into which the vector has been added. When the specific host deficient of the araD gene is cultured in the presence of arabinose, the only surviving cells are those containing the<br><br>
vector, which contains an araD gene, a mutated form of the araD gene, a complementary sequence thereof, or a catalytically active fragment thereof.<br>
In the selection system of the invention any expression vector commonly used in the production of therapeutic products can be employed, whereby the araD gene, a mutated form of the araD gene, a complementary sequence thereof, or a catalytically active fragment thereof is inserted into the vector using methods generally known in the art. In the present context, the araD gene preferably comprises the sequence identified by SEQ ID NO. 1, by SEQ ID NO. 19, or a sequence hybridizable thereto. However, any applicable araD genes are also contemplated. In the present context, the term "a catalytically active fragment of the araD gene" is any gene fragment coding a poly-peptide or a protein capable of epimerization of L-ribulose-5-phosphate to D-xyIulose-5-phosphate. In a specific embodiment of the invention the araD gene, a complementary sequence thereof, or a catalytically active fragment thereof is inserted in the vector capable of a long-term maintenance and thereby capable of providing a stable expression of the desired antigen(s).<br>
In another specific embodiment of the invention a mutated form of an araD gene, a complementary sequence thereof, or a catalytically active fragment thereof is inserted in the vector capable of a long-term maintenance and thereby capable of providing a stable expression of the desired antigen(s).<br>
In a specifically preferred embodiment of the invention the vector used is an expression vector comprising:<br>
(a)	a DNA sequence encoding a nuclear-anchoring protein opera-<br>
tively linked to a heterologous promoter, said nuclear-anchoring protein com<br>
prising (i) a DNA binding domain which binds to a specific DNA sequence, and<br>
(ii) a functional domain that binds to a nuclear component, or a functional<br>
equivalent thereof; and<br>
(b)	a multimerized DNA sequence forming a binding site for the nu<br>
clear anchoring protein, wherein said vector lacks a papiiloma virus origin of<br>
replication, and<br>
(c)	an araD gene, a mutated form of an araD gene, a complemen<br>
tary sequence thereof, or a catalytically active fragment thereof.<br>
Such vectors have been described in detail in the international patent application W002/090558, which is incorporated herein by reference.<br>
Most preferably the vector used in the selection method of the present invention is an expression vector comprising:<br><br>
(a)	the E2 protein of Bovine Papilloma Virus type 1 (BPV), and<br>
(b)	multiple binding sites of the BPV E2 protein incorporated into the<br>
vector as a cluster, where the sites can be as head-to-tail structures or can be<br>
included into the vector by spaced positioning, wherein said vector lacks a<br>
papilloma virus origin of replication, and<br>
(c)	the araD gene, a complementary sequence thereof, or a catalyti-<br>
cally active fragment thereof.<br>
In the selection system of the invention in principle any known host deficient of the araD gene and suitable for use in the production of therapeutic products could be employed. In the present connection the term "deficient" denotes a host, in which the araD gene is either totally deleted or inactivated by any known method.<br>
In a preferred embodiment of the invention an Escherichia coli strain, preferably commercially available E. coli strains DH5alpha-T1, AG1 or JM109, from which the araD gene has been deleted with generally known methods, such as those described below in the Examples, is used. In another preferred embodiment of the invention an E. coli strain, preferably E coli strain DH5alpha-T1, AG1 or JM109, into which combined deletions have been made for depletion of other genes encoding proteins with L-ribulose-5-phosphate 4-epimerase activity. Alternatively, commercially available E coli strains, preferably E coli strains DH5alpha-T1, AG1 or JM109, in which the araD gene and/or   other   genes   encoding   proteins   with   L-ribulose-5-phosphate   4-epimerase activity have been inactivated by any known method can be employed. In the method for selection of cells carrying a gene of interest for the production of recombinant therapeutic products, the gene of interest is inserted into host cells deficient of an araD and/or other genes encoding proteins with L-ribulose-5-phosphate 4-epimerase activity using method well known in the   • art and the cells are cultured in a growth medium containing arabinose under culturing medium and conditions suitable the host in question.<br>
Any growth medium suitable for culturing E. coli cells can be used. For commercial production the growth medium will naturally be optimized in terms of the yield. Examples of suitable growth media are commercially available growth media, such as M9 and LB (available from several manufacturers, such as Fermentas, Lithuania). The amount of arabinose added in the growth medium is not critical but naturally arabinose should be present in an amount that is sufficient for the total culturing period. As low amount as 0.1% has been<br><br>
found sufficient for the selection. Typically arabinose is added to the medium in an amount of about 0.1% to about 2.0%, preferably in an amount of about 0.2% to about-1,0%, most preferably 0.2% to about 0.5%. However the effect of L-arabinose is observed at concentrations as low as 0.01% and L-arabinose can be added up to 5% in the growth medium. In a special embodiment, where L-arabinose is used both as a selecting agent and as a limited carbon source, 0.2% of L-arabinose is a suitable amount to be added into the growth medium. The selection system of the invention is suitable for use in any expression system. It is especially suitable for use in the expression of recombi-nant therapeutic products, such as DMA vaccines, intended for use in vivo, since the problems associated with the use of antibiotic resistance genes are avoided. Likewise the selection system of the invention is suitable for use in the production of recombinant proteins.<br>
The possible contamination of arabinose in the final product resulting from the preparation process is inconsequential, since arabinose is editable sugar contained in foods naturally and as an additive and thus not toxic to mammalians including humans.<br>
Additionally, the araD gene is smaller in size than the commonly used antibiotic resistance genes against, for instance, ampicillin and tetracyclin and of similar size to kanamycin and chloramphenicol resistance genes. This affords an additional advantage, since it allows the construction of small plas-mids for which the energy consumption for replication is smaller than for large plasmids. Thereby both the growth rate of bacterial culture and plasmid yield are increased.<br>
The present invention may be better understood by reference to the following non-limiting Examples, which are provided as exemplary of the invention. The following examples are presented in order to more fully illustrate the preferred embodiments of the invention. They should in no way be construed, however, as limiting the broad scope of the invention.<br>
Example 1<br>
Cloning of araD selection plasmids<br>
For cloning araD selection constructs plasmid S6wtd1EGFP (Figure 2) was used. It has pMB1 origin of replication and kanamycin resistance marker as functional elements of plasmid backbone. The kanamycin resistance<br><br>
in this plasmid is conferred by gene that is derived from E coli transposon Tn903.<br>
The araD gene was amplified using polymerase chain reaction (PCR) from E. coli DH5a chromosome according to standard procedure. The PCR product was cloned into selected plasmids in two different orientations with the primer pairs s6araDL1 + s6araDR1 or s6araDL1 + s6araDR1, generating products named araD1 and araD2, respectively: s6araDL1:<br>
(Sequence Removed)<br>
(SEQ ID NO. 2);<br>
 s6araDR1:<br>
(Sequence Removed)<br>
 (SEQ ID NO. 3); <br>
s6araDL2:<br>
(Sequence Removed)<br>
 (SEQ ID NO. 4); s6araDR2:<br>
(Sequence Removed)<br>
 (SEQ ID NO. 5);<br>
The primers were designed so that P2 promoter from plasmid pBR322 (used for driving the tetracycline resistance gene in pBR322) and termination sequence from trp operon of E coli were added during PCR to the upstream and downstream of araD coding sequence, respectively.<br>
PCR products of 814 and 815 bp were cloned into pUC18 vector linearized with Hincll (Fermentas, Lithuania) and correct sequences were verified by sequencing using universal sequencing primers<br>
M13F22: GCCAGGGTTTTCCCAGTCACGA (SEQ ID NO. 6) and M13R24: GAGCGGATAACAATTTCACACAGG (SEQ ID NO. 7) and araD specific primers<br>
araD F311: CCMCTCACCGGCTGCTCTATC (SEQ ID NO. 8), araD F614: MTGCCGAAGATGCGGTGCATAAC (SEQ ID NO. 9), araD R700: TAACTGCGGCGCTMCTGAC (SEQ ID NO. 10), and araD R421: GGTTGCTGGAATCGACTGAC (SEQ ID NO. 11). The mutations in amplified sequences were repaired by recombination of different clones.<br><br>
For cloning araD into S6wtd1EGFP, the vector was linearized by partial digestion with restriction enzyme Pagl (position 4761) (Fermentas, Lithuania) and the DNA 5'-termini were dephosphorylated with Calf Intestine Alkaline Phosphatase (CIAP; Fermentas, Lithuania). araDI and araD2 fragments were cut out from pUC18 with Ncol (Fermentas, Lithuania) and ligated toS6wtd1EGFP/Pagl.<br>
Both ligation mixtures were transformed into E. coll DH5a competent cells and plated onto dishes containing LB medium containing 50 µg/ml kanamycin and incubated at 37°C over night. Colonies were first analysed with colony PCR, after which the DNA was isolated and digested with different restriction enzymes.<br>
The cloning resulted in plasmids S6wtd1EGFP/cana/araD1, S6wtd1EGFP/ra/7a/araD2, which are shown in Figures 3 and 4.<br>
To remove the kanamycin resistance marker gene from the plasmids, S6wtd1EGFP/cana/araD1 and S6wtd1EGFP/cana/araD2 were digested with restriction endonuclease Bcul (Fermentas, Lithuania) and a 6473 bp vector fragment was self-ligated.<br>
The ligation mixtures were transformed into an E. coli AG1 ∆araD strain (see Example 3) and plated onto dishes containing M9 media supplemented with 2% L-arabinose and incubated at 37°C for 36 hours. Colonies were first analyzed with colony PCR, after which the DNA was isolated and digested with different restriction enzymes. The cloning resulted in plasmids S6wtd1EGFP/araD1, S6wtd1EGFP/araD2, respectively, are shown in Figures 5 and 6.<br>
The bacterial colonies containing S6wtd1EGFP/araD1 and S6wtd1EGFP/a/'aD2 were grown in two different media: LB supplemented with 2.5% L-arabinose and M9 supplemented with 0.2% L-arabinose at 37°C with vigorous shaking. The ceils were harvested and the plasmid DNA was extracted from the cell using QIAprep Spin Miniprep Kit (QIAGEN) and analysed by agarose gel electrophoresis (Figures 7A and 7B, respectively).<br>
The plasmid DNA samples from cultures in LB and M9 media were analysed by agarose gel electrophoresis before and after digestion with restriction endonuclease Pagl (Fermentas, Lithuania), (Figure 8). The predicted sizes of the fragments obtained in the Pagl digestion were 3954 and 2519 bp for S6wtd1EGFP/araD1 and 4315 and 2157 bp for S6wtd1EGFP/araD2. Lambda DNA digested with Eco91l (M15 in Figure 8C) and lambda DNA digested with<br><br>
EcoRI/ Hindlli (Fermentas, Lithuania) (M3 in Figure 8C) were used as molecular weight markers. All analyzed bacterial clones contained the correct plasmid in the restriction enzyme analysis, but the DNA yield was very low when the plasmids were grown in LB media. Two of the analyzed bacterial clones from four S6wtd1EGFP/araD2 clones (#13 and #14 in Figure 8B) had higher growth rate when grown in M9 media supplemented with 0,2% L-arabinose (Figures 7 and 8), which resulted in higher plasmid yield per culture.<br>
Further analysis of these two clones with improved growth was performed. These two plasmids had the same structure as the other plasmids as judged by restriction analysis. The plasmids were extracted from the bacteria and further characterized by sequencing the araD gene locus. The araD locus sequence of clone #13 (SEQ ID NO. 18; SEQ ID NO. 19) indicated that araD gene coding sequence carries a STOP codon instead of a codon for Glutamine in position 8 of L-ribuIose-5-phosphate 4-epimerase. This mutation resulted from the replacement of Cytidine in codon 8 of L-ribulose-5-phosphate 4-epimerase (araD coding sequence) (5'-CAG-3') with the Thymidine, resulting in a STOP codon (5'-TAG-3'). The plasmid carrying such a mutation in araD gene effectively provided the ability to grow in the selective medium in the presence of L-arabinose although the coding sequence contains the STOP codon. It has been demonstrated that the STOP codon UAG is effectively read through by the ribosomes of Escherichia coll, when such a STOP is in the beginning of the coding sequence [for reference, see review Murgola, E. J., Annu. Rev. Genet. 19 (1985) 57-80], Without binding by the theory, we hypothesized that the high yield of the plasmid, which is an indication of rapid uninhibited growth of the bacteria, requires an optimal concentration of the araD gene product L-ribuIose-5-phosphate 4-epimerase.<br>
The analysis of clone #14 araD locus sequence indicated that the araD coding sequence is perfect as predicted. However, the sequence rearrangements near the araD promoter covering the E2 protein binding sites were observed (see Figure 13, SEQ ID NO. 18). These data suggested additionally that such rearrangements near the promoter might result in the down-regulation of the promoter activity, therefore the level of the araD product.<br><br>
Example 2<br>
Cloning of mutated araD selection plasmids<br>
For cloning of mutated araD selection constructs plasmid p3hCG (Figure 14) carrying kanamycin resistance [transposon Tn5 derived kanamycin resistance marker (neo) gene] was cleaved with the restriction endonucleases Bcul and Hindlll, the ends were filled in using Klenow Fragment (Fermentas, Lithuania) and the fragment with the size of 4647 bp was purified from the gel after agarose gel electrophoresis. The pMB1 origin of replication and the araD sequence carrying the C to T mutation, which results in a STOP codon in position 8 of the araD gene coding sequence, was excised from the plasmid paraDMgB (Figure 15) with the restriction endonucleases Bcul and Eco52l, the ends were filled in using Klenow Fragment (Fermentas, Lithuania), and the DNA 5'-termini were dephosphorylated with Calf Intestine Alkaline Phos-phatase (CIAP; Fermentas, Lithuania). The fragment with the size of 1532 bp was purified from the gel after agarose gel electrophoresis and ligated with the 4647 bp fragment obtained above. Escherichia coli AG1 araD deficient strain was transformed with this ligation mixture and plated onto agar plates containing selective M9 medium with 0.5% yeast extract, 2% L-arabinose and 25  µglm\ of kanamycin. The colonies were inspected 24 hours after the plating and showed that the size of the colonies was uniform. The plasmids were extracted from the bacteria and further characterized by sequencing of the araD gene locus.<br>
The cloning resulted in plasmids p3araD1hCG and p3araD2hCG, which are shown in Figures 16 and 17, respectively. According to the sequence analysis, the bacteria contained un-rearranged plasmids with the mutation C to T in codon 8 (p3araD1hCG; Figure 16; p3araD2hCG, Figure 17).<br>
When this experiment was repeated with the wild type sequence and transformed plates were inspected 24 hours after the transformation, the result was different. Two types of colonies were observed: first, large size colonies, and small colonies, which had a retarded growth. The sequence analysis of these plasmids indicated that araD gene coding sequence carries a STOP codon instead of a codon for glutamine (plasmid #3A, araD2) or the mutation had occurred in the Shine-Dalgarno sequence in the ribosomal binding site (AGGAG was replaced with AGTAG) (plasmid #2A, araD2). Plasmid #7 (araD1) had the correct sequence in all araD gene locus regions, however, the<br><br>
baciena grew very slowly and resulted in a 10 times lower plasmid yield when were grown in liquid media.<br>
Example 3<br>
Construction of arabinose sensitive ∆araD Eschsrichia coli strains.<br>
Three E coli strains, DH5alpha T1, AG1 and JM109, were used to construct ∆araD mutants. The araD gene in E. coli genome was disrupted using the method described by Datsenko and Wanner [PNAS 97 (2000) 6640-6645]. This method exploits a phage A Red recombination system. Briefly, the strategy of this system is to replace a chromosomal sequence with a selectable antibiotic resistance gene that is generated by PCR by using primers with homology extensions. This is accomplished by Red-mediated recombination in these flanking homologies.<br>
For transformation of the pKD46 (Datsenko and Wanner, supra), which encodes the phage/1 recombination system, E. coli, the cells were made chemically competent using RF1 and RF2 solutions:<br>
RF1 100ml<br>
(Table Removed)<br>
RF2 100ml<br>
(Table Removed)<br>
The cells were grown in 2 ml of LB medium to OD600 0.2-0.5. The culture was centrifuged and the pellet was resuspended in 1 ml of RF1. The mixture was kept on ice for 10 min and centrifuged. The pellet was suspended<br><br>
in 100µl of RF2 and the suspension was kept on ice for 30-45 min. Approximately 50 ng of pKD43 was added and the cells were kept on ice for additional -30 min followed by heat shock of 5 min.at 37°C, After-incubation for 10 min on ice 900µI of SOB medium was added to the transformed cells and the mixture was incubated at 37°C for one hour. Cells were plated on LB medium containing ampicillin (100µg/ml). The colonies were picked from the transformation plates and grown in 2 ml of the same medium to OD60 of approximately 1 and glycerol stocks were made (2 ml culture + 0.6 ml 50% glycerol). The stocks were stored at -80'C.<br>
For disruption of the araD gene a linear PCR product which contains kanamycin resistance gene was generated. Plasmid pKD13 (Datsenko and Wanner, PNAS vol. 97, no 12, June 2000) was used as the PCR template. Primers used were ara(prt) and ara(pr4): ara(prl)<br>
(Sequence Removed)<br>
 (SEQ ID NO. 12) ara(pr4)<br>
(Sequence Removed)<br>
' (SEQ ID NO. 13)<br>
These primers have the complement sequences with pKD13 for annealing in PCR and with the araD gene for homologous recombination.<br>
The PCR reaction mixture was as follows: PFU native buffer (5 µl), 10 mM dNTP (5 µ\), primer ara(prl) 10 µM (1 µl), primer ara(pr4) 10 µM (1 µl), pKD13 100 ng (2 µl), DMSO (4 µl), PFU 2.5 U (1µI), and mQ water up to 50  µl.<br>
The PCR procedure was as follows: denaturation 45 s, 96'C, annealing 4.5 s, 50°C, synthesis 2 min 30 s, 72°C, 25 cycles. The PCR product obtained was 1.4 kb.<br>
Five reactions were performed simultaneously; the DNA was purified from 2% agarose gel using Ultrapure purification Kit (MoBio Laboratories Inc.) and eluted with 60µI of water. The DNA was concentrated with ethanol precipitation and dissolved in 5µI of water. The final concentration was 0.6  µg/ µl An aliquot of 1.5µl was used in one electroporation.<br>
The PCR product was electroporated into DH5alpha T1 pKD46, AG1 pKD46 (Datsenko and Wanner, supra), and JM109 pKD46 E. col! cells. First, 200 ml of YENB medium containing 10 mM of L-arabinose for the induc-<br><br>
tion of the recombination system and 100µg/ml ampicillin was inoculated with an overnight culture of DH5alpha T1 pKD46, AG1 pKD46, and JM109 pKD46 E. co// cells. The cultures were grown at 30°C to OD600 0.8 (DH5alpha T1 and JM109) and 0.6 (AG1). The bacteria was collected by centrifugation at 4,000 g for 10 min at 4°C, washed twice with 20 ml of sterile water and once with 20 ml of sterile water containing 10% glycerol. The cells were suspended in 300 µI water containing 10% glycerol. 40µl of competent cells were used in one elec-troporation.<br>
The electroporation was performed with BioRad E coll Pulser using 0.2 cm cuvettes and 2.5 kV. The purified PCR product (1.5µI) was added to the competent cells, kept on ice for 1 min, and-immediately after the electroporation, 2 ml of warm SOB medium was added to the cells and the mixture was incubated at 37'C for 1 hour. The cells were plated on LB medium containg kanamycin (25µg/ml). 100 pg of large kanamycin resistant plasmid (GTU-MuItiHIV C-clade) was used as a positive control, no plasmid was added to the negative control. The transformation efficiency was 106 for AG1 and 107 for JM109 for positive control. There were no colonies on the negative control plate, 215 colonies were obtained on JM109+PCR product plate, 70 colonies on AG1+PCR product plate and 50 colonies on DH5alpha T1+PCR product plate.<br>
Example 4<br>
Testing of the E co// DH5alpha T1 ∆araD, AGI∆araD and JM109∆araD strains<br>
The colonies obtained from the electroporation as described in Example 2 were tested for the presence of kanamycin resistance gene by colony PCR using primers araVlisF (51 CGGCACGAAGGAGTCAACAT 3'; SEQ ID NO. 14) and araVIisR (51 TGATAGAGCAGCCGGTGAGT 3'; SEQ ID NO. 15) which contain annealing sites on the araD gene near the insertion site. A PCR product of 272 bp was expected from the E. coli DH5alpha T1, AG1 and JM109 strains without insertion in araD and a 1545 bp product, if the PCR product had been inserted in the araD gene. Three colonies of DH5alpha T1 ∆araD, nine colonies of AGI∆araD and 14 colonies of JM109∆araD out of 15 were checked and each gave the 1545 bp product. It was therefore concluded that these strains contained the kanamycin resistance gene insertion.<br><br>
To confirm the insertion of kanamycin gene another colony PCR was performed using primers kanaSF (5TCAGATCCTTGGCGGCAAGA31; SEQ ID NO. 16) and araVR-(5'TGTAATCGACGCCGGAAGGT3'; SEQ ID NO. 17). These primers produce a 435 bp product, if the kanamycin resistance gene has been inserted into the araD gene. Six colonies from AGI∆araD and JM109∆araD strains and three colonies of DH5alpha T1 ∆araD strains were tested and all gave the correct product.<br>
Six colonies of AGI∆araD and JM109∆araD, and three colonies of DH5alpha T1 ∆araD were plated on LB medium containing 25µg/ml of kanamycin and incubated at 37°C overnight to eliminate the pKD46 plasmid, which has a temperature sensitive replication origin. The cells were tested for am-picillin sensitivity by replica plating on LB medium and LB medium containing ampiciilin. None grew on the medium containing ampicillin and it was concluded that the bacteria does not contain the pKD46 plasmid any more.<br>
The arabinose sensitivity was tested on the produced AGI∆araD and JM109∆araD strains. One colony of AGI∆araD and one colony of JM109∆araD were each inoculated into 2 ml LB. The cultures were grown for 8 hours, diluted 1:100 into M9 medium containing 0.2% glycerol, 25µg/ml kanamycin, 0.01% thiamine (0.05% praline for JM109∆araD) and different concentrations of L-arabinose were added in the growth medium. The cultures were grown overnight at 37°C in shaker incubator and OD600 was measured (Table 1).<br>
Table 1; Testing of arabinose sensitivity.<br>
(Table Removed)<br>
As can be seen from Table 1, as low amount as 0.1% of L-arabinose is enough to inhibit the growth of the ∆araD strains of the invention.<br>
The arabinose sensitivity was further tested on AGI∆araD, DH5aIphaT1 ∆araD and JM109∆araD as above but using lower concentrations<br><br>
of L-arabinose. The results are given in Figure 18. As can be seen in Figure 18, as low an amount as 0.0005% of L-arabinose is enough to inhibit the growth of the ∆araD strains of the invention.<br>
Additionally the L-arabinose sensitivity was tested in M9 and yeast extract medium with different glucose and arabinose concentrations (0.2% glucose, 0.2% arabinose, 2% arabinose). The cultures were incubated at 37°C in a shaker incubator overnight. Then the OD600 was measured to quantitate the cell density. The results are given in Figure 19.<br>
Both concentrations of arabinose (0,2% and 2%) inhibited the growth of the ∆araD strains of the invention. However, the growth of strains with intact araD gene was not inhibited.<br>
Additionally the plasmid DNA yield of the ∆araD strains was tested. Plasm id S6wtd1EGFParaD2 prepared in Example 1 was transformed into AG1 ∆araD and JM109∆araD strains. Competent cells were prepared with RF1 and RF2 solutions as described in Example 3.<br>
The colonies from the transformation plates were inoculated into 2 ml of M9 medium containing 0.5% yeast extract and 25µg/ml kanamycin + 0.01 % thiamine + L-arabinose (2% and 0.2%).<br>
The cultures were incubated at 37*C for 17 hours. Then the OD600 was measured to quantitate the cell density and the plasmid DNA was extracted with Qiagen Miniprep Kit. Coefficient 2.8 (OD600/ml) was used for mini-prep isolation to get comparable results. The results are shown in Table 2.<br>
DNA concentration was measured with spectrophotometer as OD at 260 nm. For microscopic analysis a drop of bacterial culture was applied on glass slide and covered with cover slip. The culture was visually inspected at a 100xmagnification with an objective in oil immersion.<br><br>
Table 2. Plasm id DNA yield of ∆araD strains<br>
(Table Removed)<br>
0.2% L-arabinose is sufficient for obtain-i    ing the plasmid copy number at the same level as with 2% arabinose.<br>
For this plasmid AGI∆araD seems to be better, because the plasmid yield is somewhat higher and cell densities also.<br>
Example 5.<br>
Generation of an Escherichia coli strain with additional mutations within the genes potentially encoding L-ribulose-5-phosphate 4-epimerase.<br>
E. coli chromosome contains two additional coding sequences for L-ribulose-5-phosphate 4-epimerases in different operons. The ulaF and sgbE genes from L-ascorbate degradation pathway encode the genes with epime-rase activity (Wen Shan Yew, Jhon A. Gerit, J. Bacteriol. 184 (2002) 302-306. In order to increase the stringency of the selection and to avoid or knock out the possible adaptation mechanisms of E. coli strains due to other genes with epimerase activity, the coding sequences of the UlaF and SgfjbE genes in E. coli genome were interrupted. Such adaptation mechanisms could occur in long-term plasmid production under suitable conditions.<br>
The UlaF and SgbE genes in E coli strains DH5alphaT1∆araD and AGI∆araD were disrupted using the phage A Red recombination system as described in Example.3.......<br>
First, the kanamycin-resistant gene in E. coli AGI∆araD and DH5αT1∆araD strains was eliminated. FLP recombinase expression plasmid pKD20 (Datsenko and Wanner, supra) is ampicillin resistant and temperature-sensitive. Kanamycin-resistant mutants were transformed with pCP20 (kanamycin-resistant gene is FRT-flanked), and ampicillin-resistant transformants were selected at 30°C (48 hours), after which the same colonies were purified non-seiectively at 42°C (24 hours twice). Then they were tested for loss of kanamycin and ampiciilin resistances.<br>
The inactivation of the chromosomal ulaF gene (SEQ ID NO. 20) by the phage λRed recombination system was performed using the primers ulaFylem and ulaFalum: ulaFylem<br>
(Sequence Removed)<br>
 (SEQ ID NO. 21) ulaFalum<br>
(Sequence Removed)<br>
 (SEQ ID NO. 22)<br>
A lot of colonies were observed on both transformation plates. Fifteen colonies obtained from the electroporation were tested for the presence of the kanamycin resistance gene by colony PCR using primers ulaFvalisR and ulaFvalisF:<br>
ulaFvalisR<br>
(Sequence Removed)<br>
 (SEQ ID NO. 23) ulaFvalisF<br>
(Sequence Removed)<br>
 (SEQ ID NO. 24) These primers contain annealing sites on the UlaF gene near the insertion site. A PCR product of 864 bp was expected from the E coli DH5alphaT1∆araD and AGI∆araD strains without insertion in UlaF and a 1527 bp product, if the PCR product had been inserted in the UlaF gene. To confirm the insertion of the kanamycin gene another colony PCR was performed using primers ulaFvalisR (SEQ ID NO 23) and kanaSF (SEQ ID NO 16).<br>
These primers produce a 428 bp product, if the kanamycin resistance gene has been  inserted  into the  UlaF gene.  Four colonies from<br><br>
AG1∆araDAu/aF and DH5alphaT1∆araDAt//aF strains were tested and all gave the correct product. One colony from each strain was used further.<br>
The elimination of the kanamycin-resistant gene in E.coli AG1∆araDAu/aF and DH5alphaT1∆araDAi//aF strains was performed as described above. The inactivation of the chromosomal sgbE gene (SEQ ID NO. 25) by the phage A Red recombination system was performed as described in Example 3. The primers used were sgbEalum and sgbEylem: sgbEalum<br>
(Sequence Removed)<br>
 (SEQ ID NO. 26) sgbEylem<br>
(Sequence Removed)<br>
 (SEQ ID NO. 27)<br>
A lot of colonies were observed on both transformation plates. Fifteen colonies obtained from the electroporation were tested for the presence of kanamycin resistance gene by the colony PCR using primers sgbEvalisR and sgbEvalisF:<br>
sgbEvalisR<br>
(Sequence Removed)<br>
 (SEQ ID NO. 28) sgbEvalisF<br>
ATTGAAGCGCGTATGCAGGAGG (SEQ ID NO. 29) A PCR product of 792 bp was expected from the E. coli DH5alpha T1∆araD∆u/aF∆sgbE and AG1∆araD∆u/aF∆sgbE strains without insertion in SgbE and a 1413 bp product, if the PCR product had been inserted in the SgdE gene. To confirm the insertion of kanamycin gene another colony PCR was performed using primers sgbEvalisR (SEQ ID NO. 28) and kanaSF (SEQ ID NO. 16):<br>
Fifteen colonies from both strains were tested and four gave the correct product.<br>
The arabinose sensitivity was tested on the E. coli DH5alphaT1 ∆araD∆u/a∆sgbE and AG1∆araD∆t//aF∆sg6E strains produced and compared to those of E. coli DH5alphaT1∆araD and AGI∆araD strains. One colony of each strain was inoculated into 2 ml of M9 medium containing 0.5% yeast extract, 25µg/ml of kanamycin, 0.2% glucose only or 0.2% or 2% L-arabinose, respectively. The results are shown in Table 3.<br><br>
Table 3. Testing of arabinose sensitivity<br><br>
(Table Removed)<br>
As can be seen from Table 3, there were no essential differences in the arabinose sensitivity of the strains of the invention. Similarly, when the plasmid DNA yield of the ∆araD and ∆araDAu/aF∆sgdE strains was tested as described in Example 3 (the results are not shown), no differences were found between E.coli AGI∆araD and AG1∆araD∆u//aF∆sg6E or DH5alphaT1∆araD and DH5alphaT1∆araD∆u//aF∆sgbE strains.<br>
Example 6<br>
Stability of S6wtd1EGFP/araD2<br>
An important feature of the vaccination vector is the stability during propagation in bacterial cells. To test the stability of S6wtd1EGFP/araD2 in bacteria the plasmid was transformed into the E. coli AGI∆araD and JM109∆araD strains prepared in Example 3 and the intactness of the vector was followed by the plasmid DNA analysis during four generations.<br>
The plasmid S6wtd1EGFP/araD2 was mixed with competent E. coli AGI∆araD and JM109∆araD cells and incubated on ice for 30 minutes. Subsequently, the cell suspension was subjected to a heat-shock for 3 minutes at 37°C followed by a rapid cooling on ice. One milliliter of LB medium was added to the sample and the mixture was incubated for 45 minutes at 37°C with vigorous shaking. Finally, a portion of the cells was plated onto M9 medium dishes containing 0.5% yeast extract, 2% L-arabinose and 25 µg/ml of kana-<br><br>
mycin. On the next day, the cells from one colony were transferred onto the new dish containing the same medium. This procedure was repeated until four passages of bacteria had been grown. Two colonies from each passage of both bacterial strains were used to inoculate of 2 ml of M9 medium containing 0.5% yeast extract, 2% L-arabinose and 25µg/ml of kanamycin incubated overnight at 37°C with vigorous shaking. The cells were harvested and the plasmid DNA was extracted from the bacteria using QIAprep Spin Miniprep Kit (QIAGEN). The plasmid DNA samples before (Figure 9) and after the digestion with restriction endonuclease Hindlll (Figure 10) (Fermentas, Lithuania) were analyzed by agarose gel electrophoresis in comparison with the original S6wtd1EGFP/araD2 DNA used for transformation (as control in Figures 9 and 10). Lambda DNA digested with EcoRI/Hindlll (Fermentas, Lithuania) was used as a molecular weight marker (M3 in Figure 10).<br>
Samples were digested with Hindlll as shown in Figure 10A for E. co//AGI∆araD and in Figure 10B for JM109∆araD strain, patterns identical to the original S6wtd1EGFP/araD2 plasmid DNA were observed. The predicted sizes of the fragments resulted by Hindlll digestion are 3274, 1688 and 1510 bp. It can be concluded that the vaccination vector S6wtd1EGFP/araD2 is stable when propagated in E. coll AGI∆araD and JM109∆araD strains.<br>
Example 7<br>
Comparison of an antibiotic selection system with the L-arabinose selection system of the invention<br>
In the comparison of an antibiotic selection system with the L-arabinose selection system of the invention the following growth media were used.<br>
For E. coli AG1 carrying plasmid p2 MG C #11:<br>
Medium 1: M9 medium plus 0.5% yeast extract, 0.2% glucose and 25µg/ml of kanamycin (selective medium);<br>
Medium 2: M9 medium plus 0.5% yeast extract and 0.2% glucose (non-selective medium);<br>
Medium 3:<br>
M9 medium plus 0.5% yeast extract, 0.2% L-arabinose and 25 µg/ml of kanamycin; (selective medium); and<br>
Medium 4: M9 medium plus 0.5% yeast extract and 0.2% L-arabinose (non-selective medium).<br><br>
For E co// AGI∆araD carrying paraD MG C #145:<br>
Medium 5:<br>
	M9 medium plus.0.5%. yeast-extract,-0.2% L-arabinose and 25<br>
µg/m\ of kanamycin (selective medium); and<br>
Medium 6:M9 medium plus 0.5% yeast extract, 0.2% glucose and<br>
25 µg/ml of kanamycin (non-selective medium).<br>
The plasmids p2 MG C #11 (Figure 20) and paraD MG C #145 (Figure 21) were transformed into E. co//AG1 and into E. coll AGI∆araD carrying the mutation C to T in codon 8. The transformed bacterial colonies were grown at 37°C overnight in an incubator. Next morning the colonies were inoculated into the selective and non-selective liquid media as indicated above. The inoculated cultures were grown in a shaker in 2 ml of the respective medium until they reached the stationary phase, and the density of the cultures was measured at OD600. The plasmid was extracted from the cultures and the plasmid DNA yield was determined by the measurement of the plasmid DNA at 260 nm. The plasmid yield was calculated on the basis that 50µg yields to an optical density of 1 at 260 nm.<br>
Then an aliquot of 20 µl from the stationary cultures was inoculated into fresh medium (dilution 100 times), and the cultures were grown until stationary phase (8-12 hours). The density of the cultures was measured at OD6oo, the plasmid was extracted and the yield was determined, and again an aliquot was inoculated into 2 µI of the liquid medium. This procedure was repeated 7 times (preparations 1 to 7). The results of the experiment are provided in Table 5 below.<br><br>
Table 5. Comparison of an antibiotic selection system with the L-arabinose selection system of the invention<br><br>
(Table Removed)<br>
It can be concluded from these data that a plasmid carrying the kanamycin resistance gene and conferring E. coll the resistance in the presence of kanamycin is lost in the consecutive dilution/growing steps of the culture under the non-selective as well as under selective conditions. The yield of the plasmid from 1 ml culture drops 3 times under the selective conditions and 10 times under the non-selective conditions at the seventh round of dilution (preparations 1/1 vs. 1/7 and 2/1 vs. 2/7, respectively, in Table 5). The same basic result is obtained, when the carbon source for E. coli carrying a plasmid with kanamycin resistance is L-arabinose instead of glucose (preparations 3/1 vs. 3/7 and 4/1 vs. 4/7, respectively, in Table 5). However, when the araD selection system of the invention is used in the plasmid, the plasmid DNA yield is high under both selective (preparation 5/1 vs. 5/7 in Table 5) and non-selective (preparation 6/1 vs. 6/7 in Table 5) conditions. Both under selective and non-selective conditions the plasmid DNA yield dropped over 7 generations approximately 20%. This indicates clearly that the plasrhids carrying araD selec-<br><br>
tion system of the invention are much more stable and grow efficiently under the selective as well as non-selective conditions.<br>
Example 8<br>
Fed-batch fermentation of AGI∆araD S6wtd1EGFP/araD2<br>
The araD gene based selection system was also tested in fed-batch fermentation for the purpose of production of plasmid containing bacteria. A single colony was picked from AGI∆araD S6wtd1EGFP/araD2 plate and inoculated into 250 ml M9 medium containing 0.5% yeast extract, 0.2% L-arabinose and 25 //g/ml of kanamycin and incubated overnight at 37°C with vigorous shaking. After 18 hours the OD6oo of inoculum was 6.4. 160 ml of inoculum was added to fermentor containing 5 I Fermenter Starting Medium (8 g/l KH2P04; 10 g/l NaCI; 5 g/l NH4CI; 5 g/l yeast extract; 2 g/l L-arabinose; 2 g/l MgS04, 25 mg/l kanamycin and 0.1 g/l thiamine; pH 6.7 with NH4OH). After 5.5 hours of growth automatic feeding was started with given growth speed of 0.15 h"1 (allows carbon-source limited growth) with fermenter feeding medium (300 g/l L-arabinose; 150 g/l yeast extract; 50 mg/I kanamycin; 0.2 g/l thiamine). Feeding   speed   was   controlled   by   computer   according   to   formulae F(t)=myS*Sin/Sf where myS is desired growth rate, Sjn is the amount of carbon source added to the time point and Sf is carbon source concentration in feeding medium. The growth was followed by measuring ODeoo and samples for plasmid DNA were taken. The data registered during fermentation is represented in Figure 11. Fermentation was terminated when 1 I of feeding medium was consumed. Final ODeoo was 45. The bacterial mass was collected by centrifugation and washed once with 21 STE buffer. Yield of bacterial biomass was 410 g wet weight. The data for plasmid DNA content is shown in Table 6.<br><br>
Table 6. Plasmid DNA yield during AGI∆araD S6wtd1EGFP/araD2 fermentation<br>
(Table Removed)<br>
The data in Table 6 indicate that the L-arabinose selection system works very well at high cell densities. It is probably because more plasmid copies in bacterial cell gives an advantage in the conditions of L-arabinose limitation by enabling the bacterium to use sugar more rapidly.<br>
Example 9<br>
Purification of AGI∆araD S6wtd1EGFP/araD2<br>
The purification of AGI∆araD S6wtd1EGFP/araD2 was performed as follows (Figure 12):<br>
a) Feeding preparation<br>
Clear lysate was prepared according to Qiagen's Plasmid Purification Handbook, exept RNase was not used.<br>
200g of E. co// cell paste was resuspended in 2000ml of Resuspen-sion Buffer and later equal volumes of P2 and P3 for lysis and neutralization were used. The cell debris was removed by centrifugation at 6000g for 30 minutes at 4°C. Clear lysate was poured through the paper towel, 1/10 of 10% Triton X-114 (Sigma) was added and solution was left on ice for 1 hour. (Triton X-114 has been shown to effectively reduce the level of endotoxins in protein, Liu et al., Clinical Biochemistry, 1997) After one hour nucleic acids were precipitated with 0,6 volumes of cold isopropanol. Supernatant was decanted and precipitate was stored overnight at -20°C.<br><br>
b) Plasmid DNA purification<br>
Plasmid DNA purification was performed according to Amersham ..Pharmacia's...three.step..supercoiled.plasmid.purification process, where few modifications were adopted.<br>
Step 1. Precipitate was redissolved in 1500ml TE (10mM Tris-CI, 1mM EDTA; pH 8.0) and loaded for RNA removal and buffer exchange on Sepharose 6 FF (Amersham Pharmacia), previously equilibrated with Buffer A - 2M (NH4)2SO4f 100mM Tris Cl, 10mM EDTA, pH 7.5.<br>
Step' 2. Void volume was directed to the PlasmidSelect (Amersham Pharmacia) column (equilibrated with Buffer A) and after washing and elution with Buffer B2 (1.6M NaCI, 2M (NH4)2S04l 100mM Tris Cl, 10mM EDTA, pH 7.5), supercoiled plasmid DNA was captured.<br>
Step 3. Eluted plasmid was diluted with five volumes of distilled, de-ionized water and loaded to SOURCE 30Q (Amersham Pharmacia) equilibrated with buffer C1 (0,4M NaCI, 100mM Tris Cl, 10mM EDTA, pH 7.5). After washing, purified plasmid was eluted with Buffer C2 (1M NaCI, 100mM Tris Cl, 10mM EDTA, pH 7.5) and elution peak was collected. Fraction size was 150ml and it contained 100mg of endotoxins-free (
<br><br><br><br>
WE CLAIM:<br>
1. A novel, antibiotic-free selection system comprising a bacterial cell of the kind such as herein described and a mutated or non-mutated vector carrying an araD gene, or a catalytically active fragment thereof added as a selection marker; said bacterial cell being deficient of araD gene or having combined deletions of the kind such as herein described.<br>
2.	A selection system as claimed in claim 1, wherein the araD gene is L-ribulose-5- phosphate 4-epimerase gene (EC 5.1.3.4.).<br>
3.	A selection system as claimed in claim 1 or 2, wherein the araD gene of the vector is mutated said vector generating the araD gene product L-ribulose-5-phosphate 4-epimerase.<br>
4.	A selection system as claimed in claim 3, wherein the mutation introduces a stop codon into position 8 of the araD gene.<br>
5.	A selection system as claimed in any of claims 1 to 4, wherein the vector comprises a mutation in the Shine-Dalgarno sequence of the araD gene.<br>
6.	A selection system as claimed in any of claims 1 to 5, wherein the bacterial cell is an Escherichia coli cell.<br>
7.	A selection system as claimed in claim 6, wherein the E. coli is an E. coli strain JM109.<br>
8.	A selection system as claimed in claim 6, wherein the E. coli is E. coli strain DH5 alpha-Tl.<br><br>
9.	A selection system as claimed in claim 6, wherein the E. coli is E. coil strain AG 1.<br>
10.	A selection system as claimed in any of claims 7 to 9, wherein said E. coli strain is deficient of the araD gene and ulaF gene.<br>
11.	A selection system as claimed in any of claims 7 to 9, wherein said E. coli strain is deficient of the araD gene and sgbE gene.<br>
12.	A selection system as claimed in any of claims 7 to 9, wherein said E. coli strain is deficient of araD gene, ulaF gene, and sgbE gene.<br>
13.	A selection system as claimed in any of the preceding claims, wherein said vector is an expression vector comprising<br><br>
(a)	a DNA sequence encoding a nuclear-anchoring protein operatively linked to a heterologous promoter, said nuclear-anchoring protein comprising (i) a DNA binding domain which binds to a specific DNA sequence, and (ii) a functional domain that binds to a nuclear component, or a functional equivalent thereof; and<br>
(b)	a multimerized DNA sequence forming a binding site for the nuclear anchoring protein, wherein said vector lacks a papilloma virus origin of replication.<br>
14.	A selection system as claimed in claim 13, wherein<br>
(a)	the   nuclear-anchoring  protein   is  the   E2   protein   of Bovine Papilloma Virus type I (BPV), and<br>
(b)	the multimerized DNA sequence are multiple binding sites for the BPV E2 protein incorporated into the vector as a cluster,<br><br>
where the sites can be as head-to-tail structures or can be included into the vector by spaced positioning.<br>
15.	A selection system as claimed in claim 14, wherein the vector optionally comprises a deletion in the multimerized DNA sequence.<br>
16.	A selection system as claimed in claim 14, wherein the vector optionally comprises a mutation in the Shine-Dalgarno sequence of the araD gene.<br>
17.	A vector for use in a selection system as claimed in any of claims 1 to 12 comprising an araD gene as a selection marker which araD gene is mutated generating a stop codon at position 8, wherein said vector generates the araD gene product L-ribulose-5-phosphate 4-epimerase.<br>
18.	A vector as claimed in claim 17, comprising<br>
(a)	a DNA sequence encoding a nuclear-anchoring protein<br>
operatively linked to a heterologous promoter, said nuclear-<br>
anchoring protein comprising (i) a DNA binding domain which<br>
binds to a specific DNA sequence, and (ii) a functional domain<br>
that binds to a nuclear component, or a functional equivalent<br>
thereof; and<br>
(b)	a multimerized DNA sequence forming a binding site for the<br>
nuclear anchoring protein, wherein said vector lacks a<br>
papilloma virus origin of replication.<br>
19.	A vector as claimed in claim 18, wherein<br>
(a)	the   nuclear-anchoring   protein   is   the   E2   protein   of   Bovine<br>
Papillomavirus type 1 (BPV), and<br>
(b)	the multimerized DNA sequence are multiple binding sites for<br>
the BPV E2 protein incorporated into the vector as a cluster,<br><br>
where the sites can be as head-to-tail structures or can be included into the vector by spaced positioning.<br>
20.	A vector as claimed in claim 19 optionally comprising a deletion in the<br>
multimerized DNA sequence.<br>
21.	A vector as claimed in claim 19 optionally comprising a mutation in<br>
the ShineDalgarno sequence of the araD gene.<br>
22.	E. coli strain DH5alpha-Tl deficient of the araD gene and ulaF gene for use in a selection system as claimed in claim 1<br>
23.	E. coli strain DH5alpha-Tl deficient of the araD gene and sgbE gene for use in a selection system as claimed in claim 1.<br>
24.	E. coli strain DH5alpha-Tl deficient of the araD gene, ulaF gene, and sgbE gene for use in a selection system as claimed in claim 1.<br>
25.    E. coli strain AG1 deficient of the araD gene and ulaF gene for use in a selection system as claimed in claim 1.<br>
26.	E. coli strain AG1 deficient of the araD gene and sgbE gene for use in a selection system as claimed in claim 1.<br>
27.	E. coli strain AG1 deficient of the araD gene, ulaF gene, and sgbE gene for use in a selection system as claimed in claim 1.<br>
28.	E. coli strain JM109 deficient of the araD gene and ulaF gene for use in a selection system to as claimed in claim 1.<br><br>
29.	E. coli strain JM109 deficient of the araB gene and sgbE gene for use in a selection system as claimed in claim 1.<br>
30.	E. coli strain JM109 deficient of the araB gene, ulaF gene, and sgbE gene for use in a selection system as claimed in claim 1.<br>
31.	A novel, antibiotic-free selection system as claimed in claim 1 for selecting bacterial cells, wherein the bacterial cells are transformed with a plasmid containing an araD gene, or a catalytically active fragment thereof as a selection marker and the gene of interest, the method comprising inserting the plasmid into the araD deficient host cell and growing the cells in a growth medium containing arabinose<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1ERUxOUC0yMDA2LUFic3RyYWN0LSgxOS0wMS0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">1142-DELNP-2006-Abstract-(19-01-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1ERUxOUC0yMDA2LUFic3RyYWN0LTI1LTA0LTIwMDgucGRm" target="_blank" style="word-wrap:break-word;">1142-DELNP-2006-Abstract-25-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1kZWxucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1142-delnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1ERUxOUC0yMDA2LUNsYWltcy0oMTktMDEtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1142-DELNP-2006-Claims-(19-01-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1ERUxOUC0yMDA2LUNsYWltcy0yNS0wNC0yMDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">1142-DELNP-2006-Claims-25-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1kZWxucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1142-delnp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjItMDMtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">1142-DELNP-2006-Correspondence Others-(22-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTItMDEtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1142-DELNP-2006-Correspondence-Others-(12-01-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTktMDEtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1142-DELNP-2006-Correspondence-Others-(19-01-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0yNS0wNC0yMDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">1142-DELNP-2006-Correspondence-Others-25-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1kZWxucC0yMDA2LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1142-delnp-2006-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1ERUxOUC0yMDA2LURlc2NyaXB0aW9uIChDb21wbGV0ZSktKDE5LTAxLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">1142-DELNP-2006-Description (Complete)-(19-01-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1ERUxOUC0yMDA2LURlc2NyaXB0aW9uIChDb21wbGV0ZSktMjUtMDQtMjAwOC5wZGY=" target="_blank" style="word-wrap:break-word;">1142-DELNP-2006-Description (Complete)-25-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1kZWxucC0yMDA2LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">1142-delnp-2006-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1kZWxucC0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1142-delnp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1ERUxOUC0yMDA2LUZvcm0tMS0oMTktMDEtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1142-DELNP-2006-Form-1-(19-01-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1ERUxOUC0yMDA2LUZvcm0tMS0yNS0wNC0yMDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">1142-DELNP-2006-Form-1-25-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1kZWxucC0yMDA2LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">1142-delnp-2006-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1kZWxucC0yMDA2LWZvcm0tMTMtKDE5LTAxLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">1142-delnp-2006-form-13-(19-01-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1ERUxOUC0yMDA2LUZvcm0tMi0oMTktMDEtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1142-DELNP-2006-Form-2-(19-01-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1ERUxOUC0yMDA2LUZvcm0tMi0yNS0wNC0yMDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">1142-DELNP-2006-Form-2-25-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1kZWxucC0yMDA2LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">1142-delnp-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1ERUxOUC0yMDA2LUZvcm0tMjctKDIyLTAzLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">1142-DELNP-2006-Form-27-(22-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1ERUxOUC0yMDA2LUZvcm0tMy0yNS0wNC0yMDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">1142-DELNP-2006-Form-3-25-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1kZWxucC0yMDA2LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">1142-delnp-2006-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1kZWxucC0yMDA2LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">1142-delnp-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1ERUxOUC0yMDA2LUdQQS0yNS0wNC0yMDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">1142-DELNP-2006-GPA-25-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1kZWxucC0yMDA2LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">1142-delnp-2006-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1kZWxucC0yMDA2LXBjdC0xMDEucGRm" target="_blank" style="word-wrap:break-word;">1142-delnp-2006-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1kZWxucC0yMDA2LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">1142-delnp-2006-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1kZWxucC0yMDA2LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">1142-delnp-2006-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1kZWxucC0yMDA2LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">1142-delnp-2006-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1kZWxucC0yMDA2LXBjdC0zMDYucGRm" target="_blank" style="word-wrap:break-word;">1142-delnp-2006-pct-306.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1kZWxucC0yMDA2LXBjdC0zMDgucGRm" target="_blank" style="word-wrap:break-word;">1142-delnp-2006-pct-308.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1kZWxucC0yMDA2LXBjdC0zMzIucGRm" target="_blank" style="word-wrap:break-word;">1142-delnp-2006-pct-332.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1kZWxucC0yMDA2LXBjdC00MDkucGRm" target="_blank" style="word-wrap:break-word;">1142-delnp-2006-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0Mi1ERUxOUC0yMDA2LVBldGl0aW9uLTEzNy0yNS0wNC0yMDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">1142-DELNP-2006-Petition-137-25-04-2008.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="227867-cleaning-device.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="227869-seat-lock-mounting-structure-for-motorcycle.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>227868</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1142/DELNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>07/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>13-Feb-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>23-Jan-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>03-Mar-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>FIT BIOTECH OYJ PLC,</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>BIOKATU 8, FI-33520 TAMPERE, FINLAND.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>TANEL TENSON</td>
											<td>TAHE 31-6, EE-50103 TARTU, ESTONIA.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SILJA LAHT</td>
											<td>NAITUSE 22B-26, EE-50407 TARTU, ESTONIA.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>MAARJA ADOJAAN</td>
											<td>ULENURME VALD, KULITSE KULA, EE-61702 TARTU REGION, ESTONIA.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>ANDREAS MANNIK</td>
											<td>NARVA STREET 149-25, EE-51008, TARTU, ESTONIA.</td>
										</tr>
										<tr>
											<td>5</td>
											<td>URVE TOOTS</td>
											<td>AARDLA STREET 146-1, EE-50415, TARTU, ESTONIA.</td>
										</tr>
										<tr>
											<td>6</td>
											<td>MART USTAV</td>
											<td>JAAMA 58 A, EE-50064, TARTU, ESTONIA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/70</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/FI2004/000540</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-09-15</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>20031319</td>
									<td>2003-09-15</td>
								    <td>Finland</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/227868-a-novel-antibiotic-free-selection-system by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:17:54 GMT -->
</html>
