<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/227975-an-in-vitro-method-of-providing-an-index-for-characterization-of-biological-conditions by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:30:31 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 227975:AN IN VITRO METHOD OF PROVIDING AN INDEX FOR CHARACTERIZATION OF BIOLOGICAL CONDITIONS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">AN IN VITRO METHOD OF PROVIDING AN INDEX FOR CHARACTERIZATION OF BIOLOGICAL CONDITIONS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>This invention relates to a method provides an index that is indicative of the state of a subject, as to a biological condition, based on a sample from the subject. An embodiment of this method includes: deriving from the sample a profile data set, the pro~le data set including a plurality of members, each member being a quantitative measure of the amount of a distinct RNA or protein constituent in a panel of constituents selected so that measurement of the constituents enables evaluation of the biological condition; and in deriving the profile data set, achieving such measure for each constituent under measurement conditions that are substantially repeatable; and applying values from the profile data set to an index function that provides a mapping from an instance of a profile data set into a single-valued measure of biological condition, so as to produce an index pertinent to the biological condition of the subject.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Identification, Monitoring and Treatment of Disease<br>
And Characterization of Biological Condition<br>
Using Gene Expression Profiles<br>
Technical Field and Background Art<br>
The present invention relates to use of gene expression data, and in particular to use of gene expression data in identification, monitoring and treatment of disease and in characterization of biological condition of a subject.<br>
The prior art has utilized gene expression data to determine the presence or absence of particular markers as diagnostic of a particular condition, and in some circumstances have described the cumulative addition of scores for over expression of particular disease markers to achieve increased accuracy or sensitivity of diagnosis. Information on any condition of a particular patient and a patient's response to types and dosages of therapeutic or nutritional agents has become an important issue in clinical medicine today not only from the aspect of efficiency of medical practice for the health care industry but for improved outcomes and benefits for the patients.<br>
Summary of the Invention<br>
In a first embodiment, there is provided a method, for evaluating a biological condition of a subject, based on a sample from the subject. The method includes:<br>
deriving from the sample a profile data set, the profile data set including a plurality of members, each member being a quantitative measure of the amount of a distinct RNA or protein constituent in a panel of constituents selected so that measurement of the constituents enables evaluation of the biological condition; and<br>
in deriving the profile data set, achieving such measure for each constituent under measurement conditions that are substantially repeatable.<br>
There is a related embodiment for providing an index that is indicative of the state of a subject, as to a biological condition, based on a sample from the subject. This embodiment includes:<br><br>
deriving from the sample a profile data set, the profile data set including a plurality of members, each member being a quantitative measure of the amount of a distinct RNA or protein constituent in a panel of constituents selected so that measurement of the constituents enables evaluation of the biological condition; and<br>
in deriving the profile data set, achieving such measure for each constituent under measurement conditions that are substantially repeatable; and<br>
applying values from the profile data set to an index function that provides a mapping from an instance of a profile data set into a single-valued measure of biological condition, so as to produce an index pertinent to the biological condition of the subject. In further embodiments related to the foregoing, there is also included, in deriving the profile data set, achieving such measure for each constituent under measurement conditions wherein specificity and efficiencies of amplification for all constituents are substantially similar. Similarly further embodiments include alternatively or in addition, in deriving the profile data set, achieving such measure for each constituent under measurement conditions wherein specificity and efficiencies of amplification for all constituents are substantially similar.<br>
In embodiments relating to providing the index a further embodiment also includes providing with the index a normative value of the index function, determined with respect to a relevant population, so that the index may be interpreted in relation to the normative value. Optionally providing the normative value includes constructing the index function so that the normative value is approximately 1. Also optionally, the relevant population has in common a property that is at least one of age group, gender, ethnicity, geographic location, diet, medical disorder, clinical indicator, medication, physical activity, body mass, and environmental exposure.<br>
In another related embodiment, efficiencies of amplification, expressed as a percent, for all constituents he within a range of approximately 2 percent, and optionally, approximately 1 percent.<br>
In another related embodiment, measurement conditions are repeatable so that such measure for each constituent has a coefficient of variation, on repeated derivation of such measure from the sample, that is less than approximately 3 percent. In further embodiments, the panel includes at least three constituents and optionally fewer than approximately 500 constituents.<br><br>
In another embodiment, the biological condition being evaluated is with respect to a localized tissue of the subject and the sample is derived from tissue or fluid of a type distinct from that of the localized tissue.<br>
In related embodiments, the biological condition may be any of the conditions identified in Tables 1 through 12 herein, in which case there are measurements conducted corresponding to constituents of the corresponding Gene Expression Panel. The panel in each case includes at least two, and optionally at least three, four, five, six, seven, eight, nine or ten, of the constituents of the corresponding Gene Expression Panel.<br>
In another embodiment, there is provided a method of providing an index that is indicative of the inflammatory state of a subject based on a sample from the subject that includes: deriving from the sample a first profile data set, the first profile data set including a plurality of members, each member being a quantitative measure of the amount of a distinct RNA or protein constituent in a panel of constituents, the panel including at least two of the constituents of the Inflammation Gene Expression Panel of Table 1; (although in other embodiments, at least three, four, five, six or ten constituents of the panel of Table 1 may be used in a panel) wherein, in deriving the first profile data set, such measure is performed for each constituent both under conditions wherein specificity and efficiencies of amplification for all constituents are substantially similar and under substantially repeatable conditions; and applying values from the first profile data set to an index function that provides a mapping from an instance of a profile data set into a single-valued measure of biological condition (in an embodiment, this, may be an inflammatory condition), so as to produce an index pertinent to the biological condition of the sample or the subject. The biological condition may be any condition that is assessable using an appropriate Gene Expression Panel; the measurement of the extent of inflammation using the Inflammation Gene Expression Panel is merely an example. In additional embodiments, the mapping by the index function may be further based on an instance of a relevant baseline profile data set and values may be applied from a corresponding baseline profile data set from the same subject or froifa. a population of subjects or samples with a similar or different biological condition. Additionally, the index function may be constructed to deviate from a normative value generally upwardly in an instance of an increase in expression of a constituent whose increase is associated with an increase of inflammation and also in an instance of a decrease in expression of a constituent whose decrease is associated with an increase of inflammation. The index function alternatively be constructed to weigh the expression value of a constituent in the<br><br>
panel generally in accordance with the extent to which its expression level is determined to be correlated with extent of inflammation. The index function may be alternatively constructed to take into account clinical insight into inflammation biology or to take into account experimentally derived data or to take into account relationships derived from computer analysis of profile data sets in a database associating profile data sets with clinical and demographic data In this connection, the construction of the index function may be achieved using statistical methods, which evaluate such data, to establish a model of constituent expression values that is an optimized predictor of extent of inflammation.<br>
In another embodiment, the panel includes at least one constituent that is associated with a specific inflammatory disease.<br>
The methods described above may further utilize the step wherein (i) the mapping by the index function is also based on an instance of at least one of demographic data and clinical data and (ii) values are applied from the first profile data set including applying a set of values associated with at least one of demographic data and clinical data.<br>
In another embodiment of the above methods, a portion of deriving the first profile data set is performed at a first location and applying the values from the first profile data set is performed at a second location, and data associated with performing the portion of deriving the first profile data set are communicated to the second location over a network to enable, at the second location, applying the values from the first profile data set.<br>
In an embodiment of the methods, the index function is a linear sum of terms, each term being a contribution function of a member of the profile data set. Moreover, the contribution function may be a weighted sum of powers of one of the member or its reciprocal, and the powers may be integral, so that the contribution function is a polynomial of one of the member or its reciprocal. Optionally, the polynomial is a linear polynomial. The profile data set may include at least three, four or all members corresponding to constituents selected from the group consisting of ILIA, IL1B, TNF, IFNG and DL10. The index function may be proportional to 1/4{IL1A} 4- 1/4{IL1B} + 1/4{TNF} + 1/4{INFG} - 1/{IL10} and braces around a constituent designate measurement of such constituent.<br>
In an additional embodiment, a method is provided of analyzing complex data associated with a sample from a subject for information pertinent to inflammation, the method that includes: deriving a Gene Expression Profile for the sample, the Gene Expression Profile being based on a Signature Panel for Inflammation; and using the<br><br>
Gene Expression Profile to determine a Gene Expression Profile Inflammatory Index for the sample.<br>
In an additional embodiment, a method is provided of monitoring the biological condition of a subject, that includes deriving a Gene Expression Profile for each of a series of samples over time from the subject, the Gene Expression Profile being based on a Signature Panel for Inflammation; and for each of the series of samples, using the corresponding Gene Expression Profile to determine a Gene Expression Profile Inflammatory Index.<br>
In an additional embodiment, there is provided a method of determining at least one of (i) an effective dose of an agent to be administered to a subject and (ii) a schedule for administration of an agent to a subject, the method including: deriving a Gene Expression Profile for a sample from the subject, the Gene Expression Profile being based on a Signature Panel for Inflammation; using the Gene Expression Profile to determine a Gene Expression Profile Inflammatory Index for the sample; and using the Gene Expression Profile Inflammatory Index as an indicator in establishing at least one of the effective dose and the schedule.<br>
In an additional embodiment, a method of guiding a decision to continue or modify therapy for a biological condition of a subject, is provided that includes: deriving a Gene Expression Profile for a sample from the subject, the Gene Expression Profile being based on a Signature Panel for Inflammation; and using the Gene Expression Profile to determine a Gene Expression Profile Inflammatory Index for the sample.<br>
A method of predicting change in biological condition of a subject as a result of exposure to an agent, is provided that includes: deriving a first Gene Expression Profile for a first sample from the subject in the absence of the agent, the first Gene Expression Profile being based on a Signature Panel for Inflammation; deriving a second Gene Expression Profile for a second sample from the subject in the presence of the agent, the second Gene Expression Profile being based on the same Signature Panel; and using the first and second Gene Expression Profiles to determine correspondingly a first Gene Expression Profile Inflammatory Index and a second Gene Expression Profile Inflammatory Index. Accordingly, the agent may be a compound and the compound may be therapeutic.<br>
In an additional embodiment, a method of evaluating a property of an agent is provided where the property is at least one of purity, potency, quality, efficacy or safety, the method including: deriving a first Gene Expression Profile from a sample reflecting<br><br>
exposure to the agent of (i) the sample, or (ii) a population of cells from which the sample is derived, or (iii) a subject from which the sample is derived; using the Gene Expression Profile to determine a Gene Expression Profile Inflammatory Index; and using the Gene Expression Profile Inflammatory Index in determining the property.<br>
In accordance with another embodiment there is provided a method of providing an index that is indicative of the biological state of a subject based on a sample from the subject. The method of this embodiment includes:<br>
deriving from the sample a first profile data set, the first profile data set including a plurality of members, each member being a quantitative measure of the amount of a distinct RNA or protein constituent in a panel of constituents, the panel including at least two of the constituents of the Inflammation Gene Expression Panel of Table 1; and<br>
applying values from the first profile data set to an index function that provides a mapping from an instance of a profile data set into a single-valued measure of biological condition, so as to produce an index pertinent to the biological condition of the sample or the subject.<br>
In carrying out this method the index function also uses data from a baseline profile data set for the panel. Each member of the baseline data set is a normative measure, determined with respect to a relevant population of subjects, of the amount of one of the constituents in the panel. In addition, in deriving the first profile data set and the baseline data set, such measure is performed for each constituent both under conditions wherein specificity and efficiencies of amplification for all constituents are substantially similar and under substantially repeatable conditions.<br>
In another type of embodiment, there is provided a method, for evaluating a biological condition of a subject, based on a sample from the subject. In this embodiment, the method includes:<br>
deriving from the sample a first profile data set, the first profile dataset including a plurality of members, each member being a quantitative measure of the amount of a distinct RNA or protein constituent in a panel of constituents selected so that measurement of the constituents enables measurement of the biological condition; and<br>
producing a calibrated profile data set for the panel, wherein each member of the calibrated profile data set is a function of a corresponding member of the first profile data set and a corresponding member of a baseline profile data set for the panel.<br>
In this embodiment, each member of the baseline data set is a normative measure, determined with respect to a relevant population of subjects, of the amount of one of the<br><br>
constituents in the panel, and the calibrated profile data set provides a measure of the biological condition of the subject.<br>
In a similar type of embodiment, there is provided a method, for evaluating a biological condition of a subject, based on a sample from the subject, and the method of this embodiment includes::<br>
applying the first sample or a portion thereof to a defined population of indicator cells;<br>
obtaining from the indicator cells a second sample containing at least one of RNAs or proteins;<br>
deriving from the second sample a first profile data set, the first profile data set including a plurality of members, each member being a quantitative measure of the amount of a distinct RNA or protein constituent in a panel of constituents selected so that measurement of the constituents enables measurement of the biological condition; and<br>
producing a calibrated profile data set for the panel, wherein each member of the calibrated profile data set is a function of a corresponding member of the first profile data set and a corresponding member of a baseline profile data set for the panel,, wherein each member of the baseline" data set is a normative measure, determined with respect to a relevant population of subjects, of the amount of one of the constituents in the panel, the calibrated profile data set providing a measure of the biological condition of the subject.<br>
Furthermore, another and similar, type of embodiment provides a method, for evaluating a biological condition affected by an agent. The method of this embodiment includes:<br>
obtaining, from a target population of cells to which the agent has been administered, a sample having at least one of RNAs and proteins;<br>
deriving from the sample a first profile data set, the first profile data set including a plurality of members, each member being a quantitative measure of the amount of a distinct RNA or protein constituent in a panel of constituents selected so that measurement of the constituents enables measurement of the biological condition; and<br>
producing a calibrated profile data set for the panel, wherein each member of the calibrated profile data set is a function of a corresponding member of the first profile data set and a corresponding member of a baseline profile data set for the panel, wherein each member of the baseline data set is a normative measure, determined with respect to a relevant population of subjects, of the amount of one of the constituents in the panel, the calibrated profile data set providing a measure of the biological condition as affected by<br><br>
the agent.<br>
In further embodiments based on these last three embodiments, the relevant population may be a population of healthy subjects. Alternatively, or in addition, the relevant population is has in common a property that is at least one of age group, gender, ethnicity, geographic location, diet, medical disorder, clinical indicator, medication, physical activity, body mass, and environmental exposure.<br>
Alternatively or in addition, the panel includes at least two of the constituents of the Inflammation Gene Expression Panel of Table 1. (Other embodiments employ at least three, four, five, six, or ten of such constituents.) Also alternatively or in addition, in deriving the first profile data set, such measure is performed for each constituent both under conditions wherein specificity and efficiencies of amplification for all constituents are substantially similar and under substantially repeatable conditions. Also alternatively, when such measure is performed for each constituent both under conditions wherein specificity and efficiencies of amplification for all cons'tituents are substantially similar and under substantially repeatable conditions, optionally one need not produce a calibrated profile data set, but may instead work directly with the first data set.<br>
In another embodiment, there is provided a method, for evaluating the effect on a biological condition by a first agent in relation to the effect by a second agent. The method of this embodiment includes:<br>
obtaining, from first and second target populations of cells to which the first and second agents have been respectively administered, first and second samples respectively, each sample having at least one of RNAs and proteins;<br>
deriving from the first sample a first profile data set and from the second sample a second profile data set, the profile data sets each including a plurality of members, each member being a quantitative measure of the amount of a distinct RNA or protein constituent in a panel of constituents selected so that measurement of the constituents enables measurement of the biological condition; and<br>
producing for the panel a first calibrated profile data set and a second profile data set, wherein (i) each member of the first calibrated profile data set is a function of a corresponding member of the first profile data set and a corresponding member of a baseline profile data set for the panel, wherein each member of the baseline data set is a normative measure, determined with respect to a relevant population of subjects, of the ampunt of one of the constituents in the panel, and (ii) each member of the second calibrated profile data set is a function of a corresponding member of the second profile<br><br>
data set and a corresponding member of the baseline profile data set, the calibrated profile data sets providing a measure of the effect by the first agent on the biological condition in relation to the effect by the second agent.<br>
In this embodiment, in deriving the first and second profile data sets, such measure is performed for each constituent both under conditions wherein specificity and efficiencies of amplification for all constituents are substantially similar and under substantially repeatable conditions. In a further related embodiment, the first agent is a first drug and the second agent is a second drug. In another related embodiment, the first agent is a drug and the second agent is a complex mixture. In yet another related embodiment, the first agent is a drug and the second agent is a nutriceutical.<br>
Brief Description of the Drawings<br>
The foregoing features of the invention will be more readily understood by reference to the following detailed description, taken with reference to the accompanying drawings, in which:<br>
Fig. 1A shows the results of assaying 24 genes from the Source Inflammation Gene Panel (shown in Table 1) on eight separate days during the course of optic neuritis in a single male subject.<br>
IB illustrates use of an inflammation index in relation to the data of Fig. 1A, in accordance with an embodiment of the present invention.<br>
Fig. 2 is a graphical illustration of the same inflammation index calculated at 9 different, significant clinical milestones.<br>
Fig. 3 shows the effects of single dose treatment with 800 mg of ibuprofen in a single donor as characterized by the index.<br>
Fig. 4 shows the calculated acute inflammation index displayed graphically for<br>
five different conditions.	,<br>
Fig. 5 shows a Viral Response Index for monitoring the progress of an upper respiratory infection (URI).<br>
Figs. 6 and 7 compare two different populations using Gene Expression Profiles (with respect to the 48 loci of the Inflammation Gene Expression Panel of Table 1).<br>
Fig. 8 compares a normal population with a rheumatoid arthritis population derived from a longitudinal study.<br><br>
Fig. 9 compares two normal populations, one longitudinal and the other cross sectional.<br>
Fig. 10 shows the shows gene expression values for various individuals of a normal population.<br>
Fig. 11 shows the expression levels for each of four genes (of the Inflammation Gene Expression Panel of Table 1), of a single subject, assayed monthly over a period of eight months.<br>
Figs. 12 and 13 similarly show in each case the expression levels for each of 48 genes (of the Inflammation Gene Expression Panel of Table 1), of distinct single subjects (selected in each case on the basis of feeling well and not taking drugs), assayed, in the case of Fig. 12 weekly over a period of four weeks, and in the case of Fig. 13 monthly over a period of six months.<br>
Fig. 14 shows the effect over time, on inflammatory gene expression in a single human subject., of the administration of an anti-inflammatory steroid, as assayed using the Inflammation Gene Expression Panel of Table 1.<br>
Fig. 15, in a manner analogous to Fig. 14, shows the effect over time, via whole blood samples obtained from a human subject, administered a single dose of prednisone, on expression of 5 genes (of the Inflammation Gene Expression Panel of Table 1).<br>
Fig. 16 also shows the effect over time, on inflammatory gene expression in a single human subject suffering from rheumatoid arthritis, of the administration of a TNF-inhibiting compound, but here the expression is shown in comparison to the cognate locus average previously determined (in connection with Figs. 6 and 7) for the normal (i.e., undiagnosed, healthy) population.<br>
Fig. 17A further illustrates the consistency of inflammatory gene expression in a population.<br>
Fig. 17B shows the normal distribution of index values obtained from an undiagnosed population.<br>
Fig. 17C illustrates the use of the same index as Fig. 17B, where the inflammation median for a normal population has been set to zero and both normal and diseased subjects are plotted in standard deviation units relative to that median.<br>
Fig. 18 plots, in a fashion similar to that of Fig. 17A, Gene Expression Profiles, for the same 7 loci as in Fig. 17A, two different (responder v. non-responder) 6-subject populations of rheumatoid arthritis patients.<br><br>
Fig. 19 thus illustrates use of the inflammation index for assessment of a single subject suffering from rheumatoid arthritis, who has not responded well to traditional therapy with methotrexate.<br>
Fig. 20 similarly illustrates use of the inflammation index for assessment of three subjects suffering from rheumatoid arthritis, who have not responded well to traditional therapy with methotrexate.<br>
Each of Figs. 21-23 shows the inflammation index for an international group of subjects, suffering from rheumatoid arthritis, undergoing three separate treatment regimens.<br>
Fig. 24 illustrates use of the inflammation index for assessment of a single subject suffering from inflammatory bowel disease.<br>
Fig. 25 shows Gene Expression Profiles with respect to 24 loci (of the Inflammation Gene Expression Panel of Table 1) for whole blood treated with Ibuprofen in vitro in relation to other non-steroidal anti-inflammatory drugs (NSAIDs).<br>
Fig. 26 illustrates how the effects of two competing anti-inflammatory compounds can be compared objectively, quantitatively, precisely, and reproducibly.<br>
Figs. 27 through 41 illustrate the use of gene expression panels in early identification and monitoring of infectious disease.<br>
Fig. 27 uses a novel bacterial Gene Expression Panel of 24 genes, developed to discriminate various bacterial conditions in a host biological system.<br>
Fig. 28 shows differential expression for a single locus, DFNG, to LTA derived from three distinct sources: S. pyogenes, B. subtilis, and S. aureus.<br>
Figs. 29 and 30 show the response after two hours of the Inflammation 48A and 48B loci respectively (discussed above in connection with Figs. 6 and 7 respectively) in whole blood to administration of a Gram-positive and a Gram-negative organism.<br>
Figs. 31 and 32 correspond to Figs. 29 and 30 respectively and are similar to them, with the exception that the monitoring here occurs 6 hours after administration.<br>
Fig. 33 compares the gene expression response induced by E. coli and by an organism-free E. coli filtrate.<br>
Fig. 34 is similar to Fig. 33, but here the compared responses are to stimuli from E. coli filtrate alone and from E. coli filtrate to which has been added polymyxin B.<br>
Fig. 35 illustrates the gene expression responses induced by S. aureus at 2, 6, and 24 hours after administration.<br><br>
Figs. 36 through 41 compare the gene expression induced by E. coli and S. aureus under various concentrations and times.<br>
Detailed Description of Specific Embodiments<br>
Definitions<br>
The following terms shall have the meanings indicated unless the context otherwise requires:<br>
"Algorithm" is a set of rules for describing a biological condition. The rule set may be defined exclusively algebraically but may also include alternative or multiple decision points requiring domain-specific knowledge, expert interpretation or other clinical indicators.<br>
An "agenf9 is a "composition" or a "stimulus1', as those terms are defined herein, or a combination of a composition and a stimulus.<br>
"Amplification" in the context of a quantitative RT-PCR assay is a function of the number of DNA replications that are tracked to provide a quantitative determination of its concentration. "Amplification!* here refers to a degree of sensitivity and specificity of a quantitative assay technique. Accordingly, amplification provides a measurement of concentrations of constituents that is evaluated under conditions wherein the efficiency oi amplification and therefore the degree of sensitivity and reproducibility for measuring all constituents is substantially similar.<br>
A "baseline profile data sef1 is a set of values associated with constituents of a Gene Expression Panel resulting from evaluation of a biological sample (or population o samples) under a desired biological condition that is used for mathematically normative purposes. The desired biological condition may be, for example, the condition of a subject (or population of subjects) before exposure to an agent or in the presence of an untreated disease or in the absence of a disease. Alternatively, or in addition, the desired biological condition may be health of a subject or a population of subjects. Alternatively i or in addition, the desired biological condition may be that associated with a population subjects selected on the basis of at least one of age group, gender, ethnicity, geographic location, diet, medical disorder, clinical indicator, medication, physical activity, body mass, and environmental exposure.<br>
A "biological condition" of a subject is the condition of the subject in a pertinen1 realm that is under observation, and such realm may include any aspect of the subject capable of being monitored for change in condition, such as health, disease including<br><br>
cancer; trauma; aging; infection; tissue degeneration; developmental steps; physical fitness; obesity, and mood. As can be seen, a condition in this context may be chronic or acute or simply transient. Moreover, a targeted biological condition may be manifest throughout the organism or population of cells or may be restricted to a specific organ (such as skin, heart, eye or blood), but in either case, the condition may be monitored directly by a sample of the affected population of cells or indirectly by a sample derived elsewhere from the subject. The term "biological condition" includes a "physiological condition".<br>
"Body fluid9 of a subject includes blood, urine, spinal fluid, lymph, mucosal secretions, prostatic fluid, semen, haemolymph or any other body fluid known in the art for a subject.<br>
"Calibratedprofile data set* is a function of a member of a first profile data set and a corresponding member of a baseline profile data set for a given constituent in a panel.<br>
A "clinical indicator*' is any physiological datum used alone or in conjunction with other data in evaluating the physiological condition of a collection of cells or of an organism. This term includes pre-clinical indicators.<br>
A "composition" includes a chemical compound, a nutriceutical, a pharmaceutical, a homeopathic formulation, an allopathic formulation, a naturopathic formulation, a combination of compounds, a toxin, a food, a food supplement, a mineral, and a complex mixture of substances, in any physical state or in a combination of physical states.<br>
To "derive" a profile data set from a sample includes determining a set of values associated with constituents of a Gene Expression Panel either (i) by direct measurement of such constituents in a biological sample or (ii) by measurement of such constituents in a second biological sample that has been exposed to the original sample or to matter derived from the original sample.<br>
"Distinct RNA or protein constituent7 in a panel of constituents is a distinct expressed product of a gene, whether RNA or protein. An "expression" product of a gene includes the gene product whether RNA or protein resulting from translation of the messenger RNA.<br>
A "Gene Expression PaneF is an experimentally verified set of constituents, eacf~ constituent being a distinct expressed product of a gene, whether RNA or protein,<br><br>
wherein constituents of the set are selected so that their measurement provides a measurement of a targeted biological condition,<br>
A "Gene Expression Profile" is a set of values associated with constituents of a Gene Expression Panel resulting from evaluation of a biological sample (or population of samples).<br>
A "Gene Expression Profile Inflammatory Index" is the value of an index function that provides a mapping from an instance of a Gene Expression Profile into a single-valued measure of inflammatory condition.<br>
The "health" of a subject includes mental, emotional, physical, spiritual, allopathic, naturopathic and homeopathic condition of the subject.<br>
"Index" is an arithmetically or mathematically derived numerical characteristic developed for aid in simplifying or disclosing or informing the analysis of more complex quantitative information. A disease or population index may be determined by the application of a specific algorithm to a plurality of subjects or samples with a common biological condition.<br>
"Inflammation" is used herein in the general medical sense of the word and may be an acute or chronic; simple or supporative; localized or disseminated; cellular and tissue response, initiated or sustained by any number of chemical&gt; physical or biological agents or combination of agents.<br>
"Inflammatory state" is used to indicate the relative biological condition of a subject resulting from inflammation, or characterizing the degree of inflammation<br>
A "large number" of data sets based on a common panel of genes is a number of data sets sufficiently large to permit a statistically significant conclusion to be drawn with respect to an instance of a data set based on the same panel.<br>
A "normative" condition of a subject to whom a composition is to be administered means the condition of a subject before administration, even if the subject happens to be suffering from a disease.<br>
A "paneV of genes is a set of genes including at least two constituents.<br>
A "sample" from a subject may include a single cell or multiple cells or fragments of cells or an aliquot of body fluid, taken from the subject, by means including venipuncture, excretion, ejaculation, massage, biopsy, needle aspirate, lavage sample, scraping, surgical incision or intervention or other means known in the art.<br><br>
A '"Signature Profile" is an experimentally verified subset of a Gene Expression Profile selected to discriminate a biological condition, agent or physiological mechanism of action.<br>
A "Signature PaneF is a subset of a Gene Expression Panel, the constituents of which are selected to permit discrimination of a biological condition, agent or physiological mechanism of action.<br>
A "subject' is a cell, tissue, or organism, human or non-human* whether in vivo, ex vivo or in vitro, under observation. When we refer to evaluating the biological condition of a subject based on a sample from the subject, we include using blood or other tissue sample from a human subject to evaluate the human subject's condition; but we also include, for example, using a blood sample itself as the subject to evaluate, for example, the effect of therapy or an agent upon the sample.<br>
A "stimulus" includes (i) a monitored physical interaction with a subject, for example ultraviolet A or B, or light therapy for seasonal affective disorder, or treatment of psoriasis with psoralen or treatment of melanoma with embedded radioactive seeds, other radiation exposure, and (ii) any monitored physical, mental, emotional, or spiritual activity of inactivity of a subject.<br>
"Therapy" includes all interventions whether biological, chemical, physical, metaphysical, or combination of the foregoing, intended to sustain or alter the monitored biological condition of a subject.<br>
The PCT patent application publication number WO 01/25473, published April 12,2001, entitled "Systems and Methods for Characterizing a Biological Condition or Agent Using Calibrated Gene Expression Profiles," filed for an invention by inventors herein, and which is herein incorporated by reference, discloses the use of Gene Expression Panels for the evaluation of (i) biological condition (including with respect to health and disease) and (ii) the effect of one or more agents on biological condition (including with respect to health, toxicity, therapeutic treatment and drug interaction).<br>
In particular, Gene Expression Panels may be used for measurement of therapeutic efficacy of natural or synthetic compositions or stimuli that may be formulated individually or in combinations or mixtures for a range of targeted physiological conditions; prediction of toxicological effects and dose effectiveness of a composition or mixture of compositions for an individual or in a population; determination of how two or more different agents administered in a single treatment<br><br>
might interact so as to detect any of synergistic, additive, negative, neutral or toxic activity; performing pre-clinical and clinical trials by providing new criteria for preselecting subjects according to informative profile data sets for revealing disease status; and conducting preliminary dosage studies for these patients prior to conducting phase 1 or 2 trials. These Gene Expression Panels may be employed with respect to samples derived from subjects in order to evaluate their biological condition.<br>
A Gene Expression Panel is selected in a manner so that quantitative measurement of RNA or protein constituents in the Panel constitutes a measurement of a biological condition of a subject. In one kind of arrangement, a calibrated profile data set is employed. Each member of the calibrated profile data set is a function of (i) a measure of a distinct constituent of a Gene Expression Panel and (ii) a baseline quantity.<br>
We have found that valuable and unexpected results may be achieved when the quantitative measurement of constituents is performed under repeatable conditions (within a degree of repeatability of measurement of better than twenty percent, and preferably five percent or better, and more preferably three percent or better). For the purposes of this description and the following claims, we regard a degree of repeatability of measurement of better than twenty percent as providing measurement conditions that are "substantially repeatable". In particular, it is desirable that, each time a measurement is obtained corresponding to the level of expression of a constituent in a particular sample, substantially the same measurement should result for the substantially the same level of expression. In this manner, expression levels for a constituent in a Gene â€¢ Expression Panel may be meaningfully compared from sample to sample. Even if the expression level measurements for a particular constituent are inaccurate (for example, say, 30% too low), the criterion of repeatability means that all measurements for this constituent, if skewed, will nevertheless be skewed systematically, and therefore measurements of expression level of the constituent may be compared meaningfully. In this fashion valuable information may be obtained and compared concerning expression of the constituent under varied circumstances.<br>
In addition to the criterion of repeatability, it is desirable that a second criterion also be satisfied, namely that quantitative measurement of constituents is performed under conditions wherein efficiencies of amplification for all constituents are substantially similar (within one to two percent and typically one percent or less). When both of these criteria are satisfied, then measurement of the expression level of one<br><br>
constituent may be meaningfully compared with measurement of the expression level of another constituent in a given sample and from sample to sample..<br>
Present embodiments relate to the use of an index or algorithm resulting from quantitative measurement of constituents, and optionally in addition, derived from either expert analysis or computational biology (a) in the analysis of complex data sets; (b) to control or normalize the influence of uninf onnative or otherwise minor variances in gene expression values between samples or subjects; (c) to simplify the characterization of a complex data set for comparison to other complex data sets, databases or indices or algorithms derived from complex data sets; (d) to monitor a biological condition of a subject; (e) for measurement of therapeutic efficacy of natural or synthetic compositions or stimuli that may be formulated individually or in combinations or mixtures for a range of targeted physiological conditions; (f) for predictions of toxicological effects and dose effectiveness of a composition or mixture of compositions for an individual or in a population; (g) for determination of how two or more different agents administered in a single treatment might interact so as to detect any of synergistic, additive, negative, neutral of toxic activity (h) for performing pre-clinical and clinical trials by providing new criteria for pre-selecting subjects according to informative profile data sets for revealing disease status and conducting preliminary dosage studies for these patients prior to conducting phase 1 or 2 trials.<br>
Gene expression profiling and the use of index characterization for a particular condition or agent or both may be used to reduce the cost of phase 3 clinical trials and may be used beyond phase 3 trials; labeling for approved drugs; selection of suitable medication in a class of medications for a particular patient that is directed to their unique physiology; diagnosing or detennining a prognosis of a medical condition or an infection which may precede onset of symptoms or alternatively diagnosing adverse side effects associated with administration of a therapeutic agent; managing the health care of a patient; and quality control for different batches of an agent or a mixture of agents.<br>
The subject<br>
The methods disclosed here may be applied to cells of humans, mammals or other organisms without the need for undue experimentation by one of ordinary skill in the art because all cells transcribe RNA and it is known in the art how to extract RNA from all types of cells.<br><br>
Selecting constituents of a Gene Expression Panel The general approach to selecting constituents of a Gene Expression Panel has been described in PCT application publication number WO 01/ 25473. We have designed and experimentally verified a wide range of Gene Expression Panels, each panel providing a quantitative measure, of biological condition, that is derived from a sample of blood or other tissue. For each panel, experiments have verified that a Gene Expression Profile using the panel's constituents is informative of a biological condition. (We show elsewhere that in being informative of biological condition, the Gene Expression Profile can be used to used, among other things, to measure the effectiveness of therapy, as well as to provide a target for therapeutic intervention.) Examples of Gene Expression Panels, along with a brief description of each panel constituent, are provided in tables attached hereto as follows:<br>
Table 1. Inflammation Gene Expression Panel Table 2. Diabetes Gene Expression Panel Table 3. Prostate Gene Expression Panel Table 4. Skin Response Gene Expression Panel Table 5. Liver Metabolism and Disease Gene Expression Panel Table 6. Endothelial Gene Expression Panel Table 7. Cell Health and Apoptosis Gene Expression Panel Table 8. Cytokine Gene Expression Panel Table 9. TNF/IL1 Inhibition Gene Expression Panel Table 10. Chemokine Gene Expression Panel Table 11. Breast Cancer Gene Expression Panel Table 12. Infectious Disease Gene Expression Panel<br>
Other panels may be constructed and experimentally verified by one of ordinary skill in the art in accordance with the principles articulated in the present application.<br>
Design of assays We commonly run a sample through a panel in quadruplicate; that is, a sample is divided into aliquots and for each aliquot we measure concentrations of each constituent in a Gene Expression Panel. Over a total of 900 constituent assays, with each assay conducted in quadruplicate, we found an average coefficient of variation, (standard deviation/average)* 100, of less than 2 percent, typically less than 1 percent, among results for each assay. This figure is a measure of what we call "intra-assay variability". We have also conducted assays on different occasions using the same sample material.<br><br>
With 72 assays, resulting from concentration measurements of constituents in a panel of 24 members, and such concentration measurements determined on three different occasions over time, we found an average coefficient of variation of less than 5 percent, typically less than 2 percent. We regard this as a measure of what we call "inter-assay variability".<br>
We have found it valuable in using the quadruplicate test results to identify and eliminate data points that are statistical "outliers"; such data points are those that differ by a percentage greater, for example, than 3% of the average of all four values and that do not result from any systematic skew that is greater, for example, than 1%. Moreover, if more than one data point in a set of four is excluded by this procedure, then all data for the relevant constituent is discarded.<br>
Measurement of Gene Expression for a constituent in the Panel<br>
For measuring the amount of a particular RNA in a sample, we have used methods known to one of ordinary skill in the art to extract and quantify transcribed RNA from a sample with respect to a constituent of a Gene Expression Panel (See detailed protocols below. Also see PCT application publication number WO. 98/24935 herein incorporated by reference for RNA analysis protocols). Briefly, RNA is extracted from a sample such as a tissue, body fluid, or culture medium in which a population of a subject might be growing. For example, cells may be lysed and RNA eluted in a suitable solution in which to conduct a DNAse reaction. First strand synthesis may be performed using a reverse transcriptase. Gene amplification, more specifically quantitative PCR assays, can then conducted and the gene of interest size calibrated against a marker such as 18S rRNA (Hirayama et al.t Blood 92,1998:46-52). Samples are measured in multiple duplicates, for example, 4 replicates. Relative quantitation of the mRNA is determined by the difference in threshhold cycles between the internal control and the gene of interest. In an embodiment of the invention, quantitative PCR is performed using amplification, reporting agents and instruments such as those supplied commercially by Applied Biosystems (Foster City, CA). Given a defined efficiency of amplification of target transcripts, the point (e.g., cycle number) that signal from amplified target template is detectable may be directly related to the amount of specific message transcript in the measured sample. Similarly, other quantifiable signals such as fluorescence, enzyme activity, disintegrations per minute, absorbance, etc., when correlated to a known concentration of target templates (e.g., a reference standard curve) or normalized to a<br><br>
standard with limited variability can be used to quantify the number of target templates in an unknown sample.<br>
Although not limited to amplification methods, quantitative gene expression techniques may utilize amplification of the target transcript. Alternatively or in combination with amplification of the target transcript, amplification of the reporter signal may also be used. Amplification of the target template may be accomplished by isothermic gene amplification strategies, or by gene amplification by thermal cycling such as PCR.<br>
It is desirable to obtain a definable and reproducible correlation between the amplified target or reporter and the concentration of starting templates. We have discovered that this objective can be achieved by careful attention to, for example, consistent primer-template ratios and a strict adherence to a narrow permissible level of experimental amplification efficiencies (for example 99.0 to 100% relative efficiency, typically 99.8 to 100% relative efficiency). For example, in determining gene expression levels with regard to a single Gene Expression Profile, it is necessary that all constituents of the panels maintain a similar and limited range of primer template ratios (for example, within a 10-fold range) and amplification efficiencies (within, for example, less than 1%) to permit accurate and precise relative measurements for each constituent. We regard amplification efficiencies,as being "substantially similar", for the purposes of this description and the following claims, if they differ by no more than approximately 10%. Preferably they should differ by less than approximately 2% and more preferably by less than approximately 1%. These constraints should be observed over the entire range of concentration levels to be measured associated with the relevant biological condition. While it is thus necessary for various embodiments herein to satisfy criteria that measurements are achieved under measurement conditions that are substantially repeatable and wherein specificity and efficiencies of amplification for all constituents are substantially similar, nevertheless, it is within the scope of the present invention as claimed herein to achieve such measurement conditions by adjusting assay results that do not satisfy these criteria directly, in such a manner as to compensate for errors, so that the criteria are satisfied after suitable adjustment of assay results.<br>
In practice, we run tests to assure that these conditions are satisfied. For example, we typically design and manufacture a number of primer-probe sets, and determine experimentally which set gives the best performance. Even though primer-probe design and manufacture can be enhanced using computer techniques known in the art, and<br><br>
notwithstanding common practice, we still find that experimental validation is useful.<br>
Moreover, in the course of experimental validation, we associate with the selected<br>
primer-probe combination a set of features:<br>
The reverse primer should be complementary to the coding DNA strand. In one<br>
embodiment, the primer should be located" across an intron-exon junction, with not more<br>
than three bases of the three-prime end of the reverse primer complementary to the<br>
proximal exon, (If more than three bases are complementary, then it would tend to<br>
competitively amplify genomic DNA.)<br>
In an embodiment of the invention, the primer probe should amplify cDNA of less<br>
than 110 bases in length and should not amplify genomic DNA or transcripts or cDNA<br>
from related but biologically irrelevant loci.<br>
A suitable target of the selected primer probe is first strand cDNA, which may be<br>
prepared, in one embodiment, is described as follows:<br>
(a)       Use of whole blood for ex vivo assessment of a biological condition affected by an agent.<br>
Human blood is obtained by venipuncture and prepared for assay by separating samples for baseline, no stimulus, and stimulus with sufficient volume for at least three time points. Typical stimuli include lipopolysaccharide (LPS), phytohemagglutinin (PHA) and heat-killed staphylococci (HKS) or carrageean and may be used individually (typically) or in combination. The aliquots of heparinized, whole blood are mixed without stimulus and held at 37Â°C in an atmosphere of 5% C02 for 30 minutes. Stimulus* is added at varying concentrations, mixed and held loosely capped at 37Â°C for 30 min. Additional test compounds may be added at this point and held for varying times depending on the expected pharmacokinetics of the test compound. At defined times, cells are collected by centrifugation, the plasma removed and RNA extracted by various standard means.<br>
Nucleic acids, RNA and or DNA are purified from cells, tissues or fluids of the test population or indicator cell lines. RNA is preferentially obtained from the nucleic acid mix using a variety of standard procedures (or RNA Isolation Strategies, pp. 55-104, in RNA Methodologies, A laboratory guide for isolation and characterization, 2nd edition, 1998, Robert E. Farrell, Jr., Ed., Academic Press), in the present using afilter-<br>
TM<br>
based RNA isolation system from Ambion (RNAqueous    , Phenol-free Total RNA Isolation Kit, Catalog #1912, version 9908; Austin, Texas).<br>
In accordance with one procedure, the whole blood assay for Gene Expression Profiles determination was carried out as follows: Human whole blood was drawn into 10<br><br>
mL Vacutainer tubes with Sodium Heparin. Blood samples were mixed by gently inverting tubes 4-5 times. The blood was used within 10-15 minutes of draw. In the experiments, blood was diluted 2-fold, i.e. per sample per time point, 0.6 mL whole blood + 0.6 mL stimulus. The assay medium was prepared and the stimulus added as appropriate.<br>
A quantity (0.6 mL) of whole blood was then added into each 12 x 75 mm polypropylene tube. 0.6 mL of 2X LPS (from E. coli serotye 0127:B8, Sigma#L3880 or serotype 055, Sigma #L4005, lOng/ml, subject to change in different lots) into LPS tubes was added. Next, 0.6 mL assay medium was added to the "control" tubes with duplicate tubes for each condition. The caps were closed tightly. The tubes were inverted 2-3 times to mix samples. Caps were loosened to first stop and the tubes incubated @ 37Â°C, 5% C02 for 6 hours. At 6 hours, samples were gently mixed to resuspend blood cells, and 1 mL was removed from each tube (using a micropipettor with barrier tip), and transfered to a 2 mL "dolphin" microfuge tube (Costar #3213).<br>
The samples were then centrifuged for 5 min at 500 x g, ambient temperature (IEC centrifuge or equivalent, in microfuge tube adapters in swinging bucket), and as much serum from each tube was removed as possible and discarded. Cell pellets were placed on ice; and RNA extracted as soon as possible using an Ambion RNAqueous kit. (b)      Amplification strategies.<br>
Specific RNAs are amplified using message specific primers or random primers. The specific primers are synthesized from data obtained from public databases (e-.g., Unigene, National Center for Biotechnology Information, National Library of Medicine, Bethesda, MD), including information from genomic and cDNA libraries obtained from humans and other animals. Primers are chosen to preferentially amplify from specific RNAs obtained from the test or indicator samples, see, for example, RT PCR, Chapter 15 in RNA Methodologies, A laboratory guide for isolation and characterization, 2nd edition, 1998,Robert E. Farrell, Jr., Ed., Academic Press; or Chapter 22 pp.143-151, RNA isolation and characterization protocols. Methods in molecular biology, Volume 86,1998, R. Rapley and D. L. Manning Eds., Human Press, or 14 in Statistical refinement of primer design parameters, Chapter 5, pp.55-72, PCR applications: protocols for functional genomics, M.A.Innis, D.H. Gelfand and J.J. Sninsky, Eds., 1999, Academic Press). Amplifications are carried out in either isothermic conditions or using a thermal cycler (for example, a ABI9600 or 9700 or 7700 obtained from Applied Biosystems, Foster City, CA; see Nucleic acid detection methods, pp. 1-24, in Molecular methods for<br><br>
virus detection. D.L.Wiedbrauk and D.H., Farkas, Eds., 1995, Academic Press). Amplified nucleic acids are detected using fluorescent-tagged detection primers (see, for<br>
TM<br>
example, Taqman    PCR Reagent Kit, Protocol, part number 402823 revision A, 1996, Applied Biosystems, Foster City CA.) that are identified and synthesized from publicly known databases as described for the amplification primers. In the present case, amplified DNA is detected and quantified using the ABI Prism 7700 Sequence Detection System obtained from Applied Biosystems (Foster City, CA). Amounts of specific RNAs contained in the test sample or obtained from the indicator cell lines can be related to the relative quantity of fluorescence observed (see for example, Advances in quantitative PCR technology: 5' nuclease assays, Y.S. Lie and CJ. Petropolus, Current Opinion in Biotechnology, 1998,9:43-48, or Rapid thermal cycling and PCR kinetics, pp. 211-229, chapter 14 in PCR applications: protocols for functional genomics, M.A. Innis, D.H. Gelfand and J J. Sninsky, Eds., 1999, Academic Press).<br>
As a particular implementation of the approach described here, we describe in detail a procedure for synthesis of first strand cDNA for use in PCR. This procedure can be used for both whole blood RNA and RNA extracted from cultured cells (i.e. THP-1 cells).<br>
Materials<br>
1.        Applied Biosystems TAQMAN Reverse Transcription Reagents Kit (P/N 808-0234). Kit Components: 10X TaqMan RT Buffer, 25 mM Magnesium chloride, deoxyNTPs mixture, Random Hexamers, RNase Inhibitor, MultiScribe Reverse . Transcriptase (50 U/mL) (2) RNase / DNase free water (DEPC Treated Water from Ambion (P/N 9915G), or equivalent)<br>
Methods<br>
1.	Place RNase Inhibitor and MultiScribe Reverse Transcriptase on ice immediately. All other reagents can be thawed at room temperature and then placed on ice.<br>
2.	Remove RNA samples from -80Â°C freezer and thaw at room temperature and then place immediately on ice.<br>
3.	Prepare the following cocktail of Reverse Transcriptase Reagents for each 100 mL RT reaction (for multiple samples, prepare extra cocktail to allow for pipetting<br>
error):<br>
1 reaction (mL)	11X, e.g. 10 samples (mL)<br>
10XRT Buffer	10.0	110.0<br><br>
25 mM MgC12	22.0	242.0<br>
dNTPs	20.0	220.0<br>
Random Hexamers	5.0	55.0<br>
RNAse Inhibitor	2.0	22.0<br>
Reverse Transcriptase	2.5	27.5<br>
Water	18.5	203.5<br>
Total:	80.0	880.0 (80 mL per sample)<br>
4.	Bring each RNA sample to a total volume of 20 mL in a 1.5 mL microcentrifuge tube (for example, for THP-1 RNA, remove 10 mL RNA and dilute to 20 mL with RNase / DNase free water, for whole blood RNA use 20 mL total RNA) and add 80 mL RT reaction mix from step 5,2,3. Mix by pipetting up and down.<br>
5.	Incubate sample at room temperature for 10 minutes.<br>
6.	Incubate sample at 37Â°C for 1 hour.<br>
7.	Incubate sample at 90Â°C for 10 minutes.<br>
8.	Quick spin samples in microcentrifuge.<br>
9.	Place sample on ice if doing PCR immediately, otherwise store sample at â€”20oC for future use.<br>
10.	PCR QC should be run on all RT samples using 18S and b-actin (see SOP<br>
200-020).<br>
The use of the primer probe with the first strand cDNA as described above to permit measurement of constituents of a Gene Expression Panel is as follows: Set up of a 24-gene Human Gene Expression Panel for Inflammation.<br>
Materials<br>
1.	20X Primer/Probe Mix for each gene of interest.<br>
2.	20X Primer/Probe Mix for 18S endogenous control.<br>
3.	2X Taqman Universal PCR Master Mix.<br>
4.	cDNA transcribed from RNA extracted from cells.<br>
5.	Applied Biosystems 96-Well Optical Reaction Plates.<br>
6.	Applied Biosystems Optical Caps, or optical-clear film.<br>
7.	Applied Biosystem Prism 7700 Sequence Detector.<br>
Methods 1.        Make stocks of each Primer/Probe mix containing the Primer/Probe for the gene of interest, Primer/Probe for 18S endogenous control, and 2X PCR Master Mix as<br><br>
follows. Make sufficient excess to allow for pipetting error e.g. approximately 10% excess. The following example illustrates a typical set up for one gene with quadruplicate samples testing two conditions (2 plates).<br>
1X(1 well)      9X (2 plates worth)<br>
2X Master Mix	12.50	112.50<br>
20X18S Primer/Probe Mix	1.25	11.25<br>
20X Gene of interest Primer/Probe Mix        1.25	11,25<br>
Total	15.00	135.00<br>
2.	Make stocks of cDNA targets by diluting 95pl of cDNA into 2000pJ of water. The amount of cDNA is adjusted to give Ct values between 10 and 18, typically between 12 and 13.<br>
3.	Pipette 15pl of Primer/Probe mix into the appropriate wells of an Applied Biosystems 96-Well Optical Reaction Plate.<br>
4.	Pipette lOjil of cDNA stock solution into each well of the Applied Biosystems 96-Well Optical Reaction Plate.<br>
5.	Seed the plate with Applied Biosystems Optical Caps, or optical-clear film.<br>
6.	Analyze the plate on the AB Prism 7700 Sequence Detector,<br>
Methods herein may also be applied using proteins where sensitive quantitative techniques, such as an Enzyme Linked Immunosorbent Assay (ELIS A) or mass  ' spectroscopy, are available and well-known in the art for measuring the amount of a protein constituent, (see WO 98/24935 herein incorporated by reference).<br>
Baseline profile data sets<br>
The analyses of samples from single individuals and from large groups of individuals provide a library of profile data sets relating to a particular panel or series of panels. These profile data sets may be stored as records in a library for use as baseline profile datajsets. As the term "baseline" suggests, the stored baseline profile data sets serve as comparators for providing a calibrated profile data set that is informative about a biological condition or agent. Baseline profile data sets may be stored in libraries and classified in a number of cross-referential ways. One form of classification may rely on the characteristics of the panels from which the data sets are derived. Another form of classification may be by particular biological condition. The concept of biological condition encompasses any state in which a cell or population of cells may be found at<br><br>
any one time. This state may reflect geography of samples, sex of subjects or any other discriminator. Some of the discriminators may overlap. The libraries may also he accessed for records associated with a single subject or particular clinical trial. The classification of baseline profile data sets may further be annotated with medical information about a particular subject, a medical condition, a particular agent etc.<br>
The choice of a baseline profile data set for creating a calibrated profile data set is related to the biological condition to be evaluated, monitored, or predicted, as well as&gt; the intended use of the calibrated panel, e.g., as to monitor drug development, quality control or other uses. It may be desirable to access baseline profile data sets from the same subject for whom a first profile data set is obtained or from different subject at varying times, exposures to stimuli, drugs or complex compounds; or may be derived from like or dissimilar populations.<br>
The profile data set may arise from the same subject for which the first data set is obtained, where the sample is taken at a separate or similar time, a different or similar site or in a different or similar physiological condition. For example, Fig. 5 provides a protocol in which the sample is taken before stimulation or after stimulation. The profile data set obtained from the unstimulated sample may serve as a baseline profile data set for the sample taken after stimulation. The baseline data set may also be derived from a library containing profile data sets of a population of subjects having some defining characteristic or biological condition. The baseline profile data set may also correspond to some ex vivo or in vitro properties associated with an in vitro cell culture. The resultant calibrated profile data sets may then be stored as a record in a database or library (Fig. 6) along with or separate from the baseline profile data base and optionally the first profile data set although the first profile data set would normally become incorporated into a baseline profile data set under suitable classification criteria. The remarkable consistency of Gene Expression Profiles associated with a given biological condition makes it valuable to store profile data, which can be used, among other things for normative reference purposes. The normative reference can serve to indicate the degree to which a subject conforms to a given biological condition (healthy or diseased) and, alternatively or in addition, to provide a target for clinical intervention.<br>
Selected baseline profile data sets may be also be used as a standard by which to judge manufacturing lots in terms of efficacy, toxicity, etc. Where the effect of a therapeutic agent is being measured, the baseline data set may correspond to Gene Expression Profiles taken before administration of the agent. Where quality control for a<br><br>
newly manufactured product is being determined, the baseline data set may correspond with a gold standard for that product However, any suitable normalization techniques may be employed. For example, an average baseline profile data set is obtained from authentic material of a naturally grown herbal nutriceutical and compared over time and over different lots in order to demonstrate consistency, or lack of consistency, in lots of compounds prepared for release.<br>
Calibrated data Given the repeatability we have achieved in measurement of gene expression, described above in connection with "Gene Expression Panels" and ugene amplification", we conclude that where differences occur in measurement under such conditions, the differences are attributable to differences in biological condition. Thus we have found that calibrated profile data sets are highly reproducible in samples taken from the same individual under the same conditions. We have similarly found that calibrated profile data<br>
sets are reproducible in samples that are repeatedly tested. We have also found repeated<br>
i<br>
instances wherein calibrated profile data sets obtained when samples from a subject are exposed ex vivo to a compound are comparable to calibrated profile data from a sample that has been exposed to a sample in vivo. We have also found, importantly, that an indicator cell line treated with an agent can in many cases provide calibrated profile data sets comparable to those obtained from in vivo or ex vivo populations of cells. Moreover, we have found that administering a sample from a subject onto indicator cells can provide informative calibrated profile data sets with respect to the biological condition of the subject including the health, disease states, therapeutic interventions, aging or exposure to environmental stimuli or toxins of the subject.<br>
Calculation of calibrated profile data sets and computational aids<br>
The calibrated profile data set may be expressed in a spreadsheet or represented graphically for example, in a bar chart or tabular form but may also be expressed in a three dimensional representation. The function relating the baseline and profile data may be a ratio expressed as a logarithm. The constituent may be itemized on the x-axis and the logarithmic scale may be on the y-axis. Members of a calibrated data set may be expressed as a positive value representing a relative enhancement of gene expression or as a negative value representing a relative reduction in gene expression with respect to the baseline.<br>
Each member of the calibrated profile data set should be reproducible within a range with respect to similar samples taken from the subject under similar conditions. For<br><br>
example, the calibrated profile data sets may be reproducible within one order of<br>
magnitude with respect to similar samples taken from the subject under similar<br>
conditions. More particularly, the members may be reproducible within 50%, more<br>
particularly reproducible within 20%, and typically within 10%. In accordance with<br>
embodiments of the invention, a pattern of increasing, decreasing and no change in<br>
relative gene expression from each of a plurality of gene loci examined in the Gene<br>
Expression Panel may be used to prepare a calibrated profile set that is informative with<br>
regards to a biological condition, biological efficacy of an agent treatment conditions or<br>
for comparison to populations. Patterns of this nature may be used to identify likely<br>
candidates for a drug trial, used alone or in combination with other clinical indicators to<br>
be diagnostic or prognostic with respect to a biological condition or may be used to guide<br>
the development of a pharmaceutical or nutriceutical through manufacture, testing and<br>
marketing.<br>
The numerical data obtained from quantitative gene expression and numerical<br>
data from calibrated gene expression relative to a baseline profile data set may be stored<br>
i in databases or digital storage mediums and may retrieved for purposes including<br>
managing patient health care or for conducting clinical trials or for characterizing a drug.<br>
The data may be transferred in physical or wireless networks via the World Wide Web,<br>
email, or internet access site for example or by hard copy so as to be collected and pooled<br>
from distant geographic sites (Fig. 8).<br>
In an embodiment of the present invention, a descriptive record is stored in a single database or multiple databases where the stored data includes the raw gene expression data (first profile data set) prior to transformation by use of a baseline profile data set, as well as a record of the baseline profile data set used to generate the calibrated profile data set including for example, annotations regarding whether the baseline profile data set is derived from a particular Signature Panel and any other annotation that facilitates interpretation and use of the data.<br>
Because the data is in a universal format, data handling may readily be done with a computer. The data is organized so as to provide an output optionally corresponding to a graphical representation of a calibrated data set.<br>
For example, a distinct sample derived from a subject being at least one of RNA or protein may be denoted as Pi. The first profile data set derived from sample Pi is denoted Mj, where Mj is a quantitative measure of a distinct RNA or protein constituent of Pi. The record Ri is a ratio of M and P and may be annotated with additional data on<br><br>
the subject relating to, for example, age, diet, ethnicity, gender, geographic location, medical disorder, mental disorder, medication, physical activity, body mass and environmental exposure. Moreover, data handling may further include accessing data from a second condition database which may contain additional medical data not presendy held with the calibrated profile data sets. In this context, data access may be via a computer network.<br>
The above described data storage on a computer may provide the information in a form that can be accessed by a user. Accordingly, the user may load the information onto a second access site including downloading the information. However, access may be restricted to users having a password or other security device so as to protect the medical records contained within. A feature of this embodiment of the invention is the ability of a user to add new or annotated records to the data set so the records become part of the biological information.<br>
The graphical representation of calibrated profile data sets pertaining to a product such as a drug provides an opportunity for standardizing a product by means of the calibrated profile, more particularly a signature profile. The profile may be used as a feature with which to demonstrate relative efficacy, differences in mechanisms of actions, etc. compared to other drugs approved for similar or different uses.<br>
The various embodiments of the invention may be also implemented as a computer program product for use with a computer system. The product may include program code for deriving a first profile data set and for producing calibrated profiles. Such implementation may include a series of computer instructions fixed either on a tangible medium, such as a computer readable medium (for example, a diskette, CD-ROM, ROM, or fixed disk), or transmittable to a computer system via a modem or other interface device, such as a communications adapter coupled to a network. The network coupling may be for example, over optical or wired communications lines or via wireless techniques (for example, microwave, infrared or other transmission techniques) or some combination of these. The series of computer instructions preferably embodies all or part of the functionality previously described herein with respect to the system. Those skilled in the art should appreciate that such computer instructions can be written in a number of programming languages for use with many computer architectures or operating systems. Furthermore, such instructions may be stored in any memory device, such as semiconductor, magnetic, optical or other memory devices, and may be transmitted using any communications technology, such as optical, infrared, microwave, or other<br><br>
transmission technologies. It is expected that such a computer program product may be distributed as a removable medium with accompanying printed or electronic documentation (for example, shrink wrapped software), preloaded with a computer system (for example, on system ROM or fixed disk), or distributed from a server or electronic bulletin board over a network (for example, the Internet or World Wide Web). In addition, a computer system is further provided including derivative modules for deriving a first data set and a calibration profile data set.<br>
The calibration profile data sets in graphical or tabular form, the associated databases, and the calculated index or derived algorithm, together with information extracted from the panels, the databases, the data sets or the indices or algorithms are commodities that can be sold together or separately for a variety of purposes as described in WO 01/25473.<br>
Index construction In combination, (i) the remarkable consistency of Gene Expression Profiles with respect to a biological condition across a population and (ii) the use of procedures that provide substantially reproducible measurement of constituents in a Gene Expression Panel giving rise to a Gene Expression Profile, under measurement conditions wherein specificity and efficiencies of amplification for all constituents of the panel are substantially similar, make possible the use of an index that characterizes a Gene Expression Profile, and which therefore provides a measurement of a biological condition.<br>
An index may be constructed using an index function that maps values in a Gene Expression Profile into a single value that is pertinent to the biological condition at hand. The values in a Gene Expression Profile are the amounts of each constituent of the Gene Expression Panel that corresponds to the Gene Expression Profile. These constituent amounts form a profile data set, and the index function generates a single valueâ€”the indexâ€” from the members of the profile data set.<br>
The index function may conveniently be constructed as a linear sum of terms, each term being what we call a "contribution function" of a member of the profile data set. For example, the contribution function may be a constant times a power of a member of the profile data set. So the index function would have the form I = Â£CMim ,<br><br>
where I is the index, Mi is the value of the member / of the profile data set, Q is a constant, and P(i) is a power to which MÂ» is raised, the sum being formed for all integral values of i up to the number of members in the data set. We thus have a linear polynomial expression.<br>
The values Q and P(i) may be determined in a number of ways, so that the index / is informative of the pertinent biological condition. One way is to apply statistical techniques, such as latent class modeling, to the profile data sets to correlate clinical data or experimentally derived data, or other data pertinent to the biological condition. In this connection, for example, maybe employed the software from Statistical Innovations, Belmont, Massachusetts, called Latent GoldÂ®. See the web pages at www.statisticalinnovations.corn/lg/, which are hereby incorporated herein by reference. Alternatively, other simpler modeling techniques may be employed in a manner known in the art. The index function for inflammation may be constructed, for example, in a manner that a greater degree of inflammation (as determined by the a profile data set for the Inflammation Gene Expression Profile) correlates with a large value of the index function. In a simple embodiment, therefore, each P(i) may be +1 or -1, depending on whether the constituent increases or decreases'with increasing inflammation. As discussed in further detail below, we have constructed a meaningful inflammation index that is proportional to the expression<br>
1/4{IL1A} + 1/4{IL1B} + 1/4{TNF} + 1/4{INFG} - 1/{IL10}, where the braces around a constituent designate measurement of such constituent and the constituents are a subset of the Inflammation Gene Expression Panel of Table 1.<br>
Just as a baseline profile data set, discussed above, can be used to provide an appropriate normative reference, and can even be used to create a Calibrated profile data set, as discussed above, based on the normative reference, an index that characterizes a Gene Expression Profile can also be provided with a normative value of the index function used to create the index. This normative value can be determined with respect to a relevant population, so that the index may be interpreted in relation to the normative value. The relevant population may have in common a property that is at least one of age group, gender, ethnicity, geographic location, diet, medical disorder, clinical indicator, medication, physical activity, body mass, and environmental exposure.<br>
As an example, the index can be constructed, in relation to a normative Gene Expression Profile for a population of healthy subjects, in such a way that a reading of approximately 1 characterizes normative Gene Expression Profiles of healthy subjects.<br><br>
Let us further assume that the biological condition that is the subject of the index is inflammation; a reading of 1 in this example thus corresponds to a Gene Expression Profile that matches the norm for healthy subjects. A substantially higher reading then may identify a subject experiencing an inflammatory condition. The use of 1 as identifying a normative value, however, is only one possible choice; another logical choice is to use 0 as identifying the normative value. With this choice, deviations in the index from zero can be indicated in standard deviation units (so that values lying between -1 and +1 encompass 90% of a normally distributed reference population. Since we have found that Gene Expression Profile values (and accordingly constructed indices based on them) tend to be normally distributed, the 0-centered index constructed in this manner is highly informative. It therefore facilitates use of the index in diagnosis of disease and setting objectives for treatment. The choice of 0 for the normative value, and the use of standard deviation units, for example, are illustrated in Fig. 17B, discussed below.<br>
EXAMPLES<br>
Example 1:' Acute Inflammatory Index to Assist in Analysis of Large, Complex Data Sets. In one embodiment of the invention the index value or algorithm can be used to reduce a complex data set to a single index value that is informative with respect to the inflammatory state of a subject. This is illustrated in Figs. 1A and IB.<br>
Fig. 1A is entitled Source Precision Inflammation Profile Tracking of A Subject Results in a Large, Complex Data Set. The figure shows the results of assaying 24 genes from the Inflammation Gene Expression Panel (shown in Table 1) on eight separate days during the course of optic neuritis in a single male subject.<br>
Fig. IB shows use of an Acute Inflammation Index. The data displayed in Fig. 1A above is shown in this figure after calculation using an index function proportional to the following mathematical expression: (1/4{BL1A} + 1/4{IL1B} + 1/4{TNF} + 1/4{INFG} -1/{IL10}).      |<br>
Example 2: Use of acute inflammation index or algorithm to monitor a biological condition of a sample or a subject. The inflammatory state of a subject reveals . information about the past progress of the biological condition, future progress, response to treatment, etc. The Acute Inflammation Index may be used to reveal such information about the biological condition of a subject. This is illustrated in Fig. 2.<br><br>
The results of the assay for inflammatory gene expression for each day (shown for 24 genes in each row of Fig. 1 A) is displayed as an individual histogram after calculation. The index reveals clear trends in inflammatory status that may correlated with therapeutic intervention (Fig. 2).<br>
Fig. 2 is a graphical illustration of the acute inflammation index calculated at 9 different, significant clinical milestones from blood obtained from a single patient treated medically with for optic neuritis. Changes in the index values for the Acute Inflammation Index correlate strongly with the expected effects of therapeutic intervention. Four clinical milestones have been identified on top of the Acute Inflammation Index in this figure including (1) prior to treatment with steroids, (2) treatment with IV solumedrol at 1 gram per day, (3) post-treatment with oral prednisone at 60 mg per day tapered to 10 mg per day and (4) post treatment. The data set is the same as for Fig. 1. The index is proportional to 1/4{IL1A} + 1/4{DL1B} + 1/4{TNF} + 1/4{INFG} - 1/{IL10}. As expected, the acute inflammation index falls rapidly with treatment with IV steroid, goes up during less efficacious treatment with oral prednisone and returns to the pre-treatment level after the steroids have been discontinued and metabolized completely.<br>
Example 3: Use of the acute inflammatory index to set dose, including concentrations and timing, for compounds in development or for compounds to be tested in human and non-human subjects as shown in Fig. 3. The acute inflammation index may be used as a common reference value for therapeutic compounds or interventions without common mechanisms of action. The compound that induces a gene response to a compound as indicated by the index, but fails to ameliorate a known biological conditions may be compared to a different compounds with varying effectiveness in treating the biological condition.<br>
Fig. 3 shows the effects of single dose treatment with 800 mg of ibuprofen in a single donor as characterized by the Acute Inflammation Index. 800 mg of over-the-counter ibuprofen were taken by a single subject at Time=0 and Time=48 hr. Gene expression values for the indicated five inflammation-related gene loci were determined as described below at times=2,4,6,48,50, 56 and 96 hours. As expected the acute inflammation index falls immediately after taking the non-steroidal anti-inflammatory ibuprofen and returns to baseline after 48 hours. A second dose at T=48 follows the same kinetics at the first dose and returns to baseline at the end of the experiment at T=96.<br><br>
Example 4: Use of the acute inflammation index to characterize efficacy, safety, and mode of physiological action for an agent, which may be in development and/or may be complex in nature. This is illustrated in Fig. 4.<br>
Fig. 4 shows that the calculated acute inflammation index displayed graphically for five different conditions including (A) untreated whole blood; (B) whole blood treated in vitro with DMSO, an non-active carrier compound; (C) otherwise unstimulated whole blood treated in vitro with dexamethasone (0.08 ug/ml); (D) whole blood stimulated in vitro with lipopolysaccharide, a known pro-inflammatory compound, (LPS, 1 ng/ml) and (E) whole blood treated in vitro with LPS (1 ng/ml) and dexamethasone (0.08 ug/ml). Dexamethasone is used as a prescription compound that is commonly used medically as an anti-inflammatory steroid compound. The acute inflammation index is calculated from the experimentally determined gene expression levels of inflammation-related genes expressed in human whole blood obtained from a single patient. Results of mRNA expression are expressed as Ct's in this example, but may be expressed as, e.g., relative fluorescence units, copy number or any other quantifiable, precise and calibrated form, â€¢ for the genes ILIA, IL1B, TNF, IFNG and IL10. From the gene expression values, the acute inflammation values'were determined algebraically according in proportion to the expression 1/4{IL1A} + 1/4{IL1B} + 1/4{TNF} + 1/4{INFG} - 1/{IL10}.<br>
Example 5: Development and use of population normative values for Gene Expression Profiles. Figs. 6 and 7 show the arithmetic mean values for gene expression profiles (using the 48 loci of the Inflammation Gene Expression Panel of Table 1) obtained from whole blood of two distinct patient populations. These populations are both normal or undiagnosed. The first population, which is identified as Bonfils (the plot points for which are represented by diamonds), is composed of 17 subjects accepted as blood donors at the Bonfils Blood Center in Denver, Colorado. The second population is 9 donors, for which Gene Expression Profiles were obtained from assays conducted four times over a four-week period. Subjects in this second population (plot points for which are represented by squares) were recruited from employees of Source Precision Medicine, Inc., the assignee herein. Gene expression averages for each population were calculated for each of 48 gene loci of the Gene Expression Inflammation Panel. The results for loci 1-24 (sometimes referred to below as the Inflammation 48A loci) are shown in Fig. 6 and for loci 25-48 (sometimes referred to below as the Inflammation 48B loci) are shown in Fig. 7.<br><br>
The consistency between gene expression levels of the two distinct populations is dramatic. Both populations show gene expressions for each of the 48 loci that are not significantly different from each other. This observation suggests that there is a "normal" expression pattern for human inflammatory genes, that a Gene Expression Profile, using the Inflammation Gene Expression Panel of Table 1 (or a subset thereof) characterizes that expression pattern, and that a population-normal expression pattern can be used, for example, to guide medical intervention for any biological condition that results in a change from the nonnal expression pattern.<br>
In a similar vein, Fig. 8 shows arithmetic mean values for gene expression profiles (again using the 48 loci of the Inflammation Gene Expression Panel of Table 1) also obtained from whole blood of two distinct patient populations. One population, expression values for which are represented by triangular data points, is 24 normal, undiagnosed subjects (who therefore have no known inflammatory disease). The other population, the expression values for which are represented by diamond-shaped data points, is four patients with rheumatoid arthritis and who have failed therapy (who therefore have unstable rheumatoid arthritis).<br>
As remarkable as the consistency of data from the two distinct normal populations * shown in Figs. 6 and 7 is the systematic divergence of data from the normal and diseased populations shown in Fig. 8. In 45 of the shown 48 inflammatory gene loci, subjects with unstable rheumatoid arthritis showed, on average, increased inflammatory gene expression (lower cycle threshold values; Ct), than subjects without disease. The.data thus further demonstrate that is possible to identify groups with specific biological conditions using gene expression if the precision and calibration of the underlying assay are carefully designed and controlled according to the teachings herein.<br>
Fig. 9, in a maimer analogous to Fig. 8, shows the shows arithmetic mean values for gene expression profiles using 24 loci of the Inflammation Gene Expression Panel of Table 1) also obtained from whole blood of two distinct patient populations. One population, expression values for which are represented by diamond-shaped data points, is 17 nonnal, imdiagnosed subjects (who therefore have no known inflammatory disease) who are blood donors. The other population, the expression values for which are represented by square-shaped data points, is 16 subjects, also normal and undiagnosed, who have been monitored over six months, and the averages of these expression values are represented by the square-shaped data points. Thus the cross-sectional gene expression-value averages of a first healthy population match closely the longitudinal<br><br>
gene expression-value averages of a second healthy population., with approximately 7% or less variation in measured expression value on a gene-to-gene basis.<br>
Fig. 10 shows the shows gene expression values (using 14 loci of the Inflammation Gene Expression Panel of Table 1) obtained from whole blood of 44 normal undiagnosed blood donors (data for 10 subjects of which is shown). Again, the gene expression values for each member of the population are closely matched to those for the population, represented visually by the consistent peak heights for each of the gene loci. Other subjects of the population and other gene loci than those depicted here display results that are consistent with those shown here.<br>
In consequence of these principles, and in various embodiments of the present invention, population normative values for a Gene Expression Profile can be used in comparative assessment of individual subjects as to biological condition, including both for purposes of health and/or disease. In one embodiment the normative values for a Gene Expression Profile may be used as a baseline in computing a "calibrated profile data set" (as defined at the beginning of this section) for a subject that reveals the deviation of such subject's gene expression from population normative values. Population normative values for a Gene Expression Profile can also be used as baseline values in constructing index functions in accordance with embodiments of the present invention. As a result, for example, an index function can be constructed to reveal not only the extent of an    * individual's inflammation expression generally but also in relation to normative values.<br>
Example 6: Consistency of expression values, of constituents in Gene Expression Panels, over time as reliable indicators of biological condition. Fig. 11 shows the expression levels for each of four genes (of the Inflammation Gene Expression Panel of Table 1), of a single subject, assayed monthly over a period of eight months. It can be seen that the expression levels are remarkably consistent over time.<br>
Figs. 12 and 13 similarly show in each case the expression levels for each of 48 genes (of the Inflammation Gene Expression Panel of Table 1), of distinct single subjects (selected in each case on the basis of feeling well and not taking drugs), assayed, in the case of Fig. 12 weekly over a period of four weeks, and in the case of Fig. 13 monthly over a period of six months. In each case, again the expression levels are remarkably consistent over time, and also similar across individuals.<br>
Fig. 14 also shows the effect over time, on inflammatory gene expression in a single human subject, of the administration of an anti-inflammatory steroid, as assayed using the Inflammation Gene Expression Panel of Table 1. In this case, 24 of 48 loci are<br><br>
displayed. The subject had a baseline blood sample drawn in a PAX RNA isolation tube and then took a single 60 mg dose of prednisone, an anti-inflammatory, prescription steroid. Additional blood samples were drawn at 2 hr and 24 hr post the single oral dose. Results for gene expression are displayed for all three time points, wherein values for the baseline sample are shown as unity on the x-axis. As expected, oral treatment with prednisone resulted in the decreased expression of most of inflammation-related gene loci, as shown by the 2-hour post-administration bar graphs. However, the 24-hour post-administration bar graphs show that, for most of the gene loci having reduced gene expression at 2 hours, there were elevated gene expression levels at 24 hr.<br>
Although the baseline in Fig. 14 is based on the gene expression values before drug intervention associated with the single individual tested, we know from the previous example, that healthy individuals tend toward population normative values in a Gene Expression Profile using the Inflammation Gene Expression Panel of Table 1 (or a subset of it). We conclude from Fig. 14 that in an attempt to return the inflammatory gene expression levels to those demonstrated in Figs. 6 and ? (normal or set levels}, interference with the normal expression induced a compensatory gene expression response that over-compensated for the drug-induced response, perhaps because the prednisone had been significantly metabolized to inactive forms or eliminated from the subject.<br>
Fig. 15, in a manner analogous to Fig. 14, shows the effect over time, via whole blood samples obtained from a human subject, administered a single dose of prednisone, on expression of 5 genes (of the Inflammation Gene Expression Panel of Table 1). The samples were taken at the time of administration (t = 0) of the prednisone, then at two and 24 hours after such administration. Each whole blood sample was challenged by the addition of 0.1 ng/ml of lipopolysaccharide (a Gram-negative endotoxin) and a gene expression profile of the sample, post-challenge, was determined. It can seen that the two-hour sample shows dramatically reduced gene expression of the 5 loci of the Inflammation Gene Expression Panel, in relatioxi to the expression levels at the time of administration (t = 0). At 24 hours post administration, the inhibitory effect of the prednisone is no longer apparent, and at 3 of the 5 loci, gene expression is in fact higher than at t = 0, illustrating quantitatively at the molecular level the well-known rebound effect.<br>
Fig. 16 also shows the effect over time, on inflammatory gene expression in a single human subject suffering from rheumatoid arthritis, of the administration of a TNF-<br><br>
inhibiting compound, but here the expression is shown in comparison to the cognate locus average previously determined (in connection with Figs. 6 and 7) for the normal (i.e., undiagnosed, healthy) population. As part of a larger international study involving patients with rheumatoid arthritis, the subject was followed over a twelve-week period. The subject was enrolled in the study because of a failure to respond to conservative drug therapy for rheumatoid arthritis and a plan to change therapy and begin immediate treatment with a TNF-inhibiting compound. Blood was drawn from the subject prior to initiation of new therapy (visit 1). After initiation of new therapy, blood was drawn at 4 weeks post change in therapy (visit 2), 8 weeks (visit 3), and 12 weeks (visit 4) following the start of new therapy. Blood was collected in PAX RNA isolation tubes, held at room temperature for two hours and then frozen at -30Â°C.<br>
Frozen samples were shipped to the central laboratory at Source Precision Medicine, the assignee herein, in Boulder, Colorado for determination of expression levels of genes in the 48-gene Inflammation Gene Expression Panel of Table 1. The blood samples were thawed and RNA extracted according to the manufacturer's recommended procedure. RNA was converted to cDNA and the level of expression of the 48 inflammatory genes was determined. Expression results are shown for 11 of the 48 ' loci in Fig. 16. When the expression results for the 11 loci are compared from visit one to a population average of normal blood donors from the United States, the subject shows considerable difference. Similarly, gene expression levels at each of the subsequent physician visits for each locus are compared to the same normal average value. Data from visits 2,3 and 4 document the effect of the change in therapy. In each visit following the change in the therapy, the level of inflammatory gene expression for 10 of the 11 loci is closer to the cognate locus average previously determined for the normal (i.e., undiagnosed, healthy) population.<br>
Fig. 17A further illustrates the consistency of inflammatory gene expression, illustrated "here with respect to 7 loci of (of the Inflammation Gene Expression Panel of Table 1), in a population of 44 normal, undiagnosed blood donors. For each individual locus is shown the range of values lying within Â± 2 standard deviations of the mean expression value, which corresponds to 95% of a normally distributed population. Notwithstanding the great width of the confidence interval (95%), the measured gene expression value (ACT)â€”remarkablyâ€”still lies within 10% of the mean, regardless of the expression level involved. As described in further detail below, for a given biological condition an index can be constructed to provide a measurement of the condition. This is<br><br>
possible as a result of the conjunction of two circumstances: (i) there is a remarkable consistency of Gene Expression Profiles with respect to a biological condition across a population and (ii) there can be employed procedures that provide substantially reproducible measurement of constituents in a Gene Expression Panel giving rise to a Gene Expression Profile, under measurement conditions wherein specificity and efficiencies of amplification for all constituents of the panel are substantially similar and which therefore provides a measurement of a biological condition. Accordingly, a function of the expression values of representative constituent loci of Fig. 17A is here used to generate an inflammation index value, which is normalized so that a reading of 1 â€¢     corresponds to constituent expression values of healthy subjects, as shown in the right-hand portion of Fig. 17A.<br>
In Fig. 17B, an inflammation index value was determined for each member of a population of 42 normal undiagnosed blood donors, and the resulting distribution of index values, shown in the figure, can be seen to approximate closely a normal distribution, notwithstanding the relatively small population size. The values of the index are shown relative to a 0-based median, with deviations from the median calibrated in standard deviation units. Thus 90% of the population lies within +1" and -1 of a 0 value. We have constructed various indices, which exhibit similar behavior.<br>
Fig. 17C illustrates the use of the same index as Fig. 17B, where the inflammation 1     median for a normal population has been set to zero and both normal and diseased subjects are plotted in standard deviation units relative to that median. An inflammation index value was determined for each member of a normal, undiagnosed population of 70 individuals (black bars). The resulting distribution of index values, shown in Fig. 17C, can be seen to approximate closely a normal distribution. Similarly, index values were calculated for individuals from two diseased population groups, (1) rheumatoid arthritis patients treated with methotrexate (MTX) who are about to change therapy to more efficacious drugs (e.g., TNF inhibitors)(hatched bars), and (2) rheumatoid arthritis patients treated with disease modifying anti-rheumatoid drugs (DMARDS) other than MTX, who are about to change therapy to more efficacious drugs (e.g., MTX). Both populations present index values that are skewed upward (demonstrating increased inflammation) in comparison to the normal distribution. This figure thus illustrates the utility of an index to derived from Gene Expression Profile data to evaluate disease status and to provide an objective and quantifiable treatment objective. When these two<br><br>
populations were treated appropriately, index values from both populations returned to a more normal distribution (data not shown here).<br>
Fig. 18 plots, in a fashion similar to that of Fig. 17A, Gene Expression Profiles, for the same 7 loci as in Fig. 17A, two different 6-subject populations of rheumatoid â–  arthritis patients. One population (called "stable" in the figure) is of patients who have responded well to treatment and the other population (called "unstable" in the figure) is of patients who have not responded well to treatment and whose therapy is scheduled for change. It can be seen that the expression values for the stable population, lie within the range of the 95% confidence interval, whereas the expression values for the unstable population for 5 of the 7 loci are outside and above this range. The right-hand portion 'of the figure shows an average inflammation index of 9.3 for the unstable population and an average inflammation index of 1.8 for the stable population, compared to 1 for a normal undiagnosed population. The index thus provides a measure of the extent of the underlying inflammatory condition, in this case, rheumatoid arthritis. Hence the index, besides providing a measure of biological condition, can be used to measure the effectiveness of therapy as well as to provide a target for therapeutic intervention.<br>
Fig. 19 thus illustrates use of the inflammation index for assessment of a single subject suffering from rheumatoid arthritis, who has not responded well to traditional therapy with methotrexate. The inflammation index for this subject is shown on the far right at start of a new therapy (a TNF inhibitor), and then, moving leftward, successively, 2 weeks, 6 weeks, and 12 weeks thereafter. The index can be seen moving towards normal, consistent with physician observation of the patient as responding to the new treatment<br>
Fig. 20 similarly illustrates use of the inflammation index for assessment of three subjects suffering from rheumatoid arthritis, who have not responded well to traditional therapy with methotrexate, at the beginning of new treatment (also with a TNF inhibitor), and 2 weeks and 6 weeks thereafter. The index in each case can again be seen moving generally towards normal, consistent with physician observation of the patients as responding to the new treatment.<br>
Each of Figs. 21-23 shows the inflammation index for an international group of subjects, suffering from rheumatoid arthritis, each of whom has been characterized as stable (that is, not anticipated to be subjected to a change in therapy) by the subject's treating physician. Fig. 21 shows the index for each of 10 patients in the group being treated with methotrexate, which known to alleviate symptoms without addressing the<br><br>
underlying disease. Fig. 22 shows the index for each of 10 patients in the group being treated with Enbrel (an TNF inhibitor), and Fig. 23 shows the index for each 10 patients being treated with Remicade (another TNF inhibitor). It can be seen that the inflammation index for each of the patients in Fig. 21 is elevated compared to normal, whereas in Fig. 22, the patients being treated with Enbrel as a class have an inflammation index that comes much closer to normal (80% in the normal range). In Fig. 23, it can be seen that, while all but one of the patients being treated with Remicade have an inflammation index at or below normal, two of the patients have an abnormally low inflammation index, suggesting an immunosuppressive response to this drug. (Indeed, studies have shown that Remicade has been associated with serious infections in some subjects, and here the immunosuppressive effect is quantified.) Also in Fig. 23, one subject has an inflammation index that is significantly above the normal range. This subject in fact was also on a regimen of an anti-inflammation steroid (prednisone) that was being tapered; within approximately one week after the inflammation index was sampled, the subject experienced a significant flare of clinical symptoms.<br>
Remarkably, these examples show a measurement, derived from the assay of blood taken from a subject, pertinent to the subject's arthritic condition. Given that the measurement pertains to the extent of inflammation, it can be expected that other inflammation-based conditions, including, for example, cardiovascular disease, may be monitored in a similar fashion.<br>
Fig. 24 illustrates use of the inflammation index for assessment of a single subject suffering from inflammatory bowel disease, for whom treatment with Remicade was initiated in three doses. The graphs show the inflammation index just prior to first treatment, and then 24 hours after the first treatment; the index has returned to the normal range. The index was elevated just prior to the second dose, but in the normal range prior to the third dose. Again, the index, besides providing a measure of biological condition, is here used to measure the effectiveness of therapy (Remicade), as well as to provide a target for therapeutic intervention in terms of both dose and schedule.<br>
Fig. 25 shows Gene Expression Profiles with respect to 24 loci (of the Inflammation Gene Expression Panel of Table 1) for whole blood treated with Ibuprofen in vitro in relation to other non-steroidal anti-inflammatory drugs (NS AIDs), The profile for Ibuprofen is in front. It can be seen that all of the NSAIDs, including Ibuprofen share a substantially similar profile, in that the patterns of gene expression across the loci are<br><br>
similar. Notwithstanding these similarities, each individual drug has its own distinctive signature.<br>
Fig. 26 illustrates how the effects of two competing anti-inflammatory compounds can be compared objectively, quantitatively, precisely, and reproducibly. In this example, expression of each of a panel of two genes (of the Inflammation Gene Expression Panel of Table 1) is measured for varying doses (0.08 - 250 p.g/ml) of each drug in vitro in whole blood. The market leader drug shows a complex relationship between dose and inflammatory gene response. Paradoxically, as the dose is increased, gene expression for both loci initially drops and then increases in the case the case of the market leader. For the other compound, a more consistent response results, so that as the dose is increased, the gene expression for both loci decreases more consistently.<br>
Figs. 27 through 41 illustrate the use of gene expression panels in early identification and monitoring of infectious disease. These figures plot the response, in expression products of the genes indicated, in whole blood, to the administration of various infectious agents or products associated with infectious agents. In each figure, the gene expression levels are "calibrated", as that term is defined herein, in relation to baseline expression levels determined with respect to the whole blood prior to administration of the relevant infectious agent. In this respect the figures are similar in nature to various figures of our below-referenced patent application WO 01/25473 (for example, Fig. 15 therein). The concentration change is shown ratiometrically, and the baseline level of 1 for a particular gene locus corresponds to an expression level for such locus that is the same, monitored at the relevant time after addition of the infectious agent or other stimulus, as the expression level before addition of the stimulus. Ratiometric changes in concentration are plotted on a logarithmic scale. Bars below the unity line represent decreases in concentration and bars above the unity line represent increases in concentration, the magnitude of each bar indicating the magnitude of the ratio of the * change. We have shown in WO 01/25473 and other experiments that, under appropriate conditions, Gene Expression Profiles derived in vitro by exposing whole blood to a stimulus can be representative of Gene Expression Profiles derived in vivo with exposure to a corresponding stimulus.<br>
Fig. 27 uses a novel bacterial Gene Expression Panel of 24 genes, developed to discriminate various bacterial conditions in a host biological system. Two different stimuli are employed: lipotechoic acid (LTA), a gram positive cell wall constituent, and lipopolysaccharide (LPS), a gram negative cell wall constituent. The final concentration<br><br>
immediately after administration of the stimulus was 100 ng/mL, and the ratiometric changes in expression, in relation to pre-administration levels, were monitored for each stimulus 2 and 6 hours after administration. It can be seen that differential expression can be observed as early as two hours after administration, for example, in the IFNA2 locus, as well as others, permitting discrimination in response between gram positive and gram negative bacteria.<br>
Fig. 28 shows differential expression for a single locus, IFNG, to LTA derived from three distinct sources: S. pyogenes, B. subtilis, and S. aureus. Each stimulus was administered to achieve a concentration of 100 ng/mL, and the response was monitored at 1,2,4,6, and 24 hours after administration. The results suggest that Gene Expression Profiles can be used to distinguish among different infectious agents, here different species of gram positive bacteria.<br>
Figs. 29 and 30 show the response of the Inflammation 48A and 48B loci respectively (discussed above in connection with Figs. 6 and 7 respectively) in whole blood to administration of a stimulus of S. aureus and of a stimulus of E. coli (in the<br>
7	ft<br>
indicated concentrations, just after administration, of 10 and 10 CFU/mL respectively), monitored 2 hours after administration in relation to the pre-administration baseline. The figures show that many of the loci respond to the presence of the bacterial infection within two hours after infection.<br>
Figs. 31 and 32 correspond to Figs. 29 and 30 respectively and are similar to them, with the exception that the monitoring here occurs 6 hours after administration. More of the loci are responsive to the presence of infection. Various loci, such as IL2, show expression levels that discriminate between the two infectious agents.<br>
Fig. 33 shows the response of the Inflammation 48A loci to the administration of a stimulus of E. coli (again in the concentration just after administration of 106 CFU/mL) and to the administration of a stimulus of an E. coli filtrate containing E. coli bacteria by products but lacking E. coli bacteria. The responses were monitored at 2, 6, and 24 hours after administration. It can be seen, for example, that the responses over time of loci IL1B, IL18 and CSF3 to E.coli and to E. coli filtrate are different.<br>
Fig. 34 is similar to Fig. 33, but here the compared responses are to stimuli from E. coli filtrate alone and from E. coli filtrate to which has been added polymyxin B, an antibiotic known to bind to Iipopolysaccharide (LPS). An examination of the response of EL1B, for example, shows that presence of polymyxin B did not affect the response of the<br><br>
locus to E. coli filtrate, thereby indicating that LPS does not appear to be a factor in the response of IL1B to E. coli filtrate.<br>
Fig. 35 illustrates the responses of the Inflammation 48A loci over time of whole blood to a stimulus of S. aureus (with a concentration just after administration of 107 CFU/mL) monitored at 2,6, and 24 hours after administration. It can be seen that response over time can involve both direction and magnitude of change in expression. (See for example, IL5 and IL18.)<br>
Figs. 36 and 37 show the responses, of the Inflammation 48A and 48B loci   * respectively, monitored at 6 hours to stimuli from E. coli (at concentrations of 106 and 102 CFU/mL immediately after administration) and from S. aureus (at concentrations of 107 and 102 CFU/mL immediately after administration). It can be seen, among other things, that in various loci, such as B7 (Fig. 36), TACI, PLA2G7, and CIQA (Fig. 37), E. coli produces a much more pronounced response than S. aureus. The data suggest strongly that Gene Expression Profiles can be used to identify with high sensitivity the presence of gram negative bacteria and to discriminate against gram positive bacteria.<br>
Figs. 38 and 39 show the responses, of the Inflammation 48B and 48A loci respectively, monitored 2,6, and 24 hours after administration, to stimuli of high concentrations of S. aureus and E. coli respectively (at respective concentrations of 107 and 106 CFU/mL immediately after administration). The responses over time at many loci involve changes in magnitude and direction. Fig. 40 is similar to Fig. 39, but shows the responses of the Inflammation 48B loci.<br>
Fig. 41 similarly shows the responses of the Inflammation 48A loci monitored at 24 hours after administration to stimuli high concentrations of S. aureus and E. coli respectively (at respective concentrations of 107 and 106 CFU/mL immediately after administration). As in the case of Figs. 20 and 21, responses at some loci, such as GROl and GR02, discriminate between type of infection.<br>
These data support our conclusion that Gene Expression Profiles with sufficient precision and calibration as described herein (1) can determine subpopulations of individuals with a known biological condition; (2) may be used to monitor the response of patients to therapy; (3) may be used to assess the efficacy and safety of therapy; and (4) may used to guide the medical management of a patient by adjusting therapy to bring one or more relevant Gene Expression Profiles closer to a target set of values, which may be normative values or other desired or achievable values. We have shown that Gene Expression Profiles may provide meaningful information even when derived from ex<br><br>
vivo treatment of blood or other tissue. We have also shown that Gene Expression Profiles derived from peripheral whole blood are informative of a wide range of conditions neither directly nor typically associated with blood.<br>
Furthermore, in embodiments of the present invention, Gene Expression Profiles can also be used for characterization and early identification (including pre-symptomatic states) of infectious disease, such as sepsis. This characterization includes discriminating between infected and uninfected individuals, bacterial and viral infections, specific subtypes of pathogenic agents, stages of the natural history of infection (e.g., early or late), and prognosis. Use of the algorithmic and statistical approaches discussed above to achieve such identification and to discriminate in such fashion is within the scope of various embodiments herein.<br>
02331/00119 224730.1<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
What is claimed is:<br>
1.	A method, for evaluating a biological condition of a subject, based on a sample<br>
from the subject, comprising:<br>
deriving from the sample a profile data set, the profile data set including a plurality of members, each member being a quantitative measure of the amount of a distinct RNA or protein constituent in a panel of constituents selected so that measurement of the constituents enables evaluation of the biological condition; and<br>
in deriving the profile data set, achieving such measure for each constituent under measurement conditions that are substantially repeatable.<br>
2.	A method of providing an index that is indicative of the state of a subject, as to a<br>
biological condition, based on a sample from the subject, the method comprising:<br>
deriving from the sample a profile data set, the profile data set including a plurality of members, each member being a quantitative measure of the amount of a distinct RNA or protein constituent in a panel of constituents selected so that measurement of the constituents enables evaluation of the biological condition; and<br>
in deriving the profile data "set, achieving such measure for each constituent under measurement conditions that are substantially repeatable; and<br>
applying values from the profile data set to an index function that provides a mapping from an instance of a profile data set into a single-valued measure of biological condition, so as to produce an index pertinent to the biological condition of the subject<br>
3.	A method according to claim 1, further comprising in deriving the profile data set, achieving such measure for each constituent under measurement conditions wherein specificity and efficiencies of amplification for all constituents are substantially similar.<br>
4.	A method according to claim 2, further comprising in deriving the profile data set, achieving such measure for each constituent under measurement conditions wherein specificity and efficiencies of amplification for all constituents are substantially similar.<br>
5.	A method according to claims 2, further comprising providing with the index a normative value of the index function, determined with respect to a relevant population, so that the index may be interpreted in relation to the normative value.<br>
6.	A method according to claim 4, further comprising providing with the index a normative value of the index function, determined with respect to a relevant population, so that the index may be interpreted in relation to the normative value.<br><br>
7.	A method according to claim 5, wherein providing the normative value includes constructing the index function so that the normative value is approximately 1.<br>
8.	A method according to claim 6, wherein providing the normative value includes constructing the index function so that the normative value is approximately 0 and deviations in the index function from 0 are expressed in standard deviation units.<br>
9.	A method according to claim 5, wherein the relevant population has in common a property that is at least one of age group, gender, ethnicity, geographic location, diet, medical disorder, clinical indicator, medication, physical activity, body mass, and environmental exposure.<br>
10.	A method according to claim 6, wherein the relevant population is has in common a property that is at least one of age group, gender, ethnicity, geographic location, diet, medical disorder, clinical indicator, medication, physical activity, body mass, and environmental exposure.<br><br>
11.	A method according to any of claims 1 through 10, wherein efficiencies of amplification, expressed as a percent, for all constituents lie within a range of approximately 2 percent.<br>
12.	A method'according to any of claims lthrough 10, wherein efficiencies of amplification, expressed as a percent, for ail constituents lie within a range of approximately 1 percent.<br>
13.	A method according to any of claims lthrough 10, wherein measurement conditions are repeatable so that such measure for each constituent has a coefficient of variation, on repeated derivation of such measure from the sample, that is less than approximately 3 percent.<br>
14.	A method according to claim 11, wherein measurement conditions are repeatable so that such measure for each constituent has a coefficient of variation, on repeated derivation of such measure from the sample, that is less than approximately 3 percent.<br>
15.	A method according to claim 12, wherein measurement conditions are repeatable so that such measure for each constituent has a coefficient of variation, on repeated derivation of such measure from the sample, that is less than approximately 3 percent.<br>
16.	A method according to any of claims 1 through 10, wherein the panel includes at least three constituents.<br>
17.	A method according to any of claims 1 through 10, wherein the panel has fewer than approximately 500 constituents.<br><br>
18.	A method according to claim 1 through 10, wherein the biological condition being evaluated is with respect to a localized tissue of the subject and the sample is derived from tissue or fluid of a type distinct from that of the localized tissue.<br>
19.	A method according to any of claims 1 through 10, wherein the biological condition is inflammation and the panel of constituents includes at least two, and optionally at least three, four, five, six, seven, eight, nine or ten, of the constituents of the Inflammation Gene Expression Panel of Table 1.<br>
20.	A method according to any of claims 1 through 10, wherein the biological condition is diabetes and the panel of constituents includes at least two, and optionally at least three, four, five, six, seven, eight, nine or ten, of the constituents of the Diabetes Gene Expression Panel of Table 2.<br>
21.	A method according to any of claims 1 through 10, wherein the biological condition is prostate health or disease and the panel of constituents includes at least two, and optionally at least three, four, five, six, seven, eight, nine or ten, of the constituents of the Prostate Gene Expression Panel of Table 3.<br><br>
22.	A method according to any of claims 1 through 10, wherein the biological condition is manifested in skin and the panel of constituenT5~includes at least two, and optionally at least three, four, five, six, seven, eight, nine or ten, of the constituents of the Skin Response Gene Expression Panel of Table 4,<br>
23.	A method according to any of claims 1 through 10, wherein the biological condition is liver metabolism and disease and the panel of constituents includes at least two, and optionally at least three, four, five, six, seven, eight, nine or ten, of the constituents of the Liver Metabolism and Disease Gene Expression Panel of Table 5.<br>
24.	A method according to any of claims 1 through 10, wherein the biological condition is vascular and the panel of constituents includes at least two, and optionally at least three, four, five, six, seven, eight, nine or ten, of the constituents of the Endothelial Gene Expression Panel of Table 6.<br>
25.	A method according to any of claims 1 through 10, wherein the biological condition is abnormal cell development and the panel of constituents includes at least two, and optionally at least three, foitr, five, six, seven, eight, nine or ten, of the constituents of the Cell Health and Apoptosis Gene Expression Panel of Table 7.<br>
26.	A method according to any of claims 1 through 10, wherein the biological condition is inflammation and the panel of constituents includes at least two, and<br><br>
optionally at least three, four, five, six, seven, eight, nine or ten, of the constituents of the Cytokine Gene Expression Panel of Table 8.<br>
27.	A method according to any of claims 1 through 10, wherein the biological condition is inflammation and the panel of constituents includes at least two, and optionally at least three, four, five, six, seven, eight, nine or ten, of the constituents of the TNF/DL1 Inhibition Gene Expression Panel of Table 9.<br>
28.	A method according to any of claims 1 through 10, wherein the biological condition is inflammation and the panel of constituents includes at least two, and optionally at least three, four, five, six, seven, eight, nine or ten, of the constituents of the Chemokine Gene Expression Panel of Table 10.<br>
29.	A method according to any of claims 1 through 10, wherein the biological condition is cancer and the panel of constituents includes at least two, and optionally at least three, four, five, six, seven, eight, nine or ten, of the constituents of the Breast Cancer Gene Expression Panel of Table 11.<br><br>
30.	A method according to any of claims 1 through 10, wherein the biological condition is infectious disease and the panel of constituents includes at least two, and optionally'at least three, four, five, six, seven, eight, nine or ten, of the constituents of the Infectious Disease Gene Expression Panel of Table 12.<br>
31.	A method of providing an index that is indicative of the biological state of a subject based on a sample from the subject, the method comprising:<br>
deriving from the sample a first profile data set, the first profile data set including a plurality of members, each member being a quantitative measure of the amount of a distinct RNA or protein constituent in a panel of constituents, the panel including at least two of the constituents of the Inflammation Gene Expression Panel of Table 1;<br>
wherein, in deriving the first profile data set, such measure is performed for each constituent both under conditions wherein specificity and efficiencies of amplification for all constituents are substantially similar and under substantially repeatable conditions; and<br>
applying values from the first profile data set to an index function that provides a mapping from an instance of a profile data set into a single-valued measure of biological condition, so as to produce an index pertinent to the biological condition of the sample or the subject.<br>
32.	A method according to claim 31, wherein the panel includes at least three of the<br>
constituents in the Inflammation Gene Expression Panel.<br><br>
33.	A method according to claim 31, wherein the panel includes at least four of the constituents in the Inflammation Gene Expression Panel.<br>
34.	A method according to claim 31, wherein the panel includes at least five of the constituents in the Inflammation Gene Expression Panel.<br>
35.	A method according to claim 31, wherein the panel includes at least six of the constituents in the Inflammation Gene Expression Panel.<br>
36.	A method according to claim 31, wherein the panel includes at least 10 of the constituents in the Inflammation Gene Expression Panel.<br>
37.	A method according to claim 31, wherein the biological condition is an inflammatory condition.<br>
38.	A method according to claim 31, wherein<br>
(i) the mapping by the index function is further based on an instance of a relevant baseline profile data set; and<br>
(ii) applying values from the first profile data set further includes applying values from a corresponding baseline profile data set from the same subject or from a population of subjects or samples with a similar or different biological condition.<br>
39.	A method according to claim 31, wherein the index function is consfructed to deviate from a normative value generally upwardly in an instance of an increase in expression of a constituent whose increase is associated with an increase of inflammation and also in an instance of a decrease in expression of a constituent whose decrease is associated with an increase of inflammation.<br>
40.	A method according to claim 39, wherein the index function is constructed to weigh the expression value of a constituent in the panel generally in accordance'with the extent to which its expression level is determined to be correlated with extent of inflammation.<br>
41.	A method according to claim 39, wherein the index function is constructed to take into account clinical insight into inflammation biology.<br>
42.	A method according to claim 39, wherein the index function is constructed to take into account experimentally derived data.	i<br>
43.	A method according to claim 39, wherein the index function is constructed to take into account relationships derived from computer analysis of profile data sets in a data base associating profile data sets with clinical and demographic data.<br><br>
44.	A method according to claim 31, wherein the panel includes at least one constituent that is associated with a specific inflammatory disease.<br>
45.	A method according to claim 31, wherein (i) the mapping by the index function is also based on an instance of at least one of demographic data and clinical data and (ii) applying values from the first profile data set also includes applying a set of values associated with at least one of demographic data and clinical data.<br>
46.	A method according to claim 31, wherein a portion of deriving the first profile data set is performed at a first location and applying the values from the first profile data set is performed at a second location, and data associated with performing the portion of deriving the first profile data set are communicated to the second location over a network to enable, at the second location, applying the values from the first profile data set.<br>
47.	A method according to claim 31, wherein the index function is a linear sum. of terms, each term being a contribution function of a member of the profile data set.<br><br>
48.	A method according to claim 47, wherein the contribution function is a weighted power of the member.<br>
49.	A method according to claim 48, wherein the power is integral, so that the index function is a linear polynomial.<br>
50.	A method according to claim 49, wherein the profile data set includes at least three members corresponding to constituents selected from the group consisting of ILIA, IL1B, TNF, IFNG and IL10.<br>
51.	A method according to claim 49, wherein the profile data set includes at least four members corresponding to constituents selected from the group consisting of ILIA, IL1B, TNF, IFNG and IL10.<br>
52.	A method according to claim 51, wherein the index function is approximately proportional to 1/4{IL1A} + 1/4{IL1B} + 1/4{TNF} + 1/4{INFG} -l/{IL10}and braces around a constituent designate measurement of such constituent.<br>
53.	A method of analyzing complex data associated with a sample from a subject for information pertinent to inflammation, the method comprising:<br><br>
deriving a Gene Expression Profile for the sample, the Gene Expression Profile being based on a Signature Panel for Inflammation; and<br>
using the Gene Expression Profile to determine a Gene Expression Profile Inflammatory Index for the sample.<br>
54.      A method of monitoring the biological condition of a subject, the method comprising:<br>
deriving a Gene Expression Profile for each of a series of samples over time from the subject, the Gene Expression Profile being based on a Signature Panel for Inflammation; and<br>
for each of the series of samples, using the corresponding Gene Expression Profile to determine a Gene Expression Profile Inflammatory Index. â–    55.      A method of determining at least one of (i) an effective dose of an agent to be administered to a subject and (ii) a schedule for administration of an agent to a subject, the method comprising:<br>
deriving a Gene Expression Profile for a sample from the subject, the Gene Expression Profile being based on a Signature Panel for Inflammation;<br>
using the Gene Expression Profile to determine a Gene Expression Profile Inflammatory Index for the sample; and<br>
using the Gene Expression Profile Inflammatory Index as an indicator in establishing at least one of the effective dose and the schedule.<br>
56.	A method of guiding a decision to continue or modify therapy for a<br>
biological condition of a subject, the method comprising:<br>
deriving a Gene Expression Profile for a sample from the subject, the Gene Expression Profile being based on a Signature Panel for Inflammation; and<br>
using the Gene Expression Profile to determine a Gene Expression Profile Inflammatory Index for the sample.<br>
57.	A method of predicting change in biological condition of a subject as a<br>
result of exposure to an agent, the method comprising:<br>
deriving a first Gene Expression Profile for a first sample from the subject in the absence of the agent, the first Gene Expression Profile being based on a Signature Panel for Inflammation;<br><br>
deriving a second Gene Expression Profile for a second sample from the subject in the presence of the agent, the second Gene Expression Profile being based on the same Signature Panel; and<br>
using the first and second Gene Expression Profiles to determine correspondingly a first Gene Expression Profile Inflammatory Index and a second Gene Expression Profile Inflammatory Index.<br>
58.	A method according to claim 57, wherein the agent is a compound.<br>
59.	A method according to claim 58, wherein the compound is therapeutic.<br>
60.	A method of evaluating a property of an agent, the property being at least one of purity, potency, quality, efficacy or safety, the method comprising;<br>
deriving a first Gene Expression Profile from a sample reflecting exposure to the agent of (i) the sample, or (ii) a population of cells from which the sample is derived, or (iii) a subject from which the sample is derived;<br>
using the Gene Expression Profile to determine a Gene Expression Profile Inflammatory Index; and<br>
using the Gene Expression Profile Inflammatory Index in determining the property.<br>
61.	A method, for evaluating a biological condition of a subject, based on a<br>
sample from the subject, comprising:<br>
deriving from the sample a first profile data set, the first profile dataset including a plurality of members, each member being a quantitative measure of the amount of a distinct RNA or protein constituent in a panel of constituents selected so that measurement of the constituents enables measurement of the biological condition; and<br>
producing a calibrated profile data set for the panel, wherein each member of the calibrated profile data set is a function of a corresponding member of the first profile data set and a corresponding member of a baseline profile data set for the panel, wherein each member of the baseline data set is a normative measure, determined with respect to a relevant population of subjects, of the amount of one of the constituents in the panel, the calibrated profile data set providing a measure of the biological condition of the subject.<br>
62.	A method, for evaluating a biological condition of a subject, based on a<br>
sample from the subject, comprising:<br>
applying the first sample or a portion thereof to a defined population of indicator cells;<br><br>
obtaining from the indicator cells a second sample containing at least one of RNAs or proteins;<br>
deriving from the second sample a first profile data set, the first profile data set including a plurality of members, each member being a quantitative measure of the amount of a distinct RNA or protein constituent in a panel of constituents selected so that measurement of the constituents enables measurement of the biological condition; and<br>
producing a calibrated profile data set for the panel, wherein each member of the calibrated profile data set is a function of a corresponding member of the first profile data set and a corresponding member of a baseline profile data set for the panel,, wherein each member of the baseline data set is a normative measure, determined with respect to a relevant population of subjects, of the amount of one of the constituents in the panel, the calibrated profile data set providing a measure of the biological condition of the subject.<br>
63.	A method, for evaluating a biological condition affected by an agent, the<br>
method comprising:<br>
obtaining, from a target population of cells to which the agent has been administered, a sample having at least one of RNAs and proteins;<br>
deriving from the sample a first profile data set, the first profile data set including a plurality of members, each member being a quantitative measure of the amount of a distinct RNA or protein constituent in a panel of constituents selected so that measurement of the constituents enables measurement of the biological condition; and<br>
producing a calibrated profile data set for the panel, wherein each member of the calibrated profile data set is a function of a corresponding member of the first profile data set and a corresponding member of a baseline profile data set for the panel, wherein each member of the baseline data set is a normative measure, determined with respect to a relevant population of subjects, of the amount of one of the constituents in the panel, the calibrated profile data set providing a measure of the biological condition as affected by the agent.<br>
64.	A method according to any of claims 61 through 63, wherein the relevant population is a population of healthy subjects.<br>
65.	A method according to any of claims 61 through 63, wherein the relevant population is has in common a property that is at least one of age group, gender, ethnicity, geographic location, diet, medical disorder, clinical indicator, medication, physical activity, body mass, and environmental exposure.<br><br>
66.	A method, for evaluating a biological condition of a subject-based on a<br>
sample from the subject, comprising:<br>
deriving from the sample a first profile data set, the first profile dataset including a plurality of members, each member being a quantitative measure of the amount of a distinct RNA or protein constituent in a panel of constituents selected so that measurement of the constituents enables measurement of the biological condition;<br>
wherein, in deriving the first profile data set, such measure is performed for each constituent both under conditions wherein specificity and efficiencies of amplification for all constituents are substantially similar and under substantially repeatable conditions.<br>
67.	A method according to claim 66, wherein the panel includes at least two,<br>
and optionally, at least three, at least four, at least five, or at least six of the<br>
constituents of the Inflammation Gene Expression Panel of Table 1.<br>
68.	A method according to any of claims 61 through 63, the panel including at least two of the constituents of the Inflammation Gene Expression Panel of Table 1 and wherein, in deriving the first profile data set, such measure is performed for each constituent both under conditions wherein specificity and efficiencies of amplification for all constituents are substantially similar and under substantially repeatable conditions.<br>
69.	A method according to any of claims 61 through 63, the panel including at least three of the constituents of the Inflammation Gene Expression Panel of Table 1 and wherein, in deriving the first profile data set, such measure is performed for each constituent both under conditions wherein specificity and efficiencies of amplification for all constituents are substantially similar and under substantially repeatable conditions.<br>
70.	A method according to any of claims 61 through 63, the panel including at least four of the constituents of the Inflammation Gene Expression Panel of Table 1 and wherein, in deriving the first profile data set, such measure is performed for each constituent both under conditions wherein specificity and efficiencies of amplification for all constituents are substantially similar and under substantially repeatable conditions.<br>
71.	A method according to any of claims 61 through 63, the panel including at least five of the constituents of the Inflammation Gene Expression Panel of Table<br><br>
1 and wherein, in deriving the first profile data set, such measure is performed for each constituent both under conditions wherein specificity and efficiencies of amplification for all constituents are substantially similar and under substantially repeatable conditions.<br>
72.	A method according to any of claims 61 through 63, the panel including at least six of the constituents of the Inflammation Gene Expression Panel of Table 1 and wherein, in deriving the first profile data set, such measure is performed for each constituent both under conditions wherein specificity and efficiencies of amplification for all constituents are substantially similar and under substantially repeatable conditions.<br>
73.	A method, for evaluating the effect on a biological condition by a first agent in relation to the effect by a second agent, the method comprising:<br>
obtaining, from first and second target populations of cells to which the first and second agents have been respectively administered, first and second samples respectively, each sample having at least one of RNAs and proteins;<br>
deriving from the first sample a first profile data set and from the second sample a second profile data set, the profile data sets each including a plurality of members, each member being a quantitative measure of the amount of a distinct RNA or protein constituent in a panel of constituents selected so that measurement of the constituents enables measurement of the biological condition; and<br>
producing for the-panel a first calibrated profile data set and a second profile data set, wherein (i) each member of the first calibrated profile data set is a function of a corresponding member of the first profile data set and a corresponding member of a baseline profile data set for the panel, wherein each member of the baseline data set is a normative measure, determined with respect to a relevant population of subjects, of the amount of one of the constituents in the panel, and (ii) each member of the second calibrated profile data set is a function of a corresponding member of the second profile data set and a corresponding member of the baseline profile data set, the calibrated profile data sets providing a measure pf the effect by the first agent on the biological condition in relation to the effect by the second agent<br>
wherein, in deriving the first and second profile data sets, such measure is performed for each constituent both under conditions wherein specificity and efficiencies of amplification for all constituents are substantially similar and under substantially<br><br>
repeatable conditions.<br>
74.	A method according to claim 73, wherein the first agent is a first drug and<br>
the second agent is a second drug.<br>
75.	A method according to claim 73, wherein the first agent is a drug and the second agent is a complex mixture.<br>
76.	A method according to claim 74, wherein the first agent is a drug and the second agent is a nutriceutical.<br>
77.	A method of providing an index that is indicative of the inflammatory state of a subject based on a sample from the subject, the method comprising:<br>
deriving from the sample a first profile data set, the first profile data set including a plurality of members, each member being a quantitative measure of the amount of a distinct RNA or protein constituent in a panel of constituents, the panel- including at least two of the constituents of the Inflammation Gene Expression Panel of Table 1; and<br>
applying values from the first profile data set to an index function that provides a mapping from an instance of a profile data set into a single-valued measure of biological condition, so as to produce an index pertinent to the biological condition of the sample or the subject;<br>
wherein the index function also uses data from a baseline profile data set for the panel, wherein each member of the baseline data set is a normative measure, determined with respect to a relevant population of subjects, of the amount of one of the constituents in the panel; and<br>
wherein, in deriving the first profile data set and the baseline data set, such measure is performed for each constituent both under conditions wherein specificity and efficiencies of amplification for all constituents are substantially similar and under substantially repeatable conditions.<br><br>
78. A method, for evaluating a biological condition of a subject, based on a sample from the subject, substantially as herein described with reference to the accompanying drawings.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTg0LWNoZW5wLTIwMDQgIGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">984-chenp-2004  abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTg0LWNoZW5wLTIwMDQgIGNsYWltcyBncmFudGVkLnBkZg==" target="_blank" style="word-wrap:break-word;">984-chenp-2004  claims granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTg0LWNoZW5wLTIwMDQgIGNvcnJlc3BvbmRlbmNlIG90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">984-chenp-2004  correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTg0LWNoZW5wLTIwMDQgIGNvcnJlc3BvbmRlbmNlIHBvLnBkZg==" target="_blank" style="word-wrap:break-word;">984-chenp-2004  correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTg0LWNoZW5wLTIwMDQgIGZvcm0tMSAgMjUtMDMtMjAwOC5wZGY=" target="_blank" style="word-wrap:break-word;">984-chenp-2004  form-1  25-03-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTg0LWNoZW5wLTIwMDQgIGZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">984-chenp-2004  form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTg0LWNoZW5wLTIwMDQgIGZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">984-chenp-2004  form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTg0LWNoZW5wLTIwMDQgIGZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">984-chenp-2004  form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTg0LWNoZW5wLTIwMDQgIGZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">984-chenp-2004  form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTg0LWNoZW5wLTIwMDQgIHBldGl0aW9ucy5wZGY=" target="_blank" style="word-wrap:break-word;">984-chenp-2004  petitions.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTg0LWNoZW5wLTIwMDQgIHBvd2VyIG9mIGF0dG9ybmV5LnBkZg==" target="_blank" style="word-wrap:break-word;">984-chenp-2004  power of attorney.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTg0LWNoZW5wLTIwMDQgY2xhaW1zIGdyYW50ZWQucGRm" target="_blank" style="word-wrap:break-word;">984-chenp-2004 claims granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTg0LWNoZW5wLTIwMDQgZGVzY3JpcHRpb24gKGNvbXBsZXRlKSBncmFudGVkLTEucGRm" target="_blank" style="word-wrap:break-word;">984-chenp-2004 description (complete) granted-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTg0LWNoZW5wLTIwMDQgZGVzY3JpcHRpb24gKGNvbXBsZXRlKSBncmFudGVkLnBkZg==" target="_blank" style="word-wrap:break-word;">984-chenp-2004 description (complete) granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTg0LWNoZW5wLTIwMDQgZHJhd2luZ3MgZ3JhbnRlZC5wZGY=" target="_blank" style="word-wrap:break-word;">984-chenp-2004 drawings granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTg0LWNoZW5wLTIwMDQtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">984-chenp-2004-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTg0LWNoZW5wLTIwMDQtY29ycmVzcG9uZG5lY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">984-chenp-2004-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTg0LWNoZW5wLTIwMDQtY29ycmVzcG9uZG5lY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">984-chenp-2004-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTg0LWNoZW5wLTIwMDQtZGVzY3JpcHRpb24oY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">984-chenp-2004-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTg0LWNoZW5wLTIwMDQtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">984-chenp-2004-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTg0LWNoZW5wLTIwMDQtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">984-chenp-2004-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTg0LWNoZW5wLTIwMDQtZm9ybSAxOC5wZGY=" target="_blank" style="word-wrap:break-word;">984-chenp-2004-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTg0LWNoZW5wLTIwMDQtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">984-chenp-2004-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTg0LWNoZW5wLTIwMDQtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">984-chenp-2004-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTg0LWNoZW5wLTIwMDQtcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">984-chenp-2004-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="227974-biofuel-cell.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="227976-indolinone-derivatives-useful-as-protein-kinase-inhibitors.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>227975</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>984/CHENP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>10/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>06-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>27-Jan-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>06-May-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>SOURCE PRECISION MEDICINE, INC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>2425 North 55th Street, Suite 111 Boulder, CO 80301,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>BEVILACQUA, Michael, C</td>
											<td>5383 Westridge Drive, Boulder, CO 80301,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>CHERONIS, John, C</td>
											<td>24011 Pleasant Road, Conifer, CO 80433,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>TRYON, Victor</td>
											<td>6770 SW 14th Street, Loveland, CO 80537,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>BANKAITIS DAVIS, DANUTE, M</td>
											<td>3964 PEPPLE BEACH DRIVE, LONGMONT, CO 80503,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US02/36084</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-11-08</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/369,633</td>
									<td>2002-04-03</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/348,213</td>
									<td>2001-11-09</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>60/340,881</td>
									<td>2001-12-07</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>4</td>
									<td>60/376,997</td>
									<td>2002-04-30</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/227975-an-in-vitro-method-of-providing-an-index-for-characterization-of-biological-conditions by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:30:32 GMT -->
</html>
