<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/227977-an-in-vitro-method-of-identifying-anti-diabetic-compounds by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:30:43 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 227977:AN IN VITRO METHOD OF IDENTIFYING ANTI-DIABETIC COMPOUNDS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">AN IN VITRO METHOD OF IDENTIFYING ANTI-DIABETIC COMPOUNDS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>ABSTRACT 1555/CHENP/2004 &#x27;*An in vitro method of identifying anti-diabetic compounds&quot; This invention relates to an in vitro method of identifying anti-diabetic compounds such as PPARy modulators, wherein a polypeptide having PIM-3 activity is incubated in vitro with a compound to be tested, and change in PIM-3 activity is determined.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
The present invention relates to novel PIM-3 kinase subtypes and to an in vitro method of identifying anti-diabetic compounds using the same.<br>
The rat PIM-3 (originally termed KID-1, KID 'kinase induced by depolyarisation"), frog PIM-1, and human and murine PIM-1 are all known to have serine/threonine protein kinase activity in in vitro phosphorylation assays. The high polypeptide sequence similarity between human, murine and rat PIM-3, frog PIM-1, and human and murine PIM-1, demonstrates that human and murine PIM-3 are a serine/threonine protein kinase.<br>
Rat PIM-3 is described by Feldman, J.D. et al. (J. Biol. Chem. (1998) 273, 16535-16543). Rat PiM-3 is induced in specific regions of the hippocampus and cortex in response to kainic acid and electroconvulsive shock suggesting that PIM-3 is involved in neuronal function, synaptic plasticity, learning, and memory as well as kainic acid seizures and some nervous system-related diseases such as seizures and epilepsy.<br>
US 6,143,540, Konietzko, U. et al. (EMBO (1999) 18, 3359-3369) and Eichmann, A. (Oncogene (2000) 19, 1215-1224) also refer to PIM-3 kinase.<br>
The present invention provides novel PIM-3 encoding sequences and novel uses of PIM-3.<br>
The present invention provides a novel human and miu'ine PIM-3 sequence. SEQ ID NO.l depicts the DNA sequence and SEQ ID N0.2 the predicted amino acid sequence of human PIM-3. The open reading frame of SEQ ID NO:l extends from nucleotide 17 to nucleotide 995 (SEQ ID N0.3).<br><br>
SEQ ID NO. 5 depicts the DNA sequence and SEQ ID NO. 6 the predicted amino acid sequence of murine PlM-3. The open reading frame of SEQ ID NO. 5 extends from nucleotide 199 to nucleotide 1177 (SEQ ID NO. 7).<br>
The present invention demonstrates that expression of the rat PlM-3 gene is decreased in adipocytes in two independent models of insulin resistance. Treatment with an insulin sensitizer causes an increase in PIM-3 gene expression in murine 3T3-L1 cells. Because human and murine P(M-3 is the species ortholog of rat PIM-3 human and murine PIM-3 is involved in some or all of the processes and diseases in which rat PIM-3 is involved. PiM-3, in particular human and murine PIM-3 is involved in development of insulin resistance. In addition PlM-3, In particular human and murine PlM-3 is involved in development of type 2 diabetes mellitus. In addition, the human and murine PIM-3 paralogs, the PIM-1 proteins, are proto-oncogenes. Consequently, PlM-3, in particluar human and murine PIM-3 are involved in cell growth regulation, cancer, and related pathways and diseases.<br>
The present invention relates to the use of PIM-3 encoding nucleic acid molecules, PIM-3 proteins and protein homologs in a) screening assays for identifying compounds that modulate insulin" resistance or type 2 diabetes meliitus; o) detection assays for detecting insulin resistance or type 2 diabetes mellitus (e.g. chromosomal mapping, tissue typing, forensic biology); c) predictive medicine ^prediction of insulin resistance or type 2 diabetes mellitus by e.g. diagnostic assays, prognostic assays, monitoring clinical trials, and pharmacogenomics).<br>
The present invention relates in particular to an isolated nucleic acid molecule ;omprising nucleotide sequence SEQ ID. NO. 1. The present invention further relates 0 an isolated nucleic acid molecule comprising nucleotide sequence SEQ ID NO. 3.  '<br>
"he present invention relates to an isolated nucleic acid molecule comprising a<br>
lucleotide sequence selected from<br>
i)       SEQ ID NO. 5,	'<br>
))       SEQ ID NO. 7,<br><br>
c)	a DNA sequence which hybridize to SEQ ID NO. 5 or SEQ ID NO. 7, and<br>
d)	DNA sequences which are degenerated as a result of the genetic code to sequences defined in a), b) and c) and wiiich encode for a polypeptide of the PlM-3 type.<br>
The present invention furthermore relates to vectors and host cells comprising the respective DNA sequences or parts thereof.<br>
The present invention relates to polypeptides having P1IVV3 activity, said polypeptides ) being selected from<br>
a)	a polypeptide having amino acid sequence SEQ ID NO. 6,<br>
b)	a polypeptide t^iat in respect to a) is deficient in one or more amino acids.<br>
c)	a polypeptide in whicii in respect to a) one or more amino acids are replaced with different amino acids,<br>
d)	a polypeptide in which in respect to a) one or more amino acids are added to the sequence,<br>
e)	a fusion polypeptide comprising an amino acid sequence comprising all or a part<br>
of the amino acids of sequence SEQ ID NO. 6. Preferably, not more than 10 amino acids are replaced according to c).<br>
The present invenUon relates to the use of PIM-3 as a screening agent for identifying anti-diabetes mellitus drugs, e.g. ttie~use^of PIM-3 encoding^DNA or a polypeptide having PtM-3 activity for such purpose.<br>
The present invention further relates to the use of PIM-3 for preparing a medicament for the treatment of insulin resistance or type 2 diabetes mellitus and the use of PIM-3 for predicting insulin resistance or type 2 diabetes mellitus.<br>
PiM-3 protein which interacts with other cellular proteins can thus be used as a target for developing therapeutic molecules for modulating PIM-3 protein in ceils expressing PIM-3 protein or cells involved in the PIM-3 pathway, e.g., adipocytes. Nucleic add molecules of the invention can be used to express PlM-3 protein (e.g. via a<br><br>
recombinant expression vector in a host,cell in gene therapy applications), to detect PIM-3 mRNA (e.g. in a biological sample) or a genetic lesion in an PIM-3 gene, and to modulate PIM-3 activity.<br>
PIM-3 proteins can be used to screen drugs or compounds which modulate the PIM-3 activity or expression as well as to treat disorders characterized by insufficient or excessive production of PIM-3 protein or production of PiM-3 protein forms which have decreased or aberrant activity compared to PIM-3 wild type protein.<br>
The invention provides rriethods (also referred to herein as a "screening assay") for identifying modulators, i.e. candidate or test compounds or agents (e.g. peptides, peptidomimetics, small molecules or other drugs) which bind to PIM-3 proteins or have a stimulatory or inhibitory effect on, for example, PIM-3 expression or PIM-3 activity.<br>
In one embodiment, the invention provides assays for screening candidate or test compounds which bind to or modulate the activity of a PIM-3 protein or polypeptide or biologically active portion thereof. The test compounds of the present invention can be obtained using any of the numerous approaches in combinatorial library methods t;nowh in the art, including: biologicai libraries; spatially addressable parallel solid phase or solution phase libraries; synthetic library methods requiring deconvolution; the "one-bead one-compound" library method; and synthetic library methods using affinity chromatography selection.<br>
In an embodiment, an assay of the present invention is a cell-free assay comprising contacting a PlM-3 protein or biologically active portion thereof with a test compound and determining the ability of the test compound to bind to the PIM-3 protein or biologically active portion thereof. Binding of the test compound to the PIM-3 protein can be determined either directly or indirectly.<br>
In an other embodiment, the assay includes contacting the PIM-3 protein or biologically active portion thereof with a known compound which binds PlM-3 to form an assay mixture, contacting the assay mixture with a test compound, and determining the ability of the test compound to interact with' a PIM-3 protein, wherein determining the ability of<br><br>
the test compound to interact with a PIM-3 protein comprises determining the ability of the test compound to preferentially bind to PiM-3 or biologically active portion thereof as compared to the known compound.<br>
In another embodiment, an assay is a cell-free assay comprising contacting PlM-3 protein or biologically active portion thereof with a test compound and determining the ability of the test compound to modulate (e.g., stimulate or inhibit) the activity of the PIM-3 protein or biologically active portion thereof. Determining the ability of the test compound to modulate the activity of PIM-3 can be accomplished, for example, by determining the ability of the PIM-3 protein to bind to a PIM-3 target molecule, this means determining direct binding. In an alternative embodiment, determining the ability of the test compound to modulate the activity of PIM-3 can be accomplished by determining the ability of the PIM'3 protein to further modulate an PIM-3 target molecule. For example, the catalytic/enzymatic activity of the target molecule on an appropriate substrate can be determined.<br>
In another embodiment, the cell-free assay comprises contacting the PIM-3 protein or biologically active portion thereof with a known compound which binds PIM-3 to form an assay mixture, contacting the assay mixture with a test compound, and determining the ability of the test compound to interact with the PIM-3 protein, wherein determining the ability of the test compound to interact with the PIM-3 protein comprises determining the ability of the PIM-3 protein to preferentially bind to or modulate the activity of a PIM-3 target molecule.<br>
Phosphoamlnoacid analysis of the phosphorylated substrate can also be performed in order to determine which residues on the P(M-3 substrate are phosphorylated. Briefly, the radio phosphorylated protein band can be excised from the SDS gel and subjected to partial acid hydrolysis. The products can then be separated by one-dimensional electrophoresis and analyzed on, for exampie, a phosphoimager and compared to ninhydrin-stained phosphoamlnoacid standards.<br><br>
In another embodiment of the invention, the cell free assay determines the ability of the PiM-3 protein to phosphorylate an PIM-3 target molecule by, for example, an in vitro kinase assay. Briefly, a PIM-3 target molecule, e.g., an immunoprecjpitated PIM-3 target molecule from a ceil line expressing such a molecule, can be incubated with the PIM-3 protein and radioactive ATP,<br>
In another embodiment, an assay is a cell-based assay in which a cell which expresses a soluble form of PIM-3 protein, or a biologically active portion thereof, is contacted with a test compound and the ability of the test compound to bind to a PlM-3 protein determined. The cell, for example, can be a yeast ceil or a cell of mammalian origin. Determining the ability of the test compound to bind to the PlM-3 protein can be accomplished, for example, by coupling the test compound with a radioisotope or enzymatic label such that binding of the test compound to the PIM-3 protein or biologically active portion thereof can be determined by detecting the labeled compound in a complex, e. g. with .sup.125 I, .sup.35 S, .sup.14 C, or .sup.S H, or enzymaticaliy with, for example, horseradish peroxidase, alkaline phosphatase, or luciferase.<br>
In another embodiment, an assay is a cell-based assay comphsing contacting a ceil expressing a soluble form of PIM-3 protein, or a biologically active portion thereof, with a test compound and determining the ability of the test compound to modulate (e.g., stimulate or inhibit) the activity of the PlM-3 protein or biologically active portion thereof. Determining the ability of the test compound to modulate the activity of PIM-3 or a biologically active portion thereof can be accomplished, for example, by determining the ability of the PIM-3 protein to bind to or interact with an PlM-3 target molecule.<br>
As used herein, a "target molecule" Is a molecule with which an PIM-3 protein binds or interacts in nature, for example, a substrate molecule phosphoryjated by PIM-3 protein in the interior of a cell which expresses a PIM-3 protein, the intracellular domains of transmembrane receptors, a molecule associated with the internal surface of a cell<br><br>
membrane or a cytoplasmic molecule. A PlM-3 target molecule can be a non-PItvl-3 molecule or a PIM-3 protein or polypeptide of the present invention. In one embodiment, a PIM-3 target molecule is a component of a signal transduction pathway which mediates transduction of a signal.<br>
In an embodiment, determining the ability of the PIM-3 protein to bind to or interact with a PlM-3 target molecule can be accomplished by determining the activity of the target molecule. For example, the activity of the target molecule can be determined by detecting induction of a cellular second messenger of the target (e.g., intracellular Ca.sup.2+, diacylglycerol, 1P3, etc.), detecting catalytic/enzymatic activity of the target on an appropriate substrate, detecting the induction of a reporter gene (e.g., a PIM-3-responsive regulatory element operably linked to a nucleic acid encoding a detectable marker, e.g. luciferase), or detecting a cellular response, for example, cellular differentiation, or cell proliferation.<br>
In various formats of the assay methods of the present invention, it may be desirable to immobilize either PtM-3 or its target molecule to facilitate separation of compiexed from uncomplexed forms of one or both of the proteins, as well as to accommodate automation of the assay. Binding of a test compound to PlM-3, or interaction of PlM-3 A/ith a target molecule in the presence and absence of a candidate compound, can be accomplished in any vessel suitable for containing the reactants. Examples of such (■essels include microtitre plates, test tubes, and micro-centrifuge tubes. In one embodiment, a fusion protein can be provided which adds a domain that allows one or loth of the proteins to be bound to a matrix. For example, glutathione-S-ransferase/PlM-3 fusion proteins or gIutathione-S-transferase/target fusion proteins ;an be adsorbed onto glutathione sepharose beads or glutathione derivatized nicrotitre plates, which are then combined with the test compound or the test ompound and either the non-adsorbed target protein or PlM-3 protein, and the lixture incubated under conditions conducive to complex formation (e.g., at hysiological conditions for salt and pH}. Following Incubation, the beads or microtitre late wells are washed to remove any unbound components and complex formation is leasured either directly or indirectly;- Alternatively, the complexes can be dissociated<br><br>
Toni the matrix, and the level of PlM-3 binding or activity determined using standard ;ecHniques.<br>
3ther techniques for immobilizing proteins on mathces can also be used in the jcreening assays of the invention. For example, either PIM-3 or its target molecule can )e immobilized utilizing conjugation of biotin and streptavidin. Biotinylated p[M-3 or arget molecules can be prepared from biotin-NHS (N-hydroxy-succinimide) using echniques well known in the art. Alternatively, antibodies reactive with Plfvl-3 or target nolecules but which do not interfere with binding of the PliV1-3 protein to its target nolecule can be derivalized to the wells of the plate, and unbound target or PlM-3 ■apped in the wells by antibody conjugation. Methods for detecting such complexes, in ddition to those described above for the GST-immobilized complexes, include nmunodetection of complexes using antibodies reactive with the PIM-3 or target lolecuie, as well as enzyme-linked assays which rely on detecting an enzymatic ctivity associated with the PIM-3 or target molecule.<br>
i another embodiment, modulators of PIM-3 expression are identified in a lethod in which a cell is contacted with a candidate compound and the expression ' PIM-3 mRNA or protein in the cell is determined. The level of expression of lM-3 mRNA or protein in the presence of the candidate compound is compared to e level of expression of PIM-3 mRNA or protein in the absence of the candidate impound. The candidate compound can then be identified as a modulator of PIM-3 cpression based on this comparison. For example, when expression of PIW-3 mRNA ■ protein is greater (statistically significantly greater) in the presence of the candidate )mpound than in its absence, the candidate compound is identified as a stimulator of<br>
M-3 mRNA or protein expression. Alternatively, when expression of PIM-3 mRNA ot otein is less (statistically significantly less) in the presence of the candidate impound than in its absence, the candidate connpound is identified as an inhibitor of<br>
M-3 mRNA or protein expression. The level of PIM-3 mRNA or protein expression in<br>
3 cells can be determined by methods described herein for detecting PIM-3 mRNA or<br>
otein.<br><br>
In another aspect of the invention, the PIM-3 proteins or polypeptides thereof can be used as "bait proteins" in a two-hybrid assay or three hybrid assay, to identify other proteins, which bind to or interact with PlM-3 ("PlM-3-binding proteins" or "PIM-3-bp") and modulate PlM-3 activity. Such PlM-3-binding proteins are also likely to be involved in the propagation of signals by the PIM-3 proteins as, for example, upstream or downstream elements of the PlM-3 pathway. The invention also provides for the use of proteins that interact with PliVl-3, e.g., two-hybrid interactors with PlM-3, as baits in two-hybrid screens and the identification of PIM-3 interacting protein interacting proteins. PIM-3 interacting protein interacting proteins are likely to be involved in the PIM-3 signal transduction pathway.<br>
The present invention also provides the field of predictive medicine in which diagnostic assays, prognostic assays, pharmacogenomics, and monitoring clinical trials are used for prognostic (predictive) purposes to thereby treat an individual prophylactically. Accordingly, one aspect of the present invention relates to diagnostic assays for determining PIM-3 protein and/or nucleic acid expression as well as PIM-3 activity, in the context of a biological sample (e.g., blood, serum, cells, tissue form the ndividual, preferably human) to thereby determine whether an individual is affiicted mth a disease or disorder, or is at risk of developing a disorder, associated with aberrant PIM-3 expression or activity. The invention also provides for Drognostic (or predictive) assays for deternnining whether an individual is at risk of developing a disorder associated with PlM-3 protein, nucleic acid expression or activity. For example, mutations in a PIM-3 gene can be assayed in a biological &gt;ample. Such assays can be used for prognostic or predictive purpose to thereby )rophylactically treat an individual prior to the onset of a disorder characterized by or issociated with PIM-3 protein, nucleic acid expression or activity.<br>
Vnother aspect of the invention provides methods for determining PIM-3 protein, lucleic acid expression or PlM-3 activity in an individuals bioiogical sample to thereby ielect appropriate therapeutic or prophylactic agents for that individual (referred to lerein as "pharmacogenomics"). Pharmacogenomics allows for the selection of agents<br><br>
(e.g., drugs) for therapeutic or prophylactic treatment of an individual based on the<br>
genotype of the individual (e.g., the genotype of the individual<br>
examined to determine the ability of the individual to respond to a particular<br>
agent.)<br>
Another aspect of the invention provides monitoring the influence of agents (e.g., dru or other compounds) on the expression or activity of PIM-S in clinical trials.<br>
An exemplary method for detecting the presence or absence of PIM-3 in a biological sample involves obtaining a biological sample from a test subject and contacting the biological sample with a compound or an agent capable of delecting PlM-3 protein or nucleic acid (e.g., mRNA, genomic DNA) that encodes PIIVl-3 protein such that the presence of PlM-3 is detected in the biological sample. An agent for detecting PlM-3 mRNA or genomic DNA can be a labeled nucleic acid probe capable of hybridizing to PIM-3 mRNA or genomic DNA.<br>
\n agent for detecting PIM-3 protein can be an antibody capable of binding to ='lM-3'protein, preferably an antibody with a detectable label. Antibodies can le polyclonal, or more preferably, monoclonal. An intact antibody, or a fragment lereof (e.g., Fab or F(ab").sub.2) can be used.<br>
he term "biological sample" is intended to include tissues, cells and biological fluids lolated from an individual, as well as tissues, cells and fluids present within an idividual. That is, the detection method of the invention can be used to detect PIM-3 \RH/\, protein, or genomic DMA in a biological sample e. g. in vitro as well as or in vo. In one embodiment, the biological sample contains protein molecules from the ■St subject. Alternatively, the biological sample can contain mRNA molecules from the st subject or genomic DNA molecules from the test subject. A biological sample is g. a biopsy from adipose tissue isolated by conventional means from a subject.<br>
another embodiment, the methods further involve obtaining a control ^logical sample from a control subject, contacting the control sample with a<br><br>
compound or agent capable of detecting PIM-3 protein. mRNA, or genomic DNA, such that the presence of PIIW-3 protein, mRNA or genomic DNA is detected in the biological sample, and comparing the presence of PlM-3 protein, mRNA or genomic DNA in the control sample v;ith the presence of PIM-3 protein, mRNA or genomic DNA in the test sample.<br>
The invention also encompasses kits for detecting the presence of Pl!v1-3 in a biological sample (a test sample). Such kits can be used to determine \f an Individual is suffering from or is at increased nsk of developing a disorder associated with insulin resistance or type 2 diabetes. For example, the kit can comprise a labeled compound or agent capable of detecting PIM-3 protein or mRNA in a biological sample and means for determining the amount of PIM-3 in the sample (e.g., an anti-PlM-3 antibody or an oligonucleotide probe which binds to DNA encoding PIM-3, e.g., SEQ ID N0:1, SEQ ID N0:3, SEQ ID NO. 5 or SEQ iD NO. 7).<br>
The methods described herein can furthermore be utilized as diagnostic or prognostic assays to identify subjects having or at risk of developing a disease or disorder associated with aberrant PIM-3 expression or activity. For example, the assays described herein, such as the preceding diagnostic assays or the foliowing assays, can be utilized to identify a subject having or at risk of developing insulin resistance or type 2 diabetes, Thus, the present invention provides a method in which PIM-3 protein or nucleic acid (e.g., mRNA, genomic DNA) is detected in a test sample from an ndividuai, wherein the presence of PiM-3 protein ov nucleic acid is diagnostic for the ndividual for having or at risk of developing a disease or disorder associated with aberrant PlM-3 expression or activity. As used herein, a "test sample" refers to a biological sample obtained from a subject of interest. For example, a test sample can :-e a biological fluid (e.g., serum), cell sample, or tissue.<br>
furthermore, the prognostic assays can be used to determine whether a subject can )e administered an agent (e.g., an agonist, antagonist, peptldomimetic, protein, leptide, nucleic acid, small molecule, or other drug candidate) to treat insulin esistance or type 2 diabetes. For example, such methods can be used to determine<br><br>
whether a subject can be effectively treated with a specific agent or class of agents (e.g., agents of a type which decrease PIM-3 activity).<br>
Agents, or modulators which have a stimulatory or inhibitory effect on PlM-3 activity (e.g., PlM-3 gene expression) as identified by a screening assay can be used for preparing a pharmaceutical which is useful for treating (prophylactically or therapeutically) disorders (e.g., disorders involving cells or tissues in which PlM-3 is expressed, such as adipocytes) associated with aberrant P(M-3 activity. In conjunction with such treatment, the pharmacogenomics (i.e. the study of the relationship between an individual's genotype and that individual's response to a foreign compound or drug) of the individual may be considered. Differences in metabolism of therapeutics can lead to severe toxicity or therapeutic failure by altering the relation between dose and blood concentration of the pharmacologically active drug. Thus, the pharmacogenomics of the individual permits the selection of effective agents (e.g., drugs) for prophylactic or therapeutic treatments based on a consideration of the individual's genotype. Such pharmacogenomics can further be jsed to determine appropriate dosages and therapeutic regimens. Accordingly, the activity of PlM-3 protein, expression of P!M'3 nucleic acid, or mutation content of PIM-3 genes In an individual can be determined to thereby select appropriate agent(s) for :herapeutic or prophylactic treatment of the individual.<br>
\/lonitoring the influence of agents (e.g., drugs, compounds) on the expression or activity of PIM-3 (e.g., the ability to modulate aberrant cell proliferation and/or Jifferentiation) can be applied not only in basic drug screening, but also in clinical rials.<br>
\n anti type 2 diabetes agent that modulates PIM-3 protein activity can be an agent, such as a small molecule, e.g., a small molecule that modulates the protein kinase activity of PIM-3, a nucleic acid or a protein, a naturally-occurring cognate ligand of a ^IM-3 protein, a peptide, or a PIM-3 peptidomimetic. Sn one embodiment, the agent jtimulates one or more of the biological activities of PlM-3 protein, Examples of such itimulatory agents include small molecules that stimulate one or more activities of PIM-<br><br>
3, e.g., the PIM-3 protein kinase activity, active PIIVl-3 protein and a nucleic acid molecule encoding PIM-3 that has been introduced into the cell. In another embodiment, the agent inhibits one or more of the biological activities of PlM-3 protein. Examples of such inhibitory agents include a small molecule that inhibits one or more PllVl-3 activities e.g., PiM-3 protein kinase activity, antisense PIM-3 nucleic acid Tiolecules and anti-PIM-3 antibodies.<br>
This invention is further illustrated by the following examples which should not be ;onstrued as limiting. The contents of all references, patents and published patent ipplications cited throughout this application are hereby incorporated by reference.<br>
ixamples<br>
■xample 1<br>
etermlnation of the Nucleotide Sequence of human and murine PIM-3<br>
at PlM-3 nucleotide sequence (AF057026, NM_022602, SEQUENCE ID NO-.9) was sed to query a proprietary database using the BLASTN program with the BLOSUM62 atrix. This proprietary database is based on a proprietary cDNA library which Is instructed in standard cloning vectors. The most closely related cDNA clones entitled by this BLASTN were sequenced. The cDNA sequences were assembled to a contig. The human PIM-3 sequence was determined from the consensus iquence of this contig, further analysis of this contig revealed a cDNA sequence 1977 ) in length. That human PlM-3 cDNA contains a 978 base pair open reading frame edicted to encode a novel 326 amino acid protein.<br>
3t PlM-3 nucleotide sequence was also used to query the public UNIGENE Mouse itabase using the BLASTN program with the BLOSUM62 matrix. Some closely iated EST sequences were identified by this BLASTN. Sequence information of this iTs was used to screen a mouse embryo cDNA library constructed using the Gene apper II Technology (Life Technologies, Karlsruhe, Germany). A cDNA clone was<br><br>
sequenced, and the cDNA sequences were assembled into a contig. The murine PIM-3 sequence was determined from the consensus sequence of this contig, further analysis of this contig and exclusion of an intron sequence revealed a cDNA sequence 2236 bp in length. That murine PIM-3 cDNA contains a 978 base pair open reading frame predicted to encode a novel 326 amino acid protein.<br>
Example 2<br>
Characterization of the human and the murine PIM-3 protein<br>
n this example, the predicted amino acid sequence of human PIM-3 and the murine ^IM-3 protein was compared to amino acid sequences of known motifs and/or lomains present in proteins and to the polypeptide sequences of known proteins. Polypeptide domains and/or motifs present in human P!M-3 and murine PIM-3 were Jentified as were proteins with significant amino acid similarities to human PIM-3 and lurine PIIVl-3. In addition, the molecular weight of the human PIM-3 and the murine 'IM-3 protein was predicted.<br>
"he human and the murine nucleotide sequences (SEQ ID N0.1; SEQ ID NO.5), ientified as described above, encode a 326 amino acid protein (SEQ ID N0.2 and EQ ID N0.6). Human and murine PIM-3 has a predicted MW of about 35.9 kDa, sspectively, not including post-translatlonal modifications. To check for evidences for putative kinase activity and for possible posttranslational modification sites, the uman and the murine polypeptide sequences of SEQ ID N0.2 and SEQ ID N0.6, ispectively, were analyzed using the PROSITE database of protein patterns, as well s using IMPALA and PFAM.<br>
earching the PROSiTE database revealed the presence of one cAMP and cGMP spendent protein kinase phosphorylation site from amino acids 260-263 of SEQ ID 0.2 and SEQ ID N0.6; three Casein kinase II phosphorylation sites from amino acids 32-205, 211 -214 and 321-324 of SEQ ID N0.2 and four Casein kinase II losphorylation sites from amino acids 202-205, 211-214 299-302 and 321-324 of EQ ID N0.6; ten N-myristoylation sites from amino acids 43-48, 49-54, 52-57, 57-62,<br><br>
63-68, 80-85, 98-103, 101-106, 295-300 and 316-321 of SEQ ID N0.2 and SEQ ID N0.6; three Protein kinase C phosphorylation sites from amino acids 137-139, 275-277 and 279-281 of SEQ ID N0.2 and SEQ ID N0.6; one tyrosine kinase phosphorylation site from amino acids 33-40 of SEQ ID N0.2 and.SEQ ID N0.6; one protein kinases signature and profile (ATP binding site) from amino acids 46-69 of SEQ ID N0.2 and SEQ ID NO.6; one serine/threonine protein kinase active site signature from amino acids 166-178 of SEQ ID N0.2 and SEQ ID NQ,6. The search using IMPALA revealed the presence of one eukaryotic protein kinase domain from amino acid 40-293 (SEQ ID N0.4 and SEQ ID NQ.8, respectively), of SEQ ID N0.2 and SEQ ID N0.6; with a score of 186 bits and Expect value of 8e-49 and with a score of 184 bits and Expect value of 4e^8, respectively. The search using 'PFAM revealed also the presence of one eukaryotic protein kinase ilomain from amino acid 40-293, of SEQ ID N0.2 and SEQ ID NO.6; with a score of 262,5 and E-value of 5.7e-75 and with a score of 261,1 bits and E-value of 1.5e-74, ■espectively.<br>
The human, murine and rat PIM-3 polypeptide sequences (SEQ ID NO.2, SEQ ID ^0.6 and SEQ ID NO.10) were aligned in a pair wise Clustal W alignment analysis jsing blosum as the protein weight matrix. Thereby, human PIM-3 was found to be 95 4 identical to murine and rat PIM-3 (AF057026, NM_022602, SEQ ID NO.10)w;th a icore of 2011, murine PIM-3 was found to be 99 % identical to rat PIM-3 (AF057026, JM_022602, SEQ ID NO.10)with a score of 2074.<br>
"he human and the murine PIM-3 polypeptide sequences of SEQ ID N0.2 -and of SEQ D N0.6 was also used to query the Swissprot database of protein sequences using ne BLASTP program with the BLOSUM62 matrix. The four most closely related roteins to human and munne PII\/I-3 identified by this BLASTP analysis are listed: luman and muhne PIM-3 was found to be 76% identical to Xenopus laevis (frog) PIM-(Q91822; SEQ ID N0.11) with a score of 518, 72% identical to rat PIM-1 {P26794; ■EQ ID N0.12) with a score of442, 71% identical to human PlM-1 {P11309;SEQID 10.13) with a score of 441 and 71% identical to murine PIM-I (P06303; SEQ ID :0.14) with a score of 436 and 438, respectively.<br><br>
Example 3;<br>
Gene Expression of PIM-3 in an in vivo model of insulin resistance and type 2 diabetes<br>
meliitus<br>
Zucker diabetic fatty (2DF) rats are well known animal model carrying a homozygous defect in the leptin receptor fa gene. This rat strain develops age dependent an insulin resistant / hyperinsulinaemic state which than progresses to overt type 2 diabetes meilitus / hyperglycaemic state. To identify genes the expression of which is either induced or repressed and so may contribute or mark the development of insulin resistance or type 2 diabetes meilitus, gene expression profiies of ZDF rats and their lean heterozygous control littermates were collected using oligonucleotide array based profiling technique.<br>
Material and methods:<br>
Zucker Diabetic Fatty (ZDF/Gmi.TM.-fa/fa} male rats as well as their lean male fa+/fa-counterparts, used as healthy controls, aged 6,8 and 13 weeks were obtained from Genetic Models Inc. (Indianapolis, IND., US).<br>
Before the animals were included Into the study they were kept under standard animal louse conditions for one week. For collection of epidydimal fat tissue and blood samples the animals were killed by cervical dislocation. 6 ZDF rats and 6 lean fa+/fa-ittermates per age group were used for gene expression analysis as described below.<br>
Tissue collection and RNA isolation;<br>
-ollowing cervical dislocation, epididymal fat pads were surgically removed, portioned<br>
md quickly transferred in suitable tubes containing sufficient volumes of RNA later<br>
Ambion, TX, US). Samples from each animal were stored individually. Long term<br>
itorage of the samples was performed at minus SO.degree.C.<br>
'otal cellular RNA was extracted from adipose tissue using Rneasy Mini kit (Qiagen,<br>
-lilden, Germany) according to the manufacturers recommendations for RNA isolation<br>
rom fat tissue. RNA was eluted twice in SO.mul RNAse free water, RNA concentration<br>
vas determined spectroscopically {A.sub.260).<br><br>
For further purification, RNA solution was filled up to 100^1 total volume with RNase free water. RNA clean up was performed according to the manufacturers instructions using Qiagen Rneasy Mini kit. RNA was eluted twice in 50 u.\ RNase free water, RNA concentration was determined spectroscopically (A.sub.260).<br>
For concentration of the RNA eluate, NH.sub.4 acetate and ethanol were added, and RNA was precioitated overnight in an ethanol-dry ice bath. RNA v;as collected by centnfugation at maximum speed at 4.degree.C. Pelleted RNA was washed Ivjice with 80 % ethanol, air dried end dissolved in a small volume of RNAse free water. Rnase concentration was determined spectrascopically (A.sub.26Q).<br>
Gene expression profiling:<br>
The general use of oligonucleotides arrays for gene expression monitoring has been described in US 6,177,248. In our practical application, the used microarrays contain iesoxynucieotide sequences that represent approximately 8000 known genes or EST :iusters. Each gene or EST sequence is represented by up to 20 pairs of oligonucleotides, each pair consisting of one oligo that matches to a segment of the ranscript, and a control oligo that contains a centrally located 1 bp mismatch. For rat, J arrays {RG U34A, RG U34B and RG U34C) representing approximately 24000 gene ind EST sequences in total, derived from a database of known genes or EST iequences are provided by Affymetrix, Santa Clara, CA, US.<br>
:RNA preparation for hybridization:<br>
iNA was obtained from epididymal fat as decribed above. An oligo dT primer ontaining a T7 promoter site was added to total cellular RNA {10.mu.g)+-. After nnealing of the primer, the RNA was subsequently reverse transcribed using Superscript choice reverse transcriptase, following the manufacturers instructions. ,fter extraction with phenol:chloroform:isoamyialcohol, using phase lock gel tubes zppendorf, Hamburg, Germany), and ethanol precipitation, the cDNA was collected y centrifugation and washed twice with 80 % ethanol. The pellet was dissolved in '.nase free water and transcribed into biotinylated cRNA using the Enzo High Yield ibelling transcription kit (Enzo Diagnostics, Farmingdale NY, US) or MEGAscript T7 igh yield transcription kit (Amblon, Austin, TX, US) according the manufacturers<br><br>
instructions. For the latter application, biotin labelled DTP and CTP (Sigma, Munich, Germany) plus unlabeled ATP and GTP was used in a molar ratio of 1:3 (labelled vs unlabelled). cRNA was then precipitated and washed as described above. Finally, the precipitated, air dried cRNA was dissolved in a small volume of RNAse free water. RNA concentration was determined speclroscopicatly (A.sub260), and size distribution was checked by agarose geleiectrophoresis. Subsequently, cRNA was hydrolized to an average size of 50 nucleotides in length by incubating for 25 minutes in 40 mM Tris-acetate pH 8.1, 100 mM potassium acetate, 30 mM magnesium acetate at 94.degree,C. 20 jig of the fragmented cRNA was used to set up of a hybhdisation cocktail according to the instructions of Affymetrix. Prior hybridization, RNA samples were healed in the hybridization cocktails to 99.degree.C for 10 min, placed on ice for 5 min, and allowed to equilibrate to room temperature before being placed in the nybridization fiow cell. The hybridsation cocktail was then hybridised to a microarray at l-S'C and 60 rpm in a hybridisation oven overnight. After hybridization, the Hybridisation cocktail was removed and stored at minus SOX for further use. The arrays were washed and stained in a Affymethx fluidics station using the phycoerythhn steptavidin -antibody amplification protocol EukWS2 according to the manufacturers nstructions. Data were collected using a scanning confocai microscope made for Affymetrix by Hewlett Packard (Commercially available through Affymethx, Santa :iara, CA, US.). The scanner uses an argon ion laser as the excitation source, with the imission detected by a photomultiplier tube through either a 530 nm bandpass filter fluorescein) or a 560 nm longpass filter (phycoerythrin).<br>
Juantitative analysis of hybridization patterns and intensities: ollowing a quantitative scan of an array, a grid is aligned to the image using the nown dimensions of the array and the corner control regions as markers. The image , reduced to a simple text file containing position and intensity information using oftware developed at Affymetrix {available with the confocai scanner). This iformation is merged with another text file that contains information relating physical osition on the array to probe sequence and the identity of the RNA (and the specific art of the RNA) for which the oligonucleotide probe is designed. The quantitative lalysis of the hybridization results involves a simple form of pattern recognition based<br><br>
£•'-•<br><br>
on the assumption that, in the presence of a specific RNA, the PM (PM means "perfect match" in US 6,177,248) probes will hybridize more strongly on average than their MM (MM means "mismatch" in US 6,177,248) partners. The number of instances in which the PM hybridization signal is larger than the MM signal is computed along with average of the logarithm of the PM/MM ratios for each probe set. These values are used to make a decision (using a predefined decision matrix) concerning tiie presence or absence of an RNA. To determine the quantitative RNA abundance, the average of the differences (PM minus MM) for each probe family is calculated. The advantage of the difference method is that signals from random cross-hybridization contribute equally, on average, to the PM and MM probes, while specific hybridization contributes more to the PM probes. By averaging the painwise differences, the real signals add constructively while the contributions forn^ cross-hybridization tend to cancel. When assessing the differences between two different RNA samples, the hybridization signals from side-by-side experiments on identically synthesized arrays are compared directly. The magnitude of the changes in the average of the difference (PM-MM) values is interpreted by comparison with the results of spiking expehments as well as the signals observed for the internal standard bacterial and phage RNAs spiked into each sample at a known amount. Data analysis programs developed at Affymetrix perform these operations automatically.<br>
Table 1<br><br>
		ZDF  rats,    7   weeks   old	<br>
Cotnpari	ZDF   #   1	ZDF  ff	2DF   #   3	ZDF   #   4	ZDF   #   5<br>
son	vs	2  vs	vs	V3	vs<br>
	co--:trol	conCro	control	control	control<br>
	#1	1 #2	^3	#4	#5<br>
Fold	-   4.3	-4.8	-3.1	-5.3	-4.6<br>
change					<br><br><br>
) Adipocytes were isolated from epidydimal fat pads of 160-180g male Sprague Dawley rats and incubated as described (Mulier, G.and Wied, S. Diabetes (1993) 42:1852-1867). Briefiy, adipocytes were isolated from the pooled epidydymal fat of 20 male rats. The isolated adipocytes were split into two pools. One pool was made insulin resistant by incubating the cells for 5 hours in medium containing 25mM D-glucose plus lOnM insulin.<br><br>
The second pool was incubated for 5h in medium containing 5mM D-glucose, which kept the cells in the insulin sensitive state. At the end of the incubation insufin sensitivity and insulin resistance where checked by measuring insulin dependent glucose uptake with an aliquot of the two pools as described (Muller, G. and Wied, S. Diabetes (1993) 42: 1852-1S67). The majority of the adipocytes was used for isolating RNA as described in example 3.<br>
Subsequently RNA was used for gene expression monitoring using Affymetrix technology as described in example 3. Fold changes of gene expression for PIM-3 were analyzed using the Affimethx qualifier AF0S6624_S_AT. The results of these analysis are summarized in table 4:<br>
Table 4<br><br>
	In vitro insulin resistant adipocytes, 5 h    '<br>
	incubated<br>
Compariso	Resistant 1Resis-ant	Resistant	Resistant	Resistant<br>
n	1 vs	2 vs	3 vs	4 vs	S vs<br>
	control 1	concrol 2	control 3	control 4	control 5<br>
Fold	- 2.4	- 2.7	- 4.0	- 2.1	-   2.4<br>
change					<br>
zxperiments 5 : Gene expression of PIM-3 in 3T3-L1 adipocytes treated with antidiabetic dnjgs {PPARy agonists)<br>
i/laterial and Methods:<br>
iT3-L1 preadipocytes were grown at 37°C in 10% CO2 in Dulbecco's modified Eagle's nedium (DMEM) containing 1 g/l glucose and 10% fetal calf serum (FCS). For lifferentiationjnto mature adipocytes, confluent preadipocytes were cultured for four lays in DMEM supplemented with 4.5 g/l glucose, 10% FCS, 50 pg/mi ascorbic acid, 1 iM biotin, 17 pM pantothenic acid (= basal medium), 500 pM 3-isobutylmethylxanthine,<br><br>
0.25 |JM dexamethasone and 1 [sgfml human recombinant insulin. During the four day treatment the medium was changed once. Finally, 373-11 cells were treated for three days with basal medium containing 1 pg/ml insulin whereupon approximately 90% of the cells were converted into adipocytes.<br>
Differentiated 3T3-L1 cells were maintained in basal medium for one additional day and subsequently kept in serum-free basal medium for four hours. Then, PPARy agonists diluted in basal medium were added to the adipocytes as described below (concentrations and conditions see table 5);<br>
Table 5<br><br>
probe	compound*	concentration	duration of<br>
no.				treatment<br>
1	rosiglitazcne	1 fiM		6 hours<br>
2	rosiglitazone	5 ^M		6 hours<br>
3	trogiitazone	1 liH		6 hours<br>
4	trcglitazone	5 /iM		6 hours<br>
5	DMSO control	0 .02%	(v/v)	6 hours<br>
6	rosiglitasone	i /iM		24	hours<br>
7	rosiglitasone	5 /iM		24	hours<br>
8	trogiitazone	1 /iM		24	hours<br>
9	trogiitazone	5 fiM		24	hours<br>
10	DMSO control.	0.02%	(v/v)	24	hours<br>
11	rosiglitazone	1 ^M		48	hours<br>
12	rosiglitazone	5 p.Vi		43	hours<br>
13	trcglitazone	1 ^M		■4B	hours<br>
14	trogiitazone	5 jiM		48	hours<br>
15	DMSO control	0.02%	(v/v)	48	hours<br>
* 5 mM and 25 mM stock solutions of rosiglitazone and trogiitazone dissolved in DMSO were made and diluted 5000fold to the final concentrations (1 pM / 5 pM) in the culture<br><br>
medium. For the controls, DMSO without compound was equally diluted SOOOfold to a final concentration of 0.02°D (V/V).<br>
RNA was isolated using the TRlzol reagent (Life Technologies, Karlsruhe, Germany)<br>
and treated with DNase 1 by applying the DNA-free kit (Ambion, Austin, TX, US).<br>
Further purification of the RNA was achieved using the RNeasy mint kit (Qiagen,<br>
Hilden, Germany) and quality/quantity control was done with the 2100Bioanalyzer<br>
(Agilent, Boblingen, Germany). 10 jjg of total RNA was converted into biotinylated<br>
DRNA according to the GeneChip expression analysis technical manual (Affymetrix,<br>
Banta Clara, CA, US). Briefly, first and second strand synthesis was performed by<br>
applying the Superscript double stranded cDNA synthesis kit (Life Technologies,<br><arlsruhe germany and biotin labeled crna was produced with the bioarray rna></arlsruhe>
ranscript labeling kit (Enzo Diagnostics, Framingdale, NY, US). 10 (jg of cRNA was<br>
ragmented by heat, added to the GeneChip eukaryotic hybridization control solution<br>
Affymetrix) and hybridized to a GeneChip MG-U74Av2 array (Affymetrix) by rotating<br>
3r 16 hours at 45°G. Washing, staining and scanning of the array was carried out with<br>
tandard  procedures using the hardware  provided  by Affymetrix.  Raw data  was<br>
nal&gt;^ed by applying the microarray suite version 4.0.1 software (Affymetrix, see<br>
bove).<br>
he entire experiment was repeated twice to provide three biological replicates.<br>
lata analysis;<br>
ata analysis including the estimation of fold changes was performed as described oove. Foldchange values for PlM-3 were obtained by comparing the compound aated samples against the untreated controls for each time point. Therefore, ffymetrix qualifier 96S41_AT was used.<br>
ie results are summarized in table 6 a) - c).<br><br><br><br>
"Rosi" stands for "rosiglitazone", "Tro" stands for "troglitazone".<br>
Example 6 : Gene expression of Pim-3 in ZDF rats treated with anti-diabetic drugs (PPARv agonists)<br>
Zucker Diabetic Fatty (ZDF/Gmi,TM,-fa/fa) male rats as well as their lean fa+/fa-counterparts, sged 5 weeks (control 1-5, ZDF 1-I0)were obtained from Genetic Models inc. (Indianapolis, ind.). The rats were given free access to food and water. The ZDF animals were splitted into 2 groups; group 1(animals ZDF 1-5} was not treated with any agent, whereas the second group was treated for 14 weeks with the antidiabetic drug Rosigiitazone (3 mg/kg/day, animals 6-10, Rosi 1-5,). For the collection of epididymal fat pads and also blood samples, in each case 5 animals were sacnticed by cervical dislocation. To monitor the success of the antidiabetic treatment, HbAlc as marker of long-term blood glucose levels were measured by standard methods after finishing of the experiment. The results of these measurements are summarized in table 7:<br><br>
Table 7:														<br>
	% HbAI c in different rats: controls rats (C), ZDF rats untreated (Z,), ZDF rats Rosiglitazone treated (R)<br>
animal	C 1	C2	C3	C4	C5	21	Z2	Z3	Z4	Z5	R1    j R2	R3	R4	R5<br>
HbAlc (%)	4.20	4.28	4.21	4.16	4.09	5.12	8.01	7.59	S.71	9.51	4.30	4.27	4.14	4.14	4.13<br>
Coliection of epididymal adipose tissue, RNA isolation and Affymethx expehments were performed as described in experiment 1. Data analysis included the comparison of expression data of samples derived from the lean control animals versus the untreated ZDF rats as well as the comparison of the expression data derived from Rosiglitazone treated ZDF animals versus non-treated ZDF animals. Data analysis including the estimation of fold changes and weighing of the statistical significance of those fold changes was performed using a proprietary software developed in Aventis<br><br>
pharmaceuticals. Fold changes of gene expression for Pim-3 were analyzed using the Affimetrix qualifier AF086624_S_AT. The results of those analysis are summarised in the following tables Sa und 8b:<br><br><br><br>
WE CLAIM:<br>
1.	An m vitro method of identifying anti-diabetic compounds such as PPARy modulators, wherein a polypeptide having PIM-3 activity is incubated in vitro with a compound to be tested, and change in PIM-3 activity is determined.<br>
2.	The in vitro method as claimed in claim 1, which is a cell-free method and the ability of the test compound to bind to the polypeptide having PIM-3 activity is determined.<br>
3. The in vitro method as claimed in claim 1, which comprises contacting the polypeptide having PIM-3 activity with a known compound which binds PIM-3 to form an assay mixture, contacting the assay mixture with a test compound and determining the ability of the test compound to interact with a PIM-3 protein, wherein determining the ability of the test compound to interact with a PIM-3 protein comprises determining the ability of the test compound to preferentially bind to PIM-3 or biologically active portion thereof as compared to the known compound.<br>
4.	The in vitro method as claimed in claim 1, which is a cell-free method and the<br>
ability of the test compound to stimulate or inhibit the activity of the<br>
polypeptide having PIM-3 activit&gt;' is determined.<br>
5.	The in vitro method as claimed in claim I, wherein a yeast cell or an isolated<br>
mammalian cell which expresses a soluble form of PIM-3 protein or a<br>
biologically active portion thereof, is contacted with a test compound and the<br>
ability of the test compound to bind to a PIM-3 protein is determined.<br><br>
6.	The in vitro method as claimed in claim 1, whereia a yeast cell or an isolated<br>
mammalian cell which expresses a soluble fomi of PIM-3 protein or a<br>
biologically active portion thereof, is contacted with a test compound and the<br>
ability of the test compound to stimulate or inhibit the activity of the PIM-3<br>
protein or biologically active portion thereof is determined.<br>
7.	The in vitro method as claimed in any of the preceding claims, wherein the<br>
polypeptide is encoded by a nucleic acid having the sequence according to SEQ<br>
IDNo.l<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1NS1jaGVucC0yMDA0IGFic3RyYWN0IGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">1555-chenp-2004 abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1NS1jaGVucC0yMDA0IGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1555-chenp-2004 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1NS1jaGVucC0yMDA0IGNsYWltcyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1555-chenp-2004 claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1NS1jaGVucC0yMDA0IGNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1555-chenp-2004 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1NS1jaGVucC0yMDA0IGNvcnJlc3BvbmRlbmNlIG90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1555-chenp-2004 correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1NS1jaGVucC0yMDA0IGNvcnJlc3BvbmRlbmNlIHBvLnBkZg==" target="_blank" style="word-wrap:break-word;">1555-chenp-2004 correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1NS1jaGVucC0yMDA0IGRlc2NyaXB0aW9uIChjb21wbGV0ZSkgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">1555-chenp-2004 description (complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1NS1jaGVucC0yMDA0IGRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">1555-chenp-2004 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1NS1jaGVucC0yMDA0IGZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">1555-chenp-2004 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1NS1jaGVucC0yMDA0IGZvcm0tMTMucGRm" target="_blank" style="word-wrap:break-word;">1555-chenp-2004 form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1NS1jaGVucC0yMDA0IGZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">1555-chenp-2004 form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1NS1jaGVucC0yMDA0IGZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">1555-chenp-2004 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1NS1jaGVucC0yMDA0IGZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">1555-chenp-2004 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1NS1jaGVucC0yMDA0IGZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">1555-chenp-2004 form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1NS1jaGVucC0yMDA0IG90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1555-chenp-2004 others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1NS1jaGVucC0yMDA0IHBjdCBzZWFyY2ggcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">1555-chenp-2004 pct search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1NS1jaGVucC0yMDA0IHBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1555-chenp-2004 pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1NS1jaGVucC0yMDA0IHBlaXRpdG9uLnBkZg==" target="_blank" style="word-wrap:break-word;">1555-chenp-2004 peititon.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="227976-indolinone-derivatives-useful-as-protein-kinase-inhibitors.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="227978-a-combination-of-pyrolidine-of-formula-ic-and-at-least-one-statin.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>227977</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1555/CHENP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>10/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>06-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>27-Jan-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>14-Jul-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>SANOFI-AVENTIS DEUTSCHLAND GmbH</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>BRUNINGSTRASSE 50, D-65929 FRANKFURT AM MAIN,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>KORN, MARCUS</td>
											<td>EUROPAPLATZ 1, D-63069 OFFENBACH,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SCHNEIDER, RUDOLF</td>
											<td>DISTELWEG 36, D-65527 NIEDERNHAUSEN,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>MUELLER, GUENTER</td>
											<td>IM HAINDELL 1, D-65843 SULZBACH,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>TSCHANK, GEORG</td>
											<td>ANDER KLINGELPFORTE 9A, D-55270 ESSENHEIM,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/54</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP03/00492</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-01-20</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>02001401.5</td>
									<td>2002-01-19</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/227977-an-in-vitro-method-of-identifying-anti-diabetic-compounds by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:30:44 GMT -->
</html>
