<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/228095-ordered-molecular-presentation-of-antigens-method-of-preparation-and-use by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:39:53 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 228095:ORDERED MOLECULAR PRESENTATION OF ANTIGENS, METHOD OF PREPARATION AND USE .</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">ORDERED MOLECULAR PRESENTATION OF ANTIGENS, METHOD OF PREPARATION AND USE .</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention provides compositions and processes for the production of ordered and repetitive antigen or antigenic determinant arrays. The compositions of the invention are useful for the production of vaccines for the prevention of infectious diseases, the treatment of allergies and the treatment of cancers. Various embodiments of the invention provide for a virus, virus-like particle, viral capsid particle, phage or recombinant form thereof coated with any desired antigen in a highly ordered and repetitive fashion as the result of specific interactions. In one specific embodiment, a versatile new technology based on a cassette-type system (Alpha Vaccine Technology) allows production of antigen coated viral particles. Other specific embodiments allow the production of antigen coated hepatitis B virus-like particles or antigen coated Measles virus-like particles.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Field of the Invention<br>
The present invention is related to the fields of molecular biology,<br>
virology, immunology and medicine. The invention provides a composition<br>
comprising in ordered and repetitive antigen or antigenic determinant array. The<br>
invention also provides a process for producing an antigen or antigenic<br>
determinant in an ordered and repetitive array. The ordered and repetitive antigen<br>
or antigenic determinant is useful in the production of vaccines for the treatment<br>
of infectious diseases, the treatment of allergies and as a pharmaccine to prevent<br>
or cure cancer and to generate defined self-specific antibodies.<br>
RelatedArt<br>
Vaccine development for the prevention of infectious disease has had the<br>
greatest impact on human health of any medical invention. It is estimated that<br>
three million deaths per year are prevented worldwide by vaccination (Hillemann,<br>
Nature Medicine 4:507 (1998)). The most common vaccination strategy, the use<br>
of attenuated (i.e. less virulent) pathogens or closely related organisms, was first<br>
demonstrated by Edward Jenner in 1796, who vaccinated against smallpox by the<br>
administration of a less dangerous cowpox virus. Although a number of live<br>
attenuated viruses (e. g. measles, mumps, rubella, varicella, adenovirus, polio,<br>
influenza) and bacteria (e. g. bacille Calmette-Guerin (BCG) against tuberculosis)<br>
are successfully administered for vaccination, there is a risk for the development<br>
of serious complications related to a reversion to virulence and infection by the<br>
'vaccine' organism, in particular in immunocompromised individuals.<br><br>
The specific design of attenuated viruses is now enabled by recombinant<br>
DNA technology (i.e., genetic engineering) through the generation of deletion or<br>
mutation variants. For example, the administration of an engineered Simian<br>
Immunodeficiency Virus (SIV) with a deletion within the nef'gene was shown to<br>
protect macaques from subsequent infection with a pathogenic SIV strain (Daniel<br>
et al, Science 255:1938-1941 (1992)). However, the progression of acquired<br>
immunodeficiency syndrome (AIDS)-like symptoms in animals administered<br>
attenuated SIV raises safety concerns (Baba et al., Science 267:1820-1825<br>
((1995)).<br>
As an alternative approach, attenuated viruses or bacteria may be used as<br>
carriers for the antigen-encoding genes of a pathogen that is considered too unsafe<br>
to be administered in an attenuated form (e.g., Human Immunodeficiency Virus<br>
(HIV)). Upon delivery of the antigen-encoding gene to the host, the antigen is<br>
synthesized in situ. Vaccinia and related avipox viruses have been used as such<br>
carriers for various genes in preclinical and clinical studies for a variety of<br>
diseases (e.g., Shen et al, Science 252:440 (1991)). One disadvantage of this<br>
vaccination strategy is that it does not mimic the virion surface, because the<br>
recombinant protein is expressed on the surface of the host cell. Additionally,<br>
complications may develop in immunocompromised individuals, as evidenced by<br>
life-threatening disseminated vaccinia infections (Redfield. N. Eng. J. Med.<br>
316:673 ((1998)).<br>
A fourth vaccination approach involves the use of isolated components of<br>
a pathogen, either purified from the pathogen grown in vitro (e.g. influenza<br>
hemagglutinin or neuraminidase) or after heterologous expression of a single viral<br>
protein (e.g. hepatitis B surface antigen). For example, recombinant, mutated<br>
toxins (detoxified) are used for vaccination against diphtheria, tetanus, cholera<br>
and pertussis toxins (Levine et al, New generation vaccines. 2nd edn., Marcel<br>
Dekker, Inc., New York 1997), and recombinant proteins of HI V (gp 120 and full-<br>
length gpl 60) were evaluated as a means to induce neutralizing antibodies against<br>
HIV with disappointing results (Connor et al, J. Virol. 72:1552 (1998)).<br><br>
Recently, promising results were obtained with soluble oligomeric gpl60, that<br>
can induce CTL response and elicit antibodies with neutralizing activity against<br>
HIV-1 isolates (Van Cortt et al, J. Virol. 71:4319(1997)). In addition, peptide<br>
vaccines may be used in which known B- or T-cell epitopes of an antigen are<br>
coupled to a carrier molecule designed to increase the immunogenicity of the<br>
epitope by stimulating T-cell help. However, one significant problem with this<br>
approach is that it provides a limited immune response to the protein as a whole.<br>
Moreover, vaccines have to be individually designed for different MHC<br>
haplotypes. The most serious concern for this type of vaccine is that protective<br>
antiviral antibodies recognize complex, three-dimensional structures that cannot<br>
be mimicked by peptides.<br>
A more novel vaccination strategy is the use of DNA vaccines (Donnelly<br>
et al, Ann. Rev. Immunol. 15:617 (1997)), which may generate MHC Class I-<br>
restricted CTL responses (without the use of a live vector). This may provide<br>
broader protection against different strains of a virus by targeting epitopes from<br>
conserved internal proteins pertinent to many strains of the same virus. Since the<br>
antigen is produced with mammalian post-translational modification,<br>
conformation and oligomerization, it is more likely to be similar or identical to<br>
the wild-type protein produced by viral infection than recombinant or chemically<br>
modified proteins. However, this distinction may turn out to be a disadvantage<br>
for the application of bacterial antigens, since non-native post-translational<br>
modification may result in reduced immunogenicity. In addition, viral surface<br>
proteins are not highly organized in the absence of matrix proteins.<br>
In addition to applications for the prevention of infectious disease, vaccine<br>
technology is now being utilized to address immune problems associated with<br>
allergies. In allergic individuals, antibodies of the IgE isotype are produced in an<br>
inappropriate humoral immune response towards particular antigens (allergens).<br>
The treatment of allergies by allergy immunotherapy requires weekly<br>
administration of successively increasing doses of the particular allergen over a<br>
period of up to 3-5 years. Presumably, 'blocking' IgG antibodies are generated<br><br>
that intercept allergens in nasal or respiratory secretions or in membranes before<br>
they react with IgE antibodies on mast cells. However, no constant relationship<br>
exists between IgG titers and symptom relief. Presently, this is an extremely<br>
time- and cost-consuming process, to be considered only for patients with severe<br>
symptoms over an extended period each year.<br>
It is well established that the administration of purified proteins alone is<br>
usually not sufficient to elicit a strong immune response; isolated antigen<br>
generally must be given together withhelper substances called adjuvants. Within<br>
these adjuvants, the administered antigen is protected against rapid degradation,<br>
and the adjuvant provides an extended release of a low level of antigen.<br>
Unlike isolated proteins, viruses induce prompt and efficient immune<br>
responses in the absence of any adjuvants both with and without T-cell help<br>
(Bachmann &amp; Zinkernagel, Ann. Rev. Immunol. 15:235-270(1997)). Although<br>
viruses often consist of few proteins, they are able to trigger much stronger<br>
immune responses than their isolated components. For B cell responses, it is<br>
known that one crucial factor for the immunogenicity of viruses is the<br>
repetitiveness and order of surface epitopes. Many viruses exhibit a quasi-<br>
crystalline surface that displays a regular array of epitopes which efficiently<br>
crosslinks epitope-specific immunoglobulins on B cells (Bachmann &amp;<br>
Zinkernagel, Immunol. Today 77:553-558 (1996)). This crosslinking of surface<br>
immunoglobulins on B cells is a strong activation signal that directly induces cell-<br>
cycle progression and the production of IgM antibodies. Further, such triggered<br>
B cells are able to activate T helper cells, which in turn induce a switch from IgM<br>
to IgG antibody production in B cells and the generation of long-lived B cell<br>
memory - the goal of any vaccination (Bachmann &amp; Zinkernagel, Ann. Rev.<br>
Immunol. 75:235-270 (1997)). Viral structure is even linked to the generation of<br>
anti-antibodies in autoimmune disease and as a part of the natural response to<br>
pathogens (see Fehr, T., el al, J. Exp. Med. 7(55:1785-1792 (1997)). Thus,<br>
antigens on viral particles that are organized in an ordered and repetitive array are<br>
highly immunogenic since they can directly activate B cells.<br><br>
In addition to strong B cell responses, viral particles are also able to<br>
induce the generation of a cytotoxic T cell response, another crucial arm of the<br>
immune system. These cytotoxic T cells are particularly important for the<br>
elimination of non-cytopathic viruses such as HIV or hepatitis B virus and for the<br>
eradication of tumors. Cytotoxic T cells do not recognize native antigens but<br>
rather recognize their degradation products in association with MHC class I<br>
molecules (Townsend &amp; Bodmer, Ann. Rev. Immunol. 7:601-624 (1989)).<br>
Macrophages and dendritic cells are able to take up and process exogenous viral<br>
particles (but not their soluble, isolated components) and present the generated<br>
degradation product to cytotoxic T cells, leading to their activation and<br>
proliferation (Kovacsovics-Bankowski et al., Proc. Nad. Acad. Sci. USA<br>
90:4942-4946 (1993); Bachmann et al., Eur. J. Immunol. 26:2595-2600 (1996)).<br>
Viral particles as antigens exhibit two advantages over their isolated<br>
components: (1) Due to their highly repetitive surface structure, they are able to<br>
directly activate B cells, leading to high antibody titers and long-lasting B cell<br>
memory; and (2) Viral particles but not soluble proteins are able to induce a<br>
cytotoxic T cell response, even if the viruses are non-infectious and adjuvants are<br>
absent.<br>
Several new vaccine strategies exploit the inherent immunogenicity of<br>
viruses. Some of these approaches focus on the particulate nature of the virus<br>
particle; for example see Harding, C.V. and Song, R., (J. Immunology 153:4925<br>
(1994)), which discloses a vaccine consisting of latex beads and antigen;<br>
Kovacsovics-Bankowski, ML, et al. (Proc. Natl. Acad. Sci. USA 90:4942-4946<br>
(1993)), which discloses a vaccine consisting of iron oxide beads and antigen;<br>
U.S. Patent No: 5,334.394 to Kossovsky, N., et al., which discloses core particles<br>
coated with antigen; U.S. Patent No. 5,871,747. which discloses synthetic<br>
polymer particles carrying on the surface one or more proteins covalently bonded<br>
thereto; and a core particle with a non-covalently bound coating, which at least<br>
partially covers the surface of said core particle, and at least one biologically<br>
active agent in contact with said coated core particle (see. e.g., WO/94/15585).<br><br>
However, a disadvantage of these viral mimicry systems is that they are<br>
not able to recreate the ordered presentation of antigen found on the viral surface.<br>
Antigens coupled to a surface in a random orientation are found to induce CTL<br>
response and no or only weak B-cell response. For an efficient vaccine, both<br>
arms of the immune system have to be strongly activated, as described above and<br>
in Bachmann &amp; Zinkernagel, Ann. Rev. Immunol. J5:235 (1997).<br>
In another example, recombinant viruses are being utilized for antigen<br>
delivery. Filamentous phage virus containing an antigen fused to a capsid protein<br>
has been found to be highly immunogenic (see Perham R.N., et al, FEMS<br>
Microbiol. Rev. 77:25-31 (1995); Willis et al, Gene 725:85-88 (1993);<br>
Minenkova et al, Gene 725:85-88 (1993)). However, this system is limited to<br>
very small peptides (5 or 6 amino acid residues) when the fusion protein is<br>
expressed at a high level (Iannolo et al, J. Mol. Biol 2^5:835-844 (1995)) or<br>
limited to the low level expression of larger proteins (de la Cruz et al, J. Biol<br>
Chem. 263:4318-4322 (1988)). For small peptides, so far only the CTL response<br>
is observed and no or only weak B-cell response.<br>
In yet another system, recombinant alphaviruses are proposed as a means<br>
of antigen delivery (see U.S. Patent Nos. 5,766,602; 5,792,462; 5,739,026;<br>
5;789,245 and 5,814,482). Problems with the recombinant virus systems<br>
described so far include a low density expression of the heterologous protein on<br>
the viral surface and/or the difficulty of successfully and repeatedly creating a<br>
new and different recombinant viruses for different applications.<br>
In a further development, virus-like particles (VLPs) are being exploited<br>
in the area of vaccine production because of both their structural properties and<br>
their non-infectious nature. VLPs are supermolecular structures built in a<br>
symmetric manner from many protein molecules of one or more types. They lack<br>
the viral genome and, therefore, are noninfectious. VLPs can often be produced<br>
in large quantities by heterologous expression and can be easily be purified.<br>
Examples of VLPs include the capsid proteins of hepatitis B virus (Ulrich,<br>
et al. Virus Res. 50:141-182 (1998)), measles virus (Warnes, et al, Gene<br><br>
160:173-178 (1995)), Sindbis virus, rotavirus (US Patent Nos. 5.071,651 and<br>
5,374,426), foot-and-mouth-disease virus (Twomey, et al., Vaccine 13:<br>
1603-1610, (1995)), Norwalk virus (Jiang, X., et al., Science 250:1580-1583<br>
(1990);Matsui, S.M.,et al.,J. Clin. Invest. 87:1456-1461 (1991)), the retro viral<br>
GAG protein (PCT Patent Appl. No. WO 96/30523), the retrotransposon Ty<br>
protein p1, the surface protein of Hepatitis B virus (WO 92/11291) and human<br>
papilloma virus (WO 98/15631). In some instances, recombinant DNA<br>
technology may be utilized to fuse a heterologous protein to a VLP protein<br>
(Kratz, P.A., et al, Proc. Natl Acad. Sci. USA 96: 19151920 (1999)).<br>
Thus, there is a need in the art for the development of new and improved<br>
vaccines that promote a strong CTL and B-cell immune response as efficiently as<br>
natural pathogens.<br>
Summary of the Invention<br>
The invention provides a versatile new technology that allows production<br>
of particles coated with any desired antigen. The technology allows the creation<br>
of highly efficient vaccines against infectious diseases and for the creation of<br>
vaccines for the treatment of allergies and cancers.<br>
In a first embodiment, the invention provides a novel composition<br>
comprising (A) a non-natural molecular scaffold and (B) an antigen or antigenic<br>
determinant.<br>
The non-natural molecular scaffold comprises (i) a core particle selected<br>
from the group consisting of (1) a core particle of non-natural origin and (2) a<br>
core particle of natural origin; and (ii) an organizer comprising at least one first<br>
attachment site, wherein said organizer is connected to said core particle by at<br>
least one covalent bond.<br>
The antigen or antigenic determinant has at least one second attachment<br>
site which is selected from the group consisting of (i) an attachment site not<br><br>
naturally occurring with said antigen or antigenic determinant; and (ii) an<br>
attachment site naturally occurring with said antigen or antigenic determinant.<br>
The invention provides for an ordered and repetitive antigen array through<br>
an association of the second attachment site to the first attachment site by way of<br>
at least one non-peptide bond. Thus, the antigen or antigenic determinant and the<br>
non-natural molecular scaffold are brought together through this association of<br>
the first and the second attachment site to form an ordered and repetitive antigen<br>
array.<br>
In another embodiment, the core particle of the aforementioned<br>
composition comprises a virus, a virus-like particle, a bacteriophage, a viral<br>
capsid particle or a recombinant form thereof. Alternatively, the core particle<br>
may be a synthetic polymer or a metal.<br>
In a particular embodiment, the organizer may comprise at least one first<br>
attachment site. The first and the second attachment sites are particularly<br>
important elements of the composition of the invention. In various embodiments<br>
of the invention, the first and/or the second attachment site may be an antigen and<br>
an antibody or antibody fragment thereto; biotin and avidin; strepavidin and<br>
biotin; a receptor and its ligand; a ligand-binding protein and its ligand;<br>
interacting leucine zipper polypeptides; an amino group and a chemical group<br>
reactive thereto; a carboxyl group and a chemical group reactive thereto; a<br>
sulfhydryl group and a chemical group reactive thereto; or a combination thereof.<br>
In a more preferred embodiment, the invention provides the coupling of<br>
almost any antigen of choice to the surface of a virus, bacteriophage, virus-like<br>
particle or viral capsid particle. By bringing an antigen into a quasi-crystalline<br>
"virus-like' structure, the invention exploits the strong antiviral immune reaction<br>
of a host for the production of a highly efficient immune response, i.e., a<br>
vaccination, against the displayed antigen.<br>
In one preferred embodiment, the core particle may be selected from the<br>
group consisting of: recombinant proteins of Rotavirus, recombinant proteins of<br>
Norwalkvirus, recombinant proteins of Alphavirus, recombinant proteins of Foot<br><br>
and Mouth Disease virus, recombinant proteins of Retrovirus, recombinant<br>
proteins of Hepatitis B virus, recombinant proteins of Tobacco mosaic virus,<br>
recombinant proteins of Flock House Virus, and recombinant proteins of human<br>
Papilomavirus.<br>
In another preferred embodiment, the antigen may be selected from the<br>
group consisting of: (1) a protein suited to induce an immune response against<br>
cancer cells; (2) a protein suited to induce an immune response against infectious<br>
diseases; (3) a protein suited to induce an immune response against allergens; and<br>
(4) a protein suited to induce an immune response in farm animals.<br>
In a particularly preferred embodiment of the invention, the first<br>
attachment site and/or the second attachment site comprise an interacting leucine<br>
zipper polypeptide. In most preferred embodiment, the first attachment site<br>
and/or the second attachment site are selected from the group comprising: (1) the<br>
JUN leucine zipper protein domain; and (2) the FOS leucine zipper protein<br>
domain.<br>
In another preferred embodiment, the first attachment site and/or the<br>
second attachment site are selected from the group comprising: (1) a genetically<br>
engineered lysine residue and (2) a genetically engineered cysteine residue, two<br>
residues that may be chemically linked together.<br>
Other embodiments of the invention include processes for the production<br>
of the compositions of the invention and a methods of medical treatment using<br>
said compositions.<br>
It is to be understood that both the foregoing general description and the<br>
following detailed description are exemplary and explanatory only and are<br>
intended to provide further explanation of the invention as claimed.<br><br>
Brief Description of the Drawings<br>
Figure 1 Western blot demonstrating the production of viral particles<br>
containing the E2-JUN fusion protein using the pCYTts::E2JUN expression<br>
vector.<br>
Figure 2 Western blot demonstrating the production of viral particles<br>
containing the E2-JUN fusion protein expressed from pTE5'2J::E2JUN<br>
expression vector.<br>
Figure 3 Western dot blot demonstrating bacterial and eukaryotic<br>
expression of the FOS-hgh. antigen.<br>
Figure 4 Expression of HBcAg-JUN in E. coli cells.<br>
Figure 5 Western blot demonstrating that HBcAg-JUN is soluble in<br>
E. colilysates.<br>
Figure 6 SDS-PAGE analysis of enrichment of HBcAg-JUN capsid<br>
particles on a sucrose density gradient.<br>
Figure 7 Non-reducing SDS-PAGE analysis of the coupling of hGH-FOS<br>
and HBcAg-JUN particles.<br>
Detailed Description of the Preferred Embodiments<br>
1. Definitions<br>
The following definitions are provided to clarify the subj ect matter which<br>
the inventors consider to be the present invention.<br>
Alphavirus: As used herein, the term "alphavirus" refers to any of the<br>
RNA viruses included within the genus Alphavirus. Descriptions of the members<br>
of this genus are contained in Strauss and Strauss. Microbiol. Rev., 55:491-562<br>
(1994). Examples of alphaviruses include Aura virus, Bebaru virus, Cabassou<br>
virus, Chikungunya virus, Easter equine encephalomyelitis virus, Fort morgan<br>
virus, Getah virus, Kyzylagach virus, Mayoaro virus. Middleburg virus.<br><br>
Mucambo virus, Ndumu virus, Pixuna virus, Tonate virus, Triniti virus, Una<br>
virus, Western equine encephalomyelitis virus, Whataroa virus, Sindbis virus<br>
•SIN), Semliki forest virus (SFV), Venezuelan equine encephalomyelitis virus<br>
(VEE), and Ross River virus.<br>
Antigen: As used herein, the term "antigen" is a molecule capable of<br>
being bound by an antibody. An antigen is additionally capable of inducing a<br>
humoral immune response and/or cellular immune response leading to the<br>
production of B- and/or T-lymphocytes. An antigen may have one or more<br>
epitopes (B- and T- epitopes). The specific reaction referred to above is meant<br>
to indicate that the antigen will react, in a highly selective manner, with its<br>
corresponding antibody and not with the multitude of other antibodies which may<br>
be evoked by other antigens.<br>
Antigenic determinant: As used herein, the term"antigenic determinant"<br>
is meant to refer to that portion of an antigen that is specifically recognized by<br>
either B- or T-lymphocytes. B-lymphocytes respond to foreign antigenic<br>
determinants via antibody production, whereas T-lymphocytes are the mediator<br>
of cellular immunity. Thus, antigenic determinants or epitopes are those parts of<br>
an antigen that are recognized by antibodies, or in the context of an MHC, by T-<br>
cell receptors.<br>
Association: As used herein, the term "association" as it applies to the<br>
first and second attachment sites, is used to refer to at least one non-peptide bond.<br>
The nature of the association may be covalent, ionic, hydrophobic, polar or any<br>
combination thereof.<br>
Attachment Site, First: As used herein, the phrase "first attachment site"<br>
refers to an element of the "organizer", itself bound to the core particle in a non-<br>
random fashion, to which the second attachment site located on the antigen or<br>
antigenic determinant may associate. The first attachment site may be a protein,<br>
a polypeptide, a peptide, a sugar, a polynucleotide, a natural or synthetic polymer,<br>
a secondary metabolite or compound (biotin, fluorescein, retinol, digoxigenin,<br>
metal ions, phenylmethylsulfonylfluoride), or a combination thereof, or a<br><br>
chemically reactive group thereof. Multiple first attachment sites are present on<br>
the surface of the non-natural molecular scaffold in a repetitive configuration.<br>
Attachment Site, Second: As used herein, the phrase "second attachment<br>
site" refers to an element associated with the antigen or antigenic determinant to<br>
which the first attachment site of the "organizer" located on the surface of the<br>
non-natural molecular scaffold may associate. The second attachment site of the<br>
antigen or antigenic determinant may be a protein, a polypeptide, a peptide, a<br>
sugar, a polynucleotide, a natural or synthetic polymer, a secondary metabolite or<br>
compound (biotin, fluorescein, retinol, digoxigenin, metal ions,<br>
phenylmethylsulfonylfluoride), or a combination thereof, or a chemically reactive<br>
group thereof. At least one second attachment site is present on the antigen or<br>
antigenic determinant.<br>
Core particle: As used herein, the term "core particle" refers to a rigid<br>
structure with an inherent repetitive organization that provides a foundation for<br>
attachment of an "organizer". A core particle as used herein may be the product<br>
of a synthetic process or the product of a biological process.<br>
Cis-acting: As used herein, the phrase "Cis-acting" sequence refers to<br>
nucleic acid sequences to which a replicase binds to catalyze the RNA-dependent<br>
replication of RNA molecules. These replication events result in the replication<br>
of the full-length and partial RNA molecules and, thus, the alpahvirus<br>
subgenomic promoter is also a "Cis-acting" sequence. Cis-acting sequences may<br>
be located at or near the 5' end, 3' end, or both ends of a nucleic acid molecule,<br>
as well as internally.<br>
Fusion: As used herein, the term "fusion" refers to the combination of<br>
amino acid sequences of different origin in one polypeptide chain by in-frame<br>
combination of their coding nucleotide sequences. The term "fusion" explicitly<br>
encompasses internal fusions, i.e., insertion of sequences of different origin<br>
within a polypeptide chain, in addition to fusion to one of its termini.<br>
Heterologous sequence: As used herein, the term "heterologous<br>
sequence" refers to a second nucleotide sequence present in a vector of the<br><br>
invention. The term "heterologous sequence" also refers to any amino acid or<br>
RNA sequence encoded by a heterologous DNA sequence contained in a vector<br>
of the invention. Heterologous nucleotide sequences can encode proteins or RNA<br>
molecules normally expressed in the cell type in which they are present or<br>
molecules not normally expressed therein (e.g., Sindbis structural proteins).<br>
Isolated: As used herein, when the term "isolated" is used in reference to<br>
a molecule, the term means that the molecule has been removed from its native<br>
environment. For example, a polynucleotide or a polypeptide naturally present<br>
in a living animal is not "isolated," but the same polynucleotide or polypeptide<br>
separated from the coexisting materials of its natural state is "isolated." Further,<br>
recombinant DNA molecules contained in a vector are considered isolated for the<br>
purposes of the present invention. Isolated RNA molecules include in vivo or in<br>
vitro RNA replication products of DNA and RNA molecules. Isolated nucleic<br>
acid molecules further include synthetically produced molecules. Additionally,<br>
vector molecules contained in recombinant host cells are also isolated. Thus, not<br>
all "isolated" molecules need be "purified."<br>
Immunotherapeutic: As used herein, the term "immunotherapeutic" is<br>
a composition for the treatment of diseases or disorders.. More specifically, the<br>
term is used to refer to a method of treatment for allergies or a method of<br>
treatment for cancer.<br>
Individual: As used herein, the term "individual" refers to multicellular<br>
organisms and includes both plants and animals. Preferred multicellular<br>
organisms are animals, more preferred are vertebrates, even more preferred are<br>
mammals, and most preferred are humans.<br>
Low or undetectable: As used herein, the phrase "low or undetectable,"<br>
when used in reference to gene expression level, refers to a level of expression<br>
which is either significantly lower than that seen when the gene is maximally<br>
induced (e.g.. at least five fold lower) or is not readily detectable by the methods<br>
used in the following examples section.<br><br>
Lectin: As used herein, proteins obtained particularly from the seeds of<br>
leguminous plants, but also from many other plant and animal sources, that have<br>
binding sites for specific mono- or oligosaccharides. Examples include<br>
concanavalin A and wheat-germ agglutinin, which are widely used as analytical<br>
and preparative agents in the study of glycopfoteins.<br>
Natural origin: As used herein, the term "natural origin" means that the<br>
whole or parts thereof are not synthetic and exist or are produced in nature.<br>
Non-natural: As used herein, the term generally means not from nature,<br>
more specifically, the term means from the hand of man.<br>
Non-natural origin: As used herein, the term "non-natural origin"<br>
generally means synthetic or not from nature; more specifically, the term means<br>
from the hand of man.<br>
Non-natural molecular scaffold: As used herein, the phrase "non-natural<br>
molecular scaffold" refers to any product made by the hand of man that may serve<br>
to provide a rigid and repetitive array of first attachment sites. Ideally but not<br>
necessarily, these first attachment sites are in a geometric order. The non-natural<br>
molecular scaffold may be organic or non-organic and may be synthesized<br>
chemically or through a biological process, in part or in whole. The non-natural<br>
molecular scaffold is comprised of: (a) a core particle, either of natural or non-<br>
natural origin; and (b) an organizer, which itself comprises at least one first<br>
attachment site and is connected to a core particle by at least one covalent bond.<br>
In a particular embodiment, the non-natural molecular scaffold may be a virus,<br>
virus-like particle, a virus capsid particle, a phage, a recombmant form thereof,<br>
or synthetic particle.<br>
Ordered and repetitive antigen or antigenic determinant array: As<br>
used herein, the term "ordered and repetitive antigen or antigenic determinant<br>
array" generally refers to a repeating pattern of antigen or antigenic determinant,<br>
characterized by a uniform spacial arrangement of the antigens or antigenic<br>
determinants with respect to the scaffold. In one embodiment of the invention,<br>
the repeating pattern may be a geometric pattern. An ideal ordered and repetitive<br><br>
antigen or antigenic determinant array will possess a strictly repetitive<br>
paracrystalline order of antigen or antigenic determinant with a spacing of 5 to 15<br>
. nanometers.<br>
Organizer: As used herein, the term "organizer" is used to refer to an<br>
element bound to a core particle in a non-random fashion that provides a<br>
nucleation site for creating an ordered and repetitive antigen array. An organizer<br>
is any element comprising at least one first attachment site that is bound to a core<br>
particle by at least one covalent bond. An organizer may be a protein, a<br>
polypeptide, a peptide, an amino acid (i.e., a residue of a protein, a polypeptide<br>
or peptide), a sugar, a polynucleotide, a natural or synthetic polymer, a secondary<br>
metabolite or compound (biotin, fluorescein, retinol, digoxigenin, metal ions,<br>
phenylmethylsulfonylfluoride), or a combination thereof, or a chemically reactive<br>
group thereof.<br>
Permissive temperature: As used herein, the phrase "permissive<br>
temperature" refers to temperatures at which an enzyme has relatively high levels<br>
of catalytic activity.<br>
Purified: As used herein, when the term "purified" is used in reference to<br>
a molecule, it means that the concentration of the molecule being purified has<br>
been increased relative to molecules associated with it in its natural environment.<br>
Naturally associated molecules include proteins, nucleic acids, lipids and sugars<br>
but generally do not include water, buffers, and reagents added to maintain the<br>
integrity or facilitate the purification of the molecule being purified. For<br>
example, even if mRNA is diluted with an aqueous solvent during oligo dT<br>
column chromatography, mRNA molecules are purified by this chromatography<br>
if naturally associated nucleic acids and other biological molecules do not bind<br>
to the column and are separated from the subject mRNA molecules.<br>
Receptor: As used herein, the term "receptor" refers to proteins or<br>
glycoproteins or fragments thereof capable of interacting with another molecule,<br>
called the ligand. The ligand may belong to any class of biochemical or chemical<br>
compounds. The receptor need not necessarily be a membrane-bound protein.<br><br>
Soluble protein, like e.g. maltose binding protein or retinol binding protein are<br>
receptors as well.<br>
Residue. As used herein, the term "residue" is meant to mean a specific<br>
amino acid in a polypeptide backbone or side chain.<br>
Temperature-sensitive: As used herein, the phrase "temperature-<br>
sensitive" refers to an enzyme which readily catalyzes a reaction at one<br>
temperature but catalyzes the same reaction slowly or not at all at another<br>
temperature. An example of a temperature-sensitive enzyme is the replicase<br>
protein encoded by the pCYTts vector, which has readily detectable replicase<br>
activity at temperatures below 34 °C and has low or undetectable activity at 37 °C.<br>
Transcription: As used herein, the term "transcription" refers to the<br>
production of RNA molecules from DNA templates catalyzed by RNA<br>
polymerases.<br>
Recombinant host cell: As used herein, the term "recombinant host cell"<br>
refers to a host cell into which one ore more nucleic acid molecules of the<br>
invention have been introduced.<br>
Recombinant virus: As used herein, the phrase "recombinant virus"<br>
refers to a virus that is genetically modified by the hand of man. The phrase<br>
covers any virus known in the art. More specifically, the phrase refers to a an<br>
alphavirus genetically modified by the hand of man, and most specifically, the<br>
phrase refers to a Sinbis virus genetically modified by the hand of man.<br>
Restrictive temperature: As used herein, the phrase "restrictive<br>
temperature" refers to temperatures at which an enzyme has low or undetectable<br>
levels of catalytic activity. Both "hot" and "cold" sensitive mutants are known<br>
and, thus, a restrictive temperature may be higher or lower than a permissive<br>
temperature.<br>
RNA-dependent RNA replication event: As used herein, the phrase<br>
"RNA-dependent RNA replication event" refers to processes which result in the<br>
formation of an RNA molecule using an RNA molecule as a template.<br><br>
RNA-Dependent RNA polymerase: As used herein, the phrase "RNA-<br>
Dependent RNA polymerase" refers to a polymerase which catalyzes the<br>
production of an RNA molecule from another RNA molecule. This term is used<br>
herein synonymously with the term "replicase."<br>
Untranslated RNA: As used herein, the phrase "untranslated RNA" refers<br>
to an RNA sequence or molecule which does not encode an open reading frame<br>
or encodes an open reading frame, or portion thereof, but in a format in which an<br>
amino acid sequence will not be produced (e.g., no initiation codon is present).<br>
Examples of such molecules are tRNA molecules, rRNA molecules, and<br>
ribozymes.<br>
Vector: As used herein, the term "vector" refers to an agent {e.g., a<br>
plasmid or virus) used to transmit genetic material to a host cell. A vector may<br>
be composed of either DNA or RNA.<br>
one, a, or an: When the terms "one," "a," or "an" are used in this<br>
disclosure, they mean "at least one" or "one or more," unless otherwise indicated.<br>
2. Compositions of Ordered and Repetitive Antigen or Antigenic<br>
Determinant Arrays and Methods to Make the Same<br>
The disclosed invention provides compositions comprising an ordered and<br>
repetitive antigen or antigenic determinant. Furthermore, the invention<br>
conveniently enables the practitioner to construct ordered and repetitive antigen<br>
or antigenic determinant arrays for various treatment purposes, which includes the<br>
prevention of infectious diseases, the treatment of allergies and the treatment of<br>
cancers.<br>
Compositions of the invention essentially comprise two elements: (1) a<br>
non-natural molecular scaffold; and (2) an antigen or antigenic determinant with<br>
at least one second attachment site capable of association through at least one<br>
non-peptide bond to said first attachment site.<br>
The non-natural molecular scaffold comprises (a) a core particle selected<br>
from the group consisting of (1) a core particle of non-natural origin and (2) a<br><br>
core particle of natural origin; and (b) an organizer comprising at least one first<br>
attachment site, wherein said organizer is connected to said core particle by at<br>
least one covalent bond.<br>
The antigen or antigenic determinant has at least one second attachment<br>
site which is selected from the group consisting of (a) an attachment site not<br>
naturally occurring with said antigen or antigenic determinant; and (b) an<br>
attachment site naturally occurring with said antigen or antigenic determinant.<br>
The invention provides for an ordered and repetitive antigen array through<br>
an association of the second attachment site to the first attachment site by way of<br>
at least one non-peptide bond. Thus, the antigen or antigenic determinant and the<br>
non-natural molecular scaffold are brought together through this association of<br>
the first and the second attachment site to form an ordered and repetitive antigen<br>
array.<br>
The practioner may specifically design the antigen or antigenic<br>
determinant and the second attachment site such that the arrangement of all the<br>
antigens or antigenic determinants bound to the non-natural molecular scaffold<br>
will be uniform. For example, one may place a single second attachment site on<br>
the antigen or antigenic determinant at the carboxyl or amino terminus, thereby<br>
ensuring through design that all antigen or antigenic determinant molecules that<br>
are attached to the non-natural molecular scaffold are positioned in a uniform<br>
way. Thus, the invention provides a convenient means of placing any antigen or<br>
antigenic determinant onto a non-natural molecular scaffold in a defined order<br>
and repetition.<br>
As will be clear to those skilled in the art, certain embodiments of the<br>
invention involve the use of recombinant nucleic acid technologies such as<br>
cloning, polymerase chain reaction, the purification of DNA and RNA, the<br>
expression of recombinant proteins in prokaryotic and eukaryotic cells, etc. Such<br>
methodologies are well known to those skilled in the art and may be conveniently<br>
found in published laboratory methods manuals (e.g., Sambrook, J. et al, eds.,<br>
MOLECULAR CLONING, A LABORATORY MANUAL, 2nd. edition, Cold Spring<br><br>
Harbor Laboratory Press, Cold Spring Harbor, N.Y. (1989): Ausubel, F. el al.,<br>
eds., CURRENTPROTOCOLS IN MOLECULAR BIOLOGY, John H. Wiley &amp; Sons, Inc.<br>
(1997)). Fundamental laboratory techniques for working with tissue culture cell<br>
lines (Celis, J., ed., CELL BIOLOGY, Academic Press, 2nd edition, (1998)) and<br>
antibody-based technologies (Harlow, E. and Lane, D., "Antibodies: A Laboratory<br>
Manual," Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y. (1988);<br>
Deutscher, M.P., "Guide to Protein Purification," Meth. Enzymol. 128, Academic<br>
Press San Diego (1990); Scopes, R.K., "Protein Purification Principles and<br>
Practice," 3rd ed., Springer-Verlag, New York (1994)) are also adequately<br>
described in the literature, all of which are incorporated herein by reference.<br>
A. Construction of a Non-natural Molecular Scaffold<br>
One element in the composition of the invention is a non-natural,<br>
molecular scaffold comprising a core particle and an organizer. As used herein,<br>
the phrase "non-natural molecular scaffold" refers to any product made by the<br>
hand of man that may serve to provide a rigid and repetitive array of first<br>
attachment sites. More specifically, the non-natural molecular scaffold comprises<br>
(a) a core particle selected from the group consisting of (1) a core particle of non-<br>
natural origin and (2) a core particle of natural origin; and (b) an organizer<br>
comprising at least one first attachment site, wherein said organizer is connected<br>
to said core particle by at least one covalent bond.<br>
As will be readily apparent to those skilled in the art. the core particle of<br>
the non-natural molecular scaffold of the invention is not limited to any specific<br>
form. The core particle may be organic or non-organic and may be synthesized<br>
chemically or through a biological process.<br>
In one embodiment, a non-natural core particle may be a synthetic<br>
polymer, a lipid micelle or a metal. Such core particles are known in the art,<br>
providing a basis from which to build the novel non-natural molecular scaffold<br>
of the invention. By way of example, synthetic polymer or metal core particles<br><br>
are described in U.S. Patent No. 5,770,380, which discloses the use of a<br>
calixarene organic scaffold to which is attached a plurality of peptide loops in the<br>
creation of an 'antibody mimic', and U.S. Patent No. 5,334,394 describes<br>
nanocrystalline particles used as a viral decoy that are composed of a wide variety<br>
of inorganic materials, including metals or ceramics. Preferred metals in this<br>
embodiment include chromium, rubidium, iron, zinc, selenium, nickel, goldj<br>
silver, platinum. Preferred ceramic materials in this embodiment include silicon<br>
dioxide, titanium dioxide, aluminum oxide, ruthenium oxide and tin oxide. The<br>
core particles of this embodiment may be made from organic materials including<br>
carbon (diamond). Preferred polymers include polystyrene, nylon and<br>
nitrocellulose. For this type of nanocrystalline particle, panicles made from tin<br>
oxide, titanium dioxide or carbon (diamond) are particularly preferred. A lipid<br>
micelle may be prepared by any means known in the art. For example micelles<br>
may be prepared according to the procedure of Baiselle and Millar (Baiselle, C. J.<br>
and Millar, D.B., Biophys. Chem. 4:355-361 (1975)) or Corti et al. (Corti, M.,<br>
Degriorgio, V., Sonnino, S., Ghidoni R., Masserini, M. and Tettamanti, G.,<br>
Chem. Phys. Lipids 38: 197-214 (1981)) or Lopez et al. (Lopez, O. de la Maza,<br>
A., Coderch, L., Lopez-Iglesias, C, Wehrli, E. and Parra, J.L., FEBS Lett. 426:<br>
314-318 (1998)) or Topchieva and Karezin (Topchieva, I. and Karaezin, K., J.<br>
Colloid Interface Sci. 213: 29-35 (1999)) or Morein et al, (Morein, B.,<br>
Sundquist, B., Hoglund, S., Dalsgaard K. and Osterhaus, A.. Nature 308: 457-60<br>
(1984)), which are all incorporated herein by reference.<br>
The core particle may also be produced through a biological process,<br>
which may be natural or non-natural. By way of example, this type of<br>
embodiment may includes a core particle comprising a virus, virus-like particle,<br>
a phage, a viral capsid particle or a recombinant form thereof. In a more specific<br>
embodiment, the core particle may comprise recombinant proteins of Rotavirus,<br>
recombinant proteins of Norwalkvirus, recombinant proteins of Alphavirus,<br>
recombinant proteins of Foot and Mouth Disease virus, recombinant proteins of<br>
Retrovirus, recombinant proteins of Hepatitis B virus, recombinant proteins of<br><br>
Tobacco mosaic virus, recombinant proteins of Flock House Virus, and<br>
recombinant proteins of human Papilomavirus.<br>
Whether natural or non-natural, the core particle of the invention is<br>
characterized by comprising organizer that is attached to the natural or non-<br>
natural core particle by at least one covalent bond. The organizer is an element<br>
bound to a core particle in a non-random fashion that provides a nucleation site<br>
for creating an ordered and repetitive antigen array. Ideally, but not necessarily,<br>
the organizer is associated with the core particle in a geometric order. Minimally,<br>
the organizer comprises a first attachment site.<br>
As previously stated, the organizer may be any element comprising at<br>
least one first attachment site that is bound to a core particle by at least one<br>
covalent bond. An organizer may be a protein, a polypeptide, a peptide, an<br>
amino acid (i.e., a residue of a protein, a polypeptide or peptide), a sugar, a<br>
"polynucleotide, a natural or synthetic polymer, a secondary metabolite or<br>
compound (biotin, fluorescein, retinol, digoxigenin, metal ions,<br>
phenylmethylsulfonylfluoride), or a combination thereof, or a chemically reactive<br>
group thereof. In a more specific embodiment, the organizer may comprise a first<br>
attachment site comprising an antigen, an antibody or antibody fragment, biotin,<br>
avidin, strepavidin, a receptor, a receptor ligand, a Iigand. a ligand-binding<br>
protein, an interacting leucine zipper polypeptide, an amino group, a chemical<br>
group reactive to an amino group; a carboxyl group, chemical group reactive to<br>
a carboxyl group, a sulfhydryl group, a chemical group reactive to a sulfhydryl<br>
group, or a combination thereof.<br>
In a preferred embodiment, the core particle of the non-natural molecular<br>
scaffold comprises a virus, a bacteriophage, a virus-like panicle, a viral capsid<br>
particle or a recombinant form thereof. Any virus known in the art having an<br>
ordered and repetitive coat and/or core protein structure may be selected as a non-<br>
naiural molecular scaffold of the invention; examples of suitable viruses include:<br>
sindbis and other alphaviruses; vesicular somatitis virus; rhabdo-, (e.g. vesicular<br>
stomatitis virus), picorna-, toga-, orthomyxo-, polyoma-, parvovirus, rotavirus,<br><br>
norwalkvirus, foot and mouth disease virus, a retrovirus. hepatitis B virus,<br>
tobacco mosaic virus, flock house virus, human papilomavirus (for example, see<br>
Table 1 in Bachman, M.F. and Zinkernagel, R.M., Immunol. Today 77:553-558<br>
(1996)).<br>
In one embodiment, the invention utilizes genetic engineering of a virus<br>
to create a fusion between an ordered and repetitive viral envelope protein and an<br>
organizer comprising a heterologous protein, peptide, antigenic determinant or a<br>
reactive amino acid residue of choice. Other genetic manipulations known to<br>
those in the art may be included in the construction of the non-natural molecular<br>
scaffold; for example, it may be desirable to restrict the replication ability of the<br>
recombinant virus through genetic mutation. The viral protein selected for fusion<br>
to the organizer (i.e., first attachment site) protein should have an organized and<br>
repetitive structure, more preferably a paracrystalline organization optimally with<br>
a spacing of 5-15nm on the surface of the virus. The creation of this type of<br>
fusion protein will result in multiple, ordered and repetitive organizers on the<br>
surface of the virus. Thus, the ordered and repetitive organization of the first<br>
attachment sites resulting therefrom will reflect the normal organization of the<br>
native viral protein.<br>
As will be discussed in more detail herein, in a preferred embodiment of<br>
the invention, the scaffold is a recombinant alphavirus, and more specifically, a<br>
recombinant Sinbis virus. Alphaviruses are positive stranded RNA viruses that<br>
replicate their genomic RNA entirely in the cytoplasm of the infected cell and<br>
without a DNA intermediate (Strauss, J. and Strauss, E., Microbiol. Rev. JS:491 -<br>
562 (1994)). Several members of the alphavirus family, Sindbis (Xiong, C. et al,<br>
Science 243:1188-1191 (1989); Schlesinger, S., Trends Biotechnol. 77:18-22<br>
(! 993)), Semliki Forest Virus (SFV) (Liljestrom, P. &amp; Garoff. H., Bio/Technology<br>
P:1356-1361 (1991)) and others (Davis, N.L. et al. Virology 777:189-204<br>
(1989)), have received considerable attention for use as virus-based expression<br>
vectors for a variety of different proteins (Lundstrom, K., Curr. Opin. Biotechnol.<br>
5:578-582 (1997); Liljestrom, P., Curr. Opin. Biotechnol. 5:495-500 (1994)) and<br><br>
as candidates for vaccine development. Recently, a number of patents have<br>
issued directed to the use of alphaviruses for the expression of heterologous<br>
proteins and the development of vaccines (see U.S. Patent Nos. 5,766,602;<br>
5,792,462; 5,739,026; 5;789,245 and 5,814,482). The construction of the<br>
alphaviral scaffold of the invention may be done by means generally known in the<br>
art of recombinant DNA technology, as described by the aforementioned articles,<br>
which are incorporated herein by reference.<br>
A variety of different recombinant host cells can be utilized to produce a<br>
viral-based core particle for antigen or antigenic determinant attachment. For<br>
example, Alphaviruses are known to have a wide host range: Sindbis virus infects<br>
cultured mammalian, reptilian, and amphibian cells, as well as some insect cells<br>
(Clark, H., J. Natl. Cancer Inst. 57:645 (1973); Leake, C.,J. Gen. Virol. 35:335<br>
(1977); Stollar, V. in THE TOGA VIRUSES, R. W. Schlesinger, Ed., Academic Press,<br>
(1980), pp.583-621). Thus, numerous recombinant host cells can be used in the<br>
practice of the invention. BHK, COS, Vero, HeLa and CHO cells are particularly<br>
suitable for the production of heterologous proteins because they have the<br>
potential to glycosylate heterologous proteins in a manner similar to human cells<br>
(Watson, E. et al, Glycobiology 4:227, (1994)) and can be selected (Zang, M. et<br>
al, Bio/Technology 73:389 (1995)) or genetically engineered (Renner W. et al,<br>
Biotech. Bioeng. 4:476 (1995); Lee K. et al. Biotech. Bioeng. 50:336 (1996)) to<br>
grow in serum-free medium, as well as in suspension.<br>
Introduction of the polynucleotide vectors into host cells can be effected<br>
by methods described in standard laboratory manuals (see, e.g., Sambrook, J. et<br>
al. eds., MOLECULAR CLONING, A LABORATORY MANUAL. 2nd. edition, Cold<br>
Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y. (1989), Chapter 9;<br>
Ausubel, F. et al, eds.. CURRENT PROTOCOLS IN MOLECULAR BIOLOGY, John H.<br>
Wiley &amp; Sons, Inc. (1997), Chapter 16), including methods such as<br>
electroporation, DEAE-dextran mediated transfection. transfection,<br>
microinjection, cationic lipid-mediated transfection, transduction, scrape loading,<br>
ballistic introduction, and infection. Methods for the introduction of exogenous<br><br>
DNA sequences into host cells are discussed in Feigner, P. et al., U.S. Patent No.<br>
5,580,859.<br>
Packaged RNA sequences can also be used to infect host cells. These<br>
packaged RNA sequences can be introduced to host cells by adding them to the<br>
culture medium. For example, the preparation of non-infective alpahviral<br>
particles is described in a number of sources, including "Sindbis Expression<br>
System", Version C, (Invitrogen Catalog No. K750-1).<br>
When mammalian cells are used as recombinant host cells for the<br>
production of viral-based core particles, these cells will generally be grown in<br>
tissue culture. Methods for growing cells in culture are well known in the art<br>
(see, e.g., Cells, J., ed., CELL BIOLOGY, Academic Press, 2nd edition, (1998);<br>
Sambrook, J. et al., eds., MOLECULAR CLONING, A LABORATORY MANUAL, 2nd.<br>
edition, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y. (1989);<br>
Ausubel, F. et al., eds., CURRENT PROTOCOLS IN MOLECULAR BIOLOGY, John H.<br>
Wiley &amp; Sons, Inc. (1997); Freshney, R., CULTURE OF ANIMAL CELLS, Alan R.<br>
Liss, Inc. (1983)).<br>
As will be understood by those in the art, the first attachment site may be<br>
or be a part of any suitable protein, polypeptide, sugar, polynucleotide, peptide<br>
(amino acid), natural or synthetic polymer, a secondary metabolite or combination<br>
thereof that may serve to specifically attach the antigen or antigenic determinant<br>
of choice to the scaffold. In one embodiment, the attachment site is a protein or<br>
peptide that may be selected from those known in the art. For example, the first<br>
attachment site may selected from the following group: a ligand, a receptor, a<br>
lectin, avidin, streptavidin, biotin, an epitope such as an HA or T7 tag, Myc, Max,<br>
immunoglobulin domains and any other amino acid sequence known in the art<br>
that would be useful as a first attachment site.<br>
It should be further understood by those in the art that with another<br>
embodiment of the invention, the first attachment site may be created secondarily<br>
to the organizer (i.e., protein or polypeptide) utilized in constructing the in-frame<br>
fusion to the capsid protein. For example, a protein may be utilized for fusion to<br><br>
the envelope protein with an amino acid sequence known to be glycosylated in a<br>
specific fashion, and the sugar moiety added as a result may then serve at the first<br>
attachment site of the viral scaffold by way of binding to a lectin serving as the<br>
secondary attachment site of an antigen. Alternatively, the organizer sequence<br>
may be biotinylated in vivo and the biotin moiety may serve as the first<br>
attachment site of the invention, or the organizer sequence may be subjected to<br>
chemical modification of distinct amino acid residues in vitro, the modification<br>
serving as the first attachment site.<br>
One specific embodiment of the invention utilizes the Sinbis virus. The<br>
Sinbis virus RNA genome is packaged into a capsid protein that is surrounded by<br>
a lipid bilayer containing three proteins called El, E2, and E3. These so-called<br>
envelope proteins are glycoproteins, and the glycosylated portions are located on<br>
the outside of the lipid bilayer, where complexes of these proteins form the<br>
"spikes" that can be seen in electron micrographs to project outward from the<br>
surface of the virus. In a preferred embodiment of the invention, the first<br>
attachment site is selected to be the JUN or FOS leucine zipper protein domain<br>
that is fused in frame to the E2 envelope protein. However, it will be clear to all<br>
individuals in the art that other envelope proteins may be utilized in the fusion<br>
protein construct for locating the first attachment site in the scaffold of the<br>
invention.<br>
In a most preferred embodiment of the invention, the first attachment site<br>
is selected to be the JUN-FOS leucine zipper protein domain that is fused in<br>
frame to the Hepatitis B capsid (core) protein. However, it will be clear to all<br>
individuals in the art that other viral capsid proteins may be utilized in the fusion<br>
protein construct for locating the first attachment site in the scaffold of the<br>
invention.<br>
In another preferred embodiment of the invention, the first attachment site<br>
is selected to be a lysine or cysteine residue that is fused in frame to the Hepatitis<br>
core (capsid) protein. However, it will be clear to all individuals in the art that<br><br>
other viral capsid or virus-like particles may be utilized in the fusion protein<br>
construct for locating the first attachment in the scaffold of the invention.<br>
Example 1 is provided to demonstrate the construction of an in-frame<br>
fusion protein between the Sinbis virus E2 envelope protein and the JUN leucine<br>
zipper protein domain using the pTE5'2J vector of Hahn et al. (JProc. Natl. Acad.<br>
Sci. USA 89:2679-2683 (1992)). The JUN amino acid sequence utilized for the<br>
first attachment site is the following:<br>
CGGRIARLEEKVKTLKAQNSELASTANMLREQVAQLKQKVMNHVGC<br>
(SEQ ID NO:59)<br>
In this instance, the anticipated second attachment site on the antigen would be<br>
the FOS leucine zipper protein domain and the amino acid sequence would be the<br>
following:<br>
CGGLTDTLQAETDQVEDEKSALQTEIANLLKEKEKLEFILAAHGGC(SEQ<br>
ID NO:60)<br>
These sequences are derived from the transcription factors JUN and FOS,<br>
each flanked with a short sequence containing a cysteine residue on both sides.<br>
These sequences are known to interact with each other. The original hypothetical<br>
structure proposed for the JUN-FOS dimer assumed that the hydrophobic side<br>
chains of one monomer interdigitate with the respective side chains of the other<br>
monomer in a zipper-like manner (Landschulz et al, Science 240:1759-1764<br>
(1988)). However, this hypothesis proved to be wrong, and these proteins are<br>
known to form an -helical coiled coil (O'Shea et al., Science 243:538-542<br>
(1989); O'Shea et al, Cell 65:699-708 (1992); Cohen &amp; Parry, Trends Biochem.<br>
Sci. 17:245-248 (1986)), Thus, the term "leucine zipper" is frequently used to<br>
refer to these protein domains for more historical than structural reasons.<br>
Throughout this patent, the term "leucine zipper" is used to refer to the sequences<br>
depicted above or sequences essentially similar to the ones depicted above. The<br>
terms JUN and. FOS are used for the respective leucine zipper domains rather than<br>
the entire JUN and FOS proteins.<br><br>
In one embodiment, the invention provides for the production of a Sinbis<br>
virus E2-JUN scaffold using the pCYTts expression system (US Patent<br>
Application Appl. No. 60/079,562; Filed March 27,1998). The pCYTts<br>
expression system provides novel expression vectors which permit tight<br>
regulation of gene expression in eucaryotic cells. The DNA vectors of this<br>
system are transcribed to form RNA molecules which are then replicated by a<br>
temperature-sensitive replicase to form additional RNA molecules. The RNA<br>
molecules produced by replication contain a nucleotide sequence which may be<br>
translated to produce a protein of interest or which encode one or more<br>
untranslated RNA molecules. Thus the expression system enables the production<br>
of recombinant Sinbis virus particles.<br>
Example 2 provides details on the production of the E2- JUN Sinbis non-<br>
natural, molecular scaffold of the invention. Additionally provided in Example<br>
3 is another method for the production of recombinant E2-JUN Sinbis virus<br>
scaffold using the pTE5'2JE2:JLW vector produced in Example 1. Thus the<br>
invention provides two means, the pCYTts expression system (Example 2) and<br>
the pTE5'2J vector system (Example 3) by which recombinant Sinbis virus E2-<br>
JUN non-natural, molecular scaffold may be produced. An analysis of viral<br>
particles produced in each system is proved in Figure 1 and Figure 2.<br>
As previously stated, the invention includes viral-based core particles<br>
which comprise a virus, virus-like particle, a phage, a viral capsid particle or a<br>
recombinant form thereof. Skilled artisans have the knowledge to produce such<br>
core particles and attach organizers thereto. By way of providing other examples,<br>
the invention provides herein for the production of hepatitis B virus-like particles<br>
and measles viral capsid particles as core particles (Examples 17 to 22). In such<br>
an embodiment, the JUN leucine zipper protein domain or FOS leucine zipper<br>
protein domain may be used as an organizer, and hence as a first attachment site,<br>
for the non-natural molecular scaffold of the invention.<br>
Examples 23-29 provide details of the production of Hepatitis B core<br>
particles carrying an in-frame fused peptide with a reactive lysine residue and<br><br>
antigens carrying a genetically fused cysteine residue, as first and second<br>
attachment site, respectively.<br>
B. Construction of an Antigen or Antigenic Determinant with a<br>
Second Attachment Site<br>
The second element in the composition of the invention is an antigen or<br>
antigenic determinant possessing at least one second attachment site capable of<br>
association through at least one non-peptide bond to the first attachment site of<br>
the non-natural molecular scaffold. The invention provides for compositions that<br>
vary according to the antigen or antigenic determinant selected in consideration<br>
of the desired therapeutic effect. Other compositions are provided by varying the<br>
molecule selected for the second attachment site.<br>
Antigens of the invention may be selected from the group consisting of the<br>
following: (a) proteins suited to induce an immune response against cancer cells;<br>
(b) proteins suited to induce an immune response against infectious diseases; (c)<br>
proteins suited to induce an immune response against allergens, and (d) proteins<br>
suited to induce an immune response in farm animals.<br>
In one specific embodiment of the invention, the antigen or antigenic<br>
determinant is one that is useful for the prevention of infectious disease. Such<br>
treatment will be useful to treat a wide variety of infectious diseases affecting a<br>
wide range of hosts, e.g., human, cow, sheep, pig, dog, cat. other mammalian<br>
species and non-mammalian species as well. Treatable infectious diseases are<br>
well known to those skilled in the art, examples include infections of viral<br>
etiology such as HIV, influenza, Herpes, viral hepatitis, Epstein Bar, polio, viral<br>
encephalitis, measles, chicken pox, etc.; or infections of bacterial etiology such<br>
as pneumonia, tuberculosis, syphilis, etc.; or infections of parasitic etiology such<br>
as malaria, trypanosomiasis, leishmaniasis, trichomoniasis, amoebiasis, etc.<br>
Thus, antigens or antigenic determinants selected for the compositions of the<br>
invention will be well known to those in the medical art; examples of antigens or<br>
antigenic determinants include the following: the HIV antigens gp 140 and gp 160;<br><br>
the influenaza antigens hemagglutinin and neuraminidase. hepatitis B surface<br>
antigen, circumsporozoite protein of malaria.<br>
In another specific embodiment, the compositions of the invention are an<br>
immunotherapeutic that may be used for the treatment of allergies or cancer.<br>
The selection of antigens or antigenic determinants for the composition<br>
and method of treatment for allergies would be known to those skilled in the<br>
medical art treating such disorders; representative examples of this type of<br>
antigen or antigenic determinant include the following: bee venom phospholipase<br>
A2, Bet v I (birch pollen allergen), 5 Dol m V (white-faced hornet venom<br>
allergen), Der p I (House dust mite allergen).<br>
The selection of antigens or antigenic determinants for the composition<br>
and method of treatment for cancer would be known to those skilled in the<br>
medical art treating such disorders; representative examples of this type of<br>
antigen or antigenic determinant include the following: Her2 (breast cancer),<br>
GD2 (neuroblastoma), EGF-R (malignant glioblastoma), CE A (medullary thyroid<br>
cancer), CD52 (leukemia).<br>
In a particular embodiment of the invention, the antigen or antigenic<br>
determinant is selected from the group consisting of: (a) a recombinant protein<br>
of HIV, (b) a recombinant protein of Influenza virus, (c) a recombinant protein<br>
of Hepatitis C virus, (d) a recombinant protein of Toxoplasma, (e) a recombinant<br>
protein of Plasmodium falciparum, (f) a recombinant protein of Plasmodium<br>
vivax, (g) a recombinant protein of Plasmodium ovale, (h) a recombinant protein<br>
of Plasmodium malariae, (i) a recombinant protein of breast cancer cells, (j) a<br>
recombinant protein of kidney cancer cells, (k) a recombinant protein of prostate<br>
cancer cells, (1) a recombinant protein of skin cancer cells, (m) a recombinant<br>
protein of brain cancer cells, (n) a recombinant protein of leukemia cells, (o) a<br>
recombinant profiling, (p) a recombinant protein of bee sting allergy, (q) a<br>
recombinant proteins of nut allergy, (r) a recombinant proteins of food allergies,<br>
recombinant proteins of asthma, and a recombinant protein of Chlamydia.<br><br>
Once the antigen or antigenic determinant of the composition is chosen,<br>
at least one second attachment site may be added to the molecule in preparing to<br>
construct the organized and repetitive array associated with the non-natural<br>
molecular scaffold of the invention. Knowledge of what will constitute an<br>
appropriate second attachment site will be known to those skilled in the art.<br>
Representative examples of second attachment sites include, but are not limited<br>
to, the following: an antigen, an antibody or antibody fragment, biotin, avidin,<br>
strepavidin, a receptor, a receptor ligand, a ligand, a ligand-binding protein, an<br>
interacting leucine zipper polypeptide, an amino group, a chemical group reactive<br>
to an amino group; a carboxyl group, chemical group reactive to a carboxyl<br>
group, a sulfhydryl group, a chemical group reactive to a sulfhydryl group, or a<br>
combination thereof.<br>
The association between the first and second attachment sites will be<br>
determined by the characteristics of the respective molecules selected but will<br>
comprise at least one non-peptide bond. Depending upon the combination of first<br>
and second attachment sites, the nature of the association may be covalent, ionic,<br>
hydrophobic, polar, or a combination thereof.<br>
In one embodiment of the invention, the second attachment site may be<br>
the FOS leucine zipper protein domain or the JLWleucine zipper protein domain.<br>
In a most specific embodiment of the invention, the second attachment<br>
site selected is the FOS leucine zipper protein domain, which associates<br>
specifically with the JUN leucine zipper protein domain of the non-natural<br>
molecular scaffold of the invention. The association of the JUN and FOS leucine<br>
zipper protein domains provides a basis for the formation of an organized and<br>
repetitive antigen or antigenic determinant array on the surface of the scaffold.<br>
The FOS leucine zipper protein domain may be fused in frame to the antigen or<br>
antigenic determinant of choice at either the amino terminus, carboxyl terminus<br>
or internally located in the protein if desired.<br><br>
Several FOS fusion constructs are provided for exemplary purposes.<br>
Human growth hormone (Example 4), bee venom phospholipase A2 (PLA)<br>
(Example 9), ovalbumin (Example 10) and HIV gpl40 (Example 12).<br>
In order to simplify the generation of FOS fusion constructs, several<br>
vectors are disclosed that provide options for antigen or antigenic determinant<br>
design and construction (see Example 6). The vectors pAV 1 -4 were designed for<br>
the expression of FOS fusion in K coli; the vectors pAV5 and pAV6 were<br>
designed for the expression of FOS fusion proteins in eukaryotic cells. Properties<br>
of these vectors are briefly described:<br>
1. pAVl: This vector was designed for the secretion of fusion proteins<br>
with FOS at the C-terminus into the E. coli periplasmic space. The gene of<br>
interest (g.o.i.) may be Iigated into the StuI/NotI sites of the vector.<br>
2.	p,AV2: This vector was designed for the secretion of fusion proteins<br>
with FOS at the N-terminus into the E. coli periplasmic space. The gene of<br>
interest (g.o.i.) Iigated into the Notl/EcoRV (or Notl/HindlH) sites of the vector.<br>
3.	pAV3: This vector was designed for the cytoplasmic production of<br>
fusion proteins with FOS at the C-terminus in E. coli. The gene of interest (g.o.i.)<br>
may be Iigated into the EcoRV/NotI sites of the vector.<br>
4. pAV4: This vector is designed for the cytoplasmic production of<br>
fusion proteins with FOS at the N-terminus in E. coli. The gene of interest (g.o.i.)<br>
may be Iigated into the Notl/EcoRV (or Notl/HindlH) sites of the vector. The<br>
N-terminal methionine residue is proteolytically removed upon protein synthesis<br>
(Hirel et al., Proc. Nail. Acad. Sci. USA 5(5:8247-8251 (1989)).<br>
5. pAV.S: This vector was designed for the eukaryotic production of<br>
fusion proteins with FOS at the C-terminus. The gene of interest (g.o.i.) may be<br>
inserted between the sequences coding for the hGH signal sequence and the FOS<br>
domain by ligation into the Eco47III/NotI sites of the vector. Alternatively, a<br>
gene containing its own signal sequence may be fused to the FOS coding region<br>
by ligation into the StuI/NotI sites.<br><br>
6. pAV6: This vector was designed for the eukaryotic production of<br>
fusion proteins with FOS at the N-terminus. The gene of interest (g.o.i.) may be<br>
ligated into the Notl/StuI (or Notl/Hindlll) sites of the vector.<br>
As will be understood by those skilled in the art, the construction of a<br>
FOS-antigen or -antigenic determinant fusion protein may include the addition of<br>
certain genetic elements to facilitate production of the recombinant protein.<br>
Example 4 provides guidance for the addition of certain E. coli regulatory<br>
elements for translation, and Example 7 provides guidance for the addition of a<br>
eukaryotic signal sequence. Other genetic elements may be selected, depending<br>
on the specific needs of the practioner.<br>
The invention is also seen to include the production of the FOS-antigen<br>
or FOiS-antigenic determinant fusion protein either in bacterial (Example 5) or<br>
eukaryotic cells (Example 8). The choice of which cell type in which to express<br>
the fusion protein is within the knowledge of the skilled artisan, depending on<br>
factors such as whether post-translational modifications are an important<br>
consideration in the design of the composition.<br>
As noted previously, the invention discloses various methods for the<br>
construction of a FOS-anligzn or FOS-antigenic determinant fusion protein<br>
through the use of the pAV vectors. In addition to enabling prokaryotic and<br>
eukaryotic expression, these vectors allow the practitioner to choose between —<br>
and C- terminal addition to the antigen of the FOS leucine zipper protein domain.<br>
Specific examples are provided wherein — and C- terminal FOS fusions are made<br>
to PLA (Example 9) and ovalbumin (Example 10). Example 11 demonstrates the<br>
purification of the PLA and ovalbumin FOS fusion proteins.<br>
In a most specific embodiment, the invention is drawn to an antigen or<br>
antigenic determinant encoded by the HIV genome. More specifically, the HIV<br>
antigen is gpl40. As provided for in Examples 11-15, HIV gp 140 may be created<br>
with a FOS leucine zipper protein domain and the fusion protein synthesized and<br>
purified for attachment to the non-natural molecular scaffold of the invention. As<br><br>
one skilled in the art would know, other HIV antigens or antigenic determinants<br>
may be used in the creation of a composition of the invention.<br>
In a most specific embodiment of the invention, the second attachment<br>
site selected is a cysteine residue, which associates specifically with a lysine<br>
residue of the non-natural molecular scaffold of the invention. The chemical<br>
linkage of the lysine residue (Lys) and cysteine residue (Cy s) provides a basis for<br>
the formation of an organized and repetitive antigen or antigenic determinant<br>
array on the surface of the scaffold. The cysteine residue may be engineered in<br>
frame to the antigen or antigenic determinant of choice at either the amino<br>
terminus, carboxyl terminus or internally located in the protein if desired. By way<br>
of example, PL A and HIV gpl40 are provided with a cysteine residue for linkage<br>
to a lysine residue first attachment site.<br>
C. Preparation of the AlphaVaccine Particles<br>
The invention provides novel compositions and methods for the<br>
construction of ordered and repetitive antigen arrays. As one of skill in the art<br>
would know, the conditions for the assembly of the ordered and repetitive antigen<br>
array depend to a large extent on the specific choice of the first attachment site<br>
of the non-natural scaffold and the specific choice of the second attachment site<br>
of the antigen or antigenic determinant. Thus, practitioner choice in the design<br>
of the composition (/. e., selection of the first and second attachment sites, antigen<br>
and non-natural scaffold) will determine the specific conditions for the assembly<br>
of the AlphaVaccine particle (the ordered and repetitive antigen array and non-<br>
natural molecular scaffold combined). Information relating to assembly of the<br>
AlphaVaccine particle is well within the working knowledge of the practitioner,<br>
and numerous references exist to aid the practitioner (e.g.. Sambrook, J. et al,<br>
eds.. MOLECULAR CLONING, A LABORATORY MANUAL, 2nd. edition, Cold Spring<br>
Harbor Laboratory Press, Cold Spring Harbor, N.Y. (1989); Ausubel, F. et al.,<br>
eds.. CURRENT PROTOCOLS IN MOLECULARBIOLOGY, John H. Wiley &amp; Sons, Inc.<br><br>
(1997); Celis, J., ed., CELL BIOLGY, Academic Press. 2nd edition, (1998);<br>
Harlow, E. and Lane, D., "Antibodies: A Laboratory Manual," Cold Spring<br>
Harbor Laboratory, Cold Spring Harbor, N.Y. (1988). all of which are<br>
incorporated herein by reference.<br>
In a specific embodiment of the invention, the JUN and FOS leucine<br>
zipper protein domains are utilized for the first and second attachment sites of the<br>
invention, respectively. In the preparation of AlphaVaccine particles, antigen<br>
must be produced and purified under conditions to promote assembly of the<br>
ordered and repetitive antigen array onto the non-natural scaffold. In the<br>
particular JUN/FOS leucine zipper protein domain embodiment, the FOS-antigen<br>
or FOS-antigenic determinant should be treated with a reducing agent (e.g.,<br>
Dithiothreitol (DTT)) to reduce or eliminate the incidence of disulfide bond<br>
formation (Example 15).<br>
For the preparation of the non-natural scaffold (i.e.. recombinant Sinbis<br>
virus) of the JUN/FOS leucine zipper protein domain embodiment, recombinant<br>
H2-JUN viral particles should be concentrated, neutralized and treated with<br>
reducing agent (see Example 16).<br>
Assembly of the ordered and repetitive antigen array in the JUN/FOS<br>
embodiment is done in the presence of a redox shuffle. E2-JUN viral particles<br>
are combined with a 240 fold molar excess of FOS-antigen or FOS-antigenic<br>
determinant for 10 hours at 4°C. Subsequently, the AlphaVaccine particle is<br>
concentrated and purified by chromatography (Example 16).<br>
In another embodiment of the invention, the coupling of the non-natural<br>
molecular scaffold to the antigen or antigenic determinant may be accomplished<br>
by chemical crosslinking. In a most preferred embodiment, the chemical agent<br>
is a hetero-bifunctional crosslinking agent such as e-maleimidocaproic acid N-<br>
hydroxysuccinimide ester (Tanimori et al, J. Pharm. Dyn. 4:812 (1981);<br>
Fujiwara et al., J. Immunol. Meth. 45:195 (1981), which contains (1) a<br>
succinimide group reactive with amino groups and (2) a maleimide group reactive<br>
with SH groups.. A heterologous protein or polypeptide of the first attachment<br><br>
site may be engineered to contain one or more lysine residues that will serve as<br>
a reactive moiety for the succinimide portion of the hetero-bifunctional<br>
crosslinking agent. Once chemically coupled to the first attachment sites of the<br>
non-natural molecular scaffold, the maleimide group of the hetero-bifunctional<br>
crosslinking agent will be available to react with the SH group of a cysteine<br>
residue on the antigen or antigenic determinant. Antigen or antigenic determinant<br>
preparation in this instance may require the engineering of a cysteine residue into<br>
the protein or polypeptide chosen as the second attachment site so that it may be<br>
reacted to the free maleimide function on the crosslinking agent bound to the non-<br>
natural molecular scaffold first attachment sites.<br>
3.	Compositions, Vaccines, and the Administration Thereof, and Methods<br>
of Treatment<br>
In one embodiment, the invention provides vaccines for the prevention of<br>
infectious diseases in a wide range of species, particularly mammalian species<br>
such as human, monkey, cow, dog, cat, horse, pig, etc. Vaccines may be designed<br>
to treat infections of viral etiology such as HIV, influenza, Herpes, viral hepatitis,<br>
Epstein Bar, polio, viral encephalitis, measles, chicken pox, etc.; or infections of<br>
bacterial etiology such as pneumonia, tuberculosis, syphilis, etc.; or infections of<br>
parasitic etiology such as malaria, trypanosomiasis, leishmaniasis, trichomoniasis,<br>
amoebiasis, etc.<br>
In another embodiment, the invention provides vaccines for the<br>
prevention of cancer in a wide range of species, particularly mammalian species<br>
such as human, monkey, cow, dog, cat, horse, pig, etc. Vaccines may be designed<br>
to treat all types of cancer: lymphomas, carcinomas, sarcomas, melanomas, etc.<br>
In another embodiment of the invention, the compositions of the invention<br>
may be used in the design of vaccines for the treatment of allergies. Antibodies<br>
of the IgE isotype are important components in allergic reactions. Mast cells bind<br>
IgE antibodies on their surface and release histamines and other mediators of<br>
allergic response upon binding of specific antigen to the IgE molecules bound on<br><br>
the mast cell surface. Inhibiting production of IgE antibodies, therefore, is a<br>
promising target to protect against allergies. This should be possible by attaining<br>
a desired T helper cell response. T helper cell responses can be divided into type<br>
1 (TH1) and type 2 (TH2) T helper cell responses (Romagnani, Immunol. Today<br>
18:263-266 (1997)). TH1 cells secrete interferon-gamma and other cytokines<br>
which trigger B cells to produce IgG 1 -3 antibodies. In contrast, a critical cytokine<br>
produced by TH2 cells is IL-4, which drived B cells to produce IgG4 and IgE. In<br>
many experimental systems, the development of TH1 and TH2 responses is<br>
mutually exclusive sinceTHl cells suppress the induction of TH2 cells and vice<br>
versa. Thus, antigens that trigger a strong TH1 response simultaneously suppress<br>
the development of TH2 responses and hence the production of IgE antibodies.<br>
Interestingly, virtually all viruses induce a TH1 response in the host and fail to<br>
trigger the production of IgE antibodies (Coutelier et al.,J. Exp. Med. 165:64-69<br>
(1987)). This isotype pattern is not restricted to live viruses but has also been<br>
observed for inactivated or recombinant viral particles (Lo-Man et al., Eur. J.<br>
Immunol. 25:1401-1407 (1998)). Thus, by using the processes of the invention<br>
{e.g., AlphaVaccine Technology), viral particles can be decorated with various<br>
allergens and used for immunization. Due to the resulting "viral structure" of the<br>
allergen, a TH1 response will be elicited, "protective" IgG 1-3 antibodies will be<br>
produced, and the production of IgE antibodies which cause allergic reactions will<br>
be prevented. Since the allergen is presented by viral particles which are<br>
recognized by a different set of helper T cells than the allergen itself, it is likely<br>
that the allergen-specific IgGl -3 antibodies will be induced even in allergic<br>
individuals harboring pre-existing TH2 cells specific for the allergen. The<br>
presence of high concentrations of IgG antibodies may prevent binding of<br>
allergens to mast cell bound IgE, thereby inhibiting the release of histamine.<br>
Thus, presence of IgG antibodies may protect from IgE mediated allergic<br>
reactions. Typical substances causing allergies include: grass, ragweed, birch or<br>
mountain cedar pollens, house dust, mites, animal danders, mold, insect venom<br>
or drugs {e.g. penicillin). Thus, immunization of individuals with allergen-<br><br>
decorated viral particles should be beneficial not only before but also after the<br>
onset of allergies.<br>
As would be understood by one of ordinary skill in the art, when the<br>
compositions of the invention are administered to an individual, they may be in<br>
a composition which contains salts, buffers, adjuvants, or other substances which<br>
are desirable for improving the efficacy of the composition. Examples of<br>
materials suitable for use in preparing pharmaceutical compositions are provided<br>
in numerous sources including REMINGTON'S PHARMACEUTICAL SCIENCES (Osol,<br>
A, ed., Mack Publishing Co., (1980)).<br>
The compositions of the invention are said to be "pharmacologically<br>
acceptable" if their administration can be tolerated by a recipient individual.<br>
Further, the compositions of the invention will be administered in a<br>
"therapeutically effective amount" (i.e., an amount that produces a desired<br>
physiological effect).<br>
The compositions of the present invention may be administered by various<br>
methods known in the art, but will normally be administered by injection,<br>
infusion, inhalation, oral administration, or other suitable physical methods. The<br>
compositions may alternatively be administered intramuscularly, intravenously,<br>
or subcutaneously. Components of compositions for administration include<br>
sterile aqueous (e.g., physiological saline) or non-aqueous solutions and<br>
suspensions. Examples of non-aqueous solvents are propylene glycol,<br>
polyethylene glycol, vegetable oils such as olive oil, and injectable organic esters<br>
such as ethyl oleate. Carriers or occlusive dressings can be used to increase skin<br>
permeability and enhance antigen absorption.<br>
In addition to vaccine technologies, other embodiments of the invention<br>
are drawn to methods of medical treatment for cancer and allergies.<br>
All patents and publications referred to herein are expressly incorporated<br>
by reference.<br><br>
Examples<br>
Enzymes and reagents used in the experiments that follow included: T4<br>
DNA ligase obtained from New England Biolabs; Taq DNA Polymerase,<br>
QIAprep Spin Plasmid Kit, QIAGEN Plasmid Midi Kit, QiaExII Gel Extraction<br>
Kit, QlAquick PCR Purification Kit obtained from QIAGEN; QuickPrep Micro<br>
mRNA Purification Kit obtained from Pharmacia; Superscript One-step RT PCR<br>
Kit, fetal calf serum (FCS), bacto-tryptone and yeast extract obtained from Gibco<br>
BRL; Oligonucleotides obtained from Microsynth (Switzerland); restriction<br>
endonucleases obtained from Boehringer Mannheim, New England Biolabs or<br>
MBI Fermentas; Pwo polymerase and dNTPs obtained from Boehringer<br>
Mannheim. HP-1 medium was obtained from Cell culture technologies<br>
(Glattbrugg, Switzerland). All standard chemicals were obtained from<br>
Fluka-Sigma-Aldrich, and all cell culture materials were obtained from TPP.<br>
DNA manipulations were carried out using standard techniques. DNA<br>
was prepared according to manufacturer instruction either from a 2 ml bacterial<br>
culture using the QIAprep Spin Plasmid Kit or from a 50 ml culture using the<br>
QIAGEN Plasmid Midi Kit. For restriction enzyme digestion, DNA was<br>
incubated at least 2 hours with the appropriate restriction enzyme at a<br>
concentration of 5-10 units (U) enzyme per mg DNA under manufacturer<br>
recommended conditions (buffer and temperature). Digests with more than one<br>
enzyme were performed simultaneously if reaction conditions were appropriate<br>
for all enzymes, otherwise consecutively. DNA fragments isolated for further<br>
manipulations were separated by electrophoresis in a 0.7 to 1.5% agarose gel,<br>
excised from the gel and purified with the QiaExII Gel Extraction Kit according<br>
to the instructions provided by the manufacturer. For ligation of DNA<br>
fragments, 100 to 200 pg of purified vector DNA were incubated overnight with<br>
a threefold molar excess of the insert fragment at 16 °C in the presence of 1 U T4<br>
DNA ligase in the buffer provided by the manufacturer (total volume: 10-20 l).<br>
An aliquot (0.1 to 0.5 l) of the ligation reaction was used for transformation of<br><br>
E. coli XL 1 -Blue (Stratagene). Transformation was done by electroporation using<br>
a Gene Pulser (BioRAD) and 0.1 cm Gene Pulser Cuvettes (BioRAD) at 200 Q,<br>
25F, 1.7 kV. After electroporation, the cells were incubated with shaking for<br>
1 h in 1 ml S.O.B. medium (Miller, 1972) before plating on selective S.O.B. agar.<br>
Example 1:<br>
Insertion of the JUN amphiphatic helix domain within E2<br>
In the vector pTE5'2J (Hahn et al, Proc. Nad. Acad. Sci. USA<br>
89:2679-2683, (1992)), Mlul and a BstEIl restriction enzyme sites were<br>
introduced between codons 71 (Gin) and 74 (Thr) of the structural protein E2<br>
coding sequence, resulting in vector pTE5"2JBM. Introduction of these<br>
restriction enzymes sites was done by PCR using the following oligonucleotides:<br>
Oligo 1:<br>
E2insBstEII/BssHII:<br>
5'-ggggACGCGTGCAGCAggtaaccaccgTTAAAGAAGGCACC-3' (SEQ ID<br>
NO:1)<br>
Oligo 2:<br>
E2insMluIStuI:<br>
5 '-cggtggttaccTGCTGCACGCGTTGCTTAAGCGACATGTAGCGG-3' (SEQ<br>
ID NO:2)<br>
Oligo 3:<br>
E2insStuI: 5'-CCATGAGGCCTACGATACCC-3' (SEQ ID NO:3)<br>
Oligo4:<br>
E2insBssHII: 5'-GGCACTCACGGCGCGCTTTACAGGC-3' (SEQ ID NO:4)<br>
For the PCR reaction, 100 pmol of each oligo was used with 5 ng of the<br>
template DNA in a 100 1 reaction mixture containing 4 units of Taq or Pwo<br>
polymerase, 0.1 mM dNTPs and 1.5 mM MgSO4. All DNA concentrations were<br>
determined photometrically using the GeneQuant apparatus (Pharmacia).<br>
Polymerase was added directly before starting the PCR reaction (starting point<br>
was 95 °C). Temperature cycling was done in the following manner and order:<br>
95 °C for 2 minutes: 5 cycles of 95 °C (45 seconds), 53 °C (60 seconds), 72°C<br><br>
(80 seconds); and 25 cycles of 95°C (45 seconds), 57°C (60 seconds), 72°C<br>
(80 seconds).<br>
The two PCR fragments were analyzed and purified by agarose<br>
gelelectrophoresis. Assembly PCR of the two PCR fragments using oligo 3 and<br>
4	for amplification was carried out to obtain the final construct.<br>
For the assembly PCR reaction, 100 pmol of each oligo was used with<br>
2 ng of the purified PCR fragments in a 100 l reaction mixture containing 4<br>
units of Taq or Pwo polymerase, 0.1 mM dNTPs and 1.5 mM MgSO4. All DNA<br>
concentrations were determined photometrically using the GeneQuant apparatus<br>
(Pharmacia). Polymerase was added directly before starting the PCR reaction<br>
(starting point was 95 °C). Temperature cycling was done in the following<br>
manner and order: 95°C for 2 minutes; 5 cycles of 95°C (45 seconds), 57°C<br>
(60 seconds), 72°C (90 seconds); and 25 cycles of 95°C (45 seconds), 59°C<br>
(60 seconds), 72°C (90 seconds).<br>
The final PCR product was purified using Qia spin PCR columns<br>
(Qiagen) and digested in an appropriate buffer using 10 units each of BssHII and<br>
StuI restriction endonucleases for 12 hours at 37°C. The DNA fragments were<br>
gel-purified and ligated into BssHII/StuI digested and gel-purified pTE5 '2J vector<br>
(Hahn et al., Proc. Natl. Acad. Sci. USA SP:2679-2683). The correct insertion of<br>
the PCR product was first analyzed by BstEII and MM restriction analysis and<br>
then by DNA sequencing of the PCR fragment.<br>
The DNA sequence coding for the JUN amphiphatic helix domain was<br>
PCR-amplified from vector pJuFo (Crameri and Suter, Gem 137:69 (1993))<br>
using the following oligonucleotides:<br>
Oligo 5:<br>
JLiVBstEII:<br>
5	'-CCTTCTTTAAcggtggttaccTGCTGGCAACCAACGTGGTTCATGAC-3'<br>
(SEQ ID NO:5)<br>
Oligo 6:<br>
MluUUN: 5 '-AAGCATGCTGCacgcgtgTGCGGTGGTCGGATCGCCCGGC-3'<br>
(SEQ ID NO:6)<br><br>
For the PCR reaction, 100 pmol of each oligo was used with 5 ng of the<br>
template DNA in a 100 ul reaction mixture containing 4 units of Taq or Pwo<br>
polymerase, 0.1 mM dNTPs and 1.5 mM MgSO4. All DNA concentrations were<br>
determined photometrically using the GeneQuant apparatus (Pharmacia).<br>
Polymerase was added directly before starting the PCR reaction (starting point<br>
was 95°C). Temperature cycling was done in the following order and manner:<br>
95°C for 2 minutes; 5 cycles of 95°C (45 seconds), 60°C (30 seconds), 72°C<br>
(25 seconds); and 25 cycles of 95°C (45 seconds), 68°C (30 seconds), 72°C<br>
(20 seconds).<br>
The final PCR product was gel-purified and ligated into EcoRV digested<br>
and gel-purified pBIuescript II(KS"). From the resulting vector, the./LWsequence<br>
was isolated by cleavage with MluVBstEll purified with QiaExII and ligated into<br>
vector pTE5"2JBM (previously cut with the same restriction enzymes) to obtain<br>
the vector pTE5"2J:E2JUN.<br>
Example 2:<br>
Production ofviral particles containing E2-JUN using the pCYTts system<br>
The structural proteins were PCR amplified using pTE5'2J:E2JUN as<br>
template and the oligonucleotides XbalStruct<br>
(ctatcaTCTAGAATGAATAGAGGATTCTTTAAC) and StructBspl201<br>
(tcgaatGGGCCCTCATCTTCGTGTGCTAGTCAG). For the PCR 100 pmol of<br>
each loligo was used and 5 ng of the template DNA was used in the 100 ul<br>
reaction mixture, containing 4 units of Tac or Pwo polymerase, 0.1 mM dNTPs<br>
and 1.5 mM MgSO4. All DNA concentrations were determined photometrically<br>
using the GeneQuant apparatus (Pharmacia). The polymerase was added directly<br>
before starting the PCR reaction (starting point was 95 °C). The temperature<br>
cycles were as follows: 95°C for 3 minutes, followed by 5 cycles of 92°C (30<br>
seconds), 54°C (35 seconds), 72°C (270 seconds) and followed by 25 cycles of<br>
92°C (30 seconds), 63°C (35 seconds), 72°C (270 seconds. The PCR product<br>
was gel purified and digested with the restriction enzymes Xbal/Bspl201 and<br><br>
ligated into vector pCYTts previously cleaved with the same enzymes (US Patent<br>
Application Appl. No. 60/079,562; Filed March 27,1998)<br>
Twenty g of pCYTtsE2-JUN were incubated with 30 U of Seal in an<br>
appropriate buffer for at least 4 hours at 37°C. The reaction was stopped by<br>
phenol/chloroform extraction, followed by an isopropanol precipitation of the<br>
linerized DNA. The restriction reaction was checked by agarose gel<br>
eletrophoresis. For the transfection, 5.4 g of linearized pCYTtsE2'JUN was<br>
mixed with 0.6 ug of linearized pSV2Neo in 30 l H2O and 30 l of 1 M CaCl2<br>
solution were added. After addition of 60 p.1 phosphate buffer (50 mM HEPES,<br>
280 mM NaCl, 1.5 mM Na2 HPO4, pH 7.05), the solution was vortexed for 5<br>
seconds, followed by an incubation at room temperature for 25 seconds. The<br>
solution was immediately added to 2 ml HP-1 medium containing 2% FCS (2%<br>
FCS medium). The medium of an 80% confluent BHK21 cell culture in a 6-well<br>
plate was then replaced with the DNA containing medium. After an incubation<br>
for 5 hours at 37°C in a CO2 incubator, the DNA containing medium was<br>
removed and replaced by 2 ml of 15% glycerol in 2% FCS medium. The glycerol<br>
containing medium was removed after a 3 0 second incubation phase, and the cells<br>
were washed by rinsing with 5 ml of HP-1 medium containing 10% FCS. Finally<br>
2 ml of fresh HP-1 medium containing 10% FCS was added.<br>
Stably transfected cells were selected and grown in selection medium<br>
(HP-1 medium, supplemented with G418) at 3 7 ° C in a CO2 incubator. When the<br>
mixed population was grown to confiuency, the culture was split to two dishes,<br>
followed by a 12 hours growth period at 37°C. One dish of the cells was shifted<br>
to 30°C to induce the expression of the viral particles; the other dish was kept at<br>
37°C.<br>
The expression of viral particles was determined by Western blotting<br>
(Figure 1). Culture medium (0.5 ml) was methanol/chloroform precipitated, and<br>
the pellet was resuspended in SDS-PAGE sample buffer. Samples were heated<br>
for 5 minutes at 95°C before being applied to 15% acrylamide gel. After<br>
SDS-PAGE, proteins were transferred to Protan nitrocellulose membranes<br><br>
(Schleicher &amp; Schuell, Germany) as described by Bass and Yang, in Creighton,<br>
T.E., ed., Protein Function: A Practical Approach, 2nd Edn., IRL Press, Oxford<br>
(1997), pp. 29-55. The membrane was blocked with 1 % bovine albumin (Sigma)<br>
in TBS (1 OxTBS per liter: 87.7 g NaCl, 66. lg Trizma hydro chloride (Sigma) and<br>
9.7 g Trizma base (Sigma), pH 7.4) for 1 hour at room temperature, followed by<br>
an incubation with an anti-El/E2antibody (polyclonal serum) for 1 hour. The blot<br>
was washed 3 times for 10 minutes with TBS-T (TBS with 0.05% Tween20), and<br>
incubated for 1 hour with an alkaline-phosphatase-anti-rabbit IgG conjugate (0.1<br>
ug/ml, Amersham Life Science, England). After washing 2 times for 10 minutes<br>
with TBS-T and 2 times for 10 minutes with TBS, the development reaction was<br>
carried out using alkaline phosphatase detection reagents (10 ml AP buffer (100<br>
mM Tris/HCl, 100 mM NaCl, pH 9.5) with 50 l NBT solution (7.7% Nitro Blue<br>
Tetrazolium (Sigma) in 70% dimethylformamide) and 37 JJ.1 of X-Phosphate<br>
solution (5% of 5-bromo-4-chloro-3-indolyl phosphate in dimethylformamide).<br>
The production of viral particles is shown in Figure 1. The Western Blot<br>
pattern showed that E2-JUN (lane 1) migrated to a higher molecular weight in<br>
SDS-PAGE compared to wild type E2 (lane 2) and the BHK21 host cell line did<br>
not show any background.<br>
Example 3:<br>
Production of viral particles containing E2-JUN<br>
using thepTES '2JE2.-JUN vector<br>
RNase-free vector (1.0 g) was linerarized by Pvul digestion.<br>
Subsequently, in vitro transcription was carried out using an SP6 in vitro<br>
transcription kit (InvitroscripCAP by InvitroGen, Invitrogen BV, NV Leek,<br>
Netherlands). The resulting 5'-capped mRNA was analyzed on a reducing<br>
agarose-gel.<br>
In vitro transcribed mRNA (5 g) was electroporated into BHK 21 cells<br>
(ATCC: CCL10) according to Invitrogen's manual (Sindbis Expression system,<br>
Invitrogen BV, Netherlands). After 10 hours incubation at 37°C, the FCS<br>
containing medium was exchanged by HP-1 medium without FCS, followed by<br><br>
an additional incubation at 37°C for 10 hours. The supernatant was harvested<br>
and analyzed by Western blot analysis for production of viral particles exactly as<br>
described in Example 2.<br>
The obtained result was identical to the one obtained with pCYTtsE2:JZ//V<br>
as shown in Figure 2.<br>
Example 4:<br>
Fusion of human growth hormone (hGH) to the FOS leucine<br>
zipper domain (OmpA signal sequence)<br>
The hGH gene without the human leader sequence was amplified from the<br>
original plasmid (ATCC 31389) by PCR. Oligo 7 with an internal Xbal site was<br>
designed for annealing at the 5' end of the hGH gene, and oligo 9 with an internal<br>
EcoRI site primed at the 3' end of the hGH gene. For the PCR reaction, 100 pmol<br>
of each oligo and 5 ng of the template DNA was used in the 75 l reaction<br>
mixture (4 units of Taq or Pwo polymerase, 0.1 mM dNTPs and 1.5 mM MgSO4).<br>
PCR cycling was performed in the following manner: 30 cycles with an<br>
annealing temperature of 60°C and an elongation time of 1 minute at 72°C.<br>
The gel purified and isolated PCR product was used as a template for a<br>
second PCR reaction to introduce the ompA signal sequence and the<br>
Shine-Dalgarno sequence. For the PCR reaction, 100 pmol of oligo 8 and 9 and<br>
1 ng of the template PCR fragment was used in the 75 1 reaction mixture (4 units<br>
of Taq or Pwo polymerase, 0.1 mM dNTPs and 1.5 mM MgSO4). The annealing<br>
temperature for the first five cycles was 55 °C with an elongation time of 60<br>
seconds at 72°C; another 25 cycles were performed with an annealing<br>
temperature of 65 °C and an elongation time of 60 seconds at 72°C.<br>
Oligo7:<br>
gggtctagattcccaaccattcccttatccaggctttttgacaacgctatgctccgcgcccatcgtctgcaccagct<br>
ggcctttgacacc (SEQ ID NO:7)<br>
Oligo 8:<br><br>
gggtctagaaggaggtaaaaaacgatgaaaaagacagctatcgcgattgcagtggcactggctggtttcgctac<br>
cgtagcgcaggccttcccaaccattcccttatcc (SEQ ID NO:8)<br>
Oligo 9:<br>
cccgaattcctagaagccacagctgccctcc (SEQ ID N0:9)<br>
The resulting recombinant hGH gene was subcloned into pBluescript via<br>
Xbal/EcoRI. The correct sequence of both strands was confirmed by DNA<br>
sequencing.<br>
The DNA sequence coding for the FOS amphiphatic helix domain was<br>
PCR-amplified from vector pJuFo (Crameri &amp; Sitter Gene 137:69 (1993)) using<br>
the oligonucleotides:<br>
omp-FOS:<br>
5'- ccTGCGGTGGTCTGACCGACACCC-3' (SEQ ID NO: 10)<br>
FOS-hgh:<br>
5'- ccgcggaagagccaccGCAACCACCGTGTGCCGCCAGGATG-3' (SEQ ID<br>
NO: 11)<br>
For the PCR reaction, 100 pmol of each oligo and 5 ng of the template<br>
DNA was used in the 75 l reaction mixture (4 units of Taq or Pwo polymerase,<br>
0.1 mM dNTPs and 1.5 mM MgSO4). The temperature cycles were as follows:<br>
95°C for 2 minutes, followed by 5 cycles of 95°C (45 seconds), 60°C<br>
(30 seconds), 72°C (25 seconds) and followed by 25 cycles of 95 °C (45 seconds),<br>
68 °C (30 seconds), 72°C (20 seconds).<br>
The PCR product was purified, isolated and cloned into the StuI digested<br>
pBluescript-ompA-hGH. The hybrid gene was then cloned into the pKK.223-3<br>
Plasmid (Pharmacia).<br>
Example 5:<br>
Bacterial expression of FOS-hGH<br>
The ompA-FOS-hGH in pkk223-3 was expressed under the control of the<br>
inducible IPTG-dependend promoter using JM101 as E. coli host strain.<br>
Expression was performed in shaker flask. Cells were induced with 1 mM IPTG<br><br>
( final concentration) at an OD600of0.5. Expression was continued for 10 hours<br>
a:37°C. Cells were harvested by centrifugation at 3600 at 10DC for 15 min. The<br>
cell pellet was frozen (-20°C or liq. N2) and stored for 16 hours. The pellet was<br>
then thawed at 4°C and resuspended in 10 ml 10 mM Tris-HCl, pH 7.4<br>
containing 600 mM sucrose. After stirring for 15 min at 4°C, periplasmic<br>
proteins were released by an osmotic shock procedure. Chilled (4°C) deionized<br>
H,0 was added, and the suspension was stirred for 30 min at 4°C. The sludge<br>
was diluted, resuspended, and Iysozyme was added to degrade the cell wall of the<br>
bacteria. The cells and the periplasmic fraction spheroplasts were separated by<br>
centrifugation for 20 min at 11000 g at 4°C. The FOS-hGH-containing<br>
supernatant was analyzed by reducing and non-reducing SDS-Page and Dot Blot.<br>
Dot Blot was carried out as described in Example 8, using an anti-hGH antibody<br>
(Sigma) as the first antibody and an alkaline phosphatase (AP)-anti-mouse<br>
antibody conjugate as the second antibody.<br>
Full length, correctly processed FOS-hGH could be detected under<br>
reducing and non-reducing conditions. Part of FOS-hGH was bound to other,<br>
non-identified proteins due to the free cysteines present in the FOS amphiphatic<br>
helix. However, more than 50% of expressed FOS-hGH occurred in its native<br>
monomeric conformation ( Figure 3).<br>
Purified FOS-hGH will be used to perform first doping experiments with<br>
JL?v containing viral particles.<br>
Example 6:<br>
Construction of thepA V vector series for expression of FOS fusion proteins<br>
A versatile vector system was constructed that allowed either cytplasmic<br>
production or secretion of — or C-terminal FOS fusion proteins in E. coli or<br>
production of - or C-terminal FOS fusion proteins in eukaryotic cells. The<br>
vectors pAV 1 - pAV4 which was designed for production of FOS fusion proteins<br>
in E. coli, encompasses the DNA cassettes listed below, which contain the<br>
following genetic elements arranged in different orders: (a) a strong ribosome<br>
binding site and 5'-untranslated region derived from the E. coli ompA gene<br><br>
(aggaggtaaaaaacg) (SEQ ID NO: 13); (b) a sequence encoding the signal peptide<br>
of E. coli outer membrane protein OmpA (MKKTAIAIAVALAGFATVAQA)<br>
(SEQ ID NO: 14); (c) a sequence coding for the FOS dimerization domain flanked<br>
on both sides by two glycine residues and a cystein residue<br>
(CGGLTDTLQAETDQVEDEKSALQTEIANLLKEBCEKLEFILAAHGGC)<br>
(SEQ ID NO: 15); and (d) a region encoding a short peptidic linker (AAASGG<br>
(SEQ ID NO: 16) or GGSAAA (SEQ ID NO: 17)) connecting the protein of<br>
interest to the FOS dimerization domain. Relevant coding regions are given in<br>
upper case letters. The arrangement of restriction cleavage sites allows easy<br>
construction of FOS fusion genes with or without a signal sequence. The<br>
cassettes are cloned into the EcoRI/Hindin restriction sites of expression vector<br>
pKK223-3 (Pharmacia) for expression of the fusion genes under control of the<br>
strong tac promoter.<br>
pAVl<br>
This vector was designed for the secretion of fusion proteins with FOS at<br>
the C-terminus into the E. coli periplasmic space. The gene of interest (g.o.i.)<br>
may be ligated into the StuI/NotI sites of the vector.<br>
EcoRI	31/11<br>
craa, ttc agg agg taa aaa acg ATG AAA AAG ACA GCT ATC GCG ATT GCA GTG GCA CTG GCT<br>
MKKTAIAIAVALA<br>
61/21	StuI	NotI<br>
GGT TTC GCT ACC GTA GCG CAG_GCC_tgg gtg ggg GCG GCC GCT TCT GGT GGT TGC GGT GGT<br>
G FATVAQA	(goi)	AAASGGCGG<br>
121/41	151/51<br>
CTG ACC GAC ACC CTG CAG GCG GAA ACC GAC CAG GTG GAA GAC GAA AAA TCC GCG CTG CAA<br>
L TDTI.QAETDQVEDEKSAI..Q-<br>
181/61	211/71<br>
ACC GAA ATC GCG AAC CTG CTG AAA GAA AAA GAA AAG CTG GAG TTC ATC CTG GCG GCA CAC<br>
T EIANLLKEKEKLEFILAAH<br>
241/81	HiadIII<br>
GGT GGT TGC taa get t	(SEQ ID NO: 18)<br>
c- G c * A	(SEQ ID NOs:14 and l9)<br><br>
PAV2<br>
This vector was designed for the secretion of fusion proteins with FOS at<br>
the N-terminus into the E. coli periplasmic space. The gene of interest (g.o.i.)<br>
ligated into the Notl/EcoRV (or Notl/Hindlll) sites of the vector.<br>
EcoRI	31/11<br>
crae ttc agg agg taa aaa acg ATG AAA AAG ACA GCT ATC GCG ATT GCA GTG GCA CTG GCT<br>
MKKTAIAIAVALA<br>
61/21	StuI	91/31<br>
GGT TTC GCT ACC GTA GCG CAG GCC TGC GGT GGT CTG ACC GAC ACC CTG CAG GCG GAA ACC<br>
G FATVAQACGGLTDTLQAET<br>
121/41	151/51<br>
GAC CAG GTG GAA GAC GAA AAA TCC GCG CTG CAA ACC GAA ATC GCG AAC CTG CTG AAA GAA<br>
D QVEDEKSALQTE IANLLKE<br>
' 181/61	211/71	NotI<br>
AAA GAA AAG CTG GAG TTC ATC CTG GCG GCA CAC GGT GGT TGC GGT GGT TCT GCG GCC GCT<br>
K EKLEFILAAHGGCGGSAAA<br>
241/81	EcoRV HindIII-<br>
ggg tgt ggg gat ate aag ctt	(SBQ ID NO:20)<br>
(goi)	(SEQ ID NO:21)<br>
pAV3<br>
This vector was designed for the cytoplasmic production of fusion<br>
proteins with FOS at the C-terminus in E. coli. The gene of interest (g.o.i.) may<br>
be ligated into the EcoRV/NotI sites of the vector.<br>
EcoRI	EcoRV	NotI<br>
qaa ttc agg agg taa aaa aat ate ggg tgt ggg GCG GCC GCT TCT GGT GGT TGC GGT GGT<br>
(goi)	AAASGGCGG<br>
61/21	91/31<br>
CTG ACC GAC ACC CTG CAG GCG GAA ACC GAC CAG GTG GAA GAC GAA AAA TCC GCG CTG CAA<br>
L TDTLQAETDQVEDEKSALQ<br>
121/41	151/51<br>
ACC GAA ATC GCG AAC CTG CTG AAA GAA AAA GAA AAG CTG GAG TTC ATC CTG GCG GCA CAC<br>
T ZIANLLKEKEKLEFILAAH<br>
181/S1	HindIII<br>
GGT GCT TGC taa get t	(SEQ ID NO:22)<br>
G G c *	(SEQ ID NO:23)<br><br>
pAV4<br>
This vector is designed for the cytoplasmic production of fusion proteins<br>
with FOS at the N-terminusin-E. coli. Thegeneofinterest(g.o.i.)may be ligated<br>
into the Notl/EcoRV (or Notl/Hindm) sites of the vector. The N-terminal<br>
methionine residue is proteolytically removed upon protein synthesis (Hirel et al.,<br>
Proc. Natl. AcacL Sci. USA 56:8247-8251 (1989)).<br>
EcoRI	31/11<br>
gaa ttc agg agg taa aaa acg ATG GCT TGC GGT GGT CTG ACC GAC ACC CTG CAG GCG GAA<br>
S FRE* KTMACGGLTDTLQAE<br>
€1/21	91/31<br>
ACC GAC CAG GTG GAA GAC GAA AAA TCC GCG CTG CAA'ACC GAA ATC GCG AAC CTG CTG AAA<br>
TDQVEDEKSALQTEIANLLX<br>
121/41	1S1/S1	NotI<br>
GAA AAA GAA AAG CTG GAG TTC ATC CTG GCG GCA CAC GGT GST TGC GGT GGT TCT GCG GCC<br>
E KEKLEFILAAHGGCGGSAA<br>
181/61	EcoRV HindIII<br>
GCT ggg tgt ggg erab ate aacr ctt	(SEQ ED NOl24)<br>
A (got)	(SEQ ID NOs:88 and 25)<br>
The vectors pAV5 and pAV6, which are designed for eukaryotic<br>
production of FOS fusion proteins, encompasses the following genetic elements<br>
arranged in different orders: (a) a region coding for the leader peptide of human<br>
growth hormone (MATGSRTSLLLAFGLLCLPWLQEGSA) (SEQ ID NO:26);<br>
(b)	a sequence coding for the FOS dimerization domain flanked on both sides by<br>
two glycine residues and a cysteine residue<br>
(CGGLTDTLQAETDQVEDEKSALQTEIANLLKEKEKLEFILAAHGGC)<br>
(SEQ ID NO. 15); and<br>
(c)	a region encoding a short peptidic linker (AAASGG (SEQ ID NO: 16) or<br>
GOSAAA (SEQ ID NO: 17)) connecting the protein of interest to the FOS<br>
dimerization domain. Relevant coding regions are given in upper case letters.<br>
The arrangement of restriction cleavage sites allows easy construction of FOS<br>
fusion genes. The cassettes are cloned into the EcoRI/Hindlll restriction sites of<br>
the expression vector pMPSVEH (Artelt et al., Gene 55:213-219 (1988)).<br><br>
pAV5<br>
This vector is designed for the eukaxyotic production of fusion proteins<br>
with FOS at the C-terminus. The gene of interest (go.i.) may be inserted between<br>
the sequences coding for the hGH signal sequence and the FOS domain by ligation<br>
into the Eco47III/NotI sites of the vector. Alternatively, a gene containing its<br>
own signal sequence may be fused to the FOS coding region by ligation into the<br>
StuI/NotI sites.<br>
EcoRI StuI	31/11<br>
craa. ttc agg cct ATG GCT ACA GGC TCC O3G ACG TCC CTO CTC CTG GCT TTT GGC CTG CTC<br>
MATGSRTSLLLAFGI1I1<br>
61/21	Eco47III	HotI<br>
TGC CTG CCC TGG CTT CAA GAG GGC AGC GCT ggg tgt sgg GCG GCC GCT TCT GGT GGT TGC<br>
ci.PWLQ.EGSA	(goi)	A A A S G G C<br>
121/41	151/51<br>
GGT GGT CTG ACC GAC ACC CTG CAG GCG GAA ACC GAC CAG GTG GAA GAC GAA AAA TCC GCG<br>
GGLTDTLQAETDQVEDEKSA<br>
181/61	211/71<br>
CTG CAA ACC GAA ATC GCS AAC CTG CTG AAA GAA AAA GAA AAG CTG GAG TTC ATC CTG GCG<br>
I1QTEIAMLI1KEKEKLEFILA<br>
241/81	HindIII<br>
GCA CAC GGT GGT TGC taa get t	(SEQ ID NO:27)<br>
A H G G c *	(SEQ ID NOs:26 and 28)<br>
pAV6<br>
This vector is designed for the eukaryotic production of fusion proteins<br>
v/ithFOS at the N-terminus. The gene of interest (g.o.i.) may be ligated into the<br>
Notl/StuI (or NotI/HindIII) sites of the vector.<br>
EcoRI	31/11<br>
Caa ttc ATG GCT ACA GGC TCC CGG ACG TCC CTG CTC CTG GCT TTT GGC CTG CTC TGC CTG<br>
MATGSRTSLL.I,AFGIiI,CL<br>
61/21	ECO47III	91/31<br>
CCC TGG CTT CAA GAG GGC AGC GCT TGC GGT GGT CTG ACC GAC ACC CTG CAG GCG GAA ACC<br>
P WLQEGSACGGLTDTLQAET<br>
121/41	151/S1<br>
GAC CAG GTG GAA GAC GAA AAA TCC GCG CTG CAA ACC GAA ATC GCG AAC CTG CTG AAA GAA<br>
D QVEDEKSALQTE IAHLLKE<br><br>
181/61	211/71	NotI<br>
AAA GAA AAG CTG GAG TTC ATC CTG GCG GCA CAC GGT GGT TGC GGT GGT TCT GCG GCC GCT<br>
KEKLEFILAAHGGCG GSAAA<br>
24 1/81 StuI Hindlll<br>
ggg tgt ggg aqq cct aag ctt (SEQ ID NO:29)<br>
(goi)	(SEQ ID NO:30)<br>
Construction of expression vectors pA VI - pA V6<br>
The following oligonucleotides have been synthesized for construction of<br>
expression vectors pAVl - pAV6:<br>
FOS-FORl:<br>
CCTGGGTGGGGGCGGCCGCTTCTGGTGGTTGCGGTGGTCTGACC(SEQ<br>
IDN0:31);<br>
FOS-FOR2:<br>
GGTGGGAATTCAGGAGGTAAAAAGATATCGGGTGTGGGGCGGCC<br>
(SEQ ID NO:32);<br>
FO5-FOR3:<br>
GGTGGGAATTCAGGAGGTAAAAAACGATGGCTTGCGGTGGTCTGACC<br>
(SEQ ID NO:33);<br>
FOS-VOKA:<br>
GCTTGCGGTGGTCTGACC (SEQ ID NO:34);<br>
FOS-KEVl:<br>
CCACCAAGCTTAGCAACCACCGTGTGC (SEQ ID NO:35);<br>
FO5-REV2:<br>
CCACCAAGCTTGATATCCCCACACCCAGCGGCCGCAGAACCACCGC<br>
AACCACCG (SEQ ID NO:36);<br>
FOS-REV3:<br>
CCACCAAGCTTAGGCCTCCCACACCCAGCGGC (SEQ ID NO:37);<br>
OmpA-FORl:<br>
GGTGGGAATTCAGGAGGTAAAAAACGATG (SEQ ID NO:38);<br>
hGH-FORl:<br><br>
GGTGGGAATTCAGGCCTATGGCTACAGGCTCC (SEQ ID NO:39); and<br>
hGH-F0R2:<br>
GGTGGGAATTCATGGCTACAGGCTCCC (SEQ ID NO:40).<br>
For the construction of vector pAV2, the regions coding for the OmpA<br>
signal sequence and the FOS domain were amplified from the ompA-FOS-hGR<br>
fusion gene in vector pKK223-3 (see Example 5) using the primer pair<br>
OmpA-FORl/ FOS-REV2. The PCR product was digested with EcoRI/Hindlll<br>
and ligated into the same sites of vector pKK223-3 (Pharmacia).<br>
For the construction of vector pAVl, the FOS coding region was<br>
amplified from the ompA-FCAS'-hGH fusion gene in vector pKK223-3 (see<br>
Example 5) using the primer pair FOS-FORl IFOS-REV1. The PCR product was<br>
digested with HindIII and ligated into Stul/HindIII digested vector pAV2.<br>
For the construction of vector pAV3, the region coding for the FOS<br>
domain was amplified from vector pAVl using the primer pair<br>
FOS-FOR2/FOS-KEV1. The PCR product was digested with EcoRI/Hindlll and<br>
ligated into the same sites of the vector pKK223-3 (Pharmacia).<br>
For the construction of vector pAV4, the region coding for the FOS<br>
domain was amplified from the ompA-FCW-hGH fusion gene in vector<br>
pKK223-3 (see Example 5) using the primer pair FOS-VOR3/FOS-KEV2. The<br>
PCR product was digested with EcoRI/Hindlll and ligated into the same sites of<br>
the vector pKK223-3 (Pharmacia).<br>
For the construction of vector pA V5, the region coding for the hGH signal<br>
sequence is amplified from the hGH-FOS-hGH fusion gene in vector pSINrep5<br>
(see Example 7) using the primer pair hGH-FORl/hGHRE VI. The PCRproduct<br>
is digested with EcoRI/NotI and ligated into the same sites of the vector pAVl.<br>
The resulting cassette encoding the hGH signal sequence and the FOS domain is<br>
then isolated by EcoRI/HindIII digestion and cloned into vector pMPSVEH<br>
(Anelt et al, Gene (55:213-219 (1988)) digested with the same enzymes.<br><br>
For the construction of vector pAV6, the FOS coding region is amplified<br>
from vector pAV2 using the primer pair FOS-FOR4/FOSREV3. The PCR<br>
product is digested with Hindlll and cloned into Eco47III/HindIII cleaved vector<br>
pAV5. The entire cassette encoding the hGH signal sequence and the FOS<br>
domain is then reamplified from the resulting vector using the primer pair<br>
hGH-FOR2/FOSREV3, cleaved with EcoRI/Hindlll and ligated into vector<br>
pMPSVEH (Artelt et al., Gene (55:213-219 (1988)) cleaved with the same<br>
enzymes.<br>
Example 7:<br>
Construction of FOS-hGH with human (hGH) signal sequence<br>
For eukaryotic expression of the FOS-hGH fusion protein, the<br>
OmpA-.FO.S'-hGH fusion gene was isolated from pBluescript::OmpA-FOS-hGH<br>
{see Example 4) by digestion with Xbal/Bsp 1201 and cloned into vector pSINrep5<br>
(Invitrogen) cleaved with the same enzymes. The hGH signal sequence was<br>
synthesized by PCR (reaction mix: 50 pmol of each primer, dATP, dGTP, dTTP,<br>
dCTP (200 M each), 2.5 U Taq DNA polymerase (Qiagen), 50 ul total volume<br>
in the buffer supplied by the manufacturer; amplification: 92°C for 30 seconds,<br>
55°C for 30 seconds, 72°C for 30 seconds, 30 cycles) using the overlapping<br>
oligonucleotides Sig-hGH-FOR<br>
(GGGTCTAGAATGGCTACAGGCTCCCGGACGTCCCTGCTCCTGGCTT<br>
TTGGCCTGCTCTG) (SEQ ID NO:41) and Sig-hGH-REV<br>
(CGCAGGCCTCGGCACTGCCCTCTTGAAGCCAGGGCAGGCAGAGCA<br>
GGCCAAAAGCCAG) (SEQ ID NO:42). The PCR product was purified using<br>
the QiaExII Kit, digested with Stul/Xbal and ligated into vector<br>
pSINrep5::OmpA-F0S'-hGH cleaved with the same enzymes.<br><br>
Example 8:<br>
Eukaryotic expression of FOS-ItGH<br>
RNase-free vector (1.0 ng) (pSINrep5::OmpA-FaS-hGH) and 1.0 ug of<br>
DHEB (Bredenbeek et al.J Virol. 67:6439-6446 (1993)) were linerarized by<br>
Seal restriction digest. Subsequently, in vitro transcription was carried out using<br>
an SP6 in vitro transcription kit (InvitroscripCAP by InvitroGen, Invitrogen B V,<br>
NV Leek, Netherlands). The resulting 5'-capped mRNA was analyzed on<br>
reducing agarose-gel.<br>
In vitro, transcribed mRNA 5 u.g was electroporated into BHK 21 cells<br>
(ATCC: CCL10) according to Invitrogen's manual (Sindbis Expression system,<br>
Invitrogen BV, Netherlands). After 10 hours incubation at 37°C the FCS<br>
containing medium was exchanged by HP-1 medium without FCS, followed by<br>
an additional incubation at 37°C for 10 hours. The supernatant was harvested<br>
and analyzed by dot-blot analysis for production of FOS-hgh.<br>
Culture media (2.5 ul) was spotted on a nitrocellulose membrane and<br>
dried for 10 minutes at room temperature. The membrane was blocked with 1 %<br>
bovine albumin (Sigma) in TBS (IOXTBS per liter: 87.7 g NaCI, 66. lg Trizma<br>
hydrochloride (Sigma) and 9.7 g Trizma base (Sigma), pH 7.4) for 1 hour at room<br>
temperature, followed by an incubation with 2 \xg rabbit anti-human hGH<br>
antibody (Sigma) in 10 ml TBS-T (TBS with 0.05% Tween20) for 1 hour. The<br>
blot was washed 3 times for 10 minutes with TBS-T and incubated for 1 hour<br>
with alkaline phosphatase conjugated anti-rabbit IgG (Jackson ImmunoResearch<br>
Laboratories, Inc.) diluted 1:5000 in TBS-T. After washing 2 times for 10<br>
minutes with TBS-T and 2 times for 10 minutes with TBS, the blot was<br>
developed by AP staining as described in Example 2. Results are shown in<br>
Figure 3.<br><br>
Example 9:<br>
Construction of FOS-PLA (TV- and C-terminal)<br>
The following gene is constructed by chemical gene synthesis coding for<br>
a catalytically inactive variant (Forster et al, J. Allergy Clin. Immunol. 95:<br>
1229-1235 (1995)) of bee venom phospholipase A2 (PLA).<br>
1/1	31/11<br>
ATC ATC TAC CCA GGT ACT CTG TGG TGT GGT CAC GGC AAC AAA TCT TCT GGT CCG AAC GAA<br>
I IYPGTLWCGHGNKSSGPNE<br>
61/21	91/31<br>
CTC GGC CGC TTT AAA CAC ACC GAC GCA TGC TGT CGC ACC CAG GAC ATG TGT CCG GAC GTC<br>
LGRFKHTDACCRTQDMCPDV<br>
121/41	151/51<br>
ATG TCT GCT GGT GAA TCT AAA CAC GGG TTA ACT AAC ACC GCT TCT CAC ACG CGT CTC AGC<br>
M SAGESKHGLTNTAS HTRLS<br>
181/61	211/71<br>
TGC GAC TGC GAC GAC AAA TTC TAC GAC TGC CTT AAG AAC TCC GCC GAT ACC ATC TCT TCT<br>
CDCDDKFYDCLKNSADTISS<br>
241/81	271/91<br>
TAC TTC GTT GGT AAA ATG TAT TTC AAC CTG ATC GAT ACC AAA TGT TAC AAA CTG GAA CAC<br>
Y FVGKMYFNLIDTKCYKLEH<br>
301/101	331/111<br>
CCG GTA ACC GGC TGC GGC GAA CGT ACC GAA GGT CGC TGC CTG CAC TAC ACC GTT GAC AAA<br>
P VTGCGERTEGRC LHYTVDK<br>
361/121	391/131<br>
TCT AAA CCG AAA GTT TAC CAG TGG TTC GAC CTG CGC AAA TAC	(SEQ ID NO ."43)<br>
SKPKVYQWFDLRKY	(SEQ ID NO:44)<br>
For fusion of PLA to the N-terminus of the FOS dimerization domain, the<br>
region is amplified using the oligonucleotides PLA-FOR1<br>
(CCATCATCTACCCAGGTAC) (SEQ ID NO:45) and PLA-REV1<br>
(CCCACACCCAGCGGCCGCGTATTTGCGCAGGTCG) (SEQ ID NO:46).<br>
The PCR product is cleaved with NotI and ligated into vector pAVl previously<br>
cleaved with the restriction enzymes Stul/Notl. For fusion of PLA to the<br>
C-terminus of the FOS dimerization domain, the region is amplified using the<br>
oligonucleotides PLA-FOR2<br>
(CGGTGGTTCTGCGGCCGCTATCATCTACCCAGGTAC) (SEQ ID NO:47)<br>
and PLA-REV2 (TTAGTATTTGCGCAGGTCG) (SEQ ID NO:47). The PCR<br><br>
product is cleaved with NotI and ligated into vector pAV2 previously cleaved<br>
with the restriction enzymes Notl/EcoRV.<br>
Example 10:<br>
Construction of FOS-Ovalbumin fusion gene (N- and C-terminal)<br>
For cloning of the ovalbumin coding sequence, mRNA from chicken<br>
oviduct tissue is prepared using the QuickPrep™ Micro mRNA Purification Kit<br>
(Pharmacia) according to manufacturer instructions. Using the Superscript™<br>
One-step RT PCR Kit (Gibco BRL), a cDNA encoding the mature part of<br>
ovalbumin (corresponding to nucleotides 68-1222 of the mRNA (McReynolds et<br>
al, Nature 275:723-728 (1978)) is synthesized using the primers Ova-FORl<br>
(CCGGCTCCATCGGTGCAG) (SEQ ID NO:49) and Ova-REVl<br>
(ACCACCAGAAGCGGCCGCAGGGGAAACACATCTGCC) (SEQ ID<br>
NO:50). The PCR product is digested with NotI and cloned into StuI/NotI<br>
digested vector pAVl for expression of the fusion protein with the FOS<br>
dimerization domain at the C terminus. For production of a fusion protein with<br>
the FOS dimerization domain at the N terminus, the Ovalbumin coding region is<br>
amplified from the constructed vector (pAV 1: :Ova) using the primers Ova-FOR2<br>
(CGGTGGTTCTGCGGCCGCTGGCTCCATCGGTGCAG) (SEQ ID NO:51)<br>
and 0va-REV2 (TTAAGGGGAAACACATCTGCC) (SEQ ID NO:52). The<br>
PCR product is digested with NotI and cloned into the Notl/EcoRV digested<br>
vector pAV2. Cloned fragments are verified by DNA sequence analysis.<br>
Example 11<br>
Production and purification of FOS-PLA and<br>
FOS ovalbumin fusion proteins<br>
For cytoplasmic production of FOS fusion proteins, an appropriate E. coli<br>
strain was transformed with the vectors pAV3::PLA,pAV4::PL A, pAV3::0va or<br>
pAV4::Ova. The culture was incubated in rich medium in the presence of<br>
ampicillin at 37°C with shaking. At an optical density (550nm) of 1, 1 mM IPTG<br>
was added and incubation was continued for another 5 hours. The cells were<br><br>
harvested by centrifugation, resuspended in an appropriate buffer (e.g. tris-HCl,<br>
pH 7.2, 150 mM NaCl) containing DNase, RNase and lysozyme, and disrupted<br>
by passage through a french pressure cell. After centrifugation (Sorvall RC-5C,<br>
SS34 rotor, 15000 rpm, 10 min, 4°C), the pellet was resuspended in 25 ml<br>
inclusion body wash buffer (20 mM tris-HCl, 23% sucrose, 0.5% Triton X-100,<br>
1 mM EDTA, pH8) at 4°C and recentrifuged as described above. This procedure<br>
was repeated until the supernatant after centrifugation was essentially clear.<br>
Inclusion bodies were resuspended in 20 ml solubilization buffer (5.5 M<br>
guanidinium hydrochloride, 25 mM tris-HCl, pH 7.5) at room temperature and<br>
insoluble material was removed by centrifugation and subsequent passage of the<br>
supernatant through a sterile filter (0.45 urn). The protein solution was kept at<br>
4 ° C for at least 10 hours in the presence of 10 mM EDTA and 100 mM DTT and<br>
then dialyzed three times against 10 volumes of 5.5 M guanidinium<br>
hydrochloride, 25 mM tris-HCl, 10mMEDTA,pH6. The solution was dialyzed<br>
twice against 5 liters of 2 M urea, 4 mM EDTA, 0.1 M NH4C1, 20 mM sodium<br>
borate (pH 8.3) in the presence of an appropriate redox shuffle (oxidized<br>
glutathione/reduced glutathione; cystine/cysteine). The refolded protein was then<br>
applied to an ion exchange chromatography. The protein was stored in an<br>
appropriate buffer with a pH above 7 in the presence of 2-10 mM DTT to keep<br>
the cysteine residues flanking the FOS domain in a reduced form. Prior to<br>
coupling of the protein with the alphavirus particles, DTT was removed by<br>
passage of the protein solution through a Sephadex G-25 gel filtration column.<br>
Example 12:<br>
Constructions of gpl40-FOS<br>
The gpl40 gene (Swiss-Prot:P03375) without the internal protease<br>
cleavage site was amplified by PCR from the original plasmid pAbT4674 (ATCC<br>
40829) containing the full length gp l60 gene using the following<br>
oligonucleotides:<br><br>
HIV-1:<br>
5'-ACTAGTCTAGAatgagagtgaaggagaaatatc-3' (SEQ ID NO:53);<br>
HIV-end:<br>
5'-TAGCATGCTAGCACCGAAtttatctaattccaataattcttg-3' (SEQ ID NO:54);<br>
HIV-Cleav:<br>
5 '-gtagcacccaccaaggcaaagCTGAAAGCTACCCAGCTCG AGAAACTGgca-3'<br>
(SEQ ID NO:55); and<br>
HIV-Cleav2:<br>
5 '-caaagctcctattcccactgcCAGTTTCTCGAGCTGGGTAGCTTTCAG-3' (SEQ ID<br>
NO:56).<br>
For PCR I, 100 pmol of oligo HIV-1 and HIV-Cleav2 and 5 ng of the<br>
template DNA were used in the 75 l reaction mixture (4 units of Taq or Pwo<br>
polymerase, 0.1 mM dNTPs and 1.5 mM MgSO4). PCR cycling was done in the<br>
following manner: 30 cycles with an annealing temperature of 60°C and an<br>
elongation time of 2 minutes at 72°C.<br>
For PCR II, 100 pmol of oligo HIV-end and HIV-Cleav and 5 ng of the<br>
template DNA were used in the 75 l reaction mixture, (4 units of Taq or Pwo<br>
polymerase, 0.1 mM dNTPs and 1.5 mM MgSO4). PCR cycling was done in the<br>
following manner: 30 cycles with an annealing temperature of 60°C and an<br>
elongation time of 50 seconds at 72°C.<br>
Both PCR fragments were purified, isolated and used in an assembly PCR<br>
reaction. For the assembly PCR reaction, 100 pmol of oligo HIV-1 and HIV-end<br>
and 2 ng of each PCR fragment (PCRI and PCR II) were used in the 75 (il (4 units<br>
of Taq or Pwo polymerase, 0.1 mM dNTPs and 1.5 mM MgSO4). PCR cycling<br>
was done in the following manner: 30 cycles with an annealing temperature of<br>
60c C and an elongation time of 2.5 minutes at 72°C. The assembly PCR product<br>
was digested Xbal and Nhel. The FOS amphiphatic helix was fused in frame to<br>
the C-terminal end of gp-140.<br><br>
The DNA sequence coding for the FOS amphiphatic helix domain was<br>
PCR-amplified from vector pJuFo (Crameri &amp; Suter Gene 137: 69 (1993)) using<br>
the oligonucleotides:<br>
FOS-UIV:<br>
5'-ttcggtgctagcggtggcTGCGGTGGTCTGACCGAC-3' (SEQ ID NO:57); and<br>
FOS-Apa:<br>
5'-gatgctgggcccttaaccGCAACCACCGTGTGCCGCC-3' (SEQ ID NO:58).<br>
For the PCR reaction, 100 pmol of each oligo and 5 ng of the template<br>
DNA was used in the 75 ul reaction mixture (4 units of Taqor Pwo polymerase,<br>
0.1 mM dNTPs and 1.5 mM MgSO4). Temperature cycling was done as follows:<br>
95°C for 2 minutes, followed by 5 cycles of 95°C (45 seconds), 60°C (30<br>
seconds), 72 °C (25 seconds) and followed by 25 cycles of 95°C (45 seconds),<br>
68 ° C (3 0 seconds), 72 ° C (20 seconds). The obtained PCR fragment was digested<br>
with Nhel and Bsp 120L.<br>
The final expression vector for GP140-FOS was obtained in a 3 fragment<br>
ligation of both PCR fragments into pSinRep5. The resultant vector<br>
pSinRepS-GPMO-FOiS1 was evaluated by restriction analysis and DNA<br>
sequencing.<br>
GP 140-FOS was also cloned into pCYTts via Xbal and Bsp 120L to obtain<br>
a stable, inducible GP14O-F0S expressing cell line.<br>
Example 13:<br>
Expression of GPUOFOS using pSinRep5-GP140FOS<br>
RNase-free vector (1.0 ug) (pSinRep5-GP 140-FOS) and 1.0 ug of DHEB<br>
(Bredenbeek et al, J. Virol. 67:6439-6446 (1993)) were linearized by restriction<br>
digestion. Subsequently, in vitro transcription was carried out using an SP6 in<br>
vitro transcription kit (InvitroscripCAP by InvitroGen, Invitrogen BV, NV Leek,<br>
Netherlands). The resulting 5'-capped mRNA was analyzed on a reducing<br>
agarose-gel.<br>
In vitro transcribed mRNA (5 ug) was electroporated into BHK 21 cells<br>
(ATCC: CCL10) according to Invitrogen's manual (Sindbis Expression System,<br><br>
Invitrogen BV, Netherlands). After 10 hours incubation at 37°C, the FCS<br>
containing medium was exchanged by HP-1 medium without FCS, followed by<br>
an additional incubation at 37°C for 10 hours. The supernatant was harvested<br>
and analyzed by Western blot analysis for production of soluble GP140-FOS<br>
exactly as described in Example 2.<br>
Example 14:<br>
Expression ofGPUOFOS usingpCYTts-GPUOFOS<br>
pCYT-GP 140-FOS 20 g was linearized by restriction digestion. The<br>
reaction was stopped by phenol/chloroform extraction, followed by an<br>
isopropanol precipitation of the linearized DNA. The restriction digestion was<br>
evaluated by agarose gel eletrophoresis. For the transfection, 5.4 \xg of linearized<br>
pCYTtsGPl 40-FOS was mixed with 0.6 ug of linearized pSV2Neo in 30 ul H2O<br>
and 30 ul of 1 M CaCl2 solution was added. After addition of 60 ul phosphate<br>
buffer (50 mM HEPES, 280 mMNaCl, 1.5 mM Na2 HPO4, pH 7.05), the solution<br>
was vortexed for 5 seconds, followed by an incubation at room temperature for<br>
25 seconds. The solution was immediately added to 2 ml HP-1 medium<br>
containing 2% FCS (2% FCS medium). The medium of an 80% confluent<br>
BHK21 cell culture (6-well plate) was then replaced by the DNA containing<br>
medium. After an incubation for 5 hours at 37°C in a CO: incubator, the DNA<br>
containing medium was removed and replaced by 2 ml of 15% glycerol in 2%<br>
FCS medium. The glycerol containing medium was removed after a 30 second<br>
incubation phase, and the cells were washed by rinsing with 5 ml of HP-1<br>
medium containing 10% FCS. Finally 2 ml of fresh HP-1 medium containing<br>
10% FCS was added.<br>
Stably transfected cells were selected and grown in selection medium<br>
(HP-1 medium supplemented with G418) at 37°C inaCO: incubator. When the<br>
mixed population was grown to confluency, the culture was split to two dishes,<br>
followed by a 12 h growth period at 37 °C. One dish of the cells was shifted to<br>
30 °C to induce the expression of soluble GP140-FO5. The other dish was kept<br>
at 37°C.<br><br>
The expression of soluble GP14O-F0.S was determined by Western blot<br>
analysis. Culture media (0.5 ml) was methanol/chloroform precipitated, and the<br>
pellet was resuspended in SDS-PAGE sample buffer. Samples were heated for<br>
5 minutes at 95 °C before being applied to a 15% acrylamide gel. After<br>
SDS-PAGE, proteins were transferred to Protan nitrocellulose membranes<br>
(Schleicher &amp; Schuell, Germany) as described by Bass and Yang, in Creighton,<br>
T.E., ed., Protein Function: A Practical Approach, 2nd Edn., IRL Press, Oxford<br>
(1997), pp. 29-55. The membrane was blocked with 1 % bovine albumin (Sigma)<br>
in TBS (IOXTBS per liter: 87.7 gNaCl, 66. lg Trizmahydrochloride (Sigma) and<br>
9.7 g Trizma base (Sigma), pH 7.4) for 1 hour at room temperature, followed by<br>
an incubation with an anti-GP140 or GP-160 antibody for 1 hour. The blot was<br>
washed 3 times for 10 minutes with TBS-T (TBS with 0.05% Tween20), and<br>
incubated for 1 hour with an alkaline-phosphatase-anti-<br>
mouse/rabbit/monkey/human IgG conjugate. After washing 2 times for 10<br>
minutes with TBS-T and 2 times for 10 minutes with TBS, the development<br>
reaction was carried out using alkaline phosphatase detection reagents (10 ml AP<br>
buffer (100 mMTris/HCl, 100 mMNaCl,pH9.5) with 50 ul NBT solution(7.7%<br>
Nitro Blue Tetrazolium (Sigma) in 70% dimethylformamide) and 37 ul of<br>
X-Phosphate solution (5% of 5-bromo-4-chloro-3-indolyl phosphate in<br>
dimethylformamide).<br>
Example 15:<br>
Production and purification of GP140FOS<br>
An anti-gpl 20 antibody was covalently coupled to a NHS/EDC activated<br>
dextran and packed into a chromatography column. The supernatant, containing<br>
GP14OF0S is loaded onto the column and after sufficient washing, GP14OF0S<br>
was eluted using 0.1 M HC1. The eluate was directly neutralized during<br>
collection using 1 M Tris pH 7.2 in the collection tubes.<br>
Disulfide bond formation might occur during purification, therefore the<br>
collected sample is treated with 10 mM DTT in 10 mM Tris pH 7.5 for 2 hours<br>
at25°C.<br><br>
DTT is remove by subsequent dialysis against 10 mM Mes; 80 mM NaC!<br>
pH 6.0. Finally GP140FOS is mixed with alphavirus particles containing the<br>
JUN leucine zipper in E2 as described in Example 16.<br>
Example 16:<br>
Preparation of the AlphaVaccine Particles<br>
Viral particles (see Examples 2 and 3) were concentrated using Millipore<br>
Ultrafree Centrifugal Filter Devices with a molecular weight cut-off of 100 kD<br>
according to the protocol supplied by the manufacturer. Alternatively, viral<br>
particles were concentrated by sucrose gradient centrifugation as described in the<br>
instruction manual of the Sindbis Expression System (Invitrogen, San Diego,<br>
California). The pH of the virus suspension was adjusted to 7.5 and viral<br>
particles were incubated in the presence of 2-10 mM DTT for several hours.<br>
Viral particles were purified from contaminating protein on a Sephacryl S-300<br>
column (Pharmacia) (viral particles elute with the void volume) in an appropriate<br>
buffer.<br>
Purified virus particles were incubated with at least 240 fold molar excess<br>
of FOS-antigen fusion protein in an appropriate buffer (pH 7.5-8.5) in the<br>
presence of a redox shuffle (oxidized glutathione/reduced glutathione;<br>
cystine/cysteine) for at least 10 hours at 4°C. After concentration of the particles<br>
using a Millipore Ultrafree Centrifugal Filter Device with a molecular weight<br>
cut-off of 100 kD, the mixture was passed through a Sephacryl S-300 gel<br>
filtration column (Pharmacia). Viral particles were eluted with the void volume.<br>
Example 17<br>
Fusion of JUNamphipathic helix to the amino terminus of HBcAg(l-144)<br>
The JUN helix was fused to the amino terminus of the HBcAg amino acid<br>
sequence 1 to 144 (JUN-HBcAg construct). For construction of the JUN-HBcAg<br>
DNA sequence, the sequences encoding the JUN helix and HBcAgO -144) were<br>
amplified separately by PCR. The JUN sequence was amplified from the pJuFo<br><br>
plasmid using primers EcoRI-JUN(s) and JUN-SacII(as). The EcoRI-JUN(s)<br>
primer introduced an EcoRI site followed by a start ATG codon. The JUN-<br>
SacII(as) primer introduced a linker encoding the amino acid sequence GAAGS.<br>
The HBcAg (1-144) sequence was amplified from the pEco63 plasmid (obtained<br>
from ATCC No. 31518) using primers JUN-HBcAg(s) and<br>
HBcAg( 1 -144)Hind(as). JUN-HBcAg(s) contained a sequence corresponding to<br>
the 3' end of the sequence encoding the JUN helix followed by a sequence<br>
encoding the GAAGS linker and the 5' end of the HBcAg sequence. HBcAg(l-<br>
144)Hind(as) introduces a stop codon and a HindIII site after codon 144 of the<br>
HBcAg gene. For the PCR reactions, 100 pmol of each oligo and 50 ng of the<br>
template DNAs were used in the 50 l reaction mixtures with 2 units of Pwo<br>
polymerase, 0.1 mM dNTPs and 2 mM MgSO4. For both reactions, temperature<br>
cycling was carried out as follows: 94°C for 2 minutes; and 30 cycles of 94°C (1<br>
minute), 50°C (1 minute), 72°C (2 minutes).<br>
Primer sequences:<br>
EcoRI-JUN(s):<br>
(5'-CCGGAATTCATGTGCGGTGGTCGGATCGCCCGG-3') (SEQ ID<br>
NO:61);<br>
JUN-SacII(as):<br>
(5 '-GTCGCTACCCGCGGCTCCGCAACCAACGTGGTTCATGAC-3') (SEQ<br>
ID NO:62);<br>
JUN-HBcAg(s):<br>
(5'-GTTGGTTGCGGAGCCGCGGGTAGCGACATTGACCCTTATAAAGAATTTGG-3')<br>
(SEQ ID NO:63);<br><br>
HBcAg(l-144)Hind(as):<br>
(5 '-CGCGTCCCAAGCTTCTACGGAAGCGTTGATAGGATAGG-3') (SEQ<br>
ID NO:64).<br>
Fusion of the two PCR fragments was performed by PCR using primers<br>
EcoRI-JUN(s) and HBcAg(l-144)Hind(as). 100 pmol of each oligo was used with<br>
100ng of the purified PCR fragments in a 50 ul reaction mixture containing 2<br>
units of Pwo polymerase, 0.1 mM dNTPs and 2 mM MgSO4. PCR cycling<br>
conditions were: 94°C for 2 minutes; and 35 cycles of 94°C (1 minute), 50°C<br>
(1 minute), 72 °C (2 minutes). The final PCR product was analyzed by agarose<br>
gel electrophoresis, purified and digested for 16 hours in an appropriate buffer<br>
with EcoRI and HindIII restriction enzymes. The digested DNA fragment was<br>
ligated into EcoRI/HindIII-digested pKK vector to generate pKK-JUN-HBcAg<br>
expression vector. Insertion of the PCR product was analyzed by EcoRI/HindIII<br>
restriction analysis and by DNA sequencing of the insert.<br>
Example 18<br>
Fusion of JUN amphipathic helix to the carboxy terminus of HBcAg(l-144)<br>
The JUN helix was fused to the carboxy terminus of the HBcAg amino<br>
acid sequence 1 to 144 (HBcAg-JUN construct). For construction of the HBcAg-<br>
JUN DNA sequence, the sequences encoding the JUN helix and HBcAg( 1 -144)<br>
were amplified separately by PCR. The JUN sequence was amplified from the<br>
pJuFo plasmid with primers SacII-JUN(s) and JUN-Hindlll(as). SacII-JUN(s)<br>
introduced a linker encoding amino acids LAAG. This sequence also contains a<br>
SacII site. JUN-HindIII(as) introduced a stop codon (TAA) followed by a Hindlll<br>
site. The HBcAg(l -144) DNA sequence was amplified from the pEco63 plasmid<br>
using primers EcoRI-HBcAg(s) and HBcAg(l -144)-JUN(as). EcoRI-HBcAg(s)<br>
introduced an EcoRI site prior to the Start ATG of the HBcAg coding sequence.<br>
HBcAg(l-144)-JUN(as) introduces a sequence encoding the peptide linker<br>
(LAAG), which also contains a SacII site. For the PCR reactions, 100 pmol of<br><br>
each oligo and 50 ng of the template DNAs were used in the 50 ul reaction<br>
mixtures with 2 units of Pwo polymerase, 0.1 mM dNTPs and 2 mM MgSO4.<br>
Temperature cycling was carried out as follows: 94°C for 2 minutes; and 30<br>
cycles of 94°C (1 minute), 50°C (1 minute), 72°C (2 minutes).<br>
Primer sequences<br>
SacII-JUN(s):<br>
(5'-CTAGCCGCGGGTTGCGGTGGTCGGATCGCCCGG-3') (SEQ ID<br>
NO:65);<br>
JUN-Hindlll(as):<br>
(5'-CGCGTCCCAAGCTTTTAGCAACCAACGTGGTTCATGAC -3') (SEQ<br>
ID NO:66);<br>
EcoRI-HBcAg(s):<br>
(5'-CCGGAATTCATGGACATTGACCCTTATAAAG-3') (SEQ ID NO:67);<br>
and<br>
HBcAg-JUN(as):<br>
(5'-CCGACCACCGCAACCCGCGGCTAGCGGAAGCGTTGATAGGATAGG-3')<br>
(SEQ ID NO:68).<br>
Fusion of the two PCR fragments was performed by PCR using primers<br>
EcoRI-HBcAg(s) and JUN-HindIII(as). For the PCR fusion, 100 pmol of each<br>
oligo was used with lOOng of the purified PCR fragments in a 50 ul reaction<br>
mixture containing 2 units of Pwo polymerase, 0.1 mM dNTPs and 2 mM<br>
MgSO4. PCR cycling conditions were: 94°C for 2 minutes; and 35 cycles of<br>
94°C(1 minute), 50°C(1 minute), 72°C (2 minutes). The final PCR product was<br>
analyzed by agarose gel electrophoresis, and digested for 16 hours in an<br><br>
appropriate buffer with EcoRI and Hindlll restriction enzymes. The DNA<br>
fragment was gel purified and ligated into EcoRI/Hindlll-digested pKK vector to<br>
generate pKK-HBcAg-JUN expression vector. Insertion of the PCR product was<br>
analyzed by EcoRI/HindIII restriction analysis and by DNA sequencing of the<br>
insert.<br>
Example 19<br>
Insertion of JUN amphipathic helix into the c/el epitope of HBcAg(l-144)<br>
The c/el epitope (residues 72 to 88) of HBcAg is known to be located in<br>
the tip region on the surface of the hepatitis B virus capsid. A part of this region<br>
(residues 76 to 82) of the protein was genetically replaced by the JUN helix to<br>
provide an attachment site for antigens (HBcAg-JUNIns construct). The HBcAg-<br>
JUNIns DNA sequence was generated by PCRs: The JUN helix sequence and<br>
t\Vo sequences encoding HBcAg fragments (amino acid residues 1 to 75 and 83<br>
to 144) were amplified separately by PCR. The JUN sequence was amplified<br>
from the pJuFo plasmid with primers BamHI-JUN(s) and JUN-SacII(as).<br>
BamHI-JUN(s) introduced a linker sequence encoding the peptide sequence<br>
GSGGG that also contains a BamHI site. JUN-SacII(as) introduced a sequence<br>
encoding the peptide linker GAAGS followed by a sequence complementary to<br>
the 3' end of the JUN coding sequence. The HBcAg(l-75) DNA sequence was<br>
amplified from the pEco63 plasmid using primers EcoRIHBcAg(s) and<br>
HBcAg75-JUN(as). EcoRIHBcAg(s) introduced an EcoRI site followed by a<br>
sequence corresponding to the 5' end of the HBcAg sequence. HBcAg75-<br>
JUN(as) introduced a linker encoding the peptide GSGGG after amino acid 75 of<br>
HBcAg followed by a sequence complementary to the 5' end of the sequence<br>
encoding the JUN helix. The HBcAg (83-144) fragment was amplified using<br>
primers JUN-HBcAg83(s) and HBcAg(l-144)Hind(as). JUN-HBcAg83(s)<br>
contained a sequence corresponding to the 3' end of the JUN-encoding sequence<br>
followed by a linker encoding the peptide, GAAGS and a sequence corresponding<br>
to the 5' end of the sequence encoding HBcAg (83-144). HBcAg(l-144)Hind(as)<br><br>
introduced a stop codon and a HindIII site after codon 144 of the HBcAg gene.<br>
For the PCR reactions, 100 pmol of each oligo and 50 ng of the template DNAs<br>
were used in the 50 ul reaction mixtures (2 units of Pwo polymerase, 0.1 mM<br>
dNTPs and 2 mM MgSO4). Temperature cycling was performed as follows:<br>
94°C for 2 minutes; and 35 cycles of 94°C (1 minute), 50°C (1 minute), 72°C<br>
(2 minutes).<br>
Primer sequences:<br>
BamHI-JUN(s):<br>
(5'-CTAATGGATCCGGTGGGGGCTGCGGTGGTCGGATCGCCCGGCTCGAG-3')<br>
(SEQ ID NO:69);<br>
JUN-SacII(as):<br>
(5 '-GTCGCTACCCGCGGCTCCGCAACCAACGTGGTTCATGAC-3') (SEQ<br>
ID NO:70);<br>
EcoRIHBcAg(s):<br>
(5'- CCGGAATTCATGGACATTGACCCTTATAAAG-3') (SEQ ID NO:71);<br>
HBcAg75-JUN (as):<br>
(5 '-CCG ACCACCGCAGCCCCCACCGGATCCATTAGTACCCACCCAGGTAGC-3')<br>
(SEQ ID NO:72);<br>
JUN-HBcAg83(s):<br>
(5'-GTTGGTTGCGGAGCCGCGGGTAGCGACCTAGTAGTCAGTTATGTC-3')<br>
(SEQ ID NO:73); and<br><br>
HBcAg(l-144)Hind(as):<br>
(5 '-CGCGTCCCAAGCTTCTACGGAAGCGTTGATAGGATAGG-3') (SEQ<br>
ID NO:74).<br>
Fusion of the three PCR fragments was performed as follows. First, the<br>
fragment encoding HBcAg 1-75 was fused with the sequence encoding JUN by<br>
PCR using primers EcoRIHBcAg(s) and JUN-SacII(as). Second, the product<br>
obtained was fused with the HBcAg(83-144) fragment by PCR using primers<br>
EcoRI HBcAg(s) and HBcAg HindIII(as). For PCR fusions, 100 pmol of each<br>
oligo was used with 100 ng of the purified PCR fragments in a 50 l reaction<br>
mixture containing 2 units of Pwo polymerase, 0.1 mM dNTPs and 2 mM<br>
MgSO4. The same PCR cycles were used as for generation of the individual<br>
fragments. The final PCR product was digested for 16 hours in an appropriate<br>
buffer with EcoRI and Hindlll restriction enzymes. The DNA fragment was<br>
li gated into EcoRI/HindIII-digested. pKK vector, yielding the pKK-HBcAg-<br>
JUNIns vector. Insertion of the PCR product was analyzed by EcoRI/HindIII<br>
restriction analysis and by DNA sequencing of the insert.<br>
Example 20<br>
Fusion of the JUNamphipathic helix to the carboxy terminus of the<br>
measles virus nucleocapsid (N) protein<br>
The JUN helix was fused to the carboxy terminus of the truncated measles<br>
virus N protein fragment comprising amino acid residues 1 to 473 (N473-JUN<br>
construct). For construction of the DNA sequence encoding N473-JUN the<br>
sequence encoding the JUN helix and the sequence encoding N473-JUN were<br>
amplified separately by PCR. The JUN sequence was amplified from the pJuFo<br>
plasmid with primers SacII-JUN(s) and JUN-HindIII(as). SacII-JUN(s)<br>
introduced a sequence encoding peptide linker LAAG. This sequence also<br>
contained a SacII site. The JUN-HindIII(as) anti-sense primer introduced a stop<br>
codon (TAA) followed by a Hindlll site. The N (1-473) sequence was amplified<br>
from the pSC-N plasmid containing the complete measles virus N protein coding<br><br>
sequence (obtained from M. Billeter, Zurich) using primers EcoRJ-Nmea(s) and<br>
Nmea-JUN(as). EcoRI-N(mea)(s) introduced an EcoRI site prior to the Start<br>
ATG of the N coding sequence. N(mea)-JUN(as) was complementary to the 3'<br>
end of the N(l -473) coding sequence followed by a sequence complementary to<br>
the coding sequence for the peptide linker (LAAG). For the PCR reactions, 100<br>
pmol of each oligo and 50 ng of the template DNAs were used in the 50 ul<br>
reaction mixtures with 2 units of Pwo polymerase, 0.1 mM dNTPs and 2 mM<br>
MgSO4. Temperature cycling was performed as follows: 94 °C for 2 minutes; and<br>
35 cycles of 94°C (1 minute), 55°C (1 minute), 72°C (2 minutes).<br>
Primer sequences:<br>
SacII-JUN(s):<br>
(5'-CTAGCCGCGGGTTGCGGTGGTCGGATCGCCCGG-3') (SEQ ID<br>
NO:75);<br>
JUN-HindIII(as):<br>
(5'-CGCGTCCCAAGCTTTTAGCAACCAACGTGGTTCATGAC -3') (SEQ<br>
ID NO.76);<br>
EcoRI-Nmea(s):<br>
(5'-CCGGAATTCATGGCCACACTTTTAAGGAGC-3')(SEQ ID NO:77);and<br>
Nmea-JUN(as):<br>
(5 '-CGCGTCCCAAGCTTTTAGCAACCAACGTGGTTC ATGAC-3') (SEQ ID<br>
NO:78).<br>
Fusion of the two PCR fragments was performed in a further PCR using<br>
primers EcoRI-Nmea(s) and Nmea-JUN(as). For the PCR fusion, 100 pmol of<br>
each oligo was used with 100 ng of the purified PCR fragments in a 50 l<br><br>
reaction mixture containing 2 units of Pwo polymerase, 0.1 mM dNTPs and 2<br>
m.M MgSO4. Temperature cycling was performed as follows: 94°C for 2<br>
minutes; and 35 cycles of 94°C (1 minute), 50°C (1 minute), 72°C (2 minutes).<br>
The PCR product was digested for 16 hours in an appropriate buffer with EcoRI<br>
and HindIII restriction enzymes. The DNA fragment was gel purified and ligated<br>
into EcoRI/HindIII-digested pKK vector, yielding the pKK-N473-JUN plasmid.<br>
Insertion of the PCR product was analyzed by EcoRI/HindIII restriction analysis<br>
and by DNA sequencing of the insert.<br>
Example 21<br>
Expression and partial purification of HBcAg-JUN<br>
E. coli strain XL-1 blue was transformed with pKK-HBcAg-JUN. 1 ml<br>
of an overnight culture of bacteria was used to innoculate 100 ml of LB medium<br>
containing 100 g/ml ampicillin. This culture was grown for 4 hours at 37°C<br>
until an OD at 600 nm of approximately 0.8 was reached. Induction of the<br>
synthesis of HBcAg-JUN was performed by addition of IPTG to a final<br>
concentration of 1 mM. After induction, bacteria were further shaken at 37 °C for<br>
16 hours. Bacteria were harvested by centrifugation at 5000 x g for 15 minutes.<br>
The pellet was frozen at -20°C. The pellet was thawed and resuspended in<br>
bacteria lysis buffer (10 mM Na2HPO4 pH 7.0,30 mM NaCl, 0.25% Tween-20,<br>
10 mM EDTA, 10 mM DTT) supplemented with 200 ug/ml lysosyme and 10 ul<br>
of Benzonase (Merck). Cells were incubated for 30 minutes at room temperature<br>
and disrupted using a French pressure cell. Triton X-l 00 was added to the lysate<br>
to a final concentration of 0.2%, and the lysate was incubated for 30 minutes on<br>
ice and shaken occasionally. Figure 4 shows HBcAg-JUN protein expression in<br>
E. coli upon induction with IPTG. E. coli cells harboring pKK-HBcAg-JUN<br>
expression plasmid or a control plasmid were used for induction of HBcAg-JUN<br>
expression with IPTG. Prior to the addition of IPTG, a sample was removed from<br>
the bacteria culture carrying the pKK-HBcAg-JUN plasmid (lane 3) and from a<br>
culture carrying the control plasmid (lane 1). Sixteen hours after addition of<br><br>
IPTG, samples were again removed from the culture containing pKK-HBcAg-<br>
JUN (lane 4) and from the control culture (lane 2). Protein expression was<br>
monitored by SDS-PAGE followed by Coomassie staining.<br>
The lysate was then centrifuged for 30 minutes at 12,000 x g in order to<br>
remove insoluble cell debris. The supernatant and the pellet were analyzed by<br>
Western blotting using a monoclonal antibody against HBcAg (YVS1841,<br>
purchased from Accurate Chemical and Scientific Corp., Westbury, NY, USA),<br>
indicating that a significant amount of HBcAg-JUN protein was soluble (Fig. 5).<br>
Briefly, lysates from E. coli cells expressing HBcAg-JUN and from control cells<br>
were centrifuged at 14,000 x g for 30 minutes. Supernatant (= soluble fraction)<br>
and pellet (= insoluble fraction) were separated and diluted with SDS sample<br>
buffer to equal volumes. Samples were analyzed by SDS-PAGE followed by<br>
Western blotting with anti-HBcAg monoclonal antibody YVS 1841. Lane 1:<br>
soluble fraction, control cells; lane 2: insoluble fraction, control cells; lane 3:<br>
soluble fraction, cells expressing HBcAg-JUN; lane 4: insoluble fraction, cells<br>
expressing HbcAg-JUN.<br>
The cleared cell lysate was used for step-gradient centrifugation using a<br>
sucrose step gradient consisting of a 4 ml 65% sucrose solution overlaid with 3<br>
ml 15% sucrose solution followed by 4 ml of bacterial lysate. The sample was<br>
centrifuged for 3 hrs with 100,000 x g at 4°C. After centrifugation, 1 ml fractions<br>
from the top of the gradient were collected and analyzed by SDS-PAGE followed<br>
by Coomassie staining. (Fig. 6). Lane 1: total E. coli lysate prior to<br>
centrifugation. Lane 1 and 2: fractions 1 and 2 from the top of the gradient.<br>
Lane 4 to 7: fractions 5 to 8 (15% sucrose). The HBcAg-JUN protein was<br>
detected by Coomassie staining.<br>
The HBcAg-JUN protein was enriched at the interface between 15 and<br>
65% sucrose indicating that it had formed a capsid particle. Most of the bacterial<br>
proteins remained in the sucrose-free upper layer of the gradient, therefore step-<br>
gradient centrifugation of the HBcAg-JUN particles led both to enrichment and<br>
xo a partial purification of the particles.<br><br>
Example 22<br>
Covatent Coupling ofltGH-FOS to HBcAg-JUN<br>
In order to demonstrate binding of a protein to HBcAg-JUN particles, we<br>
chose human growth hormone (hGH) fused with its carboxy terminus to the FOS<br>
helix as a model protein (hGH-FOS). HBcAg-JUN particles were mixed with<br>
partially purified hGH-FOS and incubated for 4 hours at 4°C to allow binding of<br>
the proteins. The mixture was then dialyzed overnight against a 3000-fold<br>
volume of dialysis buffer (150 mM NaCl, 10 mM Tris-HCl solution, pH 8.0) in<br>
order to remove DTT present in both the HBcAg-JUN solution and the hGH-FOS<br>
solution and thereby allow covalent coupling of the proteins through the<br>
establishment of disulphide bonds. As controls, the HBcAg-JUN and the hGH-<br>
FOS solutions were also dialyzed against dialysis buffer. Samples from all three<br>
dialyzed protein solutions were analyzed by SDS-PAGE under non-reducing<br>
conditions. Coupling of hGH-FOS to HBcAg-JUN was detected in an anti-hGH<br>
immunoblot (Fig. 7). hGH-FOS bound to HBcAg-JUN should migrate with an<br>
apparent molecular mass of approximately 53 kDa, while unbound hGH-FOS<br>
migrates with an apparent molecular mass of 31 kDa. The dialysate was analyzed<br>
by SDS-PAGE in the absence of reducing agent (lane 3) and in the presence of<br>
reducing agent (lane 2) and detected by Coomassie staining. As a control, hGH-<br>
FOS that had not been mixed with capsid particles was also loaded on the gel in<br>
the presence of reducing agent (lane 1).<br>
A shift of hGH-FOS to a molecular mass of approximately 53 kDa was<br>
observed in the presence of HBcAg-JUN capsid protein, suggesting that efficient<br>
binding of hGH-FOS to HBcAg-JUN had taken place.<br>
Example 23<br>
Insertion of a peptide containing a Lysine residue into the<br>
c/el epitope ofHBcAg(l-149)<br>
The c/el epitope (residues 72 to 88) of HBcAg is located in the tip region<br>
on the surface of the hepatitis B virus capsid (HBcAg). A part of this region<br><br>
(Proline 79 and Alanine 80) was genetically replaced by the peptide Gly-Gly-Lys-<br>
Gly-Gly (HBcAg-Lys construct). The introduced Lysine residue contains a<br>
reactive amino group in its side chain that can be used for intermolecular<br>
chemical crosslinking of HBcAg particles with any antigen containing a free<br>
Cysteine group.<br>
The HBcAg-Lys DNA sequence was generated by PCRs: The two<br>
fragments encoding HBcAg fragments (amino acid residues 1 to 78 and 81 to<br>
149) were amplified separately by PCR. The primers used for these PCRs also<br>
introduced a DNA sequence encoding the Gly-Gly-Lys-Gly-Gly peptide. The<br>
HBcAg (1 to 78) fragment was amplified from pEco63 using primers<br>
EcoRIHBcAg(s) and Lys-HBcAg(as). The HBcAg (81 to 149) fragment was<br>
amplified from pEco63 using primers Lys-HBcAg(s) and HBcAg(l-<br>
149)Hind(as). Primers Lys-HBcAg(as) and Lys-HBcAg(s) introduced<br>
complementary DNA sequences at the ends of the two PCR products allowing<br>
fusion of the two PCR products in a subsequent assembly PCR. The assembled<br>
fragments were amplified by PCR using primers EcoRIHBcAg(s) and HbcAg(l -<br>
149)Hind(as).<br>
For the PCRs, 100 pmol of each oligo and 50 ng of the template DNAs<br>
were used in the 50 ul reaction mixtures with 2 units of Pwo polymerase, 0.1 mM<br>
dNTPs and 2 mM MgSO4. For both reactions , temperature cycling was carried<br>
out as follows: 94°C for 2 minutes; 30 cycles of 94°C (1 minute), 50°C (1<br>
minute), 72°C (2 minutes).<br>
Primer sequences:<br>
EcoRIHBcAg(s):<br>
(5"-CCGGAATTCATGGACATTGACCCTTATAAAG-3:) (SEQ ID NO:79);<br>
Lys-HBcAg(as):<br>
.;5"-CCTAGAGCCACCTTTGCCACCATCTTCTAAATTAGTACCCACCCAG<br><br>
GTAGC-3') (SEQ ID NO:80);<br>
Lys-HBcAg(s):<br>
(5'-GAAGATGGTGGCAAAGGTGGCTCTAGGGACCTAGTAGTCAGTTAT<br>
GTC-3')(SEQ ID NO:81);<br>
HBcAg(l-149)Hind(as):<br>
(5'-CGCGTCCCAAGCTTCTAAACAACAGTAGTCTCCGGAAG-3') (SEQ<br>
ID NO:82).<br>
For fusion of the two PCR fragments by PCR 100 pmol of primers<br>
EcoRIHBcAg(s) and HBcAg(l-149)Hind(as) were used with 100 ng of the two<br>
purified PCR fragments in a 50 l reaction mixture containing 2 units of Pwo<br>
polymerase, 0.1 mM dNTPs and 2 mM MgSCX,. PCR cycling conditions were:<br>
94°C for 2 minutes; 30 cycles of 94-°C (1 minute), 50°C (1 minute), 72°C (2<br>
minutes). The assembled PCR product was analyzed by agarose gel<br>
electrophoresis, purified and digested for 19 hours in an appropriate buffer with<br>
EcoRI and HindIII restriction enzymes. The digested DNA fragment was ligated<br>
into EcoRI/HindIII-digested pKK vector to generate pKK-HBcAg-Lys expression<br>
vector. Insertion of the PCR product into the vector was analyzed by<br>
EcoRI/HindIII restriction analysis and DNA sequencing of the insert.<br>
Example 24<br>
Expression and partial purification of HBcAg-Lys<br>
E. coli strain XL-1 blue was transformed with pKK-HBcAg-Lys. 1 ml of<br>
an overnight culture of bacteria was used to innoculate 100 ml of LB medium<br>
containing 100 ug/ml ampicillin. This culture was grown for 4 hours at 37°C<br>
until an OD at 600 nm of approximately 0.8 was reached. Induction of the<br>
synthesis of HBcAg-Lys was performed by addition of IPTG to a final<br>
concentration of 1 mM. After induction, bacteria were further shaken at 37 °C for<br><br>
16 hours. Bacteria were harvested by centrifugation at 5000 x g for 15 minutes.<br>
The pellet was frozen at -20°C. The pellet was thawed and resuspended in<br>
bacteria lysis buffer (10 mM Na2HPO4, pH 7.0, 30 mM NaCl, 0.25% Tween-20,<br>
10 mM EDTA, 10 mM DTT) supplemented with 200 g/ml lysosyme and 10 l<br>
of Benzonase (Merck). Cells were incubated for 30 minutes at room temperature<br>
and disrupted using a French pressure cell. Triton X-100 was added to the lysate<br>
to a final concentration of 0.2%, and the lysate was incubated for 30 minutes on<br>
ice and shaken occasionally. E. coli cells harboring pKK-HBcAg-Lys expression<br>
plasmid or a control plasmid were used for induction of HBcAg-Lys expression<br>
with IPTG. Prior to the addition of IPTG, a sample was removed from the<br>
bacteria culture carrying the pKK-HBcAg-Lys plasmid and from a culture<br>
carrying the control plasmid. Sixteen hours after addition of IPTG, samples were<br>
again removed from the culture containing pKK-HBcAg-Ly s and from the control<br>
culture. Protein expression was monitored by SDS-PAGE followed by<br>
Coomassie staining.<br>
The lysate was then centrifuged for 30 minutes at 12,000 x g in order to<br>
remove insoluble cell debris. The supernatant and the pellet were analyzed by<br>
Western blotting using a monoclonal antibody against HBcAg (YVS1841,<br>
purchased from Accurate Chemical and Scientific Corp., Westbury, NY, USA),<br>
indicating that a significant amount of HBcAg-Lys protein was soluble. Briefly,<br>
lysates from E. coli cells expressing HBcAg-Lys and from control cells were<br>
centrifuged at 14,000 x g for 30 minutes. Supernatant (= soluble fraction) and<br>
pellet (= insoluble fraction) were separated and diluted with SDS sample buffer<br>
to equal volumes. Samples were analyzed by SDS-PAGE followed by Western<br>
blotting with anti-HBcAg monoclonal antibody YVS 1841.<br>
The cleared cell lysate was used for step-gradient centrifugation using a<br>
sucrose step gradient consisting of a 4 ml 65% sucrose solution overlaid with 3<br>
ml 15% sucrose solution followed by 4 ml of bacterial lysate. The sample was<br>
centrifuged for 3 hrs with 100,000 x gat 4°C. After centrifugation, 1 ml fractions<br>
from the top of the gradient were collected and analyzed by SDS-PAGE followed<br><br>
by Coomassie staining. The HBcAg-Lys protein was detected by Coomassie<br>
staining.<br>
The HBcAg-Lys protein was enriched at the interface between 15 and<br>
65% sucrose indicating that it had formed a capsid particle. Most of the bacterial<br>
proteins remained in the sucrose-free upper layer of the gradient, therefore step-<br>
gradient centrifugation of the HBcAg-Lys particles led both to enrichment and to<br>
a partial purification of the particles.<br>
Example 25<br>
Chemical coupling of FLAGpeptide to HBcAg-Lys<br>
using the heterobifunctional crosslinker SPDP<br>
Synthetic FLAG peptide with a Cysteine residue at its amino terminus<br>
(amino acid sequence CGGDYKDDDDK) was coupled chemically to purified<br>
HBcAg-Lys particles in order to elicit an immune response against the FLAG<br>
peptide. 600 l of a 95% pure solution of HBcAg-Lys particles (2 mg/ml) were<br>
incubated for 30 minutes at room temperature with the heterobifunctional<br>
crosslinker N-Succinimidyl 3-(2-pyridyldithio) propionate (SPDP) (0.5 mM).<br>
After completion of the reaction, the mixture was dialysed overnight against 1<br>
liter of 50 mM Phosphate buffer (pH 7.2) with 150 mM NaCl to remove free<br>
SPDP. Then 500 l of derivatized HBcAg-Lys capsid (2 mg/ml) were mixed with<br>
0.1 mM FLAG peptide (containing an ammo-terminal cysteine) in the presence<br>
of 10 mM EDTA to prevent metal-catalyzed sufhydryl oxidation. The reaction<br>
was monitored through the increase of the optical density of the solution at 343<br>
nm due to the release of pyridine-2-thione from SPDP upon reaction with the free<br>
cysteine of the peptide. The reaction of derivatised Lys residues with the peptide<br>
was complete after approximately 30 minutes.<br>
The FLAG decorated particles were injected into mice.<br><br>
Example 26<br>
Construction of pMPSV-gpl40cys<br>
The gpl40 gene was amplified by PCR from pCytTSgpl40FOS using<br>
oligos gpl40CysEcoRI and SalIgpl40. For the PCRs, 100 pmol of each oligo<br>
and 50 ng of the template DNAs were used in the 50 u.1 reaction mixtures with 2<br>
units of Pwo polymerase, 0.1 mM dNTPs and 2 mM MgSO4. For both reactions<br>
, temperature cycling was carried out as follows: 94°C for 2 minutes; 30 cycles<br>
of 94°C (0.5 minutes), 55°C (0.5 minutes), 72°C (2 minutes).<br>
The PCR product was purified using QiaEXII kit, digested with<br>
Sall/EcoRI and ligated into vector pMPSVHE cleaved with the same enzymes.<br>
Oligo sequences:<br>
Gpl40CysEcoRI:<br>
5'-GCCGAATTCCTAGCAGCTAGCACCGAATTTATCTAA-3' (SEQ ID<br>
NO:83);<br>
SalIgpl40<br>
5'- GGTTAAGTCGACATGAGAGTGAAGGAGAAATAT-3' (SEQ ID NO:84).<br>
Example 27<br>
Expression ofpMPSVgpl40Cys<br>
pMPSVgpHOCys (20 g) was linearized by restriction digestion. The<br>
reaction was stopped by phenol/chloroform extraction, followed by an<br>
isopropanol precipitation of the linearized DNA. The restriction digestion was<br>
evaluated by agarose gel eletrophoresis. For the transfection. 5.4 ug of linearized<br>
pMPSVgpl40-Cys was mixed with 0.6 ug of linearized pSV2Neo in 30 ul H2O<br>
and 30 ul of 1 M CaCU solution was added. After addition of 60 ul phosphate<br>
buffer (50 mM HEPES, 280 mM NaCl, 1.5 mM Na2 HPO4, pH 7.05), the solution<br>
was vortexed for 5 seconds, followed by an incubation at room temperature for<br><br>
25 seconds. The solution was immediately added to 2 ml HP-1 medium<br>
containing 2% FCS (2% FCS medium). The medium of an 80% confluent<br>
BHK21 cell culture (6-well plate) was then replaced by the DNA containing<br>
medium. After an incubation for 5 hours at 37°C in a CO2 incubator, the DNA<br>
containing medium was removed and replaced by 2 ml of 15% glycerol in 2%<br>
FCS medium. The glycerol containing medium was removed after a 30 second<br>
incubation phase, and the cells were washed by rinsing with 5 ml of HP-1<br>
medium containing 10% FCS. Finally 2 ml of fresh HP-1 medium containing<br>
10% FCS was added.<br>
Stably transfected cells were selected and grown in selection medium<br>
(HP-1 medium supplemented with G418) at 37°C in a CO2 incubator. When the<br>
mixed population was grown to confluency, the culture was split to two dishes,<br>
followed by a 12 h growth period at 37°C. One dish of the cells was shifted to<br>
30° C to induce the expression of soluble GP140-FO5. The other dish was kept<br>
at 37°C.<br>
The expression of soluble GP140-Cys was determined by Western blot<br>
analysis. Culture media (0.5 ml) was methanol/chloroform precipitated, and the<br>
pellet was resuspended in SDS-PAGE sample buffer. Samples were heated for<br>
5 minutes at 95°C before being applied to a 15% acrylamide gel. After<br>
SDS-PAGE, proteins were transferred to Protan nitrocellulose membranes<br>
(Schleicher &amp; Schuell, Germany) as described by Bass and Yang, in Creighton,<br>
T.E., ed., Protein Function: A Practical Approach, 2nd Edn., IRL Press, Oxford<br>
(1997), pp. 29-55. The membrane was blocked with 1 % bovine albumin (Sigma)<br>
in TBS (IOXTBS per liter: 87.7 gNaCl, 66.1gTrizma hydrochloride (Sigma) and<br>
9.7 g Trizma base (Sigma), pH 7.4) for 1 hour at room temperature, followed by<br>
an incubation with an anti-GP140 or GP-160 antibody for 1 hour. The blot was<br>
washed 3 times for 10 minutes with TBS-T (TBS with 0.05% Tween20), and<br>
incubated for 1 hour with an alkaline-phosphatase-anti-<br>
mouse/rabbit/monkey/human IgG conjugate. After washing 2 times for 10<br>
minutes with TBS-T and 2 times for 10 minutes with TBS, the development<br><br>
reaction was carried out using alkaline phosphatase detection reagents (10ml AP<br>
buffer (100 mM Tris/HCl, 100 mM NaCl, pH 9.5) with 50 ul NBT solution (7.7%<br>
Nitro Blue Tetrazolium (Sigma) in 70% dimethylformamide) and 37 ul of<br>
X-Phosphate solution (5% of 5-bromo-4-chloro-3-indolyl phosphate in<br>
dimethylformamide).<br>
Example 28<br>
Purification ofgpl40Cys<br>
An anti-gpl20 antibody was covalently coupled to a NHS/EDC activated<br>
dextran and packed into a chromatography column. The supernatant, containing<br>
GP140Cys is loaded onto the column and after sufficient washing, GP140Cys was<br>
eluted using 0.1 M HC1. The eluate was directly neutralized during collection<br>
using 1 M Tris pH 7.2 in the collection tubes.<br>
Disulfide bond formation might occur during purification, therefore the<br>
collected sample is treated with 10 mM DTT in 10 mM Tris pH 7.5 for 2 hours<br>
at25°C.<br>
DTT is remove by subsequent dialysis against 10 mM Mes; 80 mM NaCl<br>
pH 6.0. Finally GP 1 AQCys is mixed with alphavirus particles containing the JUN<br>
residue in E2 as described in Example 16.<br>
Example 29<br>
Construction of PLA2-Cys<br>
The PLA2 gene was amplified by PCR from pAV3PLAfos using oligos<br>
EcoRIPLA and PLA-Cys-hind. For the PCRs, 100 pmol of each oligo and 50 ng<br>
of the template DNAs were used in the 50 ul reaction mixtures with 2 units of<br>
Pwo polymerase, 0.1 mM dNTPs and 2 mM MgSO4. For both reactions ,<br>
temperature cycling was carried out as follows: 94°C for 2 minutes; 30 cycles of<br>
94°C (0.5 minutes), 55°C (0.5 minutes), 72°C (2 minutes).<br>
The PCR product was purified using QiaEXII kit, digested with<br>
EcoRI/HinDIII and ligated into vector pAV3 cleaved with the same enzymes.<br><br>
Oligos<br>
EcoRIPLA:<br>
5'-TAACCGAATTCAGGAGGTAAAAAGATATGG-3' ,'SEQ ID NO:85)<br>
PLACys-hind:<br>
5'-GAAGTAAAGCTTTTAACCACCGCAACCACCAGAAG-3' (SEQ ID<br>
NO:86).<br>
Example 30<br>
Expression and purification of PLA-cys<br>
For cytoplasmic production of Cys tagged proteins. E. coli XL-1-Blue<br>
strain was transformed with the vectors pAV3: :PL A and pPL A-Cys. The culture<br>
was incubated in rich medium in the presence of ampicillin at 3 7 °C with shaking.<br>
At an optical density (550rim) of, 1 mM IPTG was added and incubation was<br>
continued for another 5 hours. The cells were harvested by centrifugation,<br>
resuspended in an appropriate buffer (e.g. Tris-HCl, pH 7.2, 150 mM NaCl)<br>
containing DNase, RNase and lysozyme, and disrupted by passage through a<br>
french pressure cell. After centrifugation (Sorvall RC-5C, SS34 rotor,<br>
15000 rpm, 10 min, 4°C), the pellet was resuspended in 25 ml inclusion body<br>
wash buffer (20 mM tris-HCl, 23% sucrose, 0.5% Triton X-100, 1 mM EDTA,<br>
pH8) at 4°C and recentrifuged as described above. This procedure was repeated<br>
until the supernatant after centrifugation was essentially clear. Inclusion bodies<br>
were resuspended in 20 ml solubilization buffer (5.5 M guanidinium<br>
hydrochloride, 25 mM tris-HCl, pH 7.5) at room temperarure and insoluble<br>
material was removed by centrifugation and subsequent passage of the<br>
supernatant through a sterile filter (0.45 um). The protein solution was kept at<br>
4 " C for at least 10 hours in the presence of 10 mM EDTA and 100 mM DTT and<br>
then dialyzed three times against 10 volumes of 5.5 M guanidinium<br>
hydrochloride, 25 mM tris-HCl, 10 mM EDTA, pH 6. The solution was dialyzed<br>
twice against 51 2 M urea, 4 mM EDTA, 0.1 M NH4C1, 20 mM sodium borate<br><br>
• pH 8.3) in the presence of an appropriate redox shuffle (oxidized<br>
glutathione/reduced glutathione; cystine/cysteine). The refolded protein was then<br>
applied to an ion exchange chromatography. The protein was stored in an<br>
appropriate buffer with a pH above 7 in the presence of 2-10 mM DTT to keep<br>
the cysteine residues in a reduced form. Prior to coupling of the protein with the<br>
alphavirus particles, DTT was removed by passage of the protein solution through<br>
a Sephadex G-25 gel filtration column.<br><br>
We claim:<br>
1.	A composition comprising:<br>
a)	a non-naturally occurring molecular scaffold comprising:<br>
(i) a core particle selected from the group consisting of:<br>
(1)	a core particle of non-natural origin; and<br>
(2)	a core particle of natural origin; and wherein said<br>
core particle comprises a virus-like particle or a recombinant<br>
form thereof;<br>
(ii) an organizer comprising at least one first attachment<br>
site,<br>
wherein said organizer is connected to said core particle<br>
by at least one covalent bond; and<br>
b)	an antigen or antigenic determinant with at least one second<br>
attachment site, said second attachment site being selected from the group<br>
consisting of:<br>
(i) an attachment site not naturally occurring with said<br>
antigen or antigenic determinant; and<br>
(ii) an attachment site naturally occurring with said antigen<br>
or antigenic determinant,<br>
wherein said second attachment site is capable of association through<br>
at least one non-peptide bond to said first attachment site; and<br>
wherein said antigen or antigenic determinant and said scaffold interact<br>
through said association to form an ordered and repetitive antigen array,<br>
wherein said antigen is selected from the group consisting of (a) proteins<br>
suited to induce an immune response against allergens and (b) proteins suited<br>
to induce an immune response against cancer cells.<br>
2.	The composition as claimed in Claim 1, wherein:<br>
a)	said core particle is a virus-like particle or a recombinant form<br>
thereof; and<br>
b)	said organizer is a polypeptide or residue thereof; and<br>
c)	said second attachment site is a polypeptide or residue thereof.<br><br>
3. The composition as claimed in Claim 2, wherein said first and/or said<br>
second attachment site comprises:<br>
a)	an antigen;<br>
b)	an antibody or antibody fragment;<br>
c)	biotin;<br>
d)	avidin;<br>
e)	strepavidin;<br>
f)	a receptor;<br>
g)	a receptor ligand;<br>
h) a ligand-binding protein;<br>
i) a ligand;<br>
j) an interacting leucine zipper polypeptide;<br>
k) an amino group;<br>
1) a chemical group reactive with an amino group;<br>
m) a carboxyl group;<br>
n) a chemical group reactive with a carboxyl group;<br>
o) a sulfhydryl group;<br>
p) a chemical group reactive with a sulfhydryl group; or<br>
q) a combination thereof.<br>
4.	The composition as claimed in Claim 3, wherein said second attachment<br>
site does not naturally occur with said antigen or antigenic<br>
determinant.<br>
5.	The composition as claimed in Claim 2, where in said core particle is a<br>
recombinant alphavirus; and wherein said recombinant alphavirus is<br>
Sindbis virus and said first attachment site and said second attachment<br>
site each comprise an interacting leucine zipper polypeptide, wherein<br>
said first attachment site and said second attachment site are the JUN<br>
and/or FOS leucine zipper polypeptides.<br>
6.	The composition as claimed in Claim 2, wherein said first attachment site<br>
is an amino group and said second attachment site is a sulfhydryl<br>
group.<br><br>
7.	The composition as claimed in Claim 2, wherein said virus-like particle is<br>
a hepatitis B virus capsid protein.<br>
8.	The composition as claimed in Claim 2, wherein said virus-like particle is<br>
a Measles virus capsid protein.<br>
9.	The composition as claimed in Claim 2, wherein said core particle is<br>
selected from the group consisting of:<br>
a)	recombinant proteins of Rotavirus,<br>
b)	recombinant proteins of Norwalk virus,<br>
c)	recombinant proteins of Alphavirus,<br>
d)	recombinant proteins of Foot and Mouth Disease virus,<br>
e)	recombinant proteins of Retrovirus,<br>
f)	recombinant proteins of Hepatitis B virus,<br>
g)	recombinant proteins of Tobacco mosaic virus,<br>
h)	recombinant proteins of Flock House Virus, and<br>
i)	recombinant proteins of human Papilomavirus.<br>
10.	The composition as claimed in Claim 1, wherein said core particle is of<br>
non-natural origin.<br>
11.	The composition as claimed in Claim 1, wherein said antigen is:<br>
a)	a recombinant protein of breast cancer cells,<br>
b)	a recombinant protein of kidney cancer cells,<br>
c)	a recombinant protein of prostate cancer cells,<br>
d)	a recombinant protein of skin cancer cells,<br>
e)	a recombinant protein of brain cancer cells,<br>
f)	a recombinant protein of leukemia cells,<br>
g)	a recombinant protein of bee sting allergy,<br>
h)	a recombinant protein of nut allergy,<br>
i) a recombinant protein of food allergies, or<br>
j) a recombinant protein of asthma..<br><br>
12.	The composition as claimed in Claim 1, wherein said antigen or antigenic<br>
determinant is selected from proteins suited to induce an immune<br>
response against cancer cells.<br>
13.	The composition as claimed in Claim 1, wherein said antigen or antigenic<br>
determinant is Her2.<br>
14.	The composition as claimed in Claim 1, wherein said antigen or antigenic<br>
determinant is selected from proteins suited to induce an immune<br>
response against allergens.<br>
15.	The composition as claimed in Claim 1, wherein said antigen or antigenic<br>
determinant is a recombinant protein of nut allergy.<br>
16.	The composition as claimed in Claim 1, wherein said antigen or antigenic<br>
determinant is a recombinant protein of asthma.<br>
17.	The composition as claimed in any one of Claim 1 to 16, wherein said first<br>
attachment site is a lysine residue and said second attachment site is a<br>
cysteine residue.<br>
18.	A process for producing a non-naturally occurring, ordered and repetitive<br>
antigen array comprising:<br>
a) providing a non-naturally occurring molecular scaffold<br>
comprising:<br>
(i) a core particle selected from the group consisting of:<br>
(1)	a core particle of non-natural origin; and<br>
(2)	a core particle of natural origin; and wherein said<br>
core particle comprises a virus-like particle or a recombinant<br>
form thereof; and<br>
(ii) an organizer comprising at least one first attachment<br>
site,<br>
wherein said organizer is connected to said core particle by at least one<br>
covalent bond; and<br><br>
b)	providing an antigen or antigenic determinant with at least one<br>
second attachment site, wherein said antigen is selected from the group<br>
consisting of (a) proteins suited to induce an immune response against<br>
allergens and (b) proteins suited to induce an immune response against cancer<br>
cells, and wherein said second attachment site being selected from the group<br>
consisting of:<br>
(i) an attachment site not naturally occurring with said<br>
antigen or antigenic determinant; and<br>
(ii) an attachment site naturally occurring with said antigen<br>
or antigenic determinant,<br>
wherein said second attachment site is capable of association through at least<br>
one non-peptide bond to said first attachment site; and<br>
c)	combining said non-naturally occurring molecular scaffold and<br>
said antigen or antigenic determinant,<br>
wherein said antigen or antigenic determinant and said scaffold interact<br>
through said association to form an ordered and repetitive antigen array.<br>
19.	A pharmaceutical composition comprising:<br>
a)	the composition as claimed in any one of Claim 1 to 17; and<br>
b)	an acceptable pharmaceutical carrier or an adjuvant.<br>
20.	A pharmaceutical composition as claimed in claim 19 is a vaccine<br>
composition.<br>
21.	The vaccine composition as claimed in Claim 20, wherein said first<br>
attachment site is an amino group and said second attachment site is a<br>
sulfhydryl group.<br>
22.	The vaccine composition as claimed in Claim 20, wherein said core<br>
particle comprises a Hepatitis B virus-like particle.<br><br>
23. The vaccine composition as claimed in Claim 20, wherein said core<br>
particle comprises a measles virus-like particle.<br><br>
The invention provides compositions and processes for the production of ordered and repetitive antigen or antigenic determinant<br>
arrays. The compositions of the invention are useful for the production of vaccines for the prevention of infectious diseases, the treatment<br>
of allergies and the treatment of cancers. Various embodiments of the invention provide for a virus, virus-like particle, viral capsid particle,<br>
phage or recombinant form thereof coated with any desired antigen in a highly ordered and repetitive fashion as the result of specific<br>
interactions. In one specific embodiment, a versatile new technology based on a cassette-type system (Alpha Vaccine Technology) allows<br>
production of antigen coated viral particles. Other specific embodiments allow the production of antigen coated hepatitis B virus-like<br>
particles or antigen coated Measles virus-like particles.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDEtNjg2LUtPTC1DT1JSRVNQT05ERU5DRS5wZGY=" target="_blank" style="word-wrap:break-word;">IN-PCT-2001-686-KOL-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDEtNjg2LUtPTC1GT1JNIDI3LnBkZg==" target="_blank" style="word-wrap:break-word;">IN-PCT-2001-686-KOL-FORM 27.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjg2LWtvbC1ncmFudGVkLWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-686-kol-granted-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjg2LWtvbC1ncmFudGVkLWFzc2lnbm1lbnQucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-686-kol-granted-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjg2LWtvbC1ncmFudGVkLWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-686-kol-granted-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjg2LWtvbC1ncmFudGVkLWNvcnJlc3BvbmRlbmNlLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-686-kol-granted-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjg2LWtvbC1ncmFudGVkLWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-686-kol-granted-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjg2LWtvbC1ncmFudGVkLWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-686-kol-granted-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjg2LWtvbC1ncmFudGVkLWV4YW1pbmF0aW9uIHJlcG9ydC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-686-kol-granted-examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjg2LWtvbC1ncmFudGVkLWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-686-kol-granted-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjg2LWtvbC1ncmFudGVkLWZvcm0gMTMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-686-kol-granted-form 13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjg2LWtvbC1ncmFudGVkLWZvcm0gMTgucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-686-kol-granted-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjg2LWtvbC1ncmFudGVkLWZvcm0gMi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-686-kol-granted-form 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjg2LWtvbC1ncmFudGVkLWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-686-kol-granted-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjg2LWtvbC1ncmFudGVkLWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-686-kol-granted-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjg2LWtvbC1ncmFudGVkLWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-686-kol-granted-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjg2LWtvbC1ncmFudGVkLXJlcGx5IHRvIGV4YW1pbmF0aW9uIHJlcG9ydC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-686-kol-granted-reply to examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjg2LWtvbC1ncmFudGVkLXNwZWNpZmljYXRpb24ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-686-kol-granted-specification.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="228094-an-insitu-method-of-nondestructive-ultrasonic-measurement-of-r-bar-on-corelating-device-features-with-time-of-flights-of-ultrasonic-wave-in-steel-sheets.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="228096-her-2-polypeptide-and-use-thereof.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>228095</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/686/KOL</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>05/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>30-Jan-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>28-Jan-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>28-Jun-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CYTOS BIOTECHNOLOGY AG</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>WAGISTRASSE 25, CH-8802 SCHLIEREN</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>RENNER, WOLFGANG, A</td>
											<td>GRANZSTEIG 9, CH-8952 KILCHBERG</td>
										</tr>
										<tr>
											<td>2</td>
											<td>BACHMANN, MARTIN F</td>
											<td>GOLDCKERWEG 8, CH-8472 SEUZACH</td>
										</tr>
										<tr>
											<td>3</td>
											<td>HENNEKE, FRANK</td>
											<td>AUFWIESENSTRASSE 14, CH-8305 DIETILIKON</td>
										</tr>
										<tr>
											<td>4</td>
											<td>NIEBA, LARS</td>
											<td>HUBACKERWEG 12A, CH-1153 REINACH</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A61K 39/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IB99/01925</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>1999-11-30</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/110,414</td>
									<td>1998-11-30</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/142,788</td>
									<td>1999-07-08</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/228095-ordered-molecular-presentation-of-antigens-method-of-preparation-and-use by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:39:54 GMT -->
</html>
