<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/228063-rapid-analysis-of-variations-in-a-genome by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:37:46 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 228063:RAPID ANALYSIS OF VARIATIONS IN A GENOME</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">RAPID ANALYSIS OF VARIATIONS IN A GENOME</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention provides a method useful for determining the sequence of large numbers of loci of interest on a single or multiple chromosomes. The method utilizes an oligonucleotide primer that contains a recognition site for a restriction enzyme such that digestion with the restriction enzyme generates a 5&#x27; overhang containing the locus of interest. The 5&#x27; overhang is used as a template to incorporate nucleotides, which can be detected. The method is especially amenable to the analysis of large numbers of sequences, such as single nucleotide polymorphisms, from one sample of nucleic acid.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>RAPID ANALYSIS OF VARIATIONS IN A GENOME<br>
CROSS-REFERENCE TO RELATED APPLICATIONS<br>
[0001] This application claims priority to U.S. Patent Application No.<br>
10/093,618, filed March 11,2002, and provisional U.S. Patent Application Nos.<br>
60/360,232 and 60/378,354, filed March 1,2002, and May 8,2002, respectively.<br>
The contents of these applications are hereby incorporated by reference in their<br>
entirety herein.<br>
BACKGROUND OF THE INVENTION<br>
FIELD OF THE INVENTION<br>
[0002] The present invention is directed to a rapid method for determining the<br>
sequence of nucleic acid. The method is especially useful for genotyping. and for<br>
the detection of one to tens to hundreds to thousands of single nucleotide<br>
polymorphisms (SNPs) or mutations on single or on multiple chromosomes, and<br>
for the detection of chromosomal abnormalities, such as truncations,<br>
transversions, trisomies, and monosomies.<br>
BACKGROUND<br>
[0003] Sequence variation among individuals comprises a continuum from<br>
deleterious disease mutations to neutral polymorphisms. There are more than<br>
three thousand genetic diseases currently known including Duchenne Muscular<br>
Dystrophy, Alzheimer's Disease, Cystic Fibrosis, and Huntington's Disease (D.N.<br>
Cooper and M. Krawczak, "Human Genome Mutations," BIOS Scientific<br>
Publishers, Oxford (1993)). Also, particular DNA sequences may predispose<br>
individuals to a variety of diseases such as obesity, arteriosclerosis, and various<br>
types of cancer, including breast, prostate, and colon. In addition, chromosomal<br>
abnormalities, such as trisomy 21, which results in Down's Syndrome, trisomy 18,<br>
which results in Edward's Syndrome, trisomy 13, which results in Patau<br>
Syndrome, monosomy X, which results in Turner's Syndrome, and other sex<br>
aneuploidies, account for a significant portion of the genetic defects in liveborn<br><br>
human beings. Knowledge of gene mutations, chromosomal abnormalities, and<br>
variations in gene sequences, such as single nucleotide polymorphisms (SNPs),<br>
will help to understand, diagnose, prevent, and treat diseases.<br>
[0004] Most frequently, sequence variation is seen in differences in the<br>
lengths of repeated sequence elements, such as minisatellites and microsatellites,<br>
as small insertions or deletions, and as substitutions of the individual bases.<br>
Single nucleotide polymorphisms (SNPs) represent the most common form of<br>
sequence variation; three million common SNPs with a population frequency of<br>
over 5% have been estimated to be present in the human genome. Small deletions<br>
or insertions, which usually cause frameshift mutations, occur on average, once in<br>
every 12 kilobases of genomic DNA (Wang, D.G. et al, Science 280:1077-1082<br>
(1998)). A genetic map using these polymorphisms as a guide is being developed<br>
(http://research.marshfieldclinic.org/genetics/; internet address as of January 10,<br>
2002).<br>
[0005] The nucleic acid sequence of the human genome was published in<br>
February, 2001, and provides a genetic map of unprecedented resolution,<br>
containing several hundred thousand SNP markers, and a potential wealth of<br>
information on human diseases (Venter et al., Science 291:1304-1351 (2001);<br>
International Human Genome Sequencing Consortium, Nature 409:860-921<br>
(2001)). However, the length of DNA contained within the human chromosomes<br>
totals over 3 billion base pairs so sequencing the genome of every individual is<br>
impractical. Thus, it is imperative to develop high throughput methods for rapidly<br>
determining the presence of allelic variants of SNPs and point mutations, which<br>
predispose to or cause disease phenotypes. Efficient methods to characterize<br>
functional polymorphisms that affect an individual's physiology, psychology,<br>
audiology, opthamology, neurology, response to drugs, drug metabolism, and<br>
drug interactions also are needed.<br>
[0006] . Several techniques are widely used for analyzing and detecting genetic<br>
variations, such as DNA sequencing, restriction fragment length polymorphisms<br>
(RFLP), DNA hybridization assays, including DNA microarrays and peptide<br><br>
nucleic acid analysis, and the Protein Truncation Test (PTT), all of which have<br>
limitations. Although DNA sequencing is the most definitive method, it is also<br>
the most time consuming and expensive. Often, the entire coding sequence of a<br>
gene is analyzed even though only a small fraction of the coding sequence is of<br>
interest. In most instances, a limited number of mutations in any particular gene<br>
account for the majority of the disease phenotypes.<br>
[0007] For example, the cystic fibrosis transmembrane conductance regulator<br>
(CFTR) gene is composed of 24 exons spanning over 250,000 base pairs<br>
(Rommens et al., Science 245:1059-1065 (1989); Riordan et al., Science<br>
245:1066-73 (1989)). Currently, there are approximately 200 mutations in the<br>
CFTR gene that are associated with a disease state of Cystic Fibrosis. Therefore,<br>
only a very small percentage of the reading frame for the CFTR gene needs to be<br>
analyzed. Furthermore, a total of 10 mutations make up 75.1% of all known<br>
disease cases. The deletion of a single phenylalanine residue, F508, accounts for<br>
66% of all Cystic Fibrosis cases in Caucasians.<br>
[0008] Hybridization techniques, including Southern Blots, Slot Blots, Dot<br>
Blots, and DNA microarrays, are commonly used to detect genetic variations<br>
(Molecular Cloning, A Laboratory Manual, Cold Spring Harbor Laboratory Press,<br>
Third Edition (2001). In a typical hybridization assay, an unknown nucleotide<br>
sequence ("the target") is analyzed based on its affinity for another fragment with<br>
a known nucleotide sequence ("the probe"). If the two fragments hybridize under<br>
"stringent conditions," the sequences are thought to be complementary, and the<br>
sequence of the target fragment may be inferred from "the probe" sequence.<br>
[0009] However, the results from a typical hybridization assay often are<br>
difficult to interpret. The absence or presence of a hybridization signal is<br>
dependent upon the definition of "stringent conditions." Any number of variables<br>
may be used to raise or lower stringency conditions such as salt concentration, the<br>
presence or absence of competitor nucleotide fragments, the number of washes<br>
performed to remove non-specific binding and the time and temperature at which<br>
the hybridizations are performed. Commonly, hybridization conditions must be<br><br>
optimized for each "target" nucleotide fragment, which is time-consuming, and<br>
inconsistent with a high throughput method. A high degree of variability is often<br>
seen in hybridization assays, as well as a high proportion of false positives.<br>
Typically, hybridization assays function as a screen for likely candidates but a<br>
positive corifirmation requires DNA sequencing analysis.<br>
[0010] Several techniques for the detection of mutations have evolved based<br>
on the principal of hybridization analysis. For example, in the primer extension<br>
assay, the DNA region spanning the nucleotide of interest is amplified by PCR, or<br>
any other suitable amplification technique. After amplification, a primer is<br>
hybridized to a target nucleic acid sequence, wherein the last nucleotide of the 3'<br>
end of the primer anneals immediately 5' to the nucleotide position on the target<br>
sequence that is to be analyzed. The annealed primer is extended by a single,<br>
labeled nucleotide triphosphate. The incorporated nucleotide is then detected.<br>
[0011] There are several limitations to the primer extension assay. First, the<br>
region of interest must be amplified prior to primer extension, which increases the<br>
time and expense of the assay. Second, PCR primers and dNTPs must be<br>
completely removed before primer extension, and residual contaminants can<br>
interfere with the proper analysis of the results. Third, and the most restrictive<br>
aspect of the assay, is that the primer is hybridized to the DNA template, which<br>
requires optimization of conditions for each primer, and for each sequence that is<br>
analyzed. Hybridization assays have a low degree of reproducibility, and a high<br>
degree of non-specificity.<br>
[0012] The Peptide Nucleic Acid (PNA) affinity assay is a derivative of<br>
traditional hybridization assays (Nielsen et al., Science 254:1497-1500 (1991);<br>
Egholm et al., J. Am. Chem. Soc. 114:1895-1897 (1992); James et al., Protein<br>
Science 3:1347-1350 (1994)). PNAs are structural DNA mimics that follow<br>
Watson-Crick base pairing rules, and are used in standard DNA hybridization<br>
assays. PNAs display greater specificity in hybridization assays because a<br>
PNA/DNA mismatch is more destabilizing than a DNA/DNA mismatch and<br>
complementary PNA/DNA strands form stronger bonds than complementary<br><br>
DNA/DNA strands. However, genetic analysis using PNAs still requires a<br>
laborious hybridization step, and as such, is subject to a high degree of non-<br>
specificity and difficulty with reproducibility.<br>
[0013] Recently, DNA microarrays have been developed to detect genetic<br>
variations and polymorphisms (Taton et al., Science 289:1757-60,2000; Lockhart<br>
et al., Nature 405:827-836 (2000); Gerhold et al., Trends in Biochemical Sciences<br>
24:168-73 (1999); Wallace, R.W., Molecular Medicine Today 3:384-89 (1997);<br>
Blanchard and Hood, Nature Biotechnology 149:1649 (1996)). DNA microarrays<br>
are fabricated by high-speed robotics, on glass or nylon substrates, and contain<br>
DNA fragments with known identities ("the probe"). The microarrays are used<br>
for matching known and unknown DNA fragments ("the target") based on<br>
traditional base-pairing rules. The advantage of DNA microarrays is that one<br>
DNA chip may provide information on thousands of genes simultaneously.<br>
However, DNA microarrays are still based on the principle of hybridization, and<br>
as such, are subject to the disadvantages discussed above.<br>
[0014] The Protein Truncation Test (PTT) is also commonly used to detect<br>
genetic polymorphisms (Roest et al., Human Molecular Genetics 2:1719-1721,<br>
(1993); Van Der Luit et al., Genomics 20:1-4 (1994); Hogervorst et al., Nature<br>
Genetics 10:208-212 (1995)). Typically, in the PTT, the gene of interest is PCR<br>
amplified, subjected to in vitro transcription/translation, purified, and analyzed by<br>
polyacrylamide gel electrophoresis. The PTT is useful for screening large<br>
portions of coding sequence and detecting mutations that produce stop codons,<br>
which significantly diminish the size of the expected protein. However, the PTT<br>
is not designed to detect mutations that do not significantly alter the size of the<br>
protein.<br>
[0015] Thus, a need still exists for a rapid method of analyzing DNA,<br>
especially genomic DNA suspected of having one or more single nucleotide<br>
polymorphisms or mutations.<br>
BRIEF SUMMARY OF THE INVENTION<br><br>
[0016] The invention is directed to a method for determining a sequence of a<br>
locus of interest, the method comprising: (a) amplifying a locus of interest on a<br>
template DNA using a first and second primers, wherein the second primer<br>
contains a recognition site for a restriction enzyme such that digestion with the<br>
restriction enzyme generates a 5' overhang containing the locus of interest; (b)<br>
digesting the amplified DNA with the restriction enzyme that recognizes the<br>
recognition site on the second primer; (c) incorporating a nucleotide into the<br>
digested DNA of (b) by using the 5' overhang containing the locus of interest as a<br>
template; and (d) deterrnining the sequence of the locus of interest by determining<br>
the sequence of the DNA of (c).<br>
[0017] The invention is also directed to a method for determining a sequence<br>
of a locus of interest, said method comprising: (a) amplifying a locus of interest<br>
on a template DNA using a first and second primers, wherein the second primer<br>
contains a portion of a recognition site for a restriction enzyme, wherein a full<br>
recognition site for the restriction enzyme is generated upon amplification of the<br>
template DNA such that digestion with the restriction enzyme generates a 5'<br>
overhang containing the locus of interest; (b) digesting the amplified DNA with<br>
the restriction enzyme that recognizes the full recognition site generated by the<br>
second primer and the template DNA; (c) incorporating a nucleotide into the<br>
digested DNA of (b) by using the 5' overhang containing the locus of interest as a<br>
template; and determining the sequence of the locus of interest by determining the<br>
sequence of the DNA of (c).<br>
[0018] The invention also is directed to a method for determining a sequence<br>
of a locus of interest, said method comprising (a) replicating a region of DNA<br>
comprising a locus of interest from a template polynucleotide by using a first and<br>
a second primer, wherein the second primer contains a sequence that generates a<br>
recognition site for a restriction enzyme such that digestion with the restriction<br>
enzyme generates a 5' overhang containing the locus of interest; (b) digesting the<br>
DNA with the restriction enzyme that recognizes the recognition site generated by<br>
the second primer to create a DNA fragment; (c) incorporating a nucleotide into<br><br>
the digested DNA of (b) by using the 5' overhang containing the locus of interest<br>
as a template; and (d) determining the sequence of the locus of interest by<br>
determining the sequence of the DNA of (c).<br>
[0019] The invention also is directed to a DNA fragment containing a locus of<br>
interest to be sequenced and a recognition site for a restriction enzyme, wherein<br>
digestion with the restriction enzyme creates a 5' overhang on the DNA fragment,<br>
and wherein the locus of interest and the restriction enzyme recognition site are in<br>
relationship to each other such that digestion with the restriction enzyme<br>
generates a 5' overhang containing the locus of interest.<br>
[0020] The template DNA can be obtained from any source including<br>
synthetic nucleic acid, preferably from a bacterium, fungus, virus, plant,<br>
protozoan, animal or human source. In one embodiment, the template DNA is<br>
obtained from a human source. In another embodiment, the template DNA is<br>
obtained from a cell, tissue, blood sample, serum sample, plasma sample, urine<br>
sample, spinal fluid, lymphatic fluid, semen, vaginal secretion, ascitic fluid,<br>
saliva, mucosa secretion, peritoneal fluid, fecal sample, or body exudates.<br>
[0021] The 3' region of the first and/or second primer can contain a mismatch<br>
with the template DNA. The mismatch can occur at but is not limited to the last<br>
1,2, or 3 bases at the 3' end.<br>
[0022] The restriction enzyme used in the invention can cut DNA at the<br>
recognition site. The restriction enzyme can be but is not limited to PfiF I, Sau96<br>
I, ScrF I, BsaJ I, Bssk I, Dde I, EcoN I, Fnu4H I, Hinf I, or Tthl 111.<br>
Alternatively, the restriction enzyme used in the invention can cut DNA at a<br>
distance from its recognition site.<br>
[0023] In another embodiment, the first primer contains a recognition site for<br>
a restriction enzyme. In a preferred embodiment, the restriction enzyme<br>
recognition site is different from the restriction enzyme recognition site on the<br>
second primer. The invention includes digesting the amplified DNA with a<br>
restriction enzyme that recognizes the recognition site on the first primer.<br><br>
[0024] Preferably, the recognition site on the second primer is for a restriction<br>
enzyme that cuts DNA at a distance from its recognition site and generates a 5'<br>
overhang, containing the locus of interest. In a preferred embodiment, the<br>
recognition site on the second primer is for a Type IIS restriction enzyme. The<br>
Type IIS restriction enzyme, e.g., is selected from the group consisting of: Alw I,<br>
Alw261, Bbs I, Bbv I, BceA I, Bmr I, Bsa I, Bst711, BsmA I, BsmB I, BsmF I,<br>
BspM I, Ear I, Fau I, Fok I, Hga I, Pie I, Sap I, SSfaN I, and Sthi321, and more<br>
preferably BceA I and BsmF I.<br>
[0025] In one embodiment, the 5' region of the second primer does not anneal<br>
to the template DNA and/or the 5' region of the first primer does not anneal to the<br>
template DNA. The annealing length of the 3' region of the first or second primer<br>
can be 25-20,20-15,15,14,13,12,11,10,9, 8,7,6,5,4, or less than 4 bases.<br>
[0026] In one embodiment, the amplification can comprise polymerase chain<br>
reaction (PCR). In a further embodiment, the annealing temperature for cycle 1 of<br>
PCR can be at about the melting temperature of the 3' region of the second primer<br>
that anneals to the template DNA. In another embodiment, the annealing<br>
temperature for cycle 2 of PCR can be about the melting temperature of the 3'<br>
region of the first primer that anneals to tile template DNA. In another<br>
embodiment, the annealing temperature for the remaining cycles can be about the<br>
melting temperature of the entire sequence of the second primer.<br>
[0027] In one embodiment, the 3' end of the second primer is adjacent to the<br>
locus of interest.<br>
[0028] The first and/or second primer can contain a tag at the 5' terminus.<br>
Preferably, the first primer contains a tag at the 5' terminus. The tag can be used<br>
to separate the amplified DNA from the template DNA. The tag can be used to<br>
separate the amplified DNA containing the labeled nucleotide from the amplified<br>
DNA that does not contain the labeled nucleotide. The tag can be but is not<br>
limited to a radioisotope, fluorescent reporter molecule, chemiluminescent<br>
reporter molecule, antibody, antibody fragment, hapten, biotin, derivative of<br>
biotin, photobiotin, iminobiotin, digoxigenin, avidin, enzyme, acridinium, sugar,<br><br>
enzyme, apoenzyme, homopolymeric oligonucleotide, hormone, ferromagnetic<br>
moiety, paramagnetic moiety, diamagnetic moiety, phosphorescent moiety,<br>
luminescent moiety, electrochemiluminescent moiety, chromatic moiety, moiety<br>
having a detectable electron spin resonance, electrical capacitance, dielectric<br>
constant or electrical conductivity, or combinations thereof. Preferably, the tag is<br>
biotin. The biotin tag is used to separate amplified DNA from the template DNA<br>
using a streptavidin matrix. The streptavidin matrix is coated on wells of a<br>
microtiter plate.<br>
[0029] The incorporation of a nucleotide in the method of the invention is by<br>
a DNA polymerase including but not limited to E. coli DNA polymerase, Klenow<br>
fragment of E. coli DNA polymerase I, T5 DNA polymerase, T7 DNA<br>
polymerase, T4 DNA polymerase, Taq polymerase, Pfu DNA polymerase, Vent<br>
DNA polymerase, bacteriophage 29, REDTaq™ Genomic DNA polymerase, and<br>
sequenase.<br>
[0030] The incorporation of a nucleotide can further comprise using a mixture<br>
of labeled and unlabeled nucleotides. One nucleotide, two nucleotides, three<br>
nucleotides, four nucleotides, five nucleotides, or more than five nucleotides may<br>
be incorporated. A combination of labeled and unlabeled nucleotides can be<br>
incorporated. The labeled nucleotide can be but is not limited to a<br>
dideoxynucleotide triphosphate and deoxynucleotide triphosphate. The unlabeled<br>
nucleotide can be but is not limited to a dideoxynucleotide triphosphate and<br>
deoxynucleotide triphosphate. The labeled nucleotide is labeled with a molecule<br>
such as but not limited to a radioactive molecule, fluorescent molecule, antibody,<br>
antibody fragment, hapten, carbohydrate, biotin, and derivative of biotin,<br>
phosphorescent moiety, luminescent moiety, electrochemiluminescent moiety,<br>
chromatic moiety, or moiety having a detectable electron spin resonance,<br>
electrical capacitance, dielectric constant or electrical conductivity. Preferably,<br>
the labeled nucleotide is labeled with a fluorescent molecule. The incorporation<br>
of a fluorescent labeled nucleotide further includes using a mixture of fluorescent<br>
and unlabeled nucleotides.<br><br>
[0031] In one embodiment, the determination of the sequence of the locus of<br>
interest comprises detecting the incorporated nucleotide. In one embodiment, the<br>
detection is by a method such as but not limited to gel electrophoresis, capillary<br>
electrophoresis, microchannel electrophoresis, polyacrylamide gel electrophoresis,<br>
fluorescence detection, sequencing, ELISA, mass spectrometry, time of flight<br>
mass spectrometry, quadrupole mass spectrometry, magnetic sector mass<br>
spectrometry, electric sector mass spectrometry, fluorometry, infrared<br>
spectrometry, ultraviolet spectrometry, palentiostatic amperometry, hybridization,<br>
such as Southern Blot, or microarray. In a preferred embodiment, the detection is<br>
by fluorescence detection.<br>
[0032] In a preferred embodiment, the locus of interest is suspected of<br>
containing a single nucleotide polymorphism or mutation. The method can be<br>
used for determining sequences of multiple loci of interest concurrently. The<br>
template DNA can comprise multiple loci from a single chromosome. The<br>
template DNA can comprise multiple loci from different chromosomes. The loci<br>
of interest on template DNA can be amplified in one reaction. Alternatively, each<br>
of the loci of interest on template DNA can be amplified in a separate reaction.<br>
The amplified DNA can be pooled together prior to digestion of the amplified<br>
DNA. Each of the labeled DNA containing a locus of interest can be separated<br>
prior to determining the sequence of the locus of interest. In one embodiment, at<br>
least one of the loci of interest is suspected of containing a single nucleotide<br>
polymorphism or a mutation.<br>
[0033] In another embodiment, the method of the invention can be used for<br>
deterrruning the sequences of multiple loci of interest from a single individual or<br>
from multiple individuals. Also, the method of the invention can be used to<br>
determine the sequence of a single locus of interest from multiple individuals.<br>
BRIEF DESCRIPTION OF THE FIGURES<br>
[0034] FIG. 1 A. A Schematic diagram depicting a double stranded DNA<br>
molecule. A pair of primers, depicted as bent arrows, flank the locus of interest,<br>
depicted as a triangle symbol at base N14. The locus of interest can be a single<br><br>
nucleotide polymorphism, point mutation, insertion, deletion, translocation, etc.<br>
Each primer contains a restriction enzyme recognition site about 10 bp from the 5'<br>
terminus depicted as region "a" in the first primer and as region "d" in the second<br>
primer. Restriction recognition site "a" can be for any type of restriction enzyme<br>
but recognition site "d" is for a restriction enzyme, which cuts "n" nucleotides<br>
away from its recognition site and leaves a 5' overhang and a recessed 3' end.<br>
Examples of such enzymes include but are not limited to BceA I and BsmF I. The<br>
5' overhang serves as a template for incorporation of a nucleotide into the 3'<br>
recessed end.<br>
[0035] The first primer is shown modified with biotin at the 5' end to aid in<br>
purification. The sequence of the 3' end of the primers is such that the primers<br>
anneal at a desired distance upstream and downstream of the locus of interest.<br>
The second primer anneals close to the locus of interest; the annealing site, which<br>
is depicted as region "c," is designed such that the 3' end of the second primer<br>
anneals one base away from the locus of interest. The second primer can anneal<br>
any distance from the locus of interest provided that digestion with the restriction<br>
enzyme, which recognizes the region "d" on this primer, generates a 5' overhang<br>
that contains the locus of interest.<br>
[0036] The first primer annealing site, which is depicted as region "b" is<br>
about 20 bases.<br>
[0037] FIG. 1B. A schematic diagram depicting the annealing and extension<br>
steps of the first cycle of amplification by PCR. The first cycle of amplification is<br>
performed at about the melting temperature of the 3' region, which anneals to the<br>
template DNA, of the second primer, depicted as region "c," and is 13 base pairs<br>
in this example. At this temperature, both the first and second primers anneal to<br>
their respective complementary strands and begin extension, depicted by dotted<br>
lines. In this first cycle, the second primer extends and copies the region b where<br>
the first primer can anneal in the next cycle.<br>
[0038] FIG. 1C. A schematic diagram depicting the annealing and extension<br>
steps following denaturation in the second cycle of amplification of PCR. The<br><br>
second cycle of amplification is performed at a higher annealing temperature<br>
(TM2), which is about the melting temperature of the 20 bp of the 3' region of the<br>
first primer that anneals to the template DNA, depicted as region "b." Therefore<br>
at TM2, the first primer, which is complementary to region b, can bind to the<br>
DNA that was copied in the first cycle of the reaction. However, at TM2 the<br>
second primer cannot anneal to the original template DNA or to DNA that was<br>
copied in the first cycle of the reaction because the annealing temperature is too<br>
high. The second primer can anneal to 13 bases in the original template DNA but<br>
TM2 is calculated at about the melting temperature of 20 bases.<br>
[0039] FIG. ID. A schematic diagram depicting the annealing and extension<br>
reactions after denaturation during the third cycle of amplification. In this cycle,<br>
the annealing temperature, TM3, is about the melting temperature of the entire<br>
second primer, including regions "c" and "d." The length of regions "c" + "d" is<br>
about 27-33 bp long, and thus TM3 is significantly higher than TM1 and TM2.<br>
At this higher TM the second primer, which contain region c and d, anneals to the<br>
copied DNA generated in cycle 2.<br>
[0040] FIG. IE. A schematic diagram depicting the annealing and extension<br>
reactions for the remaining cycles of amplification. The annealing temperature<br>
for the remaining cycles is TM3, which is about the melting temperature of the<br>
entire second primer. At TM3, the second primer binds to templates that contain<br>
regions c* and d' and the first primer binds to templates that contain regions a' and<br>
b. By raising the annealing temperature successively in each cycle for the first<br>
three cycles, from TM1 to TM2 to TM3, nonspecific amplification is significantly<br>
reduced,<br>
[0041] FIG. IF. A schematic diagram depicting the amplified locus of<br>
interest 'bound to a solid matrix.<br>
[0042] FIG. 1G. A schematic diagram depicting the bound, amplified DNA<br>
after digestion with a restriction enzyme that recognizes "d." The "downstream"<br>
end is released into the supernatant, and can be removed by washing with any<br><br>
suitable buffer. The upstream end containing the locus of interest remains bound<br>
to the solid matrix.<br>
[0043] FIG. 1H. A schematic diagram depicting the bound amplified DNA,<br>
after "filling in" with a labeled ddNTP. A DNA polymerase is used to "fill in" the<br>
base (N'14) that is complementary to the locus of interest (N14). In this example,<br>
only ddNTPs are present in this reaction, such that only the locus of interest or<br>
SNP of interest is filled in.<br>
[0044] FIG. 1I. A schematic diagram depicting the labeled, bound DNA after<br>
digestion with restriction enzyme "a." The labeled DNA is released into the<br>
supernatant, which can be collected to identify the base that was incorporated.<br>
[0045] FIG. 2. A schematic diagram depicting double stranded DNA<br>
templates with "N" number of loci of interest and "n" number of primer pairs, x1,<br>
y1 to xn, yn, specifically annealed such that a primer flanks each locus of interest.<br>
The first primers are biotinylated at the 5' end, depicted by *, and contain a<br>
restriction enzyme recognition site, "a", which is recognized by any type of<br>
restriction enzyme. The second primers contain a restriction enzyme recognition<br>
site, "d," where "d" is a recognition site for a restriction enzyme that cuts DNA at<br>
a distance from its recognition site, and generates a 5' overhang containing the<br>
locus of interest and a recessed 3' end. The second primers anneal adjacent to the<br>
respective loci of interest. The exact position of the restriction enzyme site "d" in<br>
the second primers is designed such that digesting the PCR product of each locus<br>
of interest with restriction enzyme "d" generates a 5' overhang containing the<br>
locus of interest and a 3' recessed end. The annealing sites of the first primers are<br>
about 20 bases long and are selected such that each successive first primer is<br>
further away from its respective second primer. For example, if at locus 1 the 3'<br>
ends of the first and second primers are Z base pairs apart, then at locus 2, the 3'<br>
ends of the first and second primers are Z + K base pairs apart, where K = 1,2,3<br>
or more than three bases. Primers for locus N are ZN-1 + K base pairs apart. The<br>
purpose of making each successive first primer further apart from their respective<br>
second primers is such that the "filled in" restriction fragments (generated after<br><br>
amplification, purification, digestion and labeling as described in FIGS. 1B-1I)<br>
differ in size and can be resolved, for example by electrophoresis, to allow<br>
detection of each individual locus of interest.<br>
[0046] FIG. 3. PCR amplification of DNA fragments containing SNPs using<br>
multiple annealing temperatures. A sample containing genomic DNA templates<br>
from thirty-six human volunteers was analyzed for the following four SNPs: SNP<br>
HC21S00340 (lane 1), identification number as assigned in the Human<br>
Chromosome 21 cSNP Database, located on chromosome 21; SNP TSC 0095512<br>
(lane 2), located on chromosome 1; SNP TSC 0214366 (lane 3), located on<br>
chromosome 1; and SNP TSC 0087315 (lane 4), located on chromosome 1. Each<br>
DNA fragment containing a SNP was amplified by PCR using three different<br>
annealing temperature protocols, herein referred to as the low stringency<br>
annealing temperature; medium stringency annealing temperature; and high<br>
stringency annealing temperature. Regardless of the annealing temperature<br>
protocol, each DNA fragment containing a SNP was amplified for 40 cycles of<br>
PCR. The denaturation step for each PCR reaction was performed for 30 seconds<br>
at 95°C.<br>
[0047] FIG. 3A. Photograph of a gel demonstrating PCR amplification of the<br>
4 DNA fragments containing different SNPs using the low stringency annealing<br>
temperature protocol.<br>
[0048] FIG. 3B. Photograph of a gel demonstrating PCR amplification of the<br>
4 DNA fragments containing different SNPs using the medium stringency<br>
annealing temperature protocol.<br>
[0049] FIG. 3C. Photograph of a gel demonstrating PCR amplification of the<br>
4 DNA fragments containing different SNPs using the high stringency annealing<br>
temperature protocol.<br>
[0050] FIG. 4A. A depiction of the DNA sequence of SNP HC21S00027<br>
(SEQ ID NOS:27 &amp; 28), assigned by the Human Chromosome 21 cSNP<br>
database, located on chromosome 21. A first primer (SEQ ID NO:17) and a<br>
second primer (SEQ ID NO: 18) are indicated above and below, respectively, the<br><br>
sequence of HC21S00027. The first primer is biotinylated and contains the<br>
restriction enzyme recognition site for EcoRI. The second primer contains the<br>
restriction enzyme recognition site for BsmF I and contains 13 bases that anneal<br>
to the DNA sequence. The SNP is indicated by R (A/G) and r (T/C;<br>
complementary to R).<br>
[0051] FIG. 4B. A depiction of the DNA sequence of SNP HC21S00027<br>
(SEQ ID NOS:27 &amp; 28), as assigned by the Human Chromosome 21 cSNP<br>
database, located on chromosome 21. A first primer (SEQ ID NO: 17) and a<br>
second primer (SEQ ID NO: 19) are indicated above and below, respectively, the<br>
sequence of HC21S00027. The first primer is biotinylated and contains the<br>
restriction enzyme recognition site for EcoRI. The second primer contains the<br>
restriction enzyme recognition site for BceA I and has 13 bases that anneal to the<br>
DNA sequence. The SNP is indicated by R (A/G) and r (T/C; complementary to<br>
R).<br>
[0052] FIG. 4C. A depiction of the DNA sequence of SNP TSC0095512<br>
(SEQ ID NOS:29 &amp; 30) from chromosome 1. The first primer (SEQ ID NO. 11)<br>
and the second primer (SEQ ID NO:20) are indicated above and below,<br>
respectively, the sequence of TSC0095512. The first primer is biotinylated and<br>
contains the restriction enzyme recognition site for EcoRI. The second primer<br>
contains the restriction enzyme recognition site for BsmF I and has 13 bases that<br>
anneal to the DNA sequence. The SNP is indicated by S (G/C) and s (C/G;<br>
complementary to S).<br>
[0053] FIG. 4D. A depiction of the DNA sequence of SNP TSC0095512<br>
(SEQ ID NOS:29 &amp; 30) from chromosome 1. The first primer (SEQ ID NO: 11)<br>
and the second primer (SEQ ID NO: 12) are indicated above and below,<br>
respectively, the sequence of TSC0095512. The first primer is biotinylated and<br>
contains the restriction enzyme recognition site for EcoRI. The second primer<br>
contains the restriction enzyme recognition site for BceA I and has 13 bases that<br>
anneal to the DNA sequence. The SNP is indicated by S (G/C) and s (C/G;<br>
complementary to S).<br><br>
ID NOS:31,37 &amp; 41); two nucleotides (a dNTP is incorporated followed by a<br>
ddNTP) (SEQ ID NOS:31,39 &amp; 41); three nucleotides (two dNTPs are<br>
incorporated, followed by a ddNTP) (SEQ ID NOS:31,40 &amp; 41); or four<br>
nucleotides (three dNTPs are incorporated, followed by a ddNTP) (SEQ ID<br>
NOS:31 &amp; 41). All four products can be separated by size, and the incorporated<br>
nucleotide detected (*R-dd = fluorescent dideoxy nucleotide). Detection of the<br>
first nucleotide, which corresponds to the SNP or locus site, and the next three<br>
nucleotides provides an additional level of quality assurance. The SNP is<br>
indicated by R (A/G) and r (T/C) (complementary to R).<br>
[0058] FIGS. 8A-8D. Release of the "filled in" SNP from the solid support<br>
matrix, i.e. streptavidin coated well. SNP HC21S00027 is shown in FIG. 8A<br>
(SEQ ID NOS:31,37 &amp; 41) and FIG. 8B (SEQ ID NOS:31,37 &amp; 39), while SNP<br>
TSC0095512 is shown in FIG. 8C (SEQ ID NOS:34 &amp; 38)and FIG. 8D (SEQ ID<br>
NO:34). The "filled in" SNP is free in solution, and can be detected.<br>
[0059] FIG. 9A. Sequence analysis of a DNA fragment containing SNP<br>
HC21S00027 digested with BceAI. Four "fill in" reactions are shown; each<br>
reaction contained one fluorescently labeled nucleotide, ddGTP, ddATP, ddTTP,<br>
or ddCTP, and unlabeled ddNTPs. The 5' overhang generated by digestion with<br>
BceA I and the expected nucleotides at this SNP site are indicated.<br>
[0060] FIG 9B. Sequence analysis of SNP TSC0095512. SNP TSC0095512<br>
was amplified with a second primer that contained the recognition site for BceA I,<br>
and in a separate reaction, with a second primer that contained the recognition site<br>
for BsmF I. Four fill in reactions are shown for each PCR product; each reaction<br>
contained one fluorescently labeled nucleotide, ddGTP, ddATP, ddTTP, or<br>
ddCTP, and unlabeled ddNTPs. The 5' overhang generated by digestion with<br>
BceA I and with BsmF I and the expected nucleotides are indicated.<br>
[0061] FIG 9C. Sequence analysis of SNP TSC0264580 after amplification<br>
with a second primer that contained the recognition site for BsmF I. Four "fill in"<br>
reactions are shown; each reaction contained one fluorescently labeled nucleotide,<br>
which was ddGTP, ddATP, ddTTP, or ddCTP and unlabeled ddNTPs. Two<br><br>
[0054] FIGS. 5A-5D. A schematic diagram depicting the nucleotide<br>
sequences of SNP HC21S00027 (FIG. 5A (SEQ ID NOS:31 &amp; 32) and FIG. 5B<br>
(SEQ ID NOS:31 &amp; 33)), and SNP TSC0095512 (FIG. 5C (SEQ ID NOS:34 &amp;<br>
35) and FIG. 5D (SEQ ID NOS:34 &amp; 36)) after amplification with the primers<br>
described, in FIGS. 4A-4D. Restriction sites in the primer sequence are indicated<br>
in bold.<br>
[0055] FIGS. 6A-6D. A schematic diagram depicting the nucleotide<br>
sequences of each amplified DNA fragment containing a SNP after digestion with<br>
the appropriate Type IIS restriction enzyme. FIG. 6A (SEQ ID NOS:31 &amp; 32)<br>
and FIG. 6B (SEQ ID NOS:31 &amp; 33) depict fragments of a DNA sequence<br>
containing SNP HC21S00027 digested with the Type IIS restriction enzymes<br>
BsmF I and BceA I, respectively. FIG. 6C (SEQ ID NOS:34 &amp; 35) and FIG. 6D<br>
(SEQ ID NOS:34 &amp; 36) depict fragments of a DNA sequence containing SNP<br>
TSC0095512 digested with the Type IIS restriction enzymes BsmF I and BceA I,<br>
respectively.<br>
[0056] FIGS. 7A-7D. A schematic diagram depicting the incorporation of a<br>
fluorescently labeled nucleotide using the 5' overhang of the digested SNP site as<br>
a template to "fill in" the 3' recessed end. FIG. 7A (SEQ ID NOS:31,37 &amp; 41)<br>
and FIG. 7B (SEQ ID NOS:31,37 &amp; 39) depict the digested SNP HC21S00027<br>
locus with an incorporated labeled ddNTP (*R"dd = fluorescent dideoxy<br>
nucleotide). FIG. 7C (SEQ ID NOS:34 &amp; 38) and FIG. 7D (SEQ ID NO:34)<br>
depict the digested SNP TSC0095512 locus with an incorporated labeled ddNTP<br>
(* S-dd = fluorescent dideoxy nucleotide). The use of ddNTPs ensures that the 3'<br>
recessed end is extended by one nucleotide, which is complementary to the<br>
nucleotide of interest or SNP site present in the 5' overhang.<br>
[0057] FIG. 7E. A schematic diagram depicting the incorporation of dNTPs<br>
and a ddNTP into the 5' overhang containing the SNP site. The DNA fragment<br>
containing SNP HC21S00007 was digested with BsmF I, which generates a four<br>
base 5' overhang. The use of a mixture of dNTPs and ddNTPs allows the 3'<br>
recessed end to be extended one nucleotide (a ddNTP is incorporated first) (SEQ<br><br>
different 5' overhangs are depicted: one represents the DNA molecules that were<br>
cut 11 nucleotides away on the sense strand and 15 nucleotides away on the<br>
antisense strand and the other represents the DNA molecules that were cut 10<br>
nucleotides away on the sense strand and 14 nucleotides away on the antisense<br>
strand. The expected nucleotides also are indicated.<br>
[0062] FIG 9D. Sequence analysis of SNP HC21S00027 amplified with a<br>
second primer that contained the recognition site for BsmF I. A mixture of<br>
labeled ddNTPs and unlabeled dNTPs was used to fill in the 5' overhang<br>
generated by digestion with BsmF I. Two different 5' overhangs are depicted: one<br>
represents the DNA molecules that were cut 11 nucleotides away on the sense<br>
strand and 15 nucleotides away on the antisense strand and the other represents<br>
the DNA molecules that were cut 10 nucleotides away on the sense strand and 14<br>
nucleotides away on the antisense strand. The nucleotide upstream of the SNP,<br>
the nucleotide at the SNP site (the sample contained DNA templates from 36<br>
individuals; both nucleotides would be expected to be represented in the sample),<br>
and the three nucleotides downstream of the SNP are indicated.<br>
[0063] FIG. 10. Sequence analysis of multiple SNPs. SNPsHC21S00131,<br>
and HC21S00027, which are located on chromosome 21, and SNPs TSC0087315,<br>
SNP TSC0214366, SNP TSC0413944, and SNP TSC0095512, which are on<br>
chromosome 1, were amplified in separate PCR reactions with second primers<br>
that contained a recognition site for BsmF I. The primers were designed so that<br>
each amplified locus of interest was of a different size. After amplification, the<br>
reactions were pooled into a single sample, and all subsequent steps of the method<br>
performed (as described for FIGS. 1F-1I) on that sample. Each SNP and the<br>
nucleotide found at each SNP are indicated.<br>
[0064] FIG. 11. Sequence determination of both alleles of SNPs<br>
TSC0837969, TSC0034767, TSC1130902, TSC0597888, TSC0195492,<br>
TSC0607185 using one fiuorescently labeled nucleotide. Labeled ddGTP was<br>
used in the presence of unlabeled dATP, dCTP, dTTP to fill-in the overhang<br>
generated by digestion with BsmF I. The nucleotide preceding the variable site on<br><br>
the strand that was filled-in was not guanine, and the nucleotide after the variable<br>
site on the strand that was filled in was not guanine. The nucleotide two bases<br>
after the variable site on the strand that was filled-in was guanine. Alleles that<br>
contain guanine at variable site are filled in with labeled ddGTP. Alleles that do<br>
not contain guanine are filled in with unlabeled dATP, dCTP, or dTTP, and the<br>
polymerase continues to incorporate nucleotides until labeled ddGTP is filled in at<br>
position 3 complementary to the overhang.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
[0065] The present invention provides a novel method for rapidly determining<br>
the sequence of DNA, especially at a locus of interest or multiple loci of interest.<br>
The sequences of any number of DNA targets, from one to hundreds or thousands<br>
or more; of loci of interest in any template DNA or sample of nucleic acid can be<br>
determined efficiently, accurately, and economically. The method is especially<br>
useful for the rapid sequencing of one to tens of thousands or more of genes,<br>
regions of genes, fragments of genes, single nucleotide polymorphisms, and<br>
mutations on a single chromosome or on multiple chromosomes.<br>
[0066] The invention is directed to a method for determining a sequence of a<br>
locus of interest, the method comprising: (a) amplifying a locus of interest on a<br>
template DNA using a first and second primers, wherein the second primer<br>
contains a recognition site for a restriction enzyme such that digestion with the<br>
restriction enzyme generates a 5' overhang containing the locus of interest; (b)<br>
digesting the amplified DNA with the restriction enzyme that recognizes the<br>
recognition site on the second primer; (c) incorporating a nucleotide into the<br>
digested DNA of (b) by using the 5' overhang containing the locus of interest as a<br>
template; and (d) determining the sequence of the locus of interest by determining<br>
the sequence of the DNA of (c).<br>
[0067] The invention is also directed to a method for determining a sequence<br>
of a locus of interest, said method comprising: (a) amplifying a locus of interest<br>
on a template DNA using a first and second primers, wherein the first and/or<br>
second primer contains a portion of a recognition site for a restriction enzyme,<br><br>
wherein a full recognition site for the restriction enzyme is generated upon<br>
amplification of the template DNA such that digestion with the restriction enzyme<br>
generates a 5' overhang containing the locus of interest; (b) digesting the<br>
amplified DNA with the restriction enzyme that recognizes the full recognition<br>
site generated by the second primer and the template DNA; (c) incorporating a<br>
nucleotide into the digested DNA of (b) by using the 5' overhang containing the<br>
locus of interest as a template; and determining the sequence of the locus of<br>
interest by determining the sequence of the DNA of (c).<br>
DNA Template<br>
[0068] By a "locus of interest" is intended a selected region of nucleic acid<br>
that is within a larger region of nucleic acid. A locus of interest can include but is<br>
not limited to 1-100,1-50,1-20, or 1-10 nucleotides, preferably 1-6,1-5,1-4,1-3,<br>
1-2, or 1 nucleotide(s).<br>
[006'9] As used herein, an "allele" is one of several alternate forms of a gene<br>
or non-coding regions of DNA that occupy the same position on a chromosome.<br>
The term allele can be used to describe DNA from any organism including but not<br>
limited to bacteria, viruses, fungi, protozoa, molds, yeasts, plants, humans, non-<br>
humans, animals, and archaebacteria.<br>
[0070] As used herein with respect to individuals, "mutant alleges" refers to<br>
variant alleles that are associated with a disease state.<br>
For example, bacteria typically have one large strand of DNA. The term allele<br>
with respect to bacterial DNA refers to the form of a gene found in one cell as<br>
compared to the form of the same gene in a different bacterial cell of the same<br>
species.<br>
[0071] Alleles can have the identical sequence or can vary by a single<br>
nucleotide or more than one nucleotide. With regard to organisms that have two<br>
copies of each chromosome, if both chromosomes have the same allele, the<br>
condition is referred to as homozygous. If the alleles at the two chromosomes are<br>
different, the condition is referred to as heterozygous. For example, if the locus of<br>
interest is SNP X on chromosome 1, and the maternal chromosome contains an<br><br>
adenine at SNP X (A allele) and the paternal chromosome contains a guanine at<br>
SNP X (G allele), the individual is heterozygous at SNP X.<br>
[0072] As used herein, "sequence" means the identity of, or to deterrnine the<br>
identity of (depending on whether used as a noun or a verb, respectively), one<br>
nucleotide or more than one contiguous nucleotides in a polynucleotide. In the<br>
case of a single nucleotide, e.g., a SNP, "sequence" is used as a noun<br>
interchangeably with "identity" herein, and "sequence" is used interchangeably as<br>
a verb with "identify" herein.<br>
[0073] The term "template" refers to any nucleic acid molecule that can be<br>
used for amplification in the invention. RNA or DNA that is not naturally double<br>
stranded can be made into double stranded DNA so as to be used as template<br>
DNA. Any double stranded DNA or preparation containing multiple, different<br>
double stranded DNA molecules can be used as template DNA to amplify a locus<br>
or loci of interest contained in the template DNA.<br>
[0074] The source of the nucleic acid for obtaining the template DNA can be<br>
from any appropriate source including but not limited to nucleic acid from any<br>
organism, e.g., human or nonhuman, e.g., bacterium, virus, yeast, fungus, plant,<br>
protozoan, animal, nucleic acid-containing samples of tissues, bodily fluids (for<br>
example, blood, serum, plasma, saliva, urine, tears, semen, vaginal secretions,<br>
lymph fluid, cerebrospinal fluid or mucosa secretions), fecal matter, individual<br>
cells or extracts of the such sources that contain the nucleic acid of the same, and<br>
subcellular structures such as mitochondria or chloroplasts, using protocols well<br>
established within the art. Nucleic acid can<br>
[same paragraph] also be obtained from forensic, food, archeological, or<br>
inorganic samples onto which nucleic acid has been deposited or extracted. In a<br>
preferred embodiment, the nucleic acid has been obtained from a human or animal<br>
to be screened for the presence of one or more genetic sequences that can be<br>
diagnostic for, or predispose the subject to, a medical condition or disease.<br>
[0075] The nucleic acid that is to be analyzed can be any nucleic acid, e.g.,<br>
genomic, plasmid, cosmid, yeast artificial chromosomes, artificial or man-made<br><br>
DNA, including unique DNA sequences, and also DNA that has been reverse<br>
transcribed from an RNA sample, such as cDNA. The sequence of RNA can be<br>
determined according to the invention if it is capable of being made into a double<br>
stranded DNA form to be used as template DNA.<br>
[0076] The terms "primer" and "oligonucleotide primer" are interchangeable<br>
when used to discuss an oligonucleotide that anneals to a template and can be<br>
used to prime the synthesis of a copy of that template.<br>
[0077] "Amplified" DNA is DNA that has been "copied" once or multiple<br>
times, e.g. by polymerase chain reaction. When a large amount of DNA is<br>
available to assay, such that a sufficient number of copies of the locus of interest<br>
are already present in the sample to be assayed, it may not be necessary to<br>
"amplify" the DNA of the locus of interest into an even larger number of replicate<br>
copies. Rather, simply "copying" the template DNA once using a set of<br>
appropriate primers, such as those containing hairpin structures that allow the<br>
restriction enzyme recognition sites to be double stranded, can suffice.<br>
[0078] "Copy" as in "copied DNA" refers to DNA that has been copied once,<br>
or DNA that has been amplified into more than one copy.<br>
[0079] In one embodiment, the nucleic acid is amplified directly in the<br>
original sample containing the source of nucleic acid. It is not essential that the<br>
nucleic acid be extracted, purified or isolated; it only needs to be provided in a<br>
form that is capable of being amplified. A hybridization step of the nucleic acid<br>
with the primers, prior to amplification, is not required. For example,<br>
amplification can be performed in a cell or sample lysate using standard protocols<br>
well known in the art. DNA that is on a solid support, in a fixed biological<br>
preparation, or otherwise in a composition that contains non-DNA substances and<br>
that can be amplified without first being extracted from the solid support or fixed<br>
preparation or non-DNA substances in the composition can be used directly,<br>
without further purification, as long as the DNA can anneal with appropriate<br>
primers, and be copied, especially amplified, and the copied or amplified products<br>
can be recovered and utilized as described herein.<br><br>
[0080] In a preferred embodiment, the nucleic acid is extracted, purified or<br>
isolated from non-nucleic acid materials that are in the original sample using<br>
methods known in the art prior to amplification.<br>
[0081] In another embodiment, the nucleic acid is extracted, purified or<br>
isolated from the original sample containing the source of nucleic acid and prior<br>
to amplification, the nucleic acid is fragmented using any number of methods well<br>
known in the art including but not limited to enzymatic digestion, manual<br>
shearing, and sonication. For example, the DNA can be digested with one or<br>
more restriction enzymes that have a recognition site, and especially an eight base<br>
or six base pair recognition site, which is not present in the loci of interest.<br>
Typically, DNA can be fragmented to any desired length, including 50,100,250,<br>
500,1,000, 5,000,10,000, 50,000 and 100,000 base pairs long. In another<br>
embodiment, the DNA is fragmented to an average length of about 1000 to 2000<br>
base pairs. However, it is not necessary that the DNA be fragmented.<br>
[0082] Fragments of DNA that contain the loci of interest can be purified<br>
from the; fragments of DNA that do not contain the loci of interest before<br>
amplification. The purification can be done by using primers that will be used in<br>
the amplification (see "Primer Design" section below) as hooks to retrieve the<br>
fragments containing the loci of interest, based on the ability of such primers to<br>
anneal to the loci of interest. In a preferred embodiment, tag-modified primers are<br>
used, such as e.g. biotinylated primers. See also the "Purification of Amplified<br>
DNA" section for additional tags.<br>
[0083] By purifying the DNA fragments containing the loci of interest, the<br>
specificity of the amplification reaction can be improved. This will minimize<br>
amplification of nonspecific regions of the template DNA. Purification of the<br>
DNA fragments can also allow multiplex PCR (Polymerase Chain Reaction) or<br>
amplification of multiple loci of interest with improved specificity.<br>
[0084] In one embodiment, the nucleic acid sample is obtained with a desired<br>
purpose in mind such as to determine the sequence at a predetermined locus or<br>
loci of interest using the method of the invention. For example, the nucleic acid is<br><br>
obtained for the purpose of identifying one or more conditions or diseases to<br>
which the subject can be predisposed or is in need of treatment for, or the<br>
presence of certain single nucleotide polymorphisms. Li an alternative<br>
embodiment, the sample is obtained to screen for the presence or absence of one<br>
or more DNA sequence markers, the presence of which would identify that DNA<br>
as being from a specific bacterial or fungal microorganism, or individual.<br>
[0085] The loci of interest that are to be sequenced can be selected based upon<br>
sequence alone. In humans, over 1.42 million single nucleotide polymorphisms<br>
(SNPs) have been described (Nature 409:928-933 (2001); The SNP Consortium<br>
LTD). On the average, there is one SNP every 1.9 kb of human genome.<br>
However, the distance between loci of interest need not be considered when<br>
selecting the loci of interest to be sequenced according to the invention. If more<br>
than one locus of interest on genomic DNA is being analyzed, the selected loci of<br>
interest can be on the same chromosome or on different chromosomes.<br>
[0086] In a preferred embodiment, the length of sequence that is amplified is<br>
preferably different for each locus of interest so that the loci of interest can be<br>
separated by size.<br>
[0087] In fact, it is an advantage of the invention that primers that copy an<br>
entire gene sequence need not be utilized. Rather, the copied locus of interest is<br>
preferably only a small part of the total gene. There is no advantage to<br>
sequencing the entire gene as this can increase cost and delay results. Sequencing<br>
only the desired bases or loci of interest within the gene maximizes the overall<br>
efficiency of the method because it allows for the maximum number of loci of<br>
interest to be determined in the fastest amount of time and with minimal cost.<br>
[0088] Because a large number of sequences can be analyzed together, the<br>
method of the invention is especially amenable to the large-scale screening of a<br>
number of individual samples.<br>
[0089] Any number of loci of interest can be analyzed and processed,<br>
especially concurrently, using the method of the invention. The sample(s) can be<br>
analyzed to determine the sequence at one locus of interest or at multiple loci of<br><br>
interest concurrently. For example, the 10 or 20 most frequently occurring<br>
mutation sites in a disease associated gene can be sequenced to detect the majority<br>
of the disease carriers.<br>
[0090] Alternatively, 2, 3,4, 5, 6, 7, 8,9,10-20,20-25,25-30, 30-35, 35-40,<br>
40-45,45-50, 50-100,100-250,250-500, 500-1,000,1,000-2,000,2,000-3, 000,<br>
3,000-5,000,5,000-10,000, 10,000-50,000 or more than 50,000 loci of interest can<br>
be analyzed at the same time when a global genetic screening is desired. Such a<br>
global genetic screening might be desired when using the method of the invention<br>
to provide a genetic fingerprint to identify a certain microorganism or individual<br>
or for SNP genotyping.<br>
[0091] The multiple loci of interest can be targets from different organisms.<br>
For example, a plant, animal or human subject in need of treatment can have<br>
symptoms of infection by one or more pathogens. A nucleic acid sample taken<br>
from such a plant, animal or human subject can be analyzed for the presence of<br>
multiple suspected or possible pathogens at the same time by determining the<br>
sequence: of loci of interest which, if present, would be diagnostic for that<br>
pathogen. Not only would the finding of such a diagnostic sequence in the subject<br>
rapidly pinpoint the cause of the condition, but also it would rule out other<br>
pathogens that were not detected. Such screening can be used to assess the degree<br>
to which a pathogen has spread throughout an organism or environment. In a<br>
similar manner, nucleic acid from an individual suspected of having a disease that<br>
is the result of a genetic abnormality can be analyzed for some or all of the known<br>
mutations that result in the disease, or one or more of the more common<br>
mutations.<br>
[0092] The method of the invention can be used to monitor the integrity of the<br>
genetic nature of an organism. For example, samples of yeast can be taken at<br>
various times and from various batches in the brewing process, and their presence<br>
or identity compared to that of a desired strain by the rapid analysis of their<br>
genomic sequences as provided herein.<br><br>
[0093] The locus of interest that is to be copied can be within a coding<br>
sequence or outside of a coding sequence. Preferably, one or more loci of interest<br>
that are to be copied are within a gene. In a preferred embodiment, the template<br>
DNA that is copied is a locus or loci of interest that is within a genomic coding<br>
sequence, either intron or exon. In a highly preferred embodiment, exon DNA<br>
sequences are copied. The loci of interest can be sites where mutations are known<br>
to cause disease or predispose to a disease state. The loci of interest can be sites<br>
of single nucleotide polymorphisms. Alternatively, the loci of interest that are to<br>
be copied can be outside of the coding sequence, for example, in a transcriptional<br>
regulatory region, and especially a promoter, enhancer, or repressor sequence.<br>
Primer Design<br>
[0094] Published sequences, including consensus sequences, can be used to<br>
design or select primers for use in amplification of template DNA. The selection<br>
of sequences to be used for the construction of primers that flank a locus of<br>
interest can be made by examination of the sequence of the loci of interest, or<br>
immediately thereto. The recently published sequence of the human genome<br>
provides a source of useful consensus sequence information from which to design<br>
primers to flank a desired human gene locus of interest.<br>
[0095] By "flanking" a locus of interest is meant that the sequences of the<br>
primers are such that at least a portion of the 3' region of one primer is<br>
complementary to the antisense strand of the template DNA and upstream of the<br>
locus of interest (forward primer), and at least a portion of the 3' region of the<br>
other primer is complementary to the sense strand of the template DNA and<br>
downstream of the locus of interest (reverse primer). A "primer pair" is intended<br>
to specify a pair of forward and reverse primers. Both primers of a primer pair<br>
anneal in a manner that allows extension of the primers, such that the extension<br>
results in amplifying the template DNA in the region of the locus of interest.<br>
[0096] Primers can be prepared by a variety of methods including but not<br>
limited to cloning of appropriate sequences and direct chemical synthesis using<br>
methods well known in the art (Narang et al., Methods Enzymol. 68:90 (1979);<br><br>
Brown et al., Methods Enzymol. 68:109 (1979)). Primers can also be obtained<br>
from commercial sources such as Operon Technologies, Amersham Pharmacia<br>
Biotech, Sigma, and Life Technologies. The primers of a primer pair can have the<br>
same length. Alternatively, one of the primers of the primer pair can be longer<br>
than the other primer of the primer pair. The primers can have an identical<br>
melting temperature. The lengths of the primers can be extended or shortened at<br>
the 5' end or the 3' end to produce primers with desired melting temperatures. In a<br>
preferred embodiment, the 3' annealing lengths of the primers, within a primer<br>
pair, differ. Also, the annealing position of each primer pair can be designed such<br>
that the sequence and length of the primer pairs yield the desired melting<br>
temperature. The simplest equation for detennining the melting temperature of<br>
primers smaller than 25 base pairs is the Wallace Rule (Td = 2(A+T) + 4(G+C)).<br>
Computer programs can also be used to design primers, including but not limited<br>
to Array Designer Software (Arrayit Inc.), Oligonucleotide Probe Sequence<br>
Design Software for Genetic Analysis (Olympus Optical Co.), NetPrimer, and<br>
DNAsis from Hitachi Software Engineering. The TM (melting or annealing<br>
temperature) of each primer is calculated using software programs such as Net<br>
Primer (free web based program at<br>
http://premierbiosoft.com/nerorimer/netprlaunch/netprlaunch.html<br>
(internet address as of February 13,2002).<br>
[0097] In another embodiment, the annealing temperature of the primers can<br>
be recalculated and increased after any cycle of amplification, including but not<br>
limited to cycle 1,2,3,4,5, cycles 6-10, cycles 10-15, cycles 15-20, cycles 20-25,<br>
cycles 25-30, cycles 30-35, or cycles 35-40. After the initial cycles of<br>
amplification, the 5' half of the primers is incorporated into the products from<br>
each loci of interest, thus the TM can be recalculated based on both the sequences<br>
of the 5' half and the 3' half of each primer.<br>
[0098] For example, in FIG. 1B, the first cycle of amplification is performed<br>
at about the melting temperature of the 3' region of the second primer (region "c")<br>
that anneals to the template DNA, which is 13 bases. After the first cycle, the<br><br>
annealing temperature can be raised to TM2, which is about the melting<br>
temperature of the 3' region of the first primer (region "b"') that anneals to the<br>
template DNA. The second primer cannot bind to the original template DNA<br>
because it only anneals to 13 bases in the original DNA template, and TM2 is<br>
about the melting temperature of approximately 20 bases, which is the 3'<br>
annealing region of the first primer (FIG. 1C). However, the first primer can bind<br>
to the DNA that was copied in the first cycle of the reaction. In the third cycle,<br>
the annealing temperature is raised to TM3, which is about the melting<br>
temperature of the entire sequence of the second primer ("c" and "d"). The<br>
template! DNA produced from the second cycle of PCR contains both regions c'<br>
and d', and therefore, the second primer can anneal and extend at TM3 (FIG. 1D).<br>
The remaining cycles are performed at TM3. The entire sequence of the first<br>
primer (a + b1) can anneal to the template from the third cycle of PCR, and extend<br>
(FIG. 1E). Increasing the annealing temperature will decrease non-specific<br>
binding and increase the specificity of the reaction, which is especially useful if<br>
amplifying a locus of interest from human genomic DNA, which contains 3xl09<br>
base pairs.<br>
[0099] As used herein, the term "about" with regard to annealing temperatures<br>
is used to encompass temperatures within 10 degrees Celsius of the stated<br>
temperatures.<br>
[0100] In one embodiment, one primer pair is used for each locus of interest.<br>
However, multiple primer pairs can be used for each locus of interest.<br>
[0101] In one embodiment, primers are designed such that one or both primers<br>
of the primer pair contain sequence in the 5' region for one or more restriction<br>
endonucleases (restriction enzyme).<br>
[0102] As used herein, with regard to the position at which restriction<br>
enzymes digest DNA, the "sense" strand is the strand reading 5' to 3' in the<br>
direction in which the restriction enzyme cuts. For example, BsmF I recognizes<br>
the following sequence:<br>
5' GGGAC(N)10↓ 3' (SEQ ID NO.1) or<br><br>
3* CCCTG(N)i4T 5'<br>
5' '(N^GTCCC 3' (SEQ ID N0:2)<br>
3'T(N)ioCAGGG5'<br>
[0103] Thus, the sense strand is the strand containing the "GGGAC" sequence<br>
as it reads 5' to 3' in the direction that the restriction enzyme cuts.<br>
[0104] As used herein, with regard to the position at which restriction<br>
enzymes digest DNA, the "antisense" strand is the strand reading 3' to 5' in the<br>
direction in which the restriction enzyme cuts. Thus, the antisense strand is the<br>
strand that contains the "ccctg" sequence as it reads 3' to 5'.<br>
[0105] In the invention, one of the primers in a primer pair can be designed<br>
such that it contains a restriction enzyme recognition site for a restriction enzyme<br>
such mat digestion with the restriction enzyme produces a recessed 3' end and a 5'<br>
overhang that contains the locus of interest (herein referred to as a "second<br>
primer"). For example, the second primer of a primer pair can contain a<br>
recognition site for a restriction enzyme that does not cut DNA at the recognition<br>
site but cuts "n" nucleotides away from the recognition site. "N" is a distance from<br>
the recognition site to the site of the cut by the restriction enzyme. If the<br>
recognition sequence is for the restriction enzyme BceA I, the enzyme will cut ten<br>
(10) nucleotides from the recognition site on the sense strand, and twelve (12)<br>
nucleotides away from the recognition site on the antisense strand.<br>
[0106] The 3' region and preferably the 3' half of the primers is designed to<br>
anneal to a sequence that flanks the loci of interest (FIG. 1 A). The second primer<br>
may anneal any distance from the locus of interest provided that digestion with<br>
the restriction enzyme that recognizes the restriction enzyme recognition site on<br>
this primer generates a 5' overhang that contains the locus of interest. The 5'<br>
overhang can be of any size, including but not limited to 1,2,3,4, 5,6, 7, 8, and<br>
more than 8 bases.<br>
[0107] In a preferred embodiment, the 31 end of the second primer can anneal<br>
1,2,3,4, 5,6, 7, 8,9,10,11,12,13,14, or more than 14 bases from the locus of<br>
interest or at the locus of interest.<br><br>
[0108] In a preferred embodiment, the second primer is designed to anneal<br>
closer to the locus of interest than the other primer of a primer pair (the other<br>
primer is herein referred to as a "first primer"). The second primer can be a<br>
forward or reverse primer and the first primer can be a reverse or forward primer,<br>
respectively. Whether the first or second primer should be the forward or reverse<br>
primer can be determined by which design will provide better sequencing results.<br>
[0109] For example, the primer that anneals closer to the locus of interest can<br>
contain a recognition site for the restriction enzyme BsmF I, which cuts ten (10)<br>
nucleotides from the recognition site on the sense strand, and fourteen (14)<br>
nucleotides from the recognition site on the antisense strand. In this case, the<br>
primer can be designed so that the restriction enzyme recognition site is 13 bases,<br>
12	bases, 10 bases or 11 bases from the locus of interest. If the recognition site is<br>
13	bases from the locus of interest, digestion with BsmF I will generate a 5'<br>
overhang (RXXX), wherein the locus of interest (R) is the first nucleotide in the<br>
overhang (reading 3' to 5'), and X is any nucleotide. If the recognition site is 12<br>
bases from the locus of interest, digestion with BsmF I will generate a 5' overhang<br>
(XRXX), wherein the locus of interest (R) is the second nucleotide in the<br>
overhang (reading 3' to 5'). If the recognition site is 11 bases from the locus of<br>
interest, digestion with BsmF I will generate a 5' overhang (XXRX), wherein the<br>
locus of interest (R) is the third nucleotide in the overhang (reading 3' to 5'). The<br>
distance between the restriction enzyme recognition site and the locus of interest<br>
should be designed so that digestion with the restriction enzyme generates a 5'<br>
overhang;, which contains the locus of interest. The effective distance between the<br>
recognition site and the locus of interest will vary depending on the choice of<br>
restriction enzyme.<br>
[0110] In another embodiment, the second primer, which can anneal closer to<br>
the locus of interest relative to the first primer, can be designed so that the<br>
restriction enzyme that generates the 5' overhang, which contains the locus of<br>
interest, will see the same sequence at the cut site, independent of the nucleotide<br>
at the locus of interest. For example, if the primer that anneals closer to the locus<br><br>
of interest is designed so that the recognition site for the restriction enzyme BsmF<br>
I (5' GCrGAC 3') is thirteen bases from the locus of interest, the restriction enzyme<br>
will cut the antisense strand one base upstream of the locus of interest. The<br>
nucleotide at the locus of interest is adjacent to the cut site, and may vary from<br>
DNA molecule to DNA molecule. If it is desired that the nucleotides adjacent to<br>
the cut site be identical, the primer can be designed so that the restriction enzyme<br>
recognition site for BsmF I is twelve bases away from the locus of interest.<br>
Digestion with BsmF I will generate a 5' overhang, wherein the locus of interest is<br>
in the second position of the overhang (reading 3' to 5') and is no longer adjacent<br>
to the cut site. Designing the primer so that the restriction enzyme recognition<br>
site is twelve (12) bases from the locus of interest allows the nucleotides adjacent<br>
to the cut site to be the same, independent of the nucleotide at the locus of interest.<br>
Also, primers that have been designed so that the restriction enzyme recognition<br>
site is eleven (11) or ten (10) bases from the locus of interest will allow the<br>
nucleotides adjacent to the cut site to be the same, independent of the nucleotide<br>
at the locus of interest.<br>
[0111] The 3' end of the first primer (either the forward or the reverse) can be<br>
designed to anneal at a chosen distance from the locus of interest. Preferably, for<br>
example, this distance is between 10-25,25-50,50-75,75-100,100-150,150-200,<br>
200-250, 250-300,300-350, 350-400,400-450,450-500,500-550, 550-600,600-<br>
650,650-700,700-750,750-800, 800-850, 850-900,900-950,950-1000 and<br>
greater than 1000 bases away from the locus of interest. The annealing sites of<br>
the first primers are chosen such that each successive upstream primer is further<br>
and further away from its respective downstream primer.<br>
[0112] For example, if at locus of interest 1 the 3' ends of the first and second<br>
primers are Z bases apart, then at locus of interest 2, the 3' ends of the upstream<br>
and downstream primers are Z + K bases apart, where K = 1,2, 3,4,5-10,10-20,<br>
20-30,30-40,40-50,50-60,60-70, 70-80, 80-90,90-100,100-200,200-300, 300-<br>
400,400--500, 500-600,600-700,700-800, 800-900,900-1000, or greater than<br>
1000 bases (FIG 2). The purpose of making the upstream primers further and<br><br>
further apart from their respective downstream primers is so that the PCR<br>
products of all the loci of interest differ in size and can be separated, e.g., on a<br>
sequencing gel. This allows for multiplexing by pooling the PCR products in later<br>
steps.<br>
[0113] In one embodiment, the 5' region of the first primer can have a<br>
recognition site for any type of restriction enzyme. In a preferred embodiment,<br>
the first primer has at least one restriction enzyme recognition site that is different<br>
from the restriction enzyme recognition site in the second primer. In another<br>
preferred embodiment, the first primer anneals further away from the locus of<br>
interest than the second primer.<br>
[0114] In a preferred embodiment, the second primer contains a restriction<br>
enzyme recognition sequence for a Type IIS restriction enzyme including but not<br>
limited to BceA I and BsmF I, which produce a two base 5' overhang and a four<br>
base 5' overhang, respectively. Restriction enzymes that are Type IIS are<br>
preferred because they recognize asymmetric base sequences (not palindromic<br>
like the orthodox Type II enzymes). Type IIS restriction enzymes cleave DNA at<br>
a specified position that is outside of the recognition site, typically up to 20 base<br>
pairs outside of the recognition site. These properties make Type IIS restriction<br>
enzymes, and the recognition sites thereof, especially useful in the method of the<br>
invention. Preferably, the Type IIS restriction enzymes used in this method leave<br>
a 5' overhang and a recessed 3' end.<br>
[0115] A wide variety of Type US restriction enzymes are known and such<br>
enzymes have been isolated from bacteria, phage, archaebacteria and viruses of<br>
eukaryotic algae and are commercially available (Promega, Madison WI; New<br>
England Biolabs, Beverly, MA; Szybalski W. etal, Gene 100:13-16, (1991)).<br>
Examples of Type IIS restriction enzymes that would be useful in the method of<br>
the invention include, but are not limited to enzymes such as those listed in Table<br>
I.<br><br><br>
[0116] In one embodiment, a primer pair has sequence at the 5' region of each<br>
of the primers that provides a restriction enzyme recognition site that is unique for<br>
one restriction enzyme.<br><br>
[0117] In another embodiment, a primer pair has sequence at the 5' region of<br>
each of the primers that provide a restriction site that is recognized by more than<br>
one restriction enzyme, and especially for more than one Type IIS restriction<br>
enzyme. For example, certain consensus sequences can be recognized by more<br>
than one enzyme. For example, Bsgl, Eco57I and Bpml all recognize the<br>
consensus 5' (G/C)TgnAG 3' and cleave 16 bp away on the antisense strand and<br>
14 bp away on the sense strand. A primer mat provides such a consensus<br>
sequence would result in a product that has a site that can be recognized by any of<br>
the restriction enzymes Bsgl, Eco57I and Bpml.<br>
[0118] Other restriction enzymes that cut DNA at a distance from the<br>
recognition site, and produce a recessed 3' end and a 5' overhang include Type III<br>
restriction enzymes. For example, the restriction enzyme EcoP15I recognizes the<br>
sequence 5' CAGCAG 3' and cleaves 25 bases downstream on the sense strand<br>
and 27 bases on the antisense strand. It will be further appreciated by a person of<br>
ordinary skill in the art that new restriction enzymes are continually being<br>
discovered and may readily be adopted for use in the subject invention.<br>
[0119] In another embodiment, the second primer can contain a portion of the<br>
recognition sequence for a restriction enzyme, wherein the full recognition site for<br>
the restriction enzyme is generated upon amplification of the template DNA such<br>
that digestion with the restriction enzyme generates a 5' overhang containing the<br>
locus of interest. For example, the recognition site for BsmF I is 5' GGGACN10↓<br>
3'. The 3' region, which anneals to the template DNA, of the second primer can<br>
end with the nucleotides "GGG," which do not have to be complementary with the<br>
template DNA. If the 3' annealing region is about 10-20 bases, even if the last<br>
three bases do not anneal, the primer will extend and, generate a BsmF I site.<br>
Second primer: 5' GGAAATTCCATGATGCGTGGG-* (SEQ ID NO:3)<br>
Template DNA: 3' CCTTTAAGGTACTACGCAN1,N2,N3,TG 5'<br>
5' GGAAATTCCATGATGCGTN1N2N3 AC 3' (SEQ ID NO:4)<br>
[0120] The second primer can be designed to anneal to the template DNA,<br>
wherein the next two bases of the template DNA are thymidine and guanine, such<br><br>
that an adenosine and cytosine are incorporated into the primer forming a<br>
recognition site for BsmF I, 5' GGGACN10↓3'. The second primer can be<br>
designed to anneal in such a manner that digestion with BsmF I generates a 5'<br>
overhang containing the locus of interest.<br>
[0121] In another embodiment, the second primer can contain an entire or full<br>
recognition site for a restriction enzyme or a portion of a recognition site, which<br>
generates a full recognition site upon amplification of the template DNA such that<br>
digestion with a restriction enzyme that cuts at the recognition site generates a 5'<br>
overhang that contains the locus of interest. For example, the restriction enzyme<br>
BsaJ I binds the following recognition site: 5' C↓CN1N2GG 3'. The second primer<br>
can be designed such that the 3' region of the primer ends with "CC." The SNP of<br>
interest is represented by "Nr", and the template sequence downstream of the<br>
SNP is "N2'CC."<br>
Second primer 5' GGAAATTCCATGATGCGTACC→ (SEQ ID NO:5)<br>
Template DNA 3' CCTTTAAGGTACTACGCATGGN1N2CC 5'<br>
5' GGAAATTCCATGATGCGTACCN1N2GG 3' (SEQ ID<br>
NO:6)<br>
[0122] After digestion with BsaJ I, a 5' overhang of the following sequence<br>
would be generated:<br>
5' C	3'<br>
3' GGN1-N2'C 5'<br>
[0123] If the nucleotide guanine is not reported at the locus of interest, the 3'<br>
recessed end can be filled in with unlabeled cytosine, which is complementary to<br>
the first nucleotide in the overhang. After removing the excess cytosine, labeled<br>
ddNTPs can be used to fill in the next nucleotide, Nr, which represents the locus<br>
of interest. Alternatively if guanine is reported to be a potential nucleotide at the<br>
locus of interest, labeled nucleotides can be used to detect a nucleotide 3' of the<br>
locus of interest. Unlabeled dCTP can be used to "fill in" followed by a fill in<br>
with a labeled nucleotide other that cytosine. Cytosine will be incorporated until<br>
it reaches a base that is not complementary. If the locus of interest contained a<br>
guanine, it would be filled in with the dCTP, which would allow incorporation of<br><br>
the labeled nucleotide. However, if the locus of interest did not contain a guanine,<br>
the labeled nucleotide would not be incorporated. Other restriction enzymes can<br>
be used including but not limited to BssK I (5' ↓CNGG 3'), Dde I (5' C↓TNAG<br>
3'), EcoN I (5' CCTNN↓NNNAGG 3') (SEQ ID NO:7), Fnu4H I (5' GC↓NGC 3'),<br>
Hinf I (5' G↓ANTC 3'), PflF I (5' GACN↓NNGTC 3'), Sau96I (5' G↓NCC 3'),<br>
ScrF I (5' CC↓NGG 3'), and Tthl 111 (5' GACN↓NNGTC 3')-<br>
[0124] It is not necessary that the 3' region, which anneals to the template<br>
DNA, of the second primer be 100% complementary to the template DNA. For<br>
example, the last 1,2, or 3 nucleotides of the 3' end of the second primer can be<br>
mismatches with the template DNA. The region of the primer that anneals to the<br>
template DNA will target the primer, and allow the primer to extend. Even if, for<br>
example, the last two nucleotides are not complementary to the template DNA,<br>
the primer will extend and generate a restriction enzyme recognition site.<br>
Second primer: 5' GGAAATTCCATGATGCGTACC→ (SEQ ID NO:5)<br>
Template DNA: 3' CCTTTAAGGTACTACGCATNa.NblN1-N2'CC 5'<br>
5' GGAAATTCCATGATGCGTANaNbN1N2GG 3' (SEQ ID<br>
NO:8)<br>
[0125] After digestion with BsaJ I, a 5' overhang of the following sequence<br>
would be generated:<br>
5'C	3'<br>
3' GGN1'N2-C 5'<br>
[0126] If the nucleotide cytosine is not reported at the locus of interest, the 5'<br>
overhang can be filled in with unlabeled cytosine. The excess cytosine can be<br>
rinsed away, and filled in with labeled ddNTPs. The first nucleotide incorporated<br>
(N1) corresponds to the locus of interest.<br>
[0127] Alternatively, it is possible to create the full restriction enzyme<br>
recognition sequence using the first and second primers. The recognition site for<br>
any resitriction enzyme can be generated, as long as the recognition site contains at<br>
least one variable nucleotide. Restriction enzymes that recognize sites that<br><br>
contain at least one variable nucleotide include but are not limited to BssK I<br>
(5'↓CCNGG 3'), Dde I (5'C'TNAG 3% Econ I (5,CCTNN↓NNNAGG3') (SEQ ID<br>
NO:7), Fnu4H I (5'GC↓NGC 3'), Hinf I (5'G'ANTC 3') PflF I (5' GACN↓NNGTC<br>
3'), Sau961 (5' G'GNCC 3'), ScrF I (5' CC↓NGG 3'), and Tthl 111 (5'<br>
GACN↓NNGTC 3'). In this embodiment, the first or second primer may anneal<br>
closer to the locus of interest or the first or second primer may anneal at an equal<br>
distance from the locus of interest. The first and second primers can be designed<br>
to contain mismatches to the template DNA at the 3' region; these mismatches<br>
create the restriction enzyme recognition site. The number of mismatches that can<br>
be tolerated at the 3' end depends on the length of the primer, and includes but is<br>
not limited to 1,2, or more than 2 mismatches. For example, if the locus of<br>
interest is represented by Nr, a first primer can be designed to be complementary<br>
to the template DNA, depicted below as region "a." The 3' region of the first<br>
primer ends with "CC," which is not complementary to the template DNA. The<br>
second primer is designed to be complementary to the template DNA, which is<br>
depicted below as region "b"'. The 3' region of the second primer ends with "CC,"<br>
which is not complementary to the template DNA.<br><br>
[0128] After one round of amplification the following products would be<br>
generated:<br><br>
[0129] In cycle two, the primers can anneal to the templates that were<br>
generated from the first cycle of PCR:<br><br><br>
[0130] After cycle two of PCR, the following products would be generated:<br><br>
[0131] The restriction enzyme recognition site for BsaJ I is generated, and<br>
after digestion with BsaJ I, a 5' overhang containing the locus of interest is<br>
generated. The locus of interest can be detected as described in detail below.<br>
Alternatively, the 3' region of the first and second primers can contain 1,2,3, or<br>
more than 3 mismatches followed by a nucleotide that is complementary to the<br>
template DNA. For example, the first and second primers can be used to create a<br>
recognition site for the restriction enzyme EcoN I, which binds the following<br>
DNA sequence: 5' CCTNN↓NNNAGG 3'. The last nucleotides of each primer<br>
would be "CCTN1 or CCTN1N2." The nucleotides "CCT" may or may not be<br>
complementary to the template DNA; however, N1 and N2 are nucleotides<br>
complementary to the template DNA. This allows the primers to anneal to the<br>
template DNA after the potential mismatches, which are used to create the<br>
restriction enzyme recognition site.<br>
[0132] In another embodiment, a primer pair has sequence at the 5' region of<br>
each of the primers that provides two or more restriction sites that are recognized<br>
by two or more restriction enzymes.<br>
[0133] In a most preferred embodiment, a primer pair has different restriction<br>
enzyme recognition sites at the 5' regions, especially 5' ends, such that a different<br>
restriction enzyme is required to cleave away any undesired sequences. For<br>
example, the first primer for locus of interest "A" can contain sequence<br>
recognized by a restriction enzyme, "X," which can be any type of restriction<br>
enzyme, and the second primer for locus of interest "A," which anneals closer to<br>
the locus of interest, can contain sequence for a restriction enzyme, "Y," which is<br><br>
a Type IIS restriction enzyme that cuts "n" nucleotides away and leaves a 5'<br>
overhang and a recessed 3' end. The 5' overhang contains the locus of interest.<br>
After binding the amplified DNA to streptavidin coated wells, one can digest with<br>
enzyme "Y," rinse, then fill in with labeled nucleotides and rinse, and then digest<br>
with restriction enzyme "X," which will release the DNA fragment containing the<br>
locus of interest from the solid matrix. The locus of interest can be analyzed by<br>
detecting the labeled nucleotide that was "filled in" at the locus of interest, e.g.<br>
SNP site.<br>
[0134] In another embodiment, the second primers for the different loci of<br>
interest that are being amplified according to the invention contain recognition<br>
sequence in the 5' regions for the same restriction enzyme and likewise all the first<br>
primers also contain the same restriction enzyme recognition site, which is a<br>
different enzyme from the enzyme that recognizes the second primers. The<br>
primer (either the forward or reverse primer) that anneals closer to the locus of<br>
interest contains a recognition site for, e.g., a Type IIs restriction enzyme.<br>
[0135] In another embodiment, the second primers for the multiple loci of<br>
interest that are being amplified according to the invention contain restriction<br>
enzyme recognition sequences in the 5' regions for different restriction enzymes.<br>
[0136] In another embodiment, the first primers for the multiple loci of<br>
interest that are being amplified according to the invention contain restriction<br>
enzyme recognition sequences in the 5' regions for different restriction enzymes.<br>
[0137] Multiple restriction enzyme sequences provide an opportunity to<br>
influence the order in which pooled loci of interest are released from the solid<br>
support. For example, if 50 loci of interest are amplified, the first primers can<br>
have a tag at the extreme 5' end to aid in purification and a restriction enzyme<br>
recognition site, and the second primers can contain a recognition site for a type<br>
IIS restriction enzyme. For example, several of the first primers can have a<br>
restriction enzyme recognition site for EcoR I, other first primers can have a<br>
recognition site for Pst I, and still other first primers can have a recognition site<br>
for BamH I. After amplification, the loci of interest can be bound to a solid<br><br>
support with the aid of the tag on the first primers. By performing the restriction<br>
digests one restriction enzyme at a time, one can serially release the amplified loci<br>
of interest. If the first digest is performed with EcoRI, the loci of interest<br>
amplified with the first primers containing the recognition site for EcoR I will be<br>
released, and collected while the other loci of interest remain bound to the solid<br>
support. The amplified loci of interest can be selectively released from the solid<br>
support by digesting with one restriction enzyme at a time. The use of different<br>
restriction enzyme recognition sites in the first primers allows a larger number of<br>
loci of interest to be amplified in a single reaction tube.<br>
[0138] In a preferred embodiment, any region 5' of the restriction enzyme<br>
digestion site of each primer can be modified with a functional group that<br>
provides for fragment manipulation, processing, identification, and/or<br>
purification. Examples of such functional groups, or tags, include but are not<br>
limited to biotin, derivatives of biotin, carbohydrates, haptens, dyes, radioactive<br>
molecules, antibodies, and fragments of antibodies, peptides, and immunogenic<br>
molecules.<br>
[0139] In another embodiment, the template DNA can be replicated once,<br>
without being amplified beyond a single round of replication. This is useful when<br>
there is a large amount of the DNA available for analysis such that a large number<br>
of copies of the loci of interest are already present in the sample, and further<br>
copies are not needed. In this embodiment, the primers are preferably designed to<br>
contain a "hairpin" structure in the 5' region, such that the sequence doubles back<br>
and anneals to a sequence internal to itself in a complementary manner. When the<br>
template DNA is replicated only once, the DNA sequence comprising the<br>
recognition site would be single-stranded if not for the "hairpin" structure.<br>
However, in the presence of the hairpin structure, that region is effectively double<br>
stranded, thus providing a double stranded substrate for activity by restriction<br>
enzymes.<br>
[0140] To the extent that the reaction conditions are compatible, all the primer<br>
pairs to analyze a locus or loci of interest of DNA can be mixed together for use<br><br>
in the method of the invention. In a preferred embodiment, all primer pairs are<br>
mixed with the template DNA in a single reaction vessel. Such a reaction vessel<br>
can be, for example, a reaction tube, or a well of a microtiter plate.<br>
[01411 Alternatively, to avoid competition for nucleotides and to minimize<br>
primer dimers and difficulties with annealing temperatures for primers, each locus<br>
of interest or small groups of loci of interest can be amplified in separate reaction<br>
tubes or wells, and the products later pooled if desired. For example, the separate<br>
reactions can be pooled into a single reaction vessel before digestion with the<br>
restriction enzyme that generates a 5' overhang, which contains the locus of<br>
interest or SNP site, and a 3' recessed end. Preferably, the primers of each primer<br>
pair are provided in equimolar amounts. Also, especially preferably, each of the<br>
different primer pairs is provided in equimolar amounts relative to the other pairs<br>
that are being used.<br>
[0142] In another embodiment, combinations of primer pairs that allow<br>
efficient amplification of their respective loci of interest can be used (see e.g. FIG.<br>
2). Such combinations can be determined prior to use in the method of the<br>
invention. Multi-well plates and PCR machines can be used to select primer pairs<br>
that work efficiently with one another. For example, gradient PCR machines,<br>
such as the Eppendorf Mastercycler® gradient PCR machine, can be used to<br>
select the optimal annealing temperature for each primer pair. Primer pairs that<br>
have similar properties can be used together in a single reaction tube.<br>
[0143] In another embodiment, a multi-sample container including but not<br>
limited to a 96-well or more plate can be used to amplify a single locus of interest<br>
with the same primer pairs from multiple template DNA samples with optimal<br>
PCR conditions for that locus of interest. Alternatively, a separate multi-sample<br>
container can be used for amplification of each locus of interest and the products<br>
for each template DNA sample later pooled. For example, gene A from 96<br>
different DNA samples can be amplified in microtiter plate 1, gene B from 96<br>
different DNA samples can be amplified in microtiter plate 2, etc., and then the<br>
amplification products can be pooled.<br><br>
[0144] The result of amplifying multiple loci of interest is a preparation that<br>
contains representative PCR products having the sequence of each locus of<br>
interest. For example, if DNA from only one individual is used as the template<br>
DNA and if hundreds of disease-related loci of interest were amplified from the<br>
template DNA, the amplified DNA would be a mixture of small, PCR products<br>
from each of the loci of interest. Such a preparation could be further analyzed at<br>
that time to determine the sequence at each locus of interest or at only some of<br>
loci of interest. Additionally, the preparation could be stored in a manner that<br>
preserves the DNA and can be analyzed at a later time. Information contained in<br>
the amplified DNA can be revealed by any suitable method including but not<br>
limited to fluorescence detection, sequencing, gel electrophoresis, and mass<br>
spectrometry (see "Detection of Incorporated Nucleotide" section below).<br>
Amplification of Loci of Interest<br>
[0145] The template DNA can be amplified using any suitable method known<br>
in the art including but not limited to PCR (polymerase chain reaction), 3SR (self-<br>
sustained sequence reaction), LCR (ligase chain reaction), RACE-PCR (rapid<br>
amplification of cDNA ends), PLCR (a combination of polymerase chain reaction<br>
and ligase chain reaction), Q-beta phage amplification (Shah et al., J. Medical<br>
Micro. 33:1435-41 (1995)), SDA (strand displacement amplification), SOE-PCR<br>
(splice overlap extension PCR), and the like. These methods can be used to<br>
design variations of the releasable primer mediated cyclic amplification reaction<br>
explicitly described in this application. In the most preferred embodiment, the<br>
template DNA is amplified using PCR (PCR: A Practical Approach, M. J.<br>
McPherson, et al., IRL Press (1991); PCR Protocols: A Guide to Methods and<br>
Applications, Innis, et al., Academic Press (1990); and PCR Technology:<br>
Principals and Applications of DNA Amplification, H. A. Erlich, Stockton Press<br>
(1989)). PCR is also described in numerous U.S. patents, including U.S. Pat. Nos.<br>
4,683,195; 4,683,202; 4,800,159; 4,965,188; 4,889,818; 5,075,216; 5,079,352;<br>
5,104,792,5,023,171; 5,091,310; and 5,066,584.<br><br>
[0146] The components of a typical PCR reaction include but are not limited<br>
to a template DNA, primers, a reaction buffer (dependent on choice of<br>
polymerase), dNTPs (dATP, dTTP, dGTP, and dCTP) and a DNA polymerase.<br>
Suitable PCR primers can be designed and prepared as discussed above (see<br>
"Primer Design" section above). Briefly, the reaction is heated to 95°C for 2 min.<br>
to separate the strands of the template DNA, the reaction is cooled to an<br>
appropriate temperature (determined by calculating the annealing temperature of<br>
designed primers) to allow primers to anneal to the template DNA, and heated to<br>
72°C for two minutes to allow extension.<br>
[0147] In a preferred embodiment, the annealing temperature is increased in<br>
each of the first three cycles of amplification to reduce non-specific amplification.<br>
See also Example 1, below. The TM1 of the first cycle of PCR is about the<br>
melting temperature of the 3' region of the second primer that anneals to the<br>
template DNA. The annealing temperature can be raised in cycles 2-10,<br>
preferably in cycle 2, to TM2, which is about the melting temperature of the 3'<br>
region, which anneals to the template DNA, of the first primer. If the annealing<br>
temperature is raised in cycle 2, the annealing temperature remains about the same<br>
until the next increase in annealing temperature. Finally, in any cycle subsequent<br>
to the cycle in which the annealing temperature was increased to TM2, preferably<br>
cycle 3, the annealing temperature is raised to TM3, which is about the melting<br>
temperature of the entire second primer. After the third cycle, the annealing<br>
temperature for the remaining cycles may be at about TM3 or may be further<br>
increased. In this example, the annealing temperature is increased in cycles 2 and<br>
3. However, the annealing temperature can be increased from a low annealing<br>
temperature in cycle 1 to a high annealing temperature in cycle 2 without any<br>
further increases in temperature or the annealing temperature can progressively<br>
change from a low annealing temperature to a high annealing temperature in any<br>
number of incremental steps. For example, the annealing temperature can be<br>
changed in cycles 2,3,4,5,6, etc.<br><br>
[0148] After annealing, the temperature in each cycle is increased to an<br>
"extension" temperature to allow the primers to "extend" and then following<br>
extension the temperature in each cycle is increased to the denaturization<br>
temperature. For PCR products less than 500 base pairs in size, one can eliminate<br>
the extension step in each cycle and just have denaturization and annealing steps.<br>
A typical PCR reaction consists of 25-45 cycles of denaturation, annealing and<br>
extension as described above. However, as previously noted, even only one cycle<br>
of amplification (one copy) can be sufficient for practicing the invention.<br>
[0149] Any DNA polymerase that catalyzes primer extension can be used<br>
including but not limited to E. coli DNA polymerase, Klenow fragment of E. coli<br>
DNA polymerase I, T7 DNA polymerase, T4 DNA polymerase, Taq polymerase,<br>
Pfu DNA polymerase, Vent DNA polymerase, bacteriophage 29, and REDTaq™<br>
Genomic DNA polymerase, or sequenase. Preferably, a thermostable DNA<br>
polymerase is used. A "hot start" PCR can also be performed wherein the reaction<br>
is heated to 95°C for two minutes prior to addition of the polymerase or the<br>
polymerase can be kept inactive until the first heating step in cycle 1. "Hot start"<br>
PCR can be used to minimize nonspecific amplification. Any number of PCR<br>
cycles can be used to amplify the DNA, including but not limited to 2,5,10,15,<br>
20,25,30,35,40, or 45 cycles. In a most preferred embodiment, the number of<br>
PCR cycles performed is such that equimolar amounts of each loci of interest are<br>
produced.<br>
Purification of Amplified DNA<br>
[0150] Purification of the amplified DNA is not necessary for practicing the<br>
invention. However, in one embodiment, if purification is preferred, the 5' end of<br>
the primer (first or second primer) can be modified with a tag that facilitates<br>
purification of the PCR products. In a preferred embodiment, the first primer is<br>
modified with a tag that facilitates purification of the PCR products. The<br>
modification is preferably the same for all primers, although different<br>
modifications can be used if it is desired to separate the PCR products into<br>
different groups.<br><br>
[0151] The tag can be a radioisotope, fluorescent reporter molecule,<br>
chemiluminescent reporter molecule, antibody, antibody fragment, hapten, biotin,<br>
derivative of biotin, photobiotin, iminobiotin, digoxigenin, avidin, enzyme,<br>
acridinium, sugar, enzyme, apoenzyme, homopolymeric oligonucleotide,<br>
hormone, ferromagnetic moiety, paramagnetic moiety, diamagnetic moiety,<br>
phosphorescent moiety, luminescent moiety, electrochemiluminescent moiety,<br>
chromatic moiety, moiety having a detectable electron spin resonance, electrical<br>
capacitance, dielectric constant or electrical conductivity, or combinations thereof.<br>
[0152] In a preferred embodiment, the 5' ends of the primers can be<br>
biotinylated (Kandpal et al., Nucleic Acids Res. 18:1789-1795 (1990); Kaneoka et<br>
al., Biotechniques 10:30-34 (1991); Green et al, Nucleic Acids Res. 18:6163-<br>
6164 (1990)). The biotin provides an affinity tag that can be used to purify the<br>
copied DNA from the genomic DNA or any other DNA molecules that are not of<br>
interest. Biotinylated molecules can be purified using a streptavidin coated matrix<br>
as shown in FIG. 1F, including but not limited to Streptawell, transparent, High-<br>
Bind plates from Roche Molecular Biochemicals (catalog number 1 645 692, as<br>
listed in Roche Molecular Biochemicals, 2001 Biochemicals Catalog).<br>
[0153] The PCR product of each locus of interest is placed into separate wells<br>
of a Streptavidin coated plate. Alternatively, the PCR products of the loci of<br>
interest can be pooled and placed into a streptavidin coated matrix, including but<br>
not limited to the Streptawell, transparent, High-Bind plates from Roche<br>
Molecular Biochemicals (catalog number 1 645 692, as listed in Roche Molecular<br>
Biochemicals, 2001 Biochemicals Catalog).<br>
[0154] The amplified DNA can also be separated from the template DNA<br>
using non-affinity methods known in the art, for example, by polyacrylamide gel<br>
electrophoresis using standard protocols.<br>
Digestion of Amplified DNA<br>
[0155] The amplified DNA can be digested with a restriction enzyme that<br>
recognizes a sequence that had been provided on the first or second primer using<br>
standard protocols known within the art (FIGS. 6A-6D). The enzyme used<br><br>
depends on the restriction recognition site generated with the first or second<br>
primer. See "Primer Design" section, above, for details on restriction recognition<br>
sites generated on primers.<br>
[0156] Type IIS restriction enzymes are extremely useful in that they cut<br>
approximately 10-20 base pairs outside of the recognition site. Preferably, the<br>
Type IIS restriction enzymes used are those that generate a 5' overhang and a<br>
recessed 3' end, including but not limited to BceA I and BsmF I (see e.g. Table I).<br>
In a most preferred embodiment, the second primer (either forward or reverse),<br>
which anneals close to the locus of interest, contains a restriction enzyme<br>
recognition sequence for BsmF I or BceA I. The Type IIS restriction enzyme<br>
BsmF I recognizes the nucleic acid sequence GGGAC, and cuts 14 nucleotides<br>
from the recognition site on the antisense strand and 10 nucleotides from the<br>
recognition site on the sense strand. Digestion with BsmF I generates a 5'<br>
overhang of four (4) bases.<br>
[0157] For example, if the second primer is designed so that after<br>
amplification the restriction enzyme recognition site is 13 bases from the locus of<br>
interest, then after digestion, the locus of interest is the first base in the 5'<br>
overhang (reading 3' to 5'), and the recessed 3' end is one base upstream of the<br>
locus of interest. The 3' recessed end can be filled in with a nucleotide that is<br>
complementary to the locus of interest. One base of the overhang can be filled in<br>
using dideoxynucleotides. However, 1,2,3, or all 4 bases of the overhang can be<br>
filled in using deoxynucleotides or a mixture of dideoxynucleotides and<br>
deoxynucleotides.<br>
[0158] The restriction enzyme BsmF I cuts DNA ten (10) nucleotides from<br>
the recognition site on the sense strand and fourteen (14) nucleotides from the<br>
recognition site on the antisense strand. However, in a sequence dependent<br>
manner, the restriction enzyme BsmF I also cuts eleven (11) nucleotides from the<br>
recognition site on the sense strand and fifteen (15) nucleotides from the<br>
recognition site on the antisense strand. Thus, two populations of DNA molecules<br>
exist after digestion: DNA molecules cut at 10/14 and DNA molecules cut at<br><br>
11/15. If the recognition site for BsmF I is 13 bases from the locus of interest in<br>
the amplified product, then DNA molecules cut at the 11/15 position will generate<br>
a 5' overhang that contains the locus of interest in the second position of the<br>
overhang (reading 3' to 5'). The 3' recessed end of the DNA molecules can be<br>
filled in with labeled nucleotides. For example, if labeled dideoxynucleotides are<br>
used, the 3' recessed end of the molecules cut at 11/15 would be filled in with one<br>
base, which corresponds to the base upstream of the locus of interest, and the 3'<br>
recessed end of molecules cut at 10/14 would be filled in with one base, which<br>
corresponds to the locus of interest. The DNA molecules that have been cut at the<br>
10/14 position and the DNA molecules that have been cut at the 11/15 position<br>
can be: separated by size, and the incorporated nucleotides detected. This allows<br>
detection of both the nucleotide before the locus of interest, detection of the locus<br>
of interest, and potentially the three bases pairs after the locus of interest.<br>
[0159| Alternatively, if the base upstream of the locus of interest and the locus<br>
of interest are different nucleotides, then the 3' recessed end of the molecules cut<br>
at 11/15 can be filled in with deoxynucleotide that is complementary to the<br>
upstream base. The remaining deoxynucleotide is washed away, and the locus of<br>
interest site can be filled in with either labeled deoxynucleotides, unlabeled<br>
deoxynucleotides, labeled dideoxynucleotides, or unlabeled dideoxynucleotides.<br>
After the fill in reaction, the nucleotide can be detected by any suitable method.<br>
Thus, after the first fill in reaction with dNTP, the 3' recessed end of the<br>
molecules cut at 10/14 and 11/15 is upstream of the locus of interest. The 3'<br>
recessed end can now be filled in one base, which corresponds to the locus of<br>
interest, two bases, three bases or four bases.<br>
[0160] Alternatively, if the base upstream of the locus of interest and the base<br>
downstream of the locus of interest are reported to be the same, the 3' recessed<br>
end of the molecules cut at 11/15 can be "filled in" with unlabeled<br>
deoxynucleotide, followed by a "fill in" with labeled dideoxynucleotide. For<br>
example, if the nucleotide upstream of the locus of interest is a cytosine, and a<br>
cytosine is a potential nucleotide at the locus of interest, and an adenosine is the<br><br>
first nucleotide 3' of the locus of interest, a "fill in" reaction can be performed<br>
with unlabeled deoxyguanine triphosphate (dGTP), followed by a fill in with<br>
labeled dideoxythymidine triphosphate. If the locus of interest contains a<br>
cytosine, the ddTTP will be incorporated and detected. However, if the locus of<br>
interest does not contain a cytosine, the dGTP will not be incorporated, which<br>
prevents incorporation of the ddTTP.<br>
[0161] The restriction enzyme BceA I recognizes the nucleic acid sequence<br>
ACGGC and cuts 12 (twelve) nucleotides from the recognition site on the sense<br>
strand and 14 (fourteen) nucleotides from the recognition site on the antisense<br>
strand. If the distance from the recognition site for BceA I on the second primer<br>
is designed to be thirteen (13) bases from the locus of interest (see FIGS. 4A-4D),<br>
digestion with BceA I will generate a 5' overhang of two bases, which contains<br>
the locus of interest, and a recessed 3' end that is upstream of the locus of interest.<br>
The locus of interest is the first nucleotide in the 5' overhang (reading 3' to 5').<br>
[0162] Alternative cutting is also seen with the restriction enzyme BceA I,<br>
although at a much lower frequency than is seen with BsmF I. The restriction<br>
enzyme BceA I can cut thirteen (13) nucleotides from the recognition site on the<br>
sense strand and fifteen (15) nucleotides from the recognition site on the antisense<br>
strand. Thus, two populations of DNA molecules exist: DNA molecules cut at<br>
12/14 and DNA molecules cut at 13/15. If the restriction enzyme recognition site<br>
is 13 bases from the locus of interest in the amplified product, DNA molecules cut<br>
at the 13/15 position yield a 5' overhang, which contains the locus of interest in<br>
the second position of the overhang (reading 3' to 5'). Labeled dideoxynucleotides<br>
can be used to fill in the 3' recessed end of the DNA molecules. The DNA<br>
molecules cut at 13/15 will have the base upstream of the locus of interest filled<br>
in, and the DNA molecules cut at 12/14 will have the locus of interest site filled<br>
in. The DNA molecules cut at 13/15 and those cut at 12/14 can be separated by<br>
size, and the incorporated nucleotide detected. Thus, the alternative cutting can<br>
be used to obtain additional sequence information.<br><br>
[0163] Alternatively, if the two bases in the 5' overhang are different, the 3'<br>
recessed end of the DNA molecules, which were cut at 13/15, can be filled in with<br>
the deoxynucleotide complementary to the first base in the overhang, and excess<br>
deoxynucleotide washed away. After filling in, the 3' recessed end of the DNA<br>
molecules that were cut at 12/14 and the DNA molecules that were cut at 13/15<br>
are upstream of the locus of interest. The 3' recessed ends can be filled with either<br>
labeled dideoxynucleotides, unlabeled dideoxynucleotides, labeled<br>
deoxynucleotides, or unlabeled deoxynucleotides.<br>
[0164] If the primers provide different restriction sites for certain of the loci of<br>
interest that were copied, all the necessary restriction enzymes can be added<br>
together to digest the copied DNA simultaneously. Alternatively, the different<br>
restriction digests can be made in sequence, for example, using one restriction<br>
enzyme at a time, so that only the product that is specific for that restriction<br>
enzyme is digested.<br>
Incorporation of Labeled Nucleotides<br>
[0165] Digestion with the restriction enzyme that recognizes the sequence on<br>
the second primer generates a recessed 3' end and a 5' overhang, which contains<br>
the locus of interest (FIG. 1G). The recessed 3' end can be filled in using the 5'<br>
overhang as a template in the presence of unlabeled or labeled nucleotides or a<br>
combination of both unlabeled and labeled nucleotides. The nucleotides can be<br>
labeled with any type of chemical group or moiety that allows for detection<br>
including but not limited to radioactive molecules, fluorescent molecules,<br>
antibodies, antibody fragments, haptens, carbohydrates, biotin, derivatives of<br>
biotin, phosphorescent moieties, luminescent moieties, electrochemiluminescent<br>
moieties, chromatic moieties, and moieties having a detectable electron spin<br>
resonance, electrical capacitance, dielectric constant or electrical conductivity.<br>
The nucleotides can be labeled with one or more than one type of chemical group<br>
or moiety. Each nucleotide can be labeled with the same chemical group or<br>
moiety. Alternatively, each different nucleotide can be labeled with a different<br>
chemical group or moiety. The labeled nucleotides can be dNTPs, ddNTPs, or a<br><br>
mixture of both dNTPs and ddNTPs. The unlabeled nucleotides can be dNTPs,<br>
ddNTPs or a mixture of both dNTPs and ddNTPs.<br>
[0166] Any combination of nucleotides can be used to incorporate nucleotides<br>
including but not limited to unlabeled deoxynucleotides, labeled<br>
deoxynucleotides, unlabeled dideoxynucleotides, labeled dideoxynucleotides, a<br>
mixture of labeled and unlabeled deoxynucleotides, a mixture of labeled and<br>
unlabeled dideoxynucleotides, a mixture of labeled deoxynucleotides and labeled<br>
dideoxynucleotides, a mixture of labeled deoxynucleotides and unlabeled<br>
dideoxynucleotides, a mixture of unlabeled deoxynucleotides and unlabeled<br>
dideoxynucleotides, a mixture of unlabeled deoxynucleotides and labeled<br>
dideoxynucleotides, dideoxynucleotide analogues, deoxynucleotide analogues, a<br>
mixture of dideoxynucleotide analogues and deoxynucleotide analogues,<br>
phosphorylated nucleoside analogues, 2-deoxynucleoside—5' triphosphates and<br>
modified 2'-deoxynucleoside triphosphates.<br>
[0167] For example, as shown in FIG. 1H, in the presence of a polymerase,<br>
the 3' recessed end can be filled in with fluorescent ddNTP using the 5' overhang<br>
as a template. The incorporated ddNTP can be detected using any suitable<br>
method including but not limited to fluorescence detection.<br>
[0168] All four nucleotides can be labeled with different fluorescent groups,<br>
which will allow one reaction to be performed in the presence of all four labeled<br>
nucleotides. Alternatively, five separate "fill in" reactions can be performed for<br>
each locus of interest; each of the four reactions will contain a different labeled<br>
nucleotide (e.g. ddATP*, ddTTP*, ddUTP*, ddGTP*, or ddCTP*, where *<br>
indicates a labeled nucleotide). Each nucleotide can be labeled with different<br>
chemical groups or the same chemical groups. The labeled nucleotides can be<br>
dideoxynucleotides or deoxynucleotides.<br>
[0169] In another embodiment, nucleotides can be labeled with fluorescent<br>
dyes including but not limited to fluorescein, pyrene, 7-methoxycoumarin,<br>
Cascade Blue.TM., Alexa Flur 350, Alexa Flur 430, Alexa Flur 488, Alexa Flur<br>
532, Alexa Flur 546, Alexa Flur 568, Alexa Flur 594, Alexa Flur 633, Alexa Flur<br><br>
647, Alexa Flur 660, Alexa Flur 680, AMCA-X, dialkylaminocoumarin, Pacific<br>
Blue, Marina Blue, BODIPY 493/503, BODIPY Fl-X, DTAF, Oregon Green 500,<br>
Dansyl-X, 6-FAM, Oregon Green 488, Oregon Green 514, Rhodamine Green-X,<br>
Rhodol Green, Calcein, Eosin, ethidium bromide, NBD, TET, 2', 4', 5', 7<br>
tetrabromosulfonefluorescien, BODIPY-R6G, BODIPY-F1BR2, BODIPY<br>
530/550, HEX, BODIPY 558/568, BODIPY-TMR-X., PyMPO, BODIPY<br>
564/570, TAMRA, BODIPY 576/589, Cy3, Rhodamine Red-x, BODIPY<br>
581/591, carboxyXrhodamine, Texas Red-X, BODIPY-TR-X., Cy5,<br>
SpectrumAqua, SpectrumGreen #1, SpectrumGreen #2, SpectrumOrange,<br>
SpectrumRed, or naphthofluorescein.<br>
[0170] In another embodiment, the "fill in" reaction can be performed with<br>
fluorescently labeled dNTPs, wherein the nucleotides are labeled with different<br>
fluorescent groups. The incorporated nucleotides can be detected by any suitable<br>
method including but not limited to Fluorescence Resonance Energy Transfer<br>
(FRET).<br>
[0171] In another embodiment, a mixture of both labeled ddNTPs and<br>
unlabeled dNTPs can be used for filling in the recessed 3' end of the DNA<br>
sequence containing the SNP or locus of interest. Preferably, the 5' overhang<br>
consists of more than one base, including but not limited to 2,3,4,5,6 or more<br>
than 6 bases. For example, if the 5' overhang consists of the sequence "XGAA,"<br>
wherein X is the locus of interest, e.g. SNP, then filling in with a mixture of<br>
labeled ddNTPs and unlabeled dNTPs will produce several different DNA<br>
fragments. If a labeled ddNTP is incorporated at position "X," the reaction will<br>
terminate and a single labeled base will be incorporated. If however, an unlabeled<br>
dNTP is incorporated, the polymerase continues to incorporate other bases until a<br>
labeled ddNTP is incorporated. If the first two nucleotides incorporated are<br>
dNTPs, and the third is a ddNTP, the 3' recessed end will be extended by three<br>
bases. This DNA fragment can be separated from the other DNA fragments that<br>
were extended by 1,2, or 4 bases by size. A mixture of labeled ddNTPs and<br>
unlabeled dNTPs will allow all bases of the overhang to be filled in, and provides<br><br>
additional sequence information about the locus of interest, e.g. SNP (see FIGS.<br>
7E and 9D).<br>
[0172] After incorporation of the labeled nucleotide, the amplified DNA can<br>
be digested with a restriction enzyme that recognizes the sequence provided by<br>
the first primer. For example, in FIG II, the amplified DNA is digested with a<br>
restriction enzyme that binds to region "a," which releases the DNA fragment<br>
containing the incorporated nucleotide from the streptavidin matrix.<br>
[0173] Alternatively, one primer of each primer pair for each locus of interest<br>
can be attached to a solid support matrix including but not limited to a well of a<br>
microtiter plate. For example, streptavidin-coated microtiter plates can be used<br>
for the amplification reaction with a primer pair, wherein one primer is<br>
biotinylated. First, biotinylated primers are bound to the streptavidin-coated<br>
microtiter plates. Then, the plates are used as the reaction vessel for PCR<br>
amplification of the loci of interest. After the amplification reaction is complete,<br>
the excess primers, salts, and template DNA can be removed by washing. The<br>
amplified DNA remains attached to the microtiter plate. The amplified DNA can<br>
be digested with a restriction enzyme that recognizes a sequence on the second<br>
primer and generates a 5' overhang, which contains the locus of interest. The<br>
digested fragments can be removed by washing. After digestion, the SNP site or<br>
locus of interest is exposed in the 5' overhang. The recessed 3' end is filled in<br>
with a labeled nucleotide, including but not limited to, fluorescent ddNTP in the<br>
presence of a polymerase. The labeled DNA can be released into the supernatant<br>
in the microtiter plate by digesting with a restriction enzyme that recognizes a<br>
sequence in the 5' region of the first primer.<br>
Analysis of the locus of interest<br>
[0174] The labeled loci of interest can be analyzed by a variety of methods<br>
including but not limited to fluorescence detection, DNA sequencing gel, capillary<br>
electrophoresis on an automated DNA sequencing machine, microchannel<br>
electrophoresis, and other methods of sequencing, mass spectrometry, time of<br>
flight mass spectrometry, quadrupole mass spectrometry, magnetic sector mass<br><br>
spectrometry, electric sector mass spectrometry infrared spectrometry, ultraviolet<br>
spectrometry, palentiostatic amperometry or by DNA hybridization techniques<br>
including Southern Blots, Slot Blots, Dot Blots, and DNA microarrays, wherein<br>
DNA fragments would be useful as both "probes" and "targets," ELISA,<br>
fluorimetry, and Fluorescence Resonance Energy Transfer (FRET).<br>
[0175] The loci of interest can be analyzed using gel electrophoresis followed<br>
by fluorescence detection of the incorporated nucleotide. Another method to<br>
analyze or read the loci of interest is to use a fluorescent plate reader or<br>
fluorirneter directly on the 96-well streptavidin coated plates. The plate can be<br>
placed onto a fluorescent plate reader or scanner such as the Pharmacia 9200<br>
Typhoon to read each locus of interest.<br>
[0176] Alternatively, the PCR products of the loci of interest can be pooled<br>
and after "filling in," (FIG. 10) the products can be separated by size, using any<br>
method appropriate for the same, and then analyzed using a variety of techniques<br>
including but not limited to fluorescence detection, DNA sequencing gel, capillary<br>
electrophoresis on an automated DNA sequencing machine, microchannel<br>
electrophoresis, other methods of sequencing, DNA hybridization techniques<br>
including Southern Blots, Slot Blots, Dot Blots, and DNA microarrays, mass<br>
spectrometry, time of flight mass spectrometry, quadrupole mass spectrometry,<br>
magnetic sector mass spectrometry, electric sector mass spectrometry infrared<br>
spectrometry, ultraviolet spectrometry, palentiostatic amperometry. For example,<br>
polyacrylamide gel electrophoresis can be used to separate DNA by size and the<br>
gel can be scanned to determine the color of fluorescence in each band (using e.g.<br>
ABI377 DNA sequencing machine or a Pharmacia Typhoon 9200).<br>
[0177] In another embodiment, one nucleotide can be used to determine the<br>
sequence of multiple alleles of a gene. A nucleotide that terminates the elongation<br>
reaction can be used to determine the sequence of multiple alleles of a gene. At<br>
one allele, the terminating nucleotide is complementary to the locus of interest in<br>
the 5' overhang of said allele. The nucleotide is incorporated and terminates the<br>
reaction. At a different allele, the terminating nucleotide is not complementary to<br><br>
the locus of interest, which allows a non-terminating nucleotide to be incorporated<br>
at the locus of interest of the different allele. However, the terminating nucleotide<br>
is complementary to a nucleotide downstream from the locus of interest in the 5'<br>
overhang of said different allele. The sequence of the alleles can be determined<br>
by analyzing the patterns of incorporation of the terminating nucleotide. The<br>
terminating nucleotide can be labeled or unlabeled.<br>
[0178] In a another embodiment, the terminating nucleotide is a nucleotide<br>
that terminates or hinders the elongation reaction including but not limited to a<br>
dideoxynucleotide, a dideoxynucleotide derivative, a dideoxynucleotide analog, a<br>
dideoxynucleotide homolog, a dideoxynucleotide with a sulfur chemical group, a<br>
deoxynucleotide, a deoxynucleotide derivative, a deoxynucleotide homolog, a<br>
deoxynucleotide analog, and a deoxynucleotide with a sulfur chemical group,<br>
arabinoside triphosphate, an arabinoside triphosphate analog, a arabinoside<br>
triphosphate homolog, or an arabinoside derivative.<br>
[0179] In another embodiment, a terminating nucleotide labeled wuh one<br>
signal generating moiety tag, including but not limited to a fluorescent dye, can be<br>
used to determine the sequence of the alleles of a locus of interest. The use of a<br>
single nucleotide labeled with one signal generating moiety tag eliminates any<br>
difficulties that can arise when using different fluorescent moieties. In addition,<br>
i<br>
using one nucleotide labeled with one signal generating moiety tag to determine<br>
the sequence of alleles of a locus of interest reduces the number of reactions, and<br>
eliminates pipetting errors.<br>
[0180] For example, if the second primer contains the restriction enzyme<br>
recognition site for BsmFI, digestion will generate a 5' overhang of 4 bases. The<br>
second primer can be designed such that the locus of interest is located in the first<br>
position of the overhang. A representative overhang is depicted below, where R<br>
represents the locus of interest:<br>
5'CAC<br>
3'GTG	R T G G<br>
Overhang position	12	3	4<br><br>
[0181] One nucleotide with one signal generating moiety tag can be used to<br>
determine whether the variable site is homozygous or heterozygous. For example,<br>
if the variable site is adenine (A) or guanine (G), then either adenine or guanine<br>
can be used to determine the sequence of the alleles of the locus of interest,<br>
provided that there is an adenine or guanine in the overhang at position 2,3, or 4.<br>
[0182] For example, if the nucleotide in position 2 of the overhang is<br>
thymidine, which is complementary to adenine, then labeled ddATP, unlabeled<br>
dCTP, dGTP, and dTTP can be used to determine the sequence of the alleles of<br>
the locus of interest. The ddATP can be labeled with any signal generating<br>
moiety including but not limited to a fluorescent dye. If the template DNA is<br>
homozygous for adenine, then labeled ddATP* will be incorporated at position 1<br>
complementary to the overhang at the alleles, and no nucleotide incorporation will<br>
be seen at position 2,3 or 4 complementary to the overhang.<br><br>
[0183] One signal will be seen corresponding to incorporation of labeled<br>
ddATP at position 1 complementary to the overhang, which indicates that the<br>
individual is homozygous for adenine at this position. This method of labeling<br>
eliminates any difficulties that may arise from using different dyes that have<br>
different quantum coefficients.<br>
Homozygous guanine:<br>
[0184] If the template DNA is homozygous for guanine, then no ddATP will<br>
be incorporated at position 1 complementary to the overhang, but ddATP will be<br>
incorporated at the first available position, which in this case is position 2<br><br>
complementary to the overhang. For example, if the second position in the<br>
overhang corresponds to a thymidine, then:<br><br>
[0185] One signal will be seen corresponding to incorporation of ddATP at<br>
position 2 complementary to the overhang, which indicates that the individual is<br>
homozygous for guanine. The molecules that are filled in at position 2<br>
complementary to the overhang will have a different molecular weight than the<br>
molecules filled in at position 1 complementary to the overhang.<br>
[0186] Heterozygous condition:<br><br>
[0187] Two signals will be seen; the first signal corresponds to the ddATP<br>
filled in at position one complementary to the overhang and the second signal<br>
corresponds to the ddATP filled in at position 2 complementary to the overhang.<br>
The two signals can be separated based on molecular weight; allele 1 and allele 2<br>
will be separated by a single base pair, which allows easy detection and<br>
quantitation of the signals. Molecules filled in at position one can be<br><br>
distinguished from molecules filled in at position two using any method that<br>
discriminates based on molecular weight including but not limited to gel<br>
electrophoresis, capillary gel electrophoresis, DNA sequencing, and mass<br>
spectrometry. It is not necessary that the nucleotide be labeled with a chemical<br>
moiety; the DNA molecules corresponding to the different alleles can be<br>
separated based on molecular weight.<br>
[0188] If position 2 of the overhang is not complementary to adenine, it is<br>
possible that positions 3 or 4 may be complementary to adenine. For example,<br>
position 3 of the overhang may be complementary to the nucleotide adenine, in<br>
which case labeled ddATP may be used to determine the sequence of both alleles.<br>
[0189] Homozygous for adenine:<br>
Allele 1	5' CCC	A*<br>
3'GGG	T G	T	G<br>
Overhang position	12	3	4<br>
Allele 2	5'CCC	A*<br>
3'GGG	T . G	T	G<br>
Overhang position	12	3	4<br>
[0190] Homozygous for guanine:<br>
Allele 1	5'CCC	G C	A*<br>
3'GGG	C G	T G<br>
Overhang position	12	3	4<br>
Allele 2	5'CCC	G C	A*<br>
3'GGG	C G	T G<br>
Overhang position	12	3	4<br>
[0191] Heterozygous:<br>
Allele 1	5' CCC	A*<br>
3'GGG	T G	T G<br>
Overhang position	12	3	4<br><br>
Allele 2	5'CCC G C A*<br>
3'GGG C G T G<br>
Overhang position	12	3	4<br>
[0192] Two signals will be seen; the first signal corresponds to the ddATP<br>
filled in at position 1 complementary to the overhang and the second signal<br>
corresponds to the ddATP filled in at position 3 complementary to the overhang.<br>
The two signals can be separated based on molecular weight; allele 1 and allele 2<br>
will be separated by two bases, which can be detected using any method that<br>
discriminates based on molecular weight.<br>
[0193] Alternatively, if positions 2 and 3 are not complementary to adenine<br>
{i.e. positions 2 and 3 of the overhang correspond to guanine, cytosine, or<br>
adenine) but position 4 is complementary to adenine, labeled ddATP can be used<br>
to determine the sequence of both alleles.<br>
[0194] Homozygous for adenine:<br>
Allele 1	5' CCC A*<br>
3'GGG T G G T<br>
Overhang position	12	3	4<br>
Allele 2	5' CCC A*<br>
3'GGG T	G G T<br>
Overhang position	12	3	4<br>
[0195] One signal will be seen that corresponds to the molecular weight of<br>
molecules filled in with ddATP at position one complementary to the overhang,<br>
which indicates that the individual is homozygous for adenine at the variable site.<br>
[0196] Homozygous for guanine:<br>
Allele 1	5'CCC G C C A*<br>
3'GGG C G G T<br>
Overhang position	12	3	4<br>
Allele 2	5'CCC G C C A*<br>
3'GGG C G G T<br><br>
Overhang position	12	3	4<br>
[0197] One signal will be seen that corresponds to the molecular weight of<br>
molecules filled in at position 4 complementary to the overhang, which indicates<br>
that the individual is homozygous for guanine.<br>
[0198] Heterozygous:<br>
Allele 1	5' CCC A*<br>
3'GGG T G G T<br>
Overhang position	12	3	4<br>
Allele 2	5'CCC G C C A*<br>
3'GGG C G G T<br>
Overhang position	12	3	4<br>
[0199] Two signals will be seen; the first signal corresponds to the ddATP<br>
filled in at position one complementary to the overhang and the second signal<br>
corresponds to the ddATP filled in at position 4 complementary to the overhang.<br>
The two signals can be separated based on molecular weight; allele 1 and allele 2<br>
will be separated by three bases, which allows detection and quantitation of the<br>
signals. The molecules filled in at position 1 and those filled in at position 4 can<br>
be distinguished based on molecular weight.<br>
[0200] As discussed above, if the variable site contains either adenine or<br>
guanine, either labeled adenine or labeled guanine can be used to determine the<br>
sequence of both alleles. If positions 2,3, or 4 of the overhang are not<br>
complementary to adenine but one of the positions is complementary to a guanine,<br>
then labeled ddGTP can be used to determine whether the template DNA is<br>
homozygous or heterozygous for adenine or guanine. For example, if position 3<br>
in the overhang corresponds to a cytosine then the following signals will be<br>
expected if the template DNA is homozygous for guanine, homozygous for<br>
adenine, or heterozygous:<br>
[0201] Homozygous for guanine:<br>
Allele 1	5' CCC G*<br>
3'GGG C T C T<br><br>
Overhang position	12	3	4<br>
Allele 2	5' CCC G*<br>
3'GGG C T C T<br>
Overhang position	12	3	4<br>
[0202] One signal will be seen that corresponds to the molecular weight of<br>
molecules filled in with ddGTP at position one complementary to the overhang,<br>
which indicates that the individual is homozygous for guanine.<br>
[0203] Homozygous for adenine:<br>
Allele 1	5'CCC A A G*<br>
3'GGG T T	C T<br>
Overhang position	12	3	4<br>
Allele 2	5'CCC A A G*<br>
3'GGG T T	C T<br>
Overhang position	12	3	4<br>
[0204] One signal will be seen that corresponds to the molecular weight of<br>
molecules filled in at position 3 complementary to the overhang, which indicates<br>
that the individual is homozygous for adenine at the variable site.<br>
[0205] Heterozygous:<br>
Allele 1	5' CCC G*<br>
3'GGG C T C T<br>
Overhang position	12	3	4<br>
Allele 2	5'CCC A A G*<br>
3'GGG T T	C T<br>
Overhang position	12	3	4<br>
[0206] Two signals will be seen; the first signal corresponds to the ddGTP<br>
filled in at position one complementary to the overhang and the second signal<br>
corresponds to the ddGTP filled in at position 3 complementary to the overhang.<br>
The two signals can be separated based on molecular weight; allele 1 and allele 2<br><br>
will be separated by two bases, which allows easy detection and quantitation of<br>
the signals.<br>
[0207] Some type IIS restriction enzymes also display alternative cutting as<br>
discussed above. For example, BsmFI will cut at 10/14 and 11/15 from the<br>
recognition site. However, the cutting patterns are not mutually exclusive; if the<br>
11/15 cutting pattern is seen at a particular sequence, 10/14 cutting is also seen. If<br>
the restriction enzyme BsmF I cuts at 10/14 from the recognition site, the 5'<br>
overhang will be X1X2X3X4. If BsmF I cuts 11/15 from the recognition site, the<br>
5' overhang will be X0X1X2X3. If position Xo of the overhang is complementary<br>
to the labeled nucleotide, the labeled nucleotide will be incorporated at position<br>
X0 and provides an additional level of quality assurance. It provides additional<br>
sequence information.<br>
[0208] For example, if the variable site is adenine or guanine, and position 3<br>
in the overhang is complementary to adenine, labeled ddATP can be used to<br>
determine the genotype at the variable site. If position 0 of the 11/15 overhang<br>
contains the nucleotide complementary to adenine, ddATP will be filled in and an<br>
additional signal will be seen.<br>
[0209] Heterozygous:<br>
10/14 Allele 1	5'CCA	A*<br>
3'GGT	T G T G<br>
Overhang position	12	3	4<br>
10/14 Allele 2	5'CCA	G C A*<br>
3'GGT	C G T G<br>
Overhang position	12	3	4<br>
11/15 Allele 1	5'CC	A*<br>
3'GG	T T G T<br>
Overhang position	0	12	3<br>
11/15 Allele 2	5'CC	A*<br>
3'GG	T C G T<br><br>
Overhang position	0	12	3<br>
[0210] Three signals are seen; one corresponding to the ddATP incorporated<br>
at position 0 complementary to the overhang, one corresponding to the ddATP<br>
incorporated at position 1 complementary to the overhang, and one corresponding<br>
to the ddATP incorporated at position 3 complementary to the overhang. The<br>
molecules filled in at position 0,1, and 3 complementary to the overhang differ in<br>
molecular weight and can be separated using any technique that discriminates<br>
based on molecular weight including but not limited to gel electrophoresis, and<br>
mass spectrometry.<br>
[0211] For quantitating the ratio of one allele to another allele or when<br>
determining the relative amount of a mutant DNA sequence in the presence of<br>
wild type DNA sequence, an accurate and highly sensitive method of detection<br>
must be used. The alternate cutting displayed by type IIS restriction enzymes may<br>
increase the difficulty of determining ratios of one allele to another allele because<br>
the restriction enzyme may not display the alternate cutting (11/15) pattern on the<br>
two alleles equally. For example, allele 1 may be cut at 10/14 80% of the time,<br>
and 11/15 20% of the time. However, because the two alleles may differ in<br>
sequence, allele 2 may be cut at 10/14 90% of the time, and 11/15 20% of the<br>
time.<br>
[0212] For purposes of quantitation, the alternate cutting problem can be<br>
eliminated when the nucleotide at position 0 of the overhang is not<br>
complementary to the labeled nucleotide. For example, if the variable site<br>
corresponds to adenine or guanine, and position 3 of the overhang is<br>
complementary to adenine (i.e., a thymidine is located at position 3 of the<br>
overhang), labeled ddATP can be used to deterrnine the genotype of the variable<br>
site. If position 0 of the overhang generated by the 11/15 cutting properties is not<br>
complementary to adenine, (i.e., position 0 of me overhang corresponds to<br>
guanine, cytosine, or adenine) no additional signal will be seen from the<br>
fragments that were cut 11/15 from the recognition site. Position 0<br>
complementary to the overhang can be filled in with unlabeled nucleotide,<br><br>
eliminating any complexity seen from the alternate cutting pattern of restriction<br>
enzymes. This method provides a highly accurate method for quantitating the<br>
ratio of a variable site including but not limited to a mutation, or a single<br>
nucleotide polymorphism.<br>
[0213] For instance, if SNP X can be adenine or guanine, this method of<br>
labeling allows quantitation of the alleles that correspond to adenine and the<br>
alleles that correspond to guanine, without determining if the restriction enzyme<br>
displays any differences between the alleles with regard to alternate cutting<br>
patterns.<br>
[0214] Heterozygous:<br>
10/14 Allele 1	5' CCG	A*<br>
3'GGC	T G T G<br>
Overhang position	12	3	4<br>
10/14 Allele 2	5'CCG	G C	A*<br>
3'GGC	C G T G<br>
Overhang position	12	3	4<br>
[0215] The overhang generated by the alternate cutting properties of BsmF I<br>
is depicted below:<br>
11/15 Allele 1	5'CC<br>
3'GG	C T G T<br>
Overhang position	0	12	3<br>
11/15 Allele 2	5'CC<br>
3'GG	C C G T<br>
Overhang position	0	12	3<br>
[0216] After filling in with labeled ddATP and unlabeled dGTP, dCTP, dTTP,<br>
the following molecules would be generated:<br>
11/15 Allele 1	5'CC	G A*<br><br>
3'GG	C T G T<br>
Overhang position	0	12 3<br>
11/15 Allele 2	5'CC	G G C A*<br>
3'GG	C C G T<br>
Overhang position	0	12 3<br>
[0217] Two signals are seen; one corresponding to the molecules filled in with<br>
ddATP at position one complementary to the overhang and one corresponding to<br>
the molecules filled in with ddATP at position 3 complementary to the overhang.<br>
Position 0 of the 11/15 overhang is filled in with unlabeled nucleotide, which<br>
eliminates any difficulty in quantitating a ratio for the nucleotide at the variable<br>
site on allele 1 and the nucleotide at the variable site on allele 2.<br>
[0218] Any nucleotide can be used including adenine, adenine derivatives,<br>
adenine homologues, guanine, guanine derivatives, guanine homologues,<br>
cytosine, cytosine derivatives, cytosine homologues, thymidine, thymidine<br>
derivatives, or thymidine homologues, or any combinations of adenine, adenine<br>
derivatives, adenine homologues, guanine, guanine derivatives, guanine<br>
homologues, cytosine, cytosine derivatives, cytosine homologues, thymidine,<br>
thymidine derivatives, or thymidine homologues.<br>
[0219] The nucleotide can be labeled with any chemical group or moiety,<br>
including but not limited to radioactive molecules, fluorescent molecules,<br>
antibodies, antibody fragments, haptens, carbohydrates, biotin, derivatives of<br>
biotin, phosphorescent moieties, luminescent moieties, electrochemiluminescent<br>
moieties., chromatic moieties, and moieties having a detectable electron spin<br>
resonance, electrical capacitance, dielectric constant or electrical conductivity.<br>
The nucleotide can be labeled with one or more than one type of chemical group<br>
or moiety.<br>
[0220] In another embodiment, labeled and unlabeled nucleotides can be used.<br>
Any combination of deoxynucleotides and dideoxynucleotides can be used<br>
including but not limited to labeled dideoxynucleotides and labeled<br><br>
deoxynucleotides; labeled dideoxynucleotides and unlabeled deoxynucleotides;<br>
unlabeled dideoxynucleotides and unlabeled deoxynucleotides; and unlabeled<br>
dideoxynucleotides and labeled deoxynucleotides.<br>
[0221] In another embodiment, nucleotides labeled with a chemical moiety<br>
can be used in the PCR reaction. Unlabeled nucleotides then are used to fill-in the<br>
5' overhangs generated after digestion with the restriction enzyme. An unlabeled<br>
terminating nucleotide can be used to in the presence of unlabeled nucleotides to<br>
determine the sequence of the alleles of a locus of interest.<br>
[0222] For example, if labeled dTTP was used in the PCR reaction, the<br>
following 5' overhang would be generated after digestion with BsmF I:<br>
10/14 Allele 1	5'CT*G A<br>
3'GAC	T G T G<br>
Overhang position	12	3	4<br>
10/14 Allele 2	5' CT*G G C A<br>
3'GAC	C	G T	G<br>
Overhang position	12	3	4<br>
[0223] Unlabeled ddATP, unlabeled dCTP, unlabeled dGTP, and unlabeled<br>
dTTP can be used to fill-in the 5' overhang. Two signals will be generated; one<br>
signal corresponds to the DNA molecules filled in with unlabeled ddATP at<br>
position I complementary to the overhang and the second signal corresponds to<br>
DNA molecules filled in with unlabeled ddATP at position 3 complementary to<br>
the overhang. The DNA molecules can be separated based on molecular weight<br>
and can be detected by the fluorescence of the dTTP, which was incorporated<br>
during the PCR reaction.<br>
[0224] The labeled DNA loci of interest sites can be analyzed by a variety of<br>
methods including but not limited to fluorescence detection, DNA sequencing gel,<br>
capillary electrophoresis on an automated DNA sequencing machine,<br>
microchannel electrophoresis, and other methods of sequencing, mass<br>
spectrometry, time of flight mass spectrometry, quadrupole mass spectrometry,<br>
magnetic sector mass spectrometry, electric sector mass spectrometry infrared<br><br>
spectrometry, ultraviolet spectrometry, palentiostatic amperometry or by DNA<br>
hybridization techniques including Southern Blots, Slot Blots, Dot Blots, and<br>
DNA microarrays, wherein DNA fragments would be useful as both "probes" and<br>
"targets," ELISA, fluorimetry, and Fluorescence Resonance Energy Transfer<br>
(FRET).<br>
[0225] This method of labeling is extremely sensitive and allows the detection<br>
of alleles of a locus of interest that are in various ratios including but not limited<br>
to 1:1,1:2,1:3,1:4,1:5,1:6-1:10,1:11-1:20,1:21-1:30,1:31-1:40,1:41-1:50,<br>
1:51-1:60,1:61-1:70, 1:71-1:80,1:81-1:90,1:91:1:100,1:101-1:200,1:250,<br>
1:251-1:300,1:3.01-1:400,1:401-1:500,1:501-1:600,1:601-1:700,1:701-1:800,<br>
1:801-1:900,1:901-1:1000,1:1001-1:2000,1:2001-1:3000,1:3001-1:4000,<br>
1:4001-1:5000,1:5001-1:6000,1:6001-1:7000,1:7001-1:8000,1:8001-1:9000,<br>
1:9001-1:10,000; 1:10,001-1:20,000,1:20,001:1:30,000,1:30,001-1:40,000,<br>
1:40,001-1:50,000, and greater than 1:50,000.<br>
[0226] For example, this method of labeling allows one nucleotide labeled<br>
with one signal generating moiety to be used to determine the sequence of alleles<br>
at a SNP locus, or detect a mutant allele amongst a population of normal alleles,<br>
or detect an allele encoding antibiotic resistance from a bacterial cell amongst<br>
alleles from antibiotic sensitive bacteria, or detect an allele from a drug resistant<br>
virus amongst alleles from drug-sensitive virus, or detect an allele from a non-<br>
pathogenic bacterial strain amongst alleles from a pathogenic bacterial strain.<br>
[0227] As shown above, a single nucleotide can be used to determine the<br>
sequence of the alleles at a particular locus of interest. This method is especially<br>
useful for determining if an individual is homozygous or heterozygous for a<br>
particular mutation or to determine the sequence of the alleles at a particular SNP<br>
site. This method of labeling eliminates any errors caused by the quantum<br>
coefficients of various dyes. It also allows the reaction to proceed in a single<br>
reaction vessel including but not limited to a well of a microliter plate, or a single<br>
eppendorf tube.<br><br>
[0228] This method of labeling is especially useful for the detection of<br>
multiple genetic signals in the same sample. For example, this method is useful<br>
for the detection of fetal DNA in the blood, serum, or plasma of a pregnant<br>
female, which contains both maternal DNA and fetal DNA. The maternal DNA<br>
and fetal DNA may be present in the blood, serum or plasma at ratios such as<br>
97:3; however, the above-described method can be used to detect the fetal DNA.<br>
This method of labeling can be used to detect two, three, or four different genetic<br>
signals in the sample population<br>
[0229] This method of labeling is especially useful for the detection of a<br>
mutant allele that is among a large population of wild type alleles. Furthermore,<br>
this method of labeling allows the detection of a single mutant cell in a large<br>
population of wild type cells. For example, this method of labeling can be used to<br>
detect a single cancerous cell among a large population of normal cells.<br>
Typically, cancerous cells have mutations in the DNA sequence. The mutant<br>
DNA sequence can be identified even if there is a large background of wild type<br>
DNA sequence. This method of labeling can be used to screen, detect, or<br>
diagnosis any type of cancer including but not limited to colon, renal, breast,<br>
bladder, liver, kidney, brain, lung, prostate, and cancers of the blood including<br>
leukemia.<br>
[0230] This labeling method can also be used to detect pathogenic organisms,<br>
including but not limited to bacteria, fungi, viruses, protozoa, and mycobacteria.<br>
It can also be used to discriminate between pathogenic strains of microorganism<br>
and non-pathogenic strains of microorganisms including but not limited to<br>
bacteria, fungi, viruses, protozoa, and mycobacteria.<br>
[0231] For example, there are several strains of Escherichia coli (E. coli), and<br>
most are non-pathogenic. However, several strains, such as E. coli 0157 are<br>
pathogenic. There are genetic differences between non-pathogenic E. coli strains<br>
and pathogenic E. coli. The above described method of labeling can be used to<br>
detect pathogenic microorganisms in a large population of non-pathogenic<br>
organisms, which are sometimes associated with the normal flora of an individual.<br><br>
[0232] In another embodiment, the sequence of the locus of interest can be<br>
determined by detecting the incorporation of a nucleotide that is 3' to the locus of<br>
interest, wherein said nucleotide is a different nucleotide from the possible<br>
nucleotides at the locus of interest. This embodiment is especially useful for the<br>
sequencing and detection of SNPs. The efficiency and rate at which DNA<br>
polymerases incorporate nucleotides varies for each nucleotide.<br>
[0233] According to the data from the Human Genome Project, 99% of all<br>
SNPs are binary. The sequence of the human genome can be used to determine<br>
the nucleotide that is 3' to the SNP of interest. When the nucleotide that is 3' to<br>
the SNP site differs from the possible nucleotides at the SNP site, a nucleotide<br>
that is one or more than one base 3' to the SNP can be used to determine the<br>
identity of the SNP.<br>
[0234] For example, suppose the identity of SNP X on chromosome 13 is to<br>
be determined. The sequence of the human genome indicates that SNP X can<br>
either be adenosine or guanine and that a nucleotide 3' to the locus of interest is a<br>
thymidine. A primer that contains a restriction enzyme recognition site for BsrnF<br>
I, which is designed to be 13 bases from the locus of interest after amplification, is<br>
used to amplify a DNA fragment containing SNP X. Digestion with the<br>
restriction enzyme BsmF I generates a 5' overhang that contains the locus of<br>
interest, which can either be adenosine or guanine. The digestion products can be<br>
split into two "fill in" reactions: one contains dTTP, and the other reaction<br>
contains dCTP. If the locus of interest is homozygous for guanine, only the DNA<br>
molecules that were mixed with dCTP will be filled in. If the locus of interest is<br>
homozygous for adenosine, only the DNA molecules that were mixed with dTTP<br>
will be filled in. If the locus of interest is heterozygous, the DNA molecules that<br>
were mixed with dCTP will be filled in as well as the DNA molecules that were<br>
mixed with dTTP. After washing to remove the excess dNTP, the samples are<br>
filled in with labeled ddATP, which is complementary to the nucleotide<br>
(thymidine) that is 3' to the locus of interest. The DNA molecules that were filled<br>
in by the previous reaction will be filled in with labeled ddATP. If the individual<br><br>
is homozygous for adenosine, the DNA molecules that were mixed with dTTP<br>
subsequently will be filled in with the labeled ddATP. However, the DNA<br>
molecules that were mixed with dCTP, would not have incorporated that<br>
nucleotide, and therefore, could not incorporate the ddATP. Detection of labeled<br>
ddATP only in the molecules that were mixed with dTTP indicates that the<br>
identity of the nucleotide at SNP X on chromosome 13 is adenosine.<br>
[0235] In another embodiment, large scale screening for the presence or<br>
absence of single nucleotide mutations can be performed. One to tens to hundreds<br>
to thousands of loci of interest on a single chromosome or on multiple<br>
chromosomes can be amplified with primers as described above in the "Primer<br>
Design" section. The primers can be designed so that each amplified loci of<br>
interest is of a different size (FIG. 2). The amplified loci of interest that are<br>
predicted, based on the published wild type sequences, to have the same<br>
nucleotide at the locus of interest can be pooled together, bound to a solid support,<br>
including wells of a microtiter plate coated with streptavidin, and digested with<br>
the restriction enzyme that will bind the recognition site on the second primer.<br>
After digestion, the 3' recessed end can be filled in with a mixture of labeled<br>
ddATP, ddTTP, ddGTP, ddCTP, where each nucleotide is labeled with a different<br>
group. After washing to remove the excess nucleotide, the fluorescence spectra<br>
can be detected using a plate reader or fluorimeter directly on the streptavidin<br>
coated plates. If all 50 loci of interest contain the wild type nucleotide, only one<br>
fluorescence spectra will be seen. However, if one or more than one of the 50 loci<br>
of interest contain a mutation, a different nucleotide will be incorporated and<br>
other fluorescence pattern(s) will be seen. The nucleotides can be released from<br>
the solid matrix, and analyzed on a sequencing gel to determine the loci of interest<br>
that contained the mutations. As each of the 50 loci of interest are of different<br>
size, they will separate on a sequencing gel.<br>
[0236] The multiple loci of interest can be of a DNA sample from one<br>
individual representing multiple loci of interest on a single chromosome, multiple<br>
chromosomes, multiple genes, a single gene, or any combination thereof. The<br><br>
multiple loci of interest also can represent the same locus of interest but from<br>
multiple individuals. For example, 50 DNA samples from 50 different individuals<br>
can be pooled and analyzed to determine a particular nucleotide of interest at gene<br>
"X."<br>
[0237] When human data is being analyzed, the known sequence can be a<br>
specific sequence that has been determined from one individual (including e.g. the<br>
individual whose DNA is currently being analyzed), or it can be a consensus<br>
sequence such as that published as part of the human genome.<br>
Kits<br>
[0238] The methods of the invention are most conveniently practiced by<br>
providing the reagents used in the methods in the form of kits. A kit preferably<br>
contains one or more of the following components: written instructions for the<br>
use of the kit, appropriate buffers, salts, DNA extraction detergents, primers,<br>
nucleotides, labeled nucleotides, 5' end modification materials, and if desired,<br>
water of the appropriate purity, confined in separate containers or packages, such<br>
components allowing the user of the kit to extract the appropriate nucleic acid<br>
sample, and analyze the same according to the methods of the invention. The<br>
primers that are provided with the kit will vary, depending upon the purpose of<br>
the kit and the DNA that is desired to be tested using the kit. In preferred<br>
embodiments the kits contain a primer that allows the generation of a recognition<br>
site for a restriction enzyme such that digestion with the enzyme generates in the<br>
DNA fragment generated during the sequencing method, a 5* overhang containing<br>
the locus of interest.<br>
[0239] A kit can also be designed to detect a desired or variety of single<br>
nucleotide polymorphisms, especially those associated with an undesired<br>
condition or disease. For example, one kit can comprise, among other<br>
components, a set or sets of primers to amplify one or more loci of interest<br>
associated with breast cancer. Another kit can comprise, among other<br>
components, a set or sets of primers for genes associated with a predisposition to<br>
develop type I or type II diabetes. Still, another kit can comprise, among other<br><br>
components, a set or sets of primers for genes associated with a predisposition to<br>
develop heart disease. Details of utilities for such kits are provided in the<br>
"Utilities" section below.<br>
Utilities<br>
[0240] The methods of the invention can be used whenever it is desired to<br>
know the sequence of a certain nucleic acid, locus of interest or loci of interest<br>
therein. The method of the invention is especially useful when applied to<br>
genomic DNA. When DNA from an organism-specific or species-specific locus<br>
or loci of interest is amplified, the method of the invention can be used in<br>
genotyping for identification of the source of the DNA, and thus confirm or<br>
provide the identity of the organism or species from which the DNA sample was<br>
derived. The organism can be any nucleic acid containing organism, for example,<br>
virus, bacterium, yeast, plant, animal or human.<br>
[0241] Within any population of organisms, the method of the invention is<br>
useful to identify differences between the sequence of the sample nucleic acid and<br>
that of a known nucleic acid. Such differences can include, for example, allelic<br>
variations, mutations, polymorphisms and especially single nucleotide<br>
polymorphisms.<br>
[0242] In a preferred embodiment, the method of the invention provides a<br>
method for identification of single nucleotide polymorphisms.<br>
[0243] In a preferred embodiment, the method of the invention provides a<br>
method for identification of the presence of a disease, especially a genetic disease<br>
that arises as a result of the presence of a genomic sequence, or other biological<br>
condition that it is desired to identify in an individual for which it is desired to<br>
know the same. The identification of such sequence in the subject based on the<br>
presence of such genomic sequence can be used, for example, to determine if the<br>
subject is a carrier or to assess if the subject is predisposed to developing a certain<br>
genetic trait, condition or disease. The method of the invention is especially<br>
useful in prenatal genetic testing of parents and child. Examples of some of the<br><br><br><br><br><br><br>
[02441 The method of the invention is useful for screening an individual at<br>
multiple loci of interest, such as tens, hundreds, or even thousands of loci of<br>
interest associated with a genetic trait or genetic disease by sequencing the loci of<br>
interest that are associated with the trait or disease state, especially those most<br><br>
frequently associated with such trait or condition. The invention is useful for<br>
analyzing a particular set of diseases including but not limited to heart disease,<br>
cancer, endocrine disorders, immune disorders, neurological disorders,<br>
musculoskeletal disorders, ophthalmologic disorders, genetic abnormalities,<br>
trisomies, monosomies, transversions, translocations, skin disorders, and familial<br>
diseases.<br>
[0245] The method of the invention can be used to genotype microorganisms<br>
so as to rapidly identify the presence of a specific microorganism in a substance,<br>
for example, a food substance. In that regard, the method of the invention<br>
provides a rapid way to analyze food, liquids or air samples for the presence of an<br>
undesired biological contamination, for example, microbiological, fungal or<br>
animal waste material. The invention is useful for detecting a variety of<br>
organisms, including but not limited to bacteria, viruses, fungi, protozoa, molds,<br>
yeasts, plants, animals, and archaebacteria. The invention is useful for detecting<br>
organisms collected from a variety of sources including but not limited to water,<br>
air, hotels, conference rooms, swimming pools, bathrooms, aircraft, spacecraft,<br>
trains, buses, cars, offices, homes, businesses, churches, parks, beaches, athletic<br>
facilities, amusement parks, theaters, and any other facility that is a meeting place<br>
for the public.<br>
[0246] The method of the invention can be used to test for the presence of<br>
many types of bacteria or viruses in blood cultures from human or animal blood<br>
samples.<br>
[0247] The method of the invention can also be used to confirm or identify the<br>
presence of a desired or undesired yeast strain, or certain traits thereof, in<br>
fermentation products, e.g. wine, beer, and other alcohols or to identify the<br>
absence thereof.<br>
[0248] The method of the invention can also be used to confirm or identify the<br>
relationship of a DNA of unknown sequence to a,DNA of known origin or<br>
sequence, for example, for use in criminology, forensic science, maternity or<br>
paternity testing, archeological analysis, and the like.<br><br>
[0249] The method the invention can also be used to determine the genotypes<br>
of plants, trees and bushes, and hybrid plants, trees and bushes, including plants,<br>
trees and bushes that produce fruits and vegetables and other crops, including but<br>
not limited to wheat, barley, corn, tobacco, alfalfa, apples, apricots, bananas,<br>
oranges, pears, nectarines, figs, dates, raisins, plums, peaches, apricots,<br>
blueberries, strawberries, cranberries, berries, cherries, kiwis, limes, lemons,<br>
melons, pineapples, plantains, guavas, prunes, passion fruit, tangerines, grapefruit,<br>
grapes, watermelon, cantaloupe, honeydew melons, pomegranates, persimmons,<br>
nuts, artichokes, bean sprouts, beets, cardoon, chayote, endive, leeks, okra, green<br>
onions, scallions, shallots, parsnips, sweet potatoes, yams, asparagus, avocados,<br>
kohlrabi, rutabaga, eggplant, squash, turnips, pumpkins, tomatoes, potatoes,<br>
cucumbers, carrots, cabbage, celery, broccoli, cauliflower, radishes, peppers,<br>
spinach, mushrooms, zucchini, onions, peas, beans, and other legumes.<br>
[0250] Especially, the method of the invention is useful to screen a mixture of<br>
nucleic acid samples that contain many different loci of interest and/or a mixture<br>
of nucleic acid samples from different sources that are to be analyzed for a locus<br>
of interest. Examples of large scale screening include taking samples of nucleic<br>
acid from herds of farm animals, or crops of food plants such as, for example,<br>
corn or wheat, pooling the same, and then later analyzing the pooled samples for<br>
the presence of an undesired genetic marker, with individual samples only being<br>
analyzed at a later date if the pooled sample indicates the presence of such<br>
undesired genetic sequence. An example of an undesired genetic sequence would<br>
be the detection of viral or bacterial nucleic acid sequence in the nucleic acid<br>
samples taken from the farm animals, for example, mycobacterium or hoof and<br>
mouth disease virus sequences or fungal or bacterial pathogen of plants.<br>
[0251] Another example where pools of nucleic acid can be used is to test for<br>
the presence of a pathogen or gene mutation in samples from one or more tissues<br>
from an animal or human subject, living or dead, especially a subject who can be<br>
in need of treatment if the pathogen or mutation is detected. For example,<br>
numerous samples can be taken from an animal or human subject to be screened<br><br>
for the presence of a pathogen or otherwise undesired genetic mutation, the loci of<br>
interest from each biological sample amplified individually, and then samples of<br>
the amplified DNA combined for the restriction digestion, "filling in," and<br>
detection. This would be useful as an initial screening for the assay of the<br>
presence or absence of nucleic acid sequences that would be diagnostic of the<br>
presence of a pathogen or mutation. Then, if the undesired nucleic acid sequence<br>
of the pathogen or mutation was detected, the individual samples could be<br>
separately analyzed to determine the distribution of the undesired sequence. Such<br>
an analysis is especially cost effective when there are large numbers of samples to<br>
be assayed. Samples of pathogens include the mycobacteria, especially those that<br>
cause tuberculosis or paratuberculosis, bacteria, especially bacterial pathogens<br>
used in biological warfare, including Bacillus anthracis, and virulent bacteria<br>
capable of causing food poisoning, viruses, especially the influenza and AIDS<br>
virus, and mutations known to be associated with malignant cells. Such an<br>
analysis would also be advantageous for the large scale screening of food products<br>
for pathogenic bacteria.<br>
[0252] Conversely, the method of the invention can be used to detect the<br>
presence and distribution of a desired genetic sequence at various locations in a<br>
plant, animal or human subject, or in a population of subjects, e.g. by screening of<br>
a combined sample followed by screening of individual samples, as necessary.<br>
[0253] The method of the invention is useful for analyzing genetic variations<br>
of an individual that have an effect on drug metabolism, drug interactions, and the<br>
responsiveness to a drug or to multiple drugs. The method of the invention is<br>
especially useful in pharmacogenomics.<br>
[0254] Having now generally described the invention, the same will become<br>
better understood by reference to certain specific examples which are included<br>
herein for purposes of illustration only and are not intended to be limiting unless<br>
other wise specified.<br><br>
EXAMPLES<br>
[0255] The following examples are illustrative only and are not intended to<br>
limit the scope of the invention as defined by the claims.<br>
EXAMPLE 1<br>
[0256] DNA sequences were amplified by PCR, wherein the annealing step in<br>
cycle 1 was performed at a specified temperature, and then increased in cycle 2,<br>
and further increased in cycle 3 for the purpose of reducing non-specific<br>
amplification. The TM1 of cycle 1 of PCR was determined by calculating the<br>
melting temperature of the 3' region, which anneals to the template DNA, of the<br>
second primer. For example, in FIG. 1B, the TM1 can be about the melting<br>
temperature of region "c." The annealing temperature was raised in cycle 2, to<br>
TM2, which was about the melting temperature of the 3' region, which anneals to<br>
the template DNA, of the first primer. For example, in FIG. 1C, the annealing<br>
temperature (TM2) corresponds to the melting temperature of region "b"'. In<br>
cycle 3, the annealing temperature was raised to TM3, which was about the<br>
melting temperature of the entire sequence of the second primer For example, in<br>
FIG. 1D, the annealing temperature (TM3) corresponds to the melting temperature<br>
of region "c" + region "d". The remaining cycles of amplification were performed<br>
atTM3.<br>
Preparation of Template DNA<br>
[0257] The template DNA was prepared from a 5 ml sample of blood obtained<br>
by venipuncture from a human volunteer with informed consent. Blood was<br>
collected from 36 volunteers. Template DNA was isolated from each blood<br>
sample using QIAamp DNA Blood Midi Kit supplied by QIAGEN (Catalog<br>
number 51183). Following isolation, the template DNA from each of the 36<br>
volunteers was pooled for further analysis.<br>
Design of Primers<br>
[0258] The following four single nucleotide polymorphisms were analyzed:<br>
SNP HC21S00340, identification number as assigned by Human Chromosome 21<br><br>
cSNP Database, (FIG. 3, lane 1) located on chromosome 21; SNP TSC 0095512<br>
(FIG. 3, lane 2) located on chromosome 1, SNP TSC 0214366 (FIG. 3, lane 3)<br>
located on chromosome 1; and SNP TSC 0087315 (FIG. 3, lane 4) located on<br>
chromosome 1. The SNP Consortium Ltd database can be accessed at<br>
http://snp.cshl.org/, website address effective as of February 14,2002.<br>
[0259] SNP HC21S00340 was amplified using the following primers:<br>
First primer:<br>
5' TAGAATAGCACTGAATTCAGGAATACAATCATTGTCAC 3' (SEQ ID<br>
NO:9)<br>
Second primer:<br>
5' ATCACGATAAACGGCCAAACTCAGGTTA 3' (SEQ ID NO: 10)<br>
[0260] SNP TSC0095512 was amplified using the following primers:<br>
First primer:<br>
5' AAGTTTAGATCAGAATTCGTGAAAGCAGAAGTTGTCTG 3' (SEQ ID<br>
NO: 11)<br>
Second primer:<br>
5' TCTCCAACTAACGGCTCATCGAGTAAAG 3' (SEQ ID NO:12)<br>
[0261] SNP TSC0214366 was amplified using the following primers:<br>
First primer:<br>
5'ATGACTAGCTATGAATTCGTTCAAGGTAGAAAATGGAA 3' (SEQ ID<br>
NO: 13)<br>
Second primer:<br>
5' GAGAATTAGAACGGCCCAAATCCCACTC 3' (SEQ ID NO: 14)<br>
[0262] SNP TSC 0087315 was amplified using the following primers:<br>
First primer:<br>
5' TTACAATGCATGAATTCATCTTGGTCTCTCAAAGTGC 3' (SEQ ID<br>
NO: 15)<br>
Second primer:<br>
5' TGGACCATAAACGGCCAAAAACTGTAAG 3' (SEQ ID NO: 16)<br><br>
[0263] All primers were designed such that the 3' region was complementary<br>
to either the upstream or downstream sequence flanking each locus of interest and<br>
the 5' region contained a restriction enzyme recognition site. The first primer<br>
contained a biotin tag at the 5' end and a recognition site for the restriction<br>
enzyme EcoRI. The second primer contained the recognition site for the<br>
restriction enzyme BceA I.<br>
PCR Reaction<br>
[0264] All four loci of interest were amplified from the template genomic<br>
DNA using PCR (U.S. Patent Nos. 4,683,195 and 4,683,202). The components of<br>
the PCR reaction were as follows: 40 ng of template DNA, 5 μM first primer, 5<br>
μM second primer, 1X HotStarTaq Master Mix as obtained from QIAGEN<br>
(Catalog No. 203443). The HotStarTaq Master Mix contained DNA polymerase,<br>
PCR buffer, 200 μM of each dNTP, and 1.5 mM MgCl2.<br>
[0265] Amplification of each template DNA that contained the SNP of<br>
interest was performed using three different series of annealing temperatures,<br>
herein referred to as low stringency annealing temperature, medium stringency<br>
annealing temperature, and high stringency annealing temperature. Regardless of<br>
the annealing temperature protocol, each PCR reaction consisted of 40 cycles of<br>
amplification. PCR reactions were performed using the HotStarTaq Master Mix<br>
Kit supplied by QIAGEN. As instructed by the manufacturer, the reactions were<br>
incubated at 95°C for 15 min. prior to the first cycle of PCR. The denaturation<br>
step after each extension step was performed at 95°C for 30 sec. The annealing<br>
reaction was performed at a temperature that permitted efficient extension without<br>
any increase in temperature.<br>
[0266] The low stringency annealing reaction comprised three different<br>
annealing temperatures in each of the first three cycles. The annealing<br>
temperature for the first cycle was 37°C for 30 sec; the annealing temperature for<br>
the second cycle was 57°C for 30 sec; the annealing temperature for the third<br>
cycle was 64°C for 30 sec. Annealing was performed at 64°C for subsequent<br>
cycles until completion.<br><br>
[0267] As shown in the photograph of the gel (FIG. 3A), multiple bands were<br>
observed after amplification of the DNA template containing SNP TSC 0087315<br>
(lane 4). Amplification of the DNA templates containing SNP HC21S00340 (lane<br>
1), SNP TSC0095512 (lane 2), and SNP TSC0214366 (lane 3) generated a single<br>
band of high intensity and one band of faint intensity, which was of higher<br>
molecular weight. When the low annealing temperature conditions were used, the<br>
correct size product was generated and this was the predominant product in each<br>
reaction.<br>
[0268] The medium stringency annealing reaction comprised three different<br>
annealing temperatures in each of the first three cycles. The annealing<br>
temperature for the first cycle was 40°C for 30 seconds; the annealing temperature<br>
for the second cycle was 60°C for 30 seconds; and the annealing temperature for<br>
the third cycle was 67°C for 30 seconds. Annealing was performed at 67°C for<br>
subsequent cycles until completion. Similar to what was observed under low<br>
stringency annealing conditions, amplification of the DNA template containing<br>
SNP TSC0087315 (FIG. 3B, lane 4) generated multiple bands under conditions of<br>
medium stringency. Amplification of the other three DNA fragments containing<br>
SNPs (lanes 1-3) produced a single band. These results demonstrate that variable<br>
annealing temperatures can be used to cleanly amplify loci of interest from<br>
genomic DNA with a primer that has an annealing length of 13 bases.<br>
[0269] The high stringency annealing reaction was comprised of three<br>
different annealing temperatures in each of the first three cycles. The annealing<br>
temperature of the first cycle was 46°C for 30 seconds; the annealing temperature<br>
of the second cycle was 65°C for 30 seconds; and the annealing temperature for<br>
the third cycle was 72°C for 30 seconds. Annealing was performed at 72°C for<br>
subsequent cycles until completion. As shown in the photograph of the gel (FIG.<br>
3C), amplification of the DNA template containing SNP TSC0087315 (lane 4)<br>
using the high stringency annealing temperatures generated a single band of the<br>
correct molecular weight. By raising the annealing temperatures for each of the<br>
first three cycles, non-specific amplification was eliminated. Amplification of the<br><br>
DNA fragment containing SNP TSC0095512 (lane 2) generated a single band.<br>
DNA fragments containing SNPs HC21S00340 (lane 1), and TSC0214366 (lane<br>
3) failed to amplify at the high stringency annealing temperatures, however, at the<br>
medium stringency annealing temperatures, these DNA fragments containing<br>
SNPs amplified as a single band. These results demonstrate that variable<br>
annealing temperatures can be used to reduce non-specific PCR products, as<br>
demonstrated for the DNA fragment containing SNP TSC0087315 (FIG. 3, lane<br>
4).<br>
EXAMPLE 2<br>
[0270] SNPs on chromosomes 1 (TSC0095512), 13 (TSC0264580), and 21<br>
(HC21S00027) were analyzed. SNP TSC0095512 was analyzed using two<br>
different sets of primers, and SNP HC21S00027 was analyzed using two types of<br>
reactions for the incorporation of nucleotides.<br>
Preparation of Template DNA<br>
[0271] The template DNA was prepared from a 5 ml sample of blood obtained<br>
by venipuncture from a human volunteer with informed consent. Template DNA<br>
was isolated using the QIAmp DNA Blood Midi Kit supplied by QIAGEN<br>
(Catalog number 51183). The template DNA was isolated as per instructions<br>
included in the kit. Following isolation, template DNA from thirty-six human<br>
volunteers were pooled together and cut with the restriction enzyme EcoRI. The<br>
restriction enzyme digestion was performed as per manufacturer's instructions.<br>
Design of Primers<br>
[0272] SNP HC21S00027 was amplified by PCR using the following primer<br>
set:<br>
First primer:<br>
5' ATAACCGTATGCGAATTCTATAATTTTCCTGATAAAGG 3' (SEQ ID<br>
NO: 17)<br>
Second primer:<br>
5' CTTAAATCAGGGGACTAGGTAAACTTCA 3' (SEQ ID NO: 18)<br><br>
[02731 The first primer contained a biotin tag at the extreme 5' end, and the<br>
nucleotide sequence for the restriction enzyme EcoRI. The second primer<br>
contained the nucleotide sequence for the restriction enzyme BsmF I (FIG. 4A).<br>
[0274] Also, SNP HC21S00027 was amplified by PCR using the same first<br>
primer but a different second primer with the following sequence:<br>
Second primer:<br>
5' CTTAAATCAGACGGCTAGGTAAACTTCA 3' (SEQ ID NO: 19)<br>
[027S] This second primer contained the recognition site for the restriction<br>
enzyme BceA I (FIG. 4B).<br>
SNP TSC0095512 was amplified by PCR using the following primers:<br>
First primer:<br>
5' AAGTTTAGATCAGAATTCGTGAAAGCAGAAGTTGTCTG 3' (SEQ ID<br>
NO: 11)<br>
Second primer:<br>
5' TCTCCAACTAGGGACTCATCGAGTAAAG 3' (SEQ ID NO:20)<br>
[0276] The first primer had a biotin tag at the 5' end and contained a<br>
restriction enzyme recognition site for EcoRI. The second primer contained a<br>
restriction enzyme recognition site for BsmF I (FIG. 4C).<br>
[0277] Also, SNP TSC0095512 was amplified using the same first primer and<br>
a different second primer with the following sequence:<br>
Second primer:<br>
5' TCTCCAACTAACGGCTCATCGAGTAAAG 3' (SEQ ID NO: 12)<br>
[0278] This second primer contained the recognition site for the restriction<br>
enzyme BceA I (FIG. 4D).<br>
[0279] SNP TSC0264580, which is located on chromosome 13, was amplified<br>
with the following primers:<br>
First primer:<br>
5' AACGCCGGGCGAGAATTCAGTTTTTCAACTTGCAAGG 3' (SEQ ID<br>
NO:21)<br><br>
Second primer:<br>
5' CTACACATATCTGGGACGTTGGCCATCC 3' (SEQ ID NO:22)<br>
[0280] The first primer contained a biotin tag at the extreme 5' end and had a<br>
restriction enzyme recognition site for EcoRI. The second primer contained a<br>
restriction enzyme recognition site for BsmF I.<br>
PCR Reaction<br>
[0281] All loci of interest were amplified from the template genomic DNA<br>
using the polymerase chain reaction (PCR, U.S. Patent Nos. 4,683,195 and<br>
4,683,202, incorporated herein by reference). In this example, the loci of interest<br>
were amplified in separate reaction tubes but they could also be amplified together<br>
in a single PCR reaction. For increased specificity, a "hot-start" PCR was used.<br>
PCR reactions were performed using the HotStarTaq Master Mix Kit supplied by<br>
QIAGEN (catalog number 203443). The amount of template DNA and primer<br>
per reaction can be optimized for each locus of interest but in this example, 40 ng<br>
of template human genomic DNA and 5 uM of each primer were used. Forty<br>
cycles of PCR were performed. The following PCR conditions were used:<br>
(1)	95°C for 15 minutes and 15 seconds;<br>
(2)	37°C for 30 seconds;<br>
(3)	95°C for 30 seconds;<br>
(4)	57°C for 30 seconds;<br>
(5)	95°C for 30 seconds;<br>
(6)	64°C for 30 seconds;<br>
(7)	95°C for 30 seconds;<br>
(8)	Repeat steps 6 and 7 thirty nine (39) times;<br>
(9)	72°C for 5 minutes.<br>
[0282] In the first cycle of PCR, the annealing temperature was about the<br>
melting temperature of the 3' annealing region of the second primers, which was<br>
37°C. The annealing temperature in the second cycle of PCR was about the<br>
melting temperature of the 3' region, which anneals to the template DNA, of the<br>
first primer, which was 57°C. The annealing temperature in the third cycle of<br><br>
PCR was about the melting temperature of the entire sequence of the second<br>
primer, which was 64°C. The annealing temperature for the remaining cycles was<br>
64°C. Escalating the annealing temperature from TM1 to TM2 to TM3 in the first<br>
three cycles of PCR greatly improves specificity. These annealing temperatures<br>
are representative, and the skilled artisan will understand the annealing<br>
temperatures for each cycle are dependent on the specific primers used.<br>
[0283] The temperatures and times for denaturing, annealing, and extension,<br>
can be optimized by trying various settings and using the parameters that yield the<br>
best results. Schematics of the PCR products for SNP HC21S00027 and SNP<br>
TSC095512 are shown in FIGS. 5A-5D.<br>
Purification of Fragment Containing Locus of Interest<br>
[0284] The PCR products were separated from the genomic template DNA.<br>
Each PCR product was divided into four separate reaction wells of a Streptawell,<br>
transparent, High-Bind plate from Roche Diagnostics GmbH (catalog number 1<br>
645 692, as listed in Roche Molecular Biochemicals, 2001 Biochemicals Catalog).<br>
The first primers contained a 5' biotin tag so the PCR products bound to the<br>
Streptavidin coated wells while the genomic template DNA did not. The<br>
streptavidin binding reaction was performed using a Thermomixer (Eppendorf) at<br>
1000 rpm for 20 min. at 37°C. Each well was aspirated to remove unbound<br>
material, and washed three times with IX PBS, with gentle mixing (Kandpal et<br>
al., Nucl. Acids Res. 18:1789-1795 (1990); Kaneoka et al., Biotechniques 10:30-<br>
34 (1991); Green et al., Nucl. Acids Res. 18:6163-6164 (1990)).<br>
Restriction Enzyme Digestion of Isolated Fragments Containing Loci of Interest<br>
[0285] The purified PCR products were digested with the restriction enzyme<br>
that bound the recognition site incorporated into the PCR products from the<br>
second primer. DNA templates containing SNP HC21S00027 (FIG. 6A and 6B)<br>
and SNP TSC0095512 (FIG. 6C and 6D) were amplified in separate reactions<br>
using two different second primers. FIG. 6A (SNP HC21S00027) and FIG. 6C<br>
(SNP TSC0095512) depict the PCR products after digestion with the restriction<br><br>
enzyme BsmF I (New England Biolabs catalog number R0572S). FIG. 6B (SNP<br>
HC21S00027) and FIG. 6D (SNP TSC0095512) depict the PCR products after<br>
digestion with the restriction enzyme BceA I (New England Biolabs, catalog<br>
number R0623S). The digests were performed in the Streptawells following the<br>
instructions supplied with the restriction enzyme. The DNA fragment containing<br>
SNP TSC0264580, was digested with BsmF I. After digestion with the<br>
appropriate restriction enzyme, the wells were washed three times with PBS to<br>
remove the cleaved fragments.<br>
Incorporation of Labeled Nucleotide<br>
[0286] The restriction enzyme digest described above yielded a DNA<br>
fragment with a 5' overhang, which contained the SNP site or locus of interest and<br>
a 3' recessed end. The 5' overhang functioned as a template allowing<br>
incorporation of a nucleotide or nucleotides in the presence of a DNA polymerase.<br>
[0287] For each SNP, four separate fill in reactions were performed; each of<br>
the four reactions contained a different fluorescently labeled ddNTP (ddATP,<br>
ddTTP, ddGTP, or ddCTP). The following components were added to each fill in<br>
reaction: 1 μl of a fluorescently labeled ddNTP, 0.5 μl of unlabeled ddNTPs (40<br>
uM), which contained all nucleotides except the nucleotide that was fluorescently<br>
labeled, 2 μl of 10X sequenase buffer, 0.25 μl of Sequenase, and water as needed<br>
for a 20ul reaction. All of the fill in reactions were performed at 40°C for 10 min.<br>
Non-fluorescently labeled ddNTP was purchased from Fermentas Inc. (Hanover,<br>
MD). All other labeling reagents were obtained from Amersham (Thermo<br>
Sequenase Dye Terminator Cycle Sequencing Core Kit, US 79565). In the<br>
presence of fluorescently labeled ddNTPs, the 3' recessed end was extended by<br>
one base, which corresponds to the SNP or locus of interest (FIG 7A-7D).<br>
[0288] A mixture of labeled ddNTPs and unlabeled dNTPs also was used for<br>
the "fill in" reaction for SNP HC21S00027. The "fill in" conditions were as<br>
described above except that a mixture containing 40 μM unlabeled dNTPs, 1 μl<br>
fluorescently labeled ddATP, 1 μl fluorescently labeled ddTTP, 1 ui fluorescently<br>
labeled ddCTP, and 1 μl ddGTP was used. The fluorescent ddNTPs were<br><br>
obtained from Amersham (Thermo Sequenase Dye Terminator Cycle Sequencing<br>
Core Kit, US 79565; Amersham did not publish the concentrations of the<br>
fluorescent nucleotides). The DNA fragment containing SNP HC21S00027 was<br>
digested with the restriction enzyme BsmF I, which generated a 5' overhang of<br>
four bases. As shown in FIG. 7E, if the first nucleotide incorporated is a labeled<br>
ddNTP, the 3' recessed end is filled in by one base, allowing detection of the SNP<br>
or locus of interest. However, if the first nucleotide incorporated is a dNTP, the<br>
polymerase continues to incorporate nucleotides until a ddNTP is filled in. For<br>
example, the first two nucleotides may be filled in with dNTPs, and the third<br>
nucleotide with a ddNTP, allowing detection of the third nucleotide in the<br>
overhang. Thus, the sequence of the entire 5' overhang may be determined, which<br>
increases the information obtained from each SNP or locus of interest.<br>
[0289] After labeling, each Streptawell was rinsed with IX PBS (100 ul) three<br>
times. The "filled in" DNA fragments were then released from the Streptawells<br>
by digestion with the restriction enzyme EcoRI, according to the manufacturer's<br>
instructions that were supplied with the enzyme (FIGS. 8A-8D). Digestion was<br>
performed for 1 hour at 37 °C with shaking at 120 rpm.<br>
Detection of the Locus of Interest<br>
[0290] After release from the streptavidin matrix, 2-3 μl of the 10 μl sample<br>
was loaded in a 48 well membrane tray (The Gel Company, catalog number<br>
TAM48-01). The sample in the tray was absorbed with a 48 Flow Membrane<br>
Comb (The Gel Company, catalog number AM48), and inserted into a 36 cm 5%<br>
acrylamide (urea) gel (BioWhittaker Molecular Applications, Long Ranger Run<br>
Gel Packs, catalog number 50691).<br>
[0291] The sample was electrophoresed into the gel at 3000 volts for 3 min.<br>
The membrane comb was removed, and the gel was run for 3 hours on an ABI<br>
377 Automated Sequencing Machine. The incorporated labeled nucleotide was<br>
detected by fluorescence.<br>
[0292] As shown in FIG. 9A, from a sample of thirty six (36) individuals, one<br>
of two nucleotides, either adenosine or guanine, was detected at SNP<br><br>
HC21S00027. These are the two nucleotides reported to exist at SNP<br>
HC21S00027 (www.snp.schl.org/snpsearch.shtml). One of two nucleotides,<br>
either guanine or cytosine, was detected at SNP TSC0095512 (FIG. 9B). The<br>
same results were obtained whether the locus of interest was amplified with a<br>
second primer that contained a recognition site for BceA I or the second primer<br>
contained a recognition site for BsmF I.<br>
[0293] As shown in FIG. 9C, one of two nucleotides was detected at SNP<br>
TSC0264580, which was either adenosine or cytosine. These are the two<br>
nucleotides reported for this SNP site (www.snp.schl.org/snpsearch.shtml). In<br>
addition, a thymidine was detected one base upstream of the locus of interest. In a<br>
sequence dependent manner, BsmF I cuts some DNA molecules at the 10/14<br>
position and other DNA molecules, which have the same sequence, at the 11/15<br>
position. When the restriction enzyme BsmF I cuts 11 nucleotides away on the<br>
sense strand and 15 nucleotides away on the antisense strand, the 3' recessed end<br>
is one base upstream of the SNP site. The sequence of SNP TSC0264580<br>
indicated that the base immediately preceding the SNP site was a thymidine. The<br>
incorporation of a labeled ddNTP into this position generated a fragment one base<br>
smaller than the fragment that was cut at the 10/14 position. Thus, the DNA<br>
molecules cut at the 11/15 position provided identity information about the base<br>
immediately preceding the SNP site, and the DNA molecules cut at the 10/14<br>
position provided identity information about the SNP site.<br>
[0294] SNP HC21S00027 was amplified using a second primer that contained<br>
the recognition site for BsmF I. A mixture of labeled ddNTPs and unlabeled<br>
dNTPs was used to fill in the 5' overhang generated by digestion with BsmF I. If<br>
a dNTP was incorporated, the polymerase continued to incorporate nucleotides<br>
until a ddNTP was incorporated. A population of DNA fragments, each differing<br>
by one base, was generated, which allowed the full sequence of the overhang to be<br>
determined.<br>
[0295] As seen in FIG. 9D, an adenosine was detected, which was<br>
complementary to the nucleotide (a thymidine) immediately preceding the SNP or<br><br>
locus of interest. This nucleotide was detected because of the 11/15 cutting<br>
property of BsmF I, which is described in detail above. A guanine and an<br>
adenosine were detected at the SNP site, which are the two nucleotides reported<br>
for this SNP site (FIG. 9A). The two nucleotides were detected at the SNP site<br>
because the molecular weights of the dyes differ, which allowed separation of the<br>
two nucleotides. The next nucleotide detected was a thymidine, which is<br>
complementary to the nucleotide immediately downstream of the SNP site. The<br>
next nucleotide detected was a guanine, which was complementary to the<br>
nucleotide two bases downstream of the SNP site. Finally, an adenosine was<br>
detected, which was complementary to the third nucleotide downstream of the<br>
SNP site. Sequence information was obtained not only for the SNP site but for<br>
the nucleotide immediately preceding the SNP site and the next three nucleotides.<br>
[0296] None of the loci of interest contained a mutation. However, if one of<br>
the loci of interest harbored a mutation including but not limited to a point<br>
mutation, insertion, deletion, translocation or any combination of said mutations,<br>
it could be identified by comparison to the consensus or published sequence.<br>
Comparison of the sequences attributed to each of the loci of interest to the native,<br>
non-disease related sequence of the gene at each locus of interest determines the<br>
presence or absence of a mutation in that sequence. The finding of a mutation in<br>
the sequence is then interpreted as the presence of the indicated disease, or a<br>
predisposition to develop the same, as appropriate, in that individual. The relative<br>
amounts of the mutated vs. normal or non-mutated sequence can be assessed to<br>
determine if the subject has one or two alleles of the mutated sequence, and thus<br>
whether the subject is a carrier, or whether the indicated mutation results in a<br>
dominant or recessive condition.<br>
EXAMPLE 3<br>
[0297] Four loci of interest from chromosome 1 and two loci of interest from<br>
chromosome 21 were amplified in separate PCR reactions, pooled together, and<br>
analyzed. The primers were designed so that each amplified locus of interest was<br>
a different size, which allowed detection of the loci of interest.<br><br>
Preparation of Template DNA<br>
[0298] The template DNA was prepared from a 5 ml sample of blood obtained<br>
by venipuncture from a human volunteer with informed consent. Template DNA<br>
was isolated using the QIAmp DNA Blood Midi Kit supplied by QIAGEN<br>
(Catalog number 51183). The template DNA was isolated as per instructions<br>
included in the kit. Template DNA was isolated from thirty-six human<br>
volunteers, and then pooled into a single sample for further analysis.<br>
Design of Primers<br>
[0299] SNP TSC 0087315 was amplified using the following primers:<br>
First primer:<br>
5' TTACAATGCATGAATTCATCTTGGTCTCTCAAAGTGC 3' (SEQ ID<br>
NO: 15)<br>
Second primer:<br>
5' TGGACCATAAACGGCCAAAAACTGTAAG 3' (SEQ ID NO:16)<br>
[0300] SNP TSC0214366 was amplified using the following primers:<br>
First primer:<br>
5' ATGACTAGCTATGAATTCGTTCAAGGTAGAAAATGGAA 3' (SEQ ID<br>
NO: 13)<br>
Second primer:<br>
5' GAGAATTAGAACGGCCCAAATCCCACTC 3' (SEQ ID N0.14)<br>
[0301] SNP TSC 0413944 was amplified with the following primers:<br>
First primer:<br>
5' TACCTTTTGATCGAATTCAAGGCCAAAAATATTAAGTT 3' (SEQ ID<br>
NO:23)<br>
Second primer:<br>
5' TCGAACTTTAACGGCCTTAGAGTAGAGA 3' (SEQ ID NO:24)<br>
[0302] SNP TSC0095512 was amplified using the following primers:<br>
First primer:<br>
5' AAGTTTAGATCAGAATTCGTGAAAGCAGAAGTTGTCTG 3' (SEQ ID<br>
NO: 11)<br><br>
Second primer:<br>
5' TCTCCAACTAACGGCTCATCGAGTAAAG 3' (SEQ ID NO: 12)<br>
[0303] SNP HC21S00131 was amplified with the following primers:<br>
First primer:<br>
5' CGATTTCGATAAGAATTCAAAAGCAGTTCTTAGTTCAG 3' (SEQ ID<br>
NO:25)<br>
Second primer:<br>
5' TGCGAATCTTACGGCTGCATCACATTCA 3' (SEQ ID NO:26)<br>
[0304] SNP HC21S00027 was amplified with the following primers:<br>
First primer:<br>
5' ATAACCGTATGCGAATTCTATAATTTTCCTGATAAAGG 3' (SEQ ID<br>
NO: 17)<br>
Second primer:<br>
5' CTTAAATCAGACGGCTAGGTAAACTTCA 3' (SEQ ID NO: 19)<br>
[0305] For each SNP, the first primer contained a recognition site for the<br>
restriction enzyme EcoRI and had a biotin tag at the extreme 5' end. The second<br>
primer used to amplify each SNP contained a recognition site for the restriction<br>
enzyme BceA I.<br>
PCR Reaction<br>
[0306] The PCR reactions were performed as described in Example 2 except<br>
that the following annealing temperatures were used: the annealing temperature<br>
for the first cycle of PCR was 37°C for 30 seconds, the annealing temperature for<br>
the second cycle of PCR was 57°C for 30 seconds, and the annealing temperature<br>
for the third cycle of PCR was 64°C for 30 seconds. All subsequent cycles had an<br>
annealing temperature of 64°C for 30 seconds. Thirty seven (37) cycles of PCR<br>
were performed. After PCR, ¼ of the volume was removed from each reaction,<br>
and combined into a single tube.<br><br>
Purification of Fragment Containing Locus of Interest<br>
[0307] The PCR products (now combined into one sample, and referred to as<br>
"the sample") were separated from the genomic template DNA as described in<br>
Example 2 except that the sample was bound to a single well of a Streptawell<br>
microtiter plate.<br>
Restriction Enzyme Digestion of Isolated Fragments Containing Loci of Interest<br>
[0308] The sample was digested with the restriction enzyme BceA I, which<br>
bound the recognition site in the second primer. The restriction enzyme<br>
digestions were performed following the instructions supplied with the enzyme.<br>
After the restriction enzyme digest, the wells were washed three times with IX<br>
PBS.<br>
Incorporation of Nucleotides<br>
[0309] The restriction enzyme digest described above yielded DNA molecules<br>
with a 5' overhang, which contained the SNP site or locus of interest and a 3'<br>
recessed end. The 5' overhang functioned as a template allowing incorporation of<br>
a nucleotide in the presence of a DNA polymerase.<br>
[0310] The following components were used for the fill in reaction: 1 μl of<br>
fluorescently labeled ddATP; 1 μl of fluorescently labeled ddTTP; 1 μl of<br>
fluorescently labeled ddGTP; 1 μl of fluorescently labeled ddCTP; 2 μl of 10X<br>
sequenase buffer, 0.25 μl of Sequenase, and water as needed for a 20 μl reaction.<br>
The fill in reaction was performed at 40°C for 10 min. All labeling reagents were<br>
obtained from Amersham (Thermo Sequenase Dye Terminator Cycle Sequencing<br>
Core Kit (US 79565); the concentration of the ddNTPS provided in the kit is<br>
proprietary and not published by Amersham). In the presence of fluorescently<br>
labeled ddNTPs, the 3' recessed end was filled in by one base, which corresponds<br>
to the SNP or locus of interest.<br>
[0311] After the incorporation of nucleotide, the Streptawell was rinsed with<br>
IX PBS (100 μl) three times. The "filled in" DNA fragments were then released<br>
from the Streptawell by digestion with the restriction enzyme EcoRI following the<br><br>
manufacturer's instructions. Digestion was performed for 1 hour at 37 °C with<br>
shaking at 120 rpm.<br>
Detection of the Locus of Interest<br>
[0312] After release from the streptavidin matrix, 2-3 μl of the 10 μl sample<br>
was loaded in a 48 well membrane tray (The Gel Company, catalog number<br>
TAM48-01). The sample in the tray was absorbed with a 48 Flow Membrane<br>
Comb (The Gel Company, catalog number AM48), and inserted into a 36 cm 5%<br>
acrylamide (urea) gel (BioWhittaker Molecular Applications, Long Ranger Run<br>
Gel Packs, catalog number 50691).<br>
[0313] The sample was electrophoresed into the gel at 3000 volts for 3 min.<br>
The membrane comb was removed, and the gel was run for 3 hours on an ABI<br>
377 Automated Sequencing Machine. The incorporated nucleotide was detected<br>
by fluorescence.<br>
[0314] The primers were designed so that each amplified locus of interest<br>
differed in size. As shown in FIG. 10, each amplified loci of interest differed by<br>
about 5-10 nucleotides, which allowed the loci of interest to be separated from<br>
one another by gel electrophoresis. Two nucleotides were detected for SNP<br>
TSC0087315, which were guanine and cytosine. These are the two nucleotides<br>
reported to exist at SNP TSC0087315 (www.snp.schl.org/snpsearch.shtml). The<br>
sample comprised template DNA from 36 individuals and because the DNA<br>
molecules that incorporated a guanine differed in molecular weight from those<br>
that incorporated a cytosine, distinct bands were seen for each nucleotide.<br>
[0315] Two nucleotides were detected at SNP HC21S00027, which were<br>
guanine and adenosine (FIG. 10). The two nucleotides reported for this SNP site<br>
are guanine and adenosine (www.snp.schl.org/snpsearch.shtml). As discussed<br>
above, the sample contained template DNA from thirty-six individuals, and one<br>
would expect both nucleotides to be represented in the sample. The molecular<br>
weight of the DNA fragments that incorporated a guanine was distinct from the<br>
DNA fragments that incorporated an adenosine, which allowed both nucleotides<br>
to be detected.<br><br>
[0316] The nucleotide cytosine was detected at SNP TSC0214366 (FIG. 10).<br>
The two nucleotides reported to exist at this SNP position are thymidine and<br>
cytosine.<br>
[0317J The nucleotide guanine was detected at SNP TSC0413944 (FIG. 10).<br>
The two nucleotides reported for this SNP are guanine and cytosine<br>
(http://snp.cshl.org/snpsearch.shtml).<br>
[0318] The nucleotide cytosine was detected at SNP TSC0095512 (FIG. 10).<br>
The two nucleotides reported for this SNP site are guanine and cytosine<br>
(www.snp.schl.org/snpsearch.shtml).<br>
[0319] The nucleotide detected at SNP HC21S00131 was guanine. The two<br>
nucleotides reported for this SNP site are guanine and adenosine<br>
(www.snp.schl.org/snpsearch.shtml).<br>
[0320] As discussed above, the sample was comprised of DNA templates<br>
from thirty-six individuals and one would expect both nucleotides at the SNP sites<br>
to be represented. For SNP TSC0413944, TSC0095512, TSC0214366 and<br>
HC21S00131, one of the two nucleotides was detected. It is likely that both<br>
nucleotides reported for these SNP sites are present in the sample but that one<br>
fluorescent dye overwhelms the other. The molecular weight of the DNA<br>
molecules that incorporated one nucleotide did not allow efficient separation of<br>
the DNA molecules that incorporated the other nucleotide. However, the SNPs<br>
were readily separated from one another, and for each SNP, a proper nucleotide<br>
was incorporated. The sequences of multiple loci of interest from multiple<br>
chromosomes, which were treated as a single sample after PCR, were determined.<br>
[0321] A single reaction containing fluorescently labeled ddNTPs was<br>
performed with the sample that contained multiple loci of interest. Alternatively,<br>
four separate fill in reactions can be performed where each reaction contains one<br>
fluorescently labeled nucleotide (ddATP, ddTTP, ddGTP, or ddCTP) and<br>
unlabeled ddNTPs (see Example 2, FIGS. 7A-7D and FIGS. 9A-C). Four<br>
separate "fill in" reactions will allow detection of any nucleotide that is present at<br>
the loci of interest. For example, if analyzing a sample that contains multiple loci<br><br>
of interest from a single individual, and said individual is heterozygous at one or<br>
more than one loci of interest, four separate "fill in" reactions can be used to<br>
determine the nucleotides at the heterozygous loci of interest.<br>
[0322] Also, when analyzing a sample that contains templates from multiple<br>
individuals, four separate "fill in" reactions will allow detection of nucleotides<br>
present in the sample, independent of how frequent the nucleotide is found at the<br>
locus of interest. For example, if a sample contains DNA templates from 50<br>
individuals, and 49 of the individuals have a thymidine at the locus of interest, and<br>
one individual has a guanine, the performance of four separate "fill in" reactions,<br>
wherein each "fill in" reaction is run in a separate lane of a gel, such as in FIGS.<br>
9A-9C, will allow detection of the guanine. When analyzing a sample comprised<br>
of multiple DNA templates, multiple "fill in" reactions will alleviate the need to<br>
distinguish multiple nucleotides at a single site of interest by differences in mass.<br>
[0323] In this example, multiple single nucleotide polymorphisms were<br>
analyzed. It is also possible to determine the presence or absence of mutations,<br>
including point mutations, transitions, transversions, translocations, insertions,<br>
and deletions from multiple loci of interest. The multiple loci of interest can be<br>
from a single chromosome or from multiple chromosomes. The multiple loci of<br>
interest can be from a single gene or from multiple genes.<br>
[0324] The sequence of multiple loci of interest that cause or predispose to a<br>
disease phenotype can be determined. For example, one could amplify one to tens<br>
to hundreds to thousands of genes implicated in cancer or any other disease. The<br>
primers can be designed so that each amplified loci of interest differs in size.<br>
After PCR, the amplified loci of interest can be combined and treated as a single<br>
sample. Alternatively, the multiple loci of interest can be amplified in one PCR<br>
reaction or the total number of loci of interest, for example 100, can be divided<br>
into samples, for example 10 loci of interest per PCR reaction, and then later<br>
pooled. As demonstrated herein, the sequence of multiple loci of interest can be<br>
determined. Thus, in one reaction, the sequence of one to ten to hundreds to<br><br>
thousands of genes that predispose or cause a disease phenotype can be<br>
determined.<br>
EXAMPLE 4<br>
[0325] Genomic DNA was obtained from four individuals after informed<br>
consent was obtained. Six SNPs on chromosome 13 (TSC0837969, TSC0034767,<br>
TSC1130902, TSC0597888, TSC0195492, TSC0607185) were analyzed using the<br>
template DNA. Information regarding these SNPs can be found at the following<br>
website (www.snp.schl.org/snpsearch.shtml)website active as of February 11,<br>
2003).<br>
[0326] A single nucleotide labeled with one fluorescent dye was used to<br>
genotype the individuals at the six selected SNP sites. The primers were designed<br>
to allow the six SNPs to be analyzed in a single reaction.<br>
Preparation of Template DNA<br>
[0327] The template DNA was prepared from a 9 ml sample of blood obtained<br>
by venipuncture from a human volunteer with informed consent. Template DNA<br>
was isolated using the QIAmp DNA Blood Midi Kit supplied by QIAGEN<br>
(Catalog number 51183). The template DNA was isolated as per instructions<br>
included in the kit.<br>
Design of Primers<br>
[0328] SNP TSC0837969 was amplified using the following primer set:<br>
First primer:<br>
5' GGGCTAGTCTCCGAATTCCACCTATCCTACCAAATGTC 3'<br>
Second primer:<br>
5' TAGCTGTAGTTAGGGACTGTTCTGAGCAC 3'<br>
[0329] The first primer had a biotin tag at the 5' end and contained a<br>
restriction enzyme recognition site for EcoRI. The first primer was designed to<br>
anneal 44 bases from of the locus of interest. The second primer contained a<br>
restriction enzyme recognition site for BsmF I.<br>
[0330] SNP TSC0034767 was amplified using the following primer set:<br><br>
First primer:<br>
5' CGAATGCAAGGCGAATTCGTTAGTAATAACACAGTGCA 3'<br>
Second primer:<br>
5' AAGACTGGATCCGGGACCATGTAGAATAC 3'<br>
[0331] The first primer had a biotin tag at the 5' end and contained a<br>
restriction enzyme recognition site for EcoRI. The first primer was designed to<br>
anneal 50 bases from the locus of interest. The second primer contained a<br>
restriction enzyme recognition site for BsmF I.<br>
[0332] SNP TSC1130902 was amplified using the following primer set:<br>
First primer:<br>
5' TCTAACCATTGCGAATTCAGGGCAAGGGGGGTGAGATC 3'<br>
Second primer:<br>
5' TGACTTGGATCCGGGACAACGACTCATCC 3'<br>
[0333] The first primer had a biotin tag at the 5' end and contained a<br>
restriction enzyme recognition site for EcoRI. The first primer was designed to<br>
anneal 60 bases from the locus of interest. The second primer contained a<br>
restriction enzyme recognition site for BsmF I.<br>
[0334] SNP TSC0597888 was amplified using the following primer set:<br>
First primer:<br>
5' ACCCAGGCGCCAGAATTCTTTAGATAAAGCTGAAGGGA 3'<br>
Second primer:<br>
5' GTTACGGGATCCGGGACTCCATATTGATC 3'<br>
[0335] The first primer had a biotin tag at the 5' end and contained a<br>
restriction enzyme recognition site for EcoRI. The first primer was designed to<br>
anneal 70 bases from the locus of interest. The second primer contained a<br>
restriction enzyme recognition site for BsmF I.<br>
[0336] SNP TSC0195492 was amplified using the following primer set:<br>
First primer:<br>
5'CGTTGGCTTGAGGAATTCGACCAAAAGAGCCAAGAGAA<br>
Second primer:<br><br>
5' AAAAAGGGATCCGGGACCTTGACTAGGAC 3'<br>
[0337] The first primer had a biotin tag at the 5' end and contained a<br>
restriction enzyme recognition site for EcoRI. The first primer was designed to<br>
anneal 80 bases from the locus of interest. The second primer contained a<br>
restriction enzyme recognition site for BsmF I.<br>
[0338] SNP TSC0607185 was amplified using the following primer set:<br>
First primer:<br>
5' ACTTGATTCCGTGAATTCGTTATCAATAAATCTTACAT 3'<br>
Second primer:<br>
5' CAAGTTGGATCCGGGACCCAGGGCTAACC 3'<br>
[0339] The first primer had a biotin tag at the 5' end and contained a<br>
restriction enzyme recognition site for EcoRI. The first primer was designed to<br>
anneal 90 bases from the locus of interest. The second primer contained a<br>
restriction enzyme recognition site for BsmF I.<br>
[0340] All loci of interest were amplified from the template genomic DNA<br>
using the polymerase chain reaction (PCR, U.S. Patent Nos. 4,683,195 and<br>
4,683,202, incorporated herein by reference). In this example, the loci of interest<br>
were amplified in separate reaction tubes but they could also be amplified together<br>
in a single PCR reaction. For increased specificity, a "hot-start" PCR was used.<br>
PCR reactions were performed using the HotStarTaq Master Mix Kit supplied by<br>
QIAGEN (catalog number 203443). The amount of template DNA and primer<br>
per reaction can be optimized for each locus of interest but in this example, 40 ng<br>
of template human genomic DNA and 5 μM of each primer were used. Forty<br>
cycles of PCR were performed. The following PCR conditions were used:<br>
(1)	95°C for 15 minutes and 15 seconds;<br>
(2)	37°C for 30 seconds;<br>
(3)	95°C for 30 seconds;<br>
(4)	57°C for 30 seconds;<br>
(5)	95°C for 30 seconds;<br>
(6)	64°C for 30 seconds;<br><br>
(7)	95°C for 30 seconds;<br>
(8)	Repeat steps 6 and 7 thirty nine (39) times;<br>
(9)	72°C for 5 minutes.<br>
[0341] In the first cycle of PCR, the annealing temperature was about the<br>
melting temperature of the 3' annealing region of the second primers, which was<br>
37°C. The annealing temperature in the second cycle of PCR was about the<br>
melting temperature of the 3' region, which anneals to the template DNA, of the<br>
first primer, which was 57°C. The annealing temperature in the third cycle of<br>
PCR was about the melting temperature of the entire sequence of the second<br>
primer, which was 64°C. The annealing temperature for the remaining cycles was<br>
64°C. Escalating the annealing temperature from TM1 to TM2 to TM3 in the first<br>
three cycles of PCR greatly improves specificity. These annealing temperatures<br>
are representative, and the skilled artisan will understand the annealing<br>
temperatures for each cycle are dependent on the specific primers used.<br>
[0342] The temperatures and times for denaturing, annealing, and extension,<br>
can be optimized by trying various settings and using the parameters that yield the<br>
best results. In this example, the first primer was designed to anneal at various<br>
distances from the locus of interest. The skilled artisan understands that the<br>
annealing location of the first primer can be 5-10,11-15,16-20,21-25,26-30,31-<br>
35,36-40,41-45,46-50,51-55,56-60, 61-65,66-70, 71-75,76-80, 81-85,86-90,<br>
91-95,96-100,101-105,106-110,111-115,116-120,121-125,126-130,131-140,<br>
141-160,161-180,181-200,201-220,221-240,241-260,261-280,281-300,301-<br>
350,351-400,401-450,451-500, or greater than 500 bases from the locus of<br>
interest.<br>
Purification of Fragment Containing Locus of Interest<br>
[0343] The PCR products were separated from the genomic template DNA.<br>
After the PCR reaction, 1/4 of the volume of each PCR reaction from one<br>
individual was mixed together in a well of a Streptawell, transparent, High-Bind<br>
plate from Roche Diagnostics GmbH (catalog number 1 645 692, as listed in<br>
Roche Molecular Biochemicals, 2001 Biochemicals Catalog). The first primers<br><br>
contained a 5' biotin tag so the PCR products bound to the Streptavidin coated<br>
wells while the genomic template DNA did not. The streptavidin binding reaction<br>
was performed using a Thermomixer (Eppendorf) at 1000 rpm for 20 min. at<br>
37°C. Each well was aspirated to remove unbound material, and washed three<br>
times with IX PBS, with gentle mixing (Kandpal et al., Nucl. Acids Res. 18:1789-<br>
1795 (1990); Kaneoka et al., Biotechniques 10:30-34 (1991); Green et al., Nucl.<br>
Acids Res. 18:6163-6164(1990)).<br>
Restriction Enzyme Digestion of Isolated Fragments Containing Loci of Interest<br>
[0344] The purified PCR products were digested with the restriction enzyme<br>
BsmF I, which binds to the recognition site incorporated into the PCR products<br>
from the second primer. The digests were performed in the Streptawells<br>
following the instructions supplied with the restriction enzyme. After digestion,<br>
the wells were washed three times with PBS to remove the cleaved fragments.<br>
Incorporation of Labeled Nucleotide<br>
[0345] The restriction enzyme digest with BsmF I yielded a DNA fragment<br>
with a 5' overhang, which contained the SNP site or locus of interest and a 3'<br>
recessed end. The 5' overhang functioned as a template allowing incorporation of<br>
a nucleotide or nucleotides in the presence of a DNA polymerase.<br>
[0346] Below, a schematic of the 5' overhang for SNP TSC0837969 is shown.<br>
The entire DNA sequence is not reproduced, only the portion to demonstrate the<br>
overhang (where R indicates the variable site).<br>
5'TTAA<br>
3'AATT	R A C A<br>
Overhang position	12	3	4<br>
[0347] The observed nucleotides for TSC0837969 on the 5' sense strand (here<br>
depicted as the top strand) are adenine and guanine. The third position in the<br>
overhang on the antisense strand corresponds to cytosine, which is<br>
complementary to guanine. As this variable site can be adenine or guanine,<br>
fluorescently labeled ddGTP in the presence of unlabeled dCTP, dTTP, and dATP<br><br>
was used to determine the sequence of both alleles. The fill-in reactions for an<br>
individual homozygous for guanine, homozygous for adenine or heterozygous are<br>
diagrammed below.<br>
[0348] Homozygous for guanine at TSC 0837969:<br>
Allele 1	5' TTAA	G*<br>
3'AATT	C A C A<br>
Overhang position	12	3	4<br>
Allele 2	5' TTAA	G*<br>
3'AATT	C A C	A<br>
Overhang position	12	3	4<br>
[00100] Labeled ddGTP is incorporated into the first position of the overhang.<br>
Only one signal is seen, which corresponds to the molecules filled in with labeled<br>
ddGTP at the first position of the overhang.<br>
[0349] Homozygous for adenine at TSC 083 7969:<br>
Allele 1	5'TTAA	A T G*<br>
3'AATT	T A C A<br>
Overhang position	12	3	4<br>
Allele 2	5'TTAA	A T G*<br>
3'AATT	T A C A<br>
Overhang position	12	3	4<br>
[0350] Unlabeled dATP is incorporated at position one of the overhang, and<br>
unlabeled dTTP is incorporated at position two of the overhang. Labeled ddGTP<br>
is incorporated at position three of the overhang. Only one signal will be seen; the<br>
molecules filled in with ddGTP at position 3 will have a different molecular<br>
weight from molecules filled in at position one, which allows easy identification<br>
of individuals homozygous for adenine or guanine.<br>
[0351] Heterozygous at TSC0837969:<br>
Allele 1	5' TTAA	G*<br>
3'AATT	C A C A<br>
Overhang position	12	3	4<br><br>
Allele 2	5'TTAA	A T G*<br>
3'AATT	T	A C A<br>
Overhang position	12	3	4<br>
[0352] Two signals will be seen; one signal corresponds to the DNA<br>
molecules filled in with ddGTP at position 1, and a second signal corresponding<br>
to molecules filled in at position 3 of the overhang. The two signals can be<br>
separated using any technique that separates based on molecular weight including<br>
but not limited to gel electrophoresis.<br>
[0353] Below, a schematic of the 5' overhang for SNP TSC0034767 is shown.<br>
The entire DNA sequence is not reproduced, only the portion to demonstrate the<br>
overhang (where R indicates the variable site).<br>
A C A R GTGT 3'<br>
CACA 5'<br>
4	3	2	1	Overhang Position<br>
[0354] The observed nucleotides for TSC0034767 on the 5' sense strand (here<br>
depicted as the top strand) are cytosine and guanine. The second position in the<br>
overhang corresponds to adenine, which is complementary to thymidine. The<br>
third position in the overhang corresponds to cytosine, which is complementary to<br>
guanine. Fluorescently labeled ddGTP in the presence of unlabeled dCTP, dTTP,<br>
and dATP is used to determine the sequence of both alleles.<br>
[0355] In this case, the second primer anneals upstream of the locus of<br>
interest, and thus the fill-in reaction occurs on the anti-sense strand (here depicted<br>
as the bottom strand). Either the sense strand or the antisense strand can be filled<br>
in depending on whether the second primer, which contains the type IIS<br>
restriction enzyme recognition site, anneals upstream or downstream of the locus<br>
of interest.<br>
Below, a schematic of the 5' overhang for SNP TSC1130902 is shown. The<br>
entire DNA sequence is not reproduced, only a portion to demonstrate the<br>
overhang (where R indicates the variable site).<br>
5' TTCAT<br><br>
3'AAGTA	R T C C<br>
Overhang position	12	3	4<br>
[00101] The observed nucleotides for TSC1130902 on the 5' sense strand are<br>
adenine and guanine. The second position in the overhang corresponds to a<br>
thymidine, and the third position in the overhang corresponds to cytosine, which<br>
is complementary to guanine.<br>
[0356] Fluorescently labeled ddGTP in the presence of unlabeled dCTP,<br>
dTTP, and dATP is used to determine the sequence of both alleles.<br>
[00102] Below, a schematic of the 5' overhang for SNP TSC0597888 is shown.<br>
The entire DNA sequence is not reproduced, only the portion to demonstrate the<br>
overhang (where R indicates the variable site).<br>
T	C T R ATTC3'<br>
TAAG5'<br>
4	3	2	1	Overhang position<br>
[0357] The observed nucleotides for TSC0597888 on the 5' sense strand (here<br>
depicted as the top strand) are cytosine and guanine. The third position in the<br>
overhang corresponds to cytosine, which is complementary to guanine.<br>
Fluorescently labeled ddGTP in the presence of unlabeled dCTP, dTTP, and<br>
dATP is used to determine the sequence of both alleles.<br>
[0358] Below, a schematic of the 5' overhang for SNP TSC0607185 is shown.<br>
The entire DNA sequence is not reproduced, only the portion to demonstrate the<br>
overhang (where R indicates the variable site).<br>
C C T R TGTC3'<br>
ACAG 5'<br>
4	3	2	1	Overhang position<br>
[0359] The observed nucleotides for TSC0607185 on the 5' sense strand (here<br>
depicted as the top strand) are cytosine and thymidine. In this case, the second<br>
primer anneals upstream of the locus of interest, which allows the anti-sense<br>
strand to be filled in. The anti-sense strand (here depicted as the bottom strand)<br>
will be filled in with guanine or adenine.<br><br>
[0360] The second position in the 5' overhang is thymidine, which is<br>
complementary to adenine, and the third position in the overhang corresponds to<br>
cytosine, which is complementary to guanine. Fluorescently labeled ddGTP in<br>
the presence of unlabeled dCTP, dTTP, and dATP is used to determine the<br>
sequence of both alleles.<br>
[0361] Below, a schematic of the 5' overhang for SNP TSCO195492 is shown.<br>
The entire DNA sequence is not reproduced, only the portion to demonstrate the<br>
overhang.<br>
5' ATCT<br>
3'TAGA	R A C A<br>
Overhang position	12	3	4<br>
[0362] The observed nucleotides at this site are cytosine and guanine on the<br>
sense strand (here depicted as the top strand). The second position in the 5'<br>
overhang is adenine, which is complementary to thymidine, and the third position<br>
in the overhang corresponds to cytosine, which is complementary to guanine.<br>
Fluorescently labeled ddGTP in the presence of unlabeled dCTP, dTTP, and<br>
dATP was used to determine the sequence of both alleles.<br>
[0363] As demonstrated above, the sequence of both alleles of the six SNPs<br>
can be determined by labeling with ddGTP in the presence of unlabeled dATP,<br>
dTTP, and dCTP. The following components were added to each fill in reaction:<br>
1 μl of fluorescently labeled ddGTP, 0.5 μl of unlabeled ddNTPs ( 40 uM), which<br>
contained all nucleotides except guanine, 2 μl of 10X sequenase buffer, 0.25 μl of<br>
Sequenase, and water as needed for a 20ul reaction. The fill in reaction was<br>
performed at 40°C for 10 min. Non-fluorescently labeled ddNTP was purchased<br>
from Fermentas Inc. (Hanover, MD). All other labeling reagents were obtained<br>
from Arnersham (Thermo Sequenase Dye Terminator Cycle Sequencing Core Kit,<br>
US 79565).<br>
[0364] After labeling, each Streptawell was rinsed with IX PBS (100 ul) three<br>
times. The "filled in" DNA fragments were then released from the Streptawells<br>
by digestion with the restriction enzyme EcoRI, according to the manufacturer's<br><br>
instructions that were supplied with the enzyme. Digestion was performed for 1<br>
hour at 37 °C with shaking at 120 rpm.<br>
Detection of the Locus of Interest<br>
[0365] After release from the streptavidin matrix, the sample was loaded into<br>
a lane of a 36 cm 5% acrylamide (urea) gel (BioWhittaker Molecular<br>
Applications, Long Ranger Run Gel Packs, catalog number 50691). The sample<br>
was electrophoresed into the gel at 3000 volts for 3 min. The gel was run for 3<br>
hours on a sequencing apparatus (Hoefer SQ3 Sequencer). The gel was removed<br>
from the apparatus and scanned on the Typhoon 9400 Variable Mode Imager.<br>
The incorporated labeled nucleotide was detected by fluorescence.<br>
[0366] As shown in FIG. 11, the template DNA in lanes 1 and 2 for SNP<br>
TSC0837969 is homozygous for adenine. The following fill-in reaction was<br>
expected to occur if the individual was homozygous for adenine:<br>
[0367] Homozygous for adenine at TSC 0837969:<br>
5'TTAA	A T G*<br>
3'AATT	T A C A<br>
Overhang position	12	3	4<br>
[0368] Unlabeled dATP was incorporated in the first position complementary<br>
to the overhang. Unlabeled dTTP was incorporated in the second position<br>
complementary to the overhang. Labeled ddGTP was incorporated in the third<br>
position complementary to the overhang. Only one band was seen, which<br>
migrated at about position 46 of the acrylamide gel. This indicated that adenine<br>
was the nucleotide filled in at position one. If the nucleotide guanine had been<br>
filled in, a band would be expected at position 44.<br>
[0369] However, the template DNA in lanes 3 and 4 for SNP TSC0837969<br>
was heterozygous. The following fill-in reactions were expected if the individual<br>
was heterozygous:<br>
[0370] Heterozygous at TSC0837969:<br>
Allele 1 5' TTAA	G*<br>
3'AATT	C A C A<br><br>
Overhang position	12	3	4<br>
Allele 2 5'TTAA	A T	G*<br>
3'AATT	T A C A<br>
Overhang position	12	3	4<br>
[0371] Two distinct bands were seen; one band corresponds to the molecules<br>
filled in with ddGTP at position 1 complementary to the overhang (the G allele),<br>
and the second band corresponds to molecules filled in with ddGTP at position 3<br>
complementary to the overhang (the A allele). The two bands were separated<br>
based on the differences in molecular weight using gel electrophoresis. One<br>
fluorescently labeled nucleotide ddGTP was used to determine that an individual<br>
was heterozygous at a SNP site. This is the first use of a single nucleotide to<br>
effectively detect the presence of two different alleles.<br>
[0372] For SNP TSC0034767, the template DNA in lanes 1 and 3 is<br>
heterozygous for cytosine and guanine, as evidenced by the two distinct bands.<br>
The lower band corresponds to ddGTP filled in at position 1 complementary to<br>
the overhang. The second band of slightly higher molecular weight corresponds<br>
to ddGTP filled in at position 3, indicating that the first position in the overhang<br>
was filled in with unlabeled dCTP, which allowed the polymerase to continue to<br>
incorporate nucleotides until it incorporated ddGTP at position 3 complementary<br>
to the overhang. The template DNA in lanes 2 and 4 was homozygous for<br>
guanine., as evidenced by a single band of higher molecular weight than if ddGTP<br>
had been filled in at the first position complementary to the overhang.<br>
[0373] For SNP TSC1130902, the template DNA in lanes 1,2, and 4 is<br>
homozygous for adenine at the variable site, as evidenced by a single higher<br>
molecular weight band migrating at about position 62 on the gel. The template<br>
DNA in lane 3 is heterozygous at the variable site, as indicated by the presence of<br>
two distinct bands. The lower band corresponded to molecules filled in with<br>
ddGTP at position 1 complementary to the overhang (the guanine allele). The<br>
higher molecular weight band corresponded to molecules filled in with ddGTP at<br>
position 3 complementary to the overhang (the adenine allele).<br><br>
[0374] For SNP TSC0597888, the template DNA in lanes 1 and 4 was<br>
homozygous for cytosine at the variable site; the template DNA in lane 2 was<br>
heterozygous at the variable site, and the template DNA in lane 3 was<br>
homozygous for guanine. The expected fill-in reactions are diagrammed below:<br>
[0375] Homozygous for cytosine:<br>
Allele 1	T C T	G ATTC 3'<br>
G* A C TAAG 5'<br>
4	3	2	1	Overhang position<br>
Allele 2	T C	T	G	ATTC 3'<br>
G*	A	C	TAAG 5'<br>
4	3	2	1	Overhang position<br>
[0376] Homozygous for guanine:<br>
Allele 1	T	C	T	C	ATTC 3'<br>
G* TAAG 5'<br>
4	3	2	1	Overhang position<br>
Allele 2	T C T	C ATTC 3'<br>
G* TAAG 5'<br>
4	3	2	1	Overhang position<br>
[0377] Heterozygous for guanine/cytosine:<br>
Allele1	T	C	T	G ATTC 3'<br>
G* A C TAAG 5'<br>
4	3	2	1	Overhang position<br>
Allele 2	T	C T	C ATTC 3'<br><br>
G* TAAG 5'<br>
4	3	2	1	Overhang position<br>
[0378] Template DNA homozygous for guanine at the variable site displayed<br>
a single band, which corresponded to the DNA molecules filled in with ddGTP at<br>
position 1 complementary to the overhang. These DNA molecules were of lower<br>
molecular weight compared to the DNA molecules filled in with ddGTP at<br>
position 3 of the overhang (see lane 3 for SNP TSC0597888). The DNA<br>
molecules differed by two bases in molecular weight.<br>
[0379] Template DNA homozygous for cytosine at the variable site displayed<br>
a single band, which corresponds to the DNA molecules filled in with ddGTP at<br>
position 3 complementary to the overhang. These DNA molecules migrated at a<br>
higher molecular weight than DNA molecules filled in with ddGTP at position 1<br>
(see lanes 1 and 4 for SNP TSC0597888).<br>
[0380] Template DNA heterozygous at the variable site displayed two bands;<br>
one band corresponded to the DNA molecules filled in with ddGTP at position 1<br>
complementary to the overhang and was of lower molecular weight, and the<br>
second band corresponded to DNA molecules filled in with ddGTP at position 3<br>
complementary to the overhang, and was of higher molecular weight (see lane 3<br>
for SNP TSC0597888).<br>
[0381] For SNP TSC0195492, the template DNA in lanes 1 and 3 was<br>
heterozygous at the variable site, which was demonstrated by the presence of two<br>
distinct bands. The template DNA in lane 2 was homozygous for guanine at the<br>
variable site. The template DNA in lane 4 was homozygous for cytosine. Only<br>
one band was seen in lane 4 for this SNP, and it had a higher molecular weight<br>
than the DNA molecules filled in with ddGTP at position 1 complementary to the<br>
overhang (compare lanes 2,3 and 4).<br>
[0382] The observed alleles for SNP TSC0607185 are reported as cytosine or<br>
thymidine. For consistency, the SNP consortium denotes the observed alleles as<br>
they appear in the sense strand (www.snp.schi.org/snpsearch.shtml); website<br><br>
active as of February 11,2003). For this SNP, the second primer annealed<br>
upstream of the locus of interest, which allowed the fill-in reaction to occur on the<br>
antisense strand after digestion with BsmF I.<br>
[0383] The template DNA in lanes 1 and 3 was heterozygous; the template<br>
DNA in lane 2 was homozygous for thymidine, and the template DNA in lane 4<br>
was homozygous for cytosine. The antisense strand was filled in with ddGTP, so<br>
the nucleotide on the sense strand corresponded to cytosine.<br>
[0384] Molecular weight markers can be used to identify the positions of the<br>
expected bands. Alternatively, for each SNP analyzed, a known heterozygous<br>
sample can be used, which will identify precisely the position of the two expected<br>
bands.<br>
[0385] As demonstrated in FIG. 11, one nucleotide labeled with one<br>
fluorescent dye can be used to determine the identity of a variable site including<br>
but not limited to SNPs and single nucleotide mutations. Typically, to determine<br>
if an individual is homozygous or heterozygous at a SNP site, multiple reactions<br>
are performed using one nucleotide labeled with one dye and a second nucleotide<br>
labeled with a second dye. However, this introduces problems in comparing<br>
results because the two dyes have different quantum coefficients. Even if<br>
different nucleotides are labeled with the same dye, the quantum coefficients are<br>
different. The use of a single nucleotide labeled with one dye eliminates any<br>
errors from the quantum coefficients of different dyes.<br>
[0386] In this example, fluorescently labeled ddGTP was used. However, the<br>
method is applicable for a nucleotide tagged with any signal generating moiety<br>
including but not limited to radioactive molecule, fluorescent molecule, antibody,<br>
antibody fragment, hapten, carbohydrate, biotin, derivative of biotin,<br>
phosphorescent moiety, lurninescent moiety, electrochemiluminescent moiety,<br>
chromatic moiety, and moiety having a detectable electron spin resonance,<br>
electrical capacitance, dielectric constant or electrical conductivity. In addition,<br>
labeled ddATP, ddTTP, or ddCTP can be used.<br><br>
[0387] The above example used the third position complementary to the<br>
overhang as an indicator of the second allele. However, the second or fourth<br>
position of the overhang can be used as well (see Section on Incorporation of<br>
Nucleotides). Furthermore, the overhang was generated with the type IIS enzyme<br>
BsmF I; however any enzyme that cuts DNA at a distance from its binding site<br>
can be used including but not limited to the enzymes listed in Table I.<br>
[0388] Also, in the above example, the nucleotide immediately preceding the<br>
SNP site was not a guanine on the strand that was filled in. This eliminated any<br>
effects: of the alternative cutting properties of the type IIS restriction enzyme to be<br>
removed. For example, at SNP TSC0837969, the nucleotide upstream of the SNP<br>
site on the sense strand was an adenine. If BsmF I displayed alternate cutting<br>
properties, the following overhangs would be generated for the adenine allele and<br>
the guanine allele:<br>
G allele -11/15 Cut	5' TTA<br>
3'AAT	T	C	A C<br>
Overhang position	0	12	3<br>
G allele after fill-in	5'TTA	A G*<br>
3'AAT	T	C A C<br>
Overhang position	0	12	3<br>
A allele 11/15 Cut	5'TTA<br>
3'AAT	T T	A C<br>
Overhang position	0	12	3<br>
A allele after fill-in	5'TTA	A A T G*<br>
3'AAT	T T	A C<br>
Overhang position	0	12	3<br><br>
[0389] For the guanine allele, the first position in the overhang would be filled<br>
in with dATP, which would allow the polymerase to incorporate ddGTP at<br>
position 2 complementary to the overhang. There would be no detectable<br>
difference between molecules cut at the 10/14 position or molecules cut at the<br>
11/15 position.<br>
[0390] For the adenine allele, the first position complementary to the<br>
overhang would be filled in with dATP, the second position would be filled in<br>
with dATP, the third position would be filled in with dTTP, and the fourth<br>
position would be filled in with ddGTP. There would be no difference in the<br>
molecular weights between molecules cut at 10/14 or molecules cut at 11/15. The<br>
only differences would correspond to whether the DNA molecules contained an<br>
adenine at the variable site or a guanine at the variable site.<br>
[0391] As seen in FIG. 11, positioning the annealing region of the first primer<br>
allows multiple SNPs to be analyzed in a single lane of a gel. Also, when using<br>
the same nucleotide with the same dye, a single fill-in reaction can be performed.<br>
In this example, 6 SNPs were analyzed in one lane. However, any number of<br>
SNPs including but not limited to 1,2,3,4, 5, 6,7, 8,9,10,11,12,13,14,15,16,<br>
17,18,19,20,21,22,23,24,25,26,27,28,29,30, 30-40,41-50, 51-60, 61-70,<br>
71-80,81-100,101-120,121-140,141-160,161-180,181-200, and greater than<br>
200 can be analyzed in a single reaction.<br>
[0392] Furthermore, one labeled nucleotide used to detect both alleles can be<br>
mixed with a second labeled nucleotide used to detect a different set of SNPs<br>
provided that neither of the nucleotides that are labeled occur immediately before<br>
the variable site (complementary to nucleotide at position 0 of the 11/15 cut). For<br>
example, suppose SNP X can be guanine or thymidine at the variable site and has<br>
the following 5' overhang generated after digestion with BsmF I:<br>
SNP X 10/14	5'TTGAC<br>
G allele	3'AACTG C A C	T<br>
Overhang position 12	3	4<br><br><br><br>
Overhang position 0	12<br>
[0394] Now suppose SNP Y can be adenine or thymidine and has the<br>
following 5' overhangs generated after digestion with BsmF I.<br><br>
[0395] After fill-in with labeled ddATP and unlabeled dCTP, dGTP, and<br>
dTTP, the following molecules would be generated:<br><br><br><br>
[0396] In this example, labeled ddGTP and labeled ddATP are used to<br>
determine the identity of both alleles of SNP X and SNP Y respectively. The<br>
nucleotide immediately preceding (the complementary nucleotide to position 0 of<br>
the overhang from the 11/15 cut SNP X is not guanine or adenine on the strand<br>
that is filled-in. Likewise, the nucleotide immediately preceding SNPY is not<br>
guanine or adenine on the strand that is filled-in. This allows the fill-in reaction<br>
for both SNPs to occur in a single reaction with labeled ddGTP, labeled ddATP,<br>
and unlabeled dCTP and dTTP. This reduces the number of reactions that need to<br>
be performed and increases the number of SNPs that can be analyzed in one<br>
reaction.<br>
[0397] The first primers for each SNP can be designed to anneal at different<br>
distances from the locus of interest, which allows the SNPs to migrate at different<br>
positions on the gel. For example, the first primer used to amplify SNP X can<br>
anneal at 30 bases from the locus of interest, and the first primer used to amplify<br>
SNP Y can anneal at 35 bases from the locus of interest. Also, the nucleotides can<br>
be labeled with fluorescent dyes that emit at spectrums that do not overlap. After<br>
running the gel, the gel can be scanned at one wavelength specific for one dye.<br>
Only those molecules labeled with that dye will emit a signal. The gel then can be<br>
scanned at the wavelength for the second dye. Only those molecules labeled with<br><br>
that dye will emit a signal. This method allows maximum compression for the<br>
number of SNPs that can be analyzed in a single reaction.<br>
[0398] in this example, the nucleotide preceding the variable site on the<br>
strand that -was filled-in is not be adenine or guanine. This method can work with<br>
any combination of labeled nucleotides, and the skilled artisan would understand<br>
which labeling reactions can be mixed and those that can not For instance, if one<br>
SNP is labeled with thymidine and a second SNP is labeled with cytosine, the<br>
SNPs can be labeled in a single reaction if the nucleotide immediately preceding<br>
each variable site is not thymidine or cytosine on the sense strand and the<br>
nucleotide immediately after the variable site is not thymidine or cytosine on the<br>
sense strand.<br>
[0399] This method allows the signals from one allele to be compared to the<br>
signal from a second allele without the added complexity of determining the<br>
degree of alternate cutting, or having to correct for the quantum coefficients of the<br>
dyes. This method is especially useful when trying to quantitate a ratio for one<br>
allele to another. For example, this method is useful for detecting chromosomal<br>
abnormalities. The ratio of alleles at a heterozygous site is expected to be about<br>
1:1 (one A allele and one G allele). However, if an extra chromosome is present<br>
the ratio is expected to be about 1:2 (one A allele and 2 G alleles or 2 A alleles<br>
and 1 G allele). This method is especially useful when trying to detect fetal DNA<br>
in the presence of maternal DNA.<br>
[0400] In addition, this method is useful for detecting two genetic signals in<br>
one sample. For example, this method can detect mutant cells in the presence of<br>
wild type cells (see Example 5). If a mutant cell contains a mutation in the DNA<br>
sequence of a particular gene, this method can be used to detect both the mutant<br>
signal and the wild type signal. This method can be used to detect the mutant<br>
DNA sequence in the presence of the wild type DNA sequence. The ratio of<br>
mutant DNA to wild type DNA can be quantitated because a single nucleotide<br>
labeled with one signal generating moiety is used.<br><br>
EXAMPLE 5<br>
[0401] Non-invasive methods for the detection of various types of cancer have<br>
the potential to reduce morbidity and mortality from the disease. Several<br>
techniques for the early detection of colorectal tumors have been developed<br>
including colonoscopy, barium enemas, and sigmoidoscopy but are limited in use<br>
because the techniques are invasive, which causes a low rate of patient<br>
compliance. Non-invasive genetic tests may be useful in identifying early stage<br>
colorectal tumors.<br>
[0402] In 1991, researchers identified the Adenomatous Polyposis Coli gene<br>
(APC), which plays a critical role in the formation of colorectal tumors (Kinzler et<br>
at. Science 253:661-665,1991). The APC gene resides on chromosome 5q21-22<br>
and a total of 15 exons code for an RNA molecule of 8529 nucleotides, which<br>
produces a 300 Kd APC protein. The protein is expressed in numerous cell types<br>
and is essential for cell adhesion.<br>
[0403] Mutations in the APC gene generally initiate colorectal neoplasia<br>
(Tsao, J. et al, Am, J. Pathol. 145:531-534,1994). Approximately 95% of the<br>
mutations in the APC gene result in nonsense/frameshift mutations. The most<br>
common mutations occur at codons 1061 and 1309; mutations at these codons<br>
account for 1/3 of all germline mutations. With regard to somatic mutations, 60%<br>
occur within codons 1286-1513, which is about 10% of the coding sequence.<br>
This region is termed the mutation Cluster Region (MCR). Numerous types of<br>
mutations have been identified in the APC gene including nucleotide substitutions<br>
(see Table III), splicing errors (see Table IV), small deletions (see Table V),<br>
small insertions (see Table VI), small insertions/deletions (see Table VII), gross<br>
deletions (see Table VIII), gross insertions (see Table DC), and complex<br>
rearrangements (see Table X).<br>
[0404] Researchers have attempted to identify cells harboring mutations in the<br>
APC gene in stool samples (Traverso, G. et al, New England Journal of<br>
Medicine, Vol 346:311-320,2002). While APC mutations are found in nearly all<br>
tumors, about 1 in 250 cells in the stool sample has a mutation in the APC gene;<br><br>
most of the cells are normal cells that have been shed into the feces. Furthermore,<br>
human DNA represents about one-billionth of the total DNA found in stool<br>
samples; the majority of DNA is bacterial. The technique employed by Traverso<br>
et al. only detects mutations that result in a truncated protein.<br>
[0405] As discussed above, numerous mutations in the APC gene have been<br>
implicated in the formation of colorectal tumors. Thus, there still exists a need for<br>
a highly sensitive, non-invasive technique for the detection of colorectal tumors.<br>
Below, methods are described for detection of two mutations in the APC gene.<br>
However, any number of mutations can be analyzed using the methods described<br>
herein.<br>
Preparation of Template DNA<br>
[0406] The template DNA is purified from a sample containing colon cells<br>
including but not limited to a stool sample. The template DNA is purified using<br>
the procedures described by Ahlquist et al. (Gastroenterology, 119:1219-1227,<br>
2000). If stool samples are frozen, the samples are thawed at room temperature,<br>
and homogenized with an Exactor stool shaker (Exact Laboratories, Maynard,<br>
Mass.) Following homogenization, a 4 gram stool equivalent of each sample is<br>
centrifuged at 2536 x g for 5 minutes. The samples are centrifuged a second time<br>
at 16,500 x g for 10 minutes. Supernatants are incubated with 20 μl of RNase (0.5<br>
mg per milliliter) for 1 hour at 37°C. DNA is precipitated with 1/10 volume of 3<br>
mol of sodium acetate per liter and an equal volume of isopropanol. The DNA is<br>
dissolved in 5 ml of TRIS-EDTA (0.01 mol of Tris per liter (pH 7.4) and 0.001<br>
mole of EDTA per liter.<br>
Design of Primers<br>
[0407] To determine if a mutation resides at codon 1370, the following<br>
primers are used:<br>
First primer:<br>
5' GTGCAAAGGCCTGAATTCCCAGGCACAAAGCTGTTGAA 3'<br>
Second primer:<br><br>
5' TGAAGCGAACTAGGGACTCAGGTGGACTT<br>
[0408] The first primer contains a biotin tag at the extreme 5' end, and the<br>
nucleotide sequence for the restriction enzyme EcoRI. The second primer<br>
contains the nucleotide sequence for the restriction enzyme BsmF I.<br>
[0409] To determine if a small deletion exists at codon 1302, the following<br>
primers are used:<br>
First primer:<br>
5' GATrCCGTAAACGAATTCAGTTCATTATCATCTTTGTC 3'<br>
Second primer:<br>
5' CCATTGTTAAGCGGGACTTCTGCTATTTG 3'<br>
[0410] The first primer has a biotin tag at the 5' end and contains a restriction<br>
enzyme recognition site for EcoRI. The second primer contains a restriction<br>
enzyme recognition site for BsmF I.<br>
PCR Reaction<br>
[0411] The loci of interest are amplified from the template genomic DNA<br>
using the polymerase chain reaction (PCR, U.S. Patent Nos. 4,683,195 and<br>
4,683,202, incorporated herein by reference). The loci of interest are amplified in<br>
separate reaction tubes; they can also be amplified together in a single PCR<br>
reaction. For increased specificity, a "hot-start" PCR reaction is used, e.g. by<br>
using the HotStarTaq Master Mix Kit supplied by QIAGEN (catalog number<br>
203443). The amount of template DNA and primer per reaction are optimized for<br>
each locus of interest but in this example, 40 ng of template human genomic DNA<br>
and 5 μM of each primer are used. Forty cycles of PCR are performed. The<br>
following PCR conditions are used:<br>
(1)	95°C for 15 minutes and 15 seconds;<br>
(2)	37°C for 30 seconds;<br>
(3)	95°C for 30 seconds;<br>
(4)	57°C for 30 seconds;<br>
(5)	95°C for 30 seconds;<br>
(6)	64°C for 30 seconds;<br><br>
(7)	95°C for 30 seconds;<br>
(8)	Repeat steps 6 and 7 thirty nine (39) times;<br>
(9)	72°C for 5 minutes.<br>
[0412] In the first cycle of PCR, the annealing temperature is about the<br>
melting temperature of the 3' annealing region of the second primers, which is<br>
37°C. The annealing temperature in the second cycle of PCR is about the melting<br>
temperature of the 3' region, which anneals to the template DNA, of the first<br>
primer, which is 57°C. The annealing temperature in the third cycle of PCR is<br>
about the melting temperature of the entire sequence of the second primer, which<br>
is 64°C. The annealing temperature for the remaining cycles is 64°C. Escalating<br>
the annealing temperature from TM1 to TM2 to TM3 in the first three cycles of<br>
PCR greatly improves specificity. These annealing temperatures are<br>
representative, and the skilled artisan understands that the annealing temperatures<br>
for each cycle are dependent on the specific primers used.<br>
[0413] The temperatures and times for denaturing, annealing, and extension,<br>
are optimized by trying various settings and using the parameters that yield the<br>
best results.<br>
Purification of Fragment Containing Locus of Interest<br>
[0414] The PCR products are separated from the genomic template DNA.<br>
Each PCR product is divided into four separate reaction wells of a Streptawell,<br>
transparent, High-Bind plate from Roche Diagnostics GmbH (catalog number 1<br>
645 692, as listed in Roche Molecular Biochemicals, 2001 Biochemicals Catalog).<br>
The first primers contain a 5' biotin tag so the PCR products bound to the<br>
Streptavidin coated wells while the genomic template DNA does not The<br>
streptavidin binding reaction is performed using a Thermomixer (Eppendorf) at<br>
1000 rpm for 20 min. at 37°C. Each well is aspirated to remove unbound<br>
material, and washed three times with IX PBS, with gentle mixing (Kandpal et<br>
al.,Nucl. Acids Res. 18:1789-1795 (1990); Kaneoka et al., Biotechniques 10:30-<br>
34 (1991); Green et al.,Nucl. Acids Res. 18:6163-6164 (1990)).<br><br>
[0415] Alternatively, the PCR products are placed into a single well of a<br>
streptavidin plate to perform the nucleotide incorporation reaction in a single well.<br>
Restriction Enzyme Digestion of Isolated Fragments Containing Loci of Interest<br>
[0416] The purified PCR products are digested with the restriction enzyme<br>
BsmF I (New England Biolabs catalog number R0572S), which binds to the<br>
recognition site incorporated into the PCR products from the second primer. The<br>
digests are performed in the Streptawells following the instructions supplied with<br>
the restriction enzyme. After digestion with the appropriate restriction enzyme,<br>
the wells are washed three times with PBS to remove the cleaved fragments.<br>
Incorporation of Labeled Nucleotide<br>
[0417] The restriction enzyme digest described above yields a DNA fragment<br>
with a 5' overhang, which contains the locus of interest and a 3' recessed end. The<br>
5' overhang functions as a template allowing incorporation of a nucleotide or<br>
nucleotides in the presence of a DNA polymerase.<br>
[0418] For each locus of interest, four separate fill in reactions are performed;<br>
each of the four reactions contains a different fluorescently labeled ddNTP<br>
(ddATP, ddTTP, ddGTP, or ddCTP). The following components are added to<br>
each fill in reaction: 1 μl of a fluorescently labeled ddNTP, 0.5 μl of unlabeled<br>
ddNTPs ( 40 uM), which contains all nucleotides except the nucleotide that is<br>
fluorescently labeled, 2 μl of 10X sequenase buffer, 0.25 μl of Sequenase, and<br>
water as needed for a 20ul reaction. The fill are performed in reactions at 40°C<br>
for 10 min. Non-fluorescently labeled ddNTP are purchased from Fermentas Inc.<br>
(Hanover, MD). All other labeling reagents are obtained from Amersham<br>
(Thermo Sequenase Dye Terminator Cycle Sequencing Core Kit, US 79565). In<br>
the presence of fluorescently labeled ddNTPs, the 3' recessed end is extended by<br>
one base, which corresponds to the locus of interest.<br>
[0419] A mixture of labeled ddNTPs and unlabeled dNTPs also can be used<br>
for the fill-in reaction. The "fill in" conditions are as described above except that<br>
a mixture containing 40 μM unlabeled dNTPs, 1 μl fluorescently labeled ddATP,<br><br>
1 μl ftaorescently labeled ddTTP, 1 μl fluorescently labeled ddCTP, and 1 μl<br>
ddGTP are used. The fluorescent ddNTPs are obtained from Amersham (Thermo<br>
Sequenase Dye Terminator Cycle Sequencing Core Kit, US 79565; Amersham<br>
does not publish the concentrations of the fluorescent nucleotides). The locus of<br>
interest is digested with the restriction enzyme BsmF I, which generates a 5'<br>
overhang of four bases. If the first nucleotide incorporated is a labeled ddNTP,<br>
the 3' recessed end is filled in by one base, allowing detection of the locus of<br>
interest. However, if the first nucleotide incorporated is a dNTP, the polymerase<br>
continues to incorporate nucleotides until a ddNTP is filled in. For example, the<br>
first two nucleotides may be filled in with dNTPs, and the third nucleotide with a<br>
ddNTP, allowing detection of the third nucleotide in the overhang. Thus, the<br>
sequence of the entire 5' overhang is determined, which increases the information<br>
obtained from each SNP or locus of interest This type of fill in reaction is<br>
especially useful when detecting the presence of insertions, deletions, insertions<br>
and deletions, rearrangements, and translocations.<br>
[0420] Alternatively, one nucleotide labeled with a single dye is used to<br>
determine the sequence of the locus of interest See Example 4. This method<br>
eliminates any potential errors when using different dyes, which have different<br>
quantum coefficients.<br>
[0421] After labeling, each Streptawell is rinsed with IX PBS (100 μl) three<br>
times. The "filled in" DNA fragments are released from the Streptawells by<br>
digesting with the restriction enzyme EcoRI, according to the manufacturer's<br>
instructions that are supplied with the enzyme. The digestion is performed for 1<br>
hour at 37 °C with shaking at 120 rpm.<br>
Detection of the Locus of Interest<br>
[0422] After release from the streptavidin matrix, the sample is loaded into a<br>
lane of a 36 cm 5% acrylamide (urea) gel (BioWhittaker Molecular Applications,<br>
Long Ranger Run Gel Packs, catalog number 50691). The sample is<br>
electrophoresed into the gel at 3000 volts for 3 min. The gel is run for 3 hours<br><br>
using a sequencing apparatus (Hoefer SQ3 Sequencer). The incorporated labeled<br>
nucleotide is detected by fluorescence.<br>
[0423] To determine if any cells contain mutations at codon 1370 of the APC<br>
gene when separate fill-in reactions are performed, the lanes of the gel that<br>
correspond to the fill-in reaction for ddATP and ddTTP are analyzed. If only<br>
normal cells are present, the lane corresponding to the fill in reaction with ddATP<br>
is a bright signal. No signal is detected for the "fill-in" reaction with ddTTP.<br>
However, if the patient sample contains cells with mutations at codon 1370 of the<br>
APC gene, the lane corresponding to the fill in reaction with ddATP is a bright<br>
signal, and a signal is detected from the lane corresponding to the fill in reaction<br>
with ddTTP. The intensity of the signal from the lane corresponding to the fill in<br>
reaction with ddTTP is indicative of the number of mutant cells in the sample.<br>
[0424] Alternatively, one labeled nucleotide is used to determine the sequence<br>
of the alleles at codon 1370 of the APC gene. At codon 1370, the normal<br>
sequence is AAA, which codes for the amino acid lysine. However, a nucleotide<br>
substitution has been identified at codon 1370, which is associated with colorectal<br>
tumors. Specifically, a change from A to T (AAA-TAA) typically is found at<br>
codon 1370, which results in a stop codon. A single fill-in reaction is performed<br>
using labeled ddATP, and unlabeled dTTP, dCTP, and dGTP. A single nucleotide<br>
labeled with one fluorescent dye is used to determine the presence of both the<br>
normal and mutant DNA sequence that codes for codon 1370. The relevant DNA<br>
sequence is depicted below with the sequence corresponding to codon 1370 in<br>
bold:<br>
5' CCCAAAAGTCCACCTGA<br>
3' GGGTTTTCAGGTGGACT<br>
[0425] After digest with BsmF I, the following overhang is produced:<br>
5'CCC<br>
3'GGG	T T T T<br>
Overhang position	12	3	4<br><br>
[0426] If the patient sample has no cells harboring a mutation at codon 1370,<br>
one signal is seen corresponding to incorporation of labeled ddATP.<br>
5' CCC	A*<br>
3'GGG	T T T T<br>
Overhang position	12 3	4<br>
[0427] However, if the patient sample has cells with mutations at codon 1370<br>
of the APC gene, one signal is seen, which corresponds to the normal sequence at<br>
codon 1370, and a second signal is seen, which corresponds to the mutant<br>
sequence at codon 1370. The signals clearly are identified as they differ in<br>
molecular weight.<br>
Overhang of normal DNA sequence: CCC<br>
GGG T T T T<br>
Overhang position	12	3	4<br>
Normal DNA sequence after fill-in: CCC	A*<br>
GGG T T T T<br>
Overhang position	12	3	4<br>
Overhang of mutant DNA sequence: CCC<br>
GGG	A T	T	T<br>
Overhang position	12	3	4<br>
Mutant DNA sequence after fill-in:	CCC	T A*<br>
GGG	AT	T	T<br>
Overhang position	12	3	4<br>
[0428] Two signals are seen when the mutant allele is present. The mutant<br>
DNA molecules are filled in one base after the wild type DNA molecules. The<br>
two signals are separated using any method that discriminates based on molecular<br><br>
weight One labeled nucleotide (ddATP) is used to detect the presence of both the<br>
wild type DNA sequence and the mutant DNA sequence. This method of labeling<br>
reduces the number of reactions that need to be performed and allows accurate<br>
quantitation for the number of mutant cells in the patient sample. The number of<br>
mutant cells in the sample is used to determine patient prognosis, the degree and<br>
the severity of the disease. This method of labeling eliminates the complications<br>
associated with using different dyes, which have distinct quantum coefficients.<br>
This method of labeling also eliminates errors associated with pipetting reactions.<br>
[0429] To determine if any cells contain mutations at codon 1302 of the APC<br>
gene when separate fill-in reactions are performed, the lanes of the gel that<br>
correspond to the fill-in reaction for ddTTP and ddCTP are analyzed. The normal<br>
DNA sequence is depicted below with sequence coding for codon 1302 in bold<br>
type-face.<br>
Normal Sequence:	5' ACCCTGCAAATAGCAGAA<br>
3' TGGGACGTT TATCGTCT T<br>
[0430] After digest, the following 5' overhang is produced:<br>
5' ACCC<br>
3'TGGG A C G T<br>
Overhang position	12	3	4<br>
[0431] After the fill-in reaction, labeled ddTTP is incorporated.<br>
5' ACCC T*<br>
3'TGGG A C G T<br>
Overhang position	12	3	4<br><br>
[0432] A deletion of a single base of the APC sequence, which typically codes<br>
for codon 1302, has been associated with colorectal tumors. The mutant DNA<br>
sequence is depicted below with the relevant sequence in bold:<br>
Mutant Sequence:	5' ACCCGCAAATAGCAGAA<br>
3' TGGGCGTTTATCGTCTT<br>
After digest:<br>
5'ACC<br>
3'TGG	G C G T<br>
Overhang position	12	3	4<br>
After fill-in:<br>
5' ACC	C*<br>
3'TGG	G C G T<br>
Overhang position	12	3	4<br>
[0433] If mere are no mutations in the APC gene, signal is not detected for the<br>
fill in reaction with ddCTP*, but a bright signal is detected for the fill-in reaction<br>
with ddTTP*. However, if there are cells in the patient sample that have<br>
mutations in the APC gene, signals are seen for the fill-in reactions with ddCTP*<br>
and ddTTP*.<br>
[0434] Alternatively, a single fill-in reaction is performed using a mixture<br>
containing unlabeled dNTPs, fluorescently labeled ddATP, fluorescently labeled<br>
ddTTP, fluorescently labeled ddCTP, and fluorescently labeled ddGTP. If there<br>
is no deletion, labeled ddTTP is incorporated.<br>
5' ACCC T*<br>
3'TGGG A C G T<br>
Overhang position	12	3	4<br><br>
[0435] However, if the T has been deleted, labeled ddCTP* is incorporated.<br>
5' ACC	C*<br>
3'TGG	G C G T<br>
Overhang position	12	3	4<br>
[0436] The two signals are separated by molecular weight because of the<br>
deletion of the thymidine nucleotide. If mutant cells are present, two signals are<br>
generated in the same lane but are separated by a single base pair (this principle is<br>
demonstrated in FIG 9D). The deletion causes a change in the molecular weight<br>
of the DNA fragments, which allows a single fill in reaction to be used to detect<br>
the presence of both normal and mutant cells.<br>
[0437] In the above example, methods for the detection of a nucleotide<br>
substitution and a small deletion are described. However, the methods are used<br>
for the detection of any type of mutation including but not limited to nucleotide<br>
substitutions (see Table IH), splicing errors (see Table IV), small deletions (see<br>
Table V), small insertions (see Table VI), small insertions/deletions (see Table<br>
VII), gross deletions (see Table VIII), gross insertions (see Table IX), and<br>
complex rearrangements (see Table X).<br>
[0438] In addition, the above-described methods are used for the detection of<br>
any type of disease including but not limited to those listed in Table II.<br>
Furthermore, any type of mutant gene is detected using the inventions described<br>
herein including but not limited to the genes associated with the diseases listed in<br>
Table H, BRCA1, BRCA2, MSH6, MSH2, MLH1, RET, PTEN, ATM, H-RAS,<br>
p53, ELAC2, CDH1, APC, AR, PMS2, MLH3, CYP1A1, GSTP1, GSTM1,<br>
AXIN2, CYP19, MET, NAT1, CDKN2A, NQ01, trc8, RAD51, PMS1, TGFBR2,<br>
VHL, MC4R, POMC, NROB2, UCP2, PCSK1, PPARG, ADRB2, UCP3, glurl,<br>
cart, SORBS 1, LEP, LEPR, SIM1, TNF, IL-6, IL-1, IL-2, IL-3, ILIA, TAP2,<br>
THPO, THRB, NBS1, RBM15, LIF, MPL, RUNX1, Her-2, glucocorticoid<br>
receptor, estrogen receptor, thyroid receptor, p21, p27, K-RAS, N-RAS,<br><br>
retinoblastoma protein, Wiskott-Aldrich (WAS) gene, Factor V Leiden, Factor II<br>
(prothrombin), methylene tetrahydrofolate reductase, cystic fibrosis, LDL<br>
receptor, HDL receptor, superoxide dismutase gene, SHOX gene, genes involved<br>
in nitric oxide regulation, genes involved in cell cycle regulation, tumor<br>
suppressor genes, oncogenes, genes associated with neurodegeneration, genes<br>
associated with obesity,. Abbreviations correspond to the proteins as listed on<br>
the Human Gene Mutation Database, which is incorporated herein by reference<br>
(www.archive.uwcm.ac.uk/uwcm) website address active as of February 12,<br>
2003).<br>
[0439] The above-example demonstrates the detection of mutant cells and<br>
mutant alleles from a fecal sample. However, the methods described herein are<br>
used for detection of mutant cells from any biological sample including but not<br>
limited to blood sample, serum sample, plasma sample, urine sample, spinal fluid,<br>
lymphatic fluid, semen, vaginal secretion, ascitic fluid, saliva, mucosa secretion,<br>
peritoneal fluid, fecal sample, body exudates, breast fluid, lung aspirates, cells,<br>
tissues, individual cells or extracts of the such sources that contain the nucleic<br>
acid of the same, and subcellular structures such as mitochondria or chloroplasts.<br>
In addition, the methods described herein are used for the detection of mutant<br>
cells and mutated DNA from any number of nucleic acid containing sources<br>
including but not limited to forensic, food, archeological, agricultural or inorganic<br>
samples.<br>
[0440] The above example is directed to detection of mutations in the APC<br>
gene. However, the inventions described herein are used for the detection of<br>
mutations in any gene that is associated with or predisposes to disease (see Table<br>
XI).<br>
[0441] For example, hypermethylation of the glutathione S-transferase PI<br>
(GSTP1) promoter is the most common DNA alteration in prostrate cancer. The<br>
methylation state of the promoter is determined using sodium bisulfite and the<br>
methods described herein.<br><br>
[0442] Treatment with sodium bisulfite converts unmethylated cytosine<br>
residues into uracil, and leaving the methylated cytosines unchanged. Using the<br>
methods described herein, a first and second primer are designed to amplify the<br>
regions of the GSTP1 promoter that are often methylated. Below, a region of the<br>
GSTP1 promoter is shown prior to sodium bisulfite treatment:<br>
[0443] Before Sodium Bisulfite treatment:<br>
5' ACCGCTACA<br>
3' TGGCGATCA<br>
[0444] Below, a region of the GSTP1 promoter is shown after sodium<br>
bisulfite treatment, PCR amplification, and digestion with the type IIS restriction<br>
en2yme BsmF I:<br>
Unmethylated<br>
5' ACC<br>
3'TGG	U G A T<br>
Overhang position	12	3	4<br>
Methylated<br>
5'ACC<br>
3'TGG	C G A T<br>
Overhang position	12	3	4<br>
[0445] Labeled ddATP, unlabeled dCTP, dGTP, and dTTP are used to fill-in<br>
the 5' overhangs. The following molecules are generated:<br>
Unmethylated<br>
5' ACC	A*<br>
3'TGG	U G A T<br>
Overhang position	12 3 4<br>
Methylated<br><br>
5'ACC	G C T A*<br>
3'TGG	C G A T<br>
Overhang position	1	2	3	4<br>
[0446] Two signals are seen; one corresponds to DNA molecules filled in<br>
with ddATP at position one complementary to the overhang (unmethylated), and<br>
the other corresponds to the DNA molecules filled in with ddATP at position 4<br>
complementary to the overhang (methylated). The two signals are separated<br>
based on molecular weight. Alternatively, the fill-in reactions are performed in<br>
separate reactions using labeled ddGTP in one reaction and labeled ddATP in<br>
another reaction.<br>
[0447] The methods described herein are used to screen for prostate cancer<br>
and also to monitor the progression and severity of the disease. The use of a<br>
single nucleotide to detect both the methylated and unmethylated sequences<br>
allows accurate quantitation and provides a high level of sensitivity for the<br>
methylated sequences, which is a useful tool for earlier detection of the disease.<br>
[0448] The information contained in Tables III-X was obtained from the<br>
Human Gene Mutation Database. With the information provided herein, the<br>
skilled artisan will understand how to apply these methods for determining the<br>
sequence of the alleles for any gene. A large number of genes and their associated<br>
mutations can be found at the following website:<br>
www.archive.uwcm.ac.ukyuwcm.<br><br><br><br><br><br><br><br><br><br>
TABLE V:<br>
APC SMALL DELETIONS<br>
[0449] Bold letters indicate the codon. Undercase letters represent the<br>
deletion. Where deletions extend beyond the coding region, other positional<br><br>
information is provided For example, the abbreviation 5' UTR represents 5'<br>
untranslated region, and the abbreviation E6I6 denotes exon 6/intron 6 boundary.<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
[0450] Having now fully described the invention, it will be understood by<br>
those of skill in the art that the invention can be performed with a wide and<br>
equivalent range of conditions, parameters, and the like, without affecting the<br>
spirit or scope of the invention or any embodiment thereof.<br>
[0451] All documents, e.g., scientific publications, patents and patent<br>
publications recited herein are hereby incorporated by reference in their entirety to<br>
the same extent as if each individual document was specifically and individually<br><br>
indicated to be incorporated by reference in its entirety. Where the document<br>
cited only provides the first page of the document, the entire document is<br>
intended, including the remaining pages of the document.<br><br>
WE CLAIM:<br>
1.	A method for determining a sequence of a locus of interest, said method<br>
comprising:<br>
(a)	amplifying a locus of interest on a template DNA using a first and second primers,<br>
wherein the second primer contains a recognition site for a restriction enzyme that cuts<br>
DNA at a distance from the recognition site such that digestion with the restriction<br>
enzyme generates a 5' overhang containing the locus of interests and wherein the<br>
annealing temperature for cycle 1 of amplification is at about the melting temperature of<br>
the 3' region of the second primer, which anneals to the template DNA, the annealing<br>
temperature for cycle 2 of amplification is at about the melting temperature of the 3'<br>
region of the first primer, which anneals to the template DNA, and the annealing<br>
temperature for the remaining cycles of amplification is at about the melting temperature<br>
of the entire second primer;<br>
(b)	digesting the amplified DNA with the restriction enzyme that recognizes the<br>
recognition site on the second primer;<br>
(c)	incorporating a nucleotide into the digested DNA of (b) by using the 5' overhang<br>
containing the locus of interest as a template; and<br>
(d)	determining the sequence of the locus of interest by determining the sequence of<br>
the DNA of (c).<br>
2.	The method as claimed in claim 1, wherein the template DNA is obtained<br>
from a source selected from the group consisting of a bacterium, fungus, virus, protozoan,<br>
plant, animal and human.<br>
3.	The method as claimed in claim 1 or 2, wherein the template DNA is<br>
obtained from a sample selected from the group consisting of a cell, tissue, blood, serum,<br>
plasma, urine, spinal fluid, lymphatic fluid, semen, vaginal secretion, ascitic fluid, saliva,<br>
mucosa secretion, peritoneal fluid, fecal matter, or body exudates.<br>
4.	The method as claimed in claim 1 or 2, wherein the amplification in (a)<br>
comprises polymerase chain reaction (PCR).<br>
5.	The method as claimed in claim 1 to 4, wherein the restriction enzyme cuts<br>
DNA at a distance from the recognition site.<br><br>
6.	The method as claimed in claim 5, wherein an annealing length of the 3'<br>
region of the second primer is selected from the group consisting of 25-20, 20-15, 15, 14,<br>
13, 12, 11, 10, 9, 8, 7, 6, 5, 4, and less than 4 bases.<br>
7.	The method as claimed in claim 1 to 6, wherein the 3' end of the second<br>
primer is adjacent to the locus of interest.<br>
8.	The method as claimed in claim 5, wherein the recognition site is for a<br>
Type IIS restriction enzyme.<br>
9.	The method as claimed in claim 8, wherein the Type IIS restriction<br>
enzyme is selected from the group consisting of: Acinetobacter lwoffii, Bacillus<br>
laterosporus, Bacillus brevis, —Bacillus cereus, Bacillus megaterium, Bacillus<br>
stearothermophilus, Bacillus stearothermophilus 71, Bacillus stearothermophilus A664,<br>
Bacillus stearothermophilus B61, Bacillus stearothermophilus F, Bacillus species M,<br>
Enterobacter aerogenes, Flavobacterium aquatile, Flavobacterium okeonokoites,<br>
Haemophilus gallinarum, Pseudomonas lemoignei, Saccharopolyspora species,<br>
Streptococcus faecalis ND547, and Streptococcus thermophilus.<br>
10.	The method as claimed in claim 1 to 9, wherein the first primer contains a<br>
recognition site for a restriction enzyme that is different from the recognition site for the<br>
restriction enzyme on the second primer.<br>
11.	The method as claimed in claim 10, comprising digesting the DNA of (c)<br>
with a restriction enzyme that recognizes the recognition site on the first primer.<br>
12.	The method as claimed in claim 1 to 11, wherein the second primer<br>
contains a tag at the 5' terminus.<br><br>
13.	The method as claimed in claim 1 to 11, wherein the first primer contains<br>
a tag at the 5' terminus.<br>
14.	The method as claimed in claim 12 or 13, wherein the tag is used to<br>
separate the amplified DNA from the template DNA.<br>
15.	The method as claimed in claim 14, wherein the tag is used to separate the<br>
amplified DNA containing the incorporated nucleotide from the amplified DNA that does<br>
not contain the incorporated nucleotide.<br>
16.	The method as claimed in claims 12, 13, 14 or 15, wherein the tag is<br>
selected from the group consisting of: radioisotope, fluorescent reporter molecule,<br>
chemiluminescent reporter molecule, antibody, antibody fragment, hapten, biotin,<br>
derivative of biotin, photobiotin, iminobiotin, digoxigenin, avidin, enzyme, acridinium,<br>
sugar, enzyme, apoenzyme, homopolymeric oligonucleotide, hormone, ferromagnetic<br>
moiety, paramagnetic moiety, diamagnetic moiety, phosphorescent moiety, luminescent<br>
moiety, electrochemiluminescent moiety, chromatic moiety, moiety having a detectable<br>
electron spin resonance, electrical capacitance, dielectric constant, and electrical<br>
conductivity and combinations thereof.<br>
17.	The method as claimed in claim 16, wherein a biotin tag is used to<br>
separate amplified DNA from the template DNA using a streptavidin matrix, and where<br>
optionally, the streptavidin matrix is coated on wells of a microtiter plate.<br>
18.	The method as claimed in claim 2 to 25, wherein the incorporation of a<br>
nucleotide in (c) is by a DNA polymerase selected from the group consisting of E. coli<br>
DNA polymerase, Klenow fragment of E. coli DNA polymerase I, T7 DNA polymerase,<br>
T4 DNA polymerase, Taq polymerase, Pfu DNA polymerase, Vent DNA polymerase and<br>
sequenase.<br><br>
19.	The method as claimed in claims 1 to 18, wherein the incorporation of a<br>
nucleotide in (c) comprises incorporation of a labeled nucleotide.<br>
20.	The method as claimed in claims 1 to 19, wherein the incorporation of a<br>
nucleotide in (c) further comprises incorporation of an unlabeled nucleotide.<br>
21.	The method as claimed in claim 19, wherein the labeled nucleotide is<br>
selected from the group consisting of a dideoxynucleotide and deoxynucleotide.<br>
22.	The method as claimed in claim 20, or 21 wherein the labeled nucleotide<br>
is labeled with a molecule selected from the group consisting of radioactive molecule,<br>
fluorescent molecule, antibody, antibody fragment, hapten, carbohydrate, biotin,<br>
derivative of biotin, phosphorescent moiety, luminescent moiety,<br>
electrochemiluminescent moiety, chromatic moiety, moiety having a detectable electron<br>
spin resonance, electrical capacitance, dielectric constant, and electrical conductivity and<br>
combinations thereof.<br>
23.	The method as claimed in claims to 22, wherein the determination of the<br>
sequence of the locus of interest in (d) comprises detecting the nucleotide.<br>
24.	The method as claimed in claims 19 to 23, wherein the determination of<br>
the sequence of the locus of interest in (d) comprises detecting labeled nucleotide.<br>
25.	The method as claimed in claim 23 or 24 ., wherein the detection is by a<br>
method selected from the group consisting of gel electrophoresis, polyacrylamide gel<br>
electrophoresis, fluorescence detection, sequencing, ELISA, mass spectrometry,<br>
fiuorometry, hybridization, microarray, and Southern Blot.<br>
26.	The method as claimed in claim 1 to 25, wherein the locus of interest is<br>
suspected of containing a single nucleotide polymorphism or mutation.<br><br>
27.	The method as claimed in claim 1 to 26, wherein the method is used for<br>
determining the sequence of multiple loci of interest concurrently.<br>
28.	The method as claimed in claim 25, wherein the template DNA comprises<br>
multiple loci from a single chromosome or from multiple chromosomes.<br>
29.	The method as claimed in claim 27 or 28, wherein the loci of interest on<br>
template DNA are amplified in one reaction.<br>
30.	The method as claimed in claim 27 or 28, wherein each of the loci of<br>
interest on template DNA is amplified in a separate reaction.<br>
31.	The method as claimed in claim 30, wherein the amplified DNA are<br>
pooled together prior to digestion of the amplified DNA.<br>
32.	The method as claimed in claim 19 to 31, wherein each of the labeled<br>
DNA in (c) containing a locus of interest is separated prior to (d).<br>
33.	The method as claimed in claim 27 to 32 , wherein at least one of the loci<br>
of interest may contain a single nucleotide polymorphism or a mutation.<br><br>
The invention provides a method useful for determining the sequence of large numbers of loci of interest on a single<br>
or multiple chromosomes. The method utilizes an oligonucleotide primer that contains a recognition site for a restriction enzyme<br>
such that digestion with the restriction enzyme generates a 5' overhang containing the locus of interest. The 5' overhang is used as<br>
a template to incorporate nucleotides, which can be detected. The method is especially amenable to the analysis of large numbers of<br>
sequences, such as single nucleotide polymorphisms, from one sample of nucleic acid.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NC1rb2xucC0yMDA0LWdyYW50ZWQtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">1194-kolnp-2004-granted-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NC1rb2xucC0yMDA0LWdyYW50ZWQtYXNzaWdubWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">1194-kolnp-2004-granted-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NC1rb2xucC0yMDA0LWdyYW50ZWQtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">1194-kolnp-2004-granted-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NC1rb2xucC0yMDA0LWdyYW50ZWQtY29ycmVzcG9uZGVuY2UucGRm" target="_blank" style="word-wrap:break-word;">1194-kolnp-2004-granted-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NC1rb2xucC0yMDA0LWdyYW50ZWQtZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">1194-kolnp-2004-granted-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NC1rb2xucC0yMDA0LWdyYW50ZWQtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">1194-kolnp-2004-granted-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NC1rb2xucC0yMDA0LWdyYW50ZWQtZXhhbWluYXRpb24gcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">1194-kolnp-2004-granted-examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NC1rb2xucC0yMDA0LWdyYW50ZWQtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">1194-kolnp-2004-granted-form 1.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="228062-a-continuous-process-for-the-manufacture-of-a-veil-and-a-veil.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="228064-plate-making-method-and-plate-making-apparatus-for-stencil-printing-and-stencil-printing-machine.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>228063</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1194/KOLNP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>05/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>30-Jan-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>28-Jan-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>17-Aug-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>RAVGEN, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>9241 RUMSEY ROAD, COLUMBIA, MD 21045</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>DHALLAN RAVINDER</td>
											<td>8013 THORNLEY COURT, BETHESDA, MD 20817</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2003/06376</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-02-28</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/360,232</td>
									<td>2002-03-01</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>10/093,618</td>
									<td>2002-03-11</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>60/378,354</td>
									<td>2002-05-08</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/228063-rapid-analysis-of-variations-in-a-genome by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:37:47 GMT -->
</html>
