<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/228098-neisseria-menningitidis-antigens by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:40:05 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 228098:NEISSERIA MENNINGITIDIS ANTIGENS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">NEISSERIA MENNINGITIDIS ANTIGENS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention provides proteins from Naisseria maningitidas (strains A and B), including amino acid sequences, the corresponding nucleotide sequences, expression data and serological data. The proteins are useful antigens for vaccines, immunogenic compositions, and/or diagnostics.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>NEISSER1A MENINGITIDIS ANTIGENS<br>
This invention relates to antigens from the bacterium Neisseria meningitidis.<br>
BACKGROUND<br>
Neisseria meningitidis is a non-motile, gram negative diplococcus human pathogen. It colonises<br>
the pharynx, causing meningitis and, occasionally, septicaemia in the absence of meningitis. It is<br>
closely related to N.gonorrhoeae, although one feature that clearly differentiates meningococcus<br>
from gonococcus is the presence of a polysaccharide capsule that is present in all pathogenic<br>
meningococci.<br>
N.meningitidis causes both endemic and epidemic disease. In the United States the attack rate is<br>
0.6-1 per 100,000 persons per year, and it can be much greater during outbreaks (see Lieberman<br>
et al. (1996) Safety and Immunogenicity of a Serogroups A/C Neisseria meningitidis<br>
Oligosaccharide-Protein Conjugate Vaccine in Young Children. JAMA 275(19):1499-1503;<br>
Schuchat et al (1997) Bacterial Meningitis in the United Stater in 1995. N Engl JMed 337(14):970-<br>
976). In developing countries, endemic disease rates are much higher and during epidemics<br>
incidence rates can reach 500 cases per 100,000 persons per year. Mortality is extremely high, at<br>
10-20% in the United States, and much higher in developing countries. Following the introduction<br>
of the conjugate vaccine against Haemophilia injluenzae, N. meningitidis is the major cause of<br>
bacterial meningitis at all ages in the United States (Schuchat et al (1997) supra).<br>
Based on the organism's capsular polysaccharide, 12 serogroups of N. memngitidis have been<br>
identified. Group A is the pathogen most often implicated in epidemic disease in sub-Saharan<br>
Africa. Serogroups B and C are responsible for the vast majority of cases in the United States and<br>
in most developed countries. Serogroups W135 and Y are responsible for the rest of the cases in<br>
the United States and developed countries. The meningococcal vaccine currently in use is a<br>
tetravalent polysaccharide vaccine composed of serogroups A, C, Y and W135. Although<br>
efficacious in adolescents and adults, it induces a poor immune response and short duration of<br>
protection, and cannot be used in infants [eg. Morbidity and Mortality weekly report, Vol.46, No.<br>
R R 5(1997) This is because polysaccharides are T-cell independent antigens that induce a weak<br><br>
vaccination against H.influenzae, conjugate vaccines against serogroups A and C have been<br>
developed and are at the final stage of clinical testing (Zollinger WD "New and Improved Vaccines<br>
Against Meningococcal Disease" in: New Generation Vaccines, supra, pp. 469-488; Lieberman et<br>
al (1996) supra; Costantino et al (1992) Development and phase I clinical testing of a conjugate<br>
vaccine against meningococcus A and C. Vaccine 10:691-698).<br>
Meningococcus B remains a problem, however. This serotype currently is responsible for<br>
approximately 50% of total meningitis in the United States, Europe, and South America. The<br>
polysaccharide approach cannot be used because the menB capsular polysaccharide is a polymer<br>
of α(2-8)-linked N-acetyl neuraminic acid that is also present in mammalian tissue. This results in<br>
tolerance to the antigen; indeed, if an immune response were elicited, it would be anti-self, and<br>
therefore undesirable. In order to avoid induction of autoimmunity and to induce a protective<br>
immune response, the capsular polysaccharide has, for instance, been chemically modified<br>
substituting the N-acetyl groups, with N-propionyl groups, leaving the specific antigenicity<br>
unaltered (Romero &amp; Outschoorn (1994) Current status of Meningococcal group B vaccine<br>
candidates: capsular or non-capsular? Clin Microbiol Rev 7(4):559-575).<br>
Alternative approaches to menB vaccines have used complex mixtures of outer membrane proteins<br>
(OMPs), containing either the OMPs alone, or OMPs enriched in porins, or deleted of the class 4<br>
OMPs that are believed to induce antibodies that block bactericidal activity. This approach<br>
produces vaccines that are not well characterized. They are able to protect against the homologous<br>
strain, but are not effective at large where there are many antigenic variants of the outer membrane<br>
proteins. To overcome the antigenic variability, multivalent vaccines containing up to nine different<br>
porins have been constructed (eg. Poolman JT (1992) Development of a meningococcal vaccine.<br>
Infect. Agents Dis. 4:13-28). Additional proteins to be used in outer membrane vaccines have been<br>
the opa and ope proteins, but none of these approaches have been able to overcome the antigenic<br>
variability (eg. Ala' Aldeen &amp; Borriello (1996) The meningococcal transferrin-binding proteins 1<br>
and 2 are both surface exposed and generate bactericidal antibodies capable of killing homologous<br>
and heterologous strains. Vaccine 14(l):49-53).<br>
A certain amount of sequence data is available for meningococcal and gonococcal genes and<br>
proteins (eg. EP-A-0467714, WO96/29412), but this is by no means complete. The provision of<br>
further sequences could provide an opportunity to identify secreted or surface-exposed proteins that<br><br>
are presumed targets for the immune system and which are not antigenically variable. For instance,<br>
some of the identified proteins could be components of efficacious vaccines against meningococcus<br>
B. some could be components of vaccines against all meningococcal serotypes, and others could<br>
be components of vaccines against all pathogenic Neisseriae.<br>
THE INVENTION<br>
The invention provides proteins comprising the N.meningitidis amino acid sequences disclosed in<br>
the examples.<br>
It also provides proteins comprising sequences homologous (ie. having sequence identity) to the<br>
N.meningitidis amino acid sequences disclosed in the examples. Depending on the particular<br>
sequence, the degree of sequence identity is preferably greater than 50% (eg. 60%, 70%, 80%, 90%,<br>
95%, 99% or more). These homologous proteins include mutants and allelic variants of the<br>
-sequences disclosed in the examples. Typically, 50% identity or more between two proteins is<br>
considered to be an indication of functional equivalence. Identity between the proteins is preferably<br>
determined by the Smith-Waterman homology search algorithm as implemented in the MPSRCH<br>
program (Oxford Molecular), using an affine gap search with parameters gap open penalty= 12 and<br>
gap extension penalty-1.<br>
The invention further provides proteins comprising fragments of the N.meningitidis aniino acid<br>
sequences disclosed in the examples. The fragments should comprise at least n consecutive amino<br>
acids from the sequences and, depending on the particular sequence, n is 7 or more (eg. 8,10,12,<br>
14,16,18,20 or more). Preferably the fragments comprise an epitope from the sequence.<br>
The proteins of the invention can, of course, be prepared by various means (eg. recombinant<br>
expression, purification from cell culture, chemical synthesis etc.) and in various forms (eg. native,<br>
fusions etc.). They are preferably prepared in substantially pure form (ie. substantially free from<br>
other N.meningitidis or host cell proteins)<br>
According to a further aspect, the invention provides antibodies winch bind to these proteins. These<br>
may be polyclonal or monoclonal and may be produced by any suitable means.<br><br>
According to a further aspect, the invention provides nucleic acid comprising the N.meningitidis<br>
nucleotide sequences disclosed in the examples. In addition, the invention provides nucleic acid<br>
comprising sequences homologous (ie. having sequence identity) to the N.meningitidis nucleotide<br>
sequences disclosed in the examples.<br>
Furthermore, the invention provides nucleic acid which can hybridise to the N.meningitidis nucleic<br>
acid disclosed in the examples, preferably under "high stringency" conditions (eg. 65°C in a<br>
0.1xSSC, 0.5% SDS solution).<br>
Nucleic acid comprising fragments of these sequences are also provided. These should comprise<br>
at least n consecutive nucleotides from the N.meningitidis sequences and, depending on the<br>
particular sequence, n is 10 or more {eg 12, 14, 15, 18, 20,25, 30, 35,40 or more).<br>
According to a further aspect, the invention provides nucleic acid encoding the proteins and protein<br>
fragments of the invention.<br>
It should also be appreciated that the invention provides nucleic acid comprising sequences<br>
complementary to those described above (eg. for antisense or probing purposes).<br>
Nucleic acid according to the invention can, of course, be prepared in many ways (eg. by chemical<br>
synthesis, from genomic or cDNA libraries, from the organism itself etc.) and can take various<br>
forms (eg. single: stranded, double stranded, vectors, probes etc.).<br>
In addition, the term "nucleic acid" includes DNA and RNA, and also their analogues, such as<br>
those containing modified backbones, and also peptide nucleic acids (PNA) etc.<br>
According to a further aspect, the invention provides vectors comprising nucleotide sequences of<br>
the invention (eg. expression vectors) and host cells transformed with such vectors.<br>
According to a further aspect, the invention provides compositions comprising protein, antibody,<br>
and/or nucleic acid according to the invention. These compositions may be suitable as vaccines,<br>
for instance, or as diagnostic reagents, or as immunogenic compositions.<br><br>
The invention also provides nucleic acid, protein, or antibody according to the invention for use<br>
as medicaments (eg. as vaccines) or as diagnostic reagents. It also provides the use of nucleic acid,<br>
protein, or antibody according to the invention in the manufacture of: (i) a medicament for treating<br>
or preventing infection due to Neisserial bacteria; (ii) a diagnostic reagent for detecting the<br>
presence of Neisserial bacteria or of antibodies raised against Neisserial bacteria; and/or (iii) a<br>
reagent which can raise antibodies against Neisserial bacteria. Said Neisserial bacteria may be any<br>
species or strain (such as N.gonorrhoeae) but are preferably N.meningitidis, especially strain A,<br>
strain B or strain C.<br>
The invention also provides a method of treating a patient, comprising administering to the patient<br>
a therapeutically effective amount of nucleic acid, protein, and/or antibody according to the<br>
invention.<br>
According to further aspects, the invention provides various processes.<br>
A process for producing proteins of the invention is provided, comprising the step of culturing a<br>
host: cell according to the invention under conditions which induce protein expression.<br>
A process for producing protein or nucleic acid of the invention is provided, wherein the protein<br>
or nucleic acid is synthesised in part or in whole using chemical means.<br>
A process for detecting polynucleotides of the invention is provided, comprising the steps of: (a)<br>
contacting a nucleic probe according to the invention with a biological sample under hybridizing<br>
conditions to form duplexes; and (b) detecting said duplexes.<br>
A process for detecting proteins of the invention is provided, comprising the steps of: (a) contacting<br>
an antibody according to the invention with a biological sample under conditions suitable for the<br>
formation of an antibody-antigen complexes; and (b) detecting said complexes.<br>
Unlike the sequences disclosed in PCT/IB98/01665, the sequences disclosed in the present<br>
application are believed not to have any significant homologs in N.gonorrhoeae. Accordingly, the<br>
sequences of the present invention also find use in the preparation of reagents for distinguishing<br>
between N.meningitidis and N.gonorrhoeae<br><br>
A summary of standard techniques and procedures which may be employed in order to perform the<br>
invention (eg. to utilise the disclosed sequences for vaccination or diagnostic purposes) follows.<br>
This summary is not a limitation on the invention but, rather, gives examples that may be used, but<br>
are not required<br>
General<br>
The practice of the present invention will employ, unless otherwise indicated, conventional<br>
techniques of molecular biology, microbiology, recombinant DNA, and immunology, which are<br>
within the skill of the art. Such techniques are explained fully in the literature eg. Sambrook<br>
Molecular Cloning; A Laboratory Manual, Second Edition (1989); DNA Cloning, Volumes I and<br>
ii (D.N Glover ed. 1985); Oligonucleotide Synthesis (MJ. Gait ed, 1984); Nucleic Acid<br>
Hybridization (B.D. Hames &amp; SJ. Higgins eds. 1984); Transcription and Translation (B.D. Hames<br>
&amp; S. J. Higgins eds. 1984); Animal Cell Culture (R.I. Freshney ed. 1986); Immobilized Cells and<br>
Enzymes (IRL Press, 1986); B. Perbal, A Practical Guide to Molecular Cloning (1984); the<br>
Methods in Enzymology series (Academic Press, Inc.), especially volumes 154 &amp; 155; Gene<br>
Transfer Vectors for Mammalian Cells (J.H. Miller and M.P. Calos eds. 1987, Cold Spring Harbor<br>
Laboratory); Mayer and Walker, eds. (1987), Immunochemical Methods in Cell and Molecular<br>
Biology (Academic Press, London); Scopes, (1987) Protein Purification: Principles and Practice,<br>
Second Edition (Springer-Verlag, N.Y.), and Handbook of Experimental Immunology, Volumes<br>
I-IV (DM. Weir and C. C. Blackwell eds 1986).<br>
Standard abbreviations for nucleotides and amino acids are used in this specification.<br>
All publications, patents, and patent applications cited herein are incorporated in full by reference.<br>
In particular, the contents of UK patent applications 9800760.2, 9819015.0 and 9822143.5 are<br>
incorporated herein.<br>
Definitions<br>
A composition containing X is "substantially free of Y when at least 85% by weight of the total<br>
X+Y in the composition is X. Preferably, X comprises at least about 90% by weight of the total of<br>
X+Y in the composition, more preferably at least about 95% or even 99% by weight.<br><br>
The term "comprising'" means "including" as well as "consisting" eg. a composition "comprising"<br>
X may consist exclusively of X or may include something additional to X, such as X+Y.<br>
The term "heterologous" refers to two biological components that are not found together in nature.<br>
The components may be host cells, genes, or regulatory regions, such as promoters. Although the<br>
heterologous components are not found together in nature, they can function together, as when a<br>
promoter heterologous to a gene is operably linked to the gene. Another example is where a<br>
Neisserial sequence is heterologous to a mouse host cell. A further examples would be two epitopes<br>
from the same or different proteins which have been assembled in a single protein in an<br>
arrangement not found in nature.<br>
An "origin of replication" is a polynucleotide sequence that initiates and regulates replication of<br>
polynucleotides, such as an expression vector. The origin of replication behaves as an autonomous<br>
unit of polynucleotide replication within a cell, capable of replication under its own control. An<br>
origin of replication may be needed for a vector to replicate in a particular host cell. With certain<br>
origins of replication, an expression vector can be reproduced at a high copy number in the<br>
presence of the appropriate proteins within the cell. Examples of origins are the autonomously<br>
replicating sequences, which are effective in yeast; and the viral T-antigen, effective in COS-7<br>
cells.<br>
A "mutant" sequence is defined as DNA, RNA or amino acid sequence differing from but having<br>
sequence identity with the native or disclosed sequence. Depending on the particular sequence, the<br>
degree of sequence identity between the native or disclosed sequence and the mutant sequence is<br>
preferably greater than 50% (eg. 60%, 70%, 80%, 90%, 95%, 99% or more, calculated using the<br>
Smith-Waterman algorithm as described above). As used herein, an "allelic variant" of a nucleic<br>
acid molecule, or region, for which nucleic acid sequence is provided herein is a nucleic acid<br>
molecule, ox region, that occurs essentially at the same locus in the genome of another or second<br>
isolate, and that, due to natural variation caused by, for example, mutation or recombination, has<br>
a similar but not identical nucleic acid sequence. A coding region allelic variant typically encodes<br>
a protein having similar activity to that Of the protein encoded by the gene to which it is being<br>
compared. An allelic variant can also comprise an alteration in the 5' or 3' untranslaied regions of<br>
the gene, such as in regulatory control regions (eg. see US patent 5,753,235).<br><br>
Expression systems<br>
The Neisserial nucleotide sequences can be expressed in a variety of different expression systems;<br>
for example those used with mammalian cells, baculoviruses, plants, bacteria, and yeast.<br>
i. Mammalian Systems<br>
Mammalian expression systems are known in the art. A mammalian promoter is any DNA<br>
sequence capable of binding mammalian RNA polymerase and initiating the downstream (3*)<br>
transcription of a coding sequence {eg. structural gene) into mRNA. A promoter will have a<br>
transcription initiating region, which is usually placed proximal to the 5' end of the coding<br>
sequence, and a TATA box, usually located 25-30 base pairs (bp) upstream of the transcription<br>
initiation site. The TATA box is thought to direct RNA polymerase II to begin RNA synthesis at<br>
the correct site. A mammalian promoter will also contain an upstream promoter element, usually<br>
located within 100 to 200 bp upstream of the TATA box. An upstream promoter element<br>
determines the rate at which transcription is initiated and can act in either orientation [Sambrook<br>
et al. (1989) "Expression of Cloned Genes in Mammalian Cells." In Molecular Cloning: A<br>
Laboratory Manual, 2nd ed].<br>
Mammalian viral genes are often highly expressed and have a broad host range; therefore sequences<br>
encoding mammalian viral genes provide particularly useful promoter sequences. Examples include<br>
the SV40 early promoter, mouse mammary tumor virus LTR promoter, adenovirus major late<br>
promoter (Ad Ml,P), and herpes simplex virus promoter. In addition, sequences derived from non-<br>
viral genes, such as the murine metallotheionein gene, also provide useful promoter sequences.<br>
Expression may be either constitutive or regulated (inducible), depending on the promoter can be<br>
induced with glucocorticoid in hormone-responsive cells.<br>
The presence of an enhancer element (enhancer), combined with the promoter elements described<br>
above, will usually increase expression levels. An enhancer is a regulatory DNA sequence that can<br>
stimulate transcription up to 1000-fold when linked to homologous or heterologous promoters, with<br>
synthesis beginning at the normal RNA start site. Enhancers are also active when they are placed<br>
upstream or downstream from the transcription initiation site, in either normal or flipped orien-<br>
tation, or at a distance of more than 1000 nucleotides from the promoter [Maniatis et al. (1987)<br>
Science 236:1237; Alberts et al. (1989) Molecular Biology of the Cell, 2nd ed.]. Enhancer elements<br>
derived from viruses may be particularly useful, because they usually have a broader host range.<br><br>
Examples include the SV40 early gene enhancer [Dijkema et al (1985) EMBO J. 4:761] and the<br>
enhancer/promoters derived from the long terminal repeat (LTR) of the Rous Sarcoma Virus<br>
[Gorman et al. (1982b) Proc. Nad. Acad. Sci. 79:6777] and from human cytomegalovirus fBoshart<br>
et al. (1985) Cell 41:521]. Additionally, some enhancers are regulatable and become active only<br>
in the presence of an inducer, such as a hormone or metal ion [Sassone-Corsi and Borelli (1986)<br>
Trends Genet. 2:215; Maniatis et al. (1987) Science 236:1237].<br>
A DNA molecule may be expressed intracellularly in mammalian cells. A promoter sequence may be<br>
directly linked with the DNA molecule, in which case the first amino acid at the N-terminus of the<br>
recombinant protein will always be a methionine, which is encoded by the ATG start codon. If desired,<br>
the N-terminus may be cleaved from the protein by in vitro incubation with cyanogen bromide.<br>
Alternatively, foreign proteins can also be secreted from the cell into the growth media by creating<br>
chirneric DNA molecules that encode a fusion protein comprised of a leader sequence fragment that<br>
provides for secretion of the foreign protein in mammalian cells. Preferably, there are processing<br>
sites encoded between the leader fragment and the foreign gene that can be cleaved either in vivo<br>
or in vitro. The leader sequence fragment usually encodes a signal peptide comprised of<br>
hydrophobic amino acids which direct the secretion of the protein from the cell. The adenovirus<br>
triparite leader is an example of a leader sequence that provides for secretion of a foreign protein<br>
in mammalian cells.<br>
Usually, transcription termination and polyadenylation sequences recognized by mammalian cells<br>
are regulatory regions located 3' to the translation stop codon and thus, together with the promoter<br>
elements, flank the coding sequence. The 3' terminus of the mature mRNA is formed by site-<br>
specific post-transcriptional cleavage and polyadenylation [Bimstiel et al. (1985) Cell 41:349;<br>
Proudfoot and Whitelaw (1988) Termination and 3' end processing of eukaryotic RNA. In<br>
Tratiscription and splicing (ed. BD. Hames and D.M. Glover); Proudfoot (1989) Trends Biochem.<br>
Sci. 14:105). These sequences direct the transcription of an mRNA which can be translated into the<br>
polypeptide encoded by the DNA. Examples of transcription terminater/polyadenylation signals<br>
include those derived from SV40 [Sambrook et al (1989) "Expression of cloned genes in cultured<br>
mammalian cells." In Molecular Cloning: A Laboratory Manual].<br><br>
Usually, the above described components, comprising a promoter, polyadenylation signal, and<br>
transcription termination sequence are put together into expression constructs. Enhancers, introns<br>
with functional splice donor and acceptor sites, and leader sequences may also be included in an<br>
expression construct, if desired. Expression constructs are often maintained in a replicon, such as<br>
an extrachromosomal element {eg. plasmids) capable of stable maintenance in a host, such as<br>
mammalian cells or bacteria. Mammalian replication systems include those derived from animal<br>
viruses, which require trans-acting factors to replicate. For example, plasmids containing the<br>
replication systems of papovaviruses, such as SV40 [Gluzman (1981) Cell 25:175] or<br>
poly omavirus, replicate to extremely high copy number in the presence of the appropriate viral T<br>
antigen. Additional examples of mammalian replicons include those derived from bovine<br>
papillomavirus and Epstein-Barr virus. Additionally, the replicon may have two replicaton systems,<br>
thus allowing it to be maintained, for example, in mammalian cells for expression and in a<br>
prokaryotic host for cloning and amplification. Examples of such mammalian-bacteria shuttle<br>
vectors include pMT2 {Kaufman et al. (1989) Mol. Cell Biol. 9:946] and pHEBO [Shimizu et al.<br>
(1986) Mol. Cell. Biol. 6:1074].<br>
The transformation procedure used depends upon the host to be transformed. Methods for<br>
introduction of heterologous polynucleotides into mammalian cells are known in the art and include<br>
dextran-mediated transfection, calcium phosphate precipitation, polybrene mediated transfection,<br>
protoplast fusion, electroporation, encapsulation of the polynucleotide(s) in liposomes, and direct<br>
microinjection of the DNA into nuclei.<br>
Mammalian cell lines available as hosts for expression are known in the art and include many<br>
immortalized cell lines available from the American Type Culture Collection (ATCC), including<br>
but not limited to, Chinese hamster ovary (CHO) cells, HeLa cells, baby hamster kidney (BHK)<br>
cells, monkey kidney cells (COS), human hepatocellular carcinoma cells (eg. Hep G2), and a<br>
number of other cell lines.<br>
ii. Baculovirus Systems<br>
The polynucleotide encoding the protein can also be inserted into a suitable insect expression vector,<br>
and is operably linked to the control elements within that vector. Vector construction employs<br>
techniques which are known in the art. Generally, the components of the expression system include<br>
a transfer vector, usually a bacterial plasmid, which contains both a fragment of the baculovirus<br><br>
genome, and a convenient restriction site for insertion of the heterologous gene or genes to be<br>
expressed; a wild type baculovirus with a sequence homologous to the baculovirus-specific fragment<br>
in the transfer vector (this allows for the homologous recombination of the heterologous gene in to<br>
the baculovirus genome); and appropriate insect host cells and growth media.<br>
After inserting the DNA sequence encoding the protein into the transfer vector, the vector and the<br>
wild type viral genome are transfected into an insect host cell where the vector and viral genome<br>
are allowed to recombine. The packaged recombinant virus is expressed and recombinant plaques<br>
are identified and purified. Materials and methods for baculovirus/insect cell expression systems<br>
are commercially available in kit form from, inter alia, Invitrogen, San Diego CA ("MaxBac" kit).<br>
These techniques are generally known to those skilled in the art and fully described in Summers<br>
and Smith, Texas Agricultural Experiment Station Bulletin No. 1555 (1987) (hereinafter "Summers<br>
and Smith").<br>
Prior to inserting the DNA sequence encoding the protein into the baculovirus genome, the above<br>
described components, comprising a promoter, leader (if desired), coding sequence of interest, and<br>
transcription termination sequence, are usually assembled into an intermediate transplacement<br>
construct (transfer vector). This construct may contain a single gene and operably linked regulatory<br>
elements; multiple genes, each with its owned set of operably linked regulatory elements; or multiple<br>
genes, regulated by the same set of regulatory elements. Intermediate transplacement constructs are<br>
often maintained in a replicon, such as an extrachromosomal element (eg. plasmids) capable of stable<br>
maintenance in a host, such as a bacterium. The replicon will have a replication system, thus allowing<br>
it to be maintained in a suitable host for cloning and amplification.<br>
Currently, the most commonly used transfer vector for introducing foreign genes into AcNPV is<br>
pAc373. Many other vectors, known to those of skill in the art, have also been designed. These<br>
include, for example, pVL985 (which alters the polyhedrin start codon from ATG to ATT, and<br>
which introduces a BamHI cloning site 32 basepairs downstream from the ATT; see Luckow and<br>
Summers, Virology (1989) 77:31.<br>
The plasmid usually also contains the polyhedrin polyadenylation signal (Miller et al. (1988) Ann.<br>
Rev. Microbiol, 42:177) and a prokaryotic ampicillin-resistance (amp) gene and origin of<br>
replication for selection and propagation in E. coli.<br><br>
Baculovirus transfer vectors usually contain a baculovirus promoter. A baculovirus promoter is any<br>
DNA sequence capable of binding a baculovirus RNA polymerase and initiating the downstream<br>
(5" to 3') transcription of a coding sequence (eg. structural gene) into mRNA. A promoter will have<br>
a transcription initiation region which is usually placed proximal to the 5' end of the coding<br>
sequence. This transcription initiation region usually includes an RNA polymerase binding site and<br>
a transcription initiation site. A baculovirus transfer vector may also have a second domain called<br>
an enhancer, which, if present, is usually distal to the structural gene. Expression may be either<br>
regulated or constitutive.<br>
Structural genes, abundantly transcribed at late times in a viral infection cycle, provide particularly<br>
useful promoter sequences. Examples include sequences derived from the gene encoding the viral<br>
polyhedron protein, Friesen et al., (1986) "The Regulation of Baculovirus Gene Expression," in:<br>
The Molecular Biology of Baculoviruses (ed. Walter Doerfler); EPO Publ. Nos. 127 839 and 155<br>
476; and the gene encoding the plO protein, Vlak et al.; (1988), J. Gen: Virol 69:765.<br>
DNA encoding suitable signal sequences can be derived from genes for secreted insect or<br>
baculovirus proteins, such as the baculovirus polyhedrin gene (Carbonell et al. (1988) Gene,<br>
75:409). Alternatively, since the signals for mammalian cell posttranslational modifications (such<br>
as signal peptide cleavage, proteolytic cleavage, and phosphorylation) appear to be recognized by<br>
insect cells, and the signals required for secretion and nuclear accumulation also appear to be<br>
conserved between the invertebrate cells and vertebrate cells, leaders of non-insect origin, such as<br>
those derived from genes encoding human a-interferon, Maeda et al., (1985), Nature 315:592;<br>
human gastrin-releasing peptide, Lebacq-Verheyden et al., (1988), Molec. Cell. Biol 5:3129;<br>
human IL-2, Smith et al., (1985) Proc. Natl Acad. Sci USA, 52:8404; mouse IL-3, (Miyajima et<br>
al., (1987) Gene 55:273; and human glucocerebrosidase, Martin et al. (1988) DNA, 7:99, can also<br>
be used to provide for secretion in insects.<br>
A recombmant polypeptide or polyprotein may be expressed intracellularly or, if it is expressed<br>
with the proper regulatory sequences, it can be secreted. Good intracellular expression of nonfused<br>
foreign proteins usually requires heterologous genes that ideally have a short leader sequence<br>
containing suitable translation initiation signals preceding an ATG start signal. If desired,<br>
methionine at the N-terminus may be cleaved from the mature protein by in vitro incubation with<br>
cyanogen bromide.<br><br>
Alternatively, recombinant polyproteins or proteins which are not naturally secreted can be secreted<br>
from the insect cell by creating chimeric DNA molecules that encode a fusion protein comprised<br>
of a leader sequence fragment that provides for secretion of the foreign protein in insects. The<br>
leader sequence fragment usually encodes a signal peptide comprised of hydrophobic amino acids<br>
which direct the translocation of the protein into the endoplasmic reticulum.<br>
After insertion of the DNA sequence and/or the gene encoding the expression product precursor<br>
of the protein, an insect cell host is co-transformed with the heterologous DNA of the transfer<br>
vector and the genomic DNA of wild type baculovirus ~ usually by co-transfection. The promoter<br>
and transcription termination sequence of the construct will usually comprise a 2-5kb section of the<br>
baculovirus genome. Methods for introducing heterologous DNA into the desired site in the<br>
baculovirus virus are known in the art. (See Summers and Smith supra; Ju et al. (1987); Smith et<br>
al., Mol Cell. Biol. (1983) 5:2156; and Luckow and Summers (1989)). For example, the insertion<br>
can be into a gene such as the polyhedrin gene, by homologous double crossover recombination;<br>
insertion can also be into a restriction enzyme site engineered into the desired baculovirus gene.<br>
Miller et al., (1989), Bioessays 4:91.The DNA sequence, when cloned in place of the polyhedrin<br>
gene in the expression vector, is flanked both 5' and 3' by polyhedrin-specific sequences and is<br>
positioned downstream of the polyhedrin promoter.<br>
The newly formed baculovirus expression vector is subsequently packaged into an infectious<br>
recombinant baculovirus. Homologous recombination occurs at low frequency (between about 1 %<br>
and about 5%); thus, the majority of the virus produced after cotransfection is still wild-type virus.<br>
Therefore, a method is necessary to identify recombinant viruses. An advantage of the expression<br>
system is a visual screen allowing recombinant viruses to be distinguished. The polyhedrin protein,<br>
which is produced by the native virus, is produced at very high levels in the nuclei of infected cells<br>
at late times after viral infection. Accumulated polyhedrin protein forms occlusion bodies that also<br>
contain embedded particles. These occlusion bodies, up to 15 μm in size, are highly refractile,<br>
giving them a bright shiny appearance that is readily visualized under the light microscope. Cells<br>
infected with recombinant viruses lack occlusion bodies. To distinguish recombinant virus from<br>
wild-type virus, the transfection supernatant is plaqued onto a monolayer of insect cells by<br>
techniques known to those skilled in the art. Namely, the plaques are screened under the light<br>
microscope for the presence (indicative of wild-type virus) or absence (indicative of recombinant<br><br>
virus) of occlusion bodies. "Current Protocols in Microbiology" Vol. 2 (Ausubel et al. eds) at 16.8<br>
(Supp. 10, 1990); Summers and Smith, supra; Miller et al. (1989).<br>
Recombinant baculovirus expression vectors have been developed for infection into several insect<br>
cells. For example, recombinant baculoviruses have been developed for, inter alia: Aedes aegypti<br>
 Autographa californica, Bombyx mori, Drosophila melanogaster, Spodoptera frugiperda, and<br>
Trichoplusia ni (WO 89/046699; Carbonell et al., (1985) J. Virol. J
527:718; Smith et al., (1983) Mol. Cell. Biol 5:2156; and see generally, Fraser, et al. (1989) In<br>
Vitro Cell. Dev. Biol. 25:225).<br>
Cells and cell culture media are commercially available for both direct and fusion expression of<br>
heterologous polypeptides in a baculovirus/expression system; cell culture technology is generally<br>
known to those skilled in the art. See, eg. Summers and Smith supra.<br>
The modified insect cells may then be grown in an appropriate nutrient medium, which allows for<br>
stable maintenance of the plasmid(s) present in the modified insect host Where the expression product<br>
gene is under inducible control, the host may be grown to high density, and expression induced.<br>
Alternatively, where expression is constitutive, the product will be continuously expressed into the<br>
medium and the nutrient medium must be continuously circulated, while removing the product of<br>
interest and augmenting depleted nutrients. The product may be purified by such techniques as<br>
chromatography., eg. HPLC, affinity chromatography, ion exchange chromatography, etc.;<br>
electrophoresis; density gradient centrifugation; solvent extraction, or the like. As appropriate, the<br>
product may be further purified, as required, so as to remove substantially any insect proteins which<br>
are also secreted in the medium or result from lysis of insect cells, so as to provide a product which<br>
is at least substantially free of host debris, eg. proteins, lipids and polysaccharidcs.<br>
In order to obtain protein expression, recombinant host cells derived from the transformants are<br>
incubated under conditions which allow expression of the recombinant protein encoding sequence.<br>
These conditions will vary, dependent upon the host cell selected. However, the conditions are<br>
readily ascertainable to those of ordinary skill in the art, based upon what is known in the art.<br>
iii. Plant Systems<br>
There are many plant cell culture and whole plant genetic expression systems known in the art.<br>
Exemplary plant cellular genetic expression systems include those described in patents, such as:<br><br>
US 5,693,506; US 5,659,122; and US 5,608,143. Additional examples of genetic expression in<br>
plant cell culture has been described by Zenk, Phytochemistry 30:3861-3863 (1991). Descriptions<br>
of plant protein signal peptides may be found in addition to the references described above in<br>
Vaulcombe et al., Mol. Gen. Genet. 209:33-40 (1987); Chandler et al., Plant Molecular Biology<br>
3:407-418 (1984); Rogers, J. Biol. Chem. 260:3731-3738 (1985); Rothstein et al., Gene 55:353-356<br>
(1987); Whittier et al., Nucleic Acids Research 15:2515-2535 (1987); Wirsel et al., Molecular<br>
Microbiology 3:3-14 (1989); Yu et al., Gene 122:247-253 (1992). A description of the regulation<br>
of plant gene expression by the phytohormone, gibberellic acid and secreted enzymes induced by<br>
gibberellic acid can be found in R.L. Jones and J. MacMillin, Gibberellins: in: Advanced Plant<br>
Physiology,. Malcolm B. Wilkins, ed., 1984 Pitman Publishing Limited, London, pp. 21-52.<br>
References that describe other metabolically-regulaled genes: Sheen, Plant Cell, 2:1027-<br>
1038(1990); Maas et al., EMBOJ. 9:3447-3452 (1990); Benkel and Hickey, Proc. Natl. Acad. Sci.<br>
84:1337-1339(1987)<br>
Typically, using techniques known in the art, a desired polynucleotide sequence is inserted into an<br>
expression cassette comprising genetic regulatory elements designed for operation in plants. The<br>
expression cassette is inserted into a desired expression vector with companion sequences upstream<br>
and downstream from the expression cassette suitable for expression in a plant host. The<br>
companion sequences will be of plasmid or viral origin and provide necessary characteristics to the<br>
vector to permit the vectors to move DNA from an original cloning host, such as bacteria, to the<br>
desired plant host. The basic bacterial/plant vector construct will preferably provide a broad host<br>
range prokaryote replication origin; a prokaryote selectable marker; and, for Agrobacterium<br>
transformations, T DNA sequences for Agrobacterium-mediated transfer to plant chromosomes.<br>
Where the heterologous gene is not readily amenable to detection, the construct will preferably also<br>
have a selectable marker gene suitable for determining if a plant cell has been transformed. A<br>
general review of suitable markers, for example for the members of the grass family, is found in<br>
Wilmink and Dons, 1993, Plant Mol. Biol. Reptr, 11 (2):I65-185.<br>
Sequences suitable for permitting integration of the heterologous sequence into the plant genome<br>
are also recommended. These might include transposon sequences and the like for homologous<br>
recombination as well as Ti sequences which permit random insertion of a heterologous expression<br>
cassette into a plant genome. Suitable prokaryote selectable markers include resistance toward<br><br>
antibiotics such as ampicillin or tetracycline. Other DNA sequences encoding additional functions<br>
may also be present in the vector, as is known in the art.<br>
The nucleic acid molecules of the subject invention may be included into an expression cassette<br>
for expression of the protein(s) of interest. Usually, there will be only one expression cassette,<br>
although two or more are feasible. The recombinant expression cassette will contain in addition<br>
to the heterologous protein encoding sequence the following elements, a promoter region, plant 5'<br>
untranslated sequences, initiation codon depending upon whether or not the structural gene comes<br>
equipped with one, and a transcription and translation termination sequence. Unique restriction<br>
enzyme sites at the 5' and 3' ends of the cassette allow for easy insertion into a pre-existing vector.<br>
A heterologous coding sequence may be for any protein relating to the present invention. The<br>
sequence encoding the protein of interest will encode a signal peptide which allows processing and<br>
translocation of the protein, as appropriate, and will usually lack any sequence which might result<br>
in the binding of the desired protein of the invention to a membrane. Since, for the most part, the<br>
transcriptional initiation region will be for a gene which is expressed and translocated during<br>
germination, by employing the signal peptide which provides for translocation, one may also<br>
provide for translocation of the protein of interest. In this way, the protein(s) of interest will be<br>
translocated from the cells in which they are expressed and may be efficiently harvested. Typically<br>
secretion in seeds are across the aleurone or scutellar epithelium layer into the endosperm of the<br>
seed. While it is not required that the protein be secreted from the cells in which the protein is<br>
produced, this facilitates the isolation and purification of the recombinant protein.<br>
Since the ultimate expression of the desired gene product will be in a eucaryotic cell it is desirable<br>
to determine whether any portion of the cloned gene contains sequences which will be processed<br>
out as introns by the host's splicosome machinery. If so, site-directed mutagenesis of the "intron"<br>
region may be conducted to prevent losing a portion of the genetic message as a false intron code,<br>
Reed and Maniatis, Cell 41:95-105,1985.<br>
The vector can be microinjected directly into plant cells by use of micropipettes to mechanically<br>
transfer the recombinant DNA. Crossway, Mol. Gen. Genet, 202:179-185, 1985. The genetic<br>
material may also be transferred into the plant cell by using polyethylene glycol, Krens, et al.,<br>
Nature, 296, 72-74, 1982. Another method of introduction of nucleic acid segments is high<br><br>
velocity ballistic penetration by small particles with the nucleic acid either within the matrix of<br>
small beads or particles, or on the surface, Klein, et ah. Nature, 327, 70-73, 1987 and Knudsen and<br>
Muller, 1991, Planta, 185:330-336 teaching particle bombardment of barley endosperm to create<br>
transgenic barley. Yet another method of introduction would be fusion of protoplasts with other<br>
entities, either minicells, cells, lysosomes or other fusible lipid-surfaced bodies, Fraley, et al., Proc<br>
Nad. Acad. Sci. USA,79, 1859-1863, 1982.<br>
The vector may also be introduced into the plant cells by electroporation. (Fromm et al., Proc. Natl<br>
Acad Sci. USA 82:5824, 1985). In this technique, plant protoplasts are electroporated in the<br>
presence of plasmids containing the gene construct. Electrical impulses of high field strength<br>
reversibly permeabilize biomembranes allowing the introduction of the plasmids. Electroporated<br>
plant protoplasts reform the cell wall, divide, and form plant callus.<br>
All plants from which protoplasts can be isolated and cultured to give whole regenerated plants can<br>
be transformed by the present invention so that whole plants are recovered which contain the<br>
transferred gene. It is known that practically all plants can be regenerated from cultured cells or<br>
tissues, including but .not limited to all major species of sugarcane, sugar beet, cotton, fruit and<br>
other trees, legumes and vegetables. Some suitable plants include, for example, species from the<br>
genera Fragaria, Lotus, Medicago, Onobrychis, Trifolium, Trigonella, Vigna, Citrus, Linum,<br>
Geranium, Manihot, Daucus, Arabidopsis, Brassica, Raphanus, Sinapis, Atropa, Capsicum,<br>
Datura, Hyoscyamus, Lycopersion, Nicotiana, Solarium, Petunia, Digitalis, Mqjorana, Cichorium,<br>
Helianthus, Lactuca, Bromus, Asparagus, Antirrhinum, Hererocallis, Nemesia, Pelargonium,<br>
Panicum, Pennisetum, Ranunculus, Senecio, Salpiglossis, Cucumis, Browaalia, Glycine, Lolium,<br>
Zea, Triticum, Sorghum, and Datura.<br>
Means for regeneration vary from species to species of plants, but generally a suspension of<br>
transformed protoplasts containing copies of the heterologous gene is first provided. Callus tissue<br>
is formed and shoots may be induced from callus and subsequently rooted. Alternatively, embryo<br>
formation can be induced from the protoplast suspension. These embryos germinate as natural<br>
embryos to form plants. The culture media will generally contain various amino acids and<br>
hormones, such as auxin and cytokinins. It is also advantageous to add glutamic acid and proline<br>
to the medium, especially for such species as corn and alfalfa. Shoots and roots normally develop<br>
simultaneously. Efficient regeneration will depend on the medium, on the genotype, and on the<br><br>
history of the culture. If these three variables are controlled, then regeneration is fully reproducible<br>
and repeatable.<br>
In some plaint cell culture systems, the desired protein of the invention may be excreted or<br>
alternatively, the protein may be extracted from the whole plant. Where the desired protein of the<br>
invention is secreted into the medium, it may be collected. Alternatively, the embryos and<br>
embryoless-half seeds or other plant tissue may be mechanically disrupted to release any secreted<br>
protein between cells and tissues. The mixture may be suspended in a buffer solution to retrieve<br>
soluble proteins. Conventional protein isolation and purification methods will be then used to<br>
purify the recombinant protein. Parameters of time, temperature pH, oxygen, and volumes will be<br>
adjusted through routine methods to optimize expression and recovery of heterologous protein.<br>
iv. Bacterial Systems<br>
Bacterial expression techniques are known in the art. A bacterial promoter is any DNA sequence<br>
capable of binding bacterial RNA polymerase and initiating the downstream (3') transcription of<br>
a coding sequence (eg. structural gene) into mRNA. A promoter will have a transcription initiation<br>
region which is usually placed proximal to the 5' end of the coding sequence. This transcription<br>
initiation region usually includes an RNA polymerase binding she and a transcription initiation site.<br>
A bacterial promoter may also have a second domain called an operator, that may overlap an<br>
adjacent RNA polymerase binding site at which RNA synthesis begins. The operator permits<br>
negative regulated (inducible) transcription, as a gene repressor protein may bind the operator and<br>
thereby inhibit transcription of a specific gene. Constitutive expression may occur in the absence<br>
of negative regulatory elements, such as the operator. In addition, positive regulation may be<br>
achieved by a gene activator protein binding sequence, which, if present is usually proximal (51)<br>
to the RNA polymerase binding sequence. An example of a gene activator protein is the catabolite<br>
activator protein (CAP), which helps intiate transcription of the lac operon in Escherichia coli (E.<br>
coli) [Raibaud et al. (1984) Annu. Rev. Genet. 18:113]. Regulated expression may therefore be<br>
either positive or negative, thereby either enhancing or reducing transcription.<br>
Sequences encoding metabolic pathway enzymes provide particularly useful promoter sequences.<br>
Examples include promoter sequences derived from sugar metabolizing enzymes, such as galactose,<br>
lactose (lac) [Chang et al. (1977) Nature 198:1056], and maltose. Additional examples include<br>
promoter sequences derived from biosynthetic enzymes such as tryptophan (trp) [Goeddel et al.<br><br>
(1980) Nuc. Acids Res. 5:4057; Yelverton et al. (1981) Nucl. Acids Res. 9:731; US<br>
patent 4,738,921; EP-A-0036776 and EP-A-0121775]. The g-laotamase (bid) promoter system<br>
[Weissmann (1981) "The cloning of interferon and other mistakes." In Interferon 3 (ed. I. Gresser)],<br>
bacteriophage lambda PL [Shimatake et al. (1981) Nature 292:128] and T5 [US patent 4,689,406]<br>
promoter systems also provide useful promoter sequences.<br>
In addition, synthetic promoters which do not occur in nature also function as bacterial promoters.<br>
For example, transcription activation sequences of one bacterial or bacteriophage promoter may<br>
be joined with the operon sequences of another bacterial or bacteriophage promoter, creating a<br>
synthetic hybrid promoter [US patent 4,551,433]. For example, the tac promoter is a hybrid trp-lac<br>
promoter comprised of both trp promoter and lac operon sequences that is regulated by the lac<br>
repressor [Amann et al. (1983) Gene 25:167; de Boer et al. (1983) Proc. Natl. Acad. Sci. 80:21).<br>
Furthermore, a bacterial promoter can include naturally occurring promoters of non-bacterial origin<br>
that have the ability to bind bacterial RNA polyrrierase and initiate transcription. A naturally<br>
occurring promoter of non-bacterial origin can also be coupled with a compatible RNA polymerase<br>
to produce high levels of expression of some genes in prokaryotes. The bacteriophage T7 RNA<br>
polymerase/promoter system is an example of a coupled promoter system [Studier et al. (1986) J.<br>
Mol. Biol. J89:113; Tabor et al. (1985) Proc Natl. Acad Sci. 52:1074]. In addition, a hybrid<br>
promoter can also be comprised of a bacteriophage promoter and an E. coli operator region (EPO-<br>
A-0 267 851).<br>
In addition to a functioning promoter sequence, an efficient ribosome binding site is also useful for<br>
the expression of foreign genes in prokaryotes. In E. coli, the ribosome binding site is called the<br>
Shine-Dalgarno (SD) sequence and includes an initiation codon (ATG) and a sequence 3-9<br>
nucleotides in length located 3-11 nucleotides upstream of the initiation codon [Shine et al. (1975)<br>
Nature 254:34]. The SD sequence is thought to promote binding of mRNA to the ribosome by the<br>
pairing of bases between the SD sequence and the 3' and of E. coli 16S rRNA [Steitz et al. (1979)<br>
"Genetic signals and nucleotide sequences in messenger RNA." In Biological Regulation and<br>
Development: Gene Expression (ed. R.F. Goldberger)]. To express eukaryotic genes and<br>
prokaryotic genes with weak ribosome-binding site [Sambrook et al. (1989) "Expression of cloned<br>
genes in Escherichia coli." In Molecular Cloning: A Laboratory Manual].<br><br>
A DNA molecule may be expressed intracellularly. A promoter sequence may be directly linked<br>
with the DNA molecule, in which case the first amino acid at the N-terminus will always be a<br>
methionine, which is encoded by the ATG start codon. If desired, methionine at the N-terminus<br>
may be cleaved from the protein by in vitro incubation with cyanogen bromide or by either in vivo<br>
on in vitro incubation with a bacterial methionine N-terminal peptidase (EPO-A-0 219 237).<br>
Fusion proteins provide an alternative to direct expression. Usually, a DNA sequence encoding the<br>
N-terminal portion of an endogenous bacterial protein, or other stable protein, is fused to the 5' end<br>
of heterologous coding sequences. Upon expression, this construct will provide a fusion of the two<br>
amino acid sequences. For example, the bacteriophage lambda cell gene can be linked at the 5'<br>
terminus of a foreign gene and expressed in bacteria. The resulting fusion protein preferably retains<br>
a site for a processing enzyme (factor Xa) to cleave the bacteriophage protein from the foreign gene<br>
[Nagai et al. (1984) Nature 309:810]. Fusion proteins can also be made with sequences from the<br>
iacZ [Jia et al (1987) Gene 60:197], trpE [Allen et al. (1987) J. Biotechnol. 5:93; Makoff et al.<br>
(1989) J. Gen. Microbiol. 135:11], and Chey [EP-A-0 324 647] genes. The DNA sequence at the<br>
junction of the two amino acid sequences may or may not encode a cleavable site. Another example<br>
is a ubiquitin fusion protein. Such a fusion protein is made with the ubiquitin region that preferably<br>
retains a site for a processing enzyme (eg. ubiquitin specific processing-protease) to cleave the<br>
ubiquitin from the foreign protein. Through this method, native foreign protein can be isolated<br>
[Miller et al. (1989) Bio/Technology 7:698].<br>
Alternatively, foreign proteins can also be secreted from the cell by creating chimeric DNA molecules<br>
that encode a fusion protein comprised of a signal peptide sequence fragment mat provides for secretion<br>
of the foreign protein in bacteria [US patent 4,336,336]. The signal sequence fragment usually encodes<br>
a signal peptide comprised of hydrophobic amino acids which direct the secretion of the protein from the<br>
cell. The protein is either secreted into the growth media (gram-positive bacteria) or into the periplasmic<br>
space, located between the inner and outer membrane of the cell (gram-negative bacteria). Preferably<br>
there are processing sites, which can be cleaved either in vivo or in vitro encoded between the signal<br>
peptide fragment and the foreign gene.<br>
DNA encoding suitable signal sequences can be derived from genes for secreted bacterial proteins,<br>
such as the E. coli outer membrane protein gene (ompA) [Masui et al. (1983), in: Experimental<br>
Manipulation of Gene Expression; Ghrayeb et al. (1984) EMBO J. 3:2437] and the E. coli alkaline<br><br>
phosphatase signal sequence (phoA) [Oka et al. (1985) Proc. Nad. Acad. Sci. 82:7212]. As an<br>
additional example, the signal sequence of the alpha-amylase gene from various Bacillus strains<br>
can be used to secrete heterologous proteins from B. subtilis [Palva et al. (1982) Proc. Natl. Acad.<br>
Sci. USA 79:5582; EP-A-0 244 042].<br>
Usually, transcription termination sequences recognized by bacteria are regulatory regions located<br>
3' to the translation stop codon, and thus together with the promoter flank the coding sequence.<br>
These sequences direct the transcription of an mRNA which can be translated into the polypeptide<br>
encoded by the DNA. Transcription termination sequences frequently include DNA sequences of<br>
about SO nucleotides capable of forming stem loop structures that aid in terminating transcription.<br>
Examples include transcription termination sequences derived from genes with strong promoters,<br>
such as the trp gene in E. coli as well as other biosynthetic genes.<br>
Usually, the above described components, comprising a promoter, signal sequence (if desired),<br>
coding sequence of interest, and transcription termination sequence, are put together into expression<br>
constructs. Expression constructs are often maintained in a replicon, such as an extrachromosomal<br>
element (eg. plasmids) capable of stable maintenance in a host, such as bacteria. The replicon will<br>
have a replication system, thus allowing it to be maintained in a prokaryotic host either for<br>
expression or for cloning and amplification. In addition, a replicon may be either a high or low<br>
copy number plasmid. A high copy number plasmid will generally have a copy number ranging<br>
from about 5 to about 200, and usually about 10 to about 150. A host containing a high copy<br>
number plasmid will preferably contain at least about 10, and more preferably at least about 20<br>
plasmids. Either a high or low copy number vector may be selected, depending upon the effect of<br>
the vector and the foreign protein on the host.<br>
Alternatively, the expression constructs can be integrated into the bacterial genome with an<br>
integrating vector. Integrating vectors usually contain at least one sequence homologous to the<br>
bacterial chromosome that allows the vector to integrate. Integrations appear to result from<br>
recombinations between homologous DNA in the vector and the bacterial chromosome. For<br>
example, integrating vectors constructed with DNA from various Bacillus strains integrate into the<br>
Bacillus chromosome (EP-A- 0 127 328). Integrating vectors may also be comprised of<br>
bacteriophage or transposon sequences.<br><br>
Usually, extrachromosomal and integrating expression constructs may contain selectable markers<br>
to allow for the selection of bacterial strains that have been transformed. Selectable markers can<br>
be expressed in the bacterial host and may include genes which render bacteria resistant to drugs<br>
such as ampicillin, chloramphenicol, erythromycin, kanamycin (neomycin), and tetracycline<br>
[Davies et al. (1978) Annu. Rev.,, Microbiol. 32:469). Selectable markers may also include<br>
biosynthetic genes, such as those in the histidine, tryptophan, and leucine biosynthetic pathways.<br>
Alternatively, some of the above described components can be put together in transformation<br>
vectors. Transformation vectors are usually comprised of a selectable market that is either<br>
maintained in a replicon or developed into an integrating vector, as described above.<br>
Expression and transformation vectors, either extra-chromosomal repiicons or integrating vectors,<br>
have been developed for transformation into many bacteria. For example, expression vectors have<br>
been developed for, inter alia, the following bacteria: Bacillus subtilis [Palva et al. (1982) Proc.<br>
Natl Acad. Sci. USA 79:5582; EP-A-0 036 259 and EP-A-0 063 953; WO 84/04541], Escherichia<br>
coli [Shimatake et al. (1981) Nature 292:128; Amann et al. (1985) Gene 40:183; Studier et al.<br>
(1986) J. Mol. Biol. 189:113; EP-A-0 036 776.EP-A-0 136 829 and EP-A-0 136 907],<br>
Streptococcus cremoris [Powell et al. (1988) Appl. Environ. Microbiol. 54:655]; Streptococcus<br>
lividans [Powell et al. (1988) Appl. Environ. Microbiol. 54:655], Streptomyces lividans [US patent<br>
4,745,056].<br>
Methods of introducing exogenous DNA into bacterial hosts are well-known in the art, and usually<br>
include either the transformation of bacteria treated with CaCl2 or other agents, such as divalent<br>
cations and DMSO. DNA can also be introduced into bacterial cells by electroporation.<br>
Transformation procedures usually vary with the bacterial species to be transformed. See eg.<br>
[Masson et al. (1989) FEMS Microbiol Lett. 60:273; Palva et al (1982) Proc. Natl. Acad Sci. USA<br>
79:5582r EP-A-0 036 259 and EP-A-0 063 953; WO 84/04541, Bacillus], (Miller et al. (1988)<br>
Proc. Natl. Acad Sci. 55:856; Wang el al. (1990) J. Bacteriol 172:949, Campylobacter], [Cohen<br>
et al (1973) Proc. Natl. Acad. Sci. 69:2110; Dower et al. (1988) Nucleic Acids Res. 16:6121;<br>
Kushner (1978) "An improved method for transformation of Escherichia coli with ColE1-derived<br>
plasmids. In Genetic Engineering: Proceedings of the International Symposium on Genetic<br>
Engineering (eds. H.W. Boyer and S. Nicosia); Mandel et al. (1970) J. Mol. Biol. 53:159; Taketo<br>
(1988) Biochim. Biophys. Acta 949:318; Escherichia], [Chassy et al (1987) FEMS Microbiol. Lett.<br><br>
44:173 Lactobacillus]; [Fiedler et al. (1988)Anal. Biochem 770:38, Pseudomonas]; [Augustin et<br>
al. (1990) FEMS Microbiol. Lett. 66:203, Staphyiococcus], [Barany et al. (1980) J. Bacteriol<br>
144:69%; Harlander (1987) "Transformation of Streptococcus lactis by electroporation, in:<br>
Streptococal Genetics (ed. J. Ferretti and R. Curtiss III); Perry et al. (1981) Infect, lmmun<br>
32:1295; Powell et al. (1988)Appl. Environ. Microbiol. 54:655; Somkuti et al. (1987) Proc. 4th<br>
Evr. Cong. Biotechnology 7:412, Streptococcus].<br>
v. Yeast Expression<br>
Yeast expression systems are also known to one of ordinary skill in the art. A yeast promoter is any<br>
DNA sequence capable of binding yeast RNA polymerase and initiating the downstream (3')<br>
transcription of a coding sequence (eg. structural gene) into mRNA. A promoter will have a<br>
transcription initiation region which is usually placed proximal to the 5' end of the coding sequence.<br>
This transcription initiation region usually includes an RNA polymerase binding site (the "TATA<br>
Box") and a transcription initiation site. A yeast promoter may also have a second domain called<br>
an upstream activator sequence (UAS), which, if present, is usually distal to the structural gene.<br>
The UAS permits regulated (inducible) expression. Constitutive expression occurs in the absence<br>
of a UAS. Regulated expression may be either positive or negative, thereby either enhancing or<br>
reducing transcription.<br>
Yeast is a fermenting organism with an active metabolic pathway, therefore sequences encoding<br>
enzymes in the metabolic pathway provide particularly useful promoter sequences. Examples<br>
include alcohol dehydrogenase (ADH) (EP-A-0 284 044), enolase, glucokinase, glucoses-<br>
phosphate isomerase, glyceraldehyde-3-phosphate-dehydrogenase (GAP or GAPDH), hexokinase,<br>
phosphofructokinase, 3-phosphoglycente mutase, and pyruvate kinase (PyK) (EPO-A-0 329 203).<br>
The yeast PHO5 gene, encoding acid phosphatase, also provides useful promoter sequences<br>
[Myanohara et al (1983) Proc. Natl Acad. Sci. USA 80:1].<br>
In addition, synthetic promoters which do not occur in nature also function as yeast promoters. For<br>
example, UAS sequences of one yeast promoter may be joined with the transcription activation<br>
region of another yeast promoter, creating a synthetic hybrid promoter. Examples of such hybrid<br>
promoters include the ADH regulatory sequence linked to the GAP transcription activation region<br>
(US Patent Nos. 4,876,197 and 4,880,734). Other examples of hybrid promoters include promoters<br>
which consist of the regulatory sequences of either the ADH2, GAL4, GAL JO, OR PHO5 genes,<br><br>
combined with the transcriptional activation region of a glycolytic enzyme gene such as GAP or<br>
PyK (EP-A-0 164 556). Farthermore, a yeast promoter can include naturally occurring promoters<br>
of non-yeast origin that have the ability to bind yeast RNA polymerase and initiate transcription.<br>
Examples of such promoters include, inter alia, [Cohen et al. (1980) Proc. Natl. Acad. Sci. USA<br>
77:1078; Henikoff et al. (1981) Nature 283:835; Hollenberg et al. (1981) Curr. Topics Microbiol.<br>
Immunol 96:119; Hollenberg et al. (1979) "The Expression of Bacterial Antibiotic Resistance<br>
Genes in the Yeast Saccharomyces cerevisiae," in: Plasmids of Medical, Environmental and<br>
Commercial Importance (eds. K.N. Timmis and A. Puhler); Mercerau-Puigalon et al. (1980) Gene<br>
/7:163; Panthier et al. (1980) Curr. Genet. 2:109;].<br>
A DNA molecule may be expressed intracellularly in yeast. A promoter sequence may be directly<br>
Jinked with the DNA molecule, in which case the first amino acid at the N-terminus of the<br>
recombinant protein will always be a methionine, which is encoded by the ATG start codon. If<br>
desired, methionine at the N-terminus may be cleaved from the protein by in vitro incubation with<br>
cyanogen bromide.<br>
Fusion proteins provide an alternative for yeast expression systems, as well as in mammalian,<br>
baculovirus, and bacterial expression systems. Usually, a DNA sequence encoding the N-terminal<br>
portion of an endogenous yeast protein, or other stable protein, is fused to the 5' end of<br>
heterologous coding sequences. Upon expression, this construct will provide a fusion of the two<br>
amino acid sequences. For example, the yeast or human superoxide dismutase (SOD) gene, can be<br>
linked at the 5' terminus of a foreign gene and expressed in yeast. The DNA sequence at the<br>
junction of the two amino acid sequences may or may not encode a cieavabk site. See eg. EP-A-0<br>
196 056. Another example is a ubiquitin fusion protein. Such a fusion protein is made with the<br>
ubiquitin region that preferably retains a site for a processing enzyme (eg. ubiquitin-specific<br>
processing protease) to cleave the ubiquitin from the foreign protein. Through this method,<br>
therefore, native foreign protein can be isolated (eg. WO88/024066).<br>
Alternatively, foreign proteins can also be secreted from the cell into the growth media by creating<br>
chimeric DNA molecules that encode a fusion protein comprised of a leader sequence fragment that<br>
provide for secretion in yeast of the foreign protein. Preferably, there are processing sites encoded<br>
between the leader fragment and the foreign gene that can be cleaved either in vivo or in vitro. The<br><br>
leader sequence fragment usually encodes a signal peptide comprised of hydrophobic amino acids<br>
which direct the secretion of the protein from the cell.<br>
DNA encoding suitable signal sequences can be derived from genes for secreted yeast proteins,<br>
such as the yeast invertase gene (EP-A-0 012 873; JPO. 62,096,086) and the A-factor gene (US<br>
patent 4,588,684). Alternatively, leaders of non-yeast origin, such as an interferon leader, exist that<br>
also provide for secretion in yeast (EP-A-0 060 057).<br>
A preferred class of secretion leaders are those that employ a fragment of the yeast alpha-factor<br>
gene, which contains both a "pxe" signal sequence, and a "pro" region. The types of alpha-factor<br>
fragments that can be employed include the full-length pre-pro alpha factor leader (about 83 amino<br>
acid residues) as well as truncated alpha-factor leaders (usually about 25 to about 50 amino acid<br>
residues) (US Patents 4,546,083 and 4,870,008; EP-A-0 324 274). Additional leaders employing<br>
an alpha-factor leader fragment that provides for secretion include hybrid alpha-factor leaders made<br>
with a presequence of a first yeast, but a pro-region from a second yeast alphafactor. (eg. see WO<br>
89/02463.)<br>
Usually, transcription termination sequences recognized by yeast are regulatory regions located 3'<br>
to the translation stop codon, and thus together with the promoter flank the coding sequence. These<br>
sequences direct the transcription of an mRNA which can be translated into the polypeptide<br>
encoded by the DNA. Examples of transcription terminator sequence and other yeast-recognized<br>
termination sequences, such as those coding for glycolytic enzymes.<br>
Usually, the above described components, comprising a promoter, leader (if desired), coding<br>
sequence of interest, and transcription termination sequence, are put together into expression<br>
constructs. Expression constructs are often maintained in a replicon, such as an extrachromosomal<br>
element (eg. plasmids) capable of stable maintenance in a host, such as yeast or bacteria. The<br>
replicon may have two replication systems, thus allowing it to be maintained, for example, in yeast<br>
for expression and in a prokaryotic host for cloning and amplification. Examples of such yeast-<br>
bacteria shuttle vectors include YEp24 [Botstein et al. (1979) Gene 8:17-24}, pCl/1 [Brake et al.<br>
(1984) Proc. Natl. Acad. Sci USA 87:4642-4646], and YRpl7 [Stinchcomb et al (1982) J. Mol.<br>
Biol. 158:157]. In addition, a replicon may be either a high or low copy number plasmid. A high<br>
copy number piasmid will generally have a copy number ranging from about 5 to about 200, and<br><br>
usually about 10 to about 150. A host containing a high copy number plasmid will preferably have<br>
at least about 10, and more preferably at least about 20. Enter a high or low copy number vector<br>
may be selected, depending upon the effect of the vector and the foreign protein on the host. See<br>
eg. Brake et al, supra.<br>
Alternatively, the expression constructs can be integrated into the yeast genome with an integrating<br>
vector. Integrating vectors usually contain at least one sequence homologous to a yeast<br>
chromosome that allows the vector to integrate, and preferably contain two homologous sequences<br>
flanking the expression construct. Integrations appear to result from recombinations between<br>
homologous DNA in the vector and the yeast chromosome [Orr-Weaver et al. (1983) Methods in<br>
Enzymol. 707:228-245}. An integrating vector may be directed to a specific locus in yeast by<br>
selecting the appropriate homologous sequence for inclusion in the vector. See Orr-Weaver et al,<br>
supra. One or more expression construct may integrate, possibly affecting levels of recombinant<br>
protein produced [Rine et al. (1983) Proc. Nat!. Acad. Sci. USA 80:6750]. The chromosomal<br>
sequences included in the vector can occur either as a single segment in the vector, which results<br>
in the integration of the entire vector, or two segments homologous to adjacent segments in the<br>
chromosome and flanking the expression construct in the vector, which can result in the stable<br>
integration of only the expression construct.<br>
Usually, extrachromosomal and integrating expression constructs may contain selectable markers<br>
to allow for the selection of yeast strains that have been transformed. Selectable markers may<br>
include biosynthetic genes that can be expressed in the yeast host, such as ADE2, HIS4, LEU2,<br>
TRP1, and ALG7, and the G418 resistance gene, which confer resistance in yeast cells to<br>
tunicamycin and G418, respectively. In addition, a suitable selectable marker may also provide<br>
yeast with the ability to grow in the presence of toxic compounds, such as metal. For example, the<br>
presence of CUP1 allows yeast to grow in the presence of copper ions [Butt et al. (1987) Kiicrobiol,<br>
Rev. 51-351].<br>
Alternatively, some of the above described components can be put together into transformation<br>
vectors. Transformation vectors are usually comprised of a selectable marker that is either<br>
maintained in a replicon or developed into an integrating vector, as described above.<br><br>
Expression and transformation vectors, either extrachromosomal replicons or integrating vectors,<br>
have been developed for transformation into many yeasts. For example, expression vectors have<br>
been developed for, inter alia, the following yeasts:Candida albicans [Kurtz, et al. (1986) Mol.<br>
Cell. Biol. 6:142], Candida maltosa [Kunze, et al. (1985) J. Basic Microbiol 25:141]. Hansenula<br>
polymorpha [Gleeson, et al. (1986) J. Gen. Microbiol. J32:3459; Roggenkamp et al. (1986) Mol.<br>
Gen. Genet. 202:302], Kluyveromyces fragilis [Das, et al. (1984) J. Bacteriol. 158:1165],<br>
Kluyveromyces lactis [De Louvencourt et al. (1983) J. Bacteriol. 154:131; Van den Berg el al.<br>
(1990) Bio/Technology 5:135], Pichia guillerimondii [Kunze et al. (1985) J. Basic Microbiol.<br>
25:141], Pichia pastoris [Cregg, et al. (1985) Mol. Cell Biol. 5:3316; US Patent Nos. 4,837,148<br>
and 4,929,555], Saccharomyces cerevisiae [Hinnen et al. (1978) Proc. Natl. Acad. Sci. USA<br>
75:1929; Ito et al. (1983) J. Bacteriol. 755:163], Schizosaccharomyces pombe [Beach and Nurse<br>
(1981)Nature 300:106], and Yarrowia lipolytica [Davidow, et al. (1985) Curr. Genet. 70:380471<br>
Gaillardin, et al. (1985) Curr. Genet. 10:49].<br>
Methods of introducing exogenous DNA into yeast hosts are well-known in the art, and usually<br>
include either the transformation of spheroplasts or of intact yeast cells treated with alkali cations.<br>
Transformation procedures usually vary with the yeast species to be transformed. See eg. [Kurtz<br>
et al. (1986) Mol. Cell Biol. 6:142; Kunze et al. (1985) J. Basic Microbiol 25:141; Candida];<br>
[Gleeson et al (1986) J. Gen. Microbiol. 132:3459; Roggenkamp et al. (1986) Mol. Gen. Genet.<br>
202:302; Hansenula]; [Das et al. (1984) J. Bacteriol. 158:1165; De Louvencourt et al. (1983) J.<br>
Bacteriol 154:1165; Van den Berg et al. (1990) Bio/Technology 5:135; Kluyveromyces]; [Cregg<br>
et al. (1985) Mol. Cell. Biol. 5:3376; Kunze et al. (1985) J. Basic Microbiol. 25:141; US Patent<br>
Nos. 4,837,148 and 4,929,555; Pichia]; [Hinnen et al. (1978) Proc. Natl. Acad Sci. USA 75;1929;<br>
Ito et al. (1983) J. Bacteriol 153:163 Saccharomyces]; [Beach and Nurse (1981) Nature 300:706;<br>
Schizosaccharomyces]; [Davidow et al. (1985) Curr. Genet. 10:39; Gaillardin et al. (1985) Curr.<br>
Genet. 10:49; Yarrowia].<br>
Antibodies<br>
As used herein, the term "antibody" refers to a polypeptide or group of polypeptides composed of<br>
at least one antibody combining site. An "antibody combining site" is the three-dimensional<br>
binding space with an internal surface shape and charge distribution complementary to the features<br>
of an epitope of an antigen, which allows a binding of the antibody with the antigen. "Antibody"<br><br>
includes, for example, vertebrate antibodies, hybrid antibodies, chimeric antibodies, humanised<br>
antibodies, altered antibodies, univalent antibodies, Fab proteins, and single domain antibodies.<br>
Antibodies against the proteins of the invention are useful for affinity chromatography,<br>
immunoassays, and distinguishing/identifying Neisserial proteins.<br>
Antibodies to the proteins of the invention, both polyclonal and monoclonal, may be prepared by<br>
conventional methods. In general, the protein is first used to immunize a suitable animal, preferably<br>
a mouse, rat, rabbit or goat. Rabbits and goats are preferred for the preparation of polyclonal sera<br>
due to the volume of serum obtainable, and the availability of labeled anti-rabbit and anti-goat<br>
antibodies. Immunization is generally performed by mixing or emulsifying the protein in saline,<br>
preferably in an adjuvant such as Freund's complete adjuvant, and injecting the mixture or<br>
emulsion parenterally (generally subcutaneously or intramuscularly). A dose of 50-200 μg/injection<br>
is typically sufficient. Immunization is generally boosted 2-6 weeks later with one or more<br>
injections of the protein in saline, preferably using Freund's incomplete adjuvant. One may<br>
alternatively generate antibodies by in vitro immunization using methods known in the art, which<br>
for the purposes of this invention is considered equivalent to in vivo immunization. Polyclonal<br>
antisera is obtained by bleeding the immunized animal into a glass or plastic container, incubating<br>
the blood at 25°C for one hour, followed by incubating at 4°C for 2-18 hours. The serum is<br>
recovered by centrifugation (eg. 1,000g for 10 minutes). About 20-50 ml per bleed may be obtained<br>
from rabbits.<br>
Monoclonal antibodies are prepared using the standard method of Kohler &amp; Milstein [Nature<br>
(1975) 256:495-96], or a modification thereof. Typically, a mouse or rat is immunized as described<br>
above. However, rather man bleeding the animal to extract serum, the spleen (and optionally<br>
several large lymph nodes) is removed and dissociated into single cells. If desired, the spleen cells<br>
may be screened (after removal of nonspecifically adherent cells) by applying a cell suspension to<br>
a plate or well coated with the protein antigen. B-cells expressing membrane-bound<br>
unmunoglobulin specific for the antigen bind to the plate, and are not rinsed away with the rest of<br>
the suspension. Resulting B-cells, or all dissociated spleen cells, are then induced to fuse with<br>
myeloma cells to form hybridomas, and are cultured in a selective medium (eg. hypoxanthine,<br>
aminopterin, thymidine medium, "HAT"). The resulting hybridomas are plated by limiting dilution,<br>
and are assayed for the production of antibodies which bind specifically to the immunizing antigen<br><br>
(and which do not bind to unrelated antigens). The selected MAb-secreting hybridomas are then<br>
cultured either in vitro (eg. in tissue culture bottles or hollow fiber reactors), or in vivo (as ascites<br>
in mice).<br>
If desired, the antibodies (whether polyclonal or monoclonal) may be labeled using conventional<br>
techniques. Suitable labels include fluorophores, chromophores, radioactive atoms (particularly 32P<br>
and I), electron-dense reagents, enzymes, and ligands having specific binding partners. Enzymes<br>
are typically detected by their activity. For example, horseradish peroxidase is usually detected by its<br>
ability to convert 3,3',5,5'-tetramethylbenzidine (TMB) to a blue pigment, quantifiable with a<br>
spectrophotometer. "Specific binding partner" refers to a protein capable of binding a ligand molecule<br>
with high specificity, as for example in the case of an antigen and a monoclonal antibody specific<br>
therefor. Other specific binding partners include biotin and avidin or streptavidin, IgG and protein A,<br>
and the numerous receptor-ligand couples known in the art. It should be understood that the above<br>
description is not meant to categorize the various labels into distinct classes, as the same label may<br>
serve in several different modes. For example, I25I may serve as a radioactive label or as an<br>
electron-dense reagent HRP may serve as enzyme or as antigen for a MAb. Further, one may combine<br>
various labels for desired effect. For example, MAbs and avidin also require labels in the practice of<br>
this invention: thus, one might label a MAb with biotin, and detect its presence with avidin labeled<br>
with 125I, or with an anti-biotin MAb labeled with HRP. Other permutations and possibilities will be<br>
readily apparent to those of ordinary skill in the art, and are considered as equivalents within the scope<br>
of the instant invention.<br>
Pharmaceutical Compositions<br>
Pharmaceutical compositions can comprise either polypeptides, antibodies, or nucleic acid of the<br>
invention. The pharmaceutical compositions will comprise a therapeutically effective amount of<br>
either polypeptides, antibodies, or polynucleotides of the claimed invention.<br>
The term "tberapeutically effective amount" as used herein refers to an amount of a therapeutic<br>
agent to treat, ameliorate, or prevent a desired disease or condition, or to exhibit a detectable<br>
therapeutic or preventative effect. The effect can be detected by, for example, chemical markers or<br>
antigen levels. Therapeutic effects also include reduction in physical symptoms, such as decreased<br>
body temperature. The precise effective amount for a subject will depend upon the subject's size<br>
and health, the nature and extent of the condition, and the therapeutics or combination of<br><br>
therapeutics selected for administration. Thus, it is not useful to specify an exact effective amount<br>
in advance. However, the effective amount for a given situation can be determined by routine<br>
experimentation and is within the judgement of the clinician.<br>
For purposes of the present invention, an effective dose will be from about 0.01 mg/ kg to 50 mg/kg<br>
or 0.05 mg/kg to about 10 mg/kg of the DNA constructs in the individual to which it is administered.<br>
A pharmaceutical composition can also contain a pharmaceutically acceptable carrier. The term<br>
"pharmaceutically acceptable carrier" refers to a carrier for administration of a therapeutic agent, such<br>
as antibodies or a polypeptide, genes, and other therapeutic agents. The term refers to any<br>
pharmaceutical carrier that does not itself induce the production of antibodies harmful to the<br>
individual receiving the composition, and which may be administered without undue toxicity. Suitable<br>
carriers may be large, slowly metabolized macromolecules such as proteins, polysaccharides,<br>
polylactic acids, polyglycolic acids, polymeric amino acids, amino acid copolymers, and inactive virus<br>
particles. Such carriers are well known to those of ordinary skill in the art.<br>
Pharmaceutically acceptable salts can be used therein, for example, mineral acid salts such as<br>
hydrochlorides, hydrobromides, phosphates, sulfates, and the like; and the salts of organic acids<br>
such as acetates, propionates, malonates, benzoates, and the like. A thorough discussion of<br>
pharrnaceutically acceptable excipients is available in Remington's Pharmaceutical Sciences (Mack<br>
Pub. Co., NJ. 1991).<br>
Pharmaceutically acceptable carriers in therapeutic compositions may contain liquids such as water,<br>
saline, glycerol and ethanol. Additionally, auxiliary substances, such as wetting or emulsifying agents,<br>
pH buffering substances, and the like, may be present in such vehicles. Typically, the therapeutic<br>
compositions are prepared as injectables, either as liquid solutions or suspensions; solid forms suitable<br>
for solution in, or suspension in, liquid vehicles prior to injection may also be prepared. Liposomes<br>
are included within the definition of a pharmaceutically acceptable carrier.<br>
Delivery Methods<br>
Once formulated, the compositions of the invention can be administered directly to the subject. The<br>
subjects to be treated can be animals; in particular, human subjects can be treated.<br><br>
Direct delivery of the compositions will generally be accomplished by injection, either<br>
subcutaneously, intraperitoneally, intravenously or intramuscularly or delivered to the interstitial<br>
space of a tissue. The compositions can also be administered into a lesion. Other modes of<br>
administration include oral and pulmonary administration, suppositories, and transdermal or<br>
transcutaneous applications (eg. see WO98/20734), needles, and gene guns or hyposprays. Dosage<br>
treatment may be a single dose schedule or a multiple dose schedule.<br>
Vaccines<br>
Vaccines according to the invention may either be prophylactic (ie. to prevent infection) or<br>
therapeutic (ie. to treat disease after infection).<br>
Such vaccines comprise immunising antigen(s), immunogen(s), polypeptide(s), protein(s) or nucleic acid,<br>
usually in combination with "pharmaceutically acceptable carriers," which include any carrier that does<br>
not itself induce the production of antibodies harmful to the individual receiving the composition.<br>
Suitable carriers are typically large, slowly metabolized macromolecules such as proteins,<br>
poiysaccharides, polylactic acids, polygiycolic acids, polymeric amino acids, amino acid copolymers,<br>
Jipid aggregates (such as oil droplets or liposomes), and inactive virus particles. Such carriers are well<br>
known to those of ordinary skill in the art. Additionally, these carriers may function as<br>
immunostimulating agents ("adjuvants"). Furthermore, the antigen or immunogen may be conjugated to<br>
a bacterial toxoid, such as a toxoid from diphtheria, tetanus, cholera, H. pylori, etc. pathogens.<br>
Preferred adjuvants to enhance effectiveness of the composition include, but are not limited to: (1)<br>
aluminum salts (alum), such as aluminum hydroxide, aluminum phosphate, aluminum sulfate, etc;<br>
(2) oil-in-water emulsion formulations (with or without other specific immunostimulating agents<br>
such as muramyl peptides (see below) or bacterial cell wall components), such as for example (a)<br>
MF59™ (WO 90/14837; Chapter 10 in Vaccine design: the subunit and adjuvant approach, eds.<br>
Powell &amp; Newman, Plenum Press 1995), containing 5% Squalene, 0.5% Tween 80, and 0.5% Span<br>
85 (optionally containing various amounts of MTP-PE (see below), although not required)<br>
formulated into submicron particles using a microfluidizer such as Model 110Y microfluidizer<br>
(Microfluidics, Newton, MA), (b) SAF, containing 10% Squalane, 0.4% Tween 80,5% pluronic-<br>
blocked polymer L121, and thr-MDP (see below) either microfluidized into a submicron emulsion<br>
or vortexed to generate a larger partkie size emulsion, and (c) Ribi™ adjuvant system (RAS), (Ribi<br>
Immunochem, Hamilton, MT) containing 2% Squalene, 0.2% Tween 80, and one or more bacterial<br><br>
cell wall components from the group consisting of monophosphorylipid A (MPL), trehalose<br>
dimycolate (TDM), and cell wall skeleton (CWS), preferably MPL + CWS (Detox™); (3) saponin<br>
adjuvants, such as Stimulon™ (Cambridge Bioscience, Worcester, MA) may be used or particles<br>
generated therefrom such as ISCOMs (immunostimulating complexes); (4) Complete Freund's<br>
Adjuvant (CFA) and Incomplete Freund's Adjuvant (IFA); (5) cytokines, such as interleukins (eg.<br>
IL-1, IL-2, IL-4, IL-5, IL-6, IL-7, IL-12, etc.), interferons (eg. gamma interferon), macrophage<br>
colony stimulating factor (M-CSF), tumor necrosis factor (TNF), etc; and (6) other substances that<br>
act as immunostimulating agents to enhance the effectiveness of the composition. Alum and<br>
MF59™ are preferred.<br>
As mentioned above, muramyl peptides include, but are not limited to, N-acetyl-muramyl-L-<br>
threonyl-D-isoglutamine (thr-MDP), N-acetyl-normuramyl-L-alanyl-D-isoglutamine (nor-MDP),<br>
N-acetylmuramyl-L-alanyl-D-isoglutaminyl-L-alanine-2-(r-2'-dipalmitoyl-5n-glycero-3-<br>
hydroxyphosphoryloxy)-ethylamine (MTP-PE), etc.<br>
The immunogenic compositions (eg. the immunising antigen/immunogen/polypeptide/protein/<br>
nucleic acid, pharmaceutically acceptable carrier, and adjuvant) typically will contain diluents, such<br>
as water, saline, glycerol, ethanol, etc. Additionally, auxiliary substances, such as wetting or<br>
emulsifying agents, pH buffering substances, and the like, may be present in such vehicles.<br>
Typically, the immunogenic compositions are prepared as injectables, either as liquid solutions or<br>
suspensions; solid forms suitable for solution in, or suspension in, liquid vehicles prior to injection<br>
may also be prepared. The preparation also may be emulsified or encapsulated in liposomes for<br>
enhanced adjuvant effect, as discussed above under pharmaceutically acceptable carriers.<br>
Immunogenic compositions used as vaccines comprise an immunologically effective amount of the<br>
antigenic or immunogenic polypeptides, as well as any other of the above-mentioned components,<br>
as needed, By "immunologically effective amount", it is meant that the administration of mat<br>
amount to an individual, either in a single dose or as part of a series, is effective for treatment or<br>
prevention. This amount varies depending upon the health and physical condition of the individual<br>
to be treated, the taxonomic group of individual to be treated (eg. nonhuman primate, primate, etc.),<br>
the capacity of the individual's immune system to synthesize antibodies, the degree of protection<br>
desired, the formulation of the vaccine, the treating doctor's assessment of the medical situation,<br><br>
and other relevant factors. It is expected that the amount will fall in a relatively broad range that<br>
can be determined through routine trials.<br>
The immunogenic compositions are conventionally administered parenterally, eg. by injection,<br>
either subcutaneously, intramuscularly, or transdermally/transcutaneously (eg. WO98/20734).<br>
Additional formulations suitable for other modes of administration include oral and pulmonary<br>
formulations, suppositories, and transdermal applications. Dosage treatment may be a single dose<br>
schedule or a multiple dose schedule. The vaccine may be administered in conjunction with other<br>
immunoregulatory agents.<br>
As an alternative to protein-based vaccines, DNA vaccination may be employed [eg. Robinson &amp;<br>
Torres (1997) Seminars in Immunology 9:271-283; Donnelly et al. (1997) Annu Rev Immunol<br>
15:617-648; see later herein].<br>
Gene Delivery Vehicles<br>
Gene therapy vehicles for delivery of constructs including a coding sequence of a therapeutic of<br>
the invention, to be delivered to the mammal for expression in the mammal, can be administered<br>
either locally or systemically. These constructs can utilize viral or non-viral vector approaches in<br>
in vivo or ex vivo modality. Expression of such coding sequence can be induced vising endogenous<br>
mammalian or heterologous promoters. Expression of the coding sequence in vivo can be either<br>
constitutive or regulated.<br>
The invention includes gene delivery vehicles capable of expressing the contemplated nucleic acid<br>
sequences. The gene delivery vehicle is preferably a viral vector and, more preferably, a retroviral,<br>
adenoviral, adeno-associated viral (AAV), herpes viral, or alphavirus vector. The viral vector can<br>
also be an astrovirus, coronavirus, orthomyxovirus, papovavirus, paramyxovirus, parvovirus,<br>
picomavirus, poxvirus, or togavinis viial vector. See generally, Jolly (1994) Cancer Gene Therapy<br>
1:51-64; Kimura (1994) Human Gene Therapy 5:845-852; Connelly (1995) Human Gene Therapy<br>
6:185-193; and Kaplitt (1994) Nature Genetics 6:148-153.<br>
Retroviral vectors are well known in the art and we contemplate that any retroviral gene therapy vector<br>
is employable in the invention, including B, C and D type retroviruses, xenotropic retroviruses (for<br>
example,NZB-X1,NZB-X2 and NZB9-1 (see O'Neill (1985) J. Virol. 53:160)polytropic retrovinises<br><br>
eg. MCF and MCF-MLV (see Kelly (1983) J. Virol. 45:291), spumaviruses and lentiviruses. See RNA<br>
Tumor Viruses, Second Edition, Cold Spring Harbor Laboratory, 1985.<br>
Portions of the retroviral gene therapy vector may be derived from different retroviruses. For<br>
example, retrovector LTRs may be derived from a Murine Sarcoma Virus, a tRNA binding site<br>
from a Rous Sarcoma Virus, a packaging signal from a Murine Leukemia Virus, and an origin of<br>
second strand synthesis from an Avian Leukosis Virus.<br>
These recombinant retroviral vectors may be used to generate transduction competent retroviral<br>
vector particles by introducing them into appropriate packaging cell lines (see US patent<br>
5,591,624). Retrovirus vectors can be constructed for site-specific integration into host cell DNA<br>
by incorporation of a chimeric integrase enzyme into the retroviral particle (see WO96/37626). It<br>
is preferable that the recombinant viral vector is a replication defective recombinant virus.<br>
Packaging cell lines suitable for use with the above-described retrovirus vectors are well known<br>
in the art, are readily prepared (see WO95/30763 and WO92/05266), and can be used to create<br>
producer cell lines (also termed vector cell lines or "VCLs") for the production of recombinant<br>
vector particles. Preferably, the packaging cell lines are made from human parent cells (eg. HT1080<br>
cells) or mink parent cell lines, which eliminates inactivation in human serum.<br>
Preferred retroviruses for the construction of retroviral gene therapy vectors include Avian<br>
Leukosis Virus, Bovine Leukemia, Virus, Murine Leukemia Virus, Mink-Cell Focus-Inducing<br>
Virus, Murine Sarcoma Virus, Reticuloendotheliosis Virus and Rous Sarcoma Virus. Particularly<br>
preferred Murine Leukemia Viruses include 4070A and 1S04A (Hartley and Rowe (1976) J Virol<br>
19:19-25), Abelson (ATCC No. VR-999), Friend (ATCC No. VR-245), Graffi, Gross (ATCC Nol<br>
VR-590), Kirsten, Harvey Sarcoma Virus and Rauscher (ATCC No. VR-998) and Moloney Murine<br>
Leukemia Virus (ATCC No. VR-190). Such retroviruses may be obtained from depositories or<br>
collections such as the American Type Culture Collection ("ATCC") in Rockville, Maryland or<br>
isolated from known sources using commonly available techniques.<br>
Exemplary known retroviral gene therapy vectors employable in this invention include those<br>
described in patent applications GB2200651, EP0415731, EP0345242, EP0334301, WO89/02468;<br>
WO89/05349, WO89/09271, WO90/02806, WO90/07936, WO94/03622, WO93/25698,<br><br>
WO93/25234, WO93/11230, WO93/10218, WO91/02805, WO91/02825, WO95/07994, US<br>
5,219,740, US 4,405,712, US 4,861,719, US 4,980,289, US 4,777,127, US 5,591,624. See also Vile<br>
(1993) Cancer Res 53:3860-3864; Vile (1993) Cancer Res 53:962-967; Ram (1993) Cancer Res<br>
53 (1993) 83-88; Takamiya (1992) J Neurosci Res 33:493-503; Baba (1993) J Neurosurg<br>
79:729-735; Mann (1983) Cell 33:153; Cane (1984) Proc Natl Acad Sci 81:6349; and Miller (1990)<br>
Human Gene Therapy 1.<br>
Human adenoviral gene therapy vectors are also known in the art and employable in this invention.<br>
See, for example, Berkner (1988) Biotechniques 6:616 and Rosenfeld (1991) Science 252:431, and<br>
WO93/07283, WO93/06223, and WO93/07282. Exemplary known adenoviral gene therapy vectors<br>
employable in this invention include those described in the above referenced documents and in<br>
WO94/12649, WO93/03769, WO93/19191, WO94/28938, WO95/11984, WO95/00655,<br>
WO95/27071, WO95/29993, WO95/34671, WO96/05320, WO94/08026, WO94/11506,<br>
WO93/06223, WO94/24299, WO95/14102, WO95/24297, WO95/02697, WO94/28152,<br>
WO94/24299, WO95/09241, WO95/25807, WO95/05835, WO94/18922 and WO95/09654.<br>
Alternatively, administration of DNA linked to killed adenovirus as described in Curiel (1992)<br>
Hum. Gene Ther. 3:147-154 may be employed. The gene delivery vehicles of the invention also<br>
include adenovirus associated virus (AAV) vectors. Leading and preferred examples of such<br>
vectors for use in this invention are the AAV-2 based vectors disclosed in Srivastava,<br>
WO93/09239. Most preferred AAV vectors comprise the two AAV inverted terminal repeats in<br>
which the native D-sequences are modified by substitution of nucleotides, such that at least 5 native<br>
nucleotides and up to 18 native nucleotides, preferably at least 10 native nucleotides up to 18 native<br>
nucleotides, most preferably 10 native nucleotides are retained and the remaining nucleotides of<br>
the D-sequence are deleted or replaced with non-native nucleotides. The native D-sequences of the<br>
AAV inverted terminal repeats are sequences of 20 consecutive nucleotides in each AAV inverted<br>
terminal repeat (ie. there is one sequence at each end) which are not involved in HP formation. The<br>
non-native replacement nucleotide may be any nucleotide other than the nucleotide found in the<br>
native D-sequence in the same position Oilier employable exemplary AAV vectors are pWP-19,<br>
pWN-1, both of which are disclosed in Nahreini (1993) Gene 124:257-262. Another example of<br>
such an AAV vector is psub201 (see Samdski (1987) J. Virol. 61:3096). Another exemplary AAV<br>
vector is the Double-D ITR vector. Construction of the Double-D ITR vector is disclosed in US<br>
Patent 5,478,745. Still other vectors are those disclosed in Carter US Patent 4,797,368 and<br><br>
Muzyczka US Patent 5,139,941, Chartejee US Patent 5,474,935, and Kotin WO94/288157. Yet a<br>
further example of an AAV vector employable in this invention is SSV9AFABTKneo, which<br>
contains the AFP enhancer and albumin promoter and directs expression predominantly in the liver.<br>
Its structure and construction are disclosed in Su (1996) Human Gene Therapy 7:463-470.<br>
Additional AAV gene therapy vectors are described in US 5,354,678, US 5,173,414, US 5,139,941,<br>
and US 5,252,479.<br>
The gene therapy vectors of the invention also include herpes vectors. Leading and preferred<br>
examples are herpes simplex virus vectors containing a sequence encoding a thymidine kinase<br>
polypeptide such as those disclosed in US 5,288,641 and EP0176170 (Roizman). Additional<br>
exemplary herpes simplex virus vectors include HFEM/ICP6-LacZ disclosed in WO95/04139<br>
(Wistar Institute), pHSVlac described in Geller (1988) Science 241:1667-1669 and in WO90/09441<br>
and WO92/07945, HSV Us3::pgC-lacZ described in Fink (1992) Human Gene Therapy 3:11-19<br>
and HSV 7134, 2 RH 105 and GAL4 described in EP 0453242 (Breakefield), and those deposited<br>
with the ATCC as accession numbers ATCC VR-977 and ATCC VR-26Q.<br>
Also contemplated are alpha virus gene therapy vectors that can be employed in this invention.<br>
Preferred alpha virus vectors are Sindbis viruses vectors. Togaviruses, Semliki Forest virus (ATCC<br>
VR-67; ATCC VR-1247), Middleberg virus (ATCC VR-370), Ross River virus (ATCC VR-373;<br>
ATCC VR-1246), Venezuelan equine encephalitis virus (ATCC VR923; ATCC VR-1250; ATCC<br>
VR-1249; ATCC VR-532), and those described in US patents 5,091,309, 5,217,879, and<br>
WO92/10578. More particularly, those alpha vims vectors described in US Serial No. 08/405,627,<br>
filed March 15,1995,WO94/21792, WO92/10578, WO95/07994, US 5,091,309 and US 5,217,879<br>
are employable. Such alpha viruses may be obtained from depositories or collections such as the<br>
ATCC in Rockville, Maryland or isolated from known sources using commonly available<br>
techniques. Preferably, alphavirus vectors with reduced cytotoxicity are used (see USSN<br>
08/679640).<br>
DNA vector systems such as eukaryotic layered expression systems are also useful for expressing<br>
the nucleic acids of the invention. See WO95/07994 for a detailed description of eukaryotic layered<br>
expression systems. Preferably, the eukaryotic layered expression systems of the invention are<br>
derived from alphavirus vectors and most preferably from Sindbis viral vectors.<br><br>
Other viral vectors suitable for use in the present invention include those derived from poliovirus. for<br>
example ATCC VR-58 and those described in Evans, Nature 339 (1989) 385 and Sabin (1973) J. Biol.<br>
Standardization 1:115; rhinovirus, for example ATCC VR-1110 and those described in Arnold (1990)<br>
J Cell Biochem L401; pox viruses such as canary pox virus or vaccinia virus, for example ATCC<br>
VR-111 and ATCC VR-2010 and those described in Fisher-Hoch (1989) Proc Natl Acad Sci 86:317;<br>
Flexner (1989) Ann NY Acad Sci 569:86, Flexner (1990) Vaccine 8:17; in US 4,603,112 and US<br>
4,769,330 and WO89/01973; SV40 virus, for example ATCC VR-305 and those described in<br>
Mulligan (1979) Nature 277:108 and Madzak (1992) J Gen Virol 73:1533; influenza virus, for<br>
example ATCC VR-797 and recombinant influenza viruses made employing reverse genetics<br>
techniques as described in US 5,166,057 and in Enami (1990) Proc Natl Acad Sci 87:3802-3805;<br>
Enami &amp; Palese (1991) J Virol 65:2711-2713 and Luytjes (1989) Cell 59:110, (see also McMichael<br>
(1983) NEJ Med 309:13, and Yap (1978) Nature 273:238 and Nature (1979) 277:108); human<br>
immunodeficiency virus as described in EP-0386882 and in Buchschacher (1992) J. Virol. 66:2731;<br>
measles virus, for example ATCC VR-67 and VR-1247 and those described in EP-0440219; Aura<br>
virus, for example ATCC VR-368; Bebaru virus, for example ATCC VR-600 and ATCC VR-1240;<br>
Cabassou virus, for example ATCC VR-922; Chikungunya virus, for example ATCC VR-64 and<br>
ATCC VR-1241; Fort Morgan Virus, for example ATCC VR-924; Getah virus, for example ATCC<br>
VR-369 and ATCC VR-1243; Kyzylagach virus, for example ATCC VR-927; Mayaro virus, for<br>
example ATCC VR-66; Mucambo virus, for example ATCC VR-580 and ATCC VR-1244; Ndumu<br>
virus, for example ATCC VR-371; Pixuna virus, for example ATCC VR-372 and ATCC VR-1245;<br>
Tonate virus, for example ATCC VR-925; Triniti virus, for example ATCC VR-469; Una virus, for<br>
example ATCC VR-374; Whataroa virus, for example ATCC VR-926; Y-62-33 virus, for example<br>
ATCC VR-375; O'Nyong virus, Eastern encephalitis virus, for example ATCC VR-65 and ATCC<br>
VR-1242; Western encephalitis virus, for example ATCC VR-70, ATCC VR-1251, ATCC VR-622<br>
and ATCC VR-1252; and coronavirus, for example ATCC VR-740 and those described in Hamre<br>
(1966) Proc Soc Exp Biol Med 121:190.<br>
Delivery of the compositions of mis invention into cells is not limited to the above mentioned viral<br>
vectors. Other delivery methods and media may be employed such as, for example, nucleic acid<br>
expression vectors, polycationic condensed DNA linked or unlinked to killed adenovirus alone, for<br>
example see US Serial No. 08/366,787, filed December 30,1994 and Curiel (1992) Hum Gene Ther<br>
3:147-154 ligand linked DNA, for example see Wu (1989) J Biol Chem 264:16985-16987,<br>
eucaryotic cell delivery vehicles cells, for example see US Serial No.08/240,030, filed May 9,<br><br>
1994, and US Serial No. 08/404,796, deposition of photopolymerized hydrogel materials,<br>
hand-held gene transfer particle gun, as described in US Patent 5,149,655, ionizing radiation as<br>
described in US5,206,152 and in WO92/11033, nucleic charge neutralization or fusion with cell<br>
membranes. Additional approaches are described in Philip (1994) Mol Cell Biol 14:2411-2418 and<br>
in Woffendin (1994) Proc Natl Acad Sci 91:581-1585.<br>
Particle mediated gene transfer may be employed, for example see US Serial No. 60/023,867.<br>
Briefly, the sequence can be inserted into conventional vectors that contain conventional control<br>
sequences for high level expression, and then incubated with synthetic gene transfer molecules such<br>
as polymeric DNA-binding cations like polylysine, protamine, and albumin, linked to cell targeting<br>
ligands such as asialoorosomucoid, as described in Wu &amp; Wu (1987) J. Biol. Chem.<br>
262:4429-4432, insulin as described in Hucked (1990) Biochem Pharmacol 40:253-263, galactose<br>
as described in Plank (1992) Bioconjugate Chem 3:533-539, lactose or transferrin.<br>
Naked DNA may also be employed. Exemplary naked DNA introduction methods are described in<br>
WO 90/11092 and US 5,580,859. Uptake efficiency may be improved using biodegradable latex<br>
beads. DNA coated latex beads are efficiently transported into cells after endocytosis initiation by the<br>
beads. The method may be improved further by treatment of the beads to increase hydrophobicity and<br>
thereby facilitate disruption of the endosome and release of the DNA into the cytoplasm.<br>
Liposomes that can act as gene delivery vehicles are described in US 5,422,120, WO95/13796,<br>
WO94/23697, WO91/14445 and EP-524,968. As described in USSN. 60/023,867, on non-viral<br>
delivery, the nucleic acid sequences encoding a polypeptide can be inserted into conventional<br>
vectors that contain conventional control sequences for high level expression, and then be incubated<br>
with synthetic gene transfer molecules such as polymeric DNA-binding cations like polylysine,<br>
protamine, and albumin, linked to cell targeting ligands such as asialoorosomucoid, insulin,<br>
galactose, lactose, or transferrin. Other delivery systems include the use of liposomes to encapsulate<br>
DNA comprising the gene under the control of a variety of tissue-specific or ubiquitously-active<br>
promoters. Further non-viral delivery suitable for use includes mechanical delivery systems such<br>
as the approach described in Woflfendin et al (1994) Proc. Natl. Acad. Sci. USA<br>
91 (24): 11581 -11585. Moreover, the coding sequence and the product of expression of such can be<br>
delivered through deposition of photopolymerized hydrogel materials. Other conventional methods<br>
for gene delivery that can be used for delivery of the coding sequence include, for example, use of<br><br>
hand-held gene transfer particle gun, as described in US 5,149,655; use of ionizing radiation for<br>
activating transferred gene, as described in US 5,206,152 and WO92/11033<br>
Exemplary liposome and polycationic gene delivery vehicles are those described in US 5,422,120<br>
and 4,762,915; in WO 95/13796; WO94/23697; and WO91/14445; in EP-0524968; and in Stryer,<br>
Biochemistry, pages 236-240 (1975) W.H. Freeman, San Francisco; Szoka (1980) Biochem<br>
Biophys Ada 600:1; Bayer (1979) Biochem Biophys Ada 550:464; Rivnay (1987) Meth Enzymol<br>
149:119; Wang (1987) Proc Natl Acad Sci 84:7851; Plant (1989) Anal Biochem 176:420.<br>
A polynucleotide composition can comprises therapeutically effective amount of a gene therapy<br>
vehicle, as the term is defined above. For purposes of the present invention, an effective dose will<br>
be from about 0.01 mg/ kg to 50 mg/kg or 0.05 mg/kg to about 10 mg/kg of the DNA constructs<br>
in the individual to which it is administered.<br>
Delivery Methods<br>
Once formulated, the polynucleotide compositions of the invention can be administered (1) directly<br>
to the subject; (2) delivered ex vivo, to cells derived from the subject; or (3) in vitro for expression<br>
of recombinant proteins. The subjects to be treated can be mammals or birds. Also, human subjects<br>
can be treated.<br>
Direct delivery of the compositions will generally be accomplished by injection, either<br>
subcutaneous ly, intraperitoneally, intravenously or intramuscularly or delivered to the interstitial<br>
space of a tissue. The compositions can also be administered into a lesion. Other modes of<br>
administration include oral and pulmonary administration, suppositories, and transdermal or<br>
transcutaneous applications (eg. see WO98/20734), needles, and gene guns or hyposprays. Dosage<br>
treatment may be a single dose schedule or a multiple dose schedule.<br>
Methods for the ex vivo delivery and reimplantation of transformed cells into a subject are known<br>
in the art and described in eg. WO93/14778. Examples of cells useful in ex vivo applications<br>
include, for example, stem cells, particularly hematopoetic, lymph cells, macrophages, dendritic<br>
cells, or tumor cells.<br><br>
Generally, delivery of nucleic acids for both ex vivo and in vitro applications can be accomplished<br>
by the following procedures, for example, dextran-mediated transfection, calcium phosphate<br>
precipitation, polybrene mediated transfection, protoplast fusion, electroporation, encapsulation of<br>
the polynucleotide(s) in liposomes, and direct microinjection of the DNA into nuclei, all well<br>
known in the art.<br>
Polynucleotide and polypeptide pharmaceutical compositions<br>
In addition to the pharmaceutically acceptable carriers and salts described above, the following<br>
additional agents can be used with polynucleotide and/or polypeptide compositions.<br>
A.Polvpeptides<br>
One example are polypeptides which include, without limitation: asioloorosomucoid (ASOR);<br>
transferrin; asialoglycoproteins; antibodies; antibody fragments; ferritin; interleukins; interferons,<br>
granulocyte, macrophage colony stimulating factor (GM-CSF), granulocyte colony stimulating<br>
factor (G-CSF), macrophage colony stimulating factor (M-CSF), stem cell factor and<br>
erythropoietin. Viral antigens, such as envelope proteins, can also be used. Also, proteins from<br>
other invasive organisms, such as the 17 amino acid pepude from the circumsporozoite protein of<br>
plasmodium falciparum known as RII.<br>
B. Hormones. Vitamins, etc.<br>
Other groups that can be included are, for example: hormones, steroids, androgens, estrogens,<br>
thyroid hormone, or vitamins, folic acid.<br>
C.Polyalkylenes Polysaccharides. etc.<br>
Also, polyalkylene glycol can be included with the desired polynucleotides/polypeptides. In a<br>
preferred embodiment, the polyalkylene glycol is polyethlylene glycol. In addition, mono-, di-, or<br>
polysaccharides can be included. In a preferred embodiment of this aspect, the polysaccharide is<br>
dextran or DEAE-dextran. Also, chitosan and poly(lactide-co-glycolide)<br>
D.Lipids. and Liposomes<br>
The desired polynucleotide/polypeptide can also be encapsulated in lipids or packaged in liposomes<br>
prior to delivery to the subject or to cells derived therefrom.<br><br>
Lspid encapsulation is generally accomplished using liposomes which are able to stably bind or<br>
entrap and retain nucleic acid. The ratio of condensed polynucleotide to lipid preparation can van'<br>
but will generally be around 1:1 (mg DNA:micromoles lipid), or more of lipid. For a review of the<br>
use of liposomes as carriers for delivery of nucleic acids, see, Hug and Sleight (1991) Biochim.<br>
Biophys. Acta. 1097:1-17; Straubinger( 1983) Meth. Enzymol. 101:512-527.<br>
Liposomal preparations for use in the present invention include cationic (positively charged),<br>
anionic (negatively charged) and neutral preparations. Cationic liposomes have been shown to<br>
mediate intracellular delivery of plasmid DNA (Feigner (1987) Proc. Natl. Acad. Sci. USA<br>
84:7413-7416); mRNA (Malone (1989) Proc. Natl. Acad Sci. USA 86:6077-6081); and purified<br>
transcription factors (Debs (1990) J. Biol. Chem. 265:10189-10192), in functional form.<br>
Cationic liposomes are readily available. For example, N[l-2,3-dioleyloxy)propyl]-N,N,N-triethylammnionium<br>
(DOTMA) liposomes are available under the trademark Lipofectin, from GIBCO BRL, Grand<br>
Island, NY. (See, also, Feigner supra). Other commercially available liposomes include<br>
transfectace (DDAB/DOPE) and DOTAP/DOPE (Boerhinger). Other cationic liposomes can be<br>
prepared from readily available materials using techniques well known in the art. See, eg. Szoka<br>
(1978) Proc Natl. Acad Sci. USA 75:4194-4198; WO90/11092 for a description of the synthesis<br>
of DOTAP (l,2-bis(oleoyloxy)-3<trimemylammonio liposomes.></trimemylammonio>
Similarly, anionic and neutral liposomes are readily available, such as from Avanti Polar Lipids<br>
(Birmingham, AL), or can be easily prepared using readily available materials Such materials include<br>
phosphatidyl choline, cholesterol, phosphatidy] ethanolamine, diokoylphosphatidyl choline (DOPC),<br>
dioleoylphosphatidyl glycerol (DOPG), dioleoylphoshatidyl ethanolamuie (DOPE), among others.<br>
These materials can also be mixed with the DOTMA and DOTAP starting materials in appropriate<br>
ratios. Methods for making liposomes using these materials are well known in the art.<br>
The liposomes can comprise multilammelar vesicles (MLVs), small unilamellar vesicles (SUVs),<br>
or large unilamellar vesicles (LUVs). The various liposome-nuclcic acid complexes are prepared<br>
using methods known in the art. See eg. Straubinger (1983) Meth. Immunol. 101:512-527; Szoka<br>
(1978) Proc. Natl. Acad. Sci. USA 75:4194-4198; Papahadjopoulos (1975) Biochm. Biophys. Acta<br>
394:483; Wilson (1979) Cell 17:77); Deamer &amp; Bangham (1976) Biochim. Biophys. Acta 443:629;<br>
Ostro (1977) Biochem. Biophys. Res. Commun. 76:836; Fraley (1979) Proc. Natl. Acad. Sci. USA<br><br>
76:3348); Enoch &amp; Strittmatter (1979) Proc. Nad. Acad. Sci. USA 76:145; Fraley (1980) J. Biol.<br>
Chem. (1980) 255:10431; Szoka &amp; Papahadjopoulos (1978) Proc. Nail. Acad. Sci USA 75:145:<br>
and Schaefer-Ridder (1982) Science 215:166.<br>
E. Lipoproteins<br>
In addition, lipoproteins can be included with the polynucleotide/polypeptide to be delivered.<br>
Examples of lipoproteins to be utilized include: chylomicrons, HDL, DDL, LDL, and VLDL. Mutants,<br>
fragments, or fusions of these proteins can also be used. Also, modifications of naturally occurring<br>
lipoproteins can be used, such as acetylated LDL. These lipoproteins can target the delivery of<br>
polynucleotides to cells expressing lipoprotein receptors. Preferably, if lipoproteins are including with<br>
the polynucleotide to be delivered, no other targeting ligand is included in the composition.<br>
Naturally occurring lipoproteins comprise a lipid and a protein portion. The protein portion are<br>
known as apoproteins. At the present, apoproteins A, B, C, D, and E have been isolated and<br>
identified. At least two of these contain several proteins, designated by Roman numerals, AI, All,<br>
AIV; CI, CO, CIII.<br>
A lipoprotein can comprise more than one apoprotein. For example, naturally occurring<br>
chylomicrons comprises of A, B, C, and E, over time these lipoproteins lose A and acquire C and<br>
E apoproteins. VLDL comprises A, B, C, and E apoproteins, LDL comprises apoprotein B; and<br>
HDL comprises apoproteins A, C, and £.<br>
The amino acid of these apoproteins are known and are described in, for example, Breslow (1985)<br>
Annu Rev. Biochem 54:699; Law (1986) Adv. Exp Med. Biol. 151:162; Chen (1986) J Biol Chem<br>
261:12918; Kane (1980) Proc Natl Acad Sci USA 77:2465; and Utermann (1984) Hum Genet 65:232.<br>
Lipoproteins contain a variety of lipids including, triglycerides, cholesterol (free and esters), and<br>
phosphoHpids. The composition of the lipids varies in naturally occurring lipoproteins. For<br>
example, chylomicrons comprise mainly triglycerides. A more detailed description of the lipid<br>
content of naturally occurring lipoproteins can be found, for example, in Meih. Enzymol. 128<br>
(1986). The composition of the lipids are chosen to aid in conformation of the apoprotein for<br>
receptor binding activity. The composition of lipids can also be chosen to facilitate hydrophobic<br>
interaction and association with the polynucleotide binding molecule.<br><br>
Naturally occurring lipoproteins can be isolated from serum by ultracentrifugation, for instance.<br>
Such methods are described in Meth. Enzymol. (supra); Pitas (1980) J. Biochem. 255:5454-5460<br>
and Mahey (1979) J Clin. Invest 64:743-750. Lipoproteins can also be produced by in vitro or<br>
recombinant methods by expression of the apoprotein genes in a desired host cell. See, for example,<br>
Atkinson (1986) Annu Rev Biophys Chem 15:403 and Radding (1958) Biochim Biophys Acta 30:<br>
443. Lipoproteins can also be purchased from commercial suppliers, such as Biomedical<br>
Techniologies, Inc., Stoughton, Massachusetts, USA. Further description of lipoproteins can be<br>
found in Zuckermann et al. PCT/US97/14465.<br>
F.Polvcationic Agents<br>
Polycationic agents can be included, with or without lipoprotein, in a composition with the desired<br>
polynucleotide/polypeptide to be delivered.<br>
Polycationic agents, typically, exhibit a net positive charge at physiological relevant pH and are<br>
capable of neutralizing the electrical charge of nucleic acids to facilitate delivery to a desired<br>
location. These agents have both in vitro, ex vivo, and in vivo applications. Polycationic agents can<br>
be used to deliver nucleic acids to a living subject either intramuscularly, subcutaneously, etc.<br>
The following are examples of useful polypeptides as polycationic agents: polylysinc, polyarginine,<br>
polyomithine, and protamine. Other examples include histones, protamines, human serum albumin,<br>
DNA binding proteins, non-histone chromosomal proteins, coat proteins from DNA viruses, such<br>
as (X174, transcriptional factors also contain domains that bind DNA and therefore may be useful<br>
as nucleic aid condensing agents. Briefly, transcriptional factors such as C/CEBP, c-jun, c-fos,<br>
AP-1, AP-2, AP-3, CPF, Prot-1, Sp-1, Oct-1, Oct-2, CREP, and TFIID contain basic domains that<br>
bind DNA sequences.<br>
Organic polycationic agents include: spermine, spermidine, and purtrescine.<br>
The dimensions and of the physical properties of a polycationic agent can be extrapolated from the<br>
list above, to construct other polypeptide polycationic agents or to produce synthetic polycationic<br>
agents.<br><br>
Synthetic polycationic agents which are useful include, for example, DEAE-dextran, polybrene.<br>
Lipofectin™, and lipofectAMINE™ are monomers that form polycationic complexes when<br>
combined with polynucleotides/polypeptides.<br>
Immunodiognostic Assays<br>
Neisserial antigens of the invention can be used in immunoassays to detect antibody levels (or,<br>
conversely, anti-Neisserial antibodies can be used to detect antigen levels). Immunoassays based<br>
on well defined, recombinant antigens can be developed to replace invasive diagnostics methods.<br>
Antibodies to Neisserial proteins within biological samples, including for example, blood or serum<br>
samples, can be detected. Design of the immunoassays is subject to a great deal of variation, and<br>
a variety of these are known in the art. Protocols for the immunoassay may be based, for example,<br>
upon competition, or direct reaction, or sandwich type assays. Protocols may also, for example, use<br>
solid supports, or may be by immunoprecipitation. Most assays involve the use of labeled antibody<br>
or polypeptide; the labels may be, for example, fluorescent, chemiluminescent, radioactive, or dye<br>
molecules. Assays which amplify the signals from the probe are also known; examples of which<br>
are assays which utilize biotin and avidin, and enzyme-labeled and mediated immunoassays, such<br>
as ELISA assays.<br>
Kits suitable for immunodiagnosis and containing the appropriate labeled reagents are constructed<br>
by packaging the appropriate materials, including the compositions of the invention, in suitable<br>
containers, along with the remaining reagents and materials (for example, suitable buffers, salt<br>
solutions, etc.) required for the conduct of the assay, as well as suitable set of assay instructions.<br>
Nucleic Acid Hybridisation<br>
"Hybridization" refers to the association of two nucleic acid sequences to one another by hydrogen<br>
bonding. Typically, one sequence will be fixed to a solid support and the other will be free in solution.<br>
Then, the two sequences will be placed in contact with one another under conditions that favor<br>
hydrogen bonding. Factors that affect this bonding include: the type and volume of solvent; reaction<br>
temperature; time of hybridization; agitation; agents to block the non-specific attachment of the liquid<br>
phase sequence to the solid support (Denhardfs reagent or BLOTTO); concentration of the sequences;<br>
use of compounds to increase the rate of association of sequences (dextran sulfate or polyethylene<br>
glycol); and the stringency of the washing conditions following hybridization. See Sambrook et al.<br>
[supra] Volume 2, chapter 9, pages 9.47 to 9.57.<br><br>
"Stringency" refers to conditions in a hybridization reaction that favor association of very similar<br>
sequences over sequences that differ. For example, the combination of temperature and salt<br>
concentration should be chosen that is approximately 120 to 200°C below the calculated Tm of the<br>
hybrid under study. The temperature and salt conditions can often be determined empirically in<br>
preJiminary experiments in which samples of genomic DNA immobilized on filters are hybridized<br>
to the sequence of interest and then washed under conditions of different stringencies. See<br>
Sambrook et al. at page 9.50.<br>
Variables to consider when performing, for example, a Southern blot are (1) the complexity of the<br>
DNA being blotted and (2) the homology between the probe and the sequences being detected. The<br>
total amount of the fragment(s) to be studied can vary a magnitude of 10, from 0.1 to 1 μg for a<br>
plasmid or phage digest to 10-9 to 10-8 g for a single copy gene in a highly complex eukaryotic<br>
genome. For lower complexity polynucleotides, substantially shorter blotting, hybridization, and<br>
exposure times, a smaller amount of starting polynucleotides, and lower specific activity of probes<br>
can be used. For example, a single-copy yeast gene can be detected with an exposure time of only<br>
1 hour starting with 1 ug of yeast DNA, blotting for two hours, and hybridizing for 4-8 hours with<br>
a probe of 103 cpm/ug. For a single-copy mammalian gene a conservative approach would start<br>
with 10 ug of DNA, blot overnight, and hybridize overnight in the presence of 10% dextran sulfate<br>
using a probe of greater than 108 cpm/ug, resulting in an exposure time of—24 hours.<br>
Several factors can affect the melting temperature (Tm) of a DNA-DNA hybrid between the probe<br>
and the fragment of interest, and consequently, the appropriate conditions for hybridization and<br>
washing. In many cases the probe is not 100% homologous to the fragment. Other commonly<br>
encountered variables include the length and total G+C content of the hybridizing sequences and<br>
the ionic strength and formamide content of the hybridization buffer. The effects of all of these<br>
factors can be approximated by a single equation:<br>
Tm= 81 + 16.6(log10Ci) + 0.4[%(G + C)]-0.6(%formamide) - 600/n-1.5(%mismatch).<br>
where Ci is the salt concentration (monovalent ions) and n is the length of the hybrid in base pairs<br>
(slightly modified from Meinkoth &amp; Wahl (1984) Anal Biochem. 138:267-284).<br><br>
In designing a hybridization experiment, some factors affecting nucleic acid hybridization can be<br>
conveniently altered. The temperature of the hybridization and washes and the salt concentration<br>
during the washes are the simplest to adjust. As the temperature of the hybridization increases (ie.<br>
stringency), it becomes less likely for hybridization to occur between strands that are<br>
nonhomologous, and as a result, background decreases. If the radiolabeled probe is not completely<br>
homologous with the immobilized fragment (as is frequently the case in gene family and<br>
interspecies hybridization experiments), the hybridization temperature must be reduced, and<br>
background will increase. The temperature of the washes affects the intensity of the hybridizing<br>
band and the degree of background in a similar manner. The stringency of the washes is also<br>
increased with decreasing salt concentrations.<br>
In general, convenient hybridization temperatures in the presence of 50% formamide are 42°C for<br>
a probe with is 95% to 100% homologous to the target fragment, 37°C for 90% to 95% homology,<br>
and 32°C for 85% to 90% homology. For lower homologies, formamide content should be lowered<br>
and temperature adjusted accordingly, using the equation above. If the homology between the probe<br>
and the target fragment are not known, the simplest approach is to start with both hybridization and<br>
wash conditions which are nonstringent If non-specific bands or high background are observed<br>
after autoradiography, the filter can be washed at high stringency and reexposed. If the time<br>
required for exposure makes this approach impractical, several hybridization and/or washing<br>
stringencies should be tested in parallel.<br>
Nucleic Acid Probe Assays<br>
Methods such as PCR, branched DNA probe assays, or blotting techniques utilizing nucleic acid<br>
probes according to the invention can determine the presence of cDNA or mRNA. A probe is said<br>
to "hybridize" with a sequence of the invention if it can form a duplex or double stranded complex,<br>
which is stable enough to be detected.<br>
The nucleic acid probes will hybridize to the Neisserial nucleotide sequences of the invention<br>
(including both sense and antisense stnmds). Though many different nucleotide sequences will<br>
encode the amino acid sequence, the native Neisserial sequence is preferred because it is the actual<br>
sequence present in cells. mRNA represents a coding sequence and so a probe should be<br>
complementary to the coding sequence; single-stranded cDNA is complementary to mRNA, and<br>
so a cDNA probe should be complementary to the non-coding sequence.<br><br>
The probe sequence need not be identical to the Neisserial sequence (or its complement) — some<br>
variation in the sequence and length can lead to increased assay sensitivity if the nucleic acid probe<br>
can form a duplex with target nucleotides, which can be detected. Also, the nucleic acid probe can<br>
include additional nucleotides to stabilize the formed duplex. Additional Neisserial sequence may<br>
also be helpful as a label to detect the formed duplex. For example, a non-complementary<br>
nucleotide sequence may be attached to the 5' end of the probe, with the remainder of the probe<br>
sequence being complementary to a Neisserial sequence. Alternatively, non-complementary bases<br>
or longer sequences can be interspersed into the probe, provided that the probe sequence has<br>
sufficient complementarity with the a Neisserial sequence in order to hybridize therewith and<br>
thereby form a duplex which can be detected.<br>
The exact length and sequence of the probe will depend on the hybridization conditions, such as<br>
temperature, salt condition and the like. For example, for diagnostic applications, depending on the<br>
complexity of the analyte sequence, the nucleic acid probe typically contains at least 10-20<br>
nucleotides, preferably 15-25, and more preferably at least 30 nucleotides, although it may be<br>
shorter than this. Short primers generally require cooler temperatures to form sufficiently stable<br>
hybrid complexes with the template.<br>
Probes may be produced by synthetic procedures, such as the triester method of Matteucci et al.<br>
[J. Am. Chem. Soc. (1981) 103:3185], or according to Urdea et al. [Proc. Natl. Acad. Sci. USA<br>
(1983) 80: 7461], or using commercially available automated oligonucleotide synthesizers.<br>
The chemical nature of the probe can be selected according to preference. For certain applications,<br>
DNA or RNA are appropriate. For other applications, modifications may be incorporated eg.<br>
backbone modifications, such as phosphorothioates or rnethylphosphonates, can be used to increase<br>
in vivo half-life, alter RNA affinity, increase nuclease resistance etc. [eg. see Agrawal &amp; Iyer<br>
(1995) Curr Opin Biotechnol 6:12-19; Agrawal (1996) TIBTECH 14376-387]; analogues such as<br>
peptide nucleic acids may also be used [eg. see Corey (1997) TIBTECH 15:224-229; Buchardt et<br>
al. (1993) TIBTECH 11:384-386J.<br>
Alternatively, the polymerase chain reaction (PCR) is another well-known means for detecting<br>
small amounts of target nucleic acids. The assay is described in: Mullis et al. [Meth. Enzymol.<br>
(1987) 155: 335-350]; US patents 4,683,195 and 4,683,202. Two "primer" nucleotides hybridize<br><br>
with the target nucleic acids and are used to prime the reaction. The primers can comprise sequence<br>
that does noi hybridize to the sequence of the amplification target (or its complement) to aid with<br>
duplex stability or, for example, to incorporate a convenient restriction site. Typically, such<br>
sequence will flank the desired Neisserial sequence.<br>
A thermostable polymerase creates copies of target nucleic acids from the primers using the<br>
original target nucleic acids as a template. After a threshold amount of target nucleic acids are<br>
generated by the polymerase, they can be detected by more traditional methods, such as Southern<br>
blots. When using the Southern blot method, the labelled probe will hybridize to the Neisserial<br>
sequence (or its complement).<br>
Also, mRNA or cDNA can be detected by traditional blotting techniques described in Sambrook<br>
et al [supra]. mRNA, or cDNA generated from mRNA using a polymerase enzyme, can be purified<br>
and separated using gel electrophoresis. The nucleic acids on the gel are then blotted onto a solid<br>
support, such as nitrocellulose. The solid support is exposed to a labelled probe and then washed<br>
to remove any unhybridized probe. Next, the duplexes containing the labeled probe are detected.<br>
Typically, the probe is labelled with a radioactive moiety.<br>
BRIEF DESCRIPTION OF THE ACCOMPANYING DRAWINGS<br>
Figures 1-7 show biochemical data and sequence analysis pertaining to Examples 1, 2, 3, 7, 13,<br>
16 and 19, respectively, with ORFs 40,38,44, 52,114,41 and 124.. Ml and M2 are molecular<br>
weight markers. Arrows indicate the position of the main recombinant product or, in Western blots,<br>
the position of the main N.meningitidis immunoreactive band. TP indicates N.meningitidis total<br>
protein extract; OMV indicates N.meningitidis outer membrane vesicle preparation. In bactericidal<br>
assay results: a diamond (♦) shows preimmune data; a triangle (A) shows GST control data; a<br>
circle (•) shows data with recombinant N.meningitidis protein. Computer analyses show a<br>
hydrophilicity plot (upper), an antigenic index plot (middle), and an AMPHI analysis (lower). The<br>
AMPHI program has been used to predict T-cell epitopes [Gao et al (1989) J. Immunol. 143:3007;<br>
Roberts et al. (1996) AIDS Res Hum Rctrovir 12:593; Quakyi et al. (1992) Scand J Immunol<br>
suppl.l 1:9) and is available in the Protean package of DNASTAR, Inc. (1228 South Park Street,<br>
Madison, Wisconsin 53715 USA).<br><br>
EXAMPLES<br>
The examples describe nucleic acid sequences which have been identified in N.meningitidis. along<br>
with their putative translation products. Not all of the nucleic acid sequences are complete ie. they<br>
encode less than the full-length wild-type protein It is believed at present that none of the DNA<br>
sequences described herein have significant homologs in N.gonorrhoeae.<br>
The examples are generally in the following format:<br>
•	a nucleotide sequence which has been identified in N.meningitidis (strain B)<br>
•	the putative translation product of this sequence<br>
•	a computer analysis of the translation product based on database comparisons<br>
•	a corresponding gene and protein sequence identified in N.meningitidis (strain A)<br>
•	a description of the characteristics of the proteins which indicates that they might be<br>
suitably antigenic<br>
•	results of biochemical analysis (expression, purification, ELISA.FACS etc.)<br>
The examples typically include details of sequence homology between species and strains. Proteins<br>
that are similar in sequence are generally similar in both structure and function, and the homology<br>
often indicates a common evolutionary origin. Comparison with sequences of proteins of known<br>
function is widely used as a guide for the assignment of putative protein function to a new sequence<br>
and has proved particularly useful in whole-genome analyses.<br>
Sequence comparisons were performed at NCBI (http://www.ncbi.nlm.nih.gov) using the<br>
algorithms BLAST, BLAST2, BLASTn, BLASTp, tBLASTn, BLASTx, &amp; tBLASTx [eg. see also<br>
Altschul et al. (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database<br>
search programs. Nucleic Acids Research 23:2289-3402]. Searches were performed against the<br>
following databases; non-redundant GenBank+EMBL+DDB J+PDB sequences and non-redundant<br>
GenBank CDS translations+PDB+SwissProt+SPupdate+PIR sequences<br>
Dots within nucleotide sequences (eg. position 288 in Example 12) represent nuckotides which<br>
have been arbitrarily introduced in order to maintain a reading frame. In the same way. double-<br>
underlined nucleotides were removed. Lower case letters (eg. position 589 in Example 12)<br>
represent ambiguities which arose during alignment of independent sequencing reactions (some of<br><br>
the nucleotide sequences in the examples are derived from combining the results of two or more<br>
experiments).<br>
Nucleotide sequences were scanned in all six reading frames to predict the presence of hydrophobic<br>
domains using an algorithm based on the statistical studies of Esposti et al. [Critical evaluation of<br>
the hydropathy of membrane proteins (1990) Eur J Biochem 190:207-219]. These domains<br>
represent potential transmembrane regions or hydrophobic leader sequences.<br>
Open reading frames were predicted from fragmented nucleotide sequences using the program<br>
ORFFINDER (NCBI).<br>
Underlined amino acid sequences indicate possible transmembrane domains or leader sequences<br>
in the ORFs, as predicted by the PSORT algorithm (http://www.psort.nibb.ac.jp). Functional<br>
domains were also predicted using the MOTIFS program (GCG Wisconsin &amp; PROSITE).<br>
Various tests can be used to assess the in vivo immunogenicity of the proteins identified in the<br>
examples. For example, the proteins can be expressed recombinantly and used to screen patient sera<br>
by immunoblot. A positive reaction between the protein and patient serum indicates that the patient<br>
has previously mounted an immune response to the protein in question ie. the protein is an<br>
immunogen. This method can also be used to identify immunodominant proteins.<br>
The recombinant protein can also be conveniently used to prepare antibodies eg. in a mouse. These<br>
can be used for direct confirmation that a protein is located on the cell-surface. Labelled antibody<br>
(eg. fluorescent labelling for FACS) can be incubated with intact bacteria and the presence of label<br>
on the bacterial surface confirms the location of the protein.<br>
In particular, the following methods (A) to (S) were used to express, purify and biochemically<br>
characterise the proteins of the invention:<br>
A) Chromosomal DNA preparation<br>
N.meningitidis strain 2996 was grown to exponential phase in 100ml of GC medium, harvested by<br>
centrifugation, and resuspended in 5ml buffer (20% Sucrose, 50mM Tris-HCl, 50mM EDTA, pH8).<br>
After 10 minutes incubation on ice, the bacteria were lysed by adding 10ml lysis solution (50mM<br>
NaCl, 1% Na-Sarkosyl, 50μg/ml Proteinase K), and the suspension was incubated at 37°C for 2<br><br>
hours. Two phenol extractions (equilibrated to pH 8) and one ChCl3/isoamylalcohol (24:1)<br>
extraction were performed. DNA was precipitated by addition of 0.3M sodium acetate and 2<br>
volumes ethanol, and was collected by centrifugation. The pellet was washed once with 70%<br>
ethanol and redissolved in 4ml buffer (lOmM Tris-HCl, lmM EDTA, pH 8). The DNA<br>
concentration was measured by reading the OD at 260 nm.<br>
B) Oligonucleotide design<br>
Synthetic oligonucleotide primers were designed on the basis of the coding sequence of each ORF,<br>
using (a) the meningococcus B sequence when available, or (b) the gonococcus/meningococcus A<br>
sequence, adapted to the codon preference usage of meningococcus as necessary. Any predicted<br>
signal peptides were omitted, by deducing the 5'-end amplification primer sequence immediately<br>
downstream from the predicted leader sequence.<br>
The 5' primers included two restriction enzyme recognition sites (Bam Hi-Ndel, BamHI-Nhel, or<br>
EcoRI-NheI, depending on the gene's own restriction pattern); the 3' primers included a Xhorestriction site. This procedure was established in order to direct the cloning of each amplification<br>
product (corresponding to each ORF) into two different expression systems: pGEX-KG (using<br>
cither BamHI-XhoI or EcoRI-XhoI), and pET21b+ (using either NdeI-XhoI or NheI-XhoI).<br><br>
As well as containing the restriction enzyme recognition sequences, the primers included<br>
nucleotides which hybridised to the sequence to be amplified. The number of hybridizing<br>
nucleoudes depended on the melting temperature of the whole primer, and was determined for each<br>
primer using the formulae:<br><br>
The average melting temperature of the selected oligos were 65-70°C for the whole oligo and<br>
50-55°C for the hybridising region alone.<br><br>
Table 1 shows the forward and reverse primers used for each amplification. Oligos were<br>
synthesized by a Perkin Elmer 394 DNA/RNA Synthesizer, eluted from the columns in 2ml<br>
NH4OH, and deprotected by 5 hours incubation at 56°C. The oligos were precipitated by addition<br>
of 0.3M Na-Acetate and 2 volumes ethanol. The samples were then centrifuged and the pellets<br>
resuspended in either 100μl or lml of water. OD260 was determined using a Perkin Elmer Lambda<br>
Bio spectrophotometer and the concentration was determined and adjusted to 2-10pmol/μl.<br>
C) Amplification<br>
The standard PCR protocol was as follows: 50-200ng of genomic DNA were used as a template<br>
in the presence of 20-40μM of each oligo, 400-800uM dNTPs solution, 1x PCR buffer (including<br>
1.5mM MgCl2), 2.5 units TaqI DNA polymerase (using Perkin-Elmer AmpliTaQ, GIBCO<br>
Platinum, Pwo DNA polymerase, or Tahara Shuzo Taq polymerase).<br>
In some cases, PCR was optimised by the addition of 10μl DMSO or 50μl 2M betaine.<br>
After a hot start (adding the polymerase during a preliminary 3 minute incubation of the whole mix<br>
at 95°C), each sample underwent a double-step amplification: the first 5 cycles were performed<br>
using as the hybridization temperature the one of the oligos excluding the restriction enzymes tail,<br>
followed by 30 cycles performed according to the hybridization temperature of the whole length<br>
oligos. The cycles were followed by a final 10 minute extension step at 72°C.<br><br><br><br>
The amplifications were performed using either a 9600 or a 2400 Perkin Elmer GeneAmp PCR<br>
System. To check the results, 1/10 of the amplification volume was loaded onto a 1-1.5% agarose<br>
gel and the size of each amplified fragment compared with a DNA molecular weight marker.<br>
The amplified DNA was either loaded directly on a 1 % agarose gel or first precipitated with ethanol<br>
and resuspended in a suitable volume to be loaded on a 1% agarose gel. The DNA fragment<br>
corresponding to the right size band was then eluted and purified from gel, using the Qiagen Gel<br>
Extraction Kit, following the instructions of the manufacturer. The final volume of the DNA<br>
fragment was 30μl or 50μl of either water or 10mM Tris, pH 8.5.<br>
D)	Digestion of PCR fragments<br>
The purified DNA corresponding to the amplified fragment was split into 2 aliquots and double-<br>
digested with:<br>
-	NdeI/XhoI or NheI/XhoI for cloning into pET-21 b+ and further expression of the protein<br>
as a C-terminus His-tag fusion<br>
-	BamHI/XhoI or EcoRI/XhoI for cloning into pGEX-KG and further expression of the<br>
protein as N-terminus GST fusion.<br>
-	EcoRI/PstI, EcoRI/Sall, Sall/PstI for cloning into pGex-His and further expression of<br>
the protein as N-terminus His-tag fusion<br>
Each purified DNA fragment was incubated (37°C for 3 hours to overnight) with 20 units of each<br>
restriction enzyme (New England Biolabs) in a either 30 or 40μl final volume in the presence of<br>
the appropriate buffer. The digestion product was then purified using the QlAquick PCR<br>
purification kit, following the manufacturer's instructions, and eluted in a final volume of 30 or<br>
50μl of cither water or 10mM Tris-HCl, pH 8.5. The final DNA concentration was determined by<br>
1% agarose gel electrophoresis in the presence of titrated molecular weight marker.<br>
E)	Digestion of the cloning vector* (pET22B, pGEX-KG, pTRC-His A, and pGex-His)<br>
10μg plasmid was double-digested with SO units of each restriction enzyme in 200μl reaction<br>
volume in the presence of appropriate buffer by overnight incubation at 37°C. After loading the<br><br>
whole digestion on a 1% agarose gel, the band corresponding to the digested vector was purified<br>
from the gel using the Qiagen QIAquick Gel Extraction Kit and the DNA was eluted in 50μl of<br>
10mM Tris-HCl, pH 8.5. The DNA concentration was evaluated by measuring OD260 of the sample,<br>
and adjusted to 50μg/μl. 1 μl of plasmid was used for each cloning procedure.<br>
The vector pGEX-His is a modified pGEX-2T vector carrying a region encoding six histidine<br>
residues upstream to the thrombin cleavage site and containing the multiple cloning site of the<br>
vector pTRC99 (Pharmacia).<br>
F) Cloning<br>
The fragments corresponding to each ORF, previously digested and purified, were ligated in both pET22b<br>
and pGEX-KG. In a final volume of 20μl, a molar ratio of 3:1 fragment/vector was ligated using 0.5μ1<br>
of NEB T4 DNA ligase (400 units/μl), in the presence of the buffer supplied by the manufacturer.<br>
The reaction was incubated at room temperature for 3 hours. In some experiments, ligation was<br>
performed using the Boehringer "Rapid Ligation Kit", following the manufacturer's instructions.<br>
In order to introduce the recombinant plasmid in a suitable strain, 100μl E. coli DH5 competent<br>
cells were incubated with the ligase reaction solution for 40 minutes on ice, then at 37°C for 3<br>
minutes, then, after adding 800μl LB broth, again at 37°C for 20 minutes. The cells were then<br>
centrifuged at maximum speed in an Eppendorf microfuge and resuspended in approximately 200μl<br>
of the supernatant The suspension was then plated on LB ampicillin (100mg/ml).<br>
The screening of the recombinant clones was performed by growing 5 randomly-chosen colonies<br>
overnight at 37°C in either 2ml (pGEX or pTC clones) or 5ml (pET clones) LB broth + 100μg/ml<br>
ampicillin. The cells were then pelletted and the DNA extracted using the Qiagen QIAprep Spin<br>
Mmiprep Kit, following the manufacturer's instructions, to a final volume of 30μl 5μl of each<br>
individual mmiprep (approximately 1g) were digested with either NdeIVXhol or BamHUXhol and<br>
the whole digestion loaded onto a 1 -1.5% agarose gel (depending on the expected insert size), in<br>
parallel with the molecular weight marker (1Kb DNA Ladder, GIBCO). The screening of the<br>
positive clones was made on the base of the correct insert size.<br><br>
G) Expression<br>
Each ORF cloned into the expression vector was transformed into the strain suitable for expression<br>
of the recombinant protein product, 1μl of each construct was used to transform 30ul of E. coli<br>
BL21 (pGEX vector), Exoli TOP 10 (pTRC vector) or E.coli BL21-DE3 (pET vector), as described<br>
above. In the case of the pGEX-His vector, the same E.coli strain (W3110) was used for initial<br>
cloning and expression. Single recombinant colonies were inoculated into 2ml LB+Amp<br>
(100μg/ml), incubated at 37°C overnight, then diluted 1:30 in 20ml of LB+Amp (100μg/ml) in<br>
100ml flasks, making sure that the OD600 ranged between 0.1 and 0.15. The flasks were incubated<br>
at 30°C into gyratory water bath shakers until OD indicated exponential growth suitable for<br>
induction of expression (0.4-0.8 OD for pET and pTRC vectors; 0.8-1 OD for pGEX and pGEX-<br>
His vectors). For the pET, pTRC and pGEX-His vectors, the protein expression was induced by<br>
addition of 1mM IPTG, whereas in the case of pGEX system the final concentration of IPTG was<br>
0.2mM. After 3 hours incubation at 30°C, the final concentration of the sample was checked by<br>
OD. In order to check expression, 1ml of each sample was removed, centrifuged in a microfuge,<br>
the pellet resuspended in PBS, and analysed by 12% SDS-PAGE with Coomassie Blue staining.<br>
The whole sample was centrifuged at 6000g and the pellet resuspended in PBS for further use.<br>
H) GST-fusion proteins large-scale purification.<br>
A single colony was grown overnight at 37°C on LB+Amp agar plate. The bacteria were inoculated<br>
into 20ml of LB+Amp liquid culture in a water bath shaker and grown overnight. Bacteria were<br>
diluted 1:30 into 600ml of fresh medium and allowed to grow at the optimal temperature (20-37°C)<br>
to OD550 0.8-1. Protein expression was induced with 0.2mM IPTG followed by three hours<br>
incubation. The culture was centrifuged at 8000rpm at 4°C. The supernatant was discarded and the<br>
bacterial pellet was resuspended in 7.5ml cold PBS. The cells were disrupted by sonication on ice<br>
for 30 sec at 40W using a Branson sonifier B-15, frozen and thawed twice and centrifuged again.<br>
The supernatant was collected and mixed with 150μl Glutanone-Sepharose 4B resin (Pharmacia)<br>
(previously washed with PBS) and incubated at room temperature for 30 minutes. The sample was<br>
centrifuged at 700g for 5 minutes at 4oC The resin was washed twice with 10ml cold PBS for 10<br>
minutes, resuspended in lml cold PBS, and loaded on a disposable column. The resin was washed<br>
twice with 2ml cold PBS until the flow-through reached OD280, of 0.02-0.06. The GST-fusion<br>
protein was eluted by addition of 700μl cold Glutathione elution buffer (10mM reduced<br><br><br>
glutathione, 50mM Tris-HCi) and fractions collected until the OD28() was 0.1. 21 μl of each fraction<br>
were loaded on a 12% SDS gel using either Biorad SDS-PAGE Molecular weight standard broad<br>
range (Ml) (200, 116.25, 97.4, 66.2, 45, 31, 21.5, 14.4, 6.5 kDa) or Amersham Rainbow Marker<br>
(M2) (220, 66, 46, 30, 21.5, 14.3 kDa) as standards. As the MW of GST is 26kDa, this value must<br>
be added to the MW of each GST-fusion protein.<br>
I) His-fusion solubility analysis<br>
To analyse the solubility of the His-fusion expression products, pellets of 3ml cultures were<br>
resuspended in buffer M1 [500ul PBS pH 7.2]. 25μl lysozyme (lOmg/ml) was added and the<br>
bacteria were incubated for 15 min at 4°C. The pellets were sonicated for 30 sec at 40W using a<br>
Branson sonifier B-15, frozen and thawed twice and then separated again into pellet and<br>
supernatant by a centrifugation step. The supernatant was collected and the pellet was resuspended<br>
in buffer M2 [8M urea, 0.5M NaCl, 20mM imidazole and 0.1M NaH3 PO4] and incubated for 3 to<br>
4 hours at 4°C. After centrifugation, the supernatant was collected and the pellet was resuspended<br>
in buffer M3 [6M guanidinium-HCl, 0.5M NaCl, 20mM imidazole and 0.1M NaH2PO4] overnight<br>
at 4°C. The supernatants from all steps were analysed by SDS-PAGE.<br>
J) His-fusion large-scale purification.<br>
A single colony was grown overnight at 37°C on a LB + Amp agar plate. The bacteria were<br>
inoculated into 20ml of LB+Amp liquid culture and incubated overnight in a water baft shaker.<br>
Bacteria were diluted 1:30 into 600ml fresh medium and allowed to grow at the optimal<br>
temperature (20-37°C) to OD550 0.6-0.8. Protein expression was induced by addition of 1mM IPTG<br>
and the culture further incubated for three hours. The culture was centrifuged at 8000rpm at 4°C,<br>
the supernatant was discarded and the bacterial pellet was resuspended in 7.5ml of either (i) cold<br>
buffer A (300mM NaCl, 50mM phosphate buffer, 10mM imidazole, pH 8) for soluble proteins or<br>
(ii) buffer B (urea 8M, 10mM Tris-HCl, 100mM phosphate buffer, pH 8.8) for insoluble proteins.<br>
The cells were disrupted by sonication on ice for 30 sec at 40W using a Branson sonifier B-15,<br>
frozen and thawed two times and centrifuged again.<br><br>
For insoluble proteins, the supernatant was stored at -20°C, while the pellets were resuspended in 2ml<br>
buffer C (6M guanidine hydrochloride, 10mM phosphate buffer, 10mM Tris-HCl, pH 7.5) and<br>
treated in a homogenizer for 10 cycles. The product was centrifuged at 13000rpm for 40 minutes.<br>
Supematants were collected and mixed with 150μl Ni2+-resin (Pharmacia) (previously washed with<br>
either buffer A or buffer B, as appropriate) and incubated at room temperature with gentle agitation<br>
for 30 minutes. The sample was centrifuged at 700g for 5 minutes at 4°C. The resin was washed<br>
twice with 10ml buffer A or B for 10 minutes, resuspended in 1ml buffer A or B and loaded on a<br>
disposable column. The resin was washed at either (i) 4°C with 2ml cold buffer A or (ii) room<br>
temperature with 2ml buffer B, until the flow-through reached OD280 of 0.02-0.06.<br>
The resin was washed with either (i) 2ml cold 20mM imidazole buffer (300mM NaCl, 50mM<br>
phosphate buffer, 20mM imidazole, pH 8) or (ii) buffer D (urea 8M, lOmM Tris-HCI, lOOmM<br>
phosphate buffer, pH 6.3) until the flow-through reached the O.D210 of 0.02-0.06. The His-fusion<br>
protein was eluted by addition of 700μl of either (i) cold elution buffer A (300mM NaCl, 50mM<br>
phosphate buffer, 250mM imidazole, pH 8) or (ii) elution buffer B (urea 8M, lOmM Tris-HCl,<br>
lOOmM phosphate buffer, pH 4.5) and fractions collected until the O.D280 was 0.1. 21μl of each<br>
fraction were loaded on a 12% SDS gel.<br>
K) His-fusion proteins renaturation<br>
10% glyceroi was added to the denatured proteins. The proteins were then diluted to 20ug/ml using<br>
dialysis buffer 1(10% giyceroi, O.SM arginine, 50mM phosphate buffer, SmM reduced glutathione,<br>
0.5mM oxidised glutathione, 2M urea, pH 8.8) and dialysed against the same buffer at 4°C for 12-<br>
14 hours. The protein was further dialysed against dialysis buffer II (10% giyceroi, O.SM arginine,<br>
50mM phosphate buffer, SmM reduced glutathione, 0.5mM oxidised glutathione, pH 8.8) for 12-14<br>
hours at 4°C. Protein concentration was evaluated using the formula:<br>
Protein (mg/ml) = (1.55 x OD280) - (0.76 x OD280)<br>
L) His-fusion large-teak purification<br>
500ml of bacterial cultures were induced and the fusion proteins were obtained soluble in buffer<br>
M1, M2 or M3 using the procedure described above. The crude extract of the bacteria was loaded<br><br>
onto a Ni-NTA superflow column (Qiagen) equilibrated with buffer Ml, M2 or M3 depending on<br>
the solubilization buffer of the fusion proteins. Unbound material was eluted by washing the<br>
column with the same buffer. The specific protein was eluted with the corresponding buffer<br>
containing 500mM imidazole and dialysed against the corresponding buffer without imidazole.<br>
After each run the columns were sanitized by washing with at least two column volumes of 0.5 M<br>
sodium hydroxide and reequilibrated before the next use.<br>
M) Mice immunisations<br>
20ug of each purified protein were used to immunise mice intraperitoneally. In the case of ORF 44,<br>
CD1 mice were immunised with Al(OH)3 as adjuvant on days 1,21 and 42, and immune response<br>
was monitored in samples taken on day 56. For ORF 40, CD1 mice were immunised using<br>
Freund's adjuvant, rather than Al(OH)3, and the same immunisation protocol was used, except that<br>
the immune response was measured on day 42, rather than 56. Similarly, for ORF 38, CD1 mice<br>
were immunised with Freund's adjuvant, but the immune response was measured on day 49.<br>
N) ELISA assay (sera analysis)<br>
The acapsulated MenB M7 strain was plated on chocolate agar plates and incubated overnight at<br>
37°C. Bacterial colonies were collected from the agar plates using a sterile dracon swab and<br>
inoculated into 7ml of Mueller-Hinton Broth (Difco) containing 0.25% Glucose. Bacterial growth<br>
was monitored every 30 minutes by following OD620. The bacteria were let to grow until the OD<br>
reached the value of 0.3-0.4. The culture was centrifuged for 10 minutes at 10000rpm. The<br>
supernatant was discarded and bacteria were washed once with PBS, resuspended in PBS<br>
containing 0.025% formaldehyde, and incubated for 2 hours at room temperature and then<br>
overnight at 4°C with stirring. 100μl bacterial cells were added to each well of a 96 well Greiner<br>
plate and incubated overnight at 4°C. The wells were then washed three times with PBT washing<br>
buffer (0.1% Tween-20 in PBS). 200μl of saturation buffer (2.7% Polyvinylpyrrolidone 10 in<br>
water) was added to each well and the plates incubated for 2 hours at 37°C. Wells were washed<br>
three times with PBT. 200ul of diluted sera (Dilution buffer 1% BSA, 0.1% Tween-20,0.1% NaN3<br>
in PBS) were added to each well and the plates incubated for 90 minutes at 37°C. Wells were<br>
washed three times with PBT. 100μl of HRP-conjugated rabbit anti-mouse (Dako) serum diluted<br>
1:2000 in dilution buffer were added to each well and the plates were incubated for 90 minutes at<br><br>
37°C. Wells were washed three times with PBT buffer. 100μ1 of substrate buffer for HRP (25ml<br>
of citrate buffer pH5, 10mg of O-phenildiamine and 10μl of H2O) were added to each well and the<br>
plates were left at room temperature for 20 minutes. 100μl H2SO4 was added to each well and OD490<br>
was followed. The EL1SA was considered positive when OD490 was 2.5 times the respective<br>
pre-immune sera.<br>
O) FACScan bacteria Binding Assay procedure.<br>
The acapsulated MenB M7 strain was plated on chocolate agar plates and incubated overnight at<br>
37°C. Bacterial colonies were collected from the agar plates using a sterile dracon swab and<br>
inoculated into 4 tubes containing 8ml each Mueller-Hinton Broth (Difco) containing 0.25%<br>
glucose. Bacterial growth was monitored every 30 minutes by following ODW0. The bacteria were<br>
let to grow until the OD reached the value of 0.35-0.5. The culture was centrifuged for 10 minutes<br>
at 4000rpm. The supernatant was discarded and the pellet was resuspended in blocking buffer (1%<br>
BSA, 0.4% NaN3) and centrifuged for 5 minutes at 4000rpm. Cells were resuspended in blocking<br>
buffer to reach OD620 of 0.07. 100μl bacterial cells were added to each well of a Costar 96 well<br>
plate. lOOuJ of diluted (1:200) sera (in blocking buffer) were added to each well and plates<br>
incubated for 2 hours at 4oC. Cells were centrifuged for 5 minutes at 4000rpm, the supernatant<br>
aspirated and cells washed by addition of 200μl/well of blocking buffer in each well. 100μl of R-<br>
Phicoerytrin conjugated F(ab)2 goat anti-mouse, diluted 1:100, was added to each well and plates<br>
incubated for 1 hour at 4°C. Cells were spun down by centrifugation at 4000rpm for 5 minutes and<br>
washed by addition of 200μl/well of blocking buffer. The supernatant was aspirated and cells<br>
resuspended in 200μl/well of PBS, 025% formaldehyde. Samples were transferred to FACScan<br>
tubes and read. The condition for FACScan setting were: FL1 on, FL2 and FL3 off; FSC-H<br>
threshold:92; FSC PMT Voltage: E 02; SSC PMT: 474; Amp. Gains 7.1; FL-2 PMT: 539;<br>
compensation values: 0.<br>
P) OMV preparations<br>
Bacteria were grown overnight on 5 GC plates, harvested with a loop and resuspended in 10ml 20mM<br>
Tris-HCl. Heat inactivation was performed at 56°C for 30 minutes and the bacteria disrupted by<br>
sonication for 10 minutes on ice (50% duty cycle, 50% output). Unbroken cells were removed by<br>
centrifugation at 5000g for 10 minutes and the total cell envelope fraction recovered by centrifugation<br><br>
at 50000g at 4°C for 75 minutes. To extract cytoplasmic membrane proteins from the crude outer<br>
membranes, the whole fraction was resuspended in 2% sarkosyl (Sigma) and incubated at room<br>
temperature for 20 minutes. The suspension was centrifuged at 10000g for 10 minutes to remove<br>
aggregates, and the supernatant further ultracentrifuged at 50000g for 75 minutes to pellet the outer<br>
membranes. The outer membranes were resuspended in 10mM Tris-HCl, pH8 and the protein<br>
concentration measured by the Bio-Rad Protein assay, using BSA as a standard.<br>
Q) Whole Extracts preparation<br>
Bacteria were grown overnight on a GC plate, harvested with a loop and resuspended in 1ml of<br>
20mM Tris-HCl. Heat inactivation was performed at 56°C for 30 minutes.<br>
R) Western blotting<br>
Purified proteins (500ng/lane), outer membrane vesicles (5μg) and total cell extracts (25μg) derived<br>
from MenB strain 2996 were loaded on 15% SDS-PAGE and transferred to a nitrocellulose<br>
membrane. The transfer was performed for 2 hours at 150mA at 4°C, in transferring buffer (0.3 %<br>
Tris base, 1.44 % glycine, 20% methanol). The membrane was saturated by overnight incubation<br>
at 4°C in saturation buffer (10% skimmed milk, 0.1 % Triton X100 in PBS). The membrane was<br>
washed twice with washing buffer (3% skimmed milk, 0.1 % Triton X100 in PBS) and incubated<br>
for 2 hours at 37°C with mice sera diluted 1:200 in washing buffer. The membrane was washed<br>
twice and incubated for 90 minutes with a 1:2000 dilution of horseradish peroxidase labelled anti-<br>
mouse Ig. The membrane was washed twice with 0.1% Triton X100 in PBS and developed with<br>
the Opti-4CN Substrate Kit (Bio-Rad). The reaction was stopped by adding water.<br>
S) Bactericidal assay<br>
MC58 strain was grown overnight at 37°C on chocolate agar plates. 5-7 colonies were collected and<br>
used to inoculate 7ml Muelier-Hinton broth. The suspension was incubated at 37°C on a nutator<br>
and let to grow until OD620, was 0.5-0.8. The culture was aliquoted into sterile 1.5ml Eppendorf<br>
tubes and centrifuged for 20 minutes at maximum speed in a microfuge. The pellet was washed<br>
once in Gey's buffer (Gibco) and resuspended in the same buffer to an OD6M of 0.5, diluted<br>
1:20000 in Gey's buffer and stored at 25°C.<br><br><br><br><br><br><br><br><br><br><br><br>
results of expression of the GST-fusion in E.coli. Purified His-fusion protein was used to immunise<br>
mice, whose sera were used for FACS analysis (Figure 1C), a bactericidal assay (Figure 1D), and<br>
ELISA (positive result). These experiments confirm that ORF40-1 is a surface-exposed protein, and<br>
that it is a useful immunogen.<br><br><br><br><br><br>
ORF38-1 (32kDa) was cloned in pET and pGex vectors and expressed in Exoli, as described<br>
above. The products of protein expression and purification were analyzed by SDS-PAGE. Figure<br>
2A shows the results of affinity purification of the His-fusion protein, and Figure 2B shows the<br>
results of expression of the GST-fusion in Exoli. Purified His-fusion protein was used to immunise<br>
mice, whose sera were used for Western blot analysis (Figure 2C) and FACS analysis (Figure 2D).<br>
These experiments confirm that ORF38-1 is a surface-exposed protein, and that it is a useful<br>
immunogen.<br>
Figure 2E shows plots of hydrophilicity, antigenic index, and AMPHI regions for ORF38-1.<br>
Example 3<br>
The following N.meningitidis DNA sequence was identified <seq id>:<br><br><br><br>
LecA 194 TTNAMDSANYRSQDIIVSAPNGQMLYKDCSF 224<br>
Based on homology with the adhesin, it was predicted that this protein from N.meningitidis. and<br>
its epitopes, could be useful antigens for vaccines or diagnostics.<br>
ORF44-1 (11.2kDa) was cloned in pET and pGex vectors and expressed in E.coli, as described<br>
above. The products of protein expression and purification were analyzed by SDS-PAGE. Figure<br>
3 A shows the results of affinity purification of the His-fusion protein, and Figure 3B shows the<br>
results of expression of the GST-fusion in E.coli. Purified His-fusion protein was used to immunise<br>
mice, whose sera were used for ELISA, which gave positive results, and for a bactericidal assay<br>
(Figure 3C). These experiments confirm that ORF44-1 is a surface-exposed protein, and that it is<br>
a useful immunogen.<br>
Figure 3D shows plots of hydrophilicity, antigenic index, and AMPHI regions for ORF44-1.<br><br><br><br><br><br><br><br><br>
Computer analysis predicts two transmembrane domains and also indicates that ORF50 has no<br>
significant amino acid homology with known proteins.<br>
Based on the presence of a putative transmembrane domain, it is predicted that this protein from<br>
N.meningitidis, and its epitopes, could be useful antigens for vaccines or diagnostics.<br><br><br><br><br><br><br><br><br><br><br><br>
Computer analysis of this amino acid sequence predicts a prokaryotic membrane lipoprotein lipid<br>
attachment site (underlined).<br>
ORF52-1 (7kDa) was cloned in the pGex vectors and expressed in E.coli, as described above. The<br>
products of protein expression and purification were analyzed by SDS-PAGE. Figure 4 A shows<br>
the results of affinity purification of the GST-fusion. Figure 4B shows plots of hydrophilicity,<br>
antigenic index, and AMPHI regions for ORF52-1.<br>
Based on this analysis, it is predicted that this protein from N.meningitidis, and its epitopes, could<br>
be useful antigens for vaccines or diagnostics.<br><br><br><br><br><br>
Computer analysis of this amino acid sequence predicts a transmembrane region.<br>
A corresponding ORF from strain A of N.meningitidis was also identified:<br><br><br><br><br><br><br><br><br><br><br>
orf 112-1	LKLFGGICXGLLFHLAGRLFGFTSQL<br>
Based on this analysis, it is predicted that this protein from N.meningitidis, and its epitopes. could<br>
be useful antigens for vaccines or diagnostics.<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
Amino acids 1-1423 of ORF114-1 were cloned in the pGex vector ami expressed in E.colit as<br>
described above. GST-fusion expression was visible using SDS-PAGE, and Figure S shows plots<br>
of hydrophilicity, antigenic index, and AMPHI regions for ORF114-1.<br>
Based on these results, including the homology with the putative aeactcd protein of N.meningitidis<br>
and on the presence of a transmembrane domain, it is predicted that this protein from<br>
N.meningitidis, and its epitopes, could be useful antigens for vaccines or diagnostics.<br>
Example 14<br>
The following partial DNA sequence was identified in N.meningitidis <seq id><br><br><br><br><br><br><br><br><br><br><br><br><br>
Amino acids 25-619 of ORF41-1 were amplified as described above. Figure 6 shows plots of<br>
hydrophilichy, antigenic index, and AMPHI regions for ORF41-1.<br>
Based on this analysis, it is predicted that this protein from N.meningitidis, and its epitopes, could<br>
be useful antigens for vaccines or diagnostics.<br><br><br><br><br><br><br><br><br><br><br><br><br><br>
ORF124-1 was amplified as described above. Figure 7 shows plots of hydrophilicity, antigenic<br>
index, and AMPHI regions for ORF 124-1.<br>
Based on this analysis, it is predicted that this protein from N meningitidis, and its epitopes, could<br>
be useful antigens for vaccines or diagnostics.<br>
It will be appreciated that the invention has been described by means of example only, and that<br>
modifications may be made whilst remaining within the spirit and scope of the invention.<br><br><br><br><br><br>
WE CLAIM<br>
1.	A protein comprising amino acid sequence SEQ ID NO: 2.<br>
2.	The protein of claim 1, comprising amino acid sequence SEQ ID NO: 4.<br><br>
3.	A protein comprising an amino acid sequence with 80% or greater sequence identity to SEQ<br>
ID NO: 2, provided that said protein is not (i) a protein having amino acid sequence SEQ ID NO: 7,<br>
or (ii) a fragment of SEQ ID NO: 7, wherein the fragment has 6 or more amino acids of SEQ ID NO:<br>
7 and includes an antigenic determinant.<br>
4.	The protein of claim 3, comprising an amino acid sequence with 80% or greater sequence<br>
identity to SEQ ID NO: 4.<br>
5. The protein of claim 3, comprising an amino acid sequence with 90% or greater sequence<br>
identity to SEQ ID NO: 2.<br>
6. The protein of claim 4, comprising an amino acid sequence with 90% or greater sequence<br>
identity to SEQ ID NO: 4.<br>
7. The protein of claim 5, comprising an amino acid sequence with 95% or greater sequence<br>
identity to SEQ ID NO: 2.<br>
8. The protein of claim 6, comprising an amino acid sequence with 95% or greater sequence<br>
identity to SEQ ID NO: 4.<br>
9. The protein of claim 7, comprising an amino acid sequence with 99% or greater sequence<br>
identity to SEQ ID NO: 2.<br>
10. The protein of claim 8, comprising an amino acid sequence with 99% or greater sequence<br>
identity to SEQ ID NO: 4.<br>
11.	The protein of claim 4, comprising amino acid sequence SEQ ID NO: 6.<br>
12. A protein comprising a fragment of at least 20 consecutive amino acids of amino acid<br>
sequence SEQ ID NO: 2, provided that said protein is not (i) a protein having amino acid sequence<br>
SEQ ID NO: 7, or (ii) a fragment of SEQ ID NO: 7, wherein the fragment has 6 or more amino acids<br>
of SEQ ID NO' 7 and includes an antigenic determinant.<br>
13.	The protein of claim 12, wherein said fragment comprises an epitope from SEQ ID NO: 1.<br>
14.	The protein of claim 12, comprising a fragment of at least 20 consecutive amino acids of<br>
amino acid sequence SEQ ID NO: 4.<br>
15.	The protein of claim 14, wherein said fragment comprises an epitope from SEQ ID NO: 4.<br>
116.	The protein of any preceding claim, wherein the protein is a fusion protein.<br>
17.	The protein of claim 16, which is a fusion with GST.<br>
18.	A protein of claim 17, which is a fusion with six histidine residues.<br><br>
19.	A protein comprising (i) an amino acid sequence with 80% or greater sequence identity to<br>
SEQ ID NO: 2 or (ii) a fragment of at least 20 consecutive amino acids of amino acid sequence SEQ<br>
ID NO; 2, wherein the protein is a fusion protein including six histidine residues.<br>
20.	The protein of any preceding claim, which is prepared by recombinant expression.<br>
2 1.	The protein of any one of claims 1 to 19, which is prepared by purification from cell culture.<br>
22.	An antibody which binds to a protein according to any one of claims 3 to 18.<br>
23.	The antibody of claim 22, which is monoclonal.<br>
24.	The antibody of claim 22, which is polyclonal.<br>
25.	A nucleic acid molecule which encodes a protein according to any one of claims 1 to 20.<br>
26.	The nucleic acid molecule of claim 25, comprising a fragment of at least 10 consecutive<br>
nucleotides from a nucleotide sequence selected from the group consisting of SEQ ID NOs: 1,3, &amp; 5<br>
27.	The nucleic acid molecule of claim 26, comprising a fragment of at least 12 consecutive<br>
riucieotides from said group.<br>
18. The nucleic acid molecule of claim 26, comprising a fragment of at least 14 consecutive<br>
nucleotides from said group.<br>
29. The nucleic acid molecule of claim 26, comprising a fragment of at least 15 consecutive<br>
nucleotides from said group.<br>
30.	The nucleic acid molecule of claim 26, comprising a fragment of at least 18 consecutive<br>
nucleotides from said group.<br>
31.	The nucleic acid molecule of claim 26, comprising a fragment of at least 20 consecutive<br>
nucleotides from said group.<br>
32. The nucleic acid molecule of claim 26, comprising a fragment of at least 25 consecutive<br>
nucleotides from said group.<br>
33. The nucleic acid molecule of claim 26, comprising a fragment of at least 30 consecutive<br>
nucleotides from said group.<br>
34 The nucleic acid molecule of claim 26, comprising a fragment of at least 35 consecutive<br>
nucleotides from said group.<br>
35. The nucleic acid molecule of claim 26, comprising a fragment of at least 40 consecutive<br>
nucleotides from said group.<br>
36.	The nucleic acid molecule of claim 35, comprising a nucleotide sequence selected from the<br>
group consisting of SEQ ID NO:1 and SEQ ID NO:3.<br>
37.	The nucleic acid molecule of claim 36, comprising nucleotide sequence SEQ ID NO: 1,<br>
38. A nucleic acid molecule comprising a nucleotide sequence complementary to a nucleic acid<br>
molecule according to any one of claims 25 to 37.<br><br>
39. A nucleic acid molecule comprising a nucleotide sequence having 80% or greater sequence<br>
identity to a nucleic acid molecule according to any one of claims 25 to 37.<br>
4D. A nucleic acid molecule which can hybridise to a nucleic acid molecule according to any one<br>
of claims 25 to 37 under high stringency conditions.<br>
41. The nucleic acid molecule of any one of claims 25 to 40, which is DNA.<br>
42. A vector comprising the nucleic acid of any one of claims 25 to 41.<br>
43. The vector of claim 42, which is an expression vector.<br>
44. A bacterial expression vector encoding (i) a protein comprising an amino acid sequence with<br>
80% or greater sequence identity to SEQ ID NO: 2 or (ii) a protein comprising a fragment of at least<br>
20 consecutive amino acids of amino acid sequence SEQ ID NO: 2, wherein the vector includes a<br>
Shine-Dalgamo sequence.<br>
45. A host cell transformed with the vector of claim 42 or claim 43 or claim 44.<br>
46. The host cell of claim 45, which is a bacterium, yeast, or mammalian cell, <br>
47. The host cell of claim 46, which is a bacterium.<br>
48. A bacterial host cell transformed with a vector encoding (i) a protein comprising an amino<br>
acid sequence with 80% or greater sequence identity to SEQ ID NO: 2 or (ii) a protein comprising a<br>
fragment of at least 20 consecutive amino acids of amino acid sequence SEQ ID NO: 2, provided<br>
that the host cell is not a Salmonella.<br>
19. The host cell of claim 47 or claim 48, where the vector includes a bacterial promoter.<br>
20. The host cell of claim 49, wherein the bacterial promoter is from a metabolic pathway<br>
enzyme.<br>
51.	The host cell of any one of claims 45 to 50, wherein the bacterium is an E.coli.<br>
52.	A composition comprising a protein, a nucleic acid molecule, or an antibody according to<br>
any one of claims 1 to 41.<br>
53.	A composition according to claim 52 being a vaccine composition or a diagnostic<br>
composition.<br>
54.	The composition of claim 52, which is a vaccine composition.<br>
55.	A composition according to claim 52, for use as a medicament.<br>
56.	The composition of any one of claims 52 to 54, including a pharmaceutically acceptable<br>
carrier.<br>
57 The composition of claim 56, wherein the carrier comprises a polysaccharide.<br>
58 The composition of claim 56, wherein the carrier comprises saline.<br>
59 The composition of claim 56, wherein the carrier comprises a pH buffering substance.<br><br>
60. A composition comprising; (i) a protein comprising an amino acid sequence with 80% or<br>
greater sequence identity to SEQ ID NO: 2, or a protein comprising a fragment of at least 20<br>
consecutive amino acids of amino acid sequence SEQ ID NO: 2; and (ii) a pharmaceutically<br>
acceptable carrier comprising saline, a polysaccharide, a polylactic acid, a polyglycolic acid, or a<br>
pH buffering substance.<br>
61. The composition of any one of claims 52 to 60, prepared as an injectable.<br>
62. The composition of any one of claims 52 to 59, prepared as a solid form suitable for solution<br>
n, or suspension in, a liquid vehicle prior to injection.<br>
63. A composition comprising: (i) a protein comprising an amino acid sequence with 80% or<br>
greater sequence identity to SEQ ID NO: 2, or a protein comprising a fragment of at least 20<br>
consecutive amino acids of amino acid sequence SEQ ID NO: 2; and (ii) a pharmaceuticaily<br>
acceptable carrier, wherein the composition is an injectable.<br>
64. The composition of any one of claims 52 to 60, including an adjuvant.<br>
65. . The composition of claim 64, wherein the adjuvant is selected from: an aluminum salt; an<br>
oil-in-water emulsion; a saponin; or a cytokine.<br>
66.	The composition of claim 65, wherein the adjuvant is an aluminum hydroxide or an<br>
aluminium phosphate salt.<br>
67.	A composition comprising: (i) a protein comprising an amino acid sequence with 80% or<br>
greater sequence identity to SEQ ID NO: 2, or a protein comprising a fragment of at least 20<br>
consecutive amino acids of amino acid sequence SEQ ID NO: 2; and (ii) an adjuvant selected from<br>
aluminum hydroxide, aluminum phosphate, or aluminum sulfate.<br>
68.	The composition of any one of claims 52 to 55, including chitosan.<br>
69.	The composition of any one of claims 52 to 55, including poly(lactide-co-glycolide).<br>
70. The use of the protein of any one of claims 1 to 20, in the manufacture of a medicament for<br>
the treatment or prevention of infection due to Neisserial bacteria, particularly Neisseria<br>
meningitidis.<br>
71. The use of (i) a protein comprising an amino acid sequence with 80% or greater sequence<br>
identity to SEQ ID NO: 2, or (ii) a protein comprising a fragment of at least 20 consecutive amino<br>
acids of amino acid sequence SEQ ID NO: 2, in the manufacture of a medicament for the treatment<br>
or prevention of infection due to Neisseria meningitidis serogroup B.<br>
72. A process for producing the protein of any one of claims 1 to 20, comprising the step of<br>
culturing a host cell of any one of claims 45 to 51 under conditions which induce protein expression.<br><br>
The invention provides proteins from Naisseria maningitidas<br>
(strains A and B), including amino acid sequences, the<br>
corresponding nucleotide sequences, expression data and<br>
serological data. The proteins are useful antigens for<br>
vaccines, immunogenic compositions, and/or diagnostics.</seq></seq></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDAtMTA5LUtPTC1GT1JNIDI3LnBkZg==" target="_blank" style="word-wrap:break-word;">IN-PCT-2000-109-KOL-FORM 27.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDAtMTA5LUtPTC1GT1JNLTI3LnBkZg==" target="_blank" style="word-wrap:break-word;">IN-PCT-2000-109-KOL-FORM-27.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMTA5LWtvbC1ncmFudGVkLWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2000-109-kol-granted-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMTA5LWtvbC1ncmFudGVkLWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-109-kol-granted-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMTA5LWtvbC1ncmFudGVkLWNvcnJlc3BvbmRlbmNlLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2000-109-kol-granted-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMTA5LWtvbC1ncmFudGVkLWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-109-kol-granted-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMTA5LWtvbC1ncmFudGVkLWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2000-109-kol-granted-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMTA5LWtvbC1ncmFudGVkLWV4YW1pbmF0aW9uIHJlcG9ydC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-109-kol-granted-examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMTA5LWtvbC1ncmFudGVkLWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-109-kol-granted-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMTA5LWtvbC1ncmFudGVkLWZvcm0gMTgucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-109-kol-granted-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMTA5LWtvbC1ncmFudGVkLWZvcm0gMi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-109-kol-granted-form 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMTA5LWtvbC1ncmFudGVkLWZvcm0gMjYucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-109-kol-granted-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMTA5LWtvbC1ncmFudGVkLWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-109-kol-granted-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMTA5LWtvbC1ncmFudGVkLWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-109-kol-granted-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMTA5LWtvbC1ncmFudGVkLXJlcGx5IHRvIGV4YW1pbmF0aW9uIHJlcG9ydC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-109-kol-granted-reply to examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMTA5LWtvbC1ncmFudGVkLXNwZWNpZmljYXRpb24ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-109-kol-granted-specification.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMTA5LWtvbC1ncmFudGVkLXRyYW5zbGF0ZWQgY29weSBvZiBwcmlvcml0eSBkb2N1bWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-109-kol-granted-translated copy of priority document.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="228097-an-improved-cleaning-system.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="228099-a-py-rich-immunostimulatory-nucleic-acid.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>228098</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2000/109/KOL</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>05/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>30-Jan-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>28-Jan-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>26-Jun-2000</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CHIRON SPA</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>VIA FIORENTINA 1, I-53100 SIENA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>MASIGNANI VEGA</td>
											<td>VIA PANTANETO 105 53100 SIENA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>RAPPUOLI RINO</td>
											<td>VIA DELLE ROCCHE 1, VAGLIAGLI, 53019 CASTELNUOVO BERARDENGA (SI)</td>
										</tr>
										<tr>
											<td>3</td>
											<td>PIZZA MARIAGRAZIA</td>
											<td>STRADA DI MONTALBUCCIO, 160 53100 SIENA</td>
										</tr>
										<tr>
											<td>4</td>
											<td>SCARLATO VINCENZO</td>
											<td>VIA FIRENZE, 3/37 53134 COLLE VAL D&#x27;ELSA (SI)</td>
										</tr>
										<tr>
											<td>5</td>
											<td>GRANDI GUIDO</td>
											<td>9° STRADA, 4, 20090 SEGRATE (MI)</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/31</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IB99/00103</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>1999-01-14</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>9822143.5</td>
									<td>1998-10-09</td>
								    <td>U.K.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>9800760.2</td>
									<td>1998-01-14</td>
								    <td>U.K.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>9819015.0</td>
									<td>1998-09-01</td>
								    <td>U.K.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/228098-neisseria-menningitidis-antigens by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:40:06 GMT -->
</html>
