<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/228100-an-isolated-polynucleotide by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:40:13 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 228100:AN ISOLATED POLYNUCLEOTIDE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">AN ISOLATED POLYNUCLEOTIDE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention provides methods for identifying polynucleotide and polypeptide sequences which may be associated with commercially or aesthetically relevant traits in domesticated plants or animals. The methods employ comparison of homologous genes from the domesticated organism and its ancestor to identify evolutionarily significant changes and evolutionarily neutral changes. Sequences thus identified may be useful in enhancing commercially or aesthetically desirable traits in domesticated organisms or their wild ancestors.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>METHODS TO IDENTITY EVOLUTIONARILY SIGNIFICANT<br>
CHANGES IN POLYNUCLEOTIDE AND POLYPEPTIDE<br>
SEQUENCES IN DOMESTICATED PLANTS AND ANIMALS<br>
AN ISOLATED POLYNUCLEOTIDE<br>
TECHNICAL FIELD<br>
This invention relates to using molecular and evolutionary techniques to identify<br>
polynucleotide and polypeptide sequences corresponding to commercially or aesthetically<br>
relevant traits in domesticated plants and animals.<br>
BACKGROUND ART<br>
Humans have bred plants and animals for thousands of years, selecting for certain<br>
commercially valuable and/or aesthetic traits. Domesticated plants differ from their wild<br>
ancestors in such traits as yield, short day length flowering, protein and/or oil content, ease of<br>
harvest, taste, disease resistance and drought resistance. Domesticated animals differ from<br>
their wild ancestors in such traits as fat and/or protein content, milk production, docility,<br>
fecundity and time to maturity. At the present time, most genes underlying the above<br>
differences are not known, nor, as importantly, are the specific changes that have evolved in<br>
these genes to provide these capabilities. Understanding the basis of these differences<br>
between domesticated plants and animals and their wild ancestors will provide useful<br>
information for maintaining and enhancing those traits. In the case of crop plants,<br>
identification of the specific genes that control desired traits will allow direct and rapid<br>
improvement in a manner not previously possible.<br>
Although comparison of homologous genes or proteins between domesticated species<br>
and their wild ancestors may provide useful information with respect to conserved molecular<br>
sequences and functional features, this approach is of limited use in identifying genes whose<br>
sequences have changed due to human imposed selective pressures. Wim the advent of<br>
sophisticated algorithms and analytical methods, much more information can be teased out of<br>
DNA sequence changes with regard to which genes have been positively selected. The most<br>
powerful of these methods, "KA/Ks," involves pairwise comparisons between aligned protein-<br>
coding nucleotide sequences of the ratios of<br>
nonsynonvmous nucleotide substitutions per nonsvnonymous site (KA)<br>
synonymous substitutions per synonymous site (KS)<br>
(where nonsynonvmous means substitutions that change the encoded amino acid and<br>
synonymous means substitutions that do not change the encoded amino acid). "K.A/Ks-type<br>
methods" include this and similar methods.<br><br>
These methods have been used to demonstrate the occurrence of Darwinian (i.e<br>
natural) molecular-level positive selection, resulting in amino acid differences in homologous<br>
proteins. Several groups have used such methods to document that a particular protein has<br>
evolved more rapidly than the neutral substitution rate, and thus supports the existence of<br>
Darwinian molecular-level positive selection. For example, McDonald and Kreitman (1991)<br>
Nature 351:652-654, propose a statistical test of the neutral protein evolution hypothesis<br>
based on comparison of the number of amino acid replacement substitutions to synonymous<br>
substitutions in the coding region of a locus. When they apply this test to the Adh locus of<br>
three Drosophila species, they conclude that it shows instead that the locus has undergone<br>
adaptive fixation of selectively advantageous mutations and that selective fixation of adaptive<br>
mutations may be a viable alternative to the clocklike accumulation of neutral mutations as an<br>
explanation for most protein evolution. Jenkins et al. (1995) Proc. R. Soc Lond. B 261:203-<br>
207 use the McDonald &amp; Kreitman test to investigate whether adaptive evolution is occurring<br>
in sequences controlling transcription (non-coding sequences).<br>
Nakashima et al. (1995) Proc. Natl. Acad. Sci USA 92:5606-5609, use the method of<br>
Miyata and Yasunaga to perform pairwise comparisons of the nucleotide sequences often<br>
PLA2 isozyme genes from two snake species; this method involves comparing the number of<br>
nucleotide substitutions per site for the noncoding regions including introns (KN) and the KA<br>
and Ks. They conclude that the protein coding regions have been evolving at much higher<br>
rates than the noncoding regions including introns. The highly accelerated substitution rate is<br>
responsible for Darwinian molecular-level evolution of PLA2 isozyme genes to produce new<br>
physiological activities that must have provided strong selective advantage for catching prey<br>
or for defense against predators. Endo et al (1996) Mol Biol. Evol 13(5):685-690 use the<br>
method of Nei and Gojobori, wherein dN is the number of nonsynonymous substitutions and<br>
ds is the number of synonymous substitutions, for the purpose of documenting natural<br>
selection on genes. Metz and Palumbi (1996) Mol Biol Evol 13(2):397-406 use the<br>
McDonald &amp; Kreitman (supra) test as well as a method attributed to Nei and Gojobori, Nei<br>
and Jin, and Kumar, Tamura, and Nei; examining the average proportions of Pn, the<br>
replacement substitutions per replacement site, and Ps, the silent substitutions per silent site,<br>
to look for evidence of positive selection on binding genes in sea urchins to investigate<br>
whether they have rapidly evolved as a prelude to species formation. Goodwin et al. (1996)<br>
Mol Biol Evol. 13(2):346-358 uses similar methods to examine the evolution of a particular<br>
murine gene family and conclude that the methods provide important fundamental insights<br>
into how selection drives genetic divergence in an experimentally manipulatable system,<br><br><br>
Edwards et al (1995) use degenerate primers to pull out Aff/C loci from various species of<br>
birds and an alligator species, which are then analyzed by the Nei and Gojobori methods (dN:<br>
ds ratios) to extend MHC studies to nonmammalian vertebrates. Whitfield et al. (1993)<br>
Nature 364:713-715 use KA/Ks analysis to look for directional selection in the regions<br>
flanking a conserved region in the SKY gene (that determines male sex). They suggest that the<br>
rapid evolution of SRY could be a significant cause of reproductive isolation, leading to new<br>
species. Wettsetin et al. (1996) Mol. Biol. Evol. 13(l):56-66 apply the MEGA program of<br>
Kumar, Tamura and Nei and phylogenetic analysis to investigate the diversification of MHC<br>
class I genes in squirrels and related rodents. Parham and Ohta (1996) Science 212:61-14<br>
state that a population biology approach, including tests for selection as well as for gene<br>
conversion and neutral drift are required to analyze the generation and maintenance of human<br>
MHC class I polymorphism. Hughes (1997) Mol Biol. Evol. 14(1): 1-5 compared over one<br>
hundred orthologous immunoglobulin C2 domains between human and rodent, using the<br>
method of Nei and Gojobori (dN: dS ratios) to test the hypothesis that proteins expressed in<br>
cells of the vertebrate immune system evolve unusually rapidly. Swanson and Vacquier<br>
(1998) Science 281:710-712 use dN: ds ratios to demonstrate concerted evolution between the<br>
lysin and the egg receptor for lysin and discuss the role of such concerted evolution in<br>
forming new species (speciation). Messier and Stewart (1997) Nature 385:151-154, used<br>
KA/KS to demonstrate positive selection in primate lysozymes.<br>
The genetic changes associated with domestication have been most extensively<br>
investigated in maize (the preferred agricultural term for corn) (Dorweiler (1993) Science<br>
262:232-235). For maize, (Zea mays ssp. mays), a small number of single-gene changes<br>
apparently accounts for all the differences between our present domesticated maize plant and<br>
its wild ancestor, teosinte (Zea mays ssp paruiglumis) (Dorweiler, 1993). QTL (quantitative<br>
trait locus) analysis has demonstrated (Doebley (1990) PNAS USA 87:9888-9892) that no<br>
more than fifteen genes control traits of interest in maize and explain the profound difference<br>
in morphology between maize and teosinte (Wang (1999) Nature 398:236-239).<br>
Importantly, a similarly small number of genes may control traits of interest in other<br>
grass-derived crop plants, including rice, wheat, millet and sorghum (Paterson (1995) Science<br>
269:1714-1718). In fact, for most of these relevant genes in maize, the homologous gene may<br>
control similar traits in other grass-derived crop plants (Paterson, 1995). Thus, identification<br>
of these genes in one grass-derived crop plant would facilitate identification of homologous<br>
genes in all of the others.<br><br><br>
As can be seen from the papers cited above, analytical methods of molecular evolution<br>
to identify rapidly evolving genes (KA/Ks-type methods) can be applied to achieve many<br>
different purposes, most commonly to confirm the existence of Darwinian molecular-level<br>
positive selection, but also to assess the frequency of Darwinian molecular-level positive<br>
selection, to elucidate mechanisms by which new species are formed, or to establish single or<br>
multiple origin for specific gene polymorphisms. What is clear is from the papers cited above<br>
and others in the literature is that none of the authors applied KA/Ks-type methods to identify<br>
evolutionary changes in domesticated plants and animals brought about by artificial selective<br>
pressures. While Turcich et al. (1996) Sexual Plant Reproduction 9:65-74, describes the use<br>
of Ks analysis on plant genes, it is believed that no one has used KA/Ks type analysis as a<br>
systematic tool for identifying in domesticated plants and animals those genes that contain<br>
evolutionarily significant sequence changes that can be exploited in the development,<br>
maintenance or enhancement of desirable commercial or aesthetic traits.<br>
The identification in domesticated species of genes that have evolved to confer unique,<br>
enhanced or altered functions compared to homologous ancestral genes could be used to<br>
develop agents to modulate these functions. The identification of the underlying domesticated<br>
species genes and the specific nucleotide changes that have evolved, and the further<br>
characterization of the physical and biochemical changes in the proteins encoded by these<br>
evolved genes, could provide valuable information on the mechanisms underlying the desired<br>
trait. This valuable information could be applied to developing agents that further enhance<br>
the function of the target proteins. Alternatively, further engineering of the responsible genes<br>
could modify or augment the desired trait. Additionally, the identified genes may be found to<br>
play a role in controlling traits of interest in other domesticated plants. A similar process can -<br>
identify genes for traits of interest in domestic animals.<br>
All references cited herein are hereby incorporated by reference in their entirety.<br>
DISCLOSURE OF THE INVENTION<br>
The subject invention concerns methods of identifying polynucleotides that control<br>
commercially valuable traits in domesticated plants or animals. These polynucleotides that, in<br>
accordance with the methods of the subject invention, are found to control commercially<br>
valuable traits can be used to further enhance those traits. Polynucleotides identified to<br>
control commercially valuable traits such as drought-, disease-, or stress-resistance or yield,<br>
protein content, short day length flowering, oil content, ease of harvest; taste, and the like can<br>
be used to develop compositions and methods to further enhance the commercial value of<br><br><br>
domesticated plants. While it is desired to identity polynucleotides that control valuable<br>
traits, it is challenging to identity such polynucleotides among the tens of thousands of genes<br>
in plant and animal genomes. The invention comprises narrowing the search for such<br>
polynucleotides by comparing the corresponding polynucleotide sequences of domesticated<br>
and ancestor organisms to select those sequences containing nucleotide changes that are<br>
evolutionarily significant, which is typically indicated by a Ka/Ks ratio of 1.0 or greater. For<br>
example, the subset of ancestor-modern plant polynucleotide pairs with Ka/Ks ratios of 1.0<br>
should contain polynucleotides affected by neutral evolution, that is those for which the trait<br>
has not been under pressure, imposed by man or nature, to either be conserved or to change.<br>
Such polynucleotides can then be tested for those encoding traits such as such as drought-,<br>
disease-, or stress-resistance, because these functions have been dramatically supplemented<br>
by domestication, alleviating natural selection pressures on these polynucleotides. The subset<br>
of ancestor-modem plant polynucleotide pairs with Ka/Ks ratios greater than 1.0 should<br>
contain polynucleotides affected by selection. Such polynucleotides can then be tested for<br>
those encoding traits such as yield, protein content, short day length flowering, oil content,<br>
ease of harvest, taste, and the like, because these traits have been under intense,<br>
unidirectional, unremitting selective pressure by humans in the course of domestication of<br>
plants such as food crops.<br>
Thus, in one embodiment, the present invention provides methods for identifying<br>
polynucleotide and polypeptide sequences having evolutionarily significant changes, which<br>
are associated with commercial or aesthetic traits in domesticated organisms including plants<br>
and animals. The invention uses comparative genomics to identify specific gene changes<br>
which may be associated with, and thus responsible for, structural, biochemical or<br>
physiological conditions, such as commercially or aesthetically relevant traits, and using the<br>
information obtained from these polynucleotide or polypeptide sequences to develop<br>
domesticated organisms with enhanced traits of interest.<br>
In one preferred embodiment, a polynucleotide or polypeptide of a domesticated plant<br>
or animal has undergone artificial selection that resulted in an evolutionarily significant<br>
change present in the domesticated species that is not present in the wild ancestor. One<br>
example of this embodiment is that the polynucleotide or polypeptide may be associated with<br>
enhanced crop yield as compared to the ancestor. Other examples include short day length<br>
flowering (i.e., flowering only if me daily period of light is shorter than some critical length),<br>
protein content, oil content, ease of harvest, and taste. The present invention can thus be<br>
useful in gaming insight into the genes and/or molecular mechanisms that underlie functions<br><br><br>
or traits in domesticated organisms. This information can be useful in designing the<br>
polynucleotide so as to further enhance the function or trait. For example, a polynucleotide<br>
determined to be responsible for improved crop yield could be subjected to random or<br>
directed mutagenesis, followed by testing of the mutant genes to identify those which further<br>
enhance the trait.<br>
Accordingly, in one aspect, methods are provided for identifying a polynucleotide<br>
sequence encoding a polypeptide of a domesticated organism (e.g., a plant or animal),<br>
wherein the polypeptide may be associated with a commercially or aesthetically relevant trait<br>
that is unique, enhanced or altered in the domesticated organism as compared to the ancestor<br>
of the domesticated organism, comprising the steps of: a) comparing protein-coding<br>
nucleotide sequences of said domesticated organism to protein-coding nucleotide sequences<br>
of said wild ancestor, and b) selecting a polynucleotide sequence in the domesticated<br>
organism that contains a nucleotide change as compared to a corresponding sequence in the<br>
wild ancestor; wherein said change is evolutionarily significant.<br>
In another aspect of the invention, methods are provided for identifying an<br>
evolutionarily significant change in a protein-coding nucleotide sequence of a domesticated<br>
organism (e.g., a plant or animal), comprising the steps of: a) comparing protein-coding<br>
nucleotide sequences of the domesticated organism to corresponding sequences of a wild<br>
ancestor of the domesticated organism; and b) selecting a polynucleotide sequence in said<br>
domesticated organism that contains a nucleotide change as compared to the corresponding<br>
sequence of the wild ancestor, wherein the change is evolutionarily significant.<br>
In some embodiments, the nucleotide change identified by any of the methods<br>
described herein is a non-synonymous substitution. In some embodiments, the evolutionary<br>
significance of the nucleotide change is determined according to the non-synonymous<br>
substitution rate (KA) of the nucleotide sequence. In some embodiments, the evolutionarily<br>
significant changes are assessed by determining the KA/KS ratio between the domesticated<br>
organism polynucleotide and the corresponding ancestral polynucleotide. In some of these<br>
embodiments, preferably the ratio is at least about 0.75, or more preferably 1.0. With<br>
increasing preference, the ratio is at least about 1.0,1.25,1.50,2.00, or greater.<br>
In another aspect, the invention provides a method of identifying an agent which may<br>
modulate the relevant trait in the domesticated organism, said method comprising contacting<br>
at least one candidate agent with a cell, model system or transgenic plant or animal that<br>
expresses the polynucleotide sequence having the evolutionarily significant change, or a<br><br><br>
composition comprising the evolutionarily significant polypeptide wherein the agent is<br>
identified by its ability to modulate function or synthesis of the polypeptide.<br>
Also provided is a method for large scale sequence comparison between protein-<br>
coding nucleotide sequences of a domesticated organism and protein-coding sequences from a<br>
wild ancestor, said method comprising: a) aligning the domesticated organism sequences<br>
with corresponding sequences from the wild ancestor according to sequence homology; and<br>
b) identifying any nucleotide changes within the domesticated organism's sequences as<br>
compared to the homologous sequences from the wild ancestor organism.<br>
In another aspect, the subject invention provides a method for correlating an<br>
evolutionarily significant nucleotide change to a commercially or aesthetically relevant trait<br>
that is unique, enhanced or altered in a domesticated organism, comprising: a) identifying a<br>
nucleotide sequence having an evolutionarily significant change according to the methods<br>
descrilfed herein; and b) analyzing the functional effect of the presence or absence of the<br>
identified sequence in the domesticated organism or in a model system.<br>
The domesticated plants used in the subject methods can be maize, rice, tomatoes,<br>
potatoes or any domesticated plant for which the wild ancestor is extant and known. For<br>
example, the ancestor of maize is teosinte (Zea mays parviglumis); ancestors of wheat are<br>
Tritlcum monococcum, T. speltoides and Aegilops tauschii; and an ancestor of rice is O.<br>
rufipogon. The relevant trait can be any commercially or aesthetically relevant trait such as<br>
yield, short day length flowering, protein content, oil content, drought resistance, taste, ease of<br>
harvest or disease resistance. In a preferred embodiment, the domesticated plant is rice, and<br>
the relevant trait is yield.<br>
In another embodiment of the invention, methods for the identification of<br>
polynucleotides associated with stress-resistance in an ancestor organism are provided. In this<br>
embodiment, a polynucleotide in the domesticated organism has undergone neutral evolution<br>
relative to a polynucleotide in the ancestor which is or is suspected of being associated with<br>
stress-resistance, whereby mutations have accumulated in the domesticated organism's<br>
polynucleotide. The stress-resistance trait in the ancestor may be unique, enhanced or altered<br>
relative to the domesticated organism.<br>
The method for identifying the polynucleotide sequence comprises a) comparing<br>
polypeptide-coding nucleotide sequences of the domesticated organism to polypeptide coding<br>
nucleotide sequences of the wild ancestor; and b) selecting a polynucleotide sequence in the<br>
ancestor organism that contains at least one nucleotide change as compared to a<br>
corresponding sequence in the domesticated organism, wherein the change is evolutionarily<br><br><br>
neutral. The stress-resistance trait may be drought resistance, disease resistance, pest<br>
resistance, high salt level resistance or other stress-resistance traits of commercial interest.<br>
Also provided is a method for identifying an evolutionarily neutral change in a<br>
polypeptide-coding polynucleotide sequence of a wild ancestor of a domesticated organism<br>
comprising: a) comparing polypeptide-coding polynucleotide sequences of said wild ancestor<br>
to corresponding sequences of said domesticated organism; and b) selecting a polynucleotide<br>
sequence in the domesticated organism that contains a nucleotide change as compared to the<br>
corresponding sequence of the wild ancestor, wherein the change is evolutionarily neutral and<br>
the polynucleotide is associated with a stress-resistance trait in the wild ancestor.<br>
Neutral evolution is typically indicated by a KA/KS ratio of between about 0.75 and<br>
1.25, more preferably between about 0.9 and 1.1, and most preferably about 1.0. The KA/Ks<br>
comparison may be calculated as ancestor to domestic organism, or domestic to ancestor<br>
organism.<br>
In another aspect, the invention provides for a method of identifying an agent that may<br>
modulate a stress-resistance trait in an organism (ancestor or domesticated organism), wherein<br>
at least one candidate agent is contacted with the ancestor, domesticated organism or with a<br>
cell or transgenic organism that expresses the polynucleotide sequence associated with stress-<br>
resistance, wherein the agent is identified by its ability to modulate the function of the<br>
polypeptide encoded by the polynucleotide.<br>
Also provided is a method for large scale sequence comparison between polypeptide-<br>
coding nucleotide sequences of a wild ancestor and those of a domesticated organism,<br>
wherein the ancestor polypeptide confers or is suspected of conferring a stress-related trait<br>
that is unique, enhanced or altered in the wild ancestor as compared to the domesticated<br>
organism, comprising: a) aligning the ancestor and domesticated sequences according to<br>
sequence homology, and b) identifying any nucleotide changes in the domesticated organism<br>
sequence as compared to the ancestor homologous sequence, wherein said changes are<br>
evolutionarily neutral.<br>
.In another aspect, the subject invention provides a method for correlating an<br>
evolutionarily neutral nucleotide change to a commercially or aesthetically relevant trait that<br>
is unique, enhanced or altered in a domesticated organism, comprising: a) identifying a<br>
nucleotide sequence having an evolutionarily neutral change according to the methods<br>
described herein; and b) analyzing the functional effect of the presence or absence of the<br>
identified sequence in the domesticated organism or in a model system.<br><br><br>
BRIEF DESCRIPTION OF THE FIGURES<br>
Figure 1 shows a nucleotide alignment of O. sativa cv. Nipponbare and O. rufipogon<br>
(NSGC5953) for EG307. This alignment includes untranslated regions (UTR) on the 5' end<br>
and notes the start and stop codons for this gene.<br>
Figure 2 shows a protein alignment of O. sativa cv. Nipponbare and O. rufipogon<br>
(NSGC5953) for EG307. This alignment includes the complete coding (CDS) region.<br>
Figure 3 shows a nucleotide sequence of EG307 in Zea mays mays and Zea mays<br>
parviglumis (teosinte, strain Benz967) for coding region of the gene. Start and stop codons are<br>
identified.<br>
Figure 4 shows a protein alignment of Zea mays mays and Zea mays parviglumis<br>
EG307. This alignment includes the full-length deduced protein sequence.<br>
Figure 5 shows markers CD01387 and RZ672 mapped to five different genetic rice<br>
maps, indicating that the range of these markers is consistent among the five maps. EG307 is<br>
upstream of CD01387 (about 200kb) and a QTL for 1000 Grain Weight is associated with<br>
marker RZ672.<br>
Figure 6 shows the nucleotide alignment of O. sativa (strain Nipponbare) and O.<br>
rufigogon (strain 5498) for EG117, and indicates there are three nonsynomous changes.<br>
Figure 7 shows the protein alignment of O. sativa (strain Nipponbare) and O.<br>
rufigogon (strain 5498) for EG117. This alignment includes the partial CDS rgion with the<br>
stop codon. The three amino acid difference beween O. sativa and O. rufipogon are shown in<br>
bold.<br>
Figure 8 shows the protein alignment of O. sativa (strain Nipponbare) and Araidopsis<br>
PTR2-B (histidine transporting protein, NP_178313).<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
In one embodiment, the present invention utilizes comparative genomics to identify<br>
positively selected genes and specific gene changes which are associated with, and thus may<br>
contribute to or be responsible for, commercially or aesthetically relevant traits in<br>
domesticated organisms (e.g., plants and animals).<br>
In another embodiment, the invention identifies evolutionarily neutral genes and gene<br>
changes that are associated with stress-resistance in ancestors of domesticated organisms.<br>
The practice of the present invention employs, unless otherwise indicated,<br>
conventional techniques of molecular biology, genetics and molecular evolution, which are<br>
within the skill of the art. Such techniques are explained fully in the literature, such as:<br><br><br>
"Molecular Cloning: A Laboratory Manual", second edition (Sambrook et al., 1989);<br>
"Oligonucleotide Synthesis" (MJ. Gait, ed., 1984); "Current Protocols in Molecular Biology"<br>
(F.M. Ausubel et al., eds., 1987); "PCR: The Polymerase Chain Reaction", (Mullis et al,<br>
eds., 1994); "Molecular Evolution", (Li, 1997).<br>
/. Definitions<br>
As used herein, a "polynucleotide" refers to a polymeric form of nucleotides of any<br>
length, either ribonucleotides or deoxyribonucleotides, or analogs thereof. This term refers to<br>
the primary structure of the molecule, and thus includes double- and.single-'Stranded DNA, as<br>
well as double- and single-stranded RNA. It also includes modified polynucleotides such as<br>
methylated and/or capped polynucleotides, polynucleotides containing modified bases,<br>
backbone modifications, and the like. The terms "polynucleotide" and "nucleotide sequence"<br>
are used interchangeably.<br>
As used herein, a "gene" refers to a polynucleotide or portion of a polynucleotide<br>
comprising a sequence that encodes a protein. It is well understood in the art that a gene also<br>
comprises non-coding sequences, such as 5' and 3' flanking sequences (such as promoters,<br>
enhancers, repressors, and other regulatory sequences) as well as introns.<br>
The terms "polypeptide," "peptide," and "protein" are used interchangeably herein to<br>
refer to polymers of amino acids of any length. These terms also include proteins that are<br>
post-translationally modified through reactions that include glycosylation, acetylation and<br>
phosphorylation.<br>
The term "domesticated organism" refers to an individual living organism or<br>
population of same, a species, subspecies, variety, cultivar or strain, that has been subjected to<br>
artificial selection pressure and developed a commercially or aesthetically relevant trait. In<br>
sonde preferred embodiments, the domesticated organism is a plant selected from the group<br>
consisting of maize, wheat, rice, sorghum, tomato or potato, or any other domesticated plant<br>
of commercial interest, where an ancestor is known. A "plant" is any plant at any stage of<br>
development, particularly a seed plant.<br>
In other preferred embodiments, the domesticated organism is an animal selected from<br>
the group consisting of cattle, horses, pigs, cats and dogs. A domesticated organism and its<br>
ancestor may be related as different species, subspecies, varieties, cultivars or strains or any<br>
combination thereof.<br>
The term "wild ancestor" or "ancestor" means a forerunner or predecessor organism,<br>
species, subspecies, variety, cultivar or strain from which a domesticated organism, species,<br><br><br>
subspecies, variety, cultivar or strain has evolved. A domesticated organism can have one or<br>
more than one ancestor. Typically, domesticated plants can have one or a plurality of<br>
ancestors, while domesticated animals usually have only a single ancestor.<br>
The term "commercially or aesthetically relevant trait" is used herein to refer to traits<br>
that exist in domesticated organisms such as plants or animals whose analysis could provide<br>
information (e.g., physical or biochemical data) relevant to the development of improved<br>
organisms or of agents that can modulate the polypeptide responsible for the trait, or the<br>
respective polynucleotide. The commercially or aesthetically relevant trait can be unique,<br>
enhanced or altered relative to the ancestor. By "altered," it is meant that the relevant trait<br>
differs qualitatively or quantitatively from traits observed in the ancestor.<br>
The term "KA/Ks-type methods" means methods that evaluate differences, frequently- -<br>
(but not always) shown as a ratio, between the number of nonsynonymous substitutions and<br>
synonymous substitutions in homologous genes (including the more rigorous methods that<br>
determine non-synonymous and synonymous sites). These methods are designated using<br>
several systems of nomenclature, including but not limited to KA/Ks, dN/ds, DN/DS.<br>
The terms "evolutionarily significant change" and "adaptive evolutionary change"<br>
refer to one or more nucleotide or peptide sequence change(s) between two organisms,<br>
species, subspecies, varieties, cultivars and/or strains that may be attributed to either<br>
relaxation of selective pressure or positive selective pressure. One method for determining<br>
the presence of an evolutionarily significant change is to apply a KA/Ks-type analytical<br>
method, such as to measure a KA/KS ratio. Typically, a KA/KS ratio of 1.0 or greater is<br>
considered to be an evolutionarily significant change.<br>
Stricdy speaking, KA/KS ratios of exactly 1.0 are indicative of relaxation of selective<br>
pressure (neutral evolution), and KA/KS ratios greater than 1.0 are indicative of positive<br>
selection. However, it is commonly accepted that the ESTs in GenBank and other public<br>
databases often suffer from some degree of sequencing error, and even a few incorrect<br>
nucleotides can influence KA/KS ratios. For this reason, polynucleotides with KA/KS ratios as<br>
low as 0.75 can be carefully resequenced and re-evaluated for relaxation of selective pressure<br>
(neutral evolutionarily significant change), positive selection pressure (positive evolutionarily<br>
significant change), or negative selective pressure (evolutionarily conservative change).<br>
The term "positive evolutionarily significant change" means an evolutionarily<br>
significant change in a particular organism, species, subspecies, variety, cultivar or strain that<br>
results in an adaptive change that is positive as compared to other related organisms. An<br>
example of a positive evolutionarily significant change is a change that has resulted in<br><br><br>
enhanced yield in crop plants. As stated above, positive selection is indicated by a KA/KS<br>
ratio greater than 1.0. With increasing preference, the KA/KS value is greater than 1.25,1.5<br>
and 2.0.<br>
The term "neutral evolutionarily significant change" refers to a polynucleotide or<br>
polypeptide change that appears in a domesticated organism relative to its ancestral organism,<br>
and which has developed under neutral conditions. A neutral evolutionary change is<br>
evidenced by a KA/KS value of between about 0.75-1.25, preferably between about 0.9 and<br>
1.1, and most preferably equal to about 1.0. Also, in the case of neutral evolution, there is no<br>
"directionality" to be inferred. The gene is free to accumulate changes without constraint, so<br>
both the ancestral and domesticated versions are changing with respect to one another.<br>
The term "resistant" means that an organism exhibits an ability to avoid, or diminish<br>
the extent of, a disease condition and/or development of the disease, preferably when<br>
compared to non-resistant organisms.<br>
The term "susceptibility" means that an organism fails to avoid, or diminish the extent<br>
of, a disease condition and/or development of the disease condition, preferably when<br>
compared to an organism that is known to be resistant.<br>
It is understood that resistance and susceptibility vary from individual to individual,<br>
and that, for purposes of this invention, these terms also apply to a group of individuals within<br>
a species, and comparisons of resistance and susceptibility generally refer overall to inter-<br>
specific differences, although comparisons within species may be used. Taxonomic<br>
classification of wild relatives is fairly changeable. Thus, a species difference based on a<br>
taxonomic classification may change to an intra-specific difference if taxonomic<br>
classifications are changed.<br>
The term "stress-resistance" refers to the ability to withstand drought, disease, pests<br>
(including, but not limited to, insects, animal herbivores, and microbes), high salt levels, and<br>
other adverse stimuli, internal or external, that tend to disturb the plant's homeostasis, and<br>
may lead to disorder, disease, or death if uncorrected.<br>
The term "homologous" or "homologue" or "ortholog" is known and well understood<br>
in the art and refers to related sequences that share a common ancestor and is determined •<br>
based on degree of sequence identity. These terms describe the relationship between a gene<br>
found in one species, subspecies, variety, cultivar or strain and the corresponding or<br>
equivalent gene in another species, subspecies, variety, cultivar or strain. For purposes, of this<br>
invention homologous sequences are compared. "Homologous sequences" or "homologues"<br>
or "orthologs" are thought, believed, or known to be functionally related. A functional<br><br><br>
relationship may be indicated in any one of a number of ways, including, but not limited to,<br>
(a) degree of sequence identity; (b) same or similar biological function. Preferably, both (a)<br>
and (b) are indicated. The degree of sequence identity may vary, but is preferably at least<br>
50% (when using standard sequence alignment programs known in the art), more preferably<br>
at least 60%, more preferably at least about 75%, more preferably at least about 85%.<br>
Homology can be determined using software programs readily available in the art, such as<br>
those discussed in Current Protocols in Molecular Biology (F.M. Ausubel et al., eds., 1987)<br>
Supplement 30, section 7.718, Table 7.71. Preferred alignment programs are MacVector<br>
(Oxford Molecular Ltd, Oxford, U.K.) and ALIGN Plus (Scientific and Educational Software,<br>
Pennsylvania). Another preferred alignment program is Sequencher (Gene Codes, Ann<br>
Arbor, Michigan), using default parameters.<br>
The term "nucleotide change" refers to nucleotide substitution, deletion, and/or<br>
insertion, as is well understood in the art.<br>
"Housekeeping genes" is a term well understood in the art and means those genes<br>
associated with general cell function, including but not limited to growth, division, stasis,<br>
metabolism, and/or death. "Housekeeping" genes generally perform functions found in more<br>
than one cell type. In contrast, cell-specific genes generally perform functions in a particular<br>
cell type and/or class.<br>
The term "agent", as used herein, means a biological or chemical compound such as a<br>
simple or complex organic or inorganic molecule, a peptide, a protein or an oligonucleotide<br>
that modulates the function of a polynucleotide or polypeptide. A vast array of compounds<br>
can be synthesized, for example oligomers, such as oligopeptides and oligonucleotides, and<br>
synthetic organic and inorganic compounds based on various core structures, and these are<br>
also included in the term "agent". In addition, various natural sources can provide compounds<br>
for screening, such as plant or animal extracts, and the like. Compounds can be tested singly<br>
or in combination with one another.<br>
The term "to modulate function" of a polynucleotide or a polypeptide means that the<br>
function of the polynucleotide or polypeptide is altered when compared to not adding an<br>
agent. Modulation may occur on any level that affects function. A polynucleotide or<br>
polypeptide function may be direct or indirect, and measured directly or indirectly.<br>
A "function of a polynucleotide" includes, but is not limited to, replication;<br>
translation; expression pattern(s). A polynucleotide function also includes functions<br>
associated with a polypeptide encoded within the polynucleotide. For example, an agent<br>
which acts on a polynucleotide and affects protein expression, conformation, folding (or other<br><br><br>
physical characteristics), binding to other moieties (such as ligands), activity (or other<br>
functional characteristics), regulation and/or other aspects of protein structure or function is<br>
considered to have modulated polynucleotide function.<br>
A "function of a polypeptide" includes, but is not limited to, conformation, folding (or<br>
other physical characteristics), binding to other moieties (such as ligands), activity (or other<br>
functional characteristics), and/or other aspects of protein structure or functions. For<br>
example, an agent that acts on a polypeptide and affects its conformation, folding (or other<br>
physical characteristics), binding to other moieties (such as ligands), activity (or other<br>
functional characteristics), and/or other aspects of protein structure or functions is considered<br>
to have modulated polypeptide function. The ways that an effective agent can act to modulate .<br>
the function of a polypeptide include, but are not limited to 1) changing the conformation,<br>
folding or other physical characteristics; 2) changing the binding strength to its natural ligand<br>
or changing the specificity of binding to ligands; and 3) altering the activity of the<br>
polypeptide.<br>
The term "target site" means a location in a polypeptide which can be a single amino<br>
acid and/or is a part of, a structural and/or functional motif, e.g., a binding site, a dimerization<br>
domain, or a catalytic active site. Target sites may be useful for direct or indirect interaction<br>
with an agent, such as a therapeutic agent.<br>
The term "molecular difference" includes any structural and/or functional difference.<br>
Methods to detect such differences, as well as examples of such differences, are described<br>
herein.<br>
A "functional effect" is a term well known in the art, and means any effect which is<br>
exhibited on any level of activity, whether direct or indirect.<br>
The term "ease of harvest" refers to plant characteristics or features that facilitate<br>
manual or automated collection of structures or portions (e.g., fruit, leaves, roots) for<br>
consumption or other commercial processing.<br>
The term "yield" refers to the amount of plant or animal tissue or material that is<br>
available for use by humans for food, therapeutic, veterinary or other markets.<br>
The term "enhanced economic productivity" refers to the ability to modulate a<br>
commercially or aesthetically relevant trait so as to improve desired features. Increased yield<br>
and enhanced stress resistance are two examples of enhanced economic productivity.<br><br><br>
11. General Procedures Known in the Art<br>
For the purposes of this invention, the source of the polynucleotide from the<br>
domesticated plant or animal or its ancestor can be any suitable source, e.g., genomic<br>
sequences or cDNA sequences. Preferably, cDNA sequences are compared. Protein-coding<br>
sequences can be obtained from available private, public and/or commercial databases such as<br>
those described herein. These databases serve as repositories of the molecular sequence data<br>
generated by ongoing research efforts. Alternatively, protein-coding sequences may be<br>
obtained from, for example, sequencing of cDNA reverse transcribed from mKNA expressed<br>
in cells, or after PCR amplification, according to methods well known in the art.<br>
Alternatively, genomic sequences may be used for sequence comparison. Genomic sequences<br>
can be obtained from available public, private and/or commercial databases or from<br>
sequencing of genomic DNA libraries or from genomic DNA, after PCR.<br>
In some embodiments, the cDNA is prepared from mRNA obtained from a tissue at a<br>
determined developmental stage, or a tissue obtained after the organism has been subjected to<br>
certain environmental conditions. cDNA libraries used for the sequence comparison of the<br>
present invention can be constructed using conventional cDNA library construction<br>
techniques that are explained fully in the literature of the art. Total mRNAs are used as<br>
templates to reverse-transcribe cDNAs. Transcribed cDNAs are subcloned into appropriate<br>
vectors to establish a cDNA library. The established cDNA library can be maximized for<br>
full-length cDNA contents, although less than full-length cDNAs may be used. Furthermore,<br>
the sequence frequency can be normalized according to, for example, Bonaldo et al. (1996)<br>
Genome Research 6:791-806. cDNA clones randomly selected from the constructed cDNA<br>
library can be sequenced using standard automated sequencing techniques. Preferably, full-<br>
length cDNA clones are used for sequencing. Either the entire or a large portion of cDNA<br>
clones from a cDNA library may be sequenced, almough it is also possible to practice some<br>
embodiments of the invention by sequencing as little as a single cDNA, or several cDNA<br>
clones.<br>
In one preferred embodiment of the present invention, cDNA clones to be sequenced<br>
can be pre-selected according to their expression specificity. In order to select cDNAs<br>
corresponding to active genes that are specifically expressed, the cDNAs can be subject to<br>
subtraction hybridization using mRNAs obtained from other organs, tissues or cells of the<br>
same animal. Under certain hybridization conditions with appropriate stringency and<br>
conqentration, those cDNAs that hybridize with non-tissue specific mRNAs and thus likely<br><br><br>
represent "housekeeping" genes will be excluded from the cDNA pool. Accordingly,<br>
remaining cDNAs to be sequenced are more likely to be associated with tissue-specific<br>
functions. For the purpose of subtraction hybridization, non-tissue-specific mRNAs can be<br>
obtained from one organ, or preferably from a combination of different organs and cells. The<br>
amount of non-tissue-specific mRNAs are maximized to saturate the tissue-specific cDNAs.<br>
Alternatively, information from online databases can be used to select or give priority<br>
to cDNAs that are more likely to be associated with specific functions. For example, the<br>
ancestral cDNA candidates for sequencing can be selected by PCR using primers designed<br>
from candidate domesticated organism cDNA sequences. Candidate domesticated organism<br>
cDNA sequences are, for example, those that are only found in a specific tissue, such as<br>
skeletal muscle, or that correspond to genes likely to be important in the specific function.<br>
Such tissue-specific cDN A sequences may be obtained by searching online sequence<br>
databases in which information with respect to the expression profile and/or biological<br>
activity for cDNA sequences may be specified.<br>
Sequences of ancestral homologue(s) to a known domesticated organism's gene may<br>
be obtained using methods standard in the art, such as PCR methods (using, for example,<br>
GeneAmp PCR System 9700 thermocyclers (Applied Biosystems, Inc.)). For example,<br>
ancestral cDNA candidates for sequencing can be selected by PCR using primers designed<br>
from candidate domesticated organism cDNA sequences. For PCR, primers may be made<br>
from the domesticated organism's sequences using standard methods in the art, including<br>
publicly available primer design programs such as PRIMER® (Whitehead Institute). The<br>
ancestral sequence amplified may then be sequenced using standard methods and equipment<br>
in the art, such as automated sequencers (Applied Biosystems, Inc.). Likewise, ancestors<br>
gene mimics can be used to obtain corresponding genes in domesticated organisms.<br>
///. Identification of Positively Selected Polynucleotides in<br>
Domesticated Organisms<br>
In a preferred embodiment, the methods described herein can be applied to identify the<br>
genes that control traits of interest in agriculturally important domesticated plants. Humans<br>
have bred domesticated plants for several thousand years without knowledge of the genes that<br>
control these traits. Knowledge of the specific genetic mechanisms involved would allow<br>
much more rapid and direct intervention at the molecular level to create plants with desirable -<br>
or enhanced traits.<br>
Humans, through artificial selection, have provided intense selection pressures on crop<br>
plants. This pressure is reflected in evolutionarily significant changes between homologous<br><br><br>
genes of domesticated organisms and their wild ancestors. It has been found that only a few<br>
genes, e.g., 10-15 per species, control traits of commercial interest in domesticated crop<br>
plants. These few genes have been exceedingly difficult to identify through standard methods<br>
of plant molecular biology. The KA/KS and related analyses described herein can identify the<br>
genes controlling traits of interest.<br>
For any crop plant of interest, cDNA libraries can be constructed from the<br>
domesticated species or subspecies and its wild ancestor. As is described in USSN<br>
09/240,915, filed January 29,1999, the cDNA libraries of each are "BLASTed" against each<br>
other to identify homologous polynucleotides. Alternatively, the skilled artisan can access<br>
commercially and/or publicly available genomic or cDNA databases rather than constructing<br>
cDNA libraries.<br>
Next, a KA/KS or related analysis is conducted to identify selected genes that have<br>
rapidly evolved under selective pressure. These genes are then evaluated using standard<br>
molecular and transgenic plant methods to determine if they play a role in the traits of<br>
commercial or aesthetic interest. The genes of interest are then manipulated by, e.g., random<br>
or site-directed mutagenesis, to develop new, improved varieties, subspecies, strains or<br>
cultivars.<br>
The general method of the invention is as follows. Briefly, nucleotide sequences are<br>
obtained from a domesticated organism and a wild ancestor. The domesticated organism's<br>
and ancestor's nucleotide sequences are compared to one another to identify sequences that<br>
are homologous. The homologous sequences are analyzed to identify those that have nucleic<br>
acid sequence differences between the domesticated organism and ancestor. Then molecular<br>
evolution analysis is conducted to evaluate quantitatively and qualitatively the evolutionary<br>
significance of the differences. For genes that have been positively selected, outgroup<br>
analysis can be done to identify those genes that have been positively selected in the<br>
domesticated organism (or in the ancestor). Next, the sequence is characterized in terms of<br>
molecular/genetic identity and biological function. Finally, the information can be used to<br>
identify agents that can modulate the biological function of the polypeptide encoded by the<br>
gene.<br>
The general methods of the invention entail comparing protein-coding nucleotide<br>
sequences of ancestral and domesticated organisms. Bioinformatics is applied to the<br>
comparison and sequences are selected that contain a nucleotide change or changes that is/are<br>
evolutionarily significant change(s). The invention enables the identification of genes that<br>
have evolved to confer some evolutionary advantage and the identification of the specific<br><br><br>
evolved changes. In a preferred embodiment, the domesticated organism is Oryza sativa and<br>
the wild ancestor is Oryza rufipogon. In the case of the present invention, protein-coding<br>
nucleotide sequences were obtained from O. rufipogon clones by standard sequencing<br>
techniques.<br>
Protein-coding sequences of a domesticated organism and its ancestor are compared to<br>
identify homologous sequences. Any appropriate mechanism for completing this comparison<br>
is contemplated by this invention. Alignment may be performed manually or by software<br>
(examples of suitable alignment programs are known in the art). Preferably, protein-coding<br>
sequences from an ancestor are compared to the domesticated species sequences via database<br>
searches, e.g., BLAST searches. The high scoring "hits," i.e., sequences that show a<br>
significant similarity after BLAST analysis, will be retrieved and analyzed. Sequences<br>
showing a significant similarity can be those having at least about 60%, at least about 75%, at<br>
least about 80%, at least about 85%, or at least about 90% sequence identity. Preferably,<br>
sequences showing greater than about 80% identity are further analyzed. The homologous<br>
sequences identified via database searching can be aligned in their entirety using sequence<br>
alignment methods and programs that are known and available in the art, such as the<br>
commonly used simple alignment program CLUSTAL V by Higgins et al. (1992) CABIOS<br>
8:189-191.<br>
The present invention provides a method for identifying a polynucleotide sequence<br>
encoding a polypeptide of a domesticated organism, wherein said polypeptide is or is<br>
suspected of being associated with improved yield in said domesticated organism as<br>
compared to a wild ancestor of said domesticated organism, comprising the steps of a)<br>
comparing polypeptide-coding nucleotide sequences of said domesticated organism to<br>
polypeptide-coding nucleotide sequences of said wild ancestor; and b) selecting a<br>
polynucleotide sequence in the domesticated organism that contains a nucleotide change as<br>
compared to a corresponding sequence in the wild ancestor, wherein said change is<br>
evolutionarily significant, whereby the domesticated organism's polynucleotide sequence is<br>
identified. In a preferred embodiment, the polypeptide that is associated with improved yield<br>
is an EG307 polypeptide.<br>
In the present case, for example, nucleotide sequences obtained from 0. rufipogon<br>
were used as query sequences in a search of O. sativa ESTs in GenBank to identify<br>
homologous sequences. It should be noted, that a complete protein-coding nucleotide<br>
sequence is not required. Indeed, partial cDNA sequences may be compared. Once<br>
sequences of interest are identified by the methods described below, further cloning and/or -<br><br><br>
bioinformatics methods can be used to obtain the entire coding sequence for the gene or<br>
protein of interest.<br>
Alternatively, the sequencing and homology comparison of protein-coding sequences<br>
between the domesticated organism and its ancestor may be performed simultaneously by<br>
using the newly developed sequencing chip technology. See, for example, Rava et al. US<br>
Patent 5,545,531.<br>
The aligned protein-coding sequences of domesticated organism and ancestor are<br>
analyzed to identify nucleotide sequence differences at particular sites. Again, any suitable<br>
method for achieving this analysis is contemplated by this invention. If there are no<br>
nucleotide sequence differences, the ancestor protein coding sequence is not usually further<br>
analyzed. The detected sequence changes are generally, and preferably, initially checked for<br>
accuracy. Preferably, the initial checking comprises performing one or more of the following<br>
steps, any and all of which are known in the art: (a) finding the points where there are<br>
changes between the ancestral and domesticated organism sequences; (b) checking the<br>
sequence fluorogram (chromatogram) to determine if the bases that appear unique to the<br>
ancestor or domesticated organism correspond to strong, clear signals specific for the called<br>
base; (c) checking the domesticated organism hits to see if there is more than one<br>
domesticated organism sequence that corresponds to a sequence change. Multiple<br>
domesticated organism sequence entries for the same gene that have the same nucleotide at a<br>
position where there is a different nucleotide in an ancestor sequence provides independent<br>
support that the domesticated sequence is accurate, and that the change is significant. Such<br>
changes are examined using database information and the genetic code to determine whether<br>
these nucleotide sequence changes result in a change in the amino acid sequence of the<br>
encoded protein. As the definition of "nucleotide change" makes clear, the present invention<br>
encompasses at least one nucleotide change, either a substitution, a deletion or an insertion, in<br>
a protein-coding polynucleotide sequence of a domesticated organism as compared to a<br>
corresponding sequence from the ancestor. Preferably, the change is a nucleotide substitution.<br>
More preferably, more than one substitution is present in the identified sequence and is<br>
subjected to molecular evolution analysis.<br>
Any of several different molecular evolution analyses or KA/KS-type methods can be<br>
employed to evaluate quantitatively and qualitatively the evolutionary significance of the<br>
identified nucleotide changes between domesticated species gene sequences and those of<br>
corresponding ancestors. Kreitman and Akashi (1995) Annu. Rev. Ecol. Syst. 26:403-422; Li,<br>
Molecular Evolution,-Sinauer Associates, Sunderland, MA, 1997. For example, positive<br><br>
selection on proteins (i.e., molecular-level adaptive evolution) can be detected in protein-<br>
coding genes by pairwise comparisons of the ratios of nonsynonymous nucleotide<br>
substitutions per nonsynonymous site (KA) to synonymous substitutions per synonymous site<br>
(Ks) (Li et al., 1985; Li, 1993). Any comparison of KA and KS may be used, although it is<br>
particularly convenient and most effective to compare these two variables as a ratio.<br>
Sequences are identified by exhibiting a statistically significant difference between KA and KS<br>
using standard statistical methods.<br>
In the case of the present invention, homologous sequences from O. rufipogon and O.<br>
sativa were identified. Comparison of the sequences of one O. rufipogon clone, PBI0307H9,<br>
SEQ ID NO:31, and 0. sativa in GenBank revealed a high KA/KS ratio. Further cloning and<br>
PCR of several different strains of O. sativa were completed in order to obtain the entire gene,<br>
named EG307, so that the entire gene sequence could be subjected to KA/KS analysis. These<br>
procedures are detailed in Example 10. The complete sequence of EG307 in O. rufipogon,<br>
SEQ ID NO:28, and O. sativa cv. Nipponbare 1, SEQ ED NO:25, are shown in Figure 1. The<br>
corresponding protein sequences, SEQ ID NO:30, and SEQ ID NO:27, are shown in Figure 2.<br>
A summary of the KA/KS ratios is shown in Table 1 of Example 11. Some strains were more<br>
similar to O. rufipogon due to cross-breeding between O. rufipogon and the domestic strain.<br>
High KA/KS ratios for some strains indicates an evolutionarily significant change.<br>
Preferably, the KA/KS analysis computer program by Li et al. is used to carry out the<br>
present invention, although other analysis programs that can detect positively selected genes "<br>
between species can also be used. Li et al. (1985) Mol. Biol Evol. 2:150-174; Li (1993); see<br>
also J. Mol. Evol. 36:96-99; Messier and Stewart (1997) Nature 385:151-154; Nei (1987)<br>
Molecular Evolutionary Genetics (New York, Columbia University Press). The KA/KS<br>
method, which comprises a comparison of the rate of non-synonymous substitutions per non-<br>
synonymous site with the rate of synonymous substitutions per synonymous site between<br>
homologous protein-coding region of genes in terms of a ratio, is used to identify sequence<br>
substitutions that may be driven by adaptive selections or by neutral selections during<br>
evolution. A synonymous ("silent") substitution is one that, owing to the degeneracy of the<br>
genetic code, makes no change to the amino acid sequence encoded; a non-synonymous<br>
substitution results in an amino acid replacement. The extent of each type of change can be<br>
estimated as KA and Ks, respectively, the numbers of non-synonymous substitutions per non-<br>
synonymous site and synonymous substitutions per synonymous site. Calculations of KA/KS<br>
may be performed manually or by using software. An example of a suitable program is<br>
MEGA (Molecular Genetics Institute, Pennsylvania State University).<br><br><br>
For the purpose of estimating KA and KS, either complete or partial protein-coding<br>
sequences are used to calculate total numbers of synonymous and non-synonymous<br>
substitutions, as well as non-synonymous and synonymous sites. The length of the<br>
polynucleotide sequence analyzed can be any appropriate length. Preferably, the entire<br>
coding sequence is compared, in order to determine any and all significant changes. Publicly<br>
available computer programs, such as Li93 (Li (1993) J. Mol. Evol. 36:96-99) or IN A, can be<br>
used to calculate the KA and KS values for all pairwise comparisons. This analysis can be<br>
further adapted to examine sequences in a "sliding window" fashion such that small numbers<br>
of important changes are not masked by the whole sequence. "Sliding window" refers to<br>
examination of consecutive, overlapping subsections of the gene (the subsections can be of<br>
any length).<br>
Sliding window KA/KS analysis of, for example, identified gene EG307 showed that<br>
there are a number of nonsynonymous changes on the 5'-end of EG307 in many of the O.<br>
saliva strains when compared to O. rufipogon. The 3'-end of the gene had a low ratio in all of<br>
the strains. These procedures and results are detailed in Example 11 and Tables 2-7.<br>
The comparison of non-synonymous and synonymous substitution rates is represented<br>
by the KA/KS ratio. KA/KS has been shown to be a reflection of the degree to which adaptive<br>
evolution has been at work in the sequence under study. Full length or partial segments of a<br>
coding sequence can be used for the KA/KS analysis. The higher the KA/KS ratio, the more<br>
likely that a sequence has undergone adaptive evolution and the non-synonymous<br>
substitutions are evolutionarily significant. See, for example, Messier and Stewart (1997).<br>
Preferably, the KA/KS ratio is at least about 0.75, more preferably at least about 1.0, more<br>
preferably at least about 1.25, more preferably at least about 1.50, or more preferably at least<br>
about 2.00. Preferably, statistical analysis is performed on all elevated KA/KS ratios,<br>
including, but not limited to, standard methods such as Student's t-test and likelihood ratio<br>
tests described by Yang (1998) Mol. Biol Evol. 37:441-456.<br>
For a pairwise comparison of homologous sequences, KA/KS ratios significantly<br>
greater than unity strongly suggest that positive selection has fixed greater numbers of amino<br>
acid replacements than can be expected as a result of chance alone, and is in contrast to the<br>
commonly observed pattern in which the ratio is less than one. Nei (1987); Hughes and llei<br>
(1988) Nature 335:167-170; Messier and Stewart (1994) Current Biol. 4:911-913; Kreitman<br>
and Akashi (1995) Ann. Rev. Ecol Syst. 26:403-422; Messier and Stewart (1997). Ratios less-<br>
than one generally signify the role of negative, or purifying selection: there is strong pressure<br><br><br>
on the primacy structure of functional, effective proteins to remain unchanged. Ratios of<br>
about 1 indicate evolution under neutral conditions.<br>
All methods for calculating KA/KS ratios are based on a pairwise comparison of the<br>
number of nonsynonymous substitutions per nonsynonymous site to the number of<br>
synonymous substitutions per synonymous site for the protein-coding regions of homologous<br>
genes from the ancestral and domesticated organisms. Each method implements different<br>
corrections for estimating "multiple hits" (i.e., more than one nucleotide substitution at the<br>
same site). Each method also uses different models for how DNA sequences change over<br>
evolutionary time. Thus, preferably, a combination of results from different algorithms is<br>
used to increase the level of sensitivity for detection of positively-selected genes and<br>
confidence in the result.<br>
Preferably, KA/KS ratios should be calculated for orthologous gene pairs, as opposed<br>
to paralogous gene pairs (i.e., a gene which results from speciation, as opposed to a gene that<br>
is the result of gene duplication) Messier and Stewart (1997). This distinction may be made<br>
by performing additional comparisons with other ancestors, which allows for phylogenetic<br>
tree-building. Orthologous genes when used in tree-building will yield the known "species<br>
tree", Le., will produce a tree that recovers the known biological tree. In contrast, paralogous<br>
genes will yield trees which will violate the known biological tree.<br>
It is understood that the methods described herein could lead to the identification of<br>
ancestral or domesticated organism polynucleotide sequences that are functionally related to<br>
the protein-coding sequences. Such sequences may include, but are not limited to, non-<br>
coding sequences or coding sequences that do not encode proteins. These related sequences<br>
can be, for example, physically adjacent to the protein-coding sequences in the genome, such<br>
as introns or 5'- and 3'- flanking sequences (including control elements such as promoters and<br>
enhancers). These related sequences may be obtained via searching available public, private<br>
and/or commercial genome databases or, alternatively, by screening and sequencing the<br>
organism's genomic library with a protein-coding sequence as probe. Methods and techniques<br>
for obtaining non-coding sequences using related coding sequence are well known to one<br>
skilled in the art.<br>
The evolutionarily significant nucleotide changes, which are detected by molecular<br>
evolution analysis such as the KA/KS analysis, can be further assessed for their unique<br>
occurrence in the domesticated organism or the extent to which these changes are unique in<br>
the domesticated organism. For example, the identified changes in the domesticated gene can<br>
be tested for presence/absence in other sequences of related species, subspecies or other<br><br><br>
organisms having a common ancestor with the domesticated organism. This comparison<br>
("outgroup analysis") permits the determination of whether the positively selected gene is<br>
positively selected for in the domesticated organism at issue (as opposed to the ancestor).<br>
For example, the identified changes in the EG307 gene were identified to various<br>
degrees in a number of O. saliva strains. See Tables 2-7. Additionally, a counterpart to<br>
EG307 was identified in maize, Zea mays mays, its wild ancestor, teosinte, Zea mays<br>
parviglumis, and also wild relatives of maize, Z. diploperennis and Z. luxurians. See Example<br>
13 and Table 9. While EG307 in rice and maize was somewhat different at the nucleotide<br>
level, the protein sequences were more similar. Observing that rice and corn were<br>
independently domesticated from their wild ancestors, a consistent pattern emerges: the<br>
majority of the amino acid replacements in the modem crop (whether maize or rice), as<br>
compared to the ancestral plant (teosinte or ancestral rice) result in increased charge/polarity,<br>
increased solubility, and decreased hydrophobicity. This pattern is most unlikely to have<br>
occurred by chance in these two independent domestication events. This suggests that these<br>
replacements were a similar response to human imposed domestication. This is powerful<br>
evidence that EG307 has been selected as a result of human domestication of these two<br>
cereals.<br>
The sequences with at least one evolutionarily significant change between a<br>
domesticated organism and its ancestor can be used as primers for PCR analysis of other<br>
ancestor protein-coding sequences, and resulting polynucleotides are sequenced to see<br>
whether the same change is present in other ancestors. These comparisons allow further<br>
discrimination as to whether the adaptive evolutionary changes are unique to the domesticated<br>
lineage as compared to other ancestors or whether the adaptive change is unique to the<br>
ancestor as compared to the domesticated species and other ancestors. A nucleotide change<br>
that is detected in the domesticated organism but not other ancestors more likely represents an<br>
adaptive evolutionary change in the domesticated organism. Alternatively, a nucleotide<br>
change that is detected in an ancestor that is not detected in the domesticated organism or<br>
other ancestors likely represents an ancestor adaptive evolutionary change. Other ancestors<br>
used for comparison can be selected based on their phylogenetic relationships with the<br>
domesticated organism. Statistical significance of such comparisons may be determined<br>
using established available programs, e.g., t-test as.used by Messier and Stewart (1997)<br>
Nature 385:151 -154. Those genes showing statistically high KA/KS ratios are very likely to<br>
have undergone adaptive evolution.<br><br><br>
Sequences with significant changes can be used as probes in genomes from different<br>
domesticated populations to see whether the sequence changes are shared by more than one<br>
domesticated population. Gene sequences from different domesticated populations can be<br>
obtained from databases or, alternatively, from direct sequencing of PCR-amplified DNA<br>
from a number of unrelated, diverse domesticated populations. The presence of the identified<br>
changes in different domesticated populations would further indicate the evolutionary<br>
significance of the changes.<br>
Sequences with significant changes between species can be further characterized in<br>
terms of their molecular/genetic identities and biological functions, using methods and<br>
techniques known to those of ordinary skill in the art. For example, the sequences can be<br>
located genetically and physically within the organism's genome using publicly available bio-<br>
informatics programs. The newly identified significant changes within the nucleotide<br>
sequence may suggest a potential role of the gene in the organism's evolution and a potential<br>
association with unique, enhanced or altered functional capabilities.<br>
Using the techniques of the present invention, a heretofore unknown evolutionarily<br>
significant gene in rice, termed EG307, has been discovered as detailed in EXAMPLE 10.<br>
KA/KS analysis, performed as described in EXAMPLE 11 between O. rufipogon and certain<br>
O. sativa strains indicated an evolutionarily significant change as shown in Table. 1. The gene<br>
has been positively selected. Using several different rice maps, as described in EXAMPLE<br>
12, it was found that EG307 was within about 10 cM of marker RZ672, a marker associated<br>
with a QTL for 1000 grain weight residing on chromosome 3. (1000-grain weight is the<br>
weight (mass) of three different samples of 1000 randomly chosen fully filled grains of rice.)<br>
This is a sensitive measure of yield, which takes into account the individual variation in<br>
weight that occurs among rice grains. Thus, there only is about a 10% chance that the RZ672<br>
marker will be separated from EG307 to crossing over in a single generation, strongly<br>
suggesting that EG307 plays an important role in controlling increased yield.<br>
Also using the techniques of the present invention, a heretofore unknown<br>
evolutionarily significant gene in rice, termed EG3117, has been discovered as detailed in<br>
EXAMPLE 14. KA/KS analysis, performed as described in EXAMPLE 14 between O.<br>
rufipogon and certain O. sativa strains indicated an evolutionarily significant change as shown<br>
in Table 10.. The gene has been positively selected. Using several different rice maps, as<br>
described in EXAMPLES 13 and 14, it was found that EG1117 lies on the same BAC as<br>
marker RZ672, a marker associated with a QTL for 1000 grain weight residing on<br>
chromosome 3. EG1117 lies about 2-3 cM from EG307..<br><br><br>
From the combination of the evolutionarily significant KA/KS value and mapping data,<br>
one of skill in the art can reasonably conclude that EG307 and EG1117 are yield-related<br>
genes. EG307's and EG1117's yield-increasing function could be easily confirmed by<br>
making and growing a mutant or transgenic plant. Alternative methods include association<br>
analysis and pedigree analysis using the EG307 and EG117 sequence derived from rice,<br>
EG307 and EG117 genes from maize and its wild ancestor were obtained as detailed in<br>
EXAMPLE 13.<br>
The putative gene with the identified sequences may be further characterized by, for<br>
example, homologue searching. Shared homology of the putative gene with a known gene<br>
may indicate a similar biological role or function. Another exemplary method of<br>
characterizing a putative gene sequence is on the basis of known sequence motifs. Certain<br>
sequence patterns are known to code for regions of proteins having specific biological<br>
characteristics such as signal sequences, DNA binding domains, or transmembrane domains.<br>
The identified sequences with significant changes can also be further evaluated by<br>
looking at where the gene is expressed in terms of tissue- or cell type-specificity. For<br>
example, the identified coding sequences can be used as probes to perform in situ mRNA<br>
hybridization that will reveal the expression patterns of the sequences. Genes that are<br>
expressed in certain tissues may be better candidates as being associated with important<br>
functions associated with that tissue, for example developing endosperm tissue. The timing of<br>
the gene expression during each stage of development of a species member can also be<br>
determined.<br>
As another exemplary method of sequence characterization, the functional roles of the<br>
identified nucleotide sequences with significant changes can be assessed by conducting<br>
functional assays for different alleles of an identified gene in the transfected domesticated<br>
organism, e.g., in the transgenic plant or animal. Current examples of plant functional assays<br>
include the use of microarrays, see Seki, et al., Monitoring the Exapression Pattern of 1300<br>
Arabidopsis Genes Under Drought and Cold Stresses Using a Full-Length cDNA Microarray.<br>
Plant Cell 13:61-72 (2001), and metabolite profiling, see Roessner, et al, Metabolic Profiling<br>
Allows Comprhensive Phenotyping of Geneticaly or Environmentally Modified Plant<br>
Systems. Plant Cell 13:11-29 (2001).<br>
As another exemplary method of sequence characterization, the use of computer<br>
programs may allow modeling and visualizing the three-dimensional structure of the<br>
homologous proteins from domesticated organism and ancestor. Specific, exact knowledge of<br>
which amino acids have been replaced in the ancestor protein(s) allows detection of structural<br><br><br>
changes that may be. associated with functional differences. Thus, use of modeling techniques<br>
is closely associated with identification of functional roles discussed in the previous<br>
paragraph. The use of individual or combinations of these techniques constitutes part of the<br>
present invention.<br>
A domesticated organism's gene identified by the subject method can be used to<br>
identify homologous genes in other species that share a common ancestor. For example,<br>
maize, rice, wheat, millet, sorghum and other cereals share a common ancestor, and genes<br>
identified in rice can lead directly to homologous genes in these other grasses. Likewise,<br>
tomatoes and potatoes share a common ancestor, and genes identified in tomatoes by the<br>
subject method are expected to have homologues in potatoes, and vice versa.<br>
The present invention also provides a method of detecting a yield-increasing gene in a<br>
plant cell comprising: a) contacting the EG307 gene or a portion thereof greater than 12<br>
nucleotides, preferably greater than 30 nucleotides in length with a preparation of genomic<br>
DNA from the plant cell under hybridization conditions providing detection of nucleic acid<br>
molecule sequences having about 50% or greater sequence identity to the a nucleic acid<br>
molecule selected from the group consisting of SEQ ID NO:l, SEQ ID NO:91, SEQ ID.<br>
NO:2, SEQ ID NO:4, SEQ ID NO:5, SEQ ID NO:7, SEQ LD NO: 10, SEQ ID NO:l 1, SEQ ID<br>
NO:12, SEQ ID NO:14, SEQ ID NO:15, SEQ IDNO:17, SEQ ID NO:18, SEQ ID NO.20,<br>
SEQ ID NO:21, SEQ ID. NO:23, SEQ ID NO:24, SEQ ID NO:25, SEQ ID NO:27, SEQ ID<br>
NO:28, SEQ ID NO:29, SEQ ID NO:30, SEQ ID NO:31, SEQ ID NO:33, SEQ ID NO:34,<br>
SEQ ID. NO-.35, SEQ ID NO:37, SEQ ID NO:38, SEQ ID NO:40, SEQ IDNO:41, SEQ ID<br>
N0.42, SEQ ID NO:44, SEQ ED NO:45, SEQ ID NO:46, SEQ ID NO:47, SEQ ID NO:49,<br>
SEQ ID. NO:50, SEQ ID NO:51, SEQ ID NO:53, SEQ ID NO:54, SEQ ID NO:55, SEQ ID<br>
NO:57, SEQ ID NO:58, SEQ ID NO:60, SEQ ID NO:62, SEQ ID NO:63, SEQ ID NO:64,<br>
SEQ ID NO:66, SEQ ID NO:67, SEQ ID NO:69, SEQ ID. NO:70, SEQ ID NO:71, SEQ ID<br>
NO:73, SEQ ID NO:74, SEQ ID NO:75, SEQ ID NO:77, SEQ ID NO:59, SEQ ID NO:78,<br>
SEQ ID NO:80, SEQ ID NO:81, SEQ ID NO:82, SEQ ID NO:84 and SEQ ID NO:85; and b)<br>
detecting hybridization, whereby a yield-increasing gene may be identified.<br>
The present invention also provides a method of isolating a yield-related gene from a<br>
recombinant plant cell library, comprising a) providing a preparation of plant cell DNA or a<br>
recombinant plant cell library; b) contacting the preparation or plant cell library with a<br>
detectably-labelled EG307 conserved oligonucleotide under hybridization conditions<br>
providing detection of genes having 50% or greater sequence identity; and c) isolating a yield-<br>
related gene by its association with the detectable label;<br><br><br>
The present invention also provides a method of isolating a yield-related gene from<br>
plant cell DNA comprising a) providing a sample of plant cell DNA; b) providing a pair of<br>
oligonucleotides having sequence homology to a conserved region of an EG307 gene; c)<br>
combining the pair of oligonucleotides with the plant cell DNA sample under conditions<br>
suitable for polymerase chain reaction-mediated DNA amplification; and d) isolating the<br>
amplified yield-related gene or fragment thereof.<br>
The sequences identified by the methods described herein can be used to identify<br>
agents that are useful in modulating domesticated organism-unique, enhanced or altered<br>
functional capabilities and/or correcting defects in these capabilities using these sequences.<br>
These methods employ, for example, screening techniques known in the art, such as in vitro<br>
systems, cell-based expression systems and transgenic animals and plants. The approach<br>
provided by the present invention not only identifies rapidly evolved genes, but indicates<br>
modulations that can be made to the protein that may not be too toxic because they exist in<br>
another species.<br>
The present invention also provides a method of producing an EG307 polypeptide<br>
comprising: a) providing a cell transfected with a polynucleotide encoding an EG307<br>
polypeptide positioned for expression in the cell; b) culturing the transfected cell under<br>
conditions for expressing the polynucleotide; and c) isolating the EG307 polypeptide.<br>
The present invention also provides a method of detecting a yield-increasing gene in a<br>
plant cell comprising: a) contacting the EG307 OR EG1117 gene or a portion thereof greater<br>
than 12 nucleotides, preferably greater than 30 nucleotides in length with a preparation of<br>
genomic DNA frota the plant cell under hybridization conditions providing detection of<br>
nucleic acid molecule sequences having about 50% or greater sequence identity to the a<br>
nucleic acid molecule selected from the group consisting of SEQ ID NO:92, SEQ ID NO:93,<br>
SEQ ID NO:94, SEQ ID NO:96, SEQ ID NO:97, SEQ ID NO:98, SEQ ID NO: 100, SEQ ID<br>
NO:101, SEQ ID NO:102, SEQ ID NO:103, SEQ ID NO:104, SEQ ID N6:104, SEQ ID<br>
NO:106, SEQ IDNO:107, SEQ IDNO:109, SEQ IDNO:110, SEQ IDNO:112, SEQ ID<br>
NO:113, SEQ ID NO:114, SEQ ID NO:116, SEQ ID NO:l 17, SEQ ID NO:l 19, SEQ ID<br>
NO:120, SEQ ID NO:121, SEQ ID NO:122, SEQ ID NO:123, SEQ ID NO:124, SEQ ID<br>
NO:125, SEQ IDNO.-127, SEQ IDNO:128, SEQ IDNO.-129, SEQ IDNO:130, SEQ ID<br>
NO:131, SEQ IDNO:133, SEQ EDNO:135, SEQ ED NO:136, SEQ ID NO:137, SEQ ID<br>
NO:138, SEQ ID NO:140, SEQ ID NO:141, SEQ ID NO:142, SEQ ID NO:144, SEQ ID<br>
NO:145, SEQ ID N0.146, SEQ ID NO:147, SEQ ID NO:149, SEQ ID NO:150, SEQ ID<br>
NO:151, SEQ ID NO:152, SEQ ID NO:154, SEQ ID NO:155, SEQ ID NO:157, SEQ ID<br><br><br>
NO:158, SEQ ID NO:160, SEQ IDNO:161, SEQ IDNO:162, SEQ IDNO:163, SEQ ID<br>
NO-.165, SEQ ID NO:166, SEQ ID NO:167, and SEQ ID NO.168; and b) detecting<br>
hybridization, whereby a yield-increasing gene may be identified.<br>
The present invention also provides a method of isolating a yield-related gene from a<br>
recombinant plant cell library, comprising a) providing a preparation of plant cell DNA or a<br>
recombinant plant cell library; b) contacting the preparation or plant cell library with a<br>
detectably-labelled EG307 OR EG1117 conserved oligonucleotide under hybridization<br>
conditions providing detection of genes having 50% or greater sequence identity; and c)<br>
isolating a yield-related gene by its association with the detectable label.<br>
The present invention also provides a method of isolating a yield-related gene from<br>
plant cell DNA comprising a) providing a sample of plant cell DNA; b) providing a pair of<br>
oligonucleotides having sequence homology to a conserved region of an EG307 OR EG1117<br>
gene; c) combining the pair of oligonucleotides with the plant cell DNA sample under<br>
conditions suitable for polymerase chain reaction-mediated DNA amplification; and d)<br>
isolating the amplified yieldrrelated gene or fragment thereof.<br>
The sequences identified by the methods described herein can be used to identify<br>
agents that are useful in modulating domesticated organism-unique, enhanced or altered<br>
functional capabilities and/or correcting defects in these capabilities using these sequences.<br>
These methods employ, for example, screening techniques known in the art, such as in vitro<br>
systems, cell-based expression systems and transgenic animals and plants. The approach<br>
provided by the present invention not only identifies rapidly evolved genes, but indicates<br>
modulations that can be made to the protein that may not be too toxic because they exist in<br>
another species.<br>
The present invention also provides a method of producing an EG307 OR EG1117<br>
polypeptide comprising: a) providing a cell transfected with a polynucleotide encoding an<br>
EG307 OR EG1117 polypeptide positioned for expression in the cell; b) culturing the<br>
transfected cell under conditions for expressing the polynucleotide; and c) isolating the<br>
EG307 OR EG1117 polypeptide.<br>
A. EG307Polypeptides<br>
One embodiment of the present invention is an isolated plant EG307 polypeptide. As<br>
used herein, an EG307 polypeptide, in one embodiment, is a polypeptide that is related to<br>
(i.e., bears structural similarity to) the O. sativa polypeptide of about 447 amino acids and<br>
having the sequence depicted in Figure 2 (SEQ ID NO: 6). The original identification of such<br>
a polypeptide is detailed in the Examples. A preferred EG307 polypeptide is encoded by a<br><br><br>
polynucleotide that hybridizes under stringent hybridization conditions to at least one of the<br>
following genes: (a) a gene encoding an O. sativa EG307 polypeptide (i.e., an O. sativa gene);<br>
(b) a gene encoding an O. rufipogon EG307 polypeptide (i.e., an O. rufipogon gene); (c) a<br>
gene encoding a Zea mays mays EG307 gene; (d) a gene encoding a Zea maysparviglumis<br>
EG307 polypeptide (i.e., a. Z mays parviglumis gene); (e) a gene encoding a Zea<br>
diploperennis EG307 polypeptide (i.e., a. Z diploperennis gene); and (f) a gene encoding a<br>
Zea luxurians EG307 polypeptide (i.e., a. Z luxurians gene). It is to be noted that the term<br>
"a" or "an" entity refers to one or more of that entity; for example, a gene refers to one or<br>
more genes or at least one gene. As such, the terms "a" (or "an"), "one or more" and "at least<br>
one" can be used interchangeably herein. It is also to be noted that the terms "comprising,"<br>
"including," and "having" can be used interchangeably.<br>
As used herein, stringent hybridization conditions refer to standard hybridization<br>
conditions under which polynucleotides, including oligonucleotides, are used to identify<br>
molecules having similar nucleic acid sequences. Such standard conditions are disclosed, for<br>
example, in Sambrook et al, MOLECULAR CLONING: A LABORATORY MANUAL, Cold Spring<br>
Harbor Labs Press, 1989. Examples of such conditions are provided in the Examples section<br>
of the present application.<br>
As used herein, an O. sativa EG307 gene includes all nucleic acid sequences related to<br>
a natural O. sativa EG307 gene such as regulatory regions that control production of the O.<br>
sativa EG307 polypeptide encoded by that gene (such as, but not limited to, transcription,<br>
translation or post-translation control regions) as well as the coding region itself. In one<br>
embodiment, an O. sativa EG307 gene includes the nucleic acid sequence SEQ ID NO:4.<br>
Nucleic acid sequence SEQ ID NO:4 represents the deduced sequence of a cDNA<br>
(complementary DNA) polynucleotide, the production of which is disclosed in the Examples.<br>
It should be noted that since nucleic acid sequencing technology is not entirely error-free,<br>
SEQ ID NO:4 (as well as other sequences presented herein), at best, represents an apparent<br>
nucleic acid sequence of the polynucleotide encoding an O. sativa EG307 polypeptide of the<br>
present invention.<br>
In another embodiment, an O. sativa EG307 gene can be an allelic variant that<br>
includes a similar but not identical sequence to SEQ ID NO:4. An allelic variant of an O.<br>
sativa EG307 gene including SEQ ID NO: 1 is a locus (or loci) in the genome whose activity<br>
is concerned with the same biochemical or developmental processes, and/or a gene that that<br>
occurs at essentially thesame locus as the gene including SEQ ID NO:4, but which, due to<br>
natural variations caused by, for example, mutation or recombination, has a similar but not<br><br><br>
identical sequence. Because genomes can undergo rearrangement, the physical arrangement<br>
of alleles is not always the same. Allelic variants typically encode polypeptides having<br>
similar activity to that of the polypeptide encoded by the gene to which they are being<br>
compared. Allelic variants can also comprise alterations in the 51 or 3' untranslated regions of<br>
die gene (e.g., in regulatory control regions). Allelic variants are well known to those skilled<br>
in the art and would be expected to be found within a given rice cultivar or strain since the<br>
genome is diploid and/or among a population comprising two or more rice cultivars or strains.<br>
For example, it is believed that the O. sativa polynucleotide having nucleic acid sequences<br>
reprepsented by SEQ ID NO: 18, to be described in more detail below, represents allelic<br>
variants of the Kasalath strain of O. sativa.<br>
Similarly, a Zea mays mays EG307 gene includes all nucleic acid sequences related to<br>
a natural Z mays mays EG307 gene such as regulatory regions that control production of the<br>
Z mays mays EG307 polypeptide encoded by that gene as well as the coding region itself. In<br>
one embodiment, a Zea mays mays EG307 gene includes the nucleic acid sequence SEQ ID<br>
NO:66. Nucleic acid sequence SEQ ID NO.66 represents the deduced sequence of a cDNA<br>
polynucleotide, the production of which is disclosed in the Examples. In another<br>
embodiment, a Zea mays mays EG307 gene can be an allelic variant that includes a similar<br>
but not identical sequence to SEQ ID NO:66.<br>
According to the present invention, an isolated, or biologically pure, polypeptide, is a<br>
polypeptide that has been removed from its natural milieu. As such, "isolated" and<br>
"biologically pure" do not necessarily reflect the extent to which the polypeptide has been<br>
purified. An isolated EG307 polypeptide of the present invention can be obtained from its<br>
natural source, can be produced using recombinant DNA technology or can be produced by<br>
chemical synthesis. An EG307 polypeptide of the present invention may be identified by its<br>
ability to perform the function of natural EG307 in a functional assay. By "natural EG307<br>
polypeptide," it is meant the full length EG307 polypeptide of O. sativa, O. rufipogon, Z<br>
mays mays, and/or Z mays parviglumis. The phrase "capable of performing the function of a<br>
natural EG307 in a functional assay" means that the polypeptide has at least about 10% of the<br>
activity of the natural polypeptide in the functional assay. In other preferred embodiments,<br>
the EG307 polypeptide has at least about 20% of the activity of the natural polypeptide in the<br>
functional assay. In other preferred embodiments, the EG307 polypeptide has at least about<br>
30% of the activity of the natural polypeptide in the functional assay. In other preferred<br>
embodiments, the EG307 polypeptide has at least about 40% of the activity of the natural<br>
polypeptide in the functional assay. In other preferred embodiments, the EG307 polypeptide<br><br><br>
has at least about 50% of the activity of the natural polypeptide in the functional assay. In<br>
other preferred embodiments, the polypeptide has at least about 60% of the activity of the<br>
natural polypeptide in the functional assay. In more preferred embodiments, the polypeptide<br>
has at least about 70% of the activity of the natural polypeptide in the functional assay. In<br>
more preferred embodiments, the polypeptide has at least about 80% of the activity of the<br>
natural polypeptide in the functional assay. In more preferred embodiments, the polypeptide<br>
has at least about 90% of the activity of the natural polypeptide in the functional assay.<br>
Examples of functional assays include antibody-binding assays, or yield-increasing assays, as<br>
detailed elsewhere in this specification.<br>
As used herein, an isolated plant EG307 polypeptide can be a full-length polypeptide<br>
or any homologue of such a polypeptide. Examples of EG307 homologues include EG307<br>
polypeptides in which amino acids have been deleted (e.g., a truncated version of the<br>
polypeptide, such as a peptide), inserted, inverted, substituted and/or derivatized (e.g., by<br>
glycosylation, phosphorylation, acetylation, myristylation, prenylation, palmitoylation,<br>
amidation and/or addition of glycerophosphatidyl inositol) such that the homolog has natural<br>
EG307 activity.<br>
In one embodiment, when the homologue is administered to an animal as an<br>
immunogen, using techniques known to those skilled in the art, the animal will produce a<br>
humoral and/or cellular immune response against at least one epitope of a natural EG307<br>
polypeptide. EG307 homologues can also be selected by their ability to perform the function<br>
of EG307 in a functional assay.<br>
Plant EG307 polypeptide homologues can be the result of natural allelic variation or<br>
natural mutation. EG307 polypeptide homologues of the present invention can also be<br>
produced using techniques known in the art including, but not limited to, direct modifications<br>
to the polypeptide or modifications to the gene encoding the polypeptide using, for example,<br>
classic or recombinant DNA techniques to effect random or targeted mutagenesis.<br>
In accordance with the present invention, a mimetope refers to any compound that is<br>
able to mimic the ability of an isolated plant EG307 polypeptide of the present invention to<br>
perform the function of an EG307 polypeptide of the present invention in a functional assay.<br>
Examples of mimetopes include, but are not limited to, anti-idiotypic antibodies or fragments<br>
thereof, that include at least one binding site that mimics one or more epitopes of an isolated<br>
polypeptide of the present invention; non-polypeptideaceous immunogenic portions, of an<br>
isolated polypeptide (e.g., carbohydrate structures); and synthetic or natural organic<br>
molecules, including nucleic acids, that have a structure similar to at least one epitope of an<br><br><br>
isolated polypeptide of the present invention. Such mimetopes can be designed using<br>
computer-generated structures of polypeptides of the present invention. Mimetopes can also<br>
be obtained by generating random samples of molecules, such as oligonucleotides, peptides or<br>
other organic molecules, and screening such samples by affinity chromatography techniques<br>
using the corresponding binding partner.<br>
The minimal size of an EG307 polypeptide homologue of the present invention is a<br>
size sufficient to be encoded by a polynucleotide capable of forming a stable hybrid with the<br>
complementary sequence of a polynucleotide encoding the corresponding natural polypeptide.<br>
As such, the size of the polynucleotide encoding such a polypeptide homologue is dependent<br>
on nucleic acid composition and percent homology between the polynucleotide and<br>
complementary sequence as well as upon hybridization conditions per se (e.g., temperature,<br>
salt concentration, and formamide concentration). It should also be noted that the extent of<br>
homology required to form a stable hybrid can vary depending on whether the homologous<br>
sequences are interspersed throughout the polynucleotides or are clustered (i.e., localized) in<br>
distinct regions on the polynucleotides. The rriinimal size of such polynucleotides is typically<br>
at least about 12 to about 15 nucleotides in length if the polynucleotides are GOrich and at<br>
least about 15 to about 17 bases in length if they are AT-rich. Preferably, the polynucleotide<br>
is at least 12 bases in length.<br>
As such, the minimal size of a polynucleotide used to encode an EG307 polypeptide<br>
homologue of the present invention is from about 12 to about 18 nucleotides in length. There<br>
is no limit, other than a practical limit, on the maximal size of such a polynucleotide in that<br>
the polynucleotide can include a portion of a gene, an entire gene, or multiple genes, or<br>
portions thereof. Similarly, the minimal size of an EG307 polypeptide homologue of the<br>
present invention is from about 4 to about 6 amino acids in length, with preferred sizes<br>
depending on whether a full-length, fusion, multivalent, or functional portions of such<br>
polypeptides are desired. Preferably, the polypeptide is at least 30 amino acids in length.<br>
Any plant EG307 polypeptide is a suitable polypeptide of the present invention.<br>
Suitable plants from which to isolate EG307 polypeptides (including isolation of the natural<br>
polypeptide or production of the polypeptide by recombinant or synthetic techniques) include<br>
maize, wheat, barley, rye, millet, chickpea, lentil, flax, olive, fig almond, pistachio, walnut,<br>
beet, parsnip, citrus fruits, including, but not limited to, orange, lemon, lime, grapefruit,<br>
tangerine, minneola, and tangelo, sweet potato, bean, pea, chicory, lettuce, cabbage,<br>
cauliflower, broccoli, turnip, radish, spinach, asparagus, onion, garlic, pepper, celery, squash,<br>
pumpkin, hemp, zucchini, apple, pear, quince, melon, plum, cherry, peach, nectarine, apricot,<br><br><br>
strawberry, grape, raspberry, blackberry, pineapple, avocado, papaya, mango, banana,<br>
soybean, tomato, sorghum, sugarcane, sugarbeet, sunflower, rapeseed, clover, tobacco, carrot,<br>
cotton, alfalfa, rice, potato, eggplant, cucumber, Arabidopsis, and woody plants such as<br>
coniferous and deciduous trees, with rice and maize being preferred. Preferred rice plants<br>
from which to isolate EG307 polypeptides include Nipponbarel and 2, Lemont, IR64, Teqing,<br>
Azucena, and Kasalath 1,2,3, and 4 strains of O. sativa.<br>
A preferred plant EG307 polypeptide of the present invention is a compound that<br>
when expressed or modulated in a plant, is capable of increasing the yield of the plant.<br>
One embodiment of the present invention is a fusion polypeptide that includes an<br>
EG307 polypeptide-containing domain attached to a fusion segment. Inclusion of a fusion<br>
segment as part of a EG307 polypeptide of the present invention can enhance the<br>
polypeptide's stability during production, storage and/or use. Depending on the segment's<br>
characteristics, a fusion segment can also act as an immunopotentiator to enhance the immune<br>
response mounted by an animal immunized with an EG307 polypeptide containing such a<br>
fusion segment. Furthermore, a fusion segment can function as a tool to simplify purification<br>
of an EG307 polypeptide, such as to enable purification of the resultant fusion polypeptide<br>
using affinity chromatography. A suitable fusion segment can be a domain of any size that<br>
has the desired function (e.g., imparts increased stability, imparts increased immunogenicity<br>
to a polypeptide, and/or simplifies purification of a polypeptide). It is within the scope of the<br>
present invention to use one or more fusion segments. Fusion segments can be joined to<br>
amino and/or carboxyl termini of the EG307-containing domain of the polypeptide. Linkages<br>
between fusion segments and EG307-containing domains of fusion polypeptides can be<br>
susceptible to cleavage in order to enable straightforward recovery of the EG307-containing<br>
domains of such polypeptides. Fusion polypeptides are preferably produced by culturing a<br>
recombinant cell transformed with a fusion polynucleotide that encodes a polypeptide<br>
including the fusion segment attached to either the carboxyl and/or amino terminal end of a<br>
EG307-containing domain.<br>
Preferred fusion segments for use in the present invention include a glutathione<br>
binding domain; a metal binding domain, such as a poly-histidine segment capable of binding<br>
to a divalent metal ion; an immunoglobulin binding domain, such as Polypeptide A,<br>
Polypeptide G, T cell, B cell, Fc receptor or complement polypeptide antibody-binding<br>
domains; a sugsir binding domain such as a maltose binding domain from a maltose binding<br>
polypeptide; and/or a "tag" domain (e.g., at least a portion of P-galactosidase, a strep tag<br>
peptide, other domains that can be purified using compounds that bind to the domain, such as<br><br><br>
monoclonal antibodies). More preferred fusion segments include metal binding domains,<br>
such as a poly-histidine segment; a maltose binding domain; a strep tag peptide.<br>
Preferred plant EG307 polypeptides of the present invention are rice EG307<br>
polypeptides and maize EG307 polypeptides. More preferred EG307 polypeptides are O.<br>
sativa, O. rufipogon, Z. mays mays, Zea mays parviglumis, Z. diploperennis and Z. luzurians<br>
EG307 polypeptides. O. sativa strains inlcude Nipponbare, Azucena, Kasalath 1,2,3, and 4,<br>
Teqing, Lemont, and IR64. Z mays parviglumis strains include Benz, BK4, IA19, and<br>
Wilkes. Z mays mays strains include BS7, HuoBai, Makki, Mini 3, Pint, Sari, Smena, and<br>
W22.<br>
One preferred O. sativa EG307 polypeptide of the present invention is a polypeptide<br>
encoded by an O. sativa polynucleotide that hybridizes under stringent hybridization<br>
conditions with complements of polynucleotides represented by SEQ ID NO:l, SEQ ID<br>
NO:91, SEQ ID. NO:2, SEQ ID NO:4, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:10, SEQ<br>
ID NO:11, SEQ ID NO:12, SEQ ID NO:14, SEQ ID NO:15, SEQ ID NO:17, and/or SEQ ID<br>
NO: 18. Such an EG307 polypeptide is encoded by a polynucleotide that hybridizes under<br>
stringent hybridization conditions with a polynucleotide having nucleic acid sequence SEQ ID<br>
NO:1, SEQ ID NO:91, SEQ ID. NO:2, SEQ ID NO:4, SEQ ID NO:5, SEQ ID NO:7, SEQ ID<br>
NO:10, SEQ IDNO:11, SEQ IDNO:12, SEQ IDNO:14, SEQ IDNO:15, SEQ IDNO:17,<br>
and/or SEQ ID NO: 18.<br>
Inspection of EG307 genomic nucleic acid sequences indicates that the genes<br>
comprise several regions, including a first exon region, a first intron region, a second exon<br>
region, a second intron region, and a third exon region.<br>
Polynucleotides SEQ ED NO:4 and SEQ ID NO:91 represent the 5' and 3' ends of the<br>
EG307 gene in O. sativa (cv. Nipponbare). SEQ ID NO:4 and SEQ ID NO:91 are joined by a<br>
number of nucleotides, the exact number of which is unknown due to potential<br>
insertions/deletions in the non-coding portions of the gene, but is believed to be about 6.<br>
Translation of SEQ ID NO:4 and SEQ ID NO:91 suggests that the O. sativa EG307<br>
polynucleotide includes an open reading frame. The reading frame encodes an O. sativa<br>
EG307 polypeptide of about 447 amino acids, the deduced amino acid sequence of which is .<br>
represented herein as SEQ ID NO:6, assuming an open reading frame'having an initiation<br>
(start) codon spanning from about nucleotide 37 through about nucleotide 39 of SEQ ID NO:4<br>
and a termination (stop) codon spanning from about nucleotide 2278 through about nucleotide<br>
2280 of SEQ ID NO:4, with the first exon spanning nucleotides 1-126 of SEQ ID NO: 4, the<br>
first intron spanning nucleotides 9-822 of SEQ ID NO:91, the second exon spanning<br><br><br>
nucleotides 823-1141 of SEQ ID NO:91, the second intron spanning nucleotides 1142-1222<br>
of SEQ ID NO:91, and the third exon spanning nucleotides 1223-2157 of SEQ ID NO:91.<br>
The open reading frame from nucleotide 37 through about nucleotide 2280 of SEQ ID NO:4<br>
is represented herein as SEQ ID NO:5.<br>
Similarly, translation of O. sativa (strain Azucena) polynucleotide SEQ ID NO:l<br>
suggests an open reading frame from about nucleotide 3 to about nucleotide 2410 of SEQ ID<br>
NO:1, with the first exon spanning nucleotides 1-92 of SEQ ID NO: 1, the first intron<br>
spanning nucleotides 93-1075 of SEQ ID NO:l, the second exon spanning nucleotides 1076-<br>
1394 of SEQ ID NO:l, the second intron spanning nucleotides 1395-1475 of SEQ ID NO:l,<br>
and the third exon spanning nucleotides 1476-2441 of SEQ ID NO:l. The open reading<br>
frame is represented herein as SEQ ID NO:2, and encodes a polypeptide represented herein as<br>
SEQ ID NO:3.<br>
Similarly, translation of O. sativa (strain Teqing) polynucleotide SEQ ID NO:7<br>
suggests an open reading frame from about nucleotide 21 to about nucleotide 2421, with the<br>
first exon spanning nucleotides 1-110 of SEQ ID NO:7, the first intron spanning nucleotides<br>
111-1089 of SEQ ID NO:7, the second exon spanning nucleotides 1090-1405 of SEQ ID<br>
NO:7, the second intron spanning nucleotides 1406-1486 of SEQ ID NO:7, and the third exon<br>
spanning nucleotides 1487-2461 of SEQ ID NO:7. The open reading frame is represented<br>
herein as SEQ ID NO:8, and encodes a polypeptide represented herein as SEQ ID NO:9.<br>
Similarly, polynucleotides SEQ ID NO.10 and SEQ ID NO:ll represent the 5' and 3'<br>
ends of the EG307 gene in O. sativa (strain Lemont). SEQ ID NO:10 and SEQ ID NO:l 1 are<br>
joined by an unknown number of nucleotides. In the genomic sequence, there may be<br>
insertions/deletions in the non-coding portions of the gene, thus the actual number of<br>
nucleotides is unknown, but is believed to be about 10. Translation of O. sativa (strain<br>
Lemont) polynucleotides SEQ ID NO: 10 and SEQ ID NO:l 1 suggests an open reading frame<br>
from about nucleotide 166 of SEQ ID NO: 10 to about nucleotide 1547 of SEQ ID NO:l 1,<br>
with the first exon spanning nucleotides 1-255 of SEQ ID NO: 10, the first intron spanning<br>
nucleotides 255-451 of SEQ IDNO.IO and nucleotides l-212of SEQ ID NO:l 1, the second<br>
exon spanning nucleotides 213-531 of SEQ ID NO:l 1, the second intron spanning nucleotides<br>
532-612 of SEQ ID NO:ll, and the third exon spanning nucleotides 613-1616 of SEQ ID<br>
NO:l 1. The open reading frame is represented herein as SEQ ID NO: 12, and encodes a<br>
polypeptide represented herein as SEQ ID NO: 13.,<br>
Similarly, translation of O. sativa (strain IR64) polynucleotide SEQ ID NO: 14"<br>
suggests an open reading frame from about nucleotide 1 to about nucleotide 2400, with the<br><br><br>
first exon spanning nucleotides 1-90 of SEQ ID NO:14, the first intron spanning nucleotides<br>
91-1068 of SEQ ID NO:14, the second exon spanning nucleotides 1069-1384 of SEQ ID<br>
NO: 14, the second intron spanning nucleotides 1385-1465 of SEQ ID NO: 14, and the third<br>
exon spanning nucleotides 1466-2459 of SEQ ID NO: 11. The open reading frame is<br>
represented herein as SEQ ID NO: 14, and encodes a polypeptide represented herein as SEQ<br>
ID NO:15.<br>
Similarly, translation of O. sativa (strain Kasalath) polynucleotide SEQ ID NO: 17<br>
suggests an open reading frame from about nucleotide 2 to about nucleotide 2402,, with the<br>
first exon spanning nucleotides 1-91 of SEQ ID NO:17, the first intron spanning nucleotides<br>
92-1070 of SEQ ID NO:17, the second exon spanning nucleotides 1071-1386 of SEQ ID<br>
NO:17, the second intron spanning nucleotides 1387-1467 of SEQ ID NO:17, and the third<br>
exon spanning nucleotides 1468-2432 of SEQ ID NO: 17.<br>
The open reading frame is represented as SEQ ID NO: 18, and encodes a polypeptide<br>
represented herein as SEQ ID NO: 19. In SEQ ID NO: 18, "N" at postion 889 is "G", and "N"<br>
at position 971 is "A" for strain Kasalath 1, making amino acid residue 297 in SEQ ID NO: 19<br>
a valine, and amino acid residue 324 a glutamine. In SEQ ID NO: 18, "N" at postion 889 is<br>
"G", and "N" at position 971 is "T" for strain Kasalath 2, making amino acid residue 297 in<br>
SEQ ID NO:19 a valine, and amino acid residue 324 a leucine. In SEQ ID NO: 18, "N" at<br>
postion 889 is "C", and "N" at position 971 is "A" for strain Kasalath 3, making amino acid<br>
residue 297 in SEQ ID NO: 19 a leucine, and amino acid residue 324 a glutamine. In SEQ ID<br>
NO: 18, "N" at postion 889 is "C", and "N" at position 971 is "T" for strain Kasalath 4,<br>
making amino acid residue 297 in SEQ ID NO: 19 a leucine, and amino acid residue 324 a<br>
leucine.<br>
A preferred O. sativa EG307 polypeptide of the present invention is a polypeptide<br>
encoded by a polynucleotide that hybridizes under stringent hybridization conditions with<br>
polynucleotides represented by SEQ ID NO.l, SEQ ID NO:91, SEQ ID. NO:2, SEQ ID<br>
NO:4, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:10, SEQ ID NO:l 1, SEQ ID NO:12, SEQ<br>
ID NO:14, SEQ ID NO:15, SEQ ID NO.17, and/or SEQ EDNO:18.<br>
Preferred O.rufipogon EG307 polypeptides of the present invention are polypeptides<br>
encoded by an O.rufipogon polynucleotide that hybridizes under stringent hybridization<br>
conditions with complements of polynucleotides represented by SEQ ID NO:20, SEQ ID<br>
NO:21, SEQ ID. NO:23, SEQ ID NO:24, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:28,<br>
SEQ ID NO:29, SEQ ID NO:30, and/or SEQ ID NO:31. Such an EG307 polypeptide is<br>
encoded by a polynucleotide that hybridizes under stringent hybridization conditions with a<br><br><br>
polynucleotide having nucleic acid sequence SEQ ID NO:20, SEQ ID NO:21, SEQ ID.<br>
NO:23, SEQ ID NO:24, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:28, SEQ ID N0.29,<br>
SEQ ID NO:30, and/or SEQ ID NO:31.<br>
Polynucleotides SEQ ID NO:27 and SEQ ID NO:28 represent the 5' and 3' ends of the<br>
EG307 gene in O. rufrogon (strain 5953). SEQ ID NO:27 and SEQ ID NO:28 are joined by a<br>
number of nucleotides, the exact number of which is unknown due to potential<br>
insertions/deletions in the non-coding portions of the gene, but is believed to be about 23.<br>
Translation of SEQ ID NO:27 and SEQ ID NO:28 suggests that the O. rufipogon EG307<br>
polynucleotide includes an open reading frame. The reading frame encodes an O. rufipogon<br>
EG307 polypeptide of about 446 amino acids, the deduced amino acid sequence of which is<br>
represented herein as SEQ ID NO:30, assuming an open reading frame having an initiation<br>
(start) codon spanning from about nucleotide 18 through about nucleotide 20 of SEQ ID<br>
NO:27 and a termination (stop) codon spanning from about nucleotide 1330 through about<br>
nucleotide 1332 of SEQ ID NO:28, with the first exon spanning nucleotides 1-107 of SEQ ID<br>
NO:27, no first intron, the second exon spanning nucleotides 1-316 of SEQID NO:28, the<br>
second intron spanning nucleotides 317-397 of SEQ ID NO:28, and the third exon spanning<br>
nucleotides 398-1332 of SEQ ID NO:28. The open reading frame from nucleotide 18 of SEQ<br>
ID NO:27 through about nucleotide 1332 of SEQ ID NO:28 is represented herein as SEQ ID<br>
NO.-29.<br>
Similarly, translation of O. rufipogon (strain 5948) polynucleotide SEQ ID NO:20<br>
suggests an open reading frame from about 15 nucelotides 5' of nucleotide 1 to about<br>
nucleotide 2385, first exon not represented, the first intron spanning nucleotides 1-1053 of<br>
SEQ ID NO:20, the second exon spanning nucleotides 1054-1369 of SEQ ID NO:20, the<br>
second intron spanning nucleotides 1370-1450 of SEQ ID NO:20, and the third exon spanning<br>
nucleotides 1451-2447 of SEQ ID NO:20. The open reading frame is represented herein as<br>
SEQ ID NO:21, and encodes a polypeptide represented herein as SEQ ID NO:22.<br>
Similarly, polynucleotides SEQ ID NO:23 and SEQ ID NO:24 represent the 5' and 3'<br>
ends of the EG307 gene in O. rufrogon (strain 5949). SEQ ID NO:23 and SEQ ID NO:24 are<br>
joined by a number of nucleotides, the exact number of which is unknown due to potential<br>
insertions/deletions in the non-coding portions of the gene, but is believed to be about 13.<br>
Translation of SEQ ID NO:23 and SEQ ID NO:24 suggests an open reading frame from about<br>
nucleotide 57 of SEQ ID NO:23 to about nucleotide 1562 of SEQ ID NO:24, with the first<br>
exon spanning nucleotides 1-146 of SEQ ID NO:23, the first intron spanning nucleotides 1-<br>
230 of SEQ ID NO:24, the second exon spanning nucleotides 231-546 of SEQ ID NO:24, the<br><br><br>
second intron spanning nucleotides 547-627 of SEQ ID NO:24, and the third exon spanning<br>
nucleotides 628-1615 of SEQ ID NO:24. The open reading frame is represented as SEQ ID<br>
NO:25, and encodes a polypeptide represented herein as SEQ ID NO:26.<br>
Similarly, translation of O. rufpogon (strain IRCG 105491) polynucleotide SEQ ID<br>
NO:90 suggests an open reading frame from about nucleotide 1 to about nucleotide 1341.<br>
The open reading frame is represented herein as SEQ ID NO:31 encoding a polypeptide<br>
represented herein as SEQ ID NO:32.<br>
A preferred O. rufipogon EG307 polypeptide of the present invention is a polypeptide<br>
encoded by a polynucleotide that hybridizes under stringent hybridization conditions with a<br>
polynucleotide represented by SEQ ID NO:20, SEQ ID NO:21, SEQ ID. NO:23, SEQ ID<br>
NO:24, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:28, SEQ ID NO:29, SEQ ID NO.30,<br>
and/or SEQ ID NO:31.<br>
One preferred Zea mays parviglumis EG307 polypeptide of the present invention is a<br>
polypeptide encoded by a Zea mays parviglumis polynucleotide that hybridizes under<br>
stringent hybridization conditions with complements of polynucleotides represented by SEQ ID NO:66, SEQ ID NO:67, SEQ ID NO:69, SEQ ID. NO:70, SEQ ID NO:71, SEQ ID<br>
NO:73, SEQ ID NO:74, SEQ ID NO:75, SEQ ID NO:77, SEQ ID NO:59, and/or SEQ ID<br>
NO:78. Such an EG307 polypeptide is encoded by a polynucleotide that hybridizes under<br>
stringent hybridization conditions with a polynucleotide having nucleic acid sequence SEQ ID<br>
NO:66, SEQ ID NO:67, SEQ ID NO:69, SEQ ID. NO.70, SEQ ID NO:71, SEQ ID NO:73,<br>
SEQ ID NO:74, SEQ ID NO:75, SEQ ID NO:77, SEQ ID NO:59, and/or SEQ ID NO:78.<br>
Translation of SEQ ID NO:66 suggests that the Zea mays parviglumis EG307<br>
polynucleotide (strain Benz) includes an open reading frame. The reading frame encodes an<br>
Zea mays parviglumis EG307 polypeptide of about 448 amino acids, the deduced amino acid<br>
sequence of which is represented herein as SEQ ID NO:68, assuming an open reading frame<br>
having an initiation (start) codon spanning from about nucleotide 1 through about nucleotide 3<br>
of SEQ ID NO:66 and a termination (stop) codon spanning from about nucleotide 2569<br>
through about nucleotide 2571 of SEQ ID NO:66, with the first exon spanning nucleotides 1-<br>
. 81 of SEQ ID NO:66, the first intron spanning nucleotides 82-1204 of SEQ ID NO:66, the<br>
second exon spanning nucleotides 1205-1517 of SEQ ID NO:66, the second intron spanning<br>
nucleotides 1518-1618 of SEQ ID NO:66, and the third exon spanning nucleotides 1619-2644<br>
of SEQ ID NO:66. The open reading frame from nucleotide 3 through about nucleotide 2571<br>
of SEQ ID NO-.66 is represented herein as SEQ ID NO:67.<br><br><br>
Similarly, polynucleotides SEQ ID NO:69 and SEQ ID NO:70 represent the 5' and 3'<br>
ends of the EG307 gene in Z mays parviglumis (strain BK4). SEQ ID NO:69 and SEQ ID<br>
NO:70 are joined by a number of nucleotides, the exact number of which is unknown due to<br>
potential insertions/deletions in the non-coding portions of the gene, but is believed to be<br>
about 10. Translation of Z. mays parviglumis (strain BK4) polynucleotide SEQ ID NO:69<br>
and SEQ ID NO:70 suggests an open reading frame from about nucleotide 10 of SEQ ID<br>
NO:69 to about nucleotide 1728 of SEQ ID NO:70, with the first exon spanning nucleotides<br>
1-90 of SEQ ID NO;69, the first intron spanning nucleotides 91-586 of SEQ ID NO:69 and<br>
nucleotides 1-361 of SEQ ID NO:70, the second exon spanning nucleotides 362-674 of SEQ<br>
ID NO:70, the second intron spanning nucleotides 675-775 of SEQ ID NO:70, and the third<br>
exon spanning nucleotides 776-1775 of SEQ ID NO:l 1. The open reading frame is<br>
represented as SEQ ID NO:71, and encodes a polypeptide represented herein as SEQ ID NO:72.<br>
Similarly, polynucleotides SEQ ID NO:73 and SEQ ID NO:74 represent the 5' and 3'<br>
ends of the EG307 gene in Z mays parviglumis (strain IA19). SEQ ID NO:73 and SEQ ID<br>
NO:74 are joined by a number of nucleotides, the exact number of which is unknown due to<br>
potential insertions/deletions in the non-coding portions of the gene, but is believed to be<br>
about 12. Translation of Z mays parviglumis (strain IA19) polynucleotides SEQ ID NO:73<br>
and SEQ ID NO:74 suggests an open reading frame from about nucleotide 69 of SEQ ID<br>
NO:73 to about nucleotide 1280 of SEQ ID NO:74, with the first exon spanning nucleotides<br>
1-149 of SEQ ID NO:73, the first intron spanning nucleotides 150-305 of SEQ ID NO:73, the<br>
second exon spanning nucleotides 1-226 of SEQ ID NO:74, the second intron spanning<br>
nucleotides 227-327 of SEQ ID NO:74, and the third exon spanning nucleotides 328-1309 of<br>
SEQ ID NO:74. The open reading frame is represented herein as SEQ ID NO:75, and<br>
encoding a polypeptide represented herein as SEQ ID NO:76.<br>
Similarly, polynucleotides SEQ ID NO:77 and SEQ ID NO:59 represent the 5' and 3'<br>
ends of the EG307 gene in Z mays parviglumis (strain Wilkes). SEQ ID NO:77 and SEQ ID<br>
NO:59 are joined by a number of nucleotides, the exact numberof which is unknown due to<br>
potential insertions/deletions in the non-coding portions of the gene, but is believed to be-<br>
about 14. Translation of Z mays parviglumis (strain Wilkes) polynucleotide SEQ ID NO:77<br>
and SEQ ID NO:59 suggests an open reading frame from about nucleotide 36 of SEQ ID<br>
NO:77 to about nucleotide 1598 of SEQ ID NO:59, with the first exon spanning nucleotides<br>
1-86 of SEQ ID NO:77, the first intron spanning nucleotides 1-231 of SEQ ID NO:59, the<br>
second exon spanning nucleotides 232-544 of SEQ ID NO:59, the second intron spanning<br><br><br>
nucleotides 545-645 of SEQ ID NO:59, and the third exon spanning nucleotides 656-1640 of<br>
SEQ ID NO:59. The open reading frame is represented herein as SEQ ID NO:78, and<br>
encoding a polypeptide represented herein as SEQ ID NO:79.<br>
A preferred EG307 polypeptide of the present invention is a polypeptide encoded by a<br>
polynucleotide that hybridizes under stringent hybridization conditions with a polynucleotide<br>
represented by SEQ ID NO:33, SEQ ID NO:34, SEQ ID. NO:35, SEQ ID NO:37, SEQ ID<br>
NO:38, SEQ ID NO:40, SEQ ID NO:41, SEQ ID NO:42, SEQ ID NO:44, SEQ ID NO:45,<br>
SEQ ID NO:46, SEQ ID NO:47, SEQ ID NO:49, SEQ ID. NO:50, SEQ ID NO:51, SEQ ID<br>
NO:53, SEQ ID NO:54, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:58, SEQ ID NO:60,<br>
SEQ ID NO:62, SEQ ID NO:63, and/or SEQ ID NO:64.<br>
One preferred Zea mays mays EG307 polypeptide of the present invention is a<br>
polypeptide encoded by an Zea mays mays polynucleotide that hybridizes under stringent<br>
hybridization conditions with complements of polynucleotides represented by SEQ ID NO:33,<br>
SEQ ID NO:34, SEQ ID. NO:35, SEQ ID NO:37, SEQ ID NO:38, SEQ ID NO:40, SEQ ID<br>
NO:41, SEQ ID NO:42, SEQ ID NO:44, SEQ ID NO:45, SEQ ID NO:46, SEQ ID NO:47,<br>
SEQ ID NO:49, SEQ ID. NO:50, SEQ ID NO:51, SEQ ID NO:53, SEQ ID NO:54, SEQ ID<br>
NO:55, SEQ ID NO:57, SEQ ID NO:58, SEQ ID NO:60, SEQ ID NO:62, SEQ ID NO:63,<br>
and/or SEQ ID NO:64. Such an EG307 polypeptide is encoded by a polynucleotide that<br>
hybridizes under stringent hybridization conditions with a polynucleotide having nucleic acid<br>
sequence SEQ ID NO:33, SEQ ID NO:34, SEQ ID. NO:35, SEQ ID NO:37, SEQ ID NO:38,<br>
SEQ ID NO:40, SEQ ID NO:41, SEQ ID NO:42, SEQ ID NO:44, SEQ ID NO:45, SEQ ID<br>
NO:46, SEQ ID NO:47, SEQ ID NO:49, SEQ ID. NO:50, SEQ ID NO:51, SEQ ID NO:53,<br>
SEQ ID NO:54, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:58, SEQ ID NO:59, SEQ ID<br>
NO:60, SEQ ID NO:62, SEQ ID NO:63, and/or SEQ ID NO:64.<br>
Polynucleotides SEQ ID NO:33 and SEQ ID NO:34 represent the 5' and 3' ends of the<br>
EG307 gene in Z mays mays (strain BS 7). SEQ ID NO:33 and SEQ ID NO:34 are joined by<br>
a number of nucleotides, the exact number of which is unknown due to potential<br>
insertions/deletions in the non-coding portions of the gene, but is believed to be about 21.<br>
Translation of SEQ ID NO:33 and SEQ ID NO:34 suggests that the Zea mays mays EG307<br>
polynucleotide includes an open reading frame. The reading frame encodes an Zea mays<br>
mays EG307 polypeptide of about 448 amino acids, the deduced amino acid sequence of<br>
which is represented herein as SEQ ID NO:36, assuming an open reading frame having an<br>
initiation (start) codon spanning from about nucleotide 3 uirough about.nucleotide 5 of SEQ<br>
ID NO:33 and a termination (stop) codon spanning from about nucleotide 1396 through about<br><br><br>
nucleotide 1398 of SEQ ID NO:34, with the first exon spanning nucleotides 1 -83 of SEQ ID<br>
NO:33, the first intron spanning nucleotides 84-180 of SEQ ID NO:33 and nucleotides 1-31<br>
of SEQ ID NO:34, the second exon spanning nucleotides 32-344 of SEQ ID NO:34, the<br>
second intron spanning nucleotides 345-445 of SEQ ID NO:34, and the third exon spanning<br>
nucleotides 446-1447 of SEQ ID NO:34. The open reading frame from nucleotide 3 of SEQ<br>
ID NO:33 through about nucleotide 1398 of SEQ ID NO:34 is represented herein as SEQ ID<br>
NO:35.<br>
Similarly, translation of Z mays mays (strain HuoBai) polynucleotide SEQ ID NO:37<br>
suggests an open reading frame from about nucleotide 28 to about nucleotide 2599, with the<br>
first exon spanning nucleotides 1-108 of SEQ ID NO:37, the first intron spanning nucleotides<br>
1Q9-1232 of SEQID NO:37, the second exon spanning nucleotides 1233-1545 of SEQ ID<br>
NO:37, the second intron spanning nucleotides 1546-1646 of SEQ ID NO:37, and the third<br>
exon spanning nucleotides 1647-2646 of SEQ ID NO:37. The open reading frame is<br>
represented herein as SEQ ID NO:38, and encodes a polypeptide represented herein as SEQ<br>
ID NO:39.<br>
Similarly, polynucleotides SEQ ID NO:40 and SEQ ID NO:41 represent 5' end to the<br>
3' end of the EG307 gene in Z mays mays (strain Makki). SEQ ID NO:40 and SEQ ID<br>
NO:41 are joined by a number of nucleotides, the exact number of which is unknown due to<br>
potential insertions/deletions in the non-coding portions of the gene, but is believed to be<br>
about 20. Translation of Z mays mays (strain Makki) polynucleotides SEQ ID NO:40 and<br>
SEQ ID NO:41 suggests an open reading frame from about nucleotide 61 of SEQ ID NO:40<br>
to about nucleotide 2263 of SEQ ID NO:41, with the first exon spanning nucleotides 1-141 of<br>
SEQ ID NO:40, the first intron spanning nucleotides 142-262 of SEQ ID NO:40 and<br>
nucleotides 1-896 of SEQ ID NO:41, the second exon spanning nucleotides 897-1209 of SEQ<br>
ID NO:41, the second intron spanning nucleotides 1210-1310 of SEQ ID NO:41, and the third<br>
exon spanning nucleotides 1311-2311 ofSEQID NO:41. The open reading frame is<br>
represented as SEQ ID NO:42 encoding a polypeptide represented herein as SEQ ID NO:43.<br>
Similarly, polynucleotides SEQ ID NO:44, SEQ ID NO:45 and SEQ ID NO:46<br>
represent the three parts of the EG307 gene in Z mays mays (strain Mini 3), from the 5' end<br>
to the 3' end, SEQ ID NO:44, SEQ ID NO:45 and SEQ ID NO:46 are joined by a number of<br>
nucleotides, the exact number of which is unknown due to potential insertions/deletions in the<br>
non-coding portions of the gene, but is belived to be 19 between SEQ ID NO:44 and SEQ ID<br>
NO:45, and 17 between SEQ ID NO:45 and SEQ ID NO:46. Translation of Z mays mays<br>
(strain Minl3) polynucleotides SEQ ID NO:44, SEQ ID NO:45 and SEQ ID NO:46 suggests<br><br><br>
an open reading frame from about nucleotide 45 of SEQ ID NO:44 to about nucleotide 1741<br>
of SEQ ID NO:46, with the first exon spanning nucleotides 1-125 of SEQ ID NO:44, the first<br>
intron spanning nucleotides 1-198 of SEQ ID NO:45 and nucleotides 1-374 of SEQ ID<br>
NO:46, the second exon spanning nucleotides 375-687 of SEQ ID NO:46, the second intron<br>
spanning nucleotides 688-788 of SEQ ID NO:46, and the third exon spanning nucleotides<br>
789-1787 of SEQ ID NO:46. The open reading frame is represented herein as SEQ ID<br>
NO:47, and encodes a polypeptide represented herein as SEQ ID NO:48.<br>
Similarly, polynucleotides SEQ ID NO:49 and SEQ ID NO:50 represent the 5' and 3'<br>
ends of the EG307 gene in 2. mays mays (strain Pira). SEQ ID NO:49 and SEQ ID NO:50 are<br>
joined by a number of nucleotides, the exact number of which is unknown due to potential<br>
insertions/deletions in the non-coding portions of the gene. Translation of Z mays mays<br>
(strainPira) polynucleotides SEQ ID NO:49 and SEQ ID NO:50 suggests anopenreading<br>
frame from about nucleotide 31 of SEQ ID NO.49 to about nucleotide 1722 of SEQ ID<br>
NO:50, with the first exon spanning nucleotides 1-111 of SEQ ID NO:49, the first intron<br>
spanning nucleotides 112-495 of SEQ ID NO:49 and nucleotides 1-355 of SEQ ID NO:50, the<br>
second exon spanning nucleotides 356-668 of SEQ ID NO:50, the second intron spanning<br>
nucleotides 669-769 of SEQ ID NO:50, and the third exon spanning nucleotides 770-1768 of<br>
SEQ ID NO:50. The open reading frame is represented herein as SEQ ID NO:51, and<br>
encodes a polypeptide represented herein as SEQ ID NO:52.<br>
Similarly, polynucleotides SEQ ID NO:53 and SEQ ID NO:54 represent the 5' and 3'<br>
ends of the EG307 gene in Z mays mays (strain Sari). SEQ ID NO:53 and SEQ ID NO:54 are<br>
joined by a number of nucleotides, the exact number of which is unknown due to potential<br>
insertions/deletions in the non-coding portions of the gene, but is believed to be about 22.<br>
Translation of Z mays mays (strain Pira) polynucleotides SEQ ID NO:53 and SEQ ID NO:54<br>
suggests an open reading frame from about nucleotide 19 of SEQ ID NO:53 to about<br>
nucleotide 1756 of SEQ ID NO:54, with the first exon spanning nucleotides 1-99 of SEQ ID<br>
NO:53, the first intron spanning nucleotides 100-212 of SEQ ID NO:53 and nucleotides 1-389<br>
of SEQ ID NO:54, the second exon spanning nucleotides 390-702 of SEQ ID NO:54, the<br>
second intron spanning nucleotides 703-803 of SEQ ID NO:54, and the third exon spanning<br>
nucleotides 804-1803 of SEQ ID NO:54. The open reading frame is represented herein as<br>
SEQ ID NO:55, and encodes a polypeptide represented hereiaas SEQ ID NO:56.<br>
Similarly, polynucleotides SEQ ID NO:57 and SEQ ID NO:58 represent the 5' and 3'<br>
ends of the EG307 gene m Z mays mays (strain Smena). SEQ ID NO:57 and SEQ ID NO:58<br>
are joined by a number of nucleotides, the exact number of which is unknown due to potential<br><br><br>
insertions/deletions in the non-coding portions of the gene, but is believed to be 14.<br>
Translation of Z mays mays (strain Smena) polynucleotides SEQ ID NO:57 and SEQ ID<br>
NO:58 suggests an open reading frame from about nucleotide 68 of SEQ ID NO:57 to about<br>
nucleotide 2199 of SEQ ID NO:58, with the first exon spanning nucleotides 1-148 of SEQ ID<br>
NO:57, the first intron spanning nucleotides 149-305 of SEQ ID NO:57 and nucleotides 1-834<br>
of SEQ ID NO:58, the second exon spanning nucleotides 835-1147 of SEQ ID NO:58, the<br>
second intron spanning nucleotides 1148-1248 of SEQ ID NO:58, and the third exon spanning<br>
nucleotides 1249-2208 of SEQ ID NO:58. Additionally, sequence SEQ ID NO:59 contains a<br>
deletion at starting after nucleotide 738 of SEQ ID NO:59. The open reading frame is<br>
represented herein as SEQ ID NO:60, and encodes a polypeptide represented herein as SEQ<br>
ID NO:61.<br>
Similarly, polynucleotides SEQ ID NO:62 and SEQ ID NO:63 represent the 5' and 3'<br>
ends of the EG307 gene in Z mays mays (strain W22). SEQ ID NO:62 and SEQ ID NO:63<br>
are joined by a number of nucleotides, the exact number of which is unknown due to potential<br>
insertions/deletions in the non-coding portions of the gene, but is believed to be about 22.<br>
Translation of Z mays mays (strain W22) polynucleotides SEQ ID NO:62 and SEQ ID NO:63<br>
suggests an open reading frame from about nucleotide 1 of SEQ ID NO:62 to about<br>
nucleotide 1367 of SEQ ID NO:63, with the first exon spanning nucleotides 1-81 of SEQ ID<br>
NQ:62, the first intron spanning nucleotides 82-893 of SEQ ID NO:62, the second exon<br>
spanning nucleotides 1-313 of SEQ ID NO:63, the second intron spanning nucleotides 314-<br>
414 of SEQ ID NO:63, and the third exon spanning nucleotides 415-1411 of SEQ ID NO:63.<br>
The open reading frame is represented herein as SEQ ID NO:64, and encodes a polypeptide<br>
represented herein as SEQ ID NO:65.<br>
A preferred Z mays mays EG307 polypeptide of the present invention is a polypeptide<br>
encoded by a polynucleotide that hybridizes under stringent hybridization conditions with a<br>
polynucleotide represented by SEQ ID NO:33, SEQ ID NO:34, SEQ ID. NO:35, SEQ ID<br>
NO:37, SEQ ID NO:38, SEQ ID NO:40, SEQ ID NO:41, SEQ ID NO:42, SEQ ID NO:44,<br>
SEQ ID NO:45, SEQ ID NO:46, SEQ ID NO:47, SEQ ID NO-.49, SEQ ID. NO:50, SEQ ID<br>
NO:51, SEQ ID NO:53, SEQ ID NO:54, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:58,<br>
SEQ ID NO:60, SEQ ID NO:62, SEQ ID NO:63, and/or SEQ ID NO:64.<br>
A preferred O. rufipogon EG307 polypeptide of the present invention is a polypeptide<br>
encoded by a polynucleotide that hybridizes under stringent hybridization conditions with a<br>
polynucleotide represented by SEQ ID NO:20, SEQ ID NO:21, SEQ ID. NO:23, SEQ ID<br><br><br>
NO:24, SEQ ID NO:25, SEQID NO:27, SEQ ID NO:28, SEQ ID NO:29, SEQ ID NO:30,<br>
and/or SEQ ID NO:31.<br>
One preferred Zea diploperennis EG307 polypeptide of the present invention is a<br>
polypeptide encoded by an Zea mays parviglumis polynucleotide that hybridizes under<br>
stringent hybridization conditions with complements of polynucleotides represented by SEQ<br>
ID NO:80, SEQ ID NO:81, and/or SEQ ID NO:82. Such an EG307 polypeptide is encoded<br>
by a polynucleotide that hybridizes under stringent hybridization conditions with a<br>
polynucleotide having nucleic acid sequence SEQ ID NO:80, SEQ ID NO:81, and/or SEQ ID<br>
NO:82.<br>
Polynucleotides SEQ ID NO:80 and SEQ ID NO:81 represent the 5' and 3' ends of the<br>
EG307 gene in Z. diploperennis SEQ ID NO:80 and SEQ ID NO:8l are joined by a number<br>
of nucleotides, the exact number of which is unknown due to potential insertions/deletions in<br>
the non-coding portions of the gene, but is believed to be about 24. One preferred Zea<br>
diploperennis EG307 polypeptide of the present invention is a polypeptide encoded by an Zea<br>
diploperennis polynucleotide that hybridizes under stringent hybridization conditions with<br>
complements of polynucleotides represented by SEQ ID NO:80 and. SEQ ID NO:81. Such an<br>
EG307 polypeptide is encoded by a polynucleotide that hybridizes under stringent<br>
hybridization conditions with a polynucleotide having nucleic acid sequence SEQ ID NO:80<br>
andSEQID NO:81.<br>
Translation of SEQ ID NO:80 and SEQ ID NO:81 suggests that the Zea mays<br>
diploperennis EG307 polynucleotides includes an open reading frame. The reading frame<br>
encodes an Zea diploperennis EG307 polypeptide of about 448 amino acids, the deduced<br>
amino acid sequence of which is represented herein as SEQ ID NO:83, assuming an open<br>
reading frame having an initiation (start) codon spanning from about nucleotide 21 through<br>
about nucleotide 23 of SEQ ID NO.80 and a termination (stop) codon spanning from about<br>
nucleotide 1656 through about nucleotide 1658 of SEQ ID NO:81, with the first exon<br>
spanning nucleotides 1-101 of SEQ ID NO:80, the first intron spanning nucleotides 102-225<br>
of SEQ ID NO:80 and nucleotides 1-291 of SEQ ID NO:81, the second exon spanning<br>
nucleotides 292-313 of SEQ ID NO:81, the second intron spanning nucleotides 314-705 of<br>
SEQ ID NO:81, and the third exon spanning nucleotides 706-1672 of SEQ ID NO:81. The<br>
open reading frame from nucleotide 21 of SEQ ID NO:80 through about nucleotide 1658 of<br>
SEQ ID N0.81 is represented herein as SEQ ID NO:82.<br>
A preferred Z. diploperennis EG307 polypeptide of the present invention is a...<br>
polypeptide encoded by a polynucleotide that hybridizes under stringent hybridization<br><br><br>
conditions with polynucleotides represented by SEQ ID NO:80, SEQ ID NO:81, and/or SEQ<br>
ID NO:82.<br>
One preferred Zea Itaurians EG307 polypeptide of the present invention is. a<br>
polypeptide encoded by an Zea luxurians polynucleotide that hybridizes under stringent<br>
hybridization conditions with complements of polynucleotides represented by SEQ ID NO:84<br>
and/or SEQ ID N0.85. Such an EG307 polypeptide is encoded by a polynucleotide that<br>
hybridizes under stringent hybridization conditions with a polynucleotide having nucleic acid<br>
sequence SEQ ID NO:84 and/or SEQ ID NO:85.<br>
Translation of SEQ ID NO:84 suggests that the Zea Itaurians EG307 polynucleotide<br>
includes an open reading frame. The reading frame encodes an Zea luxurians EG307<br>
polypeptide of about 448 amino acids, the deduced amino acid sequence of which is<br>
represented herein as SEQ ID NO:86, assuming an open reading frame having an initiation<br>
(start) codon spanning from about nucleotide 5 through about nucleotide 7 of SEQ ID NO-.84<br>
and a termination (stop) codon spanning from about nucleotide 2363 through about nucleotide<br>
2367 of SEQ ID NO:84, with the first exon spanning nucleotides 1-85 of SEQ ID NO:84, the<br>
first intron spanning nucleotides 86-998 of SEQ ID NO:84, the second exon spanning<br>
nucleotides 999-1311 of SEQ ID NO:84, the second intron spanning nucleotides 1312-1414<br>
of SEQ ID NO:84, and the third exon spanning nucleotides 1415-2423 of SEQ ID NO:84.<br>
The open reading frame from nucleotide 5 through about nucleotide 2367 of SEQ ID NO:84<br>
is represented herein as SEQ ID NO:85.<br>
A preferred Z. luxurians EG307 polypeptide of the present invention is a polypeptide<br>
encoded by a polynucleotide mat hybridizes under stringent hybridization conditions with<br>
polynucleotides represented by SEQ ID NO:84, and/or SEQ ID NO:85.<br>
Comparison of the various O. sativa, O. rufipogon, Z. mays mays, Z. mays<br>
parviglumis, Z, diploperennis, and Z. luxurians EG307 nucleic acid sequences and amino acid<br>
sequences indicates that these species of plants possess similar EG307 genes and<br>
polypeptides. The nucleotide sequences of the coding region of EG307 from the various<br>
strains of O. sativa and O. rufipogon have 99.0% sequence identity, when compared to each<br>
other, which makes clear that they are homologous. All rice sequences, both ancestral and<br>
modern, share the same stop codon (TAG), and (for the 5' UTR sequence that we have<br>
collected to date), the 5' UTR sequences have 98.4% sequence identity. The protein<br>
sequences of the various strains of O. sativa and O. rufipogon have 98.2% sequence identity,<br>
again demonstrating that these are homologous sequences. The protein sequence of EG307<br>
from rice is about 94% identical to the protein sequence of EG307 from maize, again<br><br><br>
demonstrating their homology. The protein sequences of maize EG307 and teosinte EG307<br>
have 99.8% sequence identity.<br>
Finding this degree of identity between O. sativa, O. rufipogon, Z. mays mays, Z. mays<br>
parviglumis, Z. diploperennis, and Z luxurians EG307 nucleic acid sequences and amino acid<br>
sequences supports the ability to obtain any plant EG307 polypeptide and polynucleotide<br>
given the polypeptide and nucleic acid sequences disclosed herein.<br>
These plant EG307 polypeptides, and the polynucleotides that encode them, represent novel<br>
compounds with utility in increasing yield in a plant<br>
Preferred plant EG307 polypeptides of the present invention include polypeptides<br>
comprising amino acid sequences that are at least about 30%, preferably at least about 50%,<br>
more preferably at least about 75%, more preferably at least about 80%, more preferably at<br>
least about 85%, more preferably at least about 90%, and more preferably at least about 95%, <br>
more preferably at least about 98% identical to one or more of the amino acid sequences<br>
disclosed herein for O. sativa, O. rufipogon, Z. mays mays, Z mays parviglumis, Z<br>
diploperennis, and Z luxurians EG307 polypeptides of the present invention. More preferred<br>
plant EG307 polypeptides of the present invention include: polypeptides encoded by at least a<br>
portion of SEQ ID NO. 1 and/or SEQ ID NO:2 and, as such, have amino acid sequences that<br>
include at least a portion of SEQ ID NO:3; polypeptides encoded by at least a portion of SEQ<br>
ID NO:4, SEQ ID NO:81 and/or SEQ ID NO:5 and, as such, have amino acid sequences that<br>
include at least a portion of SEQ ID NO:6; polypeptides encoded by at least a portion of SEQ<br>
ID NO:7 and/or SEQ ID NO:8 and, as such, have amino acid sequences that include at least a<br>
portion of SEQ ID NO:9; polypeptides encoded by at least aportion of SEQ ID NO:10, SEQ<br>
ID NO:l 1, and/or SEQ ID NO: 12 and, as such, have amino acid sequences that include at<br>
least a portion of SEQ ID NO: 13 ; polypeptides encoded by at least a portion of SEQ ID<br>
NO: 14 and/or SEQ ID NO: 15 and, as such, have amino acid sequences that include at least a<br>
portion of SEQ ID NO: 16; polypeptides encoded by at least a portion of SEQ ID NO: 17<br>
and/or SEQ ID NO: 18 and, as such, have amino acid sequences that include at least a portion<br>
of SEQ ID NO: 19; polypeptides encoded by at least a portion of SEQ ID NO:20 and/or SEQ<br>
ID NO:21 and, as such, have amino acid sequences that include at least a portion of SEQ ID<br>
NO:22; polypeptides encoded by at least a portion of SEQ ID NO:23, SEQ ID NO:24, and/or<br>
SEQ ID NO:25 and, as such, have amino acid sequences that include at least a portion of SEQ<br>
ID NO-.26; polypeptides encoded by at least a portion of SEQ ID NO:27, SEQ ID NO:28<br>
and/or SEQ ID NO:29-and, as such, have amino acid sequences that include at least a portion<br>
of SEQ ID NO:30; polypeptides encoded by at least a portion of SEQ ID NO:90 and/or SEQ<br><br><br>
ID N0:31 and, as such, have amino acid sequences that include at least a portion of SEQ ID<br>
NO:32; polypeptides encoded by at least a portion of SEQ ID NO:33, SEQ ID NO:34 and/or<br>
SEQ ID NO:35 and, as such, have amino acid sequences that include at least a portion of SEQ<br>
ID NO:36; polypeptides encoded by at least a portion of SEQ ID NO:37 and/or SEQ ID<br>
NO:38 and, as such, have amino acid sequences that include at least a portion of SEQ ID<br>
NO:39; polypeptides encoded by at least a portion of SEQ ID NO:40, SEQ ID NO:4l, and/or<br>
SEQ ID NO:42 and, as such, have amino acid sequences that include at least a portion of SEQ<br>
ID NO:43; polypeptides encoded by at least a portion of SEQ ID NO:44, SEQ ID NO:45,<br>
SEQ ID NO:46, and/or SEQ ID NO:47 and, as such, have amino acid sequences that include<br>
at least a portion of SEQ ID NO:48; polypeptides encoded by at least a portion of SEQ ID<br>
NO:49, SEQ ID NO:50, and/or SEQ ID NO:51 and, as such, have amino acid sequences that<br>
include at least a portion of SEQ ID NO:52; polypeptides encoded by at least a portion of<br>
SEQ ID NO:53, SEQ ID NO:54, and/or SEQ ID NO:55 and, as such, have amino acid<br>
sequences that include at least a portion of SEQ ID NO:56; polypeptides encoded by at .least a<br>
portion of SEQ ID NO:57, SEQ ID NO:58, and/or SEQ ID NO.60 and, as such, have amino<br>
acid sequences that include at least a portion of SEQ ID NO:61; polypeptides encoded by at<br>
least a portion of SEQ ID NO:62, SEQ ID NO:63, and/or SEQ ID NO:64 and, as such, have<br>
amino acid sequences that include at least a portion of SEQ ID NO:65; polypeptides encoded<br>
by at least a portion of SEQ ID NO:66, and/or SEQ ID NO:67 and, as such, have amino acid<br>
sequences that include at least a portion of SEQ ID NO:68; polypeptides encoded by at least a<br>
portion of SEQ ID NO:69, SEQ ID NO:70, and/or SEQ ID NO:71 and, as such, have amino<br>
acid sequences that include at least a portion of SEQ ID NO:72; polypeptides encoded by at<br>
least a portion of SEQ ID NO:73, SEQ ID NO:74, and/or SEQ ID NO:75 and, as such, have<br>
amino acid sequences that include at least a portion of SEQ ID NO:76; polypeptides encoded<br>
by at least a portion of SEQ ID NO:77, SEQ ID NO:59, and/or SEQ ID NO:78 and, as such,<br>
have amino acid sequences that include at least a portion of SEQ ID NO:79; polypeptides<br>
encoded by at least a portion of SEQ ID NO:80, SEQ ID NO:81, and/or SEQ ID NO:82 and,<br>
as such, have amino acid sequences that include at least a portion of SEQ ID NO-.83; and<br>
polypeptides encoded by at least a portion of SEQ ID NO:84, and/or SEQ ID NO:85 and, as<br>
such, have amino acid sequences that include at least a portion of SEQ ID NO:86. As used<br>
herein, "at least a portion" of a polynucleotide or polypeptide means a portion having, the<br>
minimal size characteristics of such sequences, as described above, or any larger fragment of<br>
the full length molecule, up to and including the full length molecule. For example, a portion<br>
of a polynucleotide may be 12 nucleotides, 13 nucleotides, 14 nucleotides, 15 nucleotides, and<br><br><br>
so on, going up to the full length polynucleotide. Similarly, a portion of a polypeptide may be<br>
4 amino acids, 5 amino acids, 6 amino acids, 7 amino acids, and so on, going up to the full<br>
length polypeptide. The length of the portion to be used will depend on the particular<br>
application. As discussed above, a portion of a polynucleotide useful as hybridization probe<br>
may be as short as 12 nucleotides. A portion of a polypeptide useful as an epitope may be as<br>
short as 4 amino acids. A portion of a polypeptide that performs the function of the full-<br>
length polypeptide would generally be longer than 4 amino acids.<br>
Particularly preferred plant EG307 polypeptides of the present invention are<br>
polypeptides that include SEQ ID NO:3, SEQ ID NO:6, SEQ ID NO:9, SEQ ID NO:13, SEQ<br>
ID NO:16, SEQ ID NO:19, SEQ ID NO:22, SEQ ID NO:26, SEQ ID NO:30, SEQ ID NO:32,<br>
SEQ ID NO:36, SEQ ID NO:39, SEQ ID NO:43, SEQ ID NO:48, SEQ ID NO:52, SEQ ID<br>
NO:56, SEQ ID NO:61, SEQ ID NO:65, SEQ ID NO:68. SEQ ID NO:72, SEQ ID NO:76,<br>
SEQ ID NO:79, SEQ ID NO:83and7or SEQ ID NO:86 (including, but not limited to the<br>
encoded polypeptides, full-length polypeptides, processed polypeptides, fusion polypeptides<br>
and multivalent polypeptides thereof) as well as polypeptides that are truncated homologues<br>
of polypeptides that include at least portions of the aforementioned SEQ ID NOs. Examples<br>
of methods to produce such polypeptides are disclosed herein, including in the Examples<br>
section.<br>
used herein, an EG307 polypeptide, in one embodiment, is a polypeptide that is related to<br>
(i.e., bears structural similarity to) the O. rufigogon polypeptide of about 552 amino acids and<br>
having the sequence depicted in Figure 7 (SEQ ID NO:95). The original identification of<br>
such a polypeptide is detailed in the Examples. A preferred EG1117 polypeptide is encoded<br>
by a polynucleotide that hybridizes under stringent hybridization conditions to at least one of<br>
the following genes: (a) a gene encoding an O. sativa EG1117 polypeptide (i.e., an O. sativa<br>
gene); (b) a gene encoding an O. ruflpogon EG1117 polypeptide (i.e., an O. ruflpogon gene);<br>
(c) a gene encoding a Zea mays mays EG1117 gene; (d) a gene encoding a Zea mays<br>
parviglumis EG1117 polypeptide (i.e., a. Z. mays parviglumis gene).<br>
As used herein, an O. sativa EG1117 gene includes all nucleic acid sequences related<br>
to a natural O. sativa EG307 gene such as regulatory regions that control production of the O.<br>
sativa EG1117 polypeptide encoded by that gene (such as, but not limited to, transcription,<br>
translation or post-translation control regions) as well as the coding region itself. In one<br>
embodiment, an O. sativa EG1117 gene includes the nucleic acid sequence SEQ ID NO:4.<br><br><br>
Nucleic acid sequence SEQ ID N0:4 represents the deduced sequence of a cDNA<br>
(complementary DNA) polynucleotide, the production of which is disclosed in the Examples.<br>
It should be noted that since nucleic acid sequencing technology is not entirely error-free,<br>
SEQ ID NO:4 (as well as other sequences presented herein), at best, represents an apparent<br>
nucleic acid sequence of the polynucleotide encoding an O. sativa EG307 polypeptide of the<br>
present invention.<br>
In another embodiment, an O. sativa EG1117 gene can be an allelic variant that<br>
includes a similar but not identical sequence to SEQ ID NO:92 and/or SEQ ID NO:93i<br>
An EG1117 polypeptide of the present invention may be identified by its ability to<br>
perform the function of natural EG1117 in a functional assay. By "natural EG1117<br>
polypeptide," it is meant the full length EG1117 polypeptide of O. sativa, O. rufipogon, Z.<br>
mays mays, and/or Z mays parviglumis. The phrase "capable of performing the function of a<br>
natural EG1117 in a functional assay" means that the polypeptide has at least about 10% of<br>
the activity of the natural polypeptide in the functional assay. In other preferred<br>
embodiments, the EG1117 polypeptide has at least about 20% of the activity of the natural<br>
polypeptide in the functional assay. In other preferred embodiments, the EG1117 polypeptide<br>
has at least about 30% of the activity of the natural polypeptide in the functional assay. In<br>
other preferred embodiments, the EG1117 polypeptide has at least about 40% of the activity<br>
of the natural polypeptide in the functional assay. In other preferred embodiments, the<br>
EG1117 polypeptide has at least about 50% of the activity of the natural polypeptide in the<br>
functional assay. In other preferred embodiments, the polypeptide has at least about 60% of<br>
the activity of the natural polypeptide in the functional assay. In more preferred<br>
embodiments, the polypeptide has at least about 70% of the activity of the natural polypeptide<br>
in the functional assay. In more preferred embodiments, the polypeptide has at least about<br>
80% of the activity of the natural polypeptide in the functional assay. In more preferred<br>
embodiments, the polypeptide has at least about 90% of the activity of the natural polypeptide<br>
in the functional assay. Examples of functional assays include antibody-binding assays, or<br>
yield-increasing assays, as detailed elsewhere in this specification.<br>
As used herein, an isolated plant EG1117 polypeptide can be a full-length polypeptide<br>
or any homologue of such a polypeptide. In one embodiment, when the homologue is<br>
administered to an animal as an immunogen, using techniques known to those skilled in the<br>
art, the animal will produce a humoral and/or cellular immune response against at least one<br>
epitope of a natural EG1117 polypeptide. EG1117 homologues can also be selected by their<br>
ability to perform the function of EG 1117 in a functional assay.<br><br><br>
Plant EG117 polypeptide homologues can be the result of natural allelic variation or<br>
natural mutation. EG1117 polypeptide homologues of the present invention can also be<br>
produced using techniques known in the art including, but not limited to, direct modifications<br>
to the polypeptide or modifications to the gene encoding the polypeptide using, for example,<br>
classic or recombinant DNA techniques to effect random or targeted mutagenesis.<br>
In accordance with the present invention, a mimetope refers to any compound that is<br>
able to mimic the ability of an isolated plant EG307 polypeptide of the present invention to<br>
perform the function of an EG307 polypeptide of the present invention in a functional assay.<br>
Examples of mimetopes include, but are not limited to, anti-idiotypic antibodies or fragments<br>
thereof, that include at least one binding site that mimics one or more epitopes of an isolated<br>
polypeptide of the present invention; non-polypeptideaceous immunogenic portions of an<br>
isolated polypeptide (e.g., carbohydrate structures); and synthetic or natural organic<br>
molecules, including nucleic acids, that have a structure similar to at least one epitope of an<br>
isolated polypeptide of the present invention. Such mimetopes can be designed using<br>
computer-generated structures of polypeptides of the present invention. Mimetopes can also<br>
be obtained by generating random samples of molecules, such as oligonucleotides, peptides or<br>
other organic molecules, and screening such samples by affinity chromatography techniques<br>
using the corresponding binding partner.<br>
The minimal size of an EG307 polypeptide homoiogue of the present invention is a<br>
size sufficient to be encoded by a polynucleotide capable of forming a stable hybrid with the<br>
complementary sequence of a polynucleotide encoding the corresponding natural polypeptide.<br>
Minimal size characteristics are disclosed herein.<br>
Any plant EG1117 polypeptide is a suitable polypeptide of the present invention.<br>
Suitable plants from which to isolate EG1117 polypeptides (including isolation of the natural<br>
polypeptide or production of the polypeptide by recombinant or synthetic techniques) include<br>
those described in the section entitles "EG307 Polypeptides."<br>
A preferred plant EG1117 polypeptide of the present invention is a compound that<br>
when expressed or modulated in a plant, is capable of increasing the yield of the plant.<br>
One embodiment of the present invention is a fusion polypeptide that includes an<br>
EG 1117 polypeptide-containing domain attached to a fusion segment.<br>
Preferred plant EG 1117 polypeptides of the present invention are rice EG 1117<br>
polypeptides and maize EG1117 polypeptides. More preferred EG1117 polypeptides are O.<br>
sativa, O. rufipogon, 2. mays mays, and Zea mays parviglumis EG1117 polypeptides. O.<br>
sativa strains inlcude Nipponbare, Azucena, Kasalath 1,2,3, and 4, Teqing, Lemont, and<br><br><br>
IR64. Z mays parviglumis strains include Benz, BK4, IA19, and Wilkes. Z mays mays<br>
strains include BS7, HuoBai, Makki, Mini 3, Pira, Sari, Sraena, and W22.<br>
One preferred O. ruflpogon EG1117 polypeptide of the present invention is a<br>
polypeptide encoded by an O. rufipgon polynucleotide that hybridizes under stringent<br>
hybridization conditions with complements of polynucleotides represented by SEQ ID NO:92,<br>
SEQ ID NO:93, SEQ ID NO:94, SEQ ID NO:96, SEQ ID NO:97, and/or SEQ ID NO:98.<br>
One preferred O. sativa EG1117 polypeptide of the present invention is a polypeptide<br>
encoded by an O. sativa polynucleotide that hybridizes under stringent hybridization<br>
conditions with complements of polynucleotides represented by SEQ ID NO: 100, SEQ ID<br>
NO:101, SEQ ID NO:102, SEQ ID NO:103, SEQ ID NO:104, SEQ ID NO:104, SEQ ID<br>
NO.106, SEQ ID NO:107, SEQ ID NO:109, SEQ ID NO:110, SEQ ID NO:112, SEQ ID<br>
NO:113, SEQ ID NO:114, SEQ ID NO:116, SEQ ID NO:117.<br>
One preferred Z mays mays EG1117 polypeptide of the present invention is a<br>
polypeptide encoded by an mays mays polynucleotide that hybridizes under stringent<br>
hybridization conditions with complements of polynucleotides represented by SEQ ID<br>
NO:l 19, SEQ ID NO-.120, SEQ ID NO:122, SEQ ID NO-.123, SEQ ID NO:124, SEQ ID<br>
NO:125, SEQ ID NO:127, SEQ ID NO:128, SEQ ID NO:129, SEQ ID NO:130, SEQ ID<br>
NO:133, SEQ ID NO:135, SEQ ID NO:136, SEQID NO:137, SEQ ID NO:138, SEQ ID<br>
NO:140, SEQ ID N0:141, SEQ ID N0.142, SEQ ID NO-.144, SEQ ID NO:145, SEQ ID<br>
NO:146, SEQ ID NO:147, SEQ ID NO:149, SEQ ID NO:150, SEQ ID NO:151, SEQ ID<br>
NO:152, SEQ ID NO:154, SEQ ID NO:155,<br>
One preferred Z mays parviglumis EG 1117 polypeptide of the present invention is a<br>
polypeptide encoded by an Z. mays parviglumis polynucleotide that hybridizes under stringent<br>
hybridization conditions with complements of polynucleotides represented by SEQ ID<br>
NO:157, SEQ ID NO-.158, SEQ ID NO:160, SEQ ID NO:161, SEQ ID NO:162, SEQ ID<br>
N0:163, SEQ ID NO:165, SEQ ID NO:166, SEQ ID NO:167, and/or SEQ ID NO:168.<br>
Inspection of EG1117 genomic nucleic acid sequences for rice indicates that the genes<br>
comprise several regions, including a first exon region, a first intron region, a second exon<br>
region, a second.intron region, a third exon region, a third intron region, and a fourth exon<br>
region. The locations of these regions in each of the EG1117 rice and rice ancestor genomic<br>
nucleic acid sequences is summarized in the Table below:<br><br><br>
Translation of the genomic sequences suggests that the O. ruflpogon and O. sativa<br>
EG1117 polynucleotide include open reading frames. The deduced protein sequence of O.<br>
sativa strain Nipponbare was used to perform a BLAST search, A very strong protein<br>
BLAST hit to Arabidopsis PTR2-B (histidine transporting protein, NP_78313) suggests that<br>
only about 30 codons of coding sequence (CDS) are missing from the rice sequence (Figure<br>
8),<br><br><br><br>
Translation of the genomic sequences suggests that the Z mays mays and Z mays<br>
parviglumis EG 1117 polynucleotide include open reading frames.<br>
A summary of the open reading frame information appears in the following Table:<br><br>
Preferred plant EG1117 polypeptides of the present invention include polypeptides<br>
comprising amino acid sequences that are at least about 30%, preferably at least about 50%,<br>
more preferably at least about 75%, more preferably at least about 80%, more preferably at<br>
least about 85%, more preferably at least about 90%, and more preferably at least about 95%,<br>
more preferably at least about 98% identical to one or more of the amino acid sequences<br>
disclosed herein for O. sativa, O. rufipogon, Z. mays mays, and Z. mays parviglumis, EG1117<br>
polypeptides of the present invention. More preferred plant EG1117 polypeptides of the<br>
present invention include: polypeptides encoded by at least a portion of SEQ ID NO. 92, SEQ<br>
ID NO. 93, and/or SEQ ID NO:94 and, as such, have amino acid sequences that include at<br>
least a portion of SEQ ID NO:95; polypeptides encoded by at least a portion of SEQ ID<br>
NO:96, SEQ ID NO:97 and/or SEQ ID NO:98 and, as such, have amino acid sequences that<br>
include at least a portion of SEQ ID NO:99; polypeptides encoded by at least a portion of<br>
SEQ ID NO:100 and/or SEQ ID NO:101 and, as such, have amino acid sequences that include<br>
at least a portion of SEQ ID NO: 102; polypeptides encoded by at least a portion of SEQ ID<br>
NO: 103, and/or SEQ ID NO: 104 and, as such, have amino acid sequences that include at least<br>
a portion of SEQ ID NO: 105 ; polypeptides encoded by at least a portion of SEQ ID NO:106<br>
and/or SEQ ID NO: 107 and, as such, have amino acid sequences that include at least a portion<br>
of SEQ ID NO: 108; polypeptides encoded by at least a portion of SEQ ID NO:09 and/or SEQ<br><br>
ID N0:110 and, as such, have amino acid sequences that include at least a portion of SEQ ID<br>
NO: 111; polypeptides encoded by at least a portion of SEQ ID NO: 112, SEQ ID NO: 113<br>
and/or SEQ ID NO:l 14 and, as such, have amino acid sequences that include at least a portion<br>
of SEQ ID NO: 115; polypeptides encoded by at least a portion of SEQ ID NO: 116 and/or<br>
SEQ ID NO: 117 and, as such, have amino acid sequences that include at least a portion of<br>
SEQ ID NO: 118; polypeptides encoded by at least a portion of SEQ ID NO: 119 and/or SEQ<br>
ID NO: 120 and, as such, have amino acid sequences that include at least a portion of SEQ ID<br>
NO:121; polypeptides encoded by at least a portion of SEQ ID NO:122, SEQ ID NO:123<br>
SEQ ID NO-.124 and/or SEQ ID NO:125,and, as such, have amino acid sequences that include<br>
at least a portion of SEQ ID NO: 126; polypeptides encoded by at least a portion of SEQ ID<br>
NO:127, SEQ ID NO:128, SEQ ID NO:129 and/or SEQ ID NO:130 and, as such, have amino<br>
acid sequences that include at least a portion of SEQ ID NO:131; polypeptides encoded by at<br>
least a portion of SEQ ID NO:132 and/or SEQ ID NO:133 and, as such, have amino acid<br>
sequences that include at least a portion of SEQ ID NO: 134; polypeptides encoded by at least<br>
a portion of SEQ ID NO:135, SEQ ID NO:136, SEQ ID NO:137, and/or SEQ ID NO:138<br>
and, as such, have amino acid sequences that include at least a portion of SEQ ID NO: 139;<br>
polypeptides encoded by at least a portion of SEQ ID NO:140, SEQ ID NO:141, and/or SEQ<br>
ID NO:142 and, as such, have amino acid sequences that include at least a portion of SEQ ID<br>
NO: 143; polypeptides encoded by at least a portion of SEQ ID NO: 144, SEQ ID NO: 145,<br>
SEQ ID NO:146 and/or SEQ ID NO:147 and, as such, have amino acid sequences that include<br>
at least a portion of SEQ ID NO: 148; polypeptides encoded by at least a portion of SEQ ID<br>
NO: 149, SEQ ID NO: 150, SEQ ID NO: 151, and/or SEQ ID NO: 152 and, as such, have amino<br>
acid sequences that include at least a portion of SEQ ID NO: 153; polypeptides encoded by at<br>
least a portion of SEQ ID NO: 154 and/or SEQ ID NO: 155 and, as such, have amino acid<br>
sequences that include at least a portion of SEQ ID NO: 156; polypeptides encoded by at least<br>
a portion of SEQ ID NO: 157, and/or SEQ ID NO: 158 and, as such, have amino acid<br>
sequences that include at least a portion of SEQ ID NO:l 59; polypeptides encoded by at least<br>
a portion of SEQ ID NO: 160, SEQ ID NO:161, SEQ ID NO: 162, and/or SEQ ID NO: 163<br>
and, as such, have amino acid sequences that include at least a portion of SEQ ID NO: 164;<br>
and polypeptides encoded by at least a portion of SEQ ID NO:165, SEQ ID NO:166, SEQ ID<br>
NO: 167, and/or SEQ ID NO: 168 and, as such, have amino acid sequences that include at least<br>
a portion of SEQ ID NO: 169. As used herein, "at least a portion" of a polynucleotide or<br>
polypeptide means a portion having the minimal size characteristics of such sequences, as<br>
described above, or any larger fragment of the full length molecule, up to and including the<br><br><br>
full length molecule. For example, a portion of a polynucleotide may be 12 nucleotides, 13<br>
nucleotides, 14 nucleotides, 15 nucleotides, and so on, going up to the full length<br>
polynucleotide. Similarly, a portion of a polypeptide may be 4 amino acids, 5 amino acids, 6<br>
amino acids, 7 amino acids, and so on, going up to the full length polypeptide. The length of<br>
the portion to be used will depend on the particular application. As discussed above, a portion<br>
of a polynucleotide useful as hybridization probe may be as short as 12 nucleotides. A<br>
portion of a polypeptide useful as an epitope may be as short as 4 amino acids. A portion of a<br>
polypeptide that performs the function of the full-length polypeptide would generally be<br>
longer than 4 amino acids.<br>
Particularly preferred plant EG1117 polypeptides of the present invention are<br>
polypeptides that include SEQ ID NO:95, SEQ ID NO:99, SEQ ID NO.102, SEQ ID NO:105,<br>
SEQ ID NO-.108, SEQID NO:lll, SEQID NO:115, SEQID NO:118, SEQ ID NO:121, SEQ<br>
ID NO:126, SEQ ID NO:131, SEQ ID NO:134, SEQ ID NO:139, SEQ ID NO:143, SEQ ID<br>
NO:148, SEQ ID N0.153, SEQ ID NO:156, SEQ ID NO:159, SEQ ID NO:164, SEQ ID<br>
NO: 169, and/or SEQ ID NO: 170 (including, but not limited to the encoded polypeptides, full-<br>
length polypeptides, processed polypeptides, fusion polypeptides and multivalent<br>
polypeptides thereof) as well as polypeptides that are truncated homologues of polypeptides<br>
that include at least portions of the aforementioned SEQ ID NOs. Examples of methods to<br>
produce such polypeptides are disclosed herein, including in the Examples section.<br>
C. EG307 Polynucleotides<br>
One embodiment of the present invention is an isolated plant polynucleotide that<br>
hybridizes under stringent hybridization conditions with at least one of the following genes:<br>
an 0. sativa EG307 gene, an O. rufipogon EG307 gene, a Z mays mays EG307 gene, a Z<br>
maysparviglumis EG307 gene, aZ diploperennis EG307 gene, and aZ luxwians gene. The<br>
identifying characteristics of such genes are heretofore described. A polynucleotide of the<br>
present invention can include an isolated natural plant EG307 gene or ahomologue thereof,<br>
the latter of which is described in more detail below. A polynucleotide of the present<br>
invention can include one or more regulatory regions, full-length or partial coding regions, or<br>
combinations thereof. The minimal size of a polynucleotide of the present invention is the<br>
minimal size that can form a stable hybrid with one of the aforementioned genes under<br>
stringent hybridization conditions. Suitable and preferred plants are disclosed above.<br>
In accordance with the present invention, an isolated polynucleotide is a<br>
polynucleotide that has been removed from its natural milieu (i.e., that has been subject to<br>
human manipulation). As such, "isolated" does not reflect the extent to which the<br><br><br>
polynucleotide has been purified. An isolated polynucleotide can include DNA, RNA, or<br>
derivatives of either DNA or RNA.<br>
. An isolated plant EG307 polynucleotide of the present invention can be obtained from<br>
its natural source either as an entire (i.e., complete) gene or a portion thereof capable of<br>
forming a stable hybrid with that gene. An isolated plant EG307 polynucleotide can also be<br>
produced using recombinant DNA technology (e.g., polymerase chain reaction (PCR)<br>
amplification, cloning) or chemical synthesis. Isolated plant EG307 polynucleotides include<br>
natural polynucleotides and homologues thereof, including, but not limited to, natural allelic<br>
variants and modified polynucleotides in which nucleotides have been inserted, deleted,<br>
substituted, and/or inverted in such a manner that such modifications do not substantially<br>
interfere with the polynucleotide's ability to encode an EG307 polypeptide of the present<br>
invention or to form stable hybrids under stringent conditions with natural gene isolates.<br>
A plant EG307 polynucleotide homologue can be produced using a number of<br>
methods known to those skilled in the art (see, for example, Sambrook et al., ibid). For<br>
example, polynucleotides can be modified using a variety of techniques including, but not<br>
limited to, classic mutagenesis techniques and recombinant DNA techniques, such as site-<br>
directed mutagenesis, chemical treatment of a polynucleotide to induce mutations, restriction<br>
enzyme cleavage of a nucleic acid fragment, ligation of nucleic acid fragments, polymerase<br>
chain reaction (PCR) amplification and/or mutagenesis of selected regions of a nucleic acid<br>
sequence, synthesis of oligonucleotide mixtures and ligation of mixture groups to "build" a<br>
mixture of polynucleotides and combinations thereof. Polynucleotide homologues can be<br>
selected from a mixture of modified nucleic acids by screening for the function of the<br>
polypeptide encoded by the nucleic acid (e.g., ability to elicit an immune response against at<br>
least one epitope of an EG307 polypeptide, ability to increase yield in a transgenic plant<br>
containing an EG307 gene) and/or by hybridization with an O. sativa EG307 gene, with an O.<br>
rufipogon EG307 gene, with a Z mays mays EG307 gene, with a Z mays parviglumis EG307<br>
gene, a Z diploperennis EG307 gene and/or a Z. luxurians EG307 gene.<br>
An isolated polynucleotide of the present invention can include a nucleic acid<br>
sequence that encodes at least one plant EG307 polypeptide of the present .invention,<br>
examples of such polypeptides being disclosed herein. Although the phrase "polynucleotide"<br>
primarily refers to the physical polynucleotide and the.phrase "nucleic acid sequence"<br>
primarily refers to the sequence of nucleotides on the polynucleotide, the two phrases can be<br>
used interchangeably, especially with respect to a polynucleotide, or a nucleic acid sequence,<br>
being capable of encoding an EG307 polypeptide. As heretofore disclosed, plant EG307<br><br><br>
polypeptides of the present invention include, but are not limited to, polypeptides having full-<br>
lerigth plant EG307 coding regions, polypeptides having partial plant EG3Q7 coding regions,<br>
fusion polypeptides, multivalent protective polypeptides and combinations thereof.<br>
At least certain polynucleotides of the present invention encode polypeptides that<br>
selectively bind to immune serum derived from an animal that has been immunized with an<br>
EG307 polypeptide from which the polynucleotide was isolated.<br>
A preferred polynucleotide of the present invention, when expressed in a suitable<br>
plant, is capable of increasing the yield of the plant. As will be disclosed in more detail<br>
below, such a polynucleotide can be, or encode, an antisense RNA, a molecule capable of<br>
triple helix formation, a ribozyme, or other nucleic acid-based compound.<br>
One embodiment of the present invention is a plant EG307 polynucleotide that<br>
hybridizes under stringent hybridization conditions to an EG307 polynucleotide of the present<br>
invention, or to a homologue of such an EG307 polynucleotide, or to the complement of such<br>
a polynucleotide. A polynucleotide complement of any nucleic acid sequence of the present<br>
invention refers to the nucleic acid sequence of the polynucleotide that is complementary to<br>
(i.e., can form a complete double helix with) the strand for which the sequence is cited. It is<br>
to be noted that a double-stranded nucleic acid molecule of the present invention for which a<br>
nucleic acid sequence has been determined for one strand, that is represented by a SEQ ID<br>
NO, also comprises a complementary strand having a sequence that is a complement of that<br>
SEQ ID NO. As such, polynucleotides of the present invention, which can be either double-<br>
stranded or single-stranded, include those polynucleotides that form stable hybrids under<br>
stringent hybridization conditions with either a given SEQ ID NO denoted herein and/or with<br>
the complement of that SEQ ID NO, which may or may not be denoted herein. Methods to<br>
deduce a complementary sequences are known to those skilled in the art. Preferred is an<br>
EG307 polynucleotide that includes a nucleic acid sequence having at least about 65 percent,<br>
preferably at least about 70 percent, more preferably at least about 75 percent, more<br>
preferably at least about 80 percent, more preferably at least about 85 percent, more<br>
preferably at least about 90 percent and even more preferably at least about 95 percent<br>
homology with the corresponding region(s) of the nucleic acid sequence encoding at least a<br>
portion of an EG307 polypeptide. Particularly preferred is an EG307 polynucleotide capable<br>
of encoding at least a portion of an EG307 polypeptide that naturally is present in plants.<br>
Particularly preferred EG307 polynucleotides of the present invention hybridize under<br>
stringent hybridization conditions with at least one of the followingpolynucleotides: SEQ ID<br>
NO:l, SEQ ID NO:91, SEQ ID,NO:2, SEQ ID NO:4, SEQ ID NO:5, SEQ ID NO:7, SEQ ID<br><br><br>
NO:10, SEQ ID N0:11, SEQ ID N0:I2, SEQ ID N0:14, SEQ ID N0:15, SEQ ID NO:17,<br>
SEQ ID N0:18, SEQ ID NO:20, SEQ ID NO:2l, SEQ ID. NO:23', SEQ ID NO:24, SEQ ID<br>
NO:25, SEQ ID NO:27, SEQ ID NO:28, SEQ ID NO:29, SEQ ID NO:90, SEQ ID N0:31,<br>
SEQ ID NO:33, SEQ ID NO:34, SEQ ID. NO:35, SEQ ID NO:37, SEQ ID NO:38, SEQ ID<br>
NO:40, SEQ ID N0:41, SEQ ID NO:42, SEQ ID NO:44, SEQ ID NO:45, SEQ ID NO:46,<br>
SEQ ID NO:47, SEQ ID NO:49, SEQ ID. NO:50, SEQ ID N0:51, SEQ ID NO:53, SEQ ID<br>
NO:54, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:58, SEQ ID NO:60, SEQ ID NO:62,<br>
SEQ ID NO:63, SEQ ID NO:64, SEQ ID NO:66, SEQ ID NO:67, SEQ ID NO:69, SEQ ID.<br>
NO:70, SEQ ID NO:71, SEQ ID NO-.73, SEQ ID NO:74, SEQ ID NO:75, SEQ ID NO:77,<br>
SEQ ID NO:59, SEQ ID NO:78, SEQ ID NO:80, SEQ ID N0:81, SEQ ID NO:82, SEQ ID<br>
NO:84, and/or SEQ ID NO:85, or to a homologue or complement of such polynucleotide.<br>
A preferred polynucleotide of the present invention includes at least a portion of<br>
nucleic acid sequence SEQ ID NO:l, SEQ ID NO:91, SEQ ID. NO:2, SEQ ID NO:4, SEQ ID<br>
NO:5, SEQ ID NO:7, SEQ ID NO:10, SEQ ID NO:l 1, SEQ ID NO:12, SEQ ID NO:14, SEQ<br>
ID NO:15, SEQ ID NO:17, SEQ ID NO:18, SEQ ID NO.20, SEQ ID NO:21, SEQ ID.<br>
NO:23, SEQ ID NO:24, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:28, SEQ ID NO:29,<br>
SEQ ID NO:30, SEQ ID NO:3l, SEQ ID NO:33, SEQ ID NO:34, SEQ ID. N0.35, SEQ ID<br>
NO:37, SEQ ID NO:38, SEQ ID NO:40, SEQ ID NO:41, SEQ ID NO:42, SEQ ID NO:44,<br>
SEQ ID NO:45, SEQ ID NO:46, SEQ ID NO:47, SEQ ID NO:49, SEQ ED. NO:50, SEQ ID<br>
NO:51, SEQ ID NO:53, SEQ ID NO:54, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:58,<br>
SEQ ID NO:60, SEQ ID NO:62, SEQ ID NO:63, SEQ ID NO:64, SEQ ID NO:66, SEQ ID<br>
NO:67, SEQ ID NO:69, SEQ ID. NO:70, SEQ ID NO:71, SEQ ID NO:73, SEQ ID NO:74,<br>
SEQ ID NO:75, SEQ ID NO:77, SEQ ID NO:59, and/or SEQ ID NO:78 that is capable of<br>
hybridizing (i.e., that hybridizes under stringent hybridization conditions) to an O. sativa<br>
EG307 gene, to a O. rufipogon EG307 gene, to a Z. mays mays EG307 gene, to a Z. mays<br>
parviglumis EG307 gene, to a Z. diploperennis EG307 gene and/or to a Z. luxurious EG307<br>
gene of the present invention, as well as a polynucleotide that is an allelic variant of any of<br>
those polynucleotides. Such preferred polynucleotides can include nucleotides in addition to<br>
those included in the SEQ ID NOs, such as, but not limited to, a full-length gene, a full-length<br>
coding region, a polynucleotide encoding a fusion polypeptide, and/or a polynucleotide<br>
encoding a multivalent protective compound.<br>
The present invention also includes polynucleotides encoding a polypeptide including<br>
at least a portion of SEQ ID NO:3, polynucleotides encoding a polypeptide having at least a<br><br>
portion of SEQ ID NO:6, polynucleotides encoding a polypeptide having at least a portion of<br>
SEQ ID NO:9, polynucleotides encoding a polypeptide having at least a portion of SEQ ID<br>
NO:13, polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO:16,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO: 19,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO:22,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO:26,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO:30,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO:36,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO:39,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO:43,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO:48,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO.52,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO:56,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO:61,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO:65,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO:68,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO:72,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO:76,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO:79,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO:83, and/or<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO:86, including<br>
polynucleotides that have been modified to accommodate codon usage properties of the cells<br>
in which such polynucleotides are to be expressed.<br>
Knowing the nucleic acid sequences of certain plant EG307 polynucleotides of the<br>
present invention allows one skilled in the art to, for example, (a) make copies of those<br>
polynucleotides, (b) obtain polynucleotides including at least a portion of such<br>
polynucleotides (e.g., polynucleotides including full-length genes, full-length coding regions,<br>
regulatory control sequences, truncated coding regions), and (c) obtain EG307<br>
polynucleotides for other plants, particularly since, as described in detail in the Examples<br>
section, knowledge of O. sativa EG307 polynucleotides of the present invention enabled the<br>
isolation of O. ruftpogon, lea mays mays, Zea mays parviglumis, Z. diploperennis, and<br>
ZAuxurians EG307 polynucleotides of the present invention. Such polynucleotides can be<br>
obtained in a varietyof ways including screening appropriate expression libraries with<br>
antibodies of the present invention; traditional cloning techniques using oligonucleotide<br><br><br>
probes of the present invention to screen appropriate libraries or DNA; and PCR amplification<br>
of appropriate libraries or DNA using oligonucleotide primers of the present invention.<br>
Preferred libraries to screen or from which to amplify polynucleotides include libraries such<br>
as genomic DNA libraries, BAC libraries, YAC libraries, cDNA libraries prepared from<br>
isolated plant tissues, including, but not limited to, stems, reproductive structures/tissues,<br>
leaves, roots, and tillers; and libraries constructed from pooled cDNAs from any or ail of the<br>
tissues listed above. In the case of rice, BAC libraries, available from Clemson University,<br>
are preferred. Similarly, preferred DNA sources to screen or from which to amplify<br>
polynucleotides include plant genomic DNA. Techniques to clone and amplify genes are<br>
disclosed, for example, in Sambrook et al., ibid, and in Galun &amp; Breiman, TRANSGENIC<br>
PLANTS, Imperial College Press, 1997.<br>
The present invention also includes polynucleotides that are oligonucleotides capable<br>
of hybridizing, under stringent hybridization conditions, with complementary regions of other,<br>
preferably longer, polynucleotides of the present invention such as those comprising plant<br>
EG307 genes or other plant EG307 polynucleotides. Oligonucleotides, of the present<br>
invention can be RN A, DNA, or derivatives of either. The minimal size of such<br>
oligonucleotides is the size required to form a stable hybrid between a given oligonucleotide<br>
and the complementary sequence on another polynucleotide of the present invention.<br>
Minimal size characteristics are disclosed herein. The size of the oligonucleotide must also be<br>
sufficient for the use of the oligonucleotide in accordance with the present invention.<br>
Oligonucleotides of the present invention can be used in a variety of applications including,<br>
but not limited to, as probes to identify additional polynucleotides, as primers to amplify or<br>
extend polynucleotides, as targets for expression analysis, as candidates for targeted<br>
mutagenesis and/or recovery, or in agricultural applications to alter EG307 polypeptide<br>
production or activity. Such agricultural applications include the use of such oligonucleotides<br>
in, for example, antisense-, triplex formation-, ribozyme- and/or RNA drug-based<br>
technologies. The present invention, therefore, includes such oligonucleotides and methods to<br>
enhance economic productivity in a plant by use of one or more of such technologies.<br>
D. EG1U7Polynucleotides<br>
One embodiment of the present invention is an isolated plant polynucleotide that<br>
hybridizes under stringent hybridization conditions with at least one of the following genes:<br>
an O. sativa EG1117 gene, an O. rufipogon EG1117 gene, a Z mays mays EG1117 gene, and<br>
a Z mays parviglumis EG1117 gene. The identifying characteristics of such genes are-<br>
heretofore described. A polynucleotide of the present invention can include an isolated<br><br><br>
natural plant EG1117 gene or a homologue thereof. A polynucleotide of the present invention<br>
can include one or more regulatory regions, full-length or partial coding regions, or<br>
combinations thereof. The minimal size of a polynucleotide of the present invention is the<br>
minimal size that can form a stable hybrid with one of the aforementioned genes under<br>
stringent hybridization conditions. Suitable and preferred plants are disclosed above.<br>
Characteristics of isolated EG1117 genes and homologues thereof are described above in the<br>
section entitled "EG307 polynucleotides."<br>
One embodiment of the present invention is a plant EG1117 polynucleotide that<br>
hybridizes under stringent hybridization conditions to an EG1117 polynucleotide of the<br>
present invention, or to a homologue of such an EG1117 polynucleotide, or to the<br>
complement of such a polynucleotide. Preferred is an EG1117 polynucleotide that includes a<br>
nucleic acid sequence having at least about 65 percent, preferably at least about 70 percent,<br>
more preferably at least about 75 percent, more preferably at least about 80 percent, more<br>
preferably at least about 85 percent, more preferably at least about 90 percent and even more<br>
preferably at least about 95 percent homology with the corresponding region(s) of the nucleic<br>
acid sequence encoding at least a portion of an EG1117 polypeptide. Particularly preferred is<br>
an EG1117 polynucleotide capable of encoding at least a portion of an EG1117 polypeptide<br>
that naturally is present in plants.<br>
Particularly preferred EG1117 polynucleotides of the present invention hybridize<br>
under stringent hybridization conditions with at least one of the following polynucleotides:,<br>
or to a homologue or complement of such polynucleotide.<br>
A preferred polynucleotide of the present invention includes at least a portion of<br>
nucleic acid sequence SEQ ID NO:92, that is capable of hybridizing (i.e., that hybridizes<br>
under stringent hybridization conditions) to an O. sativa EG1117 gene, to a 0. rufipogon<br>
EG1117 gene, to a Z. mays mays EG1117 gene, and/or to a Z mays parviglumis EG1117 gene<br>
of the present invention, as well as a polynucleotide that is an allelic variant of any of those<br>
polynucleotides. Such preferred polynucleotides can include nucleotides in addition to those<br>
included in the SEQ ID NOs, such as, but not limited to, a full-length gene, a full-length<br>
coding region, a polynucleotide encoding a fusion polypeptide, and/or a polynucleotide<br>
encoding a multivalent protective compound.<br>
A preferred polynucleotide of the present invention includes at least a portion of<br>
nucleic acid sequence SEQ ID NO:92, SEQ ID NO:93, SEQ ID NO:94, SEQ ID NO:96, SEQ<br>
ID NO:97, SEQ ID NO:98, SEQ ID NO:100, SEQ ID NO: 101, SEQ ID NO:102, SEQ ID<br>
NO:103, SEQ ID NO:104, SEQ ID NO:104, SEQ ID NO:106, SEQ ID NO:107, SEQ ID<br><br><br>
NO:109, SEQ ID NO:110, SEQ ID NO:l 12, SEQ ID NO:l 13, SEQ ID NO:l 14, SEQ ID NO:116, SEQID NO:117, SEQ EDN0:119, SEQ ID NO:120, SEQ ID NO:121, SEQ ID<br>
NO:122, SEQ ID NO:123, SEQ EDNO:124, SEQ ID NO:125, SEQID NO:127, SEQ ID<br>
NO:128, SEQ ID NO:129, SEQ ID NO:130, SEQ ID NO:131, SEQ ID NO:133, SEQ ID NO: 135, SEQ ID NO: 136, SEQ ID NO: 137, SEQ ID NO: 138, SEQ ID NO: 140, SEQ ID<br>
NO:141, SEQ ID NO:142, SEQ ID NO:144, SEQ ID NO:145, SEQ ID NO:146, SEQ ID<br>
NO:147, SEQ ID NO:149, SEQ ID NO:150, SEQ ID NO:151, SEQ ID NO:152, SEQ ID<br>
NO:154, SEQ ID NO:155, SEQ ID NO:157, SEQ ID NO:158, SEQ EDNO:160, SEQ ID NO:161, SEQ ID NO:162, SEQ ID NO:163, SEQ ID NO:165, SEQ ID NO:166, SEQ ED<br>
N0:167, and/or SEQ ID NO:168, that is capable of hybridizing (i.e., that hybridizes under<br>
stringent hybridization conditions) to an O. sativa EG1117 gene, to a O. ruflpogon EG1117<br>
gene, to a Z mays mays EG1117 gene, and/or to a Z mays parviglumis EG1117 gene, to a<br>
gene of the present invention, as well as a polynucleotide that is an allelic variant of any of<br>
those polynucleotides. Such preferred polynucleotides can include nucleotides in addition to<br>
those included in the SEQ ID NOs, such as, but not limited to, a full-length gene, a full-length<br>
coding region, a polynucleotide encoding a fusion polypeptide, and/or a polynucleotide<br>
encoding a multivalent protective compound.<br>
The present invention also includes polynucleotides encoding a polypeptide including<br>
at least a portion of SEQ ID NO:95, polynucleotides encoding a polypeptide having at least a<br>
portion of SEQ ID NO:99, polynucleotides encoding a polypeptide having at least a portion of<br>
SEQ ID NO: 102, polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO:105, polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO-.108,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO.l 11,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO: 115,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO-.l 18,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO: 121,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO: 126,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO-.l 31,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO: 134,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO: 139,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO: 143,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO: 148,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO: 153,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO: 156,<br><br><br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO: 159,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO: 164,<br>
polynucleotides encoding a polypeptide having at least a portion of SEQ ID NO: 169,<br>
including polynucleotides that have been modified to accommodate codon usage properties of<br>
the cells in which such polynucleotides are to be expressed.<br>
Knowing the nucleic acid sequences of certain plant EG1117 polynucleotides of the<br>
present invention allows one skilled in the art to, for example, (a) make copies of those<br>
polynucleotides, (b) obtain polynucleotides including at least a portion of such<br>
polynucleotides (e.g., polynucleotides including full-length genes, full-length coding regions,<br>
regulatory control sequences, truncated coding regions), and (c) obtain EG117<br>
polynucleotides for other plants, particularly since, as described in detail in the Examples<br>
section, knowledge of O. ruftpogon EG117 polynucleotides of the present invention enabled<br>
the isolation of O. sativa, Zea mays mays, and Zea mays parviglumis EG1117 polynucleotides<br>
of the present invention. Such polynucleotides can be obtained in a variety of ways including<br>
screening appropriate expression libraries with antibodies of the present invention; traditional<br>
cloning techniques using oligonucleotide probes of the present invention to screen appropriate<br>
libraries or DNA; and PCR amplification of appropriate libraries or DNA using<br>
oligonucleotide primers of the present invention. Preferred libraries are described above in<br>
the section entitled "EG307 polynucleotides."<br>
The present invention also includes polynucleotides that are oligonucleotides capable<br>
of hybridizing, under stringent hybridization conditions, with complementary regions of other,<br>
preferably longer, polynucleotides of the present invention such as those comprising plant<br>
EG1117 genes or other plant EG1117 polynucleotides. Oligonucleotides of the present<br>
invention can be KNA, DNA, or derivatives of either. The minimal size of such<br>
oligonucleotides is the size required to form a stable hybrid between a given oligonucleotide<br>
and the complementary sequence on another polynucleotide of the present invention.<br>
Minimal size characteristics are disclosed herein. The size of the oligonucleotide must also be<br>
sufficient for the use of the oligonucleotide in accordance with the present invention. Such<br>
applications are described above in the section entitled "EG307 polynucleotides."<br>
E. Recombinant molecules<br>
The present invention also includes a recombinant vector, which includes at least one<br>
plant EG307 or EG1117 polynucleotide of the present invention, inserted into any vector<br>
capable of delivering the polynucleotide into a host cell. Such a vector contains heterologous<br>
nucleic acid sequences, that is nucleic acid sequences that are not naturally found adjacent to<br><br><br>
polynucleotides of the present invention and that are derived from a species other than the<br>
species from which the polynucleotide(s) are derived. As used herein, a derived<br>
polynucleotide is one that is identical or similar in sequence to a polynucleotide or portion of<br>
a polynucleotide, but can contain modifications, such as modified bases, backbone<br>
modifications, nucleotide changes, and the like. The vector can be either RNA or DNA,<br>
either prokaryotic or eukaryotic, and typically is a virus or a plasmid. Recombinant vectors<br>
can be used in the cloning, sequencing, and/or otherwise manipulating of plant EG307 or<br>
EG1117 polynucleotides of the present invention. One type of recombinant vector, referred to<br>
herein as a recombinant molecule and described in more detail below, can be used in the<br>
expression of polynucleotides of the present invention. Preferred recombinant vectors are<br>
capable of replicating in the transformed cell.<br>
Suitable and preferred polynucleotides to include in recombinant vectors of the present<br>
invention are as disclosed herein for suitable and preferred plant EG307 or EG1117<br>
polynucleotides per se. Particularly preferred polynucleotides to include in recombinant<br>
vectors, and particularly in recombinant molecules, of the present invention include SEQ ID<br>
NO: 1, SEQ ID NO:91, SEQ ID. NO:2, SEQ ID NO:4, SEQ ID NO:5, SEQ ID NO:7, SEQ ID<br>
NO:10, SEQ ID NO:ll, SEQ ID NO:12, SEQ ID NO:14, SEQ ID NO:15, SEQ ID NO:17,<br>
SEQ ID NO:18, SEQ ID NO:20, SEQ ID NO:21, SEQ ID. NO:23, SEQ ID NO:24, SEQ ID<br>
NO:25, SEQ ID NO:27, SEQ ID NO:28, SEQ ID NO:29, SEQ ID NO:30, SEQ ID NO:31,<br>
SEQ ID NO:33, SEQ ID NO:34, SEQ ID. NO:35, SEQ ID NO:37, SEQ ID NO:38, SEQ ID<br>
NO:40, SEQ ID NO:41, SEQ ID NO:42, SEQ ID NO:44, SEQ ID NO:45, SEQ ID NO:46,<br>
SEQ ID NO:47, SEQ ID NO-.49, SEQ ID. NO:50, SEQ ID NO:51, SEQ ID NO:53, SEQ ID<br>
NO:54, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:58, SEQ ID NO:60, SEQ ID NO:62,<br>
SEQ ID NO:63, SEQ ID NO:64, SEQ ID NO:66, SEQ ID NO:67, SEQ ID NO:69, SEQ ID.<br>
NO:70, SEQ ID NO:71, SEQ ID NO:73, SEQ ID NO:74, SEQ ID NO:75, SEQ ID NO:77,<br>
SEQ ID NO:59, and/or SEQ ID NO:78,. Alternative preferred polynucleotides to include in<br>
recombinant vectors, and particularly in recombinant molecules, of the present invention<br>
include SEQ ID NO:92, SEQ ID NO:93, SEQ ID NO:94, SEQ ID NO:96, SEQ ID NO:97,<br>
SEQ ID NO:98, SEQ ID NO:100, SEQ ID NO:101, SEQ ID NO:102, SEQ ID NO:103, SEQ<br>
ID NO:104, SEQ ID NO:104, SEQ ID NO:106, SEQ ID NO:107, SEQ ID NO:109, SEQ ID<br>
NO:l 10, SEQ ID NO:l 12, SEQ ID NO:113, SEQ ID NO:l 14, SEQ ID NO:l 16, SEQ ID<br>
NO:l 17, SEQ ID NO:119, SEQ ID NO:120, SEQ ID NO-.121, SEQ ID NO:122, SEQ ID<br>
NO:123, SEQ ID NO:124, SEQ ID N0.125, SEQ ID NO:127, SEQ ID NO:128, SEQ ID<br>
NO:129, SEQ ID NO.130, SEQ ID NO:131, SEQ ID NO:133, SEQ ID NO:135, SEQ ID<br><br><br>
NO:136, SEQ ID NO:137, SEQ ID NO:138, SEQ ID NO:140, SEQ ID NO:141, SEQ ID<br>
NO:142, SEQ ID NO:144, SEQ ID NO:145, SEQ ID N0:146, SEQ ID NT0:147, SEQ ID<br>
NO:149, SEQ ID NO:150, SEQ ID N0:151, SEQ ID NO:152, SEQ ID NO:154, SEQ ID<br>
NO:155, SEQ ID NO:157, SEQ ID NO:158, SEQ ID NO:160, SEQ ID NO:161, SEQ ID<br>
NO:162, SEQ ID N0.163, SEQ ID NO:165, SEQ ID NO:166, SEQ ID NO:167, and/or SEQ<br>
ID NO:168.<br>
Isolated plant EG307 or EG1117 polypeptides of the present invention can be<br>
produced in a variety of ways, including production and recovery of natural polypeptideS<br>
production and recovery of recombinant polypeptides, and chemical synthesis of the<br>
polypeptides. In one embodiment, an isolated polypeptide of the present invention is produced<br>
by culturing a cell capable of expressing the polypeptide under conditions effective to produce<br>
the polypeptide, and recovering the polypeptide. A preferred cell to culture is a recombinant<br>
cell that is capable of expressing the polypeptide, the recombinant cell being produced by<br>
transforming a host cell with one or more polynucleotides of the present invention.<br>
Transformation of a polynucleotide into a cell can be accomplished by any method by which a<br>
polynucleotide can be inserted into the cell. Transformation techniques include, but axe not<br>
limited to, transfection, electroporation, microinjection, lipofection, adsorption, and protoplast<br>
fusion. A recombinant cell may remain unicellular or may grow into a tissue, organ or a<br>
multicellular organism. Transformed polynucleotides of the present invention can remain<br>
extrachromosomal or can integrate into one or more sites within a chxormosome of the<br>
transformed (i.e., recombinant) cell in such a manner that their ability to be expressed is<br>
retained. Suitable and preferred polynucleotides with which to transform a cell are as<br>
disclosed herein for suitable and preferred plant EG307 or EG1117 polynucleotides per se.<br>
Particularly preferred polynucleotides to include in recombinant cells of the present invention<br>
include SEQ ID NO:l, SEQ ID NO:91, SEQ ID. NO:2, SEQ ID NO:4, SEQ ID NO:5, SEQ<br>
ID NO:7, SEQ ID NO:10, SEQ ID NO:11, SEQ ID NO:l2, SEQ ID NO:14, SEQ ID NO:l 5,<br>
SEQ ID NO: 17, SEQ ID NO: 18, SEQ ID NO:20, SEQ ID NO:21, SEQ ID. NO:23, SEQ ID<br>
NO:24, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:28, SEQ ID NO:29, SEQ ID N0:30,<br>
SEQ ID N0:3l, SEQ ID NO:33, SEQ ID NO:34, SEQ ID. NO:35, SEQ ID NO:37, SEQ ID<br>
NO:38, SEQ ID NO:40, SEQ ID NO:41, SEQ ID NO:42, SEQ ID NO:44, SEQ ID N0:45,<br>
SEQ ID NO:46, SEQ ID NO:47, SEQ ID NO:49, SEQ ID. NO-.50, SEQ ID NO:51, SEQ ID<br>
N0:53, SEQ ID N0:54, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:58, SEQ ID NO:60,<br>
SEQ ID NO:62, SEQ ID N0:63, SEQ ID NO:64, SEQ ID NO:66, SEQ ID N0:67, SEQ ID<br>
N0:69, SEQ ID. NO:70, SEQ ID NO:71, SEQ ID NO:73, SEQ ID N0:74, SEQ ID NO:75,<br><br><br>
SEQ ID NO:77, SEQ ID NO:59, SEQ ID NO:78, SEQ ID NO:80, SEQ ID NQ:81, SEQ ID<br>
NO:82, SEQ ID NO:84, and/or SEQ ID NO:85. Alternative preferred polynucleotides to<br>
include in recombinant cells of the present invention include SEQ ID NO:92, SEQ ID NO:93,<br>
SEQ ID NO:94, SEQ ID NO:96, SEQ ID NO:97, SEQ ID NO:98, SEQ ID NO: 100, SEQ ID<br>
NO: 101, SEQ ID NO: 102, SEQ EO NO: 103, SEQ DO NO: 104, SEQ ID NO:l04, SEQ ID<br>
NO.106, SEQ ID NO:107, SEQ ID NO:109, SEQ ID NO:l 10, SEQ ID NO:l 12, SEQ ID<br>
NO:113, SEQ ID NO:114, SEQ ID NO:116, SEQ ID NO:117, SEQ ID NO:119, SEQ ID<br>
NO:120, SEQ ID NO:121, SEQ ID NO:122, SEQ ID NO:123, SEQ ID NO:124, SEQ ID<br>
NO:125, SEQ ID NO:127, SEQ ID NO: 128, SEQ ID NO: 129, SEQ ID NO: 130, SEQ ID<br>
NO:131, SEQ ID NO:133, SEQ ID NO-.135, SEQ ID NO:136, SEQ ID NO:137, SEQ ID<br>
NQ;138, SEQ ID NO:140, SEQ ID N0.141, SEQ ID NO:142, SEQ ID NO:144, SEQ ID<br>
NO:145, SEQ ID NO:146, SEQ ID NO:147, SEQ ID NO:149, SEQ ID NO:150, SEQ ID<br>
NO:151, SEQ ID NO:152, SEQ ID NO:154, SEQ ID NO:155, SEQ ID NO:157, SEQ ID<br>
NO:158, SEQ ID NO:160, SEQ ID NO:161, SEQ ID NO:162, SEQ ID NO:163, SEQ ID<br>
NO:165, SEQ ID N0.166, SEQ ID NO:167, and/or SEQ ID NO:168.<br>
Suitable host cells to transform include any cell that can be transformed with a<br>
polynucleotide of the present invention. Host cells can be either untransformed cells or cells<br>
that are already transformed with at least one polynucleotide. Host cells of the present<br>
invention either can be endogenously (i.e., naturally) capable of producing plant EG307 or<br>
EG1117 polypeptides of the present invention or can be capable of producing such<br>
polypeptides after being transformed with at least one polynucleotide of the present invention.<br>
Host cells of the present invention can be any cell capable of producing at least one<br>
polypeptide of the present invention, and include bacterial, fungal (including yeast and rice<br>
blast, Magnaporthe grisea), parasite (including nematodes, especially of the genera<br>
Xiphinema, Helicotylenchus, and Tylenchlohynchus), insect, other animal and plant cells.<br>
Suitable host viruses to transform include any virus that can be transformed with a<br>
polynucleotide of the present invention, including, but not limited to, rice stripe virus, and<br>
echinochloa hoja blanca virus.<br>
In a preferred embodiment, non-pathogenic symbiotic bacteria, which are able to live<br>
and replicate within plant tissues, so-called endophytes, or non-pathogenic symbiotic bacteria,<br>
which are capable of colonizing the phyllosphere or the rhizosphere, so-called epiphytes, are<br>
used. Such bacteria include bacteria of the genera Agrobacterium, Alcaligenes, Azospirillum,<br>
Azotobacter, Bacillus, Clavibacter, Enterobacter, Erwinia, Flavobacter, Klebsiella,<br>
Pseudomonas, Rhizobium, Serratia, Streptomyces mdXanthomonas, Symbiotic fungi, such as<br><br><br>
Trichoderma and Gliocladium are also possible hosts for expression of the inventive<br>
nucleotide sequences for the same purpose.<br>
A recombinant cell is preferably produced by transforming a host cell with one or<br>
more recombinant molecules, each comprising one or more polynucleotides of the present<br>
invention operatively linked to an expression vector containing one or more transcription<br>
control sequences. The phrase "operatively linked" refers to insertion of a polynucleotide into<br>
an expression vector in a manner such that the molecule is able to be expressed in the correct<br>
reading frame when transformed into a host cell. As used herein, an expression vector is a<br>
DNA or RNA vector that is capable of transforming a host cell and of effecting expression of<br>
a specified polynucleotide. Preferably, the expression vector is also capable of replicating<br>
within the host cell. Expression vectors can be either prokaryotic or eukaryotic, and are<br>
typically viruses or plasmids. Expression vectors of the present invention include any vectors<br>
that function (i.e., direct gene expression) in recombinant cells of the present invention,<br>
including in bacterial, fungal, parasite, insect, other animal, and plant cells. Preferred<br>
expression vectors of the present invention can direct gene expression in bacterial, yeast,<br>
fungal, insect and mammalian cells and more preferably in the cell types heretofore disclosed.<br>
Recombinant molecules of the present invention may also (a) contain secretory signals<br>
(i.e., signal segment nucleic acid sequences) to enable an expressed EG307 or EG1117<br>
polypeptide of the present invention to be secreted from the cell that produces the polypeptide<br>
and/or (b) contain fusion sequences which lead to the expression of polynucleotides of the<br>
present invention as fusion polypeptides. Examples of suitable signal segments and fusion<br>
segments encoded by fusion segment nucleic acids are disclosed herein. Eukaryotic<br>
recombinant molecules may include intervening and/or untranslated sequences surrounding<br>
and/or within the nucleic acid sequences of polynucleotides of the present invention. Suitable<br>
signal segments include natural signal segments or any heterologous signal segment capable<br>
of directing the secretion of a polypeptide of the present invention. Preferred signal and<br>
fusion sequences employed to enhance organ and organelle specific expression include, but<br>
are not limited to, arcelin-5, see Goossens, A. et. al. The arcelin-5 Gene of Phaseolus vulgaris<br>
directs high seed-specific expression jn transgenic Phaseolus acutifolius and Arabidopsis<br>
plants. Plant Physiology (1999) 120:1095-1104, phaseolin, see Sengupta-Gopalan, C. et. al.<br>
Developmentally regulated expression of the bean beta-phaseolin gene in tobacco seeds.<br>
PNAS (1985) 82:3320-3324, hydroxyproline-rich glycoprotein, serpin, see Yan, X. et. al.<br>
Gene fusions of signal sequences with a modified beta-glucuronidase gene results in retention<br>
of the beta-glucuronidase protein in the secretory pathway/plasma membrane. Plant<br><br><br>
Physiology (1997) 115:915-924, N-acetyl glucosaminyl transferase 1, see Essl, D. et al. The<br>
N-terminal 77 amino acids from tobacco N-acetylgJucosaminyltransferase I are sufficient to<br>
retain reporter protein in the Golgi apparatus of Nicotiana benthamiana cells. Febs Letters<br>
(1999) 453(l-2):169-73, albumin, see Vandekerckhove, J. et. al. Enkephalins produced in<br>
transgenic plants using modified 2S seed storage proteins. BioTechnology 7:929-932 (1989)<br>
and PR1, see Pen, J. et. al. Efficient production of active industrial enzymes in plants.<br>
Industrial Crops and Prod. (1993) 1:241-250.<br>
Polynucleotides of the present invention can be operatively linked to expression<br>
vectors containing regulatory sequences such as transcription control sequences, translation<br>
control sequences, origins of replication, and other regulatory sequences that are compatible<br>
with the recombinant cell and that control the expression of polynucleotides of the present<br>
invention. In particular, recombinant molecules of the present invention include transcription<br>
control sequences. Transcription control sequences are sequences which control the initiation,<br>
elongation, and termination of transcription. Included are those transcription control<br>
sequences which are sufficient to render promoter-dependent gene expression controllable for<br>
cell-type specific, tissue-specific or inducible by external signals or agents; such elements<br>
may be located in the 5' or 3' regions of the native gene. Particularly important transcription<br>
control sequences are those which control transcription initiation, such as promoter, enhancer,<br>
operator and repressor sequences. Suitable transcription control sequences include any<br>
transcription control sequence that can function in at least one of the recombinant cells of the<br>
present invention. A variety of such transcription control sequences are known to those<br>
skilled in the art. Preferred transcription control sequences include those which function in<br>
bacterial, yeast, fungal, insect and mammalian cells, such as, but not limited to, tac, lac, tip,<br>
tec, oxy-prOj omp/lpp, rrnB, bacteriophage lambda (X) (such as %&amp;. and \pn and fusions that<br>
include such promoters), bacteriophage T7, T71ac, bacteriophage T3, bacteriophage SP6,<br>
bacteriophage SP01, metallothionein, a-mating factor, Pichia alcohol oxidase, alphavirus<br>
subgenomic promoters (such as Sindbis virus subgenomic promoters), antibiotic resistance<br>
gene, baculovirus, Heliothis zea insect virus, vaccinia virus, herpesvirus, poxvirus,<br>
adenovirus, cytomegalovirus (such as intermediate early promoters, simian virus 40,<br>
retrovirus, actin, retroviral long terminal repeat, Rous sarcoma virus, heat shock, phosphate<br>
and nitrate transcription control sequences as well as other sequences capable of controlling<br>
gene expression in prokaryotic or eukaryotic cells.<br>
Particularly preferred transcription control sequences are plant transcription control<br>
sequences. The choice of transcription control sequence will vary depending on the temporal<br><br><br>
and spatial requirements for expression, and also depending on the target species. Thus,<br>
expression of the nucleotide sequences of this invention in any plant organ (leaves, roots,<br>
seedlings, immature or mature reproductive structures, etc.) or at any stage of plant<br>
development is preferred. Although many transcription control sequences from dicotyledons<br>
have been shown to be operational in monocotyledons and vice versa, ideally dicotyledonous<br>
transcription control sequences are selected for expression in dicotyledons, and<br>
monocotyledonous promoters for expression in monocotyledons. However, there is no<br>
restriction to the provenance of selected transcription control sequences; it is sufficient that<br>
they are operational in driving the expression of the nucleotide sequences in the desired cell.<br>
Preferred transcription control sequences that are expressed constitutively include but<br>
are not limited to promoters from genes encoding actin or ubiquitin and the CaMV 3SS and<br>
19S promoters. The nucleotide sequences of this invention can also be expressed under the<br>
regulation of promoters that are chemically regulated. This enables the EG307 or EG1117<br>
polypeptide to be synthesized only when the crop plants are treated with the inducing<br>
chemicals. Preferred technology for chemical induction of gene expression is detailed in the<br>
published application EP 0 332 104 (to Ciba-Geigy) and U.S. Pat. No. 5,614,395. A preferred<br>
promoter for chemical induction is the tobacco PR-1 a promoter.<br>
A preferred category of promoters is that which is induced by the physiological state<br>
of the plant (i.e. wound inducible, water-stress inducible, salt-stress inducible, disease<br>
inducible, and the like). Numerous promoters have been described which are expressed at<br>
wound sites and also at the sites of phytopathogen infection. Ideally, such a promoter should<br>
only be active locally at the sites of infection, and in mis way the EG307 or EG1117<br>
polypeptides only accumulate in cells in which the accumulation is desired. Preferred<br>
promoters of this kind include those described by Stanford et al. Mol. Gen. Genet 215: 200-<br>
208 (1989), Xu et al. Plant Mqlec. Biol. 22: 573-588 (1993), Logemann et al. Plant Cell 1:<br>
151-158 (1989), Rohrmeier &amp; Lehle, Plant Molec. Biol. 22:783-792 (1993), Firek et al. Plant<br>
Molec. Biol. 22:129-142 (1993), and Warner et al. Plant J. 3:191-201 (1993).<br>
Preferred tissue-specific expression patterns include but are not limited to green tissue<br>
specific, root specific, stem specific, and flower specific. Promoters suitable for expression in<br>
green tissue include many which regulate genes involved in photosynthesis and many of these<br>
have been cloned from both monocotyledons and dicotyledons. A preferred promoter is the<br>
maize PEPC promoter from the phosphoenol carboxylase gene (Hudspeth &amp; Grula, Plant<br>
Molec. Biol. 12: 579-589 (1989)). A preferred promoter for root specific expression is that<br>
described by de Framond (FEBS 290:103-106 (1991); EP 0 452 269 to Ciba-Geigy). A<br><br><br>
preferred stem specific promoter is that described in U.S. Pat. No. 5,625,136 (to Ciba-Geigy)<br>
and which drives expression of the maize trpA gene.<br>
A recombinant molecule of the present invention is a molecule that can include at least<br>
one of any polynucleotide heretofore described operatively linked to at least one of any<br>
transcription control sequence capable of effectively regulating expression of the<br>
polynucleotide(s) in the cell to be transformed, examples of which are disclosed herein.<br>
A recombinant cell of the present invention includes any cell transformed with at least<br>
one of any polynucleotide of the present invention. Suitable and preferred polynucleotides as<br>
well as suitable and preferred recombinant molecules with which to transfer cells are<br>
disclosed herein.<br>
Recombinant cells of the present invention can also be co-transformed with one or<br>
more recombinant molecules including plant EG307 or EG1117 polynucleotides encoding<br>
one or more polypeptides of the present invention and one or more other polypeptides useful<br>
when expressed in plants.<br>
It may be appreciated by one skilled in the art that use of recombinant DNA<br>
technologies can improve expression of transformed polynucleotides by manipulating, for<br>
example, the number of copies of the polynucleotides within a host cell, theefficiency with<br>
which those polynucleotides are transcribed, the efficiency with which the resultant<br>
transcripts are translated, and the efficiency of post-translational modifications. Recombinant<br>
techniques useful for increasing the expression of polynucleotides of the present invention<br>
include, but are not limited to, operatively linking polynucleotides to high-copy number<br>
plasmids, integration of the polynucleotides into one or more host cell chromosomes, addition<br>
of vector stability sequences to plasmids, substitutions or modifications of transcription<br>
control signals (e.g., promoters, operators, enhancers), substitutions or modifications of<br>
translational control signals (e.g., ribosome binding sites, Sbine-Dalgamo sequences),<br>
modification of polynucleotides of the present invention to correspond to the codon usage of<br>
the host cell, deletion of sequences that destabilize transcripts, and use of control signals that<br>
temporally separate recombinant cell growth from recombinant enzyme production during<br>
fermentation. The activity of an expressed recombinant polypeptide of the present invention<br>
may be improved by fragmenting, modifying, or derivatizing polynucleotides encoding such a<br>
polypeptide.<br>
Recombinant cells of the present invention can be used to produce one or more<br>
polypeptides of the present invention by culturing such cells under conditions effective to<br>
produce such a polypeptide, and recovering the polypeptide. Effective conditions to produce<br><br><br>
a polypeptide include, but are not limited to, appropriate media, bioreactor, temperature, pH<br>
and oxygen conditions that permit polypeptide production. An appropriate, or effective,<br>
medium refers to any medium in which a cell of the present invention, when cultured, is<br>
capable of producing an EG307 or EG1117 polypeptide of the present invention. Such a<br>
medium is typically an aqueous medium comprising assimilable carbon, nitrogen and<br>
phosphate sources, as well as appropriate salts, minerals, metals and other nutrients, such as<br>
vitamins. The medium may comprise complex nutrients or may be a defined minimal<br>
medium. Cells of the present invention can be cultured in conventional fermentation<br>
bioreactors, which include, but are not limited to, batch, fed-batch, cell recycle, and<br>
continuous fermentors. Culturing can also be conducted in shake flasks, test tubes, microtiter<br>
dishes, and petri plates. Culturing is carried out at a temperature, pH and oxygen content<br>
appropriate for the recombinant cell. Such culturing conditions are well within the expertise<br>
of one of ordinary skill in the art.<br>
Depending on the vector and host system used for production, resultant polypeptides<br>
of the present invention may either remain within the recombinant ceil; be secreted into the<br>
fermentation medium; be secreted into a space between two cellular membranes, such as the<br>
periplasmic space in E. coli; or be retained on the outer surface of a cell or viral membrane.<br>
The phrase "recovering the polypeptide" refers simply to collecting the whole<br>
fermentation medium containing the polypeptide and need not imply additional steps of<br>
separation or purification. Polypeptides of the present invention can be purified using a<br>
variety of standard polypeptide purification techniques, such as, but not limited to, affinity<br>
chromatography, ion exchange chromatography, filtration, electrophoresis, hydrophobic<br>
interaction chromatography, gel filtration chromatography, reverse phase chromatography,<br>
concanavalin A chromatography, chromatofocusing and differential solubilization.<br>
Polypeptides of the present invention are preferably retrieved in "substantially pure" form. As<br>
used herein, "substantially pure" refers to a purity that allows for the effective use of the<br>
polypeptide as a diagnostic or test compound, and means, with increasing preference, at least<br>
50%, 60%, 70%, 80%, 90%, 95%, or 98% homogeneous.<br>
F. Trattsfected plant cells and transgenic plants<br>
With regard to EG307 and EG1117, particularly preferred recombinant cells are plant<br>
cells. By "plant cell" is meant any self-propagating cell bounded by a semi-permeable<br>
membrane and containing a plastid. Such a cell also requires a cell wall if further propagation<br>
is desired. Plant cell, as used herein includes, without limitation, algae, cyanobacteria, seeds,<br><br><br>
suspension cultures, embryos, meristematic regions, callus tissue, leaves, roots, shoots,<br>
gametophytes, sporophytes, pollen, and microspores.<br>
In a particularly preferred embodiment, at least one (or both) of the EG307 or EG1117<br>
polypeptides or an allele or mutant formthereof, of the invention is expressed in a higher<br>
organism, e.g., a plant. In this case, transgenic plants expressing effective amounts of the<br>
polypeptides exhibit improved economic productivity. A nucleotide sequence of the present<br>
invention is inserted into an expression cassette, which is then preferably stably integrated in<br>
the genome of said plant. In another preferred embodiment, the nucleotide sequence is<br>
included in a non-pathogenic self-replicating virus. Plants transformed in accordance with the<br>
present invention may be monocots or dicots and include, but are not limited to, maize, wheat,<br>
barley, rye, millet, chickpea, lentil, flax, olive, fig almond, pistachio, walnut, beet, parsnip,<br>
citrus fruits, including, but not limited to, orange, lemon, lime, grapefruit, tangerine,<br>
minneola, and tangelo, sweet potato, bean, pea, chicory, lettuce, cabbage, cauliflower,<br>
broccoli, turnip, radish, spinach, asparagus, onion, garlic, pepper, celery, squash, pumpkin,<br>
hemp, zucchini, apple, pear, quince, melon, plum, cherry, peach, nectarine, apricot,<br>
strawberry, grape, raspberry, blackberry, pineapple, avocado, papaya, mango, banana,<br>
soybean, tomato, sorghum, sugarcane, sugarbeet, sunflower, rapeseed, clover, tobacco, carrot,<br>
cotton, alfalfa, rice, potato, eggplant, cucumber, Arabidopsis, and woody plants such as<br>
coniferous and deciduous trees.<br>
Once a desired nucleotide sequence has been transformed into a particular plant<br>
species, it may be propagated in that species or moved into other varieties of the same species,<br>
particularly including commercial varieties, using traditional breeding techniques.<br>
Accordingly, the present invention provides a method for producing a transfected<br>
plant cell or transgenic plant comprising the steps of a) transfecting a plant cell to contain a<br>
heterologous DNA segment encoding a protein and derived from an EG307 and/or EG1117<br>
polynucleotide not native to said cell (the polynucleotide indeed could be native but the<br>
expression pattern could be developmental^ altered, still leading to the preferred effect);<br>
wherein said polynucleotide is operably linked to a promoter that can be used effectively for<br>
expression of transgenic proteins; b) optionally growing and maintaining said cell under<br>
conditions whereby a transgenic plant is regenerated therefrom; c) optionally growing said<br>
transgenic plant under conditions whereby said DNA is expressed, whereby the total amount<br>
of EG307 and/or EG1117 polypeptide in said plant is altered. In a preferred embodiment, the<br>
method further comprises the step of obtaining and growing additional generations of<br>
descendants of said transgenic plant which comprise said heterologous DNA segment wherein<br><br><br>
said heterologous DNA segment is expressed. As used herein, "heterologous DNA", or in<br>
some cases, "transgene" refers to foreign genesor polynucleotides, or additional, or modified<br>
versions of native or endogenous genes or polynucleotides (perhaps driven by different<br>
promoters) in order to alter the traits of a plant in a specific manner.<br>
The invention also provides plant cells which comprise heterologous DNA encoding<br>
an EG307 and/or EG 1117 polypeptide. In a preferred embodiment, the transgenic plant cell is<br>
a propagation material of a transgenic plant. The present invention also provides a transfected<br>
host cell comprising a host cell transfected with a construct comprising a promoter, enhancer<br>
or intron polynucleotide from an evolutionarily significant EG307 and/or EG1117<br>
polynucleotide, and a polynucleotide encoding a reporter protein.<br>
The present invention also provides a method of providing improved economic<br>
productivity in a plant comprising: a) producing a transfected plant cell having a transgene<br>
encoding an EG307 and/or EG1117 polypeptide whereby EG307 and/or EG1117 expression<br>
in said plant cell is altered; and b) growing a transgenic plant from the transfected plant cell<br>
wherein the EG307 and/or EG1117 transgene is expressed in the transgenic plant. The<br>
expression of the transgene includes an increase in EG307 and/or EG1117 expression. In<br>
some embodiments, the expression of the transgene produces an RKA that may interfere with<br>
a native EG307 and/or EG1117 gene such that the expression of the native gene is either<br>
eliminated or reduced, resulting in a useful outcome.<br>
The invention also provides a transgenic plant containing heterologous DNA which<br>
encodes an EG307 and/or EG1117 polypeptide that is expressed in plant tissue, including<br>
expression in a vector introduced into the plant.<br>
The present invention also provides an isolated polynucleotide which includes a<br>
transcription control element operably linked to a polynucleotide that encodes the EG307<br>
and/or EG1117 gene in plant tissue. In preferred embodiment, the transcription control<br>
element is the promoter native to an EG307 and/or EG1117 gene.<br>
The present invention also provides a method of making a transfected cell comprising<br>
a) identifying an evolutionarily significant EG307 and/or EG1117 polynucleotide in a<br>
domesticated plant; b) using said EG307 and/or EG 1117 polynucleotide to identify a non-<br>
polypeptide coding sequence that may be a transcription or translation regulatory element,<br>
enhancer, intron or other 5' or 3' flanking sequence; c) assembling a construct comprising said<br>
non-polypeptide coding sequence and a polynucleotide encoding a reporter protein; and d)<br>
transfecting said construct into a host cell. The present invention also provides a transfected<br>
cell produced according to this method. In one embodiment, the host cell is a plant cell, and<br><br><br>
the method further comprises the step of growing and maintaining the cell under conditions ;<br>
suitable for regenerating a transgenic plant. Also provided is a transgenic plant produced by '<br>
the method.<br>
A nucleotide sequence of this invention is preferably expressed in transgenic plants,<br>
thus causing the biosynthesis of the corresponding EG307 and/or EG1117 polypeptide in the<br>
transgenic plants. In this way, transgenic plants with characteristics related to improved<br>
economic productivity are generated. For their expression in transgenic plants, the nucleotide<br>
sequences of the invention may require modification and optimization. Although preferred<br>
gene sequences may be adequately expressed in both monocotyledonous and dicotyledonous<br>
. plant species, sequences can be modified to account for the specific codon preferences and<br>
GC content preferences of monocotyledons or dicotyledons as these preferences have been<br>
shown to differ (Murray et al. Nucl. Acids Res. 17.477-498 (1989)). All changes required to<br>
be made within the nucleotide sequences such as those described above are made using well<br>
known techniques of site directed mutagenesis, PCR, and synthetic gene construction using<br>
the methods described in the published patent applications EP 0 385 962 (to Monsanto), EP 0<br>
359 472 (to Lubrizol), and WO 93/07278 (to Ciba-Geigy).<br>
For efficient initiation of translation, sequences adjacent to the initiating methionine<br>
may require modification. For example, they can be modified by the inclusion of sequences<br>
known to be effective in plants. Joshi has suggested an appropriate consensus for plants (NAR<br>
15:6643-6653 (1987)) and Clontech suggests a further consensus translation initiator<br>
(1993/1994 catalog, page 210). These consensuses are suitable for use with the nucleotide<br>
sequences of this invention. The sequences are incorporated into constructions comprising<br>
the nucleotide sequences, up to and including the ATG (while leaving the second amino acid<br>
unmodified), or alternatively up to and including the GTC subsequent to the ATG (with the<br>
possibility of modifying the second amino acid of the transgene).<br>
Expression of the nucleotide sequences in transgenic plants is driven by transcription<br>
control elements shown to be functional in plants. Transformation of plants with a<br>
polynucleotide under the control of these regulatory elements provides for controlled<br>
expression in the transformed plant Such transcription control elements have been described<br>
above. In addition to the selection of a suitable initiator of transcription, constructions for<br>
expression of EG307 and/or EG1117 polypeptide in plants require an appropriate<br>
transcription terminator to be attached downstream of the heterologous nucleotide sequence.<br>
Several such terminators are available and known in the art (e.g. tail from CaMV, E9 from<br><br><br>
rbcS). Any available terminator known to function in plants can be used in the context of this<br>
invention.<br>
Numerous other sequences can be incorporated into expression cassettes described in<br>
this invention. These include sequences which have been shown to enhance expression such<br>
as intron sequences (e.g. from Adhl and bronzel) and viral leader sequences (e.g. from TMV,<br>
MCMV and AMV).<br>
The present invention also provides a method of increasing yield in a plant comprising<br>
a) producing a transgenic plant cell having a transgene encoding an BG307 and/or EG1117<br>
polypeptide and the transgene is under the control of regulatory sequences suitable for<br>
controlled expression of the gene(s); and b) growing a transgenic plant from the transgenic<br>
plant cell wherein the EG307 and/or EG1117 transgene is expressed in the transgenic plant.<br>
The present invention also provides a method of increasing yield in a plant comprising<br>
a) producing a transfected plant cell having a transgene containing the EG307 and/or EG 1117<br>
gene under the control of a promoter providing constitutive expression of the EG307 and/or<br>
EG1117 gene; and b) growing a transgenic plant from the transgenic plant cell wherein the<br>
EG307 and/or EG1117 transgene is expressed constitutively in the transgenic plant<br>
The present invention also provides a method of providing controllable yield in a<br>
transgenic plant comprising: a) producing a transfected plant cell having a transgene<br>
containing the EG307 and/or EG1117 gene under the control of a promoter providing<br>
controllable expression of the EG307 and/or EG1117 gene; and b) growing a transgenic plant<br>
from the transgenic plant cell wherein the EG307 and/or EG1117 transgene is controllably<br>
expressed in the transgenic plant. In one embodiment, the EG307 and/or EG1117 gene is<br>
expressed using a tissue-specific or cell type-specific promoter, or by a promoter that is<br>
activated by the introduction of an external signal or agent, such as a chemical signal or agent.<br>
It may be preferable to target expression of the nucleotide sequences of the present<br>
invention to different cellular localizations in the plant In some cases, localization in the<br>
cytosol may be desirable, whereas in other cases, localization in some subcellular organelle<br>
may be preferred. Subcellular localization of heterologous DNA encoded polypeptides is<br>
undertaken using techniques well known in the art. Typically, the DNA encoding the target<br>
peptide from a known organelle-targeted gene product is manipulated and fused upstream of<br>
the nucleotide sequence. Many such target sequences are known for the chloroplast and their<br>
functioning in heterologous constructions has been shown. The expression of the nucleotide<br>
sequences of the present invention is also targeted to the endoplasmic reticulum or to the<br>
vacuoles of the host cells. Techniques to achieve this are well-known in the art.<br><br><br>
Vectors suitable for plant transformation are described elsewhere in this specification.<br>
For Agrobacterium-mcdiated transformation, binary vectors or vectors carrying at least one T-<br>
DNA border sequence are suitable, whereas for direct gene transfer any vector is suitable and<br>
linear DNA containing only the construction of interest may be preferred. In the case of<br>
direct gene transfer, transformation with a single DNA species or co-transformation can be<br>
used (Scbocher et al. Biotechnology 4:1093-1096 (1986)). For both direct gene transfer and<br>
Agrobacterium-mediaXed transfer, transformation is usually (but not necessarily) undertaken<br>
with a selectable marker which may provide resistance to an antibiotic (kanamycin,<br>
hygromycin or methotrexate) or a herbicide (basta). The choice of selectable marker is not,<br>
however, critical to the invention.<br>
In another preferred embodiment, a nucleotide sequence of the present invention is<br>
directly transformed into the plastid genome. A major advantage of plastid transformation is<br>
that plastids are capable of expressing multiple open reading frames under control of a single<br>
promoter. Plastid transformation technology is extensively described in U.S. Pat. Nos.<br>
5,451,513,5,545,817, and 5,545,818, in PCT application no. WO 95/16783, and in McBride<br>
et al. (1994) Proc. Natl. Acad. Sci. USA 91,7301-7305. The basic technique for chloroplast<br>
transformation involves introducing regions of cloned plastid DNA flanking a selectable<br>
marker together with the gene of interest into a suitable target tissue, e.g., using biolistics or<br>
protoplast transformation (e.g., calcium chloride or PEG mediated transformation). The 1 to<br>
1.5 kb flanking regions, termed targeting sequences, facilitate homologous recombination<br>
with the plastid genome and thus allow the replacement or modification of specific regions of<br>
the plastome. Initially, point mutations in the chloroplast 16S rRNA and rpsl2 genes<br>
conferring resistance to spectinomycin and/or streptomycin are utilized as selectable markers<br>
for transformation (Svab, Z., Hajdukiewicz, P., and Maliga, P. (1990) Prqc. Natl. Acad. Sci.<br>
USA 87,8526-8530; Staub, J. M., and Maliga, P. (1992) Plant Cell 4,39-45). This resulted in<br>
stable homoplasmic transformants at a frequency of approximately one per 100<br>
bombardments of target leaves. The presence of cloning sites between these markers allowed<br>
creation of a plastid targeting vector for introduction of foreign genes (Staub, J. M., and<br>
Maliga, P. (1993) EMBO J. 12,601-606). Substantial increases in transformation frequency<br>
are obtained by replacement of the recessive rRNA or r-polypeptide antibiotic resistance<br>
genes with a dominant selectable marker, the bacterial aad A gene encoding the<br>
spectinomycin-detoxifying enzyme aminoglycoside-3'-adenyltransferase (Svab, Z., and<br>
Maliga, P. (1993) Proc. Natl. Acad. Sci. USA 90,913-917). Previously, this marker had been<br>
used successfully for high-frequency transformation of the plastid genome of the green alga<br><br><br>
Chlamydomonas reinhardtii (Goldschmidt-Clermont, M. (1991) Nucl. Acids Res. 19: 4083-<br>
4089). Other selectable markers useful for plastid transformation are known in the art and<br>
encompassed within the scope of the invention. Typically, approximately 15-20 cell division<br>
cycles following transformation are required to reach a homoplastidic state. Plastid<br>
expression, in which genes are inserted by homologous recombination into all of the several<br>
thousand copies of the circular plastid genome present in each plant cell, takes advantage of<br>
the enormous copy number advantage over nuclear-expressed genes to permit expression<br>
levels mat can readily exceed 10% of the total soluble plant polypeptide. In a preferred<br>
embodiment, a nucleotide sequence of the present invention is inserted into a plastid targeting<br>
vector and transformed into the plastid genome of a desired plant host. Plants homoplastic for<br>
plastid genomes containing a nucleotide sequence of the present invention are obtained, and<br>
are preferentially capable of high expression of the nucleotide sequence.<br>
The present invention also provides a method of identifying a plant yield-related gene<br>
comprising: a) providing a plant tissue sample; b) introducing into the plant tissue sample a<br>
candidate plant yield-related gene; c) expressing the candidate plant yield-related gene within<br>
the plant tissue sample; and d) determining whether the plant tissue sample exhibits change in<br>
yield response, whereby a change in response identifies a plant yield-related gene. The<br>
present invention also provides plant yield-related genes isolated according to the method.<br>
Yield response, as used herein, is measured by techniques well known to those skilled<br>
in the art. In the cereals yield response is determined, for example, by one or more of the<br>
following metrics, grain weight, grain length, grain weight/1000 grains, size of panicle,<br>
number of panicles, and number of grains/panicle.<br>
G. EG307 or EG1U7Antibodies<br>
The present invention also includes isolated antibodies capable of selectively binding<br>
to an EG307 or EG1117 polypeptide of the present invention or to a mimetope thereof. Such<br>
antibodies are also referred to herein as anti-EG307 or anti-EGl 117 antibodies. Particularly<br>
preferred antibodies of this embodiment include anti-0. sativa EG307 antibodies, anti-O.<br>
rufipogon EG307 antibodies, anti-Z. mays EG307 antibodies, anti-0. sativa EGl 117<br>
antibodies, anti-O. rufipogon EGl 117 antibodies, anti-Z. mays EGl 17 antibodies.<br>
Isolated antibodies are antibodies that have been removed from their natural milieu.<br>
The term "isolated" does not refer to the state of purity of such antibodies. As such, isolated<br>
antibodies can include anti-sera containing such antibodies, or antibodies that have been<br>
purified to varying degrees.<br><br><br>
As used herein, the term "selectively binds to" refers to the ability of antibodies of the<br>
present invention to preferentially bind to specified polypeptides and mimetopes thereof of<br>
the present, invention. Binding can be measured using a variety of methods known to those<br>
skilled in the art including immunoblot assays, immunoprecipitation assays,<br>
radioimmunoassays, enzyme immunoassays (e.g., ELISA), immunofluorescent antibody<br>
assays and immunoelectron microscopy; see, for example, Sambrook et al., ibid., and Harlow<br>
&amp; Lane, 1990, ibid.<br>
Antibodies of the present invention can be either polyclonal or monoclonal antibodies.<br>
Antibodies of the present invention include functional equivalents such as antibody fragments<br>
and genetically-engineered antibodies, including single chain antibodies, that are capable of<br>
selectively binding to at least one of the epitopes of the polypeptide or mimetope used to<br>
obtain the antibodies. Antibodies of the present invention also include chimeric antibodies<br>
that can bind to more than one epitope. Preferred antibodies are raised in response to<br>
polypeptides, or mimetopes thereof, that are encoded, at least in part, by a polynucleotide of<br>
the present invention.<br>
A preferred method to produce antibodies of the present invention includes (a)<br>
administering to an animal an effective amount of a polypeptide or mimetope thereof of the<br>
present invention to produce the antibodies and (b) recovering the antibodies. In another<br>
method, antibodies of the present invention are produced recombinant! y using techniques as<br>
heretofore disclosed to produce EG307 or EG1117 polypeptides of the present invention.<br>
Antibodies of the present invention have a variety of potential uses that are within the<br>
scope of the present invention. For example, such antibodies can be used (a) as reagents in<br>
assays to detect expression of EG307 or EG1117 by plant and/or (b) as tools to screen<br>
expression libraries and/or to recover desired polypeptides of the present invention from a<br>
mixture of polypeptides and other contaminants. Furthermore, antibodies of the present<br>
invention can be used to target cytotoxic agents to plants in order to directly kill such plants.<br>
Targeting can be accomplished by conjugating (i.e., stably joining) such antibodies to the<br>
cytotoxic agents using techniques known to those skilled in the art. Suitable cytotoxic agents<br>
are known to those skilled in the art. Suitable cytotoxic agents include, but are not limited to:<br>
double-chain polypeptides (i.e., toxins having A and B chains), such as diphtheria toxin, ricin<br>
toxin, Pseudomonas exotoxin, modeccin toxin, abrin toxin, and shiga toxin; single-chain<br>
toxins, such as pokeweed antiviral polypeptide, a-amanitin, and ribosome inhibiting<br>
polypeptides; and chemical toxins, such as melphalan, methotrexate, nitrogen mustard,<br>
doxorubicin and daunomycin. Preferred double-chain toxins are modified to include the toxic<br><br><br>
domain and translocation domain of the toxin but lack the toxin's intrinsic cell binding<br>
domain.<br>
H. Formulation of Growth-Enhancing Compositions<br>
The invention also includes compositions comprising at least one or both of the<br>
EG307 or EG1117 polypeptides of the present invention. In order to effectively control<br>
growth such compositions preferably contain sufficient amounts of polypeptide. Such<br>
amounts vary depending oh the target crop, and on the environmental conditions, such as<br>
humidity, temperature or type of soil. In a preferred embodiment, compositions comprising<br>
the EG307 and/or EG1117 polypeptide comprise host cells expressing the polypeptides<br>
without additional purification. In another preferred embodiment, the cells expressing the<br>
EG307 and/or EG 1117 polypeptides are lyophilized prior to their use as a growth-enhancing<br>
agent. In another embodiment, the EG307 or EG1117 polypeptides are engineered to be<br>
secreted from the host cells. In cases where purification of the polypeptides from the host<br>
cells in which they are expressed is desired, various degrees of purification of the EG307 or<br>
EG1117 polypeptides are reached.<br>
The present invention further embraces the preparation of compositions comprising at<br>
least one EG307 or EG1117 polypeptide of the present invention, which is homogeneously<br>
mixed with one or more compounds or groups of compounds described herein. The present<br>
invention also relates to methods of treating plants, which comprise application of the EG307<br>
or EG1117 polypeptides or compositions containing the EG307 or EG 1117 polypeptides, to<br>
plants. The EG307 or EG1117 polypeptides can be applied to the crop area in the form of<br>
compositions or plant to be treated, simultaneously or in succession, with further compounds.<br>
These compounds can be both fertilizers or micronutrient donors or other preparations that<br>
influence plant growth. They can also be selective herbicides, insecticides, fungicides,<br>
bactericides, nematicides, molluscicides or mixtures of several of these preparations, if<br>
desired together with further carriers, surfactants or application-promoting adjuvants<br>
customarily employed in the art of formulation. Suitable carriers and adjuvants can be solid<br>
or liquid and correspond to the substances ordinarily employed in formulation technology,<br>
e.g. natural or regenerated mineral substances, solvents, dispersants, wetting agents, tackifiers,<br>
binders or fertilizers.<br>
A preferred method of applying EG307 or EG1117 polypeptides of the present<br>
invention is by spraying the soil, water, or foliage of plants. The number of applications and<br>
the rate of application depend on the type of plant and the desired increase in yield. The<br>
EG307 or EG1117 polypeptides can also penetrate the plant through the roots via the soil<br><br><br>
(systemic action) by impregnating the locus of the plant with a liquid composition, or by<br>
applying the compounds in solid form to the soil, e.g. in granular form (soil application). The<br>
EG307 or EG1117 polypeptides may also be applied to seeds (coating) by impregnating the<br>
seeds either with a liquid formulation containing EG307 or EG1117 polypeptides, or coating<br>
them with a solid formulation. In special cases, further types of application are also possible,<br>
for example, selective treatment of the plant stems or buds.<br>
The EG307 or EG1117 polypeptides are used in unmodified form or, preferably,<br>
together with the adjuvants conventionally employed in the art of formulation, and are<br>
therefore formulated in known manner to emulsifiable concentrates, coatable pastes, directly<br>
sprayable or dilutable solutions, dilute emulsions, wettable powders, soluble powders, dusts,<br>
granulates, and also encapsulations, for example, in polymer substances. Like the nature of<br>
the compositions, the methods of application, such as spraying, atomizing, dusting, scattering<br>
or pouring, are chosen in accordance with the intended objectives and the prevailing<br>
circumstances.<br>
The formulations, compositions or preparations containing the EG307 or EG1117<br>
polypeptides and, where appropriate, a solid or liquid adjuvant, are prepared in a known<br>
manner, for example by homogeneously mixing and/or grinding the EG307 or EG1117<br>
polypeptides with extenders, for example solvents, solid carriers and, where appropriate,<br>
surface-active compounds (surfactants).<br>
Suitable solvents include aromatic hydrocarbons, preferably the fractions having 8 to<br>
12 carbon atoms, for example, xylene mixtures or substituted naphthalenes, phthalates such as<br>
dibutyl phthalate or dioctyl phthalate, aliphatic hydrocarbons such as cyclohexane or<br>
paraffins, alcohols and glycols and their ethers and esters, such as ethanol, ethylene glycol<br>
monomethyl or monoethyl ether, ketones such as cyclohexanone, strongly polar solvents such<br>
as N-methyl-2-pyrrolidone, dimethyl sulfoxide or dimethyl formamide, as well as epoxidized<br>
vegetable oils such as epoxidized coconut oil or soybean oil or water.<br>
The solid carriers used e.g. for dusts and dispersible powders, are normally natural<br>
mineral fillers such as calcite, talcum, kaolin, montmorillonite or attapulgite. In order to<br>
improve the physical properties it is also possible to add highly dispersed silicic acid or highly<br>
dispersed absorbent polymers. Suitable granulated adsorptive carriers are porous types, for<br>
example pumice, broken brick, sepiolite or bentonite; and suitable nonsorbent carriers are<br>
materials such as calcite or sand. In addition, a great number of pregranulated materials of<br>
inorganic or organic nature can be used, e.g. especially dolomite or pulverized plant residues.<br><br><br>
Suitable surface-active compounds are nonionic, cationic and/or anionic surfactants<br>
having good emulsifying, dispersing and wetting properties. The term "surfactants" will also<br>
be understood as comprising mixtures of surfactants. Suitable anionic surfactants can be both<br>
water-soluble soaps and water-soluble synthetic surface-active compounds.<br>
Suitable soaps are the alkali metal salts, alkaline earth metal salts or unsubstiruted or<br>
substituted ammonium salts of higher fatty acids (chains of 10 to 22 carbon atoms), for<br>
example the sodium or potassium salts of oleic or stearic acid, or of natural fatty acid mixtures<br>
which can be obtained for example from coconut oil or tallow oil. The fatty acid methyltaurin<br>
salts may also be used.<br>
More frequently, however, so-called synthetic surfactants are used, especially fatty<br>
sulfonates, fatty sulfates, sulfonated benzimidazole derivatives or alkylarylsulfonates.<br>
The fatty sulfonates or sulfates are usually in the form of alkali metal salts, alkaline<br>
earth metal salts or unsubstiruted or substituted ammonium salts and have a 8 to 22 carbon<br>
alkyl radical which also includes the alkyl moiety of alkyl radicals, for example, the sodium<br>
or calcium salt of lignonsulfonic acid, of dodecylsulfate or of a mixture of tatty alcohol<br>
sulfates obtained from natural fatty acids. These compounds also comprise the salts of<br>
sulfuric acid esters and sulfonic acids of fatty alcohol/ethylene oxide adducts. The sulfonated<br>
benzimidazole derivatives preferably contain 2 sulfonic acid groups and one fatty acid radical<br>
containing 8 to 22 carbon atoms. Examples of alkylarylsulfonates are the sodium, calcium or<br>
triethanolamine salts of dodecylbenzenesulfonic acid, dibutylnapthalenesulfonic acid, or of a<br>
naphthalenesulfonic acid/formaldehyde condensation product. Also suitable are<br>
corresponding phosphates, e.g. salts of the phosphoric acid ester of an adduct of p-<br>
nonylphenol with 4 to 14 moles of ethylene oxide.<br>
Non-ionic surfactants are preferably polyglycol ether derivatives of aliphatic or<br>
cycloaliphatic alcohols, or saturated or unsaturated fatty acids and alkylphenols, said<br>
derivatives containing 3 to 30 glycol ether groups and 8 to 20 carbon atoms in the (aliphatic)<br>
hydrocarbon moiety and 6 to 18 carbon atoms in the alkyl moiety of the alkylphenols.<br>
Further suitable non-ionic surfactants are the water-soluble adducts of polyethylene<br>
oxide with polypropylene glycol, ethylenediamine propylene glycol and alkylpolypropylene<br>
glycol containing 1 to 10 carbon atoms in the alkyl chain, which adducts contain 20 to 250<br>
ethylene glycol ether groups and 10 to 100 propylene glycol ether groups. These compounds<br>
usually contain 1 to 5 ethylene glycol units per propylene glycol unit.<br>
Representative examples of non-ionic surfactants are nonylphenolpolyethoxyethanols,<br>
castor oil polyglycol ethers, polypropylene/polyethylene oxide adducts,<br><br><br>
tributylphenoxypolyethoxyethanol, polyethylene glycol and octylphenoxyethoxyethanol.<br>
Fatty acid esters of polyoxyethylene sorbitan and polyoxyethylene sorbitan trioleate are also<br>
suitable non-ionic surfactants.<br>
Cationic surfactants are preferably quaternary ammonium salts which have, as N-<br>
substituent, at least one C8-C22 alkyl radical and, as further substituents, lower unsubstituted<br>
or halogenated alkyl, benzyl or lower hydroxyalkyl radicals. The salts are preferably in the<br>
form of halides, methylsulfates or ethylsulfates, e.g. stearyltrimethylammonium chloride or<br>
benzyldi(2-chloroethyl)ethylammonium bromide. The surfactants customarily employed in<br>
the art of formulation are described, for example, in "McCutcheon's Detergents and<br>
Emulsifiers Annuali" MC Publishing Corp. Ringwood, N.J., 1979, and Sisely and Wood,<br>
"Encyclopedia of Surface Active Agents," Chemical Publishing Co., Inc. New York, 1980.<br>
IV. Identification of Genes Evolved Under Neutral Conditions<br>
As described in detail herein, KA/KS analysis allows the identification of positively<br>
selected protein-coding genes; however, this type of analysis can also be used to identify<br>
another set of evolutionarily significant genes, those genes evolving under neutral conditions.<br>
A KA/KS ratio &gt; 1 signifies the role of positive selection, while conversely, a KA/KS<br>
ratio 
conserved). As noted elsewhere herein, most genes (in fact, the vast majority) are conserved.<br>
Only rare genes exhibit a KS/KS ratio &gt; 1, since very few genes are positively selected. As<br>
described herein, genes that were positively selected during domestication of the cereals (as<br>
well as other crops) have significant commercial value; however, another set of genes<br>
contained in the genomes of domesticated plants has been neither positively (to produce a<br>
desired, enhanced trait in the domesticated descendant) nor negatively selected (conserved).<br>
This subset of plant genes, as noted above, also has a significant commercial value, and this<br>
set of genes can be identified by using KA/KS analysis, to be described here.<br>
These genes comprise those that render the plant resistant to drought, disease, pests,<br>
(including, but not limited to, insects, animal herbivores, and microbes), high salt levels, and<br>
other stresses. Attacks by pests, and damage by drought or high salt levels, etc, are<br>
responsible for annual losses of billions of dollars to farmers, seed companies, and the large<br>
agricultural companies. The identification of genes that render wild plants resistant to these<br>
stresses is thus of great value, both socially (to a hungry world), and economically.<br>
The method to detect these genes is as follows. .After plants were first domesticated<br>
(and subsequently, as the descendents are further domesticated), they were "pampered", in the<br><br>
sense, for example, that humans supply water in sufficient quantities to meet the plant's<br>
needs. Thus the plant is not required to deal with drought stress "on its own". Similarly,<br>
humans remove insect pests (either physically, or through the use of pesticides), and segregate<br>
domesticated plants away from animal herbivores, such that the domesticated plant is not<br>
constantly confronted with the need to deal with these pests. In fact, it has been well<br>
documented that domesticated cereals, for example, are usually much more vulnerable to<br>
drought, high salt levels, pests, and other stresses than are their wild relatives/ancestors. This<br>
is because organisms generally do not maintain abilities that are not required to survive. As<br>
humans take over these roles, domesticated plants can save the high metabolic costs<br>
("metabolic extravagance") of maintaining genes that code for stress-related traits.<br>
This loss of resistance must of course stem from genetic differences (i.e., changes)<br>
between the ancestor and its pampered domesticated descendent. These genetic changes that<br>
result in loss of function can occur through three different mechanisms. The genes that code<br>
for these traits may actually be lost from the genome of the descendent crop. Gene loss has<br>
been documented and is a well-known phenomenon. Similarly, the genes that code for<br>
"unneeded" traits in a descendent crop may still persist in the genome, but are no longer<br>
expressed, as a result of promoter changes, for example. Alternatively, the genes coding for<br>
these unneeded traits may still be part of the genome, and may still be expressed, but the<br>
genes may have accumulated nucleotide substitutions that render the protein product either<br>
nonfunctional or less fully functional than the ancestral homolog. These genes are thus<br>
evolving neutrally.<br>
Neutral amino acid replacements accumulate in the protein product of a gene that is<br>
free of selective pressures (either positive or negative). For a domesticated plant that has been<br>
freed of the need to maintain a functional protein product for the gene of interest, a condition<br>
of molecular neutrality exits. This includes genes that code for traits like pest, disease,<br>
drought, salt, etc., resistance. Such fully unconstrained, neutrally evolving genes are perfect<br>
candidates for detection by KA/KS analysis, as a neutrally evolving gene will ideally exhibit a<br>
KA/KS ratio =*. 1, when the homolog from the ancestral and descendant plants are compared.<br>
Thus the method invented and described here involves high-throughput sequencing of<br>
a cDNA library for an ancestral plant, BLASTING the resulting ESTs against a database of<br>
ESTs from the modern descendent, and performing KA/KS analysis for homologous pairs.<br>
The details of this process are explained elsewhere in this patent, for the case of a positively<br>
selected gene. The genes with a KA/KS ratio = 1 will be the set of genes that control important<br>
stress resistant traits, and that these genes can be effectively and swiftly identified by use of<br><br><br>
this ratio, This commercially valuable set of genes includes those coding for desirable traits<br>
such resistance to pests, disease, drought, high salt levels, etc. To best identify these genes,<br>
the EST sequencing from both the modern domesticated and the ancestral species should be<br>
performed very carefully, with a high standard of accuracy. While one can make use of cereal<br>
EST databases available in GenBank, one may also resequence ESTs from cDNA libraries<br>
prepared specifically for this purpose. The accuracy of sequencing is important, because this<br>
will give rise to a very narrow distribution of gene pair comparisons between ancestral and<br>
modern homologs that have a KA/KS ratio equal to one. This will reduce the number of false<br>
positives to a minimum, thus expediting the process.<br>
When the accuracy of the screening process is not stringently controlled, or is<br>
unknown, it is possible that sequencing errors will obscure a KA/KS ratio of 1.0, and for this<br>
reason, KA/KS values of between about 0.75 -1.25 are checked carefully for evidence of<br>
neutral evolution.<br>
" Polynucleotides that have evolved under neutral conditions can then be mapped onto<br>
one of the known quantitative trait loci, or QTL, whereby the specific stress-resistance trait<br>
controlled by that polynucleotide may be rapidly and conclusively identified.<br>
V. Screening Methods for Identification of Agents<br>
The present invention also provides screening methods using the polynucleotides and<br>
polypeptides identified and characterized using the above-described methods. These<br>
screening methods are useful for identifying agents which may modulate the runction(s) of the<br>
polynucleotides or polypeptides in a manner that would be useful for enhancing or<br>
diminishing a characteristic in a domesticated or ancestor organism. Generally, the methods<br>
entail contacting at least one agent to be tested with a domesticated organism, ancestor<br>
organism, or transgenic organism or cell that has been transfected with a polynucleotide<br>
sequence identified by the methods described above, or a preparation of the polypeptide<br>
encoded by such polynucleotide sequence, wherein an agent is identified by its ability to<br>
modulate function of either the polynucleotide sequence or the polypeptide. For example, an<br>
agent can be a compound that is applied or contacted with a domesticated plant or animal to<br>
induce expression of the identified gene at a desired time. Specifically in regard to plants, an<br>
agent could be used, for example, to induce flowering at an appropriate time.<br>
As used herein, the term "agent" means a biological or chemical compound such as a<br>
simple or complex organic or inorganic molecule, a peptide, a protein or an oligonucleotide.<br>
A vast array of compounds can be synthesized, for example oligomers, such as oligopeptides<br><br><br>
and oligonucleotides, and synthetic organic and inorganic compounds based on various core<br>
structures, and these are also included in the term "agent". In addition, various natural<br>
sources can provide compounds for screening, such as plant or animal extracts, and the like.<br>
Compounds can be tested singly or in combination with one another.<br>
To "modulate function" of a polynucleotide or a polypeptide means that the function<br>
of the polynucleotide or polypeptide is altered when compared to not adding an agent.<br>
Modulation may occur on any level that affects function. A polynucleotide or polypeptide<br>
function may be direct or indirect, and measured directly or indirectly. A "function" of a<br>
polynucleotide includes, but is not limited to, replication, translation, and expression<br>
pattem(s). A polynucleotide function also includes functions associated with a polypeptide<br>
encoded within the polynucleotide. For example, an agent which acts on a polynucleotide and<br>
affects protein expression, conformation, folding (or other physical characteristics), binding to<br>
other moieties (such as Ugands), activity (01 other functional characteristics), regulation<br>
and/or other aspects of protein structure or function is considered to have modulated<br>
polynucleotide function. The ways that an effective agent can act to modulate the expression<br>
of a polynucleotide include, but are not limited to 1) modifying binding of a transcription<br>
factor to a transcription factor responsive element in the polynucleotide; 2) modifying the<br>
interaction between two transcription factors necessary for expression of the polynucleotide;<br>
3) altering the ability of a transcription factor necessary for expression of the polynucleotide<br>
to enter the nucleus; 4) inhibiting the activation of a transcription factor involved in<br>
transcription of the polynucleotide; S) modifying a cell-surface receptor which normally<br>
interacts with a iigand and whose binding of the tigand results in expression of the<br>
polynucleotide; 6) inhibiting the inactivation of a component of the signal transduction<br>
cascade that leads to expression of the polynucleotide; and 7) enhancing the activation of a<br>
transcription factor involved in transcription of the polynucleotide.<br>
A "function" of a polypeptide includes, but is not limited to, conformation, folding (or<br>
other physical characteristics), binding to other moieties (such as ligands), activity (or other<br>
functional characteristics), and/or other aspects of protein structure or functions. For<br>
example, an agent that acts on a polypeptide and affects its conformation, folding (or other<br>
physical characteristics), binding to other moieties (such as ligands), activity (or other<br>
functional characteristics), and/or other aspects of protein structure or functions is considered<br>
to have modulated polypeptide function. The ways that an effective agent can act to modulate<br>
the function of a polypeptide include, but are not limited to 1) changing the conformation,<br>
folding or other physical characteristics; 2) changing the binding strength to its natural Iigand<br><br><br>
or changing the specificity of binding to ligands; and 3) altering the activity of the<br>
polypeptide.<br>
Generally, the choice of agents to be screened is governed by several parameters, such<br>
as the particular polynucleotide or polypeptide target, its perceived function, its three-<br>
dimensional structure (if known or surmised), and other aspects of rational compound design.<br>
Techniques of combinatorial chemistry can also be used to generate numerous permutations<br>
of candidates. Those of skill in the art can devise and/or obtain suitable agents for testing.<br>
The in vivo screening assays described herein may have several advantages over<br>
conventional drug screening assays: 1) if an agent must enter a cell to achieve a desired<br>
therapeutic effect, an in vivo assay can give an indication as to whether the agent can enter a<br>
cell; 2) an in vivo screening assay can identify agents that, in the state in which they are added<br>
to the assay system are ineffective to elicit at least one characteristic which is associated with<br>
modulation of polynucleotide or polypeptide function, but that are modified by cellular<br>
components once inside a cell in such a way that they become effective agents; 3) most<br>
importantly, an in vivo assay system allows identification of agents affecting any component<br>
of a pathway that ultimately results in characteristics that are associated with polynucleotide<br>
or polypeptide function.<br>
In general, screening can be performed by adding an agent to a sample of appropriate<br>
cells which have been transfected with a polynucleotide identified using the methods of the<br>
present invention, and monitoring the effect, i.e., modulation of a function of the<br>
polynucleotide or the polypeptide encoded within the polynucleotide. The experiment<br>
preferably includes a control sample which does not receive the candidate agent. The treated<br>
and untreated cells are then compared by any suitable phenotypic criteria, including but not<br>
limited to microscopic analysis, viability testing, ability to replicate, histological examination,<br>
the level of a particular RNA or polypeptide associated with the cells, the level of enzymatic<br>
activity expressed by the cells or cell lysates, the interactions of the cells when exposed to<br>
infectious agents, and the ability of the cells to interact with other cells or compounds.<br>
Differences between treated and untreated cells indicate effects attributable to the candidate<br>
agent. Optimally, the agent has a greater effect on experimental cells than on control cells.<br>
Appropriate host cells include, but are not limited to, eukaryotic cells, preferably plant or<br>
animal cells. The choice of cell will at least partially depend on the nature of the assay<br>
contemplated.<br>
To test for agents that upregulate the expression of a polynucleotide, a suitablehost<br>
cell transfected with a polynucleotide of interest, such that the polynucleotide is expressed (as<br><br><br>
used herein, expression includes transcription and/or translation) is contacted with an agent to<br>
be tested. An agent would be tested for its ability to result in increased expression of mRNA<br>
and/or polypeptide. Methods of making vectors and transfection are well known in the art.<br>
"Transfection" encompasses any method of introducing the exogenous sequence, including,<br>
for example, lipofection, transduction, infection or electroporation. The exogenous<br>
polynucleotide may be maintained as a non-integrated vector (such as a plasmid) or may be<br>
integrated into the host genome.<br>
To identify agents that specifically activate transcription, transcription regulatory<br>
regions could be linked to a reporter gene and the construct added to an appropriate host cell.<br>
As used herein, the term "reporter gene" means a gene that encodes a gene product that can be<br>
identified (i.e., a reporter protein). Reporter genes include, but are not limited to, alkaline<br>
phosphatase, chloramphenicol acetyltransferase, (J-galactosidase, luciferase and green<br>
fluorescence protein (GFP). Identification methods for the products of reporter genes include,<br>
but are not limited to, enzymatic assays and fluorimetric assays. Reporter genes and assays to<br>
detect their products are well known in the art and are described, for example in Ausubel et al.<br>
(1987) and periodic updates. Reporter genes, reporter gene assays, and reagent kits are also<br>
readily available from commercial sources. Examples of appropriate cells include, but are not<br>
limited to, plant, fungal, yeast, mammalian, and other eukaryotic cells. A practitioner of<br>
ordinary skill will be well acquainted with techniques for transfecting eukaryotic cells,<br>
including the preparation of a suitable vector, such as a viral vector; conveying the vector into<br>
the cell, such as by electroporation; and selecting cells that have been transformed, such as by<br>
using a reporter or drug sensitivity element. The effect of an agent on transcription from the<br>
regulatory region in these constructs would be assessed through the activity of the reporter<br>
gene product<br>
Besides the increase in expression under conditions in which it is normally repressed<br>
mentioned above, expression could be decreased when it would normally be expressed. An<br>
agent could accomplish this through a decrease in transcription rate and the reporter gene<br>
system described above would be a means to assay for this. The host cells to assess such<br>
agents would need to be permissive for expression.<br>
Cells transcribing mRNA (from the polynucleotide of interest) could be used to<br>
identify agents that specifically modulate the half-life of mRNA and/or the translation of<br>
mRNA. Such cells would also be used to assess the effect of an agent on the processing<br>
and/or post-translational modification of the polypeptide. An agent could modulate the<br>
amount of polypeptide in a cell by modifying the turn-over (i.e., increase or decrease the half-<br><br><br>
life) of the polypeptide. The specificity of the agent with regard to the mRNA and<br>
polypeptide would be determined by examining the products in the absence of the agent and<br>
by examining the products of unrelated mRNAs and polypeptides. Methods to examine<br>
mRNA half-life, protein processing, and protein turn-over are well known to those skilled in<br>
the art.<br>
In vivo screening methods could also be useful in the identification of agents that<br>
modulate polypeptide function through the interaction with the polypeptide directly. Such<br>
agents could block normal polypeptide-ligand interactions, if any, or could enhance or<br>
stabilize such interactions. Such agents could also alter a conformation of the polypeptide.<br>
The effect of the agent could be determined using immunoprecipitation reactions.<br>
Appropriate antibodies would be used to precipitate the polypeptide and any protein tightly<br>
associated with it. By comparing the polypeptides immunoprecipitated from treated ceils and<br>
from untreated cells, an agent could be identified that would augment or inhibit polypeptide-<br>
ligand interactions, if any. Polypeptide-ligand interactions could also be assessed using cross-<br>
linking reagents that convert a close, but noncovalent interaction between polypeptides into a<br>
covalent interaction. Techniques to examine protein-protein interactions are well known to<br>
those skilled in the art. Techniques to assess protein conformation are also well known to<br>
those skilled in the art.<br>
It is also understood mat screening methods can involve in vitro methods, such as cell-<br>
free transcription or translation systems. In those systems, transcription or translation is<br>
allowed to occur, and an agent is tested for its ability to modulate function. For an assay that<br>
determines whether an agent modulates the translation of mRNA or a polynucleotide, an in<br>
vitro transcription/translation system may be used. These systems are available commercially<br>
and provide an in vitro means to produce mRNA corresponding to a polynucleotide sequence<br>
of interest. After mRNA is made, it can be translated in vitro and the translation products<br>
compared. Comparison of translation products between an in vitro expression system that<br>
does not contain any agent (negative control) with an in vitro expression system that does<br>
contain an agent indicates whether the agent is affecting translation. Comparison of<br>
translation products between control and test polynucleotides indicates whether the agent, if<br>
acting on this level, is selectively affecting translation (as opposed to affecting translation in a<br>
general, non-selective or non-specific fashion). The modulation of polypeptide function can<br>
be accomplished in many ways including, but not limited to, the in vivo and in vitro assays<br>
listed above as well as in in vitro assays using protein preparations. Polypeptides can be<br>
extracted and/or purified from natural or recombinant sources to create protein preparations.<br><br><br>
An agent can be added to a sample of a protein preparation and the effect monitored; that is<br>
whether and how the agent acts on a polypeptide and affects its conformation, folding (or<br>
other physical characteristics), binding to other moieties (such as ligands), activity (or other<br>
functional characteristics), and/or other aspects of protein structure or functions is considered<br>
to have modulated polypeptide function.<br>
In an example for an assay for an agent that binds to a polypeptide encoded by a<br>
polynucleotide identified by the methods described herein, a polypeptide is first<br>
recombinant^ expressed in a prokaryotic or eukaryotic expression system as a native or as a<br>
fusion protein in which a polypeptide (encoded by a polynucleotide identified as described<br>
above) is conjugated with a well-characterized epitope or protein. Recombinant polypeptide<br>
is then purified by, for instance, immunoprecipitation using appropriate antibodies or anti-<br>
epitope antibodies or by binding to immobilized ligand of the conjugate. An affinity column<br>
made of polypeptide or fusion protein is then used to screen a mixture of compounds which<br>
have been appropriately labeled. Suitable labels include, but are not limited to fluorochromes,<br>
radioisotopes, enzymes and chemiluminescent compounds. The unbound and bound<br>
compounds can be separated by washes using various conditions (e.g. high salt, detergent)<br>
that are routinely employed by those skilled in the art. Non-specific binding to the affinity<br>
column can be minimized by pre-clearing the compound mixture using an affinity column<br>
containing merely the conjugate or the epitope. Similar methods can be used for screening for<br>
an agent(s) that competes for binding to polypeptides. In addition to affinity chromatography,<br>
there are other techniques such as measuring the change of melting temperature or the<br>
fluorescence anisotropy of a protein which will change upon binding another molecule. For<br>
example, a BIAcore assay using a sensor chip (supplied by Pharmacia Biosensor, Stitt et al.<br>
(1995) Cell 80: 661-670) that is covalently coupled to polypeptide may be performed to<br>
determine the binding activity of different agents.<br>
It is also understood that the in vitro screening methods of this invention include<br>
structural, or rational, drug design, in which the amino acid sequence, three-dimensional<br>
atomic structure or other property (or properties) of a polypeptide provides a basis for<br>
designing an agent which is expected to bind to a polypeptide. Generally, the design and/or<br>
choice of agents in this context is governed by several parameters, such as side-by-side<br>
comparison of the structures of a domesticated organism's and homologous ancestral<br>
polypeptides, the perceived function of the polypeptide target, its three-dimensional structure<br>
(if known or surmised), and other aspects of rational drug design. Techniques of<br><br><br>
combinatorial chemistry can also be used to generate numerous permutations of candidate<br>
agents.<br>
Also contemplated in screening methods of the invention are transgenic animal and<br>
plant systems, which are known in the art.<br>
The screening methods described above represent primary screens, designed to detect<br>
any agent that may exhibit activity that modulates the function of a polynucleotide or<br>
polypeptide. The skilled artisan will recognize that secondary tests will likely be necessary in<br>
order to evaluate an agent further. For example, a secondary screen may comprise testing the<br>
agent(s) in an assay using mice and other animal models (such as rat), which are known in the<br>
art or in the domesticated or ancestral plant or animal itself. In addition, a cytotoxicity assay<br>
would be performed as a further corroboration that an agent which tested positive in a primary<br>
screen would be suitable for use in living organisms. Any assay for cytotoxicity would be<br>
suitable for this purpose, including, for example the MTT assay (Promega).<br>
The screening methods detailed earlier in this specification may be applied specifically<br>
to EG307 or EG1117. Accordingly, the invention provides a method of identifying an agent<br>
that modulates the function of the non-polypeptide coding regions of an EG307 or EG1117<br>
polynucleotide, comprising contacting a host cell that has been transfected with a construct<br>
comprising the non-polypeptide coding region operabley linked to a reporter gene coding<br>
region, with at least one candidate agent, wherein the agent is identified by its ability to<br>
modulate the transcription or translation of said reporter polynucleotide. The present<br>
invention also provides agents identified by the method.<br>
The present invention also provides a method of identifying an agent that modulates<br>
the function of the non-polypeptide coding regions of an evolutionarily significant EG307 or<br>
EG1117 polynucleotide, comprising contacting a plant or transgenic plant containing an<br>
EG307 or EG1117 polynucleotide with at least one candidate agent, wherein the agent is<br>
identified by its ability to modulate the transcription or translation of said reporter<br>
polynucleotide. The present invention also provides agents identified by the method.<br>
The present invention also provides a method of identifying an agent which may<br>
modulate yield, said method comprising contacting at least one candidate agent with a plant or<br>
cell comprising an EG307 or EG1117 gene, wherein the agent is identified by its ability to<br>
modulate yield. In one embodiment the plant or cell is transfected with a polynucleotide<br>
encoding and EG307 or EG1117 gene. The present invention also provides agents identified<br>
by the method. In one embodiment, the identified agent modulates yield by modulating a<br><br><br>
function of the polynucleotide encoding the polypeptide. In another embodiment, the<br>
identified agent modulates yield by modulating a function of the polypeptide.<br>
The invention also includes agents identified by the screening methods described<br>
herein.<br>
The following examples are provided to further assist those of ordinary skill in the art.<br>
Such examples are intended to be illustrative and therefore should not be regarded as limiting<br>
the invention. A number of exemplary modifications and variations are described in this<br>
application and others will become apparent to those of skill in this art. Such variations are<br>
considered to fall within the scope of the invention as described and claimed herein.<br>
EXAMPLES<br>
EXAMPLE 1: cDNA Library Construction<br>
A domesticated plant or animal cDNA library is constructed using an appropriate<br>
tissue from the plant or animal. A person of ordinary skill in the art would know the<br>
appropriate tissue or tissues to analyze according to the trait of interest Alternately, the<br>
whole organism may be used. For example, 1 day old plant seedlings are known to express<br>
most of the plant's genes.<br>
Total RNA is extracted from the tissue (RNeasy kit, Quiagen; RNAse-free Rapid Total<br>
RNA kit, 5 Prime~3 Prime, Inc., or any similar and suitable product) and the integrity and<br>
purity of the RNA are determined according to conventional molecular cloning methods.<br>
Poly A+ RNA is isolated (Mini-01igo(dT) Cellulose Spin Columns, 5 Prime-3 Prime, Inc., or<br>
any similar and suitable product) and used as template for the reverse-transcription of cDNA<br>
with oligo (dT) as a primer. The synthesized cDNA is treated and modified for cloning using<br>
commercially available kits. Recombinants are then packaged and propagated in a host cell<br>
line. Portions of the packaging mixes are amplified and the remainder retained prior to<br>
amplification. The library can be normalized and the numbers of independent recombinants<br>
in the library is determined.<br>
EXAMPLE 2; Sequence Comparison<br>
Randomly selected ancestor cDNA clones from the cDNA library are sequenced using<br>
an automated sequencer, such as an ABI377 or MegaBACE 1000 or any similar and suitable<br>
- product. Commonly used primers on the cloning vector such as the Ml 3 Universal and<br>
Reverse primers are used to carry out the sequencing. For inserts that are not completely<br><br>
sequenced by end sequencing, dye-labeled terminators or custom primers can be used to fill in<br>
remaining gaps.<br>
The detected sequence differences are initially checked for accuracy, for example by<br>
finding the points where there are differences between the domesticated and ancestor<br>
sequences; checking the sequence fluorogram (chromatogram) to determine if the bases that<br>
appear unique to the domesticated organism correspond to strong, clear signals specific for<br>
the called base; checking the domesticated organism's hits to see if there is more than one<br>
sequence that corresponds to a sequence change; and other methods known in the art, as<br>
needed. Multiple domesticated organism sequence entries for the same gene that have the<br>
same nucleotide at a position where there is a different ancestor nucleotide provides<br>
independent support that the domesticated sequence is accurate, and that the<br>
domesticated/ancestor difference is real. Such changes are examined using public or<br>
commercial database information and the genetic code to determine whether these DNA<br>
sequence changes result in a change in the amino acid sequence of the encoded protein. The<br>
sequences can also be examined by direct sequencing of the encoded protein.<br>
EXAMPLE 3; Molecular Evolution Analysis<br>
The domesticated plant or animal and wild ancestor sequences under comparison are<br>
subjected to KA/KS analysis. In this analysis, publicly or commercially available computer<br>
programs, such as Li 93 and INA, are used to determine the number of non-synonymous<br>
changes per site (KA) divided by the number of synonymous changes per site (Ks) for each<br>
sequence under study as described above. Full-length coding regions or partial segments of a<br>
coding region can be used. The higher the KA/KS ratio, the more likely that a sequence has<br>
undergone adaptive evolution. Statistical significance of KA/KS values is determined using<br>
established statistic methods and available programs such as the t-test.<br>
To further lend support to the significance of a high KA/KS ratio, the domesticated<br>
sequence under study can be compared to other evolutionarily proximate species. These<br>
comparisons allow further discrimination as to whether the adaptive evolutionary changes are<br>
unique to the domesticated plant or animal lineage compared to other closely related species.<br>
The sequences can also be examined by direct sequencing of the gene of interest from<br>
representatives of several diverse domesticated populations to assess to what degree the<br>
sequence is conserved in the domesticated plant or animal.<br><br>
EXAMPLE 4; cDNA Library Construction<br>
A teosinte cDNA library is constructed using whole teosinte 1 day old seedlings, or<br>
other appropriate plant tissues. Total RNA is extracted from the seedling tissue and the<br>
integrity and purity of the RNA are determined according to conventional molecular cloning<br>
methods. Poly A+ RNA is selected and used as template for the reverse-transcription of<br>
cDNA with oligo (dT) as a primer. The synthesized cDNA is treated and modified for<br>
cloning using commercially available kits. Recombinants are then packaged and propagated<br>
in a host cell line. Portions of the packaging mixes are amplified and the remainder retained<br>
prior to amplification. Recombinant DNA is used to transfect K coli host cells, using<br>
established methods. The library can be normalized and the numbers of independent<br>
recombinants in the library is determined.<br>
EXAMPLE 5: Sequence Comparison<br>
Randomly selected teosinte seedling cDNA clones from the cDNA library axe<br>
sequenced using an automated sequencer, such as the ABI377. Commonly used primers on<br>
the cloning vector such as the MB Universal and Reverse primers are used to carry out the<br>
sequencing. For inserts that are not completely sequenced by end sequencing, dye-labeled<br>
terminators are used to fill in remaining gaps.<br>
The resulting teosinte sequences are compared to domesticated maize sequences via<br>
database searches. Genome databases are publicly or commercially available for a number of<br>
species, including maize. One example of a maize database can be found at the MaizeDB<br>
website at the University of Missouri. MaizeDB is a public Internet gateway to current<br>
knowledge about the maize genome and its expression. Other appropriate maize EST<br>
(expressed sequence tag) databases are privately owned and maintained. The high scoring<br>
"hits," i.e., sequences that show a significant (e.g., &gt;80%) similarity after homology analysis,<br>
are retrieved and analyzed. The two homologous sequences are then aligned using the<br>
alignment program CLUSTAL V developed by Higgins et al. Any sequence divergence,<br>
including nucleotide substitution, insertion and deletion, can be detected and recorded by the<br>
alignment.<br>
The detected sequence differences are initially checked for accuracy by finding the<br>
points where there are differences between the teosinte and maize sequences; checking the<br>
sequence fluorogram (chromatogram) to determine if the bases that appear unique to maize<br>
correspond to strong, clear signals specific for the called base;-checking the maize hits to see<br>
if there is more than one maize sequence that corresponds to a sequence change; and other<br><br>
methods known in the art as needed. Multiple maize sequence entries for the same gene that<br>
have the same nucleotide at a position where there is a different teosinte nucleotide provides<br>
independent support mat the maize sequence is accurate, and that the teosinte/maize<br>
difference is real. Such changes are examined using public/commercial database information<br>
and the genetic code to determine whether these DNA sequence changes result in a change in<br>
the amino acid sequence of the encoded protein. The sequences can also be examined by<br>
direct sequencing of the encoded protein.<br>
EXAMPLE 6: Molecular Evolution Analysis<br>
The teosinte and maize sequences under comparison are subjected to KA/KS analysis.<br>
In this analysis, publicly or commercially available computer programs, such as Li 93 and<br>
INA, are used to determine the number of non-synonymous changes per site (KA) divided by<br>
the number of synonymous changes per site (Ks) for each sequence under study as described<br>
above. This ratio, KA/KS, has been shown to be a reflection of the degree to which adaptive<br>
evolution, i.e., positive selection, has been at work in the sequence under study. Typically,<br>
full-length coding regions have been used in these comparative analyses. However, partial<br>
segments of a coding region can also be used effectively. The higher the KA/KS ratio, the<br>
more likely that a sequence has undergone adaptive evolution. Statistical significance of<br>
KA/KS values is determined using established statistic methods and available programs such<br>
as the t-test. Those genes showing statistically high KA/KS ratios between teosinte and maize<br>
genes are very likely to have undergone adaptive evolution.<br>
To further lend support to the significance of a high KA/KS ratio, the sequence under<br>
study can be compared in other ancestral maize species. These comparisons allow further<br>
discrimination as to whether the adaptive evolutionary changes are unique to the domesticated<br>
maize lineage compared to other ancestors. The sequences can also be examined by direct<br>
sequencing of the gene of interest from representatives of several diverse maize populations to<br>
assess to what degree the sequence is conserved in the maize species.<br>
EXAMPLE 7; Application of KA/KS Method to Maize and Teosinte Homologous<br>
Sequences obtained from a Database<br>
Comparison of domesticated maize and teosinte sequences available on Genbank<br>
(accessable through the Entrez Nucleotides database at the National Center for Biotechnology<br>
Information web site) revealed at least four homologous genes: waxy, A1*,A1 and globulin<br>
for which sequence was available from both maize and teosinte. All available sequences for<br><br><br>
these genes for both maize and teosinte were compared. The KA/KS ratios were determined<br>
using Li93 and/or INA:<br><br>
Although it was anticipated that Hie polymorphism (multiple allelic copies) and/or the<br>
polyploidy (more than 2 sets of chromosomes per cell) observed in maize might make a<br>
KA/KS analysis complex or difficult, it was found that this was not the case.<br>
While the above KA/KS values indicate that these genes are not positively selected, this<br>
example illustrates that the KA/KS method can be applied to maize and its teosinte sequences<br>
obtained from a database.<br>
EXAMPLE 8: Study of Protein Function using a Transgenic Plant<br>
The functional roles of a positively selected maize gehe obtained according to the<br>
methods of Examples 4-7 can be assessed by conducting assessments of each allele of the<br>
gene in a transgenic maize plant. A transgenic plant can be created using an adaptation of the<br>
method described in Peng et al. (1999) Nature 400:256-261. Physiological, morphological<br>
and/or biochemical examination of the transgenic plant or protein extracts thereof will permit<br>
association of each allele with a particular phenotype.<br>
EXAMPLE 9: Mapping of Positively Selected Genes to OTLs<br>
QTL (quantitative trait locus) analysis has defined chromosomal regions that contain<br>
the genes that control several phenotypic traits of interest in maize, including plant height and<br>
oil content By physically mapping each positively-selected gene identified by this method<br>
onto one of the known QTLs, the specific trait controlled by each positively-selected gene can<br>
be rapidly and conclusively identified.<br>
EXAMPLE 10: Discovery of New Gene EG307<br>
A normalized cDNA library was constructed from pooled tissues (including leaves,<br>
panicles, and stems) of Oryza mfipogon, the species known to be ancestral to modern rice. A<br>
clone designated PBI0307H9 was first sequenced as part of a high-throughput sequencing -<br>
project on a MegaBACE 1000 sequencer (AP Biotech). (SEQ ID NO:89) The sequence of<br>
this clone was used as a query sequence in a BLAST search of the GenBank database. Four<br><br><br>
anonymous rice ESTs (accession nos. AU093345, C29145, ISAJ0161, AU056792) were<br>
retrieved as hits. Further sequencing revealed that PBI307H9 was a partial cDNA clone.<br>
PBI307H9 had a high KA/KS ratio when compared to the domesticated rice (Oryza sativa)<br>
ESTs in GenBank. cDNA amplification and sequencing were accomplished as follows: Total<br>
RNA was isolated from O. rufipogon (strain NSGC5953) and O. sativa cv. Nipponbare<br>
(Qiagen RNeasy Plant Mini Kit: cat #74903). First strand cDNA was synthesized using a dT<br>
primer (AP Biotech Ready-to-Go T-Primed First-Strand Kit: cat #27-9263-01) and then used<br>
for PCR analysis (Qiagen HotStarTaq Master Mix Kit: cat#203445).<br>
For ease in nomenclature, the gene contained in clone PBI0307H9 is named EG307,<br>
both here and throughout. Initially, before final sequence confirmation, the KA/KS ratio for<br>
EG307 derived from modern rice (O. sativa) and ancestral rice {O. rufipogon) EG307 was<br>
1.7.<br>
Once these partial sequences were confirmed in both O. rufipogon and O. sativa, 5'<br>
RACE (Clontech SMART RACE cDNA Amplification Kit: cat # K1811-1) was performed<br>
with a gene specific primer to obtain the 5' end of this gene. The complete gene, termed<br>
EG307, has a coding region 1344 bp long. Final confirmation of the complete EG307 CDS<br>
(1344 bp) in O. sativa and O. rufipogon allowed pairwise comparisons of a number of strains<br>
of O. rufipogon and O. sativa. Many of these comparisons yield KA/KS ratios greater than<br>
one, some with statistical significance. This is compelling evidence for the role of positive<br>
selection on the EG307 gene. As the selection pressure imposed upon ancestral rice was<br>
human imposed, this is compelling evidence that EG307 is a gene that was selected for during<br>
human domestication of rice. No homologs to EG307 were identified by BLAST search to<br>
the non-redundant section of GenBank, and*, as noted above, only four rice genes were<br>
identified by BLAST in the EST section of GenBank (AU093345, AU056792, C29145, and<br>
ISA0161). All four ESTs were essentially uncharacterized.<br>
EXAMPLE 11; Further KA/KS analysis of EG307<br>
In order to ascertain the extent of genetic diversity present in O. sativa for the EG307<br>
gene, genomic DNA was isolated from several different strains of O. sativa (acquired from<br>
the National Small Grains Collection, U.S.D.A., Aberdeen, Idaho), using Qiagen's protocol<br>
(DNeasy Plant Mini Kit: cat #69103). EG307 was then sequenced in genomic DNA from six<br>
different O. sativa strains: Nipponbare, Lemont, IR64, Teqing, Azucena, and Kasalath. The<br>
KA/KS ratios for each of these strains varied when compared to O. rufipogon. Table 1 shows<br>
results for the entire 1344 bases of coding region.<br><br><br><br>
There were differences in the untranslated (UTR) regions between O. rufipogon and<br>
all these O. sativa strains. The wide range of KA/KS ratios was expected due to the differing<br>
degrees of cross breeding among the O. sativa strains. Some were more similar to O.<br>
rufipogon than others due to cross breeding between O. rufipogon with the domesticated<br>
strains. Sliding window analysis was performed for all pairwise comparisons between the<br>
protein coding region of O. rufipogon EG307 to the protein coding region of each of the O.<br>
sativa strains we sequenced. This allowed identification of the specific areas of the protein<br>
that have been selected during domestication. Such pinpointing will allow a targeted<br>
approach to characterization of the changes that are important between the ancestral protein<br>
and the protein of the domesticated descendent crop plant. This may permit development of<br>
agents that target these vital domains of the protein, with the goal of increasing yield.<br>
The length of the "window" was in most cases 150 bp, with a 50 bp overlap with<br>
adjacent windows. (Thus, as an example, if reading from the 5' end of a CDS, the first<br>
window was 150 bp in length, as was the adjacent second window to its 3' side. The second<br>
window, also 150 in length, overlapped the first window by 50 bp at the 5' end of the second<br>
window, and the third window, also 150 bp, overlapped the second window by 50 bp at the 5'<br>
end of the third window. Thus, the second window overlapped both its adjacent neighbors,<br>
each by 50 bp.) In addition a second window analysis was completed in which the CDS was<br>
divided approximately into halves. This allows a greater sample size of nucleotides, so that<br>
an accurate statistical sampling can be undertaken. It should also be noted that KA/KS,<br>
although conventionally expressed as a ratio, is really a way of asking "Does the KA value<br><br><br>
exceed the KS value by a statistically significant amount?" Thus, when KS = 0, as often<br>
happens in ancestral rice-to-modem rice comparisons (because there are only some 7,000-<br>
8,000 years of domestication), a ratio cannot be computed, since the denominator of the<br>
fraction would equal zero. However, such comparisons may still detect the action of positive<br>
selection, if the (Ka-Ks) difference is statistically significant Thus for several comparisons<br>
shown in the following tables, positive selection can be detected, as long as the comparison is<br>
statistically significant. Like those comparisons for which the KA/KS ratio is significant, these<br>
are shown in bold.<br>
It should also be noted that as a result of the stochastic nature of the nucleotide<br>
.substitution process, not all comparisons to modern rice strains are expected to reveal<br>
evidence of positive selection, particularly since some cross breeding between O. rufipogon<br>
and modern O. sativa is known to have occurred.<br><br>
It is important to note here that there is statistical support for positive selection displayed in<br>
the comparison between O. rttfipogon and Nipponbare, when the first large window is used.<br>
This is good evidence that positive selection has occurred (as a result of human<br>
domestication) between the ancestral O. rttfipogon, and the domesticated O. sativa (strain<br><br><br>
Nipponbare) EG307 homologs. As noted above, as a result of the stochastic nature of the<br>
nucleotide substitution process, not all comparisons to modern rice strains are expected to<br>
reveal evidence of positive selection. In addition, as noted above, cross breeding has occurred<br>
between O. rufipogon and some domesticated strains, further obscuring the signal of<br>
selection. What this analysis makes clear, however, is that positive selection has occurred on<br>
the EG307 gene.<br><br>
It is important to note here that there is statistical support for positive selection displayed in<br>
the comparison between O. rufipogon and Lemont, when the first large window is used. This<br>
is good evidence that positive selection has occurred (as a result of human domestication)<br>
between the ancestral O. rufipogon, and the domesticated O. sativa (strain Lemont) EG307<br>
homologs. As noted above, as a result of the stochastic nature of the nucleotide substitution<br>
process, not all comparisons to modem rice strains are expected to reveal evidence of positive<br>
selection. In addition, as noted above, cross breeding has occurred between O. rufipogonand<br><br><br>
some domesticated strains, further obscuring the signal of selection. What this analysis makes<br>
clear, however, is that positive selection has occurred on the EG307 gene.<br><br>
Note that the protein coding region sequences of EG307 from O. rufipogon and from the O.<br>
sativa strain IR64 are identical, thus, the KA/KS values are equal to 2ero. IR64 is a low<br>
yielding modern strain (personal communication, Shannon Pinson, Research Geneticist,<br>
USDA-ARS Rice Research Unit, Beaumont, TX), suspected of massive amounts of<br>
interbreeding with wild O. rufipogon.<br><br><br>
Note that no comparisons between the EG307 sequences from O. rufipogon and O. sativa<br>
strain Teqing exhibit KA/KS ratios greater than one. However, as noted above, as a result of<br>
the stochastic nature of the nucleotide substitution process, not all comparisons to modern rice<br>
strains are expected to reveal evidence of positive selection. In addition, as "noted above, cross<br>
breeding has occurred between O. rufipogon and some domesticated strains, further obscuring<br>
the signal of selection.<br><br><br>
It is important to note here that there is statistical support for positive selection displayed in<br>
the comparison between O. rufipogon and Azucena, when the first large window is used.<br>
This is again good evidence that positive selection has occurred (as a result of human<br>
domestication) between the ancestral O. rufipogon, and the domesticated 0. sativa (strain<br>
Azucena) EG307 homologs. As noted above, as a result of the stochastic nature of the<br>
nucleotide substitution process, not all comparisons to modern rice strains are expected to<br>
reveal evidence of positive selection. In addition, as noted above, cross breeding has occurred<br>
between O. rufipogon and some domesticated strains, further obscuring the signal of<br>
selection. What this analysis once again makes clear, however, is that positive selection has<br>
occurred on the EG307 gene.<br><br><br>
differences (designated as Kasalath 1,2,3, and 4) in this sequence, and as they differ only by<br>
single nucleotides, we have chosen to show only one, for purposes of clarity. The KA/KS<br>
ratios for each of the full CDS sequences, is shown, however. Note that no comparisons<br>
between the EG307 sequences from O. rufipogon and O. saliva strain Kasalath exhibit KA/KS<br>
ratios greater than one. However, as noted above, as a result of the stochastic nature of the<br>
nucleotide substitution process, not all comparisons to modern rice strains are expected to<br>
reveal evidence of positive selection. In addition, as noted above, cross breeding has occurred<br>
between O. rufipogon and some domesticated strains, further obscuring the signal of<br>
selection.<br>
Upon completion of sequencing of EG307 in the NSGC 5953 strain of O. rufipogon,<br>
the completed sequence was used to design amplification primers. These primers were then<br>
used in the Polymerase Chain Reaction (PCR) to amplify the EG307 gene from several other<br>
O. rufipogon strains, including NSGC 5948, NSGC 5949, and IRGC105491. The amplified<br>
EG307 gene was then sequenced for each of these strains.<br><br>
EXAMPLE 12: Mapping EG307<br>
EG307 was then physically mapped in rice. Clemson University has developed a Rice<br>
Nipponbare bacterial artificial chromosome (BAC) Library; See Budiman, M.A. 1999,<br>
"Construction and characterization of deep coverage BAC libraries for two model crops:<br>
Tomato and rice, and initiation of a chromosome walk to jointless-2 in tomato". Ph.D. thesis,<br>
Texas A &amp; M University, College Station, TX. Library clones are available from Clemson in<br>
the form of hybridization filters.<br>
Two different rice BAC libraries used in screening were purchased from the Clemson<br>
University Genomics Institute (CUGI). The OSJNBa library was constructed at CUGI from<br>
genomic DNA of the japonica rice strain (Nipponbare variety), and has an average insert size<br>
of 130 kb, covering 11 genome equivalents. This is one of the most widely used libraries for<br>
the International Rice Genome Sequencing Project. It was constructed in the Hindlll site of<br>
pBeloBACl 1 and contains 36,864 clones. The OSJNBb library was also constructed at CUGI<br>
from genomic DNA of the japonica rice strain (Nipponbare variety), and has an average insert<br>
size of 120 kb, covering 15 genome equivalents. This is another of the most widely used<br>
libraries for the International Rice Genome Sequencing Project. It was constructed in the<br>
EcoRl site of pIndigoBac536 and contains 55,296 clones.<br>
The DIG protocol (BMB-Roche PCR DIG Probe Synthesis Kit cat #1636090)<br>
successfully labeled a unique EG307 494bp PCR product (primers: 5'-<br>
GAGTTCACAGGACAGCAGCA-3' (SEQ ID NO:87) and 5'-<br>
CAATTCTCTGAGATGCCTTGG-3') (SEQ ID NO:88) to screen against rice BAC filters.<br>
The blots were detected easily using chemiluminescence as per the DIG protocol (BMB-<br>
Roche DIG Luminescent Detection Kit: cat #1636090). Two different O. sativa libraries,<br>
OSJNBa, and OSJNBb were screened for a total of 5 different filters, three covering the<br>
OSJNBb library, and two covering the OSJNBa library. Table 8 shows the individual BACs<br>
identified by all three screens:<br><br><br>
The reference data that allows physical mapping of a gene to a particular contig or<br>
chromosome are known to those skilled in the art, and are available on a web page made<br>
known to purchasers of filter sets or libraries from CUGI. There were also several faint, not<br>
significant hybridizations to contig 113, which was also on chromosome 3.<br>
Rice contig 80 was on chromosome 3 and contained 66 BACs and 7 markers. Judging<br>
by the overlap of all these BACs within contig 80, EG307 was approximately 200 kb<br>
upstream of marker CD01387 on the short arm of chromosome 3.<br>
Data formerly in RiceGenes, a publicly accessible genome database developed and<br>
curated by the USDA-ARS is now integrated in Gramene. Gramene was recently funded by<br>
the USDAIFAFS programme to create a curated, open-source, Web-accessible data resource<br>
for comparative genome analysis in the grasses. It provides a collection of rice genetic maps<br>
from Cornell University, the Japanese Rice Genome Research Program (JRGP), and the<br>
Korea Rice Genome Research Program (KRGRP), as well as comparisons with maps from .<br>
other grasses (maize, oat, and wheat). The CD01387 marker was mapped to several different<br>
rice maps using the RiceGenes website.<br><br>
There were also several QTLs mapped to this region, but many of them had rather<br>
wide ranges that covered almost the entire chromosome. One well-documented QTL for 1000<br>
grain weight was mapped to this region of chromosome 3 and was associated with marker<br>
RZ672 (S.R. McCouch, et al. Genetics 150:899-909 Oct 98). On one map (R3) CD01387<br>
mapped to 30.4 cM and RZ672 mapped to 39 cM, and both of these markers mapped to four<br>
other rice maps (Rice-CU-3,3RC94,3RC00, and 3RW99) in similar ranges (Figure 5). Thus,<br>
EG307 was within -10 cM of this QTL marker. The R3 map also had a BAC,<br>
OSJNBa0091Pll, mapped to 21.45 cM-21.95 cM. EG307 was negative for this BAC and<br>
any others in the same contig upon screening the rice BAC libraries. The grain weight QTL<br>
region of rice had also been involved in some synteny studies between rice and maize that<br>
indicated synteny between rice chromosome 3S and maize chromosomes IS and 9L (W.A.<br>
Wilson, etal Genetics 153(1): 453-473 Sep 99).<br>
EXAMPLE 13; Identification of EG307 in maize and teosinte<br>
Searching the maize genome in GenBank by BLAST (using rice EG307 sequences)<br>
identified two maize ESTs, accession numbers BE511288 and BG320985, which appeared to<br>
be homologous. Primers were designed that allowed successful amplification of the maize<br>
(Zea mays) and teosinte {Zea mays parviglumis) EG307 homologs (SEQ ID NO:33 and SEQ<br>
ID NO:34, having a suggested open reading frame represented by SEQ ID NO:35, and SEQ<br>
ID NO:66, having a suggested open reading frame represented by SEQ ID NO:67). (Protein<br>
sequences for maize and teosinte were deduced; and are represented by SEQ ID NO:36 and<br>
SEQ ID NO:68.) Table 9 shows KA/KS estimates for a comparison between maize and<br>
teosinte.<br><br>
Although these KA/KS values do not show ratios that are greater than one, there is still<br>
evidence for positive selection. All amino acid replacements between ancestral rice and its<br>
modern domesticated descendant were characterized, and the same analysis was performed<br>
for teosinte and its descendant, modern maize. In both (independent) cases of domestication,<br><br><br>
a consistent pattern is observed: nearly all amino acid replacements in the modern crop<br>
(whether maize or rice), as compared to the ancestral plant (teosinte or ancestral rice) result in<br>
increased charge/polarity, increased solubility, and decreased hydrophobicity. This pattern is<br>
most unlikely to have occurred by chance in these two independent domestication events.<br>
This suggests that these replacements were a similar response to human imposed<br>
domestication. This is powerful evidence that EG307 has been selected as a result of human<br>
domestication of these two cereals.<br>
Upon completion of sequencing of EO307 in one strain of teosinte, the completed<br>
sequence was used to design amplification primers. These primers were then used in the<br>
Polymerase Chain Reaction (PCR) to amplify the EG307 gene from several other teosinte<br>
strains, as well as several strains of modern maize. The amplified EG307 gene was then<br>
sequenced for each of these strains.<br>
Although the foregoing invention has been described in some detail by way of<br>
illustration and example for purposes of clarity and understanding, it will be apparent to those<br>
of ordinary skill in the art that certain changes and modifications can be practiced. Therefore,<br>
the description and examples should not be construed as limiting the scope of the invention,<br>
which is delineated by the appended claims.<br>
EXAMPLE 14; Discovery of New Gene EG1117 and KAKs analysis<br>
Clone IWF1117H5 (hereafter termed EG1117) was first sequenced during EG's high-<br>
throughput sequencing project, conducted on MegaBACE 1000 sequencers (AP Biotech).<br>
This clone was sequenced from a normalized cDNA library (Incyte Genomics) constructed<br>
from material from ancestral rice, Oryza rufipogon. GenBank BLAST results hit three<br>
anonymous rice ESTs (AU055884, AU055885, BI808367), two anonymous corn ESTs<br>
(AI783000, AW000223), and two anonymous wheat ESTs (BE444456, BE443845). Further<br>
sequencing revealed that IWF1117H5 was a partial cDNA clone. It had a KA/KS ratio<br>
divisible by zero when compared to the domesticated rice, Oryza sativa, ESTs in GenBank.<br>
Genomic DNA was isolated from several different cultivars of O. sativa following<br>
Qiagen's protocol (DNeasy Plant Mini Kit: cat #69103). Total RNA was isolated from O.<br>
rufipogon and O. sativa cv. Nipponbare (Qiagen RNeasy Plant Mini Kit: cat #74903). First<br>
strand cDNA was synthesized using a dT primer (AP Biotech Ready-To-Go T-Primed First-<br>
Strand Kit: cat #27-9263-01) and then used for PCR analysis (Qiagen HotStarTaq Master Mix<br>
Kit: cat #203445). These protocols were also performed on Zea mays (maize), Zea mays<br>
parviglumis (teosinte), and on Triticum aestivum (modern wheat).<br><br><br>
Once these partial sequences were confirmed in both O. rufipogon and O. sativa,<br>
inverse PCR was performed with gene specific primers to attempt to obtain the 5' end of this<br>
gene. To date, 1659 bp of CDS in O. rufipogon and O. sativa (Figures 6 and 7) have been<br>
identified. This partial sequence includes the stop codon.<br>
EG1117 was then partially sequenced in genomic DNA from six different O. sativa<br>
strains: Nipponbare, Lemont, IR64, Teqing, Azucena, and Kasalath. The KA/KS ratios for<br>
each of these strains varied when compared to O. rufipogon strain 5948. The KA/KS ratios for<br>
1656 bases of coding region are as follows:<br><br>
The wide range of KA/KS ratios is expected due to the amount of cross breeding<br>
among the O. sativa strains. Some resemble O. rufipogon because of cross breeding between<br>
O. rufipogon with the domesticated strains.<br>
The deduced protein sequence of O. sativa strain Nipponbare was used to perform a<br>
BLAST search. A very strong protein BLAST hit to Arabidopsis thaHandPTR2-B (histidine<br>
transporting protein, NP_178313) (SEQ ID NO: 170) suggests that only about 30 codons of<br>
CDS are missing from the rice sequence (Figure 8).<br>
Homology search results suggest that the EG1117 gene codes for a protein that is very<br>
similar to a family of peptide transport proteins that is found in a wide range of species<br>
including fungi, plants, insects and mammals. (See Koh, et al. (2002) Arabidopsis, Plant<br>
Physiol. 128:21-29; Hauser, et al. (2001) Mol Membr. Biol. 18:105-12; Hauser, et al. (2000)<br>
J. Biol. Chem. 275:3037-42; Lubkowitz, et al. (1997) Microbiology 143-387-96; Steiner, et al.<br>
(1995) Mol. Microbiol 16:825-34). EG1117 codes for a protein of 577 amino acids, that<br>
appears to have 12 putative transmembrane domain regions. KA/KS analysis of the EG1117<br>
suggests that at least a portion of the EG1117 gene was strongly selected during the<br>
domestication of rice.<br><br><br>
It is clear that this particular protein is unique even though it shows an apparent<br>
structural homology to a large number of well characterized peptide transport proteins<br>
(Steiner, et al. (1995)). The sequence appears to encode the predicted twelve transmembrane<br>
domains characteristic of this femily of proteins. The EG1117 protein clearly has homology<br>
with not only peptide transport proteins, but also the oligopeptide transport proteins and<br>
nitrate transport proteins (Lubkowitz, et al. (1997); Lin, et al. (2000) Plant Physiol. 122:379-<br>
88; West, et al. (1998) Plant J. 15:221229). There is no homology to other types of transport<br>
proteins.<br>
Peptide transport proteins are integral membrane proteins that typically contain twelve<br>
transmembrane domains in the case of the di/tripeptide transporters and may contain between<br>
twelve and fourteen transmembrane domains in the case of oligopeptide transporters. The<br>
peptide transport, protein family (PTR family) has been extensively studied in yeast and<br>
plants. Typically, these proteins aid in the transport of di/tripeptides or oligopeptides across a<br>
cell membrane in a proton-dependent fashion. These carriers couple peptide movement<br>
across the membrane to movement of protons down an inwardly directed electrochemical<br>
proton gradient allowing the transport of peptides to occur against a substrate gradient<br>
(Nakazono, et al. (1996) Curr. Genet. 29:412-16; Matsukura, et al. (2000) Plant Physiol<br>
124:85-93; Toyofuku, et al., (2000) Plant Cell Physiol. 41:940-47; Hirose, et al. (1997) Plant<br>
Cell Physiol. 38:1389-1396; Horie, et al. (2001) Plant J. 27:129-38). Peptide transporters<br>
typically carry out sequence independent transport of all possible di- and tripeptides. All<br>
show stereoselectivity with peptides containing L-enantiomers of amino acids possessing a<br>
higher affinity for binding than peptides containing D-enantiomers. Currently, it is not<br>
possible to relate the structure of the various transporters to their substrate specificity or to<br>
their affinity.<br>
Many different peptide transport proteins have been identified in a variety of species.<br>
Global alignments of these proteins allowed researchers to identify motifs in the primary<br>
amino acid sequence that are typically found in all members of this family. In PTR-2<br>
members of the peptide tranporters family, a TYING" motif, named for the conserved F-Y-x-<br>
x-I-N-x-G-S-L residues in the fifth transmembrane domain (TMD5) and either a W-Q-I-P-Q-<br>
Y motif or a E-x-C-E-R-F-x-Y-Y-G motif in transmembrane domain 10 (TMD 10) have been<br>
identified (Becket, et al. (2001) in PEPTIDES: THE WAVE OF THE FUTURE , M. Lebl and R.A.<br>
Houghten, eds. American Peptide Society, 957-58). Interestingly, site directed mutagenesis<br>
of the FYING motif in S. cerevisiae results in attenuated growth on dipeptides, decreased<br>
sensitivity to toxic dipeptides and an elimination of radiolabeled dileucine. These data<br><br><br>
suggested that the FYING motif plays a crucial role in substrate recognition and/or<br>
translocation.<br>
In the case of plants, there is evidence in the literature that peptide transporters are not<br>
only important in the nutritional uptake of peptides and nitrate, but also that these transporters<br>
affect the responses to auxins, pathogenic toxins and other developmental processes. In the<br>
case of mArabidopsis peptide transporter, AtPRT2, it was demonstrated that root growth was<br>
affected by toxic ethionine-containing peptides thought to be transported by this particular<br>
transport protein (Steiner, et al. (1994) Plant Cell 6:1289-99). In later studies, it was shown<br>
that either the over expression or inhibition of expression of the AtPTR2-B protein by<br>
recombinant expression of sense or antisense constructs of the AtPTR2-B gene resulted in<br>
delayed flowering and arrested seed development in transgenic Arabidopsis plants (Song, et<br>
al. (1997) Plant Physiol. 114:927-935). This suggests that peptide transporters can have a<br>
very profound effect on both the growth and development of plants. -<br>
Further analysis of the putative EGl 117 peptide transporter demonstrates that a<br>
FYING motif is indeed present in TMD5 of EGl 117 compared to other representative plant<br>
PTR-2 type proteins. In addition, the EGl 117 has a WQVPQY motif in TMD10 identical to<br>
the other representative plant PTR-2 type proteins. The multiple sequence alignments created<br>
by the DIALIGN local alignment program (Morgenstern (1999) Bioinformatics 15:211-218,<br>
demonstrate that there is nearly 95% alignment of the diverse PTR-2 type plant protein<br>
sequences with the rice EGl 117 protein with about 70% homology at the amino acid level. In<br>
the O. sativa and O. rufipogen proteins, there are only three non-synonymous amino acid<br>
replacements. These replacements are structurally significant replacements that may<br>
dramatically alter the function or specificity of the putative peptide transport protein. In one<br>
case, we have a change from a glutamine (polar uncharged) to a histidine (basic) amino acid.<br>
At the other two positions, we see a change from the acidic aspartic acid to an uncharged<br>
glycine and a change from a acidic glutamic acid to an uncharged glycine. In general, all<br>
three changes shift towards a more basic charge profile.<br>
EXAMPLE IS; Mapping EGl 117<br>
, EGl i 17 was then physically mapped in rice. The DIG protocol (BMB-Roche PCR<br>
DIG Probe Synthesis Kit cat #1636090) successfully labeled a unique EGl Vl7 657bp PCR<br>
product (primers: 5'- TCCTGCATCCCTCTCAACTT -3' and 5'-<br>
GGATTGGATTCGATGAATGT -3') to screen against rice BAC filters from Clemson<br>
University. The blots were definitively detected using chemiluminescence as per the DIG<br><br><br>
protocol (BMB-Roche DIG Luminescent Detection Kit: cat #1636090). Two different O.<br>
sativa libraries (OSJNBa and OSJNBb) were screened for a total of 2 different filters. Below<br>
are the BACs identified by both screens:<br><br>
Rice contig 58 is on chromosome 3 and contains 181 BACs and 1S markers. EG1117<br>
maps to the same BACs as markers CD01387, C236, C875, R2778 and R2015. These all<br>
map to 35.8 cM on map 3RJ98. This marker is mapped to several different rice maps, as<br>
accessed through the RiceGenes or Gramene website. There are also several QTLs mapped<br>
to this region. One well-documented QTL for 1000-grain weight is in this region of<br>
chromosome 3 and is associated with marker RZ672 (McCouch, S.R. et al. Genetics 150:899-<br>
909). On one map CD01387 maps to 30.4 cM and RZ672 maps to 39 cM, and both of these<br>
markers map to four other rice maps in similar ranges. This region of rice has also been<br>
involved in some studies between rice and maize that indicate synteny between rice<br>
chromosome 3S and maize chromosomes IS and 9L (Wilson, W.A. et al. Genetics 153(1):<br>
453-473).<br>
EXAMPLE 16: Relationship of EG307 and EG1117<br>
EG1117 and the previously described gene, EG307 map to the same Clemson BAC<br>
contig, 58. EG1117 lies towards the end of the p-arm about 3 cM upstream of EG307.<br>
EG1117 maps to the same BACs as many of the markers on contig 58, and EG307 maps to<br>
the same contig, but has no markers directly mapped to its positive BACs.<br>
A separate analysis was undertaken using data from a published YAC map for rice<br>
from the Rice Genome Project (RGP), a joint project of the National Institute of<br>
Agrobiological Sciences (NIAS) and the Institute of the Society for Techno-innovation of<br><br><br>
Agriculture, Forestry and Fisheries (STAFF) and a part of the Japanese Ministry of<br>
Agriculture, Forestry and Fisheries (MAFF) Genome Research Program.<br>
The RGP database puts these two genes (EG1117 and EG307) 2 cM apart on<br>
chromosome 3. This YAC map has been accepted for publication in Plant Cell (Wu, J., et al.,<br>
2002 Plant Cell, prepublication copy). Upon a BLAST search, (see above), EG1117 hit<br>
AU055884 and AU055885. Both of these GenBank EST entries come from clone S20126<br>
that maps to YACs Y2533 and Y5488. These YACs are anchored with SI0968, which maps<br>
to Chromosome 3 at 33.5 cM.<br>
The unexpected proximity of these two genes suggests a possible functional link.<br>
EG307 and EG1117 may work together to increase yield. We speculate that EG307 may be a<br>
transcription factor for EG1117, thus creating a plant operon. All indications are that both<br>
EG307 and EG1117 are logical candidates for genes that would have an impact on<br>
agriculturally important traits based upon: 1) the KA/KS analysis on rice domesticated and<br>
ancestral species, 2) linkage to a grain weight QTL, and 3) an evolutionary pattern of amino<br>
acid replacements between ancestral and domesticated species. EG1117 also shows evidence<br>
for a strong positive selection during domestication based upon the KA/KS analysis in rice.<br>
EG1117 codes for a protein homologous to a family of peptide transporters. Other members<br>
of this family have been shown in plants to influence growth, flowering and seed<br>
development. EG1117 is also linked to the QTL for grain weight. It is highly unlikely that<br>
this is a coincidence. These are ideal genes to use in the aims of this proposal to both validate<br>
the genes as agriculturally relevant.<br>
EXAMPLE 17; Validation of Yield candidates: Association Analysis &amp; Pedigree<br>
Analysis<br>
The role of EG307 and EG11J 7 in controlling yield in the cereals can be validated by<br>
creation of transgenic plants, as described elsewhere in this patent; additional validation<br>
support comes from association analysis and pedigree analysis.<br>
Association analysis involves sequencing each candidate gene in a large number of<br>
well-characterized rice strains to learn if the genes are associated with known traits. EG307<br>
was sequenced in 13 well-characterized modern rice strains and it was determined that the<br>
derived, positively-selected allele is present in each of the 9 highest yielding strains, while the<br>
ancestral allele is present in the 4 lowest yielding strains. The pattern observed by<br><br><br>
examination of Table 12 is quite striking. This adds to the evidence that EG307 does<br>
influence yield, i.e., that it may be a so-called "yield" gene.<br><br>
Pedigree analysis takes advantage of two important sets of data. In addition to the<br>
available grain weight data, the derivation of many rice strains (/.&amp;, in pedigrees) is well<br>
known. This allows a validation scheme in which yield-related candidate genes are plotted<br>
onto known rice strain pedigrees. For each strain, the known 1000-grain weight and the type<br>
of allele (i.e., the "derived", adapted, modern allele) of EG307 and EG1117 are noted. The<br>
pattern of transmission of the adapted allele can be inferred from these data.<br>
Example 18. Identification of EG1117 in maize and teostnte.<br>
- Using methods well known to those skilled in the art described in Example 13,<br>
EG1117 was amplified from a number of maize strains (Zea mays mays) SEQ ID NOs 119,<br>
122,123, 124,127,128,129,132,135,136,137,140,141, 144,145,146,149,150,151,<br>
154) and a number of teosinte strains (Zea mays parviglumis) (SEQ ID NOs 157,160,161,<br>
162,165,166,167).<br><br>
Example 19. Determination of the function of gene candidate EG307.<br>
To elucidate the function of the EG307 protein, the rice proteins it interacts with will<br>
be determined. This "guilt-by-association" approach is useful in situations where one wants<br>
to identify potential pathways or functions associated with the unknown protein (Editorial<br>
(2001) Nature 410). Two methods of determining interacting proteins include a global<br>
screening approach, such as the yeast two-hybrid approach, as well as a more direct approach<br>
using a recombinantly expressed form of the unknown protein to isolate interacting proteins<br>
based upon the affinity of their interaction. A brief outline of the experimental methods and<br>
design are presented for both methods.<br>
A. Yeast two-hybrid (YTH) screen. The YTH screening method for interacting<br>
proteins relies upon the creation of recombinant fusions of the protein of interest with one half<br>
of a transcription activation factor protein binding domain (the bait) and the use of a cDNA<br>
library of potential protein coding regions fused with the other half of the. transcription<br>
activation factor activation domain (target protein). If the bait interacts with a target protein,<br>
the two halves of the transcription factor (binding domain and activation domain) are brought<br>
together and one gets initiation of transcription of a reporter gene. There are two basic types<br>
of YTH systems typically used, a GAL4 based system for standard YTH (Fields, et al. (1989)<br>
Nature 340:2445-246) and a LexA based "Interaction-Trap" (IT) method (Golemis, et al.<br>
(1997) in CURRENT PROTOCOLS IN MOLECULAR BIOLOGY F.M. Asubel, et al., eds., John<br>
Wiley &amp; Sons, NY; Golemis, et al (1997) in CELLS: A LABORATORY MANUAL D.L. SpSeptor,<br>
R. Goldman, and L. Leinwand, eds., Cold Spring Harbor Laboratory Press).<br>
Two rice YTH cDNA libraries (L cv Mil-Yang) are available commercially from<br>
Eugentech, Inc. (Yusong Taejon, Korea). These libraries are created in the Stratagene<br>
HybriZAP® (GAM based system) from cDNA created using mRNA isolated from rice<br>
developing spikes that are either 2 cm in length. These libraries should encode<br>
proteins important in both early and late embryonic development. Importantly, we know from<br>
our RT-PCR analysis of EG307 expression, that the EG307 protein should be present in these<br>
tissues. Therefore, these libraries will likely express proteins that interact with the EG307<br>
protein.<br>
Experimental Details. Standard reagents, yeast strains, vectors and DNA<br>
isolation/sequencing specific for the HybriZAP YTH system will be obtained from<br>
Stratagene. The coding region of EG307 will be cloned using an RT-PCR amplification of O.<br>
sativa shoot mRNA. The PCR amplified insert will be cloned into a linearized pBD-GAL4<br>
Cam phagemid vector and transformants carrying inserts will be selected on chloramphenicol<br><br><br>
plates to create the "bait" plasmid., The cloning junctions and coding region of EG307 will be<br>
sequenced using standard sequencing techniques at EG to ensure usage of the proper reading<br>
frame and that no mutations have been introduced during amplification of EG307.<br>
Both commercial libraries from Eugentech are reported by the company to be single<br>
round amplified from the primary library and are supplied at 2 x 108 pfu. Library I ( 
spike) has an initial complexity of lxlO6 pfu and an amplified library titer of 3.6 x 108 pfu/ml.<br>
Insert sizes in library I range from 0.5 to 3.0 kb. Library II (&gt;2cm spike) has an initial<br>
complexity of 5 x 10s pfu and an amplified library titer of 4x106 pfu/ml. Insert sizes in library<br>
II range from 0.5 to 1.6 kb. Since these premade libraries are commercially available at a<br>
very reasonable cost, it is prudent to do an initial YTH hunt using this particular system.<br>
Both the bait plasmid and target plasmid library will be co-transfected into yeast strain<br>
YRG-2 using Stratagene's YRG-2 Yeast competent cell library kit. Yeast carrying both<br>
plasmids will be selected for by the complementation of the YRG-2 auxotrophic mutations.<br>
In this case the bait and target plasmids should complement both the tryptophan and leucine<br>
auxotrophy of the YRG-2 strain. Colonies that grow up from this co-transfection will be used<br>
to create a yeast library that will next be screened for interacting target proteins by further<br>
selection on plates lacking histidine and containing X-gal. The YRG-2 strain carries two<br>
additional reporter plasmids. One carries a GAM binding sequence upstream of a HIS gene<br>
and is used to complement the YRG-2 histidine auxotrophic mutation. The other plasmid<br>
carries a GAL4 binding sequence upstream of a LacZ reporter gene. This enables blue/white<br>
screening for reporter gene expression when the yeast are plated on X-gal containing plates.<br>
During the interaction screening phase, only yeast containing an interacting bait:target<br>
combination will complement the histidine auxotrophy and also cause the expression of LacZ<br>
and conversion of the X-gal substrate to a blue product. This double screen for interactions<br>
helps to limit the number of false positive colonies identified. In addition, to some degree tile<br>
intensity of the blue substrate production is some indication of the strength of the interaction<br>
between the bait and target proteins.<br>
Interacting colonies will be picked, the DNA will be isolated, and sequence the target<br>
plasmid inserts from several hundred colonies will be sequenced. Sequences will be<br>
translated and searched against protein sequences, both full length coding regions and<br>
potential open reading frames from ESTs. When multiple identical sequences are identified<br>
as targets, it is likely that the protein has been preferentially selected and represents an<br>
interacting protein. If a sequence is only represented once or a few times, it is either a non-<br><br>
specific interacting protein, or a transcript represented a limited number of times in the cDNA<br>
library.<br>
Multiple classes of interacting proteins should be identified this way. Ideally, proteins<br>
of known function will be highly represented and a logical function or pathway easily<br>
identified. If the interacting protein(s) are unknown, but homologous to known proteins, it<br>
may still be possible to design experiments to confirm the relevance of the interaction based<br>
on known information in the public domain.<br>
Suspected protein-protein interactions will be validated by additional in vitro and in<br>
vivo studies. Simpler assays to confirm interactions will be performed during Phase I if time<br>
permits. Assays such as affinity pull-downs and far-westerns will be performed as additional<br>
reagents such as antibodies and recombinant proteins are made. Generation of recombinant<br>
proteins for both the bait protein (EG3Q7) as well as putative interacting proteins will be done<br>
as necessary as epitope tagged fusion proteins (GST, myc, V5, biotin tags). Additional<br>
evidence for relevant in vivo interactions, such as fluorescence energy transfer between two<br>
appropriately constructed green fluorescent protein fusions, may be necessary to definitively<br>
prove an in vivo interaction and also measure factors that might influence that interaction.<br>
However, such experiments are clearly beyond the scope of Phase I.<br>
It is possible that the YTH hunt will identify that the bait protein itself binds to the<br>
GAM transcription activation sequence and causes activation of the reporter systems. If this<br>
occurs, two bait constructs expressing the two halves of the EG307 protein independently will<br>
be constructed. These constructs would be tested for direct activation of GAL4 reporters in<br>
YRG-2. If negative for direct activation, the cDNA target library would be rescreened for<br>
baitrtarget interactions.<br>
If no interacting proteins are identified from the commercial pre-made YTH libraries,<br>
it is possible that these libraries are of poor quality or that the GAL4 YTH system is not<br>
sensitive enough to identify the actual interacting proteins. In either case, having alternative<br>
libraries constructed in the interaction-trap system (LexA) can be considered. These libraries<br>
would then serve as a basis for further characterization of any other unknown candidate<br>
proteins.<br>
Example 20. Direct isolation and identification of interacting proteins using physical<br>
methods.<br>
To directly isolate interacting proteins from plant tissues, affinity isolation of proteins<br>
present in various solubilized plant tissues will be performed. Isolated interacting proteins<br>
will be subjected to limited proteolysis and the resulting peptide fragments will be analyzed<br><br><br>
by mass spectral analysis to identify whether peptide fragment patterns indicative of known or<br>
predicted proteins are produced.<br>
The EG307 protein will be cloned into a bacterial expression system to generate a<br>
GST-EG307 fusion protein using Pharmacia's pGEX-5X-l (Amersham Pharmacia). Bacterial<br>
lysates of the cultures where expression was induced by IPTG, will be used to purify the<br>
fusion protein on glutathione sepharose beads. The soluble protein will be eluted from the<br>
beads by competition for binding to the solid phase by passage of free glutathione over the<br>
column. The recombinant GST-EG307 can then be used as an affinity ligand when recoupled<br>
to fresh glutathione sepharose beads. Alternatively, free EG307 protein can also be obtained<br>
by removal of the GST domain by treatment of the fusion protein with Factor Xa. There are<br>
no Factor Xa protease sites in the predicted EG307 protein.<br>
Plant cell lysates isolated from a large amount of O. sativa seedlings (200-300 grams)<br>
will be created by standard differential tissue disruption and clarification techniques. To<br>
isolate cytosolic proteins, the cells will be disrupted by mechanical shearing using a polytron<br>
tissue homogenizer and sonicator while keeping the tissue lysate cold in the presence of a<br>
protease inhibitor cocktail. The soluble cytosolic lysates will be clarified by differential<br>
centrifugation at low speed to remove debris, followed by high speed centrifugation to<br>
remove insoluble aggregated protein. To isolate proteins in the insoluble fraction, various<br>
detergents such as NP-40, Brij® 35, and deoxycholate will be used to sohiblize membrane<br>
bound proteins isolated from these insoluble membrane fractions. Insoluble material will be<br>
removed by centrifugation.<br>
The plant cell lysates will be individually passed over the glutathioine-sepharose:GST-<br>
EG307 beads. The beads will be washed with buffers of various ionic strengths to remove<br>
weakly bound protein. Bound proteins will then be eluted with either low pH, high salt or<br>
denaturing detergents. Pure proteins will be run on an SDS-PAGE gel and the bands will be<br>
stained with a mass spectroscopy compatible silver staining reagent or commassie blue.<br>
Bands of interest will be cut out of the gels and frozen for later analysis. These proteins will<br>
be sent to a Mass Spectroscopy facility for limited proteolysis followed by mass spectroscopy<br>
to determine the proteolytic peptide signature and identity of the interacting protein.<br>
This technique should allow for the identification of the interacting proteins as long as<br>
the affinity of the interaction is specific and strong enough to ensure a tight binding between<br>
the EG307 protein and the potential interacting protein. These data would then allow for the<br>
identification of the interacting protein if that protein is homologous to other proteins. It is<br>
clearly possible that no proteins will be identified by this method because of a lack of affinity<br><br><br>
for the EG307 protein. Alternatively, it is possible that no interacting proteins are present in<br>
the lysates generated by the methods outlined above.<br>
If too many proteins appear to bind to the glutathione-seph:GST-EG307 beads, it is<br>
possible that either those proteins are non-specifically binding to either the sepharose,<br>
glutathione, glutathioine synthestase or to an artificial epitope generated by creating the N-<br>
terminal GST fusion with EG307. To eliminate some of these non-specific interactions,<br>
lysates will be pre-cleared with sepharose beads alone, glutathione-sepharose as well as an<br>
irrelevant GST-fusion protein coupled to glutathione beads. If the nonspecific bands remain<br>
following the pre-clearance steps, more stringent washing and binding conditions such as<br>
higher salt, lower salt, increased or decreased pH, addition of non-ionic detergents such as<br>
Tween-20, will be employed to restrict the proteins that bind to this bait protein.<br>
Example 21. Determine the function of gene candidate EG1117 coding for a novel<br>
protein with a putative peptide transport function.<br>
Since the EG 1H 7 protein likely encodes a form of peptide transport protein based<br>
upon the in silico homology data suggesting it is a member of the PTR-2 family of protein,<br>
experiments that directly assess this particular function will be carried out. Two<br>
complementary, but independent approaches will be used. First, a method of examining<br>
heterologous peptide transport protein functions in yeast will be used to examine the ability of<br>
both the domesticated and ancestral form of EG1117 to transport peptides across the cell<br>
membrane and complement auxotrophic amino acid requirements in PTR-2 deletion mutants<br>
of yeast. Second, the growth characteristics of both the domesticated and ancestral species of<br>
rice seedlings in the presence of toxic ethionine-containing peptides will be used to correlate<br>
the potential peptide transport protein mutations with a measurable phenotypic difference<br>
between the domesticated and ancestral species of rice.<br>
Complementation analysis of heterologous peptide transport functions in auxotrophic<br>
yeast. For these studies, we will take advantage of the previously described methods used, to<br>
identify novel Arabidopsis peptide transport proteins using specifically designed auxotrophic<br>
mutant yeast strains that also carried a mutation in their ability to transport di-/tripeptides<br>
(Steiner, et al., 1994). This heterologous system demonstrated that plant peptide transport<br>
proteins could be cloned into yeast cells and the recombinantly expressed protein function<br>
could be measured by the complementation of the auxotrophic amino acid requirements of the<br>
yeast strain. This is a simple, but powerful assay that will quickly yield information about the<br>
function of both forms of EG1117.<br><br><br>
Parental yeast strain BY4742 [Mata, his3-, leu2-, lys2-, ura2-] and the YKR093W<br>
ORF deletion mutant from the "complete yeast deletion array collection" available from<br>
ATCC in which the PTR-2 gene is deleted, designated BY4742-ptr2. Both strains are<br>
available from ATCC. The domesticated and ancestral forms of EG1117 will be cloned into<br>
the pYES2.1-TOPO-TA vector (Invitrogen, Inc.) which allows for the recombinant expression<br>
of the putative PTR-2 proteins in the B Y4742-ptr2 auxotrophic strain of yeast. Selection of<br>
transfectants will be performed on plates lacking uracil. Plasmid DNA will be reisolated from<br>
the transfectants and analyzed by EG 1117 specific primers to confirm that the strain carries<br>
the appropriate plasmid. Protein expression is controlled by the presence of galactose in the<br>
media. Transfectants will be grown in media containing galactose and EG 1117 protein<br>
expression will be monitored by western blot analysis of the C-terminal V5 epitope tag added<br>
by the vector. The following shows peptides used in complementation and root growth<br>
assays: <br>
Eth = ethionine, is a toxic derivative of methionine. All selection for auxotrophic phenotype<br>
will be done on plates lacking leucine.<br>
To test whether EG1117 codes for a peptide transport protein, the peptides listed<br>
above will be synthesized and purified commercially. Since each peptide carries a leucine, if<br>
the peptide is transported into the leucine auxotrophic strain BY4742-ptr2 transfected with a<br>
functional peptide transport protein and grown in the presence of galactose, that strain should<br>
be able to grow.<br>
A second assay that will be performed is a inhibition assay. In this case, the BY4742-<br>
ptr2 EG307 trarisfectants as well as the BY4742 parental and BY4742-ptr2 deletion mutants<br>
as controls, will be plated as a lawn on YPG (yeast extract, peptone, galactose) plates and the<br>
toxic ethionine-peptide derivatives will be spotted onto membrane discs and placed on the<br>
yeast lawns (Steiner, et al., 1994). Zones of clearing around the disc would then indicate that<br>
the yeast expressed a functional transport protein the allowed the yeast to transport the toxic<br>
peptide into the cell, killing the ceil.<br>
If both the domesticated and ancestral forms of the EG1117 protein complement the<br>
amino acid auxotrophic mutations suggesting they both are functional transport proteins, the<br><br><br>
following experiments will be performed. To assess the pH or cation dependence, the pH or<br>
ionic strength of the plating media will be varied and the ability of the transfectants carrying<br>
the EG 1117 proteins to grow in the presence of complementary peptides or die in the<br>
presence of the toxic peptides will be determined. Likewise, to assess the potential differences<br>
in affinity for peptide, the dose response effects a specific peptide or toxic variant of that<br>
peptide on growth of the yeast transformants will be assessed.<br>
It is very likely that we will find that the EG1117 protein indeed codes for a peptide<br>
transport protein. It is also likely that the two forms of the protein will display a measurable<br>
difference in this function, perhaps a change in specificity/selectivity, pH optimum or affinity<br>
will be evident Although the non-synonymous changes in the amino acid sequence from<br>
acidic to more basic characteristics are present, it is possible that these alterations are in an<br>
unimportant region of the protein. Alternatively, it is possible that these changes do not alter<br>
the 3-dimensional structure of the protein sufficiently to alter its function.<br>
It is unlikely that these proteins do not transport peptides, however it is possible that<br>
the EG1117 protein might be a transport protein for some other substrate. In this case, in the<br>
absence of any evidence that the EG1117 transports peptides, it may be possible to use the<br>
same system or at least the transfected yeast to sort out some of these other functions. For<br>
example, testing for monosaccharide or polysaccharide transport ability would be possible in<br>
the appropriate auxotrophic strains. Alternatively, other yeast deletion mutants for targeted<br>
transport functions could be mated with the existing transfectants. In this case, growth of the<br>
mated yeast on the selection plates would be indicative of complementation of that particular<br>
deletion mutation. Using this strategy, it would be possible to scan a large number of<br>
different yeast deletion mutants publicly available (Brachman, et al. (1998) Yeast 14:115-132.<br>
Example 22. Differential sensitivity of ancestral and domesticated rice seedlings to<br>
ethionine-containing toxic peptides.<br>
These studies represent an attempt to directly demonstrate in vivo that the EG1117.<br>
protein functions differently in the domesticated and ancestral strains of rice. The ability to<br>
transport toxic peptides results in death of the cells that take up the toxic peptides. A lack of<br>
function might be expressed as resistance to the effects of the toxic peptide on the continued<br>
growth of the seedling roots. A lack of a phenotypic difference would suggest either that EG<br>
1117 has not been expressed, or mat other transport proteins compensate for the altered<br>
function of the selected EG 1117 proteins.<br>
. A modification of the method used for Arabidopsis will be used in these studies<br>
(Steiner, et al., 1994). Rice seeds from 0. sativa and 0. rufipogen will be sprouted and then<br><br><br>
the seedlings are allowed to continue to grow on rice media in a dark, moist container. The<br>
seedlings will be exposed to discs impregnated with ethionine-containing toxic peptides or<br>
non-toxic peptides as a control. Initial experiments will focus on determining if dramatic<br>
differences in sensitivity to the toxic peptides exists. If no dramatic differences are observed,<br>
additional experiments using a dose range of toxic peptides will be used in a larger<br>
experiment to determine if a difference sensitivity to the toxic peptide dose exists. This<br>
would be suggestive of a difference in functional activity of the peptide transporters present it<br>
the two strains of rice.<br>
The results from this set of studies will depend upon whether there are other peptide<br>
transport mechanisms that compensate for the hypothesized differences in the EG1117<br>
encoded protein. As long as EG1117 is the primary peptide transport protein used by rice or<br>
has a unique function that is critical to the growth of the rice plant, any differences in<br>
EG1117's function between the domesticated and ancestral forms should be manifested by<br>
differences in susceptibility to the toxic peptides. Similar experiments in Arabidopsis<br>
successfully demonstrated that a single PTR-2 protein was part of a single peptide transport<br>
system (Steiner, et al., 1994). Therefore, mutants of the single peptide transporter yielded<br>
dramatic results on growth inhibition. In particular, these studies revealed that a deletion of<br>
the PTR-2 protein or a lack of PTR-2 protein expression due to a developmental block in<br>
PTR-2 expression in the early embryo, resulted in resistance of the plant to the presence of<br>
toxic peptides in the surrounding media. It is likely that rice seedling growth will be similar<br>
and easily reveal any differences in function of EG 1117 by alterations in rice seedling<br>
growth.<br><br>
WE CLAIM :<br>
1.	An isolated polynucleotide which includes a promoter operably linked to a polynucleotide<br>
that encodes an EG307 gene in plant tissue, said polynucleotide selected from the group<br>
consisting of: a) a polynucleotide selected from the group consisting of SEQ ID NO:4, SEQ ID<br>
NO:5, SEQ ID NO:91, SEQ ID NO:33, SEQ ID NO:34, and SEQ ID NO:35; b) a polynucleotide<br>
having at least 95% sequence identity to a polynucleotide of a), wherein the presence of said<br>
polynucleotide is a marker of increased yield in a plant of the genus Oryza or Zea; c) a<br>
polynucleotide encoding a polypeptide comprising SEQ ID NO:6 or SEQ ID NO:36; and d) a<br>
polynucleotide encoding a polypeptide having at least 95% sequence identity to SEQ ID NO:6 or<br>
SEQ ID NO:36, wherein the presence of a polynucleotide encoding a polypeptide of (d) is a<br>
marker of increased yield in a plant of the genus Oryza or Zea.<br>
2.	The isolated polynucleotide as claimed in claim 1, wherein said polynucleotide is a<br>
recombinant polynucleotide.<br>
3.	The isolated polynucleotide as claimed in claim 1, wherein the promoter is the promoter<br>
native to an EG307 gene.<br>
4.	A method of identifying an agent which may modulate yield, said method comprising<br>
contacting at least one candidate agent which may modulate function of an EG307 polynucleotide<br>
or polypeptide with a plant or cell comprising an EG307 gene, wherein the agent is identified by<br>
its ability to modulate yield, and wherein said EG307 gene comprises a polynucleotide selected<br>
from the group consisting of: a) a polynucleotide selected from the group consisting of SEQ ID<br>
NO:4, SEQ ID NO:5, SEQ ID NO:91, SEQ ID NO:33, SEQ ID NO:34, and SEQ ID NO:35; b) a<br>
polynucleotide having at least 95% sequence identity to a polynucleotide of a), wherein the<br>
presence of said polynucleotide is a marker of increased yield in a plant of the genus Oryza or<br>
Zea; c) a polynucleotide encoding a polypeptide comprising SEQ ID NO:6 or SEQ ID NO:36;<br>
and d) a polynucleotide encoding a polypeptide having at least 95% sequence identity to SEQ ID<br>
NO:6 or SEQ ID NO:36, wherein the presence of a polynucleotide encoding a polypeptide of (d)<br>
is a marker of increased yield in a plant of the genus Oryza or Zea.<br><br>
5.	The method as claimed in claim 4, wherein the plant or cell is transfected with a<br>
polynucleotide of a), b), c), or d).<br>
6.	The method as claimed in claim 4, wherein said identified agent modulates yield by<br>
modulating a function of the polynucleotide encoding the polypeptide.<br>
7.	The method as claimed in claim 4, wherein said identified agent modulates yield by<br>
modulating a function of the polypeptide.<br>
8.	A method of producing an EG307 polypeptide comprising: a) providing a cell transfected<br>
with a polynucleotide encoding an EG307 polypeptide positioned for expression in the cell; b)<br>
culturing the transfected cell under conditions for expressing the polynucleotide; and c) isolating<br>
the EG307 polypeptide, wherein said polypeptide is selected from the group consisting of: i) a<br>
polypeptide encoded by a polynucleotide selected from the group consisting of SEQ ID NO:4,<br>
SEQ ID NO:5, SEQ ID NO:91, SEQ ID NO:33, SEQ ID NO:34, and SEQ ID NO:35; ii) a<br>
polypeptide encoded by a polynucleotide having at least 95% sequence identity to a<br>
polynucleotide in i), wherein the presence of said polynucleotide is a marker of increased yield in<br>
a plant of the genus Oryza or Zea; iii) a polypeptide comprising SEQ ID NO:6 or SEQ ID NO:36;<br>
and iv) a polypeptide having at least 95% sequence identity to a polypeptide of iii), wherein the<br>
presence of a polynucleotide encoding a polypeptide of (d) is a marker of increased yield in a<br>
plant of the genus Oryza or Zea.<br><br>
The present invention provides methods for identifying polynucleotide and polypeptide sequences which may be<br>
associated with commercially or aesthetically relevant traits in domesticated plants or animals. The methods employ comparison of<br>
homologous genes from the domesticated organism and its ancestor to identify evolutionarily significant changes and evolutionarily<br>
neutral changes. Sequences thus identified may be useful in enhancing commercially or aesthetically desirable traits in domesticated<br>
organisms or their wild ancestors.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTk0LUtPTE5QLTIwMDQtQ09SUkVTUE9OREVOQ0UgMS4xLnBkZg==" target="_blank" style="word-wrap:break-word;">994-KOLNP-2004-CORRESPONDENCE 1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTk0LUtPTE5QLTIwMDQtQ09SUkVTUE9OREVOQ0UucGRm" target="_blank" style="word-wrap:break-word;">994-KOLNP-2004-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTk0LUtPTE5QLTIwMDQtRk9STSAyNyAxLjEucGRm" target="_blank" style="word-wrap:break-word;">994-KOLNP-2004-FORM 27 1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTk0LUtPTE5QLTIwMDQtRk9STSAyNy5wZGY=" target="_blank" style="word-wrap:break-word;">994-KOLNP-2004-FORM 27.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTk0LWtvbG5wLTIwMDQtZ3JhbnRlZC1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">994-kolnp-2004-granted-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTk0LWtvbG5wLTIwMDQtZ3JhbnRlZC1hc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">994-kolnp-2004-granted-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTk0LWtvbG5wLTIwMDQtZ3JhbnRlZC1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">994-kolnp-2004-granted-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTk0LWtvbG5wLTIwMDQtZ3JhbnRlZC1jb3JyZXNwb25kZW5jZS5wZGY=" target="_blank" style="word-wrap:break-word;">994-kolnp-2004-granted-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTk0LWtvbG5wLTIwMDQtZ3JhbnRlZC1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">994-kolnp-2004-granted-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTk0LWtvbG5wLTIwMDQtZ3JhbnRlZC1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">994-kolnp-2004-granted-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTk0LWtvbG5wLTIwMDQtZ3JhbnRlZC1leGFtaW5hdGlvbiByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">994-kolnp-2004-granted-examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTk0LWtvbG5wLTIwMDQtZ3JhbnRlZC1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">994-kolnp-2004-granted-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTk0LWtvbG5wLTIwMDQtZ3JhbnRlZC1mb3JtIDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">994-kolnp-2004-granted-form 13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTk0LWtvbG5wLTIwMDQtZ3JhbnRlZC1mb3JtIDE4LnBkZg==" target="_blank" style="word-wrap:break-word;">994-kolnp-2004-granted-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTk0LWtvbG5wLTIwMDQtZ3JhbnRlZC1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">994-kolnp-2004-granted-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTk0LWtvbG5wLTIwMDQtZ3JhbnRlZC1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">994-kolnp-2004-granted-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTk0LWtvbG5wLTIwMDQtZ3JhbnRlZC1ncGEucGRm" target="_blank" style="word-wrap:break-word;">994-kolnp-2004-granted-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTk0LWtvbG5wLTIwMDQtZ3JhbnRlZC1yZXBseSB0byBleGFtaW5hdGlvbiByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">994-kolnp-2004-granted-reply to examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTk0LWtvbG5wLTIwMDQtZ3JhbnRlZC1zZXF1ZW5jZSBsaXN0aW5nLnBkZg==" target="_blank" style="word-wrap:break-word;">994-kolnp-2004-granted-sequence listing.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTk0LWtvbG5wLTIwMDQtZ3JhbnRlZC1zcGVjaWZpY2F0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">994-kolnp-2004-granted-specification.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="228099-a-py-rich-immunostimulatory-nucleic-acid.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="228101-imide-compound-and-optical-recording-medium-using-the-same.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>228100</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>994/KOLNP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>05/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>30-Jan-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>28-Jan-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>15-Jul-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>EVOLUTIONARY GENOMICS LLC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>BIOSCIENCE PARK CENTER, 12635 EAST MONTVIEW BOULEVARD, SUITE 211, AURORA, CO</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>MESSIER WALTER</td>
											<td>2418 BOWEN STREET, LONGMONT, CO 80501</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2003/01460</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-01-16</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/349,088</td>
									<td>2002-01-16</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/368,541</td>
									<td>2002-03-29</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>10/079,042</td>
									<td>2002-02-19</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>4</td>
									<td>60/349,661</td>
									<td>2002-01-17</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/228100-an-isolated-polynucleotide by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:40:14 GMT -->
</html>
