<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/228422-sets-of-digital-antibodies-directed-against-short-epitopes-and-methods-using-same by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:00:28 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 228422:SETS OF DIGITAL ANTIBODIES DIRECTED AGAINST SHORT EPITOPES, AND METHODS USING SAME</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">SETS OF DIGITAL ANTIBODIES DIRECTED AGAINST SHORT EPITOPES, AND METHODS USING SAME</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates generally to sets of at least about 15 digital antibodies directed against short epitopes consisting of 3, or 4, or 5 consecutive amino acids, wherein each digital antibody recognizes a plurality of distinct and different proteins that comprise the same epitope to which the digital antibody binds. The present invention also relates to methods for generating a protein binding profile of a sample, characterizing a test sample, determining presence or absence of a bacteria, virus, or cell in a sample, and identifying a test protein using a set of digital antibodies. The present invention also relates to a kit comprising a set of digital antibodies.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>[0001]	This application claims the priority benefit of provisional application U.S.<br>
Serial No. 60/418,277, filed October 15,2002, and Serial No.	, filed August 18,<br>
2003, the contents of both of which are incorporated by reference in their entirety.<br>
FIELD OF INVENTION<br>
[0002]	The present invention relates generally to sets of digital antibodies<br>
directed against short epitopes, and use thereof in methods for protein analysis.<br>
STATEMENT REGARDING FEDERALLY SPONSORED RESEARCH OR<br>
DEVELOPMENT<br>
[0003]	Not applicable.<br>
BACKGROUND OF THE INVENTION<br>
[0004]	Proteomics involves the measurement of gene activity at the protein level.<br>
Today, the most common tool for proteomics purposes is the combination of two-<br>
dimensional gel electrophoresis coupled with mass spectrometry (2D-MS). This system<br>
has several limitations. First, the detection sensitivity and resolution of 2D<br>
electrophoresis is low. Second, use of mass spectrometric analysis dramatically increases<br>
the cost. Finally, 2D electrophoresis is time-consuming. A very well-equipped<br>
laboratory can only perform about 200 to 400 2D gels each week. Accordingly, there is a<br>
need for improved protein-analysis methods.<br><br>
[0005]	Detection and characterization of bacterial or viral infection is of crucial<br>
importance in the practice of clinical microbiology and in environmental testing, such as<br>
food safety and biohazard safety testing. Microorganisms are very diverse in terms of<br>
both phenotype and genotype, for instance, staphylococci consist of 32 species and 15<br>
subspecies. Current diagnostic methods, however, are generally capable of detecting<br>
only a single microorganism or virus, necessitating the use of a number of specific tests<br>
in order to detect and characterize a microorganism or virus. Thus, there is a need for<br>
new methods for detection and characterization (including identification) of protein<br>
samples, including samples comprising or derived from bacteria and/or viruses.<br>
[0006]	Cancer can be classified based on tissue type and site of cancer. Each type<br>
of cancer can be further classified to different stages based on mostly tumor size and<br>
whether it has invaded other organ. For example, a prostate cancer may be classified into<br>
stages from TO to T4 using current methods. In another example, cancers can also be<br>
classified to different grades based on, e.g., structural organization of a tumor, and/or the<br>
level of cell differentiation. However these morphological and/or histological<br>
classifications often do not correlate well with clinical treatment, and frequently fail to<br>
identify early stage cancer or pre-cancerous cells. Thus there is a need for new methods<br>
for detection and characterization (including identification) of protein samples<br>
comprising or derived from cancerous cells and/or tissues.<br>
[0007]	All references cited herein, including patent applications and publications,<br>
are incorporated by reference in their entirety.<br>
SUMMARY OF THE INVENTION<br>
[0008]	The present invention provides sets of digital antibodies, and methods<br>
comprising use of digital antibodies.<br>
[0009]	Accordingly, in one aspect, the invention provides sets of digital<br>
antibodies, wherein the set comprises at least about 15 digital antibodies, wherein each<br>
digital antibody binds a different epitope, and wherein each digital antibody binds an<br><br>
epitope consisting of 3 consecutive amino acids, or 4 consecutive amino acids. In some<br>
embodiments, the set comprises at least about any of 20,30,40, 50, 60,70, 80, 90,100,<br>
110,125,150,175,200,300,400,500,600,700, 800,900,1000, or more antibodies. In<br>
some embodiments, the set comprises 100 digital antibodies that bind epitopes consisting<br>
of 3 consecutive amino acids. In other embodiments, the set further comprises 100<br>
digital antibodies that bind epitopes consisting of 4 consecutive amino acids. In still<br>
other embodiments, the set further comprises 100 digital antibodies that bind epitopes<br>
consisting of 5 consecutive amino acids. In other embodiments, the set of digital<br>
antibodies comprises at least about 100,200,300,400, 500, 600, 700 800,900, or 1000<br>
digital antibodies, wherein each digital antibody binds a different epitope, and wherein<br>
each digital antibody binds an epitope consisting of 3 consecutive amino acids, 4<br>
consecutive amino acids, or 5 consecutive amino acids. In still other embodiments, the<br>
set of digital antibodies comprises at least 1000 digital antibodies that bind epitopes<br>
consisting of 4 consecutive amino acids. In still other embodiments, the set further<br>
comprises at least 100 digital antibodies that bind epitopes consisting of 5 consecutive<br>
amino acids. In other embodiments, the set further comprises at least 100 digital<br>
antibodies that bind epitopes consisting of 3 consecutive amino acids. It is understood<br>
that "different epitopes" encompasses epitopes with overlapping amino acid sequences, as<br>
well as epitope with non-overlapping amino acid sequences.<br>
[0010]	In another aspect, the invention provides an array comprising any of the<br>
digital antibody sets described herein. Methods for generating arrays are well known in<br>
the art, and further described herein.<br>
[0011]	In another aspect, the invention provides methods for generating a protein<br>
binding profile said methods comprising (a) contacting a sample with any of the sets of<br>
digital antibodies described herein under conditions that permit binding; (b) optionally<br>
removing unbound protein (in some embodiments, removing protein that is not<br>
specifically bound); and (c) detecting binding of protein to antibodies, whereby a protein<br>
binding profile is generated.<br><br>
10012]	In another aspect, the invention provides methods for generating a protein<br>
binding profile, said methods comprising detecting protein bound to a set of digital<br>
antibodies, whereby a protein binding profile is generated, wherein the protein bound to<br>
the set of digital antibodies is generated by a methods comprising (a) contacting a sample<br>
with any of the sets of digital antibodies described herein; and (b) optionally removing<br>
unbound protein.<br>
[0013]	In another aspect, the invention provides methods for generating a protein<br>
binding profile, said methods comprising (a) separating unbound protein from a set of<br>
digital antibodies contacted with a sample; and (b) detecting binding of protein to<br>
antibodies, whereby a protein binding profile is generated, wherein the unbound protein<br>
from a set of digital antibodies contacted with a sample is generated by a method<br>
comprising contacting a sample with any of the sets of digital antibodies described<br>
herein.<br>
[0014]	As is evident, one or more steps may be combined and/or performed<br>
sequentially (often in any order, as long as the requisite product(s) are able to be formed),<br>
and, as is evident, the invention includes various combinations of the steps described<br>
herein. It is also evident, and is described herein, that the invention encompasses<br>
methods in which the initial, or first, step is any of the steps described herein. Methods<br>
of the invention encompass embodiments in which later, "downstream" steps are an<br>
initial step.<br>
[0015]	In another aspect, the invention provides methods for generating a protein<br>
binding profile, said methods comprising (a) incubating a reaction mixture, said reaction<br>
mixture comprising: (i) any of the sets of digital antibodies described herein; and (ii) a<br>
sample; wherein the incubating is under condition permitting binding; (b) optionally<br>
separating unbound protein; and (c) detecting binding of protein to antibodies, whereby a<br>
protein binding profile is generated.<br>
[0016]	m another aspect, the invention provides methods for generating a protein<br>
binding profile, said methods comprising detecting binding of protein to a set of digital<br>
antibodies, whereby a protein binding profile is generated, wherein the protein bound to<br><br>
the set of digital antibodies is generated by a methods comprising incubating a reaction<br>
mixture, said reaction mixture comprising: (i) any of the sets of digital antibodies<br>
described herein; and (ii) a sample; wherein the incubating is under condition permitting<br>
binding; (b) optionally separating unbound protein.<br>
[0017]	As is evident to one skilled in the art, aspects that refer to combining and<br>
incubating the resultant mixture also encompasses method embodiments which comprise<br>
incubating the various mixtures (in various combinations and/or subcombinations) so<br>
that the desired products are formed.<br>
[0018]	in some embodiments, the methods of generating a protein binding profile<br>
further comprise a step of treating the sample with a protein cleaving agent, whereby<br>
polypeptide fragments are generated. The sample can treated with a protein cleaving<br>
agent prior to step (a) of contacting the sample with the set of digital antibodies under<br>
conditions that permit binding. The protein cleaving agent may be an enzyme (such as<br>
chymotrypsin or trypsin) or a chemical agent (such as cyanogen bromide).<br>
[0019]	It is understood that a protein binding profile includes information<br>
regarding absence of binding between a digital antibody and protein. It is further<br>
understood that a protein binding profile may be generated without the need to have prior<br>
knowledge of the identity of the protein being analyzed, and both known and unknown<br>
proteins may be detected. Thus, in some embodiments, the protein binding profile may<br>
be used to identify and/or detect a previously unknown agent, such as a novel pathogen.<br>
[0020]	Detection may be qualitative and/or quantitative. In some embodiments,<br>
binding of at least about 95%, at least about 90%, at least about 75%, at least about 50%,<br>
at least about 30% of antibodies in a set is detected. In some embodiments, binding of<br>
each antibody in a set is detected. In some embodiments, presence or absence and/or<br>
amount of bound labeled protein is detected. In other embodiments, labeled competitor<br>
polypeptide is detected, as further described herein. Generally, in embodiments<br>
involving use of competitor polypeptides, the methods further comprise contacting the set<br>
of digital antibodies with competitor polypeptides (in combination with the sample and/or<br>
sequentially with the sample).<br><br>
[00211	In another aspect, the invention provides methods for generating libraries<br>
of protein binding profiles, comprising the steps of (a) contacting a sample with any of<br>
the sets of digital antibodies described herein under conditions that permit binding; (b)<br>
optionally removing unbound protein (in some embodiments, removing protein that is not<br>
specifically bound); (c) detecting binding of protein to antibodies, whereby a protein<br>
binding profile is generated; and (d) repeating steps (a) through (c) with at least two<br>
different samples.<br>
[0022]	In another aspect, the invention provides methods for generating libraries<br>
of protein binding profiles, comprising: compiling a set of two or more protein binding<br>
profiles, wherein the protein binding profiles are prepared according to any of the<br>
methods for generating protein binding profiles described herein.<br>
[0023]	In another aspect, the invention provides libraries of protein binding<br>
profiles, wherein the library is prepared using any of the methods described herein.<br>
[0024]	In some embodiments, the libraries may comprise at least about 5, 10,15,<br>
20,25, 30, 35, 40, 45, 50, 75, 100,150,200, 300,400, 500, 600, 700, 800, 900, or more<br>
(such as 1000,2000, 3000,4000 or more) binding profiles.<br>
[0025]	In another aspect, the invention provides methods using the protein<br>
binding profiles (including libraries of protein binding profiles) generated according to<br>
the methods of the invention. Thus, the invention provides methods for characterizing a<br>
test sample (such as a test sample suspected of comprising a sample of interest); methods<br>
for detecting presence or absence and/or identifying a test sample; methods for<br>
characterizing a cell, bacteria and/or virus; methods for identifying a test protein;<br>
methods for characterizing, detecting presence and/or absence and/or identifying a<br>
protein complex; and methods for screening, as further described herein. The invention<br>
further provides methods for detecting presence or absence and/or characterizing of a<br>
sample comprising or derived from a cell, such as a prokaryotic cell (such as a<br>
bacterium), an eukaryotic cell, a mammalian cell (including human, and non-human<br>
mammals, such as murine), a cancer or precancerous cell, or a microorganisms, such as<br>
a virus, as well as methods for detecting and/or diagnosing presence or absence of other<br><br>
disease and/or abnormality (such as type and/or stage of cancer, or identification of a<br>
precancerous cell), as further described herein.<br>
[0026]	In one aspect, the invention provides methods for characterizing a test<br>
sample (such as a test sample suspected of comprising a sample of interest), said methods<br>
comprising (a) contacting the test sample with any of the sets of digital antibodies<br>
described herein under conditions that permit binding; (b) optionally removing unbound<br>
protein; (c) detecting binding of protein to antibodies, whereby a protein binding profile<br>
is generated; and (d) comparing the protein binding profile of the test sample with the<br>
protein binding profile of a reference sample, whereby the test sample is characterized by<br>
the comparison.<br>
[0027]	In another aspect, the invention provides methods for characterizing a test<br>
sample, said methods comprising comparing a protein binding profile of the test sample<br>
with a protein binding profile of a reference sample, whereby the test sample is<br>
characterized by the comparison; wherein the protein binding profiles are prepared by a<br>
method comprising (a) contacting a sample with any of the sets of digital antibodies<br>
described herein under conditions that permit binding; (b) optionally removing unbound<br>
protein; (c) detecting binding of protein to antibodies, whereby a protein binding profile<br>
is generated.<br>
[0028]	In another aspect, the invention provides methods for characterizing a test<br>
sample (such as a test sample suspected of comprising a sample of interest), said methods<br>
comprising (a) contacting the test sample with any of the sets of digital antibodies<br>
described herein under conditions that permit binding; (b) optionally removing unbound<br>
protein; (c) detecting binding of protein to antibodies, whereby a protein binding profile<br>
is generated; and (d) comparing the protein binding profile of the test sample with a<br>
library of protein binding profiles generated by any of the methods described herein,<br>
whereby the test sample is characterized using the comparison.<br>
[0029]	In another aspect, the invention provides methods for characterizing a test<br>
sample, said methods comprising comparing protein binding profile of the test sample<br>
with a library of protein binding profiles, whereby the test sample is characterized by the<br><br>
comparison; wherein the protein binding profiles are prepared by a methods comprising<br>
(a) contacting the test sample with any of the sets of digital antibodies described herein<br>
under conditions that permit binding; (b) optionally removing unbound protein; (c)<br>
detecting binding of protein to antibodies, whereby a protein binding profile is generated;<br>
and further wherein the library of protein binding profiles was generated using any of the<br>
methods described herein.<br>
[0030]	In some embodiments, characterizing comprises characterization of a cell,<br>
bacteria or virus (including protein or protein fraction derived from cell, bacteria and/or<br>
virus) in a sample. In other embodiments, characterizing comprises classification of a<br>
cell in a sample, as, e.g., cancerous or precancerous, or a bacteria in a sample (e.g., as<br>
pathogenic or non-pathogenic, and/or taxonomic classification). In some embodiments,<br>
the type, stage, grade, and/or other relevant diagnostic (and/or prognostic) characteristic<br>
of a cancer is determined.<br>
[0031]	In another aspect, the invention provides methods for determining<br>
presence or absence of and/or identifying a sample of interest, said methods comprising<br>
(a) contacting a test sample suspected of comprising the sample of interest with any of<br>
the sets of digital antibodies described herein under conditions that permit binding; (b)<br>
optionally removing unbound protein; (c) detecting binding of protein to antibodies,<br>
whereby a protein binding profile is generated; and (d) comparing the protein binding<br>
profile of the test sample with the protein binding profile of a reference sample, whereby<br>
presence or absence and/or identity of the sample of interest is determined by the<br>
comparison.<br>
[0032]	In another aspect, the invention provides methods for determining<br>
presence or absence of and/or identifying a sample of interest, said methods comprising:<br>
comparing the protein binding profile of a test sample suspected of comprising the<br>
sample of interest with the protein binding profile of a reference sample, whereby<br>
presence or absence and/or identity of the sample of interest is determined by the<br>
comparison; wherein the protein binding profiles were generated by a method comprising<br>
(a) contacting a sample with any of the sets of digital antibodies described herein under<br><br>
conditions that permit binding; (b) optionally removing unbound protein; (c) detecting<br>
binding-of protein to antibodies, whereby a protein binding profile is generated.<br>
[0033]	In another aspect, the invention provides methods for detecting presence<br>
or absence of and/or identifying a sample of interest, said methods comprising (a)<br>
contacting a test sample suspected of comprising the sample of interest with any of the<br>
sets of digital antibodies described herein under conditions that permit binding; (b)<br>
optionally removing unbound protein; (c) detecting binding of protein to antibodies,<br>
whereby a protein binding profile is generated; and (d) comparing the protein binding<br>
profile of the test sample with a library of protein binding profiles generated by any of the<br>
methods described herein, whereby presence or absence and/or identity of the sample of<br>
interest in the test sample is determined by the comparison.<br>
[0034]	In another aspect, the invention provides methods for detecting presence<br>
or absence of and/or identifying a sample of interest, said methods comprising:<br>
comparing a protein binding profile of a test sample suspected of comprising the sample<br>
of interest with a library of protein binding profiles generated by any of the methods<br>
described herein, whereby presence or absence and/or identity of the sample of interest in<br>
the test sample is determined by the comparison; wherein the protein binding profile of<br>
the test sample is generated by a method comprising: (a) contacting a test sample<br>
suspected of comprising the sample of interest with any of the sets of digital antibodies<br>
described herein under conditions that permit binding; (b) optionally removing unbound<br>
protein; (c) detecting binding of protein to antibodies, whereby a protein binding profile<br>
is generated; and the library of protein binding profiles is generated using any of the<br>
methods described herein.<br>
[0035]	In another aspect, the invention provides methods for determining<br>
presence or absence of and/or identifying a cell, bacteria, and/or virus in a test sample,<br>
said methods comprising (a) contacting the test sample with any of the sets of digital<br>
antibodies described herein under conditions that permit binding; (b) optionally<br>
removing unbound protein; (c) detecting binding of protein to antibodies, whereby a<br>
protein binding profile is generated; and (d) comparing the protein binding profile of the<br><br>
test sample with the protein binding profile of a reference sample, whereby presence or<br>
absence and/or identity of the cell, bacteria or virus in the test sample is determined by<br>
the comparison.<br>
[0036]	In another aspect, the invention provides methods for determining<br>
presence or absence of and/or identifying a cell, bacteria and/or virus, said methods<br>
comprising: comparing the protein binding profile of a test sample suspected of<br>
comprising the cell, bacteria and/or virus with the protein binding profile of a reference<br>
sample, whereby presence or absence and/or identity of the cell, bacteria or virus is<br>
determined by the comparison; wherein the protein binding profiles were generated by a<br>
method comprising (a) contacting a sample with any of the sets of digital antibodies<br>
described herein under conditions that permit binding; (b) optionally removing unbound<br>
protein; (c) detecting binding of protein to antibodies, whereby a protein binding profile<br>
is generated.<br>
[0037]	In another aspect, the invention provides methods for detecting presence<br>
or absence of and/or identifying a cell, bacteria and/or virus in a test sample, said<br>
methods comprising (a) contacting the test sample with any of the sets of digital<br>
antibodies described herein under conditions that permit binding; (b) optionally<br>
removing unbound protein; (c) detecting binding of protein to antibodies, whereby a<br>
protein binding profile is generated; and (d) comparing the protein binding profile of the<br>
test sample with a library of protein binding profiles generated by any of the methods<br>
described herein, whereby presence or absence and/or identity of the cell, bacteria or<br>
virus in the test sample is determined by the comparison.<br>
[0038]	In another aspect, the invention provides methods for detecting presence<br>
or absence of and/or identifying a cell, bacteria and/or virus, said methods comprising:<br>
comparing a protein binding profile of a test sample suspected of comprising the cell,<br>
bacteria and/or virus with a library of protein binding profiles generated by any of the<br>
methods described herein, whereby presence or absence and/or identity of the cell,<br>
bacteria and/or virus in the test sample is determined by the comparison; wherein the<br>
protein binding profile of the test sample is generated by a method comprising: (a)<br><br>
contacting a test sample suspected of comprising the sample of interest with any of the<br>
sets of digital antibodies described herein under conditions that permit binding; (b)<br>
optionally removing unbound protein; (c) detecting binding of protein to antibodies,<br>
whereby a protein binding profile is generated; and the library of protein binding profiles<br>
is generated using any of the methods described herein.<br>
[0039]	In another aspect, the invention provides methods for identifying a test<br>
protein, said methods comprising (a) contacting a sample comprising the test protein<br>
with any of the sets of digital antibodies described herein; (b) optionally removing<br>
unbound protein; (c) detecting presence or absence of binding of protein to antibodies in<br>
the set, wherein at least about six digital antibodies bind protein; wherein presence of<br>
binding indicates presence of at least about six epitopes in the protein, wherein the<br>
identity (i.e., linear amino acid sequence) of the at least about six epitopes is used to<br>
identify the protein.<br>
[0040]	In another aspect, the invention provides methods for identifying a test<br>
protein, said methods comprising detecting presence or absence of binding of protein<br>
with any of the sets of digital antibodies described herein, wherein at least about six<br>
digital antibodies bind protein, wherein presence of binding indicates presence of at least<br>
about six epitopes in the protein, wherein the identity (i.e., linear amino acid sequence) of<br>
the at least about six epitopes is used to identify the protein; wherein the binding of<br>
protein is generated using a method comprising (a) contacting a sample comprising the<br>
test protein with any of the sets of digital antibodies described herein; (b) optionally<br>
removing unbound protein; (c) detecting presence or absence of binding of protein to<br>
antibodies in the set. In some embodiments, the test protein consists of about 500 amino<br>
acids in length.<br>
[0041]	In another aspect, the invention provides methods for identifying a test<br>
protein, said methods comprising identifying the test protein based on the identity of at<br>
least about six epitopes recognized by at least 6 digital antibodies, wherein the at least<br>
about six digital antibodies bind protein, wherein binding of protein to antibodies is<br>
detected by a method comprising (a) contacting a sample comprising the test protein with<br><br>
any of the sets of digital antibodies described herein; (b) optionally removing unbound<br>
protein; (c) detecting presence or absence of binding of protein to antibodies in the set,<br>
wherein at least about six digital antibodies bind protein.<br>
[0042]	In some embodiments, the binding and epitope identity of about 7, about<br>
8, about 9, about 10, about 11, about 12, about 13, about 14, about 15, about 20, about 25,<br>
about 30, or more (such as about 40,50, 60, or more) digital antibodies is used to identify<br>
the test protein. In some embodiments, the methods further comprise the comparison of<br>
the epitope identity information (i.e., linear amino acid sequence bound by the antibody)<br>
with a database comprising protein sequence information (such as sequences of<br>
nucleotides or amino acids). In some embodiment, the sample comprises pure protein.<br>
In other embodiments, the sample comprises substantially pure protein.<br>
[0043]	In another aspect, the invention provides methods for screening, and<br>
methods for characterizing, detecting presence or absence of and/or identifying protein<br>
complexes, as further described herein.<br>
[0044]	An antibody can encompass monoclonal antibodies, polyclonal antibodies,<br>
antibody fragments (e.g., Fab, Fab', F(ab')2, Fv, Fc, etc.), chimeric antibodies, single<br>
chain (ScFv), mutants thereof, fusion proteins comprising an antibody portion, and any<br>
other polypeptide that comprises an antigen recognition site of the required specificity.<br>
The antibodies may be murine, rat, rabbit, chicken, human, or of any other origin<br>
(including humanized antibodies).<br>
[0045]	As used herein, "sample" encompasses a variety of sample types and/or<br>
origins, such as blood and other liquid samples of biological origin, solid tissue samples<br>
such as a biopsy specimen or tissue cultures or cells derived therefrom, and the progeny<br>
thereof. The term "sample" encompasses a clinical sample, and also includes cells in<br>
culture, cell supernatants, cell lysates, serum, plasma, biological fluid, and a pure or<br>
enriched bacterial or viral sample derived from any of these, for example, as when a<br>
sample is cultured in order to increase, enrich and/or substantially purify a bacterial or<br>
viral sample therefrom (or, in some embodiments, to increase amount of a sample<br>
comprising bacteria and/or virus). A sample can be from a microorganism, e.g., bacteria,<br><br>
yeasts, viruses, viroids, molds, fungi, plants, animals, including mammals such as<br>
humans. A sample may comprise a single cell or more than a single cell. These samples<br>
can be prepared by methods known in the art such as lysing, fractionation, purification,<br>
including affinity purification, FACS, laser capture microdissection (LCM) or isopycnic<br>
centrifugation. In some embodiments, subcellular fractionation methods are used to<br>
create enriched cellular or subcellular fractions, such as subcellular organelles including<br>
nuclei, mitochondria, golgi apparatus, endoplasmic reticulum, chloroplasts, heavy and<br>
light membranes and cytoplasm. In one embodiment, the sample comprises substantially<br>
intact protein complexes. In some embodiments, the protein complex is a receptor-<br>
associated protein complex or a signal transduction associated protein complex prepared<br>
from , e.g., nucleic acid associated receptors (e.g., the estrogen receptor), or membrane<br>
associate receptors (e.g., epidermal growth factor receptor, IL-6 receptor, stress/apoptosis<br>
pathways, chemokine pathways, MMP transcription pathway,) or cell cycle pathways.<br>
[0046]	In another aspect, the invention provides a hybridoma producing a digital<br>
antibody selected from hybridomas 2.04, 2.03, or 2.11. In some embodiments, the<br>
invention provides an digital antibody generated by hybridoma 2.04, 2.03, or 2.11. In<br>
other embodiments, the invention provides an antibody comprising a heavy chain<br>
variable region and/or a light chain variable regions of a digital antibody produced by<br>
hybridoma 2.04,2.03, or 2.11. In still other embodiments, the invention provides an<br>
antibody comprising one or more CDR(s) (such as two, three, four, five and/or all six<br>
CDRs) of a digital antibody produced by hybridoma 2.04,2.05, and/or 2.11.<br>
[0047]	In another aspect, the invention provides compositions comprising any of<br>
the sets of digital antibodies described herein. In some embodiments, the compositions<br>
are for use in any of the methods described herein. In one embodiment, the invention<br>
provides compositions comprising complexes of protein with any of the sets of digital<br>
antibodies described herein. In another embodiment, the invention provides a<br>
composition comprising any of the sets of digital antibodies described herein, and a<br>
sample. In some embodiments, the antibodies are immobilized (linked and/or attached)<br>
to a solid or semi-solid surface, such as an array. In other embodiments, the antibodies<br>
are labeled antibodies. In some embodiments, the invention provides a compositions<br><br>
comprising a sets of digital antibodies described herein, a sample, and a set of competitor<br>
polypeptides, wherein said competitor polypeptides comprise the cognate amino acid<br>
sequence(s) of one of more digital antibodies in the set of digital antibodies, In some<br>
embodiments, the competitor polypeptides are labeled.<br>
[0048]	The invention further provides kits comprising any of the sets of digital<br>
antibodies described herein. In some embodiments, the kits further comprise instructions<br>
for any of the methods described herein. In some embodiments, the antibodies are<br>
immobilized (linked and/or attached) to a solid or semi-solid surface, such as an array. In<br>
other embodiments, the antibodies are labeled antibodies. In still other embodiments, the<br>
kit comprises any of the libraries of protein binding profiles described herein. In still<br>
other embodiments, the kit further comprises a label. In still other embodiments, the kit<br>
comprises a set of competitor polypeptides, wherein said competitor polypeptides<br>
comprise the cognate amino acid sequence(s) of one of more digital antibodies in the set<br>
of digital antibodies. In some embodiments, the competitor polypeptides are labeled<br>
competitor polypeptide(s).<br>
ACCOMPANYING<br>
BRIEF DESCRIPTION OF THE/DRAWINGS<br>
[0049]	FIGURE 1: shows the reaction pattern using mapping polypeptides<br>
spanning sequences of immunization polypeptides for group 2 and group 5 mice,<br>
respectively.<br>
[0050]	FIGURE 2: depicts a computer-simulated protein binding profile<br>
prepared using a set of 100 computer-simulated digital antibodies. The X axis<br>
corresponds to each of the 100 digital antibodies, and the Y axis represents amount of<br>
protein bound. Each bar forming the protein binding profile corresponds to the amount<br>
of protein bound by each digital antibody.<br>
[0051 ]	FIGURE 3: depicts a computer-simulated protein binding profiles<br>
prepared from a simulated sample comprising one or more viral proteins (panel A), and a<br>
simulated control sample (panel B), using a computer-simulated set of 100 different<br>
digital antibodies. The X axis corresponds to each of the 100 digital antibodies, and the<br><br>
Y axis represents amount of protein bound. Each bar corresponds to the amount of<br>
protein bound by each digital antibody. Panel C depicts the protein binding profile<br>
following subtraction of the control protein binding profile from the viral sample protein<br>
binding profile.<br>
[0052]	FIGURE 4: shows one illustrative embodiment of a digital antibody<br>
microarray of the invention. A standard 96-well microtiter plate format will be used in<br>
order to fit available automatic high-throughput systems. One hundred to two hundred<br>
digital antibodies are coated in each single well in duplicate, forming an array of<br>
antibodies within each microwell. Fluorescein or enzyme-labeled single or multiple-<br>
epitope inhibitor polypeptides are mixed with trypsin-digested protein sample, and then<br>
incubated with the antibodies in a competitive antibody-antigen interaction manner.<br>
Signals can be viewed under a fluorescent reader immediately or further amplified by<br>
adding enzyme-conjugated anti-fluorescein antibody to increase sensitivity.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
[0053]	The present invention stems from the discovery that antibodies that<br>
recognize small polypeptide epitopes, often termed "cross-reactive antibodies" in the art,<br>
are useful in combination to generate a characteristic pattern of binding between<br>
antibodies in a set and protein in a sample, wherein the protein binding pattern, termed a<br>
protein binding profile, may be used to characterize, or "fingerprint" the protein sample.<br>
The present invention stems from the insight that cross-reactive antibodies (i.e.,<br>
antibodies that recognize small epitopes, and thus are expected to bind a multiplicity of<br>
proteins) may be used in combination to generate a protein binding profile of sufficient<br>
specificity such that samples may be characterized, uniquely identified (including<br>
identification of one or more components in a sample) and/or distinguished. Thus,<br>
specificity of results and information arising from antibody binding is conferred via the<br>
binding of sets of digital antibodies to protein in a sample (whereby a specific protein<br>
binding profile is generated), rather than by binding of a single antibody that binds but<br>
one or a few proteins, as commonly used in the art for specific detection. Generally, a<br>
protein binding pattern comprises information regarding presence (or absence) of protein<br><br>
binding and/or intensity (amount) of protein binding between each digital antibody and<br>
protein(s) in a sample. The protein binding profile can be used to, e.g., identify the<br>
source of the protein sample (e.g., from a pathogen; from a cancer cell), or may be used<br>
to identify or detect a previously unknown agent, such as a novel pathogen.<br>
[0054]	The digital antibodies that recognize small peptide epitopes bind<br>
(generally, specifically bind) small linear peptide epitopes consisting of 3 consecutive<br>
amino acids, 4 consecutive amino acids, or 5 consecutive amino acids, wherein each<br>
antibody in the set binds to a different small linear peptide epitope. Due to the small size<br>
of the epitope recognized by a digital antibody (i.e., 3 consecutive amino acids, 4<br>
consecutive amino acids, or 5 consecutive amino acids), a single digital antibody<br>
generally binds a plurality of proteins (in some embodiments, a large plurality of<br>
proteins) based on the presence of the cognate small epitope within the plurality of bound<br>
proteins. Thus, binding of protein to each digital antibody generally reflects binding of a<br>
plurality of protein species, each of which possesses one or more copies of the cognate<br>
small epitope sequence. Similarly, a single protein may be bound by multiple digital<br>
antibodies based on the presence of multiple small epitope sequences within the protein.<br>
Generally, the identity of the epitope recognized by a given digital antibody is known,<br>
and this information useful, e.g., in methods for identifying proteins as further describe<br>
herein.<br>
[0055]	Accordingly, in one aspect, the invention provides sets of antibodies<br>
(termed "digital antibodies") that bind (generally, specifically bind) small linear peptide<br>
epitopes consisting of 3 consecutive amino acids, 4 consecutive amino acids, or 5<br>
consecutive amino acids, wherein each antibody in the set binds to a different small linear<br>
peptide epitope. It is understood that "different epitopes" encompasses epitopes with<br>
overlapping amino acid sequence as well as epitopes with distinct amino acid sequence.<br>
In some embodiments, the set comprises at least about 15 digital antibodies, wherein each<br>
digital antibody binds a different epitope, and wherein each digital antibody binds an<br>
epitope consisting of 3 consecutive amino acids, 4 consecutive amino acids, or 5<br>
consecutive amino acids. In some embodiments, the set comprises at least about any of<br>
20,30,40, 50, 60, 70, 80, 90,100,110,125,150,175,200,300,400, 500,600, 700, 800 ,<br><br>
900,1000, or more antibodies. In other embodiments, the set comprises at least about<br>
100 antibodies. In other embodiments, the set comprises at least about 100 digital<br>
antibodies that recognize epitopes consisting of 3 consecutive amino acids (also termed a<br>
3mer epitope). In some embodiments, the set comprises at least about 100 digital<br>
antibodies that recognize epitopes consisting of 3 consecutive amino acids, and at least<br>
about 100 digital antibodies that recognize epitopes consisting of 4 consecutive amino<br>
acids (also termed a 4mer epitope). In some embodiments, the set comprises at least<br>
about at least about 100 digital antibodies that recognize epitopes consisting of 3<br>
consecutive amino acids, at least about 100 digital antibodies that recognize epitopes<br>
consisting of 4 consecutive amino acids, and at least about 100 digital antibodies that<br>
recognize epitopes consisting of 5 consecutive amino acids. In another embodiment, the<br>
set comprises at least about 1000 digital antibodies that recognize epitopes consisting of 4<br>
consecutive amino acids. Digital antibodies, and methods of making digital antibodies<br>
are further described herein.<br>
[0056]	In another aspect, the invention provides an array comprising any of the<br>
digital antibody sets described herein. Methods for generating arrays are well known in<br>
the art, and further described herein.<br>
[0057]	In another aspect, the invention further provides methods using the sets of<br>
digital antibodies. As a general overview, the methods involve generating a protein<br>
binding profile of a sample using any of the sets of digital antibodies described herein,<br>
comprising contacting the sample with a set of digital antibodies, and detection of the<br>
antibody-protein complex formed between protein in the sample and antibodies, if any.<br>
Detection of the antibody-protein complex provides information relating to amount<br>
(intensity) of binding. Detection is performed using methods well known in the art, and<br>
further described herein. Thus, the methods for generating a protein binding profile<br>
generally comprise (a) contacting a sample with any of the sets of digital antibodies<br>
described herein under conditions that permit binding; (b) optionally removing unbound<br>
protein (in some embodiments, removing protein that is not specifically bound); and (c)<br>
detecting binding of protein to antibodies, whereby a protein binding profile is generated.<br>
Generally, binding of a multiplicity of antibodies in a set is detected. In some<br><br>
embodiments, binding of at least about 95%, at least about 90%, at least about 75%, at<br>
least about 50%, at least about 30% of antibodies in a set is detected. In some<br>
embodiments, binding of each antibody in a set is detected. Detecting may be qualitative<br>
and/or qualitative. In some embodiments, the protein binding profile may include<br>
(insofar as the small epitope bound by the digital antibody is known) information relating<br>
to amino acid content of protein(s) bound by the digital antibod(ies) present in the set of<br>
digital antibodies. As further discussed herein, the protein binding profile may be<br>
correlated with information relating to the sample, such as source of sample (e.g., of a<br>
pathogen such as a bacteria and/or virus, of a cancerous cell or tumor), and may be<br>
recorded and stored in a library of protein binding profiles (such as a database).<br>
[0058]	In some embodiments, the methods of generating a protein binding profile<br>
further comprise a step of treating the sample with a protein cleaving agent, whereby<br>
polypeptide fragments are generated. The sample can treated with a protein cleaving<br>
agent prior to step (a) of contacting the sample with the set of digital antibodies under<br>
conditions that permit binding. The protein cleaving agent may be an enzyme (such as<br>
chymotrypsin or trypsin) or a chemical agent (such as cyanogen bromide). Protein<br>
cleaving agents and methods for treatment with protein cleaving agents are well known in<br>
the art and further described herein.<br>
[0059]	A library of protein binding profiles can be generated by compiling<br>
protein binding profiles for two or more different samples, such as samples of pathogenic<br>
or non-pathogenic bacteria. The protein binding profiles can be compared with each<br>
other, either qualitatively or quantitatively, in order to discern similarities and/or<br>
differences in protein binding patterns. The protein binding profiles in the library can<br>
also be used in methods involving comparative assessment of the protein binding profiles<br>
of sample from different sources (for example, from different tissues or cell types,<br>
disease states, or different microorganisms). For example, the libraries of protein binding<br>
profiles are useful in methods for characterizing protein samples, methods for diagnosis<br>
of bacterial or viral infection, and/or in methods for detection and/or taxonomic<br>
classification of a cell, bacteria or infectious agent, as further described herein. For<br>
example, a protein binding profile prepared from an unidentified sample may be used to<br><br>
characterize and/or identify the sample by comparison with a library of protein binding<br>
profiles (such as a library of known bacterial pathogens).<br>
[0060]	Thus, in another aspect, the invention provides methods for generating a<br>
library of protein binding profiles, comprising the steps of (a) contacting a sample with<br>
any of the sets of digital antibodies described herein under conditions that permit binding;<br>
(b) optionally removing unbound protein (in some embodiments, removing protein that is<br>
not specifically bound); (c) detecting binding of protein to antibodies, whereby a protein<br>
binding profile is generated; and (d) repeating steps (a) through (c) with at least two<br>
different samples. In some embodiments, the library comprises at least about 5,10,15,<br>
20,25, 30,35,40,45,50,75,100,150,200,300,400,500, 600,700, 800,900, or more<br>
(such as 1000,2000,3000,4000 or more) binding profiles. In another aspect, the<br>
invention provides libraries of protein binding profiles, wherein the library is prepared<br>
using any of the methods described herein.<br>
[0061]	In another aspect, the invention provides methods using the protein<br>
binding profiles generated according to the methods of the invention (including the<br>
libraries of protein binding profiles), including methods involving comparative<br>
assessment of the protein binding profiles of sample from different sources (for example,<br>
from different tissues or cell types, disease states, or different microorganisms). Thus,<br>
the invention provides methods for characterizing a test sample (such as a test sample<br>
suspected of comprising a sample of interest); methods for detecting presence or absence<br>
and/or identifying a test sample; methods for characterizing a cell, bacteria and/or virus;<br>
methods for identifying a test protein; methods for characterizing, detecting presence<br>
and/or absence and/or identifying a protein complex; and methods for screening, as<br>
further described herein.. In some embodiments, the methods involve the comparison of<br>
a protein binding profile generated from a test sample with a reference protein binding<br>
profile (such as protein binding profile(s) generated from a control sample, or normal<br>
values), or library of profiles generated according to any of the methods described herein.<br>
[0062]	The invention also provides methods for identifying a protein using (a) a<br>
protein binding profile generated from a sample comprising the protein; and (b) the<br><br>
amino acid sequences of the epitope recognized by digital antibodies that bind sample<br>
protein, whereby binding of a set of digital antibodies (generally, about 6 digital<br>
antibodies) provides sufficient information relating to the amino acid content of the<br>
protein such that the protein may be identified. By way of example, the simultaneous<br>
binding of digital antibodies recognizing the epitopes QAP, TPG, LTG, VSR, and WDQ<br>
to a test protein identifies that protein as HCV NS3 protein, for HCV NS3 protein is the<br>
only known protein comprising these six 3mer amino acid sequences. Accordingly, in<br>
one aspect, the invention provides methods for identifying a test protein, said methods<br>
comprising (a) contacting a sample comprising the test protein with any of the sets of<br>
digital antibodies described herein; (b) optionally removing unbound protein; (c)<br>
detecting presence or absence of binding of protein to antibodies in the set, wherein at<br>
least about six digital antibodies bind protein; wherein presence of binding indicates<br>
presence of at least about six epitopes in the protein, wherein the identity (i.e., linear<br>
amino acid sequence) of the at least about six epitopes is used to identify the protein. In<br>
some embodiments, the binding and epitope identity of about 7, about 8, about 9, about<br>
10, about 11, about 12, about 13, about 14, about 15, about 20, about 25, about 30, or<br>
more (such as about 40, 50, 60, or more) digital antibodies is used to identify the test<br>
protein. In some embodiments, the methods further comprise the comparison of the<br>
epitope identity information (i.e., linear amino acid sequence bound by the antibody) with<br>
a database comprising protein sequence information (such as sequences of nucleotides or<br>
amino acids). In some embodiment, the sample comprises pure protein. In other<br>
embodiments, the sample comprises substantially pure protein.<br>
[0063]	In another aspect, the invention provides methods for screening, and<br>
methods for characterizing, detecting presence or absence of and/or identifying protein<br>
complexes, as further described herein.<br>
[0064]	As is evident, one or more steps may be combined and/or performed<br>
sequentially (often in any order, as long as the requisite product(s) are able to be formed),<br>
and, as is evident, the invention includes various combinations of the steps described<br>
herein. It is also evident, and is described herein, that the invention encompasses<br>
methods in which the initial, or first, step is any of the steps described herein. Methods<br><br>
of the invention encompass embodiments in which later, "downstream" steps are an<br>
initial step.<br>
[0065]	In another aspect, the invention provides compositions comprising any of<br>
the sets of digital antibodies described herein. In some embodiments, the compositions<br>
are for use in any of the methods described herein. In one embodiment, the invention<br>
provides compositions comprising complexes of protein with any of the sets of digital<br>
antibodies described herein. In another embodiment, the invention provides a<br>
composition comprising any of the sets of digital antibodies described herein, and a<br>
sample. In some embodiments, the antibodies are immobilized (linked and/or attached)<br>
to a solid or semi-solid surface, such as an array. In other embodiments, the antibodies<br>
are labeled antibodies. In some embodiments, the invention provides a composition<br>
comprising a sets of digital antibodies described herein, a sample, and a set of competitor<br>
polypeptides, wherein said competitor polypeptides comprise the cognate amino acid<br>
sequence(s) of one of more digital antibodies in the set of digital antibodies. In some<br>
embodiments, the competitor polypeptides are labeled.<br>
[0066]	The invention further provides kits comprising any of the sets of digital<br>
antibodies described herein. In some embodiments, the kits further comprise instructions<br>
for any of the methods described herein. In some embodiments, the antibodies are<br>
immobilized (linked and/or attached) to a solid or semi-solid surface, such as an array. In<br>
other embodiments, the antibodies are labeled antibodies. In still other embodiments, the<br>
kit comprises any of the libraries of protein binding profiles described herein. In still<br>
other embodiments, the kit further comprises a label. In still other embodiments, the kit<br>
comprises a set of competitor polypeptides, wherein said competitor polypeptides<br>
comprise the cognate amino acid sequence(s) of one of more digital antibodies in the set<br>
of digital antibodies. In some embodiments, the competitor polypeptides are labeled<br>
competitor polypeptide(s).<br>
[0067]	Methods and conditions for antibody binding and detection of antibody<br>
binding are well known in the art and further described herein. In some embodiment,<br>
contacting with a set of digital antibodies is sequential (as when one antibody is contacted<br><br>
with the sample, then removed, another antibody is contacted with the sample and<br>
removed, and so on). In other embodiments, contacting with a set of digital antibodies is<br>
in parallel, for example, as when a group of antibodies are contacted with the sample<br>
simultaneously. In some embodiments, several groups of two or more antibodies are<br>
serially contacted with the sample, for example, group 1 is contacted and removed, group<br>
2 is contacted and removed, and so on.<br>
[0068]	An antibody can encompass monoclonal antibodies, polyclonal antibodies,<br>
antibody fragments (e.g., Fab, Fab', F(ab')2 Fv, Fc, etc.), chimeric antibodies, single<br>
chain (ScFv), mutants thereof, fusion proteins comprising an antibody portion, and any<br>
other polypeptide that comprises an antigen recognition site of the required specificity.<br>
The antibodies may be murine, rat, rabbit, chicken, human, or of any other origin<br>
(including humanized antibodies).<br>
[0069]	As noted in the definition, and as used herein, "sample" encompasses a<br>
variety of protein sample types, including those obtained from an individual (including<br>
human and non-human individuals). The sample may also be obtained from food, water,<br>
or air. In some embodiments, the sample is a protein sample, such as whole cell protein<br>
extract or a subfractionation of the whole cell extract (such as soluble protein, membrane<br>
bound protein, periplasmic protein, etc.). In other embodiments, the sample encompasses<br>
an intact protein complex, such as a receptor-associated protein complex. In some<br>
embodiments, the sample is a pure (or substantially pure) protein, for which identification<br>
is desired. Suitable samples for use in the methods of the invention are described further<br>
herein. In some embodiments, the sample comprises a protein complex that is isolated<br>
and/or substantially purified, e.g., from a cell. In some embodiments, the sample can<br>
treated with a protein cleaving agent prior to the step of contacting the sample with the<br>
set of digital antibodies under conditions that permit binding. The protein cleaving agent<br>
may be an enzyme (such as chymotrypsin or trypsin) or a chemical agent (such as<br>
cyanogen bromide). Protein cleaving agents and methods for treatment with protein<br>
cleaving agents are well known in the art and further described herein.<br><br>
[0070]	The practice of the present invention will employ, unless otherwise<br>
indicated, conventional techniques of molecular biology (including recombinant<br>
techniques), microbiology, cell biology, biochemistry and immunology, which are within<br>
the skill of the art. Such techniques are explained fully in the literature, such as,<br>
Molecular Cloning: A Laboratory Manual, second edition (Sambrook et al, 1989) Cold<br>
Spring Harbor Press; Oligonucleotide Synthesis (M.J. Gait, ed., 1984); Methods in<br>
Molecular Biology, Humana Press; Cell Biology: A Laboratory Notebook (J.E. Cellis,<br>
ed., 1998) Academic Press; Animal Cell Culture (R.I. Freshney, ed., 1987); Introduction<br>
to Cell and Tissue Culture (J.P. Mather and P.E. Roberts, 1998) Plenum Press; Cell and<br>
Tissue Culture: Laboratory Procedures (A. Doyle, J.B. Griffiths, and D.G. Newell, eds.,<br>
1993-8) J. Wiley and Sons; Methods in Enzymology (Academic Press, Inc.); Handbook of<br>
Experimental Immunology (D.M. Weir and C.C. Blackwell, eds.,) Gene Transfer Vectors<br>
for Mammalian Cells (J.M.Miller and M.P. Calos, eds., 1987); Current Protocols in<br>
Moleadar Biology (F.M. Ausubel et al., eds., 1987); PCR: Tire Polymerase Chain<br>
Reaction (Mullis et al., eds., 1994); Current Protocols in Immunology (J.E. Coligan et<br>
al, eds., 1991); Short Protocols in Molecular Biology (Wiley and Sons, 1999);<br>
Immunobiology (C.A.Janeway andP.Travers, 1997); Antibodies (P. Finch, 1997);<br>
Antibodies: a practical approach (D. Catty., ed., IRL Press, 1988-1989); Monoclonal<br>
antibodies : a practical approach (P. Shepherd and C. Dean, eds., Oxford University<br>
Press, 2000); Using antibodies: a laboratory manual (E. Harlow and D. Lane (Cold<br>
Spring Harbor Laboratory Press, 1999); and The Antibodies (M. Zanetti and J.D. Capra,<br>
eds., Harwood Academic Publishers, 1995).<br>
Definitions<br>
[0071]	An "antibody" is an immunoglobulin molecule capable of specific binding<br>
to a target, such as a carbohydrate, polynucleotide, lipid, polypeptide, etc., through at<br>
least one antigen recognition site, located in the variable region of the immunoglobulin<br>
molecule. As used herein, the term encompasses not only intact polyclonal or<br>
monoclonal antibodies, but also fragments thereof (such as Fab, Fab', F(ab')2, Fv), single<br>
chain (ScFv), mutants thereof, fusion proteins comprising an antibody portion, and any<br>
other modified configuration of the immunoglobulin molecule that comprises an antigen<br><br>
recognition site of the required specificity. An antibody includes an antibody of any<br>
class, such as IgG, IgA, or IgM (or sub-class thereof), and the antibody need not be of<br>
any particular class. Depending on the antibody amino acid sequence of the constant<br>
domain of its heavy chains, immunoglobulins can be assigned to different classes. There<br>
are five major classes of immunoglobulins: IgA, IgD, IgE, IgG, and IgM, and several of<br>
these may be further divided into subclasses (isotypes), e.g., IgGl, IgG2, IgG3, IgG4,<br>
IgAl and IgA2. The heavy-chain constant domains that correspond to the different<br>
classes of immunoglobulins are called alpha, delta, epsilon, gamma, and mu,<br>
respectively. The subunit structures and three-dimensional configurations of different<br>
classes of immunoglobulins are well known.<br>
[0072]	"Fv" is an antibody fragment that contains a complete antigen-recognition<br>
and -binding site. In a two-chain Fv species, this regions consists of a dimer of one<br>
heavy and one light chain variable domain in tight, non-covalent association. In a single-<br>
chain Fv species, one heavy and one light chain variable domain can be covalently linked<br>
by a flexible polypeptide linker such that the light and heavy chains can associate in a<br>
dimeric structure analogous to that in a two-chain Fv species. It is in this configuration<br>
that the three CDRs of each variable domain interact to define an antigen-binding<br>
specificity on the surface of the VH-VL dimer. However, even a single variable domain<br>
(or half of a Fv comprising only 3 CDRs specific for an antigen) has the ability to<br>
recognize and bind antigen, although generally at a lower affinity than the entire binding<br>
site.<br>
[0073]	A "monoclonal antibody" refers to a homogeneous antibody population<br>
wherein the monoclonal antibody is comprised of amino acids (naturally occurring and<br>
non-naturally occurring) that are involved in the selective binding of an antigen. A<br>
population of monoclonal antibodies (as opposed to polyclonal antibodies) are highly<br>
specific, in the sense that they are directed against a single antigenic site. The term<br>
"monoclonal antibody" encompasses not only intact monoclonal antibodies and full-<br>
length monoclonal antibodies, but also fragments thereof (such as Fab, Fab', F(ab')2, Fv),<br>
single chain (ScFv), mutants thereof, fusion proteins comprising an antibody portion, and<br>
any other modified configuration of the immunoglobulin molecule that comprises an<br><br>
antigen recognition site of the required specificity and the ability to bind to an antigen<br>
(see definition of antibody). It is not intended to be limited as regards to the source of the<br>
antibody or the manner in which it is made (e.g., by hybridoma, phage selection,<br>
recombinant expression, transgenic animals, etc.).<br>
[0074]	A "digital antibody" is an antibody that binds (generally specifically<br>
binds) a small linear polypeptide epitope consisting of 3 consecutive amino acids, 4<br>
consecutive amino acids, or 5 consecutive amino acids.<br>
[0075]	The terms "polypeptide", "oligopeptide", "peptide" and "protein" are used<br>
interchangeably herein to refer to polymers of amino acids of any length. The polymer<br>
may be linear or branched, it may comprise modified amino acids, and it may be<br>
interrupted by non-amino acids. The terms also encompass an amino acid polymer that<br>
has been modified naturally or by intervention; for example, disulfide bond formation,<br>
glycosylation, lipidation, acetylation, phosphorylation, or any other manipulation or<br>
modification, such as conjugation with a labeling component. Also included within the<br>
definition are, for example, polypeptides containing one or more analogs of an amino<br>
acid (including, for example, unnatural amino acids, etc.), as well as other modifications<br>
known in the art.<br>
[0076]	An epitope that "specifically binds" or "preferentially binds" (used<br>
interchangeably herein) to an antibody is a term well understood in the art, and methods<br>
to determine such specific or preferential binding are also well known in the art. A<br>
molecule is said to exhibit "specific binding" or "preferential binding" if it reacts or<br>
associates more frequently, more rapidly, with greater duration and/or with greater<br>
affinity with a particular cell or substance than it does with alternative cells or substances.<br>
An antibody "specifically binds" or "preferentially binds" to a target if it binds with<br>
greater affinity, avidity, more readily, and/or with greater duration than it binds to other<br>
substances. For example, an antibody that specifically or preferentially binds to an<br>
epitope is an antibody that binds this epitope with greater affinity, avidity, more readily,<br>
and/or with greater duration than it binds to other epitopes. It is also understood by<br>
reading this definition that, for example, an antibody (or moiety or epitope) that<br><br>
specifically or preferentially binds to a first target may or may not specifically or<br>
preferentially bind to a second target. As such, "specific binding" or "preferential<br>
binding" does not necessarily require (although it can include) exclusive binding.<br>
Generally, but not necessarily, reference to binding means preferential binding.<br>
[0077]	A "sample" encompasses a variety of sample types, including those<br>
obtained from an individual. The definition encompasses blood and other liquid samples<br>
of biological origin, solid tissue samples such as a biopsy specimen or tissue cultures or<br>
cells derived therefrom, and the progeny thereof. A sample can be from a<br>
microorganisms, e.g., bacteria, yeasts, viruses, viroids, molds, fungi, plants, animals,<br>
including mammals such as humans, rodents (such as mice and rats), and monkeys (and<br>
other primates). A sample may comprise a single cell or more than a single cell. The<br>
definition also includes samples that have been manipulated in any way after their<br>
procurement, such as by treatment with reagents, solubilization, or enrichment for certain<br>
components, such as proteins or polynucleotides. The term "sample" encompasses a<br>
clinical sample, and also includes cells in culture, cell supernatants, and cell lysates.<br>
Other types of samples are described herein.<br>
[0078]	"Detect" refers to identifying the presence, absence and/or amount of<br>
protein to be detected.<br>
[0079]	"Absent" or "absence" of binding, and "lack of detection of product" as<br>
used herein includes insignificant, or de minimus levels.<br>
[0080]	An "array" of digital antibodies is an ordered spatial arrangement of one<br>
or more antibodies on a physical substrate. Row and column arrangements are preferred<br>
due to the relative simplicity in making and assessing such arrangements. The spatial<br>
arrangement can, however, be essentially any form selected by the user, and preferably,<br>
but need not be, in a pattern.<br>
[0081 ]	As used herein, the singular form "a", "an", and "the" includes plural<br>
references unless indicated otherwise. For example, "an" antibody includes one or more<br>
antibodies and "a protein" means one or more proteins.<br><br>
Digital Antibody Sets of the Invention<br>
[0082]	The invention provides sets of digital antibodies, wherein the sets<br>
comprise at least about 15 digital antibodies, wherein each digital antibody recognizes a<br>
different small linear peptide epitope. As used herein, "digital antibody" is an antibody<br>
that binds (generally specifically binds) a small linear peptide epitope consisting of 3<br>
consecutive amino acids, 4 consecutive amino acids or 5 consecutive amino acids. The<br>
present invention stems from the insight that antibodies that recognizes small, frequently<br>
occurring polypeptide epitopes (often termed "cross-reactive antibodies" in the art) are<br>
useful in combination to generate protein binding patterns (such as a pattern of binding to<br>
antibodies and/or intensity of binding to antibodies). By virtue of the epitope specificity,<br>
digital antibodies generally recognize a multiplicity of proteins that comprise the small<br>
epitope to which the antibody binds. Thus, specificity of binding pattern is generated via<br>
the binding of sets of digital antibodies to protein in a sample, rather than by binding of a<br>
single antibody that binds but one or a few proteins, as commonly used in the art for<br>
specific detection. Insofar as the small epitope bound by the antibody is known, binding<br>
by a digital antibody provides information relating to amino acid content of protein(s)<br>
bound by the digital antibody. Digital antibodies and methods of making digital<br>
antibodies are further discussed herein and exemplified in the Examples.<br>
[0083]	In some embodiments, the set comprises at least about 15 digital<br>
antibodies, wherein each digital antibody binds a different epitope, and wherein each<br>
digital antibody binds an epitope consisting of 3 consecutive amino acids, or 4<br>
consecutive amino acids. It is understood that "different epitopes" encompasses epitopes<br>
that have overlapping amino acid sequences, as well as epitope that have different (non-<br>
overlapping) amino acid sequences. In some embodiments, the set comprises at least<br>
about 20, 30,40, 50,60, 70, 80,90,100, 110,125,150,175, 200, 300,400, 500, 600,<br>
700, 800,900,1000, or more antibodies. In other embodiments, the set comprises at least<br>
about 100 antibodies. In other embodiments, the set comprises at least about 100 digital<br>
antibodies that recognize epitopes consisting of 3 consecutive amino acids (also termed a<br>
3mer epitope). In some embodiments, the set comprises at least about 100 digital<br><br>
antibodies that recognize epitopes consisting of 3 consecutive amino acids, and at least<br>
about 100 digital antibodies that recognize epitopes consisting of 4 consecutive amino<br>
acids (also termed a 4mer epitope). In some embodiments, the set comprises at least<br>
about 100 digital antibodies that recognize epitopes consisting of 3 consecutive amino<br>
acids, at least about 100 digital antibodies that recognize epitopes consisting of 4<br>
consecutive amino acids, and at least about 100 digital antibodies that recognize epitopes<br>
consisting of 5 consecutive amino acids. In another embodiment, the set comprises at<br>
least about 1000 digital antibodies that recognize epitopes consisting of 4 consecutive<br>
amino acids. Digital antibodies, and methods of making digital antibodies are further<br>
described herein.<br>
[0084]	An antibody can encompass monoclonal antibodies, polyclonal antibodies,<br>
antibody fragments (e.g., Fab, Fab', F(ab')2, Fv, Fc, etc.), chimeric antibodies, single<br>
chain (ScFv), mutants thereof, fusion proteins comprising an antibody portion, and any<br>
other polypeptide that comprises an antigen recognition site of the required specificity<br>
(including antibody mimetics. See, e.g., Xu et al, Chem Biol. 2002 Aug. 9(8):933-42).<br>
The antibodies may be murine, rat, rabbit, chicken, human, or any other origin (including<br>
humanized antibodies).<br>
[0085]	As described herein, a digital antibody binds a short, linear peptide epitope<br>
of 3,4, or 5 consecutive (sequential) amino acids. In some embodiments, the digital<br>
antibody binds an epitope consisting of 3 sequential amino acids (termed a 3mer), 4<br>
sequential amino acids (termed a 4mer), or 5 sequential amino acids (termed a 5mer). In<br>
other embodiments, the digital antibody binds a small discontinuous linear peptide<br>
sequence, such as the linear peptide sequence YCxC or YxCC, wherein the "x" represents<br>
any of the 20 natural amino acids. Alternatively, "x" may be limited to subset of 2 or<br>
more amino acids. It is understood that the amino acid(s) forming the epitope may be<br>
linear or branched, and may comprise an amino acid(s) that has been modified naturally<br>
or by intervention; for example, disulfide bond formation, glycosylation, lipidation,<br>
acetylation, phosphorylation, or any other manipulation or modification, such as<br>
conjugation with a labeling component. The amino acid(s) forming the epitope may<br><br>
further encompass, for example, one or more analogs of an amino acid (including, for<br>
example, unnatural amino acids, etc.), as well as other modifications known in the art.<br>
[0086]	Antibodies that bind small linear peptide epitopes have been previously<br>
described, as shown in Table 2, below.<br>
[0087]	In some embodiments, the digital antibody binds its cognate epitope with<br>
an affinity of binding reaction of at least about 10"7 M, at least 10"8 M, or at least about<br>
10" M, or lower. In some embodiments, a binding interaction will discriminate over<br>
adventitious binding interactions in the reaction by at least two-fold, at least five-fold, at<br>
least 10- to at least 100-fold or more.<br>
[0088]	It is understood that other protein binding agents (such as antibodies that<br>
are not digital antibodies) may be used, in addition to the digital antibody sets described<br>
herein.<br>
[0089]	It is evident that the number of digital antibodies in the digital antibody set<br>
depends on the contemplated uses and applications. Knowledge of the sequence and/or<br>
the length of the cognate amino acid epitope recognized by the digital antibody permits<br>
an estimate regarding the expected frequency of the epitope(s) recognized by the digital<br>
antibodies within the set. As shown in Table 1, there are a total of 8,000 (203), 160,000<br>
(204) and 3,200,000 (205) random combinations for 3mer, 4mer and 5mer linear peptide<br>
sequences, respectively. Considering 500 amino acids as an average length of protein,<br>
the probability that it is detected by a single anti-3mer antibody is 0.0625, the probability<br>
increases to about 1 when 15 anti-3mer antibodies are used, and the probability increases<br>
to 6.25 when 100 anti-3mer antibodies are used. Such calculations are routine.<br><br><br>
[0090]	Thus, it is understood that the number of digital antibodies required in the<br>
present invention depends on various factors, including: the use, application, complexity<br>
of the sample (in terms of number of expected or estimated or previously determined<br>
proteins, including protein variants such as splice variants), average size of the proteins in<br>
the sample, frequency that the cognate epitope is present or predicted to be present in a<br>
sample; binding affinity and/or specificity of the digital antibody(ies); knowledge of<br>
target protein(s); and stability of the digital antibody. Such factors are well known in the<br>
art and are further discussed herein.<br>
[0091]	It is understood that the identity (sequence) of the epitopes to which the<br>
digital antibodies bind may be used in combination with any of the methods described<br>
herein to, e.g., identify a protein. Generally, at least about 15 to about 18 amino acids of<br>
epitope information must be obtained in order to identify a protein. Thus, in some<br>
embodiments, at least about 6 anti-3mer digital antibodies, at least about five anti-4mer<br>
digital antibodies, or at least about four anti-5mer digital antibodies must bind such that<br>
at least about 15-18 amino acids of amino acid epitope information is derived. In some<br><br>
embodiments, at least 6 digital antibodies are used to bind protein of average length of<br>
about 500 amino acids.<br>
[0092]	Digital antibodies may be generated using methods well known in the art.<br>
Antibodies that bind small linear peptide epitopes have been previously described, as<br>
shown in Table 2.<br><br><br><br>
[0093]	In another aspect, and as exemplified in the Examples, digital antibodies<br>
(e.g., human, humanized, mouse, chimeric) may be made by using immunogens which<br>
express one or more small peptide epitopes, such as a small linear peptide epitope<br>
consisting of 3,4, or 5 amino acids.<br>
[0094]	Methods for synthesizing polypeptides are well known in the art. In some<br>
embodiments, the polypeptide immunogen is synthesized as a multiple antigen<br>
polypeptide, or MAP.<br>
[0095]	The route and schedule of immunization of the host animal are generally<br>
in keeping with established and conventional techniques for antibody stimulation and<br>
production, as further described herein. General techniques for production of human,<br>
mouse, rabbit and chicken antibodies are known in the art and are described herein.<br>
Typically, the host animal is inoculated intraperitoneally with an amount of immunogen,<br>
including as described herein.<br>
[0096]	Hybridomas can be prepared from the lymphocytes and immortalized<br>
myeloma cells using the general somatic cell hybridization technique of Kohler, B. and<br>
Milstein, C. (1975) Nature 256:495-497 or as modified by Buck, D. W. et a!., (1982) In<br>
Vitro, 18:377-381. Available myeloma lines, including but not limited to X63-Ag8.653<br>
and those from the Salk Institute, Cell Distribution Center, San Diego, Calif, USA, may<br>
be used in the hybridization. Generally, the technique involves fusing myeloma cells and<br>
lymphoid cells using a fusogen such as polyethylene glycol, or by electrical means well<br>
known to those skilled in the art. After the fusion, the cells are separated from the fusion<br>
medium and grown in a selective growth medium, such as hypoxanthine-aminopterin-<br><br>
thymidine (HAT) medium, to eliminate unhybridized parent cells. Any of the media<br>
described herein, supplemented with or without serum, can be used for culturing<br>
hybridomas that secrete monoclonal antibodies. As another alternative to the cell fusion<br>
technique, EBV immortalized B cells may be used to produce the digital antibodies of the<br>
subject invention. The hybridomas are expanded and subcloned, if desired, and<br>
supernatants are assayed for anti-immunogen activity by conventional immunoassay<br>
procedures (e.g., radioimmunoassay, enzyme immunoassay, or fluorescence<br>
immunoassay).<br>
[0097]	Hybridomas that may be used as source of antibodies encompass all<br>
derivatives, progeny cells of the parent hybridomas that produce digital antibodies (such<br>
as monoclonal antibodies), or a portion thereof.<br>
[0098]	Hybridomas that produce such antibodies may be grown in vitro or in vivo<br>
using known procedures. The monoclonal antibodies may be isolated from the culture<br>
media or body fluids, by conventional immunoglobulin purification procedures such as<br>
ammonium sulfate precipitation, gel electrophoresis, dialysis, chromatography, and<br>
ultrafiltration, if desired. Undesired activity if present, can be removed, for example, by<br>
running the preparation over adsorbents made of the immunogen attached to a solid phase<br>
and eluting or releasing the desired antibodies off the immunogen. Immunization of a<br>
host animal with a human or other species of small epitope receptor, or a fragment of the<br>
human or other species of small epitope receptor, or a human or other species of small<br>
epitope receptor or a fragment containing the target amino acid sequence conjugated to a<br>
protein that is immunogenic in the species to be immunized, e.g., keyhole limpet<br>
hemocyanin, serum albumin, bovine thyroglobulin, or soybean trypsin inhibitor using a<br>
bifunctional or derivatizing agent, for example maleimidobenzoyl sulfosuccinimide ester<br>
(conjugation through cysteine residues), N-hydroxysuccinimide (through lysine residues),<br>
glutaradehyde, succinic anhydride, SOC12, or RlN=ONR, where R and Rl are different<br>
alkyl groups can yield a population of antibodies (e.g., monoclonal antibodies).<br>
[0099]	If desired, the digital antibody (monoclonal or polyclonal) of interest may<br>
be sequenced and the polynucleotide sequence may then be cloned into a vector for<br><br>
expression or propagation. The sequence encoding the antibody of interest may be<br>
maintained in vector in a host cell and the host cell can then be expanded and frozen for<br>
future use. In an alternative, the polynucleotide sequence may be used for genetic<br>
manipulation to "humanize" the antibody or to improve the affinity, or other<br>
characteristics of the antibody. For example, the constant region may be engineered to<br>
more resemble human constant regions to avoid immune response if the antibody is used<br>
in clinical trials and treatments in humans. It may be desirable to genetically manipulate<br>
the antibody sequence to obtain greater affinity to the small epitope and/or greater and/or<br>
altered specificity to the small epitope. It will be apparent to one of skill in the art that<br>
one or more polynucleotide changes can be made to the digital antibody and still maintain<br>
its binding ability to the small epitope.<br>
[00100]	A number of "humanized" antibody molecules comprising an antigen-<br>
binding site derived from a non-human immunoglobulin have been described, including<br>
chimeric antibodies having rodent or modified rodent V regions and their associated<br>
complementarity determining regions (CDRs) fused to human constant domains. See, for<br>
example, Winter et al. Nature 349:293-299 (1991), Lobuglio et al. Proc. Nat. Acad. Sci.<br>
USA 86:4220-4224 (1989), Shaw et al. J Immunol. 138:4534-4538 (1987), and Brown et<br>
al. Cancer Res. 47:3577-3583 (1987). Other references describe rodent CDRs grafted<br>
into a human supporting framework region (FR) prior to fusion with an appropriate<br>
human antibody constant domain. See, for example, Riechmann et al. Nature 332:323-<br>
327 (1988), Verhoeyen et al. Science 239:1534-1536 (1988), and Jones et al. Nature<br>
321:522-525 (1986). Another reference describes rodent CDRs supported by<br>
recombinantly veneered rodent framework regions. See, for example, European Patent<br>
Publication No. 519,596. These "humanized" molecules are designed to minimize<br>
unwanted immunological response toward rodent anti-human antibody molecules which<br>
limits the duration and effectiveness of therapeutic applications of those moieties in<br>
human recipients. For example, the antibody constant region can be engineered such that<br>
it is immunologically inert (e.g., does not trigger complement lysis). See, e.g.<br>
PCT/GB99/01441; UK Patent Application No. 9809951.8. There are four general steps<br>
to humanize a monoclonal antibody. These are: (1) determining the nucleotide and<br>
predicted amino acid sequence of the starting antibody light and heavy variable domains<br><br>
(2) designing the humanized antibody, i.e., deciding which antibody framework region to<br>
use during the humanizing process (3) the actual humanizing methodologies/techniques<br>
and (4) the transfection and expression of the humanized antibody. See, for example,<br>
U.S. Patent Nos. 4,816,567; 5,807,715; 5,866,692; 6,331,415; 5,530,101; 5,693,761;<br>
5,693,762; 5,585,089; and 6,180,370. Other methods of humanizing antibodies that may<br>
also be utilized are disclosed by Daugherty et al, Nucl. Acids Res. 19:2471-2476 (1991)<br>
andinU-S. Patent Nos. 6,180,377; 6,054,297; 5,997,867; 5,866,692; 6,210,671;<br>
6,350,861; and PCT Publication No. WO 01/27160.<br>
[00101]	In yet another alternative, fully human antibodies may be obtained by<br>
using commercially available mice that have been engineered to express specific human<br>
immunoglobulin proteins. Transgenic animals that are designed to produce a more<br>
desirable (e.g., fully human antibodies) or more robust immune response may also be<br>
used for generation of humanized or human antibodies. Examples of such technology are<br>
Xenomouse ™ from Abgenix, Inc. (Fremont, CA) and HuMAb-Mouse® and TC<br>
Mouse™ from Medarex, Inc. (Princeton, NJ).<br>
[0100]	In an alternative, antibodies may be made recombinantly and expressed<br>
using any method known in the art. In another alternative, antibodies may be made<br>
recombinantly by phage display technology. See, for example, U.S. Patent Nos.<br>
5,565,332; 5,580,717; 5,733,743 and 6,265,150; and Winter et al, Annu. Rev. Immunol.<br>
12:433-455 (1994).<br>
[0101]	Alternatively, the phage display technology (McCafferty et al, Nature<br>
348:552-553 (1990)) can be used to produce human antibodies and antibody fragments in<br>
vitro, from immunoglobulin variable (V) domain gene repertoires from unimmunized<br>
donors. For example, existing antibody phage display libraries may be panned in parallel<br>
against a large collection of synthetic polypeptides. According to this technique,<br>
antibody V domain genes are cloned in-frame into either a major or minor coat protein<br>
gene of a filamentous bacteriophage, such as Ml 3 or fd, and displayed as functional<br>
antibody fragments on the surface of the phage particle. Because the filamentous particle<br>
contains a single-stranded DNA copy of the phage genome, selections based on the<br><br>
functional properties of the antibody also result in selection of the gene encoding the<br>
antibody exhibiting those properties. Thus, the phage mimics some of the properties of<br>
the B cell. Phage display can be performed in a variety of formats; for review see, e.g.,<br>
Johnson, Kevin S. and Chiswell, David J., Current Opinion in Structural Biology 3,564-<br>
571 (1993). Several sources of V-gene segments can be used for phage display.<br>
Clackson et al, Nature 352:624-628 (1991) isolated a diverse array of anti-oxazolone<br>
antibodies from a small random combinatorial library of V genes derived from the<br>
spleens of immunized mice. A repertoire of V genes from unimmunized human donors<br>
can be constructed and antibodies to a diverse array of antigens (including self-antigens)<br>
can be isolated essentially following the techniques described by Mark et ah, J. Mol.<br>
Biol. 222:581-597 (1991), or Griffith et al, EMBO J. 12:725-734 (1993). In a natural<br>
immune response, antibody genes accumulate mutations at a high rate (somatic<br>
hypermutation). Some of the changes introduced will confer higher affinity, and B cells<br>
displaying high-affinity surface immunoglobulin are preferentially replicated and<br>
differentiated during subsequent antigen challenge. This natural process can be<br>
mimicked by employing the technique known as "chain shuffling." Marks, et al,<br>
BioJTechnol. 10:779-783 (1992)). In this method, the affinity of "primary" human<br>
antibodies obtained by phage display can be improved by sequentially replacing the<br>
heavy and light chain V region genes with repertoires of naturally occurring variants<br>
(repertoires) of V domain genes obtained from unimmunized donors. This technique<br>
allows the production of antibodies and antibody fragments with affinities in the pM-nM<br>
range. A strategy for making very large phage antibody repertoires (also known as "the<br>
mother-of-all libraries") has been described by Waterhouse et al., Nucl. Acids Res.<br>
21:2265-2266 (1993). Gene shuffling can also be used to derive human antibodies from<br>
rodent antibodies, where the human antibody has similar affinities and specificities to the<br>
starting rodent antibody. According to this method, which is also referred to as "epitope<br>
imprinting", the heavy or light chain V domain gene of rodent antibodies obtained by<br>
phage display technique is replaced with a repertoire of human V domain genes, creating<br>
rodent-human chimeras. Selection on antigen results in isolation of human variable<br>
regions capable of restoring a functional antigen-binding site, i.e., the epitope governs<br>
(imprints) the choice of partner. When the process is repeated in order to replace the<br><br>
remaining rodent V domain, a human antibody is obtained {see PCT Publication No. WO<br>
93/06213, published April 1,1993). Unlike traditional humanization of rodent antibodies<br>
by CDR grafting, this technique provides completely human antibodies, which have no<br>
framework or CDR residues of rodent origin. It is apparent that although the above<br>
discussion pertains to humanized antibodies, the general principles discussed are<br>
applicable to customizing antibodies for use, for example, in dogs, cats, primates, equines<br>
and bovines.<br>
[0102]	Antibodies may be made recombinantly by first isolating the antibodies<br>
made from host animals, obtaining the gene sequence, and using the gene sequence to<br>
express the antibody recombinantly in host cells (e.g., CHO cells). Another method that<br>
may be employed is to express the antibody sequence in plants (e.g., tobacco), transgenic<br>
milk, or in other organisms. Methods for expressing antibodies recombinantly in plants<br>
or milk have been disclosed. See, for example, Peeters et ah (2001) Vaccine 19:2756;<br>
Lonberg, N. and D. Huszar (1995) Int.Rev.Immunol 13:65; and Pollock et al. (1999) J<br>
Immunol Methods 231:147. Methods for making derivatives of antibodies, e.g.,<br>
humanized, single chain, etc. are known in the art.<br>
[0103]	Immunoassays and flow cytometry sorting techniques such as<br>
fluorescence activated cell sorting (FACS) can also be employed to isolate antibodies that<br>
are specific for the desired small epitope.<br>
[0104]	The antibodies can be bound to many different carriers. Carriers can be<br>
active and/or inert. Examples of well-known carriers include polypropylene, polystyrene,<br>
polyethylene, dextran, nylon, amylases, glass, natural and modified celluloses,<br>
polyacrylamides, agaroses and magnetite. The nature of the carrier can be either soluble<br>
or insoluble for purposes of the invention. Those skilled in the art will know of other<br>
suitable carriers for binding antibodies, or will be able to ascertain such, using routine<br>
experimentation.<br>
[0105]	DNA encoding digital antibodies may be isolated and sequenced, as is<br>
known in the art. Generally, the antibody (such as a monoclonal antibody) is readily<br>
isolated and sequenced using conventional procedures (e.g., by using oligonucleotide<br><br>
probes that are capable of binding specifically to genes encoding the heavy and light<br>
chains of the antibody). The hybridoma cells serve as a preferred source of such cDNA.<br>
Once isolated, the DNA may be placed into expression vectors, which are then<br>
transfected into host cells such as E. coli cells, simian COS cells, Chinese hamster ovary<br>
(CHO) cells, or myeloma cells that do not otherwise produce immunoglobulin protein, to<br>
obtain the synthesis of monoclonal antibodies in the recombinant host cells. The DNA<br>
also maybe modified, for example, by substituting the coding sequence for human heavy<br>
and light chain constant domains in place of the homologous murine sequences, Morrison<br>
et al., Proc. Nat. Acad. Sci. 81:6851 (1984), or by covalently joining to the<br>
immunoglobulin coding sequence all or part of the coding sequence for a non-<br>
immunoglobulin polypeptide, fti that manner, "chimeric" or "hybrid" antibodies are<br>
prepared that have the binding specificity of a digital antibody (such as a monoclonal<br>
antibody) herein.<br>
[0106]	Digital antibodies may be characterized using methods well-known in the<br>
art, some of which are described in the Examples. For example, one method is to identify<br>
the epitope to which it binds, including solving the crystal structure of an antibody-<br>
antigen complex, competition assays, gene fragment expression assays, and synthetic<br>
polypeptide-based assays, as described, for example, in Chapter 11 of Harlow and Lane,<br>
Using Antibodies, a Laboratory Manual, Cold Spring Harbor Laboratory Press, Cold<br>
Spring Harbor, New York, 1999. In an additional example, epitope mapping can be used<br>
to determine the sequence to which a digital antibody binds. Epitope mapping is<br>
commercially available from various sources, for example, Pepscan Systems<br>
(Edelhertweg 15,8219 PH Lelystad, The Netherlands). Polypeptides of varying lengths<br>
(e.g., at least 4-6 amino acids long) can be isolated or synthesized (e.g., recombinantly)<br>
and used for binding assays with an anti-digital antibody. In another example, the<br>
epitope to which the digital antibody binds can be determined in a systematic screening<br>
by using overlapping polypeptides derived from the small epitope extracellular sequence<br>
and determining binding by the digital antibody. Certain epitopes can also be identified<br>
by using large libraries of random polypeptide sequences displayed on the surface of<br>
phage particles (phage libraries), as is well known in the art.<br><br>
[0107]	Competition assays can be used to determine whether two antibodies bind<br>
the same epitope by recognizing identical or sterically overlapping epitopes. Typically,<br>
antigen is immobilized on a multi-well plate and the ability of unlabeled antibodies to<br>
block the binding of labeled antibodies is measured. Common labels for such<br>
competition assays are radioactive labels or enzyme labels.<br>
(0108]	The digital antibodies may be linked to an agent (e.g., biotin,<br>
oligonucleotide, apatmer), a labeling agent (alternatively termed "label") such as a<br>
fluorescent molecule (such as a hapten or fluorescent bead), a solid support (such as a<br>
bead or matrix, including a microarray or multiwell plate); or any other agents known in<br>
the art. Linking may be covalent or noncovalent. Methods of linking antibodies to such<br>
agents are well known in the art. See, e.g. Kennedy et al. (Clin. Chim. Acta 70:1 -31<br>
(1976)), and Schurs et al. (Clin. Chim. Acta 81:1-40 (1977)) (describing coupling<br>
techniques, including the glutaraldehyde method, the periodate method, the dimaleimide<br>
method, them-maleimidobenzyl-N-hydroxy-succinimide ester method, all of which<br>
methods are incorporated by reference herein). Detection, including suitable labels, is<br>
further described herein.<br>
[0109]	Solid or semi-solid supports suitable for immobilizing, binding and/or<br>
Unking antibodies (and modifications to render solid supports suitable for immobilizing<br>
antibodies) are well known in the art. Examples of a solid support include: a bead<br>
(including magnetized beads), microwell plate, a protein microarray (e.g., technology<br>
owned by Zyomyx, Inc. See, e.g. US Patent No. 6,365,418). Thus, for example, CdSe-<br>
CdS core-shell nanocrystals enclosed in a silica shell can be easily derivatized for<br>
coupling to a biological molecule. Bruchez et al. (1998) Science 281: 2013-2016.<br>
Similarly, highly fluorescent quantum dots (zinc sulfide-capped cadmium selenide) have<br>
been covalently coupled to biomolecules for use in ultrasensitive biological detection.<br>
Warren and Nie (1998) Science 281: 2016-2018. Fluorescently labeled beads are<br>
commercially available from Luminex and Quantum Dot. In addition, pads, film,<br>
nanowells, or microfluid channels can also serve as a solid support. In some<br>
embodiments, the digital antibodies are immobilized, bound or linked on a solid or semi-<br>
solid surface such as polyvinylidene difluoride, nitrocellulose, agarose, and/or<br><br>
polyacrylamide gel pads. Glass slides activated with aldehyde, polylysine, or a<br>
homofunctional cross-linker can also been used. In some embodiments, the digital<br>
antibodies can be arranged in a three-dimensional array, for example in the three<br>
dimensional polyacrylamide gel pad microarray described in Mirzabekov et al., Nucleic<br>
Acids Res 24(15): 2998-3004 (1996).<br>
Methods using the sets of digital antibodies<br>
Methods for generating protein binding profiles<br>
[0110]	In another aspect, the invention further provides methods using the sets of<br>
digital antibodies. The present invention stems from the discovery that antibodies that<br>
recognize small polypeptide epitopes, often termed "cross-reactive antibodies" in the art,<br>
are useful in combination to generate a characteristic pattern of binding between<br>
antibodies in a set and protein in a sample, wherein the protein binding pattern, termed a<br>
protein binding profile, may be used to characterize, or "fingerprint" the protein sample.<br>
Thus, specificity of results and information arising from antibody binding is conferred via<br>
the binding of sets of digital antibodies to protein in a sample (whereby a specific protein<br>
binding profile is generated), rather than by binding of a single antibody that binds but<br>
one or a few proteins, as commonly used in the art for specific detection.<br>
[0111]	As a general overview, the methods involve generating a protein binding<br>
profile of a sample using any of the sets of digital antibodies described herein,<br>
comprising (a) contacting the sample with a set of digital antibodies, and (b) detection of<br>
the antibody-protein complex formed between protein in the sample and each digital<br>
antibody, if any. Detection of the antibody-protein complex may be direct detection, or<br>
via indirect detection, such as by using competition assays. Suitable detection methods<br>
are well known in the art and described herein. Detection provides information relating<br>
to presence or absence of binding, and amount (intensity of binding). Generally, a<br>
protein binding pattern comprises information regarding presence (or absence) of protein<br>
binding and/or intensity (amount) of protein binding by each digital antibody. Figure 2<br><br>
depicts a computer-simulated protein binding profile prepared using a computer-<br>
simulated set of 100 different digital antibodies. The X axis corresponds to each of the<br>
100 digital antibodies, and the Y axis represents amount of protein bound. Each bar<br>
corresponds to the amount of protein bound by each digital antibody. Additional<br>
computer-simulated protein binding profiles are shown in Figure 3.<br>
[0112]	Accordingly, in some embodiments, the invention provides methods for<br>
generating a protein binding profile of a sample, said methods comprising (a) contacting<br>
a sample with any of the sets of digital antibodies described herein under conditions that<br>
permit binding; (b) optionally removing unbound protein (in some embodiments,<br>
removing protein that is not specifically bound); and (c) detecting binding of protein to<br>
antibodies, whereby a protein binding profile is generated. It is understood that a protein<br>
binding profile includes information regarding absence of binding between a digital<br>
antibody and protein. Detecting may be qualitative and/or qualitative. In some<br>
embodiments, protein is labeled, and detection comprises detecting presence, absence<br>
and/or amount (intensity) of label on bound protein using methods well known in the art,<br>
and described herein. In other embodiments, a competition assay is used, in which<br>
labeled peptides (or mixtures of peptides) are used to compete for antibody binding with<br>
protein sample. Generally, in embodiments involving use of competitor polypeptides, the<br>
methods further comprise contacting the set of digital antibodies with competitor<br>
polypeptides (in combination with the sample and/or sequentially with the sample).<br>
Detection assay are well known in the art, and further described herein.<br>
10113]	Generally, the methods of generating a protein binding profile further<br>
comprise a step of treating the sample with a protein cleaving agent, whereby polypeptide<br>
fragments are generated. The sample can treated with a protein cleaving agent prior to<br>
step (a) of contacting the sample with the set of digital antibodies under conditions that<br>
permit binding. The protein cleaving agent may be an enzyme (such as chymotrypsin or<br>
trypsin) or a chemical agent (such as cyanogen bromide). Protein cleaving agents and<br>
methods for treatment with protein cleaving agents are well known in the art and further<br>
described herein.<br><br>
[0114]	As farther discussed herein, the protein binding profile may be correlated<br>
with information relating to the sample, such as source of sample (e.g., of a pathogen<br>
such as a bacteria and/or virus). For example, if protein binding patterns have been<br>
determined for several members of a family of bacteria, a newly generated pattern can be<br>
readily compared to see if it is a member of the family or if it represents a novel member.<br>
The protein binding profile may be recorded and stored in a library of protein binding<br>
profiles (such as a database), as further discussed herein.<br>
[0115]	In another aspect, the invention provides a protein binding profile<br>
generated using any of the methods described herein.<br>
[0116]	It is understood that a protein binding profile may be generated without<br>
the need to have prior knowledge of the identity of the protein being analyzed, and both<br>
known and unknown proteins may be detected. Because information regarding contents<br>
of a protein sample is not required, the protein binding profile may be used to identify or<br>
detect a previously unknown agent, such as a novel pathogen. For example, a protein<br>
binding profile different from that previously seen may indicate a previously unknown<br>
agent.<br>
[0117]	Methods and conditions for antibody binding are well known in the art and<br>
further described herein, m some embodiments, contacting with two or more antibodies<br>
is sequential (as when one antibody is contacted with the sample, then removed, another<br>
antibody is contacted with the sample and removed, and so on). In other embodiments,<br>
contacting with a set of digital antibodies is in parallel, for example, as when a group of<br>
antibodies are contacted with the sample simultaneously. In some embodiments, several<br>
groups of a set of digital antibodies are serially contacted with the sample, for example,<br>
group 1 is contacted and removed, group 2 is contacted and removed, and so on.<br>
[0118]	Methods and conditions for detection of protein-antibody binding are well<br>
known in the art. Detection of protein bound to a digital antibody may be qualitative<br>
and/or qualitative. Binding of a multiplicity (generally, a large multiplicity) of antibodies<br>
in a set is detected. In some embodiments, binding of at least about 95%, at least about<br>
90%, at least about 75%, at least about 50%, or at least about 30% of antibodies in a set<br><br>
are detected. In some embodiments, binding of each antibody in a set is detected.<br>
Methods for detection of antibody binding are well known in the art and include, e.g.,<br>
ELISAs, fluorescent immunoassays, Western and dot blots, immunoprecipitations,<br>
competition assays using competitor polypeptides, and focal immunoassays. Alterations<br>
in protein binding profile can be determined by running parallel tests on test and control<br>
samples and noting any differences in results between the samples. Results of ELISAs,<br>
for example, can be directly related to the amount of protein present. In some<br>
embodiments, the protein is labeled, and the label is detected. In other embodiments,<br>
competitor polypeptides (as further described herein) and labeled, and binding of<br>
competitor polypeptides is detected. Suitable labels are well known in the art, and further<br>
described herein.<br>
Methods for generating libraries of protein binding profiles<br>
[0119]	A library of protein binding profiles can be generated by compiling<br>
protein binding profiles for two or more different samples, such as samples of pathogenic<br>
or non-pathogen bacteria. The protein binding profiles can be compared with each other,<br>
either qualitatively or quantitatively, in order to discern similarities and/or differences in<br>
protein binding patterns. The protein binding profiles in the library can also be used in<br>
methods involving comparative assessment of the protein binding profiles of sample from<br>
different sources (for example, from different tissues or cell types, disease states, or<br>
different microorganisms). For example, the libraries of protein binding profiles are<br>
useful in methods for characterizing protein samples, methods for diagnosis of bacterial<br>
or viral infection, and/or in methods for detection and/or taxonomic classification of a<br>
bacteria or infectious agent, as further described herein. For example, a protein binding<br>
profile prepared from an unidentified sample may be used to characterize and/or identify<br>
the sample by comparison with a library of protein binding profiles of known bacterial<br>
pathogens.<br>
[0120]	Thus, in another aspect, the invention provides methods for generating a<br>
library of protein binding profiles, comprising the steps of (a) contacting a sample with<br><br>
any of the sets of digital antibodies described herein under conditions that permit binding;<br>
(b) optionally removing unbound protein (in some embodiments, removing protein that is<br>
not specifically bound); (c) detecting binding of protein to antibodies, whereby a protein<br>
binding profile is generated; and (d) repeating steps (a) through (c) with at least two<br>
different samples. In some embodiments, the library comprises at least about 5,10,15,<br>
20,25,30,35,40,45,50,75,100,150,200,300,400,500,600,700, 800,900, or more<br>
(such as 1000,2000,3000,4000 or more) binding profiles. In some embodiments, the<br>
methods further comprise step of treating the sample with a protein cleaving agent prior<br>
to step (a) of contacting the sample with the set of digital antibodies under conditions that<br>
permit binding. In some embodiments, the library further comprises epitope identity<br>
information as described herein.<br>
[01211	In another aspect, the invention provides libraries of protein binding<br>
profiles, wherein the library is prepared using any of the methods described herein.<br>
[0122]	It is understood that the library of protein binding profiles may comprise<br>
binding profiles that were generated at different times, by different individual, and/or at<br>
different facilities. In some embodiments, the binding profiles are generated at the same<br>
time or different times, and are later grouped together into different libraries. A library of<br>
binding profiles can constantly be updated when a new binding profile is generated.<br>
[0123]	As is well known in the art, a library of binding profiles can be processed<br>
using software and stored in a database. The database can be constantly updated as more<br>
samples are analyzed and more binding profiles are generated. The binding profiles can<br>
be categorized through various methods, such as by the source of the sample, similarity<br>
of the binding profiles, etc. For example, the binding profile of bacteria samples may be<br>
grouped together based on their phenotypes or genotypes. Additionally, existing<br>
taxonomic classifications can be relied on in grouping the binding profiles. In some<br>
embodiments, the library comprises protein binding profiles generated from a sample<br>
comprising or derived from a prokaryotic cell, such as a bacterium, an eukaryotic cell,<br>
such as a cancer or precancerous cell, and/or other microorganisms, such as a virus<br>
(including diagnosis of bacterial and/or viral infection).<br><br>
[0124]	In another aspect, the invention provides libraries of protein binding<br>
profiles generated using any of the methods described herein.<br>
Methods using the protein binding profiles<br>
[0125]	In another aspect, the invention provides methods using the protein<br>
binding profiles (including libraries of protein binding profiles) generated according to<br>
the methods of the invention. Thus, the invention provides methods for characterizing a<br>
test sample (such as a test sample suspected of comprising a sample of interest); methods<br>
for detecting presence or absence and/or identifying a test sample; methods for<br>
characterizing a cell, bacteria and/or virus; methods for identifying a test protein;<br>
methods for characterizing, detecting presence and/or absence and/or identifying a<br>
protein complex; and methods for screening, as further described herein.<br>
Methods for identifying a protein using protein binding profile and epitope amino acid<br>
sequences<br>
[0126]	The invention also provides methods for identifying a protein using (a) a<br>
protein binding profile generated from a sample comprising the protein; and (b) the<br>
amino acid sequences of the epitope recognized by digital antibodies that bind sample<br>
protein, whereby binding of a set of about 5 or more (such as about 6, about 7, about 8, or<br>
more) digital antibodies (generally, providing about 15-18 of amino acid sequence<br>
information) provides sufficient information relating to the amino acid content of the<br>
protein such that the protein may be identified. By way of example, the simultaneous<br>
binding of digital antibodies recognizing the epitopes QAP, TPG, LTG, VSR, and WDQ<br>
to a test protein identifies that protein as HCV NS3 protein, for HCV NS3 protein is the<br>
only known protein comprising these six 3mer amino acid sequences.<br>
[0127]	Accordingly, in one aspect, the invention provides methods for identifying<br>
a test protein, said methods comprising (a) contacting a sample comprising the test<br>
protein with any of the sets of digital antibodies described herein; (b) optionally<br>
removing unbound protein; (c) detecting presence or absence of binding of protein to<br><br>
antibodies in the set, wherein at least about five digital antibodies bind protein; wherein<br>
presence of binding indicates presence or absence of at least about five epitopes in the<br>
protein, wherein the identity (i.e., linear amino acid sequence) (and, in some<br>
embodiments, amount) of the at least about five epitopes is used to identify the protein.<br>
In some embodiments, the binding and epitope identity (i.e., amino acid sequence of<br>
cognate small epitope) of about 6, about 7, about 8, about 9, about 10, about 11, about 12,<br>
about 13, about 14, about 15, about 20, about 25, about 30, or more (such as about 40, 50,<br>
60, or more) digital antibodies are used to identify the test protein.<br>
[0128]	In some embodiments, the methods further comprise the comparison of<br>
the epitope identity data with a database comprising protein sequence information (such<br>
as sequences of nucleotides or amino acids). The database may consist of nucleotide or<br>
amino acid sequences of expressed sequence tags (ESTs). Alternatively, the database<br>
may consist of sequences of genes at the nucleotide or amino acid level. The database can<br>
include, without limitation, a collection of nucleotide sequences, amino acid sequences,<br>
or translations of nucleotide sequences included in the genome of any species. A<br>
database of information relating to proteins, e.g., sequences of nucleotides or amino<br>
acids, is typically analyzed via a computer program or a search algorithm which is<br>
optionally performed by a computer. Information from sequence databases is searched for<br>
best matches with data and information obtained from the methods of the present<br>
invention (see e.g., Yates (1998) J. Mass Spec. 33:1-19; Yates et al., U.S. Pat. No.<br>
5,538,897; Yates et al., U.S. Pat. No. 6,017,693). Any appropriate algorithm or computer<br>
program useful for searching a database can be used. Search algorithms and databases are<br>
constantly updated, and such updated versions will be used in accordance with the<br>
present invention. Examples of programs or databases can be found on the World Wide<br>
Web (WWW) at http://base-peak.wiley.com/, http://mac-mann6.embl-<br>
heidelberg.de/MassSpec/Software.html, http ://www.mann.embl-<br>
heidelberg.de/Services/PeptideSearch/PeptideSearchIn-tro.html,<br>
ftp://ftp.ebi.ac.uk/pub/databases/, and http://donatello.ucsf.ed- u. U.S. Pat. Nos.<br>
5,632,041; 5,964,860; 5,706,498; and 5,701,256 also describe algorithms or methods for<br>
sequence comparison. Other examples of databases include the Genpept database, the<br>
GenBank database (described in Burks et al. (1990) Methods in Enzymology 183: 3-22,<br><br>
EMBL data library (described in Kahn et al. (1990) Methods in Enzymology 183:23-31,<br>
the Protein Sequence Database (described in Barker et al. (1990) Methods in<br>
Enzymology 183: 31-49, SWISS-PROT (described in Bairoch et al. (1993) Nucleic Acids<br>
Res., 21: 3093-3096, and PIR-International (described in (1993) Protein Seg. Data Anal.<br>
5:67-192). In some embodiments, the present invention also provides methods of<br>
determining the identity of a protein wherein a programmable digital computer is used to<br>
access a database containing one or more protein sequence databases. In some<br>
embodiments, the database further comprises epitope sequence information (i.e., the<br>
sequences of cognate small epitopes recognized by a set of digital antibodies). In another<br>
aspect, the invention provides a database comprising epitope sequence information, for<br>
use in any of the methods for identifying proteins described herein.<br>
[0129]	The protein sample may comprise pure protein, or, in some embodiments,<br>
a substantially pure protein. It is generally understood that the protein sample can be a<br>
protein sample prepared from a variety of sources described herein. In some<br>
embodiments, a protein binding profile generated from a test sample suspected of<br>
comprising a protein of interest (e.g., a bacterial or viral protein) is compared with a<br>
protein binding profile generated from a reference sample (i.e., a control sample), and the<br>
difference between the profiles is analyzed for epitope binding and epitope identity<br>
information as described above, wherein a protein of interest is identified. As an<br>
example, Figure 3 depicts a computer-simulated protein binding profiles prepared from a<br>
sample comprising one or more viral proteins (panel A), and a control sample (panel B),<br>
using a computer-simulated set of 100 different digital antibodies. The X axis<br>
corresponds to each of the 100 digital antibodies, and the Y axis represents amount of<br>
protein bound. Each bar corresponds to the amount of protein bound by each digital<br>
antibody. Panel C depicts the protein binding profile following subtraction of the control<br>
protein binding profile from the viral sample protein binding profile.<br><br>
Methods comprising detection<br>
[0130]	The invention provides methods for detecting (including diagnosing),<br>
determining presence or absence of, and/or identifying a sample of interest, such as a<br>
sample comprising or derived from cell, such as a prokaryotic cell (such as a bacterium,<br>
such as a pathogenic bacterium); a eukaryotic cell, a mammalian cell (such as a<br>
cancerous or precancerous cell), or detecting and/or diagnosing presence or absence of<br>
other infection, contamination, disease, and/or abnormality (including detection of type<br>
and/or stage of cancer, or identification of a precancerous cell). In some embodiments,<br>
the cell is mammalian (such as human),. In other embodiments, the cell is of non-human<br>
mammal (such as cow, horse, or dog). In some embodiments, the cell is human, murine,<br>
or primate.<br>
[0131]	Generally, the methods involve generation of the protein binding profile of<br>
a test sample suspect of comprising a sample of interest, such as a sample suspected of<br>
comprising a cell, bacteria and/or virus, or a sample for which detection of presence or<br>
absence of cell (such as a mammalian cell, such as human), bacteria and/or virus is<br>
desired. The protein binding profile of the test sample is compared with the protein<br>
binding profile of a known cell, bacteria and/or virus and./or compared to a library of<br>
binding profiles, e.g., a library of binding profiles of known bacteria and/or viruses. The<br>
existence of significant similarity (including a match) between the test binding profile<br>
and the control binding profile (in some embodiments, a member(s) of a library of<br>
binding profiles) indicates the presence and/or identity of the sample of interest. Absence<br>
of a match may indicate the existence of a previously unidentified sample, for example, a<br>
previously unidentified pathogenic microbe, and/or absence of a particular bacteria<br>
and/or virus in a sample. As used herein, "absence" encompasses de minimum and/or<br>
background levels of binding.<br>
[0132]	Accordingly, in one aspect, the invention provides methods for<br>
determining presence or absence of and/or identifying a sample of interest, said methods<br>
comprising (a) contacting a test sample suspected of comprising the sample of interest<br>
with any of the sets of digital antibodies described herein under conditions that permit<br><br>
binding; (b) optionally removing unbound protein; (c) detecting binding of protein to<br>
antibodies, whereby a protein binding profile is generated; and (d) comparing the protein<br>
binding profile of the test sample with the protein binding profile of a reference sample,<br>
whereby presence or absence and/or identity of the sample of interest is determined by<br>
the comparison.<br>
[0133]	In another aspect, the invention provides methods for detecting presence<br>
or absence of and/or identifying a sample of interest, said methods comprising (a)<br>
contacting a test sample suspected of comprising the sample of interest with any of the<br>
sets of digital antibodies described herein under conditions that permit binding; (b)<br>
optionally removing unbound protein; (c) detecting binding of protein to antibodies,<br>
whereby a protein binding profile is generated; and (d) comparing the protein binding<br>
profile of the test sample with a library of protein binding profiles generated by any of the<br>
methods described herein, whereby presence or absence and/or identity of the sample of<br>
interest in the test sample is determined by the comparison.<br>
[0134]	In one aspect, the invention provides methods for determining presence or<br>
absence of and/or identifying a cell, bacteria or virus in a test sample, said methods<br>
comprising (a) contacting the test sample with any of the sets of digital antibodies<br>
described herein under conditions that permit binding; (b) optionally removing unbound<br>
protein; (c) detecting binding of protein to antibodies, whereby a protein binding profile<br>
is generated; and (d) comparing the protein binding profile of the test sample with the<br>
protein binding profile of a reference sample, whereby presence or absence and/or<br>
identity of the cell, bacteria or virus in the test sample is determined by the comparison.<br>
[0135]	In another aspect, the invention provides methods for detecting presence<br>
or absence of and/or identifying a cell, bacteria and/or virus in a test sample, said<br>
methods comprising (a) contacting the test sample with any of the sets of digital<br>
antibodies described herein under conditions that permit binding; (b) optionally removing<br>
unbound protein; (c) detecting binding of protein to antibodies, whereby a protein<br>
binding profile is generated; and (d) comparing the protein binding profile of the test<br>
sample with a library of protein binding profiles generated by any of the methods<br><br>
described herein, whereby presence or absence and/or identity of the cell, bacteria or<br>
virus in the test sample is determined by the comparison.<br>
[0136]	In another aspect, the invention provides methods for identifying a protein,<br>
said method comprising comparing a protein binding profile of the test protein with a<br>
protein binding profile of a reference protein, wherein the protein binding profile of the<br>
reference protein is correlated with the identity of the reference protein; wherein the<br>
identity of the test protein is determined by the comparison; and wherein the protein<br>
binding profiles have been generated using any of the methods for generating a protein<br>
binding profile described herein.<br>
[0137]	Rapid and accurate microbial identification is critical in diagnosing<br>
diseases, predicting on-coming health hazards, monitoring potential contamination in<br>
stored foods and grains, regulating bioprocessing operations, and recognizing<br>
environmental contamination, including contamination using bio warfare. In some<br>
embodiments, the bacteria and/or virus is pathogenic. In other embodiments, the bacteria<br>
and/or virus is not pathogenic. In still other embodiments, the bacteria and/or virus is<br>
novel, or is a variant of a previously identified bacteria and/or virus.<br>
[0138]	In some embodiments, the type, subtype, variant, phylum, class, order,<br>
family, genus and/or species of a bacteria is identified. As is well known in the art, the<br>
study of microbial protein expression may be useful for determining the relationships<br>
between members of a genera and evolutionary relationships between members of<br>
different genera. In some embodiments, the helpful degree of homology and relationship<br>
between species and between members of different genera may be established. For<br>
example, the protein binding profiles of different microbe species can be assembled and<br>
compared, and the result can be input into a database and used to generate a family<br>
relationship dendrograph based on their protein binding profiles. As is apparent, once a<br>
family relationship dendrograph is established, a new microbial sample can be classified<br>
based its protein binding profile.<br>
[0139]	In still other embodiments, protein binding profiles of microbial strains<br>
may identify strains exhibiting novel phenotypes, such as chemical resistance, altered<br><br>
mode of pathogenity, or varied response to stress, nutrient limitation or genetic<br>
manipulation, can be compared via protein binding profile to correlate relative protein<br>
abundances associated with these conditions.<br>
[0140]	In other embodiments, an absence of significant similarity and/or match<br>
between binding profile of a test sample, and known protein binding profiles may identify<br>
a new strain or a new variant. In still further embodiments, the new strain/variant can be<br>
categorized based on the extent of similarity of the binding profile to other binding<br>
profiles.<br>
[0141]	Preparation of samples, and exemplary samples are described herein and<br>
well known in the art. It is understood that a sample comprising bacteria and/or virus can<br>
be removed from its source (e.g., an individual, food, air, water, and other environmental<br>
samples); grown in culture, whereby the bacteria and/or virus is multiplied, enriched<br>
and/or purified (in some embodiments, substantially purified) prior to preparation of<br>
protein sample. In some embodiments, proteins are prepared from a whole cell extract.<br>
In other embodiments, proteins can be pre-fractionated via subcellular location (e.g.,<br>
membrane and cytoplasmic) or different physical and functional properties. The protein<br>
can also be extracted from the supernatant of the culture. In some embodiments, a viral<br>
protein sample is prepared from serum and/or plasma and/or any other suitable body<br>
fluid.<br>
[0142]	In some embodiments, the methods of the invention are used to detect,<br>
determine presence or absence or and/or identify a cancer cell and/or tissue, including (in<br>
some embodiments) detecting (diagnosis of) cancer type, stage, and/or level of<br>
differentiation. In some embodiments, precancerous cells are detected.<br>
[0143]	Thus, in one aspect, the invention provides methods for determining<br>
(diagnosing) presence or absence of and/or identifying cancerous cells or tissue<br>
(including identifying type of cancer and/or stage of cancer) in a test sample, said<br>
methods comprising (a) contacting the test sample with any of the sets of digital<br>
antibodies described herein under conditions that permit binding; (b) optionally removing<br>
unbound protein; (c) detecting binding of protein to antibodies, whereby a protein<br><br>
binding profile is generated; and (d) comparing the protein binding profile of the test<br>
sample with the protein binding profile of a reference sample, whereby presence or<br>
absence and/or identity of the cancerous cells or tissue in the test sample is determined by<br>
the comparison.<br>
[0144]	In another aspect, the invention provides methods for detecting presence<br>
or absence of and/or identifying cancerous cells or tissue in a test sample, said methods<br>
comprising (a) contacting the test sample with any of the sets of digital antibodies<br>
described herein under conditions that permit binding; (b) optionally removing unbound<br>
protein; (c) detecting binding of protein to antibodies, whereby a protein binding profile<br>
is generated; and (d) comparing the protein binding profile of the test sample with a<br>
library of protein binding profiles generated by any of the methods described herein,<br>
whereby presence or absence and/or identity of the cancerous cells or tissue in the test<br>
sample is determined by the comparison.<br>
Methods for characterizing a sample<br>
[0145]	The protein binding profile may be used to characterize a sample, which<br>
generally involves comparative assessment of the protein binding profiles of sample from<br>
different sources (for example, from different tissues or cell types, disease states, cells<br>
subjected to differential treatment, or different microorganisms). In some embodiments,<br>
the methods involve the comparison of a protein binding profile generated from a test<br>
sample with a reference protein binding profile (such as protein binding profile(s)<br>
generated from a control sample, or normal values), or library of profiles generated using<br>
any of the methods described herein.<br>
[0146]	Thus, in one aspect, the invention provides methods for characterizing a<br>
test sample (suspected of comprising a sample of interest), said methods comprising (a)<br>
contacting the test sample with any of the sets of digital antibodies described herein under<br>
conditions that permit binding; (b) optionally removing unbound protein; (c) detecting<br>
binding of protein to antibodies, whereby a protein binding profile is generated; and (d)<br><br>
comparing the protein binding profile of the test sample with the protein binding profile<br>
of a reference sample, whereby the test sample is characterized by the comparison.<br>
[0147]	In another aspect, the invention provides methods for characterizing a test<br>
sample (such as a test sample suspected of comprising a sample of interest), said methods<br>
comprising (a) contacting the test sample with any of the sets of digital antibodies<br>
described herein under conditions that permit binding; (b) optionally removing unbound<br>
protein; (c) detecting binding of protein to antibodies, whereby a protein binding profile<br>
is generated; and (d) comparing the protein binding profile of the test sample with a<br>
library of protein binding profiles generated by any of the methods described herein,<br>
whereby the test sample is characterized using the comparison. In some embodiments,<br>
characterizing comprises classification of the bacteria or virus in a sample (such as<br>
classification into a phylum, class, order, family genus, species, and/or any subtype,<br>
variant and/or subgroup thereof). In other embodiments, characterizing comprises<br>
classification of a cell in a sample, as, e.g., cancerous or precancerous. In some<br>
embodiments, the type, stage, grade, and/or other relevant diagnostic characteristic of the<br>
cancer is determined.<br>
Methods for characterizing protein binding complexes and screening<br>
[0148]	Specific protein-protein interactions are important to most cellular<br>
functions. Frequently, these protein-protein interactions changes in response to<br>
extracellular stimulation. Measurement of protein-protein interactions in a cell can help<br>
to validate the physiological significance of the interaction, as well as aid in identifying<br>
changes that occur in a cell or organism in response to physiological stimuli (including<br>
stimuli due to use of a pharmaceutical drug or other chemical). Accordingly, in one<br>
aspect, the present invention provides methods for characterizing protein complexes.<br>
Generally, the methods for characterizing protein complexes involves methods for<br>
generating protein binding profiles of protein complexes. As a general overview, a<br>
protein complex (such as a receptor-associated protein complex, or a signal transduction<br>
associated protein complex) is prepared using methods known in the art, and the isolated<br><br>
protein complex, either denatured or in its native condition, and a protein binding profile<br>
is generated using any of the methods described herein.<br>
[0149]	Protein binding profiles generated from isolated protein complexes may be<br>
used in any of the methods described herein, such as methods for characterizing a sample<br>
(in this case, a sample comprising the protein complex), and methods for detecting<br>
presence or absence of and/or identifying a sample.<br>
[0150]	In another aspect, invention provides methods can be used to test chemical<br>
composition to determine whether the chemical composition promotes and/or disrupts<br>
one or more interactions within a protein complex. It is believed that a chemical<br>
composition that promotes and/or disrupts protein-protein interaction within may<br>
constitute a useful pharmaceutical Furthermore, future characterization and/or<br>
identification of components) within the complex that are reduced, increased, absent<br>
when normally present, or present when normally absent, may provide information<br>
relating to potential drug targets.<br>
[0151]	It is contemplated that a cell or population of cells may be contacted with<br>
a test chemical composition, protein prepared from the cell(s), and protein binding<br>
profiles prepared and analyzed using any of the methods described herein. Accordingly,<br>
in one aspect, the invention provides method for screening for a chemical composition<br>
that alters protein composition of a sample, said method comprising (a) treating a test<br>
sample with a chemical composition; (b) contacting the test sample (or protein prepared<br>
and/or enriched and/or purified from the test sample) treated with the chemical<br>
composition with any of the sets of digital antibodies described herein under conditions<br>
that permit binding; (b) optionally removing unbound protein; (c) detecting binding of<br>
protein to antibodies, whereby a protein binding profile is generated; and (d) comparing<br>
the protein binding profile of the test sample treated with the chemical composition with<br>
a protein binding profile of a reference sample not treated with the chemical composition,<br>
whereby the chemical composition is characterized by the comparison.<br>
[0152]	Libraries and other collections of small molecule chemical compositions<br>
are well known in the art.<br><br>
[0153J	In some embodiments, the screening methods are useful for studying any<br>
protein-protein interactions in a variety of contexts, including formation of functional<br>
transcription complexes (e.g., estrogen receptor complexes), signal transduction<br>
pathways, cytoskeletal organization (e.g., microtubule polymerization), polypeptide<br>
hormone receptor-ligand binding, organization of multi-subunit enzyme complexes, and<br>
the like, are of particular interest.<br>
[0154]	In some embodiments, the protein complexes are isolated based on their<br>
size or activity, through traditional methods such as column chromatography. In other<br>
embodiments when a DNA/RNA-binding protein complex is to be analyzed, the<br>
DNA/RNA sequence to which the complex binds to can be used to isolate the protein<br>
complex.<br>
[0155]	In some embodiment, one of the proteins in the complex is known or<br>
identified. The known/identified protein can then serve as a bait to isolate its binding<br>
partners. Ideally, endogenous proteins can serve as bait if an antibody or other reagents<br>
exist that allow specific isolation of the protein with its bound partners, for example,<br>
through affinity chromatography or immunoprecipitation. A more generic way is to<br>
"tag" the known or identified protein with a sequence readily recognized by an antibody<br>
or ligand specific for the tag. Common affinity tags are known in the art, and include<br>
Glutathione S-transferase, His6, Calmodulin-binding peptide, haemaglutinin, myc, and<br>
FLAG tags.<br>
Compositions<br>
[0156]	In another aspect, the invention provides compositions comprising any of<br>
the sets of digital antibodies described herein. In some embodiments, the compositions<br>
are for use in any of the methods described herein (such as a method for generating a<br>
protein binding profile). In some embodiments, the compositions are for use in methods<br>
for generating protein binding profiles (including methods for generating libraries of<br>
protein binding profiles). In other embodiments, the compositions are for use in methods<br><br>
for characterizing a test sample (such as a test sample suspected of comprising a sample<br>
of interest); methods for detecting presence or absence and/or identifying a test sample;<br>
methods for characterizing a cell, bacteria and/or virus; methods for identifying a test<br>
protein; methods for characterizing, detecting presence and/or absence and/or identifying<br>
a protein complex; and methods for screening, as further described herein.<br>
[0157]	In some embodiments, the invention provides compositions comprising<br>
complexes of protein with any of the sets of digital antibodies described herein. In some<br>
embodiments, the set of digital antibodies is arrayed. In still other embodiments, the<br>
antibodies and/or the protein are labeled. In some embodiments, the antibodies are<br>
immobilized (linked and/or attached) to a solid or semi-solid surface, such as an array. In<br>
other embodiments, the antibodies are labeled antibodies. In other embodiments, the<br>
protein is labeled. In still other embodiments, the antibodies and protein are labeled. In<br>
some embodiments, the invention provides compositions comprising any of the sets of<br>
digital antibodies described herein, a sample, and a set of competitor polypeptides,<br>
wherein said competitor polypeptides comprise the cognate amino acid sequence(s) of<br>
one of more digital antibodies in the set of digital antibodies. In some embodiments, the<br>
competitor polypeptides are labeled.<br>
Kits<br>
[0158]	The invention also provides kits comprising any of the sets of digital<br>
antibodies described herein. Thus, in some embodiments, the invention provides kits<br>
comprising at least about 15 digital antibodies, wherein each digital antibody binds a<br>
different epitope, and wherein each digital antibody binds an epitope consisting of 3<br>
consecutive amino acids, 4 consecutive amino acids, or 5 consecutive amino acids. In<br>
some embodiments, the kit comprises at least about any of 20, 30,40, 50,60, 70, 80, 90,<br>
100,110,125,150,175,200, 300,400, 500, 600, 700, 800,900,1000, or more<br>
antibodies. In other embodiments, the kit comprises at least about 100 antibodies. In<br>
other embodiments, the kit comprises at least about 100 digital antibodies that recognize<br>
epitopes consisting of 3 consecutive amino acids (also termed a 3mer epitope). In some<br><br>
embodiments, the kit comprises at least about 100 digital antibodies that recognize<br>
epitopes consisting of 3 consecutive amino acids, and at least about 100 digital antibodies<br>
that recognize epitopes consisting of 4 consecutive amino acids (also termed a 4mer<br>
epitope). In some embodiments, the kit comprises at least about at least about 100 digital<br>
antibodies that recognize epitopes consisting of 3 consecutive amino acids, at least about<br>
100 digital antibodies that recognize epitopes consisting of 4 consecutive amino acids,<br>
and at least about 100 digital antibodies that recognize epitopes consisting of 5<br>
consecutive amino acids. In another embodiment, the kit comprises at least about 1000<br>
digital antibodies that recognize epitopes consisting of 4 consecutive amino acids. The<br>
antibodies may be immobilized and/or linked (attached) to a surface, such as an array. In<br>
some embodiments, the kit further comprises a label (such as a label for use in labeling<br>
protein). In still other embodiments, the antibodies are labeled. In still other<br>
embodiments, the kit comprises competitor polypeptide(s). Generally, competitor<br>
polypeptide(s) comprise the amino acid sequences of one or more cognate epitopes<br>
recognized by a set of digital antibodies. In some embodiments, the competitor<br>
polypeptides are labeled. In some embodiments, the competitor polypeptides are MAPs.<br>
[0159]	In some embodiments, the kits of digital antibodies further comprise<br>
instructions for use of the digital antibodies in any of the methods described herein (such<br>
as methods of generating a protein binding profile, including methods for generating<br>
libraries of protein binding profiles). In some embodiments, the instructions are for<br>
methods for characterizing a test sample (such as a test sample suspected of comprising a<br>
sample of interest); methods for detecting presence or absence and/or identifying a test<br>
sample; methods for characterizing a cell, bacteria and/or virus; methods for identifying a<br>
test protein; methods for characterizing, detecting presence and/or absence and/or<br>
identifying a protein complex; and methods for screening. In other embodiments, the<br>
instructions are for methods for detecting (including diagnosing), determining presence<br>
or absence of, and/or identifying a sample of interest, such as a sample comprising or<br>
derived from a prokaryotic cell (such as a bacterium, such as a pathogenic bacterium); a<br>
eukaryotic cell (such as a cancerous or precancerous cell), or detecting and/or diagnosing<br>
presence or absence of other infection, contamination, disease, and/or abnormality<br>
(including detection of type and/or stage of cancer, or identification of a precancerous<br><br>
cell). Instructions supplied in the kits of the invention are typically written instructions<br>
on a label or package insert (e.g., a paper sheet included in the kit), but machine-readable<br>
instructions (e.g., instructions carried on a magnetic or optical storage disk) are also<br>
acceptable.<br>
[0160]	In some embodiments, the kit comprises an array of digital antibodies,<br>
said array comprising any of the sets of digital antibodies described herein. In some<br>
embodiments, the antibodies are immobilized (linked and/or attached) to a solid or semi-<br>
solid surface, such as an array. In other embodiments, the antibodies are labeled<br>
antibodies.<br>
[0161]	In still other embodiments, the kit comprises any of the libraries of protein<br>
binding profiles described herein. In some embodiments, the kit comprises a reference<br>
protein binding sequence.<br>
[0162]	In still other embodiments, the kit further comprises a label. In some<br>
embodiments, the label is for labeling protein.<br>
[0163]	In still other embodiments, the kit comprises a set of competitor<br>
polypeptides, wherein said competitor polypeptides comprise the cognate amino acid<br>
sequence(s) of one of more digital antibodies in the set of digital antibodies. In some<br>
embodiments, the competitor polypeptides are labeled competitor polypeptide(s).<br>
[0164]	The kits of this invention are in suitable packaging. Suitable packaging<br>
includes, but is not limited to, vials, bottles, jars, flexible packaging (e.g., sealed Mylar or<br>
plastic bags), and the like. Kits may optionally provide additional components such as<br>
buffers and interpretive information.<br><br>
[0165]	As used herein, "sample" encompasses a variety of sample types and/or<br>
origins, such as blood and other liquid samples of biological origin, solid tissue samples<br>
such as a biopsy specimen or tissue cultures or cells derived therefrom, and the progeny<br>
thereof. The definition also includes samples that have been manipulated in any way<br>
after their procurement, such as by treatment with reagents, solubilization, or enrichment<br>
for certain components, such as a subgroup of proteins. The term "sample" encompasses<br>
a clinical sample, and also includes cells in culture, cell supematants, cell lysates, serum,<br>
plasma, biological fluid, and a pure or enriched bacterial or viral sample derived from any<br>
of these, for example, as when a sample is cultured in order to increase, enrich and/or<br>
substantially purify a bacterial or viral sample therefrom (or, in some embodiments, to<br>
increase amount of a sample comprising bacteria and/or virus). A sample can be from a<br>
microorganism, e.g., bacteria, yeasts, viruses, viroids, molds, fungi, plants, animals,<br>
including mammals such as humans. A sample may comprise a single cell or more than a<br>
single cell.<br>
[0166]	These samples can be prepared by methods known in the art such as<br>
lysing, fractionation, purification, including affinity purification, FACS, laser capture<br>
microdissection (LCM) or isopycnic centrifugation. In some embodiments, subcellular<br>
fractionation methods are used to create enriched cellular or subcellular fractions, such as<br>
subcellular organelles including nuclei, mitochondria, golgi apparatus, endoplasmic<br>
reticulum, chloroplasts, heavy and light membranes and cytoplasm.<br>
[0167]	In one embodiment, the sample comprises substantially intact protein<br>
complexes. In some embodiments, the protein complex is a receptor-associated protein<br>
complex or a signal transduction associated protein complex prepared from, e.g., nucleic<br>
acid associated receptors (e.g., the estrogen receptor), or membrane associate receptors<br>
(e.g., epidermal growth factor receptor, IL-6 receptor, stress/apoptosis pathways,<br>
chemokine pathways, MMP transcription pathway,) or cell cycle pathways. Methods for<br>
preparing such complexes are well known in the art.<br>
[0168]	In some embodiments, the sample comprises or is derived from (or<br>
suspected of comprising) a pathogenic bacteria, such as Shigella species, Salmonella<br><br>
typhi, Salmonella typhimurium, Yersinia enterocolitica, Yersinia pestis, Vibrio cholerae,<br>
Campylobacter jejuni, Helicobacter jejuni, Pseudomonas aeruginosa, Haemophilus<br>
influenzae, and Bordetella pertussis (whooping cough), Vibrio cholerae, and E. coli,<br>
including Diarrheagenic E. Coli, enteroaggregative E. coli (EaggEC),<br>
enterohaemorrhagic E. coli (EHEC), enteroinvasive E. coli (EIEC), enteropathogenic E.<br>
coli (EPEC) and enterotoxigenic E. coli (ETEC), Uropathogenic E. coli (UPEC), and<br>
neonatal meningitis E. coli (NMEC). Other pathogenic bacteria include Bacilus<br>
anthracis, Clostridium botulinum, Francisella tularensis, Burkholderia pseudomallei,<br>
Coxiella burnetti, Brucella species, Burkholderia mallei, Staphylococcus, drug-resistent<br>
Streptococcus, Rickettsia prowazekii, Shigella species, Salmonella, Listeria<br>
monocytogenes, Campylobacter jeluni, and Yersinia enterocolitica.<br>
[0169]	In other embodiments, the sample comprises or is derived from (or is<br>
suspected of comprising) a viruses , such as Hepatitis C Virus, Hepatitis B Virus, Human<br>
Immunodeficiency Virus, and Cytomegalovirus. Viruses can also be food and<br>
waterbome pathogens; such as Caliciviruses and Hepatitis A viruses. Other viruses of<br>
interest include, but not limited to, Variola major (smallpox) and other pox viruses,<br>
Arenaviruses (including LCM, Junin viruses, Machupo viruses, Guanarito viruses, Lassa<br>
Fever viruses), Bunyaviruses (including Hantaviruses, Rift Valley Fever viruses),<br>
Flaviruses (including Dengue viruses), Filoviruses (including Ebola viruses and Marburg<br>
viruses), Tickborne hemorrhagic fever viruses (including Crimean-Congo Hemorrhagic<br>
fever viruses), Tickborne encephalitis viruses, yellow fever viruses, influenza viruses,<br>
Rabies virus, West Nile Viruses, La Crosse viruses, California encephalitis viruses,<br>
Venezuelan Equine Encephalomyelitis viruses, Eastern Equine Encephalomyelitis<br>
viruses, Western Equine Encephalomyelitis viruses, Japanese Encephalitis Viruses, and<br>
Kyasanur Forest Viruses.<br>
[0170]	It is understood that a sample comprising, e.g., bacteria and/or virus can be<br>
removed from its source (e.g., an individual, food, air, water, and other environmental<br>
samples); grown in culture, whereby the bacteria and/or virus is multiplied, enriched<br>
and/or purified (in some embodiments, substantially purified) prior to preparation of<br><br>
protein sample. Similarly, it is understood that a eukaryotic cell, e.g., cancer cell, may be<br>
removed from it natural setting, and cultured or propagated in vitro prior to analysis.<br>
[0171 ]	In some embodiments, the sample is comprised of (derived from)<br>
mammalian cells (in some embodiments, vertebrate cells), such as human, murine,<br>
primate, or rodent. In some embodiments, the cell is of a non-human mammal (in some<br>
embodiments, of a non-human vertebrate).<br>
[0172]	In some embodiments, proteins are prepared from a whole cell extract. In<br>
other embodiments, proteins can be pre-fractionated via subcellular location (e.g.,<br>
membrane and cytoplasmic) or different physical and functional properties. The protein<br>
can also be extracted from the supernatant of the culture. In some embodiments, a viral<br>
protein sample is prepared from serum and/or plasma and/or any other suitable body<br>
fluid. In some embodiments, serum suspected of comprising bacteria and/or virus is<br>
depleted of the major serum proteins prior to analysis using digital antibodies. Methods<br>
for depleting, reducing and/or removing the major serum proteins are well known in the<br>
art.<br>
[0173]	Prior to contacting the sample with a set of digital antibodies, the sample<br>
may also be treated with agents capable of denaturing and/or solubilizing proteins, such<br>
as detergents (ionic and non-ionic), chaotropes and/or reducing agent. Such agents are<br>
known in the art. The sample can also be heated to denature the proteins. Denaturation<br>
of the proteins allows the small epitopes on the proteins to be exposed, thus facilitating<br>
the binding of the proteins to the digital antibodies.<br>
[0174]	In other embodiments, the sample is directly applied to the digital<br>
antibodies without first being denatured. For example, analyzing the binding profile of a<br>
native protein will permit the study of small epitopes on the surface of the protein, thus<br>
providing information about the three dimensional structure of the protein.<br>
Contacting the sample with a digital antibody<br>
[0175]	Methods and conditions for contacting an antibody with a protein in a<br>
sample are well known in the art. Antibodies may be contacted with the sample one at a<br><br>
time or in groups of a set of digital antibodies). In some embodiments, contacting is<br>
serial (sequential, or iterative), e.g., a single antibody or group of antibodies is contacted<br>
with the sample; separated; and a second antibody or group of antibodies is contacted<br>
with the sample, and separated, and so on). In other embodiments, contacting is in<br>
parallel, e.g., a group of antibodies is contacted with the sample, and separated. It is<br>
appreciated that contacting may be both in parallel and serial, as when different groups of<br>
antibodies are serially contacted with a sample. Groups of antibodies may be overlapping<br>
in composition (e.g., group 1= antibody A, B, C, D; group 2 = antibody B, C, D, E, etc.)<br>
or different in composition.<br>
[0176]	In one embodiment, the set of digital antibodies set is contacted with a<br>
blocking agent before they are brought to contact with the sample. Blocking agents serve<br>
to block non-specific-binding sites, thus increase the detection sensitivity by reducing the<br>
background signal.<br>
[01771	In some embodiments involving parallel contacting, it is desirable for<br>
digital antibodies to be individually separable, for example, by linking the antibody to<br>
detectable distinct beads, use of individually separable binding partners, immobilization<br>
of antibody in, e.g., different wells of amultiwell plate, use of antibody arrays, and the<br>
like. Insofar as the small epitope bound by the antibody is known, binding by a digital<br>
antibody provides information relating to amino acid content of protein(s) bound by the<br>
digital antibody. In embodiments wherein knowledge of the cognate small epitope is<br>
desired, it may be convenient to individually separate the small antibodies (such that the<br>
protein bound by each digital antibody is kept separate). However, individual separation<br>
or separability is not required in every embodiment. For example, digital antibodies may<br>
be combined in small pools of two or more antibodies that possess overlapping antibody<br>
composition, such as (1) antibodies ABC; (2) antibodies CDE; (3) antibodies FGH, and<br>
(4) antibodies HIJ. Information regarding presence or absence of a particular small<br>
epitope may be inferred based on membership in a particular group.<br>
[0178]	In some embodiments, suitable controls can be included to, e.g., increase<br>
the precision of the detection of binding and/or intensity of binding. For example, such a<br><br>
control can include adding a specific antibody to a protein whose level is known to<br>
remain constant among the samples. Since the binding intensity of the control protein<br>
will be the same, the amount of proteins present can then be normalized.<br>
[0179]	Solid supports suitable for immobilizing (linking) antibodies (and<br>
modifications to render solid supports suitable for immobilizing antibodies) are well<br>
known in the art. Examples of a solid support include: a bead (including magnetized<br>
beads), microwell plate, and a protein microarray (e.g., technology owned by Zyomyx,<br>
Inc. See, e.g. US Patent No. 6,365,418). Thus, for example, CdSe-CdS core-shell<br>
nanocrystals enclosed in a silica shell can be easily derivatized for coupling to a<br>
biological molecule. Bruchez et al. (1998) Science 281: 2013-2016. Similarly, highly<br>
fluorescent quantum dots (zinc sulfide-capped cadmium selenide) have been covalently<br>
coupled to biomolecules for use in ultrasensitive biological detection. Warren and Nie<br>
(1998) Science 281:2016-2018. Fluorescently labeled beads are commercially available<br>
from Luminex and Quantum Dot. Methods for linking antibodies to such agents are well<br>
known in the art. See, e.g. Kennedy et al. (Clin. Chim. Acta 70:1-31 (1976)), and Schurs<br>
et al. (Clin. Chim. Acta 81 :l-40 (1977)) (describing coupling techniques, including the<br>
glutaraldehyde method, the periodate method, the dimaleimide method, the m-<br>
maleimidobenzyl-N-hydroxy-succinimide ester method, all of which methods are<br>
incorporated by reference herein).<br>
Contact with protease cleaving agent<br>
[0180]	In some embodiments, the methods of the invention further comprise a<br>
step of treating the sample with a protein cleaving agent, whereby polypeptide fragments<br>
are generated. In embodiments involving a step of separating protein from the antibody-<br>
protein complex, the sample can treated with a protein cleaving agent prior to a step of<br>
contacting a sample with the digital antibody set.<br>
[0181]	Protein cleaving reagent treatment generates protein cleavage fragments<br>
(such as polypeptides), which can facilitate subsequent mass spectral analysis of the<br><br>
amount of protein and the identity of proteins in a sample(s). In particular, treatment<br>
with a protein cleaving reagent treatment can facilitate the analysis of proteins whose<br>
molecular masses exceed 25 kDa. Protein cleaving reagent treatment also may facilitate<br>
accessibility and/or access of digital antibodies to a cognate epitope. Protein cleaving<br>
agents are well known in the art, and are further discussed herein. In some embodiments,<br>
one protein cleaving agent is used. In other embodiments, more than one protein<br>
cleaving reagent is used. Conditions for treatment with a protein cleaving agent are well<br>
known in the art.<br>
[0182]	The polypeptide cleaving reagent can be a protease. Example of proteases<br>
that can be used as polypeptide cleaving reagents, include, but are not limited to:<br>
chymotrypsin, trypsin (arg, lys cleavage sequence), thermolysin (phe, leu, iso, val<br>
cleavage sequence), V8 protease, Endoproteinase Glu-C, Endoproteinase Asp-N,<br>
Endoproteinase Lys-C, Endoproteinase Arg-C, Endoproteinase Arg-N, Factor Xa<br>
protease, thrombin, enterokinase, V5 protease, and the tobacco etch virus protease.<br>
Proteases useful in the methods of the invention can be genetically engineered and/or<br>
chemically modified to prevent autolysis. It is appreciated that an enzymatic protein<br>
cleaving regent (such as a protease) can be modified to facilitate removal of the protease<br>
from the polypeptide cleavage products following polypeptide cleavage. Such<br>
modifications are known in the art and include: (1) bead-bound (e.g., latex, silica or<br>
magnetic bead) protease, (2) haptenated protease, (3) affinity depletion of the protease<br>
(with, for example, a bead-bound anti-protease, or bead-bound non-cleavable substrate)<br>
and/or (4) size exclusion chromatography.<br>
[0183]	Polypeptide cleaving reagents can also include chemical substances and<br>
compounds that cleave polypeptides and peptide bonds such as cyanogen bromide (which<br>
cleaves at methionine residues), hydroxylamine (which cleaves between an Asn and a<br>
Gly residue), and acid pH (which can cleave an Asp-Pro bond) (see e.g., Ausubel et al.,<br>
supra). The activity of the polypeptide cleaving reagent can be inhibited by treating with<br>
heat, protease inhibitor, metal chelator (e.g., EGTA, EDTA), etc.<br><br>
[0184]	In still further embodiments, phosphatases (e.g., an alkaline phosphatase,<br>
an acid phosphatase, a protein serine phosphatase, a protein tyrosine phosphatase, and a<br>
protein threonine phosphatase, etc.), lipases, and other enzymes can be employed as<br>
protein cleaving reagents.<br>
Detection of protein binding pattern and comparison of protein binding patterns<br>
[0185J	The proteins in the sample that bind to the set of digital antibodies can be<br>
detected using any means known in the art. In some embodiments, the protein is labeled,<br>
using any methods known in the art. The term "label" refers to a composition detectable<br>
by spectroscopic, photochemical, biochemical, immunochemical, or chemical means.<br>
For example, the target protein may be labeled with one or more labeling moieties to<br>
allow detection of both protein-antibody complexes and by comparison the lack of such a<br>
complex in the comparison sample. The labeling moieties can include compositions that<br>
can be detected by photochemical, spectroscopic, biochemical, immunochemical,<br>
chemical, optical, electrical, bioelectronic, etc. means. For example, useful protein labels<br>
include P,3 S, fluorescent dyes, electron-dense reagents, enzymes (e.g., as commonly<br>
used in an ELISA), biotin, dioxigenin, or haptens and proteins for which antisera or<br>
monoclonal antibodies are available. A wide variety of labels and conjugation techniques<br>
are known and are reported extensively in both the scientific and patent literature, and are<br>
generally applicable to the present invention for the labeling of proteins. Suitable labels<br>
include radionucleotides, enzymes, substrates, cofactors, inhibitors, fluorescent moieties,<br>
chemiluminescent moieties, magnetic particles, and the like. Labeling agents optionally<br>
include e.g., monoclonal antibodies, polyclonal antibodies, proteins, or other polymers<br>
such as affinity matrices, carbohydrates or lipids. Detection of labeled proteins may<br>
proceed by any of a number of methods, including immunoblotting, tracking of<br>
radioactive or bioluminescent markers, or other methods which track a molecule based<br>
upon size, charge or affinity. The particular label or detectable moiety used and the<br>
particular assay are not critical aspects of the invention. The detectable moiety can be<br>
any material having a detectable physical or chemical property. Such detectable labels<br><br>
have been well developed in the field of gels, columns, and solid substrates, and in<br>
general, labels useful in such methods can be applied to the present invention. Thus, a<br>
label is any composition detectable by spectroscopic, photochemical, biochemical,<br>
immunochemical, electrical, optical or chemical means. Useful labels in the present<br>
invention include fluorescent dyes (e.g., fluorescein isothiocyanate, Texas red,<br>
rhodamine, and the like), radiolabels (e.g., 3H, 1251,35S, 14C, or 32P), enzymes (e.g., LacZ,<br>
CAT, horse radish peroxidase, alkaline phosphatase and others, commonly used as<br>
detectable enzymes, either as marker gene products or in an ELISA).<br>
[0186]	It will be recognized that fluorescent labels are not to be limited to single<br>
species organic molecules, but include inorganic molecules, multi-molecular mixtures of<br>
organic and/or inorganic molecules, crystals, heteropolymers, and the like. Thus, for<br>
example, CdSe-CdS core-shell nanocrystals enclosed in a silica shell can be easily<br>
derivatized for coupling to a biological molecule. Bruchez et al. (1998) Science 281:<br>
2013-2016. Similarly, highly fluorescent quantum dots (zinc sulfide-capped cadmium<br>
selenide) have been covalently coupled to biomolecules for use in ultrasensitive<br>
biological detection. Warren and Nie (1998) Science 281: 2016-2018.<br>
[0187]	The label is coupled directly or indirectly to the protein according to<br>
methods well known in the art. Methods for attaching and/or linking (either covalently or<br>
noncovalently, directly or indirectly, e.g., via a linker) label to protein are well known in<br>
the art. As indicated above, a wide variety of labels may be used, with the choice of label<br>
depending on the sensitivity required, ease of conjugation of the compound, stability<br>
requirements, available instrumentation, and disposal provisions. Non-radioactive labels<br>
are often attached by indirect means. In some embodiments, a ligand molecule (e.g.,<br>
biotin) is covalently bound to a polymer. The ligand then binds to an anti-ligand (e.g.,<br>
streptavidin) molecule which is either inherently detectable or covalently bound to a<br>
signal system, such as a detectable enzyme, a fluorescent compound, or a<br>
chemiluminescent compound. A number of ligands and anti-ligands can be used. Where<br>
a ligand has a natural anti-ligand, for example, biotin, thyroxine, and Cortisol, it can be<br>
used in conjunction with labeled anti-ligands. Alternatively, any haptenic or antigenic<br>
compound can be used in combination with an antibody.<br><br>
[0188]	Labels can also be conjugated directly to signal generating compounds,<br>
e.g., by conjugation with an enzyme or fluorophore. Enzymes of interest as labels will<br>
primarily be hydrolases, particularly phosphatases, esterases and glycosidases, or<br>
oxidoreductases, particularly peroxidases. Fluorescent compounds include fluorescein<br>
and its derivatives, rhodamine and its derivatives, dansyl, umbelliferone, fluorescent<br>
green protein, and the like. Chemiluminescent compounds include luciferin, and 2,3-<br>
dihydrophthalazinediones, e.g., luminol.<br>
[0189]	In some embodiments, the sample is labeled. In other embodiments, the<br>
antibodies are labeled. In still other embodiments, protein and antibodies are labeled.<br>
Means of detecting labels are well known to those of skill in the art. Thus, for example,<br>
where the label is a radioactive label, means for detection include a scintillation counter,<br>
proximity counter (microtiter plates with scintillation fluid built in), or photographic film<br>
as in autoradiography. Where the label is a fluorescent label, it may be detected by<br>
exciting the fluorochrome with the appropriate wavelength of light and detecting the<br>
resulting fluorescence, e.g., by microscopy, visual inspection, via photographic film, by<br>
the use of electronic detectors such as charge coupled devices (CCDS) or<br>
photomultipliers and the like. Similarly, enzymatic labels may be detected by providing<br>
appropriate substrates for the enzyme and detecting the resulting reaction product.<br>
Finally simple colorimetric labels are often detected simply by observing the color<br>
associated with the label. Thus, in various dipstick assays, conjugated gold often appears<br>
pink, while various conjugated beads appear the color of the bead.<br>
[0190]	In some embodiments, competition immunoassay is used for detection, as<br>
is well known in the art. Specifically, the test sample can be mixed with labeled small-<br>
epitope peptides (i.e., polypeptides comprising one or more cognate small epitopes<br>
recognized by one or more members of the set of digital antibodies being used in the<br>
assay) and brought into contact with the digital antibodies. The binding signal can then<br>
be detected. The signal can further be amplified to increase the sensitivity prior to the<br>
detection. Labels that are useful for the present method are known in the art, and include<br>
chemiluminescent molecules, magnetic labels, biotin, etc. In embodiments involving use<br>
of competitor polypeptides, the methods for generating protein binding profiles (and<br><br>
methods using protein binding profiles) generally further comprise contacting the set of<br>
digital antibodies with competitor polypeptides (in combination with the sample and/or<br>
sequentially with the sample).<br>
[0191]	In one embodiment, multiple-epitope inhibitors (such as MAPs) are used<br>
in a competition immunoassay. "Multi-epitope inhibitors" used herein refer to synthetic<br>
peptides or peptides generated by protein digestion, which contain multiple digital<br>
antibody epitopes. These multiple-epitope inhibitors therefore will compete with the<br>
protein sample for binding to the digital antibodies. The use of multiple-epitope inhibitor<br>
reduces the number of peptides needed for the competition immunoassay.<br>
[0192]	In another embodiments, digital antibodies are coated or linked to a solid<br>
surface, protein (including protein fragments from sample) are labeled, e.g. with biotin<br>
before contacting with antibodies, protein is contacted with antibodies, unbound protein<br>
is removed, and then streptavidin conjugated enzymes, fluorescent dyes and/or<br>
nanocrystals are used to detect protein bound to digital antibodies.<br>
[0193]	In another embodiment, a competition assay is conducted, e.g., .by coating<br>
or linking digital antibodies to a solid surface, and protein (such as protein fragments)<br>
from sample and one or more labeled synthetic peptides are added together or<br>
sequentially to the digital antibodies. Each synthetic peptide comprises binding epitopes<br>
of two or more digital antibodies used, and the mixture of the synthetic peptides<br>
comprises the epitopes of all of the digital antibodies used (in some embodiments,<br>
essentially all of the digital antibodies used). The synthetic peptide mixture thus<br>
competes for binding with protein (such as protein fragments) in a sample that contains<br>
binding epitopes of the digital antibodies in the array. Synthetic peptides (also termed<br>
competition peptides) can be labeled using nanocrystals, enzymes and/or fluorescent dye.<br>
In some embodiments, the label can be biotin or avidin or streptavidin, and then signal is<br>
amplified using avidin, streptavidin or biotin conjugated enzyme, fluorescent dye or<br>
nanocrystal will be used to demonstrate the result of the competitive binding. Figure 4<br>
depicts an illustrative embodiments of a digital antibody array and described a detection<br>
embodiments using competition polypeptides.<br><br>
[0194]	Data generated by detection of protein binding can be analyzed using any<br>
suitable means (e.g., visually, by computer, etc.). In one embodiment, data is analyzed<br>
with the use of a programmable digital computer. The data analysis can include the steps<br>
of determining the intensity of the signal. The intensity can be normalized, whereby the<br>
intensity is calibrated relative to some reference value. For example, a reference can be<br>
background noise of the binding. Alternatively, a reference can be the protein binding<br>
intensity of a control antibody.<br>
[0195]	In some embodiments, the intensity of binding of each digital antibody to<br>
protein in the sample can be digitized to generate a number. The term "digitized" as used<br>
herein, refers to the process of converting the binding signals into digital data. The<br>
number representing the intensity of binding of each digital antibody can then be stated in<br>
sequence to generate a multi-digit number which represents the binding of the digital<br>
antibodies to that sample. In another embodiment, the binding profile is represented by<br>
graphical representations similar to a bar graph. In this embodiment, each slot or position<br>
of the bar graph represents a specific digital antibody, and the height or width of the<br>
"bar" represents the presence and/or absence and/or degree, or intensity, of binding. The<br>
generation of these numbers or graphical representations permits the ready storage of<br>
information about large set of samples. Moreover, a comparison of two numbers or<br>
graphical representations provides information on the degree to which the two samples of<br>
interest differ.<br>
[0196]	The comparison of protein binding profiles can be performed by any<br>
convenient means. For example, visual comparisons of patterns can be performed to<br>
determine patterns associated with different types of toxicities. More conveniently, the<br>
correlation can be done by computer, using one of the database programs discussed in the<br>
previous section. Preferably, the correlation is performed by a computer using a neural<br>
network program, since neural network programs are specifically designed for pattern<br>
recognition. The comparison of protein binding profiles can be done by a number of<br>
means known in the art. Usually, the graphs resulting from the calculations can be<br>
stored, for example, in file folders or the like, and examined visually to discern common<br><br>
patterns of expression compared to the control, as well as differences. Conveniently,<br>
however, the data can be stored on and compared by a computer.<br>
[0197]	Standard database programs, such as Enterprise Data Management<br>
(Sybase, Inc., Emeryville, CA) or Oracle8 or 9 (Oracle Corp., Redwood Shores, CA) can<br>
be used to store and compare information. Alternatively, the data can be recorded, or<br>
analyzed, or both, in specifically designed programs available, for example, from Partek<br>
Inc. (St. Charles, MO).<br>
[0198]	Additionally, companies selling integrated analytical systems, such as<br>
mass spectrometers, provide with the machines integrated software for recording results.<br>
Such companies include Finnigan Corp. (San Jose, CA), Perkin-Elmer Corp. (Norwalk<br>
CT), Ciphergen Biosystems, Inc. (Palo Alto CA), and Hewlett Packard Corp. (Palo Alto,<br>
CA).<br>
[0199]	In a preferred embodiment, the data can be recorded and analyzed by<br>
neural network technology. Neural networks are complex non-linear modeling equations<br>
which are specifically designed for pattern recognition in data sets. One such program is<br>
the NeuroShell Classifier™ classification algorithm from Ward Systems Group, Inc.<br>
(Frederick, MD). Other neural network programs are available from, e.g., Partek, Inc.,<br>
BioComp Systems, Inc. (Redmond WA) and Z Solutions, LLC (Atlanta, GA).<br><br><br><br>
EXAMPLES<br>
Example 1: Preparation and characterization of digital antibodies<br>
[01102]	Five immunization peptides in the format of Multiple Antigenic Peptides<br>
(MAPs) were designed as shown in Table 4. These sequences in combination were also<br>
used to evaluate cross-reactivity of the induced antibodies, by virtue of the inclusion in<br>
different MAPs of the same sequence in differing locations. Each of the immunization<br>
peptides were used to immunize 4 Balb/C mice using standard methods.<br><br><br>
[01103]	Notes to Table 4:<br>
[0200]	Peptide MAPI: HSLFHPEDTGQV: From PSA, amino acids #79-89.<br>
KKTTNV: From Meningococcal Opa protein, containing KTT, a published 3mer<br>
antibody epitope (Malorny, Morelli et al. 1998).<br>
[0201]	Peptide MAP2: Alternate sequences of MAPI.<br>
[0202]	Peptide MAP3: LTPKK: Motif 1 of PS A (Nagasaki, Watanabe et al.<br>
1999). KKTTNVLTVPTNIPG: From Meningococcal Opa protein, containing two<br>
published 3mer antibody epitopes: KTT and NIP and one 4mer epitope: TNEP (Morelli et<br>
al. (1997) Mol Microbiol 25(6): 1047-64.<br>
[0203]	Peptide MAP4: LTPKK: From PSA, the same as in peptide MAP3.<br>
LTQENQNRGTH: An immunogenic sequence of alpha-1-ACT selected by DNAStar<br>
computer program. rYNQ: From Meningococcal Opa protein, containing a 2mer epitope<br>
IY and four amino acids of a 5mer epitope, TIYNQ and of a 7mer epitope TPTIYNQ<br>
(Marelli et al, id.).<br>
[0204]	Peptide MAP5 TIYNTNIPG: From Meningococcal Opa protein (Marelli<br>
et al, id.). LTQENQNRGTH: The same as in peptide MAP4.<br><br>
[0205]	Two sets of screening peptides were designed: (1)5 C-terminally<br>
biotinylated with the same sequences as the immunization peptides (shown in Table 5);<br>
and (2) 43 lOmer biotinylated peptides with sequences panning all five immunization<br>
peptides (shown in Table 6).<br><br><br><br><br><br><br>
results were consistent with mice #2-1 and #2-4 expressing antibodies that recognize<br>
distinct and concise epitopes present within more than one screening antigen used in the<br>
ELISA assays. A test of the #2-1 and #2-4 sera versus 23 lOmer biotinylated peptides<br>
that span sequences of all three immunization peptides for group 1,2 and 3 mice also<br>
demonstrated a broad cross-reactivity.<br>
[0208]	Eight test bleeds from groups 4-5 were tested by ELISA. Group 4 mice<br>
demonstrated a modest response to their relevant screening peptide, Pep4-0, while<br>
exhibiting strong cross-reactivity with Pep3-0, the screening peptide designed for group<br>
3. Group 4 mice did not show substantial cross-reactivity to Pep5-0 even though there is<br>
significant sequence identity between Pep4-0 and Pep5-0. In contrast, 3 of 4 mice in<br>
group 5 (mice 5-2, 5-3, 5-4) exhibited robust immunoreactivity to both their screening<br>
peptide, Pep5-0, and to the related screening peptide, Pep4-0. The sera from the<br>
responsive mice in group 5 did not demonstrate substantial cross-reactivity to the Pep3-0,<br>
even though there is a 5 amino acid block of sequence identity. A test of the #5-2 and 5-3<br>
sera versus 23 lOmer biotinylated peptides that span sequences of all three immunization<br>
peptides for group 4 and 5 mice demonstrated two broad but distinctive reaction patterns<br>
with the mapping peptides spanning sequences of immunization peptides for groups 4<br>
and 5 mice.<br>
[0209]	Group 2 mice #1 and #4, and Group 5 mice #2 and #3 showed the best<br>
immune responses, as summarized in Table 7 and Figure 2. These mice were selected for<br>
hybridoma fusions.	<br><br><br>
[0210]	Group 2 mice #1 and #4, and Group 5 mice #2 and #3 were selected for<br>
hybridoma fusions. The animals were sacrificed, the lymph nods and spleens harvested,<br>
then B cell hybridoma fusions using P3 mouse myeloma cell line as a fusion partner were<br>
generated using standard methods. Fusions were plated and incubated for 11-14 days<br>
before screening.<br>
[0211]	In the first round of screening, hybridomas from group 2 and 5 mice were<br>
analyzed by ELISA in 96 well plates, essentially as described above, using the<br>
corresponding screening peptides, 2-0 and 5-0. Following several rounds of screening,<br>
48 positive hybridoma lines were identified and transferred to 24 well plates for<br>
expansion and additional characterization including epitope mapping. Of the 48 positive<br>
lines, 33 were derived from the Group 2 animals that received the MAP2 immunogen<br>
while the remaining 15 originated from the Group 5 animals. Most of the hybridoma<br>
lines (-94%) were the fusion products of B cells harvested from the spleen. Thirteen of<br>
the 48 hybridoma lines expressed IgG, 25 expressed IgM, and the remaining 10<br>
hybridoma lines were expressing both IgG and IgM or were not expressing either IgG or<br>
IgM and were therefore expressing either IgA or IgE.<br><br>
[0212]	In the second round of screening, hybridomas selected for expansion were<br>
re-tested against the relevant screening peptide (either peptide 2-0 or peptide 5-0). 13 of<br>
the 48 hybridomas characterized after the 24 well expansion phase exhibited sequence<br>
specific binding to the screening peptide 2-0. Other hybridomas bound non-specifically<br>
(i.e., bound a variety of oligopeptide sequences), failed to bind (reflecting either a false<br>
positive or clonal instability and loss during the transfer and subsequent propagation in<br>
24 well plates) or bound control wells containing BSA.<br>
[0213]	The 13 hybridomas that specifically bound to screening peptide 2-0 were<br>
epitope mapped using ELISA as described above, using 3 different sets of lOmer C-<br>
terminal biotinylated mapping peptides: peptides 1-1 to 1-5; 2-1 to 2-9; and 3-1 to 3-9<br>
(see Table 6). 10 of the 12 hybridoma lines exhibited maximum reactivity with a single<br>
mapping peptide, 2-1, and that hybridomas 2.03 and 2.11 showed strong binding to<br>
different overlapping sets of mapping peptides, peptides 2-1 through 2-3 and 2-7 through<br>
2-9. Because these data showed strong reactivity to a single mapping peptide for most<br>
hybridoma lines, we considered the possibility that steric hindrance associated with<br>
immobilization of the mapping peptides (specifically, biotin-avidin immobilization) was<br>
preventing antibody binding to the epitope present within a cognate series of lOmers, thus<br>
potentially biasing the ELISA epitope map results. Thus, we evaluated epitope<br>
specificity using a competitive binding assay.<br>
[0214]	Individual mapping peptides were evaluated for their ability to inhibit<br>
antibody binding to the 2-0 screening peptide affixed to streptavidin-coated 96 well<br>
plates. In this format, the lOmer mapping peptides were not tethered within the binding<br>
pocket of streptavidin and consequently should not be sterically hindered from interacting<br>
with a reactive antibody present within the set of 13 hybridomas. Inhibition experiments<br>
were performed using standard methods using the 2-0 screening peptide was affixed to<br>
streptavidin-coated 96 well plates and lOmer mapping peptide was added to each well.<br>
[0215]	Using the competitive binding assay, the epitopes recognized by 10 of the<br>
13 hybridomas were determined. Eight of the hybridomas were specific for the epitope<br>
PEDTG, hybridoma 2.03 was specific for epitope DTG and hybridoma 2.11 recognized<br>
the epitope KTTN. Hybridomas 2.31,1.02 and 2.12 showed poor discrimination in the<br>
competitive inhibition assay. Hybridomas 2.03 (also called DA001-2.03), 2.04 (DA001-<br><br>
2.04) and 2.11 (also called DAOOl-2.11) are being prepared for deposit at the ATCC.<br>
The results of this analysis are summarized in Table 8.<br><br><br><br>
[0216]	Although the foregoing invention has been described in some detail by<br>
way of illustration and example for purposes of clarity of understanding, the descriptions<br>
and examples should not be construed as limiting the scope of the invention.<br><br>
WE CLAIM:<br>
1.	A set of isolated digital antibodies, wherein the set comprises at least 15 digital<br>
antibodies, wherein each of the 15 digital antibodies has been characterized to bind specifically<br>
to a different epitope consisting of 3 or 4, or 5 consecutive amino acids and wherein each digital<br>
antibody recognizes a plurality of distinct and different proteins that comprise the same epitope<br>
to which the digital antibody binds.<br>
2.	The set of isolated digital antibodies as claimed in claim 1, wherein the set comprises 100<br>
digital antibodies mat bind epitopes consisting of 3 consecutive amino acids.<br>
3.	The set of isolated digital antibodies as claimed in claim 2, wherein the set further<br>
comprises 100 digital antibodies that bind epitopes consisting of 4 consecutive amino acids.<br>
4.	The set of isolated digital antibodies as claimed in claim 3, wherein the set further<br>
comprises 100 digital antibodies that bind epitopes consisting of 5 consecutive amino acids.<br>
5.	The set of isolated digital antibodies as claimed in claim 1, wherein the set comprises at<br>
least 20, 30, 40, 50, 60, 70, 80, 90, 100, 200, 300, 400, 500, 600, 700, 800, 900, or 1000 digital<br>
antibodies.<br>
6.	The set of isolated digital antibodies as claimed in claim 1, wherein the set comprises at<br>
least 1000 digital antibodies that bind epitopes consisting of 4 consecutive amino acids.<br>
7.	The set of isolated digital antibodies as claimed in claim 6, wherein the set further<br>
comprises at least 100 digital antibodies that bind epitopes consisting of 5 consecutive amino<br>
acids.<br>
8.	The set of isolated digital antibodies as claimed in claim 7, wherein the set further<br>
comprises at least 100 digital antibodies that bind epitopes consisting of 3 consecutive amino<br>
acids.<br><br>
9.	The set of isolated digital antibodies as claimed in claim 1, wherein the digital antibodies<br>
are immobilized on a surface.<br>
10.	The set of isolated digital antibodies as claimed in claim 4, wherein the digital antibodies<br>
are immobilized on a surface.<br>
11.	The set of isolated digital antibodies as claimed in claim 9 or 10, wherein the surface is<br>
an array.<br>
12.	A method for generating a protein binding profile of a sample comprising a plurality of<br>
different proteins, said method comprising:<br><br>
(a)	contacting said sample with a set of isolated digital antibodies as claimed in claim 1 under<br>
conditions that permit binding;<br>
(b)	optionally removing unbound protein; and<br>
(c)	detecting binding of protein to said antibodies, whereby a protein binding profile is<br>
generated.<br><br>
13.	The method as claimed in claim 12, wherein the method further comprises the step of<br>
treating the sample with a protein cleaving agent prior to step (a) of contacting the sample with<br>
the set of isolated digital antibodies under conditions that permit binding.<br>
14.	A method for generating a library of protein binding profiles for two or more different<br>
samples as each of which comprises a plurality of proteins, said method comprising:<br><br>
(a)	contacting a sample with a set of isolated digital antibodies as claimed in claim 1 under<br>
conditions that permit binding;<br>
(b)	optionally removing unbound protein;<br>
(c)	detecting binding of protein to said antibodies, whereby a protein binding profile is<br>
generated; and<br>
(d)	repeating steps (a) through (c) with at least two samples.<br><br>
15.	The method as claimed in claim 14, wherein the method further comprises the step of<br>
treating the sample with a protein cleaving agent prior to step (a) of contacting the sample with<br>
the set of isolated digital antibodies under conditions that permit binding.<br>
16.	A library of protein binding profiles, wherein the library is prepared using the method as<br>
claimed in claim 14.<br>
17.	A method for characterizing a test sample, said method comprising<br><br>
(a)	contacting the test sample with the a set of isolated digital antibodies as claimed in claim 1<br>
under conditions that permit binding;<br>
(b)	optionally removing unbound protein;<br>
(c)	generate a protein binding profile of said test sample by detecting binding of protein to said<br>
antibodies; and<br>
(d)	comparing the generated protein binding profile of the test sample with a protein<br>
binding profile of a reference sample, whereby the test sample is characterized by the<br>
comparison.<br>
18.	The method as claimed in claim 17, wherein step (d) of comparing is with a library of<br>
protein binding profiles, wherein the library of protein binding profiles is generated using a<br>
method comprising:<br>
(i) contacting a sample with a set of isolated digital antibodies as claimed in claim 1 under<br>
conditions that permit binding;<br>
(ii) optionally removing unbound protein;<br>
(iii) detecting binding of protein to said antibodies, whereby a protein binding profile is<br>
generated; and<br>
(iv) repeating steps (i) through (iii) with at least two samples.<br>
19.	A method for determining presence or absence of a bacteria, virus, or cell in a sample,<br>
said method comprising<br>
(a)	contacting the test sample with a set of isolated digital antibodies as claimed in claim 1 under<br>
conditions that permit binding;<br>
(b)	optionally removing unbound protein;<br><br>
(c)	detecting binding of protein to said antibodies, whereby a protein binding profile is<br>
generated; and<br>
(d)	comparing the protein binding profile of the test sample with a protein binding profile of a<br>
reference sample, whereby presence or absence of the bacteria, virus or cell in the test sample is<br>
determined by the comparison.<br>
20.	The method as claimed in claim 19, wherein step (d) of comparing is with a library of<br>
protein binding profiles, wherein the library of protein binding profiles is generated using a<br>
method comprising :<br>
(i) contacting a sample with a set of isolated digital antibodies as claimed in claim 1 under<br>
conditions that permit binding;<br>
(ii) optionally removing unbound protein;<br>
(iii) detecting binding of protein to said antibodies, whereby a protein binding profile is<br>
generated; and<br>
(iv) repeating steps (i) through (iii) with at least two samples.<br>
21.	A method for identifying a bacteria, virus, or cell, said method comprising<br>
(a)	contacting the test sample with a set of isolated digital antibodies as claimed in claim 1 under<br>
conditions that permit binding;<br>
(b)	optionally removing unbound protein;<br>
(c)	detecting binding of protein to said antibodies, whereby a protein binding profile is<br>
generated; and<br>
(d)	comparing the protein binding profile of the test sample with a protein binding profile of a<br>
reference sample, whereby presence or absence of the bacteria, virus or cell in the test sample is<br>
determined by the comparison.<br>
22.	The method as claimed in claim 21, wherein step (d) of comparing is with a library of<br>
protein binding profiles, wherein the library of protein binding profiles is generated using a<br>
method comprising :<br>
(i) contacting a sample with a set of isolated digital antibodies as claimed in claim 1 under<br>
conditions that permit binding;<br>
(ii) optionally removing unbound protein;<br><br>
(iii) detecting binding of protein to said antibodies, whereby a protein binding profile is<br>
generated; and<br>
(iv) repeating steps (i) through (iii) with at least two samples.<br>
23.	A method for identifying a test protein, said methods comprising<br>
(a)	contacting a sample comprising or suspected of comprising the test protein with a set of<br>
isolated digital antibodies as claimed in claim 1 under conditions that permit binding;<br>
(b)	optionally removing unbound protein;<br>
(c)	determining the identity of the test protein by detecting presence or absence of binding of<br>
protein to said antibodies in the set, wherein at least six digital antibodies bind the test protein;<br>
and wherein presence of binding indicates presence of at least six epitopes in the protein,<br>
wherein the identity of the at least six epitopes is used to identify the test protein.<br><br>
24.	The method as claimed in claim 23, wherein at least 7, 8, 9, 10, 11, 12, 13, 14, 15, 20 or<br>
25 digital antibodies bind protein.<br>
25.	The method of any as claimed in claims 12, 14, 17, 19, 21, or 23, wherein the sample<br>
comprises cellular protein or a subfraction of cellular protein.<br>
26.	The method as claimed in any of claims 12, 14, 17, 19, 21, or 23, wherein the sample is<br>
of a cell or virus.<br>
27.	The method as claimed in any of claims 17, 19, 21, or 23 wherein the method further<br>
comprises the step of treating the sample with a protein cleaving agent prior to step (a) of<br>
contacting the sample with the set of isolated digital antibodies under conditions that permit<br>
binding.<br>
28.	A kit comprising the set of isolated digital antibodies as claimed in claim 1.<br><br>
The present invention relates generally to sets of at least about 15 digital antibodies<br>
directed against short epitopes consisting of 3, or 4, or 5 consecutive amino acids, wherein each<br>
digital antibody recognizes a plurality of distinct and different proteins that comprise the same<br>
epitope to which the digital antibody binds. The present invention also relates to methods for<br>
generating a protein binding profile of a sample, characterizing a test sample, determining<br>
presence or absence of a bacteria, virus, or cell in a sample, and identifying a test protein using a<br>
set of digital antibodies. The present invention also relates to a kit comprising a set of digital<br>
antibodies<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLUtPTE5QLTIwMDUtQ09SUkVTUE9OREVOQ0UucGRm" target="_blank" style="word-wrap:break-word;">612-KOLNP-2005-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLUtPTE5QLTIwMDUtRk9STSAyNy5wZGY=" target="_blank" style="word-wrap:break-word;">612-KOLNP-2005-FORM 27.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDUtZ3JhbnRlZC1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">612-kolnp-2005-granted-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDUtZ3JhbnRlZC1hc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">612-kolnp-2005-granted-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDUtZ3JhbnRlZC1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">612-kolnp-2005-granted-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDUtZ3JhbnRlZC1jb3JyZXNwb25kZW5jZS5wZGY=" target="_blank" style="word-wrap:break-word;">612-kolnp-2005-granted-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDUtZ3JhbnRlZC1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">612-kolnp-2005-granted-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDUtZ3JhbnRlZC1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">612-kolnp-2005-granted-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDUtZ3JhbnRlZC1leGFtaW5hdGlvbiByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">612-kolnp-2005-granted-examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDUtZ3JhbnRlZC1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">612-kolnp-2005-granted-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDUtZ3JhbnRlZC1mb3JtIDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">612-kolnp-2005-granted-form 13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDUtZ3JhbnRlZC1mb3JtIDE4LnBkZg==" target="_blank" style="word-wrap:break-word;">612-kolnp-2005-granted-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDUtZ3JhbnRlZC1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">612-kolnp-2005-granted-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDUtZ3JhbnRlZC1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">612-kolnp-2005-granted-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDUtZ3JhbnRlZC1ncGEucGRm" target="_blank" style="word-wrap:break-word;">612-kolnp-2005-granted-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDUtZ3JhbnRlZC1yZXBseSB0byBleGFtaW5hdGlvbiByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">612-kolnp-2005-granted-reply to examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDUtZ3JhbnRlZC1zcGVjaWZpY2F0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">612-kolnp-2005-granted-specification.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="228421-a-power-control-system-for-an-electric-motor-and-a-method-of-running-down-a-high-speed-dc-electric-motor.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="228423-hydraulic-tensioner-of-the-hollow-piston-type-with-a-screw-type-retaining-device.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>228422</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>612/KOLNP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>06/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>06-Feb-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>04-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>08-Apr-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>ABMETRIX, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>3509 MILLER COURT, UNION CITY, CA 94587</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>WANG, JIANFU, JEFFREY</td>
											<td>3509 MILLER COURT, UNION CITY, CA 94587</td>
										</tr>
										<tr>
											<td>2</td>
											<td>HU, WEIXING, HELEN</td>
											<td>3509 MILLER COURT, UNION CITY, CA 94587</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>G01N 33/53</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2003/032574</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-10-15</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/496,124</td>
									<td>2003-08-18</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/418,277</td>
									<td>2002-10-15</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/228422-sets-of-digital-antibodies-directed-against-short-epitopes-and-methods-using-same by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:00:29 GMT -->
</html>
