<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/228492-polynucleotide-molecule-comprising-a-nucleotide-sequence-that-is-the-streptomyces-avermetilis-avec-allele by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:05:08 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 228492:&quot;POLYNUCLEOTIDE MOLECULE COMPRISING A NUCLEOTIDE SEQUENCE THAT IS THE STREPTOMYCES AVERMETILIS AVEC ALLELE&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;POLYNUCLEOTIDE MOLECULE COMPRISING A NUCLEOTIDE SEQUENCE THAT IS THE STREPTOMYCES AVERMETILIS AVEC ALLELE&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A polynucleotide molecule comprising a nucleotide sequence that is the Streptomyces avermetilis aveC allele, the S. avermetilis AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604) or the nucleotide sequence of the aveC ORF of S. avermetilis as presented in SEQ ID NO:l, or a degenerate variant thereof and said sequence comprises mutations encoding a combination of amino acid substitutions at amino acid residues corresponding to the amino acid positions of SEQ ID NO:2, such that cells of S. avermetilis strain ATCC 53692 in which the wild-type aveC allele has been inactivated and that express the polynucleotide molecule comprising the mutated nucleotide sequence capable of producing a class 2:1 ratio of avermectins wherein when the class 2:1 avermectins are cyclohexyl B2:cyclohexyl Bl avermectins the ratio of class 2:1 avermectins is 0.2:1 or less, and wherein the combinations of amino acid substitutions comprise substitutions at both D48 and G179.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>This invention relates to "polynucleotide  molecule  comprising a nucleotide sequence that is the streptomyces avermetilis aveC allele"<br>
1.  FIELD OF THE INVENTION<br>
The present invention is directed to compositions and methods for the efficient production of avermectins such as "doramectin", which are primarily useful in the field of animal health. More particularly, the present invention relates to polynucleotide molecules comprising nucleotide sequences encoding an AveC gene product, which, can be used to modulate the ratio of class 2:1 avermectins produced by fermentation cultures of Streptomyces avermitilis. The present invention further relates to vectors, transformed host cells, and novel mutant strains of S. avermitilis in which the aveC gene has been mutated so as to modulate the ratio of ciass 2:1 avermectins produced.<br>
2. BACKGROUND OF THE INVENTION 2.1. Avermectins Streptomyces species produce a wide variety of secondary metabolites, including the avermectins, which comprise a series of eight related sixteen-membered macrocyclic lactones having potent anthelmintic and insecticidal activity. ' The eight distinct but closely related compounds are referred to as A1a, A1b, A2a, A2b, B1a, B1b, B2a and B2b. The "a" series of compounds refers to the natural avermectin where the substituent at the C25 position is (S)-sec-butyl, and the "b" series refers to those compounds where the substituent at the C25 position is isopropyl. The designations "A" and "B" refer to avermectins where the substituent at the C5 position is methoxy and hydroxy, respectively. The numeral "1" refers to avermectins where a double bond is present at the C22,23 position, and the numeral "2" refers to avermectins having a hydrogen at the C22 position and a hydroxy at the C23 position. Among the related avermectins, the B1 type of avermectin, such as doramectin, is recognized as having the most effective antiparasitic and pesticidal activity, and is therefore the most commercially desirable avermectin.<br>
The avermectins and their production by aerobic fermentation of strains of S. avermitilis are described in United States Patent Nos. 4,310,519 and 4,429,042. The biosynthesis of natural' avermectins is believed to be initiated endogenously from the CoA thioester analogs of isobutyric acid and S-{+)-2-methyl butyric acid.<br>
A combination of both strain improvement through random mutagenesis and the use of exogenously supplied fatty acids has led to the efficient production of avermectin analogs. Mutants of S. avermitilis that are deficient in branched-chain 2-oxo acid dehydrogenase (bkd deficient mutants) can only produce avermectins when fermentations are supplemented with fatty acids. Screening and isolation of mutants deficient in branched-chafn dehydrogenase activity (e.g., S. avermitilis, ATCC 53567} are described in European Patent (EP) 276103, Fermentation of such mutants in the presence of exogenously supplied fatty acids results in <br>
production of only the four avermectins corresponding to the fatty acid employed. Thus, supplementing fermentations of S. avermitilis (ATCC 53567) with S-(+)-2-methylbutyric acid results in production of the natural avermectins A1a, A2a, B1a and B2a; supplementing fermentations with isobutyric acid results in production of the natural avermectins A1b, A2b, B1b, and B2b; and supplementing fermentations with cyclopentanecarboxylic acid results in the production of the four novel cyclopentylavermectins A1, A2, B1, and B2.<br>
If supplemented with other fatty acids, novel avermectins are produced. By screening<br>
over 800 potential precursors, mdre than 60 other novel avermectins have been identified.<br>
(See, e.g., Dutton et a/., 1991, J. Antibiot. 44:357-365; and Banks et a/., 1994, Roy. Soc.<br>
Chem. 147:16-26).   In addition, mutants of S. avermitilis deficient in 5-O-methyltransferase<br>
activity produce essentially only the B analog avermectins.    Consequently, S. avermitilis<br>
mutants lacking both branched-chain 2-oxo acid dehydrogenase and 5-O-methyltransferase<br>
activity produce only the B  avermectins corresponding to the fatty acid  employed to<br>
supplement the fermentation.    Thus, supplementing such double mutants with S-{+)-2-<br>
methylbutyric acid results in production of only the natural avermectins B1a and B2a, while<br>
supplementing with isobutyric acid or cyclopentanecarboxylic acid results in production of the<br>
natural avermectins  B1b  and  B2b  or the  novel  cyclopentyl  B1   and  B2  avermectins,<br>
respectively. Supplementation of the double mutant strain with cyclohexane carboxylic acid is<br>
a   preferred   method   for   producing   the   commercially   important   novel    avermectin,<br>
cyclohexylavermectin B1  (doramectin).    The isolation and characteristics of such double<br>
mutants, e.g., S. avermitilis (ATCC 53692), is described in EP 276103.<br>
2.2. Genes Involved In Avermectin Biosynthesis<br>
In many cases, genes involved in production of secondary metabolites and genes encoding a particular antibiotic are found clustered together on the chromosome. Such is the case with the Streptomyces polyketide synthase gene cluster (PKS) (see Hopwood and Sherman, 1990, Ann. Rev. Genet. 24:37-66). Thus, one strategy for cloning genes in a biosynthetic pathway has been to isolate a drug resistance gene and then test adjacent regions of the chromosome for other genes related to the biosynthesis of that particular antibiotic. Another strategy for cloning genes involved in the biosynthesis of important metabolites has been complementation of mutants. For example, portions of a DNA library from an organism capable of producing a particular metabolite are introduced into a non-producing mutant and transformants screened for production of the metabolite. Additionally, hybridization of a library using probes derived from other Streptomyces species has been used to identify and clone genes in biosynthetic pathways.<br>
Genes involved in avermectin biosynthesis (ave genes), like the genes required for biosynthesis of other Streptomyces secondary metabolites (e.g., PKS), are found clustered on<br><br>
the chromosome. A number of ave genes have been successfully cloned using vectors to complement S. avermitilis mutants blocked in avermectin biosynthesis. The cloning of such genes is described in U.S. Patent No. 5,252,474. In addition, Ikeda et a/., 1995, J. Antlbiot. 48:532-534, describes the localization of a chromosomal region involving the C22,23 dehydration step (aveC) to a 4.82 Kb SamHI fragment of S. avermitilis, as well as mutations in the aveC gene that result in the production of a single component B2a producer. Since ivermectin, a potent anthelmintic compound, can be produced chemically from avermectin B2a, such a single component producer of avermectin B2a is considered particularly useful for commercial production of ivermectin.<br>
U.S. Patent No. 6,248,579 to Stutzman-Engwall et al., issued June 19, 2001, describes certain mutations to the aveC gene of Streptomyces avermitilis leading to a reduction in the ratio of cyclohexyl B2:cyclohexyl B1 ratio to about 0.75:1.<br>
PCT Publication WO 01/12821 by Pfizer Products Inc., published February 22, 2001, describes certain additional mutations to the aveC gene of Streptomyces avermitilis leading to further reductions in the ratio of cyclohexy! B2:cyclohexyl B1 ratio down to 0.40:1.<br>
Identification of additional mutations or combinations of mutations in the aveC gene that further minimize the complexity of avermectin production, such as, e.g., mutations that further decrease the B2:B1 ratio of avermectins, would simplify production and purification of commercially important avermectins.<br>
3. SUMMARY OF THE INVENTION<br>
The present invention provides a polynucleotide molecule comprising a nucleotide sequence that is otherwise the same as the Streptomyces avermitilis aveC allele, the S. avermitilis AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604) or the nucleotide sequence of the aveC ORF of S. avermitilis as presented in FIGURE 1 (SEQ ID NO:1), or a degenerate variant thereof, but which nucleotide sequence further comprises mutations encoding a combination of amino acid substitutions at amino acid residues corresponding to the amino acid positions of SEQ ID NO:2, such that cells of S. avermitilis strain ATCC 53692 in which the wild-type aveC allele has been inactivated and that express the polynucleotide molecule comprising the mutated nucieotide sequence are capable of producing a class 2:1 ratio of avermectins that is reduced compared to the ratio produced by cells of S. avermitilis strain ATCC 53692 that instead express only the wild-type aveC allele, wherein when the class 2:1 avermectins are cyclohexyl B2:cyclohexyl Bl avermectins, the ratio of class 2:1 avermectins is 0.35:1 or less. In a more preferred embodiment, the ratio of cyclohexyl B2:cyclohexyl B1 avermectins is about 0.30:1 or less. In a more preferred embodiment, the ratio of cyclohexyl B2:cyclohexyl B1 avermectins is about 0.25:1 or less. In a<br><br>
more preferred embodiment, the ratio of cyclohexyl B2:cyclohexyl B1 avermectins is about 0.20:1 or less.<br>
In a	particular embodiment thereof, the combination of amino acid substitutions<br>
comprises a combination selected from the group consisting of:<br>
(a)	D48E, A61T, A89T, S138T, A139T, G179S, A198G, P289L;<br>
(b)	G40S, D48E, L136P, G179S, E238D;<br>
(c)	D48E, L136P, R163Q, G179S;<br>
(d)	D48E, L136P, R163Q, G179S, E238D;<br>
(e)	D48E, L136P, R163Q, G179S, A200G, E238D;<br>
(f)	D48E, L136P, G179S, E238D;<br>
(g)	D48E, A61T, L136P, G179S, E238D;<br>
(h)         D48E, A61T.L136P, G179S;<br>
(i)	D48E, A89T, S138T.A139T, G179S;<br>
0)	D48E, A61T, L136P, G179S, A198G, P202S, E238D, P289L;<br>
(k)         D48E, A61T, L136P, S138T, A139F, G179S, E238D, P289L;<br>
(I)	D48E, L136P, G179S, A198G, E238D, P289L;<br>
(m)        D48E, A61T, S138T, A139F, G179S, A198G, P289L;<br>
(n)	D48E, L84P, G111V, S138T, A139T, G179S, A198G, P289L;<br>
(o)	Y28C, D48E,A61T,A89T, S138T.A139T, G179S, E238D;<br>
(p)	D48E, A61T, A107T, S108G, L136P, G179S, S192A, E238D, P289L;<br>
(q)	D48E, L136P, G179S, R250W;<br>
(r)	D48E, A89T, S138T, A139T, R163Q, G179S;<br>
(s)	D48E, L136P, G179S, A198G, P289L;<br>
(t)	D48E, F78L, A89T, L136P, G179S;<br>
(u)	D48E, A89T, S138T, A139T, G179S, E238D, F278L;<br>
(v)	D48E, A89T, L136P, R163Q, G179S;<br>
(w)	D48E, A61T, A89T, G111V, S138T, A139F, G179S, E238D, P289L;<br>
(x)	D25G, D48E, A89T, L136P, S138T, A139T, V141A, I159T, R163Q, G179S;<br>
(y)	D48E, A89T, S90G, L136P, R163Q, G179S, E238D;<br>
(z)	D48E, A61T, A89T, G111V, S138T, A139T, G179S, E238D, P289L;<br>
(k)	D48E, A89T, S138T, A139T, G179S;<br>
(l)	D48E, L136P, R163Q, G179S, S231L;<br>
(m)	D48E, L136P, S138T, A139F, G179S, V196A, E238D;<br>
(n)	D48E, A61T, A89T, F99S, S138T, A139T, G179S, E238D;<br>
(o)	G35S, D48E, A89T, S138T, A139T, G179S, P289L;<br>
(p)	D48E, A61T, A89T, S138T, A139T, G179S, V196A, E238D;<br><br>
 (ag)       D48E, A89T, G111V, S138T, A139T, G179S, A198G, E238D;<br>
(ah)      S41G, D48E, A89T, L136P, G179S;<br>
(ai)        D48E, A89T, L136P, R163Q, G179S, P252S;<br>
(aj)        D48E, A89T, L136P, G179S, F234S;<br>
(ak)       D48E, A89T, L136P, R163Q, G179S, E238D;<br>
(al)        Q36R, D48E, A89T, L136P, G179S, E238D;<br>
(am)      D48E, A89T, L136P, R163Q, G179S;<br>
(an)      D48E, A89T, S138T, G179S;<br>
(ao)       D48E, A89T, L136P, G179S, E238D;<br>
(ap)       D48E, A89T, L136P, K154E, G179S, E238D;'<br>
(aq)       D48E, A89T, S138T, A139T, K154R, G179S, V196A, P289L;<br>
(ar)       D48E, A89T, S138T, A139F, G179S, V196A, E238D;<br>
(as)       D48E, A61T, A89T, L136P, G179S, V196A, A198G, P289L;<br>
(at)        D48E, A61T, S138T, A139F, G179S, G196A, E238D, P289L;<br>
(au)       D48E, A89T, L136P, G179S;<br>
(av)       D48E, A89T, V120A, L136P, G179S;<br>
(aw)      D48E, A61T, A89T, S138T, A139F.G179S, V196A, A198G, E238D;<br>
(ax)       D48E, A61T, A89T, G111V, S138T, A139F, G179S, V196A, E238D;<br>
(ay)       D48E, A61T, A89T, S138T, A139T, G179S, V196A, E238D, P289L;<br>
(az)       D48E, A61T, A89T, L136P, S138T, A139F, G179S, A198G, E238D;<br>
(u)	D48E, A89T, S138T, A139F, G179S, A198G, V220A;<br>
(v)	D48E, A61T, A89T, S138T, A139T, G179S, V196A, E238D, R239H, P289L;<br>
(be)      D48E, A61T, A89T, L136P, G179S, P289L;<br><br>
(x)	D48E, A89T, S138T, A139T, G179S, V196A, E238D, P289L; and<br>
(y)	D48E, A61T, A89T, S138T, A139F, G179S, V196A, E238D.<br>
The present invention further provides a polynucleotide molecule comprising a nucleotide sequence that is otherwise the same as the Streptomyces avermitilis aveC allele, the S. avermitilis AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604) or the nucleotide sequence of the aveC ORF of S. avermitilis as presented in FIGURE 1 (SEQ ID NO:1), or a degenerate variant thereof, but which nucleotide sequence further comprises mutations encoding a combination of amino acid substitutions at amino acid residues corresponding to the amino acid positions of SEQ ID NO:2, such that cells of S. avermitifis strain ATCC 53692 in which the wild-type aveC allele has been inactivated and that express a polynucleotide molecule comprising the mutated nucleotide sequence are capable of producing a class 2:1 ratio of avermectins that is reduced compared to the ratio produced by cells of S. avermitilis strain ATCC 53692 that instead express only the wild-type aveC allele,<br><br>
wherein when the class 2:1 avermectins are cyclohexyl B2:cyclohexyl B1 avermectins, the ratio of class 2:1 avermectins is reduced to about 0.40:1 or less, and wherein the combination of amino acid substitutions comprises a combination selected from the group consisting of:<br>
(z)	D48E, S138T, A139T, G179S, E238D; and<br>
(aa)	Y28C, Q38R, D48E, L136P, G179S, E238D.<br>
The present invention further provides a recombinant vector comprising a polynucleotide molecule of the present invention.<br>
The present invention further provides a host cell comprising a polynucleotide molecule or a recombinant vector of the present invention. In a preferred embodiment, the host cell is a Streptomyces cell. In a more preferred embodiment, the host cell is a cell of Streptomyces avermitilis.<br>
The present invention further provides a method for making a novel strain of Streptomyces avermitilis, comprising (i) mutating the aveC allele in a cell of a strain of S. avermitilis, which mutation results in a combination of amino acid substitutions in the AveC gene product, or (ii) introducing into a cell of a strain of S. avermitilis a mutated aveC allele or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions, wherein the combination of amino acid substitutions is selected from (a) through (be) listed above.<br>
The present invention further provides a method for making a novel strain of Streptomyces avermitilis, comprising (i) mutating the aveC allele in a cell of a strain of S. avermitilis, which mutation results in a combination of amino acid substitutions in the AveC gene product, or (ii) introducing into a cell of a strain of S. avermitilis a mutated aveC allele or degenerate variant thereof encoding an AveC gene product comprising a combination of ammo acid substitutions, wherein cells comprising the mutated aveC allele or degenerate variant are capable of producing cyclohexyl B2:cyclohexyl B1 avermectins in a ratio of 0.35:1 or less. In a non-limiting embodiment thereof, the mutated aveC allele or degenerate variant thereof encodes an AveC gene product comprising a combination of amino acid substitutions selected from the group consisting of (a) through (be) listed above.<br>
In a preferred embodiment thereof, the ratio of cyclohexyl B2:cyclohexyl B1 avermectins is about 0.30:1 or less. In a non-limiting embodiment, the mutated aveC allele or degenerate variant thereof encodes an AveC gene product comprising a combination of amino acid substitutions selected from the group consisting of (f) through (be) listed above.<br>
In a more preferred embodiment thereof, the ratio of cyclohexyl B2:cyclohexy! B1 avermectins is about 0.25:1 or less. In a non-limiting embodiment, the mutated aveC allele or degenerate variant thereof encodes an AveC gene product comprising a combination of amino acid substitutions selected from the group consisting of (w) through (be) listed above.<br><br>
In a more preferred embodiment thereof, the ratio of cyclohexyi B2:cyclohexyl B1 avermectins is about 0.20:1 or less. In a non-limiting embodiment, the mutated aveC allele or degenerate variant thereof encodes an AveC gene product comprising a combination of amino acid substitutions selected from the group consisting of (ao) through (be) listed above.<br>
The present invention further provides a method for making a novel strain of Streptomyces avermitilis, comprising (i) mutating the aveC allele in a cell of a strain of S. avermitilis, which mutation results in a combination of amino acid substitutions in the AveC gene product, or (ii) introducing into a cell of a strain of S. avermitilis a mutated aveC allele or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions, wherein the combination of amino acid substitutions is selected from the group consisting of (bf.) and (bg) listed above. In a preferred embodiment thereof, cells of S. avermitilis comprising such a mutated aveC allele or degenerate variant are capable of producing cyclohexyi B2:cyclohexyl B1 avermectins in a ratio of about 0.40:1 or less.<br>
The present invention further provides a cell of a Streptomyces species that comprises a mutated S. avermitilis aveC allele or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions selected from (a) through (be) listed above. In a preferred embodiment thereof, the species of Streptomyces is S. avermitilis.<br>
The present invention further provides a cell of Streptomyces avermitilis capable of producing cyclohexyi B2:cyclohexyl B1 avermectins in a ratio of 0.35:1 or less. In a non-limiting embodiment thereof, the cell comprises a mutated aveC allele or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions selected from the group consisting of (a) through (be) listed above.<br>
In a preferred embodiment thereof, the ratio of cyclohexyi B2:cyclohexyl B1 avermectins is about 0.30:1 or less. In a non-limiting embodiment thereof, the cells comprise a mutated aveC allele or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions selected from the group consisting of (f) through (be) listed above.<br>
In a more preferred embodiment thereof, the ratio of cyclohexyi B2:cyciohexyl B1 avermectins is about 0.25:1 or less. In a non-limiting embodiment thereof, the cells comprise a mutated aveC allele or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions selected from the group consisting of (w) through (be) listed above.<br>
In a more preferred embodiment thereof, the ratio of cyclohexyi B2:cyclohexyl B1 avermectins is about 0.20:1 or less. In a non-limiting embodiment thereof, the cells comprise a mutated allele or degenerate variant thereof encoding an AveC gene product<br><br>
comprising a combination of amino acid substitutions selected from the group consisting of (ao) through (be) listed above.<br>
The present invention further provides a cell of a Streptomyces species, comprising a mutated S. avermitilis aveC alleie or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions selected from the group consisting of (bf) and (bg) listed above. In a preferred embodiment thereof, the species of Streptomyces is S. avermitilis. In a more preferred embodiment, the cell is a cell of S. avermitilis capable of producing cyclohexyl B2:cyclohexyl B1 avermectins in a ratio of about 0.40:1 or less. .<br>
The present invention further provides a process for producing avermectins, comprising culturing a strain of Streptomyces avermitilis cells of the present invention in culture media under conditions that permit or induce the production of avermectins therefrom, and recovering said avermectins from the culture.<br>
The present invention further provides a composition of cyclohexyl B2:cyclohexyl B1 avermectins produced by cells of Streptomyces avermitilis, comprising the cyclohexyl B2:cyclohexyl B1 avermectins present in a culture medium in which the cells have been cultured, wherein the ratio of the cyclohexyl B2:cyclohexyl B1 avermectins present in the culture medium is 0.35:1 or less, preferably about 0.30:1 or less, more preferably about 0.25:1 or less, and more preferably about 0.20:1 or less. In a particular embodiment, the composition of cyclohexyl B2:cyclohexyl B1 avermectins is produced by cells of a strain of S. avermitilis that express a mutated aveC allele or degenerate variant thereof which encodes a gene product that results in the reduction of the class 2:1 ratio of cyclohexyl B2:cyclohexyl B1 avermectins produced by the cells compared to cells of the same strain of S. avermitilis that do not express the mutated aveC allele but instead express only the wild-type aveC allele.<br>
In a preferred embodiment thereof, where the composition is cyclohexyl B2:cyclohexyl B1 avermectins in a ratio of 0.35:1 or less, the composition is produced by cells comprising a mutated aveC allele or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions selected from the group consisting of (a) through (be) listed above.<br>
In a further preferred embodiment thereof, where the composition is cyclohexyl B2:cyclohexyl B1 avermectins in a ratio of about 0.30:1 or less, the composition is produced by cells comprising a mutated aveC allele or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions selected from the group consisting of (f) through (be) listed above.<br>
In a further preferred embodiment thereof, where the composition is cyclohexyl B2:cyclohexyl B1 avermectins in a ratio of about 0.25:1 or less, the composition is produced<br><br>
by cells comprising a mutated aveC atlele or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions selected from the group consisting of (w) through (be) listed above.<br>
In a further preferred embodiment thereof, where the composition is cyclohexyi B2:cyclohexyl B1 avermectins in a ratio of about 0.20:1 or less, the composition is produced by ceils comprising a mutated aveC allele or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions selected from the group consisting of (ao) through (be) listed above.<br>
The present invention further provides a composition of cyclohexyi B2:cyciohexyl B1 avermectins produced by cells of Streptomyces avermitilis, comprising the cyclohexyi B2:cyclohexyl B1 avermectins present in a culture medium in which the cells have been cultured, wherein the ratio of the cyclohexyl B2:cyclohexyl B1 avermectins present in the culture medium is about 0.40:1 or less, and produced by cells comprising a mutated aveC allele or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions selected from the group consisting of (bf) and (bg) listed above.<br>
4. BRIEF DESCRIPTION OF THE FIGURES<br>
FIGURE 1. DNA sequence (SEQ ID NO:1) comprising the S. avermitilis aveC ORF, and deduced amino acid sequence (SEQ ID NO:2).<br>
FIGURE 2. Plasmid vector pSE186 (ATCC 209604) comprising the entire ORF of the aveC gene of S. avermitilis.<br>
FIGURE 3. Gene replacement vector pSE180 (ATCC 209605) comprising the ermE gene of Sacc. erythraea inserted into the aveC ORF of S. avermitilis.<br>
FIGURE 4. BamHl restriction map of the avermectin polyketide synthase gene cluster from S. avermitilis with five overlapping cosmid clones identified (i.e., pSE65, pSE66, pSE67, pSE68, pSE69). The relationship of pSE118 and pSE119 is also indicated.<br>
FIGURE 5. HPLC analysis of fermentation products produced by S. avermitilis strains. Peak quantisation was performed by comparison to standard quantities of cyclohexyi B1. Cyclohexyl B2 retention time was 7.4-7.7 min; cyclohexyi B1 retention time was 11.9-12.3 min. FIG. 5A. S, avermitilis strain SE180-11 with an inactivated aveC ORF. FIG. 5B. S. avermitilis strain SE180-11 transformed with pSE186 (ATCC 209604). FIG. 5C. S. avermitilis strain SE180-11 transformed with pSE187. FIG. 5D. S. avermitilis strain SE180-11 transformed with pSE188.<br>
FIGURE 6A-M. Compiled list of combinations of amino acid substitutions encoded by mutations to the aveC aiiele as identified by a second round of "gene shuffling", and their effects on the ratio of cyclohexy! B2:cyclohexyl B1 production. For each plasmid, in the column entitled "Mutations", the upper box lists the amino acid substitutions, and the lower<br><br>
box lists the nucleotide base changes resulting in those amino acid substitutions. Nucleotide base changes in parentheses are silent changes, i.e., they do not result in changes to the amino acid sequence.<br>
5.  DETAILED DESCRIPTION OF THE INVENTION<br>
The present invention relates to the identification and characterization of polynucleotide molecules having nucleotide sequences that encode the AveC gene product from Streptomyces avermitilis, the construction of novel strains of S. avermitilis that can be used to screen mutated AveC gene products for their effect on avermectin production, and the discovery that certain mutated AveC gene products can reduce the ratio of B2:B1 avermectins produced by S. avermitilis. By way of example, the invention is described in the sections below for a polynucleotide molecule having either a nucleotide sequence that is the same as the S. avermitilis AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604), or the nucleotide sequence of the ORF of FIGURE 1 (SEQ ID NO:1), and for polynucleotides molecules having mutated nucleotide sequences derived therefrom and degenerate variants thereof. However, the principles set forth in the present invention can be analogously applied to other polynucleotide molecules, including aVeC homolog genes from other Streptomyces species including, e.g., S. hygroscopicus and S. griseochromogenes, among others.<br>
5.1.       Poiynucleotide Molecules Encoding The S. avermitilis AveC Gene Product<br>
The present invention provides an isolated polynucleotide molecule comprising the complete aveC ORF of S. avermitilis or a substantial portion thereof, which isolated polynucleotide molecule lacks the next complete ORF that is located downstream from the aveC ORF in situ in the S. avermitilis chromosome.<br>
The isolated polynucleotide molecule of the present invention preferably comprises a nucleotide sequence that is the same as the S. avermitilis AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604), or that is the same as the nucleotide sequence of the ORF of FIGURE 1 (SEQ ID NO:1) or substantial portion thereof. As used herein, a "substantial portion" of an isolated polynucleotide molecule comprising a nucleotide sequence encoding the S. avermitilis AveC gene product means an isolated poiynucieotide molecule comprising at least about 70% of the complete aveC ORF sequence shown in FIGURE 1 (SEQ ID NO:1), and that encodes a functionally equivalent AveC gene product. In this regard, a "functionally equivalent" AveC gene product is defined as a gene product that, when expressed in S. avermitilis strain ATCC 53692 in which the native aveC allele has been inactivated, results in the production of substantially the same ratio and amount of<br><br>
avermectins as produced by S. avermitilis strain ATCC 53692 which instead expresses only the wild-type, functional aveC allele native to S. avermitilis strain ATCC 53692.<br>
In addition to the nucieotide sequence of the av/eC ORF, the isolated polynucleotide molecule of the present invention can further comprise nucieotide sequences that naturally flank the aveC gene in situ in S. avermitilis, such as those flanking nucieotide sequences shown in FIGURE 1 (SEQ ID NO:1).<br>
The present invention further provides an isolated polynucleotide molecule comprising the nucieotide sequence of SEQ ID NO:1 or a degenerate variant thereof, as based on the known degeneracy of the genetic code.<br>
As used herein, the terms "polynucleotide molecule," "polynucleotide sequence," "coding sequence," "open-reading frame," and "ORF" are intended to refer to both DNA and RNA molecules, which can either be single-stranded or double-stranded, and that can be transcribed and translated (DNA), or translated (RNA), into an AveC gene product, or into a polypeptide that is homologous to an AveC gene product in an appropriate host cell expression system when placed under the control of appropriate regulatory elements. A coding sequence can include but is not limited to prokaryotic sequences, cDNA sequences, genomic DNA sequences, and chemically synthesized DNA and RNA sequences.<br>
The nucieotide sequence shown in FIGURE 1 (SEQ ID NO:1) comprises four different GTG codons at bp positions 42,174,177 and 180. As previously described in U.S. Patent No. 6,248,579, multiple deletions of the 5' region of the aveC ORF (FIGURE 1; SEQ ID NO:1) were constructed to help define which of these codons could function in the aveC ORF as start sites for protein expression. Deletion of the first GTG site at bp 42 did not eliminate AveC activity. Additional deletion of all of the GTG codons at bp positions 174, 177 and 180 together eliminated AveC activity, indicating that this region is necessary for protein expression. The present invention thus encompasses variable length aveC ORFs.<br>
The present invention further provides a polynucleotide molecule having a nucieotide sequence that is homologous to the S. avermitilis AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604), or to the nucieotide sequence of the aveC ORF presented in FIGURE 1 (SEQ ID NO:1) or substantial portion thereof. The term "homologous" when used to refer to a polynucleotide molecule that is homologous to an S. avermitilis AveC gene product-encoding sequence means a polynucleotide molecule having a nucieotide sequence: (a) that encodes the same AveC gene product as the S. avermitilis AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604), or that encodes the same AveC gene product as the nucieotide sequence of the aveC ORF presented in FIGURE 1 (SEQ ID NO:1), but that includes one or more silent changes to the nucieotide sequence according to the degeneracy of the genetic code (i.e., a degenerate variant); or (b) that hybridizes to the<br><br>
compfement of a polynucleotide molecule having a nucleotide sequence that encodes the<br>
amino acid sequence encoded by the AveC gene product-encoding sequence of plasm id<br>
pSE186 (ATCC 209604) or that encodes the amino acid sequence shown in FIGURE 1 (SEQ<br>
ID NO:2) under moderately stringent conditions, i.e., hybridization to filter-bound DNA in 0.5 M<br>
NaHPCXj,   7%   sodium   dodecyl  sulfate  (SDS),   1   mM   EDTA  at  65°C,   and  washing   in<br>
0.2xSSC/0.1% SDS at 42°C (see Ausubel et al. (eds.), 1989, Current Protocols in Molecular<br>
Biology, Vol. I, Green Publishing Associates, Inc., and John Wiley &amp; Sons, Inc., New York, at<br>
p. 2.10.3), and encodes a functionally equivalent AveC gene product as defined above, 'in a<br>
preferred embodiment, the homologous polynucleotide molecule hybridizes to the complement<br>
of the AveC gene product-encoding nucleotide sequence of plasmid pSE186 (ATCC 209604)<br>
or to the complement of the nucleotide sequence of the aueC ORF presented in FIGURE 1<br>
(SEQ   ID   NO:1)   or  substantial   portion   thereof  under  highly  stringent  conditions,   i.e.,<br>
hybridization to filter-bound DNA in 0.5 M NaHPO4, 7% SDS, 1 mM EDTA at 65°C, and<br>
washing in 0.1xSSC/0.1% SDS at 68°C (Ausubel et al.,  1989, above), and encodes a<br>
functionally equivalent AveC gene product as defined above.<br>
The activity of an AveC gene product .and potential functional equivalents thereof can be determined through HPLC analysis of fermentation products, as described in the examples below. Polynucleotide molecules having nucleotide sequences that encode functional equivalents of the S. avermitilis AveC gene product may include naturally occurring aveC genes present in other strains of S. avermitilis, aveC homolog genes present in other species of Streptomyces, and mutated aveC alleles, whether naturally occurring or engineered.<br>
The present invention further provides a polynucleotide molecule comprising a nucleotide sequence that encodes a polypeptide having an amino acid sequence that is homologous to the amino acid sequence encoded by the AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604), or the amino acid sequence of FIGURE 1 (SEQ ID NO:2) or substantial portion thereof. As used herein, a "substantial portion" of the amino acid sequence of FIGURE 1 (SEQ ID NO:2) means a polypeptide comprising at least about 70% of the amino acid sequence shown in FIGURE 1 (SEQ ID N0:2), and that constitutes a functionally equivalent AveC gene product, as defined above.<br>
As used herein to refer to amino acid sequences that are homologous to the amino acid sequence of an AveC gene product from S. avermitilis, the term "homologous" refers to a polypeptide which otherwise has the amino acid sequence of FIGURE 1 (SEQ ID NO:2), but in which one or more amino acid residues has been conservatively substituted with a different amino acid residue, wherein said amino acid sequence has at least about 70%, more preferably at least about 80%, and most preferably at least about 90% amino acid sequence identity to the polypeptide encoded by the AveC gene product-encoding sequence of plasmid<br><br>
pSE186 (ATCC 209604) or the amino acid sequence of FIGURE 1   (SEQ ID NO:2) as<br>
determined by any standard amino acid sequence identity algorithm, such as the BLASTP<br>
algorithm   (GENBANK,  NCBI),  and  where   such  conservative  substitution   results  in  a<br>
functionally equivalent gene product, as defined above. Conservative amino acid substitutions<br>
are well known in the art.    Rules for making such substitutions include those described by<br>
Dayhof, M.D., 1978, Nat. Biomed. Res. Found., Washington, D.C., Vol. 5, Sup. 3, among<br>
others.  More specifically, conservative amino acid substitutions are those that generally take<br>
place within a family of amino acids that are related in the acidity or polarity.   Genetically<br>
encoded amino acids are generally divided into four groups: (1) acidic = aspartate, glutamate;<br>
(2) basic = lysine, arginine,  histidine; (3) non-polar = alanine, valine, leucine, isoleucine,<br>
proline, phenylalanine, methionine, tryptophan; and (4) uncharged polar = glycine, asparagine,<br>
glutamine, cysteine, serine, threonine, tyrosine.   Phenylalanine, tryptophan and tyrosine are<br>
also jointly classified as aromatic amino acids.    One or more replacements within any<br>
particular group, e.g., of a leucine with an isoleucine or valine, or of an aspartate with a<br>
glutamate, or of a threonine with a serine, or of any other amino acid residue with a structurally<br>
related amino acid residue, e.g., an amino acid residue with similar acidity or polarity, or with<br>
similarity in some combination thereof, will generally have an insignificant effect on the<br>
function of the polypeptide.<br>
Production and manipulation of the polynucleotide molecules disclosed herein are within the skill in the art and can be carried out according to recombinant techniques described, e.g., in Maniatis, et al., 1989, Molecular Cloning. A Laboratory Manual. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY; Ausubel, et al., 1989, Current Protocols In Molecular Biology, Greene Publishing Associates &amp; Wiley Interscience, NY; Sambrook, et al., 1989, Molecular Cloning: A Laboratory Manual. 2d ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY; Innis et al. (eds), 1995, PCR Strategies. Academic Press, Inc., San Diego; and Erlich (ed), 1992, PCR Technology, Oxford University Press, New York, all of which are incorporated herein by reference. Polynucleotide clones encoding AveC gene products or AveC homolog gene products can be identified using any method known in the art, including but not limited to the methods set forth in Section 7, below. Genomic DNA libraries can be screened for aveC and aveC homolog coding sequences using techniques such as the methods set forth in Benton and Davis, 1977, Science 196:180, for bacteriophage libraries, and in Grunstein and Hogness, 1975, Proc. Natl. Acad. Sci. USA, 72:3961-3965, for plasmid libraries. Polynucleotide molecules having nucleotide sequences known to include the aveC ORF, as present, e.g., in plasmid pSE186 (ATCC 209604), or in plasmid pSE119 (described in Section 7, below), can be used as probes in these screening experiments. Alternatively, oligonucleotide probes can be synthesized that correspond to nucleotide<br><br>
sequences deduced from partial or complete amino acid sequences of the purified AveC homoiog gene product.<br>
5.2. Recombinant Systems 5.2.1. Cloning And Expression Vectors<br>
The present invention further provides recombinant cloning vectors and expression vectors which are useful in cloning or expressing polynucleotide molecules of the present invention comprising, e.g., the aveC ORF of S. avermitilis or any aveC homoiog ORFs. In a non-limiting embodiment, the present invention provides plasmid pSE186 (ATCC 209^04), which comprises the complete ORF of the aveC gene of S. avermitilis.<br>
All of the following description regarding the aveC ORF from S. avermitilis, or a polynucleotide molecule comprising the aveC ORF from S. avermitilis or portion thereof, or an S. avermitilis AveC gene product, also refers to mutated aveC alleles as described below, unless indicated explicitly or by context.<br>
A variety of different vectors have been developed for specific use in Streptomyces, including phage, high copy number plasmids, low copy number plasmids, and E.coli-Streptomyces shuttle vectors, among others, and any of these can be used to practice the present invention. A number of drug resistance genes have also been cloned from Streptomyces, and several of these genes have been incorporated into vectors as selectable markers. Examples of current vectors for use in Streptomyces are presented, among other places, in Hutchinson, 1980, Applied Biochem. Biotech. 16:169-190.<br>
Recombinant vectors of the present invention, particularly expression vectors, are preferably constructed so that the coding sequence for the polynucleotide molecule of the invention is in operative association with one or more regulatory elements necessary for transcription and translation of the coding sequence to produce a polypeptide. As used herein, the term "regulatory element" includes but is not limited to nucleotide sequences that encode inducible and non-inducible promoters, enhancers, operators and other elements known in the art that serve to drive and/or regulate expression of polynucleotide coding sequences. Also, as used herein, the coding sequence is in "operative association" with one or more regulatory elements where the regulatory elements effectively regulate and allow for the transcription of the coding sequence or the translation of its mRNA, or both.<br>
Typical plasmid vectors that can be engineered to contain a polynucleotide molecule of the present invention include pCR-Blunt, pCR2.1 (Invitrogen), pGEMSZf (Promega), and the shuttle vector pWHMS (Vara ef a/., 1989, J. Bact. 171:5872-5881), among many others.<br>
Methods are well-known in the art for constructing recombinant vectors containing particular coding sequences in operative association with appropriate regulatory elements, and these can be used to practice the present invention. These methods include in vitro<br><br>
recombinant techniques, synthetic techniques, and in vivo genetic recombination. See, e.g., the techniques described in Maniatis et a/., 1989, above; Ausubel et a/., 1989, above; Sambrook etal., 1989, above; Innis etal., 1995, above; and Erlich, 1992, above.<br>
The regulatory elements of these vectors can vary in their strength and specificities. Depending on the host/vector system, utilized, any of a number of suitable transcription and translation elements can be used. Non-limiting examples of transcriptional regulatory regions<br>
or promoters for bacteria include the p-gal promoter, the T7 promoter, the TAG promoter, A<br>
'	/<br>
left and right promoters, trp and lac promoters, trp-lac fusion promoters and, more specifically<br>
for Streptomyces, the promoters ermE, meIC, and tipA, etc. In a specific embodiment, an expression vector can be generated that contains the aveC ORF or mutated ORF thereof cloned adjacent to a strong constitutive promoter, such as the ermE promoter from Saccharopolyspora erythraea. As described in U.S. Patent No. 6,248,579, a vector comprising the ermE promoter was transformed into S. avermitilis, and subsequent HPLC analysis of fermentation products indicated an increased titer of avermectins produced compared to production by the same strain which instead expresses only the wild-type aveC allele.<br>
Fusion protein expression vectors can be used to express an AveC gene product-fusion protein. The purified fusion protein can be used to raise antisera against the AveC gene product, to study the biochemical properties of the AveC gene product, to engineer AveC fusion proteins with different biochemical activities, or to aid in the identification or purification of the expressed AveC gene product. Possible fusion protein expression vectors include but are not limited to vectors incorporating sequences that encode p-galactosidase and trpE fusions, maltose-binding protein fusions, glutathione-S-transferase fusions and polyhistidine fusions (carrier regions). In an alternative embodiment, an AveC gene product or a portion thereof can be fused to an AveC homolog gene product, or portion thereof, derived from another species or strain of Streptomyces, such as, e.g., S. hygroscopicus or S. griseochromogenes. Such hybrid vectors can be transformed into S. avermitilis cells and tested to determine their effect, e.g., on the ratio of class 2:1 avermectin produced.<br>
AveC fusion proteins can be engineered to comprise a region useful for purification. For example, AveC-maltose-binding protein fusions can be purified using amylose resin; AveC-glutathione-S-transferase fusion proteins can be purified using glutathione-agarose beads; and AveC-polyhistidine fusions can be purified using divalent nickel resin. Alternatively, antibodies against a carrier protein or peptide can be used for affinity chromatography purification of the fusion protein. For example, a nucleotide sequence coding for the target epitope of a monoclonal antibody can be engineered into the expression vector in operative association with the regulatory elements and situated so that the expressed<br><br><br>
epitope is fused to the AveC polypeptide. For example, a nucleotide sequence coding for the FLAG™ epitope tag (International Biotechnologies Inc.), which is a hydrophilic marker peptide, can be inserted by standard techniques into the expression vector at a point corresponding, e.g., to the carboxyl terminus of the AveC polypeptide. The expressed AveC polypeptide-FLAG™ epitope fusion product can then be detected and affinity-purified using commercially available anti-FLAG™ antibodies.<br>
The expression vector encoding the AveC fusion protein can also be engineered to contain polyiinker sequences that encode specific protease cleavage sites so that the expressed AveC polypeptide can be released from the carrier region or fusion partner by treatment with a specific protease. For example, the fusion protein vector can include DNA sequences encoding thrombin or factor Xa cleavage sites, among others.<br>
A signal sequence upstream from, and in reading frame with, the aveC ORF can be engineered into the expression vector by known methods to direct the trafficking and secretion of the expressed gene product. Non-limiting examples of signal sequences include those from a-factor, immunoglobulins, outer membrane proteins, penicillinase, and T-cell receptors, among others.<br>
To aid in the selection of host cells transformed or transfected with cloning or expression vectors of the present invention, the vector can be engineered to further comprise a coding sequence for a reporter gene product or other selectable marker. Such a coding sequence is preferably in operative association with the regulatory element coding sequences, as described above. Reporter genes that are useful in the invention are well-known in the art and include those encoding green fluorescent protein, luciferase, xylE, and tyrosinase, among others. Nucleotide sequences encoding selectable markers are well known in the art, and include those that encode gene products conferring resistance to antibiotics or anti-metabolites, or that supply an auxotrophic requirement. Examples of such sequences include those that encode resistance to erythromycin, thiostrepton or kanamycin, among many others.<br>
5.2.2. Transformation Of Host Cells<br>
The present invention further provides transformed host cells comprising a polynucleotide molecule or recombinant vector of the invention, and novel strains or cell lines derived therefrom. Host cells useful in the practice of the invention are preferably Streptomyces cells, although other prokaryotic cells or eukaryotic cells can also be used. Such transformed host cells typically include but are not limited to microorganisms, such as bacteria transformed with recombinant bacteriophage DNA, plasmid DNA or cosmid DNA vectors, or yeast transformed with recombinant vectors, among others.<br>
The polynucleotide molecules of the present invention are intended to function in Streptomyces ceils, but can also be transformed into other bacterial or eukaryotic cells, e.g.,<br><br>
for cloning or expression purposes. A strain of E. coli can typically be used, such as, e.g., the DH5a strain, available from the American Type Culture Collection (ATCC), Rockville, MD, USA (Accession No. 31343), and from commercial sources (Stratagene). Preferred eukaryotic host cells include yeast cells, although mammalian cells or insect cells can also be utilized effectively.<br>
The recombinant expression vector of the invention is preferably introduced, e.g., transformed or transfected, into one or more host cells of a substantially homogeneous culture of cells. The expression vector is generally introduced into host cells in accordance with known techniques, such as, e.g., by protoplast transformation, calcium phosphate precipitation, calcium chloride treatment, microinjection, electroporation, transfection by contact with a recombined virus, liposome-mediated transfection, DEAE-dextran transfection, transduction, conjugation, or microprojectile bombardment. Selection of transformants can be conducted by standard procedures, such as by selecting for cells expressing a selectable marker, e.g., antibiotic resistance, associated with the recombinant vector, as described above.<br>
Once the expression vector is introduced into the host cell, the integration and maintenance of the aveC coding sequence either in the host cell chromosome or episomally can be confirmed by standard techniques, e.g., by Southern hybridization analysis, restriction enzyme analysis, PCR analysis, including reverse transcriptase PCR (rt-PCR), or by immunological assay to detect the expected gene product. Host cells containing and/or expressing the recombinant aveC coding sequence can be identified by any of at least four general approaches which are well-known in the art, including: (i) DNA-DNA, DNA-RNA, or RNA-antisense RNA hybridization; (ii) detecting the presence of "marker" gene functions; (iii) assessing the level of transcription as measured by the expression of aveC-specific mRNA transcripts in the host cell; and (iv) detecting the presence of mature polypeptide product as measured, e.g., by immunoassay or by the presence of AveC biological activity (e^g., the production of specific ratios and amounts of avermectins indicative of AveC activity in, e.g., S. avermitilis host cells).<br>
5.2.3.    Expression And Characterization<br>
Of A Recombinant AveC Gene Product<br>
Once the native or mutated aveC coding sequence has been stably introduced into an appropriate host cell, the transformed host cell is clonally propagated, and the resulting cells can be grown under conditions conducive to the maximum production of the native or mutated AveC gene product. Such conditions typically include growing cells to high density. Where the expression vector comprises an inducible promoter, appropriate induction conditions such as, e.g., temperature shift, exhaustion of nutrients, addition of gratuitous inducers (e.g.,<br><br>
analogs of carbohydrates, such as isopropyl-(3-D-thiogalactopyranoside (IPTG)), accumulation of excess metabolic by-products, or the like, are employed as needed to induce expression.<br>
Where the expressed AveC gene product is retained inside the host cells, the cells<br>
are harvested and lysed, and the product isolated and purified from the lysate under extraction<br>
conditions known in the art to minimize protein degradation such as, e.g., at 4°C, or in the<br>
presence of protease inhibitors, or both. Where the expressed AveC gene product is secreted<br>
from the host cells, the exhausted nutrient medium can simply be collected and the product<br>
isolated therefrom.	•'<br>
The expressed AveC gene product can be isolated or substantially purified from cell lysates or culture medium, as appropriate, using standard methods, including but not limited to any combination of the following methods: ammonium sulfate precipitation, size fractionation, ion exchange chromatography, HPLC, density centrifugation, and affinity chromatography. Where the expressed AveC gene product exhibits biological activity, increasing purity of the preparation can be monitored at each step of the purification procedure by use of an appropriate assay. Whether or not the expressed. AveC gene product exhibits biological activity, it can be detected as based, e.g., on size, or reactivity with an antibody otherwise specific for AveC, or by the presence of a fusion tag. As used herein, an AveC gene product is "substantially purified" where the product constitutes more than about 20 wt% of the protein in a particular preparation. Also, as used herein, an AveC gene product is "isolated" where the product constitutes at least about 80 wt% of the protein in a particular preparation.<br>
The present invention thus provides a recombinantly-expressed isolated or substantially purified S. avermitilis AveC gene product comprising the amino acid sequence encoded by the AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604), or the amino acid sequence of FIGURE 1 (SEQ ID NO:2) or a substantial portion thereof, and mutated versions and degenerate variants thereof.<br>
The present invention further provides a method for producing an AveC gene product, comprising culturing a host cell transformed with a recombinant expression vector, said vector comprising a polynucleotide molecule having a nucieotide sequence encoding the AveC gene product, which polynucleotide molecule is in operative association with one or more regulatory elements that control expression of the polynucleotide molecule in the host cell, under conditions conducive to the production of the recombinant AveC gene product, and recovering the AveC gene product from the cell culture.<br>
The recombinantly expressed S. avermitilis AveC gene product is useful for a variety of purposes, including for screening compounds that alter AveC gene product function and thereby modulate avermectin biosynthesis, and for raising antibodies directed against the AveC gene product.<br><br>
Once an AveC gene product of sufficient purity has been obtained,  it can be characterized by standard methods, including by SDS-PAGE, size exclusion chromatography, amino acid sequence analysis, biological activity in producing appropriate products in the avermectin biosynthetic pathway, efc.   For example, the amino acid sequence of the AveC gene product can be determined using standard peptide sequencing techniques.  The AveC gene product can be further characterized using hydrophilicity analysis (see, e.g., Hopp and Woods, 1981, Proc. Natl. Acad. Sci. USA   78:3824), or analogous software algorithms, to identify hydrophobic and hydrophilic regions of the AveC gene product.   Structural analysis can be carried out to identify regions of the AveC gene product that assume specific secondary structures.  Biophysical methods such as X-ray crystallography (Engstrom, 1974, Biochem. Exp. Biol. 11: 7-13), computer modelling (Fletterick and Zoller (eds), 1986, in: Current Communications in Molecular Biology. Cold Spring Harbor Laboratory, Cold Spring Harbor, NY), and nuclear magnetic resonance (NMR) can be used to map and study sites of interaction between the AveC gene product and its substrate.   Information obtained from these studies can be used to select new sites for mutation in the aveC ORF to help develop new strains of S. avermitilis having more desirable avermectin production characteristics .<br>
5.3. Construction And Use Of AveC Mutants<br>
A primary objective of the present invention is to identify novel mutations in the aveC allele of S. avermitilis that result in a change, and most preferably a reduction, in the ratio of B2:B1 avermectins. The present invention thus provides polynucleotide molecules useful to produce novel strains of S. averm/W/AS-cells that exhibit a detectable change in avermectin production compared to cells of the same strain but which instead express only the wild-type aveC allele. In a preferred embodiment, such polynucleotide molecules are useful to produce novel strains of S. avermitilis cells that produce avermectins in a reduced class 2:1 ratio compared to celis of the same strain which instead express only the wild-type aveC allele. The cells of such strains can also comprise additional mutations to produce an increased amount of avermectins compared to cells of the same strain that instead express only a single wild-type aveC allele.<br>
Mutations to the aveC allele or coding sequence include any mutations that introduce one or more amino acid substitutions, deletions and/or additions into the AveC gene product, or that result in truncation of the AveC gene product, or any combination thereof, and that produce the desired result. Such mutated aveC allele sequences are intended to include any degenerate variants thereof. For example, the present invention provides a polynucleotide molecule comprising the nucleotide sequence of the aveC allele or a degenerate variant thereof, or the AveC gene product-encoding sequence of plasmid pSEl86 (ATCC 209604) or a degenerate variant thereof, or the nucleotide sequence of the aveC ORF of S. avermitilis as<br><br>
present in FIGURE 1 (SEQ ID N0:1) or a degenerate variant thereof, but that further comprises mutations that encode a combination of amino acid substitutions at selected positions in the AveC gene product. In a non-limiting embodiment, such substitutions occur at one or more amino acid positions of the AveC gene product corresponding to amino acid positions 25, 28, 35, 36, 38, 40, 41, 48, 55, 61, 78, 84, 89, 90, 99, 107, 108, 111, 120, 123, 136, 138, 139, 141, 154, 159, 163, 179, 192, 196, 198, 200, 202, 220, 228, 229, 230, 231, 234, 238, 239, 250, 252, 266, 275, 278, 289 or 298 of SEQ ID NO:2. Preferred combinations of amino acid positions to be substituted comprise one or more of amino acid residues D48, A61, A89, LI 36, S138, A139, R163, G179, V196, A198, E238 and P289. Specifically preferred combinations of amino acid substitutions comprise substitutions at both D48 and G179, and more specifically D48E and G179S. Specific examples of combinations of amino acid substitutions that result in a reduction in cyclohexylB2;cyclohexyl B1 ratios are listed in FIGURE 6A-J.<br>
The  present  invention  thus  provides  a  polynucleotide  molecule  comprising  a<br>
nucleotide sequence that is otherwise the same as the Streptomyces avermitilis aveC allele,<br>
the S. avermitilis AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604)<br>
or the nucleotide sequence of the aveC ORF of S. avermitilis as presented in FIGURE 1 (SEQ<br>
ID N0:1), or a degenerate variant thereof, but which nucleotide sequence further comprises<br>
mutations encoding a combination of amino acid substitutions at amino acid  residues<br>
corresponding to the amino acid positions of SEQ ID NO:2, such that cells of S. avermitilis<br>
strain ATCC 53692 in which the wild-type aveC allele has been inactivated and that express<br>
the polynucleotide molecule comprising the mutated nucleotide sequence are capable of<br>
producing a class 2:1 ratio of avermectins that is reduced compared to the ratio produced by<br>
cells of S. avermitilis strain ATCC 53692 that instead express only the wild-type aveC allele,<br>
wherein when the class 2:1 avermectins are cyclohexyl B2:cyclohexyl B1 avermectins, the<br>
ratio of class 2:1 avermectins is 0.35:1 or less.  In a more preferred embodiment, the ratio of<br>
cyclohexyl B2:cyciohexyl  B1   avermectins is about 0.30:1   or less.  In a more preferred<br>
embodiment, the ratio of cyclohexyl B2:cyclohexyl B1 avermectins is about 0.25:1 or less. In a<br>
more preferred embodiment, the ratio of cyclohexyl B2:cyclohexyl B1 avermectins is about<br>
0.20:1 or less.<br>
In a particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (a): D48E, A61T, A89T, S138T, A139T, G179S, A198G, P289L. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE538 (see FIGURE 6).<br><br>
ln another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (b): G40S, D48E, L136P, G179S, E238D. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE559.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group, (c): D48E, L136P, R163Q, G179S. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE567.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (d): D48E, L136P, R163Q, G179S, E238D. Non-limiting examples of plasmids encoding these amino acid substitutions are pSE570 and pSE572.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (e): D48E, L136P, R163Q, G179S, A200G, E238D. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE571.<br>
In a particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (f): D48E, L136P, G179S, E238D. Non-limiting examples of plasmids encoding these amino acid substitutions are pSE501 and pSE546.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (g): D48E, A61T, L136P, G179S, E238D. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE510.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (h): D48E, A61T, L136P, G179S. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE512.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (i): D48E, A89T, S138T, A139T, G179S. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE519.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group Q): D48E, A61T, L136P, G179S, A198G, P202S, E238D, P289L. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE526.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (k): D48E, A61T, L136P, S138T, A139F, G179S, E238D, P289L. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE528.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (I): D48E, L136P, G179S, A198G, E238D, P289L A non-limiting example of a plasmid encoding these amino acid substitutions is pSE530.<br><br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (m): D48E, A61T, S138T, A139F, G179S, A198G, P289L. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE531.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (n): D48E, L84P, G111V, S138T, A139T, G179S, A198G, P289L. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE534.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (o): Y28C, D48E, A61T, A89T, S138T, A139T, G179S, E238D. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE535.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (p): D48E, A61T, A107T, S108G, L136P, G179S, S192A, E238D, P289L. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE542.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (q): D48E, L136P, G179S, R250W. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE545.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (r): D48E, A89T, S138T, A139T, R163Q, G179S. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE548.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (s): D48E, L136P, G179S, A198G, P289L A non-limiting example of a plasmid encoding these amino acid substitutions is pSE552.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (t): D48E, F78L, A89T, L136P, G179S. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE557.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (u): D48E, A89T, S138T, A139T, G179S, E238D, F278L. Non-limiting examples of plasmids encoding these amino acid substitutions are pSE564 and pSE565.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (v): D48E, A89T, L136P, R163Q, G179S. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE568.<br><br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (w): D48E, A61T, A89T, G111V, S138T, A139F, G179S, E238D, P289L . A non-limiting example of a plasmid encoding these amino acid substitutions is pSE543.<br>
In a particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (x): D25G, D48E, A89T, L136P, S138T, A139T, V141A, I159T, R163Q, G179S. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE504.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (y): D48E, A89T, S90G, L136P, R163Q, G179S, E238D. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE508.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (z): D48E, A61T, A89T, G111V, S138T, A139T, G179S, E238D, P289L. A non-limiting example of a plasmid encoding these amino acid substitutions ispSE511.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (aa): D48E, A89T, S138T, A139T, G179S. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE520.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (ab): D48E, L136P, R163Q, G179S, S231L. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE523.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (ac): D48E, L136P, S138T, A139F, G179S, V196A, E238D. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE527.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (ad): D48E, A61T, A89T, F99S, S138T, A139T, G179S, E238D. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE539.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (ae): G35S, D48E, A89T, S138T, A139T, G179S, P289L. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE540.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (af): D48E, A61T, A89T, S138T, A139T, G179S, V196A, E238D. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE547.<br><br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination' of group (ag): D48E, A89T, G111V, S138T, A139T, G179S, A198G, E238D. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE550.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (ah): S41G, D48E, A89T, L136P, G179S. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE558.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (ai): D48E, A89T, L136P, R163Q, G179S, P252S. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE563.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (aj): D48E, A89T, L136P, G179S, F234S. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE566.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (ak): D48E, A89T, L136P, R163Q, G179S, E238D. Non-limiting examples of plasmids encoding these amino acid substitutions are pSE573 and pSE578.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (al): Q36R, D48E, A89T, L136P, G179S, E238D. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE574.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (am): D48E, A89T, L136P, R163Q, G179S. Non-limiting examples of plasmids encoding these amino acid substitutions are pSE575 and pSE576.<br>
In another particular embodiment, the combination of amino acid substitutions comprises the combination of group (an): D48E, A89T, S138T, G179S. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE577.<br>
In a particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (ao): D48E, A89T, L136P, G179S, E238D. Non-limiting examples of plasmids encoding these amino acid substitutions are pSE502 and pSE524.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (ap): D48E, A89T, L136P, K154E, G179S, E238D. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE503.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (aq): D48E, A89T, S138T, A139T, K154R, G179S, V196A, P289L. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE505.<br><br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (ar): D48E, A89T, S138T, A139F, G179S, V196A, E238D. A non-iimiting example of a plasmid encoding these amino acid substitutions is pSE506.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (as): D48E, A61T, A89T, L136P, G179S, V196A, A198G, P289L. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE507.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (at): D48E, A61T, S138T, A139F, G179S, G196A, E238D, P289L. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE509.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (au): D48E, A89T, L136P, G179S. Non-limiting examples of plasmids encoding these amino acid substitutions are pSE514 and pSE525.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (av): D48E, A89T, V120A, L136P, G179S. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE515.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (aw): D48E, A61T, A89T, S138T, A139F.G179S, V196A, A198G, E238D. A non-limiting example of a plasmid encoding these amino acid substitutions ispSE517.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (ax): D48E, A61T, A89T, G111V, S138T, A139F, G179S, V196A, E238D. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE518.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (ay): D48E, A61T, A89T, S138T, A139T, G179S, V196A, E238D, P289L. Non-limiting examples of plasmids encoding these amino acid substitutions are pSE529 and pSE554.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (az): D48E, A61T, A89T, L136P, S138T, A139F, G179S, A198G, E238D. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE532.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (ba) D48E, A89T, S138T, A139F, G179S, A198G,<br><br>
V220A.   A non-limiting example of a plasmid encoding these amino acid substitutions is pSE536.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (bb): D48E, A61T, A89T, S138T, A139T, G179S, V196A, E238D, R239H, P289L. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE537.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (be): D48E, A61T, A89T, L136P, G179S, P289L. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE541.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (bd): D48E, A89T, S138T, A139T, G179S, V196A, E238D, P289L. Non-limiting examples of plasmids encoding these amino acid substitutions are pSE549 and pSE553.<br>
In another particular embodiment thereof, the combination of amino acid substitutions comprises the combination of group (be): D48E, A61T, A89T, S138T, A139F, G179S, V196A, E238D. A non-limiting example of a plasmid encoding these amino acid substitutions is pSE551.<br>
The present invention further provides a polynucleotide molecule comprising a nucleotide sequence that is otherwise the same as the Streptomyces avermitilis aveC allele, the S. avermitilis AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604) or the nucleotide sequence of the aveC ORF of S. avermitilis as presented in FIGURE 1 (SEQ ID N0:1), or a degenerate variant thereof, but which nucleotide sequence further comprises mutations encoding a combination of amino acid substitutions at amino acid residues corresponding to the amino acid positions of SEQ ID NO:2, such that cells of S. avermitilis strain ATCC 53692 in which the wild-type aveC allele has been inactivated and that express a polynucleotide molecule comprising the mutated nucleotide sequence are capable of producing a class 2:1 ratio of avermectins that is reduced compared to the ratio produced by cells of S. avermitilis strain ATCC 53692 that instead express only the wild-type aveC allele, wherein when the class 2:1 avermectins are cyclohexyl B2:cyclohexyl B1 avermectins, the ratio of class 2:1 avermectins is reduced to about 0.40:1 or less, and wherein the combination of amino acid substitutions comprises a combination selected from the group consisting of:<br>
(z)	D48E, S138T, A139T, G179S, E238D; and<br>
(aa)	Y28C, Q38R, D48E, L136P, G179S, E238D.<br>
Non-limiting examples of a plasmid encoding the amino acid substitutions of group (bf) are pSE556 and pSE569. A non-limiting example of a plasmid encoding the amino acid substitutions of group (bg) is pSE561.<br><br>
The present invention contemplates that any of the aforementioned amino acid substitutions can be accomplished by any modification to the nucleotide sequence of the aveC allele or a degenerate variant thereof that results in such substitutions. For -example, it is possible to effect most of the amino acid substitutions described herein by changing a native codon sequence or a degenerate variant thereof to any one of several alternative codons that encode the same amino acid substitution. The various possible sequences that can encode the aforementioned amino acid substitutions will be readily apparent to a person of skill in the art in view of the present disclosure and the known degeneracy of the genetic code. In a non-limiting embodiment for each particular combination recited above, the amino acid substitutions are achieved by the non-silent nucleotide changes set forth in FIGURE 6.<br>
As used herein, the phrase "the combination of amino acid substitutions comprises the combination of group ... ", and the like, means that the amino acid substitutions in the AveC gene product according to the present invention include at least those substitutions that are specifically recited, and may include other amino acid substitutions, or amino acid deletions, or amino acid additions, or some combination thereof, wherein the expression of the resulting AveC gene product in the S. avermitilis cell yields a desirable reduction in the ratio of B2:B1 avermectins.<br>
Mutations to the aveC allele or degenerate variant thereof can be carried out by any of a variety of known methods, including by use of error-prone PCR, or by cassette mutagenesis. For example, oligonucleotide-directed mutagenesis can be employed to alter the sequence of the aveC allele or ORF in a defined way such as, e.g., to introduce one or more restriction sites, or a termination codon, into specific regions within the aveC allele or ORF. Methods such as those described in U.S. Patent 5,605,793, U.S. Patent 5,830,721 and U.S. Patent 5,837,458, which involve random fragmentation, repeated cycles of mutagenesis, and nucleotide shuffling, can also be used to generate large libraries of polynucleotides having nucleotide sequences encoding aveC mutations.<br>
Targeted mutations can be useful, particularly where they serve to alter one or more conserved amino acid residues in the AveC gene product. For example, a comparison of the deduced amino acid sequence of the AveC gene product of S. avermitilis (SEQ ID N0:2) with AveC homolog gene products from S. griseochromogenes (SEQ ID NO:5) and S. hygroscopicus (SEQ ID NO:4), as described in U.S. Patent No. 6,248,579, indicates sites of significant conservation of amino acid residues between these species. Targeted mutagenesis that leads to a change in one or more of these conserved amino acid residues may be effective in producing novel mutant strains that exhibit desirable alterations in avermectin production.<br><br>
Random mutagenesis can also be useful, and can be carried out by exposing cells of S. avermitilis to ultraviolet radiation or x-rays, or to chemical mutagens such as N-methyl-N'-nitrosoguanidine, ethyl methane sulfonate, nitrous acid or nitrogen mustards. See, e.g., Ausubel, 1989, above, fora review of mutagenesis techniques.<br>
Once mutated polynucleotide molecules are generated, they are screened to determine whether they can modulate avermectin biosynthesis in S. avermiti/is. In a preferred embodiment, a polynucleotide molecule having a mutated nucleotide sequence is tested by complementing a strain of S. avermitilis in which the aveC gene has been inactivated to/give an aveC negative (aueC") background. In a non-limiting method, the mutated polynucleotide molecule is spliced into an expression plasmid in operative association with one or more regulatory elements, which plasmid also preferably comprises one or more drug resistance genes to allow for selection of transformed cells. This vector is then transformed into aveC' host cells using known techniques, and transformed cells are selected and cultured in appropriate fermentation media under conditions that permit or induce avermectin production, for example, by including appropriate starter subunits in the medium, and culturing under optimal conditions for avermectin production as known in the art. Fermentation products are then analyzed by HPLC to determine the ability of the mutated polynucleotide molecule to complement the host cell. Several plasmid vectors bearing mutated polynucleotide molecules capable of reducing the B2:B1 ratio of avermectins, including pSE188, pSE199, pSE231, pSE239, and pSE290 through pSE297, are exemplified in Section 8.3, below. Other examples of such plasmid vectors are recited in FIGURE 6.<br>
Any of the aforementioned methods of the present invention can be carried out using fermentation culture media preferably supplemented with cyclohexane carboxylic acid, although other appropriate fatty acid precursors, such as any one of the fatty acid precursors listed in TABLE 1, or methylthiolactic acid, can also used.<br>
Once a mutated polynucleotide molecule that modulates avermectin production in a desirable direction has been identified, the location of the mutation in the nucleotide sequence can be determined. For example, a poiynucleotide molecule having a nucleotide sequence encoding a mutated AveC gene product can be isolated by PCR and subjected to DNA sequence analysis using known methods. By comparing the DNA sequence of the mutated aveC allele to that of the wild-type aveC allele, the mutation(s) responsible for the alteration in avermectin production can be determined. For example, S. avermitilis AveC gene products comprising either single amino acid substitutions at any of residues 55 (S55F), 138 (S138T), 139 (A139T), or 230 (G230D), or double substitutions at positions 138 (S138T) and 139 (A139T or A139F), yielded changes in AveC gene product function such that the ratio of class 2:1 avermectins produced was altered (see Section 8, below), wherein the recited amino acid<br><br>
positions correspond to those presented in FIGURE 1 (SEQ ID NO:2). In addition, the following seven combinations of mutations have each been shown to effectively reduce the class 2:1 ratio of avermectins: (1) D48E/A89T; (2) S138T/A139T/G179S; (3) Q38P/L136P/E238D; (4) F99S/S138T/A139T/ G179S; (5) A139T7 M228T; (6) G111V/P289L; (7) A139T/K154E/Q298H. The present invention provides fifty-nine (59) additional combinations of mutations that are shown to reduce the cyclohexyl B2:cyclohexyl B1 ratio of avermectins, and these are presented in FIGURE 6 and recited in the appended claims.<br>
As used herein, the aforementioned designations, such as A139T, indicate the original amino acid residue by single letter designation, which in this example is alanine (A), at the indicated position, which in this example is position 139 (referring to SEQ ID NO:2) of the polypeptide, followed by the amino acid residue which replaces the original amino acid residue, which in this example is threonine (T).<br>
As used herein, where an amino acid residue encoded by an aveC allele in the S. avermitilis chromosome, or in a vector or isolated polynucleotide molecule of the present invention is referred to as "corresponding to" a particular amino acid residue of SEQ ID NO:2, or where an amino acid substitution is referred to as occurring at a particular position "corresponding to" that of a specific numbered amino acid residue of SEQ ID NO:2, this is intended to refer to the amino acid residue at the same relative location in the AveC gene product, which the skilled artisan can quickly determine by reference to the amino acid sequence presented herein as SEQ ID NO:2.<br>
As used herein, where specific mutations in the aveC allele encoding particular mutations are recited as base changes at specific nucleotide positions in the aveC allele "corresponding to" particular nucleotide positions as shown in SEQ ID NO:1, or where a nucleotide position in the aveC allele is otherwise referred to as "corresponding to" a particular nucleotide position in SEQ ID NO:1, this is intended to refer to the nucleotide at the same relative location in the aveC nucleotide sequence or a degenerate variant thereof, which the skilled artisan can quickly determine by reference to the nucleotide sequence presented herein as SEQ ID NO:1.<br>
As used herein to refer to ratios of cyclohexyl B2:cyclohexyl B1 avermectins, the term "about" refers to the specifically stated numerical value plus or minus 10% of that stated value. A polynucleotide molecule of the present invention may be "isolated", which means either that it is: (i) purified to the extent that it is substantially free of other polynucleotide molecules having different nucleotide sequences, or (ii) present in an environment in which it would not naturally occur, e.g., where an aveC allele from S. avermitilis, or a mutated version thereof, is present in a cell other than a cell of S. avermitilis, or (iii) present in a form in which it would not naturally occur, e.g., as a shorter piece of DNA, such as a restriction fragment<br><br>
digested out of a bacterial chromosome, comprising predominantly the aveC coding region or a mutated version thereof, with or without any associated regulatory sequences thereof, or as subsequently integrated into a heterologous piece of DMA, such as the chromosome of a bacterial cell (other than a cell of S. avermitilis) or the DNA of a vector such as a plasmid or phage, or integrated into the S. avermitilis chromosome at a locus other than that of the native aveC allele.<br>
The   present   invention   further   provides   a   recombinant   vector   comprising   a<br>
'	/<br>
polynucleotide molecule of the present invention. Such a recombinant vector can be used to<br>
target any of the polynucleotide molecules comprising mutated nucleotide sequences of the present invention to the site of the aveC allele of the S. avermitilis chromosome to either insert into or replace the aveC ORF or a portion thereof, e.g., by homologous recombination. According to the present invention, however, a polynucleotide molecule comprising a mutated nucleotide sequence of the present invention provided herewith can also function to modulate avermectin biosynthesis when inserted into the S. avermitilis chromosome at a site other than at the aveC allele, or when maintained episomally in S. avermitilis cells. Thus, the present invention further provides vectors comprising a polynucleotide molecule comprising a mutated nucleotide sequence of the present invention, which vectors can be used to insert the polynucleotide molecule at a site in the 5. avermitilis chromosome other than at the aveC gene, or to be maintained episomally.<br>
In a non-limiting embodiment, the vector is a gene replacement vector that can be used to insert a mutated aveC allele or degenerate variant thereof according to the present invention into cells of a strain of S. avermitilis, thereby generating novel strains of S. avermitilis, the cells of which can produce avermectins in a reduced class 2:1 ratio compared to cells of the same strain which instead express only the wild-type aveC allele. Such gene replacement vectors can be constructed using mutated polynucleotide molecules present in expression vectors provided herewith, such as those expression vectors exemplified in Section 8 below.<br>
The present invention further provides vectors that can be used to insert a mutated aveC allele or degenerate variant thereof into cells of a strain of S. avermitilis to generate novel strains of cells that produce altered amounts of avermectins compared to cells of the same strain which instead express only the wild-type aueC allele. In a preferred embodiment, the amount of avermectins produced by the cells is increased. In a specific though non-limiting embodiment, such a vector comprises a strong promoter as known in the art, such as, e.g., the strong constitutive ermE promoter from Saccharopoiyspora erythraea, that is situated upstream from, and in operative association with, the aveC ORF. Such vectors can be<br><br>
constructed using the mutated aveC allele of plasmid pSE189, and according to methods described in U.S. Patent No. 6,248,579,<br>
The present invention provides gene replacement vectors that are useful to inactivate the aveC gene in a wild-type strain of S. avermitilis. In a non-limiting embodiment, such gene replacement vectors can be constructed using the mutated polynucleotide molecule present in plasmid pSE180 (ATCC 209605), which is exemplified in Section 8.1, below (FIGURE 3). The<br>
present invention further provides gene replacement vectors that comprise a polynucleotide<br>
'	/<br>
molecule comprising or consisting of nucleotide sequences that naturally flank the aveC'gene<br>
in situ in the S. avermitilis chromosome, including, e.g., those flanking nucleotide sequences shown in FIGURE 1 (SEQ ID NO:1), which vectors can be used to delete the S. avermitilis aveC ORF.<br>
The present invention further provides a host cell comprising a polynucleotide molecule or recombinant vector of the present invention. The host cell can be any prokaryotic or eukaryotic cell capable of use as a host for the polynucleotide molecule or recombinant vector. In a preferred embodiment, the host cell is a bacterial cell. In a more preferred embodiment, the host cell is a Streptomyces cell. In a more preferred embodiment, the host cell is a cell of Streptomyces avermitilis.<br>
The present invention further provides a method for making a novel strain of Streptomyces avermitilis, comprising (i) mutating the aveC allele in a cell of a strain of S. avermitilis, which mutation results in a combination of amino acid substitutions in the AveC gene product, or (ii) introducing into a cell of a strain of S. avermitilis a mutated aveC allele or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions, wherein the combination of amino acid substitutions is selected from (a) through (be) listed above.<br>
The present invention further provides a method for making a novel strain of S. avermitilis, comprising (i) mutating the aveC allele in a cell of a strain of S. avermitilis, which mutation results in a combination of amino acid substitutions in the AveC gene product, or (ii) introducing into a cell of a strain of S. avermitilis a mutated aveC allele or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions, wherein cells comprising the mutated aveC allele or degenerate variant are capable of producing cyclohexyl B2:cyclohexyl B1 avermectins in a ratio of 0.35:1 or less. In a non-limiting embodiment thereof, the mutated aveC allele or degenerate variant thereof encodes an AveC gene product comprising a combination of amino acid substitutions selected from the group consisting of (a) through (be) listed above;<br>
In  a  preferred  embodiment thereof,  the  ratio  of cyclohexyl   B2:cyclohexyl   B1 avermectins is about 0.30:1 or less. In a non-limiting embodiment thereof, the mutated aveC<br><br>
allele or degenerate variant thereof encodes an AveC gene product comprising a combination<br>
of amino acid substitutions selected from the group consisting of (f) through (be) listed above.<br>
In a more preferred embodiment thereof, the ratio of cyclohexyl B2:cyclohexyl B1<br>
avermectins is about 0.25:1 or less.  In a non-limiting embodiment thereof, the mutated aveC<br>
allele or degenerate variant thereof encodes an AveC gene product comprising a combination<br>
of amino acid substitutions selected from the group consisting of (w) through (be) listed above.<br>
fn a more preferred embodiment thereof, the ratio of cyclohexyl B2:cyclohexyl B1<br>
avermectins is about 0.20:1 or less.  In a non-limiting embodiment thereof, the mutated aveC<br>
allele or degenerate variant thereof encodes an AveC gene product comprising a combination<br>
of amino acid substitutions selected from the group consisting of (ao) through (be) listed<br>
above.<br>
The present invention further provides a method for making a novel strain of Streptomyces avermitilis, comprising (i) mutating the aveC allele in a cell of a strain of S. avermitilis, which mutation results in a combination of amino acid substitutions in the AveC gene product, or (ii) introducing into a cell of a strain of S. avermitilis a mutated aveC allele or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions, wherein the combination of amino acid substitutions is selected from the group consisting of (bf) and (bg). In a preferred embodiment thereof, cells of S. avermitilis comprising such a mutated aveC allele or degenerate variant are capable of producing cyclohexyl B2:cyclohexyl B1 avermectins in a ratio of about 0.40:1 or less.<br>
By so mutating the aveC allele, or by so introducing a mutated aveC allele or degenerate variant thereof, according to the above-recited steps, a new strain of S avermitilis is made.<br>
The present invention further provides a cell of a Streptomyces species that comprises a mutated aveC allele or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions selected from (a) through (be) listed above. In a preferred embodiment thereof, the species of Streptomyces is S. avermitilis.<br>
The present invention further provides a cell of S. avermitilis capable of producing cyclohexyl B2:cyclohexyl B1 avermectins in a ratio of 0.35:1 or less. In a non-limiting embodiment thereof, the cell comprises a mutated aveC allele or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions selected from the group consisting of (a) through (be) listed above.<br>
In a preferred embodiment thereof, the ratio of cyclohexyl B2:cyclohexyl B1 avermectins is about 0.30:1 or less. In a non-limiting embodiment thereof, the cell comprises a mutated aveG allele or degenerate variant thereof encoding an AveC gene product<br><br>
comprising a combination of amino acid substitutions selected from the group consisting of (f) through (be) listed above.<br>
In a more preferred embodiment thereof, the ratio of cyclohexyl B2:cyclohexyl B1 avermectins is about 0.25:1 or less. In a non-limiting embodiment thereof, the cell comprises a mutated aveC allele or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions selected from the group consisting of (w) through (be) listed above.<br>
In a more preferred embodiment thereof, the ratio of cyclohexyl B2:cyclohexyl B1 avermectins is about 0.20:1 or less. In a non-limiting embodiment thereof, the cell comprises a mutated aveC allele or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions selected from the group consisting of (ao) through (be) listed above.<br>
The present invention further provides a cell of a Streptomyces species, comprising a mutated aveC allele or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions selected from the group consisting of (bf) and (bg) listed above. In a preferred embodiment thereof, the species of Streptomyces is S. avermitilis. In a more preferred embodiment thereof, the cell is a cell of S. avermitilis capable of producing cyclohexyl B2:cyclohexyl B1 avermectins in a ratio of about 0.40:1 or less.<br>
Although any of the recited mutations can be present in cells of the present invention on an extrachromosomal element such as a plasmid, it is preferred that such mutations are present in an aveC coding sequence integrated into the S. avermitilis chromosome, and preferably, though not necessarily, at the site of the native aveC allele.<br>
Such novel strains of cells are useful in the large-scale production of commercially desirable avermectins such as doramectin.<br>
The present invention further provides a process for producing avermectins, comprising culturing the S. avermitilis cells of the present invention in culture media under conditions that permit or induce the production of avermectins therefrom, and recovering said avermectins from the culture. In a preferred embodiment, the cells used in the process produce cyclohexyl B2:cyclohexyl B1 avermectins in a ratio of 0.35:1 or less, more preferably in a ratio of about 0.30:1 or less, more preferably in a ratio of about 0.25:1 or less, and more preferably in a ratio of about 0.20:1 or less.<br>
In a preferred embodiment thereof, cells producing cyclohexyl B2:cyclohexyl B1 avermectins in a ratio of 0.35:1 or less comprise a mutated aveC allele or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions selected from the group consisting of (a) through (be) listed above.<br><br>
In a further preferred embodiment thereof, cells producing cyclohexyl B2:cyclohexyl B1 avermectins in a ratio of about 0.30:1 or less comprise a mutated aveC allele or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions selected from the group consisting of (f) through (be) listed above.<br>
In a further preferred embodiment thereof, cells producing cyclohexyl B2:cyclohexyl B1 avermectins in a ratio of about 0.25:1 or less comprise a mutated aveC allele or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions selected from the group consisting of (w) through (be) listed above.<br>
In a further preferred embodiment thereof, cells producing cyclohexyl B2:cyclohexyl B1 avermectins in a ratio of about 0.20:1 or less comprise a mutated aveC allele or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions selected from the group consisting of (ao) through (be) listed above.<br>
In another embodiment, the cells produce cyclohexyl B2:cyclohexyl B1 avermectins in a ratio of about 0.40:1 or less and comprise a mutated aveC allele or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions selected from the group consisting of (bf) and (bg) listed above.<br>
The process of the invention provides increased efficiency in the production of commercially valuable avermectins such as doramectin.<br>
The present invention further provides a composition of cyclohexyl B2:cyclohexyl B1 avermectins produced by cells of Streptomyces avermitilis, comprising the cyclohexyl B2:cyclohexyl B1 avermectins present in a culture medium in which the cells have been cultured, wherein the ratio of the cyclohexyl B2:cyclohexyl B1 avermectins present in the culture medium is 0.35:1 or less, preferably about 0.30:1 or less, more preferably about 0.25:1 or less, and more preferably about 0.20:1 or less. In a particular embodiment, the composition of cyclohexyl B2:cyclohexyl B1 avermectins is produced by cells of a strain of S. avermitilis that express a mutated ai/eC allele or degenerate variant thereof which encodes a gene product that results in the reduction in the ratio of cyclohexyl B2:cyclohexyl B1 avermectins produced by the cells compared to cells of the same strain of S. avermitilis that do not express the mutated aveC allele but instead express only the wild-type aveC allele.<br>
In a preferred embodiment thereof, where the composition is cyclohexyl B2:cyclohexyl B1 avermectins in a ratio of 0.35:1 or less, the composition is produced by cells comprising a mutated aveC allele or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions selected from the group consisting of (a) through (be) listed above.<br>
In a further preferred embodiment thereof, where the composition is cyclohexyl B2:cyclohexyl B1 avermectins in a ratio of about 0.30:1 or less, the composition is produced<br><br>
by cells comprising a mutated aveC allele or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions selected from the group consisting of (f) through (be) listed above.<br>
In a further preferred embodiment thereof, where the composition is cyclohexyl B2:cyclohexyl B1 avermectins in a ratio of about 0.25:1 or less, the composition is produced by cells comprising a mutated aveC allele or degenerate variant thereof encoding an AveC<br>
gene product comprising a combination of amino acid substitutions selected from the group<br><br>
consisting of (w) through (be) listed above.<br>
In a further preferred embodiment thereof, where the composition is cyclohexyl B2:cyclohexyl B1 avermectins in a ratio of about 0.20:1 or less, the composition is produced by cells comprising a mutated aveC allele or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions selected from the group consisting of (ao) through (be) listed above.<br>
The present invention further provides a composition of cyclohexyl B2:cyclohexyl B1 avermectins produced by cells of Streptomyces avermitilis, comprising the cyclohexyl B2:cyclohexyl B1 avermectins present in a culture medium in which the cells have been cultured, wherein the ratio of the cyclohexyl B2:cyclohexyl B1 avermectins present in the culture medium is about 0.40:1 or less, and which is produced by cells comprising a mutated aveC allele or degenerate variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions selected from the group consisting of (bf) and (bg) listed above.<br>
Although it is preferred that the novel avermectin composition is present in a culture medium in which the cells have been cultured, e.g., in partially or totally exhausted fermentation culture fluid, the avermectin composition may alternatively be partially or substantially purified from the culture fluid by known biochemical techniques of purification, such as by ammonium sulfate precipitation, dialysis, size fractionation, ion exchange chromatography, HPLC, etc.<br>
In addition to making novel strains of S. avermitilis comprising cells that are capable of producing reduced ratios of cyclohexyl B2:cyclohexyl B1 as described above, the present invention contemplates that additional mutations can be incorporated into cells of S. avermitilis to further improve characteristics of avermectin production. In a non-limiting embodiment, cells of the present invention can further comprise modifications to increase the production level of avermectins. In one embodiment, such cells can be prepared by (i) mutating the aveC allele in a cell of S. avermitilis, or (ii) introducing a mutated aveC allele or degenerate variant thereof into cells of a strain of S. avermitilis, wherein the expression of the mutated allele results in an increase in the amount of avermectins produced by cells of a strain of S.<br><br>
avermitilis expressing the mutated aveC allele compared to cells of the same strain that instead express only a single wild-type aveC allele, and selecting transformed cells that produce avermectins in an increased amount compared to the amount of avermectins produced by cells of the strain that instead express only the single wild-type aueC allele. For example, the aveC allele can be modified so that it comprises a strong promoter, such as the strong constitutive ermE promoter from Saccharopolyspora erythraea, inserted upstream from and in operative association with the aveC ORF. In another embodiment, one or more mutations can be introduced into the aveR1 and/or aveR2 genes of S. avermitilis, thereby increasing the level of avermectin production as described in U.S. Patent No. 6,197,591 to Stutzman-Engwal! et al., issued March 6, 2001.<br>
5.4. Uses Of Avermectins<br>
Avermectins are highly active antiparasitic agents having particular utility as anthelmintics, ectoparasiticides, insecticides and acaricides. Avermectin compounds produced according to the methods of the present invention are useful for any of these purposes. For example, avermectin compounds produced according to the present invention are useful to treat various diseases or conditions in humans, particularly where those diseases or conditions are caused by parasitic infections, as known in the art. See, e.g., Ikeda and Omura, 1997, Chem. Rev. 97(7):2591-2609. More particularly, avermectin compounds produced according to the present invention are effective in treating a variety of diseases or conditions caused by endoparasites, such as parasitic nematodes, which can infect humans, domestic animals, swine, sheep, poultry, horses or cattle.<br>
More specifically, avermectin compounds produced according to the present invention are effective against nematodes that infect humans, as well as those that infect various species of animals. Such nematodes include gastrointestinal parasites such as Ancylostoma, Necator, Ascaris, Strongyloides, Trichinella, Capillaria, Trichuris, Enterobius, Dirofilaria, and parasites that are found in the blood or other tissues or organs, such as filarial worms and the extract intestinal states of Strongyloides and Trichinella.<br>
The avermectin compounds produced according to the present invention are also useful in treating ectoparasitic infections including, e.g., arthropod infestations of mammals and birds, caused by ticks, mites, lice, fleas, blowflies, biting insects, or migrating dipterous larvae that can affect cattle and horses, among others.<br>
The avermectin compounds produced according to the present invention are also useful as insecticides against household pests such as, e.g., the cockroach, clothes moth, carpet beetle and the housefly among others, as well as insect pests of stored grain and of agricultural plants, which pests include spider mites, aphids, caterpillars, and orthopterans such as locusts, among others.<br><br>
Animals that can be treated with the avermectin compounds produced according to the present invention include sheep, cattle, horses, deer, goats, swine, birds including poultry, and dogs and cats.<br>
An avermectin compound produced according to the present invention is administered in a formulation appropriate to the specific intended use, the particular species of host animal being treated, and the parasite or insect involved. For use as a parasiticide, an avermectin<br>
compound produced according to the present invention can be administered orally in the form<br><br>
of a capsule, bolus, tablet or liquid drench or, alternatively, can be administered as a pour-on, or by injection, or as an implant. Such formulations are prepared in a conventional mannerin accordance with standard veterinary practice. Thus, capsules, boluses or tablets can be prepared by mixing the active ingredient with a suitable finely divided diluent or carrier additionally containing a disintegrating agent and/or binder such as starch, lactose, talc, magnesium stearate, etc. A drench formulation can be prepared by dispersing the active ingredient in an aqueous solution together with a dispersing or wetting agent, etc. Injectable formulations can be prepared in the form of a sterile solution, which can contain other substances such as, e.g., sufficient salts and/or glucose to make the solution isotonic with blood.<br>
Such formulations will vary with regard to the weight of active compound depending on the patient, or species of host animal to be treated, the severity and type of infection, and the body weight of the host. Generally, for oral administration a dose of active compound of from about 0.001 to 10 mg per kg of patient or animal body weight given as a single dose or in divided doses for a period of from 1 to 5 days will be satisfactory. However, there can be instances where higher or lower dosage ranges are indicated, as determined, e.g., by a physician or veterinarian, as based on clinical symptoms.<br>
As an alternative, an avermectin compound produced according to the present invention can be administered in combination with animal feedstuff, and for this purpose a concentrated feed additive or premix can be prepared for mixing with the normal animal feed.<br>
For use as an insecticide, and for treating agricultural pests, an avermectin compound produced according to the present invention can be applied as a spray, dust, emulsion and the like in accordance with standard agricultural practice.<br>
6. EXAMPLE: FERMENTATION OF STREPTOMYCES AVERMITJLIS MV B2:B1 AVERMECTIN ANALYSIS<br>
Strains lacking both branched-chain 2-oxo acid dehydrogenase and 5-O-methyltransferase activities produce no avermectins if the fermentation medium is not supplemented with fatty acids. This example demonstrates that in such mutants a wide range<br><br>
of B2:B1 ratios of avermectins can be obtained when biosynthesis is initiated in the presence of different fatty acids.<br>
6.1. Materials And Methods<br>
Streptomyces avermitilis ATCC 53692 was stored at -70°C as a whole broth prepared in seed medium consisting of: Starch (Nadex, Laing National) - 20g; Pharmamedia (Trader's Protein, Memphis, TN) -15 g; Ardamine pH (Yeast Products Inc.) - 5 g; calcium carbonate - 1 g. Final volume was adjusted to 1 liter with tap water, pH was adjusted to 7.2, and the medium was autoclaved at 121°C for 25 min.<br>
Two ml of a thawed suspension of the above preparation was used to inoculate a flask containing 50 ml of the same medium. After 48 hrs incubation at 28°C on a rotary shaker at 180 rpm, 2 ml of the broth was used to inoculate a flask containing 50 ml of a production medium consisting of: Starch - 80 g; calcium carbonate - 7 g; Pharmamedia - 5 g; dipotassium hydrogen phosphate -1 g; magnesium sulfate -1 g; glutamic acid - 0.6 g; ferrous sulfate heptahydrate - 0.01 g; zinc sulfate - 0.001 g; manganous sulfate - 0.001 g. Final volume was adjusted to 1 liter with tap water, pH was adjusted to 7.2, and the medium was autoclaved at 121 °C for 25 min.<br>
Various carboxylic acid substrates (see TABLE 1) were dissolved in methanol and added to the fermentation broth 24 hrs after inoculation to give a final concentration of 0.2 g/liter. The fermentation broth was incubated for 14 days at 28°C, then the broth was centrifuged (2,500 rpm for 2 min) and the supernatant discarded. The mycelial pellet was extracted with acetone (15 ml), then with dichloromethane (30 ml), and the organic phase separated, filtered, then evaporated to dryness. The residue was taken up in methanol (1 ml) and analyzed by HPLC with a Hewlett-Packard 1090A liquid chromatograph equipped with a scanning diode-array detector set at 240 nm. The column used was a Beckman Ultrasphere C-18, 5 jam, 4.6 mm x 25 cm column maintained at 40°C. Twenty-five j.tl of the. above methanol solution was injected onto the column. Elution was performed with a linear gradient of methanol-water from 80:20 to 95:5 over 40 min at 0.85/ml min. Two standard concentrations of cyclohexyl B1 were used to calibrate the detector response, and the area under the curves for B2 and B1 avermectins was measured.<br>
6.2. Results<br>
The HPLC retention times observed for the B2 and B1 avermectins, and the 2:1 ratios, are shown in TABLE 1.<br><br>
TABLE 1(Table Remove) The data presented in TABLE 1 demonstrates an extremely wide range of B2:B1 avermectin product ratios, indicating a considerable difference in the results of dehydrative conversion of class 2 compounds to class 1 compounds, depending on the nature of the fatty acid side chain starter unit supplied. This indicates that changes in B2:B1 ratios resulting from alterations to the AveC protein may be specific to particular substrates. Consequently, screening for mutants exhibiting changes in the B2:B1 ratio obtained with a particular substrate needs to be done in the presence of that substrate. The subsequent examples described below use Cyclohexanecarboxylic acid as the screening substrate. However, this substrate is used merely to exemplify the potential, and is not intended to limit the applicability, of the present invention.<br>
7. EXAMPLE: ISOLATION OF THE aveC GENE<br>
This example describes the isolation and characterization of a region of the Streptomyces avermitilis chromosome that encodes the AveC gene product. As demonstrated below, the aveC gene was identified as capable of modifying the ratio of cyclohexyl-B2 to cyclohexyl-B1 (B2:B1) avermectins produced.<br>
7.1. Materials And Methods 7.1.1. Growth Of Streptomyces For DMA Isolation<br>
The following method was followed for growing Streptomyces. Single colonies of S. avermitilis ATCC 31272 (single colony isolate #2) were isolated on 1/2 strength YPD-6 containing: Difco Yeast Extract - 5 g; Difco Bacto-peptone - 5 g; dextrose - 2.5 g; MOPS - 5 g; Difco Bacto agar - 15 g. Final volume was adjusted to 1 liter with dH20, pH was adjusted to 7.0, and the medium was autoclaved at 121°C for 25 min.<br>
The mycelia grown in the above medium were used to inoculate 10 ml of TSB medium (Difco Tryptic Soy Broth - 30 g, in 1 liter dH20, autoclaved at 121°C for 25 min) in a 25 mm x 150 mm tube which was maintained with shaking (300 rpm) at 28°C for 48-72 hrs.<br>
7.1.2. Chromosomal DNA Isolation From Streptomvces<br>
Aliquots (0.25 ml or 0.5 ml) of mycelia grown as described above were placed in 1.5 ml microcentrifuge tubes and the cells concentrated by centrifugation at 12,000 x g for 60 sec. The supernatant was discarded and the cells were resuspended in 0.25 ml TSE buffer (20 ml 1.5 M sucrose, 2.5 ml 1 M Tris-HCI, pH 8.0, 2.5 ml 1 M EDTA, pH 8.0, and 75 ml dH2O) containing 2 mg/ml lysozyme. The samples were incubated at 37°C for 20 min with shaking, loaded into an AutoGen 540™ automated nucleic acid isolation instrument (Integrated Separation Systems, Natick, MA), and genomic DNA isolated using Cycle 159 (equipment software) according to manufacturer's instructions.<br>
Alternatively, 5 ml of mycelia were placed in a 17 mm x 100 mm tube, the cells<br>
concentrated by centrifugation at 3,000 rpm for 5 min, and the supernatant removed.  Cells<br>
were resuspended in 1 ml TSE buffer, concentrated by centrifugation at 3,000 rpm for 5 min,<br>
and the supernatant removed. Cells were resuspended in 1 ml TSE buffer containing 2 mg/ml<br>
lysozyme, and incubated at 37°C with shaking for 30-60 min.  After incubation, 0.5 ml 10%<br>
sodium dodecyi sulfate (SDS) was added and the cells incubated at 37°C until lysis' was<br>
complete. The lysate was incubated at 65°C for 10 min, cooled to rm temp, split into two 1.5<br>
ml Eppendorf tubes, and extracted 1x with 0.5 ml phenol/chloroform (50% phenol previously<br>
equilibrated with 0.5 M Tris, pH 8.0; 50% chloroform). The aqueous phase was removed and<br>
extracted 2 to 5x with chloroform:isoamyl alcohol (24:1). The DNA was precipitated by adding<br>
1/10 volume 3M sodium acetate, pH 4.8, incubating the mixture on ice for 10 min, centrifuging<br>
the mixture at 15,000 rpm at 5°C for 10 min, and removing the supernatant to a clean tube to<br>
which 1 volume of isopropanol was added.   The supernatant plus isopropanol mixture was<br>
then incubated on ice for 20 min, centrifuged at 15,000 rpm for 20 min at 5°C, the supernatant<br>
removed, and the DNA pellet washed 1x with 70% ethanol. After the pellet was dry, the DNA<br>
was resuspended in TE buffer (10 mM Tris, 1 mM EDTA, pH 8.0).<br>
7.1.3. Plasmid DNA Isolation From Streptomyces<br>
An aliquot (1.0 ml) of mycelia was placed in 1.5 ml microcentrifuge tubes and the cells concentrated by centrifugation at 12,000 x g for 60 sec. The supernatant was discarded, the cells were resuspended in 1.0 ml 10.3% sucrose and concentrated by centrifugation at 12,000 x g for 60 sec, and the supernatant discarded. The cells were then resuspended in 0.25 ml TSE buffer containing 2 mg/ml lysozyme, and incubated at 37°C for 20 min with shaking and<br>
loaded into the AutoGen 540™ automated nucleic acid isolation instrument.   Plasmid DNA<br>
/<br>
was isolated using Cycle 106 (equipment software) according to manufacturer's instructions.<br>
Alternatively, 1.5 ml of mycelia were placed in 1.5 ml microcentrifuge tubes and the<br>
cells concentrated by centrifugation at 12,000 x g for 60 sec. The supernatant was discarded,<br>
the cells were resuspended in 1.0 ml 10.3% sucrose and concentrated by centrifugation at<br>
12,000 x g for 60 sec, and the supernatant discarded. The cells were resuspended in 0.5 ml<br>
TSE buffer containing 2 mg/ml lysozyme, and incubated at 37°C for 15-30 min.    After<br>
incubation, 0.25 ml alkaline SDS (0.3N NaOH, 2% SDS) was added and the cells incubated at<br>
55°C for 15-30 min or until the solution was clear.  Sodium acetate (0.1 ml, 3M, pH 4.8) was<br>
added to the DNA solution, which was then incubated on ice for 10 min.  The DNA samples<br>
were centrifuged at 14,000 rpm for 10 min at 5°C. The supernatant was removed to a clean<br>
tube, and 0.2 ml phenolrchloroform (50% phenol:50% chloroform) was added and gently<br>
mixed.   The DNA solution was centrifuged at 14,000 rpm for 10 rnin at 5°C and the upper<br>
layer removed to a clean Eppendorf tube.  Isopropanol (0.75 ml) was added, and the solution<br>
was gently mixed and then incubated at rm temp for 20 min.    The DNA solution was<br>
centrifuged at 14,000 rpm for 15 min at 5°C, the supernatant removed, and the DNA pellet<br>
was washed with 70% ethanol, dried, and resuspended in TE buffer.<br>
7.1.4. Plasmid DNA Isolation From E. coli<br>
A single transformed E. coli colony was inoculated into 5 ml Luria-Bertani (LB) medium (Bacto-Tryptone -10 g, Bacto-yeast extract - 5 g, and NaCI - 10 g in 1 liter dH2O, pH 7.0, autoclaved at 121 °C for 25 min, and supplemented with 100 |.tg/ml ampicillin). The culture was incubated overnight, and a 1 ml aliquot placed in a 1.5 ml microcentrifuge tube. The culture samples were loaded into the AutoGen 540™ automated nucleic acid isolation instrument and plasmid DNA was isolated using Cycle 3 (equipment software) according to manufacturer's instructions.<br>
7.1.5.    Preparation And Transformation Of S. avermitilis Protoplasts<br>
Single colonies of S. avermitilis were isolated on 1/2 strength YPD-6.   The mycelia were used to inoculate 10 ml of TSB medium in a 25 mm x 150 mm tube, which was then<br>
incubated with shaking (300 rpm) at 28°C for 48 hrs. One ml of mycelia was used to inoculate 50 ml YEME medium. YEME medium contains per liter: Difco Yeast Extract - 3 g; Difco Bacto-peptone - 5 g; Difco Malt Extract - 3 g; Sucrose - 300 g. After autoclaving at 121 °C for 25 min, the following were added: 2.5 M MgCI2' 6H2O (separately autoclaved at 121°C for 25 min) - 2 ml; and glycine (20%) (filter-sterilized)- 25 ml.<br>
The mycelia were grown at 30°C for 48-72 hrs and harvested by centrifugation in a 50 ml centrifuge tube (Falcon) at 3,000 rpm for 20 min. The supernatant was discarded and the mycelia were resuspended in P buffer, which contains: sucrose - 205 g; K2SO4 - 0.25 g; MgCI2 ' 6H20 - 2.02 g; H2O - 600 ml; K2PO4 (0.5%) - 10 ml; trace element solution" - 20 ml; CaCI2 ' 2H20 (3.68%) - 100 ml; and MES buffer (1.0 M, pH 6.5) - 10 ml. (Trace element solution contains per liter: ZnCI2 - 40 mg; FeCI3 ' 6H20 - 200 mg; CuCI2 ' 2H20 - 10 mg; MnCI2' 4H20 -10 mg; Na2B4O7' 10H20 -10 mg; (NH4)6 Mo7O24 ' 4H20 - 10 mg). The pH was adjusted to 6.5, final volume was adjusted to 1 liter, and the medium was filtered hot through a 0.45 micron filter.<br>
The mycelia were pelleted at 3,000 rpm for 20 min, the supernatant was discarded, and the mycelia were resuspended in 20 ml P buffer containing 2 mg/ml lysozyme. The mycelia were incubated at 35°C for 15 min with shaking, and checked microscopically to determine extent of protoplast formation. When protoplast formation was complete, the protoplasts were centrifuged at 8,000 rpm for 10 min. The supernatant was removed and the protoplasts were resuspended in 10 ml P buffer. The protoplasts were centrifuged at 8,000 rpm for 10 min, the supernatant was removed, the protoplasts were resuspended in 2 ml P buffer, and approximately 1 x 109 protoplasts were distributed to 2.0 ml cryogenic vials (Nalgene).<br>
A vial containing 1 x 109 protoplasts was centrifuged at 8,000 rpm for 10 min, the supernatant was removed, and the protoplasts were resuspended in 0.1 ml P buffer. Two to 5 (.ig of transforming DNA were added to the protoplasts, immediately followed by the addition of 0.5 ml working T buffer. T buffer base contains: PEG-1000 (Sigma) - 25'g; sucrose - 2.5 g; H2O - 83 ml. The pH was adjusted to 8.8 with 1 N NaOH (filter sterilized), and the T buffer base was filter-sterilized and stored at 4°C. Working T buffer, made the same day used, was T buffer base - 8.3 ml; K2PO4 (4 mM) - 1.0 ml; CaCI2 • 2H20 (5 M) - 0.2 ml; and TES (1 M, pH 8) - 0.5 ml. Each component of the working T buffer was individually filter-sterilized.<br>
Within 20 sec of adding T buffer to the protoplasts, 1.0 ml P buffer was also added and the protoplasts were centrifuged at 8,000 rpm for 10 min. The supernatant was discarded and the protoplasts were resuspended in 0.1 ml P buffer. The protoplasts were then plated on RM14 media, which contains: sucrose - 205 g; K2SO4 - 0.25 g; MgCI2 ' 6H20 - 10.12 g;<br>
glucose -10 g; Difco Casamino Acids - 0.1 g; Difco Yeast Extract - 5 g; Difco Oatmeal Agar - 3 g; Difco Bacto Agar - 22 g; dH2O - 800 ml. The solution was autoclaved at 121°C for 25 min. After autoclaving, sterile stocks of the following were added: K2PO4 (0.5%) - 10 mi; CaCI2 ' 2H20 (5 M) - 5 ml; L-proline (20%) - 15 ml; MES buffer (1.0 M, pH 6.5) - 10 ml; trace element solution (same as above) - 2 ml; cycloheximide stock (25 mg/ml) - 40 ml; and 1N NaOH - 2 ml. Twenty-five ml of RM14 medium were aliquoted per plate, and plates dried for 24 hr before use.<br>
The protoplasts were incubated in 95% humidity at 30°C for 20-24 hrs. To select thiostrepton resistant transformants, 1 ml of overlay buffer containing 125 ng per ml thiostrepton was spread evenly over the RM14 regeneration plates. Overlay buffer contains per 100 ml: sucrose -10.3 g; trace element solution (same as above) - 0.2 ml; and MES (1 M, pH 6.5) - 1 ml. The protoplasts were incubated in 95% humidity at 30°C for 7-14 days until thiostrepton resistant (Thior) colonies were visible.<br>
7.1.6. Transformation Of Streptomyces lividans Protoplasts S. lividans TK64 (provided by the John Innes Institute, Norwich, U.K) was used for transformations in some cases. Methods and compositions for growing, protoplasting, and transforming S. lividans are described in Hopwood et a/., 1985, Genetic Manipulation of Streptomyces, A Laboratory Manual. John Innes Foundation, Norwich, U.K., and performed as described therein. Plasmid DMA was isolated from S. lividans transformants as described in Section 7.1.3, above.<br>
7.1.7. Fermentation Analysis Of S. avermitilis Strains<br>
S. avermitilis mycelia grown on 1/2 strength YPD-6 for 4-7 days were inoculated into 1 x 6 inch tubes containing 8 ml of preform medium and two 5 mm glass beads. Preform medium contains: soluble starch (either thin boiled starch or KOSO, Japan Com Starch Co., Nagoya) - 20 g/L; Pharmamedia - 15 g/L; Ardamine pH - 5 g/L (Champlain Ind., Clifton, NJ); CaCO3 - 2 g/L; 2x bcfa ("bcfa" refers to branched chain fatty acids) containing a final concentration in the medium of 50 ppm 2-(+/-)-methyl butyric acid, 60 ppm isobutyric acid, and 20 ppm isovaleric acid. The pH was adjusted to 7.2, and the medium was autoclaved at 121 °C for 25 min.<br>
The tube was shaken at a 17° angle at 215 rpm at 29°C for 3 days. A 2-ml aliquot of the seed culture was used to inoculate a 300 ml Erlenmeyer flask containing 25 ml of production medium which contains: starch (either thin boiled starch or KOSO) - 160 g/L; Nutrisoy (Archer Daniels Midland, Decatur, IL) -10 g/L; Ardamine pH -10 g/L; K2HPO4 - 2 g/L; MgSO4.4H2O - 2 g/L; FeSO4.7H20 - 0.02 g/L; MnCI2 - 0.002 g/L; ZnSO4.7H2O - 0.002 g/L; CaC03 -14 g/L; 2x bcfa (as above); and cyclohexane carboxylic acid (CHC) (made up as a<br>
20% solution at pH 7.0) - 800 ppm. The pH was adjusted to 6.9, and the medium was autoclaved at 121°C for 25 min. (As explained above, starter units other than CMC can be utilized instead (see, e.g., Table 1)).<br>
After inoculation, the flask was incubated at 29°C for 12 days with shaking at 200 rpm. After incubation, a 2 ml sample was withdrawn from the flask, diluted with 8 ml of methanol, mixed, and the mixture centrifuged at 1,250 x g for 10 rnin to pellet debris. The supernatant was then assayed by HPLC using,a Beckman Ultrasphere ODS column (25 cm x 4.6 mm ID) with a flow rate of 0.75 ml/min and detection by absorbance at 240 nm. The mobile phase was 86/8.9/5.1 methanol/water/acetonitrile.<br>
7.1.8. Isolation Of S. avermitilis PKS Genes<br>
A cosmid library of S. avermitilis (ATCC 31272, SC-2) chromosomal DNA was prepared and hybridized with a ketosynthase (KS) probe made from a fragment of the Saccharopolyspora erythraea polyketide synthase (PKS) gene.  A detailed description of the preparation of cosmid libraries can be found in Sambrook et a/., 1989, above.   A detailed description of the preparation of Streptomyces chromosomal DNA libraries is presented in Hopwood et a/., 1985, above.   Cosmid clones containing ketosynthase-hybridizing regions were identified by hybridization to a 2.7 Kb A/c/el/Eco47lll fragment from pEX26 (kindly supplied by Dr. P. Leadlay, Cambridge, UK).   Approximately 5 ng of pEX26 were digested using A/ctel and Eco47lll.   The reaction mixture was loaded on a 0.8% SeaPlaque GTG agarose gel (FMC BioProducts, Rockland, ME).   The 2.7 Kb A/cfel/Eco47lll fragment was excised from the gel after electrophoresis and the DNA recovered from the gel using GELase™ from Epicentre Technologies using the Fast Protocol.   The 2.7 Kb A/del/Eco47lll fragment     was     labeled     with     [a-32P]dCTP     (deoxycytidine     5'-triphosphate,     tetra (triethylammonium) salt,  [alpha-32P]-)  (NEN-Dupont,   Boston,   MA)  using the  BRL  Nick Translation System (BRL Life Technologies, Inc., Gaithersburg, MD) following the supplier's instructions. A typical reaction was performed in 0.05 ml volume.  After addition of 5 (al Stop buffer, the labeled DNA was separated from unincorporated nucleotides using a G-25 Sephadex Quick Spin™ Column (Boehringer Mannheim) following supplier's instructions.<br>
Approximately 1,800 cosmid clones were screened by colony hybridization. Ten clones were identified that hybridized strongly to the Sacc. erythraea KS probe. E. coli colonies containing cosmid DNA were grown in LB liquid medium and cosmid DNA was isolated from each culture in the AutoGen 540™ automated nucleic acid isolation instrument using Cycle 3 (equipment software) according to manufacturer's instructions. Restriction endonuclease mapping and Southern blot hybridization analyses revealed that five of the clones contained overlapping chromosomal regions. An S. avermitilis genomic SamHl<br><br>
restriction  map of the five cosmids (;.e., pSE65, pSE66,  pSE67, pSE68, pSE69) was constructed by analysis of overlapping cosmids and hybridizations (FIGURE 4).<br>
7.1.9.    Identification Of DNA That Modulates Avermectin B2:B1 Ratios And Identification Of An aveC ORF<br>
The following methods were used to test subcloned fragments derived from the pSE66 cosmid clone for their ability to modulate avermectin B2:B1 ratios in AveC mutants. pSE66 (5 ng) was digested with Sad and Ba/nHI. The reaction mixture was loaded on a 0.8% SeaPlaque™ GTG agarose gel (FMC BioProducts), a 2.9 Kb Sacl/BamHI fragment was excised from the gel after electrophoresis, and the DNA was recovered from the gel using GELase™ (Epicentre Technologies) using the Fast Protocol. Approximately 5 ^ig of the shuttle vector pWHMS (Vara era/., 1989, J. Bacteriol. 171:5872-5881) was digested with Sacl and Ba/nHI. About 0.5 jag of the 2.9 Kb insert and 0.5 ).ig of digested pWHMS were mixed together and incubated overnight with 1 unit of ligase (New England Biolabs, Inc., Beverly, MA) at 15°C, in a total volume of 20 jil, according to supplier's instructions. After incubation, 5 Hi of the ligation mixture was incubated at 70°C for 10 min, cooled to rm temp, and used to transform competent E. coli DH5a cells (BRL) according to manufacturer's instructions. Plasmid DNA was isolated from ampicillin resistant transformants and the presence of the 2.9 Kb Sacl/BamHI insert was confirmed by restriction analysis. This plasmid was designated as pSE119.<br>
Protoplasts of S. avermitilis strain 1100-SC38 (Pfizer in-house strain) were prepared and transformed with pSE119 as described in Section 7.1.5 above. Strain 1100-SC38 is a mutant that produces significantly more of the avermectin cyclohexyl-B2 form compared to avermectin cyclohexyl-B1 form when supplemented with cyclohexane carboxylic acid (B2:B1 of about 30:1). pSE119 used to transform S. avermitilis protoplasts was isolated from either E. coli strain GM2163 (obtained from Dr. B. J. Bachmann, Curator, E. coli Genetic Stock Center, Yale University), E. coli strain DM1 (BRL), or S. lividans strain TK64. Thiostrepton resistant transformants of strain 1100-SC38 were isolated and analyzed by HPLC analysis of fermentation products. Transformants of S. avermitilis strain 1100-SC38 containing pSE119 produced an altered ratio of avermectin cyclohexyl-B2:cyclohexyl-B1 of about 3.7:1 (TABLE 2).<br>
Having established that pSE119 was able to modulate avermectin B2:B1 ratios in an AveC mutant, the insert DNA was sequenced. Approximately 10 (ig of pSE119 were isolated using 3 plasmid DNA isolation kit (Qiagen, Valencia, CA) following manufacturer's instructions, and sequenced using an ABI 373A Automated DNA Sequencer (Perkin Elmer, Foster City, CA). Sequence data was assembled and edited using Genetic Computer Group<br><br>
programs (GCG, Madison, Wl).   The DNA sequence and the aveC ORF are presented in FIGURE 1 (SEQIDNO:1).<br>
A new plasmid, designated as pSE118, was constructed as follows. Approximately 5 jig of pSE66 was digested with Sphl and BamHI. The reaction mixture was loaded on a 0.8% SeaPlaque GTG agarose gel (FMC BioProducts), a 2.8 Kb Sphl/BamHI fragment was excised from the gel after electrophoresis, and the DNA was recovered from the gel using GELase™ (Epicentre Technologies) using the Fast Protocol. Approximately 5 jag of the shuttle vector pWHMS was digested with Sphl and BamHI. About 0.5 jig of the 2.8 Kb insert and 0.5 pg of digested pWHMS were mixed together and incubated overnight with 1 unit of iigase (New England Biolabs) at 15°C in a total volume of 20 |il according to supplier's instructions. After incubation, 5 (.il of the ligation mixture was incubated at 70°C for 10 min, cooled to rm temp, and used to transform competent E. coli DH5cc cells according to manufacturer's instructions. Plasmid DNA was isolated from ampiciilin resistant transformants, and the presence of the 2.8 Kb Sph\IBamH\ insert was confirmed by restriction analysis. This plasmid was designated as pSE118. The insert DNA in pSE118 and pSE119 overlap by approximately 838 nucleotides (FIGURE 4).<br>
Protoplasts of S. avermitilis strain 1100-SC38 were transformed with pSE118 as above. Thiostrepton resistant transformants of strain 1100-SC38 were isolated and analyzed by HPLC analysis of fermentation products. Transformants of S. avermitilis strain 1100-SC38 containing pSE118 were not altered in the ratios of avermectin cyclohexyl-B2: avermectin cyclohexyl-B1 compared to strain 1100-SC38 (TABLE 2).<br>
7.1.10.   PCR Amplification Of The aveC Gene From S. avermitilis Chromosomal DNA<br>
A -1.2 Kb fragment containing the aveC ORF was isolated from S. avermitilis chromosomal DNA by PCR amplification using primers designed on the basis of the aveC nucleotide sequence obtained above. The PCR primers were supplied by Genosys Biotechnologies, Inc. (Texas). The rightward primer was: 5'-TCACGAAACCGGACACAC-3' (SEQ ID NO:6); and the leftward primer was: 5'- CATGATCGCTGAACCGAG-3' (SEQ ID NO:7). The PCR reaction was carried out with Deep Vent™ polymerase (New England Biolabs) in buffer provided by the manufacturer, and in the presence of 300 jiM dNTP, 10% glycerol, 200 pmol of each primer, 0.1 jig template, and 2.5 units enzyme in a final volume of 100 nl, using a Perkin-Elmer Cetus thermal cycler. The thermal profile of the first cycle was 95°C for 5 min (denaturation step), 60°C for 2 min (annealing step), and 72°C for 2 min (extension step). The subsequent 24 cycles had a similar thermal profile except that the denaturation step was shortened to 45 sec and the annealing step was shortened to 1 min.<br><br>
The PCR product was electrophoresed in a 1 % agarose gei and a single DNA band of -1.2 Kb was detected. This DNA was purified from the gel, and ligated with 25 ng of linearized, blunt pCR-Blunt vector (Invitrogen) in a 1:10 molar vector-to-insert ratio following manufacturer's instructions. The ligation mixture was used to transform One Shot™ Competent E. coli cells (Invitrogen) following manufacturer's instructions. Plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the -1.2 Kb insert was<br>
confirmed by restriction analysis. This plasmid was designated as pSE179.<br>
/ The insert DNA from pSE179 was isolated by digestion with BamHI/Xfaal, separated<br>
by electrophoresis, purified from the gel, and ligated with shuttle vector pWHMS, which had also been digested with BamHI/Xdal, in a total DNA concentration of 1 (ig in a 1:5 molar vector-to-insert ratio. The ligation mixture was used to transform competent E. coli DH5a cells according to manufacturer's instructions. Plasmid DNA was isolated from ampicillin resistant transformants and the presence of the -1.2 Kb insert was confirmed by restriction analysis. This plasmid, which was designated as pSE186 (FIGURE 2, ATCC 209604), was transformed into E. coli DM1, and plasmid DNA was isolated from ampicillin resistant transformants.<br>
7.2. Results<br>
A 2.9 Kb Sacl/SamHI fragment from pSE119 was identified that, when transformed into S. avermitilis strain 1100-SC38, significantly altered the ratio of B2:B1 avermectin production. S. avermitilis strain 1100-SC38 normally has a B2:B1 ratio of about 30:1, but when transformed with a vector comprising the 2.9 Kb Sacl/SamHI fragment, the ratio of B2:B1 avermectin decreased to about 3.7:1. Post-fermentation analysis of transformant cultures verified the presence of the transforming DNA.<br>
The 2.9 Kb pSE119 fragment was sequenced and a -0.9 Kb ORF was identified (FIGURE 1) (SEQ ID NO:1), which encompasses a Pst\/Sph\ fragment that had previously been mutated elsewhere to produce B2 products only (Ikeda et a/., 1995, above). A comparison of this ORF, or its corresponding deduced polypeptide, against known databases (GenEMBL, SWISS-PROT) did not show any strong homology with known DNA or protein sequences.<br>
TABLE 2 presents the fermentation analysis of S. avermitilis strain 1100-SC38 transformed with various plasmids.<br><br>
TABLE2(Table Remove)<br>
8. EXAMPLE: CONSTRUCTION OF S. AVERMITILIS AveC MUTANTS<br>
This example describes the construction of several different S. avermitilis AveC mutants using the compositions and methods described above. A general description of techniques for introducing mutations into a gene in Streptomyces is described by Kieser and Hopwood, 1991, Meth. Enzym. 204:430-458. A more detailed description is provided by Anzai et a/., 1988, J. Antibiot. XLI(2):226-233, and by Stutzman-Engwall et a/., 1992, J. Bacteriol. 174(1):144-154. These references are incorporated herein by reference in their entirety.<br>
8,1. Inactivation Of The S. avermitilis aveC Gene<br>
AveC mutants containing inactivated aveC genes were constructed using several methods, as detailed below.<br>
In the first method, a 640 bp Sph\/Pst\ fragment internal to the aveC gene in pSE119 (plasmid described in Section 7.1.9, above) was replaced with the ermE gene (for erythromycin resistance) from Sacc. erythraea. The ermE gene was isolated from plJ4026 (provided by the John Innes Institute, Norwich, U.K.; see also Bibb et a/., 1985, Gene 41:357-368) by restriction enzyme digestion with Bg/ll and EcoRI, followed by electrophoresis, and was purified from the gel. This -1.7 Kb fragment was ligated into pGEM7Zf (Promega) which had been digested with BamHl and EcoRI, and the ligation mixture transformed into competent E. coli DH5a cells following manufacturer's instructions. Plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the -1.7 Kb insert was confirmed by restriction analysis. This plasmid was designated as pSE27.<br>
pSE118 (described in Section 7.1.9, above) was digested with Sph\ and BamHl, the digest electrophoresed, and the -2.8 Kb Sph\/BamH\ insert purified from the gel. pSE119 was digested with Pstl and EcoRI, the digest electrophoresed, and the -1.5 Kb Psfl/£coRI insert<br><br>
purified from the gel. Shuttle vector pWHMS was digested with SamHI and EcoRI. pSE27 was digested with Pstl and Sphl, the digest eiectrophoresed, and the ~1.7 Kb Pst\/Sph\ insert purified from the gel. All four fragments (i.e., -2.8 Kb, ~1.5Kb, ~7.2Kb, -1.7 Kb) were ligated together in a 4-way ligation. The ligation mixture was transformed into competent E. coli DH5oc cells following manufacturer's instructions. Plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the correct insert was confirmed by restriction analysis. This plasmid was designated as pSE180 (FIGURE 3; ATCC 209605).<br>
pSE180 was transformed into S. lividans TK64 and transformed colonies identified by resistance to thiostrepton and erythromycin. pSE180 was isolated from S. lividans and used to transform S. avermitilis protoplasts. Four thiostrepton resistant S. avermitilis transformants were identified, and protoplasts were prepared and plated under non-selective conditions on RM14 media. After the protoplasts had regenerated, single colonies were screened for the presence of erythromycin resistance and the absence of thiostrepton resistance, indicating chromosomal integration of the inactivated aveC gene and loss of the free replicon. One Ermr Thios transformant was identified and designated as strain SE180-11. Total chromosomal DNA was isolated from strain SE180-11, digested with restriction enzymes SamHI, HindUl, Pst\, or Sph\, resolved by electrophoresis on a 0.8% agarose gel, transferred to nylon membranes, and hybridized to the ermE probe. These analyses showed that chromosomal integration of the ermE resistance gene, and concomitant deletion of the 640 bp Pst\ISph\ fragment had occurred by a double crossover event. HPLC analysis of fermentation products of strain SE180-11 showed that normal avermectins were no longer produced (FIGURE 5A).<br>
In a second method for inactivating the aveC gene, the 1.7 Kb ermE gene was removed from the chromosome of S. avermitilis strain SE180-11, leaving a 640 bp Pstt/Sphl deletion in the aveC gene. A gene replacement plasmid was constructed as follows: pSE180 was partially digested with Xba\ and an -11.4 Kb fragment purified from the gel. The -11.4 Kb band lacks the 1.7 Kb ermE resistance gene. The DNA was then ligated and transformed into E. co// DH5a cells. Plasmid DNA was isolated from ampicillin resistant transformants and the presence of the correct insert was confirmed by restriction analysis. This plasmid, which was designated as pSE184, was transformed into E. coli DM1, and plasmid DNA isolated from ampicillin resistant transformants. This plasmid was used to transform protoplasts of S. avermitilis strain SE180-11. Protoplasts were prepared from thiostrepton resistant transformants of strain SE180-11 and were plated as single colonies on RM14. After the protoplasts had regenerated, single colonies were screened for the absence of both erythromycin resistance and thiostrepton resistance, indicating chromosomal integration of the inactivated aveC gene and loss of the free replicon containing the ermE gene. One Erms Thio3 transformant was identified and designated as SE184-1-13. Fermentation analysis of<br><br>
SE184-1-13 showed that normal avermectins were not produced and that SE184-1-13 had the same fermentation profile as SE180-11.<br>
In a third method for inactivating the aveC gene, a frameshift was introduced into the chromosomal aveC gene by adding two G's after the C at nt position 471 using PCR, thereby creating a 6spE1 site. The presence of the engineered BspEI site was useful in detecting the gene replacement event. The PCR primers were designed to introduce a frameshift mutation into the aveC gene, and were supplied by Genosys Biotechnologies, Inc. The rightward primer was: 5'-GGTTCCGGATGCCGTTCTCG-3' (SEQ ID NO:8) and the leftward p'rimer was: 5'-AACTCCGGTCGACTCCCCTTC-3' (SEQ ID NO:9). The PCR conditions were as described in Section 7.1.10 above. The 666 bp PCR product was digested with Sph\ to give two fragments of 278 bp and 388 bp, respectively. The 388 bp fragment was purified from the gel.<br>
The gene replacement plasmid was constructed as follows:   shuttle vector pWHM3 was digested with EcoRI and SamHI. pSE119 was digested with SamHI and Sph\, the digest electrophoresed, and a -840 bp fragment was purified from the gel.   pSE119 was digested with EcoRI and Xmn\, the digest was resolved by electrophoresis, and a ~1.7 Kb fragment was purified from the gel.  All four fragments (i.e., -7.2 Kb, -840 bp, -1.7 Kb, and 388 bp) were ligated together in a 4-way ligation. The ligation mixture was transformed into competent E. co// DH5cx cells. Plasmid DNA was isolated from ampicillin resistant transformants and the presence of the correct insert was confirmed by restriction analysis and DNA sequence analysis.  This plasmid, which was designated as pSE185, was transformed into E. co// DM1 and plasmid DNA isolated from ampicillin resistant transformants. This plasmid was used to transform protoplasts of S. avermitilis strain 1100-SC38. Thiostrepton resistant transformants of strain 1100-SC38 were isolated and analyzed by HPLC analysis of fermentation products. pSE185 did not significantly alter the B2:B1 avermectin ratios when transformed into S. avermitilis strain 1100-SC38 (TABLE 2).<br>
pSE185 was used to transform protoplasts of S. avermitilis to generate a frameshift mutation in the chromosomal aveC gene. Protoplasts were prepared from thiostrepton resistant transformants and plated as single colonies on RM14. After the protoplasts had regenerated, single colonies were screened for the absence of thiostrepton resistance. Chromosomal DNA from thiostrepton sensitive colonies was isolated and screened by PCR for the presence of the frameshift mutation integrated into the chromosome. The PCR primers were designed based on the aveC nucleotide sequence, and were supplied by Genosys Biotechnologies, Inc. (Texas). The rightward PCR primer was: 5'-GCAAGGATACGGGGACTAC-3' (SEQ ID NO:10) and the leftward PCR primer was: 5'-GAACCGACCGCCTGATAC-3' (SEQ ID N0:11), and the PCR conditions were as described<br><br>
in Section 7.1.10 above. The PCR product obtained was 543 bp and, when digested with 8spE1, three fragments of 368 bp, 96 bp, and 79 bp were observed, indicating chromosomal integration of the inactivated aveC gene and loss of the free replicon.<br>
Fermentation analysis of S. avermitilis mutants containing the frameshift mutation in the aveC gene showed that normal avermectins were no longer produced, and that these mutants had the same fermentation HPLC profile as strains SE180-11 and SE184-1-13. One<br>
Thios transformant was identified and designated as strain SE185-5a.<br>
/ Additionally, a mutation in the aveC gene that changes nt position 520 from G 'to A,<br>
which results in changing the codon encoding a tryptophan (W) at position 116 to: a termination codon, was produced. An S. avermitilis strain with this mutation did not produce normal avermectins and had the same fermentation profile as strains SE180-11, SE184-1-13, andSE185-5a.<br>
Additionally, mutations in the aveC gene that change both: (i) nt position 970 from G to A, which changes the amino acid at position 266 from a glycine (G) to an aspartate (D), and (ii) nt position 996 from T to C, which changes the amino acid at position 275 from tyrosine (Y) to histidine (H), were produced. An S. avermitilis strain with these mutations (G266D/Y275H) did not produce normal avermectins and had the same fermentation profile as strains SE180-11, SE184-1-13, and SE185-5a.<br>
The S. avermitilis aveC inactivation mutant strains SE180-11, SE184-1-13, SE185-5a, and others provided herewith, provide screening tools to assess the impact of other mutations in the aveC gene. pSE186, which contains a wild-type copy of the aveC gene, was transformed into E. coli DM1, and plasmid DNA was isolated from ampicillin resistant transform ants. This pSE186 DNA was used to transform protoplasts of S. avermitilis strain SE180-11. Thiostrepton resistant transformants of strain SE180-11 were isolated, the presence of erythromycin resistance was determined, and Thior Ermr transformants .were analyzed by HPLC analysis of fermentation,products. The presence of the functional aveC gene in trans was able to restore normal avermectin production to strain SE180-11 (FIGURE 5B).<br>
8.2. Analysis Of Mutations In The aveC Gene That Alter Class B2:B1 Ratios<br>
As described above, S. avermitilis strain SE180-11 containing an inactive aveC gene was complemented by transformation with a plasmid containing a functional aveC gene (pSE186). Strain SE180-11 was also utilized as a host strain to characterize other mutations in the aveC gene, as described below.<br>
Chromosomal DNA was isolated from strain 1100-SC38, and used as a template for PCR amplification of the aveC gene. The 1.2 Kb ORF was isolated by PCR amplification<br><br>
using primers designed on the basis of the aveC nucieotide sequence. The rightward primer was SEQ ID NO:6 and the leftward primer was SEQ ID NO:7 (see Section 7.1.10, above). The PCR and subcloning conditions were as described in Section 7.1.10. DNA sequence analysis of the 1.2 Kb ORF shows a mutation in the aveC gene that changes nt position 337 from C to T, which changes the amino acid at position 55 from serine (S) to phenylalanine (F). The aveC gene containing the S55F mutation was subcloned into pWHMS to produce a plasmid which was designated as pSE187, and which was used to transform protoplasts of S. avermitilis strain SE180-11. Thiostrepton resistant transformants of strain SE180-11 were isolated, the presence of erythromycin resistance was determined, and Thior Ermr transformants were analyzed by HPLC analysis of fermentation products. The presence of the aveC gene encoding a change at amino acid residue 55 (S55F) was able to restore normal avermectin production to strain SE180-11 (FIGURE 5C); however, the cyclohexyl B2:cyclohexyl B1 ratio was about 26:1, as compared to strain SE180-11 transformed with pSE186, which had a ratio of B2:B1 of about 1.6:1 (TABLE 3), indicating that the single mutation (S55F) modulates the amount of cyclohexyl-B2 produced relative to cyclohexyl-B1.<br>
Another mutation in the aveC gene was identified that changes nt position 862 from G to A, which changes the amino acid at position 230 from glycine (G) to aspartate (D). An S. avermitilis strain having this mutation (G230D) produces avermectins at a B2:B1 ratio of about 30:1.<br>
8.3. Mutations That Reduce The B2:B1 Ratio<br>
Several mutations were constructed that reduce the amount of cyclohexyl-B2 produced relative to cyclohexyl-B1, as follows.<br>
A mutation in the aveC gene was identified that changes nt position 588 from G to A, which changes the amino acid at position 139 from alanine (A) to threonine (T). The aveC gene containing the A139T mutation was subcloned into pWHMS to produce a piasmid which was designated pSE188, and which was used to transform protoplasts of S. avermitilis strain SE180-11. Thiostrepton resistant transformants of strain SE180-11 were isolated, the presence of erythromycin resistance was determined, and Thior Ermr transformants were analyzed by HPLC analysis of fermentation products. The presence of the mutated aveC gene encoding a change at amino acid residue 139 (A139T) was able to restore avermectin production to strain SE180-11 (FIGURE 5D); however, the B2:B1 ratio was about 0.94:1, indicating that this mutation reduces the amount of cyclohexyl-B2 produced relative to cyclohexyl-B1. This result was unexpected because published results, as well as the results of mutations described above, have only demonstrated either inactivation of the aveC gene or increased production of the B2 form of avermectin relative to the B1 form (TABLE 3).<br><br>
Because the A139T mutation altered the B2:B1 ratios in the more favorable B1 direction, a mutation was constructed that encoded a threonine instead of a serine at amino acid position 138. Thus, pSE186 was digested with EcoRI and cloned into pGEMSZf (Promega) which had been digested with EcoRI. This plasmid, which was designated as pSE186a, was digested with Apa\ and.Kpnl, the DNA fragments separated on an agarose gel, and two fragments of -3.8 Kb and -0.4 Kb were purified from the gel. The -1.2 Kb insert<br>
DNA from pSE186 was used as a PCR template to introduce a single base change at nt<br>
/ position 585. The PCR primers were designed to introduce a mutation at nt position 585, and<br>
were supplied by Genosys Biotechnologies, Inc. (Texas). The rightward PCR primer was: 5'-GGGGGCGGGCCCGGGTGCGGAGGCGGAAATGCCCCTGGCGACG-3' (SEQ ID NO:12); and the leftward PCR primer was: 5'-GGAACCGACCGCCTGATACA-3' (SEQ ID NO:13). The PCR reaction was carried out using an Advantage GC genomic PCR kit (Clonetech Laboratories, Palo Alto, CA) in buffer provided by the manufacturer in the presence of 200 \iM dNTPs, 200 pmol of each primer, 50 ng template DNA, 1.0 M GC-Melt and 1 unit KlenTaq Polymerase Mix in a final volume of 50 \i\. The thermal profile of the first cycle was 94°C for 1 min; followed by 25 cycles of 94°C for 30 sec and 68°C for 2 min; and 1 cycle at 68°C for 3 min. A PCR product of 295 bp was digested with Apa\ and Kpnl to release a 254 bp fragment, which was resolved by electrophoresis and purified from the gel. All three fragments (-3.8 Kb, -0.4 Kb and 254 bp) were ligated together in a 3-way ligation. The ligation mixture was transformed into competent E. coli DH5a cells. Plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the correct insert was confirmed by restriction analysis. This plasmid was designated as pSE198.<br>
pSE198 was digested with EcoRI, cloned into pWHMS, which had been digested with EcoRI, and transformed into E. coli DH5a cells. Plasmid DNA was isolated from ampicillin resistant transformants and the presence of the correct insert was confirmed by restriction analysis and DNA sequence analysis. This plasmid DNA was transformed into E. coli DM1, plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the correct insert was confirmed by restriction analysis. This plasmid, which was designated as pSE199, was used to transform protoplasts of S. avermitilis strain SE180-11. Thiostrepton resistant transformants of strain SE180-11 were isolated, the presence of erythromycin resistance was determined, and Thior Ermr transformants were analyzed by HPLC analysis of fermentation products. The presence of the mutated aveC gene encoding a change at amino acid residue 138 (S138T) was able to restore normal avermectin production to strain SE180-11; however, the B2:B1 ratio was 0.88:1 indicating that this mutation reduces the amount of cyclohexyl-B2 produced relative to cyclohexyl-B1 (TABLE 3). This B2:B1 ratio is even lower<br><br>
than the 0.94:1 ratio observed with the A139T mutation produced by transformation of strain SE180-11 with pSE188, as described above.<br>
Another mutation was constructed to introduce a threonine at both amino acid positions 138 and 139. The -1.2 Kb insert DNA from pSE186 was used as a PCR template. The PCR primers were designed to introduce mutations at nt positions 585 and 588, and were supplied by Genosys Biotechnologies, Inc. (Texas). The rightward PCR primer was: 5'-GGGGGCGGGCCCGGGTGCGGAGGCGGAAATGCCGCTGGCGACGACC-3' (SEQ ID N0:14); and the leftward PCR primer was: 5'-GGAACATCACGGCATTCACC-3' (SEQ ID NO:15). The PCR reaction was performed using the conditions described immediately above in this Section. A PCR product of 449 bp was digested with Apa\ and Kpn\ to release a 254 bp fragment, which was resolved by electrophoresis and purified from the gel. pSE186a was digested with Apa\ and Kpn\, the DNA fragments separated on an agarose gel, and two fragments of -3.8 Kb and -0.4 Kb were purified from the gel. All three fragments (-3.8 Kb, -0.4 Kb and 254 bp) were ligated together in a 3-way ligation, and the ligation mixture was transformed into competent E. coli DH5a cells. Plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the correct insert was confirmed by restriction analysis. This plasmid was designated as pSE230.<br>
pSE230 was digested with EcoRI, cloned into pWHMS, which had been digested with EcoRI, and transformed into E. coli DH5a cells. Plasmid DNA was isolated from ampicillin resistant transformants and the presence of the correct insert was confirmed by restriction analysis and DNA sequence analysis. This plasmid DNA was transformed into E. coli DM1, plasmid DNA isolated from ampicillin resistant transformants, and the presence of the correct insert was confirmed by restriction analysis. This plasmid, which was designated as pSE231, was used to transform protoplasts of S. avermitilis strain SE180-11. Thiostrepton resistant transformants of SE180-11 were isolated, the presence of erythromycin resistance. was determined, and Thior Ermr transformants were analyzed by fermentation. The presence of the double mutated aveC gene, encoding S138T/A139T, was able to restore normal avermectin production to strain SE180-11; however, the B2:B1 ratio was 0.84:1 showing that this mutation further reduces the amount of cyclohexyl-B2 produced relative to cyclohexyl-B1 (TABLE 3), over the reductions provided by transformation of strain SE180-11 with pSE188 or pSE199, as described above.<br>
Another mutation was constructed to further reduce the amount of cyciohexyl-B2 produced relative to cyclohexyl-B1. Because the S138T/A139T mutations altered the B2:B1 ratios in the more favorable B1 direction, a mutation was constructed to introduce a threonine at amino acid position 138 and a phenylalanine at amino acid position 139. The -1.2 Kb insert DNA from pSE186 was used as a PCR template. The PCR primers were designed to<br><br>
introduce mutations at nt positions 585 (changing a T to A), 588 (changing a G to T), and 589 (changing a C to T), and were supplied by Genosys Biotechnologies, Inc. (Texas).   The rightward  PCR primer was:     5'-GGGGGCGGGCCCGGGTGCGGAGGCGGAAATGCCGC TGGCGACGTTC-3'   (SEQ    ID    NO:16);   and   the   leftward    PCR   primer   was:       5'-GGAACATCACGGCATTCACC-3' (SEQ ID N0:15). The PCR reaction was carried out using an Advantage GC genomic PCR kit (Clonetech Laboratories, Palo Alto, CA) in buffer provided by the manufacturer in the presence of 200 ^M dNTPs, 200 pmol of each primer, 50 ng template DNA, 1.1 mM Mg acetate, 1.0 M GC-Melt and 1 unit Tth DNA Polymerase in a final volume of 50 (il.   The thermal profile of the first cycle was 94°C for 1 min; followed by 25 cycles of 94°C for 30 sec and 68°C for 2 min; and 1 cycle at 68°C for 3 min. A PCR product of 449 bp was digested with Apa\ and Kpnl to release a 254 bp fragment, which was resolved by electrophoresis and purified from the gel.  All three fragments (-3.8 Kb, -0.4 Kb and 254 bp) were ligated together in a 3-way ligation. The ligation mixture was transformed into competent E. co//DH5cc cells. Plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the correct insert was confirmed by restriction analysis.    This plasmid was designated as pSE238.<br>
pSE238 was digested with EcoRI, cloned into pWHMS, which had been digested with EcoRI, and transformed into E. coli DH5a cells. Plasmid DNA was isolated from ampicillin resistant transformants and the presence of the correct insert was confirmed by restriction analysis and DNA sequence analysis. This plasmid DNA was transformed into E. coli DM1, plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the correct insert was confirmed by restriction analysis. This plasmid, which was designated as pSE239, was used to transform protoplasts of S. avermitilis strain SE180-11. Thiostrepton resistant transformants of strain SE180-11 were isolated, the presence of erythromycin resistance was determined, and Thior Ermr transformants were analyzed by HPLC analysis of fermentation products. The presence of the double mutated aveC gene encoding S138T/A139F was able to restore normal avermectin production to strain SE180-11; however, the B2:B1 ratio was 0.75:1 showing that this mutation further reduced the amount of cyclohexyl-B2 produced relative to cyclohexyl-B1 (TABLE 3) over the reductions provided by transformation of strain SE180-11 with pSE188, pSE199, or pSE231 as described above.<br><br>
TABLE 3(Table Remove)<br>
Additional mutations were constructed to further reduce the amount of cyclohexyl-B2 produced relative to cyclohexyl-B1 using the technique of DNA shuffling as described in Stemmer, 1994, Nature 370:389-391; and Stemmer, 1994, Proc. Natl. Acad. Sci. USA 91:10747-10751; and further described in United States Patent Nos. 5,605,793; 5,811,238; 5,830,721; and 5,837,458.<br>
DNA shuffled plasmids containing mutated aveC genes were transformed into competent dam dcm E. coli cells. Plasmid DNA was isolated from ampicillin resistant transformants, and used to transform protoplasts of S. avermitilis strain SE180-11. Thiostrepton resistant transformants of strain SE180-11 were isolated and screened for the production of avermectins with a cyclohexyl-B2:cyclohexyl-B1 ratio of 1:1 or less. The DNA sequence of plasmid DNA from SE180-11 transformants producing avermectins with a B2:B1 ratio of 1:1 or less was determined.<br>
Eight transformants were identified that produced reduced amounts of cyciohexyl-B2 relative to cyclohexyl-B1. The lowest B2:B1 ratio achieved among these transformants was 040:1 (TABLE 4). Plasmid DNA was isolated from each of the eight transformants and the DNA sequence determined to identify the mutations in the aveC gene. The mutations are as follows.<br>
pSE290 contains 4 nucleotide mutations at nt position 317 from T to A, at nt position 353 from C to A, at nt position 438 from G to A, and at nt position 1155 from T to A. The nucleotide change at nt position 317 changes the amino acid at AA position 48 from D to E and the nucleotide change at nt position 438 changes the amino acid at AA position 89 from A to T. The B2:B1 ratio produced by cells carrying this plasmid was 0.42:1 (TABLE 4).<br><br>
pSE291 contains 4 nucleotide mutations at nt position 272 from G to A, at nt position 585 from T to A, at nt position 588 from G to A, and at nt position 708 from G to A. The nucleotide change at nt position 585 changes the amino acid at AA position 138 from S to T, the nucleotide change at nt position 588 changes the amino acid at AA position 139 from A to T, and the nucleotide change at nt position 708 changes the amino acid at AA position 179 from G to S. The B2:B1 ratio produced by cells carrying this plasmid was 0.57:1 (TABLE 4).<br>
pSE292 contains the same four nucleotide mutations as pSE290. The B2:B1 ratio<br><br>
produced by cells carrying this plasmid was 0.40:1 (TABLE 4).<br>
pSE293 contains 6 nucleotide mutations at nt 24 from A to G, at nt position 286 from A to C, at nt position 497 from T to C, at nt position 554 from C to T, at nt position 580 from T to C, and at nt position 886 from A to T. The nucleotide change at nt position 286 changes the amino acid at AA position 38 from Q to P, the nucleotide change at nt position 580 changes the amino acid at AA position 136 from L to P, and the nucleotide change at nt position 886 changes the amino acid at AA position 238 from E to D. The B2:B1 ratio produced by cells carrying this plasmid was 0.68:1 (TABLE 4).<br>
pSE294 contains 6 nucleotide mutations at nt 469 from T to C, at nt position 585 from T to A, at nt position 588 from G to A, at nt position 708 from G to A, at nt position 833 from C to T, and at nt position 1184 from G to A. In addition, nts at positions 173, 174, and 175 are deleted. The nucleotide change at nt position 469 changes the amino acid at AA position 99 from F to S, the nucleotide change at nt position 585 changes the amino acid at AA position 138 from S to T, the nucleotide change at nt position 588 changes the amino acid at AA position 139 from A to T, and the nucleotide change at nt position 708 changes the amino acid from AA position 179 from G to S. The B2:B1 ratio produced by cells carrying this plasmid was 0.53:1 (TABLE 4).<br>
pSE295 contains 2 nucleotide mutations at nt 588 from G to A and at nt 856 from T to C. The nucleotide change at nt position 588 changes the amino acid at AA position 139 from A to T and the nucleotide change at nt position 856 changes the amino acid at AA position 228 from M to T. The B2:B1 ratio produced by cells carrying this plasmid was 0.80:1 (TABLE 4).<br>
pSE296 contains 5 nucleotide mutations at nt position 155 from T to C, at nt position 505 from G to T, at nt position 1039 from C to T, at nt position 1202 from C to T, and at nt position 1210 from T to C. The nucleotide change at nt position 505 changes the amino acid at AA position 111 from G to V and the nucleotide change at nt position 1039 changes the amino acid at AA position 289 from P to L. The B2:B1 ratio produced by cells carrying this plasmid was 0.73:1 (TABLE 4).<br>
pSE297 contains 4 nucleotide mutations at nt position 377 from G to T, at nt position 588 from G to A, at nt position 633 from A to G, and at nt position 1067 from A to T. The<br><br>
nucleotide change at nt position 588 changes the amino acid at AA position 139 from A to T, the nucleotide change at nt position 633 changes the amino acid at AA position 154 from K to E, and the nucleotide change at nt position 1067 changes the amino acid at AA position 298 from Q to H. The B2:B1 ratio produced by cells carrying this plasmid was 0.67:1 (TABLE 4).<br>
TABLE 4(Table Remove)<br>
An additional round of DNA shuffling was conducted to further reduce the amount of cyclohexyl-B2 avermectin produced relative to cyclohexyl-B1 avermectin as follows.<br>
Semi-synthetic shuffling<br>
The best clone was shuffled using the method described in PCT International Publication WO 97/20078 by Maxygen Inc. Individual oligonucleotides encoding beneficial substitutions best corresponding to decreased B2:B1 ratio were added to the shuffling reaction at 5 molar excess over the aueC template strands. Each nucleotide mismatch of the oligonucleotide was flanked by 20 nucleotides of perfect identity to ensure proper incorporation during the shuffling reaction. Oligonucleotides were purchased from Operon Technologies (Alameda, CA).<br>
HTP growth ofS. avermitilis<br>
Independent clones were picked from the transformation plates and inoculated into 200 nl R5 medium (Kieser, T., et a/., "Practical Streptomyces Genetics", 2000, Norwich, U.K.,<br>
John Innes Foundation) in deep 96-welt seed plates and grown at 30°C with shaking. In each well, a glass-bead was dispensed for dispersion of mycelia and agitation of the culture. During this time, the cultures attained even and dense growth. After 4-5 days, 20 j.tl of the seed medium culture was dispensed to production plates and the remaining volume was frozen as master plates at -80°C after the addition of glycerol to the final concentration of 20%. The production plates were incubated at 30° under humidity for 12-14 days. Sporulation of the strains occurred after 5-8 days of incubation. The production plates were made essentially as described in PCT International Publication WO 99/41389 by Pfizer Inc., with the exception of adding 1 % agarose to ensure a solid surface. Extraction and ESI-MS/MS screening<br>
A total of 1 ml ethyl acetate was added to each well and incubated shaking at room temperature for 20 minutes. Approximately 750 |.tl of the ethyl acetate-phase was recovered, transferred to a 96-well plate and set to evaporate over night. The precipitate was resuspended in 100 (il methanol 1mM NaCI of which 5 ^l solution was injected into mass spectrometer by an autosampler in a 96-well format and analyzed directly in the flow injection phase without liquid chromatography or other separation. The compounds were ionized by electrospray ionization and two separate channels were monitored on two MS/MS transitions. The MS/MS transition for B1 sodiated ion is from m/z 921 to m/z 777 and for B2 sodiated ion is from m/z 939 to m/z 795 in positive mode. A Finnigan TSQ-7000, Micromass Quattro-LC mass spectrometer and a Leap Technology Twin-Pal autosampler were used for this high throughput screening. Integration of the separate B1 and B2 chromatograms for each well location identified the hits.<br>
Fifty-seven (57) new combinations of amino acid substitutions were identified that can produce ratios of cyclohexyl B2:cyclohexyl B1 avermectins of 0.35:1 or less (FIGURE 6). Several of these new mutations can produce ratios of cyclohexyl B2:cyclohexyl B1 avermectins of about 0.30:1 or less; several can produce ratios of cyclohexyl B2:cyclohexyl B1 avermectins of about 0.25:1 or less, and several can produce ratios of cyclohexyl B2:cyclohexyl B1 avermectins of about 0.20:1 or less. Two (2) new mutations were identified that can produce ratios of cyclohexyl B2:cyclohexyl B1 avermectins of 0.37 or 0.38.<br>
DEPOSIT OF BIOLOGICAL MATERIALS<br>
The following plasmids were deposited with the American Type Culture Collection (ATCC) at 12301 Parklawn Drive, Rockville, MD, 20852, USA, on January 29, 1998, and were assigned the following accession numbers:<br>
Plasmid	Accession No.<br>
plasmidpSE180	209605<br>
plasmidpSE186	209604<br>
The current address of the American Type Culture Collection is 10801 University Blvd, Manassas, VA, 20110, USA.<br>
All patents, patent applications, and publications cited above are incorporated herein<br>
by reference in their entirety.	I<br>
The present invention is not to be limited in scope by the specific embodiment described herein, which are intended as single illustrations of individual aspects of the invention, and functionally equivalent methods and components are within the scope of the invention. Indeed, various modifications of the invention, in addition to those shown and described herein will become apparent to those skilled in the art from the foregoing description and accompanying drawings. Such modifications are intended to fall within the scope of the appended claims.<br><br><br><br><br>
WE CLAIM:<br>
1.	A polynucleotide molecule comprising a nucleotide sequence that is the<br>
Streptomyces avermetilis aveC allele, the S. avermetilis AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604) or the nucleotide sequence of the aveC ORF of S. avermetilis as presented in SEQ ID NO:l, or a degenerate variant thereof and said sequence comprises mutations encoding a combination of amino acid substitutions at amino acid residues corresponding to the amino acid positions of SEQ ID NO:2, such that cells of S. avermetilis strain ATCC 53692 in which the wild-type aveC allele has been inactivated and that express the polynucleotide molecule comprising the mutated nucleotide sequence capable of producing a class 2:1 ratio of avermectins wherein when the class 2:1 avermectins are cyclohexyl B2:cyclohexyl Bl avermectins the ratio of class 2:1 avermectins is 0.2:1 or less, and wherein the combinations of amino acid substitutions comprise substitutions at both D48 and G179 selected from the group consisting of:<br>
(da)	S41G,   D48E,  A61T,   R71L,  A89T,   L136M,   S138T,  A139T,<br>
T149S,<br>
G179S, V196A, E238D, F278L, P289L;<br>
(db)	S41G,   D48E,   A61T,   R71L,   A89T,   L136M,   S138T,   A139T,<br>
T149S, F176C, G179S, V196A, E238D, P289L;<br>
(qq)	D48E,  R71L,  A89T,  L136P, T149S,  F176C,  G179S,  E238D, I280V;<br>
(rr)	D48E, A61T, R71L, W110L, T149S, G179S, V196A, L206M, E238D, V271A, I280V;<br>
(ss)	D48E,   A61T,   R71L,  A89T,   L136M,   S138T,   A139T,  T149S, G179S, V196A, E238D, H279Q, P289L;<br><br>
(tt)	D48E,   A61T,   R71L,   A89T,   L136M,   S138T,   A139T,   T149S, G179S, V196A, E238D, G287E, P289Q;<br>
(uu)	D48E,   A61T,   R7IL,   A89T,   L136M,   S138T,   A139T,   T149S, G179S, V196A, E238D, P289L;<br>
(dh)   D48E,  A61T,   R71L,  A89T,   A139T,  T149S,   F176C,   G179S, V196A, E238D, V285G, P289L;<br>
(di) Q38R, D48E, A61T, R71L, L87V, A89T, L136M, S138T, A139T, T149S, G179S, V196A, E238D, P289L;<br>
(dj) D48E, A61T, L87V, A89T, W110L, S138T, A139T, T149S, G179S, V196A, E238D, P289L;<br>
(dk) D48E, A61T, R71L, A89T, L136M, S138T, A139T, T149S, G179S, V196A, E238D, P289L; (dl) D48E, A89T, L136P, K154E, G179S, S231L, E238D;<br>
(yy)	D48E, A61T, R71L, A89T, L136P, T149S, F176C, G179S, V196A, E238D, I280V;<br>
(zz)	D48E, R71L, A89T, L136P, T149S, F176C, G179S, E238D, I280V;<br>
(aaa)	Q36P, D48E, A61T, R71L, A89T, L136P, T149S, F176C, G179S, V196A, E238D, I280V;<br>
(bbb)	D48E, A61T, R71L, A89T, A139T, T149S, F176C, G179S, V196A, E238D, I280V;<br>
(ee)V2M, D48E, A61T, R71L, A89T, L136P, T149S, F176C, G179S,<br>
V196A, E238D, I280V;<br>
(ef) D48E, A61T, R71L, A89T, L136P, T149S, F176C, G179S, V196A,<br>
E238D, I280V, A302T;<br>
(eg)D48E, R71L, A89T, L136P, T149S, F176C, G179S, E238D, P289L;<br>
(eh)D48E, R71L, A89T, L136P, T149S, F176C, G179S, E238D, A302T;<br>
(ei) D48E, R71L, A89T, L136P, T149S, F176C, G179S, V196A, E238D,<br>
I280V;<br>
(ej) D48E, A61T, R71L, A89T, L136P, T149S, F176C, G179S, V196A,<br>
E238D;<br>
(ek) V2M, D48E, R71L, A89T, L136P, T149S, F176C, G179S, V196A,<br><br>
E238D, I280V;<br>
(el) D48E, A61T, R71L, A89T, L136P, T149S, R162H, F176C, G179S, V196A, E238D, I280V;<br>
(em) D48E, R71L, A89T, V120A, L136P, T149S, K154E, G179S, S231L, E238D;<br>
(en) D48E, R71L, A89T, V120A, L136P, T149S, F176C, G179S, S231L, E238D, I280V;<br>
(eo) D48E, A61T, R71L, L87V, A89T, A139T, T149S, F176C, G179S, V196A, E238D, V285G, P289L;<br>
(ep) D48E, A6IT, R71L, L87V, A89T, S90N, A139T, T149S, F176C, G179S, V196A, E238D, V285G, P289L;<br>
(eq)D48E, R71L, A89T, L136P, K154E, G179S, S231L, E238D; (er) D48E, R71L, A89T, V120A, L136P, K154E, F176C, GI79S, S231L, E238D; and<br>
(es) D48E, R71L, A89T, V120A, L136P, T149S, K154E, F176C, G179S, S231L, E238D.<br>
2.	The polynucleotide molecule as claimed in claim 1, wherein the<br>
combination of amino acid substitutions is:<br>
(ea) D48E, A6IT, R7IL, A89T, L139P, T149S, F176C, G179S, V196A, E230D, I280V.<br>
3.	A recombinant vector comprising a polynucleotide molecule as claimed in<br>
claims 1 or 2.<br>
4.	A host cell comprising the recombinant vector as claimed in claim 3.<br>
5.	The host cell as claimed in claim 4, which is a Streptomyces cell.<br>
6.	The host cell as claimed in claim 5, which cell is a cell of Streptomyces avermetilis comprising a mutated S. Avermetilis aveC allele, or degenerate<br><br>
variant thereof encoding an AveC gene product comprising a combination of amino acid substitutions such as herein described.<br><br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1ERUxOUC0yMDA0LUFic3RyYWN0LSgwOS0wNy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1882-DELNP-2004-Abstract-(09-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1ERUxOUC0yMDA0LUFic3RyYWN0LSgxMi0wOC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1882-DELNP-2004-Abstract-(12-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1kZWxucC0yMDA0LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1882-delnp-2004-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1ERUxOUC0yMDA0LUNsYWltcy0oMDktMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1882-DELNP-2004-Claims-(09-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1ERUxOUC0yMDA0LUNsYWltcy0oMTItMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1882-DELNP-2004-Claims-(12-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1kZWxucC0yMDA0LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1882-delnp-2004-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1kZWxucC0yMDA0LWNvbXBsZXRlIHNwZWNpZmljYXRpb24gKGdyYW50ZWQpLnBkZg==" target="_blank" style="word-wrap:break-word;">1882-delnp-2004-complete specification (granted).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1kZWxucC0yMDA0LWNvcnJlc3BvbmRlbmNlLW90aGVycy0oMDctMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1882-delnp-2004-correspondence-others-(07-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDktMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1882-DELNP-2004-Correspondence-Others-(09-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1kZWxucC0yMDA0LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1882-delnp-2004-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1ERUxOUC0yMDA0LURlc2NyaXB0aW9uIChDb21wbGV0ZSktKDEyLTA4LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">1882-DELNP-2004-Description (Complete)-(12-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1kZWxucC0yMDA0LWRlc2NyaXB0aW9uIChjb21wbGV0ZSktMDktMDctMjAwOC5wZGY=" target="_blank" style="word-wrap:break-word;">1882-delnp-2004-description (complete)-09-07-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1kZWxucC0yMDA0LWRlc2NyaXB0aW9uIChjb21wbGV0ZSktMTItMDgtMjAwOC5wZGY=" target="_blank" style="word-wrap:break-word;">1882-delnp-2004-description (complete)-12-08-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1kZWxucC0yMDA0LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">1882-delnp-2004-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1kZWxucC0yMDA0LWRyYXdpbmdzLSgwNy0wOC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1882-delnp-2004-drawings-(07-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1kZWxucC0yMDA0LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1882-delnp-2004-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1ERUxOUC0yMDA0LUZvcm0tMS0oMDktMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1882-DELNP-2004-Form-1-(09-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1ERUxOUC0yMDA0LUZvcm0tMS0oMTItMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1882-DELNP-2004-Form-1-(12-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1kZWxucC0yMDA0LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">1882-delnp-2004-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1kZWxucC0yMDA0LWZvcm0tMTkucGRm" target="_blank" style="word-wrap:break-word;">1882-delnp-2004-form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1ERUxOUC0yMDA0LUZvcm0tMi0oMDktMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1882-DELNP-2004-Form-2-(09-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1ERUxOUC0yMDA0LUZvcm0tMi0oMTItMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1882-DELNP-2004-Form-2-(12-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1kZWxucC0yMDA0LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">1882-delnp-2004-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1ERUxOUC0yMDA0LUZvcm0tMy0oMDktMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1882-DELNP-2004-Form-3-(09-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1kZWxucC0yMDA0LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">1882-delnp-2004-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1kZWxucC0yMDA0LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">1882-delnp-2004-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1ERUxOUC0yMDA0LUdQQS0oMDktMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1882-DELNP-2004-GPA-(09-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1kZWxucC0yMDA0LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">1882-delnp-2004-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1ERUxOUC0yMDA0LU90aGVycy1Eb2N1bWVudC0oMDktMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1882-DELNP-2004-Others-Document-(09-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1kZWxucC0yMDA0LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">1882-delnp-2004-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1kZWxucC0yMDA0LXBjdC00MDIucGRm" target="_blank" style="word-wrap:break-word;">1882-delnp-2004-pct-402.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1kZWxucC0yMDA0LXBjdC00MDkucGRm" target="_blank" style="word-wrap:break-word;">1882-delnp-2004-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1kZWxucC0yMDA0LXBjdC00MTYucGRm" target="_blank" style="word-wrap:break-word;">1882-delnp-2004-pct-416.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1ERUxOUC0yMDA0LVBldGl0aW9uLTEzNy0oMDktMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1882-DELNP-2004-Petition-137-(09-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg4Mi1ERUxOUC0yMDA0LVBldGl0aW9uLTEzOC0oMDktMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1882-DELNP-2004-Petition-138-(09-07-2008).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="228491-a-device-for-keeping-clean-a-roller-conveying-textile-fibre-material-in-a-machine.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="228493-constant-velocity-universal-joint.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>228492</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1882/DELNP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>08/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>20-Feb-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>05-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>01-Jul-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>PFIZER PRODUCTS INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>EASTERN POINT ROAD, GROTON, CONNECTICUT 06340, UNITED STATES OF AMERICA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>JEREMY STEPHEN MINSHULL</td>
											<td>679 LOS NINOS WAY, LOS ALTOS, CALIFORNIA 94025 USA.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SUN AI RAILLARD</td>
											<td>964 TROPHY DRIVE, MOUNTAIN VIEW, CALIFORNIA 94040 USA.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>KIM JONELLE STUTZMAN-ENGWALL</td>
											<td>PFIZER GLOBAL RESEARCH AND DEVELOPMENT, EASTERN POINT ROAD, GROTON, CONNECTICUT 06340 USA.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>CLAES ERIC DANIEL GUSTAFSSON</td>
											<td>1813 BAYVIEW AVENUE, BELMONT, CALIFORNIA 94002, USA</td>
										</tr>
										<tr>
											<td>5</td>
											<td>ANKE KREBBER</td>
											<td>946 VAN AUKEN CIRCLE, PALO ALTO, CALIFORNIA 94303 USA.</td>
										</tr>
										<tr>
											<td>6</td>
											<td>YAN CHEN</td>
											<td>1765 BOWERS AVENUE, SANTA CLARA, CALIFORNIA 95051, USA</td>
										</tr>
										<tr>
											<td>7</td>
											<td>SERAN KIM</td>
											<td>1320 SOUTH WHITE OAK DRIVE #633, WAUKEGAN, ILLINOIS 60085 USA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IB03/00348</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-01-31</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/356,222</td>
									<td>2002-02-12</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/228492-polynucleotide-molecule-comprising-a-nucleotide-sequence-that-is-the-streptomyces-avermetilis-avec-allele by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:05:09 GMT -->
</html>
