<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/228501-an-isolated-polypeptide-comprising-an-immunogenic-portion by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:05:44 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 228501:&quot;AN ISOLATED POLYPEPTIDE, COMPRISING AN IMMUNOGENIC PORTION&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;AN ISOLATED POLYPEPTIDE, COMPRISING AN IMMUNOGENIC PORTION&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>An isolated polypeptide, comprising an immunogenic portion of the kind such as herein described, of a lung tumor protein, wherein the lung tumor protein comprises an amino acid sequence that is encoded by a polynucleotide sequence set forth in SEQ ID NO: 808</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>TECHNICAL FIELD OF THE INVENTION<br>
The present invention relates generally to therapy and diagnosis of<br>
cancer, such as lung cancer. The invention is more specifically related to polypeptides<br>
comprising at least a portion of a lung tumor protein, and to polynucleotides encoding<br>
such  polypeptides.      Such polypeptides  and  polynucleotides  may  be  used  in<br>
compositions for prevention and treatment of lung cancer, and for the diagnosis and<br>
monitoring of such cancers.<br>
BACKGROUND OF THE INVENTION<br>
Cancer is a significant health problem throughout the world.  Although<br>
advances have been made in detection and therapy of cancer, no vaccine or other<br>
universally successful method for prevention or treatment is currently available.<br>
Current therapies, which are generally based on a combination of chemotherapy or<br>
surgery and radiation, continue to prove inadequate in many patients.<br>
Lung cancer is the primary cause of cancer death among both men and<br>
women in the U.S., with an estimated 172,000 new cases being reported in 1994. The<br>
five-year survival rate among all lung cancer patients, regardless of the stage of disease<br>
at diagnosis, is only 13%. This contrasts with a five-year survival rate of 46% among<br>
cases detected while the disease is still localized. However, only 16% of lung cancers<br>
are discovered before the disease has spread.<br>
Early detection is difficult since clinical symptoms are often not seen<br>
until the disease has reached an advanced stage.   Currently, diagnosis is aided by the<br>
use of chest x-rays, analysis of the type of cells contained in sputum and fiberoptic<br>
examination of the bronchial passages. Treatment regimens are determined by the type<br>
and stage of the cancer, and include surgery, radiation therapy and/or chemotherapy.<br>
In spite of considerable research into therapies for this and other cancers, lung cancer remains difficult to diagnose and treat effectively. The prior art provides for detection of lung cancer through chest x-rays, sputum analysis, and fiber optic bronchial examination, and the treatments of the prior art can include chemotherapy, surgery and/or radiation. Obvious drawbacks to these treatments can include harsh side effects and lengthy recovery. Early detection greatly enhances the five year survival rate of patients, but using the methods of the prior art, only 16% of cases are discovered prior to metastasis. Accordingly, there is a need in the art for improved method for detecting and treating such cancers. The present invention fulfils these need and further provided other related advantages.    The polypeptides of the current<br><br>
invention can be utilized in a variety of ways as disclosed in the specification, providing earlier detection and improved monitoring of these disease state. The can also be used in cancer prevention. Something not satisfactorily available in the prior art. Along with the earlier detection and prevention of lung cancer, the current invention can be used for a variety of treatment options, including the induction of a cellular immune response through immunogenic compositions and effector cells.<br>
SUMMARY OF THE INVENTION<br>
Briefly stated, the present invention provides compositions and methods<br>
for the diagnosis and therapy of cancer, such as lung cancer. In one aspect, the present<br>
invention provides polypeptides comprising at least a portion of a lung tumor protein, or<br>
a variant thereof.  Certain portions and other variants are immunogenic, such that the<br>
ability of the variant to react with antigen-specific anttsera is not substantially<br>
diminished. Within certain embodiments, the polypeptide comprises a sequence that is<br>
encoded by a polynucleotide sequence selected from the group consisting of: (a)<br>
sequences recited in SEQ ID NO: 1,11-13,15,20,23-27,29,30,33, 34, 39,41,43-46,<br>
51, 52, 57, 58,60, 62, 65-67,69-71, 74, 76,79, 80, 84, 86, 89-92, 95, 97, 98,101, 110,<br>
111, 113-119, 121-128, 130-134, 136, 138, 139, 141, 143, 146-151,153, 154, 157-160,<br>
162-164, 167-178,180, 181, 183,186-190, 192,193,195-220, 224, 226-231, 234, 236,<br>
237, 240, 241, 244-246, 248, 254, 255, 261, 262, 266, 270, 275, 280, 282, 283, 288,<br>
289, 290, 292, 295, 301, 303, 304, 309, 311, 341-782, 784, 785, 790, 792, 794, 796,<br>
800-804, 807, 808 and 810-826; (b) variants of a sequence recited in SEQ ID NO: 1,<br>
11-13,15, 20,23-27, 29, 30, 33, 34, 39, 41,43-46, 51, 52, 57, 58, 60, 62, 65-67, 69-71,<br>
74, 76, 79, 80, 84, 86, 89-92, 95, 97, 98, 101, 110, 111, 113-119, 121-128, 130-134,<br>
136,138, 139,141, 143, 146-151,153,154,157-160,162-164, 167-178, 180, 181, 183,<br>
186-190, 192, 193, 195-220, 224, 226-231, 234, 236, 237, 240,241,244-246,248, 254,<br>
255, 261, 262, 266, 270, 275, 280, 282, 283, 288, 289, 290, 292, 295, 301, 303, 304,<br>
309, 311, 341-782, 784, 785, 790, 792, 794, 796, 800-804, 807, 808 and 810-826; and<br>
(c) complements of a sequence of (a) or (b). In specific embodiments, the polypeptides<br>
of the present invention comprise at least a portion of a tumor protein that includes an<br>
amino acid sequence selected from the group consisting of sequences recited in SEQ ID<br>
NO: 786,787,791, 793,795,797-799,806,809 and 827, and variants thereof.<br>
The present invention further provides polynucleotides that encode a<br>
polypeptide as described above, or a portion thereof (such as a portion encoding at least<br><br>
I5 amino acid residues of a lung tumor protein), expression vectors comprising such<br>
polynucleotides and host cells transformed or transfected with such expression Vectors.<br>
Within other aspects, the present invention provides pharmaceutical<br>
compositions comprising a polypeptide or polynucleotide as described above and a<br>
physiologically acceptable carrier.<br>
Within a related aspect of the present invention, vaccines, or<br>
immunogenic compositions, for prophylactic or therapeutic use are provided. Such<br>
vaccines comprise a polypeptide or polynucleotide as described above and an<br>
immunostimulant.<br>
The present invention further provides pharmaceutical compositions that<br>
comprise: (a) an antibody or antigen-binding fragment thereof that specifically binds to<br>
a lung tumor protein; and (b) a physiologically acceptable carrier.<br>
Within further aspects, the present invention provides pharmaceutical<br>
compositions comprising: (a) an antigen presenting cell that expresses a polypeptide as<br>
described above and (b) a pharmaceutically acceptable carrier or excipient. Antigen<br>
presenting cells include dendritic cells, macrophages, monocytes, fibroblasts and B<br>
cells.<br>
Within related aspects, vaccines, or immunogenic compositions, are<br>
provided that comprise: (a) an antigen presenting cell that expresses a polypeptide as<br>
described above and (b) an immunostimulant.<br>
The present invention further provides, in other aspects, fusion proteins<br>
that comprise at least one polypeptide as described above, as well as polynucleotides<br>
encoding such fusion proteins.<br>
Within related aspects, pharmaceutical compositions comprising a fusion<br>
protein, or a polynucleotide encoding a fusion protein, in combination with a<br>
physiologically acceptable carrier are provided.<br>
Vaccines, or immunogenic compositions, are further provided, within<br>
other aspects, that comprise a fusion protein, or a polynucleotide encoding a fusion<br>
protein, in combination with an immunostimulant.<br>
Within further aspects, the present invention provides methods for<br>
inhibiting the development of a cancer in a patient, comprising administering to a<br>
patient a pharmaceutical composition or immunogenic composition as recited above:<br>
The patient may be afflicted with lung cancer, in which case the methods provide<br>
treatment for the disease, or patient considered at risk for such a disease may be treated<br>
prophylactically.<br>
The present invention further provides, within other aspects, methods for<br>
removing tumor cells from a biological sample, comprising contacting a biological<br>
sample with T cells that specifically react with a lung tumor protein, wherein the step of<br>
contacting is performed under conditions and for a time sufficient to permit the removal<br>
of cells expressing the protein from the sample.<br>
Within related aspects, methods are provided for inhibiting the<br>
development of a cancer in a patient, comprising administering to a patient a biological<br>
sample treated as described above.<br>
Methods are further provided, within other aspects, for stimulating<br>
and/or expanding T cells specific for a lung tumor protein, comprising contacting T<br>
cells with one or more of: (i) a polypeptide as described above; (ii) a polynucleotide<br>
encoding such a polypeptide; and/or (iii) an antigen presenting cell that expresses such<br>
a polypeptide; under conditions and for a time sufficient to permit the stimulation<br>
and/or expansion of T cells. Isolated T cell populations comprising T cells prepared as<br>
described above are also provided.<br>
Within further aspects, the present invention provides methods for<br>
inhibiting the development of a cancer in a patient, comprising administering to a<br>
patient an effective amount of a T cell population as described above.<br>
The present invention further provides methods for inhibiting the<br>
development of a cancer in a patient, comprising the steps of: (a) incubating CD4+<br>
and/or CD8+ T cells isolated from a patient with one or more of: (i) a polypeptide<br>
comprising at least an immunogenic portion of a lung tumor protein; (ii) a<br>
polynucleotide encoding such a polypeptide; and (iii) an antigen-presenting cell that<br>
expressed such a polypeptide; and (b) administering to the patient an effective amount<br>
of the proliferated T cells, and thereby inhibiting the development of a cancer in the<br>
patient Proliferated cells may, but need not, be cloned prior to administration to the<br>
patient.<br>
Within further aspects, the present invention provides methods for<br>
determining the presence or absence of a cancer in a patient, comprising: (a) contacting<br>
a biological sample obtained from a patient with a binding agent that binds to a<br>
polypeptide as recited above; (b) detecting in the sample an amount of polypeptide that<br>
binds to the binding agent; and (c) comparing the amount of polypeptide with a<br>
predetermined cut-off value, and therefrom determining the presence or absence of a<br>
cancer in the patient. Within preferred embodiments, the binding agent is an antibody,<br>
more preferably a monoclonal antibody. The cancer may be lung cancer.<br>
The present invention also provides, within other aspects, methods for<br>
monitoring the progression of a cancer in a patient. Such methods comprise the steps<br>
of: (a) contacting a biological sample obtained from a patient at a first point hi tune<br>
with a binding agent that binds to a polypeptide as recited above; (b) detecting in the<br>
sample an amount of polypeptide that binds to the binding agent; (c) repeating steps (a)<br>
and (b) using a biological sample obtained from the patient at a subsequent point in<br>
time; and (d) comparing the amount of polypeptide detected in step (c) with the amount<br>
detected in step (b) and therefrom monitoring the progression of the cancer in the<br>
patient.<br>
The present invention further provides, within other aspects, methods for<br>
determining the presence or absence of a cancer hi a patient, comprising the steps of: (a)<br>
contacting a biological sample obtained from a patient with an oligonucleotide that<br>
hybridizes to a polynucleotide that encodes a lung tumor protein; (b) detecting in the<br>
sample a level of a polynucleotide, preferably mRNA, that hybridizes to the<br>
oligonucleotide; and (c) comparing the level of polynucleotide that hybridizes to the<br>
oligonucleotide with a predetermined cut-off value, and therefrom determining the<br>
presence or absence of a cancer hi the patient. Within certain embodiments, the amount<br>
of mRNA is detected via polymerase chain reaction using, for example, at least one<br>
oligonucleotide primer that hybridizes to a polynucleotide encoding a polypeptide as<br>
recited above, or a complement of such a polynucleotide. Within other embodiments,<br>
the amount of mRNA is detected using a hybridization technique, employing an<br>
oligonucleotide probe that hybridizes to a polynucleotide that encodes a polypeptide as<br>
recited above, or a complement of such a polynucleotide.<br>
In related aspects, methods are provided for monitoring the progression<br>
of a cancer in a patient, comprising the steps of: (a) contacting a biological sample<br>
obtained from a patient with an oligonucleotide that hybridizes to a polynucleotide that<br>
encodes a lung tumor protein; (b) detecting in the sample an amount of a polynucleotide<br>
that hybridizes to the oligonucleotide; (c) repeating steps (a) and (b) using a biological<br>
sample obtained from the patient at a subsequent point in time; and (d) comparing the<br>
amount of polynucleotide detected in step (c) with the amount detected in step (b) and<br>
therefrom monitoring the progression of the cancer in the patient<br>
Within further aspects, the present invention provides antibodies, such as<br>
monoclonal antibodies, that bind to a polypeptide as described above, as well as<br>
diagnostic kits comprising such antibodies. Diagnostic kits comprising one or more<br>
oligonucleotide probes or primers as described above are also provided.<br>
These and other aspects of the present invention will become apparent<br>
upon reference to the following detailed description and attached drawings. All<br>
references disclosed herein are hereby incorporated by reference in their entirety as if<br>
each was incorporated individually.<br>
SEQUENCE IDENTIFIERS<br>
SEQ ID NO: 1 is the determined cDNA sequence for clone #19038, also<br>
referred to as L845P.<br>
SEQ ID NO: 2 is the determined cDNA sequence for clone #19036.<br>
SEQ ID NO: 3 is the determined cDNA sequence for clone #19034.<br>
SEQ ED NO: 4 is the determined cDNA sequence for clone #19033.<br>
SEQ ID NO: 5 is the determined cDNA sequence for clone #19032.<br>
SEQ ID NO: 6 is the determined cDNA sequence for clone #19030, also<br>
referred to as L559S.<br>
SEQ ID NO: 7 is the determined cDNA sequence for clone #19029.<br>
SEQ ID NO: 8 is the determined cDNA sequence for clone #19025.<br>
SEQ ID NO: 9 is the determined cDNA sequence for clone #19023.<br>
SEQ ID NO: 10 is the determined cDNA sequence for clone #18929.<br>
SEQ ID NO: 11 is the determined cDNA sequence for clone #19010.<br>
SEQ ID NO: 12 is the determined cDNA sequence for clone #19009.<br>
SEQ ID NO: 13 is the determined cDNA sequence for clones #19005,<br>
19007,19016 and 19017.<br>
SEQ ID NO: 14 is the determined cDNA sequence for clone # 19004.<br>
SEQ ID NO: 15 is the determined cDNA sequence for clones #19002<br>
and 18965.<br>
SEQ ID NO: 16 is the determined cDNA sequence for clone #18998.<br>
SEQ ID NO: 17 is the determined cDNA sequence for clone #18997.<br>
SEQ ID NO: 18 is the determined cDNA sequence for clone #18996.<br>
SEQ ID NO: 19 is the determined cDNA sequence for clone #18995.<br>
SEQ ID NO: 20 is the determined cDNA sequence for clone #18994,<br>
also known as L846P.<br>
SEQ ID NO: 21 is the determined cDNA sequence for clone #18992.<br>
SEQ ID NO: 22 is the determined cDNA sequence for clone #18991.<br>
SEQ ID NO: 23 is the determined cDNA sequence for clone #18990,<br>
also referred to as clone #20111.<br>
SEQ ID NO: 24 is the determined cDNA sequence for clone #18987.<br>
SEQ ID NO: 25 is the determined cDNA sequence for clone #18985,<br>
also referred as L839P.<br>
SEQ ID NO: 26 is the determined cDNA sequence for clone #18984,<br>
also referred to as L847P.<br>
SEQ ID NO: 27 is the determined cDNA sequence for clone # 18983.<br>
SEQ ID NO: 28 is the determined cDNA sequence for clones #18976<br>
and 18980.<br>
SEQ ID NO: 29 is the determined cDNA sequence for clone #18975.<br>
SEQ ID NO: 30 is the determined cDNA sequence for clone #18974.<br>
SEQ ID NO: 31 is the determined cDNA sequence for clone #18973.<br>
SEQ ID NO: 32 is the determined cDNA sequence for clone #18972.<br>
SEQ ID NO: 33 is the determined cDNA sequence for clone #18971,<br>
also referred to as L801P.<br>
SEQ ID NO: 34 is the determined cDNA sequence for clone #18970.<br>
SEQ ID NO: 3 5 is the determined cDNA sequence for clone #18966.<br>
SEQ ID NO: 36 is the determined cDNA sequence for .clones #18964,<br>
18968 and 19039.<br>
SEQ ID NO: 37 is the determined cDNA sequence for clone #18960.<br>
SEQ ID NO: 38 is the determined cDNA sequence for clone #18959.<br>
SEQ ID NO: 39 is the determined cDNA sequence for clones #18958<br>
and 18982.<br>
SEQ ID NO: 40 is the determined cDNA sequence for clones #18956<br>
and 19015.<br>
SEQ ID NO: 41 is the determined cDNA sequence for clone #18954,<br>
also referred to L848P.<br>
SEQ ID NO: 42 is the determined cDNA sequence for clone #18951.<br>
SEQ ID NO: 43 is the determined cDNA sequence for clone #18950.<br>
SEQ ID NO: 44 is the determined cDNA sequence for clones #18949<br>
and 19024, also referred to as L844P.<br>
SEQ ID NO: 45 is the determined cDNA sequence for clone #18948.<br>
SEQ ID NO: 46 is the determined cDNA sequence for clone #18947,<br>
also referred to as L840P.<br>
SEQ ID NO: 47 is the determined cDNA sequence for clones #18946,<br>
18953,18969 and 19027.<br>
SEQ ID NO: 48 is the determined cDNA sequence for clone #18942.<br>
SEQ ID NO: 49 is the determined cDNA sequence for clone #18940,<br>
18962,18963,19006, 19008,19000, and 19031.<br>
SEQ ID NO: 50 is the determined cDNA sequence for clone #18939.<br>
SEQ ID NO: 51 is the determined cDNA sequence for clones #18938<br>
and 18952.<br>
SEQ ID NO: 52 is the determined cDNA sequence for clone #18938.<br>
SEQ ID NO: 53 is the determined cDNA sequence for clone #18937.<br>
SEQ ID NO: 54 is the determined eDNA sequence for clones #18934,<br>
18935,18993 and 19022, also referred to as L548S.<br>
SEQ ID NO: 55 is the determined cDNA sequence for clone #18932.<br>
SEQ ID NO: 56 is the determined cDNA sequence for clones #18931<br>
and 18936.<br>
SEQ ID NO: 57 is the determined cDNA sequence for clone #18930.<br>
SEQ ID NO: 58 is the determined cDNA sequence for clone #19014,<br>
also referred to as L773P.<br>
SEQ ID NO: 59 is the determined cDNA sequence for clone #19127.<br>
SEQ ID NO: 60 is the determined cDNA sequence for clones #19057<br>
and 19064.<br>
SEQ ID NO: 61 is the determined cDNA sequence for clone #19122.<br>
SEQ ID NO: 62 is the determined cDNA sequence for clones #19120<br>
and 18121.<br>
SEQ ID NO: 63 is the determined cDNA sequence for clone #19118.<br>
SEQ ID NO: 64 is the determined cDNA sequence for clone #19117.<br>
SEQ ID NO: 65 is the determined cDNA sequence for clone #19116.<br>
SEQ ID NO: 66 is the determined cDNA sequence for clone #19114.<br>
SEQ ID NO: 67 is the determined cDNA sequence for clone #19112,<br>
also known as L561S.<br>
SEQ ID NO: 68 is the determined cDNA sequence for clone #19110.<br>
SEQ ID NO: 69 is the determined cDNA sequence for clone #19107,<br>
also referred to as L552S.<br>
SEQ ID NO: 70 is the determined cDNA sequence for clone #19106,<br>
also referred to as L547S.<br>
SEQ ID NO: 71 is the determined cDNA sequence for clones #19105<br>
and 19.111.<br>
SEQ ID NO: 72 is the determined cDNA sequence for clone #19099.<br>
SEQ ID NO: 73 is the determined cDNA sequence for clones #19095,<br>
19104 and 19125, also referred to as L549S.<br>
SEQ ID NO: 74 is the determined cDNA sequence for clone #19094.<br>
SEQ ID NO: 75 is the determined cDNA sequence for clones #19089<br>
and 19101.<br>
SEQ ID NO: 76 is the determined cDNA sequence for clone #19088.<br>
SEQ ID NO: 77 is the determined cDNA sequence for clones #19087,<br>
19092,19096,19100 and 19119.<br>
SEQ ID NO: 78 is the determined cDNA sequence for clone #19086.<br>
SEQ ID NO: 79 is the determined cDNA sequence for clone #19085,<br>
also referred to as L550S.<br>
SEQ ID NO: 80 is the determined cDNA sequence for clone #19084,<br>
also referred to as clone #19079.<br>
SEQ ID NO: 81 is the determined cDNA sequence for clone #19082.<br>
SEQ ID NO: 82 is thedetermined cDNA sequence for clone #19080.<br>
SEQ ID NO: 83 is the determined cDNA sequence for clone #19077.<br>
SEQ ID NO: 84 is the determined cDNA sequence for clone #19076,<br>
also referred to as L55IS.<br>
SEQ ID NO: 85 is the determined cDNA sequence for clone #19074,<br>
also referred to as clone #20102.<br>
SEQ ID NO: 86 is the determined cDNA sequence for clone #19073,<br>
also referred to as L560S.<br>
SEQ ID NO: 87 is the determined cDNA sequence for clones #19072<br>
and 19115.<br>
SEQ ID NO: 88 is the determined cDNA sequence for clone #19071.<br>
SEQ ID NO: 89 is the determined cDNA sequence for clone #19070.<br>
SEQ ID NO: 90 is the determined cDNA sequence for clone #19069.<br>
SEQ ID NO: 91 is the determined cDNA sequence for clone #19068,<br>
also referred to L563S.<br>
SEQ ID NO: 92 is the determined cDNA sequence for clone #19066.<br>
SEQ ID NO: 93 is the determined cDNA sequence for clone #19065.<br>
SEQ ID NO: 94 is the determined cDNA sequence for clone #19063.<br>
SEQ ID NO: 95 is the determined cDNA sequence for clones #19061,<br>
19081,19108 and 19109.<br>
SEQ ID NO: 96 is the determined cDNA sequence for clones #19060,<br>
19067 and 19083, also referred to as L548S.<br>
SEQ ID NO: 97 is the determined cDNA sequence for clones #19059<br>
and 19062.<br>
SEQ ID NO: 98 is the determined cDNA sequence for clone #19058.<br>
SEQ ID NO: 99 is the determined cDNA sequence for clone #19124.<br>
SEQ ID NO: 100 is the determined cDNA sequence for clone #18929.<br>
SEQ ID NO: 101 is the determined cDNA sequence for clone #18422.<br>
SEQ ID NO: 102 is the determined cDNA sequence for clone #18425.<br>
SEQ ID NO: 103 is the determined cDNA sequence for clone #18431.<br>
SEQ ID NO: 104 is the determined cDNA sequence for clone #18433.<br>
SEQ ID NO: 105 is the determined cDNA sequence for clone #18444.<br>
SEQ ID NO: 106 is the determined cDNA sequence for clone # 18449.<br>
SEQ ID NO: 107 is the determined cDNA sequence for clone #18451.<br>
SEQ ID NO: 108 is the determined cDNA sequence for clone #18452.<br>
SEQ ID NO: 109 is the determined cDNA sequence for clone #18455.<br>
SEQ ID NO: 110 is the determined cDNA sequence for clone #18457.<br>
SEQ ED NO: 111 is the determined cDNA sequence for clone #18466.<br>
SEQ ED NO: 112 is the determined cDNA sequence for clone #18468.<br>
SEQ ID NO: 113 is the determined cDNA sequence for clone #18471.<br>
SEQ ID NO: 114 is the determined cDNA sequence for clone #18475.<br>
SEQ ID NO: 115 is the determined cDNA sequence for clone #18476.<br>
SEQ ID NO: 116 is the determined cDNA sequence for clone #18477.<br>
SEQ ID NO: 117 is the determined cDNA sequence for clone #20631.<br>
SEQ ID NO: 118 is the determined cDNA sequence for clone #20634.<br>
SEQ ID NO: 119 is the determined cDNA sequence for clone #20635.<br>
SEQ ID NO: 120 is the determined cDNA sequence for clone #20637.<br>
SEQ ID NO: 121 is the determined cDNA sequence for clone #20638.<br>
SEQ ID NO: 122 is the determined cDNA sequence for clone #20643.<br>
SEQ ID NO: 123 is the determined cDNA sequence for clone #20652.<br>
SEQ ED NO: 124 is the determined cDNA sequence for clone #20653.<br>
SEQ ID NO: 125 is the determined cDNA sequence for clone #20657.<br>
SEQ ID NO: 126 is the determined cDNA sequence for clone #20658.<br>
SEQ ID NO: 127 is the determined cDNA sequence for clone #20660.<br>
SEQ ID NO: 128 is the determined cDNA sequence for done #20661.<br>
SEQ ID NO: 129 is the determined cDNA sequence for clone #20663.<br>
SEQ ID NO; 130 is the determined cDNA sequence for clone #20665.<br>
SEQ ID NO: 131 is the determined cDNA sequence for clone #20670.<br>
SEQ ID NO: 132 is the determined cDNA sequence for clone #20671.<br>
SEQ ID NO: 133 is the determined cDNA sequence for clone #20672.<br>
SEQ ID NO: 134 is the determined cDNA sequence for clone #20675.<br>
SEQ ID NO: 135 is the determined cDNA sequence for clone #20679.<br>
SEQ ID NO: 136 is the determined cDNA sequence for clone #20681.<br>
SEQ ID NO: 137 is the determined cDNA sequence for clone #20682.<br>
SEQ ID NO: 138 is the determined cDNA sequence for clone #20684.<br>
SEQ ID NO: 139 is the determined cDNA sequence for clone #20685.<br>
SEQ ID NO: 140 is the determined cDNA sequence for clone #20689.<br>
SEQ ID NO: 141 is the determined cDNA sequence for clone #20699.<br>
SEQ ID NO: 142 is the determined cDNA sequence for clone #20701.<br>
SEQ ID NO: 143 is the determined cDNA sequence for clone #20702.<br>
SEQ ID NO: 144 is the determined cDNA sequence for clone #20708.<br>
SEQ ID NO: 145 is the determined cDNA sequence for clone #20715.<br>
SEQ ID NO: 146 is the determined cDNA sequence for clone #20716.<br>
SEQ ID NO: 147 is the determined cDNA sequence for clone #20719.<br>
SEQ ID NO: 148 is the determined cDNA sequence for clone #19129.<br>
SEQ ID NO: 149 is the determined cDNA sequence for clone #19131.1.<br>
SEQ ID NO: 150 is the determined cDNA sequence for clone # 19132.2.<br>
SEQ ID NO: 151 is the determined cDNA sequence for clone #19133.<br>
SEQ ID NO: 152 is the determined cDNA sequence for clone #19134.2.<br>
SEQ ID NO: 153 is the determined cDNA sequence for clone #19135.2.<br>
SEQ ID NO: 154 is the determined cDNA sequence for clone #19137.<br>
SEQ ID NO: 155 is a first determined cDNA sequence for clone<br>
#19138.1.<br>
#19138.2.<br>
#19140.1.<br>
#19140.2.<br>
#19145.1.<br>
#19145.2.<br>
#19153.1.<br>
#19153.2.<br>
#19161.1.<br>
#19161.2.<br>
SEQ ID NO: 156 is a second determined cDNA sequence for clone<br>
SEQ ID NO: 157 is the determined cDNA sequence for clone #19139.<br>
SEQ ID NO: 158 is a first determined cDNA sequence for clone<br>
SEQ ID NO: 159 is a second determined cDNA sequence for clone<br>
SEQ ID NO: 160 is the determined cDNA sequence for clone #19141.<br>
SEQ ID NO: 161 is the determined cDNA sequence for clone #19143.<br>
SEQ ID NO: 162 is the determined cDNA sequence for clone #19144.<br>
SEQ ID NO: 163 is a first determined cDNA sequence for clone<br>
SEQ ID NO: 164 is a second determined cDNA sequence for clone<br>
SEQ ID NO: 165 is the determined cDNA sequence for clone #19146.<br>
SEQ ID NO: 166 is the determined cDNA sequence for clone #19149.1.<br>
SEQ ID NO: 167 is the determined cDNA sequence for clone #19152.<br>
SEQ ID NO: 168 is a first determined cDNA sequence for clone<br>
SEQ ID NO: 169 is a second determined cDNA sequence for clone<br>
SEQ ID NO: 170 is the determined cDNA sequence for clone #19155.<br>
SEQ ID NO: 171 is the determined cDNA sequence for clone #19157.<br>
SEQ ID NO: 172 is the determined cDNA sequence for clone #19159.<br>
SEQ ID NO: 173 is the determined cDNA sequence for clone #19160.<br>
SEQ ID NO: 174 is a first detennined cDNA sequence for clone<br>
SEQ ID NO: 175 is a second determined cDNA sequence for clone<br>
SEQ ID NO: 176 is the determined cDNA sequence for clone #19162.1.<br>
SEQ ID NO: 177 is the determined cDNA sequence for clone #19166.<br>
#19173.1.<br>
#19173.2.<br>
#19182.1.<br>
#19182.2.<br>
#19194.1.<br>
#19194.2.<br>
SEQ ID NO: 178 is the determined cDNA sequence for clone #19169.<br>
SEQ ID NO: 179 is the determined cDNA sequence for clone #19171.<br>
SEQ ID NO: 180 is a first determined cDNA sequence for clone<br>
SEQ ID NO: 181 is a second determined cDNA sequence for clone<br>
SEQ ID NO: 182 is the determined cDNA sequence for clone #19174.1.<br>
SEQ ID NO: 183 is the determined cDNA sequence for clone #19175.<br>
SEQ ID NO: 184 is the determined cDNA sequence for clone #19177.<br>
SEQ ID NO: 185 is the determined cDNA sequence for clone #19178.<br>
SEQ ID NO: 186 is the determined cDNA sequence for clone #19179.1.<br>
SEQ ID NO: 187 is the determined cDNA sequence for clone #19179.2.<br>
SEQ ID NO: 188 is the determined cDNA sequence for clone #19180.<br>
SEQ ID NO: 189 is a first determined cDNA sequence for clone<br>
SEQ ID NO: 190 is a second determined cDNA sequence for clone<br>
SEQ ID NO: 191 is the determined cDNA sequence for clone #19183.1.<br>
SEQ ID NO: 192 is the determined cDNA sequence for clone #19185.1.<br>
SEQ ID NO: 193 is the determined cDNA sequence for clone #19187.<br>
SEQ ID NO: 194 is the determined cDNA sequence for clone #19188.<br>
SEQ ID NO: 195 is the determined cDNA sequence for clone #19190,<br>
SEQ ID NO: 196 is the determined cDNA sequence for clone #19191.<br>
SEQ ID NO: 197 is the determined cDNA sequence for clone #19192.<br>
SEQ ID NO: 198 is the determined cDNA sequence for clone #19193.<br>
SEQ ID NO: 199 is a first determined cDNA sequence for clone<br>
SEQ ID NO: 200 is a second determined cDNA sequence for clone<br>
SEQ ID NO: 201 is the determined cDNA sequence for clone #19197.<br>
SEQ ID NO: 202 is a first determined cDNA sequence for clone<br>
#19200.1.<br>
SEQ ID NO: 203 is a second determined cDNA sequence for clone<br>
#19200.2.<br>
SEQ ID NO: 204 is the determined cDNA sequence for clone #19202.<br>
SEQ ID NO: 205 is a first determined cDNA sequence for clone<br>
#19204.1.<br>
SEQ ED NO: 206 is a second determined cDNA sequence for clone<br>
#19204.2.<br>
SEQ ID NO: 207 is the determined cDNA sequence for clone #19205.<br>
SEQ ID NO: 208 is a first determined cDNA sequence for clone<br>
#19206.1.<br>
SEQ ID NO: 209 is a second determined cDNA sequence for clone<br>
#19206.2.<br>
SEQ ID NO: 210 is the determined cDNA sequence for clone #19207.<br>
SEQ ID NO: 211 is the determined cDNA sequence for clone #19208.<br>
SEQ ID NO: 212 is a first determined cDNA sequence for clone<br>
#19211.1.<br>
SEQ ID NO: 213 is a second determined cDNA sequence for clone<br>
#19211.2.<br>
SEQ ID NO: 214 is a first determined cDNA sequence for clone<br>
#19214.1.<br>
SEQ ID NO: 215 is a second determined cDNA sequence for clone<br>
#19214.2.<br>
SEQ ID NO: 216 is the determined cDNA sequence for clone #19215.<br>
SEQ ID NO: 217 is a first determined cDNA sequence for clone # 19217.<br>
2.<br>
SEQ ID NO: 218 is a second determined cDNA sequence for clone<br>
#19217.2.<br>
SEQ ID NO: 219 is a first determined cDNA sequence for clone<br>
#19218.1.<br>
SEQ ED NO: 248 is the determined cDNA sequence for clone #24851.<br>
SEQ ID NO: 249 is the determined cDNA sequence for clone #24852.<br>
SEQ ID NO: 250 is the determined cDNA sequence for clone #24853.<br>
SEQ ID NO: 251 is the determined cDNA sequence for clone #24854.<br>
SEQ ID NO: 252 is the determined cDNA sequence for clone #24855.<br>
SEQ ID NO: 253 is the determined cDNA sequence for clone #24860.<br>
SEQ ID NO: 254 is the determined cDNA sequence for clone #24864.<br>
SEQ ID NO: 255 is the determined cDNA sequence for clone #24866.<br>
SEQ ID NO: 256 is the determined cDNA sequence for clone #24867.<br>
SEQ ID NO: 257 is the determined cDNA sequence for clone #24868.<br>
SEQ ID NO: 258 is the determined cDNA sequence for clone #24869.<br>
SEQ ID NO: 259 is the determined cDNA sequence for clone #24870.<br>
SEQ ID NO: 260 is the determined cDNA sequence for clone #24872.<br>
SEQ ID NO: 261 is the determined cDNA sequence for clone #24873.<br>
SEQ ID NO: 262 is the determined cDNA sequence for clone #24875.<br>
SEQ ID NO: 263 is the determined cDNA sequence for clone #24882.<br>
SEQ ID NO: 264 is the determined cDNA sequence for clone #24885.<br>
SEQ ID NO: 265 is the determined cDNA sequence for clone #24886.<br>
SEQ ID NO: 266 is the determined cDNA sequence for clone #24887.<br>
SEQ ID NO: 267 is the determined cDNA sequence for clone #24888.<br>
SEQ ID NO: 268 is the determined cDNA sequence for clone #24890.<br>
SEQ ID NO: 269 is the determined cDNA sequence for clone #24896.<br>
SEQ ID NO: 270 is the determined cDNA sequence for clone #24897.<br>
SEQ ID NO: 271 is the determined cDNA sequence for clone #24899.<br>
SEQ ID NO: 272 is the determined cDNA sequence for clone #24901.<br>
SEQ ID NO: 273 is the determined cDNA sequence for clone #24902.<br>
SEQ ID NO: 274 is the determined cDNA sequence for clone #24906.<br>
SEQ ID NO: 275 is the determined cDNA sequence for clone #24912.<br>
SEQ ID NO: 276 is the determined cDNA sequence for clone #24913.<br>
SEQ ID NO: 277 is the determined cDNA sequence for clone #24920.<br>
SEQ ID NO: 278 is the determined cDNA sequence for clone #24927.<br>
#19218.2.<br>
#19220.1.<br>
#19220.2.<br>
SEQ ID NO: 220 is a second determined cDNA sequence for clone<br>
SEQ ID NO: 221 is a first determined cDNA sequence for clone<br>
SEQ ID NO: 222 is a second determined cDNA sequence for clone<br>
SEQ ID NO: 223 is the determined cDNA sequence for clone #22015.<br>
SEQ ID NO: 224 is the determined cDNA sequence for clone #22017.<br>
SEQ ID NO: 225 is the determined cDNA sequence for clone #22019.<br>
SEQ ID NO: 226 is the determined cDNA sequence for clone #22020.<br>
SEQ ID NO: 227 is the determined cDNA sequence for clone #22023.<br>
SEQ ID NO: 228 is the determined cDNA sequence for clone #22026.<br>
SEQ ID NO: 229 is the determined cDNA sequence for clone #22027.<br>
SEQ ID NO: 230 is the determined cDNA sequence for clone #22028.<br>
SEQ ID NO: 231 is the determined cDNA sequence for clone #22032.<br>
SEQ ID NO: 232 is the determined cDNA sequence for clone #22037.<br>
SEQ ID NO: 233 is the determined cDNA sequence for clone #22045.<br>
SEQ ID NO: 234 is the determined cDNA sequence for clone #22048.<br>
SEQ ID NO: 235 is the determined cDNA sequence for clone #22050.<br>
SEQ ID NO: 236 is the determined cDNA sequence for clone #22052.<br>
SEQ ID NO: 237 is the determined cDNA sequence for clone #22053.<br>
SEQ ID NO: 238 is the determined cDNA sequence for clone #22057.<br>
SEQ ID NO: 239 is the determined cDNA sequence for clone #22066.<br>
SEQ ID NO: 240 is the determined cDNA sequence for clone #22077.<br>
SEQ ID NO: 241 is the determined cDNA sequence for clone #22085.<br>
SEQ ID NO: 242 is the determined cDNA sequence for clone #22105.<br>
SEQ ID NO: 243 is the determined cDNA sequence for clone #22108.<br>
SEQ ID NO: 244 is the determined cDNA sequence for clone #22109.<br>
SEQ ID NO: 245 is the determined cDNA sequence for clone #24842.<br>
SEQ ID NO: 246 is the determined cDNA sequence for clone #24843.<br>
SEQ ID NO: 247 is the determined cDNA sequence for clone #24845.<br>
SEQ ID NO: 279 is the determined cDNA sequence for clone #24930.<br>
SEQ ID NO: 280 is the determined cDNA sequence for clone #26938.<br>
SEQ ID NO: 281 is the determined cDNA sequence for clone #26939.<br>
SEQ ID NO: 282 is the determined cDNA sequence for clone #26943.<br>
SEQ ID NO: 283 is the determined cDNA sequence for clone #26948.<br>
SEQ ID NO: 284 is the determined cDNA sequence for clone #26951.<br>
SEQ ID NO: 285 is the determined cDNA sequence for clone #26955.<br>
SEQ ID NO: 286 is the determined cDNA sequence for clone #26956.<br>
SEQ ID NO: 287 is the determined cDNA sequence for clone #26959.<br>
SEQ ID NO: 288 is the determined cDNA sequence for clone #26961.<br>
SEQ ID NO: 289 is the determined cDNA sequence for clone #26962.<br>
SEQ ID NO: 290 is the determined cDNA sequence for clone #26964.<br>
SEQ ID NO: 291 is the determined cDNA sequence for clone #26966.<br>
SEQ ID NO: 292 is the determined cDNA sequence for clone #26968.<br>
SEQ ID NO: 293 is the determined cDNA sequence for clone #26972.<br>
SEQ ID NO: 294 is the determined cDNA sequence for clone #26973.<br>
SEQ ID NO: 295 is the determined cDNA sequence for clone #26974.<br>
SEQ ID NO: 296 is the determined cDNA sequence for clone #26976.<br>
SEQ ID NO: 297 is the determined cDNA sequence for clone #26977.<br>
SEQ ID NO: 298 is the determined cDNA sequence for clone #26979.<br>
SEQ ID NO: 299 is the determined cDNA sequence for clone #26980.<br>
SEQ ID NO: 300 is the determined cDNA sequence for clone #26981.<br>
SEQ ID NO: 301 is the determined cDNA sequence for clone #26984.<br>
SEQ ID NO: 302 is the determined cDNA sequence for clone #26985.<br>
SEQ ID NO: 303 is the determined cDNA sequence for clone #26986.<br>
SEQ ID NO: 304 is the determined cDNA sequence for clone #26993.<br>
SEQ ID NO: 305 is the determined cDNA sequence for clone #26994.<br>
SEQ ID NO: 306 is the determined cDNA sequence for clone #26995.<br>
SEQ ID NO: 307 is the determined cDNA sequence for clone #27003.<br>
SEQ ID NO: 308 is the determined cDNA sequence for clone #27005.<br>
SEQ ID NO: 309 is the determined cDNA sequence for clone #27010.<br>
SEQ ID NO: 310 is the determined cDNA sequence for clone #27011.<br>
SEQ ID NO: 311 is the determined cDNA sequence for clone #27013.<br>
SEQ ID NO: 312 is the determined cDNA sequence for clone #27016<br>
SEQ ID NO: 313 is the determined cDNA sequence for clone #27017.<br>
SEQ ID NO: 314 is the determined cDNA sequence for clone #27019.<br>
SEQ ID NO: 315 is the determined cDNA sequence for clone #27028.<br>
SEQ ID NO: 316 is the full-length cDNA sequence for clone #19060.<br>
SEQ ID NO: 317 is the full-length cDNA sequence for clone #18964.<br>
SEQ ID NO: 318 is the full-length cDNA sequence for clone #18929.<br>
SEQ ID NO: 319 is the full-length cDNA sequence for clone #18991.<br>
SEQ ID NO: 320 is the full-length cDNA sequence for clone #18996.<br>
SEQ ID NO: 321 is the full-length cDNA sequence for clone # 18966.<br>
SEQ ID NO: 322 is the full-length cDNA sequence for clone #18951.<br>
SEQ ID NO: 323 is the full-length cDNA sequence for clone #18973<br>
(also known as L516S).<br>
SEQ ID NO: 324 is the amino acid sequence for clone #19060.<br>
SEQ ID NO: 325 is the amino acid sequence for clone #19063.<br>
SEQ ID NO: 326 is the amino acid sequence for clone #19077.<br>
SEQ ID NO: 327 is the amino acid sequence for clone #19110.<br>
SEQ ID NO: 328 is the amino acid sequence for clone #19122.<br>
SEQ ID NO: 329 is the amino acid sequence for clone #19118.<br>
SEQ ID NO: 330 is the amino acid sequence for clone #19080.<br>
SEQ ID NO: 331 is the ammo acid sequence for clone #19127.<br>
SEQ ID NO: 332 is the amino acid sequence for clone #19117.<br>
SEQ ID NO: 333 is the amino acid sequence for clone #19095, also<br>
referred to L549S.<br>
SEQ ID NO: 334 is the amino acid sequence for clone #18964.<br>
SEQ ID NO: 335 is the amino acid sequence for clone #18929.<br>
SEQ ID NO: 336 is the amino acid sequence for clone #18991.<br>
SEQ ID NO: 337 is the amino acid sequence for clone #18996.<br>
SEQ ID NO: 338 is the amino acid sequence for clone #18966.<br>
SEQ ID NO: 339 is the ammo acid sequence for clone #18951.<br>
SEQ ID NO: 340 is the amino acid sequence for clone #18973.<br>
SEQ ID NO: 341 is the determined cDNA sequence for clone 26461.<br>
SEQ ID NO: 342 is the determined cDNA sequence for clone 26462.<br>
SEQ ID NO: 343 is the determined cDNA sequence for clone 26463.<br>
SEQ ID NO: 344 is the determined cDNA sequence for clone 26464.<br>
SEQ ID NO: 345 is the determined cDNA sequence for clone 26465.<br>
SEQ ID NO: 346 is the determined cDNA sequence for clone 26466.<br>
SEQ ID NO: 347 is the determined cDNA sequence for clone 26467.<br>
SEQ ID NO: 348 is the determined cDNA sequence for clone 26468.<br>
SEQ ID NO: 349 is the determined cDNA sequence for clone 26469.<br>
SEQ ID NO: 350 is the determined cDNA sequence for clone 26470.<br>
SEQ ID NO: 351 is the determined cDNA sequence for clone 26471.<br>
SEQ ID NO: 352 is the determined cDNA sequence for clone 26472.<br>
SEQ ID NO: 353 is the determined cDNA sequence for clone 26474.<br>
SEQ ID NO: 354 is the determined cDNA sequence for clone 26475.<br>
SEQ ID NO: 355 is the determined cDNA sequence for clone 26476.<br>
SEQ ID NO: 356 is the determined cDNA sequence for clone 26477.<br>
SEQ ID NO: 357 is the determined cDNA sequence for clone 26478.<br>
SEQ ID NO: 358 is the determined cDNA sequence for clone 26479.<br>
SEQ ID NO: 359 is the determined cDNA sequence for clone 26480.<br>
SEQ ID NO: 360 is the determined cDNA sequence for clone 26481.<br>
SEQ ID NO: 361 is the determined cDNA sequence for clone 26482<br>
SEQ ID NO: 362 is the determined cDNA sequence for clone 26483.<br>
SEQ ID NO: 363 is the determined cDNA sequence for clone 26484.<br>
SEQ ID NO: 364 is the determined cDNA sequence for clone 26485.<br>
SEQ ID NO: 365 is the determined cDNA sequence for clone 26486.<br>
SEQ ID NO: 366 is the determined cDNA sequence for clone 26487.<br>
SEQ ID NO: 367 is the determined cDNA sequence for clone 26488.<br>
SEQ ID NO: 368 is the determined cDNA sequence for clone 26489.<br>
SEQ ID NO: 369 is the determined cDNA sequence for clone 26490.<br>
SEQ ID NO: 370 is the detennined cDNA sequence for clone 26491.<br>
SEQ ID NO: 371 is the determined cDNA sequence for done 26492.<br>
SEQ ID NO: 372 is the determined cDNA sequence for clone 26493.<br>
SEQ ID NO: 373 is the detennined cDNA sequence for clone 26494.<br>
SEQ ID NO: 374 is the determined cDNA sequence for clone 26495.<br>
SEQ ID NO: 375 is the determined cDNA sequence for clone 26496.<br>
SEQ ID NO: 376 is the determined cDNA sequence for clone 26497.<br>
SEQ ID NO: 377 is the determined cDNA sequence for clone 26498.<br>
SEQ ID NO: 378 is the determined cDNA sequence for clone 26499.<br>
SEQ ID NO: 379 is the determined cDNA sequence for clone 26500.<br>
SEQ ID NO: 380 is the determined cDNA sequence for clone 26501.<br>
SEQ ID NO: 381 is the determined cDNA sequence for clone 26502.<br>
SEQ ID NO: 382 is the determined cDNA sequence for clone 26503.<br>
SEQ ID NO: 383 is the detennined cDNA sequence for clone 26504.<br>
SEQ ID NO: 384 is the detennined cDNA sequence for clone 26505.<br>
SEQ ID NO: 385 is the determined cDNA sequence for clone 26506.<br>
SEQ ID NO: 386 is the determined cDNA sequence for clone 26507.<br>
SEQ ID NO: 387 is the determined cDNA sequence for clone 26508.<br>
SEQ ID NO: 388 is the determined cDNA sequence for clone 26509.<br>
SEQ ID NO: 389 is the determined cDNA sequence for clone 26511.<br>
SEQ ID NO: 390 is the detennined cDNA sequence for clone 26513.<br>
SEQ ID NO: 391 is the determined cDNA sequence for clone 26514.<br>
SEQ ID NO: 392 is the determined cDNA sequence for clone 26515.<br>
SEQ ID NO: 393 is the determined cDNA sequence for clone 26516.<br>
SEQ ID NO: 394 is the determined cDNA sequence for clone 26517.<br>
SEQ ID NO: 395 is the determined cDNA sequence for clone 26518.<br>
SEQ ID NO: 396 is the determined cDNA sequence for clone 26519.<br>
SEQ ID NO: 397 is the determined cDNA sequence for clone 26520.<br>
SEQ ID NO: 398 is the determined cDNA sequence for clone 26521.<br>
SEQ ID NO: 399 is the determined cDNA sequence for clone 26522.<br>
SEQ ID NO: 400 is the detennined cDNA sequence for clone 26523.<br>
SEQ ID NO: 401 is the determined cDNA sequence for clone 26524.<br>
SEQ ID NO: 402 is the determined cDNA sequence for done 26526.<br>
SEQ ID NO: 403 is the determined cDNA sequence for clone 26527.<br>
SEQ ID NO: 404 is the determined cDNA sequence for clone 26528.<br>
SEQ ID NO: 405 is the determined cDNA sequence for clone 26529.<br>
SEQ ID NO: 406 is the determined cDNA sequence for clone 26530.<br>
SEQ ID NO: 407 is the determined cDNA sequence for clone 26532.<br>
SEQ ID NO: 408 is the determined cDNA sequence for clone 26533.<br>
SEQ ID NO: 409 is the determined cDNA sequence for clone 26534.<br>
SEQ ID NO: 410 is the determined cDNA sequence for clone 26535.<br>
SEQ ID NO: 411 is the determined cDNA sequence for clone 26536.<br>
SEQ ID NO: 412 is the determined cDNA sequence for clone 26537.<br>
SEQ ID NO: 413 is the determined cDNA sequence for clone 26538.<br>
SEQ ID NO: 414 is the determined cDNA sequence for clone 26540.<br>
SEQ ID NO: 415 is the determined cDNA sequence for clone 26541.<br>
SEQ ID NO: 416 is the determined cDNA sequence for clone 26542.<br>
SEQ ID NO: 417 is the determined cDNA sequence for clone 26543.<br>
SEQ ID NO: 418 is the determined cDNA sequence for clone 26544.<br>
SEQ ID NO: 419 is the determined cDNA sequence for clone 26546.<br>
SEQ ID NO: 420 is the determined cDNA sequence for clone 26547.<br>
SEQ ID NO: 421 is the determined cDNA sequence for clone 26548.<br>
SEQ ID NO: 422 is the determined cDNA sequence for clone 26549.<br>
SEQ ID NO: 423 is the determined cDNA sequence for clone 26550.<br>
SEQ ID NO: 424 is the determined cDNA sequence for clone 26551.<br>
SEQ ID NO: 425 is the determined cDNA sequence for clone 26552.<br>
SEQ ID NO: 426 is the determined cDNA sequence for clone 26553.<br>
SEQ ID NO: 427 is the determined cDNA sequence for clone 26554.<br>
SEQ ID NO: 428 is the determined cDNA sequence for clone 26556.<br>
SEQ ID NO: 429 is the determined cDNA sequence for clone 26557.<br>
SEQ ID NO: 430 is the determined cDNA sequence for clone 27631.<br>
SEQ ID NO: 431 is the determined cDNA sequence for clone 27632.<br>
SEQ ID NO: 432 is the determined cDNA sequence for clone 27633.<br>
SEQ ID NO: 433 is the determined cDNA sequence for clone 27635.<br>
SEQ ID NO: 434 is the determined cDNA sequence for clone 27636.<br>
SEQ ID NO: 435 is the determined cDNA sequence for clone 27637.<br>
SEQ ID NO: 436 is the determined cDNA sequence for clone 27638.<br>
SEQ ID NO: 437 is the determined cDNA sequence for clone 27639.<br>
SEQ ID NO: 438 is the determined cDNA sequence for clone 27640.<br>
SEQ ID NO: 439 is the determined cDNA sequence for clone 27641.<br>
SEQ ID NO: 440 is the determined cDNA sequence for clone 27642.<br>
SEQ ID NO: 441 is the determined cDNA sequence for clone 27644.<br>
SEQ ID NO: 442 is the determined cDNA sequence for clone 27646.<br>
SEQ ID NO: 443 is the determined cDNA sequence for clone 27647.<br>
SEQ ID NO: 444 is the determined cDNA sequence for clone 27649.<br>
SEQ ID NO: 445 is the determined cDNA sequence for clone 27650.<br>
SEQ ID NO: 446 is the determined cDNA sequence for clone 27651.<br>
SEQ ID NO: 447 is the determined cDNA sequence for clone 27652.<br>
SEQ ID NO: 448 is the determined cDNA sequence for clone 27654.<br>
SEQ ID NO: 449 is the determined cDNA sequence for clone 27655.<br>
SEQ ID NO: 450 is the determined cDNA sequence for clone 27657.<br>
SEQ ID NO: 451 is the determined cDNA sequence for clone 27659.<br>
SEQ ID NO: 452 is the determined cDNA sequence for clone 27665.<br>
SEQ ID NO: 453 is the determined cDNA sequence for clone 27666.<br>
SEQ ID NO: 454 is the determined cDNA sequence for clone 27668.<br>
SEQ ID NO: 455 is the determined cDNA sequence for clone 27670.<br>
SEQ ID NO: 456 is the determined cDNA sequence for clone 27671.<br>
SEQ ID NO: 457 is the determined cDNA sequence for clone 27672.<br>
SEQ ID NO: 458 is the determined cDNA sequence for clone 27674.<br>
SEQ ID NO: 459 is the determined cDNA sequence for clone 27677.<br>
SEQ ID NO: 4601s the determined cDNA sequence for clone 27681.<br>
SEQ ID NO: 461 is the determined cDNA sequence for clone 27682.<br>
SEQ ID NO: 462 is the determined cDNA sequence for clone 27683.<br>
SEQ ID NO: 463 is the determined cDNA sequence for clone 27686.<br>
SEQ ID NO: 464 is the determined cDNA sequence for clone 27688.<br>
SEQ ID NO: 465 is the determined cDNA sequence for clone 27689.<br>
SEQ ID NO: 466 is the determined cDNA sequence for clone 27690.<br>
SEQ ID NO: 467 is the determined cDNA sequence for clone 27693.<br>
SEQ ID NO: 468 is the determined cDNA sequence for clone 27699.<br>
SEQ ID NO: 469 is the determined cDNA sequence for clone 27700.<br>
SEQ ID NO: 470 is the determined cDNA sequence for clone 27702.<br>
SEQ ID NO: 471 is the determined cDNA sequence for clone 27705.<br>
SEQ ID NO: 472 is the determined cDNA sequence for clone 27706.<br>
SEQ ID NO: 473 is the determined cDNA sequence for clone 27707.<br>
SEQ ID NO: 474 is the determined cDNA sequence for clone 27708.<br>
SEQ ID NO: 475 is the determined cDNA sequence for clone 27709.<br>
SEQ ID NO: 476 is the determined cDNA sequence for clone 27710.<br>
SEQ ID NO: 477 is the determined cDNA sequence for clone 27711.<br>
SEQ ID NO: 478 is the determined cDNA sequence for clone 27712.<br>
SEQ ID NO: 479 is the determined cDNA sequence for clone 27713-<br>
SEQ ID NO: 480 is the determined cDNA sequence for clone 27714.<br>
SEQ ID NO: 481 is the determined cDNA sequence for clone 27715.<br>
SEQ ID NO: 482 is the determined cDNA sequence for clone 27716.<br>
SEQ ID NO: 483 is the determined cDNA sequence for clone 27717.<br>
SEQ ED NO: 484 is the determined cDNA sequence for clone 27718.<br>
SEQ ID NO: 485 is the determined cDNA sequence for clone 27719.<br>
SEQ ID NO: 486 is the determined cDNA sequence for clone 27720.<br>
SEQ ID NO: 487 is the determined cDNA sequence for clone 27722.<br>
SEQ ED NO: 488 is the determined cDNA sequence for clone 27723.<br>
SEQ ID NO: 489 is the determined cDNA sequence for clone 27724.<br>
SEQ ID NO: 490 is the determined cDNA sequence for clone 27726.<br>
SEQ ID NO: 491 is the determined cDNA sequence for clone 25015.<br>
SEQ ID NO: 492 is the determined cDNA sequence for clone 25016.<br>
SEQ ID NO: 493 is the determined cDNA sequence for clone 25017.<br>
SEQ ID NO: 494 is the determined cDNA sequence for clone 25018<br>
SEQ ID NO: 495 is the determined cDNA sequence for done 25030.<br>
SEQ ID NO: 496 is the determined cDNA sequence for clone 25033.<br>
SEQ ID NO: 497 is the determined cDNA sequence for clone 25034.<br>
SEQ ID NO: 498 is the determined cDNA sequence for clone 25035.<br>
SEQ ID NO: 499 is the determined cDNA sequence for clone 25036.<br>
SEQ ID NO: 500 is the determined cDNA sequence for clone 25037.<br>
SEQ ID NO: 501 is the determined cDNA sequence for clone 25038.<br>
SEQ ID NO: 502 is the determined cDNA sequence for clone 25039.<br>
SEQ ID NO: 503 is the determined cDNA sequence for clone 25040.<br>
SEQ ID NO: 504 is the determined cDNA sequence for clone 25042.<br>
SEQ ID NO: 505 is the determined cDNA sequence for clone 25043.<br>
SEQ ID NO: 506 is the determined cDNA sequence for clone 25044.<br>
SEQ ID NO: 507 is the determined cDNA sequence for clone 25045.<br>
SEQ ID NO: 508 is the determined cDNA sequence for clone 25047.<br>
SEQ ID NO: 509 is the determined cDNA sequence for clone 25048.<br>
SEQ ID NO: 510 is the determined cDNA sequence for clone 25049.<br>
SEQ ID NO: 511 is the determined cDNA sequence for clone 25185.<br>
SEQ ID NO: 512 is the determined cDNA sequence for clone 25186.<br>
SEQ ID NO: 513 is the determined cDNA sequence for clone 25187.<br>
SEQ ID NO: 514 is the determined cDNA sequence for clone 25188.<br>
SEQ ID NO: 515 is the determined cDNA sequence for clone 25189.<br>
SEQ ID NO: 516 is the determined cDNA sequence for clone 25190.<br>
SEQ ID NO: 517 is the determined cDNA sequence for clone 25193.<br>
SEQ ID NO: 518 is the determined cDNA sequence for clone 25194.<br>
SEQ ID NO: 519 is the determined cDNA sequence for clone 25196.<br>
SEQ ID NO: 520 is the determined cDNA sequence for clone 25198.<br>
SEQ ID NO: 521 is the determined cDNA sequence for clone 25199.<br>
SEQ ID NO: 522 is the determined cDNA sequence for clone 25200.<br>
SEQ ID NO: 523 is the determined cDNA sequence for clone 25202.<br>
SEQ ID NO: 524 is the determined cDNA sequence for clone 25364.<br>
SEQ ID NO: 525 is the determined cDNA sequence for clone 25366.<br>
SEQ ID NO: 526 is the determined cDNA sequence for done 25367.<br>
SEQ ID NO: 527 is the determined cDNA sequence for clone 25368.<br>
SEQ ID NO: 528 is the determined cDNA sequence for clone 25369.<br>
SEQ ID NO: 529 is the determined cDNA sequence for clone 25370.<br>
SEQ ID NO: 530 is the determined cDNA sequence for clone 25371.<br>
SEQ ID NO: 531 is the determined cDNA sequence for clone 25372.<br>
SEQ ID NO: 532 is the determined cDNA sequence for clone 25373.<br>
SEQ ID NO: 533 is the determined cDNA sequence-for clone 25374.<br>
SEQ ID NO: 534 is the determined cDNA sequence for clone 25376.<br>
SEQ ID NO: 535 is the determined cDNA sequence for clone 25377.<br>
SEQ ID NO: 536 is the determined cDNA sequence for clone 25378.<br>
SEQ ID NO: 537 is the determined cDNA sequence for clone 25379.<br>
SEQ ID NO: 538 is the determined cDNA sequence for clone 25380.<br>
SEQ ID NO: 539 is the determined cDNA sequence for clone 25381.<br>
SEQ ID NO: 540 is the determined cDNA sequence for clone 25382.<br>
SEQ ID NO: 541 is the determined cDNA sequence for clone 25383.<br>
SEQ ID NO: 542 is the determined cDNA sequence for clone 25385.<br>
SEQ ID NO: 543 is the determined cDNA sequence for clone 25386.<br>
SEQ ID NO: 544 is the determined cDNA sequence for clone 25387.<br>
SEQ ID NO: 545 is the determined cDNA sequence for clone 26013.<br>
SEQ ID NO: 546 is the determined cDNA sequence for clone 26014.<br>
SEQ ID NO: 547 is the determined cDNA sequence for clone 26016.<br>
SEQ ID NO: 548 is the determined cDNA sequence for clone 26017.<br>
SEQ ID NO: 549 is the determined cDNA sequence for clone 26018.<br>
SEQ ID NO: 550 is the determined cDNA sequence for clone 26019.<br>
SEQ ID NO: 551 is the determined cDNA sequence for clone 26020.<br>
SEQ ID NO: 552 is the determined cDNA sequence for clone 26021.<br>
SEQ ID NO: 553 is the determined cDNA sequence for clone 26022.<br>
SEQ ID NO: 554 is the determined cDNA sequence for clone 26027.<br>
SEQ ID NO: 555 is the determined cDNA sequence for clone 26197.<br>
SEQ ID NO: 556 is the determined cDNA sequence for clone 26199.<br>
SEQ ID NO: 557 is the determined cDNA sequence for clone 26201.<br>
SEQ ID NO: 558 is the determined cDNA sequence for clone 26202.<br>
SEQ ID NO: 559 is the determined cDNA sequence for clone 26203.<br>
SEQ ID NO: 560 is the detennined cDNA sequence for clone 26204.<br>
.SEQ ID NO: 561 is the determined cDNA sequence for clone 26205.<br>
SEQ ID NO: 562 is the determined cDNA sequence for clone 26206.<br>
SEQ ID NO: 563 is the detennined cDNA sequence for clone 26208.<br>
SEQ ID NO: 564 is the determined cDNA sequence for clone 26211.<br>
SEQ ID NO: 565 is the determined cDNA sequence for clone 26212.<br>
; SEQ ID NO: 566 is the determined cDNA sequence for clone 26213.<br>
SEQ ID NO: 567 is the determined cDNA sequence for clone 26214.<br>
SEQ ID NO: 568 is the determined cDNA sequence for clone 26215.<br>
SEQ ID NO: 569 is the determined cDNA sequence for clone 26216.<br>
SEQ ID NO: 570 is the detennined cDNA sequence for clone 26217.<br>
SEQ ID NO: 571 is the determined cDNA sequence for clone 26218.<br>
SEQ ID NO: 572 is the determined cDNA sequence for clone 26219.<br>
SEQ ID NO: 573 is the determined cDNA sequence for clone 26220.<br>
SEQ ID NO: 574 is the determined cDNA sequence for clone 26221.<br>
SEQ ID NO: 575 is the determined cDNA sequence for clone 26224.<br>
SEQ ID NO: 576 is the determined cDNA sequence for clone 26225.<br>
SEQ ID NO: 577 is the detennined cDNA sequence for clone 26226.<br>
SEQ ID NO: 578 is the detennined cDNA sequence for clone 26227.<br>
SEQ ID NO: 579 is the detennined cDNA sequence for clone 26228.<br>
SEQ ID NO: 580 is the detennined cDNA sequence for clone 26230.<br>
SEQ ID NO: 581 is the detennined cDNA sequence for clone 26231.<br>
SEQ ID NO: 582 is the detennined cDNA sequence for clone 26234.<br>
SEQ ID NO: 583 is the determined cDNA sequence for clone 26236.<br>
SEQ ID NO: 584 is the detennined cDNA sequence for clone 26237.<br>
SEQ ID NO: 585 is the determined cDNA sequence for clone 26239.<br>
SEQ ID NO: 586 is the determined cDNA sequence for clone 26240.<br>
SEQ ID NO: 587 is the determined cDNA sequence for clone 26241.<br>
SEQ ID NO: 588 is the determined cDNA sequence for clone 26242.<br>
SEQ ID NO: 589 is the determined cDNA sequence for clone 26246.<br>
SEQ ID NO: 590 is the determined cDNA sequence for clone 26247.<br>
SEQ ID NO: 591 is the determined cDNA sequence for clone 26248.<br>
SEQ ID NO: 592 is the determined cDNA sequence for clone 26249.<br>
SEQ ID NO: 593 is the determined cDNA sequence for clone 26250.<br>
SEQ ID NO: 594 is the determined cDNA sequence for clone 26251.<br>
SEQ ID NO: 595 is the determined cDNA sequence for clone 26252.<br>
SEQ ID NO: 596 is the determined cDNA sequence for clone 26253.<br>
SEQ ID NO: 597 is the determined cDNA sequence for clone 26254.<br>
SEQ ID NO: 598 is the determined cDNA sequence for clone 26255.<br>
SEQ ID NO: 599 is the determined cDNA sequence for clone 26256.<br>
SEQ ID NO: 600 is the determined cDNA sequence for clone 26257.<br>
SEQ ID NO: 601 is the determined cDNA sequence for clone 26259.<br>
SEQ ID NO: 602 is the determined cDNA sequence for clone 26260.<br>
SEQ ID NO: 603 is the determined cDNA sequence for clone 26261.<br>
SEQ ID NO: 604 is the determined cDNA sequence for clone 26262.<br>
SEQ ID NO: 605 is the determined cDNA sequence for clone 26263.<br>
SEQ ID NO: 606 is the determined cDNA sequence for clone 26264.<br>
SEQ ID NO: 607 is the determined cDNA sequence for clone 26265.<br>
SEQ ID NO: 608 is the determined cDNA sequence for clone 26266.<br>
SEQ ID NO: 609 is the determined cDNA sequence for clone 26268.<br>
SEQ ID NO: 610 is the determined cDNA sequence for clone 26269.<br>
SEQ ID NO: 611 is the determined cDNA sequence for clone 26271.<br>
SEQ ID NO: 612 is the determined cDNA sequence for clone 26273.<br>
SEQ ID NO: 613 is the determined cDNA sequence for clone 26810.<br>
SEQ ID NO: 614 is the determined cDNA sequence for clone 26811.<br>
SEQ ID NO: 615 is the determined cDNA sequence for clone 26812.1.<br>
SEQ ID NO: 616 is the determined cDNA sequence for clone 26812.2.<br>
SEQ ID NO: 617 is the determined cDNA sequence for clone 26813.<br>
SEQ ID NO: 618 is the determined cDNA sequence for clone 26814.<br>
SEQ ID NO: 619 is the determined cDNA sequence for done 26815.<br>
SEQ ID NO: 620 is the determined cDNA sequence for clone 26816.<br>
SEQ ID NO: 621 is the detennined cDNA sequence for clone 26818.<br>
SEQ ID NO: 622 is the determined cDNA sequence for clone 26819.<br>
SEQ ID NO: 623 is the determined cDNA sequence for clone 26820.<br>
SEQ ID NO: 624 is the detennined cDNA sequence for clone 26821.<br>
SEQ ID NO: 625 is the determined cDNA sequence for clone 26822.<br>
SEQ ID NO: 626 is the determined cDNA sequence for clone 26824.<br>
SEQ ID NO: 627 is the determined cDNA sequence for clone.26825.<br>
SEQ ID NO: 628 is the determined cDNA sequence for clone 26826.<br>
SEQ ID NO: 629 is the determined cDNA sequence for clone 26827.<br>
SEQ ID NO: 630 is the determined cDNA sequence for clone 26829.<br>
SEQ ED NO: 631 is the determined cDNA sequence for clone 26830.<br>
SEQ ID NO: 632 is the determined cDNA sequence for clone 26831.<br>
SEQ ID NO: 633 is the detennined cDNA sequence for clone 26832.<br>
SEQ ID NO: 634 is the determined cDNA sequence for clone 26835.<br>
SEQ ID NO: 635 is the detennined cDNA sequence for clone 26836.<br>
SEQ ID NO: 636 is the determined cDNA sequence for clone 26837.<br>
SEQ ID NO: 637 is the determined cDNA sequence for clone 26839.<br>
SEQ ID NO: 638 is the detennined cDNA sequence for clone 26841.<br>
SEQ ID NO: 639 is the determined cDNA sequence for clone 26843.<br>
SEQ ID NO: 640 is the detennined cDNA sequence for clone 26844.<br>
SEQ ID NO: 641 is the determined cDNA sequence for clone 26845.<br>
SEQ ID NO: 642 is the determined cDNA sequence for clone 26846.<br>
SEQ ID NO: 643 is the determined cDNA sequence for clone 26847.<br>
SEQ ID NO: 644 is the determined cDNA sequence for clone 26848.<br>
SEQ ID NO: 645 is the determined cDNA sequence for clone 26849.<br>
SEQ ID NO: 646 is the detennined cDNA sequence for clone 26850.<br>
SEQ ID NO: 647 is the determined cDNA sequence for clone 26851.<br>
SEQ ID NO: 648 is the determined cDNA sequence for clone 26852.<br>
SEQ ID NO: 649 is the determined cDNA sequence for clone 26853.<br>
SEQ ID NO: 650 is the determined cDNA sequence for clone 26854.<br>
SEQ ID NO: 651 is the determined cDNA sequence for clone 26856.<br>
SEQ ID NO: 652 is the determined cDNA sequence for clone 26857.<br>
SEQ ID NO: 653 is the determined cDNA sequence for clone 26858.<br>
SEQ ID NO: 654 is the determined cDNA sequence for clone 26859.<br>
SEQ ID NO: 655 is the determined cDNA sequence for clone 26860.<br>
SEQ ID NO: 656 is the determined cDNA sequence for clone 26862.<br>
SEQ ID NO: 657 is the determined cDNA sequence for clone 26863.<br>
SEQ ID NO: 658 is the determined cDNA sequence for clone 26864.<br>
SEQ ID NO: 659 is the determined cDNA sequence for clone 26865.<br>
SEQ ID NO: 660 is the determined cDNA sequence for clone 26867.<br>
SEQ ID NO: 661 is the determined cDNA sequence for clone 26868.<br>
SEQ ID NO: 662 is the determined cDNA sequence for clone 26871.<br>
SEQ ID NO: 663 is the determined cDNA sequence for clone 26873.<br>
SEQ ID NO: 664 is the determined cDNA sequence for clone 26875.<br>
SEQ ID NO: 665 is the determined cDNA sequence for clone 26876.<br>
SEQ ID NO: 666 is the determined cDNA sequence for clone 26877.<br>
SEQ ID NO: 667 is the determined cDNA sequence for clone 26878.<br>
SEQ ID NO: 668 is the determined cDNA sequence for clone 26880.<br>
SEQ ID NO: 669 is the determined cDNA sequence for clone 26882.<br>
SEQ ID NO: 670 is the determined cDNA sequence for clone 26883.<br>
SEQ ID NO: 671 is the determined cDNA sequence for clone 26884.<br>
SEQ ID NO: 672 is the determined cDNA sequence for clone 26885.<br>
SEQ ID NO: 673 is the determined cDNA sequence for clone 26886.<br>
SEQ ID NO: 674 is the determined cDNA sequence for clone 26887.<br>
SEQ ID NO: 675 is the determined cDNA sequence for clone 26888.<br>
SEQ ID NO: 676 is the determined cDNA sequence for clone 26889.<br>
SEQ ID NO: 677 is the determined cDNA sequence for clone 26890.<br>
SEQ ID NO: 678 is the determined cDNA sequence for clone 26892.<br>
SEQ ID NO: 679 is the determined cDNA sequence for clone 26894.<br>
SEQ ID NO: 680 is the determined cDNA sequence for clone 26895.<br>
SEQ ID NO: 681 is the determined cDNA sequence for done 26897.<br>
SEQ ID NO: 682 is the determined cDNA sequence for clone 26898.<br>
SEQ ID NO: 683 is the determined cDNA sequence for clone 26899.<br>
SEQ ID NO: 684 is the determined cDNA sequence for clone 26900.<br>
SEQ ID NO: 685 is the determined cDNA sequence for clone 26901.<br>
SEQ ID NO: 686 is the determined cDNA sequence for clone 26903.<br>
SEQ ID NO: 687 is the determined cDNA sequence for clone 26905.<br>
SEQ ID NO: 688 is the detennined cDNA sequence for clone 26906.<br>
SEQ ID NO: 689 is the determined cDNA sequence for clone 26708.<br>
SEQ ID NO: 690 is the determined cDNA sequence for clone 26709.<br>
SEQ ID NO: 691 is the determined cDNA sequence for clone 26710.<br>
SEQ ID NO: 692 is the determined cDNA sequence for clone 26711.<br>
SEQ ID NO: 693 is the determined cDNA sequence for clone 26712.<br>
SEQ ID NO: 694 is the determined cDNA sequence for clone 26713.<br>
SEQ ID NO: 695 is the determined cDNA sequence for clone 26714.<br>
SEQ ID NO: 696 is the determined cDNA sequence for clone 26715.<br>
SEQ ID NO: 697 is the determined cDNA sequence for clone 26716.<br>
SEQ ID NO: 698 is the detennined cDNA sequence for clone 26717.<br>
SEQ ID NO: 699 is the determined cDNA sequence for clone 26718.<br>
SEQ ID NO: 700 is the detennined cDNA sequence for clone 26719.<br>
SEQ ID NO: 701 is the determined cDNA sequence for clone 26720.<br>
SEQ ID NO: 702 is the determined cDNA sequence for clone 26721.<br>
SEQ ID NO: 703 is the determined cDNA sequence for clone 26722.<br>
SEQ ID NO: 704 is the detennined cDNA sequence for clone 26723.<br>
SEQ ID NO: 705 is the determined cDNA sequence for clone 26724.<br>
. SEQ ID NO: 706 is the determined cDNA sequence for clone 26725.<br>
SEQ ID NO: 707 is the determined cDNA sequence for clone 26726.<br>
SEQ ID NO: 708 is the detennined cDNA sequence for clone 26727.<br>
SEQ ID NO: 709 is the determined cDNA sequence for clone 26728.<br>
SEQ ID NO: 710 is the detennined cDNA sequence for clone 26729.<br>
SEQ ID NO: 711 is the determined cDNA sequence for clone 26730.<br>
SEQ ID NO: 712 is the determined cDNA sequence for clone 26731.<br>
SEQ ID NO: 713 is the determined cDNA sequence for clone 26732.<br>
SEQ ID NO: 714 is the determined cDNA sequence for clone 26733.1.<br>
SEQ ID NO: 715 is the determined cDNA sequence for clone 26733.2.<br>
SEQ ID NO: 716 is the determined cDNA sequence for clone 26734.<br>
SEQ ID NO: 717 is the determined cDNA sequence for clone 26735.<br>
SEQ ID NO: 718 is the determined cDNA sequence for clone 26736.<br>
SEQ ID NO: 719 is the determined cDNA sequence for clone 26737.<br>
SEQ ID NO: 720 is the determined cDNA sequence for clone 26738.<br>
SEQ ID NO: 721 is the determined cDNAvSequence for clone 26739.<br>
SEQ ID NO: 722 is the determined cDNA sequence for clone 26741.<br>
SEQ ID NO: 723 is the determined cDNA sequence for clone 26742.<br>
SEQ ID NO: 724 is the determined cDNA sequence for clone 26743.<br>
SEQ ID NO: 725 is the determined cDNA sequence for clone 26744.<br>
SEQ ID NO: 726 is the determined cDNA sequence for clone 26745.<br>
SEQ ID NO: 727 is the determined cDNA sequence for clone 26746.<br>
SEQ ID NO: 728 is the determined cDNA sequence for clone 26747.<br>
SEQ ID NO: 729 is the determined cDNA sequence for clone 26748.<br>
SEQ ID NO: 730 is the determined cDNA sequence for clone 26749.<br>
SEQ ID NO: 731 is the determined cDNA sequence for clone 26750.<br>
SEQ ID NO: 732 is the determined cDNA sequence for clone 26751.<br>
SEQ ID NO: 733 is the determined cDNA sequence for clone 26752.<br>
SEQ ID NO: 734 is the determined cDNA sequence for clone 26753.<br>
SEQ ID NO: 735 is the determined cDNA sequence for clone 26754.<br>
SEQ ID NO: 736 is the determined cDNA sequence for clone 26755.<br>
SEQ ID NO: 737 is the determined cDNA sequence for clone 26756.<br>
SEQ ID NO: 738 is the determined cDNA sequence for clone 26757.<br>
SEQ ID NO: 739 is the determined cDNA sequence for clone 26758.<br>
SEQ ID NO: 740 is the determined cDNA sequence for clone 26759.<br>
SEQ ID NO: 741 is the determined cDNA sequence for clone 26760.<br>
SEQ ID NO: 742 is the determined cDNA sequence for clone 26761.<br>
SEQ ID NO: 743 is the determined cDNA sequence for clone 26762.<br>
SEQ ID NO: 744 is the determined cDNA sequence for clone 26763.<br>
SEQ ID NO: 745 is the determined cDNA sequence for clone 26764.<br>
SEQ ID NO: 746 is the determined cDNA sequence for clone 26765.<br>
SEQ ID NO: 747 is the determined cDNA sequence for clone 26766.<br>
SEQ ID NO: 748 is the determined cDNA sequence for clone 26767.<br>
SEQ ID NO: 749 is the determined cDNA sequence for clone 26768.<br>
SEQ ID NO: 750 is the determined cDNA sequence for clone 26769.<br>
SEQ ID NO: 751 is the determined cDNA sequence for clone 26770.<br>
SEQ ID NO: 752 is the determined cDNA sequence for clone 26771.<br>
SEQ ID NO: 753 is the determined cDNA sequence for clone 26772.<br>
SEQ ID NO: 754 is the determined cDNA sequence for clone 26773.<br>
SEQ ID NO: 755 is the determined cDNA sequence for clone 26774.<br>
SEQ ID NO: 756 is the determined cDNA sequence for clone 26775.<br>
SEQ ID NO: 757 is the determined cDNA sequence for clone 26776.<br>
SEQ ID NO: 758 is the determined cDNA sequence for clone 26777.<br>
SEQ ID NO: 759 is the determined cDNA sequence for clone 26778.<br>
SEQ ID NO: 760 is the determined cDNA sequence for clone 26779.<br>
SEQ ID NO: 761 is the determined cDNA sequence for clone 26781.<br>
SEQ ID NO: 762 is the determined cDNA sequence for clone 26782.<br>
SEQ ID NO: 763 is the determined cDNA sequence for clone 26783.<br>
SEQ ID NO: 764 is the determined cDNA sequence for clone 26784.<br>
SEQ ID NO: 765 is the determined cDNA sequence for clone 26785.<br>
SEQ ID NO: 766 is the determined cDNA sequence for clone 26786.<br>
SEQ ID NO: 767 is the determined cDNA sequence for clone 26787.<br>
SEQ ID NO: 768 is the determined cDNA sequence for clone 26788.<br>
SEQ ID NO: 769 is the determined cDNA sequence for clone 26790.<br>
SEQ ID NO: 770 is the determined cDNA sequence for clone 26791.<br>
SEQ ID NO: 771 is the determined cDNA sequence for clone 26792.<br>
SEQ ID NO: 772 is the determined cDNA sequence for clone 26793.<br>
SEQ ID NO: 773 is the determined cDNA sequence for clone 26794.<br>
SEQ ID NO: 774 is the determined cDNA sequence for done 26795.<br>
SEQ ID NO: 775 is the determined cDNA sequence for clone 26796.<br>
SEQ ID NO: 776 is the determined cDNA sequence for clone 26797.<br>
SEQ ID NO: 777 is the determined cDNA sequence for clone 26798.<br>
SEQ ID NO: 778 is the determined cDNA sequence for clone 26800.<br>
SEQ ID NO: 779 is the determined cDNA sequence for clone 26801.<br>
SEQ ID NO: 780 is the determined cDNA sequence for clone 26802.<br>
SEQ ID NO: 781 is the determined cDNA sequence for clone 26803.<br>
SEQ ID NO: 782 is the determined cDNA sequence for clone 26804.<br>
SEQ ID NO: 783 is the amino acid sequence for L773P.<br>
SEQ ID NO: 784 is the determined DNA sequence of the L773P<br>
expression construct.<br>
SEQ ID NO: 785 is the determined DNA sequence of the L773PA<br>
expression construct<br>
SEQ ID NO: 786 is a predicted amino acid sequence for L552S.<br>
SEQ ID NO: 787 is a predicted amino acid sequence for L840P.<br>
SEQ ID NO: 788 is the full-length cDNA sequence for L548S.<br>
SEQ ID NO: 789 is the amino acid sequence encoded by SEQ ID NO:<br>
788.<br>
SEQ ID NO: 790 is an extended cDNA sequence for L552S.<br>
SEQ ID NO: 791 is the predicted amino acid sequence encoded by the<br>
cDNA sequence of SEQ ID NO: 790.<br>
SEQ ID NO: 792 is the determined cDNA sequence for an isoform of<br>
L552S.<br>
SEQ ID NO: 793 is the predicted amino acid sequence encoded by SEQ<br>
ID NO: 792.<br>
SEQ ID NO: 794 is an extended cDNA sequence for L840P.<br>
SEQ ID NO: 795 is the predicted amino acid sequence encoded by SEQ<br>
DINO:794.<br>
SEQ ID NO: 796 is an extended cDNA sequence for L80IP.<br>
SEQ ID NO: 797 is a first predicted amino acid sequence encoded by<br>
SEQIDNO:796.<br>
SEQ ID NO: 798 is a second predicted amino acid sequence encoded by<br>
SEQ ID NO: 796.<br>
SEQ ID NO: 799 is a third predicted amino acid sequence encoded by<br>
SEQ ID NO: 796.<br>
SEQ ID NO: 800 is the determined full-length sequence for L844P.<br>
SEQ ID NO: 801 is the 5' consensus cDNA sequence for L551S.<br>
SEQ ID NO: 802 is the 3' consensus cDNA sequence for L551 S.<br>
SEQ ID NO: 803 is the cDNA sequence for STY8.<br>
SEQ ID NO: 804 is an extended cDNA sequence for L551 S.<br>
SEQ ID NO: 805 is the amino acid sequence for STY8.<br>
SEQ ID NO: 806 is the extended amino acid sequence for L551S.<br>
SEQ ID NO: 807 is the determined full-length cDNA sequence for<br>
L773P.<br>
SEQ ID NO: 808 is the full-length cDNA sequence of L552S.<br>
SEQ ID NO: 809 is the full-length amino acid sequence of L552S.<br>
SEQ ID NO: 810 is the determined cDNA sequence of clone 50989.<br>
SEQ ID NO: 811 is the determined cDNA sequence of clone 50990.<br>
SEQ ID NO: 812 is the determined cDNA sequence of clone 50992.<br>
SEQ ID NO: 813-824 are the determined cDNA sequences for clones<br>
isolated from lung tumor tissue.<br>
SEQ ID NO: 825 is the determined cDNA sequence for the full-length<br>
L55IS clone 54305.<br>
SEQ ID NO: 826 is the determined cDNA sequence for the full-length<br>
L55IS clone 54298.<br>
SEQ ID NO: 827 is the full-length amino acid sequence for L551S.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
As noted above, the present invention is generally directed to<br>
compositions and methods for using the compositions, for example in the therapy and<br>
diagnosis of cancer, such as lung cancer. Certain illustrative compositions described<br>
herein include lung tumor polypeptides, polynucleotides encoding such polypeptides,<br>
binding agents such as antibodies, antigen presenting cells (APCs) and/or immune<br>
system cells (e.g., T cells). A " lung tumor protein," as the term is used herein, refers<br>
generally to a protein that is expressed in lung tumor cells at a level that is at least two<br>
fold, and preferably at least five fold, greater than the level of expression in a normal<br>
tissue, as determined using a representative assay provided herein. Certain lung tumor<br>
proteins are tuinor proteins that react detectably (within an immunoassay, such as an<br>
ELISA or Western blot) with antisera of a patient afflicted with lung cancer.<br>
Therefore, in accordance with the above, and as described further below,<br>
the present invention provides illustrative polynucleotide compositions having<br>
sequences set forth in SEQ ID NO: 1, 11-13, 15, 20, 23-27, 29, 30, 33, 34, 39, 41, 43-<br>
46, 51, 52, 57, 58, 60, 62, 65-67, 69-71, 74, 76, 79, 80, 84, 86, 89-92, 95, 97, 98, 101,<br>
110, 111, 113-119,121-128,130-134,136, 138,139,141,143,146-151, 153, 154, 157-<br>
160, 162-164, 167-178,180, 181, 183, 186-190, 192,193,195-220, 224, 226-231, 234,<br>
236, 237, 240, 241, 244-246, 248, 254, 255, 261, 262, 266, 270, 275, 280, 282, 283,<br>
288, 289, 290, 292, 295, 301, 303, 304, 309, 311, 341-782, 784, 785, 790, 792, 794,<br>
796, 800-804, 807, 808 and 810-826, illustrative polypeptide compositions having<br>
amino acid sequences set forth in SEQ ID NO: 786, 787, 791, 793, 795, 797-799, 806, <br>
809 and 827, antibody compositions capable of binding such polypeptides, and<br>
numerous additional embodiments employing such compositions, for example in the<br>
detection, diagnosis and/or therapy of human lung cancer.<br>
POLYNUCLEOTIDE COMPOSITIONS<br>
As used herein, the terms "DNA segment" and "polynucleotide" refer to<br>
a DNA molecule that has been isolated free of total genomic DNA of a particular<br>
species. Therefore, a DNA segment encoding a polypeptide refers to a DNA segment<br>
that contains one or more coding sequences yet is substantially isolated away from, or<br>
purified free from, total genomic DNA of the species from which the DNA segment is<br>
obtained. Included within the terms "DNA segment" and "polynucleetide" are DNA<br>
segments and smaller fragments of such segments, and also recombinant vectors,<br>
including, for example, plasmids, cosmids, phagemids, phage, viruses, and the like.<br>
As will be understood by those skilled in the art, the DNA segments of<br>
this invention can include genomic sequences, extra-genomic and plasmid-encoded<br>
sequences and smaller engineered gene segments that express, or may be adapted to<br>
express, proteins, polypeptides, peptides and the like. Such segments may be naturally<br>
isolated, or modified synthetically by the hand of man.<br>
"Isolated," as used herein, means that a polynucleotide is substantially<br>
away from other coding sequences, and that the DNA segment does not contain large<br>
portions of unrelated coding DNA, such as large chromosomal fragments or other<br>
functional genes or polypeptide coding regions. Of course, this refers to the DNA<br>
segment as originally isolated, and does not exclude genes or coding regions later added<br>
to the segment by the hand of man.<br>
As will be recognized by the skilled artisan, polynucleotides may be<br>
single-stranded (coding or antisense) or double-stranded, and may be DNA (genomic,<br>
cDNA or synthetic) or RNA molecules. RNA molecules include HnRNA molecules,<br>
which contain introns and correspond to a DNA molecule hi a one-to-one manner, and<br>
mRNA molecules, which do not contain introns. Additional coding or non-coding<br>
sequences may, but need not, be present within a polynucleotide of the present<br>
invention, and a polynucleotide may, but need not, be linked to other molecules and/or<br>
support materials.<br>
Polynucleotides may comprise a native sequence (i.e., an endogenous<br>
sequence that encodes a lung tumor protein or a portion thereof) or may comprise a<br>
variant, or a biological or antigenic functional equivalent of such a sequence.<br>
Polynucleotide variants may contain one or more substitutions, additions, deletions<br>
and/or insertions, as further described below, preferably such that the immunogenicity<br>
of the encoded polypeptide is not diminished, relative to a native tumor protein. The<br>
effect on the immunogenicity of the encoded polypeptide may generally be assessed as<br>
described herein. The term "variants" also encompasses homologous genes of<br>
xenogenic origin.<br>
When comparing polynucleotide or polypeptide sequences, two<br>
sequences are said to be "identical" if the sequence of nucleotides or amino acids in the<br>
two sequences is the same when aligned for maximum correspondence, as described<br>
below. Comparisons between two sequences are typically performed by comparing the<br>
sequences over a comparison window to identify and compare local regions of sequence<br>
similarity. A "comparison window" as used herein, refers to a segment of at least about<br>
20 contiguous positions, usually 30 to about 75, 40 to about 50, in which a sequence<br>
may be compared to a reference sequence of the same number of contiguous positions<br>
after the two sequences are optimally aligned.<br>
Optimal alignment of sequences for comparison may be conducted using<br>
the Megalign program in the Lasergene suite of bioinformatics software (DNASTAR,<br>
Inc., Madison, WI), using default parameters. This program embodies several<br>
alignment schemes described in the following references: Dayhoff, M.O. (1978) A<br>
model of evolutionary change in proteins - Matrices for detecting distant relationships.<br>
In Dayhoff, M.O. (ed.) Atlas of Protein Sequence and Structure, National Biomedical<br>
Research Foundation, Washington DC Vol. 5, Suppl. 3, pp. 345-358; Hein J. (1990)<br>
Unified Approach to Alignment and Phylogenes pp. 626-645 Methods in Enzymology<br>
vol. 183, Academic Press, Inc., San Diego, CA; Higgins, D.G. and Sharp, P.M. (1989)<br>
CABIOS 5:151-153; Myers, E.W. and Muller W. (1988) CABIOS 4:11-17; Robinson,<br>
E.D. (1971) Comb. Theor 77:105; Santou, N. Nes, M. (1987) Mol. Biol Evol 4:406-<br>
425; Sneath, P.H.A. and Sokal, R.R. (1973) Numerical Taxonomy - the Principles and<br>
Practice of Numerical Taxonomy, Freeman Press, San Francisco, CA; Wilbur, W. J. and<br>
Lipman, D.J. (1983) Proc. Natl. Acad., Sci. USA 80:726-730.<br>
Alternatively, optimal alignment of sequences for comparison may be<br>
conducted by the local identity algorithm of Smith and Waterman (1981) Add. APL.<br>
Math 2:482, by the identity alignment algorithm of Needleman and Wunsch (1970) J.<br>
Mol. Biol. 48:443, by the search for similarity methods of Pearson and Lipman (1988)<br>
Proc. Natl. Acad. Sci. USA 85: 2444, by computerized implementations of these<br>
algorithms (GAP, BESTFIT, BLAST, FASTA, and TFASTA in the Wisconsin Genetics<br>
Software Package, Genetics Computer Group (GCG), 575 Science Dr., Madison, WI),<br>
or by inspection.<br>
One preferred example of algorithms that are suitable for determining<br>
percent sequence identity and sequence similarity are the BLAST and BLAST 2.0<br>
algorithms, which are described in Altschul et al. (1977) Nucl. Acids Res. 25:3389-3402<br>
and Altschul et al. (1990) J. Mol. Biol 215:403-410, respectively. BLAST and BLAST<br>
2.0 can be used, for example with the parameters described herein, to determine percent<br>
sequence identity for the polynucleotides and polypeptides of the invention. Software<br>
for performing BLAST analyses is publicly available through the National Center for<br>
Biotechnology Information. In one illustrative example, cumulative scores can be<br>
calculated using, for nucleotide sequences, the parameters M (reward score for a pair of<br>
matching residues; always &gt;0) and N (penalty score for mismatching residues; always<br>
cumulative score. Extension of the word hits in each direction are halted when: the<br>
cumulative alignment score falls off by the quantity X from its maximum achieved<br>
value; the cumulative score goes to zero or below, due to the accumulation of one or<br>
more negative-scoring residue alignments; or the end of either sequence is reached.<br>
The BLAST algorithm parameters W, T and X determine the sensitivity and speed of<br>
the alignment. The BLASTN program (for nucleotide sequences) uses as defaults a<br>
wordlength (W) of 11, and expectation (E) of 10, and the BLOSUM62 scoring matrix<br>
(see Henlkoff and Henikoff (1989) Proc. Natl. Acad. Sci. USA 89:10915) alignments,<br>
(B) of 50, expectation (E) of 10, M=5, N=-4 and a comparison of both strands.<br>
Preferably, the "percentage of sequence identity" is determined by<br>
comparing two optimally aligned sequences over a window of comparison of at least <br>
positions, wherein the portion of the polynucleotide or polypeptide sequence in the<br>
comparison window may comprise additions or deletions (i.e., gaps) of 20 percent or<br>
less, usually 5 to 15 percent, or 10 to 12 percent, as compared to the reference<br>
sequences (which does not comprise additions or deletions) for optimal alignment of the<br>
two sequences. The percentage is calculated by determining the number of positions at<br>
which the identical nucleic acid bases or amino acid residue occurs in both sequences to<br>
yield the number of matched positions, dividing the number of matched positions by the<br>
total number of positions in the reference sequence (i.e., the window size) and<br>
multiplying the results by 100 to yield the percentage of sequence identity.<br>
Therefore, the present invention encompasses polynucleotide and<br>
polypeptide sequences having substantial identity to the sequences disclosed herein, for<br>
example those comprising at least 50% sequence identity, preferably at least 55%, 60%,<br>
65%, 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, or 99% or higher, sequence<br>
identity compared to a polynucleotide or polypeptide sequence of this invention using<br>
the methods described herein, (e.g., BLAST analysis using standard parameters, as<br>
described below). One skilled in this art will recognize that these values can be<br>
appropriately adjusted to determine corresponding identity of proteins encoded by two<br>
nucleotide sequences by taking into account codon degeneracy, amino acid similarity,<br>
reading frame positioning and the like.<br>
In additional embodiments, the present invention provides isolated<br>
polynucleotides and polypeptides comprising various lengths of contiguous stretches of<br>
sequence identical to or complementary to one or more of the sequences disclosed<br>
herein. For example, polynucleotides are provided by this invention that comprise at<br>
least about 15, 20, 30, 40, 50, 75, 100, 150, 200, 300, 400, 500 or 1000 or more<br>
contiguous nucleotides of one or more of the sequences disclosed herein as well as all<br>
intermediate lengths there between. It will be readily understood that "intermediate<br>
lengths", in this context, means any length between the quoted values, such as 16, 17,<br>
18, 19, etc.; 21, 22, 23, etc.; 30, 31, 32, etc.; 50, 51, 52, 53, etc.; 100, 101, 102, 103,<br>
etc.; 150, 151,152,153, etc.; including all integers through 200-500; 500-1,000, and the<br>
like.<br>
The polynucleotides of the present invention, or fragments thereof,<br>
regardless of the length of the coding sequence itself, may be combined with other<br>
DNA sequences, such as promoters, polyadenylation signals, additional restriction<br>
enzyme sites, multiple cloning sites, other coding segments, and the like, such that their<br>
overall length may vary considerably. It is therefore contemplated that a nucleic acid<br>
fragment of almost any length may be employed, with the total length preferably being<br>
limited by the ease of preparation and use in the intended recombinant DNA protocol.<br>
For example, illustrative DNA segments with total lengths of about 10,000, about 5000,<br>
about 3000, about 2,000, about 1,000, about 500, about 200, about 100, about 50 base<br>
pairs in length, and the like, (including all intermediate lengths) are contemplated to be<br>
useful in many implementations of this invention.<br>
In other embodiments, the present invention is directed to<br>
polynucleotides that are capable of hybridizing under moderately stringent conditions to<br>
a polynucleotide sequence provided herein, or a fragment thereof, or a complementary<br>
sequence thereof. Hybridization techniques are well known in the art of molecular<br>
biology. For purposes of illustration, suitable moderately stringent conditions for<br>
testing the hybridization of a polynucleotide of this invention with other<br>
polynucleotides include prewashing in a solution of 5 X SSC, 0.5% SDS, 1.0 mM<br>
EDTA (pH 8.0); hybridizing at 50°C-65°C, 5 X SSC, overnight; followed by washing<br>
twice at 65°C for 20 minutes with each of 2X, 0.5X and 0.2X SSC containing 0.1%<br>
SDS.<br>
Moreover, it will be appreciated by those of ordinary skill in the art that,<br>
as a result of the degeneracy of the genetic code, there are many nucleotide sequences<br>
that encode a polypeptide as described herein. Some of these polynucleotides bear<br>
minimal homology to the nucleotide sequence of any native gene. Nonetheless,<br>
polynucleotides that vary due to differences in codon usage are specifically<br>
contemplated by the present invention. Further, alleles of the genes comprising the<br>
polynucleotide sequences provided herein are within the scope of the present invention.<br>
Alleles are endogenous genes that are altered as a result of one or more mutations, such<br>
as deletions, additions and/or substitutions of nucleotides. The resulting rnRNA and<br>
protein may, but need not, have an altered structure or function. Alleles may be<br>
identified using standard techniques (such as hybridization, amplification and/or<br>
database sequence comparison).<br>
PROBES AND PRIMERS<br>
In other embodiments of the present invention, the polynucleotide<br>
sequences provided herein can be advantageously used as probes or primers for nucleic<br>
acid hybridization. As such, it is contemplated that nucleic acid segments that comprise<br>
a sequence region of at least about 15 nucleotide long contiguous sequence that has the<br>
same sequence as, or is complementary to, a 15 nucleotide long contiguous sequence<br>
disclosed herein will find particular utility. Longer contiguous identical or<br>
complementary sequences, e.g., those of about 20, 30, 40, 50, 100, 200, .500, 1000<br>
(including all intermediate lengths) and even up to full length sequences will also be of<br>
use in certain embodiments.<br>
The ability of such nucleic acid probes to specifically hybridize to a<br>
sequence of interest will enable them to be of use in detecting the presence of<br>
complementary sequences in a given sample. However, other uses are also envisioned,<br>
such as the use of the sequence information for the preparation of mutant species<br>
primers, or primers for use in preparing other genetic constructions.<br>
Polynucleotide molecules having sequence regions consisting of<br>
contiguous nucleotide stretches of 10-14,15-20,30, 50, or even of 100-200 nucleotides<br>
or so (including intermediate lengths as well), identical or complementary to a<br>
polynucleotide sequence disclosed herein, are particularly contemplated as<br>
hybridization probes for use in, e.g., Southern and Northern blotting. This would allow<br>
a gene product, or fragment thereof, to be analyzed, both in diverse cell types and also<br>
in various bacterial cells. The total size of fragment, as well as the size of the<br>
complementary stretch(es), will ultimately depend on the intended use or application of<br>
the particular nucleic acid segment. Smaller fragments will generally find use in<br>
hybridization embodiments, wherein the length of the contiguous complementary<br>
region may be varied, such as between about 15 and about 100 nucleotides, but larger<br>
contiguous complementarity stretches may be used, according to the length<br>
complementary sequences one wishes to detect.<br>
The use of a hybridization probe of about 15-25 nucleotides in length<br>
allows the formation of a duplex molecule that is both stable and selective. Molecules<br>
having contiguous complementary sequences over stretches greater than 15 bases in<br>
length are generally preferred, though, in order to increase stability and selectivity of<br>
the hybrid, and thereby improve the quality and degree of specific hybrid molecules<br>
obtained. One will generally prefer to design nucleic acid molecules having genecomplementary<br>
stretches of 15 to 25 contiguous nucleotides, or even longer where<br>
desired.<br>
Hybridization probes may be selected from any portion of any of the<br>
sequences disclosed herein. All that is required is to review the sequence set forth in<br>
SEQ ID NO: 1, 11-13, 15, 20, 23-27, 29, 30, 33, 34, 39, 41, 43-46, 51, 52, 57, 58, 60,<br>
62, 65-67, 69-71, 74, 76, 79, 80, 84, 86, 89-92, 95, 97, 98,101,110, 111,113-119,121-<br>
128, 130-134, 136, 138, 139,141, 143, 146-151, 153, 154, 157-160, 162-164, 167-178,<br>
180,181,183,186-190,192,193,195-220,224,226-231,234,236,237,240,241, 244-<br>
246, 248, 254, 255, 261, 262, 266, 270, 275, 280, 282, 283, 288, 289, 290, 292, 295,<br>
301, 303,304, 309, 311,341-782, 784, 785, 790, 792, 794, 796, 800-804, 807, 808 and<br>
810-826, or to any continuous portion of the sequence, from about 15-25 nucleotides in<br>
length up to and including the full length sequence, that one wishes to utilize as a probe<br>
or primer. The choice of probe and primer sequences may be governed by various<br>
factors. For example, one may wish to employ primers from towards the termini of the<br>
total sequence.<br>
Small polynucleotide segments or fragments may be readily prepared by,<br>
for example, directly synthesizing the fragment by chemical means, as is commonly<br>
practiced using an automated oligonucleotide synthesizer. Also, fragments may be<br>
obtained by application of nucleic acid reproduction technology, such as the PCR™<br>
technology of U. S. Patent 4,683,202 (incorporated herein by reference), by introducing<br>
selected sequences into recombinant vectors for recombinant production, and by other<br>
recombinant DNA techniques generally known to those of skill in the art of molecular<br>
biology.<br>
The nucleotide sequences of the invention may be used for their ability<br>
to selectively form duplex molecules with complementary stretches of the entire gene or<br>
gene fragments of interest. Depending on the application envisioned, one will typically<br>
desire to employ varying conditions of hybridization to achieve varying degrees of<br>
selectivity of probe towards target sequence. For applications requiring high<br>
selectivity, one will typically desire to employ relatively stringent conditions to form<br>
the hybrids, e.g., one will select relatively low salt and/or high temperature conditions,<br>
such as provided by a salt concentration of from about 0.02 M to about 0.15 M salt at<br>
temperatures of from about 50°G to about 70°C. Such selective conditions tolerate<br>
little, if any, mismatch between the probe and the template or target strand, and would<br>
be particularly suitable for isolating related sequences. -<br>
Of course, for some applications, for example, where one desires to<br>
prepare mutants employing a mutant primer strand hybridized to an underlying<br>
template, less stringent (reduced stringency) hybridization conditions will typically be<br>
needed in order to allow formation of the heteroduplex. In these circumstances, one<br>
may desire to employ salt conditions such as those of from about 0.15 M to about 0.9 M<br>
salt, at temperatures ranging from about 20°C to about 55°C. Cross-hybridizing.species<br>
can thereby be readily identified as positively hybridizing signals with respect to control<br>
hybridizations. In any case, it is generally appreciated that conditions can be rendered<br>
more stringent by the addition of increasing amounts of formamide, which serves to<br>
destabilize the hybrid duplex in the same manner as increased temperature. Thus,<br>
hybridization conditions can be readily manipulated, and thus will generally be a<br>
method of choice depending on the desired results.<br>
POLYNUCLEOTIDE IDENTIFICATION AND CHARACTERIZATION<br>
Polynucleotides may be identified, prepared and/or manipulated using<br>
any of a variety of well established techniques. For example, a polynucleotide may be<br>
identified, as described in more detail below, by screening a microarray of cDNAs for<br>
tumor-associated expression (i.e., expression that is at least two fold greater in a tumor<br>
than in normal tissue, as determined using a representative assay provided herein).<br>
Such screens may be performed, for example, using a Synteni microarray (Palo Alto,<br>
CA) according to the manufacturer's instructions (and essentially as described by<br>
Schena et al., Proc. Natl. Acad. Sci. USA £3:10614-10619,1996 and Heller et al, Proc.<br>
Natl. Acad. Sci. USA 94:2150-2155, 1997). Alternatively, polynucleotides may be<br>
amplified from cDNA prepared from cells expressing the proteins described herein,<br>
such as lung tumor cells. Such polynucleotides may be amplified via polymerase chain<br>
reaction (PCR). For this approach, sequence-specific primers may be designed based<br>
on the sequences provided herein, and may be purchased or synthesized.<br>
An amplified portion of a polynucleotide of the present invention may be<br>
used to isolate a full length gene from a suitable library (e.g., a lung tumor cDNA<br>
library) using well known techniques. Within such techniques, a library (cDNA or<br>
genomic) is screened using one or more polynucleotide probes or primers suitable for<br>
amplification. Preferably, a library is size-selected to include larger molecules.<br>
Random primed libraries may also be preferred for identifying 5' and upstream regions<br>
of genes. Genomic libraries are preferred for obtaining introns and extending 5'<br>
sequences.<br>
For hybridization techniques, a partial sequence may be labeled (e.g., by<br>
nick-translation or end-labeling with 32P) using well known techniques. A bacterial or<br>
bacteriophage library is then generally screened by hybridizing filters containing<br>
denatured bacterial colonies (or lawns containing phage plaques) with the labeled probe<br>
(see Sambrook et al., Molecular Cloning: A Laboratory Manual, Cold Spring Harbor<br>
Laboratories, Cold Spring Harbor, NY, 1989). Hybridizing colonies or plaques are<br>
selected and expanded, and the DNA is isolated for further analysis. cDNA clones may<br>
be analyzed to determine the amount of additional sequence by, for example, PCR<br>
using a primer from the partial sequence and a primer from the vector. Restriction<br>
maps and partial sequences may be generated to identify one or more overlapping<br>
clones. The complete sequence may then be determined using standard techniques,<br>
which may involve generating a series of deletion clones. The resulting overlapping<br>
sequences can then assembled into a single contiguous sequence. A full length cDNA<br>
molecule can be generated by ligating suitable fragments, using well known techniques.<br>
Alternatively, there are numerous amplification techniques for obtaining<br>
a full length coding sequence from a partial cDNA sequence. Within such techniques,<br>
amplification is generally performed via PCR. Any of a variety of commercially<br>
available kits may be used to perform the amplification step. Primers may be designed<br>
using, for example, software well known in the art. Primers are preferably 22-30<br>
nucleotides hi length, have a GC content of at least 50% and anneal to the target<br>
sequence at temperatures of about 68°C to 72°C. The amplified region may be<br>
sequenced as described above, and overlapping sequences assembled into a contiguous<br>
sequence.<br>
One such amplification technique is inverse PCR (see Triglia et al.,<br>
Nucl. Acids Res. 76:8186,1988), which uses restriction enzymes to generate a fragment<br>
in the known region of the gene. The fragment is then circularized by intramolecular<br>
ligation and used as a template for PCR with divergent primers derived-from the known<br>
region. Within an alternative approach, sequences adjacent to a partial sequence may<br>
be retrieved by amplification with a primer to a linker sequence and a primer specific to<br>
a known region. The amplified sequences are typically subjected to a second round of<br>
amplification with the same linker primer and a second primer specific to the known<br>
region. A variation on this procedure, which employs two primers that initiate<br>
extension in opposite directions from the known sequence, is described hi WO<br>
96/38591. Another such technique is known as "rapid amplification of cDNA ends" or<br>
RACE. This technique involves the use of an internal primer and an external primer,<br>
which hybridizes to a polyA region or vector sequence, to identify sequences that are 5'<br>
and 3' of a known sequence. Additional techniques include capture PCR (Lagerstrom et<br>
al, PCR Methods Applic. 1:111-19,1991) and walking PCR (Parker et al., Nucl. Acids.<br>
Res. 19:3055-60, 1991). Other methods employing amplification may also be<br>
employed to obtain a full length cDNA sequence.<br>
In certain instances, it is possible to obtain a full length cDNA sequence<br>
by analysis of sequences provided in an expressed sequence tag (EST) database, such as<br>
that available from GenBank. Searches for overlapping ESTs may generally be<br>
performed using well known programs (e.g., NCBI BLAST searches), and such ESTs<br>
may be used to generate a contiguous full length sequence. Full length DMA sequences<br>
may also be obtained by analysis of genomic fragments.<br>
POLYNUCLEOTIDE EXPRESSION IN HOST CELLS<br>
In other embodiments of the invention, polynucleotide sequences or<br>
fragments thereof which encode polypeptides of the invention, or fusion proteins or<br>
functional equivalents thereof, may be used in recombinant DNA molecules to direct<br>
expression of a polypeptide hi appropriate host cells. Due to the inherent degeneracy of<br>
the genetic code, other DNA sequences that encode substantially the same or a<br>
functionally equivalent amino acid sequence may be produced and these sequences may<br>
be used to clone and express a given polypeptide.<br>
As will be understood by those of skill in the art, it may be advantageous<br>
in some instances to produce polypeptide-encoding nucleotide sequences possessing<br>
non-naturally occurring codons. For example, codons preferred by a particular<br>
prokaryotic or eukaryotic host can be selected to increase the rate of protein expression<br>
or to produce a recombinant RNA transcript having desirable properties, such as a halflife<br>
which is longer than that of a transcript generated from the naturally occurring<br>
sequence.<br>
Moreover, the polynucleotide sequences of the present invention can be<br>
engineered using methods generally known in the art in order to alter polypeptide<br>
encoding sequences for a variety of reasons, including but not limited to, alterations<br>
which modify the cloning, processing, and/or expression of the gene product. For<br>
example, DNA shuffling by random fragmentation and PCR reassembly of gene<br>
fragments and synthetic oligonucleotides may be used to engineer the nucleotide<br>
sequences. In addition, site-directed mutagenesis may be used to insert new restriction<br>
sites, alter glycosylation patterns, change cddon preference, produce splice variants, or<br>
introduce mutations, and so forth.<br>
In another embodiment of the invention, natural, modified, or<br>
recombinant nucleic acid sequences may be ligated to a heterologous sequence to<br>
encode a fusion protein. For example, to screen peptide libraries for inhibitors of<br>
polypeptide activity, it may be useful to encode a chimeric protein that can be<br>
recognized by a commercially available antibody. A fusion protein may also be<br>
engineered to contain a cleavage site located between the polypeptide-encoding<br>
sequence and the heterologous protein sequence, so that the polypeptide may be cleaved<br>
and purified away from the heterologous moiety.<br>
Sequences encoding a desired polypeptide may be synthesized, in whole<br>
or in part, using chemical methods well known in the art (see Caruthers, M. H. et al.<br>
(1980) Nucl Acids Res. Symp. Ser. 215-223, Horn, T. et al. (1980) Nucl Acids Res.<br>
Symp. Ser. 225-232). Alternatively, the protein itself may be produced using chemical<br>
methods to synthesize the amino acid sequence of a polypeptide, or a portion thereof.<br>
For example, peptide synthesis can be performed using various solid-phase techniques<br>
(Roberge, J. Y. et al. (1995) Science 269:202-204) and automated synthesis may be<br>
achieved, for example, using the ABI 431A Peptide Synthesizer (Perkin Elmer, Palo<br>
Alto, CA).<br>
A newly synthesized peptide may be substantially purified by<br>
preparative high performance liquid chromatography (e.g., Creighton, T. (1983)<br>
Proteins, Structures and Molecular Principles, WH Freeman and Co., New York, N.Y.)<br>
or other comparable techniques available in the art. The composition of the synthetic<br>
peptides may be confirmed by amino acid analysis or sequencing (e.g., the Edman<br>
degradation procedure). Additionally, the amino acid sequence of a polypeptide, or any<br>
part thereof, may be altered during direct synthesis and/or combined using chemical<br>
methods with sequences from other proteins, or any part thereof, to produce a variant<br>
polypeptide.<br>
In order to express a desired polypeptide, the nucleotide sequences<br>
encoding the polypeptide, or functional equivalents, may be inserted into appropriate<br>
expression vector, i.e., a vector which contains the necessary elements for the<br>
transcription and translation of the inserted coding sequence. Methods which are well<br>
known to those skilled in the art may be used to construct expression vectors containing<br>
sequences encoding a polypeptide of interest and appropriate transcriptional and<br>
translational control elements. These methods include in vitro recombinant DNA<br>
techniques, synthetic techniques, and in vivo genetic recombination. Such techniques<br>
are described hi Sambrook, J. et al. (1989) Molecular Cloning, A Laboratory Manual,<br>
Cold Spring Harbor Press, Plainview, N.Y., and Ausubel, F. M. et al. (1989) Current<br>
Protocols in Molecular Biology, John Wiley &amp; Sons, New York. N.Y.<br>
A variety of expression vector/host systems may be utilized to contain<br>
and express polynucleotide sequences. These include, but are not limited to,<br>
microorganisms such as bacteria transformed with recombinant bacteriophage, plasmid,<br>
or cosmid DNA expression vectors; yeast transformed with yeast expression vectors;<br>
insect cell systems infected with virus expression vectors (e.g., baculovirus); plant cell<br>
systems transformed with virus expression vectors (e.g., cauliflower mosaic virus,<br>
CaMV; tobacco mosaic virus, TMV) or with bacterial expression vectors (e.g., Ti or<br>
pBR322 plasmids); or animal cell systems.<br>
The "control elements" or "regulatory sequences" present in an<br>
expression vector are those non-translated regions of the vector-enhancers, promoters,<br>
5' and 3' untranslated regions-which interact with host cellular proteins to carry out<br>
transcription and translation. Such elements may vary in their strength and specificity.<br>
Depending on the vector system and host utilized, any number of suitable transcription<br>
and translation elements, including constitutive and inducible promoters, may be used.<br>
For example, when cloning in bacterial systems, inducible promoters such as the hybrid<br>
lacZ promoter of the PBLUESCRIPT phagemid (Stratagene, La Jolla, Calif.) or<br>
PSPORT1 plasmid (Gibco BRL, Gaithersburg, MD) and the like may be used. In<br>
mammalian cell systems, promoters from mammalian genes or from mammalian<br>
viruses are generally preferred. If it is necessary to generate a cell line that contains<br>
multiple copies of the sequence encoding a polypeptide, vectors based on SV40 or EBV<br>
may be advantageously used with an appropriate selectable marker.<br>
In bacterial systems, a number of expression vectors may be selected<br>
depending upon the use intended for the expressed polypeptide. For example, when<br>
large quantities are needed, for example for the induction of antibodies, vectors which<br>
direct high level expression of fusion proteins that are readily purified may be used.<br>
Such vectors include, but are not limited to, the multifunctional E. coli cloning and<br>
expression vectors such as BLUESCRIPT (Stratagene), in which the sequence encoding<br>
the polypeptide of interest may be ligated into the vector hi frame with sequences for<br>
the amino-tenninal Met and the subsequent 7 residues of .beta.-galactosidase so that a<br>
hybrid protein is produced; pIN vectors (Van Heeke, G. and S. M. Schuster (1989) J.<br>
Biol. Chem. 26^:5503-5509); and the like. pGEX Vectors (Promega, Madison, Wis.)<br>
may also be used to express foreign polypeptides as fusion proteins with glutathione Stransferase<br>
(GST). In general, such fusion proteins are soluble and can easily be<br>
purified from lysed cells by adsorption to glutathione-agarose beads followed by elution<br>
in the presence of free glutathione. Proteins made in such systems may be designed to<br>
include heparin, thrombin, or factor XA protease cleavage sites so that the cloned<br>
polypeptide of interest can be released from the GST moiety at will.<br>
In the yeast, Saccharomyces cerevisiae, a number of vectors containing<br>
constitutive or inducible promoters such as alpha factor, alcohol oxidase, and PGH may<br>
be used. For reviews, see Ausubel et al (supra) and Grant et al. (1987) Methods<br>
Enzymol. 153:516-544.<br>
In cases where plant expression vectors are used, the expression of<br>
sequences encoding polypeptides may be driven by any of a number of promoters. For<br>
example, viral promoters such as the 35S and 19S promoters of CaMV may be used<br>
alone or in combination with the omega leader sequence from TMV (Takamatsu, N.<br>
(1987) EMBO J. 5:307-311. Alternatively, plant promoters such as the small subunit of<br>
RUBISCO or heat shock promoters may be used (Coruzzi, G. et al. (1984) EMBO J.<br>
3:1671-1680; Broglie, R. et al. (1984) Science 22*838-843; and Winter, J. et al (1991)<br>
Results Probl. Cell Differ. 77:85-105). These constructs can be introduced into plant<br>
cells by direct DNA transformation or pathogen-mediated transfection. Such techniques<br>
are described in a number of generally available reviews (see, for example, Hobbs, S. or<br>
Murry, L. E. in McGraw Hill Yearbook of Science and Technology (1992) McGraw<br>
Hill, New York, N.Y.; pp. 191-196).<br>
An bisect system may also be used to express a polypeptide of interest.<br>
For example, in one such system, Autographa californica nuclear polyhedrosis virus<br>
(AcNPV) is used as a vector to express foreign genes in Spodoptera frugiperda cells or<br>
hi Trichoplusia larvae. The sequences encoding the polypeptide may be cloned into a<br>
non-essential region of the virus, such as the polyhedrin gene, and placed under control<br>
of the polyhedrin promoter. Successful insertion of the polypeptide-encoding sequence<br>
will render the polyhedrin gene inactive and produce recombinant virus lacking coat<br>
protein. The recombinant viruses may then be used to infect, for example, S. frugiperda<br>
cells or Trichoplusia larvae hi which the polypeptide of interest may be expressed<br>
(Engelhard, E. K. etal (1994) Proc. Natl. Acad. Sci. 91:3224-3227).<br>
In mammalian host cells, a number of viral-based expression systems are<br>
generally available. For example, in cases where an adenovirus is used as an expression<br>
vector, sequences encoding a polypeptide of interest may be ligated into an adenovirus<br>
transcription/translation complex consisting of the late promoter and tripartite leader<br>
sequence. Insertion in a non-essential El or E3 region of the viral genome may be used<br>
to obtain a viable virus which is capable of expressing the polypeptide in infected host<br>
cells (Logan, J. and Shenk, T. (1984) Proc. Natl. Acad. Sci. 57:3655-3659). In addition,<br>
transcription enhancers, such as the Rous sarcoma virus (RSV) enhancer, may be used<br>
to increase expression in mammalian host cells.<br>
Specific initiation signals may also be used to achieve more efficient<br>
translation of sequences encoding a polypeptide of interest. Such signals include the<br>
ATG initiation codon and adjacent sequences. In eases where sequences encoding the<br>
polypeptide, its initiation codon, and upstream sequences are inserted into the<br>
appropriate expression vector, no additional transcriptional or translational control<br>
signals may be needed. However, in cases where only coding sequence, or a portion<br>
thereof, is inserted, exogenous translational control signals including the ATG initiation<br>
codon should be provided. Furthermore, the initiation codon should be in the correct<br>
reading frame to ensure translation of the entire insert. Exogenous translational<br>
elements and initiation codons may be of various origins, both natural and synthetic.<br>
The efficiency of expression may be enhanced by the inclusion of enhancers which are<br>
appropriate for the particular cell system which is used, such as those described in the<br>
literature (Scharf, D. etal. (1994) Results Probl. Cell Differ. 20:125-162).<br>
In addition, a host cell strain may be chosen for its ability to modulate<br>
the expression of the inserted sequences or to process the expressed protein in the<br>
desired fashion. Such modifications of the polypeptide include, but are not limited to,<br>
acetylation, carboxylation. glycosylation, phosphorylation, lipidation, and acylation.<br>
Post-translational processing which cleaves a "prepro" form of the protein may also be<br>
used to facilitate correct insertion, folding and/or function. Different host cells such as<br>
CHO, HeLa, MDCK, HEK293, and WI38, which have specific cellular machinery and<br>
characteristic mechanisms for such post-translational activities, may be chosen to<br>
ensure the correct modification and processing of the foreign protein.<br>
For long-term, high-yield production of recombinant proteins, stable<br>
expression is generally preferred. For example, cell lines which stably express a<br>
polynucleotide of interest may be transformed using expression vectors which may<br>
contain viral origins of replication and/or endogenous expression elements and a<br>
selectable marker gene on the same or on a separate vector. Following the introduction<br>
of the vector, cells may be allowed to grow for 1-2 days in an enriched media before<br>
they are switched to selective media. The purpose of the selectable marker is to confer<br>
resistance to selection, and its presence allows growth and recovery of cells which<br>
successfully express the introduced sequences. Resistant clones of stably transformed<br>
cells may be proliferated using tissue culture techniques appropriate to the cell type.<br>
Any number of selection systems may be used to recover transformed<br>
cell lines. These include, but are not limited to, the herpes simplex virus thymidine<br>
kinase (Wigler, M. et al. (1977) Cell 17:223-32) and adenine phosphoribosyltransferase<br>
(Lowy, I. et al (1990) Cell 22317-23) genes which can be employed in tk.sup.- or<br>
apitsup.- cells, respectively. Also, antimetabolite, antibiotic or herbicide resistance can<br>
be used as the basis for selection; for example, dhfr which confers resistance to<br>
methotrexate (Wigler, M. et al. (1980) Proc. Natl Acad. Sci. 77:3567-70); npt, which<br>
confers resistance to the aminoglycosides, neomycin and G-418 (Colbere-Garapin, F. et<br>
al (1981) J. Mol. Biol 750:1-14); and als or pat, which confer resistance to<br>
chlorsulfuron and phosphinotricin acetyltransferase, respectively (Murry, supra).<br>
Additional selectable genes have been described, for example, trpB, which allows cells<br>
to utilize indole hi place of tryptophan, or hisD, which allows cells to utilize histinol in<br>
place of histidine (Hartman, S. C. and R. C. Mulligan (1988) Proc. Natl. Acad. Sci.<br>
55:8047-51). Recently, the use of visible markers has gained popularity with such<br>
markers as anthocyanins, beta-glucuronidase and its substrate GUS, and luciferase and<br>
its substrate luciferin, being widely used not only to identify transformants, but also to<br>
quantify the amount of transient or stable protein expression attributable to a specific<br>
vector system (Rhodes, C. A. etal (1995) Methods Mol. Biol 55:121-131).<br>
Although the presence/absence of marker gene expression suggests that<br>
the gene of interest is also present, its presence and expression may need to be<br>
confirmed. For example, if the sequence encoding a polypeptide is inserted within a<br>
marker gene sequence, recombinant cells containing sequences can be identified by the<br>
absence of marker gene function. Alternatively, a marker gene can be placed hi tandem<br>
with a polypeptide-encoding sequence under the control of a single promoter.<br>
Expression of the marker gene hi response to induction or selection usually indicates<br>
expression of the tandem gene as well.<br>
Alternatively, host cells which contain and express a desired<br>
polynucleotide sequence may be identified by a variety of procedures known to those of<br>
skill in the art. These procedures include, but are not limited to, DNA-DNA or DNARNA<br>
hybridizations and protein bioassay or immunoassay techniques which include<br>
membrane, solution, or chip based technologies for the detection and/or quantification<br>
of nucleic acid or protein.<br>
A variety of protocols for detecting and measuring the expression of<br>
polynucleotide-encoded products, using either polyclonal or monoclonal antibodies<br>
specific for the product are known in the art. Examples include enzyme-linked<br>
immunosorbent assay (ELISA), radioimmunoassay (RIA), and fluorescence activated<br>
cell sorting (FACS). A two-site, monoclonal-based immunoassay utilizing monoclonal<br>
antibodies reactive to two non-interfering epitopes on a given polypeptide may be<br>
preferred for some applications, but a competitive binding assay may also be employed.<br>
These and other assays are described, among other places, in Hampton, R. et al. (1990;<br>
Serological Methods, a Laboratory Manual, APS Press, St Paul. Minn.) and Maddox, D.<br>
E. etal (1983; J. Exp. Med 158:1211-1216).<br>
A wide variety of labels and conjugation techniques are known by those<br>
skilled in the art and may be used in various nucleic acid and amino acid assays. Means<br>
for producing labeled hybridization or PCR probes for detecting sequences related to<br>
polynucleotides include oligolabeling, nick translation, end-labeling or PCR<br>
amplification using a labeled nucleotide. Alternatively, the sequences, or any portions<br>
thereof may be cloned into a vector for the production of an mRNA probe. Such vectors<br>
are known in the art, are commercially available, and may be used to synthesize RNA<br>
probes in vitro by addition of an appropriate RNA polymerase such as T7, T3, or SP6<br>
and labeled nucleotides. These procedures may be conducted using a variety of<br>
commercially available kits. Suitable reporter molecules or labels, which may be used<br>
include radionuclides, enzymes, fluorescent, chemiluminescent, or chromogenic agents<br>
as well as substrates, cofactors, inhibitors, magnetic particles, and the like.<br>
Host cells transformed with a polynucleotide sequence of interest may be<br>
cultured under conditions suitable for the expression and recovery of the protein from<br>
cell culture. The protein produced by a recombinant cell may be secreted or contained<br>
intracellularly depending on the sequence and/or the vector used. As will be understood<br>
by those of skill in the art, expression vectors containing polynucleotides of the<br>
invention may be designed to contain signal sequences which direct secretion of the<br>
encoded polypeptide through a prokaryotic or eukaryotic cell membrane. Other<br>
recombinant constructions may be used to join sequences encoding a polypeptide of<br>
interest to nucleotide sequence encoding a polypeptide domain which will facilitate<br>
purification of soluble proteins. Such purification facilitating domains include, but are<br>
not limited to, metal chelating peptides such as histidine-tryptophan modules that allow<br>
purification on immobilized metals, protein A domains that allow purification on<br>
immobilized immunoglobulin, and the domain utilized in the FLAGS extension/affinity<br>
purification system (Immunex Corp., Seattle, Wash.). The inclusion of cleavable linker<br>
sequences such as those specific for Factor XA or enterokinase (Invitrogen. San Diego,<br>
Calif.) between the purification domain and the encoded polypeptide may be used to<br>
facilitate purification. One such expression vector provides for expression of a fusion<br>
protein containing a polypeptide of interest and a nucleic acid encoding 6 histidine<br>
residues preceding a thioredoxin or an enterokinase cleavage site. The histidine residues<br>
facilitate purification on IMIAC (immobilized metal ion affinity chromatography) as<br>
described in Porath, J. etal. (1992, Prot. Exp. Purif. 3:263-281) while the enterokinase<br>
cleavage site provides a means for purifying the desired polypeptide from the fusion<br>
protein. A discussion of vectors which contain fusion proteins is provided in Kroll, D. J.<br>
etal. (1993; DNA CellBiol 72:441-453).<br>
In addition to recombinant production methods, polypeptides of the<br>
invention, and fragments thereof, may be produced by direct peptide synthesis using<br>
solid-phase techniques (Merrifield J. (1963) J. Am. Chem. Soc. 55:2149-2154). Protein<br>
synthesis may be performed using manual techniques or by automation. Automated<br>
synthesis may be achieved, for example, using Applied Biosystems 431A Peptide<br>
Synthesizer (Perkin Elmer). Alternatively, various fragments may be chemically<br>
synthesized separately and combined using chemical methods to produce the full length<br>
molecule.<br>
SITE-SPECIFIC MUTAGENESIS<br>
Site-specific mutagenesis is a technique useful in the preparation of<br>
individual peptides, or biologically functional equivalent polypeptides, through specific<br>
mutagenesis of the underlying polynucleotides that encode them. The technique,<br>
well-known to those of skill in the art, further provides a ready ability to prepare and<br>
test sequence variants, for example, incorporating one or more of the foregoing<br>
considerations, by introducing one or more nucleotide sequence changes into the DNA.<br>
Site-specific mutagenesis allows the production of mutants through the use of specific<br>
oligonucleotide sequences which encode the DNA sequence of the desired mutation, as<br>
well as a sufficient number of adjacent nucleotides, to provide a primer sequence of<br>
sufficient size and sequence complexity to form a stable duplex on both sides of the<br>
deletion junction being traversed. Mutations may be employed in a selected<br>
polynucleotide sequence to improve, alter, decrease, modify, or otherwise change the<br>
properties of the polynucleotide itself, and/or alter the properties, activity, composition,<br>
stability, or primary sequence of the encoded polypeptide.<br>
In certain embodiments of the present invention, the inventors<br>
contemplate the mutagenesis of the disclosed polynucleotide sequences to alter one or<br>
more properties of the encoded polypeptide, such as the antigenicity of a polypeptide<br>
vaccine. The techniques of site-specific mutagenesis are well-known in the art, and are<br>
widely used to create variants of both polypeptides and polynucleotides. For example,<br>
site-specific mutagenesis is often used to alter a specific portion of a DNA molecule. In<br>
such embodiments, a primer comprising typically about 14 to about 25 nucleotides or so<br>
in length is employed, with about 5 to about 10 residues on both sides of the junction of<br>
the sequence being altered.<br>
As will be appreciated by those of skill in the art, site-specific<br>
mutagenesis techniques have often employed a phage vector that exists in both a single<br>
stranded and double stranded form. Typical vectors useful in site-directed mutagenesis<br>
include vectors such as the Ml 3 phage. These phage are readily<br>
commercially-available and their use is generally well-known to those skilled in the art.<br>
Double-stranded plasmids are also routinely employed in site directed mutagenesis that<br>
eliminates the step of transferring the gene of interest from a plasmid to a phage.<br>
In general, site-directed mutagenesis in accordance herewith is<br>
performed by first obtaining a single-stranded vector or melting apart of two strands of<br>
a double-stranded vector that includes within its sequence a DNA sequence that<br>
encodes the desired peptide. An oligonucleotide primer bearing the desired mutated<br>
sequence is prepared, generally synthetically. This primer is then annealed with the<br>
single-stranded vector, and subjected to DNA polymerizing enzymes such as E. coli<br>
polymerase I Klenow fragment, in order to complete the synthesis of the mutationbearing<br>
strand. Thus, a heteroduplex is formed wherein one strand encodes the original<br>
non-mutated sequence and the second strand bears the desired mutation. This<br>
heteroduplex vector is then used to transform appropriate cells, such as E. coli cells, and<br>
clones are selected which include recombinant vectors bearing the mutated sequence<br>
arrangement.<br>
The preparation of sequence variants of the selected peptide-encoding<br>
DNA segments using site-directed mutagenesis provides a means of producing<br>
potentially useful species and is not meant to be limiting as there are other ways in<br>
which sequence variants of peptides and the DNA sequences encoding them may be<br>
obtained. For example, recombinant vectors encoding the desired peptide sequence<br>
may be treated with mutagenic agents, such as hydroxylamine, to obtain sequence<br>
variants. Specific details regarding these methods and protocols are found in the<br>
teachings of Maloy etal, 1994; Segal, 1976; Prokop and Bajpai, 1991; Kuby, 1994;<br>
and Maniatis et al, 1982, each incorporated herein by reference, for that purpose.<br>
As used herein, the term "oligonucleotide directed mutagenesis<br>
procedure" refers to template-dependent processes and vector-mediated propagation<br>
which result in an increase in the concentration of a specific nucleic acid molecule<br>
relative to its initial concentration, or hi an increase in the concentration of a detectable<br>
signal, such as amplification. As used herein, the term "oligonucleotide directed<br>
mutagenesis procedure" is intended to refer to a process that involves the<br>
template-dependent extension of a primer molecule. The term template dependent<br>
process refers to nucleic acid synthesis of an RNA or a DNA molecule wherein the<br>
sequence of the newly synthesized strand of nucleic acid is dictated by the well-known<br>
rules of complementary base pairing (see, for example, Watson, 1987). Typically,<br>
vector mediated methodologies involve the introduction of the nucleic acid fragment<br>
into a DNA or RNA vector, the clonal amplification of the vector, and the recovery of<br>
the amplified nucleic acid fragment Examples of such methodologies are provided by<br>
U. S. Patent No. 4,237,224, specifically incorporated herein by reference in its entirety.<br>
POLYNUCLEOTIDE AMPLIFICATION TECHNIQUES<br>
A number of template dependent processes are available to amplify the<br>
target sequences of interest present in a sample. One of the best known amplification<br>
methods is the polymerase chain reaction (PCR™) which is described in detail in U.S.<br>
Patent Nos. 4,683,195, 4,683,202 and 4,800,159, each of which is incorporated herein<br>
by reference in its entirety. Briefly, hi PCR™* two primer sequences are prepared<br>
which are complementary to regions on opposite complementary strands of the target<br>
sequence. An excess of deoxynucleoside triphosphates is added to a reaction mixture<br>
along with a DNA polymerase (e.g., Tag polymerase). If the target sequence is present<br>
in a sample, the primers will bind to the target and the polymerase will cause the<br>
primers to be extended along the target sequence by adding on nucleotides. By raising<br>
and lowering the temperature of the reaction mixture, the extended primers will<br>
dissociate from the target to form reaction products, excess primers will bind to the<br>
target and to the reaction product and the process is repeated. Preferably reverse<br>
transcription and PCR™ amplification procedure may be performed hi order to quantify<br>
the amount of mRNA amplified. Polymerase chain reaction methodologies are well<br>
known in the art.<br>
Another method for amplification is the ligase chain reaction (referred to<br>
as LCR), disclosed in Eur. Pat. Appl. Publ. No. 320,308 (specifically incorporated<br>
herein by reference in its entirety). In LCR, two complementary probe pairs are<br>
prepared, and in the presence of the target sequence, each pair will bind to opposite<br>
complementary strands of the target such that they abut. In the presence of a ligase, the<br>
two probe pairs will link to form a single unit. By temperature cycling, as in PCR™,<br>
bound ligated units dissociate from the target and then serve as "target sequences" for<br>
ligation of excess probe pairs. U.S. Patent No. 4,883,750, incorporated herein by<br>
reference hi its entirety, describes an alternative method of amplification similar to LCR<br>
for binding probe pairs to a target sequence.<br>
Qbeta Replicase, described in PCT Intl. Pat Appl. Publ. No.<br>
PCT/US87/00880, incorporated herein by reference in its entirety, may also be used as<br>
still another amplification method in the present invention. In this method, a replicative<br>
sequence of RNA that has a region complementary to that of a target is added to a<br>
sample in the presence of an RNA polymerase. The polymerase will copy the<br>
replicative sequence that can then be detected.<br>
An isothennal amplification method, in which restriction endonucleases<br>
and ligases are used to achieve the amplification of target molecules that contain<br>
nucleotide 5'-[a-thio]triphosphates in one strand of a restriction site (Walker et al.,<br>
1992, incorporated herein by reference in its entirety), may also be useful in the<br>
amplification of nucleic acids in the present invention.<br>
Strand Displacement Amplification (SDA) is another method of carrying<br>
out isothermal amplification of nucleic acids which involves multiple rounds of strand<br>
displacement and synthesis, i.e. nick translation. A similar method, called Repair Chain<br>
Reaction (RCR) is another method of amplification which may be useful in the present<br>
invention and is involves annealing several probes throughout a region targeted for<br>
amplification, followed by a repair reaction in which only two of the four bases are<br>
present. The other two bases can be added as biotinylated derivatives for easy<br>
detection. A similar approach is used in SDA.<br>
Sequences can also be detected using a cyclic probe reaction (CPR). In<br>
CPR, a probe having a 3' and 5' sequences of non-target DNA and an internal or<br>
"middle" sequence of the target protein specific RNA is hybridized to DNA which is<br>
present in a sample. Upon hybridization, the reaction is treated with RNaseH, and the<br>
products of the probe are identified as distinctive products by generating a signal that is<br>
released after digestion. The original template is annealed to another cycling probe and<br>
the reaction is repeated. Thus, CPR involves amplifying a signal generated by<br>
hybridization of a probe to a target gene specific expressed nucleic acid.<br>
Still other amplification methods described in Great Britain Pat. Appl.<br>
No. 2 202 328, and in PCT Intl. Pat. Appl. Publ. No. PCT/US89/01025, each of which<br>
is incorporated herein by reference in its entirety, may be used in accordance with the<br>
present invention. In the former application, "modified" primers are used in a PCR-<br>
like, template and enzyme dependent synthesis. The primers may be modified by<br>
labeling with a capture moiety (e.g., biotin) and/or a detector moiety (e.g., enzyme). In<br>
the latter application, an excess of labeled probes is added to a sample. In the presence<br>
of the target sequence, the probe binds and is cleaved catalytically. After cleavage, the<br>
target sequence is released intact to be bound by excess probe. Cleavage of the labeled<br>
probe signals the presence of the target sequence.<br>
Other nucleic acid amplification procedures include transcription-based<br>
amplification systems (TAS) (Kwoh et al, 1989; PCT Intl. Pat. Appl. Publ. No. WO<br>
88/10315, incorporated herein by reference in its entirety), including nucleic acid<br>
sequence based amplification (NASBA) and 3SR. In NASBA, the nucleic acids can be<br>
prepared for amplification by standard phenol/chloroform extraction, heat denaturation<br>
of a sample, treatment with lysis buffer and minispin columns for isolation of DNA and<br>
RNA or guanidinium chloride extraction of RNA. These amplification techniques<br>
involve annealing a primer that has sequences specific to the target sequence.<br>
Following polymerization, DNA/RNA hybrids are digested with RNase H while double<br>
stranded DNA molecules are heat-denatured again. In either case the single stranded<br>
DNA is made fully double stranded by addition of second target-specific primer,<br>
followed by polymerization. The double stranded DNA molecules are then multiply<br>
transcribed by a polymerase such as T7 or SP6. In an isothermal cyclic reaction, the<br>
RNAs are reverse transcribed into DNA, and transcribed once again with a polymerase<br>
such as T7 or SP6. The resulting products, whether truncated or complete, indicate<br>
target-specific sequences.<br>
Eur. Pat. Appl. Publ. No. 329,822, incorporated herein by reference in its<br>
entirety, disclose a nucleic acid amplification process involving cyclically synthesizing<br>
single-stranded RNA ("ssRNA"), ssDNA, and double-stranded DNA (dsDNA), which<br>
may be used in accordance with the present invention. The ssRNA is a first template<br>
for a first primer oligonucleotide, which is elongated by reverse transcriptase<br>
(RNA-dependent DNA polymerase). The RNA is then removed from resulting<br>
DNArRNA duplex by the action of ribonuclease H (RNase H, an RNase specific for<br>
RNA in a duplex with either DNA or RNA). The resultant ssDNA is a second template<br>
for a second primer, which also includes the sequences of an RNA polymerase<br>
promoter (exemplified by T7 RNA polymerase) 5' to its homology to its template. This<br>
primer is then extended by DNA polymerase (exemplified by the large "Klenow"<br>
fragment of E. coli DNA polymerase I), resulting as a double-stranded DNA<br>
("dsDNA") molecule, having a sequence identical to that of the original RNA between<br>
the primers and having additionally, at one end, a promoter sequence. This promoter<br>
sequence can be used by the appropriate RNA polymerase to make many RNA copies<br>
of the DNA. These copies can then re-enter the cycle leading to very swift<br>
amplification. With proper choice of enzymes, this amplification can be done<br>
isothermally without addition of enzymes at each cycle. Because of the cyclical nature<br>
of this process, the starting sequence can be chosen to be in the form of either DNA or<br>
RNA.<br>
PCT Intl. Pat. Appl. Publ. No. WO 89/06700, incorporated herein by<br>
reference in its entirety, disclose a nucleic acid sequence amplification scheme based on<br>
the hybridization of a promoter/primer sequence to a target single-stranded DNA<br>
("ssDNA") followed by transcription of many RNA copies of the sequence. This<br>
scheme is not cyclic; i.e. new templates are not produced from the resultant RNA<br>
transcripts. Other amplification methods include "RACE" (Frohman, 1990), and<br>
"one-sided PCR" (Ohara, 1989) which are well-known to those of skill in the art.<br>
Methods based on ligation of two (or more) oligonucleotides in the<br>
presence of nucleic acid having the sequence of the resulting "di-oligonucleotide",<br>
thereby amplifying the di-oligonucleotide (Wu and Dean, 1996, incorporated herein by<br>
reference in its entirety), may also be used in the amplification of DNA sequences of<br>
the present invention.<br>
BIOLOGICAL FUNCTIONAL EQUIVALENTS<br>
Modification and changes may be made in the structure of the<br>
polynucleotides and polypeptides of the present invention and still obtain a functional<br>
molecule that encodes a polypeptide with desirable characteristics. As mentioned<br>
above, it is often desirable to introduce one or more mutations into a specific<br>
polynucleotide sequence. In certain circumstances, the resulting encoded polypeptide<br>
sequence is altered by this mutation, or in other cases, the sequence of the polypeptide<br>
is unchanged by one or more mutations in the encoding polynucleotide..<br>
When it is desirable to alter the amino acid sequence of a polypeptide to<br>
create an equivalent, or even an improved, second-generation molecule, the amino acid<br>
changes may be achieved by changing one or more of the codons of the encoding DNA<br>
sequence, according to Table 1.<br>
For example, certain amino acids may be substituted for other amino<br>
acids in a protein structure without appreciable loss of interactive binding capacity with<br>
structures such as, for example, antigen-binding regions of antibodies or binding sites<br>
on substrate molecules. Since it is the interactive capacity and nature of a protein that<br>
defines that protein's biological functional activity, certain amino acid sequence<br>
substitutions can be made in a protein sequence, and, of course, its underlying DNA<br>
coding sequence, and nevertheless obtain a protein with like properties. It is thus<br>
contemplated by the inventors that various changes may be made in the peptide<br>
sequences of the disclosed compositions, or corresponding DNA sequences which<br>
encode said peptides without appreciable loss of their biological utility or activity.<br>
(Table Removed)<br>
In making such changes, the hydropathic index of amino acids may be<br>
considered. The importance of the hydropathic amino acid index in conferring<br>
interactive biologic function on a protein is generally understood in the art (Kyte and<br>
Doolittle, 1982, incorporated herein by reference). It is accepted that the relative<br>
hydropathic character of the amino acid contributes to the secondary structure of the<br>
resultant protein, which in turn defines the interaction of the protein with other<br>
molecules, for example, enzymes, substrates, receptors, DNA, antibodies, antigens, and<br>
the like. Each amino acid has been assigned a hydropathic index on the basis of its<br>
hydrophobicity and charge characteristics (Kyte and Doolittle, 1982). These values are:<br>
isoleucine (+4.5); valine (+4.2); leucine (+3.8); phenylalanine (+2.8); cysteine/cystine<br>
(+2.5); methionine (+1.9); alanine (+1.8); glycine (-0.4); threonine (-0.7); serine (-<br>
0.8); tryptophan (-0.9); tyrosine (-1.3); proline (-1.6); histidine (-3.2); glutamate (-<br>
3.5); glutamine (-3.5); aspartate (-3.5); asparagine (-3.5); lysine (-3.9); and arginine (-<br>
4.5).<br>
It is known in the art that certain amino acids may be substituted by<br>
other amino acids having a similar hydropathic index or score and still result in a<br>
protein with similar biological activity, i.e. still obtain a biological functionally<br>
equivalent protein. In making such changes, the substitution of amino acids whose<br>
hydropathic indices are within ±2 is preferred, those within ±1 are particularly<br>
preferred, and those within ±0.5 are even more particularly preferred. It is also<br>
understood in the art that the substitution of like amino acids can be made effectively on<br>
the basis of hydrophilicity. U. S. Patent 4,554,101 (specifically incorporated herein by<br>
reference in its entirety), states that the greatest local average hydrophilicity of a<br>
protein, as governed by the hydrophilicity of its adjacent amino acids, correlates with a<br>
biological property of the protein.<br>
As detailed in U. S. Patent 4,554,101, the following hydrophilicity<br>
values have been assigned to amino acid residues: arginine (+3.0); lysine (+3.0);<br>
aspartate (+3.0 ± 1); glutamate (+3.0 ± 1); serine (+0.3); asparagine (+0.2); glutamine<br>
(+0.2); glycine (0); threonine (-0.4); proline (-0.5 ± 1); alanine (-0.5); histidine (-0.5);<br>
cysteine (-1.0); methionine (-1.3); valine (—1.5); leucine (—1.8); isoleucine (—1.8);<br>
tyrosine (-2.3); phenylalanine (-2.5); tryptophan (-3.4). It is understood that an amino<br>
acid can be substituted for another having a similar hydrophilicity value and still obtain<br>
a biologically equivalent, and in particular, an immunologically equivalent protein. In<br>
such changes, the substitution of amino acids whose hydrophilicity values are within ±2<br>
is preferred, those within ±1 are particularly preferred, and those within ±0.5 are even<br>
more particularly preferred.<br>
As outlined above, amino acid substitutions are generally therefore based<br>
on the relative similarity of the amino acid side-chain substituents, for example, their<br>
hydrophobicity, hydrophilicity, charge, size, and the like. Exemplary substitutions that<br>
take various of the foregoing characteristics into consideration are well known to those<br>
of skill in the art and include: arginine and iysine; glutamate and aspartate; serine and<br>
threonine; giutamine and asparagine; and valine, leucine and isoleucine.<br>
In addition, any polynucleotide may be fiirther modified to increase<br>
stability in vivo. Possible modifications include, but are not limited to, the addition of<br>
flanking sequences at the 5' and/or 3' ends; the use of phosphorothioate or 2' O-methyl<br>
rather than phosphodiesterase linkages in the backbone; and/or the inclusion of<br>
nontraditional bases such as inosine, queosine and wybutosine, as well as acetylmethyl-,<br>
thio- and other modified forms of adenine, cytidine, guanine, thymine and<br>
uridine.<br>
IN vivo POLYNUCLEOTIDE DELIVERY TECHNIQUES<br>
In additional embodiments, genetic constructs comprising one or more of<br>
the polynucleotides of the invention are introduced into cells in vivo. This may be<br>
achieved using any of a variety or well known approaches, several of which are<br>
outlined below for the purpose of illustration.<br>
1. ADENOVIRUS<br>
One of the preferred methods for in vivo delivery of one or more nucleic<br>
acid sequences involves the use of an adenovirus expression vector. "Adenovirus<br>
expression vector" is meant to include those constructs containing adenovirus<br>
sequences sufficient to (a) support packaging of the construct and (b) to express a<br>
polynucleotide that has been cloned therein in a sense or antisense orientation. Of<br>
course, in the context of an antisense construct, expression does not require that the<br>
gene product be synthesized.<br>
The expression vector comprises a genetically engineered form of an<br>
adenovirus. Knowledge of the genetic organization of adenovirus, a 36 kb, linear,<br>
double-stranded DNA virus, allows substitution of large pieces of adenoviral DNA with<br>
foreign sequences up to 7 kb (Grunhaus and Horwitz, 1992). In contrast to retrovirus,<br>
the adenoviral infection of host cells does not result in chromosomal integration<br>
because adenoviral DNA can replicate in an episomal manner without potential<br>
genotoxiciry. Also, adenoviruses are structurally stable, and no genome rearrangement<br>
has been detected after extensive amplification. Adenovirus can infect virtually all<br>
epithelial cells regardless of their cell cycle stage. So far, adenoviral infection appears<br>
to be linked only to mild disease such as acute respiratory disease in humans.<br>
Adenovirus is particularly suitable for use as a gene transfer vector<br>
because of its mid-sized genome, ease of manipulation, high titer, wide target-cell range<br>
and high infectivity. Both ends of the viral genome contain 100-200 base pair inverted<br>
repeats (ITRs), which are cis elements necessary for viral DNA replication and<br>
packaging. The early (E) and late (L) regions of the genome contain different<br>
transcription units that are divided by the onset of viral DNA replication. The El<br>
region (El A and E1B) encodes proteins responsible for the regulation of transcription<br>
of the viral genome and a few cellular genes. The expression of the E2 region (E2A<br>
and E2B) results in the synthesis of the proteins for viral DNA replication. These<br>
proteins are involved in DNA replication, late gene expression and host cell shut-off<br>
(Renan, 1990). The products of the late genes, including the majority of the viral capsid<br>
proteins, are expressed only after significant processing of a single primary transcript<br>
issued by the major late promoter (MLP). The MLP, (located at 16.8 m.u.) is<br>
particularly efficient during the late phase of infection, and all the mRNA's issued from<br>
this promoter possess a 5'-tripartite leader (TPL) sequence which makes them preferred<br>
mRNA's for translation.<br>
In a current system, recombinant adenovirus is generated from<br>
homologous recombination between shuttle vector and provirus vector. Due to the<br>
possible recombination between two proviral vectors, wild-type adenovirus may be<br>
generated from this process. Therefore, it is critical to isolate a single clone of virus<br>
from an individual plaque and examine its genomic structure.<br>
Generation and propagation of the current adenovirus vectors, which are<br>
replication deficient, depend on a unique helper cell line, designated 293, which was<br>
transformed from human embryonic kidney cells by AdS DNA fragments and<br>
constitutively expresses El proteins (Graham et a/., 1977). Since the E3 region is<br>
dispensable from the adenovirus genome (Jones and Shenk, 1978), the current<br>
adenovirus vectors, with the help of 293 cells, carry foreign DNA hi either the El, the<br>
D3 or both regions (Graham and Prevec, 1991). In nature, adenovirus can package<br>
approximately 105% of the wild-type genome (Ghosh-Choudhury et al, 1987),<br>
providing capacity for about 2 extra kB of DNA. Combined with the approximately 5.5<br>
kB of DNA that is replaceable in the El and E3 regions, the maximum capacity of the<br>
current adenovirus vector is under 7.5 kB, or about 15% of the total length of the<br>
vector. More than 80% of the adenovirus viral genome remains in the vector backbone<br>
and is the source of vector-borne cytotoxicity. Also, the replication deficiency of the<br>
El-deleted virus is incomplete. For example, leakage of viral gene expression has been<br>
observed with the currently available vectors at high multiplicities of infection (MOI)<br>
(Mulligan, 1993).<br>
Helper cell lines may be derived from human cells such as human<br>
embryonic kidney cells, muscle cells, hematopoietic cells or other human embryonic<br>
mesenchymal or epithelial cells. Alternatively, the helper cells may be derived from the<br>
cells of other mammalian species that are permissive for human adenovirus. Such cells<br>
include, e.g., Vero cells or other monkey embryonic mesenchymal or epithelial cells.<br>
As stated above, the currently preferred helper cell line is 293.<br>
Recently, Racher et al (1995) disclosed improved methods for culturing<br>
293 cells and propagating adenovirus. In one format, natural cell aggregates are grown<br>
by inoculating individual cells into 1 liter siliconized spinner flasks (Techne,<br>
Cambridge, UK) containing 100-200 ml of medium. Following stirring at 40 rpm, the<br>
cell viability is estimated with trypan blue. In another format, Fibra-Cei microcarriers<br>
(Bibby Sterlin, Stone, UK) (5 g/1) is employed as follows. A cell inoculum,<br>
resuspended in 5 ml of medium, is added to the carrier (50 ml) in a 250 ml Erlenmeyer<br>
flask and left stationary, with occasional agitation, for 1 to 4 h. The medium is then<br>
replaced with 50 ml of fresh medium and shaking initiated. For virus production, cells<br>
are allowed to grow to about 80% confluence, after which time the medium is replaced<br>
(to 25% of the final volume) and adenovirus added at an MOI of 0.05. Cultures are left<br>
stationary overnight, following which the volume is increased to 100% and shaking<br>
commenced for another 72 h.<br>
Other than the requirement that the adenovirus vector be replication<br>
defective, or at least conditionally defective, the nature of the adenovirus vector is not<br>
believed to be crucial to the successful practice of the invention. The adenovirus may<br>
be of any of the 42 different known serotypes or subgroups A-F. Adenovirus type 5 of<br>
subgroup C is the preferred starting material in order to obtain a conditional replicationdefective<br>
adenovirus vector for use hi the present invention, since Adenovirus type 5 is<br>
a human adenovirus about which a great deal of biochemical and genetic information is<br>
known, and it has historically been used for most constructions employing adenovirus<br>
as a vector.<br>
As stated above, the typical vector according to the present invention is<br>
replication defective and will not have an adenovirus El region. Thus, it will be most<br>
convenient to introduce the polynucleotide encoding the gene of interest at the position<br>
from which the El-coding sequences have been removed. However, the position of<br>
insertion of the construct within the adenovirus sequences is not critical to the<br>
invention. The polynucleotide encoding the gene of interest may also be inserted in lieu<br>
of the deleted E3 region in E3 replacement vectors as described by Karlsson et al.<br>
(1986) or hi the E4 region where a helper cell line or helper virus complements the E4<br>
defect.<br>
Adenovirus is easy to grow and manipulate and exhibits broad host<br>
range in vitro and in vivo. This group of viruses can be obtained in high liters, e.g., 109-<br>
1011 plaque-forming units per ml, and they are highly infective. The life cycle of<br>
adenovirus does not require integration into the host cell genome. The foreign genes<br>
delivered by adenovirus vectors are episomal and, therefore, have low genotoxicity to<br>
host cells. No side effects have been reported in studies of vaccination with wild-type<br>
adenovirus (Couch et al, 1963; Top et al, 1971), demonstrating then- safety and<br>
therapeutic potential as in vivo gene transfer vectors.<br>
Adenovirus vectors have been used in eukaryotic gene expression<br>
(Levrero et al, 1991; Gomez-Foix et al, 1992) and vaccine development (Grunhaus<br>
and Horwitz, 1992; Graham and Prevec, 1992). Recently* animal studies suggested that<br>
recombinant adenovirus could be used for gene therapy (Stratford-Perricaudet and<br>
Perricaudet, 1991; Stratford-Perricaudet et al, 1990; Rich et al, 1993). Studies hi<br>
administering recombinant adenovirus to different tissues include trachea instillation<br>
(Rosenfeld et al, 1991; Rosenfeld et al, 1992), muscle injection (Ragot et al, 1993),<br>
peripheral intravenous injections (Herz and Gerard, 1993) and stereotactic inoculation<br>
into the brain (Le Gal La Salle et al., 1993).<br>
2. RETROVIRUSES<br>
The retroviruses are a group of single-stranded RNA viruses<br>
characterized by an ability to convert their RNA to double-stranded DNA in infected<br>
cells by a process of reverse-transcription (Coffin, 1990). The resulting DNA then<br>
stably integrates into cellular chromosomes as a provirus and directs synthesis of viral<br>
proteins. The integration results in the retention of the viral gene sequences hi the<br>
recipient cell and its descendants. The retroviral genome contains three genes, gag, pol,<br>
and env that code for capsid proteins, polymerase enzyme, and envelope components,<br>
respectively. A sequence found upstream from the gag gene contains a signal for<br>
packaging of the genome into virions. Two long terminal repeat (LTR) sequences are<br>
present at the 5' and 3' ends of the viral genome. These contain strong promoter and<br>
enhancer sequences and are also required for integration in the host cell genome<br>
(Coffin, 1990).<br>
In order to construct a retroviral vector, a nucleic acid encoding one or<br>
more oligonucleotide or polynucleotide sequences of interest is inserted into the viral<br>
genome in the place of certain viral sequences to produce a virus that is replicationdefective.<br>
In order to produce virions, a packaging cell line containing the gag, pol, and<br>
env genes but without the LTR and packaging components is constructed (Mann et al.,<br>
1983). When a recombinant plasmid containing a cDNA, together with the retroviral<br>
LTR and packaging sequences is introduced into this cell line (by calcium phosphate<br>
precipitation for example), the packaging sequence allows the RNA transcript of the<br>
recombinant plasmid to be packaged into viral particles, which are then secreted into<br>
the culture media (Nicolas and Rubenstein, 1988; Temin, 1986; Mann et al, 1983).<br>
The media containing the recombinant retroviruses is then collected, optionally<br>
concentrated, and used for gene transfer. Retroviral vectors are able to infect a broad<br>
variety of cell types. However, integration and stable expression require the division of<br>
host cells (Paskind etal, 1975).<br>
A novel approach designed to allow specific targeting of retrovirus<br>
vectors was recently developed based on the chemical modification of a retrovirus by<br>
the chemical addition of lactose residues to the viral envelope. This modification could<br>
permit the specific infection of hepatocytes via sialoglycoprotein receptors.<br>
A different approach to targeting of recombinant retroviruses was<br>
designed in which biotinylated antibodies against a retroviral envelope protein and<br>
against a specific cell receptor were used. The antibodies were coupled via the biotin<br>
components by using streptavidin (Roux et al., 1989). Using antibodies against major<br>
histocompatibility complex class I and class II antigens, they demonstrated the infection<br>
of a variety of human cells that bore those surface antigens with an ecotropic virus in<br>
vitro (Roux etal, 1989).<br>
3. ADENO-ASSOCIATED VIRUSES<br>
AAV (Ridgeway, 1988; Hermonat and Muzycska, 1984) is a parovirus,<br>
discovered as a contamination of adenoviral stocks. It is a ubiquitous virus (antibodies<br>
are present in 85% of the US human population) that has not been linked to any disease.<br>
It is also classified as a dependovirus, because its replications is dependent on the<br>
presence of a helper virus, such as adenovirus. Five serotypes have been isolated, of<br>
which AAV-2 is the best characterized. AAV has a single-stranded linear DNA that is<br>
encapsidated into capsid proteins VP1, VP2 and VP3 to form an icosahedral virion of<br>
20 to 24 nm in diameter (Muzyczka and McLaughlin, 1988).<br>
The AAV DNA is approximately 4.7 kilobases long. It contains two<br>
open reading frames and is flanked by two ITRs. There are two major genes in the<br>
AAV genome: rep and cap. The rep gene codes for proteins responsible for viral<br>
replications, whereas cap codes for capsid protein VP1-3. Each ITR forms a T-shaped<br>
hairpin structure. These terminal repeats are the only essential cis components of the<br>
AAV for chromosomal integration. Therefore, the AAV can be used as a vector with<br>
all viral coding sequences removed and replaced by the cassette of genes for delivery.<br>
Three viral promoters have been identified and named p5, pi 9, and p40, according to<br>
their map position. Transcription from p5 and p!9 results hi production of rep proteins,<br>
and transcription from p40 produces the capsid proteins (Hermonat and Muzyczka,<br>
1984).<br>
There are several factors that prompted researchers to study the<br>
possibility of using rAAV as an expression vector One is that the requirements for<br>
delivering a gene to integrate into the host chromosome are surprisingly few. It is<br>
necessary to have the 145-bp ITRs, which are only 6% of the AAV genome. This<br>
leaves room in the vector to assemble a 4.5-kb DNA insertion. While this carrying<br>
capacity may prevent the AAV from delivering large genes, it is amply suited for<br>
delivering the antisense constructs of the present invention.<br>
AAV is also a good choice of delivery vehicles due to its safety. There<br>
is a relatively complicated rescue mechanism: not only wild type adenovirus but also<br>
AAV genes are required to mobilize rAAV. Likewise, AAV is not pathogenic and not<br>
associated with any disease. The removal of viral coding sequences minimizes immune<br>
reactions to viral gene expression, and therefore, rAAV does not evoke an inflammatory<br>
response.<br>
4. OTHER VIRAL VECTORS AS EXPRESSION CONSTRUCTS<br>
Other viral vectors may be employed as expression constructs hi the<br>
present invention for the delivery of oligonucleotide or polynucleotide sequences to a<br>
host cell. Vectors derived from viruses such as vaccinia virus (Ridgeway, 1988;<br>
Coupar et al, 1988), lentiviruses, polio viruses and herpes viruses may be employed.<br>
They offer several attractive features for various mammalian cells (Friedmann, 1989;<br>
Ridgeway, 1988; Coupar etal., 1988; Horwich etal, 1990).<br>
With the recent recognition of defective hepatitis B viruses, new insight<br>
was gained into the structure-function relationship of different viral sequences. In vitro<br>
studies showed that the virus could retain the ability for helper-dependent packaging<br>
and reverse transcription despite the deletion of up to 80% of its genome (Horwich et<br>
al., 1990). This suggested that large portions of the genome could be replaced with<br>
foreign genetic material. The hepatotropism and persistence (integration) were<br>
particularly attractive properties for liver-directed gene transfer. Chang et al (1991)<br>
introduced the chloramphenicol acetyltransferase (CAT) gene into duck hepatitis B<br>
virus genome in the place of the polymerase, surface, and pre-surface coding sequences.<br>
It was cotransfected with wild-type virus into an avion hepatoma cell line. Culture<br>
media containing high titers of the recombinant virus were used to infect primary<br>
duckling hepatocytes. Stable CAT gene expression was detected for at least 24 days<br>
after transfection (Chang et al., 1991).<br>
5. NON-VIRAL VECTORS<br>
In order to effect expression of the oligonucleotide or polynucleotide<br>
sequences of the present invention, the expression construct must be delivered into a<br>
cell. This delivery may be accomplished in vitro, as in laboratory procedures for<br>
transforming cells lines, or in vivo or ex vivo, as in the treatment of certain disease<br>
states. As described above, one preferred mechanism for delivery is via viral infection<br>
where the expression construct is encapsulated in an infectious viral particle.<br>
Once the expression construct has been delivered into the cell the nucleic<br>
acid encoding the desired oligonucleotide or polynucleotide sequences may be<br>
positioned and expressed at different sites. In certain embodiments, the nucleic acid<br>
encoding the construct may be stably integrated into the genome of the cell. This<br>
integration may be in the specific location and orientation via homologous<br>
recombination (gene replacement) or it may be integrated in a random, non-specific<br>
location (gene augmentation). In yet further embodiments, the nucleic acid may be<br>
stably maintained in the cell as a separate, episomal segment of DNA. Such nucleic<br>
acid segments or "episomes" encode sequences sufficient to permit maintenance and<br>
replication independent of or in synchronization with the host cell cycle. How the<br>
expression construct is delivered to a cell and where in the cell the nucleic acid remains<br>
is dependent on the type of expression construct employed.<br>
In certain embodiments of the invention, the expression construct<br>
comprising one or more oligonucleotide or polynucleotide sequences may simply<br>
consist of naked recombinant DNA or plasmids. Transfer of the construct may be<br>
performed by any of the methods mentioned above which physically or chemically<br>
permeabilize the cell membrane. This is particularly applicable for transfer in vitro but<br>
it may be applied to in vivo use as well. Dubensky et al. (1984) successfully injected<br>
polyomavirus DNA in the form of calcium phosphate precipitates into liver and spleen<br>
of adult and newborn mice demonstrating active viral replication and. acute infection.<br>
Benvenisty and Reshef (1986) also demonstrated that direct intraperitoneal injection of<br>
calcium phosphate-precipitated plasmids results in expression of the transfected genes.<br>
It is envisioned that DNA encoding a gene of interest may also be transferred hi a<br>
similar manner in vivo and express the gene product<br>
Another embodiment of the invention for transferring a naked DNA<br>
expression construct into cells may involve particle bombardment. This method<br>
depends on the ability to accelerate DNA-coated microprojectiles to a high velocity<br>
allowing them to pierce cell membranes and enter cells without killing them (Klein et<br>
al, 1987). Several devices for accelerating small particles have been developed. One<br>
such device relies on a high voltage discharge to generate an electrical current, which hi<br>
turn provides the motive force (Yang et al, 1990). The microprojectiles used have<br>
consisted of biologically inert substances such as tungsten or gold beads.<br>
Selected organs including the liver, skin, and muscle tissue of rats and<br>
mice have been bombarded in vivo (Yang et al., 1990; Zelenin et al, 1991). This may<br>
require surgical exposure of the tissue or cells, to eliminate any intervening tissue<br>
between the gun and the target organ, i.e. ex vivo treatment. Again, DNA encoding a<br>
particular gene may be delivered via this method and still be incorporated by the present<br>
invention.<br>
ANTISENSE OLIGONUCLEOTIDES<br>
The end result of the flow of genetic information is the synthesis of<br>
protein. DNA is transcribed by polymerases into messenger RNA and translated on the<br>
ribosome to yield a folded, functional protein. Thus there are several steps along the<br>
route where protein synthesis can be inhibited. The native DNA segment coding for a<br>
polypeptide described herein, as all such mammalian DNA strands, has two strands: a<br>
sense strand and an antisense strand held together by hydrogen bonding. The<br>
messenger RNA coding for polypeptide has the same nucleotide sequence as the sense<br>
DNA strand except that the DNA thymidine is replaced by uridine. Thus, synthetic<br>
antisense nucleotide sequences will bind to a mRNA and inhibit expression of the<br>
protein encoded by that mRNA.<br>
The targeting of antisense oligonucleotides to mRNA is thus one<br>
mechanism to shut down protein synthesis, and, consequently, represents a powerful<br>
and targeted therapeutic approach. For example, the synthesis of polygalactauronase<br>
and the muscarine type 2 acetylcholine receptor are inhibited by antisense<br>
oligonucleotides directed to then- respective mRNA sequences (U. S. Patent 5,739,119<br>
and U. S. Patent 5,759,829, each specifically incorporated herein by reference in its<br>
entirety). Further, examples of antisense inhibition have been demonstrated with the<br>
nuclear protein cyclin, the multiple drug resistance gene (MDG1), ICAM-1, E-selectin,<br>
STK-1, striatal GABAA receptor and human EOF (Jaskulski et al., 1988;<br>
Vasanthakumar and Ahmed, 1989; Peris et al, 1998; U. S. Patent 5,801,154; U. S.<br>
Patent 5,789,573; U. S. Patent 5,718,709 and U. S. Patent 5,610,288, each specifically<br>
incorporated herein by reference in its entirety). Antisense constructs have also been<br>
described that inhibit and can be used to treat a variety of abnormal cellular<br>
proliferations, e.g. cancer (U. S. Patent 5,747,470; U. S. Patent 5,591,317 and U. S.<br>
Patent 5,783,683, each specifically incorporated herein by reference in its entirety).<br>
Therefore, in exemplary embodiments, the invention provides<br>
oligonucleotide sequences that comprise all, or a portion of, any sequence that is<br>
capable of specifically binding to polynucleotide sequence described herein, or a<br>
complement thereof. In one embodiment, the antisense oligonucleotides comprise<br>
DNA or derivatives thereof. In another embodiment, the oligonucleotides comprise<br>
RNA or derivatives thereof. In a third embodiment, the oligonucleotides are modified<br>
DNAs comprising a phosphorothioated modified backbone. In a fourth embodiment,<br>
the oligonucleotide sequences comprise peptide nucleic acids or derivatives thereof. In<br>
each case, preferred compositions comprise a sequence region that is complementary,<br>
and more preferably substantially-complementary, and even more preferably,<br>
completely complementary to one or more portions of polynucleotides disclosed herein.<br>
Selection of antisense compositions specific for a given gene sequence is<br>
based upon analysis of the chosen target sequence (i.e. in these illustrative examples the<br>
rat and human sequences) and determination of secondary structure, Tm, binding<br>
energy, relative stability, and antisense compositions were selected based upon their<br>
relative inability to form dimers, hairpins, or other secondary structures that would<br>
reduce or prohibit specific binding to the target mRNA hi a host cell.<br>
Highly preferred target regions of the mRNA, are those which are at or<br>
near the AUG translation initiation codon, and those sequences which were<br>
substantially complementary to 5' regions of the mRNA. These secondary structure<br>
analyses and target site selection considerations were performed using v.4 of the<br>
OLIGO primer analysis software (Rychlik, 1997) and the BLASTN 2.0.5 algorithm<br>
software (Altschul etal, 1997).<br>
The use of an antisense delivery method employing a short peptide<br>
vector, termed MPG (27 residues), is also contemplated. The MPG peptide contains a<br>
hydrophobic domain derived from the fusion sequence of HIV gp41 and a hydrophilic<br>
domain from the nuclear localization sequence of SV40 T-antigen (Morris et al., 1997).<br>
It has been demonstrated that several molecules of the MPG peptide coat the antisense<br>
oligonucleotides and can be delivered into cultured mammalian cells in less than 1 hour<br>
with relatively high efficiency (90%). Further, the interaction with MPG strongly<br>
increases both the stability of the oligonucleotide to nuclease and the ability to cross the<br>
plasma membrane (Morris et al., 1997).<br>
RlBOZYMES<br>
Although proteins traditionally have been used for catalysis of nucleic<br>
acids, another class of macromolecules has emerged as useful in this endeavor.<br>
Ribozymes are RNA-protein complexes that cleave nucleic acids in a site-specific<br>
fashion. Ribozymes have specific catalytic domains that possess endonuclease activity<br>
(Kim and Cech, 1987; Gerlach etal., 1987; Forster and Symons, 1987). For example, a<br>
large number of ribozymes accelerate phosphoester transfer reactions with a high<br>
degree of specificity, often cleaving only one of several phosphoesters in an<br>
oligonucleotide substrate (Cech et al., 1981; Michel and Westhof, 1990; Reinhold-<br>
Hurek and Shub, 1992). This specificity has been attributed to the requirement that the<br>
substrate bind via specific base-pairing interactions to the internal guide sequence<br>
("IGS") of the ribozyme prior to chemical reaction.<br>
Ribozyme catalysis has primarily been observed as part of sequencespecific<br>
cleavage/ligation reactions involving nucleic acids (Joyce, 1989; Cech et al,<br>
1981). For example, U. S. Patent No. 5,354,855 (specifically incorporated herein by<br>
reference) reports that certain ribozymes can act as endonucleases with a sequence<br>
specificity greater than that of known ribonucleases and approaching that of the DNA<br>
restriction enzymes. Thus, sequence-specific ribozyme-mediated inhibition of gene<br>
expression may be particularly suited to therapeutic applications (Scanlon et al, 1991;<br>
Sarver et al., 1990). Recently, it was reported that ribozymes elicited genetic changes<br>
in some cells lines to which they were applied; the altered genes included the oncogenes<br>
H-ras, c-fos and genes of HIV. Most of this work involved the modification of a target<br>
mRNA, based on a specific mutant codon that is cleaved by a specific ribozyme.<br>
Six basic varieties of naturally-occurring enzymatic RNAs are known<br>
presently. Each can catalyze the hydrolysis of RNA phosphodiester bonds in trans (and<br>
thus can cleave other RNA molecules) under physiological conditions. In general,<br>
enzymatic nucleic acids act by first binding to a target RNA. Such binding occurs<br>
through the target binding portion of a enzymatic nucleic acid which is held in close<br>
proximity to an enzymatic portion of the molecule that acts to cleave the target RNA.<br>
Thus, the enzymatic nucleic acid first recognizes and then binds a target RNA through<br>
complementary base-pairing, and once bound to the correct site, acts enzymatically to<br>
cut the target RNA. Strategic cleavage of such a target RNA will destroy its ability to<br>
direct synthesis of an encoded protein. After an enzymatic nucleic acid has bound and<br>
cleaved its RNA target, it is released from that RNA to search for another target and can<br>
repeatedly bind and cleave new targets.<br>
The enzymatic nature of a ribozyme is advantageous over many<br>
technologies, such as antisense technology (where a nucleic acid molecule simply binds<br>
to a nucleic acid target to block its translation) since the concentration of ribozyme<br>
necessary to affect a therapeutic treatment is lower than that of an antisense<br>
oligonucleotide. This advantage reflects the ability of the ribozyme to act<br>
enzymatically. Thus, a single ribozyme molecule is able to cleave many molecules of<br>
target RNA. In addition, the ribozyme is a highly specific inhibitor, with the specificity<br>
of inhibition depending not only on the base pairing mechanism of binding to the target<br>
RNA, but also on the mechanism of target RNA cleavage. Single mismatches, or basesubstitutions,<br>
near the site of cleavage can completely eliminate catalytic activity of a<br>
ribozyme. Similar mismatches in antisense molecules do not prevent their action<br>
(Woolf et al, 1992). Thus, the specificity of action of a ribozyme is greater than that of<br>
an antisense oligonucleotide binding the same RNA site.<br>
The enzymatic nucleic acid molecule may be formed in a hammerhead,<br>
hairpin, a hepatitis 5 virus, group I intron or RNaseP RNA (in association with an RNA<br>
guide sequence) or Neurospora VS RNA motif. Examples of hammerhead motifs are<br>
described by Rossi etal. (1992). Examples of hairpin motifs are described by Hampel<br>
etal (Eur. Pat. Appl. Publ. No. EP 0360257), Hampel and Tritz (1989), Hampel etal.<br>
(1990) and U. S. Patent 5,631,359 (specifically incorporated herein by reference). An<br>
example of the hepatitis 8 virus motif is described by Perrotta and Been (1992); an<br>
example of the RNaseP motif is described by Guerrier-Takada etal. (1983);<br>
Neurospora VS RNA ribozyme motif is described by Collins (Saville and Collins,<br>
1990; Saville and Collins, 1991; Collins and Olive, 1993); and an example of the Group<br>
I intron is described in (U. S. Patent 4,987,071, specifically incorporated herein by<br>
reference). All that is important in an enzymatic nucleic acid molecule of this invention<br>
is that it has a specific substrate binding site which is complementary to one or more of<br>
the target gene RNA regions, and that it have nucleotide sequences within or<br>
surrounding that substrate binding site which impart an RNA cleaving activity to the<br>
molecule. Thus the ribozyme constructs need not be limited to specific motifs<br>
mentioned herein.<br>
In certain embodiments, it may be important to produce enzymatic<br>
cleaving agents which exhibit a high degree of specificity for the RNA of a desired<br>
target, such as one of the sequences disclosed herein. The enzymatic nucleic acid<br>
molecule is preferably targeted to a highly conserved sequence region of a target<br>
mRNA. Such enzymatic nucleic acid molecules can be delivered exogenously to<br>
specific cells as required. Alternatively, the ribozymes can be expressed from DNA or<br>
RNA vectors that are delivered to specific cells.<br>
Small enzymatic nucleic acid motifs (e.g., of the hammerhead or the<br>
hairpin structure) may also be used for exogenous delivery. The simple structure of<br>
these molecules increases the ability of the enzymatic nucleic acid to invade targeted<br>
regions of the mRNA structure. Alternatively, catalytic RNA molecules can be<br>
expressed within cells from eukaryotic promoters (e.g., Scanlon etaL, 1991; Kashani-<br>
Sabet etal, 1992; Dropulic etaL, 1992; Weerasinghe etaL, 1991; Ojwang etal, 1992;<br>
Chen etal., 1992; Sarver etal, 1990). Those skilled hi the art realize that any<br>
ribozyme can be expressed in eukaryotic cells from the appropriate DNA vector. The<br>
activity of such ribozymes can be augmented by their release from the primary<br>
transcript by a second ribozyme (Int. Pat. Appl. Publ. No. WO 93/23569, and Int. Pat.<br>
Appl. Publ. No. WO 94/02595, both hereby incorporated by reference; Ohkawa etal.,<br>
1992; Taira etal, 1991; and Ventura etal, 1993).<br>
Ribozymes may be added directly, or can be complexed with cationic<br>
lipids, lipid complexes, packaged within liposomes, or otherwise delivered to target<br>
cells. The RNA or RNA complexes can be locally administered to relevant tissues ex<br>
vivo, or in vivo through injection, aerosol inhalation, infusion pump or stent, with or<br>
without their incorporation in biopolymers.<br>
Ribozymes may be designed as described in Int. Pat. Appl. Publ. No.<br>
WO 93/23569 and Int. Pat. Appl. Publ. No. WO 94/02595, each: specifically<br>
incorporated herein by reference) and synthesized to be tested in vitro and in vivo, as<br>
described. Such ribozymes can also be optimized for delivery. While specific<br>
examples are provided, those in the art will recognize that equivalent RNA targets hi<br>
other species can be utilized when necessary.<br>
Hammerhead or hairpin ribozymes may be individually analyzed by<br>
computer folding (Jaeger etal., 1989) to assess whether the ribozyme sequences fold<br>
into the appropriate secondary structure. Those ribozymes with unfavorable<br>
intramolecular interactions between the binding arms and the catalytic core are<br>
eliminated from consideration. Varying binding arm lengths can be chosen to optimize<br>
activity. Generally, at least 5 or so bases on each arm are able to bind to, or otherwise<br>
interact with, the target RNA.<br>
Ribozymes of the hammerhead or hairpin motif may be designed to<br>
anneal to various sites in the mRNA message, and can be chemically synthesized. The<br>
method of synthesis used follows the procedure for normal RNA synthesis as described<br>
in Usman et al. (1987) and in Scaringe et al. (1990) and makes use of common nucleic<br>
acid protecting and coupling groups, such as dimethoxytrityl at the 5'-end, and<br>
phosphoramidites at the 3'-end. Average stepwise coupling yields are typically &gt;98%.<br>
Hairpin ribozymes may be synthesized in two parts and annealed to reconstruct an<br>
active ribozyme (Chowrira and Burke, 1992). Ribozymes may be modified extensively<br>
to enhance stability by modification with nuclease resistant groups, for example, 2'-<br>
amino, 2'-C-allyl, 2'-flouro, 2'-o-methyl, 2'-H (for a review see e.g., Usman and<br>
Cedergren, 1992). Ribozymes may be purified by gel electrophoresis using general<br>
methods or by high pressure liquid chromatography and resuspended in water.<br>
Ribozyme activity can be optimized by altering the length of the<br>
ribozyme binding arms, or chemically synthesizing ribozymes with modifications that<br>
prevent then* degradation by serum ribonucleases (see e.g., Int. Pat. Appl. Publ. No.<br>
WO 92/07065; Perrault et al, 1990; Pieken etal., 1991; Usman and Cedergren, 1992;<br>
Int. Pat. Appl. Publ. No. WO 93/15187; Int. Pat. Appl. Publ. No. WO 91/03162; Eur.<br>
Pat. Appl. Publ. No. 92110298.4; U. S. Patent 5,334,711; and Int. Pat. Appl. Publ. No.<br>
WO 94/13688, which describe various chemical modifications that can be made to the<br>
sugar moieties of enzymatic RNA molecules), modifications which enhance their<br>
efficacy in cells, and removal of stem II bases to shorten RNA synthesis times and<br>
reduce chemical requirements.<br>
Sullivan etal. (Int. Pat. Appl. Publ. No. WO 94/02595) describes the<br>
general methods for delivery of enzymatic RNA molecules. Ribozymes may be<br>
administered to cells by a variety of methods known to those familiar to the art,<br>
including, but not restricted to, encapsulation hi liposomes, by iontophoresis, or by<br>
incorporation into other vehicles, such as hydrogels, cyclodextrins, biodegradable<br>
nanocapsules, and bioadhesive microspheres. For some indications, ribozymes may be<br>
directly delivered ex vivo to cells or tissues with or without the aforementioned<br>
vehicles. Alternatively, the RNA/vehicle combination may be locally delivered by<br>
direct inhalation, by direct injection or by use of a catheter, infusion pump or stent<br>
Other routes of delivery include, but are not limited to, intravascular, intramuscular,<br>
subcutaneous or joint injection, aerosol inhalation, oral (tablet or pill form), topical,<br>
systemic, ocular, intraperitoneal and/or intrathecal delivery. More detailed descriptions<br>
of ribozyme delivery and administration are provided in Int. Pat. Appl. Publ. No. WO<br>
94/02595 and Int. Pat. Appl. Publ. No. WO 93/23569, each specifically incorporated<br>
herein by reference.<br>
Another means of accumulating high concentrations of a ribozyme(s)<br>
within cells is to incorporate the ribozyme-encoding sequences into a DNA expression<br>
vector. Transcription of the ribozyme sequences are driven from a promoter for<br>
eukaryotic RNA polymerase I (pol I), RNA polymerase n (pol II), or RNA polymerase<br>
III (pol III). Transcripts from pol II or pol III promoters will be expressed at high levels<br>
in all cells; the levels of a given pol II promoter in a given cell type will depend on the<br>
nature of the gene regulatory sequences (enhancers, silencers, etc.) present nearby.<br>
Prokaryotic RNA polymerase promoters may also be used, providing that the<br>
prokaryotic RNA polymerase enzyme is expressed in the appropriate cells (Elroy-Stein<br>
and Moss, 1990; Gao and Huang, 1993; Lieber etal, 1993; Zhou etal, 1990).<br>
Ribozymes expressed from such promoters can function in mammalian cells (e.g.<br>
Kashani-Saber etal., 1992; Ojwang etal., 1992; Chen etal., 1992; Yu etal., 1993;<br>
L'Huillier etal, 1992; Lisziewicz etal, 1993). Such transcription units can be<br>
incorporated into a variety of vectors for introduction into mammalian cells, including<br>
but not restricted to, plasmid DNA vectors, viral DNA vectors (such as adenovirus or<br>
adeno-associated vectors), or viral RNA vectors (such as retroviral, semliki forest virus,<br>
sindbis virus vectors).<br>
Ribozymes may be used as diagnostic tools to examine genetic drift and<br>
mutations within diseased cells. They can also be used to assess levels of the target<br>
RNA molecule. The close relationship between ribozyme activity and the structure of<br>
the target RNA allows the detection of mutations in any region of the molecule which<br>
alters the base-pairing and three-dimensional structure of the target RNA. By using<br>
multiple ribozymes, one may map nucleotide changes which are important to RNA<br>
structure and function in vitro, as well as in cells and tissues. Cleavage of target RNAs<br>
with ribozymes may be used to inhibit gene expression and define the role (essentially)<br>
of specified gene products in the progression of disease. In this manner, other genetic<br>
targets may be defined as important mediators of the disease. These studies will lead to<br>
better treatment of the disease progression by affording the possibility of combinational<br>
therapies (e.g., multiple ribozymes targeted to different genes, ribozymes coupled with<br>
known small molecule inhibitors, or intermittent treatment with combinations of<br>
ribozymes and/or other chemical or biological molecules). Other in vitro uses of<br>
ribozymes are well known in the art, and include detection of the presence of mRNA<br>
associated with an IL-5 related condition. Such RNA is detected by determining the<br>
presence of a cleavage product after treatment with a ribozyme using standard<br>
methodology.<br>
PEPTIDE NUCLEIC ACIDS<br>
In certain embodiments, the inventors contemplate the use of peptide<br>
nucleic acids (PNAs) in the practice of the methods of the invention. PNA is a DNA<br>
mimic in which the nucleobases are attached to a pseudopeptide backbone (Good and<br>
Nielsen, 1997). PNA is able to be utilized in a number methods that traditionally have<br>
used RNA or DNA. Often PNA sequences perform better in techniques than the<br>
corresponding RNA or DNA sequences and have utilities that are not inherent to RNA<br>
or DNA. A review of PNA including methods of making, characteristics of, and<br>
methods of using, is provided by Corey (1997) and is incorporated herein by reference.<br>
As such, in certain embodiments, one may prepare PNA sequences that are<br>
complementary to one or more portions of the ACE mRNA sequence, and such PNA<br>
compositions may be used to regulate, alter, decrease, or reduce the translation of ACEspecific<br>
mRNA, and thereby alter the level of ACE activity in a host cell to which such<br>
PNA compositions have been administered.<br>
PNAs have 2-aminoethyl-glycine linkages replacing the normal<br>
phosphodiester backbone of DNA (Nielsen et al, 1991; Hanvey et al., 1992; Hyrup and<br>
Nielsen, 1996; Neilsen, 1996). This chemistry has three important consequences:<br>
firstly, in contrast to DNA or phosphorothioate oligonucleotides, PNAs are neutral<br>
molecules; secondly, PNAs are achiral, which avoids the need to develop a<br>
stereoselective synthesis; and thirdly, PNA synthesis uses standard Boc (Dueholm et<br>
al., 1994) or Fmoc (Thomson et al., 1995) protocols for solid-phase peptide synthesis,<br>
although other methods, including a modified Merrifield method, have been used<br>
(Christenseneffl/., 1995).<br>
80<br>
PNA monomers or ready-made oligomers are commercially available<br>
from PerSeptive Biosystems (Framingham, MA). PNA syntheses by either Boc or<br>
Fmoc protocols are straightforward using manual or automated protocols (Norton et al.,<br>
1995). The manual protocol lends itself to the production of chemically modified PNAs<br>
or the simultaneous synthesis of families of closely related PNAs.<br>
As with peptide synthesis, the success of a particular PNA synthesis will<br>
depend on the properties of the chosen sequence. For example, while in theory PNAs<br>
can incorporate any combination of nucleotide bases, the presence of adjacent purines<br>
can lead to deletions of one or more residues in the product. In expectation of this<br>
difficulty, it is suggested that, in producing PNAs with adjacent purines, one should<br>
repeat the coupling of residues likely to be added inefficiently. This should be followed<br>
by the purification of PNAs by reverse-phase high-pressure liquid chromatography<br>
(Norton et al, 1995) providing yields and purity of product similar to those observed<br>
during the synthesis of peptides.<br>
Modifications of PNAs for a given application may be accomplished by<br>
coupling amino acids during solid-phase synthesis or by attaching compounds that<br>
contain a carboxylic acid group to the exposed N-terminal amine. Alternatively, PNAs<br>
can be modified after synthesis by coupling to an introduced lysine or cysteine. The<br>
ease with which PNAs can be modified facilitates optimization for better solubility or<br>
for specific functional requirements. Once synthesized, the identity of PNAs and thenderivatives<br>
can be confirmed by mass spectrometry. Several studies have made and<br>
utilized modifications.of PNAs (Norton et al, 1995; Haaima et al, 1996; Stetsenko et<br>
al, 1996; Petersen et al, 1995; Ulmann et al, 1996; Koch et al, 1995; Orum et al,<br>
1995; Footer et al, 1996; Griffith et al, 1995; Kremsky et al, 1996; Pardridge et al,<br>
1995; Boffa et al, 1995; Landsdorp et al, 1996; Gambacorti-Passerini et al, 1996;<br>
Armitage et al, 1997; Seeger et al, 1997; Ruskowski et al, 1997). U.S. Patent No.<br>
5,700,922 discusses PNA-DNA-PNA chimeric molecules and their uses in diagnostics,<br>
modulating protein in organisms, and treatment of conditions susceptible to<br>
therapeutics.<br>
In contrast to DNA and RNA, which contain negatively charged<br>
linkages, the PNA backbone is neutral. In spite of this dramatic alteration, PNAs<br>
recognize complementary DNA and RNA by Watson-Crick pairing (Egholm et at,<br>
1993), validating the initial modeling by Nielsen et al. (1991). PNAs lack 3' to 5'<br>
polarity and can bind in either parallel or antiparallel fashion, with the antiparallel mode<br>
being preferred (Egholm et al, 1993).<br>
Hybridization of DNA oligonucleotides to DNA and RNA is<br>
destabilized by electrostatic repulsion between the negatively charged phosphate<br>
backbones of the complementary strands. By contrast, the absence of charge repulsion<br>
hi PNA-DNA or PNA-RNA duplexes increases the melting temperature (rm) and<br>
reduces the dependence of Tm on the concentration of mono- or divalent cations<br>
(Nielsen et al., 1991). The enhanced rate and affinity of hybridization are significant<br>
because they are responsible for the surprising ability of PNAs to perform strand<br>
invasion of complementary sequences within relaxed double-stranded DNA. In<br>
addition, the efficient hybridization at inverted repeats suggests that PNAs can<br>
recognize secondary structure effectively within double-stranded DNA. Enhanced<br>
recognition also occurs with PNAs immobilized on surfaces, and Wang et al. have<br>
shown that support-bound PNAs can be used to detect hybridization events (Wang et<br>
al, 1996).<br>
One might expect that tight binding of PNAs to complementary<br>
sequences would also increase binding to similar (but not identical) sequences, reducing<br>
the sequence specificity of PNA recognition. As with DNA hybridization, however,<br>
selective recognition can be achieved by balancing oligomer length and incubation<br>
temperature. Moreover, selective hybridization of PNAs is encouraged by PNA-DNA<br>
hybridization being less tolerant of base mismatches than DNA-DNA hybridization.<br>
For example, a single mismatch within a 16 bp PNA-DNA duplex can reduce the Tm by<br>
up to 15°C (Egholm et al, 1993). This high level of discrimination has allowed the<br>
development of several PNA-based strategies for the analysis of point mutations (Wang<br>
et al, 1996; Carlsson et al, 1996; Thiede et al, 1996; Webb and Hurskainen, 1996;<br>
Perry-O'Keefeeffl/., 1996).<br>
High-affinity binding provides clear advantages for molecular<br>
recognition and the development of new applications for PNAs. For example, 11-13<br>
nucleotide PNAs inhibit the activity of telomerase, a ribonucleo-protein that extends<br>
82<br>
telomere ends using an essential RNA template, while the analogous DNA oligomers<br>
do not (Norton et al., 1996).<br>
Neutral PNAs are more hydrophobic than analogous DNA oligomers,<br>
and this can lead to difficulty solubilizing them at neutral pH, especially if the PNAs<br>
have a high purine content or if they have the potential to form secondary structures.<br>
Their solubility can be enhanced by attaching one or more positive charges to the PNA<br>
termini (Nielsen et al, 1991).<br>
Findings by Allfrey and colleagues suggest that strand invasion will<br>
occur spontaneously at sequences within chromosomal DNA (Boffa et al., 1995; Boffa<br>
et al., 1996). These studies targeted PNAs to triplet repeats of the nucleotides CAG and<br>
used this recognition to purify transcriptionally active DNA (Boffa et al, 1995) and to<br>
inhibit transcription (Boffa et at, 1996). This result suggests that if PNAs can be<br>
delivered within cells then they will have the potential to be general sequence-specific<br>
regulators of gene expression. Studies and reviews concerning the use of PNAs as<br>
antisense and anti-gene agents include Nielsen et al. (1993b), Hanvey et al (1992), and<br>
Good and Nielsen (1997). Koppelhus et al. (1997) have used PNAs to inhibit HTV-1<br>
inverse transcription, showing that PNAs may be used for antiviral therapies.<br>
Methods of characterizing the antisense binding properties of PNAs are<br>
discussed in Rose (1993) and Jensen et al (1997). Rose uses capillary gel<br>
electrophoresis to determine binding of PNAs to their complementary oligonucleotide,<br>
measuring the relative binding kinetics and stoichiometry. Similar types of<br>
measurements were made by Jensen et al. using BIAcore™ technology.<br>
Other applications of PNAs include use in DNA strand invasion (Nielsen<br>
et al, 1991), antisense inhibition (Hanvey et al, 1992), mutational analysis (Orum et<br>
al, 1993), enhancers of transcription (Mollegaard etal, 1994), nucleic acid purification<br>
(Orum et al, 1995), isolation of transcriptionally active genes (Boffa et al, 1995),<br>
blocking of transcription factor binding (Vickers et al, 1995), genome cleavage<br>
(Veselkov et al, 1996), biosensors (Wang et al, 1996), in situ hybridization (Thisted et<br>
al, 1996), and in a alternative to Southern blotting (Perry-O'Keefe, 1996).<br>
POLYPEPTIDE COMPOSITIONS<br>
The present invention, in other aspects, provides polypeptide<br>
compositions. Generally, a polypeptide of the invention will be an isolated polypeptide<br>
(or an epitope, variant, or active fragment thereof) derived from a mammalian species.<br>
Preferably, the polypeptide is encoded by a polynucleotide sequence disclosed herein or<br>
a sequence which hybridizes under moderately stringent conditions to a polynucleotide<br>
sequence disclosed herein. Alternatively, the polypeptide may be defined as a<br>
polypeptide which comprises a contiguous amino acid sequence from an amino acid<br>
sequence disclosed herein, or which polypeptide comprises an entire amino acid<br>
sequence disclosed herein.<br>
In the present invention, a polypeptide composition is also understood to<br>
comprise one or more polypeptides that are immunologically reactive with antibodies<br>
generated against a polypeptide of the invention, particularly a polypeptide having the<br>
amino acid sequence disclosed in SEQ ID NO: 786, 787, 791, 793, 795,797-799, 806<br>
or 809, or to active fragments, or to variants or biological functional equivalents<br>
thereof.<br>
Likewise, a polypeptide composition of the present invention is<br>
understood to comprise one or more polypeptides that are capable of eliciting antibodies<br>
that are immunologically reactive with one or more polypeptides encoded by one or<br>
more contiguous nucleic acid sequences contained hi SEQ ID NO: 1,11-13, 15,20, 23-<br>
27,29,30,33,34,39,41,43-46,51, 52,57, 58, 60,62,65-67, 69-71, 74,76, 79, 80, 84,<br>
86, 89-92, 95, 97, 98, 101, 110, 111, 113-119, 121-128, 130-134, 136, 138, 139, 141,<br>
143, 146-151, 153,154,157-160, 162-164, 167-178, 180, 181,183, 186-190, 192, 193,<br>
195-220,224,226-231,234,236,237,240,241,244-246,248,254,255,261, 262,266,<br>
270, 275, 280, 282, 283, 288, 289, 290, 292, 295, 301, 303, 304, 309, 311, 341-782,<br>
784, 785, 790, 792,794, 796, 800-804, 807, 808 and 810-826, or to active fragments, or<br>
to variants thereof, or to one or more nucleic acid sequences which hybridize to one or<br>
more of these sequences under conditions of moderate to high stringency. Particularly<br>
illustrative polypeptides include the amino acid sequences disclosed in SEQ ID NO:<br>
786, 787, 791, 793,795, 797-799, 806, 809 and 827.<br>
84<br>
As used herein, an active fragment of a polypeptide includes a whole or<br>
a portion of a polypeptide which is modified by conventional techniques, e.g.,<br>
mutagenesis, or by addition, deletion, or substitution, but which active fragment<br>
exhibits substantially the same structure function, antigenicity, etc., as a polypeptide as<br>
described herein.<br>
In certain illustrative embodiments, the polypeptides of the invention<br>
will comprise at least an immunogenic portion of a lung tumor protein or a variant<br>
thereof, as described herein. As noted above, a " lung tumor protein" is a protein that is<br>
expressed by lung tumor cells. Proteins that are lung tumor proteins also react<br>
detectably within an immunoassay (such as an ELISA) with antisera from a patient with<br>
lung cancer. Polypeptides as described herein may be of any length. Additional<br>
sequences derived from the native protein and/or heterologous sequences may be<br>
present, and such sequences may (but need not) possess further immunogenic or<br>
antigenic properties.<br>
An "immunogenic portion," as used herein is a portion of a protein that<br>
is recognized (i.e., specifically bound) by a B-cell and/or T-cell surface antigen<br>
receptor. Such immunogenic portions generally comprise at least 5 amino acid<br>
residues, more preferably at least 10, and still more preferably at least 20 amino acid<br>
residues of a lung tumor protein or a variant thereof. Certain preferred immunogenic<br>
portions include peptides in which an N-terminal leader sequence and/or<br>
transmembrane domain have been deleted. Other preferred immunogenic portions may<br>
contain a small N- and/or C-terminal deletion (e.g., 1-30 amino acids, preferably 5-15<br>
amino acids), relative to the mature protein.<br>
Immunogenic portions may generally be identified using well known<br>
techniques, such as those summarized in Paul, Fundamental Immunology, 3rd ed., 243-<br>
247 (Raven Press, 1993) and references cited therein. Such techniques include<br>
screening polypeptides for the ability to react with antigen-specific antibodies, antisera<br>
and/or T-cell lines or clones. As used herein, antisera and antibodies are "antigenspecific"<br>
if they specifically bind to an antigen (/.., they react with the protein in an<br>
ELISA or other immunoassay, and do not react detectably with unrelated proteins).<br>
Such antisera and antibodies may be prepared as described herein, and using well<br>
known techniques. An immunogenic portion of a native lung tumor protein is a portion<br>
that reacts with such antisera and/or T-cells at a level that is not substantially less than<br>
the reactivity of the full length polypeptide (e.g., in an ELISA and/or T-cell reactivity<br>
assay). Such immunogenic portions may react within such assays at a level that is<br>
similar to or greater than the reactivity of the full length polypeptide. Such screens may<br>
generally be performed using methods well known to those of ordinary skill in the art,<br>
such as those described in Harlow and Lane, Antibodies: A Laboratory Manual, Cold<br>
Spring Harbor Laboratory, 1988. For example, a polypeptide may be immobilized qn a<br>
solid support and contacted with patient sera to allow binding of antibodies within the<br>
sera to the immobilized polypeptide. Unbound sera may then be removed and bound<br>
antibodies detected using, for example, 125I-labeled Protein A.<br>
As noted above, a composition may comprise a variant of a native lung<br>
tumor protein. A polypeptide "variant," as used herein, is a polypeptide that differs<br>
from a native lung tumor protein in one or more substitutions, deletions, additions<br>
and/or insertions, such that the immunogenicity of the polypeptide is not substantially<br>
diminished. In other words, the ability of a variant to react with antigen-specific<br>
antisera may be enhanced or unchanged, relative to the native protein, or may be<br>
diminished by less than 50%, and preferably less than 20%, relative to the native<br>
protein. Such variants may generally be identified by modifying one of the above<br>
polypeptide sequences and evaluating the reactivity of the modified polypeptide with<br>
antigen-specific antibodies or antisera as described herein. Preferred variants include<br>
those in which one or more portions, such as an N-terminal leader sequence or<br>
transmembrane domain, have been removed. Other preferred variants include variants<br>
in which a small portion (e.g., 1-30 amino acids, preferably 5-15 amino acids) has been<br>
removed from the N- and/or C-terminal of the mature protein.<br>
Polypeptide variants encompassed by the present invention include those<br>
exhibiting at least about 70%, 75%, 80%, 85%, 90%, 91%, 92%, 93%, 94%, 95%, 96%,<br>
97%, 98%, or 99% or more identity (determined as described above) to the<br>
polypeptides disclosed herein.<br>
Preferably, a variant contains conservative substitutions. A<br>
"conservative substitution" is one in which an amino acid is substituted for another<br>
amino acid that has similar properties, such that one skilled in the art of peptide<br>
chemistry would expect the secondary structure and hydropathic-nature of the<br>
polypeptide to be substantially unchanged. Amino acid substitutions may generally be<br>
made on the basis of similarity in polarity, charge, solubility, hydrophobicity,<br>
hydrophilicity and/or the amphipathic nature of the residues. For example, negatively<br>
charged amino acids include aspartic acid and glutamic acid; positively charged amino<br>
acids include lysine and arginine; and amino acids with uncharged polar head groups<br>
having similar hydrophilicity values include leucine, isoleucine and valine; glycine and<br>
alanine; asparagine and glutamine; and serine, threonine, phenylalanine and tyrosine.<br>
Other groups of amino acids that may represent conservative changes include: (I) ala,<br>
pro, gly, glu, asp, gin, asn, ser, thr; (2) cys, ser, tyr, thr; (3) val, ile, leu, met, ala, phe;<br>
(4) lys, arg, his; and (5) phe, tyr, tip, his. A variant may also, or alternatively, contain<br>
nonconservative changes. In a preferred embodiment, variant polypeptides differ from<br>
a native sequence by substitution, deletion or addition of five amino acids or fewer.<br>
Variants may also (or alternatively) be modified by, for example, the deletion or<br>
addition of amino acids that have minimal influence on the immunogenicity, secondary<br>
structure and hydropathic nature of the polypeptide.<br>
As noted above, polypeptides may comprise a Signal (or leader)<br>
sequence at the N-terminal end of the protein, which co-translationally or posttranslationally<br>
directs transfer of the protein. The polypeptide may also be conjugated<br>
to a linker or other sequence for ease of synthesis, purification or identification of the<br>
polypeptide (e.g., poly-His), or to enhance binding of the polypeptide to a solid support.<br>
For example, a polypeptide may be conjugated to an immunoglobulin Fc region.<br>
Polypeptides may be prepared using any of a variety of well known<br>
techniques. Recombinant polypeptides encoded by DNA sequences as described above<br>
may be readily prepared from the DNA sequences using any of a variety of expression<br>
vectors known to those of ordinary skill in the art. Expression may be achieved in any<br>
appropriate host cell that has been transformed or transfected with an expression vector<br>
containing a DNA molecule that encodes a recombinant polypeptide. Suitable host<br>
cells include prokaryotes, yeast, and higher eukaryotic cells, such as mammalian cells<br>
and plant cells. Preferably, the host cells employed are E. coli, yeast or a mammalian<br>
cell line such as COS or CHO. Supernatants from suitable host/vector systems which<br>
secrete recombinant protein or polypeptide into culture media may be first concentrated<br>
using a commercially available filter. Following concentration, the concentrate may be<br>
applied to a suitable purification matrix such as an affinity matrix or an ion exchange<br>
resin. Finally, one or more reverse phase HPLC steps can be employed to further purify<br>
a recombinant polypeptide.<br>
Portions and other variants having less than about 100 ammo acids, and<br>
generally less than about 50 amino acids, may also be generated by synthetic means,<br>
using techniques well known to those of ordinary skill in the art. For example, such<br>
polypeptides may be synthesized using any of the commercially available solid-phase<br>
techniques, such as the Merrifield solid-phase synthesis method, where amino acids are<br>
sequentially added to a growing amino acid chain. See Merrifield, J. Am. Chem. Soc.<br>
55:2149-2146, 1963. Equipment for automated synthesis of polypeptides is<br>
commercially available from suppliers such as Perkin Elmer/Applied BioSystems<br>
Division (Foster City, CA), and may be operated according to the manufacturer's<br>
instructions.<br>
Within certain specific embodiments, a polypeptide may be a fusion<br>
protein that comprises multiple polypeptides as described herein, or that comprises at<br>
least one polypeptide as described herein and -an unrelated sequence, such as a known<br>
tumor protein. A fusion partner may, for example, assist hi providing T helper epitopes<br>
(an immunological fusion partner), preferably T helper epitopes recognized by humans,<br>
or may assist in expressing the protein (an expression enhancer) at higher yields than<br>
the native recombinant protein. Certain preferred fusion partners are both<br>
immunological and expression enhancing fusion partners. Other fusion partners may be<br>
selected so as to increase the solubility of the protein or to enable the protein to be<br>
targeted to desired intracellular compartments. Still further fusion partners include<br>
affinity tags, which facilitate purification of the protein.<br>
Fusion proteins may generally be prepared using standard techniques,<br>
including chemical conjugation. Preferably, a fusion protein is expressed as a<br>
recombinant protein, allowing the production of increased levels, relative to a non-fused<br>
protein, in an expression system. Briefly, DNA sequences encoding the polypeptide<br>
components may be assembled separately, and ligated into an appropriate expression<br>
vector. The 3' end of the DNA sequence encoding one polypeptide component is<br>
ligated, with or without a peptide linker, to the 5' end of a DNA sequence encoding the<br>
second polypeptide component so that the reading frames of the sequences are hi phase.<br>
This permits translation into a single fusion protein that retains the biological activity of<br>
both component polypeptides.<br>
. A peptide linker sequence may be employed to separate the first and<br>
second polypeptide components by a distance sufficient to ensure that each polypeptide<br>
folds into its secondary and tertiary structures. Such a peptide linker sequence is<br>
incorporated into the fusion protein using standard techniques well known in the art.<br>
Suitable peptide linker sequences may be chosen based on the following factors:<br>
(1) their ability to adopt a flexible extended conformation; (2) their inability to adopt a<br>
secondary structure that could interact with functional epitopes on the first and second<br>
polypeptides; and (3) the lack of hydrophobia or charged residues that might react with<br>
the polypeptide functional epitopes. Preferred peptide linker sequences contain Gly,<br>
Asn and Ser residues. Other near neutral ammo acids, such as Thr and Ala may also be<br>
used hi the linker sequence. Ammo acid sequences which may be usefully employed as<br>
linkers include those disclosed in Maratea et al,, Gene 40:39-46, 1985; Murphy et al,<br>
Proc. Natl. Acad. Sci. USA 53:8258-8262, 1986; U.S. Patent No. 4,935,233 and U.S.<br>
Patent No. 4,751,180. The linker sequence may generally be from 1 to about 50 amino<br>
acids in length. Linker sequences are not required when the first and second<br>
polypeptides have non-essential N-terminal amino acid regions that can be used to<br>
separate the functional domains and prevent steric interference.<br>
The ligated DNA sequences are operably linked to suitable<br>
transcriptional or translational regulatory elements. The regulatory elements<br>
responsible for expression of DNA are located only 5' to the DNA sequence encoding<br>
the first polypeptides. Similarly, stop codons required to end translation and<br>
transcription termination signals are only present 3' to the DNA sequence encoding the<br>
second polypeptide.<br>
Fusion proteins are also provided. Such proteins comprise a polypeptide<br>
as described herein together with an unrelated immunogenic protein. Preferably the<br>
immunogenic protein is capable of eliciting a recall response. Examples of such<br>
proteins include tetanus, tuberculosis and hepatitis proteins (see, for example, Stoute et<br>
al. New Engl. J. Med., 535:86-91,1997).<br>
Within preferred embodiments, an immunological fusion partner is<br>
derived from protein D, a surface protein of the gram-negative bacterium Haemophilus<br>
influenza B (WO 91/18926). Preferably, a protein D derivative comprises<br>
approximately the first third of the protein (e.g., the first N-terminal 100-110 amino<br>
acids), and a protein D derivative may be lipidated. Within certain preferred<br>
embodiments, the first 109 residues of a Lipoprotein D fusion partner is included on the<br>
N-terminus to provide the polypeptide with additional exogenous T-cell epitopes and to<br>
increase the expression level in E. coll (thus functioning as an expression enhancer).<br>
The lipid tail ensures optimal presentation of the antigen to antigen presenting cells.<br>
Other fusion partners include the non-structural protein from infiuenzae virus, NS1<br>
(hemaglutinin). Typically, the N-terminal 81 amino acids are used, although different<br>
fragments that include T-helper epitopes may be used.<br>
In another embodiment, the immunological fusion partner is the protein<br>
known as LYTA, or a portion thereof (preferably a C-terminal portion). LYTA is<br>
derived from Streptococcus pneumoniae, which synthesizes an N-acetyl-L-alanine<br>
amidase known as amidase LYTA (encoded by the LytA gene; Gene 43:265-292,<br>
1986). LYTA is an autolysin that specifically degrades certain bonds in the<br>
peptidoglycan backbone. The C-terminal domain of the LYTA protein is responsible<br>
for the affinity to the choline or to some choline analogues such as DEAE. This<br>
property has been exploited for the development of E. coli C-LYTA expressing<br>
plasmids useful for expression of fusion proteins. Purification of hybrid proteins<br>
containing the C-LYTA fragment at the amino terminus has been described (see<br>
Biotechnology 70:795-798, 1992). Within a preferred embodiment, a repeat portion of<br>
LYTA may be incorporated into a fusion protein. A repeat portion is found in the Cterminal<br>
region starting at residue 178. A particularly preferred repeat portion<br>
incorporates residues 188-305.<br>
In general, polypeptides (including fusion proteins) and polynucleotides<br>
as described herein are isolated. An "isolated" polypeptide or polynucleotide is one that<br>
is removed from its original environment. For example, a naturally-occurring protein is<br>
isolated if it is separated from some or all of the coexisting materials in the natural<br>
system. Preferably, such polypeptides are at least about 90% pure, more preferably at<br>
least about 95% pure and most preferably at least about 99% pure. A polynucleotide is<br>
considered to be isolated if, for example, it is cloned into a vector that is not a part of<br>
the natural environment.<br>
BINDING AGENTS<br>
The present invention further provides agents, such as antibodies and<br>
antigen-binding fragments thereof, that specifically bind to a lung tumor protein. As<br>
used herein, an antibody, or antigen-binding fragment thereof, is said to "specifically<br>
bind" to a lung tumor protein if it reacts at a detectable level (within, for example, an<br>
ELISA) with a lung tumor protein, and does not react detectably with unrelated proteins<br>
under similar conditions. As used herein, "binding" refers to a noncovalent association<br>
between two separate molecules such that a complex is formed. The ability to bind may<br>
be evaluated by, for example, determining a binding constant for the formation of the<br>
complex. The binding constant is the value obtained when the concentration of the<br>
complex is divided by the product of the component concentrations. In general, two<br>
compounds are said to "bind," in the context of the present invention, when the binding<br>
constant for complex formation exceeds about 103 L/mol. The binding constant may be<br>
determined using methods well known in the art.<br>
Binding agents may be further capable of differentiating between<br>
patients with and without a cancer, such as lung cancer, using the representative assays<br>
provided herein. In other words, antibodies or other binding agents that bind to a lung<br>
tumor protein will generate a signal indicating the presence of a cancer in at least about<br>
20% of patients with the disease, and will generate a negative signal indicating the<br>
absence of the disease in at least about 90% of individuals without the cancer. To<br>
determine whether a binding agent satisfies this requirement, biological samples (e.g.,<br>
blood, sera, sputum, urine and/or tumor biopsies) from patients with and without a<br>
cancer (as determined using standard clinical tests) may be assayed as described herein<br>
for the presence of polypeptides that bind to the binding agent. It will be apparent that a<br>
statistically significant number of samples with and without the disease should be<br>
assayed. Each binding agent should satisfy the above criteria; however, those of<br>
ordinary skill hi the art will recognize that binding agents may be used in combination<br>
to improve sensitivity.<br>
Any agent that satisfies the above requirements may be a binding agent.<br>
For example, a binding agent may be a ribosome, with or without a peptide component,<br>
an RNA molecule or a polypeptide. In a preferred embodiment, a binding agent is an<br>
antibody or an antigen-binding fragment thereof. Antibodies may be prepared by any<br>
of a variety of techniques known to those of ordinary skill hi the art. See, e.g., Harlow<br>
and Lane, Antibodies: A Laboratory Manual, Cold Spring Harbor Laboratory, 1988. In<br>
general, antibodies can be produced by cell culture techniques, including the generation<br>
of monoclonal antibodies as described herein, or via transfection of antibody genes into<br>
suitable bacterial or mammalian cell hosts, in order to allow for the production of<br>
recombinant antibodies. In one technique, an immunogen comprising the polypeptide<br>
is initially injected into any of a wide variety of mammals (e.g., mice, rats, rabbits,<br>
sheep or goats). In this step, the polypeptides of this invention may serve as the<br>
immunogen without modification. Alternatively, particularly for relatively short<br>
polypeptides, a superior immune response may be elicited if the polypeptide is joined to<br>
a carrier protein, such as bovine serum albumin or keyhole limpet hemocyanin. The<br>
immunogen is injected into the animal host, preferably according to a predetermined<br>
schedule incorporating one or more booster immunizations, and the animals are bled<br>
periodically. Polyclonal antibodies specific for the polypeptide may then be purified<br>
from such antisera by, for example, affinity chromatography using the polypeptide<br>
coupled to a suitable solid support.<br>
Monoclonal antibodies specific for an antigenic polypeptide of interest<br>
may be prepared, for example, using the technique of Kohler and Milstein, Eur. J.<br>
Immunol. 6:511-519, 1976, and improvements thereto. Briefly, these methods involve<br>
the preparation of immortal cell lines capable of producing antibodies having the<br>
desired specificity (i.e., reactivity with the polypeptide of interest). Such cell lines may<br>
be produced, for example, from spleen cells obtained from an animal immunized as<br>
described above. The spleen cells are then immortalized by, for example, fusion with a<br>
myeloma cell fusion partner, preferably one that is syngeneic with the immunized<br>
animal. A variety of fusion techniques may be employed. For example; the spleen cells<br>
and myeloma cells may be combined with a nonionic detergent for a few minutes and<br>
then plated at low density on a selective medium that supports the growth of hybrid<br>
cells, but not myeloma cells. A preferred selection technique uses HAT (hypoxanthine,<br>
aminopterin, thymidine) selection. After a sufficient time, usually about 1 to 2 weeks,<br>
colonies of hybrids are observed. Single colonies are selected and their culture<br>
supernatants tested for binding activity against the polypeptide. Hybridomas having<br>
high reactivity and specificity are preferred.<br>
Monoclonal antibodies may be isolated from the supernatants of growing<br>
hybridoma colonies. In addition, various techniques may be employed to enhance the<br>
yield, such as injection of the hybridoma cell line into the peritoneal cavity of a suitable<br>
vertebrate host, such as a mouse. Monoclonal antibodies may then be harvested from<br>
the ascites fluid or the blood. Contaminants may be removed from the antibodies by<br>
conventional techniques, such as chromatography, gel filtration, precipitation, and<br>
extraction. The polypeptides of this invention may be used in the purification process<br>
in, for example, an affinity chromatography step.<br>
Within certain embodiments, the use of antigen-binding fragments of<br>
antibodies may be preferred. Such fragments include Fab fragments, which may be<br>
prepared using standard techniques. Briefly, immunoglobulins may be purified from<br>
rabbit serum by affinity chromatography on Protein A bead columns (Harlow and Lane,<br>
Antibodies: A Laboratory Manual, Cold Spring Harbor Laboratory, 1988) and digested<br>
by papain to yield Fab and Fc fragments. The Fab and Fc fragments may be separated<br>
by affinity chromatography on protein A bead columns.<br>
Monoclonal antibodies of the present invention may be coupled to one or<br>
more therapeutic agents. Suitable agents in this regard include radionuclides,<br>
differentiation inducers, drugs, toxins, and derivatives thereof. Preferred radionuclides<br>
include "Y, 123I, 125I, 131I, l86Re, 188Re, 21lAt, and 212Bi. Preferred drugs include<br>
methotrexate, and pyrimidine and purine analogs. Preferred differentiation inducers<br>
include phorbol esters and butyric acid. Preferred toxins include ricin, abrin, dipthena<br>
toxin, cholera toxin, gelonin, Pseudomonas exotoxin, Shigella toxin, and pokeweed<br>
antiviral protein.<br>
A therapeutic agent may be coupled (e.g., covalently bonded) to a<br>
suitable monoclonal antibody either directly or indirectly (e.g., via a linker group). A<br>
direct reaction between an agent and an antibody is possible when each possesses a<br>
substituent capable of reacting with the other. For example, a nucleophilic group, such<br>
as an amino or sulfhydryl group, on one may be capable of reacting with a carbonylcontaining<br>
group, such as an anhydride or an acid halide, or with an alkyl group<br>
containing a good leaving group (e.g., a halide) on the other.<br>
Alternatively, it may be desirable to couple a therapeutic agent and an<br>
antibody via a linker group. A linker group can function as a spacer to distance an<br>
antibody from an agent hi order to avoid interference with binding capabilities. A<br>
linker group can also serve to increase the chemical reactivity of a substituent on an<br>
agent or an antibody, and thus increase the coupling efficiency. An increase in<br>
chemical reactivity may also facilitate the use of agents, or functional groups on agents,<br>
which otherwise would not be possible.<br>
It will be evident to those skilled in the art that a variety of bifunctional<br>
or polyfunctional reagents, both homo- and hetero-functional (such as those described<br>
in the catalog of the Pierce Chemical Co., Rockford, IL), may be employed as the linker<br>
group. Coupling may be effected, for example, through amino groups, carboxyl groups,<br>
sulfhydryl groups or oxidized carbohydrate residues. There are numerous references<br>
describing such methodology, e.g., U.S. Patent No. 4,671,958, to Rodwell etal.<br>
Where a therapeutic agent is more potent when free from the antibody<br>
portion of the immunoconjugates of the present invention, it may be desirable to use a<br>
linker group which is cleavable during or upon intemalization into a cell. A number of<br>
different cleavable linker groups have been described. The mechanisms for the<br>
intracellular release of an agent from these linker groups include cleavage by reduction<br>
of a disulfide bond (e.g., U.S. Patent No. 4,489,710, to Spitler), by irradiation of a<br>
photolabile bond (e.g., U.S. Patent No. 4,625,014, to Senter et a/.), by hydrolysis of<br>
derivatized ammo acid side chains (e.g., U.S. Patent No. 4,638,045, to Kohn et a/.), by<br>
serum complement-mediated hydrolysis (e.g., U.S. Patent No. 4,671,958, to Rodwell et<br>
al.), and acid-catalyzed hydrolysis (e.g., U.S. Patent No. 4,569,789, to Blattler et a/.).<br>
It may be desirable to couple more than one agent to an antibody. In one<br>
embodiment, multiple molecules of an agent are coupled to one antibody molecule. In<br>
another embodiment, more than one type of agent may be coupled to one antibody.<br>
Regardless of the particular embodiment, immunoconjugates with more than one agent<br>
may be prepared in a variety of ways. For example, more than one agent may be<br>
coupled directly to an antibody molecule, or linkers that provide multiple sites for<br>
attachment can be used. Alternatively, a carrier can be used.<br>
A carrier may bear the agents hi a variety of ways, including covalent<br>
bonding either directly or via a linker group. Suitable carriers include proteins such as<br>
albumins (e.g., U.S. Patent No. 4,507,234, to Kato et al.), peptides and polysaccharides<br>
such as aminodextran (e.g., U.S. Patent No. 4,699,784, to Shih et al). A carrier may<br>
also bear an agent by noncovalent bonding or by encapsulation, such as within a<br>
liposome vesicle (e.g., U.S. Patent Nos. 4,429,008 and 4,873,088). Carriers specific for<br>
radionuclide agents include radiohalogenated small molecules and chelating<br>
compounds. For example, U.S. Patent No. 4,735,792 discloses representative<br>
radiohalogenated small molecules and their synthesis. A radionuclide chelate may be<br>
formed from chelating compounds that include those containing nitrogen and sulfur<br>
atoms as the donor atoms for binding the metal, or metal oxide, radionuclide. For<br>
example, U.S. Patent No. 4,673,562, to Davison et al. discloses representative chelating<br>
compounds and their synthesis.<br>
A variety of routes of administration for the antibodies and<br>
immunoconjugates may be used. Typically, administration will be intravenous,<br>
intramuscular, subcutaneous or in the bed of a resected tumor. It will be evident that<br>
the precise dose of the antibody/immunoconjugate will vary depending upon the<br>
antibody used, the antigen density on the tumor, and the rate of clearance of the<br>
antibody.<br>
T CELLS<br>
Immunotherapeutic compositions may also, or alternatively, comprise T<br>
cells specific for a lung tumor protein. Such cells may generally be prepared in vitro or<br>
ex vivo, using standard procedures. For example, T cells may be isolated from bone<br>
marrow, peripheral blood, or a fraction of bone marrow or peripheral blood of a patient,<br>
using a commercially available cell separation system, such as the Isolex™ System,<br>
available from Nexell Therapeutics, Inc^ (Irvine, CA; see also U.S. Patent No.<br>
5,240,856; U.S. Patent No. 5,215,926; WO 89/06280; WO 91/16116 and WO<br>
92/07243). Alternatively, T cells may be derived from related or unrelated humans,<br>
non-human mammals, cell lines or cultures.<br>
T cells may be stimulated with a lung tumor polypeptide, polynucleotide<br>
encoding a lung tumor polypeptide and/or an antigen presenting cell (APC) that<br>
expresses such a polypeptide. Such stimulation is performed under conditions and for a<br>
time sufficient to permit the generation of T cells that are specific for the polypeptide.<br>
Preferably, a lung tumor polypeptide or polynucleotide is present within a delivery<br>
vehicle, such as a microsphere, to facilitate the generation of specific T cells.<br>
T cells are considered to be specific for a lung tumor polypeptide if the T<br>
cells specifically proliferate, secrete cytokines or kill target cells coated with the<br>
polypeptide or expressing a gene encoding the polypeptide. T cell specificity may be<br>
evaluated using any of a variety of standard techniques. For example, within a<br>
chromium release assay or proliferation assay, a stimulation index of more than two<br>
fold increase, in lysis and/or proliferation, compared to negative controls, indicates T<br>
cell specificity. Such assays may be performed, for example, as described in Chen et<br>
al, Cancer Res. 54:1065-1070, 1994. Alternatively, detection of the proliferation of<br>
T cells may be accomplished by a variety of known techniques. For example, T cell<br>
proliferation can be detected by measuring an increased rate of DNA synthesis (e.g., by<br>
pulse-labeling cultures of T cells with tritiated thymidine and measuring the amount of<br>
tritiated thymidine incorporated into DNA). Contact with a lung tumor polypeptide<br>
(100 ng/ml -100 ug/ml, preferably 200 ng/ml - 25 ug/ml) for 3 - 7 days should result in<br>
at least a two fold increase in proliferation of the T cells. Contact as described above<br>
for 2-3 hours should result in activation of the T cells, as measured using standard<br>
cytokine assays hi which a two fold increase in the level of cytokine release (e.g., TNF<br>
or IFN-y) is indicative of T cell activation (see Coligan et aL, Current Protocols in<br>
Immunology, vol. 1, Wiley Interscience (Greene 1998)). T cells that have been<br>
activated in response to a lung tumor polypeptide, polynucleotide or polypeptideexpressing<br>
APC may be CD4* and/or CD8+. Lung tumor protein-specific T cells may<br>
be expanded using standard techniques. Within preferred embodiments, the T cells are<br>
derived from a patient, a related donor or an unrelated donor, and are administered to<br>
the patient following stimulation and expansion.<br>
For therapeutic purposes, GD4+ or CD8+ T cells that proliferate in<br>
response to a lung tumor polypeptide, polynucleotide or APC can be expanded in<br>
number either in vitro or in vivo. Proliferation of such T cells in vitro may be<br>
accomplished in a variety of ways. For example, the T cells can be re-exposed to a lung<br>
tumor polypeptide, or a short peptide corresponding to an immunogenic portion of such<br>
a polypeptide, with or without the addition of T cell growth factors, such as interleukin-<br>
2, and/or stimulator cells that synthesize a lung tumor polypeptide. Alternatively, one<br>
or more T cells that proliferate in the presence of a lung tumor protein can be expanded<br>
in number by cloning. Methods for cloning cells are well known in the art, and include<br>
limiting dilution.<br>
PHARMACEUTICAL COMPOSITIONS<br>
In additional embodiments, the present invention concerns formulation<br>
of one or more of the polynucleotide, polypeptide, T-cell and/or antibody compositions<br>
disclosed herein in pharmaceutically-acceptable solutions for administration to a cell or<br>
an animal, either alone, or in combination with one or more other modalities of therapy.<br>
It will also be understood that, if desired, the nucleic acid segment,<br>
RNA, DNA or PNA compositions that express a polypeptide as disclosed herein may be<br>
administered hi combination with other agents as well, such as, e.g., other proteins or<br>
polypeptides or various pharmaceutically-active agents. In fact, there is virtually no<br>
limit to other components that may also be included, given that the additional agents do<br>
not cause a significant adverse effect upon contact with the target cells or host tissues.<br>
The compositions may thus be delivered along with various other agents as required in<br>
the particular instance.. Such compositions may be purified from host cells or other<br>
biological sources, or alternatively may be chemically synthesized as described herein.<br>
Likewise, such compositions may further comprise substituted or derivatized RNA or<br>
DNA compositions.<br>
Formulation of phannaceutically-acceptable excipients and carrier<br>
solutions is well-known to those of skill hi the art, as is the development of suitable<br>
dosing and treatment regimens for using the particular compositions described herein in<br>
a variety of treatment regimens, including e.g., oral, parenteral, intravenous, intranasal,<br>
and intramuscular administration and formulation.<br>
1. ORAL DELIVERY<br>
In certain applications, the pharmaceutical compositions disclosed herein<br>
may be delivered via oral administration to an animal. As such, these compositions<br>
may be formulated with an inert diluent or with an assimilable edible carrier, or they<br>
may be enclosed in hard- or soft-shell gelatin capsule, or they may be compressed into<br>
tablets, or they may be incorporated directly with the food of the diet.<br>
The active compounds may even be incorporated with excipients and<br>
used in the form of ingestible tablets, buccal tables, troches, capsules, elixirs,<br>
suspensions, syrups, wafers, and the like (Mathiowitz et a/., 1997; Hwang et al., 1998;<br>
U. S. Patent 5,641,515; U. S. Patent 5,580,579 and U. S. Patent 5,792,451, each<br>
specifically incorporated herein by reference in its entirety). The tablets, troches, pills,<br>
capsules and the like may also contain the following: a binder, as gum tragacanth,<br>
acacia, comstarch, or gelatin; excipients, such as dicalcium phosphate; a disintegrating<br>
agent, such as corn starch, potato starch, alginic acid and the like; a lubricant, such as<br>
magnesium stearate; and a sweetening agent, such as sucrose, lactose or saccharin may<br>
be added or a flavoring agent, such as peppermint, oil of wintergreen, or cherry<br>
flavoring. When the dosage unit form is a capsule, it may contain, in addition to<br>
materials of the above type, a liquid carrier. Various other materials may be present as<br>
coatings or to otherwise modify the physical form of the dosage unit. For instance,<br>
tablets, pills, or capsules may be coated with shellac, sugar, or both. A syrup of elixir<br>
may contain the active compound sucrose as a sweetening agent methyl and<br>
propylparabens as preservatives, a dye and flavoring, such as cherry or orange flavor!<br>
Of course, any material used in preparing any dosage unit form should be<br>
pharmaceutically pure and substantially non-toxic in the amounts employed. In<br>
addition, the active compounds may be incorporated into sustained-release preparation<br>
and formulations.<br>
Typically, these formulations may contain at least about 0.1% of the<br>
active compound or more, although the percentage of the active ingredient(s) may, of<br>
course, be varied and may conveniently be between about 1 or 2% and about 60% or<br>
70% or more of the weight or volume of the total formulation. Naturally, the amount of<br>
active compound(s) in each therapeutically useful composition may be prepared is such<br>
a way that a suitable dosage will be obtained in any given unit dose of the compound.<br>
Factors such as solubility, bioavailability, biological half-life, route of administration,<br>
product shelf life, as well as other pharmacological considerations will be contemplated<br>
by one skilled in the art of preparing such pharmaceutical formulations, and as such, a<br>
variety of dosages and treatment regimens may be desirable.<br>
For oral administration the compositions of the present invention may<br>
alternatively be incorporated with one or more excipients in the form of a mouthwash,<br>
dentifrice, buccal tablet, oral spray, or sublingual orally-administered formulation. For<br>
example, a mouthwash may be prepared incorporating the active ingredient in the<br>
required amount in an appropriate solvent, such as a sodium borate solution (Dobell's<br>
Solution). Alternatively, the active ingredient may be incorporated into an oral solution<br>
such as one containing sodium borate, glycerin and potassium bicarbonate, or dispersed<br>
in a dentifrice, or added in a therapeutically-effective amount to a composition that may<br>
include water, binders, abrasives, flavoring agents, foaming agents, and humectants.<br>
Alternatively the compositions may be fashioned into a tablet or solution form that may<br>
be placed under the tongue or otherwise dissolved in the mouth.<br>
2. INJECTABLE DELIVERY<br>
In certain circumstances it will be desirable to deliver the pharmaceutical<br>
compositions disclosed herein parenterally, intravenously, intramuscularly, or even<br>
intraperitoneally as described in U. S. Patent 5,543,158; U. S. Patent 5,641,515 and U.<br>
S. Patent 5,399,363 (each specifically incorporated herein by reference in its entirety).<br>
Solutions of the active compounds as free base or pharmacologically, acceptable salts<br>
may be prepared in water suitably mixed with a surfactant, such as<br>
hydroxypropylcellulose. Dispersions may also be prepared in glycerol, liquid<br>
polyethylene glycols, and mixtures thereof and in oils. Under ordinary conditions of<br>
storage and use, these preparations contain a preservative to prevent the growth of<br>
microorganisms.<br>
The pharmaceutical forms suitable for injectable use include sterile<br>
aqueous solutions or dispersions and sterile powders for the extemporaneous<br>
preparation of sterile injectable solutions or dispersions (U. S. Patent 5,466,468,<br>
specifically incorporated herein by reference in its entirety). In all cases the form must<br>
be sterile and must be fluid to the extent that easy syringability exists. It must be stable<br>
under the conditions of manufacture and storage and must be preserved against the<br>
contaminating action of microorganisms, such as bacteria and fungi. The carrier can be<br>
a solvent or dispersion medium containing, for example, water, ethanol, polyol (e.g.,<br>
glycerol, propylene glycol, and liquid polyethylene glycol, and the like), suitable<br>
mixtures thereof, and/or vegetable oils. Proper fluidity may be maintained, for<br>
example, by the use of a coating, such as lecithin, by the maintenance of the required<br>
particle size hi the case of dispersion and by the use of surfactants. The prevention of<br>
the action of microorganisms can be facilitated by various antibacterial and antifungal<br>
agents, for example, parabens, chlorobutanol, phenol, sorbic acid, thimerosal, and the<br>
like. In many cases, it will be preferable to include isotonic agents, for example, sugars<br>
or sodium chloride. Prolonged absorption of the injectable compositions can be<br>
brought about by the use in the compositions of agents delaying absorption, for<br>
example, aluminum monostearate and gelatin.<br>
For parenteral administration in an aqueous solution, for example, the<br>
solution should be suitably buffered if necessary and the liquid diluent first rendered<br>
isotonic with sufficient saline or glucose. These particular aqueous solutions are<br>
especially suitable for intravenous, intramuscular, subcutaneous and intraperitoneal<br>
administration. In this connection, a sterile aqueous medium that can be employed will<br>
be known to those of skill in the art in light of the present disclosure. For example, one<br>
dosage may be dissolved in 1 ml of isotonic NaCl solution and either added to 1000 ml<br>
of hypodermoclysis fluid or injected at the proposed site of infusion, (see for example,<br>
"Remington's Pharmaceutical Sciences" 15th Edition, pages 1035-1038 and 1570-<br>
1580). Some variation in dosage will necessarily occur depending on the condition of<br>
the subject being treated. The person responsible for administration will, in any event,<br>
determine the appropriate dose for the individual subject. Moreover, for human<br>
administration, preparations should meet sterility, pyrogenicity, and the general safety<br>
and purity standards as required by FDA Office of Biologies standards.<br>
Sterile injectable solutions are prepared by incorporating the active<br>
compounds in the required amount in the appropriate solvent with various of the other<br>
ingredients enumerated above, as required, followed by filtered sterilization. Generally,<br>
dispersions are prepared by incorporating the various sterilized active ingredients into a<br>
sterile vehicle which contains the basic dispersion medium and the required other<br>
ingredients from those enumerated above. In the case of sterile powders for the<br>
preparation of sterile injectable solutions, the preferred methods of preparation are<br>
vacuum-drying and freeze-drying techniques which yield a powder of the active<br>
ingredient plus any additional desired ingredient from a previously sterile-filtered<br>
solution thereof.<br>
The compositions disclosed herein may be formulated in a neutral or salt<br>
form. Pharmaceutically-acceptable salts, include the. acid addition salts (formed with<br>
the free amino groups of the protein) and which are formed with inorganic acids such<br>
as, for example, hydrochloric or phosphoric acids, or such organic acids as acetic,<br>
oxalic, tartaric, mandelic, and the like. Salts formed with the free carboxyl groups can<br>
also be derived from inorganic bases such as, for example, sodium, potassium,<br>
ammonium, calcium, or ferric hydroxides, and such organic bases as isopropylamine,<br>
trimethylamine, histidine, procaine and the like. Upon formulation, solutions will be<br>
administered in a manner compatible with the dosage formulation and in such amount<br>
as is therapeutically effective. The formulations are easily administered in a variety of<br>
dosage forms such as injectable solutions, drug-release capsules, and the like.<br>
As used herein, "carrier" includes any and all solvents, dispersion media,<br>
vehicles, coatings, diluents, antibacterial and antifungal agents, isotonic and absorption<br>
delaying agents, buffers, carrier solutions, suspensions, colloids, and the like. The use<br>
of such media and agents for pharmaceutical active substances is well known in the art<br>
Except insofar as any conventional media or agent is incompatible with the active<br>
ingredient, its use in the therapeutic compositions is contemplated. Supplementary<br>
active ingredients can also be incorporated into the compositions.<br>
The phrase "pharmaceutically-acceptable" refers to molecular entities<br>
and compositions that do not produce an allergic or similar untoward reaction when<br>
administered to a human. The preparation of an aqueous composition that contains a<br>
protein as an active ingredient is well understood in the art. Typically, such<br>
compositions are prepared as injectables, either as liquid solutions or suspensions; solid<br>
forms suitable for solution in, or suspension in, liquid prior to injection can also be<br>
prepared. The preparation can also be emulsified.<br>
3. NASAL DELIVERY<br>
In certain embodiments, the pharmaceutical compositions may be<br>
delivered by intranasal sprays, inhalation, and/or other aerosol delivery vehicles.<br>
Methods for delivering genes, nucleic acids, and peptide compositions directly to the<br>
lungs via nasal aerosol sprays has been described e.g., in U. S. Patent 5,756,353 and U.<br>
S. Patent 5,804,212 (each specifically incorporated herein by reference in its entirety).<br>
Likewise, the delivery of drugs using intranasal microparticie resins (Takenaga et at.,<br>
1998) and lysophosphatidyl-glycerol compounds (U. S. Patent 5,725,871, specifically<br>
incorporated herein by reference hi its entirety) are also well-known in the<br>
pharmaceutical arts. Likewise, transmucosal drug delivery in the form of a<br>
polytetrafluoroetheylene support matrix is described in U. S. Patent 5,780,045<br>
(specifically incorporated herein by reference hi its entirety).<br>
4. LIPOSOME-, NANOCAPSULE-, AND MICROPARTICLE-MEDIATED DELIVERY<br>
In certain embodiments, the inventors contemplate the use of liposomes,<br>
nanocapsules, microparticles, microspheres, lipid particles, vesicles, and the like, for the<br>
introduction of the compositions of the present invention into suitable host cells. In<br>
particular, the compositions of the present invention may be formulated for delivery<br>
either encapsulated in a lipid particle, a liposome, a vesicle, a nanosphere, or a<br>
nanoparticle or the like.<br>
Such formulations may be preferred for the introduction of<br>
pharmaceutically-acceptable formulations of the nucleic acids or constructs disclosed<br>
herein. The formation and use of liposomes is generally known to those of skill in the<br>
art (see for example, Couvreur et al, 1977; Couvreur, 1988; Lasic, 1998; which<br>
describes the use of liposomes and nanocapsules in the targeted antibiotic therapy for<br>
intraceUular bacterial infections and diseases). Recently, liposomes were developed<br>
with improved serum stability and circulation half-tunes (Gabizon and<br>
Papahadjopoulos, 1988; Allen and Choun, 1987; U. S. Patent 5,741,516, specifically<br>
incorporated herein by reference in its entirety). Further, various methods of liposome<br>
and liposome like preparations as potential drug carriers have been reviewed (Takakura,<br>
1998; Chandran et al, 1997; Margalit, 1995; U. S. Patent 5,567,434; U. S. Patent<br>
5,552,157; U, S. Patent 5,565,213; U. S. Patent 5,738,868 and U. S. Patent 5,795,587,<br>
each specifically incorporated herein by reference hi its entirety).<br>
Liposomes have been used successfully with a number of cell types that<br>
are normally resistant to transfection by other procedures including T cell suspensions,<br>
primary hepatocyte cultures and PC 12 cells (Renneisen et al, 1990; Muller et al,<br>
1990). In addition, liposomes are free of the DNA length constraints that are typical of<br>
viral-based delivery systems. Liposomes have been used effectively to introduce genes,<br>
drugs (Heath and Martin, 1986; Heath etal, 1986; Balazsovits etal, 1989; Fresta and<br>
Puglisi, 1996), radiotherapeutic agents (Pikul etal, 1987), enzymes (Imaizumi etal,<br>
1990a; Imaizumi etal, 1990b), viruses (Faller and Baltimore, 1984), transcription<br>
factors and allosteric effectors (Nicolau and Gersonde, 1979) into a variety of cultured<br>
cell lines and animals. In addition, several successful clinical trails examining the<br>
effectiveness of liposome-mediated drug delivery have been completed (Lopez-<br>
Berestein et al, 1985a; 1985b; Coune, 1988; Sculier et al, 1988). Furthermore, several<br>
studies suggest that the use of liposomes is not associated with autoimmune responses,<br>
toxicity or gonadal localization after systemic delivery (Mori and Fukatsu, 1992).<br>
Liposomes are formed from phospholipids that are dispersed in an<br>
aqueous medium and spontaneously form multilamellar concentric bilayer vesicles<br>
 (also termed multilamellar vesicles (MLVs). MLVs generally have diameters of from<br>
25 nm to 4 urn. Sonication of MLVs results in the formation of small unilamellar<br>
vesicles (SUVs) with diameters in the range of 200 to 500 A, containing an aqueous<br>
solution in the core.<br>
Liposomes bear resemblance to cellular membranes and are<br>
contemplated for use in connection with the present invention as carriers for the peptide<br>
compositions. They are widely suitable as both water- and lipid-soluble substances can<br>
be entrapped, i.e. hi the aqueous spaces and within the bilayer itself, respectively. It is<br>
possible that the drug-bearing liposomes may even be employed for site-specific<br>
delivery of active agents by selectively modifying the liposomal formulation.<br>
In addition to the teachings of Gouvreur et al (1977; 1988), the<br>
following information may be utilized in generating liposomal formulations.<br>
Phospholipids can form a variety of structures other than liposomes when dispersed in<br>
water, depending on the molar ratio of lipid to water. At low ratios the liposome is the<br>
preferred structure. The physical characteristics of liposomes depend on pH, ionic<br>
strength and the presence of divalent cations. Liposomes can show low permeability to<br>
ionic and polar substances, but at elevated temperatures undergo a phase transition<br>
which markedly alters their permeability. The phase transition involves a change from<br>
a closely packed, ordered structure, known as the gel state, to a loosely packed, lessordered<br>
structure, known as the fluid state. This occurs at a characteristic phasetransition<br>
temperature and results in an increase in permeability to ions, sugars and<br>
drugs.<br>
In addition to temperature, exposure to proteins can alter the<br>
permeability of liposomes. Certain soluble proteins, such as cytochrome c, bind,<br>
deform and penetrate the bilayer, thereby causing changes in permeability. Cholesterol<br>
inhibits this penetration of proteins, apparently by packing the phospholipids more<br>
tightly. It is contemplated that the most useful liposome formations for antibiotic and<br>
inhibitor delivery will contain cholesterol.<br>
The ability to trap solutes varies between different types of liposomes.<br>
For example, MLVs are moderately efficient at trapping solutes, but SUVs are<br>
extremely inefficient. SUVs offer the advantage of homogeneity and reproducibility in<br>
size distribution, however, and a compromise between size and trapping efficiency is<br>
offered by large unilamellar vesicles (LUVs). These are prepared by ether evaporation<br>
and are three to four times more efficient at solute entrapment than MLVs.<br>
In addition to liposome characteristics, an important determinant in<br>
entrapping compounds is the physicochemical properties of the compound itself. Polar<br>
compounds are trapped hi the aqueous spaces and nonpolar compounds bind to the lipid<br>
bilayer of the vesicle. Polar compounds are released through permeation or when the<br>
bilayer is broken, but nonpolar compounds remain affiliated with the bilayer unless it is<br>
disrupted by temperature or exposure to lipoproteins. Both types show maximum<br>
efflux rates at the phase transition temperature.<br>
Liposomes interact with cells via four different mechanisms:<br>
endocytosis by phagocytic cells of the reticuloendothelial system such as macrophages<br>
and neutrophils; adsorption to the cell surface, either by nonspecific weak hydrophobic<br>
or electrostatic forces, or by specific interactions with cell-surface components; fusion<br>
with the plasma cell membrane by insertion of the lipid bilayer of the liposome into the<br>
plasma membrane, with simultaneous release of liposomal contents into the cytoplasm;<br>
and by transfer of liposomal lipids to cellular or subcellular membranes, or vice versa,<br>
without any association of the liposome contents. It often is difficult to determine<br>
which mechanism is operative and more than one may operate at the same time.<br>
The fate and disposition of intravenously injected liposomes depend on<br>
their physical properties, such as size, fluidity, and surface charge. They may persist in<br>
tissues for h or days, depending on their composition, and half lives in the blood range<br>
from min to several h. Larger liposomes, such as MLVs and LUVs, are taken up<br>
rapidly by phagocytic cells of the reticuloendothelial system, but physiology of the<br>
circulatory system restrains the exit of such large species at most sites. They can exit<br>
only hi places where large openings or pores exist in the capillary endothelium, such as<br>
the sinusoids of the liver or spleen. Thus, these organs are the predominate site of<br>
uptake. On the other hand, SUVs show a broader tissue distribution but still are<br>
sequestered highly in the liver and spleen. In general, this in vivo behavior limits the<br>
potential targeting of liposomes to only those organs and tissues accessible to their large<br>
size. These include the blood, liver, spleen, bone marrow, and lymphoid organs.<br>
Targeting is generally not a limitation in terms of the present invention.<br>
However, should specific targeting be desired, methods are available for this to be<br>
accomplished. Antibodies may be used to bind to the liposome surface and to direct the<br>
antibody and its drug contents to specific antigenic receptors located on a particular<br>
cell-type surface. Carbohydrate determinants (giycoprotein or glycolipid cell-surface<br>
components that play a role in cell-cell recognition, interaction and adhesion) may also<br>
be used as recognition sites as they have potential in directing liposomes to particular<br>
cell types. Mostly, it is contemplated that intravenous injection of liposomal<br>
preparations would be used, but other routes of administration are also conceivable.<br>
Alternatively, the invention provides for pharmaceutically-acceptable<br>
nanocapsule formulations of the compositions of the present invention. Nanocapsules<br>
can generally entrap compounds in a stable and reproducible way (Henry-Michelland<br>
etal, 1987; Quintanar-Guerrero et al., 1998; Douglas et al., 1987). To avoid side<br>
effects due to intracellular polymeric overloading, such ultrafine particles (sized around<br>
0.1 urn) should be designed using polymers able to be degraded in vivo. Biodegradable<br>
polyalkyl-cyanoacrylate nanoparticles that meet these requirements are contemplated<br>
for use in the present invention. Such particles may be are easily made, as described<br>
(Couvreur etal., 1980; 1988; zur Muhlen et al., 1998; Zambaux et al 1998; Pinto-<br>
Alphandry et al., 1995 and U. S. Patent 5,145,684, specifically incorporated herein by<br>
reference in its entirety).<br>
IMMUNOGENIC COMPOSITIONS<br>
In certain preferred embodiments of the present invention, immunogenic<br>
compositions, or vaccines, are provided. The immunogenic compositions will generally<br>
comprise one or more pharmaceutical compositions, such as those discussed above, in<br>
combination with an immunostimulant. An immunostimulant may be any substance<br>
that enhances or potentiates an immune response (antibody and/or cell-mediated) to an<br>
exogenous antigen. Examples of immunostimulants include adjuvants, biodegradable<br>
microspheres (e.g., polylactic galactide) and liposomes (into which the compound is<br>
incorporated; see e.g., Fullerton, U.S. Patent No. 4,235,877). Vaccine preparation is<br>
generally described in, for example, M.F. Powell and MJ. Newman, eds., "Vaccine<br>
Design (the subunit and adjuvant approach)," Plenum Press (NY, 1995).<br>
Pharmaceutical compositions and immunogenic compositions, or vaccines, within the<br>
scope of the present invention may also contain other compounds, which may be<br>
biologically active or inactive. For example, one or more immunogenic portions of<br>
other tumor antigens may be present, either incorporated into a fusion polypeptide or as<br>
a separate compound, within the composition.<br>
Illustrative immunogenic compositions may contain DNA encoding one<br>
or more of the polypeptides as described above, such that the polypeptide is generated<br>
i/i situ. As noted above, the DNA may be present within any of a variety of delivery<br>
systems known to those of ordinary skill in the art, including nucleic acid expression<br>
systems, bacteria and viral expression systems. Numerous gene delivery techniques are<br>
well known in the art, such as those described by Holland, Crit. Rev. Therap. Drug<br>
Carrier Systems 75:143-198, 1998, and references cited therein. Appropriate nucleic<br>
acid expression systems contain the necessary DNA sequences for expression in the<br>
patient (such as a suitable promoter and terminating signal). Bacterial delivery systems<br>
involve the administration of a bacterium (such as Bacillus-Calmette-Guerrin) that<br>
expresses an immunogenic .portion of the polypeptide on its cell surface or secretes such<br>
an epitope. In a preferred embodiment, the DNA may be introduced using a viral<br>
expression system (e.g., vaccinia or other pox virus, retrovirus, or adenovirus), which<br>
may involve the use of a non-pathogenic (defective), replication competent virus.<br>
Suitable systems are disclosed, for example, in Fisher-Hoch et al., Proc. Natl. Acad.<br>
Sci. USA 86:317-321, 1989; Flexner et al, Ann. N.Y. Acad Sci. 55P:86-103, 1989;<br>
Flexner et al, Vaccine 5:17-21, 1990; U.S. Patent Nos.4,603,112, 4,769,330, and<br>
5,017,487; WO 89/01973; U.S. Patent No. 4,777,127; GB 2,200,651; EP 0,345,242;<br>
WO 91/02805; Berkner, Biotechniques 6:616-627, 1988; Rosenfeld et al., Science<br>
252:431-434, 1991; Kolls et al, Proc. Natl. Acad. Sci. USA 91:215-219, 1994;<br>
Kass-Eisler et al., Proc. Natl. Acad. Sci. USA 90:11498-11502, 1993; Guzman et al,<br>
Circulation 55:2838-2848, 1993; and Guzman et al, Cir. Res. 75:1202-1207, 1993.<br>
Techniques for incorporating DNA into such expression systems are well known to<br>
those of ordinary skill in the art. The DNA may also be "naked," as described, for<br>
example, in Ulmer et al, Science 259:1745-1749, 1993 and reviewed by Cohen,<br>
Science 259:1691-1692, 1993. The uptake of naked DNA may be increased by coating<br>
the DNA onto biodegradable beads, which are efficiently transported into the cells. It<br>
will be apparent that an immunogenic composition may comprise both a polynucleotide<br>
and a polypeptide component. Such immunogenic compositions may provide for an<br>
enhanced immune response.<br>
It will be apparent that an immunogenic composition may contain<br>
pharmaceutically acceptable salts of the polynucleotides and polypeptides provided<br>
herein. Such salts may be prepared from pharmaceutically acceptable non-toxic bases,<br>
including organic bases (e.g., salts of primary, secondary and tertiary amines and basic<br>
amino acids) and inorganic bases (e.g., sodium, potassium, lithium, ammonium,<br>
calcium and magnesium salts).<br>
While any suitable carrier known to those of ordinary skill hi the art may<br>
be employed in the immunogenic compositions of this invention, the type of carrier will<br>
vary depending on the mode of administration. Compositions of the present invention<br>
may be formulated for any appropriate manner of administration, including for<br>
example, topical, oral, nasal, intravenous, intracranial, intraperitoneal, subcutaneous or<br>
intramuscular administration. For parenteral administration, such as subcutaneous<br>
injection, the carrier preferably comprises water, saline, alcohol, a fat, a wax or a buffer.<br>
For oral administration, any of the above carriers or a solid carrier, such as mannitol,<br>
lactose, starch, magnesium stearate, sodium saccharine, talcum, cellulose, glucose,<br>
sucrose, and magnesium carbonate, may be employed. Biodegradable microspheres<br>
(e.g., polylactate polyglycolate) may also be employed as carriers for the<br>
pharmaceutical compositions of this invention. Suitable biodegradable microspheres<br>
are disclosed, for example, in U.S. Patent Nos. 4,897,268; 5,075,109; 5,928,647;<br>
5,811,128; 5,820,883; 5,853,763; 5,814,344 and 5,942,252. One may also employ a<br>
carrier comprising the particulate-protein complexes described in U.S. Patent No.<br>
5,928,647, which are capable of inducing a class I-restricted cytotoxic T lymphocyte<br>
responses in a host.<br>
Such compositions may also comprise buffers (e.g., neutral buffered<br>
saline or phosphate buffered saline), carbohydrates (e.g., glucose, mannose, sucrose or<br>
dextrans), mannitol, proteins, polypeptides or amino acids such as glycine, antioxidants,<br>
bacteriostats, chelating agents such as EDTA or glutathione, adjuvants (e.g., aluminum<br>
hydroxide), solutes that render the formulation isotonic, hypotonic or weakly hypertonic<br>
with the blood of a recipient, suspending agents, thickening agents and/or preservatives.<br>
Alternatively, compositions of the present invention may be formulated as a<br>
lyophilizate. Compounds may also be encapsulated within liposomes using well known<br>
technology.<br>
Any of a variety of immunostimulants may be employed in the<br>
immunogenic compositions of this invention. For example, an adjuvant may be<br>
included. Most adjuvants contain a substance designed to protect the antigen from<br>
rapid catabolism, such as aluminum hydroxide or mineral oil, and a stimulator of<br>
immune responses, such as lipid A, Bortadetta pertussis or Mycobacterium tuberculosis<br>
derived proteins. Suitable adjuvants are commercially available as, for example,<br>
Freund's Incomplete Adjuvant and Complete Adjuvant (Difco Laboratories, Detroit,<br>
MI); Merck Adjuvant 65 (Merck and Company, Inc., Rahway, NJ); AS-2 (SmithKline<br>
Beecham, Philadelphia, PA); aluminum salts such as aluminum hydroxide gel (alum) or<br>
aluminum phosphate; salts of calcium, iron or zinc; an insoluble suspension of acylated<br>
tyrosine; acylated sugars; cationically or anionically derivatized polysaccharides;<br>
polyphosphazenes; biodegradable microspheres; monophosphoryl lipid A and quil A.<br>
Cytokines, such as GM-CSF or interleukin-2, -7, or -12, may also be used as adjuvants.<br>
Within the immunogenic compositions provided herein, the adjuvant<br>
composition is preferably designed to induce an immune response predominantly of the<br>
Thl type. High levels of Thl-type cytokines (e.g., IFN-y, TNFa, IL-2 and IL-12) tend<br>
to favor the induction of cell mediated immune responses to an administered antigen.<br>
In contrast, high levels of Th2-type cytokines (e.g., IL-4, IL-5, IL-6 and IL-10) tend to<br>
favor the induction of humoral immune responses. Following application of an<br>
immunogenic composition as provided herein, a patient will support an immune<br>
response that includes Thl- and Th2-type responses. Within a preferred embodiment,<br>
in which a response is predominantly Thl-type, the level of Thl-type cytokines will<br>
increase to a greater extent than the level of Th2-type cytokines. The levels of these<br>
cytokines may be readily assessed using standard assays. For a review of the families<br>
of cytokines, see Mosmann and Coffinan, Ann. Rev. Immunol. 7:145-173,1989.<br>
Preferred adjuvants for use in eliciting a predominantly Thl-type<br>
response include, for example, a combination of monophosphoryl lipid A, preferably 3-<br>
de-O-acylated monophosphoryl lipid A (3D-MPL), together with an aluminum salt.<br>
MPL adjuvants are available from Corixa Corporation (Seattle, WA; see US Patent<br>
Nos. 4,436,727; 4,877,611; 4,866,034 and 4,912,094). CpG-containing<br>
oligonucleotides (in which the CpG dinucleotide is unmethylated) also induce a<br>
predominantly Thl response. Such oligonucleotides are well known and are described,<br>
for example, in WO 96/02555, WO 99/33488 and U.S. Patent Nos. 6,008,200 and<br>
5,856,462. Immunostimulatory DNA sequences are also described, for example, by<br>
Sato et al, Science 275:352, 1996. Another preferred adjuvant is a saponin, preferably<br>
QS21 (Aquila Biopharmaceuticals Inc., Framingham, MA), which may be used alone or<br>
in combination with other adjuvants. For example, an enhanced system involves the<br>
combination of a monophosphoryl lipid A and saponin derivative, such as the<br>
combination of QS21 and 3D-MPL as described in WO 94/00153, or a less reactogenic<br>
composition where the QS21 is quenched with cholesterol, as described in WO<br>
96/33739. Other preferred formulations comprise an oil-in-water emulsion and<br>
tocopherol. .A particularly potent adjuvant formulation involving QS21, 3D-MPL and<br>
tocopherol in an oil-in-water emulsion is described in WO 95/17210.<br>
Other preferred adjuvants include Montanide ISA 720 (Seppic, France),<br>
SAF (Chiron, California, United States), ISCOMS (CSL), MF-59 (Chiron), the SBAS<br>
series of adjuvants (e.g., SBAS-2 or SBAS-4, available from SmithKline Beecham,<br>
Rixensart, Belgium), Detox (Corixa, Hamilton, MT), RC-529 (Corixa, Hamilton, MT)<br>
and other aminoalkyl glucosaminide 4-phosphates (AGPs), such as those described in<br>
pending U.S. Patent Application Serial Nos. 08/853,826 and 09/074,720, the disclosures<br>
of which are incorporated herein by reference in their entireties.<br>
Any immunogenic composition provided herein may be prepared using<br>
well known methods that result in a combination of antigen, immune response enhancer<br>
and a suitable carrier or excipient. The compositions described herein may be<br>
administered as part of a sustained release formulation (i.e., a formulation such as a<br>
capsule, sponge or gel (composed of polysaccharides, for example) that effects a slow<br>
release of compound following administration). Such formulations may generally be<br>
prepared using well known technology (see, e.g., Coombes et al, Vaccine 14:1429-<br>
1438, 1996) and administered by, for example, oral, rectal or subcutaneous<br>
implantation, or by implantation at the desired target site. Sustained-release<br>
formulations may contain a polypeptide, polynucleotide or antibody dispersed in a<br>
carrier matrix and/or contained within a reservoir surrounded by a rate controlling<br>
membrane.<br>
Carriers for use within such formulations are biocompatible, and may<br>
also be biodegradable; preferably the formulation provides a relatively constant level of<br>
active component release. Such carriers include microparticles of poly(lactide-coglycolide),<br>
polyacrylate, latex, starch, cellulose, dextran and the like. Other delayedrelease<br>
carriers include supramolecular biovectors, which comprise a non-liquid<br>
hydrophilic core (e.g., a cross-linked polysaccharide or oligosaccharide) and,<br>
optionally, an external layer comprising an amphiphilic compound, such as a<br>
phospholipid (see e.g., U.S. Patent No. 5,151,254 and PCT applications WO 94/20078,<br>
WO/94/23701 and WO 96/06638). The amount of active compound contained within a<br>
sustained release formulation depends upon the site of implantation, the rate and<br>
expected duration of release and the nature of the condition to be treated or prevented.<br>
Any of a variety of delivery vehicles may be employed within<br>
pharmaceutical compositions and imraunogenic compositions to facilitate production of<br>
an antigen-specific immune response that targets tumor cells. Delivery vehicles include<br>
antigen presenting cells (APCs), such as dendritic cells, macrophages, B cells,<br>
monocytes and other cells that may be engineered to be efficient APCs. Such cells<br>
may, but need not, be genetically modified to increase the capacity for presenting the<br>
antigen, to improve activation and/or maintenance of the T cell response, to have antitumor<br>
effects per se and/or to be immunologically compatible with the receiver (i.e.,<br>
matched HLA haplotype). APCs may generally be isolated from any of a variety of<br>
biological fluids and organs, including tumor and peritumoral tissues, and may be<br>
autologous, allogeneic, syngeneic or xenogeneic cells.<br>
Certain preferred embodiments of the present invention use dendritic<br>
cells or progenitors thereof as antigen-presenting cells. Dendritic cells are highly potent<br>
APCs (Banchereau and Steinman, Nature 392:245-251, 1998) and have been shown to<br>
be effective as a physiological adjuvant for eliciting prophylactic or therapeutic<br>
antitumor immunity (see Timmerman and Levy, Ann. Rev. Med. 50:507-529, 1999). In<br>
general, dendritic cells may be identified based on their typical shape (stellate in situ,<br>
with marked cytoplasmic processes (dendrites) visible in vitro), their ability to take up,<br>
process and present antigens with high efficiency and their ability to activate nai've T<br>
cell responses. Dendritic cells may, of course, be engineered to express specific cellsurface<br>
receptors or ligands that are not commonly found on dendritic cells in vivo or ex<br>
vivo, and such modified dendritic cells are contemplated by the present invention. As<br>
an alternative to dendritic cells, secreted vesicles antigen-loaded dendritic cells (called<br>
exosomes) may be used within a vaccine, or immunogenic composition (see Zitvogel et<br>
al., Nature Med. 4:594-600,1998).<br>
Dendritic cells and progenitors may be obtained from peripheral blood,<br>
bone marrow, tumor-infiltrating cells, peritumoral tissues-infiltrating cells, lymph<br>
nodes, spleen, skin, umbilical cord blood or any other suitable tissue or fluid. For<br>
example, dendritic cells may be differentiated ex vivo by adding a combination of<br>
cytokines such as GM-CSF, IL-4, IL-13 and/or TNFa to cultures of monocytes<br>
harvested from peripheral blood. Alternatively, CD34 positive cells harvested from<br>
peripheral blood, umbilical cord blood or bone marrow may be differentiated into<br>
dendritic cells by adding to the culture medium combinations of GM-CSF, IL-3, TNFa,<br>
CD40 ligand, LPS, flt3 ligand and/or other compound(s) that induce differentiation,<br>
maturation and proliferation of dendritic cells.<br>
Dendritic cells are conveniently categorized as "immature" and "mature"<br>
cells, which allows a simple way to discriminate between two well characterized<br>
phenotypes. However, this nomenclature should not be construed to exclude all<br>
possible intermediate stages of differentiation. Immature dendritic cells are<br>
characterized as APC with a high capacity for antigen uptake and processing, which<br>
correlates with the high expression of Fey receptor and mannose receptor. The mature<br>
phenotype is typically characterized by a lower expression of these markers, but a high<br>
expression of cell surface molecules responsible for T cell activation such as class I and<br>
class IIMHC, adhesion molecules (e.g., CD54 and GDI 1) and costimulatory molecules<br>
(e.g., CD40, CD80, CD86 and 4-1BB).<br>
APCs may generally be transfected with a polynucleotide encoding a<br>
lung tumor protein (or portion or other variant thereof) such that -the lung tumor<br>
polypeptide, or an immunogenic portion thereof, is expressed on the cell surface. Such<br>
transfection may take place ex vivo, and a composition comprising such transfected<br>
cells may then be used for therapeutic purposes, as described herein. Alternatively, a<br>
gene delivery vehicle that targets a dendritic or other antigen presenting cell may be<br>
administered to a patient, resulting in transfection that occurs in vivo. In vivo and ex<br>
vivo transfection of dendritic cells, for example, may generally be performed using any<br>
methods known in the art, such as those described in WO 97/24447, or the gene gun<br>
approach described by Mahvi et al., Immunology and cell Biology 75:456-460, 1997.<br>
Antigen loading of dendritic cells may be achieved by incubating dendritic cells or<br>
progenitor cells with the lung tumor polypeptide, DNA (naked or within a plasmid<br>
vector) or RNA; or with antigen-expressing recombinant bacterium or viruses (e.g.,<br>
vaccinia, fowlpox, adenovirus or lentivirus vectors). Prior to loading, the polypeptide<br>
may be covalently conjugated to an irnmunological partner that provides T cell help<br>
(e.g., a carrier molecule). Alternatively, a dendritic cell may be pulsed with a nonconjugated<br>
immunological partner, separately or in the presence of the polypeptide.<br>
Immunogenic compositions and pharmaceutical compositions may be<br>
presented in unit-dose or multi-dose containers, such as sealed ampoules or vials. Such<br>
containers are preferably hermetically sealed to preserve sterility of the formulation<br>
until use. In general, formulations may be stored as suspensions, solutions or emulsions<br>
hi oily or aqueous vehicles. Alternatively, an immunogenic or pharmaceutical<br>
composition may be stored hi a freeze-dried condition requiring only the addition of a<br>
sterile liquid carrier immediately prior to use.<br>
CANCER THERAPY<br>
In further aspects of the present invention, the compositions described<br>
herein may be used for immunotherapy of cancer, such as lung cancer. Within such<br>
methods, compositions are typically administered to a patient. As used herein, a<br>
"patient" refers to any warm-blooded animal, preferably a human. A patient may or<br>
may not be afflicted with cancer. Accordingly, the above pharmaceutical compositions<br>
and immunogenic compositions may be used to prevent the development of a cancer or<br>
to treat a patient afflicted with a cancer. A cancer may be diagnosed using criteria<br>
generally accepted in the art, including the presence of a malignant tumor.<br>
Pharmaceutical compositions and immunogenic composition s may be administered<br>
either prior to or following surgical removal of primary tumors and/or treatment such as<br>
administration of radiotherapy or conventional chemotherapeutic drugs. Administration<br>
may be by any suitable method, including administration by intravenous,<br>
intraperitoneal, intramuscular, subcutaneous, intranasal, intradermal, anal, vaginal,<br>
topical and oral routes.<br>
Within certain embodiments, immunotherapy may be active<br>
immunotherapy, hi which treatment relies on the in vivo stimulation of the endogenous<br>
host immune system to react against tumors with the administration of immune<br>
response-modifying agents (such as polypeptides and polynucleotides as provided<br>
herein).<br>
Within other embodiments, immunotherapy may be passive<br>
immunotherapy, in which treatment involves the delivery of agents with established<br>
tumor-immune reactivity (such as effector cells or antibodies) that can directly or<br>
indirectly mediate antitumor effects and does not necessarily depend on an intact host<br>
immune system. Examples of effector cells include T cells as discussed above, T<br>
lymphocytes (such as CD8+ cytotoxic T lymphocytes and CD4+ T-helper tumorinfiltrating<br>
lymphocytes), killer cells (such as Natural Killer cells and lymphokineactivated<br>
killer cells), B cells and antigen-presenting cells (such as dendritic cells and<br>
macrophages) expressing a polypeptide provided herein. T cell receptors and antibody<br>
receptors specific for the polypeptides recited herein may be cloned, expressed and<br>
transferred into other vectors or effector cells for adoptive immunotherapy. The<br>
polypeptides provided herein may also be used to generate antibodies or anti-idiotypic<br>
antibodies (as described above and in U.S. Patent No. 4,918,164) for passive<br>
immunotherapy.<br>
Effector cells may generally be obtained in sufficient quantities for<br>
adoptive immunotherapy by growth in vitro, as described herein. Culture conditions for<br>
expanding single antigen-specific effector cells to several billion in number with<br>
retention of antigen recognition in vivo are well known in the art. Such in vitro culture<br>
conditions typically use intermittent stimulation with antigen, often in ihe presence of<br>
cytokines (such as IL-2) and non-dividing feeder cells. As noted above,<br>
immunoreactive polypeptides as provided herein may be used to rapidly expand<br>
antigen-specific T cell cultures in order to generate a sufficient number of cells for<br>
immunotherapy. In particular, antigen-presenting cells, such as dendritic, macrophage,<br>
monocyte, fibroblast and/or B cells, may be pulsed with immunoreactive polypeptides<br>
or transfected with one or more polynucleotides using standard techniques well known<br>
in the art. For example, antigen-presenting . cells can be transfected with a<br>
polynucleotide having a promoter appropriate for increasing expression in a<br>
recombinant virus or other expression system. Cultured effector cells for use in therapy<br>
must be able to grow and distribute widely, and to survive long term in vivo. Studies<br>
have shown that cultured effector cells can be induced to grow in vivo and to survive<br>
long term in substantial numbers by repeated stimulation with antigen supplemented<br>
with IL-2 (see, for example, Cheever etal, ImmunologicalReviews 157:177, 1997).<br>
Alternatively, a vector expressing a polypeptide recited herein may be<br>
introduced into antigen presenting cells taken from a patient and clonally propagated ex<br>
vivo for transplant back into the same patient. Transfected cells may be reintroduced<br>
into the patient using any means known in the art, preferably in sterile form by<br>
intravenous, intracavitary, intraperitoneal or intratumor administration.<br>
Routes and frequency of administration of the therapeutic compositions<br>
described herein, as well as dosage, will vary from individual to individual, and may be<br>
readily established using standard techniques. In general, the pharmaceutical<br>
compositions and immunogenic compositions may be administered by injection (e.g.,<br>
intracutaneous, intramuscular, intravenous or subcutaneous), intranasally (e.g., by<br>
aspiration) or orally. Preferably, between 1 and 10 doses may be administered over a<br>
52 week period. Preferably, 6 doses are administered, at intervals of 1 month, and<br>
booster vaccinations may be given periodically thereafter. Alternate protocols may be<br>
appropriate for individual patients. A suitable dose is an amount of a compound that,<br>
when administered as described above, is capable of promoting an anti-tumor immune<br>
response, and is at least 10-50% above the basal (i.e., untreated) level. Such response<br>
can be monitored by measuring the anti-tumor antibodies in a patient or by vaccinedependent<br>
generation of cytolytic effector cells capable of killing the. patient's tumor<br>
cells in vitro. Such vaccines, or immunogenic compositions, should also be capable of<br>
causing an immune response that leads to an improved clinical outcome (e.g., more<br>
frequent remissions, complete or partial or longer disease-free survival) in vaccinated<br>
patients as compared to non-vaccinated patients. In general, for compositions<br>
comprising one or more polypeptides, the amount of each polypeptide present in a dose<br>
ranges from about 25 u,g to 5 mg per kg of host. Suitable dose sizes will vary with the<br>
size of the patient, but will typically range from about 0.1 mL to about 5 mL.<br>
In general, an appropriate dosage and treatment regimen provides the<br>
active compound(s) in an amount sufficient to provide therapeutic and/or prophylactic<br>
benefit. Such a response can be monitored by establishing an improved clinical<br>
outcome (e.g., more frequent remissions, complete or partial, or longer disease-free<br>
survival) in treated patients as compared to non-treated patients. Increases in<br>
preexisting immune responses to a lung tumor protein generally correlate with an<br>
improved clinical outcome. Such immune responses may generally be evaluated using<br>
standard proliferation, cytotoxicity or cytokine assays, which may be performed using<br>
samples obtained from a patient before and after treatment.<br>
CANCER DETECTION AND DIAGNOSIS<br>
In general, a cancer may be detected in a patient based on the presence<br>
of one or more lung tumor proteins and/or polynucleotides encoding such proteins in a<br>
biological sample (for example, blood, sera, sputum urine and/or tumor biopsies)<br>
obtained from the patient. In other words, such proteins may be used as markers to<br>
indicate the presence or absence of a cancer such as lung cancer. In addition, such<br>
proteins may be useful for the detection of other cancers. The binding agents provided<br>
herein generally permit detection of the level of antigen that binds to the agent in the<br>
biological sample. Polynucleotide primers and probes may be used to detect the level<br>
of mRNA encoding a tumor protein, which is also indicative of the presence or absence<br>
of a cancer. In general, a lung tumor sequence should be present at a level that is at<br>
least three fold higher in tumor tissue than hi normal tissue<br>
There are a variety of assay formats known to those of ordinary skill in<br>
the art for using a binding agent to detect polypeptide markers in a sample. See, e.g.,<br>
Harlow and Lane, Antibodies: A Laboratory Manual, Cold Spring Harbor Laboratory,<br>
1988. In general, the presence or absence of a cancer in a patient may be determined by<br>
(a) contacting a biological sample obtained from a patient with a binding agent; (b)<br>
detecting in the sample a level of polypeptide that binds to the binding agent; and (c)<br>
comparing the level of polypeptide with a predetermined cut-off value.<br>
In a preferred embodiment, the assay involves the use of binding agent<br>
immobilized on a solid support to bind to and remove the polypeptide from the<br>
remainder of the sample. The bound polypeptide may then be detected using a<br>
detection reagent that contains a reporter group and specifically binds to the binding<br>
agent/polypeptide complex. Such detection reagents may comprise, for example, a<br>
binding agent that specifically binds to the polypeptide or an antibody or other agent<br>
that specifically binds to the binding agent, such as an anti-immunoglobulin, protein G,<br>
protein A or a lectin. Alternatively, a competitive assay may be utilized, in which a<br>
polypeptide is labeled with a reporter group and allowed to bind to the immobilized<br>
binding agent after incubation of the binding agent with the sample. The extent to<br>
which components of the sample inhibit the binding of the labeled polypeptide to the<br>
binding agent is indicative of the reactivity of the sample with the immobilized binding<br>
agent. Suitable polypeptides for use within such assays include full length lung tumor<br>
proteins and portions thereof to which the binding agent binds, as described above.<br>
The solid support may be any material known to those of ordinary skill<br>
in the art to which the tumor protein may be attached. For example, the solid support<br>
may be a test well in a microtiter plate or a nitrocellulose or other suitable membrane.<br>
Alternatively, the support may be a bead or disc, such as glass, fiberglass, latex or a<br>
plastic material such as polystyrene or polyvinylchloride. The support may also be a<br>
magnetic particle or a fiber optic sensor, such as those disclosed, for example, in U.S.<br>
Patent No. 5,359,681. The binding agent may be immobilized on the solid support<br>
using a variety of techniques known to those of skill in the art, which are amply<br>
described in the patent and scientific literature. In the context of the present invention,<br>
the term "immobilization" refers to both noncovalent association, such as adsorption,<br>
and covalent attachment (which may be a direct linkage between the agent and<br>
functional groups on the support or may be a linkage by way of a cross-linking agent).<br>
Immobilization by adsorption to a well in a microtiter plate or to a membrane is<br>
preferred. In such cases, adsorption may be achieved by contacting the binding agent,<br>
in a suitable buffer, with the solid support for a suitable amount of time. The contact<br>
time varies with temperature, but is typically between about 1 hour and about 1 day. In<br>
general, contacting a well of a plastic microtiter plate (such as polystyrene or<br>
polyvinylchloride) with an amount of binding agent ranging from about 10 ng to about<br>
10 u£, and preferably about 100 ng to about 1 ug, is sufficient to immobilize an<br>
adequate amount of binding agent.<br>
Covalent attachment of binding agent to a solid support may generally<br>
be achieved by first reacting the support with a bifunctional reagent that will react with<br>
both the support and a functional group, such as a hydroxyl or amino group, on the<br>
binding agent For example, the binding agent may be covalently attached to supports<br>
having an appropriate polymer coating using benzoquinone or by condensation of an<br>
aldehyde group on the support with an amine and an active hydrogen on the binding<br>
partner (see, e.g., Pierce Immunotechnology Catalog and Handbook, 1991, at<br>
A12-A13).<br>
In certain embodiments, the assay is a two-antibody sandwich assay.<br>
This assay may be performed by first contacting an antibody that has been immobilized<br>
on a solid support, commonly the well of a microtiter plate, with the sample, such that<br>
polypeptides within the sample are allowed to bind to the immobilized antibody.<br>
Unbound sample is then removed from the immobilized polypeptide-antibody<br>
complexes and a detection reagent (preferably a second antibody capable of binding to<br>
a different site on the polypeptide) containing a reporter group is added. The amount of<br>
detection reagent that remains bound to the solid support is then determined using a<br>
method appropriate for the specific reporter group.<br>
More specifically, once the antibody is immobilized on the support as<br>
described above, the remaining protein binding sites on the support are typically<br>
blocked. Any suitable blocking agent known to those of ordinary skill in the art, such<br>
as bovine serum albumin or Tween 20™ (Sigma Chemical Co., St. Louis, MO). The<br>
immobilized antibody is then incubated with the sample, and polypeptide is allowed to<br>
bind to the antibody. The sample may be diluted with a suitable diluent, such as<br>
phosphate-buffered saline (PBS) prior to incubation. In general, an appropriate contact<br>
time (i.e., incubation time) is a period of time that is sufficient to detect the presence of<br>
polypeptide within a sample obtained from an individual with lung cancer. Preferably,<br>
the contact time is sufficient to achieve a level of binding that is at least about 95% of<br>
that achieved at equilibrium between bound and unbound polypeptide. Those of<br>
ordinary skill in the art will recognize that the time necessary to achieve equilibrium<br>
may be readily determined by assaying the level of binding that occurs over a period of<br>
time. At room temperature, an incubation time of about 30 minutes is generally<br>
sufficient.<br>
Unbound sample may then be removed by washing the solid support<br>
with an appropriate buffer, such as PBS containing 0.1% Tween 20™. The second<br>
antibody, which contains a reporter group, may then be added to the solid support.<br>
Preferred reporter groups include those groups recited above.<br>
The detection reagent is then incubated with the immobilized antibodypolypeptide<br>
complex for an amount of tune sufficient to detect the bound polypeptide.<br>
An appropriate amount of time may generally be determined by assaying the level of<br>
binding that occurs over a period of time. Unbound detection reagent is then removed<br>
and bound detection reagent is detected using the reporter group. The method<br>
employed for detecting the reporter group depends upon the nature of the reporter<br>
group. For radioactive groups, scintillation counting or autoradiographic methods are<br>
generally appropriate. Spectroscopic methods may be used to detect dyes, luminescent<br>
groups and fluorescent groups. Biotin may be detected using avidin, coupled to a<br>
different reporter group (commonly a radioactive or fluorescent group or an enzyme).<br>
Enzyme reporter groups may generally be detected by the addition of substrate<br>
(generally for a specific period of time), followed by Spectroscopic or other analysis of<br>
the reaction products.<br>
To determine the presence or absence of a cancer, such as lung cancer,<br>
the signal detected from the reporter group that remains bound to the solid support is<br>
generally compared to a signal that corresponds to a predetermined cut-off value. In<br>
one preferred embodiment, the cut-off value for the detection of a cancer is the average<br>
mean signal obtained when the immobilized antibody is incubated with samples from<br>
patients without the cancer. In general, a sample generating a signal that is three<br>
standard deviations above the predetermined cut-off value is considered positive for the<br>
cancer. In an alternate preferred embodiment, the cut-off value is determined using a<br>
Receiver Operator Curve, according to the method of Sackett et al, Clinical<br>
Epidemiology: A Basic Science for Clinical Medicine, Little Brown and Co., 1985,<br>
p. 106-7. Briefly, in this embodiment, the cut-off value may be determined from a plot<br>
of pairs of true positive rates (i.e., sensitivity) and false positive rates (100%-<br>
specificity) that correspond to each possible cut-off value for the diagnostic test result.<br>
The cut-off value on the plot that is the closest to the upper left-hand corner (i.e., the<br>
value that encloses the largest area) is the most accurate cut-off value, and a sample<br>
generating a signal that is higher than the cut-off value determined by this method may<br>
be considered positive. Alternatively, the cut-off value may be shifted to the left along<br>
the plot, to minimize the false positive rate, or to the right, to minimize the false<br>
negative rate. In general, a sample generating a signal that is higher than the cut-off<br>
value determined by this method is considered positive for a cancer.<br>
In a related embodiment, the assay is performed in a flow-through or<br>
strip test format, wherein the binding agent is immobilized on a membrane, such as<br>
nitrocellulose. In the flow-through test, polypeptides within the sample bind to the<br>
immobilized binding agent as the sample passes through the membrane. A second,<br>
labeled binding agent then binds to the binding agent-polypeptide complex as a solution<br>
containing the second binding agent flows through the membrane. The detection of<br>
bound second binding agent may then be performed as described above. In the strip test<br>
format, one end of the membrane to which binding agent is bound is immersed in a<br>
solution containing the sample. The sample migrates along the membrane through a<br>
region containing second binding agent and to the area of immobilized binding agent.<br>
Concentration of second binding agent at the area of immobilized antibody indicates the<br>
presence of a cancer. Typically, the concentration of second binding agent at that site<br>
generates a pattern, such as a line, that can be read visually. The absence of such a<br>
pattern indicates a negative result. In general, the amount of binding agent immobilized<br>
on the membrane is selected to generate 'a visually discernible pattern when the<br>
biological sample contains a level of polypeptide that would be sufficient to generate a<br>
positive signal in the two-antibody sandwich assay, in the format discussed above.<br>
Preferred binding agents for use in such assays are antibodies and antigen-binding<br>
fragments thereof. Preferably, the amount of antibody immobilized on the membrane<br>
ranges from about 25 ng to about l|ig, and more preferably from about 50 ng to about<br>
500 ng. Such tests can typically be performed with a very small amount of biological<br>
sample.<br>
Of course, numerous other assay protocols exist that are suitable for use<br>
with the tumor proteins or binding agents of the present invention. The above<br>
descriptions are intended to be exemplary only. For example, it will be apparent to<br>
those of ordinary skill in the art that the above protocols may be readily modified to use<br>
lung tumor polypeptides to detect antibodies that bind to such polypeptides in a<br>
biological sample. The detection of such lung tumor protein specific antibodies may<br>
correlate with the presence of a cancer.<br>
A cancer may also, or alternatively, be detected based on the presence of<br>
T cells that specifically react with a lung tumor protein in a biological sample. Within<br>
certain methods, a biological sample comprising CD4+ and/or GD8+ T cells isolated<br>
from a patient is incubated with a lung tumor polypeptide, a polynucleotide encoding<br>
such a polypeptide and/or an APC that expresses at least an immunogenic portion of<br>
such a polypeptide, and the presence or absence of specific activation of the T cells is<br>
detected. Suitable biological samples include, but are not limited to, isolated T cells.<br>
For example, T cells may be isolated from a patient by routine techniques (such as by<br>
Ficoll/Hypaque density gradient centrifugation of peripheral blood lymphocytes). T<br>
cells may be incubated in vitro for 2-9 days (typically 4 days) at 37°C with polypeptide<br>
(e.g., 5-25 ng/ml). It may be desirable to incubate another aliquot of a T cell sample<br>
in the absence of lung tumor polypeptide to serve as a control. For CD4+ T cells,<br>
activation is preferably detected by evaluating proliferation of the T cells. For CD8+ T<br>
cells, activation is preferably detected by evaluating cytolytic activity. A level of<br>
proliferation that is at least two fold greater and/or a level of cytolytic activity that is at<br>
least 20% greater than in disease-free patients indicates the presence of a cancer in the<br>
patient<br>
As noted above, a cancer may also, or alternatively, be detected based on<br>
the level of mRNA encoding a lung tumor protein in a biological sample. For example,<br>
at least two oligonucleotide primers may be employed in a polymerase chain reaction<br>
(PCR) based assay to amplify a portion of a lung tumor cDNA derived from a<br>
biological sample, wherein at least one of the oligonucleotide primers is specific for<br>
(i.e., hybridizes to) a polynucleotide encoding the lung tumor protein. The amplified<br>
cDNA is then separated and detected using techniques well known in the art, such as<br>
gel electrophoresis. Similarly, oligonucleotide probes that specifically hybridize to a<br>
polynucleotide encoding a lung tumor protein may be used hi a hybridization assay to<br>
detect the presence of polynucleotide encoding the tumor protein in a biological sample.<br>
To permit hybridization under assay conditions, oligonucleotide primers<br>
and probes should comprise an oligonucleotide sequence that has at least about 60%,<br>
preferably at least about 75% and more preferably at least about 90%, identity to a<br>
portion of a polynucleotide encoding a lung tumor protein that is at least 10 nucleotides,<br>
and preferably at least 20 nucleotides, in length. Preferably, oligonucleotide primers<br>
and/or probes hybridize to a polynucleotide encoding a polypeptide described herein<br>
under moderately stringent conditions, as defined above. Oligonucleotide primers<br>
and/or probes which may be usefully employed in the diagnostic methods described<br>
herein preferably are at least 10-40 nucleotides in length. In a preferred embodiment,<br>
the oligonucleotide primers comprise at least 10 contiguous nucleotides, more<br>
preferably at least 15 contiguous nucleotides, of a DNA molecule having a sequence<br>
recited in SEQ ID NO: 1,11-13,15,20,23-27,29,30, 33, 34,39,41,4346, 51, 52,57,<br>
58, 60, 62, 65-67, 69-71, 74, 76, 79, 80, 84, 86, 89-92, 95, 97, 98, 101,110, 111, 113-<br>
119, 121-128,130-134, 136, 138,139,141, 143, 146-151, 153,154, 157-160, 162-164,<br>
167-178,180,181,183,186-190,192,193,195-220,224,226-231,234,236, 237,240,<br>
241, 244-246, 248, 254, 255, 261, 262, 266, 270, 275, 280, 282, 283, 288, 289, 290,<br>
292, 295, 301, 303, 304, 309, 311, 341-782, 784, 785, 790, 792, 794, 796, 800-804,<br>
807, 808 or 810-826. Techniques for both PCR based assays and hybridization assays<br>
are well known in the art (see, for example, Mullis et al, Cold Spring Harbor Symp.<br>
Quant. Biol.,51:263,1987;Erliched.,PCR Technology, Stockton Press* NY, 1989).<br>
One preferred assay employs RT-PCR, in which PCR is applied in<br>
conjunction with reverse transcription. Typically, RNA is extracted from a biological<br>
sample, such as biopsy tissue, and is reverse transcribed to produce cDNA molecules.<br>
PCR amplification using at least one specific primer generates a cDNA molecule,<br>
which may be separated and visualized using, for example, gel electrophoresis.<br>
Amplification may be performed on biological samples taken from a test patient and<br>
from an individual who is not afflicted with a cancer. The amplification reaction may<br>
be performed on several dilutions of cDNA spanning two orders of magnitude. A twofold<br>
or greater increase in expression in several dilutions of the test patient sample as<br>
compared to the same dilutions of the non-cancerous sample is typically considered<br>
positive.<br>
In another embodiment, the compositions described herein may be used<br>
as markers for the progression of cancer. In this embodiment, assays as described<br>
above for the diagnosis of a cancer may be performed over time, and the change in the<br>
level of reactive polypeptide(s) or polynucleotide(s) evaluated. For example, the assays<br>
may be performed every 24-72 hours for a period of 6 months to 1 year, and thereafter<br>
performed as needed. In general, a cancer is progressing in those patients in whom the'<br>
level of polypeptide or polynucleotide detected increases over tune. In contrast, the<br>
cancer is not progressing when the level of reactive polypeptide or polynucleotide either<br>
remains constant or decreases with time.<br>
Certain in vivo diagnostic assays may be performed directly on a tumor.<br>
One such assay involves contacting tumor cells with a binding agent The bound<br>
binding agent may then be detected directly or indirectly via a reporter group. Such<br>
binding agents may also be used in histological applications. Alternatively,<br>
polynucleotide probes may be used within such applications.<br>
As noted above, to improve sensitivity, multiple lung tumor protein<br>
markers may be assayed within a given sample. It will be apparent that binding agents<br>
specific for different proteins provided herein may be combined within a single assay.<br>
Further, multiple primers or probes may be used concurrently. The selection of tumor<br>
protein markers may be based on routine experiments to determine combinations that<br>
results in optimal sensitivity. In addition, or alternatively, assays for-tumor proteins<br>
provided herein may be combined with assays for other known tumor antigens.<br>
DIAGNOSTIC KITS<br>
The present invention further provides kits for use within any of the<br>
above diagnostic methods. Such kits typically comprise two or more components<br>
necessary for performing a diagnostic assay. Components may be compounds,<br>
reagents, containers and/or equipment. For example, one container within a kit may<br>
contain a monoclonal antibody or fragment thereof that specifically binds to a lung<br>
tumor protein. Such antibodies or fragments may be provided attached to a support<br>
material, as described above. One or more additional containers may enclose elements,<br>
such as reagents or buffers, to be used in the assay. Such kits may also, or alternatively,<br>
contain a detection reagent as described above that contains a reporter group suitable<br>
for direct or indirect detection of antibody binding.<br>
Alternatively, a kit may be designed to detect the level of mRNA<br>
encoding a lung tumor protein in a biological sample. Such kits generally comprise at<br>
least one oligonucleotide probe or primer, as described above, that hybridizes to a<br>
polynucleotide encoding a lung tumor protein. Such an oligonucleotide may be used,<br>
for example, within a PCR or hybridization assay. Additional components that may be<br>
present within such kits include a second oligonucleotide and/or a diagnostic reagent or<br>
container to facilitate the detection of a polynucleotide encoding a lung tumor protein.<br>
The following Examples are offered by way of illustration and not by<br>
way of limitation.<br>
EXAMPLE 1<br>
IDENTIFICATION AND CHARACTERIZATION OF LUNG<br>
TUMOR PROTEIN cDNAS<br>
This Example illustrates the identification of cDNA molecules encoding<br>
lung tumor proteins.<br>
A. Isolation of cDNA Sequences from Lung Adenocarcinoma Libraries<br>
using Conventional cDNA Library Subtraction<br>
A human lung adenocarcinoma cDNA expression library was<br>
constructed from poly A+ RNA from patient tissues (# 40031486) using a Superscript<br>
Plasmid System for cDNA Synthesis and Plasmid Cloning kit (BRL Life Technologies,<br>
Gaithersburg, MD) following the manufacturer's protocol. Specifically, lung carcinoma<br>
tissues were homogenized with polytron (Kinematica, Switzerland) and total RNA was<br>
extracted using Trizol reagent (BRL Life Technologies) as directed by the<br>
manufacturer. The poly A+ RNA was then purified using an oligo dT cellulose column<br>
as described in Sambrook etal., Molecular Cloning: A Laboratory Manual, Cold<br>
Spring Harbor Laboratories, Cold Spring Harbor, NY, 1989. First-strand cDNA was<br>
synthesized using the NotI/Oligo-dT18 primer. Double-stranded cDNA was<br>
synthesized, ligated with BstXI/EcoRI adaptors (Invitrogen, San Diego, CA) and<br>
digested with Notl. Following size fractionation with cDNA size fractionation columns<br>
(BRL Life Technologies), the cDNA was ligated into the BstXI/NotI site of pcDNA3A<br>
(Invitrogen) and transformed into ElectroMax E. coli DH10B cells (BRL Life<br>
Technologies) by elect-operation. A total of 3 x 106 independent colonies were<br>
generated.<br>
Using the same procedure, a normal human cDNA expression library<br>
was prepared from a panel of normal tissue specimens, including lung, liver, pancreas,<br>
skin, kidney, brain and resting PBMC.<br>
cDNA library subtraction was performed using the above lung<br>
adenocarcinoma and normal tissue cDNA libraries, as described by Kara et al. (Blood,<br>
54:189-199, 1994) with some modifications. Specifically, a lung adenocarcinoma-<br>
specific subtracted cDNA library was generated as follows. The normal tissue cDNA<br>
library (80 ug) was digested with BamHI and Xhol, followed by a filling-in reaction<br>
with DNA polymerase Klenow fragment. After phenol-chloroform extraction and<br>
ethanol precipitation, the DNA was dissolved in 133 ul of HiO, heat-denatured and<br>
mixed with 133 ul (133 ug) of Photoprobe biotin (Vector Laboratories, Burlingame,<br>
CA). As recommended by the manufacturer, the resulting mixture was irradiated with a<br>
270 W sunlamp on ice for 20 minutes. Additional Photoprobe biotin (67 ul) was added<br>
and the biotinylation reaction was repeated. After extraction with butanol five times,<br>
the DNA was ethanol-precipitated and dissolved in 23 ul rfcO. The resulting DNA,<br>
plus other highly redundant cDNA clones that were frequently recovered in previous<br>
lung subtractions formed the driver DNA.<br>
To form the tracer DNA, 10 ug lung adenocarcinoma cDNA library was<br>
digested with NotI and Spel, phenol chloroform extracted and passed through Chroma<br>
spin-400 columns (Clontech, Palo Alto, CA). Typically, 5 jig of cDNA was recovered<br>
after the sizing column. Following ethanol precipitation, the tracer DNA was dissolved<br>
in 5 ul H2O. Tracer DNA was mixed with 15 ul driver DNA and 20 ul of 2 x<br>
hybridization buffer (1.5 M NaCl/10 mM EDTA/50 mM HEPES pH 7.5/0.2% sodium<br>
dodecyl sulfate), overlaid with mineral oil, and heat-denatured completely. The sample<br>
was immediately transferred into a 68 °C water bath and incubated for 20 hours (long<br>
hybridization [LH]). The reaction mixture was then subjected to a streptavidin<br>
treatment followed by phenol/chloroform extraction. This process was repeated three<br>
more times. Subtracted DNA was precipitated, dissolved in 12 ul H2O, mixed with 8 ul<br>
driver DNA and 20 ul of 2 x hybridization buffer, and subjected to a hybridization at 68 "<br>
°C for 2 hours (short hybridization [SH]). After removal of biotinylated doublestranded<br>
DNA, subtracted cDNA was ligated into Notl/Spel site of chloramphenicol<br>
resistant pBCSK* (Stratagene, La Jolla, CA) and transformed into ElectroMax E. coli<br>
DH10B cells by electroporation to generate a lung adenocarcinoma specific subtracted<br>
cDNA library, referred to as LAT-S1 Similarly, LAT-S2 was generated by including<br>
23 genes that were over-expressed in the tracer as additional drivers.<br>
A second human lung adenocarcinoma cDNA expression library was<br>
constructed using adenocarcinoma tissue from a second patient (# 86-66) and used to<br>
prepare a second lung adenocarcinoma-specific subtracted cDNA library (referred to as<br>
LAT2-S2), as described above, using the same panel of normal tissues and the<br>
additional genes over-expressed inLAT-Sl.<br>
A third human metastatic lung adenocarcinoma library was constructed<br>
from a pool of two lung pleura! effusions with lung and gastric adenocarcinoma origins.<br>
The subtracted cDNA library, Mets-sub2 was generated as described above using the<br>
same panel of normal tissues. However, the Mets-sub3 subtracted library was<br>
constructed by including 51 additional genes as drivers. These 51 genes were recovered<br>
in Mets-sub2, representing over-expressed housekeeping genes in the testers. As a<br>
result, Mets-sub3 is more complexed and normalized.<br>
A total of 16 cDNA fragments isolated from LAT-S1, 585 cDNA<br>
fragments isolated from LAT-S2, 568 cDNA clones from LAT2-S2, 15 cDNA clones<br>
from Mets-sub2 and 343 cDNA clones from Mets-sub3, described above, were colony<br>
PCR amplified and their mRNA expression levels in lung tumor, normal lung, and<br>
various other normal and tumor tissues were determined using microarray technology<br>
(Incyte, Palo Alto, CA). Briefly, the PCR amplification products were dotted onto<br>
slides in an array format, with each product occupying a unique location in the array.<br>
mRNA was extracted from the tissue sample to be tested, reverse transcribed, and<br>
fluorescent-labeled cDNA probes were generated. The microarrays were probed with<br>
the labeled cDNA probes, the slides scanned and fluorescence intensity was measured.<br>
This intensity correlates with the hybridization intensity. Seventy-three non-redundant<br>
cDNA clones, of which 42 were found to be unique, showed over-expression in lung<br>
tumors, with expression in normal tissues tested (lung, skin, lymph node, colon, liver,<br>
pancreas, breast, heart, bone marrow, large intestine, kidney, stomach, brain, small<br>
intestine, bladder and salivary gland) being either undetectable, or at significantly lower<br>
levels compared to lung adenocarcinoma tumors. These clones were further<br>
characterized by DNA sequencing with a Perkin Elmer/Applied Biosystems Division<br>
Automated Sequencer Model 373A and/or Model 377 (Foster City, CA).<br>
The sequences were compared to known sequences in the gene bank<br>
using the EMBL GenBank databases (release 96). No significant homologies were<br>
found to the sequence provided in SEQ ID NO: 67, with no apparent homology to<br>
previously identified expressed sequence tags (ESTs). The sequences of SEQ ID NO:<br>
60, 62, 65, 66, 69-71, 74, 76, 79, 80, 84, 86, 89-92, 95, 97 and 98 were found to show<br>
some homology to previously identified expressed sequence tags (ESTs). The cDNA<br>
sequences of SEQ ID NO: 59, 61, 63, 64, 67, 68, 72, 73, 75, 77, 78, 81-83, 85, 87, 88,<br>
93, 94, 96, 99 and 100 showed homology to previously identified genes. The fulllength<br>
cDNA sequences for the clones of SEQ ID NO: 96 and 100 are provided in SEQ<br>
ID NO: 316 and 318, respectively. The amino acid sequences for the clones of SEQ ID<br>
NO: 59, 61, 63, 64, 68, 73, 82, 83, 94, 96 and 100 are provided in SEQ ID NO: 331,<br>
328, 329, 332, 327, 333, 330, 326, 325, 324 and 335, respectively. A predicted amino<br>
acid sequence encoded by the sequence of SEQ ID NO: 69 (referred to as L552S) is<br>
provided in SEQ ID NO: 786.<br>
Further studies led to the isolation of an extended cDNA sequence, and<br>
open reading frame, for L552S (SEQ ID NO: 790). The predicted amino acid sequence<br>
encoded by the cDNA sequence of SEQ ID NO: 790 is provided in SEQ ID NO: 791.<br>
The determined cDNA sequence of an isoform of L552S is provided in SEQ ID NO:<br>
792, with the corresponding predicted amino acid sequence being provided in SEQ ID<br>
NO: 793. Subsequent studies led to the isolation of the full-length cDNA sequence of<br>
L552S (SEQ ID NO: 808). The corresponding amino acid sequence is provided in SEQ<br>
ID NO: 809. No homologies were found to the protein sequence of L552S. However,<br>
nucleotides 533-769 of the full-length cDNA sequence were found to show homology<br>
to a previously identified DNA sequence.<br>
Full-length cloning efforts on L552S led to the isolation of three<br>
additional cDNA sequences (SEQ ID NO: 810-812) from a metastatic lung<br>
adenocarcinoma library. The sequence of SEQ ID NO: 810 was found to show some<br>
homology to previously identified human DNA sequences.- The sequence of SEQ ID<br>
NO: 811 was found to show some homology to a previously identified DNA sequence.<br>
The sequence of SEQ ID NO: 812 was found to show some homology to previously<br>
identified ESTs.<br>
The gene of SEQ ID NO: 84 (referred to as L551S) was determined by<br>
real-time RT-PCR analysis to be over-expressed in 2/9 primary adenocarcinomas and to<br>
be expressed at lower levels in 2/2 metastatic adenocarcinomas and 1/2 squamous cell<br>
carcinomas. No expression was observed in normal tissues, with the exception of very<br>
low expression in normal stomach. Further studies on L551S led to the- isolation of the<br>
5' and 3' cDNA consensus sequences provided in SEQ ID NO: 801 and 802,<br>
respectively. The L551S 5' sequence was found to show some homology to the<br>
previously identified gene STY8 (cDNA sequence provided in SEQ ID NO: 803;<br>
corresponding amino acid sequence provided in SEQ ID NO: 805), which is a mitogen<br>
activated protein kinase phosphatase. However, no significant homologies were found<br>
to the 3' sequence of L551S. Subsequently, an extended cDNA sequence for L551S<br>
was isolated (SEQ ID NO: 804). The corresponding amino acid sequence is provided<br>
in SEQ ID NO: 806. Further studies led to the isolation of two independent full-length<br>
clones for L551S (referred to as 54298 and 54305). These two clones have five<br>
nucleotide differences compared to the STY8 DNA sequence. Two of these differences<br>
are single nucleotide polymorphisms which do not effect the encoded amino acid<br>
sequences. The other three nucleotide differences are consistent between the two<br>
L551S clones but lead to encoded amino acid sequences that are different from the<br>
STY8 protein sequence. The determined cDNA sequences for the L551S full-length<br>
clones 54305 and 54298 are provided in SEQ ID NO: 825 and 826, respectively, with<br>
the amino acid sequence for L551S being provided in SEQ ID NO: 827.<br>
B. Isolation of cDNA Sequences from Lung Adenocarcinoma Libraries<br>
using PCR-Based cDNA Library Subtraction<br>
cDNA clones from a PCR-based subtraction library, containing cDNA<br>
from a pool of two human lung primary adenocarcinomas subtracted against a pool of<br>
nine normal human tissue cDNAs including skin, colon, lung, esophagus, brain, kidney,<br>
spleen, pancreas and liver, (Clontech, Palo Alto, CA) were derived and submitted to a<br>
first round of PCR amplification. This library (referred to as ALT-1) was subjected to a<br>
second round of PCR amplification, following the manufacturer's protocol. The<br>
expression levels of 760 cDNA clones in lung tumor, normal lung, and various other<br>
normal and tumor tissues, were examined using microarray technology as described<br>
above. A total of 118 clones, of which 55 were unique, were found to be overexpressed<br>
hi lung tumor tissue, with expression in normal tissues tested (lung, skin,<br>
lymph node, colon, liver, pancreas, breast, heart, bone marrow, large intestine, kidney,<br>
stomach, brain, small intestine, bladder and salivary gland) being either-undetectable, or<br>
at significantly lower levels. The sequences were compared to known sequences in the<br>
gene bank using the EMBL. and GenBank databases (release 96). No significant<br>
homologies (including ESTs) were found to the sequence provided hi SEQ ID NO: 44.<br>
The sequences of SEQ ID NO: 1, 11, 13, 15, 20, 23-27, 29, 30, 33, 34, 39, 41, 43, 45,<br>
46, 51 and 57 were found to show some homology to previously identified expressed<br>
sequence tags (ESTs). The cDNA sequences of SEQ ID NO: 2-10, 12, 14, 16-19, 21,<br>
22, 28, 31, 32, 35-38,40, 42, 44,47-50, 52-56 and 58 showed homology to previously<br>
identified genes. The full-length cDNA sequences for the clones of SEQ ID NO: 18,<br>
22, 31, 35, 36 and 42 are provided in SEQ ID NO: 320, 319, 323, 321, 317, 321 and<br>
322, respectively, with the corresponding amino acid sequences being provided in SEQ<br>
ID NO: 337,336,340,338,334, and 339, respectively.<br>
Further studies led to the isolation of an extended cDNA sequence for<br>
the clone of SEQ ID NO: 33 (referred to as L801P). This extended cDNA sequence<br>
(provided in SEQ ID NO: 796), was found to contain three potential open reading<br>
frames (ORFs). The predicted amino acid sequences encoded by these three ORFs are<br>
provided in SEQ ID NO: 797-799, respectively.<br>
In subsequent studies, a full-length cDNA sequence for the clone of SEQ<br>
ID NO: 44 (referred to as L844P) was isolated (provided in SEQ ID NO: 800).<br>
Comparison of this sequence with those in the public databases revealed that the 470<br>
bases at the 5' end of the sequence show homology to the known gene dihydrodiol<br>
dehydrogenase, thus indicating that L844P is a novel transcript of the dihydrodiol<br>
dehydrogenase family having 2007 base pairs of previously unidentified 3'<br>
untranslated region.<br>
The predicted amino acid sequence encoded by the sequence of SEQ ID<br>
NO: 46 (referred to as L840P) is provided in SEQ ID NO: 787. An extended cDNA<br>
sequence for L840P, which was determined to include an open reading frame, is<br>
provided hi SEQ ID NO: 794. The predicted amino acid sequence encoded by the<br>
cDNA sequence of SEQ ID NO: 794 is provided in SEQ ID NO: 795. The full-length<br>
cDNA sequence for the clone of SEQ ID NO: 54 (referred to as L548S) is provided in<br>
SEQ ID NO: 788, with the corresponding amino acid sequence being provided in SEQ<br>
ID NO: 789.<br>
Northern blot analyses of the genes of SEQ ID NO: 25 and 46 (referred<br>
to as L839P and L840P, respectively) were remarkably similar. Both genes were<br>
expressed in 1/2 lung adenocarcinomas as two bands of 3.6 kb and 1.6 kb. No<br>
expression of L839P was observed in normal lung or trachea. No expression of L840P<br>
was observed in normal bone marrow, resting or activated PBMC, esophagus, or<br>
normal lung." Given the similar expression patterns, L839P and L840P may be derived<br>
from the same gene.<br>
Further studies on L773P (SEQ ID NO: 58) resulted in the isolation of<br>
the extended consensus cDNA sequence provided hi SEQ ID NO: 807.<br>
Additional lung adenocarcinoma cDNA clones were isolated as follows.<br>
A cDNA library was prepared from a pool of two lung adenocarcinomas and subtracted<br>
against cDNA from a panel of normal tissues including lung, brain, liver, kidney,<br>
pancreas, skin, heart and spleen. The subtraction was performed using a PCR-based<br>
protocol (Clontech), which was modified to generate larger fragments. Within this<br>
protocol, tester and driver double stranded cDNA were separately digested with five<br>
restriction enzymes that recognize six-nucleotide restriction sites (Mlul, MscI, PvuII,<br>
Sail and StuI). This digestion resulted in an average cDNA size of 600 bp, rather than<br>
the average size of 300 bp that results from digestion with Rsal according to the<br>
Clontech protocol. The ends of the restriction digested tester cDNA were filled hi to<br>
generate blunt ends for adapter ligation. This modification did not affect the subtraction<br>
efficiency. Two tester populations were then created with different adapters, and the<br>
driver library remained without adapters. The tester and driver libraries were then<br>
hybridized using excess driver cDNA. In the first hybridization step, driver was<br>
separately hybridized with each of the two tester cDNA populations. This resulted in<br>
populations of (a) unhybridized tester cDNAs, (b) tester cDNAs hybridized to other<br>
tester cDNAs, (c) tester cDNAs hybridized to driver cDNAs and (d) unhybridized<br>
driver cDNAs. The two separate hybridization reactions were then combined, and<br>
rehybridized in the presence of additional denatured driver cDNA. Following this<br>
second hybridization, in addition to populations (a) through (d), a fifth population (e)<br>
was generated in which tester cDNA with one adapter hybridized to tester cDNA with<br>
the second adapter. Accordingly, the second hybridization step resulted in enrichment<br>
of differentially expressed sequences which could be used as templates for PCR<br>
amplification with adaptor-specific primers.<br>
The ends were then filled in, and PCR amplification was performed<br>
using adaptor-specific primers. Only population (e), which contained tester cDNA that<br>
did not hybridize to driver cDNA, was amplified exponentially. A second PCR<br>
amplification step was then performed, to reduce background and further enrich<br>
differentially expressed sequences.<br>
Fifty-seven cDNA clones were isolated from the subtracted library<br>
(referred to as LAP1) and sequenced. The determined cDNA sequences for 16 of these<br>
clones are provided in SEQ ID NO: 101-116. The sequences of SEQ ID NO: 101 and<br>
114 showed no significant homologies to previously identified sequences. The<br>
sequences of SEQ ID NO: 102-109 and 112 showed some similarity to previously<br>
identified sequences, while the sequences of SEQ ID NO: 113, 115 and 116 showed<br>
some similarity to previously isolated ESTs.<br>
C. Isolation of cDNA Sequences from Small Cell Lung Carcinoma<br>
Libraries using PCR-Based cDNA Library Subtraction<br>
A subtracted cDNA library for small cell lung carcinoma (referred to as<br>
SCL1) was prepared using essentially the modified PCR-based subtraction process<br>
described above. cDNA from small cell lung carcinoma was subtracted against cDNA<br>
from a panel of normal tissues, including normal lung, brain, kidney, liver, pancreas,<br>
skin, heart, lymph node and spleen. Both tester and driver poly A+ RNA were initially<br>
amplified using SMART PCR cDNA synthesis kit (Clontech, Palo Alto, CA). The<br>
tester and driver double stranded cDNA were separately digested with five restriction<br>
enzymes (Oral, MscI, PvuII, Smal, and StuI). These restriction enzymes generated<br>
blunt end cuts and the digestion resulted in an average insert size of 600 bp. Digestion<br>
with this set of restriction enzymes eliminates the step required to generate blunt ends<br>
by filling in of the cDNA ends. These modifications did not affect subtraction<br>
efficiency.<br>
Eighty-five clones were isolated and sequenced. The determined cDNA<br>
sequences for 31 of these clones are provided in SEQ ID NO: 117-147: The sequences<br>
of SEQ ID NO: 122, 124, 126, 127, 130, 131, 133, 136, 139 and 147 showed no<br>
significant homologies to previously identified sequences. The sequences of SEQ ID<br>
NO: 120, 129, 135, 137, 140, 142, 144 and 145 showed some similarity to previously<br>
identified gene sequences, while the sequences of SEQ ID NO: 114,118,119,121,123,<br>
125, 128, 132, 134, 138, 141, 143 and 147 showed some similarity to previously<br>
isolated ESTs.<br>
In further studies, three additional cDNA libraries were generated from<br>
poly A+ RNA from a single small cell lung carcinoma sample subtracted against a pool<br>
of poly A+ RNA from nine normal tissues (lung, brain, kidney, liver, pancreas, skin,<br>
heart pituitary gland and spleen). For the first library (referred to as SCL2), the<br>
subtraction was carried out essentially as described above for the LAPI library, with the<br>
exception that the tester and driver were digested with PvuII, StuI, MscI and Dral. The<br>
ratio of tester and driver cDNA used was as recommended by Clontech. For the second<br>
library (referred to as SCL3), subtraction was performed essentially as for SCL2 except<br>
that cDNA for highly redundant clones identified from the SCL2 library was included<br>
in the driver cDNA. Construction of the SCL4 library was performed essentially as<br>
described for the SCL3 library except that a higher ratio of driver to tester was<br>
employed.<br>
Each library was characterized by DNA sequencing and database<br>
analyses. The determined cDNA sequence for 35 clones isolated from the SCL2 library<br>
are provided in SEQ ID NO: 245-279, with the determined cDNA sequences for 21<br>
clones isolated from the SCL3 library and for 15 clones isolated from the SCL4 library<br>
being provided in SEQ ID NO: 280-300 and 301-315, respectively. The sequences of<br>
SEQ ID NO: 246,254,261,262,304,309 and 311 showed no significant homologies to<br>
previously identified sequences. The sequence of SEQ ID NO: 245,248,255,266,270,<br>
275, 280, 282, 283, 288-290, 292, 295, 301 and 303 showed some homology to<br>
previously isolated ESTs, while the sequences of SEQ ID NO: 247, 249-253, 256-260,<br>
263-265, 267-269, 271-274, 276-279,281, 284-287, 291, 293, 294, 296-300, 302, 305-<br>
308,310 and 312-315 showed some homology to previously identified gene sequences.<br>
D. Isolation of cDNA Sequences from a Neuroendocrine Library using<br>
PCR-Based cDNA Library Subtraction<br>
Using the modified PCR-based subtraction process, essentially as<br>
described above for the LAP1 subtracted library, a subtracted cDNA library (referred to<br>
as MLN1) was derived from a lung neuroendocrine carcinoma that had metastasized to<br>
the subcarinal lymph node, by subtraction with a panel of nine normal tissues, including<br>
normal lung, brain, kidney, liver, pancreas, skin, heart, lymph node and spleen.<br>
Ninety-one individual clones were isolated and sequenced. The<br>
determined cDNA sequences for 58 of these clones are provided in SEQ ID NO: 147-<br>
222. The sequences of SEQ ID NO: 150, 151, 154, 157, 158, 159, 160, 163, 174, 175,<br>
178, 186-190, 192, 193, 195-200, 208-210, 212-215 and 220 showed no significant<br>
homologies to previously identified sequences. The sequences of SEQ ID NO: 152,<br>
155,156,161, 165,166, 176,179, 182, 184,185, 191,194, 221 and 222 showed some<br>
similarity to previously identified gene sequences, while the sequences of SEQ ID NO:<br>
148, 149, 153, 164, 167-173, 177, 180, 181, 183, 201-207, 211 and 216-219 showed<br>
some similarity to previously isolated ESTs.<br>
The determined cDNA sequences of an additional 442 clones isolated<br>
from the MLN1 library are provided in SEQ ID NO: 341-782.<br>
E. Isolation of cDNA Sequences from a Squamous Cell Lung Carcinoma<br>
Library using PCR-Based cDNA Library Subtraction<br>
A subtracted cDNA library for squamous cell lung carcinoma (referred<br>
to as SQL1) was prepared, essentially using the modified PCR-based subtraction<br>
process described above, except the tester and driver double stranded cDNA were<br>
separately digested with four restriction enzymes (Dral, MscI, PvuII and StuI) cDNA<br>
from a pool of two squamous cell lung carcinomas was subtracted against cDNA from a<br>
pool of 10 normal tissues, including normal lung, brain, kidney, liver, pancreas, skin,<br>
heart, spleen, esophagus and trachea.<br>
Seventy-four clones were isolated and sequenced. The determined<br>
cDNA sequences for 22 of these clones are provided in SEQ ID NO: 223-244. The<br>
sequence of SEQ ID NO: 241 showed no significant homologies to previously<br>
identified sequences. The sequences of SEQ ID NO: 223, 225, 232,233,235,238,239,<br>
242 and 243 showed some similarity to previously identified gene sequences, while the<br>
sequences of SEQ ID NO: 224,226-231,234,236, 237,240,241 and 244 showed some<br>
similarity to previously isolated ESTs.<br>
The sequences of an additional 12 clones isolated during chracterization<br>
of cDNA libraries prepared from lung tumor tissue are provided in SEQ ID NO: 813-<br>
824. Comparison of these sequences with those in the GenBank database and the<br>
GeneSeq DNA database revealed no significant homologies to previously identified<br>
sequences.<br>
EXAMPLE 2<br>
SYNTHESIS OF POLYPEPTIDES<br>
Polypeptides may be synthesized on a Perkin Elmer/Applied Biosystems<br>
Division 430A peptide synthesizer using FMOC chemistry with HPTU (OBenzotriazole-<br>
N,N,N',N'-teteamernyluromum hexafluorophosphate) activation. A Gly-<br>
Cys-Gly sequence may be attached to the ammo terminus of the peptide to provide a<br>
method of conjugation, binding to an immobilized surface, or labeling of the peptide.<br>
Cleavage of the peptides from the solid support may be carried out using the following<br>
cleavage mixture: trifluoroacetic acid:ethanedithiol:thioanisole:water:phenol<br>
(40:1:2:2:3). After cleaving for 2 hours, the peptides may be precipitated in cold<br>
methyl-t-butyl-ether. The peptide pellets may then be dissolved in water containing<br>
0.1% trifluoroacetic acid (TFA) and lyophilized prior to purification by CIS reverse<br>
phase HPLC. A gradient of 0%-60% acetonitrile (containing 0.1% TFA) hi water<br>
(containing 0.1% TFA) may be used to elute the peptides. Following lyophilization of<br>
the pure fractions, the peptides may be characterized using electrospray or other types<br>
of mass spectrometry and by amino acid analysis.<br>
EXAMPLES<br>
PREPARATION OF ANTIBODIES AGAINST LUNG CANCER ANTIGENS<br>
Polyclonal antibodies against the lung cancer antigen L773P (SEQ ID<br>
NO: 783) were prepared as follows.<br>
Rabbits were immunized with recombinant protein expressed in and<br>
purified from E. coli as described above. For the initial immunization, 400 ng of<br>
antigen combined with muramyl dipeptide (MDP) was injected subcutaneously (S.C.).<br>
Animals were boosted S.C. 4 weeks later with 200 fig of antigen mixed with incomplete<br>
Freund's Adjuvant (IFA). Subsequent boosts of 100 fig of antigen mixed with IFA<br>
were injected S.C. as necessary to induce high antibody titer responses. Serum bleeds<br>
from immunized rabbits were tested for L773P-specific reactivity using ELISA assays<br>
with purified protein and showed strong reactivity to L733P. Polyclonal antibodies<br>
against L773P were affinity purified from high titer polyclonal sera using purified<br>
protein attached to a solid support.<br>
EXAMPLE 4<br>
PROTEIN EXPRESSION OF LUNG TUMOR-SPECIFIC ANTIGENS<br>
Full-length L773P (amino acids 2-364 of SEQ ID NO: 783), with a 6X<br>
His Tag, were subcloned into the pPDM expression vector and transfonned into either<br>
BL21 CodonPlus or BL21 pLysS host cells using standard techniques. High levels of<br>
expression were observed in both cases. Similarly, the N-terminal portion of L773P<br>
(amino acids 2-71 of SEQ ID NO: 783; referred to as L773PA), with a 6X His tag were<br>
subcloned into the vector pPDM and transfonned into BL21 CodonPlus host cells. Low<br>
levels of expression were observed by N-tenninal sequencing. The sequence of the<br>
expressed constructs for L773P and L773PA are provided in SEQ ID NO: 784 and 785,<br>
respectively.<br>
From the foregoing it will be appreciated that, although specific<br>
embodiments of the invention have been described herein for purposes of illustration,<br>
various modifications may be made without deviating from the spirit and scope of the<br>
invention. Accordingly, the invention is not limited except as by the appended claims.<br><br>
COMPOSITIONS AND METHODS FOR THE THERAPY AND DIAGNOSIS OF<br>
LUNG CANCER<br>
TECHNICAL FIELD OF THE INVENTION<br>
The present invention relates generally to therapy and diagnosis of<br>
cancer, such as lung cancer. The invention is more specifically related to polypeptides<br>
comprising at least a portion of a lung tumor protein, and to polynucleotides encoding<br>
such polypeptides. Such polypeptides and polynucleotides may be used in<br>
compositions for prevention and treatment of lung cancer, and for the diagnosis and<br>
monitoring of such cancers.<br>
BACKGROUND OF THE INVENTION<br>
Cancer is a significant health problem throughout the world. Although<br>
advances have been made in detection and therapy of cancer, no vaccine or other<br>
universally successful method for prevention or treatment is currently available.<br>
Current therapies, which are generally based on a combination of, chemotherapy or<br>
surgery and radiation, continue to prove inadequate in many patients.<br>
Lung cancer is the primary cause of cancer death among both men and<br>
women in the U.S., with an estimated 172,000 new cases being reported in 1994. The<br>
five-year survival rate among all lung cancer patients, regardless of the stage of disease<br>
at diagnosis, is only 13%. This contrasts with a five-year survival rate of 46% among<br>
cases detected while the disease is still localized. However, only 16% of lung cancers<br>
are discovered before the disease has spread.<br>
Early detection is difficult since clinical symptoms are often not seen<br>
until the disease has reached an advanced stage. Currently, diagnosis is aided by the<br>
use of chest x-rays, analysis of the type of cells contained in sputum and fiberoptic<br>
examination of the bronchial passages. Treatment regimens are determined by the type<br>
and stage of the cancer, and include surgery, radiation therapy and/or chemotherapy.<br>
In spite of considerable research into therapies for this and other cancers,<br>
lung cancer remains difficult to diagnose and treat effectively. Accordingly, there is a<br>
need in the art for improved methods for detecting and treating such cancers. The<br>
present invention fulfills these needs and further provides other related advantages.<br>
SUMMARY OF THE INVENTION<br>
Briefly stated, the present invention provides compositions and methods<br>
for the diagnosis and therapy of cancer, such as lung cancer. In one aspect, the present<br>
invention provides polypeptides comprising at least a portion of a lung tumor protein, or<br>
a variant thereof. Certain portions and other variants are immunogenic, such that the<br>
ability of the variant to react with antigen-specific antisera is not substantially<br>
diminished. Within certain embodiments, the polypeptide comprises a sequence that is<br>
encoded by a polynucleotide sequence selected from the group consisting of: (a)<br>
sequences recited in SEQ ID NO: 1,11-13,15, 20, 23-27, 29,30, 33, 34, 39,41,43-46,<br>
51, 52, 57, 58, 60, 62, 65-67, 69-71, 74, 76, 79, 80, 84, 86, 89-92, 95, 97, 98,101, 110,<br>
111, 113-119, 121-128, 130-134,136, 138, 139, 141, 143, 146-151,153, 154, 157-160,<br>
162-164, 167-178,180,181,183,186-190, 192, 193,195-220, 224, 226-231, 234, 236,<br>
237, 240, 241, 244-246, 248, 254, 255, 261, 262, 266, 270, 275, 280, 282, 283, 288,<br>
289, 290, 292, 295, 301, 303, 304, 309, 311, 341-782, 784, 785, 790, 792, 794, 796,<br>
800-804, 807, 808 and 810-826; (b) variants of a sequence recited in SEQ ID NO: 1,<br>
11-13, 15, 20,23-27,29, 30, 33, 34, 39,41,43-46, 51, 52, 57, 58, 60, 62,65-67, 69-71,<br>
74, 76, 79, 80, 84, 86, 89-92, 95, 97, 98, 101, 110, 111, 113-119, 121-128, 130-134,<br>
136,138, 139,141, 143,146-151,153,154, 157-160,162-164, 167-178,180,181, 183,<br>
186-190, 192, 193, 195-220, 224,226-231, 234, 236, 237, 240,241, 244-246,248, 254,<br>
255, 261, 262, 266, 270, 275, 280, 282, 283, 288, 289, 290, 292, 295, 301, 303, 304,<br>
309, 311, 341-782, 784, 785, 790, 792, 794, 796, 800-804, 807, 808 and 810-826; and<br>
(c) complements of a sequence of (a) or (b). In specific embodiments, the polypeptides<br>
of the present invention comprise at least a portion of a tumor protein that includes an<br>
amino acid sequence selected from the group consisting of sequences recited in SEQ ID<br>
NO: 786, 787,791, 793,795,797-799,806, 809 and 827, and variants thereof.<br>
The present invention further provides polyhucleotides that encode a<br>
polypeptide as described above, or a portion thereof (such as a portion encoding at least<br>
IS amino acid residues of a lung tumor protein), expression vectors comprising such<br>
polynucleotides and host cells transformed or transfected with such expression Vectors.<br>
Within other aspects, the present invention provides pharmaceutical<br>
compositions comprising a polypeptide or polynucleotide as described above and a<br>
physiologically acceptable carrier.<br>
Within a related aspect of the present invention, vaccines, or<br>
immunogenic compositions, for prophylactic or therapeutic use are provided. Such<br>
vaccines comprise a polypeptide or polynucleotide as described above and an<br>
immunostimulant.<br>
The present invention further provides pharmaceutical compositions that<br>
comprise: (a) an antibody or antigen-binding fragment thereof that specifically binds to<br>
a lung tumor protein; and (b) a physiologically acceptable carrier.<br>
Within further aspects, the present invention provides pharmaceutical<br>
compositions comprising: (a) an antigen presenting cell that expresses a polypeptide as<br>
described above and (b) a pharmaceutically acceptable carrier or excipient. Antigen<br>
presenting cells include dendritic cells, macrophages, monocytes, fibroblasts and B<br>
cells.<br>
Within related aspects, vaccines, or immunogenic compositions, are<br>
provided that comprise: (a) an antigen presenting cell that expresses a polypeptide as<br>
described above and (b) an immunostimulant.<br>
The present invention further provides, in other aspects, fusion proteins<br>
that comprise at least one polypeptide as described above, as well as polynucleotides<br>
encoding such fusion proteins.<br>
Within related aspects, pharmaceutical compositions comprising a fusion<br>
protein, or a polynucleotide encoding a fusion protein, in combination with a<br>
physiologically acceptable carrier are provided.<br>
Vaccines, or immunogenic compositions, are further provided, within<br>
other aspects, that comprise a fusion protein, or a polynucleotide encoding a fusion<br>
protein, in combination with an immunostimulant.<br>
Within further aspects, the present invention provides methods for<br>
inhibiting the development of a cancer in a patient, comprising administering to a<br>
patient a pharmaceutical composition or immunogenic composition as recited above:<br>
The patient may be afflicted with lung cancer, in which case the methods provide<br>
treatment for the disease, or patient considered at risk for such a disease may be treated<br>
prophylactically.<br>
The present invention further provides, within other aspects, methods for<br>
removing tumor cells from a biological sample, comprising contacting a biological<br>
sample with T cells that specifically react with a lung tumor protein, wherein the step of<br>
contacting is performed under conditions and for a time sufficient to permit the removal<br>
of cells expressing the protein from the sample.<br>
Within related aspects, methods are provided for inhibiting the<br>
development of a cancer in a patient, comprising administering to a patient a biological<br>
sample treated as described above.<br>
Methods are further provided, within other aspects, for stimulating<br>
and/or expanding T cells specific for a lung tumor protein, comprising contacting T<br>
cells with one or more of: (i) a polypeptide as described above; (ii) a polynucleotide<br>
encoding such a polypeptide; and/or (iii) an antigen presenting cell that expresses such<br>
a polypeptide; under conditions and for a time sufficient to permit the stimulation<br>
and/or expansion of T cells. Isolated T cell populations comprising T cells prepared as<br>
described above are also provided.<br>
Within further aspects, the present invention provides methods for<br>
inhibiting the development of a cancer in a patient, comprising administering to a<br>
patient an effective amount of a T cell population as described above.<br>
The present invention further provides methods for inhibiting the<br>
development of a cancer in a patient, comprising the steps of: (a) incubating CD4+<br>
and/or CD8+ T cells isolated from a patient with one or more of: (i) a polypeptide<br>
comprising at least an immunogenic portion of a lung tumor protein; (ii) a<br>
polynucleotide encoding such a polypeptide; and (iii) an antigen-presenting cell that<br>
expressed such a polypeptide; and (b) administering to the patient an effective amount<br>
of the proliferated T cells, and thereby inhibiting the development of a cancer in the<br>
patient Proliferated cells may, but need not, be cloned prior to administration to the<br>
patient.<br>
Within further aspects, the present invention provides methods for<br>
determining the presence or absence of a cancer in a patient, comprising: (a) contacting<br>
a biological sample obtained from a patient with a binding agent that binds to a<br>
polypeptide as recited above; (b) detecting in the sample an amount of polypeptide that<br>
binds to the binding agent; and (c) comparing the amount of polypeptide with a<br>
predetermined cut-off value, and therefrom determining the presence or absence of a<br>
cancer in the patient. Within preferred embodiments, the binding agent is an antibody,<br>
more preferably a monoclonal antibody. The cancer may be lung cancer.<br>
The present invention also provides, within other aspects, methods for<br>
monitoring the progression of a cancer in a patient. Such methods comprise the steps<br>
of: (a) contacting a biological sample obtained from a patient at a first point hi tune<br>
with a binding agent that binds to a polypeptide as recited above; (b) detecting in the<br>
sample an amount of polypeptide that binds to the binding agent; (c) repeating steps (a)<br>
and (b) using a biological sample obtained from the patient at a subsequent point in<br>
time; and (d) comparing the amount of polypeptide detected in step (c) with the amount<br>
detected in step (b) and therefrom monitoring the progression of the cancer in the<br>
patient.<br>
The present invention further provides, within other aspects, methods for<br>
determining the presence or absence of a cancer hi a patient, comprising the steps of: (a)<br>
contacting a biological sample obtained from a patient with an oligonucleotide that<br>
hybridizes to a polynucleotide that encodes a lung tumor protein; (b) detecting in the<br>
sample a level of a polynucleotide, preferably mRNA, that hybridizes to the<br>
oligonucleotide; and (c) comparing the level of polynucleotide that hybridizes to the<br>
oligonucleotide with a predetermined cut-off value, and therefrom determining the<br>
presence or absence of a cancer hi the patient. Within certain embodiments, the amount<br>
of mRNA is detected via polymerase chain reaction using, for example, at least one<br>
oligonucleotide primer that hybridizes to a polynucleotide encoding a polypeptide as<br>
recited above, or a complement of such a polynucleotide. Within other embodiments,<br>
the amount of mRNA is detected using a hybridization technique, employing an<br>
oligonucleotide probe that hybridizes to a polynucleotide that encodes a polypeptide as<br>
recited above, or a complement of such a polynucleotide.<br>
In related aspects, methods are provided for monitoring the progression<br>
of a cancer in a patient, comprising the steps of: (a) contacting a biological sample<br>
obtained from a patient with an oligonucleotide that hybridizes to a polynucleotide that<br>
encodes a lung tumor protein; (b) detecting in the sample an amount of a polynucleotide<br>
that hybridizes to the oligonucleotide; (c) repeating steps (a) and (b) using a biological<br>
sample obtained from the patient at a subsequent point in time; and (d) comparing the<br>
amount of polynucleotide detected in step (c) with the amount detected in step (b) and<br>
therefrom monitoring the progression of the cancer in the patient<br>
Within further aspects, the present invention provides antibodies, such as<br>
monoclonal antibodies, that bind to a polypeptide as described above, as well as<br>
diagnostic kits comprising such antibodies. Diagnostic kits comprising one or more<br>
oligonucleotide probes or primers as described above are also provided.<br>
These and other aspects of the present invention will become apparent<br>
upon reference to the following detailed description and attached drawings. All<br>
references disclosed herein are hereby incorporated by reference in their entirety as if<br>
each was incorporated individually.<br>
SEQUENCE IDENTIFIERS<br>
SEQ ID NO: 1 is the determined cDNA sequence for clone #19038, also<br>
referred to as L845P.<br>
SEQ ID NO: 2 is the determined cDNA sequence for clone #19036.<br>
SEQ ID NO: 3 is the determined cDNA sequence for clone #19034.<br>
SEQ ED NO: 4 is the determined cDNA sequence for clone #19033.<br>
SEQ ID NO: 5 is the determined cDNA sequence for clone #19032.<br>
SEQ ID NO: 6 is the determined cDNA sequence for clone #19030, also<br>
referred to as L559S.<br>
SEQ ID NO: 7 is the determined cDNA sequence for clone #19029.<br>
SEQ ID NO: 8 is the determined cDNA sequence for clone #19025.<br>
SEQ ID NO: 9 is the determined cDNA sequence for clone #19023.<br>
SEQ ID NO: 10 is the determined cDNA sequence for clone #18929.<br>
SEQ ID NO: 11 is the determined cDNA sequence for clone #19010.<br>
SEQ ID NO: 12 is the determined cDNA sequence for clone #19009.<br>
SEQ ID NO: 13 is the determined cDNA sequence for clones #19005,<br>
19007,19016 and 19017.<br>
SEQ ID NO: 14 is the determined cDNA sequence for clone # 19004.<br>
SEQ ID NO: 15 is the determined cDNA sequence for clones #19002<br>
and 18965.<br>
SEQ ID NO: 16 is the determined cDNA sequence for clone #18998.<br>
SEQ ID NO: 17 is the determined cDNA sequence for clone #18997.<br>
SEQ ID NO: 18 is the determined cDNA sequence for clone #18996.<br>
SEQ ID NO: 19 is the determined cDNA sequence for clone #18995.<br>
SEQ ID NO: 20 is the determined cDNA sequence for clone #18994,<br>
also known as L846P.<br>
SEQ ID NO: 21 is the determined cDNA sequence for clone #18992.<br>
SEQ ID NO: 22 is the determined cDNA sequence for clone #18991.<br>
SEQ ID NO: 23 is the determined cDNA sequence for clone #18990,<br>
also referred to as clone #20111.<br>
SEQ ID NO: 24 is the determined cDNA sequence for clone #18987.<br>
SEQ ID NO: 25 is the determined cDNA sequence for clone #18985,<br>
also referred as L839P.<br>
SEQ ID NO: 26 is the determined cDNA sequence for clone #18984,<br>
also referred to as L847P.<br>
SEQ ID NO: 27 is the determined cDNA sequence for clone # 18983.<br>
SEQ ID NO: 28 is the determined cDNA sequence for clones #18976<br>
and 18980.<br>
SEQ ID NO: 29 is the determined cDNA sequence for clone #18975.<br>
SEQ ID NO: 30 is the determined cDNA sequence for clone #18974.<br>
SEQ ID NO: 31 is the determined cDNA sequence for clone #18973.<br>
SEQ ID NO: 32 is the determined cDNA sequence for clone #18972.<br>
SEQ ID NO: 33 is the determined cDNA sequence for clone #18971,<br>
also referred to as L801P.<br>
SEQ ID NO: 34 is the determined cDNA sequence for clone #18970.<br>
SEQ ID NO: 3 5 is the determined cDNA sequence for clone #18966.<br>
SEQ ID NO: 36 is the determined cDNA sequence for .clones #18964,<br>
18968 and 19039.<br>
SEQ ID NO: 37 is the determined cDNA sequence for clone #18960.<br>
SEQ ID NO: 38 is the determined cDNA sequence for clone #18959.<br>
SEQ ID NO: 39 is the determined cDNA sequence for clones #18958<br>
and 18982.<br>
SEQ ID NO: 40 is the determined cDNA sequence for clones #18956<br>
and 19015.<br>
SEQ ID NO: 41 is the determined cDNA sequence for clone #18954,<br>
also referred to L848P.<br>
SEQ ID NO: 42 is the determined cDNA sequence for clone #18951.<br>
SEQ ID NO: 43 is the determined cDNA sequence for clone #18950.<br>
SEQ ID NO: 44 is the determined cDNA sequence for clones #18949<br>
and 19024, also referred to as L844P.<br>
SEQ ID NO: 45 is the determined cDNA sequence for clone #18948.<br>
SEQ ID NO: 46 is the determined cDNA sequence for clone #18947,<br>
also referred to as L840P.<br>
SEQ ID NO: 47 is the determined cDNA sequence for clones #18946,<br>
18953,18969 and 19027.<br>
SEQ ID NO: 48 is the determined cDNA sequence for clone #18942.<br>
SEQ ID NO: 49 is the determined cDNA sequence for clone #18940,<br>
18962,18963,19006, 19008,19000, and 19031.<br>
SEQ ID NO: 50 is the determined cDNA sequence for clone #18939.<br>
SEQ ID NO: 51 is the determined cDNA sequence for clones #18938<br>
and 18952.<br>
SEQ ID NO: 52 is the determined cDNA sequence for clone #18938.<br>
SEQ ID NO: 53 is the determined cDNA sequence for clone #18937.<br>
SEQ ID NO: 54 is the determined eDNA sequence for clones #18934,<br>
18935,18993 and 19022, also referred to as L548S.<br>
SEQ ID NO: 55 is the determined cDNA sequence for clone #18932.<br>
SEQ ID NO: 56 is the determined cDNA sequence for clones #18931<br>
and 18936.<br>
SEQ ID NO: 57 is the determined cDNA sequence for clone #18930.<br>
SEQ ID NO: 58 is the determined cDNA sequence for clone #19014,<br>
also referred to as L773P.<br>
SEQ ID NO: 59 is the determined cDNA sequence for clone #19127.<br>
SEQ ID NO: 60 is the determined cDNA sequence for clones #19057<br>
and 19064.<br>
SEQ ID NO: 61 is the determined cDNA sequence for clone #19122.<br>
SEQ ID NO: 62 is the determined cDNA sequence for clones #19120<br>
and 18121.<br>
SEQ ID NO: 63 is the determined cDNA sequence for clone #19118.<br>
SEQ ID NO: 64 is the determined cDNA sequence for clone #19117.<br>
SEQ ID NO: 65 is the determined cDNA sequence for clone #19116.<br>
SEQ ID NO: 66 is the determined cDNA sequence for clone #19114.<br>
SEQ ID NO: 67 is the determined cDNA sequence for clone #19112,<br>
also known as L561S.<br>
SEQ ID NO: 68 is the determined cDNA sequence for clone #19110.<br>
SEQ ID NO: 69 is the determined cDNA sequence for clone #19107,<br>
also referred to as L552S.<br>
SEQ ID NO: 70 is the determined cDNA sequence for clone #19106,<br>
also referred to as L547S.<br>
SEQ ID NO: 71 is the determined cDNA sequence for clones #19105<br>
and 19.111.<br>
SEQ ID NO: 72 is the determined cDNA sequence for clone #19099.<br>
SEQ ID NO: 73 is the determined cDNA sequence for clones #19095,<br>
19104 and 19125, also referred to as L549S.<br>
SEQ ID NO: 74 is the determined cDNA sequence for clone #19094.<br>
SEQ ID NO: 75 is the determined cDNA sequence for clones #19089<br>
and 19101.<br>
SEQ ID NO: 76 is the determined cDNA sequence for clone #19088.<br>
SEQ ID NO: 77 is the determined cDNA sequence for clones #19087,<br>
19092,19096,19100 and 19119.<br>
SEQ ID NO: 78 is the determined cDNA sequence for clone #19086.<br>
SEQ ID NO: 79 is the determined cDNA sequence for clone #19085,<br>
also referred to as L550S.<br>
SEQ ID NO: 80 is the determined cDNA sequence for clone #19084,<br>
also referred to as clone #19079.<br>
SEQ ID NO: 81 is the determined cDNA sequence for clone #19082.<br>
SEQ ID NO: 82 is thedetermined cDNA sequence for clone #19080.<br>
SEQ ID NO: 83 is the determined cDNA sequence for clone #19077.<br>
SEQ ID NO: 84 is the determined cDNA sequence for clone #19076,<br>
also referred to as L55IS.<br>
SEQ ID NO: 85 is the determined cDNA sequence for clone #19074,<br>
also referred to as clone #20102.<br>
SEQ ID NO: 86 is the determined cDNA sequence for clone #19073,<br>
also referred to as L560S.<br>
SEQ ID NO: 87 is the determined cDNA sequence for clones #19072<br>
and 19115.<br>
SEQ ID NO: 88 is the determined cDNA sequence for clone #19071.<br>
SEQ ID NO: 89 is the determined cDNA sequence for clone #19070.<br>
SEQ ID NO: 90 is the determined cDNA sequence for clone #19069.<br>
SEQ ID NO: 91 is the determined cDNA sequence for clone #19068,<br>
also referred to L563S.<br>
SEQ ID NO: 92 is the determined cDNA sequence for clone #19066.<br>
SEQ ID NO: 93 is the determined cDNA sequence for clone #19065.<br>
SEQ ID NO: 94 is the determined cDNA sequence for clone #19063.<br>
SEQ ID NO: 95 is the determined cDNA sequence for clones #19061,<br>
19081,19108 and 19109.<br>
SEQ ID NO: 96 is the determined cDNA sequence for clones #19060,<br>
19067 and 19083, also referred to as L548S.<br>
SEQ ID NO: 97 is the determined cDNA sequence for clones #19059<br>
and 19062.<br>
SEQ ID NO: 98 is the determined cDNA sequence for clone #19058.<br>
SEQ ID NO: 99 is the determined cDNA sequence for clone #19124.<br>
SEQ ID NO: 100 is the determined cDNA sequence for clone #18929.<br>
SEQ ID NO: 101 is the determined cDNA sequence for clone #18422.<br>
SEQ ID NO: 102 is the determined cDNA sequence for clone #18425.<br>
SEQ ID NO: 103 is the determined cDNA sequence for clone #18431.<br>
SEQ ID NO: 104 is the determined cDNA sequence for clone #18433.<br>
SEQ ID NO: 105 is the determined cDNA sequence for clone #18444.<br>
SEQ ID NO: 106 is the determined cDNA sequence for clone # 18449.<br>
SEQ ID NO: 107 is the determined cDNA sequence for clone #18451.<br>
SEQ ID NO: 108 is the determined cDNA sequence for clone #18452.<br>
SEQ ID NO: 109 is the determined cDNA sequence for clone #18455.<br>
SEQ ID NO: 110 is the determined cDNA sequence for clone #18457.<br>
SEQ ED NO: 111 is the determined cDNA sequence for clone #18466.<br>
SEQ ED NO: 112 is the determined cDNA sequence for clone #18468.<br>
SEQ ID NO: 113 is the determined cDNA sequence for clone #18471.<br>
SEQ ID NO: 114 is the determined cDNA sequence for clone #18475.<br>
SEQ ID NO: 115 is the determined cDNA sequence for clone #18476.<br>
SEQ ID NO: 116 is the determined cDNA sequence for clone #18477.<br>
SEQ ID NO: 117 is the determined cDNA sequence for clone #20631.<br>
SEQ ID NO: 118 is the determined cDNA sequence for clone #20634.<br>
SEQ ID NO: 119 is the determined cDNA sequence for clone #20635.<br>
SEQ ID NO: 120 is the determined cDNA sequence for clone #20637.<br>
SEQ ID NO: 121 is the determined cDNA sequence for clone #20638.<br>
SEQ ID NO: 122 is the determined cDNA sequence for clone #20643.<br>
SEQ ID NO: 123 is the determined cDNA sequence for clone #20652.<br>
SEQ ED NO: 124 is the determined cDNA sequence for clone #20653.<br>
SEQ ID NO: 125 is the determined cDNA sequence for clone #20657.<br>
SEQ ID NO: 126 is the determined cDNA sequence for clone #20658.<br>
SEQ ID NO: 127 is the determined cDNA sequence for clone #20660.<br>
SEQ ID NO: 128 is the determined cDNA sequence for done #20661.<br>
SEQ ID NO: 129 is the determined cDNA sequence for clone #20663.<br>
SEQ ID NO; 130 is the determined cDNA sequence for clone #20665.<br>
SEQ ID NO: 131 is the determined cDNA sequence for clone #20670.<br>
SEQ ID NO: 132 is the determined cDNA sequence for clone #20671.<br>
SEQ ID NO: 133 is the determined cDNA sequence for clone #20672.<br>
SEQ ID NO: 134 is the determined cDNA sequence for clone #20675.<br>
SEQ ID NO: 135 is the determined cDNA sequence for clone #20679.<br>
SEQ ID NO: 136 is the determined cDNA sequence for clone #20681.<br>
SEQ ID NO: 137 is the determined cDNA sequence for clone #20682.<br>
SEQ ID NO: 138 is the determined cDNA sequence for clone #20684.<br>
SEQ ID NO: 139 is the determined cDNA sequence for clone #20685.<br>
SEQ ID NO: 140 is the determined cDNA sequence for clone #20689.<br>
SEQ ID NO: 141 is the determined cDNA sequence for clone #20699.<br>
SEQ ID NO: 142 is the determined cDNA sequence for clone #20701.<br>
SEQ ID NO: 143 is the determined cDNA sequence for clone #20702.<br>
SEQ ID NO: 144 is the determined cDNA sequence for clone #20708.<br>
SEQ ID NO: 145 is the determined cDNA sequence for clone #20715.<br>
SEQ ID NO: 146 is the determined cDNA sequence for clone #20716.<br>
SEQ ID NO: 147 is the determined cDNA sequence for clone #20719.<br>
SEQ ID NO: 148 is the determined cDNA sequence for clone #19129.<br>
SEQ ID NO: 149 is the determined cDNA sequence for clone #19131.1.<br>
SEQ ID NO: 150 is the determined cDNA sequence for clone # 19132.2.<br>
SEQ ID NO: 151 is the determined cDNA sequence for clone #19133.<br>
SEQ ID NO: 152 is the determined cDNA sequence for clone #19134.2.<br>
SEQ ID NO: 153 is the determined cDNA sequence for clone #19135.2.<br>
SEQ ID NO: 154 is the determined cDNA sequence for clone #19137.<br>
SEQ ID NO: 155 is a first determined cDNA sequence for clone<br>
#19138.1.<br>
#19138.2.<br>
#19140.1.<br>
#19140.2.<br>
#19145.1.<br>
#19145.2.<br>
#19153.1.<br>
#19153.2.<br>
#19161.1.<br>
#19161.2.<br>
SEQ ID NO: 156 is a second determined cDNA sequence for clone<br>
SEQ ID NO: 157 is the determined cDNA sequence for clone #19139.<br>
SEQ ID NO: 158 is a first determined cDNA sequence for clone<br>
SEQ ID NO: 159 is a second determined cDNA sequence for clone<br>
SEQ ID NO: 160 is the determined cDNA sequence for clone #19141.<br>
SEQ ID NO: 161 is the determined cDNA sequence for clone #19143.<br>
SEQ ID NO: 162 is the determined cDNA sequence for clone #19144.<br>
SEQ ID NO: 163 is a first determined cDNA sequence for clone<br>
SEQ ID NO: 164 is a second determined cDNA sequence for clone<br>
SEQ ID NO: 165 is the determined cDNA sequence for clone #19146.<br>
SEQ ID NO: 166 is the determined cDNA sequence for clone #19149.1.<br>
SEQ ID NO: 167 is the determined cDNA sequence for clone #19152.<br>
SEQ ID NO: 168 is a first determined cDNA sequence for clone<br>
SEQ ID NO: 169 is a second determined cDNA sequence for clone<br>
SEQ ID NO: 170 is the determined cDNA sequence for clone #19155.<br>
SEQ ID NO: 171 is the determined cDNA sequence for clone #19157.<br>
SEQ ID NO: 172 is the determined cDNA sequence for clone #19159.<br>
SEQ ID NO: 173 is the determined cDNA sequence for clone #19160.<br>
SEQ ID NO: 174 is a first detennined cDNA sequence for clone<br>
SEQ ID NO: 175 is a second determined cDNA sequence for clone<br>
SEQ ID NO: 176 is the determined cDNA sequence for clone #19162.1.<br>
SEQ ID NO: 177 is the determined cDNA sequence for clone #19166.<br>
#19173.1.<br>
#19173.2.<br>
#19182.1.<br>
#19182.2.<br>
#19194.1.<br>
#19194.2.<br>
SEQ ID NO: 178 is the determined cDNA sequence for clone #19169.<br>
SEQ ID NO: 179 is the determined cDNA sequence for clone #19171.<br>
SEQ ID NO: 180 is a first determined cDNA sequence for clone<br>
SEQ ID NO: 181 is a second determined cDNA sequence for clone<br>
SEQ ID NO: 182 is the determined cDNA sequence for clone #19174.1.<br>
SEQ ID NO: 183 is the determined cDNA sequence for clone #19175.<br>
SEQ ID NO: 184 is the determined cDNA sequence for clone #19177.<br>
SEQ ID NO: 185 is the determined cDNA sequence for clone #19178.<br>
SEQ ID NO: 186 is the determined cDNA sequence for clone #19179.1.<br>
SEQ ID NO: 187 is the determined cDNA sequence for clone #19179.2.<br>
SEQ ID NO: 188 is the determined cDNA sequence for clone #19180.<br>
SEQ ID NO: 189 is a first determined cDNA sequence for clone<br>
SEQ ID NO: 190 is a second determined cDNA sequence for clone<br>
SEQ ID NO: 191 is the determined cDNA sequence for clone #19183.1.<br>
SEQ ID NO: 192 is the determined cDNA sequence for clone #19185.1.<br>
SEQ ID NO: 193 is the determined cDNA sequence for clone #19187.<br>
SEQ ID NO: 194 is the determined cDNA sequence for clone #19188.<br>
SEQ ID NO: 195 is the determined cDNA sequence for clone #19190,<br>
SEQ ID NO: 196 is the determined cDNA sequence for clone #19191.<br>
SEQ ID NO: 197 is the determined cDNA sequence for clone #19192.<br>
SEQ ID NO: 198 is the determined cDNA sequence for clone #19193.<br>
SEQ ID NO: 199 is a first determined cDNA sequence for clone<br>
SEQ ID NO: 200 is a second determined cDNA sequence for clone<br>
SEQ ID NO: 201 is the determined cDNA sequence for clone #19197.<br>
SEQ ID NO: 202 is a first determined cDNA sequence for clone<br>
#19200.1.<br>
SEQ ID NO: 203 is a second determined cDNA sequence for clone<br>
#19200.2.<br>
SEQ ID NO: 204 is the determined cDNA sequence for clone #19202.<br>
SEQ ID NO: 205 is a first determined cDNA sequence for clone<br>
#19204.1.<br>
SEQ ED NO: 206 is a second determined cDNA sequence for clone<br>
#19204.2.<br>
SEQ ID NO: 207 is the determined cDNA sequence for clone #19205.<br>
SEQ ID NO: 208 is a first determined cDNA sequence for clone<br>
#19206.1.<br>
SEQ ID NO: 209 is a second determined cDNA sequence for clone<br>
#19206.2.<br>
SEQ ID NO: 210 is the determined cDNA sequence for clone #19207.<br>
SEQ ID NO: 211 is the determined cDNA sequence for clone #19208.<br>
SEQ ID NO: 212 is a first determined cDNA sequence for clone<br>
#19211.1.<br>
SEQ ID NO: 213 is a second determined cDNA sequence for clone<br>
#19211.2.<br>
SEQ ID NO: 214 is a first determined cDNA sequence for clone<br>
#19214.1.<br>
SEQ ID NO: 215 is a second determined cDNA sequence for clone<br>
#19214.2.<br>
SEQ ID NO: 216 is the determined cDNA sequence for clone #19215.<br>
SEQ ID NO: 217 is a first determined cDNA sequence for clone # 19217.<br>
2.<br>
SEQ ID NO: 218 is a second determined cDNA sequence for clone<br>
#19217.2.<br>
SEQ ID NO: 219 is a first determined cDNA sequence for clone<br>
#19218.1.<br>
SEQ ED NO: 248 is the determined cDNA sequence for clone #24851.<br>
SEQ ID NO: 249 is the determined cDNA sequence for clone #24852.<br>
SEQ ID NO: 250 is the determined cDNA sequence for clone #24853.<br>
SEQ ID NO: 251 is the determined cDNA sequence for clone #24854.<br>
SEQ ID NO: 252 is the determined cDNA sequence for clone #24855.<br>
SEQ ID NO: 253 is the determined cDNA sequence for clone #24860.<br>
SEQ ID NO: 254 is the determined cDNA sequence for clone #24864.<br>
SEQ ID NO: 255 is the determined cDNA sequence for clone #24866.<br>
SEQ ID NO: 256 is the determined cDNA sequence for clone #24867.<br>
SEQ ID NO: 257 is the determined cDNA sequence for clone #24868.<br>
SEQ ID NO: 258 is the determined cDNA sequence for clone #24869.<br>
SEQ ID NO: 259 is the determined cDNA sequence for clone #24870.<br>
SEQ ID NO: 260 is the determined cDNA sequence for clone #24872.<br>
SEQ ID NO: 261 is the determined cDNA sequence for clone #24873.<br>
SEQ ID NO: 262 is the determined cDNA sequence for clone #24875.<br>
SEQ ID NO: 263 is the determined cDNA sequence for clone #24882.<br>
SEQ ID NO: 264 is the determined cDNA sequence for clone #24885.<br>
SEQ ID NO: 265 is the determined cDNA sequence for clone #24886.<br>
SEQ ID NO: 266 is the determined cDNA sequence for clone #24887.<br>
SEQ ID NO: 267 is the determined cDNA sequence for clone #24888.<br>
SEQ ID NO: 268 is the determined cDNA sequence for clone #24890.<br>
SEQ ID NO: 269 is the determined cDNA sequence for clone #24896.<br>
SEQ ID NO: 270 is the determined cDNA sequence for clone #24897.<br>
SEQ ID NO: 271 is the determined cDNA sequence for clone #24899.<br>
SEQ ID NO: 272 is the determined cDNA sequence for clone #24901.<br>
SEQ ID NO: 273 is the determined cDNA sequence for clone #24902.<br>
SEQ ID NO: 274 is the determined cDNA sequence for clone #24906.<br>
SEQ ID NO: 275 is the determined cDNA sequence for clone #24912.<br>
SEQ ID NO: 276 is the determined cDNA sequence for clone #24913.<br>
SEQ ID NO: 277 is the determined cDNA sequence for clone #24920.<br>
SEQ ID NO: 278 is the determined cDNA sequence for clone #24927.<br>
#19218.2.<br>
#19220.1.<br>
#19220.2.<br>
SEQ ID NO: 220 is a second determined cDNA sequence for clone<br>
SEQ ID NO: 221 is a first determined cDNA sequence for clone<br>
SEQ ID NO: 222 is a second determined cDNA sequence for clone<br>
SEQ ID NO: 223 is the determined cDNA sequence for clone #22015.<br>
SEQ ID NO: 224 is the determined cDNA sequence for clone #22017.<br>
SEQ ID NO: 225 is the determined cDNA sequence for clone #22019.<br>
SEQ ID NO: 226 is the determined cDNA sequence for clone #22020.<br>
SEQ ID NO: 227 is the determined cDNA sequence for clone #22023.<br>
SEQ ID NO: 228 is the determined cDNA sequence for clone #22026.<br>
SEQ ID NO: 229 is the determined cDNA sequence for clone #22027.<br>
SEQ ID NO: 230 is the determined cDNA sequence for clone #22028.<br>
SEQ ID NO: 231 is the determined cDNA sequence for clone #22032.<br>
SEQ ID NO: 232 is the determined cDNA sequence for clone #22037.<br>
SEQ ID NO: 233 is the determined cDNA sequence for clone #22045.<br>
SEQ ID NO: 234 is the determined cDNA sequence for clone #22048.<br>
SEQ ID NO: 235 is the determined cDNA sequence for clone #22050.<br>
SEQ ID NO: 236 is the determined cDNA sequence for clone #22052.<br>
SEQ ID NO: 237 is the determined cDNA sequence for clone #22053.<br>
SEQ ID NO: 238 is the determined cDNA sequence for clone #22057.<br>
SEQ ID NO: 239 is the determined cDNA sequence for clone #22066.<br>
SEQ ID NO: 240 is the determined cDNA sequence for clone #22077.<br>
SEQ ID NO: 241 is the determined cDNA sequence for clone #22085.<br>
SEQ ID NO: 242 is the determined cDNA sequence for clone #22105.<br>
SEQ ID NO: 243 is the determined cDNA sequence for clone #22108.<br>
SEQ ID NO: 244 is the determined cDNA sequence for clone #22109.<br>
SEQ ID NO: 245 is the determined cDNA sequence for clone #24842.<br>
SEQ ID NO: 246 is the determined cDNA sequence for clone #24843.<br>
SEQ ID NO: 247 is the determined cDNA sequence for clone #24845.<br>
SEQ ID NO: 279 is the determined cDNA sequence for clone #24930.<br>
SEQ ID NO: 280 is the determined cDNA sequence for clone #26938.<br>
SEQ ID NO: 281 is the determined cDNA sequence for clone #26939.<br>
SEQ ID NO: 282 is the determined cDNA sequence for clone #26943.<br>
SEQ ID NO: 283 is the determined cDNA sequence for clone #26948.<br>
SEQ ID NO: 284 is the determined cDNA sequence for clone #26951.<br>
SEQ ID NO: 285 is the determined cDNA sequence for clone #26955.<br>
SEQ ID NO: 286 is the determined cDNA sequence for clone #26956.<br>
SEQ ID NO: 287 is the determined cDNA sequence for clone #26959.<br>
SEQ ID NO: 288 is the determined cDNA sequence for clone #26961.<br>
SEQ ID NO: 289 is the determined cDNA sequence for clone #26962.<br>
SEQ ID NO: 290 is the determined cDNA sequence for clone #26964.<br>
SEQ ID NO: 291 is the determined cDNA sequence for clone #26966.<br>
SEQ ID NO: 292 is the determined cDNA sequence for clone #26968.<br>
SEQ ID NO: 293 is the determined cDNA sequence for clone #26972.<br>
SEQ ID NO: 294 is the determined cDNA sequence for clone #26973.<br>
SEQ ID NO: 295 is the determined cDNA sequence for clone #26974.<br>
SEQ ID NO: 296 is the determined cDNA sequence for clone #26976.<br>
SEQ ID NO: 297 is the determined cDNA sequence for clone #26977.<br>
SEQ ID NO: 298 is the determined cDNA sequence for clone #26979.<br>
SEQ ID NO: 299 is the determined cDNA sequence for clone #26980.<br>
SEQ ID NO: 300 is the determined cDNA sequence for clone #26981.<br>
SEQ ID NO: 301 is the determined cDNA sequence for clone #26984.<br>
SEQ ID NO: 302 is the determined cDNA sequence for clone #26985.<br>
SEQ ID NO: 303 is the determined cDNA sequence for clone #26986.<br>
SEQ ID NO: 304 is the determined cDNA sequence for clone #26993.<br>
SEQ ID NO: 305 is the determined cDNA sequence for clone #26994.<br>
SEQ ID NO: 306 is the determined cDNA sequence for clone #26995.<br>
SEQ ID NO: 307 is the determined cDNA sequence for clone #27003.<br>
SEQ ID NO: 308 is the determined cDNA sequence for clone #27005.<br>
SEQ ID NO: 309 is the determined cDNA sequence for clone #27010.<br>
SEQ ID NO: 310 is the determined cDNA sequence for clone #27011.<br>
SEQ ID NO: 311 is the determined cDNA sequence for clone #27013.<br>
SEQ ID NO: 312 is the determined cDNA sequence for clone #27016<br>
SEQ ID NO: 313 is the determined cDNA sequence for clone #27017.<br>
SEQ ID NO: 314 is the determined cDNA sequence for clone #27019.<br>
SEQ ID NO: 315 is the determined cDNA sequence for clone #27028.<br>
SEQ ID NO: 316 is the full-length cDNA sequence for clone #19060.<br>
SEQ ID NO: 317 is the full-length cDNA sequence for clone #18964.<br>
SEQ ID NO: 318 is the full-length cDNA sequence for clone #18929.<br>
SEQ ID NO: 319 is the full-length cDNA sequence for clone #18991.<br>
SEQ ID NO: 320 is the full-length cDNA sequence for clone #18996.<br>
SEQ ID NO: 321 is the full-length cDNA sequence for clone # 18966.<br>
SEQ ID NO: 322 is the full-length cDNA sequence for clone #18951.<br>
SEQ ID NO: 323 is the full-length cDNA sequence for clone #18973<br>
(also known as L516S).<br>
SEQ ID NO: 324 is the amino acid sequence for clone #19060.<br>
SEQ ID NO: 325 is the amino acid sequence for clone #19063.<br>
SEQ ID NO: 326 is the amino acid sequence for clone #19077.<br>
SEQ ID NO: 327 is the amino acid sequence for clone #19110.<br>
SEQ ID NO: 328 is the amino acid sequence for clone #19122.<br>
SEQ ID NO: 329 is the amino acid sequence for clone #19118.<br>
SEQ ID NO: 330 is the amino acid sequence for clone #19080.<br>
SEQ ID NO: 331 is the ammo acid sequence for clone #19127.<br>
SEQ ID NO: 332 is the amino acid sequence for clone #19117.<br>
SEQ ID NO: 333 is the amino acid sequence for clone #19095, also<br>
referred to L549S.<br>
SEQ ID NO: 334 is the amino acid sequence for clone #18964.<br>
SEQ ID NO: 335 is the amino acid sequence for clone #18929.<br>
SEQ ID NO: 336 is the amino acid sequence for clone #18991.<br>
SEQ ID NO: 337 is the amino acid sequence for clone #18996.<br>
SEQ ID NO: 338 is the amino acid sequence for clone #18966.<br>
SEQ ID NO: 339 is the ammo acid sequence for clone #18951.<br>
SEQ ID NO: 340 is the amino acid sequence for clone #18973.<br>
SEQ ID NO: 341 is the determined cDNA sequence for clone 26461.<br>
SEQ ID NO: 342 is the determined cDNA sequence for clone 26462.<br>
SEQ ID NO: 343 is the determined cDNA sequence for clone 26463.<br>
SEQ ID NO: 344 is the determined cDNA sequence for clone 26464.<br>
SEQ ID NO: 345 is the determined cDNA sequence for clone 26465.<br>
SEQ ID NO: 346 is the determined cDNA sequence for clone 26466.<br>
SEQ ID NO: 347 is the determined cDNA sequence for clone 26467.<br>
SEQ ID NO: 348 is the determined cDNA sequence for clone 26468.<br>
SEQ ID NO: 349 is the determined cDNA sequence for clone 26469.<br>
SEQ ID NO: 350 is the determined cDNA sequence for clone 26470.<br>
SEQ ID NO: 351 is the determined cDNA sequence for clone 26471.<br>
SEQ ID NO: 352 is the determined cDNA sequence for clone 26472.<br>
SEQ ID NO: 353 is the determined cDNA sequence for clone 26474.<br>
SEQ ID NO: 354 is the determined cDNA sequence for clone 26475.<br>
SEQ ID NO: 355 is the determined cDNA sequence for clone 26476.<br>
SEQ ID NO: 356 is the determined cDNA sequence for clone 26477.<br>
SEQ ID NO: 357 is the determined cDNA sequence for clone 26478.<br>
SEQ ID NO: 358 is the determined cDNA sequence for clone 26479.<br>
SEQ ID NO: 359 is the determined cDNA sequence for clone 26480.<br>
SEQ ID NO: 360 is the determined cDNA sequence for clone 26481.<br>
SEQ ID NO: 361 is the determined cDNA sequence for clone 26482<br>
SEQ ID NO: 362 is the determined cDNA sequence for clone 26483.<br>
SEQ ID NO: 363 is the determined cDNA sequence for clone 26484.<br>
SEQ ID NO: 364 is the determined cDNA sequence for clone 26485.<br>
SEQ ID NO: 365 is the determined cDNA sequence for clone 26486.<br>
SEQ ID NO: 366 is the determined cDNA sequence for clone 26487.<br>
SEQ ID NO: 367 is the determined cDNA sequence for clone 26488.<br>
SEQ ID NO: 368 is the determined cDNA sequence for clone 26489.<br>
SEQ ID NO: 369 is the determined cDNA sequence for clone 26490.<br>
SEQ ID NO: 370 is the detennined cDNA sequence for clone 26491.<br>
SEQ ID NO: 371 is the determined cDNA sequence for done 26492.<br>
SEQ ID NO: 372 is the determined cDNA sequence for clone 26493.<br>
SEQ ID NO: 373 is the detennined cDNA sequence for clone 26494.<br>
SEQ ID NO: 374 is the determined cDNA sequence for clone 26495.<br>
SEQ ID NO: 375 is the determined cDNA sequence for clone 26496.<br>
SEQ ID NO: 376 is the determined cDNA sequence for clone 26497.<br>
SEQ ID NO: 377 is the determined cDNA sequence for clone 26498.<br>
SEQ ID NO: 378 is the determined cDNA sequence for clone 26499.<br>
SEQ ID NO: 379 is the determined cDNA sequence for clone 26500.<br>
SEQ ID NO: 380 is the determined cDNA sequence for clone 26501.<br>
SEQ ID NO: 381 is the determined cDNA sequence for clone 26502.<br>
SEQ ID NO: 382 is the determined cDNA sequence for clone 26503.<br>
SEQ ID NO: 383 is the detennined cDNA sequence for clone 26504.<br>
SEQ ID NO: 384 is the detennined cDNA sequence for clone 26505.<br>
SEQ ID NO: 385 is the determined cDNA sequence for clone 26506.<br>
SEQ ID NO: 386 is the determined cDNA sequence for clone 26507.<br>
SEQ ID NO: 387 is the determined cDNA sequence for clone 26508.<br>
SEQ ID NO: 388 is the determined cDNA sequence for clone 26509.<br>
SEQ ID NO: 389 is the determined cDNA sequence for clone 26511.<br>
SEQ ID NO: 390 is the detennined cDNA sequence for clone 26513.<br>
SEQ ID NO: 391 is the determined cDNA sequence for clone 26514.<br>
SEQ ID NO: 392 is the determined cDNA sequence for clone 26515.<br>
SEQ ID NO: 393 is the determined cDNA sequence for clone 26516.<br>
SEQ ID NO: 394 is the determined cDNA sequence for clone 26517.<br>
SEQ ID NO: 395 is the determined cDNA sequence for clone 26518.<br>
SEQ ID NO: 396 is the determined cDNA sequence for clone 26519.<br>
SEQ ID NO: 397 is the determined cDNA sequence for clone 26520.<br>
SEQ ID NO: 398 is the determined cDNA sequence for clone 26521.<br>
SEQ ID NO: 399 is the determined cDNA sequence for clone 26522.<br>
SEQ ID NO: 400 is the detennined cDNA sequence for clone 26523.<br>
SEQ ID NO: 401 is the determined cDNA sequence for clone 26524.<br>
SEQ ID NO: 402 is the determined cDNA sequence for done 26526.<br>
SEQ ID NO: 403 is the determined cDNA sequence for clone 26527.<br>
SEQ ID NO: 404 is the determined cDNA sequence for clone 26528.<br>
SEQ ID NO: 405 is the determined cDNA sequence for clone 26529.<br>
SEQ ID NO: 406 is the determined cDNA sequence for clone 26530.<br>
SEQ ID NO: 407 is the determined cDNA sequence for clone 26532.<br>
SEQ ID NO: 408 is the determined cDNA sequence for clone 26533.<br>
SEQ ID NO: 409 is the determined cDNA sequence for clone 26534.<br>
SEQ ID NO: 410 is the determined cDNA sequence for clone 26535.<br>
SEQ ID NO: 411 is the determined cDNA sequence for clone 26536.<br>
SEQ ID NO: 412 is the determined cDNA sequence for clone 26537.<br>
SEQ ID NO: 413 is the determined cDNA sequence for clone 26538.<br>
SEQ ID NO: 414 is the determined cDNA sequence for clone 26540.<br>
SEQ ID NO: 415 is the determined cDNA sequence for clone 26541.<br>
SEQ ID NO: 416 is the determined cDNA sequence for clone 26542.<br>
SEQ ID NO: 417 is the determined cDNA sequence for clone 26543.<br>
SEQ ID NO: 418 is the determined cDNA sequence for clone 26544.<br>
SEQ ID NO: 419 is the determined cDNA sequence for clone 26546.<br>
SEQ ID NO: 420 is the determined cDNA sequence for clone 26547.<br>
SEQ ID NO: 421 is the determined cDNA sequence for clone 26548.<br>
SEQ ID NO: 422 is the determined cDNA sequence for clone 26549.<br>
SEQ ID NO: 423 is the determined cDNA sequence for clone 26550.<br>
SEQ ID NO: 424 is the determined cDNA sequence for clone 26551.<br>
SEQ ID NO: 425 is the determined cDNA sequence for clone 26552.<br>
SEQ ID NO: 426 is the determined cDNA sequence for clone 26553.<br>
SEQ ID NO: 427 is the determined cDNA sequence for clone 26554.<br>
SEQ ID NO: 428 is the determined cDNA sequence for clone 26556.<br>
SEQ ID NO: 429 is the determined cDNA sequence for clone 26557.<br>
SEQ ID NO: 430 is the determined cDNA sequence for clone 27631.<br>
SEQ ID NO: 431 is the determined cDNA sequence for clone 27632.<br>
SEQ ID NO: 432 is the determined cDNA sequence for clone 27633.<br>
SEQ ID NO: 433 is the determined cDNA sequence for clone 27635.<br>
SEQ ID NO: 434 is the determined cDNA sequence for clone 27636.<br>
SEQ ID NO: 435 is the determined cDNA sequence for clone 27637.<br>
SEQ ID NO: 436 is the determined cDNA sequence for clone 27638.<br>
SEQ ID NO: 437 is the determined cDNA sequence for clone 27639.<br>
SEQ ID NO: 438 is the determined cDNA sequence for clone 27640.<br>
SEQ ID NO: 439 is the determined cDNA sequence for clone 27641.<br>
SEQ ID NO: 440 is the determined cDNA sequence for clone 27642.<br>
SEQ ID NO: 441 is the determined cDNA sequence for clone 27644.<br>
SEQ ID NO: 442 is the determined cDNA sequence for clone 27646.<br>
SEQ ID NO: 443 is the determined cDNA sequence for clone 27647.<br>
SEQ ID NO: 444 is the determined cDNA sequence for clone 27649.<br>
SEQ ID NO: 445 is the determined cDNA sequence for clone 27650.<br>
SEQ ID NO: 446 is the determined cDNA sequence for clone 27651.<br>
SEQ ID NO: 447 is the determined cDNA sequence for clone 27652.<br>
SEQ ID NO: 448 is the determined cDNA sequence for clone 27654.<br>
SEQ ID NO: 449 is the determined cDNA sequence for clone 27655.<br>
SEQ ID NO: 450 is the determined cDNA sequence for clone 27657.<br>
SEQ ID NO: 451 is the determined cDNA sequence for clone 27659.<br>
SEQ ID NO: 452 is the determined cDNA sequence for clone 27665.<br>
SEQ ID NO: 453 is the determined cDNA sequence for clone 27666.<br>
SEQ ID NO: 454 is the determined cDNA sequence for clone 27668.<br>
SEQ ID NO: 455 is the determined cDNA sequence for clone 27670.<br>
SEQ ID NO: 456 is the determined cDNA sequence for clone 27671.<br>
SEQ ID NO: 457 is the determined cDNA sequence for clone 27672.<br>
SEQ ID NO: 458 is the determined cDNA sequence for clone 27674.<br>
SEQ ID NO: 459 is the determined cDNA sequence for clone 27677.<br>
SEQ ID NO: 4601s the determined cDNA sequence for clone 27681.<br>
SEQ ID NO: 461 is the determined cDNA sequence for clone 27682.<br>
SEQ ID NO: 462 is the determined cDNA sequence for clone 27683.<br>
SEQ ID NO: 463 is the determined cDNA sequence for clone 27686.<br>
SEQ ID NO: 464 is the determined cDNA sequence for clone 27688.<br>
SEQ ID NO: 465 is the determined cDNA sequence for clone 27689.<br>
SEQ ID NO: 466 is the determined cDNA sequence for clone 27690.<br>
SEQ ID NO: 467 is the determined cDNA sequence for clone 27693.<br>
SEQ ID NO: 468 is the determined cDNA sequence for clone 27699.<br>
SEQ ID NO: 469 is the determined cDNA sequence for clone 27700.<br>
SEQ ID NO: 470 is the determined cDNA sequence for clone 27702.<br>
SEQ ID NO: 471 is the determined cDNA sequence for clone 27705.<br>
SEQ ID NO: 472 is the determined cDNA sequence for clone 27706.<br>
SEQ ID NO: 473 is the determined cDNA sequence for clone 27707.<br>
SEQ ID NO: 474 is the determined cDNA sequence for clone 27708.<br>
SEQ ID NO: 475 is the determined cDNA sequence for clone 27709.<br>
SEQ ID NO: 476 is the determined cDNA sequence for clone 27710.<br>
SEQ ID NO: 477 is the determined cDNA sequence for clone 27711.<br>
SEQ ID NO: 478 is the determined cDNA sequence for clone 27712.<br>
SEQ ID NO: 479 is the determined cDNA sequence for clone 27713-<br>
SEQ ID NO: 480 is the determined cDNA sequence for clone 27714.<br>
SEQ ID NO: 481 is the determined cDNA sequence for clone 27715.<br>
SEQ ID NO: 482 is the determined cDNA sequence for clone 27716.<br>
SEQ ID NO: 483 is the determined cDNA sequence for clone 27717.<br>
SEQ ED NO: 484 is the determined cDNA sequence for clone 27718.<br>
SEQ ID NO: 485 is the determined cDNA sequence for clone 27719.<br>
SEQ ID NO: 486 is the determined cDNA sequence for clone 27720.<br>
SEQ ID NO: 487 is the determined cDNA sequence for clone 27722.<br>
SEQ ED NO: 488 is the determined cDNA sequence for clone 27723.<br>
SEQ ID NO: 489 is the determined cDNA sequence for clone 27724.<br>
SEQ ID NO: 490 is the determined cDNA sequence for clone 27726.<br>
SEQ ID NO: 491 is the determined cDNA sequence for clone 25015.<br>
SEQ ID NO: 492 is the determined cDNA sequence for clone 25016.<br>
SEQ ID NO: 493 is the determined cDNA sequence for clone 25017.<br>
SEQ ID NO: 494 is the determined cDNA sequence for clone 25018<br>
SEQ ID NO: 495 is the determined cDNA sequence for done 25030.<br>
SEQ ID NO: 496 is the determined cDNA sequence for clone 25033.<br>
SEQ ID NO: 497 is the determined cDNA sequence for clone 25034.<br>
SEQ ID NO: 498 is the determined cDNA sequence for clone 25035.<br>
SEQ ID NO: 499 is the determined cDNA sequence for clone 25036.<br>
SEQ ID NO: 500 is the determined cDNA sequence for clone 25037.<br>
SEQ ID NO: 501 is the determined cDNA sequence for clone 25038.<br>
SEQ ID NO: 502 is the determined cDNA sequence for clone 25039.<br>
SEQ ID NO: 503 is the determined cDNA sequence for clone 25040.<br>
SEQ ID NO: 504 is the determined cDNA sequence for clone 25042.<br>
SEQ ID NO: 505 is the determined cDNA sequence for clone 25043.<br>
SEQ ID NO: 506 is the determined cDNA sequence for clone 25044.<br>
SEQ ID NO: 507 is the determined cDNA sequence for clone 25045.<br>
SEQ ID NO: 508 is the determined cDNA sequence for clone 25047.<br>
SEQ ID NO: 509 is the determined cDNA sequence for clone 25048.<br>
SEQ ID NO: 510 is the determined cDNA sequence for clone 25049.<br>
SEQ ID NO: 511 is the determined cDNA sequence for clone 25185.<br>
SEQ ID NO: 512 is the determined cDNA sequence for clone 25186.<br>
SEQ ID NO: 513 is the determined cDNA sequence for clone 25187.<br>
SEQ ID NO: 514 is the determined cDNA sequence for clone 25188.<br>
SEQ ID NO: 515 is the determined cDNA sequence for clone 25189.<br>
SEQ ID NO: 516 is the determined cDNA sequence for clone 25190.<br>
SEQ ID NO: 517 is the determined cDNA sequence for clone 25193.<br>
SEQ ID NO: 518 is the determined cDNA sequence for clone 25194.<br>
SEQ ID NO: 519 is the determined cDNA sequence for clone 25196.<br>
SEQ ID NO: 520 is the determined cDNA sequence for clone 25198.<br>
SEQ ID NO: 521 is the determined cDNA sequence for clone 25199.<br>
SEQ ID NO: 522 is the determined cDNA sequence for clone 25200.<br>
SEQ ID NO: 523 is the determined cDNA sequence for clone 25202.<br>
SEQ ID NO: 524 is the determined cDNA sequence for clone 25364.<br>
SEQ ID NO: 525 is the determined cDNA sequence for clone 25366.<br>
SEQ ID NO: 526 is the determined cDNA sequence for done 25367.<br>
SEQ ID NO: 527 is the determined cDNA sequence for clone 25368.<br>
SEQ ID NO: 528 is the determined cDNA sequence for clone 25369.<br>
SEQ ID NO: 529 is the determined cDNA sequence for clone 25370.<br>
SEQ ID NO: 530 is the determined cDNA sequence for clone 25371.<br>
SEQ ID NO: 531 is the determined cDNA sequence for clone 25372.<br>
SEQ ID NO: 532 is the determined cDNA sequence for clone 25373.<br>
SEQ ID NO: 533 is the determined cDNA sequence-for clone 25374.<br>
SEQ ID NO: 534 is the determined cDNA sequence for clone 25376.<br>
SEQ ID NO: 535 is the determined cDNA sequence for clone 25377.<br>
SEQ ID NO: 536 is the determined cDNA sequence for clone 25378.<br>
SEQ ID NO: 537 is the determined cDNA sequence for clone 25379.<br>
SEQ ID NO: 538 is the determined cDNA sequence for clone 25380.<br>
SEQ ID NO: 539 is the determined cDNA sequence for clone 25381.<br>
SEQ ID NO: 540 is the determined cDNA sequence for clone 25382.<br>
SEQ ID NO: 541 is the determined cDNA sequence for clone 25383.<br>
SEQ ID NO: 542 is the determined cDNA sequence for clone 25385.<br>
SEQ ID NO: 543 is the determined cDNA sequence for clone 25386.<br>
SEQ ID NO: 544 is the determined cDNA sequence for clone 25387.<br>
SEQ ID NO: 545 is the determined cDNA sequence for clone 26013.<br>
SEQ ID NO: 546 is the determined cDNA sequence for clone 26014.<br>
SEQ ID NO: 547 is the determined cDNA sequence for clone 26016.<br>
SEQ ID NO: 548 is the determined cDNA sequence for clone 26017.<br>
SEQ ID NO: 549 is the determined cDNA sequence for clone 26018.<br>
SEQ ID NO: 550 is the determined cDNA sequence for clone 26019.<br>
SEQ ID NO: 551 is the determined cDNA sequence for clone 26020.<br>
SEQ ID NO: 552 is the determined cDNA sequence for clone 26021.<br>
SEQ ID NO: 553 is the determined cDNA sequence for clone 26022.<br>
SEQ ID NO: 554 is the determined cDNA sequence for clone 26027.<br>
SEQ ID NO: 555 is the determined cDNA sequence for clone 26197.<br>
SEQ ID NO: 556 is the determined cDNA sequence for clone 26199.<br>
SEQ ID NO: 557 is the determined cDNA sequence for clone 26201.<br>
SEQ ID NO: 558 is the determined cDNA sequence for clone 26202.<br>
SEQ ID NO: 559 is the determined cDNA sequence for clone 26203.<br>
SEQ ID NO: 560 is the detennined cDNA sequence for clone 26204.<br>
.SEQ ID NO: 561 is the determined cDNA sequence for clone 26205.<br>
SEQ ID NO: 562 is the determined cDNA sequence for clone 26206.<br>
SEQ ID NO: 563 is the detennined cDNA sequence for clone 26208.<br>
SEQ ID NO: 564 is the determined cDNA sequence for clone 26211.<br>
SEQ ID NO: 565 is the determined cDNA sequence for clone 26212.<br>
; SEQ ID NO: 566 is the determined cDNA sequence for clone 26213.<br>
SEQ ID NO: 567 is the determined cDNA sequence for clone 26214.<br>
SEQ ID NO: 568 is the determined cDNA sequence for clone 26215.<br>
SEQ ID NO: 569 is the determined cDNA sequence for clone 26216.<br>
SEQ ID NO: 570 is the detennined cDNA sequence for clone 26217.<br>
SEQ ID NO: 571 is the determined cDNA sequence for clone 26218.<br>
SEQ ID NO: 572 is the determined cDNA sequence for clone 26219.<br>
SEQ ID NO: 573 is the determined cDNA sequence for clone 26220.<br>
SEQ ID NO: 574 is the determined cDNA sequence for clone 26221.<br>
SEQ ID NO: 575 is the determined cDNA sequence for clone 26224.<br>
SEQ ID NO: 576 is the determined cDNA sequence for clone 26225.<br>
SEQ ID NO: 577 is the detennined cDNA sequence for clone 26226.<br>
SEQ ID NO: 578 is the detennined cDNA sequence for clone 26227.<br>
SEQ ID NO: 579 is the detennined cDNA sequence for clone 26228.<br>
SEQ ID NO: 580 is the detennined cDNA sequence for clone 26230.<br>
SEQ ID NO: 581 is the detennined cDNA sequence for clone 26231.<br>
SEQ ID NO: 582 is the detennined cDNA sequence for clone 26234.<br>
SEQ ID NO: 583 is the determined cDNA sequence for clone 26236.<br>
SEQ ID NO: 584 is the detennined cDNA sequence for clone 26237.<br>
SEQ ID NO: 585 is the determined cDNA sequence for clone 26239.<br>
SEQ ID NO: 586 is the determined cDNA sequence for clone 26240.<br>
SEQ ID NO: 587 is the determined cDNA sequence for clone 26241.<br>
SEQ ID NO: 588 is the determined cDNA sequence for clone 26242.<br>
SEQ ID NO: 589 is the determined cDNA sequence for clone 26246.<br>
SEQ ID NO: 590 is the determined cDNA sequence for clone 26247.<br>
SEQ ID NO: 591 is the determined cDNA sequence for clone 26248.<br>
SEQ ID NO: 592 is the determined cDNA sequence for clone 26249.<br>
SEQ ID NO: 593 is the determined cDNA sequence for clone 26250.<br>
SEQ ID NO: 594 is the determined cDNA sequence for clone 26251.<br>
SEQ ID NO: 595 is the determined cDNA sequence for clone 26252.<br>
SEQ ID NO: 596 is the determined cDNA sequence for clone 26253.<br>
SEQ ID NO: 597 is the determined cDNA sequence for clone 26254.<br>
SEQ ID NO: 598 is the determined cDNA sequence for clone 26255.<br>
SEQ ID NO: 599 is the determined cDNA sequence for clone 26256.<br>
SEQ ID NO: 600 is the determined cDNA sequence for clone 26257.<br>
SEQ ID NO: 601 is the determined cDNA sequence for clone 26259.<br>
SEQ ID NO: 602 is the determined cDNA sequence for clone 26260.<br>
SEQ ID NO: 603 is the determined cDNA sequence for clone 26261.<br>
SEQ ID NO: 604 is the determined cDNA sequence for clone 26262.<br>
SEQ ID NO: 605 is the determined cDNA sequence for clone 26263.<br>
SEQ ID NO: 606 is the determined cDNA sequence for clone 26264.<br>
SEQ ID NO: 607 is the determined cDNA sequence for clone 26265.<br>
SEQ ID NO: 608 is the determined cDNA sequence for clone 26266.<br>
SEQ ID NO: 609 is the determined cDNA sequence for clone 26268.<br>
SEQ ID NO: 610 is the determined cDNA sequence for clone 26269.<br>
SEQ ID NO: 611 is the determined cDNA sequence for clone 26271.<br>
SEQ ID NO: 612 is the determined cDNA sequence for clone 26273.<br>
SEQ ID NO: 613 is the determined cDNA sequence for clone 26810.<br>
SEQ ID NO: 614 is the determined cDNA sequence for clone 26811.<br>
SEQ ID NO: 615 is the determined cDNA sequence for clone 26812.1.<br>
SEQ ID NO: 616 is the determined cDNA sequence for clone 26812.2.<br>
SEQ ID NO: 617 is the determined cDNA sequence for clone 26813.<br>
SEQ ID NO: 618 is the determined cDNA sequence for clone 26814.<br>
SEQ ID NO: 619 is the determined cDNA sequence for done 26815.<br>
SEQ ID NO: 620 is the determined cDNA sequence for clone 26816.<br>
SEQ ID NO: 621 is the detennined cDNA sequence for clone 26818.<br>
SEQ ID NO: 622 is the determined cDNA sequence for clone 26819.<br>
SEQ ID NO: 623 is the determined cDNA sequence for clone 26820.<br>
SEQ ID NO: 624 is the detennined cDNA sequence for clone 26821.<br>
SEQ ID NO: 625 is the determined cDNA sequence for clone 26822.<br>
SEQ ID NO: 626 is the determined cDNA sequence for clone 26824.<br>
SEQ ID NO: 627 is the determined cDNA sequence for clone.26825.<br>
SEQ ID NO: 628 is the determined cDNA sequence for clone 26826.<br>
SEQ ID NO: 629 is the determined cDNA sequence for clone 26827.<br>
SEQ ID NO: 630 is the determined cDNA sequence for clone 26829.<br>
SEQ ED NO: 631 is the determined cDNA sequence for clone 26830.<br>
SEQ ID NO: 632 is the determined cDNA sequence for clone 26831.<br>
SEQ ID NO: 633 is the detennined cDNA sequence for clone 26832.<br>
SEQ ID NO: 634 is the determined cDNA sequence for clone 26835.<br>
SEQ ID NO: 635 is the detennined cDNA sequence for clone 26836.<br>
SEQ ID NO: 636 is the determined cDNA sequence for clone 26837.<br>
SEQ ID NO: 637 is the determined cDNA sequence for clone 26839.<br>
SEQ ID NO: 638 is the detennined cDNA sequence for clone 26841.<br>
SEQ ID NO: 639 is the determined cDNA sequence for clone 26843.<br>
SEQ ID NO: 640 is the detennined cDNA sequence for clone 26844.<br>
SEQ ID NO: 641 is the determined cDNA sequence for clone 26845.<br>
SEQ ID NO: 642 is the determined cDNA sequence for clone 26846.<br>
SEQ ID NO: 643 is the determined cDNA sequence for clone 26847.<br>
SEQ ID NO: 644 is the determined cDNA sequence for clone 26848.<br>
SEQ ID NO: 645 is the determined cDNA sequence for clone 26849.<br>
SEQ ID NO: 646 is the detennined cDNA sequence for clone 26850.<br>
SEQ ID NO: 647 is the determined cDNA sequence for clone 26851.<br>
SEQ ID NO: 648 is the determined cDNA sequence for clone 26852.<br>
SEQ ID NO: 649 is the determined cDNA sequence for clone 26853.<br>
SEQ ID NO: 650 is the determined cDNA sequence for clone 26854.<br>
SEQ ID NO: 651 is the determined cDNA sequence for clone 26856.<br>
SEQ ID NO: 652 is the determined cDNA sequence for clone 26857.<br>
SEQ ID NO: 653 is the determined cDNA sequence for clone 26858.<br>
SEQ ID NO: 654 is the determined cDNA sequence for clone 26859.<br>
SEQ ID NO: 655 is the determined cDNA sequence for clone 26860.<br>
SEQ ID NO: 656 is the determined cDNA sequence for clone 26862.<br>
SEQ ID NO: 657 is the determined cDNA sequence for clone 26863.<br>
SEQ ID NO: 658 is the determined cDNA sequence for clone 26864.<br>
SEQ ID NO: 659 is the determined cDNA sequence for clone 26865.<br>
SEQ ID NO: 660 is the determined cDNA sequence for clone 26867.<br>
SEQ ID NO: 661 is the determined cDNA sequence for clone 26868.<br>
SEQ ID NO: 662 is the determined cDNA sequence for clone 26871.<br>
SEQ ID NO: 663 is the determined cDNA sequence for clone 26873.<br>
SEQ ID NO: 664 is the determined cDNA sequence for clone 26875.<br>
SEQ ID NO: 665 is the determined cDNA sequence for clone 26876.<br>
SEQ ID NO: 666 is the determined cDNA sequence for clone 26877.<br>
SEQ ID NO: 667 is the determined cDNA sequence for clone 26878.<br>
SEQ ID NO: 668 is the determined cDNA sequence for clone 26880.<br>
SEQ ID NO: 669 is the determined cDNA sequence for clone 26882.<br>
SEQ ID NO: 670 is the determined cDNA sequence for clone 26883.<br>
SEQ ID NO: 671 is the determined cDNA sequence for clone 26884.<br>
SEQ ID NO: 672 is the determined cDNA sequence for clone 26885.<br>
SEQ ID NO: 673 is the determined cDNA sequence for clone 26886.<br>
SEQ ID NO: 674 is the determined cDNA sequence for clone 26887.<br>
SEQ ID NO: 675 is the determined cDNA sequence for clone 26888.<br>
SEQ ID NO: 676 is the determined cDNA sequence for clone 26889.<br>
SEQ ID NO: 677 is the determined cDNA sequence for clone 26890.<br>
SEQ ID NO: 678 is the determined cDNA sequence for clone 26892.<br>
SEQ ID NO: 679 is the determined cDNA sequence for clone 26894.<br>
SEQ ID NO: 680 is the determined cDNA sequence for clone 26895.<br>
SEQ ID NO: 681 is the determined cDNA sequence for done 26897.<br>
SEQ ID NO: 682 is the determined cDNA sequence for clone 26898.<br>
SEQ ID NO: 683 is the determined cDNA sequence for clone 26899.<br>
SEQ ID NO: 684 is the determined cDNA sequence for clone 26900.<br>
SEQ ID NO: 685 is the determined cDNA sequence for clone 26901.<br>
SEQ ID NO: 686 is the determined cDNA sequence for clone 26903.<br>
SEQ ID NO: 687 is the determined cDNA sequence for clone 26905.<br>
SEQ ID NO: 688 is the detennined cDNA sequence for clone 26906.<br>
SEQ ID NO: 689 is the determined cDNA sequence for clone 26708.<br>
SEQ ID NO: 690 is the determined cDNA sequence for clone 26709.<br>
SEQ ID NO: 691 is the determined cDNA sequence for clone 26710.<br>
SEQ ID NO: 692 is the determined cDNA sequence for clone 26711.<br>
SEQ ID NO: 693 is the determined cDNA sequence for clone 26712.<br>
SEQ ID NO: 694 is the determined cDNA sequence for clone 26713.<br>
SEQ ID NO: 695 is the determined cDNA sequence for clone 26714.<br>
SEQ ID NO: 696 is the determined cDNA sequence for clone 26715.<br>
SEQ ID NO: 697 is the determined cDNA sequence for clone 26716.<br>
SEQ ID NO: 698 is the detennined cDNA sequence for clone 26717.<br>
SEQ ID NO: 699 is the determined cDNA sequence for clone 26718.<br>
SEQ ID NO: 700 is the detennined cDNA sequence for clone 26719.<br>
SEQ ID NO: 701 is the determined cDNA sequence for clone 26720.<br>
SEQ ID NO: 702 is the determined cDNA sequence for clone 26721.<br>
SEQ ID NO: 703 is the determined cDNA sequence for clone 26722.<br>
SEQ ID NO: 704 is the detennined cDNA sequence for clone 26723.<br>
SEQ ID NO: 705 is the determined cDNA sequence for clone 26724.<br>
. SEQ ID NO: 706 is the determined cDNA sequence for clone 26725.<br>
SEQ ID NO: 707 is the determined cDNA sequence for clone 26726.<br>
SEQ ID NO: 708 is the detennined cDNA sequence for clone 26727.<br>
SEQ ID NO: 709 is the determined cDNA sequence for clone 26728.<br>
SEQ ID NO: 710 is the detennined cDNA sequence for clone 26729.<br>
SEQ ID NO: 711 is the determined cDNA sequence for clone 26730.<br>
SEQ ID NO: 712 is the determined cDNA sequence for clone 26731.<br>
SEQ ID NO: 713 is the determined cDNA sequence for clone 26732.<br>
SEQ ID NO: 714 is the determined cDNA sequence for clone 26733.1.<br>
SEQ ID NO: 715 is the determined cDNA sequence for clone 26733.2.<br>
SEQ ID NO: 716 is the determined cDNA sequence for clone 26734.<br>
SEQ ID NO: 717 is the determined cDNA sequence for clone 26735.<br>
SEQ ID NO: 718 is the determined cDNA sequence for clone 26736.<br>
SEQ ID NO: 719 is the determined cDNA sequence for clone 26737.<br>
SEQ ID NO: 720 is the determined cDNA sequence for clone 26738.<br>
SEQ ID NO: 721 is the determined cDNAvSequence for clone 26739.<br>
SEQ ID NO: 722 is the determined cDNA sequence for clone 26741.<br>
SEQ ID NO: 723 is the determined cDNA sequence for clone 26742.<br>
SEQ ID NO: 724 is the determined cDNA sequence for clone 26743.<br>
SEQ ID NO: 725 is the determined cDNA sequence for clone 26744.<br>
SEQ ID NO: 726 is the determined cDNA sequence for clone 26745.<br>
SEQ ID NO: 727 is the determined cDNA sequence for clone 26746.<br>
SEQ ID NO: 728 is the determined cDNA sequence for clone 26747.<br>
SEQ ID NO: 729 is the determined cDNA sequence for clone 26748.<br>
SEQ ID NO: 730 is the determined cDNA sequence for clone 26749.<br>
SEQ ID NO: 731 is the determined cDNA sequence for clone 26750.<br>
SEQ ID NO: 732 is the determined cDNA sequence for clone 26751.<br>
SEQ ID NO: 733 is the determined cDNA sequence for clone 26752.<br>
SEQ ID NO: 734 is the determined cDNA sequence for clone 26753.<br>
SEQ ID NO: 735 is the determined cDNA sequence for clone 26754.<br>
SEQ ID NO: 736 is the determined cDNA sequence for clone 26755.<br>
SEQ ID NO: 737 is the determined cDNA sequence for clone 26756.<br>
SEQ ID NO: 738 is the determined cDNA sequence for clone 26757.<br>
SEQ ID NO: 739 is the determined cDNA sequence for clone 26758.<br>
SEQ ID NO: 740 is the determined cDNA sequence for clone 26759.<br>
SEQ ID NO: 741 is the determined cDNA sequence for clone 26760.<br>
SEQ ID NO: 742 is the determined cDNA sequence for clone 26761.<br>
SEQ ID NO: 743 is the determined cDNA sequence for clone 26762.<br>
SEQ ID NO: 744 is the determined cDNA sequence for clone 26763.<br>
SEQ ID NO: 745 is the determined cDNA sequence for clone 26764.<br>
SEQ ID NO: 746 is the determined cDNA sequence for clone 26765.<br>
SEQ ID NO: 747 is the determined cDNA sequence for clone 26766.<br>
SEQ ID NO: 748 is the determined cDNA sequence for clone 26767.<br>
SEQ ID NO: 749 is the determined cDNA sequence for clone 26768.<br>
SEQ ID NO: 750 is the determined cDNA sequence for clone 26769.<br>
SEQ ID NO: 751 is the determined cDNA sequence for clone 26770.<br>
SEQ ID NO: 752 is the determined cDNA sequence for clone 26771.<br>
SEQ ID NO: 753 is the determined cDNA sequence for clone 26772.<br>
SEQ ID NO: 754 is the determined cDNA sequence for clone 26773.<br>
SEQ ID NO: 755 is the determined cDNA sequence for clone 26774.<br>
SEQ ID NO: 756 is the determined cDNA sequence for clone 26775.<br>
SEQ ID NO: 757 is the determined cDNA sequence for clone 26776.<br>
SEQ ID NO: 758 is the determined cDNA sequence for clone 26777.<br>
SEQ ID NO: 759 is the determined cDNA sequence for clone 26778.<br>
SEQ ID NO: 760 is the determined cDNA sequence for clone 26779.<br>
SEQ ID NO: 761 is the determined cDNA sequence for clone 26781.<br>
SEQ ID NO: 762 is the determined cDNA sequence for clone 26782.<br>
SEQ ID NO: 763 is the determined cDNA sequence for clone 26783.<br>
SEQ ID NO: 764 is the determined cDNA sequence for clone 26784.<br>
SEQ ID NO: 765 is the determined cDNA sequence for clone 26785.<br>
SEQ ID NO: 766 is the determined cDNA sequence for clone 26786.<br>
SEQ ID NO: 767 is the determined cDNA sequence for clone 26787.<br>
SEQ ID NO: 768 is the determined cDNA sequence for clone 26788.<br>
SEQ ID NO: 769 is the determined cDNA sequence for clone 26790.<br>
SEQ ID NO: 770 is the determined cDNA sequence for clone 26791.<br>
SEQ ID NO: 771 is the determined cDNA sequence for clone 26792.<br>
SEQ ID NO: 772 is the determined cDNA sequence for clone 26793.<br>
SEQ ID NO: 773 is the determined cDNA sequence for clone 26794.<br>
SEQ ID NO: 774 is the determined cDNA sequence for done 26795.<br>
SEQ ID NO: 775 is the determined cDNA sequence for clone 26796.<br>
SEQ ID NO: 776 is the determined cDNA sequence for clone 26797.<br>
SEQ ID NO: 777 is the determined cDNA sequence for clone 26798.<br>
SEQ ID NO: 778 is the determined cDNA sequence for clone 26800.<br>
SEQ ID NO: 779 is the determined cDNA sequence for clone 26801.<br>
SEQ ID NO: 780 is the determined cDNA sequence for clone 26802.<br>
SEQ ID NO: 781 is the determined cDNA sequence for clone 26803.<br>
SEQ ID NO: 782 is the determined cDNA sequence for clone 26804.<br>
SEQ ID NO: 783 is the amino acid sequence for L773P.<br>
SEQ ID NO: 784 is the determined DNA sequence of the L773P<br>
expression construct.<br>
SEQ ID NO: 785 is the determined DNA sequence of the L773PA<br>
expression construct<br>
SEQ ID NO: 786 is a predicted amino acid sequence for L552S.<br>
SEQ ID NO: 787 is a predicted amino acid sequence for L840P.<br>
SEQ ID NO: 788 is the full-length cDNA sequence for L548S.<br>
SEQ ID NO: 789 is the amino acid sequence encoded by SEQ ID NO:<br>
788.<br>
SEQ ID NO: 790 is an extended cDNA sequence for L552S.<br>
SEQ ID NO: 791 is the predicted amino acid sequence encoded by the<br>
cDNA sequence of SEQ ID NO: 790.<br>
SEQ ID NO: 792 is the determined cDNA sequence for an isoform of<br>
L552S.<br>
SEQ ID NO: 793 is the predicted amino acid sequence encoded by SEQ<br>
ID NO: 792.<br>
SEQ ID NO: 794 is an extended cDNA sequence for L840P.<br>
SEQ ID NO: 795 is the predicted amino acid sequence encoded by SEQ<br>
DINO:794.<br>
SEQ ID NO: 796 is an extended cDNA sequence for L80IP.<br>
SEQ ID NO: 797 is a first predicted amino acid sequence encoded by<br>
SEQIDNO:796.<br>
SEQ ID NO: 798 is a second predicted amino acid sequence encoded by<br>
SEQ ID NO: 796.<br>
SEQ ID NO: 799 is a third predicted amino acid sequence encoded by<br>
SEQ ID NO: 796.<br>
SEQ ID NO: 800 is the determined full-length sequence for L844P.<br>
SEQ ID NO: 801 is the 5' consensus cDNA sequence for L551S.<br>
SEQ ID NO: 802 is the 3' consensus cDNA sequence for L551 S.<br>
SEQ ID NO: 803 is the cDNA sequence for STY8.<br>
SEQ ID NO: 804 is an extended cDNA sequence for L551 S.<br>
SEQ ID NO: 805 is the amino acid sequence for STY8.<br>
SEQ ID NO: 806 is the extended amino acid sequence for L551S.<br>
SEQ ID NO: 807 is the determined full-length cDNA sequence for<br>
L773P.<br>
SEQ ID NO: 808 is the full-length cDNA sequence of L552S.<br>
SEQ ID NO: 809 is the full-length amino acid sequence of L552S.<br>
SEQ ID NO: 810 is the determined cDNA sequence of clone 50989.<br>
SEQ ID NO: 811 is the determined cDNA sequence of clone 50990.<br>
SEQ ID NO: 812 is the determined cDNA sequence of clone 50992.<br>
SEQ ID NO: 813-824 are the determined cDNA sequences for clones<br>
isolated from lung tumor tissue.<br>
SEQ ID NO: 825 is the determined cDNA sequence for the full-length<br>
L55IS clone 54305.<br>
SEQ ID NO: 826 is the determined cDNA sequence for the full-length<br>
L55IS clone 54298.<br>
SEQ ID NO: 827 is the full-length amino acid sequence for L551S.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
As noted above, the present invention is generally directed to<br>
compositions and methods for using the compositions, for example in the therapy and<br>
diagnosis of cancer, such as lung cancer. Certain illustrative compositions described<br>
herein include lung tumor polypeptides, polynucleotides encoding such polypeptides,<br>
binding agents such as antibodies, antigen presenting cells (APCs) and/or immune<br>
system cells (e.g., T cells). A " lung tumor protein," as the term is used herein, refers<br>
generally to a protein that is expressed in lung tumor cells at a level that is at least two<br>
fold, and preferably at least five fold, greater than the level of expression in a normal<br>
tissue, as determined using a representative assay provided herein. Certain lung tumor<br>
proteins are tuinor proteins that react detectably (within an immunoassay, such as an<br>
ELISA or Western blot) with antisera of a patient afflicted with lung cancer.<br>
Therefore, in accordance with the above, and as described further below,<br>
the present invention provides illustrative polynucleotide compositions having<br>
sequences set forth in SEQ ID NO: 1, 11-13, 15, 20, 23-27, 29, 30, 33, 34, 39, 41, 43-<br>
46, 51, 52, 57, 58, 60, 62, 65-67, 69-71, 74, 76, 79, 80, 84, 86, 89-92, 95, 97, 98, 101,<br>
110, 111, 113-119,121-128,130-134,136, 138,139,141,143,146-151, 153, 154, 157-<br>
160, 162-164, 167-178,180, 181, 183, 186-190, 192,193,195-220, 224, 226-231, 234,<br>
236, 237, 240, 241, 244-246, 248, 254, 255, 261, 262, 266, 270, 275, 280, 282, 283,<br>
288, 289, 290, 292, 295, 301, 303, 304, 309, 311, 341-782, 784, 785, 790, 792, 794,<br>
796, 800-804, 807, 808 and 810-826, illustrative polypeptide compositions having<br>
amino acid sequences set forth in SEQ ID NO: 786, 787, 791, 793, 795, 797-799, 806, <br>
809 and 827, antibody compositions capable of binding such polypeptides, and<br>
numerous additional embodiments employing such compositions, for example in the<br>
detection, diagnosis and/or therapy of human lung cancer.<br>
POLYNUCLEOTIDE COMPOSITIONS<br>
As used herein, the terms "DNA segment" and "polynucleotide" refer to<br>
a DNA molecule that has been isolated free of total genomic DNA of a particular<br>
species. Therefore, a DNA segment encoding a polypeptide refers to a DNA segment<br>
that contains one or more coding sequences yet is substantially isolated away from, or<br>
purified free from, total genomic DNA of the species from which the DNA segment is<br>
obtained. Included within the terms "DNA segment" and "polynucleetide" are DNA<br>
segments and smaller fragments of such segments, and also recombinant vectors,<br>
including, for example, plasmids, cosmids, phagemids, phage, viruses, and the like.<br>
As will be understood by those skilled in the art, the DNA segments of<br>
this invention can include genomic sequences, extra-genomic and plasmid-encoded<br>
sequences and smaller engineered gene segments that express, or may be adapted to<br>
express, proteins, polypeptides, peptides and the like. Such segments may be naturally<br>
isolated, or modified synthetically by the hand of man.<br>
"Isolated," as used herein, means that a polynucleotide is substantially<br>
away from other coding sequences, and that the DNA segment does not contain large<br>
portions of unrelated coding DNA, such as large chromosomal fragments or other<br>
functional genes or polypeptide coding regions. Of course, this refers to the DNA<br>
segment as originally isolated, and does not exclude genes or coding regions later added<br>
to the segment by the hand of man.<br>
As will be recognized by the skilled artisan, polynucleotides may be<br>
single-stranded (coding or antisense) or double-stranded, and may be DNA (genomic,<br>
cDNA or synthetic) or RNA molecules. RNA molecules include HnRNA molecules,<br>
which contain introns and correspond to a DNA molecule hi a one-to-one manner, and<br>
mRNA molecules, which do not contain introns. Additional coding or non-coding<br>
sequences may, but need not, be present within a polynucleotide of the present<br>
invention, and a polynucleotide may, but need not, be linked to other molecules and/or<br>
support materials.<br>
Polynucleotides may comprise a native sequence (i.e., an endogenous<br>
sequence that encodes a lung tumor protein or a portion thereof) or may comprise a<br>
variant, or a biological or antigenic functional equivalent of such a sequence.<br>
Polynucleotide variants may contain one or more substitutions, additions, deletions<br>
and/or insertions, as further described below, preferably such that the immunogenicity<br>
of the encoded polypeptide is not diminished, relative to a native tumor protein. The<br>
effect on the immunogenicity of the encoded polypeptide may generally be assessed as<br>
described herein. The term "variants" also encompasses homologous genes of<br>
xenogenic origin.<br>
When comparing polynucleotide or polypeptide sequences, two<br>
sequences are said to be "identical" if the sequence of nucleotides or amino acids in the<br>
two sequences is the same when aligned for maximum correspondence, as described<br>
below. Comparisons between two sequences are typically performed by comparing the<br>
sequences over a comparison window to identify and compare local regions of sequence<br>
similarity. A "comparison window" as used herein, refers to a segment of at least about<br>
20 contiguous positions, usually 30 to about 75, 40 to about 50, in which a sequence<br>
may be compared to a reference sequence of the same number of contiguous positions<br>
after the two sequences are optimally aligned.<br>
Optimal alignment of sequences for comparison may be conducted using<br>
the Megalign program in the Lasergene suite of bioinformatics software (DNASTAR,<br>
Inc., Madison, WI), using default parameters. This program embodies several<br>
alignment schemes described in the following references: Dayhoff, M.O. (1978) A<br>
model of evolutionary change in proteins - Matrices for detecting distant relationships.<br>
In Dayhoff, M.O. (ed.) Atlas of Protein Sequence and Structure, National Biomedical<br>
Research Foundation, Washington DC Vol. 5, Suppl. 3, pp. 345-358; Hein J. (1990)<br>
Unified Approach to Alignment and Phylogenes pp. 626-645 Methods in Enzymology<br>
vol. 183, Academic Press, Inc., San Diego, CA; Higgins, D.G. and Sharp, P.M. (1989)<br>
CABIOS 5:151-153; Myers, E.W. and Muller W. (1988) CABIOS 4:11-17; Robinson,<br>
E.D. (1971) Comb. Theor 77:105; Santou, N. Nes, M. (1987) Mol. Biol Evol 4:406-<br>
425; Sneath, P.H.A. and Sokal, R.R. (1973) Numerical Taxonomy - the Principles and<br>
Practice of Numerical Taxonomy, Freeman Press, San Francisco, CA; Wilbur, W. J. and<br>
Lipman, D.J. (1983) Proc. Natl. Acad., Sci. USA 80:726-730.<br>
Alternatively, optimal alignment of sequences for comparison may be<br>
conducted by the local identity algorithm of Smith and Waterman (1981) Add. APL.<br>
Math 2:482, by the identity alignment algorithm of Needleman and Wunsch (1970) J.<br>
Mol. Biol. 48:443, by the search for similarity methods of Pearson and Lipman (1988)<br>
Proc. Natl. Acad. Sci. USA 85: 2444, by computerized implementations of these<br>
algorithms (GAP, BESTFIT, BLAST, FASTA, and TFASTA in the Wisconsin Genetics<br>
Software Package, Genetics Computer Group (GCG), 575 Science Dr., Madison, WI),<br>
or by inspection.<br>
One preferred example of algorithms that are suitable for determining<br>
percent sequence identity and sequence similarity are the BLAST and BLAST 2.0<br>
algorithms, which are described in Altschul et al. (1977) Nucl. Acids Res. 25:3389-3402<br>
and Altschul et al. (1990) J. Mol. Biol 215:403-410, respectively. BLAST and BLAST<br>
2.0 can be used, for example with the parameters described herein, to determine percent<br>
sequence identity for the polynucleotides and polypeptides of the invention. Software<br>
for performing BLAST analyses is publicly available through the National Center for<br>
Biotechnology Information. In one illustrative example, cumulative scores can be<br>
calculated using, for nucleotide sequences, the parameters M (reward score for a pair of<br>
matching residues; always &gt;0) and N (penalty score for mismatching residues; always<br>
cumulative score. Extension of the word hits in each direction are halted when: the<br>
cumulative alignment score falls off by the quantity X from its maximum achieved<br>
value; the cumulative score goes to zero or below, due to the accumulation of one or<br>
more negative-scoring residue alignments; or the end of either sequence is reached.<br>
The BLAST algorithm parameters W, T and X determine the sensitivity and speed of<br>
the alignment. The BLASTN program (for nucleotide sequences) uses as defaults a<br>
wordlength (W) of 11, and expectation (E) of 10, and the BLOSUM62 scoring matrix<br>
(see Henlkoff and Henikoff (1989) Proc. Natl. Acad. Sci. USA 89:10915) alignments,<br>
(B) of 50, expectation (E) of 10, M=5, N=-4 and a comparison of both strands.<br>
Preferably, the "percentage of sequence identity" is determined by<br>
comparing two optimally aligned sequences over a window of comparison of at least <br>
positions, wherein the portion of the polynucleotide or polypeptide sequence in the<br>
comparison window may comprise additions or deletions (i.e., gaps) of 20 percent or<br>
less, usually 5 to 15 percent, or 10 to 12 percent, as compared to the reference<br>
sequences (which does not comprise additions or deletions) for optimal alignment of the<br>
two sequences. The percentage is calculated by determining the number of positions at<br>
which the identical nucleic acid bases or amino acid residue occurs in both sequences to<br>
yield the number of matched positions, dividing the number of matched positions by the<br>
total number of positions in the reference sequence (i.e., the window size) and<br>
multiplying the results by 100 to yield the percentage of sequence identity.<br>
Therefore, the present invention encompasses polynucleotide and<br>
polypeptide sequences having substantial identity to the sequences disclosed herein, for<br>
example those comprising at least 50% sequence identity, preferably at least 55%, 60%,<br>
65%, 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, or 99% or higher, sequence<br>
identity compared to a polynucleotide or polypeptide sequence of this invention using<br>
the methods described herein, (e.g., BLAST analysis using standard parameters, as<br>
described below). One skilled in this art will recognize that these values can be<br>
appropriately adjusted to determine corresponding identity of proteins encoded by two<br>
nucleotide sequences by taking into account codon degeneracy, amino acid similarity,<br>
reading frame positioning and the like.<br>
In additional embodiments, the present invention provides isolated<br>
polynucleotides and polypeptides comprising various lengths of contiguous stretches of<br>
sequence identical to or complementary to one or more of the sequences disclosed<br>
herein. For example, polynucleotides are provided by this invention that comprise at<br>
least about 15, 20, 30, 40, 50, 75, 100, 150, 200, 300, 400, 500 or 1000 or more<br>
contiguous nucleotides of one or more of the sequences disclosed herein as well as all<br>
intermediate lengths there between. It will be readily understood that "intermediate<br>
lengths", in this context, means any length between the quoted values, such as 16, 17,<br>
18, 19, etc.; 21, 22, 23, etc.; 30, 31, 32, etc.; 50, 51, 52, 53, etc.; 100, 101, 102, 103,<br>
etc.; 150, 151,152,153, etc.; including all integers through 200-500; 500-1,000, and the<br>
like.<br>
The polynucleotides of the present invention, or fragments thereof,<br>
regardless of the length of the coding sequence itself, may be combined with other<br>
DNA sequences, such as promoters, polyadenylation signals, additional restriction<br>
enzyme sites, multiple cloning sites, other coding segments, and the like, such that their<br>
overall length may vary considerably. It is therefore contemplated that a nucleic acid<br>
fragment of almost any length may be employed, with the total length preferably being<br>
limited by the ease of preparation and use in the intended recombinant DNA protocol.<br>
For example, illustrative DNA segments with total lengths of about 10,000, about 5000,<br>
about 3000, about 2,000, about 1,000, about 500, about 200, about 100, about 50 base<br>
pairs in length, and the like, (including all intermediate lengths) are contemplated to be<br>
useful in many implementations of this invention.<br>
In other embodiments, the present invention is directed to<br>
polynucleotides that are capable of hybridizing under moderately stringent conditions to<br>
a polynucleotide sequence provided herein, or a fragment thereof, or a complementary<br>
sequence thereof. Hybridization techniques are well known in the art of molecular<br>
biology. For purposes of illustration, suitable moderately stringent conditions for<br>
testing the hybridization of a polynucleotide of this invention with other<br>
polynucleotides include prewashing in a solution of 5 X SSC, 0.5% SDS, 1.0 mM<br>
EDTA (pH 8.0); hybridizing at 50°C-65°C, 5 X SSC, overnight; followed by washing<br>
twice at 65°C for 20 minutes with each of 2X, 0.5X and 0.2X SSC containing 0.1%<br>
SDS.<br>
Moreover, it will be appreciated by those of ordinary skill in the art that,<br>
as a result of the degeneracy of the genetic code, there are many nucleotide sequences<br>
that encode a polypeptide as described herein. Some of these polynucleotides bear<br>
minimal homology to the nucleotide sequence of any native gene. Nonetheless,<br>
polynucleotides that vary due to differences in codon usage are specifically<br>
contemplated by the present invention. Further, alleles of the genes comprising the<br>
polynucleotide sequences provided herein are within the scope of the present invention.<br>
Alleles are endogenous genes that are altered as a result of one or more mutations, such<br>
as deletions, additions and/or substitutions of nucleotides. The resulting rnRNA and<br>
protein may, but need not, have an altered structure or function. Alleles may be<br>
identified using standard techniques (such as hybridization, amplification and/or<br>
database sequence comparison).<br>
PROBES AND PRIMERS<br>
In other embodiments of the present invention, the polynucleotide<br>
sequences provided herein can be advantageously used as probes or primers for nucleic<br>
acid hybridization. As such, it is contemplated that nucleic acid segments that comprise<br>
a sequence region of at least about 15 nucleotide long contiguous sequence that has the<br>
same sequence as, or is complementary to, a 15 nucleotide long contiguous sequence<br>
disclosed herein will find particular utility. Longer contiguous identical or<br>
complementary sequences, e.g., those of about 20, 30, 40, 50, 100, 200, .500, 1000<br>
(including all intermediate lengths) and even up to full length sequences will also be of<br>
use in certain embodiments.<br>
The ability of such nucleic acid probes to specifically hybridize to a<br>
sequence of interest will enable them to be of use in detecting the presence of<br>
complementary sequences in a given sample. However, other uses are also envisioned,<br>
such as the use of the sequence information for the preparation of mutant species<br>
primers, or primers for use in preparing other genetic constructions.<br>
Polynucleotide molecules having sequence regions consisting of<br>
contiguous nucleotide stretches of 10-14,15-20,30, 50, or even of 100-200 nucleotides<br>
or so (including intermediate lengths as well), identical or complementary to a<br>
polynucleotide sequence disclosed herein, are particularly contemplated as<br>
hybridization probes for use in, e.g., Southern and Northern blotting. This would allow<br>
a gene product, or fragment thereof, to be analyzed, both in diverse cell types and also<br>
in various bacterial cells. The total size of fragment, as well as the size of the<br>
complementary stretch(es), will ultimately depend on the intended use or application of<br>
the particular nucleic acid segment. Smaller fragments will generally find use in<br>
hybridization embodiments, wherein the length of the contiguous complementary<br>
region may be varied, such as between about 15 and about 100 nucleotides, but larger<br>
contiguous complementarity stretches may be used, according to the length<br>
complementary sequences one wishes to detect.<br>
The use of a hybridization probe of about 15-25 nucleotides in length<br>
allows the formation of a duplex molecule that is both stable and selective. Molecules<br>
having contiguous complementary sequences over stretches greater than 15 bases in<br>
length are generally preferred, though, in order to increase stability and selectivity of<br>
the hybrid, and thereby improve the quality and degree of specific hybrid molecules<br>
obtained. One will generally prefer to design nucleic acid molecules having genecomplementary<br>
stretches of 15 to 25 contiguous nucleotides, or even longer where<br>
desired.<br>
Hybridization probes may be selected from any portion of any of the<br>
sequences disclosed herein. All that is required is to review the sequence set forth in<br>
SEQ ID NO: 1, 11-13, 15, 20, 23-27, 29, 30, 33, 34, 39, 41, 43-46, 51, 52, 57, 58, 60,<br>
62, 65-67, 69-71, 74, 76, 79, 80, 84, 86, 89-92, 95, 97, 98,101,110, 111,113-119,121-<br>
128, 130-134, 136, 138, 139,141, 143, 146-151, 153, 154, 157-160, 162-164, 167-178,<br>
180,181,183,186-190,192,193,195-220,224,226-231,234,236,237,240,241, 244-<br>
246, 248, 254, 255, 261, 262, 266, 270, 275, 280, 282, 283, 288, 289, 290, 292, 295,<br>
301, 303,304, 309, 311,341-782, 784, 785, 790, 792, 794, 796, 800-804, 807, 808 and<br>
810-826, or to any continuous portion of the sequence, from about 15-25 nucleotides in<br>
length up to and including the full length sequence, that one wishes to utilize as a probe<br>
or primer. The choice of probe and primer sequences may be governed by various<br>
factors. For example, one may wish to employ primers from towards the termini of the<br>
total sequence.<br>
Small polynucleotide segments or fragments may be readily prepared by,<br>
for example, directly synthesizing the fragment by chemical means, as is commonly<br>
practiced using an automated oligonucleotide synthesizer. Also, fragments may be<br>
obtained by application of nucleic acid reproduction technology, such as the PCR™<br>
technology of U. S. Patent 4,683,202 (incorporated herein by reference), by introducing<br>
selected sequences into recombinant vectors for recombinant production, and by other<br>
recombinant DNA techniques generally known to those of skill in the art of molecular<br>
biology.<br>
The nucleotide sequences of the invention may be used for their ability<br>
to selectively form duplex molecules with complementary stretches of the entire gene or<br>
gene fragments of interest. Depending on the application envisioned, one will typically<br>
desire to employ varying conditions of hybridization to achieve varying degrees of<br>
selectivity of probe towards target sequence. For applications requiring high<br>
selectivity, one will typically desire to employ relatively stringent conditions to form<br>
the hybrids, e.g., one will select relatively low salt and/or high temperature conditions,<br>
such as provided by a salt concentration of from about 0.02 M to about 0.15 M salt at<br>
temperatures of from about 50°G to about 70°C. Such selective conditions tolerate<br>
little, if any, mismatch between the probe and the template or target strand, and would<br>
be particularly suitable for isolating related sequences. -<br>
Of course, for some applications, for example, where one desires to<br>
prepare mutants employing a mutant primer strand hybridized to an underlying<br>
template, less stringent (reduced stringency) hybridization conditions will typically be<br>
needed in order to allow formation of the heteroduplex. In these circumstances, one<br>
may desire to employ salt conditions such as those of from about 0.15 M to about 0.9 M<br>
salt, at temperatures ranging from about 20°C to about 55°C. Cross-hybridizing.species<br>
can thereby be readily identified as positively hybridizing signals with respect to control<br>
hybridizations. In any case, it is generally appreciated that conditions can be rendered<br>
more stringent by the addition of increasing amounts of formamide, which serves to<br>
destabilize the hybrid duplex in the same manner as increased temperature. Thus,<br>
hybridization conditions can be readily manipulated, and thus will generally be a<br>
method of choice depending on the desired results.<br>
POLYNUCLEOTIDE IDENTIFICATION AND CHARACTERIZATION<br>
Polynucleotides may be identified, prepared and/or manipulated using<br>
any of a variety of well established techniques. For example, a polynucleotide may be<br>
identified, as described in more detail below, by screening a microarray of cDNAs for<br>
tumor-associated expression (i.e., expression that is at least two fold greater in a tumor<br>
than in normal tissue, as determined using a representative assay provided herein).<br>
Such screens may be performed, for example, using a Synteni microarray (Palo Alto,<br>
CA) according to the manufacturer's instructions (and essentially as described by<br>
Schena et al., Proc. Natl. Acad. Sci. USA £3:10614-10619,1996 and Heller et al, Proc.<br>
Natl. Acad. Sci. USA 94:2150-2155, 1997). Alternatively, polynucleotides may be<br>
amplified from cDNA prepared from cells expressing the proteins described herein,<br>
such as lung tumor cells. Such polynucleotides may be amplified via polymerase chain<br>
reaction (PCR). For this approach, sequence-specific primers may be designed based<br>
on the sequences provided herein, and may be purchased or synthesized.<br>
An amplified portion of a polynucleotide of the present invention may be<br>
used to isolate a full length gene from a suitable library (e.g., a lung tumor cDNA<br>
library) using well known techniques. Within such techniques, a library (cDNA or<br>
genomic) is screened using one or more polynucleotide probes or primers suitable for<br>
amplification. Preferably, a library is size-selected to include larger molecules.<br>
Random primed libraries may also be preferred for identifying 5' and upstream regions<br>
of genes. Genomic libraries are preferred for obtaining introns and extending 5'<br>
sequences.<br>
For hybridization techniques, a partial sequence may be labeled (e.g., by<br>
nick-translation or end-labeling with 32P) using well known techniques. A bacterial or<br>
bacteriophage library is then generally screened by hybridizing filters containing<br>
denatured bacterial colonies (or lawns containing phage plaques) with the labeled probe<br>
(see Sambrook et al., Molecular Cloning: A Laboratory Manual, Cold Spring Harbor<br>
Laboratories, Cold Spring Harbor, NY, 1989). Hybridizing colonies or plaques are<br>
selected and expanded, and the DNA is isolated for further analysis. cDNA clones may<br>
be analyzed to determine the amount of additional sequence by, for example, PCR<br>
using a primer from the partial sequence and a primer from the vector. Restriction<br>
maps and partial sequences may be generated to identify one or more overlapping<br>
clones. The complete sequence may then be determined using standard techniques,<br>
which may involve generating a series of deletion clones. The resulting overlapping<br>
sequences can then assembled into a single contiguous sequence. A full length cDNA<br>
molecule can be generated by ligating suitable fragments, using well known techniques.<br>
Alternatively, there are numerous amplification techniques for obtaining<br>
a full length coding sequence from a partial cDNA sequence. Within such techniques,<br>
amplification is generally performed via PCR. Any of a variety of commercially<br>
available kits may be used to perform the amplification step. Primers may be designed<br>
using, for example, software well known in the art. Primers are preferably 22-30<br>
nucleotides hi length, have a GC content of at least 50% and anneal to the target<br>
sequence at temperatures of about 68°C to 72°C. The amplified region may be<br>
sequenced as described above, and overlapping sequences assembled into a contiguous<br>
sequence.<br>
One such amplification technique is inverse PCR (see Triglia et al.,<br>
Nucl. Acids Res. 76:8186,1988), which uses restriction enzymes to generate a fragment<br>
in the known region of the gene. The fragment is then circularized by intramolecular<br>
ligation and used as a template for PCR with divergent primers derived-from the known<br>
region. Within an alternative approach, sequences adjacent to a partial sequence may<br>
be retrieved by amplification with a primer to a linker sequence and a primer specific to<br>
a known region. The amplified sequences are typically subjected to a second round of<br>
amplification with the same linker primer and a second primer specific to the known<br>
region. A variation on this procedure, which employs two primers that initiate<br>
extension in opposite directions from the known sequence, is described hi WO<br>
96/38591. Another such technique is known as "rapid amplification of cDNA ends" or<br>
RACE. This technique involves the use of an internal primer and an external primer,<br>
which hybridizes to a polyA region or vector sequence, to identify sequences that are 5'<br>
and 3' of a known sequence. Additional techniques include capture PCR (Lagerstrom et<br>
al, PCR Methods Applic. 1:111-19,1991) and walking PCR (Parker et al., Nucl. Acids.<br>
Res. 19:3055-60, 1991). Other methods employing amplification may also be<br>
employed to obtain a full length cDNA sequence.<br>
In certain instances, it is possible to obtain a full length cDNA sequence<br>
by analysis of sequences provided in an expressed sequence tag (EST) database, such as<br>
that available from GenBank. Searches for overlapping ESTs may generally be<br>
performed using well known programs (e.g., NCBI BLAST searches), and such ESTs<br>
may be used to generate a contiguous full length sequence. Full length DMA sequences<br>
may also be obtained by analysis of genomic fragments.<br>
POLYNUCLEOTIDE EXPRESSION IN HOST CELLS<br>
In other embodiments of the invention, polynucleotide sequences or<br>
fragments thereof which encode polypeptides of the invention, or fusion proteins or<br>
functional equivalents thereof, may be used in recombinant DNA molecules to direct<br>
expression of a polypeptide hi appropriate host cells. Due to the inherent degeneracy of<br>
the genetic code, other DNA sequences that encode substantially the same or a<br>
functionally equivalent amino acid sequence may be produced and these sequences may<br>
be used to clone and express a given polypeptide.<br>
As will be understood by those of skill in the art, it may be advantageous<br>
in some instances to produce polypeptide-encoding nucleotide sequences possessing<br>
non-naturally occurring codons. For example, codons preferred by a particular<br>
prokaryotic or eukaryotic host can be selected to increase the rate of protein expression<br>
or to produce a recombinant RNA transcript having desirable properties, such as a halflife<br>
which is longer than that of a transcript generated from the naturally occurring<br>
sequence.<br>
Moreover, the polynucleotide sequences of the present invention can be<br>
engineered using methods generally known in the art in order to alter polypeptide<br>
encoding sequences for a variety of reasons, including but not limited to, alterations<br>
which modify the cloning, processing, and/or expression of the gene product. For<br>
example, DNA shuffling by random fragmentation and PCR reassembly of gene<br>
fragments and synthetic oligonucleotides may be used to engineer the nucleotide<br>
sequences. In addition, site-directed mutagenesis may be used to insert new restriction<br>
sites, alter glycosylation patterns, change cddon preference, produce splice variants, or<br>
introduce mutations, and so forth.<br>
In another embodiment of the invention, natural, modified, or<br>
recombinant nucleic acid sequences may be ligated to a heterologous sequence to<br>
encode a fusion protein. For example, to screen peptide libraries for inhibitors of<br>
polypeptide activity, it may be useful to encode a chimeric protein that can be<br>
recognized by a commercially available antibody. A fusion protein may also be<br>
engineered to contain a cleavage site located between the polypeptide-encoding<br>
sequence and the heterologous protein sequence, so that the polypeptide may be cleaved<br>
and purified away from the heterologous moiety.<br>
Sequences encoding a desired polypeptide may be synthesized, in whole<br>
or in part, using chemical methods well known in the art (see Caruthers, M. H. et al.<br>
(1980) Nucl Acids Res. Symp. Ser. 215-223, Horn, T. et al. (1980) Nucl Acids Res.<br>
Symp. Ser. 225-232). Alternatively, the protein itself may be produced using chemical<br>
methods to synthesize the amino acid sequence of a polypeptide, or a portion thereof.<br>
For example, peptide synthesis can be performed using various solid-phase techniques<br>
(Roberge, J. Y. et al. (1995) Science 269:202-204) and automated synthesis may be<br>
achieved, for example, using the ABI 431A Peptide Synthesizer (Perkin Elmer, Palo<br>
Alto, CA).<br>
A newly synthesized peptide may be substantially purified by<br>
preparative high performance liquid chromatography (e.g., Creighton, T. (1983)<br>
Proteins, Structures and Molecular Principles, WH Freeman and Co., New York, N.Y.)<br>
or other comparable techniques available in the art. The composition of the synthetic<br>
peptides may be confirmed by amino acid analysis or sequencing (e.g., the Edman<br>
degradation procedure). Additionally, the amino acid sequence of a polypeptide, or any<br>
part thereof, may be altered during direct synthesis and/or combined using chemical<br>
methods with sequences from other proteins, or any part thereof, to produce a variant<br>
polypeptide.<br>
In order to express a desired polypeptide, the nucleotide sequences<br>
encoding the polypeptide, or functional equivalents, may be inserted into appropriate<br>
expression vector, i.e., a vector which contains the necessary elements for the<br>
transcription and translation of the inserted coding sequence. Methods which are well<br>
known to those skilled in the art may be used to construct expression vectors containing<br>
sequences encoding a polypeptide of interest and appropriate transcriptional and<br>
translational control elements. These methods include in vitro recombinant DNA<br>
techniques, synthetic techniques, and in vivo genetic recombination. Such techniques<br>
are described hi Sambrook, J. et al. (1989) Molecular Cloning, A Laboratory Manual,<br>
Cold Spring Harbor Press, Plainview, N.Y., and Ausubel, F. M. et al. (1989) Current<br>
Protocols in Molecular Biology, John Wiley &amp; Sons, New York. N.Y.<br>
A variety of expression vector/host systems may be utilized to contain<br>
and express polynucleotide sequences. These include, but are not limited to,<br>
microorganisms such as bacteria transformed with recombinant bacteriophage, plasmid,<br>
or cosmid DNA expression vectors; yeast transformed with yeast expression vectors;<br>
insect cell systems infected with virus expression vectors (e.g., baculovirus); plant cell<br>
systems transformed with virus expression vectors (e.g., cauliflower mosaic virus,<br>
CaMV; tobacco mosaic virus, TMV) or with bacterial expression vectors (e.g., Ti or<br>
pBR322 plasmids); or animal cell systems.<br>
The "control elements" or "regulatory sequences" present in an<br>
expression vector are those non-translated regions of the vector-enhancers, promoters,<br>
5' and 3' untranslated regions-which interact with host cellular proteins to carry out<br>
transcription and translation. Such elements may vary in their strength and specificity.<br>
Depending on the vector system and host utilized, any number of suitable transcription<br>
and translation elements, including constitutive and inducible promoters, may be used.<br>
For example, when cloning in bacterial systems, inducible promoters such as the hybrid<br>
lacZ promoter of the PBLUESCRIPT phagemid (Stratagene, La Jolla, Calif.) or<br>
PSPORT1 plasmid (Gibco BRL, Gaithersburg, MD) and the like may be used. In<br>
mammalian cell systems, promoters from mammalian genes or from mammalian<br>
viruses are generally preferred. If it is necessary to generate a cell line that contains<br>
multiple copies of the sequence encoding a polypeptide, vectors based on SV40 or EBV<br>
may be advantageously used with an appropriate selectable marker.<br>
In bacterial systems, a number of expression vectors may be selected<br>
depending upon the use intended for the expressed polypeptide. For example, when<br>
large quantities are needed, for example for the induction of antibodies, vectors which<br>
direct high level expression of fusion proteins that are readily purified may be used.<br>
Such vectors include, but are not limited to, the multifunctional E. coli cloning and<br>
expression vectors such as BLUESCRIPT (Stratagene), in which the sequence encoding<br>
the polypeptide of interest may be ligated into the vector hi frame with sequences for<br>
the amino-tenninal Met and the subsequent 7 residues of .beta.-galactosidase so that a<br>
hybrid protein is produced; pIN vectors (Van Heeke, G. and S. M. Schuster (1989) J.<br>
Biol. Chem. 26^:5503-5509); and the like. pGEX Vectors (Promega, Madison, Wis.)<br>
may also be used to express foreign polypeptides as fusion proteins with glutathione Stransferase<br>
(GST). In general, such fusion proteins are soluble and can easily be<br>
purified from lysed cells by adsorption to glutathione-agarose beads followed by elution<br>
in the presence of free glutathione. Proteins made in such systems may be designed to<br>
include heparin, thrombin, or factor XA protease cleavage sites so that the cloned<br>
polypeptide of interest can be released from the GST moiety at will.<br>
In the yeast, Saccharomyces cerevisiae, a number of vectors containing<br>
constitutive or inducible promoters such as alpha factor, alcohol oxidase, and PGH may<br>
be used. For reviews, see Ausubel et al (supra) and Grant et al. (1987) Methods<br>
Enzymol. 153:516-544.<br>
In cases where plant expression vectors are used, the expression of<br>
sequences encoding polypeptides may be driven by any of a number of promoters. For<br>
example, viral promoters such as the 35S and 19S promoters of CaMV may be used<br>
alone or in combination with the omega leader sequence from TMV (Takamatsu, N.<br>
(1987) EMBO J. 5:307-311. Alternatively, plant promoters such as the small subunit of<br>
RUBISCO or heat shock promoters may be used (Coruzzi, G. et al. (1984) EMBO J.<br>
3:1671-1680; Broglie, R. et al. (1984) Science 22*838-843; and Winter, J. et al (1991)<br>
Results Probl. Cell Differ. 77:85-105). These constructs can be introduced into plant<br>
cells by direct DNA transformation or pathogen-mediated transfection. Such techniques<br>
are described in a number of generally available reviews (see, for example, Hobbs, S. or<br>
Murry, L. E. in McGraw Hill Yearbook of Science and Technology (1992) McGraw<br>
Hill, New York, N.Y.; pp. 191-196).<br>
An bisect system may also be used to express a polypeptide of interest.<br>
For example, in one such system, Autographa californica nuclear polyhedrosis virus<br>
(AcNPV) is used as a vector to express foreign genes in Spodoptera frugiperda cells or<br>
hi Trichoplusia larvae. The sequences encoding the polypeptide may be cloned into a<br>
non-essential region of the virus, such as the polyhedrin gene, and placed under control<br>
of the polyhedrin promoter. Successful insertion of the polypeptide-encoding sequence<br>
will render the polyhedrin gene inactive and produce recombinant virus lacking coat<br>
protein. The recombinant viruses may then be used to infect, for example, S. frugiperda<br>
cells or Trichoplusia larvae hi which the polypeptide of interest may be expressed<br>
(Engelhard, E. K. etal (1994) Proc. Natl. Acad. Sci. 91:3224-3227).<br>
In mammalian host cells, a number of viral-based expression systems are<br>
generally available. For example, in cases where an adenovirus is used as an expression<br>
vector, sequences encoding a polypeptide of interest may be ligated into an adenovirus<br>
transcription/translation complex consisting of the late promoter and tripartite leader<br>
sequence. Insertion in a non-essential El or E3 region of the viral genome may be used<br>
to obtain a viable virus which is capable of expressing the polypeptide in infected host<br>
cells (Logan, J. and Shenk, T. (1984) Proc. Natl. Acad. Sci. 57:3655-3659). In addition,<br>
transcription enhancers, such as the Rous sarcoma virus (RSV) enhancer, may be used<br>
to increase expression in mammalian host cells.<br>
Specific initiation signals may also be used to achieve more efficient<br>
translation of sequences encoding a polypeptide of interest. Such signals include the<br>
ATG initiation codon and adjacent sequences. In eases where sequences encoding the<br>
polypeptide, its initiation codon, and upstream sequences are inserted into the<br>
appropriate expression vector, no additional transcriptional or translational control<br>
signals may be needed. However, in cases where only coding sequence, or a portion<br>
thereof, is inserted, exogenous translational control signals including the ATG initiation<br>
codon should be provided. Furthermore, the initiation codon should be in the correct<br>
reading frame to ensure translation of the entire insert. Exogenous translational<br>
elements and initiation codons may be of various origins, both natural and synthetic.<br>
The efficiency of expression may be enhanced by the inclusion of enhancers which are<br>
appropriate for the particular cell system which is used, such as those described in the<br>
literature (Scharf, D. etal. (1994) Results Probl. Cell Differ. 20:125-162).<br>
In addition, a host cell strain may be chosen for its ability to modulate<br>
the expression of the inserted sequences or to process the expressed protein in the<br>
desired fashion. Such modifications of the polypeptide include, but are not limited to,<br>
acetylation, carboxylation. glycosylation, phosphorylation, lipidation, and acylation.<br>
Post-translational processing which cleaves a "prepro" form of the protein may also be<br>
used to facilitate correct insertion, folding and/or function. Different host cells such as<br>
CHO, HeLa, MDCK, HEK293, and WI38, which have specific cellular machinery and<br>
characteristic mechanisms for such post-translational activities, may be chosen to<br>
ensure the correct modification and processing of the foreign protein.<br>
For long-term, high-yield production of recombinant proteins, stable<br>
expression is generally preferred. For example, cell lines which stably express a<br>
polynucleotide of interest may be transformed using expression vectors which may<br>
contain viral origins of replication and/or endogenous expression elements and a<br>
selectable marker gene on the same or on a separate vector. Following the introduction<br>
of the vector, cells may be allowed to grow for 1-2 days in an enriched media before<br>
they are switched to selective media. The purpose of the selectable marker is to confer<br>
resistance to selection, and its presence allows growth and recovery of cells which<br>
successfully express the introduced sequences. Resistant clones of stably transformed<br>
cells may be proliferated using tissue culture techniques appropriate to the cell type.<br>
Any number of selection systems may be used to recover transformed<br>
cell lines. These include, but are not limited to, the herpes simplex virus thymidine<br>
kinase (Wigler, M. et al. (1977) Cell 17:223-32) and adenine phosphoribosyltransferase<br>
(Lowy, I. et al (1990) Cell 22317-23) genes which can be employed in tk.sup.- or<br>
apitsup.- cells, respectively. Also, antimetabolite, antibiotic or herbicide resistance can<br>
be used as the basis for selection; for example, dhfr which confers resistance to<br>
methotrexate (Wigler, M. et al. (1980) Proc. Natl Acad. Sci. 77:3567-70); npt, which<br>
confers resistance to the aminoglycosides, neomycin and G-418 (Colbere-Garapin, F. et<br>
al (1981) J. Mol. Biol 750:1-14); and als or pat, which confer resistance to<br>
chlorsulfuron and phosphinotricin acetyltransferase, respectively (Murry, supra).<br>
Additional selectable genes have been described, for example, trpB, which allows cells<br>
to utilize indole hi place of tryptophan, or hisD, which allows cells to utilize histinol in<br>
place of histidine (Hartman, S. C. and R. C. Mulligan (1988) Proc. Natl. Acad. Sci.<br>
55:8047-51). Recently, the use of visible markers has gained popularity with such<br>
markers as anthocyanins, beta-glucuronidase and its substrate GUS, and luciferase and<br>
its substrate luciferin, being widely used not only to identify transformants, but also to<br>
quantify the amount of transient or stable protein expression attributable to a specific<br>
vector system (Rhodes, C. A. etal (1995) Methods Mol. Biol 55:121-131).<br>
Although the presence/absence of marker gene expression suggests that<br>
the gene of interest is also present, its presence and expression may need to be<br>
confirmed. For example, if the sequence encoding a polypeptide is inserted within a<br>
marker gene sequence, recombinant cells containing sequences can be identified by the<br>
absence of marker gene function. Alternatively, a marker gene can be placed hi tandem<br>
with a polypeptide-encoding sequence under the control of a single promoter.<br>
Expression of the marker gene hi response to induction or selection usually indicates<br>
expression of the tandem gene as well.<br>
Alternatively, host cells which contain and express a desired<br>
polynucleotide sequence may be identified by a variety of procedures known to those of<br>
skill in the art. These procedures include, but are not limited to, DNA-DNA or DNARNA<br>
hybridizations and protein bioassay or immunoassay techniques which include<br>
membrane, solution, or chip based technologies for the detection and/or quantification<br>
of nucleic acid or protein.<br>
A variety of protocols for detecting and measuring the expression of<br>
polynucleotide-encoded products, using either polyclonal or monoclonal antibodies<br>
specific for the product are known in the art. Examples include enzyme-linked<br>
immunosorbent assay (ELISA), radioimmunoassay (RIA), and fluorescence activated<br>
cell sorting (FACS). A two-site, monoclonal-based immunoassay utilizing monoclonal<br>
antibodies reactive to two non-interfering epitopes on a given polypeptide may be<br>
preferred for some applications, but a competitive binding assay may also be employed.<br>
These and other assays are described, among other places, in Hampton, R. et al. (1990;<br>
Serological Methods, a Laboratory Manual, APS Press, St Paul. Minn.) and Maddox, D.<br>
E. etal (1983; J. Exp. Med 158:1211-1216).<br>
A wide variety of labels and conjugation techniques are known by those<br>
skilled in the art and may be used in various nucleic acid and amino acid assays. Means<br>
for producing labeled hybridization or PCR probes for detecting sequences related to<br>
polynucleotides include oligolabeling, nick translation, end-labeling or PCR<br>
amplification using a labeled nucleotide. Alternatively, the sequences, or any portions<br>
thereof may be cloned into a vector for the production of an mRNA probe. Such vectors<br>
are known in the art, are commercially available, and may be used to synthesize RNA<br>
probes in vitro by addition of an appropriate RNA polymerase such as T7, T3, or SP6<br>
and labeled nucleotides. These procedures may be conducted using a variety of<br>
commercially available kits. Suitable reporter molecules or labels, which may be used<br>
include radionuclides, enzymes, fluorescent, chemiluminescent, or chromogenic agents<br>
as well as substrates, cofactors, inhibitors, magnetic particles, and the like.<br>
Host cells transformed with a polynucleotide sequence of interest may be<br>
cultured under conditions suitable for the expression and recovery of the protein from<br>
cell culture. The protein produced by a recombinant cell may be secreted or contained<br>
intracellularly depending on the sequence and/or the vector used. As will be understood<br>
by those of skill in the art, expression vectors containing polynucleotides of the<br>
invention may be designed to contain signal sequences which direct secretion of the<br>
encoded polypeptide through a prokaryotic or eukaryotic cell membrane. Other<br>
recombinant constructions may be used to join sequences encoding a polypeptide of<br>
interest to nucleotide sequence encoding a polypeptide domain which will facilitate<br>
purification of soluble proteins. Such purification facilitating domains include, but are<br>
not limited to, metal chelating peptides such as histidine-tryptophan modules that allow<br>
purification on immobilized metals, protein A domains that allow purification on<br>
immobilized immunoglobulin, and the domain utilized in the FLAGS extension/affinity<br>
purification system (Immunex Corp., Seattle, Wash.). The inclusion of cleavable linker<br>
sequences such as those specific for Factor XA or enterokinase (Invitrogen. San Diego,<br>
Calif.) between the purification domain and the encoded polypeptide may be used to<br>
facilitate purification. One such expression vector provides for expression of a fusion<br>
protein containing a polypeptide of interest and a nucleic acid encoding 6 histidine<br>
residues preceding a thioredoxin or an enterokinase cleavage site. The histidine residues<br>
facilitate purification on IMIAC (immobilized metal ion affinity chromatography) as<br>
described in Porath, J. etal. (1992, Prot. Exp. Purif. 3:263-281) while the enterokinase<br>
cleavage site provides a means for purifying the desired polypeptide from the fusion<br>
protein. A discussion of vectors which contain fusion proteins is provided in Kroll, D. J.<br>
etal. (1993; DNA CellBiol 72:441-453).<br>
In addition to recombinant production methods, polypeptides of the<br>
invention, and fragments thereof, may be produced by direct peptide synthesis using<br>
solid-phase techniques (Merrifield J. (1963) J. Am. Chem. Soc. 55:2149-2154). Protein<br>
synthesis may be performed using manual techniques or by automation. Automated<br>
synthesis may be achieved, for example, using Applied Biosystems 431A Peptide<br>
Synthesizer (Perkin Elmer). Alternatively, various fragments may be chemically<br>
synthesized separately and combined using chemical methods to produce the full length<br>
molecule.<br>
SITE-SPECIFIC MUTAGENESIS<br>
Site-specific mutagenesis is a technique useful in the preparation of<br>
individual peptides, or biologically functional equivalent polypeptides, through specific<br>
mutagenesis of the underlying polynucleotides that encode them. The technique,<br>
well-known to those of skill in the art, further provides a ready ability to prepare and<br>
test sequence variants, for example, incorporating one or more of the foregoing<br>
considerations, by introducing one or more nucleotide sequence changes into the DNA.<br>
Site-specific mutagenesis allows the production of mutants through the use of specific<br>
oligonucleotide sequences which encode the DNA sequence of the desired mutation, as<br>
well as a sufficient number of adjacent nucleotides, to provide a primer sequence of<br>
sufficient size and sequence complexity to form a stable duplex on both sides of the<br>
deletion junction being traversed. Mutations may be employed in a selected<br>
polynucleotide sequence to improve, alter, decrease, modify, or otherwise change the<br>
properties of the polynucleotide itself, and/or alter the properties, activity, composition,<br>
stability, or primary sequence of the encoded polypeptide.<br>
In certain embodiments of the present invention, the inventors<br>
contemplate the mutagenesis of the disclosed polynucleotide sequences to alter one or<br>
more properties of the encoded polypeptide, such as the antigenicity of a polypeptide<br>
vaccine. The techniques of site-specific mutagenesis are well-known in the art, and are<br>
widely used to create variants of both polypeptides and polynucleotides. For example,<br>
site-specific mutagenesis is often used to alter a specific portion of a DNA molecule. In<br>
such embodiments, a primer comprising typically about 14 to about 25 nucleotides or so<br>
in length is employed, with about 5 to about 10 residues on both sides of the junction of<br>
the sequence being altered.<br>
As will be appreciated by those of skill in the art, site-specific<br>
mutagenesis techniques have often employed a phage vector that exists in both a single<br>
stranded and double stranded form. Typical vectors useful in site-directed mutagenesis<br>
include vectors such as the Ml 3 phage. These phage are readily<br>
commercially-available and their use is generally well-known to those skilled in the art.<br>
Double-stranded plasmids are also routinely employed in site directed mutagenesis that<br>
eliminates the step of transferring the gene of interest from a plasmid to a phage.<br>
In general, site-directed mutagenesis in accordance herewith is<br>
performed by first obtaining a single-stranded vector or melting apart of two strands of<br>
a double-stranded vector that includes within its sequence a DNA sequence that<br>
encodes the desired peptide. An oligonucleotide primer bearing the desired mutated<br>
sequence is prepared, generally synthetically. This primer is then annealed with the<br>
single-stranded vector, and subjected to DNA polymerizing enzymes such as E. coli<br>
polymerase I Klenow fragment, in order to complete the synthesis of the mutationbearing<br>
strand. Thus, a heteroduplex is formed wherein one strand encodes the original<br>
non-mutated sequence and the second strand bears the desired mutation. This<br>
heteroduplex vector is then used to transform appropriate cells, such as E. coli cells, and<br>
clones are selected which include recombinant vectors bearing the mutated sequence<br>
arrangement.<br>
The preparation of sequence variants of the selected peptide-encoding<br>
DNA segments using site-directed mutagenesis provides a means of producing<br>
potentially useful species and is not meant to be limiting as there are other ways in<br>
which sequence variants of peptides and the DNA sequences encoding them may be<br>
obtained. For example, recombinant vectors encoding the desired peptide sequence<br>
may be treated with mutagenic agents, such as hydroxylamine, to obtain sequence<br>
variants. Specific details regarding these methods and protocols are found in the<br>
teachings of Maloy etal, 1994; Segal, 1976; Prokop and Bajpai, 1991; Kuby, 1994;<br>
and Maniatis et al, 1982, each incorporated herein by reference, for that purpose.<br>
As used herein, the term "oligonucleotide directed mutagenesis<br>
procedure" refers to template-dependent processes and vector-mediated propagation<br>
which result in an increase in the concentration of a specific nucleic acid molecule<br>
relative to its initial concentration, or hi an increase in the concentration of a detectable<br>
signal, such as amplification. As used herein, the term "oligonucleotide directed<br>
mutagenesis procedure" is intended to refer to a process that involves the<br>
template-dependent extension of a primer molecule. The term template dependent<br>
process refers to nucleic acid synthesis of an RNA or a DNA molecule wherein the<br>
sequence of the newly synthesized strand of nucleic acid is dictated by the well-known<br>
rules of complementary base pairing (see, for example, Watson, 1987). Typically,<br>
vector mediated methodologies involve the introduction of the nucleic acid fragment<br>
into a DNA or RNA vector, the clonal amplification of the vector, and the recovery of<br>
the amplified nucleic acid fragment Examples of such methodologies are provided by<br>
U. S. Patent No. 4,237,224, specifically incorporated herein by reference in its entirety.<br>
POLYNUCLEOTIDE AMPLIFICATION TECHNIQUES<br>
A number of template dependent processes are available to amplify the<br>
target sequences of interest present in a sample. One of the best known amplification<br>
methods is the polymerase chain reaction (PCR™) which is described in detail in U.S.<br>
Patent Nos. 4,683,195, 4,683,202 and 4,800,159, each of which is incorporated herein<br>
by reference in its entirety. Briefly, hi PCR™* two primer sequences are prepared<br>
which are complementary to regions on opposite complementary strands of the target<br>
sequence. An excess of deoxynucleoside triphosphates is added to a reaction mixture<br>
along with a DNA polymerase (e.g., Tag polymerase). If the target sequence is present<br>
in a sample, the primers will bind to the target and the polymerase will cause the<br>
primers to be extended along the target sequence by adding on nucleotides. By raising<br>
and lowering the temperature of the reaction mixture, the extended primers will<br>
dissociate from the target to form reaction products, excess primers will bind to the<br>
target and to the reaction product and the process is repeated. Preferably reverse<br>
transcription and PCR™ amplification procedure may be performed hi order to quantify<br>
the amount of mRNA amplified. Polymerase chain reaction methodologies are well<br>
known in the art.<br>
Another method for amplification is the ligase chain reaction (referred to<br>
as LCR), disclosed in Eur. Pat. Appl. Publ. No. 320,308 (specifically incorporated<br>
herein by reference in its entirety). In LCR, two complementary probe pairs are<br>
prepared, and in the presence of the target sequence, each pair will bind to opposite<br>
complementary strands of the target such that they abut. In the presence of a ligase, the<br>
two probe pairs will link to form a single unit. By temperature cycling, as in PCR™,<br>
bound ligated units dissociate from the target and then serve as "target sequences" for<br>
ligation of excess probe pairs. U.S. Patent No. 4,883,750, incorporated herein by<br>
reference hi its entirety, describes an alternative method of amplification similar to LCR<br>
for binding probe pairs to a target sequence.<br>
Qbeta Replicase, described in PCT Intl. Pat Appl. Publ. No.<br>
PCT/US87/00880, incorporated herein by reference in its entirety, may also be used as<br>
still another amplification method in the present invention. In this method, a replicative<br>
sequence of RNA that has a region complementary to that of a target is added to a<br>
sample in the presence of an RNA polymerase. The polymerase will copy the<br>
replicative sequence that can then be detected.<br>
An isothennal amplification method, in which restriction endonucleases<br>
and ligases are used to achieve the amplification of target molecules that contain<br>
nucleotide 5'-[a-thio]triphosphates in one strand of a restriction site (Walker et al.,<br>
1992, incorporated herein by reference in its entirety), may also be useful in the<br>
amplification of nucleic acids in the present invention.<br>
Strand Displacement Amplification (SDA) is another method of carrying<br>
out isothermal amplification of nucleic acids which involves multiple rounds of strand<br>
displacement and synthesis, i.e. nick translation. A similar method, called Repair Chain<br>
Reaction (RCR) is another method of amplification which may be useful in the present<br>
invention and is involves annealing several probes throughout a region targeted for<br>
amplification, followed by a repair reaction in which only two of the four bases are<br>
present. The other two bases can be added as biotinylated derivatives for easy<br>
detection. A similar approach is used in SDA.<br>
Sequences can also be detected using a cyclic probe reaction (CPR). In<br>
CPR, a probe having a 3' and 5' sequences of non-target DNA and an internal or<br>
"middle" sequence of the target protein specific RNA is hybridized to DNA which is<br>
present in a sample. Upon hybridization, the reaction is treated with RNaseH, and the<br>
products of the probe are identified as distinctive products by generating a signal that is<br>
released after digestion. The original template is annealed to another cycling probe and<br>
the reaction is repeated. Thus, CPR involves amplifying a signal generated by<br>
hybridization of a probe to a target gene specific expressed nucleic acid.<br>
Still other amplification methods described in Great Britain Pat. Appl.<br>
No. 2 202 328, and in PCT Intl. Pat. Appl. Publ. No. PCT/US89/01025, each of which<br>
is incorporated herein by reference in its entirety, may be used in accordance with the<br>
present invention. In the former application, "modified" primers are used in a PCR-<br>
like, template and enzyme dependent synthesis. The primers may be modified by<br>
labeling with a capture moiety (e.g., biotin) and/or a detector moiety (e.g., enzyme). In<br>
the latter application, an excess of labeled probes is added to a sample. In the presence<br>
of the target sequence, the probe binds and is cleaved catalytically. After cleavage, the<br>
target sequence is released intact to be bound by excess probe. Cleavage of the labeled<br>
probe signals the presence of the target sequence.<br>
Other nucleic acid amplification procedures include transcription-based<br>
amplification systems (TAS) (Kwoh et al, 1989; PCT Intl. Pat. Appl. Publ. No. WO<br>
88/10315, incorporated herein by reference in its entirety), including nucleic acid<br>
sequence based amplification (NASBA) and 3SR. In NASBA, the nucleic acids can be<br>
prepared for amplification by standard phenol/chloroform extraction, heat denaturation<br>
of a sample, treatment with lysis buffer and minispin columns for isolation of DNA and<br>
RNA or guanidinium chloride extraction of RNA. These amplification techniques<br>
involve annealing a primer that has sequences specific to the target sequence.<br>
Following polymerization, DNA/RNA hybrids are digested with RNase H while double<br>
stranded DNA molecules are heat-denatured again. In either case the single stranded<br>
DNA is made fully double stranded by addition of second target-specific primer,<br>
followed by polymerization. The double stranded DNA molecules are then multiply<br>
transcribed by a polymerase such as T7 or SP6. In an isothermal cyclic reaction, the<br>
RNAs are reverse transcribed into DNA, and transcribed once again with a polymerase<br>
such as T7 or SP6. The resulting products, whether truncated or complete, indicate<br>
target-specific sequences.<br>
Eur. Pat. Appl. Publ. No. 329,822, incorporated herein by reference in its<br>
entirety, disclose a nucleic acid amplification process involving cyclically synthesizing<br>
single-stranded RNA ("ssRNA"), ssDNA, and double-stranded DNA (dsDNA), which<br>
may be used in accordance with the present invention. The ssRNA is a first template<br>
for a first primer oligonucleotide, which is elongated by reverse transcriptase<br>
(RNA-dependent DNA polymerase). The RNA is then removed from resulting<br>
DNArRNA duplex by the action of ribonuclease H (RNase H, an RNase specific for<br>
RNA in a duplex with either DNA or RNA). The resultant ssDNA is a second template<br>
for a second primer, which also includes the sequences of an RNA polymerase<br>
promoter (exemplified by T7 RNA polymerase) 5' to its homology to its template. This<br>
primer is then extended by DNA polymerase (exemplified by the large "Klenow"<br>
fragment of E. coli DNA polymerase I), resulting as a double-stranded DNA<br>
("dsDNA") molecule, having a sequence identical to that of the original RNA between<br>
the primers and having additionally, at one end, a promoter sequence. This promoter<br>
sequence can be used by the appropriate RNA polymerase to make many RNA copies<br>
of the DNA. These copies can then re-enter the cycle leading to very swift<br>
amplification. With proper choice of enzymes, this amplification can be done<br>
isothermally without addition of enzymes at each cycle. Because of the cyclical nature<br>
of this process, the starting sequence can be chosen to be in the form of either DNA or<br>
RNA.<br>
PCT Intl. Pat. Appl. Publ. No. WO 89/06700, incorporated herein by<br>
reference in its entirety, disclose a nucleic acid sequence amplification scheme based on<br>
the hybridization of a promoter/primer sequence to a target single-stranded DNA<br>
("ssDNA") followed by transcription of many RNA copies of the sequence. This<br>
scheme is not cyclic; i.e. new templates are not produced from the resultant RNA<br>
transcripts. Other amplification methods include "RACE" (Frohman, 1990), and<br>
"one-sided PCR" (Ohara, 1989) which are well-known to those of skill in the art.<br>
Methods based on ligation of two (or more) oligonucleotides in the<br>
presence of nucleic acid having the sequence of the resulting "di-oligonucleotide",<br>
thereby amplifying the di-oligonucleotide (Wu and Dean, 1996, incorporated herein by<br>
reference in its entirety), may also be used in the amplification of DNA sequences of<br>
the present invention.<br>
BIOLOGICAL FUNCTIONAL EQUIVALENTS<br>
Modification and changes may be made in the structure of the<br>
polynucleotides and polypeptides of the present invention and still obtain a functional<br>
molecule that encodes a polypeptide with desirable characteristics. As mentioned<br>
above, it is often desirable to introduce one or more mutations into a specific<br>
polynucleotide sequence. In certain circumstances, the resulting encoded polypeptide<br>
sequence is altered by this mutation, or in other cases, the sequence of the polypeptide<br>
is unchanged by one or more mutations in the encoding polynucleotide..<br>
When it is desirable to alter the amino acid sequence of a polypeptide to<br>
create an equivalent, or even an improved, second-generation molecule, the amino acid<br>
changes may be achieved by changing one or more of the codons of the encoding DNA<br>
sequence, according to Table 1.<br>
For example, certain amino acids may be substituted for other amino<br>
acids in a protein structure without appreciable loss of interactive binding capacity with<br>
structures such as, for example, antigen-binding regions of antibodies or binding sites<br>
on substrate molecules. Since it is the interactive capacity and nature of a protein that<br>
defines that protein's biological functional activity, certain amino acid sequence<br>
substitutions can be made in a protein sequence, and, of course, its underlying DNA<br>
coding sequence, and nevertheless obtain a protein with like properties. It is thus<br>
contemplated by the inventors that various changes may be made in the peptide<br>
sequences of the disclosed compositions, or corresponding DNA sequences which<br>
encode said peptides without appreciable loss of their biological utility or activity.<br>
(Table Removed)<br>
In making such changes, the hydropathic index of amino acids may be<br>
considered. The importance of the hydropathic amino acid index in conferring<br>
interactive biologic function on a protein is generally understood in the art (Kyte and<br>
Doolittle, 1982, incorporated herein by reference). It is accepted that the relative<br>
hydropathic character of the amino acid contributes to the secondary structure of the<br>
resultant protein, which in turn defines the interaction of the protein with other<br>
molecules, for example, enzymes, substrates, receptors, DNA, antibodies, antigens, and<br>
the like. Each amino acid has been assigned a hydropathic index on the basis of its<br>
hydrophobicity and charge characteristics (Kyte and Doolittle, 1982). These values are:<br>
isoleucine (+4.5); valine (+4.2); leucine (+3.8); phenylalanine (+2.8); cysteine/cystine<br>
(+2.5); methionine (+1.9); alanine (+1.8); glycine (-0.4); threonine (-0.7); serine (-<br>
0.8); tryptophan (-0.9); tyrosine (-1.3); proline (-1.6); histidine (-3.2); glutamate (-<br>
3.5); glutamine (-3.5); aspartate (-3.5); asparagine (-3.5); lysine (-3.9); and arginine (-<br>
4.5).<br>
It is known in the art that certain amino acids may be substituted by<br>
other amino acids having a similar hydropathic index or score and still result in a<br>
protein with similar biological activity, i.e. still obtain a biological functionally<br>
equivalent protein. In making such changes, the substitution of amino acids whose<br>
hydropathic indices are within ±2 is preferred, those within ±1 are particularly<br>
preferred, and those within ±0.5 are even more particularly preferred. It is also<br>
understood in the art that the substitution of like amino acids can be made effectively on<br>
the basis of hydrophilicity. U. S. Patent 4,554,101 (specifically incorporated herein by<br>
reference in its entirety), states that the greatest local average hydrophilicity of a<br>
protein, as governed by the hydrophilicity of its adjacent amino acids, correlates with a<br>
biological property of the protein.<br>
As detailed in U. S. Patent 4,554,101, the following hydrophilicity<br>
values have been assigned to amino acid residues: arginine (+3.0); lysine (+3.0);<br>
aspartate (+3.0 ± 1); glutamate (+3.0 ± 1); serine (+0.3); asparagine (+0.2); glutamine<br>
(+0.2); glycine (0); threonine (-0.4); proline (-0.5 ± 1); alanine (-0.5); histidine (-0.5);<br>
cysteine (-1.0); methionine (-1.3); valine (—1.5); leucine (—1.8); isoleucine (—1.8);<br>
tyrosine (-2.3); phenylalanine (-2.5); tryptophan (-3.4). It is understood that an amino<br>
acid can be substituted for another having a similar hydrophilicity value and still obtain<br>
a biologically equivalent, and in particular, an immunologically equivalent protein. In<br>
such changes, the substitution of amino acids whose hydrophilicity values are within ±2<br>
is preferred, those within ±1 are particularly preferred, and those within ±0.5 are even<br>
more particularly preferred.<br>
As outlined above, amino acid substitutions are generally therefore based<br>
on the relative similarity of the amino acid side-chain substituents, for example, their<br>
hydrophobicity, hydrophilicity, charge, size, and the like. Exemplary substitutions that<br>
take various of the foregoing characteristics into consideration are well known to those<br>
of skill in the art and include: arginine and iysine; glutamate and aspartate; serine and<br>
threonine; giutamine and asparagine; and valine, leucine and isoleucine.<br>
In addition, any polynucleotide may be fiirther modified to increase<br>
stability in vivo. Possible modifications include, but are not limited to, the addition of<br>
flanking sequences at the 5' and/or 3' ends; the use of phosphorothioate or 2' O-methyl<br>
rather than phosphodiesterase linkages in the backbone; and/or the inclusion of<br>
nontraditional bases such as inosine, queosine and wybutosine, as well as acetylmethyl-,<br>
thio- and other modified forms of adenine, cytidine, guanine, thymine and<br>
uridine.<br>
IN vivo POLYNUCLEOTIDE DELIVERY TECHNIQUES<br>
In additional embodiments, genetic constructs comprising one or more of<br>
the polynucleotides of the invention are introduced into cells in vivo. This may be<br>
achieved using any of a variety or well known approaches, several of which are<br>
outlined below for the purpose of illustration.<br>
1. ADENOVIRUS<br>
One of the preferred methods for in vivo delivery of one or more nucleic<br>
acid sequences involves the use of an adenovirus expression vector. "Adenovirus<br>
expression vector" is meant to include those constructs containing adenovirus<br>
sequences sufficient to (a) support packaging of the construct and (b) to express a<br>
polynucleotide that has been cloned therein in a sense or antisense orientation. Of<br>
course, in the context of an antisense construct, expression does not require that the<br>
gene product be synthesized.<br>
The expression vector comprises a genetically engineered form of an<br>
adenovirus. Knowledge of the genetic organization of adenovirus, a 36 kb, linear,<br>
double-stranded DNA virus, allows substitution of large pieces of adenoviral DNA with<br>
foreign sequences up to 7 kb (Grunhaus and Horwitz, 1992). In contrast to retrovirus,<br>
the adenoviral infection of host cells does not result in chromosomal integration<br>
because adenoviral DNA can replicate in an episomal manner without potential<br>
genotoxiciry. Also, adenoviruses are structurally stable, and no genome rearrangement<br>
has been detected after extensive amplification. Adenovirus can infect virtually all<br>
epithelial cells regardless of their cell cycle stage. So far, adenoviral infection appears<br>
to be linked only to mild disease such as acute respiratory disease in humans.<br>
Adenovirus is particularly suitable for use as a gene transfer vector<br>
because of its mid-sized genome, ease of manipulation, high titer, wide target-cell range<br>
and high infectivity. Both ends of the viral genome contain 100-200 base pair inverted<br>
repeats (ITRs), which are cis elements necessary for viral DNA replication and<br>
packaging. The early (E) and late (L) regions of the genome contain different<br>
transcription units that are divided by the onset of viral DNA replication. The El<br>
region (El A and E1B) encodes proteins responsible for the regulation of transcription<br>
of the viral genome and a few cellular genes. The expression of the E2 region (E2A<br>
and E2B) results in the synthesis of the proteins for viral DNA replication. These<br>
proteins are involved in DNA replication, late gene expression and host cell shut-off<br>
(Renan, 1990). The products of the late genes, including the majority of the viral capsid<br>
proteins, are expressed only after significant processing of a single primary transcript<br>
issued by the major late promoter (MLP). The MLP, (located at 16.8 m.u.) is<br>
particularly efficient during the late phase of infection, and all the mRNA's issued from<br>
this promoter possess a 5'-tripartite leader (TPL) sequence which makes them preferred<br>
mRNA's for translation.<br>
In a current system, recombinant adenovirus is generated from<br>
homologous recombination between shuttle vector and provirus vector. Due to the<br>
possible recombination between two proviral vectors, wild-type adenovirus may be<br>
generated from this process. Therefore, it is critical to isolate a single clone of virus<br>
from an individual plaque and examine its genomic structure.<br>
Generation and propagation of the current adenovirus vectors, which are<br>
replication deficient, depend on a unique helper cell line, designated 293, which was<br>
transformed from human embryonic kidney cells by AdS DNA fragments and<br>
constitutively expresses El proteins (Graham et a/., 1977). Since the E3 region is<br>
dispensable from the adenovirus genome (Jones and Shenk, 1978), the current<br>
adenovirus vectors, with the help of 293 cells, carry foreign DNA hi either the El, the<br>
D3 or both regions (Graham and Prevec, 1991). In nature, adenovirus can package<br>
approximately 105% of the wild-type genome (Ghosh-Choudhury et al, 1987),<br>
providing capacity for about 2 extra kB of DNA. Combined with the approximately 5.5<br>
kB of DNA that is replaceable in the El and E3 regions, the maximum capacity of the<br>
current adenovirus vector is under 7.5 kB, or about 15% of the total length of the<br>
vector. More than 80% of the adenovirus viral genome remains in the vector backbone<br>
and is the source of vector-borne cytotoxicity. Also, the replication deficiency of the<br>
El-deleted virus is incomplete. For example, leakage of viral gene expression has been<br>
observed with the currently available vectors at high multiplicities of infection (MOI)<br>
(Mulligan, 1993).<br>
Helper cell lines may be derived from human cells such as human<br>
embryonic kidney cells, muscle cells, hematopoietic cells or other human embryonic<br>
mesenchymal or epithelial cells. Alternatively, the helper cells may be derived from the<br>
cells of other mammalian species that are permissive for human adenovirus. Such cells<br>
include, e.g., Vero cells or other monkey embryonic mesenchymal or epithelial cells.<br>
As stated above, the currently preferred helper cell line is 293.<br>
Recently, Racher et al (1995) disclosed improved methods for culturing<br>
293 cells and propagating adenovirus. In one format, natural cell aggregates are grown<br>
by inoculating individual cells into 1 liter siliconized spinner flasks (Techne,<br>
Cambridge, UK) containing 100-200 ml of medium. Following stirring at 40 rpm, the<br>
cell viability is estimated with trypan blue. In another format, Fibra-Cei microcarriers<br>
(Bibby Sterlin, Stone, UK) (5 g/1) is employed as follows. A cell inoculum,<br>
resuspended in 5 ml of medium, is added to the carrier (50 ml) in a 250 ml Erlenmeyer<br>
flask and left stationary, with occasional agitation, for 1 to 4 h. The medium is then<br>
replaced with 50 ml of fresh medium and shaking initiated. For virus production, cells<br>
are allowed to grow to about 80% confluence, after which time the medium is replaced<br>
(to 25% of the final volume) and adenovirus added at an MOI of 0.05. Cultures are left<br>
stationary overnight, following which the volume is increased to 100% and shaking<br>
commenced for another 72 h.<br>
Other than the requirement that the adenovirus vector be replication<br>
defective, or at least conditionally defective, the nature of the adenovirus vector is not<br>
believed to be crucial to the successful practice of the invention. The adenovirus may<br>
be of any of the 42 different known serotypes or subgroups A-F. Adenovirus type 5 of<br>
subgroup C is the preferred starting material in order to obtain a conditional replicationdefective<br>
adenovirus vector for use hi the present invention, since Adenovirus type 5 is<br>
a human adenovirus about which a great deal of biochemical and genetic information is<br>
known, and it has historically been used for most constructions employing adenovirus<br>
as a vector.<br>
As stated above, the typical vector according to the present invention is<br>
replication defective and will not have an adenovirus El region. Thus, it will be most<br>
convenient to introduce the polynucleotide encoding the gene of interest at the position<br>
from which the El-coding sequences have been removed. However, the position of<br>
insertion of the construct within the adenovirus sequences is not critical to the<br>
invention. The polynucleotide encoding the gene of interest may also be inserted in lieu<br>
of the deleted E3 region in E3 replacement vectors as described by Karlsson et al.<br>
(1986) or hi the E4 region where a helper cell line or helper virus complements the E4<br>
defect.<br>
Adenovirus is easy to grow and manipulate and exhibits broad host<br>
range in vitro and in vivo. This group of viruses can be obtained in high liters, e.g., 109-<br>
1011 plaque-forming units per ml, and they are highly infective. The life cycle of<br>
adenovirus does not require integration into the host cell genome. The foreign genes<br>
delivered by adenovirus vectors are episomal and, therefore, have low genotoxicity to<br>
host cells. No side effects have been reported in studies of vaccination with wild-type<br>
adenovirus (Couch et al, 1963; Top et al, 1971), demonstrating then- safety and<br>
therapeutic potential as in vivo gene transfer vectors.<br>
Adenovirus vectors have been used in eukaryotic gene expression<br>
(Levrero et al, 1991; Gomez-Foix et al, 1992) and vaccine development (Grunhaus<br>
and Horwitz, 1992; Graham and Prevec, 1992). Recently* animal studies suggested that<br>
recombinant adenovirus could be used for gene therapy (Stratford-Perricaudet and<br>
Perricaudet, 1991; Stratford-Perricaudet et al, 1990; Rich et al, 1993). Studies hi<br>
administering recombinant adenovirus to different tissues include trachea instillation<br>
(Rosenfeld et al, 1991; Rosenfeld et al, 1992), muscle injection (Ragot et al, 1993),<br>
peripheral intravenous injections (Herz and Gerard, 1993) and stereotactic inoculation<br>
into the brain (Le Gal La Salle et al., 1993).<br>
2. RETROVIRUSES<br>
The retroviruses are a group of single-stranded RNA viruses<br>
characterized by an ability to convert their RNA to double-stranded DNA in infected<br>
cells by a process of reverse-transcription (Coffin, 1990). The resulting DNA then<br>
stably integrates into cellular chromosomes as a provirus and directs synthesis of viral<br>
proteins. The integration results in the retention of the viral gene sequences hi the<br>
recipient cell and its descendants. The retroviral genome contains three genes, gag, pol,<br>
and env that code for capsid proteins, polymerase enzyme, and envelope components,<br>
respectively. A sequence found upstream from the gag gene contains a signal for<br>
packaging of the genome into virions. Two long terminal repeat (LTR) sequences are<br>
present at the 5' and 3' ends of the viral genome. These contain strong promoter and<br>
enhancer sequences and are also required for integration in the host cell genome<br>
(Coffin, 1990).<br>
In order to construct a retroviral vector, a nucleic acid encoding one or<br>
more oligonucleotide or polynucleotide sequences of interest is inserted into the viral<br>
genome in the place of certain viral sequences to produce a virus that is replicationdefective.<br>
In order to produce virions, a packaging cell line containing the gag, pol, and<br>
env genes but without the LTR and packaging components is constructed (Mann et al.,<br>
1983). When a recombinant plasmid containing a cDNA, together with the retroviral<br>
LTR and packaging sequences is introduced into this cell line (by calcium phosphate<br>
precipitation for example), the packaging sequence allows the RNA transcript of the<br>
recombinant plasmid to be packaged into viral particles, which are then secreted into<br>
the culture media (Nicolas and Rubenstein, 1988; Temin, 1986; Mann et al, 1983).<br>
The media containing the recombinant retroviruses is then collected, optionally<br>
concentrated, and used for gene transfer. Retroviral vectors are able to infect a broad<br>
variety of cell types. However, integration and stable expression require the division of<br>
host cells (Paskind etal, 1975).<br>
A novel approach designed to allow specific targeting of retrovirus<br>
vectors was recently developed based on the chemical modification of a retrovirus by<br>
the chemical addition of lactose residues to the viral envelope. This modification could<br>
permit the specific infection of hepatocytes via sialoglycoprotein receptors.<br>
A different approach to targeting of recombinant retroviruses was<br>
designed in which biotinylated antibodies against a retroviral envelope protein and<br>
against a specific cell receptor were used. The antibodies were coupled via the biotin<br>
components by using streptavidin (Roux et al., 1989). Using antibodies against major<br>
histocompatibility complex class I and class II antigens, they demonstrated the infection<br>
of a variety of human cells that bore those surface antigens with an ecotropic virus in<br>
vitro (Roux etal, 1989).<br>
3. ADENO-ASSOCIATED VIRUSES<br>
AAV (Ridgeway, 1988; Hermonat and Muzycska, 1984) is a parovirus,<br>
discovered as a contamination of adenoviral stocks. It is a ubiquitous virus (antibodies<br>
are present in 85% of the US human population) that has not been linked to any disease.<br>
It is also classified as a dependovirus, because its replications is dependent on the<br>
presence of a helper virus, such as adenovirus. Five serotypes have been isolated, of<br>
which AAV-2 is the best characterized. AAV has a single-stranded linear DNA that is<br>
encapsidated into capsid proteins VP1, VP2 and VP3 to form an icosahedral virion of<br>
20 to 24 nm in diameter (Muzyczka and McLaughlin, 1988).<br>
The AAV DNA is approximately 4.7 kilobases long. It contains two<br>
open reading frames and is flanked by two ITRs. There are two major genes in the<br>
AAV genome: rep and cap. The rep gene codes for proteins responsible for viral<br>
replications, whereas cap codes for capsid protein VP1-3. Each ITR forms a T-shaped<br>
hairpin structure. These terminal repeats are the only essential cis components of the<br>
AAV for chromosomal integration. Therefore, the AAV can be used as a vector with<br>
all viral coding sequences removed and replaced by the cassette of genes for delivery.<br>
Three viral promoters have been identified and named p5, pi 9, and p40, according to<br>
their map position. Transcription from p5 and p!9 results hi production of rep proteins,<br>
and transcription from p40 produces the capsid proteins (Hermonat and Muzyczka,<br>
1984).<br>
There are several factors that prompted researchers to study the<br>
possibility of using rAAV as an expression vector One is that the requirements for<br>
delivering a gene to integrate into the host chromosome are surprisingly few. It is<br>
necessary to have the 145-bp ITRs, which are only 6% of the AAV genome. This<br>
leaves room in the vector to assemble a 4.5-kb DNA insertion. While this carrying<br>
capacity may prevent the AAV from delivering large genes, it is amply suited for<br>
delivering the antisense constructs of the present invention.<br>
AAV is also a good choice of delivery vehicles due to its safety. There<br>
is a relatively complicated rescue mechanism: not only wild type adenovirus but also<br>
AAV genes are required to mobilize rAAV. Likewise, AAV is not pathogenic and not<br>
associated with any disease. The removal of viral coding sequences minimizes immune<br>
reactions to viral gene expression, and therefore, rAAV does not evoke an inflammatory<br>
response.<br>
4. OTHER VIRAL VECTORS AS EXPRESSION CONSTRUCTS<br>
Other viral vectors may be employed as expression constructs hi the<br>
present invention for the delivery of oligonucleotide or polynucleotide sequences to a<br>
host cell. Vectors derived from viruses such as vaccinia virus (Ridgeway, 1988;<br>
Coupar et al, 1988), lentiviruses, polio viruses and herpes viruses may be employed.<br>
They offer several attractive features for various mammalian cells (Friedmann, 1989;<br>
Ridgeway, 1988; Coupar etal., 1988; Horwich etal, 1990).<br>
With the recent recognition of defective hepatitis B viruses, new insight<br>
was gained into the structure-function relationship of different viral sequences. In vitro<br>
studies showed that the virus could retain the ability for helper-dependent packaging<br>
and reverse transcription despite the deletion of up to 80% of its genome (Horwich et<br>
al., 1990). This suggested that large portions of the genome could be replaced with<br>
foreign genetic material. The hepatotropism and persistence (integration) were<br>
particularly attractive properties for liver-directed gene transfer. Chang et al (1991)<br>
introduced the chloramphenicol acetyltransferase (CAT) gene into duck hepatitis B<br>
virus genome in the place of the polymerase, surface, and pre-surface coding sequences.<br>
It was cotransfected with wild-type virus into an avion hepatoma cell line. Culture<br>
media containing high titers of the recombinant virus were used to infect primary<br>
duckling hepatocytes. Stable CAT gene expression was detected for at least 24 days<br>
after transfection (Chang et al., 1991).<br>
5. NON-VIRAL VECTORS<br>
In order to effect expression of the oligonucleotide or polynucleotide<br>
sequences of the present invention, the expression construct must be delivered into a<br>
cell. This delivery may be accomplished in vitro, as in laboratory procedures for<br>
transforming cells lines, or in vivo or ex vivo, as in the treatment of certain disease<br>
states. As described above, one preferred mechanism for delivery is via viral infection<br>
where the expression construct is encapsulated in an infectious viral particle.<br>
Once the expression construct has been delivered into the cell the nucleic<br>
acid encoding the desired oligonucleotide or polynucleotide sequences may be<br>
positioned and expressed at different sites. In certain embodiments, the nucleic acid<br>
encoding the construct may be stably integrated into the genome of the cell. This<br>
integration may be in the specific location and orientation via homologous<br>
recombination (gene replacement) or it may be integrated in a random, non-specific<br>
location (gene augmentation). In yet further embodiments, the nucleic acid may be<br>
stably maintained in the cell as a separate, episomal segment of DNA. Such nucleic<br>
acid segments or "episomes" encode sequences sufficient to permit maintenance and<br>
replication independent of or in synchronization with the host cell cycle. How the<br>
expression construct is delivered to a cell and where in the cell the nucleic acid remains<br>
is dependent on the type of expression construct employed.<br>
In certain embodiments of the invention, the expression construct<br>
comprising one or more oligonucleotide or polynucleotide sequences may simply<br>
consist of naked recombinant DNA or plasmids. Transfer of the construct may be<br>
performed by any of the methods mentioned above which physically or chemically<br>
permeabilize the cell membrane. This is particularly applicable for transfer in vitro but<br>
it may be applied to in vivo use as well. Dubensky et al. (1984) successfully injected<br>
polyomavirus DNA in the form of calcium phosphate precipitates into liver and spleen<br>
of adult and newborn mice demonstrating active viral replication and. acute infection.<br>
Benvenisty and Reshef (1986) also demonstrated that direct intraperitoneal injection of<br>
calcium phosphate-precipitated plasmids results in expression of the transfected genes.<br>
It is envisioned that DNA encoding a gene of interest may also be transferred hi a<br>
similar manner in vivo and express the gene product<br>
Another embodiment of the invention for transferring a naked DNA<br>
expression construct into cells may involve particle bombardment. This method<br>
depends on the ability to accelerate DNA-coated microprojectiles to a high velocity<br>
allowing them to pierce cell membranes and enter cells without killing them (Klein et<br>
al, 1987). Several devices for accelerating small particles have been developed. One<br>
such device relies on a high voltage discharge to generate an electrical current, which hi<br>
turn provides the motive force (Yang et al, 1990). The microprojectiles used have<br>
consisted of biologically inert substances such as tungsten or gold beads.<br>
Selected organs including the liver, skin, and muscle tissue of rats and<br>
mice have been bombarded in vivo (Yang et al., 1990; Zelenin et al, 1991). This may<br>
require surgical exposure of the tissue or cells, to eliminate any intervening tissue<br>
between the gun and the target organ, i.e. ex vivo treatment. Again, DNA encoding a<br>
particular gene may be delivered via this method and still be incorporated by the present<br>
invention.<br>
ANTISENSE OLIGONUCLEOTIDES<br>
The end result of the flow of genetic information is the synthesis of<br>
protein. DNA is transcribed by polymerases into messenger RNA and translated on the<br>
ribosome to yield a folded, functional protein. Thus there are several steps along the<br>
route where protein synthesis can be inhibited. The native DNA segment coding for a<br>
polypeptide described herein, as all such mammalian DNA strands, has two strands: a<br>
sense strand and an antisense strand held together by hydrogen bonding. The<br>
messenger RNA coding for polypeptide has the same nucleotide sequence as the sense<br>
DNA strand except that the DNA thymidine is replaced by uridine. Thus, synthetic<br>
antisense nucleotide sequences will bind to a mRNA and inhibit expression of the<br>
protein encoded by that mRNA.<br>
The targeting of antisense oligonucleotides to mRNA is thus one<br>
mechanism to shut down protein synthesis, and, consequently, represents a powerful<br>
and targeted therapeutic approach. For example, the synthesis of polygalactauronase<br>
and the muscarine type 2 acetylcholine receptor are inhibited by antisense<br>
oligonucleotides directed to then- respective mRNA sequences (U. S. Patent 5,739,119<br>
and U. S. Patent 5,759,829, each specifically incorporated herein by reference in its<br>
entirety). Further, examples of antisense inhibition have been demonstrated with the<br>
nuclear protein cyclin, the multiple drug resistance gene (MDG1), ICAM-1, E-selectin,<br>
STK-1, striatal GABAA receptor and human EOF (Jaskulski et al., 1988;<br>
Vasanthakumar and Ahmed, 1989; Peris et al, 1998; U. S. Patent 5,801,154; U. S.<br>
Patent 5,789,573; U. S. Patent 5,718,709 and U. S. Patent 5,610,288, each specifically<br>
incorporated herein by reference in its entirety). Antisense constructs have also been<br>
described that inhibit and can be used to treat a variety of abnormal cellular<br>
proliferations, e.g. cancer (U. S. Patent 5,747,470; U. S. Patent 5,591,317 and U. S.<br>
Patent 5,783,683, each specifically incorporated herein by reference in its entirety).<br>
Therefore, in exemplary embodiments, the invention provides<br>
oligonucleotide sequences that comprise all, or a portion of, any sequence that is<br>
capable of specifically binding to polynucleotide sequence described herein, or a<br>
complement thereof. In one embodiment, the antisense oligonucleotides comprise<br>
DNA or derivatives thereof. In another embodiment, the oligonucleotides comprise<br>
RNA or derivatives thereof. In a third embodiment, the oligonucleotides are modified<br>
DNAs comprising a phosphorothioated modified backbone. In a fourth embodiment,<br>
the oligonucleotide sequences comprise peptide nucleic acids or derivatives thereof. In<br>
each case, preferred compositions comprise a sequence region that is complementary,<br>
and more preferably substantially-complementary, and even more preferably,<br>
completely complementary to one or more portions of polynucleotides disclosed herein.<br>
Selection of antisense compositions specific for a given gene sequence is<br>
based upon analysis of the chosen target sequence (i.e. in these illustrative examples the<br>
rat and human sequences) and determination of secondary structure, Tm, binding<br>
energy, relative stability, and antisense compositions were selected based upon their<br>
relative inability to form dimers, hairpins, or other secondary structures that would<br>
reduce or prohibit specific binding to the target mRNA hi a host cell.<br>
Highly preferred target regions of the mRNA, are those which are at or<br>
near the AUG translation initiation codon, and those sequences which were<br>
substantially complementary to 5' regions of the mRNA. These secondary structure<br>
analyses and target site selection considerations were performed using v.4 of the<br>
OLIGO primer analysis software (Rychlik, 1997) and the BLASTN 2.0.5 algorithm<br>
software (Altschul etal, 1997).<br>
The use of an antisense delivery method employing a short peptide<br>
vector, termed MPG (27 residues), is also contemplated. The MPG peptide contains a<br>
hydrophobic domain derived from the fusion sequence of HIV gp41 and a hydrophilic<br>
domain from the nuclear localization sequence of SV40 T-antigen (Morris et al., 1997).<br>
It has been demonstrated that several molecules of the MPG peptide coat the antisense<br>
oligonucleotides and can be delivered into cultured mammalian cells in less than 1 hour<br>
with relatively high efficiency (90%). Further, the interaction with MPG strongly<br>
increases both the stability of the oligonucleotide to nuclease and the ability to cross the<br>
plasma membrane (Morris et al., 1997).<br>
RlBOZYMES<br>
Although proteins traditionally have been used for catalysis of nucleic<br>
acids, another class of macromolecules has emerged as useful in this endeavor.<br>
Ribozymes are RNA-protein complexes that cleave nucleic acids in a site-specific<br>
fashion. Ribozymes have specific catalytic domains that possess endonuclease activity<br>
(Kim and Cech, 1987; Gerlach etal., 1987; Forster and Symons, 1987). For example, a<br>
large number of ribozymes accelerate phosphoester transfer reactions with a high<br>
degree of specificity, often cleaving only one of several phosphoesters in an<br>
oligonucleotide substrate (Cech et al., 1981; Michel and Westhof, 1990; Reinhold-<br>
Hurek and Shub, 1992). This specificity has been attributed to the requirement that the<br>
substrate bind via specific base-pairing interactions to the internal guide sequence<br>
("IGS") of the ribozyme prior to chemical reaction.<br>
Ribozyme catalysis has primarily been observed as part of sequencespecific<br>
cleavage/ligation reactions involving nucleic acids (Joyce, 1989; Cech et al,<br>
1981). For example, U. S. Patent No. 5,354,855 (specifically incorporated herein by<br>
reference) reports that certain ribozymes can act as endonucleases with a sequence<br>
specificity greater than that of known ribonucleases and approaching that of the DNA<br>
restriction enzymes. Thus, sequence-specific ribozyme-mediated inhibition of gene<br>
expression may be particularly suited to therapeutic applications (Scanlon et al, 1991;<br>
Sarver et al., 1990). Recently, it was reported that ribozymes elicited genetic changes<br>
in some cells lines to which they were applied; the altered genes included the oncogenes<br>
H-ras, c-fos and genes of HIV. Most of this work involved the modification of a target<br>
mRNA, based on a specific mutant codon that is cleaved by a specific ribozyme.<br>
Six basic varieties of naturally-occurring enzymatic RNAs are known<br>
presently. Each can catalyze the hydrolysis of RNA phosphodiester bonds in trans (and<br>
thus can cleave other RNA molecules) under physiological conditions. In general,<br>
enzymatic nucleic acids act by first binding to a target RNA. Such binding occurs<br>
through the target binding portion of a enzymatic nucleic acid which is held in close<br>
proximity to an enzymatic portion of the molecule that acts to cleave the target RNA.<br>
Thus, the enzymatic nucleic acid first recognizes and then binds a target RNA through<br>
complementary base-pairing, and once bound to the correct site, acts enzymatically to<br>
cut the target RNA. Strategic cleavage of such a target RNA will destroy its ability to<br>
direct synthesis of an encoded protein. After an enzymatic nucleic acid has bound and<br>
cleaved its RNA target, it is released from that RNA to search for another target and can<br>
repeatedly bind and cleave new targets.<br>
The enzymatic nature of a ribozyme is advantageous over many<br>
technologies, such as antisense technology (where a nucleic acid molecule simply binds<br>
to a nucleic acid target to block its translation) since the concentration of ribozyme<br>
necessary to affect a therapeutic treatment is lower than that of an antisense<br>
oligonucleotide. This advantage reflects the ability of the ribozyme to act<br>
enzymatically. Thus, a single ribozyme molecule is able to cleave many molecules of<br>
target RNA. In addition, the ribozyme is a highly specific inhibitor, with the specificity<br>
of inhibition depending not only on the base pairing mechanism of binding to the target<br>
RNA, but also on the mechanism of target RNA cleavage. Single mismatches, or basesubstitutions,<br>
near the site of cleavage can completely eliminate catalytic activity of a<br>
ribozyme. Similar mismatches in antisense molecules do not prevent their action<br>
(Woolf et al, 1992). Thus, the specificity of action of a ribozyme is greater than that of<br>
an antisense oligonucleotide binding the same RNA site.<br>
The enzymatic nucleic acid molecule may be formed in a hammerhead,<br>
hairpin, a hepatitis 5 virus, group I intron or RNaseP RNA (in association with an RNA<br>
guide sequence) or Neurospora VS RNA motif. Examples of hammerhead motifs are<br>
described by Rossi etal. (1992). Examples of hairpin motifs are described by Hampel<br>
etal (Eur. Pat. Appl. Publ. No. EP 0360257), Hampel and Tritz (1989), Hampel etal.<br>
(1990) and U. S. Patent 5,631,359 (specifically incorporated herein by reference). An<br>
example of the hepatitis 8 virus motif is described by Perrotta and Been (1992); an<br>
example of the RNaseP motif is described by Guerrier-Takada etal. (1983);<br>
Neurospora VS RNA ribozyme motif is described by Collins (Saville and Collins,<br>
1990; Saville and Collins, 1991; Collins and Olive, 1993); and an example of the Group<br>
I intron is described in (U. S. Patent 4,987,071, specifically incorporated herein by<br>
reference). All that is important in an enzymatic nucleic acid molecule of this invention<br>
is that it has a specific substrate binding site which is complementary to one or more of<br>
the target gene RNA regions, and that it have nucleotide sequences within or<br>
surrounding that substrate binding site which impart an RNA cleaving activity to the<br>
molecule. Thus the ribozyme constructs need not be limited to specific motifs<br>
mentioned herein.<br>
In certain embodiments, it may be important to produce enzymatic<br>
cleaving agents which exhibit a high degree of specificity for the RNA of a desired<br>
target, such as one of the sequences disclosed herein. The enzymatic nucleic acid<br>
molecule is preferably targeted to a highly conserved sequence region of a target<br>
mRNA. Such enzymatic nucleic acid molecules can be delivered exogenously to<br>
specific cells as required. Alternatively, the ribozymes can be expressed from DNA or<br>
RNA vectors that are delivered to specific cells.<br>
Small enzymatic nucleic acid motifs (e.g., of the hammerhead or the<br>
hairpin structure) may also be used for exogenous delivery. The simple structure of<br>
these molecules increases the ability of the enzymatic nucleic acid to invade targeted<br>
regions of the mRNA structure. Alternatively, catalytic RNA molecules can be<br>
expressed within cells from eukaryotic promoters (e.g., Scanlon etaL, 1991; Kashani-<br>
Sabet etal, 1992; Dropulic etaL, 1992; Weerasinghe etaL, 1991; Ojwang etal, 1992;<br>
Chen etal., 1992; Sarver etal, 1990). Those skilled hi the art realize that any<br>
ribozyme can be expressed in eukaryotic cells from the appropriate DNA vector. The<br>
activity of such ribozymes can be augmented by their release from the primary<br>
transcript by a second ribozyme (Int. Pat. Appl. Publ. No. WO 93/23569, and Int. Pat.<br>
Appl. Publ. No. WO 94/02595, both hereby incorporated by reference; Ohkawa etal.,<br>
1992; Taira etal, 1991; and Ventura etal, 1993).<br>
Ribozymes may be added directly, or can be complexed with cationic<br>
lipids, lipid complexes, packaged within liposomes, or otherwise delivered to target<br>
cells. The RNA or RNA complexes can be locally administered to relevant tissues ex<br>
vivo, or in vivo through injection, aerosol inhalation, infusion pump or stent, with or<br>
without their incorporation in biopolymers.<br>
Ribozymes may be designed as described in Int. Pat. Appl. Publ. No.<br>
WO 93/23569 and Int. Pat. Appl. Publ. No. WO 94/02595, each: specifically<br>
incorporated herein by reference) and synthesized to be tested in vitro and in vivo, as<br>
described. Such ribozymes can also be optimized for delivery. While specific<br>
examples are provided, those in the art will recognize that equivalent RNA targets hi<br>
other species can be utilized when necessary.<br>
Hammerhead or hairpin ribozymes may be individually analyzed by<br>
computer folding (Jaeger etal., 1989) to assess whether the ribozyme sequences fold<br>
into the appropriate secondary structure. Those ribozymes with unfavorable<br>
intramolecular interactions between the binding arms and the catalytic core are<br>
eliminated from consideration. Varying binding arm lengths can be chosen to optimize<br>
activity. Generally, at least 5 or so bases on each arm are able to bind to, or otherwise<br>
interact with, the target RNA.<br>
Ribozymes of the hammerhead or hairpin motif may be designed to<br>
anneal to various sites in the mRNA message, and can be chemically synthesized. The<br>
method of synthesis used follows the procedure for normal RNA synthesis as described<br>
in Usman et al. (1987) and in Scaringe et al. (1990) and makes use of common nucleic<br>
acid protecting and coupling groups, such as dimethoxytrityl at the 5'-end, and<br>
phosphoramidites at the 3'-end. Average stepwise coupling yields are typically &gt;98%.<br>
Hairpin ribozymes may be synthesized in two parts and annealed to reconstruct an<br>
active ribozyme (Chowrira and Burke, 1992). Ribozymes may be modified extensively<br>
to enhance stability by modification with nuclease resistant groups, for example, 2'-<br>
amino, 2'-C-allyl, 2'-flouro, 2'-o-methyl, 2'-H (for a review see e.g., Usman and<br>
Cedergren, 1992). Ribozymes may be purified by gel electrophoresis using general<br>
methods or by high pressure liquid chromatography and resuspended in water.<br>
Ribozyme activity can be optimized by altering the length of the<br>
ribozyme binding arms, or chemically synthesizing ribozymes with modifications that<br>
prevent then* degradation by serum ribonucleases (see e.g., Int. Pat. Appl. Publ. No.<br>
WO 92/07065; Perrault et al, 1990; Pieken etal., 1991; Usman and Cedergren, 1992;<br>
Int. Pat. Appl. Publ. No. WO 93/15187; Int. Pat. Appl. Publ. No. WO 91/03162; Eur.<br>
Pat. Appl. Publ. No. 92110298.4; U. S. Patent 5,334,711; and Int. Pat. Appl. Publ. No.<br>
WO 94/13688, which describe various chemical modifications that can be made to the<br>
sugar moieties of enzymatic RNA molecules), modifications which enhance their<br>
efficacy in cells, and removal of stem II bases to shorten RNA synthesis times and<br>
reduce chemical requirements.<br>
Sullivan etal. (Int. Pat. Appl. Publ. No. WO 94/02595) describes the<br>
general methods for delivery of enzymatic RNA molecules. Ribozymes may be<br>
administered to cells by a variety of methods known to those familiar to the art,<br>
including, but not restricted to, encapsulation hi liposomes, by iontophoresis, or by<br>
incorporation into other vehicles, such as hydrogels, cyclodextrins, biodegradable<br>
nanocapsules, and bioadhesive microspheres. For some indications, ribozymes may be<br>
directly delivered ex vivo to cells or tissues with or without the aforementioned<br>
vehicles. Alternatively, the RNA/vehicle combination may be locally delivered by<br>
direct inhalation, by direct injection or by use of a catheter, infusion pump or stent<br>
Other routes of delivery include, but are not limited to, intravascular, intramuscular,<br>
subcutaneous or joint injection, aerosol inhalation, oral (tablet or pill form), topical,<br>
systemic, ocular, intraperitoneal and/or intrathecal delivery. More detailed descriptions<br>
of ribozyme delivery and administration are provided in Int. Pat. Appl. Publ. No. WO<br>
94/02595 and Int. Pat. Appl. Publ. No. WO 93/23569, each specifically incorporated<br>
herein by reference.<br>
Another means of accumulating high concentrations of a ribozyme(s)<br>
within cells is to incorporate the ribozyme-encoding sequences into a DNA expression<br>
vector. Transcription of the ribozyme sequences are driven from a promoter for<br>
eukaryotic RNA polymerase I (pol I), RNA polymerase n (pol II), or RNA polymerase<br>
III (pol III). Transcripts from pol II or pol III promoters will be expressed at high levels<br>
in all cells; the levels of a given pol II promoter in a given cell type will depend on the<br>
nature of the gene regulatory sequences (enhancers, silencers, etc.) present nearby.<br>
Prokaryotic RNA polymerase promoters may also be used, providing that the<br>
prokaryotic RNA polymerase enzyme is expressed in the appropriate cells (Elroy-Stein<br>
and Moss, 1990; Gao and Huang, 1993; Lieber etal, 1993; Zhou etal, 1990).<br>
Ribozymes expressed from such promoters can function in mammalian cells (e.g.<br>
Kashani-Saber etal., 1992; Ojwang etal., 1992; Chen etal., 1992; Yu etal., 1993;<br>
L'Huillier etal, 1992; Lisziewicz etal, 1993). Such transcription units can be<br>
incorporated into a variety of vectors for introduction into mammalian cells, including<br>
but not restricted to, plasmid DNA vectors, viral DNA vectors (such as adenovirus or<br>
adeno-associated vectors), or viral RNA vectors (such as retroviral, semliki forest virus,<br>
sindbis virus vectors).<br>
Ribozymes may be used as diagnostic tools to examine genetic drift and<br>
mutations within diseased cells. They can also be used to assess levels of the target<br>
RNA molecule. The close relationship between ribozyme activity and the structure of<br>
the target RNA allows the detection of mutations in any region of the molecule which<br>
alters the base-pairing and three-dimensional structure of the target RNA. By using<br>
multiple ribozymes, one may map nucleotide changes which are important to RNA<br>
structure and function in vitro, as well as in cells and tissues. Cleavage of target RNAs<br>
with ribozymes may be used to inhibit gene expression and define the role (essentially)<br>
of specified gene products in the progression of disease. In this manner, other genetic<br>
targets may be defined as important mediators of the disease. These studies will lead to<br>
better treatment of the disease progression by affording the possibility of combinational<br>
therapies (e.g., multiple ribozymes targeted to different genes, ribozymes coupled with<br>
known small molecule inhibitors, or intermittent treatment with combinations of<br>
ribozymes and/or other chemical or biological molecules). Other in vitro uses of<br>
ribozymes are well known in the art, and include detection of the presence of mRNA<br>
associated with an IL-5 related condition. Such RNA is detected by determining the<br>
presence of a cleavage product after treatment with a ribozyme using standard<br>
methodology.<br>
PEPTIDE NUCLEIC ACIDS<br>
In certain embodiments, the inventors contemplate the use of peptide<br>
nucleic acids (PNAs) in the practice of the methods of the invention. PNA is a DNA<br>
mimic in which the nucleobases are attached to a pseudopeptide backbone (Good and<br>
Nielsen, 1997). PNA is able to be utilized in a number methods that traditionally have<br>
used RNA or DNA. Often PNA sequences perform better in techniques than the<br>
corresponding RNA or DNA sequences and have utilities that are not inherent to RNA<br>
or DNA. A review of PNA including methods of making, characteristics of, and<br>
methods of using, is provided by Corey (1997) and is incorporated herein by reference.<br>
As such, in certain embodiments, one may prepare PNA sequences that are<br>
complementary to one or more portions of the ACE mRNA sequence, and such PNA<br>
compositions may be used to regulate, alter, decrease, or reduce the translation of ACEspecific<br>
mRNA, and thereby alter the level of ACE activity in a host cell to which such<br>
PNA compositions have been administered.<br>
PNAs have 2-aminoethyl-glycine linkages replacing the normal<br>
phosphodiester backbone of DNA (Nielsen et al, 1991; Hanvey et al., 1992; Hyrup and<br>
Nielsen, 1996; Neilsen, 1996). This chemistry has three important consequences:<br>
firstly, in contrast to DNA or phosphorothioate oligonucleotides, PNAs are neutral<br>
molecules; secondly, PNAs are achiral, which avoids the need to develop a<br>
stereoselective synthesis; and thirdly, PNA synthesis uses standard Boc (Dueholm et<br>
al., 1994) or Fmoc (Thomson et al., 1995) protocols for solid-phase peptide synthesis,<br>
although other methods, including a modified Merrifield method, have been used<br>
(Christenseneffl/., 1995).<br>
80<br>
PNA monomers or ready-made oligomers are commercially available<br>
from PerSeptive Biosystems (Framingham, MA). PNA syntheses by either Boc or<br>
Fmoc protocols are straightforward using manual or automated protocols (Norton et al.,<br>
1995). The manual protocol lends itself to the production of chemically modified PNAs<br>
or the simultaneous synthesis of families of closely related PNAs.<br>
As with peptide synthesis, the success of a particular PNA synthesis will<br>
depend on the properties of the chosen sequence. For example, while in theory PNAs<br>
can incorporate any combination of nucleotide bases, the presence of adjacent purines<br>
can lead to deletions of one or more residues in the product. In expectation of this<br>
difficulty, it is suggested that, in producing PNAs with adjacent purines, one should<br>
repeat the coupling of residues likely to be added inefficiently. This should be followed<br>
by the purification of PNAs by reverse-phase high-pressure liquid chromatography<br>
(Norton et al, 1995) providing yields and purity of product similar to those observed<br>
during the synthesis of peptides.<br>
Modifications of PNAs for a given application may be accomplished by<br>
coupling amino acids during solid-phase synthesis or by attaching compounds that<br>
contain a carboxylic acid group to the exposed N-terminal amine. Alternatively, PNAs<br>
can be modified after synthesis by coupling to an introduced lysine or cysteine. The<br>
ease with which PNAs can be modified facilitates optimization for better solubility or<br>
for specific functional requirements. Once synthesized, the identity of PNAs and thenderivatives<br>
can be confirmed by mass spectrometry. Several studies have made and<br>
utilized modifications.of PNAs (Norton et al, 1995; Haaima et al, 1996; Stetsenko et<br>
al, 1996; Petersen et al, 1995; Ulmann et al, 1996; Koch et al, 1995; Orum et al,<br>
1995; Footer et al, 1996; Griffith et al, 1995; Kremsky et al, 1996; Pardridge et al,<br>
1995; Boffa et al, 1995; Landsdorp et al, 1996; Gambacorti-Passerini et al, 1996;<br>
Armitage et al, 1997; Seeger et al, 1997; Ruskowski et al, 1997). U.S. Patent No.<br>
5,700,922 discusses PNA-DNA-PNA chimeric molecules and their uses in diagnostics,<br>
modulating protein in organisms, and treatment of conditions susceptible to<br>
therapeutics.<br>
In contrast to DNA and RNA, which contain negatively charged<br>
linkages, the PNA backbone is neutral. In spite of this dramatic alteration, PNAs<br>
recognize complementary DNA and RNA by Watson-Crick pairing (Egholm et at,<br>
1993), validating the initial modeling by Nielsen et al. (1991). PNAs lack 3' to 5'<br>
polarity and can bind in either parallel or antiparallel fashion, with the antiparallel mode<br>
being preferred (Egholm et al, 1993).<br>
Hybridization of DNA oligonucleotides to DNA and RNA is<br>
destabilized by electrostatic repulsion between the negatively charged phosphate<br>
backbones of the complementary strands. By contrast, the absence of charge repulsion<br>
hi PNA-DNA or PNA-RNA duplexes increases the melting temperature (rm) and<br>
reduces the dependence of Tm on the concentration of mono- or divalent cations<br>
(Nielsen et al., 1991). The enhanced rate and affinity of hybridization are significant<br>
because they are responsible for the surprising ability of PNAs to perform strand<br>
invasion of complementary sequences within relaxed double-stranded DNA. In<br>
addition, the efficient hybridization at inverted repeats suggests that PNAs can<br>
recognize secondary structure effectively within double-stranded DNA. Enhanced<br>
recognition also occurs with PNAs immobilized on surfaces, and Wang et al. have<br>
shown that support-bound PNAs can be used to detect hybridization events (Wang et<br>
al, 1996).<br>
One might expect that tight binding of PNAs to complementary<br>
sequences would also increase binding to similar (but not identical) sequences, reducing<br>
the sequence specificity of PNA recognition. As with DNA hybridization, however,<br>
selective recognition can be achieved by balancing oligomer length and incubation<br>
temperature. Moreover, selective hybridization of PNAs is encouraged by PNA-DNA<br>
hybridization being less tolerant of base mismatches than DNA-DNA hybridization.<br>
For example, a single mismatch within a 16 bp PNA-DNA duplex can reduce the Tm by<br>
up to 15°C (Egholm et al, 1993). This high level of discrimination has allowed the<br>
development of several PNA-based strategies for the analysis of point mutations (Wang<br>
et al, 1996; Carlsson et al, 1996; Thiede et al, 1996; Webb and Hurskainen, 1996;<br>
Perry-O'Keefeeffl/., 1996).<br>
High-affinity binding provides clear advantages for molecular<br>
recognition and the development of new applications for PNAs. For example, 11-13<br>
nucleotide PNAs inhibit the activity of telomerase, a ribonucleo-protein that extends<br>
82<br>
telomere ends using an essential RNA template, while the analogous DNA oligomers<br>
do not (Norton et al., 1996).<br>
Neutral PNAs are more hydrophobic than analogous DNA oligomers,<br>
and this can lead to difficulty solubilizing them at neutral pH, especially if the PNAs<br>
have a high purine content or if they have the potential to form secondary structures.<br>
Their solubility can be enhanced by attaching one or more positive charges to the PNA<br>
termini (Nielsen et al, 1991).<br>
Findings by Allfrey and colleagues suggest that strand invasion will<br>
occur spontaneously at sequences within chromosomal DNA (Boffa et al., 1995; Boffa<br>
et al., 1996). These studies targeted PNAs to triplet repeats of the nucleotides CAG and<br>
used this recognition to purify transcriptionally active DNA (Boffa et al, 1995) and to<br>
inhibit transcription (Boffa et at, 1996). This result suggests that if PNAs can be<br>
delivered within cells then they will have the potential to be general sequence-specific<br>
regulators of gene expression. Studies and reviews concerning the use of PNAs as<br>
antisense and anti-gene agents include Nielsen et al. (1993b), Hanvey et al (1992), and<br>
Good and Nielsen (1997). Koppelhus et al. (1997) have used PNAs to inhibit HTV-1<br>
inverse transcription, showing that PNAs may be used for antiviral therapies.<br>
Methods of characterizing the antisense binding properties of PNAs are<br>
discussed in Rose (1993) and Jensen et al (1997). Rose uses capillary gel<br>
electrophoresis to determine binding of PNAs to their complementary oligonucleotide,<br>
measuring the relative binding kinetics and stoichiometry. Similar types of<br>
measurements were made by Jensen et al. using BIAcore™ technology.<br>
Other applications of PNAs include use in DNA strand invasion (Nielsen<br>
et al, 1991), antisense inhibition (Hanvey et al, 1992), mutational analysis (Orum et<br>
al, 1993), enhancers of transcription (Mollegaard etal, 1994), nucleic acid purification<br>
(Orum et al, 1995), isolation of transcriptionally active genes (Boffa et al, 1995),<br>
blocking of transcription factor binding (Vickers et al, 1995), genome cleavage<br>
(Veselkov et al, 1996), biosensors (Wang et al, 1996), in situ hybridization (Thisted et<br>
al, 1996), and in a alternative to Southern blotting (Perry-O'Keefe, 1996).<br>
POLYPEPTIDE COMPOSITIONS<br>
The present invention, in other aspects, provides polypeptide<br>
compositions. Generally, a polypeptide of the invention will be an isolated polypeptide<br>
(or an epitope, variant, or active fragment thereof) derived from a mammalian species.<br>
Preferably, the polypeptide is encoded by a polynucleotide sequence disclosed herein or<br>
a sequence which hybridizes under moderately stringent conditions to a polynucleotide<br>
sequence disclosed herein. Alternatively, the polypeptide may be defined as a<br>
polypeptide which comprises a contiguous amino acid sequence from an amino acid<br>
sequence disclosed herein, or which polypeptide comprises an entire amino acid<br>
sequence disclosed herein.<br>
In the present invention, a polypeptide composition is also understood to<br>
comprise one or more polypeptides that are immunologically reactive with antibodies<br>
generated against a polypeptide of the invention, particularly a polypeptide having the<br>
amino acid sequence disclosed in SEQ ID NO: 786, 787, 791, 793, 795,797-799, 806<br>
or 809, or to active fragments, or to variants or biological functional equivalents<br>
thereof.<br>
Likewise, a polypeptide composition of the present invention is<br>
understood to comprise one or more polypeptides that are capable of eliciting antibodies<br>
that are immunologically reactive with one or more polypeptides encoded by one or<br>
more contiguous nucleic acid sequences contained hi SEQ ID NO: 1,11-13, 15,20, 23-<br>
27,29,30,33,34,39,41,43-46,51, 52,57, 58, 60,62,65-67, 69-71, 74,76, 79, 80, 84,<br>
86, 89-92, 95, 97, 98, 101, 110, 111, 113-119, 121-128, 130-134, 136, 138, 139, 141,<br>
143, 146-151, 153,154,157-160, 162-164, 167-178, 180, 181,183, 186-190, 192, 193,<br>
195-220,224,226-231,234,236,237,240,241,244-246,248,254,255,261, 262,266,<br>
270, 275, 280, 282, 283, 288, 289, 290, 292, 295, 301, 303, 304, 309, 311, 341-782,<br>
784, 785, 790, 792,794, 796, 800-804, 807, 808 and 810-826, or to active fragments, or<br>
to variants thereof, or to one or more nucleic acid sequences which hybridize to one or<br>
more of these sequences under conditions of moderate to high stringency. Particularly<br>
illustrative polypeptides include the amino acid sequences disclosed in SEQ ID NO:<br>
786, 787, 791, 793,795, 797-799, 806, 809 and 827.<br>
84<br>
As used herein, an active fragment of a polypeptide includes a whole or<br>
a portion of a polypeptide which is modified by conventional techniques, e.g.,<br>
mutagenesis, or by addition, deletion, or substitution, but which active fragment<br>
exhibits substantially the same structure function, antigenicity, etc., as a polypeptide as<br>
described herein.<br>
In certain illustrative embodiments, the polypeptides of the invention<br>
will comprise at least an immunogenic portion of a lung tumor protein or a variant<br>
thereof, as described herein. As noted above, a " lung tumor protein" is a protein that is<br>
expressed by lung tumor cells. Proteins that are lung tumor proteins also react<br>
detectably within an immunoassay (such as an ELISA) with antisera from a patient with<br>
lung cancer. Polypeptides as described herein may be of any length. Additional<br>
sequences derived from the native protein and/or heterologous sequences may be<br>
present, and such sequences may (but need not) possess further immunogenic or<br>
antigenic properties.<br>
An "immunogenic portion," as used herein is a portion of a protein that<br>
is recognized (i.e., specifically bound) by a B-cell and/or T-cell surface antigen<br>
receptor. Such immunogenic portions generally comprise at least 5 amino acid<br>
residues, more preferably at least 10, and still more preferably at least 20 amino acid<br>
residues of a lung tumor protein or a variant thereof. Certain preferred immunogenic<br>
portions include peptides in which an N-terminal leader sequence and/or<br>
transmembrane domain have been deleted. Other preferred immunogenic portions may<br>
contain a small N- and/or C-terminal deletion (e.g., 1-30 amino acids, preferably 5-15<br>
amino acids), relative to the mature protein.<br>
Immunogenic portions may generally be identified using well known<br>
techniques, such as those summarized in Paul, Fundamental Immunology, 3rd ed., 243-<br>
247 (Raven Press, 1993) and references cited therein. Such techniques include<br>
screening polypeptides for the ability to react with antigen-specific antibodies, antisera<br>
and/or T-cell lines or clones. As used herein, antisera and antibodies are "antigenspecific"<br>
if they specifically bind to an antigen (/.., they react with the protein in an<br>
ELISA or other immunoassay, and do not react detectably with unrelated proteins).<br>
Such antisera and antibodies may be prepared as described herein, and using well<br>
known techniques. An immunogenic portion of a native lung tumor protein is a portion<br>
that reacts with such antisera and/or T-cells at a level that is not substantially less than<br>
the reactivity of the full length polypeptide (e.g., in an ELISA and/or T-cell reactivity<br>
assay). Such immunogenic portions may react within such assays at a level that is<br>
similar to or greater than the reactivity of the full length polypeptide. Such screens may<br>
generally be performed using methods well known to those of ordinary skill in the art,<br>
such as those described in Harlow and Lane, Antibodies: A Laboratory Manual, Cold<br>
Spring Harbor Laboratory, 1988. For example, a polypeptide may be immobilized qn a<br>
solid support and contacted with patient sera to allow binding of antibodies within the<br>
sera to the immobilized polypeptide. Unbound sera may then be removed and bound<br>
antibodies detected using, for example, 125I-labeled Protein A.<br>
As noted above, a composition may comprise a variant of a native lung<br>
tumor protein. A polypeptide "variant," as used herein, is a polypeptide that differs<br>
from a native lung tumor protein in one or more substitutions, deletions, additions<br>
and/or insertions, such that the immunogenicity of the polypeptide is not substantially<br>
diminished. In other words, the ability of a variant to react with antigen-specific<br>
antisera may be enhanced or unchanged, relative to the native protein, or may be<br>
diminished by less than 50%, and preferably less than 20%, relative to the native<br>
protein. Such variants may generally be identified by modifying one of the above<br>
polypeptide sequences and evaluating the reactivity of the modified polypeptide with<br>
antigen-specific antibodies or antisera as described herein. Preferred variants include<br>
those in which one or more portions, such as an N-terminal leader sequence or<br>
transmembrane domain, have been removed. Other preferred variants include variants<br>
in which a small portion (e.g., 1-30 amino acids, preferably 5-15 amino acids) has been<br>
removed from the N- and/or C-terminal of the mature protein.<br>
Polypeptide variants encompassed by the present invention include those<br>
exhibiting at least about 70%, 75%, 80%, 85%, 90%, 91%, 92%, 93%, 94%, 95%, 96%,<br>
97%, 98%, or 99% or more identity (determined as described above) to the<br>
polypeptides disclosed herein.<br>
Preferably, a variant contains conservative substitutions. A<br>
"conservative substitution" is one in which an amino acid is substituted for another<br>
amino acid that has similar properties, such that one skilled in the art of peptide<br>
chemistry would expect the secondary structure and hydropathic-nature of the<br>
polypeptide to be substantially unchanged. Amino acid substitutions may generally be<br>
made on the basis of similarity in polarity, charge, solubility, hydrophobicity,<br>
hydrophilicity and/or the amphipathic nature of the residues. For example, negatively<br>
charged amino acids include aspartic acid and glutamic acid; positively charged amino<br>
acids include lysine and arginine; and amino acids with uncharged polar head groups<br>
having similar hydrophilicity values include leucine, isoleucine and valine; glycine and<br>
alanine; asparagine and glutamine; and serine, threonine, phenylalanine and tyrosine.<br>
Other groups of amino acids that may represent conservative changes include: (I) ala,<br>
pro, gly, glu, asp, gin, asn, ser, thr; (2) cys, ser, tyr, thr; (3) val, ile, leu, met, ala, phe;<br>
(4) lys, arg, his; and (5) phe, tyr, tip, his. A variant may also, or alternatively, contain<br>
nonconservative changes. In a preferred embodiment, variant polypeptides differ from<br>
a native sequence by substitution, deletion or addition of five amino acids or fewer.<br>
Variants may also (or alternatively) be modified by, for example, the deletion or<br>
addition of amino acids that have minimal influence on the immunogenicity, secondary<br>
structure and hydropathic nature of the polypeptide.<br>
As noted above, polypeptides may comprise a Signal (or leader)<br>
sequence at the N-terminal end of the protein, which co-translationally or posttranslationally<br>
directs transfer of the protein. The polypeptide may also be conjugated<br>
to a linker or other sequence for ease of synthesis, purification or identification of the<br>
polypeptide (e.g., poly-His), or to enhance binding of the polypeptide to a solid support.<br>
For example, a polypeptide may be conjugated to an immunoglobulin Fc region.<br>
Polypeptides may be prepared using any of a variety of well known<br>
techniques. Recombinant polypeptides encoded by DNA sequences as described above<br>
may be readily prepared from the DNA sequences using any of a variety of expression<br>
vectors known to those of ordinary skill in the art. Expression may be achieved in any<br>
appropriate host cell that has been transformed or transfected with an expression vector<br>
containing a DNA molecule that encodes a recombinant polypeptide. Suitable host<br>
cells include prokaryotes, yeast, and higher eukaryotic cells, such as mammalian cells<br>
and plant cells. Preferably, the host cells employed are E. coli, yeast or a mammalian<br>
cell line such as COS or CHO. Supernatants from suitable host/vector systems which<br>
secrete recombinant protein or polypeptide into culture media may be first concentrated<br>
using a commercially available filter. Following concentration, the concentrate may be<br>
applied to a suitable purification matrix such as an affinity matrix or an ion exchange<br>
resin. Finally, one or more reverse phase HPLC steps can be employed to further purify<br>
a recombinant polypeptide.<br>
Portions and other variants having less than about 100 ammo acids, and<br>
generally less than about 50 amino acids, may also be generated by synthetic means,<br>
using techniques well known to those of ordinary skill in the art. For example, such<br>
polypeptides may be synthesized using any of the commercially available solid-phase<br>
techniques, such as the Merrifield solid-phase synthesis method, where amino acids are<br>
sequentially added to a growing amino acid chain. See Merrifield, J. Am. Chem. Soc.<br>
55:2149-2146, 1963. Equipment for automated synthesis of polypeptides is<br>
commercially available from suppliers such as Perkin Elmer/Applied BioSystems<br>
Division (Foster City, CA), and may be operated according to the manufacturer's<br>
instructions.<br>
Within certain specific embodiments, a polypeptide may be a fusion<br>
protein that comprises multiple polypeptides as described herein, or that comprises at<br>
least one polypeptide as described herein and -an unrelated sequence, such as a known<br>
tumor protein. A fusion partner may, for example, assist hi providing T helper epitopes<br>
(an immunological fusion partner), preferably T helper epitopes recognized by humans,<br>
or may assist in expressing the protein (an expression enhancer) at higher yields than<br>
the native recombinant protein. Certain preferred fusion partners are both<br>
immunological and expression enhancing fusion partners. Other fusion partners may be<br>
selected so as to increase the solubility of the protein or to enable the protein to be<br>
targeted to desired intracellular compartments. Still further fusion partners include<br>
affinity tags, which facilitate purification of the protein.<br>
Fusion proteins may generally be prepared using standard techniques,<br>
including chemical conjugation. Preferably, a fusion protein is expressed as a<br>
recombinant protein, allowing the production of increased levels, relative to a non-fused<br>
protein, in an expression system. Briefly, DNA sequences encoding the polypeptide<br>
components may be assembled separately, and ligated into an appropriate expression<br>
vector. The 3' end of the DNA sequence encoding one polypeptide component is<br>
ligated, with or without a peptide linker, to the 5' end of a DNA sequence encoding the<br>
second polypeptide component so that the reading frames of the sequences are hi phase.<br>
This permits translation into a single fusion protein that retains the biological activity of<br>
both component polypeptides.<br>
. A peptide linker sequence may be employed to separate the first and<br>
second polypeptide components by a distance sufficient to ensure that each polypeptide<br>
folds into its secondary and tertiary structures. Such a peptide linker sequence is<br>
incorporated into the fusion protein using standard techniques well known in the art.<br>
Suitable peptide linker sequences may be chosen based on the following factors:<br>
(1) their ability to adopt a flexible extended conformation; (2) their inability to adopt a<br>
secondary structure that could interact with functional epitopes on the first and second<br>
polypeptides; and (3) the lack of hydrophobia or charged residues that might react with<br>
the polypeptide functional epitopes. Preferred peptide linker sequences contain Gly,<br>
Asn and Ser residues. Other near neutral ammo acids, such as Thr and Ala may also be<br>
used hi the linker sequence. Ammo acid sequences which may be usefully employed as<br>
linkers include those disclosed in Maratea et al,, Gene 40:39-46, 1985; Murphy et al,<br>
Proc. Natl. Acad. Sci. USA 53:8258-8262, 1986; U.S. Patent No. 4,935,233 and U.S.<br>
Patent No. 4,751,180. The linker sequence may generally be from 1 to about 50 amino<br>
acids in length. Linker sequences are not required when the first and second<br>
polypeptides have non-essential N-terminal amino acid regions that can be used to<br>
separate the functional domains and prevent steric interference.<br>
The ligated DNA sequences are operably linked to suitable<br>
transcriptional or translational regulatory elements. The regulatory elements<br>
responsible for expression of DNA are located only 5' to the DNA sequence encoding<br>
the first polypeptides. Similarly, stop codons required to end translation and<br>
transcription termination signals are only present 3' to the DNA sequence encoding the<br>
second polypeptide.<br>
Fusion proteins are also provided. Such proteins comprise a polypeptide<br>
as described herein together with an unrelated immunogenic protein. Preferably the<br>
immunogenic protein is capable of eliciting a recall response. Examples of such<br>
proteins include tetanus, tuberculosis and hepatitis proteins (see, for example, Stoute et<br>
al. New Engl. J. Med., 535:86-91,1997).<br>
Within preferred embodiments, an immunological fusion partner is<br>
derived from protein D, a surface protein of the gram-negative bacterium Haemophilus<br>
influenza B (WO 91/18926). Preferably, a protein D derivative comprises<br>
approximately the first third of the protein (e.g., the first N-terminal 100-110 amino<br>
acids), and a protein D derivative may be lipidated. Within certain preferred<br>
embodiments, the first 109 residues of a Lipoprotein D fusion partner is included on the<br>
N-terminus to provide the polypeptide with additional exogenous T-cell epitopes and to<br>
increase the expression level in E. coll (thus functioning as an expression enhancer).<br>
The lipid tail ensures optimal presentation of the antigen to antigen presenting cells.<br>
Other fusion partners include the non-structural protein from infiuenzae virus, NS1<br>
(hemaglutinin). Typically, the N-terminal 81 amino acids are used, although different<br>
fragments that include T-helper epitopes may be used.<br>
In another embodiment, the immunological fusion partner is the protein<br>
known as LYTA, or a portion thereof (preferably a C-terminal portion). LYTA is<br>
derived from Streptococcus pneumoniae, which synthesizes an N-acetyl-L-alanine<br>
amidase known as amidase LYTA (encoded by the LytA gene; Gene 43:265-292,<br>
1986). LYTA is an autolysin that specifically degrades certain bonds in the<br>
peptidoglycan backbone. The C-terminal domain of the LYTA protein is responsible<br>
for the affinity to the choline or to some choline analogues such as DEAE. This<br>
property has been exploited for the development of E. coli C-LYTA expressing<br>
plasmids useful for expression of fusion proteins. Purification of hybrid proteins<br>
containing the C-LYTA fragment at the amino terminus has been described (see<br>
Biotechnology 70:795-798, 1992). Within a preferred embodiment, a repeat portion of<br>
LYTA may be incorporated into a fusion protein. A repeat portion is found in the Cterminal<br>
region starting at residue 178. A particularly preferred repeat portion<br>
incorporates residues 188-305.<br>
In general, polypeptides (including fusion proteins) and polynucleotides<br>
as described herein are isolated. An "isolated" polypeptide or polynucleotide is one that<br>
is removed from its original environment. For example, a naturally-occurring protein is<br>
isolated if it is separated from some or all of the coexisting materials in the natural<br>
system. Preferably, such polypeptides are at least about 90% pure, more preferably at<br>
least about 95% pure and most preferably at least about 99% pure. A polynucleotide is<br>
considered to be isolated if, for example, it is cloned into a vector that is not a part of<br>
the natural environment.<br>
BINDING AGENTS<br>
The present invention further provides agents, such as antibodies and<br>
antigen-binding fragments thereof, that specifically bind to a lung tumor protein. As<br>
used herein, an antibody, or antigen-binding fragment thereof, is said to "specifically<br>
bind" to a lung tumor protein if it reacts at a detectable level (within, for example, an<br>
ELISA) with a lung tumor protein, and does not react detectably with unrelated proteins<br>
under similar conditions. As used herein, "binding" refers to a noncovalent association<br>
between two separate molecules such that a complex is formed. The ability to bind may<br>
be evaluated by, for example, determining a binding constant for the formation of the<br>
complex. The binding constant is the value obtained when the concentration of the<br>
complex is divided by the product of the component concentrations. In general, two<br>
compounds are said to "bind," in the context of the present invention, when the binding<br>
constant for complex formation exceeds about 103 L/mol. The binding constant may be<br>
determined using methods well known in the art.<br>
Binding agents may be further capable of differentiating between<br>
patients with and without a cancer, such as lung cancer, using the representative assays<br>
provided herein. In other words, antibodies or other binding agents that bind to a lung<br>
tumor protein will generate a signal indicating the presence of a cancer in at least about<br>
20% of patients with the disease, and will generate a negative signal indicating the<br>
absence of the disease in at least about 90% of individuals without the cancer. To<br>
determine whether a binding agent satisfies this requirement, biological samples (e.g.,<br>
blood, sera, sputum, urine and/or tumor biopsies) from patients with and without a<br>
cancer (as determined using standard clinical tests) may be assayed as described herein<br>
for the presence of polypeptides that bind to the binding agent. It will be apparent that a<br>
statistically significant number of samples with and without the disease should be<br>
assayed. Each binding agent should satisfy the above criteria; however, those of<br>
ordinary skill hi the art will recognize that binding agents may be used in combination<br>
to improve sensitivity.<br>
Any agent that satisfies the above requirements may be a binding agent.<br>
For example, a binding agent may be a ribosome, with or without a peptide component,<br>
an RNA molecule or a polypeptide. In a preferred embodiment, a binding agent is an<br>
antibody or an antigen-binding fragment thereof. Antibodies may be prepared by any<br>
of a variety of techniques known to those of ordinary skill hi the art. See, e.g., Harlow<br>
and Lane, Antibodies: A Laboratory Manual, Cold Spring Harbor Laboratory, 1988. In<br>
general, antibodies can be produced by cell culture techniques, including the generation<br>
of monoclonal antibodies as described herein, or via transfection of antibody genes into<br>
suitable bacterial or mammalian cell hosts, in order to allow for the production of<br>
recombinant antibodies. In one technique, an immunogen comprising the polypeptide<br>
is initially injected into any of a wide variety of mammals (e.g., mice, rats, rabbits,<br>
sheep or goats). In this step, the polypeptides of this invention may serve as the<br>
immunogen without modification. Alternatively, particularly for relatively short<br>
polypeptides, a superior immune response may be elicited if the polypeptide is joined to<br>
a carrier protein, such as bovine serum albumin or keyhole limpet hemocyanin. The<br>
immunogen is injected into the animal host, preferably according to a predetermined<br>
schedule incorporating one or more booster immunizations, and the animals are bled<br>
periodically. Polyclonal antibodies specific for the polypeptide may then be purified<br>
from such antisera by, for example, affinity chromatography using the polypeptide<br>
coupled to a suitable solid support.<br>
Monoclonal antibodies specific for an antigenic polypeptide of interest<br>
may be prepared, for example, using the technique of Kohler and Milstein, Eur. J.<br>
Immunol. 6:511-519, 1976, and improvements thereto. Briefly, these methods involve<br>
the preparation of immortal cell lines capable of producing antibodies having the<br>
desired specificity (i.e., reactivity with the polypeptide of interest). Such cell lines may<br>
be produced, for example, from spleen cells obtained from an animal immunized as<br>
described above. The spleen cells are then immortalized by, for example, fusion with a<br>
myeloma cell fusion partner, preferably one that is syngeneic with the immunized<br>
animal. A variety of fusion techniques may be employed. For example; the spleen cells<br>
and myeloma cells may be combined with a nonionic detergent for a few minutes and<br>
then plated at low density on a selective medium that supports the growth of hybrid<br>
cells, but not myeloma cells. A preferred selection technique uses HAT (hypoxanthine,<br>
aminopterin, thymidine) selection. After a sufficient time, usually about 1 to 2 weeks,<br>
colonies of hybrids are observed. Single colonies are selected and their culture<br>
supernatants tested for binding activity against the polypeptide. Hybridomas having<br>
high reactivity and specificity are preferred.<br>
Monoclonal antibodies may be isolated from the supernatants of growing<br>
hybridoma colonies. In addition, various techniques may be employed to enhance the<br>
yield, such as injection of the hybridoma cell line into the peritoneal cavity of a suitable<br>
vertebrate host, such as a mouse. Monoclonal antibodies may then be harvested from<br>
the ascites fluid or the blood. Contaminants may be removed from the antibodies by<br>
conventional techniques, such as chromatography, gel filtration, precipitation, and<br>
extraction. The polypeptides of this invention may be used in the purification process<br>
in, for example, an affinity chromatography step.<br>
Within certain embodiments, the use of antigen-binding fragments of<br>
antibodies may be preferred. Such fragments include Fab fragments, which may be<br>
prepared using standard techniques. Briefly, immunoglobulins may be purified from<br>
rabbit serum by affinity chromatography on Protein A bead columns (Harlow and Lane,<br>
Antibodies: A Laboratory Manual, Cold Spring Harbor Laboratory, 1988) and digested<br>
by papain to yield Fab and Fc fragments. The Fab and Fc fragments may be separated<br>
by affinity chromatography on protein A bead columns.<br>
Monoclonal antibodies of the present invention may be coupled to one or<br>
more therapeutic agents. Suitable agents in this regard include radionuclides,<br>
differentiation inducers, drugs, toxins, and derivatives thereof. Preferred radionuclides<br>
include "Y, 123I, 125I, 131I, l86Re, 188Re, 21lAt, and 212Bi. Preferred drugs include<br>
methotrexate, and pyrimidine and purine analogs. Preferred differentiation inducers<br>
include phorbol esters and butyric acid. Preferred toxins include ricin, abrin, dipthena<br>
toxin, cholera toxin, gelonin, Pseudomonas exotoxin, Shigella toxin, and pokeweed<br>
antiviral protein.<br>
A therapeutic agent may be coupled (e.g., covalently bonded) to a<br>
suitable monoclonal antibody either directly or indirectly (e.g., via a linker group). A<br>
direct reaction between an agent and an antibody is possible when each possesses a<br>
substituent capable of reacting with the other. For example, a nucleophilic group, such<br>
as an amino or sulfhydryl group, on one may be capable of reacting with a carbonylcontaining<br>
group, such as an anhydride or an acid halide, or with an alkyl group<br>
containing a good leaving group (e.g., a halide) on the other.<br>
Alternatively, it may be desirable to couple a therapeutic agent and an<br>
antibody via a linker group. A linker group can function as a spacer to distance an<br>
antibody from an agent hi order to avoid interference with binding capabilities. A<br>
linker group can also serve to increase the chemical reactivity of a substituent on an<br>
agent or an antibody, and thus increase the coupling efficiency. An increase in<br>
chemical reactivity may also facilitate the use of agents, or functional groups on agents,<br>
which otherwise would not be possible.<br>
It will be evident to those skilled in the art that a variety of bifunctional<br>
or polyfunctional reagents, both homo- and hetero-functional (such as those described<br>
in the catalog of the Pierce Chemical Co., Rockford, IL), may be employed as the linker<br>
group. Coupling may be effected, for example, through amino groups, carboxyl groups,<br>
sulfhydryl groups or oxidized carbohydrate residues. There are numerous references<br>
describing such methodology, e.g., U.S. Patent No. 4,671,958, to Rodwell etal.<br>
Where a therapeutic agent is more potent when free from the antibody<br>
portion of the immunoconjugates of the present invention, it may be desirable to use a<br>
linker group which is cleavable during or upon intemalization into a cell. A number of<br>
different cleavable linker groups have been described. The mechanisms for the<br>
intracellular release of an agent from these linker groups include cleavage by reduction<br>
of a disulfide bond (e.g., U.S. Patent No. 4,489,710, to Spitler), by irradiation of a<br>
photolabile bond (e.g., U.S. Patent No. 4,625,014, to Senter et a/.), by hydrolysis of<br>
derivatized ammo acid side chains (e.g., U.S. Patent No. 4,638,045, to Kohn et a/.), by<br>
serum complement-mediated hydrolysis (e.g., U.S. Patent No. 4,671,958, to Rodwell et<br>
al.), and acid-catalyzed hydrolysis (e.g., U.S. Patent No. 4,569,789, to Blattler et a/.).<br>
It may be desirable to couple more than one agent to an antibody. In one<br>
embodiment, multiple molecules of an agent are coupled to one antibody molecule. In<br>
another embodiment, more than one type of agent may be coupled to one antibody.<br>
Regardless of the particular embodiment, immunoconjugates with more than one agent<br>
may be prepared in a variety of ways. For example, more than one agent may be<br>
coupled directly to an antibody molecule, or linkers that provide multiple sites for<br>
attachment can be used. Alternatively, a carrier can be used.<br>
A carrier may bear the agents hi a variety of ways, including covalent<br>
bonding either directly or via a linker group. Suitable carriers include proteins such as<br>
albumins (e.g., U.S. Patent No. 4,507,234, to Kato et al.), peptides and polysaccharides<br>
such as aminodextran (e.g., U.S. Patent No. 4,699,784, to Shih et al). A carrier may<br>
also bear an agent by noncovalent bonding or by encapsulation, such as within a<br>
liposome vesicle (e.g., U.S. Patent Nos. 4,429,008 and 4,873,088). Carriers specific for<br>
radionuclide agents include radiohalogenated small molecules and chelating<br>
compounds. For example, U.S. Patent No. 4,735,792 discloses representative<br>
radiohalogenated small molecules and their synthesis. A radionuclide chelate may be<br>
formed from chelating compounds that include those containing nitrogen and sulfur<br>
atoms as the donor atoms for binding the metal, or metal oxide, radionuclide. For<br>
example, U.S. Patent No. 4,673,562, to Davison et al. discloses representative chelating<br>
compounds and their synthesis.<br>
A variety of routes of administration for the antibodies and<br>
immunoconjugates may be used. Typically, administration will be intravenous,<br>
intramuscular, subcutaneous or in the bed of a resected tumor. It will be evident that<br>
the precise dose of the antibody/immunoconjugate will vary depending upon the<br>
antibody used, the antigen density on the tumor, and the rate of clearance of the<br>
antibody.<br>
T CELLS<br>
Immunotherapeutic compositions may also, or alternatively, comprise T<br>
cells specific for a lung tumor protein. Such cells may generally be prepared in vitro or<br>
ex vivo, using standard procedures. For example, T cells may be isolated from bone<br>
marrow, peripheral blood, or a fraction of bone marrow or peripheral blood of a patient,<br>
using a commercially available cell separation system, such as the Isolex™ System,<br>
available from Nexell Therapeutics, Inc^ (Irvine, CA; see also U.S. Patent No.<br>
5,240,856; U.S. Patent No. 5,215,926; WO 89/06280; WO 91/16116 and WO<br>
92/07243). Alternatively, T cells may be derived from related or unrelated humans,<br>
non-human mammals, cell lines or cultures.<br>
T cells may be stimulated with a lung tumor polypeptide, polynucleotide<br>
encoding a lung tumor polypeptide and/or an antigen presenting cell (APC) that<br>
expresses such a polypeptide. Such stimulation is performed under conditions and for a<br>
time sufficient to permit the generation of T cells that are specific for the polypeptide.<br>
Preferably, a lung tumor polypeptide or polynucleotide is present within a delivery<br>
vehicle, such as a microsphere, to facilitate the generation of specific T cells.<br>
T cells are considered to be specific for a lung tumor polypeptide if the T<br>
cells specifically proliferate, secrete cytokines or kill target cells coated with the<br>
polypeptide or expressing a gene encoding the polypeptide. T cell specificity may be<br>
evaluated using any of a variety of standard techniques. For example, within a<br>
chromium release assay or proliferation assay, a stimulation index of more than two<br>
fold increase, in lysis and/or proliferation, compared to negative controls, indicates T<br>
cell specificity. Such assays may be performed, for example, as described in Chen et<br>
al, Cancer Res. 54:1065-1070, 1994. Alternatively, detection of the proliferation of<br>
T cells may be accomplished by a variety of known techniques. For example, T cell<br>
proliferation can be detected by measuring an increased rate of DNA synthesis (e.g., by<br>
pulse-labeling cultures of T cells with tritiated thymidine and measuring the amount of<br>
tritiated thymidine incorporated into DNA). Contact with a lung tumor polypeptide<br>
(100 ng/ml -100 ug/ml, preferably 200 ng/ml - 25 ug/ml) for 3 - 7 days should result in<br>
at least a two fold increase in proliferation of the T cells. Contact as described above<br>
for 2-3 hours should result in activation of the T cells, as measured using standard<br>
cytokine assays hi which a two fold increase in the level of cytokine release (e.g., TNF<br>
or IFN-y) is indicative of T cell activation (see Coligan et aL, Current Protocols in<br>
Immunology, vol. 1, Wiley Interscience (Greene 1998)). T cells that have been<br>
activated in response to a lung tumor polypeptide, polynucleotide or polypeptideexpressing<br>
APC may be CD4* and/or CD8+. Lung tumor protein-specific T cells may<br>
be expanded using standard techniques. Within preferred embodiments, the T cells are<br>
derived from a patient, a related donor or an unrelated donor, and are administered to<br>
the patient following stimulation and expansion.<br>
For therapeutic purposes, GD4+ or CD8+ T cells that proliferate in<br>
response to a lung tumor polypeptide, polynucleotide or APC can be expanded in<br>
number either in vitro or in vivo. Proliferation of such T cells in vitro may be<br>
accomplished in a variety of ways. For example, the T cells can be re-exposed to a lung<br>
tumor polypeptide, or a short peptide corresponding to an immunogenic portion of such<br>
a polypeptide, with or without the addition of T cell growth factors, such as interleukin-<br>
2, and/or stimulator cells that synthesize a lung tumor polypeptide. Alternatively, one<br>
or more T cells that proliferate in the presence of a lung tumor protein can be expanded<br>
in number by cloning. Methods for cloning cells are well known in the art, and include<br>
limiting dilution.<br>
PHARMACEUTICAL COMPOSITIONS<br>
In additional embodiments, the present invention concerns formulation<br>
of one or more of the polynucleotide, polypeptide, T-cell and/or antibody compositions<br>
disclosed herein in pharmaceutically-acceptable solutions for administration to a cell or<br>
an animal, either alone, or in combination with one or more other modalities of therapy.<br>
It will also be understood that, if desired, the nucleic acid segment,<br>
RNA, DNA or PNA compositions that express a polypeptide as disclosed herein may be<br>
administered hi combination with other agents as well, such as, e.g., other proteins or<br>
polypeptides or various pharmaceutically-active agents. In fact, there is virtually no<br>
limit to other components that may also be included, given that the additional agents do<br>
not cause a significant adverse effect upon contact with the target cells or host tissues.<br>
The compositions may thus be delivered along with various other agents as required in<br>
the particular instance.. Such compositions may be purified from host cells or other<br>
biological sources, or alternatively may be chemically synthesized as described herein.<br>
Likewise, such compositions may further comprise substituted or derivatized RNA or<br>
DNA compositions.<br>
Formulation of phannaceutically-acceptable excipients and carrier<br>
solutions is well-known to those of skill hi the art, as is the development of suitable<br>
dosing and treatment regimens for using the particular compositions described herein in<br>
a variety of treatment regimens, including e.g., oral, parenteral, intravenous, intranasal,<br>
and intramuscular administration and formulation.<br>
1. ORAL DELIVERY<br>
In certain applications, the pharmaceutical compositions disclosed herein<br>
may be delivered via oral administration to an animal. As such, these compositions<br>
may be formulated with an inert diluent or with an assimilable edible carrier, or they<br>
may be enclosed in hard- or soft-shell gelatin capsule, or they may be compressed into<br>
tablets, or they may be incorporated directly with the food of the diet.<br>
The active compounds may even be incorporated with excipients and<br>
used in the form of ingestible tablets, buccal tables, troches, capsules, elixirs,<br>
suspensions, syrups, wafers, and the like (Mathiowitz et a/., 1997; Hwang et al., 1998;<br>
U. S. Patent 5,641,515; U. S. Patent 5,580,579 and U. S. Patent 5,792,451, each<br>
specifically incorporated herein by reference in its entirety). The tablets, troches, pills,<br>
capsules and the like may also contain the following: a binder, as gum tragacanth,<br>
acacia, comstarch, or gelatin; excipients, such as dicalcium phosphate; a disintegrating<br>
agent, such as corn starch, potato starch, alginic acid and the like; a lubricant, such as<br>
magnesium stearate; and a sweetening agent, such as sucrose, lactose or saccharin may<br>
be added or a flavoring agent, such as peppermint, oil of wintergreen, or cherry<br>
flavoring. When the dosage unit form is a capsule, it may contain, in addition to<br>
materials of the above type, a liquid carrier. Various other materials may be present as<br>
coatings or to otherwise modify the physical form of the dosage unit. For instance,<br>
tablets, pills, or capsules may be coated with shellac, sugar, or both. A syrup of elixir<br>
may contain the active compound sucrose as a sweetening agent methyl and<br>
propylparabens as preservatives, a dye and flavoring, such as cherry or orange flavor!<br>
Of course, any material used in preparing any dosage unit form should be<br>
pharmaceutically pure and substantially non-toxic in the amounts employed. In<br>
addition, the active compounds may be incorporated into sustained-release preparation<br>
and formulations.<br>
Typically, these formulations may contain at least about 0.1% of the<br>
active compound or more, although the percentage of the active ingredient(s) may, of<br>
course, be varied and may conveniently be between about 1 or 2% and about 60% or<br>
70% or more of the weight or volume of the total formulation. Naturally, the amount of<br>
active compound(s) in each therapeutically useful composition may be prepared is such<br>
a way that a suitable dosage will be obtained in any given unit dose of the compound.<br>
Factors such as solubility, bioavailability, biological half-life, route of administration,<br>
product shelf life, as well as other pharmacological considerations will be contemplated<br>
by one skilled in the art of preparing such pharmaceutical formulations, and as such, a<br>
variety of dosages and treatment regimens may be desirable.<br>
For oral administration the compositions of the present invention may<br>
alternatively be incorporated with one or more excipients in the form of a mouthwash,<br>
dentifrice, buccal tablet, oral spray, or sublingual orally-administered formulation. For<br>
example, a mouthwash may be prepared incorporating the active ingredient in the<br>
required amount in an appropriate solvent, such as a sodium borate solution (Dobell's<br>
Solution). Alternatively, the active ingredient may be incorporated into an oral solution<br>
such as one containing sodium borate, glycerin and potassium bicarbonate, or dispersed<br>
in a dentifrice, or added in a therapeutically-effective amount to a composition that may<br>
include water, binders, abrasives, flavoring agents, foaming agents, and humectants.<br>
Alternatively the compositions may be fashioned into a tablet or solution form that may<br>
be placed under the tongue or otherwise dissolved in the mouth.<br>
2. INJECTABLE DELIVERY<br>
In certain circumstances it will be desirable to deliver the pharmaceutical<br>
compositions disclosed herein parenterally, intravenously, intramuscularly, or even<br>
intraperitoneally as described in U. S. Patent 5,543,158; U. S. Patent 5,641,515 and U.<br>
S. Patent 5,399,363 (each specifically incorporated herein by reference in its entirety).<br>
Solutions of the active compounds as free base or pharmacologically, acceptable salts<br>
may be prepared in water suitably mixed with a surfactant, such as<br>
hydroxypropylcellulose. Dispersions may also be prepared in glycerol, liquid<br>
polyethylene glycols, and mixtures thereof and in oils. Under ordinary conditions of<br>
storage and use, these preparations contain a preservative to prevent the growth of<br>
microorganisms.<br>
The pharmaceutical forms suitable for injectable use include sterile<br>
aqueous solutions or dispersions and sterile powders for the extemporaneous<br>
preparation of sterile injectable solutions or dispersions (U. S. Patent 5,466,468,<br>
specifically incorporated herein by reference in its entirety). In all cases the form must<br>
be sterile and must be fluid to the extent that easy syringability exists. It must be stable<br>
under the conditions of manufacture and storage and must be preserved against the<br>
contaminating action of microorganisms, such as bacteria and fungi. The carrier can be<br>
a solvent or dispersion medium containing, for example, water, ethanol, polyol (e.g.,<br>
glycerol, propylene glycol, and liquid polyethylene glycol, and the like), suitable<br>
mixtures thereof, and/or vegetable oils. Proper fluidity may be maintained, for<br>
example, by the use of a coating, such as lecithin, by the maintenance of the required<br>
particle size hi the case of dispersion and by the use of surfactants. The prevention of<br>
the action of microorganisms can be facilitated by various antibacterial and antifungal<br>
agents, for example, parabens, chlorobutanol, phenol, sorbic acid, thimerosal, and the<br>
like. In many cases, it will be preferable to include isotonic agents, for example, sugars<br>
or sodium chloride. Prolonged absorption of the injectable compositions can be<br>
brought about by the use in the compositions of agents delaying absorption, for<br>
example, aluminum monostearate and gelatin.<br>
For parenteral administration in an aqueous solution, for example, the<br>
solution should be suitably buffered if necessary and the liquid diluent first rendered<br>
isotonic with sufficient saline or glucose. These particular aqueous solutions are<br>
especially suitable for intravenous, intramuscular, subcutaneous and intraperitoneal<br>
administration. In this connection, a sterile aqueous medium that can be employed will<br>
be known to those of skill in the art in light of the present disclosure. For example, one<br>
dosage may be dissolved in 1 ml of isotonic NaCl solution and either added to 1000 ml<br>
of hypodermoclysis fluid or injected at the proposed site of infusion, (see for example,<br>
"Remington's Pharmaceutical Sciences" 15th Edition, pages 1035-1038 and 1570-<br>
1580). Some variation in dosage will necessarily occur depending on the condition of<br>
the subject being treated. The person responsible for administration will, in any event,<br>
determine the appropriate dose for the individual subject. Moreover, for human<br>
administration, preparations should meet sterility, pyrogenicity, and the general safety<br>
and purity standards as required by FDA Office of Biologies standards.<br>
Sterile injectable solutions are prepared by incorporating the active<br>
compounds in the required amount in the appropriate solvent with various of the other<br>
ingredients enumerated above, as required, followed by filtered sterilization. Generally,<br>
dispersions are prepared by incorporating the various sterilized active ingredients into a<br>
sterile vehicle which contains the basic dispersion medium and the required other<br>
ingredients from those enumerated above. In the case of sterile powders for the<br>
preparation of sterile injectable solutions, the preferred methods of preparation are<br>
vacuum-drying and freeze-drying techniques which yield a powder of the active<br>
ingredient plus any additional desired ingredient from a previously sterile-filtered<br>
solution thereof.<br>
The compositions disclosed herein may be formulated in a neutral or salt<br>
form. Pharmaceutically-acceptable salts, include the. acid addition salts (formed with<br>
the free amino groups of the protein) and which are formed with inorganic acids such<br>
as, for example, hydrochloric or phosphoric acids, or such organic acids as acetic,<br>
oxalic, tartaric, mandelic, and the like. Salts formed with the free carboxyl groups can<br>
also be derived from inorganic bases such as, for example, sodium, potassium,<br>
ammonium, calcium, or ferric hydroxides, and such organic bases as isopropylamine,<br>
trimethylamine, histidine, procaine and the like. Upon formulation, solutions will be<br>
administered in a manner compatible with the dosage formulation and in such amount<br>
as is therapeutically effective. The formulations are easily administered in a variety of<br>
dosage forms such as injectable solutions, drug-release capsules, and the like.<br>
As used herein, "carrier" includes any and all solvents, dispersion media,<br>
vehicles, coatings, diluents, antibacterial and antifungal agents, isotonic and absorption<br>
delaying agents, buffers, carrier solutions, suspensions, colloids, and the like. The use<br>
of such media and agents for pharmaceutical active substances is well known in the art<br>
Except insofar as any conventional media or agent is incompatible with the active<br>
ingredient, its use in the therapeutic compositions is contemplated. Supplementary<br>
active ingredients can also be incorporated into the compositions.<br>
The phrase "pharmaceutically-acceptable" refers to molecular entities<br>
and compositions that do not produce an allergic or similar untoward reaction when<br>
administered to a human. The preparation of an aqueous composition that contains a<br>
protein as an active ingredient is well understood in the art. Typically, such<br>
compositions are prepared as injectables, either as liquid solutions or suspensions; solid<br>
forms suitable for solution in, or suspension in, liquid prior to injection can also be<br>
prepared. The preparation can also be emulsified.<br>
3. NASAL DELIVERY<br>
In certain embodiments, the pharmaceutical compositions may be<br>
delivered by intranasal sprays, inhalation, and/or other aerosol delivery vehicles.<br>
Methods for delivering genes, nucleic acids, and peptide compositions directly to the<br>
lungs via nasal aerosol sprays has been described e.g., in U. S. Patent 5,756,353 and U.<br>
S. Patent 5,804,212 (each specifically incorporated herein by reference in its entirety).<br>
Likewise, the delivery of drugs using intranasal microparticie resins (Takenaga et at.,<br>
1998) and lysophosphatidyl-glycerol compounds (U. S. Patent 5,725,871, specifically<br>
incorporated herein by reference hi its entirety) are also well-known in the<br>
pharmaceutical arts. Likewise, transmucosal drug delivery in the form of a<br>
polytetrafluoroetheylene support matrix is described in U. S. Patent 5,780,045<br>
(specifically incorporated herein by reference hi its entirety).<br>
4. LIPOSOME-, NANOCAPSULE-, AND MICROPARTICLE-MEDIATED DELIVERY<br>
In certain embodiments, the inventors contemplate the use of liposomes,<br>
nanocapsules, microparticles, microspheres, lipid particles, vesicles, and the like, for the<br>
introduction of the compositions of the present invention into suitable host cells. In<br>
particular, the compositions of the present invention may be formulated for delivery<br>
either encapsulated in a lipid particle, a liposome, a vesicle, a nanosphere, or a<br>
nanoparticle or the like.<br>
Such formulations may be preferred for the introduction of<br>
pharmaceutically-acceptable formulations of the nucleic acids or constructs disclosed<br>
herein. The formation and use of liposomes is generally known to those of skill in the<br>
art (see for example, Couvreur et al, 1977; Couvreur, 1988; Lasic, 1998; which<br>
describes the use of liposomes and nanocapsules in the targeted antibiotic therapy for<br>
intraceUular bacterial infections and diseases). Recently, liposomes were developed<br>
with improved serum stability and circulation half-tunes (Gabizon and<br>
Papahadjopoulos, 1988; Allen and Choun, 1987; U. S. Patent 5,741,516, specifically<br>
incorporated herein by reference in its entirety). Further, various methods of liposome<br>
and liposome like preparations as potential drug carriers have been reviewed (Takakura,<br>
1998; Chandran et al, 1997; Margalit, 1995; U. S. Patent 5,567,434; U. S. Patent<br>
5,552,157; U, S. Patent 5,565,213; U. S. Patent 5,738,868 and U. S. Patent 5,795,587,<br>
each specifically incorporated herein by reference hi its entirety).<br>
Liposomes have been used successfully with a number of cell types that<br>
are normally resistant to transfection by other procedures including T cell suspensions,<br>
primary hepatocyte cultures and PC 12 cells (Renneisen et al, 1990; Muller et al,<br>
1990). In addition, liposomes are free of the DNA length constraints that are typical of<br>
viral-based delivery systems. Liposomes have been used effectively to introduce genes,<br>
drugs (Heath and Martin, 1986; Heath etal, 1986; Balazsovits etal, 1989; Fresta and<br>
Puglisi, 1996), radiotherapeutic agents (Pikul etal, 1987), enzymes (Imaizumi etal,<br>
1990a; Imaizumi etal, 1990b), viruses (Faller and Baltimore, 1984), transcription<br>
factors and allosteric effectors (Nicolau and Gersonde, 1979) into a variety of cultured<br>
cell lines and animals. In addition, several successful clinical trails examining the<br>
effectiveness of liposome-mediated drug delivery have been completed (Lopez-<br>
Berestein et al, 1985a; 1985b; Coune, 1988; Sculier et al, 1988). Furthermore, several<br>
studies suggest that the use of liposomes is not associated with autoimmune responses,<br>
toxicity or gonadal localization after systemic delivery (Mori and Fukatsu, 1992).<br>
Liposomes are formed from phospholipids that are dispersed in an<br>
aqueous medium and spontaneously form multilamellar concentric bilayer vesicles<br>
 (also termed multilamellar vesicles (MLVs). MLVs generally have diameters of from<br>
25 nm to 4 urn. Sonication of MLVs results in the formation of small unilamellar<br>
vesicles (SUVs) with diameters in the range of 200 to 500 A, containing an aqueous<br>
solution in the core.<br>
Liposomes bear resemblance to cellular membranes and are<br>
contemplated for use in connection with the present invention as carriers for the peptide<br>
compositions. They are widely suitable as both water- and lipid-soluble substances can<br>
be entrapped, i.e. hi the aqueous spaces and within the bilayer itself, respectively. It is<br>
possible that the drug-bearing liposomes may even be employed for site-specific<br>
delivery of active agents by selectively modifying the liposomal formulation.<br>
In addition to the teachings of Gouvreur et al (1977; 1988), the<br>
following information may be utilized in generating liposomal formulations.<br>
Phospholipids can form a variety of structures other than liposomes when dispersed in<br>
water, depending on the molar ratio of lipid to water. At low ratios the liposome is the<br>
preferred structure. The physical characteristics of liposomes depend on pH, ionic<br>
strength and the presence of divalent cations. Liposomes can show low permeability to<br>
ionic and polar substances, but at elevated temperatures undergo a phase transition<br>
which markedly alters their permeability. The phase transition involves a change from<br>
a closely packed, ordered structure, known as the gel state, to a loosely packed, lessordered<br>
structure, known as the fluid state. This occurs at a characteristic phasetransition<br>
temperature and results in an increase in permeability to ions, sugars and<br>
drugs.<br>
In addition to temperature, exposure to proteins can alter the<br>
permeability of liposomes. Certain soluble proteins, such as cytochrome c, bind,<br>
deform and penetrate the bilayer, thereby causing changes in permeability. Cholesterol<br>
inhibits this penetration of proteins, apparently by packing the phospholipids more<br>
tightly. It is contemplated that the most useful liposome formations for antibiotic and<br>
inhibitor delivery will contain cholesterol.<br>
The ability to trap solutes varies between different types of liposomes.<br>
For example, MLVs are moderately efficient at trapping solutes, but SUVs are<br>
extremely inefficient. SUVs offer the advantage of homogeneity and reproducibility in<br>
size distribution, however, and a compromise between size and trapping efficiency is<br>
offered by large unilamellar vesicles (LUVs). These are prepared by ether evaporation<br>
and are three to four times more efficient at solute entrapment than MLVs.<br>
In addition to liposome characteristics, an important determinant in<br>
entrapping compounds is the physicochemical properties of the compound itself. Polar<br>
compounds are trapped hi the aqueous spaces and nonpolar compounds bind to the lipid<br>
bilayer of the vesicle. Polar compounds are released through permeation or when the<br>
bilayer is broken, but nonpolar compounds remain affiliated with the bilayer unless it is<br>
disrupted by temperature or exposure to lipoproteins. Both types show maximum<br>
efflux rates at the phase transition temperature.<br>
Liposomes interact with cells via four different mechanisms:<br>
endocytosis by phagocytic cells of the reticuloendothelial system such as macrophages<br>
and neutrophils; adsorption to the cell surface, either by nonspecific weak hydrophobic<br>
or electrostatic forces, or by specific interactions with cell-surface components; fusion<br>
with the plasma cell membrane by insertion of the lipid bilayer of the liposome into the<br>
plasma membrane, with simultaneous release of liposomal contents into the cytoplasm;<br>
and by transfer of liposomal lipids to cellular or subcellular membranes, or vice versa,<br>
without any association of the liposome contents. It often is difficult to determine<br>
which mechanism is operative and more than one may operate at the same time.<br>
The fate and disposition of intravenously injected liposomes depend on<br>
their physical properties, such as size, fluidity, and surface charge. They may persist in<br>
tissues for h or days, depending on their composition, and half lives in the blood range<br>
from min to several h. Larger liposomes, such as MLVs and LUVs, are taken up<br>
rapidly by phagocytic cells of the reticuloendothelial system, but physiology of the<br>
circulatory system restrains the exit of such large species at most sites. They can exit<br>
only hi places where large openings or pores exist in the capillary endothelium, such as<br>
the sinusoids of the liver or spleen. Thus, these organs are the predominate site of<br>
uptake. On the other hand, SUVs show a broader tissue distribution but still are<br>
sequestered highly in the liver and spleen. In general, this in vivo behavior limits the<br>
potential targeting of liposomes to only those organs and tissues accessible to their large<br>
size. These include the blood, liver, spleen, bone marrow, and lymphoid organs.<br>
Targeting is generally not a limitation in terms of the present invention.<br>
However, should specific targeting be desired, methods are available for this to be<br>
accomplished. Antibodies may be used to bind to the liposome surface and to direct the<br>
antibody and its drug contents to specific antigenic receptors located on a particular<br>
cell-type surface. Carbohydrate determinants (giycoprotein or glycolipid cell-surface<br>
components that play a role in cell-cell recognition, interaction and adhesion) may also<br>
be used as recognition sites as they have potential in directing liposomes to particular<br>
cell types. Mostly, it is contemplated that intravenous injection of liposomal<br>
preparations would be used, but other routes of administration are also conceivable.<br>
Alternatively, the invention provides for pharmaceutically-acceptable<br>
nanocapsule formulations of the compositions of the present invention. Nanocapsules<br>
can generally entrap compounds in a stable and reproducible way (Henry-Michelland<br>
etal, 1987; Quintanar-Guerrero et al., 1998; Douglas et al., 1987). To avoid side<br>
effects due to intracellular polymeric overloading, such ultrafine particles (sized around<br>
0.1 urn) should be designed using polymers able to be degraded in vivo. Biodegradable<br>
polyalkyl-cyanoacrylate nanoparticles that meet these requirements are contemplated<br>
for use in the present invention. Such particles may be are easily made, as described<br>
(Couvreur etal., 1980; 1988; zur Muhlen et al., 1998; Zambaux et al 1998; Pinto-<br>
Alphandry et al., 1995 and U. S. Patent 5,145,684, specifically incorporated herein by<br>
reference in its entirety).<br>
IMMUNOGENIC COMPOSITIONS<br>
In certain preferred embodiments of the present invention, immunogenic<br>
compositions, or vaccines, are provided. The immunogenic compositions will generally<br>
comprise one or more pharmaceutical compositions, such as those discussed above, in<br>
combination with an immunostimulant. An immunostimulant may be any substance<br>
that enhances or potentiates an immune response (antibody and/or cell-mediated) to an<br>
exogenous antigen. Examples of immunostimulants include adjuvants, biodegradable<br>
microspheres (e.g., polylactic galactide) and liposomes (into which the compound is<br>
incorporated; see e.g., Fullerton, U.S. Patent No. 4,235,877). Vaccine preparation is<br>
generally described in, for example, M.F. Powell and MJ. Newman, eds., "Vaccine<br>
Design (the subunit and adjuvant approach)," Plenum Press (NY, 1995).<br>
Pharmaceutical compositions and immunogenic compositions, or vaccines, within the<br>
scope of the present invention may also contain other compounds, which may be<br>
biologically active or inactive. For example, one or more immunogenic portions of<br>
other tumor antigens may be present, either incorporated into a fusion polypeptide or as<br>
a separate compound, within the composition.<br>
Illustrative immunogenic compositions may contain DNA encoding one<br>
or more of the polypeptides as described above, such that the polypeptide is generated<br>
i/i situ. As noted above, the DNA may be present within any of a variety of delivery<br>
systems known to those of ordinary skill in the art, including nucleic acid expression<br>
systems, bacteria and viral expression systems. Numerous gene delivery techniques are<br>
well known in the art, such as those described by Holland, Crit. Rev. Therap. Drug<br>
Carrier Systems 75:143-198, 1998, and references cited therein. Appropriate nucleic<br>
acid expression systems contain the necessary DNA sequences for expression in the<br>
patient (such as a suitable promoter and terminating signal). Bacterial delivery systems<br>
involve the administration of a bacterium (such as Bacillus-Calmette-Guerrin) that<br>
expresses an immunogenic .portion of the polypeptide on its cell surface or secretes such<br>
an epitope. In a preferred embodiment, the DNA may be introduced using a viral<br>
expression system (e.g., vaccinia or other pox virus, retrovirus, or adenovirus), which<br>
may involve the use of a non-pathogenic (defective), replication competent virus.<br>
Suitable systems are disclosed, for example, in Fisher-Hoch et al., Proc. Natl. Acad.<br>
Sci. USA 86:317-321, 1989; Flexner et al, Ann. N.Y. Acad Sci. 55P:86-103, 1989;<br>
Flexner et al, Vaccine 5:17-21, 1990; U.S. Patent Nos.4,603,112, 4,769,330, and<br>
5,017,487; WO 89/01973; U.S. Patent No. 4,777,127; GB 2,200,651; EP 0,345,242;<br>
WO 91/02805; Berkner, Biotechniques 6:616-627, 1988; Rosenfeld et al., Science<br>
252:431-434, 1991; Kolls et al, Proc. Natl. Acad. Sci. USA 91:215-219, 1994;<br>
Kass-Eisler et al., Proc. Natl. Acad. Sci. USA 90:11498-11502, 1993; Guzman et al,<br>
Circulation 55:2838-2848, 1993; and Guzman et al, Cir. Res. 75:1202-1207, 1993.<br>
Techniques for incorporating DNA into such expression systems are well known to<br>
those of ordinary skill in the art. The DNA may also be "naked," as described, for<br>
example, in Ulmer et al, Science 259:1745-1749, 1993 and reviewed by Cohen,<br>
Science 259:1691-1692, 1993. The uptake of naked DNA may be increased by coating<br>
the DNA onto biodegradable beads, which are efficiently transported into the cells. It<br>
will be apparent that an immunogenic composition may comprise both a polynucleotide<br>
and a polypeptide component. Such immunogenic compositions may provide for an<br>
enhanced immune response.<br>
It will be apparent that an immunogenic composition may contain<br>
pharmaceutically acceptable salts of the polynucleotides and polypeptides provided<br>
herein. Such salts may be prepared from pharmaceutically acceptable non-toxic bases,<br>
including organic bases (e.g., salts of primary, secondary and tertiary amines and basic<br>
amino acids) and inorganic bases (e.g., sodium, potassium, lithium, ammonium,<br>
calcium and magnesium salts).<br>
While any suitable carrier known to those of ordinary skill hi the art may<br>
be employed in the immunogenic compositions of this invention, the type of carrier will<br>
vary depending on the mode of administration. Compositions of the present invention<br>
may be formulated for any appropriate manner of administration, including for<br>
example, topical, oral, nasal, intravenous, intracranial, intraperitoneal, subcutaneous or<br>
intramuscular administration. For parenteral administration, such as subcutaneous<br>
injection, the carrier preferably comprises water, saline, alcohol, a fat, a wax or a buffer.<br>
For oral administration, any of the above carriers or a solid carrier, such as mannitol,<br>
lactose, starch, magnesium stearate, sodium saccharine, talcum, cellulose, glucose,<br>
sucrose, and magnesium carbonate, may be employed. Biodegradable microspheres<br>
(e.g., polylactate polyglycolate) may also be employed as carriers for the<br>
pharmaceutical compositions of this invention. Suitable biodegradable microspheres<br>
are disclosed, for example, in U.S. Patent Nos. 4,897,268; 5,075,109; 5,928,647;<br>
5,811,128; 5,820,883; 5,853,763; 5,814,344 and 5,942,252. One may also employ a<br>
carrier comprising the particulate-protein complexes described in U.S. Patent No.<br>
5,928,647, which are capable of inducing a class I-restricted cytotoxic T lymphocyte<br>
responses in a host.<br>
Such compositions may also comprise buffers (e.g., neutral buffered<br>
saline or phosphate buffered saline), carbohydrates (e.g., glucose, mannose, sucrose or<br>
dextrans), mannitol, proteins, polypeptides or amino acids such as glycine, antioxidants,<br>
bacteriostats, chelating agents such as EDTA or glutathione, adjuvants (e.g., aluminum<br>
hydroxide), solutes that render the formulation isotonic, hypotonic or weakly hypertonic<br>
with the blood of a recipient, suspending agents, thickening agents and/or preservatives.<br>
Alternatively, compositions of the present invention may be formulated as a<br>
lyophilizate. Compounds may also be encapsulated within liposomes using well known<br>
technology.<br>
Any of a variety of immunostimulants may be employed in the<br>
immunogenic compositions of this invention. For example, an adjuvant may be<br>
included. Most adjuvants contain a substance designed to protect the antigen from<br>
rapid catabolism, such as aluminum hydroxide or mineral oil, and a stimulator of<br>
immune responses, such as lipid A, Bortadetta pertussis or Mycobacterium tuberculosis<br>
derived proteins. Suitable adjuvants are commercially available as, for example,<br>
Freund's Incomplete Adjuvant and Complete Adjuvant (Difco Laboratories, Detroit,<br>
MI); Merck Adjuvant 65 (Merck and Company, Inc., Rahway, NJ); AS-2 (SmithKline<br>
Beecham, Philadelphia, PA); aluminum salts such as aluminum hydroxide gel (alum) or<br>
aluminum phosphate; salts of calcium, iron or zinc; an insoluble suspension of acylated<br>
tyrosine; acylated sugars; cationically or anionically derivatized polysaccharides;<br>
polyphosphazenes; biodegradable microspheres; monophosphoryl lipid A and quil A.<br>
Cytokines, such as GM-CSF or interleukin-2, -7, or -12, may also be used as adjuvants.<br>
Within the immunogenic compositions provided herein, the adjuvant<br>
composition is preferably designed to induce an immune response predominantly of the<br>
Thl type. High levels of Thl-type cytokines (e.g., IFN-y, TNFa, IL-2 and IL-12) tend<br>
to favor the induction of cell mediated immune responses to an administered antigen.<br>
In contrast, high levels of Th2-type cytokines (e.g., IL-4, IL-5, IL-6 and IL-10) tend to<br>
favor the induction of humoral immune responses. Following application of an<br>
immunogenic composition as provided herein, a patient will support an immune<br>
response that includes Thl- and Th2-type responses. Within a preferred embodiment,<br>
in which a response is predominantly Thl-type, the level of Thl-type cytokines will<br>
increase to a greater extent than the level of Th2-type cytokines. The levels of these<br>
cytokines may be readily assessed using standard assays. For a review of the families<br>
of cytokines, see Mosmann and Coffinan, Ann. Rev. Immunol. 7:145-173,1989.<br>
Preferred adjuvants for use in eliciting a predominantly Thl-type<br>
response include, for example, a combination of monophosphoryl lipid A, preferably 3-<br>
de-O-acylated monophosphoryl lipid A (3D-MPL), together with an aluminum salt.<br>
MPL adjuvants are available from Corixa Corporation (Seattle, WA; see US Patent<br>
Nos. 4,436,727; 4,877,611; 4,866,034 and 4,912,094). CpG-containing<br>
oligonucleotides (in which the CpG dinucleotide is unmethylated) also induce a<br>
predominantly Thl response. Such oligonucleotides are well known and are described,<br>
for example, in WO 96/02555, WO 99/33488 and U.S. Patent Nos. 6,008,200 and<br>
5,856,462. Immunostimulatory DNA sequences are also described, for example, by<br>
Sato et al, Science 275:352, 1996. Another preferred adjuvant is a saponin, preferably<br>
QS21 (Aquila Biopharmaceuticals Inc., Framingham, MA), which may be used alone or<br>
in combination with other adjuvants. For example, an enhanced system involves the<br>
combination of a monophosphoryl lipid A and saponin derivative, such as the<br>
combination of QS21 and 3D-MPL as described in WO 94/00153, or a less reactogenic<br>
composition where the QS21 is quenched with cholesterol, as described in WO<br>
96/33739. Other preferred formulations comprise an oil-in-water emulsion and<br>
tocopherol. .A particularly potent adjuvant formulation involving QS21, 3D-MPL and<br>
tocopherol in an oil-in-water emulsion is described in WO 95/17210.<br>
Other preferred adjuvants include Montanide ISA 720 (Seppic, France),<br>
SAF (Chiron, California, United States), ISCOMS (CSL), MF-59 (Chiron), the SBAS<br>
series of adjuvants (e.g., SBAS-2 or SBAS-4, available from SmithKline Beecham,<br>
Rixensart, Belgium), Detox (Corixa, Hamilton, MT), RC-529 (Corixa, Hamilton, MT)<br>
and other aminoalkyl glucosaminide 4-phosphates (AGPs), such as those described in<br>
pending U.S. Patent Application Serial Nos. 08/853,826 and 09/074,720, the disclosures<br>
of which are incorporated herein by reference in their entireties.<br>
Any immunogenic composition provided herein may be prepared using<br>
well known methods that result in a combination of antigen, immune response enhancer<br>
and a suitable carrier or excipient. The compositions described herein may be<br>
administered as part of a sustained release formulation (i.e., a formulation such as a<br>
capsule, sponge or gel (composed of polysaccharides, for example) that effects a slow<br>
release of compound following administration). Such formulations may generally be<br>
prepared using well known technology (see, e.g., Coombes et al, Vaccine 14:1429-<br>
1438, 1996) and administered by, for example, oral, rectal or subcutaneous<br>
implantation, or by implantation at the desired target site. Sustained-release<br>
formulations may contain a polypeptide, polynucleotide or antibody dispersed in a<br>
carrier matrix and/or contained within a reservoir surrounded by a rate controlling<br>
membrane.<br>
Carriers for use within such formulations are biocompatible, and may<br>
also be biodegradable; preferably the formulation provides a relatively constant level of<br>
active component release. Such carriers include microparticles of poly(lactide-coglycolide),<br>
polyacrylate, latex, starch, cellulose, dextran and the like. Other delayedrelease<br>
carriers include supramolecular biovectors, which comprise a non-liquid<br>
hydrophilic core (e.g., a cross-linked polysaccharide or oligosaccharide) and,<br>
optionally, an external layer comprising an amphiphilic compound, such as a<br>
phospholipid (see e.g., U.S. Patent No. 5,151,254 and PCT applications WO 94/20078,<br>
WO/94/23701 and WO 96/06638). The amount of active compound contained within a<br>
sustained release formulation depends upon the site of implantation, the rate and<br>
expected duration of release and the nature of the condition to be treated or prevented.<br>
Any of a variety of delivery vehicles may be employed within<br>
pharmaceutical compositions and imraunogenic compositions to facilitate production of<br>
an antigen-specific immune response that targets tumor cells. Delivery vehicles include<br>
antigen presenting cells (APCs), such as dendritic cells, macrophages, B cells,<br>
monocytes and other cells that may be engineered to be efficient APCs. Such cells<br>
may, but need not, be genetically modified to increase the capacity for presenting the<br>
antigen, to improve activation and/or maintenance of the T cell response, to have antitumor<br>
effects per se and/or to be immunologically compatible with the receiver (i.e.,<br>
matched HLA haplotype). APCs may generally be isolated from any of a variety of<br>
biological fluids and organs, including tumor and peritumoral tissues, and may be<br>
autologous, allogeneic, syngeneic or xenogeneic cells.<br>
Certain preferred embodiments of the present invention use dendritic<br>
cells or progenitors thereof as antigen-presenting cells. Dendritic cells are highly potent<br>
APCs (Banchereau and Steinman, Nature 392:245-251, 1998) and have been shown to<br>
be effective as a physiological adjuvant for eliciting prophylactic or therapeutic<br>
antitumor immunity (see Timmerman and Levy, Ann. Rev. Med. 50:507-529, 1999). In<br>
general, dendritic cells may be identified based on their typical shape (stellate in situ,<br>
with marked cytoplasmic processes (dendrites) visible in vitro), their ability to take up,<br>
process and present antigens with high efficiency and their ability to activate nai've T<br>
cell responses. Dendritic cells may, of course, be engineered to express specific cellsurface<br>
receptors or ligands that are not commonly found on dendritic cells in vivo or ex<br>
vivo, and such modified dendritic cells are contemplated by the present invention. As<br>
an alternative to dendritic cells, secreted vesicles antigen-loaded dendritic cells (called<br>
exosomes) may be used within a vaccine, or immunogenic composition (see Zitvogel et<br>
al., Nature Med. 4:594-600,1998).<br>
Dendritic cells and progenitors may be obtained from peripheral blood,<br>
bone marrow, tumor-infiltrating cells, peritumoral tissues-infiltrating cells, lymph<br>
nodes, spleen, skin, umbilical cord blood or any other suitable tissue or fluid. For<br>
example, dendritic cells may be differentiated ex vivo by adding a combination of<br>
cytokines such as GM-CSF, IL-4, IL-13 and/or TNFa to cultures of monocytes<br>
harvested from peripheral blood. Alternatively, CD34 positive cells harvested from<br>
peripheral blood, umbilical cord blood or bone marrow may be differentiated into<br>
dendritic cells by adding to the culture medium combinations of GM-CSF, IL-3, TNFa,<br>
CD40 ligand, LPS, flt3 ligand and/or other compound(s) that induce differentiation,<br>
maturation and proliferation of dendritic cells.<br>
Dendritic cells are conveniently categorized as "immature" and "mature"<br>
cells, which allows a simple way to discriminate between two well characterized<br>
phenotypes. However, this nomenclature should not be construed to exclude all<br>
possible intermediate stages of differentiation. Immature dendritic cells are<br>
characterized as APC with a high capacity for antigen uptake and processing, which<br>
correlates with the high expression of Fey receptor and mannose receptor. The mature<br>
phenotype is typically characterized by a lower expression of these markers, but a high<br>
expression of cell surface molecules responsible for T cell activation such as class I and<br>
class IIMHC, adhesion molecules (e.g., CD54 and GDI 1) and costimulatory molecules<br>
(e.g., CD40, CD80, CD86 and 4-1BB).<br>
APCs may generally be transfected with a polynucleotide encoding a<br>
lung tumor protein (or portion or other variant thereof) such that -the lung tumor<br>
polypeptide, or an immunogenic portion thereof, is expressed on the cell surface. Such<br>
transfection may take place ex vivo, and a composition comprising such transfected<br>
cells may then be used for therapeutic purposes, as described herein. Alternatively, a<br>
gene delivery vehicle that targets a dendritic or other antigen presenting cell may be<br>
administered to a patient, resulting in transfection that occurs in vivo. In vivo and ex<br>
vivo transfection of dendritic cells, for example, may generally be performed using any<br>
methods known in the art, such as those described in WO 97/24447, or the gene gun<br>
approach described by Mahvi et al., Immunology and cell Biology 75:456-460, 1997.<br>
Antigen loading of dendritic cells may be achieved by incubating dendritic cells or<br>
progenitor cells with the lung tumor polypeptide, DNA (naked or within a plasmid<br>
vector) or RNA; or with antigen-expressing recombinant bacterium or viruses (e.g.,<br>
vaccinia, fowlpox, adenovirus or lentivirus vectors). Prior to loading, the polypeptide<br>
may be covalently conjugated to an irnmunological partner that provides T cell help<br>
(e.g., a carrier molecule). Alternatively, a dendritic cell may be pulsed with a nonconjugated<br>
immunological partner, separately or in the presence of the polypeptide.<br>
Immunogenic compositions and pharmaceutical compositions may be<br>
presented in unit-dose or multi-dose containers, such as sealed ampoules or vials. Such<br>
containers are preferably hermetically sealed to preserve sterility of the formulation<br>
until use. In general, formulations may be stored as suspensions, solutions or emulsions<br>
hi oily or aqueous vehicles. Alternatively, an immunogenic or pharmaceutical<br>
composition may be stored hi a freeze-dried condition requiring only the addition of a<br>
sterile liquid carrier immediately prior to use.<br>
CANCER THERAPY<br>
In further aspects of the present invention, the compositions described<br>
herein may be used for immunotherapy of cancer, such as lung cancer. Within such<br>
methods, compositions are typically administered to a patient. As used herein, a<br>
"patient" refers to any warm-blooded animal, preferably a human. A patient may or<br>
may not be afflicted with cancer. Accordingly, the above pharmaceutical compositions<br>
and immunogenic compositions may be used to prevent the development of a cancer or<br>
to treat a patient afflicted with a cancer. A cancer may be diagnosed using criteria<br>
generally accepted in the art, including the presence of a malignant tumor.<br>
Pharmaceutical compositions and immunogenic composition s may be administered<br>
either prior to or following surgical removal of primary tumors and/or treatment such as<br>
administration of radiotherapy or conventional chemotherapeutic drugs. Administration<br>
may be by any suitable method, including administration by intravenous,<br>
intraperitoneal, intramuscular, subcutaneous, intranasal, intradermal, anal, vaginal,<br>
topical and oral routes.<br>
Within certain embodiments, immunotherapy may be active<br>
immunotherapy, hi which treatment relies on the in vivo stimulation of the endogenous<br>
host immune system to react against tumors with the administration of immune<br>
response-modifying agents (such as polypeptides and polynucleotides as provided<br>
herein).<br>
Within other embodiments, immunotherapy may be passive<br>
immunotherapy, in which treatment involves the delivery of agents with established<br>
tumor-immune reactivity (such as effector cells or antibodies) that can directly or<br>
indirectly mediate antitumor effects and does not necessarily depend on an intact host<br>
immune system. Examples of effector cells include T cells as discussed above, T<br>
lymphocytes (such as CD8+ cytotoxic T lymphocytes and CD4+ T-helper tumorinfiltrating<br>
lymphocytes), killer cells (such as Natural Killer cells and lymphokineactivated<br>
killer cells), B cells and antigen-presenting cells (such as dendritic cells and<br>
macrophages) expressing a polypeptide provided herein. T cell receptors and antibody<br>
receptors specific for the polypeptides recited herein may be cloned, expressed and<br>
transferred into other vectors or effector cells for adoptive immunotherapy. The<br>
polypeptides provided herein may also be used to generate antibodies or anti-idiotypic<br>
antibodies (as described above and in U.S. Patent No. 4,918,164) for passive<br>
immunotherapy.<br>
Effector cells may generally be obtained in sufficient quantities for<br>
adoptive immunotherapy by growth in vitro, as described herein. Culture conditions for<br>
expanding single antigen-specific effector cells to several billion in number with<br>
retention of antigen recognition in vivo are well known in the art. Such in vitro culture<br>
conditions typically use intermittent stimulation with antigen, often in ihe presence of<br>
cytokines (such as IL-2) and non-dividing feeder cells. As noted above,<br>
immunoreactive polypeptides as provided herein may be used to rapidly expand<br>
antigen-specific T cell cultures in order to generate a sufficient number of cells for<br>
immunotherapy. In particular, antigen-presenting cells, such as dendritic, macrophage,<br>
monocyte, fibroblast and/or B cells, may be pulsed with immunoreactive polypeptides<br>
or transfected with one or more polynucleotides using standard techniques well known<br>
in the art. For example, antigen-presenting . cells can be transfected with a<br>
polynucleotide having a promoter appropriate for increasing expression in a<br>
recombinant virus or other expression system. Cultured effector cells for use in therapy<br>
must be able to grow and distribute widely, and to survive long term in vivo. Studies<br>
have shown that cultured effector cells can be induced to grow in vivo and to survive<br>
long term in substantial numbers by repeated stimulation with antigen supplemented<br>
with IL-2 (see, for example, Cheever etal, ImmunologicalReviews 157:177, 1997).<br>
Alternatively, a vector expressing a polypeptide recited herein may be<br>
introduced into antigen presenting cells taken from a patient and clonally propagated ex<br>
vivo for transplant back into the same patient. Transfected cells may be reintroduced<br>
into the patient using any means known in the art, preferably in sterile form by<br>
intravenous, intracavitary, intraperitoneal or intratumor administration.<br>
Routes and frequency of administration of the therapeutic compositions<br>
described herein, as well as dosage, will vary from individual to individual, and may be<br>
readily established using standard techniques. In general, the pharmaceutical<br>
compositions and immunogenic compositions may be administered by injection (e.g.,<br>
intracutaneous, intramuscular, intravenous or subcutaneous), intranasally (e.g., by<br>
aspiration) or orally. Preferably, between 1 and 10 doses may be administered over a<br>
52 week period. Preferably, 6 doses are administered, at intervals of 1 month, and<br>
booster vaccinations may be given periodically thereafter. Alternate protocols may be<br>
appropriate for individual patients. A suitable dose is an amount of a compound that,<br>
when administered as described above, is capable of promoting an anti-tumor immune<br>
response, and is at least 10-50% above the basal (i.e., untreated) level. Such response<br>
can be monitored by measuring the anti-tumor antibodies in a patient or by vaccinedependent<br>
generation of cytolytic effector cells capable of killing the. patient's tumor<br>
cells in vitro. Such vaccines, or immunogenic compositions, should also be capable of<br>
causing an immune response that leads to an improved clinical outcome (e.g., more<br>
frequent remissions, complete or partial or longer disease-free survival) in vaccinated<br>
patients as compared to non-vaccinated patients. In general, for compositions<br>
comprising one or more polypeptides, the amount of each polypeptide present in a dose<br>
ranges from about 25 u,g to 5 mg per kg of host. Suitable dose sizes will vary with the<br>
size of the patient, but will typically range from about 0.1 mL to about 5 mL.<br>
In general, an appropriate dosage and treatment regimen provides the<br>
active compound(s) in an amount sufficient to provide therapeutic and/or prophylactic<br>
benefit. Such a response can be monitored by establishing an improved clinical<br>
outcome (e.g., more frequent remissions, complete or partial, or longer disease-free<br>
survival) in treated patients as compared to non-treated patients. Increases in<br>
preexisting immune responses to a lung tumor protein generally correlate with an<br>
improved clinical outcome. Such immune responses may generally be evaluated using<br>
standard proliferation, cytotoxicity or cytokine assays, which may be performed using<br>
samples obtained from a patient before and after treatment.<br>
CANCER DETECTION AND DIAGNOSIS<br>
In general, a cancer may be detected in a patient based on the presence<br>
of one or more lung tumor proteins and/or polynucleotides encoding such proteins in a<br>
biological sample (for example, blood, sera, sputum urine and/or tumor biopsies)<br>
obtained from the patient. In other words, such proteins may be used as markers to<br>
indicate the presence or absence of a cancer such as lung cancer. In addition, such<br>
proteins may be useful for the detection of other cancers. The binding agents provided<br>
herein generally permit detection of the level of antigen that binds to the agent in the<br>
biological sample. Polynucleotide primers and probes may be used to detect the level<br>
of mRNA encoding a tumor protein, which is also indicative of the presence or absence<br>
of a cancer. In general, a lung tumor sequence should be present at a level that is at<br>
least three fold higher in tumor tissue than hi normal tissue<br>
There are a variety of assay formats known to those of ordinary skill in<br>
the art for using a binding agent to detect polypeptide markers in a sample. See, e.g.,<br>
Harlow and Lane, Antibodies: A Laboratory Manual, Cold Spring Harbor Laboratory,<br>
1988. In general, the presence or absence of a cancer in a patient may be determined by<br>
(a) contacting a biological sample obtained from a patient with a binding agent; (b)<br>
detecting in the sample a level of polypeptide that binds to the binding agent; and (c)<br>
comparing the level of polypeptide with a predetermined cut-off value.<br>
In a preferred embodiment, the assay involves the use of binding agent<br>
immobilized on a solid support to bind to and remove the polypeptide from the<br>
remainder of the sample. The bound polypeptide may then be detected using a<br>
detection reagent that contains a reporter group and specifically binds to the binding<br>
agent/polypeptide complex. Such detection reagents may comprise, for example, a<br>
binding agent that specifically binds to the polypeptide or an antibody or other agent<br>
that specifically binds to the binding agent, such as an anti-immunoglobulin, protein G,<br>
protein A or a lectin. Alternatively, a competitive assay may be utilized, in which a<br>
polypeptide is labeled with a reporter group and allowed to bind to the immobilized<br>
binding agent after incubation of the binding agent with the sample. The extent to<br>
which components of the sample inhibit the binding of the labeled polypeptide to the<br>
binding agent is indicative of the reactivity of the sample with the immobilized binding<br>
agent. Suitable polypeptides for use within such assays include full length lung tumor<br>
proteins and portions thereof to which the binding agent binds, as described above.<br>
The solid support may be any material known to those of ordinary skill<br>
in the art to which the tumor protein may be attached. For example, the solid support<br>
may be a test well in a microtiter plate or a nitrocellulose or other suitable membrane.<br>
Alternatively, the support may be a bead or disc, such as glass, fiberglass, latex or a<br>
plastic material such as polystyrene or polyvinylchloride. The support may also be a<br>
magnetic particle or a fiber optic sensor, such as those disclosed, for example, in U.S.<br>
Patent No. 5,359,681. The binding agent may be immobilized on the solid support<br>
using a variety of techniques known to those of skill in the art, which are amply<br>
described in the patent and scientific literature. In the context of the present invention,<br>
the term "immobilization" refers to both noncovalent association, such as adsorption,<br>
and covalent attachment (which may be a direct linkage between the agent and<br>
functional groups on the support or may be a linkage by way of a cross-linking agent).<br>
Immobilization by adsorption to a well in a microtiter plate or to a membrane is<br>
preferred. In such cases, adsorption may be achieved by contacting the binding agent,<br>
in a suitable buffer, with the solid support for a suitable amount of time. The contact<br>
time varies with temperature, but is typically between about 1 hour and about 1 day. In<br>
general, contacting a well of a plastic microtiter plate (such as polystyrene or<br>
polyvinylchloride) with an amount of binding agent ranging from about 10 ng to about<br>
10 u£, and preferably about 100 ng to about 1 ug, is sufficient to immobilize an<br>
adequate amount of binding agent.<br>
Covalent attachment of binding agent to a solid support may generally<br>
be achieved by first reacting the support with a bifunctional reagent that will react with<br>
both the support and a functional group, such as a hydroxyl or amino group, on the<br>
binding agent For example, the binding agent may be covalently attached to supports<br>
having an appropriate polymer coating using benzoquinone or by condensation of an<br>
aldehyde group on the support with an amine and an active hydrogen on the binding<br>
partner (see, e.g., Pierce Immunotechnology Catalog and Handbook, 1991, at<br>
A12-A13).<br>
In certain embodiments, the assay is a two-antibody sandwich assay.<br>
This assay may be performed by first contacting an antibody that has been immobilized<br>
on a solid support, commonly the well of a microtiter plate, with the sample, such that<br>
polypeptides within the sample are allowed to bind to the immobilized antibody.<br>
Unbound sample is then removed from the immobilized polypeptide-antibody<br>
complexes and a detection reagent (preferably a second antibody capable of binding to<br>
a different site on the polypeptide) containing a reporter group is added. The amount of<br>
detection reagent that remains bound to the solid support is then determined using a<br>
method appropriate for the specific reporter group.<br>
More specifically, once the antibody is immobilized on the support as<br>
described above, the remaining protein binding sites on the support are typically<br>
blocked. Any suitable blocking agent known to those of ordinary skill in the art, such<br>
as bovine serum albumin or Tween 20™ (Sigma Chemical Co., St. Louis, MO). The<br>
immobilized antibody is then incubated with the sample, and polypeptide is allowed to<br>
bind to the antibody. The sample may be diluted with a suitable diluent, such as<br>
phosphate-buffered saline (PBS) prior to incubation. In general, an appropriate contact<br>
time (i.e., incubation time) is a period of time that is sufficient to detect the presence of<br>
polypeptide within a sample obtained from an individual with lung cancer. Preferably,<br>
the contact time is sufficient to achieve a level of binding that is at least about 95% of<br>
that achieved at equilibrium between bound and unbound polypeptide. Those of<br>
ordinary skill in the art will recognize that the time necessary to achieve equilibrium<br>
may be readily determined by assaying the level of binding that occurs over a period of<br>
time. At room temperature, an incubation time of about 30 minutes is generally<br>
sufficient.<br>
Unbound sample may then be removed by washing the solid support<br>
with an appropriate buffer, such as PBS containing 0.1% Tween 20™. The second<br>
antibody, which contains a reporter group, may then be added to the solid support.<br>
Preferred reporter groups include those groups recited above.<br>
The detection reagent is then incubated with the immobilized antibodypolypeptide<br>
complex for an amount of tune sufficient to detect the bound polypeptide.<br>
An appropriate amount of time may generally be determined by assaying the level of<br>
binding that occurs over a period of time. Unbound detection reagent is then removed<br>
and bound detection reagent is detected using the reporter group. The method<br>
employed for detecting the reporter group depends upon the nature of the reporter<br>
group. For radioactive groups, scintillation counting or autoradiographic methods are<br>
generally appropriate. Spectroscopic methods may be used to detect dyes, luminescent<br>
groups and fluorescent groups. Biotin may be detected using avidin, coupled to a<br>
different reporter group (commonly a radioactive or fluorescent group or an enzyme).<br>
Enzyme reporter groups may generally be detected by the addition of substrate<br>
(generally for a specific period of time), followed by Spectroscopic or other analysis of<br>
the reaction products.<br>
To determine the presence or absence of a cancer, such as lung cancer,<br>
the signal detected from the reporter group that remains bound to the solid support is<br>
generally compared to a signal that corresponds to a predetermined cut-off value. In<br>
one preferred embodiment, the cut-off value for the detection of a cancer is the average<br>
mean signal obtained when the immobilized antibody is incubated with samples from<br>
patients without the cancer. In general, a sample generating a signal that is three<br>
standard deviations above the predetermined cut-off value is considered positive for the<br>
cancer. In an alternate preferred embodiment, the cut-off value is determined using a<br>
Receiver Operator Curve, according to the method of Sackett et al, Clinical<br>
Epidemiology: A Basic Science for Clinical Medicine, Little Brown and Co., 1985,<br>
p. 106-7. Briefly, in this embodiment, the cut-off value may be determined from a plot<br>
of pairs of true positive rates (i.e., sensitivity) and false positive rates (100%-<br>
specificity) that correspond to each possible cut-off value for the diagnostic test result.<br>
The cut-off value on the plot that is the closest to the upper left-hand corner (i.e., the<br>
value that encloses the largest area) is the most accurate cut-off value, and a sample<br>
generating a signal that is higher than the cut-off value determined by this method may<br>
be considered positive. Alternatively, the cut-off value may be shifted to the left along<br>
the plot, to minimize the false positive rate, or to the right, to minimize the false<br>
negative rate. In general, a sample generating a signal that is higher than the cut-off<br>
value determined by this method is considered positive for a cancer.<br>
In a related embodiment, the assay is performed in a flow-through or<br>
strip test format, wherein the binding agent is immobilized on a membrane, such as<br>
nitrocellulose. In the flow-through test, polypeptides within the sample bind to the<br>
immobilized binding agent as the sample passes through the membrane. A second,<br>
labeled binding agent then binds to the binding agent-polypeptide complex as a solution<br>
containing the second binding agent flows through the membrane. The detection of<br>
bound second binding agent may then be performed as described above. In the strip test<br>
format, one end of the membrane to which binding agent is bound is immersed in a<br>
solution containing the sample. The sample migrates along the membrane through a<br>
region containing second binding agent and to the area of immobilized binding agent.<br>
Concentration of second binding agent at the area of immobilized antibody indicates the<br>
presence of a cancer. Typically, the concentration of second binding agent at that site<br>
generates a pattern, such as a line, that can be read visually. The absence of such a<br>
pattern indicates a negative result. In general, the amount of binding agent immobilized<br>
on the membrane is selected to generate 'a visually discernible pattern when the<br>
biological sample contains a level of polypeptide that would be sufficient to generate a<br>
positive signal in the two-antibody sandwich assay, in the format discussed above.<br>
Preferred binding agents for use in such assays are antibodies and antigen-binding<br>
fragments thereof. Preferably, the amount of antibody immobilized on the membrane<br>
ranges from about 25 ng to about l|ig, and more preferably from about 50 ng to about<br>
500 ng. Such tests can typically be performed with a very small amount of biological<br>
sample.<br>
Of course, numerous other assay protocols exist that are suitable for use<br>
with the tumor proteins or binding agents of the present invention. The above<br>
descriptions are intended to be exemplary only. For example, it will be apparent to<br>
those of ordinary skill in the art that the above protocols may be readily modified to use<br>
lung tumor polypeptides to detect antibodies that bind to such polypeptides in a<br>
biological sample. The detection of such lung tumor protein specific antibodies may<br>
correlate with the presence of a cancer.<br>
A cancer may also, or alternatively, be detected based on the presence of<br>
T cells that specifically react with a lung tumor protein in a biological sample. Within<br>
certain methods, a biological sample comprising CD4+ and/or GD8+ T cells isolated<br>
from a patient is incubated with a lung tumor polypeptide, a polynucleotide encoding<br>
such a polypeptide and/or an APC that expresses at least an immunogenic portion of<br>
such a polypeptide, and the presence or absence of specific activation of the T cells is<br>
detected. Suitable biological samples include, but are not limited to, isolated T cells.<br>
For example, T cells may be isolated from a patient by routine techniques (such as by<br>
Ficoll/Hypaque density gradient centrifugation of peripheral blood lymphocytes). T<br>
cells may be incubated in vitro for 2-9 days (typically 4 days) at 37°C with polypeptide<br>
(e.g., 5-25 ng/ml). It may be desirable to incubate another aliquot of a T cell sample<br>
in the absence of lung tumor polypeptide to serve as a control. For CD4+ T cells,<br>
activation is preferably detected by evaluating proliferation of the T cells. For CD8+ T<br>
cells, activation is preferably detected by evaluating cytolytic activity. A level of<br>
proliferation that is at least two fold greater and/or a level of cytolytic activity that is at<br>
least 20% greater than in disease-free patients indicates the presence of a cancer in the<br>
patient<br>
As noted above, a cancer may also, or alternatively, be detected based on<br>
the level of mRNA encoding a lung tumor protein in a biological sample. For example,<br>
at least two oligonucleotide primers may be employed in a polymerase chain reaction<br>
(PCR) based assay to amplify a portion of a lung tumor cDNA derived from a<br>
biological sample, wherein at least one of the oligonucleotide primers is specific for<br>
(i.e., hybridizes to) a polynucleotide encoding the lung tumor protein. The amplified<br>
cDNA is then separated and detected using techniques well known in the art, such as<br>
gel electrophoresis. Similarly, oligonucleotide probes that specifically hybridize to a<br>
polynucleotide encoding a lung tumor protein may be used hi a hybridization assay to<br>
detect the presence of polynucleotide encoding the tumor protein in a biological sample.<br>
To permit hybridization under assay conditions, oligonucleotide primers<br>
and probes should comprise an oligonucleotide sequence that has at least about 60%,<br>
preferably at least about 75% and more preferably at least about 90%, identity to a<br>
portion of a polynucleotide encoding a lung tumor protein that is at least 10 nucleotides,<br>
and preferably at least 20 nucleotides, in length. Preferably, oligonucleotide primers<br>
and/or probes hybridize to a polynucleotide encoding a polypeptide described herein<br>
under moderately stringent conditions, as defined above. Oligonucleotide primers<br>
and/or probes which may be usefully employed in the diagnostic methods described<br>
herein preferably are at least 10-40 nucleotides in length. In a preferred embodiment,<br>
the oligonucleotide primers comprise at least 10 contiguous nucleotides, more<br>
preferably at least 15 contiguous nucleotides, of a DNA molecule having a sequence<br>
recited in SEQ ID NO: 1,11-13,15,20,23-27,29,30, 33, 34,39,41,4346, 51, 52,57,<br>
58, 60, 62, 65-67, 69-71, 74, 76, 79, 80, 84, 86, 89-92, 95, 97, 98, 101,110, 111, 113-<br>
119, 121-128,130-134, 136, 138,139,141, 143, 146-151, 153,154, 157-160, 162-164,<br>
167-178,180,181,183,186-190,192,193,195-220,224,226-231,234,236, 237,240,<br>
241, 244-246, 248, 254, 255, 261, 262, 266, 270, 275, 280, 282, 283, 288, 289, 290,<br>
292, 295, 301, 303, 304, 309, 311, 341-782, 784, 785, 790, 792, 794, 796, 800-804,<br>
807, 808 or 810-826. Techniques for both PCR based assays and hybridization assays<br>
are well known in the art (see, for example, Mullis et al, Cold Spring Harbor Symp.<br>
Quant. Biol.,51:263,1987;Erliched.,PCR Technology, Stockton Press* NY, 1989).<br>
One preferred assay employs RT-PCR, in which PCR is applied in<br>
conjunction with reverse transcription. Typically, RNA is extracted from a biological<br>
sample, such as biopsy tissue, and is reverse transcribed to produce cDNA molecules.<br>
PCR amplification using at least one specific primer generates a cDNA molecule,<br>
which may be separated and visualized using, for example, gel electrophoresis.<br>
Amplification may be performed on biological samples taken from a test patient and<br>
from an individual who is not afflicted with a cancer. The amplification reaction may<br>
be performed on several dilutions of cDNA spanning two orders of magnitude. A twofold<br>
or greater increase in expression in several dilutions of the test patient sample as<br>
compared to the same dilutions of the non-cancerous sample is typically considered<br>
positive.<br>
In another embodiment, the compositions described herein may be used<br>
as markers for the progression of cancer. In this embodiment, assays as described<br>
above for the diagnosis of a cancer may be performed over time, and the change in the<br>
level of reactive polypeptide(s) or polynucleotide(s) evaluated. For example, the assays<br>
may be performed every 24-72 hours for a period of 6 months to 1 year, and thereafter<br>
performed as needed. In general, a cancer is progressing in those patients in whom the'<br>
level of polypeptide or polynucleotide detected increases over tune. In contrast, the<br>
cancer is not progressing when the level of reactive polypeptide or polynucleotide either<br>
remains constant or decreases with time.<br>
Certain in vivo diagnostic assays may be performed directly on a tumor.<br>
One such assay involves contacting tumor cells with a binding agent The bound<br>
binding agent may then be detected directly or indirectly via a reporter group. Such<br>
binding agents may also be used in histological applications. Alternatively,<br>
polynucleotide probes may be used within such applications.<br>
As noted above, to improve sensitivity, multiple lung tumor protein<br>
markers may be assayed within a given sample. It will be apparent that binding agents<br>
specific for different proteins provided herein may be combined within a single assay.<br>
Further, multiple primers or probes may be used concurrently. The selection of tumor<br>
protein markers may be based on routine experiments to determine combinations that<br>
results in optimal sensitivity. In addition, or alternatively, assays for-tumor proteins<br>
provided herein may be combined with assays for other known tumor antigens.<br>
DIAGNOSTIC KITS<br>
The present invention further provides kits for use within any of the<br>
above diagnostic methods. Such kits typically comprise two or more components<br>
necessary for performing a diagnostic assay. Components may be compounds,<br>
reagents, containers and/or equipment. For example, one container within a kit may<br>
contain a monoclonal antibody or fragment thereof that specifically binds to a lung<br>
tumor protein. Such antibodies or fragments may be provided attached to a support<br>
material, as described above. One or more additional containers may enclose elements,<br>
such as reagents or buffers, to be used in the assay. Such kits may also, or alternatively,<br>
contain a detection reagent as described above that contains a reporter group suitable<br>
for direct or indirect detection of antibody binding.<br>
Alternatively, a kit may be designed to detect the level of mRNA<br>
encoding a lung tumor protein in a biological sample. Such kits generally comprise at<br>
least one oligonucleotide probe or primer, as described above, that hybridizes to a<br>
polynucleotide encoding a lung tumor protein. Such an oligonucleotide may be used,<br>
for example, within a PCR or hybridization assay. Additional components that may be<br>
present within such kits include a second oligonucleotide and/or a diagnostic reagent or<br>
container to facilitate the detection of a polynucleotide encoding a lung tumor protein.<br>
The following Examples are offered by way of illustration and not by<br>
way of limitation.<br>
EXAMPLE 1<br>
IDENTIFICATION AND CHARACTERIZATION OF LUNG<br>
TUMOR PROTEIN cDNAS<br>
This Example illustrates the identification of cDNA molecules encoding<br>
lung tumor proteins.<br>
A. Isolation of cDNA Sequences from Lung Adenocarcinoma Libraries<br>
using Conventional cDNA Library Subtraction<br>
A human lung adenocarcinoma cDNA expression library was<br>
constructed from poly A+ RNA from patient tissues (# 40031486) using a Superscript<br>
Plasmid System for cDNA Synthesis and Plasmid Cloning kit (BRL Life Technologies,<br>
Gaithersburg, MD) following the manufacturer's protocol. Specifically, lung carcinoma<br>
tissues were homogenized with polytron (Kinematica, Switzerland) and total RNA was<br>
extracted using Trizol reagent (BRL Life Technologies) as directed by the<br>
manufacturer. The poly A+ RNA was then purified using an oligo dT cellulose column<br>
as described in Sambrook etal., Molecular Cloning: A Laboratory Manual, Cold<br>
Spring Harbor Laboratories, Cold Spring Harbor, NY, 1989. First-strand cDNA was<br>
synthesized using the NotI/Oligo-dT18 primer. Double-stranded cDNA was<br>
synthesized, ligated with BstXI/EcoRI adaptors (Invitrogen, San Diego, CA) and<br>
digested with Notl. Following size fractionation with cDNA size fractionation columns<br>
(BRL Life Technologies), the cDNA was ligated into the BstXI/NotI site of pcDNA3A<br>
(Invitrogen) and transformed into ElectroMax E. coli DH10B cells (BRL Life<br>
Technologies) by elect-operation. A total of 3 x 106 independent colonies were<br>
generated.<br>
Using the same procedure, a normal human cDNA expression library<br>
was prepared from a panel of normal tissue specimens, including lung, liver, pancreas,<br>
skin, kidney, brain and resting PBMC.<br>
cDNA library subtraction was performed using the above lung<br>
adenocarcinoma and normal tissue cDNA libraries, as described by Kara et al. (Blood,<br>
54:189-199, 1994) with some modifications. Specifically, a lung adenocarcinoma-<br>
specific subtracted cDNA library was generated as follows. The normal tissue cDNA<br>
library (80 ug) was digested with BamHI and Xhol, followed by a filling-in reaction<br>
with DNA polymerase Klenow fragment. After phenol-chloroform extraction and<br>
ethanol precipitation, the DNA was dissolved in 133 ul of HiO, heat-denatured and<br>
mixed with 133 ul (133 ug) of Photoprobe biotin (Vector Laboratories, Burlingame,<br>
CA). As recommended by the manufacturer, the resulting mixture was irradiated with a<br>
270 W sunlamp on ice for 20 minutes. Additional Photoprobe biotin (67 ul) was added<br>
and the biotinylation reaction was repeated. After extraction with butanol five times,<br>
the DNA was ethanol-precipitated and dissolved in 23 ul rfcO. The resulting DNA,<br>
plus other highly redundant cDNA clones that were frequently recovered in previous<br>
lung subtractions formed the driver DNA.<br>
To form the tracer DNA, 10 ug lung adenocarcinoma cDNA library was<br>
digested with NotI and Spel, phenol chloroform extracted and passed through Chroma<br>
spin-400 columns (Clontech, Palo Alto, CA). Typically, 5 jig of cDNA was recovered<br>
after the sizing column. Following ethanol precipitation, the tracer DNA was dissolved<br>
in 5 ul H2O. Tracer DNA was mixed with 15 ul driver DNA and 20 ul of 2 x<br>
hybridization buffer (1.5 M NaCl/10 mM EDTA/50 mM HEPES pH 7.5/0.2% sodium<br>
dodecyl sulfate), overlaid with mineral oil, and heat-denatured completely. The sample<br>
was immediately transferred into a 68 °C water bath and incubated for 20 hours (long<br>
hybridization [LH]). The reaction mixture was then subjected to a streptavidin<br>
treatment followed by phenol/chloroform extraction. This process was repeated three<br>
more times. Subtracted DNA was precipitated, dissolved in 12 ul H2O, mixed with 8 ul<br>
driver DNA and 20 ul of 2 x hybridization buffer, and subjected to a hybridization at 68 "<br>
°C for 2 hours (short hybridization [SH]). After removal of biotinylated doublestranded<br>
DNA, subtracted cDNA was ligated into Notl/Spel site of chloramphenicol<br>
resistant pBCSK* (Stratagene, La Jolla, CA) and transformed into ElectroMax E. coli<br>
DH10B cells by electroporation to generate a lung adenocarcinoma specific subtracted<br>
cDNA library, referred to as LAT-S1 Similarly, LAT-S2 was generated by including<br>
23 genes that were over-expressed in the tracer as additional drivers.<br>
A second human lung adenocarcinoma cDNA expression library was<br>
constructed using adenocarcinoma tissue from a second patient (# 86-66) and used to<br>
prepare a second lung adenocarcinoma-specific subtracted cDNA library (referred to as<br>
LAT2-S2), as described above, using the same panel of normal tissues and the<br>
additional genes over-expressed inLAT-Sl.<br>
A third human metastatic lung adenocarcinoma library was constructed<br>
from a pool of two lung pleura! effusions with lung and gastric adenocarcinoma origins.<br>
The subtracted cDNA library, Mets-sub2 was generated as described above using the<br>
same panel of normal tissues. However, the Mets-sub3 subtracted library was<br>
constructed by including 51 additional genes as drivers. These 51 genes were recovered<br>
in Mets-sub2, representing over-expressed housekeeping genes in the testers. As a<br>
result, Mets-sub3 is more complexed and normalized.<br>
A total of 16 cDNA fragments isolated from LAT-S1, 585 cDNA<br>
fragments isolated from LAT-S2, 568 cDNA clones from LAT2-S2, 15 cDNA clones<br>
from Mets-sub2 and 343 cDNA clones from Mets-sub3, described above, were colony<br>
PCR amplified and their mRNA expression levels in lung tumor, normal lung, and<br>
various other normal and tumor tissues were determined using microarray technology<br>
(Incyte, Palo Alto, CA). Briefly, the PCR amplification products were dotted onto<br>
slides in an array format, with each product occupying a unique location in the array.<br>
mRNA was extracted from the tissue sample to be tested, reverse transcribed, and<br>
fluorescent-labeled cDNA probes were generated. The microarrays were probed with<br>
the labeled cDNA probes, the slides scanned and fluorescence intensity was measured.<br>
This intensity correlates with the hybridization intensity. Seventy-three non-redundant<br>
cDNA clones, of which 42 were found to be unique, showed over-expression in lung<br>
tumors, with expression in normal tissues tested (lung, skin, lymph node, colon, liver,<br>
pancreas, breast, heart, bone marrow, large intestine, kidney, stomach, brain, small<br>
intestine, bladder and salivary gland) being either undetectable, or at significantly lower<br>
levels compared to lung adenocarcinoma tumors. These clones were further<br>
characterized by DNA sequencing with a Perkin Elmer/Applied Biosystems Division<br>
Automated Sequencer Model 373A and/or Model 377 (Foster City, CA).<br>
The sequences were compared to known sequences in the gene bank<br>
using the EMBL GenBank databases (release 96). No significant homologies were<br>
found to the sequence provided in SEQ ID NO: 67, with no apparent homology to<br>
previously identified expressed sequence tags (ESTs). The sequences of SEQ ID NO:<br>
60, 62, 65, 66, 69-71, 74, 76, 79, 80, 84, 86, 89-92, 95, 97 and 98 were found to show<br>
some homology to previously identified expressed sequence tags (ESTs). The cDNA<br>
sequences of SEQ ID NO: 59, 61, 63, 64, 67, 68, 72, 73, 75, 77, 78, 81-83, 85, 87, 88,<br>
93, 94, 96, 99 and 100 showed homology to previously identified genes. The fulllength<br>
cDNA sequences for the clones of SEQ ID NO: 96 and 100 are provided in SEQ<br>
ID NO: 316 and 318, respectively. The amino acid sequences for the clones of SEQ ID<br>
NO: 59, 61, 63, 64, 68, 73, 82, 83, 94, 96 and 100 are provided in SEQ ID NO: 331,<br>
328, 329, 332, 327, 333, 330, 326, 325, 324 and 335, respectively. A predicted amino<br>
acid sequence encoded by the sequence of SEQ ID NO: 69 (referred to as L552S) is<br>
provided in SEQ ID NO: 786.<br>
Further studies led to the isolation of an extended cDNA sequence, and<br>
open reading frame, for L552S (SEQ ID NO: 790). The predicted amino acid sequence<br>
encoded by the cDNA sequence of SEQ ID NO: 790 is provided in SEQ ID NO: 791.<br>
The determined cDNA sequence of an isoform of L552S is provided in SEQ ID NO:<br>
792, with the corresponding predicted amino acid sequence being provided in SEQ ID<br>
NO: 793. Subsequent studies led to the isolation of the full-length cDNA sequence of<br>
L552S (SEQ ID NO: 808). The corresponding amino acid sequence is provided in SEQ<br>
ID NO: 809. No homologies were found to the protein sequence of L552S. However,<br>
nucleotides 533-769 of the full-length cDNA sequence were found to show homology<br>
to a previously identified DNA sequence.<br>
Full-length cloning efforts on L552S led to the isolation of three<br>
additional cDNA sequences (SEQ ID NO: 810-812) from a metastatic lung<br>
adenocarcinoma library. The sequence of SEQ ID NO: 810 was found to show some<br>
homology to previously identified human DNA sequences.- The sequence of SEQ ID<br>
NO: 811 was found to show some homology to a previously identified DNA sequence.<br>
The sequence of SEQ ID NO: 812 was found to show some homology to previously<br>
identified ESTs.<br>
The gene of SEQ ID NO: 84 (referred to as L551S) was determined by<br>
real-time RT-PCR analysis to be over-expressed in 2/9 primary adenocarcinomas and to<br>
be expressed at lower levels in 2/2 metastatic adenocarcinomas and 1/2 squamous cell<br>
carcinomas. No expression was observed in normal tissues, with the exception of very<br>
low expression in normal stomach. Further studies on L551S led to the- isolation of the<br>
5' and 3' cDNA consensus sequences provided in SEQ ID NO: 801 and 802,<br>
respectively. The L551S 5' sequence was found to show some homology to the<br>
previously identified gene STY8 (cDNA sequence provided in SEQ ID NO: 803;<br>
corresponding amino acid sequence provided in SEQ ID NO: 805), which is a mitogen<br>
activated protein kinase phosphatase. However, no significant homologies were found<br>
to the 3' sequence of L551S. Subsequently, an extended cDNA sequence for L551S<br>
was isolated (SEQ ID NO: 804). The corresponding amino acid sequence is provided<br>
in SEQ ID NO: 806. Further studies led to the isolation of two independent full-length<br>
clones for L551S (referred to as 54298 and 54305). These two clones have five<br>
nucleotide differences compared to the STY8 DNA sequence. Two of these differences<br>
are single nucleotide polymorphisms which do not effect the encoded amino acid<br>
sequences. The other three nucleotide differences are consistent between the two<br>
L551S clones but lead to encoded amino acid sequences that are different from the<br>
STY8 protein sequence. The determined cDNA sequences for the L551S full-length<br>
clones 54305 and 54298 are provided in SEQ ID NO: 825 and 826, respectively, with<br>
the amino acid sequence for L551S being provided in SEQ ID NO: 827.<br>
B. Isolation of cDNA Sequences from Lung Adenocarcinoma Libraries<br>
using PCR-Based cDNA Library Subtraction<br>
cDNA clones from a PCR-based subtraction library, containing cDNA<br>
from a pool of two human lung primary adenocarcinomas subtracted against a pool of<br>
nine normal human tissue cDNAs including skin, colon, lung, esophagus, brain, kidney,<br>
spleen, pancreas and liver, (Clontech, Palo Alto, CA) were derived and submitted to a<br>
first round of PCR amplification. This library (referred to as ALT-1) was subjected to a<br>
second round of PCR amplification, following the manufacturer's protocol. The<br>
expression levels of 760 cDNA clones in lung tumor, normal lung, and various other<br>
normal and tumor tissues, were examined using microarray technology as described<br>
above. A total of 118 clones, of which 55 were unique, were found to be overexpressed<br>
hi lung tumor tissue, with expression in normal tissues tested (lung, skin,<br>
lymph node, colon, liver, pancreas, breast, heart, bone marrow, large intestine, kidney,<br>
stomach, brain, small intestine, bladder and salivary gland) being either-undetectable, or<br>
at significantly lower levels. The sequences were compared to known sequences in the<br>
gene bank using the EMBL. and GenBank databases (release 96). No significant<br>
homologies (including ESTs) were found to the sequence provided hi SEQ ID NO: 44.<br>
The sequences of SEQ ID NO: 1, 11, 13, 15, 20, 23-27, 29, 30, 33, 34, 39, 41, 43, 45,<br>
46, 51 and 57 were found to show some homology to previously identified expressed<br>
sequence tags (ESTs). The cDNA sequences of SEQ ID NO: 2-10, 12, 14, 16-19, 21,<br>
22, 28, 31, 32, 35-38,40, 42, 44,47-50, 52-56 and 58 showed homology to previously<br>
identified genes. The full-length cDNA sequences for the clones of SEQ ID NO: 18,<br>
22, 31, 35, 36 and 42 are provided in SEQ ID NO: 320, 319, 323, 321, 317, 321 and<br>
322, respectively, with the corresponding amino acid sequences being provided in SEQ<br>
ID NO: 337,336,340,338,334, and 339, respectively.<br>
Further studies led to the isolation of an extended cDNA sequence for<br>
the clone of SEQ ID NO: 33 (referred to as L801P). This extended cDNA sequence<br>
(provided in SEQ ID NO: 796), was found to contain three potential open reading<br>
frames (ORFs). The predicted amino acid sequences encoded by these three ORFs are<br>
provided in SEQ ID NO: 797-799, respectively.<br>
In subsequent studies, a full-length cDNA sequence for the clone of SEQ<br>
ID NO: 44 (referred to as L844P) was isolated (provided in SEQ ID NO: 800).<br>
Comparison of this sequence with those in the public databases revealed that the 470<br>
bases at the 5' end of the sequence show homology to the known gene dihydrodiol<br>
dehydrogenase, thus indicating that L844P is a novel transcript of the dihydrodiol<br>
dehydrogenase family having 2007 base pairs of previously unidentified 3'<br>
untranslated region.<br>
The predicted amino acid sequence encoded by the sequence of SEQ ID<br>
NO: 46 (referred to as L840P) is provided in SEQ ID NO: 787. An extended cDNA<br>
sequence for L840P, which was determined to include an open reading frame, is<br>
provided hi SEQ ID NO: 794. The predicted amino acid sequence encoded by the<br>
cDNA sequence of SEQ ID NO: 794 is provided in SEQ ID NO: 795. The full-length<br>
cDNA sequence for the clone of SEQ ID NO: 54 (referred to as L548S) is provided in<br>
SEQ ID NO: 788, with the corresponding amino acid sequence being provided in SEQ<br>
ID NO: 789.<br>
Northern blot analyses of the genes of SEQ ID NO: 25 and 46 (referred<br>
to as L839P and L840P, respectively) were remarkably similar. Both genes were<br>
expressed in 1/2 lung adenocarcinomas as two bands of 3.6 kb and 1.6 kb. No<br>
expression of L839P was observed in normal lung or trachea. No expression of L840P<br>
was observed in normal bone marrow, resting or activated PBMC, esophagus, or<br>
normal lung." Given the similar expression patterns, L839P and L840P may be derived<br>
from the same gene.<br>
Further studies on L773P (SEQ ID NO: 58) resulted in the isolation of<br>
the extended consensus cDNA sequence provided hi SEQ ID NO: 807.<br>
Additional lung adenocarcinoma cDNA clones were isolated as follows.<br>
A cDNA library was prepared from a pool of two lung adenocarcinomas and subtracted<br>
against cDNA from a panel of normal tissues including lung, brain, liver, kidney,<br>
pancreas, skin, heart and spleen. The subtraction was performed using a PCR-based<br>
protocol (Clontech), which was modified to generate larger fragments. Within this<br>
protocol, tester and driver double stranded cDNA were separately digested with five<br>
restriction enzymes that recognize six-nucleotide restriction sites (Mlul, MscI, PvuII,<br>
Sail and StuI). This digestion resulted in an average cDNA size of 600 bp, rather than<br>
the average size of 300 bp that results from digestion with Rsal according to the<br>
Clontech protocol. The ends of the restriction digested tester cDNA were filled hi to<br>
generate blunt ends for adapter ligation. This modification did not affect the subtraction<br>
efficiency. Two tester populations were then created with different adapters, and the<br>
driver library remained without adapters. The tester and driver libraries were then<br>
hybridized using excess driver cDNA. In the first hybridization step, driver was<br>
separately hybridized with each of the two tester cDNA populations. This resulted in<br>
populations of (a) unhybridized tester cDNAs, (b) tester cDNAs hybridized to other<br>
tester cDNAs, (c) tester cDNAs hybridized to driver cDNAs and (d) unhybridized<br>
driver cDNAs. The two separate hybridization reactions were then combined, and<br>
rehybridized in the presence of additional denatured driver cDNA. Following this<br>
second hybridization, in addition to populations (a) through (d), a fifth population (e)<br>
was generated in which tester cDNA with one adapter hybridized to tester cDNA with<br>
the second adapter. Accordingly, the second hybridization step resulted in enrichment<br>
of differentially expressed sequences which could be used as templates for PCR<br>
amplification with adaptor-specific primers.<br>
The ends were then filled in, and PCR amplification was performed<br>
using adaptor-specific primers. Only population (e), which contained tester cDNA that<br>
did not hybridize to driver cDNA, was amplified exponentially. A second PCR<br>
amplification step was then performed, to reduce background and further enrich<br>
differentially expressed sequences.<br>
Fifty-seven cDNA clones were isolated from the subtracted library<br>
(referred to as LAP1) and sequenced. The determined cDNA sequences for 16 of these<br>
clones are provided in SEQ ID NO: 101-116. The sequences of SEQ ID NO: 101 and<br>
114 showed no significant homologies to previously identified sequences. The<br>
sequences of SEQ ID NO: 102-109 and 112 showed some similarity to previously<br>
identified sequences, while the sequences of SEQ ID NO: 113, 115 and 116 showed<br>
some similarity to previously isolated ESTs.<br>
C. Isolation of cDNA Sequences from Small Cell Lung Carcinoma<br>
Libraries using PCR-Based cDNA Library Subtraction<br>
A subtracted cDNA library for small cell lung carcinoma (referred to as<br>
SCL1) was prepared using essentially the modified PCR-based subtraction process<br>
described above. cDNA from small cell lung carcinoma was subtracted against cDNA<br>
from a panel of normal tissues, including normal lung, brain, kidney, liver, pancreas,<br>
skin, heart, lymph node and spleen. Both tester and driver poly A+ RNA were initially<br>
amplified using SMART PCR cDNA synthesis kit (Clontech, Palo Alto, CA). The<br>
tester and driver double stranded cDNA were separately digested with five restriction<br>
enzymes (Oral, MscI, PvuII, Smal, and StuI). These restriction enzymes generated<br>
blunt end cuts and the digestion resulted in an average insert size of 600 bp. Digestion<br>
with this set of restriction enzymes eliminates the step required to generate blunt ends<br>
by filling in of the cDNA ends. These modifications did not affect subtraction<br>
efficiency.<br>
Eighty-five clones were isolated and sequenced. The determined cDNA<br>
sequences for 31 of these clones are provided in SEQ ID NO: 117-147: The sequences<br>
of SEQ ID NO: 122, 124, 126, 127, 130, 131, 133, 136, 139 and 147 showed no<br>
significant homologies to previously identified sequences. The sequences of SEQ ID<br>
NO: 120, 129, 135, 137, 140, 142, 144 and 145 showed some similarity to previously<br>
identified gene sequences, while the sequences of SEQ ID NO: 114,118,119,121,123,<br>
125, 128, 132, 134, 138, 141, 143 and 147 showed some similarity to previously<br>
isolated ESTs.<br>
In further studies, three additional cDNA libraries were generated from<br>
poly A+ RNA from a single small cell lung carcinoma sample subtracted against a pool<br>
of poly A+ RNA from nine normal tissues (lung, brain, kidney, liver, pancreas, skin,<br>
heart pituitary gland and spleen). For the first library (referred to as SCL2), the<br>
subtraction was carried out essentially as described above for the LAPI library, with the<br>
exception that the tester and driver were digested with PvuII, StuI, MscI and Dral. The<br>
ratio of tester and driver cDNA used was as recommended by Clontech. For the second<br>
library (referred to as SCL3), subtraction was performed essentially as for SCL2 except<br>
that cDNA for highly redundant clones identified from the SCL2 library was included<br>
in the driver cDNA. Construction of the SCL4 library was performed essentially as<br>
described for the SCL3 library except that a higher ratio of driver to tester was<br>
employed.<br>
Each library was characterized by DNA sequencing and database<br>
analyses. The determined cDNA sequence for 35 clones isolated from the SCL2 library<br>
are provided in SEQ ID NO: 245-279, with the determined cDNA sequences for 21<br>
clones isolated from the SCL3 library and for 15 clones isolated from the SCL4 library<br>
being provided in SEQ ID NO: 280-300 and 301-315, respectively. The sequences of<br>
SEQ ID NO: 246,254,261,262,304,309 and 311 showed no significant homologies to<br>
previously identified sequences. The sequence of SEQ ID NO: 245,248,255,266,270,<br>
275, 280, 282, 283, 288-290, 292, 295, 301 and 303 showed some homology to<br>
previously isolated ESTs, while the sequences of SEQ ID NO: 247, 249-253, 256-260,<br>
263-265, 267-269, 271-274, 276-279,281, 284-287, 291, 293, 294, 296-300, 302, 305-<br>
308,310 and 312-315 showed some homology to previously identified gene sequences.<br>
D. Isolation of cDNA Sequences from a Neuroendocrine Library using<br>
PCR-Based cDNA Library Subtraction<br>
Using the modified PCR-based subtraction process, essentially as<br>
described above for the LAP1 subtracted library, a subtracted cDNA library (referred to<br>
as MLN1) was derived from a lung neuroendocrine carcinoma that had metastasized to<br>
the subcarinal lymph node, by subtraction with a panel of nine normal tissues, including<br>
normal lung, brain, kidney, liver, pancreas, skin, heart, lymph node and spleen.<br>
Ninety-one individual clones were isolated and sequenced. The<br>
determined cDNA sequences for 58 of these clones are provided in SEQ ID NO: 147-<br>
222. The sequences of SEQ ID NO: 150, 151, 154, 157, 158, 159, 160, 163, 174, 175,<br>
178, 186-190, 192, 193, 195-200, 208-210, 212-215 and 220 showed no significant<br>
homologies to previously identified sequences. The sequences of SEQ ID NO: 152,<br>
155,156,161, 165,166, 176,179, 182, 184,185, 191,194, 221 and 222 showed some<br>
similarity to previously identified gene sequences, while the sequences of SEQ ID NO:<br>
148, 149, 153, 164, 167-173, 177, 180, 181, 183, 201-207, 211 and 216-219 showed<br>
some similarity to previously isolated ESTs.<br>
The determined cDNA sequences of an additional 442 clones isolated<br>
from the MLN1 library are provided in SEQ ID NO: 341-782.<br>
E. Isolation of cDNA Sequences from a Squamous Cell Lung Carcinoma<br>
Library using PCR-Based cDNA Library Subtraction<br>
A subtracted cDNA library for squamous cell lung carcinoma (referred<br>
to as SQL1) was prepared, essentially using the modified PCR-based subtraction<br>
process described above, except the tester and driver double stranded cDNA were<br>
separately digested with four restriction enzymes (Dral, MscI, PvuII and StuI) cDNA<br>
from a pool of two squamous cell lung carcinomas was subtracted against cDNA from a<br>
pool of 10 normal tissues, including normal lung, brain, kidney, liver, pancreas, skin,<br>
heart, spleen, esophagus and trachea.<br>
Seventy-four clones were isolated and sequenced. The determined<br>
cDNA sequences for 22 of these clones are provided in SEQ ID NO: 223-244. The<br>
sequence of SEQ ID NO: 241 showed no significant homologies to previously<br>
identified sequences. The sequences of SEQ ID NO: 223, 225, 232,233,235,238,239,<br>
242 and 243 showed some similarity to previously identified gene sequences, while the<br>
sequences of SEQ ID NO: 224,226-231,234,236, 237,240,241 and 244 showed some<br>
similarity to previously isolated ESTs.<br>
The sequences of an additional 12 clones isolated during chracterization<br>
of cDNA libraries prepared from lung tumor tissue are provided in SEQ ID NO: 813-<br>
824. Comparison of these sequences with those in the GenBank database and the<br>
GeneSeq DNA database revealed no significant homologies to previously identified<br>
sequences.<br>
EXAMPLE 2<br>
SYNTHESIS OF POLYPEPTIDES<br>
Polypeptides may be synthesized on a Perkin Elmer/Applied Biosystems<br>
Division 430A peptide synthesizer using FMOC chemistry with HPTU (OBenzotriazole-<br>
N,N,N',N'-teteamernyluromum hexafluorophosphate) activation. A Gly-<br>
Cys-Gly sequence may be attached to the ammo terminus of the peptide to provide a<br>
method of conjugation, binding to an immobilized surface, or labeling of the peptide.<br>
Cleavage of the peptides from the solid support may be carried out using the following<br>
cleavage mixture: trifluoroacetic acid:ethanedithiol:thioanisole:water:phenol<br>
(40:1:2:2:3). After cleaving for 2 hours, the peptides may be precipitated in cold<br>
methyl-t-butyl-ether. The peptide pellets may then be dissolved in water containing<br>
0.1% trifluoroacetic acid (TFA) and lyophilized prior to purification by CIS reverse<br>
phase HPLC. A gradient of 0%-60% acetonitrile (containing 0.1% TFA) hi water<br>
(containing 0.1% TFA) may be used to elute the peptides. Following lyophilization of<br>
the pure fractions, the peptides may be characterized using electrospray or other types<br>
of mass spectrometry and by amino acid analysis.<br>
EXAMPLES<br>
PREPARATION OF ANTIBODIES AGAINST LUNG CANCER ANTIGENS<br>
Polyclonal antibodies against the lung cancer antigen L773P (SEQ ID<br>
NO: 783) were prepared as follows.<br>
Rabbits were immunized with recombinant protein expressed in and<br>
purified from E. coli as described above. For the initial immunization, 400 ng of<br>
antigen combined with muramyl dipeptide (MDP) was injected subcutaneously (S.C.).<br>
Animals were boosted S.C. 4 weeks later with 200 fig of antigen mixed with incomplete<br>
Freund's Adjuvant (IFA). Subsequent boosts of 100 fig of antigen mixed with IFA<br>
were injected S.C. as necessary to induce high antibody titer responses. Serum bleeds<br>
from immunized rabbits were tested for L773P-specific reactivity using ELISA assays<br>
with purified protein and showed strong reactivity to L733P. Polyclonal antibodies<br>
against L773P were affinity purified from high titer polyclonal sera using purified<br>
protein attached to a solid support.<br>
EXAMPLE 4<br>
PROTEIN EXPRESSION OF LUNG TUMOR-SPECIFIC ANTIGENS<br>
Full-length L773P (amino acids 2-364 of SEQ ID NO: 783), with a 6X<br>
His Tag, were subcloned into the pPDM expression vector and transfonned into either<br>
BL21 CodonPlus or BL21 pLysS host cells using standard techniques. High levels of<br>
expression were observed in both cases. Similarly, the N-terminal portion of L773P<br>
(amino acids 2-71 of SEQ ID NO: 783; referred to as L773PA), with a 6X His tag were<br>
subcloned into the vector pPDM and transfonned into BL21 CodonPlus host cells. Low<br>
levels of expression were observed by N-tenninal sequencing. The sequence of the<br>
expressed constructs for L773P and L773PA are provided in SEQ ID NO: 784 and 785,<br>
respectively.<br>
From the foregoing it will be appreciated that, although specific<br>
embodiments of the invention have been described herein for purposes of illustration,<br>
various modifications may be made without deviating from the spirit and scope of the<br>
invention. Accordingly, the invention is not limited except as by the appended claims.<br><br><br><br><br>
WE CLAIM:<br>
1. A polypeptide for the prevention, treatment and diagnosis of lung cancer, comprising an immunogenic portion of a lung tumor protein, wherein the lung tumor protein comprises an arnino acid sequence recited in SEQ ID NO:809<br>
(Met Arg Cys His Ala His Gly Pro Ser Cys Leu Val Thr Ala Ile Thr Arg Glu Glu<br>
Gly Gly Pro Arg Ser Gly Gly Ala Gln Ala Lys Leu Gly Cys Cys Trp Gly Tyr Pro<br>
Ser Pro Arg Ser Thr Trp Asn Pro Asp Arg Arg Phe Trp Thr Pro Gln Thr Gly Pro<br>
Gly Glu Gly Arg His Glu Arg His Thr Gln Thr Gln Asn His Thr Ala Ser Pro Arg<br>
Ser Pro Val Met Glu Ser Pro Lys Lys Lys Asn Gln Gln Leu Lys Val Gly Ile Leu<br>
His Leu Gly Ser Arg Gln Lys Lys Ile Arg Ile Gln Leu Arg Ser Gln Cys	Ala<br>
Thr Trp Lys Val Ile Cys Lys Ser Cys Ile Ser Gln Thr Pro Gly Ile Asn Leu Asp Leu Gly Ser Gly Val Lys Val Lys Ile He Pro Lys Glu Glu His Cys Lys Met Pro Glu Ala Gly Glu Glu Gln Pro Gln Val; SEQ ID NO:809)<br>
that is encoded by a polynucleotide sequence set forth in SEQ ID NO:808<br>
(GCGGCGGAGCTGTGAGCCGGCGACTCGGGTCCCTGAGGTCTGGATTCTTTCTC<br>
CGCTACTGAGACACGGCGGGTAGGTCCACAGGCAGATCCAACTGGGAGTTGAA<br>
GTGTGAGTGAGAGTGAAGAGGAACCAGCAGGCTTCCGGAGGGTTGTGTGGTCA<br>
GTGACTCAGAGTGAGAAGGCCCTCGAAGTCGTCGTCCCTCTCATGCGGTGCCAC<br>
GCCCATGGACCTTCTTGTCTCGTCACGGCCATAACTAGGGAGGAAGGAGGGCC<br>
GAGGAGTGGAGGGGCTCAGGCGAAGCTGGGGTGCTGTTGGGGGTATCCGAGT<br>
CCCAGAAGCACCTGGAACCCCGACAGAAGATTCTGGACTCCCCAGACGGGACC<br>
AGGAGAGGGACGGCATGAGCGACACACACAAACACAGAACCACACAGCCAGTC<br>
CCAGGAGCCCAGTAATGGAGAGCCCCAAAAAGAAGAACCAGCAGCTGAAAGTC<br>
GGGATCCTACACCTGGGCAGCAGACAGAAGAAGATCAGGATACAGCTGAGATCC<br>
CAGTGCGCGACATGGAAGGTGATCTGCAAGAGCTGCATCAGTCAAACACCGGG<br>
GATAAATCTGGATTTGGGTTCCGGCGTCAAGGTGAAGATAATACCTAAAGAGGAA<br>
CACTGTAAAATGCCAGAAGCAGGTGAAGAGCAACCACAAGTTTAAATGAAGACAA<br>
GCTGAAACAACGCAAGCTGGTTRTATATTAGATATTTGACTTAAACTATCTCAATA<br>
AAGTTTTGCAGCTTTCACCAAAAAAAAAAAA; SEQ ID NO:808),<br>
and wherein the immunogenic portion comprises at least 100 amino acid residues and is specifically bound by a B-cell and/or T-cell surface antigen receptor specific for the lung tumor protein of SEQ ID NO:809.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAwMzYtZGVsLWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-00036-del-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDAwMzYtREVMLUNsYWltcy0oMjYtMTItMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-00036-DEL-Claims-(26-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAwMzYtZGVsLWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00036-del-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDAwMzYtREVMLUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjYtMTItMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-00036-DEL-Correspondence-Others-(26-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAwMzYtZGVsLWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00036-del-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAwMzYtZGVsLWRlc2NyaXB0aW9uIChjb21wbGV0ZSktMjQtMDctMjAwOC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00036-del-description (complete)-24-07-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAwMzYtZGVsLWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-00036-del-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAwMzYtZGVsLWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00036-del-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAwMzYtZGVsLWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-00036-del-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAwMzYtZGVsLWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00036-del-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAwMzYtZGVsLWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00036-del-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAwMzYtZGVsLWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00036-del-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAwMzYtZGVsLWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00036-del-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDAwMzYtREVMLU90aGVycy1Eb2N1bWVudC0oMjYtMTItMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-00036-DEL-Others-Document-(26-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAwMzYtZGVsLXBjdC0xMDEucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-00036-del-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAwMzYtZGVsLXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-00036-del-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAwMzYtZGVsLXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-00036-del-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAwMzYtZGVsLXBjdC00MDEucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-00036-del-pct-401.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAwMzYtZGVsLXBjdC00MDgucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-00036-del-pct-408.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAwMzYtZGVsLXBjdC00MDkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-00036-del-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAwMzYtZGVsLXBjdC00MTYucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-00036-del-pct-416.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMzYtREVMLUFic3RyYWN0LSgyNC0wNy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-36-DEL-Abstract-(24-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMzYtREVMLUNsYWltcy0oMjQtMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-36-DEL-Claims-(24-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMzYtREVMLUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjQtMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-36-DEL-Correspondence-Others-(24-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMzYtREVMLUZvcm0tMS0oMjQtMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-36-DEL-Form-1-(24-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMzYtZGVsLWZvcm0tMTMtKDI0LTA3LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-36-del-form-13-(24-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMzYtREVMLUZvcm0tMi0oMjQtMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-36-DEL-Form-2-(24-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMzYtREVMLUZvcm0tMy0oMjQtMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-36-DEL-Form-3-(24-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMzYtREVMLUdQQS0oMjQtMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-36-DEL-GPA-(24-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMzYtREVMLU90aGVycy1Eb2N1bWVudC0oMjQtMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-36-DEL-Others-Document-(24-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMzYtREVMLVBldGl0aW9uLTEzNy0oMjQtMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-36-DEL-Petition-137-(24-07-2008).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="228500-a-snap-action-rotary-switch.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="228502-a-circuit-method-for-providing-d-c-current-from-a-power-source.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>228501</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2002/00036/DEL</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>08/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>20-Feb-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>05-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>09-Jan-2002</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CORIXA CORPORATION</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>SUITE 200, 1124 COLUMBIA STREET, SEATTLE, WASHINGTON 98104, UNITED STATES OF AMERICA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>MARC W. RETTER</td>
											<td>33402 NE 43RD PLACE, CARNATION, WA 98104, U.S.A.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>JANE MANNION</td>
											<td>8904-192ND STREET SW, EDMONDS, WA 98026, U.S.A.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>THOMAS S. VEDVICK</td>
											<td>124 SOUTH 300TH PLACE, FEDERAL WAY, WA 98003, U.S.A.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>DARRICK CARTER</td>
											<td>321, SUMMIT AVENUE EAST, SEATTLE, WA 98102, U.S.A.</td>
										</tr>
										<tr>
											<td>5</td>
											<td>TONGTONG WANG</td>
											<td>8049 NE 28TH STREET, MEDINA, WA 98039, U.S.A.</td>
										</tr>
										<tr>
											<td>6</td>
											<td>GARY R. FANGER</td>
											<td>15906-29TH, DRIVE SE, MILL CREEK, WA 98012, U.S.A.</td>
										</tr>
										<tr>
											<td>7</td>
											<td>CHAITANYA S. BANGUR</td>
											<td>2102 NORTH 105TH STREET, APT. J101, SEATTLE, WA 98133, U.S.A.</td>
										</tr>
										<tr>
											<td>8</td>
											<td>MICHAEL J. LODES</td>
											<td>9223-36TH AVENUE SW, SEATTLE, WA 98126, U.S.A.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/12</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US00/18061</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-06-30</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>09/346,492</td>
									<td>1999-06-30</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>09/419,356</td>
									<td>1999-10-15</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>09/466,867</td>
									<td>1999-12-17</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>4</td>
									<td>09/546,259</td>
									<td>2000-04-10</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>5</td>
									<td>09/589,184</td>
									<td>2000-06-05</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/228501-an-isolated-polypeptide-comprising-an-immunogenic-portion by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:05:45 GMT -->
</html>
