<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/228636-method-for-generating-a-genetically-modified-organism-for-screening-active-substances by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:14:44 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 228636:METHOD FOR GENERATING A GENETICALLY MODIFIED ORGANISM FOR SCREENING ACTIVE SUBSTANCES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHOD FOR GENERATING A GENETICALLY MODIFIED ORGANISM FOR SCREENING ACTIVE SUBSTANCES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention relates to a method for generating a genetically modified organism for screening active substances comprising the following steps: (a) inducing the heterologous expression of at least one protein or protein fragment by genetic modification of the organism; (b) analysis of the modified gene expression pattern and identification of genes that have undergone compensator}&#x27; and differential regulation and (c) phenotyping of the organism. The invention also relates to a genetically modified organism that can be obtained by said method.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
METHOD FOR GENERATING A GENETICALLY MODIFIED ORGANISM FOR SCREENING ACTIVE SUBSTANCES<br>
Genetically modified yeasts which express heterologously the target protein which is to be inhibited by the substance to be tested are known to be used for drug screening. Heterologous expression means, within the scope of the present invention, expression of a gene foreign to the organism or expression of a gene endogenous to the organism with an altered expression pattern, in particular enhanced or reduced expression and/or an expression which is altered with respect to time and/or space (e.g. other compartments, in higher organisms other tissues, for example). In the simplest case, heterologous expression leads to a detectable modified phenotype, usually a growth inhibition, of the yeast. Growth inhibition means, within the scope of the present invention, a reduced rate of proliferation and/or a reduced growth in size and also includes cell death (apoptotic or necrotic). The type of growth inhibition occurring also depends on the organism; thus, in yeasts either a proliferation arrest or a lysis can be observed, whereas in eukaryotic cells which are originally derived from multicellular organisms apoptosis can also sometimes be observed. If heterologous expression results in a modification of the behavior and/or the morphology of the organism, which is perceptible from the outside (i.e. a modified phenotype), the genetically modified organism can readily be used for drug screening, the efficacy of the substances tested being determinable on the basis of their ability to eliminate or reduce the phenotype (e.g. growth inhibition). In the example of the yeast system with growth inhibition as modified phenotype. this is preferably carried out by simple growth assays which are also suitable for high throughput screening (HTS). Any alteration, perceptible from the outside, of the genetically modified organism (shape, size, etc.) or of its behavior (growth, rate of cell division, etc.) in comparison with the genetically unmodified organism or with the organism which does not express the heterologous protein(s) or protein fragment(s) is referred to as modified phenotype. Phenotyping thus refers to causing such a modification.<br>
However, this method of the prior art has the disadvantage of only a small proportion of heterologously expressed genes producing a phenotype of<br><br>
the genetically modified organism, whicn is usable for drug screening. Thus it is assumed that, for example, only aoprox. 20-30% of all heterologously expressed kinases cause a growth innibition in the yeast, which can be utilized for drug screening. In the case of the remaining 70-80%, growth inhibition is so low that it cannot" be used for screening (too small a difference in comparison with the control leads to a high background and thus to too large a number of false positives) or it is not present at all.<br>
There exists, therefore, the need for a method for generating a genetically engineered organism for drug screening which does not have the disadvantages of the prior art and is. in particular, suitable for making accessible to drug screening also those heterologously expressed genes which do not produce any phenotype or any phenotype usable for screening, in particular for HTS, in the organism in which they are heterologously expressed.<br>
According to the invention, this object is achieved by a method for generating a genetically modified organism for drug screening, which comprises the steps<br>
a)	causing heterologous expression of at least one protein or protein fragment by genetic modification of the organism.<br>
b)	preferably, this is followed by determining the phenotype of the genetically modified organism.<br>
c)	analyzing the modified gene expression pattern and identifying compensating differentially regulated genes.<br>
d)	phenotyping the organism (preferably by deletion, mutagenesis or overexpression of the compensatingly regulated genes to enhance or generate a phenotype in combination with the heierologously expressed protein or protein fragment).<br>
The invention is based on tne finding by the inventors that the lack of a detectable phenotype for heterologous expression of most genes is based on the fact that the genetically modified organism up- or downregulates (i.e. compensatingly differentially regulates) the expression of some genes as<br><br>
response to expression of the heterologously expressed protein or protein fragment. Differentially regulated means, in this case, regulated differently than in the genetically modified organism or without heterologous expression of the heterologously expressed protein or protein fragment. Compensatingly means that that differential gene regulation is a response to heterologous expression of the protein or protein fragment.<br>
The invention makes possible the development of a platform technology in a cellular model, preferably the yeast, in contrast to the simple biochemical model. Using the assay system it is possible, for example, to identify inhibitors from chemical libraries, from CombiChem libraries and from extracts of natural substances. The assay system can be adapted to 96-, 384- or 1 536-well plates or to other formats common for cellular assays. The format to be chosen depends partly also on the chosen organism, the selection being within the ability of the skilled worker.<br>
The method of the invention is particularly suitable for genes and proteins or protein fragments whose heterologous expression in the desired organism does not result in any detectable modification of the phenotype in comparison with the genetically unmodified organism or the organism which does not heterologously express said protein or protein fragment. It is possible, for example, to assay protein kinases as well as other gene products which cause a transcriptional response. It may, however, also be applied to a detectably modified phenotype, in particular if a modified phenotype, although detectable, is not suitable or not appropriate for the use in drug screening, due to particular reasons. Said phenotype may be enhanced by phenotyping or modified in such a way that it can be used for drug screening. Accordingly, phenotyping refers, within the scope of the Dresent invention, to causing or enhancing a phenotype in the genetically modified organism expressing heterologously the protein(s) or protein ragment(s), which phenotype can be distinguished from the organism vhich does not heterologously express the protein(s) or protein fragment(s) )r from the genetically unmodified organism.<br>
Suitable organisms are preferably ceiis, here eukaryotic as well as &gt;rokaryotic cells, or else multicellular nonhuman organisms which are suitable for drug screening, for example Drosophila and preferably s. elegans. Suitable eukaryotic cells are preferably cultured cell lines which vere originally obtained from multicellular organisms, for example 3T3,<br><br>
CHO, HeLa, or else other or eukaryotic unicellular organisms, in particular yeasts. Particularly suitable among the yeasts are, in turn, those of the strains S. cerevisiae or S. pombe. Suitable laboratory strains of yeast cells or suitable eukaryotic cell lines are sufficiently well known to the skilled "Worker.<br>
Suitable proteins and protein fragments are in principle all those whose heterologous expression in the organism results in an alteration of the expression pattern of endogenous genes. Advantageous are all proteins and protein fragments which are of interest with respect to finding new active substances, with kinases, phosphatases, GPCRs, (in particular small) GTPases, proteases and ion channels being particularly preferred within the scope of the present invention.<br>
The term drug screening comprises, within the scope of the present invention, any type of search for substances which act on the activity of one or more particular target genes and/or target proteins, using at least one genetically modified organism. In principle, any types of substances are suitable here, for example any types of natural substances (i.e. molecules occurring in nature, in particular biomolecules) as well as not naturally occurring, synthetically produced chemicals and substances/derivatives derived from natural substances, in particular biological molecules (e.g. modified peptides or oligonucleotides).<br>
Heterologous expression may comprise the introduction of a foreign gene or else the modified expression of a gene endogenous to the organism, for example by introducing an appropriate expression vector. The genetic modification required therefor may concern the modification of the genome of the organism (e.g. by means of stable vectors integrating into the genome or by various types of mutagenesis), may be episomal or may comprise simply the introduction of suitable vectors which require constant selection by means of one or more selection markers in order to remain in the organism. The most suitable type aepends on various factors, inter alia also on the type of organism, and can be readily determined by the competent skilled worker.<br>
The heterologous expression relates to at least one protein or protein fragment but may also relate to a plurality of proteins or protein fragments. It   may    be    expedient   to   verify   expression    of   the    heterologous<br><br>
protein/fragment by suitable methods (PCR, Northern blot, Western blot, etc.), before the gene expression pattern of the genetically modified organism is compared, and thus analyzed, with the organism lacking expression of said heterologous protein. The analysis is carried out by suitable measures which are sufficiently well known to the skilled worker, the use of array (preferably DNA/RNA or protein microarrays) or chip systems being particularly suitable for this purpose. By comparing the expression patterns of a control organism (e.g. a wild-type organism or an organism into which merely the empty vector has been introduced or, for inducible systems, the genetically modified organism in which expression of the heterologous gene has not been induced) and of the genetically modified organism expressing the heterologous gene. Such gene products which appear at all/to an increased/reduced extent or not at all in the expression pattern of the genetically modified organism expressing the heterologous gene in contrast to the expression pattern of the control organism are thus regarded as compensatingly differentially regulated genes and may be used for phenotyping said genetically modified organism.<br>
Phenotyping refers to causing or enhancing a phenotype distinguishable from the wild-type organism in the genetically modified organism (or, for inducible systems, a phenotype which is only produced by the genetically modified organism with heterologous expression of the protein(s) or protein fragment(s) and which is not produced in the noninduced state of said organism, when the protein(s) or protein fragment(s) are not expressed), with the phenotype being preferably suitable for evaluation in HTS drug screening. Said causing or enhancing may take place here, for example, on reducing or eliminating expression of one or more compensatingly upreguiated genes. (This may be carried out, for example, by genomic knock out of one or more of the compensatingly differentially regulated genes or by mutagenesis) or enhanced expression of one or more compensatingly downregulated genes. (This may be carried out, for example, by heterologous expression cf one or more compensatingly differentially downregulated genes, using suitable expression vectors.) In this way it is possible to produce a phenotype, endogenous to the organism and caused by the heterologously expressed gene, which phenotype has been prevented due to compensatingly differential regulation of one or more genes (preferably growth inhibition, but, in particular in multicellular organisms, other phenotypes are also possible here).<br><br>
Another possibility is also to label one or more compensatingly upregulated genes by means of a suitable marker/tag (which is coupled to the gene product, for example) or by means of a reporter which is under the control of the enhancer and/or promoter of the compensatingly upregulated gene and which is introduced into the organism. Suitable reporters are known to the skilled worker, and suitable here are, in particular, any types of luminescent proteins (e.g. GFP, BFP, etc.) or else other reporters capable of generating a detectable signal (e.g. luciferase, (J-galactosidase) and growth markers for auxotrophic strains such as, for example, HIS3, URA3, LEU2, TRP1, and antibiotic resistance genes such as, for example, for kanamycin or G418. Other types of phenotyping are also conceivable.<br>
Following phenotyping, it is expedient to check the success of said phenotyping by suitable methods (e.g. measuring the rate of proliferation, cell counting or determination of size or morphology, etc. and comparison with the phenotype of heterologous expression not taking place).<br>
According to a preferred form of carrying out the method of the invention, phenotyping is carried out by means of deletion, mutagenesis or overexpression of at least one compensatingly regulated gene.<br>
According to a preferred embodiment, phenotyping is carried out by reducing/eliminating the compensatingly differential expression or by labeling at least one compensatingly differentially regulated gene.<br>
In this connection, heterologous expression may result in compensatory up- and also downregulation of at least one gene endogenous to the organism but may also result in one or more genes being upregulated and one or more other genes being downregulated.<br>
It is also particularly convenient if heterologous expression of the protein or protein fragment is inducible. Suitable systems are known to the competent skilled worker, suitable examples thus being galactose- or copper-regulated promoters, the Tet-On Tet-Off system, etc. This may involve either inducibly switching on expression of a gene foreign or endogenous to the organism (inducible knock in) or inducibly reducing or completely switching off expression of a gene endogenous to the organism (inducible knock out). To this end, the genetic modification expediently comprises introducing a<br><br>
vector enabling inducible expression of the protein or protein fragment, preferably one with galactose- (GAL1/GAL10) or copper- (CUP1) regulated promoters,, tetracycline-inducible vector or tissue-specifically inducible promoters such as, for example, hsp 16-2, unc-119, unc-54, mec-7, or myo-3 in C\ elegans..<br>
According to a preferred embodiment, the organism is C. elegans, a prokaryotic or eukaryotic cell and, particularly preferably, a yeast cell, preferably a yeast cell of the strain S. cerevisiae.<br>
The modified gene expression is preferably analyzed by DNA/RNA profiling with the aid of cDNA or oligonucleotide microarrays, but the analysis may in principle include any modifications of the mRNA or protein steady state (transcription, translation, stabilization, etc.) and thus may also be carried out by protein profiling as well as with the aid protein arrays.<br>
In an advantageous design of the method, phenotyping is carried out by reducing or eliminating the compensatingly differential regulation. If the compensatingly differentially regulated gene is expressed stronger than in control organisms, said reduction or elimination is carried out by completely or partially inhibiting the enhanced expression. This is preferably carried out by crossing with a deletion strain and subsequent selection of the double mutants (particularly suitable when the organism is yeast), by genomic knock out using suitable vectors (these are known to the skilled worker and likewise very suitable in yeasts, here especially Saccharomyces cerevisiae), mutagenesis by radiation and/or mutagenic substances or introduction of antisense vectors or the like which inhibit protein production of the gene in question. To this end, it is particularly advantageous if the knock out of the compensatingly differentially regulated gene comprises the knock in of a reporter gene such as, for example, p-gaiactosidase/luciferase or growth markers such as HIS3, ADE2, URA3 or resistance markers such as, for example, for kanamycin. The reporter gene may then be used as signal in the subsequent assay to detect and quantify the efficacy of the drugs to be tested. This involves preferably replacing at least part of the coding sequence of the differentially regulated gene with the coding sequence (also including parts of said sequence which are sufficient for being detectable) of a reporter gene (e.g. luciferase, P-galactosidase, etc.). If the compensatingly differentially regulated gene is less   strongly   expressed   than   in   the   control   organism,   reduction   or<br><br>
elimination is effected by enhancing expression, preferably by crossing-in, introducing an episomal or another expression vector capable of selection or by genomic knock in (the methods above are particularly suitable for using yeast as organism). Preferably, reducing or eliminating the compensatingly differential regulation results in a growth inhibition of the genetically modified organism, but other phenotypes may also be advantageous.<br>
Another aspect of the invention relates to a genetically modified, phenotyped organism generated by the method of the invention.<br>
In particular, the invention relates to a genetically modified organism having genetically modified expression of at least one endogenous or foreign gene, which expression results in the compensatingly differential regulation of at least one other gene endogenous to said organism and thus preferably stops or inhibits an assessable/detectable/usable phenotype from appearing, and having a phenotype caused by reducing/eliminating the compensatingly differential expression of the gene or by labeling the compensatingly differentially regulated gene product.<br>
Another aspect of the invention relates to the use of a genetically modified organism prepared according to the invention for screening for substances having an effect on the function of the heterologous protein or protein fragments and on a method for identifying substances having an effect on the function of the heterologous protein or protein fragment.<br>
According to another aspect, the invention also relates to an assay for drug screening using a phenotyped organism of the invention by determining the phenotype (e.g. a growth inhibition due to induced heterologous overexpression of a protein), contacting the substance to be tested with said organism and observing a possible modification of said phenotype, preferably its at least partial reversion to the behavior or morphology of the wild-type organism (i.e. at least partial restoration of the phenotype of the starting organism, for example ending the growth inhibition). Furthermore, substances are concerned which are identified as being effective by a method of the invention or an assay of the invention.<br>
The invention is illustrated in more detail below on the basis of examples.<br><br>
Example 1: Development of a platform technology for identifying drugs which act on the activity of kinases, based on yeast as organism. The phenotype produced in this case is the growth inhibition of yeasts. The assay principle is thus based on the growth inhibition of yeasts which are used as living "reagent tube". Growth inhibition here means, for example, a cell cycle arrest or lysis of the cells concerned. Yeasts are used, since they are ideally suited, owing to their genetic manipulability. Human (or other exogenous) kinases are overexpressed in the yeast and under the control of a galactose-inducible promoter (GAL1/10). The yeasts are transformed and cultured according to standard methods. Examples of vectors used are those of the p41x-GAL1 or p42x-GAL11 series.<br>
In approx. 30% of all kinases to be tested, overexpression will already result in growth inhibition in yeast (Tugendreich et al. (2001)). This procedure is documented in figure 1 by steps 1, 3, 5. Kinases whose overexpression results in growth inhibition are integrated into a suitable yeast strain and then transferred to high throughput screening (HTS). This example uses yeast strains of the strain background "MATa his3Dl leu2D0 met15D0 ura3Q0" (BY4741 from EUROSCARF).<br>
During assay development for the HTS, conditions are optimized by assaying various "drug transporter" deletion mutants in the above-described strain background. For all protein kinases to be tested in this exampi-e, the strains having the following deletion combinations are assayed: 1. YRWS21 (MATa pdr1z::KanMX pdr3c::KanMX his3=1 Ieu2a0 metl5^0 Iys2=0 ura3G0) 2. YRWS39 (MATa pdr5n::KanMX yor1=::KanMX his3~1 Ieu2=0 MET15 Iys2a0 ura3c0) 3. YRWS14 (MATa pdr5z::KanMX snq2-::KanMX his3=1 leu2D0 MET15 Iys2n0 ura3n0)4. YRWS13 (MATa snq2-::KanMX yor1c::KanMX his3=1 Ieu2n0 MET15 Iys2c0 ura3z0) 5. YRWS44 (MATa pdr5z:;KanMX snq2=::KanMX yor1c::KanMX his3-1 leu2-0 met15=0 Iys2c0 ura3n0).<br>
It is then possible to search in high throughput screening for biological and chemical molecules which reduce or eliminate growth inhibition - i.e. which result in the growth of the yeast cultures. All previously described techniques are known to the competent skilled worker.<br>
As described above, approx. 30% of all exogenous kinases cause growth<br><br>
inhibition in yeast. Therefore, approx. 70% of all overexpressed kinases cause only low. if any, growth inhibition. In order to utilize the principle of growth inhibition of yeast as platform technique for compound screening of all protein kinases, the remaining 70% of protein kinases must also cause . growth inhibition. For this purpose, the present invention is needed.<br>
The desired protein kinases are cloned into a yeast expression vector of choice, in this example p413 GAL1 (D. Mumberg et al. (1994) in full length and with a C-terminal tag, e.g. MYC tag). After transformation using the lithium acetate method according to a standard protocol (see Methods in Yeast Genetics) and culturing in a suitable medium, overexpression of the exogenous kinases in the yeast is induced by adding galactose according to a standard protocol (20 g/ml of medium) at 30°C for 4 to 6 hours. Expression of the kinases is checked by immunobiots according to a standard protocol with the aid of antibodies against the chosen tag (e.g. anti-MYC: AB1364 (Chemikon) or M5546 (Sigma); anti-HA: HA-11-A (Biotrend) or 55138 (ICN)).<br>
After the immunological detection of expression in the yeast, modifications in gene expression - caused by expression of the exogenous kinases - in the yeast (compensatingly differential regulation) are studied with the aid of DNA microarrays. DNA microarrays are support materials to which specific oligonucleotides are chemically coupled. The individual oligonucleotides here represent individual genes. DNA microarrays are used as tools which can cover the current expression pattern of the entire yeast genome. For this type of experiment, kinase-transformed yeasts are compared to mock-transformed (empty plasmid) yeasts as control. Total RNA is prepared from both strains by standard methods. The RNA is then hybridized with the chip-coupied oligonucleotides (on the microarrays) at 45°C for 16 h. The direct comparison of the kinase-transformed yeast RNA with the mock-transformed yeast RNA reveals yeast genes which are regulated in a compensatingly differential manner by an overexpressed jKcttein kinase. Studies of the inventors have shown that a genetic intervention, for example, in overexpression of an exogenous protein kinase, upreguiates a particular number of RNAs for yeast genes and downregulates a particular number (table 1). This was carried out on the example of human kinase PAK1.<br>
Table   1:   2   genes   are   upreguiated.   11   genes   are   downregulated.<br><br>
Furthermore, the inventors were able to show for the first time, that many of the upregulated genes are upregulated for compensatory reasons. In this case, an S. cerevisiae wild-type strain (W303-1a (strain background or source of supply)) was compared with strain having a deletion in the Saccharomyces cerevisiae gene cla4 (Dcla4) (YEL252). Apart from the deletion in the gene for CLA4, both strains are isogenic, i.e. identical. When comparing directly the RNA preparations from the two different strains (W303-1a and YEL252), 110 different RNAs of the yeast genome turned up as upregulated (table 2).<br>
Table 2: 56 genes were downregulated (data not shown). Here, an increase of the RNA copy number for particular genes could possibly occur for compensatory reasons. In this specific example, compensatory means that the defect in the genetically modified strain, caused by deletion of the CLA4 gene, should be diminished by the increased expression of genes which can take over the function of CLA4 entirely or partially. In order to prove this thesis, some of the upregulated genes were selected for further experiments (see "2nd deletion" in table 3).<br>
Table 3: For this purpose, MATE yeast strains (which may be obtained, for example, from EUROSCARF or Research Genetics) were selected which carry deletions in the in each case upregulated genes. The deletions are marked by marker genes, i.e. marker genes, for example, for an antibiotic resistance or for required growth factors such as, for example, particular amino acids are integrated into the particular yeast genome. The deletion strains selected were crossed with the CLA4 deletion strain (YEL252, MATa) according to standard methods of yeast genetics (Methods in Yeast Genetics: A Cold Spring Harbor Course Manual (1994)).<br>
After crossing, diploid yeasts were selected which were then induced to form spores. This involves generation of 4 haploid spores from a diploid yeast cell, which can be divided into 4 haploid yeast clones for germination. Accordingly, the genes of the diploid strain become newly distributed. In 25% of all cases, the 2 deletions of the different starting strains will be united in a new haploid clone. This may be readily monitored on the basis of the various selection markers.<br>
This standard method was used to try to prepare 13 different double<br><br>
deletions. In only 10 cases, the double deletions were viable, in 3 cases, the double deletion never took place (table 3 "lethal"). In all 3 cases, 40 asci were tested. It is therefore clear that the combination of both deletions causes the affected spore to die. They are also synthetically lethal. It was demonstrated that in all 13 cases the double deletions were either synthetically lethal or have displayed other synthetic phenotypes (table 3). This study confirms the thesis that the affected genes were upregulated in order to compensate for defects caused by the lack of CLA4. It is important to the invention that in the cases studied (13 double deletions) 3 combinations and thus 23% of all possible double deletions displayed synthetic lethality (table 3).<br>
In the experiment with the Dda4 strain, 110 genes were upregulated (table 2). In the same way, overexpression of human PAK1 in the above-described approach upregulated the mRNAs of 2 genes (table 1). Consequently, these genes are also upregulated for compensatory reasons. Owing to the small number of upregulated genes and the low rate of success connected therewith for synthetically lethal combinations, we dispensed with the follow-up experiment of identifying strains which displayed a synthetically lethal phenotype in the combination of deletions in the upregulated genes (with YMR096W or HIS3 of table 1) and expression of human PAK1. Rather, a hyperactive mutant of human PAK1 was produced, namely human PAK1DCRIB. This mutant was transformed into<br>
yeast, again using standard methods. Owing to the high kinase activity, this protein caused growth inhibition in the yeast. A suitable strain for assaying low-molecular weight substances had been identified. The goal had been achieved. Nevertheless, in this case too, a differential expression profile was recorded using the DNA microarrays, in order to back up the validity of the invention (table 4).<br>
Table 4: 55 different yeast genes were compensatingly upregulated, owing to the high kinase activity, and 3 genes were downregulated (not shown). If the high activity of the PAK1 mutant had not been sufficient to cause growth inhibition in the yeast, it would now be possible to assay deletion strains for the upregulated genes. The PAK1 mutant would have to be expressed in the particular deletion strain. On the basis of the value of a 23% chance of success in a synthetic phenotype, expression of the human PAK1  mutant would then cause growth inhibition in approx.  13 yeast<br><br>
strains. Thus a strain for assaying potential kinase inhibitors would have been identified.<br>
In the case of assaying human kinases in the yeast, the starting strains would not need to be crossed, since the human kinases is expressed from a plasmid in a galactose-dependant manner. Said plasmid need only be transformed into the particular deletion strain and expression of the kinase needs to be induced. In 23% of all cases of the strains to be assayed, it will be possible to observe growth inhibition (lethality). The growth-inhibited strains can no longer compensate expression of the plasmid-encoded protein kinase, owing to the particular deletions. Therefore, these systems can be transferred to HTS.<br>
Should overexpression of particular wild-type kinases in combination with the DNA-microarray experiment not be sufficient (as described above for wild-type PAK1, see table 2) to cause growth inhibition, then mutants of the particular kinase are prepared and used instead of said wild-type kinases (also for the gene expression experiments using the DNA microarrays). These mutants may be prepared according to the principle of random mutagenesis, with the aim of obtaining hyperactive mutants. For mutagenesis, the kinase constructs are used with a C-terminal tag according to the method of Tugendreich etal. (2001).<br>
Thus, for the firs: time and surprisingly, studies of the inventors showed that the deletion of compensatingly differentially regulated genes can result in growth inhibition and in the finding connected therewith of designing a standardized platform assay for protein kinases. In the actual experiments, growth inhibition was detected with a frequency of 23%. The deletion strains which exhibit growth inhibition after transformation with the plasmid-encoded protein kinase may then, as described above, be transferred to HTS by means of optimization (testing of the various drug-transporter knockouts). Figure 1 illustrates the invention by way of example on the basis of points 1, 4. 5-10.<br>
Apart from crossing-in the deletions of compensatingly differentially reguiated genes, deletion thereof could aiso have been carried out using other methods such as genomic knockout of the kinase-expressing yeast itself. However, in yeasts the elimination of compensatingly differentially reguiated genes by crossing in deletions or the genomic knockout is<br><br>
particularly advantageous, owing to the simplicity of the procedure. In contrast, other methods may be more suitable in other organisms. Thus, in the example of eukaryotic cell lines and in the case of multicellular organisms such as Drosophila and C. elegans, the application of antisense methods such as RNAi is more suitable. The selection of measures suitable in each case for the individual organisms is within the ability of the skilled worker.<br>
The platform assay of the invention enables HTS of all protein kinases (as described on the basis of human PAK1) in homogeneous and thus cost-effective assay systems. This system is also suitable for determining IC50 values in compound screening.<br>
As described in the example, the gene expression experiments also result in the identification of RNAs of genes which are repressed by expression of exogenous kinases. The promoters of said repressed genes may serve as reporters in HTS. For this purpose, the yeast promoters are fused to "reporter genes" such as p-galactosidase, luciferase, growth markers such as H1S3, URA3, LEU2, or TRP1, etc. These constructs are transformed into the yeast strain for HTS. There they serve as growth markers for compounds which eliminate growth inhibition in the affected strain.<br>
Example 2: The platform assay may also be used as "multiplex system". Multiplex system means assaying various proteins or protein fragments, for example kinases, in the same assay in one reaction mixture at the same time. For this purpose, the individual phenotyped yeast strains are constructed first. The exogenous protein kinases are integrated using standard methods (see above). These yeast strains are then mixed to give a homogeneous culture. Expression of the protein kinases in the homogeneous yeast strain mixture results in growth inhibition, since expression of each individual kinase per se causes growth inhibition in the phenotyped yeast strain. HTS identifies compounds which result in the growth of at least one yeast strain. It is then essential to assign the kinase concerned to said compounds. This is achieved via the "colony PCR" method (A.J.P. Brown and M. Tuite (1998)). For this purpose, a few microliters from the growing yeast cultures are lysed, following instructions (A.J.P. Brown and M. Tuite (1998)). Quantitative RT-PCR using specific primers for the different protein kinases identifies unambiguously the inhibited kinase(s) concerned from (the mixture of) genomic DNA (including<br><br>
integrated protein kinases). Thus it is possible to assay in a single screening different kinases by mixing equal parts of different yeast strains. The advantage is an enormous saving of cost and time.<br>
This'technology is applicable not only to protein kinases but to any proteins or substances which cause a transcriptional response in the yeast.<br>
This platform assay enables in the subject to assays of the prior art, for example, HTS of all protein kinases (not only of those whose heterologous expression already produces a phenotype immediately) in homogeneous and therefore cost-effective assay systems. This system is also suitable for determining IC50 values in compound screening.<br>
This technology is applicable not only to protein kinases but to all proteins or substances which cause a transcriptional response in the yeast.<br>
Methods:<br>
The standard methods according to Sambrook et al. (1989) Molecular Cloning: A Laboratory Manual, Second edition, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY. 545 .pp. were used for genetic manipulations.<br>
Growth conditions, crossing conditions and genetic manipulations on yeasts (Saccharomyces cerevi.siae) were carried out according to Guthrie, C. and G.R. Fink (1991) Guide to Yeast Genetics and Molecular Biology, Volume 194, J.N. Abelson and M.I. Simon, eds. (San Diego, CA: Academic Press Inc.). The Affymetrix experiments ("gene expression analysis) were carried out exactly according to Klebl et al. (2001) Biochem. Biophys. Res. Commun. 286, 714-720.<br>
References:<br>
Brown, A.J.P. and M. Tuite (1998), PCR-Based Gene Targeting in Saccharomyces cerevisiae. Methods Microbiol. 26, 67-81.<br>
Methods in Yeast Genetics] A Cold Spring Harbor Course Manual; 1994 Edition; Kaiser, C, Michaelis, S., and A. Mitchell; Cold Spring Harbor Laboratory Press.<br><br>
Mumberg, D., Muller, R. and M. Funk (1994). Regulatable promoters of Saccharomyces cerevisiae: comparison of transcriptional activity and their use for heterologous expresson. Nucl. Acids Res. 22, 5767-5768.<br>
Tugendreich, S., Perkins, -E., Couto, J., Barthmaier, P., Sun, D., Tang, S., Tulac, S., Nguyen, A., Yeh, E., Mays, A., Wallace, E., Lila, T., Shivak, D., Prichard, M., Andrejka, L, Kim. R. and T. Melese (2001). A streamlined process to phenotypically profile heterologous cDNAs in parallel using yeast cell-based assays. Genome Res. 11 1899-1912.<br>
Table 1:<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
Patent claims<br>
1.	A method for generating a genetically modified organism for drug<br>
screening, which comprises the steps<br>
a)	causing heterologous expression of at least one protein or protein fragment by genetic modification of the organism<br>
b)	analyzing the  modified gene expression  pattern  and  identifying compensatingly differentially regulated genes<br>
c)	phenotyping the organism.<br><br>
2.	The method as claimed in claim 1, wherein phenotyping is carried out by reducing/eliminating the compensatingly differential expression or by labeling at least one compensatingly differentially regulated gene.<br>
3.	The method as claimed in either of claims 1 and 2, wherein the genetic modification causes heterologous expression of at least one protein or protein fragment which is endogenous to the organism and/or foreign.<br>
4.	The method as claimed in any of claims 1 to 3, wherein the genetic modification causes reduction or elimination of the expression of at least one protein endogenous to the organism.<br>
5.	The method as claimed in any of claims 1 to 4, wherein the modified expression is inducible.<br>
6.	The method as claimed in claim 5, wherein the genetic modification comprises introducing a vector which enables the protein or protein fragment to be inducibly expressed, preferably a vector inducible with galactose, copper tetracycline or other comparably inducible vectors.<br><br>
7.	The method as claimed in any of claims 1 to 6, wherein the genetic i     modification comprises a knock out. preferably an inducible knock out.<br>
8.	The  method  as claimed  in any of claims  1  to 7, wherein the<br><br>
organism is drosophila, C. elegans, a prokaryotic or a eukaryotic cell.<br>
9.	The method as claimed in claim 8, wherein the cell is a yeast cell, preferably a yeast cell of the strain S. cerevisiae.<br>
10.	The method as claimed in any of claims 1 to 9, wherein the modified gene expression is analyzed with the aid of DNA or protein microarrays.<br>
11.	The method as claimed in any of claims 1 to 10, wherein pheno-typing is carried out by reducing or eliminating expression of the cornpensatingly differentially regulated gene.<br>
12.	The method as claimed in claim 11, wherein expression of the cornpensatingly differentially expressed gene is enhanced to control organisms and the reduction or elimination is caused by at least partial inhibition of said enhanced expression.<br>
13.	The method as claimed in claim 7. wherein the knock out of the differentially expressed gene is carried out replacing at least part of the coding sequence of the differentially regulated gene with the coding sequence of a reporter gene or parts of the reporter gene sequence which are sufficient to be detected.<br>
14.	The method as claimed in .claim 11, wherein the differentially expressed gene is less strongly expressed than in control organisms and the reduction or elimination is carried out by enhancing its expression.<br>
15.	The method as claimed in any of claims 1 to 14, wherein the reduction or elimination leads to growth inhibition of the organism.<br>
16.	The' method as claimed in any of claims 1 to 10, wherein phenotyping is carried out by labeling the gene product of the cornpensatingly differentially regulated gene.<br>
17.	A genetically modified, phenotype organism, obtained by a method as claimed in any of claims 1 to 15.<br>
18.	A genetically modified organism, having<br><br>
a)	genetically modified expression of at least one endogenous or<br>
foreign gene, which results in compensatingly differential expression<br>
of at least one other gene endogenous to said organism, and<br>
b)	a phenotype caused -by reducing/eliminating the compensatingly<br>
differential expression of the gene or by labeling the compensatingly<br>
differentially regulated gene product.<br>
19.	The use of a genetically modified organism as claimed in either of<br>
claims 17 or 18 for screening for substances having an effect on the<br>
function of the heterologous protein or protein fragment.<br>
20.	A method for identifying substances having an effect on the<br>
function of the heterologously expressed protein or protein fragment, which<br>
method comprises the use of the organism as claimed in either of claims 17<br>
or 18.<br>
21.	An assay for drug screening using at least one phenotype organism<br>
as claimed in either of claims 17 or 18, which comprises the steps<br>
b)	determining the phenotype of said organism<br>
c)	contacting the substance to be tested with said organism<br>
d)	observing a possible modification of said phenotype.<br>
22.	A substance, which is identified by a method as claimed in claim 20<br>
or an assay as claimed in claim 21 as a substance which at least reduces<br>
the phenotype.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI0OS1jaGVucC0yMDA1IGFic3RyYWN0IGdyYW50ZWQucGRm" target="_blank" style="word-wrap:break-word;">1249-chenp-2005 abstract granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI0OS1jaGVucC0yMDA1IGNsYWltcyBncmFudGVkLnBkZg==" target="_blank" style="word-wrap:break-word;">1249-chenp-2005 claims granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI0OS1jaGVucC0yMDA1IGRlc2NyaXB0aW9uIChjb21wbGV0ZSkgZ3JhbnRlZC5wZGY=" target="_blank" style="word-wrap:break-word;">1249-chenp-2005 description (complete) granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI0OS1jaGVucC0yMDA1IGRyYXdpbmcgZ3JhbnRlZC5wZGY=" target="_blank" style="word-wrap:break-word;">1249-chenp-2005 drawing granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI0OS1jaGVucC0yMDA1LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1249-chenp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI0OS1jaGVucC0yMDA1LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1249-chenp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI0OS1jaGVucC0yMDA1LWNvcnJlc3Nwb25kYW5jZSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">1249-chenp-2005-corresspondance others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI0OS1jaGVucC0yMDA1LWNvcnJlc3Nwb25kYW5jZSBwby5wZGY=" target="_blank" style="word-wrap:break-word;">1249-chenp-2005-corresspondance po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI0OS1jaGVucC0yMDA1LWRlc2NyaXB0aW9uIGNvbXBsZXQgLnBkZg==" target="_blank" style="word-wrap:break-word;">1249-chenp-2005-description complet .pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI0OS1jaGVucC0yMDA1LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1249-chenp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI0OS1jaGVucC0yMDA1LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">1249-chenp-2005-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI0OS1jaGVucC0yMDA1LWZvcm0gMTgucGRm" target="_blank" style="word-wrap:break-word;">1249-chenp-2005-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI0OS1jaGVucC0yMDA1LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">1249-chenp-2005-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI0OS1jaGVucC0yMDA1LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">1249-chenp-2005-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI0OS1jaGVucC0yMDA1LXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1249-chenp-2005-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="228635-an-antibody-for-inducing-apoptosis.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="228637-process-for-preparing-an-expandable-thermoplastic-microspheres.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>228636</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1249/CHENP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>12/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>20-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>05-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>14-Jun-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>SANOFI-AVENTIS DEUTSCHLAND GmbH</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>D-65929 FRANKFURT AM MAIN,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>KLEBL, Bert</td>
											<td>Imhoffstrasse 16, 82294 Günzlhofen,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>STADLER, Anja</td>
											<td>Kurt-Schumacherstrasse 18, 82256 Fürstenfeldbruck,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>SOLLNER, Rosemarie</td>
											<td>Wurmseestrasse 60, 81476 München,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>LEBERER, Ekkehard</td>
											<td>Fichtenstrasse 82 B, 82110 Germering,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>NITSCHE, Almut</td>
											<td>Emanuel-Geibelstrasse 8, 65185 Wiesbaden,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2003/012870</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-11-18</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>102 58 885.6</td>
									<td>2002-12-17</td>
								    <td>Germany</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/228636-method-for-generating-a-genetically-modified-organism-for-screening-active-substances by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:14:45 GMT -->
</html>
