<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/228684-method-of-using-dna-markers-for-distinguishing-male-sterile-cms-lines-of-rice-from-their-cognate-male-fertile-maintainer-lines by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:17:56 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 228684:&quot;METHOD OF USING DNA MARKERS FOR DISTINGUISHING MALE STERILE (CMS) LINES OF RICE FROM THEIR COGNATE MALE FERTILE MAINTAINER LINES&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;METHOD OF USING DNA MARKERS FOR DISTINGUISHING MALE STERILE (CMS) LINES OF RICE FROM THEIR COGNATE MALE FERTILE MAINTAINER LINES&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>This invention relates to novel DNA markers for assessing seed purity and a method for ensuring the purity of cytoplasmic male sterile lines of rice using DNA based markers. This method is based on the identification of a DNA Sequence that is specific to WA cytoplasmic male sterile lines of rice and the development of specific DNA markers derived from the same. These DNA markers can be used to detect admixtures of male fertile Maintainer lines with CMS lines. This application is likely to be very beneficial to the hybrid rice industry as admixtures of the type described above often lead to reduced purity of the hybrid seeds and poor performance of the product in the marketplace. Methodology for the application of co-dominant sequence specific PCR markers like microsatellites and STSs for detecting impurities in parental lines and hybrids of rice and other crops is also provided.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>DNA MARKERS FOR ASSESSING SEED PURITY AND A METHOD OF USING<br>
DNA SEQUENCES FOR ASSESSING SEED PURITY<br>
TECHNICAL FIELD<br>
The invention relates to a method of using DNA sequences for assessing seed purity. More<br>
specifically, the invention relates to a DNA sequences having homology to rice<br>
mitochondrial DNA and being unique to Wild Abortive (WA) cytoplasm containing<br>
cytoplasmic male sterile lines of rice, and the use of these sequences in a Polymerase Chain<br>
Reaction (PCR) assay to distinguish Male Sterile (CMS) lines of rice from their cognate<br>
Male Fertile Maintainer Lines. This invention relates to a method for ensuring the purity of<br>
cytoplasmic male sterile lines of rice using DNA based markers<br>
BACKGROUND ART<br>
Hybrid vigor is the phenomenon by which the progeny of a cross between two inbred lines<br>
has higher yield potential than either one of the parents. Hybrids can yield upto 10-30 %<br>
more than the best variety and are a favored option for increasing yield.<br>
Rice is a major cereal crop all over the world; and in many parts of Asia it is the staple part<br>
of the diet. It has been estimated that in a number of Asian countries like India rice yields<br>
must double by the year 2025 to meet the demands of the increasing population (Hossain,<br>
1996. In Khush (ed) Rice Genetics III, Proc. Third Intl. Rice Genet. Symp., Los Banos<br>
Manila, the Philippines. 16-20 Oct. 1995. International Rice Rearch Institute, Manila, the<br>
Philippines). As has been demonstrated in the People's Republic of China, where almost<br>
fifty percent of the area under rice cultivation is covered by hybrids, the widespread<br>
cultivation of hybrid rice is a readily available option for increasing yield. By comparison,<br>
in countries like India the area under hybrid rice is less than 1% of the total area under rice<br>
cultivation. This illustrates the tremendous potential for increasing the area under hybrid<br>
rice cultivation and it is expected that the market for hybrid rice seeds will increase in a<br>
number of rice growing countries, including in India.<br>
The most widely used system for hybrid rice production is the three line system (Fig. 1).<br>
The three lines include : 1. a male sterile, female fertile line called the Cytoplasmic Male<br>
Sterile (CMS) line because it carries a male sterility conferring mutation in the cytoplasmic<br>
component of the genome, 2. a maintainer line and 3. a restorer line; the maintainer<br>
and restorer lines are male fertile as well as female fertile. The CMS and Maintainer lines<br>
are practically identical with respect to the nuclear component of the genome (and are often<br>
referred to as iso-nuclear lines) but differ from each other with respect to the cytoplasmic<br>
component of the genome. The male sterility of the CMS line is maternally inherited and is<br>
most likely due to a mutation in the mitochondrial DNA. The CMS line, being female<br>
fertile, can be propagated by fertilization with pollen emanating from the Maintainer line.<br>
Since the cytoplasmic component of the genome is not transferred through pollen, the<br>
progeny of such a cross would inherit the cytoplasm only from the CMS line and would<br>
therefore be male sterile. The nuclear component of the genome of the progeny would also<br>
be identical to that of the CMS line, even though half of it is inherited from the Maintainer<br>
line, as there is no difference between these two lines with respect to this component of the<br>
genome.<br>
The hybrid seeds are produced in a cross of the CMS line with another inbred parental line,<br>
called the Restorer line, which as indicated above is Male fertile and Female fertile. In this<br>
cross the CMS line serves as the female parent while the Restorer line is the male parent.<br>
The Restorer line also carries a Rf (restorer of fertility) gene/s in it's nuclear genome which<br>
will restore male fertility to a plant whose cytoplasm has been inherited from the CMS line.<br>
The hybrid seeds produced in the cross depicted in Figure 1 would therefore be fertile. The<br>
CMS and Restorer lines are appropriately chosen such that the hybrids exhibit sufficient<br>
hybrid vigor (or heterosis) to produce substantially higher yields than inbred varieties.<br>
The vast majority (90% or more) of the rice hybrids that are currently under commercial<br>
cultivation in the world derive their cytoplasm from a single source (Yuan, 1995. Hybrid<br>
rice seed production technology, Directorate of Rice Research, Hyderabad, India). This<br>
cytoplasm, called the WA (wild abortive) cytoplasm, was discovered in a wild rice plant in<br>
China. Subsequently, this cytoplasm has been crossed into several different nuclear genetic<br>
backgrounds by repeated back-crossing using the recurrent parent as the male donor. In this<br>
manner several different CMS lines have been developed, each of which has in turn been<br>
crossed with different Restorer lines to develop a number of hybrids, all of which share the<br>
same WA cytoplasm.<br>
It is important to maintain the purity of hybrids as any impurities therein would reduce<br>
the expected yield. It has been estimated that for every one-percent impurity in the hybrid<br>
seed the yield reduction is to the tune of 100 Kg per hectare (Mao et al, 1996 In<br>
Virmani, S.S., E.A. Siddiq, and K. Muralidharan (eds) Advances in Hybrid Rice<br>
Technology. Proc. Third Intl. Symp. on Hybrid Rice, Directorate of Rice Research,<br>
Hyderabad, India). The Indian seed act prescribes that, for hybrid rice, the expected purity<br>
should be 98% (Verma, 1996. Seed Tech. News 24:1-4); in the People's Republic of China<br>
it is mandated that the purity of hybrid rice should be at least 96% (Wengui Yan, 2000. US<br>
Patent #6066779). In order to ensure the required levels of hybrid seed purity, the parental<br>
lines that are employed in hybrid seed production should have a very high (almost 99%)<br>
level of purity.<br>
One of the common admixtures that occur during hybrid seed production is that of<br>
Maintainer lines with those of the CMS lines. As these are iso-nuclear lines, it is very<br>
difficult to distinguish between these lines based on morphological criteria i.e. other than<br>
male sterility which can only be judged at the time of flowering. DNA markers that<br>
distinguish the CMS and maintainer lines can be developed and applied at the seedling<br>
level for the practical detection of seeds of Maintainer lines that occur as contaminants<br>
within stocks of the CMS lines. DNA Markers based on the use of the Polymerase Chain<br>
Reaction would be ideally suited for this purpose as they are much more efficient for<br>
handling large numbers of samples than hybridization based methods like Restriction<br>
Fragment Length Polymorphisms.<br>
The Polymerase Chain Reaction is based on the use of short oligonucleotide sequences as<br>
primers for the enzymatic amplification of DNA sequences that occur between two<br>
appropriately spaced primer binding sites on the target DNA. The PCR works most<br>
reproducibly when the oligonucleotide primers are designed on the basis of a knowledge of<br>
the target DNA sequence. Protocols have been developed for PCR which are based on the<br>
use of short (8-10 bases long), randomly designed oligonucleotide primers (Williams et al,<br>
1990; Nucleic acids research 18: 6531-6535). These primers are not based on a knowledge<br>
of the target DNA sequence and kits containing large numbers of these randomly generated<br>
primers are now commercially available. The DNA markers that are developed by this<br>
method are known as Randomly Amplified Polymorphic DNA (RAPD) Markers. Because a<br>
large number of primers are available, genetic polymorphisms can be deteciud hy this<br>
method, even within closely related lines. However, the reproducibility of RAPD markers is<br>
poor due to the short length of the oligonucleotide (RAPD) primers and possibly also due<br>
to the lack of the required degree of specificity for the target. This severely limits the<br>
practical application of RAPD markers, as diagnostic markers for distinguishing different<br>
genotypes.<br>
RAPD markers that distinguish CMS (WA cytoplasm) and Maintainer lines of rice have<br>
been described (Jena and Pandey 1999; Hybrid Rice Newsletter. 2: 13-14). However, the<br>
low reproducibility of these markers has made it practically impossible to apply them in a<br>
routine manner for distinguishing CMS and Maintainer lines. There is thus a need for the<br>
development of reproducible PCR based methods that can be applied for distinguishing the<br>
CMS and Maintainer lines. The desired level of reproducibility can be obtained if the<br>
oligonucleotide primers are based on a knowledge of the sequence of a region of rice DNA<br>
that is polymorphic between the CMS and Maintainer lines. PCR assays based on such<br>
primers would be highly reproducible because these primers would be longer in length than<br>
the primers used in RAPD analysis and would be specific for the target DNA sequence.<br>
In this application, is described the identification and sequence determination of a region of<br>
rice mitochondrial DNA that is specific to CMS lines of rice containing the wild abortive<br>
(WA) type of cytoplasm. Based on this sequence, specific oligonucleotide primers have<br>
been developed that can be used in a PCR assay to distinguish CMS (WA) lines from their<br>
iso-nuclear Maintainer lines. These primers have been used to distinguish several different<br>
CMS lines (all containing the WA cytoplasm) of rice from their cognate Maintainer lines.<br>
In a coded test, this assay was used to predict with 100% accuracy the genotypes of a<br>
mixture of CMS (WA) and Maintainer lines of rice. The assay can therefore be used by rice<br>
breeders to successfully detect admixtures of Maintainer lines in seed stocks of the CMS<br>
line, thereby ensuring the purity of this parental line and the hybrid derived from it.<br>
Another source of impurity within the seed stocks of the CMS line is caused by cross<br>
pollination with pollen emanating from rice plants that are not the designated Maintainer<br>
line. A minimum isolation distance of 300 meters is prescribed for multiplication of rice<br>
CMS lines (Virmani, 1993. Advances in Agronomy 57:377-462) i.e. within this distance no<br>
rice lines other than the Maintainer line (the preferred pollen donor or male parent) should<br>
be cultivated.<br>
This is based on the observation that pollen originating from rice plants growing beyond<br>
this distance will not pollinate the female parent. Occasionally, this minimum isolation<br>
distance is either not strictly followed or local conditions (for e.g. wind flow and weather)<br>
might permit pollen to be transferred from plants that are growing beyond the 300 meters<br>
distance. Therefore, it is important to monitor the extent of outcrossing with rogue pollen<br>
donors that has occurred during the multiplication of the CMS lines. In this patent, are<br>
described methods for the application of sequence specific PCR markers like<br>
microsatellites and Sequence Tagged Sites (STSs) towards estimating the extent of<br>
outcrossing that has occurred during the multiplication of seed stocks of the CMS line.<br>
Although the application is for estimating the extent of outcrossing that has occurred for<br>
CMS lines of rice, a similar approach can be used for estimating the extent of outcrossing<br>
that has occurred within the CMS lines of other crops including but not limited to maize,<br>
pearl millet, sorghum, wheat, sunflower, mustard, cabbage, caulifower, tomato, pepper,<br>
okra, etc wherein the CMS lines are used for the production of hybrids and appropriate<br>
microsatellite or STS markers are available.<br>
The estimation of hybrid seed purity is conventionally done by the grow out test (GOT),<br>
which is based on the assessment of morphological and floral characteristics (that<br>
distinguish the hybrid) in a representative sample of plants that are grown to maturity. Rice<br>
plants take several months to reach maturity and the seeds have to be stored under<br>
appropriate conditions as they cannot be marketed until these results become available. In<br>
addition substantial delays can result, as occurs in India, if the first growing season after<br>
hybrid seed production which is taken up by the GOT is also the preferred season for hybrid<br>
cultivation. In such cases, the seeds have to be stored for upto a year i. e. until the<br>
subsequent growing season before they can be marketed. For seed companies, large<br>
amounts of capital are therefore locked up in the form of hybrid seed stock for prolonged<br>
periods while awaiting the results of the GOT. Another disadvantage of the GOT is that it<br>
can be subjective due to environmental influences on the expression of morphological<br>
characteristics. Further, there is also the possibility that adverse climatic conditions (like<br>
heavy wind or ram) can damage or destroy the crop and make it difficult to collect the data.<br>
With the objective of replacing the GOT with a test that is superior in terms of speed and<br>
accuracy, a PCR based assay is described for assessing hybrid seed purity. This test<br>
involves the use of either microsatellite or STS (Sequence Tagged Site) polymorphisms that<br>
distinguish the parental lines of rice hybrids. These polymorphisms are co-dominant and the<br>
alleles are detected as DNA fragments of different sizes following PCR and agarose gel<br>
electrophoresis. The hybrid can be identified because it will have alleles contributed by<br>
both parents i.e. PCR amplified fragments of two different sizes will be obtained after use<br>
of the DNA isolated from the hybrid plant as a template in PCR. One of these alleles will<br>
be contributed by the male sterile, female fertile (CMS) parent while the other will be<br>
contributed by the male fertile, female fertile (Restorer) parent. This test can be conducted<br>
on DNA isolated from six day old rice seedlings and the assay can be completed within<br>
forty eight hours. The implementation of this PCR based test for seed purity will result in<br>
considerable savings for the seed industry. Additional modifications of this assay are<br>
described wherein the test need not be conducted on individual seedlings but can be<br>
conducted on populations of seedlings obtained from the hybrid seed stock.<br>
SUMMARY OF THE INVENTION:<br>
This invention relates to novel DNA markers for assessing seed purity and a method for<br>
ensuring the purity of cytoplasmic male sterile lines of rice using DNA based markers.<br>
This method is based on the identification of a DNA Sequence that is specific to WA<br>
cytoplasmic male sterile lines of rice and the development of specific DNA markers derived<br>
from the same. These DNA markers can be used to detect admixtures of male fertile<br>
Maintainer lines with CMS lines. This application is likely to be very beneficial to the<br>
hybrid rice industry as admixtures of the type described above often lead to reduced purity<br>
of the hybrid seeds and poor performance of the product in the marketplace. Methodology<br>
for the application of co-dominant sequence specific PCR markers like microsatellites and<br>
STSs for detecting impurities in parental lines and hybrids of rice and other crops is also<br>
provided.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
Rice is a major cereal crop in many parts of the world. Yield increases of 10-30% are<br>
reported following cultivation of hybrid rice in the People's Republic of China where<br>
it is being practised on a large scale. It is expected that, in the near future, hybrid rice<br>
technology will also be practised on a large scale in a number of other rice growing<br>
countries. Currently, most rice hybrids are produced through a three line system comprising<br>
of: 1. a Cytoplasmic Male Sterile (CMS) line that is female fertile but is male sterile due<br>
to a mutation in the cytoplasmic most probably the mitochondrial) component of the rice<br>
genome; 2. a male fertile, female fertile Maintainer line that is identical to the CMS line<br>
with respect to the nuclear component of the genome but has a different cytoplasmic<br>
genotype that does not induce male sterility; 3. a male fertile, female fertile Restorer line.<br>
The CMS line serves as the female parent for the hybrid while the Restorer line is the male<br>
parent. During hybrid seed production the CMS and Restorer lines are cultivated in close<br>
proximity to each other such that pollen emanating from the Restorer line will pollinate the<br>
flowers of the CMS line. As the CMS line is male sterile it will not set seed by self<br>
pollination and any seeds that are formed on the CMS line are deemed to have arisen as a<br>
consequence of fertilisation with pollen emanating from the Restorer line. The restorer line<br>
carries one or more nuclear encoded genes that will restore male fertility to the hybrid even<br>
though it carries the CMS cytoplasm. Thus the hybrid is self fertile.<br>
The CMS line cannot be propagated by selfmg as it is male sterile. Instead, the propagation<br>
of the CMS lines is accomplished by using the CMS line as a female parent and the<br>
Maintainer line as the male parent. The genotype of the progeny that arise from this cross<br>
will be identical to the genotype of the CMS line as the Maintainer lines are practically<br>
identical to the CMS line with respect to the nuclear component of the genome. The<br>
progeny will have the male sterile characteristic of the CMS line because the cytoplasmic<br>
component of the genotype is contributed by the female parent; which in this case is the<br>
CMS line. It is also pertinent to note that the Maintainer line does not cany any Restorers<br>
of the Cytoplasmic Male Sterile phenotype of the CMS line.<br>
The identity of the nuclear genotype of the CMS and Maintainer lines means that the two<br>
lines are almost indistinguishable by morphological criteria. This creates a practical<br>
problem because it is very difficult to detect admixtures of the Maintainer line within seed<br>
stocks of the CMS line. Since the Maintainer lines are self fertile, these impurities can<br>
produce seeds in a hybrid rice production field without the necessity for fertilization with<br>
the Restorer line. This leads to a contamination of the seeds of the, Maintainer line with<br>
those of the hybrid. This type of a contamination is one of the most frequently observed<br>
during hybrid rice seed production and leads to a reduction in the expected yield and poor<br>
performance of the hybrid in the field. If the purity of the hybrid is less than the mandatory<br>
limit fixed by Seed Certification agencies, this is 98% in India and 96% in China, the entire<br>
seed lot is rejected leading to considerable loss for the seed producers (companies).<br>
The vast majority of the CMS lines that are employed in commercial production of hybrid<br>
rice are based on the use of the WA cytoplasm. In this patent we describe the identification<br>
of a DNA sequence that is unique to rice lines containing the WA cytoplasm and is highly<br>
homologous to rice mitochondrial DNA. Sequence Specific oligonucleotide primers have<br>
been developed based on this DNA sequence that can be used in a PCR assay to reliably<br>
distinguish rice cytoplasmic male sterile lines containing the WA cytoplasm from their<br>
cognate Maintainer lines. These primers can therefore be used by hybrid rice breeders/seed<br>
companies to detect impurities of the Maintainer line within the CMS line. By ensuring<br>
purity of the CMS lines, a major source of contamination of the hybrid seeds is removed<br>
leading to obvious benefits for the seed industry and farmers.<br>
BRIEF DESCRIPTION OF THE ACCOMPANYING DRAWINGS<br>
Figure 1. The three line system for hybrid rice production<br>
This figure is a graphic of how a hybrid is produced in a three line system. Cytoplasmic<br>
male sterile (CMS) line (pollen sterile), which serves as the female parent is crossed with<br>
the Maintainer line (pollen fertile) which serves as the male parent. CMS and Maintainer<br>
lines are iso-nuclear except for the pollen sterility in CMS lines hence Maintainers are<br>
necessary for propagation of the CMS lines. Once CMS lines are obtained, they are crossed<br>
with a Restorer line which can possess any number of desirable agronomic traits and which<br>
also restores fertility to the progeny. Therefore the hybrid produced by such a cross is<br>
fertile.<br>
Figure 2. PCR amplification of a DNA sequence that is specific to CMS lines of rice<br>
PCR was performed as described in Example 3 using microsatellite primer RM9 (SEQ ID<br>
No. 6 and SEQ ID No. 7). The PCR products were separated on a 2% agarose gel, stained<br>
with ethidium bromide and visualized under ultraviolet light. Lane 1 contains a DNA<br>
molecular weight marker [( DNA digested with Hind HI (New England Biolabs, USA)];<br>
lane 2 contains PCR amplified product of a Maintainer line (ER 62829 B); lane 3, the hybrid<br>
(DRR HI) and lane 4, CMS line (TR 62829 A). An extra DNA band which is present only<br>
in the hybrid and CMS line, but absent in the Maintainer line is indicated by the black<br>
arrow on the right hand comer of the gel.<br>
Figure 3. Nucleotide sequence of a rice CMS line specific DNA. (SEQ ID No. 1)<br>
A CMS specific DNA band that was identified using microsatellite primer RM9 was<br>
purified as described in Example 4. The DNA band was sequenced using an automated<br>
DNA sequencer (ABI 3700; ABI, Foster City, USA). The sequence is composed of 325<br>
bases. Homology search revealed that it has high homology to the rice mitochondrial DNA,<br>
DBJ Accession #D21251 except for a small stretch from base positions 1 to 36. PCR<br>
primers were designed based on this sequence information to obtain amplification only in<br>
CMS lines; the Forward primer was based on the region which does not exhibit homology<br>
to the rice mitochondrial DNA (position 1 to 36) and the Reverse primer was based on the<br>
mitochondrial DNA sequence.<br>
Figure 4. Homology of CMS specific DNA sequence with rice mitochondrial DNA (SFQ<br>
ID No. 2).<br>
The DNA sequence information obtained by sequencing as described in Example 4 was<br>
aligned with the rice motochondrial DNA, DBJ Accession #D21251 using the Boxshade<br>
server available at http://www.ch.embnet.org/sofrware/BOX_form.html<br>
Figure 5. A PCR assay for distinguishing CMS and Maintainer lines of rice.<br>
PCR products amplified using the SEQ ID No. 4 and SEQ ID No. 5 were separated on a 1%<br>
agarose gel, stained with ethidium bromide and visualized under ultraviolet light. Lane 1 is<br>
a kilobase DNA marker. Lane 2 contains IR58025 A (CMS); lane 3, DRR H2 (hybrid); lane<br>
4, IR 58025 B (Maintainer); lane 5, IR 62829 A (CMS); lane 6, DRR HI (hybrid); lane 7,<br>
IR 62829 B (Maintainer). Three other sets of CMS-Maintainer lines were analyzed and in<br>
all cases, amplification was observed only in the CMS lines and hybrids but not observed in<br>
the Maintainer lines.<br>
Figure 6. Detection of Restriction Fragment Length Polymorphisms (RFLP) between CMS<br>
and Maintainer lines of rice using CMS specific DNA sequence as a probe.<br>
Genomic DNA isolated from IR 58025 A (CMS) and IR 58025 B (Maintainer) were<br>
digested with EcoR V restriction enzyme and separated on an agarose gel overnight.<br>
Southern hybridization was performed as described in Example 7 using the CMS specific<br>
DNA fragment as a probe. The probe hybridized to a 2.3 kb region in the IR 58025 A<br>
(CMS) line whereas in the IR 58025 B (Maintainer) line it hybridized to a 0.6 kb region.<br>
Figure 7. A multiplex PCR assay for distinguishing CMS and Maintainer lines of rice.<br>
Lane 1 is a DNA molecular weight marker [( DNA digested with Hind HI (New England<br>
Biolabs, USA)]; lane 2, IR 58025 A (CMS); lane 3, IR 58025 B (Maintainer). Samples in<br>
lane 2 and 3 are PCR products amplified by SEQ ID No. 4 and SEQ ID No. 5 alone. Lane<br>
4, IR 58025 A (CMS); lane 5, IR 58025 B (Maintainer). Samples in lane 4 and 5 are PCR<br>
products amplified by multiplexing SEQ ID No. 4, SEQ ID No. 5, SEQ ID No. 8 and SEQ<br>
ID No. 9. The monomorphic fragment is the PCR product of SEQ ID No. 8 and SEQ ID<br>
No. 9 and the polymorphic fragment is the PCR products of SEQ ID No. 4, SEQ ID No. 5.<br>
Figure 8. A PCR assay for detecting purity of rice hybrids.<br>
Lane 1, is a 1 kb DNA molecular weight marker. PCR was performed using primers for the<br>
RM164 microsatellite locus Lane 2 is IR 40750 (Restorer), lane 3 is DRR HI (hybrid) and<br>
lane 4 is IR 62829 A (CMS). Note that a single polymorphic PCR amplified fragment is<br>
observed in lanes 2 and 4. The hybrid exhibits two DNA fragments which are characteristic<br>
of the alleles contributed by both parents. Lane 5-12 represent a PCR assay performed to<br>
determine the purity of seeds from a stock of the DRR HI hybrid. The presence of two<br>
bands indicates that lanes 5, 6, 8, 9, 10, 11 and 13 represent true hybrids. The presence of a<br>
single band indicates that the plants represented by lanes 7 and 12 (off-types) are<br>
impurities.<br>
hi short, the invention provides a DNA sequence (SEQ ID No.l &amp; SEQ ID No 3)<br>
substantially homologous to rice raitochondrial DNA, said sequence being unique to Wild<br>
Abortive (WA) cytoplasm containing cytoplasmic male sterile lines of rice. The invention<br>
also provides oligonucleotide primers (SEQ ID No.4 &amp; SEQ ID No 5) based on this DNA<br>
sequence in a Polymerase Chain Reaction (PCR) assay to distinguish Male Sterile (CMS)<br>
lines of rice from their cognate Male Fertile Maintainer Lines.<br>
Further, the invention provides a method of using oligonucleotide primers (SEQ ID No.4 &amp;<br>
SEQ ID No 5) based on this DNA sequence in a Polymerase Chain Reaction (PCR) assay<br>
to distinguish Male Sterile (CMS) lines of rice from their cognate Male Fertile Maintainer<br>
Lines.<br>
hi an embodiment, the invention provides a method of using oligonucleotide primers<br>
having SEQ ID No.4 &amp; SEQ ID No 5 based on this DNA sequence in a Polymerase Chain<br>
Reaction (PCR) assay to distinguish Male Sterile (CMS) lines of rice from their cognate<br>
Male Fertile Maintainer Lines when the said Male Sterile Lines contain the WA (wild<br>
abortive) cytoplasm.<br>
hi an embodiment, the invention provides a method of using oligonucleotide primers<br>
having SEQ ID No.4 &amp; SEQ ED No 5 based on the DNA sequence in a Polymerase Chain<br>
Reaction (PCR) assay to distinguish WA Cytoplasmic Male Sterile (CMS) lines of rice<br>
from their cognate Male Fertile Maintainer Lines wherein a DNA amplification product is<br>
obtained if the template DNA is from the CMS line and a DNA amplification product is not<br>
obtained if the template DNA is from the cognate Male Fertile Maintainer line.<br>
In another embodiment, the invention provides method of using primers having SEQ ID<br>
No.4 &amp; SEQ ID No 5 based on this DNA sequence in a Polymerase Chain Reaction (PCR)<br>
Assay to distinguish WA Cytoplasmic Male Sterile lines of rice from their cognate Male<br>
Fertile Maintainer lines wherein detection of the PCR amplified fragment/s is by agarose<br>
gel electrophoresis followed by Ethidium bromide staining.<br>
In still another embodiment, the invention provides a method of using primers having SEQ<br>
ID No.4 &amp; SEQ ID No 5 based on this DNA sequence this in a Polymerase Chain Reaction<br>
(PCR) Assay to distinguish WACytoplasmic Male Sterile lines of rice from their cognate<br>
Male Fertile Maintainer lines wherein detection of the PCR amplified fragment/s is by<br>
detection of a radioactively labeled nucleotide that is incorporated into the PCR amplified<br>
product.<br>
In yet another embodiment, the invention provides a method of using primers having SEQ<br>
ID No.4 &amp; SEQ ID No 5 based on this DNA sequence in a Polymerase Chain Reaction<br>
(PCR) Assay to distinguish WA Cytoplasmic Male Sterile lines of rice from their cognate<br>
Male Fertile Maintainer lines wherein a non-radioactively labeled nucleotide is<br>
incorporated into the PCR amplified product and detection is by colorimetry,<br>
chemiluminescence, or measurement of fluorescence.<br>
In another embodiment, the invention provides a method of using primers having SEQ ID<br>
No.4 &amp; SEQ ID No 5 based on this DNA sequence in a PCR-ELISA (Enzyme Linked<br>
Immunosorbent Assay) format to distinguish WA Cytoplasmic Male Sterile lines of rice<br>
from their cognate Male Fertile Maintainer lines, the said PCR-ELISA format may involve;<br>
(a) use of a labeled capture probe or one labeled PCR primer that can be bound to suitably<br>
coated solid surface made of polystyrene, styrene, glass, etc., (b) use of non-radioactively<br>
labeled [the label being digoxigen (DIG), flourescein, etc.] nucleotides in the PCR and<br>
subsequent detection with anti-DIG or anti-fiourescein antibodies that are conjugated to<br>
enzymes used for ELISA like horseradish peroxidase, alkaline phosphatase, betagalactosidase,<br>
glucose oxidase, etc; modifications of the PCR-ELISA format, including<br>
alternate methods for labeling the PCR amplified fragment, attachment of probe to solid<br>
surfaces, methods of detection, etc<br>
In an embodiment, the invention provides a method of using primers having SEQ ID No.4<br>
&amp; SEQ ID No 5 based on this DNA sequence in a PCR Assay to distinguish WA<br>
Cytoplasmic Male Sterile lines of rice from their cognate Male Fertile Maintainer lines<br>
wherein the method of detection is based on the use of flourescently labeled nucleotides in<br>
Fluorescence Resonance Energy Transfer (FRET) based detection systems including<br>
Taqman, Molecular Beacon, etc which are familiar to those conversant with prior art.<br>
In an embodiment, the invention provides a method of using a multiplex PCR assay of a<br>
first pair of oligonucleotide primers having SEQ ID No.4 &amp; SEQ ED No 5 based on this<br>
DNA sequence, in conjunction with a second pair of oligonucleotide primers, wherein a<br>
DNA amplification product is obtained using the first pair of primers only if the template<br>
DNA is obtained from the WA Cytoplasmic Male Sterile lines but not from the Male<br>
Fertile Maintainer Line; and another DNA amplification product is obtained using the<br>
second set of primers irrespective of whether the template DNA is from a CMS or a Male<br>
Fertile Maintainer Line, this second pair of oligonucleotide primers can be derived from<br>
any sequenced portion of the rice genome outside the region targeted by the first primer<br>
pair, another consideration is that successful amplification of the respective target DNA<br>
Sequences should occur even when the two primer pairs are included in the same PCR<br>
mixture. The sequences of several primer pairs that belong to this second set and can be<br>
successfully multiplexed in a PCR assay with the first set of oligonucleotide primer pairs<br>
In yet another embodiment, the invention provides a method of using CMS specific DNA<br>
sequence in a Southern hybridization assay (using either radioactive or non-radioactive<br>
labeling methods) to distinguish WA Cytoplasmic Male Sterile Lines of rice from their<br>
cognate Male Fertile Maintainer lines.<br>
In still another embodiment, the invention provides a method of using co-dominant<br>
sequence specific DNA markers like micro-satellites and Sequence Tagged Sites (STSs) in<br>
a Polymerase Chain Reaction Assay for assessing the extent of out-crossing with rogue<br>
pollen donors (i.e. rice lines that are not the designated Maintainers) and consequent<br>
occurrence of genetically impure seeds during the multiplication of Cytoplasmic Male<br>
Sterile lines of rice, hi yet another embodiment, the invention provides a method wherein<br>
the DNA is isolated from single seedlings, PCR analysis is performed and the genotype is<br>
assessed by electrophoresis on agarose gels or polyacrylamide gels and detection is either<br>
by ethidium bromide staining, silver staining or methods of detection that are applicable if<br>
either a radioactive label or a non-radioactive fluorescent label is incorporated into the PCR<br>
amplified fragment.<br>
In an embodiment, the invention provides a method of using co-dominant sequence specific<br>
DNA markers like micro-satellites and Sequence Tagged Sites (STSs) in a Polymerase<br>
Chain Reaction Assay for assessing the extent of out-crossing with rogue pollen donors (i.e.<br>
rice lines that are not the designated Maintainers) and consequent occurrence of genetically<br>
impure seeds during the multiplication of Cytoplasmic Male Sterile lines of rice wherein<br>
the DNA is isolated from a population of seedlings (a typical number of individuals within<br>
the population could be a 100) that are obtained by germinating seeds of the CMS stock,<br>
PCR analysis is performed, and the extent of impurities is judged by estimating allele<br>
frequencies within the population at the locus that is being assessed. The method of<br>
estimating allele frequencies within the population involves separation of flourescently<br>
labeled PCR amplified fragments by gel electrophoresis and estimation of the heights of the<br>
peaks that correspond to specific alleles.<br>
In an embodiment, the invention provides a method of using co-dominant sequence specific<br>
DNA markers like micro-satellites and STSs in a Polymerase Chain Reaction Assay for<br>
assessing the extent of out-crossing with rogue pollen donors (i.e. lines that are not the<br>
designated Maintainers) and consequent occurrence of genetically impure seeds during the<br>
multiplication of Cytoplasmic Male Sterile lines of any crop or economically valuable plant<br>
in which the three line system of hybrid production is followed. The DNA is isolated from<br>
single seedlings, PCR analysis is performed and the genotype is assessed by electrophoresis<br>
on agarose gels or polyacrylamide gels and detection is either by ethidium bromide<br>
staining, silver staining or methods of detection that are applicable if either a radioactive<br>
label or a non-radioactive fluorescent label is incorporated into the PCR amplified<br>
fragment.<br>
In still another embodiment, the invention provides a method of using co-dominant<br>
sequence specific DNA markers like micro-satellites and STSs in a Polymerase Chain<br>
Reaction Assay for assessing the extent of out-crossing with rogue pollen donors (i.e. lines<br>
that are not the designated Maintainers) and consequent occurrence of genetically impure<br>
seeds during the multiplication of Cytoplasmic Male Sterile lines of any crop or<br>
economically valuable plant in which the three line system of hybrid production is followed<br>
and the DNA is isolated from a population of seedlings (a typical number of individuals<br>
within the population could be a 100) that are obtained by germinating seeds of the CMS<br>
stock, PCR analysis is performed, and the extent of impurities is judged by estimating allele<br>
frequencies within the population at the locus that is being assessed. The method of<br>
estimating allele frequencies within the population would involve separation of<br>
flourescently labeled PCR amplified fragments by gel electrophoresis and estimation of the<br>
heights of the peaks that correspond to specific alleles.<br>
In an embodiment, the invention provides a method of using co-dominant sequence specific<br>
DNA markers like micro-satellites and STSs in a Polymerase Chain Reaction Assay for<br>
assessing the extent of purity of parental lines of rice hybrids wherein a two line system for<br>
hybrid rice production is followed. These parental lines include a female parent that has<br>
conditional male sterility wherein sterility is induced by temperature, photoperiod,<br>
treatment with chemicals that induce lethality of the male gametes etc. The DNA is isolated<br>
from single seedlings, PCR analysis is performed and the genotype is assessed by<br>
electrophoresis on agarose gels or polyacrylamide gels and detection is either by ethidium<br>
bromide staining, silver staining or methods of detection that are applicable if either a<br>
radioactive label or a non-radioactive fluorescent label is incorporated into the PCR<br>
amplified fragment.<br>
In another embodiment, the invention provides a method of using co-dominant sequence<br>
specific DNA markers like micro-satellites and STSs in a Polymerase Chain Reaction<br>
Assay for assessing the extent of purity of parental lines of rice hybrids wherein a two line<br>
system for hybrid nee production is followed. These parental lines include a female parent<br>
that has conditional male sterility wherein sterility is induced by temperature, photoperiod,<br>
treatment with chemicals that induce lethality of the male gametes, etc. What is claimed is a<br>
method wherein the DNA is isolated from a population of seedlings (a typical number of<br>
individuals within the population could be a 100) that are obtained by germinating seeds of<br>
the parental lines, PCR analysis is performed, and the extent of impurities is judged by<br>
estimating allele frequencies within the population at the locus that is being assessed. One<br>
method of estimating allele frequencies within the population would involve separation of<br>
flourescently labeled PCR amplified fragments by gel electrophoresis and estimation of the<br>
heights of the peaks that correspond to specific alleles.<br>
In yet another embodiment, the invention provides a method of using co-dominant<br>
sequence specific DNA markers like micro-satellites and STSs in a Polymerase Chain<br>
Reaction Assay for assessing the extent of purity of parental lines of hybrids of any<br>
economically important crop wherein a two line system for hybrid production is followed;<br>
these parental lines include a female parent that has conditional male sterility wherein<br>
sterility is induced by temperature, photoperiod, treatment with chemicals that induce<br>
lethality of the male gametes, etc. The DNA is isolated from single seedlings, PCR analysis<br>
is performed and the genotype is assessed by electrophoresis on agarose gels or<br>
polyacrylamide gels and detection is either by ethidium bromide staining, silver staining or<br>
methods of detection that are applicable if either a radioactive label or a non-radioactive<br>
fluorescent label is incorporated into the PCR amplified fragment.<br>
In an embodiment, the invention provides a method of using co-dominant sequence specific<br>
DNA markers like micro-satellites and STSs in a Polymerase Chain Reaction Assay for<br>
assessing the extent of purity of parental lines of hybrids of any economically important<br>
crop wherein a two line system for hybrid production is followed. These parental lines<br>
include a female parent that has conditional male sterility wherein sterility is induced by<br>
temperature, photoperiod, treatment with chemicals that induce lethality of the male<br>
gametes, etc. What is claimed is a method wherein the DNA is isolated from a population<br>
of seedlings (a typical number of individuals within the population could be a 100) that are<br>
obtained by germinating seeds of each of the parental lines, PCR analysis is performed, and<br>
the extent of impurities is judged by estimating allele frequencies within the population at<br>
the locus that is being assessed. Estimating allele frequencies within the population would<br>
involve separation of flourescently labeled PCR amplified fragments by gel electrophoresis<br>
and estimation of the heights of the peaks that correspond to specific alleles.<br>
In an embodiment, the invention provides a method of using co-dominant sequence specific<br>
DNA markers like micro-satellites and STSs in a Polymerase Chain Reaction Assay for<br>
assessing purity of hybrid seeds when the seeds (or plants) are of rice and either a Three<br>
line or Two line system is used for hybrid production. The DNA is isolated from single<br>
seedlings, PCR analysis is performed and the genotype is assessed by electrophoresis on<br>
agarose gels or polyacrylamide gels and detection is either by ethidium bromide staining,<br>
silver staining or methods of detection that are applicable if either a radioactive label or a<br>
non-radioactive fluorescent label is incorporated into the PCR amplified fragment.<br>
In an embodiment, the invention provides a method of using co-dominant sequence specific<br>
DNA markers like micro-satellites and STSs in a Polymerase Chain Reaction Assay for<br>
assessing purity of hybrid seeds when the seeds (or plants) are of rice and either a Three or<br>
Two line system is used for hybrid production; and the DNA is isolated from a population<br>
of seedlings (a typical number of individuals within the population could be 400) that are<br>
obtained by germinating seeds of the hybrid stock, PCR analysis is performed, and the<br>
extent of impurities is judged by estimating allele frequencies within the population at the<br>
locus that is being assessed. Estimating allele frequencies within the population would<br>
involve separation of flourescently labeled PCR amplified fragments by gel electrophoresis<br>
and estimation of the heights of the peaks that correspond to specific alleles.<br>
In an embodiment, the invention provides a method of using co-dominant sequence<br>
specific DNA markers like SSRs and STSs in a Polymerase Chain Reaction Assay for<br>
assessing purity of hybrid seeds when the seeds (or plants) are of any crop or economically<br>
valuable plant and either a Three or Two line system is used for hybrid production. The<br>
DNA is isolated from single seedlings, PCR analysis is performed and the genotype is<br>
assessed by electrophoresis on agarose gels or polyacrylamide gels and detection is either<br>
by ethidium bromide staining, silver staining or methods of detection that are applicable if<br>
either a radioactive label or a non-radioactive fluorescent label is incorporated into the PCR<br>
amplified fragment.<br>
In an embodiment, the invention provides a method of using co-dominant sequence<br>
specific DNA markers like SSRs and STSs in a Polymerase Chain Reaction Assay for<br>
assessing purity of hybrid seeds when the seeds (or plants) are of any crop or economically<br>
valuable plant and either a Three or Two line system is used for hybrid production; and the<br>
DNA is isolated from a population of seedlings (a typical number of individuals within the<br>
population could be 400) that are obtained by germinating seeds of the hybrid stock, PCR<br>
analysis is performed, and the extent of impurities is judged by estimating allele<br>
frequencies withm the population at the locus that is being assessed. Estimating allele<br>
frequencies within the population would involve separation of flourescently labeled PCR<br>
amplified fragments by gel electrophoresis and estimation of the heights of the peaks that<br>
correspond to specific alleles.<br>
To describe the invention in detail, initially, a PCR assay was performed on template DNA<br>
individually isolated from a set of rice lines that constitute the CMS (WA cytoplasm),<br>
Maintainer and the corresponding hybrid using oligonucleotide primers that amplify the<br>
RM9 rice microsatellite locus (McCouch et al., 1996. Rice Genetics IE, Proc. Third Intl.<br>
Rice Genet Symp. Los Banos Manila, the Philippines. 16-20 Oct. 1995. International Rice<br>
Research Institute, Manila, the Philippines). The PCR amplified fragments were separated<br>
by agarose gel electrophoresis and detected by staining with ethidium bromide. As expected<br>
for a PCR assay in which the RM9 locus is assayed, a PCR amplified fragment of<br>
approximately 250 bp was observed using template DNA from all three lines (Fig. 2).<br>
Besides this, a PCR amplified fragment of approximately 350 bp was observed only in<br>
reactions in which the template DNA was obtained from either the CMS line or the hybrid<br>
(Fig. 2). However, PCR amplification of this fragment was found to be highly sensitive to<br>
reaction conditions and was often not observed even when the template DNA was from<br>
either CMS lines or hybrids. This indicated that primers directed against the RM9<br>
microsatelite locus could not be reliably used to distinguish CMS and maintainer lines. We<br>
think that this lack of reproducibility arises due to imperfect pairing between the RM9<br>
primers and the target sequence in the lines containing the CMS cytoplasm.<br>
Subsequently, the CMS line specific PCR amplified fragment (size 325 bp) indicated in<br>
Figure 2 was purified and it's nucleotide sequence was determined on an automated DNA<br>
sequencer by using the Forward and Reverse oligonucleotide primers from the RM9 locus.<br>
The nucleotide sequence of the PCR amplified fragment is indicated in Figure 3.<br>
A search for homologous DNA sequences in the GenBank DNA database was performed<br>
using the BLAST algorithm (Altschul et al., 1997. Nucleic Acids Res. 25: 3339-3402)<br>
through the web site of the National Center for Biotechnology Information, Bethesda,<br>
Maryland, USA. The said DNA sequence is highly homologous to one particular entry<br>
[accession #D21251; DNA database of Japan (DBJ), hereby incorporated as a reference] in<br>
the database. Thus entry is a sequence of rice (Oryza saliva) mitochondrial DNA originating<br>
from a genomic region encoding genes for ribosomal protein S3, L16, S12 and NADH<br>
dehydrogenase subunit 3. The extent of homology of the said DNA sequence to rice<br>
mitochondrial DNA is depicted in Figure 4. It is clear that the regions are highly<br>
homologous excepting a stretch of DNA [corresponding to nucleotides<br>
ACGGCCCTCATCACCTTCTTTCACTTTTTGTTTTTG (SEQ ID No. 3)] that is absent<br>
in the sequence of rice mitochondrial DNA deposited as accession #D21251.<br>
Based on this sequence, one pair of oligonucleotide primers (Table 2 shown at tne end of<br>
the description) that can be used in a PCR assay to distinguish CMS (WA) and Maintainer<br>
lines of rice was designed. One of the primers (Forward primer) is based on the sequence<br>
that is specific to the CMS lines (Figure 3) while the other primer (Reverse primer) is based<br>
on the sequence of rice mitochondrial DNA in the genomic region upstream of the gene for<br>
ribosomal protein S3 between nucleotide #s!362 and 1384 as indicated in DBJ accession<br>
#021251. The oligonucleotide primers [Forward 5'-ACTTTTTGTTTTTGTGTAGG-3'<br>
(SEQ ID No. 4); Reverse 5'-TGCCATATGTCGCTTAGACTTTAC-3' (SEQ ID No. 5)]<br>
were used in a PCR assay with template DNA from the CMS and Maintainer lines of rice<br>
that are listed in Table 1 (shown at the end of the description).<br>
A PCR amplified product of 325 bp was obtained (Figure 5) using the primers SEQ ID No.<br>
4 and SEQ ID No. 5 when the template DNA was isolated from the CMS lines (IR 58025<br>
A, IR 62829 A, PMS 8 A, PMS 10 A, 78897 A) but was not obtained when the source of<br>
the template DNA was the Maintainer lines (IR 58025 B, IR 62829 B, PMS 8 B, PMS 10<br>
B, 78897 B). In a further validation of the assay, a coded test was conducted in which<br>
genomic DNA was isolated from 35 different rice plants, 15 of which were of the CMS line<br>
IR 58025 A and 20 were of the Maintainer line IR 58025 B. The PCR assay was conducted<br>
without a knowledge as to which of these 35 plants are of the CMS line and which are of<br>
the Maintainer line. This assay was used to accurately predict the genotype of each of the<br>
35 different plants indicating the applicability of the PCR assay as a method for<br>
distinguishing CMS and Maintainer lines of rice.<br>
The said CMS specific PCR amplified fragment was radiolabelled using ^p-dATP.<br>
Genomic DNAs isolated from cognate pairs of CMS (IR 58025 A and IR 62829 A) and<br>
Maintainer (IR 58025 B and IR 62829 B) lines were digested with any one of several<br>
different restriction enzymes, separated by agarose gel electrophoresis and hybridized<br>
(as described in Sambrook et al. 1989. Cold Spring Harbor, NY, USA) to the radiolabelled<br>
probe. Figure 6 indicates that a restriction fragment length polymorphism between CMS<br>
(WA) and maintainer lines of rice is detected using this probe and the EcoR V restriction<br>
enzyme. Purification of mitochondrial DNAs from CMS and maintainer lines of rice and<br>
RFLP analysis using the above mentioned probe-enzyme combination revealed a similar<br>
polymorphism confirming that the detected polymorphism is in the mitochondrial DNA.<br>
Polymorphisms were also detected using other restriction enzymes that were tested.<br>
A multiplex PCR assay was also developed for distinguishing CMS and Maintainer lines of<br>
rice. In this assay, the first set of oligonucleotide primers used in the above mentioned PCR<br>
assay were multiplexed with primers belonging to a second set of oligonucleotide primers.<br>
This second set of oligonucleotide primers were designed based on either a sequence of rice<br>
chromosome I that was obtained as part of the ongoing International Rice Genome Project<br>
(and publicly available as GeneBank Accession Number #AP001859; hereby incorporated<br>
as reference) or the sequence of rice mitochondrial DNA that is available as DBJ Accession<br>
#D21251. This second set of oligonuleotide primers were designed such that any one of the<br>
primer pairs belonging to this second set can be added to a PCR assay mixture containing<br>
primers belonging to the first set of oligonucleotide primers without affecting the formation<br>
of the CMS specific DNA fragment that is amplified by the first set of oligonucleotide<br>
primers. In this multiplex PCR assay, the second set of oligonucleotide primers amplify a<br>
specific DNA fragment irrespective of whether the template DNA is from a CMS line or a<br>
Maintainer line (Figure 7). As a PCR amplified fragment is obtained from both the CMS<br>
and Maintainer lines, the use of this second set of primers serves as a control for extraneous<br>
factors (like inhibitors of PCR, poor quality of the template DNA, etc.) that can affect the<br>
outcome of the PCR. The sequences of the second set of oligonucleotide primers, their<br>
location within the rice genome and the size of the DNA fragment that is amplified by the<br>
use of these primers is indicated in Table 2.(shown at the end of the description).<br>
Microsatellites (also called simple sequence repeats or SSRs) are simple, tandemly repeated<br>
di-to tetra-nucleotide sequence motifs flanked by unique sequences. Microsatellites are<br>
abundant and well distributed throughout the genome in rice (McCouch et al 1997) as<br>
well as many other crop plants (Powell et al, 1996. Trends Plant Sci 1: 215-222).<br>
Microsatellites are valuable as genetic markers because they are co-dominant, detect high<br>
levels of allelic diversity and are efficiently assayed by the PCR. The current level of<br>
average genome-wide coverage provided by microsatellites in rice, one in every 6 cM<br>
(Temnykh et al, 2000. Theor. Appl. Genet. 100:697-712) is sufficient to be useful for<br>
assessment of hybrid seed purity and for genotype identification. Similar to microsatellites,<br>
a STS is a short stretch of genomic sequence that can be detected by PCR and is mapped to<br>
a specified site as a landmark in the genome. Some of the STSs mapped in rice are<br>
polymorphic (Ghareyazie at al, 1995. Theor. Appl. Genet. 91:218-227; Robenoil et al,<br>
1996./H Khush, G. S. (ed) Rice genetics III. Proc. Third Intl. Rice Genet. Symp. Los Banos<br>
Manila, the Philippines. 16-20 Oct. 1995. International Rice Research Institute, Manila, the<br>
Philippines.).<br>
A minimum isolation distance of 300 metres is prescribed for multiplication of rice CMS<br>
lines (Virmani, 1993. Hybrid rice. Advances in Agronomy 57:377-462) i.e. Within this<br>
distance no rice lines other than the Maintainer line (the preferred pollen donor or male<br>
parent) should be cultivated. This is based on the observation that pollen originating from<br>
rice plants growing beyond this distance will not pollinate the CMS line. Occasionally, this<br>
minimum isolation distance is either not strictly followed or local conditions (for eg. wind<br>
flow and weather) might permit pollen to be transferred from plants that are growing<br>
beyond the 300 metres distance.<br>
The extent of cross pollination of CMS lines with rogue pollen originating from rice lines<br>
(other than Maintainer line) that are growing in nearby fields can be assessed using<br>
Microsatellite or STS markers that are polymorphic between these rice lines and the CMS<br>
line. These polymorphic markers are identified by PCR, agarose gel electrophoresis and<br>
ethidium bromide staining using template DNA from each of the potential donor 'inss (in a<br>
typical situation this would be between 1-3 lines) and the CMS line. A preferable marker<br>
would be one that will detect cross pollination arising from any one of these rogue lines. A<br>
plant arising from cross pollination would be detected using such a marker by the presence<br>
of heteozygosity (i.e presence of two DNA fragments of different sizes one of which is<br>
contributed by the female (CMS) parent while the other is contributed by the male<br>
parent (rogue pollen donor). In contrast to this situation, homozygosity (presence of a single<br>
DNA fragment) will be observed in the offspring, if pollination occurs, as desired, with<br>
pollen emanating from the Maintainer line. This is because the CMS and Maintainer lines<br>
are isonuclear to each other; meaning that they are essentially identical to each other with<br>
respect to the nuclear DNA markers. The procedure for DNA isolation, PCR and agarose<br>
gel electrophoresis is as described in Example 9.<br>
A variation of this assay is also described in Example 9 wherein the DNA is isolated from<br>
leaves obtained from a population of seedlings (obtained from a pool of upto 100 or more<br>
seeds) belonging to the CMS stock. One of the primers for detecting the polymorphic<br>
marker is labeled at the 5' end with a fluorescent tag such as flourescein, rhodamine and<br>
such other dyes that can be detected following PCR and electrophoresis using instruments<br>
like the Gene Scan facility of an ABI 377 DNA Sequencer or such other similar instruments<br>
known to those familiar with the art. By this assay, the extent of cross pollination can be<br>
detected by measuring the height of each of the peaks (each peak is characteristic of one<br>
allele) as detected by the instrument. If only one peak (corresponding to one allele) is<br>
detected, the sample of CMS seeds can be construed to be 100% pure. Impurities are<br>
detected by the occurrence of multiple alleles (peaks) in the sample. The ratio of the height<br>
of the contaminating peak (allele that is contributed to the population by the rogue donor)<br>
to the height of the expected peak (allele that is characteristic of the CMS line) will indicate<br>
the frequency of seeds within the CMS stock that have arisen following cross pollination<br>
with rogue pollen donors.<br>
Although the application, as indicated above, is for estimating the extent of out-crossing<br>
that has occurred during the multiplication of CMS lines of rice, a similar approach can be<br>
used for estimating the extent of out-crossing that has occurred within the CMS lines of<br>
other crops including but not limited to maize, pearl millet, sorghum, wheat, sunflower,<br>
mustard, cabbage, cauliflower, tomato, pepper, okra, etc wherein the CMS lines are used<br>
for the production of hybrids and appropriate micro-satellite or STS markers are available.<br>
The estimation of seed purity is an important quality control component of a hybrid rice<br>
program. This is conventionally done by the grow out test (GOT), which is based on the<br>
assessment of morphological and floral characteristics in plants grown to maturity (Ref).<br>
For the seed industry, large amounts of capital are locked up for extended periods in the<br>
form of stored hybrid seeds awaiting the results of the GOT. With the objective of replacing<br>
the GOT with DNA based assays, rice Cytoplasmic Male Sterile (CMS), Restorer and<br>
hybrid lines were screened to distinguish these, using microsatellite and Sequence Tagged<br>
Site (STS) polymorphisms. These co-dominant polymorphisms were identified after PCR,<br>
agarose gel electrophoresis and ethidium bromide staining. The principle of this method is<br>
that the hybrid would be heterozygous (i. e. both parental alleles would be present in the<br>
hybrid) when the genotype is assessed using microsatellite or STS markers that are<br>
polymorphic between the parental lines (i.e. detect a different allele in the two parents). A<br>
simple procedure for DNA isolation and detecting heterozygosity and purity, has been<br>
standardized (described below) using three-day-old rice seedlings, and has been used for<br>
detection of impurities in hybrid seed lots (Example 10). Although multiple polymorphic<br>
markers can be used (either singly or by multiplexing) for assessing seed purity, for reasons<br>
of cost considerations, it is suggested that a single appropriately chosen microsatellite<br>
marker should be sufficient for assessing hybrid seed purity.<br>
A variation of this assay is also described in Example 10 wherein the DNA is isolated from<br>
leaves obtained from a population of 400 seedlings (obtained from a pool of 400 seeds)<br>
belonging to the hybrid seed stock. One of the primers for detecting the polymorphic<br>
marker is labeled at the 5' end with a fluorescent tag such as flourescein, rhodamine and<br>
such other dyes that can be detected following PCR and electrophoresis using instruments<br>
like the Gene Scan facility of an ABI 377 DNA Sequencer or such other instruments known<br>
to those familiar with the art. By this assay, the extent of purity can be detected by<br>
measuring the height of each of the expected two peaks that are characteristic of the hybrid<br>
(each peak represents one allele that is contributed by one parent) that are detected by the<br>
instrument. Using genomic DNA isolated from a single hybrid plant as a template in the<br>
PCR assay, it is expected that the peak heights would be equal, leading to a ratio of 1:1 (or<br>
50:50). Using genomic DNA isolated from a population of four hundred seedlings as a<br>
template in the PCR assay, any deviation from a ratio of 1 : 1 (50:50) for the heights of<br>
the two peaks would be an indicator of the extent of impurities in the hybrid seed stock. For<br>
e.g., a ratio of peak heights of 1.02 : 0.98 (51 : 49) would correspond to a sample of hybrid<br>
seed stock that is 98% pure and a ratio of peak heights of 1.1 : 0.9 (55 : 45) would<br>
correspond to a sample of hybrid seed stock that is 90% pure.<br>
The markers that are used for assessing hybrid seed purity should be carefully selected after<br>
taking into consideration, the varieties grown in adjacent fields that can serve as potential<br>
pollen donors either during CMS line multiplication (wherein only the Maintainer line<br>
should be the pollen donor) or hybrid seed production (wherein only the Restorer line<br>
should be the pollen donor). If pollination is by rogue pollen donors (i.e. lines that are<br>
neither Maintainers nor Restorers is found to occur), and the rogue donor is polymorphic<br>
with the CMS line in respect to the chosen microsatellite or STS marker, then<br>
heterozygosity would be observed even in the absence of production of the desired hybrid.<br>
Therefore, it is very important that the marker/s selected for assessing hybrid seed purity<br>
should be monomorphic between the CMS line and potential rogue pollen donors but<br>
polymorphic between CMS and Restorer lines. Detection of the expected heterozygosity<br>
will then be an indicator of hybrid seed production. These specific markers can be<br>
identified in polymorphism surveys conducted using either micro-satellite or STS markers<br>
on CMS, Restorer and potential rogue donor lines. Detection of these polymorphic markers<br>
would not be difficult for those familiar with prior art as a very large number of microsatellite<br>
markers are currently available for rice (Temnykh et al, 2000).<br>
The DNA marker assay is superior to the GOT in terms of speed and accuracy and it's<br>
implementation will result in considerable savings for the seed industry. Although the<br>
method was standardized for establishing the purity of rice hybrids produced through a<br>
three line breeding system of the type described in Figure 1, it can also be applied for<br>
establishing the purity of rice hybrids obtained through a two line breeding system. In this<br>
two line breeding system, a conditionally male sterile line is used as the female parent for<br>
hybrid production by growing under conditions which induce male sterility. This line can<br>
be propagated by self fertilization when it is grown under conditions that do not induce<br>
male sterility. Therefore, the need for Cytoplasmic Male Sterile and Maintainer lines<br>
can be avoided. The two line breeding systems are, by and large, in the experimental stages<br>
(except to a small extent in China). However, even for a two line system of hybrid seed<br>
production, the scheme for estimating hybrid seed purity would be the same as described<br>
for the three line breeding system. In this case, microsatellite and STS polymorphisms that<br>
distinguish the female and male parents (instead of the CMS and Restorer lines) would be<br>
identified and used as described above.<br>
Although this method was standardized for establishing the purity of rice hybrids, it can<br>
also be used for establishing the purity of hybrids (produced either through the tv/o line or<br>
three line breeding systems) in any other crop including but not limited to maize, pearl<br>
millet, sorghum, wheat, sunflower, mustard, cabbage, cauliflower, tomato, pepper, okra, etc<br>
for which suitable microsatellite and STS markers are available. It is anticipated that this<br>
assay (or variations thereof that would be apparent to those familiar with prior art) would<br>
find wide applications in hybrid seed quality control programs and result in significant<br>
benefits for the hybrid seed industry. The farmers who use seeds tested by this process<br>
would also benefit as they would be getting a properly authenticated product.<br>
EXAMPLE 1<br>
Isolation of genomic DNA from CMS and maintainer lines of rice.<br>
Rice genomic DNA was isolated using either the protocols described by (a) Kochert et al.<br>
(1989) Rockefeller Program on Rice Biotechnology, Cornell Univ., New York, USA, or (b)<br>
Chunwongse et al (1993) Theor. Appl. Genet. 86:694-698, with slight modifications.<br>
(a) Briefly, isolation using the Kochert et al protocol is as follows: 5-10 g of leaves<br>
obtained from 20 days old greenhouse grown plants were ground in a mortar and<br>
pestle in Liquid Nitrogen until a fine powder formed, without allowing powder to<br>
thaw. Sample were then transferred to chloroform-resistant 50 ml tubes<br>
(Polypropanol tubes) containing 25 ml of extraction buffer (420 g urea, 70 ml 5 M<br>
Nacl, 50 ml 1 M Tris-Hcl pH 8.0, 80 ml 0.25 M EDTA, 200 ml 10% SDS, 50 ml<br>
Phenol reagent, volume made up to 1 litre with steriled double distilled water) prewarmed<br>
to 60°C. Any clumps that formed were broken up with a glass rod and<br>
0.750 ml of 20% Sodium Lauryl Sulfate was added and mixed well. The mixture<br>
was incubated at 60°C for 10 minutes with inverting at regular intervals. This was<br>
cooled to room temperature and 15 ml of Chlorofornrlsoamyl alcohol (24:1) was<br>
added and mixed well to get an emulsion. The tubes were centrifuged to separate<br>
the aqueous and chloroform phases. With a pipet, the aqueous upper phase was<br>
transferred into a fresh 50 ml tube. 2/3 to one volume of isopropanol was added to<br>
the final aqueous phase and inverted and mixed until DNA comes together. DNA<br>
was spooled out and transferred to a fresh tube containing 70% ethanol. DNA was<br>
pelleted by centrifugation and dried in a vacuum dryer after decanting the ethanol.<br>
DNA was dissolved in TE (Tris-Hcl lOmM pH 8.0 and EDTA 1 mM pH 8.0).<br>
(b) DNA isolation using the protocol of Chunwongse et al (1993) is as follows: seeds<br>
were germinated at 28°C in the dark on moistened filter paper in Petri dishes. Three<br>
day old seedlings were crushed individually with a pestle in a 1.5 ml tube "ontaining<br>
200 |al of extraction buffer made up of 5% W/V Chelex-100 (Bio-Rad Laboratories,<br>
USA) in sterile distilled water. The homogenate was incubated at 95°C for 10 min<br>
and centrifuged at 12,000 rpm for 1 min. 10-15 ul of the supernatant was used for<br>
each PCR reaction.<br>
EXAMPLE 2<br>
Isolation of mitochondria! DNA from CMS and maintainer lines of rice.<br>
Mitochondrial DNA was isolated from CMS and maintainer lines according to the<br>
protocols of Saleh et al. (1989) Theor. Appl. Genet. 77: 617-619. Surface sterilized seeds<br>
were germinated and grown for 14 days in a Greenhouse. The plants were then kept in the<br>
dark for three days for the plants to etiolate. After this step, all experiments were performed<br>
at 4 C. 20 g of leaves from 4 day old plants were collected and cut into small pieces. The<br>
leaves were homogenized in a mortar and pestle in 100 ml of Buffer A (10 mM TES pH<br>
7.2, 0.5 M Mannitol, 0.2% BSA, 0.05% cysteine). The homogenate was filtered through 4<br>
layers of cheesecloth. This was Centrifuged at 5,000 rpm for 10 min and the supernatant<br>
was collected. The pellet was gently resuspended in 30 ml of Buffer A and centrifuged<br>
again at 5,000 rpm for 10 min. The supematants were combined and centrifuged at 12,000<br>
rpm for 10 min and the pellet was collected. The supernatant was centrifuged again at<br>
12,000 rpm for 10 min and the two pellets were combined. The pellet was resuspended in<br>
Buffer A and centrifuged at 5,000 rpm for 10 min and the supernatant was collected. 1 M<br>
MgCl2 and 10 mg/ml freshly prepared DNase I (in 0.15 M NaCl and 50% glycerol) was<br>
added to give a final concentration of 10 mM MgCl2 and 10 ug DNase/g fresh weight of<br>
leaf tissue and incubated at 4°C for 1 h. A sucrose gradient was made with Buffer B (10<br>
mM TES pH 7.2, 20 mM EDTA, 0.6 M sucrose) and 2 ml of mitochondrial suspension was<br>
loaded in each tube. This was centrifuged at 16,000 rpm for 10 min. The pellet was<br>
resuspended in 4 ml Buffer C (50 mM Tris-Hcl pH 8.0, 10 mM EDTA pH 8.0, 2%<br>
sarkosyl, 100 (ig/ml proteinase K) and incubated at 37°C on a shaker bath (Jolabo,<br>
Germany) with gentle agitation for 2 h. The lysate was made up to 0.2 M Ammonium<br>
acetate and purified by 3 cycles of phenol-chloroform extraction. Precipitation with 95%<br>
ethanol and subsequent washings were with 70% ethanol. The pellet was dried under<br>
vacuum. DNA was dissolved in TE, treated with Rnase A and stored at -20°C.<br>
EXAMPLE 3<br>
Identification of a DNA sequence that is specific to CMS lines of rice.<br>
Primers used: Oligonucleotide primers for the RM9 microsatellite locus of rice (MoCouch<br>
et al., 1996. Rice Genetics HI, Proc. Third Intl. Rice Genet. Symp. Los Banos Manila, the<br>
Philippines. 16-20 Oct. 1995. International Rice Research Institute, Manila, the<br>
Philippines).<br>
Forward: 5'-CAAAAACAGAGCAGATGAC-3' (SEQ DD No. 6)<br>
Reverse: 5'-CTCAAGATGGACGCCAAGA-3' (SEQ ID No. 7)<br>
Primers were synthesized by Oswel DNA Service, Southamton, U. K. Polymerase Chain<br>
Reaction (PCR) conditions were as described by McCouch et al. (1996) Rice Genetics in,<br>
Proc. Third Intl. Rice Genet. Symp. Los Banos Manila, the Philippines. 16-20 Oct. 1995.<br>
International Rice Research Institute, Manila, the Philippines with slight modifications.<br>
Briefly, PCR was performed in 25 ul reaction volume containing 50 to 100 ng of template<br>
DNA, 5 picomoles of each primer, 200 uM (each) deoxyribonucleotides, 50 mM KC1, 10<br>
mM Tris-HCl, pH 8.3, 1.5 mM MgCl2, 0.01% gelatin and 1 unit of Taq polymcrase. PCR<br>
conditions were: 95°C for 7 min (initial denaturation), followed by 35 cycles of<br>
denaturation at 94 C for 1 min, annealing at 55°C for 1 min, extension at 72°C for 2 min<br>
and a final extension of 5 min at 72°C. Samples were stored at 4 C until further use. PCR<br>
products were detected by separating the DNA out in a 1% or 2% agarose gel by<br>
electrophoresis and staining with ethidium bromide to visualize the DNA fragments under<br>
Ultraviolet light.<br>
PCR was performed using a Maintainer line, CMS line and the cognate hybrid. Apart from<br>
the expected DNA fragment amplified by the RM9 primer, an extra band was observed<br>
only in the CMS line and the hybrid but absent in the Maintainer line (Figure 2).<br>
EXAMPLE 4<br>
Determination of a nucleotide sequence that is specific to CMS lines of rice.<br>
Using RM9 primers, PCR was performed in 25 u.1 reaction volume containing 50 to 100 ng<br>
of template DNA, 5 picomoles of each primer, 200 uM (each) deoxyribonucleotides, 50<br>
mM KC1, 10 mM Tris-HCl, pH 8.3, 1.5 mM MgCl2, 0.01% gelatin and 1 unit of Taq<br>
polymerase. PCR conditions were: 95 C for 7 min (initial denaturation), followed by 35<br>
O O . 0 cycles of denaturation at 94 C for 1 min, annealing at 55 C for 1 min, extension at 72 C for<br>
0 O<br>
2 min and a final extension of 5 min at 72 C. Samples were stored at 4 C until further use.<br>
Amplified PCR products were separated on an agarose gel, stained with ethidium bromide<br>
and visualized under Ultraviolet light. A DNA sequence of about 350 base pairs that is<br>
specific to CMS lines (Figure 2) was than eluted out from the gel and purified using a<br>
Qiaquick Gel Extraction Kit (Qiagen, Germany) according to the Manufacturers'<br>
instructions. The purified DNA was than sequenced using an automated DNA sequencer<br>
ABI 3700 (ABI, Foster City, USA). The sequence obtained is shown in Figure 3. A search<br>
for DNA homology in the GenBank indicates (Figure 4) that most of the sequence has<br>
homology to a region of the rice mitochondria! DNA (DBJ Accession # D21251) However,<br>
a small part of the sequence did not exhibit any homology to rice mitochondrial DNA.<br>
EXAMPLE 5<br>
Design of specific oligonucleotide primers that can be used in a PCR assay to distinguish<br>
CMS and Maintainer lines of rice.<br>
PCR primers were designed based on the sequence information obtained using the<br>
protocols described in Example 4. One primer was based on the part of the sequence that is<br>
unique to the CMS lines (base positions 1 to 36 in Figure 3) and the other primer was<br>
based on the rice mitochondria! DNA sequence. The primers are:<br>
Forward: 5'-ACTTTTTGTTTTTGTGTAGG-3' (SEQ ID No. 4)<br>
Reverse: 5'-TGCCATATGTCGCTTAGACTTTAC-3' (SEQ ID No. 5)<br>
EXAMPLE 6<br>
A PCR assay for distinguishing CMS and Maintainer lines of rice.<br>
Using the primer pair described in Example 6, PCR was performed in 25 ^il reaction<br>
volumes with CMS and Maintainer lines as template DNA. The reaction composition was,<br>
50-100 ng of template DNA, IX PCR buffer (50 mM KC1, 10 mM Tris-HCl, pH 8.3, 1.5<br>
mM MgCl2, 0.01% gelatin), 200(j.M dNTP (each), 1 unit of Taq polymerase and 5 pico<br>
0<br>
moles of each primer. The PCR conditions were: Initial denaturation at 95 C for 7 min<br>
followed by 35 cycles of 94 °C for 30 sec, 44 °C for 1 min, 72 °C for 2 min. A final<br>
0 0 extension was given at 72 C for 7 min and then the samples were stored at 4 C until<br>
further use. PCR products were separated on a 1% agarose gel and the DNA bands detected<br>
by staining with ethidium bromide and visualizing under ultraviolet light. Using this primer<br>
pair, there was an amplification of a 350 bp DNA fragment in the CMS lines but no PCR<br>
amplification was observed in the Maintainer lines (Figure 5).<br>
EXAMPLE 7<br>
A DNA-DNA hybridisation assay using CMS specific PCR amplified DNA fragment as a<br>
probe for detection of Restriction Fragment Length Polymorphisms (RFLP) that distinguish<br>
the CMS and Maintainer lines.<br>
Genomic DNA isolated (as described in Example 1 (a) from IR 58025 A (CMS) and IR<br>
58025 B (Maintainer) lines of rice were digested with various restriction enzymes viz.<br>
EcoR I, Hind HI, Dra I, EcoR V, Mine II, Sau 96 I, Taq a. I (New England Biolabs Inc.,<br>
MA, USA). Three to four micrograms of completely digested DNA from both CMS and<br>
Maintainer lines were separated electrophoretically on 0.7% agarose (Sigma, USA) gels,<br>
denatured, neutralized, and vacuum transferred to Hybond N (Amersham Life Science,<br>
Buckinghamshire, England) membranes according to the procedure given by Sambrook et<br>
al (1989) Cold Spring Harbor, NY, USA. DNA was crosslinked to membrane using a UV<br>
Stratalinker (Stratagene, La Jolla, CA, USA). Blots were pre-hybridized in a solution of<br>
0.5 M sodium phosphate (pH 7.2), 7% Sodium dodecyl sulfate (SDS), 1% bovine serum<br>
albumin and 1 mM EDTA for 3 h at 65°C. Probe (PCR amplification product of SEQ ID<br>
No. 4 and SEQ ID No. 5) was labeled with a32P-dATP using a random primer labelling kit<br>
(JONAKI-BRIT, Mumbai, India) as described by the manufacturer and hybridized for 18 h<br>
at 65°C with constant shaking. Blots were washed at 65 C with 2X SSC (IX SSC = 0.15<br>
M sodium chloride, 0.015 M sodium citrate, pH 7.0), 0.1% SDS and 5 mM sodium<br>
phosphate (pH 7.0) for 3 x 20 min and with 0.5X SSC, 0.1% SDS and 3 mM sodium<br>
phosphate buffer (pH 7.0) for 3 x 20 min. Autoradiography was done by exposing the blots<br>
to X-Ray film at -70°C. As an illustration, the RFLP obtained using the restriction enzyme<br>
EcoR V is depicted in Figure 6.<br>
EXAMPLE 8<br>
A multiplex PCR assay for distinguishing CMS and Maintainer lines of rice.<br>
We have also developed a multiplex PCR assay wherein a first set of oligonucleotide<br>
primer (Table 2 shown at the end of the description) can be successfully multiplexed in a<br>
single PCR with any primer pair belonging to a second set of oligonucleotide primers<br>
(Table 3) to distinguish CMS and Maintainer lines.<br>
The first set of primers amplify a 325 bp fragment using template DNA from CMS line as a<br>
target but no amplification product is obtained using genomic DNA of the Maimainer line<br>
as a target. Any primer pair belonging to the second set of oligonucleotide primers will<br>
amplify a single fragment irrespective of whether the target DNA is from either the CMS or<br>
the Maintainer line. This procedure is described below. Rice genome sequences available in<br>
the public domain (GeneBank) were downloaded and primers were designed based on these<br>
sequences. One primer pair was based on the sequence of rice chromosome I (Accession<br>
#AP001859; between positions 1372 and 2598 of the sequence in the database) and the<br>
oligonucleotide sequences are:<br>
Forward 5'-AACACAAGGGACAGCACATTGAGC-3' (SEQ ID No. 8)<br>
Reverse 5'-GAAAGAGGAGCTAGAGGTGGGTGC-3' (SEQ ID No. 9)<br>
This primer pair gives a PCR amplification product of 1.1 kb.<br>
Two other primer pairs were designed based on the sequence of the rice mitochondria!<br>
DNA (Accession #D21251) between positions 4981 and 5641 of the sequence for SEQ ID<br>
No. 10 and SEQ ID No. 11 and between positions 6660 and 7303 of the sequence for SEQ<br>
ID No. 12 and SEQ ID No. 13 in the database:<br>
(1) Forward 5'-GGGCAATTCCATCGTGCTATGAGC-3' (SEQ ID No. 10)<br>
Reverse 5'-GCGTTGGGTTTTCCAACGAAAAAC-3' (SEQ ID No. 11)<br>
This primer pair gives a PCR amplification product of 660 bp.<br>
(2) Forward 5'-CAGGCGAAGGTCATAATTCGCAGG-3' (SEQ ID No. 12)<br>
Reverse 5'-CGAAGAAGGCAGTCTTGCTTCCTC-3' (SEQ ID No. 13)<br>
This primer pair gives a PCR amplification product of 600 bp.<br>
Each of these three primer pairs was individually multiplexed in a PCR in combination<br>
with the Oligonucleotide primer pair indicated in Table 2 (SEQ ID No. 4 and S£Q TU No.<br>
5) using the following conditions: 50-100 ng of template DNA, IX PCR buffer (50 mM<br>
KCI, 10 mM Tns-HCl, pH 8.3, 1.5 mM MgCl2, 0.01% gelatin), 200|aM dNTP (each), 1<br>
unit of Taq polymerase, 5 pico moles each of primers from Table 2 (Seq ID No. 4 and SEQ<br>
ID No. 5) and 1 pico mole each of any of the primer pairs from Table 3. The PCR<br>
0 O<br>
conditions were: Initial denaturation at 95 C for 7 min followed by 35 cycles of 94 C for<br>
30 sec, 44 C for 1 min, 72 C for 2 min. A final extension was given at 72 C for 7 min and<br>
o<br>
then the samples were stored at 4 C until further use. PCR products were separated on a<br>
1% agarose gel and the DNA bands detected by staining with ethidium bromide and<br>
visualizing under ultraviolet light. As depicted in Figure 7, the 325 bp fragment amplified<br>
by primers SEQ ID No. 4 and SEQ ID No. 5 is obtained only when the template DNA is<br>
from the CMS line. Each of the primer pairs in Table 3 (SEQ ID No. 8 and 9, SFQ ID No.<br>
10 and 11, SEQ ID No. 12 and 13) amplified a specific DNA fragment irrespective of<br>
whether the template DNA is from the CMS line or the Maintainer line. Therefore, in this<br>
multiplex PCR assay, two DNA fragments are observed when the template is from the<br>
CMS line and only a single DNA fragment is obtained if the template DNA is from the<br>
Maintainer line. In Figure 7 (Lane 2 and 3), an example of a PCR performed with a single<br>
primer pair (SEQ ID No. 4 and 5) that is specific for CMS line and a multiplexed PCR<br>
performed with primers SEQ ID No. 4 and 5, and SEQ ID No. 8 and 9 (Lane 4 and 5) is<br>
shown.<br>
EXAMPLE 9<br>
A PCR assay for detecting impurities within CMS lines that have arisen due to cross<br>
pollination with rogue pollen donors.<br>
Genomic DNA is isolated as described in Example 1 (b) from 100 three day old seedlings<br>
obtained by germination of seeds of the CMS stock. PCR as described in Example 7 (with<br>
variation in cycling conditions as per the source of the primers) is performed by using<br>
oligonucleotide primers that target any one of a number of microsatellite markers that will<br>
detect polymorphism between the CMS line and the rogue pollen donor. After PCR, the<br>
product is separated on an agarose gel and detected by ethidium bromide staining. A single<br>
band will be observed if DNA is from the CMS line but if cross pollination has occurred<br>
from a rogue pollen donor, an extra band (contributed by the rogue pollen donor) will be<br>
observed apart from the CMS specific band.<br>
A PCR assay for detecting impurities within CMS lines that have arisen due to cross<br>
pollination with rogue pollen donors can also be performed on DNA isolated from a pool of<br>
100 seedlings obtained by germination of the seeds of the CMS stock. Grow 100 CMS rice<br>
seedlings up to 15 days and from each plant, remove two cm from the tip of the second leaf<br>
and then cut a one cm long leaf piece below this area. Pool the cut leaves from all the plants<br>
and isolate genomic DNA as described in Example 1 according to the protocol of Kochert<br>
et al (1989). Label 5' end of one of the primers (either forward or reverse) with fluorescent<br>
dye using standard protocols. Do PCR reactions with the Fluorescent labeled primers,<br>
generally as described in Example 7. The cycling conditions can vary according to the<br>
primer in use. Important to note, the template DNA used should be only 2-3 ng and<br>
AmpliTaq Gold (ABI, Foster City, USA) enzyme should be used. After PCR, take 1 |al of<br>
PCR product and mix with loading dye (Formamide and Blue dextron 5:1 ratio) and 1.5 jal<br>
marker (ABI, Foster City, USA). Heat denature and load 1.5 nl of the samnle in an<br>
automated DNA sequencer 377 (ABI, Foster City, USA). Run the gel till the bands are<br>
separated properly. Analyse the peak heights produced by the bands using the GeneScan<br>
analyser (ABI, Foster City, USA).<br>
EXAMPLE 10<br>
A PCR assay for detecting purity of rice hybrids.<br>
Genomic DNA from CMS line IR 58025 A, Restorer IR40750 and the hybrid (DRR HI) of<br>
these two parents, were isolated as described in Example 1 (b). PCR was performed using<br>
the microsatellite marker RM164 [Forward 5'-TCTTGCCCGTCACTGCAGATATCC-3'<br>
(SEQ ID No. 14), Reverse 5'-GCAGCCCTAATGCTACAATTCTTC-3' (SEQ ID No. 15)]<br>
(Wu and Tanksley, 1993. Mol. Gen. Genet. 241:225-235). A polymorphism between the<br>
two parents is observed at this locus. The PCR conditions were: DNA samples (50 ng) were<br>
amplified in 25 ml reaction volumes containing 1XPCR buffer [10 mM Tris.HCl (pH 8.3),<br>
50 mM KC1, 1.5 mM MgCl2, 0.01% gelatin], 0.2 mM of each dNTPs (Amersham<br>
Pharmacia Biotech, Sweden), 10 Pico moles of each primer and 1U of Taq polymerase.<br>
Initial denaturation at 95 C for 7 min followed by 35 cycles of 94 C for 1 min, 55 C for 1<br>
0 0 min, 72 C for 2 min. A final extension was given at 72 C for 7 min and then the samples<br>
were stored at 4 C until further use. The PCR product was separated on an agarose gel,<br>
stained with ethidium bromide and visualized under ultraviolet light. The result of the PCR<br>
performed using the SEQ ID No. 14 and SEQ ID No. 15 as primers is shown in Figure 8.<br>
True hybrids exhibited two DNA fragments, one fragment contributed by the Restorer<br>
parent and the other fragment contributed by the CMS parent. The off-types exhibits a<br>
single fragment which co-migrates with the DNA band of either of the parents (Figure 8).<br>
In Figure 8, the off-type co-migrates with the CMS DNA band which suggest that this<br>
particular hybrid seed lot is contaminated with the Maintainer line of IR 58025 A (CMS).<br>
The microsatellite marker used in this experiment is shown only as an example of the<br>
hundreds of microsatellite markers that can be used for detecting hybrid seed purity in rice.<br>
A PCR method that can detect purity of rice hybrids can also be done in pooled samples.<br>
Grow 400 rice seedlings obtained by germination of seeds belonging to the hybrid seed<br>
stock for up to 15 days and from each plant, remove 2 cm from the tip of the second leaf<br>
and then cut 1 cm leaf piece below this area. Pool the cut leaves from all the plants and<br>
isolate genomic DNA as described in Example 1 according to the protocol of Kochert et al<br>
(1989). Label 5' end of one of the primers (either forward or reverse) with fluorescent dye<br>
using standard protocols. Do PCR reactions with the Fluorescent labeled primers, generally<br>
as described in Example 7. The cycling conditions can vary according to the primer in<br>
use. Important to note, the template DNA used should be only 2-3 ng and AmpliTaq Gold<br>
(ABI, Foster City, USA) enzyme should be used. After PCR, take 1 (al of PCR product and<br>
mix with loading dye (Formamide and Blue dextron 5:1 ratio) and 1.5 ul marker (ABI,<br>
Foster City, USA). Heat denature and load 1.5 jal in an automated DNA sequencer 377<br>
(ABI, Foster City, USA). Run the gel till the bands are separated properly. Analyse the peak<br>
heights produced by the bands using the GeneScan analyser (ABI, Foster City, USA).<br>
By this assay, the extent of purity can be detected by measuring the height of each of the<br>
expected two peaks that are characteristic of the hybrid (each peak represents one allele that<br>
is contributed by one parent) that are detected by the instrument. Using genomic DNA<br>
isolated from a single hybrid plant as a template in the PCR assay, it is expected that the<br>
peak heights would be equal, leading to a ratio of 1:1 (or 50:50). Using genomic DNA<br>
isolated from a population of four hundred seedlings as a template in the PCR assay, any<br>
deviation from a ratio of 1 : 1 (50:50) for the heights of the two peaks would be an indicator<br>
of the extent of impurities in the hybrid seed stock. For eg., a ratio of peak heights of 1.02 :<br>
0.98 (51 : 49) would correspond to a sample of hybrid seed stock that is 98% pure and a<br>
ratio of peak heights of 1.1 : 0.9 (55 : 45) would correspond to a sample of hybrid seed<br>
stock that is 90% pure.<br>
In conclusion, it can be said that the invention provides a method for ensuring the purity of<br>
parental lines is an essential prerequisite for ensuring the purity of rice hybrids. The<br>
cytoplasmic male sterile lines that are used in the three line breeding system for hybrid rice<br>
production are often contaminated with seeds of the iso-nuclear maintainer lines. Herein is<br>
reported a DNA Sequence that is homologous to rice mitochondrial DNA but is unique to<br>
the WA cytoplasmic male sterile lines of rice. In a Polymerase Chain Reaction (PCR)<br>
using total genomic DNA as a template, oligonucleotide primers based on this said DNA<br>
sequence PCR amplify a fragment from cytoplasmic male sterile lines of rice but not from<br>
their cognate maintainer lines indicating that this PCR assay can be used to detect<br>
impurities of the maintainer lines within seed stocks of the CMS line. In a coded test on a<br>
mixed sample of plants containing both CMS and maintainer lines, this PCR assay was<br>
used to correctly predict the genotypes of these plants. In Southern hybridization analysis,<br>
the PCR amplified fragment obtained using these oligonucleotide primers detect a<br>
polymorphism between the CMS and maintainer lines. Also described is the application of<br>
co-dominant PCR assayable markers like Microsatellites and Sequence Tagged Site<br>
polymorphisms for detecting contaminating cross pollination during CMS line<br>
multiplication as well as for assessing the purity of rice hybrids is described.<br>
36<br>
Table 1 : Rice lines analysed in this study*<br>
(Table Removed)<br>
*A!1 rice lines used in this study were provided by the Directorate of Rice Research,<br>
Indian Council of Agricultural Research, Hyderabad, India.<br>
Table 2. Sequence speciflc oligonucleotide primers for a PCR assay to distinguish CMS and Maintainer lines or rice.<br>
(Table Removed)<br>
Table 3. Sequence of oligonacleottde primers that can be multiplexed with SEQ ID No.4 and SEQ ID No.5 in a PCR assay for<br>
the differentiation of CMS and Maintainer lines.<br>
Oligonucleotide<br>
Primer pair<br>
Locus being amplified Approximate Polymorphism<br>
size of between CMS<br>
amplified &amp; Maintainer<br>
fragment Jines<br>
(I) F 5 *-AACACAAGGGACAGCACATTGAGC-3'<br>
(SEQ No.8)<br>
R 5'-GAAAGAGGAGCTAGAGGTGGGTGC-3'<br>
(SEQ ID No.9)<br>
Rice Chromosome t 1178bp No<br>
(2) F 5'-GGGCAATTCCATCGTGCTATGAGC-3'<br>
(SEQ No. 10)<br>
R 5'-GCGTTGGGTTTTCCAACGAAAAAC-3'<br>
(SEQ No. 11)<br>
Rice mhochondrial 660 bp<br>
DNA (positions 4981<br>
to 5641 of Accession<br>
8D21251)<br>
(3) F S'-CAGGCGAAGGTCATAATTCGCAGGO'<br>
(SEQ ID No. 12)<br>
R 5'-CGAAGAAGGCAGTCTTGCTTCCTC-3'<br>
(SEQ ID No. 13)<br>
Rice mitochondrial 643 bp<br>
DNA (positions 6660 lo 7303<br>
of Accession #D21251)<br>
F = Forward primer, R = Reverse primer<br>
No<br>
No<br><br><br><br><br><br>
Amended claims:-<br>
1.	A method of distinguishing Male Sterile (CMS) lines of rice from their cognate Male Fertile Maintainer Lines comprising the steps of: (a) providing a pair of oligonucleotide primers, said pair comprising a first primer comprising SEQ ID No: 4 and a second primer comprising SEQ ID No: 5; (b) conducting a Polymerase Chain Reaction (PCR) assay on a rice sample using the oligonucleotide primer pair obtained in step 4 (a); and (c) detecting PCR-amplified fragment(s); wherein the presence of PCR-amplified fragments indicates the sample is from a CMS line, and the absence of PCR-amplified fragments indicates the sample is from the cognate Male Fertile Maintainer line.<br>
2.	The method of claim 1, wherein said Male Sterile Lines contain the WA (wild abortive) cytoplasm.<br>
3.	The method of claim 1, wherein detecting PCR amplified fragment(s) comprises incorporating at least one radioactively labeled nucleotide into the PCR amplified fragment(s) and detecting the radioactively labeled nucleotide(s).<br>
4.	The method of claim 1, wherein detecting PCR amplified fragment(s) comprises incorporating at least one non-radioactively labeled nucleotide into the PCR amplified fragment(s) and detecting the non-radioactively labeled nucleotide by colorimetry, chemiluminescence or measurement of fluorescence.<br>
5.	The method of claim 1, wherein it is useful in assessing seed purity of cytoplasmic male sterile lines of rice.<br>
6.	The method of claim 1, wherein several oligonucleotide primer pairs are multiplexed in a PCR assay with the first pair of oligonucleotide primers such as herein described.<br>
7.	The method of claim 1, wherein the PCR-ELISA comprises one or more of: using a labeled capture probe or labeled PCR primer that can be bound to suitably coated solid surface made of polystyrene, styrene or glass; using a non-radioactively labeled nucleotides in the PCR and subsequently detecting with anti-label antibodies that are conjugated to enzymes used for ELISA; and modifying the PCR-ELISA format by using an alternate method for labeling the PCR amplified fragment,<br><br>
attaching probe to  solid  surfaces  or detecting the  PCR amplified fragment.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NDktZGVsbnAtMjAwMy1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">01449-delnp-2003-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NDktZGVsbnAtMjAwMy1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">01449-delnp-2003-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NDktZGVsbnAtMjAwMy1jb3JyZXNwb25kZW5jZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">01449-delnp-2003-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NDktZGVsbnAtMjAwMy1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">01449-delnp-2003-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NDktZGVsbnAtMjAwMy1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">01449-delnp-2003-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NDktZGVsbnAtMjAwMy1mb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">01449-delnp-2003-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NDktZGVsbnAtMjAwMy1mb3JtLTE4LnBkZg==" target="_blank" style="word-wrap:break-word;">01449-delnp-2003-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NDktZGVsbnAtMjAwMy1mb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">01449-delnp-2003-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NDktZGVsbnAtMjAwMy1mb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">01449-delnp-2003-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ0OS1ERUxOUC0yMDAzLUFic3RyYWN0LSgwNC0wMi0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">1449-DELNP-2003-Abstract-(04-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ0OS1ERUxOUC0yMDAzLUNsYWltcy0oMDQtMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1449-DELNP-2003-Claims-(04-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ0OS1ERUxOUC0yMDAzLUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDQtMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1449-DELNP-2003-Correspondence-Others-(04-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ0OS1ERUxOUC0yMDAzLUZvcm0tMS0oMDQtMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1449-DELNP-2003-Form-1-(04-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ0OS1ERUxOUC0yMDAzLU90aGVycy1Eb2N1bWVudC0oMDQtMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1449-DELNP-2003-Others-Document-(04-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ0OS1ERUxOUC0yMDAzLVBDVC0yMTAtKDA0LTAyLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">1449-DELNP-2003-PCT-210-(04-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ0OS1ERUxOUC0yMDAzLVBDVC00MDktKDA0LTAyLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">1449-DELNP-2003-PCT-409-(04-02-2009).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="228683-diazepinone-compounds-as-chk-1-protein-kinase-inhibitor.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="228685-optical-fiber-comprising-an-internal-glass-portion.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>228684</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>01449/DELNP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>08/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>20-Feb-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>06-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>09-Nov-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>COUNCIL OF SCIENTIFIC &amp; INDUSTRIAL RESEARCH</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>ANUSANDHAN BHAWAN, RAFI MARG, NEW DELHI-110 001, INDIA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>YOSHITOLA JAMIR</td>
											<td>INDIA.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>RAMESH VENKATA SONTI</td>
											<td>INDIA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/09</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IN01/00048</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-03-26</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>PCT/IN01/00048</td>
									<td>2001-03-26</td>
								    <td>India</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/228684-method-of-using-dna-markers-for-distinguishing-male-sterile-cms-lines-of-rice-from-their-cognate-male-fertile-maintainer-lines by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:17:57 GMT -->
</html>
