<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/228756-a-method-for-reducing-the-complexite-of-a-sample-that-comprises-a-mixture-of-proteins by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:22:39 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 228756:A METHOD FOR REDUCING THE COMPLEXITE OF A SAMPLE THAT COMPRISES A MIXTURE OF PROTEINS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR REDUCING THE COMPLEXITE OF A SAMPLE THAT COMPRISES A MIXTURE OF PROTEINS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates generally to methods for reducing the complexity of a sample. More specifically, the present invention relates to proteomics, the measurement of the protein levels in biological samples, and analysis of proteins in a sample using antibodies that recognize small epitopes.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>A METHOD FOR REDUCING THE COMPLEXITY OF A SAMPLE THAT<br>
COMPRISES A MIXTURE OF PROTEINS<br>
CROSS-REFERENCE TO RELATED APPLICATIONS<br>
[0001] This application claims the benefit of U.S. Provisional Application Nos. 60/496,154,<br>
filed on August 18, 2003, and 60/511,720, filed on October 15, 2003, which are hereby<br>
incorporated by reference in their entireties.<br>
FIELD OF THE INVENTION<br>
[0002] The present invention relates generally to methods for reducing the complexity of a<br>
sample. More specifically, the present invention relates to proteomics, the measurement of the<br>
protein levels in biological samples, and analysis of proteins in a sample using antibodies that<br>
recognize small epitopes.<br>
BACKGROUND OF THE INVENTION<br>
[0003] Proteomics offers a more direct look at the biological functions of a cell or organism<br>
than does genomics, the traditional focus for evaluation of gene activity. Proteomics involves the<br>
qualitative and quantitative measurement of gene activity by detecting and quantitating<br>
expression at the protein level, rather than at the messenger RNA level. Proteomics also involves<br>
the study of non-genome encoded events including the post-translational modification of<br>
proteins, protein degradation and protein byproducts, interactions between proteins, and the<br>
location of proteins within the cell. The structure, function, or level of activity of the proteins<br>
expressed by a cell are also of interest.<br>
[0004] The study of gene expression at the protein level is important because many of the<br>
most important cellular processes are regulated by the protein status of the cell, not by the status<br>
of gene expression. Also, the protein content of a cell is highly relevant to drug discovery efforts<br>
since most drugs are designed to be active against protein targets.<br><br>
[0005] Current technologies for the analysis of protein mixtures, such as the intracellular<br>
proteins of a cell or population of cells and the proteins secreted by the cell or population of cells<br>
or biological fluids, are based on a variety of protein separation techniques followed by<br>
identification and/or analysis of the separated proteins. The most popular method is based on<br>
2D-gel electrophoresis followed by "in-gel" proteolytic digestion and mass spectroscopy.<br>
Alternatively, Edman and related methods may be used for the sequencing. This 2D-gel<br>
technique requires large sample sizes, is time consuming, and is currently limited in its ability to<br>
reproducibly resolve a significant fraction of the proteins expressed by a human cell. Techniques<br>
involving some large-format 2D-gels can produce gels which separate a larger number of<br>
proteins than traditional 2D-gel techniques, but reproducibility is still poor and over 95% of the<br>
spots cannot be sequenced due to limitations with respect to sensitivity of the available<br>
sequencing techniques. The electrophoretic techniques are also plagued by a bias towards<br>
proteins of high abundance.<br>
[0006] Thus, there is a need for the ability to assay more completely proteins expressed by a<br>
cell or a population of cells hi an organism or in a fluid comprising protein (such as serum,<br>
plasma, lymph, and other biological fluids), including up to the total set of proteins expressed by<br>
the cell or cells or found in the fluid comprising protein.<br>
BRIEF SUMMARY OF THE INVENTION<br>
[0007] In one aspect, the invention provides methods for reducing the complexity of a<br>
sample, said methods comprising: (a) contacting a sample with one or more small epitope<br>
antibody under conditions that permit binding; and (b) separating an antibody-protein complex,<br>
whereby proteins comprising one or more epitope(s) bound by the one or more small epitope<br>
antibody are isolated, separated, enriched and/or purified.<br>
[0008] In another aspect, the invention provides methods comprising (a) contacting a sample<br>
with one or more small epitope antibody under conditions that permit binding; (b) separating an<br>
antibody-protein complex, whereby proteins comprising one or more epitope(s) bound by the<br>
one or more small epitope antibody are isolated, separated, enriched and/or purified; and (c)<br>
separating proteins from the antibody-protein complex.<br><br>
[0009] In another aspect, the invention provides methods for reducing the complexity of a<br>
sample, said methods comprising: separating a small epitope antibody-protein complex, whereby<br>
proteins comprising an epitope bound by the small epitope antibody are enriched; wherein the<br>
complex was generated by contacting a sample with the small epitope antibody.<br>
[0010] In another aspect, the invention provides methods for reducing the complexity of a<br>
sample, said methods comprising: (a) separating a small epitope antibody-protein complex,<br>
whereby proteins comprising an epitope bound by the small epitope antibody are enriched;<br>
wherein the complex was generated by contacting a sample with the small epitope antibody; and<br>
(b) separating proteins from the antibody-protein complex.<br>
[0011] In another aspect, the invention provides methods for reducing the complexity of a<br>
sample, said methods comprising separating protein from a small epitope antibody-protein<br>
complex, whereby protein comprising an epitope bound by the small epitope antibody is<br>
enriched; wherein the small epitope antibody-protein complex is generated by (a) contacting a<br>
sample with the small epitope antibody under conditions that permit binding, whereby the small<br>
epitope antibody-protein complex is generated; and (b) separating an antibody-proteincomplex.<br>
[0012] As is evident, one or more steps may be combined and/or performed sequentially<br>
(often in any order, as long as the requisite product(s) are able to be formed), and, as is evident,<br>
the invention includes various combinations of the steps described herein. It is also evident, and<br>
is described herein, that the invention encompasses methods in which the initial, or first, step is<br>
any of the steps described herein. Methods of the invention encompass embodiments in which<br>
later, "downstream" steps are an initial step.<br>
[0013] In some embodiments, the methods further comprise a step of treating the sample<br>
with a protein cleaving agent, whereby polypeptide fragments are generated. In embodiments<br>
involving a step of separating protein from the antibody-protein complex, the sample can be<br>
treated with a protein cleaving agent prior to a step of contacting a sample with the at least one<br>
small epitope antibody, and/or following a step of separating protein from the antibody-protein<br>
complex. Methods for treatment with protein cleaving agents are well known in the art and<br>
described herein. One or more protein cleaving agent may be used. The protein cleaving agent<br>
may be an enzyme (such as chymotrypsin or trypsin) or a chemical agent (such as cyanogen<br>
bromide).<br>
[0014] Thus, in another aspect, the invention provides methods for reducing the complexity<br>
of a sample, said methods comprising (a) contacting a sample with one or more small epitope<br><br>
antibody under conditions that permit binding; (b) separating an antibody-protein complex,<br>
whereby proteins comprising one or more epitope(s) bound by the one or more small epitope<br>
antibody are enriched; (c) separating protein from protein-antibody complex; and (d) treating the<br>
protein with a protein cleaving agent, whereby polypeptide fragments are generated.<br>
[0015] In another aspect, the invention provides methods for reducing the complexity of a<br>
sample, said methods comprising (a) contacting a sample with one or more small epitope<br>
antibody under conditions that permit binding, to form an antibody-protein complex; and (b)<br>
treating the antibody-protein complex with a protein cleaving agent to produce polypeptide<br>
fragments.<br>
[0016] In another aspect, the invention provides methods for reducing the complexity of a<br>
protein sample, said methods comprising: (a) treating the sample with a protein cleaving agent,<br>
whereby polypeptide fragments are generated; (b) contacting the polypeptide fragments with one<br>
or more small epitope antibody under conditions that permit binding, whereby antibody-<br>
polypeptide complexes are generated; and (c) separating the antibody-polypeptide complex,<br>
whereby polypeptides comprising one or more epitope bound by the one or more small epitope<br>
antibody are enriched.<br>
[0017] In another aspect, the invention provides methods for reducing the complexity of a<br>
sample, said method comprising: (a) incubating a reaction mixture, said reaction mixture<br>
comprising: (i) a small epitope antibody; and (ii) a sample, wherein incubating is under<br>
conditions permitting binding; and (b) separating an antibody-protein complex, whereby protein<br>
is enriched.<br>
[0018] In another aspect, the invention provides methods for reducing the complexity of a<br>
sample, said method comprising: separating an antibody-protein complex, whereby protein is<br>
enriched; wherein the antibody-protein complex is generated by incubating a reaction mixture,<br>
said reaction mixture comprising: (a) a small epitope antibody; and (b) a sample, wherein<br>
incubating is under conditions permitting binding.<br>
[0019] In another aspect, the invention provides methods for reducing the complexity of a<br>
sample, said method comprising: (a) incubating a reaction mixture, said reaction mixture<br>
comprising: (i) a small epitope antibody; and (ii) a sample, wherein incubating is under<br>
conditions permitting binding; (b) separating an antibody-protein complex; and (c) separating<br>
protein from the protein-antibody complex, whereby protein is enriched.<br><br>
[0020] In another aspect, the invention provides separating protein from a separated protein-<br>
antibody complex, wherein the protein-antibody complex is generated by incubating a reaction<br>
mixture, said reaction mixture comprising: (a) a small epitope antibody; and (b) a sample,<br>
wherein incubating is under conditions permitting binding; and separation of a protein-antibody<br>
complex.<br>
[0021] In another aspect, the invention provides a method for reducing the complexity of a<br>
sample that comprises a mixture of proteins, comprising separating a small epitope antibody-<br>
protein complex, wherein proteins comprising an epitope bound by the small epitope antibody<br>
are enriched. In some embodiments, the method further comprises separating protein from the<br>
antibody-protein complex. In some embodiments, the small epitope antibody binds an epitope<br>
consisting of about 3 to about 5 amino acids. In some embodiments, the sample is contacted<br>
with a plurality of small epitope antibodies to form a plurality of small epitope antibody-protein<br>
complexes. In some embodiments, the small epitope antibodies are detectably labeled. In some<br>
embodiments, a plurality of small epitope antibodies is immobilized on a solid matrix. In some<br>
embodiments, the sample is contacted with a plurality of small epitope antibodies in parallel. In<br>
some embodiments, the sample is contacted with a plurality of small epitope antibodies serially.<br>
In some embodiments, the sample is contacted with at least 100 small epitope antibodies. In<br>
some embodiments, the method further comprises contacting protein separated from the<br>
antibody-protein complex with a protein cleaving agent to form polypeptide fragments. In some<br>
embodiments, the method further comprises contacting the small epitope antibody-protein<br>
complex with a protein cleaving agent to form polypeptide fragments. In some embodiments,<br>
the method further comprises contacting the sample with a protein cleaving agent to form<br>
polypeptide fragments prior to formation of the small epitope antibody-protein complex,<br>
optionally further comprising separating polypeptide fragments from the small epitope antibody-<br>
protein complex. In one embodiment, the protein cleaving agent comprises a protease. In<br>
another embodiment, the protein cleaving agent comprises a chemical agent.<br>
[0022] In another aspect, the invention provides a method for reducing the complexity of a<br>
sample that comprises a mixture of proteins, comprising (a) contacting the sample with at least<br>
one small epitope antibody to form an antibody-protein complex; and (b) separating the<br>
antibody-protein complex from unbound protein in the sample. In some embodiments, steps (a)<br>
and (b) are performed sequentially. In some embodiments, steps (a) and (b) are performed<br>
simultaneously. In some embodiments, the method further comprises separating protein from the<br><br>
antibody-protein complex. In some embodiments, the small epitope antibody binds an epitope<br>
consisting of about 3 to about 5 amino acids. In some embodiments, the at least one small<br>
epitope antibody comprises at least about 100 small epitope antibodies.<br>
[0023] In other aspects, the invention provides small epitope antibody-protein complexes,<br>
proteins, and/or polypeptide fragments prepared using methods for reducing the complexity of a<br>
sample described herein.<br>
[0024] As is evident to one skilled in the art, aspects that refer to combining and incubating<br>
the resultant mixture also encompass method embodiments which comprise incubating the<br>
various mixtures (in various combinations and/or subcombinations) so that the desired products<br>
are formed.<br>
[0025] One, or more than one (such as about two, about three, about four, about five, about<br>
ten, about twenty or more) small epitope antibod(ies) may be used in the methods of the<br>
invention. In some embodiments, the sample is contacted with about 20, about 30, about 40,<br>
about 50, about 75, about 100, about 125, about 150, about 200, about 300, about 400, about 500,<br>
about 1000, or more small epitope antibodies. In some embodiments, the sample is contacted<br>
with at least about 20, about 30, about 40, about 50, about 75, about 100, about 125, about 150,<br>
about 200, about 300, about 400, about 500, about 1000, or more small epitope antibodies. In<br>
some embodiments, the sample is contacted with less than about 100, about 95, about 90, about<br>
85, about 80, about 75, about 70, about 65, about 60, about 55, about 50, about 45, about 40,<br>
about 35, about 30, about 25, about 20, about 15, about 10, or fewer small epitope antibodies. In<br>
some embodiments, the sample is contacted with at least about any of 10,20,30,40, 50,75,100,<br>
125,150,200, 300,400 or 500 small epitope antibodies, with an upper limit of about any of 20,<br>
30,40,50,75,100,125,150,200,300,400, 500, or 1000 small epitope antibodies.<br>
[0026] The invention also provides methods using the protein prepared using any of the<br>
methods described herein, for example, methods for characterizing a protein, methods of<br>
expression profiling, methods of identifying proteins; methods for identifying protein<br>
degradation products; methods for identifying change in post-translational modification, and<br>
methods for determining the mass, the amount and/or identity of protein(s) in a sample. Methods<br>
of genotyping (protein mutation detection), identifying splice variants, determining the presence<br>
or absence of a protein of interest, expression profiling; methods for identifying protein<br>
degradation products; methods for identifying change in post-translational modification, and<br>
protein discovery are also encompassed by the methods of the invention.<br><br>
[0027] Thus, in one aspect, the invention provides methods for characterizing a protein<br>
comprising: (a) reducing the complexity of a sample using any of the methods described herein,<br>
whereby proteins are enriched and/or purified; and (b) analyzing the proteins (interchangeably<br>
termed "products").<br>
[0028] In another aspect, the invention provides methods for characterizing protein<br>
comprising analyzing protein; wherein the protein was prepared using any of the methods<br>
described herein.<br>
[0029] In some embodiments, the step of analyzing comprises determining amount of said<br>
proteins, whereby the amount of protein(s) prepared, enriched and/or separated is quantified. In<br>
some embodiments, the step of analyzing comprises identifying one or more of said proteins. In<br>
some embodiments, the identity of the epitope(s) to which the small epitope antibody(ies) bind is<br>
used to assist identification of the enriched proteins. In some embodiments, a protein is<br>
identified using any one or more of the following characteristics: sequence; mass; m/z ratio (in<br>
embodiments involving mass spectrometric analysis), and/or amino acid composition. In other<br>
embodiments, the step of analyzing comprises determining the mass of one or more protein(s).<br>
In some embodiments, the step of analyzing includes analysis for the detection of any alterations<br>
in the protein, as compared to a reference protein which is identical (at least in part) to the<br>
protein sequence other than the sequence alteration. Sequence alterations include mutations<br>
(such as deletion, substitution, insertion and/or transversion of one or more amino acid), splice<br>
variants, degradation products, and change in glycosylation.<br>
[0030] In another aspect, the invention provides methods for characterizing a protein using<br>
mass spectrometry, comprising: (a) reducing the complexity of a sample using any of the<br>
methods described herein, whereby proteins are enriched and/or purified; and (b) analyzing the<br>
proteins (interchangeably termed "products") which are isolated, purified, prepared and/or<br>
separated using any of the methods herein, wherein the analyzing is by mass spectrometry.<br>
[0031] In another aspect, the invention provides methods for characterizing protein<br>
comprising analyzing protein using mass spectrometry; wherein the protein was prepared using<br>
any of the methods described herein; wherein the analyzing is by mass spectrometry. In some<br>
embodiments, quantity, mass, and/or identity of a protein is determined. In some embodiments,<br>
the methods further comprise use of epitope identity information.<br>
[0032] In some embodiments, mass spectrometric is matrix assisted laser<br>
desorption/ionization ("MALDI") mass spectrometry; surface-enhanced laser<br><br>
desorption/ionization ("SELDI"); and/or tandem mass spectrometry (e.g., MS/MS, MS/MS/MS,<br>
ESI-MS/MS).<br>
[0033] In another aspect, the invention provides methods for determining the identity of a<br>
protein in a sample using mass spectrometry, said methods comprising: (a) reducing the<br>
complexity of a sample using any of the methods described herein, whereby proteins are<br>
enriched and/or purified; (b) analyzing the proteins (interchangeably termed "products"), wherein<br>
the analyzing is by mass spectrometry; and (c) determining identity of enriched protein. In some<br>
embodiments, the methods further comprise use of epitope identity information.<br>
[0034] In another aspect, the invention provides methods for determining the identity of a<br>
protein in a sample using mass spectrometry, said methods comprising determining the identity<br>
of protein using mass spectrometry; wherein the protein is prepared using any of the methods for<br>
reducing complexity of a sample described herein. In some embodiments, the methods further<br>
comprise use of epitope identity information.<br>
[0035] In another aspect, the invention provides methods for protein expression profiling,<br>
wherein in the level of expression of one or more proteins is determined, wherein the protein is<br>
prepared using any of the methods for reducing complexity of a sample described herein. In<br>
some embodiments, the level of expression is determined using mass spectrometry. In some<br>
embodiments, the invention provides methods for comparing the amounts of proteins in two or<br>
more samples.<br>
[0036] In another aspect, the invention provides methods for protein expression profiling,<br>
wherein in the identity of one or more proteins is determined, wherein the protein is prepared<br>
using any of the methods for reducing complexity of a sample described herein. In some<br>
embodiments, protein identity is determined using mass spectrometry. In some embodiments,<br>
the methods further comprise use of epitope identity information. In some embodiments, the<br>
invention provides methods for comparing the identity of protein(s) in two or more samples.<br>
[0037] In another aspect, the invention provides a method for determining the presence or<br>
absence of a protein of interest in a sample, wherein the method comprises detecting the protein<br>
of interest, of any, in an enriched protein fraction, wherein the enriched protein fraction is<br>
prepared using any of the methods for reducing the complexity of a sample described herein, and<br>
wherein detection of the protein of interest indicates presence of the protein in the sample. In<br>
one embodiment, detection comprises mass spectrometry.<br><br>
[0038] In another aspect, the invention provides a method for determining the amount of a<br>
protein of interest in a sample, wherein the method comprises quantifying the amount of the<br>
protein of interest in an enriched protein fraction, wherein the enriched protein fraction is<br>
prepared using any of the methods for reducing the complexity of a sample described herein. In<br>
one embodiment, quantification of the protein of interest comprises mass spectrometry.<br>
[0039] In another aspect, the invention provides a method for identifying the protein in a<br>
small epitope antibody-protein complex, wherein the small epitope antibody-protein complex is<br>
prepared using any of the methods for reducing the complexity of a sample described herein. In<br>
one embodiment, the identification comprises mass spectrometry.<br>
[0040] In another aspect, the invention provides a method for identification of a biomarker,<br>
wherein the method comprises comparing the proteins in two or more enriched protein fractions,<br>
wherein each of the two or more enriched protein fractions is prepared from a sample using any<br>
of the methods for reducing the complexity of a sample described herein. In some embodiments,<br>
the two or more samples comprise samples from at least one individual who has a disease<br>
condition and at least one individual who does not have the disease condition, and presence or<br>
absence of the biomarker is indicative of the disease condition. In one embodiment, the<br>
invention provides a method for determining presence or absence of a disease condition in an<br>
individual, comprising determining the level of a biomarker in a sample from the individual,<br>
wherein the biomarker is identified as described herein, and wherein the level of the biomarker is<br>
indicative of the presence or absence of the disease condition. In some embodiments, the two or<br>
more samples comprise samples from at least one individual who has received treatment for a<br>
disease condition and at least one individual who has not received treatment for the disease<br>
condition, and presence or absence of the biomarker is indicative of efficacy of the treatment. In<br>
one embodiment, the invention provides a method for determining efficacy of treatment for a<br>
disease condition in an individual, comprising detennining the level of a biomarker in a sample<br>
from the individual, wherein the biomarker is identified as described herein, and wherein the<br>
level of the biomarker is indicative of the efficacy of treatment. In some embodiments, the two<br>
or more samples comprise samples from at least one individual who has been exposed to a toxin<br>
or pathogen and at least one individual who has not been exposed to the toxin or pathogen, and<br>
presence or absence of the biomarker is indicative of exposure of an individual to the toxin or<br>
pathogen. In one embodiment, the invention provides a method for determining exposure of an<br>
individual to a toxin or pathogen, comprising detennining the level of a biomarker in a sample<br><br>
from the individual, wherein the biomarker is identified as described herein, and wherein the<br>
level of the biomarker is indicative of exposure to the toxin or pathogen.<br>
[0041] In another aspect, the invention provides compositions and kits comprising one or<br>
more small epitope antibodies for use in any of the methods of the invention.<br>
[0042] In some embodiments, the invention provides a composition comprising a plurality of<br>
small epitope antibodies. In some embodiments, the plurality of small epitope antibodies binds<br>
epitopes consisting of about 3 to about 5 amino acids. In some embodiments, the small epitope<br>
antibodies are detectably labeled. In some embodiments, the plurality of small epitope<br>
antibodies comprises at least about 100 small epitope antibodies.<br>
[0043] In some embodiments, the invention provides a kit comprising a plurality of small<br>
epitope antibodies. In some embodiments, the plurality of small epitope antibodies binds<br>
epitopes consisting of about 3 to about 5 amino acids. In some embodiments, the small epitope<br>
antibodies are detectably labeled. In some embodiments, the plurality of small epitope<br>
antibodies comprises at least about 100 small epitope antibodies.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
[0044] FIGURE 1 shows the reaction pattern using mapping polypeptides spanning<br>
sequences of immunization polypeptides for group 2 and group 5 mice, respectively.<br>
[0045] FIGURE 2 shows the results of a secondary screen of positive antibodies in a phage<br>
ELISA, as described in Example 2.<br>
[0046] FIGURE 3 shows an SPR trace of a single chain antibody derived from phage<br>
L50P1_15 against peptides 1, 6, 7, 8, and 9, as described in Example 2.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
[0047] The invention provides methods using one or more antibodies that bind (generally,<br>
specifically bind) small epitopes, termed "small epitope antibodies", to fractionate a protein<br>
mixture based on the presence and/or quantity of small epitopes within protein in the protein<br>
mixture, whereby protein(s) comprising the small epitope are isolated, separated, prepared,<br>
purified and/or enriched. Use of the methods of the invention thereby provides a means for<br>
reducing the complexity of a protein mixture, facilitating subsequent use and/or characterization<br><br>
of the enriched protein components of the sample. Insofar as the small epitope bound by the<br>
antibody is known, binding by a small epitope antibody provides information relating to amino<br>
acid content of protein(s) bound by the small epitope antibody. Small epitope antibodies are<br>
further described herein.<br>
[0048] As a general overview, the methods comprise: (a) contacting a sample with at least<br>
one small epitope antibody under conditions that permit binding; and (b) separating an antibody-<br>
protein complex from proteins that are not bound by the small epitope antibody(ies). Generally,<br>
proteins comprising one or more epitope bound by the at least one small epitope antibody are<br>
isolated, separated, enriched and/or purified. In some embodiments, the methods further<br>
comprise: step (c) of separating protein from the antibody-protein complex. In some<br>
embodiments, the methods further comprise a step of treating the sample with a protein cleaving<br>
agent prior to step (a) of contacting a sample with the at least one small epitope antibody, or, in<br>
embodiments involving separation of protein from the antibody-protein complex, after step (c) of<br>
separating protein from the antibody-protein complex.<br>
[0049] The methods of the invention are useful for fractionating samples comprising protein,<br>
which is accomplished by the use of antibodies (termed "small epitope antibodies") that<br>
recognize epitopes that are present in a multiplicity of proteins (such as, for example, an epitope<br>
consisting of or consisting essentially of 3 linear amino acids, 4 linear amino acids, or 5 linear<br>
amino acids). Small epitope antibodies suitable for use in the methods of the invention are<br>
extensively described herein and exemplified in the Examples. By virtue of the specificity of the<br>
small epitope antibodies, proteins (e.g., polypeptides) are separated, enriched and/or purified<br>
depending on the presence and/or amount of the small epitope within the protein that is<br>
recognized by the small epitope antibody(ies) used in the methods of the invention. Methods<br>
using the protein prepared using the methods of the invention are further described herein. As is<br>
evident, "reducing the complexity of a sample", as used herein, encompasses isolating, purifying,<br>
separating, enriching and/or purifying proteins (e.g., polypeptides) from a sample. Accordingly,<br>
the invention provides methods for purifying and/or enriching protein, methods for isolating<br>
protein, methods for separating protein, methods for preparing protein for characterization,<br>
methods for preparing protein for mass spectrometry analysis, methods for identifying protein<br>
(such as one or a group of proteins), methods for discovering new protein, and methods for<br>
quantification of protein in a sample.<br><br>
[0050] In one aspect, the invention provides methods for reducing the complexity of a<br>
sample, said methods comprising: (a) contacting a sample with one or more small epitope<br>
antibody under conditions that permit binding; and (b) separating an antibody-protein complex,<br>
whereby proteins comprising one or more epitope(s) bound by the one or more small epitope<br>
antibody are isolated, separated, enriched and/or purified.<br>
[0051] In another aspect, the invention further provides methods for purifying and/or<br>
enriching protein; isolating protein; separating protein, preparing protein for characterization;<br>
preparing protein for mass spectrometry analysis; identifying protein (such as one or more<br>
protein, or a group of proteins); discovering a new protein; and/or quantification of protein in a<br>
sample, wherein said methods comprising: (a) contacting a sample with one or more small<br>
epitope antibody under conditions that permit binding; and (b) separating an antibody-protein<br>
complex.<br>
[0052] In another aspect, the invention also encompasses methods using the protein prepared<br>
using any of the methods of the invention, for example, for characterizing a protein, methods of<br>
expression profiling, methods of identifying proteins; methods for identifying protein<br>
degradation products; methods for identifying change in post-translational modification, and<br>
methods for detemiining the mass, the amount and/or identity of protein(s) in a sample. For<br>
example, these methods can be applied in such areas as protein discovery, expression profiling,<br>
drug discovery and diagnostics.<br>
[0053] In another embodiment, mass spectrometry is used to characterize the protein<br>
prepared using any of the methods of the invention. The protein fraction generated using a small<br>
epitope antibody is particularly amenable to analysis using mass spectrometry because the<br>
number of proteins (including protein variants) is reduced (as compared with the starting sample)<br>
by use of the small epitope antibodies described herein. Insofar as the epitope present within the<br>
protein is identified, the amino acid sequence or content of the epitope (termed "epitope<br>
sequence" or "epitope amino acid content") provides further information useful for<br>
characterizing and identifying the protein. Mass spectrometry methods have been used to<br>
quantify and/or identify proteins. In some embodiments, mass spectrometry analysis generates a<br>
polypeptide mass map. Using these results, polypeptide mass mapping may permit identification<br>
of the corresponding protein. In other embodiments, mass spectrometry analysis is by tandem<br>
mass spectrometer, and generates specific sequence information. Use of this information may<br>
result in identification of the corresponding protein at the sequence level. In some embodiments,<br><br>
protein is identified using a method comprising MS analysis of protein prepared using any of the<br>
methods of the invention, in combination with epitope sequence or amino acid content<br>
information.<br>
[0054] One or more than one (such as about 2, about 5, about 7, about 10, about 20, about<br>
30, about 50, about 100, or more) small epitope antibodies may be used in the methods of the<br>
invention. In some embodiments, the sample is contacted with about 20, about 30, about 40,<br>
about 50, about 75, about 100, about 125, about 150, about 200, about 300, about 400, about 500,<br>
about 1000, or more small epitope antibodies. In some embodiments, the sample is contacted<br>
with at least about 20, about 30, about 40, about 50, about 75, about 100, about 125, about 150,<br>
about 200, about 300, about 400, about 500, about 1000, or more small epitope antibodies. In<br>
some embodiments, the sample is contacted with less than about 100, about 95, about 90, about<br>
85, about 80, about 75, about 70, about 65, about 60, about 55, about 50, about 45, about 40,<br>
about 35, about 30, about 25, about 20, about 15, about 10, or fewer small epitope antibodies. In<br>
some embodiments, the sample is contacted with at least about any of 10,20, 30,40, 50, 75,100,<br>
125,150,200,300,400 or 500 small epitope antibodies, with an upper limit of about any of 20,<br>
30, 40, 50, 75,100,125,150,200, 300,400,500, or 1000 small epitope antibodies.<br>
[0055] In another aspect, the invention provides compositions and kits comprising one or<br>
more small epitope antibody for use in any of the methods of the invention. In some<br>
embodiments, the kits further comprise instructions for any of the methods described herein.<br>
General Techniques<br>
[0056] The practice of the present invention will employ, unless otherwise indicated,<br>
conventional techniques of molecular biology (including recombinant techniques), microbiology,<br>
cell biology, biochemistry and immunology, which are within the skill of the art. Such<br>
techniques are explained fully in the literature, such as, Molecular Cloning: A Laboratory<br>
Manual, second edition (Sambrook et al., 1989) Cold Spring Harbor Press; Oligonucleotide<br>
Synthesis (M.J. Gait, ed., 1984); Methods in Molecular Biology, Humana Press; Cell Biology: A<br>
Laboratory Notebook (J.E. Cellis, ed., 1998) Academic Press; Animal Cell Culture (R.I.<br>
Freshney, ed., 1987); Introduction to Cell and Tissue Culture ( J.P. Mather and P.E. Roberts,<br>
1998) Plenum Press; Cell and Tissue Culture: Laboratory Procedures (A. Doyle, J.B. Griffiths,<br>
and D.G. Newell, eds., 1993-8) J. Wiley and Sons; Methods in Enzymology (Academic Press,<br><br>
Inc.); Handbook of Experimental Immunology (D.M. Weir and C.C. Blackwell, eds.,)/ Gene<br>
Transfer Vectors for Mammalian Cells (J.M. Miller and M.P. Calos, eds., 1987); Current<br>
Protocols in Molecular Biology (F.M. Ausubel et al., eds., 1987); PCR: The Polymerase Chain<br>
Reaction (Mullis et al., eds., 1994); Current Protocols in Immunology (J.E. Coligan et al., eds.,<br>
1991); Short Protocols in Molecular Biology (Wiley and Sons, 1999); Immunobiology (C.A.<br>
Janeway and P. Travers, 1997); Antibodies (P. Finch, 1997); Antibodies: a practical approach<br>
(D. Catty., ed., IRL Press, 1988-1989); Monoclonal antibodies : a practical approach (P.<br>
Shepherd and C. Dean, eds., Oxford University Press, 2000); Using antibodies: a laboratory<br>
manual (E. Harlow and D. Lane (Cold Spring Harbor Laboratory Press, 1999); and The<br>
Antibodies (M. Zanetti and J.D. Capra, eds., Harwood Academic Publishers, 1995).<br>
Definitions<br>
[0057] An "antibody" is an immunoglobulin molecule capable of specific binding to a target,<br>
such as a carbohydrate, polynucleotide, lipid, polypeptide, etc., through at least one antigen<br>
recognition site, located in the variable region of the immunoglobulin molecule. As used herein,<br>
the term encompasses not only intact polyclonal or monoclonal antibodies, but also fragments<br>
thereof (such as Fab, Fab', F(ab')2, Fv), single chain (ScFv), mutants thereof, fusion proteins<br>
comprising an antibody portion, and any other modified configuration of the immunoglobulin<br>
molecule that comprises an antigen recognition site of the required specificity. An antibody<br>
includes an antibody of any class, such as IgG, IgA, or IgM (or sub-class thereof), and the<br>
antibody need not be of any particular class. Depending on the antibody amino acid sequence of<br>
the constant domain of its heavy chains, immunoglobulins can be assigned to different classes.<br>
There are five major classes of immunoglobulins: IgA, IgD, IgE, IgG, and IgM, and several of<br>
these may be further divided into subclasses (isotypes), e.g., IgGl, IgG2, IgG3, IgG4, IgAl and<br>
IgA2. The heavy-chain constant domains that correspond to the different classes of<br>
immunoglobulins are called alpha, delta, epsilon, gamma, and mu, respectively. The subunit<br>
structures and three-dimensional configurations of different classes of immunoglobulins are well<br>
known.<br>
[0058] "Fv" is an antibody fragment that contains a complete antigen-recognition and -<br>
binding site. In a two-chain Fv species, this region consists of a dimer of one heavy and one<br>
light chain variable domain in tight, non-covalent association. In a single-chain Fv species, one<br><br>
heavy and one light chain variable domain can be covalently linked by a flexible polypeptide<br>
linker such that the light and heavy chains can associate in a dimeric structure analogous to that<br>
in a two-chain Fv species. It is in this configuration that the three CDRs of each variable domain<br>
interact to define an antigen-binding specificity on the surface of the VH-VL dimer. However,<br>
even a single variable domain (or half of a Fv comprising only 3 CDRs specific for an antigen)<br>
has the ability to recognize and bind antigen, although generally at a lower affinity than the<br>
entire binding site.<br>
[0059] The Fab fragment also contains the constant domain of the light chain and the first<br>
constant domain (CHI) of the heavy chain. Fab' fragments differ from Fab fragments by the<br>
addition of a few residues at the carboxy terminus of the heavy chain CHI domain including one<br>
or more cysteines from the antibody hinge regions.<br>
[0060] A "monoclonal antibody" refers to a homogeneous antibody population wherein the<br>
monoclonal antibody is comprised of amino acids (naturally occurring and non-naturally<br>
occurring) that are involved in the selective binding of an antigen. A population of monoclonal<br>
antibodies (as opposed to polyclonal antibodies) are highly specific, in the sense that they are<br>
directed against a single antigenic site. The term "monoclonal antibody" encompasses not only<br>
intact monoclonal antibodies and full-length monoclonal antibodies, but also fragments thereof<br>
(such as Fab, Fab', F(ab')2, Fv), single chain (ScFv), mutants thereof, fusion proteins comprising<br>
an antibody portion, and any other modified configuration of the immunoglobulin molecule that<br>
comprises an antigen recognition site of the required specificity and the ability to bind to an<br>
antigen (see definition of antibody). It is not intended to be limited as regards to the source of<br>
the antibody or the manner in which it is made (e.g., by hybridoma, phage selection, recombinant<br>
expression, transgenic animals, etc.).<br>
[0061] The terms "polypeptide", "oligopeptide", "peptide" and "protein" are used<br>
interchangeably herein to refer to polymers of amino acids of any length. The polymer may be<br>
linear or branched, it may comprise modified amino acids, and it may be interrupted by non-<br>
amino acids. The terms also encompass an amino acid polymer that has been modified naturally<br>
or by intervention; for example, disulfide bond formation, glycosylation, lipidation, acetylation,<br>
phosphorylation, or any other manipulation or modification, such as conjugation with a labeling<br>
component. Also included within the definition are, for example, polypeptides containing one or<br>
more analogs of an amino acid (including, for example, unnatural amino acids, etc.), as well as<br>
other modifications known in the art.<br><br>
[0062] An epitope that "specifically binds" or "preferentially binds" (used interchangeably<br>
herein) to an antibody is a term well understood in the art, and methods to determine such<br>
specific or preferential binding are also well known in the art. A molecule is said to exhibit<br>
"specific binding" or "preferential binding" if it reacts or associates more frequently, more<br>
rapidly, with greater duration and/or with greater affinity with a particular cell or substance than<br>
it does with alternative cells or substances. An antibody "specifically binds" or "preferentially<br>
binds" to a target if it binds with greater affinity, avidity, more readily, and/or with greater<br>
duration than it binds to other substances. For example, an antibody that specifically or<br>
preferentially binds to an epitope is an antibody that binds this epitope with greater affinity,<br>
avidity, more readily, and/or with greater duration than it binds to other epitopes. It is also<br>
understood by reading this definition that, for example, an antibody (or moiety or epitope) that<br>
specifically or preferentially binds to a first target may or may not specifically or preferentially<br>
bind to a second target. As such, "specific binding" or "preferential binding" does not<br>
necessarily require (although it can include) exclusive binding. Generally, but not necessarily,<br>
reference to binding means preferential binding.<br>
[0063] A "sample" encompasses a variety of sample types, including those obtained from an<br>
individual. The definition encompasses blood and other liquid samples of biological origin, solid<br>
tissue samples such as a biopsy specimen or tissue cultures or cells derived therefrom, and the<br>
progeny thereof. A sample can be from a microorganism (e.g., bacteria, yeasts, viruses, viroids,<br>
molds, fungi) plant, or animal, including mammals such as humans, rodents (such as mice and<br>
rats), and monkeys (and other primates). A sample may comprise a single cell or more than a<br>
single cell. The definition also includes samples that have been manipulated in any way after<br>
their procurement, such as by treatment with reagents, solubilization, or enrichment for certain<br>
components, such as proteins or polynucleotides. The term "sample" encompasses a clinical<br>
sample, and also includes cells in culture, cell supernatants, cell lysates, serum, plasma,<br>
biological fluid, human tissue propagated in animals, and tissue samples. Examples of a sample<br>
include blood, plasma, serum, urine, stool, cerebrospinal fluid, synovial fluid, amniotic fluid,<br>
saliva, lung lavage, semen, milk, nipple aspirate, prostatic fluid, mucous, and tears.<br>
[0064] The "complexity" of a sample means the number of different protein species,<br>
including number of different proteins as well as number of different protein variants (including<br>
splice variants, polymorphisms, and protein degradation products).<br><br>
[0065] "Detect" refers to identifying (determining) the presence, absence and/or amount of<br>
the object or substance to be detected, and as described herein, detection may be qualitative<br>
and/or quantitative.<br>
[0066] As used herein, the singular form "a", "an", and "the" includes plural references<br>
unless indicated otherwise. For example, "an" antibody includes one or more antibodies and "a<br>
protein" means one or more proteins.<br>
Methods of the invention<br>
[0067] With respect to all methods described herein, reference to a small epitope antibody<br>
also includes compositions comprising one or more of these antibodies. These compositions<br>
may further comprise suitable excipients, such as pharmaceutically acceptable excipients, as well<br>
as buffers and/or components to enhance stability, which are well known in the art.<br>
Methods for reducing the complexity of a sample<br>
[0068] The invention provides methods using one or more antibodies that bind (generally,<br>
specifically bind) small epitopes, termed "small epitope antibodies", to fractionate a protein<br>
mixture based on the presence or absence or amount of small epitopes within proteins within the<br>
protein mixture, whereby a fraction comprising protein(s) comprising and enriched for the small<br>
epitope is generated. As used herein, "enriched" refers to an increase in concentration and/or<br>
purity of a protein or peptide in comparison with the concentration and/or purity of the protein or<br>
peptide in the sample from which it was derived. Use of the methods of the invention thereby<br>
provides a means for reducing the complexity of a protein mixture, facilitating subsequent use<br>
and/or characterization of the enriched protein components of the sample. Insofar as the amino<br>
acid sequence or composition of the small epitope bound by the antibody is known, binding by a<br>
small epitope antibody provides information relating to amino acid sequence and/or content of<br>
protein(s) bound by the small epitope antibody. As described herein, epitope identity<br>
information {i.e., the amino acid content and/or sequence recognized by a small epitope<br>
antibody) may be used in combination with other methods of the invention to, e.g., identify<br>
proteins. Small epitope antibodies are further described herein.<br>
[0069] The invention further provides methods for purifying and/or enriching protein;<br>
isolating protein; separating protein, preparing protein for characterization (e.g., subsequent<br><br>
analysis); preparing protein for mass spectrometry analysis; identifying protein; discovering new<br>
protein; and/or quantification of protein in a sample.<br>
[0070] As a general overview, the methods comprise: .(a) contacting a sample with at least<br>
one small epitope antibody under conditions that permit binding; and (b) separating an antibody-<br>
protein complex. In one embodiment, steps (a) and (b) occur sequentially. In another<br>
embodiment, steps (a) and (b) occur simultaneously. Generally, proteins comprising one or more<br>
epitope bound by the one or more small epitope antibody are isolated, separated, enriched and/or<br>
purified (i.e., removed from the environment of the original sample). In some embodiments, the<br>
methods further comprise: step (c) of separating protein from the antibody-protein complex. In<br>
some embodiments, the methods further comprise treating the sample with a protein cleaving<br>
agent. In one embodiment, the protein cleaving agent is added prior to step (a) of contacting a<br>
sample with the at least one small epitope antibody. In another embodiment, the protein cleaving<br>
agent is added after step (c) of separating protein from the antibody-protein complex.<br>
[0071] The methods of the invention are useful for fractionating samples comprising protein<br>
(such as polypeptides), which is accomplished by the use of antibodies (termed "small epitope<br>
antibodies") that recognize epitopes that are present in a multiplicity of proteins (such an epitope<br>
consisting of or consisting essentially of 3 linear amino acids, 4 linear amino acids, or 5 linear<br>
amino acids). Small epitope antibodies suitable for use in the methods of the invention are<br>
extensively described herein and exemplified in the Examples. By virtue of the specificity of the<br>
small epitope antibodies, proteins or peptides (e.g., polypeptides) are separated, enriched and/or<br>
purified depending on the presence and/or amount of the small epitope within the protein that is<br>
recognized by the small epitope antibody(ies) used in the methods of the invention. As is<br>
evident, "reducing the complexity of a sample", as used herein, encompasses isolating, purifying,<br>
separating, enriching and/or purifying proteins or peptides (e.g., polypeptides) from a sample<br>
(including removing the proteins or peptides from the environment of the sample).<br>
[0072] Accordingly, in one aspect, the invention provides methods for reducing the<br>
complexity of a sample, said methods comprising: (a) contacting a sample with one or more<br>
small epitope antibody under conditions that permit binding; and (b) separating an antibody-<br>
protein complex, whereby proteins comprising one or more epitope(s) bound by the one or more<br>
small epitope antibody are isolated, separated, enriched and/or purified. In some embodiments,<br>
the methods further comprise: step (c) of separating protein from the antibody-protein complex.<br><br>
[0073] In one embodiment, the invention provides methods for reducing the complexity of a<br>
protein sample, said methods comprising separating a small epitope antibody-protein complex,<br>
whereby proteins comprising an epitope bound by the small epitope antibody are enriched;<br>
wherein the complex was generated by contacting a sample with the small epitope antibody. In<br>
another embodiment, the invention provides methods for reducing the complexity of a protein<br>
sample, said methods comprising separating a plurality of small epitope antibody-protein<br>
complexes, whereby proteins comprising epitopes bound by a plurality of small epitope<br>
antibodies are enriched, and wherein the complexes were generated by contacting a sample with<br>
the plurality of small epitope antibodies.<br>
[0074] In another aspect, the invention provides methods for reducing the complexity of a<br>
protein sample, said methods comprising separating protein from a small-epitope antibody-<br>
protein complex, whereby protein comprising an epitope bound by the small epitope antibody is<br>
enriched; wherein the small epitope antibody-protein complex is generated by contacting a<br>
sample with the small epitope antibody under conditions that permit binding, whereby the small<br>
epitope antibody-protein complex is generated; and separating an antibody-protein complex from<br>
unbound proteins in the sample, if any. In one embodiment, the invention provides methods for<br>
reducing the complexity of a protein sample, said methods comprising separating a plurality of<br>
proteins from small epitope antibody-protein complexes, whereby protein comprising epitopes<br>
bound by a plurality of small epitope antibodies is enriched, and wherein the small epitope<br>
antibody-protein complexes are generated by contacting a sample with a plurality of small<br>
epitope antibodies under conditions that permit binding to proteins in the sample, whereby small<br>
epitope antibody-protein complexes are generated, and separating the antibody-protein<br>
complexes from unbound proteins in the sample, if any.<br>
[0075] As is evident, one or more steps may be combined and/or performed sequentially<br>
(often in any order, as long as the requisite product(s) are able to be formed), and, as is evident,<br>
the invention includes various combinations of the steps described herein. It is also evident, and<br>
is described herein, that the invention encompasses methods in which the initial, or first, step is<br>
any of the steps described herein. Methods of the invention encompass embodiments in which<br>
later, "downstream" steps are an initial step.<br>
[0076] In some embodiments, the methods further comprise a step of treating the sample<br>
with a protein cleaving agent, whereby polypeptide fragments are generated. In embodiments<br>
involving step (c) of separating protein from the antibody-protein complex, the sample can<br><br>
treated with a protein cleaving agent prior to step (a) of contacting a sample with the at least one<br>
small epitope antibody, and/or following step (c) of separating protein from the antibody-protein<br>
complex. The protein cleaving agent may be an enzyme (such as chymotrypsin or trypsin) or a<br>
chemical agent (such as cyanogen bromide). Protein cleaving agents and methods for treatment<br>
with protein cleaving agents are well known in the art and further described herein.<br>
[0077] In another aspect, the invention provides methods for reducing the complexity of a<br>
sample, said methods comprising (a) contacting a sample with one or more small epitope<br>
antibody under conditions that permit binding; (b) separating an antibody-protein complex,<br>
whereby proteins comprising one or more epitope(s) bound by the one or more small epitope<br>
antibody are enriched; (c) separating protein from protein-antibody complex; and (d) treating the<br>
protein with a protein cleaving agent, whereby polypeptide fragments are generated.<br>
[0078] In another aspect, the invention provides methods for reducing the complexity of a<br>
sample, said methods comprising (a) contacting a sample with one or more small epitope<br>
antibody under conditions that permit binding, thereby forming an antibody-protein complex;<br>
and (b) treating the antibody-protein complex with a protein cleaving agent to produce<br>
polypeptide fragments.<br>
[0079] In another aspect, the invention provides methods for reducing the complexity of a<br>
protein sample, said methods comprising: (a) treating the sample with a protein cleaving agent,<br>
whereby polypeptide fragments are generated; (b) contacting the polypeptide fragments with one<br>
or more small epitope antibody under conditions that permit binding, whereby antibody-<br>
polypeptide complexes are generated; and (c) separating the antibody-polypeptide complex,<br>
whereby polypeptides comprising one or more epitope bound by the one or more small epitope<br>
antibody are enriched.<br>
[0080] One, or more than one (such as about two, about three, about four, about five, about<br>
ten, about twenty, about one hundred, or more) small epitope antibod(ies) may be used in the<br>
methods of the invention. In some embodiments, the sample is contacted with about 20, about<br>
30, about 40, about 50, about 75, about 100, about 125, about 150, about 200, about 300, about<br>
400, about 500, about 1000, or more small epitope antibodies. In some embodiments, the sample<br>
is contacted with at least about 20, about 30, about 40, about 50, about 75, about 100, about 125,<br>
about 150, about 200, about 300, about 400, about 500, about 1000, or more small epitope<br>
antibodies. In some embodiments, the sample is contacted with less than about 100, about 95,<br>
about 90, about 85, about 80, about 75, about 70, about 65, about 60, about 55, about 50, about<br><br>
45, about 40, about 35, about 30, about 25, about 20, about 15, about 10, or fewer small epitope<br>
antibodies. In some embodiments, the sample is contacted with at least about any of 10, 20,30,<br>
40, 50,75,100,125,150,200,300,400, or 500 small epitope antibodies, with an upper limit of<br>
about any of 20, 30,40, 50, 75,100, 125,150, 200, 300,400, 500, or 1000 small epitope<br>
antibodies.<br>
[0081] It is understood that the sample may also be contacted with other protein binding<br>
agents, including antibodies that are not small epitope antibodies, and other protein binding<br>
agents. Such agents may be used simultaneously, sequentially, before or after treatment with<br>
small epitope antibodies.<br>
[0082] In some embodiments, the sample is treated with one or more antibodies that bind to<br>
one or more proteins, preferably proteins that are known to be abundant in the sample, prior to or<br>
simultaneously with the step of contacting the sample with one or more small epitope antibodies.<br>
For example, in a serum sample, pretreatment may comprise antibodies that bind to albumin,<br>
immunoglobulin, and/or other abundant proteins. In one embodiment, proteins in the sample are<br>
cleaved with a protein cleaving agent prior to contact with the one or more antibodies that bind<br>
to one or more known abundant proteins. In another embodiment, proteins in the sample are<br>
cleaved with a protein cleaving agent after contact with the one or more antibodies that bind to<br>
one or more known proteins, such as abundant proteins. In one embodiment, the bound<br>
protein(s) (such as abundant protein(s)) are removed from the sample prior to contact with the<br>
one or more small epitope antibodies. In one embodiment, the method comprises "debulking" of<br>
a sample by treatment with one or more antibodies that bind to one or more known proteins in<br>
the sample, such as abundant protein(s) (optionally followed by removal of bound proteins),<br>
cleavage of proteins in the sample with a protein cleaving agent, and contact of cleaved proteins<br>
with one or more small epitope antibodies. In another embodiment, the method comprises<br>
treatment of the sample with a protein cleaving agent, debulking of the sample by treatment with<br>
one or more antibodies that bind to one or more known proteins, such as abundant protein(s)<br>
and/or cleaved polypeptide fragments in the sample (optionally followed by removal of the<br>
bound protein(s) and/or polypeptide fragments), and contact of the remaining proteins and/or<br>
cleaved polypeptide fragments with one or more small epitope antibodies. In another<br>
embodiment, the method comprises debulking of the sample by treatment with one or more<br>
antibodies that bind to one or more known proteins, such as abundant protein(s) (optionally<br>
followed by removal of the bound protein(s)), contacting the sample with at least one small<br><br>
epitope antibody to form an antibody-protein complex, and treatment of the antibody-protein<br>
complex with a protein cleaving agent.<br>
[0083] It is further understood that the protein components of the sample that remain<br>
following treatment with small-epitope antibodies (i.e., the unbound components) may also be<br>
suitable for use in the methods of the invention using protein generated using the methods of the<br>
invention. Thus, in some embodiments, the methods using the protein generated using the<br>
methods of the invention encompass use of this unbound protein fraction.<br>
[0084] Methods and conditions for antibody binding and separation of antibody-protein<br>
complexes are well known in the art and further described herein. Generally, the sample is<br>
partially or wholly denatured when it is contacted with the small epitope antibody(ies), but<br>
denaturation is not required in every embodiment. In some embodiment, step (a) of contacting a<br>
sample with two or more antibodies is sequential (as when one antibody is contacted with the<br>
sample, then removed, another antibody is contacted with the sample and removed, and so on).<br>
In other embodiments, step (a) of contacting with two or more antibodies is in parallel, for<br>
example, as when a group of antibodies are contacted with the sample simultaneously. In some<br>
embodiments, several groups of two or more antibodies are serially contacted with the sample,<br>
for example, group 1 is contacted and removed, group 2 is contacted and removed, and so on.<br>
[0085] As noted in the definition, and as used herein, "sample" encompasses a variety of<br>
sample types, including those obtained from an individual. In some embodiment, the sample<br>
comprises blood, plasma, serum, urine, stool, cerebrospinal fluid, synovial fluid, amniotic fluid,<br>
saliva, lung lavage, semen, milk, nipple aspirate, prostatic fluid, mucous, and tears. Suitable<br>
samples for use in the methods of the invention are described further herein.<br>
Methods using proteins isolated ('enriched') using the methods of the invention<br>
[0086] The proteins isolated or enriched using the methods of the invention can be used for a<br>
variety of purposes. For purposes of illustration, methods of characterizing proteins using the<br>
proteins enriched and/or purified by the methods of the invention, are described. In some<br>
embodiments, the proteins are characterized using mass spectrometry, whereby the proteins may<br>
be quantified and/or identified. Methods of genotyping (protein mutation detection), identifying<br>
splice variants, determining the presence or absence of a protein of interest, expression profiling;<br><br>
methods for identifying protein degradation products; methods for identifying change inpost-<br>
translational modification, and methods of protein discovery are also described.<br>
[0087] For simplicity and convenience, reference is generally made to "protein(s)". It is<br>
understood that reference to protein encompasses "polypeptides" (interchangeably termed<br>
"polypeptide fragments"). As is evident from the discussion herein, in some embodiments, a<br>
protein cleaving agent is used to generate polypeptide fragments.<br>
Methods of characterizing a protein<br>
[0088] The invention provides methods for characterizing (for example, detecting (presence<br>
or absence) and/or quantifying) a protein of interest (generally, a polypeptide fragment). In some<br>
embodiments, use of the methods of the invention generates one or more fractions of the sample,<br>
each of which comprises fewer proteins than in the starting sample, facilitating subsequent<br>
characterization of the protein comprised in the fraction. In particular, characterization using<br>
mass spectrometry is expected to be enhanced, as further described herein.<br>
[0089] Thus, the invention provides methods for characterizing a protein comprising: (a)<br>
reducing the complexity of a sample using any of the methods described herein, whereby<br>
proteins are enriched and/or purified; and (b) analyzing the proteins (interchangeably termed<br>
"products") which are isolated by any one or of the methods described herein.<br>
[0090] In another aspect, the invention provides methods for characterizing a protein<br>
comprising: analyzing proteins (interchangeably termed "products"), wherein the protein is<br>
prepared using any of the methods for reducing complexity of a sample described herein<br>
(including: methods for purifying and/or enriching a protein, methods for isolating a protein,<br>
methods for separating a protein, methods for preparing a protein fraction for characterization,<br>
methods for preparing a protein fraction for mass spectrometry analysis, methods for identifying<br>
a protein (such as one or more protein, or a group of proteins), methods for discovering a new<br>
protein, and methods for quantification of protein in a sample.)<br>
[0091] The step of analyzing can be performed by any method known in the art or described<br>
herein. Methods for analyzing proteins are well known in the art, and include: sodium dodecyl<br>
sulphate-polyacrylamide gel electrophoresis ("SDS-PAGE"), isoelectric focusing, separated by<br>
such techniques as high pressure liquid chromatography, FPLC, thin layer chromatography,<br>
affinity chromatography, gel-filtration chromatography, ion exchange chromatography, and other<br><br>
standard biochemical analyses, immunodetection, protein sequencing, analysis with protein<br>
arrays, mass spectrometry, and the like. Thus, the invention includes those further analytical<br>
and/or quantification methods as applied to any of the products of the methods herein.<br>
[0092] In some embodiments, the step of analyzing comprises determining amount of said<br>
proteins, whereby the amount of protein(s) prepared, enriched and/or separated is quantified. It<br>
is understood that the amount of enriched protein(s) may be determined using quantitative and/or<br>
qualitative methods. Determining amount of protein product includes determining whether<br>
product is present or absent.<br>
[0093] In some embodiments, the step of analyzing comprises identifying one or more of<br>
said proteins. Methods for identifying a protein are known in the art, and include:<br>
immunodetection, protein sequencing, and the like. In some embodiments, essentially all of the<br>
enriched proteins (purified or enriched from a sample) are identified. In some embodiments, the<br>
identity of the epitope(s) to which the small epitope antibody(ies) bind is used to assist<br>
identification of the enriched proteins. In some embodiments, a protein is identified using any<br>
one or more of the following characteristics: sequence; mass; m/z ratio (in embodiments<br>
involving mass spectrometric analysis), amino acid composition, and any other method that<br>
provide sufficient information to identify a protein. As used herein, "identify" includes<br>
identifying known (previously characterized proteins) as well as discovery of previously<br>
unknown or uncharacterized proteins (including protein variants such as mutant proteins,<br>
differentially modified proteins {e.g., varying carbohydrate content) and splice variants). In<br>
some embodiments, a multiplicity, a large multiplicity or a very large multiplicity of proteins are<br>
identified. In other embodiments, at least about 2, 3, 4, 5,10,20, 30,40, 50, 60,70, 80, 90,100,<br>
500, or 1000 or more proteins are identified.<br>
[0094] In other embodiments, the step of analyzing comprises determining the mass of one<br>
or more protein(s).<br>
[0095] In some embodiments, the step of analyzing includes analysis for the detection of any<br>
alterations in the protein, as compared to a reference protein which is identical (at least in part) to<br>
the protein sequence other than the sequence alteration. The sequence alterations may be<br>
sequence alterations present in the genomic sequence or may be sequence alterations which are<br>
not reflected in the genomic DNA sequences, for example, alterations due to post transcriptional<br>
alterations, and/or mRNA processing, including splice variants, and/or post-translational<br>
modifications, such as variation in amount of glycosylation, and protein degradation or by-<br><br>
products. Sequence alterations include mutations (such as deletion, substitution, insertion<br>
and/or transversion of one or more amino acid).<br>
[0096] It is understood that the identity (sequence) of the epitope(s) to which the small<br>
epitope antibody(ies) may be used in combination with any of the methods described herein to,<br>
e.g., identify proteins.<br>
Method of characterizing a protein using mass spectrometry<br>
[0097] In some embodiments, mass spectrometry (MS) is used to characterize the proteins<br>
isolated using the methods of the invention. Generally, in embodiments involving mass<br>
spectrometric analysis, the sample will be treated with a protein cleaving agent (whereby<br>
polypeptide fragments are generated), but treatment with a cleaving agent is not required in<br>
every embodiment. In some embodiments, the sample is. treated with a protein cleaving agent<br>
prior to contacting the sample with small epitope antibodies. In other embodiments, the sample<br>
is treated with a protein cleaving agent following enrichment of a protein fraction by contacting<br>
with a small epitope antibody, separation of antibody-protein complex, and separation of protein<br>
from the protein-antibody complex. As noted herein, the protein (such as polypeptide fragments)<br>
generated using the methods of the invention are particularly amenable to analysis using mass<br>
spectrometry because use of the methods of the invention generates fractions of proteins that are<br>
less complex than are the starting sample. Insofar as the epitope present within the protein is<br>
known, e.g., the cognate epitope recognized by the small epitope antibody used to purify and/or<br>
enrich the protein fraction comprising the protein, the amino acid sequence or content of the<br>
epitope (termed "epitope sequence" or "epitope content") provides further information useful for<br>
characterizing and identifying the protein.<br>
[0098] Methods for mass spectrometric protein analysis are well known in the art and further<br>
described herein. Mass spectrometry methods have been used to quantify and/or identify<br>
proteins. (See, e.g., Li et al. (2000) Tibtech 18:151-160; Rowley et al. (2000) Methods 20: 383-<br>
397; and Kuster and Mann (1998) Curr. Opin. Structural Biol. 8: 393-400.) Mass spectrometric<br>
techniques have also been developed that permit at least partial de novo sequencing of isolated<br>
proteins. Chait et al. (1993) Science 262:89-92; Keough et al.(1999) Proc. Natl. Acad. Sci. USA<br>
96:7131-6; reviewed in Bergman (2000) EXS 88:133-44.<br><br>
[0099] Polypeptide mass mapping provides a polypeptide mass fingerprint of the protein or<br>
protein fraction under analysis, based on its amino acid composition. Polypeptide mass mapping<br>
can be obtained using, for example, the MALDI-TOF platform, in which matrix-assisted laser<br>
desorption/ionization (MALDI) is used to ionize polypeptides of interest while the time of flight<br>
distribution of the ionized polypeptides provides mass to charge ratio specifications for each<br>
polypeptide which can be used to query protein sequence databases. The polypeptide mass<br>
fingerprints yielded comprise the amino acid composition based on mass and charge<br>
determination. Using these results, a small set of polypeptide mass matches may provide<br>
sufficient information for the identification of the corresponding protein.<br>
[00100] In a second method for protein identification by MS, individual polypeptides in the<br>
mixture are fragmented to generate sequence infonnation. Polypeptides are ionized by<br>
electrospray (ESI) from the liquid phase, and then sprayed into a tandem mass spectrometer that<br>
is capable of resolving polypeptides in a mixture, isolating polypeptides of interest and<br>
dissociating individual polypeptide species into constituent amino- and carboxy-terminal-<br>
containing fragments by predominantly disrupting polypeptide bonds (collision induced<br>
dissociation). The resulting mass spectrum is comprised of the parent ion as well as two<br>
overlapping mass ladders of ions derived from the amino- and carboxy-terminal containing<br>
fragments. Because each member of a ladder differs in mass-to-charge ratio (termed "m/z") by 1<br>
amino acid from its nearest mass neighbor in the series, a partial primary sequence can be<br>
generated and used to query both protein and translated DNA sequence databases. This mass<br>
spectrometry platform provides specific sequence information derived from several polypeptides,<br>
which is often more useful for protein identification that a list of polypeptide masses that reflect<br>
the amino acid composition of the polypeptide (as generated by other platforms, including<br>
SELDI-TOF).<br>
[00101] Mass spectrometry methods further permit quantification of proteins that are<br>
analyzed, as further described below.<br>
[00102] Other mass spectrometry methods are well known in the art, and include: matrix<br>
assisted laser desorption/ionization ("MALDI") mass spectrometry; surface-enhanced laser<br>
desorption/ionization ("SELDI"); Tandem mass spectrometry (e.g., MS/MS, MS/MS/MS, ESI-<br>
MS/MS, etc.). In some embodiments, tandem mass spectrometry is carried out using a laser<br>
desorption/ionization mass spectrophotometer .that is further coupled to a quadrupole time-of-<br>
flight mass spectrometer QqTOF MS (see e.g., Krutchinsky et al, WO 99/38185). Methods such<br><br>
as MALDI-QqTOFMS (Krutchinsky et al., WO 99/38185; Shevchenko et al. (2000) Anal. Chem.<br>
72:2132-2141), ESI-QqTOF MS (Figeys et al. (1998) Rapid Comm'ns. Mass Spec. 12-1435-<br>
144) and chip capillary electrophoresis (chip-CE)-QqTOF MS (Li et al. (2000) Anal. Chem. 72:<br>
599-609) have been described previously. Mass spectrometers and techniques for using them in<br>
methods of the invention are well known to those of skill in the art. A person skilled in the art<br>
would understand that any of the components of a mass spectrometer (e.g., desorption source, .<br>
mass analyzer, detect, etc.) can be combined with other suitable components described herein or<br>
those known in the art. For additional information regarding mass spectrometers, see, e.g.,<br>
Principles of Instrumental Analysis, 3rd ed., Skoog, Saunders College Publishing, Philadelphia,<br>
1985; and Kirk-Othmer Encyclopedia of Chemical Technology, 4th ed. Vol. 15 (John Wiley &amp;<br>
Sons, New York 1995), pp. 1071-1094.<br>
Data Analysis of Mass Spectra<br>
[00103] The mass spectra data obtained using the mass spectrometry analysis can be used to<br>
obtain information on the quantity and/or identity of the enriched protein products obtained using<br>
the methods of the invention. Data generated by desorption and detection of polypeptides can be<br>
analyzed using any suitable means {e.g., visually, by computer, etc). In one embodiment, data is<br>
analyzed with the use of a programmable digital computer. The computer contains code that<br>
receives as input, data on the strength of the signal at various molecular masses received from a<br>
particular addressable location on the substrate. This data can indicate the number of products<br>
detected, optionally including the strength of the signal of a peak value and the determined<br>
molecular mass for each product detected.<br>
[00104] Data analysis can include the steps of determining signal strength (e.g., height of<br>
peaks) of a peak value (e.g., of a particular mass-to-charge value or range of values) detected and<br>
removing "outliers" (data deviating from a predetermined statistical distribution). The observed<br>
peaks can be normalized, a process whereby the height of each peak relative to some reference is<br>
calculated. For example, a reference can be background noise generated by instrument and<br>
chemicals (e.g., energy absorbing molecule) which is set as zero in the scale. Then the signal<br>
strength detected for each polypeptide or other substances can be displayed in the form of<br>
relative intensities in the scale desired (e.g., 100). Alternatively, a standard may be admitted<br>
with the sample so that a peak from the standard can be used as a reference to calculate relative<br><br>
intensities of the signals observed for each affinity tagged product detected. Software programs<br>
such as the Biomarker Wizard program (Ciphergen Biosystems, Inc., Fremont, Calif.) can be<br>
used to aid in analyzing mass spectra.<br>
[00105] In some embodiments the amounts of one or more proteins present in a sample is<br>
determined, in part, by executing an algorithm with a programmable digital computer. The<br>
algorithm identifies at least one peak value in a first mass spectrum of a first sample and in a<br>
second mass spectrum of a second sample. The algorithm then compares the signal strength of<br>
the peak value of the first mass spectrum to the signal strength of the peak value of the second<br>
mass spectrum. The relative signal strengths are an indication of the amount of the protein that is<br>
present in the first and second samples. A standard containing a known amount of a protein can<br>
be analyzed as the second sample to better quantify the amount of the protein present in the first<br>
sample. In certain embodiments, the identities of the proteins in the first and second samples can<br>
also be determined (see below).<br>
[00106] The present invention also provides methods of determining the identity of a protein.<br>
In certain embodiments, a programmable digital computer is used to access a database containing<br>
one or more mass spectra. An algorithm is then executed with a programmable digital computer<br>
to determine at least a first measure for each of the predicted mass spectra. The first measure is<br>
an indication of the closeness-of-fit between a mass spectrum of the protein and each of the<br>
plurality of predicted mass spectra.<br>
[00107] The data of a mass spectrum can be used to identify the proteins by executing an<br>
algorithm with a programmable digital computer that compares the MS data to records in a<br>
database. Each molecule provides characteristic mass-spectrometric (MS) data (also referred to<br>
as a mass spectral "signature" or "fingerprint") when analyzed by MS methods. This data can be<br>
analyzed by comparing it to databases containing, inter alia, actual or theoretical MS data or<br>
protein sequence information. Additionally, a protein may be cleaved into fragments for MS<br>
analysis. Information obtained from the MS analysis of fragments is also compared to a database<br>
to identify proteins {e.g., proteins) in the sample (see e.g., Yates (1998) J. Mass Spec. 33:1-19;<br>
Yates et al., U.S. Pat. No. 5,538,897; Yates et al., U.S. Pat. No. 6,017,693; PCT Publication No.<br>
WO 00/11208 and Gygi et al. (1999) Nat. Biotechnol 10:994-999). Software resources that<br>
facilitate interpretation of mass spectra, especially protein mass spectra, and mining of public<br>
domain sequence databases are now readily accessible on the Internet to facilitate protein<br>
identification. Among these are Protein Prospector (http://prospector.ucsf/edu), PROWL<br><br>
(http://prowl.rockefeller.edu), and the Mascot Search Engine (Matrix Science Ltd., London, UK,<br>
www.matrixscience.com).<br>
[00108] In certain embodiments, MS data and information obtained from that data are<br>
compared to a database consisting of data and information relating to proteins. For example, the<br>
database may consist of sequences of nucleotides or amino acids. The database may consist of<br>
nucleotide or amino acid sequences of expressed sequence tags (ESTs). Alternatively, the<br>
database may consist of sequences of genes at the nucleotide or amino acid level. The database<br>
can include, without limitation, a collection of nucleotide sequences, amino acid sequences, or<br>
translations of nucleotide sequences included in the genome of any species.<br>
[00109] A database of information relating to proteins, e.g., sequences of nucleotides or amino<br>
acids, is typically analyzed via a computer program or a search algorithm which is optionally<br>
performed by a computer. Information from sequence databases is searched for best matches<br>
with data and information obtained from the methods of the present invention (see e.g., Yates<br>
(1998) J. Mass Spec. 33:1-19; Yates et al., U.S. Pat. No. 5,538,897; Yates et al., U.S. Pat. No.<br>
6,017,693). Any appropriate algorithm or computer program useful for searching a database can<br>
be used. Search algorithms and databases are constantly updated, and such updated versions will<br>
be used in accordance with the present invention. Examples of programs or databases can be<br>
found on the World Wide Web (WWW) at http://base-peak.wiley.com/, http://mac-mann6.embl-<br>
heidelberg.de/MassSpec/Software.html, http://www.mann.embl-<br>
heidelberg.de/Services/PeptideSearch/PeptideSearchIn-tro.html,<br>
ftp://ftp.ebi.ac.uk/pub/databases/, and http://donatello.ucsf.ed- u. U.S. Pat. Nos. 5,632,041;<br>
5,964,860; 5,706,498; and 5,701,256 also describe algorithms or methods for sequence<br>
comparison. Other examples of databases include the Genpept database, the GenBank database<br>
(described in Burks et al. (1990) Methods in Enzymology 183: 3-22, EMBL data library<br>
(described in Kahn et al. (1990) Methods in Enzymology 183:23-31, the Protein Sequence<br>
Database (described in Barker et al. (1990) Methods in Enzymology 183: 31-49, SWISS-PROT<br>
(described in Bairoch et al. (1993) Nucleic Acids Res. 21: 3093-3096, and PIR-International<br>
(described in (1993) Protein Seg. Data Anal. 5:67-192).<br>
[00110] In some embodiments, the amino acid sequence of the epitope recognized by the<br>
small epitope antibody (termed "epitope sequence") is used in conjunction with the database<br>
search information and search algorithms to enhance identification of proteins. For example,<br>
prior to or following analysis of MS data and information obtained from that data by comparison<br><br>
to a database consisting of data and information relating to proteins, the amino acid sequence of<br>
the epitope may be used to refine the data analysis. For example, a preliminary list of protein<br>
identity candidates may be refined by excluding members from that list that do not include the<br>
epitope sequence. In another example, a database may be compiled or theoretically generated of<br>
all proteins comprising a given epitope sequence. This database may then be subjected to further<br>
analysis using data analysis methods known in the art.<br>
[00111] A database of information relating to proteins, e.g., sequences of nucleotides or amino<br>
acids, is typically analyzed via a computer program or a search algorithm which is optionally<br>
performed by a computer.<br>
[00112] In a further embodiment, novel databases are generated for comparison to mass<br>
spectrometrically determined MS data, e.g., mass or mass spectra of cleaved protein and<br>
polypeptide fragments. For example, a theoretical database of all polypeptide fragments<br>
comprising an epitope recognized by a small epitope antibody is generated. This database may<br>
be used in conjunction with any of the data analysis tools and methods described herein.<br>
[00113] In some embodiments, the mass of a polypeptide derived from a mass spectrum is<br>
used to query a database for those masses of proteins or predicted proteins from nucleic acid<br>
sequences that provide the closest fit. In this manner, an unknown protein can be rapidly<br>
identified without an amino acid sequence. In other embodiments of the invention, the masses<br>
provided from polypeptide fragments thereof can be compared to the predicted mass spectra of a<br>
database of proteins or predicted proteins from a nucleic acid sequences that provide the closest<br>
fit.<br>
[00114] Sequences or simulated cleavage fragments from the sequence database that fall<br>
within a desired range of similar sequence homologies to sequences generated from the MS data<br>
of parent or fragment molecules are designated "matches" or "hits." In this manner, the identity<br>
of the proteins or fragments thereof can be rapidly determined. The investigator can customize<br>
or vary the range of acceptable sequence homology comparison values according to each<br>
particular analysis.<br>
[00115] It is understood that for convenience, reference is made to protein "identity". It is<br>
understood that the methods described herein are equally applicable to the determination of<br>
presence or absence of a mutation (such as an amino acid substitution, transversion, insertion or<br>
deletion), and other protein variants, such as splice variants, degradation products, and/or<br>
differential post-translational modification (for example, variation in glycosylation level).<br><br>
[00116] In some embodiments, the presence or absence of a mutation is determined by<br>
detection of a change in m/z ratio relative to a reference m/z ratio.<br>
[00117] In some embodiments, level (or changes in level) of post-translational modification is<br>
determined by comparing endoglycosylase-treated sample with a reference sample (e.g., a<br>
sample that has not been treated with endoglycosylase), whereby level of post translational<br>
modification is determined.<br>
Expression profiling<br>
[00118] The methods of the invention are suitable for use in determining the levels of<br>
expression of one or more proteins in a sample. As described above, enriched and/or purified<br>
protein fractions can be detected and/or quantified by various methods, as described herein<br>
and/or known in the art. In some embodiments, protein fractions are analyzed (including<br>
quantification and/or identification) using mass spectrometry. It is understood that amount of<br>
protein product may be determined using quantitative and/or qualitative methods. Determining<br>
amount of product includes determining whether product is present or absent. Thus, an<br>
expression profile can includes information about presence or absence of one or more protein<br>
sequences of interest. "Absent" or "absence" of product, and "lack of detection of product" as<br>
used herein includes insignificant, or de minimus levels.<br>
[00119] In some embodiments, the amounts of proteins in two or more samples are compared.<br>
Typically, the samples have overlapping protein profiles. Using the methods of the present<br>
invention, the amounts of the proteins can be compared to determine how the profiles differ in<br>
the nature and amount of proteins that are present. These methods are useful for identifying a<br>
change in the nature or amount of a protein that is indicative of a disease state (e.g., a disease<br>
biomarker, PSA, BRCA1, etc.) or treatment efficacy, or toxic effects of an agent, or presence of<br>
a pathogen (e.g., HIV, bacterial pathogens, viral pathogens, prions, etc), etc. These methods are<br>
also useful for discovering proteins that are associated with disease states for drug discovery<br>
purposes, diagnostic purposes, etc. In particular, it is useful to compare the protein profiles of<br>
samples that are from different subjects or have been subjected to different conditions or<br>
treatments.<br>
[00120] For example, in certain embodiments, the first sample is an untreated control sample<br>
and the second sample has been subjected to an agent or condition. Examples of agents include,<br><br>
but are not limited to: a chemotherapeutic agent, ultraviolet light, a medical device {e.g., a stent<br>
defibrillator), an exogenous gene, and a growth factor. Those of skill in the art will recognize<br>
that there are many ways to introduce an exogenous gene into a cell (see, e.g., Ausubel et al.,<br>
eds., (1994), supra). In other embodiments, the first sample is a diseased sample and the second<br>
sample is a non-diseased sample. In addition, agents can take the form of candidate drugs. For<br>
example, the proteins in a first sample treated with a candidate drug and can be compared to a<br>
second sample which is a negative or positive control. The influence of the candidate drug on<br>
the amount of a protein (e.g., a protein) present in the first and second sample can be an<br>
indication of the candidate drugs efficacy or toxicity. Those of skill in the art will appreciate that<br>
these methods can be adapted to analyze the effects of any agent on a disease state or amount of<br>
a disease marker present in a sample. In one embodiment, the methods are used to identify<br>
protein(s) that are associated with treatment with an agent (such as a candidate drug). Such<br>
proteins may be, e.g., may be associated with efficacy of the agent, and thereby serve as a proxy<br>
for a clinical endpoint.<br>
Biomarkers<br>
[00121] Biomarker protein (or proteins) can be identified using the expression profiling and<br>
characterization methods described herein. A biomarker is a protein of interest, for which the<br>
detection, monitoring, quantitation, and/or characterization is of interest. In some embodiments,<br>
a biomarker is correlated with a specific condition or treatment, such as a disease or condition,<br>
treatment with a drug (including efficacy of drug treatment and/or toxicity), treatment with a<br>
medical device, and the like. In other embodiments, a biomarker is expressed in a tissue or cell<br>
of interest (e.g., a tumor, an organ, etc.). As used herein, a biomarker protein may be a newly<br>
identified protein or protein variant (such as a mutant protein, splice variant, a protein with<br>
altered post-translational modification, etc.). In other embodiments, a biomarker is a tissue-<br>
specific marker.<br>
[00122] A biomarker can be used as a surrogate marker in diagnosis (including staging of<br>
disease, in some embodiments), prognosis, evaluation and/or selection of therapies, monitoring<br>
of disease progression, monitoring of efficacy of treatment, and/or treatment of disease. In some<br>
embodiments, a biomarker is detected and/or quantified by any method known in the art, and/or<br><br>
any method described herein, whereby expression of the biomarker (presence or absence of<br>
biomarker, or differential expression of the biomarker) indicates the presence of a disorder or a<br>
condition. In one embodiment, increase in level of a biomarker indicates the presence of a<br>
disorder or condition. In another embodiment, decrease in level of a biomarker indicates the<br>
presence of a disorder or condition. In some embodiments, biomarker expression is used to<br>
evaluate the efficacy of a particular therapeutic treatment regimen in animal studies, in clinical<br>
trials, or to monitor the treatment of an individual subject. In some embodiments, the biomarker<br>
serves as a proxy for a desired clinical endpoint. In other embodiments, the biomarker is<br>
correlated with efficacy of an agent, as when biomarker expression is predictive of efficacy of<br>
treatment with an agent (such as a drug). In one embodiment, increase in le\d of a biomarker<br>
indicates efficacy or progress of treatment. In another embodiment, decrease in level of a<br>
biomarker indicates efficacy or progress of treatment.<br>
[00123] The biomarker can be used as a marker for toxicity, including, toxicity of an agent<br>
such as a pharmaceutical, new drug candidate, cosmetic, or other chemical. In some<br>
embodiments, detection of biomarker expression may also be used to monitor for environmental<br>
exposure to an agent, such as a toxin or a pathogen. In one embodiment, increase in level of a<br>
biomarker indicates toxicity or exposure to an agent. In another embodiment, decrease in level<br>
of a biomarker indicates toxicity or exposure to an agent.<br>
[00124] A biomarker can be used to screen a plurality or library of molecules and compounds<br>
for specific binding affinity, including, for example, DNA molecules, RNA molecules, peptide<br>
nucleic acids, polypeptides, mimetics, small molecules, and the like. In one embodiment, an<br>
assay involves providing a plurality of molecules and/or compounds, combining a biomarker<br>
with the plurality of molecules and/or compounds under conditions to allow specific binding, and<br>
detecting specific binding to identify at least one molecule or compound which specifically binds<br>
the biomarker.<br>
[00125] Similarly, one or more biomarkers, or portions thereof, can be used to screen a<br>
plurality or library of molecules and/or compounds in any of a variety of screening assays to<br>
identify a ligand. Methods for screening are well known in the art. The assay can be used to<br>
screen, for example, aptamers, DNA molecules, RNA molecules, peptide nucleic acids,<br>
polypeptides, mimetics, proteins, antibodies, agonists, antagonists, immunoglobulins, inhibitors,<br>
small molecules, pharmaceutical agents or drug compounds and the like, which specifically bind<br>
the biomarker.<br><br>
[00126] In another embodiment, one or more antibodies comprising an antigen binding site<br>
that specifically binds a biomarker can be used for the detection of the biomarker (including in<br>
vitro and in vivo detection). In another example, an antibody that specifically binds a biomarker<br>
can be linked to an in vivo imaging reagent, such as, for example, 3H, mIn,1251, (see Esteban et<br>
al. (1987) J. Nucl. Med 28.861-870), and used in an in vivo imaging application.<br>
Compositions and kits<br>
[00127] The invention also provides compositions for use in any of the methods described<br>
herein, such as methods for reducing the complexity of a sample, methods for purifying and/or<br>
enriching a protein or a plurality of proteins, methods for isolating and/or separating a protein or<br>
a plurality of proteins, and/or methods for preparing a protein, a plurality of proteins, or a protein<br>
fraction for characterization, methods for preparing a protein, a plurality of proteins, or a protein<br>
fraction for mass spectrometry analysis, methods for identifying a protein or a plurality of<br>
proteins, methods for discovering one or more new proteins, methods for detection and/or<br>
quantification of a protein or a plurality of proteins in a sample, methods for characterizing a one<br>
or more proteins, methods for expression profiling, methods for identifying protein degradation<br>
products, methods for identifying change(s) in post-translational modification, and/or methods<br>
for determining the mass, the amount and/or identity of protein(s) in a sample. The compositions<br>
used in the methods of the invention may comprise one or more (such as about 2, about 3, about<br>
4, about 5, about 7, about 10, about 15 or more) small epitope antibody(ies). In some<br>
embodiments, the composition comprises less than about 100, about 95, about 90, about 85,<br>
about 80, about 75, about 70, about 65, about 60, about 55, about 50, about 45, about 40, about<br>
35, about 30, about 25, about 20, about 15, about 10, about 5, or fewer small epitope antibodies.<br>
In some embodiments, the composition comprises at least about 20, about 30, about 40, about 50,<br>
about 75, about 100, about 125, about 150, about 200, about 300, about 400, about 500, about<br>
1000, or more small epitope antibodies. In some embodiments, the composition comprises about<br>
20, about 30, about 40, about 50, about 75, about 100, about 125, about 150, about 200, about<br>
300, about 400, about 500, about 1000, or more small epitope antibodies. In some embodiments,<br>
the composition comprises at least about any of 10,20,30,40,50,75,100,125,150,200, 300,<br>
400 or 500 small epitope antibodies, with an upper limit of about any of 20, 30,40, 50,75,100,<br>
125,150,200,300,400,500 or 1000 small epitope antibodies.<br><br>
[00128] The invention also provides kits for use in the instant methods. Kits of the invention<br>
include one or more containers comprising one or more small epitope antibody(ies). In some<br>
embodiments, the kit comprises less than about 100, about 95, about 90, about 85, about 80,<br>
about 75, about 70, about 65, about 60, about 55, about 50, about 45, about 40, about 35, about<br>
30, about 25, about 20, about 15, about 10, about 5, or fewer small epitope antibodies. In some<br>
embodiments, the kit comprises at least about 20, about 30, about 40, about 50, about 75, about<br>
100, about 125, about 150, about 200, about 300, about 400, about 500, about 1000, or more<br>
small epitope antibodies. In some embodiments, the kit comprises about 20, about 30, about 40,<br>
about 50, about 75, about 100, about 125, about 150, about 200, about 300, about 400, about 500,<br>
about 1000, or more small epitope antibodies. In some embodiments, the kit comprises at least<br>
about any of 10,20, 30,40,50, 75,100,125,150,200,300,400 or 500 small epitope antibodies<br>
with an upper limit of about any of 20, 30,40, 50, 75,100, 125,150, 200, 300,400, 500 or 1000<br>
small epitope antibodies. In some embodiments, the kit further comprises instructions for use in<br>
accordance with any of the methods of the invention described herein, such as methods for<br>
reducing the complexity of a sample, methods for purifying and/or enriching a protein or a<br>
plurality of proteins, methods for isolating and/or separating a protein or a plurality of proteins,<br>
and/or methods for preparing a protein, a plurality of proteins, or a protein fraction for<br>
characterization, methods for preparing a protein, a plurality of proteins, or a protein fraction for<br>
mass spectrometry analysis, methods for identifying a protein or a plurality of proteins, methods<br>
for discovering one or more new proteins, methods for detection and/or quantification of a<br>
protein or a plurality of proteins in a sample, methods for characterizing a one or more proteins,<br>
methods for expression profiling,, methods for identifying protein degradation products, methods<br>
for identifying change(s) in post-translational modification, and/or methods for determining the<br>
mass, the amount and/or identity of protein(s) in a sample.<br>
[00129] The invention also comprises any of the protein "products" (e.g., proteins enriched,<br>
purified, isolated, prepared, separated, and/or fractionated using any of the methods of the<br>
invention described herein. The invention also provides proteins or protein fragments<br>
characterized (e.g., detected, identified, quantified, etc.) using any of the methods of the<br>
invention described herein and compositions comprising such products. Such proteins comprise<br>
a cognate small epitope that is recognized by the small epitope antibody (to which the protein<br>
was bound). The invention also provides small epitope antibody-protein complexes or small<br>
epitope antibody-protein fragment complexes (for methods wherein the proteins are contacted<br><br>
with a protein cleaving agent prior to contact with the small epitope antibody(ies)) prepared or<br>
isolated by any of the methods described herein. The invention also provides proteins or protein<br>
fragments separated from a small epitope antibody-protein complex or small epitope antibody-<br>
protein fragment complex according to any of the methods described herein, and/or protein<br>
fragments prepared from proteins after separation from small epitope antibody(ies).<br>
[00130] In another aspect, the invention includes compositions and/or kits comprising<br>
intermediates (such as complexes, e.g., small epitope antibody-protein complex) produced by<br>
any aspect of the methods of the invention. The invention also provides incubation mixtures<br>
comprising protein-containing samples and small epitope antibodies and/or small epitope<br>
antibody-protein complexes as described herein.<br>
[00131] The kits of this invention are in suitable packaging. Suitable packaging includes, but<br>
is not limited to, vials, bottles, jars, flexible packaging (e.g., sealed Mylar or plastic bags), and<br>
the like. In some embodiments, the kit comprises a container and a label or package insert(s) on<br>
or associated with the container. The label or package insert may indicate that the small epitope<br>
antibody(ies) are useful for any of the methods described herein, e.g., method for reducing the<br>
complexity of a sample, or method for identifying a protein, characterizing a protein, and/or<br>
expression profiling. Instructions may be provided for practicing any of the methods described<br>
herein.<br>
Components and reaction mixtures useful in the methods of the invention<br>
Small epitope antibody<br>
[00132] The methods of the invention use a small epitope antibody. As used herein, a "small<br>
epitope antibody" is an antibody that binds (generally specifically binds) a small peptide epitope.<br>
By virtue of the epitope specificity, small epitope antibodies generally recognize a multiplicity of<br>
proteins that comprise the small epitope to which the antibody binds. Insofar as the small<br>
epitope bound by the antibody is known, binding by a small epitope antibody provides<br>
information relating to amino acid content and/or sequence of protein(s) bound by the small<br>
epitope antibody. Small epitope antibodies are described, for example, in co-pending U.S. Patent<br>
Application No. 10/687,174. Small epitope antibodies and methods of making small epitope<br>
antibodies are further discussed herein and exemplified in the Examples.<br><br>
[00133] An antibody can encompass monoclonal antibodies, polyclonal antibodies, antibody<br>
fragments {e.g., Fab, Fab', F(ab')2, Fv, Fc, etc.), chimeric antibodies, single chain (ScFv),<br>
mutants thereof, fusion proteins comprising an antibody portion, and any other modified<br>
configuration of the immunoglobulin molecule that comprises an antigen recognition site of the<br>
required specificity. The antibodies may be murine, rat, human, or any other origin (including<br>
humanized antibodies). Small epitope antibodies may be produced by a number of methods<br>
known in the art, including, for example, production by a hybridoma, recombinant production, or<br>
chemical synthesis.<br>
[00134] Generally, a small epitope antibody binds a short, linear peptide epitope of 3, 4, or 5<br>
sequential (consecutive) amino acids. Alternatively, in some embodiments, a small epitope<br>
antibody binds a discontinuous amino acid sequence witliin a polypeptide. In some<br>
embodiments, a small epitope antibody binds an epitope consisting of or consisting essentially of<br>
about any of 2, 3,4, 5, 6, 7, 8, 9, or 10 amino acids. In some embodiments, a small epitope<br>
antibody binds an epitope consisting of or consisting essentially of 2 to 10, 3 to 8, or 3 to 5<br>
amino acids. In some embodiments, a small epitope antibody binds an epitope consisting of or<br>
consisting essentially of less than about any of 10, 9, 8, 7, 6, 5, 4, or 3 amino acids. In some<br>
embodiments, a population of small epitope antibodies binds epitopes consisting of or consisting<br>
essentially of about 3 to about 5 amino acids. In some embodiments, a population of small •<br>
epitope antibodies binds epitopes consisting of or consisting essentially of 2 to 10, 3 to 8, or 3 to<br>
5 amino acids. In some embodiments, a population of small epitope antibodies binds epitopes<br>
consisting of or consisting essentially of about any of 2,3,4,5,6, 7, 8,9, or 10 amino acids. In<br>
some embodiments, a population of small epitope antibodies binds epitopes consisting of or<br>
consisting essentially of less than about any of 10, 9, 8, 7, 6, 5, 4, or 3 amino acids. A population<br>
of small epitope antibodies comprises a plurality of small epitope antibodies. In one<br>
embodiment, the plurality of small epitope antibodies binds epitopes of the same number of<br>
amino acids. In other embodiments, the plurality of small epitope antibodies binds epitopes of a<br>
mixture of different numbers of amino acids. In any of the embodiments described herein, an<br>
epitope may be a sequential or discontinuous sequence within a polypeptide, as described below.<br>
In some embodiments, one or more small epitope antibody(ies) may be comprised within a<br>
mixture of antibodies that comprises antibodies that bind to epitopes larger that the epitopes<br>
recognized by the one or more small epitope antibody(ies).<br><br>
[00135] In some embodiments, the small epitope antibody binds an epitope consisting of or<br>
consisting essentially of 3 sequential amino acids (termed a 3mer), four sequential amino acids<br>
(termed a 4mer), or five sequential amino acids (termed a 5mer). In other embodiments, the<br>
small peptide antibody binds a small "discontinuous" or "degenerate" linear peptide sequence,<br>
such as the linear peptide sequence YCxC, wherein x represents any of the 20 natural amino<br>
acids (a degenerate linear sequence). In other embodiments, the small epitope antibody binds a<br>
non-sequential (discontinuous) sequence within a polypeptide based on conformational<br>
proximity of amino acids within the polypeptide to form the epitope (for example, a<br>
conformational epitope formed by proximity of amino acid residues due to secondary structure<br>
within a folded polypeptide). In still other embodiments, the small epitope antibody may bind an<br>
epitope consisting of an amino acid sequence that is predicted to be antigenic, using methods<br>
well known in the art for predicting antigenicity. Antibodies that bind small linear peptide<br>
epitopes have been previously described, as shown in Table 2, below. In some embodiments,<br>
the same antibody may bind a sequential sequence on one or more proteins and a discontinuous<br>
sequence on one or more proteins.<br>
[00136] Small epitope antibodies generally recognize a multiplicity of proteins that comprise<br>
the small epitope to which the antibody binds. In some embodiments, the small epitope antibody<br>
binds to an epitope present one or more times in about any of 0.1%, 0.5%, 1,2%, 3%, 4%, 5%,<br>
10%, or more of proteins in a sample. In still other embodiments, the small epitope antibody<br>
binds to an epitope present one or more times in about 0.1 % to 1 % of proteins in a sample. In<br>
still other embodiments, the small epitope antibody binds to an epitope present one or more times<br>
in approximately 1-5% of proteins in a sample. In still other embodiments, the small epitope<br>
antibody binds to an epitope present one or more times in about 0.1% to 1 % of proteins in a<br>
sample, wherein the small antibody epitope binds to a linear peptide epitope consisting of or<br>
consisting essentially of 3 amino acids, 4 amino acids or 5 amino acids. In still other<br>
embodiments, the small epitope antibody binds to an epitope present one or more times in about<br>
1-5% of proteins in a sample, wherein the small antibody epitope binds to a linear peptide<br>
epitope consisting of or consisting essentially of 3 amino acids, 4 amino acids or 5 amino acids.<br>
In still other embodiments, the small epitope antibody binds to an epitope present one or more<br>
times in about 5-7% or about 5-10% of proteins in a sample, wherein the small antibody epitope<br>
binds to a linear peptide epitope consisting or consisting essentially of 3 amino acids, 4 amino<br>
acids or 5 amino acids. In some embodiments, a plurality of small epitope antibodies<br><br>
collectively bind to one or more epitopes present one or more times in about any of at least about<br>
any of 0.1%, 0.5%, 1, 2%, 3%, 4%, 5%, 10%, or more of proteins in a sample. In some<br>
embodiments, a plurality of small epitope antibodies binds to an epitope present one or more<br>
times in about any of 0.1 to 1%, 1 to 5%, 5 to 7%, or 5 to 10% of proteins in a sample. Methods<br>
for empirically assessing frequency of an epitope in a sample include: assessment using<br>
biochemical approaches, such as binding of an antibody followed by analysis using, for example,<br>
2D gels or mass spectrometry, and sequence based analysis, using, for example, amino acid or<br>
nucleic acid sequence databases such as GenBank and SwissProt. Suitable databases are further<br>
described herein.<br>
[00137] In some embodiments, the epitope recognized by a small epitope antibody further<br>
comprises a C-terminal amino acid recognized as a cleavage site by an endopeptidase. For<br>
example, the epitope could comprise a C-terminal arginine and/or a lysine, which are each<br>
recognized by trypsin as a cleavage site. Following endopeptidase digestion of a protein<br>
mixture, the amino acid recognized by the endopeptidase is generally found at the C-terminus of<br>
the target peptide; accordingly, an epitope encompassing such an amino acid will also be found<br>
at the C-terminus of a target polypeptide, which may increase immunogeniciry, and increase the<br>
binding energy associated with antibody-target peptide binding.<br>
[00138] In some embodiments, the small epitope antibody binds its cognate epitope with an<br>
affinity of a binding reaction of at least about 10"7 M, at least about 10"8 M, or at least about 10"9<br>
M, or lower. Binding affinity may be measured by well-known methods in the art, including, for<br>
example, by surface plasmon resonance (Malmborg and Borrebaeck (1995) J. Immunol. Methods<br>
183(1):7-13; Lofas and Johnsson (1990) J. Chem. Soc. Chem. Commun. 1526. In some<br>
embodiments, a binding interaction will discriminate over adventitious binding interactions in<br>
the reaction by at least two-fold, at least five-fold, at least 10- to at least 100-fold or more.<br>
[00139] One, or a population of more than one (such as about two, about three, about four,<br>
about five, about ten, about twenty, about one hundred or more) small epitope antibod(ies) may<br>
be used in the methods of the invention. Thus, in some embodiments, the methods comprise use<br>
of at least one small epitope antibody. In other embodiments, the methods comprise use of at<br>
least two small epitope antibodies. In still other embodiments, at least about 5, about 10, about<br>
20, about 30, about 40, about 50, about 60, about 75, about 100, about 125, about 150, about 200<br>
about, 300, about 400, about 500, about 750, about 1000, or more small epitope antibodies are<br>
used in the methods of the invention. In some embodiments, the sample is contacted with less<br><br>
than about 100, about 95, about 90, about 85, about 80, about 75, about 70, about 65, about 60,<br>
about 55, about 50, about 45, about 40, about 35, about 30, about 25, about 20, about 15, about<br>
10, about 5, or fewer small epitope antibodies. In some embodiments, the sample is contacted<br>
with at least about 20, about 30, about 40, about 50, about 75, about 100, about 500, about 1000,<br>
or more small epitope antibodies. In some embodiments, a sample is contacted with at least<br>
about any of 5,10,20,30,40,50,60,75,100,125,150,200,300,400,500, or 750 small epitope<br>
antibodies, with an upper limit of about any of 10, 20, 30, 40, 50, 75,100, 125,150,200,300,<br>
400, 500, 750, or 1000 small epitope antibodies. It is understood that mixture of small epitope<br>
antibodies and other protein binding agents (such as antibodies that are not small epitope<br>
antibodies) may be used.<br>
[00140] It is understood that the identity (sequence) of the epitope(s) to which the small<br>
epitope antibody(ies) may be used in combination with any of the methods described herein to,<br>
e.g., identify proteins. In some embodiments, the small epitope identity is known. In other<br>
embodiments, the identity of the epitope is predictable using methods known in the art.<br>
[00141] As discussed herein, antibodies may be contacted with the sample one at a time or in<br>
groups of two or more antibodies. In some embodiments, contacting is serial (sequential or<br>
iterative), e.g., a single antibody or group of antibodies is contacted with the sample and<br>
separated, and a second antibody or group of antibodies is contacted with the sample and<br>
separated. In other embodiments, contacting is in parallel, e.g., a group of antibodies is<br>
contacted with the sample and separated. It is appreciated that contacting may be both in parallel<br>
and serial, as when different groups of antibodies are serially contacted with a sample. Groups<br>
of antibodies may be overlapping in composition (e.g., group 1= antibody A, B, C, D,; group 2 =<br>
antibody B, C, D, E, etc.).<br>
[00142] It is evident that the number of small epitope antibodies that are useful in the methods<br>
of reducing complexity of a sample depends on the use, application, and/or subsequent analysis<br>
contemplated for the protein prepared using one or more small epitope antibodies. In some<br>
applications, such as detection of a protein(s) comprising a cognate epitope recognized by a<br>
small antibody, a single small epitope antibody (or, in some embodiments, a small number of<br>
small epitope antibodies) may be used to prepare, purify and/or enrich a fraction of protein(s)<br>
that comprises the protein for which subsequent detection (or other analysis) is desired. Then,<br>
the separated protein can be subjected to further analysis. In other embodiments, use of a set of<br>
two or more small epitope antibodies may be useful. For example, in applications such as<br><br>
protein discovery and, in some embodiments, expression profiling, it may be desirable to use a<br>
multiplicity of small epitope antibodies, such that a large multiplicity of proteins (such as<br>
essentially all protein in the starting sample) will be enriched and/or purified. Use of a<br>
multiplicity of small epitope antibodies is also useful in application in which purification and/or<br>
enrichment of new protein(s) or protein forms is desired (for example, because information<br>
regarding target protein sequence is unknown). As an illustrative example relating to<br>
embodiments involving fractionation of a multiplicity of proteins in a sample (such as essentially<br>
all proteins in a sample) shown, knowledge of the sequence and/or the length of the cognate<br>
amino acid epitope recognized by the small epitope antibody permits an estimate regarding the<br>
expected frequency of the epitope(s) recognized by the small epitope antibody(ies) within the<br>
protein sample. As shown in Table 1, there are a total of 8,000 (203), 160,000 (204) and<br>
3,200,000 (205) random combinations for 3mer, 4mer and 5mer linear peptide sequences,<br>
respectively. Considering 500 amino acids as an average length of protein, the probability that it<br>
is detected by a single anti-3mer antibody is 0.0625, the probability increases to about 1 when 15<br>
anti-3mer antibodies are used, and the probability increases to 6.25 when 100 anti-3mer<br>
antibodies are used. Such calculations are routine. A small epitope antibody may also recognize<br>
a degenerate linear epitope, for example a short peptide, such as YCxC, where x represents two<br>
or more of the 20 standard amino acids.<br><br><br>
[00143] Thus, it is understood that the number of small epitope antibodies useful in the<br>
methods of the invention depends on various factors, including, for example, the use, application,<br>
and/or subsequent analysis contemplated for the protein fraction bound by the small epitope<br>
antibody(ies), complexity of the sample (in terms of number of expected or estimated or<br>
previously determined proteins, including protein variants such as splice variants), average size<br>
of the proteins in the sample, frequency that the cognate epitope is present or predicted to be<br>
present in a sample, binding affinity and/or specificity of the small epitope antibody(ies);<br>
knowledge of target protein(s), and stability of the small epitope antibody. Such factors are well<br>
known in the art and are further discussed herein.<br>
[00144] Antibodies that bind small linear peptide epitopes have been previously described, as<br>
shown in Table. 2.<br><br><br>
[00145] Methods of making small epitope antibodies are known in the art. In another aspect,<br>
and as exemplified in the Examples, small epitope antibodies (e.g., human, humanized, mouse,<br>
chimeric) may be made by using immunogens which express one or more small peptide epitopes,<br>
such as a small linear peptide epitope consisting of or consisting essentially of 3,4, or 5 amino<br>
acids.<br>
[00146] Immunogens may be produced, for example, by chemical synthesis. Methods for<br>
synthesizing polypeptides are well known in the art. In some embodiments, the polypeptide<br>
immunogen is synthesized with a terminal cysteine to facilitate coupling to either KLH or BSA,<br><br>
as is known in the art. The terminal cysteine can be incorporated at the amino terminus of the<br>
polypeptide (which may minimize steric effects during immunization and screening), or at the<br>
carboxy terminus. In other embodiments, the polypeptide immunogen is synthesized as a<br>
multiple antigen polypeptide, or MAP.<br>
[00147] The route and schedule of immunization of the host animal are generally in keeping<br>
with established and conventional techniques for antibody stimulation and production, as further<br>
described herein. General techniques for production of human and mouse antibodies are known<br>
in the art and are described herein. Typically, the host animal is inoculated intraperitoneally with<br>
an amount of immunogen, including as described herein.-<br>
[00148] Hybridomas can be prepared from the lymphocytes and immortalized myeloma cells<br>
using the general somatic cell hybridization technique of Kohler, B. and Milstein, C. (1975)<br>
Nature 256:495-497 or as modified by Buck, D. W. et al, (1982) In Vitro, 18:377-381.<br>
Available myeloma lines, including but not limited to X63-Ag8.653 and those from the Salk<br>
Institute, Cell Distribution Center, San Diego, Calif., USA, may be used in the hybridization.<br>
Generally, the technique involves fusing myeloma cells and lymphoid cells using a fusogen such<br>
as polyethylene glycol, or by electrical means well known to those skilled in the art. After the<br>
fusion, the cells are separated from the fusion medium and grown in a selective growth medium,<br>
such as hypoxanthine-aminopterin-thymidine (HAT) medium, to eliminate unhybridized parent<br>
cells. Any of the media described herein, supplemented with or without serum, can be used for<br>
culturing hybridomas that secrete monoclonal antibodies. As another alternative to the cell'<br>
fusion technique, EBV immortalized B cells may be used to produce the small epitope antibodies<br>
of the subject invention. The hybridomas are expanded and subcloned, if desired, and<br>
supernatants are assayed for anti-immunogen activity by conventional immunoassay procedures<br>
{e.g., radioimmunoassay, enzyme immunoassay, or fluorescence immunoassay).<br>
[00149] Hybridomas or progeny cells of the parent hybridomas that produce small epitope<br>
antibodies (such as monoclonal antibodies) may be used as source of antibodies or derivatives<br>
thereof, or a portion thereof.<br>
[00150] Hybridomas that produce such antibodies may be grown in vitro or in vivo using<br>
known procedures. The monoclonal antibodies may be isolated from the culture media or body<br>
fluids, by conventional immunoglobulin purification procedures such as ammonium sulfate<br>
precipitation, gel electrophoresis, dialysis, chromatography, and ultrafiltration, if desired.<br>
Undesired activity if present, can be removed, for example, by running the preparation over<br><br>
adsorbents made of the immunogen attached to a solid phase and eluting or releasing the desired<br>
antibodies off the immunogen. Immunization of a host animal with a human or other species of<br>
small epitope receptor, or a fragment of the human or other species of small epitope receptor, or<br>
a human or other species of small epitope receptor or a fragment containing the target amino acid<br>
sequence conjugated to a protein that is immunogenic in the species to be immunized, e.g.,<br>
keyhole limpet hemocyanin, serum albumin, bovine thyroglobulin, or soybean trypsin inhibitor<br>
using a bifunctional or derivatizing agent, for example maleimidobenzoyl sulfosuccinimide ester<br>
(conjugation through cysteine residues), N-hydroxysuccinimide (through lysine residues),<br>
glutaradehyde, succinic anhydride, SOC12, or R1N=C=NR, where R and Rl are different alkyl<br>
groups can yield a population of antibodies (e.g., monoclonal antibodies).<br>
[00151] If desired, the small epitope antibody (monoclonal or polyclonal) of interest may be<br>
sequenced and the polynucleotide sequence may then be cloned into a vector for expression or<br>
propagation. The sequence encoding the antibody of interest may be maintained in vector in a<br>
host cell and the host cell can then be expanded and frozen for future use. In an alternative, the<br>
polynucleotide sequence may be used for genetic manipulation to "humanize" the antibody or to<br>
improve the affinity, or other characteristics of the antibody. For example, the constant-region<br>
may be engineered to more resemble human constant regions to avoid immune response if the<br>
antibody is used in clinical trials and treatments in humans. It may be desirable to genetically<br>
manipulate the antibody sequence to obtain greater affinity to the small epitope and/or greater<br>
and/or altered specificity to the small epitope. It will be apparent to one of skill in the art that<br>
one or more polynucleotide changes can be made to the small epitope antibody and still maintain<br>
its binding ability to the small epitope.<br>
[00152] A number of "humanized" antibody molecules comprising an antigen-binding site<br>
derived from a non-human immunoglobulin have been described, including chimeric antibodies<br>
having rodent or modified rodent V regions and their associated complementarity determining<br>
regions (CDRs) fused to human constant domains. See, for example, Winter et al. Nature<br>
349:293-299 (1991), Lobuglio et al.(l989)Proc. Nat. Acad. Sci. USA 86:4220-4224, Shaw et al.<br>
(1987) J Immunol. 138:4534-4538, and Brown et al. (1987) Cancer Res. 47:3577-3583. Other<br>
references describe rodent CDRs grafted into a human supporting framework region (FR) prior<br>
to fusion with an appropriate human antibody constant domain. See, for example, Riechmann et<br>
al. (1988) Nature 332:323-327, Verhoeyen et al. Science (1988) 239:1534-1536, and Jones etal.<br>
Nature (1986) 321:522-525. Another reference describes rodent CDRs supported by<br><br>
recombinantly veneered rodent framework regions. See, for example, European Patent<br>
Publication No. 519,596. These "humanized" molecules are designed to minimize unwanted<br>
immunological response toward rodent anti-human antibody molecules which limits the duration<br>
and effectiveness of therapeutic applications of those moieties in human recipients. For<br>
example, the antibody constant region can be engineered such that it is immunologically inert<br>
{e.g., does not trigger complement lysis). See, e.g. PCT/GB99/01441; UK Patent Application<br>
No. 9809951.8. There are four general steps to humanize a monoclonal antibody. These are:<br>
(1) determining the nucleotide and predicted amino acid sequence of the starting antibody light<br>
and heavy variable domains (2) designing the humanized antibody, i.e., deciding which antibody<br>
framework region to use during the humanizing process (3) the actual humanizing<br>
methodologies/techniques and (4) the transfection and expression of the humanized antibody.<br>
See, for example, U.S. Patent Nos. 4,816,567; 5,807,715; 5,866,692; 6,331,415; 5,530,101;<br>
5,693,761; 5,693,762; 5,585,089; and 6,180,370. Other methods of humanizing antibodies that<br>
may also be utilized are disclosed by Daugherty et ah, Nucl. Acids Res. (1991) 19:2471-2476 and<br>
in U.S. Patent Nos. 6,180,377; 6,054,297; 5,997,867; 5,866,692; 6,210,671; 6,350,861; andPCT<br>
Publication No. WO 01/27160.<br>
[00153] In yet another alternative, fully human antibodies may be obtained by using<br>
commercially available mice that have been engineered to express specific human<br>
immunoglobulin proteins. Transgenic animals that are designed to produce a more desirable<br>
(e.g., fully human antibodies) or more robust immune response may also be used for generation<br>
of humanized or human antibodies. Examples of such technology are Xenomouse ™ from<br>
Abgenix, Inc. (Fremont, CA) and HuMAb-Mouse® and TC Mouse™ from Medarex, Inc.<br>
(Princeton, NJ).<br>
[00154] In an alternative, antibodies may be made recombinantly and expressed using any<br>
method known in the art. In another alternative, antibodies may be made recombinantly by<br>
phage display technology. See, for example, U.S. Patent Nos. 5,565,332; 5,580,717; 5,733,743<br>
and 6,265,150; and Winter etal.( 1994) Annu. Rev. Immunol. 12:433-455.<br>
[00155] Alternatively, the phage display technology (McCafferty et al.(l990) Nature<br>
348:552-553) can be used to produce human antibodies and antibody fragments in vitro, from<br>
immunoglobulin variable (V) domain gene repertoires from unimmunized donors. For example,<br>
existing antibody phage display libraries may be panned in parallel against a large collection of<br>
synthetic polypeptides. According to this technique, antibody V domain genes are cloned in-<br><br>
frame into either a major or minor coat protein gene of a filamentous bacteriophage, such as Ml 3<br>
or fd, and displayed as functional antibody fragments on the surface of the phage particle.<br>
Because the filamentous particle contains a single-stranded DNA copy of the phage genome,<br>
selections based on the functional properties of the antibody also result in selection of the gene<br>
encoding the antibody exhibiting those properties. Thus, the phage mimics some of the<br>
properties of the B cell. Phage display can be performed in a variety of formats; for review see,<br>
e.g., Johnson, Kevin S. and Chiswell, David J., Current Opinion in Structural Biology (1993) 3,<br>
564-571. Several sources of V-gene segments can be used for phage display. Clackson et al.,<br>
Nature (1991) 352:624-628 isolated a diverse array of anti-oxazolone antibodies from a small<br>
random combinatorial library of V genes derived from the spleens of immunized mice. A<br>
repertoire of V genes from unimmunized human donors can be constructed and antibodies to a<br>
diverse array of antigens (including self-antigens) can be isolated essentially following the<br>
techniques described by Mark et al (1991) J. Mol. Biol. 222:581-597, or Griffith et al (1993)<br>
EMBO J. 12:725-734. In a natural immune response, antibody genes accumulate mutations at a<br>
high rate (somatic hypermutation). Some of the changes introduced will confer higher affinity,<br>
and B cells displaying high-affinity surface immunoglobulin are preferentially replicated and<br>
differentiated during subsequent antigen challenge. This natural process can be mimicked by<br>
employing the technique known as "chain shuffling." Marks, et al. (1992) Bio/Technol. 10:779-<br>
783. In this method, the affinity of "primary" human antibodies obtained by phage display can<br>
be improved by sequentially replacing the heavy and light chain V region genes with repertoires<br>
of naturally occurring variants (repertoires) of V domain genes obtained from unimmunized<br>
donors. This technique allows the production of antibodies and antibody fragments with<br>
affinities in the pM-nM range. A strategy for making very large phage antibody repertoires (also<br>
known as "the mother-of-all libraries") has been described by Waterhouse et al. (1993) Nucl.<br>
Acids Res. 21:2265-2266. Gene shuffling can also be used to derive human antibodies from<br>
rodent antibodies, where the human antibody has similar affinities and specificities to the starting<br>
rodent antibody. According to this method, which is also referred to as "epitope imprinting", the<br>
heavy or light chain V domain gene of rodent antibodies obtained by phage display technique is<br>
replaced with a repertoire of human V domain genes, creating rodent-human chimeras. Selection<br>
on antigen results in isolation of human variable regions capable of restoring a functional<br>
antigen-binding site, i.e., the epitope governs (imprints) the choice of partner; When the process<br>
is repeated in order to replace the remaining rodent V domain, a human antibody is obtained (see<br><br>
PCT Publication No. WO 9306213, published April 1,1993). Unlike traditional humanization of<br>
rodent antibodies by CDR grafting, this technique provides completely human antibodies, which<br>
have no framework or CDR residues of rodent origin. It is apparent that although the above<br>
discussion pertains to humanized antibodies, the general principles discussed are applicable to<br>
customizing antibodies for use, for example, in dogs, cats, primates, equines and bovines.<br>
[00156] Antibodies may be made recombinantly by first isolating the antibodies made from<br>
host animals, obtaining the gene sequence, and using the gene sequence to express the antibody<br>
recombinantly in host cells (e.g., CHO cells). Another method that may be employed is to<br>
express the antibody sequence in plants (e.g., tobacco), transgenic milk, or in other organisms.<br>
Methods for expressing antibodies recombinantly in plants or milk have been disclosed. See, for<br>
example, Peeters et ah (2001) Vaccine 19:2756; Lonberg, N. and D. Huszar (1995) Int. Rev.<br>
Immunol 13:65; and Pollock et ah (1999) J Immunol Methods 231:147. Methods for making<br>
derivatives of antibodies, e.g., humanized, single chain, etc. are known in the art.<br>
[00157] Immunoassays and flow cytometry sorting techniques such as fluorescence activated<br>
cell sorting (FACS) can also be employed to isolate antibodies that are specific for the desired<br>
small epitope.<br>
[00158] The antibodies can be bound to many different carriers. Carriers can:be active and/or<br>
inert. Examples of well-known carriers include polypropylene, polystyrene, polyethylene,<br>
dextran, nylon, amylases, glass, natural and modified celluloses, polyacrylamides, agaroses and<br>
magnetite. The nature of the carrier can be either soluble or insoluble for purposes of the<br>
invention. Those skilled in the art will know of other suitable carriers for binding antibodies, or<br>
will be able to ascertain such, using routine experimentation.<br>
[00159] DNA encoding small epitope antibodies may be isolated and sequenced, as is known<br>
in the art. Generally, the monoclonal antibody is readily isolated and sequenced using<br>
conventional procedures (e.g., by using oligonucleotide probes that are capable of binding<br>
specifically to genes encoding the heavy and light chains of the monoclonal antibodies). The<br>
hybridoma cells serve as a preferred source of such cDNA. Once isolated, the DNA may be<br>
placed into expression vectors, which are then transfected into host cells such as E. coli cells,<br>
simian COS cells, Chinese hamster ovary (CHO) cells, or myeloma cells that do not otherwise<br>
produce immunoglobulin protein, to obtain the synthesis of monoclonal antibodies in the<br>
recombinant host cells. The DNA also may be modified, for example, by substituting the coding<br>
sequence for human heavy and light chain constant domains in place of the homologous murine<br><br>
sequences, Morrison etal (1984) Proc. Nat. Acad. Sci. 81: 6851, or by covalently joining to the<br>
immunoglobulin coding sequence all or part of the coding sequence for a non-immunoglobulin<br>
polypeptide. In that manner, "chimeric" or "hybrid" antibodies are prepared that have the<br>
binding specificity of a small epitope antibody (such as a monoclonal antibody) herein.<br>
[00160] Small epitope antibodies may be characterized using methods well-known in the art,<br>
some of which are described in the Examples. For example, one method is to identify the<br>
epitope to which it binds, including solving the crystal structure of an antibody-antigen complex,<br>
competition assays, gene fragment expression assays, and synthetic polypeptide-based assays, as<br>
described, for example, in Chapter 11 of Harlow and Lane, Using Antibodies, a Laboratory<br>
Manual, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York, 1999. In an<br>
additional example, epitope mapping can be used to determine the sequence to which a small<br>
epitope antibody binds. Epitope mapping is commercially available from various sources, for<br>
example, Pepscan Systems (Edelhertweg 15, 8219 PH Lelystad, The Netherlands). Polypeptides<br>
of varying lengths (e.g., at least 4-6 amino acids long) can be isolated or synthesized (e.g.,<br>
recombinantly) and used for binding assays with an anti-small epitope antibody. In another<br>
example, the epitope to which the small epitope antibody binds can be determined in a<br>
systematic screening by using overlapping polypeptides derived from the small epitope<br>
extracellular sequence and determining binding by the small epitope antibody. Certain epitopes<br>
can also be identified by using large libraries of random polypeptide sequences displayed on the<br>
surface of phage particles (phage libraries), as is well known in the art.<br>
[00161] Yet another method which can be used to characterize an anti-small epitope antibody<br>
is to use competition assays with other antibodies known to bind to the same antigen, i.e., to<br>
determine if the anti-small epitope antibody binds to the same epitope as other antibodies.<br>
Competition assays are well known to those of skill in the art.<br>
[00162] The small epitope antibodies useful in this invention may be linked to a labeling agent<br>
(alternatively termed "label") such as a fluorescent molecule (such as a hapten or fluorescent<br>
bead), a binding partner, a solid support, or other agents to facilitate separation that are known in<br>
the art. Such agents are further described herein.<br>
[00163] In some embodiments, one or more of the following considerations are used in the<br>
design of small epitope antibodies (whether designed to be used singly or in a population) that<br>
result in an epitope frequency with sufficient redundancy to yield optimal coverage of the<br>
proteins present in a sample. In one embodiment, a group of small epitope antibodies designed<br><br>
according to one or more of the following considerations is capable of binding to cognate<br>
epitopes on at least about any of 10, 20, 30,40, 50, 60, 70, 80, 90, or 95% of the proteins in a<br>
sample.<br>
•	Epitope size: Small epitope antibodies that are small enough to occur frequently in the<br>
proteome but large enough to confer sufficient binding energy when recognized by their<br>
cognate epitope. In some embodiments, the epitope size recognized by each antibody is<br>
3,4, or 5'amino acids.<br>
•	Epitope abundance: In an embodiment, optimal epitope abundance enables each small<br>
epitope antibody to bind to about 100 to about 150 serum-derived polypeptides of about<br>
20 to about 100 amino acids in length. This abundance level matches the resolving<br>
power of most mass spectrometers without requiring MS-MS and collision-induced<br>
dissociation (CID). Epitopes of the appropriate abundance are preferable for<br>
achievement of the desired MS resolution and sensitivity.<br>
•	Sampling redundancy: In some embodiments, a sufficiently large collection of small<br>
epitope antibodies is used to permit binding to about 3 to about 5 epitopes per protein per<br>
proteome of interest. This design feature provides for sampling redundancy to<br>
accommodate the variability expected in both expression levels for different proteins and<br>
binding efficiency for each antibody in the collection.<br>
•	Affinity: In some embodiments, the tightness of binding between small epitope<br>
antibodies and their epitopes affects the sensitivity of protein profiling. In some<br>
embodiments, each antibody in a collection binds with high enough affinity to ensure<br>
that sufficient analyte is captured for MS analysis.<br>
•	Frequency of Binding: In some embodiments, frequency of binding of small epitope<br>
antibodies is high so that peptides present within each bound peptide fraction contain a<br>
common epitope. This provides sampling redundancy and facilitates bioinformatic<br>
determination of peptide identity.<br>
Contacting the sample with a small epitope antibody and separation of protein from a<br>
protein-antibody complex<br>
[00164] Methods and conditions for contacting an antibody with a protein in a sample are well<br>
known in the art. Antibodies may be contacted with the sample one at a time or in groups of two<br><br>
or more antibodies). In some embodiments, contacting is serial (sequential, or iterative), e.g., a<br>
single antibody or group of antibodies is contacted with the sample; separated; and a second<br>
antibody or group of antibodies is contacted with the sample, and separated, and so on). In other<br>
embodiments, contacting is in parallel, e.g., a group of antibodies is contacted with the sample,<br>
and separated. It is appreciated that contacting may be both in parallel and serial, as when<br>
different groups of antibodies are serially contacted with a sample. Groups of antibodies may be<br>
overlapping in composition (e.g., group 1= antibody A, B, C, D; group 2 = antibody B, C, D, E,<br>
etc.) or different in composition. Contacting of an antibody with protein may occur with both<br>
antibody and protein in a liquid medium or may occur with one component (antibody or protein)<br>
bound or associated with a solid support and the other component in a liquid medium. In one<br>
embodiment, a liquid (e.g., aqueous) protein containing sample is contacted with a small epitope<br>
antibody that is bound or associated with a solid support.<br>
[00165] In some embodiments involving parallel contacting, it is desirable for small epitope<br>
antibodies to be individually separable, for example, by linking the antibody to detectable<br>
distinct beads, use of individually separable binding partners, immobilization of antibody in, e.g.,<br>
different wells of a multiwell plate, use of antibody arrays, and the like. Insofar as the small<br>
epitope bound by the antibody is known, binding by a small epitope antibody provides<br>
information relating to amino acid content and/or sequence of protein(s) bound by the small<br>
epitope antibody. In embodiments wherein knowledge of the cognate small epitope is desired, it<br>
may be convenient to individually separate the small antibodies (such that the protein bound by<br>
each small epitope antibody is kept separate). However, individual separation or separability is<br>
not required in every embodiment. For example, small epitope antibodies may be combined in<br>
small pools of two or more antibodies that possess overlapping antibody composition, such as (1)<br>
antibodies ABC; (2) antibodies CDE; (3) antibodies FGH, and (4) antibodies HIJ. Following<br>
separation of antibody-protein complexes, and separation of antibody from antibody-protein<br>
complexes, information regarding presence or absence of a particular small epitope may be<br>
inferred based on membership in a particular group.<br>
[00166] To facilitate separation of the antibody-protein complex from unbound protein in the<br>
sample, the antibody may be linked to an agent that facilitates separation, such as a binding<br>
partner (e.g., biotin, oligonucleotide, aptamer), a solid support (such as a bead or matrix,<br>
including a microarray or multiwell plate); or any other agent known in the art. Linking may be<br>
covalent or noncovalent, and may be direct or indirect. Methods for linking antibodies to such<br><br>
agents are well known in the art. See, e.g. Kennedy et al. (1976) Clin. Chim. Acta 70:1-31, and<br>
Schurs et al. (1977) Clin. Chim. Acta 81:1-40 (describing coupling techniques, including the<br>
glutaraldehyde method, the periodate method, the dimaleimide method, the m-maleimidobenzyl-<br>
N-hydroxy-succinimide ester method, all of which methods are incorporated by reference<br>
herein).<br>
[00167J Methods for separating an antibody-protein complex from a sample are known in the<br>
art and include use of a capture agent that binds a binding partner (e.g., avidin to capture a<br>
biotin-linked antibody; an oligonucleotide to capture an oligonucleotide linked to an antibody;<br>
Physical separation may also be used, such as sedimentation, filtration, FACS (for example,<br>
using beads that are labeled with a spectral signature), and magnetic separation (when the<br>
antibody is linked to a matrix with magnetic properties, such as a magnetic bead).<br>
[00168] Many binding partners are known in the art (e.g., a dinitrophenyl group, digoxigenin,<br>
fluorophores, Oregon Green dyes, Alexa Fluor 488 (Molecular Probes), fluorescein, a dansyl<br>
group, Marina Blue (Molecular Probes), tetramethylrhodamine, Texas Red (Molecular Probes),<br>
BODIPY (4,4-difluoro-4-bora-3a,4a-diaza-s-indacene; U.S. Pat. No. 4,774,339) dyes, etc) that<br>
can be used in the present invention. Antibodies that can be used as capture reagents can<br>
specifically bind to binding agents are commercially available from vendors such as Molecular<br>
Probes, Eugene, Oreg. These antibodies include antibodies that can specifically bind to a<br>
dinitrophenyl group, a digoxigenin, a fluorophore, Oregon Green dyes, Alexa Fluor 488<br>
(Molecular Probes), fluorescein, a dansyl group, Marina Blue (Molecular Probes),<br>
tetrahmethylrhodamine, Texas Red (Molecular Probes), and a BODIPY dye (Molecular Probes).<br>
Any suitable ligand and anti-ligand may also be used.<br>
[00169] Oligonucleotides can be used as binding partner and capture reagents.<br>
Oligonucleotides include nucleic acids such as DNA, RNA, and mixed RNA/DNA molecules.<br>
The oligonucleotide that is used as the affinity label should be able to hybridize to the sequence<br>
of the oligonucleotide present on the capture reagent. Those of skill in the art will recognize that<br>
many different oligonucleotide sequences can be designed that will hybridize to each other.<br>
Important considerations for designing such oligonucleotide pairs include the actual nucleotide<br>
sequence, the length of the oligonucleotides, the hybridization conditions (e.g., temperature, salt<br>
concentration, presence of organic chemicals, etc.) and the melting temperature of the<br>
oligonucleotide.<br><br>
[00170] Solid supports suitable for immobilizing (linking) antibodies or proteins from a<br>
sample (and modifications to render solid supports suitable for immobilizing antibodies) are well<br>
known in the art. Examples of a solid support include: a bead (including magnetized beads),<br>
microwell plate, and a protein microarray (e.g., technology owned by Zyomyx, Inc. See, e.g. US<br>
Patent No. 6,365,418). Thus, for example, CdSe-CdS core-shell nanocrystals enclosed in a silica<br>
shell can be easily derivatized for coupling to a biological molecule. Bruchez et al (1998)<br>
Science 281: 2013-2016. Similarly, highly fluorescent quantum dots (zinc sulfide-capped<br>
cadmium selenide) have been covalently coupled to biomolecules for use in ultrasensitive<br>
biological detection. Warren andNie (1998) Science 281: 2016-2018. Fluorescently labeled<br>
beads are commercially available from Luminex and Quantum Dot.<br>
[00171] The bound protein (or in some embodiments, polypeptide fragments) may be released<br>
from the antibody-protein complex using conventional immunoaffinity elution conditions such as<br>
acidic pH, ionic strength, detergents or combinations of the above. Generally, peptide or protein<br>
is de-salted for subsequent fractionation, characterization, or other analysis.<br>
a. Protein cleaving agent<br>
[00172] In some embodiments, the methods of the invention further comprise treating the<br>
sample with a protein cleaving agent, whereby polypeptide fragments are generated. In one<br>
embodiment, the sample is contacted with a protein cleaving agent prior to contacting a sample<br>
with at least one small epitope antibody. In another embodiment, protein is contacted with a<br>
protein cleaving agent after separation of protein from an antibody-protein complex.<br>
[00173] Protein cleaving agent treatment generates protein cleavage fragments (such as<br>
polypeptides), which can facilitate subsequent mass spectral analysis of the amount of protein<br>
and the identity of proteins in a sample(s). In particular, treatment with a protein cleaving agent<br>
treatment can facilitate the analysis of proteins whose molecular masses exceed 25 kDa. Protein<br>
cleaving reagent treatment also may facilitate accessibility and/or access of small epitope<br>
antibodies to a cognate epitope. Protein cleaving agents are well known in the art, and are<br>
further discussed herein. In some embodiments, one protein cleaving agent is used. In other<br>
embodiments, more than one protein cleaving reagent is used. In some embodiments, more than<br>
one type of protein cleaving agent is used with respect to a single sample (e.g., two or more types<br>
of proteases, two or more types of chemical cleavage agents, or a combination of one or more<br><br>
protease and one or more chemical cleavage agent). Conditions for treatment with a protein<br>
cleaving agent are well known in the art.<br>
[00174] In one embodiment, a protein cleaving agent is a protease Example of proteases that<br>
can be used as protein cleaving agents, include, but are not limited to: chymotrypsin, trypsin<br>
(arg, lys cleavage sequence), thermolysin (phe, leu, iso, val cleavage sequence), V8 protease,<br>
Endoproteinase Glu-C, Endoproteinase Asp-N, Endoproteinase Lys-C, Endoproteinase Arg-C,<br>
Endoproteinase Arg-N, Factor Xa protease, thrombin, enterokinase, V5 protease, and the tobacco<br>
etch virus protease. Proteases useful in the methods of the invention can be genetically<br>
engineered and/or chemically modified to prevent autolysis. It is appreciated that an enzymatic<br>
protein cleaving agent (such as a protease) can be modified to facilitate removal of the protease<br>
from the polypeptide cleavage products following polypeptide cleavage. Such modifications are<br>
known in the art and include: (1) bead-bound (e.g., latex, silica or magnetic bead) protease, (2)<br>
haptenated protease, (3) affinity depletion of the protease (with, for example, a bead-bound anti-<br>
protease, or bead-bound non-cleavable substrate) and/or (4) size exclusion chromatography. The<br>
activity of a protease can be inhibited, for example, by treating with heat, a protease inhibitor, a<br>
metal chelator (e.g., EGTA, EDTA), etc.<br>
[00175] In another embodiment, a protein cleaving agent is a chemical cleaving agent, such as<br>
chemical substances and compounds that cleave polypeptides and peptide bonds. Nonlimiting<br>
examples of chemical cleaving agents include cyanogen bromide (which cleaves at methionine<br>
residues), hydroxylamine (which cleaves between an Asn and a Gly residue), and acid pH (which<br>
can cleave an Asp-Pro bond) (see e.g., Ausubel et al., supra).<br>
[00176] In still further embodiments, phosphatases (e.g., alkaline phosphatase, acid<br>
phosphatase, protein serine phosphatase, protein tyrosine phosphatase, protein threonine<br>
phosphatase, etc.), lipases, and other enzymes can be employed as protein cleaving agents.<br>
Sample<br>
[00177] As noted in the definition and as used herein, "sample" encompasses a variety of<br>
sample types and/or origins, such as blood and other liquid samples of biological origin, solid<br>
tissue samples such as a biopsy specimen or tissue cultures or cells derived therefrom, and the<br>
progeny thereof. The definition also includes samples that have been manipulated in any way<br>
after their procurement, such as by treatment with reagents, solubilization, or enrichment for<br><br>
certain components, such as proteins or polynucleotides. The term "sample" encompasses a<br>
clinical sample, and also includes cells in culture, cell supernatants, cell lysates, serum, plasma,<br>
biological fluid, and tissue samples. A sample can be from a microorganism, e.g., bacteria,<br>
yeasts, viruses, viroids, molds, fungi, plants, animals, including mammals such as humans. A<br>
sample may comprise a single cell or more than a single cell. Examples of a sample include<br>
blood, plasma, serum, urine, stool, cerebrospinal fluid, synovial fluid, amniotic fluid, saliva, lung<br>
lavage, semen, milk, nipple aspirate, prostatic fluid, mucous, cheek swabs, and/or tears.<br>
[00178] These samples can be prepared by methods known in the art such as lysing,<br>
fractionation, purification, including affinity purification, FACS, laser capture microdissection<br>
(LCM) or isopycnic centrifugation. In some embodiments, subcellular fractionation methods are<br>
used to create enriched cellular or subcellular fractions, such as subcellular organelles including<br>
nuclei, mitochondria, heavy and light membranes and cytoplasm.<br>
[00179] Prior to contacting the sample with one or more small epitope antibodies, the sample<br>
may be treated with agents capable of denaturing and/or solubilizing proteins, such as detergents<br>
(ionic and non-ionic), chaotropes and/or reducing agent. Such agents are known in the art.<br>
[00180] Under certain circumstances, it may be desirable to remove or minimize abundant<br>
proteins present in a sample, for example, by targeted immunodepletion, or other methods known<br>
in the art. Generally, such removal (or reduction) occurs prior to contacting the sample with one<br>
or more small epitope antibodies (however, such reduction or removal can occur during or after<br>
treatment with small epitope antibodies). Any suitable reagent may be used, including one or<br>
more small epitope antibody(ies). In one embodiment, removal and/or reduction of one or more<br>
sample components is effected by treating the sample with one or more small epitope antibodies.<br>
[00181] In some embodiments, it may be desirable to treat the sample with a polysaccharide<br>
cleaving agent, for example, to reduce, minimize, and or eliminate glycosylation of sample<br>
protein. Removal of any carbohydrate moieties may be accomplished chemically or<br>
enzymatically. Examples of polysaccharide cleaving agents include glycosidases,<br>
endoglycosidases, exoglycosylases, and chemicals such as trifluoromethanesulfonic acid.<br>
Endoglycosidases such as Endoglycosidase H (New England Biolabs, Beverly, Mass.), and Endo<br>
Hf (New England Biolabs) are commercially available. These endoglycosidases cleave the<br>
chitobiose core of high mannose and some hybrid oligosaccharides from N-linked glycoproteins.<br>
Exoglycosidases are also commercially available from vendors such as New England Biolabs<br>
and include, beta-N-Acetylhexosaminidase, alpha- 1-2-Fucosidase, alpha-1-3,4 Fucosidase alpha-<br><br>
1-2,3 Mannosidase, alpha-1-6 Mannosidase, Neuraminidase, alpha-2-3 Neuraminidase, beta 1-3<br>
Galactosidase, and alpha -N-Acetyl-galactosaminidase<br>
[00182] The following Examples are provided to illustrate, but not limit, the invention.<br><br>
EXAMPLES<br>
Example 1: Preparation and characterization of small epitope antibodies<br>
[00183] Five immunization polypeptides in the format of Multiple Antigenic Peptide (MAP)<br>
were designed as shown in Table 3. These sequences in combination were also used to evaluate<br>
cross-reactivity of the induced antibodies, by virtue of the inclusion in different MAPs of the<br>
same sequence in differing locations. Each of the immunization polypeptides was used to<br>
immunize 4 Balb/C mice using standard methods.<br><br>
[00184] Notes to Table 3:<br>
Polypeptide MAPI: HSLFHPEDTGQV: From PSA, amino acids #79-89. KKTTNV:<br>
From Meningococcal Opa protein, containing KTT, a published 3mer antibody epitope<br>
(Malorny, Morelli et al. 1998).<br>
Polypeptide MAP2: Alternate sequences of MAPI.<br>
Polypeptide MAP3: LTPKK: Motif 1 of PSA (Nagasaki, Watanabe et al. 1999).<br>
KKTTNVLTVPTNIPG: From Meningococcal Opa protein, containing two published 3mer<br>
antibody epitopes: KTT and NIP and one 4mer epitope: TNIP (Morelli, et al. (1997) Mol<br>
Microbiol 25(6): 1047-64.<br><br>
Polypeptide MAP4: LTPKK: From PSA, the same as in peptide MAP3.<br>
LTQENQNRGTH: An immunogenic sequence of alpha-1-ACT selected by DNAStar computer<br>
program. IYNQ: From Meningococcal Opa protein, containing a 2mer epitope IY and four<br>
amino acids of a 5mer epitope, TIYNQ and of a 7mer epitope TPTIYNQ (Marelli, et al, id).<br>
Polypeptide MAP5 TIYNTNIPG: From Meningococcal Opa protein (Marelli, et al,<br>
id.). LTQENQNRGTH: The same as in peptide MAP4.<br>
[00185] Two sets of screening polypeptides were designed: (1) 5 C-terminally biotinylated<br>
with the same sequences as the immunization polypeptides (shown in Table 4); and (2) 43 lOmer<br>
biotinylated polypeptides with sequences panning all five immunization polypeptides (shown in<br>
Table 5).<br><br><br><br><br><br>
[00186] After a standard period of immunization, immune serum was collected from each<br>
mouse using standard methods, and tested using ELISA as follows:<br>
ELISA plates (Corning 3369 or similar) were coated with 100µl/well or 50µl/well of<br>
streptavidin (Sigma Catalog No. S4762 or similar, 5ug/ml in 50mM carbonate buffer, pH 9.6).<br>
Plates were incubated at 4°C overnight or at room temperature for 2 hours. Following<br>
incubation, plates were washed 3 times with PBS+0.05% Tween-20 (PBST buffer). Following<br>
washing, plates were blocked with 250µl/well of PBST, and incubated at room temperature for 1<br>
hour, or at 4°C overnight. PBST was removed, and 100µl/well or 50µl/well of a test<br>
biotinylated polypeptide selected from Table 4, at a concentration of 5 |J.g/ml (diluted in PBS)<br>
was added. Plates were incubated for about 30 to 60 min at room temperature. Following<br>
incubation, plates were washed 3 times with PBST. Then, 100µl or 50µl/well of test serum {i.e.,<br>
from test bleeds) was added, and the plates were incubated for one hour at room temperature, or<br>
overnight at 4°C. To titer immunoreactivity, the serum was generally diluted prior to testing to<br><br>
1:500,1:2000,1:8000, or 1:32000. Following incubation, plates were washed 3 times with<br>
PBST. To detect antibody binding, a 1:10,000 dilution of goat anti-mouse IgG (and IgM)-HRP<br>
conjugate (Jackson'Immuno order No. 115-036-071, or similar) was added to each well. Plates<br>
were incubated at room temperature for another hour, then washed 5 times with PBST. HRP<br>
substrate (Sigma Fast OPD,) was added and incubated in the dark at room temperature for 30-60<br>
minutes. Plates were read at OD450 with a 96-well colorimetric detector if HRP reaction was<br>
not stopped. Alternatively, HRP reaction was stopped with 1.25M sulfuric acid, and plates were<br>
read at OD492.<br>
[00187] 12 test bleeds from Groups 1, 2, and 3 mice were tested. No immune response was<br>
observed from mice in groups 1 and 3, and these mice were not studied further. All 4 mice in<br>
group 2 showed strong immune response to screening polypeptide Pep2-0 (&gt;1:32,000). In<br>
addition, immune sera from two of the four mice in group 2 (mice #2-1 and #2-4) showed cross-<br>
reactivity with screening polypeptides designed for groups 1 and 3 due to the sequence<br>
homology between MAP2 and MAP1/MAP3. These results were consistent with mice #2-1 and<br>
#2-4 expressing antibodies mat recognize distinct and concise epitopes present within more than<br>
one screening antigen used in the ELISA assays. A test of the #2-1 and #2-4 sera versus 23<br>
lOmer biotinylated polypeptides that span sequences of all three immunization polypeptides for -<br>
group 1,2 and 3 mice also demonstrated a broad cross-reactivity.<br>
[00188] Eight test bleeds from groups 4-5 were tested by ELISA. Group 4 mice demonstrated<br>
a modest response to their relevant screening polypeptide, Pep4-0, while exhibiting strong cross-<br>
reactivity with Pep3-0, the screening polypeptide designed for group 3. Group 4 mice did not<br>
show substantial cross-reactivity to Pep5-0 even though there is significant sequence identity<br>
between Pep4-0 and Pep5-0. In contrast, 3 of 4 mice in group 5 (mice 5-2, 5-3,5-4) exhibited<br>
robust immunoreactivity to both their screening polypeptide, Pep5-0, and to the related screening<br>
polypeptide, Pep4-0. The sera from the responsive mice in group 5 did not demonstrate<br>
substantial cross-reactivity to the Pep3-0, even though there is a 5 amino acid block of sequence<br>
identity. A test of the #5-2 and 5-3 sera versus 23 lOmer biotinylated polypeptides that span<br>
sequences of all three immunization polypeptides for group 4 and 5 mice demonstrated two<br>
broad but distinctive reaction patterns with the mapping polypeptides spanning sequences of<br>
immunization polypeptides for groups 4 and 5 mice.<br><br>
[00189] Group 2, mice #1 and #4, and Group 5, mice #2 and #3, showed the best immune<br>
responses, as summarized in Table 6 and Figure 1. These mice were selected for hybridoma<br>
fusions.<br><br>
[00190] The animals were sacrificed, the lymph nodes and spleens harvested, then B cell<br>
hybridoma fusions using P3 mouse myeloma cell line as a fusion partner were generated using<br>
standard methods. Fusions were plated and incubated for 11-14 days before screening.<br>
[00191] In the first round of screening, hybridomas from group 2 and 5 mice were analyzed<br>
by ELISA in 96 well plates, essentially as described above, using the corresponding screening<br>
polypeptides, 2-0 and 5-0. Following several rounds of screening, 48 positive hybridoma lines<br>
were identified and transferred to 24 well plates for expansion and additional characterization<br>
including epitope mapping. Of the 48 positive lines, 33 were derived from the Group 2 animals<br>
that received the MAP2 immunogen while the remaining 15 originated from the Group 5<br>
animals. Most of the hybridoma lines (-94%) were the fusion products of B cells harvested from<br>
the spleen. Thirteen of the 48 hybridoma lines expressed IgG, 25 expressed IgM, and the<br>
remaining 10 hybridoma lines were expressing both IgG and IgM or were not expressing either<br>
IgG or IgM and were therefore expressing either IgA or IgE.<br><br>
[00192] In the second round of screening, hybridomas selected for expansion were re-tested<br>
against the relevant screening polypeptide (either polypeptide 2-0 or polypeptide 5-0). 13 of the<br>
48 hybridomas characterized after the 24 well expansion phase exhibited sequence specific<br>
binding to the screening polypeptide 2-0. Other hybridomas bound non-specifically (i.e., bound<br>
a variety of oligopeptide sequences), failed to bind (reflecting either a false positive or clonal<br>
instability and loss during the transfer and subsequent propagation in 24 well plates) or bound<br>
control wells containing BSA.<br>
[00193] The 13 hybridomas that specifically bound to screening polypeptide 2-0 were epitope<br>
mapped using ELISA as described above, using 3 different sets of 10mer C-terminal biotinylated<br>
mapping polypeptides: polypeptides 1-1 to 1-5; 2-1 to 2-9; and 3-1 to 3-9 (see Table 5). 10 of<br>
the 12 hybridoma lines exhibited maximum reactivity with a single mapping polypeptide, 2-1,<br>
and that hybridomas 2.03 and 2.11 showed strong binding to different overlapping sets of<br>
mapping polypeptides, polypeptides 2-1 through 2-3 and 2-7 through 2-9. Because these data<br>
showed strong reactivity to a single mapping polypeptide for most hybridoma lines, we<br>
considered the possibility that steric hindrance associated with immobilization of the mapping , .<br>
polypeptides (specifically, biotin-avidin immobilization) was preventing antibody binding to the<br>
epitope present within a cognate series of 10mers, thus potentially biasing the ELISA epitope<br>
map results. Thus, we evaluated epitope specificity using a competitive binding assay.<br>
[00194] Individual mapping polypeptides were evaluated for their ability to inhibit antibody<br>
binding to the 2-0 screening polypeptide affixed to streptavidin-coated 96 well plates. In this<br>
format, the 10mer mapping polypeptides were not tethered within the binding pocket of<br>
streptavidin and consequently should not be sterically hindered from interacting with a reactive<br>
antibody present within the set of 13 hybridomas. Competition experiments were performed<br>
using standard methods using the 2-0 screening polypeptide affixed to streptavidin-coated 96<br>
well plates and 10mer mapping polypeptide added to each well.<br>
[00195] Using the competitive binding assay, the epitopes recognized by 10 of the 13<br>
hybridomas were determined. Eight of the hybridomas were specific for the epitope PEDTG,<br>
hybridoma 2.03 was specific for epitope DTG and hybridoma 2.11 recognized the epitope<br>
KKTTN. Hybridoma 2.31 exhibited a complex inhibition pattern suggesting that this line is a<br>
mixture of 2 or more specificities and should be subcloned to segregate the individual<br>
reactivities. Finally, hybridomas 1.02 and 2.12 showed poor discrimination in the competitive<br>
inhibition assay. The results of this analysis are summarized in Table 7.<br><br><br><br>
[00196]	The competitive binding assays were repeated twice, and it was confirmed that<br>
hybridoma 2.11 recognized the epitope KTTN, not the epitope KKTTN as suggested in the<br>
preliminary experiments. The epitope competitive binding assays confirmed the epitope<br>
characterization described above for the other, hybridomas. The results of this updated analysis<br>
are summarized in Table 8. Hybridomas 2.03 (also called DA001-2.03), 2.04 (DA001-2.04) and<br>
2.11 (also called DA001 -2.11) are being prepared for deposit at the ATCC.<br><br><br><br>
Example 2. Preparation of Small Epitope Antibodies<br>
[00197] An approach to identify antibodies based on phage display antibody screening was<br>
performed. Five peptide sequences used for the selection of positive antibodies are shown in<br>
Table 9. These sequences in combination were also used to evaluate cross-reactivity of the<br>
selected antibodies.<br><br>
[00198] Note to Table 9: The letter 'X' denotes a mixture of the naturally-occurring L-amino<br>
acids excluding cysteine, methionine, and tryptophan.<br>
[00199] Positives were selected after six rounds of enrichment. The results of phage ELISA<br>
screens against the five screening peptides is shown in Table 10. A total of 96 phage were<br>
screened for P1; 48 were screened for polypeptides P6-P9. In all cases, positive phage were<br>
identified above background.<br><br><br><br><br>
[00200] In a secondary screen of positives identified in the primary screen, a phage ELISA<br>
assay was done against all five polypeptides. Up to five positives were selected for the secondary<br>
screen. Figure 2 shows the results of the most selective clones using this assay. All five positives<br>
yielded significant signal to polypeptide above BSA, and the phage selected from PI<br>
(L50P1-15), Pδ (L50P8-5), and P9 (L50P9_5) appear to show specificity in this semi-<br>
quantitative assay.<br>
[00201] The reactive antibody for L50P 1_15 was subcloned into a vector for bacterial<br>
expression of single chain antibodies. The crude periplasmic preparation was analyzed using a<br>
surface plasmon resonance (SPR) biosensor assay to monitor the formation of complex<br>
association and the dissociation of the protein from immobilized peptides (Malmborg et al,<br>
1995). Figure 3 shows the SPR profile of single chain antibody against the five polypeptides and<br>
BSA. The antibody has the highest affinity for peptide 1, with an estimated Kd of 2xl0"8.<br>
Example 3. Protein profiling and biomarker development<br>
[00202] In one exemplary method for protein profiling, serums derived from healthy and<br>
affected individuals for a particular disease of clinical interest are subjected to: (a) debulking of<br>
the most abundant protein constituents; (b) deglycosylation of the less abundant proteins that<br>
remain; (c) reduction and alkylation of cysteine residues present in the debulked proteome; (d)<br>
digestion of the debulked proteome to completion; (e) fractionation of the resulting peptide<br>
fragments with small epitope antibodies as described above; and (f) comparison of the<br>
composition and relative abundance of peptide constituents from epitope enriched fractions<br>
derived from healthy and affected patients to identify candidate biomarkers associated with a<br>
specific disease.<br>
[00203] Fractionation with small epitope antibodies is performed in parallel with a set of<br>
approximately 100 small epitope antibodies of different specificities. Each antibody is chosen<br>
based on a set of criteria including epitope size, epitope abundance in the serum proteome,<br>
specificity, affinity, and sampling redundancy. The epitopes recognized by the antibodies are<br>
predominantly 3mers, although some are 4mers or 5mers that satisfy the abundance criteria, with<br>
each epitope occurring in 0.5-3% of the constituents of the serum proteome. Each antibody<br>
recognizes its cognate epitope in a context-independent manner and with high affinity. The<br><br>
complete set of small epitope antibodies used for fractionation provides 3-5 fold sampling<br>
redundancy to accommodate the variability expected in both expression levels for different<br>
proteins and capture efficiencies for each antibody in the set.<br>
[00204] Mass spectroscopy is used to analyze the peptide composition and peptide constituent<br>
expression levels for each small epitope antibody fraction. Biomarkers are identified that are<br>
differentially expressed in healthy and diseased individuals. ELISA assays are developed that<br>
can discriminate between healthy and affected individuals based on specific levels of identified<br>
biomarkers present in plasma or serum.<br>
[00205] Although the foregoing invention has been described in some detail by way of<br>
illustration and examples for purposes of clarity of understanding, it will be apparent to those<br>
skilled in the art that certain changes and modifications may be practiced without departing from<br>
the spirit and scope of the invention. Therefore, the description should not be construed as<br>
limiting the scope of the invention, which is delineated by the appended claims.<br>
[00206] All publications, patents, and patent applications cited herein are hereby incorporated<br>
by reference in their entireties for all purposes and to the same extent as if each individual<br>
publication, patent, or patent application were specifically and individually indicated to be so<br>
incorporated by reference.<br><br>
WE CLAIM:<br>
1.	A method for reducing the complexity of a sample that comprises a mixture of<br>
proteins, said method comprising separating a multiplicity of different proteins from said<br>
mixture of proteins, wherein said multiplicity of proteins is bound to a small epitope antibody<br>
in small epitope antibody-protein complexes, wherein the small epitope antibody binds an<br>
epitope consisting of 3 to 5 amino acids, wherein the multiplicity of proteins bound by the<br>
small epitope antibody is enriched; and separating said multiplicity of proteins from said<br>
antibody-protein complexes.<br>
2.	A method as claimed in claim 1, comprising contacting the multiplicity of<br>
proteins separated from the antibody-protein complexes with a protein cleaving agent to form<br>
polypeptide fragments.<br>
3.	A method as claimed in claim 2, wherein the protein cleaving agent<br>
comprises a protease.<br>
4.	A method as claimed in claim 2, wherein the protein cleaving agent<br>
comprises a chemical agent, such as herein described.<br>
5.	A method as claimed in claim 1, wherein said mixture of proteins comprises<br>
a mixture of polypeptide fragments produced by cleavage of proteins in said sample with a<br>
protein cleaving agent<br>
6.	A method as claimed in claim 5, wherein the protein cleaving agent comprises<br>
a protease.<br>
7.	A method as claimed in claim 5, wherein the protein cleaving agent<br>
comprises a chemical agent, such as herein described.<br>
8.	Multiplicity of proteins prepared by the method as claimed in claim 1.<br>
9.	Polypeptide fragments prepared by the method as claimed in claim 2.<br><br>
10.	A method for reducing the complexity of a sample that comprises a mixture of<br>
proteins, said method comprising separating a multiplicity of different proteins from said<br>
mixture of proteins, wherein said multiplicity of proteins is bound to a plurality of different<br>
small epitope antibodies in small epitope antibody-protein complexes, wherein each small<br>
epitope antibody in said plurality of small epitope antibodies binds a different epitope<br>
consisting of 3 to 5 amino acids, wherein said multiplicity of proteins bound by said plurality<br>
of small epitope antibodies is enriched; and separating said multiplicity of proteins from the<br>
antibody-protein complexes.<br>
11.	A method as claimed in claim 10, wherein the small epitope antibodies are<br>
immobilized on a solid matrix.<br>
12.	A method as claimed in claim 10, wherein the small epitope antibodies are<br>
detectably labeled.<br>
13.	A method as claimed in claim 10, wherein said plurality of different small<br>
epitope antibodies comprises at least 100 different small epitope antibodies.<br>
14.	A method for reducing the complexity of a sample that comprises a mixture of<br>
proteins, said method comprising:<br><br>
(a)	contacting the sample comprising a mixture of proteins with at least<br>
one small epitope antibody to form small epitope antibody-protein complexes, wherein a<br>
multiplicity of different proteins from said sample is bound to said small epitope antibody in<br>
small epitope antibody-protein complexes, wherein the small epitope antibody binds an<br>
epitope consisting of 3 to 5 amino acids;<br>
(b)	separating said small antibody-protein complexes from unbound<br>
protein in the sample; and<br>
(c)	separating said multiplicity of proteins from said small antibody-<br>
protein complexes.<br>
15.	A method as claimed in claim 14, wherein steps (a) and (b) are performed<br>
sequentially.<br><br>
16.	A method as claimed in claim 14, wherein steps (a) and (b) are performed<br>
simultaneously.<br>
17.	A method as claimed in claim 14, wherein the at least one small epitope<br>
antibody comprises at least 100 small epitope antibodies.<br>
18.	A method for determining presence or absence of a protein of interest in a<br>
sample, said method comprising detecting the protein of interest, if any, in an enriched<br>
protein fraction, wherein the enriched protein fraction is prepared by the method of claim 1,<br>
and wherein detection of the protein of interest indicates presence of the protein in the<br>
sample.<br>
19.	A method as claimed in claim 18, wherein said detection comprises mass<br>
spectrometry.<br>
20.	A method for determining the amount of a protein of interest in a sample, said<br>
method comprising quantifying the amount of the protein of interest in an enriched protein<br>
fraction, wherein the enriched protein fraction is prepared by the method as claimed in claim<br>
1.<br>
21.	A method as claimed in claim 20, wherein said quantifying comprises mass<br>
spectrometry.<br>
22.	A method for identifying a protein in small epitope antibody-protein<br>
complexes, wherein the small epitope antibody-protein complexes are prepared as claimed<br>
in claim 1.<br>
23.	A method as claimed in claim 22, wherein said identifying comprises mass<br>
spectrometry.<br>
24.	A method for identification of a biomarker, said method comprising<br>
comparing the proteins in the two or more enriched protein fractions, wherein each of the<br>
enriched protein fractions is prepared from a sample by the method as claimed in claim 1.<br><br>
25.	A method as claimed in claim 24, wherein the two or more samples comprise<br>
samples from at least one individual who has a disease condition and at least one individual<br>
who does not have the disease condition, and wherein presence or absence of the biomarker<br>
is indicative of the disease condition.<br>
26.	A kit for use in a method as claimed in any of claims 1-7 and 10-25,<br>
comprising at least one small epitope antibody, wherein said at least one small epitope<br>
antibody binds epitopes consisting of 3 to 5 amino acids.<br>
27.	A kit as claimed in claim 26, wherein said at least one small epitope antibody<br>
is detectably labeled.<br>
28.	A kit as claimed in claim 26 or 27, wherein said at least one small epitope<br>
antibody comprises a plurality of small epitope antibodies.<br>
29.	A kit as claimed in claim 28, wherein said plurality of small epitope<br>
antibodies comprises at least 100 small epitope antibodies.<br>
30.	A method as claimed in claim 10, comprising contacting the proteins<br>
separated from the antibody-protein complexes with a protein cleaving agent to form<br>
polypeptide fragments.<br>
31.	A method as claimed in claim 30, wherein the protein cleaving agent<br>
comprises a protease.<br>
32.	A method as claimed in claim 30, wherein the protein cleaving agent<br>
comprises a chemical agent, such as herein described.<br>
33.	A method as claimed in claim 10, wherein said mixture of proteins comprises<br>
a mixture of polypeptide fragments produced by cleavage of protein in said sample with a<br>
protein cleaving agent.<br>
34.	A method as claimed in claim 33, wherein the protein cleaving agent<br>
comprises a protease.<br><br>
35.	A method as claimed in claim 33, wherein the protein cleaving agent<br>
comprises a chemical agent, such as herein described.<br>
36.	The method as claimed in any one of claims 1,10, and 14, wherein the epitope<br>
to which said small epitope antibody or antibodies bind is present within the amino acid<br>
sequence of each protein in said multiplicity of proteins.<br><br>
The present invention relates generally to methods for reducing the complexity of a sample. More specifically, the<br>
present invention relates to proteomics, the measurement of the protein levels in biological samples, and analysis of proteins in a<br>
sample using antibodies that recognize small epitopes.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTAxLUtPTE5QLTIwMDYtQ09SUkVTUE9OREVOQ0UucGRm" target="_blank" style="word-wrap:break-word;">501-KOLNP-2006-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTAxLUtPTE5QLTIwMDYtRk9STSAyNy5wZGY=" target="_blank" style="word-wrap:break-word;">501-KOLNP-2006-FORM 27.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTAxLUtPTE5QLTIwMDYtRk9STS0yNy5wZGY=" target="_blank" style="word-wrap:break-word;">501-KOLNP-2006-FORM-27.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTAxLWtvbG5wLTIwMDYtZ3JhbnRlZC1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">501-kolnp-2006-granted-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTAxLWtvbG5wLTIwMDYtZ3JhbnRlZC1hc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">501-kolnp-2006-granted-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTAxLWtvbG5wLTIwMDYtZ3JhbnRlZC1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">501-kolnp-2006-granted-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTAxLWtvbG5wLTIwMDYtZ3JhbnRlZC1jb3JyZXNwb25kZW5jZS5wZGY=" target="_blank" style="word-wrap:break-word;">501-kolnp-2006-granted-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTAxLWtvbG5wLTIwMDYtZ3JhbnRlZC1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">501-kolnp-2006-granted-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTAxLWtvbG5wLTIwMDYtZ3JhbnRlZC1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">501-kolnp-2006-granted-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTAxLWtvbG5wLTIwMDYtZ3JhbnRlZC1leGFtaW5hdGlvbiByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">501-kolnp-2006-granted-examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTAxLWtvbG5wLTIwMDYtZ3JhbnRlZC1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">501-kolnp-2006-granted-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTAxLWtvbG5wLTIwMDYtZ3JhbnRlZC1mb3JtIDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">501-kolnp-2006-granted-form 13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTAxLWtvbG5wLTIwMDYtZ3JhbnRlZC1mb3JtIDE4LnBkZg==" target="_blank" style="word-wrap:break-word;">501-kolnp-2006-granted-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTAxLWtvbG5wLTIwMDYtZ3JhbnRlZC1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">501-kolnp-2006-granted-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTAxLWtvbG5wLTIwMDYtZ3JhbnRlZC1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">501-kolnp-2006-granted-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTAxLWtvbG5wLTIwMDYtZ3JhbnRlZC1ncGEucGRm" target="_blank" style="word-wrap:break-word;">501-kolnp-2006-granted-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTAxLWtvbG5wLTIwMDYtZ3JhbnRlZC1yZXBseSB0byBleGFtaW5hdGlvbiByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">501-kolnp-2006-granted-reply to examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTAxLWtvbG5wLTIwMDYtZ3JhbnRlZC1zcGVjaWZpY2F0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">501-kolnp-2006-granted-specification.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="228755-device-for-measuring-the-consumption-of-electrical-energy.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="228757-pressure-fluid-driven-device-for-generating-reciprocating-movement-and-pneumatic-tool.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>228756</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>501/KOLNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>07/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>13-Feb-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>10-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>03-Mar-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>TETHYS BIOSCIENCE, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1900, POWELL STREET, SUITE 210, EMERYVILLE, CA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>URDEA, MICHAEL S.</td>
											<td>100, BUNCE MEADOW DRIVE, ALAMO, CA 94507</td>
										</tr>
										<tr>
											<td>2</td>
											<td>LANDES, GREGORY MICHAEL</td>
											<td>1603, VETTA DRIVE, LIVERMORE, CA 94550</td>
										</tr>
										<tr>
											<td>3</td>
											<td>WENT, GREGORY T.</td>
											<td>257, BUENA VISTA, MILL VALLEY, CA 94941</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>G01N 33/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2004/027005</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-08-18</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/511,720</td>
									<td>2003-10-15</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/496, 154</td>
									<td>2003-08-18</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/228756-a-method-for-reducing-the-complexite-of-a-sample-that-comprises-a-mixture-of-proteins by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:22:40 GMT -->
</html>
