<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/228884-a-fusion-polypeptide-and-a-process-for-preparing-them by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:31:10 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 228884:A FUSION POLYPEPTIDE AND A PROCESS FOR PREPARING THEM</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A FUSION POLYPEPTIDE AND A PROCESS FOR PREPARING THEM</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Fusion polypeptides and salts thereof comprising at least one IgE-binding domain fused to at least one human serum albumin component, optionally via a peptide linker, and in particular, dimeric fusion polypeptides comprising HSA protein fused, at each of its amino and carboxy termini, to an extracellular domain of the &amp;#945;-chain of the human high affinity receptor for IgE (Fc&amp;#949;Rl&amp;#945;); process for the preparation thereof, functionally equivalent polypeptides which are intermediates in their preparation, and polynucleotide and oligonucleotide intermediates and vectors therefor. They are indicated for use in the prevention and/or treatment of IgE-mediated allergic diseases and related disorders such as atopic dermatitis, atopic asthma and chronic urticaria.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
The invention relates to fusion polypeptides.  It concerns fusion polypeptides comprising an IgE-binding domain and a human serum albumin (HSA) component and salts thereof.  It also concerns polynucleotides and physiologically functional equivalent polypeptides which are intermediates in the preparation of such fusion polypeptides; appropriate recombinant expression vectors therefor, corresponding procaryotic and eucaryotic expression systems, and processes for synthesizing the fusion polypeptides.<br>
The interaction between immunoglobulin E (IgE) and its receptors has an established role in the defense against parasitic infections in humans (M. Capron and A. Capron, Science 264 [1994] 1876-1877).   In industrialized countries, however, with improved hygienic conditions, the encounter with parasites is less frequent than the disturbance of the IgE network by over-production of IgE in response to environmental allergens, resulting in allergies and other IgE- or IgE-receptor- mediated disease states.<br>
IgE is the primary antibody involved in initiation of an immediate allergic response and is a major participant in the maintenance of the late phase response.  IgE is synthesized in B lymphocytes and exerts its effects after binding to the high affinity receptor for IgE, i.e. FceRI, which is found on the surface of allergy effector cells such as mast cells, basophils and eosinophils.  IgE also exerts inducer functions through binding to its receptor on antigen presenting cells such as Langerhans cells, B cells and monocytes (G.C. Mudde etal. Allergy 50 \ 19951 193-199).<br>
An allergic response or condition is manifested when IgE molecules bind to the surface of allergy effector cells via the IgE receptor, FceRI, and become cross-linked by<br><br>
allergen, thereby effecting signalling to initiate degranulation ot cytoplasmic granules in the cells, with accompanying release of mediators of allergy such as histamine, serotonin, prostaglandins and cytokines, and consequent local tissue edema and influx of inflammatory cells.  An alternate means of stimulating allergy and associated conditions is the interaction of the IgE receptor, FceRI, with circulating autoantibodies against FceRI.<br>
FceRI typically exists as a tetramer comprising an a-, a p- and two y-chains (i.e. "subunits"), although on monocytes and Langerhans cells, the p-subunit is absent.<br>
The IgE binding site of FceRI has been shown to be contained entirely within its ot-subunit (referred to as FceRIa) (J. Hakimi et al., J. Biol. Chem.  265 [1990] 22079-22081; U. Blank et al., J. Biol Chem. 266 [1991] 2639-2646).    Recombinant "knockout" mice genetically deleted for the entire a-subunit have been found to be unable to mount an allergic response to allergen challenge (D. Dombrowicz et al., Cell  75 [1993] 969-976).<br>
FceRIa is a heavily glycosylated polypeptide of molecular weight about 60 kD, comprising a hydrophobic transmembrane domain as well as hydrophilic extracellular ("ecto-") and cytoplasmic domains which are exposed to the outer surface of the cell.  The IgE binding capability of FceRIa has been been further localized to its extracellular portion (J. Hakimi et al. [1990], supra;  Leder et al., USP 4'962'035).  It is possible to produce a soluble, secretable molecule by excising the transmembrane portion and sequences downstream therefrom (C. Ra et al., Int. Immunol. 5. [1993] 47-54); and the resulting truncation, consisting essentially of the human FceRIa extracellular domain, has IgE-binding activity in vitro and in vivo (M. Haak-Frendscho et al., Immunol. 151 [1993] 351-358: amino acid residues 1-204 of FceRIa fused to a truncated IgGl H chain C region;  C. Ra. et al. [1993] supra: residues 1-172 of FceRIa therein, corresponding to residues 26-197 of SEQ. ID. NO. 1 hereof).   Structural features of this fragment include two potential disulfide bridges and seven potential glycosylation sites (M. Haak-Frendscho et al. H993L supra).<br>
Therefore, truncations of FceRIa consisting essentially of the extracellular domain can potentially be administered therapeutically to a mammal to bind serum IgE in order to prevent its binding to its high affinity receptor on allergy effector cells, and also for suppressing de novo IgE biosynthesis in human lymphocytes (Y. Yanagihara et al., J. Clin. Invest. 94 [1994] 2162-2165).<br><br>
However, effective use of an IgE-binding.polypeptide such as the extracellular domain of FceRIot for systemic treatment of IgE* or IgE receptor- mediated allergic disorders in mammals has been hindered by its extreme transience in vivo due to rapid clearance from circulating plasma.  Considering that IgE- or IgE receptor- mediated diseases account for 10-20 % of physician-patient contact, an effective treatment would constitute a significant benefit to patients suffering from such conditions and an important advance in the clinical treatment of IgE- and IgE receptor- mediated disorders such as allergy and allergy-related conditions.<br>
It would therefore be beneficial to obtain an IgE-binding polypeptide having prolonged effective serum life and thus improved clinical utility in the treatment of allergy, and in particular the systemic treatment of atopic dermatitis, atopic asthma and chronic urticaria, as well as improved activity in an efficient, cost-effective manner.<br>
Summary<br>
It has now been found that by fusing an IgE-binding domain to a human serum albumin (HSA) component, fusion polypeptides can be obtained having extended serum half-life relative to the IgE-binding domain alone, without loss of IgE-binding activity, resulting in IgE-binding polypeptides indicated for use in the systemic treatment of allergy and other IgE-mediated disorders.  Systemically administered IgE-binding polypeptide will bind to serum IgE as well as to circulating auto-antibodies against the IgE receptor, FceRIa, preventing them from binding to cell-bound FceRIa, and thus preventing and/or inhibiting an allergic reaction and its associated manifestations.<br>
It has further been found that significant improvements in IgE-binding activity can be obtained by using fusion polypeptides of the invention comprising more than one IgE-binding domain per molecule.  For example, a "dimer" molecule of the invention has been found to have significantly increased IgE-binding activity relative to a  "monomer" of the<br>
invention.<br><br>
The term "dimer" herein refers to a fusion polypeptide ot the invention possessing two IgE-binding domains.  The term "monomer" refers to a fusion polypeptide of the invention possessing one IgE binding domain.  The monomer or dimer may be constructed of a single or multiple HSA components.  For example, a monomeric fusion polypeptide of the invention may comprise an IgE-binding domain fused at either its amino or carboxy terminus to an HSA component.  Alternatively, the monomer may comprise an IgE-binding domain fused at both of its termini to HSA components.  A dimeric fusion polypeptide according to the invention may comprise, for example, two IgE-binding domains fused via the carboxy terminus of one and the amino terminus of the other to an intervening HSA component.  Alternatively, the dimer may comprise, in addition to its two IgE-binding domains, multiple HSA components.<br>
It has further been found that a dimer molecule of the invention possesses unexpectedly favorable activity.<br>
The invention therefore is directed to fusion polypeptides and salts thereof comprising at least one IgE-binding domain fused to at least one HSA component; preferably, to monomeric fusion polypeptides, wherein a single IgE-binding domain is fused to one or more HSA components, or to multimeric fusion polypeptides, wherein two or more IgE-binding domains are fused to at least one HSA component; more preferably, dimers possessing two IgE-binding domains and at least one HSA component.<br>
The invention is further directed to polynucleotide intermediates therefor.<br>
The term "fused" or "fusion" herein refers to polypeptides in which: (i)  a given functional domain (i.e. an IgE-binding domain) is bound at its carboxy terminus by a covalent (preferably, peptide, i.e. amide) bond either to the amino terminus of another functional domain (i.e. an HSA component) or to a linker peptide which itself is bound by a covalent (preferably, peptide) bond to the amino terminus of the other functional domain; and/or,<br>
(ii)  a given functional domain (i.e. an IgE-binding domain) is bound at its amino terminus by a covalent (preferably, peptide, i.e. amide) bond either to the carboxy terminus of another functional domain (i.e. an HSA component) or to a linker peptide which itself is bound by a covalent (preferably, peptide) bond to the carboxy terminus of the other functional domain.<br><br>
Similarly, "tused" when used in connection witn tne polynucleotide intermediates or the invention means that the 3'- [or 5'-] terminus of a nucleotide sequence encoding a first functional domain is bound to the respective 5'-[or 3'-] terminus of a nucleotide sequence encoding a second functional domain, either by a covalent bond or indirectly via a nucleotide linker which itself is covalently bound preferably at its termini to the first functional domain-encoding polynucleotide and the second functional domain-encoding polynucleotide.<br>
Preferably, the fusion polypeptide or its salt is a monomer or a dimer; when it is a dimer it comprises two IgE-binding domains fused to an HSA component, such as a first IgE-binding domain fused at its carboxy terminus to the amino terminus of an HSA component, and a second IgE-binding domain fused at its amino terminus to the carboxy terminus of that HSA component.<br>
The fusion polypeptides of the invention may also comprise further functional domains in addition to IgE-binding domains and HSA components, e.g. full length or truncated (e.g. soluble, extracellular fragments of) human proteins, such as cytokines or the amino-terminal fragment of urokinase.  These additional functional domains may themselves serve as linker peptides, for example, for joining an IgE-binding domain to another IgE-binding domain or to an HSA component; alternatively, they may be located elsewhere in the fusion molecule, e.g. at the amino or carboxy terminus thereof.<br>
Examples of fusion polypeptides of the invention may be represented by the following formulas:<br>
I.	R, - L - R2<br>
II.	R2 - L - R,<br>
III.	R, - L - R2 - L - R,<br>
IV.	R, - L - R, - L - R2<br>
V.	R2 - L - R, - L - R,	wherein<br>
Ri   is the amino acid sequence of an IgE-binding domain,<br>
R2  is the amino acid sequence of an HSA component,<br>
each L is independently a covalent (preferably, peptide) bond, or is a peptide linker which is<br>
bound by a covalent (preferably, peptide) bond to a terminus of Rj and/or R2 , whereby the above molecule fragments are read directionally, i.e. with the left side<br>
corresponding to the amino terminus and the right side to the carboxy terminus of<br>
the molecule.<br><br>
It has been found that IgE binding is at high levels when the fusion polypeptide is led by an IgE-binding domain; that is, when the IgE binding domain constitutes the N-terminal portion of the mature fusion protein (such as in compounds of formulae I,  III and IV, above).  A particularly preferred embodiment of the invention comprises a dimer of formula III above, i.e. wherein the protein region which leads expression is an IgE-binding domain, fused at its carboxy terminus to the amino terminus of an HSA component, which in turn is itself fused at its carboxy terminus to the amino terminus of a second IgE-binding domain.  When there is more than one Rj moiety, or more than one R2 moiety, or more than one L moiety, these moieties can be, but are not necessarily, identical<br>
Preferably the fusion polypeptides of the invention comprise at least one soluble, secretable mammalian IgE-binding domain fused to at least one HSA component, and pharmaceutical^ acceptable salts thereof.<br>
The invention is further directed to a method for the prevention and/or treatment of IgE- or IgE-receptor- mediated disorders, and in particular, allergic reactions such as atopic dermatitis, atopic asthma and chronic urticaria, comprising the administration of the fusion<br>
polypeptides of the invention or pharmaceutically acceptable salts thereof to a subject in need of such treatment; and to pharmaceutical compositions comprising the fusion polypeptides of the invention or pharmaceutically acceptable salts thereof, together with at least one pharmaceutically acceptable carrier or diluent.<br>
The invention is further directed to physiologically functional equivalents of the fusion polypeptides of the invention which are intermediates in the synthesis of the polypeptides; to polynucleotides which are intermediates in the preparation of the fusion polypeptides or physiologically functional equivalents thereof as defined above; to oligonucleotide intermediates to be used to construct them; to the peptides encoded by such oligonucleotides; to recombinant vectors for expressing the fusion molecules; to procaryotic or eucaryotic (especially, mammalian) expression systems; and to processes for preparing the fusion polypeptides or physiologically functional equivalents thereof using such expression systems.<br><br>
Description of SEP. ID. NOs. and related definitions<br>
SEP. ID. NO. 1:   Amino acid sequence of dominant form of full length native human FceRIa, including  signal sequence.<br>
The term "pre-IgER" refers to residues MetrLeu204 of SEQ. ID. NO. 1. The term "IgER" refers to the mature form of pre-IgER and constitutes residues Val26-Leu2o4 of SEQ. ID. NO. 1 (i.e. the extracellular domain of FceRIot).<br>
SEP. ID. NO. 2:   Amino acid sequence of dominant form of native prepro-HSA (referred to herein as "prepro HSA I"), comprising residues Met|-Leu609.<br>
The dominant form of the mature native protein (referred to herein as "HSA I") is represented by residues Asp25-LeugQ9 of SEQ. ID. NO. 2.<br>
The term "prepro-HSA 11" represents a truncation of the native sequence by one amino acid (Leu609) at the carboxy terminus, and therefore refers to residues Metj-Gly60g of SEQ. ID. NO. 2.<br>
The mature form of prepro-HSA II, referred to herein as "HSA II", is represented by residues Asp25-Gly608 of SEQ. ID. NO. 2.<br>
SEQ. ID. NO. 3:   Amino acid sequence encoded by the EcoRI fragment of plasmid R-H-R/SK #50 prepared in Example 5, comprising:   "pre-IgER" sequence at residues 1-204; linker AlaSer(Gly)4Ser (referred to hereinafter as "Lj") at residues 205-211; HSA II sequence at residues 212-795; linker (Gly^Ser (referred to hereinafter as "L^") at residues 796-799; and the "IgER" sequence at residues 800-978.<br>
A mature dimeric fusion polypeptide of the invention, referred to herein as IgER -  Lr HSA II - L2 - IgER" or, alternatively, as "IgER - HSA - IgER Dimer", expressed from CHO cells in the manner described in Example 7, has the amino acid sequence Val26-Leu978 of SEQ. ID. NO. 3.<br>
SEP. ID. NO. 4:  Nucleotide sequence of the EcoRI fragment of plasmid R-H-R/SK #50 of Example 5, comprising:   a polynucleotide sequence encoding "pre-IgE " at positions 10-621;  an oligonucleotide encoding Lj at positions 622-642;  a polynucleotide encoding<br><br>
HSA II at positions 643-2394;  an oligonucleotide encoding L2 at positions 2395-2406;  and a polynucleotide encoding "IgER" at positions 2407-2943; with 2 stop codons at positions 2944-2949.<br>
Restriction sites at the ends of the coding fragments and in the linker regions are at positions 1-6; 622-627; 637-642; 2387-2393; 2401-2406; and 2950-2955.  A Kozak sequence is at nucleotide positions 7-9.<br>
Point mutations differing from the consensus HSA nucleotide sequence are at positions 804, 1239; 1290; 1446, 1815, 2064 and 2079.  Because the point mutations are in the wobble position, they do not affect the amino acid sequence.<br>
Seq. ID. NO. 5:  Nucleotide sequence of the dominant form of native prepro-HSA corresponding to FIG. 12 and to the amino acid sequence of SEQ.ID.NO.2.<br>
Seq. ID. NO. 6:   Nucleotide sequence of the dominant form of full length native human FceRIa, including signal sequence, corresponding to FIG. 13 and to the amino acid sequence of SEQ.ID.NO. 1.<br>
Description of the Fi2ures<br>
In the following Figures, the indicated molecules are read directionally, i.e. the left side corresponding to the amino (or 5'-) terminus and the right side corresponding to the<br>
carboxy (or 3'-) terminus.<br>
FIGS. 1A-C:   Serum half life in mice:  Protein concentration in vivo (in picomoles of protein per ml of serum) over time (10-800 minutes after injection) of<br>
(A)	free HSA I protein, and<br>
(B)	IgER protein (referred to as "Free alpha chain") and IgER - Lj - HSA II - L2 - IgER dimeric fusion polypeptide (referred to as "IgER - HSA - IgER" dimer) prepared from CHO cells as described in Example 7.<br>
(C)	Serum half life of free HSA I from (A) compared with that of dimeric fusion polypeptide ("IgER - HSA - IgER") from (B) by normalizing to 1 with respect to serum concentrations at 10 minutes after injection.<br><br>
FIG. 2:  Extravasation resulting from passive cutaneous anaphylaxis reaction in mice administered intravenous serial dilutions (10 Mg/kg,  50 |ig/kg or 500 ng/kg) of IgER - L1 - HSA n - L2 - IgER  polypeptide  ("IgER - HSA - IgER Dimer") prepared as described in Example 7 (the control group receiving 0 |ig/kg); area in mm ; at intervals of 5, 15 or 30 minutes prior to sensitization by intradermal injection of monoclonal mouse IgE anti-dinitrophenyl (DNP) antibody and subsequent challenge with DNP-bovine serum albumin solution containing 1% Evans blue.<br>
FIG. 3:   Schematic representation of three fusion polypeptides of the invention (two monomers and one dimer), with the polypeptide linkers also shown:<br>
I.	Monomers:<br>
(1.)  HSA-leading monomer comprising HSA II (referred to in the Figure as "HSA") fused via L2 ("GGGS") to IgER.  The nucleotides encoding positions  Leu607Gly60g of HSA II contain a unique Mstll site as  indicated, and the nucleotides encoding  GlySer of L2 contain a BamHI site (encoded amino acids underlined);<br>
(2.)  IgE-binding domain-leading monomer comprising IgE   fused at its carboxy terminus via L1 (i.e. "ASGGGGS") to HSA II (referred to in the Figure as "HSA").  The oligonucleotide encoding L1 contains a Nhel site and a BamHI site respectively (encoded amino acids  AlaSer  and  GlySer of L1 are underlined).<br>
II.	Dimer:   IgE-binding domain-leading dimer comprising a first IgER fused at its carboxy<br>
terminus via L1 to the amino terminus of HSA II (referred to in the Figure as "HSA"), the<br>
carboxy terminus of which is fused to a second IgER via L2 with restriction sites in the<br>
encoding polynucleotide as described above for the monomer.<br>
FIG.  4:   PCR primers to truncate full length human FceRIq cDNA (referred to as "IgE Receptor cDNA") to obtain DNA encoding:<br>
(0 iss!<br>
[BamHI site added to 5' end of coding strand for FceRIa by oligonucleotide #18; and stop codon and EcoRI and Sail sites added to 3' end of non-coding strand by oligonucleotide #19];<br><br>
(ii) pre-IgE"<br>
[oligonucleotide #20 adding SstI, EcoRI, and Kozak sites to 5' end of coding strand for FceRIa; oligonucleotide #31 adding NotI and Nhel (and deleting a second Nhel site) in non-coding strand for FceRIa]; (iii)  pre-IgER [oligonucleotides #20 and #19 used as described above]:<br>
(A.)   "HSA-leading":   subcloning of (i) above into SK vector, providing cloning vector TA clone pEKl used in construction of HSA II - leading monomer;<br>
(B.)   "IgER-leading":   subcloning of (ii) into SK vector, providing construct IgER/TA#l used to prepare IgE -leading monomer;<br>
(C.)   "IgER alone":   subcloning of (iii) into SK vector, providing construct IgER FL/TA#34 used in expression of mature IgER for use as a standard. Double asterisks represent two stop codons.<br>
FIG.  5:   PCR primer pairs to truncate full length human prepro-HSA I cDNA to yield<br>
cDNA encoding:<br>
(i)  prepro-HSA II fused at carboxv terminus to L2<br>
[oligonucleotide # 24 adding Spel, EcoRI and Kozak sequence to 5'-end of coding strand;<br>
oligonucleotides #28 and #29 adding a linker encoding Mstll and Hindlll sites at the<br>
3' terminus of HSA II);<br>
(ii)  L]  fused to 5* terminus of HSA II<br>
[oligonucleotide #26 adding NotI, Nhel, linker and BamHI sites to coding strand;<br>
oligonucleotide #27 containing Ncol site in non-coding strand];<br>
(iii)  prepro-HSA I<br>
[oligonucleotide #24 used as above; and oligonucleotide #25 adding stop, EcoRI and Hindlll<br>
sites to 3' end of non-coding strand]:<br>
(A.)   "HSA leading":   subcloning of (i) into SK vector, providing pEK7 used to construct<br>
HSA II - leading monomer;<br>
(B.)  "IgER leading":  subcloning of (ii) into SK vector, providing HSA/SK#5 used to<br>
construct IgER-leading monomer;<br>
(C.)  "HSA alone":    subcloning of (iii) into SK vector, providing HSA/SK#17 encoding<br>
mature native human serum albumin protein (i.e. HSA I) for use as a standard.<br><br>
FIG. 6:  Human serum albumin (HSA) sequencing oligonucleotides #1-10, 12, 16, 17, 22<br>
and 30,  used to sequence materials cloned in TA and after linkage to IgE-binding fragments.<br>
FIG. 7:  IgE receptor sequencing oligonucleotides #11, 13 and 23, used to sequence fragments cloned in TA and after linkage to HSA component.<br>
FIG. 8:<br>
(A)	Mutagenic oligonucleotides #14 and #15 for human serum albumin;<br>
(B)	PCR and linker oligonucleotides #18-20, 24-29 and 31 for constructing fusion polypeptides.<br>
FIG. 9:   Construction of vector HSA-IgER/SK#49, comprising a polynucleotide encoding prepro-HSA II (referred to in the Figure as "HSA") fused at 3'-terminus via oligonucleotide encoding linker L2 (represented in FIG.9 by "GGG") to 5'-terminus of polynucleotide encoding IgER (referred to in the Figure as "IgER"), by ligating the BamHI, Sail fragment of pEKl into BamHI,SalI-cut pEK7.<br>
FIG. 10:   Construction of vector IgER-HSA/SK#l, comprising a polynucleotide encoding pre-IgER (referred to in the Figure as "IgER") fused at 3'-terminus via oligonucleotide for linker L] (represented by "GGG") to 5'-terminus of polynucleotide encoding HSA II ("HSA"), by ligating the SstI, Nhel fragment of IgER/TA#l into SstI,NheI - cut HSA/SK#5.<br>
FIG.  11:  Construction of vector HSA-IgER Pst Sal/SK#37, comprising a polynucleotide encoding HSA II (indicated by "HSA3") which is fused at the 3' terminus via oligonucleotide for L2 (not depicted) to 5'-terminus of polynucleotide encoding IgER (indicated as "IgER"), by ligating the PstI, Sail fragment of HSA-IgER/SK#49 into SK vector.  Also shown is construction of a vector R-H-R/SK #50 comprising a polynucleotide encoding pre-IgER (referred to as "IgER") fused at its 3' terminus via oligo for L1 (not depicted) to 5'-terminus of polynucleotide encoding HSA II ("HSA"), which in turn is fused via oligonucleotide for L2 (not depicted) to a polynucleotide encoding IgER ("IgER").  The vector is prepared by ligating the PstI,KpnI fragment of HSA-IgER Pst Sal/SK#37 into  PstI,KpnI - cut  IgER-HSA/SK#l.<br><br>
FIG. 12:  Nucleotide sequence of human serum albumin and amino acid sequence corresponding to SEQ. ID. NO. 2 and SEQ. ID. NO. 5.<br>
FIG. 13:  Nucleotide sequence of IgER and amino acid sequence corresponding to SEQ. ID. NO. 1 and SEQ. ID. NO. 6.<br>
FIG. 14:  Nucleotide and amino acid sequences of the EcoRI fragment of R-H-R/SK #50 corresponding to SEQ. ID. NO. 3 and SEQ. ID. NO. 4, encoding the R7 - HSA - Rl dimeric fusion protein.  The HSA sequence is shown in italics.  Linker sequences are shown in lower case.  Point mutations differing from the consensus HSA nucleic acid sequence are shown in bold lower case.  Because the point mutations are in the wobble position, they do not affect the amino acids sequence.  Restriction sites at the ends of the fragments and in the linker region are underlined.<br>
FIG. 15:   S PS-PAGE of purified mature fusion polypeptide of Example 7:<br>
Total amounts applied to gel:  8 ng (lane 1),  6 \ig (lane 2), 4 fig (lane 3) and 2 \xg (lane 4); molecular weight standards:  97.4, 66.2, 45, 31, 21.5 and 14.4 kDa (lane 5).<br><br>
Detailed description<br>
The invention concerns fusion polypeptides and salts thereof, comprising at least one IgE-binding domain fused to at least one HSA component.<br>
The term "IgE-binding domain" refers to an amino acid sequence capable of binding or otherwise associating with mammalian, e.g. human IgE in a manner preventing the binding of IgE to its receptor, FceRL<br>
The cDNA and deduced amino acid sequence of human FCERICC are known (J.Kochan et al.. Nucleic Acids Research 16 [1988] 3584; and Leder et al., USP 4'962'035). Human FceRJa encodes an NH2-terminal signal peptide [amino acid residues (including the first Met) Metj-Ala25], two immunoglobulin-like extracellular domains (residues Val26-Leu204), a hydrophobic transmembrane region (Gln205-Ile224), anc* a hydrophilic cytoplasmic tail (residues Ser225~Asn257) (with reference to SEQ. ID. NO. 1).  The signal peptide is cleaved during intracellular processing.  The full length amino acid sequence of the dominant form of native human FceRIa is included herein as SEQ. ID. NO. 1.<br>
"IgER" herein refers to amino acid sequence Val26-Leu2Q4 Â°^ SEQ. ID. NO. 1 (encoding the extracellular domain); and the term "pre-IgE " refers to residues Met|-Leu2Q4 of SEQ. ID. NO. 1 (encoding the signal sequence upstream of the extracellular domain).<br>
The IgE-binding domain is e.g. an amino acid sequence having at least 80%, more preferably at least 85%, more preferably at least 95%, or most preferably at least 99%, homology with  IgER as defined above (homology is calculated as the % identity between the native sequence and the altered sequence, as a function of total sequence length of the native molecule, after aligning the sequences and introducing gaps, if necessary, to achieve the maximum percent sequence identity).<br>
In a subgroup of fusion polypeptides of the invention the IgE-binding domain component is<br>
(Xa) wherein (Xa) is:<br>
(a)	IgER ; or<br>
(b)	a naturally occurring allele of IgER, or<br><br>
(c)	a truncation of (a) or (b) at the carboxy terminus by 1-12 (e.g. 1-10, 1-7 or 1-4) amino<br>
acids; or<br>
(d)	a variant of (a), (b) or (c).<br>
By "variant" is meant:<br>
-	the sequence of (a), (b) or (c) as modified by one or more deletions (which are other than truncations at the carboxy terminus) of up to 10 (e.g. 1-7 or 1-5) amino acids;<br>
-	insertions of a total of up to 10 (e.g. 1-5) amino acids internally within the amino acid sequence, or of up to a total of 100 amino acids at either terminus; or<br>
-	conservative substitutions of a total of up to 15  (e.g. 1-5) amino acids.<br>
The IgE-binding domain is more preferably<br>
(Xb)<br>
wherein (Xb) is:<br>
(a)	IgER;<br>
(b)	a truncation of IgE   at the carboxy terminus by 1-7 amino acids; or<br>
(c)	a variant of (a) or (b).<br>
The IgE-binding domain is even more preferably<br>
(Xc) wherein (Xc) is:<br>
(a)	IgE ; or<br>
(b)	a truncation of IgE   at the carboxy terminus by 1-7 amino acids (e.g. the sequence Val-&gt;6-Ala]97 with reference to SEQ. ID. NO. 1).<br>
The IgE-binding domain is most preferably IgER.<br>
Preferably, any homolog, truncation or variant is prepared recombinantly as a "secretable" polypeptide, namely the mature peptide sequence encoding the IgE-binding domain can be secreted from the host cell in which it (or a precursor form such as a preform) is synthesized from a polynucleotide.<br>
The other major constituent of the recombinant fusions of the invention, human serum albumin (HSA), constitutes the most abundant plasma protein, contributing 60 % on a per weight basis of the total protein content of plasma.  A molecule of human serum albumin consists of a single non-glycosylated polypeptide chain of 585 amino acids of<br><br>
molecular weight 66.5 kDa. A feature specific to albumin is its complex disulfide bond pattern (F.F. Clerc et al, J. Chromatogr. 662 [1994] 245-259).  Human serum albumin is widely distributed throughout the body, in particular in the intestinal and blood compartments where it is mainly involved, as the most abundant protein of the serum, in the maintenance of osmolarity and plasma volume. Furthermore, it is slowly cleared by the liver and displays an in vivo half-life of 14-20 days in humans (T.A. Waldmann, "Albumin Structure, Function and Uses", Pergamon Press [1977] 255-275).  Human serum albumin is devoid of enzymatic or immunological function.  It is a natural carrier involved in the endogenous transport and delivery of various natural as well as therapeutic molecules (H.Luetal.,FEBS_Lett. 356 [1994] 56-59;  WO 93/15199;  EP 648499;  P. Yeh et al., PNAS USA 89 H9921 1904-1908; EP 413622).<br>
Human cells synthesize serum albumin initially in the form of a prepro- polypeptide. A signal sequence of 18 amino acids (including the first Met) is removed when the protein passes through the lumen of the endoplasmic reticulum, leaving still 6 amino acids at the N-terminus (in the predominant form:  Arg-GIy-Val-Phe-Arg-Arg) which are then subject to proteolytic excision during or immediately following transport through the secretory apparatus.<br>
Human serum albumin is well-known to be polymorphic (D.C. Carter and J.X. Ho, Adv. Prot. Chem. 45 [1994] 153-203). For example, albumin Naskapi has Lys372 in place of Glu372, and proalbumin Christchurch has an altered pro- sequence, i.e. Glu24 instead of Arg24 (Latta et al., USP 5'100'784).<br>
The complete amino acid sequence of the dominant form of naturally occurring protein, referred to herein as "prepro-HSA I", is known (A. Dugaiczyk et al., PNAS USA 79 [1982] 71-75)  (SEQ. ID. NO. 2).  The dominant form of the mature protein ("HSA F) is represented by Asp^-Leu^g of SEQ. ID. NO. 2.<br>
The HSA carrier of the fusion polypeptide of the invention may be an amino acid sequence having at least 80 %, more preferably at least 85 %, even more preferably at least 95 %, or most preferably at least 99 %, homology with residues 25-609 of SEQ. ID. NO. 2 (i.e. HSA I) (homology is calculated as the % identity between the native sequence and the altered sequence, as a function of total sequence length of the native molecule, after aligning the sequences and introducing gaps, if necessary, to achieve the maximum percent sequence identity).<br><br>
In a subgroup of fusion polypeptides ot the invention, tne  HSA component is<br>
(Ya) wherein (Ya) is:<br>
(a)	HSA I; or<br>
(b)	a naturally occurring allele of (a); or<br>
(c)	a truncation of (a) or (b) at the carboxy terminus thereof by 1-10 (e.g. 1-5, or 1 or 2) amino acids; or<br>
(d)	a truncation of (a) terminating at amino acid residue n, where n is 369 to 419, and especially where n is 373, 387, 388, 389, 390 and 407 (as described in USP 5'380'712); or<br>
(e)	a variant of (a), (b), (c) or (d).<br>
By "variant" is meant:<br>
-	the sequence of (a), (b), (c) or (d) as modified by one or more deletions (which are other than truncations at the carboxy terminus) of up to 10 (e.g. 1-7, or 1-5) amino acids;<br>
-	insertions of a total of up to 10 (e.g. 1-5) amino acids internally within the sequence, or of up to a total of 100 amino acids at either terminus; or<br>
-	conservative substitutions of a total of up to 15  (e.g. 1-5) amino acids.<br>
The  HSA component is more preferably<br>
(Yb)<br>
wherein (Yb) is:<br>
(a)	HSA I; or<br>
(b)	a naturally occurring allele of (a); or<br>
(c)	a truncation of (a) or (b) at the carboxy terminus thereof by 1-10 (e.g. 1-5, or 1 or 2)<br>
amino acids); or<br>
(d)	a variant of (a), (b) or (c).<br>
The HSA component is even more preferably<br>
(Yc) wherein  (Yc) is:<br>
(a)	HSA I, or<br>
(b)	a truncation of HSA I at the carboxy terminus thereof by 1-10 (e.g., 1) amino acids<br>
(an example of such a truncation is "HSA II", which is the amino acid sequence represented by Asp25-Gly608 of SEQ. ID. NO. 2.)<br><br>
In an even more preferred subgroup, the HSA component is HSA I or HSA II, particularly HSA II.<br>
Especially preferred is the fusion polypeptide of Example 7, especially without leader sequence, i.e. the polypeptide Val26-Leu978 of SEQ.ID.NO.3.<br>
Preferably, any homolog, truncation or variant of native HSA which is used to prepare a fusion protein of the invention will be devoid of enzymatic function; and will not prevent or inhibit binding of the IgE-binding domain to serum IgE.  It is further preferred that any such homolog or variant will have a serum half-life of at least 14 days.<br>
With respect to either the IgE-binding domain or HSA component, the term "conservative substitutions" refers to the substitution of one or more amino acids by others having similar properties such that one would expect at least the secondary structure, and preferably the tertiary structure of the polypeptide, to be substantially unchanged.  For example, typical such substitutions include alanine or valine for glycine, asparagine for glutamine, serine for threonine and arginine for lysine.  All of the amino acids (except for glycine) are preferably naturally-occurring L-amino acids.<br>
Preferably each IgE-binding domain is at least 95% homologous to IgER, and each HSA component is at least 95% homologous to HSA I.<br>
In other preferred subgroups:<br>
-	each IgE-binding domain is preferably (Xa), and each HSA component is (Ya), or more preferably (Yb), or even more preferably (Yc), as defined above;<br>
-	each IgE-binding domain is more preferably (Xb), and each HSA component is (Ya), or more preferably (Yb), or even more preferably (Yc), as defined above;<br>
-	each IgE-binding domain is even more preferably (Xc), and each HSA component is (Ya) or more preferably (Yb), or even more preferably (Yc).<br>
In a further preference, the IgE-binding domain is IgER and the HSA component is (Ya), or more preferably (Yb), or even more preferably (Yc), as defined above, and is most preferably HSA I or HSA II, in particular HSA II.<br>
The preferences expressed above also apply in particular to the polypeptides of each of formulae I, II, III, IV and V herein.<br><br>
Preferred polypeptides of the invention are those composing a tirst molecule of lgH" which is fused via its carboxy terminus to the amino terminus of a molecule of HSA II, which HSA EI is fused at its carboxy terminus to the amino terminus of a second molecule of IgER(IgER being residues Va^-Le^^ of SEQ. ID. NO. 1; and HSA II being residues Asp25-Gly608 of SEQ. ID. NO. 2).<br>
Particularly preferred polypeptides of the invention comprise dimers of formula III hereinabove wherein each Rj is IgER and R2 is HSA II.  Especially preferred are the dimers of formula III wherein each L is 1-25 amino acids.<br>
Any peptide linker (expressed as "L" in formulae I-V herein) preferably allows independent folding and activity of the IgE-binding domain; is free of a propensity for developing an ordered secondary structure which could interfere with the IgE-binding domain or cause an immunological reaction in the patient, and has minimal hydrophobic or charge characteristics which could interact with the IgE-binding domain.<br>
The peptide linker is preferably 1-500 amino acids; more preferably 1-250; and even more preferably 1-100 (e.g. 1-25,   1-10,   1-7 or   1-4) amino acids.  The linker is preferably linear, i.e. non-branched.   In general, peptide linkers comprising Gly, Ala and Ser can be expected to satisfy the criteria for a linker.  For example, linkers of the present invention include:   GlyGlyGlySer ("Lf herein) and AlaSerGlyGlyGlyGlySer (ML2" herein).   Linkers of various other lengths and sequence composition may also be used.<br>
The invention also includes oligonucleotides encoding the peptide linkers of the invention.  Such oligonucleotides should be "fused in frame" with the polynucleotides encoding the IgE-binding domain and HSA component, and preferably include restriction sites unique in the molecule.  By "fused in frame" is meant that:<br>
(1)	there is no shift in reading frame of the IgE-binding domain or the HSA component caused by the linker oligonucleotide; and<br>
(2)	there is no translation termination between the reading frames of the IgE-binding domain and the HSA-component.<br><br>
The invention further encompasses physiologically functional equivalents or me novel fusion polypeptides which are normally intermediates in the synthesis of the novel polypeptides.  The term "physiologically functional equivalent" preferably refers to a larger molecule comprising the fusion polypeptide of the invention to which has been added such amino acid sequence as is necessary or desirable for effective expression and secretion of the mature recombinant fusion polypeptide of the invention from a particular host cell. Such added sequence is typically at the amino terminus of the mature polypeptide, and usually constitutes a leader (i.e. signal) sequence which serves to direct them into the secretory pathway, and is normally cleaved upon or prior to secretion of the polypeptide from the cell.  The signal sequence can be derived from the natural N-terminal region of the relevant polypeptide, or it can be obtained from host genes coding for secreted proteins, or it can derive from any sequence known to increase the secretion of the polypeptide of interest, including synthetic sequences and all combinations between a "pre-" and a "pro-" region.  The juncture between the signal sequence and the sequence encoding the mature polypeptide should correspond to a site of cleavage in the host.<br>
In fusion polypeptides wherein an IgE-binding domain "leads" expression, i.e. is upstream from other coding sequences in the fusion molecule, it is expedient to utilize the<br>
leader sequence of native human FceRIot (i.e. Met]  +  Ala2-Ala25 of SEQ. ID. NO. 1), and<br>
this has been employed effectively to obtain the mature polypeptide from mammalian expression systems (e.g. CHO, COS), as well as from yeast (Pichia pastoris).<br>
An example of a physiologically functional equivalent of the mature dimeric fusion polypeptide  IgER - L, - HSA II - L2 - IgER  is:  pre- IgER - L, - HSA II - L2 - IgER. However, the additional leader sequence is not necessarily that of human FceRIa and may be obtained from any suitable source, provided it is suitable to effect expression/secretion of the mature polypeptide from the particular host cell. For example, the prepro- sequence of HSA may also be used to prepare the above dimeric fusion polypeptides, particularly for expression in yeast.<br>
In fusion polypeptides of the invention wherein an HSA component leads expression, a suitable leader sequence may comprise the native leader sequences.  For example, the native prepro- region of HSA may be used to accomplish secretion of the mature heterologous polypeptide from mammalian (e.g. CHO, COS) cells or yeast (Pichia pastoris).<br><br>
/\n example ui a pnysioiugitaiiy luncuunai equivalent ui uic mature IUMUII polypeptide represented by  HSA II - L2 - IgER is:    prepro HSA II - L2 - IgER.   However, other leader sequences, not necessarily native to HSA or to the host cell, may provide effective expression of the mature fusion protein in certain hosts (USP 5'100'784; USP 5'330'901).<br>
The invention also comprises polynucleotides which are intermediates in the preparation of the recombinant fusion polypeptides of the invention, including polynucleotides encoding the fusion polypeptides or physiologically functional equivalents thereof and oligonucleotides encoding linker peptides, e.g. as depicted in Figure 8.<br>
As a further aspect, there is provided a process for preparing a recombinant fusion polypeptide as defined above or salt thereof, which comprises:<br>
(a)	transforming a host cell with a vector comprising DNA encoding a fusion polypeptide<br>
as defined above or a physiologically functional equivalent thereof;<br>
(b)	expressing the fusion polypeptide or its physiologically functional equivalent in that cell, whereby the physiologically functional equivalent polypeptide is modified (e.g. by cleavage of a signal sequence) to yield a fusion polypeptide as defined above; and<br>
(c)	recovering the resultant polypeptide from the host cell, preferably as a secreted product, optionally in the form of a salt thereof.<br>
A still further aspect of the present invention provides vectors, preferably plasmids, for use in the expression of the fusion polypeptides.  These vectors comprise DNA encoding the polynucleotides defined above or physiologically functional equivalents thereof.  In general, appropriate vectors which can transform microorganisms capable of expressing the fusion polypeptides include expression vectors comprising nucleotide sequences coding for the fusion polypeptides joined to transcriptional and translational regulatory sequences, such as promoters, which together constitute an expression cassette.  The promoters may derive from genes of the particular host used, or such control regions may be modified, for example, by in vitro site-directed mutagenesis, by introduction of additional control elements or synthetic sequences.  The expression cassette specifically used in the present invention thus also includes a transcription and translation termination region which is functional in the intended host and which is positioned at the 3' end of the sequence<br><br>
encoding the hybrid macromolecule.  In addition-to the expression cassette, the vector will include one or several markers enabling the transformed host to be selected.  Such markers include markers conferring resistance to antibiotics such as G418.  These resistance genes will be placed under the control of the appropriate transcription and translation signals allowing for expression in a given host.<br>
More particularly, the preparation of recombinant fusion polypeptides of the invention may be effected e.g. as follows:<br>
A.  Construction of fusion protein expression vectors<br>
The first step in the construction of recombinant fusion polypeptides is to subclone portions of the fusion polypeptides in cloning vectors.  In this context, a "cloning vector" is a DNA molecule, such as a plasmid, cosmid or bacteriophage, that can replicate autonomously in a host prokaryotic cell.  Cloning vectors typically contain one or a small number of restriction endonuclease recognition sites at which foreign DNA sequences can be inserted in a determinable fashion without loss of an essential biological function of the vector, as well as a marker gene that is suitable for use in the identification and selection of cells transformed with the cloning vector. Marker genes typically include genes that provide tetracycline resistance or ampicillin resistance.  Suitable cloning vectors are described in J. Sambrook et al. (eds.), Molecular Cloning: A Laboratory Manual, 2nd Edition (Cold Spring Harbor Laboratory Press [1989]) and can be obtained, for example, from various sources.<br>
The FceRIa cDNA clone pGEM-3-110B-l, is described in A. Shimizu et al., Proc.Nat.Acad.Sci.USA 85 [1988] 1907-1911 and can be obtained from the American Type Tissue Collection (ATCC stock #67566).  Single-stranded human liver cDNA can be obtained from Clontech (PCR-ready Quick Clone cDNA, Cat. D#7113-1).  The sequence of HSA is available from GenBank under Accession #s:  V00495, J00078, L00132, LOO 133.  HSA cDNA can be obtained by PCR amplification using oligonucleotides #24 and 25, as described in Example 2.<br>
A polynucleotide encoding a suitable IgE-binding domain or HSA component DNA can be prepared using the polymerase chain reaction (PCR). PCR utilizes a single-stranded cDNA template and a mixture of oligonucleotide primers.  The PCR procedure is perfomed via well-known methodology (see e.g. C.R.M. Bangham, "The Polymerase Chain Reaction:<br><br>
Getting Started" in Protocols in Human Molecular Genetics, Human Press [1991], Ch.I, p. 1-8).  PCR kits and material for use in the kits are also commercially available from various sources.  Kits and methods for using them are further described in e.g. USP 5'487'993).<br>
DNA sequences encoding signal peptides can be added by PCR or if necessary using synthetic oligonucleotides that encode known signal peptide sequences.  DNA sequences encoding a heterologous signal peptide are subcloned in frame with DNA sequences encoding the N-terminus of the fusion polypeptide.<br>
Fusion of the polynucleotides may be accomplished by subcloning in intermediate vectors. Alternatively, one gene can be cloned directly into a vector containing the other gene.  Linkers and adapters can be used for joining the DNA sequences.<br>
Subcloning is performed in accordance with conventional techniques, such as by use of restriction enzyme digestion to provide appropriate termini, the use of alkaline phosphatase treatment to avoid undesirable joining of DNA molecules, and ligation with appropriate ligases.  Techniques for such manipulation are described in the literature and are known in the art.<br>
B.   Expression cloning of fusion polypeptides<br>
The cloned fusion protein is then cleaved from the cloning vector and inserted into an expression vector.   Suitable expression vectors typically contain<br>
(1)	prokaryotic DNA elements coding for a bacterial replication origin and an antibiotic resistance marker to provide for the growth and selection of the expression vector in a bacterial host;<br>
(2)	eukaryotic DNA elements that control initiation of transcription, such as a promoter; and<br>
(3)	DNA elements that control the processing of transcripts, such as a transcription termination/polyadenylation sequence.<br>
Therefore, another aspect of the present invention concerns vectors, preferably plasmid vectors, for use in the expression of the fusion polypeptides of the invention, which contain the polynucleotide sequences described herein which code for the fusion polypeptides of the invention.  Appropriate expression vectors which can transform procaryotic or eucaryotic cells include expression vectors comprising nucleotide sequences coding for the fusion molecules joined to transcriptional and translational regulatory<br><br>
sequences which are selected accoding to the host cells used.  For expression in E. coli, vectors such as that described by M.W. Robertson, J. Biol. Chem. 268 [1993] 12736-12743 can be used.  The product is expected to be disulfide-bonded, but not glycosylated.  For expression in yeast, an expression vector such as pHIL-D2 supplied with the Invitrogen (San Diego, CA, USA) Pichia pastoris expression kit (catalogue # K1710-01) can be used.  The protein product is expected to be disulfide-bonded and glycosylated.   Other suitable yeast expression systems include Saccharomvces cerevisiae and Kluveromyces lactis.  For expression in baculovirus, vectors such as pAC360, pVL1392 and pVL1393 (Invitrogen, San Diego, CA, USA) are useful for infection of insect cells, which would be expected to secrete a glycosylated and disulfide-bonded product.<br>
The fusion polypeptide of the invention normally is a glycoprotein, particularly when expressed in mammalian cells, and the invention includes fusion polypeptides in any glycosylation, or disulfide bridging, state.  In particular, mutational analysis suggests that N-l inked glycosylation at the first, second and seventh positions of the N-linked glycosylation sites of the IgE receptor a chain (A. Shimizu et al., PNAS USA 85 [1988] 1907-1911, Figure 2) (corresponding to amino acid residues 46, 67 and 191 of SEQ.ID.NO. 1), promotes biological activity of the IgE   molecule and monomers and dimers comprising it.  The most preferred expression system is one in which any sugars added will be most similar to those in the native molecule from which the polypeptide is derived. Yeast and insect cells are known to modify glycoproteins differently from mammalian cells, whereas E. coli does not add sugar molecules after secretion.  Therefore, while expression from any of these expression systems can yield a protein product which is useful for diagnostic applications (for example, the detection of an allergic condition), the most preferred form for expressing this product for use as a therapeutic molecule is expression in mammalian cells or possibly expression in the milk of transgenic mammals.<br>
For expression in a mammalian host, the transcriptional and translational regulatory signals used in an expression cassette may be derived from viral sources, such as adenovirus, bovine papilloma virus or simian virus, in which the regulatory signals are associated with a particular gene which has a high level of expression.  Suitable transcriptional and translational regulatory sequences also can be obtained from mammalian genes, such as actin, collagen, myosin, and metallothionein genes.  Transcriptional regulatory sequences include a promoter region sufficient to direct the initiation of RNA synthesis in a mammalian cell.  Examples of typical mammalian cells are Chinese hamster<br><br>
'    ovary (CHO) cells, SV40-transformed monkey kidney cells (COS), HeLa, BHK, NIH Swiss mouse embryo cells, rat, monkey or human fibroblasts, or rat hepatoma cells.<br>
For expression in mammalian cells, the preferred method is secretion from CHO cells. Neither CHO nor human cells add the al,3-linked galactose residues which are typical of expression in murine cells such as C127 and SP2/0 cells.  Antibodies to this sugar linkage are present in human serum  [C.F. Goochee et al., BioTechnol. 9  [1991] 1347-1355] and can affect the half-life, accessibility and clearance of recombinant products expressed from these murine cells.  CHO, dhfr- cells are mutant for dihydrofolate reductase (DHFR), and therefore are unable to synthesize purines, thymidine and glycine de novo. The copy number of chromosomally integrated plasmids bearing wild type copies of the DHFR gene can be increased or amplified by exposing cells transformed with these plasmids to increasing levels of methotrexate, a folate analogue which competes for folate binding at the active site of the enzyme.  A suitable vector for expression in CHO, dhfr-cells is pMT2 (R.J. Kaufman et al.  EMBO J. 6 [1987] 187-193)].  The pMT2 vector has a wild type copy of the DHFR gene which is transcribed as a single mRNA with the foreign gene.  Therefore, upon treatment of transformed CHO, dhfr- cells with increasing concentrations of methotrexate, both the foreign gene and the DHFR gene are co-amplified. The product secreted from these cells is expected to be disulfide-bonded and glycosylated in a mammalian pattern.<br>
An expression vector can be introduced into host cells using a variety of techniques including calcium phosphate transfection, liposome-mediated transfection, electroporation, and the like.  Preferably, transfected cells are selected and propagated wherein the expression vector is stably integrated in the host cell genome to produce stable transformants.  The cells can be cultured, for example, in DMEM media.  The polypeptide secreted into the media can be recovered by standard biochemical approaches following transient expression 24 - 72 hours after transfection of the cells or after establishment of stable cell lines following selection by e.g. antibiotic resistance.<br>
Conventional methods for recovering an expressed polypeptide from a culture include fractionation of the polypeptide-containing portion of the culture using well known biochemical techniques.  For instance, the methods of gel filtration, gel chromatography, ultrafiltration, electrophoresis, ion exchange or affinity chromatography, such as are known for protein fractionations, can be used to isolate the expressed proteins found in the culture. In addition, conventional immunochemical methods, such as immunoaffmity or<br><br>
immunoabsorption can be performed.  Techniques for transformation, culture, amplification, screening and product production and purification are also well-known (see e.g. J. Sambrook et al. [1989], supra;  RJ. Kaufman, Genetic Engineering:  Principles and Methods, Plenum Press, Vol. 9 [1987] 156-198).<br>
The fusion polypeptides of the invention are indicated for use therapeutically to treat mammalian, and in particular, human, patients suffering from allergies. The IgE-binding domain of the fusion polypeptides competes for IgE with the IgE receptor naturally present on mast cells, basophils and Langerhans cells, so that IgE is bound to the administered protein and unable to bind to these allergy effector cells to mediate the allergic response. The IgE-binding domain also competes with the IgE receptor, FceRI, by binding to autoantibodies to FCERI.<br>
Therefore the present invention provides a pharmaceutical composition for competitively binding IgE (and/or autoantibodies to FceRI), and/or inhibiting production of IgE, and thus for suppressive and/or preventative treatment of IgE- or IgE-receptor-mediated disorders, and more specifically, allergy and conditions associated with allergy, such as atopic dermatitis, atopic asthma and chronic urticaria.<br>
By "IgE- or IgE receptor- mediated disorders" is meant disorders associated with the binding of the cell-bound IgE receptor, FceRI, to IgE or to auto-antibodies to  FceRI, e.g. type   allergic reactions (nhyper-IgE syndrome") such as bronchial asthma, atopic asthma, hay fever, pollen allergy, allergic rhinitis, atopic dermatitis, eczema, anaphylaxis, as well as chronic urticaria, and also non-allergic Kimura's disease, and other pulmonary, dermatological or autoimmune diseases.<br>
In particular, atopic dermatitis, one of the most dramatic manifestations of atopy, is a chronic inflammatory skin disease associated with high serum IgE levels and a sensitization to various environmental allergens (M.-A. Morren et al., J. Am. Acad. Dermatol. 31 [1994] 467-473).  Current treatment of atopic dermatitis concentrates on the use of steroid-containing creams, and severe cases of atopic dermatitis have been successfully treated with cyclosporin A (H. Granlund et al., Br. J. Dermatol 132 [1995] 106-112), but the side effects restrict this treatment to a minority of the patients. An agent that inhibits the functions of IgE such as mast cell degranulation and IgE-mediated antigen presentation by B cells and other antigen presenting cells would be superior to any presently known treatment of atopic dermatitis and in addition also be useful for other milder forms of allergy.<br><br>
In addition to atopic dermatitis and atopic asthma, the polypeptides of the invention can be used to treat or prevent chronic urticaria (CU), in which mast cell degranulation through activation of FceRIa plays a role.  The fusion polypeptides of the invention can clear circulating autoantibodies against FceRIa, in contrast to anti-IgE monoclonal antibodies alternatively proposed for treatment of the disease.<br>
The polypeptides may be administered in the form of a pharmaceutical composition comprising a polypeptide of the invention, preferably an unmodified polypeptide, or a pharmaceutically acceptable salt thereof, in a pharmaceutically acceptable carrier or diluent.<br>
The term "salt" refers in particular to pharmaceutically acceptable salts prepared from pharmaceutically acceptable, non-toxic acids to form acid addition salts of e.g. an amino group of the polypeptide chain, or from pharmaceutically acceptable non-toxic bases to form basic salts of e.g. a carboxyl group of the polypeptide chain.   Such salts may be formed as internal salts and/or as salts of the amino or carboxylic acid terminus of the polypeptide of the invention.<br>
Suitable pharmaceutically acceptable acid addition salts are those of pharmaceutically acceptable, non-toxic organic acids, polymeric acids, or inorganic acids. Examples of suitable organic acids comprise acetic, ascorbic, benzoic, benzenesulfonic, citric, ethanesulfonic, fumaric, gluconic, glutamic, hydrobromic, hydrochloric, isothionic, lactic, maleic, malic, mandelic, methanesulfonic, mucic, nitric, oxalic, pamoic, pantothenic, phosphoric, salicylic, succinic, sulfuric, tartaric and p-toluenesulfonic, as well as polymeric acids such as tannic acid or carboxymethyl cellulose.  Suitable inorganic acids include mineral acids such as hydrochloric, hydrobromic, sulfuric, phosphoric and nitric acid.<br>
Examples of suitable inorganic bases for forming salts of a carboxyl group include the alkali metal salts such as sodium, potassium and lithium salts; the alkaline earth salts such as calcium, barium and magnesium salts; and ammonium, copper, ferrous, ferric, zinc, manganous, aluminum and manganic salts.  Preferred are the ammonium, calcium, magnesium, potassium, and sodium salts. Examples of pharmaceutically acceptable organic bases suitable for forming salts of a carboxyl group include organic amines, such as trimethylamine, triethylamine, tri(n-propyl)amine, dicyclohexylamine, P-(dimethylamino)ethanol, tris(hydroxymethyl)aminomethane, triethanolamine, (3-(diethylamino)ethanol, arginine, lysine, histidien, N-ethylpiperidine, hydrabamine, choline,<br><br>
betaine, ethylenediamine, glucosamine, methylglucamine, theobromine, purines, piperazines, piperidines, caffeine and procaine.<br>
Acid addition salts of the polypeptides may be prepared in conventional manner by contacting the polypeptide with one or more equivalents of the desired inorganic or organic acid such as hydrochloric acid.   Salts of carboxyl groups of the peptide may be conventionally prepared by contacting the peptide with one or more equivalents of a desired base such as a metallic hydroxide base, e.g. sodium hydroxide; a metal carbonate or bicarbonate base such as sodium carbonate or sodium bicarbonate; or an amine base such as triethylamine or triethanolamine.<br>
The invention thus also concerns pharmaceutical compositions comprising a novel fusion polypeptide as defined above or a pharmaceutically acceptable salt thereof, together with a pharmaceutically acceptable carrier or diluent.  The carrier is preferably a sterile, pyrogen-free, parenterally acceptable liquid.  Water, physiological saline, aqueous dextrose, and glycols are preferred liquid carriers or diluents, particularly (when isotonic) for injectable solutions.  The composition may be a conventionally prepared lyophilizate.<br>
The compositions can be administered systemically, i.e. parenterally (e.g. intramuscularly, intravenously, subcutaneously or intradermally), or by intraperitoneal administration.   For parenteral administration, it is preferred that the fusion polypeptides be essentially soluble in patient serum or plasma, e.g. that at least 1 milligram of polypeptide is soluble in one mililiter of serum or plasma.<br>
The compositions can also be administered by known techniques for topical administration.  Examples of suitable dosage forms include sprays, opthalmic solutions, nasal solutions and ointments.  For example, a spray can be manufactured by dissolving the peptide in an appropriate solvent and putting it in a spray to serve as an aerosol for commonly employed inhalation therapy.  An opthalmic or nasal solution can be manufactured by dissolving the active ingredient in distilled water, adding any auxiliary agent required, such as a buffer, isotonizing agent, thickener, preservative, stabilizer, surfactant or antiseptic, adjusting the mixture to pH 4 to 9.  Ointments can be obtained e.g. by preparing a composition from a polymer solution, such as 2 % aqueous carboxyvinyl polymer, and a base, such as 2 % sodium hydroxide, mixing to obtain a gel, and mixing with the gel an amount of purified fusion polypeptide.<br><br>
The composition is preferably administered suDcutaneousiy or intravenously, ana most typically, subcutaneously.<br>
The invention also comprises a method of treatment of allergic conditions comprising administration of a therapeutically effective ^amount of a fusion polypeptide of the invention or a pharmaceutically acceptable salt thereof to a patient in need of such treatment.  The method of treatment is practiced during the course of an allergic disease state (i.e. when the relief of symptoms is specifically required); or as a continuous or prophylactic treatment (i.e. prior to the onset of an anticipated IgE- or IgE-receptor -mediated disorder, such as an allergic reaction).<br>
The effective dosage in accordance herewith can vary over a wide range taking into consideration e.g. the degree or severity of the condition being treated, the age, sex and condition of the subject, the length of treatment, and the potency of the particular fusion protein, factors which are determinable by conventional methods.<br>
Since individual subjects widely vary in their IgE (as well as antibody to FCERI)<br>
content, a therapeutically effective dosage in accordance herewith can best be described as<br>
2	4<br>
being between 5x10   and 1 x 10   times the total content of serum IgE and antibodies to<br>
FceRI, on a molar scale.  The patient may be treated on a daily basis in single or multiple<br>
administration.  The composition may also be administered on a per month basis (or at such<br>
weekly intervals as may be appropriate), also in either single or multiple administrations.<br>
For an average subject (70 kg), a suitable monthly dosage can range from a lower dosage of about 0.5 mg per month to an upper dosage of about 500 mg per month, preferably of from about 1 mg to about 300 mg, more preferably of from about 20 mg to about 250 mg, per month, of the fusion polypeptide of the invention, such as the dimer of Example 7, conveniently administered in divided dosages once or twice a month.  Higher dosage ranges, e.g. 500 mg per month to 2 g per month, may be indicated in patients having high serum IgE concentrations or in early treatment phases.<br>
The dosage and timing of administration may vary.  Initial administrations of the composition may be at higher dosages within the above ranges, and administered more frequently than administrations later in the treatment of the disease.  Appropriate dosages may be determined by measuring the content of IgE and antibodies to FceRI in the patient's serum, as indicated above.  For example, early in the course of disease, the fusion polypeptide of the invention, such as the dimer of Example 7, may be administered in<br><br>
weekly doses of 200-500 mg of polypeptide in the average patient (70 kg). After clearance of serum IgE and antibody to FCERI, the treatment regimen may be reduced to weekly treatments or treatments every other week, with dosages ranging from 50 jag to 100 mg of polypeptide per treatment.<br>
The compositions of the present invention can be administered either alone or in combination with other compounds of the present invention or further pharmaceutical agents such as antihistamines or corticosteroids.<br>
The invention also comprises the use of the fusion polypeptides of the invention in an in vitro diagnostic assay of any standard format, e.g. ELISA, to determine the level of IgE or auto-antibodies to FceRI (e.g. in chronic urticaria patients) in a biological sample obtained from a human patient, e.g. blood or tissue samples.  The HSA component advantageously facilitates binding and detection of IgE or autoantibodies to FceRI in an ELISA formatted assay.  The amount of IgE or auto-antibodies present in the sample can serve as a measure of the allergic response of the patient to a substance to which the patient has been exposed. IgE or auto-antibody levels can also be measured to determine the efficiency of anti-allergy therapies, and to monitor a patient's allergic status over time.<br>
The invention further comprises a method of performing gene therapy in humans using a polynucleotide encoding a fusion polypeptide of the invention, for treatment of IgE- or IgE receptor- mediated disorders.  The gene therapy method comprises modifying cells of a patient by introducing therein a polynucleotide encoding a fusion polypeptide of the invention and expressing the polypeptide from such cells.  For example, somatic cells may first be removed from a patient, then genetically modified in culture by insertion of the polynucleotide, and the resultant modified cells reintroduced into the patient, whereby a polypeptide of the invention is expressed by the cells of the patient.<br>
Alternatively, the cells may be modified in vivo by direct insertion of vector DNA encoding the polypeptide.<br>
Suitable cells for modification include endothelial cells or leukocytes.  For gene therapy applications the polynucleotide of the invention is preferably under the control of a regulable (e.g. inducible) promoter.  Expression of the polypeptide may thereby be made dependent on exposure of the patient to an exogenous factor using known regulable promoter systems.<br><br>
The invention also includes utilizing trie metnoas or gene uierapy iu picpaic non-human somatic recombinant or transgenic animals expressing the fusion polypeptides of the invention, e.g. into milk.  Such modified non-human animals also form part of the invention.  Examples of useful animals include mice, rats, rabbits, pigs, sheep, goats and cattle, all of which have been made transgenic using standard techniques (e.g., D.R. Hurwitz et al., Transgenic Research 3 [1994] 365-375).  The animals may be used for modelling purposes or actual production of a protein.  In particular, the invention concerns a transgenic mouse, goat, cow or pig expressing a fusion polypeptide of the invention.  Methods for making such animals are well known.  A polynucleotide encoding the fusion protein of the invention can be introduced into the somatic cells of the animals, in vitro or in vivo, to produce somatic recombinant animals which are modified genetically but which cannot pass the genetic modification on to offspring.  Alternatively, the polynucleotide can be inserted into cells of embryos for production of transgenic animals able to pass on the capability of expressing the proteins of the invention to offspring.<br>
Transfection of cells for gene therapy can be accomplished in conventional manner, e.g. by electroporation, calcium phosphate precipitation, a lipofectin-based procedure, intramuscular injection, microinjection or through use of a "gene gun."  Plasmid-based vectors preferably contain a marker such as the neomycin gene for selection of stable transfectants with the cytotoxic aminoglycoside G418 in eukaryotic cells.<br>
Infection is accomplished by incorporating the genetic sequence for the fusion polypeptide into a retroviral vector.  Various procedures are known in the art for such incorporation.  One such procedure which has been widely used employs a defective murine retrovirus for packaging the retrovirus, and an amphotropic packaging cell line to prepare infectious amphotropic virus for use in infecting the target donor cells.<br><br>
Further characterization may be effected e,g. as follows:<br>
In vivo determination of serum half life<br>
General procedure:<br>
Female SKHl/hr/hr Charles River mice weighing approximately 25 g are injected intravenously with test proteins diluted in sterile Ca+2, Mg2+ - free PBS.  The mice are divided into groups as follows:<br>
-	Group (i) receiving 130 |ig (1 nmole) of fusion polypeptide, e.g. the dimer<br>
R	R<br>
IgE" - L1 - HSA II - L2 - IgE" prepared as in Example 7, or<br>
-	Group (ii) receiving 60 fag IgER (2 nmoles), or<br>
-	Group (iii) receiving 65 |ag HSA I (1 nmole).<br>
100 |al of blood are taken from each mouse at 10 minutes, 30 minutes, 3 hours, 6 hours and 12 hours after injection. Serum is prepared by centrifugation.  Serum concentration levels of the various proteins are determined by  a) IgE receptor ELISA binding assay;   b) inhibition ELISA; or  c) HSA Sandwich ELISA, as follows: a)  IgE   ELISA binding assay:<br>
IgE serum concentration is determined by detection of IgE binding by the following Sandwich ELISA:   200 ng human IgE is immobilized in COSTAR Strip Plate-8 wells (Cambridge, MA, USA) in 100 \x\ coating buffer in a humidifed chamber at 4Â°C overnight. Each well is washed twice with 300 yb Ca2+, Mg2+ - free PBS, pH 7.2.  The plates are blocked for one hour at room temperature with 200 |il Ca2+, Mg2+ - free PBS containing 5% BSA (Sigma).  After washing twice with 300 |il Ca2+, Mg2+ - free PBS containing 0.05 % Tween 20 (PBST), samples diluted in 100 |il of 1:10 diluted (in Ca2+, Mg2+- free PBS) mouse serum are added and incubated for one hour at room temperature.<br>
The wells are washed twice with 300 \si PBS and incubated with 100 |il (lng) of a monoclonal anti-human IgE receptor antibody such as 5H5/F8 for one hour at room temperature.  Again, the wells are washed twice with 300 \xl PBST and incubated with 100 \il of goat anti-mouse IgG-HRP (Biorad, 1:2000 diluted in Ca2+, Mg2+ - free PBS) for 1 hour at room temperature.  Plates were washed three times with 300 (il of PBST,  and horse radish peroxidase (HRP) conjugates are detected with 100 \x\ of ABTS (Biorad) substrate. The reaction is stopped after 5 minutes with 100 |il of 3 % oxalic acid.  Color intensities are measured with an EASY READER photometer at 405 nm.<br><br>
D)  mniDition CJUSA:<br>
100 ng FceRIot is immobilized in Nunc 96 well Immunoplates (F96 cert. Maxisorb) in 100 nl coating buffer (0.1 M NaHC03, 0.01 % NaN3 pH:9.6) in a humidified chamber at 4Â°C overnight.  Each well is washed 4 times with 300 \A PBS, 0.05 % Tween 20 (washing buffer).  To different sets of wells, either a negative control (50 \d of mouse serum diluted 1:25 in PBS, 0.05 % Tween 20, 2 % FCS), a dilution series of the fusion polypeptide standard, e.g.  IgER - L^ - HSAII - L2 - IgER  standard (dilutions from 400 ng/ml to 1.6 ng/ml) or dilution series of the samples are added.  The standard and the samples are diluted in mouse serum diluted 1:25 in PBS, 0.05 % Tween 20, 2 % FCS.  Immediately afterwards 50 |il of 400 ng/^1 of human IgE-Biotin conjugate in dilution buffer is added and mixed.  The final mouse serum dilution in the incubation mixture is 1:50.<br>
After incubation for two hours at  37Â°C, the plates are washed 4 times with 300 |il washing buffer, and 50 |il steptavidine-alkaline phosphatase conjugate (Gibco) diluted 1:1000 in dilution buffer is added and incubated for one hour at 37Â°C.  The plates are washed 4 times with 300 |jl washing buffer and after addition of 100 \xl substrate (1 mg/ml p-nitrophenyl-phosphate in diethanolamine buffer, pH 9.8 [BIORAD]) the reaction is stopped after incubation for 30 minutes at 37Â°C with 50 \x\ of 2 M NaOH.   Optical densities are measured with a BIOMEK-1000 workstation photometer at 405 nm.  The quantitative evaluation from the standard curves (4-parameter logistic curve fitting) is made with the Beckman IMMUNOFIT ELISA evaluation program. Calculation:<br>
% binding  =   [OD (sample or standard value) / OD (buffer value)]  x   100 c)   HSA Sandwich ELISA:<br>
500 ng monoclonal anti-mouse HSA (HSA-9) is immobilized in Nunc 96 well Immunoplates (F96 cert. Maxisorb) in 100 |il coating buffer (0.1 M NaHC03, 0.01 % NaN3 pH:9.6) in a humidified chamber at 4Â°C overnight.  Each well is washed 4 times with 300 \x\ washing buffer (PBS, 0.05 % Tween 20).   100 ^1 of 1:100 diluted mouse serum (PBS, 0.05 % Tween 20, 2 % FCS = dilution buffer) as negative control, or 100 pi standard (1 fig/ml - 2 ng/ml human serum albumin, KABI) or 100 |il of sample diluted in 1:100 diluted mouse serum is added.  After incubation for two hours at 37Â°C, the plates are washed 4 times with 300 |il washing buffer, and 100 \xl of 1 ng/^1 rabbit anti-HSA-Biotin conjugate diluted in dilution buffer is added. The anti-HSA-Biotin conjugate is purified by immunoaffinity chromatography with CH-Seph4B-HSA, and cross reactivities with mouse serum are removed by immunosorbtion with mouse serum-agarose.   After incubation for<br><br>
two hours at 37Â°C, the plates are washed 4 times .with 300 jixl washing buffer, and 50 jil streptavidin-alkaline phosphatase conjugate (Gibco) diluted 1:1000 in dilution buffer is added and incubated for one hour at 37Â°C.  The plates are washed 4 times with 300 jal washing buffer and after addition of 100 |xL substrate (Img/ml p-nitrophenylphosphate in diethanolamine buffer, pH 9.8, BIO-RAD),  the reaction is stopped after incubation for 15 minutes at 37Â°C with 50 |il 2M NaOH.  Optical densities are measured with a BIOMEK-1000 workstation photometer at 405 nm.  The quantitative evaluation from the standard curve (4-parameter logistic curve fitting) is made with the Beckman IMMUNOFIT ELISA evaluation program.<br>
Results:<br>
Concentrations for the dimeric fusion polypeptide,  IgER - L1 - HSA II - L2 - IgER, the free IgER (referred to as "free alpha chain") or HSA I (referred to as "HSA") as a function of elapsed time after injection, are given in pmoles/ml serum  in Figure 1(A) and (B).<br>
Figure 1(B) shows that the free receptor is only detectable in mouse serum for about 10 minutes, whereas the dimeric fusion polypeptide is still detectable about 12 hours after administration.<br>
Figure 1(C) shows the relative clearance kinetics for HSA and the dimeric fusion polypeptide.   After stabilization of tissue-blood distribution in the first 10 minutes, the dimer and HSA show an almost identical clearance curve. This confirms that the serum half life of an IgE-binding domain can be substantially prolonged by fusion to HSA.<br>
Inhibition of passive cutaneous anaphylaxis (PCA) in mice<br>
General procedure:<br>
(a)  Administration of polypeptide:<br>
Serial dilutions of 10, 50 or 500 (ig/kg of fusion polypeptide, e.g.  IgER - L1 - HSA II - L2 - IgER  obtained from CHO cells in Example 7, are intravenously injected into female SKHl/hr/hr Charles River mice weighing approximately 25 g at varying intervals prior to sensitization.<br>
Three groups of mice are established depending on the amount of polypeptide injected prior to sensitization (i.e. 10 ng/kg, 50 Mg/kg or 500 |ig/kg).  A fourth group of<br><br>
control mice receive 200 |ig/kg PBS intravenously instead of polypeptide.  The four groups of mice are each divided into three sub-groups, which differ by the interval between intravenous injection of the compound and intracutaneous sensitization with IgE [step (b) below].  The tested intervals are:  5 minutes, 15 minutes and 30 minutes.<br>
(b)	Intracutaneous sensitization:<br>
The mice are anaesthetized and sensitized into the back skin by 4 intradermal injections of 5 ng each monoclonal mouse anti-dinitrophenyl (DNP) IgE antibody in 10ml PBS (BioMakor, Rehovot, Israel).  In the control group, saline is injected intradermally in one site.<br>
(c)	Allergen challenge:<br>
90 minutes after sensitization, the mice are anesthesized again and challenged by intravenous injection of a solution containing 50 (ig of dinitrophenyl-bovine serum albumin (DNP-BSA) (Calbiochem-Behring, San Diego, USA) containing 1% Evans blue.  The PCA response was quantified by measuring the diameter of the stained test site due to extravasation.<br>
Results:<br>
These are depicted in the bar graph in Figure 2 for the mature fusion polypeptide of Example 7 (amino acids Va^g-Leu^g of SEQ.ID.NO.3).   At a concentration of 500 fig/kg, the dimeric fusion polypeptide completely blocks PCA at all time points of application. At 50 |Jg/kg, treatment at 30 minutes prior to challenge gives a statistically significant reduction of 36 %. At 15 minutes prior to challenge, a trend is seen with administration of both 10 and 50 ug/kg of dimer.  Thus, the dimeric fusion polypeptide is efficacious in preventing PCA.<br>
The procedures and techniques for carrying out the present invention are known in the art.   Insofar as their preparation is not particularly described herein, the compounds, reagents, vectors, cell-lines, etc. to be used are known and readily available or may be obtained in conventional manner from known and readily available materials, or equivalent materials may be prepared in conventional manner from known and readily available materials.<br><br>
The following non-limitative Examples illustrate the invention.  All temperatures are in degrees Centigrade.<br>
Materials and methods<br>
PCR amplification:<br>
De novo chemical synthesis of the primers of the invention can be conducted using any suitable method, such as, for example, the phosphotriester or phosphodiester methods.<br>
Methods and systems for amplifying a specific nucleic acid sequence are described in USP 4'683'195 and USP 4'683'202; and in Polymerase Chain Reaction, H.A. Erlich et al., Eds., Cold Spring Harbor Laboratory Press [1989].<br>
Following PCR, the DNA fragments are excised and purified using the QiaEx protocol (Qiagen, Inc., Chatsworth, CA, USA), then subcloned into a TA vector [TA CloningÂ® Kit (Invitrogen) (product literature, Version 2.2)].  The primers used in generating PCR amplified nucleic acids are set forth in FIG. 8.  DNA is sequenced using the Sequenase method (USB, Cleveland, OH, USA). Plasmids and Reagents:<br>
The FceRIcx cDNA clone,  pGEM-3-110B-l   (A. Shimizu et al., Proc.Nat.Acad.Sci.USA 85 [19881 1907-1911) is obtained from the American Type Tissue Collection (ATCC stock #67566).   Single-stranded human liver cDNA is obtained from Clontech (PCR-ready Quick Clone cDNA, Cat. D#7113-l). The sequence of HSA is available under GenBank Accession #s V00495, J00078, L00132 and L00133. Restriction enzymes are obtained from Boehringer-Mannheim or Gibco/BRL.  Taq DNA polymerase is obtained from Perkin-Elmer Cetus (PECI) or from Boehringer-Mannheim. The SK vector is obtained from Stratagene.<br>
pHIL-D2 is available from Invitrogen (San Diego, CA, USA; Catalog no. K1710-01 [1994])  (see "Pichia Expression Kit - Protein Expression - A Manual of Methods for Expression of Recombinant Proteins in Pichia pastoris - Version 3.0" [December 1994]) (hereinafter referred to as the "Invitrogen Manual").<br>
The pXMT3 vector is derived from pMT2 (Sambrook et al. (Eds.) [1989] supra) by cloning the PstI to EcoRI linker from pUC8 (Pharmacia) into the PstI and EcoRI sites of pMT2 (R.J. Kaufman et al. [1987] supra).<br>
Standard techniques as used below are described in Sambrook et al. (Eds.) r 19891 supra.<br><br>
Example A:  TA Cloning Vector<br>
Plasmid pCR2 is ligated separately with each of the PCR amplification products<br>
obtained in Examples 1 and 2.  The ligation reactions are performed using T4 DNA ligase<br>
in the following reaction mixture:<br>
25 mM Tris-HCl (pH 7.8)<br>
10 mM MgCl2<br>
1 mMDTT<br>
1 mM ATP<br>
50 ng vector DNA<br>
100-200 ng PCR reaction products (unpurified)<br>
4 units T4 DNA ligase (New England Biolabs)<br>
Each reaction mixture is maintained at 15Â° for 18 hours before being transformed<br>
into competent cells.<br>
Example 1:   PCR amplification and cloning of FceRIa cDNA<br>
FceRIa cDNA is purified from pGEM-3-110B-l using the Qiagen method.  Then: (A)  For preparing an HSA-leading construct,  PCR amplification of FceRIa cDNA  is carried out with oligonucleotides #18 and #19 (FIGS. 4, 8) using the following reaction<br>
mixture:<br>
1	jil ( 50 ng) of FceRIa cDNA;<br>
50 pmoles of each of oligonucleotides #18 and #19<br>
5 pi lOx PCR buffer (PECI)<br>
0.5 |il 20 mM dNTP stock solution  = 200 \xM final dNTPs concentration<br>
0.5 pi (2.5 U) Taq DNA polymerase (PECI)<br>
water to 50 fil.<br>
The reaction mixture is overlaid with mineral oil to prevent evaporation, and<br>
thermocycled in a Perkin Elmer Cetus DNA thermocycler model 480.  Cycling conditions<br>
for this reaction are: heat to 95Â° for 5 minutes, then 30 cycles of 94Â° for 1.5 minutes, 53Â° for<br>
2	minutes, 72Â° for 3 minutes, followed by a three minute extension at 72Â° and an overnight<br>
soak at 4Â°.<br>
Following electrophoresis an amplified product of -550 bp is confirmed by ethidium bromide staining.  The fragment is subcloned into PCR2 vector to yield pEKl encoding IgER (Figs. 4, 8B).<br><br>
(B)	For preparing an IgE-leading construct, PCR amplification of the FceRIot cDNA is<br>
carried out according to the procedure of (A) above except that oligonucleotides #20 and<br>
#31 (FIG. 8B) are used.  Following electrophoresis on 0.7 % agarose gel, an amplified<br>
product of -600 bp is confirmed by ethidium bromide staining.  The fragment is subcloned<br>
into PCR2 vector to form IgER/TA#l encoding pre-IgER (FIG. 4) .<br>
(C)	For preparing a construct encoding pre-IgER in order to express the mature, truncated<br>
protein for use as a control in assays, PCR amplification of FceRIa cDNA is carried out as<br>
in (A) above with oligonucleotides #20 and #19 (FIG. 8B).  Following electrophoresis, an<br>
amplified product of -620 bp is confirmed by ethidium bromide staining.  The PCR fragment<br>
is excised and subcloned into PCR II vector to provide IgERFL/TA #34, encoding pre-IgER<br>
followed by a stop codon (FIG. 4).<br>
Example 2:  PCR amplification and cloning of human serum albumin cDNA<br>
For obtaining a sequence encoding prepro-HSA II, PCR amplification of full length human serum albumin cDNA is carried out with oligonucleotides #24 and #25 (FIG. 8B), following the general procedure of Example 1(A).  The resulting clone HSA/TA#1 had the sequence most closely resembling the sequence in Genbank.  In this clone, seven mutations in the wobble position and one mutation which resulted in a change from lysine to glutamic acid at base 1333 were detected.  The seven mutations in the wobble position were: base 309: A to T;  base 744: A to G; base 795: G to A;  base 951: G to A;  base 1320: C to T;  base 1569: A to C;  base 1584: G to A (with reference to HSA/TA#1).  The lysine to glutamic acid mutation was corrected back to the sequence in Genbank using site-directed mutagenesis with the oligonucleotides shown in FIG. 8A and the Bio-Rad Mutagene Phagemid in vitro mutagenesis kit (Biorad, Cat #170-3581).  This method relies on the incorporation of uracil residues in the parental strand and their subsequent removal in the mutagenized strand (T.A. Kunkel Proc.Nat.Acad.Sci.USA 82 [19851 488-492).  Because the point mutations in the wobble position do not alter the native amino acid sequence of the encoded protein, the above seven mutations were not corrected.  Following electrophoresis, an amplified product of -1.8  kb is confirmed by ethidium bromide staining.<br>
The 1.8 kb product is subcloned into pCR2 vector to form HSA/TA mut #16, which is verified by DNA sequencing to contain the complete prepro-HSA II sequence.<br><br>
HSA/TA mut #16 is subcloned into Bluescript SK as a Spel, Hindm fragment, yielding HSA/SK #17 (FIG. 5).<br>
(A)	For preparing an HSA-leading construct HSA/SK#17 is digested with Mstll and Hindlll to remove the nucleotide sequence encoding the 3'-terminal amino acid (Leu609); and an oligonucleotide encoding linker L^, as defined above, is introduced at the 3'-terminus of the linearized HSA/SK#17 by kinasing oligonucleotides #28 and #29 (FIG. 8B) and annealing and ligating these fragments into the Mstll/Hindlll sites of the linearized HSA/SK#17, to yield pEK7 encoding prepro-HSA II fused to L2 (FIG. 8B). Miniprep DNA is checked by BamHI digestion and by sequencing.<br>
(B)	For preparing an IgE-leading construct, HSA/SK#17 encoding prepro-HSA II is amplified by PCR with oligonucleotides #26 and #27 (FIGS.  5, 8).   Oligonucleotide #26 removes the prepro sequence from HSA, and adds an oligonucleotide encoding L1 at the 5? end of HSA.  Oligonucleotide #27 ends at a naturally occurring, unique Ncol site at about nucleotide position 800 of the HSA coding sequence.  PCR amplification is carried out according to the procedure in Example 1(A).  An  approximately 800 bp fragment isolated by gel electrophoresis and Qia is subcloned into a pCR2 vector, to provide clone HSA Nco/TA #13, the sequence of which is verified by DNA sequencing.  The Ncol to NotI fragment from this clone is subcloned into cut Ncol and NotI HSA/SK #17 DNA, yielding HSA/SK #5 (FIG. 5) encoding L1 linked to the 5'-terminus of cDNA encoding HSA II.<br>
(C)	For expressing HSA alone, the HSA/SK #17 construct prepared above is employed.<br>
Example 3:  Fusion construct HSA-IgER/SK#22 encoding  prepro-HSA + Linker + IgER<br>
pEK7 (containing prepro-HSA II cDNA + oligonucleotide for L2) and pEKl (containing IgER) are digested with BamHI and Sail.  The obtained 1.8 kb fragment from pEK7 is phosphatased and then ligated to the 550 Kb fragment obtained from pEKl.  One positive miniprep, #23, was prepared but was contaminated with another unknown plasmid which prevented recovery of the HSA-IgER band.  Therefore, the 2.4 kb Spel to Sail fragment containing the HSA - IgE   fusion and the 2.9 kb fragment containing the vector DNA were purified from miniprep#23 and ligated together, resulting in clone<br><br>
HSA - IgE/SK#49 (FIG. 9) [vector sites were found to be missing from either end of the subcloned region, and accordingly, the 2.4 kb Spel to Sail fragment from HSA-IgE/SK#49 was subcloned into Spel plus Sail - cut Bluescript SK, resulting in HSA-IgE/SK#22 (not shown in the Figures)].<br>
The sequence of the junction of the HSA and linker with the IgE receptor DNA and the sequence of the ends of the fusion with the vector DNA were as expected in HSA-IgE/SK#22 as verified by DNA sequence analysis.<br>
Example 4:  Fusion construct IgE-HSA/SK#l encoding IgE R +  prepro-HSA II<br>
HSA/SK #5 and IgER/TA #1 are digested with SstI and Nhel. The 600 bp fragment from IgE/TA #1 is purified by gel electrophoresis and ligated into the cut and phosphatase-treated HSA/SK #5, yielding clone IgE-HSA/SK#l (FIG. 10).<br>
Example 5:   Fusion construct R-H-R/SK #50 encoding pre - IgER - LrHSA II -Lr IgER<br>
The PstI site unique to the HSA region of IgER-HSA/SK#l and HSA-IgER/SK#49 is used to join IgER and prepro - HSA II via the oligonucleotide for L^.  HSA/IgER/SK #49 and Bluescript SK are digested with PstI and Sail.  A 1.2 kb fragment containing the 3' portion of HSA II, the linker and the IgER sequence are ligated into PstI plus Sail - cut Bluescript DNA, resulting in HSA-IgER Pst Sal/SK #37 (FIG. 11), which is digested with PstI and Kpnl, and the 1.2 kb fragment is  prepared.<br>
IgER-HSA/SK#l DNA is digested with PstI and Kpnl, and a 4.8 kb fragment containing the vector, IgER, linker and 5' half of HSA are isolated, phophatase-treated, and ligated to the 1.2 kb fragment from HSA-IgER Pst Sal/SK #37.  The resultant dimeric construct R-H-R/SK #50 is obtained (FIG. 11).<br><br>
Example 6:  HSA II - L2 - IgEK monomeric fusion polypeptides by transfection and culture of Pichia pastoris<br>
Plasmid MB#2 encoding HSA II - L2 - IgER is prepared by cutting the plasmid HSA/SK#49 with EcoRI and isolating the 2.4 kb fragment encoding the fusion protein. This fragment is ligated into the unique EcoRI site of the Pichia pastoris expression vector pHIL-D2 (Invitrogen), after digestion of plasmid pHIL-D2 with EcoRI and alkaline phosphatase treatment.  The resultant MB#2 plasmid is linearized by digestion with NotI and transformed into his4 GS115 cells as described in the "Invitrogen Manual". His+ transformants are screened for growth on methanol.  Strains exhibiting slow growth on methanol are grown in minimal glycerol medium as specified in the "Invitrogen Manual" to stationary phase, transferred by centrifugation to buffered complex methanol medium and grown for 4 days.  The supernatant from the cells is then assayed by ELISA for the presence of HSA and for IgE-binding ability.  The IgE-binding ability of the product obtained secreted from P. pastoris was equivalent on a molar basis to the HSA concentration and fully biologically active.<br>
Example 7:   IgER- L1 â  HSA II - L2   - IgER dimeric fusion polypeptide by transfection and culture of CHO cells<br>
Plasmid pXMT3-RIa-HSA-RIa (containing the polynucleotide of SEQ. ID. NO. 4<br>
encoding pre- IgER- L1 - HSA II - L2   -IgER) is  prepared by digestion of R-H-R/SK #50<br>
(see Example 5) with EcoRI and isolation of the 3 kb EcoRI fragment encoding the dimeric fusion polypeptide.  This fragment is ligated into the unique EcoRI site of pXMT3 after digestion of pXMT3 with EcoRI and alkaline phosphatase-treatment.<br>
The plasmid is transfected into CHO DUKX Bl 1 cells.  These cells lack a functional dhfr-gene (dihydrofolate reductase) required for nucleoside synthesis. Therefore, cells are maintained in Alpha+ medium (MEM ALPHA MEDIUM with ribonucleosides and deoxyribonucleosides/Gibco) containing 10 % fetal calf serum (FCS). For transfection, the cells are washed twice in Ca++-Mg++-free PBS (CMF-PBS) and cell concentration is adjusted to 2x10   cells/ml in CMF-PBS.  0.8 ml of the cell suspension is added to 15 fig of<br><br>
piasmia un/\. iransrection is aone oy eiectroporation using a BIO RAD Gene Pulser (voltage = 1000 V; capacitor = 25 jiF). After transfection cells are cultured in 15 ml Alpha+ medium with 10 % FCS for 3 days.<br>
The dhfr-gene located on the pXMT3 plasmid allows selection for recombinant cells in a nucleoside depleted medium. 3 days after transfection cells are placed into Alpha-medium (MEM ALPHA MEDIUM without ribonucleosides and deoxyribonucleo-sides/Gibco) containing 10 % dialyzed fetal calf serum (FCSD). After 2 weeks of cultivation, recombinant cell colonies are visible. Cells are kept in Alpha- medium containing 10 % FCSD for 4 additional passages before gene amplification is started.<br>
In the presence of methotrexate (MTX), dhfr and the genes linked to it are amplified, resulting in an increased expression of the transgene. Therefore, selection for recombinant cells in a nucleoside-depleted medium is followed by cultivation in the presence of 20 nM MTX in Alpha- medium containing 10 % FCSD.  Further amplification is achieved by stepwise increases in methotrexate to 100 nM and 500 nM MTX.<br>
Protein is produced by seeding pool Tl/3-500 nM in Alpha- medium containing<br>
3      2 10 % FCS (GIBCO) at a density of 9x10 /cm   into roller bottles. The first supernatant is<br>
collected 5 days after seeding, followed by a switch to serum free Alpha- .  The second<br>
harvest is collected 3 days later, yielding a total of 1 liter supernatant for purification.<br>
A second batch is purified from 2 liter supernatant derived from pool T1/3-500 nM<br>
4 adapted to serum free growth conditions. Cells are seeded at 5x10   cells/ml and the<br>
supernatant collected 6 days after seeding.<br>
Example 8:   Purification of fusion protein<br>
The culture supernatants from Example 7 are purified by immunoaffinity chromatography on immobilized anti-FceRI monoclonal antibodies (e.g. 5H5-F8 ; mouse IgGl), produced and purified according to standard techniques: a)    Preparation of the chromatographic support:<br>
The monoclonal antibodies are coupled to CNBr-activated Sepharose 4B (Pharmacia, Uppsala, Sweden) at a density of 10 mg antibodies / ml gel according to the manufacturer's instructions. Excess reactive groups are subsequently blocked with ethanolamine and the resin stored in PBS supplemented with 0.02 % NaN3 until use.<br><br>
b)	Affinity chromatography:<br>
Clear culture supernatant is applied to a 5 ml antibody column equilibrated in PBS at a flow rate of 0.5 ml/min. The absorbed material is eluted in 50 mM citric acid, 140 mM NaCl, pH 2.70. Protein containing fractions are immediately adjusted to pH 7.0 (NaOH) followed by sterile filtration.<br>
c)	Quantification / characterization:<br>
The concentration of the dimeric fusion polypeptide is determined by absorption at 280 nm in its native conformation in 30 mM (3-[N-morpholino]propane)sulfonic acid (MOPS), pH 7.0 and in the denatured form (6 M guanidine.HCl). The corresponding molar absorption coefficient is calculated from the number of tryptophan, tyrosine and cystine residues using the tabulated absorption coefficients of these amino acids in model compounds and corrected for the difference in optical density between folded and unfolded protein. The fusion protein contains 17 tryptophans, 40 tyrosines and 21 cystines, which results in a theoretical extinction coefficient of 150840 M  cm   .  The quality of the purified material is assessed by standard SDS-PAGE and by N-terminus automated gas-phase Edman degradation sequencing and mass spectrometry.  Upon SDS-PAGE (FIG. 15) the polypeptide migrates with an apparent molecular weight of about 140 kDa, reflecting nhout 28 % elvcosvlation (theoretical MW without elvcosylation:   108'863.61 Da).<br><br>
SEQUENCE LISTING<br>
(1)	GENERAL INFORMATION;<br>
(i) APPLICANT:<br>
(A)	NAME: Novartis AG<br>
(B)	STREET: Schwarzwaldallee 215<br>
(C)	CITY: Basle<br>
(E)	COUNTRY: Switzerland<br>
(F)	POSTAL CODE (ZIP): CH-4058<br>
(G)	TELEPHONE: 61-324 5269<br>
(H) TELEFAX: 61-322 7532<br>
(ii) TITLE OF INVENTION:      FUSION POLYPEPTIDES<br>
(iii) NUMBER OF SEQUENCES: 6<br>
(iv) COMPUTER READABLE FORM:<br>
(A)	MEDIUM TYPE: Floppy disk<br>
(B)	COMPUTER: IBM PC compatible<br>
(C)	OPERATING SYSTEM: PC-DOS/MS-DOS<br>
(D)	SOFTWARE: Patentln Release #1.0, Version #1.30 (EPO)<br>
(v) CURRENT APPLICATION DATA:<br>
APPLICATION NUMBER: WO PCT/EP97/....<br>
(vi) PRIOR APPLICATION DATA:<br>
(A)	APPLICATION NUMBER: US 08/690216<br>
(B)	FILING DATE: 26-JUL-1996<br>
(2)	INFORMATION FOR SEP ID NO. 1:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 257 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear (ii) MOLECULE TYPE: protein<br>
(iii) HYPOTHETICAL: NO (iii) ANTI-SENSE: NO (v) FRAGMENT TYPE: internal<br><br>
i) SEQUENCE DESCRIPTION: SEQ ID NO: 1:<br>
Met Ala  Pro Ala Met  Glu  Ser  Pro  Thr  Leu Leu  Cys  Val  Ala  Leu  Leu<br>
15	10	15<br>
Phe  Phe Ala  Pro Asp  Gly Val  Leu Ala Val  Pro  Gin Lys   Pro  Lys  Val<br>
20	25	30<br>
Ser Leu Asn Pro Pro Trp Asn Arg lie Phe Lys Gly Glu Asn Val Thr<br>
35	40	45<br>
Leu Thr Cys Asn Gly Asn Asn Phe Phe Glu Val Ser Ser Thr Lys Trp<br>
50	55	60<br>
Phe His Asn Gly Ser Leu Ser Glu Glu Thr Asn Ser Ser Leu Asn lie<br>
65	70	75	80<br>
Val Asn Ala Lys Phe Glu Asp Ser Gly Glu Tyr Lys Cys Gin His Gin<br>
85	90	95<br>
Gin Val Asn Glu Ser Glu Pro Val Tyr Leu Glu Val Phe Ser Asp Trp<br>
100	105	110<br>
Leu Leu Leu Gin Ala Ser Ala Glu Val Val Met Glu Gly Gin Pro Leu<br>
115	120	125<br>
Phe Leu Arg Cys His Gly Trp Arg Asn Trp Asp Val Tyr Lys Val lie<br>
130	135	140<br>
Tyr Tyr Lys Asp Gly Glu Ala Leu Lys Tyr Trp Tyr Glu Asn His Asn<br>
145	150	155	160<br>
lie Ser lie Thr Asn Ala Thr Val Glu Asp Ser Gly Thr Tyr Tyr Cys<br>
165	170	175<br>
Thr Gly Lys Val Trp Gin Leu Asp Tyr Glu Ser Glu Pro Leu Asn lie<br>
180	185	190<br>
Thr Val lie Lys Ala Pro Arg Glu Lys Tyr Trp Leu Gin Phe Phe lie<br>
195	200	205<br>
Pro Leu Leu Val Val lie Leu Phe Ala Val Asp Thr Gly Leu Phe lie<br>
210	215	220<br>
Ser Thr Gin Gin Gin Val Thr Phe Leu Leu Lys lie Lys Arg Thr Arg<br>
225	230	235	240<br>
Lys Gly Phe Arg Leu Leu Asn Pro His Pro Lys Pro Asn Pro Lys Asn<br>
245	250	255<br>
Asn 257<br><br>
(2) INFORMATION FOR SEQ ID NO. 2:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 609 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear (ii) MOLECULE TYPE: protein<br>
(iii) HYPOTHETICAL: NO (iii) ANTI-SENSE: NO (v) FRAGMENT TYPE: internal<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 2:<br>
Met  Lys  Trp Val Thr  Phe  lie  Ser Leu Leu  Phe Leu Phe  Ser  Ser Ala<br>
15	10	15<br>
Tyr  Ser Arg Gly Val   Phe Arg Arg Asp Ala His  Lys   Ser  Glu  Val  Ala<br>
20	25	30<br>
His Arg Phe Lys Asp Leu Gly Glu Glu Asn Phe Lys Ala Leu Val Leu<br>
35	40	45<br>
lie Ala Phe Ala Gin Tyr Leu Gin Gin Cys Pro Phe Glu Asp His Val<br>
50	55	60<br>
Lys Leu Val Asn Glu Val Thr Glu Phe Ala Lys Thr Cys Val Ala Asp<br>
65	70	75	80<br>
Glu Ser Ala Glu Asn Cys Asp Lys Ser Leu His Thr Leu Phe Gly Asp<br>
85	90	95<br>
Lys Leu Cys Thr Val Ala Thr Leu Arg Glu Thr Tyr Gly Glu Met Ala<br>
100	105	110<br>
Asp Cys Cys Ala Lys Gin Glu Pro Glu Arg Asn Glu Cys Phe Leu Gin<br>
115	120	125<br>
His Lys Asp Asp Asn Pro Asn Leu Pro Arg Leu Val Arg Pro Glu Val<br>
130	135	140<br>
Asp Val Met Cys Thr Ala Phe His Asp Asn Glu Glu Thr Phe Leu Lys<br>
145	150	155	160<br>
Lys Tyr Leu Tyr Glu lie Ala Arg Arg His Pro Tyr Phe Tyr Ala Pro<br>
165	170	175<br>
Glu Leu Leu Phe Phe Ala Lys Arg Tyr Lys Ala Ala Phe Thr Glu Cys<br>
180	185	190<br>
Cys Gin Ala Ala Asp Lys Ala Ala Cys Leu Leu Pro Lys Leu Asp Glu<br>
195	200	205<br>
Leu Arg Asp Glu Gly Lys Ala Ser Ser Ala Lys Gin Arg Leu Lys Cys<br>
210	215	220<br><br>
Ala Ser Leu Gin Lys Phe Gly Glu Arg Ala Phe Lys Ala Trp Ala Val<br>
225	230	235	240<br>
Ala Arg Leu Ser Gin Arg Phe Pro Lys Ala Glu Phe Ala Glu Val Ser<br>
245	250	255<br>
Lys Leu Val Thr Asp Leu Thr Lys Val His Thr Glu Cys Cys His Gly<br>
260	265	270<br>
Asp Leu Leu Glu Cys Ala Asp Asp Arg Ala Asp Leu Ala Lys Tyr lie<br>
275	280	285<br>
Cys Glu Asn Gin Asp Ser lie Ser Ser Lys Leu Lys Glu Cys Cys Glu<br>
290	295	300<br>
Lys Pro Leu Leu Glu Lys Ser His Cys lie Ala Glu Val Glu Asn Asp<br>
305	310	315	320<br>
Glu Met Pro Ala Asp Leu Pro Ser Leu Ala Ala Asp Phe Val Glu Ser<br>
325	330	335<br>
Lys Asp Val Cys Lys Asn Tyr Ala Glu Ala Lys Asp Val Phe Leu Gly<br>
340	345	350<br>
Met Phe Leu Tyr Glu Tyr Ala Arg Arg His Pro Asp Tyr Ser Val Val<br>
355	360	365<br>
Leu Leu Leu Arg Leu Ala Lys Thr Tyr Glu Thr Thr Leu Glu Lys Cys<br>
370	375	380<br>
Cys Ala Ala Ala Asp Pro His Glu Cys Tyr Ala Lys Val Phe Asp Glu<br>
385	390	395	400<br>
Phe Lys Pro Leu Val Glu Glu Pro Gin Asn Leu lie Lys Gin Asn Cys<br>
405	410	415<br>
Glu Leu Phe Lys Gin Leu Gly Glu Tyr Lys Phe Gin Asn Ala Leu Leu<br>
420	425	430<br>
Val Arg Tyr Thr Lys Lys Val Pro Gin Val Ser Thr Pro Thr Leu Val<br>
435	440	445<br>
Glu Val Ser Arg Asn Leu Gly Lys Val Gly Ser Lys Cys Cys Lys His<br>
450	455	460<br>
Pro Glu Ala Lys Arg Met Pro Cys Ala Glu Asp Tyr Leu Ser Val Val<br>
465	470	475	480<br>
Leu Asn Gin Leu Cys Val Leu His Glu Lys Thr Pro Val Ser Asp Arg<br>
485	490	495<br>
Val Thr Lys Cys Cys Thr Glu Ser Leu Val Asn Arg Arg Pro Cys Phe<br>
500	505	510<br>
Ser Ala Leu Glu Val Asp Glu Thr Tyr Val Pro Lys Glu Phe Asn Ala<br>
515	520	525<br>
Glu Thr Phe Thr Phe His Ala Asp lie Cys Thr Leu Ser Glu Lys Glu<br>
530	535	540<br>
Arg Gin lie Lys Lys Gin Thr Ala Leu Val Glu Leu Val Lys His Lys<br>
545	550	555	560<br><br>
Fro Lys Ala Tnr bys uiu Gin i.eu bys Aia vai Met Asp Asp ?ne  Aia<br>
565	570	575<br>
Ala Phe Val Glu Lys Cys Cys Lys Ala Asp Asp Lys Glu Thr Cys Phe<br>
580	585	590<br>
Ala Glu Glu Gly Lys Lys Leu Val Ala Ala Ser Gin Ala Ala Leu Gly<br>
595	600	605<br>
Leu 609<br>
(2) INFORMATION FOR SEQ ID NO. 3:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 978 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear (ii) MOLECULE TYPE: protein<br>
(iii) HYPOTHETICAL: NO (iii) ANTI-SENSE: NO (v) FRAGMENT TYPE: internal<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 3:<br>
Met  Ala  Pro Ala Met  Glu  Ser  Pro  Thr  Leu  Leu  Cys  Val  Ala  Leu  Leu<br>
15	10	15<br>
Phe  Phe Ala  Pro  Asp  Gly Val  Leu Ala Val   Pro  Gin  Lys   Pro  Lys  Val<br>
20	25	30<br>
Ser Leu Asn Pro Pro Trp Asn Arg lie Phe Lys Gly Glu Asn Val Thr<br>
35	40	45<br>
Leu Thr Cys Asn Gly Asn Asn Phe Phe Glu Val Ser Ser Thr Lys Trp<br>
50	55	60<br>
Phe His Asn Gly Ser Leu Ser Glu Glu Thr Asn Ser Ser Leu Asn lie<br>
65	70	75	80<br>
Val Asn Ala Lys Phe Glu Asp Ser Gly Glu Tyr Lys Cys Gin His Gin<br>
85	90	95<br>
Gin Val Asn Glu Ser Glu Pro Val Tyr Leu Glu Val Phe Ser Asp Trp<br>
100	105	110<br>
Leu Leu Leu Gin Ala Ser Ala Glu Val Val Met Glu Gly Gin Pro Leu<br>
115	120	125<br>
Phe Leu Arg Cys His Gly Trp Arg Asn Trp Asp Val Tyr Lys Val lie<br>
130	135	140<br>
Tyr Tyr Lys Asp Gly Glu Ala Leu Lys Tyr Trp Tyr Glu Asn His Asn<br>
145	150	155	160<br><br>
lie Ser lie Thr Asn Ala Thr Val Glu Asp Ser Gly Thr Tyr Tyr Cys<br>
165	170	175<br>
Thr Gly Lys Val Trp Gin Leu Asp Tyr Glu Ser Glu Pro Leu Asn lie<br>
180	185	190<br>
Thr Val lie Lys Ala Pro Arg Glu Lys Tyr Trp Leu Ala Ser Gly Gly<br>
195	200	205<br>
Gly Gly Ser Asp Ala His Lys Ser Glu Val Ala His Arg Phe Lys Asp<br>
210	215	220<br>
Leu Gly Glu Glu Asn Phe Lys Ala Leu Val Leu lie Ala Phe Ala Gin<br>
225	230	235	240<br>
Tyr Leu Gin Gin Cys Pro Phe Glu Asp His Val Lys Leu Val Asn Glu<br>
245	250	255<br>
Val Thr Glu Phe Ala Lys Thr Cys Val Ala Asp Glu Ser Ala Glu Asn<br>
260	265	270<br>
Cys Asp Lys Ser Leu His Thr Leu Phe Gly Asp Lys Leu Cys Thr Val<br>
275	280	285<br>
Ala Thr Leu Arg Glu Thr Tyr Gly Glu Met Ala Asp Cys Cys Ala Lys<br>
290	295	300<br>
Gin Glu Pro Glu Arg Asn Glu Cys Phe Leu Gin His Lys Asp Asp Asn<br>
305	310	315	320<br>
Pro Asn Leu Pro Arg Leu Val Arg Pro Glu Val Asp Val Met Cys Thr<br>
325	330	335<br>
Ala Phe His Asp Asn Glu Glu Thr Phe Leu Lys Lys Tyr Leu Tyr Glu<br>
340	345	350<br>
lie Ala Arg Arg His Pro Tyr Phe Tyr Ala Pro Glu Leu Leu Phe Phe<br>
355	360	365<br>
Ala Lys Arg Tyr Lys Ala Ala Phe Thr Glu Cys Cys Gin Ala Ala Asp<br>
370	375	380<br>
Lys Ala Ala Cys Leu Leu Pro Lys Leu Asp Glu Leu Arg Asp Glu Gly<br>
385	390	395	400<br>
Lys Ala Ser Ser Ala Lys Gin Arg Leu Lys Cys Ala Ser Leu Gin Lys<br>
405	410	415<br>
Phe Gly Glu Arg Ala Phe Lys Ala Trp Ala Val Ala Arg Leu Ser Gin<br>
420	425	430<br>
Arg Phe Pro Lys Ala Glu Phe Ala Glu Val Ser Lys Leu Val Thr Asp<br>
435	440	445<br>
Leu Thr Lys Val His Thr Glu Cys Cys His Gly Asp Leu Leu Glu Cys<br>
450	455	460<br>
Ala Asp Asp Arg Ala Asp Leu Ala Lys Tyr lie Cys Glu Asn Gin Asp<br>
465	470	475	480<br>
Ser lie Ser Ser Lys Leu Lys Glu Cys Cys Glu Lys Pro Leu Leu Glu<br>
485	490	495<br><br>
Lys Ser His Cys lie Ala Glu Val .Glu Asn Asp Glu Met Pro Ala Asp<br>
500	505	510<br>
Leu Pro Ser Leu Ala Ala Asp Phe Val Glu Ser Lys Asp Val Cys Lys<br>
515	520	525<br>
Asn Tyr Ala Glu Ala Lys Asp Val Phe Leu Gly Met Phe Leu Tyr Glu<br>
530	535	540<br>
Tyr Ala Arg Arg His Pro Asp Tyr Ser Val Val Leu Leu Leu Arg Leu<br>
545	550	555	560<br>
Ala Lys Thr Tyr Glu Thr Thr Leu Glu Lys Cys Cys Ala Ala Ala Asp<br>
565	570	575<br>
Pro His Glu Cys Tyr Ala Lys Val Phe Asp Glu Phe Lys Pro Leu Val<br>
580	585	590<br>
Glu Glu Pro Gin Asn Leu lie Lys Gin Asn Cys Glu Leu Phe Lys Gin<br>
595	600	605<br>
Leu Gly Glu Tyr Lys Phe Gin Asn Ala Leu Leu Val Arg Tyr Thr Lys<br>
610	615	620<br>
Lys Val Pro Gin Val Ser Thr Pro Thr Leu Val Glu Val Ser Arg Asn<br>
625	630	635	640<br>
Leu Gly Lys Val Gly Ser Lys Cys Cys Lys His Pro Glu Ala Lys Arg<br>
645	650	655<br>
Met Pro Cys Ala Glu Asp Tyr Leu Ser Val Val Leu Asn Gin Leu Cys<br>
660	665	670<br>
Val Leu His Glu Lys Thr Pro Val Ser Asp Arg Val Thr Lys Cys Cys<br>
675	680	685<br>
Thr Glu Ser Leu Val Asn Arg Arg Pro Cys Phe Ser Ala Leu Glu Val<br>
690	695	700<br>
Asp Glu Thr Tyr Val Pro Lys Glu Phe Asn Ala Glu Thr Phe Thr Phe<br>
705	710	715	720<br>
His Ala Asp lie Cys Thr Leu Ser Glu Lys Glu Arg Gin lie Lys Lys<br>
725	730	735<br>
Gin Thr Ala Leu Val Glu Leu Val Lys His Lys Pro Lys Ala Thr Lys<br>
740	745	750<br>
Glu Gin Leu Lys Ala Val Met Asp Asp Phe Ala Ala Phe Val Glu Lys<br>
755	760	765<br>
Cys Cys Lys Ala Asp Asp Lys Glu Thr Cys Phe Ala Glu Glu Gly Lys<br>
770	775	780<br>
Lys Leu Val Ala Ala Ser Gin Ala Ala Leu Gly Gly Gly Gly Ser Val<br>
785	790	795	800<br>
Pro Gin Lys Pro Lys Val Ser Leu Asn Pro Pro Trp Asn Arg lie Phe<br>
805	810	815<br>
Lys Gly Glu Asn Val Thr Leu Thr Cys Asn Gly Asn Asn Phe Phe Glu<br>
820	825	830<br><br>
Val Ser Ser Thr Lys Trp Phe His _Asn Gly Ser Leu Ser Glu Glu Thr<br>
835	840	845<br>
Asn Ser Ser Leu Asn lie Val Asn Ala Lys Phe Glu Asp Ser Gly Glu<br>
850	855	860<br>
Tyr Lys Cys Gin His Gin Gin Val Asn Glu Ser Glu Pro Val Tyr Leu<br>
865	870	875	880<br>
Glu Val Phe Ser Asp Trp Leu Leu Leu Gin Ala Ser Ala Glu Val Val<br>
885	890	895<br>
Met Glu Gly Gin Pro Leu Phe Leu Arg Cys His Gly Trp Arg Asn Trp<br>
900	905	910<br>
Asp Val Tyr Lys Val lie Tyr Tyr Lys Asp Gly Glu Ala Leu Lys Tyr<br>
915	920	925<br>
Trp Tyr Glu Asn His Asn lie Ser lie Thr Asn Ala Thr Val Glu Asp<br>
930	935	940<br>
Ser Gly Thr Tyr Tyr Cys Thr Gly Lys Val Trp Gin Leu Asp Tyr Glu<br>
945	950	955	960<br>
Ser Glu Pro Leu Asn lie Thr Val He Lys Ala Pro Arg Glu Lys Tyr<br>
965	970	975<br>
Trp Leu 978<br><br>
(2) INFORMATION FOR SEP ID NO.  4:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 2955 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: double<br>
(D)	TOPOLOGY: linear (ii) MOLECULE TYPE: DNA<br>
(iii) HYPOTHETICAL: NO (iii) ANTI-SENSE: NO (v) FRAGMENT TYPE: internal<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 4:<br>
GAATTCACCA TGGCTCCTGC CATGGAATCC CCTACTCTAC TGTGTGTAGC CTTACTGTTC	60<br>
TTCGCTCCAG ATGGCGTGTT AGCAGTCCCT CAGAAACCTA AGGTCTCCTT GAACCCTCCA	120<br>
TGGAATAGAA TATTTAAAGG AGAGAATGTG ACTCTTACAT GTAATGGGAA CAATTTCTTT	180<br>
GAAGTCAGTT CCACCAAATG GTTCCACAAT GGCAGCCTTT CAGAAGAGAC AAATTCAAGT	240<br>
TTGAATATTG TGAATGCCAA ATTTGAAGAC AGTGGAGAAT ACAAATGTCA GCACCAACAA	300<br>
GTTAATGAGA GTGAACCTGT GTACCTGGAA GTCTTCAGTG ACTGGCTGCT CCTTCAGGCC	3 60<br>
TCTGCTGAGG TGGTGATGGA GGGCCAGCCC CTCTTCCTCA GGTGCCATGG TTGGAGGAAC	42 0<br>
TGGGATGTGT ACAAGGTGAT CTATTATAAG GATGGTGAAG CTCTCAAGTA CTGGTATGAG	480<br>
AACCACAACA TCTCCATTAC AAATGCCACA GTTGAAGACA GTGGAACCTA CTACTGTACG	540<br>
GGCAAAGTGT GGCAGCTGGA CTATGAGTCT GAGCCCCTCA ACATTACTGT AATAAAAGCT	600<br>
CCGCGTGAGA AGTACTGGCT TGCTAGCGGT GGAGGTGGAT CCGATGCACA CAAGAGTGAG	660<br><br>
GTTGCTCATC GGTTTAAAGA TTTGGGAGAA GAAAATTTCA AAGCCTTGGT GTTGATTGCC	720<br>
TTTGCTCAGT ATCTTCAGCA GTGTCCATTT GAAGATCATG TAAAATTAGT GAATGAAGTA	780<br>
ACTGAATTTG CAAAAACATG TGTTGCTGAT GAGTCAGCTG AAAATTGTGA CAAATCACTT	840<br>
CATACCCTTT TTGGAGACAA ATTATGCACA GTTGCAACTC TTCGTGAAAC CTATGGTGAA	900<br>
ATGGCTGACT GCTGTGCAAA ACAAGAACCT GAGAGAAATG AATGCTTCTT GCAACACAAA	960<br>
GATGACAACC CAAACCTCCC CCGATTGGTG AGACCAGAGG TTGATGTGAT GTGCACTGCT	102 0<br>
TTTCATGACA ATGAAGAGAC ATTTTTGAAA AAATACTTAT ATGAAATTGC CAGAAGACAT	1080<br>
CCTTACTTTT ATGCCCCGGA ACTCCTTTTC TTTGCTAAAA GGTATAAAGC TGCTTTTACA	1140<br>
GAATGTTGCC AAGCTGCTGA TAAAGCTGCC TGCCTGTTGC CAAAGCTCGA TGAACTTCGG	12 00<br>
GATGAAGGGA AGGCTTCGTC TGCCAAACAG AGACTCAAGT GTGCCAGTCT CCAAAAATTT	1260<br>
GGAGAAAGAG CTTTCAAAGC ATGGGCAGTA GCTCGCCTGA GCCAGAGATT TCCCAAAGCT	1320<br>
GAGTTTGCAG AAGTTTCCAA GTTAGTGACA GATCTTACCA AAGTCCACAC GGAATGCTGC	1380<br>
CATGGAGATC TGCTTGAATG TGCTGATGAC AGGGCGGACC TTGCCAAGTA TATCTGTGAA	1440<br>
AATCAAGATT CGATCTCCAG TAAACTGAAG GAATGCTGTG AAAAACCTCT GTTGGAAAAA	1500<br>
TCCCACTGCA TTGCCGAAGT GGAAAATGAT GAGATGCCTG CTGACTTGCC TTCATTAGCT	1560<br>
GCTGATTTTG TTGAAAGTAA GGATGTTTGC AAAAACTATG CTGAGGCAAA GGATGTCTTC	1620<br>
CTGGGCATGT TTTTGTATGA ATATGCAAGA AGGCATCCTG ATTACTCTGT CGTGCTGCTG	1680<br>
CTGAGACTTG CCAAGACATA TGAAACCACT CTAGAGAAGT GCTGTGCCGC TGCAGATCCT	1740<br>
CATGAATGCT ATGCCAAAGT GTTCGATGAA TTTAAACCTC TTGTGGAAGA GCCTCAGAAT	1800<br>
TTAATCAAAC AAAATTGTGA GCTTTTTAAG CAGCTTGGAG AGTACAAATT CCAGAATGCG	1860<br>
CTATTAGTTC GTTACACCAA GAAAGTACCC CAAGTGTCAA CTCCAACTCT TGTAGAGGTC	192 0<br><br>
TCAAGAAACC TAGGAAAAGT GGGCAGCAAA TGTTGTAAAC ATCCTGAAGC AAAAAGAATG	1980<br>
CCCTGTGCAG AAGACTATCT ATCCGTGGTC CTGAACCAGT TATGTGTGTT GCATGAGAAA	2040<br>
ACGCCAGTAA GTGACAGAGT CACCAAATGC TGCACAGAAT CCTTGGTGAA CAGGCGACCA	2100<br>
TGCTTTTCAG CTCTGGAAGT CGATGAAACA TACGTTCCCA AAGAGTTTAA TGCTGAAACA	2160<br>
TTCACCTTCC ATGCAGATAT ATGCACACTT TCTGAGAAGG AGAGACAAAT CAAGAAACAA	2220<br>
ACTGCACTTG TTGAGCTTGT GAAACACAAG CCCAAGGCAA CAAAAGAGCA ACTGAAAGCT	22 80<br>
GTTATGGATG ATTTCGCAGC TTTTGTAGAG AAGTGCTGCA AGGCTGACGA TAAGGAGACC	2340<br>
TGCTTTGCCG AGGAGGGTAA AAAACTTGTT GCTGCAAGTC AAGCTGCCTT AGGTGGAGGT	2400<br>
GGATCCGTCC CTCAGAAACC TAAGGTCTCC TTGAACCCTC CATGGAATAG AATATTTAAA	2460<br>
GGAGAGAATG TGACTCTTAC ATGTAATGGG AACAATTTCT TTGAAGTCAG TTCCACCAAA	2520<br>
TGGTTCCACA ATGGCAGCCT TTCAGAAGAG ACAAATTCAA GTTTGAATAT TGTGAATGCC	2580<br>
AAATTTGAAG ACAGTGGAGA ATACAAATGT CAGCACCAAC AAGTTAATGA GAGTGAACCT	2640<br>
GTGTACCTGG AAGTCTTCAG TGACTGGCTG CTCCTTCAGG CCTCTGCTGA GGTGGTGATG	2700<br>
GAGGGCCAGC CCCTCTTCCT CAGGTGCCAT GGTTGGAGGA ACTGGGATGT GTACAAGGTG	2760<br>
ATCTATTATA AGGATGGTGA AGCTCTCAAG TACTGGTATG AGAACCACAA CATCTCCATT	282 0<br>
ACAAATGCCA CAGTTGAAGA CAGTGGAACC TACTACTGTA CGGGCAAAGT GTGGCAGCTG	2880<br>
GACTATGAGT CTGAGCCCCT CAACATTACT GTAATAAAAG CTCCGCGTGA GAAGTACTGG	2940<br>
CTATAGTAAG AATTC  2955<br><br>
(2) INFORMATION FOR SEP ID NO.  5:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 1827 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: double<br>
(D)	TOPOLOGY: linear (ii) MOLECULE TYPE: DNA<br>
(iii) HYPOTHETICAL: NO (iii) ANTI-SENSE: NO (v) FRAGMENT TYPE: internal<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 5:<br>
ATGAAGTGGG TAACCTTTAT TTCCCTTCTT TTTCTCTTTA GCTCGGCTTA TTCCAGGGGT	60<br>
GTGTTTCGTC GAGATGCACA CAAGAGTGAG GTTGCTCATC GGTTTAAAGA TTTGGGAGAA	120<br>
GAAAATTTCA AAGCCTTGGT GTTGATTGCC TTTGCTCAGT ATCTTCAGCA GTGTCCATTT	180<br>
GAAGATCATG TAAAATTAGT GAATGAAGTA ACTGAATTTG CAAAAACATG TGTAGCTGAT	240<br>
GAGTCAGCTG AAAATTGTGA CAAATCACTT CATACCCTTT TTGGAGACAA ATTATGCACA	3 00<br>
GTTGCAACTC TTCGTGAAAC CTATGGTGAA ATGGCTGACT GCTGTGCAAA ACAAGAACCT	360<br>
GAGAGAAATG AATGCTTCTT GCAACACAAA GATGACAACC CAAACCTCCC CCGATTGGTG	420<br>
AGACCAGAGG TTGATGTGAT GTGCACTGCT TTTCATGACA ATGAAGAGAC ATTTTTGAAA	480<br>
AAATACTTAT ATGAAATTGC CAGAAGACAT CCTTACTTTT ATGCCCCGGA ACTCCTTTTC	540<br>
TTTGCTAAAA GGTATAAAGC TGCTTTTACA GAATGTTGCC AAGCTGCTGA TAAAGCTGCC	600<br>
TGCCTGTTGC CAAAGCTCGA TGAACTTCGG GATGAAGGGA AGGCTTCGTC TGCCAAACAG	660<br>
AGACTCAAAT GTGCCAGTCT CCAAAAATTT GGAGAAAGAG CTTTCAAAGC ATGGGCAGTG	72 0<br><br>
GCTCGCCTGA GCCAGAGATT TCCCAAAGCT GAGTTTGCAG AAGTTTCCAA GTTAGTGACA	780<br>
GATCTTACCA AAGTCCACAC GGAATGCTGC CATGGAGATC TGCTTGAATG TGCTGATGAC	840<br>
AGGGCGGACC TTGCCAAGTA TATCTGTGAA AATCAGGATT CGATCTCCAG TAAACTGAAG	900<br>
GAATGCTGTG AAAAACCTCT GTTGGAAAAA TCCCACTGCA TTGCCGAAGT GGAAAATGAT	960<br>
GAGATGCCTG CTGACTTGCC TTCATTAGCT GCTGATTTTG TTGAAAGTAA GGATGTTTGC	102 0<br>
AAAAACTATG CTGAGGCAAA GGATGTCTTC CTGGGCATGT TTTTGTATGA ATATGCAAGA	1080<br>
AGGCATCCTG ATTACTCTGT CGTGCTGCTG CTGAGACTTG CCAAGACATA TGAAACCACT	1140<br>
CTAGAGAAGT GCTGTGCCGC TGCAGATCCT CATGAATGCT ATGCCAAAGT GTTCGATGAA	1200<br>
TTTAAACCTC TTGTGGAAGA GCCTCAGAAT TTAATCAAAC AAAACTGTGA GCTTTTTAAG	1260<br>
CAGCTTGGAG AGTACAAATT CCAGAATGCG CTATTAGTTC GTTACACCAA GAAAGTACCC	1320<br>
CAAGTGTCAA CTCCAACTCT TGTAGAGGTC TCAAGAAACC TAGGAAAAGT GGGCAGCAAA	13 80<br>
TGTTGTAAAC ATCCTGAAGC AAAAAGAATG CCCTGTGCAG AAGACTATCT ATCCGTGGTC	1440<br>
CTGAACCAGT TATGTGTGTT GCATGAGAAA ACGCCAGTAA GTGACAGAGT CACAAAATGC	1500<br>
TGCACAGAGT CCTTGGTGAA CAGGCGACCA TGCTTTTCAG CTCTGGAAGT CGATGAAACA	1560<br>
TACGTTCCCA AAGAGTTTAA TGCTGAAACA TTCACCTTCC ATGCAGATAT ATGCACACTT	1620<br>
TCTGAGAAGG AGAGACAAAT CAAGAAACAA ACTGCACTTG TTGAGCTTGT GAAACACAAG	1680<br>
CCCAAGGCAA CAAAAGAGCA ACTGAAAGCT GTTATGGATG ATTTCGCAGC TTTTGTAGAG	1740<br>
AAGTGCTGCA AGGCTGACGA TAAGGAGACC TGCTTTGCCG AGGAGGGTAA AAAACTTGTT	1800<br>
GCTGCAAGTC AAGCTGCCTT AGGCTTA  1827<br><br>
(2) INFORMATION FOR SEP ID NO.  6:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 773 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: double<br>
(D)	TOPOLOGY: linear (ii) MOLECULE TYPE: DNA<br>
(iii) HYPOTHETICAL: NO (iii) ANTI-SENSE: NO (v) FRAGMENT TYPE: internal (xi) SEQUENCE DESCRIPTION: SEQ ID NO: 6:<br>
ATGGCTCCTG CCATGGAATC CCCTACTCTA CTGTGTGTAG CCTTACTGTT	CTTCGCTCCA	60<br>
GATGGCGTGT TAGCAGTCCC TCAGAAACCT AAGGTCTCCT TGAACCCTCC	ATGGAATAGA	120<br>
ATATTTAAAG GAGAGAATGT GACTCTTACA TGTAATGGGA ACAATTTCTT	TGAAGTCAGT	180<br>
TCCACCAAAT GGTTCCACAA TGGCAGCCTT TCAGAAGAGA CAAATTCAAG	TTTGAATATT	240<br>
GTGAATGCCA AATTTGAAGA CAGTGGAGAA TACAAATGTC AGCACCAACA	AGTTAATGAG	300<br>
AGTGAACCTG TGTACCTGGA AGTCTTCAGT GACTGGCTGC TCCTTCAGGC	CTCTGCTGAG	360<br>
GTGGTGATGG AGGGCCAGCC CCTCTTCCTC AGGTGCCATG GTTGGAGGAA	CTGGGATGTG	420<br>
TACAAGGTGA TCTATTATAA GGATGGTGAA GCTCTCAAGT ACTGGTATGA	GAACCACAAC	480<br>
ATCTCCATTA CAAATGCCAC AGTTGAAGAC AGTGGAACCT ACTACTGTAC	GGGCAAAGTG	540<br>
TGGCAGCTGG ACTATGAGTC TGAGCCCCTC AACATTACTG TAATAAAAGC	TCCGCGTGAG	600<br>
AAGTACTGGC TACAATTTTT TATCCCATTG TTGGTGGTGA TTCTGTTTGC	TGTGGACACA	660<br>
GGATTATTTA TCTCAACTCA GCAGCAGGTC ACATTTCTCT TGAAGATTAA	GAGAACCAGG	720<br>
AAAGGCTTCA GACTTCTGAA CCCACATCCT AAGCCAAACC CCAAAAACAA	CTG  773<br><br><br><br><br>
WE CLAIM:<br>
1.	A fusion polypeptide or a salt thereof comprising the polypeptide of SEQ ID. NO.3 or the polypeptide of residues Val26-Leu978 of SEQ ID .NO.3 fused to at least one human serum albumin (HSA) component, or a physiologically functional equivalent thereof.<br>
2.	An isolated polynucleotide which is an intermediate in the preparation of a polypeptide or salt thereof according to claim 1 which comprises residues Val26-Leu978ofSEQ.ID.N0.3.<br>
3.	A process for the preparation of a polypeptide or salt thereof according to claim 1, which comprises<br><br>
(a)	transforming a host cell with a vector comprising DNA encoding a polypeptide according to claim 1 or a physiologically functional equivalent thereof;<br>
(b)	expressing the polypeptide or its physiologically functional equivalent in that cell, whereby the physiologically functional equivalent polypeptide is modified to yield a fusion polypeptide as claimed in claim 1; and<br>
(c)	recovering the resultant polypeptide from the host cell, optionally in the form of a salt thereof.<br>
4.	A vector comprising DNA encoding a polypeptide or salt thereof according to<br>
claim 1 or encoding a physiologically functional equivalent thereof.<br><br>
5. A pharmaceutical composition comprising a polypeptide according to claim 1 or a pharmaceutical^ acceptable salt thereof, together with a known pharmaceutically acceptable carrier or diluent.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxMC1tYXMtMTk5NyBhYnN0cmFjdCBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1610-mas-1997 abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxMC1tYXMtMTk5NyBjbGFpbXMgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">1610-mas-1997 claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxMC1tYXMtMTk5NyBkZXNjcnB0aW9uIChjb21wbGV0ZSkgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">1610-mas-1997 descrption (complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxMC1tYXMtMTk5NyBkcmF3aW5ncyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1610-mas-1997 drawings duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxMC1tYXMtMTk5Ny0gYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">1610-mas-1997- abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxMC1tYXMtMTk5Ny0gYXNzaWdubWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">1610-mas-1997- assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxMC1tYXMtMTk5Ny0gY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">1610-mas-1997- claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxMC1tYXMtMTk5Ny0gY29ycmVzcG9uZGVuY2Ugb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">1610-mas-1997- correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxMC1tYXMtMTk5Ny0gY29ycmVzcG9uZGVuY2UgcG8ucGRm" target="_blank" style="word-wrap:break-word;">1610-mas-1997- correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxMC1tYXMtMTk5Ny0gZGVzY3JpcHRpb24gY29tcGxldGUucGRm" target="_blank" style="word-wrap:break-word;">1610-mas-1997- description complete.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxMC1tYXMtMTk5Ny0gZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">1610-mas-1997- drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxMC1tYXMtMTk5Ny0gZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">1610-mas-1997- form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxMC1tYXMtMTk5Ny0gZm9ybSAyNi5wZGY=" target="_blank" style="word-wrap:break-word;">1610-mas-1997- form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxMC1tYXMtMTk5Ny0gZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">1610-mas-1997- form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxMC1tYXMtMTk5Ny0gZm9ybSA0LnBkZg==" target="_blank" style="word-wrap:break-word;">1610-mas-1997- form 4.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="228883-a-method-for-suppressing-silence-frames-in-a-stream-of-media.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="228885-high-purity-oxygen-production-by-pressure-swing-adsorption.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>228884</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1610/MAS/1997</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>12/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>20-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>11-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>17-Jul-1997</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NOVARTIS AG</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>SCHWARZWALDALLEE 215, CH-4058 BASEL,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>MARY ELLEN DIGAN</td>
											<td>46 JUNARD DRIVE, MORRISTOWN, NJ 07960,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>PHILIP LAKE</td>
											<td>70 BROOKLAWN DRIVE, MORRIS PLAINS, NJ 07950,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>HERMANN GRAM</td>
											<td>AM BISCHOFSACKER 20, D-7858 WEIL AM RHEIN,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/62</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>08/690216</td>
									<td>1996-07-26</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/228884-a-fusion-polypeptide-and-a-process-for-preparing-them by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:31:11 GMT -->
</html>
