<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/228871-an-isolated-polynucleotide-sequence-of-the-par1-gene by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:30:18 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 228871:AN ISOLATED POLYNUCLEOTIDE SEQUENCE OF THE PAR1 GENE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">AN ISOLATED POLYNUCLEOTIDE SEQUENCE OF THE PAR1 GENE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention relates to polynucleotide sequences comprising genetic variations of PAR 1 gene on positions 3090 and/or 3329. Surprisingly, the presence of said variants in humans correlates with specific cardiovascular diseases. The invention also relates to methods for the detection of said genetic variations.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Description<br>
Analysis and use of PARI polymorphisms for evaluating the risk of cardiovascular disorders.<br>
The invention relates to polynucleotide sequences comprising genetic variations of the PARI gene at positions 3090 and/or 3329.<br>
The protease-activated receptor 1 (PARI) is a thrombin receptor which belongs to the class of G protein-coupled receptors (GCPR). The gene for PARI is located on chromosome 5q13, consists of two exons and covers a region of approx. 27 kb. PARI is expressed in, inter alia, endothelial cells, smooth muscles cells, fibroblasts, neurons and human platelets. In platelets, PARI is an important signal transduction receptor which is involved in the initiation of platelet aggregation.<br>
PARs are activated via proteolytic removal of a part of the N terminus of said PARs, whereby a new N-terminal sequence is exposed which then activates the receptor.<br>
PARI and PAR4 play a central part in the activation of platelets; the activation of these receptors in platelets leads to morphological changes, release of ADP and aggregation of said platelets.<br>
A connection of coronary heart diseases with single nucleotide polymorphisms (SNP) in the promoter region of PARI in a group of Korean patients was not confirmed. In another study, a PARI promoter variant was shown to have a protective action for the development of venous thromboembolisms.<br>
The sequence of the human PARI gene is known. The polynucleotide sequence of this gene can be accessed under the number NM-001992 at the NCBI nucleotide database. Likewise, the protein sequence is available under the number NP-001983 at the NCBI protein database. NCBI is the National Center for Biotechnology Information (postal address: National Center for Biotechnology Information, National Library of Medicine, Building 38A, Bethesda, MD 20894, USA; Web address: www.ncbi.nhm.nih.gov). The cloning of the PARI gene has been described, inter alia, in "Schmidt et al., J. Biol. Chem. 271, 9307-9312, 1996".<br><br>
There are various new polymorphisms of the PARI gene, by means of which it is possible to determine a relatively strong disposition of an individual for coronary heart diseases. The affected individuals are thus enabled to counteract this risk factor in time by adapting their life style accordingly, for example by compensating via increased control of other damaging influences such as smoking, alcohol consumption, cholesterol-rich food, high blood pressure etc.<br>
Such health-related preventive mechanisms would not be possible without knowledge of the PARI polymorphisms which are explained in more detail below and the use thereof in corresponding methods.<br>
Variants of a particular nucleotide sequence with substitutions at individual positions are known to the skilled worker under the term SNP (= single nucleotide polymorphism).<br>
The invention relates to an isolated polynucleotide sequence of the PARI gene, which comprises a C for T substitution at position 3090 of the PARI sequence according to NM-001992 which, as prior art, is publicly available. In a preferred embodiment, the polynucleotide sequence of the PARI gene having a T to C substitution at position 3090 encompasses a sequence according to SEQ ID NO: 2 and, in a particularly preferred embodiment of said polynucleotide sequence, the latter comprises a sequence of SEQ ID NO: 2.<br>
The invention furthermore relates to an isolated polynucleotide sequence of the PARI gene, which comprises an C for A substitution at position 3329 of the PARI sequence according to NM-001992 which, as prior art, is publicly available. In a preferred embodiment, the polynucleotide sequence of the PARI gene having an A to C substitution at position 3329 encompasses a sequence according to SEQ ID NO: 3 and, in a particularly preferred embodiment of said polynucleotide sequence, the latter comprises a sequence of SEQ ID NO: 3.<br>
The invention also relates to an isolated polynucleotide sequence of the PARI gene, which comprises a C for T substitution at position 3090 of the PARI sequence according to NM-001992 and, simultaneously, a V for A substitution at position 3329 of said PARI sequence. In a preferred embodiment, the polynucleotide sequence of the PARI gene having a T to<br><br>
C substitution at position 3090 and a simultaneous A to C substitution at position 3329 encompasses a sequence according to SEQ ID NO: 4 and, in a particularly preferred embodiment of said polynucleotide sequence, the latter comprises a sequence of SEQ ID NO; 4.<br>
The invention also relates to an isolated part of the polynucleotide sequence of the PARI gene, which comprises a sequence according to SEQ ID NO: 5.<br>
The invention also relates to an isolated part of the polynucleotide sequence of the PARI gene, which sequence comprises a C for T substitution at position 3090, based on the PARI sequence according to NM-001992, which part comprises a sequence according to SEQ ID NO: 6.<br>
The invention also relates to an isolated part of the polynucleotide sequence of the PARI gene, which sequence comprises a C for A substitution at position 3329, based on the PARI sequence according to NM-001992, which part comprises a sequence according to SEQ ID NO: 7.<br>
The invention also relates to an isolated part of the polynucleotide sequence of the PARI gene, which sequence comprises a C for T substitution at position 3090, based on the PARI sequence according to NM-001992, and simultaneously a C for A substitution at position 3329 of said PARI sequence, which part comprises a sequence according to SEQ ID NO: 8.<br>
The invention furthermore comprises the preparation of a 3592 base pair polynucleotide sequence of the PARI cDNA gene, which sequence may or may not comprise the polymorphisms at positions 3090 and 3329, as defined above, individually or in combination, which preparation comprises the following method steps:<br>
a]	Providing human DNA comprising a PARI sequence according to SEQ ID NO: 2 and/or a PARI sequence according to SEQ ID NO: 3 and/or a PARI sequence according to SEQ ID NO: 4,<br>
b]	Providing a primer pair having a sequence according to SEQ ID NO: 9 and SEQ ID NO: 10.<br>
c]	Amplifying the PARI polynucleotide sequence by the polymerase chain extension reaction (PCR),<br>
d]	Isolating and/or purifying the 3.56 kb fragment obtained from c],<br><br>
e]        Sequencing the fragment from d].<br>
The invention also relates to the preparation of a polynucleotide sequence according to SEQ ID NO; 5, SEQ ID NO: 6, SEQ ID NO: 7 or SEQ ID NO: 8, which preparation comprises the following method steps:<br>
a]	Providing human genomic DNA comprising a PARI sequence according to SEQ ID NO: 1 and/or a PARI sequence according to SEQ ID NO: 2 and/or a PARI sequence according to SEQ ID NO: 3 and/or a PARI sequence according to SEQ ID NO: 4<br>
b]	Providing a primer pair according to SEQ ID NO: 11 and SEQ ID NO: 12<br>
c]	Amplifying the fragment of the PARI polynucleotide sequence by the polymerase chain extension reaction (PCR),<br>
d]	Isolating and/or purifying the fragment obtained from c],<br>
e]	Sequencing the fragment from d].<br>
The invention furthermore relates to a method for detecting whether or not there is in a PARI gene a T to C substitution at position 3090 of the sequence according to NM-001992 and/or an A to C substitution at position 3329 of the sequence according to NM-001992, which method comprises the following method steps:<br>
a]	Providing biological material comprising human cells,<br>
b]	Obtaining chromosomal DNA from the material of a],<br>
c]	Amplifying a polynucleotide fragment by means of the primers according to SEQ ID NO: 11 and SEQ ID NO: 12, using a PCR reaction,<br>
d]	Sequencing the polynucleotide fragment from c].<br>
The invention furthermore relates to a method for detecting, whether or not there is in a PARI gene a T to C substitution at position 3090 of the sequence according to NM-001992 and/or an A to C substitution at position 3329 of the sequence according to NM-001992, which method comprises the following method steps:<br>
a]	Providing biological material comprising human cells,<br>
b]	Obtaining RNA from the material of a],<br>
c]	Transcribing said RNA to cDNA by means of reverse transcriptase,<br>
d]	Possibly amplifying a polynucleotide fragment by means of the primers according to SEQ ID NO: 10 and SEQ ID NO: 11, using said PCR reaction,<br><br>
e] Sequencing the cDNA from c] and/or the polynucleotide fragment from d].<br>
The invention also relates to a method for detecting whether or not there is in a PARI gene a T to C substitution at position 3090 of the sequence according to NM-001992 and/or an A to C substitution at position 3329 of the sequence according to NM-001992, which method comprises the following method steps:<br>
a]	Providing biological material comprising human cells,<br>
b]	Obtaining chromosomal DNA from the material of a],<br>
c]	Southern blotting the chromosomal DNA from b](<br>
d]	Providing a probe according to SEQ ID NO: 5 and/or SEQ ID NO: 6 and/or SEQ ID NO: 7 and/or SEQ ID NO: 8,<br>
e]	Hybridizing the Southern blot from c] with the probe form d] under stringent hybridization conditions,<br>
f]	Determining the presence or absence of a genetic variation in the PARI gene at position 3090 and/or 3329 according to NM-001992 by comparing the results of the hybridization from e].<br>
The invention furthermore relates to a method for detecting whether or not there is in a PARI gene a T to C substitution at position 3090 of the sequence according to NM-001992 and/or an A to C substitution at position 3329 of the sequence according to NM-001992, which method comprises the following method steps:<br>
a]	Providing biological material comprising human cells,<br>
b]	Obtaining RNA from the material of a],<br>
c]	Northern blotting the RNA from b],<br>
d]	Providing a probe according to SEQ ID NO: 5 and/or SEQ ID NO: 6 and/or SEQ ID NO: 7 and/or SEQ ID NO: 8,<br>
e]	Hybridizing the Northern blot form c] with the probe from d] under stringent hybridization conditions,<br>
f]	Determining the presence or absence of a genetic variation in the PARI gene at position 3090 and/or 3329 according to NM-001992 by comparing the results of the hybridization.<br>
Detection of the genetic variations or polymorphisms in the PARI gene at positions 3090 and/or 3329 may be used as (a) genetic marker for evaluating the risk of atrial fibrillation, acute coronary syndrome, cardiomyopathy and/or unstable angina,  as (b) marker for preventive<br><br>
treatment for atrial fibrillation, acute coronary syndrome, cardiomyopathy and/or stable angina of the carriers of the corresponding genetic variants, as (c) marker for adjusting the dose of a pharmaceutically active substance to be administered for atrial fibrillation, acute coronary syndrome, cardiomyopathy and/or unstable angina, as (d) marker for determining the high throughput-screening strategy for identifying a pharmaceutical^ active substance for atrial fibrillation, acute coronary syndrome, cardiomyopathy and/or unstable angina, as (e) marker for identifying the relevant individuals or patients for clinic studies in order to test the tolerability, safety and efficacy of a pharmaceutical substance for atrial fibrillation, acute coronary syndrome, cardiomyopathy and/or unstable angina, and as (f) basis for developing assays systems for analyzing the genetic variation in the PARI gene at the DNA, RNA or protein level.<br>
The invention also relates to an isolated polynucleotide sequence having from 21 to 50 nucleotides, which comprises a sequence according to SEQ ID NO: 11. Said sequence preferably comprises SEQ ID NO: 11. The invention furthermore relates to an isolated polynucleotide sequence having from 20 to 50 nucleotides, which comprises a sequence according to SEQ ID NO: 12. Said sequence preferably comprises SEQ ID NO: 12.<br>
The invention also relates to the use of an isolated polynucleotide sequence having from 21 to 50 nucleotides, which encompasses or comprises a sequence according to SEQ ID NO: 11, in combination with an isolated polynucleotide sequence having from 20 to 50 nucleotides, which encompasses or comprises a sequence according to SEQ ID NO: 12, for amplifying a corresponding fragment of the PARI gene by means of the polymerase chain extension reaction (PCR). This use preferably relates to the amplification of a fragment of a PARI gene having a T to C substitution at position 3090 of the sequence according to NM-001992 and/or having an A to C substitution at position 3329 of the sequence according to NM-001992.<br>
Moreover, the invention comprises a kit of parts which comprises<br>
a]	an isolated polynucleotide sequence of from 21 to 50 nucleotides in length, which encompasses or comprises a sequence according to SEQ ID NO: 11,<br>
b]	an isolated polynucleotide sequence of from 20 to 50 nucleotides in length, which encompasses or comprises a sequence according to<br><br>
SEQ ID NO: 12,<br>
c]	at least one enzyme for carrying out the polymerase chain extension reaction (PCR),<br>
d]	possibly substances and/or solutions for carrying out the polymerase chain extension reaction,<br>
e]	possibly polynucleotide sequences encompassing the PARI gene with or without substitution at position 3090 of the PARI sequence according to NM-001992 and/or position 3329 according to NM-001992 in full length and/or parts thereof<br>
f]	and possibly reagents for carrying out the sequencing reaction.<br>
Kit of parts here and below means the combination of said components which have been combined into a functional unit in spatial juxtaposition to each other.<br>
The invention furthermore relates to the preparation of the above-described kit of parts, which comprises<br>
a]	providing an isolated polynucleotide sequence of from 21 to 50 nucleotides in length, which encompasses or comprises a sequence according to SEQ ID NO: 11,<br>
b]	providing an isolated polynucleotide sequence of from 20 to 50 nucleotides in length, which encompasses or comprises a sequence according to SEQ ID NO: 12,<br>
c]	providing an enzyme for carrying out the polymerase chain extension reaction (PCR),<br>
d]	providing, where appropriate, reagents for carrying out a sequencing<br>
e]	possibly providing substances and/or solutions for carrying out said polymerase chain extension reaction (PCR)<br>
f]	possibly providing polynucleotide sequences comprising the PARI gene with or without a T to C substitution at position 3090 of the PARI sequence according to NM-001992 and/or an A to C substitution at position 3329 according to NM-001992, in each case in the full length, or parts thereof,<br>
g]	introducing the components from a] to f] in each case separately into suitable containers,<br>
h] combining, where appropriate, the containers from g] in one or more pack units.<br>
The above-described kit of parts may be used for amplifying a fragment of<br><br>
the PARI gene.<br>
The technical aspects of the invention are discussed in more detail in the following embodiments.<br>
Isolated polynucleotide sequences of the PARI gene may be prepared, for example, by amplification by means of the polymerase chain extension reaction (PCR). Suitable primers for this purpose are described in SEQ ID NO: 9 and SEQ ID NO: 10.<br>
The PCR is an in-vitro technique which may be used to selectively duplicate polynucleotide sections which are flanked by two known sequences. Amplification requires short, single-stranded DNA molecules which are complementary to the ends of a defined sequence of a DNA or RNA template (primers). A DNA polymerase extends the primers, under the correct reaction conditions and in the presence of deoxynucleotide triphosphates (dNTPs), along the single-stranded and denatured polynucleotide template and thus synthesizes new DNA strands whose sequence is complementary to said template. During this process, the temperature is changed at regular intervals so that, time after time, the polynucleotide strands are denatured and the primers can be attached and extended. Heat-stable DNA polymerases, for example Taq polymerase, are used. A typical PCR reaction mixture contains, apart from a polynucleotide template, two suitable primer nucleotides, for example at concentrations between 0.2 to 2 ^M, furthermore dNTPs, for example at concentrations of 200 p.M per dNPT, furthermore MgCfe having a concentration of 1 -2 mM, and 1-10 units of a heat-stable DNA polymerase such as, for example, Taq polymerase (Thermus aquaticus polymerase). Heat-stable DNA polymerase and the components for carrying out the same, and also protocols, are commercially supplied by numerous companies such as, for example, Roche Diagnostics, Clontech, Life Technologies, New England Biolabs, Promega, Stratagene, etc.<br>
The polynucleotide template for amplifying the polynucleotide sequence to be isolated may be present in the form of RNA or DNA. If the polynucleotide template is RNA, then the latter is transcribed to DNA by means of reverse transcriptase, prior to the actual PCR reaction. The amount of polynucleotide template for carrying out the PCR reaction may be from 0.01 to 20 ng, for example.<br><br>
The polynucleotide template is obtained using techniques known to the skilled worker for obtaining DNA and/or RNA from biological material. Biological material should include here, inter alia, the cells of a tissue or organ (e.g. brain, blood, liver, spleen, kidney, heart, blood vessels) of a vertebrate, including humans, or cells from a eukaryotic cell culture (e.g. Hela cells, CHO cells, 3T3 cells) or cells comprising bacteria or yeasts in which the DNA sequence to be isolated is present in cloned form.<br>
Cells of a tissue assemblage or organ of a vertebrate, including humans, may be obtained by taking blood, tissue puncture or surgical techniques. A polynucleotide template may be obtained therefrom, for example, by disrupting the cells, possibly concentrating individual organelles, in particular the nucleus, and recovering the DNA or RNA by precipitation and centrifugation.<br>
Another method for preparing isolated polynucleotide sequences of the PARI gene comprises cloning the PARI gene, subsequently expressing it in bacteria or yeast and purifying the expressed polynucleotide. The previously mentioned PCR reaction, for example, is suitable for preparing a polynucleotide fragment which is clonable. It is advantageous to use, for a fragment to be cloned, primers which carry the recognition sequence of a reaction enzyme 5' of the complementary sequence. The two primers may use in each case the same or different recognition sequences for restriction enzymes.<br>
Examples of common restriction enzymes are: BamHI (GGATCC), Clal (ATCGAT), EcoRI (GAATTC), EcoRV (GATATC), Hindlll (AAGCTT) Ncol (CCATGG) Sail (GTCGAC), Xbal (TCTAG1).<br>
For cloning, a vector is treated with the restriction enzymes which correspond to the recognition sequences attached to the primers. The fragment is connected to the vector by means of ligase by isolation and treatment with the same restriction enzymes. Vector means a DNA molecule such as, for example, a plasmid, bacteriophage or a cosmid, with the aid of which it is possible to clone genes or other DNA sequences and to introduce them into a bacterial or eukaryotic cell for replication. Examples of vectors are DNA molecules such as pBR322, pUC18/19, pBluescript, pcDNA3.1. Vectors are commercially available from specialist companies for biotechnological material, such as Roche Diagnostics, New England Biolabs, Promega, Stratagene etc.<br><br>
The instructions required for carrying out the PCR reaction, for providing polynucleotides or for carrying out cloning procedures can be found by the skilled worker in the form of recipes and protocols in standard manuals such as, for example, in a] "Current Protocols in Molecular Biology by Frederick M. Ausubel (Editor), Roger Brent (Editor), Robert E. Kingston (Editor), David D. Moore (Editor), J. G. Seidman (Editor), Kevin Struhl (Editor), loose leaf edition, continuously updated, John Wiley &amp; Sons, Inc., New York or in b] Short Protocols in Molecular Biology, 5th edition, by Frederick M. Ausubel (Editor), Roger Brent (Editor), Robert E. Kingston (Editor), David D. Moore (Editor), J. G. Seidman (Editor), John A. Smith (Editor), Kevin Struhl (Editor), October 2002, John Wiley &amp; Sons, Inc., New York" or in c] "Molecular Cloning by J. Sambrock, E. F. Fritsch, T. Maniatis; Cold Spring Harbor Laboratory Press".<br>
Suitable primer sequences are provided, for example, via chemical synthesis thereof which may be carried out commercially to order by companies such as MWG Biotech, etc.<br>
Human cDNA from different organs is commercially available from companies such as, for example, Promega, Stratagene or others.<br>
The sequencing of a polynucleotide is carried out by means of routine methods known to the skilled worker by using, for example, laboratory robots from companies such as, for example, Life Technologies, Applied Biosystems, BioRad or others.<br>
Isolated polynucleotide sequences of the PARI variant and fragments therefrom may also be used for hybridization at different stringencies. Stringency describes reaction conditions which influence the specificity of hybridization or attachment of two single-stranded nucleic acid molecules. The stringency and thus also specificity of a reaction can be increased by increasing the temperature and lowering the ionic strength. Low stringency conditions are present, for example, if the hybridization is carried out at room temperature in 2 x SSC solution. High stringency conditions are present, for example, if hybridization is carried out at 68°C in 0.1 x SSC/0.1% SDS solution.<br>
Hybridization under stringent hybridization conditions in accordance with<br><br>
the present application means:<br>
1] Hybridizing the labeled probe with the sample to be studied at 65°C (or, in the case of oligonucleotides, 5°C below the melting temperature) overnight in 50 mM Tris pH 7.5, 1NaCI, 1% SDS, 10% dextran sulfate, 0.5 mg/ml denatured salmon sperm DNA.<br>
2]       Washing at room temperature in 2 x SSC for 10 min.<br>
3] Washing at 65°C (or, in the case of oligonucleotides, 5°C below the melting temperature) in 1 x SSC/1% SDS for 30 min.<br>
4] Washing at 65°C (or, in the case of oligonucleotides, 5°C below the melting temperature) in 0.2 x SSC/0.1% SDS for 30 min.<br>
5] Washing at 65°C (or, in the case of oligonucleotides, 5°C below the melting temperature) in 0.1% SSC/0.1% SDS for 30 min.<br>
DNA fragments of 20 nucleotides in overall length are to be regarded as being oligonucleotides for this purpose. The melting temperature results from the formula Tm = 2 (number of A+T) + 4 (number of G+C)C°.<br>
A 2 x SSC or 0.1 x SSC solution is prepared by diluting a 20 x SSC solution accordingly. The 20 x SSC solution comprises a 3M NaCI/0.3 sodium citrate 2 H2O solution. SDS is sodium dodecyl sulfate.<br>
The hybridization is carried out by transferring the polynucleotides to be studied to a nylon or nitrocellulose membrane (Southern blot - DNA; Northern blot - RNA), after electrophoretic fractionation and subsequent denaturation. The hybridization is carried out using a probe which is radiolabeled or has been labeled in another way, for example with the aid of fluorescent dyes. The probe comprises a usually single-stranded and/or denatured DNA or RNA polynucleotide sequence which binds to the complementary nucleotide sequence of the once again single-stranded and/or denatured DNA or RNA polynucleotide sequence to be studied.<br>
Single nucleotide polymorphisms of the PARI gene may be detected with the aid of the primers of the invention, also by SSCP analysis. SSCP stands for Single Stranded Conformation Polymorphism which is an electrophoretic technique for identifying individual base pair substitutions. The polynucleotides to be studied are amplified by PCR by means of labeled primers and, after denaturation into single strands, fractionated in a polyacrylamide gel electrophoresis (PAGE). If the DNA fragments to be<br><br>
studied exhibit individual base pair substitutions, they then possess different conformations and thus migrate in the PAGE at different rates.<br>
Examples of substances for carrying out the PCR are buffers such as Hepes or Tris, furthermore dAPP, dGTP, dTTP, dCTP, and Mg2+ and possibly further divalent or monovalent irons. Solutions contain these substances in dissolved form-Examples Amplification of genomic regions of the PARI gene<br>
The T to C nucleotide substitution at position 3090 and the A to C substitution at position 3329 in the PARI sequence were detected using the following primers:<br>
Primer 1:       5'-ACAGAGTGGAATAAGACAGAG-3' (SEQ ID NO: 11) Primer 2:       5'-CCAGTGCTAGCTTCTACTTAC-3 (SEQ ID NO: 12) Primer 1 (SEQ ID NO: 11) corresponds to positions 2767 to 2789 of the NM-001992 reference sequence. Primer 2 is derived from Exon No. 1 of the PARI gene.<br>
PCR protocol for the amplification:<br>
The reagents used are from Applied Biosystems (Foster City, USA): 20 ng of genomic DNA; 1 unit of TaqGold DNA polymerase; 1 x Taq polymerase buffer; 500 µM of dNTPs; 2.5 µM MgCI2: 200 nM of each amplification primer pair; H2Oto 5 µl<br>
PCR amplification program for the genotyping<br>
95°C for 10 min	x 1 cycle<br>
95°C for 30 sec<br>
70°C for 30 sec	x 2 cycles<br>
95°C for 30 sec<br>
65°C for 30 sec	x 2 cycles;<br><br>
x 2 cycles;<br>
)5°C for 30 sec 50°C for 30 sec<br>
x 40 cycles;<br>
35°C for 30 sec 56°C for 30 sec 72°C for 30 sec<br>
x 1 cycle;<br>
72°Cfor10min 4°C for 30 sec<br>
Identification of SNPs<br>
Protocol for the minisequencing and detection of the SNPs.<br>
All reagents are from Applied Biosystems (Foster City, USA). 2 \x\ of purified PCR product, 1.5 µl of BigDye Terminator Kit, 200 nM sequencing primer; H2Oto 10 µl.<br>
Amplification program for the sequencing:<br>
96°C for 2 min	x 1 cycle<br>
96°C for 10 sec<br>
55°C for 10 sec<br>
65°C for 4 min	x 30 cycles<br>
72°C for 7 min<br>
4°C for 30 sec	x 1 cycle;<br>
Analysis of the sequencing products:<br>
The sequences were first analyzed using the sequence analysis software<br>
(Applied Biosystems, Foster City,  USA) to obtain the raw data, then<br>
processed using Phred, Phrap, Polyphred and Consed. Phred, Phrap,<br>
Polyphred and Consed are software written by Phil Green at Washington<br>
University<br>
(http://www.genome.washington.edu).<br>
Assigning PARI SNPs to coronary disorders<br><br>
1. a clinical study, two PARI polymorphisms from the 3'-noncoding region f the gene were studied for a connection with thrombotic and ardiovascular complications in a cohort of patients.<br>
he following abbreviations are used below (all positions indicated refer to le nucleotide positions in the reference sequence NM-001992).<br>
PAR1 T3090T describes the group of individuals whose alleles of the PARI<br>
iene both have a thymidine (T) at position 3090. These individuals are<br>
lomozygous with respect to this PARI variant.<br>
PAR1 T3090C describes the group of individuals whose one allele of the<br>
PAR1 gene has a cytidine (C) at position 3090 and whose other allele of<br>
he PARI gene has a thymidine (T) at position 3090. These individuals are<br>
heterozygous with respect to this PARI variant.<br>
PAR1 C3090C describes the group of individuals whose alleles of the<br>
PAR1 gene both have a cytidine (C) at position 3090. These individuals are<br>
lomozygous with respect to this PARI variant.<br>
PAR1 A3329A describes the group of individuals whose alleles of the<br>
PARI gene both have an adenosine (A) at position 3329. These individuals<br>
are homozygous with respect to this PARI variant.<br>
PARI A3329C describes the group of individuals whose one allele of the<br>
PARI gene has a cytidine (C) at position 3329 and whose other allele of<br>
the PARI gene has an adenosine (A) at position 3329. These individuals<br>
are heterozygous with respect to this PARI variant.<br>
PARI C3329C describes the group of individuals whose alleles of the<br>
PARI gene both have a cytidine (C) at position 3329. These individuals are<br>
homozygous with respect to this PARI variant.<br>
In the group of patients analyzed (Fig. 1), statistically significant associations of the homozygous carriers of the PARI variant C3090C with atrial fibrillation and cardiomyopathy were observed. After carrying out a logistic regression, a 1.97 fold increased risk of atrial fibrillation and a 1.84 fold increased risk of cardiomyopathy were found in homozygous carriers of the PARI variant C3090C compared to carriers of the PARI variants T3090/T3090T (Fig. 3).<br>
It was shown that, for carriers of the PARI variant C3329C, said variant is associated with a 2.35 fold increased risk of atrial fibrillation compared to carriers of the PARI variants C3329A/A3329A. In carriers of the PARI variant C3329C, said variant seems, in addition, to be protective with<br><br>
respect to the appearance of acute coronary syndrome and unstable angina. Carriers of the PARI variant C3329C have a 2.78 fold reduced risk of the appearance of acute coronary syndrome and/or unstable angina compared to carriers of the PARI variants A3329C/A3329A (Fig. 4).<br>
It is therefore possible, by means of a method of the invention and using an isolated PARI sequence of the particular SNP type or a fragment thereof, to determine for human individuals whether there is as assignment a risk group in accordance with the results presented.<br>
Preparation of plasmid DNA<br>
1 ml of a bacterial overnight culture is transferred to an Eppendorf tube and centrifuged (5 000 rpm for 5 min) in a Heraeus Biofuge. The bacterial cell pellet is to be resuspended in 100 \i\ of cooled solution I and then to be placed on ice for 5 min.<br>
Solution I: 25 mM tris-HCI, pH 8.0, 50 mM glucose (sterile-filtered) 10 mM EDTA 100 \xglm\ Rnase A.<br>
After addition of 200 \i\ of solution II, the entire mixture is mixed well, resulting in alkaline denaturation of the DNA.<br>
Solution II:    200 mM NaOH, 1% SDS.<br>
After subsequent incubation for 5 min on ice, 150 p,l of solution III are added to the mixture. This is followed by mixing once more and incubating on ice for a further 15 min.<br>
Solution III:   3 M sodium acetate (pH 4.8).<br>
Centrifugation in the Heraeus Biofuge at 12 000 rpm for 15 minutes removes the cell debris, the genomic DNA and the denatured proteins. The supernatant produced, which contains the plasmid DNA, is decanted into a second Eppendorf tube and admixed with 1 ml of 96% strength EtOH (or 300 JJ,I of isopropanol). The precipitation mixture is mixed thoroughly and again centrifuged (15 min at 12 000 rpm in Heraeus Biofuge). This results in precipitation of the plasmid DNA. The plasmid DNA sediment is washed with ice-cold 70% strength EtOH and then dried in air. Finally, the dry<br><br>
sediment is taken up in 50 µlof sterile distilled water.<br>
Alcohol precipitation of DNA<br>
Precipitation mixture:	DNA  solution,   1/10  volume  of  3 M  sodium<br>
acetate (pH 5.4), 2 to 3 volumes of 96% EtOH (1 volume of isopropanol).<br>
The mixture is mixed well and can be stored at -20°C, although this does not increase the precipitation yield. The plasmid DNA is sedimented by centrifugation at 12 000 rpm for 20 minutes.<br>
In order to remove residues of the sodium acetate used, the plasmid DNA must be washed once more with 1 ml of 70% strength EtOH after precipitation.<br>
Phenol extraction of DNA<br>
A DNA solution is admixed with the same volume of phenol (Rotiphenol®, equilibrated with TE buffer, pH 7.6, Roth, Karlsruhe, Germany), shaken for 5 min and centrifuged at 5000 rpm. Most of the now denatured proteins accumulate in the interface. The upper, aqueous phase contains the DNA and is carefully removed by suction, and then mixed with a chloroform/isoamyl alcohol mixture (24:1) in order to remove phenol residues. This is followed by another centrifugation, after which the aqueous supernatant is removed and the DNA is isolated from the solution by alcohol precipitation.<br>
Purification of amplified DNA molecules<br>
DNA amplicons are purified using a PCR purification kit (Qiagen). This removes the starter molecules, nucleotides (dNTPs), polymerases and salts. For this purpose, the PCR reaction mixture is admixed with five times the volume of PB buffer, mixed well and applied to the Qiaquick column. The amplified DNA is then selectively bound to the column material, and the dNPTs are removed by washing twice with 750 p,l of PE buffer. The amplified DNA is then eluted with the desired volume of water, with the best volume being the same as that of the PCR reaction mixture starting material.<br><br>
DNA cleavage with restriction enzymes<br>
Mix: 3µl of DNA, 2 µl of 10 x cleavage buffer, 2.5-5 U of restriction enzyme (e.g. EcoRI, BamHI, Sail, Xbal, Xhol etc.), add distilled water to a volume of 20 µl<br>
Depending on the restriction enzyme, the cleavage reaction runs at 25-55°C for 1-2 h. For analysis, the fragments are electrophoretically fractionated in an agarose or polyacrylamide gel in parallel with a length standard. If the reaction is a double cleavage, then first one enzyme is added to the mixture. After 1 hour, an aliquot is applied to an appropriate gel, and, if the cleavage has occurred, the second enzyme can be added. If the second enzyme does not cleave in the same cleavage buffer, then an alcohol precipitation is required first.<br>
Agarose gel electrophoresis of DNA<br>
The agarose (Roth) is dissolved in 1 x agarose buffer at the desired concentration and boiled in a microwave oven, until the agarose has completely dissolved. The solution is then poured into a sealed Plexiglass flat bed gel chamber.<br>
The DNA samples are admixed with 1/10 volume of loading blue (50% v/v glycerol; 50 mM EDTA; 0.005% w/v BPB [Merck, Darmstadt, Germany] and 0.005% xylene cyanol) and pipetted into the gel pockets which are generated by means of a comb.<br>
The electrophoresis is carried out horizontally in 1 x agarose buffer as running buffer at a constant voltage of 80-140 V, depending on the size of the gel and the distance between the electrodes.<br>
1 x agarose buffer: 40 mM Tris-HCI (pH 7.8), 5 mM sodium acetate, 1 mM EDTA.<br>
Polyacrylamide gel electrophoresis of DNA<br>
7.5% polyacrylamide gel solution; 0.94 ml of 40% strength acrylamide-bisacrylamide stock solution, 0.5 ml of 10 x TBE buffer (400 mM Tris-HCI, pH 8.3; 200 mM sodium acetate, 20 mM EDTA), 0.25 ml of 1% AMPS, 10 \i\ of TEMED, 3.33 ml of distilled water.<br><br>
This mixture is poured between well-cleaned, vertical glass plates mounted in vertical apparatuses for polymerization (approx. 10-20 min). The gel is run in 1 x TBE buffer at a constant voltage of 140 V.<br>
DNA sequencing<br>
1-2 jag of DNA are to be dissolved in 81 pi of distilled H2O and 9 pi of NaOH (2 N) is to be added for denaturation. After incubation at room temperature for 10 minutes, the mixture is precipitated, with thorough washing of the resulting DNA sediment with ice-cold 80% strength ethanol being important for the subsequent sequencing reactions. 2 pi of 5 x Sequenase buffer (200 mM Tris-CI pH 7.5/100 mM MgCI2/250 mM NaCI), 1 pi of oligonucleotide (1 pM/pl) and, finally, distilled H2O are to be added to the sediment to a total volume of 10 pi. During the subsequent incubation in a 37°C water bath for 30 minutes, the starter oligonucleotide hybridizes to the DNA.<br>
Reagents added to the hybridization mixture for the sequencing reaction: 1.0 pi of DTT (0.1 M), 2.0 pi of labeling mixture (diluted 1:5), 0.5 pi of [ct-35S]dATP, 2 pi of Sequenase™ (13U/pl, United States Biochemical), (diluted 1:8 with enzyme dilution buffer).<br>
During the subsequent incubation at room temperature for 5 minutes, the counter strand is synthesized, with the synthetic DNA being labled by incorporation of the radiolabeled dATP. This is followed by adding in each case 3.5 µl of the labeling mixture to 2.5 pi of the four different termination mixtures. Another incubation at 37°C for 5 minutes results in the randomly distributed termination reactions of counter stand synthesis. The reactions are stopped by adding 4µlof stop buffer, after which the mixtures are denatured at 80-90°C and then applied to a 6% strength denatured sequencing gel. After loading the samples, the main run is carried out at 30-50 W and, respectively, 1300-1600 V for 2-5 h. The gel is then fixed in a 10% strength acetic acid bath (15 min), freed of urea residues under running water and then dried (for 45 min, using a heat gun, or for 2 h, in a 70°C incubator). The subsequent autoradiography is carried out at 4°C for 16-24 h (Fuji Medical X-ray-Film RX, 30 x 40; Kodak Scientific Imaging Film X-omat AR).<br><br>
Labeling-mixture stock solution: in each case 7.5 µM dATP, dTTP, dGTP, dCTP<br>
Termination mixtures: in each case 80 µM dATP, dTTP, dGTP, dCTP and in each case 8 µ.M of the respective ddNTP<br>
Sequenase dilution buffer: 10 mM Tris/HCI; pH 7.5, 5 mM DTT, 0.5 mg/ml BSA<br>
Stop buffer: 95% formamide, 20 mM EDTA, 0.005% (w/v) xylene cyanol FF<br>
Automated DNA sequencing<br>
Mix: 1 ng of plasmid DNA (in the case of PCR fragments, for example, 100ng/500 nucleotides), 3-5 pmol of starter molecule (PCR primer, Tm of 55°C, if possible), 4 y\ of Dye Terminator ready-mix (FddNTPs-Ampli-TaqFS mixture), add distilled water to a volume of 20 \i\.<br>
The PCR reaction [25 x (15 sec at 94°C, 15 sec. at 50°C, 4 min at 60°C] is precipitated with alcohol and taken up in 4 \x\ of loading buffer. The samples are then denatured at 95°C for 3 min, removed by centrifugation and applied to a vertical polyacrylamide gel (34 cm in length, provided with 24 parallel lanes).<br>
After excitation by an argon laser beam at 488 nm, the dyes emit light of different wavelengths of between 525 nm and 605 nm which is separated into its spectral colors via a grating, a "spectrograph". The spectral colors are subsequently detected simultaneously with the aid of the high-resolution pixel field of a CCD camera. The data are recorded with the aid of a computer (Macintosh Quadra/650 Macllcx Apple Share) and the corresponding data analysis software (PE Biosystems, Weiterstadt, Germany).<br>
Sequencing gel: 30 g of urea (Sigma), 21.5 ml of distilled H2O, 6 ml of<br>
10xTBE<br>
The mixture is dissolved in a wide-necked flask on a heating block at 50°C,<br>
with the following being added: 9 ml of 40% bisacrylamide (filtered), 180 µl<br>
of 10% APS, 24 µlof Temed.<br><br>
Polymerase chain reaction (PCR reaction)<br>
The following DNA polymerases may be used:<br>
Taq (Thermus aquaticus) DNA polymerase (recombinant, Gibco/BRL) and<br>
10 x PCR buffer<br>
[200 mM Tris/HCI (pH 8.4), 500 mM KCI]<br>
Tfl (Thermus flavus) DNA polymerase (Master Amp™, Biozym, Oldendorf,<br>
Germany) and 20 x PCR<br>
Buffer [20 mM (NH2)S04) 1 M Tris/HCI (pH 9.0)<br><br>
The following applies here: 1 U catalyses the conversion of 10 nM deoxyribonucleoside triphosphates, at 74°C within 30 min, to an acid-insoluble DNA product The PCR reaction usually commences with the "hot start": the mixture is incubated first without the polymerase at 94°C in order to enable the DNA to be denatured for the first time. After the temperature has reached 80°C, the DNA polymerase is added to the mixture in order to avoid nonspecific amplification at a still low temperature. Thereafter, the actual PCR reaction is carried out over 25-35 cycles.<br>
For each cycle, the following reaction conditions apply:<br><br><br>
Finally and in addition, the chain extension is carried out at 72°C for 10 min, finally followed by cooling.<br>
Isolation of total RNA<br>
All centrifugation steps are carried out at 13 000 rpm and 16°C.<br>
Cells    are    lysed    with    600 µl    of   lysis    buffer    (100 RLT    buffer:<br>
1 mercaptoethanol). The cell lysate is applied to a QiaSchredder column<br>
and removed by centrifugation for 2 min.<br>
The eluate is admixed with 600 µl of 70% ethanol, mixed well, and the DNA<br>
is applied to an RNAeasy mini spin column and centrifuged for 15 s<br>
(binding of RNA to the silica matrix). The column is washed three times<br>
(once with 700 ml of RW1 buffer and twice with 500 \x\ of RPE buffer). The<br>
column is then transferred to an autoclaved 1.5 ml Eppendorf tube and the<br>
RNA is eluated with 15µl of distilled H20. The average concentration of<br>
total RNA obtained in this way is 1 µg/µl<br>
RNA fractionation via agarose gel electrophoresis<br>
Denaturing a rag rose gel:<br>
1 g of agraose, 37 ml of distilled water, 10 ml of 10 x MOPS (0.2 mM<br>
MOPS, 10 mM EDTA, 100 mM NaAc),<br>
the mixture is boiled and cooled to 60°C<br>
16 ml of 37% strength formaldehyde are added.<br>
After it has solidified, the gel is inserted with RNA gel running buffer into the electrophoresis apparatus. The RNA is applied together with a special sample buffer.<br>
RNA gel running buffer: 40 ml of 10 x MOPS, 65 ml of 37% strength formaldehyde, 295 ml of distilled water<br><br>
RNA sample buffer: 1-5 pg of RNA, 5 jd of RNA-NEW buffer (7.5 \i\ 37% strength formaldehyde, 4.5 µl of 10 x MOPS, 25.9 µl of formamide, 7.5 \i\ of distilled water), 2 µl of formamide dye marker [50% (v/v) glycerol, 1 mM EDTA (pH 8.0), 0.25% (v/v) bromophenol blue, 0.25% (v/v) xylene cyanol].<br>
The gel runs at 80 V for approx. 3 h. Since this work uses only eukaryotic RNA isolates, the dominant bands visible on the gel should be those of 28S and18SrRNA.<br>
Reverse Transcriptase with MMLV-RT<br>
(Moloney murine leukemia virus - Reverse Transcriptase)<br>
Reverse transcriptase mixture: 5 µg of RNA, 100 µM of starter molecule<br>
The RNA preparation and the starter molecule are incubated at 75°C for 10 min, in order to avoid possible formation of secondary structures in the RNA template as factors interfering with the transcriptase. However, even without this step, a transcription reaction usually takes place.<br>
Reverse transcriptase reaction mixture: 28 U of Rnasin® (Promega), 25 mM dNTPS, 5 µl of 10 x reverse transcriptase buffer [10 m Tris/HCI (pH 8.3), 75 mM KG, 3 mM MgCI2], 50 U of reverse transcriptase (StrataScript™1, Stratagene), to 50 µl with distilled water.<br>
The reverse transcription is carried out by incubating the mixture at 42°C for 15 min and at 37°C for 45 min. A longer incubation of 2 h at 42°C with a 30-sec interruption at 55°C is recommended for relatively long RNA templates. Subsequent incubation of the mixture at 95°C for 5 minutes results in inactivation of said reverse transcriptase. Subsequently, 5-20 ^il of the reverse transcription mixture are used for a PCR reaction.<br>
Preparation of genomic DNA from tissue<br>
100 mg of tissue are crushed in liquid nitrogen to give a powder. The tissue powder is introduced into a Falcon tube containing 6 ml of reaction buffer (30 µl of proteinase K [20 mg/ml] are added freshly to the buffer) and incubated with careful shaking at 56°C overnight (12-18 hours). After incubation, 100 µl of RNase A (10 µg/µl) are added and the mixture is incubated with further shaking at 37°C for one hour. This is followed by adding 4 ml of phenol and turning the tube manually<br><br>
upside down and up again for approximately 5 min. 4 ml of CI (chloroform/isoamyl alcohol) are added immediately and the tube is turned upside down and up again for another 5 minutes and then centrifuged for 15 min (3 000 rpm). The supernatant is carefully removed and transferred to 10 ml Falcon tubes. If the supernatant is still not clear, the phenol extraction must be repeated, otherwise another 4 ml of CI are added and the tube is manually turned upside down and up for 5 min and then centrifuged for 15 min (3 000 rpm). The supernatant is carefully removed and the CI extraction repeated. The final supernatant obtained is admixed with 1/10 volumes of sodium acetate solution (3 M, pH 6) and 2.5 volumes of ethanol (99.8%). The tube is carefully rotated, until the DNA precipitates as a tangle. This DNA tangle is transferred to approximately 25 ml of ethanol (70%) with the aid of a glass hook and left resting for 3 min. The washing was repeated twice. The DNA was then dried in air and dissolved in 0.5 ml of double-distilled water at room temperature.<br>
Southern Blot<br>
DNA fractionation via an agarose gel<br>
Leave the gel on short-wave UV for approx. 5 min for strand breaks to<br>
occur in the larger DNA molecules (&gt; 6kBp).<br>
Continuously tilt the gel in denaturating solution for 30 min for DNA<br>
denaturation.<br>
Continuously tilt the gel in neutralizing solution for 30 min for neutralization.<br>
Blot construction (from bottom to top): gel, nylon membrane, dry filter paper, blotting paper, plate, weight (approx. 1 kg). Blotting with 20 x SSC overnight.<br>
Wash membrane in 2 x SSC for 10 min<br>
Dry membrane on filter paper<br>
Fixing of nucleic acid by baking at 80°C for 1 h or UV crosslinking (e.g. in<br>
"Stratalinker", automatic position). The membrane may then be stored until<br>
hybridization.<br>
Prehybridization of membrane in hybridization solution for approx. 1 - 2 h<br>
Covering of nonspecific binding sites on the membrane.<br>
Hybridization solution: 5 x SSC,  5 x Denhardt's solution, 0.5% SDS,<br><br>
100 µg/ml herring sperm DNA<br>
Denaturing solution: 0.5 M NaOH (20 g), 1 M NaCI<br>
Neutralizing solution: 1.5 M NaCI/0.5 M Tris pH 7.4<br>
20 x SSC is 3 M NaCI, 0.3 M Na-citrate: 175.3 g of NaCI, 88.2 g of sodium citrate X 2 H20, to 1 I with double-distilled water, adjust pH to 7.0 with HCI.<br>
50  x  Denhard's  solution:  5 g  of  Ficoll  400,  5 g  of  PVP  (polyvinyl pyrrolidone), 5 g of BSA, to 500 ml with double-distilled water<br>
Northern Blot<br>
RNA fractionation using a formaldehyde agarose gel<br>
Blot construction (from bottom to top): gel, nylon membrane, dry filter<br>
paper, blotting paper, plate, weight (appox. 1 kg).<br>
Blotting with 20 x SSC overnight.<br>
Fix RNA on filter by baking at 80°C (1 h)<br>
Introduce filter into boiling 20 mM Tris pH 8 for RNA deglyoxylation and let<br>
cool to RT.<br>
Description of the Figures<br>
Fig. 1<br>
Characteristics of study group<br>
Fig. 2<br>
Distribution of PARI variants T3090C and A3329C in 1362 individuals analyzed<br>
Fig. 3<br>
Association   of   PARI   variants   C3090C   with   atrial   fibrillation   and cardiomyopathy<br><br>
Fig. 4<br>
Association of PARI variants C3329C with atrial fibrillation, acute coronary syndrome and unstable angina.<br>
Fig. 5<br>
Polynucleotide sequence of the cDNA of the human PARI gene in 573' orientation. The sequence corresponds to the sequence made publicly available by the NCBI Nucleotide Database under number NM-001992. The prepared sequence is identical to SEQ ID NO: 1.<br>
Fig. 6<br>
Polynucleotide sequence of the cDNA of the human PARI gene in 573' orientation with a polymorphism at position 3090 of the sequence according to NM-001992, which polymorphism comprises a T to C substitution. The depicted sequence is identical to SEQ ID NO: 2.<br>
Fig. 7<br>
Polynucleotide sequence of the cDNA of the human PARI gene in 573' orientation with a polymorphism at position 3329 of the sequence according to NM-001992, which polymorphism comprises an A to C substitution. The depicted sequence is identical to SEQ ID NO: 3.<br>
Fig. 8<br>
Polynucleotide sequence of the cDNA of the human PARI gene in 573' orientation with a polymorphism at position 3090 of the sequence according to NM-001992, which polymorphism comprises a T to C substitution, and with a simultaneous second polymorphism at position 3329 of the sequence according to NM-001992, which polymorphism comprises an A to C substitution. The depicted sequence is identical to SEQ ID NO: 4.<br><br>
Fig. 9<br>
Polynucleotide sequence of a fragment of the human PARI gene in 573' orientation. The depicted sequence is identical to SEQ ID NO: 5.<br>
Fig. 10<br>
Polynucleotide sequence of a fragment of the human PARI gene in 573' orientation with a polymorphism at position 3090 of the sequence according to NM-01992, which polymorphism comprises a T to C substitution. The depicted sequence is identical to SEQ ID NO: 6.<br>
Fig. 11<br>
Polynucleotide sequence of a fragment of human PARI gene in 573' orientation with a polymorphism at position 3329 of the sequence according to NM-001992, which polymorphism comprises an A to C substitution. The depicted sequence is identical to SEQ ID NO: 7.<br>
Fig. 12<br>
Polynucleotide sequence of a fragment of the human PARI gene in 573' orientation with a polymorphism at position 3090 of the sequence according to NM-001992, which polymorphism comprises a T to C substitution, and with a simultaneous second polymorphism at position 3329 of the sequence according to NM-001992, which polymorphism comprises an A to C substitution. The depicted sequence is identical to SEQ ID NO: 8.<br>
Fig. 13<br>
Polynucleotide sequence in 573' orientation of the 5' end of the cDNA of the human PARI gene. The depicted sequence is identical to SEQ ID NO; 9.<br>
Fig. 14<br>
Polynucleotide sequence in 573' orientation of the 3' end of the cDNA of the human PARI gene. The depicted sequence is identical to SEQ ID NO; 10.<br><br>
Fig. 15<br>
Polynucleotide sequence in 573' orientation of the cDNA of the human PARI gene, relating to positions 2767 to 2789 according to NM-001992. The depicted sequence is identical to SEQ ID NO; 11.<br>
Fig. 16<br>
Polynucleotide sequence in 573' orientation of the Exon No. 1 of the human PARI gene. The depicted sequence is identical to SEQ ID NO; 12.<br>
Fig. 17<br>
Protein sequence of the human PARI receptor. The sequence corresponds to the sequence made publicly available by the NCBI Protein Database under number NP-001983. The depicted sequence is identical to SEQ ID NO: 13.<br><br><br><br><br>
What is claimed is:<br>
1.	An isolated polynucleotide sequence of the PARI gene, which comprises a C for T substitution at position 3090 of the sequence according to NM-001992.<br>
2.	The isolated polynucleotide sequence as claimed in claim 1, wherein the polynucleotide sequence of the PARI gene encompasses a sequence according to SEQ ID NO: 2.<br>
3.	The isolated polynucleotide sequence as claimed in claim 1, wherein the polynucleotide sequence of the PARI gene comprises a sequence according to SEQ ID NO: 2.<br>
4.	An isolated polynucleotide sequence of the PARI gene, which comprises a C for A substitution at position 3329 of the sequence according to NM-001992.<br>
5.	The isolated polynucleotide sequence as claimed in claim 4, wherein the polynucleotide sequence of the PARI gene encompasses a sequence according to SEQ ID NO: 3.<br>
6.	The isolated polynucleotide sequence as claimed in claim 4, wherein the polynucleotide sequence of the PARI gene comprises a sequence according to SEQ ID NO: 3.<br>
7.	An isolated polynucleotide sequence of the PARI gene, which comprises a C for T substitution at position 3090 and a C for A substitution at position 3329, in each case based on NM-001992.<br>
8.	The isolated polynucleotide sequence as claimed in claim 7, wherein the polynucleotide sequence of the PARI gene encompasses a sequence according to SEQ ID NO: 4.<br>
9.	The isolated polynucleotide sequence as claimed in claim 7, wherein the polynucleotide sequence of the PARI gene comprises a sequence according to SEQ ID NO: 4.<br><br>
10.	An isolated part of the polynucleotide sequence of the PARI gene, comprising a sequence according to SEQ ID NO: 5.<br>
11.	An isolated part of the polynucleotide sequence of the PARI gene, comprising a sequence according to SEQ ID NO: 6.<br>
12.	An isolated part of the polynucleotide sequence of the PARI gene, comprising a sequence according to SEQ ID NO: 7.<br>
13.	An isolated part of the polynucleotide sequence of the PARI gene, comprising a sequence according to SEQ ID NO: 8.<br>
14.	A method for preparing a polynucleotide sequence as claimed in any of claims 1 to 9 by means of the following method steps:<br><br>
a]	Providing human cDNA comprising a PARI sequence according to SEQ ID NO: 2 and/or a PARI sequence according to SEQ ID NO: 3 and/or a PARI sequence according to SEQ ID NO: 4,<br>
b]	Providing a primer pair according to SEQ ID NO: 9 and SEQ ID NO: 10.<br>
c]	Amplifying the PARI polynucleotide sequence by the polymerase chain extension reaction (PCR),<br>
d]	Isolating and/or purifying the 3.56 kb fragment obtained from<br>
c],<br>
e]	Sequencing the fragment from d].<br>
15.	A method for preparing a polynucleotide sequence as claimed in any<br>
of claims 10 to 13 by means of the following method steps:<br>
a]	Providing human genomic DNA comprising a PARI sequence according to SEQ ID NO: 1 and/or a PARI sequence according to SEQ ID NO: 2 and/or a PARI sequence according to SEQ ID NO: 3 and/or a PARI sequence according to SEQ ID NO: 4<br>
b]	Providing a primer pair according to SEQ ID NO: 11 and SEQ ID NO: 12<br>
c]	Amplifying the fragment of the PARI polynucleotide sequence by the polymerase chain extension reaction (PCR),<br>
d]	Isolating and/or purifying the fragment obtained from c],<br>
e]	Sequencing the fragment from d].<br><br>
16.	A method for detecting whether or not there is in a PARI gene a T to<br>
C substitution at position 3090 of the sequence according to NM-<br>
001992 and/or an A to C substitution at position 3329 of the<br>
sequence according to NM-001992, which method comprises the<br>
following method steps:<br>
a]	Providing biological material comprising human cells,<br>
b]	Obtaining chromosomal DNA from the material of a],<br>
c]	Amplifying a polynucleotide fragment by means of the primers according to SEQ ID NO: 11 and SEQ ID NO: 12, using a PCR reaction,<br>
d]	Sequencing the polynucleotide fragment from c].<br>
17.	A method for detecting whether or not there is in a PARI gene a T to<br>
C substitution at position 3090 of the sequence according to NM-<br>
001992 and/or an A to C substitution at position 3329 of the<br>
sequence according to NM-001992, which method comprises the<br>
following method steps:<br>
a]	Providing biological material comprising human cells,<br>
b]	Obtaining RNA from the material of a],<br>
c]	Transcribing said RNA into cDNA by means of reverse transcriptase,<br>
d]	Possibly amplifying a polynucleotide fragment by means of the primers according to SEQ ID NO: 10 and SEQ ID NO: 11, using the PCR reaction,<br>
e]	Sequencing the cDNA from c] and/or the polynucleotide fragment from d].<br>
18.	A method for detecting whether or not there is in a PARI gene a T to<br>
C substitution at position 3090 of the sequence according to NM-<br>
001992 and/or an A to C substitution at position 3329 of the<br>
sequence according to NM-001992, which method comprises the<br>
following method steps:<br>
a]	Providing biological material comprising human cells,<br>
b]	Obtaining chromosomal DNA from the material of a],<br>
c]	Southern blotting the chromosomal DNA from b],<br>
d]	Providing a probe according to SEQ ID NO: 5 and/or SEQ ID NO: 6 and/or SEQ ID NO: 7 and/or SEQ ID NO: 8,<br><br>
e]	Hybridizing the Southern blot from c] with the probe from d] under stringent hybridization conditions,<br>
f]	Determining the presence or absence of a genetic variation in the PARI gene at position 3090 and/or 3329 according to NM-001992 by comparing the results of the hybridization from<br>
e].<br>
19.	A method for detecting, whether or not there is in a PARI gene a T<br>
to C substitution at position 3090 of the sequence according to NM-<br>
001992 and/or an A to C substitution at position 3329 of the<br>
sequence according to NM-001992, which method comprises the<br>
following method steps:<br>
a]	Providing biological material comprising human cells,<br>
b]	Obtaining RNA from the material of a],<br>
c]	Providing a probe according to SEQ ID NO: 5 and/or SEQ ID NO: 6 and/or SEQ ID NO: 7 and/or SEQ ID NO: 8,<br>
e] Hybridizing the Northern blot form c] with the probe from d] under stringent hybridization conditions,<br>
f\ Determining the presence or absence of a genetic variation in the PARI gene at position 3090 and/or 3329 according to NM-001992 by comparing the results of the hybridization.<br>
20.	An isolated polynucleotide sequence having from 21 to 50 nucleotides, which comprises a sequence according to SEQ ID NO: 11.<br>
21.	An isolated polynucleotide sequence comprising SEQ ID NO: 11.<br>
22.	An isolated polynucleotide sequence having from 21 to 50 nucleotides, which comprises a sequence according to SEQ ID NO: 12.<br>
23.	An isolated polynucleotide sequence comprising SEQ ID NO: 12.<br>
24.	The use of an isolated polynucleotide sequence as claimed in claim 20 or 21 in combination with an isolated polynucleotide sequence as claimed in claim 22 or 23 for amplifying a fragment of the PARI gene by means of the PCR reaction.<br><br>
25.	The use as claimed in 24, wherein the PARI gene has a T to C substitution at position 3090 of the sequence according to NM-001992 and/or an A to C substitution at position 3329 of the sequence according to NM-001992.<br>
26.	A kit of parts, comprising<br><br>
a]	a polynucleotide sequence as claimed in claim 20 or 21,<br>
b]	a polynucleotide sequence as claimed in claim 22 or 23,<br>
c]	at least one enzyme for carrying out the PCR reaction<br><br>
d]	and possibly substances and/or solutions for carrying out the polymerase chain extension reaction (PCR),<br>
e]	and   possibly   furthermore   polynucleotide   sequences   as claimed in one or more of claims 1 to 17<br>
f]	and possibly reagents for carrying out a sequencing.<br>
27.	A method for preparing the kit of parts as claimed in claim 26, which<br>
method comprises<br>
a]	preparing a polynucleotide sequence as claimed in claim 20<br>
or 21,<br>
b]	providing a polynucleotide sequence as claimed in claim 22<br>
or 23,<br>
c]	providing an enzyme for carrying out the PCR reaction,<br>
d]	providing, where appropriate, reagents for carrying out a sequencing<br>
e]	possibly providing substances and/or solutions for carrying out the polymerase chain extension reaction (PCR),<br>
f]	possibly providing polynucleotide sequences as claimed in<br>
one or more of claims 1 to 13,<br>
g]	introducing the components from a] to e] in each  case<br>
separately into suitable containers,<br>
h]       combining, where appropriate, the containers from g] in one or more pack-units.<br>
28.	The use of the kit of parts as claimed in claim 26 or 27 for amplifying<br>
a fragment of the PARI gene and possibly for the further analysis,<br>
as to whether genetic variations are present in the PARI gene at<br><br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjczMS1DSEVOUC0yMDA1ICBBQlNUUkFDVC5wZGY=" target="_blank" style="word-wrap:break-word;">2731-CHENP-2005  ABSTRACT.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjczMS1DSEVOUC0yMDA1ICBDTEFJTVMucGRm" target="_blank" style="word-wrap:break-word;">2731-CHENP-2005  CLAIMS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjczMS1jaGVucC0yMDA1LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2731-chenp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjczMS1jaGVucC0yMDA1LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2731-chenp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjczMS1jaGVucC0yMDA1LWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2731-chenp-2005-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjczMS1jaGVucC0yMDA1LWNvcnJlc3BvbmRuZWNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">2731-chenp-2005-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjczMS1jaGVucC0yMDA1LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">2731-chenp-2005-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjczMS1jaGVucC0yMDA1LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">2731-chenp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjczMS1jaGVucC0yMDA1LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">2731-chenp-2005-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjczMS1jaGVucC0yMDA1LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">2731-chenp-2005-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjczMS1jaGVucC0yMDA1LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">2731-chenp-2005-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjczMS1jaGVucC0yMDA1LW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2731-chenp-2005-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjczMS1jaGVucC0yMDA1LXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">2731-chenp-2005-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="228870-phthalimidyl-azo-dyes-processes-for-the-preparation-thereof-and-process-of-dying-or-printing-with-the-dye.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="228872-phospholipase-and-method-of-producing-it.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>228871</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2731/CHENP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>12/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>20-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>11-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>21-Oct-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>SANOFI-AVENTIS DEUTSCHLAND GmbH</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>BRUNNINGSTRASSE 50, D-65929 FRANKFURT AM MAIN,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>CZECH, JOERG</td>
											<td>KREUTZACKER 2A, 35041 MARBURG,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>KOZIAN, DETLEF</td>
											<td>SCHULSTRASSE 23, 67595 HATTERSHEIM,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>SIEGLER, KARL-ERNST</td>
											<td>HORST-SCHORK-STRASSE 19, 67069 LUDWIGSHAFEN,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>DELEUZE, JEAN-FRANCOIS</td>
											<td>4, RUE DE I&#x27;YERRES, F-77380 COMBS LA VILLE,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>RICARD, SYLVAIN</td>
											<td>120 BOULEVARD VINCENT AURIOL, F-75013 PARIS,</td>
										</tr>
										<tr>
											<td>6</td>
											<td>MACE, SANDRINE</td>
											<td>11 RUE LOUIS CHAGNOT, F-78350 JOUY-EN-JOSAS,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 14/72</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP04/04035</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-04-16</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>103 18 496.1</td>
									<td>2003-04-24</td>
								    <td>Germany</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/228871-an-isolated-polynucleotide-sequence-of-the-par1-gene by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:30:19 GMT -->
</html>
