<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/228882-corynebacterium-glutamicum-genes-encoding-stress-resistance-and-tolerance-proteins by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:31:02 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 228882:CORYNEBACTERIUM GLUTAMICUM GENES ENCODING STRESS, RESISTANCE AND TOLERANCE PROTEINS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">CORYNEBACTERIUM GLUTAMICUM GENES ENCODING STRESS, RESISTANCE AND TOLERANCE PROTEINS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>This invention relates to an isolated nucleic acid molecules, designated SRT nucleic acid molecules, which encode novel SRT proteins from Corynebacterium glutamicum are described. The invention also provides antisense nucleic acid molecules, recombinant expression vectors containing SRT nucleic acid molecules, and host cells into which the expression vectors have been introduced. The invention still further provides isolated SRT proteins, mutated SRT proteins, fusion proteins, antigenic peptides and methods for the improvement of production of a desired compound from C. glutamicum based on gene!ic engineering of SRT genes in this organIsm.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
CORYNEBACTERIUM GLUTAMICUMGENES ENCODING STRESS, RESISTANCE AND TOLERANCE PROTEINS<br>
Related Applications<br>
This application claims priority to prior filed U.S. Provisional Patent Application Serial No. 60/141031, filed June 25, 1999, U.S. Provisional Patent Application Serial No. 60/142692, filed July 1, 1999. and also to U.S. Provisional Patent Application Serial No. 60/151214. filed August 27, 1999. This application also claims priority to German Patent Application No. 19930429.7, filed July 1, 1999, German Patent Application No. 19931413.6, filed July 8, 1999, German Patent Application No. 19931457.8, filed July 8, 1999, German Patent Application No. 19931541.8, filed July 8, 1999, German Patent Application No. 19932209.0, filed July 9, 1999, German Patent Application No. 19932230.9, filed July 9, 1999, German Patent Application No. 19932914.1, filed July 14. 1999. German Patent Application No. 19940764.9, filed August 27. 1999. and German Patent Application No. 19941382.7, filed August 31, 1999. The entire contents of all of the aforementioned applications are hereby expressly incorporated herein in their entirety by this reference.<br>
Background of the Invention<br>
Certain products and by-products of naturally-occurring metabolic processes in cells have utility in a wide array of industries, including the food, feed, cosmetics, and pharmaceutical industries. These molecules, collectively termed Tine chemicals', include organic acids, both proteinogenic and non-proteinogenic amino acids, nucleotides and nucleosides, lipids and fatty acids, diols, carbohydrates, aromatic compounds, vitamins and cofactors, and enzymes. Their production is most conveniently performed through large-scale culture of bacteria developed to produce and secrete large quantities of a particular desired molecule. One particularly useful organism for this purpose is Corynebacterium glutamicum, a gram positive, nonpathogenic bacterium. Through strain selection, a number of mutant strains have been developed which produce an array of desirable compounds. However, selection of strains improved for the production of a particular molecule is a time-consuming and difficult process.<br><br>
Summary of the Invention<br>
The invention provides novel bacterial nucleic acid molecules which have a variety of uses. These uses include the identification of microorganisms which can be used to produce fine chemicals, the modulation of fine chemical production in C. glutamicum or related bacteria, the typing or identification of C glutamicum or related bacteria, as reference points for mapping the C. glutamicum genome, and as markers for transformation. These novel nucleic acid molecules encode proteins, referred to herein as stress, resistance and tolerance (SRT) proteins.<br>
C. glutamicum is a gram positive, aerobic bacterium which is commonly used in industry for the large-scale production of a variety of fine chemicals, and also for the degradation of hydrocarbons (such as in petroleum spills) and for the oxidation of terpenoids. The SRT nucleic acid molecules of the invention, therefore, can be used to identify microorganisms which can be used to produce fine chemicals, e.g.. by fermentation processes. Modulation of the expression of the SRT nucleic acids of the invention, or modification of the sequence of the SRT nucleic acid molecules of the invention, can be used to modulate the production of one or more fine chemicals from a microorganism (e.g., to improve the yield or production of one or more fine chemicals from a Corynebacterium or Brevibacterium species).<br>
The SRT nucleic acids of the invention may also be used to identify an organism as being Corynebacterium glutamicum or a close relative thereof, or to identify the presence of C. glutamicum or a relative thereof in a mixed population of microorganisms. The invention provides the nucleic acid sequences of a number of C. glutamicum genes; by probing the extracted genomic DNA of a culture of a unique or mixed population of microorganisms under stringent conditions with a probe spanning a region of a C. glutamicum gene which is unique to this organism, one can ascertain whether this organism is present. Although Corynebacterium glutamicum itself is nonpathogenic, it is related to species pathogenic in humans, such as Corynebacterium diphtheriae (the causative agent of diphtheria); the detection of such organisms is of significant clinical relevance.<br>
The SRT nucleic acid molecules of the invention may also serve as reference points for mapping of the C glutamicum genome, or of genomes of related organisms.<br><br>
Similarly, these molecules, or variants or portions thereof, may serve as markers for genetically engineered Corynebacterium or Brevibacterium species.<br>
The SRT proteins encoded by the novel nucleic acid molecules of the invention are capable of, for example, permitting C glutamicum to survive in a setting which is either chemically or environmentally hazardous to this microorganism. Given the availability of cloning vectors for use in Corynebacterium glutamicum^ such as those disclosed in Sinskey et a/., U.S. Patent No. 4,649,119, and techniques for genetic manipulation of C glutamicum and the related Brevibacterium species (e.g.. lactofermenium) (Yoshihama et al, J. Bacterial. 162: 591*597 (1985); Katsumata et a!.. J. BacterioL 159:306-311 (1984); and Santamaria et al, J. Gen. Microbiol 130:2237-2246 (1984)), the nucleic acid molecules of the invention may be utilized in the genetic engineering of this organism to make it a better or more efficient producer of one or more fine chemicals, through the ability of these proteins to permit growth and multiplication of C glutamicum (and also continuous production of one or more fine chemicals) under circumstances which would normally impede growth of the organism. such as those conditions frequently encountered during large-scale fermentative growth. For example, by overexpressing or engineering a heat-shock induced protease molecule such that it is optimized in activity, one may increase the ability of the bacterium to degrade incorrectly folded proteins when the bacterium is challenged with high temperatures. By having fewer misfolded (and possibly misregulated or nonfunctional) proteins to interfere with normal reaction mechanisms in the cell, the cell is increased in its ability to function normally in such a culture, which should in turn provide increased viability. This overall increase in number of cells having greater viability and activity in the culture should also result in an increase in yield, production, and/or efficiency of production of one or more desired fine chemicals, due at least to the relatively greater number of cells producing these chemicals in the culture.<br>
This invention provides novel SRT nucleic acid molecules which encode SRT proteins which are capable of, for example, permitting C glutamicum to survive in a setting which is either chemically or environmentally hazardous to this microorganism. Nucleic acid molecules encoding an SRT protein are referred to herein as SRT nucleic acid molecules. In a preferred embodiment, the SRT protein participates in metabolic pathways permitting C. glutamicum to survive in a setting which is either chemically or<br><br>
environmentally hazardous to this microorganism. Examples of such proteins include those encoded by the genes set forth in Table 1.<br>
Accordingly, one aspect of the invention pertains to isolated nucleic acid molecules (e.g., cDNAs, DNAs, or RNAs) comprising a nucleotide sequence encoding an SRT protein or biologically active portions thereof, as well as nucleic acid fragments suitable as primers or hybridization probes for the detection or amplification of SRT-encoding nucleic acid (e.g., DNA or mRNA). In particularly preferred embodiments. the isolated nucleic acid molecule comprises one of the nucleotide sequences set forth as the odd-numbered SEQ ID NOs in the Sequence Listing (e.g., SEQ ID NO:l, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7....), or the coding region or a complement thereof of one of these nucleotide sequences. In other particularly preferred embodiments, the isolated nucleic acid molecule of the invention comprises a nucleotide sequence which hybridizes to or is at least about 50%, preferably at least about 60%. more preferably at least about 70%. 80% or 90%, and even more preferably at least about 95%. 96%. 97%. 98%, 99% or more homologous to a nucleotide sequence set forth as an odd-numbered SEQ ID NO in the Sequence Listing (e.g., SEQ ID NO: 1. SEQ ID NO:3, SEQ ID NO:5. SEQ ID NO:7....). or a portion thereof. In other preferred embodiments, the isolated nucleic acid molecule encodes one of the amino acid sequences set forth as an even-numbered SEQ ID NO in the Sequence Listing (e.g., SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8....).. The preferred SRT proteins of the present invention also preferably possess at least one of the SRT activities described herein.<br>
In another embodiment, the isolated nucleic acid molecule encodes a protein or portion thereof wherein the protein or portion thereof includes an amino acid sequence which is sufficiently homologous to an amino acid sequence of the invention (e.g., a sequence having an even-numbered SEQ ID NO: in the Sequence Listing), e.g., sufficiently homologous to an amino acid sequence of the invention such that the protein or portion thereof maintains an SRT activity. Preferably, the protein or portion thereof encoded by the nucleic acid molecule maintains the ability to increase the survival of C. glutamicum in a setting which is either chemically or environmentally hazardous to this microorganism.   In one embodiment, the protein encoded by the nucleic acid molecule is at least about 50%, preferably at least about 60%, and more preferably at least about 70%, 80%, or 90% and most preferably at least about 95%, 96%, 97%, 98%, or 99% or<br><br>
more homologous to an amino acid sequence of the invention (e.g., an entire amino acid sequence selected from those having an even-numbered SEQ ID NO in the Sequence Listing). In another preferred embodiment, the protein is a full length C. glutamicum protein which is substantially homologous to an entire amino acid sequence of the invention (encoded by an open reading frame shown the corresponding odd-numbered SEQ ID NOs in the Sequence Listing {e.g., SEQ ID NO: 1, SEQ ID NO:3, SEQ ID NO:5,SEQIDNO:7....).<br>
In another preferred embodiment, the isolated nucleic acid molecule is derived from C. glutamicum and encodes a protein {e.g., an SRT fusion protein) which includes a biologically active domain which is at least about 50% or more homologous to one of the amino acid sequences of the invention (e.g., a sequence of one of the even-numbered SEQ ID NOs in the Sequence Listing) and has the ability to increase the survival of C glutamicum in a setting which is either chemically or environmentally hazardous to this microorganism, or possesses one or more of the activities set forth in Table 1. and which also includes heterologous nucleic acid sequences encoding a heterologous polypeptide or regulatory regions.<br>
In another embodiment, the isolated nucleic acid molecule is at least 15 nucleotides in length and hybridizes under stringent conditions to a nucleic acid molecule comprising a nucleotide sequence of the invention (e.g., a sequence of an odd-numbered SEQ ID NO in the Sequence Listing). Preferably, the isolated nucleic acid molecule corresponds to a naturally-occurring nucleic acid molecule. More preferably, the isolated nucleic acid encodes a naturally-occurring C. glutamicum SRT protein, or a biologically active portion thereof.<br>
Another aspect of the invention pertains to vectors, e.g., recombinant expression vectors, containing the nucleic acid molecules of the invention, and host cells into which such vectors have been introduced. In one embodiment, such a host cell is used to produce an SRT protein by culturing the host cell in a suitable medium. The SRT protein can be then isolated from the medium or the host cell.<br>
Yet another aspect of the invention pertains to a genetically altered microorganism in which an SRT gene has been introduced or altered. In one embodiment, the genome of the microorganism has been altered by the introduction of a nucleic acid molecule of the invention encoding wild-type or mutated SRT sequence as<br><br>
a transgene. In another embodiment, an endogenous SRT gene within the genome of the microorganism has been altered, e.g., functionally disrupted, by homologous recombination with an altered SRT gene. In another embodiment, an endogenous or introduced SRT gene in a microorganism has been altered by one or more point mutations, deletions, or inversions, but still encodes a functional SRT protein. In still another embodiment, one or more of the regulatory regions {e.g., a promoter, repressor, or inducer) of a SRT gene in a microorganism has been altered {e.g., by deletion, truncation, inversion, or point mutation) such that the expression of the SRT gene is modulated. In a preferred embodiment, the microorganism belongs to the genus Corynebacterium or Brevibacterium, with Corynebacterium glutamicum being particularly preferred. In a preferred embodiment, the microorganism is also utilized for the production of a desired compound, such as an amino acid, with lysine being particularly preferred.<br>
In another aspect, the invention provides a method of identifying the presence or activity of Cornyebacterium diphtheriae in a subject. This method includes detection of one or more of the nucleic acid or amino acid sequences of the invention (e.g.. the sequences set forth in the Sequence Listing as SEQ ID NOs 1 through 304)) in a subject, thereby detecting the presence or activity of Corynebacterium diphtheriae in the subject.<br>
Still another aspect of the invention pertains to an isolated SRT protein or a portion, e.g., a biologically active portion, thereof. In a preferred embodiment, the isolated SRT protein or portion thereof possesses the ability to increase the survival of C. glutamicum in a setting which is either chemically or environmentally hazardous to this microorganism. In another preferred embodiment, the isolated SRT protein or portion thereof is sufficiently homologous to an amino acid sequence of the invention {e.g., a sequence of an even-numbered SEQ ID NO: in the Sequence Listing) such that the protein or portion thereof maintains the ability to increase the survival of C. glutamicum in a setting which is either chemically or environmentally hazardous to this microorganism.<br>
The invention also provides an isolated preparation of an SRT protein. In preferred embodiments, the SRT protein comprises an amino acid sequence of the invention {e.g., a sequence of an even-numbered SEQ ID NO: of the Sequence Listing). In another preferred embodiment, the invention pertains to an isolated full length protein<br><br>
which is substantially homologous to an entire amino acid sequence of the invention (e.g., a sequence of an even-numbered SEQ ID NO: of the Sequence Listing) (encoded by an open reading frame set forth in a corresponding odd-numbered SEQ ID NO: of the Sequence Listing).). In yet another embodiment, the protein is at least about 50%, preferably at least about 60%, and more preferably at least about 70%, 80%, or 90%. and most preferably at least about 95%, 96%, 97%, 98%, or 99% or more homologous to an entire amino acid sequence of the invention (e.g., a sequence of an even-numbered SEQ ID NO: of the Sequence Listing). In other embodiments, the isolated SRT protein comprises an amino acid sequence which is at least about 50% or more homologous to one of the amino acid sequences of the invention (e.g., a sequence of an even-numbered SEQ ID NO: of the Sequence Listing) and is able to improve the survival rate of C glutamicum in a setting which is either chemically or environmentally hazardous to this microorganism, or has one or more of the activities set forth in Table 1.<br>
Alternatively, the isolated SRT protein can comprise an amino acid sequence which is encoded by a nucleotide sequence which hybridizes, e.g., hybridizes under stringent conditions, or is at least about 50%, preferably at least about 60%. more preferably at least about 70%, 80%, or 90%, and even more preferably at least about 95%, 96%, 97%, 98,%, or 99% or more homologousto a nucleotide sequence of one of the even-numbered SEQ ID NOs set forth in the Sequence Listing. It is also preferred that the preferred forms of SRT proteins also have one or more of the SRT bioactivities described herein.<br>
The SRT polypeptide, or a biologically active portion thereof, can be operatively linked to a non-SRT polypeptide to form a fusion protein. In preferred embodiments, this fusion protein has an activity which differs from that of the SRT protein alone. In other preferred embodiments, this fusion protein results in increased yields, production, and/or efficiency of production of a desired fine chemical from C. glutamicum. In particularly preferred embodiments, integration of this fusion protein into a host cell modulates the production of a desired compound from the cell.<br>
In another aspect, the invention provides methods for screening molecules which modulate the activity of an SRT protein, either by interacting with the protein itself or a substrate or binding partner of the SRT protein, or by modulating the transcription or translation of an SRT nucleic acid molecule of the invention.<br><br>
Another aspect of the invention pertains to a method for producing a fine chemical. This method involves the culturing of a cell containing a vector directing the expression of an SRT nucleic acid molecule of the invention, such that a fine chemical is produced. In a preferred embodiment, this method further includes the step of obtaining a cell containing such a vector, in which a cell is transfected with a vector directing the expression of an SRT nucleic acid. In another preferred embodiment, this method further includes the step of recovering the fine chemical from the culture. In a particularly preferred embodiment, the cell is from the genus Corynehacterium or Brevibacterium. or is selected from those strains set forth in Table 3.<br>
Another aspect of the invention pertains to methods for modulating production of a molecule from a microorganism. Such methods include contacting the cell with an agent which modulates SRT protein activity or SRT nucleic acid expression such that a cell associated activity is altered relative to this same activity in the absence of the agent. In a preferred embodiment, the cell is modulated in resistance to one or more toxic chemicals or in resistance to one or more environmental stresses, such that the yields or rate of production of a desired fine chemical by this microorganism is improved. The agent which modulates SRT protein activity can be an agent which stimulates SRT protein activity or SRT nucleic acid expression. Examples of agents which stimulate SRT protein activity or SRT nucleic acid expression include small molecules, active SRT proteins, and nucleic acids encoding SRT proteins that have been introduced into the cell. Examples of agents which inhibit SRT activity or expression include small molecules, and antisense SRT nucleic acid molecules.<br>
Another aspect of the invention pertains to methods for modulating yields of a desired compound from a cell, involving the introduction of a wild-type or mutant SRT gene into a cell, either maintained on a separate plasmid or integrated into the genome of the host cell. If integrated into the genome, such integration can random, or it can take place by homologous recombination such that the native gene is replaced by the introduced copy, causing the production of the desired compound from the cell to be modulated. In a preferred embodiment, said yields are increased. In another preferred embodiment, said chemical is a fine chemical. In a particularly preferred embodiment, said fine chemical is an amino acid. In especially preferred embodiments, said amino acid is L-lysine.<br><br>
Detailed Description of the Invention<br>
The present invention provides SRT nucleic acid and protein molecules which are involved in the survival of C glutamicum upon exposure of this microorganism to chemical or environmental hazards. The molecules of the invention may be utilized in the modulation of production of fine chemicals from microorganisms, since these SRT proteins provide a means for continued growth and multiplication of C glutamicum in the presence of toxic chemicals or hazardous environmental conditions, such as may be encountered during large-scale fermentative growth. By increasing the growth rate or at least maintaining normal growth in the face of poor, if not toxic, conditions, one may increase the yield, production, and/or efficiency of production of one or more fine chemicals from such a culture, at least due to the relatively greater number of cells producing the fine chemical in the culture. Aspects of the invention are further explicated below.<br>
1.	Fine Chemicals<br>
The term 'fine chemical" is art-recognized and includes molecules produced by an organism which have applications in various industries, such as. but not limited to. the pharmaceutical, agriculture, and cosmetics industries. Such compounds include organic acids, such as tartaric acid, itaconic acid, and diaminopimelic acid, both proteinogenic and non-proteinogenic amino acids, purine and pyrimidine bases, nucleosides, and nucleotides (as described e.g. in Kuninaka, A. (1996) Nucleotides and related compounds, p. 561-612, in Biotechnology vol. 6, Rehm et ah, eds. VCH: Weinheim, and references contained therein), lipids, both saturated and unsaturated fatty acids (e.g., arachidonic acid), diols (e.g., propane diol, and butane diol), carbohydrates (*•£•» hyaluronic acid and trehalose), aromatic compounds (e.g., aromatic amines, vanillin, and indigo), vitamins and cofactors (as described in Ullmann's Encyclopedia of Industrial Chemistry, vol. A27, "Vitamins", p. 443-613 (1996) VCH: Weinheim and references therein; and Ong, A.S., Niki, E. &amp; Packer, L. (1995) "Nutrition, Lipids, Health, and Disease" Proceedings of the UNESCO/Confederation of Scientific and Technological Associations in Malaysia, and the Society for Free Radical Research -Asia, held Sept. 1-3, 1994 at Penang, Malaysia, AOCS Press, (1995)), enzymes, polyketides (Cane et al (1998) Science 282: 63-68), and all other chemicals described in<br><br>
Gutcho (1983) Chemicals by Fermentation, Noyes Data Corporation, ISBN: 0818805086 and references therein. The metabolism and uses of certain of these fine chemicals are further explicated below.<br>
A. Amino Acid Metabolism and Uses<br>
Amino acids comprise the basic structural units of all proteins, and as such are essential for normal cellular functioning in all organisms. The term "amino acid" is art-recognized. The proteinogenic amino acids, of which there are 20 species, serve as structural units for proteins, in which they are linked by peptide bonds, while the nonproteinogenic amino acids (hundreds of which are known) are not normally found in proteins (see Ulmann's Encyclopedia of Industrial Chemistry, vol. A2, p. 57-97 VCH: Weinheim (1985)). Amino acids may be in the D- or L- optical configuration, though L-amino acids are generally the only type found in naturally-occurring proteins. Biosynthetic and degradative pathways of each of the 20 proteinogenic amino acids have been well characterized in both prokaryotic and eukaryotic cells (see. for example. Strycr. L. Biochemistry. 3rd edition, pages 578-590 (1988)). The "essential" amino acids (histidine. isoleucine. leucine, lysine, methionine, phenylalanine., threonine, tryptophan, and valine), so named because they are generally a nutritional requirement due to the complexity of their biosyntheses, are readily converted by simple biosynthetic pathways to the remaining 11 'nonessential' amino acids (alanine, arginine, asparagine, aspartate, cysteine, glutamate, glutamine, glycine, proline, serine, and tyrosine). Higher animals do retain the ability to synthesize some of these amino acids, but the essential amino acids must be supplied from the diet in order for normal protein synthesis to occur.<br>
Aside from their function in protein biosynthesis, these amino acids are interesting chemicals in their own right, and many have been found to have various applications in the food, feed, chemical, cosmetics, agriculture, and pharmaceutical industries. Lysine is an important amino acid in the nutrition not only of humans, but also of monogastric animals such as poultry and swine. Glutamate is most commonly used as a flavor additive (mono-sodium glutamate, MSG) and is widely used throughout the food industry, as are aspartate, phenylalanine, glycine, and cysteine. Glycine, L-methionine and tryptophan are all utilized in the pharmaceutical industry. Glutamine, valine, leucine, isoleucine, histidine, arginine, proline, serine and alanine are of use in<br><br>
both the pharmaceutical and cosmetics industries. Threonine, tryptophan, and D/ L-methionine are common feed additives. (Leuchtenberger, W. (1996) Amino aids -technical production and use, p. 466-502 in Rehm et al (eds.) Biotechnology vol. 6, chapter 14a, VCH: Weinheim). Additionally, these amino acids have been found to be useful as precursors for the synthesis of synthetic amino acids and proteins, such as N-acetylcysteine, S-carboxymethyl-L-cysteine, (S)-5-hydroxytryptophan, and others described in Ulmann's Encyclopedia of Industrial Chemistry, vol. A2, p. 57-97. VCH: Weinheim. 1985.<br>
The biosynthesis of these natural amino acids in organisms capable of producing them, such as bacteria, has been well characterized (for review of bacterial amino acid biosynthesis and regulation thereof, see Umbarger, H.E.(1978) Ann. Rev. Biochem. 47: 533-606). Glutamate is synthesized by the reductive amination of a-ketoglutarate, an intermediate in the citric acid cycle. Glutamine. proline, and arginine are each subsequently produced from glutamate. The biosynthesis of serine is a three-step process beginning with 3-phosphoglycerate (an intermediate in glycolysis), and resulting in this amino acid after oxidation, transamination, and hydrolysis steps.  Both cysteine and glycine are produced from serine; the former by the condensation of homocysteine with serine, and the latter by the transferal of the side-chain 0-carbon atom to tetrahydrofolate, in a reaction catalyzed by serine transhydroxymethylase. Phenylalanine, and tyrosine are synthesized from the glycolytic and pentose phosphate pathway precursors erythrose 4-phosphate and phosphoenolpyruvate in a 9-step biosynthetic pathway that differ only at the final two steps after synthesis of prephenate. Tryptophan is also produced from these two initial molecules, but its synthesis is an 11-step pathway. Tyrosine may also be synthesized from phenylalanine, in a reaction catalyzed by phenylalanine hydroxylase. Alanine, valine, and leucine are all biosynthetic products of pyruvate, the final product of glycolysis. Aspartate is formed from oxaloacetate, an intermediate of the citric acid cycle. Asparagine, methionine, threonine, and lysine are each produced by the conversion of aspartate. Isoleucine is formed from threonine. A complex 9-step pathway results in the production of histidine from 5-phosphoribosyl-l -pyrophosphate, an activated sugar.<br>
Amino acids in excess of the protein synthesis needs of the cell cannot be stored, and are instead degraded to provide intermediates for the major metabolic pathways of<br><br>
the cell (for review see Stryer, L. Biochemistry 3rd ed. Ch. 21 "Amino Acid Degradation and the Urea Cycle" p. 495-516 (1988)). Although the cell is able to convert unwanted amino acids into useful metabolic intermediates, amino acid production is costly in terms of energy, precursor molecules, and the enzymes necessary to synthesize them. Thus it is not surprising that amino acid biosynthesis is regulated by feedback inhibition, in which the presence of a particular amino acid serves to slow or entirely stop its own production (for overview of feedback mechanisms in amino acid biosynthetic pathways, see Stryer. L. Biochemistry. 3rd ed. Ch. 24: 'Biosynthesis of Amino Acids and Heme" p. 575-600 (1988)). Thus, the output of any particular amino acid is limited by the amount of that amino acid present in the cell.<br>
B.   I 'itamin, Cofactor* and Nutraceutical Metabolism and Uses<br>
Vitamins, cofactors, and nutraceuticals comprise another group of molecules which the higher animals have lost the ability to synthesize and so must ingest, although they are readily synthesized by other organisms, such as bacteria. These molecules are either bioactive substances themselves, or are precursors of biologically active substances which may serve as electron carriers or intermediates in a variety of metabolic pathways. Aside from their nutritive value, these compounds also have significant industrial value as coloring agents, antioxidants, and catalysts or other processing aids. (For an overview of the structure, activity, and industrial applications of these compounds, see, for example, Ullman's Encyclopedia of Industrial Chemistry. "Vitamins" vol. A27, p. 443-613, VCH: Weinheim, 1996.) The term "vitamin" is art-recognized, and includes nutrients which are required by an organism for normal functioning, but which that organism cannot synthesize by itself. The group of vitamins may encompass cofactors and nutraceutical compounds. The language "cofactor" includes nonproteinaceous compounds required for a normal enzymatic activity to occur. Such compounds may be organic or inorganic; the cofactor molecules of the invention are preferably organic. The term "nutraceutical" includes dietary supplements having health benefits in plants and animals, particularly humans. Examples of such molecules are vitamins, antioxidants, and also certain lipids (e.g., polyunsaturated fatty acids).<br><br>
The biosynthesis of these molecules in organisms capable of producing them, such as bacteria, has been largely characterized (Ullman's Encyclopedia of Industrial Chemistry, "Vitamins" vol. A27, p. 443-613, VCH: Weinheim, 1996; Michal, G. (1999) Biochemical Pathways: An Atlas of Biochemistry and Molecular Biology, John Wiley &amp; Sons; Ong, A.S., Niki, E. &amp; Packer, L. (1995) "Nutrition, Lipids, Health, and Disease" Proceedings of the UNESCO/Confederation of Scientific and Technological Associations in Malaysia, and the Society for Free Radical Research - Asia, held Sept. 1-3. 1994 at Penang. Malaysia. AOCS Press: Champaign. IL X, 374 S).<br>
Thiamin (vitamin B\) is produced by the chemical coupling of pyrimidine and thiazole moieties. Riboflavin (vitamin B2) is synthesized from guanosine-5'-triphosphate (GTP) and ribose-5'-phosphate. Riboflavin, in turn, is utilized for the synthesis of flavin mononucleotide (FMN) and flavin adenine dinucleotide (FAD). The family of compounds collectively termed 'vitamin B6' (e.g., pyridoxine, pyridoxamine. pyridoxa-5*-phosphate. and the commercially used pyridoxin hydrochloride) are all derivatives of the common structural unit. 5-hydroxy-6-methylpyridine. Pantothenate (pantothenic acid. (R)-(+)-N-(2.4-dihydroxy-3.3-dimethyl-l-oxobutyl)-P-alanine) can be produced either by chemical synthesis or by fermentation. The final steps in pantothenate biosynthesis consist of the ATP-driven condensation of P-alanine and pantoic acid. The enzymes responsible for the biosynthesis steps for the conversion to pantoic acid, to P-alanine and for the condensation to panthotenic acid are known. The metabolically active form of pantothenate is Coenzyme A, for which the biosynthesis proceeds in 5 enzymatic steps. Pantothenate, pyridoxal-S'-phosphate, cysteine and ATP are the precursors of Coenzyme A. These enzymes not only catalyze the formation of panthothante, but also the production of (R)-pantoic acid, (R)-pantolacton, (R)-panthenol (provitamin B5), pantetheine (and its derivatives) and coenzyme A.<br>
Biotin biosynthesis from the precursor molecule pimeloyl-CoA in microorganisms has been studied in detail and several of the genes involved have been identified. Many of the corresponding proteins have been found to also be involved in Fe-cluster synthesis and are members of the nifS class of proteins. Lipoic acid is derived from octanoic acid, and serves as a coenzyme in energy metabolism, where it becomes part of the pyruvate dehydrogenase complex and the a-ketoglutarate dehydrogenase complex. The folates are a group of substances which are all derivatives<br><br>
of folic acid, which is turn is derived from L-glutamic acid, p-amino-benzoic acid and 6-methylpterin. The biosynthesis of folic acid and its derivatives, starting from the metabolism intermediates guanosine-S'-triphosphate (GTP), L-glutamic acid and p-amino-benzoic acid has been studied in detail in certain microorganisms.<br>
Corrinoids (such as the cobalamines and particularly vitamin B12) and porphyrines belong to a group of chemicals characterized by a tetrapyrole ring system. The biosynthesis of vitamin Bi: is sufficiently complex that it has not yet been completely characterized, but many of the enzymes and substrates involved are now known.  Nicotinic acid (nicotinate). and nicotinamide are pyridine derivatives which are also termed 'niacin'. Niacin is the precursor of the important coenzymes NAD (nicotinamide adenine dinucleotide) and NADP (nicotinamide adenine dinucleotide phosphate) and their reduced forms.<br>
The large-scale production of these compounds has largely relied on cell-free chemical syntheses, though some of these chemicals have also been produced by large-scale culture of microorganisms, such as riboflavin. Vitamin B6. pantothenate, and biotin. Only Vitamin Bi? is produced solely by fermentation, due to the complexity of its synthesis. In vitro methodologies require significant inputs of materials and time, often at great cost.<br>
C. Purine, Pyrimidine, Nucleoside and Nucleotide Metabolism and Uses<br>
Purine and pyrimidine metabolism genes and their corresponding proteins are important targets for the therapy of tumor diseases and viral infections. The language "purine" or "pyrimidine" includes the nitrogenous bases which are constituents of nucleic acids, co-enzymes, and nucleotides. The term "nucleotide" includes the basic structural units of nucleic acid molecules, which are comprised of a nitrogenous base, a pentose sugar (in the case of RNA, the sugar is ribose; in the case of DNA, the sugar is D-deoxyribose), and phosphoric acid. The language "nucleoside" includes molecules which serve as precursors to nucleotides, but which are lacking the phosphoric acid moiety that nucleotides possess. By inhibiting the biosynthesis of these molecules, or their mobilization to form nucleic acid molecules, it is possible to inhibit RNA and DNA synthesis; by inhibiting this activity in a fashion targeted to cancerous cells, the ability of tumor cells to divide and replicate may be inhibited. Additionally, there are<br><br>
nucleotides which do not form nucleic acid molecules, but rather serve as energy stores (i.e., AMP) or as coenzymes (i.e., FAD and NAD).<br>
Several publications have described the use of these chemicals for these medical indications, by influencing purine and/or pyrimidine metabolism (e.g. Christopherson, R.I. and Lyons, S.D. (1990) "Potent inhibitors of de novo pyrimidine and purine biosynthesis as chemotherapeutic agents." Med. Res. Reviews 10: 505-548). Studies of enzymes involved in purine and pyrimidine metabolism have been focused on the development of new drugs which can be used, for example, as immunosuppressants or anti-proliferants (Smith. J.L.. (1995) "Enzymes in nucleotide synthesis." Curr. Op'm. Struct. Biol. 5: 152-151 \ (1995) Biochem Soc. Transact. 23: 877-902). However, purine and pyrimidine bases, nucleosides and nucleotides have other utilities: as intermediates in the biosynthesis of several fine chemicals (e.g., thiamine, S-adenosyl-methionine. folates, or riboflavin), as energy carriers for the cell (e.g., ATP or GTP). and for chemicals themselves, commonly used as flavor enhancers (e.g.. IMP or GMP) or for several medicinal applications (see, for example, Kuninaka, A. (1996) Nucleotides and Related Compounds in Biotechnology vol. 6. Rehm et al, eds. VCH: Weinheim. p. 561-612), Also, enzymes involved in purine, pyrimidine. nucleoside, or nucleotide metabolism are increasingly serving as targets against which chemicals for crop protection, including fungicides, herbicides and insecticides, are developed.<br>
The.metabolism of these compounds in bacteria has been characterized (for reviews see, for example, Zalkin, H. and Dixon, J.E. (1992) "de novo purine nucleotide biosynthesis", in: Progress in Nucleic Acid Research and Molecular Biology, vol. 42, Academic Press:, p. 259-287; and Michal, G. (1999) "Nucleotides and Nucleosides", Chapter 8 in: Biochemical Pathways: An Atlas of Biochemistry and Molecular Biology, Wiley: New York). Purine metabolism has been the subject of intensive research, and is essential to the normal functioning of the cell. Impaired purine metabolism in higher animals can cause severe disease, such as gout. Purine nucleotides are synthesized from ribose-5-phosphate, in a series of steps through the intermediate compound inosine-5'-phosphate (IMP), resulting in the production of guanosine-5'-monophosphate (GMP) or adenosine-5'-monophosphate (AMP), from which the triphosphate forms utilized as nucleotides are readily formed. These compounds are also utilized as energy stores, so their degradation provides energy for many different biochemical processes in the cell.<br><br>
Pyrimidine biosynthesis proceeds by the formation of uridines-monophosphate (UMP) from ribose-5-phosphate. UMP, in turn, is converted to cytidine-5'-triphosphate (CTP). The deoxy- forms of all of these nucleotides are produced in a one step reduction reaction from the diphosphate ribose form of the nucleotide to the diphosphate deoxyribose form of the nucleotide. Upon phosphorylation, these molecules are able to participate in DNA synthesis.<br>
D.  Trehalose Metabolism and Uses<br>
Trehalose consists of two glucose molecules, bound in a. a-LI linkage. It is commonly used in the food industry as a sweetener, an additive for dried or frozen foods, and in beverages. However, it also has applications in the pharmaceutical, cosmetics and biotechnology industries (see, for example, Nishimoto el al, (1998) U.S. Patent No. 5,759.610; Singer. M.A. and Lindquist, S. (1998) Trends Biotech. 16: 460-467; Paiva. C.L.A. and Panek, A.D. (1996) Biotech. Ann. Rev. 2: 293-314; and Shiosaka, M. (1997) J. Japan 172: 97-102). Trehalose is produced by enzymes from many microorganisms and is naturally released into the surrounding medium, from which it can be collected using methods known in the art.<br>
II.        Resistance to Damage from Chemicals, Environmental Stress, and Antibiotics Production of fine chemicals is typically performed by large-scale culture of bacteria developed to produce and secrete large quantities of these molecules. However, this type of large-scale fermentation results in the subjection of the microorganisms to stresses of various kinds. These stresses include environmental stress and chemical stress.<br>
A.  Resistance to Environmental Stress<br>
Examples of environmental stresses typically encountered in large-scale fermentative culture include mechanical stress, heat stress, stress due to limited oxygen, stress due to oxygen radicals, pH stress, and osmotic stress. The stirring mechanism used in most large-scale fermentors to ensure aeration of the culture produces heat, thus increasing the temperature of the culture. Increases in temperature induce the well-characterized heat shock response, in which a set of proteins are expressed which not<br><br>
only aid in the survival of the bacterium in the face of high temperatures, but also increase survival in response to a number of other environmental stresses (see Neidhardt, F.C., et al, eds. (1996) E. coli and Salmonella. ASM Press: Washington, D.C., p. 1382-1399; Wosten, M. M. (1998) FEMS Microbiology Reviews 22(3): 127-50; Bahl, H. etal. (1995) FEMS Microbiology Reviews 17(3): 341-348; Zimmerman, J.L., Cohill, P.R. (1991) New Biologist 3(7): 641-650; Samali, A., and Orrenius. S. (1998) Cell. Stress Chaperones 3(4): 228-236. and references contained therein from each of these citations). Regulation of the heat shock response in bacteria is facilitated by specific sigma factors and other cellular regulators of gene expression (Hecker. M.. Volker, U(1998). Molecular Microbiology 29(5): 1129-1136). One of the largest problems that the cell encounters when exposed to high temperature is that protein folding is impaired; nascent proteins have sufficient kinetic energy in high temperature circumstances that it is difficult for the growing polypeptide chain to remain in a stable conformation long enough to fold properly. Thus, two of the key types of proteins expressed during the heat shock response consist of chaperones (proteins which assist in the folding or unfolding of other proteins - see. e.g.. Fink. A.L. (1999) Physiol. Rev. 79(2): 425-449). and proteases, which can destroy any improperly folded proteins. Examples of chaperones expressed during the heat shock response include GroEL and DNAK; proteases known to be expressed during this cellular reaction to heat shock include Lon, FtsH, and ClpB.<br>
Other environmental stresses besides heat may also provoke a stress response. Though the fermentor stirring process is meant to introduce oxygen into the culture, oxygen may remain in limited supply, particularly when the culture is advanced in growth and the oxygen needs of the culture are thereby increased; an insufficient supply of oxygen is another stress for the microorganism. Cells in fermentor cultures are also subjected to a number of osmotic stresses, particularly when nutrients are added to the culture, resulting in a high extracellular and low intracellular concentration of these molecules. Further, the large quantities of the desired molecules produced by these organisms in culture may contribute to osmotic stress of the bacteria. Lastly, aerobic metabolism such as that used by C. glutamicum results in carbon dioxide as a waste product; secretion of this molecule may acidify the culture medium due to conversion of this molecule to carboxylic acid. Thus, bacteria in culture are also frequently subjected<br><br>
to acidic pH stress. The converse may also be true - when high levels of basic waste molecules such as ammonium are present in the culture medium, the bacteria in culture may be subjected to basic pH stress as well.<br>
To combat such environmental stresses, bacteria have elegant gene systems which are expressed upon exposure to one or more stresses, such as the aforementioned heat shock system. Genes expressed in response to osmotic stress, for example, encode proteins capable of transporting or synthesizing compatible solutes such that osmotic intake or export of a particular molecule is slowed to manageable levels. Other examples of stress-induced bacterial proteins are those involved in trehalose biosynthesis, those encoding enzymes involved in ppGpp metabolism, those involved in signal transduction, particularly those encoding two-component systems which are sensitive to osmotic pressure, and those encoding transcription factors which are responsive to a variety of stress factors (e.g., RssB analogues and/or sigma factors). Many other such genes and their protein products are known in the art.<br>
B.  Resistance to Chemical Stress<br>
Aside from environmental stresses, cells may also experience a number of chemical stresses. These may fall into two categories. The first are natural waste products of metabolism and other cellular processes which are secreted by the cell to the surrounding medium. The second are chemicals present in the extracellular medium which do not originate from the cell. Generally, when cells excrete toxic waste products from the concentrated intracellular cytoplasm into the relatively much more dilute extracellular medium, these products dissipate such that extracellular levels of the possibly toxic compound are quite low. However, in large-scale fermentative culture of the bacterium, this may not be the case: so many bacteria are grown in a relatively small environment and at such a high metabolic rate that waste products may accumulate in the medium to nearly toxic levels. Examples of such wastes are carbon dioxide, metal ions, and reactive oxygen species such as hydrogen peroxide. These compounds may interfere with the activity or structure of cell surface molecules, or may re-enter the cell, where they can seriously damage proteins and nucleic acids alike. Certain other chemicals hazardous to the normal functioning of cells may be naturally found in the extracellular medium. For example, metal ions such as mercury, cadmium, nickel or<br><br>
copper are frequently found in water sources, and may form tight complexes with cellular enzymes which prevent the normal functioning of these proteins.<br>
C. Resistance to Antibiotics<br>
Bacteriocidal proteins or antibiotics, may also be found in the extracellular milieu, either through the intervention of the researcher, or as a natural product from another organism, utilized to gain a competitive advantage. Microorganisms have several art-known mechanisms to protect themselves against antimicrobial chemicals. Degradation, modification, and export of compounds toxic to the cell are common methods by which microorganisms eliminate or detoxify antibiotics. Cytoplasmic 'efflux-pumps' are known in several prokaryotes and show similarities to the so-called 'multidrug resistance" proteins from higher eukaryotes (Neyfakh, A. A. , et al. (1991) Proc. Natl Acad. Sci. USA 88: 4781-4785). Examples of such proteins include emrAB from E. coli (Lomovskaya. O. and K. Lewis (1992) Proc, Natl Acad Sci. L'SA 89: 8938-8942), lmrB from B. subtilis (Kumano, M. et al (1997) Microbiology 143: 2775-2782). smr from.V. aureus (Grinius. L.G. et al (1992) Plasmidll: 119-129) or cmr from C. glufamicum (Kaidoh. K. et al (1997) Micro. Drug Resist. 3: 345-350). C. glutamicum itself is non-pathogenic, in contrast to several other members of the genus Corynebacterium , such as C. diphtheriae or C. pseudotuberculosis. Several pathogenic Corynebacteria are known to have multiple resistances against a variety of antibiotics, such as C. jeikeium and C. urealyticum (Soriano, F. et al (\995) Antimicrob. Agents Chemother. 39:208-214).<br>
Lincosamides are recognized as effective antibiotics against Corynebacterium species (Soriano, F. et al (1995) Antimicrob. Agents Chemother. 39: 208-214). An unexpected result of the present invention was the identification of a gene encoding a Iincosamide-resistance protein (in particular, a lincomycin-resistance protein). The LMRB protein from C glutamicum shows 40% homology to the product of the lmrB gene from B. subtilis (see Genbank accession no. AL009126), as calculated using version 1.7 of the program CLUSTALW (Thompson, J.D., Higgins, D.G., Gibson, T. J. (1994) Nucl Acids Res. 22: 4673-4680) using standard parameters (PAIRWISE ALIGNMENT PARAMETERS: slow/accurate alignments: Gap Open Penalty = 10.00, Gap Extension Penalty = 0.10, Protein weight matrix = BLOSUM 30, DNA weight<br><br>
matrix = IUB, Fast/Approximate alignments: Gap penalty = 3, K-tuple (word) size = 1, No. of top diagonals = 5, Window size = 5, Toggle Slow/Fast pairwise alignments = slow. Multiple alignment parameters: Gap Opening Penalty = 10.00, Gap Extension Penalty = 0.05, Delay divergent sequences = 40%, DNA transitions weight = 0.50, Protein weight matrix = BLOSUM series, DNA weight matrix = IUB, Use negative matrix = OFF).<br>
Environmental stress, chemical stress, and antibiotic or other antimicrobial stress may influence the behavior of the microorganisms during fermentor culture, and may have an impact on the production of the desired compound from these organisms. For example, osmotic stress of a microorganism may cause inappropriate or inappropriately rapid uptake of one or more compounds which can ultimately lead to cellular damage or death due to osmotic shock. Similarly, chemicals present in the culture, either exogenously added (e.g., antimicrobial compounds intended to eliminate unwanted microbes) or generated by the bacteria themselves {e.g.. waste compounds such as heavy metals or oxygen radicals, or even antimicrobial compounds) may result in inhibition of fine chemical production or even death of the organism. The genes of the invention encode C. glutamicum proteins which act to prevent cell damage or death, by specifically counteracting the source or effect of the environmental or chemical stress.<br>
III. Elements and Methods of the Invention<br>
The present invention is based, at least in part, on the discovery of novel molecules, referred to herein as SRT nucleic acid and protein molecules, which increase the ability of C glutamicum to survive in chemically or environmentally hazardous settings. In one embodiment, the SRT molecules function to confer resistance to one or more environmental or chemical stresses to C. glutamicum. In a preferred embodiment, the activity of the SRT molecules of the present invention has an impact on the production of a desired fine chemical by this organism. In a particularly preferred embodiment, the SRT molecules of the invention are modulated in activity, such that the yield, production, and/or efficiency of production of one or more fine chemicals from C. glutamicum is also modulated.<br><br>
The language, "SRT protein" or "SRT polypeptide" includes proteins which participate in the resistance of C. glutamicum to one or more environmental or chemical stresses. Examples of SRT proteins include those encoded by the SRT genes set forth in Table 1 and by the odd-numbered SEQ ID NOs. The terms "SRT gene" or "SRT nucleic acid sequence' include nucleic acid sequences encoding an SRT protein, which consist of a coding region and also corresponding untranslated 5' and 3' sequence regions. Examples of SRT genes include those set forth in Table 1. The terms "production" or "productivity" are art-recognized and include the concentration of the fermentation product (for example, the desired fine chemical) formed within a given time and a given fermentation volume {e.g., kg product per hour per liter). The term "efficiency of production" includes the time required for a particular level of production to be achieved (for example, how long it takes for the cell to attain a particular rate of output of a fine chemical). The term "yield" or "product/carbon yield" is art-recognized and includes the efficiency of the conversion of the carbon source into the product (i.e.. fine chemical). This is generally written as, for example, kg product per kg carbon source. By increasing the yield or production of the compound, the quantity of recovered molecules, or of useful recovered molecules of that compound in a given amount of culture over a given amount of time is increased.   The terms "biosynthesis" or a "biosynthetic pathway" are art-recognized and include the synthesis of a compound, preferably an organic compound, by a cell from intermediate compounds in what may be a multistep and highly regulated process. The terms "degradation" or a "degradation pathway" are art-recognized and include the breakdown of a compound, preferably an organic compound, by a cell to degradation products (generally speaking, smaller or less complex molecules) in what may be a multistep and highly regulated process. The language "metabolism" is art-recognized and includes the totality of the biochemical reactions that take place in an organism. The metabolism of a particular compound, then, (e.g., the metabolism of an amino acid such as glycine) comprises the overall biosynthetic, modification, and degradation pathways in the cell related to this compound. The terms "resistance" and "tolerance" are art-known and include the ability of a cell to not be affected by exposure to a chemical or an environment which would otherwise be detrimental to the normal functioning of these organisms. The terms "stress" or "hazard" include factors which are detrimental to the normal functioning of<br><br>
cells such as C. glutamicum. Examples of stresses include "chemical stress", in which a cell is exposed to one or more chemicals which are detrimental to the cell, and "environmental stress" where a cell is exposed to an environmental condition outside of those to which it is adapted. Chemical stresses may be either natural metabolic waste products such as, but not limited to reactive oxygen species or carbon dioxide, or chemicals otherwise present in the environment, including, but not limited to heavy metal ions or bacteriocidal proteins such as antibiotics. Environmental stresses may be. but are not limited to temperatures outside of the normal range, suboptimal oxygen availability, osmotic pressures, or extremes of pH. for example.<br>
In another embodiment, the SRT molecules of the invention are capable of modulating the production of a desired molecule, such as a fine chemical, in a microorganism such as C. glutamicum. Using recombinant genetic techniques, one or more of the SRT proteins of the invention may be manipulated such that its function is modulated. The alteration of activity of stress response, resistance or tolerance genes such that the cell is increased in tolerance to one or more stresses may improve the ability of that cell to grow and multiply in the relatively stressful conditions of large-scale fermentor culture. For example, by overexpressing or engineering a heat-shock induced chaperone molecule such that it is optimized in activity, one may increase the ability of the bacterium to correctly fold proteins in the face of nonoptimal temperature conditions. By having fewer misfolded (and possibly misregulated or nonfunctional) proteins, the cell is increased in its ability to function normally in such a culture, which should in turn provide increased viability. This overall increase in number of cells having greater viability and activity in the culture should also result in an increase in the yield, production, and/or efficiency of production of one or more desired fine chemicals, due at least to the relatively greater number of cells producing these chemicals in the culture.<br>
The isolated nucleic acid sequences of the invention are contained within the genome of a Corynebacterium glutamicum strain available through the American Type Culture Collection, given designation ATCC 13032. The nucleotide sequence of the isolated C. glutamicum SRT DNAs and the predicted amino acid sequences of the C. glutamicum SRT proteins are shown the Sequence Listing as odd-numbered SEQ ID NOs and even-numbered SEQ ID NOs, respectively.,.<br><br>
Computational analyses were performed which classified and/or identified these nucleotide sequences as sequences which encode chemical and environmental stress, resistance, and tolerance proteins.<br>
The present invention also pertains to proteins which have an amino acid sequence which is substantially homologous to an amino acid sequence of the invention (e.g., the sequence of an even-numbered SEQ ID NO of the Sequence Listing). As used herein, a protein which has an amino acid sequence which is substantially homologous to a selected amino acid sequence is least about 50% homologous to the selected amino acid sequence, e.g., the entire selected amino acid sequence. A protein which has an amino acid sequence which is substantially homologous to a selected amino acid sequence can also be least about 50-60%, preferably at least about 60-70%, and more preferably at least about 70-80%, 80-90%, or 90-95%, and most preferably at least about 96%. 97%, 98%. 99% or more homologous to the selected amino acid sequence. Ranges and identity values intermediate to the above-recited values, (e.g.. 75%-80% identical, 85-87% identical. 91-92% identical) are also intended to be encompassed by the present invention. For example, ranges of identity values using a combination of any of the above values recited as upper and/or lower limits are intended to be included.<br>
The SRT proteins or biologically active portions or fragments thereof of the invention can confer resistance or tolerance to one or more chemical or environmental stresses, or may have one or more of the activities set forth in Table 1.<br>
Various aspects of the invention are described in further detail in the following subsections:<br>
A. Isolated Nucleic Acid Molecules<br>
One aspect of the invention pertains to isolated nucleic acid molecules that encode SRT polypeptides or biologically active portions thereof, as well as nucleic acid fragments sufficient for use as hybridization probes or primers for the identification or amplification of SRT-encoding nucleic acid (e.g., SRT DNA). As used herein, the term "nucleic acid molecule" is intended to include DNA molecules (e.g., cDNA or genomic DNA) and RNA molecules (e.g., mRNA) and analogs of the DNA or RNA generated using nucleotide analogs. This term also encompasses untranslated sequence located at both the 3' and 5' ends of the coding region of the gene: at least about 100 nucleotides<br><br>
of sequence upstream from the 5' end of the coding region and at least about 20 nucleotides of sequence downstream from the 3'end of the coding region of the gene. The nucleic acid molecule can be single-stranded or double-stranded, but preferably is double-stranded DNA. An "isolated" nucleic acid molecule is one which is separated from other nucleic acid molecules which are present in the natural source of the nucleic acid. Preferably, an "isolated" nucleic acid is free of sequences which naturally flank the nucleic acid (i.e., sequences located at the 5' and 3' ends of the nucleic acid) in the genomic DNA of the organism from which the nucleic acid is derived. For example, in various embodiments, the isolated SRT nucleic acid molecule can contain less than about 5 kb, 4kb, 3kb, 2kb, 1 kb, 0.5 kb or 0.1 kb of nucleotide sequences which naturally flank the nucleic acid molecule in genomic DNA of the cell from which the nucleic acid is derived (e.g, a C. glutamicum cell). Moreover, an "isolated" nucleic acid molecule, such as a DNA molecule, can be substantially free of other cellular material, or culture medium when produced by recombinant techniques, or chemical precursors or other chemicals when chemically synthesized.<br>
A nucleic acid molecule of the present invention, e.g.. a nucleic acid molecule having a nucleotide sequence of an odd-numbered SEQ ID NO of the Sequence Listing. or a portion thereof, can be isolated using standard molecular biology techniques and the sequence information provided herein. For example, a C. glutamicum SRT DNA can be isolated from a G glutamicum library using all or portion of one of the odd-numbered SEQ ID NO sequences of the Sequence Listing as a hybridization probe and standard hybridization techniques (e.g., as described in Sambrook, J., Fritsh, E. F., and Maniatis, T. Molecular Cloning: A Laboratory Manual. 2nd, ed, Cold Spring Harbor Laboratory, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989). Moreover, a nucleic acid molecule encompassing all or a portion of one of the nucleic acid sequences of the invention (e.g., an odd-numbered SEQ ID NO:) can be isolated by the polymerase chain reaction using oligonucleotide primers designed based upon this sequence (e.g., a nucleic acid molecule encompassing all or a portion of one of the nucleic acid sequences of the invention (e.g., an odd-numbered SEQ ID NO of the Sequence Listing) can be isolated by the polymerase chain reaction using oligonucleotide primers designed based upon this same sequence). For example, mRNA can be isolated from normal endothelial cells (e.g., by the guanidinium-thiocyanate<br><br>
extraction procedure of Chirgwin et al (1979) Biochemistry 18: 5294-5299) and DNA can be prepared using reverse transcriptase (e.g., Moloney MLV reverse transcriptase, available from Gibco/BRL, Bethesda, MD; or AMV reverse transcriptase, available from Seikagaku America. Inc., St. Petersburg, FL). Synthetic oligonucleotide primers for polymerase chain reaction amplification can be designed based upon one of the nucleotide sequences shown in the Sequence Listing. A nucleic acid of the invention can be amplified using cDNA or. alternatively, genomic DNA, as a template and appropriate oligonucleotide primers according to standard PCR amplification techniques. The nucleic acid so amplified can be cloned into an appropriate vector and characterized by DNA sequence analysis. Furthermore, oligonucleotides corresponding to an SRT nucleotide sequence can be prepared by standard synthetic techniques, e.g.. using an automated DNA synthesizer.<br>
In a preferred embodiment, an isolated nucleic acid molecule of the invention comprises one of the nucleotide sequences shown in the Sequence Listing. The nucleic acid sequences of the invention, as set forth in the Sequence Listing, correspond to the Corynebacterium glutamicum SRT DNAs of the invention. This DNA comprises sequences encoding SRT proteins (i.e.. the "coding region", indicated in each odd-numbered SEQ ID NO: sequence in the Sequence Listing), as well as 5' untranslated sequences and 3' untranslated sequences, also indicated in each odd-numbered SEQ ID NO: in the Sequence Listing. Alternatively, the nucleic acid molecule can comprise only the coding region of any of the nucleic acid sequences of the Sequence Listing.<br>
For the purposes of this application, it will be understood that each of the nucleic acid and amino acid sequences set forth in the Sequence Listing has an identifying RXA, RXN, or RXS number having the designation "RXA", "RXN", or "RXS" followed by 5 digits (/.., RXA01524, RXN00493, or RXS01027). Each of the nucleic acid sequences comprises up to three parts: a 5' upstream region, a coding region, and a downstream region. Each of these three regions is identified by the same RXA, RXN, or RXS designation to eliminate confusion. The recitation "one of the odd-numbered sequences of the Sequence Listing", then, refers to any of the nucleic acid sequences in the Sequence Listing,, which may be also be distinguished by their differing RXA, RXN, or RXS designations. The coding region of each of these sequences is translated into a corresponding amino acid sequence, which is also et forth in the Sequence Listing, as an<br><br>
even-numbered SEQ ID NO: immediately following the corresponding nucleic acid sequence. For example, the coding region for RXA01524 is set forth in SEQ ID NO: 1, while the amino acid sequence which it encodes is set forth as SEQ ID NO:2. The sequences of the nucleic acid molecules of the invention are identified by the same RXA, RXN, or RXS designations as the amino acid molecules which they encode, such that they can be readily correlated. For example, the amino acid sequence designated RXA01524 is a translation of the coding region of the nucleotide sequence of nucleic acid molecule RXA01524, the amino acid sequence designated RXN00034 is a translation of the coding region of the nucleotide sequence of nucleic acid molecule RXN00034, and the amino acid sequence in designated RXS00568 is a translation of the coding region of the nucleotide sequence of nucleic acid molecule RXS00568. The correspondence between the RXA, RXN, and RXS nucleotide and amino acid sequences of the invention and their assigned SEQ ID NOs is set forth in Table 1.<br>
Several of the genes of the invention are "F-designated genes'*. An F-designated gene includes those genes set forth in Table 1 which have an hF" in front of the RXA. RXN. or RXS designation. For example. SEQ ID NO:7. designated, as indicated on Table 1, as fcT RXA00498". is an F-designated gene, as are SEQ ID NOs: 25. 33. and 37 (designated on Table 1 as fcT RXA01345". "F RXA02543", and "F RXA02282". respectively).<br>
In one embodiment, the nucleic acid molecules of the present invention are not intended to include those compiled in Table 2. In the case of the dapD gene, a sequence for this gene was published in Wehrmann, A.,etai (1998) J. Bacleriol 180(12): 3159-3165. However, the sequence obtained by the inventors of the present application is significantly longer than the published version. It is believed that the published version relied on an incorrect start codon, and thus represents only a fragment of the actual coding region.<br>
In another preferred embodiment, an isolated nucleic acid molecule of the invention comprises a nucleic acid molecule which is a complement of one of the nucleotide sequences of the invention (e.g., a sequence of an odd-numbered SEQ ID NO: of the Sequence Listing, or a portion thereof A nucleic acid molecule which is complementary to one of the nucleotide sequences of the invention is one which is sufficiently complementary to one of the nucleotide sequences shown in the Sequence<br><br>
Listing (e.g., the sequence of an odd-numbered SEQ ID NO:) such that it can hybridize to one of the nucleotide sequences of the invention, thereby forming a stable duplex.<br>
In still another preferred embodiment, an isolated nucleic acid molecule of the invention comprises a nucleotide sequence which is at least about 50%, 51%, 52%, 53%. 54%, 55%, 56%, 57%, 58%, 59%, or 60%, preferably at least about 61%, 62%, 63%. 64%, 65%. 66%, 67%. 68%, 69%, or 70%%, more preferably at least about 71%, 72%. 73%. 74%. 75%. 76%, 77%. 78%. 79%, or 80%, 81%, 82%, 83%, 84%. 85%. 86%. 87%. 88%. 89%. or 90%, or 91%. 92%. 93%. 94%. and even more preferably at least about 95%. 96%. 97%. 98%. 99% or more homologous to a nucleotide sequence of the invention (e.g., a sequence of an odd-numbered SEQ ID NO: of the Sequence Listing), or a portion thereof. Ranges and identity values intermediate to the above-recited ranges, (e.g., 70-90% identical or 80-95% identical) are also intended to be encompassed by the present invention. For example, ranges of identity values using a combination of any of the above values recited as upper and/or lower limits are intended to be included. In an additional preferred embodiment, an isolated nucleic acid molecule of the invention comprises a nucleotide sequence which hybridizes, e.g.. hybridizes under stringent conditions, to one of the nucleotide sequences of the invention., or a portion thereof.<br>
Moreover, the nucleic acid molecule of the invention can comprise only a portion of the coding region of the sequence of one of the odd-numbered SEQ ID NOs of the Sequence Listing for example a fragment which can be used as a probe or primer or a fragment encoding a biologically active portion of an SRT protein. The nucleotide sequences determined from the cloning of the SRT genes from C. glutamicum allows for the generation of probes and primers designed for use in identifying and/or cloning SRT homologues in other cell types and organisms, as well as SRT homologues from other Corynebacteria or related species. The probe/primer typically comprises substantially purified oligonucleotide. The oligonucleotide typically comprises a region of nucleotide sequence that hybridizes under stringent conditions to at least about 12, preferably about 25, more preferably about 40, 50 or 75 consecutive nucleotides of a sense strand of one of the nucleotide sequences of the invention (e.g., a sequence of one of the odd-numbered SEQ ID NOs of the Sequence Listing),, an anti-sense sequence of one of these sequences, or naturally occurring mutants thereof. Primers based on a nucleotide sequence of the invention can be used in PCR reactions to clone SRT homologues.<br><br>
Probes based on the SRT nucleotide sequences can be used to detect transcripts or genomic sequences encoding the same or homologous proteins. In preferred embodiments, the probe further comprises a label group attached thereto, e.g. the label group can be a radioisotope, a fluorescent compound, an enzyme, or an enzyme co-factor. Such probes can be used as a part of a diagnostic test kit for identifying cells which misexpress an SRT protein, such as by measuring a level of an SRT-encoding nucleic acid in a sample of cells, e.g.. detecting SRT mRNA levels or determining whether a Genomic SRT cene has been mutated or deleted.<br>
In one embodiment, the nucleic acid molecule of the invention encodes a protein or portion thereof which includes an amino acid sequence which is sufficiently homologous to an amino acid sequence of the invention (e.g., a sequence of an even-numbered SEQ ID NO of the Sequence Listing) such that the protein or portion thereof maintains the ability to confer resistance or tolerance of C. glutamicum to one or more chemical or environmental stresses. As used herein, the language "sufficiently homologous" refers to proteins or portions thereof which have amino acid sequences which include a minimum number of identical or equivalent (e.g.. an amino acid residue which has a similar side chain as an amino acid residue in a sequence of one of the even-numbered SEQ ID NOs of the Sequence Listing) amino acid residues to an amino acid sequence of the invention such that the protein or portion thereof is capable of participating in the resistance of C. glutamicum to one or more chemical or environmental stresses. Protein members of such metabolic pathways, as described herein, function to increase the resistance or tolerance of C. glutamicum to one or more environmental or chemical hazards or stresses. Examples of such activities are also described herein. Thus, "the function of an SRT protein" contributes to the overall resistance of C glutamicum to elements of its surroundings which may impede its normal growth or functioning, and/or contributes, either directly or indirectly, to the yield, production, and/or efficiency of production of one or more fine chemicals. Examples of SRT protein activities are set forth in Table 1.<br>
In another embodiment, the protein is at least about 50-60%, preferably at least about 60-70%, and more preferably at least about 70-80%, 80-90%, 90-95%. and most preferably at least about 96%, 97%, 98%, 99% or more homologous to an entire amino acid sequence of the invention (e.g., a sequence of an even-numbered SEQ ID NO: of<br><br>
the Sequence Listing).   Ranges and identity values intermediate to the above-recited values, (e.g., 75%-80% identical, 85-87% identical, or 91-92% identical) are also intended to be encompassed by the present invention. For example, ranges of identity values using a combination of any of the above values recited as upper and/or lower limits are intended to be included.<br>
Portions of proteins encoded by the SRT nucleic acid molecules of the invention are preferably biologically active portions of one of the SRT proteins. As used herein, the term "biologically active portion of an SRT protein" is intended to include a portion. e.g.* a domain/motif, of an SRT protein that is capable of imparting resistance or tolerance to one or more environmental or chemical stresses or hazards, or has an activity as set forth in Table 1. To determine whether an SRT protein or a biologically active portion thereof can increase the resistance or tolerance of C. glutamicum to one or more chemical or environmental stresses or hazards, an assay of enzymatic activity may be performed. Such assay methods are well known to those of ordinary skill in the art. as detailed in Example 8 of the Exemplification.<br>
Additional nucleic acid fragments encoding biologically active portions of an SRT protein can be prepared by isolating a portion of one of the amino acid sequences of the invention (e.g., a sequence of an even-numbered SEQ ID NO: of the Sequence Listing), expressing the encoded portion of the SRT protein or peptide (e.g., by recombinant expression in vitro) and assessing the activity of the encoded portion of the SRT protein or peptide.<br>
The invention further encompasses nucleic acid molecules that differ from one of the nucleotide sequences of the invention (e.g., a sequence of an odd-numbered SEQ ID NO: of the Sequence Listing) (and portions thereof) due to degeneracy of the genetic code and thus encode the same SRT protein as that encoded by the nucleotide sequences of the invention. In another embodiment, an isolated nucleic acid molecule of the invention has a nucleotide sequence encoding a protein having an amino acid sequence shown in the Sequence Listing (e.g., an even-numbered SEQ ID NO:).. In a still further embodiment, the nucleic acid molecule of the invention encodes a full length C. glutamicum protein which is substantially homologous to an amino acid sequence of the invention (encoded by an open reading frame shown in an odd-numbered SEQ ID NO: of the Sequence Listing).<br><br>
It will be understood by one of ordinary skill in the art that in one embodiment the sequences of the invention are not meant to include the sequences of the prior art, such as those Genbank sequences set forth in Tables 2 or 4 which were available prior to the present invention. In one embodiment, the invention includes nucleotide and amino acid sequences having a percent identity to a nucleotide or amino acid sequence of the invention which is greater than that of a sequence of the prior art (e.g., a Genbank sequence (or the protein encoded by such a sequence) set forth in Tables 2 or 4). For example, the invention includes a nucleotide sequence which is greater than and/or at least 39% identical to the nucleotide sequence designated RXA00084 (SEQ ID NO: 189). a nucleotide sequence which is greater than and/or at least 56% identical to the nucleotide sequence designated RXA00605 (SEQ ID NO:l 1), and a nucleotide sequence which is greater than and/or at least 50% identical to the nucleotide sequence designated RXA00886 (SEQ ID NO:39). One of ordinary7 skill in the art would be able to calculate the lower threshold of percent identity for any given sequence of the invention by examining the GAP-calculated percent identity scores set forth in Table 4 for each of the three top hits for the given sequence, and by subtracting the highest GAP-calculated percent identity from 100 percent. One of ordinary skill in the art will also appreciate that nucleic acid and amino acid sequences having percent identities greater than the lower threshold so calculated (e.g., at least 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, or 60%, preferably at least about 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, or 70%, more preferably at least about 71%, 72%, 73%. 74%, 75%, 76%, 77%, 78%, 79%, or 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, or 90%, or 91%, 92%, 93%, 94%, and even more preferably at least about 95%, 96%, 97%, 98%, 99% or more identical) are also encompassed by the invention.<br>
In addition to the C. glutamicum SRT nucleotide sequences set forth in the Sequence Listing as odd-numbered SEQ ID NOs, it will be appreciated by one of ordinary skill in the art that DNA sequence polymorphisms that lead to changes in the amino acid sequences of SRT proteins may exist within a population (e.g., the C. glutamicum population). Such genetic polymorphism in the SRT gene may exist among individuals within a population due to natural variation. As used herein, the terms "gene" and "recombinant gene" refer to nucleic acid molecules comprising an open reading frame encoding an SRT protein, preferably a C. glutamicum SRT protein. Such<br><br>
natural variations can typically result in 1-5% variance in the nucleotide sequence of the SRT gene. Any and all such nucleotide variations and resulting amino acid polymorphisms in SRT that are the result of natural variation and that do not alter the functional activity of SRT proteins are intended to be within the scope of the invention.<br>
Nucleic acid molecules corresponding to natural variants and non-C glutamicum homologues of the C. glutamicum SRT DNA of the invention can be isolated based on their homology to the C. glutamicum SRT nucleic acid disclosed herein using the C. glutamicum DNA. or a portion thereof, as a hybridization probe according to standard hybridization techniques under stringent hybridization conditions. Accordingly, in another embodiment, an isolated nucleic acid molecule of the invention is at least 15 nucleotides in length and hybridizes under stringent conditions to the nucleic acid molecule comprising a nucleotide sequence of an odd-numbered SEQ ID NO: of the Sequence Listing. In other embodiments, the nucleic acid is at least 30. 50. 100. 250 or more nucleotides in length. As used herein, the term "hybridizes under stringent conditions" is intended to describe conditions for hybridization and washing under which nucleotide sequences at least 60% homologous to each other typically remain hybridized to each other. Preferably, the conditions are such that sequences at least about 65%, more preferably at least about 70%, and even more preferably at least about 75% or more homologous to each other typically remain hybridized to each other. Such stringent conditions are known to those of ordinary skill in the art in the art and can be found in Ausubel et al\ Current Protocols in Molecular Biology, John Wiley &amp; Sons, N.Y. (1989), 6.3.1-6.3.6. A preferred, non-limiting example of stringent hybridization conditions are hybridization in 6X sodium chloride/sodium citrate (SSC) at about 45°C, followed by one or more washes in 0.2 X SSC, 0.1% SDS at 50-65°C. Preferably, an isolated nucleic acid molecule of the invention that hybridizes under stringent conditions to a nucleotide sequence of the invention corresponds to a naturally-occurring nucleic acid molecule. As used herein, a "naturally-occurring" nucleic acid molecule refers to an RNA or DNA molecule having a nucleotide sequence that occurs in nature (e.g., encodes a natural protein). In one embodiment, the nucleic acid encodes a natural C glutamicum SRT protein.<br>
In addition to naturally-occurring variants of the SRT sequence that may exist in the population, one of ordinary skill in the art will further appreciate that changes can be<br><br>
introduced by mutation into a nucleotide sequence of the invention, thereby leading to changes in the amino acid sequence of the encoded SRT protein, without altering the functional ability of the SRT protein. For example, nucleotide substitutions leading to amino acid substitutions at "non-essential" amino acid residues can be made in a nucleotide sequence of the invention. A "non-essential" amino acid residue is a residue that can be altered from the wild-type sequence of one of the SRT proteins (e.g., an even-numbered SEQ ID NO: of the Sequence Listing) without altering the activity of said SRT protein, whereas an "essential" amino acid residue is required for SRT protein activity. Other amino acid residues, however, (e.g., those that are not conserved or only semi-conserved in the domain having SRT activity) may not be essential for activity and thus are likely to be amenable to alteration without altering SRT activity.<br>
Accordingly, another aspect of the invention pertains to nucleic acid molecules encoding SRT proteins that contain changes in amino acid residues that are not essential for SRT activity. Such SRT proteins differ in amino acid sequence from a sequence of an even-numbered SEQ ID NO: of the Sequence Listing yet retain at least one of the SRT activities described herein. In one embodiment, the isolated nucleic acid molecule comprises a nucleotide sequence encoding a protein, wherein the protein comprises an amino acid sequence at least about 50% homologous to an amino acid sequence of the invention and is capable of increasing the resistance or tolerance of C. glutamicum to one or more environmental or chemical stresses, or has one or more of the activities set forth in Table 1. Preferably, the protein encoded by the nucleic acid molecule is at least about 50-60% homologous to the amino acid sequence of one of the odd-numbered SEQ ID NOs of the Sequence Listing, more preferably at least about 60-70% homologous to one of these sequences, even more preferably at least about 70-80%, 80-90%, 90-95% homologous to one of these sequences in, and most preferably at least about 96%, 97%, 98%, or 99% homologous to one of the amino acid sequences of the invention.<br>
To determine the percent homology of two amino acid sequences (e.g., one of the amino acid sequences of the invention and a mutant form thereof) or of two nucleic acids, the sequences are aligned for optimal comparison purposes (e.g., gaps can be introduced in the sequence of one protein or nucleic acid for optimal alignment with the other protein or nucleic acid). The amino acid residues or nucleotides at corresponding amino acid positions or nucleotide positions are then compared. When a position in one<br><br>
sequence (e.g., one of the amino acid sequences of the invention) is occupied by the same amino acid residue or nucleotide as the corresponding position in the other sequence (e.g., a mutant form of the amino acid sequence), then the molecules are homologous at that position (i.e., as used herein amino acid or nucleic acid "homology" is equivalent to amino acid or nucleic acid "identity"). The percent homology between the two sequences is a function of the number of identical positions shared by the sequences (i.e.. % homology = # of identical positions/total # of positions x 100).<br>
An isolated nucleic acid molecule encoding an SRT protein homologous to a protein sequence of the invention (e.g.. a sequence of an even-numbered SEQ ID NO: of the Sequence Listing)can be created by introducing one or more nucleotide substitutions, additions or deletions into a nucleotide sequence of the invention such that one or more amino acid substitutions, additions or deletions are introduced into the encoded protein. Mutations can be introduced into one of the nucleotide sequences of the invention by standard techniques, such as site-directed mutagenesis and PCR-mediated mutagenesis. Preferably, conservative amino acid substitutions are made at one or more predicted non-essential amino acid residues. A "conservative amino acid substitution" is one in which the amino acid residue is replaced with an amino acid residue having a similar side chain. Families of amino acid residues having similar side chains have been defined in the art. These families include amino acids with basic side chains (e.g., lysine, arginine, histidine), acidic side chains (e.g., aspartic acid, glutamic acid), uncharged polar side chains (e.g., glycine, asparagine, glutamine, serine, threonine, tyrosine, cysteine), nonpolar side chains (e.g., alanine, valine, leucine, isoleucine, proline, phenylalanine, methionine, tryptophan), beta-branched side chains (e.g., threonine, valine, isoleucine) and aromatic side chains (e.g., tyrosine, phenylalanine, tryptophan, histidine). Thus, a predicted nonessential amino acid residue in an SRT protein is preferably replaced with another amino acid residue from the same side chain family. Alternatively, in another embodiment, mutations can be introduced randomly along all or part of an SRT coding sequence, such as by saturation mutagenesis, and the resultant mutants can be screened for an SRT activity described herein to identify mutants that retain SRT activity. Following mutagenesis of one the nucleotide sequence of one of the odd-numbered SEQ ID NOs of the Sequence Listing, the encoded protein can be expressed recombinantly and the activity of the protein can<br><br>
be determined using, for example, assays described herein (see Example 8 of the Exemplification).<br>
In addition to the nucleic acid molecules encoding SRT proteins described above, another aspect of the invention pertains to isolated nucleic acid molecules which are antisense thereto. An "antisense" nucleic acid comprises a nucleotide sequence which is complementary to a "sense" nucleic acid encoding a protein, e.g., complementary to the coding strand of a double-stranded DNA molecule or complementary to an mRNA sequence. Accordingly, an antisense nucleic acid can hydrogen bond to a sense nucleic acid. The antisense nucleic acid can be complementary to an entire SRT coding strand, or to only a portion thereof. In one embodiment, an antisense nucleic acid molecule is antisense to a "coding region" of the coding strand of a nucleotide sequence encoding an SRT protein. The term "coding region" refers to the region of the nucleotide sequence comprising codons which are translated into amino acid residues (e.g.. the entire coding region of SEQ ID NO.: 120 (RXA00600) comprises nucleotides 1 to!098). In another embodiment, the antisense nucleic acid molecule is antisense to a "noncoding region" of the coding strand of a nucleotide sequence encoding SRT. The term "noncoding region" refers to 5* and 3' sequences which flank the coding region that are not translated into amino acids (i.e., also referred to as 5' and 3' untranslated regions).<br>
Given the coding strand sequences encoding SRT disclosed herein (e.g., the sequences set forth as odd-numbered SEQ ID NOs in the Sequence Listing), antisense nucleic acids of the invention can be designed according to the rules of Watson and Crick base pairing. The antisense nucleic acid molecule can be complementary to the entire coding region of SRT mRNA, but more preferably is an oligonucleotide which is antisense to only a portion of the coding or noncoding region of SRT mRNA. For example, the antisense oligonucleotide can be complementary to the region surrounding the translation start site of SRT mRNA. An antisense oligonucleotide can be, for example, about 5, 10, 15, 20, 25, 30, 35,40, 45 or 50 nucleotides in length. An antisense nucleic acid of the invention can be constructed using chemical synthesis and enzymatic ligation reactions using procedures known in the art. For example, an antisense nucleic acid (e.g., an antisense oligonucleotide) can be chemically synthesized using naturally occurring nucleotides or variously modified nucleotides designed to<br><br>
increase the biological stability of the molecules or to increase the physical stability of the duplex formed between the antisense and sense nucleic acids, e.g., phosphdrothioate derivatives and acridine substituted nucleotides can be used. Examples of modified nucleotides which can be used to generate the antisense nucleic acid include 5-fluorouracil, 5-bromouraciI, 5-chlorouraciI, 5-iodouracil, hypoxanthine, xanthine, 4-acetylcytosine, 5-(carboxyhydroxylmethyl) uracil, 5-carboxymethylaminomethyl-2-thiouridine. 5-carboxymethylaminomethyluracil, dihydrouracil, beta-D-galactosylqueosine. inosine. N6-isopentenyiadenine. 1-methylguanine, 1-methylinosine. 2.2-dimethylguanine. 2-methyladenine. 2-methylguanine. 3-methylcytosine, 5-methylcytosine, N6-adenine, 7-methylguanine, 5-methylaminomethyluracil, 5-methoxyaminomethyl-2-thiouracil, beta-D-mannosylqueosine, 5'-methoxycarboxymethyluracil, 5-methoxyuracil, 2-methylthio-N6-isopentenyladenine. uracil-5-oxyacetic acid (v), wybutoxosine, pseudouraciL queosine, 2-thiocytosine. 5-methyl-2-thiouracil. 2-thiouracil. 4-thiouracil. 5-methyluraciL uracil-5- oxyacetic acid methylesten uracil-5-oxyacetic acid (v), 5-methyl-2-thiouracil, 3-(3-amino-3-N-2-carboxypropyl) uracil. (acp3)w. and 2.6-diaminopurine. Alternatively, the antisense nucleic acid can be produced biologically using an expression vector into which a nucleic acid has been subcloned in an antisense orientation (i.e., RNA transcribed from the inserted nucleic acid will be of an antisense orientation to a target nucleic acid of interest, described further in the following subsection).<br>
The antisense nucleic acid molecules of the invention are typically administered to a cell or generated in situ such that they hybridize with or bind to cellular mRNA and/or genomic DNA encoding an SRT protein to thereby inhibit expression of the protein, e.g., by inhibiting transcription and/or translation. The hybridization can be by conventional nucleotide complementarity to form a stable duplex, or, for example, in the case of an antisense nucleic acid molecule which binds to DNA duplexes, through specific interactions in the major groove of the double helix. The antisense molecule can be modified such that it specifically binds to a receptor or an antigen expressed on a selected cell surface, e.g., by linking the antisense nucleic acid molecule to a peptide or an antibody which binds to a cell surface receptor or antigen. The antisense nucleic acid molecule can also be delivered to cells using the vectors described herein. To achieve sufficient intracellular concentrations of the antisense molecules, vector constructs in<br><br>
which the antisense nucleic acid molecule is placed under the control of a strong prokaryotic, viral, or eukaryotic promoter are preferred.<br>
In yet another embodiment, the antisense nucleic acid molecule of the invention is anα -anomeric nucleic acid molecule. An a-anomeric nucleic acid molecule forms specific double-stranded hybrids with complementary RNA in which, contrary to the usual p-units. the strands run parallel to each other (Gaultier et al. (1987) Nucleic Acids. Res. 15:6625-6641). The antisense nucleic acid molecule can also comprise a 2'-o-methylribonucleotide (Inoue et al. (1987) Nucleic Acids Res. 15:6131-6148) or a chimeric RNA-DNA analogue (Inoue et al. (1987) FEBSLett. 215:327-330).<br>
In still another embodiment, an antisense nucleic acid of the invention is a ribozyme. Ribozymes are catalytic RNA molecules with ribonuclease activity which are capable of cleaving a single-stranded nucleic acid, such as an mRNA, to which they have a complementary region. Thus, ribozymes (e.g., hammerhead ribozymes (described in Haselhoff and Gerlach (1988) Nature 334:585-591)) can be used to catalytically cleave SRT mRNA transcripts to thereby inhibit translation of SRT mRNA. A ribozyme having specificity for an SRT-encoding nucleic acid can be designed based upon the nucleotide sequence of an SRT cDNA disclosed herein (i.e.. SEQ ID NO:l 19 (RXA00600)). For example, a derivative of a Tetrahymena L-19 IVS RNA can be constructed in which the nucleotide sequence of the active site is complementary to the nucleotide sequence to be cleaved in an SRT-encoding mRNA. See, e.g., Cech et al. U.S. Patent No. 4,987,071 and Cech et al. U.S. Patent No. 5,116,742. Alternatively, SRT mRNA can be used to select a catalytic RNA having a specific ribonuclease activity from a pool of RNA molecules. See, e.g., Bartel, D. and Szostak, J.W. (1993) Science 261:1411-1418.<br>
Alternatively, SRT gene expression can be inhibited by targeting nucleotide sequences complementary to the regulatory region of an SRT nucleotide sequence (e.g., an SRT promoter and/or enhancers) to form triple helical structures that prevent transcription of an SRT gene in target cells. See generally, Helene, C. (1991) Anticancer Drug Des. 6(6):569-84; Helene, C. et al. (1992) Ann. N.Y. Acad. Sci. 660:27-36; and Maher, L.J. (1992) Bioassays 14(12):807-15.<br><br>
B. Recombinant Expression Vectors and Host Cells<br>
Another aspect of the invention pertains to vectors, preferably expression vectors, containing a nucleic acid encoding an SRT protein (or a portion thereof)- As used herein, the term "vector" refers to a nucleic acid molecule capable of transporting another nucleic acid to which it has been linked. One type of vector is a "plasmid", which refers to a circular double stranded DNA loop into which additional DNA segments can be ligated. Another type of vector is a viral vector, wherein additional DNA segments can be ligated into the viral genome. Certain vectors are capable of autonomous replication in a host cell into which they are introduced (e.g.. bacterial vectors having a bacterial origin of replication and episomal mammalian vectors). Other vectors (e.g., non-episomal mammalian vectors) are integrated into the genome of a host cell upon introduction into the host cell, and thereby are replicated along with the host genome. Moreover, certain vectors are capable of directing the expression of genes to which they are operatively linked. Such vectors are referred to herein as "expression vectors". In general, expression vectors of utility in recombinant DNA techniques are often in the form of plasmids. In the present specification, "plasmid" and "vector" can be used interchangeably as the plasmid is the most commonly used form of vector. However, the invention is intended to include such other forms of expression vectors, such as viral vectors (e.g., replication defective retroviruses, adenoviruses and adeno-associated viruses), which serve equivalent functions.<br>
The recombinant expression vectors of the invention comprise a nucleic acid of the invention in a form suitable for expression of the nucleic acid in a host cell, which means that the recombinant expression vectors include one or more regulatory sequences, selected on the basis of the host cells to be used for expression, which is operatively linked to the nucleic acid sequence to be expressed. Within a recombinant expression vector, "operably linked" is intended to mean that the nucleotide sequence of interest is linked to the regulatory sequence(s) in a manner which allows for expression of the nucleotide sequence (e.g., in an in vitro transcription/translation system or in a host cell when the vector is introduced into the host cell). The term "regulatory sequence" is intended to include promoters, enhancers and other expression control elements (e.g., polyadenylation signals). Such regulatory sequences are described, for example, in Goeddel; Gene Expression Technology: Methods in Enzymology 185,<br><br>
Academic Press, San Diego, CA (1990). Regulatory sequences include those which direct constitutive expression of a nucleotide sequence in many types of host cell and those which direct expression of the nucleotide sequence only in certain host cells. Preferred regulatory sequences are, for example, promoters such as cos-, tac-, trp-, tet-. trp-tet-, lpp-, lac-, lpp-lac-, laclq-, T7-, T5-, T3-, gal-, trc-, ara-, SP6-, amy, SP02, X-PR-or X PL, which are used preferably in bacteria. Additional regulatory sequences are, for example, promoters from yeasts and fungi, such as ADO, MFa, AC. P-60. CYC1. GAPDH. TEF. rp28. ADH. promoters from plants such as CaMV/35S, SSU. OCS. lib-*. usp. STLS1. B33, nos or ubiquitin- or phaseolin-promoters. It is also possible to use artificial promoters. It will be appreciated by one of ordinary skill in the art that the design of the expression vector can depend on such factors as the choice of the host cell to be transformed, the level of expression of protein desired, etc. The expression vectors of the invention can be introduced into host cells to thereby produce proteins or peptides, including fusion proteins or peptides, encoded by nucleic acids as described herein (e.g., SRT proteins, mutant forms of SRT proteins, fusion proteins, etc.). The recombinant expression vectors of the invention can be designed for expression of SRT proteins in prokaryotic or eukaryotic cells. For example. SRT genes can be expressed in bacterial cells such as C. glutamicum, insect cells (using baculovirus expression vectors), yeast and other fungal cells (see Romanos, M.A. et al. (1992) 'Foreign gene expression in yeast: a review", Yeast 8: 423-488; van den Hondel, C.A.M.J.J. et al. (1991) "Heterologous gene expression in filamentous fungi" in: More Gene Manipulations in Fungi, J.W. Bennet &amp; L.L. Lasure, eds., p. 396-428: Academic Press: San Diego; and van den Hondel, C.A.M.J.J. &amp; Punt, P.J. (1991) "Gene transfer systems and vector development for filamentous fungi, in: Applied Molecular Genetics of Fungi, Peberdy, J.F. et al, eds., p. 1-28, Cambridge University Press: Cambridge), algae and multicellular plant cells (see Schmidt, R. and Willmitzer, L. (1988) High efficiency Agrobacterium tumefaciens -mediated transformation of Arabidopsis thaliana leaf and cotyledon explants" Plant Cell Rep:. 583-586), or mammalian cells. Suitable host cells are discussed further in Goeddel, Gene Expression Technology: Methods in Enzymology 185, Academic Press, San Diego, CA (1990). Alternatively, the recombinant expression vector can be transcribed and translated in vitro, for example using T7 promoter regulatory sequences and T7 polymerase.<br><br>
Expression of proteins in prokaryotes is most often carried out with vectors containing constitutive or inducible promoters directing the expression of either fusion or non-fusion proteins. Fusion vectors add a number of amino acids to a protein encoded therein, usually to the amino terminus of the recombinant protein. Such fusion vectors typically serve three purposes: 1) to increase expression of recombinant protein; 2) to increase the solubility of the recombinant protein; and 3) to aid in the purification of the recombinant protein by acting as a ligand in affinity purification. Often, in fusion expression vectors, a proteolytic cleavage site is introduced at the junction of the fusion moiety and the recombinant protein to enable separation of the recombinant protein from the fusion moiety subsequent to purification of the fusion protein. Such enzymes, and their cognate recognition sequences, include Factor Xa, thrombin and enterokinase.<br>
Typical fusion expression vectors include pGEX (Pharmacia Biotech Inc; Smith. D.B. and Johnson, K.S. (1988) Gene 67:31-40), pMAL (New England Biolabs. Beverly. MA) and pRIT5 (Pharmacia. Piscataway. N.1) which fuse glutathione S-transferase (GST), maltose E binding protein, or protein A, respectively, to the target recombinant protein. In one embodiment, the coding sequence of the SRT protein is cloned into a pGEX expression vector to create a vector encoding a fusion protein comprising, from the N-terminus to the C-terminus, GST-thrombin cleavage site-X protein. The fusion protein can be purified by affinity chromatography using glutathione-agarose resin. Recombinant SRT protein unfused to GST can be recovered by cleavage of the fusion protein with thrombin.<br>
Examples of suitable inducible non-fusion E. coli expression vectors include pTrc (Amann et a/., (1988) Gene 69:301-315) pLG338, pACYC184, pBR322, pUC18. pUC19, pKC30, pRep4, pHSl, pHS2, pPLc236, pMBL24, pLG200, pUR290, pIN-III113-B1, ^gtl 1, pBdCl, and pET lid (Studier et al, Gene Expression Technology: Methods in Enzymology 185, Academic Press, San Diego, California (1990) 60-89; and Pouwels et al.y eds. (1985) Cloning Vectors. Elsevier: New York IBSN 0 444 904018). Target gene expression from the pTrc vector relies on host RNA polymerase transcription from a hybrid trp-lac fusion promoter. Target gene expression from the pET 1 Id vector relies on transcription from a T7 gnlO-lac fusion promoter mediated by a coexpressed viral RNA polymerase (T7 gnl). This viral polymerase is supplied by host strains BL21(DE3) or HMS174(DE3) from a resident X prophage harboring a T7<br><br>
gnl gene under the transcriptional control of the lacUV 5 promoter. For transformation of other varieties of bacteria, appropriate vectors may be selected. For example, the plasmids pIJlOl, pIJ364, pIJ702 and pIJ361 are known to be useful in transforming Streptomyces, while plasmids pUBl 10, pC194, or pBD214 are suited for transformation of Bacillus species. Several plasmids of use in the transfer of genetic information into Corynebacterium include pHM1519, pBLl, pSA77, or pAJ667 (Pouwels et ai. eds. (1985) Cloning Vectors. Elsevier: New York IBSN 0 444 904018).<br>
One strategy to maximize recombinant protein expression is to express the protein in a host bacteria with an impaired capacity to proteolytically cleave the recombinant protein (Gottesman, S., Gene Expression Technology: Methods in Enzymology 185, Academic Press, San Diego, California (1990) 119-128). Another strategy is to alter the nucleic acid sequence of the nucleic acid to be inserted into an expression vector so that the individual codons for each amino acid are those preferentially utilized in the bacterium chosen for expression, such as C. ghitamicum (Wadae/tf/. (1992) Nucleic Acids Res. 20:2111-2118). Such alteration of nucleic acid sequences of the invention can be carried out by standard DNA synthesis techniques.<br>
In another embodiment, the SRT protein expression vector is a yeast expression vector. Examples of vectors for expression in yeast S. cerevisiae include pYepSecl (Baldari, etal.9 (1987) EmboJ. 6:229-234), 2 n, pAG-1, Yep6, Yepl3, pEMBLYe23, pMFa (Kurjan and Herskowitz, (1982) Cell 30:933-943), pJRY88 (Schultz et ai, (1987) Gene 54:113-123), and pYES2 (Invitrogen Corporation, San Diego, CA). Vectors and methods for the construction of vectors appropriate for use in other fungi, such as the filamentous fungi, include those detailed in: van den Hondel, C.A.MJ.J. &amp; Punt, P.J. (1991) "Gene transfer systems and vector development for filamentous fungi, in: Applied Molecular Genetics of Fungi, J.F. Peberdy, et ai, eds., p. 1-28, Cambridge University Press: Cambridge, and Pouwels et ai, eds. (1985) Cloning Vectors. Elsevier: New York (IBSN 0 444 904018).<br>
Alternatively, the SRT proteins of the invention can be expressed in insect cells using baculovirus expression vectors. Baculovirus vectors available for expression of proteins in cultured insect cells (e.g., Sf 9 cells) include the pAc series (Smith et ai (1983) Moi Cell Biol. 3:2156-2165) and the pVL series (Lucklow and Summers (1989) Virology 170:31-39).<br><br>
In another embodiment, the SRT proteins of the invention may be expressed in unicellular plant cells (such as algae) or in plant cells from higher plants (e.g., the spermatophytes, such as crop plants). Examples of plant expression vectors include those detailed in: Becker, D., Kemper, E., Schell, J. and Masterson, R. (1992) "New plant binary vectors with selectable markers located proximal to the left border", Plant Mol Biol 20: 1195-1197; and Bevan, M.W. (1984) "Binary Agrobacterium vectors for plant transformation", Nucl. Acid. Res. 12: 8711-8721, and include pLGV23, pGHlac*. pB!N19, pAK2004. and pDH51 (Pouwels et al.. eds. (1985) Cloning Vectors. Elsevier: New York 1BSN 0 444 904018).<br>
In yet another embodiment, a nucleic acid of the invention is expressed in mammalian cells using a mammalian expression vector. Examples of mammalian expression vectors include pCDM8 (Seed, B. (1987) Nature 329:840) and pMT2PC (Kaufman et al. (1987) EMBOJ. 6:187-195). When used in mammalian cells, the expression vector's control functions are often provided by viral regulatory elements. For example, commonly used promoters are derived from polyoma, Adenovirus 2. cytomegalovirus and Simian Virus 40. For other suitable expression systems for both prokaryotic and eukaryotic cells see chapters 16 and 17 of Sambrook. J.. Fritsh. E. F.. and Maniatis, T. Molecular Cloning; A Laboratory Manual 2nd, ed., Cold Spring Harbor Laboratory, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989.<br>
In another embodiment, the recombinant mammalian expression vector is capable of directing expression of the nucleic acid preferentially in a particular cell type (e.g., tissue-specific regulatory elements are used to express the nucleic acid). Tissue-specific regulatory elements are known in the art. Non-limiting examples of suitable tissue-specific promoters include the albumin promoter (liver-specific; Pinkert et al (1987) Genes Dev. 1:268-277), lymphoid-specific promoters (Calame and Eaton (1988) Adv. Immunol 43:235-275), in particular promoters of T cell receptors (Winoto and Baltimore (1989) EMBOJ. 8:729-733) and immunoglobulins (Banerji et al. (1983) Cell 33:729-740; Queen and Baltimore (1983) Cell 33:741-748), neuron-specific promoters (e.g., the neurofilament promoter; Byrne and Ruddle (1989) PNAS 86:5473-5477), pancreas-specific promoters (Edlund et al (1985) Science 230:912-916), and mammary gland-specific promoters (e.g., milk whey promoter; U.S. Patent No. 4,873,316 and<br><br>
European Application Publication No. 264,166). Developmentally-regulated promoters are also encompassed, for example the murine hox promoters (Kessel and Gruss (1990) Science 249:374-379) and the a-fetoprotein promoter (Campes and Tilghman (1989) Genes Dev. 3:537-546).<br>
The invention further provides a recombinant expression vector comprising a DNA molecule of the invention cloned into the expression vector in an antisense orientation. That is. the DNA molecule is operatively linked to a regulatory sequence in a manner which allows for expression (by transcription of the DNA molecule) of an RNA molecule which is antisense to SRT mRNA. Regulator)7 sequences operatively linked to a nucleic acid cloned in the antisense orientation can be chosen which direct the continuous expression of the antisense RNA molecule in a variety of cell types, for instance viral promoters and/or enhancers, or regulatory sequences can be chosen which direct constitutive, tissue specific or cell type specific expression of antisense RNA. The antisense expression vector can be in the form of a recombinant plasmid. phageinid or attenuated virus in which antisense nucleic acids are produced under the control of a high efficiency regulator)' region, the activity of which can be determined by the cell type into which the vector is introduced. For a discussion of the regulation of gene expression using antisense genes see Weintraub, H. et aly Antisense RNA as a molecular tool for genetic analysis. Reviews - Trends in Genetics, Vol. 1(1) 1986.<br>
Another aspect of the invention pertains to host cells into which a recombinant expression vector of the invention has been introduced. The terms "host cell" and "recombinant host cell" are used interchangeably herein. It is understood that such terms refer not only to the particular subject cell but to the progeny or potential progeny of such a cell. Because certain modifications may occur in succeeding generations due to either mutation or environmental influences, such progeny may not, in fact, be identical to the parent cell, but are still included within the scope of the term as used herein.<br>
A host cell can be any prokaryotic or eukaryotic cell. For example, an SRT protein can be expressed in bacterial cells such as C glutamicum, insect cells, yeast or mammalian cells (such as Chinese hamster ovary cells (CHO) or COS cells). Other suitable host cells are known to those of ordinary skill in the art. Microorganisms related<br><br>
to Corynebacterium gtutamicum wnicn may De conveniently usea as nosx ceus ior me nucleic acid and protein molecules of the invention are set forth in Table 3.<br>
Vector DNA can be introduced into prokaryotic or eukaryotic cells via conventional transformation or transfection techniques. As used herein, the terms "transformation" and "transfection" are intended to refer to a variety of art-recognized techniques for introducing foreign nucleic acid (e.g., linear DNA or RNA {e.g.. a linearized vector or a gene construct alone without a vector) or nucleic acid in the form of a vector (e.g., a plasmid. phage, phasmid. phagemid, transposon or other DNA)) into a host cell, including calcium phosphate or calcium chloride co-precipitation. DEAE-dextran-mediated transfection, lipofection, or electroporation. Suitable methods for transforming or transfecting host cells can be found in Sambrook, et al {Molecular Cloning: A Laboratory Manual 2nd, ed., Cold Spring Harbor Laboratory, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989), and other laborator}' manuals.<br>
For stable transfection of mammalian cells, it is known that, depending upon the expression vector and transfection technique used, only a small fraction of cells may integrate the foreign DNA into their genome. In order to identify and select these integrants, a gene that encodes a selectable marker (e.g.. resistance to antibiotics) is generally introduced into the host cells along with the gene of interest. Preferred selectable markers include those which confer resistance to drugs, such as G418, hygromycin and methotrexate. Nucleic acid encoding a selectable marker can be introduced into a host cell on the same vector as that encoding an SRT protein or can be introduced on a separate vector. Cells stably transfected with the introduced nucleic acid can be identified by drug selection (e.g., cells that have incorporated the selectable marker gene will survive, while the other cells die).<br>
To create a homologous recombinant microorganism, a vector is prepared which contains at least a portion of an SRT gene into which a deletion, addition or substitution has been introduced to thereby alter, e.g., functionally disrupt, the SRT gene. Preferably, this SRT gene is a Corynebacterium glutamicum SRT gene, but it can be a homologue from a related bacterium or even from a mammalian, yeast, or insect source. In a preferred embodiment, the vector is designed such that, upon homologous recombination, the endogenous SRT gene is functionally disrupted (i.e., no longer encodes a functional protein; also referred to as a "knock out" vector). Alternatively,<br><br>
the vector can be designed such that, upon homologous recombination, the endogenous SRT gene is mutated or otherwise altered but still encodes functional protein (e.g., the upstream regulatory region can be altered to thereby alter the expression of the endogenous SRT protein). In the homologous recombination vector, the altered portion of the SRT gene is flanked at its 5' and 3' ends by additional nucleic acid of the SRT gene to allow for homologous recombination to occur between the exogenous SRT gene carried by the vector and an endogenous SRT gene in a microorganism. The additional flanking SRT nucleic acid is of sufficient length for successful homologous recombination with the endogenous gene. Typically, several kilobases of flankine DNA (both at the 5' and 3' ends) are included in the vector (see e.g., Thomas, K.R., and Capecchi, M.R. (1987) Cell 51: 503 for a description of homologous recombination vectors). The vector is introduced into a microorganism (e.g., by electroporation) and cells in which the introduced SRT gene has homologously recombined with the endogenous SRT gene are selected, using art-known techniques.<br>
In another embodiment, recombinant microorganisms can be produced which contain selected systems which allow for regulated expression of the introduced gene. For example, inclusion of an SRT gene on a vector placing it under control of the lac operon permits expression of the SRT gene only in the presence of IPTG. Such regulatory systems are well known in the art.<br>
In another embodiment, an endogenous SRT gene in a host cell is disrupted (e.g., by homologous recombination or other genetic means known in the art) such that expression of its protein product does not occur. In another embodiment, an endogenous or introduced SRT gene in a host cell has been altered by one or more point mutations, deletions, or inversions, but still encodes a functional SRT protein. In still another embodiment, one or more of the regulatory regions (e.g., a promoter, repressor, or inducer) of an SRT gene in a microorganism has been altered (e.g., by deletion, truncation, inversion, or point mutation) such that the expression of the SRT gene is modulated. One of ordinary skill in the art will appreciate that host cells containing more than one of the described SRT gene and protein modifications may be readily produced using the methods of the invention, and are meant to be included in the present invention.<br><br>
A host cell of the invention, such as a prokaryotic or eukaryotic host cell in culture, can be used to produce (i.e., express) an SRT protein. Accordingly, the invention further provides methods for producing SRT proteins using the host cells of the invention. In one embodiment, the method comprises culturing the host cell of invention (into which a recombinant expression vector encoding an SRT protein has been introduced, or into which genome has been introduced a gene encoding a wild-type or altered SRT protein) in a suitable medium until SRT protein is produced. In another embodiment, the method further comprises isolating SRT proteins from the medium or the host cell.<br>
C. Isolated SR T Proteins<br>
Another aspect of the invention pertains to isolated SRT proteins, and biologically active portions thereof. An "isolated" or "purified" protein or biologically active portion thereof is substantially free of cellular material when produced by recombinant DNA techniques, or chemical precursors or other chemicals when chemically synthesized. The language "substantially free of cellular material" includes preparations of SRT protein in which the protein is separated from cellular components of the cells in which it is naturally or recombinantly produced. In one embodiment, the language "substantially free of cellular material" includes preparations of SRT protein having less than about 30% (by dry weight) of non-SRT protein (also referred to herein as a "contaminating protein"), more preferably less than about 20% of non-SRT protein, still more preferably less than about 10% of non-SRT protein, and most preferably less than about 5% non-SRT protein. When the SRT protein or biologically active portion thereof is recombinantly produced, it is also preferably substantially free of culture medium, i.e., culture medium represents less than about 20%, more preferably less than about 10%, and most preferably less than about 5% of the volume of the protein preparation. The language "substantially free of chemical precursors or other chemicals" includes preparations of SRT protein in which the protein is separated from chemical precursors or other chemicals which are involved in the synthesis of the protein. In one embodiment, the language "substantially free of chemical precursors or other chemicals" includes preparations of SRT protein having less than about 30% (by dry weight) of chemical precursors or non-SRT chemicals, more preferably less than<br><br>
about 20% chemical precursors or non-SRT chemicals, still more preferably less than about 10% chemical precursors or non-SRT chemicals, and most preferably less than about 5% chemical precursors or non-SRT chemicals. In preferred embodiments, isolated proteins or biologically active portions thereof lack contaminating proteins from the same organism from which the SRT protein is derived. Typically, such proteins are produced by recombinant expression of, for example, a C. glutamicum SRT protein in a microorganism such as C. glutamicum.<br>
An isolated SRT protein or a portion thereof of the invention can contribute to the resistance or tolerance of C glutamicum to one or more chemical or environmental stresses or hazards, or has one or more of the activities set forth in Table 1. In preferred embodiments, the protein or portion thereof comprises an amino acid sequence which is sufficiently homologous to an amino acid sequence of the invention (e.g.. a sequence of an even-numbered SEQ ID NO: of the Sequence Listing) such that the protein or portion thereof maintains the ability to mediate the resistance or tolerance of C. glutamicum to one or more chemical or environmental stresses or hazards. The portion of the protein is preferably a biologically active portion as described herein. In another preferred embodiment, an SRT protein of the invention has an amino acid sequence set forth as an even-numbered SEQ ID NO: of the Sequence Listing. In yet another preferred embodiment, the SRT protein has an amino acid sequence which is encoded by a nucleotide sequence which hybridizes, e.g., hybridizes under stringent conditions, to a nucleotide sequence of the invention (e.g., a sequence of an odd-numbered SEQ ID NO: of the Sequence Listing). In still another preferred embodiment, the SRT protein has an amino acid sequence which is encoded by a nucleotide sequence that is at least about 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, or 60%, preferably at least about 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, or 70%, more preferably at least about 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, or 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, or 90%, or 91%, 92%, 93%, 94%, and even more preferably at least about 95%, 96%. 97%, 98%, 99% or more homologous to one of the nucleic acid sequences of the invention , or a portion thereof. Ranges and identity values intermediate to the above-recited values, (e.g., 70-90% identical or 80-95% identical) are also intended to be encompassed by the present invention. For example, ranges of identity values using a combination of any of the above values recited as upper<br><br>
and/or lower limits are intended to be included. The preferred SRT proteins of the present invention also preferably possess at least one of the SRT activities described herein. For example, a preferred SRT protein of the present invention includes an amino acid sequence encoded by a nucleotide sequence which hybridizes, e.g., hybridizes under stringent conditions, to a nucleotide sequence of the invention, and which can increase the resistance or tolerance of C. glutamicum to one or more environmental or chemical stresses, or which has one or more of the activities set forth in Table 1.<br>
In other embodiments, the SRT protein is substantially homologous to an amino acid sequence of the invention (e.g., a sequence of an even-numbered SEQ ID NO: of the Sequence Listing) and retains the functional activity of the protein of one of the amino acid sequences of the invention yet differs in amino acid sequence due to natural variation or mutagenesis, as described in detail in subsection I above. Accordingly, in another embodiment, the SRT protein is a protein which comprises an amino acid sequence which is at least about 50%. 51%. 52%. 53%. 54%. 55%. 56%. 57%. 58%. 59%. or 60%. preferably at least about 61%, 62%. 63%, 64%. 65%, 66%. 67%. 68%. 69%. or 70%. more preferably at least about 71%. 72%. 73%. 74%. 75%. 76%. 77%. 78%. 79%. or 80%. 81%, 82%. 83%. 84%. 85%. 86%. 87%. 88%. 89%. or 90%. or 91%, 92%, 93%, 94%, and even more preferably at least about 95%, 96%. 97%, 98%. 99% or more homologous to an entire amino acid sequence of the invention and which has at least one of the SRT activities described herein. Ranges and identity values intermediate to the above-recited values, (e.g., 70-90% identical or 80-95% identical) are also intended to be encompassed by the present invention. For example, ranges of identity values using a combination of any of the above values recited as upper and/or lower limits are intended to be included. In another embodiment, the invention pertains to a full length C. glutamicum protein which is substantially homologous to an entire amino acid sequence of the invention .<br>
Biologically active portions of an SRT protein include peptides comprising amino acid sequences derived from the amino acid sequence of an SRT protein, e.g., an amino acid sequence of an even-numbered SEQ ID NO: of the Sequence Listing or the amino acid sequence of a protein homologous to an SRT protein, which include fewer amino acids than a full length SRT protein or the full length protein which is homologous to an SRT protein, and exhibit at least one activity of an SRT protein.<br><br>
Typically, biologically active portions (peptides, e.g., peptides which are, for example, 5, 10, 15, 20, 30, 35, 36, 37, 38, 39,40, 50, 100 or more amino acids in length) comprise a domain or motif with at least one activity of an SRT protein. Moreover, other biologically active portions, in which other regions of the protein are deleted, can be prepared by recombinant techniques and evaluated for one or more of the activities described herein. Preferably, the biologically active portions of an SRT protein include one or more selected domains/motifs or portions thereof having biological activity.<br>
SRT proteins are preferably produced by recombinant DNA techniques. For example, a nucleic acid molecule encoding the protein is cloned into an expression vector (as described above), the expression vector is introduced into a host cell (as described above) and the SRT protein is expressed in the host cell. The SRT protein can then be isolated from the cells by an appropriate purification scheme using standard protein purification techniques. Alternative to recombinant expression, an SRT protein, polypeptide, or peptide can be synthesized chemically using standard peptide synthesis techniques. Moreover, native SRT protein can be isolated from cells (e.g., endothelial cells), for example using an anti-SRT antibody, which can be produced by standard techniques utilizing an SRT protein or fragment thereof of this invention.<br>
The invention also provides SRT chimeric or fusion proteins. As used herein, an SRT "chimeric protein" or "fusion protein" comprises an SRT polypeptide operatively linked to a non-SRT polypeptide. An "SRT polypeptide" refers to a polypeptide having an amino acid sequence corresponding to SRT, whereas a "non-SRT polypeptide" refers to a polypeptide having an amino acid sequence corresponding to a protein which is not substantially homologous to the SRT protein, e.g., a protein which is different from the SRT protein and which is derived from the same or a different organism. Within the fusion protein, the term "operatively linked" is intended to indicate that the SRT polypeptide and the non-SRT polypeptide are fused in-frame to each other. The non-SRT polypeptide can be fused to the N-terminus or C-terminus of the SRT polypeptide. For example, in one embodiment the fusion protein is a GST-SRT fusion protein in which the SRT sequences are fused to the C-terminus of the GST sequences. Such fusion proteins can facilitate the purification of recombinant SRT proteins. In another embodiment, the fusion protein is an SRT protein containing a heterologous signal sequence at its N-terminus. In certain host cells (e.g., mammalian host cells), expression<br><br>
and/or secretion of an SRT protein can be increased through use of a heterologous signal sequence.<br>
Preferably, an SRT chimeric or fusion protein of the invention is produced by standard recombinant DNA techniques. For example, DNA fragments coding for the different polypeptide sequences are ligated together in-frame in accordance with conventional techniques, for example by employing blunt-ended or stagger-ended termini for ligation, restriction enzyme digestion to provide for appropriate termini, filling-in of cohesive ends as appropriate, alkaline phosphatase treatment to avoid undesirable joining, and enzymatic ligation. In another embodiment, the fusion gene can be synthesized by conventional techniques including automated DNA synthesizers. Alternatively. PCR amplification of gene fragments can be carried out using anchor primers which give rise to complementary overhangs between two consecutive gene fragments which can subsequently be annealed and reamplified to generate a chimeric gene sequence (see. for example. Current Protocols in Molecular Biology, eds. Ausubcl ei al. John Wiley &amp; Sons: 1992). Moreover, many expression vectors are commercially available that already encode a fusion moiety (e.g.. a GST polypeptide). An SRT-encoding nucleic acid can be cloned into such an expression vector such that the fusion moiety is linked in-frame to the SRT protein.<br>
Homologues of the SRT protein can be generated by mutagenesis, e.g., discrete point mutation or truncation of the SRT protein. As used herein, the term "homologue" refers to a variant form of the SRT protein which acts as an agonist or antagonist of the activity of the SRT protein. An agonist of the SRT protein can retain substantially the same, or a subset, of the biological activities of the SRT protein. An antagonist of the SRT protein can inhibit one or more of the activities of the naturally occurring form of the SRT protein, by, for example, competitively binding to a downstream or upstream member of the SRT system which includes the SRT protein. Thus, the C. glutamicum SRT protein and homologues thereof of the present invention may increase the tolerance or resistance of C. glutamicum to one or more chemical or environmental stresses.<br>
In an alternative embodiment, homologues of the SRT protein can be identified by screening combinatorial libraries of mutants, e.g., truncation mutants, of the SRT protein for SRT protein agonist or antagonist activity. In one embodiment, a variegated lihrarv of SRT variants is generated bv combinatorial mutagenesis at the nucleic acid<br><br>
level and is encoded by a variegated gene library. A variegated library of SRT variants can be produced by, for example, enzymatically Hgating a mixture of synthetic oligonucleotides into gene sequences such that a degenerate set of potential SRT sequences is expressible as individual polypeptides, or alternatively, as a set of larger fusion proteins (e.g., for phage display) containing the set of SRT sequences therein. There are a variety of methods which can be used to produce libraries of potential SRT homologues from a degenerate oligonucleotide sequence. Chemical synthesis of a degenerate gene sequence can be performed in an automatic DNA synthesizer, and the synthetic gene then ligated into an appropriate expression vector. Use of a degenerate set of genes allows for the provision, in one mixture, of all of the sequences encoding the desired set of potential SRT sequences. Methods for synthesizing degenerate oligonucleotides are known in the art (see, e.g., Narang, S.A. (1983) Tetrahedron 39:3; Itakura et al. (1984) Arrnu. Rev. Biochem. 53:323; Itakura et al (1984) Science 198:1056; Ike et al (1983) Nucleic Acid Res. 11:477.<br>
In addition, libraries of fragments of the SRT protein coding can be used to generate a variegated population of SRT fragments for screening and subsequent selection of homologues of an SRT protein. In one embodiment, a library of coding sequence fragments can be generated by treating a double stranded PCR fragment of an SRT coding sequence with a nuclease under conditions wherein nicking occurs only about once per molecule, denaturing the double stranded DNA, renaturing the DNA to form double stranded DNA which can include sense/antisense pairs from different nicked products, removing single stranded portions from reformed duplexes by treatment with SI nuclease, and Hgating the resulting fragment library into an expression vector. By this method, an expression library can be derived which encodes N-terminal, C-terminal and internal fragments of various sizes of the SRT protein.<br>
Several techniques are known in the art for screening gene products of combinatorial libraries made by point mutations or truncation, and for screening cDNA libraries for gene products having a selected property. Such techniques are adaptable for rapid screening of the gene libraries generated by the combinatorial mutagenesis of SRT homologues. The most widely used techniques, which are amenable to high through-put analysis, for screening large gene libraries typically include cloning the gene library into replicable expression vectors, transforming appropriate cells with the resulting library of<br><br>
vectors, and expressing the combinatorial genes under conditions in which detection of a desired activity facilitates isolation of the vector encoding the gene whose product was detected. Recursive ensemble mutagenesis (REM), a new technique which enhances the frequency of functional mutants in the libraries, can be used in combination with the screening assays to identify SRT homologues (Arkin and Yourvan (1992) PNAS 59:7811-7815; Delgrave et al (1993) Protein Engineering 6(3):327-331).<br>
In another embodiment, cell based assays can be exploited to analyze a variegated SRT library, using methods well known in the art.<br>
D.   Uses and Methods of the Invention<br>
The nucleic acid molecules, proteins, protein homologues, fusion proteins, primers, vectors, and host cells described herein can be used in one or more of the following methods: identification of C glutamicum and related organisms; mapping of genomes of organisms related to C. glutamicum: identification and localization of (' glutamicum sequences of interest; evolutionary studies; determination of SRT protein regions required for function; modulation of an SRT protein activity; modulation of the activity of an SRT pathway; and modulation of cellular production of a desired compound, such as a fine chemical.<br>
The SRT nucleic acid molecules of the invention have a variety of uses. First, they may be used to identify an organism as being Corynebacterium glutamicum or a close relative thereof. Also, they may be used to identify the presence of C. glutamicum or a relative thereof in a mixed population of microorganisms. The invention provides the nucleic acid sequences of a number of C. glutamicum genes; by probing the extracted genomic DNA of a culture of a unique or mixed population of microorganisms under stringent conditions with a probe spanning a region of a C. glutamicum gene which is unique to this organism, one can ascertain whether this organism is present.<br>
Although Corynebacterium glutamicum itself is nonpathogenic, it is related to pathogenic species, such as Corynebacterium diphtheriae. Corynebacterium diphtheriae is the causative agent of diphtheria, a rapidly developing, acute, febrile infection which involves both local and systemic pathology. In this disease, a local lesion develops in the upper respiratory tract and involves necrotic injury to epithelial cells; the bacilli secrete toxin which is disseminated through this lesion to distal susceptible tissues of the<br><br>
body. Degenerative changes brought about by the inhibition of protein synthesis in these tissues, which include heart, muscle, peripheral nerves, adrenals, kidneys, liver and spleen, result in the systemic pathology of the disease. Diphtheria continues to have high incidence in many parts of the world, including Africa, Asia, Eastern Europe and the independent states of the former Soviet Union. An ongoing epidemic of diphtheria in the latter two regions has resulted in at least 5,000 deaths since 1990.<br>
In one embodiment, the invention provides a method of identifying the presence or activity of Cornyebacterium diphtheriae in a subject. This method includes detection of one or more of the nucleic acid or amino acid sequences of the invention (e.g., the sequences set forth as odd-numbered or even-numbered SEQ ID NOs, respectively, in the Sequence Listing) in a subject, thereby detecting the presence or activity of Corynebacterium diphtheriae in the subject. C. glutamicum and C. diphtheriae are related bacteria, and many of the nucleic acid and protein molecules in C. glutamicum are homologous to C. diphtheriae nucleic acid and protein molecules, and can therefore be used to detect C. diphtheriae in a subject.<br>
The nucleic acid and protein molecules of the invention may also serve as markers for specific regions of the genome. This has utility not only in the mapping of the genome, but also for functional studies of C. glutamicum proteins. For example, to identify the region of the genome to which a particular C glutamicum DNA-binding protein binds, the C glutamicum genome could be digested, and the fragments incubated with the DNA-binding protein. Those which bind the protein may be additionally probed with the nucleic acid molecules of the invention, preferably with readily detectable labels; binding of such a nucleic acid molecule to the genome fragment enables the localization of the fragment to the genome map of C glutamicum, and, when performed multiple times with different enzymes, facilitates a rapid determination of the nucleic acid sequence to which the protein binds. Further, the nucleic acid molecules of the invention may be sufficiently homologous to the sequences of related species such that these nucleic acid molecules may serve as markers for the construction of a genomic map in related bacteria, such as Brevibacterium lactofermentum.<br>
The SRT nucleic acid molecules of the invention are also useful for evolutionary and protein structural studies. The resistance processes in which the molecules of the invention participate are utilized by a wide variety of cells; by comparing the sequences<br><br>
of the nucleic acid molecules of the present invention to those encoding similar enzymes from other organisms, the evolutionary relatedness of the organisms can be assessed. Similarly, such a comparison permits an assessment of which regions of the sequence are conserved and which are not, which may aid in determining those regions of the protein which are essential for the functioning of the enzyme. This type of determination is of value for protein engineering studies and may give an indication of what the protein can tolerate in terms of mutagenesis without losing function.<br>
The genes of the invention, e.g.. the gene encoding LMRB (SEQ ID NO: 1) or other gene of the invention encoding a chemical or environmental resistance or tolerance protein (e.g., resistance against one or more antibiotics), may be used as genetic markers for the genetic transformation of (e.g., the transfer of additional genes into or disruption of preexisting genes of) organisms such as C glutamicum or other bacterial species. Use of these nucleic acid molecules permits efficient selection of organisms which have incorporated a given transgene cassette (e.g.. a plasmid. phage, phasmid. phagemid. transposon, or other nucleic acid element), based on a trait which permits the survival of the organism in an otherwise hostile or toxic environment (e.g.. in the presence of an antimicrobial compound). By employing one or more of the genes of the invention as genetic markers, the speed and ease with which organisms having desirable transformed traits (e.g., modulated fine chemical production) are engineered and isolated are improved. While it is advantageous to use the genes of the invention for selection of transformed C. glutamicum and related bacteria, it is possible, as described herein, to use homologs (e.g., homologs from other organisms), allelic variants or fragments of the gene retaining desired activity. Furthermore, 5' and 3' regulatory elements of the genes of the invention may be modified as described herein (e.g., by nucleotide substitution, insertion, deletion, or replacement with a more desirable genetic element) to modulate the transcription of the gene. For example, an LMRB variant in which the nucleotide sequence in the region from -1 to -200 5' to the start codon has been altered to modulate (preferably increase) the transcription and/or translation of LMRB may be employed, as can constructs in which a gene of the invention (e.g., the LMRB gene (SEQ ID NO:l)) is functionally coupled to one or more regulatory signals (e.g., inducer or repressor binding sequences) which can be used for modulating gene expression.<br><br>
Similarly, more than one copy of a gene (functional or inactivated) of the invention may be employed.<br>
An additional application of the genes of the invention (e.g., the gene encoding LMRB (SEQ ID NO:l) or other drug- or antibiotic-resistance gene) is in the discovery of new antibiotics which are active against Corynebacteria and/or other bacteria. For example, a gene of the invention may be expressed (or overexpressed) in a suitable host to generate an organism with increased resistance to one or more drugs or antibiotics (in the case of LMRB. lincosamides in particular, especially lincomycin). This type of resistant host can subsequently be used to screen for chemicals with bacteriostatic and or bacteriocidal activity, such as novel antibiotic compounds. It is possible, in particular, to use the genes of the invention (e.g.* the LMRB gene) to identify new antibiotics which are active against those microorganisms which are already resistant to standard antibiotic compounds.<br>
The invention provides methods for screening molecules which modulate the activity of an SRT protein, either by interacting with the protein itself or a substrate or binding partner of the SRT protein, or by modulating the transcription or translation of a SRT nucleic acid molecule of the invention. In such methods, a microorganism expressing one or more SRT proteins of the invention is contacted with one or more test compounds, and the effect of each test compound on the activity or level of expression of the SRT protein is assessed.<br>
Manipulation of the SRT nucleic acid molecules of the invention may result in the production of SRT proteins having functional differences from the wild-type SRT proteins. These proteins may be improved in efficiency or activity, may be present in greater numbers in the cell than is usual, or may be decreased in efficiency or activity. The goal of such manipulations is to increase the viability and activity of the cell when the cell is exposed to the environmental and chemical stresses and hazards which frequently accompany large-scale fermentative culture. Thus, by increasing the activity or copy number of a heat-shock-regulated protease, one may increase the ability of the cell to destroy incorrectly folded proteins, which may otherwise interfere with normal cellular functioning (for example, by continuing to bind substrates or cofactors although the protein lacks the activity to act on these molecules appropriately). The same is true for the overexpression or optimization of activity of one or more chaperone molecules<br><br>
induced by heat or cold shock. These proteins aid in the correct folding of nascent polypeptide chains, and thus their increased activity or presence should increase the percentage of correctly folded proteins in the cell, which in turn should increase the overall metabolic efficiency and viability of the cells in culture. The overexpression or optimization of the transporter molecules activated by osmotic shock should result in an increased ability on the part of the cell to maintain intracellular homeostasis, thereby increasing the viability of these cells in culture. Similarly, the overproduction or increase in activity by mutagenesis of proteins involved in the development of cellular resistance to chemical stresses of various kinds (either by transport of the offending chemical out of the cell or by modification of the chemical to a less hazardous substance) should increase the fitness of the organism in the environment containing the hazardous substance (i.e., large-scale fermentative culture), and thereby may permit relatively larger numbers of cells to survive in such a culture. The net effect of all of these mutagenesis strategies is to increase the quantity of fine-chemical-producing compounds in the culture, thereby increasing the yield, production, and/or efficiency of production of one or more desired fine chemicals from the culture.<br>
This aforementioned list of mutagenesis strategies for SRT proteins to result in increased yields of a desired compound is not meant to be limiting; variations on these mutagenesis strategies will be readily apparent to one of ordinary skill in the art. By these mechanisms, the nucleic acid and protein molecules of the invention may be utilized to generate C. glutamicum or related strains of bacteria expressing mutated SRT nucleic acid and protein molecules such that the yield, production, and/or efficiency of production of a desired compound is improved. This desired compound may be any natural product of C glutamicum, which includes the final products of biosynthesis pathways and intermediates of naturally-occurring metabolic pathways, as well as molecules which do not naturally occur in the metabolism of C. glutamicum, but which are produced by a C. glutamicum strain of the invention.<br>
This invention is further illustrated by theTfollowing examples which should not be construed as limiting. The contents of all references, patent applications, patents, published patent applications, Tables, and the sequence listing cited throughout this application are hereby incorporated by reference.<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
ATCC: American Type Culture Collection, Rockville, MD, USA<br>
FERM: Fermentation Research Institute. Chiba, Japan<br>
NRRL: ARS Culture Collection, Northern Regional Research Laboratory, Peoria, IL, USA<br>
CECT: Coleccion Espanola de Cultivos Tipo, Valencia, Spain<br>
NCIMB: National Collection of Industrial and Marine Bacteria Ltd., Aberdeen, UK<br>
CBS: Centraalbureau voor Schimmelcultures, Baarn, NL<br>
NCTC: National Collection of Type Cultures, London, UK<br>
DSMZ: Deutsche Sammlung von Mikroorganismen und Zellkulturen, Braunschweig, Germany<br>
For reference see Sugawara, H. et al. (1993) World directory of collections of cultures of microorganisms: Bacteria, fungi and yeasts (4th edn), World federation for culture collections world data center on microorganisms, Saimata, Japen.<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
Exemplification<br>
Example 1: Preparation of total genomic DNA of Corynebacterium glutamic urn ATCC 13032<br>
A culture of Corynebacterium glutamicum (ATCC 13032) was grown overnight at 30°C with vigorous shaking in BHI medium (Difco). The cells were harvested by centrifugation, the supernatant was discarded and the cells were resuspended in 5 ml buffer-I (5% of the original volume of the culture — all indicated volumes have been calculated for 100 ml of culture volume). Composition of buffer-I:  140.34 g/1 sucrose. 2.46 g/1 MgS04 x 7H:0, 10 ml/1 KH2P04 solution (100 g/l, adjusted to pH 6.7 with KOH), 50 ml/1 M12 concentrate (10 g/1 (NH4)2S04, 1 g/1 NaCl, 2 g/1 MgS04 x 7HA 0.2 g/1 CaCl,, 0.5 g/1 yeast extract (Difco), 10 ml/1 trace-elements-mix (200 mg/1 FeS04 x HA 10 mg/1 ZnS04 x 7 HA 3 mg/1 MnCl2 x 4 HA 30 mg/1 H3BO, 20 mg/1 CoCl: x 6 HA 1 mg/1 NiCl: x 6 HA 3 mg/1 Na.MoO, x 2 HA 500 mg/1 complexing agent (EDTA or critic acid), 100 ml/1 vitamins-mix (0.2 mg/1 biotin, 0.2 mg/1 folic acid. 20 mg/1 p-amino benzoic acid, 20 mg/1 riboflavin, 40 mg/1 ca-panthothenate. 140 mg/1 nicotinic acid, 40 mg/1 pyridoxole hydrochloride, 200 mg/1 myo-inositol). Lysozymc was added to the suspension to a final concentration of 2.5 mg/ml. After an approximately 4 h incubation at 37°C, the cell wall was degraded and the resulting protoplasts are harvested by centrifugation. The pellet was washed once with 5 ml buffer-I and once with 5 ml TE-buffer (10 mM Tris-HCl, 1 mM EDTA, pH 8). The pellet was resuspended in 4 ml TE-buffer and 0.5 ml SDS solution (10%) and 0.5 ml NaCl solution (5 M) are added. After adding of proteinase K to a final concentration of 200 jig/ml, the suspension is incubated for ca.l 8 h at 37°C. The DNA was purified by extraction with phenol, phenol-chloroform-isoamylalcohol and chloroform-isoamylalcohol using standard procedures. Then, the DNA was precipitated by adding 1/50 volume of 3 M sodium acetate and 2 volumes of ethanol, followed by a 30 min incubation at -20°C and a 30 min centrifugation at 12,000 rpm in a high speed centrifuge using a SS34 rotor (Sorvall). The DNA was dissolved in 1 ml TE-buffer containing 20 Hg/ml RNaseA and dialysed at 4°C against 1000 ml TE-buffer for at least 3 hours. During this time, the buffer was exchanged 3 times. To aliquots of 0.4 ml of the dialysed DNA solution, 0.4 ml of 2 M LiCl and 0.8 ml of ethanol are added. After a 30<br><br>
min incubation at -20°C, the DNA was collected by centrifugation (13,000 rpm, Biofiige Fresco, Heraeus, Hanau, Germany). The DNA pellet was dissolved in TE-buffer. DNA prepared by this procedure could be used for all purposes, including southern blotting or construction of genomic libraries.<br>
Example 2: Construction of genomic libraries in Escherichia coli of Corynebacterium glutamicum ATCC13032.<br>
Using DNA prepared as described in Example 1. cosmid and plasmid libraries were constructed according to known and well established methods (see e.g., Sambrook. J. et al. (1989) "Molecular Cloning : A Laboratory Manual", Cold Spring Harbor Laboratory Press, or AusubeL F.M. et al (1994) "Current Protocols in Molecular Biology", John Wiley &amp; Sons.)<br>
Any plasmid or cosmid could be used. Of particular use were the plasmids pBR322 (Sutcliffe, J.G. (1979) Proc. Natl. Acad. Sci. USA, 75:3737-3741); pACYC177 (Change &amp; Cohen (1978) J. Bacteriol 134:1141-1156). plasmids of the pBS series (pBSSK+, pBSSK- and others; Stratagene. LaJolla, USA), or cosmids as SuperCosl (Stratagene, LaJolla, USA) or Lorist6 (Gibson. T.J., Rosenthal A. and Waterson. R.H. (1987) Gene 53:283-286. Gene libraries specifically for use in C. glutamicum may be constructed using plasmid pSL109 (Lee, H.-S. and A. J. Sinskey (1994) J. Microbiol Biotechnol. 4: 256-263).<br>
Example 3: DNA Sequencing and Computational Functional Analysis<br>
Genomic libraries as described in Example 2 were used for DNA sequencing<br>
according to standard methods, in particular by the chain termination method using<br>
ABI377 sequencing machines (see e.g., Fleischman, R.D. et al (1995) "Whole-genome<br>
Random Sequencing and Assembly of Haemophilus Influenzae Rd., Science, 269:496-<br>
512). Sequencing primers with the following nucleotide sequences were used: 5'-<br>
F	(SEQ  ID 1*0:305)<br>
GGAAACAGTATGACCATG-3^or5'-GTAAAACGACGGCCAGT-3'.(SEQ  ID MO:306)<br>
Example 4: In vivo Mutagenesis<br>
In vivo mutagenesis of Corynebacterium glutamicum can be performed by passage of plasmid (or other vector) DNA through E. coli or other microorganisms (e.g. Bacillus spp. or yeasts such as Saccharomyces cerevisiae) which are impaired in their capabilities to maintain<br><br>
the integrity of their genetic information. Typical mutator strains have mutations in the genes for the DNA repair system (e.g.. mutHLS, mutD, mutT, etc.; for reference, see Rupp, W.D. (1996) DNA repair mechanisms, in: Escherichia coh and Salmonella, p. 2277-2294, ASM: Washington.) Such strains are well known to those of ordinary skill in the art. The use of such strains is illustrated, for example, in Greener, A. and Callahan, M. (1994) Strategies 7: 32-34.<br>
Example 5: DNA Transfer Between Escherichia coli and Corynehacterium glutamicum<br>
Several Corynehacterium and Brevibacterium species contain endogenous plasmids (as e.g., pHM1519 or pBLl) which replicate autonomously (for review see, e.g., Martin, J.F. et al. (1987) Biotechnology, 5:137-146). Shuttle vectors for Escherichia coli and Corynehacterium glutamicum can be readily constructed by using standard vectors for E. coli (Sambrook, J. et al. (1989), "Molecular Cloning: A Laboratory Manual", Cold Spring Harbor Laboratory Press or Ausubel, F.M, et al. (1994) "Current Protocols in Molecular Biology", John Wiley &amp; Sons) to which a origin or replication for and a suitable marker from Corynehacterium glutamicum is added. Such origins of replication are preferably taken from endogenous plasmids isolated from Corynehacterium and Brevibacterium species. Of particular use as transformation markers for these species are genes for kanamycin resistance (such as those derived from the Tn5 or Tn903 transposons) or chloramphenicol (Winnacker, E.L. (1987) "From Genes to Clones — Introduction to Gene Technology, VCH, Weinheim). There are numerous examples in the literature of the construction of a wide variety of shuttle vectors which replicate in both E. coli and C. glutamicum, and which can be used for several purposes, including gene over-expression (for reference, see e.g., Yoshihama, M. et al. (1985) J. BacterioL 162:591-597, Martin J.F. et al. (1987) Biotechnology, 5:137-146 and Eikmanns, B.J. et al. (1991) Gene, 102:93-98).<br>
Using standard methods, it is possible to clone a gene of interest into one of the shuttle vectors described above and to introduce such a hybrid vector into strains of Corynehacterium glutamicum. Transformation of C glutamicum can be achieved by protoplast transformation (Kastsumata, R. et al. (1984) J. BacterioL 159306-311), electroporation (Liebl, E. et al (1989) FEMS Microbiol. Letters, 53:399-303) and in cases where special vectors are used, also by conjugation (as described e.g. in Schafer, A et al.<br><br>
(1990) J. Bacteriol. 172:1663-1666). It is also possible to transfer the shuttle vectors for C. glutamicum to E. coli by preparing plasmid DNA from C. glutamicum (using standard methods well-known in the art) and transforming it into E. coli. This transformation step can be performed using standard methods, but it is advantageous to use an Mcr-deficient E. coli strain, such as NM522 (Gough &amp; Murray (1983) J. Mol. BioL 166:1-19).<br>
Genes may be overexpressed in C. glutamicum strains using plasmids which comprise pCGl (U.S. Patent No. 4,617,267) or fragments thereof, and optionally the gene for kanamycin resistance from TN903 (Grindley, N.D. and Joyce, CM. (1980) Proc. Natl Acad. Sci. USA 77(12): 7176-7180). In addition, genes may be overexpressed in C. glutamicum strains using plasmid pSL109 (Lee, H.-S. and A. J. Sinskey (1994) J. Microbiol Biotechnol 4: 256-263).<br>
Aside from the use of replicative plasmids, gene overexpression can also be achieved by integration into the genome. Genomic integration in C. glutamicum or other Corynebacterium or Brevibacterium species may be accomplished by well-known methods, such as homologous recombination with genomic region(s), restriction endonuclease mediated integration (REMI) (see, e.g., DE Patent 19823834). or through the use of transposons. It is also possible to modulate the activity of a gene of interest by modifying the regulatory regions (e.g., a promoter, a repressor, and/or an enhancer) by sequence modification, insertion, or deletion using site-directed methods (such as homologous recombination) or methods based on random events (such as transposon mutagenesis or REMI). Nucleic acid sequences which function as transcriptional terminators may also be inserted 3' to the coding region of one or more genes of the invention; such terminators are well-known in the art and are described, for example, in Winnacker, E.L. (1987) From Genes to Clones - Introduction to Gene Technology. VCH: Weinheim.<br>
Example 6: Assessment of the Expression of the Mutant Protein<br>
Observations of the activity of a mutated protein in a transformed host cell rely on the fact that the mutant protein is expressed in a similar fashion and in a similar quantity to that of the wild-type protein. A useful method to ascertain the lev^i of transcription of the mutant gene (an indicator of the amount of mRNA available for translation to the gene product) is to perform a Northern blot (for reference see, for example, Ausubel et al.<br><br>
(1988) Current Protocols in Molecular Biology, Wiley: New York), in which a primer designed to bind to the gene of interest is labeled with a detectable tag (usually radioactive or chemiluminescent), such that when the total RNA of a culture of the organism is extracted, run on gel, transferred to a stable matrix and incubated with this probe, the binding and quantity of binding of the probe indicates the presence and also the quantity of mRNA for this gene. This information is evidence of the degree of transcription of the mutant gene. Total cellular RNA can be prepared from Corynebacterium glutamicum by several methods, all well-known in the art, such as that described in Bormann. E.R. et al (1992) Mol Microbiol 6: 317-326.<br>
To assess the presence or relative quantity of protein translated from this mRNA. standard techniques, such as a Western blot, may be employed (see, for example, Ausubel et al. (1988) Current Protocols in Molecular Biology, Wiley: New York). In this process, total cellular proteins are extracted, separated by gel electrophoresis, transferred to a matrix such as nitrocellulose, and incubated with a probe, such as an antibody, which specifically binds to the desired protein. This probe is generally tagged with a chemiluminescent or colorimetric label which may be readily detected. The presence and quantity of label observed indicates the presence and quantity of the desired mutant protein present in the cell.<br>
Example 7: Growth of Genetically Modified Corynebacterium glutamicum — Media and Culture Conditions<br>
Genetically modified Corynebacteria are cultured in synthetic or natural growth media. A number of different growth media for Corynebacteria are both well-known and readily available (Lieb et al. (1989) Appl. Microbiol. BiotechnoL, 32:205-210; von der Osten et al (1998) Biotechnology Letters, 11:11-16; Patent DE 4,120,867; Liebl (1992) "The Genus Corynebacterium, in: The Procaryotes, Volume II, Balows, A. et al, eds. Springer-Verlag). These media consist of one or more carbon sources, nitrogen sources, inorganic salts, vitamins and trace elements. Preferred carbon sources are sugars, such as mono-, di-, or polysaccharides. For example, glucose, fructose, mannose, galactose, ribose, sorbose, ribulose, lactose, maltose, sucrose, raffinose, starch or cellulose serve as very good carbon sources.   It is also possible to supply sugar to the media via complex compounds such as molasses or other by-products from sugar refinement. It can also be<br><br>
advantageous to supply mixtures of different carbon sources. Other possible carbon sources are alcohols and organic acids, such as methanol, ethanol, acetic acid or lactic acid. Nitrogen sources are usually organic or inorganic nitrogen compounds, or materials which contain these compounds. Exemplary nitrogen sources include ammonia gas or ammonia salts, such as NH4C1 or (NH4)2S04, NH4OH, nitrates, urea, amino acids or complex nitrogen sources like corn steep liquor, soy bean flour, soy bean protein, yeast extract, meat extract and others.<br>
Inorganic salt compounds which may be included in the media include the chloride-, phosphorous- or sulfate- salts of calcium, magnesium, sodium, cobalt. molybdenum, potassium, manganese, zinc, copper and iron. Chelating compounds can be added to the medium to keep the metal ions in solution. Particularly useful chelating compounds include dihydroxyphenols, like catechol or protocatechuate, or organic acids, such as citric acid. It is typical for the media to also contain other growth factors, such as vitamins or growth promoters, examples of which include biotin, riboflavin, thiamin, folic acid, nicotinic acid, pantothenate and pyridoxin. Growth factors and salts frequently originate from complex media components such as yeast extract, molasses, corn steep liquor and others. The exact composition of the media compounds depends strongly on the immediate experiment and is individually decided for each specific case. Information about media optimization is available in the textbook "Applied Microbiol. Physiology, A Practical Approach {eds. P.M. Rhodes, P.F. Stanbury, IRL Press (1997) pp. 53-73, ISBN 0 19 963577 3). It is also possible to select growth media from commercial suppliers, like standard 1 (Merck) or BHI (grain heart infusion, DIFCO) or others.<br>
All medium components are sterilized, either by heat (20 minutes at 1.5 bar and 12TC) or by sterile filtration. The components can either be sterilized together or, if necessary, separately. All media components can be present at the beginning of growth, or they can optionally be added continuously or batchwise.<br>
Culture conditions are defined separately for each experiment. The temperature should be in a range between 15°C and 45°C The temperature can be kept constant or can be altered during the experiment. The pH of the medium should be in the range of 5 to 8.5, preferably around 7.0, and can be maintained by the addition of buffers to the media. An exemplary buffer for this purpose is a potassium phosphate buffer. Synthetic buffers such as MOPS, HEPES, ACES and others can alternatively or simultaneously be used. It<br><br>
is also possible to maintain a constant culture pH through the addition of NaOH or NH4OH during growth. If complex medium components such as yeast extract are utilized, the necessity for additional buffers may be reduced, due to the fact that many complex compounds have high buffer capacities. If a fermentor is utilized for culturing the microorganisms, the pH can also be controlled using gaseous ammonia.<br>
The incubation time is usually in a range from several hours to several days. This time is selected in order to permit the maximal amount of product to accumulate in the broth. The disclosed growth experiments can be carried out in a variety of vessels, such as microtiter plates, glass tubes, glass flasks or glass or metal fermentors of different sizes. For screening a large number of clones, the microorganisms should be cultured in microtiter plates, glass tubes or shake flasks, either with or without baffles. Preferably 100 ml shake flasks are used, filled with 10% (by volume) of the required growth medium. The flasks should be shaken on a rotary shaker (amplitude 25 mm) using a speed-range of 100 - 300 rpm. Evaporation losses can be diminished by the maintenance of a humid atmosphere; alternatively, a mathematical correction for evaporation losses should be performed.<br>
If genetically modified clones are tested, an unmodified control clone or a control clone containing the basic plasmid without any insert should also be tested. The medium is inoculated to an OD600 of 0.5 - 1.5 using cells grown on agar plates, such as CM plates (10 g/1 glucose, 2,5 g/1 NaCl, 2 g/1 urea, 10 g/1 polypeptone, 5 g/1 yeast extract, 5 g/1 meat extract, 22 g/1 NaCl, 2 g/1 urea, 10 g/1 polypeptone, 5 g/1 yeast extract, 5 g/1 meat extract, 22 g/1 agar, pH 6.8 with 2M NaOH) that had been incubated at 30*C. Inoculation of the media is accomplished by either introduction of a saline suspension of C. glutamicum cells from CM plates or addition of a liquid preculture of this bacterium.<br>
Example 8 - In vitro Analysis of the Function of Mutant Proteins<br>
The determination of activities and kinetic parameters of enzymes is well established in the art. Experiments to determine the activity of any given altered enzyme must be tailored to the specific activity of the wild-type enzyme, which is well within the ability of one of ordinary skill in the art. Overviews about enzymes in general, as well as specific details concerning structure, kinetics, principles, methods, applications and examples for the determination of many enzyme activities may be<br><br>
found, for example, in the following references: Dixon, M, and Webb, E.C., (1979) Enzymes. Longmans: London; Fersht, (1985) Enzyme Structure and Mechanism. Freeman: New York; Walsh, (1979) Enzymatic Reaction Mechanisms. Freeman: San Francisco; Price, N.C., Stevens, L. (1982) Fundamentals of Enzymology. Oxford Univ. Press: Oxford; Boyer, P.D., ed. (1983) The Enzymes, 3rd ed. Academic Press: New York; Bisswanger, H., (1994) Enzymkinetik, 2nd ed. VCH: Weinheim (ISBN 3527300325); Bergmeyer, H.U., Bergmeyer, J., GraBl, M., eds. (1983-1986) Methods of Enzymatic Analysis. 3rd ed.. vol. I-XIL Verlag Chemie: Weinheim; and Ullmann's Encyclopedia of Industrial Chemistry (1987) vol. A9, '"Enzymes'". VCH: Weinheim. p. 352-363.<br>
The activity of proteins which bind to DNA can be measured by several well-established methods, such as DNA band-shift assays (also called gel retardation assays). The effect of such proteins on the expression of other molecules can be measured using reporter gene assays (such as that described in Kolmar, H. et al (1995) EMBQ J. 14: 3895-3904 and references cited therein). Reporter gene test systems are well known and established for applications in both pro- and eukaryotic cells, using enzymes such as beta-galactosidase, green fluorescent protein, and several others.<br>
The determination of activity of membrane-transport proteins can be performed according to techniques such as those described in Gennis, R.B. (1989) "Pores, Channels and Transporters", in Biomembranes, Molecular Structure and Function, Springer: Heidelberg, p. 85-137; 199-234; and 270-322.<br>
Example 9: Analysis of Impact of Mutant Protein on the Production of the Desired Product<br>
The effect of the genetic modification in G glutamicum on production of a desired compound (such as an amino acid) can be assessed by growing the modified microorganism under suitable conditions (such as those described above) and analyzing the medium and/or the cellular component for increased production of the desired product (i.e., an amino acid). Such analysis techniques are well known to one of ordinary skill in the art, and include spectroscopy, thin layer chromatography, staining methods of various kinds, enzymatic and microbiological methods, and analytical chromatography such as high performance liquid chromatography (see, for example,<br><br>
Ullman, Encyclopedia of Industrial Chemistry, vol. A2, p. 89-90 and p. 443-613, VCH: Weinheim (1985); Fallon, A. et al., (1987) "Applications of HPLC in Biochemistry" in: Laboratory Techniques in Biochemistry and Molecular Biology, vol. 17; Rehm et al. (1993) Biotechnology, vol. 3, Chapter III: "Product recovery and purification", page 469-714, VCH: Weinheim; Belter, P.A. et al (1988) Bioseparations: downstream processing for biotechnology, John Wiley and Sons; Kennedy, J.F. and Cabral, J.M.S. (1992) Recovery processes for biological materials, John Wiley and Sons: Shaeiwitz. J.A. and Henry. J.D. (1988) Biochemical separations, in: Ulmamvs Encyclopedia of Industrial Chemistry, vol. B3, Chapter 11, page 1-27. VCH: Weinheim; and Dechow. F.J. (1989) Separation and purification techniques in biotechnology, Noyes Publications.)<br>
In addition to the measurement of the final product of fermentation, it is also possible to analyze other components of the metabolic pathways utilized for the production of the desired compound, such as intermediates and side-products, to determine the overall yield, production, and/or efficiency of production of the compound. Analysis methods include measurements of nutrient levels in the medium (e.g., sugars, hydrocarbons, nitrogen sources, phosphate, and other ions), measurements of biomass composition and growth, analysis of the production of common metabolites of biosynthetic pathways, and measurement of gasses produced during fermentation. Standard methods for these measurements are outlined in Applied Microbial Physiology, A Practical Approach, P.M. Rhodes and P.F. Stanbury, eds., IRL Press, p. 103-129; 131-163; and 165-192 (ISBN: 0199635773) and references cited therein.<br>
Example 10: Purification of the Desired Product from C glutamicum Culture<br>
Recovery of the desired product from the C. glutamicum cells or supernatant of the above-described culture can be performed by various methods well known in the art. If the desired product is not secreted from the cells, the cells can be harvested from the culture by low-speed centrifugation, the cells can be lysed by standard techniques, such as mechanical force or sonication. The cellular debris is removed by centrifugation, and the supernatant fraction containing the soluble proteins is retained for further purification of the desired compound. If the product is secreted from the C glutamicum<br><br>
cells, then the cells are removed from the culture by low-speed centrifugation, and the supemate fraction is retained for further purification.<br>
The supernatant fraction from either purification method is subjected to chromatography with a suitable resin, in which the desired molecule is either retained on a chromatography resin while many of the impurities in the sample are not, or where the impurities are retained by the resin while the sample is not. Such chromatography steps may be repeated as necessary, using the same or different chromatography resins. One of ordinary skill in the art would be well-versed in the selection of appropriate chromatography resins and in their most efficacious application for a particular molecule to be purified. The purified product may be concentrated by filtration or ultrafiltration, and stored at a temperature at which the stability of the product is maximized.<br>
There are a wide array of purification methods known to the art and the preceding method of purification is not meant to be limiting. Such purification techniques are described, for example, in Bailey, J.E. &amp; Ollis, D.F. Biochemical Engineering Fundamentals, McGraw-Hill: New York (1986).<br>
The identity and purity of the isolated compounds may be assessed by techniques standard in the art. These include high-performance liquid chromatography (HPLC). spectroscopic methods, staining methods, thin layer chromatography, NIRS, enzymatic assay, or microbiologically. Such analysis methods are reviewed in: Patek et al (1994) Appl Environ. Microbiol, 60: 133-140; Malakhova et al (1996) Biotekhnologiya 11: 27-32; and Schmidt et al (1998) Bioprocess Engineer. 19: 67-70. Ulmann's Encyclopedia of Industrial Chemistry, (1996) vol. A27, VCH: Weinheim, p. 89-90, p. 521-540, p. 540-547, p. 559-566, 575-581 and p. 581-587; Michal, G. (1999) Biochemical Pathways: An Atlas of Biochemistry and Molecular Biology, John Wiley and Sons; Fallon, A. et al. (1987) Applications of HPLC in Biochemistry in: Laboratory Techniques in Biochemistry and Molecular Biology, vol. 17.<br>
EXAMPLE 11:	Cloning of a Corynebacterium glutamicum Gene Involved in<br>
Lincomycin Resistance Using a Reporter Gene Approach<br>
A. Identification of the Gene Encoding the LMRB Protein<br><br>
Plasmid pSL130 was constructed by ligation of the aceB promoter region (paceB) of C glutamicum (Kim, H.J. et al (1997)7. Microbiol. Biotechnol 7: 287-292) into the polylinker of the lac operon fusion vector pRS415, which lacks a promoter (Simon, R.W. et al. (1987) Gene 53: 85-96). Plasmid pSL145 was constructed by ligating the resulting paceB-lac region into the E. coli cloning vector pACYCl 84. E. coli DH5aF' was transformed with pSL145 and the resulting strain was used as a host for screening of a genomic C. glutamicum library (in pSL109).<br>
Transformants were screened by growth on agar medium containing 5-bromo-4-chloro-3-indolyl-beta-D-glalactopyranoside (X-Gal). A white colony, containing DNA influencing lacZ expression, was selected for further analysis. This clone was found to contain a 4 kB fragment from the gene library. Subclones were constructed in pSL109 and a subclone which retained the white phenotype on X-Gal plates was identified. This subclone was found to contain a 2.6 kB BamHl-XhoI fragment (plasmid pSL 149-5). The fragment was sequenced and identified as a membrane protein-encoding gene (LMRB gene).<br>
The 1442 nucleotides of the coding sequence of the LMRB gene encode a polypeptide of 481 amino acid residues with a high percentage of hydrophobic amino acids. A Genbank search determined that the LMRB protein is 40% identical to the protein product of the lmrB gene from Bacillus subtilis (Genbank Accession AL009126, TREMBL Accession P94422), as determined using a CLUSTAL W analysis (using standard parameters).<br>
The LMRN protein contains a sequence pattern: 158-A-P-A-L-G-P-T-L-S-G-167 (SEQ ID NO:301), which resembles the known multi-drug-resistance-protein consensus motif G-X-X-X-G-P-X-X-G-G (SEQ ID NO:302) (Paulsen, I.T., and Skurray, R.A. (1993) Gene 124: 1-11). Therefore, the LMRB protein was classified as a drug resistance protein.<br>
B. In vivo Analysis oflmrB Function<br>
The lmrB gene was overexpressed in C. glutamicum AS019E12 (Kim, H.J. et al. (1997) J. Microbiol. Biotechnol. 7: 287-292) using the plasmid pSL149-5, described above.<br><br>
Disruption of the LMRB gene was accomplished by use of the vector pSL18-ImrB. This vector was constructed as follows: an internal fragment of the LMRB gene was amplified by PCR under standard conditions using primers 5'-CTCCAGGATTGCTCCGAAGG-3' (SEQ ID NO:303) and 5'-CACAGTGGTTGACCACTGGC-3' (SEQ ID NO:304). The resulting PCR product was treated with T7 DNA polymerase and T7 polynucleotide kinase, and was cloned into the Smal site of plasmid pSL18 (Kim, Y.H. and H.-S. Lee (1996) J. Microbiol. Biotechnol. 6: 315-320). The disruption of the LMRB gene in C. glulamicum AS019E12 was performed by conjugation, as previously described (Schwarzer and Puhler (1991) Bio/Technology 9:84-87).<br>
C. glulamicum cells transformed with pSL 149-5 displayed similar resistances as untransformed cells against erythromycin, penicillin G, tetracycline, chloramphenicol, spectinomycin, nalidixic acid, gentamycin, streptomycin, ethidium bromide, carbonyl cyanide m-chlorophenylhydrazone (CCCP), and sodium dodecyl sulfate. Significant differences were observed, however, in the resistance of transformed and untransformed cells to lincomycin.<br>
LMRB-overexpressing C. glulamicum cells were found to be able to grow in the presence of 20 ng/ml lincomycin. In contrast, cells which do not overexpress LMRB (or cells carrying a LMRB disruption) were not able to grow on agar media containing 5 ^g/ml lincomycin. This effect was clearly visible in liquid culture. LMRB overexpression led to a 9-fold increased resistance (compared to wild-type) against lincomycin and LMRB disruption resulted in a decreased resistance (28% of wild-type) to this antibiotic.<br>
Example 12: Analysis of the Gene Sequences of the Invention<br>
The comparison of sequences and determination of percent homology between two sequences are art-known techniques, and can be accomplished using a mathematical algorithm, such as the algorithm of Karlin and Altschul (1990) Proc. Natl Acad. Sci. USA 87:2264-68, modified as in Karlin and Altschul (1993) Proc. Natl. Acad. Sci. USA 90:5873-77. Such an algorithm is incorporated into the NBLAST and XBLAST programs (version 2.0) of Altschul, et al. (1990)7. Mol. Biol 215:403-10. BLAST nucleotide searches can be performed with the NBLAST program, score = 100,<br><br>
wordlength = 12 to obtain nucleotide sequences homologous to SRT nucleic acid molecules of the invention. BLAST protein searches can be performed with the XBLAST program, score = 50, wordlength = 3 to obtain amino acid sequences homologous to SRT protein molecules of the invention. To obtain gapped alignments for comparison purposes, Gapped BLAST can be utilized as described in Altschul et al, (1997) Nucleic Acids Res. 25(17):3389-3402. When utilizing BLAST and Gapped BLAST programs, one of ordinary skill in the art will know how to optimize the parameters of the program (e.g.. XBLAST and NBLAST) for the specific sequence being analyzed.<br>
Another example of a mathematical algorithm utilized for the comparison of sequences is the algorithm of Meyers and Miller ((1988) Comput. Appl. Biosci. 4:11-17). Such an algorithm is incorporated into the ALIGN program (version 2.0) which is part of the GCG sequence alignment software package. When utilizing the ALIGN program for comparing amino acid sequences, a PAM120 weight residue table, a gap length penalty of 12, and a gap penalty of 4 can be used. Additional algorithms for sequence analysis are known in the art, and include ADVANCE and ADAM, described in Torelli and Robotti (1994) Comput. Appl. Biosci. 10:3-5; and FASTA. described in Pearson and Lipman (1988)/&gt;. WAS. 85:2444-8.<br>
The percent homology between two amino acid sequences can also be accomplished using the GAP program in the GCG software package (available at http://www.gcg.com), using either a Blosum 62 matrix or a PAM250 matrix, and a gap weight of 12, 10, 8, 6, or 4 and a length weight of 2, 3, or 4. The percent homology between two nucleic acid sequences can be accomplished using the GAP program in the GCG software package, using standard parameters, such as a gap weight of 50 and a length weight of 3.<br>
A comparative analysis of the gene sequences of the invention with those present in Genbank has been performed using techniques known in the art (see, e.g., Bexevanis and Ouellette, eds. (1998) Bioinformatics: A Practical Guide to the Analysis of Genes and Proteins. John Wiley and Sons: New York). The gene sequences of the invention were compared to genes present in Genbank in a three-step process. In a first step, a BLASTN analysis (e.g., a local alignment analysis) was performed for each of the sequences of the invention against the nucleotide sequences present in Genbank, and the<br><br>
top 500 hits were retained for further analysis. A subsequent FASTA search (e.g., a combined local and global alignment analysis, in which limited regions of the sequences are aligned) was performed on these 500 hits. Each gene sequence of the invention was subsequently globally aligned to each of the top three FASTA hits, using the GAP program in the GCG software package (using standard parameters). In order to obtain correct results, the length of the sequences extracted from Genbank were adjusted to the length of the query sequences by methods well-known in the art. The results of this analysis are set forth in Table 4. The resulting data is identical to that which would have been obtained had a GAP (global) analysis alone been performed on each of the genes of the invention in comparison with each of the references in Genbank, but required significantly reduced computational time as compared to such a database-wide GAP (global) analysis. Sequences of the invention for which no alignments above the cutoff values were obtained are indicated on Table 4 by the absence of alignment information. It will further be understood by one of ordinary skill in the art that the GAP alignment homology percentages set forth in Table 4 under the heading "% homology (GAP)" are listed in the European numerical format, wherein a Y represents a decimal point. For example, a value of "40,345" in this column represents "40.345%".<br>
Example 13:  Construction and Operation of DNA Microarrays<br>
The sequences of the invention may additionally be used in the construction and application of DNA microarrays (the design, methodology, and uses of DNA arrays are well known in the art, and are described, for example, in Schena, M. et ah (1995) Science 270: 467-470; Wodicka, L. et ah (1997) Nature Biotechnology 15: 1359-1367; DeSaizieu, A. et ah (1998) Nature Biotechnology 16: 45-48; and DeRisi, J.L. et ah (1997) Science 278: 680-686).<br>
DNA microarrays are solid or flexible supports consisting of nitrocellulose, nylon, glass, silicone, or other materials. Nucleic acid molecules may be attached to the surface in an ordered manner. After appropriate labeling, other nucleic acids or nucleic acid mixtures can be hybridized to the immobilized nucleic acid molecules, and the label may be used to monitor and measure the individual signal intensities of the hybridized molecules at defined regions. This methodology allows the simultaneous quantification of the relative or absolute amount of all or selected nucleic acids in the applied nucleic<br><br>
acid sample or mixture. DNA microarrays, therefore, permit an analysis of the expression of multiple (as many as 6800 or more) nucleic acids in parallel (see, e.g., Schena, M. (1996) BioEssays 18(5): 427-431).<br>
The sequences of the invention may be used to design oligonucleotide primers which are able to amplify defined regions of one or more C. glutamicum genes by a nucleic acid amplification reaction such as the polymerase chain reaction. The choice and design of the 5' or 3' oligonucleotide primers or of appropriate linkers allows the covalent attachment of the resulting PCR products to the surface of a support medium described above (and also described, for example, Schena, M. et al (1995) Science 270: 467-470).<br>
Nucleic acid microarrays may also be constructed by in situ oligonucleotide synthesis as described by Wodicka, L. et al. (1997) Nature Biotechnology 15: 1359-1367. By photolithographic methods, precisely defined regions of the matrix are exposed to light. Protective groups which are photolabile are thereby activated and undergo nucleotide addition, whereas regions that are masked from light do not undergo any modification. Subsequent cycles of protection and light activation permit the synthesis of different oligonucleotides at defined positions. Small, defined regions of the genes of the invention may be synthesized on microarrays by solid phase oligonucleotide synthesis.<br>
The nucleic acid molecules of the invention present in a sample or mixture of nucleotides may be hybridized to the microarrays. These nucleic acid molecules can be labeled according to standard methods. In brief, nucleic acid molecules (e.g., mRNA molecules or DNA molecules) are labeled by the incorporation of isotopically or fluorescently labeled nucleotides, e.g., during reverse transcription or DNA synthesis. Hybridization of labeled nucleic acids to microarrays is described (e.g., in Schena, M. et al. (1995) supra; Wodicka, L. et al (1997), supra; and DeSaizieu A. et al (1998), supra). The detection and quantification of the hybridized molecule are tailored to the specific incorporated label. Radioactive labels can be detected, for example, as described in Schena, M. et al (1995) supra) and fluorescent labels may be detected, for example, by the method of Shalon et al (1996) Genome Research 6: 639-645).<br>
The application of the sequences of the invention to DNA microarray technology, as described above, permits comparative analyses of different strains of C<br><br>
glutamicum or other Corynebacteria. For example, studies of inter-strain variations based on individual transcript profiles and the identification of genes that are important for specific and/or desired strain properties such as pathogenicity, productivity and stress tolerance are facilitated by nucleic acid array methodologies. Also, comparisons of the profile of expression of genes of the invention during the course of a fermentation reaction are possible using nucleic acid array technology.<br>
Example 14:  Analysis of the Dynamics of Cellular Protein Populations (Proteomics)<br>
The genes, compositions, and methods of the invention may be applied to study the interactions and dynamics of populations of proteins, termed 'proteomics'. Protein populations of interest include, but are not limited to, the total protein population of C glutamicum (e.g., in comparison with the protein populations of other organisms), those proteins which are active under specific environmental or metabolic conditions {e.g.. during fermentation, at high or low temperature, or at high or low pH), or those proteins which are active during specific phases of growth and development.<br>
Protein populations can be analyzed by various well-known techniques, such as gel electrophoresis. Cellular proteins may be obtained, for example, by lysis or extraction, and may be separated from one another using a variety of electrophoretic techniques. Sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE) separates proteins largely on the basis of their molecular weight. Isoelectric focusing polyacrylamide gel electrophoresis (IEF-PAGE) separates proteins by their isoelectric point (which reflects not only the amino acid sequence but also posttranslational modifications of the protein). Another, more preferred method of protein analysis is the consecutive combination of both IEF-PAGE and SDS-PAGE, known as 2-D-gel electrophoresis (described, for example, in Hermann et al (1998) Electrophoresis 19: 3217-3221; Fountoulakis et al (1998) Electrophoresis 19: 1193-1202; Langen et al. (1997) Electrophoresis 18: 1184-1192; Antelmann et al (1997) Electrophoresis 18: 1451-1463). Other separation techniques may also be utilized for protein separation, such as capillary gel electrophoresis; such techniques are well known in the art.<br>
Proteins separated by these methodologies can be visualized by standard techniques, such as by staining or labeling. Suitable stains are known in the art, and<br><br>
include Coomassie Brilliant Blue, silver stain, or fluorescent dyes such as Sypro Ruby (Molecular Probes). The inclusion of radioactively labeled amino acids or other protein precursors (e.g., 35S-methionine, 35S-cysteine, 14C-labelled amino acids, 15N -amino acids, I5NC&gt;3 or I5NH4+ or I3C-labelled amino acids) in the medium of C. glutamicum permits the labeling of proteins from these cells prior to their separation. Similarly, fluorescent labels may be employed. These labeled proteins can be extracted, isolated and separated according to the previously described techniques.<br>
Proteins visualized by these techniques can be further analyzed by measuring the amount of dye or label used. The amount of a given protein can be determined quantitatively using, for example, optical methods and can be compared to the amount of other proteins in the same gel or in other gels. Comparisons of proteins on gels can be made, for example, by optical comparison, by spectroscopy, by image scanning and analysis of gels, or through the use of photographic films and screens. Such techniques are well-known in the art.<br>
To determine the identity of any given protein, direct sequencing or other standard techniques may be employed. For example, N- and/or C-terminal amino acid sequencing (such as Edman degradation) may be used, as may mass spectrometry (in particular MALDI or ESI techniques (see, e.g., Langen et ai (1997) Electrophoresis 18: 1184-1192)). The protein sequences provided herein can be used for the identification of C. glutamicum proteins by these techniques.<br>
The information obtained by these methods can be used to compare patterns of protein presence, activity, or modification between different samples from various biological conditions (e.g., different organisms, time points of fermentation, media conditions, or different biotopes, among others). Data obtained from such experiments alone, or in combination with other techniques, can be used for various applications, such as to compare the behavior of various organisms in a given (e.g., metabolic) situation, to increase the productivity of strains which produce fine chemicals or to increase the efficiency of the production of fine chemicals.<br><br>
Equivalents<br>
Those of ordinary skill in the art will recognize, or will be able to ascertain using no more than routine experimentation, many equivalents to the specific embodiments of the invention described herein. Such equivalents are intended to be encompassed by the following claims.<br><br><br><br>
What is claimed:<br>
1.	An isolated nucleic acid molecule from Corynebacterium glutamicum encoding a stress, resistance, or tolerance gene, or a portion thereof, provided that the nucleic acid molecule does not consist of any of the F-designated genes set forth in Table 1.<br>
2.	The isolated nucleic acid molecule of claim 1, wherein said stress, resistance, or tolerance gene is selected from the group consisting of nucleic acid molecules involved in a stress response, tolerance, or resistance to temperature stresses, pH stresses, oxygen stresses, osmotic stresses, toxic chemicals, oxygen radicals, antibiotics, or to lincomycin.<br>
3.	An isolated Corynebacterium glutamicum nucleic acid molecule selected from the group consisting of those sequences set forth as odd-numbered SEQ ID NOs of the Sequence Listing, or a portion thereof provided that the nucleic acid molecule does not consist of any of the F-designated genes set forth in Table 1.<br>
4.	An isolated nucleic acid molecule which encodes a polypeptide sequence selected from the group consisting of those sequences set forth as even-numbered SEQ ID NOs of the Sequence Listing, provided that the nucleic acid molecule does not consist of any of the F-designated genes set forth in Table 1.<br>
5.	An isolated nucleic acid molecule which encodes a naturally occurring allelic variant of a polypeptide selected from the group of amino acid sequences consisting of those sequences set forth as even-numbered SEQ ID NOs of the Sequence Listing, provided that the nucleic acid molecule does not consist of any of the F-designated genes set forth in Table 1.<br>
6.	An isolated nucleic acid molecule comprising a nucleotide sequence which is at least 50% homologous to a nucleotide sequence selected from the group consisting of those sequences set forth as odd-numbered SEQ ID NOs of the Sequence Listing, or<br><br>
a portion thereof, provided that the nucleic acid molecule does not consist of any of the F-designated genes set forth in Table 1.<br>
7.	An isolated nucleic acid molecule comprising a fragment of at least 15 nucleotides of a nucleic acid comprising a nucleotide sequence selected from the group consisting of those sequences set forth as odd-numbered SEQ ID NOs of the Sequence Listing, provided that the nucleic acid molecule does not consist of any of the F-designated genes set forth in Table 1.<br>
8.	An isolated nucleic acid molecule which hybridizes to the nucleic acid molecule of any one of claims 1-7 under stringent conditions.<br>
9.	An isolated nucleic acid molecule comprising the nucleic acid molecule of any one of claims 1-8 or a portion thereof and a nucleotide sequence encoding a heterologous polypeptide.<br>
10.	A vector comprising the nucleic acid molecule of any one of claims 1-9.<br>
11.	The vector of claim 10, which is an expression vector.<br>
12.	A host cell transfected with the expression vector of claim 11.<br>
13.	The host cell of claim 12, wherein said cell is a microorganism.<br>
14.	The host cell of claim 13, wherein said cell belongs to the genus Corynebacterium or Brevibacterium.<br>
15.	The host cell of claim 12, wherein the expression of said nucleic acid molecule results in the modulation in production of a fine chemical from said cell.<br>
16.	The host cell of claim 15, wherein said fine chemical is selected from the group consisting of: organic acids, proteinogenic and nonproteinogenic amino acids, purine<br><br>
and pyrimidine bases, nucleosides, nucleotides, lipids, saturated and unsaturated fatty acids, diols, carbohydrates, aromatic compounds, vitamins, cofactors, polyketides, and enzymes.<br>
17.	A method of producing a polypeptide comprising culturing the host cell of claim 12 in an appropriate culture medium to, thereby, produce the polypeptide.<br>
18.	An isolated stress, resistance, or tolerance polypeptide from Corynebacterium glatamicum* or a portion thereof.<br>
19.	The protein of claim 18, wherein said stress, resistance, or tolerance polypeptide is selected from the group consisting of proteins involved in a stress response, tolerance, or resistance to temperature stresses, pH stresses, oxygen stresses, osmotic stresses, toxic chemicals, oxygen radicals, antibiotics, or to lincomycin.<br>
20.	An isolated polypeptide comprising an amino acid sequence selected from the group consisting of those sequences set forth as even-numbered SEQ ID NOs of the Sequence Listing, provided that the amino acid sequence is not encoded by any of the F-designated genes set forth in Table 1.<br>
21.	An isolated polypeptide comprising a naturally occurring allelic variant of a polypeptide comprising an amino acid sequence selected from the group consisting of those sequences set forth as even-numbered SEQ ID NOs of the Sequence Listing, or a portion thereof, provided that the amino acid sequence is not encoded by any of the F-designated genes set forth in Table 1.<br>
22.	The isolated polypeptide of any of claims 18-21, further comprising heterologous amino acid sequences.<br>
23.	An isolated polypeptide which is encoded by a nucleic acid molecule comprising a nucleotide sequence which is at least 50% homologous to a nucleic acid selected from the group consisting of those sequences set forth as odd-numbered SEQ ID<br><br>
NOs of the Sequence Listing,, provided that the nucleic acid molecule does not consist of any of the F-designated nucleic acid molecules set forth in Table 1.<br>
24.	An isolated polypeptide comprising an amino acid sequence which is at least 50% homologous to an amino acid sequence selected from the group consisting of those sequences set forth as even-numbered SEQ ID NOs of the Sequence Listing, provided that the amino acid sequence is not encoded by any of the F-designated genes set forth in Table 1.<br>
25.	A method for producing a fine chemical, comprising culturing a cell containing a vector of claim 12 such that the fine chemical is produced.<br>
26.	The method of claim 25, wherein said method further comprises the step of recovering the fine chemical from said culture.<br>
27.	The method of claim 25, wherein said method further comprises the step of transfecting said cell with the vector of claim 11 to result in a cell containing said vector.<br>
28.	The method of claim 25, wherein said cell belongs to the genus Corynebacterium or Brevibacterium.<br>
29.	The method of claim 25, wherein said cell is selected from the group consisting of: Corynebacterium glutamicum, Corynebacterium herculis, Corynebacterium, lilium, Corynebacterium acetoacidophilum, Corynebacterium acetoglutamicum, Corynebacterium acetophilum, Corynebacterium ammoniagenes, Corynebacterium fujiokense, Corynebacterium nitrilophilus, Brevibacterium ammoniagenes, Brevibacterium butanicum, Brevibacterium divaricatum, Brevibacterium flavum, Brevibacterium healii, Brevibacterium ketoglutamicum, Brevibacterium ketosoreductum, Brevibacterium lactofermentum, Brevibacterium linens, Brevibacterium paraffinolyticum, and those strains set forth in Table 3.<br><br>
30.	The method of claim 25, wherein expression of the nucleic acid molecule from said vector results in modulation of production of said fine chemical.<br>
31.	The method of claim 25, wherein said fine chemical is selected from the group consisting of: organic acids, proteinogenic and nonproteinogenic amino acids, purine and pyrimidine bases, nucleosides, nucleotides, lipids, saturated and unsaturated fatty acids, diols, carbohydrates, aromatic compounds, vitamins, cofactors. polyketides and enzymes.<br>
32.	The method of claim 25, wherein said fine chemical is an amino acid.<br>
33.	The method of claim 32, wherein said amino acid is drawn from the group consisting of: lysine, glutamate, glutamine, alanine, aspartate, glycine, serine, threonine, methionine, cysteine, valine, leucine, isoleucine, arginine, proline, histidine. tyrosine, phenylalanine, and tryptophan.<br>
34.	A method for producing a fine chemical, comprising culturing a cell whose genomic DNA has been altered by the inclusion of a nucleic acid molecule of any one of claims 1-9.<br><br>
35.	A method for diagnosing the presence or activity of Corynebacterium diphtheriae in a subject, comprising detecting the presence of one or more SEQ ID NOs 1 through 304 of the Sequence Listing in the subject, provided that the sequences are not or are not encoded by any of the F-designated sequences set forth in Table 1, thereby diagnosing the presence or activity of Corynebacterium diphtheriae in the subject.<br>
36.	A host cell comprising a nucleic acid molecule selected from the group consisting of the nucleic acid molecules set forth as odd-numbered SEQ ID NOs of the Sequence Listing, wherein the nucleic acid molecule is disrupted.<br>
37.	A host cell comprising a nucleic acid molecule selected from the group consisting of the nucleic acid molecules set forth as odd-numbered SEQ ID NOs of the<br><br>
Sequence Listing , wherein the nucleic acid molecule comprises one or more nucleic acid modifications from the sequence set forth as odd-numbered SEQ ID NOs of the Sequence Listing .<br>
38. A host cell comprising a nucleic acid molecule selected from the group consisting of the nucleic acid molecules set as odd-numbered SEQ ID NOs of the Sequence Listing, wherein the regulatory region of the nucleic acid molecule is modified relative to the wild-type regulatory region of the molecule.<br><br>
39.	An isolated nucleic acid molecule substantially as herein described<br>
and exemplified.<br>
40.	An   isolated   polypeptide   substantially   as   herein   described   and<br>
exemplified.<br>
41.	A method of producing a polypeptide substantially as herein described<br>
and exemplified.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDEyNS1jaGUgYWJzdHJhY3QtZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0125-che abstract-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDEyNS1jaGUgY2xhaW1zLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-0125-che claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDEyNS1jaGUgZGVzY3JpcHRpb24gKGNvbXBsZXRlKS1kdXBsaWNhdGUtMS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-0125-che description (complete)-duplicate-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDEyNS1jaGUgZGVzY3JpcHRpb24gKGNvbXBsZXRlKS1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-0125-che description (complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTI1LWNoZS1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-125-che-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTI1LWNoZS1jbGFpbXMgLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-125-che-claims .pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTI1LWNoZS1jb3JyZXNwb25kYW5jZSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-125-che-correspondance others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTI1LWNoZS1jb3JyZXNwb25kYW5jZSBwby5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-125-che-correspondance po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTI1LWNoZS1kZXNjcmlwdGlvbiBjb21wbGV0ZSAucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-125-che-description complete .pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTI1LWNoZS1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-125-che-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTI1LWNoZS1mb3JtIDE5LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-125-che-form 19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTI1LWNoZS1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-125-che-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTI1LWNoZS1wY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-125-che-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="228881-a-method-for-quantifying-the-teichoic-acids-in-a-preparation-comprising-one-or-more-capsular-polysaccharides-of-streptococcus-pneumoniae.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="228883-a-method-for-suppressing-silence-frames-in-a-stream-of-media.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>228882</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2002/125/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>12/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>20-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>11-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>23-Jan-2002</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>BASF AKTIENGESELLSCHAFT</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>67056 Ludwigshafen,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>POMPEJUS, Markus</td>
											<td>Wenjenstrasse 21, D-67251 Freinsheim,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SCHRODER, Hartwig</td>
											<td>Goethestrasse 5, D-69226 Nussloch,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>ZELDER, Oskar</td>
											<td>Rossmarktstrasse 27, D-67346 Speyer,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>LEE, Heung-Shick</td>
											<td>Korea University, Graduate School of Biotechnology Anam Dong, Sungbook-Gu, Seoul 136-701,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>KIM, Hyung-Joon</td>
											<td>Korea University, Graduate School of Biotechnology Anam Dong, Sungbook-Gu, Seoul 136-701,</td>
										</tr>
										<tr>
											<td>6</td>
											<td>KROGER, Burkhard</td>
											<td>Im Waldhof, 1, D-67117 Limburgerhof,</td>
										</tr>
										<tr>
											<td>7</td>
											<td>HABERHAUER, Gregor</td>
											<td>Moselstrasse 42, D-67117 Limburgerhof,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K14/34</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IB00/00922</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-06-23</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>199 31 457.8</td>
									<td>1999-07-08</td>
								    <td>Germany</td>
								</tr>
								<tr>
									<td>2</td>
									<td>199 31 541.8</td>
									<td>1999-07-08</td>
								    <td>Germany</td>
								</tr>
								<tr>
									<td>3</td>
									<td>199 32 230.9</td>
									<td>1999-07-09</td>
								    <td>Germany</td>
								</tr>
								<tr>
									<td>4</td>
									<td>60/141,031</td>
									<td>1999-06-25</td>
								    <td>Germany</td>
								</tr>
								<tr>
									<td>5</td>
									<td>60/151,214</td>
									<td>1999-08-27</td>
								    <td>Germany</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/228882-corynebacterium-glutamicum-genes-encoding-stress-resistance-and-tolerance-proteins by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:31:03 GMT -->
</html>
