<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/228872-phospholipase-and-method-of-producing-it by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:30:22 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 228872:PHOSPHOLIPASE AND METHOD OF PRODUCING IT</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">PHOSPHOLIPASE AND METHOD OF PRODUCING IT</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention is related to a method for producing a phospholipase by processing an expressed fungal peptide and to certain specified phospholipases. Furthermore the invention provides a method for producing cheese with a phospholipase.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
PHOSPHOLIPASE AND METHOD OF PRODUCING IT<br>
FIELD OF THE INVENTION<br>
The present invention relates to a method of hydrolyzing a phospholipid, a method of producing a phospholipase, a method of making cheese, and to a phospholipase.<br>
BACKGROUND OF THE INVENTION<br>
Soragni, E., et al. (2001) EMBO J. 20: 5079-5090 discloses a phospholipase (TbSP1) from Tuber borchii and the nucleotide sequence of a cDNA of a gene encoding it. The following peptide sequence are published in the indicated sources, derived from the indicated source organism:<br>
•	COGEME Phytopathogenic Fungi and Oomycete EST Database, Unisequence ID: VD0100C34, Verticillium dahliae.<br>
•	NCBI Protein database, gi: 18307435, Neurospora crassa<br>
•	NCBI Protein database, gi:16519372, Helicosporum sp. HN1<br>
•	WO 0056762, SEQ ID NO: 5954, Aspergillus oryzae<br>
•	TREMBL Protein database, EAA28927, Neurospora crassa<br>
US 6399121 discloses the use of phospholipase in cheese making.<br>
SUMMARY OF THE INVENTION<br>
The inventors have analyzed known sequence data for fungal Group XIII phospholipases A2, and they have identified additional sequences, either from published sequence data or by screening for relevant sequences from natural sources. By expressing genes encoding fungal Group XIII phospholipases A2 in a suitable host organism they found that the expressed sequences consist of a core peptide coupled to a peptide sequence at the N- or C-terminal side, or both, and that expression of the gene in a suitable host organism can lead to cleavage of the expressed peptide to obtain the core peptide without any peptide extension at the N- or C-terminal. They further found that the core peptide without any peptide extensions) has a significantly higher phospholipase activity than the core peptide linked to the peptide extension(s). Finally, they found that the core peptide discovered by this method is similar in length and sequence to a known mature peptide from Helicosporium sp. (Waka-tsuki, S. et al. (2001) Biochim. Biophys. Acta 1522: 74-81) of unknown function, and to bacterial Group XIII phospholipases A2, which lack peptide extensions other than secretion signals (Sugiyama, M. et al. (2002) J. Biol. Chem. 277:20051-20058).<br>
The inventors additionally found that phospholipase sharing the active site sequence similarity and cysteine residue conservation of fungal Group XIII phospholipase A2 is useful in<br><br>
cheese making.<br>
Additionally, the inventors discovered and isolated a gene encoding a novel phospholipase from Fusarium venenatum A3/5, which was originally deposited as Fusarium graminearum ATCC 20334 and recently reclassified as Fusarium venenatum by Yoder and Christianson, 1998, Fungal Genetics and Biology 23: 62-80; and O'Donnell et al., 1998, Fungal Genetics and Biology 23: 57-67. The phospholipase belongs to the fungal/bacterial group XIII PLA2 as defined by Soragni et al., The EMBO Journal, 20 (2001), 5079-5090. The inventors also cloned the novel phospholipase encoding gene into an £ coli strain, and used the cloned gene to make a construct for expressing the Fusarium phospholipase gene in Aspergillus oryzae. The inventors transformed Aspergillus oryzae with this construct, and isolated the phospholipase from transformed Aspergillus cells.<br>
Accordingly, the invention provides a method of producing a phospholipase which comprises processing an expressed fungal peptide so as to cleave off a peptide from the C-terminal end and/or a peptide from the N-terminal end to obtain a core peptide, wherein the core peptide comprises:<br>
a)	the amino acid sequence given by amino acids 146-153 of SEQ ID NO: 1, amino acids 87-94 of SEQ ID NO: 3, or amino acids 79-86 of SEQ ID NO: 12; or a sequence identical to any of these amino acid sequences except for the substitution of a single amino acid with another amino acid; and<br>
b)	at least two cysteine residues located on the N-terminal side of the sequence given in a); and<br>
c)	at least two cysteine residues located on the C-terminal side of the sequence given in a).<br>
The invention also provides a method for hydrolyzing a phospholipid with a phospholipase of the invention. Furthermore the invention provides a method for producing cheese by contacting cheese milk or a fraction of cheese milk with a phospholipase and producing cheese from the cheese milk.<br>
Finally, the invention provides phospholipase which is a polypeptide having an amino acid sequence which is at least 80 % identical with certain specified sequences.<br>
BRIEF DESCRIPTION OF DRAWINGS<br>
Fig. 1 shows an alignment of amino acid sequences of fungal group XIII phospholi-pases A2, showing processing sites (|) where known. The active site consensus is underlined. Conserved cysteine residues are indicated with | under the consensus. Alignment was made with the AlignX program of the Vector NTI program suite v8. The algorithm used is ClustalW with the blosum62mt2 matrix and AlignX default settings.<br><br>
DETAILED DESCRIPTION OF THE INVENTION<br>
Expressed peptide<br>
The invention uses an expressed fungal peptide belonging to a group defined by the active site sequence similarity and cysteine residue conservation used in the definition of the group "fungal/bacterial group XIII phospholipase A2" given by Soragni, E., et al. (2001) EMBO J. 20: 5079-5090. The peptide is fungal, e.g. derived from Tuber, Verticillium, Neurospora, Helicosporum, or Aspergillus, particularly T. borchii, T. albidum, V dahliae, V. tenerum, N. crassa, Helicosporium sp.HNI or A oryzae.<br>
The peptide may have phospholipase activity, e.g. phospholipase A activity, such as phospholipase A1 and/or phospholipase A2 activity.<br>
Some particular examples are known peptides having amino acid sequences listed in the sequence listing as follows. The source organisms and literature references are also indicated:<br>
•	SEQ ID NO:1. Tuber borchiL Soragni, E., et al. (2001) EMBO J. 20: 5079-5090<br>
•	SEQ ID NO: 3. Verticillium dahliae. COGEME Phytopathogenic Fungi and Oomy-cete EST Database, Unisequence ID: VD0100C34.<br>
•	SEQ ID NO: 4. Neurospora crassa. NCBI Protein database, gi:18307435.<br>
•	SEQ ID NO: 5. Helicosporum sp. HN1. NCBI Protein database, gi:16519372.<br>
•	SEQ ID NO: 7. Aspergillus oryzae. WO 0056762, SEQ ID NO: 5954.<br>
•	SEQ ID NO 8. Neurospora crassa. TREMBL Protein database, EAA28927 Further, the following fungal phospholipases having the indicated sequences were<br>
isolated by the inventors from natural sources purchased from public collections or collected in the indicated country and year:<br>
•	SEQ ID NO: 10. Tuber albidum. Purchased from Centraalbureau voor Schimmel-cultures, Utrecht, The Netherlands, isolate CBS27272<br>
•	SEQ ID NO: 12. Verticillium tenerum. Ireland, 1996<br>
The inventors inserted the gene from T. albidum (SEQ ID NO: 9) into E. coll and deposited the clone under the terms of the Budapest Treaty on the 12 February 2003. The deposit was made at the Deutsce Sammlung von Mikroorganismen und Zellkulturen (DSMZ), Mascheroder Weg 1b, D-38124 Braunschweig, Germany, and was accorded deposit number DSM 15441.<br>
In one embodiment the invention provides a phospholipase which is a polypeptide having an amino acid sequence which is at least 80 %, such as at least 85%, preferably 90%, more preferably at least 95%, identical with amino acids 91-210 in SEQ ID NO: 10 (T. a/-bidum), amino acids 92-211 in SEQ ID NO: 1 (7. borchii), amino acids 30-137 in SEQ ID NO: 12 (V. tenerum), amino acids 38-145 in SEQ ID NO: 3 (V dahliae), amino acids 44-151 in<br><br>
SEQ'ID NO: 4 (N. crassa), amino acids 37-157 in SEQ ID NO: 7 (A oryzae), or amino acids 58-168 in SEQ ID NO: 8 (A/, crassa).<br>
Peptide processing<br>
By analyzing the phospholipase sequences in the sequence listing, the inventors found that each expressed amino acid sequence consists of a signal peptide, a core peptide, and additionally a peptide sequence with unknown function attached to the C- or N-terminal, or both, of the core peptide.<br>
Core peptide<br>
The core peptides are characterized by the same active site sequence similarity and cysteine residue conservation observed by Soragni, E., et al. (2001) EMBO J. 20: 5079-5090 for the fungal/bacterial group XIII phospholipase A2.<br>
In a preferred embodiment of the invention the core peptides comprises: a) the sequence given by amino acids 146-153 of SEQ ID NO: 1, amino acids 87-94 of SEQ ID NO: 3, or amino acids 79-86 of SEQ ID NO: 12; or a sequence identical to any of these amino acid sequences except for the substitution of a single amino acid with another amino acid; and b) two cysteine residues located on the N-terminal side of the sequence given in a); and c) two cysteine residues located on the C-terminal side of the sequence given in a).<br>
One of the cysteine residues located on the N-terminal side of the sequence given in a), may e.g. be separated from the sequence given in a) by 0-5 amino acids, such as 0-3 amino acids, preferably 0-2 amino acids, and even more preferably 1 amino acid. Another of the cysteine residues located on the N-terminal side of the sequence given in a) may e.g. be separated from the sequence given in a) by 14-20 amino acids, such as 15-19 amino acids, preferably 16-18 amino acids, and even more preferably 17 amino acids.<br>
One of the cysteine residues located on the C-terminal side of the sequence given in a), may e.g. be separated from the sequence given in a) by 22-29 amino acids, such as 23-28 amino acids, preferably 24-27 amino acids, and even more preferably 25-26 amino acids. Another of the cysteine residues located on the C-terminal side of the sequence given in a) may e.g. be separated from the sequence given in a) by 27-49 amino acids, such as 29-46 amino acids, preferably 30-43 amino acids, even more preferably 32-42 amino acids, and most preferably 35-40 amino acids.<br>
In a preferred embodiment the core peptide comprises four cysteine residues aligning with the cysteine residues of SEQ ID NO:1 with amino acid numbers 128, 144, 180, and 194, respectively, when the complete expressed phospholipase sequence is aligned simultaneously with the sequences given in SEQ ID NO: 1, SEQ ID NO: 3, SEQ ID NO: 4, SEQ ID NO: 5, SEQ ID NO: 7, SEQ ID NO: 8, SEQ ID NO: 10, and SEQ ID NO: 12.<br><br>
According to the invention, the expressed polypeptide is cleaved so as to separate :he core peptide from the attached peptide(s). The cleavage may be done in vivo by expressing it in a suitable filamentous fungal host or in vitro, e.g. by a treatment with a suitable protease such as e.g. Kex2.<br>
The cleavage points may be found within 11 amino acids of a sequence which is FG or within 10 amino acids of a sequence which is a Kex2 site. Kex2 sites are e.g. RR, KR, KK or RK. In one embodiment the core peptide has a length of 100-150 amino acids, such as 110-140 amino acids, 115-133 amino acids, 118-129 amino acids, or 118*126 amino acids.<br>
In one embodiment of the invention the expressed phospholipase is cleaved within 0-18 amino acids, such as 3-16 amino acids, preferably 5-14 amino acids on the N-terminai side of the sequence aligning with amino acids 97-101 of SEQ ID NO: 1, when the complete expressed phospholipase sequence is aligned simultaneously with the sequences given in SEQ ID NO: 1, SEQ ID NO: 3, SEQ ID NO: 4, SEQ ID NO: 5, SEQ ID NO: 7, SEQ ID NO: 8, SEQ ID NO: 10, and SEQ ID NO: 12.<br>
In a preferred embodiment the expressed phospholipase is cleaved within 0-11 amino acids, such as 0-9 amino acids, preferably 0-7 amino acids, on the C-terminal side of the sequence aligning with amino acids 204-209 of SEQ ID NO: 1, when the complete expressed phospholipase sequence is aligned simultaneously with the sequences given in SEQ ID NO: 1, SEQ ID NO: 3, SEQ ID NO: 4, SEQ ID NO: 5, SEQ ID NO: 7, SEQ ID NO: 8, SEQ ID NO: 10, and SEQ ID NO: 12.<br>
In a preferred embodiment the processed phospholipase has a specific phospholipase activity, which is higher than the activity of the expressed peptide before processing, e.g. in one embodiment the specific phospholipase activity is at least 2 times, more preferably at least 5 times, most preferably at least 10 times the specific phospholipase activity of the expressed peptide before processing. In one embodiment of the invention the expressed peptide does not have measurable phospholipase activity before processing.<br>
Phospholipase activity may e.g. be measured in the LEU assay by hydrolyzing soy lecithin (L-alfa-phosphotidyl-choline) at pH 8.0 and 40 °C for 2 minutes. Phospholipase activity is expressed as the rate of titrant consumption (0.1 M NaOH) necessary for keeping constant pH, relative to a standard.<br>
Expression in filamentous fungal host cell<br>
The filamentous fungal host cell may e.g. be a cell of Acremonium, Aspergillus, Fusarium, Humicola, Myceliophthora, Neurospora, Penicifflum, Rhizomucor, Thermomyces, Thielavia, Tolypocladium, or Trichoderma, particularly A. awamori, A. foetidus, A. japonicus, A. nidutans, A. niger, A. oryzae. F, bactridioides, F. cerealis, F. crookwellense, F. culmorum, F. graminearum, F. graminum, F. heterosporum, F. negund'u F. oxysporum, F. reticulatum, F.<br><br>
roseum, F. sambucinum, F. sarcochroum, F. sporotrichioides, F sulphureum, F. torulosum, h. trichothecioides, F. venenatum, H. insolens, M. thermophila, N. crassa, P. purpurogenum, R. miehei, Thermomyces lanuginosus, Thielavia terrestris, Trichoderma harzianum, Trichoderma koningii, Trichoderma longibrachiatum, Trichoderma reeseit or Trichoderma viride.<br>
In a preferred embodiment the host organism is a strain of Aspergillus, Fusarium, or Trichoderma, particularly A niger, A. o/yzae, F. venenatum, F. sambucinum or F. cerealis<br>
The transformation, cultivation, expression, recovery may be performed by conventional methods, e.g. by the general methods described in EP 238023. EP 305216. WO 9600787, EP 244234 or T. Christensen et al., BioTechnology, vol. 6, Dec. 1988,1419-22.<br>
Phospholipase polypeptide and DNA<br>
In one embodiment, the present invention relates to polypeptides having phospholipase activity and where the polypeptides comprises, preferably consists of, an amino acid sequence which has a degree of identity to amino acids 29 to 149 of SEQ ID NO: 16 (i.e., the mature polypeptide) of at least 80%, such as at least 85%, even more preferably at least 90%, most preferably at least 95%, e.g. at least 96%, such as at least 97%, and even most preferably at least 98%, such as at least 99%.<br>
Preferably, the polypeptides comprise the amino acid sequence of SEQ ID NO: 16; an allelic variant thereof; or a fragment thereof that has phospholipase activity. In another preferred embodiment, the polypeptide of the present invention comprises amino acids 29 to 149 of SEQ ID NO: 16. In a further preferred embodiment, the polypeptide consists of amino acids 29 to 149 of SEQ ID NO: 16.<br>
The present invention also relates to a polynucleotide comprising, preferably consisting of, a nucleotide sequence which has at least 80% identity with nucleotides 133 to 495 of SEQ ID NO: 15. Preferably, the nucleotide sequence has at least 85% identity, such as at least 90% identity, more preferably at least 95% identity, such as at least 96% identity, e.g. at least 97% identity, even more preferably at least 98% identity, such as at least 99% with nucleotides 133 to 495 of SEQ ID NO: 15. Preferably, the nucleotide sequence encodes a polypeptide having phospholipase activity.<br>
The phospholipase may be derived from a strain of Fusarium, particularly F. venenatum, using probes designed on the basis of the DNA sequences in this specification. In one embodiment the phospholipase has phospholipase A activity.<br>
The phospholipase may be produced by transforming a suitable host cell with a DNA sequence encoding the phospholipase, cultivating the transformed organism under conditions permitting the production of the enzyme, and recovering the enzyme from the culture.<br>
The host organism is preferably a eukaryotic cell, in particular a fungal cell, such as a yeast cell or a filamentous fungal cell, such as a strain of Aspergillus, Fusarium, Trichoderma<br><br>
or Saccharomyces, particularly A. niger, A. oryzae, F. venenatum, F. sambucinum, K cereans or S. cerevisiae, e.g. a glucoamylase-producing strain of A. niger such as those described in US 3677902 or a mutant thereof. The production of the phospholipase in such host organisms may be done by the general methods described in EP 238,023 (Novo Nordisk), WO 96/00787 (Novo Nordisk) or EP 244,234 (Alko).<br>
The expression vector of the invention typically includes control sequences functioning as a promoter, a translation initiation signal, and, optionally, a selectable marker, a transcription terminator, a repressor gene or various activator genes. The vector may be an autonomously replicating vector, or it may be integrated into the host cell genome.<br>
Sequence alignment and identity<br>
Nucleotide sequences may be aligned with the AlignX application of the Vector NTI Program Suite 7.0 using the default settings, which employ a modified ClustalW algorithm (Thompson, J.D., Higgins, D.G., and Gibson T.J. (1994) Nuc. Acid Res. 22: 4673-4680), the swgapdnarnt score matrix, a gap opening penalty of 15 and a gap extension penalty of 6.66.<br>
Amino acid sequences may be aligned with the AlignX application of the Vector NTI Program Suite v8 using default settings, which employ a modified ClustalW algorithm (Thompson, J.D., Higgins, D.G., and Gibson T.J., 1994), the blosum62mt2 score matrix, a gap opening penalty of 10 and a gap extension penalty of 0.1.<br>
In one embodiment of the invention alignments of sequences and calculation of ho-| mology scores are done using the Lipman-Pearson Method (Lipman, D.J. and W.R. Pearson (1985) Rapid and sensitive protein similarity searches. Science 227: 1435-1441) using a PAM250 residue weight table (Dayhoff, M.O., R.M. Schwartz, and B.C. Orcutt (1978) A model of evolutionary change in proteins. In Dayhoff, M.O. (ed.), Atlas of Protein Sequence and Structure. National Biomedical Research Foundation. Washington, D.C. Vol 5. Suppl. 3: pp. 345-358) and the default settings of the MegAlign program, v4.03, in the Lasergene software package (DNASTAR Inc., 1228 South Park Street, Madison, Wisconsin 53715). The default settings are a K-tuple of 2, gap penalty of 4, and a gap length penalty of 12.<br>
Phospholipid hydrolysis<br>
The invention may be used in the hydrolysis of any phospholipid such as a lecithin, a cephalin or an inositide.<br>
The invention may be used in analogy with prior art processes by replacing the phospholipase, e.g. in the production of baked products (WO 0032758, WO 9953769V mayonnaise (GB 1525929, US 4034124) or treatment of vegetable oil (US 5264367).<br><br>
Use of phospholipase<br>
The phospholipase of the invention can be used in various industrial application of phospholipases, e.g. as described below.<br>
Use in baking<br>
The phospholipase of the invention can be used in the preparation of dough, bread and cakes, e.g. to improve the elasticity of the bread or cake. Thus, the phospholipase can be used in a process for making bread, comprising adding the phospholipase to the ingredients of a dough, kneading the dough and baking the dough to make the bread. This can be done in analogy with US 4567056 or WO 99/53769.<br>
Use in detergent<br>
The variant may be used as a detergent additive, e.g. at a concentration (expressed as pure enzyme protein) of 0.001-10 (e.g. 0.01-1) mg per gram of detergent or 0.001-100 (e.g. 0.01-10) mg per litre of wash liquor.<br>
The detergent composition of the invention may for example be formulated as a hand or machine laundry detergent composition including a laundry additive composition suitable for pre-treatment of stained fabrics and a rinse added fabric softener composition, or be formulated as a detergent composition for use in general household hard surface cleaning operations. In a laundry detergent, the variant may be effective for the removal of fatty stains, for whiteness maintenance and for dingy cleanup. A laundry detergent composition may be formulated as described in GB 2247025, WO 9901531 or WO 9903962.<br>
The detergent composition of the invention may particularly be formulated for hand or machine dishwashing operations, e.g. as described in GB 2,247,025 (Unilever) or WO 99/01531 (Procter &amp; Gamble). In a dishwashing composition, the variant may be effective for removal of greasy/oily stains, for prevention of the staining /discoloration of the dishware and plastic components of the dishwasher by highly colored components and the avoidance of lime soap deposits on the dishware.<br>
Other uses<br>
The phospholipase of the invention can be used to improve the filterability of an aqueous solution or slurry of carbohydrate origin by treating it with the phospholipase. This is particularly applicable to a solution of slurry containing a starch hydrolyzate, especially a wheat starch hydrolyzate, since this tends to be difficult to filter and to give cloudy filtrates. The treatment can be done in analogy with EP 219,269 (CPC International).<br>
Further, the phospholipase of the invention may be used for partial hydrolysis of phospholipids, preferably lecithin, to obtain improved phospholipid emulsifiers. This application is further described in Ullmann's Encyclopedia of Industrial Chemistry (Publisher: VCH<br><br>
Weiriheim (1996)), JP patent 2794574, and JP-B 6-087751.<br>
Further, the phospholipase of the invention may be used in a process for the production of an animal feed which comprises mixing the phospholipase with feed substances and at least one phospholipid. This can be done in analogy with EP 743 017.<br>
Even further the phospholipase of the invention can be used in a process for reducing the content of phospholipid in an edible oil, comprising treating the oil with the phospholipase so as to hydrolyze a major part of the phospholipid, and separating an aqueous phase containing the hydrolyzed phospholipid from the oil. This process is applicable to the purification of any edible oil which contains phospholipid, e.g. vegetable oil such as soy bean oil, rape seed oil and sunflower oil. The phospholipase may e.g. be used in the processes described in JP-A 2-153997 and US 5264367.<br>
Method for producing cheese<br>
The phospholipase of the invention may be used for producing cheese in analogy with the process given in US 6399121.<br>
In a preferred embodiment of the invention cheese is produced by contacting cheese milk or a fraction of cheese milk with a phospholipase of the invention and producing cheese from the cheese milk.<br>
In a further preferred embodiment cheese is produced by contacting cheese milk or a fraction of cheese milk with a phospholipase, wherein the phospholipase comprises:<br>
a)	the sequence given by amino acids 146-153 of SEQ ID NO: 1, amino acids 87-94 of SEQ ID NO: 3, or amino acids 79-86 of SEQ ID NO: 12; or a sequence identical to any of these amino acid sequences except for the substitution of a single amino acid with another amino acid; and<br>
b)	two cysteine residues located on the N-terminal side of the sequence given in a); and<br>
c)	two cysteine residues located on the C-terminal side of the sequence given in a).<br>
In the present context the term cheese milk is meant to cover any milk based composition used for production of cheese. A fraction of the cheese milk may be any fraction of the cheese milk such as e.g. cream, skim milk, milk, butter milk, butter or milk fat.<br>
In a preferred embodiment cheese milk or a fraction of cheese milk is contacted with a phospholipase of the invention in an amount sufficient to decrease the oiling-off effect in cheese and/or to increase cheese yield. The oiling-off effect is the tendency of the cheese to form free oil upon storage and/or melting.<br>
In one aspect the invention relates to a process for producing cheese comprising<br><br>
treatfng a dairy composition with a phospholipase of the invention and producing cheese from the dairy composition.<br>
Another aspect of the invention relates to a process for producing cheese comprising treating a dairy composition with phospholipase and producing cheese from the dairy composition, wherein the phospholipase is selected from the group of fungal/bacterial group XIII PLA2 phospholipases. In a preferred embodiment of the invention the fungal/bacterial group XIII PLA2 is from a fungus, more preferably from a fungus belonging to the Ascomy-cetes. A phospholipase belonging to the fungal/bacterial group XIII PLA2 may be any phospholipase belonging to this group as defined by Soragni et al., The EMBO Journal, 20 (2001), 5079-5090, and may e.g. be from the species Tuber, e.g. 7. borchii, Streptomyces, e.g. S. coelicor, Verticillium, e.g. V. dahliae, Aspergillus, e.g. A. oryzaet Neurospora, e.g. A/, crassa, or Helicosporurn.<br>
A dairy composition according to the invention may be any composition comprising milk constituents. Milk constituents may be any constituent of milk such as milk fat, milk protein, casein, whey protein, and lactose. A milk fraction may be any fraction of milk such as e.g. skim milk, butter milk, whey, cream, milk powder, whole milk powder, skim milk powder. In a preferred embodiment of the invention the dairy composition comprises milk, skim milk, butter milk, whole milk, whey, cream, or any combination thereof. In a more preferred embodiment the dairy composition consists of milk, such as skim milk, whole milk, cream, buttermilk, or any combination thereof.<br>
The enzymatic treatment in the process of the invention may be conducted by dispersing the phospholipase into the dairy composition, and allowing the enzyme reaction to take place at an appropriate holding-time at an appropriate temperature. The treatment with phospholipase may be carried out at conditions chosen to suit the selected enzyme(s) according to principles well known in the art.<br>
The enzymatic treatment may be conducted at any suitable pH, such as e.g., in the range 2-10, such as, at a pH of 4-9 or 5-7. In one embodiment the phospholipase treatment is conducted at 3-60°C, such as at 25-45°C (e.g., for at least 5 minutes, such as, e.g., for at least 10 minutes or at least 30 minutes, e.g., for 5-120 minutes). The phospholipase is added in a suitable amount to produce the cheese having the desired properties. Preferably, the phospholipase is added in an amount effective to decrease the oiling-off effect in cheese and/or to increase cheese yield. A suitable dosage of phospholipase will usually be in the range 0.001-0.5 mg enzyme protein per g milk fat, preferably 0.01-0.3 mg enzyme protein per g milk fat, more preferably, 0.02-0.1 mg enzyme protein per g milk fat<br>
The cheeses produced by the process of the present invention comprise all varieties of cheese, such as, e.g. Campesino, Chester, Danbo, Drabant, Herregard, Manchego, Provo-lone, Saint Paulin, Soft cheese, Svecia, Taleggio, White cheese, including rennet-curd<br><br>
cheese produced by rennet-coagulation of the cheese curd; ripened cheeses such as Cheddar, Colby, Edam, Muenster, Gruyere, Emmenthal, Camembert, Parmesan and Romano; blue cheese, such as Danish blue cheese; fresh cheeses such as Feta; acid coagulated cheeses such as cream cheese, Neufchatel, Quarg, Cottage Cheese and Queso Blanco. In a preferred embodiment the invention relates to a process for producing pasta filata cheese, such as e.g. Mozzarella and Pizza cheese. Pasta filata, or stretched curd, cheeses are normally distinguished by a unique plasticizing and kneading treatment of the fresh curd in hot water, which imparts the finished cheese its characteristic fibrous structure and melting and stretching properties, cf. e.g. "Mozzarella and Pizza cheese" by Paul S. Kindstedt, Cheese: Chemistry, physics and microbiology, Volume 2: Major Cheese groups, second edition, page 337-341, Chapman &amp; Hall.<br>
Sequence Listing and Deposited Microorganisms<br>
The present application contains information in the form of a sequence listing, which is appended to the application and also submitted on a data carrier accompanying this application. In addition, the present application refers to deposited microorganisms. The contents of the data carrier and the deposited microorganisms are fully incorporated herein by reference.<br>
Deposit of Biological Material<br>
The following biological material has been deposited under the terms of the Budapest Treaty with the Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 1 B, D-38124 Braunschweig, Germany, and given the following accession number:<br>
Deposit	Accession Number	Date of Deposit<br>
£ coli	DSM 15441	12 February 2003<br>
£ coli	DSM 15442	12 February 2003<br>
MATERIALS AND METHODS<br>
Media and substrates<br>
Medium YP+2%G<br>
10g yeast extract<br>
20g peptone<br>
water to 1L<br><br>
autoclave at 121°C, 20 minutes<br>
add 100ml 20% sterile glucose solution<br>
RA sporulation medium: 50 g succinic acid 12.1 g sodium nitrate 1 g glucose<br>
20 ml 50x Vogel's salts (Davis, R. H. and F. J. de Serres (1970), Meth. Enzymol. 17A:79-143) components are blended in one liter distilled water and filter sterilized<br>
Britton Robinson buffer<br>
0.023 M phosphoric acid<br>
0.023 M acetic acid<br>
0.023 M boric acid<br>
Titrated with NaOH or HCI to desired pH<br>
Methods<br>
Phospholipase activity (LEU)<br>
Lecithin is hydrolyzed under constant pH and temperature, and the phospholipase<br>
activity is determined as the rate of titrant (0.1N NaOH) consumption during neutralization of<br>
the liberated fatty acid.<br>
The substrate is soy lecithin (L-ct-Phosphotidyl-Choline), and the conditions are pH<br>
8.00,40.0°C, reaction time 2 min. The unit is defined relative to a standard.<br>
EXAMPLES<br>
Example 1: Expression of a phospholipase A2 from Tuber albidum in Aspergillus<br>
oryzae<br>
The DNA sequence disclosed in Soragni et al. (supra) was used to design primers for PCR amplification of TbSP1 from genomic DNA, with appropriate restriction sites added to the primer ends to facilitate cloning of the PCR product (SEQ ID NO: 13 and 14). A Tuber albidum strain, CBS 272.72, was obtained from the CBS (Centraalbureau voor Schimmelcul-tures, Utecht, The Netherlands), and cultured on X-agar at 20°C, as recommended by the CBS in List of Cultures, 1996. Mycelium was removed from the surface of the plate, and total DNA was isolated using a FastDNA Spin Kit B10101 Inc.. Vista. GA), following the manufac-<br><br>
turer's instructions. PCR amplification was performed using Extensor Hi-Fidelity PCR Master Mix (ABgene, Surrey, U.K.) following the manufacturers instructions and using an annealing temperature of 52°C for the first 5 cycles and 62°C for the last 25 cycles. A single PCR product was obtained, and the sequence was determined and is presented as SEQ ID 9 excluding the added synthetic restriction sites. Comparison of this genomic sequence to the cDNA sequence presented by E. Soragni et al. revealed a single intron. When the intron is removed, the nucleotide sequence from 7. albidum CBS272.72 is 92.5% identical to that from 7. borchii ATCC 96540, the strain used by E. Soragni et al. The corresponding peptide predicted from the 7 albidum CBS272.72 gene sequence is 93.8% identical to the peptide sequence reported by Soragni et al.<br>
The PCR fragment was restricted with BamH\ and Xho\ and cloned into the Aspergillus expression vector pMStr57 using standard techniques. The expression vector pMStr57 contains the same elements as pCaHj483 (WO 98/00529), with minor modifications made to the Aspergillus NA2 promoter, and has sequences for selection and propogation in E. coli, and selection and expression in Aspergillus. Specifically, selection in Aspergillus is facilitated by the amdS gene of Aspergillus nidulans, which allows the use of acetamide as a sole nitrogen source. Expression in Aspergillus is mediated by a modified neutral amylase II (NA2) promoter from Aspergillus niger which is fused to the 5' leader sequence of the triose phosphate isomerase (tpi) encoding-gene from Aspergillus nidulans, and the terminator from the amyloglucosidase-encoding gene from Aspergillus niger. The phospholipase A2-encoding gene of the resulting Aspergillus expression construct, pMStr70, was sequenced and the sequence was compared to that determined previously for the uncloned PCR fragment, SEQ ID 9. A single T to C mutation was found 52bp downstream of the stop codon.<br>
Aspergillus oryzae was transformed with pMStr70 using standard techniques de-scribeed in Christensen, T. et al., (1988), Biotechnology 6, 1419-1422. Transformants were cultured in YP+2%G medium shaken at 275 RPM at 30°C and expression of the Tuber phospholipase A2, TbPLA2, was monitored by SDS-PAGE.<br>
Protein characterization<br>
SDS-PAGE revealed two bands, with approximate Mw of 25 and16 kDa. The supernatant was purified by ion exchange chromatography on a SP-sepharose column equilibrated with 50 mM Acetate-buffer, and eluted with 1M NaCI pH 5.0. The two proteins eluted in two separate fractions. Protein concentration was determined using Protein Assay ESL from Roche. Activity was determined in the LEU assay.<br><br><br>
The proteins were subjected to N-terminal sequencing. The N-terminal sequence of pool 1 (23-25 kDa band) was found to correspond to amino acids 32-50 of SEQ ID NO: 10. Blotting of pool 2 (16 kDa band) revealed two bands with N-terminal sequences corresponding to amino acids 86-98 and 91-103, respectively. Mass spectral analysis of the two bands showed masses of 13934 and 14348 Da respectively, matching within 5 Da of values calculated from the sequences of amino acids 86-210 and 91-210 of SEQ ID NO: 10, respectively.<br>
Example 2: Purification procedure for two forms of T. albidium PLA2 expressed in Aspergillus oryzae.<br>
In most fermentations of the Aspergillus oryzae transformant described in Example I that produces the T. albidum PLA2, two forms of the enzyme were detected during purification. One form ran at 22-23 kDa in SDS-PAGE and corresponds to the peptide reported by Soragni et al. {supra). Additionally, a new form was detected which ran at 16-17 kDa in SDS-PAGE and which has a high specific activity and a high isoelectric point.<br>
Purification of the 22-23 kDa peptide<br>
Fermentation supernatant containing phospholipase from T.albidium expressed in A. oryzae (prepared in Example 1) was sterile filtered using EKS filter purchased from Seitz Schenk Bad Kreuznach, Bettringerstrasse 42, Germany D-73550, Waldstetten.<br>
The sterile filtered supernatant was then adjusted to pH of 8 and ionic strength under 4mSL<br>
Anion Exchange chromatography<br>
First step of purification was carried out on anion exchange chromatography using<br>
50 ml Fast flow Q ™ sepharose column purchased from Amersham Pharmacia. The column was prequilibrated with 50 mM Tris acetate buffer pH 8. The sterile filtered fermentation broth was then applied on the column and the column was washed with the same buffer until all unbound material was washed out.<br>
Bound proteins were eluted with the same buffer containing 1 M Sodium chloride pH<br><br>
8 with flow rate of 5ml/minute and to a final volume of 500 ml total buffer. Fractions of 5ml each were collected using fraction collector and Phospholipase activity of all fractions containing was assayed qualitatively using Lecithin as substrate using L-ar- Phosphatidyl choline purchased from Sigma product P-5638 and activity was assayed using NEFA C kit purchased from Wako Chemicals GmbH, Nissan Strasse 2, 41468 Neuss, Germany. Exact assay is described below.<br>
Substrate solutions containing 10 mg/ml of Lecithin substrate were prepared in different buffers such as 50 mM Acetate pH 5 or 50 mM Hepes pH 7 or 50 mM Tris acetate pH 9 as buffers containing 2mM CaCI2 and 0.1 % Triton X-100 purchased from Fluka chemicals. Substrate was then emulsified by stirring and warming at 50°C and then cooling to 40°C and used as substrate.<br>
Assay of activity was carried out using Z00JJ\ of the substrate emulsion incubated with 25 u\ of the enzyme fractions for 20 minutes at 40°C then 30 pi of the assay mixture was transferred to 300 //I of the NEFA C color reagent A prepared as described by the manufacturer and incubated for 10 minutes at 37°C and 600 //I of the color reagent NEFA C B solution was added to the mixture and further incubated for 10 minutes. The blue color formed was then measured in a spectrophotometer at 505 nm.<br>
Protein characterization<br>
Fractions containing activity were then pooled and characterized for the molecular<br>
weight using SDS-PAGE electrophoresis using Novex Pre casted gels 4 to 20 % Tris-Glycine<br>
gels purchased from Invitrogen Life Technologies, Carlsbad CA 92008, USA.<br>
22-23 kDa protein was detected and blotted and N-terminal analysis was carried out using an Applied Biosystem sequenator.<br>
The first 19 amino acid residues from N-terminal were determined and found to have the sequence of amino acids 32-50 of SEQ ID NO: 10.<br>
Purification of the the 16-17 kDa peptide<br>
Sterile filtered fermentation supernatant of the T. albidum phospholiplase expressed<br>
in A. oryzae was adjusted to pH 4.7 and ionic strength was adjusted below 4 mSi.<br>
Cation exchange chromatography<br>
SP-sepharose ™ fast flow was purchased from Amersham Pharmacia. 50 ml Column was packed and equilibrated with 50 mM acetate buffer pH 47 the fermentation supernatant was then applied on column and unbound material was washed using the same buffer.<br>
Bound protein with high pi was then eluted with a linear salt gradient using 50 mM acetate buffer pH 4.7 containing 1 M Sodium chloride. Fractions and flow rate were similar to those used for the low pi form of the phospholipase. Phospholipase activity in the fractions was assayed qualitatively using NEFA kit as above. Fractions containing Phospholipase ac-<br><br>
vity were pooled and SDS-PAGE was carried out as described above.<br>
16-17 kDa protein was observed which had a high isoelectric point, above 9.<br>
The N-terminal analysis of the protein was carried out after blotting the protein and sing Applied biosystem sequentaor which showed an N-terminal which was completely dif-3rent from the one published in Soragni et al. (supra). Thus, the T albidum PLA2 was found D have two forms deriving from differential N-terminal processing with N-terminal sequences corresponding to amino acids 86-105 and 91-110 of SEQ ID NO: 10, respectively.<br>
Example 3: Cheese making with T. albidum phospholipase<br>
Pasteurized, non-homogenized cream (North Carolina State University Dairy Plant) vas used to standardize five hundred grams pasteurized, non-homogenized skim milk (North Carolina State University Dairy Plant) to 3.5% fat thus producing full fat mozzarella cheese. The cheese milk for each experiment was treated with either the 16-17 kD T. albidum phospholipase prepared according to example 2, or the commercial phospholipase Lecitase® 10L (Novozymes A/S, Bagsvaerd, Denmark), and placed in a 35°C water bath until equilibrated to that temperature. The initial pH of the cheese milk was taken and 0.01% (w/w) of starter culture was added.<br>
pH was monitored until a pH of 6.4 was reached. 250 µI rennet (Novozym 89L) was diluted to in 9 ml total solution with deionised water, one ml of this solution was added to the cheese milk and the cheese milk was stirred vigorously for 3 minutes. The stir bar was removed and the rennetted milk was allowed to sit at 35°C.<br>
After the above treatments, curd was ready to cut when a spatula was inserted and sharp edges were seen. The cheese was cut by pushing the cutter down and while holding the beaker quickly turning the cutter and finally pulling the cutter up. The curd was allowed to rest 5 minutes then stirred gently with spoon. Temperature was raised to 41 °C with intermittent gentle agitation for - 45 min or until the pH dropped to 6.0-5.9. The curd was drained using cheesecloth then replaced in the beaker and kept at 41 °C in water bath while pouring off whey as needed.<br>
When the curd reached pH 5.3, the stainless steel bowl with the curd in it was flooded in a water bath at 69°C for 5 minutes then hand stretched. Curd was tempered in cold icewater for 30 minutes. The cheese curd was dried out with paper towel, weighed and refrigerated overnight.<br>
Control cheese making experiments were made from the same batch of milk following the same procedures except that no phospholipase was added.<br><br>
t<br>
Actual cheese yield was calculated as the weight of cheese after stretching relative to the total weight of cheese milk.<br>
Moisture adjusted cheese yield was expressed as the actual yield adjusted to standard constant level of moisture. Moisture adjusted yield was calculated by multiplying the actual yield and the ratio of actual moisture content to standard moisture, according to the following formula:<br>
Yadj = (YactXl-Mact)/(1-Mstd)<br>
where Yadj = moisture adjusted cheese yield, Yact = actual cheese yield, Mact = actual moisture<br>
fraction &amp; Mstd = standard moisture fraction (0.48).<br>
The moisture adjusted cheese yield of all experiments and controls are shown in table 1<br><br>
Example 4: Cloning and expression of a phospholipase (FvPLA2) from Fusarium vene-natum in Aspergillus oryzae<br>
Cells of the Fusarium venenatum A3/5 (originally deposited as Fusarium graminea-rum ATCC 20334 and recently reclassified as Fusarium venenatum by Yoder and Christianson, 1998, Fungal Genetics and Biology 23: 62-80; and O'Donnell et al., 1998, Fungal Genetics and Biology 23: 57-67) were grown for two days in VogePs minimal medium (Davis, R. H. and F. J. de Serres (1970), Meth. Enzymol. 17A:79-143) at 28°C in shaking culture, filtered on sterile Miracloth (Calbiochem, San Diego, California, USA), and transferred to<br><br>
'RA'sporulation medium" in which they were incubated in shaking culture for an additional 24 hr at 28°C. Cells and spores were collected by centrifugation and lysed, and RNA was extracted and transcribed into cDNA that was cloned into pZErO-2 by the methods described in WO 00/56762. The number of independent clones in this library before amplification was 2.5x105, of which 92% contained inserts ranging in size from 550-2500 bp. Partial DNA sequences were determined for approximately 1000 randomly chosen clones and the sequences were stored in a computer database by methods described in WO 00/56762.<br>
The nucleotide sequence of a cDNA encoding TbSP1, a phospholipase A2 from Tuber borchii, and the corresponding peptide translation were reported by E. Soragni et al., 2001. This translated peptide sequence was compared to translations of the Fusarium vene-natum partial cDNA sequences using the TFASTXY program, version 3.3t08 (Pearson et al., 1997). One translated F. venenatum sequence was identified as having 42% identity to TbSP1 through a 125 amino acid overlap. The complete sequence of the cDNA insert of the corresponding clone, FM0700, was determined and is presented as SEQ ID NO: 15, and the peptide translated from this sequence, FvPLA2, is presented as SEQ ID NO: 16. This sequence was used to design the primers FvPLAI and FvPLA2.2 for PCR amplification of the FvPLA2 encoding-gene from FM0700, with appropriate restriction sites added to the primer ends to facilitate sub-cloning of the PCR product.<br>
FvPLAI: CTGGGATCCTCAAGATGAAGTTCAGCG<br>
FvPLA2.2: GACCTCGAGACCCGCCATTTAAGATT<br>
PCR amplification was performed using Extensor Hi-Fidelity PCR Master Mix (AB-gene, Surrey, U.K.) following the manufacturers instructions and using an annealing temperature of 52°C and an extension temperature of 60°C for 20 cycles.<br>
The PCR fragment was restricted with BamHI and Xhol and cloned into the Aspergillus expression vector pMStr57 using standard techniques. The expression vector pMStr57 contains the same elements as pCaHj483 (WO 98/00529), with minor modifications made to the Aspergillus NA2 promoter as described for the vector pMT2188 in WO 01/12794, and has sequences for selection and propagation in £ coll, and selection and expression in Aspergillus. Specifically, selection in Aspergillus is facilitated by the amdS gene of Aspergillus nidu-lans% which allows the use of acetamide as a sole nitrogen source. Expression in Aspergillus is mediated by a modified neutral amylase II (NA2) promoter from Aspergillus niger which is fused to the 5' leader sequence of the those phosphate isomerase (tpi) encoding-gene from Aspergillus niduians, and the terminator from the amyloglucosidase-encoding gene from Aspergillus niger. The phospholipase-encoding gene of the resulting Aspergillus expression construct, pMStr77, was sequenced and the sequence agreed completely with that determined previously for the insert of FM0700.<br>
The Aspergillus oryzae strain BECh2 (WO 00/39322) was transformed with pMStr77<br><br>
using standard techniques (T. Christensen et a!., 1988). TransTormanxs were cuuurea in YP+2%G medium shaken at 275 RPM at 30°C and expression of FvPLA2 was monitored by SDS-PAGE.<br>
A strain of Eschericia coli containing a gene encoding the phospholipase from F. ve-nenatum was deposited by the inventors under the terms of the Budapest Treaty with Deutsche Sammlung von Microorganismen und Zellkuituren GmbH, Mascheroder Weg 1b, D-38124 Braunschweig, Germany. The deposit date was 12 February 2003, and the accession number was DSM 15442.<br>
Example 5: Purification and sequence comparison of FvPLA2<br>
FvPLA2 from the fermentation of example 4 was purified by ion exchange chromatography on a SP-sepharose column equilibrated with 50 mM Acetate-buffer pH 4.7, and eluted with 1M NaCI pH 4.7. Fractions were analyzed on SDS-PAGE, and fractions containing a 14 kDa protein were pooled. The identity of the pure protein was confirmed by determining the the N-terminal sequence, which was identical to the sequence from amino acid (aa) 29-40 of SEQ ID NO: 16. Additionally, the mass of the peptide was determined by mass spectral analysis, because the apparent size estimated from SDS-PAGE, 14 kDa, is smaller than that of the peptide predicted by processing the theoretical peptide in SEQ ID NO: 16. The mass of the purified, active FvPLA2 was found to be 13336 Da. This molecular mass indicates additional processing at the C-terminus, and is consistent with a cleavage between amino acids 149 and 150 in SEQ ID NO: 16, as the peptide sequence from amino acid 29 to 149 has a theoretical mass of 13335,66 Da.<br>
A comparison of the mature processed peptide (amino acids 29-149 of SEQ ID NO: 16) with known sequences showed that the closest prior-art sequence was a phospholipase from Verticillium dahliae translated from Unisequence ID: VD0100C34 from the COGEME Phytopathogenic Fungi and Oomycete EST Database Version 1.2 (http://cogeme.ex.ac.uk/) (Soanes et al. (2002) Genomics of phytopathogenic fungi and the development of bioinfor-matic resources. Mol Plant Microbe Interact. 15(5):421-7). The processing of the partial peptide predicted from the V. dahliae sequence was estimated by comparison to the found processing for FvPLA2. The identity between amino acids 29 to 149 of SEQ ID NO: 16 and the estimated sequence of the mature peptide of the V. dahliae phospholipase was calculated to be 77%.<br>
Example 6: Physical properties of FvPLA2<br>
Catalytic activity<br>
Phospholipase activity as a function of enzyme concentration was determined in the LEU assay for FvPLA2 of example 4. Results are shown in table 1.<br><br><br><br><br>
Example 7: Cheese making with FvPLA2<br>
Pasteurized, non-homogenized cream (North Carolina State University Dairy Plant) was used to standardize five hundred grams pasteurized, non-homogenized skim milk (North Carolina State University Dairy Plant) to 3.5% fat thus producing full fat mozzarella cheese.<br>
The cheese milk for each experiment was treated with either the F. venenatum phospholipase (FvPLA2) prepared according to example 5, or of the commercial phospholi-<br><br>
pase' Lecitase® 10L (Novozymes A/S, Bagsvaerd, Denmark), and placed in a 35°C water bath until equilibrated to that temperature. The initial pH of the cheese milk was taken and 0.01% (w/w) of starter culture at was added.<br>
pH was monitored until a pH of 6.4 was reached. 250µI rennet (Novozym 89L) was diluted to in 9 ml total solution with deionized water, one ml of this solution was added to the cheese milk and the cheese milk was stirred vigorously for 3 minutes. The stir bar was removed and the rennetted milk was allowed to sit at 35°C.<br>
After the above treatments, curd was ready to cut when a spatula was inserted and sharp edges were seen. The cheese was cut by pushing the cutter down and while holding the beaker quickly turning the cutter and finally pulling the cutter up. The curd was allowed to rest 5 minutes then stirred gently with spoon. Temperature was raised to 41 °C with intermittent gentle agitation for ~ 45 min or until the pH dropped to 6.0-5.9. The curd was drained using cheesecloth then replaced in the beaker and kept at 41°C in water bath while pouring off whey as needed.<br>
When the curd reached pH 5.3, the stainless steel bowl with the curd in it was flooded in a water bath at 69°C for 5 minutes then hand stretched. Curd was tempered in cold icewater for 30 minutes. The cheese curd was dried out with paper towel, weighed and refrigerated overnight.<br>
Control cheese making experiments were made from the same batch of milk following the same procedures except that no phospholipase was added.<br>
Actual cheese yield was calculated as the weight of cheese after stretching relative to the total weight of cheese milk.<br>
Moisture adjusted cheese yield was expressed as the actual yield adjusted to standard constant level of moisture. Moisture adjusted yield was calculated by multiplying the actual yield and the ratio of actual moisture content to standard moisture, according to the following formula:<br>
Yadj = Yact x (1- Mact) / (1-Mstd )<br>
where Yadj = moisture adjusted cheese yield, Yact = actual cheese yield, Mact = actual moisture fraction &amp; Mstd = standard moisture fraction (0.48).<br>
The moisture adjusted cheese yield of all experiments and controls are shown in table 5.<br><br><br>
Example 8: Cheese making with FvPLA2<br>
Milk was pasteurized at 72°C for 15 seconds and then cooled to below 10°C. Milk was standardized to 2.4% fat with cream. After standardization the milk was preheated in a heat exchanger at a pre-ripening temperature of 34.5°C. 150 kg milk was poured into each cheese vat and 15 g culture (F-DVS ST-M6) was added. The phospholipase from example 5 was added in a dosage of 5 LEU/g fat and the milk was incubated for 1 h at 34.5°C. Rennet (Chy-Max Plus, 200 IMCU) was added and agitation was continued for not more than 4 min.<br>
After approx. 60 min when the coagulum was judged ready it was cut using 10 mm knives. The agitator was returned to the vat and after 10 min. the scalding was started by increasing the temperature to 41 °C within 30 min. After reaching 41 °C a further stirring for approximately 20 min. took place until a titratable acidity of 0.15-0.16% was reached. The curd was allowed to settle in the vat, and whey was drained. The curd was cut in uniform blocks and the blocks were turned and stacked into two. Subsequently, at intervals of 10 min. the blocks were turned and kept in stacks of two. At a pH of around 5.15-5.20, the curd was milled in a milling machine. The curd pieces were added two percent of salt (weight/weight).<br><br>
\fter milling all the curd was added into the stretcher, which contains 70 I preheated water at 74°C. Around 201 of hot water was transferred to the upper chamber and the cheese is added. When the curd temperature reached 62°C, the stretching was stopped and the curd moved to the extruder. Cheeses were extruded into 8-9 cheese loaves, each of 2.3 kg, and cooled in 5-7°C water for 20 min. After 20 min. cooling the cheeses were moved to the saturated brine and brined for 1.5 hours at 5-6°C. The brine was made by blending 120 kg water, adding salt to 22Be, 750 g CaCI2 (34% solution) and adjusted to pH 5.1. After brining each cheese was dried for around 30 min. and weighed before vacuum packaging. Samples were taken for pH and compositional analyses (moisture, salt, fat and protein) after about 1 week's storage in cold room.<br>
Actual yield (AY) was adjusted to 48% moisture in cheese:<br>
Adj Yield = AYx (100-% moisture)<br>
•\   <br>
DAP2C-1<br>
11gMgS047H20<br>
1g KH2P04<br>
2g Citric acid, monohydrate<br>
30g maltodextrin<br>
6g l
0.5g yeast extract<br>
0.5ml trace metals solution<br>
1ml Pluronic PE 6100 (BASF, Ludwigshafen, Germany)<br>
Components are blended in one liter distilled water and portioned out to flasks, adding 250 mg CaC03 to each 150ml portion.<br>
The medium is sterilized in an autoclave. After cooling the following is added to 1 liter of medium:<br>
23 ml 50% w/v (NH4)2HP04) filter sterilized<br>
33 ml 20% lactic acid, filter sterilized<br>
Trace metals solution 6.8g ZnCl2<br>
2.5g CuS04-5H20<br>
0.24g NiCI2-6H20<br>
13.9gFeS04-7H20<br>
8.45g MnS04-H20<br>
3g Citric acid, monohydrate<br>
Components are blended in one liter distilled water.<br>
The cloning and partial sequencing of a cDNA encoding a phospholipase A2 from Aspergillus oryzae is described in WO 00/56762. The full sequence of the clone, AS3812, is given in SEQ ID NO: 6.<br>
This sequence was used to design the primer AoPLA1 for use with the vector primer pYESrev in PCR amplification of the PLA2 encoding-gene from AS3812 with the addition of a restriction site to facilitate sub-cloning of the PCR product:<br>
AoPLAI: TGAGGATCCATCATGAAGAACATCTTCG<br>
pYESrev: gggcgtgaatgtaagcgtgac<br>
PCR amplification was accomplished using Extensor Hi-Fidelity PCR Master Mix (ABgene, Surrey, U.K.) following the manufacturers instructions and using an annealing temperature of 52°C for the first 5 cycles and 62°C for the last 25 cycles, and an extension time of 1.5 minutes.<br><br>
The PCR fragment was restricted with BamHI and Xhol and cloned into the Aspergillus expression vector pMStr57 (described in Example 1) using standard techniques. The phospholipase-encoding gene of the resulting Aspergillus expression construct, pMStr71, was sequenced and the sequence agreed completely with that determined previously for the insert ofAS3812.<br>
The Aspergillus oryzae strain BECh2 (WO 00/39322) was transformed with pMStr71 using standard techniques (T. Christensen et al., 1988). Transformants were cultured in DAP2C-1 medium shaken at 270 RPM at 37°C for 4 days and expression of phospholipase was monitored by SDS-PAGE.<br>
Example 10: Purification and determination of peptide processing<br>
The Aspergillus oryzae phospholipase from the fermentation of example 9 was filtered through 0.22// sterile filter Seitz-EKS obtained from Pall Corporation (Pall SeitzSchenk Filter Systems GmbH Pianiger Str.137 D-55543 Bad Kreuznach, Germany). The sterile filtered solution was then adjusted to pH 4.7 using dilute acetic acid. Ionic strength of the fermentation supernatant was then adjusted so that salt concentration was low and ionic strength was under 4 mSi. Purification of the desired PLA2 protein was obtained by cation exchange chromatography using SP sepharose fast Flow matrix obtained from Amersham-Pharmacia (Sweden). The cation exchanger matrix was packed washed and pre-equilibrated with 50 mM Sodium acetate buffer pH 4.7 ( Buffer A) on XK26 column obtained from Amersham Pharmacia. Fermentation supernatant containing the desired PLA2 adjusted for pH and ionic strength was then applied on the column. Unbound material was then washed with the buffer A until all the UV absorbing material was washed out, which was monitored by UV detector attached to fraction collector equipment. Bound proteins were then eluted with a linear salt gradient using Buffer B, which contained 1 M Sodium chloride as salt in 50 mM Sodium acetate buffer pH 4.7. Total volume of the linear gradient reaching 1 M salt concentration was around 500 ml (10 column volume). Fractions of 10 ml each were collected during the elution. All the fractions were assayed for phospholipase activity using Lecithin as substrate obtained from Sigma chemicals. Fatty acids released from Lecithin on incubation with the phospholipase were detected using NEFA C kit obtained from Waco chemicals. Fractions containing phospholipase activity were then checked for purity of the protein using standard SDS-PAGE technique. Fractions were pooled that contained a single band of the desired PLA2 showing molecular weight of around 16 kDa, as determined by comparison to molecular weight standards from Amersham-Pharmacia.<br>
The identity of the pure protein was confirmed by determining the N-terminal sequence, which was identical to the sequence from amino acid (aa) 37-45 of SEQ ID NO: 7. Additionally, the mass of the peptide was determined by mass spectral analysis. The purified,<br><br>
active Aspergillus PLA2 gave two masses, 14114 and 14242 Da. These molecular masses indicate additional processing at the C-terminus, consistent with cleavage between amino acids 121 and 122 in SEQ ID NO: 7, as the peptide sequence from amino acid 37 to 121 has a theoretical mass of 14114.11 Da and cleavage between amino acids 122 and 123, predicting the peptide sequence from amino acid 37 to 123 with a theoretical mass of 14242.29 Da.<br>
Example 11: Expression of incompletely processed phospholipase from Aspergillus oryzae and Fusarium venenatum<br>
Processing of the Aspergillus oryzae PLA2 (AoPLA2) and the Fusarium venenatum PLA (FvPLA2) at both the N- and C-termini occurs at single or multiple basic residues (lys or arg), typical of the cleavage sites of the Kexin-like maturases, which are often responsible for processing propeptides (Jalving, R., et al. (2000) Appl. Environ. Microbiol. 66: 363-368). In order to determine the effect of processing on the activity of AoPLA2 and FvPLA2, the enzymes were expressed in a Kexin deficient strain of Aspergillus oryzae. Processing was then assessed by SDS-PAGE, and phospholipase activity was measured for cultures of strains expressing AoPLA2 and FvPLA2 in both wild-type and Kexin deficient backgrounds,<br>
A Kexin deficient strain of Aspergillus oryzae {kexB") was constructed by a disrupting the kexB gene of A. oryzae (EMBLAB056727) by methods established in the art, such as those described in WO 98/12300 and US6013452. Disruption of kexB was confirmed by Southern blot analysis and by monitoring the expression of peptides where KexB is known to be responsible for maturation. The kexB' strain was transformed with the AoPLA2 expression construct described in Example 9, and with the FvPLA2 expression construct described in Example 4. These strains were fermented in YP+2%G at 30°C, along with the /cexB* expression strains for both AoPLA2 and FvPLA2 described in Examples 9 and 4, and untransformed strains as controls. AoPLA2 expressing strains were shaken at 200RPM for 4 days while FvPLA2 expressing strains were shaken at 275RPm for 3 days. Phospholipase expression and processing were assessed by SDS-PAGE.<br>
In SDS-PAGE analysis, AoPLA2 was resolved as a distinct single band in both /cexB+ and kexB' strains. When expressed in the /cexB+ strain, AoPLA2 ran at ca. 16 kDa, consistent with the migration observed earlier for fully processed AoPLA2 (Example 10), while in the kexB' strain, AoPLA2 ran at ca. 27-28 kDa, consistent with a lack of processing or incomplete processing. When expressed in the kexB+ strain FvPLA2 was resolved as two bands with apparent molecular weights of 17kDa and 14kDa. The 14kDa band corresponds to the fully processed peptide (Example 5), while the 17kDa peptide is a partially processed form.<br><br>
When expressed in the kexB' strain, FvPLA2 ran as a single band at ca. !8-19kDa, a size consistent with incomplete processing. No similar bands were seen in any of the control samples from untransformed strains. Relative band intensities suggest that expression of AoPLA2 in the kexB' strain was 1/5 to 1/10 the level of that in the kexB* strain, while expression of FvPLA2 in the kexB' strain was the same to 1/2 the level of that in the kexB* strain.<br>
The activity of the phospholipases produced by each strain was determined in the LEU assay and is shown in table 7.<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
CLAIMS<br>
1.	A method of producing a phospholipase which comprises processing an expressed fungal<br>
peptide so as to cleave off a peptide from the C-terminal end and/or a peptide from the N-<br>
terminal end to obtain a core peptide with phospholipase activity, wherein the core peptide<br>
comprises<br>
a)	the amino acid sequence given by amino acids 146-153 of SEQ ID NO: 1f amino acids 87-94 of SEQ ID NO: 3, or amino acids 79-86 of SEQ ID NO: 12; or a sequence identical to any of these amino acid sequences except for the substitution of a single amino acid with another amino acid; and<br>
b)	at least two cysteine residues located on the N-terminal side of the sequence given in a); and<br>
c)	at least two cysteine residues located on the C-terminal side of the sequence given in a).<br><br>
2.	The method of claim 1 wherein the expressed peptide is expressed in a filamentous fungal host cell transformed with DNA encoding the expressed peptide.<br>
3.	The method of claim 2 wherein the host cell is an Aspergillus, Fusarium or Trichoderma, particularly A oryzae, A. niger, F. venenatum or T. reesei.<br>
4.	The method of claim 2 wherein the expressed peptide is processed in vivo by the host cell.<br>
5.	The method of any of the claims 1-4 wherein the core peptide has a length of 100-150 amino acids.<br>
6.	The method of any of the claims 1-5 wherein the phospholipase has a specific phospholipase activity which is at least 2 times the activity of the expressed peptide before processing.<br>
7.	The method of any of the claims 1-6 wherein the expressed peptide is derived from Tuber, Verticillium, Neurospora, Aspergillus, or Helicosporum, particularly 7". borchii, T. albidum, V. dahliae, V. tenerum, N. crassa, A. oryzae, or Helicosporium sp.HNL<br>
8.	The method of any of the claims 1-7 wherein the expressed peptide is cleaved within 0-18 amino acids on the N-terminal side of the sequence aligning with amino acids 97-101 of SEQ ID NO: 1, when the complete expressed peptide sequence is aligned simultaneously with the sequences given in SEQ ID NO: 1, SEQ ID NO: 3, SEQ ID NO: 4, SEQ ID NO: 5, SEQ ID<br><br><br>
NO: 7, SEQ ID NO: 8, SEQ ID NO: 10, and SEQ ID NO: 12.<br>
9.	The method of any of the claims 1-8 wherein the expressed peptide is cleaved within 0-11 amino acids on the C-terminal side of the sequence aligning with amino acids 204-209 of SEQ ID NO: 1, when the complete expressed peptide sequence is aligned simultaneously with the sequences given in SEQ ID NO: 1, SEQ ID NO: 3, SEQ ID NO: 4, SEQ ID NO: 5, SEQ ID NO: 7, SEQ ID NO: 8, SEQ ID NO: 10, and SEQ ID NO: 12.<br>
10.	The method of any of claims 1-9 wherein the expressed peptide is cleaved within 10 amino acids of a Kex2 processing site or within 11 amino acids of an FG sequence or both.<br>
11.	A method of hydrolyzing a phospholipid comprising contacting the phospholipid with a phospholipase produced by the method of any of the claims 1-10.<br>
12.	A method of producing cheese comprising contacting cheese milk or a fraction of cheese milk with a phospholipase, wherein the phospholipase comprises:<br><br>
a)	the sequence given by amino acids 146-153 of SEQ ID NO: 1f amino acids 87-94 of SEQ ID NO: 3, or amino acids 79-86 of SEQ ID NO: 12; or a sequence identical to any of these amino acid sequences except for the substitution of a single amino acid with another amino acid; and<br>
b)	two cysteine residues located on the N-terminal side of the sequence given in a); and<br>
c)	two cysteine residues located on the C-terminal side of the sequence given in a).<br><br>
13.	A method of producing cheese comprising contacting cheese milk or a fraction of cheese milk with a phospholipase produced by the method of any of the claims 1-10.<br>
14.	A phospholipase which is a polypeptide having an amino acid sequence which is at least 80 % identical with amino acids 91-210 in SEQ ID NO: 10 (T. albidum), amino acids 92-211 in SEQ ID NO: 1 (7. borchii), amino acids 30-137 in SEQ ID NO: 12 (V. tenerum), amino acids 38-145 in SEQ ID NO: 3 (V. dahliae), amino acids 44-151 in SEQ ID NO: 4 (N. crassa), amino acids 37-157 in SEQ ID NO: 7 (A oryzae), or amino acids 58-168 in SEQ ID NO: 8 (A/. crassa).<br>
15.	A phospholipase which comprises:<br><br>
a)	a polypeptide encoded by the phospholipase encoding part of the DNA sequence cloned into a plasmid present in Escherichia coli deposit number DSM 15442; or<br>
b)	a polypeptide comprising the amino acid sequence of amino acids 29 to 149 of SEQ ID NO: 16,or an amino acid sequence which can be obtained therefrom by<br><br>
substitution, deletion, and/or insertion of one or more amino acids; or<br>
c)	an analogue of the polypeptide defined in (a) or (b) which:<br>
i)        has at least 80 % homology with said polypeptide, or<br>
ii)        is immunologically reactive with an antibody raised against said<br>
polypeptide in purified form, or iii)       is an allelic variant of said polypeptide; or<br>
d)	a polypeptide which is encoded by a nucleic acid sequence which hybridizes un<br>
der low stringency conditions with a complementary strand of the nucleic acid se<br>
quence of nucleic acids 133 to 495 SEQ ID NO: 15 encoding the mature polypeptide<br>
or a subsequence thereof having at least 100 nucleotides.<br>
6.	The phospholipase of claim 15 which is native to a strain of Fusarium, particularly F. ve-enatum.<br>
7.	A nucleic acid sequence comprising a nucleic acid sequence which encodes the phos-iholipase of claims 15 or 16.<br>
8.	A nucleic acid sequence which comprises:<br><br>
a)	the partial DNA sequence encoding a mature phospholipase cloned into a plas-mid present in Escherichia coli DSM 15442,<br>
b)	the partial DNA sequence encoding a mature phospholipase of nucleic acids 133 to 495 of SEQ ID NO: 15,<br>
c)	an analogue of the sequence defined in a) or b) which encodes a phospholipase and<br>
i)        has at least 80 % homology with said DNA sequence, or ii)       hybridizes at high stringency with a complementary strand of said DNA sequence or a subsequence thereof having at least 100 nucleotides, iii)       is an allelic variant thereof, or<br>
d)	a complementary strand of a), b) or c).<br>
19.	A nucleic acid construct comprising the nucleic acid sequence of claims 17 or 18 operably linked to one or more control sequences capable of directing the expression of the phospholipase in a suitable expression host.<br>
20.	A recombinant expression vector comprising the nucleic acid construct of claim 19, a promoter, and transcriptional and translational stop signals.<br><br>
21 f A recombinant host cell comprising the nucleic acid construct of claim 20.<br>
22.	A method for producing a phospholipase comprising cultivating the host cell of claim 21<br>
under conditions conducive to production of the phospholipase, and recovering the phosphol<br>
ipase.<br>
23.	A method for preparing a dough or a baked product made from the dough, comprising<br>
adding the phospholipase of claim 15 to the dough.<br>
24.	A dough composition comprising the phospholipase of claim 15.<br>
25.	A detergent composition comprising a surfactant and the phospholipase of claim 15.<br><br>
26.	A process for reducing the content of phosphorus in a vegetable oil, comprising contacting the oil with the phospholipase of claim 15 in the presence of water, and then separating an aqueous phase from the oil.<br>
27.	A process for producing cheese comprising treating a dairy composition with a phospholipase and producing cheese from the dairy composition, wherein the phospholipase is the phospholipase of claim 15.<br>
28.	A process for producing cheese comprising treating a dairy composition with a phospholi<br>
pase and producing cheese from the dairy composition, wherein the phospholipase is se<br>
lected from the group of fungal/bacterial group XIII PLA2 phospholipases.<br><br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjc4Mi1DSEVOUC0yMDA1ICBDTEFJTVMucGRm" target="_blank" style="word-wrap:break-word;">2782-CHENP-2005  CLAIMS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjc4Mi1DSEVOUC0yMDA1ICBDT1JSRVNQT05ERU5DRSBPVEhFUlMucGRm" target="_blank" style="word-wrap:break-word;">2782-CHENP-2005  CORRESPONDENCE OTHERS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjc4Mi1DSEVOUC0yMDA1ICBDT1JSRVNQT05ERU5DRSBQTy5wZGY=" target="_blank" style="word-wrap:break-word;">2782-CHENP-2005  CORRESPONDENCE PO.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjc4Mi1DSEVOUC0yMDA1ICBGT1JNLTMucGRm" target="_blank" style="word-wrap:break-word;">2782-CHENP-2005  FORM-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjc4Mi1DSEVOUC0yMDA1ICBGT1JNLTUucGRm" target="_blank" style="word-wrap:break-word;">2782-CHENP-2005  FORM-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjc4Mi1DSEVOUC0yMDA1ICBQT1dFUiBPRiBBVFRPUk5FWS5wZGY=" target="_blank" style="word-wrap:break-word;">2782-CHENP-2005  POWER OF ATTORNEY.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjc4Mi1jaGVucC0yMDA1LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2782-chenp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjc4Mi1jaGVucC0yMDA1LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2782-chenp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjc4Mi1jaGVucC0yMDA1LWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2782-chenp-2005-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjc4Mi1jaGVucC0yMDA1LWNvcnJlc3BvbmRuZWNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">2782-chenp-2005-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjc4Mi1jaGVucC0yMDA1LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">2782-chenp-2005-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjc4Mi1jaGVucC0yMDA1LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">2782-chenp-2005-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjc4Mi1jaGVucC0yMDA1LWZvcm0gMTgucGRm" target="_blank" style="word-wrap:break-word;">2782-chenp-2005-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjc4Mi1jaGVucC0yMDA1LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">2782-chenp-2005-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjc4Mi1jaGVucC0yMDA1LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">2782-chenp-2005-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjc4Mi1jaGVucC0yMDA1LXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">2782-chenp-2005-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="228871-an-isolated-polynucleotide-sequence-of-the-par1-gene.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="228873-a-method-and-a-device-for-suspending-a-gas-in-a-liquid.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>228872</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2782/CHENP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>12/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>20-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>11-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>27-Oct-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NOVOZYMES A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>KROGSHOEJVEJ 36, DK-2880 BAGSAVERD,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>STRINGER , MARY, ANN</td>
											<td>SOEBORG HOVEDGADE 39 C, 3 TV., DK-2860 SOEBORG,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>FATUM, TINE, MUXOLL</td>
											<td>SOEVANGEN 50, DK-3450 ALLERODE,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>PATKAR, SHAMKANT, ANANT</td>
											<td>CHRISTOFFERS ALLE 91, DK-2800 LYNGBY,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N9/20</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/DK04/00279</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-04-23</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>PA 2003 01163</td>
									<td>2003-08-14</td>
								    <td>Denmark</td>
								</tr>
								<tr>
									<td>2</td>
									<td>PA 2003 00634</td>
									<td>2003-04-28</td>
								    <td>Denmark</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/228872-phospholipase-and-method-of-producing-it by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:30:23 GMT -->
</html>
