<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/229018-a-method-for-expressing-nucleic-acid-sequence-in-prokaryotic-host-cell-deficient-in-l-rhamnose-isomerase by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:40:06 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 229018:A METHOD FOR EXPRESSING NUCLEIC ACID SEQUENCE IN PROKARYOTIC HOST CELL DEFICIENT IN L-RHAMNOSE ISOMERASE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR EXPRESSING NUCLEIC ACID SEQUENCE IN PROKARYOTIC HOST CELL DEFICIENT IN L-RHAMNOSE ISOMERASE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to methods for expressing nucleic acid sequences in prokaryotic host cells, where at least one DNA construct which is capable of episomal replication in said host cells and which comprises a nucleic acid sequence to be expressed under the transcriptional control of an L-rhamnose-inducible pro¬moter, where said promoter is heterologous with regard to said nucleic acid sequence, is introduced into said host cells and the expression of said nucleic acid sequence is induced by addition of L-rhamnose, wherein the prokaryotic host cell is at least de¬ficient with regard to an L-rhamnose isomerase.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
L-Rhamnose-inducible expression systems<br>
The present invention relates to methods for expressing nucleic acid sequences in prokaryotic host cells, where at least one DNA construct which is capable of episomal replication in said host cells and which comprises a nucleic acid sequence to be expressed under the transcriptional control of an L-rhamnose-inducible promoter, where said promoter is heterologous with regard to said nucleic acid sequence, is introduced into said host cells and the expression of said nucleic acid sequence is induced by addition of L-rhamnose, wherein the prokaryotic host cell is at least deficient with regard to an L-rhamnose isomerase. The invention furthermore relates to prokaryotic host cells which are at least deficient with regard to an L-rhamnose isomerase and which comprise at least one DNA construct which is capable of replication in said host cell and which comprises a nucleic acid to be expressed under the transcriptional control of an L-rhamnose-inducible promoter, where said promoter is heterologous with regard to said nucleic acid sequence.<br>
The heterologous expression of genes is an economical way of producing enzymes and other proteins for pharmaceutical and industrial purposes. Said expressions are still predominantly carried out using strains of Escherichia coli. A multiplicity of systems which rely on different host organisms and gene expression cassettes are known for the production of recombinant proteins. Although a large number of systems and methods for expressing recombinant proteins in microbiological systems have been described, the expression systems for Gram-negative bacteria such as Escherichia coli are based on a very limited range of bacterial promoters. Most widely used are the lactose promoter [lac] (Yanisch-Perron et al. (1985) Gene 33: 103-109) and the tryptophan promoter [trp] (Goeddel et al. (1980) Nature (London) 287: 411-416) and hybrid promoters of the above [lac and trp] (Brosius (1984) Gene 27:161-172; Amanna &amp; Brosius (1985) Gene 40: 183-190). Further examples are the PL and PR promoters of X. phage (Elvin et al. (1990) Gene 37:123-126), the Phage T7 promoter (Tabor &amp; Richardson (1998) Proc Natl Acad Sci USA 82:1074-1078) and the alkaline phosphatase promoter [pho] (Chang et al. (1986) Gene 44:121-125) .<br>
Heterologous expression entails various problems such as, for example, the toxicity of the gene product, unduly low expression rates or the formation of insoluble protein aggregates ("inclusion bodies"). Many of the above-described promoters are unsuitable for applications where the recombinant protein to be<br><br>
expressed has a toxic effect on the host in question. The strictest possible regulation of expression is desirable in these cases. Promoter systems which can be employed for this purpose are what are known as inducible promoter systems, which can be induced by means of addition of an inductor or another exogenous stimulus (for example heat). As a rule, said inducible promoter systems consist of a promoter/regulator combination, where the regulator is for example a protein which, in combination with an exogenous stimulus, induces the transcription starting from the promoter in question. An example which may be mentioned is the combination of a promoter with a repressor such as, for example, the lac repressor (Studier FW et al. (1990) Methods in Enzymol 185:60-89; Dubendorff JW &amp; Studier FW (1991) J Mol Biol 219:45-59). The repressing effect of this repressor can be removed by addition of a natural inductor (for example lactose) or an artificial inductor (for example isopropyl-β -D-thiogalacto-pyranoside; IPTG), thus initiating expression. In contrast to lactose, IPTG cannot be metabolized and thus ensures long-term induction. A further example of these inducible promoters is the arabinose-inducible araB promoter (US 5,028,530; Guzman LM et al, (1995) J Bacterid 177:4121-4130).<br>
IPTG and other synthetic inductors are very expensive and, in some cases, have an adverse effect on the growth of the organisms, which makes an application on the industrial scale uneconomic.<br>
While, as a rule, physiological inductors such as amino acids (for example tryptophan) and sugars (arabinose) are cheaper, they are metabolized by the organism so that substantial amounts must be added and/or fed subsequently when cells are grown, in particular in the case of high-density cell fermentations. Moreover, metabolites of these compounds may later also be harmful for the culture, for example when acetate is produced from sugars.<br>
WO 01/73082 describes a method for expressing recombinant proteins under the control of the inducible araB promoter in an E.coli host organism with deficiency for the active transport of the inductor arabinose. The advantage here is said to be that no active transport, but only passive transport (by means of diffusion), can take place. This means better control for the intracellular arabinose concentration and thus also expression induction. In some of the examples stated, an E. coli strain (E104) with deficiency in the arabinose-metabolizing enzymes ribulokinase (AraB) and L-ribulose-5-phosphate 4-epimerase (AraD) is employed. In accordance with the expression data, however,<br><br>
this deficiency has no substantial effect on the expression levels. The arabinose-inducible system has various disadvantages:<br>
a)	Arabinose has a growth-inhibitory effect on the bacterial culture from concentrations of as little as 0.1 mM and above, which can be compensated for only to a certain extent, even when using the method described in WO 01/73082 (cf. Table 4, WO 01/73082).<br>
b)	The arabinose-inducible promoter is not entirely inactive in the absence of arabinose, but has a fairly high basal activity (cf. Table 5, WO 01/73082).<br>
c)	The quality of the recombinant proteins expressed depends on the cell density and decreases with increasing cell densities (De Lisa MP et al. (1999) Biotechnol Bioeng 65:54-64).<br>
The Escherichia coli strain JB1204 (CGSC6999, Bulawa &amp; Raetz (1984) J Biol Chem 259:11257-11264), which has the transposon insertion "rhα-14::Tnl0", is described, but no detailed information on the sequence or function of "rhα-14" is provided.<br>
The uptake and metabolization of L-rhamnose in bacteria such as E.coli is described. L-Rhamnose is taken up into the cells via an active transport system (RhaT), converted into L-rhamnulose by an isomerase (RhaA), and L-rhamnulose is then phosphorylated further by rhamnulose 1-phosphatase (RhaB) and hydrolyzed by an aldolase (RhaD) to give dihydroxyacetone phosphate and lactaldehyde. The genes rhaBAD form an operon and are transcribed with the aid of what is known as the rhaPBAD promoter. In comparison with other systems, the rhamnose system is distinguished by the fact that two activators RhaS and RhaR are required for regulation. These two form a transcriptional unit and are transcribed in the opposite direction to rhaBAD. When L-rhamnose is present, RhaR binds to the rhaPRS promoter and initiates its own expression as well as the expression of RhaS. RhaS, in turn, once activated by L-rhamnose, binds as effector to the rhaPBAD promoter and the separate rhaPT promoter of the rhaT gene and activates the transcription of the structural gene (Moralejo P et al. (1993) J Bacteriol 175:5585-5594; Tobin JF et al. (1990) J Mol Biol 211:1-4; Chen YM et al. (1987) J Bacteriol 169:3712-3719; Egan SM et al. (1993) J Mol Biol 243:87-98). The combination of two activators causes an unusually strict expressional control by the rhaPBAD promoter. A comparison between the arabinose-inducible araB promoter and the rhamnose-inducible rhaPBAD promoter shows that the latter is subjected to substantially stricter regulation and, in the absence of the inductor rhamnose, virtually<br><br>
represents a zero phenotype (Haldimann A et al. (1998) J Bacteriol 180(5):1277-1286).<br>
WO 01/32890 describes the production of L-pantolactone hydrolase using Escherichia coli TGI pDHE681 or derivatives, where L-rhamnose is employed as inductor for the gene expression of the enzyme. Since L-rhamnose is metabolized well by E. coli, the L-rhamnose converted must be supplemented by feeding in. This makes the experimentation considerably more complicated and increases the costs for the culture medium.<br>
Furthermore described are expression systems for the fermentation under high cell densities using the L-rhamnose-inducible rhaBAD promoter and an E.coli strain with a site-specifically introduced deficiency in L-rhamnulose kinase (rhaB) (Stumpp T et al. (2000) Biospectrum 6(l):33-36; Wilms B et al. (2001) Biotechnol Bioeng 73(2): 95-103). RhaB was deliberately selected here since it is the first irreversible step in the metabolization of L-rhamnose (cf. Wilms B et al. (2001) Biotechnol Bioeng 73(2) p.98, left column, lines 4-8). Optimal induction can be achieved in these systems using L-rhamnose concentrations of 2 g/L (cf. Wilms B et al. (2001) Biotechnol Bioeng 73(2) p.102, left column, 2nd paragraph, lines 1-4). These concentrations are still very high. With an average L-rhamnose price of approximately 100 euros/kg, a 10 m3 fermenter would mean that 2000 euros are spent on L-rhamnose alone.<br>
Furthermore described are tightly-regulated rhamnose-inducible expression systems where the rhamnose operon (BAD), which is located behind the endogenous rhaPBAD promoter, is replaced by the PhoB gene (transcription activator) by means of homologous recombination (Haldimann A et al. (1998) J Bacteriol 180(5):1277-1286). While the system described herein is well suited to regulator studies since very tight regulation is ensured, it is less suitable for overexpression — in particular under high-density cell culture conditions — since in each case only one copy of the rhaPBAD promoter-controlled expression cassette can be introduced as the result of the replacement of the chromosomal rhamnose operon. Furthermore, the replacement of genes by homologous recombination is complicated and requires a tedious selection and characterization of suitably modified organisms. This makes the method described unsuitable for routine purposes.<br><br>
It was an object to provide an improved method for expressing nucleic acids — and preferably recombinant proteins — where small L-rhamnose quantities give high expression levels. This object is achieved by the present invention.<br>
A first aspect of the invention relates to methods for expressing nucleic acid sequences in prokaryotic host cells, where<br>
a)	at least one DNA construct which is capable of episomal replication in said host cells and which comprises a nucleic acid sequence to be expressed under the transcriptional control of an L-rhamnose-inducible promoter, where said promoter is heterologous with regard to said nucleic acid sequence, is introduced into said host cells and<br>
b)	prokaryotic host cells which comprise said DNA construct in episomal form are selected and<br>
c)	the expression of said nucleic acid sequence is induced by addition of L-rhamnose to a culture of said selected host cells,<br>
wherein the prokaryotic host cell is at least deficient with regard to L-rhamnose isomerase.<br>
In a preferred embodiment, the expression of the nucleic acid sequence to be expressed causes the production of a protein encoded by said nucleic acid sequence so that the method according to the invention for the production of recombinant proteins can be employed.<br>
In a furthermore preferred embodiment, an additional deficiency may be present in one or more further L-rhamnose-metabolizing, or —transporting, protein(s).<br>
A further aspect of the invention relates to a prokaryotic host cell which is at least deficient with regard to L-rhamnose isomerase and which comprises at least one DNA construct which is capable of replication in said host cell and which comprises a nucleic acid sequence to be expressed under the transcriptional control of an L-rhamnose inducible promoter, where said promoter is heterologous with regard to said nucleic acid sequence.<br>
In a preferred embodiment, the prokaryotic host cell according to the invention may have an additional deficiency in one or more further L-rhamnose-metabolizing, or —transporting, protein(s).<br><br>
Furthermore, the invention relates to a method for the production of recombinant proteins, enzymes and other fine chemicals such as, for example, chiral carboxylic acids, using one of the prokaryotic host cells according to the invention or a preparations thereof.<br>
The method according to the invention has various advantages:<br>
1.	It is simple to employ since the expression strain in question can be generated, starting from a host strain, by simple transformation without an insertion into the genome by means of homologous recombination (as by Haldimann A et al. (1998) J Bacterid 180(5):1277-1286) and a laborious selection of correctly modified organisms being required,<br>
2.	The expression cassettes and expression vectors provided within the scope of the invention are easy to handle. The rhaPBAD promoter, which is employed by way of example, has a length of just 123 base pairs.<br>
3.	Since L-rhamnose is metabolized by E.coli, in particular in the case of C-source-limited fermentations, standard methods result in a high L-rhamnose consumption (feeding) and thus high medium costs. Since the method according to the invention has a low L-rhamnose requirement (
4.	The regulation of the system described proved to be extraordinarily tight and continued to provide maximum induction even at very low concentrations of the inductor L-rhamnose of up to 0.05 g/1, while no promoter activity whatsoever was detected in the absence of the inductor. Thus, the system is also outstandingly suitable for the expression of potentially toxic proteins and makes possible an inexpensive production, in particular under industrial conditions, since only low L-rhamnose concentrations are required.<br>
For the purposes of the present invention, "prokaryotic host cell" or "prokaryotic host organism" means Gram-positive or Gram-negative bacteria, but in particular those Gram-positive or<br><br>
Gram-negative bacteria which are naturally capable of metabolizing L-rhamnose as carbon source. L-Rhamnose can be utilized as carbon source by most prokaryotic organisms.<br>
Preferably, prokaryotic host cell or prokaryotic host organism means all genera and species of the Enterobacteriaceae and the families Actinomycetales, very especially preferably the Enterobacteriaceae species Escherichia, Serratia, Proteus, Enterobacter, Klebsiella, Salmonella, Shigella, Edwardsielle, Citrobacter, Morganella, Providencia and Yersinia..<br>
Furthermore preferred are the species Pseudomonas, Burkholderia, Nocardia, Acetobacter, Gluconobacter, Corynebacterium, Brevibacterium, Bacillus, Clostridium, Cyanobacter, Staphylococcus, Aerobacter, Alcaligenes, Rhodococcus and Penicillium.<br>
Most preferred are Escherichia species, in particular Escherichia coli.<br>
"L-Rhamnose-inducible promoter" generally means all those promoters which have a higher expression activity in the presence of L-rhamnose than in the absence of L-rhamnose. Expression in the presence of L-rhamnose is at least twice as high, preferably at least five times as high, very especially preferably at least ten times as high, most preferably at least one hundred times as high as in the absence of L-rhamnose. Nucleic acid sequences which are preferably employed for the purposes of determining the expression level are those nucleic acid sequences in functional linkage with the promoter to be tested which encode readily quantifiable proteins. Very especially preferred in this context are reporter proteins (Schenborn E, Groskreutz D (1999) Mol Biotechnol 13(1): 29-44) such as "green fluorescence protein" (GFP) (Chui WL et al. (1996) Curr Biol 6:325-330; Leffel SM et al. (1997) Biotechniques 23(5):912-8), chloramphenicol transferase, luciferase (Millar et al. (1992) Plant Mol Biol Rep 10:324-414), β -glucuronidase or β —galactosidase.<br>
In this context, the L-rhamnose concentration in the medium can generally be in the range of from approximately 0.0001 g/1 to approximately 50 g/1, preferably 0.001 g/1 to 5 g/1, especially preferably 0.01 g/1 to 0.5 g/1.<br>
Especially preferred is the rhaPBAD promoter from the L-rhamnose operon rhaBAD in E. coli (Egan &amp; Schleif (1994) J Mol Biol 243:821-829) and its functional equivalents from other<br><br>
brokaryotic organisms, in particular organisms of tne Snterobacteriaceae family.<br>
fery especially preferred promoters are those which comprise at Least one RhaS binding element as shown in SEQ ID NO: 5 or a Eunctional equivalent thereof, and also a functionally equivalent Eragment of the above.<br>
Especially preferred promoters are those which comprise a sequence as shown in SEQ ID NO: 2, 3 or 4 and functional equivalents thereof, and also functional equivalent fragments of the above.<br>
Functional equivalents to a promoter comprising a sequence as shown in SEQ ID NO: 2, 3, 4 or 5 preferably comprise those promoters which<br>
a)	have essentially the same promoter activity as the promoter comprising a sequence as shown in SEQ ID NO: 2, 3, 4 or 5 and<br>
b)	have at least 50%, preferably 70%, by preference at least 80%, especially preferably at least 90%, very especially preferably at least 95%, most preferably 99% homology with the sequence of said promoter, where the homology extends over a length of at least 30 base pairs, preferably at least 50 base pairs, especially preferably at least 100 base pairs.<br>
Functional equivalents to a promoter comprising a sequence as shown in SEQ ID NO: 2, 3, 4 or 5 means in particular natural or artificial mutations of said promoter and homology sequences and functionally equivalent sequences from other organisms, preferably from other prokaryotic organisms, in particular organisms of the Enterobacteriaceae family, which have essentially the same promoter activity as said promoter.<br>
"Essentially the same promoter activity" means the inducibility of the expression activity by L-rhamnose in accordance with the above general definition for L-rhamnose-inducible promoters.<br>
As described above, the RhaR protein binds to the rhaPRs promoter in the presence of L-rhamnose and initiates its own expression as well as the expression of RhaS. RhaS, in turn, binds to the rhaPBAD promoter, with L-rhamnose as effector, and now activates the rhaPBAD promoter and thus the transcription of the nucleic acid sequences regulated by said promoter. This upstream regulatory unit — consisting of RhaR, RhaS and the rhaPRS promoter — can be provided naturally by the prokaryotic host<br><br>
organism, inserted into the genome of the latter by recombinant methods, or else be provided by means of the DNA construct employed within the scope of the invention. One promoter cassette which is suitable in this context is the sequence described by SEQ ID NO: 1.<br>
If the L-rhamnose uptake required for induction in the cell should be insufficient, it may be advantageous in organisms which, for example, naturally express no L-rhamnose transporter, to transgenically express the latter. However, experience to date shows that the active rhamnose transport should not represent the limiting factor for the efficiency of the expression system according to the invention.<br>
"L-Rhamnose isomerase" generally means all those proteins which are capable of converting L-rhamnose into a different hexose. Preferably, L-rhamnose isomerase means proteins which are capable of converting L-rhamnose into L-rhamnulose (EC 5.3.1.14). Especially preferred is the RhaA gene from organisms of the Enterobacteriaceae family, in particular E.coli. Most preferably, L-rhamnose isomerase means the protein as shown in SEQ ID NO: 9 and homologous sequences from other organisms, preferably from other prokaryotic organisms.<br>
Functional equivalent to the L-rhamnose isomerase as shown in SEQ ID NO: 9 preferably comprises those sequences which<br>
a)	have essentially the same enzyme activity as the L-rhamnose isomerase as shown in SEQ ID NO: 9 and<br>
b)	have at least 50%, preferably 70%, by preference at least 80%, especially preferably at least 90 %, very especially preferably at least 95%, most preferably 99% homology with the sequence of the L-rhamnose isomerase as shown in SEQ ID NO: 9, where the homology extends over a length of at least 30 amino acids, preferably at least 50 amino acids, especially preferably at least 100 amino acids, very especially preferably at least 200 amino acids, most preferably over the entire length of the protein.<br>
Besides the L-rhamnose isomerase, further deficiencies with regard to genes which have a function in the metabolization of L-rhamnose may also be present. Deficiencies which may be mentioned in particular in this context are rhamnulose 1-phosphatase/kinase deficiency (e.g. RhaB; for example described by SEQ ID NO: 11), a rhamnulophosphate aldolase deficiency (e.g. RhaD; for example described by SEQ ID NO: 13) or a deficiency in<br><br>
at least one regulatory element which controls the expression of the abovementioned proteins (such as, for example, promoter, regulator or similar).<br>
Under certain circumstances, it can furthermore be advantageous to generate a deficiency in an active rhamnose transport system (e.g. RhaT; for example described by SEQ ID NO: 19).<br>
"Deficiency" with regard to an L-rhamnose isomerase or another enzyme of L-rhamnose uptake/metabolization means the essentially complete inhibition or blocking of the expression of the target gene in question or of the mRNA derived therefrom and/or of the protein product encoded thereby or the modification of the protein sequence of the gene product in such a manner that its function and/or activity is essentially inhibited or modified in such a way that L-rhamnose can essentially no longer be converted, this inhibition or blocking being based on different cell-biological mechanisms.<br>
Inhibition or blocking for the purposes of the invention comprises in particular the quantitative reduction of an mRNA expressed by the target gene and/or of the protein product encoded thereby down to an essentially complete absence thereof. In this context, the expression, in a cell or an organism, of a certain mRNA and/or of the protein product included thereby is preferably reduced by more than 50%, especially preferably by more than 80%, very especially preferably by more than 90%, most preferably by more than 95% in comparison with the same cell or organism which have not been subjected to the method. Very especially preferably, reduction means the complete inactivation of an endogenous gene (knock-out mutation).<br>
Inhibition or blocking can be based on different mechanisms. Preferably, inhibition or blocking are based on a mutation of the target gene in question, it being possible for the mutation to consist in a substitution, deletion and/or addition of one or more nucleotide(s). Especially preferred is an inhibition or blocking by means of transposon-aided mutagenesis or by means of site-specific knock-out.<br>
The reduction can be determined by methods with which the skilled worker is familiar. Thus, the reduction of the protein quantity can be determined for example by an immunological detection of the protein. Furthermore, it is possible to employ biochemical techniques such as Northern hybridization, nuclease protection assay, reverse transcription (quantitative RT-PCR), ELISA (enzyme linked immunosorbent assay), Western blotting, radioimmunoassay<br><br>
(RIA) or other immunoassays and fluorescence-activated cell analysis (FACS). Depending on the type of the produced protein product, the activity of the latter, or the influence of the phenotype of the organism or the cell, may also be determined.<br>
"Protein quantity" means the amount of a particular polypeptide in an organism, a tissue, a cell or a.cell compartment.<br>
"Reduction" of the protein quantity means the reduction of the amount of a particular polypeptide in an organism, a tissue, a cell or a cell compartment in comparison with the wild type of the same genus and species to which this method has not been applied, under otherwise identical framework conditions (such as, for example, culture conditions, age, nutrient supply and the like). In this context, the reduction amounts to at least 50%, preferably at least 70%, especially preferably at least 90%, very especially preferably at least 95%, most preferably at least 99%. Methods for determining the protein quantity are known to the skilled worker. Examples which may be mentioned are: the micro-Biuret method (Goa J (1953) Scand J Clin Lab Invest 5:218-222), the Folin-Ciocalteu method (Lowry OH et al. (1951) J Biol Chem 193:265-275) or measuring the adsorption of CBB G-250 (Bradford MM (1976) Analyt Biochem 72:248-254).<br>
The reduction of the L-rhamnose isomerase activity can be determined in particular by means of enzymatic assay systems. Suitable assay systems are known to the skilled worker (Bhuiyan SH et al. (1997) J Ferment Bioeng 84(4):319-323).<br>
"DNA construct which is capable of episomal replication in prokaryotic host cells" means all those DNA constructs which differ from the chromosomal DNA of said host cell and which exist in parallel with the former in said host cell and are capable of replicating in said host cell using homologous or other replication mechanisms (for example replication mechanisms which are encoded via the DNA construct itself). The DNA construct can constitute a single- or double-stranded DNA structure. Preferably, the DNA construct has a double-stranded DNA structure at least some of the time (for example at a point in time during its replication cycle).<br>
Preferably, said DNA constructs which are capable of episomal replication are present in the host cell in a copy number of at least 1, preferably at least 5, especially preferably at least 10.<br><br>
"Selection of prokaryotic host cells comprising said DNA construct in episomal form" means choosing host cells comprising said DNA construct in episomal form. They can be chosen for example using a selection marker described hereinbelow. Preferably, the DNA construct does not insert into the chromosomal DNA of the host cell. This can be prevented for example by the DNA construct lacking sequences which are identical with chromosomal sequences of the host cell over a substantial section.<br>
Preferably, said DNA constructs which are capable of episomal replication have a size/length of no more than 100 000 bases or base pairs, especially preferably no more than 50 000 bases or base pairs, very especially preferably 10 000 bases or base pairs (the number of bases or base pairs depends on whether the DNA construct is a single- or double-stranded DNA structure).<br>
The DNA construct is preferably a vector. By way of example, vectors can be plasmids, cosmids, phages, viruses, retroviruses or else agrobacteria. The vector is preferably a circular plasmid which comprises the nucleic acid sequence to be expressed in recombinant form and capable of autonomously replicating in the prokaryotic host cell. Within the scope of the present invention, vector can also be referred to as recombinant vector or recombinant expression vector. The skilled worker is familiar with various sequences which permit the replication of DNA in prokaryotes. Examples which may be mentioned are ORI (origin of DNA replication), the pBR322 ori or the P15A ori (Sambrook et al.: Molecular Cloning. A Laboratory Manual, 2nd ed. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989).<br>
Correspondingly suitable replication origins which ensure a low copy number can be isolated from BACs (bacterial artificial chromosomes), F-plasmids, cosmids such as, for example, pWE15. Correspondingly suitable replication origins which ensure a medium copy number can be isolated for example from pBR322 (Lin-Chao S, Bremer H, Mol Gen Genet 1986 203(1): 143-149) and derivatives such as the pJOE series, pKK223-3, pQE30, pQE40 or plasmids with an Rl origin such as pRSFlOlO and derivatives such as, for example, pMLl22, pl5A, pSClOl. Correspondingly suitable replication origins which ensure a high copy number can be isolated for example from phagemids such as pBluescript II SK/KS+/-, pGEM etc. The copy number which is present in a cell in each case is determined in part by what is known as the replication origin (also referred to as replicon). Plasmids of the pBR322 series comprise the ColEl replication origin from pMBl. This replication origin is relatively tightly regulated and<br><br>
results in a copy number of approximately 25 per cell. pUC plasmids comprise a mutated ColEl version and can be present as 200 to 700 plasmid copies per cell. Some plasmids comprise the pl5A replication origin, which results in a low copy number.<br>
Examples of vectors which may be mentioned:<br>
a)	the following are preferred in E.coli: pQE70, pQE60 and pQE-9 (QIAGEN, Inc.); pBluescript vectors, Phagescript vectors, pNH8A, pNH16a, pNH18A, pNH46A (Stratagene Cloning Systems, Inc.); ptrc99a, pKK223-3, pKK233-3, pDR540, pRIT5 (Pharmacia Biotech, Inc.); pLG338, pACYC184, pBR322, pUC18, pUC19, pKC30, pRep4, pHSl, pHS2, pPLc236, pMBL24, pLG200, pUR290, pIN-III113-Bl, Xgtll or pBdCI,<br>
b)	the following are preferred in Streptomyces: pIJlOl, pU364, pIJ702 or pIJ361,<br>
c)	the following are preferred in Bacillus: pUBHO, pC194 or pBD214,<br>
d)	in Corynebacterium: pSA77 or pAJ667,<br>
or derivatives of the abovementioned plasmids. The plasmids mentioned are a small selection of the plasmids which are possible. Further plasmids are well known to the skilled worker and can be found for example in the book Cloning Vektors (Eds. Pouwels P. H. et al. Elsevier, Amsterdam-New York-Oxford, 1985, ISBN 0 444 904018).<br>
"Transformation" or "transformed" means the introduction of genetic material such as, for example, a vector (for example a plasmid) into a prokaryotic host cell. The skilled worker has available for this purpose a variety of methods described in detail hereinbelow. A prokaryotic host cell into which said genetic material has been introduced, and also the "progeny" and colonies resulting from this cell and which comprise said genetic material are referred as to "transformants".<br>
"Transduction" or "transduced" means the introduction of genetic material into a prokaryotic host cell starting from the genetic material of a bacteriophage. A prokaryotic host cell into which said genetic material has been introduced, and also the "progeny" and colonies resulting from this cell and which comprise said genetic material are referred as to "transductants".<br><br>
"Recombinant protein" means any protein product which, starting from the nucleic acid sequence to be expressed, can be expressed under the functional control of the L-rhamnose-inducible promoter and includes peptides, polypeptides, proteins, oligoproteins and/or fusion proteins. "Recombinant protein" preferably means a protein of microbial, bacterial, animal or vegetable origin.<br>
"Fusion proteins" means a fusion of the desired protein and leader sequences which make possible an expression in specific compartments (for example periplasm or cytoplasm) of the host cell or into the surrounding medium. An example which may be mentioned is the pelB leader sequence (US 5,576,195; US 5,846,818).<br>
"Expression cassette" means in each case the combination of a promoter with at least one nucleic acid sequence which can be transcribed under the control of the former.<br>
"Heterologous" with regard to the ratio of the<br>
L-rhamnose-inducible promoter and the nucleic acid sequence to be expressed under the control of said promoter, or an expression cassette or an expression vector, means all those constructs which have been generated by recombinant methods in which either<br>
a)	at least one of the nucleic acid sequences to be expressed, or<br>
b)	at least one of the L-rhamnose-inducible promoters which controls the expression of said nucleic acid sequence to be expressed, or<br>
c)	(a) and (b)<br>
are not in their natural genetic environment (for example at their natural chromosomal locus) or have been modified by recombinant methods, it being possible for the modification to comprise, for example, substitutions, additions, deletions, inversions or insertions of one or more nucleotide residues.<br>
In the method according to the invention, the prokaryotic host cells according to the invention are grown in a medium which permits the growth of these organisms. This medium may be a synthetic or a natural medium. Depending on the organism, media known to the skilled worker are used. To allow microbial growth, the media used comprise a carbon source, a nitrogen source,<br><br>
inorganic salts and, if appropriate, minor amounts of vitamins and trace elements.<br>
Advantageous carbon sources are, for example, polyols such as glycerol, sugars such as mono-, di- or polysaccharides such as glucose, fructose, mannose, xylose, galactose, ribose, sorbose, ribulose, lactose, maltose, sucrose, raffinose, starch or cellulose, complex sugar sources such as molasses, sugar phosphates such as fructose-l,6-bisphosphate, sugar alcohols such as mannitol, alcohols such as methanol or ethanol, carboxylic acids such as citric acid, lactic acid or acetic acid, fats such as soya oil or rapeseed oil, amino acids such as a mixture of amino acids, for example so-called casamino acids (Difco), or individual amino acids such as glycine or aspartic acid or amino sugars which may simultaneously also be used as the nitrogen source. Especially preferred are polyols, in particular glycerol.<br>
The medium employed as basal medium should preferably not comprise L-rhamnose to ensure the tightest possible expressional regulation. If required, L-rhamnose is then added at the desired point in time or cell density and in the concentration desired in each case.<br>
Advantageous nitrogen sources are organic or inorganic nitrogen compounds or materials which comprise these compounds. Examples are ammonium salts such as NH4C1 or (NH4)2S04, nitrates, urea or complex nitrogen sources such as cornsteep liquor, brewer's yeast autolyzate, soybean flour, wheat gluten, yeast extract, meat extract, casein hydrolyzate, yeast or potato protein, all of which can frequently also act as the nitrogen source.<br>
Examples of inorganic salts are the salts of calcium, magnesium, sodium, cobalt, molybdenum, manganese, potassium, zinc, copper and iron. Anions of these salts to be mentioned are, in particular, the chloride, sulfate and phosphate ion. An important factor for increasing the productivity in the method according to the invention is the control of the Fe2+-or Fe3+ ion concentration in the production medium.<br>
If appropriate, other growth factors are added to the nutrient medium, such as, for example, vitamins or growth promoters such as biotin, 2-KLG, thiamin, folic acid, nicotinic acid, pantothenate or pyridoxin, amino acids such as alanine, cysteine, proline, aspartic acid, glutamine, serine, phenylalanine, ornithine or valine, carboxylic acids such as citric acid, formic<br><br>
acid, pimelic acid or lactic acid, or substances such as dithiothreitol.<br>
The mixing ratio of said nutrients depends on the type of fermentation and is decided for each individual case. All of the components of the medium may be introduced into the fermentation vessel at the beginning of the fermentation, if appropriate after having been sterilized separately or jointly, or else they may be fed continuously or batchwise during the fermentation, as required.<br>
The culture conditions are specified in such a way that the organisms' growth is optimal and that the best possible yields are achieved (this can be determined for example on the basis of the activity level of the recombinant protein expressed) . Preferred culture temperatures are at 15°C to 40°C. Temperatures between 25 °C and 37 °C are especially advantageous. The pH is preferably maintained in a range of from 3 to 9. pH values of between 5 and 8 are especially advantagous. In general, an incubation time of a few hours to several days, preferably 8 hours up to 21 days, especially preferably 4 hours to 14 days, will suffice. The maximum amount of product accumulates in the medium within this period.<br>
Advantageous media optimization can be found by the skilled worker for example in the textbook Applied Microbiol. Physiology, "A Practical Approach (Eds. PM Rhodes, PF Stanbury, IRL-Press, 1997, pages 53 - 73, ISBN 0 19 963577 3).<br>
The method according to the invention can be carried out continuously or discontinuously, batchwise or fed-batch-wise.<br>
"Mutation" or "mutations" means the substitution, addition, deletion, inversion or insertion of one or more amino acid residue(s) or base(s)/base pair(s).<br>
"Homology" between two nucleic acid sequences means the identity of the nucleic acid sequence over in each case the sequence length indicated, which is calculated by comparison with the aid of the program algorithm GAP (Wisconsin Package Version 10.0, University of Wisconsin, Genetics Computer Group (GCG), Madison, USA; Altschul et al. (1997) Nucleic Acids Res. 25:3389ff), setting the following parameters:<br><br>
Average Match: 10	Average Mismatch: 0<br>
For example, a sequence with at least 50% homology with the sequence SEQ ID NO: 2 at the nucleic acid level is understood as meaning a sequence which, upon alignment with the sequence SEQ ID NO: 2 using the above program algorithm with the above parameter set has at least 50% homology.<br>
"Homology" between two polypeptides means the identity of the amino acid sequence over in each case the sequence length indicated, which is calculated by comparison with the aid of the program algorithm GAP (Wisconsin Package Version 10.0, University of Wisconsin, Genetics Computer Group (GCG), Madison, USA), setting the following parameters:<br>
Gap Weight: 8	Length Weight: 2<br>
Average Match: 2912	Average Mismatch: -2003<br>
For example, a sequence with at least 50% homology with the sequence SEQ ID NO: 9 at the protein level is understood as meaning a sequence which, upon alignment with the sequence SEQ ID NO: 9 using the above program algorithm with the above parameter set has at least 50% homology.<br>
For optimal expression of heterologous genes in organisms, it may be advantageous to modify the nucleic acid sequences in accordance with the specific codon usage of the organism. The codon usage can easily be established on the basis of computer analyses of other, known genes of the organism in question.<br>
The DNA construct which comprises the L-rhamnose-inducible promoter and the nucleic acid sequence to be expressed under its control ensures the transcription and/or translation of said nucleic acid sequence as the result of a functional linkage of said promoter and said nucleic acid sequence.<br>
A functional linkage is generally understood as meaning an arrangement in which a genetic control sequence can exert its function with regard to the nucleic acid sequence to be expressed. In this context, function can mean, for example, expressional control, i.e. transcription and/or translation of the nucleic acid sequence. In this context, control comprises for example the initiation, enhancement, control or suppression of expression, i.e. transcription and, if appropriate, translation. A functional linkage is understood as meaning, for example, the sequential arrangement of a promoter, the nucleic acid sequence<br><br>
to be expressed and, if appropriate, further regulatory elements such as, for example, a terminator, in such a way that each of the regulatory elements can fulfill its function when the nucleic icid sequence is expressed. The skilled worker is familiar with various ways of arriving at one of the DNA constructs according to the invention. The construction can be carried out by means of customary recombination and cloning techniques as are described, Eor example, in T Maniatis, EF Fritsch and J Sambrook, Molecular Cloning: A Laboratory Manual, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY (1989) and in TJ Silhavy, ML Berman and LW Snquist, Experiments with Gene Fusions, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY (1984) and in Ausubel, FM et al., Current Protocols in Molecular Biology, Greene Publishing issoc. and Wiley Interscience (1987).<br>
Said DNA construct can comprise further functional elements. The concept of the functional elements is to be interpreted broadly md means all those sequences which have an effect on the production, the multiplication or the function of the DNA constructs or organisms according to the invention. Functional elements ensure, enhance, regulate or modify for example the cranscription and, if appropriate, translation in corresponding lost organisms.<br>
Tunction elements are described for example in "Goeddel; Gene Expression Technology: Methods in Enzymology 185, Academic Press, 5an Diego, CA (1990)" or "Gruber and Crosby, in: Methods in Plant tolecular Biology and Biotechnolgy, CRC Press, Boca Raton, Tlorida, eds.: Glick and Thompson, Chapter 7, 89-108" and the references cited therein. Depending on the host organism or starting organism described hereinbelow in greater detail, which Ls converted into a genetically modified or transgenic organism Dy introducing the expression cassettes or vectors, different control sequences are suitable.<br>
"Genetic control sequences" comprise for example the<br>
5'-untranslated region or the noncoding 3' region of genes.<br>
"Genetic control sequences" furthermore mean sequences which<br>
2ncode fusion proteins consisting of a signal peptide sequence.<br>
The following may be mentioned by way of example but not by<br>
Limitation:<br>
a)  Selection markers<br>
As a rule, selection markers are necessary for selecting successfully transformed cells and preventing the loss of the DNA construct from the host cell in the course of time and while cell division takes place. Such a loss can occur in<br><br>
particular when the recombinant protein encoded by the nucleic acid sequence to be expressed has a toxic effect on the prokaryotic organism. The selectable marker which is introduced together with the expression construct confers a resistance to a biocide (for example an antibiotic such as, for example, ampicillin, kanamycin or hygromycin) to the successfully transformed cells. Examples of selection markers which may be mentioned are:<br>
-	Amp	(ampicillin resistance; β lactamase)<br>
-	Cab	(carbenicillin resistance)<br>
-	Cam	(chloramphenicol resistance)<br>
-	Kan	(kanamycin resistance)<br>
-	Rif	(rifampicin resistance)<br>
-	Tet	(tetracyclin resistance)<br>
-	Zeo	(zeocin resistance)<br>
-	Spec (spectinomycin)<br>
The selection pressure is maintained by suitable amounts of the antibiotic. Examples which may be mentioned are: ampicillin 100 mg/1, carbenicillin 100 mg/1, chloramphenicol 35 mg/1, kanamycin 30 mg/1, rifampicin 200 mg/1, tetracyclin 12.5 mg/1, spectinomycin 50 mg/1.<br>
Selection markers furthermore comprise those genes and gene products which make possible a selection of a suitably transformed host cell, for example by complementing a genetic deficiency in amino acid or nucleotide synthesis. Generally, media which do not comprise said amino acid or nucleotide unit are employed for this purpose. The skilled worker is familiar with a variety of such systems. Examples which may be mentioned are the deficiencies in the biosynthesis of tryptophan (for example trpC), leucine (for example leuB), histidine (for example hisB) as they are present, for example, in E.coli strain KC8 (Clontech). These deficiencies can be complemented, inter alia, by the selectable markers TRP1, Leu2 and HIS3.<br>
Transcription terminators<br>
The transcription terminator reduces unwanted transcription<br>
and increases the plasmid and mRNA stability.<br>
Shine-Dalgarno sequences<br>
A Shine-Dalgarno (SD) sequence is required for initiating translation and is complementary to the 3' end of the 16S ribosomal RNA. The efficiency of initiating translation at the start codon depends on the actual sequence. A suitable<br><br>
consensus sequence for E.coli is, for example, 5'-TAAGGAGG-3'. It is located approximately 4 to 14 nucleotides upstream of the start codon, the optimum being 8 nucleotides. To avoid the formation of secondary structures (which may reduce expression), this region should preferably be rich in A/T nucleotides.<br>
Start codon<br>
The start codon is the point at which translation is initiated. In E. coli, ATG is the most widely used start codon; as an alternative GTG may also be used.<br>
"Tags" and fusion proteins<br>
N- or C-terminal fusions between recombinant proteins to be expressed and shorter peptides ("tags") or other proteins (fusion partners) may be advantageous. For example, they may make possible an improved expression, solubility, detectability and purification. Preferably, such fusions are combined with protease cleavage sequences (for example for thrombin or factor X), which make possible a removal of the "tag" or the fusion partner after expression and purification has taken place.<br>
Multiple cloning regions (multiple cloning sites; MCS) permit and facilitate the insertion of one or more nucleic acid sequences.<br>
Stop codon / translation terminators<br>
Of the three possible stop codons, TAA is preferred since TAG and TGA can, under some circumstances, result in a read-through without terminating the translation. To ensure reliable termination, it is also possible to employ a plurality of stop codons in sequence.<br>
Reporter genes<br>
Reporter genes encode readily quantifiable proteins which ensure an assessment of the transformation efficiency, the expression level and the place or time of expression via their intrinsic color or enzyme activity. Reporter genes can, for example, encode the following proteins: hydrolases, fluorescence proteins, bioluminescence proteins, glucosidases or peroxidases. Preferred are luciferases, β galactosidases, β -glucuronidase, green fluorescence protein,<br>
acetyltransferases, phosphotransferases or adenyltransferases (see also Schenborn E, Groskreutz D (1999) Mol Biotechnol 13(l):29-44).<br><br>
In the case of selection markers or reporter proteins, the nucleic acid sequence encoding said proteins is preferably linked functionally with a promoter which is functional in the prokaryotic host organism in question and, if appropriate, further control sequences to give an expression cassette. Advantageous promoters and control sequences are generally known to the skilled worker. Examples which may be mentioned are promoters such as the cos, tac, trp, tet, Ipp, lac, laclq, T7, T5, T3, gal, trc, ara, SP6, yt-PR or y-PL promoter.<br>
The production of a transformed host cell or a transformed host organism requires introduction of the DNA in question (for example one of the expression cassettes or vectors according to the invention) into the host cell in question.  A large number of methods is available for this process, which is referred to as transformation (see also Keown et al. (1990) Methods in Enzymology 185:527-537). Thus, the DNA can be introduced for example directly by means of microinjection, electroporation or by bombardment with DNα-coated microparticles (biolistic methods with the gene gun; particle bombardment). Also, the cell can be permeabilized chemically, for example with polyethylene glycol, so that the DNA can enter the cell by diffusion. The DNA can also take place by means of fusion with other DNα-comprising units such as minicells, cells, lysosomes or liposomes. Electroporation is another suitable method for introducing DNA, in which the cells are reversibly permeabilized by an electrical pulse. Preferred general methods which may be mentioned are calcium-phosphate-mediated transformation, DEAE-dextran-mediated transformation, cationic lipid-mediated transformation, electroporation, transduction, infection. Such methods are known to the skilled worker and described by way of example (Davis et al. (1986) Basic Methods In Molecular Biology; Sambrook J et al. (1989) Molecular cloning: A laboratory manual, Cold Spring Harbor Laboratory Press; Ausubel FM et al. (1994) Current protocols in molecular biology, John Wiley and Sons; Glover DM et al. (1995) DNA Cloning Vol.1, IRL Press ISBN 019-963476-9).<br>
Transformed cells, i.e. those which comprise the DNA which has been introduced, can be selected from untransformed cells when a selectable marker is part of the DNA which has been introduced. Various selection markers are described above.<br>
The method according to the invention is not limited regarding the nature and sequence of the nucleic acid sequence to be expressed, or of the recombinant protein expressed on the basis thereof. The nucleic acid sequences to be expressed under the control of the L-rhamnose-inducible promoter can be diverse. In<br><br>
this context, expression means transcription and, if appropriate, translation. Besides the expression of nucleic acid sequences which encode recombinant proteins, it is also possible to express nucleic acid sequences which, for example, bring about the transcription of an antisense RNA and thus reduce the expression of an endogenous gene of the prokaryotic host cell. It is possible to express sequences of prokaryotic, but also of eukaryotic origin. It is preferred to express sequences which encode recombinant proteins which are to be produced in substantial quantities. The following may be mentioned by way of example, but not by limitation:<br>
a)  enzymes such as, for example, chymosin, proteases,<br>
polymerases, saccharidases, dehydrogenases, nucleases, glucanases, glucose oxidase, α-amylase, oxidoreductases (such as peroxidases or laccases), xylanases, phytases, cellulases, collagenases, hemicellulases and lipases. Especially preferred are<br>
enzymes as are used in laundry detergents or other<br>
detergents such as, for example, horseradish peroxidase,<br>
proteases, amylases, lipases, esterases or cellulases<br>
enzymes as are used in the food industry such as<br>
proteases, lipases, lactases, β -glucanase, cellulases or<br>
pectinases<br>
enzymes as are employed in industrial processes such as<br>
lipases, α-amylases, amyloglucosidases, glucoamylases,<br>
pullulanases, glucose isomerases,<br>
enzymes as are employed in industrial processes for the<br>
production of chemicals and fine chemicals such as<br>
lipases, amidases, nitrile hydratases, esterases or<br>
nitrilases<br>
enzymes as are employed in animal nutrition such as<br>
β -glucanases<br>
enzymes as are employed in papermaking or in the leather<br>
industry such as amylases, collagenases, cellulases or<br>
xylanases.<br>
D)  mammalian proteins such as, for example, blood proteins (for example serum albumin, factor VII, factor VIII, factor IX, factor X, tissue plasminogen factor, protein C, von Willebrand factor, anti-thrombin 111 or erythropoietin), colony stimulating factors (CFS) (for example granulocyte colony-stimulating factor (G-CSF),  macrophage colony-stimulating factor (M-CSF) or granulocyte macrophage colony-stimulating factor (GM-CSF)), cytokins (for example interleukins), integrins, addressins, selectins, antibodies<br><br>
or antibody fragments, structural proteins (for example collagen, fibroin, elastin, tubulin, actin or myosin), growth factors, cell-cycle proteins, vaccines, fibrinogen, thrombin, insulins.<br>
The nucleic acid sequence to be expressed especially preferably encodes a recombinant protein selected from the group consisting of chymosines, proteases, polymerasen, saccharidases, dehydrogenases, nucleases, glucanases, glucose oxidases, ct-amylases, oxidoreductases, peroxidases, laccases, xylanases, phytases, cellulases, collagenases, hemicellulases, lipases, lactases, pectinases, amyloglucosidases, glucoamylases, pullulanases, glucose isomerases, nitrilases, esterases, nitrile hydratases, amidases, oxygenases, oxynitrilases, lyases, lactonases, carboxylases, collagenases, cellulases, serum albumins, factor VII, factor VIII, factor IX, factor X, tissue plasminogen factors, protein C, von Willebrand factors, antithrombins, erythropoietins, colony-stimulating factors, cytokins, interleukins, insulins, integrins, addressins, selectins, antibodies, antibody fragments, structural proteins, collagen, fibroins, elastins, tubulins, actins, myosins, growth factors, cell-cycle proteins, vaccines, fibrinogens and thrombins.<br>
In a preferred embodiment, the recombinant protein is a nitriliase, preferably a nitrilase described by an amino acid sequence which is encoded by a nucleic acid sequence selected from the group consisting of<br>
a)	a nucleic acid sequence with the sequence shown in SEQ ID NO: 6,<br>
b)	nucleic acid sequences which, owing to the degeneracy of the genetic code, are derived from the nucleic acid sequence shown in SEQ ID NO: 6,<br>
c)	derivatives of the nucleic acid sequence shown in SEQ ID NO: 6 which encode polypeptides with the amino acid sequences shown in SEQ ID NO: 7 and which have at least 35% homology at the amino acid level without the enzymatic activity of the polypeptides being substantially reduced.<br>
A further aspect of the invention relates to the use of the above-described host cells or host organisms according to the invention for the production of foodstuffs, feedstuffs, pharmaceuticals or fine chemicals. Fine chemicals preferably means proteins, enzymes, vitamins, amino acids, sugars, fatty<br><br>
acids, natural and synthetic flavorings, aroma chemicals and colorants•<br>
The invention furthermore relates to methods for the production of recombinant proteins, enzymes and other fine chemicals such as, for example, aldehydes, ketones or carboxylic acids (preferably chiral carboxylic acids) using one of the prokaryotic host cells according to the invention or a preparations thereof. The preferred proteins and enzymes are detailed hereinabove.<br>
In this context, the prokaryotic host cell can be present in a growing, quiescent, immobilized or disrupted state. Disrupted cells are understood as meaning, for example, cells which have been made permeable via treatment with, for example, solvents, or cells which have been disrupted via an enzymatic treatment, a mechanical treatment (for example French press or sonication) or via any other method. The resultant crude extracts are advantageously suitable for the method according to the invention. Partially purified enzyme preparations may also be used for the method. Immobilized microorganisms or enzymes which can advantageously be used in the reaction are likewise suitable.<br>
A further aspect of the invention relates to methods for the production of chiral carboxylic acids, where a racemic nitrile (or, as an alternative, its precursors aldehyde and hydrocyanic acid/cyanide salt) is converted into said chiral carboxylic acid by treatment with a prokaryotic host cell which is at least deficient with regard to one L-rhamnose isomerase and comprises at least one DNA construct which can replicate in said host cell and which comprises a nucleic acid sequence encoding a nitrilase under the transcriptional control of an L-rhamnose-inducible promoter, where said promoter is heterologous with regard to said nucleic acid sequence.<br>
The nucleic acid sequence which encodes the nitrilase is preferably selected from the group of the above-shown sequences which encode nitrilases.<br>
Zhiral carboxylic acids are sought-after compounds for organic synthetic chemistry. They are starting materials for a nultiplicity of pharmaceutical active ingredients or active ingredients for crop protection. Chiral carboxylic acids can be ased for traditional racemate resolution via diastereomer salts. Thus, for example, R-(-)- or S-(-)-mandelic acid is employed for the racemate resolution of racemic amines. R-(-)-Mandelic acid is furthermore used as intermediate for synthesis purposes.<br><br>
In a preferred embodiment, the chiral carboxylic acids of the general formula I are prepared starting from a racemic nitrile of bhe general formula II.<br><br>
R1, R2, R3 are independently of one another hydrogen, substituted or unsubstituted, branched or unbranched Cl-C10-alkyl-, C2-C10-alkenyl-, substituted or unsubstituted aryl-, hetaryl-, OR4 or NR4R5 and where the radicals R1, R2 and R3 are always different,<br>
R4  is hydrogen, substituted or unsubstituted, branched or unbranched Cl-C10-alkyl-, C2-C10-alkenyl-, Cl-C10-alkylcarbonyl-, C2-C10-alkenylcarbonyl-, aryl-, arylcarbonyl-, hetaryl- or hetarylcarbonyl-,<br>
R5  is hydrogen, substituted or unsubstituted, branched or<br>
unbranched Cl-C10-alkyl-, C2-C10-alkenyl-, aryl- or hetaryl-.<br>
Most preferred as the nitrile are mandelonitrile, o-chloromandelonitrile, β chloromandelonitrile or m-chloromandelonitrile. Most preferred as the chiral carboxylic acid are R-mandelic acid, S-mandelic acid, R-β chloromandelic acid, S-β chloromandelic acid, R-m-chloromandelic acid, S-m-chloromandelic acid, R-o-chloromandelic acid or S-o-chloroirvandelic acid.<br>
Details for carrying out these conversions or for purifying the products and the like are described in detail for example in WO 00/23577. The starting materials, products and process parameters described therein are expressly referred to.<br>
Examples<br>
General nucleic acid methods such as, for example, cloning, restriction cleavages, agarose gel electrophoresis, linking DNA fragments, transformation of microorganisms, bacterial cultures and sequence analysis of recombinant DNA were carried out as described by Sambrook et al. (1989) (Cold Spring Harbor<br><br>
Laboratory Press: ISBN 0-87969-309-6), unless otherwise specified. Recombinant DNA molecules were sequenced with an ABI laser fluorescence DNA sequencer following the method of Sanger (Sanger et al. (1977) Proc Natl Acad Sci USA 74:5463-5467). To avoid polymerase errors in constructs to be expressed, fragments resulting from a polymerase chain reaction were sequenced and verified.<br>
Examples 1: Characterization of the E.coli strain JB1204<br>
In accordance with the literature, Escherichia coli JB1204 (CGSC6999, Bulawa CE &amp; Raetz CRH (1984) J Biol Chem 259:11257-11264) has a transposon insertion "rhα-14::TnlO", no more detailed information being given of the sequence or function of "rhα-14". JB1204 (a K12 derivative) is inferior to strains such as TGI and W3110 with regard to growth, as the result of a number of other mutations, which is why this strain itself is not used for the production of proteins on an industrial scale.<br>
To test if the strain E.coli JB1204 still metabolizes rhamnose and if the induction of a rhamnose-dependent expression system in E. coli JB1204 is adversely affected, competent JB1204 cells were prepared and transformed with the plasmid pDHEl650, which is a pJOE derivative and carries the gene for a nitrilase under the control of the rhamnose promoter (plasmid corresponds to pDHE19.2 in DE 19848129). After 15 hours of culture at 37°C in LB-ampicillin-tetracyclin with and without rhamnose, the optical density of the cultures was measured and, after the cells had been washed, the nitrilase activity was tested in what is known as the resting-cell assay (see Table 1). When grown in the presence of L-rhamnose, nitrilase expression takes place in JB1204 and in the comparison strain TGI, but this expression does not take place in the absence of L-rhamnose.<br><br>
1,	E.coli JB1204 pDHE1650 in LB Amp Tet;<br>
2,	E.coli TGI pDHE1650 in LB Amp (positive control)<br><br>
Assay conditions:    10 mM Tris-HCl, 6 mM mandelonitrile, 40°C<br>
Analysis:	stop sample with 40 nl of 1M HCl/ml,<br>
remove cells and then analyze by HPLC as<br>
described in DE 19848129.<br>
Example 2: Preparation of the rhamnose-deficient host strain TG10 for the production of recombinant proteins<br>
The strain TGI, which is utilized for the production of recombinant biocatalysts, was modified by PI transduction in such a way that it no longer metabolizes rhamnose, while the rhamnose-induction-based expression system of the pJOE and pDHE vectors continues to function without being adversely affected (name of this new strain derivative: TG10).<br>
The choice of the E.coli strain is important for being able to conduct fermentative methods in an inexpensive manner and in high yields. This is why E.coli TGI, which is known for productive high-density cell fermentations (Korz et al. (1995) J Biotechnol 39:59-65) was chosen as the host strain. The rhamnose deficiency from JB1204 was transferred to TGI pDHE1650 by Pi transduction and selection on 15 fxg/ml tetracyclin (=TG10 pDHE1650= Lul0569).<br>
2.1 Pi transduction protocol for transferring the rhamnose deficiency from JB1204 (rhal4::TnlO) to TGI<br>
a)	Preparation of the donor lysate<br>
Grow the donor, i.e. JB1204, in 3 ml LB-Tet (15 \iq/ml)   for 15<br>
hours at 37°C (preculture).<br>
Incubate 3 ml LB-Tet + 5 mM CaCl2 + 60 β ,1 preculture (=1:50)<br>
up to OD600= 0.3 - 0.5 at 37°C (approx. 45 minutes)<br>
+ 100 µl  (fresh) lysate of phage PI, continue shaking<br>
thoroughly for 10-120 minutes until cell lysis takes place<br>
(clarification, up to 5 hours for old lysate)<br>
+ 60 fil chloroform, vortex for 30 seconds to destroy residual<br>
cells, storage at 4°C.<br>
b)	Infection of the recipient<br>
Grow the recipient, i.e. TGI pDHE1650 (=Xu9682) in 3 ml<br>
LB-Amp for approx. 15 hours at 37°C (preculture)<br>
Incubate 5 ml LB-Amp+ 5 mM CaCl2 + 10 mM MgCl2 + 10 mM MgS04 +<br>
100 ill preculture (=1:50) up to OD600= 0.3 - 0.5 at 37°C<br>
(approx. 30 minutes), remaining preculture on ice<br>
Harvest preculture and main culture, resuspend in 2.5 ml of<br>
LB-Amβ Cα-Mg<br><br>
Treat in each case 2x 100 nl of recipient with 0, 5, 30, 100 µl of donor lysate and incubate together with a control without recipient + 100 µl of donor lysate for 8 minutes and 24 minutes, respectively, without shaking  at 30°C (infection) + 100 µL  1 M sodium citrate pH 7.0, centrifuge for 2 minutes at 7000 rpm, wash 2-3x in 1 ml of 0.1 M citrat buffer pH 7.0 and resuspend, 1 hour 37°C, without shaking Harvest, resuspend in 100 µl 0.02 M sodium citrate pH 7.0 Plate in each case 80 β 1 on LB-Amβ Tet and in each case 10 µl of the mixtures without donor lysate addition on LB-Amp, incubation overnight at 37°C<br>
LB-Amp gives rise to a lawn (control). Pick colonies from LB-Amβ Tet and verify resistances, rhamnose deficiency, rhamnose inducibility and activity.<br>
Also, TGI pDHE1650 pAgro4 pHSG575, the equivalent to TGI pDHE1650 with chaperone coexpression (GroESL), was transduced in parallel (+spectinomycin 50 ng/ml and chloramphenicol 10 β tg/ml in the medium; name TG10 pDHE1650 pAgro4 pHSG575=Lul0571) .<br>
After the clones obtained were cultured overnight in 3 ml of LB/ampicillin/rhamnose (approx. 2 g/1) medium (± tetracyclin 10 ng/ml), the optical densities (X=600 nm) of the cultures were determined. HPLC analysis of the culture supernatants revealed that the resulting E. coli strain TG10 pDHE1650 cannot metabolize rhamnose. The cells were subsequently washed in buffer and assayed for their nitrilase activity in a resting-cell-assay (Table 2).<br>
The rhamnose deficient clones showed a similar nitrile hydrolyzing activity to the corresponding comparison strain (TGlpDHE1650). The rhamnose concentration hardly decreased in the clones.<br><br><br>
The transduction with E. coli TGI pDHE1650 had the advantage of selecting against the original strain JB1204 with ampicillin. However, subsequent work required a plasmid-free host strain, i.e. the plasmid pDHEl650 was to be removed from TG10 pDHE1650 (curing of TG10 pDHE1650). To this end, E. coli TG10 pDHEl650 was inoculated from ice into 3 ml of LB-Tet without  ampicillin and incubated overnight at 37°C. This culture was used to inoculate a 3 ml main culture 1:100 in LB-Tet, which was subjected to a heat<br><br>
shock treatment (2.5 minutes, 42°C). After shaking for 16 hours at 37°C, the ODgoo of the culture was 1.3 (corresponds to approx. 1.3xl09 cells/ml). In each case 100 µl of the dilution steps 10~4 to 10"7 were plated onto LB-Tet, and the resulting colonies (560+140+15+0) were transferred to LB-Tet with  ampicillin by the replica method. A clone which showed weak growth on this medium was again plated onto LB-Amβ Tet. It neither grew on LB-Amβ Tet nor did it show any plasmid DNA following minipreparation (LB-Tet culture). This ampicillin-sensitive clone is named TG10 (=Lul0568) and is used as starting strain for new overexpression strains.<br>
Example 4: Production of recombinant L-pantolactone<br>
hydrolase using the rhamnose-deficient host strain E. coli TG10<br>
Competent E. coli TG10 cells were prepared and transformed with the plasmids pDHE681, pAgro4 and pHSG575 ( = sample 1 in Table 3). After overnight culture at 37°C, the cells showed a high L-pantolactone-hydrolyzing activity in comparison with the control strain in question (TGI pDHE681 pAgro4 pHSG575== sample 2 in Table 3), whose maximum activity is, as a rule, reached after incubation for 6-7 hours (approx. 1500 U/L) and drops drastically upon longer incubation. The rhamnose (0.5 g/L) was not metabolized by TG10 pDHE681 pAgro4 pHSG575.<br><br>
1,	TG10 pDHE681 pAgro4 pHSG575; LB with ampicillin (Amp;<br>
100 fµg/ml) tetracyclin (Tet 10 β µg/ml), L-rhamnose (Rha 0.5 g/1) and isopropyl thiogalactoside (IPTG 0.15 mM)<br>
2,	TGI pDHE681 pAgro4 pHSG575; LB with ampicillin<br>
(Amp; 100 \µg/ml),   L-rhamnose (Rha 0.5 g/1) and isopropyl thiogalactoside (IPTG 0.15 mM)<br>
The assay was repeated in greater detail. The addition of tetracyclin (15 \µg/ml) to the medium is not necessary for maintaining the rhamnose deficiency.<br><br>
i 1<br>
I<br>
Example 5: Determining the dependency of the induction on the<br>
L-rhamnose concentration<br>
The strain E. coli TG10 (pDHEl650, pAgro4, pHSG575) was grown analogously to Example 1 on LB ampicillin (100 mg/1), chloramphenicol 10 mg/1, spectinomycin (50 mg/l)r IPTG 0.15 mM in the presence of various rhamnose concentrations (0 to 2 g/1 rhamnose) and analyzed (in duplicate) for its specific nitrilase activity. A concentration of as little as 0.01 g/1 L-rhamnose results in, on average, a significant induction of expression, while no significant expression was determined (via the enzyme activity) in the absence of rhamnose.<br><br>
Example 6: Analysis of the integration site of the transposon in<br>
the L-rhamnose-isomerase-deficient strain E. coli TG10<br>
To characterize the integration site of the transposon TnlO in greater detail, the rhamnose genes rhaT, rhaB, rhaA and rhaD were studied via PCR (Pfu polymerase) in comparison with TGI (pDHE681)<br><br>
and TG10 (pDHE681). When rhaA (L-rhamnose isomerase) or the region rhaα-rhaD were amplified with the primers MKe 259/260 and MKe 258/259, respectively, the mutagenized strain TG10 gave no specific amplificate, as opposed to the wild-type strain TGI.<br><br>
To inactivate the L-rhamnose-isomerase (rhaA) the rhaA gene is first amplified with the primers MKeOOl and MKe002 and cloned into pBluescriptSK+ (Xbal/Hindlll digestion and ligation). Thereafter, a frame shift is introduced by restriction digestion with BamHI and filling in with Klenow fragment, followed by ligation, and the corresponding rha* fragment is recloned into the gene replacement vector pK03 (Link et al. (1997) J Bacteriol 179:6228-6237). The knock-out of the rhaA gene in TGlpDHE1650 by homologous recombination with the rha* construct is carried out as described by Link et al. (Link et al. (1997) J Bacteriol 179:6228-6237) by means of selection on chloramphenicol at 43°C, replica plating on sucrose at 30°C and subsequent verification on : McConkey agar supplemented with 1 g/L rhamnose.<br><br>
Example 8: Production of recombinant nitrilase with the<br>
rhamnose-deficient host strain TG10<br>
The fed-batch fermentation of TG10 derivatives such as TG10 pDHE1650 pAgro4 pHSG575 is carried out on a modified Riesenberg medium with glycerol as the carbon source and rhamnose as inductor for overexpressing the target protein, in this case nitrilase. Comparably high, and higher, cell densities and enzyme activities were achieved using this strain.<br>
8.1 Fermentation of E. coli TG 1<br>
The fermentation of Escherichia  coli   (TGI pDHE1650 pAgro4 pHSG575) was carried out in a 20 L bioreactor. The reactor, with a working volume of 10 L, was inoculated with 200 ml of preculture from shake flasks. The preculture medium corresponds to the main culture medium.<br><br><br>
The medium is sterilized for 30 min at 121°C. Thereafter, 0.1 g of ampicillin are added under sterile conditions<br><br>
The fermentation is carried at a temperature of 37°C. The aeration  is adjusted to between 8-30 L/min and the stirrer speed to 400 to 1500 l/min in order to avoid the p02 dropping to below 20%. After a fermentation time of 1 hour, the culture is induced with IPTG (0.15 mM). Thereafter, 76 ml of rhamnose feed solution are added. When the rhamnose concentration in the fermenter falls below 5 1.0 g/L, rhamnose feed solution is metered in. After the amount<br><br>
of glycerol which had been introduced at the beginning has been consumed, glycerol is fed continuously.<br><br>
The fermentation of Escherichia coli  TG10 (pDHE1650 pAgro4 OHSG575) was carried out following the same protocol as in Example 1, except that induction was carried out with 18.5 g of rhamnose feed solution. No rhamnose was subsequently feed in.<br><br><br>
8.3 Activity assay:<br>
50 µl of cell suspension are pipetted to 880 µl of sodium/potassium phosphate buffer (10 mM) and the mixture is heated to 30°C. The reaction is started by addition of 20 µl of methanolic mandelonitrile solution (12%). After 10 minutes, the enzyme reaction is stopped by addition of 50 µl of 1M HC1. The cell biomass is centrifuged off and the mandelic acid concentration in the supernatant is measured by HPLC (ODS Hypersil  100*2.0 mm, mobile phase: 75% H3PO4 (14.8 mM)/25% methanol; flow rate: 0.5 ml/min; injection volume: 2 µl; column temperature: 40°C; detection: 210 nm; retention time mandelic acid: 0.9 minutes).<br><br>
A ceramic filter and a continuously operated roller pump are used for online sampling of the fermenter. The HPLC system is programmed in such a way that a new sample is injected after each analysis has been concluded. In between, the filtrate is pumped from the fermenter into a waste container.<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
claim:<br>
1.	A method for expressing nucleic acid sequences in prokaryotic<br>
host cells, where<br>
a)	at least one DNA construct which is capable of episomal replication in said host cells and which comprises a nucleic acid sequence to be expressed under the transcriptional control of an L-rhamnose-inducible promoter, where said promoter is heterologous with regard to said nucleic acid sequence, is introduced into said host cells and<br>
b)	prokaryotic host cells which comprise said DNA construct in episomal form are selected and<br>
c)	the expression of said nucleic acid sequence is induced by addition of L-rhamnose to a culture of said selected host cells,<br>
wherein the prokaryotic host cell is at least deficient with regard to L-rhamnose isomerase.<br>
2.	The method according to claim 1, wherein the prokaryotic host cell is selected from the species of the family Enterobacteriaceae or the order Actinomycetales.<br>
3.	The method according to claim 1 or 2, wherein the prokaryotic host cell is Escherichia coli.<br>
4.	The method according to any of claims 1 to 3, wherein the L-rhamnose-inducible promoter is the rhaPBAD promoter from E. coli or a functional equivalent thereof or a functionally equivalent fragment of the above promoters.<br>
5.	A method according to any of claims 1 to 4, wherein the L-rhamnose-inducible promoter comprises at least one RhaS binding element as shown in SEQ ID NO: 5 or a functional equivalent thereof or a functionally equivalent fragment of the above elements.<br>
6.	A method according to any of claims 1 to 5, wherein the L-rhamnose-inducible promoter comprises at least one sequence described by SEQ ID NO: 1, 2, 3 or 4.<br><br>
The method according to any of claims 1 to 6, wherein the L-rhamnose isomerase is described by the amino acid sequence as shown in SEQ ID NO: 9 or a functional equivalent thereof.<br>
The method according to one of claims 1 to 7, wherein the DNA construct which is capable of episomal replication has a size of not more than 100 000 bases or base pairs.<br>
The method according to any of claims 1 to 8, wherein the DNA construct which is capable of episomal replication is selected from the group consisting of circular plasmid vectors, phagemids and cosmids.<br>
The method according to any of claims 1 to 9, wherein the prokaryotic host cell has at least one further deficiency with regard to a gene which has a function in the metabolization of rhamnose, where said gene encodes a protein selected from the group consisting of rhamnulose 1-phosphatase (RhaB) and rhamnulose-phosphate aldolase (RhaD).<br>
The method according to any of claims 1 to 10, wherein the expression of the nucleic acid sequence to be expressed causes the production of a protein encoded by said nucleic acid sequence.<br>
The method according to any of claims 1 to 11, wherein the nucleic acid sequence to be expressed encodes a recombinant protein selected from the group consisting of chymosines, proteases, polymerasen, saccharidases, dehydrogenases, nucleases, glucanases, glucose oxidases, a-amylases, oxidoreductases, peroxidases, laccases, xylanases, phytases, cellulases, collagenases, hemicellulases, lipases, lactases, pectinases, amyloglucosidases, glucoamylases, pullulanases, glucose isomerases, nitrilases, esterases, nitrile hydratases, amidases, oxygenases, oxynitrilases, lyases, lactonases, carboxylases, collagenases, cellulases, serum albumins, factor VII, factor VIII, factor IX, factor X, tissue plasminogen factors, protein C, von Willebrand factors, antithrombins, erythropoietins, colony-stimulating factors, cytokins, interleukins, insulins, integrins, addressins, selectins, antibodies, antibody fragments, structural proteins, collagen, fibroins, elastins, tubulins, actins, myosins, growth factors, cell-cycle proteins, vaccines, fibrinogens and thrombins.<br><br>
13.	A prokaryotic host cell which is at least deficient with regard to L-rhamnose isomerase and which comprises at least one DNA construct which is capable of replication in said host cell and which comprises a nucleic acid sequence to be expressed under the transcriptional control of an L-rhamnose-inducible promoter, where said promoter is heterologous with regard to said nucleic acid sequence.<br>
14.	The use of a prokaryotic host cell according to claim 13 for the production of foodstuffs, feedstuffs, enzymes, chemicals, pharmaceuticals or fine chemicals.<br>
15.	A method for the production of recombinant proteins, enzymes and fine chemicals using a prokaryotic host cell according to claim 13 or a preparations thereof.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ2NC1jaGVucC0yMDA1IGFic3RyYWN0LWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">1464-chenp-2005 abstract-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ2NC1jaGVucC0yMDA1IGNsYWltcy1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1464-chenp-2005 claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ2NC1jaGVucC0yMDA1IGRlc2NyaXB0aW9uIChjb21wbGV0ZSktZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">1464-chenp-2005 description (complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ2NC1jaGVucC0yMDA1IGRyYXdpbmdzLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">1464-chenp-2005 drawings-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ2NC1jaGVucC0yMDA1LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1464-chenp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ2NC1jaGVucC0yMDA1LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1464-chenp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ2NC1jaGVucC0yMDA1LWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1464-chenp-2005-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ2NC1jaGVucC0yMDA1LWNvcnJlc3BvbmRuZWNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">1464-chenp-2005-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ2NC1jaGVucC0yMDA1LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">1464-chenp-2005-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ2NC1jaGVucC0yMDA1LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1464-chenp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ2NC1jaGVucC0yMDA1LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">1464-chenp-2005-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ2NC1jaGVucC0yMDA1LWZvcm0gMTgucGRm" target="_blank" style="word-wrap:break-word;">1464-chenp-2005-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ2NC1jaGVucC0yMDA1LWZvcm0gMjYucGRm" target="_blank" style="word-wrap:break-word;">1464-chenp-2005-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ2NC1jaGVucC0yMDA1LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">1464-chenp-2005-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ2NC1jaGVucC0yMDA1LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">1464-chenp-2005-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ2NC1jaGVucC0yMDA1LXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1464-chenp-2005-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="229017-improved-guard-for-vertical-machining-centers.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="229019-reticulated-elastomeric-matrices-and-a-process-for-preparing-the-same.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>229018</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1464/CHENP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>12/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>20-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>13-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>30-Jun-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>BASF AKTIENGESELLSCHAFT</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>D-67056, Ludwigshafen,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>KESSELER, Maria</td>
											<td>Max-Joseph-Strasse 6, 68167 Mannheim,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>ZELINSKI, Thomas</td>
											<td>Kirchengasse 16, 67271 Neuleiningen,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>HAUER, Bernhard</td>
											<td>Merowingerstr.1, 67136 Fussgönheim,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/63</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2003/013367</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-11-27</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>102 56 381.0</td>
									<td>2002-12-02</td>
								    <td>Germany</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/229018-a-method-for-expressing-nucleic-acid-sequence-in-prokaryotic-host-cell-deficient-in-l-rhamnose-isomerase by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:40:07 GMT -->
</html>
