<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/229251-a-method-of-reducing-or-eliminating-the-tripeptidyl-amino-peptidase-tpap-production-from-a-tpap-producing-cell-of-the-genus-aspergillus by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:55:35 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 229251:A METHOD OF REDUCING OR ELIMINATING THE TRIPEPTIDYL AMINO PEPTIDASE (TPAP) PRODUCTION FROM A TPAP PRODUCING CELL OF THE GENUS ASPERGILLUS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD OF REDUCING OR ELIMINATING THE TRIPEPTIDYL AMINO PEPTIDASE (TPAP) PRODUCTION FROM A TPAP PRODUCING CELL OF THE GENUS ASPERGILLUS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a method of reducing or eliminating the tripeptidyl aminopeptidase (TPAP) production from a TPAP producing ceil of the genus Aspergillus, in which method a DNA sequence encoding TPAP or a part thereof essential for exhibiting TPAP activity or the TPAP promoter sequence required for the expression of TPAP from a TPAP encoding DNA sequence present in said cell are modified or inactivated so as to result in a reduced or no TPAP production from said cell.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br><br><br>
?he present invention relates to a tripeptidyl aminopeptidase [TPAP), a DNA construct encoding the TPAP, a method of produc-.ng TPAP and methods of reducing the TPAP production in cells, .n  which  TPAP  activity  is  undesired.<br>
JACKGROUND  OF  THE   INVENTION<br>
?ripeptidyl aminopeptidases are enzymes capable of cleaving 'ragments from unsubstituted N-termini of peptides, oligopepti-les, or proteins. Tripeptidyl aminopeptidases may be unspeci-:ic, i.e. cleaving any tripeptide sequence from the unsubstitu-:ed N-terminal end, or specific, i.e. capable of cleaving jpecific  types  of  tripeptide  sequences.<br>
tripeptidyl aminopeptidases of animal origin have been reported )reviously. For instance, Doebber et al. (1978) , Endocrinology .03: 1794-1804, disclose a tripeptidyl aminopeptidase isolated :rom bovine pituitary glands, which was shown to cleave :ripeptides from the N-terminal end of bovine growth hormone. :n Mammalian Proteases Vol. 2, Exopeptidases (AP 1986, eds. r.K. McDonald and A.J. Barrett) tripeptidyl aminopeptidases are lisclosed which are isolated from beef pituitary glands, )regnant  hog  ovaries  and  hog  spleen,   respectively.<br>
I bacterial tripeptidyl aminopeptidase isolated from 'treptomyces lividans 66 is described by Krieger et al., Febs setters Vol. 352, No. 3, pp 385-388, 1994. Butler et al., applied and Environmental Microbiology, August 1995, p. 3145-1150 disclose the gene encoding said peptidase and the deduced imino acid sequence. The peptidase is characterized as a serine )rotease  with  a  pH  optimum  between  7.5  and  8.5.<br>
:t is well-known that the stability of microbially produced &gt;roducts,   such  as  enzymes   or  other  proteins,   may  vary,   inter<br><br>
alia, as a function of the method by which the product is produced and/or the origin or microbial producer of the product. A reduced stability of a microbially produced product may, e.g., be due to a reduced resistance towards heat, light or other external conditions, or may be due to the composition of the product itself. In this connection, the presence of even trace amounts of protease activity in a protein product may result in a significantly reduced stability of said product. Often, however, it is not possible to exactly identify the cause  of  the  varying  stability.<br>
SUMMARY  OF  THE   INVENTION<br>
The present inventors have now surprisingly found that various fungal species produce TPAP and that protein products produced by these organisms may contain minor amounts of TPAP which in some cases have been found to lead to a reduced stability of these products. The present inventors have succeeded in isolating and characterizing said TPAP. It has been established that the TPAP is capable of unspecifically cleaving tripeptides from the unsubstituted N-terminus of protein products, a cleavage which in some instances may be desirable, and in other instances (e.g. when resulting in a reduced stability of a protein  product  comprising  the  TPAP)   is  highly  undesirable.<br>
Accordingly, one object of the present invention is to provide a substantially pure tripeptidylpeptidase which may be used for cleaving peptide or protein sequences. Another object is to provide a method of producing protein products essentially free from TPAP, in particular products which are substrates for TPAP and  which  have  a  reduced  stability  in  the  presence  of  TPAP.<br>
In a first general aspect the present invention relates to an isolated TPAP of fungal origin. In particular, the TPAP may be obtainable from strains of the fungal species Aspergillus. In further aspects the invention relates TPAP identified by amino acid and/or DNA sequence information and/or by enzyme protein charactestics,  to  DNA  constructs  encoding  the  TPAP  and  to  a  DNA<br><br>
construct comprising a nucleotide sequence which is complemenÂ¬tary to a sufficient part of the DNA sequence encoding TPAP to be able to hybridize to said sequence and thereby abolish the TPAP  producing  capability  of  a  given  host  cell.<br>
In a further important aspect the invention relates to a method of reducing the TPAP production from a TPAP producing cell, in which method a DNA sequence present in said cell and necessary for expression of TPAP is modified or inactivated so as to result  in  a  reduced  TPAP  production  from  said  cell.<br>
DEFINITIONS<br>
In the present context the term "TPAP" is intended to indicate an aminopeptidase which cleaves tripeptides from the N-terminal end of a peptide or protein sequence, for instance from an extended protein sequence, e.g. found in a prohormon or a proenzyme. Expressed in a general manner the TPAP is capable of cleaving the tripeptide XYZ from the unsubstituted N-terminal amino group of a peptide or protein, wherein X, Y, Z represents any amino acid residue (i.e. selected from Ala, Arg, Asn, Asp, Cys, Gin, Glu, Gly, His, lie, Leu, Lys, Met, Phe, Pro, Ser, Thr, Trp, Tyr, Val) . In the TPAP substrate all of X, Y and Z may be different or identical or two of X, Y and Z may be identical. It will be understood that the TPAP of the invention is unspecific as to the amino acid sequence of the tripeptide to be cleaved. In example 7 specific examples of tripeptide products obtained from cleavage of naturally occurring peptides and  proteins  are  given.<br>
The term "obtainable" as used about the origin of DNA sequences i^onstituting part of the DNA construct of the invention is intended to indicate that the DNA sequence in question may be isolated from nucleic acid (DNA or RNA) material of the relevant organism or may be prepared on the basis of such material. For instance, the DNA sequence may be isolated from a   genomic   or   cDNA   library   prepared   from   the   organism   using<br><br>
procedures known in the art, or may be prepared on the basis of such  material.<br>
Analogously, when used in connection with the TPAP of the invention the term "obtainable" is intended to indicate that the TPAP may be recovered from the organism in question or may be encoded by a DNA sequence obtainable from said organism and recovered  from  an  organism  expressing  said  DNA  sequence.<br>
DETAILED  DISCLOSURE  OF  THE  INVENTION<br>
The  TPAP  of  the  invention<br>
In the course of the research leading to the present invention two different TPAPs have been isolated and characterized - one from a strain of A. niger, another from a strain of A. oryzae. The two TPAPs are contemplated to be representative examples of a  generally  novel  class  of  tripeptidyl  amino  peptidases.<br>
Thus, in one aspect the invention relates to a TPAP which is encoded  by  a  DNA  construct  comprising<br>
i)   at  least  one  of,   but  preferably  two  or  more  of  the  partial DNA  sequences  shown  in  SEQ  ID  Nos.   1,   2  and  3,   or ii)   the  partial  DNA  sequence  encoding  the  N-terminal  part  of  a mature  TPAP  present  in  DSM  9570,   or<br>
iii) a nucleotide sequence which hybridizes to an oligonucleÂ¬otide probe prepared on the basis of the DNA sequences shown in SEQ ID No. 1, 2 or 3 or on the basis of any of the amino acid sequences  shown  in  SEQ  ID  Nos.   4-14,   and  which  encodes  a  TPAP.<br>
k more detailed explanation of the nucleotide sequence iii) is given futher below in the section entitled "The DNA construct and  vector  of  the  invention".<br>
En another aspect the invention relates to a substantially pure rPAP  which  has  one  or  more  of  the  following  characteristics<br>
-  a  capability  of  cleaving  the  substrate  Phe-Pro-Ala-pNA,<br><br>
-	a molecular weight of about 65 kDa (determined essentially as described by Laemmli, U.K., 1970, "Cleavage of structural proteins during the assembly of the head of bacteriophage T4"., Nature,   227,   p.   680-685)<br>
-	a  pi  in  the  range  of  4-6<br>
-	a  pH  optimum  in  the  range  of  about  5.0-7.5,<br>
-	is immunologically cross-reactive with the purified A. niger TPAP  or  the  purified A.  oryzae  TPAP,<br>
-	comprises  the  N-terminal  sequence<br>
Ala-Xaa(1)-Asn-Xaa(2)-Ser-His-Cys-Asp-Ser-Ile-Ile-Thr-Pro-Xaa (3) -Cys-Leu-Lys-Xaa (4) -Leu-Tyr-Asn-Ile-Gly-Asp-Tyr-Gln-Ala-Asp-Xaa(5)-Xaa(6) (SEQ ID NO 15), in which any one of Xaa(l), Xaa(2), Xaa(3), Xaa(4), Xaa(5) and Xaa(6) may be different or identical and selected from any of the naturally occurring amino acid residues. Preferably, Xaa(l) is Lys or Gin, Xaa(2) is lie or Thr, Xaa(3) is Pro or His, Xaa(4) is Glu or Gin, Xaa(5)   is  Pro  or  Ala  and/or  Xaa(6)   is  Lys  or  Asn.<br>
Antibodies    to   be   used    in   determining    immunological    cross-reactivity  may  be  prepared  by  use  of  the  purified  TPAP.   More specifically,  antiserum  against  the  enzyme  of  the  invention may be  raised  by  immunizing rabbits  (or  other  rodents)   according to the  procedure  described  by  N.   Axelsen  et  al.   in:   A  Manual  of Quantitative      Immunoelectrophoresis.      Blackwell      Scientific Publications,   1973,   Chapter  23,   or  A.  Johnstone  and  R.  Thorpe, Immunochemistry in Practice.  Blackwell  Scientific Publications, 1982    (more   specifically  pp.   27-31).   Purified   immunoglobulins nay  be  obtained  from  the  antisera,  for  example  by  salt  precipiÂ¬tation     [(NH4)2S04],     followed    by    dialysis    and    ion    exchange chromatography,   e.g.  on  DEAE-Sephadexâ¢.   Immunochemical  charac-:erization   of   proteins    may   be   done   either   by   Outcherlony iouble-diffusion    analysis    (O.    Ouchterlony    in:    Handbook    of Sxperimental  Immunology   (D.M.  Weir,  Ed.),  Blackwell  Scientific Publications,   1967,  pp.   655-706),  by  crossed  immunoelectropho-â¢esis    (N.   Axelsen   et   al..   supra,    Chapters   3   and   4) ,    or   by â¢ocket  immunoelectrophoresis  (N.  Axelsen et al. .  supra,  Chapter<br>
i).<br><br>
In one embodiment the TPAP of the invention comprises at least one of the partial amino acid sequences shown in SEQ ID Nos. 4-9 and/or has a pH optimum of about 5.0-5.5 and/or a pi of about 5.1  or  5.2.<br>
In another embodiment the TPAP of the invention comprises at least one of the partial amino acid sequences SEQ ID NOs 10-14 and/or has a pH optimum in the range of about 5.5-7.5 and/or a pl  of  about  4.5.<br>
The TPAP which is encoded by a DNA sequence comprising at least one of the sequences shown in SEQ ID NO 1, 2 and 3 or the one harboured in DSM 9570 or which comprises the peptides shown in SEQ ID NOs 4-9 was isolated from a protein product produced by a strain of A. niger (cf. Example 1 hereinafter). The TPA comprising the peptides SEQ ID Nos 10-14 was isolated from a strain  of A.  oryzae,   cf  Example  2  hereinafter.<br>
Et is presently believed that a TPAP belonging to the generally lovel class of tripeptidyl aminopeptides defined herein may be 3f any origin, including animal or plant origin, but preferably Ls of microbial, i.e. bacterial or fungal, origin. As far as :he present inventors are aware the present disclosure is the :irst report on TPAP of fungal origin. TPAP of the invention lay be purified from strains which are natural TPAP producers, )r may more conveniently be produced by means of recombinant )NA techniques as a homologous or heterologous gene product as rill  be  further  explained  below.<br>
n particular, the TPAP of the invention may be obtainable from a strain of Aspergillus, such as a strain of A. oryzae, A. niger, A. japonicus, A. aculeatus, A. nidulans or A. foetidus or a strain of Trichoderma, e.g. T. viride, T. reesei, T. longibrachiatum or T. harzianum, or a species of Fusarium, e.g. F. oxysporum, F. graminearum or F. solani, or a strain of Thermomyces,   e.g.   T.   lanuginosus  or T.  insolens.<br><br>
	^^Â«4.  wjL  .-lie  invention  IS  preferably  provided  in  an  isolated<br>
and substantially pure form, e.g. at least 90% pure such as at least  95%  pure.<br>
While the presence of TPAP in protein products may be conÂ¬sidered undesirable due to the resulting reduced stability of said product, the use of the purified TPAP of the invention for controlled destabilization of protein products may be advanÂ¬tageous. For instance, it is contemplated that the purified TPAP of the invention may be used for deactivation of enzymes after they have exerted their desired effect, and thus function as a "killer enzyme". Such deactivation is conventionally accomplished by thermoinactivation (or alternatively, the undesired enzyme activity is removed by purification). The use of TPAP for destabilization of thermophilic enzymes may be particularly  advantageous.<br>
As an example of this use may be mentioned the deactivation of AMG used for starch liquefaction, which is presently performed by heating the reaction mixture to high temperatures (80-85Â°C). This deactivation may, e.g., be obtained by adding TPAP, preferably in a batch proces after AMG has hydrolyzed dextrins to glucose. A thermoinactivation of AMG is then possible by increasing  the temperature  to  only  about  66Â°C  for  a  short time.<br>
Another example is the deactivation of AMG used in the fermenÂ¬tation of beer such as low calorie beer. In the normal beer fermentation procedure AMG is inactivated by pasteurization. The addition of TPAP may reduce the thermostability of the used ?^G and thus reduce the temperature for the pasterization. By this treatment the organoleptic characteristics may be improved.<br>
Furthermore, the purified TPAP of the invention may be useful for a number of purposes in which a specific cleavage of tripeptide sequences are desirable. For instance, some proteins or peptides are synthesized in the form of precursors comprisÂ¬ing   a   number    of    additional   amino   acid   residues   on   the   N-<br><br>
terminal amino acid residue, the presence of which is undesirÂ¬able for the protein to be used. This may, e.g., be in the pro-cessesing of proteins or peptides, which may be synthesized in protected  form  as  precursor  proteins.<br>
The  DNA  construct  and  vector  of  the  invention<br>
In    accordance   with   a    still    further   aspect,    the    invention relates  to  a  DNA  construct  encoding  a  TPAP,   which  comprises i)   at   least  one  of,   but  preferably  two  or  all  of  the  partial DNA  sequences  shown  in  SEQ  ID  Nos.   1,   2  and  3,   or ii)   the  partial   DNA  sequence  encoding  the  N-terminal  part  of TPAP  and  present  in  DSM  9570,   or<br>
iii) a nucleotide sequence which hybridizes to an oligonucleÂ¬otide probe prepared on the basis of any of the DNA sequences shown in SEQ ID No. 1, 2 or 3 or on the basis of any of the amino acid sequences shown in SEQ ID Nos. 4-14, and which encodes  a  TPAP,   or<br>
iv) a nucleotide sequence complementary to the DNA/nucleotide sequence  of  i),   ii)   or  iii).<br>
The DNA construct of the invention may be used either for the recombinant production of TPAP (DNA/nucleotide sequences i) -iii)) or for reducing the TPAP producing capability of a cell, in which said production is undesired (nucleotide sequence iv)).<br>
The nucleotide sequence iii) may, e.g., be isolated from another or related (e.g. the same) organism known or contemÂ¬plated to produce TPAP on the basis of any of the partial DNA or partial amino acid sequences shown in SEQ ID Nos. 1-14 or the partial TPAP encoding DNA sequence of DSM 9570, e.g. using the procedures described herein, or constructed on the basis of any of said DNA or amino acid sequences, e.g. by introduction of nucleotide substitutions which do not give rise to another amino acid sequence of the TPAP encoded by the DNA sequence, but which correspond to the codon usage of the host organism intended   for   production   of   the   TPAP,    or   by   introduction   of<br><br>
iiuuij-fejotiae substitutions which do give rise to a different amino acid sequence and therefore, possibly, a different protein structure which might give rise to a TPAP mutant with different properties than the native TPAP, but with the TPAP activity intact, other examples of possible modifications are insertion of one or more nucleotides into the sequence, addition of one or more nucleotides at either end of the sequence, or deletion of one or more nucleotides at either end or  within  the  sequence.<br>
It will be understood that the DNA sequences shown in SEQ ID Nos. 1-3 and the DNA sequence of DSM 9570 are partial sequences which may be included in and used for isolating an entire TPAP encoding DNA sequence. This may easily be achieved by methods known in the art. In accordance with the present invention, the nucleotide sequence iii) is intended to include said entire DNA sequence.<br>
The hybridization referred to above is intended to indicate that the nucleotide sequence iii) hybridizes to the same probe as the DNA sequence encoding the TPAP (or to said DNA sequence sncoding TPAP) under certain specified conditions which are iescribed in detail in the Materials and Methods section her-sinafter. Normally, the nucleotide sequence iii) is highly homologous to the DNA sequence shown in SEQ ID No. 1, 2 or 3 or :he TPAP encoding DNA sequence of DSM 9570 (when comparing the )art of the nucleotide sequence iii) which corresponds to the )artial DNA sequence(s) disclosed herein), such as at least )5%, e.g. at least 70% homologous to said DNA sequence, e.g. at .east 75%, at least 80%, at least 85%, at least 90% or even at .east 95% homologous to any or all of said sequences. Corre-ipondingly,   the  TPAP  of  the  invention  is  comtemplated  to  be  at<br>
east 65%, e.g. at least 70% homologous to the amino acid ;equence encoded by said DNA sequence(s) (as evaluated on the &gt;asis of a comparison between the part of the TPAP amino acid lequence  corresponding  to  the  part  encoded  by  the  DNA  sequence<br>
n  question),  e.g.   at  least  75%,  at  least  80%,  at  least  85%,  at<br>
east  90%  or  even  at  least  95%  homologous.<br><br>
The  nucleotide  sequence  iii)   may  be  a  DNA  or  an  RNA  sequence.<br>
The DNA sequences i)-iii) of the DNA construct of the invention may be prepared by well-known methods. Thus, the relevant DNA sequence may, for instance, be isolated by establishing a cDNA or genomic library from an organism expected to harbour the sequence, e.g. a cell as described above, and screening for positive clones by conventional procedures. Examples of such procedures are hybridization to suitable oligonucleotide probes in accordance with standard techniques (cf. Sambrook et al., Molecular Cloning: A Laboratory Manual, Cold Spring Harbor, New York 1989), and/or selection for clones expressing the relevant activity, and/or selection for clones producing a protein which is reactive with an antibody raised against the relevant enzyme.<br>
A preferred method of isolating a DNA sequence from a cDNA or genomic library is by use of polymerase chain reaction (PCR) using degenerate oligonucleotide probes. For instance, the PCR may be carried out using the techniques described in PCR Protocols 1993, ed. Bruce A. White and The polymerase chain reaction,   1994,   ed.   Kary  B.   Mullis.<br>
Alternatively, the DNA sequences i) and ii) may simply be isolated  from  DSM  9570.<br>
Furthermore, DNA sequences of the DNA construct of the invenÂ¬tion, e.g. the nucleotide sequence iv) being complementary to the DNA/nucleotide sequence i) , ii) or iii), may be synthesized by established techniques, e.g. based on the principles disclosed by Narang, SA, 1983, Tetrahedron 39:3 and Itakura et al.,   1984,   Annu.   Rev.   Biochem.   53:323.<br>
The DNA construct may be of mixed genomic and synthetic, mixed synthetic and cDNA or mixed genomic and cDNA origin prepared by ligating fragments of synthetic, genomic or cDNA origin (as appropriate),   the  fragments  corresponding  to  various  parts  of<br><br>
the entire recombinant DNA molecule, in accordance with standard  techniques.<br>
It will be understood that a preferred use of the DNA sequence i) , ii) or iii) is in the preparation of recombinant TPAP, whereas a preferred use of the DNA sequence iv) is for the reduction of the TPAP producing capability of cells intended for use in the production of protein products which are sensible  to  TPAP.<br>
Although the DNA sequence iv) may be complementary to the entire TPAP encoding sequence, it is normally sufficient that the DNA sequence iv) is complementary to only part of said sequence. Expressed in a functional manner, the DNA sequence iv) must be complementary to a sufficient length of the DNA sequence encoding the TPAP to allow for hybridization to the TPAP encoding DNA and thus reduction or prohibition of the transcription of said mRNA. Typically, it is sufficient that the DNA sequence iv) comprises a DNA fragment of at least 17 nucleotides  such  as  at  least  3 00  nucleotides.<br>
The vector carrying a DNA construct of the invention, preferabÂ¬ly a recombinant expression vector, may be any vector which may conveniently be subjected to recombinant DNA procedures, and the choice of expression vector will often depend on the host cell into which it is to be introduced. Thus, the vector may be an autonomously replicating vector, i.e. a vector which exists as an extrachromosomal entity, the replication of which is independent of chromosomal replication, e.g. a plasmid or a bacteriophage. Alternatively, the vector may be one which, when Introduced into a host cell, is integrated into the host cell jenome and replicated together with the chromosome(s) into 
[n the DNA construct or the vector, the DNA sequence should be jperably connected to a suitable promoter sequence. The pro~ aoter may be any DNA sequence which shows transcriptional ac-:ivity  in  the  host  cell  of  choice  and may  be  derived  from genes<br><br>
encoding proteins either homologous or heterologous to the host cell. The promoter may be derived from genes encoding both exÂ¬tracellular and intracellular proteins, such as amylases, glucoamylases, proteases, lipases, cellulases and glycolytic enzymes. Examples of suitable promoters for directing the tranÂ¬scription of the DNA construct of the invention are promoters derived from genes for A. oryzae TAKA amylase, Rhizomucor miehei aspartic proteinase, A. niger glucoamylase, A. niger neutral a-amylase, A. niger acid stable a-amylase, and RhizomuÂ¬cor miehei lipase. Examples of promoters from genes for glycolytic enzymes are TPI, ADH, and PGK. The promoter may also be a homologous promoter, i.e. a gene native to the host strain being  used.<br>
The promoter sequence may be provided with linkers for the purpose of introducing specific restriction sites facilitating ligation of the promoter sequence with the gene of choice or tfith  a  selected  signal  peptide  or  preregion.<br>
The DNA construct and/or expression vector of the invention may ilso comprise a suitable terminator operably connected to the )NA sequence encoding the TPAP and/or a polyadenylation sequence. The terminator and polyadenylation sequences may be ierived from the same sources as the promoters. Enhancer sequences  may  also  be  inserted  into  the  construct.<br>
?he DNA construct and/or vector may further comprise a DNA sequence enabling the vector to replicate in the host cell in [uestion. Examples of such sequences are the origins of repli-â¢.ation of plasmids pUC19, pACYC177, pUBllO, pE194, pAMBl and &gt;IJ702.<br>
'he DNA construct and/or vector may also comprise a selectable larker. Examples of selection markers include the amdS or argB, rpC or pyrG (the latter three markers, e.g. from A. nidulans r A.  niger).<br><br>
The procedure used to construct the DNA construct of the invention comprise ligating the DNA sequences mentioned above the promoter, the terminator and other elements, respectively, and to insert it into suitable vectors containing the informaÂ¬tion necessary for replication, are well known to persons skilled in the art (cf., for instance, Sambrook et al. op. cit.) .<br>
The  cell  and  a  method  of  producing  TPAP  of  the  invention In  one  embodiment  the  cell  of  the  invention  either  comprising a   DNA   construct   or   an   expression  vector   of   the   invention  as defined  above  is  used  as  a  host  cell  in  the  recombinant  producÂ¬tion  of  TPAP  of  the  invention.   In  this  case  the  DNA  construct or  expression   vector   comprises   any   of   the   TPAP   encoding   DNA sequences  i)-iii)   or  the  insert  of  DSM  9570  defined  above.  The 3ell may be transformed with the DNA construct,  conveniently by Integrating  the  DNA  construct  in  the  host  chromosome,   although :he  DNA  construct may  also  exist  as  an  extrachromosomal  entity, lowever,    the   integration   is   generally   considered   to   be   an idvantage  as  the  DNA  sequence  is  more  likely  to  be  stably main-:ained  in  the  cell.   Integration  of  the  DNA  constructs  into  the lost   chromosome   may   be   performed   according   to   conventional lethods,   e.g.   by  homologous  recombination.   Alternatively,   the lell  may  be  transformed  with  an  expression  vector  as  described telow  in  connection  with  the  different  types  of  host  cells.<br>
he cell of the invention may be a cell of a higher organism uch as a mammal or an insect, but is preferably a microbial ell,   e.g.   a  bacterial  or  a  fungal   (including  yeast)   cell.<br>
xamples of suitable bacteria are grampositive bacteria such as acillus subtilis, Bacillus licheniformis, Bacillus lentus, acillus brevis, Bacillus stearothermophilus, Bacillus alkalo-hilus, Bacillus amyloliquefaciens, Bacillus coagulans, Bacil-us circulans, Bacillus lautus, Bacillus thuringiensis or treptomyces lividans or Streptomyces murinus, or gramnegative acteria  such as E.coli.  The  transformation of  the  bacteria may<br><br>
for instance be effected by protoplast transformation or by using  competent  cells  in  a  manner  known per se.<br>
The yeast organism may favourably be selected from a species of Saccharomyces or Schizosaccharomyces, e.g. Saccharomyces ce-revisiae. The filamentous fungus may advantageously belong to a species of Aspergillus, e.g. Aspergillus oryzae, A. nldulans, A. foetidus, A. aculeatus, A. japonicus or A. niger, a species of Trichoderma, e.g. T. reesei, T. longibrachiatum or T. harzianum, or a species of Fusarium, e.g. F. oxysporum, F. graminearum or F. solani. Fungal cells may be transformed by a process involving protoplast formation and transformation of the protoplasts followed by regeneration of the cell wall in a manner  known per se.<br>
In a further aspect the invention relates to a method of producing  TPAP,   which  method  comprises<br>
L) culturing a cell of the invention as defined above in a suitable culture medium under conditions permitting expression 3f  the  TPAP,   and  recovering  the  TPAP  from  the  culture.<br>
[â¢he medium used to cultivate the cells may be any conventional ledium suitable for growing the host cell in question. Suitable ledia are available from commercial suppliers or may be &gt;repared according to published recipes (e.g. in catalogues of â¢.he American Type Culture Collection) . It is believed that the &gt;resence of protein in the medium may result in an increased 'PAP  production.<br>
'he TPAP may be recovered from the medium by conventional irocedures including separating the cells from the medium by lentrifugation or filtration, if necessary after disruption of he cells, precipitating the proteinaceous components of the upernatant or filtrate by means of a salt, e.g. ammonium sul-hate, followed by purification by a variety of chromatographic rocedures, e.g. ion exchange chromatography, affinity chroma-ography,   or  the  like.<br><br>
Removal  or  reduction  of  TPAP  activity<br>
The identification of TPAP as a destabilizing factor in microbially produced protein products may have important consequences for the production of a large number of different protein products. Thus, as it has been demonstrated by the present inventors, even minor amounts of TPAP present in a protein product may result in a reduced thermostability of said product. Accordingly, by the present invention it is possible to construct production strains which have a reduced TPAP-producing  capability.<br>
The reduction of TPAP production from a TPAP producing cell may conveniently be accomplished by modification or inactivation of a DNA sequence present in said cell and necessary for expressÂ¬ion of TPAP so as to result in a reduced TPAP production from said  cell.<br>
The DNA sequence to be modified may, e.g., be a DNA sequence encoding TPAP or a part thereof essential for exhibiting TPAP activity or may be a regulatory sequence required for the expression  of  TPAP  from  a  TPAP  encoding  DNA  sequence.<br>
As an example the regulatory sequence may be a promoter sequence or a functional part thereof, i.e. a part which is sufficient  for  effecting  expression  of  TPAP.<br>
The modification or inactivation of the DNA sequence may be performed by subjecting the TPAP producing cell to mutagenesis and selecting for cells for which the TPAP producing capability las been reduced. The mutagenesis, which may be specific or random, may, e.g., be performed by use of a suitable physical 3r chemical mutagenizing agent, by use of a suitable oligonuÂ¬cleotide, or by subjecting the DNA sequence to PCR generated nutagenesis. Furthermore, the mutagenesis may be performed by ise  of  any  combination  of  these  mutagenizing  agents.<br><br>
Examples of a physical or chemical mutagenizing agent suitable for the present purpose includes ultraviolet (UV) irradiation, hydroxylamine, N-methyl-N'-nitro-N-nitrosoguanidine (MNNG), O-methyl hydroxylamine, nitrous acid, ethyl methane sulphonate (EMS), sodium bisulphite, formic acid, and nucleotide analoÂ¬gues.<br>
When such agents are used the mutagenesis is typically perÂ¬formed by incubating the cell to be mutagenized in the presence of the mutagenizing agent of choice under suitable conditions for the mutagenesis to take place, and selecting for mutated cells  having  a  reduced  TPAP  production.<br>
When the modification or inactivation is accomplished by introduction, substitution or removal of one or more nucleoÂ¬tides in the TPAP encoding sequence or a regulatory element required for the transcription or translation thereof, nucleoÂ¬tides may, e.g., be inserted or removed so as to result in the Introduction of a stop codon, the removal of the start codon or a change of the open reading frame. The modification or Lnactivation of the TPAP encoding sequence or a regulatory element may be accomplished by site-directed mutagenesis or PCR generated mutagenesis in accordance with methods known in the irt. Although in principle, the modification may be performed :n vivo, i.e. directly on the cell carrying the TPAP gene to be lodified, it is presently preferred to conduct the modification .n vitro  as  exemplified  below.<br>
in example of a convenient way to inactivate or reduce the TPAP (reduction of a host cell of choice is based on the principles Â»f gene replacement or gene interruption. For instance, the [ene interruption method involves the use of a DNA sequence lorresponding to the endogenous gene or gene fragment which it s desired to destroy. Said DNA sequence is in vitro mutated to defective gene and transformed into the host cell. By omologous recombination the defective gene replaces the ndogenous gene or gene fragment. It may be desirable that the efective  gene  or  gene  fragment  encodes  a  marker  which  may  be<br><br>
used  for  selection  of  transformants  in  which  the  TPAP  gene  has been  modified  or  destroyed.<br>
Alternatively, the modification or inactivation of the DNA sequence may be performed by use of established anti-sense techniques using a nucleotide sequence complementary to the TPAP encoding sequence, e.g. the nucleotide sequence iv) described above. More specifically, the TPAP production from a TPAP producing cell may be reduced or eliminated by introducing a nucleotide sequence complementary to the TPAP encoding sequence and thus capable of hybridizing to TPAP mRNA produced in the cell into the cell in such a manner that the nucleotide sequence may be transcribed in the cell under conditions allowing the nucleotide sequence to hybridize to the TPAP mRNA ind thus reduce the amount of TPAP translated from said mRNA or Jliminate  any  such  translation.<br>
[â¢he TPAP-deficient mutants so created are particularly useful Ln the expression of heterologous proteins. In the present jontext the term "heterologous proteins" is intended to mean a &gt;rotein which is not native to the host cell, a native protein .n which modifications have been made to alter the native ;equence, or a native protein whose expression is quantitative-y altered as a result of a manipulation of the host cell by ecombinant  DNA  techniques.<br>
t is preferred that the TPAP producing cell to be modified in ccordance with the present invention is a strain which is uitable for the production of desired protein products, either omologous or heterologous to the cell. For instance, cells of he fungal genera Aspergillus, Trichoderma and Fusarium are xamples of prefered production cells. Accordingly, the cell to e modified according to the present invention is preferable a ell of an Aspergillus sp., in particular a cell of A. niger, . oryzae, A. japonicus, A. foetidus or A. nidulans or a cell f a Trichoderma sp., e.g. T. reesei, T. longibrachiatum or T. arzianum, or a cell of a Fusarium sp., e.g. F. oxysporum, F. raminearum  or F.   solani.<br><br>
in a specific embodiment of the invention the cell to be modified is a cell of A. niger or A. oryzae which is used for the  production  of  enzymes  such  as  AMG.<br>
In a further aspect the invention relates to a method of preparing a product essentially free from TPAP activity, which method comprises transforming a host cell as described above having a reduced or no TPAP producing capability with a DNA sequence encoding the product, culturing the transformed cell under suitable conditions for expression of the product, and recovering  the  product  from  the  culture.<br>
In an alternativ aspect the invention relates to a method of preparing a product essentially free from TPAP activity, which product is encoded by a DNA sequence present in a TPAP expressÂ¬ing cell, which method comprises modifying or inactivation a DNA sequence present in said cell and necessary for expression 3f TPAP as described above, and subsequently culturing the cell ander suitable conditions for expression of the product, and recovering  the  product  from  the  culture.<br>
[n a still further aspect the invention relates to a method of jreparing a product essentially free from TPAP by fermentation )f a TPAP-producing cell which also produces the product, which lethod comprises adding an effective amount of an inhibitor :apable of inhibiting TPAP activity to the fermentation broth iither during or after the fermentation has been completed, â¢ecovering the product of interest from the fermentation broth, md optionally subjecting the recovered product to further turification. This method is further illustrated in the ixamples  below.<br>
n a still further alternative aspect the invention relates to . method of preparing a product essentially free from TPAP ctivity, which product is encoded by a DNA sequence present in TPAP expressing cell, which method comprises cultivating the PAP expressing cell encoding the product under conditions ermitting    the    expression    of    the    product,     subjecting    the<br><br>
resulting culture broth to a combined pH and temperature treatment so as to reduce the TPAP activity substantially, and recovering the product from the culture broth. Alternatively, the combined pH and temperature treatment may be performed on an enzyme preparation recovered from the culture broth. The combined pH and temperature treatment may optionally be used in combination  with  a  treatment  with  a  TPAP  inhibitor.<br>
In accordance with this aspect of the invention it is possible to remove at least 60% of the TPAP activity, such as at least 75% of the activity, more preferably at least 85% of the activity, still more preferably at least 95% of the activity, and most preferably essentially at least 99% of the TPAP activity. It is contemplated that a complete removal of TPAP activity  may  be  obtained  by  use  of  this  method.<br>
The combined pH and temperature treatment is preferably carried out at a pH in the range of 6.5-7 and a temperature in the range of 25-40Â°C for a sufficient period of time for obtaining the desired effect. Typically, 0.5-1 hour is sufficient for obtaining  the  desired  effect.<br>
The methods used for cultivation and purification of the product of interest may be performed by methods known in the art,  e.g.  as  described  herein  before.<br>
The methods of the invention for producing an essentially TPAP-free product is of particular interest in the production of eukaryotic proteins, in particular fungal proteins such as enzymes. The enzyme product may, e.g., be selected from an amylolytic enzyme, a lipolytic enzyme, a proteolytic enzyme, a cellulytic enzyme, an oxidoreductase or a plant cell-wall degrading enzyme. Examples of such enzymes include AMG, amylase, lipase, cutinase, esterase, cellulase, hemicellulase, protease, peroxidase, laccase, phenoloxidase, catalase, glucose oxidase, phytase, lyase, pectinase, glucosidase, mannosidase, isomerase, invertase, trasferase, ribonuclease, galactosidase, transglutaminase   and   chitinase.   The   TPAP-deficient   cells   may<br><br>
also be used to express heterologous proteins of pharmaceutical interest such as hormones, growth factors, receptors, and the like.<br>
It will be understood that the term "eukaryotic proteins" is intended to include not only native proteins, but also those proteins (e.g. enzymes) which have been modified by amino acid substitutions, deletions, additions, or other modifications which may be made to enhance activity, thermostability, pH tolerance  and  the  like.<br>
In a further aspect the invention relates to a protein product essentially free from TPAP activity which is produced by the method  of  the  invention.<br>
EXAMPLE   1<br>
PURIFICATION  AND  CHARACTERIZATION  OF  A.   niger  TPAP<br>
Haterials  and  methods<br>
Phe-Pro-Ala-pNA sul^strate  (available from Bachem,  Switzerland) .<br>
Protease  inhibitors<br>
Ma-Ala-Phe-chloromethyIketone<br>
3enzyloxycarbonyl-Ala-Pro-Phe-chloromethylketone<br>
3enzyloxycarbonyl-Gly-Gly-Phe-chloromethyl-ketone.<br>
?urification  of  TPAP<br>
PPAP was purified from a commercial A. niger AMG preparation [available from Novo Nordisk A/S, Denmark). A sample of 300 ml formulated AMG product was repeatedly diluted and concentrated it 4'C in a FiltronÂ® concentrator equipped with a 3 kDa cutoff lembrane until the conductivity was less than 1.5 mS/cm. All )ther purification steps were carried out at ambient tempera-:ure.<br><br>
Cation exchange chromatography employing a HaCl gradient The concentrate (600 ml) was adjusted to pH 4.0 and filtrated before application to a 200 ml (2.6 x 37 cm) S-Sepharose column equilibrated with 20 nM sodium acetate, pH 4.0. A flow of 10 ml/min was used. The TPAP was eluted with a linear NaCl gradient from 0 to 0.2 M in 10 column volumes. One pool conÂ¬taining the largest part of the peptidase activity was made. A buffer change to 20 mM sodium acetate, pH 5.5 was made in an Amicon cell equipped with a Diaflo membrane with a cutoff of 10 kDa.<br>
Anion  exhange  chromatography<br>
The pool from the S-Sepharose column was further purified on a HiLoad Q-Sepharose HP column (50 ml, 2.6 x 10 cm) equilibrated with 20 mM sodium acetate, pH 5.5. Elution of TPAP was perÂ¬formed with a linear NaCl gradient from 0 to 0.5 M in 15 column volumes. The protein was applied with a flow of 8.0 ml/min and eluted with 5.0 ml/min. A pool containing the largest part of TPAP was made. A buffer change to 20 mM sodium acetate, pH 4.0 was  made  in  an  Amicon  cell  as  described  above.<br>
Cation exchange chromatography employing a PH gradient The pool from the HiLoad Q-Sepharose HP column was finally pur if ied on a Mono S column (5/ 5) equi librated with 20 mM sodium acetate, pH 4.0. A gradient from pH 4.0 to pH 6.0 was made in 30 column volumes using 20 mM sodium acetate, pH 4.0 and 20 raM sodium acetate, pH 6.0. The flow was 1.0 ml/min. Two isoenzymes  of  TPAP  eluted  at  pH  5.1  and  5.2,   respectively.<br>
Furificatiott  of  AHG<br>
AMG Gl has been purified from a commercial AMG preparation (Novo Nordisk A/S) by anion exchange chromatography using Q-Sepharose. A 50 ml column was equilibrated with 20 mM sodium acetate, pH 5.5 and the Gl form eluted with a linear NaCl gradient from 0 to 0.6 M NaCl in 8 column volumes. The flow was 8.0  ml/min.<br><br>
AMG, was purified after TPAP treatment of AMG Gl on a Mono Q column equilibrated with 20 nM sodium acetate, pH 4.3. A linear NaCl gradient from 0 to 1.0 M in 30 column volumes was used for elution.   The  flow  was  1.0  ml/min.<br>
Destabilization  assay<br>
Aliquots of purified AMG Gl were incubated with different mixtures of TPAP and buffer in O.i M sodium acetate, pH 4.3 for several weeks at 37C. The final volume was either 1 or 2 ml with a concentration of AMG Gl of 10 AGU/ml. 100 Ml of the incubation mixture were withdrawn and diluted to 2 AGU/ml with 0.1 M sodium acetate, pH 4.3. A heat treatment at 65Â°C for 30 min was carried out on the diluted sample. After cooling the samples to ambient temperature, the activity of untreated and heat treated samples were measured in microtiter plates using the chroinogenic substrate p-nitrophenyl-a-D-glycopyranoside (pNPG) (Merck, Art. 6792). 50 ^1 3 mM pNPG in 0,1 M sodium acetate, pH 4.3 was incubated with 25 ^i sample for 30 min at ambient temperature. The reaction was stopped by addition of 75 Âµ1 0.1 M sodium tetraborate. The absorbance at 405 nm was measured in a UV-max kinetic microplate reader. T30 was calÂ¬culated as the percentage of activity retained after heat -treatment.   All  measurements  were  made  in  duplicate.<br>
TPAP  assay<br>
The assay was performed either in a microtiter plate reader or in a spectrophotometer using a substrate concentration of 0.2 mM Phe-Pro-Ala-pNA in 0.1 M sodium acetate buffer, pH 4.3. A 5 mM stock solution of Phe-Pro-Ala-pNA was made in DMSO and diluted before use. The reaction was followed for 4 min at 405 nm either in a UV-max kinetic microplate reader or a spectrophotometer  and  the  initial  rate  of  cleavage  calculated.<br>
inhibition  of  TPAP<br>
Aliquots containing 4 ng TPAP were incubated with a number of protease inhibitors: 1 mM Ala-Ala-Phe-chloromethylketone, l mM benzyloxycarbonyl-Ala-Pro-Phe-chloromethylketone  and  1  mM  ben-<br><br>
zyloxycarbonyl-Gly-Gly-Phe-chloroinethyl-ketone. Following 30 min  incubation  the  residual  activity  was  determined.<br>
Storage  stability  of  filtrated  fermentation  broths<br>
The culture broths were centrifuged and sterile filtrated through a o. 22 [MTR filter. The broths were made 0.1% with respect to potassium sorbate and sodium benzoate and pH was adjusted to 4.3. Aliguots of 2 ml were either added 200 Âµ1 of 10 mM Ala-Ala-Phe-chloromethylketone or 200 Âµ1 of 0.1 M sodium acetate, pH 4.3. Samples were withdrawn and diluted to 2 AGU/ml before  the T30  determination  as  described  above.<br>
Determination  of  AMG  activity   (AGO)<br>
AMG   activity  was  determined  as  described   By  K.A.   Holm,   1980, Anal.   Chem.   Acta.,    117,   pp   359-362.   In   brief,   the   method   is based  on  hydrolysis  of  maltose  by AMG  under  formation  of  alpha-D-glucose.   After   a   short,   continuous   dialysing  procedure   the concentration  of  glucose  is  determined  by  a  glucose  dehydrogeÂ¬nase   (GlucDH)   reaction   (performed  at  pH  7.6).   Standard  condiÂ¬tions  for  the  automated  Auto-Analyzer  method  are: substrate:   maltose  28  mM Incubation  buffer:   acetate  O.IM,   pH  4.3 Incubation  temperature:   37'C Incubation  time:   5  min. The  enzyme-working  area  of  the  method  is  0.5-4  AGU/ml.<br>
Results<br>
TPAP was purified to homogeniety from a commercial AMG preparaÂ¬tion (Novo Nordisk A/S) according to the purification scheme shown in Table 1. The elution profile from the final cation exchange column revealed the presence of two isoenzymes of TPAP (TPAP-I and TPAP-II) with pi 5.1 and 5.2, respectively. Both isoforms were pure as judged by SDS-PAGE and N-terminal sequencing, but the specific activities of the enzymes differed (of. Table 1). TPAP-II had 20% higher specific activity than TPAP-I.<br><br>
A deamidation of one or several Asn- or Gin-residues either in the fermentation broth or during purification can explain the small  difference  in  pI  between  the  two  isoforms  of  TPAP.<br><br><br><br>
pH  optimum<br>
The pH optimum of TPAP was determined using the same procedure as in the TPAP assay described above. 0.1 M acetate buffer was used to regulate the pH, The resulting pH optimum curve is shown in Fig. 1. It is seen that the TPAP functions optimally at  a  pH  in  the  range  of  5.0-5.5,   in  particular  5.25.<br>
Determination  of  temperature  optimum  for  TPAP The temperature/activity relationship of TPAP was determined  in 0,1   M   sodium   acetate   buffer,    pH   5.5   using   the   TPAP   assay described  above.  As  can  be  seen  from  the  table  TPAP  has  maximal activity  in  the  temperature  range  45-55   Â°C.<br><br>
Mass  spectrometry<br>
Matrix assisted laser desorption ionisation time-of-flight mass spectrometry of TPAP purified from A. niger gave a broad signal indicating that the TPAP is glycosylated. The average mass was found  to  be  54.5  kDa.<br>
EXAMPLE   2<br>
PURIFICATION  AND  CHARACTERIZATION  OF  A.   oryzae  TPAP<br>
Materials  &amp;  Methods<br>
As a starting material for the purification, the supernatant of an A. oryzae IPC 4177 fermentation fermented at pH 5 in a soy containing medium was used. After fermentation, the culture broth  was  centrifuged  to  remove  the  majority  of  cells.<br><br>
Purification<br>
Approx. 4L supernatant was germ filtered on a Seitz EKS plate. The EKS-filtrate was ultrafiltrated on a 3k cut-off Filtron cassette (Minisette) to minimal volume. Ultrafiltrate = 240 ml. 170 ml of the ultraf iltrate was precipitated with solid ammonium sulphate (AMS) to give an AMS saturation of approx. 90%. After stirring for at least 30 min., the AMS precipitate was recovered by centrifugation in a Sorvall RC3B centrifuge (4500 rpm, 15 min., room temp) . 100 ml deionized water was added to the AMS precipitate to dissolve the protein and 1% (w/v) FGV120 activated charcoal was added to remove colour. After stirring for approx. 1 hour the suspension was filtered on a Seitz EKl plate to remove charcoal (and colour). The EKl-filtrate was dialysed against 1) 100 mM HjBO,, 10 mM dimethyl f glutaric acid, 2 niM CaClj, pH 5 and 2} deionized water. After an EKi-filtration the dialysate (260 ml) was freezed in aliquots.<br>
A 120 ml aliquot of the dialysate was thawed and applied to a 1.4L G25 Sephadex column equilibrated in 20 mM CHjCOOH/NaOH, pH 5.0. To remove colour and very acidic proteins, the G25-filtrate was applied to a 40 ml Q-sepharose FF column equilibrated in the same buffer (CHjCOOH/NaOH, pH 5.0). After washing the column, bound protein was eluted with a linear NaCl gradient (0 â&gt; 200mM). Fractions from the column were analysed for TPAP activity. Most of the TPAP activity was seen in the run-through (70%), whereas most of the protein was bound to the column.<br>
The run-through from the Q-sepharose column was applied to a 50 ml S-sepharose HP column equilibrated in 20 mM CHjCOOH/NaOH, pH 5.0. After washing the column, bound protein was eluted with a linear NaCl gradient (0 â&gt; 200mM). Fractions from the column were analysed for TPAP activity. Most of the TPAP activity was again seen in the run-through (75%) . The rest of the TPAP activity was in the start of the NaCl gradient. The run-through + fractions 1-11 were pooled and dialysed against 50 mM H3BO3, 5  mM  dimethyl  glutaric  acid,   l  mM  CaCl2,   pH  6.0.<br><br>
After adjusting the pH of the dialysed pool to pH 7.0 the enzyme was applied to a 23 ml SOURCE Q column equilibrated in 50 mM H3BO3, 5 MM dimethyl glutaric acid, l mM CaCl^, pH 7.0. After washing the column, bound protein was eluted with a linear NaCl gradient (0 â&gt; 500 mM). Fractions from the column were analysed for TPAP activity. All the TPAP activity was seen in the run-through (95%). The reason for this has to be, that the column had been overloaded with protein. The run-through was  dialysed  against  20  mM  CHjCOOH/NaOH,   pH  4.0.<br>
The dialysed enzyme was applied to a 50 ml S-sepharose HP column equilibrated in 20 mM CHjCOOH/NaOH, pH 4.0. After washing the column, bound protein was eluted with a NaCl gradient (0 â&gt; 200 mM). Fractions from the column were analysed for TPAP activity. The TPAP activity eluted with 100 mM NaCl. The TPAP activity was pooled and dialysed against 20 mM CHjCOOH/NaOH, pH 4.0. To get a better resolution than what the S-sepharose column could give, the dialysed pool was applied to an 8 ml SOURCE S column equilibrated in the same buffer (20 mM CHjCOOH/-NaOH, pH 4.0). After washing the column, bound protein was eluted with a NaCl gradient (0 â&gt; 200 mM). Fractions from the column were analysed for TPAP activity. The TPAP activity eluted with 60 mM NaCl. The TPAP activity was pooled and dialysed  against  20  mM  CHsCOOH/NaOH,   pH   5.5.<br>
The dialysed enzyme was applied to a 23 ml SOURCE Q column equilibrated in 20 mM CH3COOH/NaOH, pH 5.5. After washing the column, bound protein was eluted with a NaCl gradient (0 â&gt; 200 mM) . The TPAP activity eluted with 30 mM NaCl. Fractions from the column were analysed by SDS-PAGE and for TPAP activÂ¬ity. The SDS-PAGE gel showed that the fractions containing TPAP activity contained two major bands: a 65 kDa band (TPAP) and a 3 0 kDa band. The TPAP band was diffuse whereas the 30 kDa band was sharp, indicating that TPAP is glycosylated whereas the 30 kDa  band  is  not.<br>
The TPAP containing fractions were concentrated to 1 ml by ultrafiltration  and  applied  to  a  150  ml  Sephacryl  S-loo  column<br><br>
equilibrated in 20 mM Tris-acetate, 100 mM NaCl, pH 5.5. With a 1.0 ml/min flow rate the two bands were separated. The TPAP activity  containing  fractions  were  pooled  as A.oryzae  TPAP.<br>
Mass  spectrometry<br>
Matrix assisted laser desorption ionisation time-of-flight mass spectrometry of TPAP purified from A. oryzae gave a broad signal indicating that the TPAP is glycosylated. The average mass  was  found  to  be  55.0  kDa.<br>
pH  profile  of  A.   oryzae  TPAP<br>
The pH dependency of the activity of TPAP from A. oryzae was investigated using the chromogenic substrate Phe-Pro-Ala-pNA. To 50 Âµ1 of enzyme 150 Âµl Britton-Robinson buffer at the indicated pH were added before incubation with 50 /il Phe-Pro-Ala-pNA at a final substrate concentration of 0.2 mM. The assays were performed in an UV-max kinetic microtiter plate reader and the reaction followed for 3.5 min at 405 nm. The relative  activity   (RA)   is  given  in  the  table  3  below.<br><br>
EXAMPLE   3<br>
Peptide  sequences  of A.   ni(jer  TPAP<br>
N-terminal  amino  acid  sequencing  of  intact A.  niger  TPAP  forms<br>
I	and II (Example 1) as well as of peptides derived from TPAP<br>
form I was performed in an Applied Biosystems 473A sequencer<br>
operated  according  to  the  manufacturers  instructions.<br>
The  N-terminal  amino  acid  sequences  of  intact  TPAP  forms  I  and<br>
II	were determined for 30 residues. The two sequences were<br>
identical  and  found  to  be:<br><br>
Ala-Gln-Asn-Thr-Ser-His-Cys-Asp-Ser-Ile-Ile-Thr-Pro-His-Cys-Leu-Lys-Gln-Leu-Tyr-Asn-Ile-Gly-Asp-Tyr-Gln-Ala-Asp-Pro-Lys-(SEQ  ID  No,   4)<br>
The  sequence  did  not  show  any  homology  to  other  proteins.<br>
Following denaturation, reduction and s-carboxymethylation, peptides were derived from TPAP form I by proteolytic cleavage using the lysyl-specific protease froro Achromobactex-. The resulting peptides were fractionated and repurified using revesed phase HPLC. The purity and mass of the peptides were evaluated using matrix assisted laser desorption ionisation time-of-flight mass spectrometry using a VG Analytical TofSpec operated according to the manufacturers recommandations. The following  7  peptides  were  sequenced.<br>
Peptide  1:<br>
Ala-Gln-Asn-Thr-Ser-His-Cys-Asp-Ser-Ile-Ile-Thr-Pro-His-Cys-Leu-Lys<br>
Asn3 is glycosylated as shown by mass spectrometry and amino acid sequencing. Peptide 1 is identical to amino acid residues 1-17   of   intact  TPAP   and  thus   of  the   sequence   shown   in  SEQ   ID<br>
No.   4.<br>
Peptide 2:<br>
Gln-Leu-Tyr-Asn-Ile-Gly-Asp-Tyr-Gln-Ala-Asp-Pro-Lys<br>
Peptide 2 is identical to amino acid residues 18-30 of intact TPAP (and thus of the amino acid sequence shown in SEQ ID No. 4) . Peptide 2 is recovered in two forms as revealed by mass spectrometry and amino acid sequencing. One form with an N-terminal Gin residue and one form with this Gin residue conÂ¬verted  to  a  pyroglutamate  residue.<br>
Peptide 3:<br>
Thr-Ser-Pro-Glu-Gln-Ala-Val-Ser-Phe-Ser-Ser-Gly-Gly-Phe-Ser-Asp-Leu-Trp-Pro-Arg-Pro-Ser-Tyr-Gln-His-   (SEQ   ID  No.   5)<br><br>
Peptide 4:<br>
Phe-Ser-Gly-Leu-Phe-Asn-Ala-Ser-Gly-Arg-Ala-Phe-Pro-Asp-Val-Ser-Ala-Gln-Gly-Val-Asn-Tyr-Ala-Val-Tyr-Asp-Lys   fSEQ  ID  No.   6)<br>
Asn6 is glycosylated as shown by mass spectrometry and amino acid  sequencing.<br>
Peptide 5:<br>
Ile-Gly-Phe-Ala-Ser-Tyr-Leu-Gln-Glu-Tyr-Ala-Arg-Tyr-Ala-Asx-Leu-Glu-Arg-Phe-Glu-Gln-His-Leu-   (SEQ  ID  No.   7)<br>
It was not possible to discriminate whether Asxis was an Asp or Asn  residue.<br>
Peptide  6:<br>
Xaa-Leu-Asx-Leu-Gln-Tyr-Ile-Leu-Gly-Val-Ser-Ala-Pro-Val-Pro-Ile-Thr-Glu-Tyr-Ser-Thr-Gly-Gly-Arg-Gly-Glu-Leu-Val-Pro- (SEQ ID  No.   8)<br>
It was not possible to discriminate whether Asx3 was an Asp or Asn  residue.   Xaal  was  unidentified.<br>
Peptide  7:<br>
Gly-Ala-Leu-Asx-Asp-Ile-Val-Asn-Gly-Thr-Ser-Val-Gly-Gln-Asp-Gly-Arg-Asn-Arg-Phe-Gly-Gly-Thr-Pro-Asn-Gly-Ser-  (SEQ ID No.  9)<br>
It was not possible to discriminate whether Asx4 was an Asp or Asn residue. Note that Asn25 is not glycosylated although it is found  in  the  concensus  sequence  for  N-glycosylation.<br>
Peptide  sequences  of  A.   oryzae  TPAP<br>
N-terminal amino acid sequencing intact TPAP as well as of peptides derived from TPAP was carried out in an Applied Biosystems 473A protein sequencer according to the manufacÂ¬turers  instructions.<br><br>
The N-terminal amino acid sequence of intact TPAP were deterÂ¬mined following SDS-PAGE and electroblotting onto a PVDF-membrane using standard procedures. The following 23 residue amino  acid  sequence  was  found:<br>
Ala-Lys-Xaa-ile-Ser-His-Yaa-Asp-Ser-Ile-Ile-Thr-Pro-Pro-Yaa-Leu-Lys-Glu-Leu-Tyr-Asn-Ile-Gly-   (SEQ  ID  NO  14)<br>
This sequence is clearly homologous to the N-terminal amino acid sequence of TPAP from A. niger. Based on this homology Xaa most likely is a glycosylated Asn-residue while Yaa probably represents  Cys-residues.<br>
Following denaturation, reduction and S-carboxymethylation, peptides were derived from TPAP by proteolytic cleavage using the lysyl-specific protease from Achromojbacter. The resulting peptides were fractionated and repurified using revesed phase HPLC. The purity and mass of the peptides were evaluated using matrix assisted laser desorption ionisation time-of-flight mass spectrometry using a VG Analytical TofSpec operated according to the manufacturers recommandations. The following 4 peptides were  sequenced.<br>
Peptide  8:<br>
Glu-Leu-Tyr-Asn-Ile-Gly-Asp-Tyr-Gln-Ala-Asp-Ala-Asn-Ser-Gly-Ser-Lys   (SEQ   ID  NO   10)<br>
This peptide overlaps with the 6 last amino acid residues determined by the N-terminal sequencing of intact TPAP thereby extending  the  N-terminal  sequence  to  34  residues.<br>
Peptide  9:<br>
Thr-Thr-Pro-Glu-Arg-Gly-Thr-Tyr-Phe-Ser-Ser-Gly-Gly-Phe-Ser-Asn-Tyr-Trp-Pro-Arg-Pro-Glu-Trp-Gln-Asn-Gln-Ala-Val-Ala-Ser-Tyr-Leu-   (SEQ   ID  NO   11)<br>
This  peptide  is  homologous  to  peptide  3   from A.   niger  TPAP.<br><br>
Peptide  10:<br>
Gly-Thr-Leu-Gly-Glu-Phe-Asp-Gly-Thr-Ser-Ala-Ser-Ala-pro-Ala-Phe-Ser-Ala-Val-Ile-Ala-Leu-Leu-Asn-Asp-Ala-Arg-Leu-Arg-Ala-Gly-Lys-Pro-Thr-Leu-Gly-Phe-Leu-Asn-Pro-Trp-Leu-Tyr-Lys (SEQ ID NO   12)<br>
Peptide  11:<br>
Thr-Gly-Arg-Gln-Gly-Leu-Gln-Asn-Ile-Thr-Leu-Gly-Ala-Ser-Ile-Gly-Xaa-Thr-Gly-Arg-Ala-Arg-Phe-Gly-Gly-Ala-Pro-Asn-Gly-Gly-Pro-Val-Val-Pro-Tyr-Ala-Ser-   (SEQ   ID  NO   13),<br>
Xaa  designates  an  unidentified  residue.<br>
EXAMPLE  4<br>
The amino acid compositions of A. niger TPAP forms i and II were determined. Duplicates of lyophilyzed aliqouts of TPAP forms I and II were hydrolysed in 6 N HCl containing 0.1% phenol at 110Â°C in vacuo for 16 h. Trypthophan was determined following hydrolysis in 3M methanesulfonic acid. Following hydrolysis the amino acid compositions were determined using an Applied Biosystems 420A amino acid analysis system operated according to the manufacturers instructions. The results show that within experimental error TPAP forms I and II have idenÂ¬tical  amino  acid  compositions  as  shown  in  Table  4  below.<br><br><br><br>
1 h, 2 h and 4 h. Following hydrolysis the monosaccharide compositions were analysed by high performance ion exchange chromatography using a CarboPacTM PAl column eluted with 16 mM NaOH. The monosaccharides were detected by pulsed amperometric detection. Due to the different stability and release of the monosaccharides in 2 M TFA the amount of galactose was deterÂ¬mined after 1 h hydrolysis, the amount of mannose after 2 h hydrolysis and the amount of glucosamine after 4 h hydrolysis. The results obtained indicate very minor differences in mannose content  of  TPAP  form  I  and  II  as  shown  in  the  table  below.<br>
Table   5<br>
Monosaccharide compositions of TPAP forms I and II. The results are given in pmol monosaccharide/pmol protein as determined from  amino  acid  analysis.<br><br>
EXAMPLE   6<br>
CLEAVAGE  BY  A.   niger  TPAP<br>
The ability of A. niger TPAP to destabilize the AMG Gl form was investigated using purified AMG and TPAP (obtained as described in the Materials and Methods section in Example 1) at TPAP/AMG ratios equal to those in formulated products. Amino acid seÂ¬quencing of the destabilized preparations revealed a modificaÂ¬tion in the N-terminal of the catalytic domain of AMG. A tripeptide comprising the first three amino acid residues (Ala-Thr-Leu) had been cleaved off by the peptidase indicating that the enzyme is a tripeptidyl aminopeptidase. The classification I   as  a  tripeptidyl  aminopeptidase,   based  on  the  cleavage  of  AMG<br><br>
as well as of the chromogenic substrate Phe-Pro-Ala-pNA, was further supported by the lack of activity towards another chromogenic substrate Succinyl-Ala-Ala-Ala-pNA. In this substrate the free amino group at the N-terminus has been succinylated and thereby made the substrate inaccesible to cleavage by TPAP. The TPAP cleavage of the AMG batches was not complete after 3 weeks storage, as a mixture of intact and truncated  AMG  (AMG,njâ)  was  detected  by  amino  acid  sequencing.<br>
The destabilization of AMG Gl obtained with various dosages of TPAP at different temperatures has been investigated. A TPAP dosage in the range from 3 to 10 f^g/ml gave a significant destabilization of AMG and also TPAP/AMG ratios similar to those measured in several fermentations. The destabilization effect was most pronounced at 37Â°C but also at 25*'C a clear effect on the thermostability was observed after 27 days of storage. Storage at 4Â°C did not affect the thermostability of AMG.<br>
EXAMPLE   7<br>
The specificity of A. niger TPAP was investigated using various native protein and peptide substrates. Using these substrates it was found that TPAP is highly unspecific with respect to the amino acid residue N-terminal to the cleavage point. TPAP is capable of cleavage even following Pro residues and CM-Cys residues. However, a Pro residue C-terminal to the cleavage point completely inhibits cleavage by TPAP. Specific cleavage products obtained from subjecting native proteins to TPAP treatment include the tripeptide sequences IPE, yVD, WRQ, KGA, LPS, ANL, NGT, LMQ, YFE, GPG, GGG, ADG, RST, SVE, KKP, EGV, NTG, AGD, RHN, LKT, VEK, KPE, GVN, TGA, GDR, HNL, HSQ, GTF, TSD,   YSK,   YLD,   SRR,   AQD,   FVQ,   WLM  and  ATL.<br><br>
EXAMPLE  8<br>
INHIBITION  OF A.   niger  TPAP  ACTIVITY<br>
Inhibition of TPAP by the protease inhibitor, cloromethylketone Ala-Ala-Phe-CMK (AAF-CMK), was tested under the conditions described  above.  AAF-CMK was  found  to  inhibit  TPAP  completely.<br>
Addition of AAF-CMK to fermentation broths can totally inhibit the TPAP activity. The thermostability of 5 different AMG batches were investigated with and without the TPAP activity present. The thermostability of all 5 AMG batches were unchanged after 2 weeks storage at 37 Â°C when TPAP had been inhibited. The corresponding samples without inhibitor were all clearly  destabilized.<br>
EXAMPLE   9<br>
Portions of 10 ml culture broths obtained from a cultivation of an AMG producing A. niger strain were adjusted to either pH 6.5 or pH 7.0 with sodium hydroxide. The samples were incubated at 25'C, 40Â°C and 50Â°C, respectively, for one hour and subsequently the samples were adjusted to pH 4.3 with acetic acid. The storage stability of the treated samples was measured. It was shown that this simple recovery process {based on a pH treatment at elevated temperatures) resulted in efficient removal of TPAP activity (Table 6). Treatment of the culture broth at pH 6.5/40'C for one hour reduced the TPAP activity to 5% and gave a very stable product with an unchanged thermostability  after  two  weeks  of  storage  at  40Â°C.<br><br><br>
All numbers in the table are %. The values indicated for TPAP and AMG, respectively, are the relative activity at the time indicated.<br>
EXAMPLE   10 PCR   CLONINQ<br>
From the K-terminal amino acid sequence of A. niger TPAP the sequence of which is shown in SEQ ID Wo. 4 four PCR primers were designed (Table 7} Genomic DMA from a strain of A. niger was used as template in four PCR reactions. PCR fragments in the expected size 65 bp was purified and cloned into the plasTtiid pCRâ¢II (Invitrogen Corporation) . Sequencing of the insert for three clones showed that two of them contains the degenerate sequence in the areas corresponding to the primers, while the sequence in between was invariant to the N-terminal amino  acid  sequence.<br>
In order to clone a larger DNA fragment encoding the tripepti-dyl aminopeptidase the primer #6010 (GCACTGTCTGAAGCAGCTGTACAAC-ATCGGTG) corresponding to the invariant sequence was made. From two other peptide sequences three PCR primers were designed (Table 7). Three PCR reactions (#6010/#59S8, #6010/#5989, and #60l0/#5990 were performed using genomic A. niger DNA as template. The reactions were done at two annealing temperatures -iior-  =â^  Ac^an    Roarri-inriR  *6niQ/#5989   (one  fragment  or  approx.<br><br>
80 bp) and #6010/#5990 (three fragments of approx. 120 bp, 500 bp, and 950 bp) show the same band pattern on an agarose gel at the two different temperatures. In reaction #6010/#5988 a fragment of approx 120 bp was seen at 42"C, but at 45Â°C a fragment of approx. 950 bp was seen. The 950 bp fragment turned out to be the right one and was inserted into the pCR II AT vector. An E, coli strain harbouring the 950 kb fragment was deposited with the Deutsche Sammlung von Microorganismen und ZelIkulturen GmbH, Mascheroder Weg lb, D-38124 Braunschweig, Germany on December 5, 1994 as DSM 9570, under the provisions of  the  Budapest  Treaty.<br><br><br><br>
The 950 bp fragment was sequenced from both ends. It was found that this fragment encodes the N-terminal part of TPAP. The partial sequence of the N-terminal part is shown in SEQ ID No. 2, the partial sequence obtained from sequencing from the other end is shown in SEQ ID No. 3. The sequence of the entire "950 base fragment" which was found to be constituted by 908 bp, is shown  in  SEQ  ID  No.   1.<br>
Southern  analysis<br>
Genomic DNA from a strain of A.niger and from A. oryzae IFO 4177 was isolated as described previously fVelton et al., 1984, Proc. Natl. Acad. Sci. USA. 81: 1470-1474). Genomic DNA was digested with appropriate restriction enzymes, fractionated on a 0.7% agarose gel, and blotted to ImmobilonTM as described by the manofacturer. The membranes were probed for the presence of the 950 bp TPAP gene sequence labelled with alpha-32 dATP (NewEngland) by random priming according to the method described by Feinberg et al., 1983, Anal. Biochem. 132:6. The membranes were then incubated for 2 hours at 65'C in hybridizaÂ¬tion solution (5xSSC (0.15 M NaCl, 0.015 M trisodium citrate), lOxDenhardt (0.2% Ficoll, 0.2% polyvinyl pyrrolidone, 0.2% bovine serum albumin) , 10 mM EDTA, 1% SDS, l50/ig/ml Poly A, 50 Mg/ml yeast RNA). Next, the radioactively labelled probe was added and incubated at 65Â°C overnight with gentle agitation. The memebranes were washed twice at 30Â°C for 15 minutes in 2XSSC, 1% SDS. Finally, the membranes were dried, covered with plastic  wrap,   and  exposed  to  x-ray  film  (Fuli-RX)   at ~70Â°c.<br><br>
the   results    show   that   both    strains    contain    only   one   gene encoding  the  TPAP.<br><br>
SEQUENCE  LISTING<br>
SEQ  ID  No.    I 90S  bases.<br>
CACTGTCTGAAGCAGCTGTACAACATCGGTGACTACCAGGCCGATCCCAAGTCCGGCAG-CAAGATCGGCTTTGCCAGCTACCTTGAGGAATACGCCCGGTATGCCGATCTCGAGAGG-TTCGAGCAGCACCTGGC TCCCAATGCCATCGGCCAGAACTTCAGCGTCGTCCAATTCAACG-GCGGCCTCAACGATCAGCTTTCA TCGAGTGACAGCGGCGAAGCCAACCTCGACCTGCAGTA-CATCCTGGGCGTCAGCGCTCCCGTCCCCA TCACCGAGTACAGCACCGGCGGACGCGGCGAA-CTAGTCCCCGACCTGAGCTCCCCCGACCCCAACGA CAACAGCAACGAGCCCTACCTTGACT-TCCTTCAGGGAATCCTCAAGCTTAACAACTCCGACCTCCCA CAAGTCATCTCTACCTCCTA-CGGTGAAGACGAACAGGTATGCACCTCACCTGACCCATTCCATTTTA CATCCCTCACCTCT-CTCAACCAAACTAACAACACCAACAGACTATCCCCGTCCCCTACGCCCGCACC GTCTGCAA-CCTCTACGCCCAACTCGGCAGCCGCGGCGTCTCTGTAATCTTCTCCAGCGGCGACTCCG GCGTCGGCGCCGCCTGCCTCACCAACGACGGCACCAACCGCACGCACTTCCCTCCTCAATT CCCCGC CTCCTGCCCCTGGGTAACCTCCGTCGGCGCAACCTCCAAGACCTCCCCCGAGCAA-GCCGTCTCCTTC TCCTCCGGCGGCTTCTCCGACCTCTGGCCCCGCCCCTCCTACCAACACG-CCGCCGTGCAAACCTACC TCACCAAGCACCTGGGCAACAAGTTCTCGGGGCTTTTCAACGC-CTCCGGCCGCGCCTTCCCCGACGTCTCCGCGCAGGGCGTCAACTACGCTGTTTACGACAAA<br>
SEQ   ID  No.    2<br>
CACTGTCTGAAGCAGCTGTACAACATCGGTGACTACCAGGCCGATCCCAAGTCCGGCAGCA AGATCGGCTTGGGCAGCTACCTTGAGGAATACGCCCGGTATGCCGATCTCGAGAGG TTCGAGCAGCACCTGGCTCCAATGCATCGGCAGAACTCAGCGTCGTCCAATTCACGGCGGCT-CACGATCAGCTTCATCGAGTGACAGCGGCGAGCAACTCGACTGCAGTAC<br>
SEQ  ID  NO.   3<br>
CCTCCTACCAACACGCCGCCGTGCAACCTACCTGACCAAGCACCTGGCAACAAGTTCTCGG-GGCTTTTCAACGCCTCCGGCCGCGCCTTCCCCGACGTCTCCGCGCAGGGCGTCAACTACGCT-GTTTACGACAA<br>
SEQ   ID  No.    4<br>
Ala-Gln-Asn-Thr-Ser-His-Cys-Asp-Ser-Ile-Ile-Thr-Pro-His-Cys-Leu-Lys-Gln-Leu-Tyr-Asn-Ile-Gly-Asp-Tyr-Gln-Ala-Asp-Pro-Lys<br><br>
Thr-Ser-Pro-Glu-Gln-Ala-Val-Ser-Phe-Ser-Ser-Gly-Gly-Phe-Ser-Asp-Leu-Trp-Pro-Arg-Pro-Ser-Tyr-Gln-His-<br>
SEQ  ID  NO,   6<br>
Phe-Ser-Gly-Leu-Phe-Asn-Ala-Ser-Gly-Arg-Ala-Phe-Pro-Asp-Val-<br>
Ser-Ala-Gln-Gly-Val-Asn-Tyr-Ala-Val-Tyr-Asp-Lys<br>
SEQ   ID  No.   7<br>
Ile-Gly-Phe-Ala-Ser-Tyr-Leu-Gln-Glu-Tyr-Ala-Arg-Tyr-Ala-Asx-Leu-Glu-Arg-Phe-Glu-Gln-His-Leu-<br>
SEQ   ID  No.   8<br>
Xaa-Leu-Asx-Leu-Gln-Tyr-Ile-Leu-Gly-Val-Ser-Ala-Pro-Val-Pro-Ile-Thr-Glu-Tyr-Ser-Thr-Gly-Gly-Arg-Gly-Glu-Leu-Val-Pro-<br>
SEQ   ID  No.   9<br>
Gly-Ala-Leu-Asx-Asp-Ile-Val-Asn-Gly-Thr-Ser-Val-Gly-Gln-Asp-Gly-Arg-Asn-Arg-Phe-Gly-Gly-Thr-Pro-Asn-Gly-Ser-<br>
SEQ   ID  NO.   10<br>
Glu-Leu-Tyr-Asn-Ile-Gly-Asp-Tyr-Gln-Ala-Asp-Ala-Asn-Ser-Gly-<br>
Ser-Lys<br>
SEQ   ID  NO.    11<br>
Thr-Thr-Pro-Glu-Arg-Gly-Thr-Tyr-Phe-Ser-Ser-Gly-Gly-Phe-Ser-Asn-Tyr-Trp-Pro-Arg-Pro-Glu-Trp-Gln-Asn-Gln-Ala-Val-Ala-Ser-Tyr-Leu-<br>
SEQ   ID  NO.   12<br>
Gly-Thr-Leu-Gly-Glu-Phe-Asp-Gly-Thr-Ser-Ala-Ser-Ala-Pro-Ala-Phe-Ser-Ala-Val-Ile-Ala-Leu-Leu-Asn-Asp-Ala-Arg-Leu-Arg-Ala-Gly-Lys-Pro-Thr-Leu-Gly-Phe-Leu-Asn-Pro-Trp-Leu-Tyr-Lys<br><br>
SEQ   ID  NO.    13<br>
Thr-Gly-Arg-Gln-Gly-Leu-Gln-Asn-Ile-Thr-Leu-Gly-Ala-Ser-Ile-Gly-Xaa-Thr-Gly-Arg-Ala-Arg-Phe-Gly-Gly-Ala-Pro-Asn-Gly-Gly-Pro-Val-Val-Pro-Tyr-Ala-Ser-<br>
SEQ   ID  NO   X4<br>
Ala-Lys-Xaa-Ile-Ser-His-Yaa-Asp-Ser-Ile-Ile-Thr-Pro-Pro-Yaa-<br>
Leu-Lys-Glu-Leu-Tyr-Asn-Ile-Gly-<br>
SEQ   ID  NO   15<br>
Ala-Xaa(1)-Asn-Xaa(2)-Ser-His-Cys-Asp-Ser-Ile-Ile-Thr-Pra-Xaa<br>
(3} -Cys-Leu-Lys-Xaa (4) -Leu-Tyr-Asn-IIe-Gly-Asp-Tyr-Gln-Aia-Asp-<br>
Xaa(5)-Xaa(6)<br><br><br><br>
WE CLAIM:<br>
1.	A method of reducing or eliminating the tripeptidyl aminopeptidase (TPAP)<br>
production from a TPAP producing cell of the genus Aspergillus, in which method a<br>
DNA sequence encoding TPAP or a part thereof essential for exhibiting TPAP activity<br>
or the TPAP promoter sequence required for the expression of TPAP from a TPAP<br>
encoding DNA sequence present in said cell are modified or inactivated so as to result in<br>
a reduced or no TPAP production from said cell.<br>
2.	The method as claimed in claim 1, in which the modification or inactivation of the DNA sequence is obtained by subjecting the TPAP producing cell to mutagenesis and selecting for cells for which the TPAP producing capability has been reduced or removed.<br>
3.	The method-as claimed in claim 2, in which the mutagenesis is performed by random or site-directed mutagenesis.<br>
4.	A method of reducing or eliminating the TPAP production from a TPAP producing cell of the genus Aspergillus, in which method a nucleotide sequence complementary to the TPAP encoding sequence and thus capable of hybridizing to TPAP mRNA produced in the cell is introduced into the cell in such a manner that the nucleotide sequence may be transcribed in the cell vinder conditions allowing the nucleotide sequence to hybridize to the TPAP mRNA and thus reduce the amount of TPAP translated from said mRNA or eliminate any such translation.<br>
5.	A microbial host cell prepared according to claims 1 to 4.<br>
6.	A method of preparing a product wherein at least 60% of the TPAP activity is removed, which method comprises transforming a host cell as claimed in claim 5 with a DNA sequence encoding the product, culturing the transformed cell under suitable conditions for expression of the product, and recovering the product from the culture.<br>
-47-<br><br>
7.	A method of preparing a product wherein at least 60% of the TPAP activity is removed, which product is encoded by a DNA sequence present in a TPAP expressing cell, which method comprises modifying or inactivation a DNA sequence present in said cell and encoding TPAP or a part thereof essendal for exhibiting TPAP activity or the TPAP promoter required for the expression of TPAP from a TPAP encoding DNA sequence as claimed in the method of any of claims 1 -4 and subsequently culturing the cell under suitable condidons for expression of the product, and recovering the product from the culture.<br>
8.	A method of preparing a product wherein at least 60% of the TPAP activity is removed by fermentation of a TPAP-producing cell of the genus Aspergillus which also produces the product, which method comprises adding an effecdve amount of Ala-Ala-Phe-chloromethylketone to the fermentation broth either during or after the fermentation has been completed, recovering the product of interest from the fermentation broth, and optionally subjecting the recovered product to further purification.<br>
9.	A method of preparing a product wherein at least 60% of the TPAP activity is removed by fermentation of a TPAP-producing cell of the genus Aspergillus which also produces the product, which method comprises subjecting the fermentation broth or an enzyme preparation isolated there from containing TPAP and the product of interest to a combined pH and temperature treatment for a sufficient period of time to reduce the TPAP activity, and optionally recovering the product from the treated broth.<br>
10.	The method according to claim 9, in which the TPAP reducing treatment is<br>
performed at a pH in the range of 6.5-7 and at a temperature in the range of 25-40Â° C.<br>
11.	The method according to any of claims 6-10, in which the product is an enzyme.<br>
12.	The method according to claim 11, in which the product is selected from an amylolytic enzyme, a lipolytic enzyme, a proteolytic enzyme, a cellulytic enzyme, an oxidoreductase, and a plant cell-wall degrading enzyme.<br>
-48-<br><br>
13. The method according to claim 12, in which the product is AMG, lipase, cutinase, cellulase, amylase, protease, peroxidase, transglutaminase, laccase, catalase, glucose oxidase, or phytase.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3Ny1tYXMtMTk5NSBhYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1477-mas-1995 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3Ny1tYXMtMTk5NSBhc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">1477-mas-1995 assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3Ny1tYXMtMTk5NSBjbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">1477-mas-1995 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3Ny1tYXMtMTk5NSBjb3JyZXNwb25kZW5jZSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">1477-mas-1995 correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3Ny1tYXMtMTk5NSBjb3JyZXNwb25kZW5jZSBwby5wZGY=" target="_blank" style="word-wrap:break-word;">1477-mas-1995 correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3Ny1tYXMtMTk5NSBkZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">1477-mas-1995 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3Ny1tYXMtMTk5NSBkcmF3aW5nLnBkZg==" target="_blank" style="word-wrap:break-word;">1477-mas-1995 drawing.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3Ny1tYXMtMTk5NSBmb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">1477-mas-1995 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3Ny1tYXMtMTk5NSBmb3JtLTE5LnBkZg==" target="_blank" style="word-wrap:break-word;">1477-mas-1995 form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3Ny1tYXMtMTk5NSBmb3JtLTI2LnBkZg==" target="_blank" style="word-wrap:break-word;">1477-mas-1995 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3Ny1tYXMtMTk5NSBmb3JtLTQucGRm" target="_blank" style="word-wrap:break-word;">1477-mas-1995 form-4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3Ny1tYXMtMTk5NSBmb3JtLTYucGRm" target="_blank" style="word-wrap:break-word;">1477-mas-1995 form-6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3Ny1tYXMtMTk5NSBvdGhlcnMtMS5wZGY=" target="_blank" style="word-wrap:break-word;">1477-mas-1995 others-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3Ny1tYXMtMTk5NSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">1477-mas-1995 others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3Ny1tYXMtMTk5NSBwZXRpdGlvbi5wZGY=" target="_blank" style="word-wrap:break-word;">1477-mas-1995 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="229250-inhibitors-of-pi3k-and-method-of-inhibiting-pi3k-in-leukocytes-osteoclasts-and-cancer-cells.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="229252-an-isolated-polypeptide-and-applications-thereof.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>229251</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1477/MAS/1995</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>12/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>20-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>16-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>15-Nov-1995</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NOVOZYMES A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>KROGSHOEJVEJ 36, DK-2880 BAGSVAERD,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>LEHMBECK, JAN</td>
											<td>C/O NOVO NORDISK A/S, NOVO ALLE, DK-2880 BAGSVAERD,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>HOLM, KAJ ANDRE</td>
											<td>C/O NOVO NORDISK A/S, NOVO ALLE, DK-2880 BAGSVAERD,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>RASMUSSEN, GRETHE</td>
											<td>C/O NOVO NORDISK A/S, NOVO ALLE, DK-2880 BAGSVAERD,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>HALKIER, TORBEN</td>
											<td>C/O NOVO NORDISK A/S, NOVO ALLE, DK-2880 BAGSVAERD,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N09/48</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/229251-a-method-of-reducing-or-eliminating-the-tripeptidyl-amino-peptidase-tpap-production-from-a-tpap-producing-cell-of-the-genus-aspergillus by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:55:36 GMT -->
</html>
