<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/229347-cytokine-receptor-zalpha11 by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:01:55 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 229347:CYTOKINE RECEPTOR ZALPHA11</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">CYTOKINE RECEPTOR ZALPHA11</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to an isolated polynucleotide that encodes a zalpha 11 polypeptide consisting of a sequence of amino acid residues selected form the group consisting of (a) the amino acid sequence as shown in SEQ ID NO:2 form amino acid number 20 (Cys), to amino acid number 237 (His): (b) the amino acid sequence as shown in SEQ IN NO:2 form amino acid number 20 (Cys), to amino acid number 225 (Leu); (c)the amino acid sequence as shown in SEQ ID NO-2 form amino acid 256 (Lys), to amino acid number 538 (Ser); and (d) the amino acid sequence as shown in SEQ ID NO-2 form amino acid number 20 (Cys), to amino acid number 538 (Ser).</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
BACKGROUND OF THE INVENTION<br>
Proliferation and differentiation of cells of multicellular organisms are controlled by hormones and polypeptide growth factors. These diffusable molecules allow cells to communicate with each other and act in concert to form cells and organs, and to repair damaged tissue. Examples of hormones and growth factors include the steroid hormones (e.g. estrogen, testosterone), parathyroid hormone, follicle stimulating hormone, the interleukins, platelet derived growth factor (PDGF), epidermal growth factor (EOF), granulocyte-macrophage colony stimulating factor (GM-CSF), erythropoietin (EPO) and calcitonin.<br>
Hormones and growth factors influence cellular metabolism by binding to receptors. Receptors may be integral membrane proteins that are linked to signaling pathways within the cell, such as second messenger systems. Other classes of receptors are soluble molecules, such as the transcription factors. Of particular interest are receptors for cytokines, molecxiles that promote the proliferation and/or differentiation of cells. Examples of cytokines include erythropoietin (EPO), which stimulates the development of red blood cells; thrombopoietin (TPO), which stimulates development of cells of the megakaryocyte lineage; and granulocyte-colony stimulating factor (G-CSF), which stimulates development of neutrophils. These cytokines are useful in restoring normal blood cell levels in patients suffering from anemia, thrombocytopenia, and neutropenia or receiving chemotherapy for cancer.<br>
The demonstrated in vivo activities of these cytokines illustrate the enormous clinical potential of, and need for, other cytokines, cytokine agonists, and cytokine antagonists. The present invention addresses these needs by providing new a hematopoietic cytokine receptor, as well as related compositions and methods.<br><br>
The present invention provides such polypeptides for these and other uses that should be apparent to those skilled in the art from the teachings herein.<br>
SUMMARY OF THE INVENTION<br>
Within one aspect, the present invention provides an isolated polynucleotide that encodes a zalphal 1 polypeptide comprising a sequence of amino acid residues that is at least 90% identical to an amino acid sequence selected from the group consisting of: (a) the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 20 (Cys), to amino acid number 237 (His); (b) the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 20 (Cys), to amino acid number 255 (Leu); (c) the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 256 (Lys), to amino acid number 538 (Ser); (d) the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 20 (Cys), to amino acid number 538 (Ser); and (e) the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 1 (Met) to amino acid number 538 (Ser), wherein the amino acid percent identity is determined using a FASTA program with ktup=l, gap opening penalty=10, gap extension penalty=l, and substitution matrix=BLOSUM62, with other parameters set as default. Within one embodiment, the isolated polynucleotide disclosed above comprises a sequence of polynucleotides that is selected from the group consisting of: (a) a polynucleotide sequence as shown in SEQ ID N0:4 from nucleotide 1 to nucleotide 1614; (b) a polynucleotide sequence as shown in SEQ ID N0:1 from nucleotide 126 to nucleotide 779; (c) a polynucleotide sequence as shown in SEQ ID N0:1 from nucleotide 126 to nucleotide 833; (d) a polynucleotide sequence as shown in SEQ ED N0:1 from nucleotide 834 to nucleotide 1682; (e) a polynucleotide sequence as shown in SEQ ID N0:1 from nucleotide 126 to nucleotide 1682; and (f) a polynucleotide sequence as shown in SEQ ID N0:1 from nucleotide 69 to nucleotide 1682. Within another embodiment, the isolated polynucleotide disclosed above comprises a sequence of amino acid residues selected from the group consisting of: (a) the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 20 (Cys), to amino acid number 237 (His); (b) the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 20 (Cys), to amino acid number 255 (Leu); (c) the<br><br>
amino acid sequence as shown in SEQ ID N0:2 from amino acid number 256 (Lys), to amino acid number 538 (Ser); (d) the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 20 (Cys), to amino acid number 538 (Ser); and (e) the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 1 (Met) to amino acid number 538 (Ser). Withm another embodiment, the isolated polynucleotide disclosed above consists of a sequence of amino acid residues selected from the group consisting of: (a) the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 20 (Cys), to amino acid number 237 (His); (b) the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 20 (Cys), to amino acid number 255 (Leu); (c) the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 256 (Lys), to amino acid number 538 (Ser); (d) the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 20 (Cys), to amino acid number 538 (Ser); and (e) the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 1 (Met) to amino acid number 538 (Ser). Within another embodiment, the isolated polynucleotide disclosed above further comprises a WSWSX domain. Within another embodiment, the isolated polynucleotide disclosed above further comprises a transmembrane domain. Within another embodiment, the isolated polynucleotide disclosed above comprises a transmembrane domain consisting of residues 238 (Leu) to 255 (Leu) of SEQ ID N0:2. Within another embodunent, the isolated polynucleotide disclosed above further comprises an intracellular domain. Within another embodiment, the isolated polynucleotide disclosed above comprises an intracellular domain consists of residues 256 (Lys) to 538 (Ser) of SEQ ID N0:2. Within another embodiment, the isolated polynucleotide disclosed above comprises an intracellular domain which domain ftirther comprises Box I and Box II sites, comprises an intracellular domain wherein the polypeptide further comprises an affinity tag.<br>
Within a second aspect, the present invention provides an expression vector comprising the following operably linked elements: a transcription promoter; a DNA segment encoding a zalphall polypeptide having an amino acid sequence as shown in SEQ ID N0:2 from amino acid number 20 (Cys), to amino acid number 538 (Ser); and a transcription terminator, wherein the promoter is operably linked to the DNA segment, and the DNA segment is operably linked to the transcription terminator.<br><br>
Within one embodiment, the expression vector disclosed above further comprisies a secretory signal seque'nce operably linked to the DNA segment.<br>
Within a third aspect, the present invention provides a cultured cell comprising an expression vector as disclosed above, wherein the cell expresses a polypeptide encoded by the DNA segment.<br>
Within a fourth aspect, the present invention provides an expression vector comprising: a transcription promoter; a DNA segment encoding a zalphall polypeptide having an amino acid sequence as shown in SEQ ID N0:2 from amino acid number 20 (Cys), to amino acid number 237 (His); and a transcription terminator, wherein the promoter, DNA segment, and terminator are operably linked. Within one embodiment, the expression vector disclosed above further comprises a secretory signal sequence operably linked to the DNA segment. Within another embodiment, the expression vector disclosed above further comprises a transmembrane domain operably linked to the DNA segment. Within another embodiment, the expression vector disclosed above further comprises a transmembrane domain consisting of residues 238(Leu) to 255 (Leu) of SEQ ID N0:2. Within another embodiment, the expression vector disclosed above further comprises an intracellular domain operably linked to the DNA segment. Within another embodiment, the expression vector disclosed above further comprises an intracellular domain consisting of residues 256 (Lys) to 538 (Ser) of SEQ IDN0:2.<br>
Within another aspect, the present invention provides a cultured cell into which has been introduced an expression vector as disclosed above, wherein the cell expresses a soluble receptor polypeptide encoded by the DNA segment. Within one embodiment, the cultured cell disclosed above is dependent upon an exogenously supplied hematopoietic growth factor for proliferation.<br>
Within another aspect, the present invention provides a DNA construct encoding a fusion protein, the DNA construct comprising: a first DNA segment encoding a polypeptide having a sequence of amino acid residues selected from the group consisting of: (a) the amino acid sequence of SEQ ID N0;2 from amino acid number 1 (Met), to amino acid number 19 (Gly); (b) the amino acid sequence of SEQ ID N0:2 firom ammo acid number 20 (Cys) to amino acid number 237 (His); (c) the<br><br>
amino acid sequence of SEQ ID N0:2 from amino acid number 20 (Cys) to amino acid number 255 (Leu); (d) the amino acid sequence of SEQ ID N0:2 from amino acid number 238 (Leu) to'-amino acid number 255 (Leu); (e) the amino acid sequence of SEQ ID N0:2 from.ainino acid number 238 (Leu) to amino acid number 538 (Ser); (f) the amino acid sequence of SEQ ID N0:2 from amino acid number 256 (Lys) to amino acid number 538 (Ser); and (g) the amino acid sequence of SEQ ID N0:2 from amino acid number 20 (Cys), to amino acid number 538 (Ser); and at least one other DNA segment encoding an additional polypeptide, wherein the first and other DNA segments are connected in-frame; and wherein the first and other DNA segments encode the fusion protein.<br>
Within another aspect, the present invention provides an expression vector comprising the following operably linked elements: a transcription promoter; a DNA construct encoding a fusion protein as disclosed above; and a transcription terminator, wherein the promoter is operably linked to the DNA construct, and the DNA construct is operably linked to the transcription terminator.<br>
Within another aspect, the present invention provides a cultured cell comprising an expression vector as disclosed above, wherein the cell expresses a polypeptide encoded by the DNA construct.<br>
Within another aspect, the present invention provides a method of producing a fusion protein comprising: culturing a cell as disclosed above; and isolating the polypeptide produced by the cell.<br>
Within another aspect, the present invention provides an isolated polypeptide comprising a sequence of amino acid residues that is at least 90% identical to an amino acid sequence selected from the group consisting of: (a) the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 20 (Cys), to amino acid number 237 (His); (b) the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 20 (Cys), to amino acid number 255 (Leu); (c) the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 256 (Lys), to amino acid number 538 (Ser); (d) the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 20 (Cys), to amino acid number 538 (Ser); and (e) the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 1 (Met) to amino acid number 538 (Ser),<br><br>
wherein the amino acid percent identity is determined using a FASTA program with ktup=l, gap opening penalty=10, gap extension penalty=l, and substitution matrix=BLOSlJM62, with other parameters set as default. Within one embodiment, the isolated polypeptide disclosed above comprises a sequence of amino acid residues selected from the group consisting of: (a) the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 20 (Cys), to amino acid number 237 (His); (b). the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 20 (Cys), to amino acid number 255 (Leu); (c) the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 256 (Lys), to amino acid number 538 (Ser); (d) the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 20 (Cys), to amino acid number 538 (Ser); and (e) the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 1 (Met) to amino acid number 538 (Ser). Within another embodiment, the isolated polypeptide disclosed above consists of a sequence of amino acid residues selected from the group consisting of: (a) the amino acid sequence as shown m SEQ ID N0:2 from amino acid number 20 (Cys), to amino acid number 237 (His); (b) the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 20 (Cys), to amino acid number 255 (Leu); (c) the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 256 (Lys), to amino acid number 538 (Ser); (d) the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 20 (Cys), to amino acid number 538 (Ser); and (e) the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 1 (Met) to amino acid number 538 (Ser). Within another embodiment, the isolated polypeptide disclosed above fiirther contains a WSXWS motif. Within another embodiment, the isolated polypeptide disclosed above fiirther comprises a transmembrane domain. Within another embodiment, the isolated polypeptide disclosed above fiirther comprises a transmembrane domain, wherein the transmembrane domain consists of residues 238(Leu) to 255 (Leu) of SEQ ID N0:2. Within another embodiment, the isolated polypeptide disclosed above fiirther comprises an intracellular domain. Within another embodiment, the isolated polypeptide disclosed above fiirther comprises an intracellular domain, wherein the intracellular domain consists of residues 256 (Lys) to 538 (Ser) of SEQ ID N0:2.    Within another<br><br>
embodiment, the isolated polypeptide disclosed above further comprises an intracellular domain, wherein themtracellular domain further comprises Box I and Box II sites.<br>
WithinVanother aspect, the present invention provides a method of producing a zalpha11.polypeptide comprising: culturing a cell as disclosed above; and isolating the zalphal 1 polypeptide produced by the cell.<br>
Within another aspect, the present invention provides an isolated polypeptide comprising an amino acid sequence selected from the group consisting of: (a) the amino acid sequence as shown in SEQ ID NO:2 from amino acid number 20 (Cys), to amino acid number 237 (His); and wherein the polypeptide is substantially free of transmembrane and intracellular domains ordinarily associated with hematopoietic receptors. Within another embodiment, the isolated polypeptide disclosed above comprises an affinity tag.<br>
Within another aspect, the present invention provides a method of producing a zalphal 1 polypeptide comprising: culturing a cell as disclosed above; and isolating the zalphal 1 polypeptide produced by the cell.<br>
Within another aspect, the present invention provides a method of producinjg_an^aiitibody to zalphal 1 polypeptide comprising: inoculating an animal with a polypeptide selected from the group consisting of: (a) a polypeptide consisting of 9 to 519 amino acids, wherein the polypeptide consists of a contiguous sequence of amino acids in SEQ ID N0:2 from amino acid number 20 (Cys), to amino acid number 538 (Ser); (b) a polypeptide consisting of the amino acid sequence of SEQ ID N0:2 from amino acid number 20 (Cys), to amino acid number 237 (His); (c) a polypeptide .consisting of the amino acid sequence of SEQ ID N0:2 from amino acid number 101 (Leu) to amino acid number 122 (Gly); (d) a polypeptide consisting of the amino acid sequence of SEQ ID N0:2 from amino acid number 141 (Asn) to amino acid number 174 (Ala); (e) a polypeptide consisting of the amino acid sequence of SEQ ID N0:2 from amino acid number 193 (Cys) to amino acid number 261 (Val); (f) a polypeptide consisting of the amino acid sequence of SEQ ID N0:2 from amino acid number 51 (Trp) to amino acid number 61 (Glu); (g) a polypeptide consisting of the amino acid sequence of SEQ ID N0:2 from amino acid 136 (Ile) to amino acid number 143 (Glu); (h) a polypeptide consisting of the amino acid sequence of SEQ ID N0:2 from amino<br><br>
acid 187 (Pro) to amino-acid number 195 (Ser); (i) a polypeptide consisting of the amino acid sequence' of SEQ ID N0:2 from amino acid number 223 (Phe) to amino acid number 232 (Glu); and (j) a polypeptide consisting of the amino acid sequence of SEQ ID N0:2 from .amino acid number 360 (Glu) to amino acid number 368 (Asp); and wherein the polypeptide elicits an immune response in the animal to produce the antibody; and isolating the antibody from the animal.<br>
Within another aspect, the present invention provides an antibody produced by the method disclosed above, which specifically binds to a zalphall polypeptide. Within one embodiment, the antibody disclosed above is a monoclonal antibody.<br>
Within another aspect, the present invention provides an antibody which specifically binds to a polypeptide as disclosed above.<br>
Within another aspect, the present invention provides a method of detecting, in a test sample, the presence of a modulator of zalphal 1 protein activity, comprising: culturing a cell into which has been introduced an expression vector as disclosed above, wherein the cell expresses the zalphal 1 protein encoded by the DNA segment in the presence and absence of a test sample; and comparing levels of activity of zalphal 1 in the presence and absence of a test sample, by a biological or biochemical assay; and determining from the comparison, the presence of modulator of zalphal 1 activity in the test sample.<br>
Within another aspect, the present invention provides a method for detecting a zalphal 1 receptor ligand within a test sample, comprising: contacting a test sample with a polypeptide comprising an amino acid sequence as shown in SEQ ED N0:2 from amino acid number 20 (Cys), to amino acid number 237 (His); and detecting the bmding of the polypeptide to a ligand in the sample. Within one embodiment, the method disclosed above further comprises a polypeptide comprising transmembrane and intracellular domains. Within another embodiment, the method disclosed above futher comprises a polypeptide wherein the polypeptide is membrane bound within a cultured cell, and the detecting step comprises measuring a biological response in the cultured cell. Within another embodiment, the method disclosed above further comprises a polypeptide wherein the polypeptide is membrane bound within a<br><br>
cultured cell, and the detecting step comprises measuring a biological response in the cultured cell, wherein the biological response.is cell proliferation or activation of transcription of a reporter gene. Within another embodiment, the method disclosed above further comprises a polypeptide wherein the polypeptide is immobilized on a solid support.<br>
These and other aspects of the invention will become evident upon reference to the following detailed description of the invention.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figure 1 is a Hopp/Woods hydrophilicity plot of human zalphall.<br>
Figure 2 is an alignment of human zalphal 1 (zalpha) (SEQ ED NO: 2) and mouse zalphal 1 (muzalp) (SEQ ID NO: 85).<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
Prior to setting forth the invention in detail, it may be helpful to the understanding thereof to define the following terms:<br>
The term "affinity tag" is used herein to denote a polypeptide segment that can be attached to a second polypeptide to provide for purification or detection of the second polypeptide or provide sites for attachment of the second polypeptide to a substrate. In principal, any peptide or protein for which an antibody or other specific binding agent is available can be used as an affinity tag. Affinity tags include a poly-histidine tract, protein A (Nilsson et al., EMBO J. 4:1075,1985; Nilsson et al., Methods Enzvmol. 198:3. 1991), glutathione S transferase (Smith and Johnson, Gene 67:31, 1988), Glu-Glu affmity tag (Grussenmeyer et al., Proc. Natl. Acad. Sci. USA 82:7952-4, 1985), substance P, Flag™ peptide (Hopp et al., Biotechnology 6:1204-10, 1988), streptavidin binding peptide, or other antigenic epitope or binding domain. See, in general. Ford et al.. Protein Expression and Purification 2: 95-107, 1991. DNAs encoding affinity tags are available from commercial suppliers (e.g., Pharmacia Biotech, Piscataway, NJ).<br>
The term "allelic variant" is used herein to denote any of two or more alternative forms of a gene occupying the same chromosomal locus. Allelic variation<br><br>
arises naturally through mutation, and may result in phenotypic polymorphism within populations. Gene mutations can be silent (no change in the encoded polypeptide) or may encode polypeptides having altered amino acid sequence. The tenn allelic variant is also used herein to Henote a protein encoded by an allelic variant of a gene.<br>
The terms "amino-terminal" and "carboxyl-terminal" are used herein to denote positions within polypeptides. Where the context allows, these terms are used with reference to a particular sequence or portion of a polypeptide to denote proximity or relative position. For example, a certain sequence positioned carboxyl-terminal to a reference sequence within a polypeptide is located proximal to the carboxyl terminus of the reference sequence, but is not necessarily at the carboxyl terminus of the complete polypeptide.<br>
The term "complement/anti-complement pair" denotes non-identical moieties that form a non-covalently associated, stable pair under appropriate conditions. For instance, biotin and avidin (or streptavidin) are prototypical members of a complement/anti-complement pair. Other exemplary complement/anti-compiement pairs include receptor/ligand pairs, antibody/antigen (or hapten or epitope) pairs, sense/antisense polynucleotide pairs, and the like. Where subsequent dissociation of the complement/anti-complement pair is desirable, the complement/anti-complement<br>
pair preferably has a binding affinity of 
The term "complements of a polynucleotide molecule" is a polynucleotide molecule having a complementary base sequence and reverse orientation as compared to a reference sequence. For example, the sequence 5' ATGCACGGG 3' is complementary to 5' CCCGTGCAT 3'.<br>
The term "contig" denotes a polynucleotide that has a contiguous stretch of identical or complementary sequence to another polynucleotide. Contiguous sequences are said to "overlap" a given stretch of polynucleotide sequence either in their entirety or along a partial stretch of the polynucleotide. For example, representative contigs to the polynucleotide sequence 5'-ATGGCTTAGCTT-3' are 5'-TAGCTTgagtct-3' and 3'-gtcgacTACCGA-5'.<br>
The term "degenerate nucleotide sequence" denotes a sequence of nucleotides that includes one or more degenerate codons (as compared to a reference<br><br>
polynucleotide molecule that encodes a polypeptide). Degenerate codons contain different triplets of nucleotides, but encode the same amino acid residue (i.e., GAU and GAC triplets each encode Asp).<br>
The term "expression vector" is used to denote a DNA molecule, linear or circular, that comprises a segment encoding a polypeptide of interest operably linked to additional segments that provide for its transcription. Such additional segments include promoter and terminator sequences, and may also include one or more origins of replication, one or more selectable markers, an enhancer, a polyadenylation signal, etc. Expression vectors are generally derived from plasmid or viral DNA, or may contain elements of both.<br>
The term  isolated", ,when applied to a polynucleotide denotes that the polynucleotide has been removed from its natural genetic milieu and is thus free of other extraneous or unwanted coding sequences, and is in a form suitable for use within genetically engineered protein production systems. Such isolated molecules are those that are separated from their natural environment and include cDNA and genomic clones. Isolated DNA molecules of the present invention are free of other genes with which they are ordinarily associated, but may include naturally occurring 5' and 3' untranslated regions such as promoters and terminators. The identification of associated regions will be evident to one of ordinary skill in the art (see for example, Dynan and Tijan, Nature 316:774-78,1985).    <br>
An isolated" polypeptide) or protein is a polypeptide or protein that is found in a condition other than its native environment, such as apart from blood and animal tissue. In a preferred form, the isolated polypeptide is substantially free of other polypeptides, particularly other polypeptides of animal origin. It is preferred to provide the polypeptides in a highly purified form, i.e. greater than 95% pure, more preferably greater than 99% pure. When used in this context, the term "isolated" does not exclude the presence of the same polypeptide in alternative physical forms, such as dimers or altematively glycosylated or derivatized forms.<br>
The term "operably linked", when referring to DNA segments, indicates that the segments are arranged so that they fuction in concert for their intended<br><br>
purposes, e.g., transcription initiates in the promoter and proceeds through the coding segment to the terminator.<br>
The term "ortholog" denotes a polypeptide or protein obtained from one species that is the functional counterpart of a polypeptide or protein from a different species. Sequence differences among orthologs are the result of speciation.<br>
"Paralogs" are distinct but structurally related proteins made by an organism. Paralogs are believed to arise through gene duplication. For example, a-globin, (3-globin, and myoglobin are paralogs of each other.<br>
A "polynucleotide" is a single- or double-stranded polymer of deoxyribonucleotide or ribonucleotide bases read from the 5' to the 3' end. Polynucleotides include RNA and DNA, and may be isolated from natural sources, synthesized in vitro, or prepared from a combination of natural and synthetic molecules. Sizes of polynucleotides are expressed as base pairs (abbreviated "bp"), nucleotides ("nt"), or kilobases ("kb"). Where the context allows, the latter two terms may describe polynucleotides that are single-stranded or double-stranded. When the term is applied to double-stranded molecules it is used to denote overall length and will be imderstood to be equivalent to the term "base pairs". It will be recognized by those skilled in the art that the two strands of a double-stranded polynucleotide may differ slightly in length and that the ends thereof may be staggered as a result of enzymatic cleavage; thus all nucleotides within a double-stranded polynucleotide molecule may not be paired.<br>
A "polypeptide" is a polymer of amino acid residues joined by peptide bonds, whether produced naturally or synthetically. Polypeptides of less than about 10 amino acid residues are commonly referred to as "peptides".<br>
The term "promoter" is used herein for its art-recognized meaning to denote a portion of a gene containing DNA sequences that provide for the binding of RNA polymerase and initiation of transcription. Promoter sequences are commonly, but not always, found in the 5' non-coding regions of genes.<br>
A "protein" is a macromolecule comprising one or more polypeptide chains. A protein may also comprise non-peptidic components, such as carbohydrate groups. Carbohydrates and other non-peptidic substituents may be added to a protein by the cell in which the protein is produced, and will vary with the type of cell.<br><br>
Proteins are defined herein in terms of their amino acid backbone structures;<br>
substituents such as carbohydrate groups are generally not specified, but may be present<br>
nonetheless.	<br>
The term "receptor" is used herein to denote a cell-associated protein, or a polypeptide subunit of such a protein, that binds to a bioactive molecule (the "ligand") and mediates the effect of the ligand on the cell. Binding of ligand to receptor results in a conformational change in the receptor (and, in some cases, receptor multimerization, i.e., association of identical or different receptor subunits) that causes interactions between the effector domain(s) and other molecule(s) in the cell. These interactions in turn lead to alterations in the metabolism of the cell. Metabolic events that are linked to receptor-ligand interactions include gene transcription, phosphorylation, dephosphorylation, cell proliferation, increases in cyclic AMP production, mobilization of cellular calcium, mobilization of membrane lipids, cell adhesion, hydrolysis of inositol lipids and hydrolysis of phospholipids. Cell-surface cytokine receptors are characterized by a multi-domain structure as discussed in more detail below. These receptors are anchored in the cell membrane by a transmembrane domain characterized by a sequence of hydrophobic amino acid residues (typically about 21-25 residues), which is commonly flanked by positively charged residues (Lys or Arg). In general, receptors can be membrane bound, cytosolic or nuclear; monomeric (e.g., thyroid stimulating hormone receptor, beta-adrenergic receptor) or multimeric (e.g., PDGF receptor, growth hormone receptor, IL-3 receptor, GM-CSF receptor, G-CSF receptor, erythropoietin receptor and IL-6 receptor). The term "receptor polypeptide" is used to denote complete receptor polypeptide chains and portions thereof, including isolated functional domains (e.g., ligand-binding domains).<br>
A "secretory signal sequence" is a DNA sequence that encodes a* polypeptide (a "secretory peptide") that, as a component of a larger polypeptide, directs the larger polypeptide through a secretory pathway of a cell in which it is synthesized. The larger peptide is commonly cleaved to remove the secretory peptide during transit through the secretory pathway.<br>
A "soluble receptor" is a receptor polypeptide that is not bound to a cell membrane. Soluble receptors are most commonly ligand-binding receptor polypeptides<br><br>
that lack transmembrane -and cytoplasmic domains. Soluble receptors can comprise additional amino acid residues, such as affinity tags that provide for purification of the polypeptide or provide .sites for attachment of the polypeptide to a substrate, or immunoglobulin constant region sequences. Many cell-surface receptors have naturally occurring, soluble counterparts that are produced by proteolysis. Soluble receptor polypeptides are said to be substantially free of transmembrane and intracellular polypeptide segments when they lack sufficient portions of these segments to provide membrane anchoring or signal transduction, respectively.<br>
The term "splice variant" is used herein to denote alternative forms of RNA transcribed from a gene. Splice variation arises naturally through use of alternative splicing sites within a transcribed RNA molecule, or less commonly between separately transcribed RNA molecules, and may result in several mRNAs transcribed from the same gene. Splice variants may encode polypeptides having altered amino acid sequence. The term splice variant is also used herein to denote a protein encoded by a splice variant of an mRNA transcribed from a gene.<br>
Molecular weights and lengths of polymers determined by imprecise analytical methods (e.g., gel electrophoresis) will be understood to be approximate values. When such a value is expressed as "about" X or "approximately" X, the stated value of X will be understood to be accurate to ± 10%.<br>
All references cited herein are incorporated by reference in their entirety.<br>
The present invention is based in part upon the discovery of a novel DNA sequence that encodes a protein having the structure of a class I cytokine receptor. The deduced amino acid sequence indicated that the encoded receptor belongs to the receptor subfamily that includes the IL-2 receptor p-subunit and the P-common receptor (i.e., IL3, lL-5, and GM-CSF receptor p-subxmits). Analysis of the tissue distribution of the mRNA corresponding to this novel DNA showed expression in lymph node, peripheral blood leukocytes (PBLs), spleen, and thymus. Moreover, the mRNA was abundant in the Raji cell line (ATCC No. CCL-86) derived from a Burkitt's lymphoma. The polypeptide has been designated zalpha11<br>
The novel zalphall polypeptides of the present invention were initially identified by querying an EST database.   An EST was found and its corresponding<br><br>
cDNA was sequenced. - The novel polypeptide encoded by the cDNA showed homology with class I cytokine receptors. The zalphal 1 polynucleotide sequence encodes the entire coding sequence of the predicted protein. Zalphal 1 is a novel cytokine receptor that may be involved in an apoptotic cellular pathway, cell-cell signaling molecule, growth factor receptor, or extracellular matrix associated protein with growth factor hormone activity, or the like.<br>
The sequence of the zalphal 1 polypeptide was deduced from a single clone that contained its corresponding polynucleotide sequence. The clone was obtained from a spinal cord library. Other libraries that might also be searched for such sequences include PBL, thymus, spleen, lymph node, human erythroleukemia cell lines (e.g., TF-1), Raji cells, acute monocytic leukemia cell lines, other lymphoid and hematopoietic cell lines, and the like.<br>
The nucleotide sequence of a representative zalphal 1-encoding DNA is described in SEQ ID N0:1 (from nucleotide 69 to 1682), and its deduced 538 amino acid sequence is described in SEQ ID N0:2. In its entirety, the zalphal 1 polypeptide (SEQ ID NO:2) represents a full-length polypeptide segment (residue 1 (Met) to residue 538 (Ser) of SEQ ID N0:2). The domains and structural features of the zalphal 1 polypeptide are further described below.<br>
Analysis of the zalphal 1 polypeptide encoded by the DNA sequence of SEQ ID N0:1 revealed an open reading frame encoding 538 amino acids (SEQ ID N0:2) comprising a predicted secretory signal peptide of 19 amino acid residues (residue 1 (Met) to residue 19 (Gly) of SEQ ID N0:2), and a mature polypeptide of 519 amino acids (residue 20 (Cys) to residue 538 (Ser) of SEQ ID N0:2). In addition to the WSXWS motif (SEQ ID N0:3) corresponding to residues 214 to 218 of SEQ ID N0:2, the receptor comprises a cytokine-binding domain of approximately 200 amino acid residues (residues 20 (Cys) to 237 (His) of SEQ ID N0:2); a domain linker (residues 120 (Pro) to 123 (Pro) of SEQ ID N0:2); a penultimate strand region (residues 192 (Lys) to 202 (Ala) of SEQ ID N0:2); a transmembrane domain (residues 238 (Leu) to 255 (Leu) of SEQ ID N0:2); complete intracellular signaling domain (residues 256 (Lys) to 538 (Ser) of SEQ ID N0:2) which contains a "Box I" signaling site (residues 267 (Ile) to 273 (Pro) of SEQ ID N0:2), and a "Box 11" signaling site (residues 301<br><br>
(Leu) to 304 (Gly) of SEQ ID N0:2). Those skilled in the art will recognize that these domain bovmdaries are approximate, and are based on alignments with known proteins and predictions of protein folding. In addition to these domains, conserved receptor features in the encoded receptor include (as shown in SEQ ID N0:2) a conserved Trp residue at position 138, and a conserved Arg residue at position 201. The corresponding polynucleotides encoding the zalphal 1 polypeptide regions, domains, motifs, residues and sequences described above are as shown in SEQ ID N0:1.<br>
The presence of transmembrane regions, and conserved and low variance motifs generally correlates with or defines important structural regions in proteins. Regions of low variance (e.g., hydrophobic clusters) are generally present in regions of structural importance (Sheppard, P. et al., supra.). Such regions of low variance often contain rare or infrequent amino acids, such as Tryptophan. The regions flanking and between such conserved and low variance motifs may be more variable, but are often functionally significant because they may relate to or define important structures and activities such as binding domains, biological and enzymatic activity, signal transduction, cell-cell interaction, tissue localization domains and the like.<br>
The regions of conserved amino acid residues in zalphal 1, described above, can be used as tools to identify new family members. For instance, reverse transcription-polymerase chain reaction (RT-PCR) can be used to amplify sequences encoding the conserved regions from RNA obtained from a variety of tissue sources or cell lines. In particular, highly degenerate primers designed from the zalphal 1 sequences are useful for this purpose. Designing and using such degenerate primers may be readily performed by one of skill in the art.<br>
The present invention provides polynucleotide molecules, including DNA and RNA molecules, that encode the zalphal 1 polypeptides disclosed herein. Those skilled in the art will recognize that, in view of the degeneracy of the genetic code, considerable sequence variation is possible among these polynucleotide molecules. SEQ ID N0:4 is a degenerate DNA sequence that enconipasses all DNAs that encode the zalphal 1 polypeptide of SEQ ID N0:2. Those skilled in the art will recognize that the degenerate sequence of SEQ ID N0:4 also provides all RNA sequences  encoding  SEQ ID N0:2 by substituting U for T.     Thus, zalphal 1<br><br>
polypeptide-encoding polynucleotides comprising nucleotide 1 to nucleotide 1614 of SEQ ID N0:4 and their RNA equivalents are contemplated by the present invention. Table 1 sets forth the'one-letter codes used within SEQ ID N0:4 to denote degenerate nucleotide positions.. "Resolutions" are the nucleotides denoted by a code letter. "Complement" indicates the code for the complementary nucleotide(s). For example, the code Y denotes either C or T, and its complement R denotes A or G, A being complementary to T, and G being complementary to C.<br><br><br><br><br><br>
One of ordinary skill in the art will appreciate that some ambiguity is introduced in determining a degenerate codon,. representative of all possible codons encoding each amino acid.; For example, the degenerate codon for serine (WSN) can, in some circumstances, 'encode arginine (AGR), and the degenerate codon for arginine ' (MGN) can, in some circumstances, encode serine (AGY). A similar relationship exists between codons encoding phenylalanine and leucine. Thus, some polynucleotides encompassed by the degenerate sequence may encode variant amino acid sequences, but one of ordinary skill in the art can easily identify such variant sequences by reference to the amino acid sequence of SEQ ID N0:2. Variant sequences can be readily tested for functionality as described herein.<br>
One of ordinary skill in the art will also appreciate that different species can exhibit "preferential codon usage." In general, see, Grantham, et al., Nuc. Acids Res. 8:1893-912, 1980; Haas, et al. Curr. Biol. 6:315-24, 1996; Wain-Hobson, et al.. Gene 13:355-64. 1981; Grosjean and Fiers, Gene 18:199-209, 1982; Hohn, Nuc. Acids Res. 14:3075-87, 1986; Ikemura, J. Mol. Biol. 158:573-97, 1982. As used herein, the term "preferential codon usage" or "preferential codons" is a term of art referring to protein translation codons that are most frequently used in cells of a certain species, thus favoring one or a few representatives of the possible codons encoding each amino acid (See Table 2). For example, the amino acid Threonine (Thr) may be encoded by ACA, ACC, ACG, or ACT, but m mammalian cells ACC is the most commonly used codon; in other species, for example, insect cells, yeast, viruses or bacteria, different Thr codons may be preferential. Preferential codons for a particular species can be introduced into the polynucleotides of the present invention by a variety of methods known in the art. Introduction of preferential codon sequences into recombinant DNA can, for example, enhance production of the protein by making protein translation more efficient within a particular cell type or species. Therefore, the degenerate codon sequence disclosed in SEQ ID N0:4 serves as a template for optimizing expression of polynucleotides in various cell types and species commonly used in the art and disclosed herein. Sequences containing preferential codons can be tested and optimized for expression in various species, and tested for functionality as disclosed herein.<br><br>
Within preferred embodiments of the invention the isolated polynucleotides will hybridize to similar sized regions of SEQ ID N0:1, or a sequence complementary thereto, under stringent conditions. In general, stringent conditions are selected to be about '5°C lower than the thermal melting point (Tm) for the specific<br>
sequence at a defined ionic strength and pH. The Tm is the temperature (under defined<br>
ionic strength and pH) at which 50% of the target sequence hybridizes to a perfectly matched probe. Numerous equations for calculating Tn, are known in the art, and are specific for DNA, RNA and DNA-RNA hybrids and polynucleotide probe sequences of varying length (see, for example, Sambrook et al, Molecular Cloning: A Laboratory Manual, Second Edition (Cold Spring Harbor Press 1989); Ausubel et al, (eds.), Current Protocols in Molecular Biology john Wiley and Sons, Inc. 1987); Berger and Kimmel (eds.), Guide to Molecular Cloning Techniques. (Academic Press, Inc. 1987); and Wetmur, Crit. Rev. Biochem. Mol. Biol. 26:227 (1990)). Sequence analysis software such as OLIGO 6.0 (LSR; Long Lake, MN) and Primer Premier 4.0 (Premier Biosoft International; Palo Alto, CA), as well as sites on the Internet, are available tools for analyzing a given sequence and calculating T„ based on user defined criteria. Such programs can also analyze a given sequence under defined conditions and identify suitable probe sequences. Typically, hybridization of longer polynucleotide sequences (e.g., &gt;50 base pairs) is performed at temperatures of about 20-25°C below the calculated T„. For smaller probes (e.g., 
 <br>
2X SSC, 0.1% SDS, at 55°C to 65°C. Different degrees of stringency can be used during hybridization and washing to achieve maximum specific binding to the target sequence. Typically,'"Ihe washes following hybridization are performed at increasing degrees of stringency to remove non-hybridized polynucleotide probes from hybridized complexes. Stringent hybridization and wash conditions depend on the length of the probe, reflected in the Tm, hybridization and wash solutions used, and are routinely determined empirically by one of skill in the art.<br>
As previously noted, the isolated polynucleotides of the present invention include DNA and RNA. Methods for preparing DNA and RNA are well known in the art. In general, RNA is isolated from a tissue or cell that produces large amounts of zalphal 1 RNA. Such tissues and cells are identified by Northern blotting (Thomas, Proc. Natl. Acad. Sci. USA 77:5201, 1980), and include PBLs, spleen, thymus, and lymph tissues, Raji cells, human erythroleukemia cell lines (e.g., TF-1), acute monocytic leukemia cell lines, other lymphoid and hematopoietic cell lines, and the like. Total RNA can be prepared using guanidinium isothiocyanate extraction followed  by  isolation by  centrifiigation  in  a  CsCl  gradient  (Chirgwin  et  al.,<br>
Biochemistry 18:52-94, 1979). Poly (A)+ RNA is prepared from total RNA using the method  of Aviv  and Leder  (Troc.  Natl.  Acad.   Sci.  USA  69:1408-12,   1972).<br>
Complementary DNA (cDNA) is prepared from poly(A)'" RNA using known methods. In the alternative, genomic DNA can be isolated. Polynucleotides encoding zalphal 1 polypeptides are then identified and isolated by, for example, hybridization or polymerase chain reaction (PCR) (Mullis, U.S. Patent No. 4,683,202).<br>
A full-length clone encoding zalphal 1 can be obtained by conventional cloning procedures.  Complementary DNA (cDNA) clones are preferred, although for<br>
»<br>
some applications (e.g., expression in transgenic animals) it may be preferable to use a genomic clone, or to modify a cDNA clone to include at least one genomic intron. Methods for preparing cDNA and genomic clones are well known and within the level of ordinary skill in the art, and include the use of the sequence disclosed herein, or parts thereof, for probing or priming a library. Expression libraries can be probed with antibodies to zalphall, receptor fragments, or other specific binding partners.<br><br>
The polynucleotides of the present mvention can also be synthesized using DNA synthesis machines. Currently the method of choice is the phosphoramidite method. If chemically synthesized double stranded DNA is required for an application such as the synthesis of a gene or a gene fragment, then each complementary strand is made separately. The production of short polynucleotides (60 to 80 bp) is technically straightforward and can be accomplished by synthesizing the complementary strands and then annealing them. However, for producing longer polynucleotides (&gt;300 bp), special strategies are usually employed, because the coupling efficiency of each cycle during chemical DNA synthesis is seldom 100%. To overcome this problem, synthetic genes (double-stranded) are assembled in modular form from single-stranded fragments that are from 20 to 100 nucleotides in length.<br>
One method for building a synthetic gene requires the initial production of a set of overlapping, complementary oligonucleotides, each of which is between 20 to 60 nucleotides long. Each internal section of the gene has complementary 3' and 5' terminal extensions designed to base pair precisely with an adjacent section. Thus, after the gene is assembled, process is completed by sealing the nicks along the backbones of the two strands with T4 DNA ligase. In addition to the protein coding sequence, synthetic genes can be designed with terminal sequences that facilitate insertion into a restriction endonuclease site of a cloning vector. Moreover, other sequences should can be added that contain signals for proper initiation and termination of transcription and translation.<br>
An alternative way to prepare a full-length gene is to synthesize a specified set of overlapping oligonucleotides (40 to 100 nucleotides). After the 3' and 5' short overlapping complementary regions (6 to 10 nucleotides) are annealed, large gaps still remain, but the short base-paired regions are both long enough and stable enough to hold the structure together. The gaps are filled and the DNA duplex is completed via enzymatic DNA synthesis by E. coli DNA polymerase I. After the enzymatic synthesis is completed, the nicks are sealed with T4 DNA ligase. Double-stranded constructs are sequentially linked to one another to form the entire gene sequence which is verified by DNA sequence analysis. See Glick and Pasternak, Molecular Biotechnology, Principles &amp; Applications of Recombinant DNA. (ASM Press, Washington, D.C.<br><br>
1994); Itakura et al., Annu. Rev. Biochem. 53: 323-56, 1984 and Climie et al., Proc. Natl. Acad. Sci. USA'87:633-7.1990.<br>
The present invention further provides counterpart polypeptides and polynucleotides from.other species (orthologs). These species include, but are not limited to mammalian, avian, amphibian, reptile, fish, insect and other vertebrate and invertebrate species. Of particular interest are zalphall polypeptides from other mammalian species, including mvirine, porcine, ovine, bovine, canine, feline, equine, and other primate polypeptides. Orthologs of human zalphal 1 can be cloned using information and compositions provided by the present invention in combination with conventional cloning techniques. For example, a cDNA can be cloned using mRNA obtained from a tissue or cell type that expresses zalphal 1 as disclosed herein. Suitable sources of mRNA can be identified by probing Northern blots with probes designed from the sequences disclosed herein. A library is then prepared from mRNA of a positive tissue or cell line. A zalphall-encoding cDNA can then be isolated by a variety of methods, such as by probing with a complete or partial human cDNA or with one or more sets of degenerate probes based on the disclosed sequences. A cDNA can also be cloned using PCR (MuUis, supra.*), xising primers designed from the representative human zalphall sequence disclosed herein. Within an additional method, the cDNA library can be used to transform or transfect host cells, and expression of the cDNA of mterest can be detected with an antibody to zalphal 1 polypeptide. Similar techniques can also be applied to the isolation of genomic clones.<br>
Cytokine receptor subunits are characterized by a multi-domain structure comprising an extracellular domain, a transmembrane domain that anchors the polypeptide in the cell membrane, and an mtracellular domain. The extracellular domain may be a ligand-binding domain, and the intracellular domain may be an effector domain involved in signal transduction, although ligand-binding and effector functions may reside on separate subunits of a multimeric receptor. The ligand-binding domain may itself be a multi-domain structure. Multimeric receptors include homodimers (e.g., PDGF receptor αα and ββ isoforms, erythropoietin receptor, MPL, and G-CSF receptor), heterodimers whose subunits each have ligand-binding and effector domains (e.g., PDGF receptor αβ  isoform), and multimers having component<br><br>
subunits with disparate functions (e.g., IL-2, IL-3, IL-4, IL-5, IL-6, IL-7, and GM-CSF receptors). Some receptor subunits are common to a plurality of receptors. For example, the AIC2B 'subunit, which cannot bind ligand on its own but includes an intracellular signal, transduction domain, is a component of IL-3 and GM-CSF receptors. Many cytokine receptors can be placed into one of four related families on the basis of the structure and function. Hematopoietic receptors, for example, are characterized by the presence of a domain containing conserved cysteine residues and the WSXWS motif (SEQ ID NO:3). Cytokine receptor structure has been reviewed by Urdal, Ann. Reports Med. Chem. 26:221-228, 1991 and Cosman, Cvtokine 5:95-106, 1993. Under selective pressure for organisms to acquire new biological functions, new receptor family members likely arise from duplication of existing receptor genes leading to the existence of multi-gene families. Family members thus contain vestiges of the ancestral gene, and these characteristic features can be exploited in the isolation and identification of additional family members. Thus, the cytokine receptor superfamily is subdivided into several families, for example, the immunoglobulin family (including CSF-1, MGF, IL-1, and PDGF receptors); the hematopoietin family (including IL-2 receptor p-subunit, GM-CSF receptor a-subunit, GM-CSF receptor β -subunit; and G-CSF, EPO, IL-3, IL-4, IL-5, IL-6, IL-7, and IL-9 receptors); TNF receptor family (including TNF (p80) TNF (p60) receptors, CD27, CD30, CD40, Fas, and NGF receptor).<br>
Analysis of the zalphal 1 sequence suggests that it is a member of the same receptor subfamily as the iL-2 receptor P-subunit, IL-3, IL-4, and IL-6 receptors. Certain receptors in this subfamily (e.g., G-CSF) associate to form homodimers that transduce a signal. Other members of the subfamily (e.g., IL-6, IL-11, and LIF receptors) combine with a second subunit (termed a p-subunit) to bind ligand and transduce a signal. Specific β -subunits associate with a plurality of specific cytokine receptor subxmits. For example, the P-subunit gpl30 (Hibi et al.. Cell 63:1149-1157, 1990) associates with receptor subunits specific for IL-6, IL-11, and LIF (Gearing et al., EMBO J. 10:2839-2848, 1991; Gearing et al., U.S. Patent No. 5,284,755). Oncostatin M binds to a heterodimer of LIF receptor and gpl30. CNTF binds to trimeric receptors comprising CNTF receptor, LIF receptor, and gpl30 subvmits.<br><br>
A polynucleotide sequence for the mouse ortholog of human zalphal 1 has been identified and is shown in SEQ ID NO:84 and the corresponding amino acid sequence shown in jEQ ID NO: 85. AnalYsis of the mouse zalphal 1 polypeptide encoded by the DNA\ sequence of SEQ ID N0(84)revealed an open reading frame encodmg 529 amino acids (SEQ ID NO:85) comprising a predicted secretory signal peptide of 19 amino acid residues (residue 1 (Met) to residue 19 (Ser) of SEQ ID NO:85), and a mature polypeptide of 510 amino acids (residue 20 (Cys) to residue 529 (Ser) of SEQ ID N0:2). In addition to the WSXWS motif (SEQ ID N0:3) corresponding to residues 214 to 218 of SEQ ID N0:85, the receptor comprises a cytokine-binding domain of approximately 200 amino acid residues (residues 20 (Cys) to 237 (His) of SEQ ID N0:85); a domam linker (residues 120 (Pro) to 123 (Pro) of SEQ ID NO:85); a penultimate strand region (residues 192 (Lys) to 202 (Ala) of SEQ ID NO:85); a transmembrane domain (residues 238 (Met) to 254 (Leu) of SEQ ID NO:85); complete intracellular signaling domain (residues 255 (Lys) to 529 (Ser) of SEQ ID NO;85) which contains a "Box I" signaling site (residues 266 (Ile) to 273 (Pro) of SEQ ID NO:85), and a "Box II" signaling site (residues 301 (IIe) to 304 (Val) of SEQ ID N0:2). A comparison of the human and mouse amino acid sequences reveals that both the himian and orthologous polypeptides contain corresponding structural features described above (See, Figure 2). The mature sequence for the mouse zalphal 1 begins at Cysjo (as shown in SEQ ID NO:85), which corresponds to Cysjo (as shown in SEQ ID N0:2) in the human sequence. There is about 63% identity between the mouse and human sequences over the entire amino acid sequence corresponding to SEQ ID N0:2 and SEQ ID NO:85, There is about 69% identity a between the mouse and human zalphall sequences over the extracellular cytokine binding domain corresponding to residues 20 (Cys) to 237 (His) of SEQ ID Nb:2 and residues 20 (Cys) to 237 (His) of SEQ ID NO:85. There is about 60% identity a between the mouse and human zalphall sequences over the intracellular signalling domain corresponding to residues 256 (Lys) to 538 (Ser) of SEQ ID N0:2, and residues 255 (Lys) to 529 (Ser) of SEQ ID NO:85. The above percent identities were determined using a FASTA program with ktup=l, gap opening penalty=12, gap extension penalty=2, and substitution matrix=BLOSUM62,  with  other parameters  set  as  default.     The  corresponding<br><br>
polynucleotides encoding the mouse zalphal 1 polypeptide regions, domains, motifs, residues and sequences described above are as shown in SEQ ID NO:84.<br>
Those skilled in the art will recognize that the sequence disclosed in SEQ ID NO: 1 represents a single allele of human zalphal 1 and that allelic variation and alternative splicing are expected to occur. Allelic variants of this sequence can be cloned by probing cDNA or genomic libraries from different individuals according to standard procedures. Allelic variants of the DNA sequence shown in SEQ ID N0:1, including those containing silent mutations and those in which mutations resuU in amino acid sequence changes, are within the scope of the present invention, as are proteins which are allelic variants of SEQ ID N0:2. cDNAs generated from alternatively spliced mRNAs, which retain the properties of the zalphal 1 polypeptide are included within the scope of the present invention, as are polypeptides encoded by such cDNAs and mRNAs. Allelic variants and splice variants of these sequences can be cloned by probing cDNA or genomic libraries from different individuals or tissues according to standard procedures known in the art.<br>
The present invention also provides isolated zalphal 1 polypeptides that are substantially similar to the polypeptides of SEQ ID N0:2 and their orthologs. The term "substantially similar" is used herein to denote polypeptides having at least 70%, more preferably at least 80%, sequence identity to the sequences shown in SEQ ID N0:2 or their orthologs. Such polypeptides will more preferably be at least 90% identical, and most preferably 95% or more identical to SEQ ID N0:2 or its orthologs.) Percent sequence identity is determined by conventional methods. See, for example, Altschul et al., Bull. Math. Bio. 48: 603-616, 1986 and Henikoff and Henikoff, Proc. Natl. Acad. Sci. USA 89:10915-10919. 1992. Briefly, two amino acid sequences are aligned to optimize the alignment scores using a gap opening penalty of 10, a gap extension penalty of 1, and the "blosum 62" scoring matrix of Henikoff and Henikoff (ibid.) as shown in Table 3 (amino acids are indicated by the standard one-letter codes). The percent identity is then calculated as:<br><br>
Total number of identical matches<br>
		xlOO<br>
[length, of the longer sequence plus the number of gaps introduced into the longer sequence in order to align the two sequences]<br><br><br><br>
Sequence identity of polynucleotide molecules is determined by similar methods using a ratio "as disclosed above.<br>
Those'skilled in the art appreciate that there are many established algorithms available 'to align two amino acid sequences. The "FASTA" similarity search algorithm of Pearson and Lipman is a suitable protein alignment method for examining the level of identity shared by an amino acid sequence disclosed herein and the amino acid sequence of a putative variant zsig57. The FASTA algorithm is described by Pearson and Lipman, Proc. Nat'l Acad. Sci. USA 85:2444 (1988), and by Pearson. Meth. EnzvmoL 183:63 (1990).<br>
Briefly, FASTA first characterizes sequence similarity by identifymg regions shared by the query sequence (e.g., SEQ ID N0:2) and 'a test sequence that have either the highest density of identities (if the ktup variable is 1) or pairs of identities (if ktup=2), without considering conservative amino acid substitutions, insertions, or deletions. The ten regions with the highest density of identities are then rescored by comparing the similarity of all paired amino acids using an amino acid substitution matrix, and the ends of the regions are "trimmed" to include only those residues that contribute to the highest score. If there are several regions with scores greater than the "cutoff' value (calculated by a predetermined formula based upon the length of the sequence and the ktup value), then the trimmed initial regions are examined to determine whether the regions can be joined to form an approximate alignment with gaps. Finally, the highest scoring regions of the two amino acid sequences are aligned using a modification of the Needleman-Wunsch-Sellers algorithm (Needleman and Wunsch, J. Mol. Biol. 48:444 (1970); Sellers. SIAM J. Appl. Math. 26:787 (1974)), which allows for amino acid insertions and deletions. Preferred parameters for FASTA analysis are: ktup=l, gap opening penalty=10, gap extension penalty=l, and substitution matrix=BLOSUM62, with other parameters set as default. These parameters can be introduced into a FASTA program by modifying the scoring matrix file ("SMATRIX"), as explained in Appendix 2 of Pearson, Meth. Enzvmol. 183:63 (1990).<br>
FASTA can also be used to determine the sequence identity of nucleic acid molecules using a ratio as disclosed above. For nucleotide sequence comparisons,<br><br>
the ktup value can range between one to six, preferably from three to six, most preferably three, with other parameters set as defauh.<br>
The BLOSUM62 table (Table 3) is an amino acid substitution matrix derived from about 2,000 local multiple alignments of protein sequence segments, representing highly conserved regions of more than 500 groups of related proteins (Henikoff and Henikoff, Proc. Nat'I Acad. Sci. USA SS&gt;:10915 (1992)). Accordingly, the BLOSUM62 substitution frequencies can be used to define conservative amino acid substitutions that may be introduced into the amino acid sequences of the present invention. Although it is possible to design amino acid substitutions based solely upon chemical properties (as discussed below), the language "conservative amino acid substitution" preferably refers to a substitution represented by a BLOSUM62 value of greater than -1, For example, an amino acid substitution is conservative if the substitution is characterized by a BLOSUM62 value of 0, 1, 2, or 3. According to this system, preferred conservative amino acid substitutions are characterized by a BLOSUM62 value of at least 1 (e.g., 1, 2 or 3), while more preferred conservative amino acid substitutions are characterized by a BLOSUM62 value of at least 2 (e.g., 2 or 3).<br>
Variant zalphall polypeptides or substantially homologous zalphall polypeptides are characterized as having one or more amino acid substitutions, deletions or additions. These changes are preferably of a minor nature, that is conservative amino acid substitutions (see Table 4) and other substitutions that do not significantly affect the folding or activity of the polypeptide; small deletions, typically of one to about 30 amino acids; and small amino- or carboxyl-terminal extensions, such as an amino-terminal methionine residue, a small linker peptide of up to about 20-25 residues, or an affinity tag. The present invention thus includes polypeptides of from about 489 to about 568 amino acid residues that comprise a sequence that is at least 80%, preferably at least 90%, and more preferably 95% or more identical to the corresponding region of SEQ ID N0:2. Polypeptides comprising affinity tags can further comprise a proteolytic cleavage site between the zalphal 1 polypeptide and the affinity tag. Suitable sites include thrombin cleavage sites and factor Xa cleavage sites.<br><br><br>
The present invention further provides a variety of other polypeptide fusions and related multimeric proteins comprising one or more polypeptide fusions. For example, a zalphall polypeptide can be prepared as a fusion to a dimerizing protein as disclosed in U.S. Patents Nos. 5,155,027 and 5,567,584. Preferred dimerizing proteins in this regard include immunoglobulin constant region domains, hnmunoglobulin-zalphal 1 polypeptide fusions can be expressed m genetically engineered cells to produce a variety of multimeric zalphall analogs. Auxiliary domains can be fused to zalphal 1 polypeptides to target them to specific cells, tissues,<br><br>
or macromolecules (e.g., "collagen), A zalphal 1 polypeptide can be fused to two or more moieties, such' as an affinity tag for purification and a targeting domain. Polypeptide fusions can also comprise one or more cleavage sites, particularly between domains. See, Tuan et al., Connective Tissue Research 34-1-9,1996.<br>
The proteins of the present invention can also comprise non-naturally<br>
occurring amino acid residues. Non-naturally occurring amino acids include, without<br>
limitation, /raw-3-methylproline, 2,4-methanoproline, cw-4-hydroxyproline, transA-<br>
hydroxyproline,	TZ-methylglycine,	a//o-threonine,	methylthreonine,<br>
hydroxyethylcysteine, hydroxyethylhomocysteine, nitroglutamine, homoglutamine, pipecolic acid, thiazolidine carboxylic acid, dehydroproline, 3- and 4-methylproline, 3,3-dimethylproline, /er/-leucine, norvaline, 2-azaphenylalanine, 3-azaphenylalanine, 4-azaphenylalanine, and 4-fluorophenylalanine. Several methods are known in the art for incorporating non-naturally occurring amino acid residues into proteins. For example, an in vitro system can be employed wherein nonsense mutations are suppressed using chemically aminoacylated suppressor tRNAs. Methods for synthesizing amino acids and aminoacylating tRNA are known in the art. Transcription and translation of plasmids containing nonsense mutations is carried out in a cell-free system comprising an E. coli S30 extract and commercially available enzymes and other reagents. Proteins are purified by chromatography. See, for example, Robertson et al., J. Am. Chem. Soc. 113:2722. 1991; Ellman et al.. Methods Enzvmol. 202:301, 1991; Chung et al., Science 259:806-9,1993; and Chung et al., Proc. Natl. Acad. Sci. USA 90:10145-9.1993). In a second method, translation is carried out in Xenopus oocytes by microinjection of mutated mRNA and chemically aminoacylated suppressor tRNAs (Turcatti et al., L Biol. Chem. 271:19991-8, 1996). Within a third method, E. coli cells are cultured in the absence of a natural amino acid that is to be replaced (e.g., phenylalanine) and in the presence of the desired non-naturally occurring amino acid(s) (e.g., 2-azaphenylalanine, 3-azaphenylalanine, 4-azaphenylalanine, or 4-fluorophenylalanine). The non-naturally occurring amino acid is incorporated into the protein in place of its natural counterpart. See, Koide et al., Biochem. 33:7470-7476, 1994. Naturally occurring amino acid residues can be converted to non-naturally occurring species by in vitro chemical modification.  Chemical modification can be combined with site-directed mutagenesis<br><br>
to further expand the range of substitutions (Wynn and Richards, Protein Sci. 2:395-403,1993).<br>
A limited.number of non-conservative amino acids, amino acids that are not encoded by the. genetic code, non-naturally occurring amino acids, and unnatural amino acids may be substituted for zalphal 1 amino acid residues.<br>
Essential amino acids in the polypeptides of the present invention can be identified according to procedures known in the art, such as site-directed mutagenesis or alanine-scaiming mutagenesis (Cuimingham and Wells, Science 244: 1081-5, 1989; Bass et al., Proc. Natl. Acad. Sci. USA 88:4498-502, 1991). In the latter technique, single alanine mutations are introduced at every residue in the molecule, and -the resultant mutant molecules are tested for biological activity (e.g. ligand binding and signal transduction) as disclosed below to identify amino acid residues that are critical to the activity of the molecule. See also, Hilton et al., J. Biol. Chem. 271:4699-4708. 1996. Sites of ligand-receptor, protein-protein or other biological interaction can also be determined by physical analysis of structure, as determined by such techniques as nuclear magnetic resonance, crystallography, electron diffraction or photoaffinity labeling, in conjunction with mutation of putative contact site amino acids. See, for example, de Vos et al.. Science 255:306-312, 1992; Smith et al., J. Mol. Biol. 224:899-904, 1992; Wlodaver et al., FEBS Lett. 309:59-64, 1992. The identities of essential amino acids can also be inferred from analysis of homologies with related receptors.<br>
Determination of amino acid residues that are within regions or domains that are critical to maintaining structural integrity can be determined. Within these regions one can determine specific residues that will be more or less tolerant of change and maintain the overall tertiary structure of the molecule. Methods for analyzing sequence structure include, but are not limited to, alignment of multiple sequences with high amino acid or nucleotide identity and computer analysis using available software (e.g., the Insight II® viewer and homology modeling tools; MSI, San Diego, CA), secondary structure propensities, binary patterns, complementary packing and buried polar interactions (Barton, Current Opin. Struct. Biol. 5:372-376, 1995 and Cordes et al.. Current Opin. Struct. Biol. 6:3-10, 1996). In general, when designing modifications<br><br>
to molecules or identifying specific fragments determination of structure will be accompanied by evaluating activity of modified molecules.<br>
Amino acid sequence changes are made in zalphal 1 polypeptides so as to minimize disruptioti of higher order structure essential to biological activity. For example, when the zalphal 1 polypeptide comprises one or more helices, changes in amino acid residues will be made so as not to disrupt the helix geometry and other components of the molecule where changes in conformation abate some critical function, for example, binding of the molecule to its binding partners. The effects of amino acid sequence changes can be predicted by, for example, computer modeling as disclosed above or determined by analysis of crystal structure (see, e.g., Lapthom et al., Nat. Struct. Biol. 2:266-268, 1995). Other techniques that are well known in the art compare folding of a variant protein to a standard molecule (e.g., the native protein). For example, comparison of the cysteine pattern in a variant and standard molecules can be made. Mass spectrometry and chemical modification using reduction and alkylation provide methods for determining cysteine residues which are associated with disulfide bonds or are free of such associations (Bean et al. Anal. Biochem. 201:216-226, 1992; Gray, Protein Sci. 2:1732-1748, 1993; and Patterson et al., Anal. Chem. 66:3727-3732,1994). It is generally believed that if a modified molecule does not have the same disulfide bonding pattem as the standard molecule folding would be affected. Another well known and accepted method for measuring folding is circular dichrosism (CD). Measuring and comparing the CD spectra generated by a modified molecule and standard molecule is routine (Johnson, Proteins 7:205-214, 1990). Crystallography is another well known method for analyzing folding and structure. Nuclear magnetic resonance (NMR), digestive peptide mapping and epitope mapping are also known methods for analyzing folding and structural similarities between proteins and polypeptides (Schaanan et al., Science 257:961-964, 1992).<br>
A Hopp/Woods hydrophilicity profile of the zalphal 1 protein sequence as shown in SEQ ID N0:2 can be generated (Hopp et al., Proc. Nati. Acad. Sci78:3824-3828, 1981; Hopp, J. Immun. Meth. 88:1-18, 1986 and Triquier et al., Protein Engineering 11:153-169, 1998). See, Figiare 1. The profile is based on a sliding six-residue window.   Buried G, S, and T residues and exposed H, Y, and W<br><br>
residues were ignored. For example, in zalphall, hydrophilic regions include amino acid residues 55 through 60 of SEQ ID NO: 2, amino acid residues 56 through 61 of SEQ ID NO: 2, amino acid residues 139 through 144 of SEQ ID NO: 2, amino acid residues 227 throug.h'i32 of SEQ ID NO: 2, and amino acid residues 364 through 369 of SEQ ID NO: 2.<br>
Those skilled in the art will recognize that hydrophilicity or hydrophobicity will be taken into accoimt when designing modifications in the amino acid sequence of a zalphall polypeptide, so as not to disrupt the overall structural and biological profile. Of particular interest for replacement are hydrophobic residues selected from the group consisting of Val, Leu and He or the group consisting of Met, Gly, Ser, Ala, Tyr and Trp. For example, residues tolerant of substitution could include such residues as shown in SEQ ID NO: 2. However, Cysteine residues could be relatively intolerant of substitution.<br>
The identities of essential amino acids can also be inferred from analysis of sequence similarity between class I cytokine receptor family members with zalphall. Using methods such as "FASTA" analysis described previously, regions of high similarity are identified within a family of proteins and used to analyze amino acid sequence for conserved regions. An alternative approach to identifying a variant zalphal 1 polynucleotide on the basis of structure is to determine whether a nucleic acid molecule encoding a potential variant zalphall polynucleotide can hybridize to a nucleic acid molecule having the nucleotide sequence of SEQ ID N0:1, as discussed above.<br>
Other methods of identifying essential amino acids in the polypeptides of the present invention are procedures known in the art, such as site-directed mutagenesis or alanine-scanning mutagenesis (Cunningham and Wells, Science 244:1081 (1989), Bass et al., Proc. Natl Acad. Sci. USA 88:4498 (1991), Coombs and Corey, "Site-Directed Mutagenesis and Protein Engineering," in Proteins: Analysis and Design. Angeletti (ed.), pages 259-311 (Academic Press, Inc. 1998)). In the latter technique, single alanine mutations are introduced at every residue in the molecule, and the resultant mutant molecules are tested for biological activity as disclosed below to<br><br>
identify amino acid residues that are critical to the activity of the molecule. See also, Hilton et al. J. Biol them: 277:4699 (1996).<br>
The present invention also includes functional fragments of zalphal 1 polypeptides and nucleic acid molecules encoding such functional fragments. A "functional" zalphall or fragment thereof defined herein is characterized by its proliferative or differentiating activity, by its ability to induce or inhibit specialized cell functions, or by its ability to bind specifically to an anti-zalphal 1 antibody or zalphal 1 ligand (either soluble or immobilized). As previously described herein, zalphall is characterized by a class I cytokine receptor structure. Thus, the present invention further provides fusion proteins encompassing: (a) polypeptide molecules comprising an extracellular or intracellular domain described herein; and (b) functional fragments comprising one or more of these domains. The other polypeptide portion of the fusion protein may be contributed by another class I cytokine receptor, for exapmple, IL-2 receptor P-subunit and the P-common receptor (i.e., IL3, IL-5, and GM-CSF receptor P-subunits), or by a non-native and/or an unrelated secretory signal peptide that facilitates secretion of the fusion protein.<br>
Routine deletion analyses of nucleic acid molecules can be performed to<br>
obtain functional fragments of a nucleic acid molecule that encodes a zalphall<br>
polypeptide.   As an illustration, DNA molecules having the nucleotide sequence of<br>
SEQ ID N0:1 or fragments thereof, can be digested with Bal31 nuclease to obtain a<br>
series of nested deletions.   These DNA fragments are then inserted into expression<br>
vectors m proper reading frame, and the expressed polypeptides are isolated and tested<br>
for zalphal 1 activity, or for the ability to bind anti-zalphal 1 antibodies or zalphal 1<br>
ligand.   One altemative to exonuclease digestion is to use oligonucleotide-directed<br>
mutagenesis to introduce deletions or stop codons to specify production of a desired<br>
zalphal 1 fragment.   Alternatively, particular fragments of a zalphal 1 polynucleotide<br>
can be synthesized using the polymerase chain reaction.<br>
Standard methods for identifying functional domains are well-known to<br>
those of skill in the art. For example, studies on the truncation at either or both termini<br>
of interferons have been summarized by Horisberger and Di Marco, Pharmac. Ther.<br>
66:507 (1995).  Moreover, standard techniques for functional analysis of proteins are<br>
described by, for example, Treuter et al., Molec. Gen. Genet. 240:113 (1993); Content<br>
et al., "Expression and preliminary deletion analysis of the 42 kDa 2-5A synthetase<br><br>
induced by human interferon," in Biological Interferon Systems. Proceedings of ISIR-TNO Meeting on Interferon Systems. Cantell (ed.), pages 65-72 (Nijhoff 1987); Herschman, "The EGF Receptor," in Control of Animal Cell Proliferation 1 Boynton et al., (eds.) pages 16:9-199 (Academic Press 1985); Coumailleau et al., J. Biol. Chem. 270:29270 (1995); Fukunaga et al.. J. Biol. Chem 270:25291 (1995); Yamaguchi et al., Biochem. Pharmacol. 50:1295 (1995); and Meisel et al., Plant Molec. Biol. 30:1 (1996).<br>
Multiple amino acid substitutions can be made and tested using known methods of mutagenesis and screening, such as those disclosed by Reidhaar-OIson and Sauer fScience 241:53-57, 1988) or Bowie and Sauer rProc. Natl. Acad. Sci. USA 86:2152-2156, 1989). Briefly, these authors disclose methods for simultaneously randomizing two or more positions in a polypeptide, selecting for functional polypeptide, and then sequencing the mutagenized polypeptides to determine the spectrum of allowable substitutions at each position. Other methods that can be used include phage display (e.g., Lowman et al., Biochem. 30:10832-10837, 1991; Ladner et al., U.S. Patent No. 5,223,409; Huse, WIPO Publication WO 92/062045) and region-directed mutagenesis (Derbyshire et al.. Gene 46:145, 1986; Ner et al., DNA 7:127, 1988).<br>
Variants of the disclosed zalphal 1 DNA and polypeptide sequences can be generated through DNA shuffling as disclosed by Stemmer, Nature 370:389-91. 1994, Stemmer. Proc. Natl. Acad. Sci. USA 91:10747-51, 1994 and WIPO PuWication WO 97/20078. Briefly, variant DNAs are generated by in vitro homologous recombination by random fragmentation of a parent DNA followed by reassembly using PCR, resulting in randomly introduced point mutations. This technique can be modified by using a family of parent DNAs, such as allelic variants or DNAs from different species, to introduce additional variability into the process. Selection or screening for the desired activity, followed by additional iterations of mutagenesis and assay provides for rapid "evolution" of sequences by selecting for desirable mutations while simultaneously selecting against detrimental changes.<br>
Mutagenesis methods as disclosed herein can be combined with high-throughput, automated screening methods to detect activity of cloned, mutagenized<br><br>
zalphal 1 receptor polypeptides in host cells. Preferred assays in this regard include cell proliferation assays and biosensor-based ligand-binding assays, which are described below. Mutagenized'DNA molecules that encode active receptors or portions thereof (e.g., ligand-binding fragments, signaling domains, and the like) can be recovered from the host cells and rapidly sequenced using modem equipment. These methods allow the rapid determination of the importance of individual amino acid residues in a polypeptide of interest, and can be applied to polypeptides of unknown structure.<br>
In addition, the proteins of the present invention (or polypeptide fragments thereof) can be joined to other bioactive molecules, particularly other cytokines, to provide multi-fiinctional molecules. For example, one or more helices from zalphal 1 can be joined to other cytokines to enhance their biological properties or efficiency of production.<br>
The present invention thus provides a series of novel, hybrid molecules in which a segment comprising one or more of the helices of zalphal 1 is fused to another polypeptide. Fusion is preferably done by splicing at the DNA level to allow expression of chimeric molecules in recombinant production systems. The resultant molecxiles are then assayed for such properties as improved solubility, improved stability, prolonged clearance half-life, improved expression and secretion levels, and pharmacodynamics. Such hybrid molecules may further comprise additional amino acid residues (e.g. a polypeptide linker) between the component proteins or polypeptides.<br>
Using the methods discussed herein, one of ordinary skill in the art can identify and/or prepare a variety of polypeptide fragments or variants of SEQ ID N0:2 that retain the signal transduction or ligand binding activity. For example, one can make a zalphal 1 "soluble receptor" by preparing a variety of polypeptides that are substantially homologous to the cytokine-binding domain (residues 20 (Cys) to 237 (His) of SEQ ID N0:2 or allelic variants or species orthologs thereof) and retain ligand-binding activity of the wild-type zalphal 1 protein. Such polypeptides may include additional amino acids from, for example, part or all of the transmembrane and intracellular domains. Such polypeptides may also include additional polypeptide segments as generally disclosed herein such as labels, affinity tags, and the like.<br>
V<br><br>
For any zaiphal 1 polypeptide, including variants, soluble receptors, and fusion polypeptides or proteins, one of ordinary -skill in the art can readily generate a fully degenerate polynucleotide sequence encoding that variant using the information set forth in Tables 1. and 2 above.<br>
The zaiphal 1 polypeptides of the present invention, including full-length polypeptides, biologically active fragments, and fusion polypeptides, can be produced in genetically engineered host cells according to conventional techniques. Suitable host cells are those cell types that can be transformed or transfected with exogenous DNA and grown in culture, and include bacteria, fongal cells, and cultured higher eukaryotic cells. Eukaryotic cells, particularly cultured cells of multicellular organisms, are preferred. Techniques for manipulating cloned DNA molecules and introducing exogenous DNA into a variety of host cells are disclosed by Sambrook et al.. Molecular Cloning: A Laboratory Manual. 2nd ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989, and Ausubel et al., eds., Current Protocols in Molecular Bioloev. John Wiley and Sons, Inc., NY, 1987.<br>
In general, a DNA sequence encoding a zaiphal 1 polypeptide is operably linked to other genetic elements required for its expression, generally including a transcription promoter and terminator, within an expression vector. The vector will also commonly contain one or more selectable markers and one or more origins of replication, although those skilled in the art will recognize that within certain systems selectable markers may be provided on separate vectors, and replication of the exogenous DNA may be provided by integration into the host cell genome. Selection of promoters, terminators, selectable markers, vectors and other elements is a matter of routine design within the level of ordinary skill in the art.   Many such elements are<br>
t<br>
described in the literature and are available through commercial suppliers.<br>
To direct a zaiphal 1 polypeptide into the secretory pathway of a host cell, a secretory signal sequence (also known as a leader sequence, prepro sequence or pre sequence) is provided in the expression vector. The secretory signal sequence may be that of zaiphal 1, or may be derived from another secreted protein (e.g., t-PA) or synthesized de novo. The secretory signal sequence is operably linked to the zaiphal 1 DNA sequence, i.e., the two sequences are joined in the correct reading frame and<br><br>
positioned to direct the newly synthesized polypeptide into the secretory pathway of the host cell. Secretory signal sequences are commonly positioned 5' to the DNA sequence encoding the polypeptide of interest, although certain secretory signal sequences may be positioned elsewhere in the DNA sequence of interest (see, e.g., Welch et al., U.S. Patent No. 5,037,743; Holland et al., U.S. Patent No. 5,143,830).<br>
Alternatively, the secretory signal sequence contained in the polypeptides of the present invention is used to direct other polypeptides into the secretory pathway. The present invention provides for such fiision polypeptides. A signal fusion polypeptide can be made wherein a secretory signal sequence derived from amino acid 1 (Met) to amino acid 19 (Gly) of SEQ ID N0:2 is operably linked to another polypeptide using methods known in the art and disclosed herein. The secretory signal sequence contained in the fusion polypeptides of the present invention is preferably fused amino-termuially to an additional peptide to direct the additional peptide into the secretory pathway. Such constructs have numerous applications known in the art. For example, these novel secretory signal sequence fusion constructs can direct the secretion of an active component of a normally non-secreted protein. Such fusions may be used in vivo or in vitro to direct peptides through the secretory pathway.<br>
Cultured mammalian cells are suitable hosts within the present invention. Methods for introducing exogenous DNA into mammalian host cells include calcium phosphate-mediated transfection (Wigler et al.. Cell 14:725, 1978; Corsaro and Pearson, Somatic Cell Genetics 7:603, 1981: Graham and Van der Eb, Virology 52:456, 1973), electroporation (Neumann et al., EMBO J. 1:841-845, 1982), DEAE-dextran mediated transfection (Ausubel et al., ibid.), and liposome-mediated transfection (Hawley-Nelson et al.. Focus 15:73, 1993; Ciccarone et al.. Focus 15:80, 1993, and viral vectors (Miller and Rosman, BioTechniaues 7:980-90, 1989; Wang and Finer, Nature Med. 2:714-716, 1996). The production of recombinant polypeptides in cultured mammalian cells is disclosed, for example, by Levinson et al., U.S. Patent No. 4,713,339; Hagen et al., U.S. Patent No. 4,784,950; Palmiter et al., U.S. Patent No. 4,579,821; and Ringold, U.S. Patent No. 4,656,134. Suitable cultured mammalian cells include tiie COS-1 (ATCC No. CRL 1650), COS-7 (ATCC No. CRL 1651), BHK (ATCC No. CRL 1632), BHK 570 (ATCC No. CRL 10314), 293 (ATCC No. CRL<br><br>
1573; Graham et al., J. Gen. Virol. 36:59-72, 1977) and Chinese hamster ovary (e.g. CHO-Kl; ATCC No.' CCL 61) cell lines. Additional suitable cell lines are known in the art and availableVfrpm public depositories such as the American Type Culture Collection, Rockville^ Maryland. In general, strong transcription promoters are preferred, such as promoters from SV-40 or cytomegalovirus. See, e.g., U.S. Patent No. 4,956,288. Other suitable promoters include those from metallothionein genes (U.S. Patent Nos. 4,579,821 and 4,601,978) and the adenovirus major late promoter.<br>
Drug selection is generally used to select for cultiured mammalian cells into which foreign DNA has been inserted. Such cells are commonly referred to as "transfectants". Cells that have been cultured in the presence of the selective agent and are able to pass the gene of interest to their progeny are referred to as "stable transfectants." A preferred selectable marker is a gene encoding resistance to the antibiotic neomycin. Selection is carried out in the presence of a neomycin-type drug, such as G-418 or the like. Selection systems can also be used to increase the expression level of the gene of interest, a process referred to as "amplification." Amphfication is carried out by culturing transfectants in the presence of a low level of the selective agent and then increasing the amount of selective agent to select for cells that produce high levels of the products of the introduced genes. A preferred amplifiable selectable marker is dihydrofolate reductase, which confers resistance to methotrexate. Other drug resistance genes (e.g. hygromycin resistance, multi-drug resistance, puromycin acetyltransferase) can also be used. Altemative markers that introduce an altered phenotype, such as green fluorescent protein, or cell surface proteins such as CD4, CD8, Class IMHC, placental alkaline phosphatase may be used to sort transfected cells from untransfected cells by such means as FACS sorting or magnetic bead separation technology.<br>
Other higher eukaryotic cells can also be used as hosts, including plant cells, insect cells and avian cells. The use of Agrobacterium rhizogenes as a vector for expressing genes in plant cells has been reviewed by Sinkar et al., J. Biosci. (Bmigalore) n.:47-58, 1987. Transformation of insect cells and production of foreign polypeptides therein is disclosed by Guarino et al., U.S. Patent No. 5,162,222 and WIPO publication WO 94/06463. Insect cells can be infected with recombinant<br><br>
baculovirus, commonly derived from Autographa californica nuclear polyhedrosis virus (AcNPV). See, King, L.A. and Possee, R.D., The Baculovirus Expression Svstem: A Laboratbrv Guide, London, Chapman &amp; Hall; O'Reilly, D.R. et al., Baculovirus Expression Vectors: A Laboratory Manual, Nev/ York, Oxford University Press., 1994; and, Richardson, C. D., Ed., Baculovirus Expression Protocols. Methods in Molecular Biologv, Totowa, NJ, Humana Press, 1995. A second method of making recombinant zalphall baculovirus utilizes a transposon-based system described by Luckow (Luckow, V.A, et al.. J Virol 67:4566-79, 1993). This system, which utilizes transfer vectors, is sold in the Bac-to-Bac™ kit (Life Technologies, Rockville, MD). This system utilizes a transfer vector, pFastBacF" (Life. Technologies) containing a Tn7 transposon to move the DNA encoding the zalphal 1 polypeptide into a baculovirus genome maintained in E. coli as a large plasmid called a "bacmid." See, Hill-Perkins, M.S. and Possee, R.D., J Gen Virol 71:971-6, 1990; Bonning, B.C. et al., J Gen Virol 75:1551-6, 1994; and, Chazenbalk, G.D., and Rapoport, B., J Biol Chem 270:1543-9. 1995. In addition, transfer vectors can include an in-frame fiision with DNA encoding an epitope tag at the C- or N-terminus of the expressed zalphal 1 polypeptide, for example, a Glu-Glu epitope tag (Grussenmeyer, T. et al., Proc. Natl. Acad. Sci. 82:7952-4, 1985). Using a technique known in the art, a transfer vector containing zalphall is transformed into E. Coli. and screened for bacmids which contain an interrupted lacZ gene indicative of recombinant baculovirus. The bacmid DNA containing the recombinant baculovirus genome is isolated, using conmion techniques, and used to transfect Spodoptera frugiperda cells, e.g. Sf9 cells. Recombinant virus that expresses zalphal 1 is subsequently produced. Recombinant viral stocks are made by methods commonly used in the art.<br>
The recombinant virus is used to infect host cells, typically a cell line derived from the fall armyworm, Spodoptera frugiperda. See, in general, Glick and Pasternak, Molecular Biotechnoloev: Principles and Applications of Recombinant DNA. ASM Press, Washington, D.C., 1994. Another suitable cell line is the High FiveO™ cell line (Invitrogen) derived from Trichoplusia ni (U.S. Patent No. 5,300,435). Commercially available serum-free media are used to grow and maintain the cells. Suitable media are Sf900 11™ (Life Technologies) or ESF 921™ (Expression<br><br>
Systems) for the Sf9 cells; and Ex-ceIlO405™ (JRH Biosciences, Lenexa, KS) or Express FiveOtm (Life Technologies) for the T. ni cells. Procedures used are generally described in available laboratory manuals (King, L. A. and Possee, R.D., ibid.: O'Reilly, D.R. et al.ibid.; Richardson, C. D., ibid). Subsequent purification of the 2alphal 1 polypeptide from the supernatant can be achieved using methods described herein.<br>
Fungal cells, including yeast cells, can also be used within the present invention. Yeast species of particular interest in this regard include Saccharomyces cerevisiae, Pichia pastoris, and Pichia methanolica. Methods for transforming S. cerevisiae cells with exogenous DNA and producing recombinant polypeptides therefrom are disclosed by, for example, Kawasaki, U.S. Patent No. 4,599,311; Kawasaki et al., U.S. PatentNo. 4,931,373; Brake, U.S. Patent No. 4,870,008; Welch et al., U.S. Patent No. 5,037,743; and Murray et al., U.S. Patent No. 4,845,075. Transformed cells are selected by phenotype determined by the selectable marker, commonly drug resistance or the ability to grow in the absence of a particular nutrient (e.g., leucine). A preferred vector system for use in Saccharomyces cerevisiae is the POTl vector system disclosed by Kawasaki et al. (U.S. Patent No. 4,931,373), which allows transformed cells to be selected by growth in glucose-containing media. Suitable promoters and terminators for use in yeast include those from glycolytic enzyme genes (see, e.g., Kawasaki, U.S. Patent No. 4,599,311; Kingsman et al., U.S. Patent No. 4,615,974; and Bitter, U.S. Patent No. 4,977,092) and alcohol dehydrogenase genes. See also U.S. Patents Nos. 4,990,446; 5,063,154; 5,139,936 and 4,661,454. Transformation systems for other yeasts, including Hansenula polymorpha, Schizosaccharomyces pombe, Kluyveromyces lactis, Kluyveromyces fragilis, Ustilago<br>
t<br>
maydis, Pichia pastoris, Pichia methanolica, Pichia guillermondii and Candida maltosa are known in the art. See, for example, Gleeson et al., J. Gen. Microbiol. 132:3459-3465, 1986 and Gregg, U.S. Patent No. 4,882,279. Aspergillus cells may be utilized according to the methods of McKnight et al., U.S. Patent No. 4,935,349. Methods for transforming Acremonium chrysogenum are disclosed by Sumino et al., U.S. Patent No. 5,162,228. Methods for transforming Neurospora are disclosed by Lambowitz, U.S. Patent No. 4,486,533.<br><br>
The use of-Pichia methanolica as host for the production of recombinant proteins is disclosed in WIPO Publications WO 97/17450, WO 97/17451, WO 98/02536, and WO 98/02565. DNA molecules for use in transforming P. methanolica will commonly be prepared as double-stranded, circular plasmids, which are preferably linearized prior to transformation. For polypeptide production in P. methanolica, it is preferred that the promoter and terminator in the plasmid be that of a P. methanolica gene, such as a P. methanolica alcohol utilization gene (AUGl oxAUG2). Other useful promoters include those of the dihydroxyacetone synthase (DHAS), formate dehydrogenase (FMD), and catalase (CAT) genes. To facilitate integration of the DNA into the host chromosome, it is preferred to have the entire expression segment of the plasmid flanked at both ends by host DNA sequences. A preferred selectable marker for use in Pichia methanolica is a P. methanolica ADE2 gene, which encodes phosphoribosyl-5-aniinoimidazoIe carboxylase (AIRC; EC 4.1,1.21), which allows adel host cells to grow in the absence of adenine. For large-scale, industrial processes where it is desirable to minimize the use of methanol, it is preferred to use host cells in which both methanol utilization genes {A UGl and A UG2) are deleted. For production of secreted proteins, host cells deficient in vacuolar protease genes {PEP4 and PRBl) are preferred. Electroporation is used to facilitate the introduction of a plasmid containing DNA encoding a polypeptide of interest into P. methanolica cells. It is preferred to transform P. methanolica cells by electroporation using an exponentially decaying, pulsed electric field having a field strength of from 2.5 to 4.5 kV/cm, preferably about 3.75 kV/cm, and a time constant (t) of fi-om 1 to 40 milliseconds, most preferably about 20 milliseconds.<br>
Prokaryotic host cells, including strains of the bacteria Escherichia coli. Bacillus and other genera are also useful host cells within the present invention. Techniques for transforming these hosts and expressing foreign DNA sequences cloned therein are well known in the art (see, e.g., Sambrook et al., ibid.). When expressing a zalphal 1 polypeptide in bacteria such as E. coli, the polypeptide may be retained in the cytoplasm, typically as insoluble granules, or may be directed to the periplasmic space by a bacterial secretion sequence. In the former case, the cells are lysed, and the granules are recovered and denatured using, for example, guanidine isothiocyanate or<br><br>
urea. The denatured polypeptide can then be refolded and dimerized by diluting the denaturant, such as by dialysis against a solution-of urea and a combination of reduced and oxidized glutathione, followed by dialysis against a buffered saline solution. In the latter case, the polypeptide can be recovered from the periplasmic space in a soluble and functional form by disrupting the cells (by, for example, sonication or osmotic shock) to release the contents of the periplasmic space and recovering the protein, thereby obviating the need for denaturation and refolding.<br>
Transformed or transfected host cells are cultured according to conventional procedures in a culture medium containing nutrients and other components required for the growth of the chosen host cells. A variety of suitable media, including defined media and complex media, are known in the art and generally include a carbon source, a nitrogen source, essential amino acids, vitamins and minerals. Media may also contain such components as growth factors or serum, as required. The growth medium will generally select for cells containing the exogenously added DNA by, for example, drug selection or deficiency in an essential nutrient which is complemented by the selectable marker carried on the expression vector or co-transfected into the host cell. P. methanolica cells are cultured in a mediimi comprising adequate sources of carbon, nitrogen and trace nutrients at a temperature of about 25°C to 35°C. Liquid cultures are provided with sufficient aeration by conventional means, such as shaking of small flasks or sparging of fermentors. A preferred culture medium for P. methanolica is YEPD (2% D-glucose, 2% Bacto™ Peptone (Difco Laboratories, Detroit, MI), 1% Bacto™ yeast extract (Difco Laboratories), 0.004% adenine and 0.006% L-leucine).<br>
Within one aspect of the present invention, a zalphal 1 cytokine receptor (including transmembrane and intracellular domains) is produced by a cultured cell, and the cell is used to screen for ligands for the receptor, including the natural ligand, as well as agonists and antagonists of the natural ligand. To summarize this approach, a cDNA or gene encoding the receptor is combined with other genetic elements required for its expression (e.g., a transcription promoter), and the resulting expression vector is inserted into a host cell. Cells that express the DNA and produce functional receptor are selected and vised within a variety of screening systems.<br><br>
Mammalian cells suitable for use in expressing the novel receptors of the present invention and traiisducing a receptor-mediated signal include ceils that express a β -subunit, such as gpl30, and cells that co-express gpl30 and LIF receptor (Gearing et al., EMBOJ. i0:28;39-2848,1991; Gearing et al., U.S. Patent No. 5,284,755). In this regard it is generally preferred to employ a cell that is responsive to other cytokines that bind to receptors in the same subfamily, such as IL-6 or LIF, because such cells will contain the requisite signal transduction pathway(s). Preferred cells of this type include the human TF-1 cell line (ATCC number CRL-2003) and the DA-1 cell line (Branch et al.. Blood 69:1782,1987; Broudy et al.. Blood 75:1622-1626,1990). In the alternative, suitable host cells can be engineered to produce a p-subunit or other cellular component needed for the desired cellular response. For example, the murine cell line BaF3 (Palacios and Steinmetz, Cell 41:727-734, 1985; Mathey-Prevot et al., Mol. Cell. Biol. 6: 4133-4135, 1986), a baby hamster kidney (BHK) cell line, or the CTLL-2 cell line (ATCC TIB-214) can be transfected to express the mouse gpl30 subunit, or mouse gpl30 and LIF receptor, in addition to zalphall. It is generally preferred to use a host cell and receptor(s) from the same species, however this approach allows cell lines to be engineered to express multiple receptor subunits from any species, thereby overcoming potential limitations arising from species specificity. In the alternative, species homologs of the human receptor cDNA can be cloned and used within cell lines from the same species, such as a mouse cDNA in the BaF3 cell line. Cell lines that are dependent upon one hematopoietic growth factor, such as IL-3, can thus be engineered to become dependent upon a zalphal 1 ligand.<br>
Cells expressing functional zalphal 1 are used within screening assays. A variety of suitable assays are known in the art. These assays are based on the detection of a biological response in the target cell. One such assay is a cell proliferation assay. Cells are cultured in the presence or absence of a test compound, and cell proliferation is detected by, for example, measuring incorporation of tritiated thymidine or by colorimetric assay based on the metabolic breakdown of Alymar Blue™ (AccuMed, Chicago, IL) or 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyl tetrazolium bromide (MTT) (Mosman, J. Immunol. Meth. 65: 55-63, 1983). An alternative assay format uses cells that are further engineered to express a reporter gene.<br><br>
The reporter gene is linked to a promoter element that is responsive to the receptor-linked pathway, and the assay detects activation of transcription of the reporter gene. A preferred promoter element in this regard is a serum response element, or SRE (see, for example, Shaw et al. Cell 56:563-572, 1989). A preferred such reporter gene is a luciferase gene (de Wet et al., Mol. Cell. Biol. 2:725, 1987). Expression of the luciferase gene is detected by luminescence using methods known in the art (e.g., Baumgartner et al., J. Biol. Chem. 269:19094-29101, 1994; Schenbom and Goiffin, Promega Notes 41.: 11, 1993). Luciferase assay kits are commercially available from, for example, Promega Corp., Madison, WI. Target cell lines of this type can be used to screen libraries of chemicals, cell-conditioned culture media, fungal broths, soil samples, water samples, and the like. For example, a bank of cell- or tissue-conditioned media samples can be assayed on a target cell to identify cells that produce ligand. Positive cells are then used to produce a cDNA hbrary in a mammalian cell expression vector, which is divided into pools, transfected into host cells, and expressed. Media samples from the transfected cells are then assayed, with subsequent division of pools, retransfection, subculturing, and re-assay of positive cells to isolate a clonal cell line expressing the ligand. Media samples conditioned by kidney, liver, spleen, thymus, other lymphoid tissues, or T-cells are preferred sources of ligand for use in screening procedures.<br>
A natural ligand for zalphal 1 can also be identified by mutagenizing a cytokine-dependent cell line expressing zalphal 1 and cnlturing it under conditions that select for autocrine growth. See WIPO pubUcation WO 95/21930. Within a typical procedure, cells expressing zalphal 1 are mutagenized, such as with EMS. The cells are then allowed to recover in the presence of the required cytokine, then transferred to a culture medium lacking the cytokine. Surviving cells are screened for the production of a ligand for zalphal 1, such as by adding soluble (ligand-binding) receptor polypeptide to the culture medium or by assaying conditioned media on wild-type cells and transfected cells expressing the zalphal 1. Preferred cell lines for use within this method include cells that are transfected to express gpl30 or gpl30 in combination with LIF receptor. Preferred such host cell lines include transfected CTLL-2 cells (Gillis and Smith, Nature 268:154-156,1977) and transfected BaF3 cells.<br><br>
Moreover, a secretion trap method employing zalphal 1 soluble receptor polypeptide can be used to isolate a 2alphal 1 ligand (Aldrich, et al, Cell 87: 1161-1169, 1996). A cDNA expression library prepared from a known or suspected ligand source is transfected into COS-7 cells. The cDNA library vector generally has an SV40 origin for amplification in COS-7 cells, and a CMV promoter for high expression. The transfected COS-7 cells are grown in a monolayer and then fixed and permeabilized. Tagged or biotin-labeled zalphal 1 soluble receptor, described herein, is then placed in contact with the cell layer and allowed to bind cells in the monolayer that express an anti-complementary molecule, i.e., a zalphal 1 ligand. A cell expressing a ligand will thus be bound with receptor molecules. An anti-tag antibody (anti-Ig for Ig fusions, M2 or anti-FLAG for FLAG-tagged fusions, streptavidin, and the like) which is conjugated with horseradish peroxidase (HRP) is used to visualize these cells to which the tagged or biotin-labeled zalphal 1 soluble receptor has bound. The HRP catalyzes deposition of a tyramide reagent, for example, tyramide-FITC. A commercially-available kit can be used for this detection (for example. Renaissance TSA-Directtm Kit; NEN Life Science Products, Boston, MA). Cells which express zalphal 1 receptor ligand will be identified under fluorescence microscopy as green cells and picked for subsequent cloning of the ligand using procedures for plasmid rescue as outlined in Aldrich, et al, supra., followed by subsequent rounds of secretion trap assay until single clones are identified.<br>
As a receptor, the activity of zalphal 1 polypeptide can be measured by a silicon-based biosensor microphysiometer which measures the extracellular acidification rate or proton excretion associated with receptor binding and subsequent physiologic cellular responses. An exemplary device is the Cytosensor Microphysiometer manufactured by Molecular Devices, Simnyvale, CA. A variety of cellular responses, such as cell proliferation, ion transport, energy production, inflammatory response, regulatory and receptor activation, and the like, can be measured by this method. See, for example, McConnell, H.M. et al., Science 257:1906-1912, 1992; Pitchford, S. et al.. Meth. Enzvmol. 228:84-108. 1997; Arimilli, S. et al., J. Immunol. Meth. 212:49-59, 1998; Van Liefde, I. Et al., Eur. J. Pharmacol. 346:87-95,   1998.     The microphysiometer can be used for assaying eukaryotic.<br><br>
prokaryotic, adherent or non-adherent cells. By measuring extracellular acidification changes in cell media over time, the microphysiometer directly measures cellular responses to various stimuli, including agonists, ligands, or antagonists of the zalphall polypeptide. Preferably, the microphysiometer is used to measure responses of a zalphal 1-expressing eukaryotic cell, compared to a control eukaryotic cell that does not express zalphall polypeptide. Zalphall-expressing eukaryotic cells comprise cells into which zalphal 1 has been transfected, as described herein, creating a cell that is responsive to zalphall-modulating stimuli, or are cells naturally expressing zalphall, such as zalphall-expressing cells derived from lymphoid, spleen, thymus tissue or PBLs. Differences, measured by an increase or decrease in extracellular acidification, in the response of cells expressing zalphall, relative to a control, are a direct measurement of zalphall-modulated cellular responses. Moreover, such zalphal 1-modulated responses can be assayed under a variety of stimuli. Also, using the microphysiometer, there is provided a method of identifying agonists and antagonists of zalphal 1 polypeptide, comprising providing cells expressing a zalphal 1 polypeptide, culturing a first portion of the cells in the absence of a test compovmd, culturing a second portion of the cells in the presence of a test compound, and detecting an increase or a decrease in a cellular response of the second portion of the cells as compared to the first portion of the cells. Antagonists and agonists, including the natural ligand for zalphal 1 polypeptide, can be rapidly identified using this method.<br>
Additional assays provided by the present invention include the use of hybrid receptor polypeptides. These hybrid polypeptides fall into two general classes. Within the first class, the intracellular domain of zalphall, comprising approximately residues 256 (Lys) to 528 (Ser) of SEQ ID N0:2, is joined to the ligand-binding domain of a second receptor. It is preferred that the second receptor be a hematopoietic cytokine receptor, such as mpl receptor (Souyri et al.. Cell 63:1137-1147, 1990). The hybrid receptor will further comprise a transmembrane domain, which may be derived firom either receptor. A DNA construct encoding the hybrid receptor is then inserted into a host cell. Cells expressing the hybrid receptor are cultured in the presence of a ligand for the binding domain and assayed for a response. This system provides a means for analyzing signal transduction mediated by zalphall while using readily<br><br>
available ligands. This system can also be used to determine if particular cell lines are capable of responding to signals transduced by.zalphall. A second class of hybrid receptor polypeptides comprise the extracellular (ligand-binding) domain of zalphal 1 (approximately residues 20 (Cys) to 237 (His) of SEQ ID N0:2) with a cytoplasmic domain of a second receptor, preferably a cytokine receptor, and a transmembrane domain. The transmembrane domain may be derived from either receptor. Hybrid receptors of this second class are expressed in cells known to be capable of responding to signals transduced by the second receptor. Together, these two classes of hybrid receptors enable the use of a broad spectrum of cell types within receptor-based assay systems.<br>
Cells found to express a ligand for zalphal 1 are then used to prepare a cDNA library from which the ligand-encoding cDNA may be isolated as disclosed above. The present invention thus provides, in addition to novel receptor polypeptides, methods for cloning polypeptide ligands for the receptors.<br>
The tissue specificity of zalphal 1 expression suggests a role in early thymocyte development and immune response regulation. These processes involve stimulation of cell proliferation and differentiation in response to the binding of one or more cytokines to their cognate receptors. In view of the tissue distribution observed for this receptor, agonists (including the natural ligand) and antagonists have enormous potential in both in vitro and in vivo applications. Compounds identified as receptor agonists are useful for stimulating proliferation and development of target cells in vitro and in vivo. For example, agonist compounds are useful as components of defined cell culture media, and may be used alone or in combination with other cytokines and hormones to replace serum that is commonly used in cell culture. Agonists are thus useful in specifically promoting the growth and/or development of T-cells, B-cells, and other cells of the lymphoid and myeloid lineages, and hematopoietic cells in culture.<br>
Agonist ligands for zalphal 1 may be useful in stimulating cell-mediated immunity and for stimulating lymphocyte proliferation, such as in the treatment of infections involving immunosuppression, including certain viral infections. Additional uses include tumor suppression, where malignant transformation results in tumor cells that are antigenic. Agonist ligands could be used to induce cytotoxicity, which may be<br><br>
mediated through activation of effector cells such as T-cells, NK (natural killer) cells, or LAK (lymphoid 'activated killer) cells, or- induced directly through apoptotic pathways. Agonist ligands may also be useful in treating leukopenias by increasing the levels of the affecte'd cell type, and for enhancing the regeneration of the T-cell repertoire after bone marrow transplantation.<br>
Antagonist Ugands or compounds may find utility in the suppression of the immune system, such as in the treatment of autoimmune diseases, mcluding rheumatoid arthritis, multiple sclerosis, diabetes mellitis, inflammatory bowel disease, Crohn's disease, etc. Immune suppression can also be used to reduce rejection of tissue or organ transplants and grafts and to treat T-cell specific leukemias or lymphomas by inhibiting proliferation of the affected cell type.<br>
Zalphal 1 may also be used within diagnostic systems for the detection of circulating levels of ligand. Within a related embodiment, antibodies or other agents that specifically bind to zalphal 1 can be used to detect circulating receptor polypeptides. Elevated or depressed levels of ligand or receptor polypeptides may be indicative of pathological conditions, including cancer. Soluble receptor polypeptides may contribute to pathologic processes and can be an indirect marker of an underlying disease. For example, elevated levels of soluble IL-2 receptor in human serum have been associated with a wide variety of inflammatory and neoplastic conditions, such as myocardial infarction, asthma, myasthenia gravis, rheumatoid arthritis, acute T-cell leukemia, B-cell lymphomas, chronic lymphocytic leukemia, colon cancer, breast cancer, and ovarian cancer (Heaney et al.. Blood 87:847-857,1996).<br>
A ligand-binding polypeptide of a zalphall receptor, or "soluble receptor," can be prepared by expressing a truncated DNA encoding the zalphall cytokine binding domain (approximately residue 20 (Cys) through residue 237 (His) of the human receptor (SEQ ID N0:2)) or the corresponding region of a non-human receptor. It is preferred that the extracelldar domain be prepared in a form substantially free of transmembrane and intracellular polypeptide segments. Moreover, ligand-binding polypeptide fiagments within the zalphall cytokine binding domain, described above, can also serve as zalphal 1 soluble receptors for uses described herein. To direct the export of a receptor polypeptide from the host cell, the receptor DNA is<br><br>
linked to a second DMA segment encoding a secretory peptide, such as a t-PA secretory peptide or a zalphafl secretory peptide. To facilitate purification of the secreted receptor polypeptide, a C-terminal extension, such as a poly-histidine tag, substance P, Flag™ peptide (Hopp et al., Bio/Technology 6:1204-1210, 1988; available from Eastman Kodak Co., New Haven, CT) or another polypeptide or protein for which an antibody or other specific binding agent is available, can be fused to the receptor polypeptide.<br>
In an alternative approach, a receptor extracellular domain can be expressed as a fiision with immunoglobulin heavy chain constant regions, typically an Fc fragment, which contains two constant region domains and lacks the variable region.<br>
Such fiisions are typically secreted as multimeric molecules wherein the Fc portions are disulfide bonded to each other and two receptor polypeptides are arrayed in close proximity to each other. Fusions of this type can be used to affinity purify the cognate ligand from solution, as an in vitro assay tool, to block signals in vitro by specifically titrating out ligand, and as antagonists in vivo by administering them parenterally to bind circulating ligand and clear it from the circulation. To purify ligand, a zalphal 1-Ig chimera is added to a sample containing the ligand (e.g., cell-conditioned culture media or tissue exfracts) under conditions that facilitate receptor-ligand binding (typically near-physiological temperature, pH, and ionic strength). The chimera-ligand complex is then separated by the mixture using protein A, which is immobilized on a solid support (e.g., insoluble resin beads). The ligand is then eluted using conventional chemical techniques, such as with a salt or pH gradient. In the alternative, the chimera itself can be bound to a solid support, with binding and elution carried out as above. Collected fractions can be re-fractionated until the desired level of purity is reached.<br>
. Moreover, zalphal 1 soluble receptors can be used as a "ligand sink," i.e., antagonist, to bind ligand in vivo or in vitro in therapeutic or other applications where the presence of the ligand is not desired. For example, in cancers that are expressing large amount of bioactive zalphal 1 ligand, zalphal 1 soluble receptors can be used as a direct antagonist of the ligand in vivo, and may aid in reducing progression and symptoms associated with the disease. Moreover, zalphal 1 soluble receptor can be used to slow the progression of cancers that over-express zalkphal 1 receptors, by<br><br>
binding ligand in vivo that would otherwise enhance proliferation of those cancers. Similar in vitro applications for a zalphal 1 soluble receptor can be used, for instance, as a negative selection tp'.select cell lines that grow in the absence of zalphal 1 ligand.<br>
Moreover, zalphal 1 soluble receptor can be used in vivo or in diagnostic applications to detect zalphal 1 ligand-expressing cancers in vivo or in tissue samples. For example, the zalphal 1 soluble receptor can be conjugated to a radio-label or fluorescent label as described herem, and used to detect the presence of the ligand m a tissue sample using an in vitro ligand-receptor type binding assay, or fluorescent imaging assay. Moreover, a radio-labeled zalphal 1 soluble receptor could be administered in vivo to detect ligand-expressing soUd tumors through a radio-imaging method known in the art.<br>
Analysis of the tissue distribution of the mRNA corresponding to this novel DNA showed expression in lymphoid tissues, including thymus, spleen, lymph nodes, and peripheral blood leukocytes. These data indicate a role for the zalphal 1 receptor in proliferation, differentiation, and/or activation of immune cells, and suggest a role in development and reg\ilation of immune responses. The data also suggest that the interaction of zalphal 1 with its ligand may stimulate proliferation and development of myeloid cells and may, like IL-2, IL-6, LIF, IL-11 and OSM (Baumaim et al., I Biol. Chem. 268:8414-8417, 1993), induce acute-phase protein synthesis in hepatocytes.<br>
It is preferred to purify the polypeptides of the present invention to &gt;80% purity, more preferably to &gt;90% purity, even more preferably &gt;95% purity, and particularly preferred is a pharmaceutically pure state, that is greater than 99.9% pure with respect to contaminating macromolecules, particularly other proteins and nucleic acids, and free of infectious and pyrogenic agents. Preferably, a purified polypeptide is substantially free of other polypeptides, particularly other polypeptides of animal origin.<br>
Expressed recombinant zalphal 1 polypeptides (or zalphal 1 chimeric or fusion polypeptides) can be purified using fractionation and/or conventional purification methods and media. Ammonium sulfate precipitation and acid or chaotrope extraction may be used for fractionation of samples. Exemplary purification<br><br>
steps may include hydroxyapatite, size exclusion, FPLC and reverse-phase high performance liquid" chromatography. Suitable chromatographic media include derivatized dextrans, agarose, cellulose, polyacrylamide, specialty silicas, and the like. PEI, DEAE, QAE and Q derivatives are preferred. Exemplary chromatographic media include those media derivatized with phenyl, butyl, or octyl groups, such as Phenyl-Sepharose FF (Pharmacia), Toyopearl butyl 650 (Toso Haas, Montgomeryville, PA), Octyl-Sepharose (Pharmacia) and the like; or polyacrylic resins, such as Amberchrom CG 71 (Toso Haas) and the like. Suitable solid supports include glass beads, silica-based resins, cellulosic resins, agarose beads, cross-linked agarose beads, polystyrene beads, cross-linked polyacrylamide resins and the like that are insoluble under the conditions in which they are to be used. These supports may be modified with reactive groups that allow attachment of proteins by amino groups, carboxyl groups, sulfhydryl groups, hydroxyl groups and/or carbohydrate moieties. Examples of coupling chemistries include cyanogen bromide activation, N-hydroxysuccinimide activation, epoxide activation, sulfhydryl activation, hydrazide activation, and carboxyl and amino derivatives for carbodiimide coupling chemistries. These and other solid media are well known and widely used in the art, and are available from commercial suppliers. Methods for binding receptor polypeptides to support media are well known in the art. Selection of a particular method is a matter of routine design and is determined in part by the properties of the chosen support. See, for example, Affinitv Chromatography: Principles &amp; Methods. Pharmacia LKB Biotechnology, Uppsala, Sweden, 1988.<br>
The polypeptides of the present invention can be isolated by exploitation of their biochemical, structural, and biological properties.- For example, immobilized metal ion adsorption (IMAC) chromatography can be used to purify histidine-rich proteins, including those comprising polyhistidine tags. Briefly, a gel is first charged with divalent metal ions to form a chelate (Sulkowski, Trends in Biochem. 3:1-7, 1985). Histidine-rich proteins will be adsorbed to this matrix with differing affinities, depending upon the metal ion used, and will be eluted by competitive elution, lowering the pH, or use of strong chelating agents. Other methods of purification include purification of glycosylated proteins by lectin affinity chromatography and ion exchange  chromatography (Methods in Enzvmol.  Vol.   182,  "Guide to Protein<br><br>
Purification", M. Deutscher, (ed.), Acad. Press, San Diego, 1990, pp.529-39). Within additional embodiments of the invention, a fusion of the polypeptide of interest and an affinity tag (e.g., maltpse-binding protein, an immunoglobulin domain) may be constructed to facilitate purification.<br>
Moreover, using methods described in the art, polypeptide fusions, or hybrid zalphal 1 proteins, are constructed using regions or domains of the inventive zalphal 1 in combination with those of other human cytokine receptor family proteins, or heterologous proteins (Sambrook et al., ibid.. Altschul et al., ibid.. Picard, Cur. Opin. Biology. 5:511-5, 1994, and references therein). These methods allow the determination of the biological importance of larger domains or regions in a polypeptide of interest. Such hybrids may alter reaction kinetics, binding, constrict or expand the substrate specificity, or alter tissue and cellular locali2ation of a polypeptide, and can be applied to polypeptides of unknown structure.<br>
Fusion poljT)eptides or proteins can be prepared by methods known to those skilled in the art by preparing each component of the fusion protein and chemically conjugating them. Alternatively, a polynucleotide encoding one or more components of the fusion protein in the proper reading frame can be generated using known techniques and expressed by the methods described herein. For example, part or all of a domain(s) conferring a biological fianction may be swapped between zalphal 1 of the present invention with the functionally equivalent domain(s) from another cytokine family member. Such domains include, but are not limited to, the secretory signal sequence, extracellular cytokine binding domain, transmembrane domain, and intracellular signaling domain, Box I and Box II sites, as disclosed herein. Such fusion proteins would be expected to have a biological functional profile that is the same or similar to polypeptides of the present invention or other known family proteins, depending on the fusion constructed. Moreover, such fusion proteins may exhibit other properties as disclosed herein.<br>
Standard molecular biological and cloning techniques can be used to swap the equivalent domains between the zalphal 1 polypeptide and those polypeptides to which they are fused. Generally, a DNA segment that encodes a domain of interest, e.g., a zalphal 1 domain described herein, is operably linked in frame to at least one<br><br>
other DNA segment encoding an additional polypeptide (for instance a domain or region from another cytokine receptor, such as the IL-2 receptor), and inserted into an appropriate expression-ivector, as described herein. Generally DNA constructs are made such that the several^ DNA segments that encode the corresponding regions of a polypeptide are operably linked in fi-ame to make a single construct that encodes the entire fusion protein, or a functional portion thereof. For example, a DNA construct would encode from N-terminus to C-terminus a fusion protein comprising a signal polypeptide followed by a cytokine binding domain, followed by a fransmembrane domain, followed by an intracellular signaling domain. Such fusion proteins can be expressed, isolated, and assayed for activity as described herein.<br>
Zalphall polypeptides or fragments thereof may also be prepared through chemical synthesis, zalphal 1 polypeptides may be monomers or multimers; glycosylated or non-glycosylated; pegylated or non-pegylated; and may or may not include an initial methionine amino acid residue.<br>
Polypeptides of the present mvention can also be synthesized by exclusive solid phase synthesis, partial solid phase methods, fragment condensation or classical solution synthesis. Methods for synthesizing polypeptides are well known in the art. See, for example, Merrifield, J. Am. Chem. Soc. 85:2149, 1963; Kaiser et al.. Anal. Biochem. 34:595, 1970. After the entire synthesis of the desired peptide on a solid support, the peptide-resin is with a reagent which cleaves the polypeptide from the resin and removes most of the side-chain protecting groups. Such methods are well established in the art.<br>
The activity of molecules of the present invention can be measured using a variety of assays that measure cell differentiation and proliferation. Such assays are well known in the art.<br>
Proteins of the present invention are useful for example, in treating<br>
lymphoid, immvme, inflammatory, spleenic, blood or bone disorders, and can be<br>
measured in vitro using cultured cells or in vivo by administering molecules of the<br>
claimed invention to the appropriate animal model. For instance, host cells expressing<br>
a zalphal 1 soluble receptor polypeptide can be embedded in an alginate environment<br>
and    injected    (implanted)    into    recipient    animals.	Alginate-poly-L-lysine<br><br>
microencapsulation, permselective membrane encapsulation and diffusion chambers are a means to entrap transfected mammalian cells or primary mammalian cells. These types of non-immunogenic "encapsulations" permit the diffusion of proteins and other macromolecules secreted or released by the captured cells to the recipient animal. Most importantly, the capsules mask and shield the foreign, embedded cells from the recipient animal's immime response. Such encapsiilations can extend the life of the injected cells from a few hours or days (naked cells) to several weeks (embedded cells). Alginate threads provide a simple and quick means for generating embedded cells.<br>
The materials needed to generate the alginate threads are known in the art.   In an exemplary procedure, 3% alginate is prepared in sterile H2O, and sterile<br>
filtered.   Just prior to preparation of alginate threads, the alginate solution is again<br>
filtered. An approximately 50% cell suspension (containing about 5 x 10^ to about 5 x<br>
10^ cells/ml) is mixed with the 3% alginate solution. One ml of the alginate/cell suspension is extruded into a 100 mM sterile filtered CaCl2 solution over a time period<br>
of ~15 min, forming a "thread". The extruded thread is then transferred into a solution of 50 mM CaCl2, and then into a solution of 25 mM CaCl2. The thread is then rinsed<br>
with deionized water before coating the thread by incubating in a 0.01% solution of poly-L-lysine. Finally, the thread is rinsed with Lactated Ringer's Solution and drawn from solution into a syringe barrel (v^dthout needle). A large bore needle is then attached to the syringe, and the thread is intraperitoneally injected into a recipient in a minimal volume of the Lactated Ringer's Solution.<br>
An in vivo approach for assaying proteins of the present invention involves viral delivery systems. Exemplary viruses for this purpose include adenovirus, herpesvirus, retroviruses, vaccinia virus, and adeno-associated virus (AAV). • Adenovirus, a double-stranded DNA virus, is ciirrently the best studied gene transfer vector for delivery of heterologous nucleic acid (for review, see T.C. Becker et al., Meth. Cell Biol. 43:161-89, 1994; and J.T. Douglas and D.T. Curiel, Science &amp; Medicine 4:44-53, 1997). The adenovirus system offers several advantages: (i) adenovirus can accommodate relatively large DNA inserts; (ii) can be grown to high-titer; (iii) infect a broad range of mammalian cell types; and (iv) can be used with a<br><br>
large number of different promoters including ubiquitous, tissue specij5c, and regulatable promoters. Also, because adenoviruses are stable in the bloodstream, they can be administered by.u3travenous injection.<br>
Using adenovirus vectors where portions of the adenovirus genome are deleted, inserts are incorporated into the viral DNA by direct ligation or by homologous recombination with a co-transfected plasmid. In an exemplary system, the essential El gene has been deleted from the viral vector, and the virus will not replicate unless the El gene is provided by the host cell (the human 293 cell line is exemplary). When intravenously administered to intact animals, adenovirus primarily targets the liver. If the adenoviral delivery system has an El gene deletion, the virus cannot repUcate in the host cells. However, the host's tissue (e.g., liver) will express and process (and, if a secretory signal sequence is present, secrete) the heterologous protein. Secreted proteins will enter the circulation in the highly vascularized liver, and effects on the infected animal can be determined.<br>
Moreover, adenoviral vectors containing various deletions of viral genes can be used in an attempt to reduce or eliminate immune responses to the vector. Such adenoviruses are El deleted, and in addition contain deletions of E2A or E4 (Lusky, M. et al., J. Virol. 72:2022-2032, 1998; Raper, S.E. et al.. Human Gene Therapv 9:671-679, 1998). In addition, deletion of E2b is reported to reduce immune responses (Amalfitano, A. et al., J. Virol. 72:926-933, 1998). Moreover, by deleting the entire adenovirus genome, very large inserts of heterologous DNA can be accommodated. Generation of so called "gutless" adenoviruses where all viral genes are deleted are particularly advantageous for insertion of large inserts of heterologous DNA. Foi review, see Yeh, P. and Perricaudet, M., FASEB J. 11:615-623,1997.<br>
The adenovirus system can also be used for protein production in vitro By culturing adenovirus-infected non-293 cells under conditions where the cells are no rapidly dividing, the cells can produce proteins for extended periods of time. For instance, BHK cells are grown to confluence in cell factories, then exposed to the adenoviral vector encoding the secreted protein of interest. The cells are then grown under serum-free conditions, which allows infected cells ta survive for several weeks without significant cell division.   Alternatively, adenovirus vector infected 293 cells<br><br>
can be grown as adherent cells or in suspension culture at relatively high cell density to produce significant amounts of protein (See Gamier et al., Cvtotechnol. 15:145-55. 1994). With either'.protocol, an expressed, secreted heterologous protein can be repeatedly isolated .ffom the cell culture supernatant, lysate, or membrane fractions depending on the disposition of the expressed protein in the cell. Within the infected 293 cell production protocol, non-secreted proteins may also be effectively obtained.<br>
In view of the tissue distribution observed for zalphall, agonists (including the natural ligand/ substrate/ cofactor/ etc.) and antagonists have enormous potential in both in vitro and in vivo appUcations. Compounds identified as zalphal 1 agonists are useful for stimulating growth of immune and hematopoietic cells in vitro and in vivo. For example, zalphal 1 and agonist compounds are useful as components of defined cell culture media, and may be used alone or in combination with other cytokines and hormones to replace serum that is commonly used in cell culture. Agonists are thus useful in specifically promoting the growth and/or development of T-cells, B-cells, and other cells of the lymphoid and myeloid lineages in culture. Moreover, zalphal 1 soluble receptor, agonist, or antagonist may be used in vitro in an assay to measure stimulation of colony formation from isolated primary bone marrow cultures. Such assays are well known in the art.<br>
Antagonists are also useful as research reagents for characterizing sites of ligand-receptor interaction. Inhibitors of zalphal 1 activity (zalphal 1 antagonists) include anti-zalphal 1 antibodies and soluble zalphall receptors, as well as other peptidic and non-peptidic agents (including ribozymes).<br>
Zalphal 1 can also be used to identify modulators (e.g, antagonists) of its activity. Test compounds are added to the assays disclosed herein to identify compoimds that inhibit the activity of zalphall. In addition to those assays disclosed herein, samples can be tested for inhibition of zalphal 1 activity within a variety of assays designed to measure zalphall binding, oligomerization, or the stimulation/uihibition of zalphall-dependent cellular responses. For example, zalphall-expressing cell lines can be transfected with a reporter gene construct that is responsive to a zalphal 1-stimulated cellular pathway. Reporter gene constructs of this type are known in the art, and will generally comprise a zalphal 1-DNA response<br><br>
element operably linked to a gene encoding an assay detectable protein, such as luciferase. DNA response elements can include, but are not limited to, cyclic AMP response elements (CRE), hormone response elements (HRE) insulin response element (IRE) (Nasrin et al., Pbc. Natl. Acad. Sci. USA 87:5273-7, 1990) and serum response elements (SRE) (Shaw et al. CeU 56: 563-72, 1989). Cyclic AMP response elements are reviewed m Roestler et al., J. Biol. Chem. 263 (19):9063-6; 1988 and Habener, Molec. Endocrinol. 4 (8):1087-94; 1990. Hormone response elements are reviewed in Beato, Cell 56:335-44; 1989. Candidate compounds, solutions, mixtures or extracts or conditioned media from various cell types are tested for the ability to enhance the activity of zalphal 1 receptor as evidenced by a increase in zalphal 1 stimulation of reporter gene expression. Assays of this type will detect compounds that directly stimulate zalphal 1 signal transduction activity through binding the receptor or by otherwise stimulating part of the signal cascade. As such, there is provided a method of identifying agonists of zalphal 1 polypeptide, comprising providing cells responsive to a zalphal 1 polypeptide, cxilturing a first portion of the cells in the absence of a test compovmd, culturing a second portion of the cells in the presence of a test compound, and detecting a increase in a cellular response of the second portion of the cells as compared to the first portion of the cells. Moreover third cell, containing the reporter gene construct described above, but not expressing zaplphal 1 receptor, can be used as a control cell to assess non-specific, or non-zalpha 11-mediated, stirhulation of the reporter. Agonists, including the natural ligand, are therefore useful to stimulate or increase zalphal 1 polypeptide function.<br>
A zalphal 1 ligand-binding polypeptide, such as the cytokine binding domain disclosed herein, can also be used for purification of ligand. The polypeptide is immobilized on a solid support, such as beads of agarose, cross-linked agarose, glass, cellulosic resins, silica-based resins, polystyrene, cross-linked polyacrylamide, or like materials that are stable under the conditions of use. Methods for linking polypeptides to solid supports are known m the art, and include amine chemistry, cyanogen bromide activation, N-hydroxysuccinimide activation, epoxide activation, sulfhydryl activation, and hydrazide activation. The resulting medium will generally be configured in the form of a column, and fluids containing ligand are passed through the column one or<br><br>
more times to allow ligand to bind to the receptor polypeptide. The ligand is then eluted using changes in salt concentration, chaotropic agents (guanidine HCl), or pH to disrupt ligand-receptpir binding.<br>
An assJay system that uses a ligand-binding receptor (or an antibody, one member of a complement/ anti-complement pair) or a binding fragment thereof, and a commercially available biosensor instrument may be advantageously employed (e.g., BIAcore'^", Pharmacia Biosensor, Piscataway, NJ; or SELDI™ technology, Ciphergen, Inc., Palo Alto, CA). Such receptor, antibody, member of a complement/anti-complement pair or fragment is immobilized onto the surface of a receptor chip. Use of this instrument is disclosed by Karlsson, J. Immvmol. Methods 145:229-240, 1991 and Cuimingham and Wells. J. Mol. Biol. 234:554-63. 1993. A receptor, antibody, member or fragment is covalently attached, using amine or sulfhydryl chemistry, to dextran fibers that are attached to gold film within the flow cell. A test sample is passed through the cell. If a ligand, epitope, or opposite member of the complement/anti-complement pair is present in the sample, it will bind to the immobilized receptor, antibody or member, respectively, causing a change in the refractive index of the medium, which is detected as a change in surface plasmon resonance of the gold film. This system allows the determination of on- and off-rates, from which binding affinity can be calculated, and assessment of stoichiometry of binding.<br>
Ligand-binding receptor polypeptides can also be used within other assay systems known in the art. Such systems include Scatchard analysis for determination of binding affinity (see Scatchard, Ann. NY Acad. Sci. 51: 660-672, 1949) and calorimetric assays (Cunningham et al.. Science 253:545-48, 1991; Cunningham et al.. Science 245:821-25.1991).<br>
v/Zalphal 1 polypeptides can also be used to prepare antibodies that bind to zalphal 1 epitopes, peptides or polypeptides. The zalphal 1 polypeptide or a fragment thereof serves as an antigen (immunogen) to inoculate an animal and elicit an immune response. One of skill in the art would recognize that antigens or immunogenic epitopes can consist of stretches of amino acids within a longer polypeptide, from about 10 amino acids and up to about the entire length of the polypeptide or longer depending on the polypeptide.  Suitable antigens include the zalphal 1 polypeptide encoded by SEQ<br><br>
ID N0:2 from amino acid number 20 (Cys) to amino acid number 538 (Ser), or a contiguous 9 to 519'AA amino acid fragment thereof. Preferred peptides to use as antigens are the cytokine; binding domain, intracellular signaling domain, Box I and Box II sites, disclosed herein, and zalphall hydrophilic peptides such as those predicted by one of skill in the art from a hydrophobicity plot, determined for example, from a Hopp/Woods hydrophilicity profile based on a sliding six-residue window, with buried G, S, and T residues and exposed H, Y, and W residues ignored (See, Figure 1). Zalphall hydrophilic peptides include peptides comprising amino acid sequences selected from the group consisting of: (1) amino acid number 51 (Trp) to amino acid number 61 (Glu) of SEQ ID N0:2; (2) amino acid number 136 (He) to amino acid number 143 (Glu) of SEQ ID N0:2; (3) amino acid number 187 (Pro) to amino acid number 195 (Ser) of SEQ ID N0:2; (4) amino acid number 223 (Phe) to amino acid number 232 (Glu) of SEQ ID N0:2; and (5) amino acid number 360 (Glu) to aming^ acid number 368 (Asp) of SEQ ID N0:2. In addition, conserved motifs, and variable regions between conserved motifs of zalphall are suitable antigens. Moreover, corresponding regions of the mouse zalphall polypeptide (SEQ ID NO: 85) can be used to^ generatejygdJbQJiei^gainsL.thsjnojytse „zalphal 1 ( Antibodies generated from this immime response can be isolated and purified as described herein, \lethods for preparing and isolating polyclonal and monoclonal antibodies are well known in the art. See, for example, Ciirrent Protocols in Immunology. Cooligan, et al. (eds.). National Institutes of Health, John Wiley and Sons, Inc., 1995; Sambrook et al., Molecular Cloning: A Laboratory Manual. Second Edition. Cold Spring Harbor, NY, 1989; and Hurrell, J. G. R., Ed., Monoclonal Hvbridoma Antibodies: Techniques and Applications. CRC Press, Inc., Boca Raton, FL, 1982.<br>
As would be evident to one of ordinary skill in the art, polyclonal antibodies can be generated from inoculating a variety of warm-blooded animals such as horses, cows, goats, sheep, dogs, chickens, rabbits, mice, and rats with a zalphal 1 polypeptide or a fragment thereof. The immunogenicity of a zalphal 1 polypeptide may be increased through the use of an adjuvant, such as alum (aluminum hydroxide) or Frexmd's complete or incomplete adjuvant. Polypeptides useful for immunization also include fiision polypeptides, such as ftisions of zalphal 1 or a portion thereof with an<br><br>
immimoglobulin polypeptide or with maltose binding protein. The polypeptide immunogen may be a fiall-length molecule or. a portion thereof. If the polypeptide portion is "hapten-lilce"^ such portion may be advantageously joined or linked to a macromolecular carrier (such as keyhole limpet hemocyanin (KLH), bovine serum albumin (BSA) or tetanus toxoid) for immunization.<br>
As used herein, the term "antibodies" includes polyclonal antibodies, affinity-purified polyclonal antibodies, monoclonal antibodies, and antigen-binding fragments, such as F(ab')2 and Fab proteolytic fragments. Genetically engineered intact<br>
antibodies or fragments, such as chimeric antibodies, Fv fragments, single chain antibodies and the like, as well as synthetic antigen-binding peptides and polypeptides, are also included. Non-human antibodies may be humanized by grafting non-human CDRs onto human framework and constant regions, or by incorporating the entire non-human variable domains (optionally "cloaking" them with a human-like surface by replacement of exposed residues, wherein the result is a "veneered" antibody). In some instances, humanized antibodies may retain non-human residues within the hviman variable region framework domains to enhance proper binding characteristics. Through hxmianizing antibodies, biological half-life may be increased, and the potential for adverse immime reactions upon administration to humans is reduced.<br>
Alternative techniques for generating or selecting antibodies useftil herein include in vitro exposure of lymphocytes to zalphal 1 protein or peptide, and selection of antibody display libraries in phage or similar vectors (for instance, through use of immobilized or labeled zalphall protein or peptide). Genes encoding polypeptides having potential zalphal 1 polypeptide binding domains can be obtained by screening random peptide libraries displayed on phage (phage display) or on bacteria, such as E. coli. Nucleotide sequences encoding the polypeptides can be obtained in a number of ways, such as through random mutagenesis and random polynucleotide synthesis. These random peptide display libraries can be used to screen for peptides which interact with a known target which can be a protein or polypeptide, such as a ligand or receptor, a biological or synthetic macromolecule, or organic or inorganic substances. Techniques for creating and screening such random peptide display libraries are known in the art (Ladner et al., US Patent NO. 5,223,409; Ladner<br><br>
et al., US Patent NO. 4,946,778; Ladner et al., US Patent NO. 5,403,484 and Ladner et al., US Patent NO. '5,571,698) and random peptide display libraries and kits for screening such libraries are available commercially, for instance from Clontech (Palo Alto, CA), Invitrogen'inc. (San Diego, CA), New England Biolabs, Inc. (Beverly, MA) and Pharmacia LKB Biotechnology Inc. (Piscataway, NJ). Random peptide display libraries can be screened using the zalphall sequences disclosed herein to identify proteins which bind to zalphall. These "binding peptides" which interact with zalphal 1 polypeptides can be used for tagging cells; for isolating homolog polypeptides by affinity purification; they can be durectly or indirectly conjugated to drugs, toxins, radionuclides and the like. These binding peptides can also be used in analytical methods such as for screening expression libraries and neutralizing activity. The binding peptides can also be used for diagnostic assays for determining circulating levels of zalphall polypeptides; for detecting or quantitating soluble zalphall polypeptides as marker of underlying pathology or disease. These binding peptides can also act as zalphall "antagonists" to block zalphal 1 binding and signal transduction in vitro and in vivo. These anti-zalphal 1 binding peptides would be useful for inhibiting the action of a ligand that binds with zalphal 1.<br>
Antibodies are determined to be specifically binding if: 1) they exhibit a threshold level of binding activity, and/or 2) they do not significantly cross-react with related polypeptide molecules. First, antibodies herein specifically bind if they bind if they bind to a zalphal 1 polypeptide, peptide or epitope with an affinity at least 10-fold greater than the binding affinity to control (non-zalphall) polypeptide. It is prefened<br><i></i>
that the antibodies exhibit a binding affinity (Ka) of 10   M"   or greater, preferably 10<br>
-1	8-1	9    -1<br>
M'   or greater, more preferably 10   M    or greater, and most preferably 10   M    or<br>
greater.   The binding affinity of an antibody can be readily determined by one of<br>
ordinary skill in the art, for example, by Scatchard analysis (Scatchard, G., Ann. NY<br>
Acad. Sci. 51: 660-672,1949).<br>
Second, antibodies are determined to specifically bind if they do not<br>
significantly cross-react with related polypeptides. Antibodies do not significantly<br>
cross-react with related polypeptide molecules, for example, if they detect zalphall but<br><br>
not known related polypeptides using a standard Western blot analysis (Ausubel et al., ibid.'). Examples of known related polypeptides are orthologs, proteins from the same species that are menibers of a protein family (e.g. IL-6), zalphall polypeptides, and non-human zalphall.'^Moreover, antibodies may be "screened against" known related polypeptides to isolate a population that specifically binds to the inventive polypeptides. For example, antibodies raised to zalphall are adsorbed to related polypeptides adhered to insoluble matrix; antibodies specific to zalphall will flow through the matrix under the proper buffer conditions. Such screening allows isolation of polyclonal and monoclonal antibodies non-crossreactive to closely related polypeptides (Antibodies: A Laboratorv Manual. Harlow and Lane (eds.). Cold Spring Harbor Laboratory Press, 1988; Current Protocols in Immunology. Cooligan, et al. (eds.). National Institutes of Health, John Wiley and Sons, Inc., 1995). Screening and isolation of specific antibodies is well known in the art. See, Fundamental Immunology, Paul (eds.). Raven Press, 1993; Getzoff et al.. Adv. in Immunol. 43: 1-98, 1988; Monoclonal Antibodies: Principles and Practice. Goding, J.W. (eds.), Academic Press Ltd.. 1996; Benjamin et al.. Ann. Rev. Immunol. 2: 67-101, 1984.<br>
A variety of assays known to those skilled in the art can be utilized to detect antibodies which specifically bind to zalphal 1 proteins or peptides. Exemplary assays are described in detail in Antibodies: A Laboratorv Manual. Harlow and Lane (Eds.), Cold Spring Harbor Laboratory Press, 1988. Representative examples of such assays include: concurrent Immunoelectrophoresis, radioimmunoassay, radioimmimo-precipitation, enzyme-linked immunosorbent assay (ELISA), dot blot or Western blot assay, inhibition or competition assay, and sandwich assay. In addition, antibodies can be screened for binding to wild-type versus mutant zalphal 1 protein or polypeptide.<br>
Antibodies to zalphall may be used for tagging cells that express zalphall; for isolating zalphall by affinity purification; for diagnostic assays for determining circulating levels of zalphal 1 polypeptides; for detecting or quantitating soluble zalphal 1 as marker of underlying pathology or disease; in analytical methods employmg FACS; for screening expression libraries; for generating anti-idiotypic antibodies; and as neutralizmg antibodies or as antagonists to block zalphal 1 activity in vitro and in vivo.    Suitable direct tags or labels include radionuclides, enzymes.<br><br>
substrates, cofactors, inhibitors, fluorescent markers, chemiluminescent markers, magnetic particles and the like; indirect tags or-labels may feature use of biotin-avidin or other complement/emti-complement pairs as intermediates. Antibodies herein may also be directly or indirectly conjugated to drugs, toxins, radionuclides and the like, and these conjugates used for in vivo diagnostic or therapeutic applications. Moreover, antibodies to zalphal 1 or fragments thereof may be used in vitro to detect denatured zalphal 1 or fragments thereof m assays, for example. Western Blots or other assays known in the art.<br>
Antibodies to zalphall are usefial for tagging cells that express the receptor and assaying Zalphall expression levels, for affinity purification, within diagnostic assays for determining circulating levels of soluble receptor polypeptides, analytical methods employing fluorescence-activated cell sorting. Divalent antibodies may be used as agonists to mimic the effect of the zalphal 1 ligand.<br>
Antibodies herein can also be directly or indirectly conjugated to drugs, toxins, radionuclides and the like, and these conjugates used for in vivo diagnostic or therapeutic applications. For instance, antibodies or binding polypeptides which recognize zalphal 1 of the present invention can be used to identify or treat tissues or organs that express a corresponding anti-complementary molecule (i.e., a zalphall receptor). More specifically, anti-zalphal 1 antibodies, or bioactive fragments or portions thereof, can be coupled to detectable or cytotoxic molecules and delivered to a mammal having cells, tissues or organs that express the zalphal 1 molecvde.<br>
Suitable detectable molecules may be directly or indirectly attached to polypeptides that bind zalphall ("binding polypeptides," including binding peptides disclosed above), antibodies, or bioactive fragments or portions thereof Suitable detectable molecules include radionuclides, enzymes, substrates, cofactors, inhibitors, fluorescent markers, chemiluminescent markers, magnetic particles and the like. Suitable cytotoxic molecules may be directly or indirectly attached to the polypeptide or antibody, and include bacterial or plant toxins (for instance, diphtheria toxin, Pseudomonas exotoxin, ricin, abrin and the like), as well as therapeutic radionuclides, such as iodine-131, rhenium-188 or yttrium-90 (either directly attached to the polypeptide or antibody, or indirectly attached through means of a chelating moiety, for<br><br>
instance). Binding polypeptides or antibodies may also be conjugated to cytotoxic drugs, such as adriamycin. For indirect attachment of a detectable or cytotoxic molecule, the detectable .or cytotoxic molecule can be conjugated with a member of a complementary/ anticomplementary pair, where the other member is bound to the binding polypeptide or antibody portion. For these purposes, biotin/streptavidin is an exemplary complementary/ anticomplementary pair.<br>
In another embodiment, binding polypeptide-toxin fusion proteins or antibody-toxin fusion proteins can be used for targeted cell or tissue inhibition or ablation (for instance, to treat cancer cells or tissues). Alternatively, if the binding polypeptide has multiple fimctional domains (i.e., an activation domain or a ligand binding domain, plus a targeting domain), a fusion protein including only the targeting domain may be suitable for directing a detectable molecule, a cytotoxic molecule or a complementary molecule to a cell or tissue type of interest. In instances where the fusion protein including only a single domain includes a complementary molecule, the anti-complementary molecule can be conjugated to a detectable or cytotoxic molecule. Such domain-complementary molecule fusion proteins thus represent a generic targeting vehicle for cell/tissue-specific delivery of generic anti-complementary-detectable/ cytotoxic molecule conjugates.<br>
In another embodiment, zalphall binding polypeptide-cytokine or antibody-cytokine fusion proteins can be used for enhancing in vivo killing of target tissues (for example, blood, lymphoid, colon, and bone marrow cancers), if the binding polypeptide-cytokine or anti-zalphal 1 antibody targets the hyperproliferative cell (See, generally, Homick et al., Blood 89:4437-47, 1997). They described fusion proteins enable targeting of a cytokine to a desired site of action, thereby providing an elevated local concentration of cytokine. Suitable anti-zalphal 1 antibodies target an undesirable cell or tissue (i.e., a tumor or a leukemia), and the fused cytokine mediates unproved target cell lysis by effector cells. Suitable cytokines for this purpose include interleukin 2 and granulocyte-macrophage colony-stimulating factor (GM-CSF), for instance.<br>
Alternatively, zalphal 1 binding polypeptide or antibody fusion proteins described herein can be used for enhancing in vivo killing of target tissues by directly<br><br>
stimulating a zalphall-modulated apoptotic pathway, resxolting in cell death of hyperproliferative cells expressing zalphal 1.    .<br>
The bfqactive binding polypeptide or antibody conjugates described herein can be delivered orally, intravenously, intraarterially or intraductally, or may be introduced locally at the intended site of action.<br>
Four-helix bundle cytokines that bind to cytokine receptors as well as other proteins produced by activated lymphocytes play an important biological role in cell differentiation, activation, recruitment and homeostasis of cells throughout the body. Therapeutic utility includes treatment of diseases which require immune regulation including autoimmune diseases, such as, rheumatoid arthritis, multiple sclerosis, myasthenia gravis, systemic lupus erythomatosis and diabetes. Zalphall antagonists or agonists, including soluble receptors and the natural hgand, may be important in the regulation of inflammation, and therefore would be useful in treating rheumatoid arthritis, asthma, ulcerative colitis, inflammatory bowel disease, Crohn's disease, and sepsis. There may be a role of zalphal 1 antagonists or agonists, including soluble receptors and the natural ligand, in mediating tumorgenesis, and therefore would be useful in the treatment of cancer. Zalphal 1 antagonists or agonists, including soluble receptors and the natural ligand, may be a potential therapeutic in suppressing the immune system which would be important for reducing graft rejection. Zalphal 1 Ligand may have usefiihiess in prevention of graft vs. host disease.<br>
Alternatively, zalphall antagonists or agonists, including soluble receptors and the natural ligand may activate the immune system which would be important in boosting immunity to infectious diseases, treating immunocompromised patients, such as HIV+ patient, or in improving vaccines. In particular, zalphall antagonists or agonists, including soluble receptors and the natural ligand can modulate, stimulate or expand NK cells, or their progenitors, and would provide therapeutic value in treatment of viral infection, and as an anti-neoplastic factor. NK cells are thought to play a major role in elimination of metastatic tumor cells and patients with both metastases and solid tumors have decreased levels of NK cell activity (Whiteside et. al., Curr. TOD. Microbiol. Immunol. 230:221-244, 1998).<br><br>
r<br>
Polynucleotides encoding zalphal 1 polypeptides are useful within gene therapy applications'where it is desired to increase or inhibit 2alphall activity. If a mammal has a mutated or absent zalphal 1 gene, the 2alphal 1 gene can be introduced into the cells of the^mammal. In one embodiment, a gene encoding a zalphall polypeptide is introduced in vivo in a viral vector. Such vectors include an attenxiated or defective DNA virus, such as, but not limited to, herpes simplex virus (HSV), papillomavirus, Epstein Barr virus (EBV), adenovirus, adeno-associated virus (AAV), and the like. Defective viruses, which entirely or almost entirely lack viral genes, are preferred. A defective virus is not infective after introduction into a cell. Use of defective viral vectors allows for administration to cells in a specific-, localized area, without concern that the vector can infect other cells. Examples of particular vectors include, but are not limited to, a defective herpes simplex virus 1 (HSVl) vector (Kaplitt et al., Molec. Cell. Neurosci. 2:320-30, 1991); an attenuated adenovirus vector, such as the vector described by Stratford-Perricaudet et al., J. Clin. Invest. 90:626-30, 1992; and a defective adeno-associated virus vector (Samulski et al., J. Virol. 61:3096-101,1987; Samulski et al., J. Virol. 63:3822-8,1989).<br>
In another embodiment, a zalphall gene can be introduced in a retroviral vector, e.g., as described in Anderson et al., U.S. Patent No. 5,399,346; Maim et al. Cell 33:153, 1983; Temin et al., U.S. Patent No. 4,650,764; Temin et al., U.S. Patent No. 4,980,289; Markowitz et al., J. Virol. 62:1120, 1988; Temm et al., U.S. Patent No. 5,124,263; International Patent Publication No. WO 95/07358, published March 16,1995 by Dougherty et al.; and Kuo et al.. Blood 82:845,1993. Alternatively, the vector can be introduced by lipofection in vivo using Hposomes. Synthetic cationic lipids can be used to prepare liposomes for in vivo transfection of a gene encoding a marker (Feigner et al., Proc. Natl. Acad. Sci. USA 84:7413-7, 1987; Mackey et al., Proc. Natl. Acad. Sci. USA 85:8027-31, 1988). The use of lipofection to introduce exogenous genes into specific organs in vivo has certain practical advantages. Molecular targeting of liposomes to specific cells represents one area of benefit. More particularly, directing transfection to particular cells represents one area of benefit. For instance, directing transfection to particular cell types would be particularly advantageous in a tissue with cellular heterogeneity, such as the pancreas, liver, kidney,<br><br>
and brain. Lipids may be chenaically coupled to other molecules for the purpose of targeting. Targeted peptides (e.g., hormones .or neurotransmitters), proteins such as antibodies, or non-pep'tide molecules can be coupled to liposomes chemically.<br>
It is possible to remove the target cells from the body; to introduce the vector as a naked DNA plasmid; and then to re-implant the transformed cells into the body. Naked DNA vectors for gene therapy can be introduced into the desired host cells by methods known m the art, e.g., transfection, electroporation, microinjection, transduction, cell fusion, DEAE dextran, calcium phosphate precipitation, use of a gene gan or use of a DNA vector transporter. See, e.g., Wu et al., J. Biol. Chem. 267:963-7. 1992: Wu etal.. J. Biol. Chem. 263:14621-4. 1988.<br>
Antisense methodology can be used to mhibit zalphall gene transcription, such as to inhibit cell proliferation in vivo. Polynucleotides that are complementary to a segment of a zalphall-encoding polynucleotide (e.g., a polynucleotide as set froth in SEQ ID N0:1) are designed to bind to zalphal 1-encoding mRNA and to inhibit translation of such mRNA. Such antisense polynucleotides are used to inhibit expression of zalphal 1 polypeptide-encoding genes in cell culture or in a subject.<br>
In addition, as a cell surface molecule, zalphal 1 polypeptide can be used as a target to introduce gene therapy into a cell. This application would be particularly appropriate for introducing therapeutic genes into cells in which zalphal 1 is normally expressed, such as lymphoid tissue and PBLs, or cancer cells which express zalphal 1 polypeptide. For example, viral gene therapy, such as described above, can be targeted to specific cell types in which express a cellular receptor, such as zalphal 1 polypeptide, rather tiian the viral receptor. Antibodies, or other molecules that recognize zalphal 1 molecules on the target cell's surface can be used to direct the virus to infect and administer gene therapeutic material to that target cell. See, Woo, S.L.C, Nature Biotech. 14:1538, 1996; Wickham, T.J. et al, Nature Biotech. 14:1570-1573, 1996; Douglas, J.T et al.. Nature Biotech. 14:1574-1578, 1996; Rihova, B., Crit. Rev. Biotechnol. 17:149-169, 1997; and Vile, R.G. et al., Mol. Med. Todav 4:84-92, 1998. For example, a bispecific antibody containing a virus-neutralizing Fab fragment coupled to a zalphall-specific antibody can be used to direct the virus to cells<br><br>
expressing the zalphal 1 receptor and allow efficient entry of the virus containing a genetic element into'the cells. See, for example, Wickham, T.J., et al., J. Virol. 71:7663-7669,1997;.and Wickham, T.J., etal., J. Virol. 70:6831-6838. 1996.<br>
The present invention also provides reagents which will find use in diagnostic applications. For example, the 2alphal 1 gene, a probe comprising zalphal 1 DNA or RNA or a subsequence thereof can be used to determine if the zalphal 1 gene is present on chromosome 16 or if a mutation has occurred. Zalphal 1 is located at the 16pll.l region of chromosome 16 (See, Example 3). Detectable chromosomal aberrations at the zalphal 1 gene locus include, but are not limited to, aneuploidy, gene copy number changes, insertions, deletions, restriction site changes and rearrangements. Such aberrations can be detected using polynucleotides of the present invention by employing molecular genetic techniques, such as restriction fi:agment length polymorphism (RFLP) analysis, fluorescence in situ hybridization methods, short tandem repeat (STR) analysis employing PCR techniques, and other genetic linkage analysis techniques known in the art (Sambrook et al., ibid.: Ausubel et. al., ibid: Marian. Chest 108:255-65.1995).<br>
The precise knowledge of a gene's position can be useful for a number of purposes, including: 1) determining if a sequence is part of an existing contig and obtaining additional siirrounding genetic sequences in various forms, such as YACs, BACs or cDNA clones; 2) providing a possible candidate gene for an inheritable disease which shows linkage to the same chromosomal region; and 3) cross-referencing model organisms, such as mouse, which inay aid in determining what function a particular gene might have.<br>
The zalphal 1 gene is located at the 16pll.l region of chromosome 16. Several genes of known function map to this region. For example, the interleukin 4 (IL-4) cytokine receptor alpha-subunit, a member of the hematopoietin receptor family, maps to 16p 12.1-pi 1.2. This subunit may form a heterodimer with zalphal 1. Moreover, zalphal 1 polynucleotide probes can be used to detect abnormalities or genotypes associated with defects in IL-4 receptor, such as those that are implicated in some allergic inflammatory disorders and asthma (Deichman, K.A. et al., Exp. Allergv 28:151-155; 1998; Mitsuyasu, H. et al.. Nature Genet. 19:119-120, 1998). In addition,<br><br>
zalphal 1 polynucleotide -probes can be used to detect abnonnalities or genotypes associated with inflammatory bowel disease, where a susceptibility marker maps to 16pl2-ql3 (Cho, J.H; et al, Proc. Nat. Acad. Sci. 95:7502-7507, 1998). Further, zalphal 1 polynucleotide probes can be used to detect abnormalities or genotypes associated with hemoglobin loci located at 16pter-pl3.3, and particularly hemoglobin-alpha defects associated with alpha-thalassemia syndromes, such as hydrops fetalis (for review, see Chui, M.P., and Waye, J.S. Blood 91:2213-2222, 1998). Moreover, amongst other genetic loci, those for Wihns tumor, type III (16q), Rubenstein-Taybi syndrome (16pl3.3), severe infantile polycystic kidney disease (16pl3.3), all manifest themselves in human disease states as well as map to this region of the human genome. See the Online Mendellian Inheritance of Man (OMIM) gene map, and references therein, for this region of chromosome 16 on a publicly available WWW server (http://www3 .ncbi.nlm.nih.gov/htbin-post/Omim/getmap?chromosome=l 6p 11.1). All of these serve as possible candidate genes for an inheritable disease which show linkage to the same chromosomal region as the zalphal 1 gene.<br>
Similarly, defects in the zalphal 1 locus itself may result in a heritable human disease state. Molecules of the present invention, such as the polypeptides, antagonists, agonists, polynucleotides and antibodies of the present invention would aid in the detection, diagnosis prevention, and treatment associated with a zalphal 1 genetic defect.<br>
Mice engineered to express the zalphal 1 gene, referred to as "transgenic mice," and mice that exhibit a complete absence of zalphal 1 gene function, referred to as "knockout mice," may also be generated (Snouwaert et al.. Science 257:1083, 1992; Lowell et al., Nature 366:740-42, 1993; Capecchi, M.R., Science 244: 1288-1292, 1989; Palmiter, R.D. et al. Annu Rev Genet. 20: 465-499, 198^). For example, transgenic mice that over-express zalphal 1, either ubiquitously or under a tissue-specific or tissue-restricted promoter can be used to ask whether over-expression causes a phenotype. For example, over-expression of a wild-type zalphal 1 polypeptide, polypeptide fragment or a mutant thereof may alter normal cellular processes, resulting in a phenotype that identifies a tissue in which zalphal 1 expression is functionally relevant and may indicate a therapeutic target for the zalphal 1, its agonists or<br><br>
antagonists. For example, a preferred transgenic mouse to engineer is one that expresses a "domiiiant-negative" phenotype, such as one that over-expresses the zalphal 1 extracellular cytokine binding domain with the transmembrane domain attached (approximately amino acids 20 (Cys) to 255 (Leu) of SEQ ID N0:2). Moreover, such over-expression may result in a phenotype that shows similarity with human diseases. Similarly, knockout zalphal 1 mice can be used to determine where zalphal 1 is absolutely required in vivo. The phenotype of knockout mice is predictive of the in vivo effects of that a zalphal 1 antagonist, such as those described herein, may have. The mouse or the human zalphal 1 cDNA can be used to isolate murine zalphal 1 mRNA, cDNA and genomic DNA, which are subsequently used to generate knockout mice. These mice may be employed to study the zalphal 1 gene and the protein encoded thereby in an in vivo system, and can be used as in vivo models for corresponding human diseases. Moreover, transgenic mice expression of zalphal 1 antisense polynucleotides or ribozymes directed against zalphal 1, described herein, can be used analogously to transgenic mice described above.<br>
For pharmaceutical use, the soluble receptor polypeptides of the present invention are formulated for parenteral, particularly intravenous or subcutaneous, delivery according to conventional methods. Intravenous administration will be by bolus injection or infusion over a typical period of one to several hours. In general, pharmaceutical formulations will include a zalphal 1 soluble receptor polypeptide in combination with a pharmaceutically acceptable vehicle, such as saline, biiffered saline, 5% dextrose in water or the like. Formulations may further include one or more excipients, preservatives, solubilizers, buffering agents, albumin to prevent protein loss on vial surfaces, etc. Methods of formulation are well known in the art and are disclosed, for example, in Remington: The Science and Practice of Pharmacy, Gemiaro, ed., Mack Publishing Co., Easton, PA, 19th ed., 1995. Therapeutic doses will generally be in the range of 0.1 to 100 |ig/kg of patient weight per day, preferably 0.5-20 mg/kg per day, with the exact dose determined by the clinician according to accepted standards, taking into account the nature and severity of the condition to be treated, patient traits, etc. Determination of dose is within the level of ordinary skill in the art. The proteins may be administered for acute treatment, over one week or less, often over<br><br>
a period of one to three days or may be used in chronic treatment, over several months or years. In general,' a therapeutically effective amount of zalphal 1 soluble receptor polypeptide is an amount sufficient to produce a clinically significant effect.<br>
The invention is further illxistrated by the following non-limiting examples.<br>
EXAMPLES<br>
Example 1<br>
Identification of Human zalphal 1<br>
Using an EST Sequence to Obtain Full-length zalphal 1<br>
Scanning of a tianslated DNA database resulted in identification of an expressed sequence tag (EST) sequence found to be a member of the Class I Cytokine Receptor family and designated zalphal 1.<br>
Confirmation of the EST sequence was made by sequence analyses of the cDNA fi:om which the EST originated. This cDNA clone was obtained and sequenced using the following primers: ZC 447 (SEQ ID N0:5), ZC 976 (SEQ ID N0:6), ZC 19345 (SEQ ID N0:7), ZC 19346 (SEQ ID N0:8), ZC 19349 (SEQ ID N0:9), and ZC 19350 (SEQ ID NO: 10), ZC 19458 (SEQ ID N0:11), ZC 19459 (SEQ ID N0:12), ZC 19460 (SEQ ID N0:.13), ZC 19461 (SEQ ID NO: 14), ZC 19572 (SEQ ID N0:15), ZC 19573 (SEQ ID N0:16), ZC 19657 (SEQ ID N0:17). The insert was 2945 bp, and was full-length.<br>
Example 2 Tissue Distribution Northern blot analysis was performed using Human Multiple Tissue Northern'^" Blots (MTN I, MTN II, and MTN III) (Clontech). The cDNA described in Example 1 was used in a PCR reaction using oligos ZC19,181 (SEQ ID N0:18) and ZC19,182 (SEQ ID NO: 19) as primers. PCR conditions were as follows: 94°C for 1.5 minutes; 35 cycles at 94°C for 15 seconds then 68°C for 30 seconds; 72°C for 10 minutes; 4°C overnight   A sample of the PCR reaction product was run on a 1.5%<br><br>
agarose gel. A band of-the expected size of 175 bp was seen. The 175 bp PCR fragment, was gel purified using a commercially available kit (QiaexII™; Qiagen) and then radioactively labeled with ^^P-dCTP using Rediprime II™ (Amersham), a random prime labeling system^ according to the manufacturer's specifications. The probe was then purified using a Nuc-Trap™ column (Stratagene) according to the manufacturer's instructions. ExpressHyb^'^ (Clontech) solution was used for prehybridization and as a hybridizing solution for the Northern blots. Hybridization took place overnight at 65°C using 1-2x10* cpm/ml of labeled probe. The blots were then washed 4 times for 15 minutes in 2X SSC/1% SDS at 25°C, followed by a wash in O.IX SSC/0.1% SDS at 50 °C. Transcripts of approximately 3 kb and 5 kb were detected in lymph node, peripheral blood leukocytes, and thymus.<br>
Dot Blots were also performed using Human RNA Master Blots™ (Clontech). The methods and conditions for the Dot Blots are the same as for the Multiple Tissue Blots described above. Dot blot had strongest signals in thymus, lymph node, and spleen.<br>
Northem analysis was also performed using Human Cancer Cell Line MT&gt;r" (Clontech). The cDNA described in Example 1 was used in a PCR reaction using oUgos ZC 19,907 (SEQ ID NO:20) and ZC 19,908 (SEQ ID N0:21) as primers. PCR conditions were as follows: 35 cycles at 95°C for 1 minute, then 60°C for 1 minute; 72°C for 1.5 minutes; 72°C for 10 minutes; 4°C overnight A sample of the PCR reaction product was run on a 1.5% agarose gel. A band of the expected size of 1.2 kb was seen. The 1.2 kb PCR fragment, was gel purified using a commercially available kit (QiaQuick™ Gel Extraction Kit; Qiagen) and then radioactively labeled with "P-dCTP using Prime-It H"™ (Stratagene), a random prime labeling system, according to the manufacturer's specifications. The probe was then purified using a Nuc-Trap™ column (Stratagene) according to the manufacturer's instructions. ExpressHyb™ (Clontech) solution was used for prehybridization and as a hybridizing solution for the Northem blots. Hybridization took place for 2 hours at 65°C using 1-2 X 10* cpm/ml of labeled probe. The blots were then washed 4 times for 15 minutes in 2X SSC/1% SDS at 25'C, followed by two 30 minute washes in O.IX SSC/0.1% SDS<br><br>
at 50°C.   A strong signal was seen in the Raji cell line derived from Burkitt's lymphoma.<br>
Example 3<br>
PCR-Based Chromosomal Mapping of the zalphal 1 Gene<br>
Zalphall was mapped to chromosome 16 using the commercially available "GeneBridge 4 Radiation Hybrid Panel" (Research Genetics, Inc., Himtsville, AL). The GeneBridge 4 Radiation Hybrid Panel contains PCRable DNAs from each of 93 radiation hybrid clones, plus two control DNAs (the HFL donor and the A23 recipient). A publicly available WWW server (http://www-genome.wi.mit.edu/cgi-bin/contig/rhmapper.pl) allows mapping relative to the Whitehead Institute/MT Center for Genome Research's radiation hybrid map of the human genome (the "WICGR" radiation hybrid map) which was constructed with the GeneBridge 4 Radiation Hybrid Panel.<br>
For the mapping of Zalphal 1 with the "GeneBridge 4 RH Panel", 20 ^l reactions were set up in a 96-well microliter plate (Stratagene, La JoUa, CA) and used in a "RoboCycler Gradient 96" thermal cycler (Stratagene). Each of the 95 PCR reactions consisted of 2 [il lOX PCR reaction buffer (Clontech Laboratories, Inc., Palo Alto, CA), 1.6 ^1 dNTPs mix (2.5 mM each, Perkin-Ehner, Foster City, CA), 1 |il sense primer, ZC 19,954, (SEQ ID NO:22), 1 ixl antisense primer, ZC 19,955 (SEQ ID NO:23), 2 ^il "RediLoad" (Research Genetics, Inc., Huntsville, AL), 0.4 ^il 50X Advantage KlenTaq Polymerase Mix (Clontech), 25 ng of DNA from an individual hybrid clone or control an4 ddH20 for a total volume of 20 jil. The reactions were overlaid with an equal amount of mineral oil and sealed. The PCR cycler conditions were as follows: an initial 1 cycle 4 minute 'denaturation at 94°C; 35 cycles of a 45 seconds at 94°C, 45 seconds at 68°C, and 1 mmute at 72°C; followed by 7 minutes at 72°C. The reactions were separated by electrophoresis on a 2% agarose gel (Life Technologies).<br>
The results showed that zalphall maps 9.54 cR_3000 from the framework marker WI-3768 on the chromosome 16 WICGR radiation hybrid map. Proximal  and  distal   framework  markers  were   WI-3768   and   TIGR-A002K05,<br><br>
respectively. The use of surrounding markers positions Zalphall in the 16pll.l region on the integrated LDB chromosome 16 map (The Genetic Location Database, University of '.Southhampton, WWW server: http://cedar.genetics. soton.ac.ulc/public_htinl/).<br>
Example 4<br>
Construction of Human MPL-zalphal 1 Polypeptide Chimera: MPL Extracellular and<br>
TM Domain Fused to the zalphal 1 Intracellular Signaling Domain<br>
The extracellular and transmembrane domains of the MPL receptor were isolated from a plasmid containing the MPL receptor (PHZl/MPL plasmid) using PCR with primers ZC17,212 (SEQ ID NO:24) and ZC19,914 (SEQ ID NO:25). The reaction conditions were as follows: 95°C for 1 min.; 35 cycles at QS'C for 1 min., 45°C for 1 min., 72°C for 2 min.; followed by 72°C at 10 min.; then a lO'C soak. The PCR product was run on a 1% low melting point agarose (Boerhinger Mannheim, Indianapolis, IN) and the approximately 1.5 kb MPL receptor fragment isolated using Qiaquick''"** gel extraction kit (Qiagen) as per manufacturer's instructions.<br>
The intracellular domains of zalphall were isolated from a plasmid containing zalphall receptor cDNA using PCR with primers ZC19,913 (SEQ ID NO:26) and ZC20,097 (SEQ ID NO:27). The polynucleotide sequence corresponding to the zalphal 1 receptor coding sequence is shown in SEQ ID NO: 1 from nucleotide 69 to 1682. The reaction conditions were as per above. The PCR product was run on a 1% low melting point agarose (Boerhinger Maimheim) and the approximately 900 bp zalphall fragment isolated using Qiaquick gel extraction kit as per manufacturer's instructions.<br>
Each of the isolated fragments described above were mixed at a 1:1 volumetric ratio and used in a PCR reaction using ZC 17,212 (SEQ ID NO:24) and ZC20,097 (SEQ ID NO:27) to create the MPL-zalphall chimera. The reaction conditions were as follows: 95°C for 1 min.; 35 cycles at 95°C for 1 min., 55°C for 1 min., 72°C for 2 min.; followed by 72°C at 10 min.; then a 10°C soak. The entire PCR product was run on a 1% low melting point agarose (Boehringer Mannheim) and the approximately 2.4 kb MPL-zalphall chimera fragment isolated using Qiaquick gel<br><br>
extraction kit (Qiagen) as^jer manufacturer's instructions. The MPL-zalphal 1 chimera fragment was digested with EcoRI (BRL) and Xbal (Boerhinger Mannheim) as per manufacturer's instructions. The entire digest was run on a 1% low melting point agarose (Boehringer Mannheim) and the cleaved MPL-2alphal 1 chimera isolated using Qiaquick"™ gel extraction kit (Qiagen) as per manufacturer's instructions. The resultant cleaved MPL-zalphall chimera was inserted into an expression vector as described below.<br>
Recipient expression vector pZP-5N was digested with EcoRI (BRL) and Hindlll (BRL) as per manufacturer's instructions, and gel purified as described above. This vector fragment was combined with the EcoRI and Xbal cleaved MPL-zalphal 1 chimera isolated above and a Xbal/Hindlll linker fragment in a ligation reaction. The ligation was run using T4 Ligase (BRL), at IS^C overnight. A sample of the ligation was electroporated in to DHIOB ElectroMAX'^'^ electrocompetent E. coli cells (25|j,F, 200Q, 2.3V). Transformants were plated on LB+Ampicillin plates and single colonies screened by PCR to check for the MPL-zalphall chimera using ZC17,212 (SEQ ID NO:24) and ZC20,097 (SEQ ID NO:27) using the PCR conditions as described above.<br>
Confirmation of the MPL-zalphal 1 chimera sequence was made by sequence analyses using the following primers: ZC12,700 (SEQ ID NO:28), ZC5,020 (SEQ ID NO:29), ZC6,675 (SEQ ID NO:30), ZC7,727 (SEQ ID N0:31), ZC8,290 (SEQ ID NO:32), ZC19,572 (SEQ ID N0:15), ZC6,622 (SEQ ID NO:33), ZC7,736 (SEQ ID NO:34), and ZC9,273 (SEQ ID NO:35). The insert was approximately 2.4 bp, and was fiill-length.<br>
Example 5 MPL-zalphal 1 Chimera Based Proliferation in BAF3 Assay Using Alamar Blue A. Construction of BaF3 Cells Expressing MPL-zalphal 1 Chimera<br>
BaF3, an interleukin-3 (IL-3) dependent pre-lymphoid cell line derived from murine bone marrow (Palacios and Steinmetz, Cell 41: 727-734, 1985; Mathey-Prevot et al., Mol. Cell. Biol. 6: 4133-4135, 1986), was maintained in complete media (RPMI medium (JRH Bioscience Inc., Lenexa, KS) supplemented with 10% heat-<br><br>
inactivated fetal calf serum, 2ng/ml murine IL-3 (mIL-3) (R&amp;D, Minneapolis, MN), 2mM L-glutaMax-l™ (Gibco BRL), 1 mM Sodium Pyruvate (Gibco BRL), and PSN antibiotics (GIBCO BRL)). Prior to electroporation, pZP-5N/MPL-zalphal 1 DNA (Example 4) was prepared and purified using a Qiagen Maxi Prep kit (Qiagen) as per manufacturer's instructions. BaF3 cells for electroporation were washed once in RPMI<br>
media and then resuspended in RPMI media at a cell density of 10^ cells/ml. One ml of resuspended BaF3 cells was mixed with 30 |xg of the pZP-5N/MPL-zalphall plasmid DNA and transferred to separate disposable electroporation chambers (GIBCO BRL). Following a 15 minute incubation at room temperature the cells were given two serial shocks (800 lFad/300 V.; 1180 lFad/300 V.) delivered by an electroporation apparatus (CELL-PORATOR™; GIBCO BRL). After a 5 minute recovery time, the electroporated cells were transferred to 50 ml of complete media and placed in an incubator for 15-24 hours (37°C, 5% CO2).     The cells were then spun down and<br>
resuspended in 50 ml of complete media containing Geneticin'™ (Gibco) selection (500 ^g/ml G418) in a T-162 flask to isolate the G418-resistant pool. Pools of the transfected BaF3 cells, hereinafter called BaF3/MPL-zalphal 1 cells, were assayed for signaling capability as described below.<br>
B. Testing the Signaling Capability of the BaF3/MPL-zalphal 1 Cells Using an Alamar Blue Proliferation Assay<br>
BaF3/MPL-zalphal 1 cells were spun down and washed in the complete media, described above, but without mIL-3 (hereinafter referred to as "mIL-3 free media"). The cells were spun and washed 3 times to ensure the removal of the mIL-3. Cells were then counted in a hemac^ometer. Cells were plated in a 96-well format at 5000 cells per well in a volume of 100 jal per well using the mIL-3 free media.<br>
Proliferation of the BaF3/MPL-zalphal 1 cells was assessed using thrombopoietin (TPO) diluted with mIL-3 free media to 500 ng/ml, 250ng/ml, 125 ng/ml, 62 ng/ml, 30 ng/ml, 15 ng/ml, 7.5 ng/ml, 3.75 ng/ml, 1.8 ng/ml, 0.9 ng/ml, 0.5 ng/ml and 0.25 ng/ml concentrations. 100 pi of the diluted TPO was added to the BaF3/MPL-zalphal 1 cells. The total assay volume is 200 yii. Negative controls were<br><br>
run in parallel using mIL-3 free media only, without the addition of TPO. The assay plates were incubated at 37°C, 5% CO2 for -3 days at which time Alamar Blue (Accumed, Chicago, IL). was added at 20nl/well. Alamar Blue gives a fluourometric readout based on number of live cells, and is thus a direct measxirement of cell proliferation in comparison to a negative control. Plates were again incubated at 37°C, 5% CO2 for 24 hours. Plates were read on the Fmax"™ plate reader (Molecular Devices Sunnyvale, CA) using the SoftMax"™ Pro program, at wavelengths 544 (Excitation) and 590 (Emmission).<br>
Results confirmed the signaling capability of the intracellular portion of the 2alphall receptor as the thrombopoietin induced proliferation at approximately 10 fold over back ground at 62 ng/ml and greater.<br>
Example 6 Construction of Expression Vector Expressing Full-length zalphal 1<br>
The entire zalphal 1 receptor was isolated from a plasmid containing zalphall receptor cDNA using PCR with primers ZC 19,905 (SEQ ID NO:36) and ZC19,906 (SEQ ID NO:37). The reaction conditions were as follows: 95°C for 1 min; 35 cycles at 95°C for 1 min, 55°C for 1 min, 72°C for 2 min; followed by 72°C at 10 min; then a 10°C soak. The PCR product was run on a 1% low melting point agarose (Boerhinger Mannheim) and the approximately 1.5 kb zalphal 1 cDNA isolated using Qiaquick''** gel extraction kit (Qiagen) as per manufacturer's instructions.<br>
The purified zalphall cDNA was digested with BamHI (Boerhinger Mannheim) and EcoRI (BRL) as per manufacturer's instructions. The entire digest was run on a 1% low melting point agarose (Boerhinger Mannheim) and purified the cleaved zalphall fragment using Qiaquick gel exfraction kit as per manufacturer's instructions. The resultant cleaved zalphal 1 chimera was inserted into an expression vector as described below;<br>
Recipient expression vector pZP-5N was digested with BamHI (Boerhinger Mannheim) and EcoRI (BRL) as per manufacturer's instructions, and gel purified as described above. This vector fragment was combined with the BamHI and EcoRI cleaved zalphal 1 fragment isolated above in a ligation reaction.  The ligation<br><br>
was run using T4 Ligase (BRL), at 15°C overnight. A sample of the ligation was electroporated in to D"HlOB electroMAX™ electrocompetent E. coli cells (25|iF, 200^, 2.3V). Transformants .were plated on LB+Ampicillin plates and single colonies screened by PCR to.clieck for the zalphal 1 sequence using ZCl9,905 (SEQ ID NO:36) and ZCl9,906 (SEQ ID NO:37) using the PCR conditions as described above.<br>
Confirmation of the MPL-zalphal 1 sequence was made by sequence analyses usmg the following primers: ZCl2,700 (SEQ ID NO:28), ZC5,020 (SEQ ID NO:29), ZC20,114 (SEQ ID NO:38), ZC19,459 (SEQ ID N0:12), ZC19,954 (SEQ ID NO:39), and ZC20,116 (SEQ ID NO:40). The insert was approximately 1.6 kb, and was full-length.<br>
Example 7 Zalphal 1 based proliferation in BAF3 assay using Alamar Blue A. Construction of BaF3 Cells Expressing zalphal 1 receptor<br>
BaF3 cells expressing the full-length zalphal 1 receptor were constructed as per Example 5 A above, using 30ng of the zalphal 1 expression vector, described in Example 6 above. The BaF3 cells expressing the zalphal 1 receptor mRNA were designated as BaF3/zalphal 1. These cells were used to screen for a zalphal 1 activity as described below in Examples 8 and 12.<br>
Example 8 Screening for zalphal 1 activity using BaF3/Zalphal 1 cells using an Alamar Blue<br>
Proliferation Assay<br>
A. Monkey Primary Source used to test for presence of zalphal 1 activity<br>
Conditioned media from primary monkey spleen cells was used to test for the presence of activity as described below. Monkey spleen cells were activated with 5 ng/ml Phorbol-12-myristate-13-acetate (PMA) (Calbiochem, San Diego, CA), and 0.5|ig/ml lonomycin'™ (Calbiochem) for 72 h. The supernatant from the stimulated monkey spleen cells was used to assay proliferation of the BaF3/zalphal 1 cells as described below.<br><br>
B.	Screening for zalphall activity using BaF3/Zalphal 1 cells using an Alamar Blue<br>
Proliferation Assay .'/_ .<br>
BaF3/2falphal 1 cells were spun down and washed in mIL-3 free media. The cells were spun and washed 3 times to ensure the removal of the mIL-3. Cells were then counted in a hemacytometer. Cells were plated in a 96-well format at 5000 cells per well in a volimie of 100 |J,1 per well using the niIL-3 free media.<br>
Proliferation of the BaF3/Zalphal 1 cells was assessed using conditioned media from activated monkey spleen (see Example 8A, above) was diluted with mIL-3 free media to 50%, 25%, 12.5%, 6.25%, 3.125%, 1.5%, 0.75% and 0.375% concentrations. 100 |j,l of the diluted conditioned media was added to the BaF3/Zalphal 1 cells. The total assay volume is 200 fil. The assay plates were incubated at 37°C, 5% COj for 3 days at which tune Alamar Blue (Accumed, Chicago, IL) was added at 20 ^il/well. Plates were again incubated at 37°C, 5% CO2 for 24 hours. Plates were read on the Fmax"™ plate reader (Molecular devices) as described above (Example 5).<br>
Results confirmed the proliferative response of the BaF3/Zalphal 1 cells to a factor present in the activate monkey spleen conditioned media. The response, as measured, was approximately 4-fold over background at the 50% concentration. The BaF3 wild type cells did not prohferate in response to this factor, showing that this factor is specific for the Zalphal 1 receptor.<br>
C.	Human Primary Source used to isolate zalphal 1 activity<br>
100 ml blood draws were taken from each of six donors. The blood was drawn using lOX 10 ml vacutainer tubes containing heparin. Blood was pooled from six donors (600ml), diluted 1:1 in PBS, and separated using a FicoU-Paque® PLUS (Pharmacia Biotech, Uppsala, Sweden). The isolated primary human cell yield after separation on the ficoU gradient was 1.2X10' cells.<br>
Cells were suspended in 9.6 ml MACS buffer (PBS, 0.5% EDTA, 2mM EDTA). 1.6 ml of cell suspension was removed and 0.4 ml CD3 microbeads (Miltenyi Biotec, Auburn, CA) added.   The mixture was incubated for 15 min. at 4°C.   These<br><br>
cells labeled with CDS -beads were washed with 30 ml MACS buffer, and then resuspended in 2 ml MACS buffer.<br>
A VS+'.cqlumn (Miltenyi) was prepared according to the manufacturer's instructions. The yS'+ column was then placed in a VarioMACS"™ magnetic field (Miltenyi). The column was equilibrated with 5 ml MACS buffer. The isolated primary human cells were then applied to the column. The CD3 negative cells were allowed to pass through. The column was rinsed with 9 ml (3 X 3 ml) MACS buffer. The colimm was then removed firom the magnet and placed over a 15 ml falcon tube. CD3+ cells were eluted by adding 5 ml MACS buffer to the column and bound cells flushed out using the plunger provided by the manufacturer. The incubation of the cells with the CDS magnetic beads, washes, and VS+ column steps (incubation through elution) above were repeated five more times. The resulting CD3+ fi-actions firom the six column separations were pooled. The yield of CDS+ selected human T-cells were 3X10'total cells.<br>
A sample of the pooled CDS+ selected human T-cells was removed for staining and sorting on a fluorescent antibody cell sorter (FACS) to assess their purity. The CD3+ selected human T-cells were 91% CD3+ cells.<br>
The CD3+ selected human T-cells were activated by incubating m RPMI + 5% FBS + PMA 10 ng/ml and lonomycin 0.5 |ig/ml (Calbiochem) for 13 hours 37°C. The supernatant from these activated CD3+ selected human T-cells was tested for zalphal 1 activity as described below.<br>
D. Testing supernatant from activated CD3+ selected human T-cells for zalphall activitv using BaFS/Zalphal 1 cells and an Alamar Blue Proliferation Assay<br>
* BaFS/Zalphal 1 cells were spim down and washed in mlL-S free media. The cells were spun and washed 3 times to ensure the removal of the mIL-3. Cells were then coxmted in a hemacytometer. Cells were plated in a 96-well format at 5000 cells per. well in a volume of 100 fj.1 per well using the mlL-S firee media.<br>
Proliferation of the BaF3/Zalphal 1 cells was assessed using conditioned media ftom activated CD3+ selected human T-cells (see Example 8C, above) diluted with mIL-3 fi-ee media to 50%, 25%, 12.5%, 6.25%, 3.125%, 1.5%, 0.75% and 0.375%<br><br>
concentrations. 100 pi of the diluted conditioned media was added to the BaF3/Zalphal 1 cells! The total assay volume is 200 \i\. The assay plates were incubated and assayedas. described in Example 8B above.<br>
Results^ confirmed the proliferative response of the BaF3/Zalphall cells to a factor present in the activated CD3+ selected human T-cell conditioned media. The response, as measured, was approxunately 10-fold over background at the 50% concentration. The BaF3 wild type cells did not proliferate in response to this factor, showing that this factor is specific for the Zalphal 1 receptor.<br>
Example 9<br>
Construction of mammalian expression vectors that express zalphal 1 soluble receptors:<br>
zalphalICEE. zalphal ICFLG. zalphallCHIS andzalphl 1-Fc4<br>
A. Construction of zalphal 1 Mammalian Expression Vector containing zalphl ICEE, zalphl ICFLG and zalphl ICHIS<br>
An expression vector was prepared for the expression of the soluble, extracellular domain of the zalphal 1 polypeptide, pC4zalphllCEE, wherein the construct is designed to express a zalphal 1 polypeptide comprised of the predicted initiating methionine and truncated adjacent to the predicted transmembrane domain, and with a C-terminal Glu-Glu tag (SEQ ID N0:41).<br>
A 700 bp PCR generated zalphal 1 DNA fragment was created using ZC19,931 (SEQ ID NO:42) and ZC19,932 (SEQ ID NO:43) as PCR primers to add Asp718 and BamPII restriction sites. A plasmid containing the zalphal 1 receptor cDNA was used as a template. PCR amplification of the zalphal 1 fragment was performed as follows: Twenty five cycles at 94C for 0.5 minutes; five cycles at 94°C for 10 seconds, 50°C for 30 seconds, 68°C for 45 seconds, followed by a 4°C hold. The reaction was purified by chloroform/phenol extraction and isopropanol precipitation, and digested with Asp718 and BamHI (Gibco BRL) following manufacturer's protocol. A band of the predicted size, 700 bp, was visualized by 1% agarose gel electrophoresis, excised and the DNA was purified using a QiaexIF'^ purification system (Qiagen) according the manufacturer's instructions.<br><br>
The excised DNA was subcloned into plasmid pC4EE which had been cut with BamHI and'Asp718. The pC4zalphllCEE expression vector uses the native zalphall signal peptide, and attaches the Glu-Glu tag (SEQ ID N0:41) to the C-terminus of the zalphal 1 polypeptide-encoding polynucleotide sequence. Plasmid pC4EE, is a mammalian expression vector containing an expression cassette having the mouse metallothionein-1 promoter, multiple restriction sites for insertion of coding sequences, a stop codon and a human growth hormone terminator. The plasmid also has an E. coli origin of replication, a mammalian selectable marker expression unit having an SV40 promoter, enhancer and origin of replication, a DHFR gene and the SV40 terminator.<br>
About 30 ng of the restriction digested zalphal 1 insert and about 12 ng of the digested vector were ligated overnight at 16°C. One microliter of each ligation reaction was independently electroporated into DHIOB competent cells (GIBCO BRL, Gaithersburg, MD) according to. manufacturer's direction and plated onto LB plates containing 50 mg/ml ampicillin, and incubated overnight. Colonies were screened by restriction analysis of DNA prepared from 2 ml liquid cultures of individual colonies. The insert sequence of positive clones was verified by sequence analysis. A large scale plasmid preparation was done using a QIAGEN® Maxi prep kit (Qiagen) according to manufacturer's instructions.<br>
The same process was used to prepare the zalphal 1 soluble receptors with a C-terminal his tag, composed of 6 His residues in a row; and a C-terminal flag (SEQ ID NO:49) tag, ' zalphal ICFLAG. To construct these constructs, the aforementioned vector has either the HIS or the FLAG® tag in place of the glu-glu tag (SEQIDN0:41).<br>
B. Mammalian Expression Construction of Soluble zalphal 1 receptor zalphal 1-Fc4<br>
An expression plasmid containing all or part of a polynucleotide encoding zalphall was constructed via homologous recombination. A fragment of zalphall cDNA was isolated using PCR that includes the polynucleotide sequence from extracellular domain of the zalhall receptor. The two primers used in the<br><br>
production of the zalphall fragment were: (1) The primers for PCR each include from 5' to 3' end: 40 bp' of the vector flanking sequence (5' of the insert) and 17 bp corresponding to the 5' end of the zaiphal 1 extracellular domain (SEQ ID NO:44); and (2) 40 bp of the 5' end of the Fc4 polynucleotide sequence (SEQ ID NO:45) and 17 bp corresponding to the 3' end of the zalphall extracellular domain (SEQ ID NO:46). The fragment of Fc-4 for fusion with the zaiphal 1 was generated by PCR in a similar fashion. The two primers used in the production of the Fc4 fragment were: (1) a 5' primer consisting of 40 bp of sequence from the 3' end of zalphall extracellular domain and 17 bp of the 5' end of Fc4 (SEQ ID NO:47); and (2) a 3' primer consisting of 40 bp of vector sequence (3' of the insert) and 17 bp of the 3' end of Fc4 (SEQ ID NO:48).<br>
PCR amplification of the each of the reactions described above was performed as follows: one cycle at 94°C for 2 minutes; twenty-five cycles at 94°C for 30 seconds, 60°C for 30 seconds, 72°C for 1 minute; one cycle at 72°C for 5 minutes; followed by a 4°C hold. Ten ^1 of the 100 pi PCR reaction was run on a 0.8% LMP agarose gel (Seaplaque GTG) with 1 x TBE biiffer for analysis. The remaining 90 [il of PCR reaction is precipitated with the addition of 5 pi 1 M NaCl and 250 pi of absolute ethanol. The expression vector used was derived from the plasmid pCZR199 (deposited at the American Type Culture Collection, 10801 University Boulevard, Manassas, VA 20110-2209, and is designated No. 98668), and was cut with Smal (BRL). The expression vector was derived from the plasmid pCZR199, and is a mammalian expression vector containing an expression cassette having the CMV immediate early promoter, a consensus intron from the variable region of mouse immunoglobulin heavy chain locus, multiple restriction sites for insertion of coding sequences, a stop codon and a h\mian growth hormone terminator. The expression vector also has an E. coli origin of replication, a mammalian selectable marker expression unit having an SV40 promoter, enhancer and origin of replication, a DHFR gene and the SV40 terminator. The expression vector used was constructed from pCZR199 by the replacement of the metallothionein promoter with the CMV immediate early promoter.<br><br>
One hundred microliters of competent yeast cells (5. cerevisiae) were combined with 10 \il containing approximately 1 (ig each of the zalphall and Fc4 inserts, and 100 ng of-lSmjal (BRL) digested expression vector and transferred to a 0.2 cm electroporation cuvette. The yeast/DNA mixtures were electropulsed at 0.75 kV (5 kV/cm), "infinite" ohms, 25 |aF. To each cuvette is added 600 \i\ of 1.2 M sorbitol and the yeast was plated in two 300 |il aliquots onto two URA-D plates and incubated at 30°C.<br>
After about 48 hours, the Ura+ yeast transformants firom a single plate were resuspended in 1 ml H2O and spun briefly to pellet the yeast cells. The cell pellet was resuspended in 1 ml of lysis buffer (2% Triton X-100, 1% SDS, 100 mM NaCl, 10 mM Tris, pH 8.0, 1 mM EDTA). Five hundred microliters of the lysis mixture was added to an Eppendorf tube containing 300 \il acid washed glass beads and 200 nl phenol-chloroform, vortexed for 1 minute intervals two or three times, followed by a 5 minute spin in a Eppendorf centrifuge at maximum speed. Three hundred microliters of the aqueous phase was transferred to a fresh tube, and the DNA precipitated with 600 |J,1 ethanol (EtOH), followed by centrifugation for 10 minutes at 4'C. The DNA pellet was resuspended in 100 fil HjO.<br>
Transformation of electrocompetent E. coli cells (DHIOB, GibcoBRL) is done with 0.5-2 ml yeast DNA prep and 40 ul of DHIOB cells. The cells were electropulsed at 2.0 kV, 25 mF and 400 ohms. Following electroporation, 1 ml SOC (2% Bactoe Tryptone (Difco, Detroit, MI), 0.5% yeast extract (Difco), 10 mM NaCl, 2.5 mM KCl, 10 mM MgC12, 10 mM MgS04, 20 mM glucose) was plated in 250 \d aliquots on four LB AMP plates (LB broth (Leimox), 1.8% Bacto Agar (Difco), 100 mg/L Ampicillin).<br>
Individual clones harboring the correct expression construct for zalphall-Fc4 were identified by restriction digest to verify the presence of the zalphal 1-Fc4 insert and to confirm that the various DNA sequences have been joined correctly to one another. The insert of positive clones were subjected to sequence analysis. Larger scale plasmid DNA is isolated using the Qiagen Maxi kit (Qiagen) according to manufacturer's instructions.<br><br>
Example 10 Transfection AJcid Expression Of Zalphal 1 Soluble Receptor Polypeptides<br>
BHK ^70 cells (ATCC No. CRL-10314), passage 27, were plated at 1.2X10* cells/well (6-well plate) in 800 ^1 of serum free (SF) DMEM media (DMEM, Gibco/BRL High Glucose) (Gibco BRL, Gaithersburg, MD). The cells were transfected with expression plasmids containing zalphal ICEE/CFLG/CHIS described above (see, Example 9), using Lipofectin'"** (Gibco BRL), in serum free (SF) DMEM. Three micrograms of zalphal ICEE/CFLG/CHIS each were separately diluted into 1.5 ml tubes to a total final volume of 100 ^1 SF DMEM. In separate tubes, 15 |il of Lipofectin™ (Gibco BRL) was mixed with 100 |il of SF DMEM. The Lipofectin™ mix was incubated at room temperature for 30-45 minutes then tlie DNA mix was added and allowed to incubate approximately 10-15 minutes at room temperature.<br>
The entire DNA: Lipofectin'™* mixture was added to the plated cells and distributed evenly over them. The cells were incubated at 37°C for approximately five hours, then transferred to separate 150 mm MAXI plates in a final volume of 30 ml DMEM/5% fetal bovine serum (FBS) (Hyclone, Logan, UT). The plates were incubated at 37°C, 5% COj, overnight and the DNA: Lipofectin'™ mixture was replaced with selection media (5% FBS/DMEM with 1 jiM methotrexate (MTX))the next day.<br>
Approximately 10-12 days post-transfection, the plates were washed with 10 ml SF DMEM. The wash media was aspirated and replaced vnUi 7.25 ml serum-free DMEM. Sterile Teflon meshes (Spectrum Medical Industries, Los Angeles, CA) pre-soaked in SF DMEM were then placed over the clonal cell colonies. A sterile nitrocellulose filter pre-soaked in SF DMEM was then placed over the mesh. Orientation marks on the nitrocellulose were transferred to the culture dish. The plates were then incubated for 5-6 hours in a 37°C, 5% COj incubator.<br>
Following incubation, the filters/meshes were removed, and the media aspirated and replaced with 5% FBS/DMEM with 1 \iM MTX. The filters were then blocked in 10% nonfat dry milk/Westem A buffer (Western A: 50mM Tris pH 7.4, 5 mM EDTA, 0.05% NP-40, 150 mM NaCl and 0.25% gelatin) for 15 minutes at room<br><br>
temperature on a rotating shaker. The filters were then incubated with an anti-Glu-Glu, anti-FLAG®, or anti-HIS antibody-HRP conjugates, respectively, rn 2.5% nonfat dry milkAVestem A buffer for one hour at room temperature on a rotating shaker. The filters were then washed three times at room temperature with Western A for 5-10 minutes per wash. The filters were developed with ultra ECL reagent (Amersham Corp., Arlington Heights, IL) according the manufacturer's directions and visualized on the Lumi-Imager (Roche Corp.)<br>
Positive expressing clonal colonies were mechanically picked to 12-well plates in one ml of 5%FCS/DMEM with 5 |iM MTX, then grown to confluence. Conditioned media samples were then tested for expression levels via SDS-PAGE and Western anlaysis. The three highest expressing clones for each construct were picked; two out of three were frozen down as back up and one was expanded for mycoplasma testing and large-scale factory seeding.<br>
B. Mammalian Expression of soluble zalphal 1 receptor zalphal 1-Fc4<br>
BHK 570 cells (ATCC NO: CRL-10314) were plated in 10 cm tissue culture dishes and allowed to grow to approximately 50 to 70% confluency overnight at 37_C , 5% CO2, m DMEM/FBS media (DMEM, Gibco/BRL High Glucose, (Gibco BRL, Gaithersburg, MD), 5% fetal bovine serum (Hyclone, Logan, UT), 1 mM L-glutamine (JRH Biosciences, Lenexa, KS), 1 mM sodium pyruvate (Gibco BRL)). The cells were then transfected with the plasmid containing zalphall-Fc4 (see, Example 9), using Lipofectamine™ (Gibco BRL), in serum free (SF) media formulation (DMEM, 10 mg/ml transferrin, 5 mg/ml insulin, 2 mg/ml fetuin, 1% L-glutamine and 1% sodiiun pyruvate). The plasmid containing zalphal 1-Fc4 was diluted into 15 ml tubes to a total final volume of 640 ml with SF media. 35 ml of Lipofectamine™ (Gibco BRL) was mixed with 605 ml of SF medium. The Lipofectamine™ mix was added to the DNA mix and allowed to incubate approximately 30 minutes at room temperature. Five milliliters of SF media was added to the DNA:Lipofectamine™ mixture. The cells were rinsed once with 5 ml of SF media, aspirated, and the DNA:Lipofectamine™<br>
mixture is added. The cells were incubated at 370C for five hours, then 6.4 ml of DMEM/10% FBS, 1% PSN media was added to each plate. The plates were mcubated<br><br>
at 370c overnight and the DNA:Lipofectamine™ mixture was replaced with fresh 5% FBS/DMEM media the next day. On day 2 post-transfection, the cells were split into the selection media (DMEM/FBS media from above with the addition of 1 mM methotrexate (Sigma Chemical Co., St. Louis, Mo.)) in 150 mm plates at 1:10,1:20 and 1:50. The media on the cells was replaced with fresh selection media at day 5 post-transfection. Approximately 10 days post-transfection, two 150 mm culture dishes of methotrexate resistant colonies from each transfection were trypsinized and the cells are pooled and plated into a T-162 flask and transferred to large scale culture.<br>
Example 11 Purification of zalphal 1 soluble receptors from BHK 570 cells A. Purification of zalphal ICEE polypeptide from BHK 570<br>
Unless otherwise noted, all operations were carried out at 4°C. The following procedure was used for purifying zalphall polypeptide containing C-terminal GluGlu (EE) tags. Thirty liters of cell factory conditioned media was concentrated to 1.6 liters with an Amicon S10Y3 spiral cartridge on a ProFlux A30. A Protease inhibitor solution was added to the concentrated 1.6 liters of cell factory conditioned media from transfected BHK 570 cells (see, Example 10) to final concentrations of 2.5 mM ethylenediaminetetraacetic acid (EDTA, Sigma Chemical Co. St. Louis, MO), 0.003 mM leupeptin (Boehringer-Mannheim, Indianapolis, IN), 0.001 mM pepstatin (Boehringer-Mannheim) and 0.4 mM Pefabloc (Boehringer-Mannheim). Samples were removed for analysis and the bulk volume was frozen at -80°C until the purification was started. Total target protein concentrations of the concentrated cell factory conditioned media was determined via SDS-PAGE and Western blot analysis with the anti-EE HRP conjugated antibody.<br>
A 100 ml column of anti-EE G-Sepharose (prepared as described below) was poured in a Waters AP-5, 5 cm x 10 cm glass column. The column was flow packed and equilibrated on a BioCad Sprint (PerSeptive BioSystems, Framingham, MA) with phosphate buffered saline (PBS) pH 7.4. The concentrated cell factory conditioned media was thawed, 0.2 micron sterile filtered, pH adjusted to 7.4, then loaded on the column overnight with 1 ml/minute flow rate.  The column was washed<br><br>
with 10 column volumes (CVs) of phosphate buffered saline (PBS, pH 7.4), then plug eluted with 200 ml of PBS (pH 6.0) containing 0.5 mg/ml EE peptide (Anaspec, San Jose, CA) at 5 ml/minute.. The EE peptide used has the sequence EYMPME (SEQ ID N0:41). The column, was washed for 10 CVs with PBS, then eluted with 5 CVs of 0.2M glycine, pH 3.0. The pH of the glycine-eluted column was adjusted to 7.0 with 2 CVs of 5X PBS, then equilibrated in PBS (pH 7.4). Five ml fractions were collected over the entire elution chromatography and absorbance at 280 and 215 nM were monitored; the pass through and wash pools were also saved and analyzed. The EE-polypeptide elution peak fractions were analyzed for the target protein via SDS-PAGE Silver staining and Western Blotting with the anti-EE HRP. conjugated antibody. The polypeptide elution fractions of interest were pooled and concentrated from 60 ml to 5.0 ml using a 10,000 Dalton molecular weight cutoff membrane spin concentrator (Millipore, Bedford, MA) according to the manufacturer's instructions.<br>
To separate zalphallCEE from other co-purifying proteins, the concentrated polypeptide elution pooled fractions were subjected to a POROS HQ-50 (sfrong anion exchange resin from PerSeptive BioSystems, Framingham, MA) at pH 8.0. A 1.0 X 6.0 cm column was poured and flow packed on a BioCad Sprint. The column was counter ion charged then equibrated in 20mM TRIS pH 8.0 (Tris (Hydroxymethyl Aminomethane)). The sample was diluted 1:13 (to reduce the ionic strength of PBS) then loaded on the Poros HQ column at 5 ml/minute. The column was washed for 10 CVs with 20mM Tris pH 8.0 then eluted with a 40 CV gradient of 20 mM Tris/ 1 M sodium chloride (NaCl) at 10 ml/minute. 1.5 ml fractions were collected over the entire chromatography and absorbance at 280 and 215 nM were monitored. The elution peak fractions were analyzed via SDS-PAGE Silver staining. Fractions of interest were pooled and concentrated to 1.5-2 ml using a 10,000 Dalton mo'lecular weight cutoff membrane spin concentrator (Millipore, Bedford, MA) according to the manufacturer's instructions.<br>
To separate zalphallCEE polypeptide from free EE peptide and any contaminating co-purifying proteins, the pooled concentrated fractions were subjected to chromatography on a 1.5 x 90 cm Sephadex S200 (Pharmacia, Piscataway, NJ) colimm equilibrated and loaded in PBS at a flow rate of 1.0 ml/mm using a BioCad<br><br>
Sprint. 1.5 ml fractions were collected across the entire chromatography and the absorbance at 280 and 215 nM were monitored. The peak fractions were characterized via SDS-PAGE Silver staining, and only the most pure fractions were pooled. This material represented purified zalphal ICEE polypeptide.<br>
This purified material was finally sujectd to a 4 ml ActiClean Etox (Sterogene) column to remove any remaining endotoxins. The sample was passed over the PBS equilibrated gravity column four times then the column was washed with a single 3 ml volume of PBS, which was pooled with the "cleaned" sample. The material was then 0.2 micron sterile filtered and stored at -80°C until it was aliquoted.<br>
On Western blotted, Coomassie Blue and Silver stained SDS-PAGE gels, the zalphal ICEE polypeptide was one major band of an apparent molecular weight of 50,000 Daltons. The mobility of this band was the same on reducing and non-reducing gels.<br>
The protein concentration of the purified material was performed by BCA analysis (Pierce, Rockford, IL) and the protein was aliquoted, and stored at -80°C according to our standard procedures. On lEF (isoelectric focusing) gels the protein runs with a PI of less than 4.5. The concentration of zalphal ICEE polypeptide was 1.0 mg/ml.<br>
Purified zalphal ICEE polypeptide was prepared for injection into rabbits and sent to R &amp; R Research and Development (Stanwood, WA) for antibody production. Rabbits were injected to produce anti-huzalphall-CEE-BHK serum (Example 15, below).<br>
To prepare anti-EE Sepharose, a 100 ml bed volume of protein G-Sepharose (Pharmacia, Piscataway, NJ) was washed 3 times with 100 ml of PBS containing 0.02% sodium azide using a 500 ml Nalgene 0.45 micron filter unit. The gel was washed with 6.0 volumes of 200 mM triethanolamine, pH 8.2 (TEA, Sigma, St. Louis, MO), and an equal volume of EE antibody solution containing 900 mg of antibody was added. After an overnight incubation at 4°C, unbound antibody was removed by washing the resin with 5 volumes of 200 mM TEA as described above. The resin was resuspended in 2 volumes of TEA, transferred to a suitable container, and<br><br>
dimethylpiiniliinidate-2HCl (Pierce, Rockford, IL) dissolved in TEA, was added to a final concentration of 36 rhg/ml of protein G-Sepharose gel. The gel was rocked at room temperature fohiAS] rain and the liquid was removed using the filter unit as described above. Nonspecific sites on the gel were then blocked by incubating for 10 min. at room temperature with 5 volumes of 20 mM ethanolamine in 200 mM TEA. The gel was then washed with 5 volumes of PBS containing 0.02% sodium azide and stored in this solution at 4°C.<br>
B. Purification of zalnhal ICFLAG polypeptide fi-om BHK 570<br>
Unless otherwise noted, all operations were carried out at 4°C. The following procedure was used for purifying zalphall polypeptide containing C-terminal FLAG® (FLG) (Sigma-Aldrich Co.) tags. Thirty liters of cell factory conditioned media was concentrated to 1.7 liters with an Amicon S10Y3 spiral catridge on a ProFlux A30. A Protease inhibitor solution was added to the 1.7 liters of concentrated cell factory conditioned media fi-om transfected BHK 570 cells (see, Example 10) to final concentrations of 2.5 mM ethylenediaminetetraacetic acid (EDTA, Sigma Chemical Co. St. Louis, MO), 0.003 mM leupeptin (Boehringer-Mannheim, Indianapolis, IN), 0.001 mM pepstatin (Boehringer-Mannheim) and 0.4 mM Pefabloc (Boehringer-Maimheim). Samples were removed for analysis and the biilk volume was frozen at -80°C until the purification was started. Total target protein concentrations of the cell factory conditioned media was determined via SDS-PAGE and Western blot analysis with the anti-FLAG® (Kodak) HRP conjugated antibody. A 125 ml column of anti- FLAG® M2-Agarose affinity gel (Sigma-Aldrich Co.) was poiired in a Waters AP-5, 5 cm X 10 cm glass column. The colunm was flow packed and equilibrated on a BioCad Sprint (PerSeptive BioSystems, Framingham, MA) with phosphate buffered saline (PBS) pH 7.4. The concentrated cell factory conditioned media was thawed, 0.2 micron sterile filtered, pH adjusted to 7.4, then loaded on the column overnight with 1 ml/minute flow rate. The column was washed with 10 column volumes (CVs) of phosphate buffered saline (PBS, pH 7.4), then plug eluted with 250 ml of PBS (pH 6.0) containing 0.5 mg/ml FLAG® (Sigma-Aldrich Co.) peptide at 5 ml/minute. The FLAG® peptide used has the sequence DYKDDDDK (SEQ ID NO:49). The column<br><br>
was washed for 10 CVs with PBS, then eluted with 5 CVs of 0.2M glycine, pH 3.0. The pH of the glycine-eluted column was adjusted to 7.0 with 2 CVs of 5X PBS, then equilibrated in PBS (pH.7.4). Five ml fractions were collected over the entire elution chromatography and absorbence at 280 and 215 nM were monitored; the pass through and wash pools were also saved and analyzed. The FLAG®-polypeptide elution peak fractions were analyzed for the target protein via SDS-PAGE Silver staining and Westem Blotting with the anti-FLAG HRP conjugated antibody. The polypeptide elution fractions of interest were pooled and concentrated from 80 ml to 12 ml using a 10,000 Dalton molecular weight cutoff membrane spin concentrator (Millipore, Bedford, MA) according to the manufacturer's instructions.<br>
To separate zalphallCFLG from other co-purifying proteins, the polypeptide elution pooled fractions were subjected to a POROS HQ-50 (strong anion exchange resin from PerSeptive BioSystems, Framingham, MA) at pH 8.0. A 1.0 x 6.0 cm column was poured and flow packed on a BioCad Sprint. The column was counter ion charged then equilibrated in 20mM TRIS pH 8.0 (Tris (Hydroxymethyl Aminomethane)). The sample was diluted 1:13 (to reduce the ionic strength of PBS) then loaded on the Poros HQ-50 column at 5 ml/minute. The column was washed for 10 column volumes (CVs) with 20mM Tris pH 8.0 then eluted with a 40 CV gradient of 20 mM Tris/ 1 M sodium chloride (NaCl) at 10 ml/minute. 1.5 ml fractions were collected over the entire chromatography and absorbance at 280 and 215 nM were monitored. The elution peak fractions were analyzed via SDS-PAGE Silver staining. Fractions of interest were pooled and concentrated to 1.5-2 ml using a 10,000 Dalton molecular weight cutoff membrane spin concentrator (Millipore, Bedford, MA) according to the manufacturer's instructions.<br>
To separate zalphallCFLG polypeptide from free FLAG® pep'tide and any contaminating co-purifying proteins, the pooled concentrated fractions were subjected to chromatography on a 1.5 x 90 cm Sephacryl S200 (Pharmacia, Piscataway, NJ) colimm equilibrated and loaded in PBS at a flow rate of 1.0 ml/min using a BioCad Sprint. 1.5 ml fractions were collected across the entire chromatograohy and the absorbance at 280 and 215 nM were monitored. The peak fractions were characterized<br><br>
via SDS-PAGE Silver staining, and only the most pure fractions w^ere pooled. This material represented purified zalphal 1CFLG polypeptide.<br>
This pimfied material was finally sujectd to a 4 ml ActiClean Etox (Sterogene) column.ta remove any remaining endotoxins. The sample was passed over the PBS equilibrated gravity colimm four times then the colimm was washed with a single 3 ml volume of PBS, which was pooled with the "cleaned" sample. The material was then 0.2 micron sterile filtered and stored at -SCC until it was aliquoted.<br>
On Western blotted, Coomassie Blue and Silver stained SDS-PAGE gels, the zalphal 1 CFLG polypeptide was one major band of an apparent molecular weight of 50,000 Daltons. The mobility of this band was the same on reducing and non-reducing gels.<br>
The protein concentration of the purified material was performed by BCA analysis (Pierce, Rockford, IL) and the protein was aliquoted, and stored at -80°C according to our standard procedures. On lEF (isoelectric focusing) gels the protein runs with a PI of less than 4.5. The concentration of zalphal ICFLG polypeptide was 1.2 mg/ml.<br>
C. Purification of zalphal 1-Fc4 polvpeptide from transfected BHK 570 cells<br>
Unless otherwise noted, all operations were carried out at 4°C. The following procedure was used for purifying zalphal 1 polypeptide containing C-terminal fusion to human IgG/Fc (zalphall-Fc4; Examples 8 and 9). 12,000 ml of conditioned media from BHK 570 cells transfected vvdth zalphal 1-Fc4 (Example 10) was filtered through a 0.2 mm sterilizing filter and then supplemented with a solution of protease inhibitors, to final concentrations of, 0.001 mM leupeptin (Boerhinger-Mannheim, Indianapolis, IN), '0.001 mM pepstatin (Boerhinger-Mannheim) and 0.4 mM Pefabloc (Boerhinger-Mannheim). A protein G sepharose (6 ml bed volume, Pharmacia Biotech) was packed and washed with 500 ml PBS (Gibco/BRL) The supplemented conditioned media was passed over.the column with a flow rate of 10 ml/minute, follOowed by washing with 1000 ml PBS (BRL/Gibco). zalphal 1-Fc4 was eluted from the column with 0.1 M Glycine pH 3.5 and 2 ml fractions were collected directly into 0.2 ml 2M Tris pH 8.0, to adjust the final pH to 7.0 in the fractions.<br><br>
The eluted fractions were characterized by SDS-PAGE and western blotting with anti-human Fc (Amersham) antibodies. Western blot analysis of reducing SDS-PAGE gels reveal an immunoreactive protein of 80,000 KDa in fractions 2-10. Silver stained SDS-PAGE gels also revealed an 80,000 KDa zalpal l:Fc polypeptide in fractions 2-10. Fractions 2-10 were pooled.<br>
The protein concentration of the pooled fractions was performed by BCA analysis (Pierce, Rockford, IL) and the material was aliquoted, and stored at -80°C according to our standard procedures. The concentration of the pooled fractions was 0.26 mg/ml.<br>
Example 12 Assay using zalphal 1 soluble receptor zalphallCEE. zalphalICFLG and zalphal 1-Fc4<br>
Cmutanf) Soluble receptors in competitive inhibition assay<br>
BaF3/Zalphal 1 cells were spun down and washed in mIL-3 free media. The cells were spun and washed 3 times to ensure the removal of the mIL-3. Cells were then counted in a hemacytometer. Cells were plated in a 96-well format at 5000 cells per well in a volume of 100 \xl per well using the mIL-3 free media.<br>
Both media from the monkey spleen cell activation and the CD3+ selected cells, described in Example 8 above, were added in separate experiments at 50%, 25%, 12.5%, 6.25%, 3.125%, 1.5%, 0.75% and 0.375% concentrations, with or without zalphal 1 soluble receptors (CEE, C-flag, and Fc4 constructs; See, Example 10 and 11) at 10}j,g/ml. The total assay volume was 200 ^1.<br>
The assay plates were incubated 37°C, 5% CO2 for 3 days at which time Alamar Blue (Accumed) was added at 20)il/well. Plates were again incubated at 37°C, 5% CO2 for 24 hours. Plates were read on the Fmax™ plate reader (Molecular Devices) as described above (Example 5). Results demonsfrated complete inhibition of cell growth from each of the different zalphal 1 soluble receptor constructs at lO^g/ml, confirming that the factor in each sample was specific for the zalphal 1 receptor.<br>
Titration curves, diluting out the soluble receptors, were also run using the above stated assay.  Both the zalphallCEE and zalphal ICFLG soluble zalphal 1<br><br>
receptors were able to completely inhibit growth as low as 20 ng/ml. The mutant zalphall-Fc4 soluble'zalphal 1 receptor was only as effective at 1.5 μg/ml.<br>
Example 13 Expression of human zalphal 1 in E. coli A.   Construction of expression vector pCZR225 that expresses huzalphal 1/MBP-6H fusion polypeptide<br>
An expression plasmid containing a polynucleotide encoding a human zalphal 1 soluble receptor fused C-terminally to maltose binding protein (MBP) was constructed via homologous recombination. The polynucleotide sequence for the MBP-zalphall soluble receptor fiision polypeptide is shown in SEQ ID NO:50, with the corresponding protein sequence shown in SEQ ID N0:51. The fusion polypeptide, designated huzalphal 1/MBP-6H, in Example 14, contains an MBP portion (amino acid 1 (Met) to amino acid 388 (Ser) of SEQ ID N0:51) fused to the human zalphal 1 soluble receptor (amino acid 389 (Cys) to amino acid 606 (His) of SEQ ID N0:51). A fragment of human zalphal 1 cDNA (SEQ ID NO:52) was isolated using PCR. Two primers were used in the production of the human zalphal 1 fragment in a PCR reaction:<br>
(1)	Primer ZC20,187 (SEQ ID NO:53), containing 40 bp of the vector flanking<br>
sequence and 25 bp corresponding to the amino terminus of the human zalphall, and<br>
(2)	primer ZC20,185 (SEQ ID NO:54), containing 40 bp of the 3' end corresponding to<br>
the flanking vector sequence and 25 bp correspondmg to the carboxyl terminus of the<br>
human zalphal 1. The PCR Reaction conditions were as follows: 25 cycles of 94°C for<br>
30 seconds, SO'C for 30 seconds, and 72°C for 1 minute; followed by 4°C soak, run in<br>
dupUcate. Two ^1 of the 100 |il PCR reaction was run on a 1.0% agarose gel with 1 x<br>
TBE buffer for analysis, and the expected approximately 660 bp fragment was seen.<br>
The remaining 90 |il of PCR reaction was combined with the second PCR tube<br>
precipitated with 400 JAI of absolute ethanol. The precipitated DNA used for<br>
recombining into the Smal cut recipient vector pTAP98 to produce the construct<br>
encoding the MBP-zalphal 1 fusion, as described below.<br><br>
Plasmid pTAP98 was derived from the plasmids pRS316 and pMAL-c2. The plasmid pRS3I5 is a Sdccharomyces cerevisiae shuttle vector (Hieter P. and Sikorski, R., Genetics-122:19-27. 1989). pMAL-C2 (NEB) is an E. coti expression plasmid. It carries the tac promoter driving MalE (gene encoding MBP) followed by a His tag, a thrombin cleavage site, a cloning site, and the rmB tenninator. The vector pTAP98 was constructed using yeast homologous recombination. lOOng of EcoRl cut pMAL-c2 was recombined with l|ig Pvul cut pRS316, \\i% linker, and Ifig Scal/EcoRl cut pRS316. The linker consisted of oligos ZC 19,372 (SEQ ID NO:55) (100 pmol): ZC19,351 (SEQ ID NO:56) (1 pmol): ZC19,352 (SEQ ID NO:57) (1 pmol), and ZC19,371 (SEQ ID NO:58) (100 pmol) combined in a PCR reaction. PCR reaction conditions were as follows: 10 cycles of 94°C for 30 seconds, 50°C for 30 seconds, and 72°C for 30 seconds; followed by 4°C soak. PCR products were concentrated via 100% ethanol precipitation.<br>
One hundred microliters of competent yeast cells {S. cerevisiae) were combined with 10 μl of a mixture containing approximately 1 |ig of the human zalphall receptor PCR product above, and 100 ng of Smal digested pTAP98 vector, and transferred to a 0.2 cm electroporation cuvette. The yeast/DNA mixture was electropulsed at 0.75 kV (5 kV/cm), infinite ohms, 25 |iF. To each cuvette was added 600 |il of 1.2 M sorbitol and the yeast was then plated in two 300 |il aliquots onto two -URA D plates and incubated at 30°C.<br>
After about 48 hours, the Ura+ yeast transformants from a single plate were resuspended in 1 ml HjO and spun briefly to pellet the yeast cells. The cell pellet was resuspended in 1 ml of lysis buffer (2% Triton X-100, 1% SDS, 100 mM NaCl, 10 mM Tris, pH 8.0, 1 mM EDTA). Five hundred microliters of the lysis mixture was added to an Eppendorf tube containing 300 μl acid washed glass beads and 200 μ\ phenol-chloroform, vortexed for 1 minute intervals two or three times, followed by a 5 minute spin in a Eppendorf centrifuge at maximum speed. Three hundred microliters of the aqueous phase was transferred to a fresh tube, and the DNA precipitated with 600 μl ethanol (EtOH), followed by centrifugation for 10 minutes at 4°C. The DNA pellet was resuspended in 100 μl H2O.<br>
V<br><br>
Transformation of electrocompetent E. coli cells (MCI061, Casadaban et. al. J. Mol. Biol. T38. 179-207) was done with 1 μl yeast DNA prep and 40 μ1 of MCI061 cells. The cells were electropulsed at 2.0 kV, 25 μF and 400 ohms. Following electroporation, 0.6 ml SOC (2% Bacto™ Tryptone (Difco, Detroit, MI), 0.5% yeast extract (Difco), 10 mM NaCl, 2.5 mM KCl, 10 mM MgC12, 10 mM MgS04,20 mM glucose) was plated in one aliquot on MM/CA +AMP 100 mg/L plates (Pryor and Leiting, Protein Expression and Pruification 10:309-319,1997).<br>
Cells harboring the correct expression construct for human zalphall receptor were identified by expression. Cells were grown in MM/CA with 100 μg/ml Ampicillin for two hours, shaking, at 37°C. 1ml of the culture was induced with ImM IPTG. 2-4 hours later the 250 μl of each culture was mixed with 250 μl acid washed glass beads and 250 μl Thomer buffer with 5% pME and dye (8M urea, 100 mM Tris pH7.0, 10% glycerol, 2niM EDTA, 5% SDS). Samples were vortexed for one minute and heated to 65°C for 10 minutes. 20 μl were loaded per lane on a 4%-12% PAGE gel (NOVEX). Gels were run in IXMES buffer. The positive clones were designated pCZR225 and subjected to sequence analysis. The polynucleotide sequence of MBP-zalphal 1 fusion is shown in SEQ ID NO:50. B. Bacterial Expression of human huzalphal 1/MBP-6H fusion polypeptide<br>
One microliter of sequencing DNA was used to transform strain BL21. The cells were electropulsed at 2.0 kV, 25 μF and 400 ohms. Following electroporation, 0.6 ml MM/CA with 100 mg/L Ampicillm.<br>
Cells were grown in MM/CA with 100|μg/ml Ampicillin for two hours, shaking, at 37°C. 1ml of the culture was induced with ImM IPTG. 2-4 hours later the 250 \x\ of each culture was mixed with 250μl acid washed glass beads and 250 |il Thomer buffer with 5% pME and dye (8M urea, 100 mM Tris pH7.0, 10% glycerol, 2mM EDTA, 5% SDS). Samples were vortexed for one minute and heated to 65°C for 10 minutes. 20μ1 were loaded per lane on a 4%-12% PAGE gel (NOVEX). Gels were run in IXMES buffer. The positive clones were used to grow up for protein purification of the huzalphal 1/MBP-6H fusion protein (Example 14, below).<br><br>
Example 14 Purification of huzalphal 1/MBP-6H soluble receptor from E.coli fermentation<br>
Unless-otherwise noted, all operations were carried out at 4°C. The following procedure'Was used for purifying huzalphall/MBP-6H soluble receptor polypeptide. E. coli cells containing the pCZR225 construct and expressing huzalphal 1/MBP-6H soluble receptor (Example 13) were grown up in SuperBroth II (12 g/L Casien, 24 g/L Yeast Extract, 11.4 g/L di-potassium phosphate, 1.7 g/L Mono-potassium phosphate; Becton Dickenson, Cockeysville, MD), and frozen in 0.5% glycerol. Twenty grams of the frozen cells in SuperBroth II + Glycerol were used to purify the protein. The frozen cells were thawed and diluted 1:10 in a protease inhibitor solution (Exfraction buffer) prior to lysing the cells and releasing the huzalphal 1/MBP-6H soluble receptor protein. The diluted cells contained final concenfrations of 20 mM Tris (JT Baker, Philipsburg, NJ) 100 mM Sodium Chloride (NaCl, Mallinkrodt, Paris, KY), 0.5 mM pheynlmethylsulfonyl fluoride (PMSF, Sigma Chemical Co., St. Louis, MO), 2 (J.g/ml Leupeptin (Fluka, Switzerland), and 2 |ig/ml Aprotinin (Sigma). A French Press cell breaking system (Constant Systems Ltd., Warwick, UK) with temperature of -7 to -10°C and 30K PSI was used to lyse the cells. The diluted cells were checked for breakage by A500 readings before and after the French Press. The lysed cells were centrifuged @ 18,000G for 45 minutes to remove the broken cell debris, and the supernatant used to purify the protein. Total target protein concentrations of the supernatant was determined via BCA Protein Assay (Pierce, Rockford, IL), according to manufacturer's instructions.<br>
A 25 ml column of Talon Metal Affinity resin (Clontech, Palo Alto, CA) (prepared as described below) was poured in a Bio-Rad, 2.5 cm D x 10 cm H glass column. The column was packed and equilibrated by gravity with 10 column volumes (CVs) of Talon Equilibration buffer (20mM Tris, 100mM NaCl, pH 8.0). The supernatant was batch loaded to Talon metal affinity resin and was rocked overnight. The resin was poured back into the column and was washed with 10 CVs of Talon Equilibration buffer by gravity, then gravity eluted with 140 ml of Elution buffer (Talon Equilibration buffer + 200mM Imidazole-Fluka Chemical). The talon colunm was cleaned with 5 CVs of 20mM 2-(N-Morhpholino) ethanesulfonic acid pH 5.0<br><br>
(MES, Sigma), 5 CVs of distilled H2O, then stored in 20% Ethanol/0.1% Sodium Azide. Fourteen ml'fractions were collected over the entire elution chromatography and the fractions were'.read with absorbance at 280 and 320 nM and BCA protein assay; the pass through and'wash pools were also saved and analyzed. The protein elution fractions of mterest were pooled and loaded straight to Amylose resin (New England Biolabs, Beverly, MA),<br>
To obtain more pure huzalphal 1/MBP-6H polypeptide, the talon affinity elution pooled fractions were subjected to Amylose resin (22mls) at pH 7.4. A 2.5 cm D X 10 cm H Bio-Rad column was poured, packed and equilibrated in 10 CVs of Amylose equilibration buffer-20mM Tris (JT Baker), lOOmM NaCl (Mallinkrodt), ImM PMSF (Sigma), 10mM beao-Mercaptoethanol (BME, ICN Biomedicals Inc., Aurora, OH) pH 7.4. The sample was loaded by gravity flow rate of 0.5 ml/min. The column was washed for 10 CVs with Amylose equilibration buffer, then eluted with ~2 CV of Amylose equilibration buffer + 10 mM Maltose (Fluka Biochemical, Switzerland) by gravity. 5 ml fractions were collected over the entire chromatography and absorbance at 280 and 320 nM were read. The Amylose column was regenerated with 1 CV of distilled H2O, 5 CVs of 0.1% (w/v) SDS (Sigma), 5 CVs of distilled H2O, and then 5 CVs of Amylose equilibration buffer.<br>
Fractions of interest were pooled and dialyzed in a Slide-A-Lyzer (Pierce) with 4 x 4L PBS pH 7.4 (Sigma) to remove low molecular weight contaminants, buffer exchange and desalt. After the changes of PBS, the material harvested represented the purified huzalphal 1/MBP-6H polypeptide. The purified huzalphall/MBP-6H polypeptide was analyzed via SDS-PAGE Coomassie staining and Western blot analy$is with the anti-rabbit HRP conjugated antibody (Rockland, Gilbertsville, PA). The concentration of the huzalphal 1/MBP-6H polypeptide was 1.92 mg/ml as determined by BCA analysis.<br>
Purified huzalphal 1/MBP-6H polypeptide was prepared for injection into rabbits and sent to R &amp; R Research and Development (Stanwood, WA) for antibody production. Rabbits were injected to produce anti anti-huzalphal 1/MBP-6H serum (Example 15, below).<br><br>
Example 15 zalphal 1 Polyclonal Antibodies<br>
Polyclonal antibodies were prepared by immunizing two female New Zealand white rabbits with the purified huzalphal 1/MBP-6H polypeptide (Example 14), or the purified recombinant zalphal ICEE soluble receptor (Example UA). Corresponding polyclonal antibodies were designated rabbit anti-huzalphal 1/MBP-6H and rabbit anti-huzalphal 1-CEE-BHK respectively. The rabbits were each given an initial intraperitoneal (IP) injection of 200 mg of purified protein in Complete Freund's Adjuvant (Pierce, Rockford, IL) followed by booster IP injections of 100 mg purified protein in Incomplete Freund's Adjuvant every three weeks. Seven to ten days after the administration of the third booster injection, the animals were bled and the serum was collected. The rabbits were then boosted and bled every three weeks.<br>
The zalphal 1-specific polyclonal antibodies were affinity purified firom the rabbit serum using an CNBr-SEPHAROSE 4B protein column (Pharmacia LKB) that was prepared using 10 mg of the purified huzalphal 1/MBP-6H polypeptide (Example 14) per gram CNBr-SEPHAROSE, followed by 20X dialysis in PBS overnight. Zalphall-specific antibodies were characterized by an ELISA titer check using 1 mg/ml of the appropriate protein antigen as an antibody target. The lower limit of detection (LLD) of the rabbit anti-huzalphal 1/MBP-6H affinity piirified antibody is a dilution of 500 pg/ml. The LLD of the rabbit anti-huzalphal 1-CEE-BHK affinity purified antibody is a dilution of 50 pg/ml.<br>
Example 16 Identification of cells expressing zalphal 1 receptor using RT-PCR<br>
Specific human cell types were isolated and screened for zalphall expression by RT-PCR. B-cells were isolated from fi'esh human tonsils by mechanical disruption through 100 ^m nylon cell strainers (Falcon™; Bectin Dickenson, Franklin Lakes, NJ). The B-cell suspensions were enriched for CD 19+ B-cells by positive selection with VarioMACS VS+ magnetic column and CD 19 microbeads (Miltenyi Biotec, Auburn, CA) as per manufacturer's instructions. T-cells and monocytes were<br><br>
isolated from human apheresed blood samples. CD3+ T-cells were purified by CDS microbead VarioMACS positive selection and monocytes were purified by VarioMACS negativp.selection columns (Miltenyi) as per manufacturer's instructions. Samples from each population were stained and analyzed by fluorescent antibody cell sorting (FACS) (Bectin Dickinson, San Jose, CA) analysis to determine the percent enrichment and resulting yields. CD 19+ B-cells were approximately 96% purified CD3+ T-cells were approximately 95% purified, and monocytes were approximately 96% purified.<br>
RNA was prepared, using a standard method in the art, from all three cell types that were either resting or activated. RNA was isolated from resting cells directly from the column preparations above. The CD 19+ and CD3+ cells were activated by culturing at 500,000 cells/ml in RPMI + 10%FBS containing PMA 5ng/ml (Calbiochem, La JoUa, CA) and lonomycin 0.5ug/ml (Calbiochem) for 4 and 24 hours. The monocytes were activated by culturing in RPMI + 10% FES containing LPS lOng/ml (Sigma St. Louis MO) and rhIFN-g 10 ng/ml (R&amp;D, Minneapolis, MN) for 24 hours. Cells were harvested and washed in PBS. RNA was prepared from the cell pellets using RNeasy Midiprep™ Kit (Qiagen, Valencia, CA) as per manufacturer's instructions and first strand cDNA synthesis was generated with Superscript II™ Kit (GIBCO BRL, Grand Island, NY) as per manufacturers protocol.<br>
Oligos ZC19907 (SEQ ID NO:20) and ZC19908 (SEQ ID N0:21) were used in a PCR reaction to screen the above described samples for a 1.2 kb. fragment corresponding to zalphall message. PCR amplification was performed with Taq Polymerase (BRL Grand Island NY), and conditions as follows: 35 cycles of 95°C for 1 min., 60°C for 1 min., 72°C for 30 sec; 1 cycle at 12°C for 10 min.; and 4'C soak. lOul of each 50 jil reaction volume was run on a 2% agarose IXTAE gel to identify resultant products. PCR products were scored as (-) for no product, (+) for band visible, (++) increased presence of band and (+++) being the most predominant band, with results shown in Table 5 below.<br><br><br>
These results indicated that zalphall message is present in resting human CD 19+ B-cells and increases with mitogenic activation. It also appears to be expressed by human CD3+ T-cells only after 4 hour activation. There was no apparent message in either resting or activated human monocytes.<br>
Example 17 Zalphal 1 Immunohistochemistrv<br>
A. Cell and tissue preparations<br>
Positive control tissues consisted of BaF3 cells transfected with zalphall (Example 7) and lymphoid tissues known to express zalphall including mouse lymph node, spleen and thymus received from HSD (Harlan Sprague Dawley, Indianapolis, IN), monkey lymph node and spleen received from Regional Primate Research Center (University of Washington, Seattle, WA), human lymph node and spleen received from CHTN (Cleveland, OH). Negative controls performed on each tissue sample included: (1) untransfected BaF3 cells, (2) liver and brain from mouse and human known not to express zalphall, (3) staining with antibody dilution buffer (Ventann Bioteck Systems, Tucson AZ) in the absence of primary antibody, and (4) using zalphal 1 soluble protein in competition experiments.<br><br>
Other cell samples were examined. Both non-stimulated and stimulated HL60 cells were assayed. HL60 cells are a.promyelocytic cell line, which can be differentiated into myeloid or granulocyte lineages with different reagents. Stimulated HL60 samples were prepared as follows: (1) HL60 cells were treated with l0ng/ml of phorbol-myristate-acetate (PMA) (Sigma, St. Louis, MO) for 48 hours to differentiate into monocyte lineage cells; and (2) HL60 cells treated with 1.25% DMSO (Sigma) for 4 days to differentiate into neutrophil-like cells. In addition, human polymorphonuclear (PMN) cells, human granulocytes, human peripheral blood lymphocytes (PBL) and human monocytes from fresh himian blood were examined (prepared in house using routine methods in the art). The cells and tissues described above were fixed overnight in 10% NBF (Surgipath, Richmond, IL), and embedded in parapalst X-fra (Oxford Scientific, St. Louis, MO), and sectioned at 5 |im with a Reichart-Jung 2050 microme (Leica Instnmients GmbH, Nussloch, Germany).<br>
B. Immunohistochemistrv<br>
Tissue slides were deparaffinized, hydrated to buffer (water), and subjected to steam HIER treatment in Antigen Retrieval Citra buffer (BioGenex, San Roman, CA) for 20 minutes. 5% normal goat serum (Vector, Burlingame, CA) was used to block non-specific binding for 10 minutes. Immunocytochemical screening analyses were performed using polyclonal antibodies to zalphall soluble receptor protein (rabbit anti-huzalphall-MBP-6H and rabbit anti-huzalphall-CEE-BHK; see. Example 15) as the primary antibodies, at dilutions of 1:200 and 1:400 respectively. Biotin conjugated goat anti-rabbit IgG (Vector; Cat. No. BA-1000, 1.5 mg/ml) was used as the secondary antibody at dilution of 1:200. In separate samples, protein competition was performed by using additional zalphallCEE soluble receptor protein (in 10X fold excess) (Example 11 A) to the primary antibody to pre-block primary antibody immunoreaction. This competition was used as a control for the rabbit polyclonal antibody specificity to zalphall. Detection was performed on the Ventana ChemMate 500 instrument using a ChemMate DAB Kit (labeled Streptavidin-Biotin Kit with application of a streptavidin-horseradish peroxidase conjugate, and DAB<br><br>
substrate) according to 'manufacturer's instruction and using the manufacturer's hematoxylin counterstain for 30 seconds (Ventana Biotek Systems, Tucson, AZ).<br>
High expression of 2alphal 1 was observed in the PMA-activated HL60 cells. Low level expression was observed in PBL and HL60 cells without stimulation. A subset of cells in the spleen, thymus and lymph node of mouse showed positive staining. Lymph node and spleen of both human and monkey, and HL60 cells with DMSO stimulation showed minimal or no staining. The signal seen in the cells and tissues was mostly competed out by using the excess zalphal 1 soluble receptor protein. The negative control tissues of brain and liver showed no staining.<br>
Example 18<br>
Identifying Peripheral Blood Mononuclear Cells (PBMNC's) that express zalphal 1<br>
receptor using polvclonal rabbit anti-sera to zalphal 1 soluble receptor<br>
200 ml fresh heparinized blood was obtained from a normal donor. Blood was diluted 1:1 in PBS, and separated using a Ficoll-Paque PLUS gradient (Pharmacia Biotech, Uppsala, Sweden), and the lymphocyte interface collected. Cells were washed 2X in PBS and resuspended in RPMI + 5% FBS media at a concentration of 2X10 cells/ml.<br>
In order to determine whether expression of zalphal 1 receptor is affected by the activation state of the lymphocyte cells, i.e., between resting and activated cells several stimulation conditions were used: 1) un-stimulated, i.e., media alone (RPMI + 5% FBS media); 2) stimulated with PMA 10 ng/ml + lonomycin 0.5 μg/ml (both from Calbiochem); and 3) PHA activation (phytohemagglutinin-P, DifcoA^WR). The cells were iacubated at 37°C for 17 hours then collected for staining to detect expression of zalphal 1 receptor.<br>
An indirect staining protocol was used. Briefly, the human lymphocyte cells were suspended in staining buffer (PBS + 0.02% NaN3 + BSA 1% normal human serum 2%) and plated at 2X10^ cells in 50 μl/well in a 96 well plate. Antibodies to the zalphal ICEE soluble receptor (Example 15) were used to determine whether they co-stained with a B-cell (CD-19), T-cell (CD-3) or monocyte marker (CD-14) on the<br><br>
isolated human lymphocytes. A rabbit polyclonal sera to zalphal 1 soluble receptor (Rb anti-huzalphall-CEE-BHK) (Example 15) at 10 μg/ml was used as the antibody to identify zalphal 1 on the lymphocytes. A secondary antibody, goat anti-rabbit Ig-FITC (Biosource, Camarilla, CA), was used to visualize the Rb anti-huzalphall-CEE-BHK antibody binding to the zalphal 1 receptors. Other antibodies were simultaneously used to stain T cells (CD3-PE; PharMingen, San Diego, CA), B cells (CD19-PE) (PharMingen), and monocsrtes (CD-14-PE) (PharMingen) in order to identify co-staining of the anti-zalphal 1 receptor antibody on these cell types. Various controls were used to determine non-specific binding and background levels of staining: (1) an irrelevant rabbit polyclonal sera was used as a non-specific control; and (2) secondary antibody alone was used to determine background binding of that reagent. Purified, zalphal ICEE soluble receptor (Example 11) was used in about a 10-fold excess as a competitive inhibitor to verify the specificity of the rabbit anti-huzalphall-CEE-BHK antibody to zalphal 1 soluble receptor.<br>
After plating the cells and adding the primary and co-staining antibodies, the cells were incubated on ice for 30 minutes, washed 2X with staining buffer, and stained with the secondary antibody, goat anti-rabbit Ig-FITC (Biosource), for 30 minutes on ice. Cells were washed 2X staining buffer, and resuspended at 200 μl per well in staining buffer containing the viability stain 7AAD at about 1 μg/ml final concentration (Sigma, St. Louis, MO). Samples were read on the FACS-Caliber (Becton-Dickinson, San Jose, CA) and viable cells analyzed.<br>
The rabbit polyclonal to zalphal 1 receptor stained resting B cells. The signal on resting B cells was brighter than the signal achieved using the irrelevant rabbit sera, and the signal was diminished to a greater extent on B cells than on T cells with the "addition of excess zalpHall-CEE soluble receptor. This experiment was repeated using separated B and T cells, and the results were very similar. Again the staining with the polyclonal rabbit anti-huzalphall-CEE-BHK antibody to zalphal 1 receptor was highest on resting B cells.<br><br>
Example 19 Zalphall  receptor expression in various tissues using Real-Time Quantitative RT/PCR/<br>
A. Primers and Probes for Quantitative RT-PCR-<br>
Real-time quantitative RT-PCR using the ABI PRISM 7700 Sequence Detection System (PE Applied Biosystems, Inc., Foster City, CA) has been previously described (See, Heid, CA. et al.. Genome Research 6:986-994, 1996; Gibson, U.E.M. et al.. Genome Research 6:995-1001, 1996; Simdaresan, S. et al., Endocrinoloev 139:4756-4764, 1998. This method incorporates use of a gene specific probe containing both reporter and quencher fluorescent dyes. When the probe is intact the reporter dye emission is negated due to the close proximity of the quencher dye. During PCR extension using additional gene-specific forward and reverse primers, the probe is cleaved by 5' nuclease activity of Taq polymerase which releases the reporter dye fi:om the probe resulting in an increase in fluorescent emission.<br>
The primers and probes used for real-time quantitative RT-PCR analyses of zalphall expression were designed using the primer design software Primer Express™ (PE Applied Biosystems, Foster City, CA), Primers for human zalphal 1 were designed spanning an intron-exon junction to eliminate amplification of genomic DNA. The forward primer, ZC22,277 (SEQ ID NO:59) and the reverse primer, ZC22,276 (SEQ ID NO:60) were used in a PCR reaction (below) at about 300 nM concentration to synthesize a 143 bp product. The corresponding zalphal 1 TaqMan® probe, designated ZG31 (SEQ ID N0:61) was synthesized and labeled by PE Applied Biosystems. The ZG31 probe was labeled at the'5'end with a reporter fluorescent dye (6-carboxy-fluorescein) (FAM) (PE Applied Biosystems) and at the 3' end with a quencher fluorescent dye (6-carboxy-tetramethyl-rhodamine) (TAMRA) (PE Applied Biosystems).<br>
As a control to test the integrity and quality of RNA samples tested, all RNA samples (below) were screened for rRNA using a primer and probe set ordered from PE Applied Biosystems (cat No. 4304483).  The kit contains an rRNA forward<br><br>
primer (SEQ ID NO:66) and the rRNA reverse primer (SEQ ID NO:67), rRNA TaqMan® probe (SEQ ID NO:68) The rRNA probe was labeled at the 5'end with a reporter fluorescent dye. VIC (PE Applied Biosystems) and at the 3' end with the quencher fluorescent dye TAMRA (PE Applied Biosystems). The rRNA results also serve as an endogenous control and allow for the normalization of the zalphal 1 mRNA expression results seen in the test samples.<br>
RNA samples from human CDS, CD 19 and monocyte cell types were prepared and described as per Example 16 above. Control RNA was prepared, using RNeasy Miniprep™ Kit (Qiagen, Valencia, CA) as per manufacturer's instructions, from approximately 10 million BaF3 cells expressing human zalphal 1 receptor (Example 7).<br>
B. Real-time quantitative RT-PCR-<br>
Relative levels of zalphal 1 mRNA were determined by analyzing total RNA samples using the one-step RT-PCR method (PE Applied Biosystems). Total RNA from BaF3 cells expressing human zalphal 1 receptor was isolated by standard methods and used to generate a standard curve used for quantitation. The curve consisted of 10-fold serial dilutions ranging from 2.5-2.5x10 ng/μ,l for the rRNfA screen and 250-0.025 ng/)al for the zalphal 1 screen with each standard curve point analyzed in triplicate. The total RNA samples from the human CDS, CD 19 and monocyte cells were also analyzed in tripUcate for human zalphal 1 transcript levels and for levels of rRNA as an endogenous control. In a total volume of 25 )J.1, each RNA sample was subjected to a One-Step RT-PCR reaction containing: approximately 25 ng of total RNA in buffer A (50 mM KCL, 10 mM Tris-HCL); the internal standard dye, carboxy-x-rhodamine (ROX)); appropriate primers (approximately 50 nM rRNA primers (SEQ ID NO:66 and SEQ ID NO:67) for the rRNA samples; and approximately 300 nM ZC22,277 (SEQ ID NO:59) and ZC22,276 (SEQ ID NO:60) primers for zalphal 1 samples); the appropriate probe (approximately 50 nM rRNA TaqMan® probe (SEQ ID NO:68) for rRNA samples, approximately 100 nM ZG31 (SEQ ID N0:61) probe for zalphal 1 samples); 5.5 mM MgCl2; 300 μM each d-CTP, d-ATP, and d-GTP and 600 μM of d-UTP; MuLV reverse transcriptase (0.25 U/μ1);<br><br>
AmpliTaq™ Gold DNA polymerase (0.025 U/μl) (PE Applied Biosystems); and RNase Inhibitor (0.4 U/μl) (PE Applied Biosystems). PCR thermal cycling conditions were as follows: an initial reverse transcription (RT) step of one cycle at 48°C for 30 minutes; followed by an AmpliTaq Gold™ (PE Applied Biosystems) activation step of one cycle at 95°C for 10 minutes; followed by 40 cycles of amplification at 95°C for 15 seconds and 60°C for 1 minute.<br>
Relative zalphal 1 RNA levels were determined by using the Standard Curve Method as described by the manufacturer, PE Biosystems (User Bulletin No.2: ABI Prism 7700 Sequence Detection System, Relative Quantitation of Gene Expression, December 11,1997). The rRNA measurements were used to normalize the zalphal 1 levels and the resting CD3+ RNA sample was used as a calibrator. Resting CDS was arbitrarily chosen as the calibrator and given a value of 1.00. The rest of the samples were compared relative to the calibrator. Data are shown in Table 6 below.<br><br>
There was a 15-fold increase in zalphal 1 receptor expression in CD34- at 4 and 24 hrs. Resting CD 19 had 20 fold increase in receptor expression relative to resting CD3+. There was a 3 fold increase with 4 hr stimulation that fell back to resting levels by 24 hrs. Monocytes showed no detectable zalphal 1 receptor expression in this assay.<br>
Example 20 Identification of cells expressing zalphal 1 receptor using in situ hybridization<br>
Specific human tissues were isolated and screened for zalphall expression by in situ hybridization. Various human tissues prepared, sectioned and subjected to in situ hybridization included thymus, spleen, tonsil, lymph node and lung.<br><br>
The tissues were fixed in 10% buffered formalin and blocked in paraffin using standard techniques (Example' 17). Tissues were sectioned at 4 to 8 microns. Tissues were prepared using a standard protocol ("Development of non-isotopic in situ hybridization" at http'//dir.niehs.nih,gov/dirlep/ish.html). Briefly, tissue sections were deparafFmized with HistoClear (National Diagnostics, Atlanta, GA) and then dehydrated with ethanol. Next they were digested with Proteinase K (50 fig/ml) (Boehringer Diagnostics, Indianapolis, IN) at 37°C for 2 to 20 minutes. This step was followed by acetylation and re-hydration of the tissues.<br>
Two in situ probes generated by PCR were designed against the human zalphal 1 sequence. Two sets of oligos were designed to generate probes for separate regions of the zalphal 1 cDNA: (1) Oligos ZC23,684 (SEQ ID NO:62) and ZC23,656 (SEQ ID NO:63) were used to generate a 413 bp probe for zalphal 1; and (2) OUgos ZC23,685 (SEQ ID NO:64) and ZC23,657 (SEQ ID NO:65) were used to generate a 430 bp probe for zalphal 1. The second probe is 1500 bp 3' of the first zalphal 1 probe. The antisense oligo firom each set edso contained the working sequence for the T7 RNA polymerase promoter to allow for easy transcription of antisense RNA probes fi:om these PCR products. The PCR reaction conditions were as follows: 30 cycles at 94°C for 30 sec, 60°C for 1 min., 72°C for 1.5 mm. The PCR products were purified by Qiagen spin columns followed by phenol/chloroform extraction and ethanol precipitation. Probes were subsequentiy labeled vnih digoxigenin (Boehringer) or biotin (Boehringer) using an In Vitro transcription System (Promega, Madison, WI) as per manufacturer's instruction.<br>
In situ hybridization was performed with a digoxigenin- or biotin-labeled zalphal 1 probe (above). The probe was added to the slides at a concentration of 1 to 5 pmol/ml for 12 to 16 hours at 55-60°C. Slides were subsequentiy washed in 2XSSC and O.IXSSC at 50°C. The signals were amplified using tyramide signal amplification (TSA) (TSA, in situ indirect kit; NEN) and visualized with Vector Red substrate kit (Vector Lab) as per manufacturer's instructions. The slides were then counter-stained with hematoxylin (Vector Laboratories, Burlingame, CA).<br>
A signal was seen in the thymus, tonsil, lung, and lymph node. The positive-staining cells appeared to be lymphocytes and related cells.<br><br>
Example 21 Isolation of the mouse Zalphal 1 Receptor<br>
A. Mouse genomic library screen<br>
An initial partial mouse zalphal 1 sequence was obtained by probing a mouse genomic library with a human zalphal 1 receptor polynucleotide probe containing the entire cDNA. The hiunan zalphal 1 cDNA was generated by PCR with ZC19,905 (SEQ ID NO:36) and ZC19,906 (SEQ ID NO:37) primers and a plasmid containing full length human zalphal 1 (e.g., Example 1) was used for the template. The PCR reaction conditions were as follows: 35 cycles at 98°C for 1 min., 68°C for 1 min., and 72°C for 2 min.; followed by one cycle at 72°C for 10 min.. The PCR product was run on a 1% low melting point agarose (Boerhinger Mannheim) and the approximately 1.5 kb human zalphal 1 cDNA isolated using Qiaquick"™ gel extraction kit (Qiagen) as per manufacturer's instructions. This human zalphal 1 cDNA was used to screen a mouse genomic DNA library (below).<br>
The mouse genomic DNA library used was embl3 SP6/T7 lambda BamHI cloned library (Clontech, Palo Alto, CA). This library representing 7.2 x 105 pfu was plated onto an E. coli K802 host lawn on 24 NZY plates. Plaque lifts were performed using Hybond-N filters (Amersham Pharmacia, Buckinghamshire, England, UK) as per manufacturer's instructions. The filters were denatured in 1.5 M NaCl and 0.5 M NaOH for 10 min. and then neutralized in 1.5 M NaCl and 0.5 M Tris-HCL (pH 7.2 ) for 10 min.. The DNA was affixed to the filter using a STRATALINKER UV crosslinker (Stratagene) at 1200 joules. The filters were pre-washed to remove cell debris at 65°C in pre-wash buffer (0.25 x SSC, 0.25% SDS and 1 mM'EDTA), changing solution three times for a total of 45 min. The filters were prehybridized overnight at 50°C in Expresshybtm solution (Clontech) containmg 0.1 mg/ml denatured salmon sperm DNA. Approximately 50 ng of the purified hiunan zalphal 1 cDNA (above) was labeled with "P using the Rediprime II Random Prime Labeling System (Amersham Pharmacia) as per manufacturers instructions. Unincorporated radioactivity was removed from the zaplhal 1 cDNA probe using a NucTrap™ push<br><br>
column (Stratagene, La JoUa, CA). Filters were hybridized in Expresshyb™ solution (Clontech) containing about 0.5 to about 1 x 106 cpm/ml zalphall cDNA probe, about 0.1 mg/ml denatured: salmon sperm DNA and denatured 0.5 μg/ml cot-1 DNA. Hybridization took place overnight at 50°C. Filters were washed in 2x SSC, 0.1% SDS at room temperature for 2 hours (changing wash several times) then the temperature was raised to 60°C for one hour (changing buffer once). Overnight exposure at -80°C showed 6 plaques representing primary isolates.<br>
To obtain secondary plaque isolates, the 6 plaques representing primary isolates were picked with a Pasteur pipette and eluted overnight at 4°C in 1 ml SM (0.1 M NaCl, 50mM Tris pH 7.5, lOmM MgS04, 0.02% gelatin) containing a few drops of chloroform. After determining phage titers, about 12.5X the estimated amount of phage in the origuial plug (12.5X coverage) of 6 primary isolates was plated on a lawn of £. coli K802 cells embedded in 10mMMgSO4/NZY top agarose on NZY maxi plates, and grown overnight at 37°C. Plaque lifts were done using Hybond-N filters (Amersham Pharmacia) as per manufacturer's instructions. Filters were fixed as per above. The second roxmd filters were pre-washed to remove cell debris at 65°C in pre-wash buffer (2 x SSC, 0.1% SDS and 1 mM EDTA), changing solution three times for a total of 45 min.. The second round filter lifts were then prehybridized, and the zalphal 1 cDNA probe prepared as described above.<br>
The second round filters were hybridized as above in Expresshyb™ solution (Clontech) containing about 106 cpm/ml zalphal 1 cDNA probe containing about 0.1 mg/ml denatured salmon sperm DNA. Hybridization took place overnight at 50°C. Wash conditions described above for the primary screen were repeated for this secondary screen. After an overnight exposure at -80°C, two of the 6 original primary plaques isolates were'verified as positive in the secondary screen. Positive plaques hybridizing to human zalphall cDNA in the secondary screen were picked with a Pasteur pipette and designated 7b 1 and 20b 1.<br>
The isolated plaques No.7bl and 20b 1 were eluted in 200 μl SM overnight at 4 °C. Serial 10-fold serial dilutions ranging firom 10-2 to 10-6 of each isolate were plated on host E. coli K802 cells to determine the titer. Isolate 20b 1 had a titer of 4 X 103 pfu/μl and was fiarther pursued.  4 plates were prepared by plating 10'<br><br>
pfu/plate on confluent ho'st E, coli K802 cells in order to make a phage DNA prep. Plates were grown at 37°C for about 6.5 hours until phage lysis was starting to get confluent. The phage was then eluted overnight at 4°C in 12 ml of SM per plate. Plates were then shaken at room temperature one hour, the supernatant was removed; 1% chloroform was added and supernatant was shaken again for 15 min.. The 20b 1 phage DNA was prepped using the Wizard Lambda Preps DNA Purification System (Promega, Madison, WI; sections IV and VI).<br>
Samples of 20b 1 phage DNA were cut with several restriction enzymes to generate DNA fragments for Southern blotting. The digests were run on a 1% TBE agarose gel. The gel was soaked in 0.25 M HCl for 30 min.; rinsed in distilled H20; soaked in 0.5M NaOH and 1.5 M NaCl for 40 min. with one solution change and neutralized in 1.5 NaCl and 0.5 Tris-HCL (pH 7.2) for 40 min. with one solution change. A TURBOBLOTTER™ Rapid Downward Transfer System (Schleicher &amp; Schuell, Keene, NH) was set up to transfer the DNA onto a Nytran/BA-S membrane (Schleicher &amp; Schuell) overnight. The DNA was affixed to the Nytran using a STRATALINKER UV crosslinker (Stratagene, La JoUa, CA) at 1200 joules. The blot was prehybridized as described above. About 50 ng of the hmnan zalphal 1 cDNA was labeled and purified for a probe, as described above. Filters were hybridized as above in Expresshyb™ solution (Clontech) containing about 106 cpm/ml zalphal 1 cDNA probe and about 0.1 mg/ml denatured salmon sperm DNA. Hybridization took place overnight at 50°C. The blot was washed as described above and exposed to film overnight at -80°C.<br>
The Southern showed a DNA fragment generated from a BamHI/StuI digest which hybridized to the human zalphal 1 cDNA probe in the expected size range of 1.3 to 1.6 kb. This fiagment was pursued. Approximately 3 μg of 20bl lambda DNA was cut with 20 units of BamHI (Boehringer Mannheim, Indianapolis, IN ) and 20 Units StuI (NEB, Beverly, MA) for 2 hours at 37°C. The digest was run on a 1% TBE gel and a 1.3 kb doublet and 1.6 kb doublet bands were excised firom the gel and the DNA was extracted from the agarose using the Qiaquick Gel Extraction Kit (Qiagen, Valencia, CA). Due to the low yield of DNA firom the prep, it was not possible to determine by additional restriction digest analysis whether fragments which<br><br>
hybridized to the human zalphall cDNA probe were BamHI/StuI or StuI/StuI fragments. Thus, blunt ligations using 5 μl of the 1.3 kb doublet fragment and 5 μl of the 1.6 kb doublet .fragment were done usmg the Zero Blunt PCR Cloning Kit (Invitrogen, Carlsbad/CA). The blunt ligation yielded positive clones with both of the 1.6 kb fragments and one of the 1.3 kb fragments. These clones were digested with EcoRI (Life Technologies) which flanks the T-overhang site where the 1.6 and 1.3 kb fragments were inserted. Another Southern blot was performed to determine which was the original fragment hybridizing to the human zalphal 1 cDNA probe. The 1% TEE gel was treated and the DNA was transferred to the Nytran blot as described above.<br>
The blot was prehybridized as above in 10 ml of hybridization solution. A different human zalphal 1 polynucleotide probe was prepared. Another frill length zalphal 1 cDNA human zalphal 1 fragment was generated for use as a probe by PCR with the oligos ZC 19,905 (SEQ ID NO: 36) and ZC20,097 (SEQ ID NO: 27). The PCR reaction conditions were as follows: 95oC for 1 min.; 35 cycles of 95°C for 1 min., 55°C for 1 min., and 72°C for 2 min.; followed by one cycle at 72°C for 10 min.. The PCR product was run on a 1% low melting point agarose (Boerhinger Mannheim) and the approximately 1.5 kb human zalphall cDNA isolated using Qiaquick"™ gel extraction kit (Qiagen) as per manufacturer's instructions. About 50 ng of this isolated human zalphal 1 cDNA fragment was labeled with 32P and purified as described above. Filters were hybridized as above in Expresshyb™ solution (Clontech) containing about 106 cpm/ml zalphall cDNA probe, about 0.1 mg/ml denatured salmon sperm, and denatured 0.5 μ,g/ml cot-1 DNA. Hybridization and washing was as described above. The blot was exposed to film 1.5 hours at -80°C and the 1.3 kb insert was strongly<br>
t<br>
hybridizing to the human zalphal 1 probe.<br>
This clone was sequenced and fovmd to contain a mouse zalphall 3' coding exon with a termination codon and upstream intron sequence. Sequencing primers used were: ZC3,424 (SEQ ID NO:86), ZC694 (SEQ ID NO:87), ZC24,399 (SEQ ID NO:88), and ZC24,400 (SEQ ID NO:89). The genomic sequence of mouse zalphall including the 3' exon is shown in SEQ ID NO:69. The the 3' exon coding<br><br>
sequence starts at nucleotide 543 and ends at nucleotide 1262 in SEQ ID NO:69, encoding a 240 amino acid exon (SEQ ID N0:70).<br>
B.	PCR screen of mouse cDNA panel<br>
A panel of available in-house and commercial mouse cDNAs (Clontech; Life technologies, Gaithersburg, MD) was screened by PCR using ZC24,432 (SEQ ID N0:71) and ZC24,433 (SEQ ID NO:72) as primers (about 20 pmol each). The PCR reaction conditions were as follows: 94°C, 2 min.; 32 cycles of 94°C for 20 sec, 64°C for 30 sec, and 72°C for 30 sec; followed by one cycle at 72°C for 5 min.. Mouse spleen, dendritic cells, neonatal skin, bone marrow, wild type BaF3 cells, EL4 cells, and lung showed strong PCR products of the predicted 450 bp size.<br>
C.	5'nested RACE<br>
5' RACE reactions were performed using 20 pmol each of primers ZC9,739 (SEQ ID NO:73) and ZC24,434 (SEQ ID NO:74) and CD90+ selected mouse spleen marathon cDNA as a template. The marathon cDNA was prepared using a Marathon cDNA Amplification Kit (Clontech) according to the manufacturer's instructions. The PCR reaction conditions were as follows: 94°C for 1 min.; 5 cycles of 94°C for 20 sec, and 70°C for 1.5 min.; followed by 25 cycles of 94oC for 20 sec, 64°C for 20 sec, and 70^ for 1.5 min.; followed by one cycle at 72°C for 5 min..<br>
To enrich for mouse zalphall 5' RACE product, a nested 5' RACE reaction was performed using PCR reaction conditions as described above for the initial 5'RACE, except using nested primers ZC24,431 (SEQ ID NO:75) and ZC9,719 (SEQ ID NO:76), and one μl of a 1/20 dilution of the initial 5' RACE reaction (above) as a template. The products were purified by gel electrophoresis, the DNA was'eluted using the Qiaex II Agarose Gel Extraction Kit (Qiagen) and subcloned using the TOPO TA Cloning Kit (Invitrogen). Positive clones were identified by colony PCR using 10 pmol each of ZC24,431 (SEQ ID NO:75) and ZC24,511 (SEQ ID NO:77). The PCR reaction conditions were as follows: 94°C, 2 min.; 35 cycles of 94°C for 20 sec, 64°C for 20 sec, and 72°C for 30 sec; followed by one cycle at 72°C for 5 min.. Two subclones from each of the nested 5'RACE reactions were sequenced. All the clones contained<br><br>
some zalphal 1 sequence but none were complete. A compiled sequence was generated from the incomplete'5'RACE clones and the 3' exon sequence (SEQ ID NO:70) representing a prelinmiary. partial sequence of the mouse zalphal 1 polynucleotide and corresponding polypeptide. The preliminary sequence of the partial mouse zalphal 1 cDNA is show in SEQ ID NO:78 (5'end) and SEQ ID NO:80 (3'end); there was approximately 330 nucleotides of yet unknown sequence between SEQ ID NO:78 (5'end) and SEQ ID NO:80 (3'end) to comprise the entire mouse zalphall cDNA (see below). The corresponding amino acid sequences for SEQ ID NO:78 and SEQ ID NO:80 are shown in SEQ ID NO:79 (N-terminus) and SEQ ID N0:81 (C-terminus) respectively.<br>
D. Full Length PCR<br>
Primers were designed from the mouse upstream UTR of the initiation Met and downstream of the termination codon for full length PCR. 20 pmol each of primers ZC24,616 (SEQ ID NO:82) and ZC24,615 (SEQ ID NO:83) were used in PCR reactions using a mouse dendritic cell marathon cDNA or a neonatal mouse skin in-house cDNA library as a template. PCR reaction conditions were as follows: 94°C, 1 min.; 30 cycles of 94°C for 20 sec, and 66°C for 2 min.; followed by one cycle at 72°C for 5 min.. The PCR products were purified by gel electrophoresis, and the cDNA was eluted using the Qiaquick Gel Extraction Kit and subcloned using the TA Cloning Kit (Invitrogen). 2 subclones from each PCR reaction were sequenced. Sequencing primers used were: ZC694 (SEQ ID NO:87), ZC3,424 (SEQ ID NO:86), ZC24,431 (SEQ ID NO:75), ZC24,511 (SEQ ID NO:77), ZC24,806 (SEQ ID NO:90), and ZC24,807 (SEQ ID N0:91). The sequence of the full length mouse zalphall cDNA is<br>
t<br>
show in SEQ ID NO:84. The corresponding amino acid sequence is shown in SEQ ID NO:85.<br>
From the foregomg, it will be appreciated that, although specific embodiments of the invention have been described herein for purposes of illustration, various modifications may be made without deviating from the spirit and scope of the invention. Accordingly, the invention is not limited except as by the appended claims.<br><br><br>
WE CLAIM:<br>
1.	An isolated polynucleotide that encodes a zalpha 11 polypeptide consisting of a<br>
sequence of amino acid residues selected from the group consisting of:<br>
(a)	the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 20 (Cys), to amino acid number 237 (His);<br>
(b)	the amino acid sequence as shown in SEQ ID NO:2 from amino acid number 20 (Cys), to amino acid number 255 (Leu);<br>
(c)	the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 256 (Lys), to amino acid number 538 (Ser); and<br>
(d)	the amino acid sequence as shown in SEQ ID N0:2 from amino acid number 20 (Cys), to amino acid number 538 (Ser).<br><br>
2.	The isolated polynucleotide as claimed in claim 1, wherein the polypeptide comprises a WSWSX domain.<br>
3.	The isolated polynucleotide as claimed in claim 1, wherein the polypeptide comprises a transmembrane domain.<br>
4.	The isolated polynucleotide as claimed in claim 3, wherein the transmembrane domain consists of residues 238 (Leu) to 255 (Leu) of SEQ ID N0:2.<br>
5.	The isolated polynucleotide as claimed in claim 1, wherein the polypeptide comprises an intracellular domain.<br>
6.	The isolated polynucleotide as claimed in claim 5, wherein the intracellular domain consists of residues 256 (Lys) to 538 (Ser) of SEQ ID NO:2.<br><br>
7.	The isolated polynucleotide as claimed in claim 6, wherein the intracellular domain comprises Box I and Box II sites.<br>
8.	The isolated polynucleotide as claimed in claim 1, wherein the polypeptide comprises an affinity tag.<br>
9.	An expression vector comprising the following operably linked elements:<br>
a transcription promoter;<br>
a DNA segment encoding a zalpha 11 polypeptide consisting of an amino acid sequence as shown in SEQ ID NO:2 from amino acid number 20 (Cys), to amino acid number 538 (Ser); and<br>
a transcription terminator,<br>
wherein the promoter is operably linked to the DNA segment, and the DNA segment is operably linked to the transcription terminator.<br>
10.	The expression vector as claimed in claim 9, comprising a secretory signal sequence operably linked to the DNA segment.<br>
11.	A cultured microbial cell comprising an expression vector as claimed in claim 9, wherein the cell expresses a polypeptide encoded by the DNA segment.<br>
12.	An expression vector comprising: a transcription promoter;<br><br>
a DNA segment encoding a zalpha 11 polypeptide consisting of an amino acid sequence as shown in SEQ ID N0:2 from amino acid number 20 (Cys), to amino acid number 237 (His); and<br>
a transcription terminator,<br>
wherein the promoter, DNA segment, and terminator are operably linked.<br>
13.	The expression vector as claimed in claim 12, comprising a secretory signal sequence operably linked to the DNA segment.<br>
14.	The expression vector as claimed in claim 12, wherein the polypeptide comprises a transmembrane domain operably linked to the DNA segment.<br>
15.	The expression vector as claimed in claim 14, wherein the transmembrane domain comprises residues 238(Leu) to 255 (Leu) of SEQ ID N0:2.<br>
16.	The expression vector as claimed in claim 12, wherein the polypeptide comprises an intracellular domain operably linked to the DNA segment.<br>
17.	The expression vector as claimed in claim 16, wherein the intracellular domain consists of residues 256 (Lys) to 538 (Ser) of SEQ ID NO:2.<br>
18.	A cultured microbial cell into which has been introduced an expression vector as claimed in claim 12, wherein the cell expresses a soluble receptor polypeptide encoded by the DNA segment.<br>
19.	The microbial cell as claimed in claim 18, wherein the cell is dependent upon<br>
an exogenously supplied hematopoietic growth factor for proliferation.<br><br>
20.      A DNA construct encoding a fusion protein, the DNA construct comprising:<br>
a first DNA segment encoding a polypeptide consisting of a sequence of amino acid residues selected from the group consisting of:<br>
(a)	the amino acid sequence of SEQ ID No.2 from amino acid number l(Met), to amino acid number 19 (Gly);<br>
(b)	the amino acid sequence of SEQ ID NO:2 from amino acid number acid number 20 (Cys) to amino acid number 237 (His);<br>
(c)	the amino acid sequence of SEQ ID N0:2 from amino acid number acid number 20 (Cys) to amino acid number 255 (Leu);<br>
(d)	the amino acid sequence of SEQ ID NO:2 from amino acid number acid number 238 (Leu) to amino acid number 255 (Leu);<br>
(e)	the amino acid sequence of SEQ ID N0:2 from amino acid number acid 238 (Leu) to amino acid number number 538 (Ser);<br>
(f)	the amino acid sequence of SEQ ID N0:2 from amino acid number 256 (Lys) to amino acid number number 538 (Ser); and<br>
(g)	the amino acid sequence of SEQ ID N0:2 from amino acid number 20 (Cys), to amino acid number 538 (Ser); and<br>
at least one other DNA segment encoding an additional polypeptide, wherein the first and other DNA segments are connected in-frame; and wherein the first and other DNA segments encode the fusion protein.<br><br>
21.	An expression vector comprising the following operably linked elements:<br>
a transcription promoter;<br>
a DNA construct encoding a fusion protein as claimed in claim 20; and<br>
a transcription terminator,<br>
wherein the promoter is operably linked to the DNA construct, and the DNA construct is operably linked to the transcription terminator.<br>
22.	A cultured microbial cell comprising an expression vector as claimed in claim 21, wherein the cell expresses a polypeptide encoded by the DNA construct.<br>
23.	A method of producing a fusion protein comprising : culturing a cell as claimed in claim 22, and isolating the polypeptide produced by the cell<br>
24.	An isolated polypeptide consisting of a sequence of amino acid residues selected from the group consisting of:<br><br>
(a)	the amino acid sequence of SEQ ID NO:2 from amino acid number acid number 20 (Cys) to amino acid number 237 (His);<br>
(b)	the amino acid sequence as shown in SEQ ID N0:2 from amino acid number acid number 20 (Cys) to amino acid number 255 (Leu);<br>
(c)	the amino acid sequence as shown in SEQ ID N0:2 from amino acid number acid number 256 (Lys), to amino acid number 538 (Ser); and<br>
(d)	the amino acid sequence as shown in SEQ ID N0:2 from amino acid number acid 20 (Cys), to amino acid number number 538 (Ser).<br><br>
25.	The isolated polypeptide as claimed in claim 24, wherein the polypeptide contains a WSXWS motif.<br>
26.	The isolated polypeptide as claimed in 24, wherein the polypeptide comprises a transmembrane domain.<br>
27.	The isolated polypeptide as claimed in claim 26, wherein the transmembrane domain consists of residues 238(Leu) to 255 (Leu) of SEQ ID N0:2.<br>
28.	The isolated polypeptide as claimed in claim 24, wherein the polypeptide comprises an intracellular domain.<br>
29.	The isolated polypeptide as claimed in claim 28, wherein the intracellular domain comprises residues 256 (Lys) to 538 (Ser) of SEQ ID N0:2.<br>
30.	The isolated polypeptide as claimed in claim 29, wherein the intracellular domain comprises Box I and Box II sites.<br>
31.	The isolated polypeptide comprising an amino acid sequence selected from the group consisting of:<br>
(a)      the amino acid sequence as shown in SEQ ID NO:2 from amino acid number 20 (Cys), to amino acid number 237 (His); and<br>
wherein   the   polypeptide   is   substantially   free   of   transmembrane   and intracellular domains ordinarily associated with hematopoietic receptors.<br>
32.	The polypeptide as claimed in claim 31, comprising an affinity tag.<br><br>
33.	A method of producing a zalpha 11 polypeptide comprising:<br>
culturing a cell as claimed in claim 18; and<br>
isolating the zalpha 11 polypeptide produced by the cell.<br>
34.	An antibody which specifically binds to a polypeptide as claimed in claim 24.<br>
35.	The antibody as claimed in claim 34, wherein the antibody is a monoclonal antibody.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQwNi1jaGUgIGFic3RyYWN0LWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-0406-che  abstract-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQwNi1jaGUgIGNsYWltcy1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-0406-che  claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQwNi1jaGUgIGRlc2NyaXB0aW9uIChjb21wbGV0ZSktZHVwbGljYXRlLTEucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-0406-che  description (complete)-duplicate-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQwNi1jaGUgIGRlc2NyaXB0aW9uIChjb21wbGV0ZSktZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-0406-che  description (complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQwNi1jaGUgIGRyYXdpbmdzLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-0406-che  drawings-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQwNi1jaGUgYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-0406-che abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQwNi1jaGUgY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-0406-che claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQwNi1jaGUgY29ycmVzcG9uZGVuY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-0406-che correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQwNi1jaGUgY29ycmVzcG9uZGVuY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-0406-che correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQwNi1jaGUgZGVzY3JpcHRpb24oY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-0406-che description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQwNi1jaGUgZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-0406-che drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQwNi1jaGUgZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-0406-che form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQwNi1jaGUgZm9ybS0xOS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-0406-che form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQwNi1jaGUgZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-0406-che form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQwNi1jaGUgZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-0406-che form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQwNi1jaGUgb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-0406-che others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQwNi1jaGUgcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-0406-che pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQwNi1jaGUgcGV0aXRpb24ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-0406-che petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="229346-a-cap-assembly-and-a-method-of-cooling-a-hot-drawn-fiber-in-a-heat-exchange-unit.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="229348-a-method-for-thermally-and-or-acoustically-insulating-a-building-an-insulating-material-and-a-process-for-producing-the-same.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>229347</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/406/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>12/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>20-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>16-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>22-Mar-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>ZYMOGENETICS, INC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1201 EASTLAKE AVENUE EAST, SEATTLE, WASHINGTON 98102,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>SCOTT R PRESNELL</td>
											<td>2902 N PUGET SOUND AVENUE, TACOMA, WASHINGTON 98407,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>DARRELL C. CONKLIN</td>
											<td>117 E LOUISA STREET, #421, SEATTLE, WASHINGTON 98102,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>JULIA E NOVAK</td>
											<td>10699 BATTLE POINT DRIVE NE, BAINBRIDGE ISLAND, WASHINGTON 98110,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>ANGELA K HAMMOND</td>
											<td>26774 230th PLACE SE, MAPLE VALLEY, WASHINGTON 98038,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>CO7K14/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US99/22149</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>1999-09-23</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>US 09/347,930</td>
									<td>1999-07-06</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>US 09/159,254</td>
									<td>1998-09-23</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>US 09/265,117</td>
									<td>1999-03-09</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/229347-cytokine-receptor-zalpha11 by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:01:56 GMT -->
</html>
