<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/229362-polypeptides-involved-in-spiramycin-biosynthesis-nucleotide-sequences-encoding-said-polypeptides by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:02:54 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 229362:POLYPEPTIDES INVOLVED IN SPIRAMYCIN BIOSYNTHESIS, NUCLEOTIDE SEQUENCES ENCODING SAID POLYPEPTIDES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">POLYPEPTIDES INVOLVED IN SPIRAMYCIN BIOSYNTHESIS, NUCLEOTIDE SEQUENCES ENCODING SAID POLYPEPTIDES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to the isolation and identification of novel genes of the biosynthetic pathway for spiramycins and to novel polypeptides involved in this biosynthesis. The invention also relates to a method for producing these polypeptides. It also relates to the use of these genes for the purpose of increasing the levels of production and the purity of the spiramycin produced. The invention relates in particular to a microorganism which produces spiramycin I but which does not produce spiramycin II and III , and to the use of such a microorganism. The invention also relates to the use of the genes of the biosynthetic pathway for spiramycins for constructing mutants which can lead to the synthesis of novel antibiotics or to derived forms of spiramycins. The invention also relates to the molecules produced through the expression of these genes and to pharmacologically active compositions of a molecule produced through the expression of such genes.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
POLYPEPTIDES INVOLVED IN THE BIOSYNTHESIS OF SPIRAMYCINS. NUCLEOTIDE SEQUENCES ENCODING THESE POLYPEPTIDES AND<br>
APPLICATIONS THEREOF<br>
The present invention relates to the isolation and identification of novel genes of the biosynthetic'c pathway for spiramycins and to novel polypeptides involved in this biosynthesis. It also relates to the use of these genes for the purpose of increasing the levels of production and the purity of the spiramycin produced.<br>
The invention also relates to the use of these genes for constructing mutants which can lead to the synthesis of novel antibiotics or to derived forms of spiramycins. The invention also relates to the molecules produced through the expression of these genes and, finally, to pharmacologically active compositions of a molecule produced through the expression of such genes.<br>
Streptomyces are Gram-positive filamentous soil bacteria. They play an important role in the decomposition and mineralization of organic materials due to the great diversity of degrading enzymes that they secrete. They exhibit morphological differentiation phenomena which are unique in prokaryotes, accompanied by a metabolic differentiation characterized by the production of secondary metabolites having an extraordinary diversity of chemical structures and biological activities. Among these metabolites are the spiramycins produced naturally by the bacterium Streptomyces ambofaciens.<br>
Spiramycin is an antibiotic of the macrolide family which is of use both in veterinary medicine and in human medicine. Macrolides are characterized by the presence of a lactone ring onto which are grafted one or more sugars. Streptomyces ambofaciens naturally produces spiramycin I, II and IE (cf. figure 1); however, the antibiotic activity of spiramycin I is clearly greater than that of spiramycins II and HI (Liu et al, 1999). The spiramycin I molecule consists of a lactone-based macrocycle,<br><br>
called platenolide, and three sugars: forosamme, mycaminose and mycarose (cf. figure 1). The antibiotic activity of spiramycins is due to inhibition of protein synthesis in prokaryotes via a mechanism involving binding of the antibiotic to the bacterial ribosome.<br>
Certain compounds which are members of the macrolide family and which also possess a lactone ring have given rise to uses outside the field of antibiotics. Thus, the product FK506 has immunosuppressor effects and offers perspectives of therapeutic application in the field of organ transplantation, of rheumatoid arthritis and, more generally, in pathological conditions related to autoimmune reactions. Other macrolides, such as avermectin, have insecticidal and anti-helminth activity.<br>
Many biosynthetic pathways, concerning antibiotics belonging to varied classes, and also other secondary metabolites (for a review, K, Chater, 1990), have to date already been studied in Streptomycetes. However, knowledge of the biosynthetic pathways for spiramycins is, to date, only very partial.<br>
Spiramycin biosynthesis is a complex process comprising many steps and involving many enzymes (Omura et al9 1979a, Omura et al9 1979b). Spiramycins belong to the large class of polyketides which includes complex molecules which are particularly abundant in microorganisms found in the soil. These molecules are grouped together not through structural analogy, but through a certain similarity in the steps of their biosynthetic pathway. Specifically, polyketides are produced by a complex series of reactions, which have in common the fact that, in their biosynthetic pathway, there is a series of reactions catalyzed by.one-or.more enzymes .called."polyketide synthases" (PKS). In Streptomyces ambofaciens, the biosynthesis of the lactone-based macrocycle of spiramycins (platenolide) is carried out by a series of eight modules encoded by five PKS genes (S. Kuhstoss, 1996, US patent 5,945,320). Spiramycins are obtained from this lactone ring. Howeverthe various steps and enzymes involved in the synthesis of the sugars, and also their attachment to the lactone ring and the modification of this ring so as to obtain spiramycins still remain unknown to date.<br><br>
US patent 5,514,544 describes the cloning of a sequence called srmR in Streptomyces ambofaciens. In that patent, the hypothesis that the srmR gene encodes a protein which regulates the transcription of the genes involved in macrolide biosynthesis is put forward.<br>
In 1987, Richardson and colleagues (Richardson et aL, 1987) showed that the spiramycin resistance of S. ambofaciens is imparted by at least three genes; said genes were called srmA, srmB and srrnC. US patent 4,886,757 describes more particularly the characterization of a DNA fragment of 5. ambofaciens containing the srmC gene. However, the sequence of this gene was not disclosed. In 1990, Richardson and colleagues (Richardson et al, 1990) put forward the hypothesis that there are three genes for spiramycin biosynthesis close to srmB. US patent 5,098,837 reports the cloning of five genes potentially involved in spiramycin biosynthesis. These genes were named srmD, srmE, srmF, srmG and srmH.<br>
One of the major difficulties in producing compounds such as spiramycins lies in the fact that very large fermentation volumes are necessary to produce a relatively small amount of product. It is therefore desirable to be able to increase the efficiency of production of such molecules in order to decrease the cost of production thereof.<br>
The biosynthetic pathway for spiramycins is a complex process and it would be desirable to identify and to eliminate the parasitic reactions which might exist during this process. The aim of such a manipulation is to obtain a purer antibiotic and/or an improvement in productivity. In this respect, Streptomyces ambofaciens naturally produces spiramycin I, II and HI (cf. figure 1); however, the antibiotic activity of spiramycin I is clearly greater than that of spiramycins II and EI (Liu et al.9 1999). It would therefore be desirable to be able to have strains which produce only spiramycin I.<br>
Due to the commercial value of macrolide antibiotics, the production of novel derivatives, in particular of analogs of spiramycins having advantageous properties, is intensely sought. It would be desirable to be able to generate, in sufficient amount, the biosynthetic intermediates of the biosynthetic pathway for spiramycins or spiramycin<br><br>
derivatives, in particular for the purpose of producing spiramycin-derived hybrid antibiotics.<br>
General description of the invention<br>
The present invention results from the cloning of genes the product of which is involved in spiramycin biosynthesis. The invention relates, first of all, to novel genes of the biosynthetic pathway for spiramycins and novel polypeptides involved in this biosynthesis.<br>
The genes of the biosynthetic pathway and the associated coding sequences have been cloned and the DNA sequence of each one has been determined. The cloned coding sequences will hereafter be designated orfl*c, or/2*c, or/3*cf orf4*c, or/5*, orf6*t or/7*c, or/8*, or/9*, or/10*, orfl, or/2, or/3, orf4, orp, orf6, or/7, or/8, or/9c, orflO, orfllc, orfl2, orfl3c, orfl4, orfl 5c, orfl6, orfl 7, orfl8, orf!9, or/20, orftlc, or/22c, or/23cf orf24c, or/25ct or/26, or/27, orf28ct or/29, or/30c, or/31, or/32c, or/33 and or/34c. The function of the proteins encoded by these sequences in the biosynthetic pathway for spiramycins is developed in the discussion hereinafter, which is illustrated by figures 4, 5,6 and 8.<br>
1) A first subject of the invention concerns a polynucleotide encoding a polypeptide involved in spiramycin biosynthesis, wherein the sequence of said polynucleotide is:<br>
(afone of theSequences SEQ"IIXNdr3;5;7,^;ir;13rWrn;"'19^1;-23;-25f"28r<br>
30, 34, 36, 40, 43, 45, 47, 49, 53, .60, 62, 64, 66, 68, 70, 72, 74, 76, 78, 80, 82, 84, 107, 109, 111, 113, 115, 118, 120, 141,143,145, 147 and 149,<br>
(b)	one of the sequences consisting of the variants of the sequences (a),<br>
(c)	one of the sequence derived from the sequences (a) and (b) due to the degeneracy of the genetic code.<br><br>
2)	A subject of the present invention is also a polynucleotide which hybridizes, under high stringency hybridization conditions, to at least one of the polynucleotides according to paragraph!) above.<br>
3)	The invention also relates to a polynucleotide exhibiting at least 70%, 80%, 85%, 90%, 95% or 98% nucleotide identity with a polynucleotide comprising at least 10, 12, 15, 18, 20 to 25, 30, 40, 50, 60, 70, 80, 90, 100, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 80O, 850, 900, 950, 1000, 1050, 1100, 1150, 1200, 1250, 1300, 1350, 1400, 1450. 1500. 1550, 1600, 1650, 1700, 1750, 1800, 1850 or 1900 consecutive nucleotides of a polynucleotide according to paragraph 1) above.<br>
4)	The invention also relates to a polynucleotide according to paragraph 2) or 3) above, which is isolated from a bacterium of the genus Streptomyces.<br>
5)	The invention also relates to a polynucleotide according to paragraph 2), 3) or 4) above, which encodes a protein involved in the biosynthesis of a macrolide.<br>
6)	The invention also relates to a polynucleotide according to paragraph 2), 3) or 4) above, which encodes a protein having an activity similar to the protein encoded by the polynucleotide with which it hybridizes or with which it exhibits the identity.<br>
7)	The invention also relates to a polypeptide resulting from the expression of a polynucleotide according to paragraph 1), 2), 3), 4), 5) or 6) above.<br>
8)	Another aspect of the invention concerns a polypeptide involved in a<br>
spiramycin biosynthesis, wherein the sentence of said polypeptide is:<br>
(a) one or me sequences SEQ ID No. 4, 6; 8; 10, 12,14, 16, 18, 20, 22, 24, 26, 27, 29, 31, 32, 33, 35, 37, 38, 39, 41, 42, 44, 46, 48, 50, 51, 52, 54, 55, 56, 57, 58, 59, 61, 63, 65, 67, 69, 71, 73, 75, 77, 79, 81, 83, 85, 108, 110, 112, 114, 116, 117, 119, 121, 142, 144, 146, 148 and 150,<br>
(b) one of the sequences as defined in (a), except that throughout said sequence, one or more amino acids have been substituted, inserted or deleted without affecting the functional properties thereof,<br><br>
(c) one of the sequences consisting of the variants of the sequences (a) and (b).<br>
9)	Another subject of the invention concerns a polypeptide exhibiting at least<br>
70%, 80%, 85%, 90%, 95% or 98% amino acid identity with a polypeptide comprising<br>
at least 10, 15, 20, 30 to 40, 50, 60, 70, 80, 90, 100, 120, 140, 160, 180, 200, 220, 240,<br>
260, 280, 300, 320, 340, 360, 380, 400, 420, 440, 460, 480, 500, 520, 540, 560, 580,<br>
600, 620 or 640 consecutive amino acids of a polypeptide according to paragraph 8)<br>
above.<br>
10)	Another aspect of the invention concerns a polypeptide according to paragraph 9) above, which is isolated from a bacterium of the genus Streptomyces.<br>
11)	Another aspect of the invention concerns a polypeptide according to paragraph 9) or 10) above, which encodes a protein involved in the biosynthesis of a macrolide.<br>
12)	Another aspect of the invention concerns a polypeptide according to pargraph<br>
9), 10) or 11) above, which has an activity similar to that of the polypeptide with which<br>
it shares the identity.<br>
13)	Another aspect of the invention concerns a recombinant DNA, which comprises at least one polynucleotide according to one of paragraphs 1), 2), 3), 4), 5) and 6) above.<br>
14)	Another aspect of the invention concerns a recombinant DNA according to paragraph 13) above, wherein said recombinant DNA is included in a vector.<br>
15)	Another aspect of the invention concerns a recombinant DNA according to paragraph 14) above, wherein said vector is chosen from bacteriophages, plasmids, phagemids, integrative vectors, fosmids, cosmids, shuttle vectors, BACs and PACs.<br>
16)	Another aspect of the invention concerns a recombinant DNA according to paragraph 15) above, which is chosen from pOS49.1, pOS49.ll, pOSC49.12, pOS49.14, pOS49.16,  pOS49.28,  pOS44.1, pOS44.2, pOS44.4,  pSPM5,  pSPM7,<br><br>
pOS49.67, pOS49.88, pOS49.106, pOS49.120, pOS49.107, pOS49.32, pOS49.43, pOS49.44, pOS49.50, pOS49.99, pSPM17, pSPM21, pSPM502, pSPM504, pSPM507, pSPM508, pSPM509, pSPMl, pBXLllll, pBXL1112, pBXL1113, pSPM520, pSPM521, pSPM522, pSPM523, pSPM524, pSPM525, pSPM527, pSPM528, pSPM34, pSPM35, pSPM36. pSPM37, pSPM38, pSPM39, pSPM40, pSPM41, pSPM42, pSPM43, pSPM44, pSPM45, pSPM47, pSPM48, pSPMSO, pSPM51, pSPM52, pSPM53, pSPM55, pSPM56, pSPM58, pSPM72, pSPM73, pSPM515, pSPM519, pSPM74, pSPM75, pSPM79, pSPM83, pSPM107, pSPM543 and pSPM106.<br>
17)	Another aspect of the invention concerns an expression vector, which comprises at least one nucleic acid sequence encoding a polypeptide according to paragraph 7), 8), 9), 10), 11) or 12) above.<br>
18)	The invention also relates to an expression system comprising a suitable expression vector and a host cell - which allows the expression of one or more polypeptides according to paragraph 7), 8), 9), 10), 11) or 12) above.<br>
19)	The invention also relates to an expression system according to paragraph 18 above, which is chosen from prokaryotic expression systems and eukaryotic expression systems.<br>
20)	The invention also relates to an expression system according to paragraph 19) above, which is chosen from systems of expression in the bacterium £". coli, baculovirus expression systems which allow expression in insect cells, expression systems which allow expression in yeast cells and expression systems which allow expression in mammalian.cells.	<br>
21)	The invention also relates to a host cell into which at least one polypeptide and/or at least one recombinant DNA and/or at least one expression vector according to one of paragraphs 1), 2), 3), 4), 5), 6), 13), 14), 15), 16) and 17) above has been introduced.<br><br>
22)	The invention also relates to a method of producing a polypeptide according<br>
to paragraph 7), 8), 9), 10), 11) or 12) above, wherein said method comprises the<br>
following steps:<br>
a)	inserting at least one nucleic acid encoding said polypeptide into a suitable vector;<br>
b)	culturing, in a suitable culture medium, a host cell transformed or transfected beforehand with the vector of step a);<br>
c)	recovering the conditioned culture medium or a cell extract;<br>
d)	separating and purifying said polypeptide from said culture medium or else from the cell extract obtained in step c);<br>
e)	where appropriate, characterizing the recombinant polypeptide produced.<br>
23)	Another aspect of the invention concerns a microorganism blocked in a step<br>
of the biosynthetic pathway for at least one macrolide.<br>
24)	Another aspect of the invention concerns a microorganism according to paragraph 23) above, which is obtained by inactivating the function of at least one protein involved in the biosynthesis of this (these) macrolide(s) in a microorganism which produces this (these) macrolide(s).<br>
25)	Another aspect of the invention concerns a microorganism according to paragraph 24) above, wherein the inactivation of this (these) protein(s) is carried out by mutagenesis in the gene(s) encoding said protein(s) or by expression of one or more antisense RNAs complementary to the messenger RNA(s) encoding said protein(s).<br>
26).Another aspect of the invention..concerns...a microorganism according to paragraph 25) above, wherein the inactivation of this (these) protein(s) is carried out by mutagenesis via irradiation, by action of a mutagenic chemical agent, by site-directed mutagenesis or by gene replacement.<br>
27) Another aspect of the invention concerns a microorganism according to paragraph 25) or 26) above, wherein the mutagenesis or mutageneses is or are carried<br><br>
out in vitro or in situ, by suppression, substitution, deletion and/or addition of one or more bases in the gene(s) under consideration or by gene inactivation.<br>
28)	Another aspect of the invention concerns a microorganism according to paragraph 23), 24), 25), 26) or 27) above, wherein said microorganism is a bacterium of the genus Streptomyces.<br>
29)	Another aspect of the invention concerns a microorganism according to paragraph 23), 24), 25), 26), 27) or 28) above, wherein the macrolide is spiramycin,<br>
30)	Another aspect of the invention concerns a microorganism according to paragraph 23), 24), 25), 26), 27), 28) or 29) above, wherein said microorganism is a strain of 5. ambofaciens.<br>
31)	Another aspect of the invention concerns a microorganism according to paragraph 23), 24), 25), 26), 27), 28), 29) or 30) above, wherein the mutagenesis is carried out in at least one gene comprising a sequence according to one of paragraphs 1), 2), 3), 4), 5) and 6) above.<br>
32)	Another aspect of the invention concerns a microorganism according to paragraph 25), 26), 27), 28), 29), 30) or 31) above, wherein the mutagenesis is carried out in one or more genes comprising one of the sequences corresponding to one or more of the sequences SEQ ID No. 3, 5, 7,9, 11,13,15,17,19,21,23,25, 28, 30, 34, 36,40, 43, 45, 47, 49, 53, 60, 62, 64, 66, 68, 70, 72, 74, 76, 78, 80, 82, 84, 107, 109, 111, 113, 115,118,120, 141, 143, 145, 147 and 149.<br>
33)	Another aspect of the invention concerns a microorganism according to paragraph 25), 26), 27), 28), 29), 30), 31) or 32) above, wherein the mutagenesis consists of the gene inactivation of a gene comprising a sequence corresponding to the sequence SEQ ID No. 13.<br>
34)	Another aspect of the invention concerns a strain of Streptomyces<br>
ambofaciens, which is a strain chosen from one of the strains deposited with the<br><br>
Collection Nationale de Cultures de Microorganismes [National Collection of Cultures and Microorganisms] (CNCM) on July 10, 2002, under the registration number 1-2909, 1-2911,1-2912,1-2913,1-2914,1-2915,1-2916 or 1-2917.<br>
35)	Another aspect of the invention concerns a method of preparing a macrolide<br>
biosynthesis intermediate, which comprises the following steps:<br>
a)	culturing, in a suitable culture medium, a microorganism according to one of paragraphs 23), 24), 25), 26), 27), 28), 29), 30), 31), 32), 33) or 34) above,<br>
b)	recovering the conditioned culture medium or a cell extract,<br>
c)	separating and purifying said biosynthesis intermediate from said culture<br>
medium or else from the extract obtained in step b).<br>
36)	Another aspect of the invention concerns a method of preparing a molecule derived from a macrolide, wherein a biosynthesis intermediate is prepared according to the method of paragraph 35) above and the intermediate thus produced is modified.<br>
37)	Another aspect of the invention concerns a method of preparation according to paragraph 36) above, wherein said intermediate is modified chemically, biochemically, enzymatically and/or microbiologically.<br>
38)	Another aspect of the invention concerns a method of preparation according to paragraph 36) or 37) above, wherein one or more genes encoding proteins capable of modifying the intermediate by using it as substrate is (are) introduced into said microorganism.<br>
39)	Another aspect of the invention concerns a method of preparation according to paragraph 36), 37) or 38) above, wherein the macrolide is spiramycin.<br>
40)	Another aspect of the invention concerns a method of preparation according to paragraph 36), 37), 38) or 39) above, wherein the microorganism used is a strain of S. ambofaciens.<br><br>
41)	Another aspect of the invention concerns a microorganism which produces spiramycin I but which does not produce spiramycin II and IE<br>
42)	Another aspect of the invention concerns a microorganism according to paragraph 41) above, which comprises all of the genes required for the biosynthesis of spiramycin I, but wherein the gene comprising the sequence SEQ ID No. 13 or one of its variants, or one of the sequences derived therefrom due to the degeneracy of the genetic code, and encoding a polypeptide of sequence SEQ ID No. 14 or one of its variants, is not expressed or has been made inactive.<br><br>
43)	Another aspect of the invention concerns a microorganism according to paragraph 42) above, wherein said inacrivation is carried out by mutagenesis in the gene encoding said protein or by the expression of an antisense RNA complementary to the messenger RNA encoding said protein.<br>
44)	Another aspect of the invention concerns a microorganism according to paragraph 43) above, .wherein said mutagenesis is carried out in the promoter of .this gene, in the coding sequence or in a noncoding sequence so as to make the encoded protein inactive or to prevent its expression or its translation therefrom.<br>
45)	Another aspect of the invention concerns a microorganism according to paragraph 43) or 44) above, wherein the mutagenesis is carried out by irradiation, by action of a mutagenic chemical agent, by site-directed mutagenesis or by gene replacement.<br>
46)	Another aspect of the invention concerns a microorganism according to paragraph 43), 44) or 45) above, wherein the mutagenesis is carried out in vitro or in situ, by suppression, substitution, deletion and/or addition of one or more bases in the gene under consideration or by gene inactivation.<br>
47)	Another aspect of the invention concerns a microorganism according to<br>
paragraph 41) or 42) above, wherein said microorganism is obtained by expressing the<br>
genes of the biosynthetic pathway for spiramycin without these genes comprising the<br><br>
gene comprising the sequence corresponding to SEQ ED No. 13 or one of its variants, or one of the sequences derived therefrom due to the degeneracy of the genetic code, and encoding a polypeptide of sequence SEQ ID No. 14 or one of its variants.<br>
48)	Another aspect of the invention concerns a microorganism according to paragraph 41), 42), 43), 44), 45), 46) or 47) above, wherein said microorganism is a bacterium of the genus Streptomyces.<br>
49)	Another aspect of the invention concerns a microorganism according to paragraph 41), 42), 43), 44), 45), 46), 47) or 48) above, wherein said microorganism is obtained from a starting strain which produces spiramycins I, II and HI.<br>
50)	Another aspect of the invention concerns a microorganism according to paragraph 41), 42), 43), 44), 45), 46), 47), 48) or 49) above, which is obtained by mutagenesis in a gene comprising the sequence corresponding to SEQ ID No. 13 or one of its variants, or one of the sequences derived therefrom due to the degeneracy of the genetic code, and encoding a polypeptide of sequence SEQ ID No. 14 or one of its variants having the same function.<br>
51)	Another aspect of the invention concerns a microorganism according to paragraph 41), 42), 43), 44), 45), 46), 47), 48), 49) or 50) above, wherein said microorganism is obtained from a strain of 5. amhofaciens which produces spiramycins I, II and EI, in which gene inactivation of the gene comprising the sequences corresponding to SEQ ID No. 13 or one of the sequences derived therefrom due to the degeneracy of the genetic code is carried out.<br><br>
52)	Another aspect of the invention concerns a strain of S. ambofaciens, which is the strain deposited with the Collection Nationale de Cultures de Microorganismes (CNCM) on July 10, 2002, under the registration number 1-2910.<br>
53)	Another aspect of the invention concerns a method of producing spiramycin I which comprises the following steps:<br><br>
(a)	culturing, in a suitable culture medium, a microorganism according to one of paragraphs 41), 42), 43), 44), 45), 46), 47), 48), 49), 50), 51) and 52) above,<br>
(b)	recovering the conditioned culture medium or a cell extract,<br>
(c)	separating and purifying the spiramycin I from said culture medium or else from the cell extract obtained in step b).<br><br>
54)	Another aspect of the invention concerns the use of a polynucleotide according to one of paragraphs 1), 2), 3), 4), 5) and 6) above, for improving the macrolide production of a microorganism.<br>
55)	Another aspect of the invention concerns a macrolide-producing mutant microorganism, which has a genetic modification in at least one gene comprising a sequence according to one of paragraphs 1), 2), 3), 4), 5) or 6) above and which overexpresses at least one gene comprising a sequence according to one of paragraphs 1), 2), 3), 4), 5) and 6) above.<br>
56)	Another aspect of the invention concerns a mutant microorganism according to paragraph 55) above, wherein the genetic modification consists of a suppression, a substitution, a deletion and/or an addition of one or more bases in the gene(s) under consideration, with the aim of expressing one or more proteins having greater activity or of expressing a higher level of this (these) protein(s).<br>
57)	Another aspect of the invention concerns a mutant microorganism according to paragraph 55) above, wherein the overexpression of the gene under consideration is obtained by increasing the copy number for the gene and/or introducing a promoter which is more active than.the wild-type promoter.	<br>
58)	Another aspect of the invention concerns a mutant microorganism according to paragraph 55) or 57) above, wherein the overexpression of the gene under consideration is obtained by transforming a macrolide-producing microorganism with a recombinant DNA construct according to paragraph 13, 14 or 17 above, allowing the overexpression of this gene.<br><br>
59)	Another aspect of the invention concerns a mutant microorganism according to paragraph 55), 56), 57) or 58) above, wherein the genetic modification is carried out in one or more genes comprising one of the sequences corresponding to one or more of the sequences SEQ ID No. 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 25, 28, 30, 34, 36, 40, 43, 45, 47, 49, 53, 60, 62, 64, 66, 68, 70, 72, 74, 76, 78, 80, 82, 84, 107, 109, 111,113, 115, 118, 120, 141, 143, 145, 147 and 149, or one of its variants, or one of the sequences derived therefrom due to the degeneracy of the genetic code.<br>
60)	Another aspect of the invention concerns a mutant microorganism according to paragraph 55), 56), 57), 58) or 59) above, wherein the microorganism overexpresses one or more genes comprising one of the sequences corresponding to one or more of the sequences SEQ ID No. 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 25, 28, 30, 34, 36, 40, 43, 45,47,49, 53, 60, 62, 64,66,68, 70, 72,74,76, 78, 80, 82, 84,107, 109, 111, 113, 115, 118, 120, 141, 143, 145, 147 and 149, or one of its variants, or one of the sequences derived therefrom due to the degeneracy of the genetic code.<br>
61)	Another aspect of the invention concerns a mutant microorganism according to paragraph 55), 56), 57), 58), 59) or 60) above, wherein said microorganism is a bacterium of the genus Streptomyces.<br>
62)	Another aspect of the invention concerns a mutant microorganism according to paragraph 55), 56), 57), 58), 59), 60) or 61) above, wherein the macrolide is spiramycin.<br><br>
63)	Another aspect of the invention concerns a mutant microorganism according to paragraph 55)r56),57)r58),o9), 60), 61) or 62) above, wherein said microorganism is a strain of S. ambofaciens.<br>
64)	Another aspect of the invention concerns a method of producing macrolides, which comprises the following steps:<br>
(a) culturing, in a suitable culture medium, a microorganism according to one of paragraphs 55), 56), 57), 58), 59), 60), 61), 62), 63) and 64) above,<br><br>
(b)	recovering the conditioned culture medium or a cell extract,<br>
(c)	separating and purifying said macrolide(s) produced from said culture<br>
medium or else from the cell extract obtained in step b).<br>
65)	Another aspect of the invention concerns the use of a sequence and/or of a<br>
recombinant DNA and/or of a vector according to one of paragraphs 1), 2), 3), 4), 5), 6),<br>
7), 8), 9), 10), 11), 12), 13), 14), 15), 16) and 17) above, for preparing hybrid<br>
antibiotics.<br>
66)	Another aspect of the invention concerns the use of at least one polynucleotide and/or at least one recombinant DNA and/or at least one expression vector and/or at least one polypeptide and/or at least one host cell according to one of paragraphs 1), 2), 3), 4), 5), 6), 7), 8), 9), 10), 11), 12), 13), 14), 15), 16), 17) and 21) above, for carrying out one or more byconversions.<br>
67)	Another aspect of the invention concerns a polynucleotide which is a polynucleotide complementary to one of the polynucleotides according to paragraph 1), 2), 3), 4), 5) or 6) above.<br>
68)	Another aspect of the invention concerns a microorganism producing at<br>
least one spiramycin, which overexpresses:<br>
-	a gene which can be obtained by polymerase chain reaction (PCR) using the<br>
following pair of sequence primers:<br>
5' AAGCTTGTGTGCCCGGTGTACCTGGGGAGC 3' (SEQ ID No. 138) and<br>
5' GGATCCCGCGACGGACACGACCGCCGCGCA 3' (SEQ ID No. 139)<br>
and, as matrix, the cosmid pSPM36 or the total DNA of Streptomyces ambofaciens,<br>
-	or a gene derived therefrom due to the degeneracy of the genetic code.<br>
69)	Another aspect of the invention concerns a microorganism according to<br>
paragraph 68 or 90, which is a bacterium of the genus Streptomyces.<br><br>
70)	Another aspect of the invention concerns a microorganism according to paragraph 68, 69 or 90, which is a bacterium of the species Streptomyces ambofaciens.<br>
71)	Another aspect of the invention concerns a microorganism according to paragraph 68, 69, 70 or 90, wherein the overexpression of said gene is obtained by transformation of said microorganism with an expression vector.<br><br>
72)	Another aspect of the invention concerns a strain of Streptomyces ambofaciens, which is the strain OSC2/pSPM75(l) or the strain OSC2/pSPM75(2) deposited with the Collection Nationale de Cultures de Microorganismes (CNCM) [National Collection of Cultures and Microorganisms] Pasteur Institute, 25,*rue du Docteur Roux 75724 Paris Cedex 15, France, on October 6, 2003, under the registration number 1-3101.<br>
73)	Another aspect of the invention concerns a recombinant DNA which comprises:<br>
-	a polynucleotide which can be obtained by polymerase chain reaction using<br>
the following pair of sequence primers:<br>
5' AAGCTTGTGTGCCCGGTGTACCTGGGGAGC 3' (SEQ ID No. 138) and<br>
5' GGATCCCGCGACGGACACGACCGCCGCGCA 3' (SEQ ID No. 139) and, as<br>
matrix, the cosmid pSPM36 or the total DNA of Streptomyces ambofaciens,<br>
-	or a fragment of at least 10, 12, 15, 18, 20 to 25, 30, 40, 50, 60, 70,80,90,<br>
100,150, 200, 250, 300, 350,400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900,<br>
950, 1.0QO,.i05.0,.JJ(OOf_A15Q,.J200^.125p, 1300, ...1350, 1400, 1450, 1460, 1470,<br>
1480,1490 or 1500 consecutive nucleotides of this polynucleotide.<br>
74)	Another aspect of the invention concerns a recombinant DNA according to paragraph 73 or 91, which is a vector.<br>
75)	Another aspect of the invention concerns a recombinant DNA according to paragraph 73, 74 or 91, which is an expression vector.<br><br>
76)	Another aspect of the invention concerns a host cell into which at least one recombinant DNA according to one of paragraphs 73, 74, 75 and 91 has been introduced.<br>
77)	Another aspect of the invention concerns a method of producing a polypeptide, wherein said method comprises the following steps:<br><br>
a)	transforming a host cell with at least one expression vector according to paragraph 75;<br>
b)	cultivating, in a suitable culture medium, said host cell;<br>
c)	recovering the conditioned culture medium or a cell extract;<br>
d)	separating and purifying said polypeptide from said culture medium or else from the cell extract obtained in step c);<br>
e)	where appropriate, characterizing the recombinant polypeptide produced.<br><br>
78)	Another aspect of the invention concerns a microorganism according to paragraph 51, wherein the gene inactivation is carried out by in-phase deletion of the gene or of a part of the gene comprising the sequence corresponding to SEQ ID No. 13 or one of the sequences derived therefrom due to the degeneracy of the genetic code.<br>
79)	Another aspect of the invention concerns a microorganism according to one of paragraphs 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51 and 78, which also overexpresses:<br>
-	a gene which can be obtained by polymerase chain reaction using the following pair<br>
of sequence primersL.<br>
5' AAGCTTGTGTGCCCGGTGTACCTGGGGAGC 3' (SEQ ID No. 138) and<br>
5' GGATCCCGCGACGGACACGACCGCCGCGCA 3' (SEQ ID No. 139) and, as<br>
matrix, the cosmid pSPM36 or the total DNA of Streptomyces ambofaciens,<br>
-	or a gene derived therefrom due to the degeneracy of the genetic code.<br>
80) Another aspect of the invention concerns an expression vector, wherein the polynucleotide of sequence SEQ ID No. 47, or a polynucleotide derived<br><br>
therefrom due to the degeneracy of the genetic code, is placed under the control of a promoter which allows the expression of the protein encoded by said polynucleotide in Streptomyces ambofaciens.<br>
81)	Another aspect of the invention concerns an expression vector according<br>
to paragraph 80, which is the plasmid pSPM524 or pSPM525.<br>
82)	Another aspect of the invention concerns a strain of Streptomyces ambofaciens transformed with a vector according to paragraph 80 or 81.<br>
83)	Another aspect of the invention concerns a microorganism according to one of paragraphs 41,42,43, 44,45, 46,47,48,49, 50, 51,78, 79 and 92, which also overexpresses the gene having coding sequence SEQ ID No. 47 or a coding sequence derived therefrom due to the degeneracy of the genetic code.<br>
84)	Another aspect of the invention concerns a microorganism according to paragraph 83, which is the strain SPM502 pSPM525 deposited with the Collection Nationale de Cultures de Microorganismes (CNCM) Pasteur Institute, 25 rue du Docteur Roux 75724 Paris Cedex 15, France, on February 26, 2003, under the registration number 1-2977.<br>
85) Another aspect of the invention concerns a method of producing spiramycin(s), which comprises the following steps:<br>
(a)	culturing, in a suitable culture medium, a microorganism according to one of paragraphs 68,69,70,71,72, 78, 79, 82, 83, 84,90 and 92,<br>
(b)	recovering the conditioned culture medium or a cell extract,<br>
(c)	separating and purifying the spiramycins from said culture medium or else from the cell extract obtained in step b).<br>
86) Another aspect of the invention concerns a polypeptide, the sequence of which comprises the sequence SEQ ID No. 112 or the sequence SEQ ED No. 142.<br><br>
87)	Another aspect of the invention concerns a polypeptide, the sequence of<br>
which corresponds to the sequence translated from the coding sequence:<br>
- of a gene which can be obtained by polymerase chain reaction (PCR) using the following pair of sequence primers:<br>
5' AAGCTTGTGTGCCCGGTGTACCTGGGGAGC 3' (SEQ ID No. 138) and<br>
5' GGATCCCGCGACGGACACGACCGCCGCGCA 3r(SEQ ID No. 139) and, as<br>
matrix, the cosmid pSPM36 or the total DNA of Streptomyces ambofaciens,<br>
-or of a gene derived therefrom due to the degeneracy of the genetic code.<br>
88)	Another aspect of the invention concerns an expression vector which allows the expression of a polypeptide according to paragraph 86, 87 or 93 in Streptomyces ambofaciens.  .   .<br>
89)	Another aspect of the invention concerns an expression vector according to paragraph 88, which is the plasmid pSPM75.<br>
90)	Another aspect of the invention concerns a microorganism according to paragraph 68, wherein the gene which can be obtained by polymerase chain amplification is the gene of coding sequence SEQ ID No. 141, or a gene derived therefrom due to the degeneracy of the genetic code.<br>
91)	Another aspect of the invention concerns a recombinant DNA according<br>
to paragraph 73, wherein the polynucleotide which can be obtained by polymerase<br>
chain amplification is a polynucleotide of sequence SEQ ID No. 141.<br>
92} Another aspect of the invention concerns a microorganism according to paragraph 79, wherein the gene which can be obtained by polymerase chain amplification is the gene of coding sequence SEQ ID No. 141, or a gene derived therefrom due to the degeneracy of the genetic code.<br>
93) Another aspect of the invention concerns a polypeptide, the sequence of which is SEQ ID No. 142.<br><br>
General definitions<br>
For the purpose of the present invention, the term "isolated" denotes a biological material (nucleic acid or protein) which has been removed from its original environment (the environment in which it is naturally located).<br>
For example, a polynucleotide present in the natural state in a plant or an animal is not isolated. The same polynucleotide separated from the adjacent nucleic acids within which it is naturally inserted in the genome of the plant or the animal is considered to be "isolated".<br>
Such a polynucleotide can be included in a vector and/or such a polynucleotide can be included in a composition, and can nevertheless remain in the isolated state due to the fact that the vector of the composition does not constitute its natural environment.<br>
The term "purified" does not require the material to be present in a form of absolute purity, excluding the presence of other compounds. It is rather a relative definition.<br>
A polynucleotide in the "purified" state after purification of the starting material or of the natural material by at least an order of magnitude, preferably 2 or 3, and preferentially 4 or 5, orders of magnitude.<br>
For the purposes of the present invention, the term ORF ("Open Reading Frame") has been used to denote in particular the coding sequence of a gene.<br>
For the purposes of the present invention, the expression "nucleotide sequence" can be used to denote equally a polynucleotide or a nucleic acid. The expression "nucleotide sequence" encompasses the genetic material itself and is not therefore restricted to the information concerning its sequence.<br><br>
The terms "nucleic acid", "polynucleotide", "oligonucleotide" or alternatively "nucleotide sequence" encompass RNA, DNA or cDNA sequences or RNA/DNA hybrid sequences of more than one nucleotide, equally in the single-chain form or in the form of a duplex.<br>
The term "nucleotide" denotes both the natural nucleotides (A, T, G, C) and also modified nucleotides which comprise at least one modification, such as (1) a purine analog, (2) a pyrimidine analog, or (3) a sugar analog, examples of such modified nucleotides being described, for example, in PCT application No. WO 95/04064.<br>
For the purposes of the present invention, a first polynucleotide is considered to be "complementary" to a second polynucleotide when each base of the first polynucleotide is paired to the complementary base of the second polynucleotide, the orientation of which is reversed. The complementary bases are A and T (or A and U), or C and G.<br>
The term "genes of the biosynthetic pathway for spiramycins" also comprises the regulatory genes and the genes imparting resistance to the producer microorganisms.<br>
According to the invention, the term "fragment" of a reference nucleic acid will be intended to mean a nucleotide sequence which is shorter compared to the reference nucleic acid and comprises, on the common portion, a nucleotide sequence identical to the reference nucleic acid.<br>
According to the invention, such a nucleic acid "fragment" may, where appropriate, be included in a larger polynucleotide of which it is a constituent.<br>
Such fragments comprise, or alternatively consist of, polynucleotides of length ranging from 8, 10, 12, 15, 18, 20 to 25, 30, 40, 50, 60, 70, 80, 90, 100, 150, 200, 250, 300, 350,400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900, 950, 1000, 1050, 1100,<br>
1150, 1200, 1250, 1300, 1350, 1400, 1450, 1500,1550, 1600,1650, 1700, 1750, 1800,<br>
»<br>
1850" or 1900 consecutive nucleotides of a nucleic acid accordins to the invention.<br><br>
According to the invention, the term "fragment" of a polypeptide will be intended to mean a polypeptide the amino acid sequence of which is shorter than that of the reference polypeptide and which comprises, over the entire portion common with these reference polypeptides, an identical amino acid sequence.<br>
Such fragments may, where appropriate, be included in a larger polypeptide of which they are part.<br>
Such fragments of a polypeptide according to the invention may be 10, 15, 20, 30 to 40, 50,60, 70, 80,90,100,120, 140, 160, 180,200, 220, 240, 260, 280, 300, 320, 340, 360, 380,400, 420,440,460,480, 500, 520, 540, 560, 580,600,620 or 640 amino acids in length.<br>
For the purpose of the present invention, the expression  "high  stringency hybridization conditions" will be intended to mean hybridization conditions which are unfavorable  to  the  hybridization  of nonhomologous  nucleic  acid  strands.  High stringency hybridization conditions can, for example, be described as conditions of hybridization in the buffer described by Church &amp; Gilbert (Church &amp; Gilbert, 1984) at a temperature of between 55°C and 65°C; preferably the hybridization temperature is 55°C, even more preferably the hybridization temperature is 60°C, and most preferably the hybridization temperature is 65°C, followed by one or more washes carried out in 2X SSC buffer (IX SSC buffer corresponds to an aqueous solution of 0.15M NaCU 15 mM of sodium citrate) at a temperature of between 55°C and 65°C; preferably this temperature is 55°C, even more preferably this temperature is 60°C, and most preferably this temperature is 65°C, followed by one or more washes in 0.5X SSC buffer at a temperature of between 55°C and 65°C; preferably this temperature is 55°C, even more preferably this temperature is 60°C, and most preferably this temperature is"65°C.<br>
It goes without saying that the hybridization conditions described above can be adjusted as a function of the length of the nucleic acid the hybridization of which is sought, or of the type of labeling chosen, according to techniques known to those skilled<br><br>
in the art. The suitable hybridization conditions can, for example, be adjusted according to the work by F. Ausubel et al (2002).<br>
The term "variant" of a nucleic acid according to the invention will be intended to mean a nucleic acid which differs by one or more bases compared to the reference polynucleotide. A variant nucleic acid may be of natural origin, such as a naturally found allelic variant, or may also be a non-natural variant obtained, for example, by mutagenesis techniques.<br>
In general, the differences between the reference nucleic acid and the variant nucleic acid are small such that the nucleotide sequences of the reference nucleic acid and of the variant nucleic acid are very close and, in many regions, identical. The nucleotide modifications present in a variant nucleic acid may be silent, which means that they do not modify the amino acid sequences encoded by said variant nucleic acid.<br>
However, the nucleotide changes in a variant nucleic acid may also result in substitutions, additions and/or deletions in the polypeptide encoded by the variant nucleic acid compared to the peptides encoded by the reference nucleic acid. In addition, nucleotide modifications in the coding regions may produce conservative or nonconservative substitutions in the amino acid sequence.<br>
Preferably, the variant nucleic acids according to the invention encode polypeptides which conserve substantially the same function or biological activity as the polypeptide of the reference nucleic acid, or else the ability to be recognized by antibodies against the polypeptides encoded by the initial nucleic acid.<br>
Some variant nucleic acids will thus encode mutated forms of the polypeptides, the systematic study of which will make it possible to deduce structure-activity relationships of the proteins in question.<br>
The term "variant" of a polypeptide according to the invention will mainly be intended to mean a polypeptide the amino acid sequence of which contains one or more substitutions, additions or deletions of at least one amino acid residue, compared to the<br><br>
amino acid sequence of the reference polypeptide, it being understood that the amino acid substitutions can be indifferently conservative or nonconservative.<br>
Preferably, the variant polypeptides according to the invention conserve substantially the same function or biological activity as the reference polypeptide, or else the ability to be recognized by antibodies against the initial polypeptide.<br>
For the purpose of the invention, polypeptide having "an activity similar" to a reference polypeptide is intended to mean a polypeptide having a biological activity close, but not necessarily identical, to that of the reference polypeptide as measured in a biological assay suitable for measuring the biological activity of the reference polypeptide.<br>
For the purpose of the invention, the term "hybrid antibiotic" is intended to mean a compound generated by constructing an artificial biosynthetic pathway using recombinant DNA technology.<br><br>
Detailed description of the invention<br>
A subject of the present invention is more particularly novel genes of the biosynthetic pathway for spiramycins and novel polypeptides involved in this biosynthesis as presented in the detailed description below.<br>
The genes of the biosynthetic pathway were cloned and the DNA sequence of these genes was determined. The sequences obtained were analyzed using the FramePlot program (J. Ishikawa &amp; K. Hotta, 1999). Among the open reading frames, those exhibiting a codon usage typical of Streptomyces were identified. This analysis showed that this region comprises 44 ORFs located on either side of five genes encoding the enzyme  "polyketide  synthase" (PKS),  and  exhibiting  a codon usage typical  of Streptomyces. On either side of these five genes encoding PKSs, 10 and 34 ORFs, respectively, were identified downstream and upstream (downstream and upstream being defined by the orientation of the 5 PKS genes all oriented in the same direction) (cf. figures 3 and 37). Thus, the 34 open reading frames of this type, occupying a region of approximately 41.7 kb (cf. SEQ ID No. 1 exhibiting a first region of 31 kb containing 25 ORFs and SEQ ID No. 140 exhibiting a region of approximately 12.1 kb, 1.4 kb of which overlap the preceding sequence (SEQ ID No. 1) and approximately 10.7 kb of which correspond to the subsequent sequence, the latter portion of   approximately 10.7 kb containing 9 additional ORFs (including one ORF of partial sequence) cf. also figures 3 and 37 below), were identified upstream of the 5 genes encoding the PKSs, and 10 occupying a region approximately 11.1 kb (SEQ ID No. 2 and figure 3) were identified downstream of the PKS genes. The 10 genes located downstream of the 5 PKS genes were thus named or/7*c, orf2*c, orf3*c, orf4*c, orfi*, orf6*, orf7*ct orf8*, orfi* and orflO* (SEQ ED No. 3, 5, 7, 9,11,13,15,17,19 and 21). The "c" added in the name of the gene signifies, for the ORF in question, that the coding sequence is in the reverse orientation (the coding strand is therefore the strand complementary to the sequence given in SEQ ID No. 2 for these genes). Using the same nomenclature, the 34 ORFs upstream of the PKS genes were named orfl, or/2, orf3, orf4, orf5, orf6, orf7, or/8, orftc, orflO, orfllc, orfl2, off 13c, orf!4,orf!5cf orfl6, orfl 7, orfl8, orfl9, orf20,<br><br>
orplc, orf22c, orf23c, orf24c, orf25ct orf26t orf27, or/28c, orf29, orf30c9 or/311 orf32c, orf33 and orf34c (SEQ ID Nos 23, 25, 28, 30, 34, 36, 40, 43, 45, 47, 49, 53, 60, 62, 64, 66,68,70,72,74,76,78,80,82,84,107, 109, 111,113,115,118,120, 141,143, 145, 147 and 149) (cf. figures 3 and 37).<br>
The protein sequences deduced from these open reading frames were compared with those present in various databases using various programs: BLAST (Altschul et a/., 1990) (Altschul et aL, 1997), CD-search, COGs (Cluster of Orthologous Groups) (these three programs are accessible in particular from the National Center for Biotechnology Information (NCBI) (Bethesda, Maryland, USA)), FASTA ((W.R. Pearson &amp; D.J. Lipman, 1988) and (W.R. Pearson, 1990), BEAUTY (K.C. Worley et aL, 1995)), (these two programs are accessible in particular from the INFOBIOGEN resource center; Evry, France). These comparisons made it possible to formulate hypotheses regarding the function of the products of these genes and to identify those liable to be involved in spiramycin biosynthesis.<br>
Genes located downstream of the genes encoding the PKSs<br>
A diagrammatic representation of the organization of the region is given in figure 3. As will be demonstrated below, on the 10 genes identified downstream of the genes encoding the PKSs, 9 appear to be involved in the biosynthesis of or the resistance to spiramycins. They are the following 9 genes: orfl*c, orf2*cr orf3*c, orf4*c, orfi*, orf6*, orf7*c, orf8* and orf9In table 1 below the references to the DNA sequence and the amino acid sequence of the 10 genes identified downstream of the 5 PKS genes are given.<br>
Table 1<br><br>
Gene	Position in the sequence SEQ ID No. 2	DNA sequence	Polypeptide sequences<br>
*•      -<br>
orfl*c	10882 to 10172	SEQ ID No. 3	SEQ ID No. 4<br>
orjl*c	10052 to 8781	SEQ ID No. 5	SEQ ID No. 6<br><br>
II 1	I		1<br>
The "c" added to the name of the gene indicates that the coding sequence is in the reverse orientation (the coding strand is therefore the strand complementary to the sequence given in SEQ ID No. 2 for these genes).<br>
With the aim of determining the function of the polypeptides identified, three types of experiment were carried' out: comparison of the identified sequences with sequences of known functions, gene inactivation experiments, leading to the construction of mutant strains, and analyses of the production of spiramycins and of spiramycin biosynthesis intermediates by these mutant strains.<br>
The protein sequences deduced from these open reading frames were first of all compared with those present in various databases using various programs: BLAST (Altschul et al, 1990) (Altschul et al, 1997), CD-search, COGs (Cluster of Orthologous Groups), _F AST A ((W.R, Pearson &amp;.DJ. Lipman, 1988) and. (W.R. Pearson, 1990), BEAUTY (K.C. Worley et al, 1995)). These comparisons made it possible to formulate hypotheses regarding the function of the products of these genes and to identify those liable to be involved in spiramycin biosynthesis. Table 2 shows the proteins exhibiting strong similarity with the products of the 10 genes located downstream of the 5 PKS genes.<br><br><br>
* a greater sequence similarity is associated with a higher BLAST score (Altschul etai, 1990).<br>
Gene inactivation experiments were carried out in order to confirm these results. The methods used consist in performing a gene replacement. The target gene to be Interrupted is replaced with a copy of this gene interrupted with a cassette imparting resistance to an antibiotic (for example apramycin, geneticin or hygromycin). The<br><br>
cassettes used are bordered on either side by translation termination codons in all the reading frames and by transcription terminators which are active in Streptomyces, Insertion of the cassette into the target gene may or may not be accompanied by a deletion in this target gene. The size of the regions flanking the cassette may range from a few hundred to several thousand base pairs. A second type of cassette can be used for the gene inactivation: cassettes termed "excisable cassettes". These cassettes have the advantage of being able to be excised in Streptomyces by a site-specific recombination event after having been introduced into the genome of 5. ambofaciens. The aim is to inactivate certain genes in strains of Streptomyces without leaving, in the final strain, selection markers or large DNA sequences not belonging to the strain. After excision, only a short sequence of about 30 base pairs (called "cicatricial" site) remains in the genome of the strain (cf. figure 10). The use of this system consists, initially, in replacing the wild-type copy of the target gene (by virtue of two homologous recombination events, cf. figure 9) with a construct in which an excisable cassette has been inserted into this target gene. The insertion of this cassette is accompanied by a deletion in the target gene (cf. figure 9). Secondly, the excision of the excisable cassette from the genome of the strain is brought about. The excisable cassette functions by virtue of a system of site-specific recombination and has the advantage of making it possible to obtain Streptomyces mutants which do not in the end carry a resistance gene. Possible polar effects on the expression of the genes located downstream of the inactivated gene(s) are also avoided (cf. figure 10). The strains thus constructed were tested for their spiramycin production.<br>
The or/7*c, orf2*c, orf3*c and orf4*c genes were not inactivated since the sequence comparison experiments made it possible to determine that these genes had a relatively high similarity with genes involved in the biosynthesis of a relatively close antibiotic. Thus, the orfl*c gene encodes a protein exhibiting 66% identity (determined using the BLAST program) with the protein encoded by the tylMI gene which encodes an N-methyltransferase which is involved in the biosynthesis of tylosine and which catalyzes 3-N-methylation during the production of mycaminose in Streptomyces fradiae (A.R. Gandecha et ai, 1997; GenBank accession number: CAA57473; BLAST score:<br><br>
287). This similarity with a protein involved in the biosynthetic pathway for another antibiotic relatively close, and more particularly in the biosynthesis of mycaminose, suggests that the orfl*c gene encodes an N-methyltransferase responsible for an N-methylation during the biosynthesis of forosamine or of mycaminose (cf. figures 5 and 6). This hypothesis is supported by the fact that the protein encoded by the orfl*c gene exhibits strong similarity with other proteins of similar function in other organisms (cf. table 3).<br>
I 		i       .	i	i	■<br>
* a greater sequence similarity is associated with a higher BLAST score (Altschul etaU 1990).<br>
The orf2*c gene encodes a protein exhibiting relatively strong similarity (35% identity) with a protein encoded by the tylMIII gene encoding an NDP hexose 3,4-isomerase involved in the biosynthesis of tylosine in Streptomyces fradiae (A.R. Gandecha et aL9 1997; GenBank accession number: CAA57471; BLAST score: 130). This similarity with a protein involved in the biosynthetic pathway of another<br>
close antibiotic, and more particularly in the biosynthesis of mycaminose, strongly<br>
»<br>
suggests that the orf*2c gene encodes an NDP hexose 3,4-isomeraSe" responsible for isomerization during the biosynthesis of one of the sugars of spiramycin, possibly mycaminose (cf. figures 5 and 6).<br><br>
The orfi*c gene encodes a protein exhibiting relatively strong similarity (59% identity) with a protein encoded by the tylMII gene encoding a glycosyltransferase involved in tylosine biosynthesis in Streptomyces fradiae (A.R. Gandecha et al, 1997; GenBank accession number: CAA57472; BLAST score: 448). This similarity with a protein involved in the biosynthetic pathway for another close antibiotic strongly suggests that the orf*3c gene encodes a glycosyltransferase. This hypothesis is supported by the fact that the protein encoded by the orf3*c gene exhibits strong similarity with other proteins of similar function in other organisms (cf. table 4).<br><br>
* greater sequence similarity is associated with a higher BLAST score (Altschul et c/., 1990).<br>
The orf4*cgene; encodes a protein exhibiting relatively strong similarity with several crotonyl-CoA reductases. In particular, the protein encoded by orf4*c has considerable similarity with a crotonyl-CoA reductase from Streptomyces coelicolor (M. Redenbach et ai9 1996; GenBank accession number: NP_630556; BLAST score: 772). This similarity with a protein involved in the biosynthetic pathway for another close antibiotic strongly suggests that the orf4*c gene also encodes a crotonyl-CoA reductase. This hypothesis is supported by the fact that the protein encoded by the orf4*c<br><br>
gene exhibits strong similarity with other proteins of similar function in other organisms (cf. table 5).<br>
— -        — ■	— -■	•	•     ■   — - ■	— —   -   ~<br>
* greater sequence similarity is associated with a higher BLAST score (Altschul et<br>
a/., 1990).<br>
The orf6* gene exhibits a certain similarity with the mdmB gene present in Streptomyces mycarofaciens (Hara and Hutchinson, 1992; GenBank accession number: A42719; BLAST score: 489) which produces macrolide antibiotic. In this producer, the gene is involved in the acylation of the lactone ring. The orf6* gene is therefore thought to encode an acyltransferase. This hypothesis is supported by the fact that the protein encoded by the orf6* gene exhibits strong similarity with other proteins of similar function in other organisms (cf. table 6).<br><br>
' - ■■-	-	._■■■!	I	j			        -    ■ ■	'<br>
* greater sequence similarity is associated with a higher BLAST score (Altschul et aL, 1990).<br>
Inactivation of the orf6* gene was produced by an in-phase deletion/inversion and it showed that the resulting strain no longer produces spiramycin II and EQ but only spiramycin I (cf. figure 1). This confirms that the orf6* gene is indeed involved in the synthesis of spiramycin II and IE. The enzyme encoded by this gene is responsible for the formation of spiramycin II and III by attachment of an acetyl or butyryl group to the carbon in the 3-position, The strains which no longer express the protein encoded by the orf6* gene are particularly advantageous since they no longer produce spiramycin II and HI but only spiramycin I. As specified above, the antibiotic activity of spiramycin I is clearly greater than that of spiramycins II and in (Liu et aL, 1999).<br>
The orfi* gene encodes a protein exhibiting relatively strong similarity with several O-methyltransferases. In particular, the protein encoded by or/5* has considerable similarity with an O-methyltransferase (EC 2.1.1.-) MdmC from Streptomyces mycarofaciens (Hara &amp; Hutchinson, 1992; GenBank accession number:<br><br>
142719; BLAST score: 355). This similarity with a protein involved in the biosynthetic iathway for another antibiotic strongly suggests that the orf5* gene also encodes an )-methyltransferase. The orfi* gene is thought to be involved in the formation of irecursors incorporated into the lactone ring. In fact, according to the sequence ;omparisons, the product of the orf5* gene is also relatively close to FkbG, which is esponsible for the methylation of hydroxymalonyl-ACP according to (Wu et al, 2000; ioffineister et al9 2000; GenBank accession number: AAF86386; BLAST score: 247) cf figure 8). This hypothesis is supported by the fact that the protein encoded by the orf* gene exhibits strong similarity with other proteins of similar function in other 3rganisms (cf. table 7).<br><br>
* greater sequence similarity is associated with a higher BLAST score (Altschul et al, 1990).<br>
Due to the polar effect of the insertion of a nonexcised cassette into the orf 6* gene, it has been possible to determine that the orf 5* gene is essential to the biosynthetic pathway for spiramycins. Specifically, insertion of the excisable cassette into the coding portion of the orf 6* gene leads to a complete arrest of spiramycin production. However, once the inserted cassette has been excised (and therefore when only the orf6* gene is inactivated, cf. examples 14 and 15), production of spiramycin I is observed again. This shows that the orf'5* gene is essential to the biosynthetic pathway<br><br>
for spiramycins since its inactivation leads to a complete arrest of spiramycin<br>
production.<br>
The orf5* gene encodes a protein exhibiting relatively strong similarity with several O-methyltransferases. The orfi* gene is thought to be an O-methyltransferase involved either directly in the synthesis of the platenolide, or in the synthesis of a methylated precursor (methoxymalonyl, see figure 8) incorporated into the platenolide by PKS. To verify this hypothesis, LC/SM and NMR analytical experiments were carried out on a strain of 5. ambofaciens of genotype : orf6*::attlQhyg+. In this strain, the orf5* gene is not expressed, owing to the polar effect of the insertion, into the orf6* gene, of the cassette which contains transcription terminators (cf. example 27). It has been shown that this strain produces a molecule for which the UV spectrum has an appearance similar to that of spiramycin I, but the mass spectrum shows a molecular ion at 829. The difference in mass of 14 compared to the mass of spiramycin can be explained by the absence of methyl on the oxygen borne by carbon No. 4 of the lactone ring (the structure of this compound is given in figure 39). The Tesults obtained by NMR are compatible with this hypothesis. The presence of a compound at 829 makes it possible to validate the hypothesis of the role of orp* in spiramycin biosynthesis. In addition, the product corresponding to spiramycin without the methyl group exhibits very weak microbiological activity (10-fold weaker) compared with the unmodified spiramycin, when tested on the microorganism Micrococcus luteus.<br>
The or/7*c gene encodes a protein exhibiting relatively strong similarity with a protein encoded by the mdmA of Streptomyces mycarofaciens, the latter gene encoding a protein involved in midecamycin resistance in the producer enzyme (Hara et a/., 1990; GenBank accession number: A60725; BLAST score: 380). This similarity with the protein involved in the biosynthetic pathway for another antibiotic strongly suggests that the orf7*c gene also encodes a protein involved in spiramycin resistance. More particularly, the enzyme encoded by the orf7*c gene has methyltransferase activity and is involved in spiramycin resistance in Streptomyces ambofaciens. It has been demonstrated that this gene imparts resistance of the MLS I type, which resistance is<br><br>
known to be due to monomethyiation, at position A2058 of 23 S ribosomal RNA (Pernodet et al, 1996). This hypothesis is supported by the fact that the protein encoded by the orf7*c gene exhibits strong similarity with other proteins of similar function in other organisms (cf, table 8).<br><br>
* greater sequence similarity is associated with a higher BLAST score (Altschul et al9 1990).<br>
The or/8* gene encodes a protein exhibiting relatively strong similarity with a protein of the ABC transporter type in Streptomyces griseus (Campelo, 2002, GenBank accession number: CAC22119; BLAST score: 191). This similarity with a protein of the ABC transporter type strongly suggests that the orf8* gene also encodes a protein of the ABC transporter type which may be involved in spiramycin resistance. This hypothesis is supported by the fact that the protein encoded by the orf8* gene exhibits strong similarity with other proteins of similar function in other organisms (cf. table 9).<br><br><br>
* greater sequence similarity is associated with a higher BLAST score (Altschul ei al, 1990).<br>
The orf9* gene encodes a protein exhibiting relatively" strong similarity with a protein of the ABC transporter type in Streptomyces griseus (Campelo, 2002, GenBank accession number: CAC22118; BLAST score: 269). This similarity with a protein of the ABC transporter type strongly suggests that the or/9* gene also encodes a protein of the ABC transporter type which may be involved in spiramycin resistance. This hypothesis is supported by the fact that the protein encoded by the orf9* gene exhibits strong similarity with other proteins of similar function in other organisms (cf. table 10).<br><br><br>
* greater sequence similarity is associated with a higher BLAST score (Altschul et aL, 1990).<br>
The orflO* gene encodes a protein exhibiting relatively strong similarity with a protein of unknown function. However, genes similar to orflO* are found in the middle of several groups of genes involved in the biosynthesis of antibiotics. Thus, a gene close -to orflO* is found in S. coelicolor (Redenbach et aL9 1996, GenBank accession number: NP_627432, BLAST score: 109). A close gene (CouY) is also found in S. rishiriensis (Wang et al, 2000, GenBank accession number: AAG29779, BLAST score 97).<br>
Genes located upstream of the genes encoding PKSs<br>
In the DNA sequence located upstream of the genes encoding PKSs (downstream and upstream being defined by the orientation of the 5 PKS genes all oriented in the same direction) (cf. figure 3), 34 ORFs have been identified (cf. above). Thus, the 34 open reading frames of this type occupy a region of approximately 41.7 kb (cf. SEQ ED No. 1 exhibiting a first region of 31 kb containing 25 ORFs and SEQ ID No. 140 exhibiting a region of approximately 12.1 kb, 1.4 kb of which overlap the preceding sequence (SEQ ID No. 1) and approximately 10.7 kb of which correspond to the subsequent sequence), the latter portion of approximately 10.7 kb containing 9 additional ORFs (including an ORF partial sequence), cf. also figures 3 and 37 below).<br><br><br>
1 The "c" added to the name of the gene*indicates that the coding sequence is in the reverse orientation (the coding strand is therefore the strand complementary to the sequence given in SEQ ID No. 1 or SEQ ID No. 140 for these genes).<br><br>
" When several protein'sequences are indicated for a single orf. the corresponding proteins are derived from several possible translation initiation sites.<br>
Three types of experiment were carried out with the aim of determining the function 0f the polypeptides identified in table 11 above:- comparison of the identity of the identified sequences with sequences of known functions, gene inactivation experiments, and analyses of spiramycin production by these mutant strains.<br>
The protein sequences deduced from these open reading frames were all first of all compared with those present in various databases using various programs: BLAST (Altschul et aL, 1990) (Altschul et a/., 1997), CD-search, COGs (Cluster of Orthologous Groups), FASTA ((W.R. Pearson &amp; D.J. Lipman, 1988) and (W.R. Pearson, 1990), BEAUTY (K.C. Worley et air 1995)), (cf. above). These comparisons made it possible to formulate hypotheses regarding the function of the products of these genes and to identify those liable to be involved in spiramycin biosynthesis. Table 12 shows the proteins exhibiting strong similarity with the 34 genes located upstream of the 5 PKF genes.<br><br><br><br><br>
* greater sequence similarity is associated with a higher BLAST score (Altschul ei al., 1990).<br>
Gene inactwation experiments were carried out in order to confirm these results. The methods used consist in performing a gene replacement. The target gene to be interrupted is replaced with a copy of this gene interrupted with a cassette imparting resistance to an antibiotic (for example apramycin or hygromycin). The cassettes used are bordered en either side by translation termination codens in all reading frames and by transcription terminators which are active in Streptomyces. Insertion of the cassette into the target gene may or may not be accompanied by deletion in this target gene. The size of the regions flanking the cassette may range from a few hundred to several thousand base pairs. A second type of cassette may be used for the gene inactivation: cassettes termed "excisable cassettes" (cf. above). The strains thus constructed were tested for their spiramycin production.<br>
The orfl gene encodes a protein exhibiting relatively strong similarity with several cytochrome P450s...In particular, the protein encodied by orfl has considerable similarity with the protein encoded by the tyil gene involved in the biosynthesis of tylosine in Streptomyces fradiae (L.A. Merson-Davies et al, 1994; GenBank accession number: S49051; BLAST score: 530). This similarity with a protein involved in the biosynthetic pathway for another close antibiotic strongly suggests that the orfl gene also encodes a cytochrome P450. This hypothesis is supported by the fact that the protein encoded by the orfl gene exhibits strong similarity with other proteins of similar function in other organisms (cf. table 13).<br><br><br><br>
* greater sequence similarity is associated with a higher BLAST score (Altschul et aL, 1990).<br>
The orfl gene encodes a protein exhibiting relatively strong similarity with a dTDP-6-deoxy-3,4-ketohexulose isomerase of Aneurinibacillus thermoaerophilus (Pfoestl, A. et al.9 2003, GenBank accession number: AAO06351; BLAST score: 113). This similarity strongly suggests that the orfl gene encodes an isomerase responsible for the isomerization reaction required for the biosynthesis of one of the sugars present in the spiramycin molecule, this sugar possibly being mycarose (cf. figure 5). Inactivation of the orfl gene was performed. It could be shown that the resulting strain no longer produces spiramycins. This confirms that the orfl gene is indeed involved in spiramycin biosynthesis.<br>
The or/3 gene encodes a protein exhibiting relatively strong similarity with several aminotransferases. In particular, the protein encoded by orf3 has considerable similarity with an aminotransferase of Streptomyces antibioticus involved in oleandomycin biosynthesis (G. Draeger et al, 1999; GenBank accession number: AAF59939; BLAST score: 431). This similarity with a protein involved in the biosynthetic pathway for another close antibiotic strongly suggests that the or/3 gene encodes a 3-aminotransferase responsible for the transamination reaction necessary for the biosynthesis of one of the amino sugars of spiramycins (cf. figure 5). This hypothesis is supported by the fact that the protein encoded by the orfl gene exhibits strong similarity with other proteins of similar function in other organisms (cf. table 15).<br><br>
L ....    -   		■- -   .     	_. -	-  _	1	.    .<br>
* greater sequence similarity is associated with a higher BLAST score (Altschul et •  aL, 1990).<br>
The or/3 gene was inactivated. It was thus possible to show that the resulting strain no longer produces spiramycins. This confirms that the orf3 gene is indeed involved in spiramycin biosynthesis. The enzyme encoded by this gene is therefore clearly responsible for a byconversion step essential to spiramycin biosynthesis" Spiramycin production can be complemented by the expression of the TylB protein of S. fradiae (cf. example 23). This demonstrates that the or/3 gene encodes a 3-aminotransferase responsible for the transamination reaction necessary for mycaminose biosynthesis (cf. figure 5). Since mycaminose is the first sugar to be attached to platenolide, the strain with the orf3 knockout (OS49.67) is expected to accumulate platenolide.<br>
The biosynthesis intermediates of the strain with the orfS gene knockout were studied (cf. example 20). These experiments made it possible to demonstrate that this strain produces two forms of platenolide: platenolide A and platenolide B, the deduced structure of these two molecules is given in figure 36. This strain also produces platenolide A + mycarose and platenolide B + mycarose (cf. example 20 and figure 40). These compounds comprise a sugar but do not comprise an^mycaminose. In addition, if they are compared with spiramycin (cf. figure 1), these compounds comprise mycarose in place of mycaminose. These results are in agreement with the product of the or/3 gene being involved in mycaminose biosynthesis and it having a role as a 3 aminotransferase<br><br>
responsible for the transamination reaction required for mycaminose biosynthesis (cf. figure 5). It may be noted that the specificity of glycosylation does not appear to be absolute since molecules with mycarose attached at the position normally occupied by mycaminose are found (cf. figure 40).<br>
The orf4 gene encodes a protein exhibiting relatively strong similarity with several NDP-glucose synthetases. In particular, the protein encoded by orf4 has considerable similarity with an alpha-D-glucose-1 -phosphate thymidylyltransferase of Streptomyces venezuelae (Y. Xue et al, 1998; GenBank accession number: AAC68682; BLAST score: 404). This similarity with a protein involved in the biosynthetic pathway for another close antibiotic strongly suggests that the orf4gene encodes an NDP-glucose synthetase responsible for the synthesis of NDP-glucose necessary for the biosynthesis of the three atypical sugars incorporated into the spiramycin molecule (cf. figures 4, 5 and 6). This hypothesis is supported by the fact that the protein encoded by the orf4 gene^ exhibits strong similarity with other proteins of similar function in other cttiganisms (cf. table 16).<br><br><br>
* greater sequence similarity is associated with a higher BLAST score (Altschul et aU 1990).<br>
The orp gene encodes a protein exhibiting relatively strong similarity with several glucose dehydratases. In particular, the protein encoded by orf5 has considerable similarity with a dTDP-glucose 4,6-dehydratase of Streptomyces tenebrarius (T.B. Li et al, 2001; GenBank accession number: AAG18457, BLAST score: 476). This similarity with a protein involved in the biosynthetic pathway for another close antibiotic strongly suggests that the orp gene encodes an NDP-glucose dehydratase necessary for the biosynthesis of the three atypical sugars incorporated into the spiramycin molecule (cf. figures 4, 5 and 6). This hypothesis is supported by the fact that the protein encoded by the orp gene exhibits strong similarity with other proteins of similar function in other organisms (cf. table 17).<br><br><br><br><br>
* greater sequence similarity is associated with a higher BLAST score (Altschul et al, 1990).<br>
The orp gene encodes a protein exhibiting relatively strong similarity with several hexose dehydratases. In particular, the protein encoded by orp has considerable similarity with a dNTP-hexose 2,3-dehydratase (encoded by the TyiCVI gene) of Streptomyces fradiae involved in tyiosine biosynthesis (LA. Merson-Davies et al, 1994; GenBank accession number: AAF29379; BLAST score: 461). This similarity with a protein involved in the biosynthetic pathway for another close antibiotic strongly suggests that the orp gene also encodes a hexose 2-3-dehydratase necessary for the biosynthesis of two atypical sugars incorporated into the spiramycin molecule (cf. figures 4 and 6). This hypothesis is supported by the fact that the protein encoded by the orp gene exhibits strong similarity with other proteins of similar function in other organisms (cf. table 19).<br><br>
* greater sequence similarity is associated with a higher BLAST score (Altschul et al., 1990).<br>
The orfE gene encodes a protein exhibiting relatively strong similarity with several aminotransferases. In particular, the protein encoded by or/8 has considerable similarity with an aminotransferase probably involved in forosamine biosynthesis in Saccharopolyspora spinosa (C. Waldron et al, 2001; GenBank accession number: AAG23279; BLAST score: 465).  This similarity with a protein involved  in the<br><br>
biosynthetic pathway for another close antibiotic strongly suggests that the orf8 gene encodes a 4-aminotransferase responsible for the transamination reaction necessary for forosamine biosynthesis (cf. figure 6). This hypothesis is supported by the fact that the protein encoded by the orfB gene exhibits strong similarity with other proteins of similar function in other organisms (cf. table 20).<br><br>
* greater sequence similarity is associated with a higher BLAST score (Altschul et al., 1990).<br>
The orf8 gene was inactivated. It was thus possible to show that the resulting strain no longer produces spiramycins. This confirms that the orf8 gene is indeed involved in spiramycin biosynthesis. The enzyme encoded by this gene is therefore clearly responsible for a bioconversion step essential to spiramycin biosynthesis. Validation of the hypothesis of the role played by the product of the orf8 gene in forosamine biosynthesis is provided by the fact that an inactivated mutant for the orf8 gene produces forocidin, this mutant is therefore blocked at the forocidin stage and does not produce any neo-spiramycin (cf. figure 7 and example 25). These results are in agreement with the product of the orfB gene being involved in forosamine biosynthesis (cf. figure 6).<br>
The orf9c gene has already been identified in Streptomyces ambofaciens and has been named srmX by Geistlich et ah (M. Geistlich ex ah, 1992). The similarity of the protein encoded by this gene with several - methyltransferases involved in ,the biosynthetic pathway for other close antibiotics strongly suggests that the or9c gene encodes a methyltransferase responsible for the methylation reaction necessary for mycaminose or forosamine biosynthesis (cf. figures 5 and 6). This hypothesis is<br><br>
supported by the fact that the protein encoded by the orf9c gene exhibits strong similarity with other proteins of similar function in other organisms (cf. table 21).<br>
- ■        — - — -■ --■—- ~   —      -	i	■	■     .	...<br>
* greater sequence similarity is associated with a higher BLAST score (Altschul et<br>
al, 1990).<br>
The orflQ gene has already been identified in Streptomyces ambofaciens and has been named srmR by Geistlich et al (M. Geistlich et al, 1992). The protein encoded by this gene is involved in regulation of the biosynthetic pathway for spiramycins in Streptomyces ambofaciens. The orflO gene was inactivated. It was thus possible to show that the resulting strain no longer produces spiramycins. This confirms that the orflQ gene is indeed involved in spiramycin bisynthesis. The protein encoded by this gene is therefore clearly essential for spiramycin biosynthesis.<br>
In addition, the translation initiation point of orflO was determined and it was possible to show that overexpression of this gene leads to an improvement in the production of spiramycins. The translation initiation site corresponds to an ATG located upstream of the ATG proposed by Geistlich et al (M. Geistlich et al, 1992). It was also demonstrated that this 5' end is essential to the function of OrflO since a 5'-truncated<br><br>
messenger is inactive (cf. example 17). To obtain the desired effect on the production of spiramycins, it is therefore essential for the overexpression of orf10 to be effected while taking care not to express a 5'-truncated messenger of orf10<br>
The affile gene has already been identified in Streptomyces ambofaciens and has been named srmB by Geistlich et al (M. Geistlich et al, 1992) and Schoner et al (B. Schoner et al, 1992). The protein encoded by this gene is involved in spiramycin resistance in Streptomyces ambofacietis and is an ABC-type transporter.<br>
The orfl2 gene encodes a protein exhibiting relatively strong similarity with several hexose dehydratases. In particular, the protein encoded by orfl2 has considerable similarity with an NDP-hexose 3,4-dehydratase encoded by the UrdQ -gene of Streptomyces fradiae and involved in urdamycin biosynthesis (D. Hoffineister et al, 2000; GenBank accession number: AAF72550; BLAST score: 634). This similarity with a protein involved in the biosynthetic pathway for another close antibiotic strongly suggests that the orfl2 gene encodes a 3,4 dehydratase responsible for the dehydration reaction necessary for forosamine biosynthesis (cf. figure 6). This hypothesis is supported by the fact that the protein encoded by the orfl2 gene exhibits strong similarity with other proteins of similar function in other organisms (cf. table 22).<br><br><br><br>
* greater sequence similarity is associated with a higher BLAST score (Altschul et<br>
a/., 1990).	.  -<br>
The off!2 gene was inactivated. It was possible to show that the resulting strain no longer produces spiramycins. This confirms that the orf12 gene is indeed involved in spiramycin biosynthesis. The  enzyme  encoded by this gene is  therefore clearly responsible for a byconversion step essential to spiramycin biosynthesis. Validation of the hypothesis of the role played by Orfl2 in forosamine biosynthesis is provided by the fact that an inactivated mutant for the orfll gene no longer produces forosamine. However, it produces a small amount of forocidin. This mutant is therefore blocked at the forocidin stage and does not produce any neo-spiramycin (cf. figure 7 and example 26). This mutant also produces a compound having the structure shown in figure 38. The latter compound contains two sugars, mycaminose and mycarose, but does not contain any forosamine. In addition, if it is compared with the structure of spiramycin (c£ figure \\ this compound contains the sugar.mycarose in..the expected place of forosamine. These results are in agreement with the product of the orfll gene being involved in forosamine biosynthesis (cf. figure 6). It may be noted that the specificity of glycosylation is not absolute since molecules in which mycarose is attached at the position normally occupied by forosamine are observed (see figure 38). *<br>
The orfl3c gene encodes a protein exhibiting relatively strong similarity with a protein of unknown function in Streptomyces coelicolor. This protein was named<br><br>
SC4H2.17 (GeneBank accession number: T35116; BLAST score: 619). The protein encoded by the orfl3c gene also exhibits strong similarity with other proteins of other organisms (cf. table 23).<br><br>
* greater sequence similarity is associated with a higher BLAST score (Altschul et al, 1990).<br>
No precise function has been attributed to the proteins close to that encoded by orf!3c. The orfttc gene was inactivated for the purpose of studying the function of this gene in the biosynthetic pathway for spiramycins in Streptomyces ambofaciens. It was possible to show that the resulting strain produces spiramycins. This indicates that the orf 13c gene is not essential to spiramycin biosynthesis and that it is not essential to the survival of the bacterium. The enzyme encoded by this gene is not therefore responsible for a bioconversion step essential to spiramycin biosynthesis.<br>
The orf 14 gene encodes a protein exhibiting relatively strong similarity with a putative reductase (M. Redenbach et al, 1996; Bentley et al, 2002; GenBank accession number: CAB90862; BLAST score: 147).<br><br>
The orf 14 gene was inactivated. It was thus possible to show that the resulting strain no longer produces spiramycins. This confirms that the orfl4 gene is indeed involved in spiramycin biosynthesis. The enzyme encoded by this gene is therefore clearly responsible for a bioconversion step essential to spiramycin biosynthesis. The biosynthesis intermediates of the strain with the orfl4 gene knockout were studied (cf. example 20). These experiments made it possible to demonstrate that this strain produces platenolide A but does not produce platenolide B (cf. figure 36).<br>
The orf15c gene encodes a protein exhibiting relatively strong similarity with several keto reductases. In particular, the protein encoded by orflSc has considerable similarity with a 3-keto reductase in Streptomyces antibioticus (GenBank accession number: T51102, BLAST score: 285). This similarity strongly suggests that the orfl5c gene encodes a 3-keto reductase responsible for the reduction reaction necessary for forosamine biosynthesis (cf. figure 6). This hypothesis is-supported by the fact that the protein encoded by the orfl5c gene exhibits strong similarity with other proteins of similar function in other organisms (cf table 24).<br><br><br><br>
* greater sequence similarity is associated with a higher BLAST score (Altschul et aL, 1990).<br>
The orf 16 gene encodes a protein exhibiting relatively strong similarity with several isomerases. In particular, the protein encoded by orfl6 has considerable similarity with an NDP-hexose 3,4-isomerase in Streptomyces fradiae (Gandecha et aL, 1997; GenBank accession number: CAA57471, BLAST score: 209). This similarity strongly suggests that the orf16 gene encodes a protein involved in the biosynthesis of one of the sugars of spiramycin (cf. figures 5 and 6). This hypothesis is supported by the fact that the protein encoded by the orf16 gene exhibits strong similarity with other proteins of similar function in other organisms (cf. table 25).<br><br>
* greater sequence similarity is associated with a higher BLAST score (Altschul et al, 1990).<br>
The orf 17 gene encodes a protein exhibiting-relatively strong similarity with several glycosyltransferases. In particular, the protein encoded by orf!7 has considerable similarity with a glycosyltransferase of Streptomyces venezuelae (Y. Xue et aL, 1998;<br><br>
GenBank accession number AAC68677; BLAST score: 400). The similarity of the protein encoded by the orf1 7 gene with several glycosyltransferases involved in the biosynthetic pathway for other close antibiotics strongly suggests that this gene also encodes a glycosyltransferase. This hypothesis is supported by the fact that the protein encoded by the orfl 7 gene exhibits some similarity with other proteins of similar function in other organisms (cf. table 26).<br>
* -    -	»	i	■		.	_—j<br>
* greater sequence similarity is associated with a higher BLAST score (Altschul et<br>
aL91990).<br>
The orfl 8 gene encodes a protein exhibiting relatively strong similarity with several glycosyltransferases. In particular, the protein encoded by orfl8 has considerable similarity with a glycosyltransferase of Streptomyces rishiriensis (Wang et aL, 2000; GenBank accession number: AAG29785; BLAST score: 185). The similarity of the protein encoded by the orfl8 gene with several glycosyltransferases involved in the biosynthetic pathway for other close antibiotics strongly suggests that this gene also<br><br>
encodes a glycosyltransferase. This hypothesis is supported by the fact that the protein encoded by the orfl8 gene exhibits strong similarity with other proteins of similar function in other organisms (cf. table 27).<br><br>
* greater sequence similarity is associated with a higher BLAST score (Altschul et aU 1990).<br>
The orfl9 gene encodes a protein exhibiting relatively strong similarity with several keto reductases. In particular, the protein encoded by orfl9 has considerable similarity with an NDP-hexose 4-keto reductase (TylCTV) Streptomyces fradiae (Bate et al, 2000; GenBank accession number: AAD41822; BLAST score: 266). The similarity of the protein encoded by the orfl9 gene with this keto reductase involved in the biosynthetic pathway for a close antibiotic strongly suggests that this gene also encodes a 4-keto reductase responsible for the reduction reaction necessary for mycarose biosynthesis (cf. figure 4). This hypothesis is supported by the fact that the protein<br><br>
encoded by the orfl9 gene exhibits strong similarity with other proteins of similar function in other organisms (cf. table 28).<br><br>
* greater sequence similarity is associated with a higher BLAST score (Altschul et al., 1990),<br>
The or/20 gene encodes a protein exhibiting relatively strong similarity with several hexose reductases. In particular, the protein encoded by orfiO has considerable similarity with the EryBII of Saccharopolyspora erythraea which encodes a dTDP-4-keto-L-6-deoxyhexose 2,3-reductase (R.G. Summers et al. 1997), GenBank accession number: AAB84068; BLAST score: 491). The similarity of the protein encoded by the orf2Oc gene with several hexose reductases involved in the biosyhthejic pathway for other close antibiotics strongly suggests that this gene encodes a 2,3-reductase responsible for the reduction necessary for mycarose biosynthesis (cf. figure 4). This<br><br>
hypothesis is supported by the fact that the protein encoded by the orflOc gene exhibits strong similarity with other proteins of similar function in other organisms (cf. table 29).<br>
\ .    -	-	|-	.     —     I —.	-!■,--.      1-1     I— —    ,.      ,	,	■■■■■———■	„.	.	.1—     -|	.,_      -	|	I	.,     m^^    -       i.	_     ,.      —    ,	.	,     I.	—t—I..I.-I,     —	-      —     —-	I	I.	-	-    —I,,—■<br>
* greater sequence similarity is associated with a higher BLAST score (Altschul et aU 1990).<br>
The orf21c gene encodes a protein exhibiting relatively strong similarity with several hexose methyltransferases. In particular, the protein encoded by orf21c has considerable similarity with the 7ylC111 gene of Streptomyces fradiae which encodes an NDP-hexose 3-C-methyltransferase (N. Bate et al, 2000; GenBank accession number: AAD41823; BLAST score: 669). The similarity of the protein encoded by the orfllc gene with several hexose methyltransferases involved in the biosynthetic pathway for other close antibiotics strongly suggests that this gene encodes a hexose methyltransferase responsible for the methylation reaction necessary for mycarose biosynthesis. (cf. figure 4). This hypothesis is supported by the fact that the protein encoded by the orf2lc gene exhibits strong similarity with other proteins of similar function in other organisms (cf. table 30).<br><br><br>
* greater sequence similarity is associated with a higher BLAST score (Altschul et al., 1990).<br>
The orf22c gene encodes a protein exhibiting relatively strong similarity with the protein encoded .by the fkbH gene of Streptomyces hygroscopicus var. ascomyceticus which encodes an enzyme involved in methoxymalonyl biosynthesis (K. Wu et aL, 2000; GenBank accession number: AAF86387; BLAST score: 463). The similarity of the protein encoded by the orf22c gene with this protein involved in the biosynthetic pathway for another close macrolide strongly suggests that this gene also encodes an enzyme involved in methoxymalonyl biosynthesis in Streptomyces ambofaciens (cf. figure 8).<br>
The orfl23c gene encodes a protein exhibiting relatively strong similarity with the protein encoded by the fkbl gene of Streptomyces hygroscopicus var. ascomyceticus which encodes an acyl-CoA dehydrogenase involved in methoxymalonyl (K. Wu, et a/., 2000; GenBank accession number: AAF86388; BLAST score: 387). The similarity of the protein encoded by the orfllc gene with several acyl-CoA dehydrogenases involved in the biosynthetic pathway for other close antibiotics strongly suggests that this gene encodes an acyl-CoA dehydrogenase involved in methoxymalonyl biosynthesis (cf. figure 8). This hypothesis is supported by the fact that the protein encoded by the<br><br>
orf23c gene exhibits strong similarity with other proteins of similar function in other organisms (cf. table 31).<br><br>
* greater sequence similarity is associated with a higher BLAST score (Altschul et al., 1990).<br>
The orj24c gene encodes a protein exhibiting relatively strong similarity with the protein encoded by the jkbJ gene of Streptomyces hygroscopicus var. ascomyceticus which is thought to encode the acyl carrier protein (ACP) involved in methoxymalonyl biosynthesis (K. Wu et al., 2000; GenBank accession number: AAF86389; BLAST score: 87). The similarity of the protein encoded by the orf24c gene with this protein involved in the biosynthetic pathway for another close macrolide strongly suggests that this gene encodes a protein involved in methoxymalonyl biosynthesis in Streptomyces ambofaciens (cf. figure 8).<br>
The orf25c gene encodes a protein exhibiting a relatively strong similarity with the protein encoded by thefkbK gene of Streptomyces hygroscopicus var. ascomyceticus which encodes an acyl-CoA dehydrogenase involved in methoxymalonyl biosynthesis (K. Wu et al, 2000; GenBank accession number: AAF86390; BLAST score: 268). The similarity of the protein encoded by the orf25c gene with several acyl-CoA dehydrogenases involved in the biosynthetic pathway for other close antibiotics strongly suggests   that   this    gene   encodes    an   acyl-CoA   dehydrogenase    involved    in<br><br>
methoxymalonyl biosynthesis (cf. figure 8). This hypothesis is supported by the fact that the protein encoded by the orf25c gene exhibits strong similarity with other proteins of similar function in other organisms (cf. table 32).<br><br>
* greater sequence similarity is associated with a higher BLAST score (Altschul et al., 1990).<br>
The orf26 gene encodes a protein exhibiting 65% identity (determined using the BLAST program) with the protein encoded by the tylCV gene which encodes a mycarosyltransferase involved in tylosine biosynthesis in Streptomyces fradiae (N. Bate, et al., 2000; GenBank accession number: AAD41824, BLAST score: 471). More particularly, TylCV is a glycosyltransferase which binds the mycarose molecule during tylosine synthesis. This similarity with a protein involved in the biosynthetic pathway for another relatively close antibiotic, and more particularly in mycarose transfer, suggests that the orf26 gene is a glycosyltransferase. This hypothesis is supported by the fact that the protein encoded by the or/26 gene exhibits strong similarity with other proteins of similar function in other organisms (cf. table 33).<br><br><br>
* greater sequence similarity is associated with a higher BLAST score (Altschul et al.y 1990).<br>
The orf27 gene encodes a protein exhibiting a 70% identity (determined using the BLAST program) with the protein encoded by the tylCVIl gene which encodes an NDP-hexose 3,5- (or 5-) epimerase involved in tylosine biosynthesis in Streptomyces fradiae (N. Bate et a/., 2000; GenBank accession number: AAD41825, BLAST score: 243). More particularly, TylCVIl is a hexose 3,5- (or 5-) epimerase involved in mycarose biosynthesis. This similarity with a protein involved in the biosynthetic pathway for another relatively close antibiotic, and more particularly in mycarose biosynthesis, suggests that the orf27 gene encodes an epimerase. This hypothesis is supported by the fact that the protein encoded by the orf27 gene exhibits strong similarity with other proteins of similar function in other organisms (cf. table 34). Analysis of the close sequences obtained using the BLAST program strongly suggests that the or/27 gene encodes a 5-epimerase responsible for the epimerization reaction necessary for mycarose biosynthesis (cf. figure 4).<br><br><br>
* greater sequence similarity is associated with a higher BLAST score (Altschul et ai, 1990).<br>
The sequence of orj28c was initially determined partially, since the sequence of a region of approximately 450 base pairs was only determined after resequencing (this region is symbolized by "N" in the incomplete sequence SEQ ID No. 106). The partial sequence of this ORF (SEQ ID No. Ill) was nevertheless used for the analysis with the various computer programs as explained above. It was thus possible to determine that the orf28c gene encodes a protein exhibiting 64% identity, over the determined sequence (SEQ ID No. 112, which is the partial sequence of the Orf28c protein) (determined using the BLAST program), with the protein encoded by the acyB2 gene which encodes a regulatory protein involved in carbomycin biosynthesis in Streptomyces thermotolerans (A. Arisawa et ai9 1993; GenBank accession number: JC2032, BLAST score; 329). This similarity with a protein involved in the biosynthetic pathway for a relatively close antibiotic suggests that the orf28c gene encodes a regulatory protein involved in spiramycin biosynthesis. This hypothesis is supported by the fact that the protein encoded by the orf28c gene also exhibits strong similarity with the TylR protein, which<br><br>
is a regulatory protein involved in tylosine biosynthesis in Streptomyces fradiae (N. Bate et al., 1999; GenBank accession number: AAF29380, BLAST score: 167).<br>
It was possible to amplify the orf28c gene using oligonucleotides located on either side of the undetermined sequence and to subclone it into an expression vector. It was thus possible to demonstrate that overexpression of the orf28c gene significantly increases spiramycin production in the OSC2 strain (cf. example 24). This demonstrates that overexpresion of orf28c leads to an increase in spiramycin production and confirms its role as a regulator of the biosynthetic pathway for spiramycins.<br>
The partial sequence oiorf28c was subsequently completed and the missing region of approximately 450 base pairs was determined (cf. SEQ ID, No. 140 and SEQ ID No. 141). The complete sequence of this ORF (SEQ ED No. 141) was used for the analysis with the various computer programs as explained above. It was thus possible to determine that the orf28c gene encodes a protein exhibiting 69% identity over the sequence determined (SEQ ID No. 142, which is the complete sequence of the Or£28c protein) (determined using the BLAST program), with the protein encoded by the acyBl gene which encodes a regulatory protein involved in carbomycin biosynthesis in Streptomyces thermotolerans (Arisawa, A., et a/., 1993; GenBank accession number: JC2032, BLAST score: 451). This similarity with a protein involved in regulating the biosynthesis of a relatively close antibiotic suggests that the orf28c gene encodes a regulatory protein involved in spiramycin biosynthesis. This hypothesis is supported by the fact that the protein encoded by the orf28c gene also exhibits strong similarity with the TylR protein, which is a regulatory protein involved in regulating tylosine oiosyninesis in streptomyces jraaiae (jsate, N. et al., lyyy; Uentfank accession number: AAF29380, BLAST score: 224). The results of overexpression of this gene (cf. example 24) confirm its role as a regulator of the spiramycin biosynthesis pathway.<br>
The orf28c gene was inactivated. It was thus possible to show that the resulting strain no longer produces spiramycins. This confirms that the orf28c gene is clearly involved in spiramycin biosynthesis and is essential to spiramycin biosynthesis. These<br><br>
results, combined with the results of overexpression of this gene (cf. example 24), are in agreement with Or£28c having a role as an activator essential to spiramycin biosynthesis.<br>
The or/29 gene encodes a protein exhibiting 31% identity (determined using the BLAST program) with a probable glycosyl hydrolase located in the gene group involved in the biosynthesis of soraphen A (an antifungal agent of the polyketide class) in Sorangium cellulosum (J. Ligon et a/., 2002; GenBank accession number: AAK19890, BLAST score: 139). This similarity with a protein involved in the biosynthetic pathway for a relatively close molecule suggests that the orf29 gene encodes a protein having glycosyl hydrolase activity. This hypothesis is supported by the fact that the protein encoded by the or/29 gene exhibits strong similarity with other proteins of similar function-in other organisms (cf. table 35). Analysis of the sequence of the protein encoded by or/29 using the CD-search program (cf. above) also suggests that the or/29 gene encodes a glycosyl hydrolase.<br><br>
* greater sequence similarity is associated with a higher BLAST score (Altschul et aL9 1990).<br>
Analysis of the protein sequence deduced from or/29, using the SignalP program (http://www.cbs.dtu.dk/services/SignalP/) (Nielsen, H., et ai, 1997), shows that this protein has a G-terminal signal sequence with a predicted cleavage site between positions 30 and 31 (QSA/QA). It may be predicted that this protein is extracellular. It might, as a glycosylhydrolase, have a role in the reactivation of spiramycin inactivated<br><br>
by glycosylation by the glycosyltransferases GimA and/or GimB (Gourmelen et aly 1998)!<br>
The orfiOc gene encodes a protein exhibiting 31% identity (determined using the BLAST program) with an nucleoside-diphosphate sugar epimerase in Corynebacterium glutamicum (GenBank accession number: NP_600590, BLAST score: 89). This similarity suggests that the orfSOc gene encodes an epimerase. This hypothesis is supported by the fact that analysis of the sequence using the CD-search program (cf. above) also suggests that the orfSOc gene encodes an epimerase.<br>
orfiQc exhibits two possible initiation codons (cf. SEQ ED No. 115) which give two possible proteins of 345 and 282 amino acids respectively (SEQ ID No.-116 and 117). However, the codon usage is typical of Streptomyces only from the second ATG; in addition, the deduced protein sequence of the sequence between the first ATG and the second does not align with the identified close sequences, whereas the shortest protein sequence (from the 2nd ATG: SEQ ID No. 144) aligns correctly with the beginning of these proteins. It can therefore be deduced from this that the second ATG is the correct initiation codon and that the sequence of this orf is therefore that presented in SEQ ID No. 143, which, once translated, corresponds to the protein of sequence SEQ ID No. 144.<br>
The orftl gene encodes a protein exhibiting 52% identity (determined using the BLAST program) with an oxidoreductase in Streptomyces coelicolor (GenBank accession number: NP_631148, BLAST score: 261), This similarity suggests that the orfil gene encodes a reductase. This hypothesis is supported by the fact that analysis of the sequence using the CD-search program (cf. above) also suggests that the orf31 gene encodes a reductase. This hypothesis is also supported by the fact that the protein encoded by the orfll gene exhibits strong similarity with other proteins of similar function in other organisms (cf. table 36).<br><br><br>
* greater sequence similarity is associated with a higher BLAST score (Altschul et a/., 1990).<br>
The orftl gene was inactivated. It was thus possible to show that the resulting strain no longer produces spiramycins. This confirms that the or/31 gene is indeed involved in spiramycin biosynthesis. The enzyme encoded by this gene is therefore clearly responsible for a byconversion step essential to spiramycin biosynthesis.<br>
The sequence oiorf32c was first of all determined partially (cf. example 19), since the coding sequence in 5' position was only determined in a second step. The partial sequence of this orf (SEQ ID No. 120) was, however, used for the analysis with the various computer programs as explained above. It was thus possible to determine that the orf32c gene encodes a protein exhibiting 47% identity, over the determined sequence (SEQ ID No. 121, which is the partial sequence of the Orf32c protein) (determined using the BLAST program), with a regulatory protein of the GntR family in Streptomyces coelicolor (GenBank accession number: NP_625576, BLAST score: 229). This similarity suggests that the orf32c gene encodes a transcriptional regulator of the GntR family. This hypothesis is supported by the fact that the protein encoded by the orf32c gene exhibits strong similarity with other proteins of similar function in other organisms.<br>
The partial sequence of orf32c was subsequently completed and the missing region was determined (cf. SEQ ID No. 140 and SEQ ED "No. 145). The complete sequence of this orf encodes a protein exhibiting 44% identity (determined using the BLAST<br><br>
program) with a regulatory protein of the GntR family in Streptomyces avermitilis (GenBank accession number: NP_824604, BLAST score: 282). This similarity suggests that the orfSlc gene encodes a transcriptional regulator of the GntR family. This hypothesis is supported by the fact that the protein encoded by the orf32c gene exhibits strong similarity with other proteins of similar function in other organisms (cf. table 37).<br>
i 	i	i	i	1<br>
* greater sequence similarity is associated with a hieher BLAST score (Altschul et al.<br>
1990).<br>
The orf32c gene was inactivated with the aim of studying the function of this gene in the spiramycin biosynthesis pathway in Streptomyces ambofaciens. It was possible t9 , show that the resulting strain^produces spiramycins. This indicates that the orf32c gene is not essential to spiramycin biosynthesis and that it is not essential to survival of the bacterium.<br><br>
The orfi3 gene encodes a protein exhibiting 49% identity (determined using the BLAST program) with a hypothetical protein of Xanthomonas campestris (GenBank accession number: NP_635564, BLAST score: 54).<br>
The sequence of orf34c is partial. In fact, the comparisons carried out between the product of this orf and the databases suggest that the C-terminal portion of this protein is not in the product deduced from the nucleotide sequence and therefore that this orf is longer and continues beyond the region sequenced. The partial sequence of this ORF was, however, used for the analysis with the various computer programs as explained above. It was possible to determine that the orf34c gene encodes a protein exhibiting 91% identity, over the sequence determined (SEQ ID No. 150, which is the partial sequence of the Orf34c protein) (determined using the BLAST program), with an arabmofuranosidase from Streptomyces coelicolor (Bentley et al*, 2002; GenBank accession number: NP_630049, BLAST score: 654). In £ coelicolor, the gene encoding this arabmofuranosidase does not appear to be involved in secondary metabolite biosynthesis. In 5. ambofaciens, this gene is therefore probably not involved in spiramycin biosynthesis.<br>
A subject of the present invention is also polynucleotides which hybridize, under high stringency hybridization conditions, to at least one of the polynucleotides of sequence SEQ ID Nos 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 25, 28, 30, 34, 36, 40, 43, 45,47,49, 53,60,62,64,66,68, 70, 72,74,76,78, 80, 82, 84,107,109,111,113, 115, 118, 120, 141, 143, 145, 147 and 149, or one of its variants, or one of the sequences derived therefrom due to the degeneracy of the genetic code. Preferentially, these said polynucleotides are isolated from a bacterium of the genus Streptomyces, more preferentially these polynucleotides encode proteins involved in the biosynthesis of a macrolide, and even more preferentially these polynucleotides encode a protein having activity similar to the protein encoded by the polynucleotides with which they hybridize. The high stringency hybridization conditions can be defined as hybridization conditions which are not favorable to the hybridization of nonhomologous nucleic acid strands. High stringency hybridization conditions may, for example, be described as<br><br>
hybridization conditions in the buffer described by Church &amp; Gilbert (Church &amp; Gilbert, 1984) at a temperature of between 55°C and 65°C; the hybridization temperature is preferably 55°C, the hybridization temperature is even more preferably 60°C, and the hybridization temperature is most preferably 65°C, followed by one or more washings carried out in 2X SSC buffer at a temperature of between 55°C and 65°C; this temperature is preferably 55°C, this temperature is even more preferably 60°C and this temperature is most preferably 65°C, followed by one or more washes in 0.5X SSC buffer at a temperature of between 55°C and 65°C; this temperature is preferably 55°C, this temperature is even more preferably 60°C and this temperature is most preferably 65°C The hybridization conditions described above can be adjusted as a function of the length of the nucleic acid the hybridization of which is sought, or of the type of labeling chosen, according to techniques known to those skilled in the art. Suitable hybridization conditions can, for example, be adjusted according to the work by F. Ausubel et a/., 2002.<br>
The invention also relates to a polynucleotide having at least 70%, more preferably 80%, more preferably 85%, even more preferably 90%, even more preferably 95%, and most preferably 98%, nucleotide identity with a polynucleotide comprising at least 10, 12, 15, 18, 20 to 25, 30, 40, 50, 60, 70, 80, 90, 100, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900, 950, 1000, 1050, 1100, 1150, 1200, 1250, 1300, 1350, 1400, 1450, 1500, 1550, 1600, 1650,1700, 1750, 1800, 1850 or 1900 consecutive nucleotides of a polynucleotide chosen from the group consisting of the nucleotide sequences SEQ ID Nos 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 25, 28, 30, 34, 36, 40, 43, 45, 47, 49, 53, 60? 62, 64, 66, 68, 70, 72, 74, 76, 78, 80, 82, 84, 107, 109,111, 113, 115, 118,120,141, 143, 145, 147 and 149, or one of its variants, or one of the sequences derived therefrom due to the degeneracy of the genetic code, or a polynucleotide of complementary sequence. Preferentially, these said polynucleotides are isolated from a bactrium of -the genus Streptomyces, more preferentially these polynucleotides encode proteins involved in the biosynthesis of a macrolide, and even more preferentially these polynucleotides encode proteins having activity similar to the proteins encoded by the polynucleotides with which they exhibit the identity. Most<br><br>
preferably, a polynucleotide according to the invention is chosen from the group consisting of the nucleotide sequences SEQ ID Nos 3, 5, 7,9,11,13, 15,17,19, 21, 23,<br>
25,	28, 30, 34, 36, 40, 43, 45, 47, 49, 53, 60, 62, 64, 66, 68, 70, 72, 74, 76, 78, 80, 82,<br>
84, 107, 109, 111, 113, 115, 118, 120, 141, 143, 145, 147 and 149, or a polynucleotide<br>
of complementary sequence.<br>
Thev optimal alignment of the sequences for the comparison can be carried out on a computer using known algorithms, for example those of the FASTA package (W.R. Pearson &amp; D.J. Lipman, 1988) and (W.R. Pearson, 1990), accessible in particular from the INFOBIOGEN resource center, Evry, France. By way of illustration, the percentage sequence identity may be determined using the LFASTA (K.-M. Chao et a/., 1992) or LALIGN (X. Huang and W. Miller, 1991) software. The LFASTA and LALIGN programs are part of the FASTA package. LALIGN provides optimal local alignments; this program is more rigorous, but also slower than LFASTA.<br>
Another aspect of the invention concerns a polypeptide resulting from the expression of a nucleic acid sequence as defined above. Preferentially, the polypeptides according to the invention exhibit at least 70%, more preferably 80%, more preferably 85%, even more preferably 90%, even more preferably 95% and most preferably 98%, amino acid identity with a polypeptide comprising at least 10, 15, 20, 30 to 40, 50, 60, 70, 80, 90, 100, 120, 140, 160, 180, 200, 220, 240, 260, 280, 300, 320, 340, 360, 380, 400, 420, 440, 460, 480, 500, 520, 540, 560, 580, 600, 620 or 640 consecutive amino acids of a polypeptide chosen from SEQ ID Nos 4, 6, 8,10,12,14, 16, 18, 20, 22, 24,<br>
26,	27, 29, 31, 32, 33, 35, 37, 38, 39, 41, 42, 44, 46, 48, 50, 51, 52, 54, 55, 56, 57, 58,<br>
59, 61, 63, 65, 67, 69,71, 73, 75, 77, 79, 81, 83, 85,108, 110,112, 114, 116, 117, 119,<br>
121, 142, 144, 146, 148 and 150, or one of these sequences except that, all along said<br>
sequence, one or more amino acids have been substituted, inserted or deleted without<br>
affecting the functional properties thereof, or one of the variants of these sequences.<br>
Preferentially, the polypeptides according to the invention are expressed in a natural<br>
state by a bacterium of the genus Streptomyces, more preferentially these polypeptides<br>
are involved in the biosynthesis of a macrolide, and even more preferentially these<br><br>
polypeptides have activity similar to that of the polypeptide with which they share the identity. Preferably, a polypeptide according to the invention is chosen from the group consisting of the polypeptide sequences SEQ ID Nos 4, 6, 8, 10, 12, 14, 16, 18, 20, 22, 24, 26, 27, 29, 31, 32, 33, 35, 37, 3S, 39, 41, 42, 44, 46, 48, 50, 51, 52, 54, 55, 56, 57, 58, 59, 61, 63, 65, 67, 69, 71, 73, 75, 77, 79, 81, 83, 85, 108, 110, 112, 114, 116, 117, 119, 121, 142, 144, 146, 148 and 150, or one of these sequences except that, all along said sequence, one or more amino acids have bee~ substituted, inserted or deleted without affecting the functional properties thereo; T one of the variants of these sequences. Most preferably, a polypeptide according : the invention is chosen from the group consisting of the polypeptide sequences SEQ ID Nos 4, 6, 8, 10, 12, 14, 16, 18, 20, 22, 24, 26, 27, 29, 31, 32, 33, 35, 37, 38, 39, 41, 42, 44, 46, 48, 50, 51, 52, 54, 55, 56, 57, 58, 59, 61, 63,65,57,69, 71, 73, 75, 77,79, 81, 83, 85,108,110,112,114,116, 117,119, 121, 142, 144,146, 148 and 150.<br>
The optimal alignment of the sequences for the comparison can be carried out on a computer using known algorithms, for example those of the FASTA package (W.R. Pearson &amp; D J. Lipman, 1988) and (W.R. Pearson, 1990), accessible in particular from the INFOBIOGEN resource center, Evry, France. By way of illustration, the percentage sequence identity may be determined using the LFASTA (K.-M. Chao et ai, 1992) or LALIGN (X. Huang and W. Miller, 1991) software using the default parameters as defined by the INFOBIOGEN resource center, Evry, France. The LFASTA and LALIGN programs are part of the FASTA package. LALIGN provides optimal local alignments; this program is more rigorous, but also slower than LFASTA.<br>
Another aspect of the invention concerns a recombinant DNA comprising at least one polynucleotide as described above. Preferentially, this recombinant DNA is a vector. Even more preferentially, the vector is chosen from bacteriophages, plasmids, phagemids, integrative vectors, fosmids, cosmids, shuttle vectors, BACs (Bacterial Artificial Chromosomes) and PACs (PI-derived Artificial Chromosomes). By way of illustration, the lambda phage and the Ml3 phage may be mentioned as bacteriophages. As plasmids, mention may be made of plasmids which replicate in £. coli, for example<br><br>
pBR322 and its derivatives, pUC18 and its derivatives, pUC19 and its derivatives, pGB2 and its derivatives (G. Churchward et al,  1984), pACYC177 (GenBank accession number: X06402) and its derivatives, and pACYC184 (GenBank accession number; X06403) and its derivatives. Mention may also be made of plasmids which replicate in Streptomyces, such as, for example, pUlOl and its derivatives, pSG5 and its derivatives, SLP1 and its derivatives, and SCP2* and its derivatives (Kieser et al 2000). As phagemids, mention may be made, by way of illustration, of pBluescript II and its derivatives (marketed in particular by the company Stratagene (LaJolla, California, USA)), pGEM-T and its derivatives (marketed by the company Promega (Madison, Wisconsin, USA)), [lacuna] the IS117 integration system (Kieser et al, 2000). As fosmids, mention maybe made, by way of illustration, of the fosmid pFOSl (marketed by the company New England Bioloabs Inc., Beverly, Massachusetts, USA) and its derivatives. As cosmids; mention may be made, by way of illustration, of the cosmidSuperCos and its derivatives (marketed in particular by the company Stratagene (LaJolla, California, USA)) and the cosmid pWED15 (Wahl et al, 1987) and its derivatives. As shuttle vectors, mention may be made, by way of illustration, of £. colilStreptomyces shuttle plasmids, such as, for example, pU903 and its derivatives, the series of plasmids pUWL, pCAO106, pWHM3 and pOJ446 and their derivatives (Kieser et al 2000), and £. coWStreptomyces shuttle BACs, such as, for example, those described  in  patent  application  WO  01/40497.  As  BACs  (Bacterial  Artificial Chromosomes), mention may be made, by way of illustration, of the BAC pBeloBACl 1 (GenBank accession number: U5I113). As PACs (Plnierived Artificial Chromosomes), mention may be made, by way of illustration, of the vector pCYPAC6 (GenBank accession number AFl33437). Most preferably, a vector according to the invention is chosen from pOS49.1, pOS49.11, pOSC49.12, pOS49.14, pOS49.16, pOS49.28, pOS44.1, pOS44.2, pOS44.4, pSPM5, pSPM7, pOS49.67, pOS49.88, pOS49.106, pOS49.120,  pOS49.107, pOS49.32,  pOS49.43,  pOS49.44, pOS49.50,  pOS49.99, pSPM17, pSPM21, pSPM502, pSPM504, pSPM507, -pSPM508, pSPM509, pSPMl, pBXLllll, pBXL1112, pBXL1113, pSPM520, pSPM521, pSPM522,  pSPM523, pSPM524, pSPM525, pSPM527, pSPM528, pSPM34, pSPM35, pSPM36, pSPM37,<br><br>
pSPM38, pSPM39, pSPM40, pSPM41, pSPM42, pSPM43, pSPM44, pSPM45, pSPM47, pSPM48, pSPM50, pSPM51, pSPM52, pSPM53, pSPM55, pSPM56, pSPM58, pSPM72, pSPM73, pSPM515, pSPM519, pSPM74, pSPM75, pSPM79, pSPM83, pSPM107, pSPM543 and pSPM106.<br>
Another aspect of the invention concerns an expression system comprising a suitable expression vector and a host cell allowing the expression of one or more polypeptides as described above in a biological system. The expression vectors according to the invention comprise a nucleic acid sequence encoding one or more polypeptides as described above, and may be intended for the expression of the various polypeptides according to the invention in diverse host cells well known to those skilled in the art. By way of example, mention may be made of prokaryotic expression systems, such as the expression system in the bacterium E. coli, and eukaryotic expression systems, such as the baculovirus expression system which allows expression in insect cells, and the expression systems which allow expression in yeast cells, or the expression systems which allow expression in mammalian cells, in particular human   -cells. The expression vectors which can be used in such systems are well known to those skilled in the art; as regards prokayrotic cells, mention may be made, by way of illustration, of the expression vectors in £. coli, for example of the pET marketed by the company Stratagene (LaJolla, California, USA), the vectors of the GATEWAY family marketed by the company Invitrogen (Carlsbad, California, USA), the vectors of the pBAD family marketed by the company Invitrogen (Carlsbad, California, USA), the vectors of the pMAL family marketed by the company New England Bioloabs Inc. (Beverly, Massachusetts, USA),   and the rhamnose-inducible  expression vectors mentioned in the publication B. Wilms et aL, 2001 and their derivatives; mention may also be made of the expression vectors in Streptomyces, such as, for example, the vectors pU4123, pIJ6021s pPM927, pANT849, pANT 850, pANT 851, pANT1200, pANT1201 and pANT1202 and their derivatives (Kieser et a/., 2000). As regards yeast cells, mention may be made, by way of illustration, of the vector pESC marketed by the company Stratagene (LaJolla, California, USA). As regards the baculovirus expression system which allows expression in insect cells, mention may be made, by way of<br><br>
illustration, of the vector BacPAK6 marketed by the company BD Biosciences Clontech, (Palo Alto, California, USA). As regards mammalian cells, mention may be made, by way of illustration, of the vectors comprising the CMV (Cytomegalovirus) immediate early gene promoter (for example the vector pCMV and its derivatives marketed by the company Stratagene (LaJolla, California, USA)), or the SV40 early promoter of the vacuolating simian virus (for example the vector pSG5 marketed by the company Stratagene (LaJolla, California, USA).<br>
The invention also relates to a method of producing a polypeptide as described above, said method comprising the following steps:<br>
a)	inserting a nucleic acid encoding said polypeptide into a suitable vector; •<br>
b)	culturing, in a suitable culture medium, a host cell transformed or transfected beforehand with the vector step a);<br>
c)	recovering the conditioned culture medium or a cell extract, for example by sonication or by osmotic shock;<br><br>
d)	separating and purifying said polypeptide from said culture medium or else from the cell extract obtained in step c);<br>
e)	where appropriate, characterizing the recombinant polypeptide produced.<br>
A recombinant polypeptide according to the invention can be purified by passage over an appropriate series of chromatography columns, according to the methods known to those skilled in the art and described, for example, in F. Ausubel et ah% (2002). By way of illustration, mention may be made of the "Histidine-Tag" technique, which consists in adding a short polyhistidine sequence to the polypeptide to be produced, it being possible for this polypeptide to be purified on a nickel column. A polypeptide according to the invention may also be prepared by in vitro synthesis techniques. By way of illustration of such techniques, a polypeptide according to the invention may be<br><br>
prepared using the <srapid translation system marketed in particular by the company roche diagnostics france s.a. meylan france.></srapid>
Another aspect of the invention concerns host cells into which has been introduced at least one polynucleotide and/or at least one recombinant DNA and or at least one expression vector according to the invention.<br>
Another aspect of the invention concerns microorganisms blocked in a step of the biosynthetic pathway for at least one macrolide. The advantage lies, firstly, in studying the function of the mutated proteins and, secondly, in producing microorganisms which produce biosynthesis intermediates. These intermediates can be modified, optionally after separation, either by adding particular components to the production media, or by introducing into the microorganisms thus mutated other genes encoding proteins capable of modifying the intermediate by using it as a substrate. These intermediates can    thus    be    modified    chemically,    biochemically,    enzymatically    and/or microbiologically. The microorganisms blocked in a step of the biosynthetic pathway for macrolides can be obtained by inactivating the function of one or more proteins involved in the biosynthesis of this or these macrolide(s) in microorganisms which produce   this   or   these  macrolide(s).   Depending  on   the  protein(s)   inactivated, microorganisms blocked in the various steps of the biosynthetic pathway for this or these macrolide(s) may thus be obtained. The inactivation of this or these protein(s) can be carried out by any method known to those skilled in the art, for example by mutagenesis in the gene(s) encoding said protein(s) or by the expression of one or more antisense RNA(s) complementary to the messenger RNA(s) encoding said protein(s). The mutagenesis can, for example, be carried out by irradiation, by action of a mutagenic chemical agent, by solid-directed mutagenesis, by gene replacement, or any other method known to those skilled in the art. The conditions suitable for such a<br>
mutagenesis can, for. example, be adjusted according to the teaching contained in the<br>
»<br>
works by T. Kieser et aL, (2000) and Ausubel et ai9 (2002). The mutagenesis can be carried out in vitro or in situ^ by suppression, substitution, deletion and/or addition of one or more bases in the gene under consideration, or by gene inactivation. This<br><br>
mutagenesis can be carried out in a gene comprising a sequence as described above. Preferentially, the microorganisms blocked in a step of the biosynthetic pathway for macrolides are bacteria of the genus Streptomyces. More preferentially, the inactivation of the function of one or more proteins involved in the biosynthesis of the macrolide(s) in question is carried out by mutagenesis. Even more preferentially, the macroiide in question is spiramycin and the microorganisms in which the mutagenesis or mutageneses is (are) carried out are strains of S. ambofaciens. More preferentially, the mutagenesis is carried out in one or more genes comprising one of the sequences corresponding to one or more of the sequences SEQ ID Nos 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 25, 28, 30, 34, 36, 40, 43, 45, 47, 49, 53, 60, 62, 64, 66, 68, 70, 72, 74, 76, 78, 80, 82, 84, 107, 109, 111, 113, 115, 118, 120,^141, 143, 145, 147 and 149. Preferably, the mutagenesis or mutageneses is (are) carried out by gene inactivation. Most preferably, the mutagenesis consists of gene inactivation of a gene comprising a sequence corresponding to the sequence SEQ ID No. 13.<br>
By way of illustration, the following microorganisms may be mentioned as examples of such microorganisms: OS49.16 (orftw. fihyg, cf. example 2), OS49.67 (or/3 in-phase deletion, cf. example 6), OS49.107 (orfE:: &amp;hyg, cf. example 7), OS49.50 (orflO:: fihyg, cf. example 8), SPM21 (or/2::att3Daac-,cf. example 10), SPM22 (orf2::att3 in-phase deletion, cf. example 10), SPM501 (orf6*::attlf2hyg+, cf. example 14), SPM502 (prf6*:;attl in-phase deletion, cf. example 14), SPM507 (orfl2::att3fhac-y cf. example 11), SPM508 {orfl3c::att3naac-, cf. example 12), and SPM509 (orfl4::att3{2aac~, cf. example 13), SPM107 (orf28c::att3aac+, cf. example 29), SPM543 (or/31;:att3aac+, cf. example 30), SPM106 (orf32c::att3aacJr, cf. example 31).<br>
Another aspect of the invention concerns a method of preparing a macroiide<br>
biosynthesis  intermediate,   using  the  microorganisms  blocked   in   a  step   of the<br>
*<br>
biosynthetic*pathway for macrolides, as described above. The method consists in culturing, in a suitable culture medium, a microorganism blocked in a step of the biosynthetic pathway for macrolides, as described above, recovering the conditioned<br><br>
culture medium or a cell extract, for example by sonification or by osmotic shock, and separating and purifying said biosynthesis intermediate from said culture medium or else from the cell extract obtained in the preceding step. The conditions for culturing such microorganisms may be determined according to techniques well known to those skilled in the art. The culture medium may, for example, be the MPS medium or the SL11 medium for Streptomyces, and in particular for Streptomyces ambofaciens (Pernodet et ai9 1993). Those skilled in the art may in particular refer to the work by Kieser et al, (2000) as regards the culturing of Streptomyces. The intermediate produced can be recovered by any technique known to those skilled in the art. Those skilled in the art may, for example, refer to the techniques taught in US patent 3,000,785, and more particularly to the methods for-extracting spiramycins described in * that patent.<br>
Another subject of the invention concerns a method of preparing a molecule derived from a macrolide, using the microorganisms blocked in a step of the biosynthetic pathway for this macrolide, as described above. The method consists in obtaining a biosynthesis intermediate according to the method above and in modifying the intermediate thus produced, optionally after separation from the culture medium. The conditions for culturing such microorganisms may be determined according to techniques well known to those skilled in the art The culture medium may, for example, be the MP5 medium or the SL11 medium for Streptomyces, and in particular for Streptomyces ambofaciens (Pemodet et al, 1993). Those skilled in the art may in particular refer to the work by Kieser et a/., 2000 with regard to the culturing of Streptomyces. The intermediates produced may be modified, optionally after separation, either by adding suitable components to the production media, or by introducing into the   microorganisms   other   genes'  encoding  proteins   capable   o f modifying   the intermediate by using it as a substrate. These intermediates can thus be modified chemically, biochemically, enzymatically and/or microbiologically. More preferentially,. the macrolide in question  is  spiramycin and the microorganisms  in which  the mutagenesis or mutageneses is (are) carried out are strains of S. ambofaciens.<br><br>
The invention also relates to a microorganism which produces spiramycin I but which does not produce spiramycin II and EH. This microorganism comprises all of the genes necessary for the biosynthesis of spiramycin I but does not produce spiramycin II and HI since the gene comprising the sequence corresponding to SEQ ID No. 13. or one of its variants, or one of the sequences derived therefrom due to the degeneracy of the genetic code, and encoding a polypeptide of sequence SEQ ID No. 14, or one ot its variants, is not expressed or has been made inactive. The inactivation of this protein can be carried out by any method known to those skilled in the art, for example by mutagenesis in the gene encoding said protein or by the expression of antisense RNA complementary to the messenger RNA encoding said protein, it being understood that, if the expression of orf5* is affected by this manipulation, it will be necessary to perform another modification so that the orf5* gene is correctly expressed.  The mutagenesis may be carried out in the coding sequence or in a noncoding sequence so as to make the encoded protein inactive or to prevent its expression or its translation therefrom. The mutagenesis may be carried out by site-directed mutagenesis, by gene replacement or any other method known to those skilled in the art. The conditions suitable for such a mutagenesis may, for example, be adjusted according to the teaching contained in the works by T. Kieser et al, (2000) and Ausubel et aL, 2002. The mutagenesis may be carried out in vitro or in situ, by suppression, substitution, deletion and/or addition of one or more bases in the gene under consideration or by gene inactivation. The microorganism may also be obtained by expressing the genes of the biosynthetic pathway for spiramycin without them comprising the gene comprising the sequence SEQ ED No. 13, or one of its variants, or one of the sequences derived therefrom due to the degeneracy of the genetic code, and encoding a polypeptide of sequence SEQ ID No. 14, or one of its variants. Preferentially, the microorganism is a bacterium of the genus Streptomyces. More preferentially, the microorganism which produces spiramycin I but which does not produce spiramycin II and III is obtained from a starting microorganism which produces spiram veins I, II and 3U. Even more preferentially, the microorganism is obtained by mutagenesis in a gene comprising the sequence corresponding to SEQ ID No.  13, or one of its variants, or one of the<br><br>
sequences derived therefrom due to the degeneracy of the genetic code, and encoding a polypeptide of sequence SEQ ID No. 14, or one of its variants having the same function. Even more preferentially, this mutagenesis is carried out by gene inactivarion. Preferably, the microorganism is obtained from a strain o: S. ambofaciens which produces spiramycins I. II and HI. in which gene inacuvaticn of the gene comprising the sequence corresponding to SEQ ID No. 13, or one of the sequences derived therefrom due to the degeneracy of the genetic code, is carried out. Most preferably, the gene inactivation is carried out by in-phase deletion of the gene or of a part of the gene comprising the sequence corresponding to SEQ ID No. 13, or one of the sequences derived therefrom due to the degeneracv of the senetic code. Bv wav of illustration, the strain SPM502 (orf6*::attl, cf. example 14) may be mentioned as a microorganism which produces spiramycin I but which does not produce spiramycin II and HI.<br>
The invention also relates to a microorganism which produces spiramycin I but which does not produce spiramycin II and EEL as described above, which also overexpresses:<br>
-	a gene which can be obtained by polymerase chain reaction using the pair of primers of the following sequence: 51 AAGCTTGTGTGCCCGGTGTACCTGGGGAGC 3' (SEQ ID No. 138) and 5' GGATCCCGCGACGGACACGACCGCCGCGCA 3' (SEQ ID No. 139), and as matrix the cosmid pSPM36 or the total DNA of Streptomyces ambofaciens, more preferably it is the gene of coding sequence SEQ ID No. 141,<br>
-	or a gene derived therefrom due to the degeneracy of the genetic code.<br>
An example of sequence of such a gene is given in SEQ ID No. Ill (DNA); however, this sequence is partial since it does not comprise the 31 portion of the corresponding coding sequence. The translation of this portion of coding sequence into protein is given in SEQ ED No. 112. Those skilled in the art will easily be able to complete it in particular using the teaching given in example 24. The sequence undetermined in SEQ ID No. Ill was determined in a second step, and the complete sequence of this orf iorf28c) is given in SEQ ID No. 141. The translation into protein of this coding sequence is given in SEQ ID No. 142. Example 24 gives a method for cloning the orf28c<br><br>
gene and for producing an expression vector which allows the expression of orf28c. This example also shows that overexpression of the orj28c gene in the strain OSC2 leads to the improvement of spiramycin production in this strain. The overexpression of the orfZSc gene can be obtained by increasing the number of copies of this gene and or by introducing a promoter which is more active than the wild-type promoter. Preferably, the overexpression of said gene is obtained by introducing into the microorganism a recombinant DNA construct which allows overexpression of this gene. Preferably, this recombinant DNA construct increases the number of copies of said gene and makes it possible to obtain overexpression of said gene. In this recombinant DNA construct, the coding sequence of the gene can be placed under the control of a promoter which is more active than the wild-type promoter. By way of illustration, mention may be made of the ptrc promoter which is active in Streptomyces ambofaciens (E. Amann el aL, 1988) and also the ermE* promoter. Thus, preferably, the copy or copies of the orf28c gene introduced is (are) placed under the control of the erm£* promoter, as is the case in the construct pSPM75 (cf example 24).<br>
The invention also relates to a microorganism which produces spiramycin I but which does not produce spiramycin II and HI, as described above, which also overexpresses the gene having coding sequence SEQ ID No. 47 or having a coding sequence derived therefrom due to the degeneracy of the genetic code. Preferentially, this microorganism is the strain SPM502 pSPM525 deposited with the Collection Nationale de Cultures de Microorganismes [National Collection of Cultures and Microorganisms] (CNCM) Pasteur Institute, 25, rue du Docteur Roux 75724 Paris Cedex 15, France, on-February 26, 2003, under the registration number 1-2977.<br>
The invention also relates to a method of producing spiramycin I; the method consists in culturing, in a suitable culture medium, a microorganism which produces spiramycin I but which does not produce spiramycin II and HI, as described above, recovering the conditioned culture medium or a cell extract, and separating and purifying the spiramycin I from said culture medium or else from the cell extract obtained in the preceding step. The conditions for culturing such a microorganism may<br><br>
be determined according to techniques well known to those skilled in the an. The culture medium may, for example, be the MP5 medium or the SL11 medium for Streptomyces, and in particular for Streptomyces ambofaciens (Pemodet ei ai, 1993). These skilled in the art may in particular refer to the work by Kieser et al. 2000 regarding the culturing of Streptomyces. The spiramycin I produced can be recovered by any techniques known to those skilled in the art. These skilled in the art may, for example, refer to the techniques tausht in US oatent 3.0OO.7S5. and more oarticulariv to the methods for extracting spiramycins described in that patent.<br>
Another aspect of the  inversion concerns the  use of a nucleotide sequence according to the invention, for improving the macrolide production of a microorganism. Thus, the invention relates to a macrolide-producing mutant microorganism which has a genetic modification in at least one gene comprising a sequence as defined above, and/or which overexpresses at least one gene comprising a sequence as defined above. The genetic modification may consist of a suppression, a substitution, a deletion and/or an addition of one or more bases in the gene(s) under consideration with the aim of expressing one or more proteins having greater activity or of expressing a higher level of this or these protein(s). The overexpression of the gene under consideration can be obtained by increasing the number of copies of this gene and/or by introducing a promoter which is more active than the wild-type promoter. By way of illustration, mention may be made of the ptrc promoter which is active in Streptomyces ambofaciens (E. Amann et al, 1988) and also the ermE* promoter (Bibb et al, 1985), (Bibb et al, 1994). Thus, the overexpression of the gene under consideration can be obtained by introducing,   into   a   macrolide-producmg   microorganism   under   consideration,   a recombinant DNA construct according to the invention which allows overexpression of this gene. Specifically, certain steps of macrolide biosynthesis are limiting and, if one or more proteins which are more active or a higher expression level of the wild-type protein(s) involved in these limiting steps are expressed, it is possible to improve the production of the macrolide(s) concerned. An increase in the rate of production of tylosine has, for example, been obtained in Streptomyces fradiae by duplication of the ^ene encoding a limiting methvltransferase which converts macrocine to tvlosine<br>
O	3	J	-	-<br><br>
(R. Baltz, 1997). The production of expression of a protein which is more active can be obtained in particular by mutagenesis; those skilled in the art may, for example, refer in :his respect to the work by F. Ausubel et aL, (2002). Preferentially, these mutant microorganisms improved in terms of their macroiide production are bacteria of the genus Streptomycas. More preferentially, the macroiide in question is spiramycin and the microorganisms in which the mutagenesis or mutageneses is fare) carried out are strains of 5. ambofaciens. More preferentially, the genetic modification is carried out in one or more genes comprising one of the sequences corresponding to one or more oi the sequences SEQ ID Nos 3, 5, 7, 9, 11, 13, 15. 17, 19, 21, 23, 25, 28, 30, 34, 36, 40, 43, 45. 47, 49, 53, 60, 62, 64, 66, 68, 70, 72, 74, 76, 78, 80, 82, 84, 107, 109, 111,113,115, 118, 120, 141, 143, 145, 147 and 149, or one of its variants, or one of the sequences derived   therefrom   due  to   the  degeneracy  of the  genetic   code.   Preferably,   the microorganism overexpresses one or more genes comprising one of the sequences corresponding to one or more of the sequences SEQ ID Nos 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 25, 28, 30, 34, 36, 40, 43, 45, 47, 49, 53, 60, 62, 64, 66, 68, 70, 72, 74, 76, 78, 80,82,84, 107, 109, 111, 113, 115, 118, 120, 141, 143, 145, 147 and 149, or one of its variants, or one of the sequences derived therefrom due to the degeneracy of the genetic code. Preferably, the microorganism overexpresses the gene comprising a sequence corresponding to SEQ ID.No. 111 or 141, or one of its variants, or one of the sequences derived therefrom due to the degeneracy of the genetic code. The sequence given in SEQ ID No. 111 is partial; however, those skilled in the art will easily be able to complete it in particular using the teaching given in example 24. The sequence undetermined in SEQ ID No. 111 was determined in a second step, and the complete sequence of this orf (orflSc) is given in SEQ ED No. 141. The translation into protein of this coding sequence is given in SEQ ID No. 142. Example 24 gives a method for cloning the orf28c gene and for producing an expression vector which allows the expression oiorf28c. This example also shows that overexpression of the orf28c gene in the strain OSC2 leads to the improvement of spiramycin production in this, strain. The overexpression of the or[28c gene can be obtained by increasing the number of copies of this gene and/or by introducing a promoter which is more active than the wild-type promoter. Preferably, the<br><br>
overexpression of said gene is obtained by introducing into the microorganism a recombinant DNA construct which allows overexpression of this gene. Preferably, this recombinant DNA construct increases the number of copies of said gene and makes it possible to obtain overexpression cfsaid gene. In this recombinant DNA construct, the coding sequence of the gene can be placed under the control of a promoter which is more active than the wild-type promoter. By way of illustration, mention may be made of the ptrc promoter which is active in Streptomyces ambofaciens (E. Amann et al. 1983) and also the ermE* promoter. Thus, preferable the cov\' or comes of the orCSc gene introduced is (are) placed under the control of the ermE* promoter, as is the case in the construct pSPM75 (cf. example 24).<br>
Another aspect of the invention concerns a method of producing macrolides, using the microorganisms described in the preceding paragraph. This method consists in culturing, in a suitable- culture medium, a microorganism defined in the preceding paragraph, recovering the conditioned culture medium or a cell extract, and separating and purifying the macrolide(s) produced from said culture medium or else from the cell extract   obtained   in   the   preceding   step.   The   conditions   for   culturing   such microorganisms may be determined according to techniques well known to those skilled in the art. The culture medium may, for example, be the MPS medium or the SL11 medium for Streptomyces, and in particular for Streptomyces ambofaciens (Pernodet et ai, 1993). Those skilled in the art may in particular refer to the work by Kieser et al, 2000 regarding the culturing of Streptomyces. The macrolide(s) produced can be recovered by any techniques known to those skilled in the art. Those skilled in the art may, for example, refer to the techniques taught in US patent 3,000,785, and more particularly to  the  methods   for  extracting spiramycins  described in that  patent. Preferentially, the microorganisms used in such a method are bacteria of the genus Streptomyces, More preferentially, the macrolide in question is spiramycin and the mutant microorganisms improved in terms of their spiramycin production are strains of S. ambofaciens.<br><br>
Another aspect of the invention concerns the use of a sequence and/or of a vector according  to   the   invention,   for  preparing   hybrid   antibiotics.   Specifically,   the polynucleotides according to the invention may be used to obtain microorganisms expressing one or more mutant proteins giving rise to a modification in the substrate specificity, or else may be expressed in many antibiotic-producing microorganisms with the aim of generating hybrid antibiotics. Thus, the polynucleotides according to the invention may make it possible, by gene transfer between producer microorganisms, to produce hybrid antibiotics having advantageous pharmacological properties (Hopwood et al, 1985a, Hopwood et al, 1985b, Hutchinson et al, 1989). The principle by which genetic engineering can bring about the production of hybrid antibiotics was first of all proposed by Hopwood (Hopwood 1981). It was thus proposed that the enzymes involved in the biosynthesis of antibiotics often accept substrates which are structurally related but which differ from their natural substrate. It is generally accepted (Hopwood 1981, Hutchinson 1988, Robinson 1988) that the enzymes encoded by the genes of the biosynthetic pathway for antibiotics have a less strict substrate specificity than the enzymes of primary metabolism. It has thus been possible to show that a large number of non-natural substrates are converted by antibiotic-producing microorganisms, their mutants  or purified  enzymes  of the  biosynthetic pathway for  these  antibiotics (Hutchinson  1988). Using this teaching,  those skilled in the  art may construct microorganisms expressing one or more mutant proteins giving rise to a modification in the substrate specificity with the aim of generating hybrid antibiotics.<br>
The invention also relates to the use of at least one polynucleotide and/or at least one recombinantDNA_ and/or at. least one expression vector and/or at least one polypeptide and/or at least one host cell according to the invention, for carrying out one or more bioconversions. Thus, the invention makes it possible to construct bacterial or fungal strains in which one or more proteins according to the invention are expressed under the control of suitable expression signals. Such strains can then be used to,carry out one or more bioconversions. These bioconversions may be carried out either using whole cells, or using acellular extracts of said cells. These bioconversions may make it possible to convert a molecule into a derived form, with an enzyme of a biosynthetic<br><br>
pathway. For example, Carreras et al describe the use of a strain of Saccharopofyspora erythraea and of Streptomyces coelicolor for producing new erythromycin derivatives (Carreras et al, 2002). Walczak et al describe the use of a Streptomyces P450 monooxygenase for the byconversion of desacetyladriarnycin (an anthracycline analog) to novel anthracyclines (Walczak e: al, 2001). Olonao et al describe the use of a modified strain of Streptomyces lividans for the bioconversion of epsilon-rhodomycinone to rhodomycin D (Olonao et al, 1999). Those skilled in the art may apply this principle to any biosynthesis intermediate.<br>
The invention also relates to a recombinant DNA which comprises:<br>
-	a polynucleotide which can be obtained by polymerase chain reaction using the pair<br>
of primers of the following sequence:<br>
5' AAGCTTGTGTGCCCGGTGTACCTGGGGAGC 3' (SEQ ID No. 138) and 5' GGATCCCGCGACGGACACGACCGCCGCGCA 3' (SEQ ID No. 139) and as matrix the cosmid pSPM36 or the total DNA of Streptomyces ambofaciens, more preferably it is a polynucleotide of sequence SEQ ID No. 141, "<br>
-	or a fragment of at least 10, 12, 15, 18, 20 to 25, 30, 40, 50, 60, 70, 80, 90, 100,<br>
150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900, 950,<br>
1000, 1050, 1100, 1150, 1200, 1250, 1300, 1350, 1400, 1450, 1460, 1470, 1480,<br>
1490 or 1500 consecutive nucleotides of the polynucleotide.<br>
Preferentially, this recombinant DNA is a vector. Even more preferentially, the vector is chosen from bacteriophages, plasmids, phagemids, integrative vectors, fosmids, cosmids, shuttle vectors, BACs (Bacterial Artificial Chromosomes) and PACs (Pl-derived Artificial Chromosomes). By way of illustration, the lambda phage and the Ml3 phage may be mentioned as bacteriophages. As plasmids, mention may be made of plasmids which replicate in £*. coli, for example pBR322 and its derivatives, pUC18 and its derivatives, pUCl9 and its derivatives, pGB2 and its derivatives (£L Churchward et al, 1984), pACYC177 (GenBank accession number: X06402) and its derivatives, and pACYC184 (GenBank accession number: X06403) and its derivatives. Mention may also be made of plasmids which replicate<br><br>
in Streptomyces, such as, for example, pIJlOl and its derivatives, pSG5 and its derivatives, SLP1 and its derivatives, and SCP2* and its derivatives (Kieser et al 2000). As phagemids, mention may be made, by way of illustration, of pBluescript II and its derivatives (marketed in particular by the company Stratagene (LaJolla, California, USA)), pGEM-T and its derivatives (marketed by the company Promega (Madison, Wisconsin, USA)), and AZAPII and its derivatives (marketed in particular by the company Stratagene (LaJolla, California, USA)). As integrative vectors, mention may be made, by way of illustration, of vectors which integrate in Streptomyces, such as, for example, those derived from SLP1 (Kieser et al, 2000), those derived from pSAM2 (Kieser et al, 2000), vectors which use PhiC31 phage<br>
integration systems (Kieser et al, 2000) (for example pSET152 (Bierman et al,<br>
		*»<br>
1992)) or VWB integration systems (L. van Mellaert et al 1998), and also vectors<br>
which use the IS117 integration system (Kieser et al, 2000). As fosmids, mention<br>
maybe made, by way of illustration, of the fosmid pFOSl (marketed by the company<br>
New England Bioloabs Inc., Beverly, Massachusetts, USA) and its derivatives. As<br>
cosmids, mention may be made, by way of illustration, of the cosmid SuperCos and<br>
its  derivatives  (marketed  in  particular  by the  company Stratagene  (LaJolla,<br>
California, USA)) and the cosmid pWED15 (Wahl et al, 1987) and its derivatives.<br>
As   shuttle   vectors,   mention   may   be   made,   by   way   of   illustration,   of<br>
E. coli/Streptomyces  shuttle  plasmids,   such  as,  for  example,  pU903   and  its<br>
derivatives, the series of plasmids pUWL, pCAO106, pWHM3 and pOJ446 and their<br>
derivatives (Kieser et al 2000), and £. coli/Streptomyces shuttle BACs, such as, for<br>
example, those described in patent application WO 01/40497. As BACs (Bacterial<br>
Artificial Chromosomes), mention may be made, by way of illustration, of the BAC<br>
pBeloBACl 1   (GenBank   accession   number:   U51113).   As   PACs   (Pl-derived<br>
Artificial Chromosomes), mention may be made, by way of illustration, of the vector<br>
pCYPAC6  (GenBank accession number:  AF133437).  More preferentially,  this<br>
recombinant DNA is an expression vector. The expression vectors which can be used<br>
in such systems are well known to those skilled in the art; as regards prokayrotic<br>
cells, mention may be made, by way of illustration, of the expression vectors in<br><br>
re appropriate, characterizing the recombinant polypeptide produced.<br><br>
Th<br>
appropri those sic of illustr in addin possible also be technique<br>
(RTS)'Meylan; I<br><br>
^mbinant polypeptide thus produced can be purified by passage over an *ries of chromatography columns, according to the methods known to n the art and described, for example, in F. Ausubel et al.f (2002). By way mention may be made of the "Histidine-Tag" technique, which consists iort polyhistidine sequence to the polypeptide to be produced, it being lis polypeptide to be purified on a nickel column. This polypeptide may .red by in vitro synthesis techniques. By way of illustration of such ;e polypeptide may be prepared using the "rapid translation system :eted in particular by the company Roche Diagnostics France S.A.,<br><br><br>
- a gene • primers o i 5'AAGC 5' GGAT matrix th* preferably<br>
Ar      *r aspect of the invention concerns a microorganism which produces at least one       ;mycin, which overexpresses:<br>
h can be obtained by polymerase chain reaction (PCR) using the pair of following sequence:<br>
3TGTGCCCGGTGTACCTGGGGAGC 3' (SEQ ED No. 138) and CGCGACGGACACGACCGCCGCGCA 3' (SEQ ID No. 139), and as osmid pSPM36 or the total DNA of Streptomyces ambofaciens, more s the gene of coding sequence SEQ ID No. 141, - or a gene    rived therefrom due to the degeneracy of the genetic code. An examp:   of sequence of such a gene is given in SEQ ID No. Ill (DNA); however, this sequer. * * is partial since it does not comprise the 3' portion of the corresponding coding seq\  nee. The translation of this portion of coding sequence into protein is given in SEQ ID No. 112. Those skilled in the art will easily be able to complete it in particular U:ing the teaching given in example 24. This example thus gives a method for cloning the orf28c gene and for producing an expression vector which allows the expression oiorf28c. This example also shows that overexpression of the orf28c gene m the strain OSC2 leads to the improvement of spiramycin production in this strain. Preferably, the microorganism overexpressing<br><br>
-	a gene which can be obtained by polymerase chain reaction (PCR) using the pair of<br>
primers of the following sequence:<br>
5' AAGCTTGTGTGCCCGGTGTACCTGGGGAGC 3' (SEQ ID No. 138) and<br>
5" GGATCCCGCGACGGACACGACCGCCGCGCA 3" (SEQ ID No. 139), and as<br>
matrix the cosmid pSPM36 or the total DNA of Streptomyces ambofaciens, more<br>
preferably it is the gene of coding sequence SEQ 3D No. 141,<br>
-	or a gene derived therefrom due to the degeneracy of the genetic code<br>
is a bacterium of the genus Streptomyces; even more preferably, it is a bacterium of the species Streptomyces ambofaciens. Preferably, the overexpression of said gene is obtained by transformation of said microorganism with an expression vector; most preferably, the strain of microorganism is the strain OSC2/pSPM75(l) or of the strain OSC2/pSPM75(2) deposited with the Collection Nationale de Cultures de Microorganismes (CNCM) [National Collection of Cultures and Microorganisms] Pasteur Institute, 25, rue du Docteur Roux 75724 Paris Cedex 15, France, on October 6, 2003, under the registration number 1-3101.<br>
Another aspect of the invention concerns a method of producing spiramycin(s), using the microorganisms described in the preceding paragraph. This method consists in culturing, in a suitable culture medium, a microorganism defined in the preceding paragraph, recovering the conditioned culture medium or a cell extract, and separating and purifying the spiramycin(s) produced from said culture medium or else from the cell extract obtained in the preceding step. The conditions for culturing  such microorganisms may be determined according to techniques well known to those skilled in the art. The culture medium may, for example, be the MPS medium or the SL11 medium for Streptomyces and in particular for Streptomyces ambofaciens (Pemodet et ai, 1993). Those skilled in the art may in particular refer to the work by Kieser et al, 2000 regarding the culturing of Streptomyces. The spiramycin(s) produced can be recovered using any techniques known to those skilled in the art. Those skilled in the art may, for example, refer to the techniques taught in US patent 3,000,785, and more particularly to  the methods  for  extracting spiramycins described in  that patent.<br><br>
been given above. Preferentially, the expression vector in question is the plasmia<br>
pSPM75.<br>
LIST OF FIGURES<br>
Figure 1: Chemical structure of spiramycins I, II and IE.<br>
Figure 2: Cosmids used for sequencing the region.<br>
Figure 3: Organization of a group of genes involved in the biosynthetic pathway for<br>
spiramycins.<br>
Figure 4: Proposed biosynthetic pathway for mycarose.<br>
Figure 5: Proposed biosynthetic pathway for mycaminose.<br>
Figure 6: Proposed biosynthetic pathway for forosamine.<br>
Figure 7: Preferential order of the addition of sugars to the spiramycin molecule and<br>
intermediates.<br>
Figure 8: Proposed biosynthetic pathway for methoxymalonyl in S. ambofaciens. This<br>
pathway is  proposed  by  analogy with  the  biosynthesis  of methoxymalonyl   in<br>
Streptomyces hygroscopicus var. ascomyceticus (K. Wu et aL, 2000).<br>
Figure 9: Steps leading to the inactivation of a gene;<br>
A)	Cloning of the target gene into a vector which replicates E. coli but not in Streptomyces;<br>
B)	Insertion of the resistance cassette into the target gene (by cloning or recombination between short identical sequences);<br>
C)	Introduction of the plasmid into Streptomyces ambofaciens (by transformation or conjugation with E. coli) and selection of the clones having integrated the cassette then screening of clones having lost the vector portion so as to have a gene replacement;<br>
D)	Region of the chromosome of the mutant strain in which the target gene is inactivated by gene replacement.<br>
Figure 10: Optional steps following the inactivation of a gene according to the method described in figure 9, which may be carried out if an excisable cassette has been used to interrupt the target gene;<br><br>
E)	Introduction into the mutant strain of the plasmid pOSV508 carrying the xis<br>
and int genes of pSAM2, the products of which will allow effective excision by<br>
site-specific recombination between the attL and attR sequences bordering the<br>
cassette;<br>
F)	Production of clones which have lost the excisable cassette and which are<br>
sensitive to the antibiotic against which the cassette provided resistance;<br>
G)	After growth and sporulation on solid medium without antibiotic, the<br>
plasmid pOSV508 is lost at high frequency. It is thus possible to obtain clones<br>
sensitive to thiostrepton in which the target gene contains an in-phase deletion.<br>
The sequence of the deleted target gene can be controlled by PCR amplification<br>
and sequencing of the PCR product.<br>
Figure 11: Amplification of the excisable cassette with the aim of using it for a homologous recombination experiment. The technique of homologous recombination via short homologous sequences has been described by Chaveroche et a/., 2000. The 39 or 40 deoxynucleotides located at the 5' end of these oligonucleotides comprise a sequence corresponding to a sequence of the gene to be inactivated, and the 20 deoxynucleotides located in the most 3' position correspond to the sequence of one of the ends of the excisable cassette.<br>
Figure 12: Production of a construct for the inactivation of a target gene using the technique described by Chaveroche et aL9 2000.<br>
Figure 13: Map of the plasmid pWHM3. Strepto ori: Streptomyces origin of replication.<br>
Figure 14: Map of the plasmid pOSV508.<br>
Figure 15: Example of the structure of an excisable cassette. This consists of the Qhyg cassette (Blondelet-Rouault et ai, 1997) bordered by the attR and attL sites (Raynal et<br>
ai, 1998), between which a recombination event will allow the excision of the cassette,<br>
*	»   -<br>
by virtue of the expression of the xis and int genes. Figure 16: Map of the plasmid pBXLl 111. Figure 17: Map of the plasmid pBXLl 112.<br><br>
Figure 18: Microbiological test for spiramycin production, based on the sensitivity of a strain of Micrococcus luteus to spiramycin. The strain of Micrococcus luteus used is a strain naturally sensitive to spiramycin but resistant to congocidine. The various strains of Streptomyces to be tested were cultured in 500 ml Erlenmeyer flasks containing 70 ml of MPS medium, inoculated at an initial concentration of 2.5 x 106 spores/ml and grown at 27°C with orbital shaking at 250 rpm. Samples of fermentation musts were taken after 48, 72 and 96 hours of culturing, and centrifuged. A ten-fold dilution of these supernatants in sterile culture medium is used for the test. The Micrococcus luteus indicator strain resistant to congocidine but sensitive to spiramycin (Gourmelen et ai, 1998) was cultured in a square 12 x 12 cm dish. Disks of Whatman AA paper were soaked with 70   1 of the ten-fold dilution of each supernatant and placed on the surface of the dish. Disks soaked with a solution of spiramycin of various concentrations (2-4-8   g/ml in MP5 culture medium) were diluted as a standard range. The dishes were incubated at 37°C for 24 to 48 h. If the disk contains spiramycin, this diffuses into the agar and inhibits the growth of the Micrococcus luteus indicator strain. This inhibition creates a "halo" around the disk, this halo reflecting the area where the Micrococcus luteus strain has not grown. The presence of this halo is therefore an indication of the presence of spiramycin and makes it possible to determine whether the strain of S. ambofaciens in question is or is not a spiramycin producer. Comparison with the inhibition diameters obtained for the standard range makes it possible to obtain an indication of the amount of spiramycin produced by this strain.<br>
Figure 19: HPLC chromatogram of the filtered culture medium supernatant of the strain OSC2.<br>
Figure 20: HPLC chromatogram of the filtered culture medium supernatant of the strain<br>
SPM501.<br>
Figure 21: HPLC chromatogram of the filtered culture medium supernatant of the strain<br>
SPM502.<br>
Figure 22: HPLC chromatogram of the filtered culture medium supernatant of the strain<br>
SPM507.<br><br>
Figure 23: HPLC chromatogram of the filtered culture medium supernatant of the strain<br>
SPM508.<br>
Figure 24: HPLC chromatogram of the filtered culture medium supernatant of the strain<br>
SPM509.<br>
Figure 25: .-Uignment of the Orf3 protein (SEQ ID No. 29) with the TylB protein (SEQ<br>
ID No. 87) of 5. fradiae produced using the FASTA program.<br>
Figure 26: Alignment of the MdmA protein of S. mycarofaciens (SEQ ID No. 88) with<br>
the SrmD protein (SEQ ED No. 16) produced using the FASTA program.<br>
Figure 27: Example of sequences of the residual sites after excision of the excisable<br>
cassette. Indicated in bold is the minimum att26 site as defined in Raynal et a/., 1998.<br>
The sequence of phase .1 (attl) of 33 nucleotides is given in SEQ ID No. 104, the<br>
sequence of phase 2 (att2) of 34 nucleotides is given in SEQ ID No. 105, and the<br>
sequence of phase 3 (att3) of 35 nucleotides is given in SEQ ED No. 95.Figure 28:<br>
Diagrammatic representation of the orflO gene, location of the PCR primers used and<br>
construct obtained with each pair of primers.<br>
Figure 29: Construction of the cassette pac-oritT.<br>
Figure 30: Map of the cosmid pWED2.<br>
Figure 31:  Diagrammatic representation of the group of genes involved in the<br>
biosynthetic pathway for spiramycins and of the location of the three probes used to<br>
isolate the cosmids of the genomic DNA library of the Streptomyces ambofaciens strain<br>
OSC2 in E. coli (cf. example 19).<br>
Figure 32: Location of the inserts of the cosmids isolated from the genomic DNA<br>
library of the Streptomyces ambofaciens strain OCS2 in E. coli (cf. example 19).New<br>
cosmid DNA library. Figure 33: Subcloning of the Pstl-Pstl fragment of the cosmid<br>
pSPM36 (insert of the plasmid pSPM58), subcloning of the Stul-Stul fragment of the<br>
cosmid pSPM36 (insert of the plasmid pSPM72) and subcloning of an EcoRl-StuI<br>
(insert of the plasmid pSPM73).<br>
Figure 34: Location of the open reading frames identified in the Pstl-Pstl insert of the<br>
plasmid pSPM58 and in the EcoRl-StuI insert of the plasmid pSPM73.<br><br>
Figure 35: Superposition of the HPLC chromatograms of the filtered culture medium<br>
supernatant of the strain OS49.67 produced at 238 and 280 nm (top) and UV spectra of<br>
the molecules eluted at 33.4 minutes and 44.8 minutes (bottom).<br>
Figure 36: Molecular structure of the platenolide A and platenolide B molecules.<br>
Figure 37: Organization of the group of genes involved in the biosynthetic pathway for<br>
spiramycins.<br>
Figure 38: Molecular structure of a biosynthesis intermediate produced by the strain<br>
SPM507.<br>
Figure   39:   Structure   of a  bios&gt;nthesis  intermediate  produced  by   a  strain   of<br>
S. ambofaciens   of  genotype   orf6*::attlQhyg±   obtained   from   a   strain   which<br>
overproduces spiramycins. Insertion of the cassette attlQhyg^ into orf6% exerts a polar<br>
effect which prevents the expression of orf5 *.<br>
Figure 40: Molecular structure of the platenolide A + mycarose and platenolide B +<br>
mycarose molecules produced by the strain OS49.67<br>
Figure 41: Subcloning of a Pstl-PstI fragment of the cosmid pSPM36 (insert of the<br>
plasmid (pSPM79)), localization of the open reading frames identified in the Pstl-PslI<br>
insert of the plasmid pSPM79 and localization of the sequence SEQ ID N° 140.<br><br>
The present invention is illustrated using the following examples, which should be considered as nonlimiting illustrations.<br>
In summary, the genes of the biosynthetic pathway for spiramycins were isolated from a genomic DNA library of Streptomyces ambofaciens. This library was obtained by partial digestion of the genomic DNA of Streptomyces ambofaciens, with the BamYD. restriction enzyme. Large DNA fragments, of 35 to 45 kb on average, were cloned into the cosmid pWEDl (Gourmelen et a/., 1998) derived from the cosmid pWED15 (Wahl et al9 1987). These cosmids were introduced into E. coli using phage particles. The library thus obtained was hybridized with a probe (of sequence SEQ ID No. 86) corresponding to a portion of the tylB gene of S. fradiae (Merson-Davies &amp; Cundliffe, 1994, GenBank accession number: U08223). After hybridization, one cosmid out of the 4 which hybridized with the probe was more particularly selected. This cosmid, named pOS49.1, was then digested with Sad and a 3.3 kb fragment containing the region which hybridized with the probe was subcloned into the vector pUC19 and sequenced. Four open reading frames were identified and one of them encodes a protein (SEQ ID No. 29) exhibiting strong sequence similarity with the TylB protein of S.fradiae (SEQ ID No. 87) (cf. figure 25). This gene was named orf3 (SEQ ED No. 28) and was inactivated in S. ambofaciens. It was possible to demonstrate that the clones in which the orfl gene was inactivated no longer produced spiramycins. This shows the involvement of the orf3  gene,  or of genes  located downstream, in spiramycin biosynthesis.<br>
Once this confirmation had been obtained, a larger region of the cosmid pOS49.1 was sequenced, either side of the 'Sad fragment previously studied. Thus, it was possible to obtain, from the cosmid pOS49.1, the sequence of a region comprising seven whole open reading frames and two other incomplete open reading frames located on either side of these seven whole open reading frames. By virtue of a search in the databases, it was possible to show that one of the incomplete open reading frames corresponded to the srmG locus (a region encoding an enzyme called "polyketide synthase" (PKS)). The corresponding genes were subcloned by S. Burgett et al in 1996<br><br>
(US patent 5,945,320). Moreover, the other open reading frames, the seven whole ORFs named: orfl, orf2, orf3. orf4, orfS, orf6 and orf7 (SEQ ID No. 23, 25, 28, 30, 34, 36, 40) and the start of the eighth ORF named orfS (sequence SEQ ID No. 43) were not found in the databases.<br>
Subsequently, and with the aim df cloning other genes involved in spriramycin biosynthesis, other cosmids comprising fragments of the S. ambofaciens genome in this same region were isolated. For this, a further series of hybridizations on colonies was carried out using three probes. The first probe corresponds to a 3.7 kb DNA fragment containing orfl, orfl and the start of or/3, subcloned from pOS49.1. The second probe corresponds to a 2 kb DNA fragment containing a portion of or/7 and a portion of orfS, subcloned from pOS49.L A third probe was also used. This latter probe corresponds to a 1.8 kb DNA fragment containing the srmD gene. The srmD gene is a gene isolated from S. ambofaciens capable of imparting spiramycin resistance. Specifically, prior studies had enabled the cloning of several resistance determinants of S. ambofaciens, imparting spiramycin resistance to a strain of S. griseofuscus (spiramycin-sensitive strain) (Pernodet et a/., 1993) (Pernodet et a/., 1999). To isolate resistance genes, a cosmid library of the genomic DNA of the strain S. ambofaciens had been produced in the cosmid pKC505 (Richardson MA et al, 1987). This pool of cosmids had been introduced into 5. griseofuscus, naturally sensitive to spiramycin. Five cosmids capable of imparting apramycin resistance and spiramycin resistance in S. griseofuscus had thus been obtained. Among these 5 cosmids, a cosmid named pOS44.1 contains, in its insert, the srmD gene which encodes a protein exhibiting a certain similarity with the protein encoded  by  the  mdmA   gene  of Streptomyces  mycarofaciens   and   involved   in midecamycin resistance in this producer organism (Hara et al.,  1990, GenBank accession number: A60725) (figure 26). The third probe used to locate the spiramycin biosynthesis genes was an insert of approximately 1.8 kb comprising the srmD gene.<br>
These three probes were used to hybridize the genomic DNA library described above and made it possible to choose two cosmids (pSPM7 and pSPM5) liable to<br><br>
contain the longest inserts and having no common bands. The cosmid pSMP5 hybridized with the first and the second probe, but did not hybridize with the third probe, whereas the cosmid pSPM7 hybridized with the third probe only. These two cosmids were entirely sequenced using the "shotgut sequencing" technique. The sequences of the inserts of these two cosmids: pSPM7 and pSPM5, could be assembled since, although they did not overlap, each one of the inserts comprised a known sequence at one of its ends. Specifically, each one of these inserts comprised a fragment of the sequence of one of the genes encoding an enzyme called "polyketide synthase" (PKS). These 5 genes were cloned by S. Burgett et al in 1996 (US patent 5,945,320) (cf. figure 2), Thus, it was possible to determine a single S. ambofaciens genomic DNA sequence. A sequence of 30 943 nucleotides starting, in the 5 * position, from an £coRI site located in the first PKS gene and ranging up to a BamHl site in the 3' position is given in SEQ ID No. 1. This sequence corresponds to the region upstream of the PKS genes (cf. figure 2 and 3). A second region of 11 171 nucleotides, starting from a Pstl site in the 5' position and ranging up to a BstEH site in the 3* position, located in the fifth PKS gene, is given in SEQ ID No. 2. This region is the region downstream of the PKS genes (downstream and upstream being defined by the orientation of the 5 PKS genes or oriented in the same direction) (cf. figures 2 and 3).<br>
Next, and with the aim of cloning other genes involved in spiramycin biosynthesis, other cosmids comprising fragments of the S. ambofaciens genome in this same region were isolated (cf. example 18 and 19).<br><br>
EXAMPl<br>
ambofacu<br><br>
: Construction of a genomic DNA library of the Streptomyces rain ATCC23877 in E. coli<br><br><br><br>
1.1.   Extr ATCC23S<br>
T from the *■ the numbc al, 2000) standard te<br><br>
of the   genomic   DNA   of the  Strevtomvces   ambofaciens   strain<br>
s<br>
reptomyces ambofaciens strain ATCC23877 (accessible in particular can Type Culture Collection (ATCC) (Manassas, Virginia, USA), under ill) was cultured in YEME (Yeast Extract-Malt Extract) (Kieser, T, et ■he genomic DNA of this strain was extracted and purified according to ques of lysis and precipitation (Kieser T. et aly 2000).<br><br>
1.2. Const:      on of the genomic DNA library<br>
Tl    genomic DNA of the Streptomyces ambofaciens strain ATCC23877 as isolated ab    * was partially digested with the BamEl restriction enzyme so as to obtain DNA frag:   its having a size between approximately 35 and 45 kb. These fragments were clone   into the cosmid pWEDl (Gourmelen et al, 1998) digested beforehand with BamY   The cosmid pWEDl is derived from the cosmid pWE15 (Wahl, et al9 1987) by c .etion of the 4.1 kb Hpal-Hpal fragment containing the expression module which is ac .ive in mammals (Gourmelen et al, 1998). The ligation mixture was then encapsidate : in vitro in lambda phage particles using the "Packagene® Lambda DNA packaging s stem" marketed by the company Promega, according to the manufacturer's recommenc itions. The phage particles obtained were used to infect the SURE® strain of £._ coli marketed by the company Stratagene (LaJolla, California, USA). The clones were selected on LB medium + ampicillin (50 ng/ml) since the cosmid pWEDl imparts ampicillin resistance.<br><br>
EXAMPLE 2: Isolation and characterization of genes involved in spiramycin biosynthesis in Streptomyces ambofaciens<br>
2.1 Colony hybridization of the E. coli clones of the genomic library of Streptomyces ambofaciens ATCC23877<br>
Approximately 2 000 E. coli clones of the library obtained above were transferred onto a filter for colony hybridization. The probe used for the hybridization is an Nael-Nael DNA fragment (SEQ ID No. 86) comprising a portion of the tylB gene of Streptomyces fradiae. This fragment corresponds to nucleotides 2663 to 3702 of the DNA fragment described by L.A. Merson-Davies &amp; E. Cundliffe (L.A. Merson-Davies &amp; E. Cundliffe, 1994, GenBank accession number: U08223), in which the coding sequence of the tylB gene corresponds to nucleotides 2677 to 3843.<br>
The Nael-Nael DNA fragment carrying a portion of the tylB gene of Streptomyces fradiae (SEQ ID No. 86) was labeled with 32P using the random priming technique (Kit marketed by the company Roche) and used as a probe to hybridize the 2 000 clones of the library, transferred onto a filter. The membranes used are Hybond N nylon membranes marketed by the company Amersham (Amersham Biosciences, Orsay, France) and the hybridization was carried out at 55°C in the buffer described by Church &amp; Gilbert (Church &amp; Gilbert, 1984). A wash was carried out in 2X SSC at 55°C for a period of 15 minutes and two successive washes in 0.5X SSC were then carried out at 55°C, each for a period of 15 minutes. Under these hybridization and washing conditions, 4 clones out of the 2 000 hybridized exhibited a strong hybridization signal. " These 4 clones were cultured in LB medium + ampicillin (50 fig/ml) and the corresponding 4 cosmids were extracted by standard alkaline lysis (Sambrook et aL, 1989). It was then verified that the hybridization was indeed due to a DNA fragment present in the insert of these four cosmids. For this, the cosmids were digested independently with several enzymes (BamHL, Pstl and SacT). The digestion products were separated on agarose gel, transferred onto a nylon membrane and hybridized with the Nael-Nael DNA fragment comprising a portion of the tylB gene of Streptomyces<br><br>
fradiae (cf. above) under the same conditions as above. It was possible to validate the four cosmids and one of these cosmids was more particularly selected and named pOS49.1.<br>
2.2	Verification of the involvement of the identified region and sequencing of the insert<br>
of the cosmid pOS49.1<br>
Several fragments of the insert of the cosmid pOS49.1 were subcloned and their sequences were determined. The cosmid pOS49.1 was digested with the Sad enzyme and it was shown, by Southern blotting, under the conditions described above, that a<br>
3.3	kb fragment contains the region which hybridizes with the tylB probe. This 3.3 kb<br>
fragment was isolated by electroelution from a 0.8% "agarose gel, and then cloned into<br>
the vector pUC19 (GenBank accession number; M77789) and sequenced. The plasmid<br>
thus obtained was named pOS49.11. Four open reading frames exhibiting a codon use<br>
typical of Streptomyces could be identified in this fragment (two complete and two<br>
truncated open reading frames), using the FramePlot program (J. Ishikawa &amp; K. Hotta,<br>
1999). Sequence comparisons using the FASTA program (cf.. (W.R. Pearson &amp;<br>
DJLipman, 1988) and (W.R. Pearson, 1990), accessible in particular from the<br>
INFOBIOGEN resource center, Evry, France) made it possible to show that the protein<br>
deduced from one of these four open reading frames exhibits strong sequence<br>
similarities with the TylB protein (SEQ ID No. 87; GenBank accession number:<br>
U08223) of S. fradiae (cf. figure 25). This protein was named Or£3 (SEQ ID No. 29).<br>
With the aim of testing whether the corresponding gene (the orfi gene (SEQ ID<br>
Nor~28)) was "involved" in spiramycin biosynthesis iriS. a^	was<br>
interrupted with an Qhyg cassette (M-H. Blondelet-Rouault et aL, 1997, GenBank accession number: X99315). For this, the plasmid pOS49.11 was digested with the Xhol enzyme and the fragment containing the four open reading frames (two complete and two truncated, including orf3 in its entirety) was subcloned into the Xhol site of the" vector pBC SK+ marketed by the company Stratagene (LaJoUa, California, USA). The plasmid thus obtained was named pOS49.12. For the purpose of inactivating or/3, a<br><br>
PmK-BstER fragment inside orfi was replaced with the Ohyg cassette by blunt-ended cloning into the latter plasmid. For this, the plasmid pOS49.12 was digested with the Pmli and BstEU. enzymes, the unique site for which is in the coding sequence of the or/3 gene. The ends of the fragment corresponding to the vector were blunt-ended by treatment with the Klenow enzyme (DNA polymerase I large fragment). The Qhyg cassette was obtained by digestion, with the BarriHl enzyme, of the plasmid pHP45 Qhyg (Blondelet-Rouault et a/., 1997, GenBank accession number: X99315). The fragment corresponding to the Qhyg cassette was recovered on agarose gel and its ends were blunt-ended by treatment with the Klenow enzyme. The two blunt-ended fragments thus obtained (the Qhyg cassette and the plasmid pOS49.12) were ligated and the ligation product was used to transform £. coli bacteria. The plasmid thus obtained was named pOS49.14 and contains the or/3 gene interrupted with the Qhyg cassette.<br>
The insert of the plasmid pOS49.14, in the form of an Xhol-Xhol fragment the ends of which were made non-sticky by treatment with the Klenow enzyme, was cloned into the EcoRW site of the plasmid pOJ260 (the plasmid pOJ260 is a conjugative plasmid capable of replicating in E. coli but incapable of replicating in S. ambofaciens (M. Bierman et al9 1992). This plasmid imparts apramycin resistance in E. coli and Streptomyces). The plasmid obtained (insert of the plasmid pOS49.14 cloned into the plasmid pOJ260)  was  named  pOS49.16.   The  latter was  transferred  into   the S. ambofaciens strain ATCC23877 by conjugation using the conjugated £. coli strain SI7-1, as described by Mazodier et ah (Mazodier et ai91989). The E. coli strain SI7-1 is derived from the E. coli strain 294 (Simon et al, 1983) (Simon, et aL, 1986). It was possible to obtain transconjugant clones possessing the hygromycin resistance marker carried by the Qhyg cassette and having lost the apramycin resistance marker carried by the vector pOJ260. For this, after conjugation, the clones were selected for their hygromycin  resistance.   The   hygromycin-resistant   clones   were   then   subcultured respectively on "medium with hygromycin (antibiotic B) and on medium with apramycin (antibiotic A) (cf. figure 9). The clones resistant to hygromycin (HygR) and sensitive to apramycin (ApraS) are, in principle, those in which a double recombination event has occurred and which therefore f ossess the orf3 gene interrupted with the Qhyg cassette.<br><br>
Replacement of the wild-type copy of or/3 with the interrupted copy was verified by two successive hybridizations. Thus, the total DNA of the clones obtained was digested with various enzymes, separated on agarose gel, transferred onto a membrane and hybridized with a probe corresponding to the Qhyg cassette (cf. above) in order to verify the presence of the cassette in the genomic DNA of the clones obtained. A second hybridization was carried out using, as probe, the Xhol-Xhol of the plasmid pOS49.11 containing the four open reading frames (two complete and two truncated, including or/3 in its entirety). The verification of the genotype can also be carried out by any method known to those skilled in the art, and in particular by PCR using the appropriate oligonucleotides and sequencing of the PCR product. One of the or/3:: Ohyg clones thus obtained, and the genotype of which was verified, was chosen and was called OS49.16.<br>
The spiramycin production of the OS49.16 clone thus obtained was tested using the production test described below (cf. example 15). It was thus possible to demonstrate that this strain no longer produces spiramycin, confirming the involvement of orf3 and/or of the genes located downstream, such as, for example, orf49 in spiramycin biosynthesis.<br>
Once this confirmation had been obtained, a larger region of the cosmid pOS49.1 was sequenced, on either side of the Sad fragment previously studied. Thus, it was possible to obtain, from the cosmid pOS49.1, the sequence of a region comprising seven whole open reading frames and two other incomplete open reading frames, located on either side of these seven open reading frames. By virtue of a search in the databases, it was possible to show that one of the incomplete open reading frames corresponded to the srmG locus (a region encoding an enzyme called "polyketide synthase" (PKS)). The corresponding genes were cloned by S. Burgett et al in 1996 (US patent 5,945,320). Moreover, the other open reading frames: the seven whole ORFs named: orfl orp, orp, 6rf4, orp, orf6 and orp (SEQ ID Nos 23, 25, 28, 30, 34, 36 and 40) and the start of the eighth ORF named orp (the whole sequence of this orf is given in SEQ ID No. 43), were not found in the databases.<br><br>
EXAMPLE   3:   Isolation   and   characterization   of other   genes   involved   in spiramycin biosynthesis in Streptomyces ambofaciens<br>
Secondly, and with the aim of cloning other genes involved in spiramycin biosynthesis, other cosmids comprising fragments of the S. ambofaciens genome in this same region were isolated. For this, a further series of colony hybridizations was carrieds out using three probes:<br>
-	The first probe used corresponds to a 3.7 kb BamHl-Pstl DNA fragment containing a fragment of the PKS gene (the genes corresponding to PKS were cloned by S. Burgett et al in 1996 (US patent 5,945,320)), orfl, or/2 and the start of orf3, subcloned from pOS49.1 and ranging from a BamVLl site located 1 300 base pairs upstream of the EcoRL site defining position 1 of SEQ ID No. 1 up to the Pstl site located at 2472 (SEQ ID No. 1). This BamKI-Pstl fragment was subcloned, from pOS49.1, into the plasmid pBC SK+, which made it possible to obtain the plasmid pOS49.28.<br>
-	The second probe used corresponds to a Pstl-BamHl DNA fragment of approximately 2 kb containing a fragment of or/7 and of orfS, subcloned from pOS49.1 and ranging from a Pstl site located at position 6693 of SEQ ID No. 1 up to the BarniU site located at position 8714 of SEQ ID No. 1. This Pstl-BamHl fragment was subcloned, from pOS49.1 into the plasmid pBC SK+, which made it possible to obtain the plasmid pOS49.76.<br>
-	A third probe was also used. This corresDonds to a 1 R Ich /TrnRT-JT/'w^/TTT HNA<br>
fragment containing the srmD gene. The srmD gene is a gene isolated from<br>
S. ambofaciens capable of imparting spiramycin resistance. Specifically, prior studies<br>
had enabled the cloning of several resistance determinants of S. ambofaciens, imparting<br>
spiramycin resistance to a strain of S. griseofuscus (spiramycin-sensitive strain)<br>
(Pernodet et al, 1993) (Pernodet et al, 1999). To isolate resistant genes, a cosmid<br>
library of the genomic DNA of the S. ambofaciens strain ATCC23877 had been<br>
prepared in the cosmid pKC505 (M.A. Richardson et al, 1987). For this, the genomic<br><br>
DNA of the 5. ambofaciens strain ATCC23877 had been partially digested with Sau3Al so as to obtain fragments having a size of between approximately 30 and 40 kb. The genomic DNA thus digested (3 fxg) had been ligated with 1 tig of pKC505 digested beforehand with the BaniHl enzyme (Pernodet et al, 1999). The ligation mixture had then been encapsidated in vitro in phage particles. The phage particles obtained had been used to infect the £. coli strain HB101 (accessible in particular from the American Type Culture Collection (ATCC) (Manassas, Virginia, USA), under the number 33694). Approximately 20 000 apramycin-resistant £. coli clones had been pooled and the cosmids of these clones had been extracted. This pool of cosmids had been introduced by protoplast transformation into the S. griseofuscus strain DSM 10191 (K.L. Cox &amp; R.H. Baltz, 1984), naturally sensitive to spiramycin (R.N. Rao et ai, 1987, this strain is available in particular from the German Collection of Microorganisms and Cell Cultures  (Deutsche  Sammlung  von  Mikro-organismen  und  Zellkulturen  GmbH, DSMZ), (Braunschweig, Germany), under the number DSM 10191). The transformants had been selected on a medium containing apramycm. 1 300 of the clones growing on medium containing apramycin had been transferred onto medium containing 5 f-ig/ml of spiramycin. Several apramycin-resistant clones had also grown on medium containing spiramycin and the cosmids of these colonies had been extracted and used to transform £. coli and S. griseofuscus (Pemodet et al9 1999). Five cosmids capable of imparting apramycin resistance to E. coli and co-resistance to apramycin and spiramycin in S. griseofuscus have thus been obtained. Among these 5 cosmids, it was determined that a cosmid named pOS44.1 contains, in its insert, a gene (SEQ ID No. 15) which encodes a protein (SEQ ID No. 16) exhibiting a certain similarity with a protein 'encoded'by the mdmA gene of Strepiomyces mycarofaciens (SEQ ID No. 88); this gene was named srmD (cf. alignment shown in figure 26, carried out using the FASTA program (cf.. (W.R. Pearson &amp; D.J. Lipman,  1988) and (W.R. Pearson,   1990), accessible in particular from the INFOBIOGEN resource center, Evry, France)).<br>
To isolate the resistance determinant contained in the plasmid pOS44.1, the latter was partially digested with the SauSAl restriction enzyme so as to obtain fragments<br><br>
having a size of approximately 1.5 to 3 kb, and these fragments were ligated into the vector pU486 linearized with the BamHl enzyme (Ward et a/., 1986). A plasmid was selected for its ability to impart spiramycin resistance in the 5. griseofuscus strain DSM 10191 (R.N. Rao et al, 1987), naturally sensitive to spiramycin (cf. above). For this, the pool of plasmids corresponding to the Sau3Al fragment of pOS44.1 ligated into the vector pU486 (cf. above) was introduced by protoplast transformation into the strain DSM 10191 and the transformants were selected for their thiostrepton resistance (due to the tsr gene carried by pU486). The clones growing on medium containing thiostrepton were transferred onto medium containing spiramycin. Several thiostrepton-resistant clones also grew on medium containing spiramycin and the plasmids of these colonies were extracted. A plasmid imparting resistance and containing an approximately 1.8 kb insert was selected and named pOS44.2. This 1.8 kb insert can be excised easily by virtue of a HindUl site and an EcoRL site present in the vector on either side of the insert. This 1.8 kb HindUl-EcoRI insert was sequenced and the resistance gene that it contains was named srmD. This fragment containing the srmD gene could thus be easily subcloned into the Vector pUC19 (GenBank accession number: M77789) opened with EcoRl-HindUl, and the plasmid obtained was named pOS44.4. The 1.8 kb HindBlr £coRI insert, containing the srmD gene, of this plasmid was used as a probe to locate the spiramycin biosynthesis genes (cf. below).<br>
A sample of an Escherichia Coli DH5a strain containing the plasmid pOS44.4 was deposited with the Collection Nationale de Cultures de Microorganismes [National Collection of Cultures and Microorganisms] (CNCM) Pasteur Institute, 25, rue du Docteur Roux 75724 Paris Cedex 15, France, on July 10, 2002, under the registration number 1-2918.<br>
Approximately 2 000 clones of the library, obtained above (cf. example 1), were<br>
transferred onto a filter for colony hybridization according to conventional techniques<br>
*<br>
(Sambrookera/., 1989).<br><br>
The three probes described above were labeled with P using the random priming technique (Kit marketed by Roche) and used to hybridize the 2 000 clones of the library, transferred onto the filter. The hybridization was carried out at 65 °C in the buffer described by Church &amp; Gilbert (Church &amp; Gilbert, 1984). A wash was carried out in 2X SSC at 65°C for 15 minutes and two successive washes were then carried out in 0.5X SSC at 65°C, each for a period of 15 minutes. Under these hybridization and washing conditions, 16 clones out of the 2 000 hybridized exhibited a strong hybridization signal with at least one of the probes. However, no cosmid hybridized with the three probes. The 16 cosmids were extracted and digested with the BamHl restriction enzyme. Comparison of the restriction profiles of these various cosmids with one another led to two cosmids being chosen which were liable to contain the longest inserts of the region and which had no common bands. Thus, two cosmids, one named pSPM5 and the other pSPM7, were chosen. The cosmid pSPM5 hybridized with the probes orfl to orf4 and the probe orf8, but did not hybridize with the probe srmD. pSPM7 hybridized only with the probe srmD and not with the other two probes.<br>
These two cosmids were entirely sequenced using the "shotgun sequencing" technique. The sequences of the inserts of these two cosmids: pSPM7 and pSPM5, could be assembled since, although they did not overlap, each one of the inserts comprised a known sequence at one of its ends. Specifically, each one of these inserts comprised a fragment of the sequence of one of the genes encoding an enzyme called "polyketide synthase" (PKS). These 5 genes were cloned by S. Burgett et al in 1996 (US patent 5,945,320) (cf. figure 2). Thus, it was possible to determine a single S. ambofaciens genomic DNA sequence. A sequence of 30 943 nucleotides beginning, in the 5' position, from an EcoRl site located in the first PKS gene and ranging up to a BamiU. site in the 3' position is given in SEQ ID N° 1. This sequence corresponds to the region upstream of the PKS genes (cf. figure 2 and 3). A second region of 11 171 'nucleotides, starting from a Pstl site in the 5' position and ranging up to an Ncel site in the 3' position located in the fifth PKS gene, is given in SEQ ID No. 2. This second region is the region downstream of the PKS genes (downstream and upstream being<br><br>
defined by the orientation of the 5 PKS genes all oriented in the same direction) (cf. figure 2 and 3).<br>
EXAMPLE 4: Analysis of the nucleotide sequences, determination of the open reading frames and characterization of the genes involved in spiramycin biosynthesis<br>
The sequences obtained were analyzed using the FramePlot program (J. Ishikawa &amp; K. Hotta 1999). This made it possible to identify, among the open reading frames, the open reading frames exhibiting a codon use typical ofStreptomyces. This analysis made it possible to detennine that this region comprises 35 ORFs located on either side of five genes encoding the enzyme "polyketide synthase" (PKS). 10 and 25 ORFs were identified respectively downstream and upstream of these genes (downstream and upstream being defined by the orientation of the 5 PKS genes all oriented in the same direction) (cf. figure 3). Thus, the 25 open, reading frames of thisjype, occupying a region of approximately 31 kb (SEQ ID No. 1 and figure 3), were identified upstream of the 5 PKS genes and 10 occupying a region of approximately 11.1 kb (SEQ ID No. 2 and figure 3) were identified downstream of the PKS genes. The genes of the upstream region were named orfl, orfl, orfi, orf4, orp, orf6, orf7, or/8, orpc, orflO, orfllc, orfl 2, orf!3ct orf!4t orfl5c, orfl 6, orfl 7, orfl 8, orfl9t or/20, orfllc, orfllc, orf23ct orf24c and orf25c (SEQ ID Nos 23, 25, 28, 30, 34, 36, 40, 43, 45, 47, 49, 53, 60, 62, 64, 66, 68, 70, 72, 74, 76, 78, 80, 82 and 84). The genes of the downstream region were named orfl*c, orfl*c, orf3*c, orf4*c, prf5*, orf6*, orf7*c, or/8*, orft* and orflO* (SEQ ID Nos 3, 5, 7, 9, 11, 13, 15, 17, 19 and 21). The "c" added to the name of the gene means, for the ORF in question, that the coding sequence is in the reverse orientation (the coding strand is therefore the strand complementary to the sequence given in SEQ ID No. 1 or SEQ ID No. 2 for these genes) (cf. figure 3).<br>
The protein sequences deduced from these open reading frames were compared with those present in various databases using various programs: BLAST (Altschul et<br><br>
ai, 1990) (Altschul et a/., 1997), CD-search, COGs (Cluster of Orthologous Groups) (these three programs are accessible in particular from the National Center for Biotechnology Information (NCBI) (Bethesda, Maryland, USA)), FASTA ((W.R. Pearson &amp; D. J. Lipman, 1988) and (W.R. Pearson, 1990), BEAUTY (K.C. Worley et ai, 1995)), (these two programs are accessible in particular from the INFOBIOGEN resource center, Evry, France). These comparisons made it possible to formulate hypotheses regarding the function of the products of these genes and to identify those which may be involved in spiramycin biosynthesis.<br>
EXAMPLE 5: Gene inactivation: principle of the construction of a knocked-out strain of Streptomyces ambofaciens<br>
The methods used consist in carrying out a gene replacement. The target gene to be interrupted is replaced with a copy of this gene interrupted with a cassette imparting resistance to an antibiotic (for example apramycin or hygromycin), as illustrated in figure 9. The cassettes used are bordered on either side by translation termination codons in all the reading frames and by transcription terminators which are active in Streptomyces.<br>
The insertion of the cassette into the target gene may or may not be accompanied by a deletion in this target gene. The size of the regions flanking the cassette can range from a few hundred to several thousand base pairs.<br>
The constructs required for the inactivation of the gene with the cassette were obtained in E. coli, the reference organism for obtaining recombinant DNA constructs. The interrupted gene was obtained in a plasmid which replicates in E. coli but which cannot replicate in Streptomyces.<br>
The constructs were then subcloned into vectors so as to allow "the transformation and the inactivation of the desired gene in S. ambofaciens. For this, two plasmids were used:<br><br>
-	pOJ260 (M. Bierman et al, 1992) (cf. example 2) which imparts apramycin resistance in £. coli and Streptomyces and which was used when the target gene was interrupted with a cassette imparting hygromycin resistance.<br>
-	pOSK1205 (4726 bp). This plasmid derives from the plasmid pBK-CMV (marketed by the company Stratagene (LaJolla, California, USA)) in which an Avrll fragment containing the sequence encoding Neomycin/Kanamycin resistance has been replaced with a sequence encoding hygromycin resistance, while at the same time conserving the P SV40 promoter. For this, the plasmid pHP45-Qhyg (Blondelet-Rouault et aiy 1997) was digested with the Notl and Pflml enzymes and the fragment conferring hygromycin resistance was subcloned into, the i4vrll site of the vector pBK-CMV after all the ends had been blunt-ended by treatment with the Klenow enzyme. In pOSK1205, the cassette which confers hygromycin resistance is preceded by the pSV40 promoter. This plasmid confers hygromycin resistance in E. coli and Streptomyces and was used when the target gene was interrupted with a cassette conferring apramycin resistance.<br>
The cassettes were introduced into the target gene either by cloning using restriction sites present in the target gene, or by recombination between short identical sequences as described, for example, by Chaveroche et al (M.K. Chaveroche et al, 2000).<br>
The plasmid carrying the gene interrupted by the cassette can then be introduced into Streptomyces ambofaciens, for example by conjugation between E. coli and Streptomyces (P. Mazodier et al., 1989). This technique was used when the basic vector is the vector pOJ260. A second technique may be used: the technique of protoplast transformation after denaturation of the DNA by alkali treatment (T. Kieser et al., 2000), in order to increase the frequency of recombination as described, for example, by Oh &amp; Chater (Oh &amp; Chater, 1997). This technique was used when the basic vector is pOJ260 or pOSK1205 (cf. below). The transformants are then selected with the antibiotic corresponding to the cassette present in the target gene (cf. figure 9, antibiotic B). A mixture of clones, among which there has been integration by a single or by two<br><br>
recombination events, is thus selected- Next, the clones sensitive to the antibiotic for which the resistance gene is present in the vector (outside the recombination cassette) (cf. figure 9, antibiotic A) are sought. It is thus possible to select the clones for which there have, in principle, been two recombination events resulting in the replacement of the wild-type gene with the copy interrupted by the cassette. These steps are shown diagrammatically in figure 9.<br>
Several cassettes can be used to interrupt the target genes. The Qhyg cassette which confers hygromycin resistance (Blondelet-Rouault et al, 1997, GenBank accession number: X99315) can, for example, be used.<br>
EXAMPLE 6: Construction of a strain of Streptomyces ambofaciens with an in-phase knockout in the orf3 gene<br>
The orf 3 gene had been interrupted withthe Qhyg cassette (cf. example 2.2) and it was possible to demonstrate that an orftw Qhyg strain no longer produces spiramycin, confirming the involvement of one or more genes of the cloned region in spiramycin biosynthesis (cf. example 2.2). In view of their orientation, cotranscription of ORFs 1 to 7 is probable (cf. figure 3) and the phenotype observed (non-producer of spiramycins) may be due to the inactivation of one or more of the genes cotranscribed with orf3. To confirm the involvement of orf3 in spiramycin biosynthesis, a further inactivation of the orfS gene was carried out, the latter inactivation being carried out in phase. For this, a Drain fragment inside orf3 of 504 base pairs was deleted. A DNA fragment obtained from"pOS49.1, ranging from the EcoBl site located at position 1 (SEQ ID No. 1) up to the Sacl site located at position 5274 (SEQ ID No. 1), and which comprises a deletion between the two DraM sites at positions 2563 and 3067 (504 nucleotides removed), was cloned into the plasmid pOJ260 (M. Bierman et al, 1992). The plasmid thus . obtained was named pOS49.67.<br>
The insert of pOS49.67 therefore consists of a DNA fragment of S. ambofaciens containing the orfl gene, or/2 gene, or/3 gene with the in-phase deletion, orf4 gene and<br><br>
a portion of orp. The vector in which this insert was subcloned is pOJ260, the plasmid pOS49.67 therefore imparts apramycin resistance and was introduced into the strain OS49.16 by protoplast transformation (cf. example 2). Since the strain OS49.16 is resistant to hygromycin, hygR and apraR transformants were obtained. After passaging such clones twice on nonselective medium, the clones sensitive to apramycin and to hygromycin (apraS and hygS) were sought. In some of these clones, a recombination event between homologous sequences is in fact expected to lead to the replacement of the copy of or/3 interrupted with the Qhyg cassette (contained in the genome of the strain OS49.16) with the copy oiorfl with the in-phase deletion present on the vector. The clones resulting from this recombination are expected to be apraS and hygS after elimination of the vector sequences. The genotype of the strains thus obtained can be verified by hybridization or by PCR and sequencing of the PCR product (to verify that only one in-phase deleted copy of or/3 is present in the genome of the clones obtained). Clones having only the copy of or/3 with an in-phase deletion were thus obtained and their genotype was verified. A clone exhibiting the desired characteristics was more particularly selected and was named OS49.67.<br>
A sample of the strain OS49.67 was deposited with the Collection Nationale de Cultures de Microorganismes (CNCM) Pasteur Institute, 25, rue du Docteur Roux 75724 Paris Cedex 15, France, on July 10,2002, under the registration number 1-2916.<br>
EXAMPLE 7: Construction of a strain of Streptomyces ambofaciens with a knockout in the orfS gene<br>
To carry out the inactivation of the orfS gene, a construct in which the JOhyg cassette was introduced into the coding sequence of orfS was obtained. For this, the plasmid pOS49.88 was first of all constructed. The plasmid pOS49.88 is derived from the plasmid pUC19 (GenBank accession number: M77789) by insertion of a 3.7 kb fragment (Pstl-EcoRI fragment obtained from the cosmid pSPM5) containing the end of orf7, or/8 and the beginning of orf9, cloned into the Pstl-EcoRl sites of pUC19. The<br><br>
jQhyg cassette (in the form of a BamHl fragment blunt-ended by treatment with the Klenow enzyme) was cloned into the unique Sail site of pOS49.88 located in or/5, after blunt-ending of all the ends by treatment with the Klenow enzyme.<br>
Since the cloning was blunt-ended, two types of plasmid were obtained depending on the direction of insertion of the cassette: pOS49.106 in which the hyg and or/8 genes are in the same orientation, and pOS49.120 in which the hyg and or/8 genes are in opposite orientations. The insert of the plasmid pOS49.106 was then subcloned into the plasmid pOJ260, to give pOS49.107. For this, the plasmid pOS49.106 was digested with the AspllBl enzyme and the ends were blunt-ended by treatment with the Klenow enzyme; this digestion product was redigested with the Pstl enzyme and the fragment containing the 6rj8 gene into which the Hhyg cassette was inserted was cloned into the vector pOJ260 (cf. above). For this, the vector pOJ260 was digested with the EcoKV and Pstl enzymes and used for the ligation. This manipulation therefore makes, it possible to obtain an oriented ligation since each one of the two fragments is blunt-ended on one side and Pstl on the other. The plasmid obtained was named pOS49.107.<br>
The plasmid pOS49.107 was intoduced into the S. ambofaciens strain ATCC23877 by protoplast transformation (T. Kieser, et a/., 2000). After protoplast transformation, the clones are selected for their hygromycin resistance. The hygromycin-resistant clones are then subcultured respectively on medium with hygromycin (antibiotic B) and on medium with apramycin (antibiotic A) (cf. figure 9). The clones resistant to hygromycin (HygR) and sensitive to apramycin (ApraS) are in principle those in which a double crossing over event has occurred and which contain the or/8 gene interrupted with the Qhyg cassette. The replacement of the wild-type copy of or/8 with the copy interrupted by the Qhyg cassette was verified by Southern blotting. Thus, the total DNA of the clones  obtained was  digested  with  several  enzymes,  separated  on  agarose  gel, transferred onto a membrane and hybridized with a probe corresponding to the* Qhyg cassette in order to verify the presence of the cassette in the genomic DNA of the clones obtained. A second hybridization was carried out using as probe the Pstl-EcoRI insert<br><br>
containing the end of or/7, orf8 and the beginning of orft, approximately 3.7 kb in size, of the plasmid pOS49.88. The verification of the genotype can be carried out by any method known to those skilled in the art, and in particular by PCR using the appropriate oligonucleotides and sequencing of the PCR product.<br>
An^orf8::Qhyg clone was chosen and named OS49.107. A sample of the strain OS49.107 was deposited with the Collection Nationale de Cultures de Microorganismes (CNCM) Pasteur Institute, 25, rue du Docteur Roux 75724 Paris Cedex 15, France, on July 10, 2002, under the registration number 1-2917.<br>
EXAMPLE 8:  Construction  of a strain  of Streptomyces ambofaciens with  a knockout in the orflO gene<br>
A 1.5 kb DNA fragment inside the orflO gene was obtained by PCR using as matrix the genomic DNA of 5. ambofaciens and the following primers:<br>
SRMKl : 5' CTGCCAGTCCTCTCCCAGCAGTACG 3' (SEQ ID No. 89) SRMR2 : 5' TGAAGCTGGACGTCTCCTACGTCGG 3' (SEQ ID No. 90)<br>
This PCR-derived DNA fragment was cloned into the vector pCR2.1 marketed by the company Invitrogen (Carlsbad, California, USA). The plasmid thus obtained was named pOS49.32. The Dhyg cassette (in the form of a BamHl fragment, cf. above) was cloned into the unique BstEU. site inside the fragment of the orflO gene, after all the ends had been blunt-ended by treatment with the Klenow enzyme. Since the cloning was blunt-ended, two types of plasmid were obtained depending on the direction of insertion of the cassette: pOS49.43 in which the hyg and orflO genes are in the same orientation and pOS49.44 in which the hyg and orflO genes are in opposite orientations. The insert of the plasmid pOS49.43 was transferred (in the form of an Aspl\%\-Xbal fragment the ends of which were blunt-ended by treatment with the Klenow enzyme)<br><br>
into the EcoRV site of the plasmid pOJ260, which made it possible to obtain the plasmid pOS49.50. The plasmid pOS49.50 containing the fragment of the orflO gene interrupted with the Qhyg cassette was introduced into the Streptomyces ambofaciens strain ATCC23877. After transformition, the clones were selected for their hygromycin resistance. The hygromycin-resistant clones were then subcultured respectively on medium with hygromycin (antibiotic B) and on medium with apramycin (antibiotic' A) (cf. figure 9). The clones resistant to hygromycin (HygR) and sensitive to apramycin (ApraS) are in principle those in which a double crossing over event has occurred and which contain the orflO gene ir.rerruped with the Qhyg cassette. Clones which contained the hygromycin resistance marker carried by the cassette and which had lost the apramycin resistance marker carried by the vector pOJ260 were thus obtained. The event consisting of replacement of the wild-type copy of orflO with the orflOv.Qhyg interrupted copy was verified by Southern blotting. Thus, the total genomic DNA of the clones obtained was digested with _ several enzymes, separated on agarose gel, transferred onto a membrane and hybridized with a probe corresponding to the Qhyg cassette in order to verify the presence of the cassette in the genomic DNA of the clones obtained. A second hybridization was carried out using as probe the 1.5 kb PCR product in the orflO gene (cf. above).<br>
A clone exhibiting the expected characteristics {orflOv.Qhyg) was more particularly selected and named OS49.50. It was in fact possible to verify, by virtue of two hybridizations, that the Qhyg cassette was indeed present in the genome of this clone and that the expected digestion profile is indeed obtained in the case of replacement, subsequent to a double recombination event, of the wild-type gene with the copy interrupted with the Qhyg cassette, in the genome of this clone. The verification of the genotype can also be carried out by any method known to those skilled in the art, and in particular by PCR using the appropriate oligonucleotides and sequencing of the PCR product.<br><br>
EXAMPLE 9: Gene inactivation: principle of the construction of a strain of Streptomyces ambofaciens knocked out according to the "excisable cassette" technique (cf. figures 9 and 10)<br>
A second type of cassette can be used for the gene inactivation: cassettes termed "excisable cassettes". These .cassettes have the advantage of being able to be excised in Streptomyces by a site-specific recombination event after having been introduced into the genome of S. ambofaciens. The aim is to inactivate certain genes in strains of Streptomyces without leaving, in the final strain, selection markers or large DNA sequences not belonging to the strain. After excision, only a short sequence of approximately thirty or so base pairs-(called "scar" site) remains in the genome of the strain (cf. figure 10).<br>
The implementation of this system consists, firstly, in replacing the wild-type copy of the target gene (by virtue of two homologous recombination events, cf. figure 9) with a construct in which an excisable cassette has been inserted into this target gene. The insertion of this cassette is accompanied by a deletion in the target gene (cf. figure 9). Secondly, the excision of the excisable cassette from the genome of the strain is brought about. The excisable cassette functions by virtue of a site-specific recombination system and has the advantage of allowing Streptomyces mutants which do not ultimately carry a resistance gene to be obtained. Possible polar effects on the expression of the genes located downstream of the inactivated gene(s) are also avoided (cf. figure 10).<br>
The employment of excisable cassettes has been described and used in many organisms including mammalian cells and yeast cells and in £. coli (Bayley et al.9 1992; Brunelli and Pall, 1993; Camilli et a/., 1994; Dale and Ow, 1991; Russell et al9 1992; Lakso et al, 1992). These excisable cassettes all use the Cre site-specific recombinase which acts on lox sites. This recombination system originates from the PI bacteriophage.<br><br>
To construct a system of the "excisable cassette" type in Streptomyces, the site-specific recombination system for the mobile genetic element pSAM2 of Streptomyces ambofaciens (Boccard et al, 1989a and b) is exploited. The system set up consists, initially, in constructing a recombinant vector comprising the gene to be interrupted, into which an excisable cassette is inserted. The insertion into the target gene of the excisable cassette is accompanied by a deletion in the target gene. It can be carried out by cloning using restriction sites present in the target gene or by recombination between short identical sequences as described, for example, by Chaveroche et al (Chaveroche MK et al, 2000). The excisable cassette can be constructed using, for example, the Qhyg cassette (Blondelet Rouault et al, 1997). This cassette was bordered by attR and attL sequences which normally flank the integrated copy of pSAM2 (cf. figure 15). The attL and attR sequences contain all the sites required for the site-specific recombination allowing excision of pSAM2 or of any DNA fragment located between these two regions (Sezonov et al, 1997, Raynal et al, 1998). The construction of such a cassette is obviously not limited to the use of an Qhyg cassette, but other resistance cassettes may be used as a basis to. construct this cassette (for example the Qaac or Qvph cassette (Blondelet Rouault et al, 1997)).<br>
After having obtained this construct, the strain of Streptomyces is transformed with the recombinant plasmid. The transformants are then selected with the antibiotic corresponding to the cassette present in the target gene (cf. figure 9, antibiotic B, this involves, for example, selection with hygromycin if the excisable cassette derives from the Qhyg cassette). A mixture of clones among which there has been integration by a single or by two recombination events is thus selected. Subsequently, the clones sensitive to the antibiotic for which the resistance gene is present in the vector (outside the recombination cassette) (cf. figure 9, antibiotic A) are sought. It is thus possible to select the clones for which there have in principle been two recombination events resulting in the- replacement of the wild-type gene with the copy interrupted with the cassette. These steps are shown diagrammatically in figure 9; the genotype of the clones thus obtained is verified by Southern blotting and a strain exhibiting the desired<br><br>
characteristics (replacement of the wild-type gene with the copy interrupted with the excisable cassette) is selected.<br>
Next, the strain selected above is transformed with a plasmid which allows expression of the xis and int genes, which are both necessary for the site-specific recombination between the attR and attL sites. This recombination leads to the excisable cassette leaving the genome of the strain, by virtue of a recombination event (cf. figure 10) (Raynal et aL, 1998). It is advantageous to choose the vector carrying the xis and int genes from the vectors which are relatively stable in Streptomyces (for example derived from the Streptomyces vector pWHM3. (Vara et al.9 1989)), this makes it possible to obtain a strain which has lost the latter vector after a few sporulation cycles in the absence of selection pressure.<br>
To excise the cassette, it is possible, for example, to use the plasmid pOSV508 (cf. figure 14), which 4s introduced by protoplast transformation into the strain of S. ambofaciens containing a gene interrupted with the excisable cassette. The plasmid pOSV508 is derived from the plasmid pWHM3 (J. Vara et a/., 1989) (cf. figure 13) to which the xis and int genes of pSAM2 (F. Boccard et aL, 1989b) have been added, placed under the control of the ptrc promoter (E. Amann et aL, 1988). The xis and int genes placed under the control of the ptrc promoter were subcloned into the. plasmid pWHM3 from the plasmid pOSint3 (Raynal et aL, 1998) (cf. figure 14). Introduction of the plasmid pOSV508 carrying the xis and int genes of pSAM2, into the mutant strain, will allow effective excision, by site-specific recombination, of the excisable cassette between the attL and attR sites flanking this cassette (A. Raynal et aL, 1998) (figure 10). Among the transformants, selected for their thiostrepton resistance due to the tsr gene carried by pOSV508, those which have become sensitive to the antibiotic to which the presence of the cassette imparts resistance are chosen (cf. figure 10). The excision is<br>
effective and it was observed that more than 90% of the transformants are of this type.<br>
&gt;<br>
After one or more cycles of growth and sporulation on a solid medium lacking thiostrepton, clones which have lost the plasmid pOSV508 are obtained. These clones<br><br>
can be detected by their sensitivity- to thiostrepton. The sequence of the deleted target gene can be verified by PCR and sequencing of the PCR product.<br>
Finally, the resulting strain carries, in the inactivated gene (internal deletion for example), a "scar" att site corresponding to the minimum attB site (Raynal et al.? 1998), derived from the recombination between the attR and attL sites. This minimum attB site which remains is similar to that naturally present in strains of Streptomyces ambofaciens, Streptomyces pristineespiralis and Streptomyces lividans (Sezonov et aL9 1997).<br>
The gene whose inactivation is desired can be cotranscribed with other genes located downstream. To avoid the inactivation of one* of the genes having a polar effect on the expression of the genes downstream in the operon, it is important to obtain an in-phase deletion after excision of the cassette. The excisable cassette system as described above makes it possible to satisfy such a requirement. Those skilled in the art may, in fact, easily construct three different excisable cassettes, said- cassettes leaving, after excision, a sequence of 33, 34 or 35 nucleotides respectively, without-a stop codon whatever the reading frame. Knowing the sequence of the target gene and the size of the deletion associated with the insertion of the cassette, it is possible to choose between these three excisable cassettes so that the excision produces an in-phase deletion. Of the 33, 34 or 35 nucleotides added, 26 correspond to the minimum attB sequence (cf. figure 27).<br>
In the case of the present application, two excisable cassettes were used. These two cassettes are as follows: attlQhyg^ (SEQ ID No. 91) and att3Qaac- (SEQ ID No. 92); these cassettes leave respectively 33 and 35 nucleotides after excision. They comprise respectively the Qhyg cassette or the Qaac cassette, the -f and - signs corresponding to the orientation of the resistance cassette. These two cassettes were constructed and cloned into the £coRV site of the vector pBC SK+, the HindUl site of which was deleted beforehand. The plasmids obtained were named pattl£lhyg+ and patt3Haac-, respectively. The excisable cassettes can be easily taken out by digestion of the plasmid with EcoRV.<br><br>
EXAMPLE 10: Construction of a strain of Streptomyces ambofaciens with a knockout in the orfl gene<br>
The inactivation of the or/2 gene was carried out using the excisable cassette technique (cf. above). The starting strain used is the Streptomyces ambofaciens strain' OSC2 which derives from the strain ATCC23877. However, the strain OSC2 differs from the strain ATCC23877 in that it has lost the mobile genetic element pSAM2 (Boccard et al, 1989a and b). This mobile element can be lost spontaneously during protoplastization (action of the lysozyme to digest the bacterial wall and fragment the mycelium (Kieser et al, 2000)) and regeneration of the protoplasts of the strain ATCC23877. To select the clones which had lost the pSAM2 element, a screen was set up, based on repression of the pra gene by the KorSA transcription repressor (Sezonov et al, 1995) (G. Sezonov et al, 2000). For this, a DNA fragment containing the promoter of the pra gene placed upstream of the aph gene (imparting kanamycin resistance and lacking its  own promoter) was cloned into the  unstable  vector pWHM3Hyg, the latter deriving from the plasmid pWHM3 (Vara et a/.r1989) in which the tsr gene has been replaced with the hyg gene (imparting hygromycin resistance). The plasmid thus obtained was named pOSV510.  The strain ATCC23877 was transformed, after protoplastization, with the plasmid pOSV510. The Pra promoter is a promoter which is repressed by the KorSA repressor, the gene encoding the latter being located within the pSAM2  mobile  element (Sezonov  G.  et al,  2000).   After transformation with the plasmid pOSV510, the transformed bacteria are selected for their kanamycin resistance (due to the aph gene carried by pOSV510). The clones having lost the pSAM2 integrative element have lost the KorSA repressor and the Pra promoter is no longer repressed and allows expression of the aph kanamycin resistance gene. Selection with kanamycin after transformation with the plasmid pOSV510 therefore makes it possible to select the clones which have lost the pSAM2 integrative element (and therefore KorSA) and which contain the-plasmid pOSV510. Since the plasmid pOSV510 is unstable, after a few cycles of sporulation without antibiotic, isolated  clones  are  subcultured  on  medium  with  kanamycin,   on  medium  with hygromycin and on medium without antibiotic. The clones sensitive to kanamycin and<br><br>
to hygromycin have lost pOSV510. The loss of the pSAM2 element was verified by hybridization and PCR. A clone exhibiting the desired characteristics was selected and was named OSC2.<br>
A sample of the strain OSC2 was deposited with the Collection Nationale de Cultures de Microorganismes (CNCM) Pasteur Institute, 25, rue du Docteur Roux 75724 Paris Cedex 15, France, on July 10, 2002, under the registration number 1-2908.<br>
The inactivation of the orj2 gene was carried out using the excisable cassette technique (cf. above and figure 10). For this, a 4.5 kb insert, the sequence of which starts from the EcoKL site located at position 1 up to the BamHI site located at position 4521 (SEQ ID No. 1), was subcloned into the EcoRI and BamEI sites of the plasmid pUC19 (GenBank accession number: M77789) from the cosmid pSPM5. The plasmid thus obtained was named pOS49.99.<br>
This plasmid was introduced into the E. coli strain KS272, which already contained the plasmid pKOBEG (Chaveroche et al, 2000) (cf. figure 12).<br>
In parallel, the attiQaac- excisable cassette (SEQ ED No. 92, cf. above) was amplified by PCR using as matrix the plasmid pOSKl 102 (the plasmid pOSKl 102 is a plasmid derived from the vector pGP704Not (Chaveroche et aU 2000) (V.L. Miller &amp; J.J. Mekalanos, 1988) in which the att3Qaac- cassette has been cloned, as an EcoRV fragment into the unique EcoKV site of pGP704Not) and using the following primers:<br>
ORF2A<br>
5'   GCGGCGCGGCAGCCTCTGGGTGATCGAGTCCGGCGTGACCATCGCGCGCGCTTCGTTCGG-3'<br>
(SEQ  ID No.   93)<br>
ORF2B<br>
5 '   GCTCCGTGCGTCATGCAGGAAGGTGTCGTAGTCGCGGTAGATCTGCCTCTTCGTCCCGAA-3 '<br>
(SEQ   ID No.   94)<br>
The 40 deoxynucleotides  located at the 5'* end of these oligonucleotides % comprise a sequence corresponding to a sequence in the target gene (or/2 in the present case) and the 20 deoxynucleotides located in the most 3' position (shown in bold<br><br>
above) correspond to the sequence of one of the ends of the att3Qaac- excisable<br>
cassette (cf. figure 11).<br>
The PCR product thus obtained was used to transform the E. coli strain containing the plasmids pKOBEG and pOS49.99 as described (Chaveroche et a/., 2000) (cf. figure 12). Thus, the bacteria were transformed by electroporation and selected for their apramycin resistance. The plasmids of the clones obtained were extracted and digested with several restriction enzymes, for the purpose of verifying that the digestion profile obtained corresponds to the profile expected if there has been insertion of the cassette (att3Qaac-) into the target gene (or/2), i.e. if there has indeed been homologous recombination between the ends of the PCR product and the target gene (Chaveroche et ai, 2000). The verification of the construct can also be carried out by any method known to those skilled in the art, and in particular by PCR using the appropriate oligonucleotides and sequencing of the PCR product. A clone in which the plasmid has the expected profile was selected and the corresponding plasmid was named pSPM17. This plasmid derives from pOS49.99 in which orfl is interrupted with the apramycin cassette (cf figure 12).<br>
The insertion of the cassette is accompanied by a deletion in orf2, between the nucleotides 211 and 492 of the coding portion of orfl.<br>
The plasmid pSPM17 was digested with the EcdRI enzyme and the ends were then blunt-ended by treatment with the Klenow enzyme; this digestion product was then digested with the Xbal enzyme and the insert containing the deleted or/2 gene was cloned into the vector pOSK1205 (cf above). For this, the vector pOSK1205 was digested with the BamHl enzyme and the ends were then blunt-ended by treatment with the Klenow enzyme; this product was then digested with the Xbal enzyme and used for the ligation with the insert obtained from pSPM17 as above. This manipulation therefore makes it possible to obtain an oriented ligation since each one of the two fragments is blunt-ended on one side and Xbal on the other. The plasmid thus obtained was named pSPM21; it carries H hygromycin resistance gene (vector portion) anH an insert in which the deleted or/2 gene is replaced with the att3Qaac- cassette.<br><br>
The vector pSPM21 was introduced into the Streptomyces ambofaciens strain OSC2 (cf. above) by protoplast transformation (T. Kieser et aL. 2000). After transformation, the clones were selected for their apramycin resistance. The apramycin-resistant clones were then subcultured respectively on medium with apramycin (antibiotic B) and on medium with hygromycin (antibiotic A) (cf. figure 9). The clones resistant to apramycin (ApraR) and sensitive to hygromycin (HygS) are in principle those in which a double crossing over event has occurred and which contain the or/2 gene interrupted with the att3Qaac- cassette. These clones were selected and the replacement of the wild-type copy of orf2 with the copy interrupted with the cassette was verified. The presence of the att3Qaac- cassette was verified by colony PCR. A hybridization was also carried out. For this, the total DNA of the clones obtained was digested with several enzymes, * separated on agarose gel, transferred onto a membrane and hybridized with a 3 kb probe corresponding to an EcoKL-BamHI fragment of the insert of the plasmid pOS49.99 (cf. above). The verification of the genotype can_also be carried out by any method known to those skilled in the art, and in particular by PCR using the appropriate oligonucleotides and sequencing of the PCR product.<br>
A clone exhibiting the expected characteristics was selected and named SPM2J - It was possible to verify, using PCR and hybridization, that the att3Qaac- cassette was indeed present in the genome of this clone and that the digestion profile expected in the case of replacement, subsequent to a double recombination event, of the wild-type gene with the copy interrupted with the att3Qaac- cassette in the genome of this clone is indeed obtained. This clone therefore has the genotype: orf2::att3£2aac~.<br>
A sample of the strain SPM21 was deposited with the Collection Nationale de Cultures de Microorganismes (CNCM) Pasteur Institute, 25, rue du Docteur Roux 75724 Paris Cedex 15, France, on July 10, 2002, under the registration number 1-2914.<br>
The strain SPM21 was transformed with the vector pOSV508 by protoplast transformation in order to bring about excision of the cassette (cf. figure 14). The plasmid pOSV508  is  derived  from the  plasmid pWHM3  (J.  Vara et  a/.,1989)<br><br>
(cf. figure 13) in which the xis and int genes of pSAM2 (F. Boccard et al% 1989b) have been added, placed under the control of the ptrc promoter (E. Amann et ai, 1988) (cf. figure 14). The introduction, into the strain SPM21, of the plasmid pOSV508 carrying the xis and int genes of pSAM2 allows effective excision, by site-specific recombination, of the excisable cassette between the attL and attR sites flanking this cassette (A. Raynal et ai, 1998) (figure 10). Among the transformants selected for their thiostrepton resistance due to the tsr gene carried by pOSV508, those which have become sensitive to apramycin, the resistance gene for which is carried by the aitiQaac- cassette, are chosen; the excision in fact leads to the loss of this resistance gene (cf. figure 10). The plasmid pOSV508 is unstable and, after two successive passages on medium without antibiotic, isolated clones are subcultured on medium with<br>
4	-r<br>
thiostrepton and on medium without thiostrepton. The thiostrepton-sensitive clones have lost pOSV508. It was verified that the excision of this cassette indeed results in an in-phase deletion in the orf2 gene, by PCR and sequencing of the PCR product; the excision of the cassette in fact leaves a characteristic "scar" att3 sequence (which is similar to the attB site of origin after recombination between the attL and attR sites):<br>
5' ATCGCGCGCGCTTCGTTCGGGACGAAGAGGTAGAT 3' (SEQ ID No. 95).<br>
The strain thus obtained which has the desired genotype (or/2::att3) was named SPM22.<br>
A sample of the strain SPM22 was deposited with the Collection Nationale de Cultures de Microorganismes (CNCM) Pasteur Institute, 25, rue du Docteur Roux 75724 Paris Cedex 15, France, July 10, 2002, under the registration number 1-2915.<br><br>
EXAMPLE 11: Construction of a strain of Streptomyces ambofaciens with a knockout in the or/12 gene<br>
For the inactivation of orr'12, orfI3c and orfl4y the same starting plasmid (pSPM504) was used to introduce a cassette of the "excisable cassette" type, at various positions. This plasmid contains a 15.1 kb insert which corresponds to the region from orf7 to orfl7. To construct this plasmid, a 15.1 kb BglE fragment originating from digestion of the cosmid pSPM7 (cf. above) was cloned into the plasmid pMBL18 (Nakano et aL, 1995) digested with BamHL. Since BamHI and BgHl ends are compatible, the plasmid pSPM502 is obtained after ligation. The entire insert of pSPM502 was then subcloned (in the form of a HindWNhel fragment) into the plasmid pOSK1205 (digested with HindULNhel), which made it possible to obtain the plasmid pSPM504.<br>
This plasmid was introduced into the E. coli strain KS272 which already contained the plasmid pKOBEG (Chaveroche et al, 2000) (cf. figure 12).<br>
In parallel, the attiQaac- excisable cassette was amplified by PCR using as matrix the plasmid pOSK1102 (the plasmid pOSK1102 is a plasmid derived from the vector pGP704Not (Chaveroche et al, 2000) (V.L. Miller &amp; J.J. Mekalanos, 1988) in which the att3Qaac- cassette has been cloned, as an EcoRV fragment, into the unique EcoRV site of pGP704Not); the primers used are as follows:<br>
EDR8 : 5 ' CGGGATGATCGCTTGTCCGGCGGCCGGATGCCTAGCCTCATCGCGCGCGCTTCGTTCGG 3 ' (SEQ ID No. 96)<br>
EDR9 : 5 '   CCCGATCCAGAACGTCTGGTCGGTGATCAGGTCGCTGTTCATCTGCCTCTTCGTCCCGAA 3' (SEQ ID No. 97)<br>
The 40 (only 39 for EDR8) deoxynucleotides located at the 5' end of these oligonucleotides comprise a sequence corresponding to a sequence in the target gene<br><br>
{prfll in the present case) and the 20 deoxynucleotides located in the most 3* position (shown in bold above) correspond to the sequence of one of the ends of the att3Qaac-excisable cassette (cf. figure 11).<br>
The PCR product thus obtained was used to transform the E. coli strain KS272 containing  the plasmids pKOBEG, and pSPM504  (cf.  above),  as  described by Chaveroche et al (Chaveroche et al, 2000) (cf. figure 12 for the principle, the plasmid pOS49.99 should be replaced by the plasmid pSPM504 and the plasmid obtained is no longer pSPM17 but pSPM507). Thus, the bacteria were transformed by electroporation with this PCR product and the clones were selected for their apramycin resistance. The plasmids of the clones obtained were extracted and digested with several restriction enzymes for the purpose of verifying that the digestion profile obtained corresponds to the profile expected if there has been insertion of the cassette (att3Qaac-) into the target gene (prfll), i.e. if there has indeed been homologous recombination between the ends of the PCR product and the target gene (Chaveroche et al., 2000). The verification of the construct can also be carried out by any method known to those skilled in the art, and in particular by PCR using the appropriate oligonucleotides and sequencing of the PCR product. A clone in which the plasmid has the expected profile was selected and the corresponding plasmid was named pSPM507. This plasmid derives from pSPM504 in which orfll is interrupted with the att3Qaac- cassette (cf. figure 12). The insertion of the cassette is accompanied by a deletion in the orfl 2 gene, the interruption begins at the thirtieth codon of orfl2. The last 46 codons of orfl 2 remain after the cassette.<br>
The vector pSPM507 was introduced into the Streptomyces ambofaciens strain OSC2 (cf. above) by protoplast transformation (T. Kieser et al, 2000). After transformation, the clones were selected for their apramycin resistance. The apramycin-resistant clones were then subcultured respectively on medium with apramycin (antibiotic B) and on medium with hygromycin (antibiotic A) (cf figure 9). The clones resistant to apramycin (ApraR) and sensitive to hygromycin (HygS) *are in principle those in which a double crossing over event has occurred and which contain the orfl 2 gene interrupted with the att3Qaac- cassette. These clones were more panicularly selected and the replacement of the wild-type copy of orfl2 with the copy interrupted<br><br>
with the cassette was verified by hybridization. Thus, the total DNA of the clones obtained was digested with several enzymes, separated on agarose gel, transferred onto a membrane and hybridized with a probe corresponding to the att3Qaac- cassette in order to verify the presence of the cassette in the genomic DNA of the clones obtained. A second hybridization was carried out using as probe a DNA fragment obtained by PCR arid corresponding to a very large portion of the coding sequence of the orfl2 gene.<br>
The verification of the genotype can also be carried out by any method known to those skilled in the art, and in particular by PCR using the appropriate oligonucleotides and sequencing of the PCR product.<br>
A clone exhibiting the expected characteristics (prfl2::att3Qaac~) was more particularly selected and named SPM507. It was in fact possible to verify, by virtue of the two hybridizations, that the att3Qaac~ cassette was indeed present in the genome of this clone and that the digestion profile expected in the case of replacement, subsequent to a double recombination event, of the wild-type gene-with the copy interrupted with the att3Qaac- cassette in the genome of this clone is indeed obtained. This clone therefore has the genotype: orfl2::att3Qaac- and was named SPM507. Given the orientation of the genes (cf. figure 3), there is no need to excise the cassette to study the effect of the inactivation of or/72. Specifically, the fact that orfl3c is oriented in the opposite direction to orfl2 shows that these genes are not cotranscribed. The use of an excisable cassette makes it possible, on the other hand, to have the possibility of being rid of the selection marker at any moment, in particular by transformation of the plasmid pOSV508. A sample of the strain SPM507 was deposited with the Collection Nationale de Cultures de Microorganismes (CNCM) Pasteur Institute, 25, rue du Docteur Roux 75724 Paris Cedex 15, France, on July 10, 2002 under the registration number 1-2911.<br><br>
EXAMPLE 12: Construction of a strain of Streptomyces ambofaciens with a knockout in the orfl3c gene<br>
The att3Qaac- excisable cassette was amplified by PCR using as matrix the plasmid pOSKl 102 (cf. above), using the following primers:<br>
EDR3 : 5 '   ACCGGGGCGGTCCTCCCCTCCGGGGCG7CACGGCCGCGGAATCTGCCTCTTCGTCCCGAA 3' (SEQ ID No. 98)<br>
EDR4 : 5 '   CACGCAGCGAGCCGACGCACTGATGGACGACACGATGGCCATCGCGCGCGCTTCGTTCGG 3' (SEQ ID No. 99)<br>
The 40 deoxynucleotides located at the 5' end of these oligonucleotides comprise a sequence corresponding to a sequence in the target gene {orflSc in the present case) and the 20 deoxynucleotides located in the most 3' position (shown in- bold above) correspond to the sequence of one of the ends of the attZQaac- excisable cassette (cf. figure 11).<br>
The PCR product thus obtained was used to transform the E. coli strain KS272 containing the plasmids  pKOBEG  and pSPM504  (cf.  above),  as  described by Chaveroche et al (Chaveroche et ai, 2000) (cf. figure 12 for the principle, the plasmid pOS49.99 should be replaced by the plasmid pSPM504 and the plasmid obtained is no longer pSPM17 but pSPM508). Thus, the bacteria were transformed by electroporation with this PCR product and the clones were selected for their apramycin resistance. The plasmids of the clones obtained were extracted and digested with several restriction enzymes for the purpose of verifying that the digestion profile obtained corresponds to the profile expected if there has been insertion of the cassette (att3Qaac-) into the target gene (orfl3c), i.e. if there has indeed been homologous recombination between the ends of the PCR product and the target gene (Chaveroche et ai, 2000). The verification of the construct can also be carried out by any method known to those skilled in the art, and in particular by PCR using the appropriate oligonucleotides and sequencing of the PCR product. A clone in which the plasmid has the. expected profile was selected and the corresponding plasmid was named pSPM508. This plasmid derives from pSPM504<br><br>
in which orfl3c is interrupted with the apramycin cassette (cf. figure 12). The insertion of the cassette is accompanied by a deletion in the off 13c gene, the interruption begins at the sixth codon of orfl 3c. The last 3 codons of orfl 3c remain after the cassette.<br>
The vector pSPM508 was introduced into the Streptomyces ambofaciens strain OSC2  (cf.  above)  by protoplast transformation (T. Kieser et al, 2000).  After transformation, the clones were selected for their apramycin resistance. The apramycin-resistant clones were then subcultured respectively on medium with  apramycin (antibiotic B) and on medium with hygromycin (antibiotic A) (cf. figure 9). The clones resistant to apramycin (ApraR) and sensitive to hygromycin (HygS) are in principle those in which a double crossing over event has occurred and which contain the orfl 3c gene interrupted with att3Qaac- cassette. These clones were more particularly selected and the replacement of the wild-type copy of orfl3c with the copy interrupted with the cassette was verified by hybridization. Thus, the total DNA of the clones obtained was digested with several enzymes, separated on agarose gel, transferred onto a membrane and hybridized with a probe corresponding to the att3Qaac- cassette in order to verify the presence of the cassette in the genomic DNA of the clones obtained. A second hybridization was carried out using as probe a PCR product corresponding to a sequence extending by about a hundred base pairs upstream and downstream of the coding sequence of orfl3c. The verification of the genotype can also be carried out by any method known to those skilled in the art, and in particular by PCR using the appropriate oligonucleotides and sequencing of the PCT product.<br>
A clone exhibiting the expected characteristics (orfl3c::att3Qaac-) was more particularly selected arid named SPM508. It was in fact possible to verify, by virtue of the two hybridizations, that the att3Qaac- cassette was indeed present in the genome of this clone and that the digestion profile expected in the case of replacement, subsequent to a double recombination event, of the wild-type gene with the copy interrupted with the att3Qaac- cassette in the genome of this clone is indeed obtained. This clone therefore has the genotype: orfl3c::att3Qaac- and was named SPM508. Given the orientation of the genes (cf. figure 3), there is no need to excise the cassette to study the<br><br>
effect of the inactivation of orfl3c. The fact that orfl4 is oriented in the opposite direction to orfl3c shows that these genes are not cotranscribed. The use of an excisable cassette makes it possible, on the other hand, to add the possibility of being rid of the selection marker at any moment. A sample of the strain SPM508 was deposited with the Collection Nationale de Cultures de Microorganismes (CNCM) Pasteur Institute, 25, rue du Docteur Roux 75724 Paris Cedex 15, France, on July 10, 2002 under the registration number 1-2912.<br>
EXAMPLE 13 Construction of a strain of Streptomyces ambofaciens with a knockout in the orfl4 gene<br>
The att3Qaac- excisable cassette was amplified by PCR using as matrix the plasmid pOSKl 102 (cf. above), using the following primers:<br>
EDR5 : 5 ' GGGCGTGAAGCGGGCGAGTGTGGATGTCATGCGAGTACTCATCGCGCGCGCTTCGTTCGG 3' (SEQ ID No. 100)<br>
EDR6 : 5 ' CGGGAAACGGCGTCGCACTCCTCGGGGGCCGCGTCAGCCCATCTGCCTCTTCGTCCCGAA 3' (SEQ ID No. 101)<br>
The 40 deoxynucleotides located at the 5' end of these oligonucleotides comprise a sequence corresponding to a sequence in the target gene (orf!4 in the present case) and the .20 deoxynucleotides located in.the most 37 position (shown in. bold above) correspond to the sequence of one of the ends of the att3Qaac- excisable cassette (cf. figure 11).<br>
The PCR product thus obtained was used to transform the E. coli strain KS272 containing the plasmids pKOBEG and pSPM504 (cf. above), as described by Chaveroche et al (Chaveroche et ai, 2000) (cf. figure 12 for the principle, the plasmid<br><br>
coding sequence of orfl4. The verification of the genotype can also be carried out by any method known to those skilled in the art, and in particular by PCR using the appropriate oligonucleotides and sequencing of the PCT product.<br>
A clone exhibiting the expected characteristics (orf!4::att3Qaac-) was more particularly selected and named SPM509. It was in-fact possible to verify, by virtue of the two hybridizations, that the att3Qaac- cassette was indeed present in the genome of this clone and that the digestion profile expected in the case of replacement, subsequent to a double recombination event, of the wild-type gene with the copy interrupted with the att3Qaac- cassette in the genome of this clone is indeed obtained. This clone therefore has the genotype: orfl4::att3Daac- and was named SPM509. Given the orientation of the genes (cf. figure 3), there is no need to excise the cassette to study the effect of the inactivation of arfl4. The fact that orfl5c is oriented in the opposite direction to orfl4 shows that these genes are not cotranscribed. The use of an excisable cassette makes it possible, on the other hand, to add the possibility of being rid of the selection marker at any moment. A sample of the strain SPM509 was deposited with the Collection Nationale de Cultures de Microorganismes (CNCM) Pasteur Institute, 25, rue du Docteur Roux 75724 Paris Cedex 15, France, on July 10, 2002 under the registration number 1-2913.<br>
EXAMPLE 14: Construction of a strain of Streptomyces ambofaciens with a knockout in the orf6* gene<br>
The inactivation of the orf6* gene was carried out using the excisable cassette technique (cf. above and figure 10). For this, the cosmid pSPM7 was used as matrix to amplify a fragment of the orf6* gene using the following oligonucleotides:<br>
C9583: 5' CTGCAGGTGCTCCAGCGCGTCGATCT 3' (oligo ser.se) (SEQ ID No. 102)<br>
C9534:  5'  CTGCAGACGGAGGCGGACCTGCGGC? 3' . (oligo antisenSe)  (SEQ  ID No. 103)<br><br>
The 20 deoxynucledtides located at the 3' end of these oligonucleotides correspond to a sequence located in the coding portion of the orf6* gene (SEQ ED No. 13) and the 6 deoxynucleotides located in the most 5' position correspond to the sequence of a Pstl site which facilitates subsequent cloning. The amplified DNA fragment is approximately 1.11 kb in size. This PCR product was cloned into the vector pGEM-T Easy (marketed by the company Promega (Madison, Wisconsin, USA)), which made it possible to obtain the plasmid named pBXLl 111 (cf. figure 16).<br>
The attinhyg+ excisable cassette was then introduced into the coding sequence of the orf6* gene. For this, the plasmid pBXLllll was digested with the Smal and AspllSl restriction enzymes and the digestion product was treated with the Klenow enzyme. This manipulation makes it possible to produce an internal deletion of 120 bp in the coding sequence of the orf6* gene (cf. figure 15). In addition, on either side of the restriction sites, there remain respectively 511 bp and 485 bp of the sequence of orf6* which will allow the homologous recombination for the inactivation of the orf6* gene. The attlQhyg+ excisable cassette was prepared from the plasmid pattlQhyg+ (cf. above) by digestion of this plasmid with £coRV. The latter was then subcloned into the vector pBXLllll prepared beforehand as described above (Smal and Aspll&amp;l digestion and then treatment with the Klenow enzyme). The plasmid obtained was named pBXL1112 (cf. figure 17). In this construct, the orf6* gene comprises a 120 bp deletion and is interrupted with the attlGhyg+ cassette.<br>
The plasmid pBXL1112 was then digested with the Pstl enzyme (site bordering the cassette since present in the PCR oligonucleotides) and the 3.7 kb Pstl insert comprising a portion of orf6* interrupted with the attlQhyg+ cassette was then cloned into the Pstl site of the plasmid pOJ260 (cf. above). The plasmid thus obtained was named pBXLl 113.<br>
The vector pBXLl 113 was introduced into the Streptomyces ambofaciens strain OSC2  (cf.  above)  by protoplast  transformation  (T.  Kieser et aL,  2000).   After.<br><br>
transformation, the clones were selected for their hygromycin resistance. The hygromycin-resistant clones were then subcultured respectively on medium with hygromycin (antibiotic B) and on medium with apramycin (antibiotic A) (cf. figure 9). The clones resistant to hygromycin (HygR) and sensitive to apramycin (ApraS) are in principle those in which a double crossing over event has occurred and which contain the orf6* gene interrupted with the attlf2hygJr cassette. These clones were more particularly selected and the replacement of the wild-type copy of orf6* with the copy interrupted with the cassette was verified by the Southern blotting technique. Thus, the total DNA of the clones obtained was digested with several enzymes, separated on agarose gel, transferred onto a membrane and hybridized with a probe corresponding to the hyg gene (obtained by PCR) in order to verify the presence of the cassette in the genomic DNA of the clones obtained. A second hybridization was carried out using as probe the Pstl-Pstl insert containing the orf6* gene, approximately 1.1 kb in size, and obtained from the plasmid pBXLl 111 (cf. above and figure 16). The verification of the genotype can also be carried out by any method known to those skilled in the art, and in particular by PCR using the appropriate oligonucleotides and sequencing of the PCR product.<br>
A clone exhibiting the expected characteristics (orf6*::attl{2hyg+) was more particularly selected and named SPM501. It was in fact possible to verify, using the two hybridizations, that the attl!2hyg+ cassette was indeed present in the genome of this clone and that the digestion profile expected in the case of replacement, subsequent to a double recombination event, of the wild-type gene with the copy interrupted with the aalf2hyg+ cassette in the genome of this clone is indeed obtained. This clone therefore has the genotype: orf6*::attlf2hyg+ and was named SPM501. A sample of the strain SPM501 was deposited with the Collection Nationale de Cultures de Microorganismes (CNCM) Pasteur Institute, 25, rue du Docteur Roux 75724 Paris Cedex 15, France, on July 10, 2002, under the registration number 1-2909.<br><br>
The strain SPM501 was transformed with the vector pOSV508 by protoplast transformation in order to bring about excision of the cassette (cf. figure 14). The plasmid pOSV508 is derived from the plasmid pWHM3 (J. Vara et aL. 1989) (cf. figure 13) in which the xis and int genes of pSAM2 (F. Boccard et aL. 1989b), placed under the control of the ptrc promoter (E. Amann et aL, 1988), have been added (cf. figure 14). The introduction, into the strain SPM50L of the plasmid pOSV508 carrying the xis and int genes of pSAM2 allows effective excision, by site-specific recombination, of the excisable cassette between the attL and attR sites flanking this cassette (A. Raynal et aL, 1998) (figure 10). Among the transformants, selected for their thiostrepton resistance due to the tsr gene carried by pOSV508, those which have become sensitive to hygromycin, for which the resistance gene is carried by the attlHhyg^ cassette, are chosen; the excision in fact leads to the loss of this resistance gene (cf. figure 10). The plasmid pOSV508 is unstable and, after two successive passages on medium without antibiotic, isolated clones are subcultured on medium with thiostrepton and on medium without thiostrepton. The thiostrepton-sensitive clones have lost pOSV508. It was verified that the excision of the cassette had indeed occurred in phase in the orf6* gene, by PCR and sequencing of the PCR product. The interruption begins at the 158th codon, 40 codons are deleted (120 bp), and the excision of the cassette leaves a characteristic "scar" attl sequence of 33 bp:<br>
5' ATCGCGCGCTTCGTTCGGGACGAAGAGGTAGAT 3' (SEQ ID No. 104).<br>
The strain thus obtained and having the desired genotype (orf6*::attI) was named SPM502.<br>
A sample of the strain SPM502 was deposited with the Collection Nationale de Cultures de Microorganismes (CNCM) Pasteur Institute, 25, rue du Docteur Roux, 75724 Paris Cedex 15, France, on July 10, 2002, under the registration number 1-2910.<br><br>
EXAMPLE  15: Analysis of the strains of Streptomyces ambofaciens with  a knockout in the or/2, orf3, orf8, orflO, orfl2, orfl3cy orfl4 or orf6* gene<br>
In order to test the spiramycin production of the various strains obtained, a microbiological test based on the sensitivity of a strain of Micrococcus luteus was developed'(cf. (A. Gourmelen et al^ 1998)). The strain of Micrococcus luteus used is a strain derived from the strain DSM1790 naturally sensitive to spiramycin (this strain is available in particular from the German Collection of Microorganisms and Cell Cultures (Deutsche Sammlung von Mikro-organismen und Zellkuituren GmbH, DSMZ), (Braunschweig, Germany), under the number DSM1790); the strain used in the present test differs from the strain DSM1790 in that it is resistant to congocidine. This strain is a spontaneous mutant obtained by selection on medium containing increasing doses of congocidine. Such a strain was selected due to the fact that Streptomyces ambofaciens produces both spiramycin and congocidine. Since the aim is to assay the spiramycin production of the various strains obtained using a microbiological test based on the sensitivity of a strain of Micrococcus luteus/il is necessary to have a congocidine-resistant strain.<br>
The various strains of Streptomyces to be tested were cultured in 500 ml Erlenmeyer flasks with baffles (baffled Erlenmeyers) containing 70 ml of MP5 medium (Pernodet et ai9 1993). The baffled Erlenmeyers were inoculated at an initial concentration of 2.5 x 106 spores/ml of the various strains of S. ambofaciens and grown at 27°C with orbital shaking at 250 rpm. 2 ml samples of suspensions were taken after 48, 72 and 96 hours of culturing, and centrifuged. The various supematants were then frozen at -20°C. A ten-fold dilution of these supematants in sterile culture medium is used for the test (cf. figure 18).<br>
The Micrococcus luteus indicator strain resistant to congocidine but sensitive to<br>
»	*   -<br>
spiramycin was cultured in 2TY medium (Sambrook et al., 1989) containing congocidine at 5 (ig/ml, for 48h at 37°C. The optical density (OD) of the culture is measured and this culture is diluted so as to adjust the optical density to 4. 0.4 ml of this<br><br>
preculture is diluted in 40 ml of DAMS medium (Difco Antibiotic Medium 5, marketed by the company Difco), brought to a temperature of approximately 45 °C beforehand. This medium is then poured into a 12 x 12 cm square dish and left to stand at ambient<br>
temperature.<br>
Once the medium had cooled and solidified, disks of Whatman AA paper (cf. A. Gourmelen et a/., 1998), 12 mm in diameter, were soaked with 70 \il of the ten¬fold dilution of each supernatant and placed on the surface of the dish. Disks soaked with a solution of spiramycin of various concentrations (2-4-8 ng/ml in MP5 culture medium) are used as a standard range. The dishes were left at 4°C for 2 h so as to allow diffusion of .the antibiotics into the agar, and were then incubated at 37°C for 24 to 48 h.<br>
If the disk contains spiramycin, this diffuses into the agar and inhibits the growth of the Micrococcus lutetts indicator strain. This inhibition creates a "halo" around the disk, this halo reflecting the area in which the Micrococcus luteus strain has not grown. The presence of this halo-is therefore an indication of the presence of spiramycin and makes it possible to determine whether the strain of S. ambofaciens corresponding to the disk in question is or is not a spiramycin producer. Comparison with the inhibition diameters obtained for the standard range makes it possible to obtain an indication of the amount of spiramycin produced by this strain.<br>
The various strains described in the preceding examples were used in this test in order to detect their spiramycin production. The results obtained were grouped together in Table 38.<br><br>
Table 38<br><br>
Strain	Inactivated gene	Example in<br>
which the<br>
strain is<br>
described	Phenotype: Producer<br>
(+) or nonproducer (-<br>
) of spiramycin<br>
ATCC2387<br>
7	None	1	(-)<br>
OS49.16	orjly.nhyg	2	(-)<br>
OS49.67	In-phase deletion orf3	6	(-)<br>
OS49.107	orfSy.Ohyg	7	(-)<br>
OS49.50	orflO:: Qhyg	8	(-)<br>
0SC2	None	10	(+)<br>
SPM21	orf2::att3fhac-	10	o<br>
SPM22	In-phase deletion orf2::att3	10	(-)<br>
SPM501	orf6*::attl&amp;hyg+	14	(-)<br>
SPM502	In-phase deletion orf6*::attl	14	<br>
SPM507	orfl2::att3fhac-	11                               (-)<br>
1                                                         i<br>
SPM508	orfl3c::att3Qaac-	12                             (+)<br>
SPM509	orfl4: :att3Qaac-	j<br>
13           |                 (-)<br><br>
These results make it possible to draw a certain number of conclusions regarding the function of the various genes involved in the spiramycin biosynthesis. Thus, the orf3 gene is essential to spiramycin biosynthesis. Specifically, an in-phase inactivation in this gene leads to a strain (OS-9.67, (cf. example 6)) which no longer produces spiramycin. The in-phase inactivation makes it possible to discard the hypothesis of a possible influence of the cassette introduced on the expression of the genes located downstream of orf3.<br>
Similarly, the orf8 and orflO genes encode proteins essential to spiramycin biosynthesis since the strains OS49.107 and OS49.50 have a nonproducer phenotype. In addition, in these latter two strains, it is clearly the inactivation of the corresponding gene which is responsible for this nonproducer phenotype, since, in view of the orientation of the various orfs (cf. figure 3), the construct introduced cannot have a polar effect.<br>
The study of the strains having an excisable cassette also makes it possible to draw a certain number of conclusions regarding the function of the interrupted gene. The strain SPM507 has the genotype: orf\2::ati3Qaac-. In view of the orientation of the genes (cf. figure 3), there is no point in excising the cassette to study the effect of the inactivation of orfl2. The fact that orfl3c is oriented in the opposite direction to orf!2 shows that these genes are not cotranscribed. The use of an excisable cassette makes it possible, on the other hand, to have the possibility of being rid of the selection marker at any moment. The phenotype of the strain SPM507 is nonproducer; it may therefore be concluded therefrom that the orfll gene is a gene essential to spiramycin biosynthesis in S. ambofaciens.<br>
The strain SPM508 has the genotype: orfl3c::att3£2aac~. In view of the orientation of the genes (cf. figure 3), there is no point *in excising the cassette to study the effect of the inactivation of orfl3c. The fact that orf\4 is oriented in the opposite direction to orfl3c shows that these genes are not cotranscribed. The use of an excisable<br><br>
cassette makes it possible, on the other hand, to have the possibility of being rid of the selection marker at any moment. The phenotype of the strain SPM508 is producer; it may therefore be concluded therefrom that the orfl3c gene is not a gene essential to spiramycin biosynthesis in 5. ambofaciens.<br>
The strain SPM509 has the genotype: orfl4::att3Qaac-. In view of the orientation of the genes (cf. figure 3), there is no need to excise the cassette to study the effect of the inactivation of orfl4\ the fact that orflSc is oriented in the opposite direction to or[14 shows that these genes are not cotranscribed. The use of an excisable cassette makes it possible, on the other hand, to have the possibility of being rid of the selection marker at any moment. The phenotype of the strain SPM509 is nonproducer; it may therefore be concluded therefrom that the orfl4 gene is a gene essential to spiramycin biosynthesis in 5. ambofaciens.<br>
The strain SPM21 has the genotype: orf2::att3Qaac-. This strain has a spiramycin nonproducer phenotype. However, the orientation of the genes orfl to orfS (cf. figure 3) implies that these genes are cotranscribed. Thus, the phenotype observed may be due to a polar effect of the cassette introduced into orf2 on the expression of genes located downstream in the operon. The strain SPM22 has the genotype orf2::att3 and was obtained after in-phase excision of the cassette introduced. The excision of the cassette leaves only a characteristic "scar" sequence (cf. example 10). Since the strain SPM22 also has a nonproducer phenotype, it may be concluded therefrom that the orf2 gene is a gene essential to spiramycin biosynthesis in S. ambofaciens. Only the effect due to the - inactivation of or/2 is observed here.<br>
The strain SPM501  has the genotype: orft^'.iattlQhyg^. This strain has a<br>
spiramycin nonproducer phenotype.  However,  since  the  orfj*  and  orf6*   genes<br>
. Ccf: figure 3) have the same orientation, the phenotype observed may be due to a polar<br>
effect of the cassette introduced into orf6* on the expression of orf5*. The arrangement<br>
of these genes implies that they may be cotranscribed. To reply to this question, the<br><br>
strain SPM502 was obtained after in-phase excision of the cassette introduced. In this strain, only the effect of the inactivation of orf6* is observed. This strain has the genotype orf6*::attl (cf. example 14). Excision of the cassette leaves only an in-phase "scar" sequence (cf. example 14). The strain SPM502 has a producer phenotype (however, this strain produces only spiramycin I (cf*. example 16)). It may therefore be concluded that the orp* gene is a gene essential to spiramycin biosynthesis in S. ambofaciens y since indirect inactivation thereof in the strain SPM501 leads to a nonproducer phenotype. On the other hand, the orf6* gene is not a gene essential to the biosynthesis of spiramycin I in S. ambofaciens (on the other hand, it is essential to the production of spiramycin II and III (cf. example 16)).<br>
EXAMPLE 16. Assaying of the production of spiramycins I, II and III in the mutant strains obtained<br>
The various strains to be tested were each cultured in seven 500 ml baffled Erlenmeyers containing 70 ml of MP5 medium (Pernodet et ai9 1993). The Erlenmeyers were inoculated with 2.5 x 106 spores/ml of the various strains of S. ambofaciens and grown at 27°C with orbital shaking at 250 rpm for 72 hours. The cultures corresponding to the same clone were pooled, optionally filtered through a pleated filter, and centrifiiged for 15 min at 7000 rpm. The various supernatants were then stored at-30°C.<br>
The assays were carried out by ion-pairing high performance liquid chromatography (HPLC). The HPLC analysis of the culture medium makes it possible to determine precisely the concentration of the three forms of spiramycin. The column used (Macherey-Nagel) is filled with a Nucleosil octyl grafted silica phase. The particle size is 5 /zm and the pore size is 100 A. The internal diameter of the column is 4.6 mm and it is 25 cm long. The mobile phase is a 70/30 (v/v) mixture of H3PO4 buffer (pH 2.2) and acetonitrile containing 6.25g /L of NaCIO* perchlorate. The analysis is carried<br><br>
out in an isocratic system with a flow rate fixed at 1 ml/min. The column is thermocontrolled at 23°C. Detection is by UV spectrophotometry at 238 nm. The sample is refrigerated at -flO°C and the quantification is determined from the area of the peaks (by external calibration). Under these conditions, the retention times for spiramycin I, II and EI are, respectively, approximately 17; 21 and 30 minutes, as could be verified using a commercial sample comprising the three forms of spiramycin.<br>
The strain OSC2 has a spiramycin producer phenotype. It is the parental strain used to obtain the strains having an excisable cassette (cf. example 15). This strain was therefore used as a positive control for production of the three forms of spiramycin. This strain clearly produces the three forms of spiramycin, as was verified by HPLC (cf. figure 19).<br>
The study of the strains having an excisable cassette makes it possible to draw a certain number of conclusions regarding the function of the interrupted genes. The strain SPM507 has the genotype: orfl2::att3Daac-. The phenotype of the strain SPM507 is nonproducer (cf. example 15); it may therefore be concluded therefrom that the orf 12 gene is a gene essential to spiramycin biosynthesis in S. ambofaciens. This strain no longer produces spiramycins, as was verified by HPLC (cf. figure 22). This result confirms the essential nature of the off 12 gene in spiramycin biosynthesis.<br>
The strain SPM508 has the genotype: orfl3c::att3Qaac~. The strain SPM508 has a spiramycin producer phenotype (cf. example 15); it may therefore be concluded therefrom that the orfl3c gene is not a gene essential to spiramycin biosynthesis in S. ambofaciens. This strain produces spiramycin I, II and IE, as was verified by HPLC (cf. figure 23). This result confirms that the orf 13c gene is not a gene essential to the biosynthesis of spiramycins I, II and HI in S. ambofaciens.<br>
The* strain SPM509 has the genotype: orfl4::att3Qaac-. The phenotype of the strain SPM509 is nonproducer; it may therefore be concluded therefrom that the orf!4 gene is a gene essential to spiramycin biosynthesis in S. ambofaciens. This strain no<br><br>
longer produces spiramycins, as was verified by HPLC (cf. figure 24), This result confirms the essential nature of the orfl4 gene in spiramycin biosynthesis.<br>
The strain SPM501  has the genotype: orf6*::attlilhygr. This strain has a spiramycin nonproducer phenotype. This strain no longer produces spiramycins, as was verified by HPLC (cf. figure 20). However, since the or/3* and orf6* genes (cf. figure 3) have the same orientation, the phenotype observed may be due to a polar effect of the cassette introduced into orf6% on the expression of orf5* in the operon. This implies that these genes are contranscribed. To reply to this question, the strain SPM502 was obtained by excision of the cassette introduced, producing an in-phase deletion in the or6* gene and restoring the expression of or/5*. This strain has the genotype orf6*::attl (cf. example 14 and 15). Excision of the cassette leaves only an in-phase "scar" att sequence (cf. example 14). The strain SPM502 has a spiramycin producer phenotype. However, as was proved by HPLC, this strain produces only spiramycin I and does not produce spiramycin II and HI (cf. figure 21). It may therefore be concluded from these results that the orf5* gene is a gene essential to spiramycin biosynthesis in S. ambofaciens, since indirect inactivation thereof in the strain SPM501 leads to a spiramycin nonproducer phenotype (cf figure 20). On the other hand, the off6* gene is not a gene essential to biosynthesis of spiramycin I in £ ambofaciens, since the inactivation of this gene leads to a spiramycin I producer phenotype (cf. figure 21). However, orf6* is essential to the production of spiramycin II and HI (cf. example 16)). The orf6* gene therefore clearly encodes an acyltransferase responsible for the modification of the platenolide at position 3 (cf. figure 1).<br>
EXAMPLE 17: Determination of the translation initiation point of orf 10 and improvement of spiramycin production<br>
17.1 Construction of the plasmids pSPM523. pSPMS24 and pSPM525:<br><br>
The orflO gene was identified in Streptomyces ambofaciens and was named srmR by Geistlich et al (M. Geistlich et a/., 1992). Inactivation of the orflO gene was carried out (cf. example 8). It was thus possible to show that the resulting strain no longer produces spiramycins (cf. example 15). This confirms that the orflO gene is clearly involved in spiramycin biosynthesis. The protein encoded by this gene is therefore clearly essential to spiramycin biosynthesis. However, analysis of the sequence shows that two ATG codons located in the same reading frame may be used for the translation of orflO (cf. Figure 28). One of the two possible codons (the most upstream codon) begins at position 10656 of the sequence given in SEQ ID No. 1, whereas the other possible codon, located more downstream, begins at position 10809 (cf. SEQ ID No. 1). Before testing a possible effect of the overexpression of srmR on spiramycin production, it is important to first determine the translation initiation point.<br>
With the aim of determining the translation iniation site, three constructs comprising three forms of orflO were produced. These forms were obtained by PCR using oligonucleotides comprising either a ifmdlQ restriction site or a BamHl restriction site.<br>
The first pair used for the amplification corresponds to the following oligonucleotides: EDR39:<br>
5' CCC AAGCTTGAG AAGGGAGCGGAC ATTC ATGGCCCGCGCCG AACGC3' (SEQ ID No. 122) (the HindHL site is shown in bold) EDR42:<br>
5'CGGGATCCGGCTGACCATGGGAGACGGGCGCATCGCCGAGTTCAGC3' (SEQ ID No. 123) (the BamHl site is shown in bold)<br>
The pair of primers EDR39-EDR42 allows the amplification of an fragment of off 10 comprising the ATG located in the most 3' position (position 10809 in the sequence given in SEQ ID No. 1) (cf. Figure 28), The fragment obtained is approximately 2 kb in size and will subsequently be referred to as "short qrfl0"\ it does not contain the.or/70 promoter. This 2kb fragment was cloned into the vector pGEM-T easy, to give the plasmid pSPM520.<br><br>
The   second   pair   used   for   the   amplification   corresponds   to   the   following<br>
oligonucleotides:<br>
EDR40:<br>
S'CCCAAGCTTGAGAAGGGAGCGGACATTCAATGCTTTGGTAAAGCACS'<br>
(SEQ ID No. 124) (the HindUl site is shown in bold) EDR42:<br>
5 'CGGGATCCGGCTGACCATGGGAGACGGGCGCATCGCCGAGTTCAGC3'<br>
(SEQ ID No. 123) (the BamHl site is shown in bold)<br>
The pair of primers EDR40-EDR42 allows the amplification of a fragment of orflO<br>
comprising the ATG located in the most 5' position (position 10656 in the sequence<br>
given in SEQ ID No. 1) (cf. Figure 28). This 2.2 kb fragment, subsequently referred to<br>
as "long orflO", was cloned into the vector pGEM-T easy, to give the plasmid<br>
pSPM521; this plasmid does not contain the orflO promoter.<br>
The third pair used for the amplification corresponds to the following oligonucleotides:<br>
EDR41:<br>
S'-CCCAAGCTTTCAAGGAACGACGGGGTGGTCAGTCAAGT-S'<br>
(SEQ ID No. 125) (the HindSL site is shown in bold)<br>
EDR42:<br>
S'CGGGATCCGGCTGACCATGGGAGACGGGCGCATCGCCGAGTTCAGCS'<br>
(SEQ ID No. 123) (the BamHl site is shown in bold)<br>
The pair of primers EDR41-EDR42 allows the amplification of the orflO gene with the<br>
two ATGs, and also its own promoter (cf.  figure 28). This 2.8 kb  fragment,<br>
subsequently referred to as "pro orflO", was cloned into the vector pGEM-T easy, to<br>
give the plasmid pSPM522.<br>
The "pro or/70" fragment was obtained using as matrix the chromosomal DNA of the<br>
strain OSC2. The "short or/70" and "long or/70" fragments were, themselves, obtained<br>
using as matrix the DNA of the "pro orflO" fragment purified beforehand.<br>
The HiwHBrBamHl inserts of the plasmids pSPM520, pSPM521 and pSPM522 were then subcloned into the vector pUWL201 (plasmid derived from the plasmid pUWL199 (U.F. Wehmeier, 1995) in to which the Kpnl-BamHl fragment of the region<br><br>
of the ermE promoter (cf. Bibb et ai, 1985, in particular figure 2) carrying a mutation which increases the strength of the promoter (ermE* promoter) (Bibb et aU 1994) has been introduced (cf. Doumith et al, 2000)) predigested with the HindJR-BamHI enzymes. Thus, three plasmids were obtained: pSPM523 (derived from pUWL201 with the "short orflCT form as insert), pSPM524 (derived from pUWL201 with the 'long orflT form as insert) and pSPM525 (derived from pUWL201 with the "pro orflT form) (Figure 28).<br>
17.2 Transformation of the strain OS49.50 with the constructs pSPM523, pSPM524andpSPM525:<br>
The strain OS49.50 (strain with a knockout in the orflQ gene, cf. example 8) was transformed independently by protoplast transformation (T. Kieser et ai, 2000) with each of the plasmids pSPM523, pSPM524 and pSPM525. A negative control was also prepared by transforming the strain OS49.50 with the plasmid pUWL201 without insert. After protoplast transformation, the clones were selected for their thiostrepton resistance. The transformation of the clones with each of the plasmids was verified by extraction of these plasmids. Thus, four new strains were obtained: the strain OSC49.50 pUWL201, derived from the transformation with the plasmid pUWL201 without insert; the strain OSC49.50 pSPM523, derived from the transformation with the plasmid pSPM523; the strain OSC49.50 pSPM524, derived from the transformation with the plasmid pSPM524; and, finally, the strain OS49.50 pSPM525, derived from the transformation with the plasmid pSPM525.<br>
The spiramycin production of each of these four strains was tested by HPLC. For this, the various strains of Streptomyces to be tested were cultured in 500 ml Erlenmeyers with baffles (baffled Erlenmeyers) containing 70 ml of MP5 medium (Pemodet et ai, 1993). When the strain contains the plasmid pUWL201, or one of its derivatives, 5 ^g/rnl of thiostTepton are added. The baffled Erlenmeyers were inoculated at an initial concentration of 2.5 x 106 spores/ml of the various strains of 5 ambofaciens, and the cultures were incubated at 27°C with orbital shaking" at 250 rpn<br><br>
for 96 hours. The cells were then separated from the medium by centrifugation and the supernatant was analyzed by HPLC (cf. example 16) in order to determine the amount of spiramycin produced. By virtue of a standard sample and measuring the area of the peaks, it was possible to determine the amount of each of the spiramycins produced by these strains. The results'of this analysis are given in table 39. the data are expressed in mg per liter of supernatant. The results correspond to the total production of spiramycins (obtained by adding the production of spiramycin I. II and HI).<br>
Table 39. Spiramycin production of the strains derived from OS49.50, (results expressed in mg/1).<br><br>
As shown by the results given in table 39, the negative control (strain OS49.50 transformed with the plasmid pUWL201) does not produce spiramycin. When the plasmid pSPM523 (which contains the "short or/70" form) is introduced into the strain OS49.50, no spiramycin production is observed. On the other hand, the presence of the plasmid pSPM524 (which contains the "long orfW form) and of the plasmid pSPM525 (which contains the "pro or/70" form) restores the spiramycin production in the host strain OS49.50. Thus, only the orflO fragments containing the most upstream ATG make it possible to restore spiramycin synthesis.<br><br>
With the aim of confirming these results, the nlasmid oS?M52I  (oiasmid pGEM-T easy containing .the "long orflT form* was digested with the A7:cl restriction<br>
---      tTi i^ira    ^ \ j 1	-».-^      ,'    j/~)i     .*r*n~    -'-.o-.-i    '"tarj    '-&gt; * i' — -    ^(li.-i    \* i   «-**e»'i j*-—-* -»n f    *.V*~,    ■'"■.a    V* *-a*-* fa V<br>
-&gt;*---",.■■•-*■&gt; *a      ' r\&amp; V) \ '"wr*"1 11""   "V*N r^iAr   "^o^   TIP'.' M*-\   ^, r-   --~=i**- %T- i-Vv* T .-* • 
*-,--,    ,-yi * -^    I'KP    nl '"isT"^ * ^    "*S"P\f^^™*      i "    ' ^f3    r&gt;^f cr    ' :*^c*r^^,r-«     iTi""-    i **o ^I t"1 i^r*     ' 0/-&gt; ^ ri    • r^    I-^IJ<br>
—	•-	»	.i<br>
;^^a-nV-v   ^i / wi*.   .ii   w-. — ^    LLS   .^o.    * .    .J    liiU.—-a    _»,i^,vj.   ui    *-*. J-iiotLi..^ i.^   .i^Uu^Uu   site,    .UIJ<br>
treatment will lead to a shin in the reading trance at the A7:cl site and w;A have the effect of producing a protein exhibiting no activating acivity. On the other hand, if the translation initiation takes piace at the mos: downstream ATG (position 10S09 in the sequence given in SEQ ID No. 1), this treatment should have little or no effect on the expression of OrflO (given the location of the transcription initiation point) and no effect on the protein produced.<br>
The BamHl-HindYh insert of DSPM527 was then subcioned into the vector plTWL201, to give the plasmid pSPM52S. This piasmid was introduced into the strain OS49.50 and a clone having the desired plasmid was more narticularly selected. The spiramycin production c'the resulting strain was then tested by HPLC (cf. example 16 and above). Unlike what was observed with the plasmid pS?M524 (cf. table 39), the presence of the plasmid pSPM528 in the strain OS49.50 does not reestablish the spiramycin production. This confirms that the translation initiation of the orflO gene is the ATG located in the most downstream position (ATG 1 cf. Figure 2S).<br>
17,3 Improvement in spiramycin production of the S. ambofaciens strain<br>
	        I.I—	*	I	"      ■	I        I	—	■	I      P      I.    I        I      J—	»	——I-       I—	|	II.    I      .	■	—-J   .   .	.,—	.	.      ■   M—J.	Il.l	—        I     ■—■		-	—     ""<br>
0SC2:<br>
In order to test the effect of overexpression of the orflO gene on spiramycin production, the plasmids pSPM523, pSPM524, pS?M525 and pS?M52S were introduced into the strain 0SC2. For this, nrotoniasts of the strain OSC2 were transformed (T. Kieser et aL9 2000) independently with each of the plasmids pSPM523, pSPM524, pSPM525 and pSPM528. A negative control was also produced by transforming the  strain  OSC2'with  the plasmid pUWL201   without  insert.   After<br><br><br><br><br>
Thus, it is observed that the presence of the plasmid pSPM524 or of the plasmid pSPM525 significantly increases the spiramycin production of the strain OSC2. This clearly demonstrates that overexpression of OrflO has a positive effect on the spiramycin production. On the other hand, the plasmid pSPM528 has no effect on the spiramycin production.<br>
In the same way, the plasmids pSPM525 and pUWL201 were introduced into the strain SPM502 (cf. example 14). Thus, two. new strains were obtained: the strain SPM502 pUWL201, derived from the transformation with the plasmid pUWL201 without insert; and the strain SPM502 pSrM^25^aerived frnm th*» transformation with the plasmid pSPM525.<br>
A sample of the strain SPM502 pSPM525 (this strain contains the plasmid pSPM525, cf. above) was deposited with the Collection Nationale de Cultures de Microorganismes (CNCM) Pasteur Institute, 25, rue du Docteur Roux 75724 Paris Cedex 15, France, on February 26,2003 under the registration number 1-2977.<br>
The spiramycin production of the strains SPM502 pUWL201 and SPM502 pSPM525 was analyzed by HPLC (in the same way as in example 17.2). Analysis of the spiramycin production of the strain SPM502 was also carried out in parallel for comparison. The results of this analysis are given in table 41, the data are expressed in mg per liter of supernatant. The results correspond to the production of spiramycin I. In fact, none of these strains produces spiramycin II and HI.<br>
Table 41. Spiramycin I production of the strains derived from SPM502, (results expressed in mg/1).<br><br><br>
Thus, it was possible to observe that overexpression of the orflO gene in the strain SPM502 considerably increases the production of spiramycin I.<br>
EXAMPLE 18: Construction of a genomic DNA library of the Streptomyces ambofaciens strain OSC2 in £. coli in the cosmid pWED2.<br>
18.1 Construction of the cosmid pWED2:<br>
With the aim of facilitating the inactivation of genes in Streptomyces, a cosmid carrying the oriT sequence of the plasmid RK2 (which allows its introduction by conjugation into Streptomyces from a suitable strain of E. coli) and also carrying a gene for resistance to a an antibiotic imparting a detectable phenotype in Streptomyces was constructed. Such a cosmid containing large inserts of genomic DNA of Streptomyces ambofaciens can be used in gene inactivation experiments.<br>
To construct this vector, a pac-oriT cassette (EcoRV fragment) was introduced into the cosmid pWEDl (Gourmelen et al, 1998), derived from the cosmid p WED 15 (Wahl et al, 1987), at the unique Hpal site. The pac-oriT cassette was obtained by PCR. For this, tiicpac gene was amplified by PCR from the plasmid pVF 10.4 (Vara et al, 1985; Lacalle et al, 1989) using, as first primer, primer A (of sequence 5'-CCAGTAGATATCCCGCCAACCCGGAGCTGCAC-3' (SEQ ID No. 126), the EcoRV restriction site has been underlined and the 20 nucleotide sequence in bold<br><br><br><br>
conjugation into Streptomyces from a suitable strain of £. colu but also a gene for resistance to an antibiotic imparting a detectable phenotype in Streptomyces. Such a cosmid containing large inserts of genomic DN'A of Streptomyces ambofaciens can be used in gene inactivarion experiments.<br>
Thus, a cosmid derived from pWED2 containing the target gene may, for example, be introduced into the E. coli strain KS272 containing the plasmid pKOBEG (cf. Chaveroche et al 2000) and a cassette will be introduced into the target gene according to the technique described by Chaveroche et al. 2000. The cosmid obtained by this technique (cosmid in which the target gene is inactivated) may then be introduced into an E. coli strain such as the SI7.1 strain or any other strain making it possible to transfer plasmids containing the oriT sequence to Streptomyces by conjugation.<br>
After conjugation between E. coli and Streptomyces, clones of Streptomyces in which the wild-type copy of the target gene will have been replaced with the interrupted copy maybe obtained as described in example 2.<br>
The resistance gene expressed in Streptomyces, present on this new cosmid, is the pac gene of Streptomyces alboniger (J. Vara et al, 1985; Lacalle et al, 1989), which encodes puromycin N-acetyltransferase and which imparts puromycin resistance. In gene inactivation experiments, the clones in which a double recombination event has taken place will be sought. These clones will have eliminated the cosmid pWED2 and will therefore have become sensitive to puromycin again.<br>
18.2 Construction of a genomic DNA library of the Streptomyces ambofaciens strain OSC2 in E. coli in the cosmid pWED2<br>
The genomic DNA of the Streptomyces ambofaciens strain OSC2 was partially digested with the BamHl restriction enzyme so as to obtain DNA fragments of between approximately 35 and 45 kb in size. These fragments were then cloned into the cosmid p\VED2, the latter having been digested beforehand with BamHl, and then treated with alkaline phosphatase. The ligation mixture.was then encapsidated in vitro in lambda phage particles using *the "Packagene® Lambda DN'A packaging system" marketed by<br><br>
the company Promega (Madison, Wisconsin, USA) according to the manufacturer's<br>
recommendations. The phage particles obtained were used to infect the E. col: strain SURE'D marketed by the company Stratagene (LaJoila, California, USA). The clones were selected on LB medium - anpicillin (50 ug;ml), the cosmid pWEDI imparting ampicillin resistance.<br>
EXAMPLE 19: Isolation of cosmids of the new library covering the region of the biosynthetic pathway for spiramycins. Subcloning and sequencing of fragments of these cosmids.<br>
19.1   Hybridization  on  colonies  of the   genomic   library  of Strevtomvces<br>
«•			■'■■ ■    '        ■'■'■	-    '	•	mi "i	-■■■■"	~ '	—	■	— -1 -i——	 ..—.        .	,i	i	a			I,,.<br>
ambofaciens OSC2:<br>
Cosmids of the new library of Streptomyces ambofaciens OSC2 (cf. example 18) covering orfl* to orflO* or a part or all of orfl to or/25c, or a region more upstream of or/25c, were isolated. For this, successive hybridizations on the colonies of E. coli obtained according to example 18 were performed using the following 3 probes (cf. figure 31):<br>
-	The first probe used corresponds to a DNA fragment of approximately 0.8 kb<br>
amplified by PCR using as matrix the cosmid pSPM5, and the following primers:<br>
ORF23c:   S'-ACGTGCGCGGTGAGTTCGCCGTTGCo'   (SEQ  ID  No.   130)   and<br>
ORF25c: S'-CTGAACGACGCCATCGCGGTGGTGC-S1 (SEQ ID No. 131).<br>
The PCR product thus obtained contains a fragment of the start of orf23ct orf24c in its entirety and the end oiorf25c (cf. figure 31, probe I).<br>
-	The second probe used corresponds to a DNA fragment of approximately 0.7 kb<br>
amplified by PCR using as matrix the total DNA. of the S. ambofaciens strain<br>
ATCC23877, and the following primers:<br><br>
ORFl*c:  5'-GACCACCTCGAACCGTCCGGCGTCA-3'   (SEQ  ED  No.   132)  and ORF2*c: 5'-CK5CCCGGTCCAGCGTGCCGAAGC-3' (SEQ ID No. 133). The PCR product thus obtained contains a fragment of the end of orfl *c and of the start of orC*c (cf. figure 31, probe H).<br>
- The third probe used corresponds to an EcoRl-BamHI fragment of approximately 3 kb containing orfl, orfl and orf3 and obtained by digestion cf the plasmid pOS49.99 (cf. figure 31, probe HI).<br>
Approximately 2000 clones of the library obtained in example 18.2 were transferred onto a filter for colony hybridization according to conventional techniques (Sambrookera/., 1989).<br>
The first probe (cf. figure 31, probe I) was labeled with 3:P by the random priming technique (kit marketed by the company Roche) and used for the hybridization on 2000 clones of the library, after transfer onto a filter. The hybridization was carried out at 65°C in the buffer described by Church &amp; Gilbert (Church &amp; Gilbert, 1984). Two washes were carried out in 2 x SSC, 0,1% SDS at 65°C, the first for 10 minutes and the second for 20 minutes, and a third wash lasting 30 minutes was then carried out in 0.2X SSC, 0.1% SDS at 65°C. Under these hybridization and washing conditions, 20 clones out of the 2000 hybridized exhibited a strong hybridization signal with the first probe. These 20 clones were cultured in LB medium + ampicillin (50 jig/ml) and the corresponding 20 cosmids were extracted by alkaline lysis (Sambrook et aL, 1989) and then digested with the BamEI restriction enzyme. The digestion products were then separated on agarose gel, transferred onto a nylon membrane and hybridized with the first probe (cf above: the PCR product ORF23c-ORF25c, probe I) under the same conditions as above. Twelve cosmids contained a BamKl fragment which hybridized strongly with the probe used.  These  12 cosmids were named pSPM34, pSPM35, pSPM36,  pSPM37,   pSPM38,   pSPM39,   pSPM40,   pSPM41,   pSPM42,   pSPM43, pSPM44 and pSPM45. The profiles, after digestion with BamHI, of these 12 cosmids were compared with one another and with that of the cosmid pSPM5. In addition, PCR<br><br>
amplification experiments using various primers corresponding to various genes already-identified in the orfl-orf25c region made it possible to position the insert of some of these cosmids with respect to one another and :o also determine the location of these inserts in the already known orfI-orf25c regicr. (cf. figure 32). The cosmic pSPM36 was more particularly chosen since it was liable to contain a large region upstream of orflSc (cf. figures 31 and 32).<br>
Next, using :he same conditions as those described above, the 2000 clones of the library of Streptomyces ambofaciens OSC2 were hybridized with the second probe corresponding to the PCR product: ORFl*c-ORF2*c (cf. figure 31, probe II). This hybridization made it possible to isolate cosmids whose insert is located in the region from orfl*c to orfl0*c. Under the hybridization conditions used, 16 clones out of the 2000 hybridized exhibited a strong hybridization signal with this second probe. These 16 clones were cultured in LB medium + ampicillin (50 ug'ml) and the corresponding 16 cosmids were extracted by alkaline lysis (Sambrook et ai, 1989) and then digested with the BamYLl restriction enzyme. The digestion profiles (after digestion with BamHT) of these 16 cosmids were compared with one another and with that of the cosmid pSPM7. PCR amplification experiments with the primers ORFl*c and ORF2*c made it possible to choose two cosmids which clearly contained the orfl*c and orf2*c genes and the profiles of which had common bands but also different bands. In addition, other PCR amplification experiments using various primers corresponding to various genes already identified in the region orfl*c to orfI0xc made it possible to position the insert of these cosmids with respect to one another and to also determine the location of these inserts in the already known region orfl*c to orfl0*c (cf. figure 32). The two cosmids more particularly selected were named pSPM47 and pSPM48 (cf. Figure 32).<br>
Using the same conditions as those described above, the 2000 clones of the library of Streptomyces ambofaciens OSC2 were also hybridized with the third probe corresponding to the EcoRl-BamHl DNA fragment of the plasmid pOS49.99 (cf. figure 31 probe EQ). This hybridization made it possible to isolate the cosmids<br><br>
containing the region from orfl up to orft and liable to contain either a large part of the PKS genes or a large part of the genes orfl to orfl 5 c of the biosynthetic pathway for spiramycins. Under these hybridization conditions, 35 clones out of the 2000 hybridized exhibited a strong hybridization signal with the third probe. These 35 clones were cultured in LB medium - ampiciilin (50 ug;ml) and the corresponding 35 cosmids were extracted by alkaline lysis (Sambrook et al, 1989) then digested with the BamEl restriction enzyme. The profiles, after digestion with BamHl. of these 35 cosmids were compared with one another and with that of the cosmid pSPM5. In addition, PCR amplification experiments using various primers corresponding to various genes already identified in the region orfl to orfl 5c made it possible to verify that these cosmids clearly contained inserts originating from the region orfl to orfl5c and to position the inserts of these cosmids with respect to one another and with respect to the already known region orfl to orf!5c (cf. figure 32). Five cosmids were more particularly selected, they were  named pSPM50, pSPMSl, pSPM53, pSPM55  and pSPM56 (cf. figure 32).<br>
19.2 Subcloning and sequencing of a part of the insert of the cosmid pSPM36<br>
The probe of approximately 0.8 kb obtained by PCR with the primers ORF23c and ORF25c (cf. above and figure 31, probe I) was also used in Southern blotting experiments on the total DNA of 5. ambofaciens OSC2 digested with the Pstl enzyme. Under the hybridization conditions described above, this probe reveals a single Pstl fragment of approximately 6 kb when hybridized on the total DNA of £ ambofaciens OSC2 digested with the Pstl enzyme. A Pstl site exists in the region of orf23c (cf. SEQ ID No. 80), but no other Pstl site exists up to the end (BamHl site) of the known sequence (cf. SEQ ID No. 1). This Pstl-BamEl fragment is approximately 1.4 kb in size. The 6 kb Pstl fragment hybridized on the total DNA of S. ambofaciens digested with the Pstl enzyme therefore contains a region of approximately 4.6 kb located upstream of orf!5c. This region is liable to contain other genes the products of which are involved in the biosynthetic pathway for spiramycin. It was verified, by digestion, that the cosmid pSPM36 indeed contained this 6 kb Pstl fragment. This fragment was<br><br>
isolated from pSPM36, with the aim of determining the sequence further upstream of<br>
orj25c. For this, the cosmid pSPM36 was digested with the Pstl restriction enzyme. The PstVPstl fragment, approximately 6 kb in size, was isolated by electroelution from a 0.8% agarose gel and then cloned into the vector pBK-CMV (4512 bp) (marketed bv the company Stratagene (La Jolla, California, USA')). The plasmid thus obtained was named pSPM58 (cf. figure 33) and the sequence of its insert was determined. The sequence of this insert is given in SEQ ID No. 134. However, not all the sequence was determined and a gap of approximately 450 nucleotides remains, the part of the sequence undetermined was noted by a succession of "NTs in the corresponding sequence.<br>
19.3 Analysis of the new nucleotide sequences, determination of the open reading frames and characterization of the genes involved in spiramycin biosynthesis<br>
The sequence of the insert of the cosmid pSPM58 obtained was analyzed using the FramePlot program (J. Ishikawa &amp; K. Hotta 1999). This made it possible to identify, among the open reading frames, the open reading frames exhibiting a codon use typical ofStreptomyces. This analysis made it possible to determine that this insert comprises 4 new ORFs upstream of orf25c (cf. figure 34). These genes were named or/26 (SEQ ID No. 107), or/27 (SEQ ID No. 109), or/28c (SEQ ID No. Ill, the sequence of this orf was not completely determined since a gap of approximately 450 nucleotides remains in the sequencing of the insert of pSPM58, these 450 nucleotides appearing in the form of a series of "N"s in the sequence SEQ ID No. Ill) and or/29 (the sequence of the latter orf was incomplete in this insert). The "c" added to the name of the gene means, for the ORF in question, that the coding sequence is in the reverse orientation (cf. figure 34).<br>
The protein sequences deduced from these open reading frames were compared with those present in various databases using various programs: BLAST (Altschul et al.i 1990)/Altschul et al, 1997), CD-search, (these two programs are accessible in particular from the National Center for Biotechnology Information (NCBI) (Bethesda, Maryland, USA)), FASTA ((W.R. Pearson &amp; D.J. Lipman, 1988) and (W.R. Pearson,<br><br>
1990) (this program is accessible in particular from the INFOBIOGEN resource center, Evry, France). These comparisons made it possible to formulate hypotheses regarding the function of the products of these genes and to identify those liable to be involved in spiramycin biosynthesis.<br>
19.4 Subcloning -and sequencing of another part of the insert of the cosmid PSPM36.<br>
The probe of approximately 0.8 kb obtained by PCR with the primers ORF23c and ORF25c (cf. above and figure 31, probe I) was also used in Southern blotting experiments on the total DNA of the strain OSC2 digested with the Stul enzyme. Under the hybridization conditions described above for this probe, this probe reveals a single Stul fragment of approximately 10 kb when hybridized on the total DNA of the strain OSC2 digested with the Stul enzyme. Given the presence of an Stul site in orf23c (cf. SEQ ID No. 80) and the location of this site relative to the Pstl site, this 10 kb fragment includes all of the Pstl fragment previously studied (insert of pSPM58) and makes it possible to have access to an approximately 4 kb region not yet studied (cf. figure 33). It was verified, by digestion, that the cosmid pSPM36 indeed contained this 10 kb Stul fragment. This fragment was isolated from the cosmid pSPM36, with the aim of determining the sequence of the end of orf29 and of other genes further upstream of orf29. For this, the cosmid pSPM36 was digested with the Stul restriction enzyme. The Stul-Stul fragment, approximately 10 kb in size, was isolated by electroelution from a 0.8% agarose gel and then cloned into the vector pBK-CMV (4512 bp) (marketed by the company Stratagene (La Jolla, California, USA)). The plasmid thus obtained was named pSPM72 (cf. figure 33). The latter was then digested with EcoRl (EcoRl site in the insert of pSPM58) and HindJR (HindIR site in the multiple cloning site of the vector, immediately after the Stul site of end of the insert) (cf. figure 33). The EcoBI-HindUl DNA fragment thus obtained corresponds to a fragment of the insert of the plasmid pSPM72 (cf. figure 33) and was subcloned into the vector pBC-SK&gt; (marketed by the company Stratagene (La Jolla, California, USA)) digested beforehand with EcoKL and HindTR. The plasmid thus obtained was named pSPM73 and the<br><br>
sequence of its insert was determined. The sequence of this insert is given in SEQ ID No. 135.<br>
An assembly of the sequences of the inserts of pS?M53 and pSPM73 is g:ven in SEQ ID No. 106. This sequence starts rem the Pstl site in orf23c (cf. SEQ ID No. 80) and continues to the Stul site in or/32c (cf. figure 3-1. Since the sequence of crf28c (SEQ ID No. Ill) is not complete (cf. example 19.3). a region of approximately 450 nucleotides is not sequenced, these 450 nucleotides appear in the form of a series of "&gt;Ts in the sequence SEQ ID No. 106.<br>
19.5 Analysis of the new nucleotide sequences, determination of the ooen reading frames and characterization of the genes involved in spiramycin biosynthesis<br>
The partial sequence of the insert of the cosmid pSPM73 obtained was analyzed using the FramePlot program (J. Ishikawa &amp; K. Hotta, 1999). This made k possible to identify, among the open reading frames, the open reading frames exhibiting a codon use typical of Streptomyces. This analysis made it possible to determine that this insert comprises 4 ORFs, one incomplete and three complete (cf. figure 34). The incomplete ORF corresponds to the 3' portion of the coding sequence of or/29, which made it possible to complete the sequence of this gene by virtue of the partial sequence of this same orf obtained during the sequencing of the insert of the plasmid pSPM58 (cf. examples 19.2 and 19.3); the combination of the two sequences thus made it possible to obtain the complete sequence of orf29. The 4 genes were thus named or/29 (SEQ ID No. 113), or/30c (SEQ ID No. 115), or/3! (SEQ ID No. 118) and ori32c (SEQ ID No. 120). The "c" added to the name of the gene means, for the ORE in question, that the coding sequence is in the reverse orientation (cf. figure 34).<br>
The protein sequences deduced from these open reading frames (SEQ ID No. 114 for or/29, SEQ ID No. 116 and 117 for or/30c, SEQ ID No. 119 for orftl and SEQ ID No. 121 for orf32c) were compared with those present in various databases using various programs: BLAST (Altschul et a!.y 1990) (Altschul et al, 1997), CD-search, (these  two  programs   are  accessible  in  panicular  from  the  National   Center  for<br><br>
Biotechnology Information (NCBI) (Bethesda, Maryland, USA)), FASTA ((W.R. Pearson &amp; D.J. Lipman, 1988) and (W.R. Pearson. 1990) (this program is accessible in particular from the INFOBIOGEN resource center. Evry, France). These comparisons made it possible to formulate hypotheses regarding the function of the products of these genes and to identity- those liable 10 be involved in spiramycin biosynthesis.<br>
19.6 Subcloning and sequencing of a third portion of the insert of the cosmid PSPM36.<br>
A probe (0.8 kb DNA fragment) corresponding to a sequence internal to orfilc was obtained by PCR using, as matrix, the total DNA of the Streptomyces ambofaciens strain and the following primers<br>
KF36: 5'- TTGCCGTAGCCGAGGACCAGCG-3' (SEQ ID No. 151) and KF37: 5'- CACATGGCCCTGGAGGACCCTG-3' (SEQ ID No. 152).<br>
The PCR product thus obtained represents an internal sequence of orf32c. This probe was used in Southern blotting experiments on the chromosomal DNA of the strain OSC2 and on the DNA of the cosmid pSPM36, digested with the Pstl enzyme. Using the same hybridization conditions as those described above (cf. example 19.1), this probe reveals two Pstl fragments of approximately 3.4 kb and 2.5 kb when hybridized on the total DNA of the strain OSC2 and on the DNA of the cosmid pSPM36, digested with the Pstl enzyme. Given the presence of a Pstl site in the probe used, these results can be explained. The first DNA fragment, which has a size of approximately 3.4 kb, is a fragment the sequence of which is already entirely known. The sequence of the 2.5 kb fragment is only partially known, over a region of 700 bp. This fragment was isolated from the cosmid pSPM36 with the aim of determining the sequence of the end of orfSlc and other genes upstream of the latter. For this, the cosmid pSPM36 was digested with the Pstl restriction enzyme. The Pstl-Pstl fragment, approximately 2.5 kb in size, was isolated by purification from a 0.8% agarose gel and<br><br>
then cloned into the vector pBK-CMV (4518bp) (marketed by the company Stratagene ('La Jolla, California. USA)). The piasmid thus obtained was called pS?M~9 (cf. figure 41) and the sequence of its insert was determined.<br>
The sequence of orf28c (SEQ D No. 111; was not complete (cf. example 19.3). Specifically, it had not been possible to determine a region of approximately 450 nucleotides, these 450 nucleotides appear in the form of a series of ""NTs in :he sequence SEQ ID No. 106. The seauence of the entire missing resion was determined bv resequencmg this region. The sequence of the inserts of pSPMSS and pSPM73 was therefore determined in its entirety. The complete sequence of the coding portion of orf28c is given in SEQ ID No. 141 and the protein deduced from this sequence in SEQ ID No. 142. The sequence of the insert of the piasmid pSPM79 is given in SEQ ED No. 161.<br>
An assembly of the sequences of the inserts of pSPM58, pSPM73 and pSPM79 is given in SEQ ID No. 140 (cf. figure 41). This sequence starts from the Psil site in orfISc (cf. SEQ ID No. 80) and continues up to the Pstl site in orf34c (cf. figure 41).<br>
19.7 Analysis of the new nucleotide sequences, determination of the ooen reading frames and characterization of the genes which mav be involved in spiramycin biosynthesis.<br>
The sequence of the insert of the piasmid pSPM79 obtained was analyzed using the FramePlot program (Ishikawa J &amp; Hotta K. 1999). This made it possible to identify, among the open reading frames, the open reading frames exhibiting a codon usage typical of Streptomyces. This analysis made it possible to determine that this insert contains 3 ORFs, two incomplete iorf32c and orf34c) and one complete (or/33) (cf. figure 41).<br>
The first incomplete ORF corresponds to the 5' portion of the coding sequence of orf32c. This made it possible to complete the sequence of this gene by virtue of the<br><br>
partial sequence of this same orf obtained during the sequencing of the insert of the plasmid pSPM73 (cf. examples 19.4 and 19.5), the combination of the two sequences thus made it possible to obtain the complete sequence of orf32c. (SEQ ID No. 145). The *4c" added to the name of the gene means, for the ORF in question, that the coding sequence is in the reverse orientation (cf. figure 41). The complete orf was called or/33 (SEQ ID No. 147). The third ORF was called orf34c (SEQ ID No. 149). The "c" added to the name of the gene means, for the ORF in question, that the coding sequence is in the reverse orientation (cf. figure 37). The comparisons performed between the product of this orf and the databanks suggest that the C-terminal portion of this protein is not in the product deduced from the nucleotide sequence and therefore that this orf is longer and continues beyond the region sequenced.<br>
The protein sequences deduced from these open reading frames were compared with those present in various databases using various programs: BLAST (Altschul et al., 1990) (Altschul et al., 1997), CD-search (these two programs are accessible in particular from the National Center for Biotechnology Information (NCBI) (Bethesda, Maryland, USA)), FASTA ((Pearson W. R. &amp; D. J. Lipman, 1988) and (Pearson W. R., 1990) (this program is accessible in particular from the INFOBIOGEN resource center, Evry, France). These comparisons made it possible to formulate hypotheses regarding the function of the products of these genes and to identify those which may be involved in spiramycin biosynthesis.<br>
EXAMPLE 20: Analysis of the production of spiramycin biosynthesis intermediates:<br>
20.1 Sample preparation:<br>
The various strains to be tested were each cultured in seven 500 ml baffled Erlenmeyers containing 70 ml of MP5 medium (Pernodet et ai, 1993). The Erlenmeyers  were inoculated  with 2.5 x 10   spores/ml  of the  various  strains  of<br><br>
S. ambofaciens and grown at 27°C with orbital shaking at 250 rpm for 96 hours. The cultures corresponding to the same clone were pooled, optionally filtered through a pleated filter, and centrifuged for 15 min at 7000 rpm.<br>
The pH of the must, was then adjusted to 9 with sodium hydroxide and the supernatant was ex:racted with methyl isoburyryl ketone (ME3K). Tne organic phase (MIBK) was then recovered and evaporated. The drv'extract was then taken UD in 1ml of acetonitrile, then diluted to 1/10 (100 /zl qs I ml with water) before being used for the liquid chromatography/mass spectrometry (LC/MS) analyses.<br>
20.2	Analysis of the samples bv LC/MS:<br>
The samples were analyzed by LC/MS with the aim of determining the mass of the various products synthesized by the strains to be tested.<br>
, The high performance liquid chromatography column used is a Kromasil C8 150* 4.6mmm, 5mmm, 100 A column.<br>
The mobile phase is a gradient consisting of a mixture of acetonitrile and an aqueous<br>
0.05% trifluoroacetic acid solution, the flow rate is fixed at 1 ml/min. The temperature<br>
of the column oven is maintained at 30°C.<br>
The UV detection at the column outlet was carried out at two different wavelengths:<br>
238 nm and 280 run.<br>
The mass spectrometer coupled to the chromatography column is a Single Quadripole<br>
device marketed by the company Agilent, with cone voltages at 30 and at 70 V.<br>
20.3	Analysis of the biosynthesis intermediates produced bv the strain OS49.67:<br>
The strain OS49.67 in which the or/3 gene is inactivated by an in-phase deletion does not produce spiramycins (cf. examples 6 and 15).<br>
A sample was prepared ..according to the method described above (cf. paragraph 20.1) and was analyzed by LC/MS as described above (cf. paragraph 20.2).<br><br>
More particularly, the analysis by chromatography was carried out in a solvent gradient with, as mobile phase: 20% of acetonitrile from time T=0 to 5min, then linear increase to 30% at T=35 minutes, followed by a plateau up to T=50 minutes.<br>
Under these conditions, two products were more particularly observed: an absorbent at 238 nm (retention time of 33.4 min) and an absorbent at 230 nn (retention time of 44.8 min) (cf. figure 35). Figure 35 shows the superposition ot the HPLC chromatograms produced at 238 and 280 nm (top) and also the UV spectra of the molecules eluted at 33.4 minutes and 44.8 minutes (bottom).<br>
The coupled mass spectrometry analysis conditions were as follows: the scanning is carried out in scan mode, covering a mass range of between 100 and 1000 Da. The gain of the electromultiplier was 1 V. As regards the electrospray source, the pressure of the nebulizing gas was 35 psig, the flow rate of the drying gas was 12.0 I.mm"1, the temperature of the drying gas was 350°C, and the capillary voltage was brought to +/-3000 V. These experiments made it possible to determine the mass of the two products separated. These masses are, respectively, 370 g/mol for the product eluted first ([M-H20]+=353 major product) and 368 g/mol for the second product ([M-H20]"=351 major product).<br>
In order to obtain the structure, the products mentioned above were isolated and purified under the following conditions: the mobile phase is a 70/30 (v/v) mixture of an aqueous 0.05% trifluoroacetic acid solution and acetonitrile. The chromatography is carried out in an isocratic system with a fixed flow rate of 1 ml/min. Under these conditions, the retention times of the 2 products are, respectively, 8 and 13.3 minutes. In addition, in this case, the sample prepared (cf. paragraph 20.1) is not diluted in water before injection of 10 fil.<br>
The 2 products are recovered at the chromatographic column outlet and isolated under the following conditions: an Oasis HLB 1  cc 30 mg cartridge (Waters) is<br>
conditioned sequentially with 1 ml of acetonitrile, then 1 ml of water/acetonitrile<br>
»<br>
(20v/80v) and l"ml of 80/20 water/acetonitrile. The sample is then introduced and the cartridge washed successively with 1 ml of water/acetonitrile (95/5) and 1 ml of water /<br><br>
deuterated acetonitrile (95/5), then eluted with 600 fi\ of 40/60 water, deuterated<br>
acetonitrile. The solution recovered is then directly analyzed by NMR.<br>
The NMR snectra obtained for these rwo compounds are as follows:<br>
-	First product eluted: Platenolide A: (spectrum 9646V)<br>
IH spectrum in CD3CN (chemical shifts in ppm): 0.90 (3H. t. J=6Hz), 0.93 (3H, d. J=5Hz), 1.27 (3H, c. J=5Hz), between 1.27 and 1.40 (3H, m). 1.51 (IH, m). 1.95 (IH. m), 2.12 (IH, m), 2.30 (IH, d, J=12Hz). 2.50 (IH, d, J=llHz). 2.58 (IH, dd, J=9 and 12 Hz), 2.96 (IH, d. J=7Hz), 3.43 (3H, s), 3.70 (IH, d, J=9Hz), 3.86 (IH. d, J=7Hz), 4.10 (IH, m), 5.08 (IH, m), 5.58 (IH, dt, J=3 and 12Hz), 5.70 (IH, dd, J=8 and 12 Hz), 6.05 (IH, dd, J=9 and 12 Hz), 6.24 (IH, dd, J=9 and 12Hz).<br>
-	Second product eluted: Platenolide B: (spectrum 9647V)<br>
IH spectrum in CD3CN (chemical shifts in ppm): 0.81 (3H, t, J=6Hz), 0.89 (IH, m), 1.17 (3H, d, J=5Hz), 1.30 (4H, m), 1.47 (2H, m), 1.61 (IH, t, J=10Hz), 2.20 (IH, m), 2.38 (IH, d, J=13Hz), 2.52 (IH, m), 2.58 (IH, m), 2.68 (IH, dd, J=8 and 13Hz), 3.10 (IH, d, J=7Hz), 3.50 (3H, s), 3.61 (IH, d, J=8Hz), 3.82 (IH, d, J=7Hz), 5.09 (IH, m), 6,20 (IH, m), 6.25 (IH, dd, J=9 and 12 Hz), 6.58 (IH, d, J=12 Hz), 7.19 (IH, dd, J=9 and 12Hz).<br>
These experiments thus made it possible to determine the structure of the these two compounds. The first product eluted is platenolide A and the second is platenolide B; the deduced structure of these two molecules is given in figure 36.<br>
It was also possible to determine, using the LC/MS technique combined with NMR, under conditions slightly different from those described above, but the setting up of which is well known to those skilled in the art, that the strain OS49.67 produces, in addition to platenolide A and B, a derivative of these two compounds. Tfiey are platenolide A-mycarose and platenolide  B-mycarose (the structure of these two<br><br>
compounds is given in figure 40). The results of analysis of the must of the strain OS49.67 are given in table 42.<br><br><br>
20.4 Analysis of the biosynthesis intermediates produced bv the strain SPM509:<br>
The strain SPM509 in which the orfl4 gene is inactivated {orfl4::ztt3fhac-) does not produce spiramycins (cf. examples 13, 15 and 16). A sample was prepared according to the method described above (cf. paragraph 20.1) and was analyzed by LC/MS as described above (cf. paragraphs 20.2 and 20.3). Analysis of the biosynthesis intermediates Dresent in the culture supernatant of the strain SPM509 cultured in MPS medium showed that this strain produced only form B of platenolide ("platenolide B'\ cf. figure 36) and not form A ("platenolide A'\ cf. figure 36).<br>
EXAMPLE 21: Interruption of the orfl4 gene in a strain with a knockout in the or/3 gene (OS49.67)<br>
The  product  of the  orfl4  gene   is   essential   to   spiramycin   biosynthesis (cf. examples 13, 15 and 16; the strain SPM509 in which this gene is interrupted no longer produces spiramycins). Analysis of the biosynthesis intermediates present in the culture supernatant of the strain SPM509 cultured in MPS medium showed that this strain produced only the form B of platenolide and not form A (cf. example 20). One of the hypotheses which may explain this observation is that the product of orfl4 is involved in the conversion of form B of platenolide to form A via an enzymatic reduction step. To test this hypothesis, the orf!4 gene was inactivated in a mutant no longer producing spiramycin, but producing forms A and B of platenolide. This is the case of the strain OS49.67 (cf. examples 6 and 20) in which the orf3 gene is inactivated by an in-phase deletion (Aor/3). To inactivate the orfl4 gene in this strain, the plasmid pSPM509 was introduced by protoplast transformation of the strain OS49.67 (T. Kieser et aL, 2000). The inactivation of the orfl4 gene has already been described in the case of the strain OSC2 (cf. example 13) and the same procedure was carried out for the inactivation of the orfl4 gene in the strain OS49.67. After transformation with the plasmid pSPM509, the clones were selected for their apramycin resistance.  The apramycin-resistant   clones   were   then   subcultured   respectively  on   medium   with<br><br>
apramycin and on medium with hygromycin. The clones resistant to apramycin (ApraR) and sensitive to hygromycin (HygS) are in principle those in which a double crossing over event has occurred and in which the orfl4 gene has been replaced with a copy of orfl4 interrupted with the att3{jaac- cassette. These clones were more particularly selected and the replacement of the wild-type copy of orfl4 with the copy interrupted with the cassette was verified by hybridization. Thus, the total DNA of the clones obtained was digested with several enzymes, separated on agarose gel, transferred onto a membrane and hybridized with a probe corresponding to the attZQaac- cassette in order to verify that the gene replacement had indeed taken place. The verification of the genotype can also be carried out by any method known to those skilled in the art, and in particular by PCR using the appropriate oligonucleotides and sequencing of the PCR product.<br>
A clone exhibiting the expected characteristics (Aorfi, orfl4:\att3Qaac-) was more particularly selected and named SPM510. It was in fact possible to verify, by virtue of the two hybridizations, that the att3Qaac- cassette was indeed present in the genome of this clone and that the digestion profile expected in the case of a replacement, subsequent to a double recombination event, of the wild-type orfl4 gene with the copy interrupted with the att3Daac- cassette in the genome of this clone is indeed obtained.<br>
EXAMPLE 22: Functional complementation of the interruption of the orfl4 gene<br>
22.1 Construction of the plasmid pSPM519:<br>
The orfl4 gene was amplified by PCR using the following pair of oligonucleotides: EDR31: 5' CCCAAGCTTCTGCGCCCGCGGGCGTGAA 3' (SEQ ID No. 136) and EDR37: 5' GCTCTAGAACCGTGTAGCCGCGCCCCGG 3' (SEQ ID No. 137) and, as matrix, the chromosomal DNA of the strain OSC2. The oligonucleotides   EDR31   and  EDR37   carry,  respectively,  the  HindUI  and Xbal<br><br>
restriction site (sequence in bold). The 1.2 kb fragment thus obtained was cloned into the vector pGEM-T easy (marketed by the company Promega (Madison, Wisconsin, USA)), to give the plasmid pSPM515. This plasmid was then digested with the HindUl and Xbal restriction enzymes. The 1.2 kb HindWXkal insert obtained was cloned into the vector pU\VL201 (cf. example 17.1) digested beforehand with the same enzymes. The plasmid thus obtained was named pSPM519.<br>
22.1 Transformation of the strains SPM509 and 5PM31Q with plasmid DSPM519:<br>
The plasmid pSPM519 was introduced into the strains SPM509 (cf. example 13) and SPM510 (cf. example 17) by protoplast transformation (T. Kieser ei al, 2000). After transformation, the clones were selected for their thiostrepton resistance. The clones were then subcultured on a medium containing thiostrepton.<br>
The strain SPM509 is a spiramycin nonproducer strain (cf. examples 13, 15 and 16 and figure 24). The spiramycin production of the strain SPM509 transformed with"the vector pSPM519 (strain named SPM509 pSPM519) was analyzed by culturing this strain in MPS medium in the presence of thiostrepton. The culture supematants were then analyzed by HPLC (cf. examples 16 and 17). The results of this analysis are given in table 43, the data are expressed in mg per liter of supernatant. The results correspond to the total production of spiramycins (obtained by adding the production of spiramycin I, II and EI). It was observed that the presence of the vector pSPM519 in the strain SPM509 restores the spiramycin production (cf table 43).<br>
Table 43. Spiramycin production of the strain SPM509 transformed with the vector pSPM519, (results expressed in mg/1 of supernatant).<br>
SPM509pSPM519<br><br>
The strain SPM510 transformed with the plasmid pSPM519 was named SPM510 pSPM509.<br>
EXAMPLE 23: Functional complementation of the interruption of the or/3 gene by the tylB gene of S.fradiae<br>
23.1 Construction of the plasmid pOS49.52:<br>
The plasmid pOS49.52 corresponds to a plasmid which allows expression of the TylB protein in S. ambofaciens. In order to construct it, the coding sequence of the tylB gene of S. fradiae (Merson-Davies &amp; Cundliffe, 1994, GenBank accession number: U08223 (sequence of the region), SFU08223 (DNA sequence) and AAA21342 (protein sequence)) was introduced in the plasmid pKCl218 (Bierman et al9 1992, Kieser et aL, 2000, a strain of E. coli containing this plasmid is accessible in particular from the ARS (NRRL) Agricultural Research Service Culture Collection) (Peoria, Illinois, USA), under the number B-14790). In addition, this coding sequence was placed under the control of the erm£* promoter (cf. above, in particular example 17.1, Bibb et aiy 1985, Bibb era/., 1994).<br>
23.1 Transformation of the strain OS49.67 with the plasmid pOS49.52:<br>
The strain OS49.67 in which the orf3 gene is inactivated by an in-phase deletion does not produce spiramycins (cf. examples 6 and 15). The plasmid pOS49.52 was introduced into the strain OS49.67 by protoplast transformation (T. Kieser et al9 2000). After transformation, the clones were selected for their apramycin resistance. The clones were then subcultured on a medium containing apramycin. A clone was more particularly selected and was named OS49.67 pOS49.52.<br>
As was demonstrated above, the strain OS49.67 does not produce spiramycins (cf; examples 6 and 15). The spiramycin production of the strain OS49.67 transformed with the vector pOS49.52 was analyzed by the technique described in example 15. It was thus possible to demonstrate that this strain has a spiramycin producer phenotype.<br><br>
Thus, the TylB protein allows functional complementation of the interruption of the cr/3 aene.<br>
EXAMPLE 24: Improvement of spiramycin production by overexpression of the orf28c gene<br>
24.1 Construction of the plasmid pSPM"::<br>
The orf28c gene was amplified by PCR using a pair of oligonucleotides comprising a HindTH restriction site or a BamHI restriction site. Tnese primers have the following sequence:<br>
KF30: 51 AAGCTTGTGTGCCCGGTGTACCTGGGGAGC 3* (SEQ ID No. 138) with a HindJE restriction site (which is shown in bold)<br>
KF31: 5' GGATCCCGCGACGGACACGACCGCCGCGCA 3' (SEQ ID No. 139) with the BamHI restriction site (which is shown in bold)<br>
The primers KF30 and KF31 carry, respectively, the HindEL and BamHI restriction sites (sequence in bold). The pair of primers KF30 and KF31 makes it possible to amplify a DNA fragment approximately 1.5 kb in size containing the orf28c gene, using as matrix the cosmid pSPM36 (cf. above). The 1.5 kb fragment thus obtained was cloned into the vector pGEM-T easy (marketed by the company Promega (Madison, Wisconsin, USA)) to give the plasmid pSPM74. The plasmid pSPM74 was then digested with the HindYR and BamHI restriction enzymes and the approximately 1.5 kb HindUUBarnlU insert obtained was subcloned into the vector pUWL201 (cf. example 17.1) digested beforehand with the same enzymes. The plasmid thus obtained was named pSPM75; it contains all the coding sequence of orf28c placed under the control of the ermE* promoter.<br>
24.2 Transformation of the strain OSC2 with the plasmid oSPM75:<br>
The plasmid pSPM75 was introduced into the strains OSC2 by protoplast transformation (T. Kieser et al, 2000). After protoplast transformation, the clones were selected for their thiostrepton resistance. The clones were then subcultured on a<br><br>
medium containing thiostrepton and the transformation with the plasmid was verified by plasmid extraction. Two clones were more particularly selected and named OSC2/pSPM75(l) and OSC2/pSPM75(2).<br>
A sample of the strain OSC2/pSPM75(2) was deposited with the Collection Nationale de Cultures de Microorganismes (CNCM) [National Collection of Cultures and Microorganisms] Pasteur Institute, 25, rue du Docteur Roux 75724 Paris Cedex 15, France, on October 5, 2003 under the registration number 1-3101.<br>
In order to test the effect of overexpression of the orf28c gene on spiramycin production, the spiramycin production of the OSC2/pSPM75(l) and OSC2/pSPM75(2) clones was tested by the technique described in example 15. Analysis of the spiramycin production of the strain OSC2 was also carried out in parallel for comparison. It was thus possible to observe that the presence of the plasmid pSPM75 significantly increases the spiramycin production of the strain OSC2. This demonstrates than overexpression of orf28c leads to an increase in spiramycin production and confirms its role as a regulator.<br>
The spiramycin production of the OSC2/pSPM75(l) and OSC2/pSPM75(2) clones was also analyzed by HPLC (in the same way as in example 17.2). Analysis of the spiramycin production of the strain OSC2 was also carried out in parallel for comparison. The results of this analysis are given in table 44, the data are expressed in mg per liter of supernatant. The results correspond to the total production of spiramycins (obtained by adding the production of spiramycin I, II and III).<br>
Table 44. Spiramycin production of the strains derived from OSC2 transformed with pSPM75, (results expressed in mg/1).<br><br><br>
Thus, it is observed that the presence of the plasmid pSPM75 significantly increases the spiramycin production of the strain OSC2. This clearly demonstrates that overexpression of or/25c has a positive effect on the spiramycin production.<br>
EXAMPLE    25:    Analysis   of   the   production    of   spiramycin    biosynthesis intermediates by a strain which has been inactivated in the orf8 gene:<br>
The strain OS49.107, in which the orfB gene is inactivated by insertion of the Qhyg cassette, does not produce spiramycins (cf. examples 7 and 15). The orf8 gene encodes a protein exhibiting relatively strong similarity with several aminotransferases and strongly suggests that the orf8 gene encodes a 4-aminotransferase responsible for the transamination reaction required for forosamine biosynthesis (cf. figure 6). It is therefore expected that spiramycin biosynthesis will be blocked at the forocidin stage (cf. figure 7). The strain OS49.107, which is a spiramycin non-producer, should therefore produce forocidin.<br>
A sample of supernatant of the strain OS49.107 was prepared according to the method described above (cf. example 16, without MIBK extraction) and was analyzed by LC/MS as described above (cf. paragraph 20.2 and 203). In SIM mode, the mass 553 relating to the molecular ion of forocidin was selected and several peaks were detected. The presence of compounds of mass 553 is compatible with the hypothesis of orf? having a role in forosamine synthesis.<br><br>
EXAMPLE   26:    Analysis   of   the   production   of   spiramycin   biosynthesis intermediates by a strain which has been inactivated in the orfl2 gene:<br>
The strain SPM507, in which the off 12 gene is inactivated, does not produce spiramycins (cf. examples 11 ar.c 15), The orfl2 gene is thought to encode a 3,4-dehydratase responsible for the dehydration reaction required for forosamine biosynthesis (cf. figure 6). It is therefore expected that the spiramycin biosynthesis will be blocked at the forocidin stage (cf. figure 7). The strain SPM507, which is a spiramycin non-producing strain, should therefore produce forocidin.<br>
A sample of supernatant of the strain SPM507 was prepared according to the method described above (cf. example 16, without MIBK extraction) and was analyzed by LC/MS as described above (cf. paragraphs 20.2 and 20.3). Under these conditions, the forocidin retention time is approximately 12.9 minutes. In SIM mode, the mass 558 relating to the molecular ion [M-H]* of forocidin was selected and a peak was detected. The presence of a compound at 558 makes it possible to validate the hypothesis that the product of orf!2 has a-roie in spiramycin biosynthesis.<br>
However, the forocidin is present in a relatively low amount and, under these conditions, a product which absorbs at 238 nm was more particularly observed (retention time of 17.1 min). The LC/MS analysis made it possible to determine the mass of this compound, which is 744.3 g/mol ([M+H]*= 744.3 major product).<br>
In order to obtain the structure, the products mentioned above were isolated and purified under the conditions described above (cf. paragraph 20.1). The organic phase (MIBK) is then recovered and evaporated. The dry extract is taken up with water and extracted with heptane, The aqueous solution is then extracted by binding to an Oasis HLB 1 g cartridge (Waters SAS, St-Quentin en-Yvelines, France). The compound is recovered by elution with a 30/70 water/acetonitrile mixture. This solution is then injected (100 fil) onto the analytical column and the fractions are recovered on an Oasis HLB 1 -cc 30 mg cartridge (Waters). Before use, the Oasis HLB 1 cc 30 mg cartridges (Waters) are conditioned sequentially with acetonitrile, and then a water/acetonitrile mixture (20v/80v) and an 80/20 water/acetonitrile mixture.<br><br>
The Oasis HLB 1 cc 30 mg cartridge (Waters) is then washed successively with I ml of water; acetonitrile (95/5) and I ml of water/deuterated acetonitriie (95/5), and ;hen eiuted with 600 fil of 40/60 water/deuterated acetonitrile. The recovered solution is :hen directly anaKiied by NMR.<br>
The NMR spectrum obtained for this compound is as follows (19312V NMR spectrum):<br>
1H spectrum in CD3CN/D20 (chemical shifts in ppm): 0.92 (3H, d, J=6Hz), 1.10 (1H, unresolved peak), 1.14 (3H, s), 1.17 (3H, d, J=6Hz), 1.22 (3H. d, J=6Hz), 1.25 (3H, d, J=6Hz), 1.40 (1H, unresolved peak), 1.75 (1H, dd, J=12 and 2Hz), 1.81 (1H. unresolved peak), 1.90 (1H, d, J-12Hz), 2.05 (1H, unresolved peak), 2.12 (3H, s), 2.15 (1H, unresolved peak), 2.35 (2H, unresolved peak), 2.45 (6H, broad s), 2.53 (1H, unresolved peak), 2.64 (1H, dd, J=12 and 9Hz), 2.80 (1H, dd. J=9 and 16Kz), 2.95 (1H. d, J=8Hz), 3.23 (2H, unresolved peak), 3.34 (1H, d, J=7Hz), 3.45 (3H, s), 3.49 (1£L unresolved peak), 3.93 (1H, dd, J=7 and 3Hz), 4.08 (1H, unresolved peak), 4.37 (1H, d, J=6Hz), 4.88 (1H, unresolved peak), 5.05 (2H, unresolved peak), 5.65 (2H, unresolved peak), 6.08 (1H, dd, J=8 and 12Hz), 6.40 (1H, dd, J=12 and 9Hz), 9.60 (1H, s).<br>
These experiments thus made it possible to determine the structure of this compound, this structure is given in figure 38.<br>
EXAMPLE   27:    Analysis   of   the   production   of   spiramycin    biosynthesis intermediates by a strain which has been inactivated in the orf5* gene:<br>
The strain SPM501 has the genotype orf6*::attI hyg-r. By virtue of the polar effect of the insertion of the attl hyg+ cassette into the orf 6* gene, it was possible to determine that the orf5* gene is essential to the spiramycin biosynthesis pathway. Specifically, insertion of the excisable cassette into the coding portion of the orf6* gene leads to complete arrest of spiramycin production by virtue of a polar effect on the expression of the orf5* gene. However, once the inserted cassette has been excised (and<br><br>
therefore when only the orf6* gene is inactivated, cf. examples 14 and 15), production of spiramycin I is restored. This demonstrates that the ort:* gene is essential to spiramycin biosynthesis since its inactivation leads to a complete arrest of spiramycin production.<br>
The orf5* gene encodes a protein exhibiting relatively strong similarity with several O-methyltransferases. The orfj* gene is thought to be an 0-methyltransferase involved in platenolide biosynthesis. To verify this hypothesis. LC/MS and NMR analytical experiments were carried out on a strain of S. ambofaciens of genotype orf6*::attl   hyg+, obtained from a strain which overproduces spiramycins.<br>
A sample of the supernatant of this strain was prepared according to the method described above (cf. example  16, without MIBK extraction) and was analyzed by LC/MS as described above (cf. paragraph 20.2 and 20.3). However, the column used is an X-Terra column (Waters SAS, St-Quentin en-Yvelines, France), and the cone voltage of the spectrometer is set at 380V so as to obtain fermentation of the compound analyzed.   Under   these   conditions,   a  product   for  which  the  retention  time   is approximately 13.1 minutes is observed. The mass spectrum of this compound has an appearance similar to that of spiramycin I, but the molecular ion is 829. The difference in mass of 14 compared with the mass of spiramycin can be explained by the absence of methyl group on the oxygen borne by carbon No. 4 of the lactone ring (the structure of this compound is given in figure 39). The presence of a compound at 829 makes it possible to validate the hypothesis that orf5* has a role in spiramycin biosynthesis. Using   a  microbiological   test   carried   out   on   a   sensitive   strain   of M.   luteus (cTexample 15 and figure 18)* it was demonstrated that the intermediate molecule (spiramycin minus methyl group, the structure of which is given in figure 39) produced by the strain orf6* y.Datthyg-  is much less active (by a factor of 10) than the spiramycin of origin with the methyl group at position 4.<br><br>
EXAMPLE 28 : Construction of new "excisable cassettes":<br>
New excisable cassettes were constructed. These cassettes are very similar to the excisable cassettes already described in example 9. The main difference between the former cassettes and the new cassettes is the absence, in the latter, of the sequences corresonding to the ends of the Q interposon, which sequences contain a transcription terminator originating from the T4 phage.<br>
In the cassettes without terminator, the gene which confers the resistance to an antibiotic is flanked by the attR and attL sequences which allow the excision. The resistance gene is the aac(3)IV gene which encodes an acetyitransferase which confers apramycin resistance. This gene is present in the ffaac cassette (GenBank accession number: X99313, Blondelet-Rouault, M.H. et ai, 1997) and was amplified by PCR using, as matrix, the plasmid pOSK1102 (cf. above) and, as primers, the oligonucleotides KF42 and KF43 each containing the HindUl restriction site (in bold) (AAGCTT) in the 5' position.<br>
KJ42:5^\AGCTTGTACGGCCCACAGAATGATGTCAC-3' (SEQ CD No. 153) and<br>
KF43:5t-AAGCTTCGACTACCTTGGTGATCTCGCCTT-3' (SEQIDNo. 154).<br>
The PCR product obtained, of approximately 1 kb, was cloned into the £. coli vector pGEMT Easy, producing the plasmid pSPM83.<br>
The vector pSPM83 was digested with the HindUl restriction enzyme. The HindUl-HindUl fragment of the insert was isolated by purification from a 0.8% agarose gel, and then cloned into the HindUl site located between the attL and attR sequences of the various plasmids carrying the various possible excisable cassettes (cf. example 9 and figure 27) so as to replace the HindUl fragment corresponding to ace with the HindUl fragment corresponding to the aac gene alone. This made it possible to obtain the art 1 aac, attlaac and att3aac cassettes (depending on the desired phase, cf. example 9). Depending on the orientation of the aac gene relative to the attL and<br><br>
attR sequences, attlaac+, attlaac-, att2aac+, attlaac-, att3aac+ and att3aac- are distinguished (according to the same conventions as those adopted in example 9),<br>
EXAMPLE 29: Construction of a strain of 5. ambofaciens with a knockout in the orf28c gene:<br>
The orf28c gene was inactivated using the excisable cassette technique. The excisable cassette attSaac* (cf. example 23) was amplified by PCR using, as matrix, the plasmid pSPMlOl (the plasmid pSPMlOl is a plasmid derived from the vector pGP704Not (Chaveroche et al.9 2000) (Miller VL &amp; Mekalanos JJ, 1988) in which the att3aac- cassette has been cloned as an EcoRV fragment into the unique EcoRV site of pGP704Not) and using the following primers: KF32:<br>
5TCAACCGCTTGAGCTGCTCCATCAACTGCTGGGCCGAGGTATCGCGCGCG CTTCGTTCGGGACGAA 3' (SEQ ID No. 155) andKF33:<br>
5TTGGGTCCCGCCGCGCGGCACGACTTCGACTCGCTCGTCTATCTGCCTCTT CGTCCCGAAGCAACT 3' (SEQ ID No. 156)<br>
The 39 nucleotides located at the 5' end of these oligonucleotides contain a sequence corresponding to a sequence in the orf28c gene and the 26 nucleotides located in the most 3' position (shown in bold and underlined above) correspond to the sequence of one of the ends of the excisable cassette att3aac+<br>
The PCR product thus obtained was used to transform the hyper-recombinant E. coli strain DY330 (Yu et aL, 2000) (this strain contains the exo, bet and gam genes of the lambda phage, integrated into its chromosome, these genes are expressed at 42°C, it was used in place of the E. coli strain KS272 (Chaveroche et ai, 2000)) containing the cosmid pSPM36. Thus, the bacteria were transformed by electroporation with this PCR product and the clones were selected for their resistance to apramycin. The cosmids of the clones obtained were extracted and digested with the BamHl restriction enzyme<br><br>
with the aim of verifying that the digestion profile obtained corresponded to the profile<br>
expected if insertion of the cassette (an3aac^-) into the orf2Sc gene had taken place, i.e. if there had indeed been homologous recombination between the ends of the PCR oroduct and the tarzet 2ene. The construct can also be verified bv anv method known to those skilled in the art, and in particular by PCR using the appropriate oligonucleotides and sequencing of the PCR product. A clone for which the cosmid has the expected profile was selected and the corresponding cosmid was called pSPM107. This cosmid is a derivative of pSPM36, in which orf28c is interrupted by the atiSaac- cassette. Insertion of the cassette is accompanied by a deletion in the orf2Sc gene, the interruption begins at the 28th codon of or£2Sc. After the cassette, there remains the last 137 codons of orGSc.<br>
The cosmid pSPM107 was, in a first step, introduced into the E. coli strain DH5a, and then into the Streptomyces ambofaciens strain OSC2 by protoplast transformation. After transformation, the clones were selected for their resistance to apramycin. The apramycin-resistant  clones  were  then   subcultured  respectively  on  medium  with apramycin (antibiotic B) and on medium with puromycin (antibiotic A) (cf. figure 9). The clones resistant to apramycin (ApraR) and sensitive to puromycin (PuroS) are, in principle, those in which a double crossing over event has occurred and which possess the orf28c gene interrupted by the att3aac-  cassette.  These clones  were more particularly selected and the replacement of the wild-type copy of orf28c with the copy interrupted by the cassette was verified by hybridization. Thus, the total DNA of the clones  obtained  was  digested  with  several  enzymes,  separated  on  agarose   gel, transferred onto a membrane and hybridized with a probe corresponding to the att3aacJr cassette so as to verify the presence of the cassette at the expected locus in the genomic DNA of the clones obtained. The genotype can also be verified by any method known to those skilled in the art, and in particular by PCR using the appropriate oligonucleotides and sequencing of the PCR product.<br>
A clone exhibiting the expected characteristics (orf28c::att3aac-) was more particularly selected and called  SPM107.  This  clone therefore has  the  genotype:<br><br>
orf28c::att3aac+ and was called SPM107. In view of the orientation of the genes (cf. figure 3), it is pointless to excise the cassette to study the effect of the inactivation<br>
of orf28c. The fact that or/29 is oriented in the opposite direction to orf28c shows that these genes are not cotranscribed. The use of an excisable cassette makes it possible, on the other hand, to have the possibility of being rid of the selection marker at any time, in particular by transformation with the plasmid pOSV508.<br>
In order to test the effect of turning off the orf28c gene on spiramycin production, the spiramycin production of the strain SPM107 was tested by the technique described in example 15. It was thus possible to demonstrate that this strain has a spiramycin non-producing phenotype. This demonstrates that the orf28c gene is a gene essential to spiramycin biosynthesis in S. ambofaciens.<br>
EXAMPLE 30: Construction of a strain of 5. ambofaciens with knockout in the orfil gene:<br>
The orfil gene was inactivated using the excisable cassette technique. The excisable cassette att3aac+ was amplified by PCR using, as matrix, the plasmid pSPMlOl, and the oligonucleotides EDR71 and EDR72. EDR71:<br>
5' CGTCATCGACGTGCGGGGAAGACAGAGGTGATACCGATGATCGCGCGC GCTTCGTTCGGGACGAA 3' (SEQ ID No 157)<br>
EDR72:<br>
S'GCCAGCACCTCGTCCAGCTGCTCGACGG.WCTCACCCCCATCTGCCTCT TCGTCCCGAAGCAACT 3' (SEQ ID No. 158)<br>
The 39 nucleotides located at the 5' end of these oligonucleotides contain a sequence corresponding to a sequence in the orfil gene and the 26 nucleotides located in the most 3' position (shown in bold and underlined above) correspond to the sequence of one of the ends of the excisable cassette att3aac~.<br><br>
The PCR product thus obtained was used to transform the E. coli strain KS272 containing the plasmid pKOBEG and the cosmic pSPM36, as described by Chaveroche et al. (Chaveroche et aL 2000) (cf. figure 12 for the principle, the plasmid pOS49.99 should be replaced with the cosmid pSPM36 and the plasmid obtained is no longer pSPMIT but pSPM543). Thus, the bacteria were transformed with this PCR product by electroporation and the clones were selected for their resistance to apramycin. The cosmids of the clones obtained were extracted and digested with several restriction enzvmes with the aim of verifvins that the disesrion nrofile obtained corresDonds to the profile expected if insertion of the cassette (attSaac-) into the or/31 gene has taken place, i.e. if there has indeed been homologous recombination between the ends of the PCR product and the target gene,. The construct can also be verified by any method known to those skilled in the art, and in particular by PCR using the appropriate oligonucleotides and sequencing of the PCR product. A clone for which the cosmid has the expected profile was selected and the corresponding cosmid was called pSPM543. This cosmid is a derivative of pSPM36, in which or/31 is interrupted by the att3aac+ cassette (cf. figure 12). The insertion of the cassette is accompanied by a deletion in the or/31 gene, the interruption begins at the thirty-sixth codon of or/31. After the cassette, there remain the last 33 codons of or/3L<br>
The cosmid pSPM543 was introduced into the Streptomyces ambofaciens strain OSC2 (cf. above) by protoplast transformation (Kieser, T et al., 2000). After transformation, the clones were selected for their resistance to apramycin. The apramycin-resistant clones were then subcultured respectively on medium with apramycin (antibiotic B) and on medium with puromycin (antibiotic A) (cf. figure 9). The clones resistant to apramycin (ApraR) and sensitive to puromycin (PuroS) are, in principle, those in which a double crossing over event has occurred and which possess the or/31 gene interrupted by the att3aac~ cassette. These clones were more particularly selected and the replacement of the wild-type copy of 5rf31 with the copy interrupted by the cassette was verified by hybridization. Thus, the total DNA of the clones   obtained   was  digested  with   several   enzymes,   separated   on   agarose   gel,<br><br>
transferred onto membrane and hybridized with a probe corresponding to the att3aac+ cassette so as to verify the presence of the cassette at the expected locus in the genomic DNA of the clones obtained. A second hybridization was carried out using, as probe, a DNA fragment obtained by PCR and corresponding to a very large portion of the coding sequence of the orfll gene.<br>
The genotype can also be verified by any method known to those skilled in the art, and in particular by PCR using the appropriate oligonucleotides and sequencing of the PCR product.<br>
A clone exhibiting the expected characteristics (or/3I::ait3aac~) was more particularly selected and called SPM543. It was in fact possible to verify', by virtue of the two hybridizations, that the att3aacJr cassette was indeed present in the genome of this clone and that the digestion profile expected in the case of a replacement, subsequent to a double recombination event, of the wild-type gene with the copy interrupted by the att3aac+ cassette in the genome of this clone was indeed obtained. This clone therefore has the genotype: orf3l::att3aacJr and was called SPM543. In view of the orientation of the genes (cf. figure 3), it is pointless to excise the cassette to study the effect of the inactivation of orf3l. The fact that orf32c is oriented in the opposite direction to or/31 shows that these genes are not cotranscribed. The use of an excisable cassette makes it possible, on the other hand, to have the possibility of being rid of the selection marker at any time, in particular by transformation with the plasmid pOSV508.<br>
In order to test the effect of turning off the orf31 gene on spiramycin production, the spiramycin production of the strain SPM543 was tested by the technique described in example 15. It was thus possible to demonstrate that this strain has a spiramycin non-producing phenotype. This demonstrates that the or/31 gene is a gene essential to spiramycin biosynthesis in S. ambofaciens.<br><br>
EXAMPLE 31: Construction of a strain of S. ambofaciens with a knockout in the orJ32c gene:<br>
The orf32c gene was inactivated using the excisable cassette technique. The excisable cassette att3aac- was amplified by ?CR using, as matrix, the plasmid pSPMlOl and using the following primers: KF52:<br>
S'GATCCGCCAGCCTCACGTCACGCCGCGCCGCCTCCCTGACATCGCGCGC GCTTCGTTCGGGACGAA 3f (SEP ID No 159). andKF53:<br>
5'GAGGCGGACGTCGGTACGCGGTGGGAGCCGGAGTTCGACAATCTGCCTC TTCGTCCCGAAGCAACT 3' (SEQ ID No. 160).<br>
The 40 nucleotides located at the 5' end of these oliaonucleotides contain a sequence corresponding to a sequence in the orf32c gene and the 26 nucleotides located in the most 3' position (shown in bold and underlined above) correspond to the sequence of one of the ends of the excisable cassette att3aac+.<br>
The PCR product thus obtained was used to transform the hyper-recombinant E. coli strain DY330 (Yu et al., 2000) containing the cosmid pSPM36. Thus, the bacteria were transformed with this PCR product by electroporation and the clones were selected for their resistance to apramycin. The cosmids of the clones obtained were extracted and digested with the BamHl restriction enzymes with the aim of verifying that the digestion profile obtained corresponds to the expected profile if insertion of the cassette (att3aac-r) into the orf32c gene has occurred, i.e. if there has indeed been homologous recombination between the ends of the PCR product and the target gene. The construct can also be verified by any method known to those skilled in the art, and in particular by PCR using the appropriate oligonucleotides and sequencing of the PCR<br>
to<br>
product. A clone.for which the cosmid has the expected profile was selected and the corresponding cosmid was called pSPM106. This cosmid is a derivative of pSPM36, in which orf32c is interrupted by the att3aac+ cassette. The insertion of the cassette is<br><br>
accompanied by deletion in the or/32c gene, the interruption begins at the 112th codon oforf32c. After the cassette, there remains the last 91 codons of or/3 2c.<br>
The cosmid pSPM106 was. in a first step, introduced into the E. coli srrain DH5a, and then into the Streptomyces ambofaciens strain OSC2 by transformation. After transformation, the clones are selected for their resistance to apramycin. The apramycin-resistant clones are then subcultured respectively on medium with apramycin (antibiotic B) and on medium with puromycin (antibiotic A) (cf. figure 9). The clones resistant to apramycin (ApraR) and sensitive to puromycin (PuroS) are, in principle, those in which a double crossing over event has occurred and which possess the orf32c gene interrupted by the att3aac+ cassette. These clones were more particularly selected and the replacement of the wild-type copy of orf32c with the copy interrupted by the cassette was verified by hybridization. Thus, the total DNA of the clones obtained was digested with several enzymes, separated on agarose gel, transferred onto membrane and hybridized with a probe corresponding to the att3aac+ cassette so as to verify the presence of the cassette in the genomic DNA of the clones obtained. The genotype can also be verified by any method known to those skilled in the art, and in particular by PCR using the appropriate oligonucleotides and sequencing ofthePCR product.<br>
A clone exhibiting the expected characteristics (or/32c::att3aac+) was more particularly selected. This clone therefore has the genotype: or/32c::att3aac+ and was called SPM106. In view of the orientation of the genes (cf. figure 3), it is pointless to excise the cassette to study the effect of the inactivation of the orf32c. The fact that orf33 is oriented in the opposite direction to orf32c shows that these genes are not cotranscribed. The use of an excisable cassette makes it possible, on the other hand, to have the possibility of being rid of the selection marker at any time, in particular by transformation with the plasmid pOSV508.<br>
In order to  test the effect of turning off the orf32c gene on spiramycin production,  the spiramycin production of the strain SPM106  was  tested by the<br><br>
technique described in example 15. It was thus possible to demonstrate that this strain has a spiramycin-producing phenotype. This demonstrates that the orf32c gene is not a gene essential to spiramycin biosynthesis in S. ambofaciens.<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
Depositing of biological material<br>
The following organisms were deposited with the Collection Nationale de Cultures<br>
de Microorganismes [National Collection of Cultures and Microorganisms] (CNCM), 2f<br>
rue du Docteur Roux, 75724 Paris Cedex 15, France, on July 10, 2002, according to th€<br>
provisions of the Treaty of Budapest.	_   .    	   ..._„.-...<br>
-	Strain OSC2 under the registration number 1-2908.	-...-...<br>
-	Strain SPM501 under the registration number 1-2909.	•	- Jr-tir<br>
-	Strain SPM502 under the registration number 1-2910.	'~ ^"r<br>
-	Strain SPM507 under the registration number 1-2911.   •	'.'-.-'r~«a<br>
-	Strain SPM508 under the registration number 1-2912.    :      "  * "-.	'^-i&amp;a&amp;ls<br>
-	Strain SPM509 under the registration number I-29t3"/~ ~* 'i"TT^- ^|SK<br>
-	Strain SPM21 imder the registration number 1-2914.    ^^ ^'issL ^^JISHB<br>
•	Strain SPM22 under the registration number 1-2915.	;	*?fw||<br>
-	Strain OS49.67 under the registration number 1-2916.   &gt; _  ,-	l'WSS<br>
-	Strain OS49.107 under the registration number 1-2917.- ; *   -  ^ -      ^^Sj|<br>
-	Escherichia  Coli strain DH5a containing the plasmid pOS44.4, undCT^ths<br>
registration number 1-2918.		y "        T'lSGl<br>
The following organisms were deposited with the Collection Nationategjfe Cultures de Microorganismes (CNCM), 25 rue du Docteur Roux, 75724 Paris £je&amp;£ 15, France, on February 26, 2003, according to the provisicmsjof^the Tie^^^E<br>
Budapest.	-	. .jSilS<br>
-	Strain SPM502 pSPM525 under the registration number 1-297"'   -    *      -*■*<br><br>
The following organisms were deposited with the Collection Nationale de Cultures de Microorganismes (CNCM), Pasteur Institute, 25 rue du Docteur Roux, 75724 Paris Cedex 15. France, on October 5, 2003, according to the provisions of :he Treaty of Budapest.<br>
Strain GSC2/pSPM75(2) under the registration number 1-3101.<br><br>
All the publications and patents cited are incorporated into the present application by<br>
way of reference.<br>
Bibliography:<br>
-	Altschul SF. Gish W? Miller W, Myers E\V, Lipman DJ. Basic local alignment search tool. JMolBiol (1990). 215 (3):403-4i0.<br>
-	.Altschul SF. Madden TL. Schaffer AA. Zhang J, Zhang Z, Miller \V, Lipman DJ. Gapped BLAST and PSI-3LAST: a new generation of protein database search programs. Nucleic Acids Res. (1997). 25 (17):3389-402.<br>
-	Amann,E., Ochs,B* and AbeLKJ. Tightly regulated tac promoter vectors useful for the expression of unfused and fused proteins in Escherichia coli Gene (1988) 69 (2), 301-315.<br>
-	Arisawa,A., Kawamura,N., Tsunekawa,H., Okamura,K., Tone,H. and Okamoto,R. Cloning and nucleotide sequences of two genes involved in the 4"-0-acyIation of macrolide antibiotics from Streptomyces thermotolerans. Biosci. Biotechnoi Biochem. (1993). 57 (12): 2020-2025. *<br>
-	Arisawa A, Kawamura N, Takeda K, Tsunekawa H, Okamura K, Okamoto R. Cloning of the macrolide antibiotic biosynthesis gene acyA, which encodes 3-0-acyltransferase, from Streptomyces thermotolerans and its use for direct fermentative production of a hybrid macrolide antibiotic. Appl Environ Microbiol. (1994).60(7):2657-2660.<br>
-	AugustJP.R., TangJL, Yoon,YJ., Ning,Streptomyces, Mueller,RM Yu,T.W., Tayior,M., Hoffrnann,D., Kim,C.G., Zhang,X., Hutchinson,C.R. and Floss,H.G. Biosynthesis of the ansamycin antibiotic rifamycin: deductions from the molecular analysis of the rif biosynthetic gene cluster of Amycolatopsis mediterranei S699. Chem. Biol (1998) 5 (2), 69-79.<br>
-	Ausubel Fred M., Brent Roger, Kingston Robert E., Moore David D., Seidman J.G., Smith John A., Struhl Kevin (Editeurs). Current Protocols in Molecular Biology, published by John Wiley &amp; Sons Inc. Current Protocols Customer Service, 605 Third Avenue, 9th Floor New York, NY 10158 USA (updated edition, March 2002).<br>
-	Baltz RH, McHenney NLA, Cantwell CA? Queener SW, Solenberg PJ. Applications<br>
of transposition mutagenesis in antibiotic producing streptomycetes. Antonie Van^   .<br>
Leeuwenhoek. (1997). 71 (1-2): 179-18".<br><br>
-	Bate N, Butler AR, Gandecha AR, Cundliffe E. Multiple regulatory genes in the   ,<br>
tylosin biosynthetic cluster of Streptomyces fradiae. Chem Biol. (1999). 6 (9): 6P-<br>
624.<br>
Bate,N., Bu:Ier,A.R., Smith.!.?, and Cundliffe.E. The mvcarose-biosvnthetic senes of Streptomyces fradiae, producer of tylcsin. Microbiology (2000). 146 (Pt 1), 139-146.<br>
Bayley C, Morgan, Dale E. C, Ow D. W. Exchange of gene activity in transgenic plants catalysed by the Cre-Iox site specific system. Plant Mol Biol (1992). 18 •353-361.<br>
-	Bentley,S.D.. Chater,K.F., Cerdeno-Tarraga,A.M., Challis,G.L., Thomson,N.R.,<br>
James,K.D.. Harris?D.E., Quail,M.A„ Kieser,H-T Harper,D., Bateman.A.,<br>
Brown,S., Chandra,G., Chen,C.W., Cotlins,M., Cronin.A., FrasenA., Goble,A.,<br>
HidalgoJ., Kornsby,T., Howarth,S., Huang,C.H., Kieser,T., Larke,L„ MurphyX.,<br>
OIiver,K., 0'Neil,S., Rabbinovvitsch.E., Rajandream,M.A., Rutherford,K.,<br>
Rutter,S., Seeger,K., Saunders,D., Sharp,S., Squares,R., Squares,S., Taylor,K.,<br>
Warren,T., Wietzorrek,A., Woodward J., BarrelLB.G., ParkhillJ. and<br>
Hopwood,D.A. Complete genome sequence of the model actinomycete<br>
Streptomyces coelicolor A3(2). Nature (2002). 417 (6885), 141-147.<br>
-	Bibb MJ, Findlay PR, Johnson MW. The relationship between base composition<br>
and codon usage in bacterial genes and its use for the simple and reliable<br>
identification of protein-coding sequences. Gene. (1984) 30 (1-3):157-166.<br>
Bibb MJ, Janssen GR, Ward JM. Cloning and analysis of the promoter region of the erythromycin resistance gene (ermE) of Streptomyces erythraeus. Gene. (1985). 38 (1-3): 215-26.<br>
-	Bibb MJ, White J, Ward JM, Janssen GR. The mRNA for the 23S rRNA<br>
methylase encoded by the ermE gene of Saccharopolyspora erythraea is translated<br>
in the absence of a conventional ribosome-binding site. Mol Microbiol. (1994). 14<br>
(3): 533-45.<br>
. - Bierman M, Logan R, O'Brien K, Seno ET, Rao RN, Schoner BE. Plasmid cloning vectors for the conjugal transfer of DNA from Escherichia coli to Streptomyces spp. Gene, 1992;116(l):43-9.<br>
-	Blondelet-Rouault M-H., Weiser J., Lebrihi A., Brannv P. and Pernodet J-L.<br>
.Antibiotic resistance gene cassettes derived from the Q interposon for use in E.<br>
coli and Streptomyces. Gene (1997) 190 : 315-117.<br>
-	Boccard F., Smokvina T., J.-L., Pemodet Fridmann A., and Guerineau M„<br>
Structural analysis of loci involved in pSAM2 site-specific integration in<br>
Streptomyces. Plasmid, (1989a). 21 : 59-70.<br><br>
-	Boccard F., Smokvina T- J.-L., Pernodet Fridmann A., and Guerineau M.. The<br>
integrated conjugative plasmid pSAM2 of Streptomyces ambofaciens is related to<br>
temperate bacteriophages. EMBOJ. (1989b) 8 : 973-980.<br>
Brunelli J.?., Pall MX.. A series of Yeast' E. coli lambda expression vectors designed for directional cloning of cDNA and cre/lox mediated plasmid excision. Yeast. (1992) 9: 1309-131S.<br>
Camilli A.. Beattie D. T. and Mekalanos J. J. Use of genetic recombination as a reporter of gene expression. Proc. Natl. Acad Sci. USA (1994) 91(7). 2634-2638.<br>
CampeIo,A.3. and GilJ.A. The candicidin gene cluster from Streptomyces griseus IMRU 3570. Microbiology (2002) 148 (Pt 1), 51-59.<br>
-	Carreras C, Frykman S, Ou S, Cadapan L, Zavala S, Woo E, Leaf T, Carney J,<br>
Burlingame M, Patei S, Ashley G, Licari P. Saccharopolyspora erythraea-catalyzed<br>
byconversion of 6-deoxyerythronolide B analogs for production of novel<br>
erythromycins. JBiotechnoL (2002). 92(3): 217-28.<br>
Chao K.-M., Pearson W.R., Miller W. Aligning two sequences within a specified diagonal band. Comput Appi Biosci. (1992) 8 : 481-487.<br>
Chater K. F. The improving prospects for yield increase by genetic engineering in antibiotic-producing Streptomycetes. Biotechnology. (1990). 8 (2) : 115-121.<br>
-	Chaveroche MK, Ghigo JM, d'Enfert C. A rapid method for efficient gene<br>
replacement in the filamentous fungus Aspergillus nidulans. Nucleic Acids Res.<br>
2000; 28(22) :E97.<br>
-	Church GM, Gilbert W. Genomic sequencing. Proc Natl Acad Sci USA. (1984) 81(7):1991-5.<br>
-	Churchward G, Belin D, Nagamine Y. A pSClOl-derived plasmid which shows no sequence homology to other commonly used cloning vectors. Gene. (1984) 31 (1-3): 165-71.<br>
-	Comstock,L.E., Coyne?M.J., Tzianabos.A.O. and Kasper,D.L. Interstrain variation of the polysaccharide B biosynthesis locus of Bacteroides fragilis: characterization of the region from strain 638RJ. Bacterial. (1999). 181 (19) : 6192-6196.<br>
Cox KL, Baltz RH. Restriction of bacteriophage plaque formation in Streptomyces spp. J Bacterial (1984) 159(2): 499-504.<br>
-	Dale E.C., Ow D.W., Gene transfer with subsequent removal of the selection gene<br>
from the host genome. Proc. Nati Acad Sci. USA (1991). 88 : 10558-10562.<br><br>
Doumith M, Weingarten P, Wehmeier UF, Salah-Bey K, Benhamou B, Capdevila C, Michel JM, Piepersberg W, Raynal MC. Analysis of genes involved in 6-deoxyhexose biosynthesis and transfer in Saccharopolyspora erythraea. Mol Gen Genet. (2000) 264(4): 477-485.<br>
■ Draeger,G., Park,Streptomyces-H.H. and Floss,H.G. Mechanism of the 2-deoxygenation step in the biosynthesis of the deoxyhexose moieties of the antibiotics granaticin and oleandomycin J. Am. Chem. Soc. (1999) 121 : 2611-2612.<br>
-	Gandecha,A.R., Large,S-L- and Cundliffe,E. Analysis of four tylosin biosynthetic genes from the tylLM region of the Streptomyces fradiae genome. Gene. (1997). 1*4 (2): 197-203.<br>
-	Geistlich,M., LosickJL, TurnerJ.R. and Rao,R.N. Characterization of a novel regulatory gene governing the expression of a polyketide synthase gene in Streptomyces ambofaciens. Mol Microbiol (1992). 6(14): 2019-2029.<br>
-	Gourmelen A, Blondelet-Rouault MH, Pemodet JL. Characterization of a glycosyl transferase inactivating macrolides, encoded by gimA from Streptomyces ambofaciens. Antimicrob Agents Chemother. (1998). 42(10): 2612-9.<br>
-	Huang X. and Miller W. A time-efficient, linear-space local similarity algorithm. Adv. Appl Math. (1991). 12 : 337-357.<br>
-	Hara O and Hutchinson CR. Cloning of midecamycin(MLS)-resistance genes from Streptomyces mycarofaciens, Streptomyces lividans and Streptomyces coelicolor A3(2). JAntibiot (Tokyo). (1990). 43 (8):977-991.<br>
-	Hara O and Hutchinson CR. A macrolide 3-O-acyltransferase gene from the midecamycin-producing species Streptomyces mycarofaciens. J Bacteriol (1992). 174(15) :5141-5144.<br>
-	Hoffineister D, Ichinose K, Domann S, Faust B, Trefzer A, Drager G, Kirschning A, Fischer C, Kunzel E, Bearden D, Rohr J and Bechthold A. The NDP-sugar co-substrate concentration and the enzyme expression level influence the substrate specificity of glycosyltransferases: cloning and characterization of deoxysugar biosynthetic genes of the urdamycin biosynthetic gene cluster. Chem Biol (2000). 7(11):821-831.<br>
-	Hopwood, D.A. Future possibilities for the diskovery of new antibiotics by genetic<br>
*    engineering.   Li Beta-lactam antibiotics.  Edited  by M.R.J.   Salton   and  G.D.<br>
Shockman. New York: Academic Press. (1981). 585-598.<br><br>
-	Hopwood D. A., Malpartida F., Kieser H. M., Dceda H., Duncan J., Fujii L, Rudd A. M., Floss H. G. and Omura S. Production of 'hybrid' antibiotics by genetic engineering. Nature. (1985a). 314 (6012): 642-644.<br>
-	Hopwood D. A., Malpartida F., Kieser H. M., Dceda H. and Omura S. (1985b) In Microbiology (ed S. Silver). American Society for Microbiology, Washington D. C, 409-413.<br>
-	Houben Weyl, 1974, in Methode der Organischen Chernie [Methods in Organic Chemistry], E. Wunsch Ed., Volume 15-1 and 15-11,<br>
-	Hutchinson, C.R. Prospects for the diskovery of new (hybrid) antibiotics by genetic engineering of antibiotic-producing bacteria. Med Res Rev. (1988). 8 (4):<br>
557-567.<br>
-	Hutchinson C R., Borell C. W., Otten S. L., Stutzman-Engwall K. J. and Wang Y. Drug diskovery and development through the genetic engineering of antibiotic-producing microorganisms J. Med, Chem. (1989) 32 (5): 929-937.<br>
-	Ishikawa J, Hotta K. FramePlot: a new implementation of the frame analysis for predicting protein-coding regions in bacterial DNA with a high G + C content. FEMS Microbiol Lett. (1999) 174(2):251-3.<br>
-	Kieser, T, Bibb-, MJ, Buttner MJ, Chater KF, Hopwood DA. Practical Streptomyces Genetics. 2000. The John Innes Foundation, Norwich UK.<br>
-	Kuhstoss and ah Production of a novel polyketide through the construction of a hybrid polyketide synthase. Gene. (1996). 183(1-2): 231-236.<br><br>
-	Lacalle RA, Pulido D, Vara J, Zalacain M, Jimenez A. Molecular analysis of the pac gene encoding a puromycin N-acetyl transferase from Streptomyces alboniger. Gene. (1989). 79(2):375-380.<br>
-	Lakso M, Sauer B, Mosinger B Jr, Lee EJ, Manning RW, Yu SH, Mulder KL, Westphal H. Targeted oncogene activation by site-specific recombination in transgenic mice. Proc Natl Aca d Sci USA. (1992)89(14) :6232-6.<br>
-	Li,T.B., Shang,G.D., Xia,H.Z. and Wang,Y.G. Cloning of the sugar related biosynthesis gene cluster from Streptomyces tenebrarius H6. Sheng Wu- Gong ChengXueBao (2001). 17 (3): 329-331.<br>
-	LigonJ., Hill,S., BeckJ., Zirkle,R., MolnarJ., ZawodnyJ., Money,S. and Schupp,T. Charactenzation of the. biosynthetic gene cluster for the antifungal polyketide soraphen A from Sorangium cellulosum So ce26. Gene (2002). 285 (1-2), 257-267.<br><br>
-	Liu L, Saevels J, Louis P, Nelis H, Rico S, Dierick K, Guyomard S, Roets E, Hoogmartens J. Interlaboratory study comparing the microbiological potency of spiramycins I, II and EL JPharm Biomed Anal (1999). 20 (l-2):217-24.<br>
-	Mazodier P, Petter R, Thompson C. Intergeneric conjugation between Escherichia coli and Streptomyces species J BacterioL (1989). 171 (6):3583-5.<br>
-	Merrifield RB, 1965a, Nature, 207(996): 522-523.<br>
-	Merrifield RB., 1965b, Science, 150(693): 178-185.<br>
-	Merson-Davies,L.A. and CundliffeJE. Analysis of five tylosin biosynthetic genes from the tyllBA region of the Streptomyces fradiae genome, Molecular microbiology. (1994). 13 (2): 349-355.<br>
-	Miller VL, Mekalanos JJ. A novel suicide vector and its use in construction of insertion mutations: osmoregulation of outer membrane proteins and virulence determinants in Vibrio cholerae requires toxR. J BacterioL (1988) 170(6):2575-83.<br>
-	Nakano, Y., Yoshida, Y,, Yamashita, Y. &amp; Koga, T. Construction of a series of pACYC-derived plasmid vectors. Gene (1995). 162 :157-8.<br>
-	Nielsen H., Engelbrecht J., Brunak S. et von Heijne G. Identification of prokaryotic and eukaryotk signal peptides and prediction of their "cleavage sites. Protein Engineering. (1997). 10: 1-6.<br>
-	Oh SH, Chater KF. Denaturation of circular or linear DNA facilitates targeted integrative transformation of Streptomyces coelicolor A3(2): possible relevance to other organisms. J BacterioL 1997; 179(l):122-7.<br><br>
-	Olano C, Lomovskaya N, Fonstein L, Roll JT, Hutchinson CR. A two-plasmid system for the glycosylation of polyketide antibiotics: byconversion of epsilon-rhodomycinone to rhodomycin D. Chem Biol (1999). 6 (12):845-55.<br>
-	Omura S, Kitao C, Hamada H, Dceda H. Byconversion and biosynthesis of 16-membered macrolide antibiotics. X. Final steps in the biosynthesis of spiramycin, using enzyme inhibitor:' cerulenin. Chem Pharm Bull (Tokyo). (1979a). 27(1): 176-82.<br>
-	Omura S, Dceda H, Kitao C. Isolation and properties of spiramycin I 3-hydroxyl acylase from Streptomyces and ambofaciens. J Biochem (Tokyo). (1979b). 86(6):<br>
1753-8.    .   .<br>
-	Omura,S., DcedaJL, IshikawaJ., Hanamoto,A., Takahashi,C, Shinose,M.,<br>
Takahashi,Y., Horikawa,H., Nakazawa,H., Osonoe,T., Kikuchi,H., Shiba,T.,<br>
Sakaki,Y. and Hattori,M. Genome sequence of an industrial microorganism<br><br>
Streptomyces avermitilis: Deducing the ability of producing secondary metabolites. Proc. Natl Acad. Sci. U.SA. (2001). 98 (21), 12215-12220.<br>
-	Pearson W.R. and D. J. Lipman. Improved Tools for Biological Sequence Analysis. Proc. Natl. Acad. Sci. USA 1988, 85: 2444-2448.<br>
-	Pearson W. R. Rapid and Sensitive Sequence Comparison with FASTP and FAST A. Methods in Enzymology. 1990,183: 63-98.<br>
-	Pemodet JL, Simonet JM, Guerineau M. Plasmids in different strains of Streptomyces ambofaciens: free and integrated form of plasmid pSAM2. Mol Gen Genet. (1984);198 (1):35-41.<br>
-	Pernodet JL, Alegre MT, Blondelet-Rouault MH, Guerineau M. Resistance to spiramycin in Streptomyces ambofaciens, the producer organism, involves at least two different mechanisms. J Gen Microbiol. (1993) ; 139 ( Pt 5):10O3-l 1.<br>
-	Pernodet JL, Fish S, Blondelet-Rouault MH, Cundliffe E. The macrolide-lincosamide-streptogramin B resistance phenotypes characterized by using a specifically deleted, antibiotic-sensitive strain of Streptomyces lividans. Antimicrob Agents Chemother. (1996), 40 (3): 581-5.<br>
-	Pernodet JL, Gourmelen A, Blondelet-Rouault MH, Cundliffe E. Dispensable ribosomal resistance to spiramycin conferred by sntiA in the spiramycin producer Streptomyces ambofaciens. Microbiology. (1999), 145 (Pt 9):2355-64.<br>
-	Pfoestl^A., Hofinger,A., Kosma,P. and Messner,P. Biosynthesis of dTDP-3-acetamido-3,6-dideoxy-alpha-D-galactose in Aneurinibacillus thermoaerophilus L420-91T. J. Biol Chem. (2003), 278 (29): 26410-26417.<br><br>
-	Rao RN, Richardson MA, Kuhstoss S. Cosmid shuttle vectors for cloning and analysis of Streptomyces DNA. Methods Enzymol (1987); 153:166-98.<br>
-	Raynal A, Tuphile K, Gerbaud C, Luther T, Guerineau M, and Pemodet JL.. Structure of the chromosomal insertion site for pSAM2: functional analysis in Escherichia coli. Molecular Microbiology (1998) 28 (2) 333-342.<br>
-	Redenbach,M., Kieser,H.M., Denapaite,D., Eichner,A., CullumJ., KinashiJL and Hopwood,D.A. A set of ordered cosmids and a detailed genetic and physical map for the 8 Mb Streptomyces coelicolor A3(2) chromosome Mol Microbiol. (1996).<br>
21(1): 77-96.<br>
-	Richardson MA, Kuhstoss S, Solenberg P, Schaus NA, Rao RN. A" new shuttle<br>
cosmid vector, pKC505, for strep tomycetes: its use in the cloning of three different<br>
spiramycin-resistance genes from a Streptomyces ambofaciens library. Gene.<br>
(1987);61(3):231-41.<br><br>
-	Robinson, LA. Enzymes of Secondary Metabolism in Microorganisms. Chem Soc Rev. (1988). 17 :383-452.<br>
-	Russell SH„ Hoopes JL, Odell JT. Directed excision of a transgene from the plant genome. Mol Gen Genet (1992). 234 (1): 49-59.<br>
-	Sambrook J, Frisch EF, Maniatis T . Molecular Cloning : a laboratory manual 2nd ed. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York USA (1989).<br>
-	Schoner B, Geistlich M, Rosteck P Jr, Rao RN, Seno E, Reynolds P, Cox K, Burgett S and Hershberger C. Sequence similarity between macrolide-resistance determinants and ATP-binding transport proteins. Gene. (1992). 115 (1-2), 93-96.<br>
-	Sezonov G, Hagege J, Pernodet JL, Friedmann A and Guerineau M. Characterization of pra, a gene for replication control in pSAM2, the integrating element of Streptomyces ambofaciens. Mol Microbiol. (1995). 17(3): 533-44.<br>
Sezonov G., Blanc V., Bamas-Jacques N., Friedmann A., Pernodet J.L. and M. Guerineau, Complete conversion of antibiotic precursor to pristinamycin HA by over expression of Streptomyces pristinaespiralis biosynthetic genes, Nature Biotechnology. (1997) 15 : 349-353.<br>
-	Sezonov G, Possoz C, Friedmann A, Pernodet JL, Guerineau M. KorSA from the<br>
Streptomyces integrative element pSAM2 is a central transcriptional repressor:<br>
target genes and binding sites. JBacterioL (2000). 182(5):1243-50.<br>
-	Simon R., Priefer U and Puhler. A broad host range mobilisation system for in vivo genetic engeneering: transposon mutagenesis in gram negative bacteria. Bio/Technology (1983). 1 : 784-791.<br>
-	Simon, and a/., Plasmid vectors for the genetic analysis and manipulation of rhizobia and other gram-negative bacteria, p. 640-659. In A. Weissbach, and H. Weissbach (eds.), Methods in enzymology, vol 118, Academic Press, Inc., Orlando, 1986<br>
-	Summers,R.G., Donadio,Streptomyces, Staver,MJ., Wendt-Pienkowski,E., Hutchinson,C.R. and Katz,L. Sequencing and mutagenesis of genes from the erythromycin biosynthetic gene cluster of Saccharopolyspora erythraea that are involved in L-mycarose and D-desosamine production. Microbiology (1997). 143 (Pt 10): 3251-3262.<br>
*	*  -<br>
-	Van Mellaert L, Mei L, Lammertyn E, Schacht S, Anne J. Site-specific integration<br>
of bacteriophage VWB genome into Streptomyces venezuelae and construction of a VWB-based integrative vector. Microbiology. (1998). 144 (Pt 12): 3351-8.<br><br>
-	Vara J, Lewandowska-Skarbek M, Wang YG, Donadio S, Hutchinson CR.Cloning<br>
of genes governing the deoxysugar portion of the erythromycin biosynthesis<br>
pathway in Saccharopolyspora erythraea (Streptomyces erythreus). J Bacteriol.<br>
(1989). 171(11): 5872-81.<br>
Vara J, Maipartida F, Hopwood DA. Jimenez A. Cloning and expression of a puromycin N-acetyi transferase gene from Streptomyces alboniger in Streptomyces lividans and Escherichia coli. Gene. (1985). 33(2): 197-206.<br>
-	Wahl, G. M., K. A. Lewis, J. C. Ruiz. B. Rothenberg, J. Zhao, and G. A. Evans. Cosmid vec:ors for rapid genomic walking, restriction mapping, and gene transfer. Proc. Natl. Acad, Sci. USA (1987). 84: 2160-2164.<br>
-	Waldron,C. MatsushimaJ?., Rosteck,P.R., Broughton,M.C, Turner J,, Madduri,K., CrawfordiCP., Merlo,D.J. and Baltz,R.H. Cloning and analysis of the spinosad biosynthetic gene cluster of Saccharopolyspora spinosa(l) Chem. Biol (2001) 8 (5): 487-499.	~   .      '<br>
-	Walczak RJ, Hines JV, Strohl \VR, Priestley ND.Bioconversion of the anthracycline analogue desacetyladriamycin by recombinant DoxA, a P450-monooxygenase from Streptomyces sp. strain C5. Org Lett. (2001). 3 (15):2277-9.<br>
-	Wang,Z.X., Li,S.M. and Heide,L. Identification of the coumermycin A(l) biosynthetic gene cluster of Streptomyces rishiriensis DSM 40489. Antimicrob. Agents Chemother. (2000) 44 (11), 3040-3048.<br><br>
-	Ward, J.M., Janssen, G.R., Kieser, T, Bibb, M.J., Buttner, MJ. &amp; Bibb, MJ. Construction and characterization of a series of multi-copy promoter-probe plasmid vectors for Streptomyces using the aminoglycoside phosphotransferase from Tn5 as indicator. Mol Gen Genet (1986). 203 : 468-478.<br>
-	Wehmeier UF. New multifunctional Escherichia coli-Streptomyces shuttle vectors allowing blue-white screening on XGal plates. Gene. (1995). 165(1): 149-150.<br>
-	Wilms B, Hauck A, Reuss M, Syldatk C, Mattes R, Siemann M, Altenbuchner J. High-cell-density fermentation for production of L-N-carbamoylase using an expression system based on the Escherichia coli rhaBAD promoter. Biotechnol Bioeng. (2001). 73(2) :95-103.<br>
-	Worley K. C, Wiese B. A., and Smith R. F. BEAUTY: An enhanced BLAST-based search tool that integrates multinle biolosical information resources into sequence similarity search results. Genome Research (1995). 5: 173-184.<br>
-	Wu K, Chung L, Revill WP, Katz L and Reeves CD. Tne FK520 gene cluster of Streptomyces hygroscopicus var. ascomyceticus (ATCC 14891) contains genes for biosynthesis of unusual polyketide extender units. Gene. (2000). 251 (1): 81-90.<br><br>
-	Xue Y, Zhao L, Liu HW and Sherman DK. A eene cluster for macrolide antibiotic biosynthesis in Streptomvces venezuelae: architecture of metabolic diversity. Proc Natl Acad Sci U S A. (1998). 95(21): 12111-12116.<br>
-	Yu D, Ellis HM, Lee EC, Jenkins NA. Copeland NG et Court DL. An efficient recombination system for chromosome engineering in Escherichia coli. Proc Sari AcadSci U S A.'(2000). 97(11): 597S-83.<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
CLAIMS<br>
1.   A   polynucleotide   encoding   a   polypeptide   involved   ;n   spiramycin biosynthesis, wherein the sequence of said polynucleotide is:<br>
(a)	one of the sequences SEQ ID No. 3.5, 7,9, 11, 13, 15, 17, 19.21,23,25,23, 30, 34, 36, 40, 43. 45, 47, 49, 53, 60. 62, 64, 66, 68, 70, 72, 74. 76, 78, SO, 82, 84, 107, 109, 111, 113, 115, 118, 120, 141, 143, 145, 147 and 149,<br>
(b)	one of the sequences consisting of the variants of the sequences (a),<br>
(c)	one of the sequences derived from the sequences (a) and (b) due to the<br>
degeneracy of the genetic code.<br>
2.	A polynucleotide which hybridizes, under high stringency conditions, with at least one of the polynucleotides as claimed in claim 1.<br>
3.	A polynucleotide exhibiting at least 70%, 80%, 85%, 90%, 95% or 98% nucleotide-identity with a polynucleotide comprising at least 10, 12, 15, 18, 20 to 25, 30, 40, 50, 60, 70, 80, 90, 100, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900, 950, 1000, 1050, 1100, 1150, 1200, 1250, 1300, 1350, 1400, 1450, 1500, 1550, 1600, 1650, 1700, 1750, 1800, 1850 or 1900 consecutive nucleotides of a polynucleotide as claimed in claim 1.<br>
4.	The polynucleotide as claimed in claim 2 or 3, which is isolated from a bacterium of the genus Streptomyces.<br>
5.	The polynucleotide as claimed in 2. 3 or 4, which encodes a protein involved in the biosynthesis of a macrolide.<br>
6.	The polynucleotide as claimed in claim 2, 3, 4 or 5, which encodes a protein having an activity similar to the protein encoded by the polynucleotide with which it hybridizes or with which it exhibits the identity.<br><br>
7.	A polypeptide resulting from the expression of a polynucleotide as claimed in claim 1, 2, 3, 4, 5 or 6.<br>
8.	A polypeptide involved in spiramycin biosynthesis, wherein :he sequence oi said polypeptide is:<br><br>
(a)	one of the sequences SEQ ID. No. 4. 6. 8. 10, 12. 14, 16, IS. 20, 22, 24, 26, 27, 29, 31, 32, 33, 35, 37, 33, 39, 41. 42, 44. 46, 48. 50, 51. 52, 54, 55. 56, 57, 58, 59, 61,63,65,67,69.71,73,75.77,79.81,83.85, 108, 110, 112, 114. 116, 117, 119, 121, 142, 144, 146, 148 and 150,<br>
(b)	one of the sequences as denned in (a), except that throughout said sequence, one or more amino acids have been substituted, inserted or deleted without affecting the functional properties thereof,<br>
(c)	one of the sequences consisting of the variants of the sequences (a) and (b).<br>
9.	A poKpeptide exhibiting at least 70%, 80%, 85%, 90%, 95% or 98% amino<br>
acid identity with a polypeptide comprising at least 10, 15, 20, 30 to 40, 50, 60, 70, 80,<br>
90, 100, 120, 140, 160, 180, 200, 220, 240, 260, 280, 300, 320, 340, 360, 380, 400, 420,<br>
440, 460, 480, 500, 520, 540, 560, 580, 600, 620 or 640 consecutive amino acids of a<br>
polypeptide as claimed in claim 8.<br>
10.	The polypeptide as claimed in claim 9, which is isolated from a bacterium of the genus Streptomyces.<br>
11.	The polypeptide as claimed in claim 9 or 10, which encodes a protein involved in the biosynthesis of a macrolide.<br>
12.	The polypeptide as claimed in claim 9, 10 or 11, which has an activity similar to that of the polypeptide with which it shares the identity.<br>
13.	A recombinant DNA, which comprises at least one polynucleotide as claimed	,   . in one of claims 1, 2, 3, 4, 5 and 6.<br><br>
14.	The recombinant DNA as claimed in claim 13, wherein said recombinant<br>
DNA is included in a vector.<br>
15.	The recombinant DNA as claimed in claim 14, wherein said vector is chosen<br>
from bacteriophages, plasmids, phagemids, integrative vectors, fosmids, cosmids,<br>
shuttle vectors, BACs and PACs.<br><br>
17.	An expression vector, which comprises at least one nucleic acid sequence<br>
encoding a polypeptide as claimed in claim 7, 8,9,10,11 or 12.<br>
18.	An expression system comprising a suitable expression vector and a host cell<br>
which allows the expression of one or more polypeptides as claimed in claim 7, 8, 9,<br>
10,11 or 12.<br>
19.	The expression system as claimed in claim 18, which is chosen from<br>
prokaryotic expression systems and eukaryotic expression systems.<br>
20.	The expression system as claimed in claim 19, which is chosen from systems<br>
of expression in the bacterium E. coli, baculovirus expression systems which allow<br>
expression in insect cells, expression systems which allow expression in yeast cells and<br>
expression systems which allow expression in mammalian cells.<br><br>
21.	A host cell into which at least one polynucleotide and/or at least one recombinant DNA and/or at least one expression vector as claimed in one of claims 1, 2, 3,4, 5,6,13, 14,15, 16 and 17 has been introduced.<br>
22.	A method of producing a polypeptide as claimed in claim 7, 8, 9, 10, 11 or 12, wherein said method comprises the following steps:<br><br>
a)	inserting at least one nucleic acid encoding said polypeptide into a suitable vector;<br>
b)	culturing, in a suitable culture medium, a host cell transformed or transfected beforehand with the vector of step a);<br>
c)	recovering the conditioned culture medium or a cell extract;<br>
d)	separating and purifying said polypeptide from said culture medium, or else from the cell extract obtained in step c);<br>
e)	where appropriate, characterizing the recombinant polypeptide produced.<br><br>
23.	A microorganism blocked in a step of the biosynthetic pathway for at least one macrolide.<br>
24.	The microorganism as claimed in claim 23, which is obtained by inactivating the function of at least one protein involved in the biosynthesis of this (these) macrolide(s) in a microorganism which produces this (these) macrolide(s).<br>
25.	A microorganism as claimed in 24, wherein the inactivation of this (these) protein(s) is carried out by mutagenesis in the gene(s) encoding said protein(s) or by expression of one or more antisense RNAs complementary to the messenger RNA(s) encoding said protein(s).<br>
26.	The microorganism as claimed in claim 25, wherein the inactivation of this (these) protein(s) is carried out by mutagenesis via irradiation, by action of a mutagenic chemical agent, by site-directed mutagenesis or by gene replacement.<br>
27.	The microorganism as claimed in claim 25 or 26, wherein the mutagenesis or<br>
mutageneses is or are carried out in vitro or in situ, by suppression, substitution,<br><br>
deletion and/or addition of one or more bases in the gene(s) under consideration or by gene inactivation.<br>
28.	The microorganism as claimed in claim 23, 24, 25, 26 or 27, wherein said microorganism is a bacterium of the genus Streptomyces.<br>
29.	The microorganism as claimed in claim 2J, 24, 25, 26, 27 or 28, wherein the macro lide is spiramycin.<br>
30.	The microorganism as claimed in claim 23, 24, 25, 26, 27, 28 or 29, wherein said microorganism is a strain of S. ambofaciens.<br>
31.	The microorganism as claimed in claim 23, 24, 25, 26, 27, 28, 29 or 30, wherein the mutagenesis is carried out in at least one gene comprising a sequence as claimed in one of claims 1,2,3,4, 5 and 6.<br>
32.	The microorganism as claimed in claim 25, 26, 27, 28, 29, 30 or 31, wherein the mutagenesis is carried out in one or more genes comprising one of the sequences corresponding to one or more of the sequences SEQ ID No. 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 25, 28, 30, 34, 36, 40, 43, 45,47,49, 53, 60, 62, 64, 66, 68, 70, 72, 74, 76, 78, 80, 82, 84,107,109, 111, 113,115,118,120,141,143, 145,147 and 149.<br><br>
33.	The microorganism as claimed in claim 25, 26, 27, 28, 29, 30, 31 or 32, wherein the mutagenesis consists of the gene inactivation of a gene comprising a sequence corresponding to the sequence SEQ ID No. 13.<br>
34.	A strain of Streptomyces ambofaciens, which is a strain chosen from:<br>
-  one  of the  strains  deposited  with the  Collection  Nationale   de  Cultures   de Microorganismes (CNCM) [National Collection of Cultures and Microorganisms] on July 10, 2002, under the registration number 1-2909,1-2911,1-2912, 1-2913,1-2914, I-2915,1-2916 or 1-2917 - the strain SPM510.<br><br>
35.	A method of preparing a macrolide biosynthesis intermediate, which<br>
comprises the following steps:<br>
a)	culturing, in a suitable culture medium, a microorganism as claimed in one of claims 23, 24, 25, 26, 27, 23, 29, 30, 31, 32, 33 or 34,<br>
b)	recovering the conditioned culture medium or a cell extract,<br>
c)	separating and purifying said biosynthesis intermediate from said culture<br>
medium or else from the cell extract obtained in step b).<br>
36.	A method of preparing a molecule derived from a macrolide, wherein a biosynthesis intermediate is prepared according to the method of claim 35, and the intermediate thus produced is modified.<br>
37.	The method of preparation as claimed in claim 36, wherein said intermediate is modified chemically, biochemically, enzymatically and/or microbiologically.<br><br>
38.	The method of preparation as claimed in claim 36 or 37, wherein one or more genes encoding proteins capable of modifying "the intermediate by using it as substrate is (are) introduced into said microorganism.<br>
39.	The method of preparation as claimed in claim 36, 37 or 38, wherein the macrolide is spiramycin.<br>
40.	The method of preparation as claimed in claim 36, 37, 38 or 39, wherein the microorganism used is a strain of S. ambofaciens.<br>
41.	A microorganism which produces spiramycin I but which does not produce spiramycin II and HI.<br>
42.	The microorganism as claimed in claim 41, which comprises all of the genes required for the biosynthesis of spiramycin I, but wherein the gene comprising the Sequence SEQ ID No. 13 or one of its variants, or one of the sequences derived therefrom due to the degeneracy of the genetic code, and encoding a polypeptide of<br><br>
sequence SEQ ID No. 14 or one of its variants, is not expressed or has been made inactive.<br>
43.	The microorganism as claimed in claim 42, wherein said inactivation is carried out by mutagenesis in the gene encoding said protein or by the expression of an antisense RNA complementary to the messenger RNA encoding said protein.<br>
44.	The microorganism as claimed in claim 43, wherein said mutagenesis is carried out in the promoter of this gene, in the coding sequence or in a noncoding sequence so as to make the encoded protein inactive or to prevent its expression or its translation therefrom.<br>
45.	The microorganism as claimed in claim 43 or 44, wherein the mutagenesis is carried out by irradiation, by the action of a mutagenic chemical agent, by site-directed mutagenesis or by gene replacement.<br>
46.	The microorganism as claimed in claim 43, 44 or 45, wherein the mutagenesis is carried out in vitro or in situ9 by suppression, substitution, deletion and/or addition of one or more bases in the gene under consideration or by gene inactivation.<br>
47.	The microorganism as claimed in claim 41 or 42, wherein said microorganism is obtained by expressing the genes of the biosynthetic pathway for spiramycin without these genes comprising the gene comprising the sequence corresponding to SEQ ID No. 13 or one of its variants, or one of the sequences derived therefrom due to the degeneracy of the genetic code, and encoding a polypeptide of sequence SEQ ID No. 14 or one of its variants.<br><br>
48.	The microorganism as claimed in claim 41, 42, 43, 44, 45, 46 or 47, wherein said microorganism is a bacterium of the genus Streptomyces.<br>
49.	The microorganism as claimed in claim 41, 42, 43, 44, 45, 46, 47 or 48, wherein said microorganism is obtained from a starting strain which produces spiramycins I, 11 and 111.<br><br><br><br><br><br><br><br>
The microorganism as claimed in claim 68 or 90, which is a bacterium of the genus Streptom'. ces.<br>
70.	The microorganism as claimed in claim 53, 69 or 90. which is a bacterium of :he species Strep:omyces amkcfaciai-iz.<br>
71.	The microorganism as claimed in claim 63, 69. 70 or c0. wherein the overexpression of said gene is obtained by transformation of said microorganism with an expression vector.<br>
72.	A strain of Streptomyces ambcfcciens, which is the strain OSC2/pSPM75(l) or the strain OSC2/pSPM75(2) deposited with the Collection Nationale de Cultures de Microorganismes (CNCM) [National Collection of Cultures and Microorganisms] Pasteur Institute. 25, rue du Docteur Roux 75724 Paris Cedex 15, France, on October 6, 2003, under the registration number 1-3101.<br>
73.	A recombinant DNA which comprises:<br>
-	a polynucleotide which can be obtained bv polymerase chain reaction using the<br>
following pair of sequence primers:<br>
5' AAGCTTGTGTGCCCGGTGTACCTGGGGAGC 3' (SEQ ID No. 138) and<br>
5' GGATCCCGCGACGGACACGACCGCCGCGCA 3? (SEQ ID No. 139) and, as<br>
matrix, the cosmid pSPM36 or the total DNA of Streptomyces ambofaciens,<br>
-	or a fragment of at least 10, 12, 15, 18, 20 to 25, 30, 40, 50, 60, 70, 80, 90,<br>
100, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900,<br>
950, 1000, 1050, 1100, 1150, 1200, 1250, 1300, 1350, 1400, 1450, 1460, 1470, 1480,<br>
1490 or 1500 consecutive nucleotides of this polynucleotide.<br>
74.	The recombinant DNA as claimed in claim 73 or 91, which is a vector.<br>
75.	The recombinant DNA as claimed in claim 73, 74 or 91, which is an expression vector.<br><br>
76.	A host cell into which at least one recombinant DNA as claimed in one of claims 73, 74, 75. and 91 has been introduced.<br>
77.	A method of producing a polypeptide, wherein said method comprises the following steps:<br><br>
a)	transforming a host cell with at least one expression vector as claimed in claim 75;<br>
b)	cultivating, in a suitable culture medium, said host cell;<br>
c)	recovering the conditioned culture medium or a cell extract;<br>
d)	separating and purifying said polypeptide from said culture medium or else from the cell extract obtained in step c);<br>
e)	where appropriate, characterizing the recombinant polypeptide produced.<br><br>
78.	The microorganism as claimed in claim 51, wherein the gene inactivation is carried out by in-phase deletion of the gene or of a part of the gene comprising the sequence corresponding to SEQ ID No. 13 or one of the sequences derived therefrom due to the degeneracy of the genetic code.<br>
79.	The microorganism as claimed in one of claims 41, 42, 43, 44, 45, 46, 47, 48,49, 50, 51 or 78, which also overexpresses:<br>
-	a gene which can be obtained by polymerase chain reaction using the following pair of<br>
sequence primers:<br>
5' AAGCTTGTGTGCCCGGTGTACCTGGGGAGC 3' (SEQ ID No. 138) and<br>
5' GGATCCCGCGACGGACACGACCGCCGCGCA 3' (SEQ ID No. 139) and, as<br>
matrix, the cosmid pSPM36 or the total DNA of Streptomyces ambofacieris,<br>
-	or a gene derived therefrom due to the degeneracy of the genetic code.<br>
80. An expression vector, wherein the polynucleotide of sequence SEQ ID No. 47, or a polynucleotide derived therefrom due to the degeneracy of the genetic code, is placed under the control of promoter which allows the expression of the protein encoded by said polynucleotide in Streptomyces ambofaciens.<br><br>
81.	The expression vector as claimed in claim 80, which is the plasmid<br>
pSPM524orpSPM525.<br>
82.	A strain of Streptomyces ambofaciens transformed with a vector as claimed<br>
in claim 80 or 81.<br>
83) The microorganism as claimed in one of claims 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 78, 79 and 92, which also overexpresses the gene having the coding sequence SEQ ID No. 47 or a coding sequence derived therefrom due to the degeneracy of the genetic code.<br>
84.	The microorganism as claimed in claim 83, .which is the strain SPM502 pSPM525 deposited with the Collection Nationale de Cultures de Microorganismes (CNCM) Pasteur Institute, 25 rue du Docteur Roux 75724 Paris Cedex 15, France, on February 26,2003, under the registration number 1-2977.<br>
85.	A method of producing spiramycin^), which comprises the following steps:<br><br>
(a)	culturing, in a suitable culture medium, a microorganism as claimed in one of claims 68,69, 70,71, 72,78,79,82, 83,84,90 and 92,<br>
(b)	recovering the conditioned culture medium or a cell extract,<br>
(c)	separating and purifying the spiramycins from said culture medium or else from the cell extract obtained in step b).<br>
86.	A polypeptide, the sequence of which comprises the sequence SEQ ID<br>
No. 112 or the sequence SEQ ID No. 142.<br>
87.	A polypeptide, the sequence of which corresponds to the sequence translated<br>
from the coding sequence:<br>
- of a gene which can be obtained by polymerase chain reaction (PCR) using the following pair of sequence primers: 5' AAGCTTGTGTGCCCGGTGTACCTGGGGAGC 3* (SEQ ID No. 138) and<br><br>
5' GGATCCCGCGACGGACACGACCGCCGCGCA 3' (SEQ ID No. 139) and, as matrix, the cosmid pSPM36 or the total DNA of Streptomyces ambofaciens,<br>
- or of a gene derived therefrom due to the degeneracy of the genetic code.<br>
88.	An expression vector which allows the expression of a polypeptide as<br>
claimed in claim 86, 87 or 93 in Streptomyces ambofaciens.<br>
89.	The expression vector as claimed in claim 88, which is the plasmid<br>
pSPM75.<br>
90.	The microorganism as claimed in claim 68, wherein the gene which can be<br>
obtained by polymerase chain amplification is the gene of coding sequence SEQ ID<br>
No. 141, or a gene derived therefrom due to the degeneracy of the genetic code.<br>
91.	The recombinant DNA as claimed in claim 73&gt; wherein the polynucleotide<br>
which can be obtained by polymerase chain amplification is a polynucleotide of<br>
sequence SEQ ID No, 141.<br>
92.	The microorganism as claimed in claim 79, wherein the gene which can be<br>
obtained by polymerase chain amplification is the gene of coding sequence SEQ ID<br>
No. 141, or a gene derived therefrom due to the degeneracy of the genetic code.<br>
93.	A polypeptide, the sequence of which is SEQ ID No. 142.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDU1OC1jaGVucC0yMDA1IGFic3RyYWN0IGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">0558-chenp-2005 abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDU1OC1jaGVucC0yMDA1IGNsYWltcyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">0558-chenp-2005 claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDU1OC1jaGVucC0yMDA1IGRlc2NyaXBpdGlvbiBjb21wbGV0ZWQgZHVwbGljYXRlLTEucGRm" target="_blank" style="word-wrap:break-word;">0558-chenp-2005 descripition completed duplicate-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDU1OC1jaGVucC0yMDA1IGRlc2NyaXBpdGlvbiBjb21wbGV0ZWQgZHVwbGljYXRlLTIucGRm" target="_blank" style="word-wrap:break-word;">0558-chenp-2005 descripition completed duplicate-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDU1OC1jaGVucC0yMDA1IGRlc2NyaXBpdGlvbiBjb21wbGV0ZWQgZHVwbGljYXRlLTMucGRm" target="_blank" style="word-wrap:break-word;">0558-chenp-2005 descripition completed duplicate-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDU1OC1jaGVucC0yMDA1IGRlc2NyaXBpdGlvbiBjb21wbGV0ZWQgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">0558-chenp-2005 descripition completed duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDU1OC1jaGVucC0yMDA1IGRyYXdpbmdzIGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">0558-chenp-2005 drawings duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LUNIRU5QLTIwMDUgQ09SUkVTUE9OREVOQ0UgT1RIRVJTLnBkZg==" target="_blank" style="word-wrap:break-word;">558-CHENP-2005 CORRESPONDENCE OTHERS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LUNIRU5QLTIwMDUgQ09SUkVTUE9OREVOQ0UgUE8ucGRm" target="_blank" style="word-wrap:break-word;">558-CHENP-2005 CORRESPONDENCE PO.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LUNIRU5QLTIwMDUgRk9STSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">558-CHENP-2005 FORM 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LUNIRU5QLTIwMDUgUEVUSVRJT05TLnBkZg==" target="_blank" style="word-wrap:break-word;">558-CHENP-2005 PETITIONS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LUNIRU5QLTIwMDUgUE9XRVIgT0YgQVRUT1JORVkucGRm" target="_blank" style="word-wrap:break-word;">558-CHENP-2005 POWER OF ATTORNEY.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LWNoZW5wLTIwMDUtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">558-chenp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LWNoZW5wLTIwMDUtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">558-chenp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LWNoZW5wLTIwMDUtY29ycmVzcG9uZG5lY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">558-chenp-2005-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LWNoZW5wLTIwMDUtY29ycmVzcG9uZG5lY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">558-chenp-2005-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LWNoZW5wLTIwMDUtZGVzY3JpcHRpb24oY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">558-chenp-2005-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LWNoZW5wLTIwMDUtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">558-chenp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LWNoZW5wLTIwMDUtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">558-chenp-2005-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LWNoZW5wLTIwMDUtZm9ybSAxOC5wZGY=" target="_blank" style="word-wrap:break-word;">558-chenp-2005-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LWNoZW5wLTIwMDUtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">558-chenp-2005-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LWNoZW5wLTIwMDUtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">558-chenp-2005-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LWNoZW5wLTIwMDUtcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">558-chenp-2005-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="229361-re-epithelializing-pharmaceutical-compositions-comprising-xanthan-gum.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="229363-a-gateway-device-placed-on-the-site-of-a-phone-switching-station-and-a-voice-conversation-system-and-a-voice-conversation-method.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>229362</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>558/CHENP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>12/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>20-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>17-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>06-Apr-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>AVENTIS PHARMA S.A</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>20 Avenue Raymond Aron, F-92160 ANTONY,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>DOMINGUEZ, Helene</td>
											<td>30 rue Campane, F-84000 AVIGNON,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>DARBON-RONGERE, Emmanuelle</td>
											<td>6 rue du Grand Parc, F-78960 VOISINS LE BRETONNEUX,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>GERBAUD, Claude</td>
											<td>3 allee Dunand, F-93220 GAGNY,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>GONDRAN, Anne</td>
											<td>45 boulevard Saint Jacques, F-75014 PARIS,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>KARRAY, Fatma</td>
											<td>Residence Universitaire les Rives de l&#x27;Yvette, Bâtiment 23, ch. 2008, F-91440 BURES SUR YVETTE,</td>
										</tr>
										<tr>
											<td>6</td>
											<td>LACROIX, Patricia</td>
											<td>28 allee de Bellevue, F-94170 LE PERREUX,</td>
										</tr>
										<tr>
											<td>7</td>
											<td>OESTREICHER-MERMET- BOUVIER, Nathalie</td>
											<td>9 rue Louise Filliaux-Tiger, F-92140 CLAMART,</td>
										</tr>
										<tr>
											<td>8</td>
											<td>PERNODET, Jean-Luc</td>
											<td>21 rue des Jardins, F-94230 CACHAN,</td>
										</tr>
										<tr>
											<td>9</td>
											<td>TUPHILE, Karine</td>
											<td>18A rue de l&#x27;Oratoire, F-69300 CALUIRE,</td>
										</tr>
										<tr>
											<td>10</td>
											<td>BLONDELET-ROUAULT, Marie-Helene</td>
											<td>26 avenue de l&#x27;Esterel, F-91400 ORSAY,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/31</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/FR2003/002962</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-03-08</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/493,490</td>
									<td>2003-08-07</td>
								    <td>France</td>
								</tr>
								<tr>
									<td>2</td>
									<td>0212489</td>
									<td>2002-10-08</td>
								    <td>France</td>
								</tr>
								<tr>
									<td>3</td>
									<td>0302439</td>
									<td>2003-02-27</td>
								    <td>France</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/229362-polypeptides-involved-in-spiramycin-biosynthesis-nucleotide-sequences-encoding-said-polypeptides by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:02:55 GMT -->
</html>
