<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/229519-method-of-breeding-lipid-producing-fungus by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:13:09 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 229519:METHOD OF BREEDING LIPID-PRODUCING FUNGUS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHOD OF BREEDING LIPID-PRODUCING FUNGUS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention relates to a method of breeding lipid-producing fungus belonging to the genus Mortierella without the use resort to antibiotics. The process comprises carrying out performing transformation to create a population of transformant in a host by transfering a marker selected from orothidine-5&#x27;-phosphate decarboxylase gene or orotidylic acid pyrophosphorylase gene, the host being an uracil auxotrophic strain. of the lipid-producing fungus, and the marker gene being complementary to the auxotrophy of the uracil auxotrophic strain; and recovering lipid-producing fungus belonging to Mortierella spp.from the host by conventional methods.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
DESCRIPTION METHOD FOR BREEDING LIPID-PRODUCING FUNGUS<br>
TECHNICAL FIELD<br>
The present invention relates to a method for breeding a lipid-producing fungus, especially, a method for breeding a lipid-producing fungus belonging to Mortierella spp., the method being such that the breeding is carried out by transformation, but not selection using antibiotic.<br>
BACKGROUND ART<br>
Techniques (fermentation techniques in a broad sense) for producing useful compounds by microorganism's metabolism have been developed and reduced in practice. Lipid-producing fungi capable of producing a lipid in a large quantity by its metabolism are some concrete examples. Fungi belonging to Mortierella spp. such as Mortierella alpina are typical lipid-producing fungi. Mortierella spp., which are known to produce polyunsaturated fatty acid (PUFA) such as arachidonic acid, are industrially useful fungi (e.g., see Japanese Patent Publication, Tokukouhei, No. 7-34752 (published on April 19, 1995) corresponding to Unexamined Japanese Patent Publication, Tokukaisho, No. 63-44891, (published on February 25, 1988)).<br>
Breeding (i.e., breed improvement) for improving their genetic properties has been carried out for various useful organisms including microorganisms such as the lipid-producing fungi and the like. Breeding is very important especially in the fermentation technology in order to attain better efficiency, lower cost, and the like in microbial production of a compound.<br>
The breeding method basically includes the step of creating a population including genetic variation (hereinafter, this step is referred to as the population creating step, for the sake of easy explanation), and the step of selecting, out of the population, a variety having a desirable property (hereinafter, this step is referred to as the selecting step, for the sake of easy explanation). Various processes can be adopted as the population creating step and selecting step depending on which kind of a useful organism is to be bred. For microorganisms such as the lipid-producing fungus and the like, (1) mutagen treating process or (2) transformation process is mainly adopted as the population creating step.<br><br>
(1)	Mutagen treating process<br>
In the population creation step adopting the mutagen treating process, the population is created by inducing mutation in the microorganism by various methods. The mutagenesis causes many kinds of mutations randomly. Therefore, even if a variety (strain) showing a targeted trait is obtained in the selecting step, there is a possibility that the variety would have an unexpected damage in a gene other than the gene regarding the targeted trait. For example, the mutation would deteriorate a lipid-producing fungus in terms of multiplication, spore formation capability, or the like, even though the mutation causes the lipid-producing fungus to produce a different kind of lipid. Therefore, the population creating step adopting the mutagen treating process does not promise that a productive strain can be obtained.<br>
Further, this method creates a population consisted of individuals in which many kinds of mutations are randomly caused. Therefore, if the selecting step after the population creating step did not have appropriate selecting method (screening method) to obtain the mutant (strain) having the targeted character, all the individuals consisting the population should be checked to find out which kind of mutations they have. This is highly labor-consuming.<br>
(2)	Transformation process<br>
Meanwhile, the population creating step adopting the transformation process creates a population of transformant by transfer, into a useful organism to be bred, a DNA fragment necessary to obtain the targeted character (i.e., transformation ). That is, the population creating step adopting the transformation process creates a population controlled to express only specific gene for the targeted character. Therefore, the selecting step after the population creating step is only required to select the desired variety (strain) out of the obtained transformants. Thus, the screening is easy in the selecting step. Further, it is possible to avoid unexpected damage in the gene other than the gene regarding the targeted character. Therefore, this method saves the labor of the breeding significantly.<br>
As described above, the population creating step in the breeding preferably adopts the transformation process. For example, many techniques have been reported as to how to transform filamentous fungi to which Mortierella spp. belong.<br>
Specifically, (a) Documents 1 to 3 listed below disclose techniques for transforming filamentous fungi, such as Aspergillus nidulans, Neurospora crassa, by particle delivery method. In these techniques, uracil-auxotrophic strains are used as host strains that are to be transformed, and genes complementary to uracil auxotrophy strain are used as marker genes, in order to select the transformant.<br>
Moreover, (b) Document 4 listed below discloses a technique of the transformation process of<br><br>
M. alpina In this technique, plotoplasts are prepared from spores and a gene is transferred into cells by electroporation method. Moreover, screening of the transformants is performed using, as the marker gene, hygromycin B-resistant gene (hpt) derived from Esherichia coli. With this, an organism that can grow on a hygromycin containing medium can be selected out as the transformants.<br>
Document 1: Fungaro M. H. et al. Ferns microbial Lett., 25, 293-297, 1995 Document 2: Herzog R. W. et al. Appl. Microbiol. Biotechnoi., 45, 333-337, 1996 Document 3: Armaleo, D. et al. Curr. Genet., 17, 97-103, 1990 Document 4: Mackenzie D. A. et al. Appl. Environ. Microbiol., 66,4655-4661, 2000<br>
However, the technique adopting the transformation is disadvantageous that it is difficult to effectively and efficiently perform the breeding of the lipid-producing fungus that produces PUFA.<br>
Specifically, in the technique (a) adopting the transformation of the filamentous fungus by the particle delivery method, the filamentous fungi such as A. nidulans and N. crassa to be transformed are not suitable for industrial production of the lipids such as PUFA, because lipid productivity of the filamentous fungi is low.<br>
On the other hand, the technique (b) disclosed in Document 4 is a technique for transforming M. alpina, which is known as an industrially-applicable lipid-producing fungus. Thus, the technique (b) is advantageous over technique (a) in terms of industrial application.<br>
However, all Mortierella spp., typified by M. alpina, are not hygromycin sensitive. Therefore, if a hygromycin tolerant fungus of Mortierella spp. is used as the host, the hygromycin tolerance cannot be used as the marker.<br>
Most of PUFA are essential fatty acids, which relate to complicated physiological functions in vivo. Therefore, PUFA receives much attention as an important nutrient recently. Therefore, there is a demand for a fermentation technique that produces PUFA more efficiently/To attain such a fermentation technique, it is necessary to establish a technique for efficiently and effectively breeding fungi of Mortierella spp., which are highly reliable as a lipid-producing fungus.<br>
The present invention is accomplished in view of the above problem. An object of the present invention is to provide a method of breeding a lipid-producing fungus belonging to Mortierella spp., the method being capable of breeding the lipid-producing fungus effectively and efficiently without using an antibiotic.<br><br>
DISCLOSURE OF INVENTION<br>
In view of the above problems, the inventors of the present invention found, as a result of diligent works, that it would be possible to efficiently and effectively perform transformation for all strains of the fungi of Mortierella spp., if an auxotrophic strain of a fungus of Mortierella spp., which is a lipid producing fungus, was obtained and a system was established in which transformation is performed using as a marker a gene complementary to auxotrophy for the auxotrophic strain. The inventors also found that use of this system makes it possible to adopt self-cloning, thereby making the breeding more efficient and effective. The present invention is based on these findings.<br>
A method according to the present invention is a method for breading a lipid-producing fungus belonging to Mortierella spp., the method including the step of: performing transformation. The step of performing the transformation includes the steps of: transferring a marker gene into a host, where the host is an auxotrophic strain of the lipid-producing fungus, and the marker gene is a gene complementary to the auxotrophy of the auxotrophic strain; and selecting a transformant by using, as a marker, recovery from the auxotrophy of the host.<br>
The method is preferably arranged such that the lipid-producing fungus is Mortierella alpina, Mortierella hygrophila, or Mortierella chlamydospora. Moreover, the method is preferably arranged such that the auxotrophic strain is a uracil auxotrophic strain. Furthermore, the method is preferably arranged such that the marker gene is orothidine-5'-phosphate decarboxylase gene or orotidylic acid pyrophosphorylase gene.<br>
Moreover, the method may be arranged such that an electroporation method or particle delivery method is used in the step of transferring.<br>
Furthermore, the method may further includes the step of obtaining the auxotrophic strain from a wild-type lipid-producing fungus.<br>
Additional objects, features, and strengths of the present invention will be made clear by the description below. Further, the advantages of the present invention will be evident from the following explanation in reference to the drawings.<br>
BEST MODE FOR CARRYING OUT THE INVENTION<br>
One embodiment of the present invention is described below. It should be noted that the embodiment is not to limit the present invention.<br>
In the present invention, the step of performing transformation in which auxotrophy is utilized as a marker is performed as the step of creating a population including a genetic variations to be bred (i.e., as the population creating step). Specifically, an auxotrophic strain of a fungus of Mortierella spp. is obtained. This fimgus is a lipid-producing fungus. Using<br><br>
this fungus as a host, a gene complementary to the auxotrophy is transferred into the cells of the fungus (transformation). After that, a transformant is selected out of the population using recovery from the auxotrophy in the host as the marker.<br>
The following describes in detail the fungus of Mortierella spp., which the present invention uses, the breeding method according to the present invention, and a use thereof.<br>
(1)	Fungus of Mortierella spp.<br>
The fungus of Mortierella spp. to be bred by the breeding method of the present invention is not particularly limited. Various filamentous fungi belonging to Mortierella spp. may be used as the fungus, for example. Mortierella spp. is divided into two subgenus, namely, Mortierella and Micromucor. All fungi belonging to Mortierella subgenus produce fatty acids of carbon number of 20 such as arachidonic acid, meanwhile fungi belonging to Micromucor subgenus produce fatty acids of carbon number of 18 or less. The fungus of Mortierella spp. used in the present invention may be any fungi belonging to either Mortierella subgenus or Micromucor subgenus.<br>
Specific examples of the fiingi belonging to Mortierella spp. are: M alpinaDM. elongataUM. exiguaOM. hygrophilaU M. isabelinaU M. tutficolaUM. gamsiiUM. cogitansUM. capitataU M. vinaceaUM. chlamydospora, and the like. It should be noted that the present invention is not limited to these fiingi. Among them, M. alpina {Mortierella alpina), which is widely used as a lipid-producing fungus, can be preferably used in the present invention. It is known that strains of M. alpina accumulate, within the fungus bodies, PUFA such as arachidonic acid (ARA). M. alpina is widely used not only in the researches of biosynthesis of PUFA but also industrial production of PUFA.<br>
How to obtain the fungus of Mortierella spp., including M. alpina, is not particularly limited. It is possible to obtain the fungus from depository institutions for microorganisms and the like, such as Institute for fermentation, Osaka, ATCC (American Type Culture Collection), or the like institution. For strains for which patent application have been filed, it is possible to obtain the strain from the International Patent Organism Depository of the National Institute of Advanced Industrial Science and Technology. Further, it is possible to obtain and use an unknown strain belonging to Mortierella spp. from a natural environment using a well-known screening method.<br>
(2)	Lipid-Producing Fungus Breeding Method According the Present Invention<br>
The breeding method according to the present invention may be arranged in any way, provided that the population creating step comprises a step of performing transformation (hereinafter, transformation step). The breeding method according to the present invention, however, may more preferably comprise a step of obtaining an auxotrophic strain (hereinafter, auxotrophic strain obtaining step). Of course, the breeding method may comprise a step(s) other than these steps. In the following, these steps are described in detail.<br><br><auxotrophic strain obtaining step><br>
The auxotrophic strain obtaining step is not particularly limited, provided that it obtains an auxotrophic strain (auxotrophic mutant) from a wild-type fungus of Mortierella spp. A well-known technique for obtaining an auxotrophic strain may be adopted. Especially, in case where a gene relating to the auxotrophy is known (e.g., in case the gene is found out by the analysis of whole genome, or the like case), it is possible to obtain the auxotrophic strain by knock out the gene.<br>
The present invention is not limited to a particular kind of auxotrophic strain. For example, the auxotrophic strain may be an auxotrophic strain for an amino acid such as leucine, histidine, methionine, arginine, tryptophan, lysine, or the like; an auxotrophic strain for a nucleic acid base such as uracil, adenine, or the like; an auxotrophic strain for a vitamin; or the like. In the present invention, for example, an auxotrophic strain for uracil (hereinafter, uracil auxotrophic strain) is used, as in Examples later described. The uracil auxotrophic strain is especially advantageous for Mortierella spp., because it allows to use 5-fluoro orotic acid (5+FOA) tolerance as a marker for positive selection in the screening.<br>
Therefore, while the auxotrophic strain obtaining step may be arranged to obtain a strain that cannot grow in a synthetic medium in which a particular nutrition is lacked, the auxotrophic strain obtaining step may be arranged to obtain the strain with screening that limits the mutation gene. For example, screening with 5-FOA may be performed in order to obtain a uracil auxotrophic strain in which mutation occurs at URA3 gene or URA 5 gene. Similarly, screening with aminoadipic acid may be performed in order to obtain a lysine auxotrophic strain in which mutation occurs on LYS2 gene or LYS5 gene. It is preferable to obtain an auxotrophic strain in which a mutation gene is limited. This is because it is possible to specify the marker gene for use in the transformation step,<br>
transformation Step 1: Gene transfer Step&gt;<br>
The transformation step is, as described above, the step for performing the transformation in the population creating step for the breeding. The transformation step at least includes a gene transfer step and the selecting-out step. Of course, the transformation step may include a step(s) other than these steps, if necessary.<br>
The gene transfer step in the present invention is not particularly limited, provided that the gene transfer step transfer a marker gene into a host, where the host is an auxotrophic strain of a fungus (lipid-producing fungus) of Mortierella spp., and the marker gene is a gene complementary to the auxotrophy.<br><br>
The marker gene is not particularly limited, as long as the gene is complementary to the mutation (auxotrophic mutation) in the auxotrophic strain. Specifically, examples of the marker gene are: Ieu2 gene for leucine auxotrophic strain, his3 gene for histidine auxotrophic strain, Iys2 gene for lysine auxotrophic strain, trpl gene for tryptophan auxotrophic strain, ura3 gene or ura5 gene for uracil auxotrophic strain, ade2 gene for adenine auxotrophic strain.<br>
For example, the present invention may be arranged such that the host is the uracil auxotrophic strain and the marker gene is ura5 gene, as described in Examples.<br>
There is no particular limitation as to how to obtain the marker gene. The marker gene may be obtained, by using a well-known method, from a (micro)organism (such as yeast) other than the fungus. A commercially-available marker gene may be used. Alternatively, because self-cloning is applicable to the present invention, the marker gene may be obtained from the fungus of the Mortierella spp., which is used as the host. For instance, the later-described Examples are arranged such that ura5 gene was obtained from M. alpina (host), using a base sequence of ura5 gene in three kinds of yeasts.<br>
How to transfer the gene (i.e., how to transform) in the gene transfer step is not particularly limited. A conventionally well-known method such as the electroporation method, particle delivery method, or the like method may be used appropriately. For the fungi of Mortierella spp., it is preferable that the fungus bodies be protoplasted, if the electroporation method is adopted. A specific method of the particle delivery method is the particle gun method. Of course, the particle delivery method is not limited to the particle gun method.<br>
The marker gene is not particularly limited in terms of its specific structure, provided that the marker gene can be transferred into the fiingus bodies of Mortierella spp. so that the marker gene can be expressed. In the present invention, the marker gene should be constructed as a gene construction which is prepared by ligating (a) a region (hereinafter, recombinant region) to be recombined with a chromosome with (b) an expression cassette prepared by ligating a promoter thereto. Therefore, the gene construction may be transferred into the host via a vector in which the gene construction has been inserted. Alternatively, the gene construction may be transferred into the host directly.<br>
The expression cassette of the marker gene, which is contained in the gene construction, should be structured such that the promoter is ligated in the upstream of the marker gene. It is preferable that a terminator be ligated in the downstream of the marker gene. Moreover, in addition to the expression cassettes of the marker gene and the recombinant region, the gene construction may contain a multi-closing site, so that a cassette for expression of various gene can be ligated thereto. Further, the gene construction may contain a DNA fragment other than the expression cassettes of the marker gene (which contains the promoter, the marker gene, and the terminator), the recombinant region, and the multi-cloning site. The gene construction is not particularly limited in terms of its specific structure.<br><br>
There is no particular limitation as to the specific kinds of the DNA fragments mentioned above. For example, there is no particular limitation in case of the promoter, provided that the promoter can cause expression of the marker gene effectively. It is possible to use a known promoter appropriately in the present invention. It should be noted that the present invention is not limited to hisH4.1 promoter used in the later-described Examples. Similarly, there is no particular limitation in case of the terminator, provided that the terminator can function as a transcription termination site. A known terminator may be used in the present invention. It should be noted that present invention is not limited to trpC terminator used in the later-described Examples. Furthermore, there is no particular limitation in the case of the recombinant region, provided that the recombinant region is a DNA segment by which the marker gene to be transferred into the fungus body of the Mortierella spp. can be inserted in chromosome. In the later described Example, the recombinant region is 18S ribosomal DNA (18Sr).<br>
There is no particular limitation regarding the recombinant expression vector in case where the gene construction is introduced into a vector to prepare an expression vector. The expression vector may be equivalent to a general recombinant expression vector. The recombinant expression vector may be prepared by using a plasm id, a phage, a cosmid, or the like. However, there is no particular limitation regarding how to prepare the recombinant expression vector. Moreover, the preparation of the recombinant expression vector may be carried out by using a known method. In general, a plasmid can be used preferably in the preparation of the recombinant expression vector.<br><transformation step selecting-out><br>
The selecting-out step in the transformation step is not particularly limited, provided that the selecting-out step selects out the transformant using the recovery from the auxotrophy of the host as the marker. The selecting-out step may be carried out by using a known method appropriately. Specifically, the selecting-out step may be carried out by, in a synthetic medium, the transformants obtained after the gene transfer step, the synthetic medium being such that a specific nutrition is omitted therein in view of the auxotrophy that is to be used as the marker. With this, it is possible to screen the transformants easily and efficiently.<br>
As described above, the auxotrophy is used as the marker in the transformation step. The auxotrophic marker is advantageous in easy operation and low background. Further, auxotrophic marker is advantageous in that a gene derived from the fungus of Mortierella spp. used as the host can be utilized. Moreover, the use of the auxotrophy as the marker is preferable for food product development, compared with the case where the antibody tolerance is used as the marker.<br>
This is more specifically explained here referring to the case of the technique of Document 4 explained in Background Art, that is, the case where the hygromycin tolerance is used as the marker, by way of example. In order to stably grow the transformants to be selected, it is<br><br>
necessary to add the hygromycin in the medium. As a result, the lipid produced by using the transformants would be contaminated with hygromycin. For the lipid to be consumed as food, it is better to avoid that the lipid is contaminated with hygromycin.<br>
For example, in case of the auxotrophy is the uracil auxotrophic strain, a uracil-lacking medium is used in the actual production. This makes it possible to produce the lipid while selectively growing the fungus in which the gene (such as ura5 gene or the like) complementary to the auxotrophy is transferred. That is, by using the auxotrophy as the marker, it is possible to avoid hygromycin contamination and perform selective culturing always. Thus, it is preferable to use the auxotrophic marker as in the present invention.<br><selecting step><br>
According to the breeding method of the present invention for breeding the lipid-producing fungus, populations of strains of various characters can be obtained by transferring various genes in the transformation step in which the auxotrophy is used as the marker. Therefore, the present invention may be arranged to comprise the selecting-out step for selecting, from the obtained population, a variety having a desired character. The "selecting step" is different from the "selecting-out step" in the transformation step. The selecting step is not for selecting out the transformant, using the auxotrophy as the marker. The selecting step is for selecting a transformant having more preferable character, out of the plurality of the obtained transformants.<br>
There is no particular limitation regarding how to carry out the selecting step specifically. In order to select out the transformant having the preferable character, the selecting step may be arranged, according to the purpose of the breeding, to be carried out by a known method under conditions that allow selection of the transformant having the preferable character.<br>
(3) Use of Present Invention<br>
The breeding method according to the present invention for breeding a lipid-producing fungus is, as described above, arranged such that a desired variety having a desirable character is selected (screened) out of a population having a genetic variations. As the marker in the selection (screening), auxotophy is used. With this arrangement, it is possible to efficiently and effectively produce a novel variety (novel strain) obtained from, as the wild type, a lipid-producing fungus belonging to Mortierella spp. Mortierella spp. are well known as a lipid producing fungus. Especially, M. alpina. is known for its high reliability as a lipid producing fungus. It is possible to produce efficiently and easily a strain improved, for example, in the lipid productivity.<br>
Moreover, the breeding method according to the present invention can perform the transformation using only a gene of the lipid-producing fungus to be transformed. Moreover,<br><br>
self-cloning is applicable in the present invention. Thus, it is possible to obtain an appropriate DNA fragment from the wild-type fungus of Mortierella spp., and transfer the DNA fragment into the auxotrophic strain, thereby controlling expression of a specific gene (high expression, controlling of expression timing, inhibition of expression, etc.). This makes it possible to obtain a transformant, that is, a novel strain, which can attain such improvements such as an increase in lipid production, modification of the lipid production in terms of kind or composition of the lipid to produce, etc. Of course, the present invention may be arranged such that a DNA fragment derived from other variety is transferred into the auxotrophic strain, without self-cloning.<br>
The breeding method according to the present invention is not particularly limited in terms of technique to adopt and may adopt a known technique. For example, in the case of self-cloning, how to obtain an appropriate DNA fragment from the wild-type fungus of Mortierella spp. (host) is not particularly limited and may be performed by a known technique. Moreover, the appropriate DNA fragment (for example a gene to be transferred, etc.) is not particularly limited, provided that the DNA fragment can realize a targeted and desired character, or has a possibility to realized the character.<br>
For example, the DNA fragment may be such that an expression cassette ligated at least with a promoter and preferably with a terminator is inserted in a recombinant expression vector. However, the present invention is not limited to this. Further, the present invention is not limited to GLELO gene, which was used as the DNA fragment in the later-described Examples.<br>
The present invention is described in detail referring to Examples. It should be noted that these Examples are not to limit the present invention.<br>
[Example 1]<br>
Here, a specific example of the breeding method of the present invention is explained. In the present example, a uracil auxotrophic strain was obtained, and transformation and selection of the uracil auxotrophic strain were performed.<br>
(1) Obtaining Uracil Auxotrophic strain (Auxotrophic Strain Obtaining Step)<br>
In order to form spores of M alpina, M alpina was inoculated on Czapek-Dox medium (3% sucrose, 0.2% NaNO3, 0.1%KH2PO4, 0.05%KCl, 0.05%MgSO4-7H2O, 0.001%FeSO4-7H2O, 2% agar; adjusted to pH6.0) and incubated at 28°C for approximately 2 weeks. This was then suspended in Tween 80 aqueous solution (ldrop/lOOml H2O). Then, hyphae were removed using a glass filter (Iwaki Glass Co., Ltd.; Product No.: 3G1), thereby to obtain a spore solution. Mutagen treatment was carried out with N-methyl-N'-nitro-N-nitrosoguanidine (MNNG) with respect to spores of 10  *   108 to  109 according to the method of<br><br>
Jareonkitmongkol et at. (S. Jareonkitmongkol et al. JAOCS, 69, 939-944, 1992).<br>
The spores treated by mutagen were spread on GY medium (2% glucose, 1% yeast extract, 2% agar; adjusted to pH 6.0) containing 5-FOA of l.Omg/ml and uracil of O.OSmg/ml, and incubated at 28°C for 4 days. Thereby, six 5-FOA tolerant strains were obtained. The obtained strains were evaluated with respect to uracil auxotrophy.<br>
That is, growths of these strains on SC medium and uracil-added SC medium were compared. (SC medium: Yeast Nitrogen Base w/o Amino Acids and Ammonium Sulfate (Difco) 5.0g, (NH4)2SO4 1.7g, glucose 20g, agar 20g, adenine 20mg, tyrosine 30mg, methionine l.Omg, arginine 2.0mg, histidine 2.0mg, lysine 4.0mg, tryptophan 4.0mg, threonine 5.0mg, isoleucine 6.0mg, leucine 6.0mg, phenylalanine 6.0mg/liter; uracil-added SC medium: uracil 50mg/liter).<br>
The evaluation showed that all the six 5-FOA tolerant strains could grow on the uracil-added media like the parent strain, but grew poorly on the medium containing no uracil. Two of the six 5-FOA tolerant strains could not grow on the medium containing no uracil. These strains were named as Aura-1 strain and Aura-2 strain.<br>
(2) Evaluation of Enzyme Activity<br>
When a strain shows 5-FOA tolerance and uracil auxotrophy as its phenotypes, one possibility is that the strain lacks an activity of orothidine-5'-phosphate decarboxylase (which is an enzyme coded by ura3 gene; abbreviated to ura3p hereinafter) or orotidylic acid pyrophosphorylase (which is an enzyme coded by ura5 gene; abbreviated to ura5p). Thus, the activities of these enzymes in the Aura-1 strain and Aura-2 strain were evaluated.<br>
These strains were incubated on uracil-added GY medium at 28°C for 4 days with shaking. Preparation of cell-free extract solution of the fiingus bodies was carried out according to a method modified from the method of Wynn et al. (J. P. Wynn et al. Microbiology, 146, 2325-2331,2000).<br>
Specifically, the fungus bodies obtained from the 4-day incubation at 28°C on the GY liquid medium containing uracil of 0.05mg/ml were suspended in 0.1M trisbuffer (pH 7.5) containing P-mercaptoethanol. Then, the cells were crushed by applying 35MPa using a French press. The obtained solution was centrifuged to obtain a supernatant, which was the cell-free extract solution. Further, the supernatant was subjected to super centrifugation under conditions 100,000xg for 60 min, thereby to remove organella. In this way, a soluble fraction was obtained.<br>
The ura3p activity was measured using the obtained soluble fraction according to the method of Yoshimoto et. el. (A. Yoshimoto et al. Methods in Enzymology Vol 51, 74-79, ACADEMIC PRESS 1978). The ura5p activity was measured using the obtained soluble<br><br>
fraction according to the method of Umezu et. al. (K. Umezu et al. J. Biochem., 70, 249-262, 1971). The results, that is, comparison of the enzymic activities regarding pyrimidine biosynthesis, are given in Table 1. As seen from the results in Table 1, both the two strains had the ura3p activity equivalently to the parent strain, but no ura5p activity was detected in these two strains.<br>
Further, 108 to 109 spores prepared separately were subjected to the same mutagen treatment. This gave five 5-FOA tolerant strains, three of which could not grow on the medium containing no uracil.<br><br>
(3)	Obtaining genome DNA of M. alpina and cDNA library<br>
M. alpina and the uracil auxotrophic strains (the Aura-1 strain and Aura-2 strain) derived from M. alpina as the wild type were respectively incubated in GY liquid media at 28°C for 5 days. From the obtained fungus bodies, genome DNAs were prepared according to the method of Sakuradani et. al. (E. Sakuradani et al. Eur. J. Biochem., 260, 208-216, 1999).<br>
Spores of M. alpina were inoculated in a liquid medium (2% glucose, 1% yeast extract; adjusted to pH 6.0) and incubated at 28°C for 4 days, and fungus bodies were collected therefrom. Then, all RNAs were prepared by hydrochloride guanidine/CsCI method. Using Oligotex-dT30<super>mRNA Purification Kit (From Total RNA) (Product Name (Takara Bio Inc.)), mRNAs were purified from total RNA. Then, a cDNA library was prepared using ZAP-cDNA Synhesis Kit (Product Name (STRATAGENE)).<br>
(4)	Obtaining ura5 cDNA from M. alpina<br>
In amino acid sequences of ura5p of three kinds of yeasts, namely, Glomerella graminicota, Saccharomyces cerevisiae, and Sordaria macrospora, the followings are amino acid sequences of two regions having a high homology with each other.<br>
Region 1: FGPAYKGIP (see SEQ. ID. NO. 1) Region 2; KEAKDHGEG (see SEQ. ID NO. 2)<br><br>
Based on these amino acid sequences of the two regions, sense primers and antisense primers respectively having the following base sequences were synthesized.<br>
Senseprimer: 5 -TTYGGHCCIGCITAYAARGGHATYCC-3  (see SEQ. ID. NO. 3) Antisenseprimer: 5 -CCCTCDCCRTGRTCYTTIGCYTCYTT-3   (see SEQ. ID. NO. 4)<br>
"I" in the primer stands for inosine. Note that in the Sequence Table, "n" stands for the site corresponds to inosine.<br>
Then, PCR was carried out with (a) the genome DNA of M alpina as a template, (b) the two primers mentioned above, and (c) Ex Taq polymerase (Takara Bio Inc.), using T Gradient thermocycler (product name; Biometra). The PCR was carried out with 35 cycles of 1 min at 94°C, 1 min at 52°C, and 2 min at 72°C, and then 10 minute elongation at 72°C thereafter.<br>
This produced approximately 130bp DNA fragment, a base sequence of which was then determined. The DNA fragment was found to be highly homologous with the known ura5 gene. Using the DNA as a probe, the cDNA library of M. alpina prepared in (3) was screened by plaque hybridization, thereby obtaining plasmid pBMAURA5 including cDNA of the ura5 gene.<br>
cDNA of the ura5 gene had the base sequence shown in SEQ. ID. NO. 5. The ura5 gene codes ura5p, which has the amino acid sequence shown in SEQ. ID. NO. 6.<br>
(5)	Cloning of ura5 genome DNA<br>
Based on the cDNA sequence (SEQ. ID. NO. 5) of the ura5 gene, the following primer CRON1 and CRON2 were synthesized:<br>
Primer CRON 1: CTTCTCTTCAGCCCCCAGCCGCATCGGTCC   (SEQ. ID. NO. 7) Primer CRON 2: GAGTCCACGAAAACTGCTGATGAGCAGCAC   (SEQ. ID. NO. 8)<br>
Using the genome DNA of M alpina as a template, ura5 genome DNA was cloned by PCR. The PCR was carried out with 30 cycles of 1 min at 94°C, 1 min at 55°C, and 2 min at 72°C, and then 10 minute elongation at 72°C thereafter. As a result, it was found that no intron existed in ura5 genome gene.<br>
(6)	Identification of Mutation Site in Auxotrophic strain<br>
Using the primers CRON 1 and CRON 2, PCR was performed using, as templates, the genome DNA of Aura-1 strain and Aura-2 strain, which were uracil auxotrophic strains.<br><br>
Thereby, ura5 genome genes from the respective strains were cloned. The PCR was carried out with 30 cycles 1 min at 94°C, 1 min at 55°C, and 2 min at 72°C, and then 10 minute elongation at 72°C thereafter. It was found that in the Aura-1 strain one adenine was inserted at 93th from the initiation codon, and a frame shift was observed in ORF. On the other hand, in the Aura-2 strain, guanine at 398th was replaced with adenine, and glycine at 133th in a motif sequence was replaced with aspartic acid. This suggested that these mutations inactivated the ura5p activity in these two strains.<br>
(7)	Recombinant Expression Vector: Constructing pDura5<br>
Plasmid pD4 (supplied from D. B. Archer, see Document 4) was digested with restriction enzymes Ncol and BamHI, and its ends were blunted by using a DNA Blunting Kit (Product Name; Takara Bio Inc.). Then, a DNA fragment of 5.4 kbp except hpt mod fragment, which is the hygromycin B tolerant gene was purified by using GFX PCR DNA and Gel Band Purification Kit (Product Name (made by Amasham Pharmacia Biotech &amp; Science). With the restriction enzymes EcoRI and Xhol, pBMAURA5 obtained in (4) was digested. After blunting its ends, a fragment of approximately 0.65 kbp including ura5 cDNA was purified. These two fragments were ligated with each other by using ligation high (Product Name (Toyobo Ltd.). The direction of ura5 gene is confirmed to select, as plasmid pDura5, a fragment that contains hisH4.1 promoter on the 5' side of ura5 gene, and trpC terminator on the 3' side of ura5 gene.<br>
(8)	Transformation using pDura 5 (Transformation Step)<br>
SC agar medium was used as the selective medium for the transformant. Spores (xlO8) of Aura-1 strain were spread on the SC agar medium. pDura5 was transferred into the host by the particle deliver method (transformation). The gene transfer, carried out by using PDS-1000/He particle delivery system (Product Name; BIO RAD), was performed under the following conditions: Helium pressure: 7590kPa (1100 psi); vacuum in chamber: 28inch HG; Distance from target: 3cm; and tungsten particle diameter 1.1 fim. Four colonies grew on the SC agar medium after 2-3 day incubation at 28°C after the gene transfer. These colonies were collected as the transformant, which were labeled as #1 to #4.<br>
The transformant #1 to #4 were examined by using PCR as to whether the gene was transferred into them. That is, a genome DNA was prepared from each transformant. PCR was carried out using the genome DNA as the template, EX Taq polymerase (Takara Bio Inc.), and the following primers RDNA1 and RDNA2.<br>
Primer RDNA1: ACAGGTACACTTGTTTAGAG (SEQ. ID. NO. 9) Primer RDNA2: CGCTGCGTTCTTCATCGATG   (SEQ. ID. NO. 10)<br><br>
The PCR was carried out with 35 cycles of 1 min at 94°C, 1 min at 54°C, and 2min at 72°C, and then 10 minute elongation at 72°C thereafter. As a control, PCR was performed on the Aura-1 strain.<br>
It was found that no DNA amplification was observed for the Aura-1 strain used as the host. Meanwhile, DNA fragment of l.Skbp was amplified in each of the transformant #1 to #4. This confirmed that recombination with plasmid pDura5 occurred in 18SrDNA region of each of the transformant #1 to #4, thereby transferring the ura5 gene.<br>
Further, the transformant #1 to #4 were incubated in SC liquid media at 28°C for 4 days with shaking, in order to evaluate the ura5p enzyme activity. The results, that is, the comparison of the ura5p enzyme activities is shown in Table 2. As seen from the result in the Table 2, the transformant #1 to #4 had recovered the activity equivalently to or several times greater than the wild type. This proved that the transferred ura5 gene functioned effectively.<br><br>
Further, the same gene transfer operation for approximately 108 spores of Aura-1 strain was repeated two times. The resultants were respectively incubated for 2-3 day at 28°C, which grew 3 and 4 colonies respectively. This confirmed that the transformation is reproducible.<br>
[Example 2]<br>
The present example describes an example where a novel strain, namely GLELO gene-transferred strain was prepared by the breeding method in which the uracil auxotrophic strain prepared in Example 1 was used.<br>
(1) Constructing pDura 5GLELO vector<br>
GLELO gene codes for a fatty acid chain elongation enzyme, which converts y-linolenic acid into dihomo-y-linolenic acid. GLELO gene was prepared by PCR amplification using cDNA of M alpina as a template based on the base sequence of the sequence GB (ID: AF206662)<br><br>
disclosed in J. M. Parker-Barnes et al. Proc. NatL Acad. Sci. USA., 97 (15), 8284-8289, 2000. In the PCR amplification, the following primers MAGLELO1 and MAGLELO2 were used.<br>
Primer MAGLELO1:   CCATGGATGGAGTCGATTGCGCCATTCC   (SEQ. ID. NO. 11) Primer MAGLELO2:   GGATCCTTACTGCAACTTCCTTGCCTTCTC  (SEQ. ID. NO. 12)<br>
The amplified GLELO gene was digested with restriction enzymes Ncol and BamHI, thereby to obtain a fragment of approximately lkb. Moreover, pD4 was digested with the restriction enzymes Ncol and BamHI, thereby to obtain a fragment of approximately 7.7kb. Each fragment was ligated by using ligation high (Product Name; Toyobo Ltd.), thereby to obtain plasmid pDGLELO.<br>
On the other hand, pDura5 was partially digested with EcoRI. Then, DNA fragments of approximately 6.2 kb, which were cut only at one of two EcoRI sites, were purified. After ends of the fragments were blunted by using DNA Blunting Kit (Product Name; Takara Bio Inc.), the fragments were subjected to self ligation by using ligation high (Product Name; Toyobo Ltd.). Then, fragments in which only that one of the sites which was located in the 5' side of histone 4.1 promoter were collected as pDura 5'. Moreover, pDGLELO was digested with EcoRI, thereby to obtain fragments of 2. 7kb. The fragments were then inserted at EcoRI site of pDura5\ From these fragments, a plasmid in which GLELO expression cassettes and ura5 expression cassettes were aligned in the same direction was selected as recombinant expression vector pDura5GLELO.<br>
(2)	Transformation using pDura5GLELO (Transformation Step)<br>
Following the same method as in (8) in Example 1, transformation was carried out for the Aura-1 strain thereby obtaining three transformant. For the sake of easy explanation, these transformants were labeled as #5 to #7, respectively.<br>
The transformant #5 to #7 were examined by using PCR as to whether or not pDura5GLELO was transferred into them. The PCR was carried out in the same manner as in (8) in Example 1, using the primers RDNA1 and RDNA2. It was found that the DNA fragment of 1.5kbp was amplified in each of the three strains #5 to #7. This confirmed that recombination with pDuraSGLELO occurred in 18SrDNA region of the three strains #5 to #7, thereby transferring the ura5 gene.<br>
(3)	Analysis of Expression of GLELO gene<br>
Using RNeasy plant mini kit (Product Name (made by QIAGEN)), total RNA was extracted from each of the two ura5 gene-transferred strains #1 and #2 created in Example 1 and the GLELO gene-transferred strains #5 to #7 created in (2) above. The strains #5 to #7 had been<br><br>
incubated under the same incubation conditions as in the lipid analysis.<br>
Using Super Script First-Strand Synthesis' System for RT-PCR (Product Name (made by Invitrogen)), reverse transcription of the total RNA (lfxg) was performed with random hexamer, thereby synthesizing cDNA. Then, PCR was performed using 1 JJ.1 (1/20 in amount) of synthesized cDNA as a template, EX Taq polymerase (Takara Bio Inc.), and the following primers MAGLELO 5 and MAGLELO 6:<br>
Primer MAGLELO 5:   CTTTGTGGGCATGCAGATCA   (SEQ. ID. No. 13) Primer MAGLELO 6:   TGAAGATGGAGCTGTGGTGGTA   (SEQ. ID. NO. 14)<br>
The PCR was carried out with 20 cycles of lmin at 94°C, lmin at 55°C, 2min at 72°C, and then 10 min elongation at 72°C thereafter.<br>
Electrophoresis of the product of the PCR was carried out. Fluorescent density for a band of fragment of approximately 300bp was compared. As a result, the band was very light in the fluorescent density in both the ura5 gene-transferred strains, while the band was in a clear density in all the three GLELO gene-transferred strains. On the other hand, when the PCR was carried out in the same manner but with 30 cycles, the band was in clear density in all the strains. This suggests that expression amount of GLELO gene was increased in the GLELO gene-transferred strains.<br>
(4) Lipid Analysis of GLELO Gene Transfer-Strain<br>
The spores of the two ura5 gene-transferred strains #1 and #2 created in Example 1, and the GLELO gene- transferred strains #5 to #7 obtained in (2) above were respectively inoculated in liquid media (5% glucose, 1% yeast extract; adjusted to pH 6.0), and incubated at 28°C for 7 days with shaking. After the incubation, the fungus bodies were collected by filtration. The collected fungus bodies were then dried and weighted. According to hydrochloric acid-methanol method, fatty acid residues in the fungus bodies were converted into methyl ester. The obtained methyl ester was extracted with hexane, which was distilled off thereafter, thereby obtaining methyl ester fatty acids. The obtained methyl ester fatty acids were analyzed with gas chromatography. The result of the analysis is shown in Table 3, showing weight of dried fungus bodies when the respective gene-transferred strains (transformant) were grown, and fatty acid compositions and total lipid production amount in the fungus bodies of each strain.<br>
In Table 3, 16:0 is palmitic acid, 18:0 is stearic acid, 18:1 is oleic acid, 18:2 is linoleic acid, 18:3 is y-linoleic acid, 20:3 is dihomo-y-linoleic acid, 20:4 is arachidonic acid, and 24:0 is lignoceric acid.<br><br><br><br>
gene increased the production amount and contents of dihomo-y-linoleic acid and arachidonic acid. The GLELO gene-transferred strains showed a significant increase in the ratio (20:3/18:3) between dihomo-y-linoleic acid and y-linoleic acid, which are the product of the chain-length elongation reaction and a ground substance thereof, or the ratio ((20:3+20:4)718:3) of a sum of dihomo-y-linoleic acid and arachidonic acid, which is a metabolite from dihomo-y-linoleic acid, over y-linoleic acid.<br>
[Example 3]<br>
The present example explains a specific example in which the breeding method according to the present invention was applied to a fungus of Mortierella spp. other than M. alpina,<br>
(1) Obtaining Uracil Auxotrophic strain (Auxotrophic Strain Obtaining Step)<br>
In order to form spores of M. hygrophila and M. chlamydospora, these fungi were respectively inoculated on Czapek-Dox media (3% sucrose, 0.2% NaNO3, 0.1%KH2PO4, 0.05%KCl, 0.05%MgSO4-7H2O, 0.001%FeSO4-7H2O, 2% agar; adjusted to pH6.0) and incubated at 28°C for approximately 2 weeks. These were then suspended in Tween 80 aqueous solution (1 drop/100ml H2O). Then, hyphae were removed using a glass filter (Iwaki Glass Co., Ltd.; Product No.: 3G1), thereby to obtain a spore solution. Mutagen treatment was carried out with N-methyl-N'-nitro-N-nitrosoguanidine (MNNG) with respect to spores of 10 x 108 to 109 according to the method of Jareonkitmongkol et at. (S. Jareonkitmongkol et al. JAOCS, 69, 939-944, 1992).<br>
The spores treated by mutagen were spread on GY media (2% glucose, 1% yeast extract, 2% agar; adjusted to pH 6.0) containing 5-FOA of l.Omg/ml and uracil of 0.05mg/ml, and incubated at 28°C for 4 days. Thereby, two 5-FOA tolerant strains were obtained from M. hygrophila and one 5-FOA tolerant strain was obtained from M. chlamydospora. The obtained strains were evaluated with respect to uracil auxotrophy.<br>
That is, growths of these strains on SG medium and uracil-added SC medium were compared (SC medium: Yeast Nitrogen Base w/o Amino Acids and Ammonium Sulfate (Difco) 5.0g, (NH4)2SO4 1.7g, glucose 20g, agar 20g, adenine 20mg, tyrosine 30mg, methionine l.Omg, arginine 2.0mg, histidine 2.0mg, lysine 4.0mg, tryptophan 4.0mg, threonine 5.0mg, isoleucine 6.0mg, leucine 6.0mg, phenylalanine 6.0mg/liter; uracil-added SC medium: uracil 50mg/liter).<br>
The evaluation showed that all the obtained 5-FOA tolerant strains could grow on the uracil-added medium like the parent strain, but grew poorly on the medium containing no uracil. One of the two strains from M. hygrophila and the strain from M. chlamydospora could not grow on the medium containing no uracil. This one of the two strains from M. hygrophila was named as MH-Aura-1 strain, while the strain from M chlamydospora was named as MC-Aura-1 strain.<br><br>
(2)	Evaluation of Enzyme Activity<br>
These two strains were incubated on uracil-added GY medium at 28°C for 4 days with shaking. Preparation of cell-free extract solution of the fiingus bodies was carried out according to a method modified from the method of Wynn et al. (J. P. Wynn et al. Microbiology, 146, 2325-2331, 2000).<br>
Specifically, the fungus bodies obtained from the 4-day incubation at 28°C on the GY liquid medium containing uracil of 0.05mg/mI were suspended in 0.1M trisbuffer (pH 7.5) containing p-mercaptoethanol. Then, the cells were crushed by applying 35MPa using a French press. The obtained solution was centrifiiged to obtain a supernatant, which was the cell-free extract solution. Further, super centrifugation was carried out under conditions of 100,000xg for 60 min, thereby to remove organella. In this way, a soluble fraction was obtained.<br>
Using the obtained soluble fraction, orotidylic acid pyrophosphorylase (ura5p) activity was evaluated according to the method of Umezu et al. (K. Umezu et al. J. Biochem., 70, 249-262, 1971). The evaluation showed that the ura5p activity was below measurable limit in each of these strains.<br>
(3)	Transformation using pDura 5 (Transformation Step)<br>
M. hygrophila MH-Aura-1 or M chlamydospora MC-Aura-1 was inoculated on Czapek-Dox medium and incubated at 28°C for 2 weeks, thereby forming spores. The spores were collected as described in (1) above.<br>
SC agar medium was used as the selective medium for the transformed strains. pDura5 was transferred into the host by the particle delivery method (transformation).The gene transfer, carried out by using PDS-1000/He particle delivery system (Product Name; BIO RAD), was performed under the following conditions: Helium pressure: 7590kPa (1100 psi); vacuum in chamber: 28inch HG; Distance from target: 3cm; and tungsten particle diameter 1.1 nm.) After the gene transfer, about dozen colonies grew on the SC agar medium after 2-3 day incubation at 28°C for each strain. These colonies were collected as the transformant.<br>
The embodiments and concrete examples of implementation discussed in the foregoing detailed explanation serve solely to illustrate the technical details of the present invention, which should not be narrowly interpreted within the limits of such embodiments and concrete examples, but rather may be applied in many variations within the spirit of the present invention, provided such variations do not exceed the scope of the patent claims set forth below.<br><br>
. INDUSTRIAL APPLICABILITY<br>
A method according to the present invention for breading a lipid-producing fungus is, as described above, includes a transformation step including transferring a marker gene into a host, where the host is an auxotrophic strain of the lipid-producing fungus, and the marker gene is a gene complementary to the auxotrophy of the auxotrophic strain (gene transfer step); and selecting a transformant, using, as a marker, recovery from the auxotrophy of the host (selecting-out step). With this arrangement, the present invention makes it possible to efficiently and effectively produce a novel variety (novel strain) from the lipid-producing fungus belonging to Mortierella spp. as the wild type.<br>
Moreover, self-cloning is applicable in the present invention. Thus, it is possible to obtain an appropriate DNA fragment from the wild-type fungus of Mortierella spp., and transfer the DNA fragment into the auxotrophic strain, thereby controlling expression of a specific gene (high expression, controlling of expression timing, inhibition of expression, etc.). This makes it possible to obtain a transformant, that is, a novel strain, which can attain such improvements such as an increase in lipid production, modification of the lipid production in terms of kind or composition of the lipid to produce, etc.<br>
Mortierella spp., are well known as a lipid producing fungus, and includes highly-reliable lipid producing fungus such as M. alpirta. Thus, according to the present invention, for example, it is possible to attain effective and easy production of a strain which is improved in the lipid productivity, or like effect.<br>
Therefore, the present invention is applicable to industries relating to various fermentation techniques using Mortierella spp., and food industry, pharmaceutical industry, and the like industries using the fermentation techniques.<br><br>
WE CLAIM:<br>
1.	A method for breading a lipid-producing fungus belonging to Mortierella spp., comprising<br>
the step of:<br>
performing transformation, the step of performing the transformation comprising the steps of: transferring a marker gene into a host, where the host is an auxotrophic strain of the lipid-producing fungus, and the marker gene is a gene complementary to the auxotrophy of the auxotrophic strain; and selecting a transformant by using, as a marker, recovery from the auxotrophy of the host.<br>
2.	The method as set forth in Claim 1, wherein:<br>
the lipid-producing fungus is Mortierella alpina, Mortierella hygrophila, or Mortierella chlamydospora.<br>
3.	The method as set forth in Claim 1 or 2, wherein:<br>
the auxotrophic strain is a uracil auxotrophic strain.<br>
4.	The method as set forth in Claim 3, wherein:<br>
the   marker   gene   is   orothidine-5'-phosphate   decarboxylase   gene   or  orotidylic   acid pyrophosphorylase gene.<br>
5.	The method as set forth in any one of Claims 1 to 4, wherein:<br>
an electroporation method or particle delivery method is used in the step of transferring.<br>
6.	The method as set forth in any one of Claims 1 to 5, further comprising the step of:<br>
obtaining the auxotrophic strain from a wild-type lipid-producing fbngus.<br>
Dated this the 15 day of March 2006 <br></super></selecting></transformation></auxotrophic></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDkwMC1jaGVucC0yMDA2IGFic3RyYWN0IGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">0900-chenp-2006 abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDkwMC1jaGVucC0yMDA2IGNsYWltcyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">0900-chenp-2006 claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDkwMC1jaGVucC0yMDA2IGRlc2NyaXBpdGlvbiBjb21wbGV0ZWQgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">0900-chenp-2006 descripition completed duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAwLUNIRU5QLTIwMDYgQ09SUkVTUE9OREVOQ0UgT1RIRVJTLnBkZg==" target="_blank" style="word-wrap:break-word;">900-CHENP-2006 CORRESPONDENCE OTHERS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAwLUNIRU5QLTIwMDYgQ09SUkVTUE9OREVOQ0UgUE8ucGRm" target="_blank" style="word-wrap:break-word;">900-CHENP-2006 CORRESPONDENCE PO.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAwLUNIRU5QLTIwMDYgRk9STSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">900-CHENP-2006 FORM 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAwLUNIRU5QLTIwMDYgRk9STSAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">900-CHENP-2006 FORM 13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAwLUNIRU5QLTIwMDYgRk9STSAxOC5wZGY=" target="_blank" style="word-wrap:break-word;">900-CHENP-2006 FORM 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAwLUNIRU5QLTIwMDYgUE9XRVIgT0YgQVRUT1JORVkucGRm" target="_blank" style="word-wrap:break-word;">900-CHENP-2006 POWER OF ATTORNEY.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAwLWNoZW5wLTIwMDYtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">900-chenp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAwLWNoZW5wLTIwMDYtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">900-chenp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAwLWNoZW5wLTIwMDYtY29ycmVzcG9uZG5lY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">900-chenp-2006-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAwLWNoZW5wLTIwMDYtZGVzY3JpcHRpb24oY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">900-chenp-2006-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAwLWNoZW5wLTIwMDYtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">900-chenp-2006-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAwLWNoZW5wLTIwMDYtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">900-chenp-2006-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAwLWNoZW5wLTIwMDYtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">900-chenp-2006-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAwLWNoZW5wLTIwMDYtcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">900-chenp-2006-pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAwLWNoZW5wLTIwMDYtcHJpb3JpdHkucGRm" target="_blank" style="word-wrap:break-word;">900-chenp-2006-priority.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="229518-device-for-winding-up-several-bunches-of-threads.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="229520-a-process-for-producing-biodiesel-fuel.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>229519</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>900/CHENP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>18-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>15-Mar-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>SUNTORY LIMITED</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1-40, Dojimahama, 2-chome, Kita-ku Osaka-shi, Osaka 530 8203,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>Hiroshi KAWASHIMA</td>
											<td>6-11-1-113 Makami-cho, Takatsuki-shi Osaka 569-1121,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>Sakayu SHIMIZU</td>
											<td>6-9, Tokiwayamashita-cho Ukyo-ku Kyoto-shi Kyoto 616-8212,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>Misa OCHIAI</td>
											<td>4-20-5-401, YAMAZAKI SHIMAMOTO-CHO MISHIMA-GUN, OSAKA 618-001,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/09</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/JP2004/012021</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-08-20</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>299345-2003</td>
									<td>2003-08-22</td>
								    <td>Japan</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/229519-method-of-breeding-lipid-producing-fungus by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:13:10 GMT -->
</html>
