<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/229708-novel-carbonyl-reductase-gene-encoding-the-same-and-process-for-producing-optically-active-alcohols-using-the-same by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:25:24 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 229708:NOVEL CARBONYL REDUCTASE, GENE ENCODING THE SAME, AND PROCESS FOR PRODUCING OPTICALLY ACTIVE ALCOHOLS USING THE SAME</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">NOVEL CARBONYL REDUCTASE, GENE ENCODING THE SAME, AND PROCESS FOR PRODUCING OPTICALLY ACTIVE ALCOHOLS USING THE SAME</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a polypeptide, comprising an amino acid sequence of (A) or (B) below: (A) an amino acid sequence described in seq id. 1; and (B) an amino acid sequence of a polypeptide having a carbonyl reductase activity, the amino acid sequence has 80% or more homology to the amino acid sequence described in SEQ ID No.1.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
DESCRIPTION<br>
NOVEL CARBONYL REDUCTASE, GENE ENCODING THE SAME, AND PROCESS FOR PRODUCING OPTICALLY ACTIVE ALCOHOLS USING THE SAME<br>
TECHNICAL FIELD<br>
The present invention relates to: a polypeptide having an activity to reduce carbonyl group-containing compounds to convert them to optically active alcohols that are industrially useful compounds as intermediate materials for drugs, pesticides, and the like; a DNA encoding the polypeptide; a recombinant DNA obtained by incorporating the DNA into a vector; and a transformant having the recombinant DNA. Further, the present invention relates to a process for producing optically active alcohols using the transformant, culture of the transformant, or a treated product of the transformant.<br>
BACKGROUND ART<br>
As     a    method     of     chemically     producing<br>
(E)-7-[2-cyclopropyl-4-(4-fluorophenyl)-guinolin-3-yl]-3,5-dih ydroxy-hept-6-enoic acid esters,  for example,  the following production route has been known as disclosed in Patent Document<br>
1.<br><br><br><br>
Furthermore, Patent Document 2 discloses another production process using an optically active Schiff base.<br>
Still furthermore, Patent Document 3 discloses a production<br>
process	using        methyl	(R)-3-tert-butyl<br>
dimethylsilyloxy-6-dimethoxyphosphinyl-5-DXohexanoate at a very low temperature.<br>
On the other hand, a method of producing an optically active alcohol product by a stereo-selective reduction of a compound having a carbonyl group using microbial cells and/or a cell preparation is applicable as a cost effective method of production with few byproducts to the manufacturing of the above compound. As a method for a compound including a quinolin ring in a side chain of a carbonyl group, Non-Patent Document 1 has described that the following reaction can be performed using Microbacterium campoquemadoensis<br><br>
Furthermore, Non-patent Document 2 has a description that the following reaction can be performed using a bakers yeast.<br><br><br>
However, with respect to a compound in which carbonyl groups continuously exist in the molecule thereof in addition to the presence of olefin on. the a-position of the carbonyl groups, such as (E)-7-[2-cyclopropyl-4-(4-fluorophenyl}-quinolin-3-yl]-3,5-dio xohepto-6-enoic acid esters, no example in which such a compound can be reduced using a microorganism in a stereo-selective manner has been known in the art.<br><non-patent document> Appl microbiol Biotechnol (1998} 49: p. 709-717<br><non-patent document> Bioorg Med Chem Lett, vol. 8, p. 1403- (1998) <patent document> JP 1-279866 A <patent document> JP 8-92217 A <patent document> JP 8-127585 A<br>
DISCLOSURE OF THE INVENTION<br>
Therefore, it has been desired to develop a novel production<br>
process	with	which<br>
(3R, 5S) -(E)-7-[2-cyclopropyl-4-(4-fluorophenyl]-quinolin-3-yl] -3,5-dihydroxyhept-6--enoic acid esters can be cost-effectively produced on an industrial scale.<br><br>
To solve the above-mentioned problems, the inventors of the present invention have made intensive studies on a process for producing<br>
(3R,5S)-{E)-7-[2~cyclopropyl-4-{4-fluorophenyl)-quinolin-3-yl] -3,5-dihydroxyhept-6-enoic acid esters. As a result, they have found that an objective compound can be obtained with a high optical purity and a high concentration by: isolating a novel enzyme that catalyzes a reduction reaction of carbonyl group-containing compounds that can be used as starting materials; and allowing the enzyme, which is expressed by using a recombinant bacteria, to act on the carbonyl group-containing compounds that can be used as starting materials. Thus, they have achieved the present invention. That is, the gist of the present invention resides in:<br>
(1)	A polypeptide, comprising an amino acid sequence of (A)<br>
or (B) below:<br>
(A)	an amino acid sequence described in SEQ ID NO: 1; and<br>
(B)	an amino acid sequence of a polypeptide having a carbonyl reductase activity, the amino acid sequence is an amino acid sequence including deletion, addition, or substitution of one or a plurality of amino acids in the amino acid sequence described in SEQ ID NO: 1.<br>
(2)	A DNA, comprising a nucleotide sequence of any one of (a)<br>
to (e) below:<br>
(a) a nucleotide sequence encoding a polypeptide consisting<br><br>
of an amino acid sequence described in SEQ ID NO: 1;<br>
(b)	a nucleotide sequence encoding a polypeptide having a carbonyl reductase activity, the polypeptide consists of an amino acid sequence including deletion, addition, or substitution of one or a plurality of amino acids in the amino acid sequence described in SEQ ID N0:1;<br>
(c)	a nucleotide sequence described in SEQ ID NO: 2;<br>
(d)	a nucleotide sequence encoding a polypeptide having a carbonyl reductase activity, the nucleotide sequence is a nucleotide sequence including deletion, addition, or substitution of one or a plurality of bases in the nucleotide sequence described in SEQ ID NO:2; and<br>
(e)	a nucleotide sequence that hybridizes with the nucleotide<br>
sequence described in SEQ ID NO:2 or a complementary nucleotide<br>
sequence thereof or a part of these sequences under stringent<br>
conditions, the nucleotide sequence encoding a polypeptide having<br>
a carbonyl reductase activity.<br>
(3)	Arecombinant DNA obtained by incorporating a DNA according<br>
to (2) into a vector.<br>
(4)	A transformant having a recombinant DNA according to (3) .<br>
(5)	A transformant obtained by incorporating a DNA according to (2) into a chromosomal DNA.<br>
(6)	A process for producing a compound represented by the<br>
following formula (IV)<br><br><br>
(where R represents a hydrogen atom, an alkyl group, or an aryl group), comprising:<br>
reacting a carbonyl group-containing compound selected from the group consisting of a compound represented by the following<br><br>
(where R has the same meaning as that described above}, a compound<br><br>
(where Rhas the same meaning as that described above) , and a compound represented by the following formula (III)<br><br>
iwnere K nas une same meaning as tnac aescrii^ea aDove) with any one selected from the group consisting of a transformant cell according to (4) or {5), a culture broth of the transformant cell, and a treated product of the transformant cell to asymmetrically reduce the carbonyl group-containing compound.<br>
(7) A process according to (6), wherein the compounds represented by formulae (II) and (III) consist of optically active<br><br><br>
BEST MODE FOR CARRYING OUT THE INVENTION<br>
Hereinafter, the present invent ion will be described in detail.<br>
The polypeptide of the present invention is one having the amino acid sequence described in SEQ ID NO: 1 or a homolog thereof having a carbonyl reductase activity.<br>
The "carbonyl reductase activity" as used herein refers to activity that asymmetrically reduces carbonyl groups in carbonyl group-containing compounds to produce optically active alcohols. Such an activity can be measured by: reacting an objective polype tide, a transformant, culture of the transformant, or treated product of the transformant as an enzyme in a reaction system containing a carbonyl group-containing compound as a substrate and NADPH as a coenzyme; and measuring an initial rate of decrease in NADPH.<br>
Since its amino acid sequence and nucleotide sequence encoding the amino acid sequence have been elucidatedby the present invention, the polypeptide of the present invention can be obtainedby: isolating DNA encoding a reductase from any microorganism having a carbonyl group reducing activity by use of a probe prepared based on a nucleotide sequence that encodes a partial or whole amino acid sequence of the polypeptide of the present invention; and then using a conventional gene engineering technique based on the isolated DNA, as will be described hereinbelow. Further, as has been made upon completion of the present invention, the polypeptide of the present invention can be purified from culture broths of<br><br>
microorganisms having a carbonyl group reducing activity, that is, microorganisms having DNA that encodes a carbonyl reductase, for example, yeasts belonging to the genus Ogataea.<br>
Among the yeasts that belong to the genus Ogataea, for example, Ogataea minuta has particularly excellent productivity of the carbonyl reductase of the present invention. The yeast is available from Institute for Fermentation, Osaka as IFO 1473 strain.<br>
Any conventional purification method can be used as a method of obtaining the polypeptide of the present invention from the culture broths of microorganisms. For example, the following method can be used. After the above-mentioned microorganism has been sufficiently proliferated by cultivating it in a common medium for use in cultivation of fungi, such as YM medium, the microorganism is collected and disrupted in a buffer, which is added with a reducing agent such as DTT (dithiothreitol) or a protease inhibitor such as phenylmethanesulfonyl fluoride (PMSF), to form a cell-free extract. The polypeptide can be purified from the cell-free extract by appropriate combinations of fractionation based on solubility of aprotein (such as precipitation with organic solvents, or salting out with ammonium sulfate or the like), cation exchange* anion exchange, gel filtration, hydrophobic chromatography, affinity chromatography with chelate, dyestuff, antibody, or the like, and so on. For example, the polypeptide can be purified to almost an electrophoretically  single  band  through  anion  exchange<br><br>
chromatography using DEAE-Sepharose, hydrophobic chromatography using octyl-Sepharose, anion exchange chromatography using Q-Sepharose, gel filtration using Superdex 200, or the like.<br>
The thus-purifiedpolypetide of the present invention derived from Ogataea minuta typically has the following properties. (1) Optimum pH<br>
5.0 to 6.0. (2J Molecular weight<br>
About 27,000 Da by measurement of molecular weight by sodium dodecylsulfate-polyacrylamide gel electrophoresis (hereinafter, abbreviated as "SDS-PAGE").<br>
Further the enzyme can be characterized by the following properties.<br>
(3)	Stable pH range<br>
Relatively stable in the range of pH 5.5 to 6.5.<br>
(4)	Operation temperature range<br>
Optiniuin temperature is 60 to 70°C.<br>
(5)	Temperature stability<br>
Relatively stable up to 40°C.<br>
(6)	Inhibition<br>
The enzyme is inhibited by a mercury (I) ion and a lead (II) ion.<br>
The polypeptide of the present invention isolated by the above-mentioned method is an excellent enzyme that has an ability<br><br>
to asymmetrically reduce not only carbonyl group-containing compounds but also dicarbonyl compounds.<br>
A homolog of the polypeptide of the present invention refers to a polypeptide, comprising an amino acid sequence including deletion, addition, or substitution of one or a plurality of amino acids in the amino sequence described in SEQ ID NO: 1 so far as the carbonyl reductase activity is not deteriorated. The term "a plurality of" as used herein means specifically 20 or less, preferably 10 or less, more preferably 5 or less.<br>
The homolog of the polypeptide of the present invention refers to a polypeptide comprising an amino acid sequence that has at least 50%, preferably at least 60% or 70%, more preferably 80% or more homology to the amino acid sequence described in SEQ ID MO: 1.<br>
Incidentally, homology search of the above-mentioned polypeptide can be performed by searching DNA Databank of JAPAN (DDBJ) using FASTA program or BLAST program. Search of the amino acid sequence described in SEQ ID NO: 1 on DDBJ using Blast program indicated that among the known proteins, the one that showed the highest homology was the probable short chain dehydrogenase (T41540} of Schizosaccharomyces pombe,   which showed 37.4% homology.<br>
A DNA of the present invention is a DNA that encodes the above-mentioned polypeptide or a homolog thereof having carbonyl reductase activities.<br>
Specific examples of the DNAs that encode the above-mentioned<br><br>
polypeptide include those DNAs that include the nucleotide sequence described in SEQ ID NO: 2.<br>
The DNA that encodes the polypeptide of the present invention can be isolated by, for example, the following method.<br>
First, the polypeptide of the present invention is purified by the above-mentioned method or the like, and then the N-terminal amino acid sequence thereof is analyzed. Further, after the polypeptide has been cleaved with an enzyme such as lysylendopeptidase or V8 protease, the resultant peptide fragments are purified by reversed-phase liquid chromatography or the like. Then, the amino acid sequences of the fragments are analyzed by using a protein sequencer to determine a plurality of amino acid sequences.<br>
Primers for PCR are designed based on the determined amino acid sequences and PCR is performed using chromosomal DNA or cDNA library of a carbonyl reductase producing microbial strain as a template and the PCR primer designed from the determined amino acid sequence to obtain a part of the DNA of the present invention. Further, the DNA of the present invention can be obtained by colony hybridization, plaque hybridization, or the like by using the thus-obtained DNA fragment as a probe and utilizing a library, which is obtained by introducing the chromosomal DNA of the carbonyl reductase producing microbial strain digested with a restriction enzyme into phage, plasrnid, or the like or a cDNA library. Further,<br><br>
the DNAs of the present invention can also be obtained by reverse PCR (Genetics 120, 621-623 (1988)) including: analyzing the nucleotide sequence of the DNA fragment obtained by PCR; designing a PCR primer for elongating a chain outward from the DNA based on the obtained sequence; and digesting the chromosomal DNA of the carbonyl reductase producing microbial strain with an appropriate restriction enzyme and utilizing an autocyclized DNA as a template, or by an RACE method (RapidAmpliflcation of cDNAEnd, "PCR Experiment Manual", p25-33, HBJ Publishing Section), or the like.<br>
Note that since their nucleotide sequences have been elucidated by the present invention, the DNAs of the present invention can be obtained not only by the above-mentioned cloning method as the genome DNA or cDNA, but by chemical synthesis or the like based on SEQ ID NO: 2.<br>
The DNA homologs encoding the polypeptide of the present invention include those DNAs that encode polypeptides comprising an amino acid including deletion, addition, or substitution of one or a plurality of amino acids in the amino sequence described in SEQ ID NO: 1 so far as the carbonyl reductase activity is not deteriorated. Specifically, the DNA homologs encoding the above-mentioned polypeptide include those DNAs having nucleotide sequences that encode polypeptides corresponding to the polypeptide encoded by the nucleotide sequence described in SEQ ID NO: 2 and include deletion, addition, or substitution of one or a plurality<br><br>
of amino acids so far as the carbonyl reductase activity thereof is not deteriorated. The term "a plurality of" as used herein means specifically 60 or less, preferably 30 or less, more preferably 10 or less.<br>
One skilled in the art can obtain the homologs of DNA of the present inventionby introducing appropriate substitution, deletion, insertion, and/or addition mutations into the DNA described in SEQ IDNO: 2 by using a site-specificmutation introducingmethod {Nuclei c Acid Res. 10, pp. 6487 (1982), Methods in Enzymol. 100, pp. 448 (1983), Molecular Cloning 2nd Edt., Cold Spring Harbor Laboratory Press (1989) , and PCR A Practical Approach IRL Press pp. 200 (1991) ) , and the like.<br>
Further, the homologs of DNA of the present invention can also be obtained by performing hybridization with DNA prepared from any microorganism having a carbonyl group reducing activity by a colony hybridization method, a plaque hybridization method, a Southern blot hybridization method, or the like using the DNA encoding the polypeptide of the present invention or a part thereof as a probe under stringent conditions to select a DNA that hybridizes with the probe. The term "a part" of DNA that encodes the polypeptide of the present invention means a DNA that has a sufficient length to be used as a probe, specifically, 15 bp or more, preferably 50 bp or more, more preferably 100 bp or more.<br>
Each hybridization may be preformed according to the method<br><br>
described ia Molecular Cloning: A Laboratory Manual, 2nd Ed., Cold Spring Harbor Laboratory, Cold Spring Harbor, NY,, 1989 (hereinafter, abbreviated as "Molecular Cloning, 2nd Ed.") or the like.<br>
The term "nucleotide sequence that hybridizes under stringent conditions" means a nucleotide sequence of a DNA that is obtained by using a colony hybridization method, a plaque hybridization method, a Southern blot hybridization method, or the like using a DNA as a probe under stringent conditions. Examples of the stringent conditions include those conditions in which in the colony hybridization method or plaque hybridization method, hybridization is performed using a filter having fixed thereon a DNA derived from a colony or plaque or a fragment of the DNA in the presence of 0.7 to 1. 0 M sodium chloride at 65°c, and then the filter is washed with 0.1 to 2 X SSC solution (the composition of 1 x SSC is 150 mM sodium chloride and 15 mM sodium citrate) under the condition of 65°C.<br>
Carbonyl reductase expression vectors can be provided by incorporating the thus-isolated DNA that encodes the polypeptide of the present invention into known expression vectors as enable the DNA to express. Further, cultivation of transf ormants that are transformed with the expression vectors enables one to obtain carbonyl reductase from the transf ormants. The expression vectors are not particularly limited so far as they can express the DNAs of the present invention and can be selected as appropriate depending on the kind of host microbial to be transformed and the like.<br><br>
Alternatively, the transformants can also be obtained by incorporating the DNA of the present invention to a chromosomal DNA of a known host microbial as enable the DNA to express.<br>
Specific examples of the method of making the transformants include: a method involving introducing the DNA of the present invention into a plasmid vector or phage vector that exists in a microbial stably and introducing the constructed expression vector into the microbial; and a method involving introducing the DNA of the present invention into a host genome directly to allow genetic information thereof to be transcribed and translated.<br>
In the case where the DNA of the present invention does not contain a promoter that can be expressed in a host microbial, an appropriate promoter can be incorporated in the upstream of the 5'-terminal of the DNA chain of the present invention or more preferably, a terminator can be incorporated in the downstream of the 3'-terminal of the DNA chain of the present invention. The promoter and terminator are not particularly limited so far as they are a promoter and a terminator that are known to function in the microbial utilized as a host. Such vectors/ promoters, and terminators that can be used in various microbials are described in detail in, for example, "Microbiology Fundamental Course 8, Gene Engineering, Kyoritsu Publishing". In particular, yeasts are described in detail in Adv. Biochem. Eng. 43, 75-102 (1990), Yeast 8, 423-488 {1992) and the like.<br><br>
The host microbial as a target of transformation for expressing the carbonyl reductase of the present invention is not particularly limited so far as the host itself gives no adverse influence on the reaction. Specific examples thereof include microbials as shown below.<br>
Bacteria that belong to the genera Escherichia, Bacillus,<br>
Pseudomonas,	Serratia,	Brevibacterium,	Corynebacterium,<br>
Streptococcus,  Lactobacillus,  and so on and of which host vector systems are established;<br>
ActinomyceCes that belong to the genera Rhodococcus, Streptoinyces, and so on and of which host vector systems are established;<br>
Yeasts that belong to the genera Saccharomyces, Kluyveromyces, Schizosaccharomyces, Zygosaccharomyces, Yarrowia, Trichosporon, Rhodosporidium, Hansenula, Pichia, Candida, and so on and of which host vector systems are established; and<br>
Molds that belong to the genera Nevrospora, Aspergillus, Cephalosporium, Trichoderma, and so on and of which host vector systems are established.<br>
Among the above-mentioned microbials, preferable hosts include Escherichia, Bacillus, Brevibacterium, and Corynebacterium, particularly preferably Escherichia  and Corynebacterium.<br>
Procedures for making transformants, construction of recombinant vectors suitable for hosts, and cultivation of hosts<br><br>
can be performed according to the technologies commonly used in the fields of molecular biology, biotechnology, and gene engineering (for example, Sambrook et al., Molecular Cloning, Cold Spring Harbor Laboratories).<br>
Hereinafter, specific examples of host microbial, preferable transforming techniques, vector, promoter, terminator, and so on for eachmicrobial will be described. However, the present invention should not be considered to be limited thereto.<br>
In the case of Escherichia, particularly Escherichia coli, plasmid vectors include pBR, pUC plasmids, and promoters derived from lac (|3-galactosidase) , trp (tryptophan operon) , tac, trc (fusion of lac and trp) , and A-phage PL and PR. Further, the terminators include trpA derived terminator, phage derived terminator, and rrnB ribosomal RNA derived terminator.<br>
In the case of Bacillus, the vectors include pUBllO plasmid and pC194 plasmid. Integration into chromosome is also possible. The promoters and terminators that can be utilized include promoters and terminators of genes of enzymes such as alkaline protease, neutral protease, and a-amylase.<br>
In the case of Pseudomonas, the vectors include commonly used host vector systems established for Pseudoino/idsputida, Pseudomonas cepacia, and so on, a broad host range vector (containing a gene necessary for autonomic replication derived from, for example, RSFIOIO)  pKT240 based on TOL plasmid that participates in<br><br>
decomposition of toluene compounds (Gene, 26, 273-82 (1983)).<br>
In the case of Brevibacterivm, particularly Brevibacterium lactofermentum, the vectors include plasmid vectors such as pAJ43 (Gene, 39, 281 (1985) ) . The promoters and terminators that can be used include various promoters and terminators used for Escherichia coli .<br>
In the case of Corynebacterium, particularly Corynebacterium glutamicum, the vectors include plasmid vectors such as pCSll (JP 57-183799 A) and pCBlOl (Mol. Gen. Genet. 196, 175 (1984).<br>
In the case of Saccharomyces, particularly Saccharomyces cerevisiae, the vectors include plasmids such as YRp, YEp, YCp, and Yip plasmids. Promoters and terminators of genes of various enzymes such as alcohol dehydrogenase, glyceryl aldehyde-3-phosphate dehydrogenase, acidic phosphatase, |3-galactosidase, phosphoglycerate kinase, and enolase can also be used.<br>
In the case of Schizosaccaromyces, the vectors include plasmid vectors derived from Schizosaccharomyces pombe described in Mol. Cell. Biol. 6, 80 (1986). In particular, pAUR224 is commercially available from Takara Shuzo and can be readily used.<br>
In the case of Aspergillus, Aspergillus niger, Aspergillus orizae, and so on have been best studied among molds and integration into a plasmid and chromosome can be used, and promoters derived from extracellular protease or amylase can be used (Trends in<br><br>
Biotechnology 7, 283-287 (1989)).<br>
Further, besides those described above, host vector systems depending on various microbials have been established and they can be used appropriately. In addition, various host vector systems have been established in plants and animals in addition to the microbials. In particular, systems that allow hetero proteins to be expressed in large amounts in animals, particularly insects such as a silkworm (Nature 315, 592-594 (1985)) and in plants such as rapeseed, corn, andpotato, as well as systems using cell-free protein synthesizing systems such as Escherichia coli cell-free extract solution and wheat germ have been established and can be used appropriately.<br>
Further, the present invention relates to a process for producing an optically active alcohol by reacting the transformant cells of the present invention obtainedby the above-mentionedroethod or the like, culture broth of the transformant cells, or treated products of the transformant cells with a compound represented by one of the general formulae (I), {II), and (III) as a reaction substrate to cause an asymmetric reduction of the carbonyl group of the compound. The transformant cells, culture broth of the transformant cells, and treated products of the transformant cells may be used singly or in combination.<br>
A compound represented by the following formula (I)<br><br><br>
In the compounds representedby the above formulae (I) to (Ill)y which are raw materials to be used in the production process of the present invention, R represents a hydrogen atom, an alkyl group, or an aryl group.<br>
Examples of the alkyl group include: a straight-, branched-, or cyclic alkyl group which may be substituted with an alkyl group<br><br>
or an aryl group, such as a methyl group, an ethyl group, an isopropyl group, a cyclopropyl group, a butyl group, an isobutyl group, a t-butyl group, a cyclohexyl group, a benzyl group, or a phenethyl group.<br>
Examples of the aryl group include a phenyl group or a naphthyl group which may be substituted with an alkyl group, such as a phenyl group, a mesityl group, or a naphthyl group.<br>
The above R is preferably a C1-C4 alkyl group, a benzyl group, or a phenyl group, more preferably a C1-C4 alkyl group, particularly preferably a methyl group or an ethyl group.<br>
In the production process of the present invention, the compounds represented by the formulae (II) and (III) contain at least optically active forms represented by the following formulae {II') and (III')/ respectively. It is preferable that among the compounds represented by the formulae (II) and (III), the contents of the compounds represented by the formulae (II') and (Hi') be large. Further, it is preferable that the compounds represented by the formulae (II) and (III) consist of those represented by the formulae [II'} and (III'). For the compounds represented by the formulae (II' ) and (III' ) , R has the same meaning as that described above.<br><br><br>
Note that the products obtained from the compounds represented by the formulae (I) to (III) and {II') and [III') according to the production process of the present invention may be mixtures but contain at least the compounds represented by the following formula (IV) . In the compounds represented by the following formula (IV) , R has the same meaning as that described above.<br><br>
(where R has the same meaning as that described above).<br>
The compounds represented by the formulae (I) to (III) and<br><br>
{II') and (III') can be optionally produced with a coinbination of a process disclosed in JP 1-279866 A, JP 8-127585 A, JP 5-178841 A, or the like and a process well known in the art. Those compounds may be used singly or two or more may be combined and used as the raw materials.<br>
In the case where a compound represented by the formula (IV) is produced by using a compound represented by the formula (I) as a rawraaterial, there are two routes, i.e,, a route in which production is performed through a compound represented by the formula (II') as an intermediate product and a route in which production is performed through a compound represented by the formula (III') as an intermediate product.<br>
The compound represented by the formula (II') and the compound represented by the formula (III') may be prepared in advance from the compound represented by the formula (I) , isolated, and further derived into the compound represented by the formula (IV) . Alternatively, the compound represented by the formula {II') and the compound represented by the formula (III') may be used as they are without isolation directly for producing the compound represented by the formula (IV).<br>
Further, the reaction substrate is used at a substrate concentration of usually in the range of 0 . 01 to 90 w/v%, preferably 0.1 to 30 w/v%. The reaction substrate may be added at a time upon initiation of the reaction. However, from the viewpoints of reducing<br><br>
the possible influence of substrate inhibition of the enzyme and of increasing accumulation concentration of the product, it is desirable to add continuously or intermittently.<br>
In the production process of the present invention, when the above-mentioned transformant is acted on the carbonyl group-containing compound, the transformant cells themselves, culture broth of the transformant, or treated transformant cells obtained by treating the transformant cells by a known method, for example, treated bacterial cells such as one obtained by treating the transformant cells with organic solvents such as acetone, dimethyl sulfoxide (DMSO) , and toluene or surfactants, one obtained by freeze-drying, one obtained by physically or enzymatically disrupting the transformant cells, one obtained by extracting the enzyme fraction of the present invention in the transformant cells as a crude purified product or purified product, and further one obtained by immobilizing these to a carrier typified by polyacrylamide gel or carrageenan can be used.<br>
Further, in the production process of the present invention, it is preferable that coenzyme NADP"^ or NADPH be added. Usually, the coenzyme NADP"^ or NADPH is added in an amount of 0.001 roM to 100 mM, preferably 0.01 to 10 mM.<br>
When the above-mentioned coenzyme is added, it is preferable that NADP"^ produced from NADPH be regenerated to NADPH in order to increase production efficiency. The regeneration method includes :<br><br>
1) a method that utilizes NADP* reducing ability of host microbial itself; 2) a method including addition of a microbial having the ability of producing NADPH from NADP"^ or a treated product thereof or enzymes that can be used for regeneration of NADPH (regeneration enzymes) including glucose dehydrogenase, formic acid dehydrogenase, alcohol dehydrogenase, amino acid dehydrogenase, and organic acid dehydrogenase (such as malic acid dehydrogenase) into the reaction system; and 3) a method that includes incorporation of gene of the above-mentioned regeneration enzymes, which can be used for regeneration of NADPH, at the same time along with the DNA of the present invention into the host when the transformant is prepared.<br>
Among the methods, in the method of 1) above, it is preferable that glucose, ethanol, formic acid, or the like be added to the reaction system.<br>
Further, in the method of 2) above, microbial containing the above-mentioned regeneration enzymes, treated bacterial cell such as one obtained by treating the bacterial cell with acetone, one obtained by freeze-drying the bacterial cell, one obtained by physically or enzymatically disrupting bacterial cell, one obtained by extracting the enzyme fraction as a crude product or purified product, and, further, one obtainedby immobilizing these to acarrier typified by polyacrylamide gel or carrageenan gel may be used. Alternatively, commercially available enzymes may be used.<br>
In this case, the above-mentioned enzyme, specifically, is<br><br>
added in such an amount that the amount of the regeneration enzyme is about 0.01 to 100 folds, preferably about 0.5 to 20 folds the amount of the carbonyl reductase of the present invention.<br>
Further, it becomes necessary to add the compound that can be used as a substrate of the above-mentioned regeneration enzyme, for example, glucose in the case where glucose dehydrogenase is used, formic acid in the case where formic acid dehydrogenase is used, or ethanol or isopropanol in the case where alcohol dehydrogenase is used. The substrate compound is added in an amount of 0.1 to 20 fold molar equivalent, preferably 1 to 5 fold molar equivalent with respect to the carbonyl group-containing compound being used as a reaction raw material.<br>
Further, in the method of 3) above, a method that includes incorporating the DNA of the present invention and the DNA encoding the above-mentioned regeneration enzyme in a chromosome, a method that includes incorporating both the DNAs in a single vector and transforming the host therewith, and a method that includes incorporating the DNAs into separate vectors and thereafter transforming the host with the vectors may be used. In the case of the method that includes incorporating the DNAs into separate vectors and then transforming the host therewith, it is necessary to select vectors taking into consideration incompatibility between both the vectors.<br>
In the case where a plurality of genes are incorporated into<br><br>
a single vector, it is possible to use a method that includes connecting regions that participate in control of expression, such as a promoter and a terminator to each gene and a method that includes expressing an operon containing a plurality of cistrons, such as lactose operon.<br>
The production process of the present invention is performed in an aqueous solvent that contains a reaction substrate, the transformant cells of the present invention, culture broth of the transformant cells, or treated transformant, and various coenzymes added as necessary as well as a regeneration system for the coenzymes, or in a mixture of the aqueous solvent and an organic solvent. Buffers using sodium phosphate, potassium phosphate, and so on can be used as the above-mentioned aqueous solvent, in which optional components used in ordinary enzyme reactions, such as organic solvents and surfactants, are added appropriately. The organic solvents include; water-soluble solvents such as dimethyl sulfoxide (DMSO) and tetrahydrofuran (THF); and water-insoluble organic solvents such as butyl acetate and hexane. Those having high solubilities of reaction substrates are preferably used. The surfactants include Tween 80 and sugar esters.<br>
The process of the present invention can be performed usually at a reaction temperature of 4 to 60°C, preferably 10 to 45°C and usually at pH 3 to 11, preferably pH 5 to 8. It is also possible to perform the process by using a membrane reactor or the like.<br><br>
The optically active alcohols produced by the process of the present invention can be purified by: separating the bacterial cells or protein in the reaction mixture after completion of the reaction by centirifugation, membrane treatment, or the like; and subjecting the resultant to a combination of extraction with an organic solvent such as ethyl acetate or toluene, distillation, column chromatography, crystallization, and so on appropriately.<br>
Examples<br>
Hereinafter, the present invention will be described in more detail based on examples. However, general modifications are allowed in the technical field of the present invention without departing from the gist of the present invention.<br>
In	the	meantime,<br>
(E) -7~[2~cyclopropyl-4-(4-fluorophenyl}-quinolin-3-yl]-3,5-dih<br>
ydroxyhept-6-enoic acid esters have isomers : {3S, 5R) -isomer, {3R,<br>
5R)-isomer, and (3S, 5S}-isomer, in addition to the objective (3R,<br>
5S) -isomer.	As       an       example       of<br>
(E)-7-[2-cyclopropyl-4-{4-fluorophenyl)-quinolin-3-yl]-3,5-dih ydroxyhept-6-enoic acid esters {hereinafter, abbreviated as "DOLE") , the structural formula is as follows.<br>
3s, 5R-D0LE and 3R, 5S-D0LE are syn-isomers of DOLE, and 3S, 5S-D0LE and 3R, 5R-D0LE are anti-isomers of DOLE.<br>
In the examples, the purity of the [3R, 5S)-isomer which is<br><br>
the objective product is expressed by the excess enantiomer ratio. Specifically, the excess enantiomer ratio is represented by ( (3R, 5S)-isomer- (3S, 5R)-isomer) / [[3R, 5S)-isomer + (3S, 5R)-isomer)<br><br>
xohept-6-enoic acid ethyl ester (hereinafter, abbreviated as DOXE) In a 500-ml four-neck flask equipped with a stirrer, a dropping funnel, and a thermo-meter, 5.02 g {11.22 mmol) of (E)-7-[2-cyclopropyl-4-(4-fluorophenyl)-quinolin-3-yl]-5-hydroxy-3-oxohept-6-enoic acid ethyl ester (hereinafter, abbreviated as 5-MOLE) and 420 mL of acetone were added and stirred.<br><br>
Then, 10.5 mL of a prepared Jones oxidizing agent (i.e., a reagent obtained by mixing 3 mL of a concentrated sulfuric acid and 3.35 g of chromium oxide together, followed by diluting up to 25 mL with water) was dropped at O'C in 20 minutes, and was then stirred under ice cooling for 2 hours, followedby gradually adding lOmL of methanol to terminate the reaction. Subsequently, a reaction mixture solution was placed at reduced pressure to allow acetone to be distilled off, followed by the addition of 250 mL of ethyl acetate. The resulting solution was washed twice with 60 mL of saturated sodium bicarbonate aqueous solution, and was then washed twice with 60 mL of a brine to be extracted and separated, followed by drying an ethyl acetate solution with anhydrous magnesium sulfate. Subsequently, the solvent was distilled off, and purification was performed using a silica gel column chromatography (an eluting solvent; hexane: ethyl acetate =2 : 1), resulting in 3.03 g of an entitled compound (yield rate: 60.6%).<br>
^H-NMR (300MHz, CDCI3, 5 ppm) : 7.79-7.19 (3H, m) , 7.?1 (IH, d) , 6.03 (IH, d), 5.51 (IH, s) , 4.21 (2H, q) , 3.40 (2H, s) , 2.35-2.40 (IH, m) , 1.39-1.41 (2H, m), 1.28 (3H, t), 1.07-1.09 (2H, m) .<br><production example>       Synthesis       of 5S-(E)-7-[2-cyclopropyl-4-(4-fluorophenyl}-quinolin-3-yl]-5-hy droxy-3-oxo-hept-6-enoic  acid  ethyl   ester   (hereinafter, abbreviated as 5S-M0LE)<br><br>
In a Schlenk tube introduced with nitrogen gas after being<br>
heated and dried at a reduced pressure, 0.87 g (3.3 mmol) of<br>
(S)-2-[N-(3-methyl-5-tert-butyl	salicylidene)<br>
amino]-3-methyl-l-butanol, 5 ml of methylene chloride, and 0.63 ml (6.0 mmol) of titanium tetraethoxide were added, and stirred and mixed at room temperature for 1 hour. After cooling the Schlenk tube down to -50°C, 0.95 g (3.0 mmol) of (E)-3-[2-cyclopropyl-4-(4-fluorophenyl)-quinolin-3-yl]-prop-2-en-l-al dissolved in 2 ml of methylene chloride was dropped. After stirring the resultant for 5 minutes, 0.51 g (6 mmol) of diketene was further added, and stirred for 22 hours while keeping the temperature at -50°C for the reaction. The resulting reaction mixture solution was added in a mixture solution of 25 ml of methylene chloride and 25 ml of a 0,24M sodium bicarbonate aqueous solution, and was vigorously stirred for 2 hours at room temperature to obtain a two-layer solution. The resulting two-layer solution was separated. Awater layer was extracted twice with 10 ml of methylene chloride. The methylene chloride layer and the methylene chloride extract were combined together, resulting in a methylene chloride solution. The methylene chloride solution was dried with anhydrous magnesium sulfate and the solvent was distilled off, followed by purification with a silica gel column chromatography (an eluting solvent; hexane : ethyl acetate =3 : 2), resulting in 0.75 g of 5S-(E)-7-[2-cyclopropyl-4-(4-fluorophenyl)-quinolin-3-yl]-5-hy<br><br>
droxy-3-oxo-hept-6-enoic acid ethyl ester (5S-M0LE)  {optical purity:       73%      e.e.,       and      yield      for (E)-3-[2-cyclopropyl-4-{4-fluorophenyl)-quinolin-3-yl]-prop-2-en-l-al: 56%).<br><example> Isolation of carbonyl reductase<br>
Ogataea minuta var. nonfermentans IFO 1473 strain was cultivated in 20 liters of YMmedium {24 g glucose, 3 g yeast extract, 3 g malt extract, 5 g/1 peptone, pH 6.0) and bacterial cells were prepared by centrifugation. 3 00 g out of the obtained wet bacterial cells were suspended in 50 mM potassium phosphate buffer (pH 7.0, 1 itiM DTT} and disrupted in Dyno-Mill (manufactured by Dyno-Mill) . Thereafter, the bacterial cell residue was removedby centrifugation to obtain a cell-free extract. 12-w/v% Polyethylene glycol (PEG) 6000 was added to the cell-free extract and the resultant was centrifuged to obtain a supernatant. The supernatant was added to DEAE-Sepharose 6FF (2.6 cm x 28 cm) equilibrated with a standard buffer (10 mM potassium phosphate buffer (pH7.0), ImMDTT). After a column was washed with the standard buffer, it was further washed with a standard buffer that contains 0.07 M sodium chloride and then gradient elutionwithO . 07 to 0 . ISMsodiumchloridewas performed. The eluted fractions were collected and measured for carbonyl reductase activity.<br>
Carbonyl reductase activity was measured as to the following<br><br>
composition of reaction mixture, that is, 200 pi of an enzyme activity measuring fraction to which 10 pi of an aqueous solution of 0.1 mM potassium hydroxide containing 8 mMNADPH, 25 pi of 0.1 M phosphate buffer (pH 7.0) , and 10 pi of 20 g/1 DOXE (DMSO solution) were added, the reaction mixture was shaken overnight at 30°C for reaction. The activity was detected by the following method. That is, 0.5 ml of ethyl acetate was added to the reaction mixture after completion of the reaction and vigorously mixed and separated into an organic layer and a water layer by centrifugation. The organic layer was transferred in another vessel and the solvent was distilled by a condensation centrifuge. The dried product was dissolved in 0.01 ml of ethyl acetate and thin layer chromatography (TLC) was performed. For the TLC, silica gel plate (Silica Gel 60 F254 manufactured by Merck &amp; Co.) was used and hexane/ethyl acetate = 1/1 was used as development solvent. After completion of the development, the product was confirmed by using an UV lamp. The compound (I) had Rf = 0.7 to 0.86, the compounds (II) and (III) had Rf = 0.54 to 0.61, and the compound (IV) (wherein compound R = an ethyl group: DOLE) had Rf =   0.33.<br>
The activity of carbonyl reductase showed a peak in the vicinity of 0 .13M sodium chloride. The eluted active fractions were collected and ammonium sulfate was added thereto until a final concentration of 0.3 M was reached. Then, the resultant was added to octyl-Sepharose CL4B (0.8 cm x 20 cm) equilibrated with a standard<br><br>
buffer (lOmMpotassiumphosphatebuffer (pHV.O) , ImMDTT) containing 0.3 M aminonium sulfate, and concentration gradient elution of 0.3 to 0 M of ammoniuin sulfate was performed. The activity of carbonyl reductase showed a peak in the vicinity of 0 M ammonium sulfate.<br>
The eluted peak was collected and desalted to the concentration in the standard solution by ultrafiltration and concentrated. Thereafter, the resultant was added to MonoQ HR5/5 equilibrated with the same buffer and washed with the same buffer, and then with the same buffer but contains 0.05 M sodium chloride, followed by performing 0.05 M to 0.25 M sodium chloride concentration gradient elution. The active fraction of carbonyl reductase was eluted in 0.17 M sodium chloride fraction so that the fraction was collected and concentrated. The concentrated enzyme solution was subjected to gel filtration using Superdex 2 00 HRlO/30 with the standard buffer containing 0.15 M sodium chloride. The active fraction obtained by the gel filtration had a molecular weight of about 107,000 Da.<br>
Results of analysis of the above-mentioned active fraction on polyacrylamide gel electrophoresis (SDS-PAGE) indicated a substantially single band and the molecular weight thereof wa5 about 27,000 Da.<br>
Further, the reduction activity of the enzyme was measured by reacting the enzyme with various ketones and aldehyde, and measuring initial rate of NADPH decrease, specifically, inareaction mixture that contains 5 [al of an isopropanol solution of 100 mM<br><br>
substrate, 10 |il of aqueous solution of 8 mMNADPH and 0 .1 mM potassium hydroxide, and 0.1 M phosphate buffer (pH 7.0) to make a reaction mixture of a total volume of 250 ^1. Table 1 shows specific activities of the enzyme assuming that a variation of optical absorption in 2,2,2-trifluoroacetophenone is defined as 100.<br><br><br>
The optimijin pH of the enzyme was determined by measuring 2,2,2-trifluoroacetophenone reducing activity in a plurality of buffers differing in pH from each other. Specifically, 0.1 M potassium buffers, sodium acetate buffers, and citrate buff ers each differing in pH were prepared and activities were measured. The optimum pH of the reaction was 5.0 to 6.5.<br>
Further, only the reaction temperature was varied and 2,2,2-trifluoroacetophenone reducing activity was measured to measure the optimum temperature of the action of the carbonyl reductase. As a result, the optimum temperature was 60 to 70°C.<br><br>
The pH stability of the enzyme was examined by preparing a plurality of potassium phosphate buffers, sodium acetate buffers, and citrate buffers differing in pH and incubating each of them for 30 minutes at 30°C to measure the residual activity. As a result, the highest stability was confirmed at pH 5.5 to 6.5.<br>
The temperature stability of the enzyme was examined by allowing to stand in 20 mM phosphate buffer at pH 7 . 0 at temperatures of aC'C, 40'C, sec, 60°C, 70°C, and 80°C for 10 minutes and measuring 2,2,2-trifluoroacetophenone reducing activity. The results indicated that the enzyme of the present invention showed a residual activity of 79% or more up to 40°C and rapidly deactivated at 50°C or more, and lost the activity completely at 70°C.<br>
Further, the enzyme was treated in various reagents at 30°C for 10 minutes and then 2,2,2-trifluoroacetophenone reducing activity was measured. As a result, the activity of enzyme was considerably inhibited in the presence of mercury (I) ion or lead (II} ion such as mercury nitrate and lead chloride.<br><example> Analysis of partial amino acid sequence of carbonyl reductase<br>
The fraction containing the carbonyl reductase obtained in Example 1 was desalted, concentrated and then treated with lysylendopeptidase at 30°C overnight. The digested peptide was subjected to acetonitrile gradient elution in 0.1% trifluoroacetic<br><br>
acid {TFA) using reversed-phase HPLC (uRPC C2/C18 manufactured by Amersham-Pharmacia, 46 mm x 100 mm.) to resolve and fractionate a peptide. Three fractionated peptide peaks were named Frac20, 27, and 30 and each of them was analyzed for amino acid sequence by the Edman method. The respective amino acid sequences of Frac20, 27, and 30 are shown in SEQ ID NOs: 3, 4, and 5. Similarly, the fraction containing the carbonyl reductase was analyzed for N-terminal amino acid by the Edman method after desalting and the result is shown in SEQ ID NO: 6.<br><example> Sequence analysis of the DNA of the present invention derived from Ogataea minuta var. nonfermentans IFO 1473 strain and Making of transformant<br>
Ogataea minuta var. nonfermentans IFO 1473 strain was cultivated in YMmedium and the bacterial cell thereof was prepared.<br>
Purification of total RNA from the bacterial cells was performed using Mag Extractor (manufactured by Toyobo). First-Strand cDNA was synthesized based on the purified RNA using SuperScriptll (manufactured by Life Technology). The method was performed according to the attached manual.<br>
Four kinds in total of primers, i.e., a sense degenerate primer based on the N-terminal amino acid sequence obtained in Example 2, and antisense degenerate primers based on SEQ ID NOs; 3, 4, and 5, respectively, were synthesized. The nucleotide sequences are<br><br>
shown in SEQ ID NOs: 7, 8, 9, and 10, respectively. Primers were selected based on the coinbinations (three types) of N-terminal and three partial amino acid sequences and PCR was performed using 50 1J1 of a reaction mixture containing each 400 pmol of primer, 0.4 mmol dNTP, 1 ul of cDNAs of Ogataea minuta var. nonfermentans IFO 1473 strain, 5 yl of lOx buf f er f or TaKaRa Taq (Manuf acturedby Takara Shuzo), and 2 units of heat-resistant DNA polymerase (trade name "TaKaRa Taq", manuf acturedby Takara Shuzo) , by conducting 30 cycles of denaturation (94°C, 30 seconds), anneal (50°C, 30 seconds), and elongation (72°C, Iminute) bymeans of RTC-200 Peltier Thermal Cycler (manuf actured by M J Research) . Analysis ofapartofthe PCR reaction mixture by agarose gel electrophoresis indicated that bands considered to be specific were detected in the combination of the primer described in SEQ ID NO: 7 and the primer described in SEQ ID NO: 8, in the combination of the primer described in SEQ ID NO: 7 and the primer described in SEQ ID NO: 9, and in the combination of the primer described in SEQ ID NO: 7 and the primer described in SEQ ID NO: 10.<br>
The three kinds of DNA fragments obtained as described above were subjected to agarose gel electrophoresis and objective bands were cut out, purified by a Gel Extraction kit (manufactured by QIAGEN) and recovered. The obtained DNA fragments were subjected to TA cloning by a pGEM-T Vector System [manufactured by Promega) and used for transforming Escherichia coli DH5a strain (manufactured<br><br>
byToyobo) . The trans formants were grown on a mediumplate containing 1% bactotriptone, 0.5% bacto yeast extract and 1% sodium chloride (hereinafter, referred to as "LB medium") to which ampicillin (100 |ig/ml) was added and the medium was converted to the plate by addition of 1.5% bacto agar. Using some colonies, colony direct PCR was performed with 3EQ IDNOs : 11 and 12 synthesized based on the sequences of the vectors as primers and the size of the inserted fragment was confirmed. Colonies assumed to have the objective DNA fragment inserted therein were cultivated in liquid LB medium containing 100 jag/ml ampicillin and plasmids were purified by Mini-Prep (manufactured by QIAGEN) . The plasmids obtained by using the , combination of the primer described in SEQ ID NO: 7 and the primer described in SEQ ID NO: 8, in the combination of the primer described in SEQ ID NO: 7 and the primer described in SEQ ID NO: 9, and in the combination of the primer described in SEQ ID NO: 7 and the primer described in SEQ ID NO: 10 were named phir21-23, phir21-25, and phir21-27, respectively.<br>
The nucleotide sequence of the inserted DNA was analyzed by the diterminator method using the purified plasmids . The nucleotide sequences of respective inserted fragment portions of phir21-23, phir21-25, and phir21-27 thus determined are shown in SEQ ID NOs: 13, 14, and 15, respectively.<br>
Total RNAs prepared from Ogataea minuta  var.   nonfermentans IFO 1473 strain was subjected to the 5' RACE method and 3' RACE<br><br>
method using primers designed based on the above-mentioned nucleotide sequences.<br>
The 5' RACE method was performed using the primers having nucleotide sequences described in SEQ ID NOs: 16 and 17 as gene specific primers (hereinafter, referred to as "GSPs") 1 and 2 in 5' RACE System for Rapid Amplification of cDNA Ends, ver2.0 (manufactured by Life Technology) and then the resultant was used as templates and the 5' RACE method was again performed using the primer having the nucleotide sequence described in SEQ ID NO: 18 as GSP.<br>
The 3' RACE method was performed using the primer having the nucleotide sequence shown in SEQ ID NO:19 as GSP and also the 3' RACE System for Rapid Amplification of cDNA Ends (manufactured by Life technology) . Note that each of the RACE methods was carried out based on the manual attached.<br>
The obtained DNA fragments were subjected to TA cloning by a pGEM-T Vector System (manufactured by Promega) and used for transforming Escherichia coli DH5a strain (manufactured by Toyobo) . The transformants were grown on an LB medium plate containing 100 Ug/ml ampicillin. Using some colonies, colony direct PCR was per formed with SEQ ID NOs : 11 and 12 synthesized based on the sequences of the vectors as primers and the size of the inserted fragment was confirmed. Colonies assumed to have the objective DNA fragment inserted therein were cultivated in liquid LB medium containing<br><br>
10 0 \ig/ml ample ill in and plasmids were purified by Mini-Prep (manufactured by QIAGEN) and then analyzed for DNA nucleotide sequence by the diterminator method.<br>
ORF search was performed on the 5' upstream nucleotide sequence and 3' downstream nucleotide sequence of phir21-25 obtained in the 5' RACE and 3' RACE as well as DNA sequence prepared based on the nucleotide sequence information described in SEQ ID NO:14 using Genetyx (manufactured by Software Development Co., Ltd.). The DNA sequence of the present invention and the amino acid sequence of the protein encoded thereby were provisionally determined to be the nucleotide sequence and amino acid sequence shown in SEQ ID NO: 2 0.<br>
Results of homology search of the above-mentioned polypeptide performed on DDBJ using the BLAST program with the amino acid sequence shown in SEQ ID NO: 20 indicated that among the known proteins, the one that showed the highest homology was the probable short chain dehydrogenase {T41540) of Schizosaccharomyces pombe, which showed 37.4% homology. Subsequently, the nucleotide sequence described in SEQ ID NO: 21 and the nucleotide sequence described in SEQ ID NO: 22 were synthesized as primers for cloning based on the sequence described in SEQ ID NO: 20, and PCR was performed using 50 yl of a reaction mixture containing 50 pmol of each primer, 200 nmol of dNTP, 1 pi of cDNA of Ogataea minuta var. nonfermentans IFO 1473 strain,  5 ]il    of lOx buffer for KOD-DNA polymerase<br><br>
(manufactured by Toyobo) , 2.5 units of KOD-DNA polymerase (manufactured by Toyobo), by conducting 30 cycles of denaturation {96°C, 30 seconds), anneal {54°C, 30 seconds) , and elongation (74°C, 1 minute) by means of RTC-200 Peltier Thermal Cycler (manufactured by MJ Research) . Analysis of a part of the PCR reaction mixture by agarose gel electrophoresis indicated that bands considered to be specific could be detected.<br>
The band detected as described above was recovered by using a QIAGEN Gel Extraction kit (manufactured by QIAGEN) . The recovered DNA fragment was digestedwith restriction enzymes Hindi 11 andEcoRI, and subjected to agarose gel electrophoresis. The objective band was cut out, purified again by using a QIAGEN Gel Extraction kit (manufactured by QIAGEN) and then recovered. The obtained DNA fragment was ligated to pKK223-3 (manufactured by Pharmacia) digested with EcoRI and Hindlll using a Takara Ligation kit and Escherichia   coli   JM109 strain was transformed therewith.<br>
The transformants were grown on an LB medium plate containing ampicillin (50 yg/ml). Using some colonies, colony direct PCR was performed with the primer having the nucleotide sequence described in SEQ ID NO: 23 and the primer having the nucleotide sequence described in SEQ ID NO: 24 synthesized based on the sequences of the vectors as primers and the size of the inserted fragment was confirmed. Transformants assumed to have the obj ective DNA fragment inserted therein were cultivated in liquid LB medium containing<br><br>
50 ug/ml ampicillin and plasmids were purified by using Qiagen 500 (manufactured by QIAGEN) to obtain pKK223-30CRl.<br>
Analysis of nucleotide sequence of the DNA inserted in the plasmid by the diterminator method indicated that the inserted DWA fragment had a nucleotide sequence consisting of a gene having the nucleotide sequence represented by SEQ ID NO: 2 that has 6 bases for cloning added to the 5' upstream thereof and double stop codon (TAGTAAT) derived from Ogataea minuta var. nonfermentans IFO 1473 strain and 6 bases for cloning added to the 3'-downstream thereof.<br>
The nucleotide sequence of the DNA fragment inserted into pKK233-30CRl is shown in SEQ ID NO: 25 and the amino acid sequence of the protein encoded by the DNA fragment is shown in SEQ ID NO: 1.<br><example> Production of DOLE from DOXE using Escherichia  coli transformed with the DNA of the present invention<br>
The transformant obtained in Example 4 was grown in LB medium containing   ampicillin   {50   (jg/ml),   0.1   mM   isopropyl<br>
p-D-thiogalactopyranoside (IPTG) at 37°C for 17 hours and 2 ml of the obtained bacterial cell broth was centrifuged to collect the bacterial cells, which were suspended in 180 yl of 100 mM potassium phosphate buffer (pH 7.0). Thereafter, reducing activity of the transformant was confirmed by using (E)-7-[2-cyclopropyl-4-(4-fluorophenyl)-quinolin-3-yl]-3,5-dio<br><br>
xohept-6-enoic acid ethyl ester (DOXE) as a substrate.<br>
After addition of 10 pi of 2 g/1 NADP"^ (manufactured by Oriental Yeast) and 1 ul of toluene, the above-mentioned bacterial cell suspension was stirred with a vortexmixer for 5 minutes. After addition of 10 ]il of 50% glucose, 10 yl of a glucose dehydrogenase solution (manufactured by Amano Pharmaceutical; 25 U/ml), and 50 \il of a 20 g/1 DMSO solution of the above-mentioned substrate (corresponding to 1 mg of substrate), the mixture was shaken at 40°C for 20 hours for reaction. After dilution of the reaction suspension after completion of the reaction by addition of 1.75 ml of acetonitrile, the resultant was centrifuged and the product in the supernatant was measured by using high performance liquid chromatography (HPLC). The conditions of HPLC were as follows. Column: MCIGEL CHP2MGM (manufactured by Mitsubishi Chemical Corporation)<br>
Eluting solution; Methanol/acetonitrile/water/ phosphoric acid = 800/100/100/1 Flow rate: 0.6 ml/min Detection: UV 254 nm Temperature: 60°C<br>
As a result, 319 yg yields and 31.9% yield rate were achieved.<br>
Further, for measurement of the optical purity, after terminating the reaction, 0.5 mL of ethyl acetate was added in the reaction solution and was mixed therewith vigorously, followed by<br><br>
separation into an organic layer and a water layerbyacentrifugation. The organic layer was transferred to another vessel. A solvent was distilled off with a condensation centrifuge. Then^ the dried product was dissolved in 0.01 mL of ethyl acetate, and was then subjected to thin layer chromatography (TLC) . The TLC used was a silica gel plate {silica gel 60 F254 manufactured by Merck &amp; Co.) , and development solvent used was of hexane/ethyl acetate ^ 1/1.<br>
After terminating the development, the product was confirmed with an UV lamp. As for the compound (I), Rf = 0.76 to 0.86. As for compounds (II) and (III), Rf = 0.54to0,61. As for the compound (IV) (wherein the compound R = ethyl group: hereinafter abbreviated as DOLE), Rf = 0.33. A spot of the DOLE on the TLC was scraped and eluted with 0.25 mL of isopropanol. After the centrifugation, a supernatant was subjected to a high performance liquid chromatography (HPLC) to analyze its optical purity and the concentration of a TLC-scraped-off sample.<br>
The following are the conditions of the HPLC. Column: Daicel CHIRALCEL AD<br>
Eluting solution: Hexane/ethanol/trifluoroacetate - 900/100/1 Flow rate: 1 ml/min Detection: UV 254 nm Temperature: Room temperature<br>
This resulted in 10 0%e- e. optical purity, and 0.6% anti-isomer.<br>
Further, bacterial cells of Escherichia coli having theplasmid<br><br>
pKK223-3 that did not contain the gene cultivated overnight in LB medium to which 0.1 niM IPTG was added was allowed to react in the same manner. However, the above-mentioned product was not detected.<br><exainple> Production of DOLE from 5S-M0LE using Escherichia coli transformed with the DNA of the present invention<br>
The transformant obtained in Example 4 was grown in LB medium containing   ampicillin   {50   ]iq/ml),        0,1   mM   isopropyl<br>
[3-D-thiogalactopyranoside (IPTG) at 37'C for 17 hours and 2 ml of the obtained bacterial cell broth was centrifuged to collect the bacterial cells, which were suspended in 180 pi of 100 mM potassium phosphate buffer (pH 7.0) . Thereafter, reducing activity of the transformant was confirmed by using 5S-(E)-7-[2-cyclopropyl-4-(4-fluorophenyl)-quinolin-3~yl]-5-hy droxy-3-oxo-hept-6-enoic acid ethyl ester (5S-M0LE) as a substrate by the following method.<br>
After addition of 10 )il of 2 g/lNADP"" (manufactured by Oriental Yeast) and 1 ^il of toluene, the above-mentioned bacterial cell suspension was stirred with a vortexmixer for 5 minutes. After addition of 10 ]il of 50% glucose, 10 yl of a glucose dehydrogenase solution (manufactured by Amano Pharmaceutical; 25 U/ml), and 50 Ul of a 20 g/1 DMSO solution of the above-mentioned substrate {corresponding to 1 mg of substrate), the mixture was shaken at 40°C   for 20 hours for reaction. After dilution of the reaction<br><br>
suspension after completion of the reaction by addition of<br>
acetonitrile, the resultant was centrifuged and the product in the<br>
supernatant was measured by using high performance  liquid<br>
chromatography (HPLC) .  The conditions of HPLC were as follows.<br>
Column:  Cosmosil 5C18MS-II, 4 . 6 x 250 mm [manufactured by Nacalai<br>
Tesque, Inc.)<br>
Elating solution: Kethanol/acetonitrile/water/phosphoric acid =<br>
350/150/500/1<br>
Flow rate: 0.6 ml/min<br>
Detection; UV 254 nm<br>
Temperature: 50°C<br>
This resulted in 807 ^g yields, 80.7% yield rate, and 15.3% anti-isomer.<br>
Further, for measurement of the optical purity, after terminating the reaction, 0.5 mL of ethyl acetate was added in the reaction solution and was mixed therewith vigorously, followed by separation into anorganic layer and a water layerby acentrifugation. The organic layer was transferred to another vessel, A solvent was distilled off with a condensation centrifuge. Then, the dried product was dissolved in 0.01 mL of ethyl acetate, and was then subjected to thin layer chromatography {TLC) . The TLC used was a silica gel plate (silica gel 60 F25J manufactured by Merck &amp; Co.), and development solvent used was of hexane/ethyl acetate = 1/1.<br>
After terminating the development, the product was confirmed<br><br>
with an UV lamp. As for the compound (I), Rf = 0.76 to 0.86. As for compounds (11) and (III), Rf = 0.5-5 to 0.61. As for the compound (IV) (wherein the compound R = ethyl group: hereinafter abbreviated as DOLE) , Rf = 0.33. A spot of the DOLE on the TLC was scraped and eluted with 0.25 mL of isopropanol. After the centrifugation, a supernatant was subjected to high performance liquid chromatography (HPLC) to analyze its optical purity and the concentration of a TLC-scraped-off sample.<br>
The following are the conditions of the HPLC. Column: Daicel CHIRALCEL AD Eluting solution; Hexane/ethanol = 95/5 Flow rate: 1 ml/min Detection: UV 254 nm Temperature: 50°C<br>
This resultedin97%e .e . optical purity, andlS .2% anti-isomer.<br>
Reaction of Escherichia coli having a plasmid pKK223-3 not containing the gene and an overnight cultivated bacterial cells with an LB medium having O.lmM IPTG added therein was attempted in the same way but the above-mentioned product was not recognized.<br>
INDUSTRIAL APPLICABILITY<br>
Aproduct ion process that canprovide optically active alcohols, which are industrially useful compounds as intermediate materials for drugs and pesticides, at high optical purity and in high yield<br><br><br>
WE CLAIM:<br>
1.	A polypeptide, comprising an amino acid sequence<br>
of (A) or (B) helow:<br>
(A) an amino acid sequence described in SFQ ID HO: 1; and (Bi an amino acid sequence of a polypeptiede having a carbonyl, reductase activity, the amino acid sequence has 80% or mare homology CO the amino acid sequence described in SEQ ID NO: 1.<br>
2.	A DNA, comprising a nucleotide sequence of any one<br>
of (a) to (c) below;<br>
(a)	a nucleotide sequence encoding a polypeptide consisting of an amino acid sequence described in SEQ ID HD: I;<br>
(b)	a nucleotide sequence described in SEQ ID NO; 2;   and<br>
(c) a nucleotide sequence that hybridizes with the nucleotide sequence described in SEQ ID NO; 2 or a complementary nucleotide sequence thereof or a part of these sequences under stringent conditions, the nucleotide sequence encoding a polypeptide having a carbonyl reductase activity.<br>
3.	A recombinant DNA obtained by mcorporating the DNA<br>
according to claim 2 into a vector.<br>
1, A trans formed microbial, cell having a recombinant DNA according to claim 3.<br>
5.	A transformed microbial cell obtained by incorporating a DNA according to claim 2 into a chromosomal DNA,<br>
6.	A process for producing a compound represented by the following formula (IV)<br><br><br>
from the group consisting of a transformed microbial cell, according to claim 4 or 5 , a culture broth of the transformed microbial cell, and a treated product of the transformed microbial cell to asymmetrically reduce the carbonyl group-containing compound.<br><br>
7, The process according to claim 6, wherein the compounds represented by formulae (III and (III) consist of optically active forms represented fay the following formula (II')<br><br><br></exainple></example></example></example></example></production></patent></patent></patent></non-patent></non-patent></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjMyMC1jaGVucC0yMDA0IGFic3RyYWN0LWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">2320-chenp-2004 abstract-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjMyMC1jaGVucC0yMDA0IGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2320-chenp-2004 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjMyMC1jaGVucC0yMDA0IGNsYWltcy1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">2320-chenp-2004 claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjMyMC1jaGVucC0yMDA0IGNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2320-chenp-2004 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjMyMC1jaGVucC0yMDA0IGNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2320-chenp-2004 correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjMyMC1jaGVucC0yMDA0IGNvcnJlc3BvbmRlbmNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">2320-chenp-2004 correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjMyMC1jaGVucC0yMDA0IGRlc2NyaXBpdGlvbihjb21wbGV0ZWQpLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">2320-chenp-2004 descripition(completed)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjMyMC1jaGVucC0yMDA0IGRlc2NyaXBpdGlvbihjb21wbGV0ZWQpLnBkZg==" target="_blank" style="word-wrap:break-word;">2320-chenp-2004 descripition(completed).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjMyMC1jaGVucC0yMDA0IGRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">2320-chenp-2004 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjMyMC1jaGVucC0yMDA0IGZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">2320-chenp-2004 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjMyMC1jaGVucC0yMDA0IGZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">2320-chenp-2004 form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjMyMC1jaGVucC0yMDA0IGZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">2320-chenp-2004 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjMyMC1jaGVucC0yMDA0IGZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">2320-chenp-2004 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjMyMC1jaGVucC0yMDA0IGZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">2320-chenp-2004 form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjMyMC1jaGVucC0yMDA0IHBjdCBzZWFyY2ggcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">2320-chenp-2004 pct search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjMyMC1jaGVucC0yMDA0IHBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">2320-chenp-2004 pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjMyMC1jaGVucC0yMDA0IHBldGl0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">2320-chenp-2004 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="229707-novel-dendritic-polymers-having-biphosphonic-terminations-derivatives-thereof-method-for-preparing-them-and-their-use.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="229709-a-recombinant-subgroup-b-adenovirus-particle-comprising-an-adenovirus-genome.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>229708</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2320/CHENP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>19-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>13-Oct-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MITSUBISHI CHEMICAL CORPORATION</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>33-8, Shiba 5-chome, Minato-ku, Tokyo 108-0014,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>HIRAOKA, Hirotoshi</td>
											<td>C/O MITSUBISHI CHEMICAL CORPORATION, 1000, Kamoshida-cho, Aoba-ku, Yokohama-shi, Kanagawa 227-8502,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>UEDA, Makoto</td>
											<td>C/O MITSUBISHI CHEMICAL CORPORATION, 1000, Kamoshida-cho, Aoba-ku, Yokohama-shi, Kanagawa 227-8502,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>HARA, Mari</td>
											<td>C/O MITSUBISHI CHEMICAL CORPORATION, 1000, Kamoshida-cho, Aoba-ku, Yokohama-shi, Kanagawa 227-8502,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/53</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/JP2003/03262</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-03-18</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>2002-75921</td>
									<td>2002-03-19</td>
								    <td>Japan</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/229708-novel-carbonyl-reductase-gene-encoding-the-same-and-process-for-producing-optically-active-alcohols-using-the-same by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:25:25 GMT -->
</html>
