<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/229730-glyphosate-resistant-class-i-5-enolpyruvylshikimate-3-phosphate-synthase-epsps by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:26:44 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 229730:GLYPHOSATE RESISTANT CLASS I 5-ENOLPYRUVYLSHIKIMATE-3-PHOSPHATE SYNTHASE (EPSPS)</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">GLYPHOSATE RESISTANT CLASS I 5-ENOLPYRUVYLSHIKIMATE-3-PHOSPHATE SYNTHASE (EPSPS)</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>ABSTRACT The compositions and methods disclosed herein provide novel DMA molecules that encode glyphosate resistant EPSPS proteins and plants containing these new proteins. The plants that express the new EPSPS proteins are themselves tolerant to the herbicidal effects of glyphosate.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
FIELD OF THE INVENTION<br>
This invention relates to plant molecular biology and plant genetic engineering for herbicide resistance and, more particularly, to class I 5-enolpyruvylshikimate-3-phosphate synthases modified for glyphosate resistance. Plant genetic engineering methods are used to modify class [ 5-enolpyruvylshikimate-3-phosphate synthase DNA and the encoded proteins, and to transfer these molecules into plants of agronomic importance. More specifically, the invention comprises DNA and protein compositions of glyphosate resistant 5-eno]p\ruv\lshikimaie-?-phospluie ssnlhases. and lo the plants containing these compositions.<br>
BACKGROUND OF THE INVENTION<br>
N-phosphonomethylglycine. also known as glyphosate. is a well-known herbicide that has activity on a broad spectrum of plant species. Glyphosate is the active ingredient of Roundup® (Monsanto Co., St Louis. MO), a herbicide having a long history of safe use and a desirably short half-life in the environment. When applied to a plant surface, glyphosate moves systemically through the plant. Glyphosate is phytotoxic due to its inhibition of the shikimic acid pathway, which provides a precursor for the synthesis of aromatic amino acids. Glyphosate inhibits the class I 5-enolpyruvylshikimate-3-phosphate synthase (EPSPS) found in plants and some bacteria. Glyphosate tolerance in plants am be achieved by the expression of a modified class I EPSPS that has lower affinity for glyphosate. yet still retains its catalytic activity in the presence of glyphosate (U.S. Patent 4.535.060. and 6.040,497). "Tolerant" or "tolerance" refers to a reduced effect of an agent on the growth and development, and yield of a plant, in particular, tolerance to the phytotoxic effects of a herbicide, especially glyphosaie.<br><br>
Enzymes, such as, class II EPSPSs have been isolated from bacteria that are naturally resistant to glyphosate and when the enzyme is expressed as a transgene in plants provides glyphosate tolerance to the plants (U.S. Patent 5,633,435 and 5,094,945). Enzymes that degrade glyphosate in plant tissues (U.S. Patent 5,463,175) are also capable of conferring plant tolerance to glyphosate. DNA constructs that contain the necessary genetic elements to express the glyphosate resistant enzymes or degradative enzymes create chimeric transgenes useful in plants. Such transgenes are used for the production of transgenic crops that are tolerant to glyphosate, thereby allowing glyphosate to be used for effective weed control with minimal concern of crop damage. For example, glyphosate tolerance has been genetically engineered into corn (U.S. Patent 5,554,798), wheat (Zhou et al. Plant Cell Rep, 15:159-163, 1995), soybean (WO 9200377) and canola (WO 9204449). The transgenes for glyphosate tolerance and transgenes for tolerance to other herbicides, for example the bar gene (Sh.Bar) may be included in DNA constructs for use as a selectable marker for plant transformation (present invention pMON81519; and Toki et al. Plant Physiol., 100:1503-1507, 1992; Thompson et al. EMBO J. 6:2519-2523, 1987; phosphmothricin acetyltransferase DeBlock et al. EMBO J., 6:2513-2522, 1987, glufosinate herbicide) are also useful as selectable markers or scorable markers and can provide a useful phenotype for selection of transgenic plants when the marker gene is linked with other agronomically useful traits.<br>
Development of herbicide-tolerant crops has been a major breakthrough in agriculture biotechnology as it has provided farmers with new weed control methods. One enzyme that has been successfully engineered for resistance to its inhibitor herbicide is class I EPSPS. Variants of class I EPSPS have been isolated (Pro-Ser, U.S. Patent 4,769,061; Gly-Ala, U.S. Patent 4,971,908; Gly-Ala, Gly-Asp, U.S. Patent 5,310,667; Gly-Ala, Ala-Thr, U.S. Patent 5,8866,775) that are resistant to glyphosate.  However, many EPSPS variants<br>
3<br><br>
either do not demonstrate a sufficiently high K, for glyphosate or have a Km for phosphoeno] pyruvate (PEP) too high to be effective as a glyphosate resistance enzyme for use in plants (Padgette ct. al, In "Herbicide-resistant Crops", Chapter 4 pp 53-83. ed. Stephen Duke, Lewis Pub, CRC Press Boca Raton, Fl 1996). However, one class I EPSPS variant, T1021/P106S (TIPS) that is operably linked to a heterologous promoter has been shown to provide glyphosate tolerance to transgenic maize plants (U.S. Patent No. 6,040,497). A glyphosate tolerant EPSPS has also been isolated from the weed Eleitsine sndica [WO 01/66704].<br>
There is a need in the field of plant molecular biology for a diversity of genes that can provide a positive selectable marker phenotype. In particular, glyphosate tolerance is used extensively as a positive selectable marker in plants and is a valuable phenotype for use in crop produclion. The stacking and combining of existing transgene traits with newly developed traits is enhanced when distinct positive selectable marker genes are used. The marker genes provide either a distinct phenotype, such as, antibiotic or herbicide tolerance, or a molecular distinction discernabie by methods used for DNA detection. The transgenic plants can be screened for the stacked traits by analysis for multiple antibiotic or herbicide tolerance or for the presence of novel DNA molecules by DNA detection methods. The present invention provides DNA and protein compositions of glyphosate resistant variant class I EPSP synthases. The present invention also provides DNA constructs useful in plants and transgenic plants that exhibit glyphosate tolerance.<br>
SUMMARY OF THE INVENTION<br>
In one aspect of the invention there is provided an isolated modified EPSPS DNA molecule encoding a glyphosate tolerant EPSPS protein having an isoleucine or leucine at position 102, and an amino acid at position 106 selected from the group consisting of threonine, glycine, cysteine, alanine, and isoleucine.   In another aspect of the  invention  is a DNA  construct that<br>
4<br><br>
comprises a promoter that functions in plant cells operably linked to a modified EPSPS DNA molecule encoding a glyphosate tolerant EPSPS protein having an isoleucine or leucine at position 102, and an amino acid at position 106 selected from the group consisting of threonine, glycine, cysteine, alanine, and isoleucine. In yet another aspect of the invention there is provided a transgenic plant than contains the DMA construct, wherein the transgenic plant is tolerant to glyphosate herbicide.<br>
In another aspect of the invention is a method of preparing a fertile transgenic plant comprising providing a plant expression cassette having a modified EPSPS gene encoding an EPSPS protein having isoleucine or leucine at position 102, and an amino acid at position 106 selected from the group consisting of threonine, glycine, cysteine, alanine, and isoleucine; and contacting recipient plant cells with the plant expression cassette under conditions permitting the uptake of the plant expression cassette by the recipient cells; and selecting the recipient plant cells that contain the plant expression cassette; and regenerating plants from the selected recipient plant cells; and identifying a fertile transgenic plant that is tolerant to glyphosate.<br>
In another aspect of the invention is a fertile glyphosate tolerant transgenic plant that contains a plant expression cassette having a modified plant EPSPS gene encoding an EPSPS protein having isoleucine or leucine at position 102, and an amino acid at position 106 selected from the group consisting of threonine, glycine, cysteine, alanine, and isoleucine that is crossed to another plant to provide progeny that are tolerant to glyphosate.<br>
In another aspect of the invention, there is provided a method for controlling weeds in a field of crop plants, wherein the field of crop plants is treated with an effective amount of a glyphosate containing herbicide and the crop plants contain a plant expression cassette having a modified EPSPS gene encoding an EPSPS protein having isoleucine or leucine at position 102, and an amino<br>
5<br><br>
acid at position 106 selected from the group consisting of threonine, glycine, cysteine, alanine, and isoleucine.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figure I shows polynucleotide and polypeptide sequence of maize EPSPS<br>
Figure 2 shows polypeptide alignment of class I plant and bacterial EPSP<br>
synthases.<br>
Figure 3 shows DNA construct map of pMON70461 (wild-type EPSPS).<br>
Figure 4 shows DNA primer sequences<br>
Figure 5 shows DNA construct map of pMON58452 (ZmTIPT variant)<br>
Figure 6 shows DNA construct map of pMON30167 (CP4 EPSPS)<br>
Figure 7 shows DNA construct map of pMON70472 (ZmTIPT variant)<br>
Figure 8 shows DNA construct map of pMON70475 (ZmTlPA variant)<br>
Figure 9 shows DNA construct map of pMON70467 (ZmPT variant)<br>
Figure 10 shows DNA construct map of pMON81519 (ZmTIPT variant)<br>
Figure II shows DNA construct map of pMON8154S (LsTiPA varianl)<br>
Figure 12 shows DNA construct map of pMON58491 (AtTIPA variant)<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The present invention is based, in part, on the construction of a glyphosate resistant EPSPS and utilizing DNA molecules that encode the EPSPS in a DNA construct to provide herbicide tolerance to transgenic plants expressing the glyphosate resistant EPSPS in its tissues. The following descriptions are provided to better define the present invention and to guide those of ordinary skill in the art in the practice of the present invention. Unless otherwise noted, terms are to be understood according to conventional usage by those of ordinary skill in the relevant art. Definitions of common terms in molecular biology may also be found in Rieger et al., Glossary of Genetics: Classical and Molecular, 5th edition, Springer-Verlag: New York, (1991); and Lewin, Genes V. Oxford University Press: New York, (1994). The nomenclature for DNA bases as set forth at 37 CFR §   1.822  is used. "Nucleic acid" refers to<br>
6<br><br>
deoxyribonucleic acid (DNA) and ribonucleic acid (RNA). The standard one-and three-letter nomenclature for amino acid residues is used.<br>
Methods of the present invention include designing EPSPS proteins that confer a glyphosate tolerant trait to the plant into which they are introduced. Polynucleotide molecules encoding proteins involved in herbicide tolerance are known in the art, and include, but are not limited to a polynucleotide molecule encoding 5-enolpyruvylshikimate-3-phosphate synthase (EPSPS, described in U.S. Patent 5,627,061, 5,633,435, and 6,040,497; Padgette et al. Herbicide Resistant Crops, Lewis Publishers, 53-85, 1996; and Penaloza-Vazquez, et al. Plant Cell Reports 14:482-487, 1995; and aroA (U.S. Patent 5,094,945) for glyphosate tolerance.<br>
"Glyphosate" refers to N-phosphonomethylglycine and its' salts. Glyphosate is the active ingredient of Roundup® herbicide (Monsanto Co.). Plant treatments with "glyphosate" refer to treatments with the Roundup® or Roundup Ultra® herbicide formulation, unless otherwise stated. Glyphosate as N-phosphonomethylglycine and its' salts (not formulated Roundup® herbicide) are components of synthetic culture media used for the selection of bacteria and plant tolerance to glyphosate or used to determine enzyme resistance in vitro biochemical assays. Examples of commercial formulations of glyphosate include, without restriction, those sold by Monsanto Company as ROUNDUP®, ROUNDUP® ULTRA, ROUNDUP® ULTRAMAX, ROUNDUP® CT, ROUNDUP® EXTRA, ROUNDUP® BIACT1VE, ROUNDUP® BIOFORCE, RODEO®, POLARIS®, SPARK® and ACCORD® herbicides, all of which contain glyphosate as its isopropylammonium salt; those sold by Monsanto Company as ROUNDUP® DRY and RIVAL® herbicides, which contain glyphosate as its ammonium salt; that sold by Monsanto Company as ROUNDUP® GEOFORCE, which contains glyphosate as its sodium salt; and that sold by Zeneca Limited as<br>
7<br><br>
TOUCHDOWN® herbicide, which contains glyphosate as its trimethylsulfonium salt.<br>
Through plant genetic engineering methods, it is possible to produce glyphosate tolerant plants by inserting into the plant genome a DNA molecule that causes the production of higher levels of wild-type EPSPS (U.S. Patent 4,940,835; Shah et al., Science 233:478-481, 1986). Glyphosate tolerance can also be achieved by the expression of EPSPS variants that have lower affinity for glyphosate and therefore retain their catalytic activity in the presence of glyphosate, for example, aroA P-S (U.S. Patent 5,094,945), CP4 EPSPS (U.S. Patent 5,633,435), maize TIPS (U.S. Patent 6,040,497), 101/192 and 101/144 variants (U.S. Patent 5.866,775 and U.S. Patent 6,225,112, Howe et al., Mol. Breeding 10:153-164, 2002). For example, glyphosate tolerance has been genetically engineered into com (U.S. Patent No. 5,554,798, 6,040,497), wheat (Zhou et al. Plant Cell Rep. 15:159-163, 1995), soybean (WO 9200377), cotton (WO 0234946), and canoia (WO 9204449).<br>
Variants of the wild-type EPSPS enzyme have been isolated that are glyphosate-resistant as a result of alterations in the EPSPS amino acid coding sequence (Kishore et al., Annu. Rev. Biochem. 57:627-663, 1988; Schulz et al., Arch. Microbiol. 137:121-123, 1984; Sost et al., FEBS Lett. 173:238-241, 1984; Kishore et al., In "Biotechnology for Crop Protection" ACS Symposium Series No. 379. eds. Hedfin et al., 37-48,1988). These variants typically have a higher Ki for glyphosate than the wild-type EPSPS enzyme that confers the glyphosate-tolerant phenotype, but these variants are also characterized by a high Km for PEP that makes the enzyme kinetically less efficient. For example, the apparent Km for PEP and the apparent K, for glyphosate for the native EPSPS from E. coli are 10 u.M and 0.5 u,M while for a glyphosate-resistant isolate having a single amino acid substitution of an alanine for the glycine at position 96 these values are 220 u.M and 4.0 mM, respectively. US Patent No. 6,040,497 reports that the EPSPS variant, known as the TIPS<br>
8<br><br>
mutation (a substitution of isoleucine for threonine at amino acid position 102 and a substitution of serine for proline at amino acid position 106) comprises two mutations that when introduced into the polypeptide sequence of Zea mays EPSPS confers glyphosate resistance to the enzyme. Transgenic plants containing this mutant enzyme are tolerant to glyphosate. Identical mutations may be made in the genes encoding glyphosate sensitive EPSPS enzymes from other sources to create glyphosate resistant enzymes. In vitro site-directed mutagenesis of DNA molecules have clearly demonstrated utility for introducing specific changes in a DNA sequence of a genome and other methods under development may also provide in situ site-directed mutagenesis methods {US Patent Pub. 20020151072). These methods may be used to generate the DNA coding sequences that encode for the glyphosate resistant EPSPS variants of the present invention in the context of the endogenous host cell EPSPS gene.<br>
The present invention provides amino acid substitutions in a class I EPSPS that demonstrates enhanced glyphosate resistance over any previously described modified class I EPSPSs. The present invention relates specifically to certain double variants of class I EPSPSs that are glyphosate resistant, but still retain a functional level of PEP substrate binding activity. During the development of the novel double variants of class I EPSPSs for glyphosate resistance, it was necessary to construct a number of single variants useful as controls for the assay and for demonstration that the double variant is necessary to obtain both a glyphosate resistant enzyme and an enzyme that still retains a sufficient level of substrate binding activity to serve as a functional replacement for a native class 1 EPSPS.<br>
The EPSPS enzyme functions in plant chloroplast, therefore, chloroplast transit peptides (CTP) are engineered in a DNA molecule to encode a fusion of the CTP to the N terminus of an EPSPS creating a chimeric molecule. A chimeric poJynucieic acid coding sequence is comprised of two or more open<br>
9<br><br>
reading frames joined in-frame that encode a chimeric protein, for example, a chloroplast transit peptide and an EPSPS enzyme. A chimeric gene refers to the multiple genetic elements derived from heterologous sources operably linked to comprise a gene. The CTP directs the glyphosate resistant enzyme into the plant chloroplast. In the native plant EPSPS gene, chloroplast transit peptide regions are contained in the native coding sequence (for example, CTP2, Klee et al., Mol. Gen. Genet. 210:47-442, 1987). The CTP is cleaved from the EPSPS enzyme at the chloroplast membrane to create a "mature EPSPS or EPSPS enzyme" that refers to the polypeptide sequence of the processed protein product remaining after the chloroplast transit peptide has been removed.<br>
The native CTP may be substituted with a heterologous CTP during construction of a transgene plant expression cassette. Many chloroplast-localized proteins, including EPSPS, are expressed from nuclear genes as precursors and are targeted to the chloroplast by a chloroplast transit peptide (CTP) that is removed during the import steps. Examples of other such chloroplast proteins include the small subunit (SSU) of ribulose-1, 5,-bisphosphate carboxylase (rubisco), ferredoxin, ferredoxin oxidoreductase, the light-harvesting complex protein I and protein ([, and thioredoxin F. It has been demonstrated in vivo and in vitro that non-chloroplast proteins may be targeted to the chloroplast by use of protein fusions with a CTP and that a CTP sequence is sufficient to target a protein to the chloroplast. Incorporation of a suitable chloroplast transit peptide, such as, the Arabidopsis thaliana EPSPS CTP (Klee et al., Mol. Gen. Genet. 210:437-442 (1987), and the Petunia hybrida EPSPS CTP (della-Cioppa et a]., Proc. Natl. Acad. Sci. USA 83:6873-6877 (1986) has been shown to target heterologous EPSPS protein to chloroplasts in transgenic plants. The production of glyphosate tolerant plants by expression of a fusion protein comprising an amino-terminal CTP with a glyphosate resistant EPSPS enzyme is well known by those skilled in the art,<br>
10<br><br>
(U.S. Patent 5,627,061, U.S. Patent 5,633,435, U.S. Patent 5,312,910, EP 0218571, EP 189707, EP 508909, and EP 924299). Those skilled in the art will recognize that various chimeric constructs can be made that utilize the functionality of a particular CTP to import glyphosate resistant EPSPS enzymes into the plant cell chloroplast.<br>
Modification and changes may be made in the structure of the DNA polynucleotides of the invention and still obtain a DNA molecule that transcribes a mRNA that encodes the modified functional EPSPS protein of the present invention. The amino acid substitutions disclosed herein provide an improved characteristic to the protein, for example, enhanced glyphosate resistant EPSP synthase. Amino-acid substitutions or amino-acid variants are preferably substitutions of a single amino-acid residue for another amino-acid residue at one or more positions within the protein. Substitutions, deletions, insertions or any combination thereof can be combined to arrive at a final construct. The present invention involves the substitution of amino acids in a class I EPSPS protein to provide a new feature of the protein, such as, glyphosate resistance.<br>
It is known that the genetic code is degenerate. The amino acids and their RNA codon(s) are listed below in Table 1.<br>
TABLE 1: Amino acids and the RNA codons that encode them<br>
Amino Acid	Codons<br>
Full name; 3 letter code; 1 letter code<br>
Alanine; Ala; A	GCA	GCC   GCG   GCU<br>
Cysteine; Cys ; C	UGC	UGU<br>
Aspartic acid; Asp; D	GAC	GAU<br>
Glutamic acid; Glu; E	GAA	GAG<br>
Phenylalanine; Phe; F	UUC	UUU<br>
Glycine; Gly; G	GGA	GGC   GGG   GGU<br>
11<br><br>
Histidine; His; H Isoleucine; lie; I Lysine; Lys; K Leucine ; Leu ; L Methionine; Met; M Asparagine; Asn; N Proline; Pro; P Gluiamine; Gin; Q Arginine; Arg; R Serine; Ser; S Threonine; Thr; T Valine; Val; V Tryptophan; Tip; W Tyrosine; Tyr; Y<br><br>
CAC CAU<br>
AUA AUC AUU<br>
AAA AAG<br>
UUA UUG CUA CUC CUG CUU<br>
AUG<br>
AAC AAU<br>
CCA CCC CCG CCU<br>
CAA CAG<br>
AGA AGG CGA CGC CGG CGU<br>
AGC AGU UCA UCC UCG UCU<br>
ACA ACC ACG ACU<br>
GUA GUC GUG GUU<br>
UGG<br>
UAC UAU<br><br>
The codons are described in terms of RNA bases, for example adenine, uracil, guanine and cytosine; it is the mRNA that is directly translated into polypeptides. It is understood that when designing a DNA polynucleotide for use in a construct, the DNA bases would be substituted, for example, thymine instead of uracil. Codon refers to a sequence of three nucleotides that specify a particular amino acid. Codon usage or "codon bias" refers to the frequency of use of codons encoding amino acids in the coding sequences of organisms. A codon usage table would be consulted when selecting substituting codons for an artificial DNA sequence. The sequence of codons provides a coding sequence that refers to the region of continuous sequential nucleic acid triplets encoding a protein, polypeptide, or peptide sequence. The term "encoding DNA" refers to chromosomal DNA, plasmid DNA, cDNA, or artificial DNA polynucleotide that encodes any of the proteins discussed herein. "Plasmid" refers to a circular, extra chromosomal, self-replicating piece of DNA.<br>
The  term   "'endogenous"   refers  to  materials  originating   from   within  an organism or cell. "Exogenous" refers to materials originating from outside of<br>
12<br><br>
an organism or cell. This typically applies to nucleic acid molecules used in producing transformed or transgenic host cells and plants.<br>
The term "genome" as it applies to bacteria encompasses both the chromosome and plasmids within a bacterial host cell. Encoding nucleic acids of the present invention introduced into bacterial host cells can therefore be either chromosomally-integrated or plasmid-localized. The term "genome" as it applies to plant cells encompasses not only chromosomal DNA found within the nucleus, but organelle DNA found within sub cellular components of the cell. The term "gene" refers to polynucleic acids that comprise chromosomal DNA, plasmid DNA, cDNA, an artificial DNA polynucleotide, or other DNA that is transcribed into an RNA molecule, wherein the RNA may encode a peptide, polypeptide, or protein, and the genetic elements flanking the coding sequence that are involved in the regulation of expression of the mRNA or polypeptide of the present invention. A "fragment" of a gene is a portion of a full-length polynucleic acid molecule that is of at least a minimum length capable of transcription into RNA, translation into a peptide or useful as a probe or primer in a DNA detection method.<br>
Polynucleic acids of the present invention introduced into plant cells can therefore be either chromosomal ly-integrated or organelle-localized. The modified EPSPSs of the present invention are targeted to the chloroplast by a chloroplast transit peptide located at the N-terminus of the coding sequence. Alternatively, the gene encoding the modified EPSPSs may be integrated into the chloroplast genome, thereby eliminating the need for a chloroplast transit peptide.<br>
"Heterologous DNA" sequence refers to a polynucleotide sequence that originates from a foreign source or species or, if from the same source, is modified from its original form. "Homologous DNA" refers to DNA from the same source as that of the recipient cell.<br>
13<br><br>
"Hybridization" refers to the ability of a strand of nucleic acid to join with a complementary strand via base pairing. Hybridization occurs when complementary sequences in the two nucleic acid strands bind to one another. The nucleic acid probes and primers of the present invention hybridize under stringent conditions to a target DNA sequence. Any conventional nucleic acid hybridization or amplification method can be used to identify the presence of DNA from a transgenic event in a sample. A transgenic "event" is produced by transformation of a plant cell with heterologous DNA, i.e., a nucleic acid construct that includes a transgene of interest; regeneration of a population of plants resulting from the insertion of the transgene into the genome of the plant cell, and selection of a particular plant characterized by insertion into a particular genome location. The term "event" refers to the original transformant plant and progeny of the transformant that include the heterologous DNA. The term "event" also includes progeny produced by a sexual outcross between the event and another plant that wherein the progeny includes the heterologous DNA. Nucleic acid molecules or fragments thereof are capable of specifically hybridizing to other nucleic acid molecules under certain circumstances. As used herein, two nucleic acid molecules are said to be capable of specifically hybridizing to one another if the two molecules are capable of forming an anti-parallel, double-stranded nucleic acid structure. A nucleic acid molecule is said to be the "complement" of another nucleic acid molecule if they exhibit complete complementarity. As used herein, molecules are said to exhibit "complete complementarity" when every nucleotide of one of the molecules is complementary to a nucleotide of the other. Two molecules are said to be "minimally complementary" if they can hybridize to one another with sufficient stability to permit them to remain annealed to one another under at least conventional "low-stringency" conditions. Similarly, the molecules are said to be "complementary" if they can hybridize to one another with sufficient stability to permit them to remain annealed to one another under conventional "high-stringency" conditions.<br>
14<br><br>
Conventional stringency conditions are described by Sambrook et al., Molecular Cloning - A Laboratory Manual, 2nd. ed., Cold Spring Harbor Laboratory, Cold Spring Harbor, New York (1989), herein referred to as Sambrook et al., (1989), and by Haymes et al., In: Nucleic Acid Hybridization, A Practical Approach, IRL Press, Washington, DC (1985). Departures from complete complementarity are therefore permissible, as long as such departures do not completely preclude the capacity of the molecules to form a double-stranded structure. In order for a nucleic acid molecule to serve as a primer or probe it need only be sufficiently complementary in sequence to be able to form a stable double-stranded structure under the particular solvent and salt concentrations employed.<br>
As used herein, a substantially homologous sequence is a nucleic acid sequence that will specifically hybridize to the complement of the nucleic acid sequence to which it is being compared under high stringency conditions. The term "stringent conditions" is functionally defined with regard to the hybridization of a nucleic-acid probe to a target nucleic acid (such as, to a particular nucleic-acid sequence of interest) by the specific hybridization procedure discussed in Sambrook et al., 1989, at 9.52-9.55. See also, Sambrook et al., 1989 at 9.47-9.52, 9.56-9.58; Kanehisa, (Nucl. Acids Res. 12:203-213, 1984); and Wetmur and Davidson, (J. Mol. Biol. 31:349-370, 1988). Accordingly, the nucleotide sequences of the invention may be used for their ability to selectively form duplex molecules with complementary stretches of DNA fragments. Depending on the application envisioned, one will desire to employ varying conditions of hybridization to achieve varying degrees of selectivity of probe towards target sequence. For applications requiring high selectivity, one will typically desire to employ relatively stringent conditions to form the hybrids, for example, one will select relatively low salt and/or high temperature conditions, such as provided by about 0.02 M to about 0.15 M NaCl at temperatures of about 50DC to about 70°C. A<br>
15<br><br>
stringent condition, for example, is to wash the hybridization filter at least twice with high-stringency wash buffer (0.2X SSC, 0.1% SDS, 65° C). Appropriate stringency conditions that promote DNA hybridization, for example, 6.0 x sodium chloride/sodium citrate (SSC) at about 45°C, followed by a wash of 2.0 x SSC at 50°C, are known to those skilled in the art or can be found in Current Protocols in Molecular Biology, John Wiley &amp; Sons, N.Y. (1989), 6.3.1-6.3.6. For example, the salt concentration in the wash step can be selected from a low stringency of about 2.0 x SSC at 50°C to a high stringency of about 0.2 x SSC at 50°C. In addition, the temperature in the wash step can be increased from low stringency conditions at room temperature, about 22°C, to high stringency conditions at about 65°C. Both temperature and salt may be varied, or either the temperature or the salt concentration may be held constant while the other variable is changed. Such selective conditions tolerate little, if any, mismatch between the probe and the template or target strand. Detection of DNA molecules via hybridization is well known to those of skill in the art, and the teachings of U.S. Patent 4,965,188 and 5,176,995 are exemplary of the methods of hybridization analyses.<br>
"Identity" refers to the degree of similarity between two polynucleic acid or protein sequences. An alignment of the two sequences is performed by a suitable computer program. A widely used and accepted computer program for performing sequence alignments is CLUSTALW vl.6 (Thompson, et al. Nucl. Acids Res., 22: 4673-4680, 1994). The number of matching bases or amino acids is divided by the total number of bases or amino acids, and multiplied by 100 to obtain a percent identity. For example, if two 580 base pair sequences had 145 matched bases, they would be 25 percent identical. If the two compared sequences are of different lengths, the number of matches is divided by the shorter of the two lengths. For example, if there are 100 matched amino acids between 200 and 400 amino acid proteins, they are 50<br>
16<br><br>
percent identical with respect to the shorter sequence. If the shorter sequence is less than 150 bases or 50 amino acids in length, the number of matches are divided by 150 (for nucleic acid bases) or 50 (for amino acids), and multiplied by 100 to obtain a percent identity.<br>
"Intron" refers to a genetic element that is a portion of a gene not translated into protein, even though it is transcribed into RNA, the intron sequence being "spliced out" from the mature messenger RNA.<br>
An "isolated" nucleic acid molecule is substantially separated away from other nucleic acid sequences with which the nucleic acid is normally associated, such as, from the chromosomal or extrachromosomal DNA of a cell in which the nucleic acid naturally occurs. A nucleic acid molecule is an isolated nucleic acid molecule when it comprises a transgene or part of a transgene present in the genome of another organism. The term also embraces nucleic acids that are biochemically purified so as to substantially remove contaminating nucleic acids and other cellular components. The term "transgene" refers to any polynucleic acid molecule nonnative to a cell Or organism transformed into the cell or organism. "Transgene" also encompasses the component parts of a native plant gene modified by insertion of a nonnative polynucleic acid molecule by directed recombination or site specific mutation.<br>
"Isolated," "Purified," "Homogeneous" polypeptides: A polypeptide is "isolated" if it has been separated from the cellular components (nucleic acids, lipids, carbohydrates, and other polypeptides) that naturally accompany it or that is chemically synthesized or recombinant. A polypeptide molecule is an isolated polypeptide molecule when it is expressed from a transgene in another organism. A monomeric polypeptide is isolated when at least 60% by weight of a sample is composed of the polypeptide, preferably 90% or more, more preferably 95% or more, and most preferably more than 99%. Protein purity or<br>
17<br><br>
homogeneity is indicated, for example, by polyacrylamide gel electrophoresis of a protein sample, followed by visualization of a single polypeptide band upon staining the polyacrylamide gel; high pressure liquid chromatography; or other conventional methods. Proteins can be purified by any of the means known in the art, for example as described in Guide to Protein Purification, ed. Deutscher, Meth. Enzymol. 185, Academic Press, San Diego, 1990; and Scopes, Protein Purification: Principles and Practice, Springer Verlag, New York, 1982.<br>
The term "native" generally refers to a naturally-occurring ("wild-type") polynucleic acid or polypeptide. However, in the context of the present invention a modification of a native isolated polynucleotide and polypeptide has occurred to provide a variant polypeptide with a particular phenotype, for example, amino acid substitution in a native glyphosate sensitive EPSPS to provide a glyphosate resistant EPSPS. The polynucleotide modified in this manner is nonnative with respect to the genetic elements normally found linked to a naturally occurring unmodified polynucleotide.<br>
Using well-known methods, the skilled artisan can readily produce nucleotide and amino acid sequence variants of genes and proteins that provide a modified gene product. For example, "variant" DNA molecules of the present invention are DNA molecules containing changes in an EPSPS coding sequence, such as, changes that include one or more nucleotides of a native EPSPS coding sequence being deleted, added, and/or substituted, such that the variant EPSPS gene encodes a modified protein that retains EPSPS activity and is now resistant to glyphosate herbicide. Variant DNA molecules can be produced, for example, by standard DNA mutagenesis techniques or by chemically synthesizing the variant DNA molecule or a portion thereof. Methods for chemical synthesis of nucleic acids are discussed, for example, in Beaucage et al., Tetra. Letts. 22:1859-1862 (1981), and Marteucci et al., J. Am. Chem. Soc. 103:3185- (1981). Chemical synthesis of nucleic acids can be<br>
18<br><br>
performed, for example, on automated oligonucleotide synthesizers. Such variants preferably do not change the reading frame of the protein-coding region of the nucleic acid. The present invention also encompasses fragments of a protein that lacks at least one residue of a full-length protein, but that substantially maintains activity of the protein.<br>
A first nucleic-acid molecule is "operably linked" with a second nucleic-acid molecule when the first nucleic-acid molecule is placed in a functional relationship with the second nucleic-acid molecule. For example, a promoter is operably linked to a protein-coding nucleic acid sequence if the promoter effects the transcription or expression of the coding sequence. Generally, operably linked DNA molecules are contiguous and, where necessary to join two protein-coding regions, in reading frame.<br>
The term "plant" encompasses any higher plant and progeny thereof, including monocots (for example, corn, rice, wheat, barley, etc.), dicots (for example, soybean, cotton, canola, tomato, potato, Arabidopsis, tobacco, etc.), gymnosperms (pines, firs, cedars, etc.) and includes parts of plants, including reproductive units of a plant (for example, seeds, bulbs, tubers, fruit, flowers, etc.) or other parts or tissues from that the plant can be reproduced.<br>
"Polyadenylation signal" or "polyA signal" refers to a nucleic acid sequence located 3' to a coding region that causes the addition of adenylate nucleotides to the 3' end of the mRNA transcribed from the coding region.<br>
"Polymerase chain reaction (PCR)" refers to a DNA amplification method that uses an enzymatic technique to create multiple copies of one sequence of nucleic acid (amplicon). Copies of a DNA molecule are prepared by shuttling a DNA polymerase between two amplimers. The basis of this amplification method is multiple cycles of temperature changes to denature, then re-anneal amplimers (DNA primer molecules), followed by extension to synthesize new DNA strands in the region located between the flanking amplimers. Nucleic-<br>
19<br><br>
acid amplification can be accomplished by any of the various nucleic-acid amplification methods known in the art, including the polymerase chain reaction (PCRJ. A variety of amplification methods are known in the art and are described, inter alia, in U.S. Patents 4,683,195 and 4,683,202 and in PCR Protocols: A Guide to Methods and Applications, ed. Innis et al., Academic Press, San Diego, 1990. PCR amplification methods have been developed to amplify up to 22 kb of genomic DNA and up to 42 kb of bacteriophage DNA (Cheng et al., Proc. Natl. Acad. Sci. USA 91:5695-5699, 1994). These methods as well as other methods known in the art of DNA amplification may be used in the practice of the present invention.<br>
The term "promoter'- or "promoter region" refers to a polynucleic acid molecule that functions as a regulatory element, usually found upstream (5') to a coding sequence, that controls expression of the coding sequence by controlling production of messenger RNA (mRNA) by providing the recognition site for RNA polymerase and/or other factors necessary for start of transcription at the correct site. As contemplated herein, a promoter or promoter region includes variations of promoters derived by means of ligation to various regulatory sequences, random or controlled mutagenesis, and addition or duplication of enhancer sequences. The promoter region disclosed herein, and biologically functional equivalents thereof, are responsible for driving the transcription of coding sequences under their control when introduced into a host as part of a suitable recombinant DNA construct, as demonstrated by its ability to produce mRNA.<br>
A "recombinant" nucleic acid is made by a combination of two otherwise separated segments of nucleic acid sequence, for example, by chemical synthesis or by the manipulation of isolated segments of polynucleic acids by genetic engineering techniques. The term "recombinant DNA construct" refers to any agent such as a plasmid, cosmid, virus, autonomously replicating sequence, phage, or linear or circular single-stranded or double-stranded DNA<br>
20<br><br>
or RNA nucleotide sequence, derived from any source, capable of genomic integration or autonomous replication, comprising a DNA molecule that one or more DNA sequences have been linked in a functionally operative manner. Such recombinant DNA constructs are capable of introducing a 5' regulatory sequence or promoter region and a DNA sequence for a selected gene product into a cell in such a manner that the DNA sequence is transcribed into a functional mRNA that is translated and therefore expressed. Recombinant DNA constructs may be constructed to be capable of expressing antisense RNAs, or stabilized double stranded antisense RNA in order to inhibit expression of a specific target RNA of interest.<br>
"Resistance" refers to an enzyme that is able to function in the presence of a toxin, for example, naturally occurring glyphosate resistant class 11 EPSP synthases resistant to glyphosate or a modified EPSPS enzyme having catalytic activity that is unaffected by at a herbicide concentration that normally disrupts the same activity in the wild type enzyme, for example, the modified class I EPSP synthases of the present invention. An enzyme that has resistance to a herbicide may also have the function of detoxifying the herbicide, for example, phosphinothricin acety[transferase, and glyphosate oxidoreductase.<br>
"Selectable marker" refers to a polynucleic acid molecule that encodes a protein, which confers a phenotype facilitating identification of cells containing the polynucleic acid molecule. Selectable markers include those genes that confer resistance to antibiotics (for example, ampicillin, kanamycin), complement a nutritional deficiency (for example, uracil, histidine, leucine), or impart a visually distinguishing characteristic (for example, color changes or fluorescence). Useful dominant selectable marker genes include genes encoding antibiotic resistance genes (for example, neomycin phosphotransferase, npt); and herbicide resistance genes (for example, phosphinothricin acetyl transferase, class II EPSP synthase, modified<br>
21<br><br>
class 1 EPSP synthase). A useful strategy for selection of transformants for herbicide resistance is described, for example, in Vasil, Cell Culture and Somatic Cell Genetics of Plants, Vols. I-iII, Laboratory Procedures and Their Applications Academic Press, New York (1984).<br>
An "artificial polynucleotide" as used in the present invention is a DNA sequence designed according to the methods of the present invention and created as an isolated DNA molecule for use in a DNA construct that provides expression of a protein in host cells, or for the purposes of cloning into appropriate constructs or other uses known to those skilled in the art. Computer programs are available for these purposes, including but not limited to the "BestFit" or ltGap" programs of the Sequence Analysis Software Package, Genetics Computer Group (GCG), Inc., University of Wisconsin Biotechnology Center, Madison, WI 53711. The artificial polynucleotide may be created by one or more methods known in the art that include, but are not limited to: overlapping PCR. An artificial polynucleotide as used herein, is non-naturally occurring and can be substantially divergent from other polynucleotides that code for the identical or nearly identical protein.<br>
Expression of a modified class I EPSPS coding sequence in plants<br>
DNA constructs are made that contain various genetic elements necessary for the expression of the EPSPS coding sequence in plants. "DNA construct" refers to the heterologous genetic elements operably linked to each other making up a recombinant DNA molecule and may comprise elements that provide expression of a DNA polynucleotide molecule in a host cell and elements that provide maintenance of the construct in the host cell. A plant expression cassette comprises the operable linkage of genetic elements that when transferred into a plant cell provides expression of a desirable gene product. "Plant expression cassette" refers to chimeric DNA segments comprising the regulatory elements that are operably linked to provide the expression of a transgene product in plants. Promoters, leaders, introns, transit<br>
22<br><br>
peptide encoding polynucleic acids, 3' transcriptional termination regions are all genetic elements that may be operably linked by those skilled in the art of plant molecular biology to provide a desirable level of expression or functionality to a glyphosate resistant class I EPSPS of the present invention. A DNA construct can contain one or more plant expression cassettes expressing the DNA molecules of the present invention or other DNA molecules useful in the genetic engineering of crop plants.<br>
A variety of promoters specifically active in vegetative tissues, such as leaves, stems, roots and tubers, can be used to express the EPSPS polynucleic acid molecules of the present invention. Examples of tuber-specific promoters include, but are not limited to the class I and II patatin promoters (Bevan et al., EMBO J. 8:1899-1906, 1986; Koster-Topfer et al., Mol Gen Genet. 219:390-396, 1989; Mignery et al., Gene. 62:27-44, 1988; Jefferson et al., Plant Mol. Biol. 14: 995-1006, 1990), the promoter for the potato tuber ADPGPP genes, both the large and small subunits; the sucrose synthase promoter (Salanoubat and Belliard, Gene. 60:47-56, 1987; Salanoubat and Belliard, Gene 84: 181-185, 1989); and the promoter for the major tuber proteins including the 22 kd protein complexes and proteinase inhibitors (Hannapel, Plant Physiol. 101:703-704, 1993). Examples of leaf-specific promoters include, but are not limited to the ribulose biphosphate carboxylase {RBCS or RuBISCO) promoters (see, for example, Matsuoka et al., Plant J. 6:311-319, 1994); the light harvesting chlorophyll a/b binding protein gene promoter (see, for example, Shiina et al., Plant Physiol. 115:477-483, 1997; Casal et al., Plant Physiol. 116:1533-1538, 1998); and the Arabidopsis thaliana myb-related gene promoter (AtmybS) (Li et al., FEBS Lett. 379:117-121, 1996). Examples of root-specific promoter include, but are not limited to the promoter for the acid chitinase gene (Samac et al., Plant Mol. Biol. 25:587-596, 1994); the root specific subdomains of the CaMV35S promoter that have been identified (Lam et al., Proc. Natl. Acad. Sci. (U.S.A.) 86:7890-7894, 1989); the ORF13<br>
23<br><br>
promoter from Agrobacterium rhizogenes that exhibits high activity in roots (Hansen et al., Mol. Gen. Genet. 254:337-343 (1997); the promoter for the tobacco root-specific gene TobRB7 (Yamamoto et al., Plant Cell 3:371-382, 1991); and the root cell specific promoters reported by Conkling et al. (Conkling et al.. Plant Physiol. 93:1203-1211, 1990).<br>
Another class of useful vegetative tissue-specific promoters is meristematic<br>
(root     tip     and     shoot     apex)     promoters.	For     example,     the<br>
"SHOGTMERISTEMLESS" and "SCARECROW" promoters, which are active in the developing shoot or root apical meristems can be used (Di Laurenzio et al., Cell 86:423- 433, 1996; Long, Nature 379:66-69, 1996). Another example of a useful promoter is that which controls the expression of 3-hydroxy-3-methylglutaryl coenzyme A reductase HMG2 gene, whose expression is restricted to meristematic and floral (secretory zone of the stigma, mature pollen grains, gynoecium vascular tissue, and fertilized ovules) tissues (see, for example, Enjuto et al., Plant Cell. 7:517-527, 1995). Also another example of a useful promoter is that which controls the expression of knl-related genes from maize and other species that show meri stem -specific expression (see, for example, Granger et al., Plant Mol. Biol. 31:373-378, 1996; Kerstetter et al., Plant Cell 6:1877-1887, 1994; Hake el al., Philos. Trans. R. Soc. Lond. B. Biol. Sci. 350:45-51, 1995). Another example of a meristematic promoter is the Arabidopsis thaliana KNAT1 promoter. In the shoot apex, KNAT1 transcript is localized primarily to the shoot apical meristem; the expression of KNATI in the shoot meristem decreases during the floral transition and is restricted to the cortex of the inflorescence stern (see, for example, Lincoln et al., Plant Cell 6:1859-1876, 1994).<br>
Suitable seed-specific promoters can be derived from the following genes: MAC1 from maize (Sheridan et al., Genetics 142:1009-1020, 1996; Cat3 from maize (GenBank No. L05934, Abler et al„ Plant Mol. Biol. 22:10131-1038, 1993); viviparous-1 from Arabidopsis (Genbank No. U93215); Atimycl from<br>
24<br><br>
Arabidopsis (Urao et al., Plant Mol. Biol. 32:571-57, 1996; Conceicao et al., Plant 5:493-505, 1994); napA from Brassica napus (GenBank No. J02798); the napin gene family from Brassica napus (Sjodahl et al., Planta 197:264-271,1995, and others (Chen et al., Proc. Natl. Acad Sci. 83:8560-8564, 1986).<br>
The ovule-specific promoter for BEL1 gene can also be used (Reiser et al. Cell 83:735-742, 1995, GenBank No. U39944; Ray et al, Proc. Natl. Acad. Sci, USA 91:5761-5765, 1994). The egg and central cell specific MEA (FIS1) and FIS2 promoters are also useful reproductive tissue-specific promoters (Luo et al., Proc. Natl. Acad. Sci. USA, 97:10637-10642, 2000; Vielle-Calzada, et al., Genes Dev. 13:2971-2982, 1999).<br>
A maize pollen-specific promoter has been identified in maize (Guerrero et al., Mol. Gen. Genet. 224:161-168, 1990). Other genes specifically expressed in pollen have been described (see, for example, Wakeley et al., Plant Mol. Biol, 37:187-192, 1998; Ficker et al., Mol. Gen. Genet. 257:132-142, 1998; Kulikauskas et al., Plant Mol. Biol. 34:809-814, 1997; Treacy et al., Plant Mol. Biol. 34:603-611, 1997).<br>
It is recognized that additional promoters that may be utilized are described, tor example, in U.S. Patents 5,378,619, 5,391,725, 5,428,147, 5,447,858, 5,608,144, 5,608,144, 5,614,399, 5,633,441, 5,633,435, and 4,633,436. It is further recognized that the exact boundaries of regulatory sequences may not be completely defined; DNA fragments of different lengths may have identical promoter activity.<br>
The translation leader sequence means a DNA molecule located between the promoter of a gene and the coding sequence. The translation leader sequence is present in the fully processed mRNA upstream of the translation start sequence. The translation leader sequence may affect processing of the primary transcript to mRNA, mRNA stability or translation efficiency. Examples of translation leader sequences include maize and petunia heat<br>
25<br><br>
shock protein leaders, plant virus coat protein leaders, plant rubisco gene leaders among others (Turner and Foster, Molecular Biotechnology 3:225, 1995).<br>
The "3' non-translated sequences" means DNA sequences located downstream of a structural polynucleotide sequence and include sequences encoding polyadenylation and other regulatory signals capable of affecting mRNA processing or gene expression. The polyadenylation signal functions in plants to cause the addition of polyadenylate nucleotides to the 3' end of the mRNA precursor. The polyadenylation sequence can be derived from the natural gene, from a variety of plant genes, or from T-DNA. An example of the polyadenylation sequence is the nopaline synthase 3' sequence (nos 3'; Fraley et al., Proc. Natl. Acad. Sci. USA 80: 4803-4807, 1983). The use of different 3' non-translated sequences is exemplified by Ingelbrecht et al., Plant Cell 1:671-680, 1989.<br>
The laboratory procedures in recombinant DNA technology used herein are those well known and commonly employed in the art. Standard techniques are used for cloning, DNA and RNA isolation, amplification and purification. Generally enzymatic reactions involving DNA ligase, DNA polymerase, restriction endonucleases and the like are performed according to the manufacturer's specifications. These techniques and various other techniques are generally performed according to Sambrook et al. (1989).<br>
The DNA construct of the present invention may be introduced into the genome of a desired plant host by a variety of conventional transformation techniques that are well known to those skilled in the art. "Transformation" refers to a process of introducing an exogenous polynucleic acid molecule (for example, a DNA construct, a recombinant polynucleic acid molecule) into a cell or protoplast and that exogenous polynucleic acid molecule is incorporated into a host cell genome or an organelle genome (for example.<br>
26<br><br>
chloroplast or mitochondria) or is capable of autonomous replication. "Transformed" or "transgenic" refers to a cell, tissue, organ, or organism into which a foreign polynucleic acid, such as a DNA vector or recombinant polynucleic acid molecule. A "transgenic" or "transformed" cell or organism also includes progeny of the cell or organism and progeny produced from a breeding program employing such a "transgenic" plant as a parent in a cross and exhibiting an altered phenotype resulting from the presence of the foreign polynucleic acid molecule.<br>
Methods of transformation of plant cells or tissues include, but are not limited to Agrobacterium mediated transformation method and the Biolistics or particle-gun mediated transformation method. Suitable plant transformation vectors for the purpose of Agrobacterium mediated transformation include those elements derived from a tumor inducing (Ti) plasmid of Agrobacterium tumefaciens, for example, right border (RB) regions and left border (LB) regions, and others disclosed by Herrera-Estrella et al., Nature 303:209 (1983); Bevan, Nucleic Acids Res.l2:8711-8721 (1984); Klee et al., Bio-Technology 3(7):637-642 (1985). In addition to plant transformation vectors derived from the Ti or root-inducing (Ri) plasmids of Agrobacterium, alternative methods can be used to insert the DNA constructs of this invention into plant cells. Such methods may involve, but are not limited to, for example, the use of liposomes, electroporation, chemicals that increase free DNA uptake, free DNA delivery via microprojectile bombardment, and transformation using viruses or pollen.<br>
DNA constructs can be prepared that incorporate the class I EPSPS variant coding sequences of the present invention for use in directing the expression of the sequences directly from the host plant cell plastid. Examples of such constructs suitable for this purpose and methods that are known in the art and are generally described, for example, in Svab et al., Proc. Natl. Acad. Sci. USA 87:8526-8530, (1990) and Svab et al., Proc. Natl. Acad. Sci. USA<br>
27<br><br>
90:913-917 (1993) and in U.S. Patent No. 5,693,507. It is contemplated that piastid transformation and expression of the class I EPSPS variants of the present invention will provide glyphosate tolerance to the plant cell.<br>
A plasmid expression vector suitable for the introduction of a polynucleic acid encoding a polypeptide of present invention in monocots using electroporation or particle-gun mediated transformation is composed of the following: a promoter that is constitutive or tissue-specific; an intron that provides a splice site to facilitate expression of the gene, such as the maize Hsp70 intron (U.S. Patent 5,593,874); and a 3' polyadenylation sequence such as the nopaline synthase 3' sequence (T-nos 3'; Fraley et al., Proc. Natl. Acad. Sci. USA 80: 4803-4807, 1983). This expression cassette may be assembled on high copy replicons suitable for the production of large quantities of DNA.<br>
When adequate numbers of cells containing the exogenous polynucleic acid molecule encoding polypeptides from the present invention are obtained, the cells can be cultured, then regenerated into whole plants. "Regeneration" refers to the process of growing a plant from a plant cell (for example, plant protoplast or explant). Such regeneration techniques rely on manipulation of certain phytohormones in a tissue culture growth medium, typically relying on a biocide and/or herbicide marker that has been introduced together with the desired nucleotide sequences. Choice of methodology for the regeneration step is not critical, with suitable protocols being available for hosts from Leguminoseae (for example, alfalfa, soybean, clover), Umbelliferae (carrot, celery, parsnip), Cruciferae (for example, cabbage, radish, canola/rapeseed), Cucurbitaceae (for example, melons and cucumber), Gramineae (for example, wheat, barley, rice, maize), Solanaceae (for example, potato, tobacco, tomato, peppers), various floral crops, such as sunflower, and nut-bearing trees, such as almonds, cashews, walnuts, and pecans. See, for example, Ammirato et al., Handbook of Plant Cell Culture - Crop Species. Macmillan Publ. Co. (1984); Shimamoto et al., Nature 338:274-276 (1989); Fromm, UCLA Symposium on<br>
28<br><br>
Molecular Strategies for Crop Improvement, April 16-22, 1990. Keystone, CO (1990); Vasil et al., Biotechnology 8:429-434 (1990); Vasil et al., Bio/Technology 10:667-674 (1992); Hayashimoto, Plant Physiol. 93:857-863 (1990); and Datta et al., Bio-technology 8:736-740 (1990). Such regeneration techniques are described generally in Klee et al., Ann. Rev. Plant Phys. 38:467-486(1987).<br>
The development or regeneration of transgenic plants containing the exogenous polynucleie acid molecule that encodes a polypeptide of interest is well known in the art. Preferably, the regenerated plants are self-pollinated to provide homozygous transgenic plants, as discussed above. Otherwise, pollen obtained from the regenerated plants is crossed to seed-grown plants of agronomically important lines. Conversely, pollen from plants of these important lines is used to pollinate regenerated plants.<br>
Plants that can be made to have enhanced glyphosate tolerance by practice of the present invention include, but are not limited to, Acacia, alfalfa, aneth, apple, apricot, artichoke, arugula, asparagus, avocado, banana, barley, beans, beet, blackberry, blueberry, broccoli, brussels sprouts, cabbage, canola, cantaloupe, carrot, cassava, cauliflower, celery, cherry, cilantro, citrus, Clementines, coffee, corn, cotton, cucumber, Douglas fir, eggplant, endive, escarole, eucalyptus, fennel, figs, forest trees, gourd, grape, grapefruit, honey dew, jicama, kiwifruit, lettuce, leeks, lemon, lime, Loblolly pine, mango, melon, mushroom, nut, oat, okra, onion, orange, an ornamental plant, papaya, parsley, pea, peach, peanut, pear, pepper, persimmon, pine, pineapple, plantain, plum, pomegranate, poplar, potato, pumpkin, quince, radiata pine, radicchio, radish, raspberry, rice, rye, sorghum, Southern pine, soybean, spinach, squash, strawberry, sugarbeet, sugarcane, sunflower, sweet potato, sweetgum, tangerine, tea, tobacco, tomato, turf, a vine, watermelon, wheat, yams, and zucchini.<br>
29<br><br>
The following examples are provided to better elucidate the practice of the present invention and should not be interpreted in any way to limit the scope of the present invention. Those skilled in the art will recognize that various modifications, additions, substitutions, truncations, etc., can be made to the methods and genes described herein while not departing from the spirit and scope of the present invention.<br>
EXAMPLES<br>
EXAMPLE 1<br>
Site-directed mutagenesis of a class I EPSPS<br>
Mutagenesis of a DNA molecule encoding a class I EPSPS was directed at a region of the protein defined by a polypeptide sequence -G-T-X1-X2-R-P-(SEQ ID NO:l) of the class I EPSPS, where X, and X2 are any amino acid. The invention described herein provides for the mutagenesis of a gene encoding a class I EPSPS, wherein the mutagenesis results in a polypeptide sequence of -G-X4-X1-X2-R-X3- (SEQ ID NO:2) in this region of the class I EPSPS protein related to the binding of the enzyme substrate and the glyphosate molecule. The amino acid substitutions in SEQ ID NO: I that will result in a glyphosate resistant class I EPSPS include replacing the native threonine (T) at X4 with amino acids isoleucine (I) or leucine (L), and replacing the native proline (P) at X3 with threonine, glycine, cysteine, alanine, or isoleucine. The amino acid positions 102 and 106 are designated according to the maize EPSPS polypeptide sequence shown in Figure 1, however, other plant class I EPSPS coding sequences (Figure 2), for example, petunia and soybean can be used as templates for site-directed mutagenesis as the relative positions of the threonine and proline amino acids, respectively, are conserved; however, a slightly different amino acid position number in the EPSPS polypeptide sequence may occur because of variations in the starting point of mature EPSPSs from various sources (U.S. Patent No: 5,866,775,<br>
30<br><br>
Figure 1), those variations are recognized by those skilled in the art and are within the scope of the present invention. In a similar manner, site-directed mutagenesis of prokaryote class I EPSPS DNA coding sequences, for example, E. coli (Figure 2) can be performed using mutagenesis primers designed to hybridize to these DNA molecules to create the EPSPS variants -G-X4-X1-X2-R-X3- (SEQ ID NO:2) as described herein.<br>
Mutations were made using plant EPSPS DNA coding sequence template as an example of class 1 EPSPSs. Mutations of the DNA coding sequence result in variant EPSPS protein molecules by the substitution of codons (Table 1) encoding for amino acids in the DNA sequence. The variant protein sequences that have two amino acid substitutions compared to the wild type protein sequence are referred to as double variants, a single amino acid substitution is referred to as a single variant. All the variants were made using the PCR-based QuickChange™ Site-directed Mutagenesis Kit (Stratagene, La Jolla, CA, Cat. No. 200518) following the manufactures instructions. The DNA sequence of each mutagenesis primer was designed, and then ordered from Invitrogen Corp., Custom Primers (Carlsbad, CA). Mutagenesis of a maize wild-type DNA molecule (SEQ ID NO:3) encoding the EPSPS enzyme was performed using pMON70461 (Figure 3) as the template pMON70461 contains the unmodified wild-type maize EPSPS coding sequence. The previously known T102I, P106S variant (TIPS) was created by the PCR mediated mutagenesis method using primer pairs, TIPSMut-1-U (SEQ ID NO: 4) and TIPSMut-2-L (SEQ ID NO: 5) as shown in Figure 4A, and is contained in pMON70462 plasmid. The single EPSPS variants were created by mutagenesis of the maize wild-type EPSPS DNA coding sequence as controls for measuring the efficacy of the double variant EPSPSs. The following single EPSPS variants were created using PCR mutagenesis: the T102I variant (primers Il-U (SEQ ID NO:6) and I2-L (SEQ ID NO:7), pMON58455), the PI06T variant (primers mEmut-9-U (SEQ ID NO:8) and mEmut-10-L (SEQ ID NO:9), pMON70467,<br>
31<br><br>
Figure 9), the P106S variant (primers mEmut-7-U (SEQ ID NO: 10) and mEmut-S-L (SEQ ID NO: 11), pMON70466), and the P106L variant (primers Hl-U (SEQ ID NO: 12) and H2-L (SEQ ID NO: 13), pMON5845t) were created using the unmodified wild-type maize EPSPS coding sequence contained in pMON70461 as the template for site directed mutagenesis.<br>
The double variants of the present invention were made using pMON58452 (Figure 5) as the template. This pMON58452 EPSPS gene template contains the maize EPSPS double variant T102I, P106T (TIPT) that was constructed by mutagenesis of pMON70467 with the mutagenesis primers mEmut-9-U and mEmut-10-L. The various mutagenesis primer sequences were designed and then were synthesized by Invitrogen Corp., Custom Primers and used in combination in a PCR to create the variant EPSPS coding sequences. The PCR was set up in a 50 ul reaction in the following manner: dt^O 38 u.1; 2 mM dNTP I uX; 10X buffer 5 (iL; pMON58452 1 uL (10 ng); primer-U 2 uL; primer-L 2 ^L; pfu Turbo enzyme 1 ^L. PCR was carried out on a MJ Research PTC-200 thermal cycler using the following program: Step 1 - 94°C for 30 seconds; Step 2 - 94°C for 30 seconds; Step 3 - 55°C for 1 minute; Step 4 - 68°C for 14 minute; Step 5 - go to step 2, 16 times; Step 6 - End. At the end of the PCR, I (il of the restriction enzyme Dpnl was added to each 50 (jl of PCR reaction and the mixture was incubated at 37°C for 1 hour. The proline (106) amino acid codon of pMON58452 was substituted in subsequent steps to provide additional variants that include TIPG (primer P106G-U, SEQ ID NO:14 and -L, SEQ ID NO:15), TIPA (primer P106A-U, SEQ ID NO:16 and -L, SEQ ID NO: 17), TIPV (primer PI 06V-U, SEQ ID NO: 18 and -L, SEQ ID NO: 19), TIPL (primer P106L-U, SEQ ID NO:20 and -L, SEQ ID NO:2l), TIPI (primer PI06I-U, SEQ ID NO:22 and -L, SEQ ID NO:23), TIPM (primer P106M-U, SEQ ID NO:24 and -L, SEQ ID NO:25), and TIPC (primer P106C-U, SEQ ID NO:26 and -L, SEQ ID NO:27). The mutagenesis primers' DNA sequences are shown in Figures 4A and 4B. The double variants of the<br>
32<br><br>
gene encoding the EPSPS protein were generated using the PCR conditions described above.<br>
At the end of the PCR, 1 ul of the restriction enzyme Dpnl was added lo each 50 uj of PCR reaction and the mixture was incubated at 37°C for 1 hour. Dpnl is a methylation- and hemimethylation-specific restriction enzyme and will cleave only those double-stranded DNA plasmid containing at least one wild-type, methylated, strand, leaving the mutated plasmid intact. After the Dpnl treatment, 1 ul of the treated reaction mixture was used to transform the competent E. coli strain XL 1-blue (Stratagene Corp, La Jolla, CA) following the manufacturer's instruction. The transformed cells were plated onto a Petri dish containing carbenicillin at a final concentration of 0.1 mg/mL. The dish was then incubated at 37°C overnight. Single colonies were picked the next day and used to inoculate a 3 mL liquid culture containing 0.1 mg/mL carbenicillin. The liquid culture was incubated overnight at 37°C with agitation at 250 rpm. Plasmid DNA was prepared from 1 mL of the liquid culture using Qiagen's miniprep Kit (Qiagen Corp. Cat. No. 27160). The DNA was elutcd in 50 u.1 of dP^O. The entire coding region of three independent clones from each mutagenesis was sequenced by DNA sequence analysis (ABI Prism™ 377, PE Biosystems, Foster City, CA and DNASTAR sequence analysis software, DNASTAR Inc., Madison, WI) and confirmed to contain the desired mutation.<br>
Other plant class 1 EPSPS coding sequences were modified to contain the TIPA variant. The EPSPS coding sequence of Arabidopsis thaliana (Columbia) EPSPS 1 and EPSPS coding sequences of lettuce (Lactuca sativa) were isolated for mutagensis. RT-PCT was used to isolate the coding sequence of the mature protein of both AtEPSPSl and lettuce EPSPS. All of the primers were ordered from Invitrogen. The leaf tissues of both Arabidopsis and lettuce were ground into powder in liquid nitrogen with a mortar and pestle. Total RNA was isolated using Qiagene's RNeasy mini kit (cat. #74904) using lOmg<br>
33<br><br>
leaf powder and RNA was eluted into 50u.l water. RT-PCR reactions were performed using one-step-RT-PCR kit (Invitrogen #10928-034) in a 50 uL reaction containing: dHjO 24 uL; reaction buffer 50 uL; total RNA 20 uL; AtEPSPS-F primer (SEQ ID NO: 28) (lOuM) 1 uL; AtEPSPS-R (SEQ ID NO: 29) (10|iM) 1 |iL; Taq 1 \iL. RT-PCR was carried out on a MJ Research PTC-200 thermal cycler using the following program: Step 1 - 40°C for 30 seconds; Step 2 - 94°C for 2 minutes; Step 3 - 94°C for 20 seconds; Step 4 -65°C for 30 seconds; Step 5 - 68°C for 1 minute 30 seconds; Step 6 - go to step 3, 30 times; Step 7 - End. Both PCR reactions yielded an approximately 1.3 kilo base band on a I percent agarose electrophoresis gel. The lettuce EPSPS coding sequence was isolated using the primers LsEPSPS-F (SEQ ID NO:30) and LsEPSPS-R (SEQ ID NO:31) in the above described method. The RT-PCR products were cloned into PCR-1I vector (Invitrogen Corp.) and the DNA molecules sequenced.<br>
The Arabidopsis and lettuce EPSPS TIPA variants were generated using the PCR-based QuickChange™ Site-directed Mutagenesis Kit (Stratagene Cat, No.200518) and the DNA mutagenesis primers AtEPSPS-TIPA-F (SEQ ID NO:32) and AtEPSPS-TIPA-R (SEQ ID NO:33) for the Arabidopsis EPSPS coding sequence mutation, and LsEPSPS-TIPA-F (SEQ ID NO:34) and LsEPSPS-TlPA-R ($£Q ID NO:35) for the lettuce EPSPS coding sequence mutation and the PCR conditions were as described above. Two additional site-directed mutagenesis reactions were performed to engineer a restriction enzyme site, Ndel, at the 5' end and to remove an internal Ndel site in the lettuce EPSPS coding sequence. The DNA fragments of the Arabidopsis and the lettuce EPSPS TIPA variants were then digested with Ndel and Xhol and cloned into a pET19 vector. The DNA molecules that encode for the variant Arabidopsis EPSPS and variant lettuce EPSPS are shown in SEQ ID NO: 36 and SEQ ID NO: 37, respectively. Examples of plant and bacterial expression DNA constructs that were made with these variant EPSPS coding sequences<br>
34<br><br>
are illustrated in pMON81548 (LsTlPA variant, promoter of US patent 6,660,911) shown in Figure 11 and pMON58491 (AtTIPA variant) shown in Figure 12. Any of the variant EPSPS coding sequences of the present invention can be inserted into a plant expression cassette, for example, that contained in pMON81519 or pMON81548 by replacing the existing coding sequence.<br>
EXAMPLE 2<br>
EPSPS purification and enzyme analysis<br>
The wild type and variant EPSPS coding sequences were cloned into a pET-19b base vector (Novagen, Madison, WI). The plant (maize) class I EPSPS variants so created were assigned pMON plasmid numbers (Table 2). The variant EPSPS proteins were purified from the E. coli host using the protocols outlined in the pET system manual, 9th edition (Novagen) or by the following method. A single colony or a few microliters from a glycerol stock was inoculated into 4 mL LB medium containing 0.1 mg/mL carbenicillin antibiotic. The culture was incubated with shaking at 37 C for 4 hours. The cultures were stored at 4° C overnight. The following morning, 1 mL of the overnight culture was used to inoculate 100 mL of fresh LB medium containing 0.1 mg/mL carbenicillin. The cultures were incubated with shaking at 37 C for 4-5 hours, then the cultures were placed at 4°C for 5-10 minutes. The cultures were then induced with IPTG (1 mM final concentration) and incubated with shaking at 30 C for 4 hours or 20 C overnight. The cells were harvested by centrifugation at 7000 rpm for 20 minutes at 4° C. The supernatant was removed and the cells were frozen at -70°C until further use. The proteins were extracted by resuspending the cell pellet in BugBuster reagent (Novagen) using 5 mL reagent per gram of cells. Benzonase (125 Units) was added to the resuspension and the cell suspension was then incubated on a rotating mixer for 20 minutes at room temperature.   The cell<br>
35<br><br>
debris was removed by centifugation at 10,000 rpm for 20 minutes at room temperature. The supernatant was passed through a 0.45 u,m syringe-end filter and transferred to a fresh tube. A pre-packed columns containing 1.25 mL of His-Bind resin was equilibrated with 10 mL of 5 mM imidazole, 0.5 M NaCI, 20 mM Tris-HCl pH 7.9 (IX Binding Buffer). The column was then loaded with the prepared cell extract. After the cell extract had drained, the column was then washed with 10 mL of IX Binding Buffer, followed with 10 mL of 60 mM imidazole, 0.5 M NaCI, 20 mM Tris-HCl pH 7.9 (IX Wash Buffer). The protein was eluted with 5 mL of 1 M imidazole, 0.5 M NaCI, 20 mM Tris-HCl pH 7.9 (IX elution buffer). Finally the protein was dialyzed into 50 mM Tris-HCl pH 6.8. The resulting protein solution was concentrated to - 0.1-0.4 mL using Ultrafree - centrifugal device (Biomax-IOK. MW cutoff, Millipore Corp, MA). Proteins were diluted to 10 mg/mL and 1 mg/mL in 50 mM Tris pH 6.8, 30 percent final glycerol and stored at -20°C. Protein concentration was determined using Bio-Rad protein assay (Bio-Rad Laboratories, CA). Bovine serum albumin was used to generate a standard curve 1-5 u.g. Samples (10 (xL) were added to wells in a 96 well-plate and mixed with 200 uL of Bio-Rad protein assay reagent (1 part dye reagent concentrate: 4 parts water). The samples were read at OD595 after ~ 5 minutes using a spectraMAX 250 plate reader (Molecular Devices Corporation, Sunnyvale, CA) and compared to the standard curve.<br>
The EPSPS enzyme assays contained 50 mM K+-HEPES pH 7.0 and I mM shikimate-3-phosphate (Assay mix). The Km-PEP were determined by incubating assay mix (30 u.L) with enzyme (10 y.L) and varying concentrations of [,4C] PEP in a total volume of 50 u.L. Th*t reactions were quenched after various times with 50 uL of 90 percent ethanol/0.1 M acetic acid pH 4.5 (quench solution). The samples were centrifuged at 14,000 revolutions per minute and the resulting supernatants were analyzed for l4C-EPSP production by HPLC. The percent conversion of UC-PEP to l4C-EPSP was determined by<br>
36<br><br>
HPLC radioassay using an AX100 weak anion exchange HPLC column (4.6 x 250 mm, SynChropak) with 0.26 M isocratic potassium phosphate eluant, pH 6.5 at 1 mL/minute mixed with Ultima-Flo AP cocktail at 3 mL/min (Packard). Initial velocities were calculated by multiplying fractional turnover per unit time by the initial concentration of the substrate.<br>
The inhibition constant (K.j) were determined by incubating assay mix (30 u-L) with and without glyphosate and ,4C-PEP (10 uX of 2.6 mM). The reaction was initiated by the addition of enzyme (10 jiL). The assay was quenched after 2 minutes with quench solution. The samples were centrifuged at 14,000 rpm and the conversion of 14C-PEP to l4C-EPSP was determined as shown above. The steady-state and IC50 data were analyzed using the GraFit software (Erithacus Software, UK). The K| values were calculated from the IC50 values using the following algorithm: Kj = [IC] V (1 + [S]/Km). The assays were done such that the MC-PEP to 14C-EPSP turnover was 
The maize EPSPS variants that were cloned into pET-19b and from which proteins were expressed and assayed, included the double variants TIPS, TIPT, T1PG, TIPC, TIPA, T1PV, TIPM, TIPL, and TIPI; and single variants T1021, P106S, P106T, and P106L (Table 2). The enzymes were purified and assayed for apparent K.m of PEP (Km-PEP) and inhibition by glyphosate (K,). The TIPS variant is well known and is currently in the commercial Roundup Ready® corn product GA21 (U.S. Patent 6,040,497) and its kinetic parameters serve as the baseline value for a glyphosate resistant class I EPSPS enzyme that is sufficient to provide glyphosate tolerance to a transgenic plant. All the variants were characterized and the kinetic parameters are shown in Table 2. Substantial differences were observed between these variants. Surprisingly, the results showed that two of the new variants, TIPA and TIPT, were more<br>
37<br><br>
resistant to glyphosate than the TIPS variant and demonstrated a similar Km-PEP. These EPSPS enzyme double variants will provide enhanced glyphosate tolerance when appropriately expressed in transgenic plants. The variant TIPG has similar Km-PEP as the wild-type enzyme (WT), but has a K, of only 38.6 uM, not an improvement over TIPS, but this K, should be sufficient to provide glyphosate tolerance in transgenic plants when appropriately expressed. The variants TIPC and T1PI show a high level of resistance to glyphosate but have 1.7-fold and 2.2-fold higher Km-PEP than the wild-type enzyme, respectively. Although TIPC and TIPI are somewhat less efficient than the wild-type enzyme for Km-PEP, they do show a high level of resistance to glyphosate and when these are overexpressed as a transgene in plant cells, these enzymes should be sufficient to provide glyphosate tolerance. Other double variants that include TIPV, TIPM, TIPL showed very high resistance to glyphosate, but have significantly higher Km for PEP and therefore do not have sufficient substrate binding activity to provide useful EPSPS enzyme activity to a transgenic plant. Additional double variants TIPD and TIPN had a Km for PEP of 355 and 566, respectively and were not assayed for K, for glyphosate because the substrate binding activity was too inefficient for these variants to have effective EPSPS activity. For comparison purposes, the enzyme kinetics of the naturally occurring class II glyphosate resistant EPSPS isolated from Agrobacterium strain CP4 (CP4 EPSPSJ was expressed from pMON2I104 (RecA promoter/GIO leader/CP4 EPSPSA7 terminator) and assayed under the same conditions as the maize variant EPSPSs and demonstrated a Km-PEP of 14.4 JJ.M and K, for glyphosate of 5100 uM.<br><br><br><br>
EXAMPLE 3<br>
The maize EPSPS double variants ZmTIPT (SEQ ID NO: 38) and ZmTIPA (SEQ ID NO: 39) were made with as a CTP translational fusion into plant expression DNA constructs, pMON70472 (Figure 7) and pMON70475 (Figure 8), respectively. pMON30167 (Figure 6) was digested with Sphl/NotI and EcoRI, a Not 1/ Eco RI backbone DNA fragment of pMON30167 and a Sph I/Not I DNA fragment (rice actin promoter, P-OsActI and intron, I-Os.Actl, US Patent 5,641,876) were gel purified. A maize EPSPS-TIPT DNA molecule was isolated from pMON58452 by incorporating Sph I and Eco Rl endonuclease sites in the ends with DNA primer molecules ZmAroA-1 (SEQ ID NO:40) and ZmAroA-2 (SEQ ID NO:4I). The amplified mEPSPS-TIPT DNA fragment was digested with SphI and Eco RI and gel purified. A triple ligation was performed with the two pMON30167 fragments and the modified maize EPSPS-TIPT DNA fragment. The ligated plasmid was transformed into E. coli strain XLl-blue following the manufacturer's instruction and screened for colonies with the correct plasmid. The mature maize EPSPS N-terminus was restored (should be Ala not Met) by mutagenesis with Stratagene QuikChange kit according to manufacturer's instruction and using DNA<br><br>
primer molecules ZmAroA-3 (SEQ ID NO: 42) and ZmAroA-4 (SEQ ID NO: 43) contained in pMON70472 (Figure 7).<br>
EXAMPLE 4<br>
A DNA construct containing the TIPT variant (pMON70472, Figure 7) under the control of rice actin promoter was transformed into corn plant cells (LHI98 x Hill) by an Agrobacterium mediated transformation method. For example, a disarmed Agrobacterium strain C58 harboring the binary DNA construct of the present invention is used. The DNA construct is transferred into Agrobacterium by a triparental mating method (Ditta et al., Proc. Natl. Acad. Sci. 77:7347-7351, 1980). Liquid cultures of Agrobacterium are initiated from glycerol stocks or from a freshly streaked plate and grown overnight at 26°C-28°C with shaking (approximately 150 rpm) to mid-log growth phase in liquid LB medium, pH 7.0 containing the appropriate antibiotics. The Agrobacterium cells are resuspended in the inoculation medium (liquid CM4C) and the density is adjusted to ODMI of 1. Freshly isolated Type II immature HiIIxLH198 and Hill corn embryos are inoculated with Agrobacterium containing a construct and co-cultured several days in the dark at 23 °C. The embryos are then transferred to delay media and incubated at 28 °C for several or more days. All subsequent cultures are kept at this temperature. The embryos are transferred to a first selection medium containing carbenicillin 500/0.5 mM glyphosate). Two weeks later, surviving tissue is transferred to a second selection medium containing carbenicillin 500/1.0 mM glyphosate). Subculture surviving callus every 2 weeks until events can be identified. This may take about 3 subcultures on 1.0 mM glyphosate. Once events are identified, bulk up the tissue to regenerate. The plantlets (events) are transferred to MSOD media in culture vessel and kept for two weeks. The transformation efficiency is determined by dividing the number of events produced by the number of embryos inoculated. Then the plants with roots are transferred into soil. Those skilled in the art of monocot<br>
40<br><br>
transformation methods can modify this method to provide substantially identical transgenic monocot plants containing the DNA compositions of the present invention, or use other methods, such as, particle gun, that are known to provide transgenic monocot plants.<br>
The results of molecular analysis and glyphosate selection of the regenerated corn events transformed with a DNA construct containing ZmTIPT are shown in Table 3. The events were analyzed for single copy insertion in the corn genome and for vegetative and male fertility. Eight of the thirteen transgenic events (61%) containing the pMON70472 (TIPT) plant expression cassette were assayed using Taqman® analysis (ABI, Foster City, CA) and determined to be a single copy insert into the corn genome. The events were treated with a foliar application of 32 oz/acre of Roundup® Ultra at the V4 stage of corn development and then another foliar application of 64 oz/acre Roundup® Ultra at about V7. The treated plants were scored for vegetative (glypT) and reproductive tolerance (fertile) to glyphosate. Four of the eight single copy events (50%) showed no vegetative injury due to the glyphosate application at V4 (%V4 glypT). A 64 oz Roundup® Ultra treatment was applied at around the V7 stage and the plants were scored for vegetative glyphosate tolerance and male fertility (%V7gIypT/ferti(e). All four of the events 100% (4/4) that were single copy and vegetatively tolerant to glyphosate at V4 were also vegetatively tolerant and fully male fertile after the V7 treatment. Compared to the commercial standard glyphosate resistant EPSPS (CP4 EPSPS) the ZmTIPT variant performed surprisingly well. DNA constructs containing the TIPT variant of a class I EPSPS provide plants that are vegetatively and reproductively tolerant to a glyphosate containing herbicide.<br>
Table 3: Results of glyphosate tolerance treatment of corn events containing the TIPT EPSPS variant<br>
DNA Construct      #events      single copy     %V4 glypT     %V7 glypT/fertile<br>
41<br><br>
pMON70472	13	61%(8/13)      50% (4/8)	100% (4/4)<br>
(ZmTlPT)<br>
PMON30167	24	33% (8/24)      62% (5/8)	100?/o (5/5)<br>
(CP4 EPSPS)<br>
EXAMPLE 5<br>
The ZmTIPT variant was constructed into a plant expression construct suitable for dicot plant expression. The DNA construct was designated pMON81519 and is illustrated in Figure 10. The DNA construct and control constructs were transferred into Agrobacterium by a triparental mating method as previously described. The transformed Agrobacterium cells were used to transfer the plant expression cassette into Arabidopsis and tobacco cells.<br>
Arabidopsis embryos were transformed by an Agrobacterium mediated method essentially as described by Bechtold N, et al., CR Acad Sci Paris Sciences di la vie/life sciences 316: 1194-1199, (1993). An Agrobacterium strain ABI containing a DNA construct is prepared as inoculum by growing in a culture tube containing 10 mis Luria Broth and antibiotics. The Agrobacterium inoculum is pelleted by centrifugation and resuspended in 25 ml Infiltration Medium (MS Basal Salts 0.5%, Gamborg's B-5 Vitamins i%, Sucrose 5%, MES 0.5 g/L, pH 5.7) with 0.44 nM benzyl am inopurine (10 ul of a 1.0 mg/L stock in DMSO per liter) and 0.02% Silwet L-77 to an OD6oo of 0.6.<br>
Mature flowering Arabidopsis plants are vacuum infiltrated in a vacuum chamber with the Agrobacterium inoculum by inverting the pots containing the plants into the inoculum. The chamber is sealed, a vacuum is applied for several minutes, release the vacuum suddenly, blot the pots to remove excess inoculum, cover pots with plastic domes and place pots in a growth chamber at 21 °C 16 hours light and 70% humidity. Approximately 2 weeks after vacuum infiltration of the inoculum, cover each plant with a Lawson 511 pollination<br>
42<br><br>
bag. Approximately 4 weeks post infiltration; withhold water from the plants to permit dry down. Harvest seed approximately 2 weeks after dry down.<br>
The transgenic Arabidopsis plants produced by the infiltrated seed embryos are selected from the non-transgenic plants by a germination selection method. The harvested seed is surface sterilized then spread onto the surface of selection media plates containing MS Basal Salts 4.3 g/L, Gamborg's B-5 (500 X) 2.0 g/L, MES 0.5 g/L, and 8 g/L Phytagar with Carbenicillin 250mg/L, Cefotaxime 100 mg/L, and PPM 2 ml/L and 300 uM glyphosate added as a filter sterilized liquid solution, after autoclaving. The pMON8l519 VI events and control construct pMON8l517 glyphosate tolerant transgenic Arabidopsis plants are selected by spray application of glyphosate herbicide at a rate of 24 ounces/acre, the surviving plants are transplanted into individual pots. The VI plants are sprayed a second time corresponding to the observation of bolting, approximately 16 days after the at a rate of 24 ounces/acre. The second spray will determine the efficacy of the two constructs for conferring reproductive tolerance. The plants are observed for vegetative and reproductive effects of glyphosate application. Sixty-two plants were assayed that were transformed with the control construct pMONSI517 that contains the CP4 EPSPS (class II EPSPS) coding sequence; forty-nine plants were assayed that were transformed with pMON81519. The results shown in Table 4 demonstrate that the percentage of plants showing glyphosate tolerance and fertility is about the same for the Zm T1PT class I EPSPS variant as for the class II EPSPS.<br>
Tobacco is a well known model plant for testing of transgene constructs and the methods of transformation are well known in the art of plant transformation. Briefly, tobacco leaf tissue is cut and placed onto solid pre-culture plates containing the appropriate culture medium. The day before Agrobacterium inoculation, a 10 ul loop of a transformed Agrobacterium culture containing pMON8!519 or control construct is placed into a tube<br>
43<br><br>
containing 10 mis of YEP media with appropriate antibiotics to maintain selection of the DNA construct. The tube is put into a shaker to grow overnight at 28 °C. The OD60o of the Agrobacterium is adjusted to 0.15 - 0.30 ODeoo with TXD medium. Inoculate tobacco leaf tissue explants by pipetting 7-8 mis of the liquid Agrobacterium suspension directly onto the pre-culture plates covering the explant tissue. Allow the Agrobacterium to remain on the plate for 15 minutes. Tilt the plates and aspirate liquid off using a sterile 10 ml wide bore pipette. The explants are co-cultured on these same plates for 2-3 days, The explants are then transferred to fresh medium containing appropriate selection agents and maintained for 3-4 weeks at which time the callus tissue is transferred to fresh medium. At 6-8 weeks, shoots should be excised from the callus allowed to root in culture media. Rooted shoots are then transferred to soil after 2-3 weeks. The plants are treated with 16-24 oz/Acre glyphosate and scored for vegetative and reproductive tolerance. The results shown in Table 4 demonstrate that the percentage of plants showing glyphosate tolerance and fertility is about the same for the ZmTIPT class I EPSPS variant as for the class II EPSPS.<br>
Table 4: Results of glyphosate tolerance treatment of Arabidopsis and tobacco events containing the TIPT EPSPS variant<br>
Tobacco	Arabidopsis<br>
DNA Construct		% glvpT/fertile	%glypT/fertile<br>
PMON81517 (CP4 EPSPS)       56%(N=4I)	61%(N=62)<br>
PMON81519 (ZmTIPT)	49% (N=39)	65% (N=49)<br>
EXAMPLE 6<br>
Class I EPSPSs can be modified by site-directed mutagenesis methods or random mutagenesis method to provide an enzyme that is resistant to glyphosate. The present invention preferably provides amino acid substitutions of the Thrl02 and Prol06 positions. In addition to the previously described<br>
44<br><br>
TIP-T,G,C,A and I variants of the present invention, an additional substitution was performed of the Thrl02 codon was replaced with a leuerne(L, Leu) codon and the Prol06 codon was replaced with an alanine (A, Ala) codon by site-directed modification of the corresponding codons in a maize EPSPS DNA coding sequence resulting in a variant ZmTLPA (SEQ ID NO:44) that provides a glyphosate resistant enzyme. In another variant, the Thrl02 codon was replaced with a codon for Glutamine (Q, Gin), the Prol06 codon modified to an Ala codon, resulting in a TQPA variant.<br>
These maize EPSPS variants, TLPA and TQPA were generated using the PCR-based QuickChangeIM Site-directed Mutagenesis Kit by Stratagene (Cat. No: 200518). The unmodified maize EPSPS coding sequence was used as the template for PCR to generate the variants. The mutagenesis oligo primers were ordered from Invitrogen. The PCR was set up in a 50^1 reaction in the following manner: dH20 38 ul; 2 mM dNTP I pi,; 10X buffer 5 uL; pMON70461 1 pL (10 ng); ZmTLPA-1 (SEQ ID NO: 45) 2 pL; ZmTLPA-2 (SEQ ID NO: 46) 2 pL; pfu Turbo enzyme 1 pL. The PCR was carried out on a MJ Research PTC-200 thermal cycler using the following program: Step 1 -94°C for 2'; Step 2 - 94°C for30"; Step 3 - 55°C for 30"; Step 4 - 68°C for 14'; Step 5 - go to step 2, 16 times; Step 6 - End. At the end of the PCR, 1 ul of the restriction enzyme Dpnl was added to each 50 ul of PCR reaction and the mixture was incubated at 37°C for 1 hour. After the Dpnl treatment, 1 ul of the treated reaction mixture was used to transform the competent E. coii strain XL 1-blue strain (Stratagene) following the manufacturer's instruction. The transformed cells were plated on a Petri dish containing carbenicillin at a final concentration of 0.1 mg/mL. The plate was then incubated at 37°C overnight. Single colonies were picked the next day and used to inoculate a 3 mL liquid culture containing G.l mg/mL carbenicillin. The liquid culture was incubated overnight at 37"C with agitation at 250 rpm. Plasmid DNA was prepared from I mL of the liquid culture using Qiagen's miniprep Kit (Cat. No: 27160). The<br><br>
DNA was eluted in 50 yd of dH20. The entire coding region of three independent clones from each mutagenesis was sequenced and confirmed to contain the desired mutation. The variant coding sequences were inserted into a pET19 expression vector in the proper orientation to provide expression of the variant enzyme as a translational fusion with a purification tag.<br>
The mutant maize EPSPS enzymes (TLPA and TQPA) were assayed for catalytic activity, substrate binding, and resistance to glyphosate (K,) using the assay conditions previously described. The results are shown in Table 5, These mutants were compared to the wild type (WT) unmodified maize EPSPS, and the TIPA variant. The results provide evidence that the TLPA variant is resistant to glyphosate and has sufficient enzyme kinetics that when expressed in a transgenic plant will provide glyphosate tolerance to the transgenic plant when fused with a CTP or modified for chloroplast expression. Further amino acid substitutions at the 106 position that include threonine, glycine, cysteine and isolcucine are expected to result in a glyphosate resistant enzyme as observed in combination with the T-I modification at position 102.<br><br>
Having illustrated and described the principles of the present invention, it should be apparent to persons skilled in the art that the invention can be modified in arrangement and detail without departing from such principles.<br><br>
We claim all modifications that are within the spirit and scope of the appended claims.<br>
All publications and published patent documents cited in this specification are incorporated herein by reference to the same extent as if each individual publication or patent application was specifically and individually indicated to be incorporated by reference.<br><br><br><br><br><br><br>
I/We claim:<br>
1.	A DNA moiecule that encodes a glyphosate resistant class I EPSPS protein, wherein said glyphosate resistant class 1 EPSPS protein comprises a polypeptide sequence GX4X1X2^X3, where Xi and X; are any amino acid, X4 is isoleucine or leucine, and X-, is selected from the group consisting of threonine, cysteine, alanine, and isoleucine.<br>
2.	The DNA molecule of claim 1, wherein said DNA molecule is derived from a plant genome.<br>
3.	The DNA molecule of claim 2. wherein said DNA molecule is derived from Zea mays and is modified to encode said glyphosate resistant class I EPSPS protein comprising an isoleucine at position 102 and an amino acid at position 106 selected from the group selected from threonine, cysteine, alanine and isoleucine.<br>
4.	The DNA molecule of claim 2. wherein said DNA molecule is derived from Arabidopsis and is modified to encode said glyphosate resistant class I F.PSPS protein comprising an isoleucine at position 102 and an alanine at position 106.<br>
5.	The DNA molecule of claim 2. wherein said DNA molecule is derived from Lettuce and is modified to encode said glyphosate resistant class I EPSPS protein comprising an isoleucine at position 102 and an alanine at position 106.<br>
6.	The DNA molecule of claim 2, wherein said DNA molecule is derived from Zea mays and is modified to encode said glyphosate resistant class I EPSPS protein comprising an leucine at position 102 and an alanine at position 106.<br><br>
7.	The DNA molecule of claim 2. wherein said DNA molecule is selected from the group consisting of SEQ ID NO:36, SEQ ID NO:37, SEQ ID NO:38, SEQ ID NO:39 and SEQ ID NO:44.<br>
8.	The DNA molecule of claim 1, wherein said DNA molecule is derived from a bacterial genome.<br>
9.	A DNA construct comprising a promoter that functions in plant cells opcrably linked to a DNA molecule that encodes a glyphosate resistant class I EPSPS protein, wherein said glyphosate resistant class I EPSPS protein comprises a polypeptide sequence GXJXJX^RXJ, where X] and X; are any amino acid, and X4 is isoleucine or leucine and Xj is selected from the group consisting of threonine, cysteine, alanine, and isoleucine.<br>
10.	The DNA construct of claim 9, comprising a DNA molecule selected from the group consisting of SEQ ID NO:36. SEQ ID NO:37. SEQ ID NO:38, SEQ ID NO:39 and SEQ ID NO:44.<br>
11.	A method of preparing a glyphosate tolerant plant comprising the steps of:<br><br>
1)	contacting a recipient plant cell with the DNA construct of claim 9. wherein said DNA construct is incorporated into the genome of the recipient plant cell;<br>
2)	regenerating the recipient plant cell into a plant: and<br>
3)	applying an effective dose of glyphosate to the plant, wherein the plant displays a glyphosate tolerant phenotype.<br>
12.	The method of claim 11, wherein said DNA construct comprises a DNA<br>
molecule selected from the group consisting of SEQ ID NO:36. SEQ ID<br>
NO:37. SEQ ID NO:38. SEQ ID NO:39 and SEQ ID NO:44.<br><br>
13.	A method of treating a Held of glyphosate tolerant crop plants in order to<br>
control weeds therein, comprising applying to said field of glyphosate<br>
tolerant crop plant an effective dose of a glyphosate containing herbicide,<br>
wherein said glyphosate tolerant crop plant contains a DNA construct<br>
comprising a promoter that functions in plant cells operably linked to a<br>
DNA molecule that encodes a chloroplast transit peptide linked to a<br>
glyphosate resistant class I EPSPS protein, wherein said glyphosate<br>
resistant class I EPSPS protein comprises a polypeptide sequence<br>
GIX1X2RX3, where Xi and X: are any amino acid, and X3 is selected from<br>
the group consisting of threonine, cysteine, alanine, and isoleucine.<br>
14.	The method of claim 13, wherein said DNA construct comprises a DNA<br>
molecule selected from the group consisting of SEQ ID NO:36, SEQ ID<br>
NO:37, SEQ ID NO:38, SEQ ID NO:39 and SEQ ID NO:44.<br>
15.	A purified EPSPS enzyme comprising a polypeptide sequence<br>
GX4X1X2RX3. where X] and X2 are any amino acid, X4 is isoleucine or<br>
leucine, and X_; is selected from the group consisting of threonine, cysteine,<br>
alanine, and isoleucine.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkzNi1jaGVucC0yMDA1IGFic3RyYWN0LWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">1936-chenp-2005 abstract-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkzNi1jaGVucC0yMDA1IGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1936-chenp-2005 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkzNi1jaGVucC0yMDA1IGNsYWltcy1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1936-chenp-2005 claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkzNi1jaGVucC0yMDA1IGNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1936-chenp-2005 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkzNi1jaGVucC0yMDA1IGNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1936-chenp-2005 correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkzNi1jaGVucC0yMDA1IGNvcnJlc3BvbmRlbmNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">1936-chenp-2005 correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkzNi1jaGVucC0yMDA1IGRlc2NyaXB0aW9uIChjb21wbGV0ZSktZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">1936-chenp-2005 description (complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkzNi1jaGVucC0yMDA1IGRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">1936-chenp-2005 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkzNi1jaGVucC0yMDA1IGRyYXdpbmdzLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">1936-chenp-2005 drawings-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkzNi1jaGVucC0yMDA1IGRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1936-chenp-2005 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkzNi1jaGVucC0yMDA1IGZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">1936-chenp-2005 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkzNi1jaGVucC0yMDA1IGZvcm0tMTMucGRm" target="_blank" style="word-wrap:break-word;">1936-chenp-2005 form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkzNi1jaGVucC0yMDA1IGZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">1936-chenp-2005 form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkzNi1jaGVucC0yMDA1IGZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">1936-chenp-2005 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkzNi1jaGVucC0yMDA1IGZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">1936-chenp-2005 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkzNi1jaGVucC0yMDA1IGZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">1936-chenp-2005 form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkzNi1jaGVucC0yMDA1IHBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1936-chenp-2005 pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="229729-substituted-n-arylhetrocycles-method-for-production-and-use-thereof-as-medicaments.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="229731-peptide-inhibitors-of-thrombin-as-potent-anticoagulants.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>229730</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1936/CHENP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>19-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>16-Aug-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MONSANTO TECHNOLOGY, LLC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>800 North Lindbergh Boulevard, Saint Louis, Missouri 63167,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>HECK, Gregory, R</td>
											<td>2200 Divot Drive, Crystal Lake Park, MO 63131-3201,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>QI, Youlin</td>
											<td>936 Wellesley Place Drive, Chesterfield, MO 63017,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>STALLINGS, William, C</td>
											<td>19165 Old Logging Road, Wildwood, MO 63038,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>ALIBHAI, Murtaza, F</td>
											<td>16218 Forest Meadows Drive, Chesterfield, MO 63005,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>CAJACOB, Claire</td>
											<td>15207 Brightfield Manor Drive, Chesterfield, MO 63017,</td>
										</tr>
										<tr>
											<td>6</td>
											<td>FENG, Paul, C., C</td>
											<td>16519 Oak Forest Court, Wildwood, MO 63011,</td>
										</tr>
										<tr>
											<td>7</td>
											<td>FLASINSKI, Stanislaw</td>
											<td>2325 Laurenwood Drive, Chesterfield, MO 63017,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2004/004636</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-02-17</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/448,438</td>
									<td>2003-02-18</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/229730-glyphosate-resistant-class-i-5-enolpyruvylshikimate-3-phosphate-synthase-epsps by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:26:45 GMT -->
</html>
