<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/229754-biomarker-panels-for-detection-of-colorectal-cancer by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:28:12 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 229754:BIOMARKER PANELS FOR DETECTION OF COLORECTAL CANCER</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">BIOMARKER PANELS FOR DETECTION OF COLORECTAL CANCER</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A panel of biomarkers has been identified for analysis of colorectal cancer. The panel, originally identified using a mouse colon cancer model, has been used to assess changes in human tissue from surgical and biopsy samples against a normal human control panel of biomarkers. The panel may be used for providing a cost effective, rapid, noninvasive procedure for risk assessment, early diagnosis, establishing prognosis, monitoring patient treatment, detecting relapse, and for the discovery of therapeutic intervention of colorectal cancer.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>BIOMARKER PANEL FOR COLORECTAL CANCER<br>
Reference to Related Applications<br>
This application claims priority to U.S. Provisional Patent Application 60/488,660 entitled Molecular Marker Panel for Determination of Colorectal Cancer, by Nancy M. Lee et at., filed July 18.2003 (Attorney Docket CPMC-01000US0); and U.S. Patent Application 10/690,880 entitled Biomarker Panel for Colorectal Cancer, by Nancy M. Lee et al., filed October 22,2003 (Attorney Docket CPMC-01000US1), both of which are incorporated herein by reference.<br>
Background<br>
The field of art of this disclosure concerns biomarkers for colorectal cancer (CRC). These biomarkers are useful for risk assessment, early detection, establishing prognosis, evaluation of intervention, recurrence of CRC, and discovery of therapeutic intervention, and methods of use thereof.<br>
In the field of medicine, clinical procedures providing for the risk assessment and early detection of CRC have been long sought. Currently, CRC is the second leading cause of cancer-related deaths in the Western world. One picture that has clearly emerged through decades of research into CRC is that early detection is critical to enhanced survival rates.<br>
The currently accepted methods for CRC screening include the fecal occult blood test (FOBT), x-ray using double contrast between barium enema and air (DCBE), sigmoidoscopy, and colonoscopy. Sigmoidoscopy is an invasive procedure that visually examines the lower third of the colon using a lighted, flexible endoscope, while a related method, colonoscopy, is a procedure that examines the entire colon. In both cases, biopsy samples can be taken during the procedure.<br>
Concerning the accepted methods for screening, none clearly possess what is desired in a screening examination for CRC. While FOBT is<br><br>
rapid, it is a very general, and therefore a very non-specific screening method for CRC. Though DCBE has proven useful in specifically imaging abnormalities in the colon, the drawbacks of the DCBE method include: 1.) Patient discomfort in preparation of and during the examination, creating reluctance for compliance of DCBE as a screening method. 2.) Exposure of a patient to x-ray radiation, limiting DCBE in terms of frequency of use as a screening method. 3.) Research indicating that DCBE is more effective in detecting larger growths, which contraindicates its use for early detection. 4.) Biopsy samples cannot be taken during the procedure. 5.) Due to the cost involved, not all insurance providers pay for DCBE screening exams. Though sigmoidoscopy has gained favor from many physicians, the drawbacks of this method include: 1.) Patient discomfort in preparation of and during the examination, creating reluctance for compliance of sigmoidoscopy as a screening method. 2.) Due to the cost involved, not all insurance providers pay for sigmoidoscopy screening exams. 3.) Since only the lower third of the colon is inspected, there is a suggestion by studies that many significant lesions are in the proximal end of the colon, rendering sigmoidoscopy inadequate. Though colonoscopy addresses the issue of complete inspection of the colon, the drawbacks of colonoscopy as a screening method include: 1.) Creating even more patient discomfort than sigmoidoscopy, therefore generally requiring sedation, and thereby exacerbating the issue with patient compliance. 2.) Due to the cost involved, not all insurance providers pay for colonoscopy screening exams. 3.) There are risks of colonoscopy that include bleeding, and puncture of the lining of the colon.<br>
Emerging spectroscopic technologies, such as magnetic resonance imaging and tomographic imaging each have drawbacks that are drawn from the list of drawbacks for the currently accepted screening methodologies.<br>
Accordingly, there is a need in the art for approaches that have value in early detection and treatment of CRC that are cost effective, rapid, and<br><br>
minimally or noninvasive. Additional utility would be realized from an approach that would also serve as the basis for establishing prognosis, monitoring patient treatment, and detecting relapse, as well as the discovery of therapeutic intervention of CRC.<br>
Brief Description of Figures<br>
FIG 1 is a summary of the sequence listings.<br>
FIGS 2A-2C show data that illustrate a panel of biomarkers for samples taken from adenomous polyps, and suspect tissues vs. normal controls. FIGS 2A-2B are tables that compare the results of model studies done in mouse (2A) for a selection of members of the set of 22 biomarkers listed in the sequence listings with the comparable selection in of biomarkers for human subjects (2B).FIG 2C shows the multivariate analysis for 9 markers, for 78 biopsies taken from 12 normal patients and 63 biopsies taken from 6 patients with CRC.<br>
FIGS 3B-3C show expression levels for representative biomarkers, IL-8 (3A), CXCR-2 (3B), and COX-2 (3c) for a series of samples taken from a human subject comparing a histologically identified cancerous lesion, a polyp, and an adjacent non-cancerous tissue vs. a normal control.<br>
FIGS 4A-4C show the results of multiple analysis across a 53 cm distance of a colon for a patient with CRC: 4A shows expression levels for IL-8; 4B shows expression levels for COX-2; and 4c shows expression levels for CXCR-2.<br>
Detailed Description<br>
Still another sought after approach apart from currently accepted methods for screening for CRC, has been the search for biomarkers that have value in detection and treatment of CRC. For more than four decades, since the discovery of alpha-fetoprotein (AFP) and carcinogenic embryonic antigen (CEA), the search for biomarkers for cancer detection and treatment in general has been in a state of evolution. Biomarkers for cancer have five potential uses in the management of patient care. Ideally, they would be<br><br>
used for risk assessment, for early diagnosis, for establishing prognosis, for monitoring treatment, and for detecting relapse. Additionally, such markers could play a valuable role in developing therapeutic interventions.<br>
It is further advantageous for the sampling methods used in conjunction with biomarker analysis to be minimally invasive or non-invasive. Examples of such sampling methods include serum, stool, swabs, and the like. Non-invasive and minimally invasive methods increase patient compliance, and generally reduce cost.<br>
Clinically, the two criteria that are important for assessing the effectiveness of biomarkers are selectivity and sensitivity. Selectivity of a biomarker defined clinically refers to percentage of patients correctly diagnosed. Sensitivity of a biomarker in a clinical context is defined as the probability that the disease is detected at a curable stage. Ideally, biomarkers would have 100% clinical selectivity and 100% clinical sensitivity. To date, no single biomarker has been identified that has an acceptably high degree of selectivity and sensitivity required to be effective in for the broad range of needs in patient care management. However, from the clinical perspective, single serum biomarkers, such as AFP and CEA have proven to provide value in some aspects of patient care management.<br>
For example, elevated serum levels of CEA were first discovered in 1965 in patients with adenocarcinoma of the colon. Elevated levels can be found in a variety of benign and malignant conditions other than colon cancer. Additionally, the production of CEA by early localized tumors of the colon is in the normal range. Therefore CEA lacks both the sensitivity and selectivity required to be of value for risk assessment or early diagnosis. Further, elevated levels of CEA correlate poorly with colon tumor differentiation and stage, rendering CEA as a biomarker for prognosis of colon cancer of limited value. The two areas for which CEA has proven helpful clinically in managing patient care are in evaluating the effectiveness of treatment, and for detecting relapse. Illustrative of this, numerous studies<br><br>
have found that there is high correlation between elevated serum levels of CEA preceding clinical detection of recurrence of colon cancer. This has proven to be of value in managing the care of high-risk patents with second-look surgical procedures based on rising levels of CEA.<br>
Currently, investigations across numerous areas of oncology research, including CRC, ovarian, breast, and head and neck, are finding increased sensitivity and selectivity in panels of markers. It is now generally held that many mutations must take place before normal cell processes are altered, resulting in a disease, such as cancer. Still, given the complexity of biological systems, discovery of panels useful in providing value in patient care management for CRC is in the nascent stage.<br>
To date, a greater understanding of the biology of CRC has been gained through the research on adenomous polyposis coli (APC), p53, and Ki-ras genes, as well as the corresponding proteins, and related pathways involved in the regulation thereof. However, there is a distinct difference between research on a specific a gene, its expression, protein product, and regulation, and understanding what genes are critical to include in a panel used to for the analysis of CRC that is useful in the management of patient care for the disease. To date, panels that have been suggested for CRC are comprised of specific point mutations of APC, p53, and Ki-ras, as well as BAT-26, which is a gene that is a microstatelite instability marker.<br>
What is disclosed herein is based on studies conducted in mouse multiple intestinal neoplasia (MIN) model, in which expressions levels of genes were screened in adenomous polyps. In the mouse MIN subjects, a chemically induced mutation of the APC gene is effected. The normal control is defined by littermates for which there was no aberration of the APC gene, and are therefore designated wildtype. From studies based on the mouse MIN model, candidate genes were selected for studying human subjects. From these human subject studies, a panel of biomarkers is disclosed herein. Further, what is disclosed are methods for measuring gene and<br><br>
protein expression levels based on the panel. Additionally, another aspect of what is disclosed are kits which provide the reagents and instructions for measuring gene and protein expression levels based on the panel. The panel, methods and kits are useful in the management of patient care for CRC. Additionally, the panel, methods, and kits are believed useful as the basis for discovery of therapeutic interventions for CRC.<br>
FIG 1 is a table that gives an overview of the sequence listing for the disclosed biomarkers. The combination of biomarkers disclosed forms the basis for monitoring CRC with enhanced selectivity and sensitivity, and therefore providing enhanced management of patient care for CRC. It is to be understood that fragments and variants of the biomarkers described in the sequence listings are also useful biomarkers in a panel used for the analysis of CRC. What is meant by fragment is any incomplete or isolated portion of a polynucleotide or polypeptide in the sequence listing. It is recognized that almost daily, new discoveries are announced for gene variants, particularly for those genes under intense study, such as genes implicated in diseases like cancer. Therefore, the sequence listings given are exemplary of what is now reported for a gene, but it recognized that for the purpose of an analytical methodology, variants of the gene, and their fragments are also included.<br>
One embodiment of what is disclosed is a panel of biomarkers with the selectivity and sensitivity required for managing patient care for CRC. In Table 1, entries 1-22 are the polynucleotide coding sequences for a panel of biomarkers, and include the name and abbreviation of the gene. Entries 23-44 in Table 1 are the protein, or polypeptide, amino acid sequences that correspond to the coding sequences for entries 1-22. A biomarker, as defined by the National Institutes of Health (NIH) is a molecular indicator of a specific biological property; a biochemical feature or facet that can be used to measure the progress of disease or the effects of treatment. A panel of biomarkers is a selection of biomarkers. Biomarkers may be from a variety of<br><br>
classes of molecules. As previously mentioned, there is still a need for biomarkers for CRC having the selectivity and sensitivity required to be effective for all aspects of patient care management. Therefore, the selection of an effective set of biomarkers is differentiating in providing the basis for effective determination of CRC.<br>
In another embodiment of this disclosure, expression levels of polynucleotides for the biomarkers indicated in SEQ ID NOs 1-22, are used in the determination of CRC. Such analysis of polynucleotide expression levels is frequently referred to in the art as gene expression profiling. In gene expression profiling, levels of mRNA in a sample are measured as a leading indicator of a biological state, in this case, as an indicator of CRC. One of the most common methods for analyzing gene expression profiling is to create multiple copies from mRNA in a biological sample using a process known as reverse transcription. In the process of reverse transcription, the mRNA from the sample is used to create copies of the corresponding DNA sequence from which the mRNA was originally transcribed. In the reverse transcription amplification process, copies of ONA are created without the regulatory regions in the gene known as introns. These multiple copies made from mRNA are therefore referred to as copy DNA, or cDNA. Entries 45-88 are the sets of primers used in the reverse transcription process for each gene listed in entries 1-22.<br>
Since the reverse transcription procedure amplifies copies of cDNA proportional to the original level of mRNA in a sample, it has become a standard method that allows the analysis of even low levels of mRNA present in a biological sample. Genes may either be up regulated or down regulated in any particular biological state, and hence mRNA levels shift accordingly.<br>
In still another embodiment of this disclosure, expression levels of proteins listed in SEQ ID NOs 23-44, which correspond to the genes indicated in SEQ ID NOs 1-22, are disclosed. The term  polypeptide  or<br><br>
 polypeptides  is used interchangeably with the term  protein  or  proteins  herein. As discussed previously, proteins have been long investigated for their potential as biomarkers, with limited success. There is value in protein biomarkers as complementary to polynucleotide biomarkers. Reasons for having the information provided by both types of biomarkers include the current observations that mRNA expression levels are not good predictors of protein expression levels, and that mRNA expression levels tell nothing of the post-translational modifications of proteins that are key to their biological activity. Therefore, in order to understand the expression levels of proteins, and their complete structure, the direct analysts of proteins is required.<br>
FIGS 2A-2B show an exemplary panel of biomarkers from the list of 22 biomarkers for which gene expression levels are compared in the mouse MIN model, and in human subjects. The selection for the panel is taken from across the list of the 22 biomarkers and is taken for the purpose of easy visual assimilation of data in order to demonstrate the utility of a panel. Typically, for complex data sets represented in the 22 member panel of biomarkers, multivariate analysis (MANOVA)*is applied, such as that demonstrated in FIG 2C.<br>
In FIG 2A, the data reported for the mouse MIN studies represent statistical averaging of a number of animal subjects, and the standard error is reported. The p value on the right indicates the degree of confidence that the values are significantly different. As an example, the first gene listed, SDF-1, is related to the human IL-8 gene, and is in the same super family. For SDF-1, the p value of 0.003 indicates that the probability that the differences in the values of the wildtype control and that of the adenomous polyps of the MIN mice occurred by chance alone is only 3 in 1000. Screening the expression levels in adenomous polyps in the subject mice was specifically targeted, since it has been established that adenomous polyps are useful in risk assessment for CRC. What is demonstrated in FIG<br><br>
2A is that the panel of 6 clearly differentiate the results of the MIN mice over that of the wildtype control.<br>
FIGS 2B-2C address the issue of selectivity for biomarker panels. Regarding biomarkers that have an acceptable level of selectivity for CRC, the incidence of CRC for individuals in families with a history of CRC is 3-4 times that of the general population. However, It is now estimated that 6% of all Americans will develop CRC, and of those 70-80% will occur in people of average risk. There is clearly a need for biomarkers that have the necessary selectivity required for confidence in the determination of CRC.<br>
In FIG 2B, the same panel of 6 biomarkers established in the mouse MIN model in FIG 2A are the basis for determination of CRC in human subjects. In FIG 2B, the results of biopsy tissue determined to be normal by histological evaluation taken from patients known to have CRC are compared to biopsy tissue from individuals validated as normal controls. It should be noted that histological methodologies are the accepted standard for the identification of a cancerous colonic lesion. There are two aspects of FIG 2B to further discuss. First, values for gene expression profiling for patient vs. normal control may vary either up, as in the case of IL 8, or down, as in the case of PPAR-5. It is the determination of the collective shift for the patient vs. normal control that is significant when using a panel of biomarkers. Second, in glancing through the patient data, sample-to-sample variation can be noted, which is anticipated, given all the patient-to-patient variables. It is clear at a glance that the expression levels for the panel taken as a group distinguish the patient samples overall from the normal control group, even though a value for any one specific biomarker may not in itself distinguish the patient sample from the normal control. For example, the patient designated as H008 has an expression level for PPAR-5 that is not distinct from the normal control. However, at a glance it is dear that the results of the panel for H008 distinguish it from the normal control set. This demonstrates in principle why a validated panel of markers, given the<br><br>
complexity and variability of biology, enhance the selectivity of a determination vs. a single marker alone.<br>
Fig 2c further serves to emphasize the value of a panel of biomarkers in enhancing the selectivity of a determination between patient vs. normal samples. An example of demonstrating the use of MANOVA for a panel of 9 biomarkers selected from the group of 22 is demonstrated in FIG 2C. In this study, 78 sigmoidal-rectal biopsies from 12 normal patients, and 63 sigmoidal-rectal biopsies from non-cancerous sections of 6 patients with sigmoidal-rectal carcinoma were compared. The Wilks' Lambda criterion was used to assess the difference between the patient samples and normal control samples using the 9 biomarkers listed. The lambda value close to 1.0 signifies a significant difference between the patient and normal samples is indicated, with the probability of about 9 chances in 1000 that the difference is by chance alone.<br>
FIGS 3A-3C and FIGS 4A-4C address the issue of sensitivity for biomarker panels. As previously mentioned, since survival rates are greatly enhanced with the earliest indication of CRC, biomarkers for risk assessment and early detection of CRC have been long sought. The difference between risk assessment and early detection is the degree of certainty regarding acquiring CRC. Biomarkers that are used for risk assessment confer less than 100% certainty of CRC within a time interval, whereas biomarkers used for early detection confer an almost 100% certainty of the onset of the disease within a specified time interval. Risk factors may be used as surrogate end points for individuals not diagnosed with cancer, providing they there is an established relationship between the surrogate end point and a definitive outcome. An example of an established surrogate end point for CRC is the example of adenomous polyps. What has been established is that the occurrence of adenomous polyps are a necessary, but not sufficient condition for an individual to later develop CRC. This is demonstrated by the fact that 90% percent of all preinvasive<br><br>
cancerous lesions are adenomous polyps or precursors, but not all individuals with adenomous polyps go on to later develop CRC.<br>
FIGS 3A-3C show graphs of gene expression levels taken for multiple biopsy samples taken from the colon of one exemplary patient diagnosed with CRC. The determination of cancerous lesions, polyps, and adjacent tissues was made by conventional histological methods. The expression levels for three of the panel of biomarkers are shown for the biopsy samples categorized in that fashion. Again, as was demonstrated with the examples given in FIGS 2A-2C, it is evident that the three markers taken together for the cancerous lesions sampled are significantly different than the normal controls, even though one by itself (CXCR2) would not have been differentiating for this patient. What is additionally indicated in this representation is the distinction between the results of the polyp vs. the normal control. Given that polyps are already accepted as surrogate endpoints for CRC, then a determination of the presence of polyps by a validated analytical methodology using a minimally invasive method, such as a swab, or a non-invasive sampling method, such as a stool sample, would also serve as surrogate end point for risk assessment.<br>
FIGS 4A-4C show the results of gene expression levels for three of the biomarkers in biopsy samples taken over a 53 cm region of the colon of a patient with CRC. The irregularly shaped objects represent biopsy samples that were confirmed to be cancerous lesions by histological methodology, while the oval shapes represent samples that were determined to be non-cancerous by histological methodology. Gene expression profiling was done for each of the biopsy samples, as well. The results of the expression profiling, where the legend indicates relative levels in the patient biopsy samples as compared to normal controls, are depicted in FIGS 4A-4C.<br>
The representation of FIGS 4A-4C indicates the distance over which the biomarkers are able to distinguish differences in the colon tissue for the patient, where these biopsy samples were rendered normal by conventional<br><br>
histological analysis. These results demonstrate that it is possible to sample cells through a minimally invasive swabbing collection method from an area distant from a cancerous lesion, but capable of indicating a non-normal colon condition. Moreover, collection of a stool sample is an already validated sampling method for collecting sloughed cells or cell debris from which these determinations may be made. In that regard, samples taken either minimally invasively or non-invasively would render samples that could be analyzed using the disclosed panel of biomarkers. Such non-invasive procedures not only reduce the cost of determination of CRC, but reduce the discomfort and risk associated with current methodology. All these factors together increase the attractiveness of regular testing, and hence patient compliance. Increased patient compliance, coupled with an effective determination for CRC, enhance the prospects for early detection, and enhanced survival rates.<br>
Methods and kits for the polynucleotide and polypeptide expression profiling for the panel of molecular markers are also contemplated as part of the present disclosure.<br>
In one embodiment, a method for gene expression profiling comprises measuring cDNA levels for biomarkers selected in the claimed panel. Such a method requires the use of primers, enzymes, and other reagents for the preparation, detection, and quantitation of cDNAs. The method of creating cDNA from mRNA in a sample is referred to as the reverse transcriptase polymer chain reaction (RT-PCR). The primers listed in SEQ ID NOs 45-88 are particularly suited for use in gene expression profiling using RT-PCR based on the claimed panel. A series of primers were designed using Primer Express Software (Applied Biosystems, Foster City, CA). Specific candidates were chosen, and then tested to verify that only cDNA was amplified, and not contaminated by genomic DNA. The primers listed in SEQ ID NOs 45-88 were specifically designed, selected, and tested accordingly. In addition to the primers, reagents such as one including a dinucleotide triphosphate<br><br>
mixture having all four dinucleotide triphosphates (e.g. dATP, dGTP, dCTP, and dTTP), one having the reverse transcriptase enzyme, and one having a thermostable DNA polymerase are required for RT-PCR. Additionally buffers, inhibitors and activators are also required for the RT-PCR process. Once the cDNA has been sufficiently amplified to a specified end point, the cDNA sample must be prepared for detection and quantitation. Though a number of detection schemes are contemplated, as will be discussed in more detail below, one method contemplated for detection of poiynucleotides is fluorescence spectroscopy, and therefore chromophores that are suited to fluorescence spectroscopy are desirable for labeling poiynucleotides. One example of such a fluorescent label is SYBR Green, though numerous related chromophores exist, and are known in the art.<br>
In another embodiment, a method for protein expression profiling comprises using an antibody panel based on the claimed panel of biomarkers for measuring targeted polypeptide levels from a biological sample. In one embodiment contemplated for the method, the antibodies for the panel are bound to a solid support. The method for protein expression profiling may use a second antibody having specificity to some portion of the bound polypeptide. Such a second antibody may be labeled with molecules useful for detection and quantitation of the bound polypeptides, and therefore in binding to the polypeptide label it for detection and quantitation. Additionally, other reagents are contemplated for labeling the bound polypeptides for detection and quantitation. Such reagents may either directly label the bound polypeptide or, analogous to a second antibody, may be a moiety with specificity for the bound polypeptide having labels. Examples of such moieties include but are not limited to small molecules such as cofactors, substrates, complexing agents, and the like, or large molecules, such as iectins, peptides, olionucleotides, and the like. Such moieties may be either naturally occurring or synthetic.<br>
Examples of detection modes contemplated for the disclosed<br><br>
methods include, but are not limited to spectroscopic techniques, such as fluorescence and UV-Vis spectroscopy, scintillation counting, and mass spectroscopy. Complementary to these modes of detection, examples of labels for the purpose of detection and quantitation used in these methods include, but are not limited to chromophoric labels, scintillation labels, and mass labels. The expression levels of polynucleotides and polypeptides measured using these methods may be normalized to a control established for the purpose of the targeted determination. These methods are believed useful in providing determinations as the basis of effective management of patient care for CRC. These methods may also be used in the discovery of therapeutic interventions for CRC. Additionally, not only biopsy samples from sigmoidoscopy, colonoscopy, or surgery may be analyzed by these methods, but biological samples from non-invasive or minimally evasive collection methods are indicated for these methods, as well.<br>
It is further contemplated in what is disclosed to provide kits having the reagents and procedures that facilitate the ready implementation of the methods, and provide consistency and quality control thereby.<br>
In one embodiment, a kit for gene expression profiling comprises the reagents and instructions necessary for the gene expression profiling of the claimed panel. Thus, for example, the reagents may include primers, enzymes, and other reagents for the preparation, detection, and quantitation of cDNAs for the claimed panel of biomarkers. As discussed above, the method of creating cDNA from mRNA in a sample is referred to as the reverse transcriptase polymer chain reaction (RT-PCR). The primers listed in SEQ ID NOs 45-88 are particularly suited for use in gene expression profiling using RT-PCR based on the claimed panel. The primers listed in SEQ ID NOs 45-88 were specifically designed, selected, and tested accordingly. In addition to the primers, reagents such as one including a dinucleotide triphosphate mixture having all four dinucleotide triphosphates (e.g. dATP, dGTP, dCTP, and dTTP), one having the reverse transcriptase<br><br>
enzyme, and one having a thermostable DNA polymerase are required for RT-PCR. Additionally buffers, inhibitors and activators used for the RT-PCR process are suitable reagents for inclusion in the kit embodiment. Once the cONA has been sufficiently amplified to a specified end point, the cDNA sample must be prepared for detection and quantitation. One method contemplated for detection of polynucleotides is fluorescence spectroscopy, and therefore chromophores that are suited to fluorescence spectroscopy are desirable for labeling polynucleotides and may also be included in reagents of the kit embodiment. Instructions included with the kit embodiment for gene expression profiling preferably teach the user the following steps: to obtain a biological sample; to isolate cellular RNA from the sample; to amplify copies of cDNA from the sample for each biomarker in the panel, and the panel for which the reagents are provided; and to quantify levels of cDNA amplified from the sample. Though tissue samples from a variety of procedures may be used, the instructions for obtaining a biological sample are preferably whereby the user obtains a sample of colorectal cells in a minimally invasive manner, such as by use of a swab or collection of a stool sample. The instructions may also preferably include the step of comparing the cDNA levels quantified to a control.<br>
In another embodiment, a kit for protein expression profiling comprises the reagents and instructions necessary for protein expression profiling of the claimed panel. Thus, in this embodiment, the kit for protein expression profiling includes supplying an antibody panel based on the claimed panel of biomarkers for measuring targeted polypeptide levels from a biological sample. One embodiment contemplated for such a panel includes the antibody panel bound to a solid support. Additionally, the reagents included with the kit for protein expression profiling may use a second antibody having specificity to some portion of the bound polypeptide. Such a second antibody may be labeled with molecules useful for detection and quantitation of the bound polypeptides, and therefore in binding to the<br><br>
polypeptide label it for detection and quantitation. Additionally, other reagents are contemplated for labeling the bound polypeptides for detection and quantitation. Such reagents may either directly label the bound polypeptide or, analogous to a second antibody, may be a moiety with specificity for the bound polypeptide having labels. Examples of such moieties include but are not limited to small molecules such as cofactors, substrates, complexing agents, and the like, or large molecules, such as lectins, peptides, olionucleotides, and the like. Such moieties may be either naturally occurring or synthetic. Instructions for the protein expression profiling kit preferably teach the user to obtain a biological sample; to use the antibody panel supplied with the kit for each biomarker in the panel to bind the polypeptides from the sample; and to quantify levels of polypeptides bound from the sample to the antibody panel. Preferably, the kit instructions also include a step of comparing the polypeptide levels to a control. Preferably the biological sample is obtained by a minimally invasive procedure such as use of a swab to through a stool sample.<br>
Additionally, consumable labware required for sample collection, preparation, and analysis may be provided with the kits.<br>
What has been disclosed herein has been provided for the purposes of illustration and description. It is not intended to be exhaustive or to limit what is disclosed to the precise forms described. Many modifications and variations will be apparent to the practitioner skilled in the art. What is disclosed was chosen and described in order to best explain the principles and practical application of the disclosed embodiments of the art described, thereby enabling others skilled in the art to understand the various embodiments and various modifications that are suited to the particular use contemplated. It is intended that the scope of what is disclosed be defined by the following claims and their equivalence.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDU2OC1jaGVucC0yMDA2IGFic3RyYWN0IGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">0568-chenp-2006 abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDU2OC1jaGVucC0yMDA2IGNsYWltcyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">0568-chenp-2006 claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDU2OC1jaGVucC0yMDA2IGRlc2NyaXB0aW9uKGNvbXBsZXRlKSBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">0568-chenp-2006 description(complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDU2OC1jaGVucC0yMDA2IGRyYXdpbmdzIGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">0568-chenp-2006 drawings duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY4LUNIRU5QLTIwMDYgQ09SUkVTUE9OREVOQ0UgT1RIRVJTLnBkZg==" target="_blank" style="word-wrap:break-word;">568-CHENP-2006 CORRESPONDENCE OTHERS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY4LUNIRU5QLTIwMDYgQ09SUkVTUE9OREVOQ0UgUE8ucGRm" target="_blank" style="word-wrap:break-word;">568-CHENP-2006 CORRESPONDENCE PO.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY4LUNIRU5QLTIwMDYgRk9STSAxOC5wZGY=" target="_blank" style="word-wrap:break-word;">568-CHENP-2006 FORM 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY4LUNIRU5QLTIwMDYgUE9XRVIgT0YgQVRUT1JORVkucGRm" target="_blank" style="word-wrap:break-word;">568-CHENP-2006 POWER OF ATTORNEY.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY4LWNoZW5wLTIwMDYtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">568-chenp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY4LWNoZW5wLTIwMDYtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">568-chenp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY4LWNoZW5wLTIwMDYtY29ycmVzcG9uZG5lY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">568-chenp-2006-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY4LWNoZW5wLTIwMDYtZGVzY3JpcHRpb24oY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">568-chenp-2006-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY4LWNoZW5wLTIwMDYtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">568-chenp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY4LWNoZW5wLTIwMDYtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">568-chenp-2006-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY4LWNoZW5wLTIwMDYtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">568-chenp-2006-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY4LWNoZW5wLTIwMDYtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">568-chenp-2006-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY4LWNoZW5wLTIwMDYtcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">568-chenp-2006-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="229753-confidential-block-trading-system-and-method.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="229755-disposable-undergarment-and-method-for-the-manufacture-thereof.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>229754</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>568/CHENP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>20-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>15-Feb-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>INTELLIGENESCAN, INC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>3702 AUTUMN GLEN COURT, SANTA ROSA, CALIFORNIA 95403,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>LEE, Nancy, M</td>
											<td>1830 Funston Avenue, San Francisco, CA 94116,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>CHEN, Ling, C</td>
											<td>510 Lowell Place, Fremont, CA 94536,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>G01N</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US04/22594</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-07-14</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>10/690,880</td>
									<td>2003-10-22</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/488,660</td>
									<td>2003-07-18</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/229754-biomarker-panels-for-detection-of-colorectal-cancer by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:28:13 GMT -->
</html>
