<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/229709-a-recombinant-subgroup-b-adenovirus-particle-comprising-an-adenovirus-genome by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:25:28 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 229709:A RECOMBINANT SUBGROUP B ADENOVIRUS PARTICLE COMPRISING AN ADENOVIRUS GENOME&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A RECOMBINANT SUBGROUP B ADENOVIRUS PARTICLE COMPRISING AN ADENOVIRUS GENOME&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>ABSTRACT 2401/CHENP/2004 &quot;A recombinant subgroup B adenovirus particle comprising an adenovirus genome&quot; The present invention relates to a recombinant subgroup B adenovirus particle comprising an adenovirus genome wherein the genome comprises a deletion in the El-region, which adenovirus genome does not encode a functional E1B 55K gene product, said adenovirus genome comprising a pIX coding sequence under control of an expression sequence, said expression sequence comprising part of the 3&#x27; end of the E1B 55K coding sequence containing from 150 to 600 nucleotides of adenovirus sequences directly upstream of the pIX open reading frame of a wild-type subgroup B adenovirus, and said expression sequence being capable of increasing expression of the pIX coding sequence in a packaging cell, relative to the expression of said pIX coding sequence when preceded by sequences between the E1B 55K stop codon and the start codon of pIX and without nucleotides from the 3&#x27; end of the E1B 55K coding sequence.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM 2<br>
THE PATENTS ACT, 1970<br>
(39 of 1970)<br>
COMPLETE SPECIFICATION<br>
(See Section 10; rule 13)<br>
TITLE<br>
"A recombinant subgroup B adenovirus particle comprising an adenovirus genome"<br>
4-	APPLICANT<br>
£	CRUCELL HOLLAND B.V.<br>
of Archimedesweg 4,<br>
*	NL-2333 CN Leiden ,<br>
A.	The Netherlands;<br>
°*	Nationality : a Dutch company<br>
The following specification particularly describes<br>
the invention and the manner in which<br>
it is to be performed<br><br>
FIELD OF THE INVENTION The invention relates to the field of medicine, more in particular the invention relates to recombinant adenoviral vectors and the use thereof.<br>
BACKGROUND OF THE INVENTION<br>
Human adenoviruses are non-enveloped icosahedrai particles of 60-90 nM size. To date 51 serotypes have been identified that are subdivided into 6 subgroups based on hemagglutination properties and sequence homology (Francki et al., 1991). The genome has a length of 34 to 36 kb and is flanked on both sites by inverted terminal repeat sequences (ITR). The virus infectious cycle is divided into an early and a late phase. In the early phase (6-8 hours after infection) the virus is uncoated and the genome transported to the nucleus after which the early gene regions E1-E4 become transcriptionally active.<br>
The early region-1 (El) contains two transcription regions named E1A and E1B. The E1A region encodes two major proteins that are involved in modification of the host cell cycle and activation of the other viral transcription regions (reviewed by Russell, 2000). The E1B region encodes two major proteins, 19K and 55K, that prevent, via different routes, the induction of apoptosis resulting from the activity of the E1A proteins (Rao et al.,1992; Yew and Berk, 1992; reviewed in Shenk, 1996). In addition, the E1B-55K protein is required in the late phase for selective viral inRNA transport and inhibition of host protein expression (Pilder et al., 1986).  Early region-2 (E2) is also divided in an E2A and E2B region that together encode three proteins, DNA binding protein, viral<br>
2-<br><br>
polymerase and pre-terminal protein, all involved in replication of the viral genome (reviewed by van der Vliet, 1995).  The E3 region, is not necessary for replication in vitro but encodes several proteins that subvert the Host defence mechanism towards viral infection (reviewed by Horwitz, 2001). The E4 region encodes at least 6 proteins involved in several distinct functions related to viral mRNA splicing and transport, host cell mRNA transport, viral and cellular transcription and transformation (reviewed by Leppard, 1997) .<br>
The late proteins, necessary for formation of the viral capsids and packaging of viral genomes, are all generated fron the major late transcription unit (MLTU) that becomes fully active after the onset of replication. A complex process of differential splicing and poly-adenylation gives rise to more than 15 mRNA species that share a tripartite leader sequence. The early proteins E1B 55K and E4 Orf 3 and Orf6 play a pivotal role in the regulation of late viral mRNA processing and transport from the nucleus (reviewed in Leppard, 1998).<br>
Packaging of newly formed viral genomes in pre-formed capsids is mediated by at least two adenoviral proteins, the late protein 52/55K and an intermediate protein IVa2, through interaction with the packaging sequence located at the left end of the genome (Grable and Hearing, 1990; Gustin and Imperiale, 1998; Zang et al., 2001). A second intermediate protein, pIX, is part of the capsid and is known to stabilize the hexon-hexon interactions (Furcinitti et al., 1989) . In addition, pIX has been described to transactivate TATA containing promoters like the E1A promoter and MLP (Lutz et al., 1997).<br>
Due to the extensive knowledge of the viral biology and the high efficiency of nuclear delivery after entry into<br>
3<br><br>
cells, adenoviruses have become popular tools for gene delivery into human cells. In addition, adenoviral vectors are stable and can be produced relatively easy at large scale. In most cases vectors are deleted for at least the El region, which renders them replication deficient. Production of El-deleted vectors based on subgroup C serotypes Ad5 or Ad2 is achieved in El complementing cell lines such as 293 (Graham et al., 1970), 911 (Fallaux et al., 1996) and PER.C6TM (Fallaux et al., 1998). As disclosed in US patent 5,994,128 vectors and cell line need to be carefully matched to avoid generation of replication competent adenoviruses trough homologous recombination between adenovirus sequences in the cell line and the vector. Thus, PER.C6™ cells and matched adenoviral vectors provide a preferred system for the production of group C adenoviral vectors (Fallaux et al., 1998). The deletion of El sequences provides space for the introduction of foreign genes in the viral vector. Since the maximum size of Ad5 genomes that can be incorporated into virions is limited to about 105% of the wt length, El-deleted viruses can accommodate approximately 4.8 kb of foreign DNA (Bett et al., 1993) .<br>
The maximum packaging capacity in virions that lack pIX is reduced to approximately 95% of the normal genome length (Ghosh-Choudhury et al., 1987). This is most likely caused by the reduced stability of pIX- ("pIX-minus") virions. The deficiency in pIX-roinus mutant Ad5 can be complemented by episomal expression of pIX in a packaging cell line used for producing viruses (Caravokyri et al., 1995).<br>
Although the serotypes Ad5 and Ad2 are most commonly used as gene transfer vectors, other serotypes may have preferred characteristics that make them more useful as a therapeutic or prophylactic tool. Subgroup 8 viruses Ad35 and Adll for<br><br>
example are much less prone to neutralisation by human sera than Ad5 and Ad2 viruses (disclosed in WO 00/70071). Neutralisation of adenoviral transfer vectors diminishes transduction efficiency in vivo. Furthermore, the infection efficiency of antigen presenting cells, like dendritic cells, by recombinant viruses carrying the fiber of Ad35 was found to be greatly enhanced in vitro compared to Ad5 viruses (WO 00/70071, WO 02/24730). Thus, Ad35-based vectors combine highly improved infection efficiency with low neutralisation in human sera, making such vectors suitable for vaccination purposes.<br>
Generation and propagation of fully El-deleted Ad35-based vectors is possible using technology discussed below. However, careful analysis of a variety of recombinant Ad35-based vectors has revealed that such vectors are less stable, i.e. can contain less foreign DNA compared to the Ad5-based vectors. In the current patent application means and methods are presented to overcome this problem.<br>
In addition, there is a need to further develop the presently available technology for adenoviruses that have broader serotype utility. Existing packaging cell lines typically comprises El encoded proteins derived from adenovirus serotype 5. Examples of such ^standard' packaging cell lines are 293, 911 and PER.C6™. Attempts to produce vectors derived from other serotypes on these standard packaging cell lines have proven arduous if not unsuccessful. Occasionally, some production is seen, depending on the particular serotype used. However, the yields of recombinant adenovirus vectors derived from adenovirus subgroups other than subgroup C, produced on cell lines transformed and immortalized by El from Ad5, is poor. In a paper by Abrahamsen et al. (1997), improved plaque purification of an ElA-deleted<br><br>
adenovirus serotype 7 vector {subgroup B) was observed on 2 93 cells comprising E4-orf6 derived from adenovirus serotype 5, as compared to 293 cells lacking the E4-orf6 sequence from Ad5. However, a problem was encountered with the stability of the vector as unexpected recombinations were observed in plaque-purified stocks. An additional problem was encountered with wild type adenovirus contamination during production. Moreover, for large scale production of adenoviruses it is not useful to co-transfect E4-orf6 to obtain titers that are high enough for application. One option for growing such adenoviruses is to provide cells with the E4-orf6 gene stably integrated into the genome of the complementing/packaging cell line. Such cells have been described in the art (e.g. WO 96/22378). A disadvantage of that system is the fact that new stable cell lines have to be generated, and numerous selection rounds have to performed before stable and proper cells have been generated. This process is laborious and time-consuming. In general^ it can be stated that generation and propagation of adenoviruses from serotypes other than serotype 5 (subgroup C) , such as subgroup B viruses, has proven to be difficult on Ad5 complementing cells. As has been disclosed by the applicants in WO 00/70071, recombinant viruses based on subgroup B virus Ad35 can be made by co-transfection of an expression construct containing the Ad35 early region-1 sequences (Ad35-El) . Furthermore, Ad35-based viruses that are deleted only for ElA sequences and not for E1B were shown to replicate efficiently on PER.C6 cells, suggesting that the ElA proteins of Ad5 are able to complement the Ad35-E1A functions (applicant's application WO 02/40665). Moreover, the experiments show that lack of Ad35-E1B results in poos yields on Ad5 complementing cells. WO 00/70071 also discloses cell lines for the production of El-deleted non-group C adenoviral<br><br>
vectors by further modifying cell lines that are capable of complementing adenovirus serotype 5. WO 00/70071 further suggests that one should establish new cell lines harbouring Ad35-El sequences for the complementation of recombinant adenovirus serotype 35 vectors lacking the El region {see also WO 02/40665). However, as also discussed above, if one desires to apply a specific serotype for a specific need, one would have to establish a new cell line for every specific serotype, or one would have to modify the available cell lines that can complement adenovirus serotype 5 for complementation of the serotype of interest. It would clearly be advantageous to use the established cell lines that are available in the art, and not to modify these and use them for the production of all other, non-Ad5 serotypes, applying the established and efficient methods known in the art.<br>
It is concluded that there still exists a need for a production system to produce useful yields of adenovirus serotypes that are different from the serotypes of subgroup C. Furthermore, there is still a need for suitable packaging systems comprising convenient packaging cells and recombinant subgroup B adenoviruses that are stable and can be propagated on such packaging cells.<br>
DESCRIPTION OF THE FIGURES Fig. 1. Map of pWE.Ad35.pIX-rITRAE3<br>
Fig. 2. Gel analysis of PCR fragments generated on Ad35 El-deleted viruses with and without the E3 region. PI = plasmid control; M = marker: 1 kb plus ladder (Invitrogen); mQ = H20. Indicated genome lenghts are in kb.<br><br>
Fig. 3A. Sequence alignments of the proximal pIX upstream sequence regions of various adenoviruses generated with MEGalign software (DNAstar) using Clustal method. Source of sequences are indicated in the text. The Spl site and TATA-box in Ad5 and Ad2 (as in Babiss and Vales, 1991) are boxed. Fig. 3B. Schematic comparison of putative Spl- and TATA-boxes in proximal pIX regions from sequences given in 3A.<br>
Fig. 4. Map of pAdAptS35.Luc<br>
Fig. 5. Map of pAdApt535<br>
Fig. 6A. Gel analysis of PCR fragments generated on DNA isolated from Ad35.AdApt.Luc and Ad35.AdApt535.Luc viruses or generated on plasmid controls. M = Marker (1 kb plus ladder, Invitrogen). Each virus preparation or plasmid control is analysed with two specific PCR amplifications. Fig. 6B. Gel analysis of PCR fragments generated on Ad35.AdApt.LacZAE3 (35LacZ) and Ad35.AdApt535.LacZAE3 (535LacZ) viruses or generated on plasmid controls.<br>
Fig. 7. Map of pBr.Ad35.ASM.AdApt.LacZ<br>
Fig. 8. Schematic representation of the putative promoters in the ElB promoter and 55K coding region.<br>
Fig. 9. Schematic representation of the restriction sites in the 55K region that can be used to generate distinct fragments for identification of a putative promoter. Numbering of the sites is according to their position in wild type Ad35.<br><br>
Fig. 10. Sequence alignment of the region between the polyA signals of the ElA region and of the ElB/pIX region in three different subgroup B serotypes.<br>
Fig. 11. Schematic representation of plasmid pAMT.Orf6.Hygro {ECACC deposit no. P02041226).<br>
Fig.	12.	Schematic representation	of plasmid pAd35.AMT.Orf6.<br>
Fig.	13.	Schematic representation	of pUC.35-5E4.<br>
Fig.	14.	Cloning steps leading to	pUC.35-5E4.<br>
Fig.	15.	Schematic representation	of pBr.Ad35.PRn.<br>
Fig. 16. Schematic representation of pBr.Ad35.PR5E4 (ECACC deposit no. P02041229).<br>
Fig.	17.	Schematic	representation	of	pWE.Ad35.pIX-rITR5E4.<br>
Fig.	18.	Schematic	representation	of	pCRscriptAmp.NFI-NcoIR.<br>
Fig.	19.	Schematic	representation	of	pCRscriptAmp.NcoIF-NR2.<br>
Fig.	20.	Schematic	representation	of	pCR.NFl-NR2.<br>
Fig. 21. Alignment between E4-orf6 of Ad5 (upper sequence) with E4-orf6 from Ad5 cloned into the Ad35 backbone (middle sequence) and the Ad35 E4-orf6 sequence (lower sequence), showing that the entire fragment has been replaced.<br>
9<br><br>
Fig. 22. Alignment between E4-orf6/7 of Ad5 (upper sequence) with the part of Ad5 E4-orf6/7 cloned into the Ad35 backbone (middle sequence) and the Ad35 E4-orf6/7 sequence flower sequence), showing that the orf6/7 sequence is partly chimeric, with the fusion approximately at the lysine (K) residue at position 138,<br>
Fig. 23. Schematic representation of pBr,Ad35.PR.50rf6 (ECACC deposit no. P02041227).<br>
Fig.	24. Schematic	representation	of	pWE.Ad35.pIX-rlTR.50rf6.<br>
Fig.	25. Schematic	representation	of	pBr.Ad35.PRnAE3.<br>
Fig.	26. Schematic	representation	of	pBr,Ad35.AE3.PR5E4.<br>
Fig.	27. Schematic	representation	of	p3r.Ad35.AE3.PR50rf6.<br>
Fig. 28. Schematic representation of the system for producing recombinant adenoviral particles in cells such as PER.C6 through a double homologous recombination event.<br>
Fig. 29. Schematic representation of pWE.Ad35.pIX-EcoRV.<br>
Fig. 30.   Alignment of Ad35 and Adll pIX-cDNA sequences with wt Ad35 sequence.<br>
The sequences obtained from cloned cDNA fragments as described in example 18 were aligned using SeqMan software from DNASTAR. Ad35 cDNA sequences were derived from RNAs isolated from wtAd35- or Ad35ElB+Luc-infected cultures (sequence of one out of seven clones is shown), the Adll cDNA sequence from RNA isolated from an wtAdll-infected culture. The sequence<br>
(0<br><br>
numbering is arbitrary. For the Ad35 wt sequence, nucleotide 3339 to nucleotide 3628 of the wt Ad35 sequence is shown. The intron sequence (seen as a gap in the cDNA sequences) is flanked by splice donor (SD) and splice acceptor (SA) sites closely matching the known consensus sequences. The SeqMan software has placed the first two nucleotides from the SD (AG) in the cDNA sequences at the 3' end of the intron sequence instead of the 5' end.<br>
Fig. 31. Location of the pIX cap site in Ad35 viruses. Schematic representation of the genome organization around the ElB gene and pIX sequences in Ad35; depicts the transcription start site and intron boundaries in the pIX mRNA. Nucleotide sequences are according to wt Ad35 DNA (WO 00/70071). M= Muni, B=Bsu36I, SD= splice donor and SA= splice acceptor. The transcription start site (nucl. 3339, cap site), the stop codon of 55K {TAA} and the start codon of pIX (nucl. 3484, ATG) are in bold.  A dotted line indicates that sequences are not shown.<br>
Fig. 32.  Transgene PCR results from Ad35 viruses with a 166 bp 3' ElB sequence retained.<br>
A representative example of the results of the transgene PCR assays on Ad35.AdAptBsuLuc50rf6 (lane 1-9) and Ad35.AdAptBsuLuc viruses (lanes 10-14). M= lkb+ marker (Invitrogen), P= pAdAptBsuluc control plasmid.<br>
SUMMARY OF THE INVENTION It is shown in the present invention that a recombinant group B adenovirus that has a deletion in the El region up to the stopcodon of ElB 55K, can accommodate less exogenous sequences than a similar Ad5 recombinant adenovirus. It appears that<br><br>
this is due to a relative underexpression of the pIX gene in said group B virus in a given packaging cell, when the pIX coding region is preceded by sequences between the E1B 55K stopcodon and the pIX startcodon only. It is shown that such viruses can be rendered more stable and/or capable of accommodating more exogenous sequences when either the pIX promoter is at least partly restored by including sequences from the E1B 55K coding region into such a virus, or by using a heterologous promoter to regulate pIX, such that a normal, or even a relative ov&amp;rexpression, of pIX is achieved in a given packaging cell.<br>
The present invention provides a recombinant adenovirus having at least a deletion in the El region, characterized in that at least part of the sequences encoding the E1B 55K gene product that increase the expression of the pIX gene are present in said adenovirus, with the proviso that -said recombinant adenovirus does not express a functional E1B 55K gene product. Such adenoviruses are more stable and/or can harbour more exogenous DNA than the corresponding adenovirus that lacks all E1B coding sequences. Preferably, said adenovirus comprises about 700 base pairs or less of the sequences that are directly upstream of the pIX coding sequence. In preferred embodiments, said adenovirus is a group B adenovirus, more preferably an adenovirus derived from or based upon Ad35 or Adll. The invention further provides a method for increasing the stability and/or the packaging capacity of a recombinant adenovirus having at least a deletion in the El-region, said method comprising retaining or reintroducing at least part of the sequences that encode the E1B 55K gene product and increase the expression of the pIX gene in said adenovirus. Instead of or in addition to the presence of E1B 55K sequences increasing the expression of the pIX gene, it is also possible<br><br>
to change the sequences preceding the pIX coding sequence into a stronger promoter, to increase the expression of pIX, resulting in an increase of the stability of a recombinant adenovirus and/or an increase of the packaging capacity of the adenoviral particle produced by the method of the invention. Hence,' the present invention further provides methods and means for increasing the stability and/or the packaging capacity of a recombinant adenoviral vector lacking at least the El-region and comprising exogenous genetic information, comprising the step of expressing the elements necessary for production and assembly of said recombinant adenoviral vector into virus particles in a packaging cell in the presence of an elevated level of pIX gene product in said packaging cell, wherein said elevated level of pIX gene product is brought about by the overexpression of genetic information encoding the pIX protein, by the use of a modified pIX gene in said vector, said modification causing the pIX gene product to be overexpressed. In a preferred embodiment, said modified pIX gene comprises a heterologous promoter driving the expression of the genetic information encoding pIX, said heterologous promoter being a promoter causing said genetic information encoding pIX to be overexpressed in said packaging cell. In one embodiment, said heterologous promoter is at least in part derived from or based upon the pIX promoter of a adenovirus serotype that confers higher levels of pIX expression than the endogenous proximal plx upstream sequence of said recombinant adenoviral vector. In one embodiment, said heterologous promo'ter is at least in part derived from or based upon the pIX promoter of Ad5. In one aspect, the invention provides a recombinant adenoviral vector lacking at least the El-region and comprising a gene of interest, wherein the pIX gene is modified, and wherein said recombinant adenoviral vector is<br>
&amp;<br><br>
not derived from an adenovirus serotype 5. It is yet another aspect of the present invention to provide a recombinant nucleic acid sequence comprising a modified adenoviral pIX gene, wherein the genetic information encoding the pIX protein is not derived from an adenovirus serotype 5 or an adenovirus serotype 7 pIX encoding sequence. In a preferred embodiment, said modified pIX gene comprises a heterologous promoter driving the expression of the genetic information encoding pIX.<br>
The present invention further provides recombinant adenovirus vectors comprising structural and non-structural elements of an .adenovirus of a first serotype, wherein said vector further comprises a sequence encoding an E4-orf6 protein, wherein said sequence is selected from the group consisting of: a) an E4-orf6 encoding sequence derived from an adenovirus of a second serotype different from said first serotype; b) an E4~orf6 encoding sequence derived from an adenovirus of said first serotype by way of a deletion, mutation, addition and/or substitution in one or more codons; and c) an E4-orf6 encoding sequence comprising a fusion between a part of an E4-orf6 encoding sequence derived from a second serotype different from said first serotype and a part of an E4-orf6 encoding sequence derived from a third serotype, wherein said third serotype may be identical to-, or different from said first serotype.<br>
The invention further provides methods for the production of such recombinant adenovirus vectors comprising structural and non-structural elements of an adenovirus of a first serotype, said method comprising the steps of: a) providing a complementing cell harbouring an E1B-55K encoding sequence, derived from an adenovirus of a second serotype in expressible form, with the necessary elements of an adenovirus such as to<br><br>
allow assembly of said recombinant adenovirus vector by said complementing cell, wherein said elements comprise at least some structural and non-structural elements from an adenovirus of said first serotype different from said second serotype and a sequence encoding a functional E4-orf6 protein, or a functional part, derivative and/or analogue thereof, which is compatible with said expressible E1B-55K protein in said complementing cell; b) culturing said complementing cell in a medium under conditions allowing for production and assembly of the adenovirus vector to take place; and c) harvesting the recombinant adenovirus vector so produced from the medium and/or the complementing cell, wherein the sequence encoding the compatible E4-orf6 protein is present in the recombinant adenovirus vector so produced.<br>
The invention further provides a recombinant adenovirus comprising a recombinant nucleic acid molecule derived from an adenovirus, said recombinant nucleic acid molecule having at least a deletion in the El region, characterized in that at least part of the sequence encoding the E1B-55K gene product is present in said recombinant nucleic acid molecule and/or that the pIX coding sequence is under the control of a heterologous promoter, said recombinant adenovirus further comprising structural and non-structural elements of an adenovirus of a first serotype, wherein said recombinant . adenovirus further comprises a sequence encoding a functional E4-orf6 protein, or a functional part, derivative and/or analogue thereof, wherein said sequence is chosen from the group of: a) an E4-orf6 coding sequence derived from an adenovirus of a second serotype different from said first serotype; b) an E4-orf6 encoding sequence derived from an adenovirus of said first serotype comprising a deletion, mutation, addition and/or substitution in one   or more codons;<br><br>
and c) an E4-orf6 encoding sequence comprising a fusion between a part of an E4-orf6 encoding sequence derived from a second serotype and a part of an E4-orf6 encoding sequence derived from a third serotype, wherein said third serotype may be identical to- or different from said first serotype. The invention also provides a packaging system comprising a recombinant adenovirus according to the invention and a packaging cell, wherein said packaging cell and said recombinant adenovirus together comprise all necessary elements to allow production and assembly of said recombinant adenovirus in said packaging cell, and wherein said packaging cell expresses nucleic acid encoding at least an adenoviral E1B-55K protein or a functional, part, derivative and/or analogue thereof, which is compatible with the E4-orf6 protein, or a functional part, derivative and/or analogue thereof of said recombinant adenovirus.<br>
The invention also provides a method for producing a stable recombinant adenovirus comprising structural and non¬structural elements of an adenovirus of a first serotype, wherein said recombinant adenovirus comprises a recombinant nucleic acid molecule derived from an adenovirus which nucleic acid molecule has a deletion in the El region and comprises nucleic acid derived from at least part of the sequence encoding the E1B-55K gene product increasing expression of the pIX protein not leading to the expression of a functional E1B-55K protein from said nucleic acid molecule and/or has a pIX coding sequence under control of a heterologous promoter, said method comprising the steps of: a) providing a complementing cell expressing an E1B-55K encoding sequence, of a functional part, derivative and/or analogue thereof, derived from an adenovirus of a second serotype in expressible form, with the necessary elements of an adenovirus as to allow assembly of<br><br>
said recombinant adenovirus vector by said complementing cell, wherein said elements comprise at least some structural and non-structural elements from an adenovirus of said first serotype different from said second serotype and a sequence encoding a functional E4-orf6 protein, or a functional part, derivative and/or analogue thereof, which is compatible with said expressible E1B-55K protein in said complementing cell; and b) culturing said complementing cell in a medium under conditions allowing for production and assembly of the recombinant adenovirus to take place; and c) harvesting the recombinant adenovirus so produced from the medium and/or the complementing cell.<br>
The invention further provides a method of producing stable adenoviral vectors containing exogenous genetic information without the generation of replication competent adenovirus, wherein said vectors are of a serotype other than adenovirus 5, comprising the steps of growing cells under conditions conducive to adenoviral vector production, said cells, or an ancestor thereof, having been provided with and expressing nucleic acid coding for all elements essential for adenoviral vector production and packaging, the elements essential for generating replication competent adenoviral vectors being present on at least two separate nucleic acid molecules not giving rise to intermolecular recombination, the essential elements comprising at least one serotype determining structural and one non-structural element from an adenovirus of a first serotype other than Ad5, and comprising an ElB 55K gene product from a second serotype different from the first serotype and an E4 orf6 gene product compatible with said ElB 55K gene product. In certain embodiments, the nucleic acid coding for all essential elements essential for adenoviral vector production and packaging comprise a<br>
l^<br><br>
recombinant pIX gene. In one embodiment, a recombinant pIX gene is a pIX coding sequence under control of a heterologous promoter.<br>
In another aspect the present invention provides methods for treating or preventing diseases or disorders in human or animal subjects, comprising a step of administering recombinant adenoviral vectors according to the invention to a human or animal subject. In other aspects, the invention provides vaccines and pharmaceutical compositions comprising the adenoviral vectors according to the invention. In another aspect, the invention provides for the use of recombinant adenoviral vectors according to the invention for the preparation of a medicament for the prevention or treatment of diseases or disorders in human or animal subjects. The invention also relates to a kit of parts comprising cell lines and adenoviral vectors provided by the invention for executing the methods provided by the invention.<br>
DETAILED DESCRIPTION OF THE INVENTION It is one aspect of the invention to provide a method for increasing the stability and/or the packaging capacity of a recombinant adenovirus having at least a deletion in the El-region/ comprising the step of expressing the elements necessary for production and assembly of said recombinant adenovirus into virus particles in a packaging cell in the presence of an elevated level of pIX gene product in said packaging cell/ relative to the level of pIX gene product obtained when the pIX coding sequence is behind its endogenous proximal upstream sequence without E1B 55K sequences. In certain embodiments of the method of the invention, said elevated level of pIX gene product is brought about by retaining or reintroducing part of the E1B 55K sequences in<br><br>
said adenovirus. In other embodiments, said elevated level of pIX gene product is brought about by the expression of the pIX coding sequences under control of a heterologous promoter. The invention further provides a recombinant adenovirus comprising a functional pIX coding sequence under control of an expression sequence, said expression sequence comprising part of an ElB 55K sequence capable of increasing expression of the pIX coding sequence in a given packaging cell, relative to the expression of said pIX coding sequence behind its endogenous proximal pIX upstream sequence without the part of said ElB 55K sequence, with the proviso that said part of an ElB 55K sequence does not code for a functional ElB 55K gene product. It is shown in the present invention that pIX promoter sequences can be present in said ElB 55K sequences, and including these sequences in said expression sequence can therefore increase pIX expression. The presence of said ElB 55K sequences increases the stability and/or packaging capacity of said recombinant adenovirus, compared to the situation where said pIX coding sequence is behind its endogenous proximal pIX upstream sequence without the part of said ElB 55K sequence.<br>
An adenovirus of serotype 35 with a deletion in the El region but with an intact ElB 55K coding region<br>
(pBr.Ad35.1eftITR.AElAA21K) has been disclosed in WO 02/40665. However, a functional 55K gene product, such as present in the disclosed vector, inhibits apoptosis, and hence it is desired to obtain recombinant adenovirus that lacks functional ElB 55K expression, e.g. by mutating the ElB 55K gene or preferably by including only part of the ElB 55K sequences, more preferably including only sequences downstream of the ElB 55K startcodon. It will be clear to the skilled person that it is beneficial to minimalize the amount of ElB 55K  sequences in said vector<br>
n<br><br>
in order to generate viruses with a maximal El deletion to accommodate more foreign nucleic acid, while at the same time retaining sufficient E1B 55K sequences to have the benefit according to the present invention of an increased stability and/or increased packaging capacity within the recombinant adenovirus. With the teachings of the present invention, the person skilled in the art will be able to find the minimally required sequences within E1B 55K that lead to the stabilization and/or increase in packaging capacity of the recombinant adenovirus, e.g. by using standard molecular cloning techniques to obtain serial deletions within the E1B 55K region starting from the disclosed vector (pBr.Ad35.1eftITR.AElAA21K) and determining the stability or packaging capacity. It is therefore an object of the present invention to provide a recombinant adenoviral vector according to the invention wherein said sequences encoding the E1B 55K. gene product comprise about 0.7 kb or less of the adenovirus sequences that are directly upstream of the plX open reading frame. In other embodiments, said sequences comprise not more than about 680, 600, 550, 500, 450, 400, 350, 300, 250, 200, 150, or 100 nt of the adenovirus sequences that are directly upstream of the plX open reading frame. In one embodiment, such a recombinant adenoviral vector retains 166 bp of the 3' end of the 55K coding sequence. As will be clear to the person skilled in the art, the invention is not limited to the presence of a sequence that is found in a contiguous stretch directly upstream of the plX coding sequences in the natural adenovirus. Instead, it will also be possible according to the invention to have sequences that are more upstream, i.e. from the E1B 55K region {e.g. a restriction or PCR fragment, see e.g. example 10 and fig. 9), fused to the more proximal plX regulatory sequences, thereby creating an artificial<br>
90<br><br>
combination of regulatory sequences, as long as this results in increased pIX expression in a given packaging cell, when compared to the absence  of any E1B 55K sequences. In a preferred embodiment, said adenovirus is a subgroup B adenovirus, more preferably an Ad35 or Adll adenovirus. Adenoviruses of serotypes 35 or 11 have been shown to be particular useful for administration to humans, as there are much less individuals that have neutralizing antibodies to these serotypes than to the serotype 5 hitherto most used {WO 00/70071). It is another aspect of the invention to provide the nucleic acid that can act as the genome of the adenovirus according to the present invention.<br>
As an alternative or in addition to the presence of E1B 55K saquences that increase the expression of the pIX gene, it is also possible to overexpress pIX itself by mutating the pIX gene, preferably its promoter, to increase the stability and/or packaging capacity of a recombinant adenovirus. A modified pIX gene is a pIX gene having a different promoter and/or transcription terminator, and/or mutated coding sequences, e.g. obtained by codon optimisation, introduction of introns that stabilize RNA, and the like. A pIX gene according to the invention comprises genetic information encoding pIX, and includes nucleic acid such as the pIX gene such as found in natural adenoviruses, cDNA and information encoding mutant pIX in the form of allelic variants or nucleic acid encoding mutant pIX that has at least part of the function of pIX which may differ from normal pIX in quantitative or qualitative aspects, derived from or based on pIX by mutation, deletion, addition, or translocation of amino acids, or combinations thereof.<br>
If a recombinant adenovirus has a deletion of at least the E1B 55K region up to and including the stopcodon of the ElB<br>
3 <br>
55K gene product, the pIX open reading frame will be preceded by sequences between the E1B 55K stopcodon and the startcodon of pIX. These sequences are herein referred to as the "endogenous proximal pIX upstream sequences" (e.g. the Ad35 pIX upstream sequence in Ad35 based recombinant adenoviral vectors, the Adll pIX upstream sequence in Adll based recombinant adenoviral vectors, etc. are referred to as endogenous in this respect; the definition includes allelic variants that may be found in nature and which are not created in the laboratory; see Fig. 3A for some proximal pIX upstream sequences).<br>
A "heterologous promoter" as used herein is defined as any sequence different from the sequences naturally found upstream of the pIX gene, including the sequences of the E1B 55K region and the endogenous proximal pIX upstream sequences, and being capable of acting as a promoter and thereby regulating transcription of pIX coding sequences. In one aspect, a heterologous promoter may be a sequence at least in part derived from or based upon a proximal pIX upstream sequence (i.e. the sequences between the E1B 55K stopcodon and the startcodon of pIX) from an adenovirus from another serotype than the serotype from which the recombinant adenoviral vector is derived (e.g. an Ad5 pIX upstream sequence in an Ad35 recombinant adenoviral vector). This is referred to herein as a "non-endogenous proximal pIX promoter". In one embodiment of the invention, Ad35 derived pIX expression is driven by an Ad5 non-endogenous proximal pIX • promoter. Any non-endogenous proximal pIX promoter derived from or based upon the proximal pIX upstream sequence from a serotype that confers higher levels of pix expression than the endogenous pIX proximal sequences of the adenoviral vector may be used. Identification of such non-endogenous proximal pIX<br><br>
promoters may be based upon sequence information, such as will be evident to the person skilled in the art from Example 4. In particular embodiments, the adenoviral vector according to the invention is derived from or based upon an adenovirus subgroup E serotype/ or preferably of subgroup B serotype. In specific embodiments, said adenoviral vector is derived from or based upon an adenovirus serotype 35 (Ad35), Adll, Ad7, or Ad4. In alternative embodiments, said non-endogenous proximal pIX promoter is at least in part derived from or based upon a proximal pIX upstream sequence of an adenovirus classified in subgroup C, A, D, or F. In particular embodiments said non-endogenous proximal pIX promoter is at least in part derived from or based upon a proximal pIX upstream sequence of an adenovirus serotype 12 (Adl2), Ad9, or Ad40, or more preferably of Ad5 or Ad2. Alternatively, sequences acting as non-endogenous proximal pIX promoter can be found empirically, by general molecular biology methods known to persons skilled in the art, such as by transcription assays wherein promoters can be routinely tested for strength. It will be clear to the skilled person that elements from a promoter may be swapped without exchanging the whole promoter, e.g. adding, deleting, or mutating known transcription factor binding sequences to a promoter may influence its strength. Mutating at least part of promoter sequences can be done by changing the sequence by mutations, such as by additions, deletions, or exchanging of one or more nucleotides, including stretches of nucleotides with a known function. Substituting promoter sequences is done by replacing part or all of these sequences by a different promoter. Such replacing can be done according to standard molecular biology techniques all well known to the person skilled in the art. Any promoter can be constructed in<br>
3.1<br><br>
operable association with the pIX gene of choice, and tested for its effect.<br>
In another aspect, a heterologous promoter is unrelated to adenoviral non-endogenous proximal pIX promoters. Therefore, heterologous promoters may also be viral promoters, including but not limited to promoters based upon or derived from Cytomegalovirus (CMV, e.g. the human CMV immediate early gene promoter, further herein referred to as the CMV promoter), Rous Sarcoma Virus (RSV, e.g. the RSV long terminal repeat promoter, further referred to herein as the RSV promoter), TK, HBV, SV40 and the like. In one embodiment, an adenoviral E1B promoter is used as a heterologous promoter. Cellular promoters can also be used as heterologous promoters, and these include but are not limited promoters from PGK, metallothionein, EFl-tx,p-actin, and the like. Synthetic or hybrid promoters, comprising elements from more than one promoter, can also be used for the invention, and are all included within the scope of the term heterologous promoter. Promoters used, may be constitutive or inducible. In the context of an inducible promoter a promoter is considered suitable for the invention if it gives overexpression in its induced state. Any promoter sequence resulting in overexpression of pIX according to the invention can be used as a heterologous promoter according to the invention. Besides promoter strength, one other aspect determining the usefulness of a particular promoter when present in the recombinant adenoviral vector, itself is its size, as longer promoters will take up space available for the transgene. A person skilled in the art will be able to use the present invention to allow for optimising the vector experimentally with respect to promoter length and strength with regard to insert size, to find the most stable recombinant vector.<br><br>
It will be clear that a heterologous promoter may still contain or include part or all of the endogenous proximal pIX upstream sequences/ or alternatively wholly replace these sequences, as long as the heterologous promoter according to the invention can cause the overexpression of genetic information encoding pIX in a packaging cell of choice. It will also be clear to the person skilled in the art that when an endogenous non-pIX adenoviral promoter (e.g. the ElA, E2A, etc. promoter) or a heterelogous promoter that for instance regulates the transgene is used for driving expression of pIX by use of an internal ribosome entry site (IRES) between the non-pIX adenovirus gene or the transgene and the pIX coding sequence (in either order), this is to be regarded as being within the scope of the term "heterologous promoter" according to the invention.<br>
An elevated or increased level of pIX gene product in the invention is the result of overexpression of the pIX gene in a packaging cell of choice. Overexpression of the pIX gene as used herein is defined as an expression level of pIX, either on RNA or protein level or both, that is higher than the pIX expression level obtained when the coding region of pIX is behind the "endogenous proximal pIX upstream sequences" as defined herein, in a given packaging cell. "Overexpression" is meaningful in the context of the present invention for a particular heterologous pronoter-pIX combination of an adenovirus in combination with a particular packaging cell of choice. Methods to determine expression levels are generally well known to persons skilled in the art, and include but are not limited to RT-PCR, Northern blotting, Western blotting, and the like. For the present invention, overexpression would be measured by determining the expression levels in a recombinant adenovirus with a given insert (e.g. luciferase)<br><br>
in a given packaging cell of choice, wherein the genetic information encoding pIX is behind the endogenous proximal pIX upstream sequences without E1B 55K sequences, and comparing these expression levels to those in a recombinant adenovirus that is the same except that the pIX coding region is regulated by a heterologous promoter or by sequences from the endogenous E1B 55K gene (its 'natural' promoter that has been at least partly reconstituted). Overexpression of pIX is indicated by a ratio higher than 1 for the expression level obtained by the heterologous or 'natural' promoter over that obtained by the endogenous proximal pIX upstream sequences without E1B 55K sequences. The choice of a packaging cell is determined by factors such as the serotype of the elements of the recombinant adenovirus that interact with the complementing adenoviral functions in the packaging cell, product purity (such as absence of replication competent adenovirus from the generated batch), ease of use, growth characteristics, and the like. Examples of packaging cells are known to the person skilled in the art, and include 293 cells, 911 cells, and PER.C6™ cells as used herein, as well as derivatives thereof adapted for complementation of adenoviral vectors of specific serotypes, such as PER55K.<br>
It will be clear that in all embodiments of the invention, the pIX coding sequences and regulatory sequences driving the expression of pIX can be positioned on their natural location within the adenovirus genome, as well as in different parts of the adenovirus genome, e.g. in the region that originally contained the E3 sequences.<br>
Recombinant adenoviruses with increased stability are capable of incorporating larger genomes into virus particles (virions). Hence, increasing the stability of recombinant adenoviruses as used herein will allow the recombinant ,<br>
5b<br><br>
adenoviruses according to the invention to include more foreign genetic information, comprising a gene of interest. Furthermore, recombinant adenoviruses with increased stability according to the invention may be capable of being propagated for more passages without signs of instability. Stability can be measured by several methods known to people skilled in the art, including but not limited to PCR on recombinant virus to demonstrate the presence of desired recombinant adenoviral vectors. Instability will lead to by-products, which can also be visualized by PCR methods, and the like. Restriction analysis of viral DNA, determination of the relative infectivity of virus particles, determination of the thermostability of adenoviral particles also can be used to determine the stability of recombinant adenoviral vectors<br>
(Caravokyri and Leppard, 1995). Methods to determine stability/instability of recombinant adenoviral vectors are also given in Example 3 of this application. A recombinant adenovirus, also called recombinant adenoviral vector, or adenoviral vector, as used herein is derived from or based upon an adenovirus, lacks at least part of the El region<br>
(comprising the ElA and ElB genes) of an adenovirus and can comprise foreign genetic information of which delivery and/cr expression by said vector is desired. Exogenous (or foreign) genetic information as used herein, is any genetic information that is not naturally present in an adenovirus, and is also referred to as transgene. This includes but is not limited to genes of interest, expression cassettes, and the like. Such exogenous genetic information can fill the space in the genome that has become available by the deletion of adenoviral El sequences. Recombinant adenoviral vectors are useful for various purposes, such as in gene therapy applications, vaccine preparation, and the like. In addition to the El<br><br>
region deletion, also E3 sequences can be deleted from such adenoviral vectors, to increase the capacity for foreign genetic information in certain preferred embodiments. Other deletions and various combinations of part or complete deletions of E2, E3, and E4 regions combined with the El deletion can be used, if necessary in combination with a packaging cell comprising the genetic information lacking in the adenoviral vector, when necessary for replication of the adenoviral vector. All recombinant adenoviruses having a deletion in the El-region combined with optionally any other deletions in the adenovirus genome, are meant to be included within the scope of the present invention. The adenoviruses of the present invention can be used in different settings such as gene therapy or prophylactic and/or therapeutic vaccination, including tumour vaccination and anti-viral vaccination. For this, the adenoviral vector functions as a gene delivery vehicle, wherein a non-native gene is incorporated into the adenoviral genome. The adenoviral particle can subsequently be targeted specifically to target cells of interest; the adenovirus binds to that specific cell either through capsid-receptor binding or through other means, and delivers the transgene. Targeting of adenoviruses can be performed in many different ways. Persons skilled in the art of adenoviral vector targeting will be aware of all the different possibilities that are applied to deliver the adenoviral vectors to the cells of interest. Such possibilities include but are not limited to capsid alterations (fiber, hexon and/or penton modifications, such as deletions, swaps between fibers of different serotypes, and additions of peptides and/or other binding moieties), wherein chimeric fibers are produced that recognize a receptor present on the cell of interest, or wherein the binding of the penton-<br><br>
base is utilized. Other possibilities are linking targeting moieties to the capsid proteins, wherein for instance binding peptides, known and strong binding proteins, or antibodies or parts thereof are linked to the capsid proteins to achieve specific targeting. Such vectors can all be produced using the methods and means provided by the present invention. Therefore, the present invention also discloses recombinant adenovirus vectors according to the invention, further comprising a sequence encoding a non-adenoviral protein. Such sequences can be present on different locations within the adenoviral backbone, but preferably they are located in the El region, which is lacking in the recombinant adenoviral vectors of the invention. The El region is complemented by the complementation elements present in the complementing cells. The direction of the promoter, transgene and other regulatory sequences can be directed towards the left-, as well as to the right inverted terminal repeat.<br>
The present invention can also be used for the production of viral vectors based on adenovirus and/or on other viruses such as the Adeno-Associated Virus (AAV), wherein the combination, such as an Ad-AAV chimeric virus, can integrate into the host cell genome. Several methods are known in the art for generating integrating adenoviruses. Generally, the invention is also useful for the production of adenovirus forms that (specifically, or non-specifically) can integrate.<br>
As mentioned, several non-adenoviral transgenes can be cloned into the recombinant adenoviral vectors of the present invention. These do not only include regulatory nucleic acid sequences such as enhancers, promoters (e.g. strong non-adenoviral promoters such as the cytomegalovirus promoter, the SV40 promoter and the RSV promoter) and polyadenylation signals, but also heterologous genes for therapeutic purposes.<br><br>
Therefore, in one aspect of the invention, recombinant adenovirus vectors according to the invention are provided, wherein said non-adenoviral protein is selected from the group consisting of: a cell-death inducing polypeptide, a tumor specific antigen, a viral protein, a hormone and a cytokine. Non-limiting examples of non-adenoviral factors, proteins, polypeptides and peptides are transcription factors, intracellular signalling proteins, phosphatases, kinases, apoptosis inhibiting factors, receptor antagonists, soluble forms of membrane-bound receptors, RNA inhibitors, anti-sense RNA's, decoy factors, ribozymes, and more specifically, thymidine kinase, erythropoietin, novel-erythropoiesis stimulating protein {NESP), IL3, ceNOS, gamma-interferon and gplOO. Non-adenoviral viral proteins can be cloned into the recombinant adenoviral vectors provided by the methods and means of the present invention for vaccination purposes. Such viral proteins include, but are not limited to, gag, pol, env, nef, etc. for HIV vaccines, E6 and E7 proteins for Human Papilloma Virus vaccines, circumsporozoite proteins from Plasmodium protozoa for malaria vaccines, rotavirus components for rotavirus vaccines, ebola proteins for ebola vaccines, the F and G gene products from Respiratory syncytial virus for Respiratory Syncytial virus vaccines, HA and NA for influenza vaccines,etc.<br>
Adenoviruses according to the invention are preferably human adenoviruses, i.e. derived from or based upon adenovirus that is capable of infecting human cells, but the invention is equally useful for non-human adenoviruses. A person skilled in the art will be aware of the fact that in addition to all human adenoviruses numerous non-human adenoviruses have been identified in the art. Obviously, also non-human adenoviruses can be applied to reach the same results as disclosed by the<br>
30<br><br>
present invention. Non-limiting examples of non-human adenoviruses that can be produced using the methods and means of the present invention are canine-, bovine-, monkey- and avian adenoviruses. Serotypes as used herein therefore goes beyond species-restricted serotypes.<br>
"Derived from" as used herein, means that nucleic acid sequences, genes, or proteins that are normally found in an adenovirus, are used for the generation of recombinant adenoviruses according to the invention. Methods to generate such recombinant adenoviruses are well known to persons skilled in the art, and include but are not limited to general molecular biology methods such as cloning of genetic information into desired constellations by  use of restriction enzymes, and the like. Recombinant adenoviruses can also be based upon adenoviral sequences. "Based upon" as used herein, is meant to include the synthetic construction of genetic information based upon knowledge of such genetic information. Such methods include but are not limited to, the use of adenoviral genetic material as a template for PCR to construct a new adenoviral construct that is based upon the sequence of the template adenovirus, the construction of completely synthetic genetic information with a desired sequence e.g. by linking synthetic oligonucleotides to a desired construct, and the like. It is to be understood that ^derived from' does not necessarily mean a direct cloning of the wild type DNA. A person skilled in the art will also be aware of the possibilities of molecular biology to obtain mutant forms of a certain piece of nucleic acid. These mutations may render a different functionality, but they may also be silent in a way that certain mutations do not alter the functionality of that particular piece of DNA and its encoded protein. Therefore, the terms 'functional part, derivative and/or analogue<br>
Si<br><br>
thereof are to be understood as equivalents of the nucleic acid they are related to. A person skilled in the art will appreciate the fact that certain deletions, swaps, (point)mutations, additions, etc. may still result in a nucleic acid that has a similar function as the original nucleic acid. It is therefore to be understood that such alterations that do not significantly alter the functionality of the proteins such as the pIX protein, E4-orf6 and E1B-55K gene product are within the scope of the present invention. It will be clear to those skilled in the art, that the method for obtaining the genetic information encoding the recombinant adenovirus can be varied without departing from the scope of the present invention.<br>
Human adenoviruses have been classified into subgroups A-F, which encompass 51 serotypes (see e.g. EP 0978566). For some applications, it can be beneficial to use adenoviral vectors derived from or based upon adenoviruses from specific subgroups or from certain serotypes that have a tissue tropism for a desired cell type, e.g. dendritic cells (WO 02/24730). The general absence of neutralizing antibodies in the population against adenovirus from certain subgroups or from a specific serotype is also an important parameter to determine the serotype of choice (WO 00/70071). Because of the similarity between the subgroup B viruses (see e.g. Examples 4 and 11) it is expected that the invention is particularly suitable for the adenoviruses of subgroup B. Hence, preferred embodiments of the present invention relate to adenoviral vectors derived from or based upon an adenovirus classified in subgroup B. Subgroup B of human adenoviruses comprises Ad3, Ad7, Adll, Adl4, Adl6, Ad21, Ad34, Ad35, and Ad50. Preferred embodiments of the present application relate to recombinant adenoviral vectors derived from or based upon Ad35 or Adll<br><br>
serotypes. Besides choosing for a serotype for specific applications, so-called chimeric adenoviruses can be used. These comprise parts or all of genetic sequences coding for coat proteins, such as fiber, penton, or hexon, from one or more adenoviral serotypes linked to the remaining genetic information (the *main' adenoviral vector part) from other serotypes, which can be used to decrease immunogenicity or change the tissue tropism of the 'main' adenoviral vector {EP 0978566). The 'main' part as used herein means that it contributes most of the genetic information to said chimeric virus, and a chimeric adenovirus will therefore be included in the serotype group of the 'main' part of such a virus. It will be clear to those skilled in the art that the present invention can also be used for such chimeric adenoviruses when these could face similar instability problems. It can for instance be expected that a chimeric adenovirus comprising Ad35 sequences as the main part and comprising a fiber that is derived from or based on e.g. an Adll adenovirus may have similar instability upon- propagation as is reported for the Ad35 recombinant adenoviral vectors described here. Hence, when recombinant adenoviruses are mentioned in this application, chimeric adenoviral vectors are meant to be included in the present invention.<br>
The elements necessary for production and assembly of recombinant adenoviral vectors are well known to the person skilled in the art (described supra; US patent 5,994,128; Russell, 2000) . Production of El deleted recombinant adenoviruses in the form of virions is done in packaging cells, also called complementing cells. Such cells provide in trans  the genetic information of the adenovirus lacking in said recombinant adenoviruses, necessary to produce recombinant virus (recombinant virions). Well-known packaging<br><br>
cells are 293 cells, 911 cells and PER.C61M cells (supra). For most purposes it is preferable to use a packaging cell and a recombinant adenoviral vector that lack overlapping sequences that would otherwise lead to homologous recombination resulting in replication competent adenovirus (US patent 5,994,128). PER.C6™, as deposited under no. 96022940 at the European Collection of Animal Cell Cultures at the Center for Applied Microbiology and Research, is therefore a very suitable packaging cell for propagating recombinant adenoviruses. Other methods to decrease the generation of replication competent adenovirus have also been envisaged and concern for instance manipulation of adenoviral sequences to reduce the homology between sequences present in the packaging cell and the vector (e.g. Hehir et al., 1996; Robert et al., 2001). Packaging cells can, besides the obligatory El region, comprise other adenoviral sequences to complement other adenoviral functions when these are functionally lacking in the recombinant adenovirus used, such as e.g. parts or all of E2, E4, and the like. The complementing information in packaging cells can be present either integrated in the genome, or as extrachromosomal copies, e.g. on plasmids, vectors, cosmids, and the like. Other methods make use of so-called helper viruses, which comprise genetic information lacking in the recombinant adenovirus. Recombinant adenoviral vectors are also used as so-called helper viruses used for the production of recombinant adenoviruses that contain a genome deleted for most or all adenoviral genes (gutless vectors or helper dependent adenoviruses). In the final production of such gutless adenoviruses it is necessary to avoid packaging of the helper adenovirus. A person skilled in the art is familiar with the methods to achieve this, for example using a site-specific recombinase on an engineered site in the<br>
3
 <br>
packaging signal to delete this packaging signal. Often it is necessary to separate remaining helper virus from the desired gutless virus using CsCl-gradient separation. This is easier to achieve when the genome lengths of the helper and gutless virus differ maximally. Therefore, a large helper virus is preferred above a smaller one.   Us   will be clear to the person skilled in the art, the present invention can equally be applied to increase the stability of the recombinant adenovirus by use of a recombinant helper virus having the increased pIX expression, which can be accomplished by the methods described in the present invention. Any cell containing genetic information that can be used to complement the recombinant adenovirus, to generate recombinant virus particles, is meant to be included in the scope of the meaning of packaging cell. It will be clear to the person skilled in the art that the advantage gained by the present invention is not dependent on the packaging cell used.<br>
The genetic information encoding pIX can either be present on the recombinant adenoviral vector but can also be present independent from said recombinant adenoviral vector, and such extraviral genetic information can be present either integrated in the genome, or as extrachromosomal copies, e.g. on plasmids, vectors, cosmids, and the like. Introducing genetic information into a packaging cell can be done according to a variety of methods, such as transfection by lipofectamin, calcium phospate precipitation, viral infection, and the like. Such methods are generally well known to the person skilled in the art, and the method used for introduction of genetic information is not critical for the scope of the invention. Functional pIX in expressible format as used herein means genetic information encoding pIX in<br>
3*<br><br>
operable linkage to a promoter or other regulatory sequence capable of driving expression of said genetic information encoding pIX in the packaging cell. Introduction of genetic information into the packaging cell can be done either prior to, concomitantly with, or after the introduction of the recombinant adenoviral vector. It was found that constitutive episomal expression of pIX in a 293-based packaging cell line complements the deficiency of pIX mutant adenovirus type 5 (Caravokyri and Leppard, 1995) . However, for such applications special episomal plasmids containing an EBNA1 expression cassette are required, and propagation of adenoviral vectors in such cell lines suffers from the disadvantage that parts of the episome very likely will become part of the recombinant adenoviral vector. Hence, in preferred embodiments of the present invention, genetic information encoding functional pIX is present on the adenoviral vector.<br>
In attempts to decrease the amount of recombination leading to replication competent adenovirus, some authors have used a pIX gene derived from Ad7 - a group B virus- which was driven by a mutated pIX promoter of Ad5, to diminish the overlap between the nucleic acids of the (Ad5-derived sequence containing) packaging cell and the recombinant adenoviral vector (Robert et al., 2001). However, those experiments were not done to increase the stability of the viral vector, and this stability was not measured in those experiments. In the current application, the transcription regulating sequences of pIX in the adenoviral vector are changed with the purpose of increasing the stability of the virus and/or increasing the capacity for foreign genetic material in virions. The invention demonstrates that a recombinant adenoviral vector derived from Ad35 comprising a pIX gene under the control of an Ad5-derived proximal pIX promoter derived from Ad5 is more<br><br>
stable/can harbour more foreign genetic information than the corresponding virus with the endogenous (i.e. Ad35 derived) proximal pIX upstream sequences. Therefore, the present invention also provides a recombinant adenovirus comprising a functional pIX coding sequence and having at least a deletion in the El-region, wherein the pIX coding sequence is under control of a heterologous promoter, and wherein said recombinant adenovirus is derived from or based upon an adenovirus other than an adenovirus serotype 5. In certain embodiments, said heterologous promoter is a non-endogenous proximal pIX promoter. In preferred embodiments the genetic information encoding pIX is derived from or based on Ad35 or Adll. In such embodiments, a preferred non-endogenous pIX promoter is an Ad5 promoter.<br>
In another aspect, the invention provides a recombinant adenoviral vector obtainable by a method according to the invention. Such recombinant adenoviral vectors are useful, e.g. in the preparation of vaccines (WO 00/70071; WO 01/38362; WO 02/24730), as gene delivery vehicles, and the like. Choosing a desired main serotype for such recombinant adenoviral vectors can be used for obtaining vectors with an altered tissue tropism as compared to the much-used Ad5 adenoviral vectors, and/or can be used because they are less immunogenic than such Ad5 derived vectors (WO 00/70071). For the generation of recombinant adenoviral vectors, it is convenient to clone the transgene into a plasmid (adapter plasmid), containing the left part of an adenovirus lacking El sequences and having restriction enzyme sites for cloning. The recombinant adenoviral vector is then generated by homologous recombination with a cosmid comprising the right part of the adenovirus having at the 5'end overlapping sequences with the 3'end of the adapter plasmid (see examples in present<br>
3*<br><br>
application; method described in HO 99/38362). Thus it is another aspect of the present invention to provide a recombinant nucleic acid sequence comprising an adenoviral left ITR, a packaging signal, other adenoviral sequences with a deletion in the El region, at least part of the ElB 55K open reading frame and pIX coding sequences. In yet another aspect the invention provides a modified adenoviral pIX gene, wherein the genetic information encoding the pIX protein is not derived from an adenovirus serotype 5 or an adenovirus serotype 7 pIX encoding sequence. In preferred embodiments, said modified pIX gene comprises a heterologous promoter. The present invention also relates to a pharmaceutical composition comprising a recombinant adenoviral vector according to the invention or obtainable by a method provided by the invention. The pharmaceutical composition further comprises an acceptable pharmaceutical carrier, generally applied by persons skilled in the art of preparation of pharmaceuticals. Furthermore, the present invention relates to a method of treating a human body comprising administering to a human body a recombinant adenoviral vector according to the invention, or a pharmaceutical composition provided by the invention.<br>
The invention further provides a recombinant adenovirus packaging cell comprising an recombinant adenovirus according to the invention. In one embodiment said recombinant adenovirus packaging cell comprises a nucleic acid capable of complementing an ElB 55K deficiency of said recombinant adenovirus and wherein said recombinant adenovirus comprises nucleic acid molecule comprising a part of the sequence encoding a ElB 55K gene product increasing the expression of the pIX gene, with the proviso that the latter recombinant<br>
3*<br><br>
nucleic acid molecule does not encode a functional ElB 55K gene product, and wherein said cell and said recombinant adenovirus do not comprise sequence overlap leading to the formation of a recombinant adenovirus comprising a nucleic acid encoding a functional ElB 55K protein. This embodiment is particularly useful for preventing the formation of recombinant adenoviruses comprising additional adenovirus function.<br>
The present invention furthermore discloses methods and means that solve certain difficulties related to diminished complementation of non-group C adenoviral vectors in Ad5 packaging/complementing cells. Although in the Ad5 complementing cell lines functional Ad5 E1B-55K expression is present, it was found that only very low titers of adenoviral vectors could be produced when the adenoviral backbone was of a non-group C adenoviral origin; this finding implies a serotype-specificity in the interaction of E1B-55K with another (viral) protein. It is disclosed here that this serotype-dependency can be circumvented by providing E4-orf6 protein compatible with the E1B-55K protein provided by the complementing cell line. As discussed herein, E1B-55K and E4-orf6 form a complex that is involved in inhibiting transport of cellular mRNA's from the nucleus to the cytoplasm, while the complex is also involved in stimulation of transport of viral mRNA's from the nucleus to the cytoplasm {reviewed in Leppard 1997 and 1998). It has been observed by the present inventors that proper complementation of viral vectors in packaging cells requires the presence of E1B-55K and E4-orf6 gene products that are compatible. Packaging cells are also referred to as complementing cells, if the cells comprise certain sequences encoding proteins that complement functions<br>
31<br><br>
not provided by  the vector that should be packaged. 'Compatible' as used herein therefore means that a complex between the available E1B-55K gene product is able to form a functional complex with the available E4-orf6 gene product in a sense that this protein complex supports viral replication, propagation and/or packaging to a level that is comparable to the wild-type situation or that is comparable to the situation found when a recombinant adenovirus serotype 5 vector is produced on a Ad5 complementing cell line such as 293 or PER.C6™. Vector replication in packaging cells is efficient if, during production period in which the virus is formed, the cell comprises at least E1B-55K protein and E4-orf6 protein that are compatible. Preferably, the E1B-55K and E4-orf6 sequences are from adenoviruses within the same adenovirus subgroup {such as A, B, C, D, E or F). More preferably, the E1B-55K and E4-orf6 sequences are from the same serotype. Since established cell lines are available in the art that are capable of supporting the growth of adenoviruses of subgroup C, such as serotype 5, it is even more preferred that the E1B-55K and E4-orf6 genes are derived from adenovirus serotype 5. As will be understood by the skilled person compatibility may be determined in complementation tests or assays as such in the realm of those skilled in the art of adenoviral vector production. The person skilled in the art will also understand that the present invention can also be used for the production of any adenovirus serotype on any complementing cell line as long as the E1B-55K and ,E4-orf6 proteins are compatible.<br>
It has further been observed that the E4-orf6 gene product, ^matching' with the ElB in the complementing cell line, can be provided by the adenoviral vector, by replacing the £4-orf6 in the adenoviral vector of choice with an E4-orf6 encoding sequence that is compatible with the ElB gene present<br><br>
within the packaging cell line. This modification was surprisingly found not to have a severe effect on the stability, replication, packaging, assembly and production of the vector.<br>
It is a specific aspect of the invention that one is now able to efficiently produce adenovirus serotypes different than those from subgroup C on cell lines normally applied for the production of adenovirus serotype 5 or other serotype from subgroup C, such as serotype 1, 2 and 6. The present invention provides methods for the production of non-group C adenoviruses without the necessity of separately providing the complementing (packaging) cell with E4-orf6 because the E4-orf6 sequence, that is compatible with the complementing E1B-55K sequence, is incorporated in the adenoviral backbone.<br>
The present invention provides a recombinant adenovirus vector comprising structural and non-structural elements of an adenovirus of a first serotype, wherein said vector further comprises a sequence encoding a functional E4-orf6 protein, or a functional part, derivative and/or analogue thereof, wherein said sequence is selected from the group consisting of: a) an E4-orf6 encoding sequence derived from an adenovirus of a second serotype different from said first serotype; b) an E4-orf6 encoding sequence derived from an adenovirus of said first serotype comprising a deletion, mutation, addition and/or substitution in one or more codons; and c) an E4-orf6 encoding sequence comprising a fusion between a part of an E4-orf6 encoding sequence derived from a second serotype different from said first serotype and a part of an E4-orf6 encoding sequence derived from a third serotype, wherein said third serotype may be identical to-, or different from said first serotype. In one embodiment, the present invention provides a recombinant adenovirus vector according to the<br><br>
invention, wherein said first serotype and said second serotype are from different adenovirus subgroups. In a preferred embodiment, a recombinant adenovirus vector according to the invention is provided, wherein said first serotype is from a subgroup other than subgroup C and wherein said E4-or£6 encoding sequence is derived from an adenovirus serotype of subgroup C. More preferred is a recombinant adenovirus according to the invention, wherein said first serotype is from subgroup B and said second serotype is from subgroup C. Even more preferably, said E4-orf6 encoding sequence is derived from adenovirus serotype 5. The recombinant adenoviruses of the invention comprise structural and non-structural elements. Examples of structural elements are the genes encoding the capsid proteins, such as fiber, hexon and penton proteins, as well as the gene products itself. Examples of non-structural elements are the early genes that are expressed upon infection into a cell and that are downregulated when the infection cycle proceeds, other examples of non-structural elements are the genes encoding the proteins active during replication, such as pol and pTP.<br>
Some alterations in the nucleic acid, such as a deletion, a mutation, addition and/or substitution in one or more codons may significantly change the structure and/or functionality of the encoded gene product. The present invention therefore also relates to E4-orf6 encoded sequences that are derived from the same adenovirus serotype as the backbone harbouring the genes for instance the structural and non-structural elements, but wherein the E4-orf6 encoding sequence has been mutated such that it has become compatible with the El proteins (such as the E1B-55K gene product) present in the complementing cell in which the adenoviral vector is to be produced. The codon may be altered completely to change the encoded amino acid, but it<br><br>
may also be mutated partly to change the encoded amino acid. Deletions of nucleic acids may result in loss of one or more encoded amino acids; while it may also result in frame-shifts. The present invention also relates to E4-orf6 sequences present in the adenoviral nucleic acid that comprise different parts derived from different serotypes, wherein the domains that render the protein functional in compatibility may be used from one serotype, while the remainder of the E4-orf6 sequence or a part thereof is derived from another (un)related serotype (for instance from the same subgroup, from different subgroups or from different species, or combinations thereof). It is therefore also within the scope of the present invention to apply E4-orf6 fusion proteins that are compatible. Such fusion protein may be the product of several pieces of nucleic acid.<br>
A person skilled in the art will be aware of the fact that besides all human adenoviruses numerous non-human adenoviruses have been identified in the art. Obviously, also non-human adenoviruses can be applied to reach the same results as disclosed by the present invention. It will be clear to the skilled person that compatibility between E1B-55K and E4-orf6 may not  be  limited to human   adenoviruses but that also elements from adenoviruses specific for different species can be compatible. Thus, it also another aspect of the invention that non-human adenoviruses can be produced to high titers on known packaging cell lines available in the art as long as the E1B-55K and E4-orf6 gene products are compatible. Non-limiting examples of non-human adenoviruses that can be produced using the methods and means of the present invention are canine-, bovine-, ovine-, frog-, porcine, equine-, monkey- and avian adenoviruses. Serotypes as used herein therefore goes beyond species-restricted serotypes. If for instance a monkey<br>
M3<br><br>
adenovirus E4-orf6 gene product is compatible with the E1B-55K provided by the packaging cell, then this combination is within the scope of the present invention. Also, when fusions are applied between different serotypes, or between E4-orf6 sequences derived from for instance a human and an avian adenovirus that is compatible with the E1B gene of the packaging cell, then that particular combination is also within the scope of the present invention.<br>
The present invention provides a method for producing a recombinant adenovirus vector comprising structural and non¬structural elements of an adenovirus of a first serotype, said method comprising the steps of: a) providing a complementing cell harbouring an E1B-55K encoding sequence, or a functional part, derivative and/or analogue thereof, derived from an adenovirus of a second serotype in expressible form, with the necessary elements of an adenovirus such as to allow assembly of said recombinant adenovirus vector by said complementing cell, wherein said elements comprise at least some structural and non-structural elements from an adenovirus of said first serotype different from said second serotype and a sequence encoding a functional E4-orf6 protein, or a functional part, derivative and/or analogue thereof, which is compatible with said expressible E1B-55K protein in said complementing cell; b) culturing said, complementing cell in a medium under conditions allowing for production and assembly of the adenovirus vector to take place; and c) harvesting the recombinant adenovirus vector so produced from the medium and/or the complementing cell, wherein the sequence encoding the compatible E4-orf6 protein is present in the recombinant adenovirus vector so produced.<br>
In one aspect of the invention, a method according to the invention is provided, wherein said E4-orf6 encoding sequence<br><br>
is selected from the group consisting of: i) an E4-orf6 encoding sequence derived from an adenovirus of said second serotype; ii) an E4-orf6 encoding sequence derived from an adenovirus of a third serotype different from said first and second serotype; iii) an E4-orf6 encoding sequence derived from an adenovirus of said first serotype comprising a deletion, mutation, addition and/or substitution in one or more codons; and iv) an E4~orf6 encoding sequence comprising a fusion between a part of an E4-orf6 encoding sequence derived from a third serotype and a part of an E4-orf6 encoding sequence derived from an adenovirus of said second serotype, wherein said third serotype may be identical to-, or different from said first serotype. In a preferred embodiment, said first and said second serotypes are from different subgroups. In a more preferred embodiment, said second serotype is an adenovirus serotype of subgroup C. In an even more preferred embodiment, said second serotype is adenovirus serotype 5. In other particular aspect of the invention, said first serotype is an adenovirus serotype of subgroup B. Preferably, said first serotype is selected from the group consisting of adenovirus serotypes 11, 14, 16, 21, 34, 35 and 50.<br>
There are several packaging cells known in the art that are used for complementing recombinant adenoviral vectors and to produce, assemble and package the adenoviral particles. Non-limiting examples of such cell lines are HEK-293, 911 and PER.C6m cells. It is preferred to use cell lines that have already proven to deliver high titers of adenoviral stocks. Such cell lines express El proteins in a stable manner, it is therefore a preferred aspect of the invention to use cell lines and methods, wherein said E1B-55K encoding sequence is integrated into the genome of said complementing cell. More preferred are complementing cells that are derived from a<br>
MS"<br><br>
primary, diploid human cell, or a progenitor cell thereof. Even more preferred, said complementing cell is derived from a primary human retinoblast cell, a primary human embryo-iic kidney cell, a primary human neuronal cell or a primar/ human amniocyte. Highly preferred is the use of a complementing cell in the methods provided by the present invention, wherein said complementing cell is a PER.C6™ cell or a derivative thereof. PER.C6™ cells are well known in the art for not givinc rise to replication competent adenovirus when adenoviral DNA is used that has no overlap with the nucleic acid provided by |the cells. Many of the adenoviral vectors used in the art lack the El region, therefore in one aspect of the invention, s]aid complementing cell comprises, integrated into its gencme, a<br><br>
em.<br>
nucleic acid encoding at least one adenovirus E1A prot<br>
Preferably, said nucleic acid encoding at least one aqenovirus E1A protein is derived from an adenovirus serotype of a subgroup different than subgroup B. More preferably, s!aid nucleic acid encoding at least one adenovirus E1A protjein is derived from an adenovirus serotype of subgroup C. Hie lily■ preferred are embodiments wherein said nucleic acid ercoding at least one adenovirus E1A protein is derived from ar adenovirus serotype 5. In another embodiment of the ii.vention, the invention provides a method, wherein said E4-orf6 encoding sequence and said E1B-55K encoding sequence are derived from different adenovirus serotypes and wherein said different adenovirus serotypes are members of the same adenovirus subgroup. Preferably, said E4-orf6 encoding sequence and said E1B-55K encoding sequence are derived from different adenovirus serotypes and wherein said different adenovirus serotypes are both members of subgroup C. More preferably, said £4-orf6 encoding sequence and said E1B-55K encoding sequence are derived from the same adenovirus serotypfi. Highly<br>
qb<br><br>
preferred are methods, wherein said E4-orf6 encoding sequence and said E1B-55K encoding sequence axe  derived from adenovirus serotype 5.<br>
The invention also relates to methods in which adenoviral vectors can be produced using the proper<br>
complementing/packaging cells and the adenoviral vector of interest. For an efficient production process it is useful to apply the correct cells with the proper adenoviral vector. Therefore, the invention also relates to a kit of parts (also referred to as 'packaging system') comprising: a) a complementing cell for producing a recombinant adenovirus vector comprising structural and non-structural elements of an adenovirus of a first serotype, said cell harbouring an ElB-55K encoding sequence, or a functional part, derivative and/or analogue thereof, derived from an adenovirus of a second serotype in expressible form; and b) on one or more replicable nucleic acid vectors all necessary adenoviral elements such as to allow assembly of said recombinant adenovirus vector by said complementing cell,   wherein said elements comprise at least some structural and non-structural elements from an adenovirus of said first serotype different from said second serotype and a sequence encoding a functional E4-orf6 protein, or a functional part, derivative and/or analogue thereof, which is compatible with said expressible E1B-55K protein in said complementing cell. Preferably, a kit of parts is used, wherein said E4-orf6 encoding sequence is selected from the group consisting of: a) an E4-orf6 encoding sequence derived from an adenovirus of said second serotype; b) an E4-orf6 encoding sequence derived from an adenovirus of a third serotype different from said first and second serotype; c) an E4-orf6 encoding sequence derived from an adenovirus of said first serotype comprising a deletion, mutation, addition<br><br>
and/or substitution of one or more codons; and d) an E4-orf6 encoding sequence comprising a fusion between a part of an E4-orf6 encoding sequence derived from a third serotype and a part of an E4-orf6 encoding sequence derived from an adenovirus of said second serotype, wherein said third serotype may be identical to-, or different from said first serotype.<br>
The invention is particularly useful for the replication of El deleted chimeric adenoviruses that are derived almost entirely from a serotype other than adenovirus 5. Such vectors need only to be provided with a nucleic acid encoding adenovirus 5 E4orf6 or a functional part, derivative and/or analogue thereof. Once provided therewith the vector can be efficiently replicated on normal adenovirus 5 El-complementing packaging cell lines. Stability of the vectors is improved, and vectors may be complemented for deletions in both E1A and ElB. By providing such vectors with a nucleic acid encoding adenovirus E4orf6, it is possible to enable efficient plaque purification and good yields in the absence of an additional wild type contamination problem, when grown on 293 or 911 cells. In PER.C6, of course, wild type adenovirus contamination can also be prevented in other ways.<br>
An additional advantage of a recombinant vector of the invention is that there is no need to generate special cell lines adenovirus E4-orf6 from a nucleic acid integrated into the genome. Although such cell lines exist, production parameters such as scaling up and the like, and/or regulatory issues may not have been resolved up the same degree as for cell lines such as PER.C6™. This may at least in part be due to the fact that with more and more foreign genes inserted into the genome of cell line it is difficult to maintain stability of all foreign sequences {or the expression<br><br>
thereof). in the present invention it was found that at least some of the problems associated with low yields of non-adenovirus serotype 5 based vectors and stability of adenovirus serotype vectors from subgroup B, such as adenovirus serotype 7, 11 and 35 on adenovirus serotype 5 packaging cell lines can be overcome with a recombinant adenovirus vector of the invention.<br>
It will be clear that also the two main aspects of the invention can be combined to provide for stable recombinant adenoviruses that can grow on convenient packaging cells that are readily available. The invention thus provides a recombinant adenovirus comprising a recombinant nucleic acid molecule having at least a deletion in the El region, characterized in that at least part of the sequence encoding the ElB 55K gene product increasing the expression of the pIX gene is present in said recombinant nucleic acid molecule, with the proviso that said recombinant nucleic acid molecule does not encode a functional ElB 55K gene product; said recombinant adenovirus further comprising structural and non¬structural elements of an adenovirus of a first serotype, wherein said adenovirus further comprises a sequence encoding a functional E4-orf6 protein, or a functional part, derivative and/or analogue thereof, wherein said sequence is an E4-orf6 coding sequence derived from an adenovirus of a second serotype different from said first serotype. Alternatively or in addition to having at least part of ElB 55K sequences, said nucleic acid may have a pIX gene product that is regulated by an heterologous promoter. Preferably, said second serotype and hence the E4-orf6 sequence is derived from a group C adenovirus, more preferably from an adenovirus serotype 5. Preferably, said first serotype is from a subgroup other than<br><br>
group C, preferably a subgroup B serotype such as Adll, Adl4, Adl6, Ad21, Ad34, Ad35 or Ad50. In one embodiment, the recombinant adenovirus further comprises a sequence encoding a non-adenoviral protein, polypeptide or peptide. Such recombinant adenoviruses are stable and can grow on readily available packaging cells, such as Ad5 El-containing packaging cells, preferably PER.C6™ cells, when E4-orf6 in the recombinant adenovirus is compatible with Ad5 El gene products, for instance when E4-orf6 in said adenovirus is derived from Ad5. The prevention of the generation of replication competent adenovirus or functional El protein containing adenoviral particles is a recognized problem in the art, and has been solved by preventing overlap between the El sequences present in the adenovirus with those in the packaging cell (US patent 5,994,128). In embodiments of the. invention, the combination of e.g. an Ad35 derived adenovirus comprising E1B 55K sequences that increase the stability of the virus by influencing pIX expression, with a packaging cell comprising El regions from Ad5, will not lead to homologous recombination with the concomitant formation of a recombinant adenovirus comprising a nucleic acid encoding a functional E1B 55K protein. It is thus another aspect of the present invention to provide a packaging system comprising a packaging cell and a recombinant adenovirus comprising structural and non-structural elements of an adenovirus of a first serotype, wherein said packaging cell expresses nucleic acid encoding at least a E1B-55K protein or a functional part, derivative and/or analogue thereof, derived from an adenovirus of a second serotype, and wherein said recombinant adenovirus can be replicated in said packaging cell to generate stable recombinant adenovirus, said adenovirus comprising a nucleic acid molecule having a deletion in the El-region and further<br><br>
comprising a part of the sequence encoding an E1B-55K gene product, said nucleic acid further comprising sequences encoding a functional E4-orf6 protein, or a functional part, derivative and/or analogue thereof, which is compatible with said expressible E1B-55K protein in said packaging cell. Preferably said first serotype is Ad 35 or Adll. Preferably said second serotype is Ad5. More preferably, said packaging cell is PER.C6™. Preferably, said sequences encoding a functional E4-orf6 protein, or a functional part, derivative and/or analogue thereof, are derived from an adenovirus of the first serotype. This system will allow replication of the recombinant adenovirus without the concomitant generation of functional El protein-containing adenovirus. Alternatively, or additionally, said nucleic acid could contain pIX coding sequences under the control of an exogenous promoter, it is another aspect of the invention to provide a method for producing a stable recombinant adenovirus comprising structural and non-structural elements of an adenovirus of a first serotype, wherein said recombinant adenovirus comprises a recombinant nucleic acid molecule derived from an adenovirus which nucleic acid molecule has a deletion in the El region and comprises nucleic acid derived from at least part of the sequence encoding the E1B-55K gene product increasing expression of the pIX protein not leading to the expression of a functional E1B-55K protein from said nucleic acid molecule and/or has a pIX coding sequence under control of a heterologous promoter, said method comprising the steps of: a) providing a complementing cell expressing an E1B-55K encoding sequence, of a functional part, derivative and/or analogue thereof, derived from an adenovirus of a second serotype in expressible form, with the necessary elements of an adenovirus as to allow assembly of said recombinant adenovirus vector by<br><br>
said complementing cell, wherein said elements comprise at least some structural and non-structural elements from an adenovirus of said first serotype different from said second serotype and a sequence encoding a functional E4-orf6 protein, or a functional part, derivative and/or analogue thereof, which is compatible with said expressible E1B-55K protein in said complementing cell and wherein; b) culturing said complementing cell in a medium under conditions allowing for production and assembly of the recombinant adenovirus to take place; and c) harvesting the recombinant adenovirus so produced from the medium and/or the complementing cell. The invention further provides methods for using the recombinant adenoviruses from the invention, for treatment of the human or animal body, for vaccination, gene therapy, and for the preparation of medicaments for treatment of diseases or disorders. The invention also provides pharmaceutical preparations comprising the adenoviruses according to the invention.<br>
The invention will now be illustrated with some examples/ which are not intended to limit the scope of the invention.<br>
EXAMPLES Standard molecular biology methods were used (e.g. Sambrook and Russel, 2001), unless otherwise indicated. Primer sequences are provided in Table IV.<br>
Example 1. PER.C6™-based complementing cell lines for El-deleted Ad35 viruses.<br>
PER.C6 cells were seeded in 10 cm culture dishes at a density of 3x106 cells/dish in PER.C6 culture medium (DMEM (Gibco BRL) complemented with FBS (Gibco BRL) up to 10% and<br>
9-<br><br>
lOmM MgC12 (4.9 M stock solution, Sigma)). Two days later, 9 dishes were transfected with 1 ug Seal linearised pIG35.55K DNA (described infra) and 9 dishes were transfected with 1.5 ug Seal linearised pIG35.55K DNA. Separate control dishes were transfected with 1 or 1.5 ug Seal linearised pAdApt35. LacZ (described in WO 00/70071) to monitor transfection efficiency and with I   or 1.5 ug Seal linearised pcDNA.nlsLacZ. pcDNA.nlsLacZ {described in W099/55132) is a pcDNA3-based plasmid (Invitrogen) with the nlsLacZ gene driven by the CMV promoter. pcDNA.nlsLacZ also contains a neoE expression cassette. As a negative control one extra dish was transfected with linearised pAdApt35.LacZ, a construct that lacks the neor selection gene. All transfections were performed with the LipofectAmine transfection kit (Invitrogen/Life Technologies) according to manufacturers instructions using 5 ml LipofectAmine reagent/u.g DNA. Cells were incubated for 4 hrs with the transfection mixture after which the medium was replaced with PER.C6 culture medium. The next day medium was replaced with culture medium containing 0.5 mg/ml G418 (Gibco BRL) except in the two dishes that were transfected with 1 or 1.5 ug pAdApt35.LacZ. These latter dishes were used to monitor LacZ expression two days following transfection. After X-gal staining of these cultures transfection efficiency was estimated at approximately 40% with slightly more blue cells<br>
in the dish transfected with 1.5 (J.g DNA. Selection medium was refreshed twice weekly in the remaining transfected dishes. Within two weeks following first addition of selection medium most cells in the negative control dish (transfected with 1.5 Ug pAdApt35.LacZ) were dead. In the dishes transfected with pcDNA.nlsLacZ cell clones were becoming visible. Since the cells transfected with pIG35.55K seemed to be more resistent to G418, the concentration was raised to 0.75 mg/ml 3 weeks<br><br>
following transfection. Three days and seven days later a total of 196 cell clones were picked from the dishes transfected with pIG35.55K and seeded in separate wells of 96-well plates.<br>
Cells remaining after colony picking of two 10 cm dishes of the transfection with 1 jxg pIG35.55K DNA were trypsinised, pooled and expanded to give pool PER55K(1.0)  The same was done for two dishes of the 1.5 fig transfection. The PER55K(1.0) cell pool was expanded and seeded in 4 T25 flasks at a density of 3,5x106 cells/flask for transfection to test virus generation. In addition, 3 T25 flasks with parental PER.C6 cells were seeded at the same density.  pAdApt35.eGFP (an adapter plasmid based on pAdApt35IPl (described in WO 00/70071) but also containing the green fluorescent protein as marker gene, which was cloned into pAdApt35lPl as Hindlll-BamHI fragment derived from pIPspAdapt.eGFP {described in WO 02/24933)) was digested with Pad to liberate the adenoviral sequences from the plasmid backbone. pWE.Ad35.pIX-rlTR (described in WO 00/70071) was digested with NotI to liberate the adenoviral sequences  from the cosmid backbone. 2 flasks with PER.C6 cells and 2 flasks with PER55K(1.0) cells were transfected with 2 ug digested pAdApt35.eGFP and 6 ug digested pWE.Ad35.pIX-rITR each. One flask of each cell line was transfected with 8 ug pAdApt35.LacZ to monitor transfection efficiency. The remaining flask with PER55K(1.0) cells served as a negative control and was treated as the others but did not receive the transfection mixture. All transfections were performed with LipofectAmine (Invitrogen/Life Techn.) according to manufacturers instructions using for each transfection a total of 8 ug DNA and 4 0 ul LipofectAmine reagent. The transfection mixture was removed after 4 hrs<br><br>
incubation and fresh culture medium was added. Transfections were done the day after seeding of the cells and again two days later cells in the T25 flasks were transferred to a T80 flask except for the LacZ control transfections. These were stained with X-gal solution after mild fixation. After five hours incubation with staining solution, the percentage of blue cells was estimated at approximately 90% in both flasks showing that transfection went well for both cell lines. Four days following the passage to the T80 flasks the transfected PER55K(1.0) cultures showed starting CPE (cytopathogenic effect, indicative of virus replication) with approximately 100 events/flask. The untransfected PER55K{1.0) cells were grown confluent with no evidence of CPE. In the transfected PER.C6 cultures only three CPE events were visible in the confluent monolayer of cells.  Again three days later, the transfected PER55K(1.0) cultures showed full CPE, with all cells rounded and detached in clumps. In contrast, in the PER.C6 cultures the few events of CPE had not progressed and cells were still in monolayer. This confirms earlier observations that generation of El-deleted Ad35-based viruses on PER.C6 is very inefficient. Also the untransfected PER55K(1.0) cultures showed, as expected, a confluent monolayer with no CPE. The cells and medium in the PER55K(1.0) flasks with full CPE were harvested and subjected to two freeze/thaw cycles after which the cell debris was removed by centrifugation at 3000 rpm for 10 minutes in a table centrifuge. One of the resulting crude lysates was used to infect a fresh culture of PER55K(1.0) cells in a T175 flask (1.5 ml/flask), Cells and medium were harvested at full CPE four days later. This shows that infectious virus had formed in the initial transfections. GFP expression was confirmed by fluorescent microscopy of A549 cells infected with the crude<br><br>
lysate. The crude lysate was then used to analyse complementation of this El-deleted Ad35.AdApt.eGFP virus in the individual clones as described below.<br>
The supra described clones that were picked from the pIG35.55K transfected PER.C6 cells were expanded and were functionally tested for the ability to sustain replication of Ad35.AdApt.eGFP. Hereto, the clones were seeded at two densities in 6-well plates and one day later infected with 15 ml of the above described crude lysate. CPE was monitored the day after. Of the 146 clones tested in this way 19 gave full CPE at day 2 or 3 and 68 gave full CPE at day 5 or 6. The remaining clones had only partial CPE or showed a few non-progressing events. The latter were indistinguishable from PER.C6 cells that were taken along as a negative control.<br>
Based on these results a selection of 24 clones was made that were further screened for the ability to generate recombinant El-deleted viruses following transfection of the pAdApt35.GFP adapter plasmid and the large pWE.Ad35.pIX-rITR cosmid clone. Hereto, clones were plated in T25 flasks and transfected with 2 \xg  of the adapter and 6 jig of the backbone plasmid using LipofectAmine as described above. Two days following the transfection, cells were transferred to T80 flasks to prevent overconfluency of the cultures. Of the 24 clones 5 gave full CPE three days after passage to T80 and another 13 clones gave progressing to full CPE the day after. The remaining 6 clones showed no CPE or only starting. In comparison: routine generation of El-deleted Ad5 vectors on PER.C6 cells generally results in full CPE four to six days after transfer to T80 flasks.<br>
This shows that the new clones efficiently complement El-deleted adenovirus vectors. One of the clones {clone #16)<br><br>
described above was used to generate and produce multiple batches of El and E1/E3 deleted Ad35 viruses containing different transgenes. Hereto, virus in crude lysates resulting from transfections as described above, but using different adapter plasmids, were plaque purified on the new cell line. Single plaques were tested for transgene activity and then amplified for medium scale production in 4-8 triple layer flasks (3x175 cm/flasK). Cells were harvested at full CPE and the virus was released and purified as routinely done for adenoviruses. The amount of virus particles was determined by HPLC (Shabram et al., 1997). Table I presents the yields after downstream processing of medium scale productions of El- and El/E3-deleted Ad35 viruses on triple layer flasks with PER55K clone #16 cells. The amount of purified virus particles is comparable with the yields of Ad5-based vectors on PER.C6 cells.<br>
We conclude that we have generated multiple cell lines that efficiently complement fully El-deleted Ad35-based vectors. Thus, Ad35 E1B-55K expression in an Ad5 complementing cell line facilitates replication of Ad35 vectors.<br>
Example 2. Generation of pWE.Ad.pIX-rITRAE3.<br>
The early region^3 of human adenoviruses contains multiple coding regions for proteins that interfere with the host immune response to adenoviral infection. When adenoviral vectors are used as vaccine carrier such interference is unwanted. Therefore, we constructed an Ad35 backbone cosmid lacking the E3 region.<br>
Hereto, construct pBr.Ad35.PRn (Fig. 15; described in example 13 in publication EP1054064) was digested with StuI and Mlul and the 17.3 kb vector fragment was purified from low melting point (LMP) gel using agarase enzyme (Roche) according<br><br>
to manufacturers instructions. Next, a PCR fragment was generated on pBr.Ad35.PRn using primers 35E3for and 35E3rev. For the amplification Pwo DNA polymerase (Roche) was used according to manufacturers instructions and program set at: 94°C for 2 minutes, 30 cycles of (94°C for 30 seconds, 58°C for 30 seconds and 72°C for 1 minute) and a final incubation at 68°C for 8 minutes. The 833 bp PCR product was purified using the QIAquick PCR purification kit (Qiagen) and digested with Mlul and StuI. The digested DNA was purified from gel using the QIAquick gel extraction kit (Qiagen). Both isolated fragments were ligated and transformed into DH5a competent cells (Invitrogen/Life Technologies) to give pBr.Ad35.PRnAE3 (Fig. 25). The plasmid was checked by restriction analysis and sequencing of the PCR amplified insert. The E3 deletion was then cloned into the pWE.Ad35.pIX-rlTR cosmid backbone. Hereto, pWE.Ad35.pIX-rlTR was digested with Pad and the DNA was purified by precipitation with isopropanol and washing with 70% EtOH. Following resuspension in milliQ water, the DNA was digested with Swal and the 22.8 kb vector containing fragment was purified from LMP gel using agarase enzyme as above. Construct pBr.Ad35.PRnAE3 was digested with PacI and Swal in the same manner and the 16.6 kb fragment was also isolated using agarase enzyme. Both isolated fragments were ligated using 0.5-0.6 ug of each fragment. Ligated fragments<br>
were then packaged using X-phage packaging extracts (Stratagene) according to manufacturers instructions and mixed with STBL-2 cells. Bacteria were plated on LB+Amp plates and resulting colonies were analyzed for the presence of the correct construct. This gave construct pWE.Ad35.pIX-rITRAE3 (Fig. 1). The E3 deletion extends from nucl. 27648 to 30320 of<br>
5*<br><br>
the Ad35 sequence (described in WO 00/70071) and thus spans a 2.6 kb region.<br>
Cotransfection of NotI digested pWE.Ad35.pIX-rITRAE3 and pIPsp-1 (New England Biolabs) digested pAdApt35.eGFP onto PER55-clone #16 cells (described supra) gave rise to GFP expressing Ad35-based viruses. Upon isolation of viral DNA from these viruses, PCR amplification of the E3 region showed that the viruses were deleted for 2.6 kb of E3 sequences as expected.<br>
Example 3. Limits in packaging size El-deleted Ad35-based vectors.<br>
Ad35-based El-deleted and El/E3-deleted vectors containing different inserts were generated by transfection of PER55K-clone#16 cells (see example 1) with<br>
1.5 Jig of an Ad35 adapter plasmid carrying a specific transgene digested with Pad or pIPsp-1 enzyme to liberate the adenovirus insert from the plasmid vector sequences and,<br>
4.5 Jig of either pWE.Ad35.pIX-rlTR digested with Not! or with pWE.Ad35,pIX-rITRAE3 digested with NotI enzyme.<br>
The right flank of the adapter plasmids and the left end of the backbone plasmid contain homologous sequences mediating recombination events that lead to a complete El-deleted viral genome (as described in WO 00/70071).<br>
Transfections were done with 30 j-ll Lipofectamine reagent (Invitrogen/Life Technologies) for each set of constructs according to manufacturers instructions. Transfection mixtures were added to PER55K clone 16 cells at 70% confluency in T25 flasks. The following combinations were transfected:<br>
1.	pAdApt35IPl + pWE.Ad35.pIX-rITR<br>
2.	pAdApt35IPl + PWE.Ad35.pIX-rITRAE3<br><br>
3.	pAdApt35eGFP + pWE.Ad35.pIX~rITR<br>
4.	pAdApt35eGFP + pWE.Ad35.pIX-rITRAE3<br>
5.	pAdApt35Luc + pWE.Ad35.pIX-rlTR<br>
6.	pAdApt35Luc + pWE.Ad35.pIX-rITRAE3<br>
7.	pAdApt35Lac2 + pWE.Ad35.pIX-rlTR<br>
8.	pAdApt35LacZ + pWE.Ad35.pIX-rITRAE3<br>
Adapter plasmids were digested with pIPsp-1 enzyme to liberate the adenovirus sequences from the plasmid vector backbone. pWE.Ad35.pIX-rITR and pWE.Ad35.pIX-rITRAE3 were digested with NotI prior to transfection for the same reason. Generation of the adapter plasmids and of the pWE.Ad35.pix-rlTR backbone cosmid is described previously in WO 00/70071. Generation of pWE.Ad35.pIX-rITRAE3 is described supra.<br>
Two days following transfection cells were passaged to T80 and further incubated until full CPE was obtained. Cells and medium were harvested 1-2 days after full CPE was noticed. The mixtures were subjected to one freeze/thaw cycle and spun down at 1500 rpm for 15 minutes to pellet cell debris, after which supernatants were collected. The crude lysates obtained in this way were used to isolate viral DNA. Hereto, 275 M-l of crude lysate material was incubated with 10p.l lOmg/ral DNasel at 37°C for 30 minutes. Subsequently, 6.0 ul 0.5 M EDTA (pH 8.0), 7.5 |il 20% SDS and 1.5 ^1 20 mg/ml Proteinase K was added and mixed by vortexing. The mixture was then incubated at 50°C for 1 hour. Finally, the viral DNA was isolated using the GeneClean Spin Kit (Bio 101, Inc.). Following elution of the viral DNA in 20 u,l milliQ H20, the transgene region was analysed by PCR amplification. Hereto, primers AdApt35CMVF and 35pIXR were used. The amplifications were done with 2 (j,l of the isolated viral DNA using Taq DNA polymerase (Invitrogen). The<br><br>
reaction mixtures contained 5 jxl lOx buffer (Invitrogen), 2 |4.1 50mM MgC12, 5 [Xl 2mM dNTPs, 3 |Xl of each primer (lOjlM stock) and 2.5 units Taq enzyme in a total volume of 50 \il.   The program was set at 94 °C for 2 minutes followed by 30 cycles of (94°C for 30 seconds, 60°C for 30 seconds and 72°C for 4 minutes). Control reactions were done on 5 ng of adapter piasmids. After completion of the PCR, 5 u.1 of the reaction was loaded on gel for analysis. Figure 2 shows the results for the above mentioned transfections. The primers amplify sequences from the 5'end of the CMV promoter to the 5' end of the pIX coding region. As can be seen in Figure 2, viruses without transgene or with GFP insert show the expected band (compare with the plasmid controls; lanes PL for each virus). Smaller fragments are seen with the larger inserts, luciferase and LacZ, and these deletions become more prominent with larger total length of the virus (compare LacZ or Luc viruses with and without E3). Thus, increasing genome length corresponds with the occurrence of deletions in the transgene region. The fact that the total genome length (also indicated in Fig. 2) of Ad35.AdApt.eGFP and Ad35.AdApt.LacZAE3, 33.7 and 33.4 kb respectively, is comparable while deletions are only found in the LacZ virus sample, indicates that either the sequence or the size of the insert in the former El region can also influence the occurrence of the deletions.<br>
Example 4. Sequence comparison of the pIX gene region of adenovirus serotype 5 and 35.<br>
The finding that with increasing genome length of recombinant Ad35 viruses, deletions occur in the transgene region could suggest, in analogy with the pIX-deficient Ad5<br>
61<br><br>
viruses (Ghosh-Choudhury et al., 1987), that there is a problem with the stability of the capsids. The fact that Ad35 .E1B+.AdApt.Luc viruses {where ElA sequences are replaced by the AdApt.Luc cassette), with a total length of 36.7 kb can be made, indicates that the deletion of ElB plays a role. This might be either through a function of one of the ElB proteins itself or through a function of unknown regulatory sequences in this region having influence on expression of other adenoviral proteins. It has, to our knowledge, not been described that ElB proteins itself influence the packaging capacity of adenoviruses. However, it has been described that the E1B-21K protein non-specifically stabilizes transfected DNA (Herrraan and  Mathews,   1989)   and that mutations in the 21K protein result in degradation of cellular and viral DNA during infection (Pilder et al., 1984/ White et al., 1984). Since the Ad5 E1B-21K protein is expressed in PER.C6 cells, these findings do not provide an explanation for our observations. The pix gene is located directly 3' of the E1B-55K coding region. For Ad5 it is known that the pIX promoter and coding sequences are located within the ElB transcription region since pIX and ElB share the poly-adenylation signal. The minimal promoter sequences necessary for pIX expression have been studied in case of Ad5 (Babiss and Vales, 1991). It was shown that a promoter fragment containing the upstream Spl site and the TATA-box sequence was sufficient for prx expression. The spacing between the Spl site and the TATA-box as well as the sequence of the TATA-box itself, were shown to influence the level of pIX expression. Whether the corresponding region in Ad35 is also sufficient to drive pIX expression to a level high enough for stable viruses is not known. Sequence comparison revealed that both the Spl site and the TATA sequence are different from those found in the Ad5<br><br>
pIX promoter. Using sequence information available from Genbank, a comparison was made of the proximal pIX upstream sequences (i.e. between the stop codon of E1B-55K and the start codon of the plx gene) of serotypes from different subgroups. The following adenoviruses with SEQ. ID. NOs.and Genbank reference sequences were used for the comparison: Ad2 (SEQ. ID. NO. 45; Genbank NC_001405) , Ad5 (SEQ. ID. NO. 46; Genbank M73260), Adl2 (SEQ. ID. NO. 47; Genbank NC_001460) , Ad9 (SEQ. ID. NO. 48; Genbank  AF099665), Ad40 (SEQ. ID. WO. 49; Genbank L19443), Ad4 (SEQ. ID. NO. 50; NC_003266), Simian 25 (SEQ. ID. NO. 51; Genbank AF394196) , Ad7 (SEQ. ID. NO. 54; Genbank AD7001). The Ad35 sequence (SEQ. ID. NO. 52) was as published in WO 00/70071. The Adll sequence (SEQ. ID. NO. 53) was not published before and is provided herein. Fig. 3A shows an alignment of the above mentioned sequences between the stop codon of the E1B-55K protein (first 3 nucleotides in all sequences) and the start codon of the pIX protein {last 3 nucleotides). The Spl site and the TATA sequence in Ad2 and Ad5 are boxed. In most cases there is insufficient homology to directly point the Spl and TATA boxes in the other sequences. Therefore, the consensus sequences for GO and TATA-boxes as published by Bucher, p (1990) was used to identify the putative Spl and TATA-box in the various sequences. Fig. 3b shows the putative Spl and TATA-box sequences and the spacing between them. Adl2, Ad9 and Ad40, belonging to respectively subgroup A, D and F, have Spl and TATA sequences that fairly match the consensus sequence. However, the distance between the two boxes is smaller than for Ad5 and Ad2. This is not unusual since the Ad5 E1B promoter also contains an Spl box and a TATA sequence with a spacing of 11 nucleotides. However, a deletion of 9 nucleotides (of the 20) in the Ad5 pIX promoter sequence between the Spl- and TATA-boxes gave reduced<br>
63<br><br>
pIX levels (Babiss and Vales, 1991). The subgroup B serotypes Ad35, Adll and Ad7 as well as subgroup E virus Ad4, have divergent TATA-box sequences and different spacing between the putative Spl sequence and the TATA-box. The proximal pix region in human adenovirus type 4 is identical to that in the simian adenovirus 25 (CV68), a serotype that recently was proposed as therapeutic vector (Farina et al., 2001), Thus, also for replication-deficient vectors based on non-human adenoviruses pIX expression may be insufficient for stable capsids.<br>
It may well be that pIX expression is regulated differently in Ad35 viruses and other human and non-human adenoviruses and that regulatory sequences, or even the promoter sequences itself, are located further upstream in the ElB sequences or even more upstream. Alternatively, it is also possible that, since pIX expression is activated by E1A proteins, high levels of pix expression are obtained in the presence of E1A proteins belonging to the same serotype or subgroup.<br>
We tested whether changing the endogenous proximal pIX upstream sequences into a heterologous promoter to increase pIX expression in the vector, leads to more stable viruses and a better packaging capacity (infra). Alternatively, the pIX function may be delivered in trans via the packaging cell line. As a non-limiting example we describe recombinant Ad35-based viruses that have a non-endogenous proximal pIX promoter as found in Ad5 viruses, and show that these viruses have a better stability than the unchanged recombinant vectors.<br>
Example 5. Generation of adapter plasmids with an Ad5 pIX promoter<br>
to<br><br>
pAdApt535 is an Ad35 adapter plasmid having part of the Ad5 pIX promoter sequences but is otherwise identical to Ad35 adapter plasmid pAdApt35IPl (see WO 00/70071). Its construction is described below:<br>
A first PCR fragment was generated with primers SV40for and pIX5Rmfe. The reaction was done with Pwo DNA polymerase (Roche) according to manufacturers instructions but with 3% DMSO in the final mix. pAdApt, an adapter plasmid for Ad5 El-deleted viruses (100 ng; see WO 00/70071) was taken as template. The program was set as follows: 2 minutes at 94 °C and then 30 cycles of (94°C for 30 seconds (melting), 52°C for 30 seconds (annealing) and 72°C for 30 seconds (elongation)) followed by 8 minutes at 72 °C. The resulting PCR fragments contain the 3' end of the SV40 polyadenylation signal from pAdApt and the Ad5 pIX promoter region as present in Genbank Accession number M73260 from nucleotide 3511 to nucleotide 3586 and an Mfel site at the 3'end.<br>
A second PCR fragment was generated as described above but with primers pIX35Fmfe and 35R4. 100 ng pAdApt35IPl was taken as template, the annealing was set at 58°C for 30 seconds and the elongation of the PCR program was set at 72°C for 90 seconds. This PCR amplifies Ad35 sequences from nucleotide 3467 to nucleotide 4669 (sequence numbering as in WO 00/70071) and adds an Mfel site to the 5'end.<br>
Both PCR fragments were then digested with Mfel and purified using the Qiagen PCR purification kit (Qiagen) according to manufacturers instructions. Concentration of the purified fragments was estimated by running a sample on agarose gel and approximate equimolar amounts of the two fragments were mixed in a ligation reaction containing 5 |lg DNA., 4 |il lOx ligase buffer and 2 ^,1 ligase enzyme (New England<br><br>
Biolabs) in a 40 u.1 volume. Following an incubation of &gt; 2 hours at RT, the mixture was loaded on a 1.2% agarose gel in TAE and the DNA fragmens of 1.4 kb length were isolated with the Geneclean II kit (BiolOl, Inc.) according to manufacturers instructions.<br>
The DNA was eluted in 30 jxl sterile H20 and 1 (Xl was used in a PCR amplification reaction with primers SV40£or and 35R4 as described above. The PCR was done as described above with an annealing temperature of 52°C and an elongation time at 90 seconds. The resulting product was isolated from gel using the Qiagen gel extraction kit and digested with Agel and Bglll. The resulting 0.86 kb band was isolated from gel using the Geneclean II kit according to manufacturers instructions.<br>
pAdApt35.Luc (described in WO 00/70071) was also digested with Bglll and Agel and the 5.8 kb vector fragment was isolated from gel using the Geneclean II kit as above. This fragment was ligated with the isolated Bglll-Agel fragment described supra containing the Ad5-Ad35 chimeric pIX promoter, to give pAdApt535.Luc (Fig. 4).<br>
Other adapter plasmids containing the Ad5 pIX promoter were then made as follows:<br>
pAdApt535.Luc was digested with Bglll and Apal and the 1.2 kb insert was purified from gel using the Geneclean II kit according to manufacturers instructions. pAdApt35IPl was also digested with Bglll and Apal and the 3.6 kb vector fragment was isolated as above. Ligation of both isolated fragments resulted in pAdApt535 (Figure 5). Next, pAdApt535 was used to clone other marker genes like eGFP (derived from pAdApt35.eGFP) and LacZ (derived from pAdApt35.LacZ) into the multiple cloning site using standard cloning techniques giving rise to pAdApt535.eGFP and pAdApt535.LacZ.<br><br>
Example 6. Generation of El-deleted Ad35-based vectors with adapter plasmids containing the Ad5 pIX promoter.<br>
Recombinant viruses were generated by transfection of adapter plasmids and Ad35 vector backbone cosmids on PER55K clone 16 cells as described above. Hereto, the following set of plasmids were used:<br>
Tl.	pAdApt535eGFP + pWE.Ad35.pIX-rlTR<br>
T2.	pAdApt535eGFP + pWE.Ad35.pIX-rITRAE3<br>
T3.	pAdApt35Luc + pWE.Ad35.pIX-rITR<br>
T4.	pAdApt535Luc + pWE.Ad35.pIX-rlTR<br>
T5.	pAdApt535Luc + pWE.Ad35.pIX-rITRAS3<br>
T6.	pAdApt535LacZ + pWE.Ad35.pIX-rlTR<br>
T7.	pAdApt535LacZ + pWE.Ad35.pIX-rITRAE3<br>
T8.	pAdApt35LacZ + pWE.Ad35.pIX-rlTR<br>
T9.	pAdApt35LacZ + pWE.Ad35.pIX-rITRAE3<br>
AdApter plasmids were digested with Pad except pAdApt535.Luc and pAdApt35.Luc which were digested with pIPsp-1 enzyme and pWE.Ad35.pIX-rlTR and pWE.Ad35.pIX-rITRAE3 were digested with NotI prior to transfection. 2 u,g of each adapter plasmid and 6 ^g of the backbone DNA were mixed with 40 (il Lipofectamine (Invitrogen/Life Technologies) according to manufacturers instructions and incubated with PER55K clone 16 cells in T25 flasks at 70% confluency. Transfection medium was removed after 4 hrs and cells were further incubated at 37°c/10% C02. Two days after transfection, cells were passaged to a T80 flask and scored for occurrence of cytopathogenic effect (CPE) the days after. Five days later all cultures showed progressing or full CPE except T6 (no CPE) and T8 (CPE events). Again two days later, T6 and T8 showed starting CPE<br><br>
and all others full CPE. All cultures were harvested by collecting medium and cells. The mixtures were stored at -20°C. Upon thawing of the samples, the mixtures were spun down at 1500 rpm for 15 minutes to pellet cell debris and supernatant were collected. Of some of the samples (the four LacZ expressing viruses, T6-T9) 2 ml was used to infect again PER55K clone 16 cells at 80% confluency in a T80 flask to further amplify the virus titre. Cells and medium were harvested upon progressing (T6+T8) or full CPE (T7+T9) and crude lysates were prepared as described above.<br>
The crude lysates obtained in this way were used to isolate viral DNA. Hereto, 275 |xl of crude lysate material was incubated with lOul lOmg/ml DNasel at 37°C for 30 minutes. Subsequently, 6.0 \ll   0.5 M EDTA (pH 8.0), 7.5 \il   20% SDS and 1.5 (il 20 rug/ml Proteinase K was added and mixed by vortexing. The mixture was then incubated at 50°C for 1 hour. Finally, the viral DNA was isolated using the GeneClean Spin Kit {Bio 101, Inc.). Viral DNA was eluted  in 50 pi milliQ H20 and 5 |il samples were used to analyse the transgene region. It should be noted that the pWE.Ad35.pIX-rITR+/-E3 backbone cosmids were unchanged and therefore still contain the Ad35 pIX promoter. Since this promoter is located at the very 5' end of the cosmid, the chances for a recombination event resulting in the wt Ad35 promoter were considered to be small. However, it could not be excluded in this set-up that viruses would be generated that still contain the Ad35 pIX promoter. Therefore, two specific PCR amplification were performed on each virus preparation: The first was done with primer set 1 {Ad35 specific): AdApt35CMVF and AdApt35pIXrev. This PCR specifically amplifies the transgene region in viruses containing the Ad35 pix promoter. The PCR reaction was done on<br>
6*<br><br>
5 ul  of the isolated viral DNA samples with recombinant Taq polymerase (Invitrogen) according to manufacturers instructions but with using 4mM MgC12 and 4 units Taq enzyme in the reactions. The PCR program was set at 94 °C for 2 minutes followed by 30 cycles of {94°C for 30 seconds, 60°C for 30 seconds and 72°C for 5 minutes) and ended with a final step of 8 minutes at 68aC.<br>
The second was done with the primers AdApt35CMVF and plX5Rmfe and thus specifically amplifies the transgene region in viruses containing the Ad5 pIX promoter (primer set 2).<br>
PCR amplification was done on 5 |J.l of the isolated viral DNA using Pwo DNA polymerase {2.5 units/[il Genaxis) in 50 p.1 volume containing 0.3 pi of each primer (100 |XM stock), 5 fil 2mM dNTP mixture, 5 u.1 lOx complete buffer (incl. Mg2+), 1.5 jll DMSO and 0. 8 |Xl Pwo enzyme. The PCR program was set at 94 °C for 2 minutes followed by 30 cycles of (94°C for 30 seconds, 60°C for 30 seconds and 726C for 5 minutes) and ended with a final step of 8 minutes at 68°C. During PCR the heating and cooling ramps were set at 2 °C/second. Then, 5 [Xl loading buffer was added to the samples and 8 (il of the mixture was loaded on gel for analysis.<br>
The El and E1/E3 deleted Ad35 viruses containing the Ad5 pIX promoter sequence and eGFP transgene (transfections Tl and T2) had PCR amplified bands at the expected height with no shorter fragments. Fig. 6a shows the results for the PCR amplifications on the El-deleted Luciferase carrying viruses (transfections T3 and T4). Lanes 5-8 are the control PCRs on AdApt535Luc (lanes 5 and 6) and AdApt35Luc plasmids (lanes 7 and 8) with both primer sets. Lanes 1-4 are PCRs on the viral DNA isolates. Primer set 1 {specific for Ad35 pIX region) amplifies a band of the expected length and shows in addition<br><br>
shorter fragments on Ad35.AdApt35.Luc viruses {lane 4; compare also Fig. 2 Luciferase +E3). In contrast, primer set 2 (specific for the Ad5 pIX promoter) only shows a band of the expected length with no deletion fragments when viruses are made with the AdApt535.Luc plasmid (lane 1). From this we conclude that the insertion of Ad5 pix promoter sequences increases the stability and the packaging capacity of Ad35-El deleted viruses. Figure 6b confirms these results for Ad35 E1/E3 deleted viruses carrying LacZ as transgene. Lanes 1-4 are the control PCRs on AdApt535.LacZ and AdApt35.LacZ plasmids with each primer set. Some background bands are seen especially with primer set 1 (lanes 2 and 4) but a strong specific band is also seen at the expected height for each primer set on the homologous samples (lanes 1 and 4). Viral DNA was isolated after transfection and after one amplification round as described above. Strikingly, primer set 2 generates the expected fragment on Ad35.AdApt535.LacZ viruses with no deletion fragments (lanes 5 and 9) whereas the sample with viruses containing the Ad35 pix promoter sequence clearly shows deleted fragments in addition to a fragment of the correct length (visible after amplification (lanell)■<br>
Altogether, these results show that substitution of the Ad35-pIX promoter sequences for Ad5-pIX promoter sequences increase the stability of the transgene region in viruses with larger genomes. Stronger promoters or additional promoter elements may even enhance this effect.<br>
Exanple 7. Generation of pWE.Ad35-348l. As indicated above, the adenovirus insert in the cosmid pWE.Ad35.pIX-rITR contains the Ad35 pIX promoter at its 5' end. This could lead to re-insertion of the Ad35 pIX promoter into viruses generated with the pAdApt535-based adapter<br><br>
plasmids. Therefore, a new version of the Ad35 backbone cosmid is made that lacks pIX promoter sequences. Hereto, a PCR fragment was generated with the pIXcosF-2 and Adapt35-3 primers. The amplification was done with Pwo DNA polymerase (2.5 units/fil; Genaxis) in 50 UM volume containing 3 |Xl of each primer (10 \m  stock), 5 (il 2mM dNTP mixture, 5 \il   lOx complete buffer (incl. Mg2+), 1.5 fil DMSO, 0.5 (11 Pwo enzyme and 10 ng pAdApt35IPl template. The PCR program was set at 94°C for 2 minutes followed by 5 cycles of (94°C for 30 seconds, 58°C for 30 seconds and 72°C for 1.5 minutes) and then 25 cycles of (94°C for 30 seconds, 60°C for 30 seconds and 72°C for 1.5 minutes) and ended with a final step of 8 minutes at 68"C. The resulting 1.2 kb PCR product contains Ad35 sequences from nucleotide 3481 to nucleotide 4663 (numbering according to Ad35 sequence as published in WO 00/70071) with an Aatll and NotI site attached to the 5'end. The PCR product was purified using the PCR purification kit (Qiagen) according to manufacturers instructions and cloned into the pPCR-Script Amp vector (Stratagene) according to manufacturers instructions. The sequence of the cloned fragment is then verified by sequencing and subsequently removed from the construct by digestion with Aatll and Agel. The resulting 780 bp fragment is purified from gel using the Geneclean spin kit (BiolOl, Inc.) according to manufacturers instructions.<br>
Construct pWE.Ad35ANdeI (described infra) is also digested with Aatll and Agel and the resulting 12 kb vector fragment is isolated from gel using the Geneclean spin kit (BiolOl, Inc.) according to manufacturers instructions. Ligation of both isolated fragments results in construct pY3E.Ad35-3
The construction of construct pWE.Ad35ANdeI is described in WO 00/70071 and contains Ad35 sequences from  nucleotide<br><br>
3401 to the Ndel site at nucleotide 6541 and Ad35 sequences from the Ndel site at nucleotide 33167 to the end of the right ITR whereby both Ad35 fragments are linked via the Ndel site (see also Figure 13 in WO 00/70071).<br>
pWE.Ad35-348lANdeI is then linearised with Ndel, dephosphorylated with CIP enzyme (New England Biolabs) and purified from gel using the Geneclean spin kit (BioIOl, Inc.) according to manufacturers instructions. This vector fragment is then ligated to a 2 6.6 kb Ndel fragment isolated from Ad35 wt DNA after which the mixture is used to package the cosmid using X-phage packaging extracts (Stratagene) according to manufacturers instructions. The resulting mixture is used to transform STBL-2 bacteria (Invitrogen), giving rise to PWE.Ad35~3481.<br>
Example 8. Construction of p!G35.55K.<br>
Construct pIG35.55K contains the coding sequences of the Ad35 E1B-55K gene operatively linked to the human phosphoglycerate kinase promoter (hPGK) and the HBV poly-adenylation sequence. In addition, it contains the neomycine resistence gene operatively linked to the RSV promoter and HBV pA. The construction of pIG35.55K is described below.<br>
Construct pIG270 (described in WO 00/70071) was digested with EcoRI, treated with Klenow enzyme and purified using a PCR purification kit (Qiagen) according to the manufacturers instructions. The recovered DNA was then digested with Agel and the ~5 kb vector fragment was isolated from gel using the Geneclean kit (BioIOl, Inc.) according to manufacturers instructions. Next, Ad35 E1B-55K sequences were amplified by PCR on pIG270 template DNA using the 35D21 and 35B3 primers. The PCR amplification was done with Pwo DNA polymerase (Roche) on 2 ng template DNA according to manufacturers instructions<br><br>
but with using DMSO at a final concentration of 3% in the PCR mixture. The program was set at:. 94°C for 2 minutes followed by 25 cycles of (94DC for 30 seconds, 56DC for 30 seconds and 72°C for 30 seconds) and ended by a final incubation of 72 °C for 10 minutes. The resulting PCR fragment was purified using the PCR purification kit (Qiagen) and digested with Ncol. Following Klenow treatment to fill in the protruding ends, the DNA was further digested with Agel and again column purified. The thus treated PCR fragment was then cloned into the above prepared EcoRI/Agel digested vector fragment to give pIG27 0.AElAA21K. pIG270.AElAA21K was digested with Avrll and Xbal and protruding ends were filled in using Klenow enzyme. The 2.9 kb fragment containing the PGK promoter and Ad35 E1B-55K sequences was isolated from gel as described above. Next, pRSVneo4 (construction described infra) was digested with Bglll, blunted with Klenow enzyme, dephosphorylated and isolated from gel. The blunted Avrll/Xbal fragment from pIG270-AE1AA21K was then ligated into the above prepared pRSVneo4 vector fragment to give pIG35.55K.<br>
pRSVneo4 was generated as follows: Construct pRSVhbvNeo (described in WO 00/70071) was digested with Seal and BamHI and protruding ends were filled in using Klenow enzyme. The 1070 bp fragment containing part of the Ampicilin gene and the RSV promoter was isolated from gel using the Geneclean kit (BIO 101, Inc.). Next, pRSVhbvNeo was digested with Seal and EcoRl, blunted with Klenow and the 3.2 kb fragment containing the neo gene, HBVpA, vector and part of the Ampicilin gene was isolated as above. The two fragments were then ligated to give pR3Vneo4.<br>
Example 9. Increased pIX expression mediated by the RSV promoter increases stability of Ad35 viruses.<br><br>
As an example of a heterologous promoter driving expression of the pIX gene, the RSV promoter was inserted into the Ad35 adapter plasmids containing the Lac2 or Luciferase reporter gene. The RSV promoter corresponds to an NruI/ApaLI fragment obtainable from pRc-RSV (Invitrogen) . Protruding ends were filled in using Klenow enzyme (New England Biolabs) according to manufacturers instructions. The 388 bp fragment containing the RSV promoter was isolated from agarose gel using the QIAquick Gel Extraction kit (Qiagen). Adapter plasmids pAdApt35.Luc and pAdApt35.LacZ were linearized with Bglll followed by Klenow treatment to blunt the ends. Bglll digests just behind the SV40 poly-adenylation sequence of the transgene expression cassette. For a description of pAdApt35-based adapter plasmids see WO 00/70071. The treated adapter plasmids were then dephosphorylated using Shrink Alkaline Phosphatase (SAP) according to manufacturers (Roche) instructions. The isolated RSV promoter fragment was then ligated with each of the treated vectors and transformed into DH5a competent bacteria (Invitrogen). Colonies were analyzed for forward oriented insertion of the RSV promoter relative to the pIX gene resulting in pAdApt35.Luc.rsv and pAdApt35.LacZ.rsv.<br>
In addition, an adapter plasmid was generated from sequences that were isolated by PCR from an Ad35 recombinant virus that resulted after deletion of the transgene region. Analysis of a crude lysate preparation resulting from a transfection of pAdApt35.LacZ and the pWE.Ad35.pIX-rlTR Ad35 backbone constructs and subsequent plaque purification showed that the virus had a deletion in the transgene region of approximately 2.8 kb. The 5' sequences from this virus were PCR amplified from isolated DNA using primers 35F1 and 35R4. The reaction was performed with Pwo polymerase (Roche)<br>
9V<br><br>
according to manufacturers instructions. Program settings were as follows: 94DC for 2' then 5 cycles of (94°C for 30 sec; 48°C for 30 sec; 72°C for 2.5 rain.) followed by 25 cycles of<br>
(94°C for 30  sec; 56°C for 30 sec; 72°C for 2.5 min.), and ended by 8 min. at 68°C.<br>
The resulting 2 kb fragment was purified by the PCR purification kit (Qiagen) and cloned into the pCR-Script-Amp vector (Stratagene) according to manufacturers instructions resulting in pCR.Ad35A2.8kb. This plasmid was sequenced to determine the extent of the deletion. The deletion affected most of the CMV promoter, the transgene and SV40 polyA. This resulted in linking of the 5' 317 bp of the CMV promoter to the Ad35 sequences upstream of the pIX gene. This CMV fragment contains three GC-boxes and a 21-bp repeat (Boshart et ai., 1985). Possibly, the remaining sequences of the CMV promoter could augment the plx expression resulting in a more stable virus. An alternative possibility was that the virus being smaller, on it self resulted in increased stability. To investigate this, a complete expression cassette was cloned back in the following manner, and viruses with this new-adapter plasmid were generated. The pCR-Script-based vector containing the amplified sequences (renamed in pCR.C4) had a unique Avrll site preceding the ACMV-pIX sequences. The vector was linearized with Avrll, blunted with Klenow enzyme and dephosphorylated using SAP enzyme (Roche) as described above. Adapter plasmids pAdApt535,LacZ (Example 5) and pAdApt.Luc (WO 00/70071) were digested with Avrll and Bgln and DNA was treated with Klenow to fill protruding ends. The fragments corresponding to LacZ and Luciferase expression cassettes<br>
(CMV-TG-pA) were isolated from gel as above and ligated with the Avrll linearized pCR.C4 vector. Transformation in competent cells as above and selection of colonies that had<br><br>
the cassettes in the forward orientation relative to the left ITR, resulted in pCR.C4.LacZ and pCR.C4.Luc.<br>
Ad35 viruses were generated as described in example 6 using the new adapter plasmids; pCR.C4.LacZ digested with Pad, pCR.C4.Luc digested with Apal and<br>
pAdApt35.Lac2.rsv and pAdApt35.Luc. rsv each digested with PI-PspI.<br>
The adapter plasmids were cotransfected onto PER55K cells (WO 02/40665) with pWE.Ad35.pIX-rITR or pWE.Ad35.pIX~rITRAE3 digested with Notl. In addition, adapter plasmid pBr.Ad35.AElAA2lK.Luc (construction described below) was digested with PI~PspI and cotransfected with pWE.Ad35.pIX-rlTR digested with Notl. Upon full cyto-pathogenic effect (CPE) cultures were harvested by one freeze/thaw cycle and centrifugated to remove cell debris. 300 ul of the resulting cleared lysates were then used to re-infect PER55K cells seeded the day before in T80 flasks. Upon full CPE, crude lysates were prepared and used to infect A549 cells to test for transgene expression and to perform a plaque assay on PER55K cells.<br>
A549 cells were seeded in 6-well plates at 5x105 cells/well and after 5hours infected with 10, 1 or 0.1 ul of each of the LacZ virus stocks and incubated for two days. A549 cells were then stained for LacZ activity and blue cells were counted. The percentage of blue cells is given in Table II.<br>
LacZ expressing viruses are clearly more stable when the RSV promoter is driving expression of the pIX gene as compared to the deleted CMV promoter. These results are confirmed with the Luciferase viruses. To measure activity of the Luciferase viruses A549 cells were seeded in 24-well plates at 1x105 cells/well and infected with 10, 1, 0.1, or 0,01 pi of the virus stocks and incubated. After two days, cells were washed<br>
1b<br><br>
with PBS twice and resuspended in 100 ul lysis buffer (Promega) and stored at -20DC until use. Luciferase was measured using the Steady-Glo luciferase assay system (Promega) according to manufacturers instructions. Results are presented in Table III.<br>
In example 3 we described that fully El-deleted Ad35 viruses containing the E3 region and an AdApt.Luc or AdApt.LacZ expression cassette were not stable. Apparently, in the new described constructs the deleted CMV promoter in front of pIX did not prevent the deletion of the transgene region. With the RSV promoter driving the pIX gene however, we now are able to generate viruses of more than wild type length. The Ad35.AdApt.Luc.rsv and Ad35.AdApt.LacZ.rsv are 35 kb and 36.5<br>
kb respectively (see also figure 2). Ad35.AE1AA21K.Luc (36.4 kb) also showed high transgene activity.<br>
We next tested whether viruses would be intact after a plaque purification. Hereto, PER55K cells were seeded in 6-well plates at 0.9x106 cells/well and infected with different 10-fold dilutions of the Ad35.AdApt.Lac2 crude lysates. Dilutions from 10-5 to 10-8 were plated and the next day an agar overlay was added. Hereto, cells were first washed with PBS and then 3 ml of a pre-warmed agar solution prepared by mixing 2xMEM (GibcoBRL; 9.14 ml), FBS (Gibco; 0.36 ml), MgCl (4,9 M; 0.037 ml) with agarose (SeaPlague GTG; 2.5% in H2o, 7.2 ml), was added. After solidification plates were further incubated at 37°C7 5% C02. Four days later plagues were visible and LacZ staining solution was added to the wells on top of the agar and allowed to drain. All viruses showed clear, separate plaques in the range of 10-7 to 10-9 but only in case of the viruses with the RSV promoter driving the pIX gene all plaques stained blue, in both cases where the deleted CMV promoter drives pXX at least part of the plaques did not<br>
11<br><br>
stain. This clearly shows that the packagable genome size/stability is increased in viruses that have the RSV promoter regulating plX.<br>
This invention for the first time provides a stable recombinant adenovirus derived from or based upon an adenovirus serotype 3 5 lacking expression of a functional E1B gene. Such an adenovirus has at least a deletion in the El-region. In particular embodiments provided by the invention, said stable recombinant adenoviruses have foreign insert sequences of more than 4.2 kb, and a packaged genome size of more than 33.4 kb, using methods according to the invention. It is therefore an object of the present invention to provide a stable recombinant adenovirus that: a) harbours a foreign nucleic acid sequence of more than 4.2 kb, and/or b) has a packaged genome size of more than 33.4 kb. In particular embodiments, the invention provides an Ad35 or Adll based recombinant adenovirus that: a) harbours a foreign nucleic acid sequence of at least 4.6 kb, and/or b) has a packaged genome size of more than 33.8 kb. Alternatively or in addition thereto, said packaged genome sizes are at least 34.6, 35.0, 36.1 and 36.5 kb, respectively. The foreign sequences in these embodiments may include a heterologous promoter driving expression of plX. In another aspect, said stable recombinant adenovirus is of serotype 11. A stable adenovirus according to this aspect of the invention can be passaged on a packaging cell to provide a batch of the recombinant adenoviruses, with less than 10%, preferably less than 5%, preferably none of separate clones giving rise to deletions in the foreign sequences in the recombinant adenovirus, as can be measured e.g. by  the PCR method exemplified in Example 3.<br>
-I*<br><br>
pBr.Ad35.AElAA21K.Luc (see above} was made as follows. Construct pBr.Ad35.AE1AA21K (WO 02/40665) was digested with Hpal, dephosphorylated with CIP (New England Biolabs) and the 5 kb vector fragment was isolated from gel. Construct pBr.Ad35.AElA.Luc was also digested with Hpal and the 3.3 kb insert was isolated from gel and ligated with the isolated vector fragment. Following transformation into competent STBL-2 cells (Invitrogen), a colony was   selected with the insert in the correct orientation. This gave construct pBr.Ad35.AElAA21K.Luc. pBr.Ad35.AElA.Luc (also called pBr.Ad35.E1B+.Luc, because it still contains the ElB region) was made by inserting the AdApt.Luc cassette, taken from pAdApt.Luc after Avrll and Bglll digestion and blunting with Klenow enzyme, into the vector fragment pBr.Ad35.leftlTR-pIX (WO 02/40665) digested with SnaBI and Hindlll  and blunted with Klenow. Colonies with the expression cassette in the forward orientation were selected, giving pBr.Ad35.AElA.Luc.<br>
Example 10. Identification of pIX regulatory sequences.<br>
The previous examples show that Ad35 recombinant viruses in which the coding regions for ElA and ElB are completely removed become progressively more unstable if the genome size is increased. We show in this application that addition of a heterologous promoter driving pIX expression can overcome the instability. In WO 02/40665 and supra we disclosed that Ad35 viruses that retain the complete E1B-55K coding sequence  can be produced on PER.C6 and are stable. The same is true for viruses that retain the full ElB coding sequence (WO 00/70071; Abrahamsen et al., 1997). Together these results raise the possibility that expression of the pIX gene is regulated differently in subgroup B viruses as compared to the pIX gene<br><br>
in subgroup C. Since the viruses that retain the E1B-55K gene driven by the E1B promoter are stable (see above), pIX regulatory sequences will probably be located in this region. To investigate this we generate a series of constructs that retain different lengths of i:he 3' end of the 55K sequence. Hereto, pBr.Ad35ASM.AdAptLacS is first generated as follows. Construct pBr.Ad35.lITR-pIX {described in WO 00/70071) is digested with SnaBI and Mfel, blunted with Klenow and de-phosphorylated with SAP enzyme (Roche). The 4.4 kb vector fragment is then isolated from agarose gel. Construct pAdApt.LacZ (An Ad5-based adapter plasmid pAdApt with LacZ transgene insert; WO 99/55132) is digested with Avrll and Bglll {and/ optionally/ Sail to increase the difference in fragment size) and blunted with Klenow enzyme. The 4.2 kb CMV.LacZ.pA insert is then isolated from gel. Both isolated fragments are then ligated to give pBr.Ad35ASM.AdAptLac2 (Figure7). The orientation can be checked by restriction digestion since ligation in the correct orientation restores both the Avrll site and the Mfel site. Construct pBr.Ad35ASM.AdAptLacZ retains the 0.6 kb 3' E1B-55K sequences (wt Ad35 nucl 2804-3400) in the wt position relative to the pIX gene. Previously, we have shown that these 55K sequences do not lead to expression of a functional E1B-55K protein since propagation on PER.C6 cells appeared not possible (pBr.Ad35ASM; WO 02/40665). Starting from<br>
pBr.Ad35ASM.AdAptLacZ different deletions can be made of the 680 bp (0.7 kb) E1B-55K region by digestion with Mfel (isoschizomer of Muni) and either StuI, Nsil or Bglll followed by blunting of the protruding"ends using Klenow or T4 DNA polymerase in case of 5' or 3' overhang respectively, Religation of the digested DNA gives functional adapter<br>
9*<br><br>
plasmids that are then used to generate recombinant viruses on PER55K cells by co-transfection with pWE.Ad35.pIX-rITR as described above. Additional constructs are made by using enzymes Drain, Bsu36I, BssHII or BamHI, and digest the vector partially (the LacZ gene also contains a recognition site for these enzymes) using methods known in the art,- followed by selection of the correct clone. The stability is tested as described above for the Ad35.AdApt.LacZ.rsv construct. Constructs that are stable (i.e. do not acquire deletions in the transgene region) contain proper regulatory regions for pIX expression. In addition, it is possible to directly test promoter activity in a given sequence by inserting the sequence upstream of a reporter gene. pGL3basic (Promega) is such a reporter gene construct.  The region between Muni and the start of the pIX gene was amplified using primer set Ad3555KmfeF and Ad35pIXNcoR. This PCR (2 minutes 94°C; then 30 cycles of [30 seconds 94   °C, 30 seconds 59°C, 60   seconds 72°C]; followed by 8 minutes 68°C; enzyme: Pwo (Genexis) according to manufacturer's instructions, with additional 3% DMSO)   amplified Ad35 sequences from 2804 to 3491 (numbering as in wt Ad35) thereby changing the sequence around the start codon of pIX into an Ncol site and introducing an Hindlll site at the 5' end. This amplified fragment is digested with Hindlll and Ncol and cloned into pGL3basic digested with the same enzymes generating pGL3-MN. pGL3-MN is than used to delete sequences upstream of the Luciferase coding region by combining Hindlll digestion with e.g. Pad, Nsil, Stul, Bsu36I, BssHII or Bglll/ followed by blunting of the protruding ends and religation. Promoter activity is tested by transient transfection of the obtained constructs into PER.C6 cells using lipofectamine reagent according to manufacturers instructions. Luciferase activity is analysed two days after<br>
91<br><br>
transfection using Steady-Glo luciferase assay system (Promega) according to manufacturers instructions. Alternatively, into the pGL3basic vector different regions are inserted, which regions are generated by PCR amplification using a 5'(forward) primer directed to a specific sequence in the Ad35 E1B-55K region and having a Hindlll site attached at the 5' end combined with the Nco-pIXrev primer. In this way one is not limited to the presence of a unique restriction site for cloning.<br>
The location of the pIX promoter was further investigated by using software to find putative promoter sequences (Reese and Eeckman, 1995). Figure 8 shows the promoter scores (minimum set at 0.65}   for the E1B promoter directly linked to the 55K coding sequence (as in pBr.Ad35.AE1AA21K). The regions marked A corresponds to the E1B promoter and regions B and C locate within the 55K coding region. The pIX upstream region is not recognized as a promoter sequence. Region C has the highest score (0.96) of the three (even higher than the known E1B promoter) and may therefore comprise sequences that influence pIX expression.<br>
To locate and identify a possible pIX promoter experimentally, a series of small fragments corresponding to different (overlapping) parts of the 3'-end of the 55K coding sequence are generated. Figure 9 schematically depicts these fragments and their location relative to the putative promoter region and the pIX gene. The fragments are generated by restriction digestion using the indicated en2ymes. Fragments are blunted with Klenow (5' protruding ends) or T4 DNA polymerase (3' protruding ends) and cloned into the Ncol site of the pGL3basic vector (Promega) also blunted by Klenow treatment and de-phosphorylated by SAP treatment (Roche). Following transformation into competent bacteria obtained plasmids are<br><br>
checked for the orientation of the insert by restriction digestion. Promoter activity is then analysed by transient transfection of the obtained luciferase constructs into PER.C6 cells using lipofectamine reagent according to manufacturers instructions. Empty pGL3basic plasmid serves as a negative control. Additional controls are made by cloning i) a Bglll-Mfel fragment from pAdApt535 containing the Ad5 pIX promoter, ii) a 388 bp NruI-ApaLI RSV promoter fragment {described above), or iii) the Ad35 pIX upstream region as a PCR fragment into the blunted Ncol site of pGL3basic as  described above. The Ad35 upstream pIX region is amplified on pAdApt35lPl using primers SV40-for and V-phosphorylated Ad35pIXrev. Following amplification the DNA is digested with Bglll and treated with Klenow.<br>
Constructs are also transfected into human cells not containing adenovirus El (e.g. A549,   Hela) to investigate the dependency on E1A expression.<br>
The fragments are also cloned into an adapter plasmid pBr.Ad35ASM.AdAptLacZ (see above) to be able to generate recombinant viruses and study viral genome stability. Hereto, construct pBr.Ad35AsM.AdAptLacZ is digested with Mfel and Bglll, blunted with Klenow enzyme and dephosphorylated. After gel isolation of the vector fragment, DNA can be ligated with the fragments described above (see Fig. 9) to give rise to a set of adapter plasmids that have varying lengths of 55K fragments upstream of the pIX gene. Viruses can be generated with the construct pWE.Ad35.pIX-rlTR as described above. Control transfections are done with the pBr.Ad35ASM.AdAptLacZ, pAdApt35.LacZ and pAdApt35.LacZ.rsv constructs. Upon appearance of full CPE, cells and medium are harvested by one freeze/thaw cycle and used to re-infect fresh PER55K cells.<br><br>
Cells and medium are again harvested at full CPE, crude lysates are prepared and used to perform a plaque assay. After appearance of plaques, X-gal staining solution is added to check for LacZ expression.<br>
The results of the above experiments aid in finding the position of pIX regulatory sequences in the Ad35 E1B-55K region.<br>
As yet another alternative, pIX gene expression may be driven by the E1B promoter as a heterologous promoter for the generation of recombinant viruses. Hereto, pAdApt535.Lac2 is digested with Bglll and Mfel followed by Klenow treatment to blunt ends and dephosphorylation. The thus treated 4.8 kb vector fragment is then isolated from gel. The E1B promoter region is isolated as a PCR fragment using pBr.Ad35.leftlTR-pIX as   target DNA and the Epr-F and Epr-R primers, whereby both primers are phosphorylated. PCR is done with Pwo DNA polymerase (Genaxis/Inno-train Diagnostic Gmbh) according to manufacturers instructions. The resulting 151 bp fragment is then cloned into the isolated vector to give pAdApt35Epr. LacZ<br>
This plasmid is then used to generate Ad35-based viruses and test stability as described before.<br>
To identify the different transcripts that contain pIX sequences, RNA is isolated from infected cells and pIX containing RNAs are identified by hybridisation with a labelled specific probe. Hereto, PER55K cells are infected at a multiplicity of infection of 10 and 50 with the following viruses: wtAd35, Ad35.E1B.AdApt.Luc, Ad35AE3.AdApt.Luc, Ad35AE3.AdApt535.Luc, Ad35.AdApt.Luc.rsv. Infected cells are harvested after 8 hrs {wtAd35 also after 2 and 18 hrs) and RNA is isolated using TRI-zol Reagent (Invitrogen). This RNA is<br><br>
size fractionated on an 1.5% agarose gel, transferred to a Northern blot and  hybridised to a 32P-labeled probe derived from the plX coding region. Procedures are known in the art (described in Molecular Cloning: A laboratory manual, by Sambrook and Russell, 2001 or earlier versions). The length of the RNA can be determined if known RNA size markers are included and will give an indication of the RNA species that contain plX sequences. To   identify the mRNAs that start in the E1B promoter, the blot can be stripped and re-hybridised with a 5' 21K probe. pIX-containing transcripts that do not hybridise to the E1B-21K probe are likely generated by a promoter different from the E1B promoter.<br>
Due to possible (and expected) splicing events it is still difficult to precisely determine transcription start sites via this method. This can be achieved as follows. The isolated RNA is reverse transcribed into cDNA and the cDNA is used to specifically amplify 5' ends of plX containing RNAs using the GeneRacer System (Invitrogen) according to manufacturers instructions with the reverse primers directed to plX coding sequences: pIXrev and the nested primer pIXrev-N2. Cloning and sequencing of the amplified fragments gives the location of the transcription start sites and 5' sequences of mRNAs that contain plX sequences. In this way the possible plX coding mRNAs are identified. The correlation between the levels of plX expression and stability of the corresponding recombinant adenoviruses can thus also be determined.<br>
Example 11, E1B and plX sequences from group B adenoviruses. Above examples use the Ad35 virus as an example. Other members of the subgroup B that have considerable homology to each other are could have comparable plX regulation.<br><br>
To investigate this, we aligned ElB and pIX sequences of subgroup B members. The sequence of Ad7 (SEQ. ID, NO. 57) is available via GenbanJc Accession number X03000. The sequence of Adll {SEQ. ID. NO. 56) was revealed by shotgun sequencing of DNA isolated from Adllp wt viruses performed by Lark Technologies (UK) similar as described (WO 00/70071) for the Ad35 sequence (SEQ. ID. NO. 55). Adll and Ad35 are highly homologous to each other (overall 98.1% similarity), and the main differences are located in hexon and fiber knob. The Adll sequence is also disclosed in WO 02/053759. The sequence between the poly-adenylation site (pA) of ElA and the pA downstream of the pIX gene is used in the alignment (Fig. 10) . Ad35 has an overall similarity (in this region) of 98.4 % to Adll and 82.9 % to Ad7. This makes it very likely that pIX expression is regulated in the same way in these viruses.<br>
Hence, the methods and means according to the invention as exemplified in the previous examples can be used accordingly to increase the stability and/or insert capacity of other recombinant adenoviruses of subgroup B, herein exemplified by Adll and Ad7.<br>
Example 12. Generation of El-deleted Ad35 viruses expressing Ad5-E4/Orf6 on an Ad5 complementing cell line.<br>
The sequencing of the Adenovirus serotype 35 genome as well as the construction of a plasmid-based vector system and generation of recombinant Ad35-based viruses have been described in detail in WO 00/70071.<br>
The cloning of the Ad5-E4-orf6 coding sequence into pAdApt35IPl (ECACC deposit no. P02041228, for cloning details of this plasmid see WO 00/70071) was performed as follows. The plasmid was digested with Nhel and Avrll and dephosphorylated<br>
9*<br><br>
with Calf Intestine Phosphatase (New England Biolabs). Digested DNA was isolated from gel using the GeneClean kit. Plasmid pAMT.Orf6.Hygro {Fig. 11, ECACC deposit no. P02041226) was digested with Nhel and subsequently partially digested with Xbal. After separation of the resulting bands on gel, the 1350 bp fragment corresponding to the AMT promoter linked to the E4-orf6 sequence was purified from gel. Next, both isolated fragments were ligated and transformed into electro-competent DH10B cells Invitrogen/LifeTechnologies) after which a colony with the insert in the correct orientation with respect to the SV40 poly(A) signal was selected for large scale DNA preparation. This resulted in construct pAd35.AMT.Orf6 (Fig. 12), which contains the Ad5 E4-orf6 coding sequence functionally linked to a mutated metallothionein promoter {AMT). The AMT promoter has been described by Hagmeyer et al. (1996) . The Ad5 E4-orf6 sequence corresponds to nucleotide 33193 to nucleotide 34077 in the Ad5 sequence (Genbank accession number M7326Q). To test whether the expression of Ad5 E4-orf6 proteins would make production of fully El-deleted Ad35 vectors possible on Ad5 complementing cells, pAd35.AMT.0rf6 was co-transfected with the Ad35 backbone construct pWE.Ad35.pIX-rlTR onto PER.C6 cells. Hereto, pAd35.AMT.Orf6 was digested with Pt-Psp-1 and pWE.Ad35.pIX-riTR was digested with NotI to liberate the adenoviral inserts from the backbone. 2 ug of digested pAd35.AMT.Orf6 and 6 ug of digested pWE.Ad35.pIX-rlTR were transfected using LipofectAmine. The transfection mixture was added to PER.C6 cells that were seeded the day before at a density of 3.5xl06 cells per T25 flask. The next day the medium was changed for PER.C6 culture medium {DMEM with 10% FBS and 10 mM MgCl2) and cells were further incubated at 37°C/10% C02. Control transfections were performed with pAdApt35.Luc co-<br><br>
transfected with pWE.Ad35.pIX-rITR, and pWE.Ad35.pIX-rITR alone. Two days after transfection, cells were passed from T25 to T80 flasks and incubated as described. Again three days later the culture transfected with pAd35.AMT.Orf6 together with the Ad35 backbone showed cytopathogenic effect (CPE) indicative of virus replication, and was harvested {including cells and medium) after a further incubation of 2 days. The cell suspension was subjected to 2 rounds of freeze/thaw cycles and the resulting material (crude lysate) was kept at -20°C until further use. The other flasks did not show CPE and were passed 1:3 in T80 flasks 6 days after transfer to T80. Again 5 days later the pAdApt35.Luc + pWE.Ad35.pIX-rlTR transfected flask showed a few CPE-like events but this did not progress further. 0.2 and 0.5 ml of the crude lysate resulting from the pAd35.AMT.Orf6 transfection was used to re¬infect PER.C6 cells at approximately 85% confluency in T80 flasks. This resulted in full CPE after one day of incubation indicating that infectious virus was present in the crude lysates. These cultures were also harvested by two freeze/thaw cycles. Additional control transfeetions with construct pAd35.AMT.Orf6 alone onto PER.C6 were performed to confirm that orf6 expression by itself did not result in cell toxicity and CPE-like cell death. In conclusion, only the transfections with pAd35.AMT.Orf6 together with pWE.Ad35.pIX-rlTR did result in CPE and virus replication.<br>
PCR analysis was performed to confirm the presence of Ad35-based viral genomes with Ad5-E4-orf6 replacing the former El region. Hereto, viral DNA was isolated from the crude lysate samples as follows. 275 ul of crude lysate material was incubated with 10 ul DNasel (10 mg/ml) at 37°C for 30 min. Subsequently, 6.0 ul 0.5 M EDTA (pH 8.0), 7.5 \il   20% SDS and 1.5 ul 20 mg/ml Proteinase K was added and mixed by vortexing.<br><br>
The mixture was then incubated at 50°C for 1 h. Finally, the viral DNA was isolated using the GeneClean Spin Kit (Bio 101, Inc.). 2 ul of the isolated DNA was then PCR amplified using primers 35psi-For and 35R4. The program was set at 94QC for 2 min followed by 30 cycles of 94°C for 30 sec, 58°C for 30 sec and 72°C for 5 min, and ended by an incubation at 72°C for 10 min. The primers are specific for Ad35 sequences and generate a fragment of 2.9 kb ranging from the packaging sequence to nt 4669 (numbering as in wt Ad35 sequence) thus including the Ad5 orf6 tranagene cassette. Electrophoresis of the obtained PCR fragments showed that the fragments had the expected length matching with the control PCR fragments generated on the adapter plasmid pAd35.AMT.Orf6. Thus, fully El-deleted Ad35-based vectors can be made on Ad5 complementing cells if the virus also expresses Ad5-E4orf6.<br>
Example 13. Construction of pWE.Ad35,pIX-rITR5E4.<br>
A first PCR fragment was amplified using primers DF35-1 and 35FR. Amplification was done with pWE.Ad35.pIX-rlTR (see WO 00/70071) as template DNA using Pwo DNA polymerase (Roche) with additional DMSO (Sigma, final concentration 3%). The program was as follows: 94°C for 2 min followed by 30 cycles of (94°C for 30 sec, 52°C for 30 sec, 72°C for 3 min) and a final step of 72°C for 8 min to ensure complete fragments. Amplification resulted in a 1.6 kb fragment corresponding to nt 30224 to 31805 of the Ad35 sequence. A BamHI site was introduced at the 3' end. The amplified DNA was purified from gel using the GeneClean kit and ligated to the pCRScript/Amp cloning vector kit (Stratagene). Following transformation into electro-competent DH10B cells white colonies were selected for further analysis. This resulted in construct pCR-fiber35. Due to the blunt cloning the PCR fragment could be inserted in 2<br><br>
orientations. A clone that had the insert with the BamHI site in the polylinker of the pCRScript/Amp vector at the 5' end was selected. Digestion with BamHI thus results in a 1.6 kb fragment. Sequencing confirmed correct amplification of the PCR fragment. A second PCR fragment was amplified using primers: 5E4F and 5E4R. Amplification was done with pWE.Ad5.Af111-rITRsp, which is a cosmid vector containing an extra Pad site in pWE.Ad5.Aflll-rlTR (ECACC deposit no. P97082116 described in WO 02/40665). pWE.Ad5.Aflll-rlTRsp served as a template using Pwo DNA polymerase as described above, although pWE.Ad5.Aflll-rlTR could also be used for the same purpose. After purification from gel, the DNA was digested with SstI and BamHI (both sites introduced during the PCR) and the 3 kb fragment was purified from agarose gel using the GeneClean kit. The Ad5 E4 region that is amplified corresponds to bp 32794 to bp 35828 of the Ad5 sequence. A third PCR fragment was generated on pWE.Ad35.pIX-rlTR using primers: 355ITR and 353ITR. PCR amplification was performed as described above. The resulting 160 bp fragment is flanked by an SstI site (5' end) and an EcoRI site (3' end). After purification from gel as above, the DNA was digested with SstI and EcoRI. The 160 bp fragment corresponding to the right ITR of Ad35 was then separated from digested ends on a .low melting point agarose gel and collected in gel. Next, pUC119 was digested with BamHI and EcoRI and the 3.1 kb fragment was purified from gel using the GeneClean kit. The above treated second and third PCR fragments were then ligated with BamHI/EcoRI digested pUC119 resulting in pUC.Ad5E4-35ITR. The cloned PCR derived inserts were sequenced to verify correct amplification. Next, the 1.6 kb insert in pCR-fiber35 was excised with BamHI and the fragment was purified from gel as above. pUC.Ad5E4-35ITR was also digested with BamHI and the<br><br>
linear fragment was purified from gel. Ligation of both fragments and selection of the clones that had the correct orientation relative to each other resulted in pUC.35-5E4 (Fig. 14). The steps leading to the construction of pUC.35-5E4 are schematically represented in fig. 13. The adenovirus insert in pUC.35-5E4 was sub-cloned into pBr.Ad35.PRn (Fig. 15; see WO 00/70071), a construct with Ad35 3' sequences. Hereto, construct pOC.35-5E4 is digested with Mlul and NotI and the 4.7 kb fragment is purified from gel using the GeneClean kit. This fragment is then ligated with the vector fragment resulting from Mlul and NotI digestion of construct pBr.Ad35.PRn. This 16.3 kb fragment was purified from gel using agarase enzyme (Roche). Ligations were then transformed into competent DH10B cells. The resulting construct was named pBr.Ad35.PR5E4 (Fig. 16, ECACC deposit no. P02041229). The last step entails cloning of the modified 3' end of the Ad35 sequence into the viral cosmid clone pWE.Ad35.pIX-rlTR. Hereto, two fragments are combined in a lambda phage packaging reaction (Stratagene) according to manufacturer's instructions. The first is the 16.8 kb modified Ad35 insert from pBr.Ad35.PR5E4 obtained by digestion with Pad and Swal and the second is a 22.8 kb fragment obtained by digestion of pWE.Ad35.pIX~rITR with Pad and Swal. The correct combination of the two fragments yields pWE.Ad35.pIX-rITR5E4 (Fig. 17). Thus, in this construct the E4 region in the Ad35 backbone is replaced with the corresponding region derived from Ad5.<br>
Example 14. Construction of pWE.Ad35.pIX-rITR50rf6.<br>
To obtain an adenoviral backbone construct that contains the Ad35 sequences from the pIX gene (nt 3401 in the Ad35 sequence) to the end of the right ITR but with the sequences for E4-orf6 and -orf6/7 exchanged for the corresponding<br><br>
sequences of Ad5, Ad35 and Ad5 sequences were PCR amplified and combined as described below. PCR fragments were generated with Pwo DNA polymerase with addition of DMSO up to 3%. The first PCR was done with pBr.Ad35.PRn (Fig. 15; see WO 00/70071) as template and the primers E4-F1 and E4-R2, The program was set as follows: 94°C for 2 min, 5 cycles of {94°C for 30 sec, 50°C for 30 sec and 72°C for 1 min) followed by 30 cycles of (94°C for 30 sec, 60°C for 30 sec and 72DC for 1 min) and ended with a final step at 68°C for 8  min. The resulting 1.8 kb fragment was purified using the GeneClean kit. The second PCR was done with pWE.Ad5.Aflll-rlTRsp, which is a cosmid vector containing a Pad site in pWE.Ad5.Aflll-rlTR (ECACC deposit no. P97082116, described in WO 02/40665), as template and the primers E4-F3 and E4-R4. The program was set as follows: 94°C for 2 min followed by 30 cycles of (94°C for 30 sec, 62°C for 30 sec and 72°C for 1 min) and ended with a final step at 68°C for 8 min. The 1.1 kb fragment was purified as above. The third PCR was done with pBr.Ad35.PRn as template and the primers E4-F5 and E4-R6. The program was set as follows: 94°C for 2 min, 5 cycles of (94°C for 30 sec, 48°C for 30 sec and 72°C for 45 sec) followed by 30 cycles of (94°C for 30 sec, 56°C for 30 sec and 72°C for 45 sec) and ended with a final step at 68°C for 8 min. The 366 bp fragment was purified as above.   Samples  of the purified fragments were loaded on a gel to estimate the concentration and then the fragments were mixed together to contain 700 ng PCR-1, 650 ng PCR-2 and 430 ng PCR-3 in a total of 30 pi. To this mixture 3 ]xl   EcoPol buffer (New England Eiolabs), 3 ul 2 mM dNTP solution and 3 ul milliQ H2O was added. The resulting mixture was incubated at 94°C for 3 min and then cooled down to 65°C in a PCR machine at a rate of 0.5°C/sec. Following incubation at 65°C for 10 min, the mixture was further cooled down to<br><br>
20°C at a rate of 0.05°C per sec and incubated for 10 min at 20°C. Then 1 u.1 (5 units) Klenow enzyme {New England Biolabs) was added followed by an incubation of 60 rain at 37°C. 5 u.1 of this Klenow mixture was used as a template to separately amplify two fragments as follows. Primer set 1: NF-1 and Ncol-R was used in a reaction using Pwo DNA polymerase (Roche) with addition of DMSO to a final concentration of 3% and using the following settings of the PCR machine: 94°C for 2 min followed by 30 cycles of (94°C   for 30 sec, 66°C for 30 sec and 72°C for 3 min) followed by a final incubation at 68°C for 8 min. Primer set 2: Ncol-F and NR-2 was used in a reaction using Pwo DNA polymerase (Roche) with addition of DMSO to a final concentration of 3% and using the following settings of the PCR machine: 94°C for 2 min followed by 30 cycles of (94°C for 30 sec, 62°C   for  30   sec  and 72°C for 90   sec)   followed  by a final incubation at 68°C for 8 min. The resulting fragments of 2,7 kb (primer set 1) and 1.1 kb (primer set 2) were purified from gel using the GeneClean kit and each was ligated to the pCRscriptAmp vector (Stratagene) and transformed into DJilOB electrocompetent cells. This resulted in construct pCRscriptAmp.NFI-NcoIR (Fig. 18) and construct pCRscriptAmp.NcoIF-NR2 (Fig. 19}. Since the inserts contained blunt ends two orientations were obtained of each cloning, Using Kpnl digestions the constructs with the orientation needed for further cloning were selected (see Figs. 18 and 19), The inserts were then sequenced to verify correct amplification. Next, part of the insert from pCRscriptAmp-NcoIF-NR2 was excised using BamHI and Ncol and purified from gel as above. pCRscriptAmp-NFI-NcoIR was digested with the same enzymes and the vector containing fragment was also purified from gel. Ligation of these fragments resulted in PCR.NF1-NR2 (Fig. 20) . pCR.NFl-NR2 contains Ad35 sequences<br><br>
between nt 30162 and 33234 of the Ad35 sequence with E4-orf6 and E4-orf6/7 sequences between nt 31879 and 32974 replaced for Ad5 derived sequences located between 32968 and 34077 from the published Ad5 sequence in Genbank (Accession Number M73260). Thus, as can be seen in the amino acid alignments presented in Figure 21 and 22, the amino acid sequence of the cloned E4~orf6 protein is identical to the E4-orf6 sequence found in Ad5 (SEQ. ID. NO. 61; amino acid sequence of E4-orf6 of Ad35 is SEQ. ID. NO. 62) and the E4-orf6/7 amino acid sequence is for the greater part identical to the E4-orf6/7 sequence present in Ad5 (E4-orf6/7 sequence is given for Ad 5 as SEQ. ID. NO. 63, for Ad35 as SEQ. ID. NO. 64, for the cloned fusion protein as SEQ. ID. NO. 65). Obviously, different hybrid Ad35-Ad5 E4 constructs can be designed using the general method outlined above without departing from the invention. This chimeric insert from pCR.NFl-NR2 was then cloned into pWE.Ad35.plx-rITR: pCR.NFl-NR-2 was digested with Mlul and Ndel and the resulting 2.8 kb fragment was purified from gel using the GeneClean kit. Construct pBr.Ad35.PRn was also digested with Mlul and Ndel and the 18 kb vector fragment was isolated from gel using agarase enzym (Roche). Ligation of both fragments resulted in construct pBr.Ad35.PR.50rf6 (Fig. 23, ECACC deposit no. P02041227). The Ad35 sequences between Pad and Swal containing the chimeric E4 region in this construct are then cloned into construct pWE.Ad35.pIX-rITR using lambda-phage packaging as described above. The resulting pWE.Ad35pIX-rITR.50rf6 (Fig. 24) is then used to generate recombinant Ad35-based viruses by co~transfection on PER.C6 packaging cells with an Ad35 adapter plasmid.<br>
Example 15. Construction of pWE.Ad35.pIX-rITRAE3.5E4 and pWE.Ad35.pIX-rITKAE350rf6.<br><br>
The Ad35 backbone was further modified by a deletion of E3 sequences. E3 proteins are known to modulate the host immune response to adenovirus infection and are therefore not necessary for in   vitro  propagation of recombinant viruses. Furthermore, the deletion of E3 sequences allows for insertion of larger heterologous sequences in the vectors without compromising the packaging efficiency. Also, for the application of adenoviral vectors as vaccination vehicles, expression of immunomodulatory genes encoded by the E3 region is not preferred.<br>
The construction of pWE.Ad35.pIX-rITRAE3 (Fig. 1) was described supra. To construct the E3-deleted versions of the E4-modified backbone constructs described above, the E4 modifications were introduced into the pBr.Ad35.PRnAE3 (Fig. 25) construct as follows. Construct pUC.35-5E4 (Fig. 13) was digested with Mlul and NotI and the 4.7 kb fragment was isolated from gel using the GeneClean II kit. Construct pBr.Ad35.PRnAE3 was also digested with Mlul and NotI and the 13.6 kb vector fragment was isolated from gel using the GeneClean spin kit. Ligation of these fragments resulted in construct pBr.Ad35.AE3.PR5E4 (Fig. 26). Construct pCR.NFl-NR2 (Fig. 20) was digested with Mlul, Ndel and Bgll (the latter to digest the vector fragment into smaller fragments), and the 2.8 kb fragment was isolated from gel using the GeneClean spin kit.   Construct pBr.Ad35.PRnAE3 was digested with Mlul and Ndel, dephosphorylated using CIP enzym (New England Biolabs) and the 15.2 kb vector fragment was also isolated using the GeneClean spin kit. Ligation of these fragments gave construct pBr.Ad35.AE3.PR50rf6 (Fig. 27).<br>
pBr.Ad35.AE3.PR5E4 -and pBr-Ad35 . AE3 . PRSOrf 6 are then used to swap the 3' Pacl-Swal fragment in pWE.Ad35.pIX-rlTR for the corresponding regions from pBr.Ad35.AE3.PR5E4 and<br>
1*<br><br>
pBr.Ad35.AE3.PR50rf6 as described intra. This leads to constructs pWE.Ad35.pIX-rITRAE3.5E4 and pWE.Ad35.pIX-rITRAE3.50rf6. An alternative method to generate these large cosmids is to use three fragments in the ligation reaction for packaging: a 14.7 kta Notl-PacI fragment from pWE.Ad35.pIX-rlTR, the PacI-NotI insert from pBr.Ad35.AE3.PR5E4 or pBr.Ad35.AE3.PR50rf6 and the NotI digested pWE15 cosmid vector fragment (Stratagene). This latter fragment can also be isolated from the Notl/PacI digestion of pWE.Ad35.pIX-rITR. Co-transfection of e.g. the Notl-digested pWE.Ad35.pIX-rITRAE3.50rf6 with e.g. PI-PspI digested pAdApt35.LacZ.rsv (example 9) into PER.C6™ cells will generate recombinant adenovirus derived from Ad35, which recombinant adenovirus comprises E4-orf6 derived from Ad5 (conferring the ability of propagation on PER.C6), said recombinant adenovirus furthermore having a heterologous pIX promoter resulting in increased pIX expression levels and stable virions.<br>
Example 16. Generation of El- and El/E3-deleted Ad35-based vectors on PER.C6 cells.<br>
To enable generation of recombinant Ad35 viruses on the complementing cell line PER.C6 using the pBr.Ad35.PRn-based constructs, we first made a new construct containing Ad35 sequences from bp 3401 to bp 24650 of the Ad35 sequence (WO 00/70071) and thus overlaps with both the adapter plasmids and the pBr.Ad35,PRn-based constructs. Transfection of these three plasmids into PER.C6 cells and a double homologous recombination event leads to a complete viral genome and replication of recombinant viruses as outlined in Figure 18. The required plasmid was made by deletion of a large part of the Ad35 sequences in pWE.Ad35.pIX-rITR. Hereto, pWE.Ad35.pIX-rlTR was digested with EcoRV and the 29 kb vector containing<br><br>
fragment was purified from a low melting point gel using the Geneclean spin kit. The purified DNA was self-ligated and used to transform DH10B electro-competent bacteria (Invitrogen/LTI) resulting in pWE.Ad35.pIX-EcoRV {Fig. 29).<br>
All DNAs used for transfection were digested as indicated in Table V, heat-inactivated at 65CC for 15 min and used without further treatment in the transfection. PER.C6 cells were ■ seeded the day prior to transfection in T25 flasks at a density of 3xl06 cells/flask and transfected as indicated in Table V using LipofectAmine {Invitrogen/LTI&gt; according to manufacturers instructions, except that the transfection mixture in serum-free DMEM medium (Gibco/BRL) was replaced for PER.C6 culture medium {DMEM, 10%  FBS and  10 mM MgCl2) after 5 h. The day after, transfection efficiency was estimated at 50% by fluorescence microscopy. Two days later, cells were trypsinized and reseeded in T80 flasks and further incubated at 37°C/10%CO2, Six days following transfection all cultures showed full cytopathogenic effect (CPE, indicative for virus propagation) except for the PER.C6 culture transfected with Ad35.AdApt.eGFP + pWE.Ad35.pIX-rlTR. One day later, cells and medium in the flasks with CPE were harvested and subjected to two freeze/thaw cycles, clarified from cell debris by centrifugation (10 min at 1500 rpm) and 100 ul of these crude lysates were used to re-infect fresh PER.C6 cells at 85% confluency in T80 flasks. The transfection of Ad35.AdApt.eGFP + pWE.Ad35.pIX-rITR that did not show signs of CPE was harvested by trypsini3ation and also treated as above. Two days following infection of fresh PER.C6 cells all flasks showed full CPE except for the one that showed no signs of CPE at the time of initial harvesting. This clearly shows that fully El-deleted Ad35-based viruses can be made on PER.C6<br><br>
cells when the Ad5 E4-orf6 gene product is expressed from the Ad35 backbone.<br>
Example 17. El-deleted Ad35 viruses with a heterologous promoter driving pIX expression.<br>
To investigate the effect of a heterologous promoter sequence activating the pIX gene in full El-deleted viruses, a series of adapter plasmids was used to generate recombinant Ad35 viruses. Hereto, pAdApt35LacZ, pAdApt35.LacZ.rsv (example 9), pAdApt535.Lac2 {example 5) and pAdApt35BLacZ (containing the Ad35 ElB promoter sequence in front of the pIX gene; described below) were digested with pIPsp-1 and used to generate viruses with NotI digested cosmid pWE/Ad35-3481 and pWE/Ad35-348lAE3 (example 7) as described in example 2 (and in WO 00/70071). In addition, viruses were generated with adapter plasmid pBr.Ad35ASM.AdAptLacZ (Figure 7; example 10). This adapter plasmid is deleted for E1A and a large part of the ElB sequences. It retains 0,6 kb of the 3' E1B-55K sequence and also has wt sequences between the stop codon of 55K and the start codon of pIX.<br>
Upon full CPE, the cells and medium were harvested, freeze/thawed and centrifuged to remove the cell debris. The supernatant (cleared lysates) of each of the transfections was then used to perform a plaque assay as described in example 9. Cleared lysates were diluted serially tenfold and 10~5 to 10"9 dilutions were plated.<br>
One week after addition of the agar overlay, plaques became visible and were stained with X-gal to monitor LacZ activity. Table VI summarizes the results of these experiments. All Ad35 viruses having additional or other sequences than just the endogenous proximal pIX upstream sequence regulating pIX perform better and have a higher number of expressing plaques<br><br>
in our assay as compared to the El-deleted Ad35.AdApt.LacZ viruses. Note that the total genome length of the Ad35ASM.LacZ viruses (106% compared to wt Ad35) exceeds the maximum packable length as determined for Ad5 viruses (105%). This may influence the results obtained for this virus. The Ad35.AdApt.LacZ.rsv (105%) are also at the border of the theoretical packagable size. Altogether the results show that a heterologous promoter driving pix expression improves the maximum tolerated packaging size and the stability stability of El-deleted Ad35 viruses. The same is true for viruses that have a longer endogenous proximal sequence (Ad35ASM.LacZ) suggesting that the additional (ElB 55K) sequences herein contain regulatory elements for pIX expression.<br>
pAdapt35BLacZ is an Ad35 adapter plasmid with the Ad35 ElB promoter sequence regulating the pIX gene. Adapter plasmid pAdApt35BLacZ was generated as follows: The ElB promoter fragment was amplified using the primers 35ElBlong and Ad35Elbpromrev. Both primers were phosphorylated. The' reaction was done with Pwo DNA polymerase (Inno-train, Diagnostic GmbH) according to manufacturer instructions. pBr.Ad35.leftlTR-pIX was used as template DNA, (25 ng, described in WO 02/40665). The program was set as follows: 2 minutes ac 94°C and then 3'0   cycles of (94°C for 30 seconds, 60°C for 30 seconds and 72°C for 1 minute) and ended by 10 minutes at 72°C. The cooling/heating slope was set at 2°C/sec. This PCR results in amplification of the potential ElB promoter of Ad35 of 125 nucleotides. Construct pAdapt535.LacZ (example 5) was then digested with Mfel and Bglll. After digestion the vector was treated with Klenow enzyme to create blunt ends. A dephosphorylation step was done using SAP<br><br>
(Roche). The thus treated 8 kb vector fragment was then isolated from gel. The E1B promoter region was also isolated from gel. These two fragments were ligated and transformed into DH5a-Tlr competent cells (Invitrogen). The correct orientation of the E1B promoter in the resulting plasmid was confirmed by digesting with Hpal and ApaLI. After selection of the correct clone the inserted E1B promoter sequence was also verified by sequencing.<br>
Example 18. The promoter of the pIX gene is located in the 3' end of the E1B-55K coding sequence in Ad35 and Adll viruses. Based on the results described supra  we expected that the pIX promoter in subgroup B viruses would be located in the E1B-55K coding region. To investigate this directly, we set out to identify the pIX raRNA cap site as described in example 10. Hereto, wtAd35, wtAdll and Ad3 5.ElB+.AdApt.Luc viruses were used to infect PER55K clone 16 cells at an MOI of 50 VP/cell. As a control wtAd5 was taken along since the promoter and mRNA start site of this virus is known. RNA was isolated from the infected cultures at 16-18 hrs post-infection using TRIzol agent (Invitrogen) as described by the manufacturer. At the end of the procedure, the isolated RNA was stored in 100% formamide. The GeneRacer Kit (Invitrogen) was used to amplify the 5' end of pIX transcripts, in order to locate the start of transcription. Before starting GeneRacer protocol, 5 p,g RNA was purified from the formamide by sodium acetate precipitation as described in the GeneRacer protocol. Purified RNA was treated according to manufacturer's protocol for amplification of the 5' end of the pIX mRNA. After phosphatase treatment and subsequent removal of the cap structure with tobacco acid pyrophosphatase and ligation of the Generacer RNA oligo,<br><br>
Superscript™ II Reverse Transcriptase from the kit was used for cDNA synthesis. cDNA was synthesized by reverse transcription using a gene specific (reverse) primer for pIX. For Ad35 (wt and ElB+.Luc virus) and wtAdll primer pIXrev was used, for Ad5 the primer plXrev-Ad5 was used. The resulting cDNA (1 nl of unknown concentration) was used as a template for PCR to generate dsDNA. This PCR was done using Pwo DNA polymerase (Roche) according to manufacturers instructions, and with addition of DMSO (Sigma; 3% v/v). The amplification was done with the GeneRacer 5' primer from the kit which is specific for the oligonucleotide ligated to the 5' end of the mRNA, and the gene specific reverse primers, as mentioned above. Reaction conditions were as follows: denaturation at 94°C for 2 minutes, followed by 30 cycles of (94°C for 30 seconds, 60°C for 30 seconds, 72°C for 2 minutes) and finished by elongation at 68°C for 8 minutes. The resulting DNA fragments were size separated by electrophoresis on a 1.0% agarose gel. For Ad5 a 480 bp fragment and for Ad35 (both viruses) and Adll a 200 bp fragment was obtained. Adll also showed a 2 kb fragment. All fragments were excised and purified from agarose gel. The purified DNA fragments were cloned into the pCR4Blunt-TOPO® vector (Invitrogen). The vectors were sequenced using commercial M13 Forward and Reverse primers. The resulting sequences were aligned against wildtype sequences to locate the start of pIX transcription. The 200 bp band isolated from Ad35 and Adll cDNA preparations constituted the genuine pIX mRNA, the 2 kb fragment isolated from Adll turned out to originate from the E1B promoter. Fig. 30 shows an alignment of the Ad35 (SEQ. ID. NO. 59) and Adll (SEQ. ID. NO. 58) cDNA sequences with the wt Ad35 sequence (SEQ. ID. NO. 60). The alignment reveals the location of a<br><br>
spliced intron sequence in the pIX mRNA. In Fig. 31 the location of the identified cap site and splice sites is schematically shown in the ElB-pIX region of Ad35. For Ad5 the expected pIX mRNA {Babiss and Vales, 1991) was identified with the cap site located at position 3580 {not shown; numbering as in genbank accession no. M73260) . For Ad35 the cap site was located in the 3' end of the E1B-55K gene at position 3339 on an A residue (Ad35 sequence WO 00/70071). In Adll the cap site was similarly found on a T residue (position 3339 in Genbank Ace No. AY163756). Interestingly, the sequence between the stop codon of the 55K gene and the start codon of the pIX, where in Ad5 viruses the promoter for pIX is located, is spliced out of the mRNA in Ad35 and Adll viruses. These results provide strong evidence that in Ad35 and Adll pIX gene expression is regulated from a promoter located in the 3' end of the 55K gene.<br>
Example 19: El-deleted Ad35 viruses that retain a short stretch of 3' E1B-55K sequence have a larger packaging capacity.<br>
With the identification of the pix mRNA cap site it becomes possible to include the natural Ad35 promoter for correct pIX expression and also limit as much as possible the E1B-55K sequences in the viral vector. Here we show, as a non-limiting example, the construction of an Ad35 adapter plasmid that retains 166 bp of the 3'end of the 55K coding sequence (pAdApt35Bsu.Luc) and the generation of an El-deleted Ad35Luc virus with increased stability and/or packaging capacity. This 166 bp sequence does not code a functional 55K gene product but contains the pIX mRNA cap site identified in the previous example in its natural position relative to the pIX coding sequence.<br>
(0&gt;<br><br>
Construct pAdApt35Bsu.Luc was generated as follows: First a PCR fragment was generated using 4 0ng<br>
pBr.Ad35.1eftITR-pIX as target DNA {described in WO 02/40665) and primers Bsu55KF and Age-pIXR. The PCR was performed with Pwo DNA polymerase (Genaxis) according to manufacturer instructions. In addition, 3% v/v DMSO (Sigma) was used. The program was set as follows: 2 minutes at 94°C, then 30 cycles of (94°C for 30 seconds, 60°C for 30 seconds and 72°C for 1.5 minute) and ended by 8 minutes at 68°C. The resulting 1.2 kb product was cloned directly into the pCR4Blunt-T0P0 vector (Invitrogen) according to the manufacturer's protocol, resulting in pCR4Blunt-TOPO.Bsu-Age. The construct was checked by digestion with PvuII (New England Biolabs). The Bsu-Age fragment was isolated from the pCR4Blunt-TOPO.Bsu-Age plasmid by digestion with Bsu36I (New England Biolabs) and treated with Klenow enzyme {New England Biolabs) to blunt ends. The DNA was then purified using the PCR purification kit (Qiagen) and was digested with Agel (New England Biolabs). The 1 kb fragment was isolated from gel using the Gene clean II kit (BiolOl, Inc.). In parallel, construct pAdApt35.Luc (described in WO 00/70071) was digested with Bglll and treated with Klenow enzyme. The DNA was purified using the PCR purification kit {Qiagen). The purified DNA was digested with Agel (New England Biolabs) and dephosphorylated with SAP (Roche). The 5.8-Kb fragment was isolated from gel with the Gene clean II kit (BiolOl). The two fragments were mixed in equimolar amounts in a ligation reaction and transformed into Tl resistant EM DH10B cells (Invitrogen) . This resulted in plasmid pAdApt35Bsu.Luc.<br>
To generate El-deleted viruses pAdApt35Bsu.Luc was digested with plPsp-I and co-transfected onto PER55K clone 16 cells<br><br>
with NotI digested pWE.Ad35-3481 (example 7) and with pWE.Ad35-348lAE3 as described before. pWE.Ad35-348lAE3 contains the same E3 deletion as described for pWE.Ad35.pIX-rITRAE3 (example 2} and was generated according to the method described in example 7 using construct pWE.Ad35-348lANdeI and a 26,6 kb Ndel fragment from pWE.Ad35.pIX-rITRAE3. In addition, El-deleted Ad35 viruses were generated that contained the E4-Orf6 and E4-Orf6/7 sequences from Ad5 in the viral backbone replacing the native Ad35 sequences (see example 16 for the generation of such viruses on unmodified PER.C6 cells}. In the current example pAdApt35Bsu.Luc digested with pIPsp-I is cotransfected with Notl/EcoRV digested pWE.Ad35.pIX-EcoRV and with PacI/NotI digested pBr.Ad35.PR50rf6 (with and without E3 region). All transfections gave rise to full CPE within a week following transfection and cells and medium were harvested as described before. Viruses were then plaque purified and viral stocks amplified on the appropriate complementation cells and originating from single plaques were analysed by PCR for integrity of the transgene region. Transgene PCRs were done as described in example 3 using primers AdApt35CMVF and pIXrevN2. Fig. 32 shows an example of the PCR results on plaques originating from Ad35Bsu.Luc and from Ad35Bsu.Luc.50rf6<br>
viruses.<br>
Irrespective of the presence of the E3 region or of the E4-0rf6 sequence in the viral backbone, all tested plaques (5-10 for each virus) contained an intact transgene. One exception occurred in one of the plaques from Ad35Bsu.Luc viruses that showed a faint band at approximately 1,6 kb (Fig. 32; lane 12} probably originating from minor amount of viruses with a deletion. The faint band at approximately 500 bp that occurs<br><br>
in all virus samples, is a background band of the primers on the viral backbone. The observation that E3-containing Ad35 viruses with a luciferase expression cassette proved stable following plaque purification is in contrast to previous results with Ad35.AdApt.Luc viruses that are fully El-deleted and do not contain the 166 bp 3' 55K coding sequence. Using standard pAdApt35.Luc plasmids, we were not able to generate plaque-purified viruses containing the E3 region. Thus with incorporation of the extra 55K sequences in the backbone we can now make viruses of more than 34.6 kb total length without severe instability. This closely matches the length of a wt Ad35 virus. If an E3-deleted backbone were used, the capacity for foreign sequences theoretically would be over 5 kb. Obviously, it is possible to incorporate more E1B-55K sequences than in the current example and/or to combine the 3' 55K sequences with heterologous enhancer sequences without departing from the invention disclosed inhere.<br>
c
 <br>
TABLES Table I: Yields of El- and E1/E3- deleted Ad35 viruses on clone #16 cells produced on triple layer flasks.<br><br>
Virus	Scale<br>
(T175m<br>
flasks)	Total # of Virus Particles after DSP	VP/cell<br>
Ad35.AdApt.eGFP<br>
4	7.5x10"	2500<br>
Ad35.AE3.AdApt.empty	8	2x10"	3300<br>
Ad35.AE3.AdApt.LacZ<br>
8	3.8x10"	600<br>
Ad35.AE3.AdApt.MV-F<br>
4	8.8x10"	2900<br>
Ad35.AE3.AdApt.MV-H<br>
8	2.6x10"	4250<br>
Table II: Transgene (LacZ) activity test on A549 using crude lysates from second passage virus. % blue cells is given for each amount of virus used for infection.<br><br>
Virus	10 M-l	1 Hi	0.1 ill<br>
Ad35.AdApt.LacZ.rsv<br>
95	15	
Ad35.AE3.AdApt.LacZ.rsv<br>
90	10	
Ad35.AdApt.LacZ.C4	2	O.l	0<br>
Ad35.AE3.AdApt.LacZ.C4	15	
(Ofe<br><br>
Table III: Transgene {Luciferase) activity test on A549 using crude lysates from second passage virus. Activity is expressed in relative light units (RLU).<br><br>
Virus	10 Ml	mi	0.1 ul	0.01 ul<br>
Ad35.AdApt.Luc,rsv	845453	27940	178	26<br>
Ad35.AE3.AdApt.Luc.rsv<br>
258269	2217	46	6<br>
Ad35.AdApt.Luc.C4	6130	175	18	33<br>
Ad35.AE3.AdApt.Luc.C4	814642	6278	H7	23<br>
Ad35.AElAA21K.Luc<br>
1514698	50196	503	57<br>
(0^<br><br><br><br><br><br><br><br>
REFERENCES<br>
Abrahamsen K, Kong H-L, Mastrangeli A, Brough D, Lizonova A, Crystal R and Falck-Pedersen E. (1997) Construction of an adenovirus type 7a E1A" vector. J Virol 11:8946-8951.<br>
Babiss, L.E. and Valesj L.D. (1991). Promoter of the adenoviral polypeptide IX gene: Similarity to ElB and inactivation by substitution of the simian virus TATA element. J. Virol. 65(2), p598-605.<br>
Bett, A.J., Prevec, L. and Graham, F.L. (1993). Packaging capacity and stability of human adenovirus type 5 vectors. J. Virol. 67(10), p5911-5921.<br>
Boshart, M., Weber, F., Jahh, G., Dorsch-Hasler, K., Fleckenstein B. and Schaffnqr W. (1985). A very strong enhancer is located upstream of an immediate early gene of human cytomegalovirus. Cell 41, 521-530.<br>
Caravokyri, C. and Leppard, K.N. (1995) . Constitutive episomal expression of polypeptide IX (pIX) in a 293-based cell line complements the deficiency of pIX mutant adenovirus type 5. J. Virol. 69(11), p6627-6633.<br>
Fallaux, F. J., Kran.enbu.rg, 0 ., Cramer, S . J., Kouweling, A., van Ormondt, H., Hoeben, R.C. and van der Eb, A.J. (1996). Characterization of 911: A new helper cell line for the titration and propagation of early region 1-deleted adenoviral vectors. Hum. Gene Ther. 7, p 215-222.<br>
Fallaux, F.J., Bout, A., van der Velde, I., van den Wollenberg, D.J., Hehir, K.M., Keegan, J., Auger, C., Cramer, S.J.,van Ormondt, H., van der Eb, A.J., Valerio, D. and Hoeben, R.C. (1998) . New helper cells and matched early region 1-deleted adenovirus vectors prevent generation of replication competent adenoviruses. Hum. Gene Ther. 9, p 1909-1917.<br>
Farina, S.F., Gao, G.P., Xiang, Z.Q., RUX, J.J., Burnett, R.M., Alvira, M.R., Marsh, J., Ertl, H.c. and Wilson, J.M.<br><br>
(2001). Replication-defective vector based on a chimpanzee adenovirus. J. Virol. 75(23), pll603-11613.<br>
Francki, R.I.B., Fauquet, CM., Knudson, L. and Brown, F. (1991). Classification and .nomenclature of viruses. Fifth report of the international! committee on taxonomy of viruses. Arch. Virol. Suppl. 2, p 140-144.<br>
Furcinitti, P.S., van Oostrom, J. and Burnett R.M. (1989). Adenovirus polypeptide IX revealed as capsid cement by difference images from electron microscopy and crystallography. EMBO J. 8(12), p3563-3570.<br>
Grable, M. and Hearing, P. (1990). Adenovirus type 5 packaging domain is composed of a repeated element that is functionally redundant. J. Virol. Vol.64, No. 5, p 2047-2056.<br>
Graham, F.O., Smiley, J., Russell, W. and Nairn, R. (1970. Characteristics of a human cell line transformed by DNA from human adenovirus type 5. J. Gen. Virol. 36, p59-72.<br>
Gustin, K.E. and Imperiale, M.J. (1998). Encapsidation of viral DNA requires the adenovirus LI 52/55-kilodalton protein. J. Virol. Vol 72, No. 10, p 7860-7870.<br>
Ghosh-Choudhury, G., H0j-Ahmad, Y. and Graham, F.L. (1987), Protein IX, a minor component of the human adenovirus capsid, is essential for the packaging of full length genomes. EMBO J. 6(6}, pl733-1739.<br>
Hagmeyer BM, Duyndam MC, Angel P, De Groot RP, Verlaan M, Elfferich P, Van der Eb AJ and Zantema A. (1996) Oncogene 12:1025-1032.<br>
Hehir, K.M., Armentano, D., Cardoza, L.M., Choquette, T. L., Berthelette, P.B., White, G.A., Couture, L.A., Everton, M.B., Keegan, J., Martin, J.M., Pratt, D.A., Smith, M.P., Smith, A.E. and Wadsworth, S.C. (1996). Molecular characterization of replication-competent variants of<br><br>
adenovirus vectors and genome modifications to prevent their occurrence. J. Virol. 70 (12), p8459-8467.<br>
Herrman, C.H. and Mathews, M.B. (1989). The adenovirus ElB 19-kilodalton protein stimulates gene expression by increasing DNA levels. Mol. Cell. Biol. 9, p5412-5423.<br>
Horwitz, M.S. (2001). Adenovirus immunoregulatory genes and their cellular targets. Virology 279(1), p 1-8.<br>
Leppard, K.N. (1997). E4 gene function in adenovirus, adenovirus vector and adeno-associated virus infections. J. Gen. Virol. 78, p 2131-2138.<br>
Leppard, K.N. (1998). Regulated RNA processing and RNA transport during adenovirus infection. Seminars in Virology 8, p 301-307.<br>
Lutz, P., Rosa-Calatrava, M. and Kedinger, C. (1997). The product of the adenovirus intermediate gene IX is a transcriptional activator. J. Virol. 71(7), p5102-5109.<br>
Pilder, S., Logan, J. and Shenk, T. (1984). Deletion of the gene encoding the adenovirus type 5 early region IB 21,000-molecular-weight polypeptide leads to degradation of viral and host cell DNA. J. Virol. 52(2), p664-671.<br>
Pilder, S., Moore, M., Logan, J. and Shenk, T. (1986). The adenovirus ElB 55K transforming polypeptide modulates transport or cytoplasmic stabilization of viral and host cell mRNAs. Mol. Cell. Biol. 6, p470-476.<br>
Rao, L., Debbas, M., Sabbatini, P., Hockenbery, D., Korsmeyer, S. and White, E.. (1992). The adenovirus E1A proteins induce apoptosis, which is inhibited by the ElB 19-kDa and Bcl-2 proteins. Proc. Natl. Acad. Sci. USA Vol. 89, pp 7742-7746.<br>
Reese, M.G. and Eeckman, F.H. (1995). Novel neural network algorithms for improved eukaryotic promoter site recognition. Talk and Abstract from The seventh international genome<br>
0»<br><br>
sequencing and analysis conference. Hyatt Regency/ Hilton Head<br>
Island, South Carolina, Sept 16-20, 1995.<br>
(http://www.fruitfly.org/seq_tools/promoter.html).<br>
Robert, J.-J., Gauffeny, I., Maccario, J., Jullien, C, Benoit, P., Vigne, E., Crouzet, J., Perricaudet, M. and Yeh, P. (2001). Degenerated pIX-IVa2 adenoviral vector sequences lowers reacquisition of the El genes during virus amplification in 293 cells. Gene Ther. 8, pl713-1720.<br>
Russell, W,C. (2000). Update on adenoviruses and its vectors. J. Gen. Virol. 81, pp 2573-2604.<br>
Sambrook, J. and Russell, D. Molecular Cloning: A laboratory manual. Third Edition. Cold Spring Harbor Press (2001). ISBN 0-87969-576-5.<br>
Shabram, P.W., Giroux, D.D., Goudreau, A.M., Gregory, R.J., Horn, M.T., Huyghe, B.G., Liu, X., Nunnally, M.H., Sugarman, B.J. and Sutjipto, S. (1997) Analytical anion-exchange HPLC of recombinant type-5 adenoviral particles. Hum. Gene Ther. 8(4): 453-465.<br>
Shenk, T. (1996) . Adenoviridae: The viruses and their replication. In Virology, eds. Fields, B.N., Knipe, D.M. and Howley, P.M. (Lippincott-Raven, New York), Vol. 2, pp 2111-2148.<br>
Van der Vliet, P.C. (1995) . Adenovirus DNA replication. In The molecular repertoire of adenoviruses II, eds. Doerfler, W. and Bohm, P. (Springer-Verlag, Berlin). Current Topics in Microbiology and Immunology 199/IT, pp 1-30.<br>
White, E., Grodzicker, T. and Stillman, B.W. (1984). Mutations in the gene encoding the adenovirus early region IB 19,000 molecular-weight tumor antigen cause degradation of chromosomal DNA. J.Virol. 52(2), p410-419.<br><br>
Yew, P.R. and Berk, A.J. (1992). Inhibition of p53 transactivation required for transformation by adenovirus early region IB protein. Nature 357, pp 82-85. Zhang, W., Low, J.A., Christensen, J.B. and Imperiale, M.I. (2001). Role for the adenovirus IVa2 protein in packaging of viral DNA. J. Virol. Vol. 75 No. 21, pp!0446-10454.<br>
U*<br><br>
WE CLAIM :<br>
1.	A recombinant subgroup B adenovirus particle comprising an adenovirus genome wherein the genome comprises a deletion in the El-region, which adenovirus genome does not encode a functional ElB 55K gene product, said adenovirus genome comprising a plX coding sequence under control of an expression sequence, said expression sequence comprising part of the 3' end of the ElB 55K coding sequence containing from 150 to 600 nucleotides of adenovirus sequences directly upstream of the plX open reading frame of a wild-type subgroup B adenovirus, and said expression sequence being capable of increasing expression of the plX coding sequence in a packaging cell, relative to the expression of said plX coding sequence when preceded by sequences between the ElB 55K stop codon and the start codon of plX and without nucleotides from the 3' end of the ElB 55K coding sequence.<br>
2.	The recombinant adenovirus particle as claimed in claim 1, wherein said expression sequence contains from 150 to 250 nucleotides of the adenovirus sequences that are directly upstream of the plX open reading frame of a wild-type subgroup B adenovirus.<br>
3.	The recombinant adenovirus particle as claimed in claim 1, wherein said expression sequence contains 166 base pairs of the 3' end of the ElB 55K open reading frame sequence.<br>
4.	The recombinant adenovirus particle as claimed in claim 1, wherein said expression sequence contains from 150 to 200 nucleotides of adenovirus sequences directly upstream of the plX open reading frame of a wild-type subgroup B adenovirus.<br>
Kb<br><br>
5.	The recombinant adenovirus particle as claimed in claim 1, wherein said adenovirus particle is an adenovirus serotype 35 particle or an adenovirus serotype 11 particle.<br>
6.	The recombinant adenovirus particle as claimed in claim 1, further comprising a sequence encoding a functional E4-orf6 protein of adenovirus subgroup C origin.<br>
7.	An isolated nucleic acid, comprising a genome of the recombinant adenovirus particle as claimed in claim 1.<br>
8.	A vaccine comprising the recombinant adenovirus particle as claimed in claim 1.<br>
9.	The recombinant adenovirus particle as claimed in claim 1, wherein said heterologous promoter comprises a nucleic acid sequence selected from the group consisting of a non-endogenous adenoviral proximal pIX promoter, a non-adenovirus viral promoter, a cellular promoter, a synthetic promoter and a hybrid promoter.<br>
117<br><br>
10. The recombinant adenovirus particle as claimed in claim 9, wherein said heterologous promoter is selected from the group consisting of an Ad5 proximal pIX promoter or a Rous Sarcoma Virus promoter.<br><br>
118<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwMS1jaGVucC0yMDA0IGFic3RyYWN0LWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">2401-chenp-2004 abstract-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwMS1jaGVucC0yMDA0IGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2401-chenp-2004 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwMS1jaGVucC0yMDA0IGNsYWltcy1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">2401-chenp-2004 claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwMS1jaGVucC0yMDA0IGNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2401-chenp-2004 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwMS1jaGVucC0yMDA0IGNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2401-chenp-2004 correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwMS1jaGVucC0yMDA0IGNvcnJlc3BvbmRlbmNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">2401-chenp-2004 correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwMS1jaGVucC0yMDA0IGRlc2NyaXB0aW9uIChjb21wbGV0ZSktMS5wZGY=" target="_blank" style="word-wrap:break-word;">2401-chenp-2004 description (complete)-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwMS1jaGVucC0yMDA0IGRlc2NyaXB0aW9uIChjb21wbGV0ZSktZHVwbGljYXRlLTEucGRm" target="_blank" style="word-wrap:break-word;">2401-chenp-2004 description (complete)-duplicate-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwMS1jaGVucC0yMDA0IGRlc2NyaXB0aW9uIChjb21wbGV0ZSktZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">2401-chenp-2004 description (complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwMS1jaGVucC0yMDA0IGRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">2401-chenp-2004 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwMS1jaGVucC0yMDA0IGRyYXdpbmdzLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">2401-chenp-2004 drawings-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwMS1jaGVucC0yMDA0IGRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">2401-chenp-2004 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwMS1jaGVucC0yMDA0IGZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">2401-chenp-2004 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwMS1jaGVucC0yMDA0IGZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">2401-chenp-2004 form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwMS1jaGVucC0yMDA0IGZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">2401-chenp-2004 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwMS1jaGVucC0yMDA0IGZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">2401-chenp-2004 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwMS1jaGVucC0yMDA0IGZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">2401-chenp-2004 form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwMS1jaGVucC0yMDA0IG90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2401-chenp-2004 others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwMS1jaGVucC0yMDA0IHBjdCBzZWFyY2ggcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">2401-chenp-2004 pct search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwMS1jaGVucC0yMDA0IHBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">2401-chenp-2004 pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwMS1jaGVucC0yMDA0IHBldGl0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">2401-chenp-2004 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="229708-novel-carbonyl-reductase-gene-encoding-the-same-and-process-for-producing-optically-active-alcohols-using-the-same.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="229710-method-and-apparatus-for-wideband-docsis-on-catv-systems-using-port-trunking.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>229709</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2401/CHENP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>19-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>21-Oct-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CRUCELL HOLLAND B.V</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>Archimedesweg 4, NL-2333 CN Leiden,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>VOGELS, Ronald</td>
											<td>Van Rietlaan 4, NL-3461 HW Linschoten,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>HAVENGA, Menzo, Jans, Emco</td>
											<td>Wilhelmina Druckerstraat 66, NL-2401 KG Alphen aan den Rijn,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>ZUIJDGEEST, David, Adrianus, Theodorus, Maria</td>
											<td>Klimopstraat 85, NL-2565 VH Den Haag,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/861</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2003/050126</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-04-24</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>PCT/NL02/00281</td>
									<td>2002-04-25</td>
								    <td>Netherlands</td>
								</tr>
								<tr>
									<td>2</td>
									<td>02102631.5</td>
									<td>2002-11-25</td>
								    <td>Netherlands</td>
								</tr>
								<tr>
									<td>3</td>
									<td>PCT/NL02/00656</td>
									<td>2002-10-15</td>
								    <td>Netherlands</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/229709-a-recombinant-subgroup-b-adenovirus-particle-comprising-an-adenovirus-genome by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:25:29 GMT -->
</html>
