<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/229731-peptide-inhibitors-of-thrombin-as-potent-anticoagulants by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:26:48 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 229731:PEPTIDE INHIBITORS OF THROMBIN AS POTENT ANTICOAGULANTS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">PEPTIDE INHIBITORS OF THROMBIN AS POTENT ANTICOAGULANTS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to an isolated polypeptide inhibitor of thrombin consisting of all naturally-occurring amino acids, said inhibitor having general formula (1) of: SBM-PBM-EBM (1) wherein SBM is a sequence moiety comprising P-subsite residues P4-P3-Pro-Arg of the thrombin inhibitor, PBM is a sequence moiety comprising P&#x27;-subsite residues Pro-P2&#x27;-P3&#x27;-P4&#x27;-P5&#x27;-P6&#x27;-P7&#x27; of the thrombin inhibitor, and EBM is a peptide sequence moiety that binds to the fibrinogen recognition exosite of thrombin, or a pharmaceutically acceptable salt thereof, wherein P4, P3, P2&#x27;, P3&#x27;, P4&#x27;, P5&#x27;, P6&#x27; and P7&#x27; are natural amino acid residues, where P3&#x27; is Arg, Leu, He or Met, and wherein P and P&#x27; represent a division of the peptide sequence at the peptide bond hydrolyzed specifically by thrombin. The invention also relates to a method for selecting a polypeptide inhibitor of thrombin.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Peptide Inhibitors of Thrombin as Potent Anticoagulants<br>
BACKGROUND OF THE INVENTlON<br>
(a)	Field of the Invention<br>
The invention relates to peptide inhibitors of thrombin that can be used as potent anticoagulants, and that are composed of natural amino acids or that can be made by recombinant techniques.<br>
(b)	Description of Prior Art<br>
A wide range of medical conditions including atherosclerosis, infections and cancer can trigger thrombotic complications, leading to heart attack, stroke, deep-vein thrombosis, or pulmonary embolism (Libby, P. (2002) Nature 420, 868-874; Levi, M., Keller, T. T., van Gorp, E.. and ten Gate. H. (2003) Cardiovasc. Res. 60, 26-39; Opal, S. M. and Esmon, C. T. (2003) Crit Care 7, 23-38; Loynes, J. and Zacharski, L. (2003) Expert. Opin. Then Targets. 7, 399-404; and Schultz, M. J., Levi. M., and van der, P. T. (2003) Curr. Drug Targets. 4, 315-321). As such, thrombosis, or the aberrant formation of a blood clot, has been the single largest cause of human disability and death in the worid. In many situations, it is the occlusive blood clot that is life-threatening for .patients with atherosclerosis and related cardiovascular diseases (Libby, P. (2002) Sci. Am. 286, 46-55; Libby, P. (2002) Nature 420, 868-874; and Virmani, R., Burke, A. P., and Farb, A. (2001) Cardiovasc. Pathol. 10, 211-218) or on long-term anti-HIV treatments (Madamanchi, N. R., Patterson, C, and Runge, M. S. (2002) Arterioscler. Thromb. Vase. Biol. 22, 1758-1760; and Zhong, D. S., Lu, X. H., Conklin, B. S., Lin, P. H., Lumsden, A. B., Yao, Q., and Chen, C. (2002) Arterioscler. Thromb. Vase. Biol. 22, 1560-1566). Pathogenic blood coagulation or thrombosis aggravates the symptoms of chromic liver infections and underiines the lethality of many infectious diseases (Levi, M., Keller, T. T., van Gorp, E., and ten Gate, H. (2003) Cardiovasc. Res. 60, 26-39; Marsden, P. A., Ning, Q., Fung, L. S., Luo, X., Chen, Y., Mendicino, M., Ghanekar, A., Scott, J. A., Miller, T., Chan, C. W., Chan, M. W., He, W., Gorczynski, R. M., Grant, D. R., Clark, D. A., Phillips, M. J., and Levy, G. A. (2003) J. Clin. Invest 112, 58-66; and Opal, S. M. and Esmon, C. T. (2003) Crit Care 7, 23-38). Malignant cells have been found to constitutively express the procoagulant tissue factor, generating hypercoagulability In cancer patients (Agorogiannis, E. I. and<br><br>
Agorogiannis, G. I. (2002) Lancet 359, 1440; Lorenzet, R. and Donati, M. B. (2002) Thromb. Haemost 87, 928-929; and Omstein, D. L., Meehan, K. R., and Zacharski, L. R. (2002) Semin. Thromb. Hemost. 28, 19-28). These recent observations have attracted significant attention to the potential use of anticoagulants or antithrombotic agents as part of new treatment strategies for devastating human cancers (Loynes, J. and Zacharski, L. (2003) Expert. 0pm. Then Targets. 7, 399-404; Kakkar, A. K. (2003) Cancer Treat. Rev. 29 SuppI 2, 23-26; Levine, IV!. N. (2003) Cancer Treat. Rev. 29 SuppI 2, 19-22; Lee, A. Y. (2003) Expert. Opin. Phanriacother. 4, 2213-2220; and Deitcher, S. R. (2003) J. Thromb. Thrombolysis. 16, 21-31) and Infectious diseases (Marsden, P. A., Ning, Q., Fung, L. S., Luo, X., Chen, Y., Mendicino, M., Ghanekar, A., Scott, J. A., Miller, T., Chan, C. W., Chan, M. W., He, W., Gorczynski, R. M., Grant, D. R., Clark, D. A., Phillips, M. J., and Levy, G. A. (2003) J. Clin. Invest 112, 58-66; Opal, S. M. and Esmon, C. T. (2003) Crit Care 7, 23-38; Geisbert, T. W., Hensley, L. E., Jahriing, P. B., Larsen, T., Geisbert, J. B., Parages, J., Young, H. A., Fredeking, T. M., Rote, W. E., and Vlasuk, G. P. (2003) Lancet 362, 1953-1958; Robertson, M. (2003) Drug Discov. Today 8, 768-770; and Schultz, M. J., Levi, M., and van der, P. T. (2003) Curr. Drug Targets. 4, 315-321). However, the current generation of antithrombotic agents, among which many are thrombin inhibitors, lacks the required efficacy/safety and cost-effectiveness (Gresele, P. and Agnelli, G. (2002) Trends Pharmacol. Sci. 23, 25-32; Vorchheimer, D. A. and Fuster, V. (2002) Eur Heart J. 23, 1142-1144; and Weitz. J. I. and Buller, H. R. (2002) Circulation 105, 1004-1011) for realizing the tremendous potential of anticoagulant therapy in many disease indications. Blood coagulation is one of the best-characterized physiological responses that involve tightly-regulated cascades of protein-protein interaction and enzyme activation reactions (Mann, K. G. (1999) Thromb. Haemost. 82, 165-174; and Furie, B. and Furie, B. C. (1988) Cell 53, SOS-SIS). The coagulation processes can be triggered by the exposure of blood to open air and/or upon injury of the vascular wall (e.g. at the sites of atherosclerotic lesions). The clotting of the free blood is the result of the so-called "intrinsic" coagulation pathway started by the activation of factors XII and XI. Blood clots formed in closed circulation are initiated by the<br><br>
"extrinsic" coagulation pathway though contact of blood with exposed tissue factors (TF) on injured blood vessels. The two pathways converge on the activation of the circulating coagulation factor X into the factor Xa enzyme, which in turn is assembled into a macromolecular enzyme-cofactor complex, called the prothrombinase, containing factor Xa, factor Va, calcium ions and a phospholipid surface (Mann, K. G. (1999) Thromb. Haemost. 82, 165-174). There also appears to be a third pathway of blood coagulation, in which factor Xa of the prothrombinase is replaced by a tissue-specific Xa-like protein, the fgl2/fibroleukin, induced by the invasion of pathogenic viruses (Marsden, P. A., Ning, Q., Fung, L. S., Luo, X., Chen, Y., Mendicino, M., Ghanekar, A., Scott, J. A., Miller, T., Chan, C. W., Chan, M. W.. He, W., Gorczynski, R. M., Grant, D. R., Clark, D. A., Phillips, M. J., and Levy, G. A. (2003) J. Clin. Invest 112, 58-66; and Chan, C. W., Chan,. M. W., Liu, M., Fung, L., Cole, E. H., Leibowitz, J. L., Marsden, P. A., Clark, D. A., and Levy, G. A. (2002) J. Innmunol. 168, 5170-5177). As well, factor Xa may be generated from the inactive precursor factor X by endogenous proteases secreted by invading microbes (Ntefidou, M., Eisner, C, Spreer, A., Weinstock, N., Kratzin,.H. D., and Ruchel, R. (2002) Mycoses 45 SuppI 1, 53-56; and Schoen, C, Reichard, U., Monod, M., Kratzin, H. D., and Ruchel, R. (2002) Med. Mycol. 40, 61-71). In all the coagulation pathways, the prothrombinase assembly rapidly converts prothrombin into active thrombin, the ultimate protease resulting from the coagulation cascades. Upon generation, thrombin induces formation of the fibrin clot from the soluble fibrinogen, activates the fibrin cross-linking factor XIII, stimulates the aggregation of platelets and catalyzes the conversion of factors V, VIII and XI into Va, Villa and XIa to amplify its own production. Thrombin also binds to the cell-anchored thrombomodulin to form the thrombin-thrombomodulin complex, which in turn activates protein C and the thrombin-activatable fibrinolysis inhibitor (TAFI), initiating the natural anticoagulation and anti-fibrinolysis pathways (Nesheim, M., Wang, W., Boffa, M., Nagashima, M., Morser, J., and Bajzar, L. (1997) Thromb. Haemost. 78, 386-391). The critical role of thrombin in making blood clots and in thrombotic diseases has stimulated in-depth studies on the structure and function of thrombin (Berliner, J. L. (1992) Thrombin: structure and function Plenum Press, New York) and the<br><br>
design of thrombin inhibitors as novel anticoagulants (Weitz, J. I. and Bulier, H. R. (2002) Circulation 105, 1004-1011; Fenton. J. W., Ni. F., Witting, J. I., Breznial
Tlie mainstays of clinical anticoagulant treatments are heparin, which is a cofactor of plasma-derived and naturally-occurring inhibitors of thrombin, and coumarins, such as arfarin, which antagonize the biosynthesis of vitamin K-dependent coagulation factors. Although effective and widely used, heparins and coumarins have practical limitations because their pharmacokinetics and anticoagulation effects are unpredictable, with the risk of many undesirable side effects, such as hemorraghing and thrombocytopenia resulting in the need for close monitoring of their use. Low-molecular-weight heparins (LMWHs) provide a more predictable anticoagulant response; however, discontinuation of heparin treatment can result in a thrombotic rebound due to the inability of these compounds to inhibit clot-bound thrombin. More seriously, heparins are involved in many aspects of cellular physiology (Kakkar, A. K. (2003) Cancer Treat. Rev. 29 SuppI 2, 23-26), making their long-term uses as anticoagulants plagued with potential side effects.<br>
There is a need for direct thrombin inhibitors (DTI) that are able to target: (1) free and (2) clot-bound thrombin. Hirudin is a member of only the first class and is a naturally occurring polypeptide produced by the blood sucking leech hirudo medicinalis. Hirudin and its recombinant fomis bind in-eversibly to both the catalytic and substrate-recognition sites of thrombin. Other DTIs with lower molecular weights, such as DuP714, PPACK, and efegatran, have subsequently been developed, and these agents are better inhibitors of clot-bound thrombin and the thrombotic processes at sites of arterial damage. Such compounds inhibit thrombin by covalent attachment and can result in toxicity and nonspecific inhibition of other proteins. Eventually, further development of these small molecules was mostly abandoned. In recent years, the development of low molecular weight, active site-directed, and reversible DTIs has resulted in a number of highly potent and selective compounds, such as inogatran and melagatran. Ongoing clinical trails suggest that the binding characteristics<br><br>
of  these  low   molecular weight   DTls   may  also   result   in   bleeding complications.<br>
The main problem of small molecular weight compounds is their limited specificity since thrombin belongs to the family of serine proteases and these compounds do not have strong specificity toward thrombin alone. Also available data so far indicate that the development of the small-molBcule DTls of thrombin is more time-consuming than other inhibitory molecules. For example, it has proven difficult to disrupt high-affinity and highly-specific protein-protein interactions by use of small-molecule inhibitors (Benard, V., Bokoch, G. M., and Diebold, B. A. (1999) Trends Pharmacol. Sci. 20, 365-370; Cochran, A. G. (2001) Curr. Op'in. Chem. Biol. 5, 654-659; and Veselovsky, A. V., Ivanov, Y. D., Ivanov, A. S., Archakov, A. I.. Lewi, P., and Janssen, P. (2002) J. Mol. Recognit 15, 405-422). On the other hand, in addition to natural proteins (e.g. antibodies), novel polypeptide ligands have been discovered and shown to possess the ability to interfere selectively with the targeted protein-protein interactions (Cochran, A. G. (2001) Curr. Opin. Chem. Biol. 5, 654-659; Veselovsky, A. V., Ivanov, Y. D., Ivanov, A. S., Archakov, A. I., Lewi, P., and Janssen, P. (2002) J. Mol. Recognit. 15, 405-422; Juliano, R. L., Astriab-Fisher, A., and Faike, D. (2001) Mol. Interv. 1,40-53; and Sidhu, S. S., Fairbrother, W. J., and Deshayes, K. (2003) Chembiochem. 4, 14-25). Linking of weak-binding molecules to create bivalent or multivalent molecules (DiMaio, J., Gibbs, B., Munn, D., Lefebvre, J., Ni, F., and Konishi, Y. (1990) J. Biol. Chem. 265, 21698-21703; and Maraganore, J. M., Bourdon, P., Jablonski, J., Ramachandran, K. L., and Fenton, J. W. (1990) Biochemistry 29, 7095-7101) has also emerged as a general strategy for the design of potent inhibitors of enzymes (Jahnke, W., Florsheimer, A.. Blommers, M. J., Paris, C. G., Heim, J., Nalin, C. M., and Perez, L. B. (2003) Cum Top. Med. Chem. 3, 69-80; and Shuker, S. B., Hajduk, P. J., Meadows, R. P., and Fesik, S. W. (1996) Science 274, 1531-1534), receptors (Jahnke, W., Florsheimer, A., Blommers, M. J., Paris, C. G., Heim, J., Nalin, C. M., and Perez, L. B. (2003) Curr. Top. Med. Chem. 3, 69-80; and Kramer, R. H. and Karpen, J. W. (1998) Nature 395, 710-713) and protein-protein interactions (Song, J. and Ni, F. (1998) Biochem.   Cell Biol.   76,   177-188;   Mammen,   M.,   Choi,   S.   K.,   and<br><br>
Whitesides. G. M. (1998) Angew. Chem. Int. Ed. 37, 2754-2794; and Mourez, M., Kane, R. S., Mogridge, J., Metallo, S., Deschatelets, P., Sellman, B. R., Whitesides, G. M.. and Collier, R. J. (2001) Nat Biotechnol. 19, 958-961). Intervention of cellular and physiological processes with multivalent polypeptides in particular allows access to the built-in evolutionary specificity of naturally-occurring protein-protein interactions, potentially avoiding the non-specific binding or side effects often seen with small molecules. The synthetic compound bivalirudin is one example of the better DTIs with two covalently linked groups that bind to both the catalytic and substrate-recognition sites of thrombin.<br>
In late 1988, a research program was initiated at the Biotechnology Research Institute (BRI) on the design of thrombin inhibitors as antithrombotic agents. In this research program, a series of novel compounds (the Canadian version of Angiomax™/Bivalirudin) (Fenton, J. W., Ni, F., Witting, J. I., Brezniak, D. V., Andersen, T. T., and Malik, A. B. (1993)&gt;AQfv. Exp. Med. Biol. 340,1-13; Song, J. and Ni, F. (1998) Biochem. Cell Biol. 76, 177-188; DiMaio, J., Konishi, Y., U.S. 6,060,451; and CA 2,085,465) that mimic the multivalent action of hirudin, a natural antithrombin from medicinal leeches were discovered. These early research efforts have started to pay off as the related bivalent peptide, bivalirudin or hirulog, mimicking the action of hirudin has recently proved its clinical efficacy (Weitz, J. I. and Buller, H. R. (2002) Circulation 105, 1004-1011; Hirsh, J. (2003) Thromb. Res. 109 SuppI 1, S1-S8; Salam, A. M. (2003) Expert. Opin. Investig. Drugs 12, 1027-1033; and Wykrzykowska, J. J., Kathiresan, S., and Jang, I. K. (2003) J. Thromb. Thrombolysis. 15,47-57).<br>
Thrombin inhibitors issued from the research program were also disclosed in W099/19356. These peptide inhibitors contain two covalently-linked motifs that bind to a large surface area encompassing the catalytic active site and a protein recognition exosite of thrombin. One of these peptide molecules, P53, has an amino acid sequence of (d)F-P-R-P-Q-S-H-N-D-G-D-F-E-E-l-P-E-E-Y-L-Q (DiMaio, J., Gibbs, B., Munn, D., Lefebvre, J., Ni, F., and Konishi, Y. (1990) J. Biol. Chem. 265, 21698-21703; U.S. 6,060,451; and CA 2,085,465), which inhibits human a-thrombin with a K\ of -2.8 nM. The clinically-tested peptide known as<br><br>
hirulog or bivalirudin has a sequence of (d)F-P-R-P-G-G-G-G-N-G-D-F-E-E-l-P-E-E-Y-L, which is also a strong inhibitor of human a-thrombin (K|~2.3 nIVl) (l\/laraganore, J. M., Bourdon, P., Jablonski, J., Ramachandran, K. L., and Fenton, J. W. (1990) Biochemistry 29, 7095-7101). Hinjlog-8 was approved and adopted for clinical uses since January 2001 under the trademark, Anglomax™. However, the cost of Angiomax^"^ is prohibitive since it contains an amino acid residue in the (d)-configuration, i.e." (d)F or (d)Phe, requiring chemical synthesis, limiting broader clinical applications.<br>
The latest clinical experiences showed that uses of the current generation of antithrombotic agents, among which many are direct thrombin inhibitors, can cause prolonged systemic bleeding during anticoagulation and may be associated with rebound activation of coagulation and the re-occlusion of opened blood vessels after anticoagulant therapy (Gresele, P. and Agnelli, G. (2002) Trends Pfiarmacol. Sci. 23, 25-32; Vorchheimer, D. A. and Fuster, V. (2002) Eur. Heart J. 23, 1142-1144; and Weitz, J. I. and Buller, H. R. (2002) . Circulation 105, 1004-1011). In Canadian patent application CA 2,340,461, Shen et al. replaced the d-Phe moiety at the N-terminus of Bivalirudin with a 12 natural amino acids sequence derived from the thrombin receptor. This peptide, containing the LDPR (SEQ ID N0:1) sequence, exhibited an improved safety/efficacy profile with reduced bleeding complications as compared to Hirulog, despite having a significantly decreased binding affinity to thrombin (Xue, M., Ren, S., Welch, S.. and Shen, G. X. (2001) J. Vase. Res. 38, 144-152; and Chen, X., Ren, S., Ma, M. G., Dharmalingam, S., Lu, L, Xue, M., Ducas, J., and Shen, G. X. (2003) Atherosclerosis 169, 31-40). The (d)Phe-Pro-Arg sequence of P53 or Hirulog can also be replaced by the natural sequence of human FpA, i.e. acetyl-Asp-Phe-Leu-Ala-Glu-Gly-Gly-Gly-Val-Arg (SEQ IS N0:2) as proposed by Fenton et al (Fenton, J. W., Ni, F., Witting, J. I., Brezniak, D. V., Andersen, T. T., and Malik, A. B. (1993) Adv. Exp. Med. Biol. 340, 1-13) and synthesized previously (U.S. Patent 5,433,940). The bivalent conjugate of FpA has been used along with an N-terminal extension to include a binding moiety for integrins on platelets (Mu, R., Qin, Y., Cha, Y., and Jing, Q. (2002) Zhonghua Yi. Xue. Za Zhi. 82, 593-596).<br><br>
It would thus be highly desirable to be provided with peptide inhibitors of thrombin with good binding affinity and composed of genetically-encodable natural amino acids, as these peptides can be expressed through recombinant DNA or used in gene therapy.<br>
SUMMARY OF THE INVENTION<br>
One aim of the present invention is to provide new peptide inhibitors of thrombin composed of genetically-encodable natural amino acids, as these peptides can be expressed through recombinant DNA or used in gene therapy.<br>
In accordance with the invention, there is proposed to use the template of a tetra-peptide sequence, Phe-Asn-Pro-Arg (SEQ ID N0:3), as the inhibitory element binding to the S-subsites of the thrombin active site, which is linked to the C-terminal fragment of hirudin or other related peptide fragments that bind specifically to the Exosite I of thrombin. The linker can be either the natural amino acid sequences of hirudin, a polyglycine moiety or other suitable polypeptides that bind to the S'-subsites of thrombin.<br>
In accordance with the present invention, there is provided a polypeptide inhibitor of thrombin obtained by the method described herein.<br>
Still in accordance with the present invention, there is provided an isolated or purified polypeptide inhibitor of thrombin consisting of all naturally-occurring amino acids, said inhibitor having general formula (1) of:<br>
SBM-PBM-EBM	(1)<br>
Wherein<br>
-	SBM is a sequence moiety comprising P-subsite residues of the thrombin<br>
inhibitor,<br>
-	PBM  is a  sequence moiety comprising  P'-subsite  residues of the thrombin inhibitor,<br>
-	EBM   is  a  peptide  sequence  moiety that  binds  to  the  fibrinogen recognition exosite of thrombin<br>
or a pharmaceutically acceptable salt thereof.<br>
Preferably, SBM has a sequence comprising Phe-Xaa-Pro-Arg (SEQ ID NO:39) or Trp-Xaa-Pro-Arg (SEQ ID N0:121) at the carboxy-<br><br><br><br><br><br><br><br><br><br><br><br>
Pro-Ile-Pro-Arg-Pro-Gln-lle-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-lle-Pro-Glu-Glu-Tyr-Leu-GIn (SEQ ID NO:110),<br>
Pro-His-Pro-Arg-Pro-Gln-lle-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-lle-Pro-Glu-Glu-Tyr-Leu-GIn (SEQ ID NO:111),<br>
Pro-Trp-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-lle-Pro-Glu-Glu-Tyr-Leu-GIn (SEQ ID N0:112),<br>
Pro-Asn-Pro-Arg-Pro-GIn-Val-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-lle-Pro-Glu-Glu-Tyr-Leu-GIn (SEQ ID N0:113),<br>
Thr-lle-Pro-Arg-Pro-Gln-lle-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-lle-Pro-Glu-Glu-Tyr-Leu-GIn (SEQ ID N0:114),<br>
Asn-Asp-Pro-Arg-Pro-Gln-Lys-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-lle-Pro-Glu-Glu-Tyr-Leu-GIn (SEQ ID NO:115), and<br>
Asn-Ser-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-GIy-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-GIn (SEQ ID N0:116).<br>
In accordance with the present invention, there is also provided a method for selecting a polypeptide inhibitor of thrombin consisting of all naturally occurring amino acids and possessing a strong anti-clotting activity, said method comprising the steps of:<br>
a) generating  a   phage-displayed   library of  polypeptides  of sequence: SBM-PBM-EBM, wherein,<br>
SBM has a sequence selected from the group consisting of Xaa-Xaa-Pro-Arg, Xaa-Xaa-Xaa-Pro-Arg, and Xaa-Xaa-Xaa-Xaa-Pro-Arg; PBM has a sequence selected from the group consisting of Pro-Aaal-Xaa-Aaa2-Aaa3-Aaa4-Aaa5 and Pro-Aaa1-Aaa2-Aaa3-Aaa4-Aaa5-Aaa6, wherein Xaa is a random (i.e. variable) natural amino acid and Aaa1, Aaa2, Aaa3, Aaa4, Aaa5 and Aaa6 represent fixed natural amino acids in the librairy; and<br>
EBM has a sequence selected from the group consisting of Asp-Phe-GIu-Glu-IIe-Pro-Glu-Glu-Tyr-Leu-GIn (SEQ ID NO:64). Asp-Phe-GIu-Pro-lle-Pro-Glu-Glu-Tyr-Leu-GIn (SEQ ID NO:65), Asp-Tyr-Glu-Pro-lle-Pro-Glu-Glu-Tyr-Leu-GIn (SEQ ID NO:66), Asp-Phe-Glu-Pro-lle-Pro-Glu-Glu-Tyr-Leu (SEQ ID NO:67), and Asp-Tyr-GIu-Pro-IIe-Pro-Glu-Glu-Tyr-Leu (SEQ ID NO:68);<br><br>
b)	immobilizing active thrombin on a surface;<br>
c)	contacting said library withi the immobilized thrombin;<br>
d)	removing the non-bound and/or cleaved phage particles;<br>
e)	propagating bound and uncleaved phage particles remaining after step (d); and<br>
f)	identifying the sequence of a polypeptide from the propagated phage particles, said polypeptide being an inhibitor of thrombin.<br>
In accordance with the present invention, there is also provided expression cassettes comprising, in addition to the required sequences for obtaining expression in a target nucleic acid sequence, a nucleic acid sequence encoding the polypeptide inhibitors of the present invention, operably linked to the aforementioned required sequences.<br>
Of course, the present invention is to also include recombinant hosts containing one or more of such expression cassettes, as well as the use of such cassettes, recombinant hosts and the polypeptide inhibitors.<br>
For the purpose of the present invention the following terms are defined below.<br>
When making reference to P-subsite and P'-subsite in reference to residues of the thrombin inhibitor, the p- and p'-subsites refers to or represent a division of the peptide sequence at the peptide bond hydrolyzed specifically by thrombin<br>
The tem: "good affinity" in reference to an inhibitor is intended to mean an inhibitor that has an affinity good enough for commercial application. In some cases, some inhibitors may have an affinity, but the affinity is so low that these inhibitors would not be industrially useful in commerce.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Fig. 1 illustrates a comparison of the basic compositions of the new bivalent inhibitors of thrombin in accordance with the present invention, hereinafter named, FX22, and their inhibition activities, with respect to a hirudin peptide, Bivalirudin and P53;<br>
Fig. 2 illustrates an SDS-PAGE analysis of the expression of FN22 as a fusion protein in E. coli BL21 (DE3) cells;.<br><br>
Fig. 3 illustrates an SDS-PAGE analysis of the expression of FD22 as a fusion protein in E. coli BL21 (DE3) cells;<br>
Fig. 4 illustrates an HPLC purification profile of the crude peptide after the FN22 fusion protein was digested by CNBr;<br>
Fig. 5 illustrates an HPLC purification profile of the secx&gt;nd step purification of FN22;<br>
Fig. 6 illustrates an HPLC purification profile of the second step purification of FD22;<br>
Fig. 7 illustrates progression curves of clotting of human fibrinogen by hunnan thrombin in the presence of varying concentrations of hinjdin54-85<br>
Fig. 8 illustrates progression curves of clotting of human fibrinogen by human thrombin in the presence of varying concentrations of FN22;<br>
Fig. 9 illustrates progression curves of clotting of human fibrinogen by human thrombin in the presence of varying concentrations of FD22;<br>
Fig. 10 illustrates progression curves of clotting of human fibrinogen by human thrombin in the presence of varying concentrations of Bivalinjdin;<br>
Fig. 11 illustrates inhibition of thrombin by hirudin54-85 analyzed with the clotting assay;<br>
Fig. 12 illustrates inhibition of thrombin by FN22 analyzed with the clotting assay;<br>
Fig. 13 illustrates inhibition of thrombin by FD22 analyzed with the clotting assay;<br>
Fig. 14 illustrates inhibition of thrombin by Bivalirudin analyzed with the clotting assay;<br>
Fig. 15A illustrates the amino acid sequences of bivalent peptide inhibitors of thrombin. Sequences of the FN22 and FD22 peptides in comparison with two related peptides, P53 and hirulog, both of which contain a residue in the (d)-configuration, i.e. dF or (d)Phe;<br>
Fig. 15B illustrates two primers used to synthesize the gene encoding the FN22 peptide;<br><br>
Fig 16A illustrates the preparation and ciiaracterization of recombinant FN22 and FD22 peptides (collectively referred to as FX22) using the expression vector (pi\/lFH) for their preparation;<br>
Fig. 16B illustrates an SDS-PAGE analysis of the expressed fusion proteins containing FN22 and FD22; In Fig. 16B, lane M is loaded with molecular weight markers (LMW, Amersham Bioscience); lane 1 is loaded with cell lysate without IPTG induction; lane 2 is loaded with cell lysate with IPTG induction, wherein the total protein of cells harbouring pMFH-FN22 was prepared by dissolving the cell pellets in the SDS sample buffer; lanes 3 &amp; 4 are loaded with cell lysate with IPTG induction, wherein the total protein of cells harbouring pMFH-FN22 was prepared by dissolving the cell pellets in Buffer A under a denaturing condition; lane 5 &amp; 6 are loaded with cell lysate with IPTG induction, wherein the total protein of cells harbouring pMFH-FD22 was prepared by dissolving the cell pellets in Buffer A under a denaturing condition;<br>
Fig. 16C illustrates a HPLC purification profile of the FN22 peptide after CNBr treatment and cleavage of the fusion protein, where peak 1 with a retention time of 17.16 min was collected and confirmed to contain the intact FN22 peptide by mass spectroscopy;<br>
Fig. 16D illustrates a ^H-'^N HSQC spectrum of ^^N-labelled FD22 peptide, wherein the HSQC cross-peaks of residues D45, Q49 and D53 are marked with asterisks and can only be seen at lower contour levels.<br>
Figs. 17A to 17D illustrates inhibition curves of thrombin by hirudin^'*"^^(Fig. 17A) FN22 (Fig. 17B), FD22 (Fig. 17C) and Hirulog (also called bivalirudin) (Fig. 17D) at various increasing peptide concentrations;<br>
Fig. 17E illustrates the transition clotting time as a function of the peptide concentration for Himdin^'*'^^ (so//of square), FN22 {open square), FD22 {solid circle) and Hirulog {open circle);<br>
Fig. 17F illustrates the Dixon plot of the inhibition of human a-thrombin-catalyzed hydrolysis of Tos-Gly-Pro-Arg-pNA by FD22 wherein the assays as described herein were performed in the presence of 5 {open square), 10 {open circle), 20 {full square), 50 {full circle) and 80 pM {open circle) of the chromogenic substrate;<br><br>
Figs. 18A and 18B illustrates HPLC profiles (on a Ce column) after 2.0 pM of FN22 and FD22 peptides was incubated with 2.0 |JM liuman a-thrombin for 40 lirs at 25°C, respectively;<br>
Fig. 18C illustrates an analysis of the stability of the FN22 and FD22 peptides in the presence of human a-thrombin by the clotting assay;<br>
Fig. 19A illustrates the cloning sites as DNA sequences used for the construction of the phage-displayed bivalent peptide library;<br>
Fig. 19B illustrates the sequences of nine randomly-selected clones from a small scale production of the phage library of Fig. 19A;<br>
Fig. 20A illustrates inhibition curves of thrombin by synthetic WD22-R-P (SEQ ID NO:22) at various increasing peptide concentrations;<br>
Fig. 20B illustrates the transition clotting time as a function of the peptide concentration for synthetic WD22-R-P (SEQ ID NO:22), wherein two independent sets of data are plotted therein;<br>
Fig. 21A illustrates inhibition curves of thrombin by synthetic WD22-R-YP (SEQ ID NO:23) at various increasing peptide concentrations;<br>
Fig. 21B illustrates the transition clotting time as a function of the peptide concentration for synthetic WD22-R-YP (SEQ ID NO:22), wherein two independent sets of data are plotted therein;<br>
Fig. 22A illustrates inhibition curves of thrombin by recombinant WD22-R-P (SEQ ID NO:22) at various increasing peptide concentrations;<br>
Fig. 22B illustrates the transition clotting time as a function of the peptide concentration for recombinant WD22-R-P (SEQ ID NO:22);<br>
Fig. 23A illustrates inhibition curves of thrombin by recombinant WD22-R-YP (SEQ ID NO:23) at various increasing peptide concentrations; and<br>
Fig. 23B illustrates the transition clotting time as a function of the peptide concentration for recombinant WD22-R-YP(SEQ ID NO:23).<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
Structurally, the tripeptide, (d)Phe-Pro-Arg, mimics the specific binding of human fibrinopeptide A to thrombin. The aromatic side chain of the (d)Phe residue occupies a binding subsite on thrombin for an (L)Phe at the Pg site, i.e. 8 amino acids away to the N-terminal side of the Arg(Pi)-Gly(Pi') peptide bond (Ni. F., Zhu, Y., and Scheraga, H. A. (1995) J. Mol.<br><br>
Biol. 252, 656-671; Ni, F., Meinwald, Y. C, Vasquez, M., and Scheraga, H. A. (1989) Biochemistry 28. 3094-3105; and Stubbs, M. T., Oschkinat, H., Mayr, I., Huber, R., Angliker, H., Stone, S. R.. and Bode, W. (1992) Eur. J. Biochem. 206, 187-195). Alternatively, the same binding subsites on thrombin can accommodate tetrapeptide sequences as long as there is an aliphatic, or more preferably an aromatic residue, at the P4 subsite. The P4 residue of the tetrapeptides fulfills the binding of the natural (L)Phe residue at the Pg position of fibrinopeptide A (FpA) (Ni, P., Zhu, Y., and Scheraga, H. A. (1995) J. Mol. Biol. 252, 656-671; Ni, P., Ripoll, D. R., and Purisima, E. O. (1992) Biochemistry 31, 2545-2554; and Rose, T. and Di Cera, E. (2002) J. Biol. Chem. 277, 18875-18880). Some bivalent peptides, although weak binding, have been derived from the activation sequences of thrombin-activated receptors, also referred to as protease-activated receptors (PARs). These peptides carry the LDPR (SEQ ID N0:1) sequence and binding motifs targeting the fibrinogin-recognition exosite of thrombin (Liu, L. W., Vu, T. K., Esmon, C. T., and Coughlin, S. R. (1991) J. Biol. Chem. 266, 16977-16980; and Mathews, I. I., Padmanabhan, K. P., Ganesh, V., Tulinsky, A., Ishii, M., Chen, J., Turck, C. W., Coughlin, S. R., and Fenton, J. W. (1994) Biochemistry 33, 3266-3279). More recent work using substrate libraries has shown that thrombin preferentially cleaves after tetrapeptides carrying the L/FXPR sequence motif (Backes, B. J., Harris, J. L., Leonetti, F., Craik, C. S., and Ellman, J. A. (2000) Nat. Biotechnol. 18, 187-193; Edwards, P. D., Mauger, R. C, Cottrell, K. M., Morris, F. X., Pine, K. K., Sylvester, M. A., Scott, C. W., and Furlong, 8. T. (2000) Bioorg. Med. Chem. Lett. 10, 2291-2294; and Furlong, S. T., Mauger, R. C, Strimpler, A. M., Liu, Y. P., Morris, F. X., and Edwards, P. D. (2002) Bioorg. Med. Chem. 10, 3637-3647). Indeed, there is an FNPR (SEQ ID N0:3) tetrapeptide sequence in human prothrombin, which is cleaved off efficiently by thrombin to generate the free N-terminus of the A-chain of human thrombin (Ni, P., Zhu, Y., and Scheraga, H. A. (1995) J. Mol. Biol. 252, 656-671; and Rose, T. and Di Cera, E. (2002) J. Biol. C/7em. 277, 18875-18880).<br>
In accordance with the present invention, P4-P1 tetrapeptides, such as F-D-P-R (SEQ ID N0:4) or W-D-P-R (SEQ ID N0:5) have now been incorporated into bivalent and bridge-binding inhibitors of thrombin in<br><br>
replacement of the (d)F-P-R moiety. Rational design and combinatorial selection led to peptides with strong antithrombin and anticlotting activities in the low nanomolar range comparable with those of bivallrudin or P53. The availability of the potent and genetically-enoodable polypeptide inhibitors of thrombin of the present invention opens the door for much broader applications of this clinically-successful class of anticoagulants, e.g. through more cost-effective recombinant peptide production, in areas such as gene therapy as well as to improve clinical efficacy/safety through the incorporation of homing peptides for targeted delivery.<br>
In the present invention, the Phe-Asn-Pro-Arg sequence (SEQ ID NO:3) was first substituted for the active site binding element of the P53 bivalent peptide inhibitor of thrombin {DiMaio, J., Gibbs, B., Munn, D., Lefebvre, J., Ni, F., and Konishi, Y. (1990) J. Biol. Chem. 265, 21698-21703). The resulting new bivalent peptide served as a template for studying the nature of the P4, P3 and P3' sites required for enhanced thrombin inhibition. Furthermore, a phage-displayed library of bivalent peptides was constructed in order to explore residue preferences at the P4, P3, and P3' sites that confer strong bivalency for thrombin binding and at the same time resistance against thrombin cleavage. Synthetic peptides derived from the phage-selected sequences were found to have unique and rather unexpected antithrombin and anticoagulant activities.<br>
Fig. 1 represents the basic compositions of the new inhibitors of thrombin in accordance with the present invention, hereafter also referred to as FX22. A comparison of their compositions with Bivallrudin and P53 was also indicated.<br>
Specific examples of the novel inhibitors are called FN22, FD22, FD22-R, WD22 and WD22-R. The doubling clotting time, DCT, of these inhibitors are also listed in Fig. 1 and Table 1 along with other examples.<br>
The new peptide inhibitors of thrombin of the present invention can be produced cost-effectively through recombinant DNA technologies.<br>
In Fig. 2, the expression was induced with ImM IPTG when the cell density reached 0.8 to 1.0 O.D.eoo in LB medium. The cells were harvested after an additional three hours of growth. In Fig. 2, lane M was loaded with low molecular weight markers, lane 1 was loaded with the total cell extracts, whereas lanes 2 and 3 were loaded with the supernatant and<br><br>
the pellet from the fraction purification of the cell lysis. The expressed FN22 fusion proteins were indicated by the arrow.<br>
In Fig. 3, the expression was induced with 1 mM IPTG when the cell density reached 0.8 to 1.0 O.D.eoo in LB medium. The cells were harvested after an additional three hours of growth. In Fig. 3, lane M was loaded with low molecular weight markers, lane 1 was loaded with the cell extracts without IPTG induction, whereas lanes 2 to 3 were loaded with the cell extracts with the IPTG induction, lanes 5 and 6 were loaded with the products of the partial purification and lanes 7 to 9 were loaded with different fractions from the purifications by Sep-Pak™. The expressed fusion proteins were indicated by the arrows.<br>
In Fig. 4, the reversed phase semi-preparative column (C18) was used and the sample was eluted with a concentration gradient of acetonitrile from 10% to 70% with a flow rate of 5 ml/min. The retention time of FN22 is 17.15 min. (the peak was indicated by the asterisk in the figure. The wavelength of the detector was set at 278 nm.<br>
In Fig. 5, the reversed phase semi-preparative column (C18) was used and the sample was eluted with a concentration gradient of acetonitrile from 10% to 70% with a flow rate of 5 ml/min. The wavelength of the detector was set at 215 nm. The fraction between the two arrows was collected and the lyophilized product was tested for the inhibition of thrombin.<br>
Similarly in Fig. 6, the reversed phase semi-preparative column was used and the sample was eluted with a concentration gradient of acetonitrile from 10% to 70% with a flow rate of 5 ml/min. The wavelength of the detector was set at 215 nm. The fraction between the two arrows was collected and the lyophilized product was tested for the inhibition of thrombin.<br>
The DOT reported in Fig. 1 were determined from data extracted from Figs. 11 to 14.<br>
The amino acid composition of the Phe-Asn-Pro-Arg or Phe-Asp-Pro-Arg sequences, specifically the Phe-Asn or Phe-Asp residues was also varied in order to achieve a control between potent thrombin inhibition and reversible dissociation (through cleavage of the peptides, see Fig. 1 and the Results Section).<br><br>
Materials and methods<br>
Materials<br>
All the restriction enzymes were purchased from New England Blolabs (MA, USA). The T4 DNA ligase was obtained from Amersham Biosciences (NJ, USA). Bovine and human a-thrombins were supplied by Haematologic Technologies Inc (VT, USA). The stock solution of human a-thrombih was 12.6 mg/ml In concentration with a specific activity of 3300 NIH units/mg. The stock solution of bovine a-thrombin was 14.6 mg/mj in concentration with a specific activity of 3290 NIH units/mg. The chromogenic substrate Tos-Gly-Pro-Arg-pNA (Tos is Tosyl, and pNa is p-. nitroalanine), poly(ethylene glycol)-8000, clottable bovine fibrinogen and the C-terminal peptide of hirudin (Hirudin 54-65) were purchased from Sigma. Peptide preparation<br>
The two leading peptides, referred to as FN22(a) and FD22(a) (Table 1 hereinafter), were prepared with a recombinant DNA approach essentially as described previously (Osborne, M. J., Su, Z., Sridaran, V., and Ni, F. (2003) J. Biomol. NMR 26, 317-326). The carrier protein termed MFH (Fig. 16A), a variant of the SFC120 fusion carrier (Osborne, M. J., Su. Z., Sridaran, V., and Ni, F. (2003) J. Biomol. A/MR 26, 317-326), was used to fuse with the targeted peptide. Essentially, MFH is a methionine-free mutant of SFC120 with a six-histidine tag. Other peptides were synthesized by the solid-phase method using FMOC-chemistry either at the Sheldon Biotechnology Centre of McGill University or at the Peptides Facility of the Biotechnology Research Institute, except where indicated.<br>
DNA fragments encoding recombinant peptides were prepared by annealing and amplifying two oligonucleotide primers through the standard FOR procedure. The primers were designed by use of the codon preference of £ Coli. PCR products were purified by the PCR clean-up kit from Qiagen and double digested with EcoR / and BamH I. The DNA inserts were ligated into the expression vector. The expression constructs were confirmed by DNA sequencing and transformed into the E coli BL21 host strain for expression.<br>
Expression of fusion proteins was achieved by transformation of the plasmid into E. coli BL21 (DE3) competent cells. A 50 ml of overnight<br><br>
culture grown in LB containing 100 pg/ml amplcillin was used to inoculate 1 L of LB medium supplemented with 100 pg/ml amplcillin. 15N-labeled peptides were expressed in M9 medium using 15(NH4)2S04 (1 g/L) as the sole nitrogen source. The cells were grown at 37°C to a density of ODeoo = 0.8 and induced by adding IPTG to a final concentration of 2 mM. The induced cells were Incubated for 12 h at 37°C and collected by centrifugatlon (8000 rpm for 20 min). The cell pellet was frozen at -20'C before further processing.<br>
Thawed cell pellets were resuspended in 6 M urea in 20 mM Tris, 100 mM NaCI buffer, pH 8.0 for 20 min and then sonicated for one minute on ice. The solution was then centrifuged at 7,000 rpm for 20 min. The supernatant was subjected to purification by Ni-NTA affinity chromatography under a denaturing condition. The eluate containing fusion protein was applied to a Sep-Pak column to remove salts. The purified fusion protein was then lyophilized.<br>
CMBr cleavage was used to release the target peptide from the fusion protein. The fusion protein was dissolved in 0.1 M HCI and 6 M guanidine hydrochloride (10 mg protein / ml). Crystalline CNBr was added to a final molar ratio of 100:1 of the fusion protein. The solution was allowed to stand for 12 ~ 24 hours. The samples were then purified with Ni-NTA beads to remove MFH fusion carrier and undigested fusion protein if any. The flow-through was desalted, lyophilized and confirmed by electrospray mass spectrometry and ^H-^^N heteronuclear single-quantum correlation (HSQC) NMR experiments.<br>
Both the recombinant and synthetic peptides were finally purified using HPLC on a Ci8 reverse-phase column with a water-acetonitrile gradient with added 0.1% trifluoroacetic acid. For the synthetic peptides, 3-5 mg of the crude materia! were dissolved in 1 ml of 0,1% TFA in water and the peptide solutions were filtered through a 0.45 pm filter. The filtered solution was applied on a Ci8 HPLC column pre-equilibrated with the water solution of 0.1% TFA. Identities of all purified peptides were verified by electrosprary mass spectrometry.<br>
Clotting assays<br><br>
The clotting assays were carried out by use of tlie protocols described previously (DiMaio, J., Gibbs, B., Munn, D., Lefebvre, J., Ni, F., and Konishi, Y. (1990) J. Biol. Chem. 265. 21698-21703; and Witting, J. I., Bourdon, P., Breznial^;, D. V., Maraganore, J. M., and Fenton, J. W. (1992) Biochem. J. 283 ( R 3), 737-743) The assay employs bovine plasma fibrinogen dissolved at 0.1% in 50 mM Tris-CI, 100 mM NaCI, 0.1% PEG-8000 at pH 7.6 (i.e. the clotting buffer). Each assay mixture contained a certain concentration of the peptide, and the reaction was started by the addition of human thrombin to a final concentration of 0.1 NIH unit/ml. Optical absorbance of the assay mixtures was measured at 420 nm and 37°C using the Spectramax™ plate reader. The onset clotting time was determined as an intersection of the baseline and the initial slope of the O.D. change as a result of fibrin clot fomiation. The concentration of a peptide needed to double the clotting time (DCT) was defined as IC50. Peptide concentrations were determined by comparing the O.D.280 values of peptide stock solutions in the clotting buffer (vide supra) with the predicted extinction coefficient for each peptide (Gill, S. C. and von Hippel, P. H. (1989)/\na/. Biochem. 182, 319-326).<br>
Inhibition of the amidolydic activity of thrombin<br>
Kinetics of thrombin-catalyzed hydrolysis of the chromogenic substrate Tos-Gly-Pro-Arg-p-nitroaniiide were followed by absorbance at 405 nm on a Spectramax plate reader thermostated at either 25°C or 37°C according to the method of Maragnore et al (supra). The concentrations of Tos-Gly-Pro-Arg-p-nitroanilide ranged from 2 to 400 pM. Initial rates were calculated under conditions of 
 <br>
and the total volume was adjusted to 120 pL using the clotting buffer. Eighty microliters of human a-thrombin in the clotting buffer (~ 0.3 niVl final concentration) was added to initiate the reaction. The concentration of the peptides ranged from 0.5 nM to 10 pM. Kinetics data from initial rate experiments were used to construct Lineweaver-Burke plot; i.e. the relationship of (substrate concentration)"^ versus (initial velocity)'^ which were analysed by linear regression with MicroCal Origin 4.1 program (MicroCal, MD) or using the GraphPad Prism software. The Kj values of the inhibitors were determined using the equation:<br>
K, = [!]/{(SLo/SLi)-i}. where      [I] is the inhibitor concentration,<br>
SLo is the slope of the reaction in the absence of inhibitors, and SLi is the slope of the reaction in the presence of the inhibitor. Alternatively, Dixon plots of inverse velocity versus the inhibitor concentrations were generated with the SigmaPlot™ software. Every Dixon plot contained an array of data corresponding to different substrate concentrations and the K/values were determined from the intersections.<br>
Following peptide cleavage by HPLC<br>
Previously reported methods (DiMaio, J., Gibbs, B., Munn, D., Lefebvre, J., Ni, F., and Konishi. Y. (1990) J. Biol. Chem. 265, 21698-21703; and Witting, J. I., Bourdon, P., Brezniak, D. V., Maraganore, J. M., and Fenton, J. W. (1992) Biochem. J. 283 (Pt3), 737-743) were employed with some modifications (as detailed in the Results section and Fig. 18) to determine whether some bivalent peptides are cleaved by thrombin. Each peptide was dissolved in the clotting buffer at a final concentration of 2.0 pM and incubated at 25°C with or without 2 pM of human a-thrombin for up to 40 hrs. The samples were then boiled for 2 min to stop the reaction, and centrifuged. The supematant was acidified by addition of concentrated acetic acid before being loaded on an analytical HPLC Ci8 column and eluted with a linear gradient of 0% - 50% (v/v) acetonitrile.<br>
NMR experiments<br>
HMR spectra were acquired with Bruker 500 or 800 MHz NMR spectrometers     using     standard     pulse     sequences     (Mori,      S.,<br><br>
Abeygunawardana, C, Johnson, M. O., and van ZijI, P. C. (1995) J. Magn Reson. B 108, 94-98). Heteronuclear NMR experiments including HSQC (2D). HSQC-N0ESY(3D) and HSQC-T0CSY(3D) were carried out on the 800 IViHz HMR spectrometer. Spectral processing, display and analysis were performed using the XwinNIVIR software package supplied with the spectrometer system. Sequence-specific assignment of the peptide HSQC spectmm was carried out with the NMRview 4.0 software program.<br>
Construction of phage library<br>
The phage vector fd-tetGIIID (MacKenzie, R. and To, R. (1998) J. Immunol. Methods 220. 39-49) was a generous gift from Dr. R. MacKenzie (Institute of Biological Sciences, Ottawa). The phage library was constructed essentially as described by Tanha et at. (Tanha, J., Xu, P., Chen, Z., Ni, F., Kaplan, H., Narang, S. A., and MacKenzie, C. R. (2001) J. Biol. Chem. 276, 24774-24780). Briefly, double-stranded DNA fragments, encoding the amino acid sequences to be displayed on phage particles were generated and amplified by PCR using a mixture of four synthetic and partly complementary synthetic DNA primers: 1) 5'-catgaccaca gtgcacagca ccaccaccat caccatggct ctggc-3 (SEQ ID N0:6)', 2) 5'-ttcctcaaaa tcaccgtcgt tatgmnnttg agggcgcggm nnmnnagagc cagagccatg gtgatg-3' (SEQ ID NO:7), 3) 5'-aacgacggtg attttgagga aattcctgaa gagtatttac aaggtggt-3'(SEQ ID N0:8), and 4) 5'-cgattctgcg gccgcagaag aaccaccttg taaatactc-3'(SEQ ID NO:9). The concentration of the primers in the PCR reaction mixture was 10 pM for primers 1) and 4), and 0.1 pM for primers 2) and 3). The resulting DNA fragments encode a bivalent peptide library with an N-tenninal Hise-tag and randomized at a number of residue locations (see Fig. 198). In addition, the fragments contained ApaU and AfofI restriction sites at the N-terminal and C-terminal ends, respectively. The PCR product and the phage vector were purified, digested by ApaU and Not\, purified again and ligated. The ligation mixture was desalted using the Nanosep™ concentrator (Pall Gelman Laboratory). Subsequently, 0.21 pg of the desalted and ligated product was mixed with 50 pi of competent Escherichia coll strain TG1 (Stratagene, CA) and the cells were transformed by electroporation. Transformed cells were plated, and the produced phage was collected, purified and stored as reported<br><br>
elsewhere (Tanha, J., Dubuc, G., Hirama, T., Narang, S. A., and MacKenzie, C. R. (2002) J. Immunol. Methods 263, 97-109). A total of 1.7 |jg of the ligated product yielded a library of approximately 1.0x10^ transformants, which allowed -99 % sampling of the library (Deperthes, D. (2002) BioL Chem. 383, 1107-1112). The transformation efficiency was determined by plating a portion of cells onto LB plates containing 50 pg/ml tetracycline.<br>
Selection of proteolytically stable peptides from the phage library<br>
Phage particles (10^) were incubated with 0.5 pg of human thrombin at ZT'C for 30 minutes in 26 pi of PBS buffer, pH 7.4. The proteolytic reaction was stopped with an excess of the inhibitor PPACK (Calbiochem) specific for the thrombin active site. The reaction mixture was mixed with 700 pi of the Ni-NTA agarose resin (QIAGEN) in PBS (50% slurry), and phage particles were allowed to bind with gentle agitation for two hours at O'C. Cleaved phage particles were separated from the resin by washing with 9 ml of PBS buffer, pH 7.4. Bound phage particles were eluted from the resin by 0.7 ml of PBS adjusted to pH 4.4, and immediately neutralized by the addition of 30 pi of 1 M Tris-HCI, pH 8.0. Exponentially growing TGI cultures (0.3 ml) were infected with the eluted phage at 37°C for 30 minutes. Serial dilutions were used to estimate phage recovery.<br>
Panning the phage library against human thrombin<br>
Individual wells of MaxiSorp plates were coated with 150 pi of 80 pg/ml human a-thrombin in PBS buffer, pH 7.4, at 4°C under shaking for 2 hours. Wells were rinsed three times with PBS, blocked with 400 pi PBS-2% (w/v) skim milk (2% MPBS) at 4°C for 2 hours, and rinsed as above. Two hundred microliters of the phage particles (-10"12 plaque forming units) in 2% MPBS were added to the thrombin-coated wells and incubated with shaking either for 2 hours at 4°C or for 0.5-1 hours at room temperature (25°C) without shaking. The wells were rinsed 15 (or 25) times with PBS-0.1% (v/w) Tween™ 20 and then 15 (or 25) times with PBS at the corresponding temperature. Bound phage was eluted by adding 200 pi of freshly prepared 100 mM triethylamine, neutralized with 100 μ 1 M Tris-<br><br>
HCI, pH 7.4, and used to infect TG1 cells as described above. Alternatively, TG1 cells were infected directly by the addition of 300 |jl cell culture in the wells (Stoop, A. A. and Craik, C. S. (2003) Nat. Biotechnol. 21.1063-1068).<br>
Results<br>
1: Incorporator! of the tetrapeptide Phe-Asn-Pro-Arg (SEQ ID NO:3) into bivalent inhibitors of thrombin: recombinant expression and characterization<br>
The Phe-Asn-Pro-Arg peptide fragment (SEQ ID NO:3) was conjugated to a peptide derived from the C-terminus of hirudin (or hirudin 48-85 as done previously (DiMaio, J., Gibbs, B., Munn, D., Lefebvre, J., Ni, F., and Konishi, Y. (1990) J. Biol. Chem. 265, 21698-21703). The resulting 22-residue chimeric peptide was designated FN22 and contained one proline residue inserted between the two parts in order to decrease the rate of peptide cleavage by thrombin (Fig. 1 and Fig. 15A). The FN22 peptide was then expressed as a fusion protein to a mutant, named MFH, of the SFC120 fusion carrier (Osbome, M. J., Su, Z., Srldaran, V., and Ni, F. (2003) J. Biomol. NMR 26, 317-326). The mutant carrier had all the methionine residues of SFC120 replaced by leucines with a His-tag attached in order to simplify peptide purification (Fig. 2 and Fig. 16A). A single methionine residue was inserted between the carrier protein and the FN22 peptide sequence to facilitate release of the peptides by CNBr cleavage.<br>
In Fig. 15A, all peptides contain three distinctive segments, the active site binding moiety, the fibrinogen-recognition exosite binding moiety and a linker sequence. The FN22 and FD22 peptides were prepared by recombinant DNA techniques as described herein (Figs. 2-6 and Fig. 16).<br>
The recombinant FN22 peptide had an enhanced anticlotting activity as it further increased the time of fibrinogen clotting catalysed by a-thrombin (Figs. 8, 178, 17D 17E and Table 1) compared to the hirudin peptide (Figs. 7, 17A, 17E and Table 1). The FN22 peptide showed inhibitory activities in the concentration range from 50 nM to 6 pM (Figs. 8, 17A and 17E). The IC50 values for FN22 determined was 289 nM for bovine a-thrombin and 150 nM for human a-thrombin (Fig. 1 and Table 1). In comparison, a hirudin tail peptide has an IC50 of 447 nM for human a-<br><br>
thrombin (Fig.1 and Table 1). Similar experiments showed that the FNPRP peptide alone was a much weaker inhibitor (Table 1) with submilllmolar IC50 values.<br>
In each of Figs. 17A to 17F, the leftmost curve represents the clotting progress curve in the absence of the peptide and others appear from left to right with increasing peptide concentrations. In Fig. 17A, the . concentrations of the hirudin peptide were 0 p-M, 0.1 yM, 0.2 \xM, 0.5 nM, 0.7 \xM and 1.0 \M, respectively. In Fig. 17B, the peptide concentrations of FN22 were 0 ^M, 0.04 }xM, 0.08 ^M, 0.12 ixM, 0.2 ^M, 0.4 nM, 0.8 jiM and 1.0 ^M, respectively. In Fig. 17C, the peptide concentrations of FD22 were 0 nM, 0.01 \iM, 0.02 ^M, 0.04 ^iM, 0.08 nM, 0.1 jilVI, 0.3 ^iM and 0.5 \M, respectively. In Fig. 17D, the peptide concentrations of Hinjlog.(also called bivalirudin, Fig. 1) were 0 nM, 1.35 nM, 2.7 nM, 13.5 nM and 27 nM, respectively.<br>
In Fig 17E, In the absence of the peptides, the onset of the clotting time for all assays is around 75 s. The peptide concentration for doubling the clotting time (DCT) was defined as the IC50 value. The IC50 value of each peptide was determined by the peptide concentration when the clotting time was 150 s and is listed in Fig. 1 and Table 1.<br><br><br><br><br>
(a). These two peptides were prepared by use of a recombinant DNA procedure (see Rgs. 2-6,15A, 15B and<br>
16A).<br>
(b). No delay in clotting time up to the peptide concentrations shown<br>
(c). The Ki values shown were detenriined based on competitive inhibition achieved when the inhibitor<br>
concentrations were larger than 20 nM. if [I] 
inhibition.<br>
(d) These two peptides were prepared by use of a recombinant DNA procedure (see Fig. 16A). Their<br>
anticoagulant activities are siiown in Figs. 22 and 23.<br>
(e). Data from the published reports of DiMaio (supra) and Maraganore (supra).<br>
The FN22 peptide also inhibited a-thrombin-catalysed hydrolysis of the chromogenic substrate, Tos-Gly-Pro-Arg-p-nitrioaniiide, at submicromolar concentrations (Table 1). Under the same experimental conditions, micromolar concentrations of the himdin tail peptide exhibited no measurable inhibition of thrombin-catalyzed substrate turnover (Table 1). Likewise, the presence of the FNPRP pentapeptide (SEQ IS N0:11) at concentrations as high as 100 pM, exhibited no significant inhibition. These observations indicated that at micromolar concentrations, the individual components of FN22, i.e. the active site- and the FRE-directed moieties, were unable by themselves to inhibit the thrombin hydrolysis of the tripeptide substrate. In contrast, the combination of these components in a single polypeptide as in FN22 resulted in potent inhibition of the thrombin active site.<br>
The inhibition constant (/
 <br>
hydrolysis of the chromogenic substrate increased from 4.2 pM to 6.4 pM and the kcat decreased from 130 to 115 s"The FN22 peptide was incubated with thrombin at 25 ^'C in order to investigate the stability of the peptide in complex with thrombin. The progress of peptide cleavage by thrombin was monitored and analyzed through the HPLC profiles of the FN22 peptide in the absence or presence of thrombin for different incubation times. It was found that about 30 % of the peptide was cleaved by thrombin after 40 hours (Fig. 18A). In the clotting assay, the FN22 peptide at a concentration of 400 nM was premixed with thrombin for different times of incubation before initiating the clotting reaction. The cleavage of the peptide by thrombin is indicated by a reduced clotting time with increasing incubation times. After 30 hrs of incubation, the clotting time was reduced by 41 % compared to the control (Fig. 18C).<br>
In both Figs. 18A and 18B, the retention time for the intact peptides was 23.0 min. The large fragment produced from cleavage of the peptides by thrombin (at the Arg47-Prd48 bond, Fig. 15A) was eluted slightly earlier than the intact peptides, which appeared as a shoulder of the HPLC peak of the undigested peptide.<br>
In Fig. 18C, curve (1) represents the clotting assay curve recorded for a freshly-prepared stock solution of thrombin in the absence of peptide inhibitors, curves (2) and (3) represent the clotting curves recorded under the same condition as in (1) except that thrombin stood at 4°C and 25°C, respectively, for 30 hrs, curves (4) and (5) represent the clotting curves recorded under the same condition as that in (3) except that the FN22 peptide (400 nM, curve 4) and FD22 (100 nM, curve 5) were mixed with thrombin during incubation, and curves (6) and (7) represent the clotting cun/es recorded under the same condition as that in (3) except that FN22 (400 nM, curve 6) and FD22 (100 nM, curve 7) were freshly-prepared intact peptides.<br>
2: Asp residue at the P3 site confers enhanced bivalency for thrombin binding<br>
The inventors then studied whether a charged residue at the P3 site would enhance the inhibition of a substituted FN22 peptide to human thrombin. The  replacement by negatively charged  residues was first<br><br>
considered (e.g. Asp), as an Asp residue at this site is unfavourable for peptide cleavage (Le Bonniec, B. F. and Esmon, C. T. (1991) Proc. Natl. Acad. Sci. U. S. A 88, 7371-7375). Even so, an Asp at the P3 site can still confer binding to the thrombin active site as shown for the LDPR sequence (SEQ ID N0:1)(Ni, F., Ripoll, D. R., and Purisima, E. O. (1992) Biochemistry 31, 2545-2554). The new bivalent peptide, referred to as FD22, was prepared by use of the sanne recombinant procedure as for the FN22 peptide (Figs. 3, 6, 16A &amp; 168). Fig. 16D shows the ^H-'^N NMR heteronuclear single-quantum correlation (HSQC) spectrum of the ""^N-labelled FD22 peptide. The narrow proton chemical shift dispersion (~ 1 ppm, Fig. 16D) indicates that the FD22 peptide lacks a folded structure in solution, as shown previously for related peptides (DiMaio, J., Ni, F., and Konlshi, Y. Thrombin inhibitors based on the amino acid sequence of himdin. [2,085,465]. 2001. Canadian. 14-6-1991). Clotting assay showed that FD22 had a stronger inhibition of thrombin compared to FN22 (Figs. 9, 17C and Table 1), with an ICgoof ~ 50 nM (at 37°C) and K; values of ~ 55 nM at25°C.<br>
Compared to FN22, the FD22 peptide is much more stable against thrombin cleavage. After the peptide was incubated with thrombin at 25°C for 40 hrs, around 90% of the intact FD22 peptide still remained as shown by use of HPLC analysis (Fig. 18B). In the clotting assay, the FD22 peptide at 100 nM was premixed with thrombin for different times before initiation of clotting. The peptide thus prepared had the clotting time reduced by only 12 % after 30 hrs of incubation with thrombin. Indeed, the ^^N-labelled FD22 with a concentration of ~1 mM (Fig. 16D) was digested completely at the Arg(Pi)-Pro(Pi') bond by a 5 pM concentration of human a-thrombin only after more than 60 hrs of incubation.<br>
3: Nature ofttie P3 and P3'residues and effects on tlirombin inhibition<br>
The P3 and P3' positions of the active-site binding sequence Phe(P4)-Asp(P3)-Pro(P2)-Arg(Pi)-Pro(Pi')-Gln(P2')-Ser(P3') were then explored for their contributions to thrombin binding and inhibition by the bivalent peptide. For these experiments, both the FN22 and FD22 peptides and all other substituted ones (vide infra) were generated synthetically using  solid-phase methods.  The synthetic  FN22  and   PD22  peptides<br><br>
appear to have somewhat higher activities than the recombinant equivalents { FN22(a) and FD22(a)' in Table 1). However, this can be understood as caused by the premixing of the substrates (chromogenic or fibrinogen) with the synthetic peptide inhibitors instead of pre-mixing of the recombinant peptides with thrombin (see Figs. 7-10, 18C). These peptides should be slowly proteolyzed at the Arg(Pi)-Pro(Pi') peptide bonds as shown herein (Fig. 18C) and for the P53 and Hirulog peptides in previous studies (DiMaio, J., Gibbs, B., Munn, D., Lefebvre, J., Ni, F., and Konishi, Y. (1990) J. Biol. Chem. 265, 21698-21703; and Witting, J. I., Bourdon, P., Brezniak, D. V., Maraganore, J. M., and Fenton, J. W. (1992) Biochem. J. 283 (R 3), 737-743).<br>
Activities of the synthetic peptides were ranked by use of only the ICso (or DCT) values for the inhibition of fibrinogen clotting. Therefore, elongation of the negatively-charged side chain through the Asp(P3) to Glu(P3) substitution decreased the anticlotting activity of the bivalent peptide from an IC50 of 45 nM (for FD22) to 80 nM (for FE22) (Table 1). The positively-charged and somewhat aromatic His residue at P3 is not well tolerated, as the FH22 peptide had a further decreased anticlotting activity (Table 1). Interestingly, the P3' site appears to be insensitive to the presence of a negatively-charged.residue, as the FD22-D peptide with a Ser(P3') to Asp(P3') substitution had a similar activity as the FD22 peptide. On the other hand, the FE22-D peptide had a significantly reduced activity (Table 1), indicating that an elongated and negatively-charged Glu(P3) is unfavourable when a negatively-charged Asp is present at the P3 position.<br>
The residue requirements at the P3' position were examined further by reversing the charge of the residue at this site. A His or a Lys residue is not as favourable as a Ser residue at P3' since both FD22-H and FD22-K had somewhat reduced activities compared to the FD22 peptide (Table 1). Very surprisingly, however, an Arg(P3') residue appears to confer enhanced anticlotting activity as the FD22-R peptide had an IC50 of ~ 32 nM compared to - 45 nM for the FD22 peptide (Table 1). Inhibition of the thrombin turnover of the chromogenic substrate Tos-Gly-Pro-Arg-pNA was then studied for the two bivalent peptides, FD22-D and FD22-R. Changes of /
 <br>
with Arg(P3') had a similar K, as FD22 with SerCPg') despite that FD22-R appears to be slightly more active (IC50 ~ 32 nM) in the anticlotting assay. The FD22-D peptide with Asp(P3') had the largest K; value (or the least active) among the three, indicating that the negatively-charged Asp is not a favourable P3' residue.<br>
4: Enhancement of inhibitory activities by a Trp residue at the P4 site<br>
The inventors then examined whether a Trp residue at the P4 site would contribute positively to the anticoagulant activity of the bivalent peptide. Indeed, the WD22-R peptide with a Trp(P4) residue had its DCT value decreased almost 30% and its K/ value decreased more then 50% compared to the FD22-R peptide (Table 1). A further change of Glu58 to Pro as well as the Phe56Tyr substitution within the exosite-binding segment led to another more than 50% drop in DCT for the WD22-R-P and WD22-R-YP peptides, respectively (Table 1). interestingly, both the WD22-R-P and WD22-R-YP peptides had significantly increased K, values in the inhibition of the amidolytic activity of thrombin. These results indicate that the WD22-R-P and WD22-R-YP molecules represent a new class of anticoagulant agents with strong binding (low DCT values) to the fibrinogen-recognition exosite of thrombin, while interfering only partially the thrombin catalytic active site (with high K; values). Furthermore, the strong anticoagulant activities of WD22-R-P and WD22-R-YP appear to depend on the nature of the so-called linker sequence (Fig. 1 and Fig. 15A), as a shift of the Arg(P3') residue to the P4' position led to almost a factor of 10 increase of the DCT values of both the WD22-SR-P and WD22-SR-YP peptides (Table 1). In addition, replacement of the important Arg(P3') residue by Leu(P3') resulted in a decreased anticoagulant activity of the WD22-L-P peptide (DCT ~. 26 nM) by almost 4 fold as compared to the WD22-R-P peptide (DCT - 6.7 nM) (Table 1). In ail, these data showed that the strong anticoagulant activities of the WD22-R-P and WD22-R-YP molecules critically depend on the presence of an Arg residue at the P3' position with presumably specific interactions with the S'-subsites of the thrombin active site. The critical interdependence between the S-subsite binding residues, i.e. those up to Arg(Pi) before the peptide bond cleaved by thrombin, and the S'-subsite binding residues (those after<br><br>
Arg(Pi)) is also seen with the FD22 and FD21-4GN (also refered to as FD22.1 in Fig. 1) molecules. The Phe(P4)-Asp(p3)-Pro(P2)-Arg(Pi)-Pro(Pi') (SEQ ID NO:122) pentapeptide segment confers synergistic binding between the active site and the exosite I of thrombin (with a DCT value of - 50 nM for FD22). However, replacement of residues QSHNDG (SEQ ID NO: 123) in FD22 by the linker residues GGGGNG (SEQ ID NO: 124) and the exosite-binding sequence by DFEEIPEEYL (SEQ ID NO:125) iri hirulog resulted in more than 4 fold decrease of the anticoagulant activity (i.e. 4 times increase in the DCT value for the FD21-4GN peptide, Table 1).<br>
5: Selection of bivalent peptide inhibitors of thrombin by phage display<br>
Residue preferences at the P4, P3, and P3' sites were then searched through sequences displayed on phages and panning the displayed peptide library. The framework of the peptide library (Fig. 19A) includes two flexible linkers of glycine/serine residues to separate the displayed peptide ligand from the His-tag and the gill protein. These Gly/Ser linkers were introduced to minimize possible interference with thrombin binding. Sequencing of ten clones randomly chosen showed the lack of a noticeable sequence bias (Fig. 19B). Three slightly different protocols were used for phage selection. In the first approach, the phage particles were incubated at 37°C with active thrombin, followed by purification using Ni-NTA affinity resin of the phages containing uncleaved sequences. The amount of added thrombin was chosen empirically to retain 10-50 % of the intact phages. The collected phage was amplified and panned against active thrombin immobilized on a solid support. Only two sequences were identified after these two steps of selection (Table 2). In Fig. 19A, underlined are the inserted flexible linkers of Gly/Ser residues, which were introduced to minimize interference with thrombin binding to the displayed peptide. Boxed are randomized amino acids, with putative subs'rte positions displayed on top.<br><br><br><br>
Phase selection was then carried out with less stringent cleavage conditions, i.e., no thrombin cleavage was performed before the phage panning. It was reasoned that upon binding to immobilized active thrombin, some cleavage would take place and therefore enrich propagated phages with the sequences possessing lower proteolytic susceptibility plus higher affinity. One round of panning was performed at room temperature (25°C), another at 4°C, and the TG1 cells were infected with the retained phages directly in the panning well. For panning rounds carried out at 25°C and 4°C, 60 and 4x103 plaques were recovered, respectively. Peptide sequences were determined for eight and ten randomly-chosen phage particles from these two experiments (Table 2). A clear tendency of the P3' position was observed, with lle/Leu/Met dominating the collected phages. In addition, P4 is occupied preferentially by aromatic, heterocyclic or long-chain aliphatic amino acids. Interestingly, the rationally designed tetrapeptide sequence Phe-Asn-Pro-Arg (SEQ ID N0:3) was found in one of the clones obtained from panning at 4°C (Table<br>
2).<br>
The phage particles collected after the first round of panning at 4°C were amplified and used for a second round of panning, at two temperatures, A total of 2x107 plaques and 3x107 plaques were recovered at 25°C and 4'C, respectively. The increase in phage recovery as compared to the first round of panning is consistent with amplification of sequences with higher affinity towards thrombin, while less dependence of phage recovery on temperature suggested decreased proteolytic cleavage. In the second round of selection, the P3' position retained its strong preference for lle/Leu/Met amino acids (Table 2). The P3 position showed a preference for Gin, while aromatic and heterocyclic amino acids at P4 were outnumbered by long-chain aliphatic residues. Apparently, trivalent peptides may have been selected by the process of panning, with the invariant C-terminal hirudin-derived tail targeting the fibrinogen-recognition exosite, optimized prime-site sequences with lle/Leu/Met in the P3' position, and the P1-P4 tetrapeptide sequences binding to the active site of thrombin.<br><br>
6: Antithrombin activities of representative peptides selected from ttie phage library<br>
Five   bivalent  peptides  were   derived   from   two   sequences<br>
obtained from the first round of panning (Table 2), and three sequences<br>
from the second round of selection (Table 3). The two peptides, FN22-I<br>
and FQ22-M, had a Phe residue at P4, Gin or Asn at P3 and He or Met at<br>
P3', which resemble the FNPR sequence (SEQ ID N0:3) of the prototypic<br>
peptide FN22 (Fig. 1. 15A and Table 1). The three peptides, GS-AV22-I,<br>
GS-IQ22-I and iQ22-l, from the second round had an Ala or lie at P4, Val<br>
or Gin at P3 and lie at P3', all of which are frequently occurring in the<br>
panning hits (Table 3). Two of the peptides, GS-AV22-I and GS-IQ22-I<br>
included the two residues Gly-Ser at the N-termini, in accordance with the<br>
context of phage-displayed sequences (Fig. 19A).<br><br><br>
All five peptides displayed IC50 values in the range of 10 to 45 nM (Table 1). However, they were, with the exception of IQ22-I, generally much less efficient at inhibiting the amidolytic activity of thrombin (Table 1). The significantly increased K/with respect to IC50 may reflect less efficient binding and/or the existence of additional binding modes of these peptides near the active site of thrombin as observed with some peptides in the WD22 series (Table 1). Regardless, these bivalent peptides derived from the phage library exhibit potent inhibition of fibrinogen clotting catalyzed by thrombin, in accordance with the phage selection process. The FQ22-M and the GS-IQ22-I peptides with sequences of Phe(P4)-Gln(P3)-Pro-Arg-Pro-Gln-Met(P3') (SEQ ID NO:35) and Gly-Ser-lle(P4)-Gln(P3)-Pro-Arg-Pro-Gln-lle(P3')(SEQ ID NO:36), respectively, even had IC50 values down to 10 nM, that approaches the inhibitory potencies of other bivalent thrombin inhibitors such as hirulog or P53 (DilVlaio, J., Gibbs, B., Munn, D., Lefebvre, J., Ni, F., and Konishi, Y. (1990) J. Biol. Chem. 265, 21698-21703; and Maraganore, J. M., Bourdon, P., Jablonski, J., Ramachandran, K. L., and Fenton, J. W. (1990) Biochemistry 29, 7095-7101).<br>
Finally, the LD22 peptide was synthesized, whereby the Phe(P4) residue of FD22 was replaced by Leu(P4). The LD22 peptide is related to the Hirulog-like peptide (HLP) reported by Shen at al (Xue, M., Ren, S., Welch, S., and Shen, G. X. (2001) J. Vase. Res. 38, 144-152) in that it also contained the PrP4 tetrapeptide LDPR (SEQ ID N0:1) from the thrombin receptor (Ni, F., Ripoll, D. R., and Purisima, E. O. (1992) Biocliemistry 31, 2545-2554). However, this peptide had significantly-reduced antithrombin and anticoagulant activities (Table 1) in accordance with observations with the Hirulog-like peptide (Xue, M., Ren, S., Welch, S., and Shen, G. X. (2001) J. Vase. Res. 38,144-152).<br>
Discussion<br>
The   specificities  of  substrate   recognition   by  thrombin   are<br>
commonly analyzed at the P1, P2 and P3 subsites using a tripeptide<br>
sequence framework (Liu, L. W., Vu, T. K., Esmon, C. T., and Coughlin, S.<br>
R. (1991) J. Biol. Chem. 266, 16977-16980; Le Bonniec, B. F. and Esmon,<br>
i     C. T. (1991) Proc. Natl. Acad. Sci. U. S. A 88, 7371-7375; Vindigni, A.,<br><br>
Dang, Q. D., and Di Cera, E. (1997) Nat. Biotechnol. 15, 891-895; and Le Bonniec, B. F., MacGillivray, R. T., and Esmon, C. T. (1991) J. Biol. Chem. 266, 13796-13803). Optimal binding interactions witin thrombin occur only if these tripeptide substrates contain an amino acid residue in the (D)-configuration, such as (d)Phe at P3 (Blomback, B., Blomback, M., Olsson, P., Svendsen, L., and Aberg, G. (1969) Scand. J. Clin. Lab Invest SuppI 107, 59-61), which mimics the natural Pg"residue in FpA (Ni, F., Meinwald, Y. C, Vasquez, M., and Scheraga, H. A. (1989) Biochemistry 28, 3094-3105; Stubbs, M. T., Oschkinat, H., Mayr, I., Huber, R., Angliker, H., Stone, S. R., and Bode, W. (1992) Eur. J. Biochem. 206, 187-195; and Martin, P. D., Robertson, W., Turk, D., Huber, R., Bode, W., and Edwards, B. F. (1992) J. Biol. Chem. 267, 7911-7920). However, these minimalistic peptide substrates probe only the active site apparatus of thrombin and related binding events, which were found to be mildly sensitive to interactions of thrombin with regulatory proteins (Liu, L. W., Vu, T. K., Esmon, C. T., and Coughlin, S. R. (1991) J. Biol. Chem. 266, 16977-16980). On the other hand, cleavages after tetrapeptide sequences by thrombin, e.g. VDPR (SEQ ID NO:37) in human protein C, LDPR (SEQ ID N0:1) in human PARI and iSPR (SEQ ID NO:38) in the thrombin-activatable fibrinolysis inhibitor, are dramatically enhanced by binding interactions at thrombin exosites remote from the active site (Le Bonniec, B. F. and Esmon, C. T. (1991) Proc. Natl. Acad. Sci. U. S. A 88, 7371-7375; Le Bonniec, B. F., MacGillivray, R. T., and Esmon, C. T. (1991) J. Biol. Chem. 266, 13796-13803; Schneider, M., Nagashima, M., Knappe, S., Zhao, L., Morser,.J., and Nesheim, M. (2002) J. Biol. Chem. 277, 9944-9951; Boffa, M. B., Bell, R., Stevens, W. K., and Nesheim, M. E. (2000) J. Biol. Chem. 275, 12868-12878; and Ishii, K., Gerszten, R., Zheng, Y. W., Welsh, J. B., Turck, C. W., and Coughlin, S. R. (1995) J. Biol. Chem. 270, 16435-16440). Based on NMR and structural studies, it was found that binding specificity at the active srte of thrombin may be captured by minimally a four-residue consensus motif, Phe(P4)-Xaa(P3)-Pro(P2)-Arg(Pi) or FXPR (SEQ ID N0:39), where the P3 residue (i.e. Xaa) can be a charged or a neutral polar residue contacting specific structural features of the S1-S4 subsites (Ni, F., Zhu, Y., and Scheraga, H. A. (1995) J. Mol. Biol. 252,  656-671).   It  was  also  found  that  the   FXPR  (SEQ   ID   N0:39)<br><br>
tetrapeptide sequences could be good candidates for active site binding in designing novel bivalent peptide inhibitors of tiirombin (Song, J. and Ni, F. (1998) Biochem. Cell Biol. 76, 177-188), that are composed of only [L]-amino acids.<br>
Indeed, both the FNPR (SEQ ID N0:3) and FDPR (SEQ ID N0:4) sequence motifs turned out to confer bivalent and bridge binding, as the peptides FN22 and FD22 had significantly enhanced anticlotting activities compared to the hirudin peptide that bind only to the fibrinogen recognition exosite of thrombin (Table 1). The FN22 peptide was also found to be cleaved by thrombin at the Arg(Pi)-Pro(Pi') peptide bond as with all substrate-type bivalent inhibitors like P53 or Hirulog (DiMaio, J., Gibbs, B., Munn, D.. Lefebvre, J., Ni, F., and Konishi, Y. (1990) J. Biol. Chem. 265, 21698-21703; and Maraganore, J. M., Bourdon, P., Jablonski, J., Ramachandran, K. L., and Fenton, J. W. (1990) Biochemistry 29, 7095-7101). In addition, thrombin can cleave much more efficiently (&gt; 50 fold) recombinant fusion proteins containing the FNPR (SEQ ID N0:3) sequence in place of the commonly-used LVPR sequence (SEQ ID NO:40). Screening combinatorial substrate libraries containing the P1-P4 tetrapeptides has confirmed thrombin's preference for a Pro(P2) residue and Phe or long-chain aliphatic residues at the P4 position (Backes, B. J., Harris, J. L., Leonetti, F., Graik, C. S., and Ellman, J. A. (2000) Nat. Biotechnol. 18, 187-193; Edwards, P. D., Mauger, R. C, Cottrell, K. M., Morris, F. X., Pine, K. K., Sylvester, M. A., Scott, C. W., and Furlong, S. T. (2000) Bioorg. Med. Chem. Lett. 10, 2291-2294; nd Furlong, S. T., Mauger, R. C, Strimpler, A. M., Liu, Y. P., Morris, F. X., and Edwards, P. D. (2002) Bioorg. Med. Chem. 10, 3637-3647). On the other hand, the P3 subsite was found to be rather promiscuous with both Asn and Asp among the least preferred residues for substrate cleavage. Even so, substitution of Asn(P3) for Asp(P3) led to an enhancement of the anticlotting activity of the FD22 peptide as well as a significant decrease of the proteolytic sensitivity to thrombin. This latter observation is very much in line with the notion that a P3 aspartate in protein C partially inhibits the proteolytic cleavage at the VDPR (SEQ ID NO:37) site by thrombin (Le Bonniec, B. F. and Esmon, C. T. (1991) Proc. Natl. Acad. Sci. U. S. A 88. 7371-7375; Ishii, K., Gerszten, R., Zheng, Y. W., Welsh, J. B.. Turck, C. W., and<br><br>
Coughlin. S. R. (1995) J. Biol. Chem. 270, 16435-16440; and Rezaie, A. R. and Esmon, C. T. (1994) Eur. J. Biochem. 223, 575-579). In addition, enlianced inhibition of the FD22 peptide with Asp(P3) unfavorable for substrate turnover indicates that residue preferences for efficient catalysis may be too stringent a criterion for the selection of optimized binding interactions.<br>
Using extended peptide substrates, it was shown previously that thrombin prefers a positively-charged residue, e.g. a lysine, at the Py position over negatively-charged residues (Le Bonniec, B. F., Myles, T., Johnson, T., Knight, C. G., Tapparelli, C, and Stone, S. R. (1996) Biochemistry 35, 7114-7122). Interestingly, a Ser(P3') to Asp(P3') substitution resulted in little changes of the inhibitory activities of the FD22-D peptide, indicating that the thrombin active site can also tolerate negatively-charged aspartates at both the P3 and P3' positions, at least for inhibitory binding. However, a further change of Asp(P3) to Glu(P3) impaired active site binding as the FE22-D peptide showed diminished anticlotting activities to a level approaching the himdin peptide alone (Table 1). On the other hand, the bivalent peptide with an aspartate at P3 indeed accommodates well a large and positively charged residue at the P3' position, as both the FD22-K and FD22-R peptides had comparable or higher inhibitory activities compared to FD22 (Table 1). With an arginine at the P3 position, the FD22-R peptide, showed a somewhat enhanced anticlotting activity as compared to the FD22 and FD22-D peptides (Table 1) with Ser(P3') and Asp(P3'), respectively. Furthermore, the Arg residue at the P3' position may create a more favorable binding environment for the Asp(P3) residue as the FD22-R peptide showed the strongest inhibition (or lowest Ki) of the thrombin active site (Table 1). These findings indicate that there exist intricate communications between the P3 and P3' sites not only through charge-charge interactions, but as well with defined spatial requirements as shown in studies of protein C activation by thrombin (Le Bonniec, B. F. and Esmon, C. T. (1991) Proc. Natl. Acad. Sci. U. S. A 88, 7371-7375; Rezaie, A. R. and Esmon, C. T. (1994) Eur. J. Biochem. 223, 575-579; Le Bonniec, B. F., Guinto, E. R., and Esmon, C. T. (1992) J. Biol. Chem. 267, 6970-6976; and Rezaie, A. R. and Yang, L. (2003) Proc. Natl. Acad. Sci. U. S. A 100,12051-12056). The three-dimensional structures of<br><br>
an uncleavable analogue of human FpA containing Pi' — P3' residues (Martin, P. D., Malkowski, M. G., DilVlaio, J., Konisiii, Y., Ni, F., and Edwards, B. F. (1996) Biochemistry 35, 13030-13039) and of iieparin cofaotor II (Baglin, T. P., Carrel!, R. W., Church, F. C, Esmon, C. T., and Huntington, J. A. (2002) Proc. Natl. Acad. Sci. U. S. A 99,11079-11084) in complexes with thrombin showed that the P3' residue projects its side-chain in the direction of the P3 site, bringing both residues into side-chain to side-chain contacts. Such specific interactions would account for the enhanced anticlotting activity of the FD22-R peptide containing Asp(P3) and Arg(P3') residues.<br>
Complementary to rational design, panning of a phage library revealed additional bivalent sequences with unique P4, P3 and P3' residues (Tables 2 and 3). Phage libraries displaying bivalent peptides have recently been used for the discovery of specific inhibitors of factor Vila (Maun, H. R., Eigenbrot, C, and Lazarus, R. A. (2003) J. Biol. Chem. 278, 21823-21830). A peptide with a defined sequence targeting an exosite of factor Vila was used as an anchor for a bivalent and partially randomized peptide. The panning was to select peptide sequences capable of reaching and binding to the catalytic active site of factor Vila, thereby undergoing proteolytic cleavage. The new panning procedure described herein, on the other hand, selects peptide sequences that bind strongly to thrombin and are resistant to proteolysis in the presence of thrombin. The P4 site of the panned phage sequences had a preference for a hydrophobic or an aromatic residue, such as lie/Leu, Phe or Tyr, in agreement with screening of substrate libraries (Backes, B. J., Harris, J. L., Leonetti, F., Craik, C. S., and Ellman, J. A. (2000) Nat. Biotechnol. 18, 187-193; Edwards, P. D., Mauger, R. C, Cottrell, K. M., Morris, F. X., Pine, K. K., Sylvester, M. A., Scott, C. W., and Furlong, S. T. (2000) Bioorg. Med. Chem. Lett. 10, 2291-2294; and Furlong, S. T., Mauger, R. C, Strimpler. A. M., Liu, Y. P., Morris, F. X., and Edwards, P. D. (2002) Bioorg. Med. Chem. 10, 3637-3647). The P3 site was a great deal more variable, having either hydrophobic or polar residues such as Asn and Gin. The high occurrence of a long-chain hydrophobic residue, i.e. Leu/Met/lle, at the P3' position (Tables 2 and 3) was not noted in previous studies using extended peptide substrates of thrombin (Le Bonniec, B. F., Myles, T., Johnson, T., Knight,<br><br>
C. G., Tapparelli, C, and Stone, S. R. (1996) Biochemistry 35, 7114-7122), nor within protein sequences tliat are natural substrates of tlirombin (Rose, T. and Di Cera, E. (2002) J. Biol. Chem. 277, 18875-18880; and Le Bonniec, B. F., Myles, T., Johnson, T., Knight, C. G., Tapparelli, C, and Stone, S. R. (1996) Biochemistry 35, 7114-7122). In contrast, an aromatic, especially phenylalanine, or a long-chain aliphatic residue at the P2' position was found to favor substrate cleavage (Le Bonniec, B. F., Myles, T., Johnson, T., Knight, C. G., Tapparelli, C, and Stone, S. R. (1996) Biochemistry 35, 7114-7122) and many protein substrates of thrombin contain a hydrophobic residue at the P2' site. Interestingly, a similar preference for long-chain hydrophobic residues was found for the P^' subsite of a bivalent peptide inhibitor of thrombin containing a terminally-blocked arginine as the active-site binding moiety (Slon-Usakiewicz, J. J., Sivaraman, J., Li, Y., Cygler, M., and Konishi, Y. (2000) Biochemistry 39, 2384-2391). Surprisingly, many phage sequences contain an alanine at the P4 position (Table 3), which is not a favorable residue for substrate turnover (Backes, B. J., Harris, J. L., Leonetti, F., Craik, C. S., and Ellman, J. A. (2000) Nat. Biotechr)ol. 18, 187-193; Edwards. P. D., Mauger, R. C, Cottrell, K. M., Morris, F. X., Pine, K. K., Sylvester, M. A., Scott, C. W., and Furlong, S. T. (2000) Bioorg. Med. Chem. Lett. 10, 2291-2294; and Furlong, S. T., Mauger, R. C, Strimpler, A. M., Liu, Y. P., Morris, F. X., and Edwards, P. D. (2002) Bioorg. Med. Chem. 10, 3637-3647) and neither found in sequences of natural substrates of thrombin (Ni, F., Zhu, Y., and Scheraga, H. A. (1995) J. Mol. Biol. 252, 656-671; and Rose, T. and Di Cera, E. (2002) J. Biol. Chem. 277, 18875-18880).<br>
All representative peptides derived from panning hits showed strong anticlotting activities in the low nanomolar range (Table 1). Two of the five phage-derived peptides, i.e. FQ22-M and GS-IQ22-I, had anticlotting IC50 values as low as 10 nM, an inhibitory potency that is close to that of hirulog (Table 1). Very interestingly, however, the FQ22-M and GS-IQ22-I peptides had rather large K; values for the inhibition of the amidoiytic activity of thrombin (Table 1). Even the less potent peptide, FN22-I, exhibited a K, value (360 nM), which is eight times the IC50 (45 nM) for clotting inhibition. These consistently increased K, values over the anticlotting ICso's indicate that these bivalent peptides may also bind to<br><br>
alternative sites on thrombin, through the PrP4 and Pi'-Ps' sequences, other than the catalytic active site. In fact, the very potent GS-IQ22-I peptide containing residues Gly-Ser before the putative lle(P4) residue may not even bind to the active site of thrombin as the amidolytic activity of thrombin was little affected in the presence of GS-IQ22-I up to 400 nM in concentration (Table 1). Very surprisingly, the truncated peptide, IQ22-I, being devoid of only the Gly-Ser linker residues, returned to "normal" thrombin binding as it had similar anticlotting and anti-amidolytic activities. Such active site binding favored by the IQPR sequence may be driven by the formation of stronger hydrogen-bonding and electrostatic interactions involving the primary amine of an unblocked lle(P4) residue, as observed for the (d)FPR tripeptide motifs (Rehse, P. H., Steinmetzer, T., Li, Y., Konishi, Y., and Cygler, M. (1995) Biochemistry 34, 11537-11544; and Nienaber, V. L., Mersinger, L. J., and Kettner, C. A. (1996) Biochemistry 35, 9690-9699). The GS-AV22-I peptide also had the normal bivalent mode of binding to thrombin (Table 1), but it very interestingly contains the Gly-Ser residues, which blocks the N-terminus of the alanine at the P4 position and would make it impossible to engage in enhanced hydrogen bonding interactions (Nienaber, V. L., Mersinger, L. J., and Kettner, C. A. (1996) Biochemistry 35, 9690-9699). In all, some selected phage peptides appear to utilize alternative binding sites on thrombin to achieve synergistic binding and efficient inhibition of proteolytic cleavages of macromolecular substrates (e.g. fibrinogen) by thrombin. Indeed, it was reported recently that a peptide derived from the N-terminus of the thrombin receptor PARS also confers strong inhibition of the fibrinogen-clotting activity while leaving open the catalytic active site of thrombin (Owen, W. G. (2003) Biochem. Biophys. Res. Commun. 305,166-168).<br>
The frequent occurrence of Ala at P4 in phage sequences (Table 3) is of further interest since thrombin is not known to favor this residue and yet the GSAVPR sequence (as in the GS-AV22-I peptide) confers enhanced bridge-binding inhibition of the thrombin active site (Table 1). On the other hand, thrombin is a highly allosteric enzyme, featuring, in addition to the catalytic active site, a regulatory protein binding site (exosite I), also called the anino-binding exosite (ABE) or the fibrinogen-recognition exosite (PRE), which the hirudin sequence of the bivalent peptides targets and<br><br>
binds. Thrombin also lias a sodium-binding site in the vicinity of the catalytic active site, responsible for additional complexities of thrombin function (Dang, 0. D., Vindigni, A., and Di Cera, E. (1995) Proc. Natl. Acad. Sci. U. S. A 92, 5977-5981). It was shown in a recent work that the active site of thrombin could assume a closed and inactive conformation, which in solution rnay be in equilibrium with active conformations of thrombin (Huntington, J. A. and Esmon, C. T. (2003) Structure. (Camb.) 11, 469-479). This inactive conformation of thrombin has an occluded P4-binding subsite, leaving little room for the bulky side chain of an aromatic or aliphatic P4 residue. The binding properties and catalytic efficiencies of the thrombin active site may thereby be linked through fast interconversions between active and inactive conformations to the occupancies of regulatory binding sites, especially ligand binding at the fibrinogen-recognition exosite. As such, it is conceivable that binding of the GS-AV22-I peptide may stabilize selectively the inactive and closed conformation of thrombin, achieving inhibition of catalysis indirectly instead of direct binding to catalytically active structures at the active site.<br>
The new peptide inhibitors of thrombin of the present invention are composed of natural amino acids and can be prepared more cost-effectively by recombinant DNA technology. Moreover, a controlled balance between thrombin inhibition and peptide cleavage (after the Arg-Pro bond) can result in potent and more efficacious anticoagulation with reduced undesirable side effects.<br>
While the invention has been described in connection with specific embodiments thereof, it will be understood that it is capable of further modifications and this application is intended to cover any varia¬tions, uses, or adaptations of the invention following, in general, the principles of the invention and including such departures from the present disclosure as come within known or customary practice within the art to which the invention pertains and as may be applied to the essential features hereinbefore set forth, and as follows in the scope of the appended claims.<br><br><br>
WE CLAIM:<br>
1.	An isolated polypeptide inhibitor of thrombin consisting of all  naturally-<br>
occurring amino acids, said inhibitor having general formula (1) of:<br>
SBM-PBM-EBM     (1)<br>
wherein<br>
SBM is a sequence moiety comprising P-subsite residues P4-P3-Pro-Arg of the thrombin inhibitor,<br>
PBM is a sequence moiety comprising P'-subsite residues Pro-P2'-P3'-P4'-P5'-P6'-P7' of the thrombin inhibitor, and<br>
EBM is a peptide sequence moiety that binds to the fibrinogen recognition exosite of thrombin,<br>
or a pharmaceutically acceptable salt thereof<br>
wherein P4, P3, P2', P3', P4', P5', P6' and P7' are natural amino acid residues, where P3' is Arg, Leu, He or Met,<br>
and wherein P and P' represent a division of the peptide sequence at the peptide bond hydrolyzed specifically by thrombin.<br>
2.	The inhibitor as claimed in claim 1, wherein SBM is Trp-Xaa-Pro-Arg, where Xaa is a natural amino acid residue.<br>
3.	The inhibitor as claimed in claim 1, wherein SBM is<br>
A f <br>
Trp-Asp-Pro-Arg (SEQ ID N0:5), Tyr-Asn-Pro-Arg (SEQ ID NO:44), Tyr-Ser-Pro-Arg (SEQ ID NO:45), Ile-Gln-Pro-Arg (SEQ ID NO:46), Gly-Ser-Ile-Gln-Pro-Arg (SEQ ID NO:47), Ile-Asn-Pro-Arg (SEQ ID NO:48), Val-Gln-Pro-Arg (SEQ ID NO:49), Ala-Val-Pro-Arg (SEQ ID NO:50), Gly-Ser-Ala-Val-Pro-Arg (SEQ ID N0:51), Ala-Leu-Pro-Arg (SEQ ID NO:52), or Ala-Ile-Pro-Arg (SEQ ID NO:53).<br>
4.	The inhibitor as claimed in claim 2, wherein Xaa is Asp or Asn.<br>
5.	The inhibitor as claimed in claim 1, 2, 3, or 4, wherein PBM is<br>
Pro-Gln-Arg-His-Asn-Asp-Gly (SEQ ID NO:55), Pro-Gln-Arg-Pro-Asn-Asp-Gly (SEQ ID NO:56), Pro-Gln-Ile-His-Asn-Asp-Giy (SEQ ID NO:58), Pro-Gln-Leu-His-Asn-Asp-Gly (SEQ ID NO:59), or Pro-Gln-Met-His-Asn-Asp-Gly (SEQ ID NO:60).<br>
6.	The inhibitor as claimed in claim 1, 2, 3, 4 or 5, wherein EBM is<br>
Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:64), Asp-Phe-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:65), Asp-Tyr-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:66), Asp-Phe-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu (SEQ ID NO:67), or Asp-Tyr-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu (SEQ ID NO:68).<br><br>
7.	The isolated inhibitor as claimed in claim 1, wherein said inhibitor is<br>
Trp-Asp-Pro-Arg-Pro-Gln-Arg-His-Asn-Asp-Gly-Asp-Phe-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:22),<br>
Trp-Asp-Pro-Arg-Pro-Gln-Arg-His-Asn-Asp-Gly-Asp-Tyr-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:23),<br>
Trp-Asp-Pro-Arg-Pro-Gln-Arg-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID N0:21),<br>
Trp-Asp-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:31),<br>
Phe-Gln-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-IIe-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:29),<br>
Phe-Asn-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-GIu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:26),<br>
Ile-Gln-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:27),<br>
Gly-Ser-Ala-Val-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:28), or<br>
Gly-Ser-Ile-Gln-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:30).<br>
8.	The inhibitor as claimed in claim 1, wherein said inhibitor is<br><br>
Tyr-Asn-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:72),<br>
His-Tyr-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:73),<br>
Ile-Leu-Pro-Arg-Pro-GIn-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-GIu-Glu-Tyr-Leu-Gln (SEQ ID NO:76),<br>
Leu-Thr-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-GIu-Glu-IIe-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:77),<br>
Gln-Ser-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:79),<br>
His-Val-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-IIe-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:80),<br>
Ile-Asn-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:83),<br>
Thr-Asp-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:85),<br>
Pro-Glu-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-GIy-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:86);<br>
Tyr-Ser-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:88),<br><br>
Ile-Gln-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-IIe-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:90),<br>
Ile-Met-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID N0:91),<br>
Ile-Ile-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:92),<br>
Ile-His-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:93),<br>
Leu-His-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:97),<br>
Leu-Ile-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:98),<br>
Met-Gln-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:99),<br>
Met-Met-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 100),<br>
Ala-Val-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:101),<br>
Ala-Leu-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 102),<br><br>
Ala-Gln-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 107),<br>
Ala-Thr-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 108),<br>
Pro-Ile-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 110),<br>
Pro-His-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:111),<br>
Pro-Trp-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 112),<br>
Thr-Ile-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 114), or<br>
Asn-Ser-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 116).<br>
9.        A purified polypeptide inhibitor of thrombin comprising all naturally-occurring amino acids, said inhibitor having general formula (1) of:<br>
SBM-PBM-EBM     (1)<br>
wherein<br>
SBM is a sequence moiety comprising P-subsite residues P4-P3-Pro-Arg of the thrombin inhibitor,<br><br>
PBM is a sequence moiety comprising P'-subsite residues Pro-P2'-P3'-P4'-P5'-P6'-P7' of the thrombin inhibitor, and<br>
EBM is a peptide sequence moiety that binds to the fibrinogen recognition exosite of thrombin,<br>
or a pharmaceutically acceptable salt thereof,<br>
wherein P4, P3, P2', P3', P4', P5', P6' and PV are natural amino acid residues, where P3' is Arg, Leu, He or Met,<br>
and wherein P and P' represent a division of the peptide sequence at the peptide bond hydrolyzed specifically by thrombin.<br>
10.	The purified inhibitor as claimed in claim 9, wherein SBM is Trp-Xaa-Pro-Arg, where Xaa is a natural amino acid residue.<br>
11.	The purified inhibitor as claimed in claim 9, wherein SBM is<br>
Trp-Asp-Pro-Arg (SEQ ID N0:5), Tyr-Asn-Pro-Arg (SEQ ID NO:44), Tyr-Ser-Pro-Arg (SEQ ID NO:45), Ile-Gln-Pro-Arg (SEQ ID NO:46), Gly-Ser-Ile-Gln-Pro-Arg (SEQ ID NO:47), Ile-Asn-Pro-Arg (SEQ ID NO:48), Val-Gln-Pro-Arg (SEQ ID NO:49), Ala-Val-Pro-Arg (SEQ ID NO:50), Gly-Ser-Ala-Val-Pro-Arg (SEQ ID N0:51), Ala-Leu-Pro-Arg (SEQ ID NO:52), or Ala-Ile-Pro-Arg (SEQ ID NO:53).<br><br>
12.	The purified inhibitor as claimed in claim 10, wherein Xaa is Asp or Asn.<br>
13.	The purified inhibitor as claimed in claim 9, 10, 11 or 12, wherein PBM is<br>
Pro-Gln-Arg-His-Asn-Asp-Gly (SEQ ID NO:55), Pro-Gln-Arg-Pro-Asn-Asp-Gly (SEQ ID NO:56), Pro-Gln-Ile-His-Asn-Asp-Gly (SEQ ID NO:58), Pro-Gln-Leu-His-Asn-Asp-Gly (SEQ ID NO:59), or Pro-Gln-Met-His-Asn-Asp-Gly (SEQ ID NO:60).<br>
14.	The purified inhibitor as claimed in claim 9, 10, 11, 12 or 13, wherein EBM is<br>
Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:64), Asp-Phe-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:65), Asp-Tyr-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:66), Asp-Phe-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu (SEQ ID NO:67), or Asp-Tyr-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu (SEQ ID NO;68).<br>
15.	The purified inhibitor as claimed in claim 9, wherein said inhibitor is<br>
Trp-Asp-Pro-Arg-Pro-Gln-Arg-His-Asn-Asp-GIy-Asp-Phe-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:22),<br>
Trp-Asp-Pro-Arg-Pro-Gln-Arg-His-Asn-Asp-Gly-Asp-Tyr-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:23),<br>
Trp-Asp-Pro-Arg-Pro-Gln-Arg-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID N0:21),<br>
Trp-Asp-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID N0:31),<br><br>
Phe-Gln-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-IIe-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:29),<br>
Phe-Asn-Pro-Arg-Pro-Gln-IIe-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:26),<br>
Ile-Gln-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-GIn (SEQ ID NO:27),<br>
Gly-Ser-AIa-Val-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-GIu-Glu-Tyr-Leu-Gln (SEQ ID NO:28), or<br>
Gly-Ser-Ile-Gln-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-GIu-Glu-Tyr-Leu-Gln (SEQ ID NO:30).<br>
16.      The purified inhibitor as claimed in claim 9, wherein said inhibitor is<br>
Tyr-Asn-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:72),<br>
His-Tyr-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:73),<br>
Ile-Leu-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:76),<br>
Leu-Thr-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:77),<br>
Gln-Ser-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:79),<br><br>
His-Val-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:80),<br>
Ile-Asn-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:83),<br>
Thr-Asp-Pro-Arg-Pro-Gln-IIe-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:85),<br>
Pro-Glu-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:86);<br>
Tyr-Ser-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:88),<br>
Ile-Gln-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-GIy-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:90),<br>
Ile-Met-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-GIy-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID N0:91),<br>
Ile-Ile-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:92),<br>
Ile-His-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:93),<br>
Leu-His-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:97),<br><br>
Leu-Ile-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:98),<br>
Met-Gln-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-IIe-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:99),<br>
Met-Met-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 100),<br>
Ala-Val-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 101),<br>
Ala-Leu-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 102),<br>
Ala-Gln-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 107),<br>
Ala-Thr-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 108),<br>
Pro-Ile-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 110),<br>
Pro-His-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 111),<br>
Pro-Trp-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID N0:112),<br><br>
Thr-Ile-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-IIe-Pro-Glu-GIu-Tyr-Leu-Gln (SEQ ID NO:l 14), or<br>
Asn-Ser-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-GIu-Glu-Tyr-Leu-GIn (SEQ ID NO:l 16).<br>
17. A method for selecting a polypeptide inhibitor of thrombin consisting of all naturally occurring amino acids and possessing a strong anti-clotting activity, said method comprising the steps of:<br>
(a) generating a library of polypeptides of sequence:<br>
SBM-PBM-EBM<br>
wherein,<br>
SBM has a sequence selected from the group consisting of Xaa-Xaa-Pro-Arg, Xaa-Xaa-Xaa-Pro-Arg, and Xaa-Xaa-Xaa-Xaa-Pro-Arg;<br>
PBM has a sequence selected from the group consisting of Pro-Aaa -Xaa-Aaa^- Aaa^ Aaa"*- Aaa^ wherein Xaa is Arg, Leu, He or Met, and wherein Aaa , Aaa^, Aaa^, Aaa', and Aaa'^ represent fixed natural amino acids; and<br>
EBM has a sequence selected from the group consisting of Asp-Phe-GIu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:64), Asp-Phe-Glu-Pro-IIe-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:65), Asp-Tyr-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:66), Asp-Phe-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu (SEQ ID NO:67), and Asp-Tyr-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu (SEQ ID NO:68);<br><br>
(b)	immobilizing active thrombin on a surface;<br>
(c)	contacting said library with the immobilized thrombin;<br>
(d)	removing the non-bound and/or cleaved phage particles;<br>
(e)	propagating bound and uncleaved phage particles remaining after step (d); and<br>
(f)	identifying the sequence of a polypeptide from the library, said polypeptide being a highest affinity inhibitor of thrombin.<br><br>
18.	The method as claimed in claim 17, wherein PBM has a sequence selected from the group consisting of: Pro-Gln-Arg-His-Asn-Asp-Gly, Pro-Gln-Ile-His-Asn-Asp-Gly, Pro-Gln-Leu-His-Asn-Asp-Gly, and Pro-Gln-Met-His-Asn-Asp-Gly.<br>
19.	The polypeptide inhibitor of thrombin obtained by the method as claimed in claim 17 or 18.<br>
20.	The isolated inhibitor as claimed in claim 1, wherein SBM has a sequence comprising Trp-Xaa-Pro-Arg at the carboxy-terminal end thereof, where Xaa is a natural amino acid residue.<br>
21.	The isolated inhibitor as claimed in claim 20, wherein SBM has a sequence containing at the carboxy-terminal end thereof a sequence comprising<br>
Trp-Asp-Pro-Arg (SEQ ID NO:5), Tyr-Asn-Pro-Arg (SEQ ID NO-.44), Tyr-Ser-Pro-Arg (SEQ ID NO:45), Ile-Gln-Pro-Arg (SEQ ID NO:46),<br><br>
Gly-Ser-Ile-Gln-Pro-Arg (SEQ ID NO:47), Ile-Asn-Pro-Arg (SEQ ID NO:48), Val-Gln-Pro-Arg (SEQ ID NO:49), Ala-Val-Pro-Arg (SEQ ID NO:50), Gly-Ser-Ala-Val-Pro-Arg (SEQ ID N0:51), Ala-Leu-Pro-Arg (SEQ ID NO:52), or Ala-Ile-Pro-Arg (SEQ ID NO:53).<br>
22.	The isolated inhibitor as claimed in claim 20, wherein Xaa is Asp or Asn.<br>
23.	The isolated inhibitor as claimed in claim 20, 21 or 22, wherein PBM has a sequence comprising<br>
Pro-Gln-Arg-His-Asn-Asp-Gly (SEQ ID NO:55), Pro-Gin-Arg-Pro-Asn-Asp-Gly (SEQ ID NO:56), Pro-Gln-Ile-His-Asn-Asp-Gly (SEQ ID NO:58), Pro-Gln-Leu-His-Asn-Asp-Gly (SEQ ID NO:59), or Pro-Gln-Met-His-Asn-Asp-Gly (SEQ ID NO:60).<br>
24.	The isolated inhibitor as claimed in claim 20, 21, 22 or 23, wherein EBM is at<br>
the amino-terminal end thereof and is<br>
Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:64), Asp-Phe-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:65), Asp-Tyr-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:66), Asp-Phe-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu (SEQ ID NO:67), or Asp-Tyr-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu (SEQ ID NO:68).<br>
25.	The isolated inhibitor as claimed in claim 20, wherein said inhibitor comprises<br><br>
Trp-Asp-Pro-Arg-Pro-Gln-Arg-His-Asn-Asp-Gly-Asp-Phe-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:22),<br>
Trp-Asp-Pro-Arg-Pro-Gln-Arg-His-Asn-Asp-Gly-Asp-Tyr-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:23),<br>
Trp-Asp-Pro-Arg-Pro-GIn-Arg-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID N0:21),<br>
Trp-Asp-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID N0:31),<br>
Phe-Gln-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-IIe-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:29),<br>
Phe-Asn-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:26),<br>
Ile-Gln-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:27),<br>
Gly-Ser-Ala-Val-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:28), or<br>
Gly-Ser-Ile-Gln-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:30).<br>
26.      The isolated inhibitor as claimed in claim 20, wherein said inhibitor comprises<br><br>
Tyr-Asn-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-GIn (SEQ ID NO:72),<br>
His-Tyr-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:73),<br>
Ile-Leu-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-GIu-GIu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:76),<br>
Leu-Thr-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-GIu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:77),<br>
Gln-Ser-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:79),<br>
His-Val-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:80),<br>
Ile-Asn-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:83),<br>
Thr-Asp-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:85),<br>
Pro-Glu-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-IIe-Pro-Glu-Glu-Tyr-Leu-GIn (SEQ ID NO:86);<br>
Tyr-Ser-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:88),<br><br>
Ile-Gln-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:90),<br>
Ile-Met-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:91),<br>
Ile-Ile-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:92),<br>
Ile-His-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-GIy-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:93),<br>
Leu-His-Pro-Arg-Pro-GIn-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-GIu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:97),<br>
Leu-Ile-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Giy-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:98),<br>
Met-Gln-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:99),<br>
Met-Met-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 100),<br>
Ala-Val-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 101),<br>
Ala-Leu-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 102),<br><br>
Ala-Gln-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 107),<br>
Ala-Thr-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 108),<br>
Pro-Ile-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 110),<br>
Pro-His-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-GIu-Glu-Tyr-Leu-Gln (SEQ ID NO: 111),<br>
Pro-Trp-Pro-Arg-Pro-GIn-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-IIe-Pro-GIu-Glu-Tyr-Leu-Gln (SEQ ID NO: 112),<br>
Thr-Ile-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Giy-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 114), or<br>
Asn-Ser-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 116).<br>
27.	The purified inhibitor as claimed in claim 9, wherein SBM has a sequence containing Trp-Xaa-Pro-Arg at the carboxy-terminal end thereof, where Xaa is a natural amino acid residue.<br>
28.	The purified inhibitor as claimed in claim 27, wherein SBM has a sequence containing at the carboxy-terminal end thereof a sequence comprising<br>
Trp-Asp-Pro-Arg (SEQ ID NO:5), Tyr-Asn-Pro-Arg (SEQ ID NO:44),<br><br>
Tyr-Ser-Pro-Arg (SEQ ID NO:45), Ile-Gln-Pro-Arg (SEQ ID NO:46), Gly-Ser-Ile-Gln-Pro-Arg (SEQ ID NO:47), Ile-Asn-Pro-Arg (SEQ ID NO:48), Val-Gln-Pro-Arg (SEQ ID NO:49), Ala-Val-Pro-Arg (SEQ ID NO:50), Gly-Ser-Ala-Val-Pro-Arg (SEQ ID N0:51), Ala-Leu-Pro-Arg (SEQ ID NO:52), or Ala-Ile-Pro-Arg (SEQ ID NO:53).<br>
29.	The purified inhibitor as claimed in claim 27, wherein Xaa is Asp or Asn.<br>
30.	The  purified  inhibitor as  claimed in claim 27,  28  or  29,  wherein PBM comprises<br>
Pro-Gln-Arg-His-Asn-Asp-Gly (SEQ ID NO:55), Pro-Gln-Arg-Pro-Asn-Asp-Gly (SEQ ID NO:56), Pro-Gln-Ile-His-Asn-Asp-Gly (SEQ ID NO:58), Pro-Gln-Leu-His-Asn-Asp-Gly (SEQ ID NO:59), or Pro-Gln-Met-His-Asn-Asp-Gly (SEQ ID NO:60).<br>
31.	The purified inhibitor as claimed in claim 27, 28, 29 or 30, wherein EBM has a<br>
sequence containing at the amino-terminal end thereof a sequence comprising<br>
Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:64), Asp-Phe-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:65), Asp-Tyr-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:66), Asp-Phe-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu (SEQ ID NO:67), or Asp-Tyr-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu (SEQ ID NO:68).<br><br>
32.      The purified inhibitor as claimed in claim 27, wherein said inhibitor has a sequence containing a sequence comprising<br>
Trp-Asp-Pro-Arg-Pro-Gln-Arg-His-Asn-Asp-Gly-Asp-Phe-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:22),<br>
Trp-Asp-Pro-Arg-Pro-Gln-Arg-His-Asn-Asp-Gly-Asp-Tyr-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:23),<br>
Trp-Asp-Pro-Arg-Pro-Gln-Arg-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID N0:21),<br>
Trp-Asp-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID N0:31),<br>
Phe-Gln-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-IIe-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:29),<br>
Phe-Asn-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:26),<br>
Ile-Gln-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:27),<br>
Gly-Ser-Ala-Val-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:28), or<br>
Gly-Ser-Ile-Gln-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:30).<br><br>
33.      The purified inhibitor as claimed in claim 27, wherein said inhibitor comprises<br>
Tyr-Asn-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:72),<br>
His-Tyr-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:73),<br>
Ile-Leu-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-GIn (SEQ ID NO:76),<br>
Leu-Thr-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:77),<br>
Gln-Ser-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:79),<br>
His-Val-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:80),<br>
Ile-Asn-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:83),<br>
Thr-Asp-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:85),<br>
Pro-Glu-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:86);<br><br>
Tyr-Ser-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:88),<br>
Ile-Gln-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:90),<br>
Ile-Met-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID N0:91),<br>
Ile-Ile-Pro-Arg-Pro-GIn-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-IIe-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:92),<br>
Ile-His-Pro-Arg-Pro-GIn-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:93),<br>
Leu-His-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:97),<br>
Leu-Ile-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:98),<br>
Met-Gln-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO:99),<br>
Met-Met-Pro-Arg-Pro-Gln-Met-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 100),<br>
Ala-Val-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-GIn (SEQ ID NO:101),<br><br>
Ala-Leu-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 102),<br>
Ala-Gln-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 107),<br>
Ala-Thr-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 108),<br>
Pro-Ile-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 110),<br>
Pro-His-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 111),<br>
Pro-Trp-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 112),<br>
Thr-Ile-Pro-Arg-Pro-Gln-Ile-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID N0:114), or<br>
Asn-Ser-Pro-Arg-Pro-Gln-Leu-His-Asn-Asp-Gly-Asp-Phe-Glu-Glu-Ile-Pro-Glu-Glu-Tyr-Leu-Gln (SEQ ID NO: 116).<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjA2OC1jaGVucC0yMDA1IGFic3RyYWN0LWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">2068-chenp-2005 abstract-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjA2OC1jaGVucC0yMDA1IGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2068-chenp-2005 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjA2OC1jaGVucC0yMDA1IGNsYWltcy1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">2068-chenp-2005 claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjA2OC1jaGVucC0yMDA1IGNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2068-chenp-2005 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjA2OC1jaGVucC0yMDA1IGNvcnJlc3BvbmRlbmNlIG90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2068-chenp-2005 correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjA2OC1jaGVucC0yMDA1IGNvcnJlc3BvbmRlbmNlIHBvLnBkZg==" target="_blank" style="word-wrap:break-word;">2068-chenp-2005 correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjA2OC1jaGVucC0yMDA1IGRlc2NyaXBpdGlvbihjb21wbGV0ZWQpLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">2068-chenp-2005 descripition(completed)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjA2OC1jaGVucC0yMDA1IGRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">2068-chenp-2005 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjA2OC1jaGVucC0yMDA1IGRyYXdpbmdzLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">2068-chenp-2005 drawings-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjA2OC1jaGVucC0yMDA1IGRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">2068-chenp-2005 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjA2OC1jaGVucC0yMDA1IGZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">2068-chenp-2005 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjA2OC1jaGVucC0yMDA1IGZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">2068-chenp-2005 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjA2OC1jaGVucC0yMDA1IGZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">2068-chenp-2005 form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjA2OC1jaGVucC0yMDA1IHBjdCBzZWFyY2ggcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">2068-chenp-2005 pct search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjA2OC1jaGVucC0yMDA1IHBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">2068-chenp-2005 pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjA2OC1jaGVucC0yMDA1IHBldGl0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">2068-chenp-2005 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="229730-glyphosate-resistant-class-i-5-enolpyruvylshikimate-3-phosphate-synthase-epsps.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="229732-a-peptidic-compound-with-affinity-to-bacterial-and-fungal-toxins.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>229731</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2068/CHENP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>19-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>26-Aug-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NATIONAL RESEARCH COUNCIL OF CANADA</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1200 Montreal Road, Ottawa, Ontario K1A 0R6,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>NI, Feng</td>
											<td>4507 Maplewood, Pierrefonds, Quebec H9A 1B2,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>TOLKATCHEV, Dmitri</td>
											<td>110 Balmoral Street, Apartment #103, Laprairie, Québec J5R 4L5,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>NATAPOVA, Anna</td>
											<td>3255 Goyer Street, Apartment #14, Montreal, Québec H3S 1H7,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>KOUTYCHENKO, Anatol</td>
											<td>DRUZHBA, 2/19 APP15, MOSCOW, 119330,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K14/815</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/CA2004/00301</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-02-27</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/449, 878</td>
									<td>2003-02-27</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/229731-peptide-inhibitors-of-thrombin-as-potent-anticoagulants by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:26:49 GMT -->
</html>
