<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/229905-a-method-for-in-vivo-production-of-a-library-of-variants-of-a-gene-encoding-a-polypeptide by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:38:00 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 229905:A METHOD FOR IN VIVO PRODUCTION OF A LIBRARY OF VARIANTS OF A GENE ENCODING A POLYPEPTIDE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR IN VIVO PRODUCTION OF A LIBRARY OF VARIANTS OF A GENE ENCODING A POLYPEPTIDE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a method for in vivo production of a library of variants of a gene encoding a polypeptide of interest in a cell wherein an error prone polymerase is used to replicate all or a part of a genetic element independently of the cells chromosomal replication, the genetic element comprising i) an origin of replication from which replication is initiated, ii) optionally a genetic marker e.g. a gene conferring resistance towards an antibiotic, and iii) the gene encoding the polypeptide of interest.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
The present invention relates to methods for in vivo production 3f libraries of polypeptide variants, the screening of these /variants and selection of those exhibiting desired properties. The invention furthermore relates to methods for producing the desired polypeptide variants.<br>
  increasing number of polypeptides, including enzymes and non-enzymatic proteins, are being produced industrially, for use in various industries, household, food/feed, cosmetics, medicine 2tc. One of the major sources for these proteins is and have been nicroorganism found in nature.<br>
rhe classical approach for finding polypeptides with new and special properties, have been to screen wild' type organisms present in nature. This has been a very successful way of procuring polypeptides to be used in such diverse areas as the above mentioned applications.<br>
However, often it has not been possible to produce such polypeptides in sufficient amounts because the quantities produced in the natural host systems were too minute to allow a production, and even if the cost was no problem, difficulties could be encountered in providing sufficient amounts in relation to the demand (e.g. human growth hormone).<br>
Such problems have to a large degree been overcome by the advent of recombinant techniques for the production of polypeptides. In this art polypeptides are produced by the use of biological systems.  Genes encoding certain polypeptides are cloned and<br><br>
transferred into cells that will produce the polypeptides in quantities much larger than those, wherein they are produced in the original organism. Over the latest twenty years a large number of methods for the production of polypeptides according to such techniques have been developed.<br>
Often, proteins from natural sources do not meet the requirements for certain applications, and it will be necessary to modify existing proteins towards certain activities or biophysical properties.<br>
It is possible to generate new variants of a protein by classical mutagenesis of the microorganism using radiation (X-ray and UV) or chemical mutagens. However, since this approach is a very labour and time consuming process, in the same last two decades researchers have been developing improvements on existing polypeptides by using more specific and selective recombinant techniques, such as protein and genetic engineering for creating artificial diversity.<br>
Based upon considerations using knowledge of the structure-function relationships and general protein chemistry, researchers have come a long way in designing polypeptide variants exhibiting improvements in various properties.<br>
However, it has also been realised that the various interactions into which polypeptides take part, are so complex that rational design according to such knowledge has serious limitations, and in recent years methods employing random mutagenesis followed by screening of or selection from very large numbers of variants produced therefrom has gained interest.<br>
For this purpose a microbial library of mutants is generated for subsequent expression and screening to determine variants possessing the desired properties.<br><br>
Over the years many both in vitro and in vivo DNA mutagenesis techniques for creating high numbers of different variants of polypeptides have been developed.<br>
Considering the fact that a typical naturally occurring polypeptide consists of between 100 and 1000 amino acids, and each may be varied in 2 0 ways (only to stay within the naturally occurring amino acids), the number of possible variants for a specific polypeptide is enormous. Since the main parameter that defines or measures the usefulness of a microbial collection or library used to identify improved variants of polypeptide is the number of different variants, N, which is comprised in the collection, a need for large libraries has emerged.<br>
Especially in cases when a powerful selection system is available, the limiting factor for the identification of the desired polypeptide is the size of the library.<br>
In in vitro systems the practical, state of art, limit for N is about 10^. This is mainly due to inefficiency of transformation (introduction of DNA into the cell) of the manipulated DNA into the host organism. This number varies a lot from organism to organism: in the presently best case, E. coli, the usual efficiency of transformation of in vitro manipulated DNA, e.g. a ligation of DNA fragments or chemical treatment of DNA, leads at the most to library sizes up to 10^ bacteria (Greg Winter, Current methods in Immunology 5: 253-255, 1993). Very few examples of libraries of this size have been reported.<br>
In vitro library constructions in other prokaryotes, such as Bacillus sp., Streptococcus sp. or Staphylococcus sp. will for practical reasons be orders of magnitude below this number.<br>
Considering eukaryotic hosts such as Saccharomyces cerevisiae or various Aspergillus sp. , an even lower number of transformants can be expected from in vitro  manipulated DNA.<br><br>
'A special case of a large library has been reported based on in vivo recombination between libraries of antibody light and heavy chains based on a specially designed system useful for that particular case (Griffiths, A.D. et al., 1994, EMBO J. 14: 3245-3260).<br>
A number of methods are available to generate variants of a polypeptide in microorganisms in vivo, ranging from very simple, such as treating cells with chemical or physical mutagens, to rather complex, relying on cells that contain an error-prone DNA polymerase but lack the mismatch repair sygteii:! which corrects the errors (Stratagene, XLl-red (mutS, mutD, mutT) Catalog #200129) . But these techniques have a major drawback as the mutagenesis is not targeted to a specific part of the genome (coding for the polypeptide of interest) and high frequencies of mutations are generated also in essential genes for the cell as well as in the target gene, resulting in massive cell death, together with a high number of cells, where the mutations do not influence the polypeptide of interest. Such "noise" will limit the accumulation of mutations in the target region.<br>
It is therefore the object of the invention to provide an in vivo target region-specific mutagenesis procedure in order to produce very large numbers, N, of polypeptide variants.<br>
A second object of the invention relates to the screening or selection of variants with the desired properties, both by existing and future technologies.<br>
SUMMARY OF THE INVENTION<br>
The present invention therefore relates to a method for in vivo production of a library in cells comprising a multitude of nutated genetic elements, wherein an error-prone polymerase is<br><br>
r<br><br>
used in each ancestral cell to replicate all or a part of a<br>
genetic element comprising<br>
i)     an  origin  of  replication  from which  replication  is<br>
initiated, ii)    optionally a genetic marker,  e.g.  a gene  conferring<br>
resistance towards an antibiotic, iii)   a gene encoding the polypeptide of interest, independently of the host chromosomal replication machinery.<br>
The invention furthermore relates to a method for the generation<br>
of a DNA sequence encoding a desired variant of a polypeptide of<br>
interest, wherein<br>
i)     a mutant library is produced by the above method,<br>
ii)    said library is cultivated under conditions conducive for<br>
the expression of said gene of  interest  to produce<br>
polypeptide variants, iii)   said variant polypeptides are screened or selected for a<br>
desired  property,  and  hosts  producing  such  desired<br>
variants identified and isolated, iv)    said genetic  element  in  said hosts  is  sequenced to<br>
elucidate the DNA sequence of the mutant gene encoding a<br>
desired variant.<br>
and a method for the determination of the DNA sequence encoding a<br>
desired variant of a polypeptide of interest, wherein<br>
(i)    a mutant library is produced by the above method,<br>
(ii)   said library is cultivated under conditions conducive for<br>
the expression of said gene of interest to produce variant<br>
polypeptides, (iii)  said variant polypeptides are screened or selected for a<br>
desired property, and hosts producing desired variants<br>
identified and isolated, (iv)   said genetic element  in said hosts  is  sequenced to<br>
elucidate the DNA sequence of the mutant gene encoding a<br>
desired variant.<br><br>
The screening of the library or the selection of the variants depends on the specific polypeptide and which properties thereof it is desired to improve and/or retain. It is therefore necessary to set up a screening protocol for each case. Such protocols involving a number of assays are described in the literature (Clackson at al. , Nature 352:624-628, 1991, Bryan, P et al., Proteins  1:326-334:,    1986).<br>
An elegant approach to the combination of the generation of diversity and the selection of variants with the desired properties would be a combination of the in vivo method of the invention for generating the diversity with a phage display system (Greg Winter, Supra).<br>
A specific example of a polypeptide of interest is the alkaline proteases used in the detergent industry for the removal of proteinaceous stains from fabric. In that case the screening may be performed in actual detergent compositions to investigate properties such as thermal stability, oxidation stability, storage stability, substrate specificity and affinity, stability to non-aqueous solvents, pH profile, ionic strength dependence, catalytic efficiency, and wash performance.<br>
Furthermore the invention relates to a process for the production<br>
of a desired polypeptide variant, wherein<br>
(i) a DNA sequence encoding a polypeptide of interest that has been determined according to the method above is intro¬duced into a suitable host in a manner whereby it can be expressed in said host,<br>
(ii) said host is cultivated under conditions conducive to the expression of said DNA sequence, and<br>
(iii)  said polypeptide variant is recovered.<br>
Methods for the introduction of the DNA sequence selected into suitable  host  systems  are  described  in,  (Sambrook  et al.<br><br>
Molecular Cloning: A Laboratory Manual. Cold Spring Harbor Lab., Cold Spring Harbor, NY.).<br>
It is also within the abilities of the skilled person to select suitable growth media and other conditions for the host system selected that are conducive for the expression of the polypeptide variant od interest. Guidance hereto may f. ex. be found in (Sambrook at al.,   supra).<br>
Also for the recovery of	the polypeptide a large number of<br>
methods are available for	the separation and purification of<br>
proteins, e.g.    in (Scopes,	R.K., Protein Purification (1987),<br>
Springer-Verlag)<br>
Lastly, the invention relates to the polypeptides produced by the above method.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The invention comprises a method to construct in vivo libraries of variants in a gene of interest. The method involves the use of a genetic element, such as a bacteriophage or a plasmid that is able to replicate independently of the host chromosomal replication system. By the use of the possibility to separate the replication of the host chromosome and the replication of the genetic element (phage/plasmid), it is possible by modifications of one replication system to selectively introduce mutations in the genetic element (phage/plasmid) keeping the chromosome of the host intact. This means that the generation of variation in the gene of interest does not compromise the viability of the host.<br>
DNA replication is a highly accurate process, and the misincorporation rate of the chromosomal replication in E. coli has been estimated to be in the order of 10'^° pr. base pr. round of replication. The base pairing carried out during DNA replication  leads  to  a  preference  for  the  polymerase  to<br><br>
incorporate the correct base at a certain position, which accounts for approximately 10^ of the overall replication fidelity. If an incorrect base has been incorporated, the polymerase will stall and a 3'-5' exonuclease will often remove the 3' misincorporated base. This part denoted proof-reading accounts for approximately 10^ of the overall replication fidelity. The repair system of the cell accounts for the last 10^ of the fidelity rate.<br>
Replication of the chromosome in E. coli is for the most part carried out by DNA polymerase III holoenzyme, which is a multi-protein complex containing 10 different polypeptides including a polymerase (alpha sub-unit, polC gene) and a 3'-5' exonuclease {dnaQ  gene).<br>
A further polymerase, DNA polymerase I (DNA pol I, polA gene), contains three different activities, viz. a DNA polymerase activity, a 3'-5' exonuclease activity, and a 5'-3' exonuclease activity despite the fact that it is one single polypeptide. This polymerase has several functions in the cell. Besides DNA repair, DNA pol I is also needed for the chromosomal DNA replication, as it is involved in the assembly of DNA fragments during synthesis of the lagging DNA strand. However, it replicates only a very minor portion of the genome.<br>
This polymerase is furthermore involved in initiation of DNA replication of certain classes of plasmids, e.g. ColEI origin of replication-based plasmids such as pBR322 in Escherichia coli and Gram-negative bacteria or pAMSl like plasmids in Gram-positive bacteria. Such plasmids may be able to replicate completely through the activity of DNA polymerase I without DNA polymerase III being present in active form, e.g. if this enzyme is dysfunctional due to genetic causes, e.g. temperature sensitive variants at a non-permissive temperature), or under conditions where only limited amounts of DNA Polymerase III are present in the cell.<br><br>
It is well known that certain mutations lead to a decrease (or an increase) in the fidelity of DNA replication. These mutations have been mapped to reside mainly in polymerases, exonucleases or in elements of the repair system. Unfortunately, most of these mutations alter/impair the fidelity rate for the complete genome present, and such non-targeted mutations are not desirable.<br>
One example of such a mutation could be an inactivation of the 3'-5' exonuclease activity of DNA pol I.<br>
However, according to the invention use is being made of the fact that some elements in the replication system may be temporarily "switched" off, fully or partially, thereby stopping or greatly slowing down the replication of the genome, while replication of certain genetic elements as defined herein is continued.<br>
An E.   coli   strain containing a temperature sensitive DNA pol III<br>
(i.e. the polymerase a-sub-unit or another temperature sensitive<br>
sub-unit that render the holoenzyme conditionally non¬functional) , or a function required for initiation of chromosomal replication, such as DnaA, an error prone DNA pol I and a colEI based plasmid containing a gene of interest, is an example of a genetic system according to the invention designed to specifi¬cally introduce mutations in the plasmid (and the gene of interest).<br>
In such a system raising the temperature to a non-permissive value will have the effect that DNA pol III ceases to function fully, while the error prone DNA pol I will retain its function and replicates the plasmid with reduced fidelity resulting in mutated copies of the plasmid.<br>
Since the generation of mutations is random, each cell will generate unique mutations and upon lowering uhe temperature, the temperature sensitive function will become active again, and normal replication of the cells continue.<br><br>
A variation would be an E. coli strain with temperature sensitive alleles of polIII and poll and an inducible expression (by a ts repressor (temperature-sensitive) or by chemical induction) of the error-prone polymerase. At a restrictive temperature and the presence of the inducer mutations will accumulate in the genetic element. At permissive temperature and the absence of the inducer, the complete systems functions as the wild type cell.<br>
Accordingly the invention in its first aspect relates to a method for in vivo production of a mutant library in cells comprising a multitude of mutated genetic elements, wherein an error-prone polymerase is used in each ancestral cell to replicate all or a part of a genetic element comprising i)     an  origin  of  replication  from which  replication  is<br>
initiated, ii)    optionally a genetic marker,  e.g.  a gene conferring<br>
resistance towards an antibiotic, iii)   a gene encoding the polypeptide of interest, ■ndependently of the host chromosomal replication machinery.<br>
he  invention consequently comprises  a method for in    vivo<br>
reduction of a library in cells comprising a multitude of<br>
utated genetic elements comprising<br>
)     providing a cell having<br>
i) an error-prone polymerase that independently of the chromosomal replication machinery of said cell will replicate all or a part of a genetic element comprising<br>
a)	an origin of replication from which replication is initiated,<br>
b)	optionally  a  genetic  marker,  e.g.  a  gene conferring resistance towards an antibiotic,<br>
c)	a gene encoding the polypeptide of interest, and<br><br>
ii) a chromosomal replication machinery that can be reversibly induced to be substantially non¬functional,<br>
B)	growing such a cell under conditions conducive to its replication to obtain a multitude of ancestral cells,<br>
C)	reversibly inducing said chromosomal replication machinery in said ancestral cells to be substantially non-functional for a period of time sufficient to allow for the replication of said genetic element by said error-prone polymerase to generate mutations in said genetic element,<br>
D)	reversibly inducing said chromosomal replication machinery in such mutated cells to be substantially functional, and<br>
E)	growing such mutated cells under conditions conducive to<br>
their replication.<br>
In this context the expression "mutant library" means a set of cells, bacteria or phages (typically 10^ to lO" cells or phages) that differs with respect to one particular gene encoding a polypeptide of interest. Typically one would like to introduce one or more different amino acid alterations in this particular polypeptide in each member of the library.<br>
In this context the expression "error-prone polymerase" means a polymerase that during DNA replication will incorporate mistakes (one of the wrong nucleotides in a given position or cause a deletion or an insertion of one or several nucleotides) with higher frequency than the polymerase normally used for this purpose (e.g. E. coli DNA pol I, Bacillus subtilis DNA pol I, T4 DNA polymerase, T7 DNA polymerase).<br>
The expression "ancestral cell" here means such cells wherein no mutations have been introduced. In some embodiments of the invention the mutation cycle may be reiterated, and in that case, such cells that were initially mutated become ancestral cells for the second mutation cycle, etc.<br><br>
In this context the expression "host chromosomal replication machinery" means the DNA polymerase or DNA polymerase holoenzyme that is mainly responsible for the replication of the host chromosome, e.g. DNA polymerase III in E. coli.<br>
In this context the expression "genetic element" means a small (from 1 or 2 kilo bases to 100 kilo bas-^s) entity consisting of RNA or DNA, that is able to replicate independently, i.e. it contains an origin of replication. The genetic element would typically be a bacteriophage, a phagemid, or a plasmid. The genetic element must also according to the invention comprise a gene encoding the polypeptide of interest, and it may further comprise a genetic marker, e.g. a gene conferring resistance towards an antibiotic.<br>
A virus, a retrovirus, or a transposon that is able to replicate independently of the host replication machinery, e.g. retrotransposons could also be used as the "genetic element".<br>
Kim and Loeb (1995, PNAS 92: 684-688) have demonstrated that HIV reverse transcriptase (HIV-RT) is able to complement E. coli DNA pol I with respect to chromosomal DNA replication and initiation of plasmid DNA replication. The misincorporation rate of HIV-RT (and related retroviral reverse transcriptases) is several orders of magnitude higher than the rate of DNA pol I, i.e. 10"^ to 10"* mis incorporations pr. base pr. round of replication. The use of such a polymerase in stead of a mutated error prone E. coli DNA pol I in an embodiment of the invention would significantly increase the frequency of replication errors in the system described above.<br>
In a further embodiment of the invention the mutation cycle described above can be reiterated, i.e. the mutagenic polymerase switched on and off several times, thereby generating even more mutants. Such a step could furthermore help the segregation of plasmids if a multicopy plasmid is used as the genetic element.<br><br>
In certain genetic elements one can envision that only the part of the genetic element located in the vicinity of the origin of replication is replicated by the error-prone polymerase. In such cases, the gene of interest should be situated within this region.<br>
In a specific embodiment of the invention the genetic element is a phage, wherein the gene encoding the polypeptide of interest is positioned at a locus where the polypeptide upon expression is displayed from the surface of the phage, whereby a screening can be performed directly (see Greg Winther, supra) . To ensure the correspondence between DNA sequence of the phage and the protein displayed the primary phage stock should be passed through wild type E. coli, at low multiplicity of infection, prior to selection or screening.<br>
To further increase the frequency of mutations, the method of the invention comprises embodiments where the method is used in conjunction with a repair deficient host, e.g.   mutL,   mutS,   mutH, or a combination of mutator genome types.<br>
In this context the expression "repair deficient host" means a cell containing one or more alterations in genes encoding proteins known to be directly or indirectly involved in the DNA repair. The result of such mutations is that a higher frequency of introduced mutations (by the polymerases, chemicals. X-ray, UV light, etc.) will not be repaired and will be "permanently" incorporated in the genome, the so called mutator phenotype. Examples of such genes are inutL, mutS,   mutH,   mutT.<br>
As the genetic element one could as indicated above use a phagemid in stead of a plasmid in order to couple the variant generation to a display system, e.g. M13, fl, fd.<br>
In this context the expression "phagemid" means a plasmid that besides its plasmid origin of replication contains a phage origin<br><br>
of replication, phagemids are dependent of the conditions able to replicate as a plasmid or as a phage (upon infection with a helper-phage).<br>
A phagemid based system would involve the construction of a phagemid containing:<br>
1)	a plasmid origin of replication, e.g. ColEI<br>
2)	a MIS phage origin of replication<br>
3)	a chimeric gene consisting of the gene of interest fused to the gene encoding GUI protein. (0rum, P. et al. , Nucl. Acid.   Res.   21: 4491-4498, 1993).<br>
The first step would be the generation of diversity by growing/maintaining an E.     coli    strain transformed with this<br>
phagemid as described above. The second step would be the infection with the helper phage in order to create single stranded phagemid that will be packed into phage particles. The phages displaying the variant proteins can then be subjected to a selection procedure.<br>
Also, certain bacteriophages such as T4 or T7 in Eschericia or SPOII and Phi29 in Bacillus contain their own DNA polymerases, and according to the invention one could envision embodiments where the genetic element described above is a bacteriophage containing an error-prone DNA polymerase.<br>
According to the invention the error-prone polymerase is typically selected from the group comprising DNA polymerase I or reverse transcriptases. A preferred error-prone polymerase is a variant of E. coli DNA polymerase I or HIV reverse transcriptase.<br>
As a polypeptide of interest a large number is possible, and especially such polypeptides exhibiting biological activities could be mentioned. Among these are enzymes, hormones, receptors, blood-clotting factors,  anti-microbial agents, and other such<br><br>
polypeptides important for the prophylaxis and treatment of various disorders and diseases in humans and animals.<br>
Also, enzymes used for industrial purposes could be mentioned. Among such industrial enzymes, enzymes belonging to the groups carbonyl  hydrolases,  carbohydrases,  oxidoreductases,  trans¬ferases, phytases, anti-microbial polypeptides, oxidoreductases, isomerases, lyases, and ligases.<br>
In this context the expression "carbonyl hydrolase" means enzymes that hydrolyze compounds containing a -C(=0)-X group, where X is oxygen or nitrogen.<br>
Specific classes of enzymes belonging to the group of carbonyl hydrolases are such as hydrolases (lipases) and peptide hydrolases (proteases).<br>
Proteases are here meant as enzymes classified under the Enzyme Classification number E.G. 3.4 in accordance with the Recommendations (1992) of the International Union of Biochemistry and Molecular Biology (lUBMB).<br>
Examples include proteases selected from those classified under the Enzyme Classification (E.C.) numbers:<br>
3.4.11 (i.e. so-called aminopeptidases), including 3.4.11.5 (Prolyl aminopeptidase), 3.4.11.9 (X-pro aminopeptidase), 3.4.11.10 (Bacterial leucyl aminopeptidase), 3.4.11.12 (Thermophilic aminopeptidase), 3.4.11.15 (Lysyl aminopeptidase), 3.4.11.17 (Tryptophanyl aminopeptidase), 3.4.11.18 (Methionyl aminopeptidase).<br>
3.4.21 (i.e. so-called serine endopeptidases), including 3.4.21.1 (Chymotrypsin), 3.4.21.4 (Trypsin), 3.4.21.25 (Cucumisin), 3.4.21.32 (Brachyurin), 3.4.21.48 (Cerevisin) and 3.4.21.62 (Subtilisin);<br><br>
3.4.22	(i.e. so-called cysteine endopeptidases), including 3.4.22.2 (Papain), 3.4.22.3 (Ficain), 3.4.22.6 (Chymopapain), 3.4.22.7 (Asclepain), 3.4.22.14 (Actinidain), 3.4.22.30 (Caricain) and 3.4.22.31 (Ananain);<br>
3.4.23	(i.e. so-called aspartic endopeptidases), including 3.4.23.1 (Pepsin A), 3.4.23.18 (Aspergillopepsin I), 3.4.23.20 (Penicillopepsin) and 3.4.23.25 (Saccharopepsin); and<br>
3.4.24	(i.e. so-called metallo endopeptidases), including 3.4.24.28 (Bacillolysin).<br>
Examples of relevant subtilisins comprise subtilisin BPN', subtilisin amylosacchariticus, subtilisin 168, subtilisin mesentericopeptidase, subtilisin Carlsberg, subtilisin DY, subtilisin 309, subtilisin 147, thermitase, aqualysin. Bacillus PB92 protease, proteinase K, Protease TW7, and Protease TW3.<br>
Specific examples of such readily available commercial proteases include Esperase®, Alcalase®, Neutrase®, Dyrazym®, Savinase®, Pyrase®, Pancreatic Trypsin NOVO (PTN), Bio-Feed™ Pro, Clear-Lens Pro (all enzymes available from Novo Nordisk A/S).<br>
Examples  of  other  commercial	proteases  include  Maxatase®,<br>
Maxacal®, Maxapem® marketed by	Gist-Brocades N.V., Opticlean®<br>
marketed by Solvay et Cie. and	Purafect® marketed by Genencor<br>
International.<br>
I It is to be understood that also protease variants are contemplated as the polypeptide of interest. Examples of such protease variants are disclosed in EP 130.756 (Genentech), EP 214.435 (Henkel), WO 87/04461 (Amgen), WO 87/05050 (Genex), EP 251.446 (Genencor), EP 260.105 (Genencor), Thomas et al., (1985),<br>
r Nature. 318, p. 375-376, Thomas et al., (1987), J. Mol. Biol., 193, pp. 803-813, Russel et al., (1987), Nature, 328, p. 496-500,<br><br>
wo 88/08028  (Genex),  WO 88/08033  (Amgen),  WO 89/06279  (Novo Nordisk A/S),   WO 91/00345  (Novo Nordisk A/S),  EP 525 610 (Solvay) and WO 94/02618 (Gist-Brocades N.V.).<br>
The activity of proteases can be determined as described in "Methods of Enzymatic Analysis", third edition, 1984, Verlag Chemie, Weinheira, vol. 5.<br>
Lipases are here meant as enzymes classified under the Enzyme Classification number E.G. 3.1.1 (Carboxylic Ester Hydrolases) in accordance with the Recommendations (1992) of the International Union of Biochemistry and Molecular Biology (lUBMB).<br>
Examples include lipases selected from those classified under the Enzyme Classification (E.G.) numbers:<br>
3.1.1 (i.e. so-called Carboxylic Ester Hydrolases), including (3.1.1.3) Triacylglycerol lipases, (3.1.1.4.) Phospholipase Aj<br>
Examples of lipases include lipases derived from the following microorganisms. The indicated patent publications are in¬corporated herein by reference:<br>
Humicola, e.g. H. brevispora, H. lanuginosa, H. brevis var. thermoidea  and H.   insolens   (US 4,810,414)<br>
Pseudomonas, e.g. Ps. fragi, Ps. stutzeri, Ps. cepacia and Ps. fluorescens (WO 89/04361), or Ps. plantarii or Ps. gladioli (US patent no. 4,950,417 (Solvay enzymes)) or Ps. alcaligenes and Ps. pseudoalcaligenes (EP 218 272) or Ps. mendocina (WO 88/09367; US 5,389,536) .<br>
Fusarium, e.g. F.   oxysporum   (EP 130,064) or F.   solani pisi     (WO 90/09446) .<br>
Mucor   (also called Rhizomucor),   e.g.   M.   miehei   (EP 238 023).<br><br>
Chromobacterium   (especially C. viscosum)<br>
Aspergillus   (especially A.   niger) .<br>
Candida, e.g. C. cylindracea (also called C, rugosa) or C. antarctica (WO 88/02775) or C, antarctica lipase A or B (WO 94/01541 and WO 89/02916).<br>
Geotriciim, e.g. G. candidum (Schimada at al. , (1989), J. Biochem.,   106, 383-388)<br>
Penicillium, e.g. P. camenibertii (Yamaguchi et al., (1991), Gene 103, 61-67).<br>
Rhizopus, e.g. R. delemar (Hass et al., (1991), Gene 109, 107-113) or R. niveus (Kugimiya et al. , (1992) Biosci. Biotech. Biochem  56, 716-719) or R.   oryzae.<br>
Bacillus, e.g. B. subtilis (Dartois et al., (1993) Biochemica et Biophysica acta 1131, 253-260) or B. stearothermophilus (JP 64/7744992) or B.   pumilus   (WO 91/16422).<br>
Specific examples of readily available commercial lipases include Lipolase®, Lipolase™ Ultra, Lipozyme®, Palatase®, Novozym® 435, Lecitase® (all available from Novo Nordisk A/S) .<br>
Examples of other lipases are Lumafast™, Ps.    mendocina	lipase<br>
from Genencor Int. Inc.; Lipomax™, Ps.   pseudoalcaligenes	lipase<br>
from Gist Brocades/Genencor Int. Inc.; Fusarium   solani	lipase<br>
(cutinase) from Unilever; Bacillus sp. lipase from	Solvay<br>
enzymes. Other lipases are available from other companies.<br><br>
It is to be understood that also lipase variants are contemplated as the polypeptide of interest. Examples of such are described in e.g.   WO 93/01285 and WO 95/22615.<br>
'The activity of the lipase can be determined as described in "Methods of Enzymatic Analysis", Third Edition, 1984, Verlag Chemie, Weinhein, vol. 4, or as described in AF 95/5 GB (avail¬able on request from Novo Nordisk A/S).<br>
In this context the expression "carbohydrase" means all enzymes capable of breaking down carbohydrate chains {e.g. starches) of especially five and six member ring structures (i.e. enzymes classified under  the  Enzyme  Classification number  E.G.  3.2<br>
(glycosidases) in accordance with the Recommendations (1992) of the International Union of Biochemistry and Molecular Biology<br>
(lUBMB)). Also included in the group of carbohydrases according to the invention are enzymes capable of isomerizing carbohydrates e.g. six member ring structures, such as D-glucose to e.g. five member ring structures like D-fructose.<br>
Examples include carbohydrases selected from those classified under the Enzyme Glassification (E.G.) numbers:<br>
a-amylase (3.2.1.1) (3-amylase (3.2.1.2), glucan 1,4-a-glucosidase (3.2.1.3), cellulase (3.2.1.4), endo-l,3(4)- P-glucanase  (3.2.1.6),  endo-1,4-P-xylanase  (3.2.1.8),  dextranase<br>
(3.2.1.11), chitinase (3.2.1.14), polygalacturonase (3.2.1.15), lysozyme  (3.2.1.17),  (i-glucosidase  (3.2.1.21),  a-galactosidase<br>
(3.2.1.22),  P-galactosidase  (3.2.1.23),  amylo-1,6-glucosidase<br>
(3.2.1.33), xylan 1,4--xylosidase (3.2.1.37), glucan endo-l,3-P-D-glucosidase    (3.2.1.39),    a-dextrin   endo-1,6-glucosidase<br>
(3.2.1.41), sucrose a-glucosidase (3.2.1.48), glucan endo-1,3-a-glucosidase (3.2.1.59), glucan 1,4-P-glucosidase (3.2.1.74), glucan endo-1,6-P-glucosidase  (3.2.1.75),  arabinan endo-1,5-a-<br><br>
arabinosidase   (3.2.1.99),   lactase   (3.2.1.108),   chitonanase (3.2.1.132) and xylose isomerase (5.3.1.5).<br>
Examples of relevant carbohydrases include a-1,3-glucanases<br>
derived from Trichoderma. harzianum; a-1,6-glucanases derived from<br>
a strain of Paecilomyces; P-glucanases derived from Bacillus<br>
subtilis; P-glucanases derived from Humicola insolens; P-<br>
glucanases derived from Aspergillus niger; P-glucanases derived<br>
from a strain of Trichoderma; p-glucanases derived from a strain<br>
of Oerskovia xanthineolytica; exo-1,4-a-D-glucosidases (gluco-<br>
amylases) derived from Aspergillus niger; a-amylases derived from<br>
Bacillus        subtilis;	a-amylases   derived   from   Bacillus<br>
amyloliquefaciens;	a-amylases    derived    from    Bacillus<br>
stearotheijnophilus; a-araylases derived from Aspergillus oryzae; a-amylases derived from non-pathogenic microorganisms; a-galactosidases derived from Aspergillus niger; Pentosanases, xylanases, cellobiases, cellulases, hemi-cellulases derived from Humicola insolens; cellulases derived from Trichoderma reesei; cellulases derived from non-pathogenic mold; pectinases, cellulases, arabinases, hemi-celluloses derived from Aspergillus niger; dextranases derived from Penicillium lilacinuirt; endo-glucanase derived from non-pathogenic mold; pullulanases derived from Bacillus acidopullyticus; P-galactosidases derived from Kluyveromyces fragilis; xylanases derived from Trichoderma reesei;<br>
Specific examples of readily available commercial carbohydrases include Alpha-Gal™, Bio-Feed™ Alpha, Bio-Feed™ Beta, Bio-Feed™ Plus, Bio-Feed™ Plus, Novozyme® 188, Carezyme®, Celluclast®, Cellusoft®, Ceremyl®, Citrozym™, Denimax™, Dezyme™, Dextrozyme™, Finizym®, Fungamyl™, Gamanase™, Glucanex®, Lactozym®, Maltogenase™, Pentopan™, Pectinex™, Promozyme®, Pulpzyme™,  Novamyl™,  Termamyl®,  AMG  (Amyloglucosidase Novo),<br><br>
Maltogenase®, Sweetzyme®, Aquazym® (all enzymes available from Novo Nordisk A/S) . Other carbohydrases are available from other companies.<br>
It is to be understood that also variants of such carbohydrases are contemplated as the polypeptide of interest.<br>
The activity of carbohydrases can be determined as described in "Methods of Enzymatic Analysis", third edition, 1984, Verlag Chemie, Weinheim, vol. 4.<br>
Oxidoreductases are here meant to be enzymes classified under the Enzyme Classification number E.G. 1 (Oxidoreductases) in accordance with the Recommendations (1992) of the International Union of Biochemistry and Molecular Biology (lUBMB).<br>
Examples include oxidoreductases selected from those classified under the Enzyme Classification (E.C.) numbers:<br>
Glycerol-3-phosphate dehydrogenase  (NAD*) (1.1.1.8),  Glycerol-3-<br>
phosphate dehydrogenase NAD(P)* (1.1.1.94), Glycerol-3-phosphate<br>
1-dehydrogenase  (NADP)  (1.1.1.94),  Glucose oxidase  (1.1.3.4),<br>
Hexose oxidase (1.1.3.5), Catechol oxidase (1.1.3.14), Bilirubin<br>
oxidase  (1.3.3.5),  Alanine dehydrogenase  (1.4.1.1),  Glutamate<br>
dehydrogenase   (1.4.1.2),   Glutamate  dehydrogenase   (NAD(P)*)<br>
(1.4.1.3),  Glutamate dehydrogenase  (NADP")  (1.4.1.4),  L-Amino<br>
acid dehydrogenase  (1.4.1.5),  Serine dehydrogenase  (1.4.1.7),<br>
Valine dehydrogenase  (NADP*)  (1.4.1.8),  Leucine dehydrogenase<br>
(1.4.1.9), Glycine dehydrogenase (1.4.1.10), L-Amino-acid oxidase<br>
(1.4.3.2.),  D-Amino-acid oxidase (1.4.3.3),  L-Glutamate oxidase<br>
(1.4.3.11), Protein-lysine 6-oxidase (1.4.3.13), L-lysine oxidase<br>
(1.4.3.14),   L-Aspartate   oxidase   (1.4.3.16),   D-amino-acid<br>
dehydrogenase (1.4.99.1), Protein disulfide reductase (1.6.4.4),<br>
Thioredoxin reductase  (1.6.4.5),  Protein disulfide  reductase<br>
(glutathione) (1.8.4.2), Laccase (1.10.3.2), Catalase (1.11.1.6),<br>
Peroxidase  (1.11.1.7),  Lipoxygenase  (1.13.11.12),  Superoxide<br>
dismutase (1.15.1.1)<br><br>
Said Glucose oxidases may be derived from Aspergillus niger.<br>
Said  Laccases  may  be  derived  from  Polyporus      pinsitus, Myceliophtora   thermophila,   Coprinus  cinereus,   Rhizoctonia  solani,<br>
Ihizoctonia      praticola,       Scytalidium       thermophilum      and Rhus<br>
lernicifera.<br>
bilirubin oxidases may be derived from Myrothechecium verrucaria.<br>
The Peroxidase may be derived from e.g. Soy bean, Horseradish or "^loprinus cinereus.<br>
The Protein Disulfide reductase may be any mentioned in any of :he DK patent applications no. 768/93, 265/94 and 264/94 (Novo Tordisk A/S), which are hereby incorporated as reference, inclu-iing Protein Disulfide reductases of bovine origin. Protein Disulfide reductases derived from Aspergillus oryzae or Asper-jillus niger,   and DsbA or DsbC derived from Escherichia  coli.<br>
Specific examples of readily available commercial oxidoreductases Include Gluzyme™ (enzyme available from Novo Nordisk A/S) . lowever, other oxidoreductases are available from others.<br>
[t is to be understood that also variants of oxidoreductases are :ontemplated as the polypeptide of interest.<br>
The activity of oxidoreductases can be determined as described in 'Methods of Enzymatic Analysis", third edition, 1984, Verlag rhemie, Weinheim, vol. 3.<br>
En this context transferases are enzymes classified under the Enzyme Classification number E.G. 2 in accordance with the R.ecommendations (1992) of the International Union of Biochemistry and Molecular Biology (lUBMB).<br><br>
The transferases may be any transferase in the subgroups of transferases: transferases transferring one-carbon groups (E.G. 2.1); transferases transferring aldehyde or residues (E.G. 2.2); acyltransferases (E.G. 2.3); glucosyltransferases (E.G. 2.4); ' transferases transferring alkyl or aryl groups, other that methyl groups (E.G. 2.5); transferases transferring nitrogenous groups (2.6).<br>
In a preferred embodiment the transferease is a transglutaminase E.G 2.3.2.13 (Protein-glutamine y-glutamyltransferase) .<br>
Transglutaminases are enzymes capable of catalysing an acyl transfer reaction in which a y-carboxyamide group of a peptide-bound glutamine residue is the acyl donor. Primary amino groups in a variety of compounds may function as acyl acceptors with the subsequent formation of mono-substituted y-amides of peptide-bound glutamic acid. When the epsilon-amino group of a lysine residue in a peptide-chain serves as the acyl acceptor, the transferases form intramolecular or intermolecular y-glutamyl-e-lysyl crosslinks.<br>
Examples of transglutaminases are described in the pending DK patent application no. 990/94 (Novo Nordisk A/S).<br>
The transglutaminase may the of human, aminal (e.g. bovine) or microbially origin.<br>
Examples of such transglutaminases are animal derived transglutaminases, FXIIIa; microbial transglutaminases derived from Physarum polycephalum (Klein et al., Journal of Bacteriol¬ogy, 174, p. 2599-2605); transglutaminases derived from Strep-tomyces sp., including Streptomyces lavendulae, Streptomyces lydicus (former Streptoznyces libani) and StreptoverticiIlium sp., including Streptoverticillium mobaraense, Streptoverticillium cinnamoneum, and Streptoverticillium griseocarneum (Motoki et al.,   US 5,156,956; Andou et al. ,   US 5,252,469; Kaempfer et al.,<br><br>
uourna.1 oz ueneraj. lyiicromoiogy, J.J /, laji-iay^; ocni et al. , International Journal of Sytematic Bacteriology, 44, 285-292; Andou et al. , US 5,252,469; Williams et al. , Journal of General Microbiology,   129, 1743-1813) .<br>
It is to be understood that also transferase variants are contemplated as the polypeptide of interest.<br>
The activity of transglutaminases can be determined as described in "Methods of Enzymatic Analysis", third edition, 1984, Verlag Chemie, Weinheim, vol. 1-10.<br>
In this context phytases are enzymes classified under the Enzyme Classification number E.G. 3.1.3 (Phosphoric Monoester Hydrolases) in accordance with the Recommendations (1992) of the International Union of Biochemistry and Molecular Biology (lUBMB).<br>
Phytases are enzymes produced by microorganisms which catalyse the conversion of phytate to inositol and inorganic phosphorus<br>
Phytase producing microorganisms comprise bacteria such as Bacillus subtilis. Bacillus natto and Pseudomonas; yeasts such as Saccharomyces cerevisiae; and fungi such as Aspergillus niger, Aspergillus ficuum, Aspergillus awamori, Aspergillus oryzae, Aspergillus terreus or Aspergillus nidulans, and various other Aspergillus  species).<br>
Examples of phytases include phytases selected from those classified under the Enzyme Classification (E.C.) numbers: 3-phytase (3.1.3.8) and6-phytase (3.1.3.26).<br>
The activity of phytases can be determined as described in "Methods of Enzymatic Analysis", third edition, 1984, Verlag Chemie, Weinheim, vol. 1-10, or may be measured according to the method described in EP-Al-0 420 358, Example 2 A.<br><br>
In the present context an anti-microbial polypeptides may be any polypeptide exhibiting anti-microbial activities, such as anti¬fungal, anti-bacterial, and/or anti-insecticidal activity.<br>
Such polypeptides may also exhibit other activities such as enzymatic activity.<br>
Examples of anti-microbial polypeptides according to the invention include: fungicidally active polypeptides derived from the mold genus Curvularia described in WO 94/01459 (Novo Nordisk A/S) ; anti-bacterial polypeptides described in EP 403.458 (Kabigen AB); anti-microbial proteins isolated from the Mirabilis seed, described in WO 92/15691 (Imperial Chem Ind. PLC) ; anti¬bacterial polypeptides isolated from an extract of pig small intestine, described in WO 92/22578 (Boman et al.); polypeptide with yeast lethal action accumulated by yeast of Hansenula spp. as described in JP-60130599; Phytolacca insularis antiviral protein, which can be used as an anti-microbial described in US patent no. 5,348,865 (Jin Ro LTD.); bacteriolytic enzymes preparations derived from Nocardiopsis dassonvillei described in US patent no. 5,354,681 (Novo Industri A/S).<br>
Examples of other anti-microbial polypeptides are maganinin, protegrin, defensin, pseudomycin, mutanolysin and N-acetylmuramidase.<br>
The present invention in a further aspect relates to a method for generating a DNA sequence encoding a desired variant of a polypeptide of interest, wherein<br>
(i)    a mutant library is produced by the above method, (ii)   said library is cultivated under conditions conducive for the expression of said gene of interest to produce variant polypeptides.<br><br>
(iii) said variant polypeptides are screened or selected for a desired property, and hosts producing desired variants identified and isolated,<br>
(iv)   the DNA encoding said variants is isolated.<br>
The present invention in a still further aspect relates to a method for the determination of a DNA sequence encoding a desired variant of a polypeptide of interest, wherein (i)    a mutant library is produced as described above, (ii)   said library is cultivated under conditions conducive for<br>
the expression of said gene of interest to produce variant<br>
polypeptides, (iii)  said variant polypeptides are screened or selected for a<br>
desired property, and hosts producing desired variants<br>
identified and isolated, (iv)   said genetic element  in said hosts  is  sequenced to<br>
elucidate the DNA sequence of the mutant gene encoding a<br>
desired variant.<br>
This aspect of the invention can be performed by making dilutions of the library (e.g. in microtiter plates) and culturing these, whereby populations are made each originating from one member of the library, and the variant polypeptide produced from each of the populations screened for the desired properties. Alternatively the library might be plated on agar-plates containing a desired growth medium that allows for the screening of or selection for desired properties of the variant polypeptide.<br>
If the phage display method is used, the screening or selection is performed directly with the phages.<br>
The criteria used for the selection will vary according to the end use of the polypeptide variant of interest, but properties typically being tested may include solubility and half-life in various media, antigenicity and allergenicity, thermal stability, oxidation stability, storage stability, substrate specificity and<br><br>
affinity, stability to non-aqueous solvents, pH profile, ionic strength dependence, catalytic efficiency, and compatibility with other components of envisaged end products wherein the polypeptide variant will form a part.<br>
For enzymes to be used in detergents further properties to be investigated are, wash performance and compatibility with various surfaces, especially fabrics.<br>
Numerous other criteria could be mentioned.<br>
Upon identification of populations that produce variant polypeptides fulfilling the criteria selected, the DNA encoding the polypeptide variant of interest is isolated and sequenced by use of methods well known in the art.<br>
The invention furthermore comprises a process for the production of a desired polypeptide variant, wherein (i)    a DNA sequence determined as indicated above is introduced<br>
into a suitable host in a manner whereby it can be<br>
expressed in said host, (ii)   said host is cultivated under conditions conducive to the<br>
expression of said DNA sequence, and (iii)  said polypeptide variant is recovered.<br>
The present invention can be used with any cell, especially any microbial cell, but it is often suitable to use a prokaryote, especially a bacterium, preferably of the genus Bacillus,   etc.<br>
Among the Bacilli it is preferred to use a strain chosen from the<br>
group   comprising   B.   lentus,	B.	licheniformis,	B.<br>
amyloliquefaciens,   B.   subtilis  ,   etc.<br>
For some uses it is preferable to use a microbial cell which is a fungus, especially a filamentous fungus, preferably of the genus Aspergillus,   Trichoderma,   etc.<br><br>
Among the Aspergilli it is preferable to use a strain chosen from the group comprising A.   oryzae,   A.   niger,   A.   awamori,   etc.<br>
Among the Trichoderma it is preferable to use a strain chosen from the group comprising T. reseei,   etc.<br>
In yet other situations it is more expedient to use a mammalian cell chosen from the group comprising BHK, etc. cells.<br>
The invention should not be construed to be limited to specific examples or embodiments mentioned in the specification above or the following examples.<br>
MATERIALS AND METHODS<br>
EXAMPLES<br>
EXAMPLE 1<br>
The system used is an Eschericia    coli    host cell,  which is<br>
characterized by a number of chromosomal mutations:<br>
i)     a ts (thermosensitive) mutation in the polC  gene (encoding<br>
DNA polymerase III, being the main replication polymerase) ii)    a mutation in the polA   gene (encoding DNA polymerase I)<br>
causing an increased error rate by a reduction in the 3'-<br>
5' exonuclease activity, iii)   repair deficiency by the mutL mutation.<br>
The target for the in vivo mutagenesis is plasmid pBR322 (colEl origin) having either (i) a frame shift mutation, or (ii) a stop codon introduced into the tet gene, encoding a protein conferring resistance towards tetracyclin.<br><br>
En each case the repair of the mutation leads to a dominant :etracycline resistant phenotype.<br>
pBR322 contains also the bla gene conferring resistance towards ampicillin. The higher mutagenesis frequency at the target region is seen as a higher frequency of tetracycline resistant colonies after plating a culture exposed to "mutation-introduction" conditions.<br>
\n E.    coli    culture grown at 37°C to an optical density of 1<br>
neasured at 60 0 nm is exposed to 2, 4 or 16 hours at restrictive<br>
:emperature, e.g. 42°C. At these time points dilution series of<br>
:he cultures are plated on LBagar supplementet with<br>
L) ampicillin (AmpR colonies)<br>
2) tetracycline and ampicillin.(AmpR and tetR colonies)<br>
rhe ratio of tetracycline resistant colonies to ampicillin resistant colonies indicate the number of cells in the culture :hat contains one copy of a repaired tet gene, indicated one specific mutagenesis event.<br>
This means, if a clone have become tetracycline resistant, at Least one specific mutation has occurred to repair the originally introduced gene defect.<br><br>
REFERENCES CITED IN THE SPECIFICATION<br>
Greg Winter, Current methods  in Immunology  5: 253-255, 1993<br>
Griffiths, A.D. et  al.,   1994, EMBOJ.   14: 3245-3260.<br>
Clackson et al.,   Nature  352: 624-628, 1991<br>
Bryan, P et al.,   Proteins  1: 326-334, 1986<br>
Sambrook et al.    Molecular Cloning: A Laboratory Manual. Cold<br>
Spring Harbor Lab., Cold Spring Harbor, NY.<br>
Scopes, R.K.,Protein Purification (1987), Springer-Verlag<br>
Kim and Loeb (1995, PNAS  92: 684-688)<br>
0rum, P. et al., Nucl.   Acid.   Res.   21: 4491-4498, 1993<br>
EP 130.756 (Genentech), EP 214.435 (Henkel), WO 87/04461 (Amgen), WO  87/05050   (Genex),   EP  251.446   (Genencor),   EP  260.105 (Genencor), Thomas et al.    (1985) Nature   318, 375-376, Thomas et al.     (1987) J.    Mol.    Biol.,     193 803-813, Russel et al.     (1987) Nature   328 496-500, WO 88/08028 (Genex), WO 88/08033 (Amgen), WO 89/06279 (Novo Nordisk A/S), WO 91/00345 (Novo Nordisk A/S), EP 525  610  (Solvay)  and WO  94/02618  (Gist-Brocades  N.V.),  US 4,810,414, WO 89/04361, US patent no. 4,950,417 (Solvay enzymes), EP 218 272, WO 88/09367, US 5,389,536, EP 130,064, WO 90/09446), EP 238 023, WO 88/02775, WO 94/01541, WO 89/02916, Schimada et al.,    (1989), J.   Biochem.,   106, 383-388, Yamaguchi et al.,    (1991), Gene  103, 61-67, Hass et al.,    (1991), Gene  109, 107-113, Kugimiya et al.,    (1992) Biosci.   Biotech.   Biochem   56, 716-719, Dartois et al.,      (1993) Biochemica    et    Biophysica    acta    1131,  253-260,  JP 64/7744992,  WO 91/16422,  WO 93/01285, WO 95/22615,  DK patent applications no. 768/93, 265/94 and 264/94 (Novo Nordisk A/S), DK patent application no. 990/94 (Novo Nordisk A/S), Klein et al.. Journal  of Bacteriology,   174, p. 2599-2605, Motoki et al., US 5,-156,956; Andou et al.,   US 5,252,469; Kaempfer et al.,   Journal   of General  Microbiology,   137, 1831-1892; Ochi et al.,   International Journal   of Sytematic Bacteriology,   44, 285-292; Andou et al.,   US 5,252,469; Williams et al.,   Journal  of General Microbiology,   129, 1743-1813, WO 94/01459 (Novo Nordisk A/S), EP 403.458 (Kabigen<br><br>
AB) , WO 92/15691 (Imperial Chem Ind. PLC), WO 92/22578 (Boman et al.), JP-60130599, US patent no. 5,348,865 (Jin Ro LTD.), US patent no. 5,354,681 (Novo Industri A/S),<br>
Methods of Enzymatic Analysis,  third edition,  1984,  Verlag<br>
Chemie, Weinheim, vol. 1-10.<br>
AF 95/5 GB (available on request from Novo Nordisk A/S)<br><br><br>
WE CLAIM :<br>
1.	A method for in vivo production of a library of variants of a gene encoding a<br>
polypeptide of interest in a cell  wherein an error-prone polymerase is used to<br>
replicate all or a part of a genetic element independently of the cells<br>
chromosomal replication, the genetic element comprising<br>
i)        an origin of replication from which replication is initiated,<br>
ii)       optionally a genetic marker, e.g. a gene conferring resistance towards an<br>
antibiotic, and iii)      the gene encoding the polypeptide of interest.<br>
2.	The method as claimed in claim 1, wherein said cell further comprises a deficient repair system.<br>
3.	The method as claimed in any of claims 1 - 2, wherein said genetic element is a plasmid, a phagemid, a phage, a virus, a retrovirus or a retrotransposon.<br>
4.	The method as claimed in any of claims 1-3, wherein said cell is a microbial cell.<br>
5.	The method as claimed in any of claims 1-4, wherein said cell is a prokaryote, especially a bacterium, preferably of the genus Bacillus, Escherichia, Staphylococcus, and Streptococus.<br>
6.	The method as claimed in claim 5, wherein said Escherichia is a strain chosen from E. coli.<br><br>
7.	The method as claimed in claim 5, wherein said Bacillus is a strain chosen from the group comprising B. lentus, B. licheniformis, B. amyloliquefaciens, B. subtilis.<br>
8.	The method as claimed in any as claimed in claims 1-4, wherein said cell is a fungus, especially a yeast or a filamentous fungus, preferably of the genus Aspergillus, Trichoderma.<br>
9.	The method as claimed in claim 8, wherein said Aspergillus is chosen from the group comprising A. oryzae, A. niger, A. awamori.<br>
10.	The method as claimed in claim 8, wherein said Trichoderma is chosen from the group comprising T. reseei.<br>
f 11.')   The method as claimed in any of claims 1-3, wherein said cell is a mammalian cell chosen from the group comprising BHK cells or an insect cell.<br>
12.	The method as claimed in any of claims 1-11, wherein said error-prone polymerase is selected from the group comprising DNA pol I, DNA pol II, reverse transcriptase and more specifically E.coli DNA pol I, Bacillus subtilis DNA pol I, HIV reverse transcriptase, T4 DNA polymerase, T7 DNA polymerase, Phi29 DNA polymerase.<br>
13.	The method as claimed in any of claims 1-12, wherein said polypeptide of interest is an enzyme.<br><br>
14.	The method as claimed in claim 13, wherein said enzyme is a carbonyl<br>
hydrolases, carbohydrases, oxidoreductases, transferases, phytases, ligases,<br>
lyases, and anti-microbial polypeptides.<br>
15.	The method as claimed in claim 14, wherein said carbonyl hydrolase is a<br>
protease, or a lipase.<br>
16.	The method as claimed in claim 14, wherein said carbohydrase is an amylase, glucosidase, cellulase, glucanase, xylanase, dextranase, chitinase, polygalacturonase, lysozyme, glucosidase, galactosidase, xylosidase, arabinosidase, lactase, chitonanase, xylose isomerase, pectin esterase, rhamnogalacturonase, endo-glucanase.<br>
17.	The method as claimed in claim 14, wherein said oxidoreductase is a dehydrogenase, oxidase, reductase, Laccase, Catalase, Peroxidase, Lipoxygenase, Superoxide dismutase.<br>
18.	The method as claimed in claim 14, wherein said transferase is a transferase transferring one-carbon groups, a transferase transferring aldehyde or residues, acyltransferase, glucosyltransferase, transferase transferring alkyl or aryl groups, other that methyl groups, transferase transferring nitrogeneous groups.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDkyLW1hcy0xOTk3IGFic3RyYWN0LWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">092-mas-1997 abstract-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDkyLW1hcy0xOTk3IGNsYWltcy1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">092-mas-1997 claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDkyLW1hcy0xOTk3IGRlc2NyaXB0aW9uIChjb21wbGV0ZSktZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">092-mas-1997 description (complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTItbWFzLTE5OTcgYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">92-mas-1997 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTItbWFzLTE5OTcgYXNzaWdubWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">92-mas-1997 assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTItbWFzLTE5OTcgY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">92-mas-1997 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTItbWFzLTE5OTcgY29ycmVzcG9uZGVuY2Ugb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">92-mas-1997 correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTItbWFzLTE5OTcgY29ycmVzcG9uZGVuY2UgcG8ucGRm" target="_blank" style="word-wrap:break-word;">92-mas-1997 correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTItbWFzLTE5OTcgZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">92-mas-1997 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTItbWFzLTE5OTcgZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">92-mas-1997 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTItbWFzLTE5OTcgZm9ybS0xOC5wZGY=" target="_blank" style="word-wrap:break-word;">92-mas-1997 form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTItbWFzLTE5OTcgZm9ybS0yNi5wZGY=" target="_blank" style="word-wrap:break-word;">92-mas-1997 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTItbWFzLTE5OTcgZm9ybS00LnBkZg==" target="_blank" style="word-wrap:break-word;">92-mas-1997 form-4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTItbWFzLTE5OTcgZm9ybS02LnBkZg==" target="_blank" style="word-wrap:break-word;">92-mas-1997 form-6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTItbWFzLTE5OTcgb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">92-mas-1997 others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTItbWFzLTE5OTcgcGV0aXRpb24ucGRm" target="_blank" style="word-wrap:break-word;">92-mas-1997 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="229904-carbohydrate-derivatives.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="229906-tobacco-products-with-vitamin-e.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>229905</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>92/MAS/1997</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>24-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>20-Jan-1997</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NOVOZYMES A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>KROGSHOEJVEJ 36, DK-2880 BAGSVAERD,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>STANISLAS DUSKO EHRLICH</td>
											<td>C/O NOVO NORDISK A/S, NOVO ALLE, DK-2880 BAGSVAERD,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>TORBEN VEDEL BORCHERT</td>
											<td>C/O NOVO NORDISK A/S, NOVO ALLE, DK-2880 BAGSVAERD,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12P21/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/229905-a-method-for-in-vivo-production-of-a-library-of-variants-of-a-gene-encoding-a-polypeptide by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:38:01 GMT -->
</html>
