<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/230236-gene-conferring-resistance-to-phytophthora-infestans-late-blight-in-solanacea by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:59:45 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 230236:GENE CONFERRING RESISTANCE TO PHYTOPHTHORA INFESTANS (LATE-BLIGHT) IN SOLANACEA</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">GENE CONFERRING RESISTANCE TO PHYTOPHTHORA INFESTANS (LATE-BLIGHT) IN SOLANACEA</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>ABSTRACT 1968/CHENP/2004 &quot;Gene conferring resistance to phvtophthora infestans (late-blight) in Solanacea&quot; The present invention relates an isolated or recombinant nucleic acid comprising (a) a nucleic acid as depicted in Fig. 6; (b) a nucleic acid coding for the amino acid sequence of Fig. 8; or (c) a homologous nucleic acid sequence of (a) and/or (b) that is capable of providing at least partial resistance against an oomycete infection when incorporated and expressed in a Solanaceae plant or plant cell, and that encodes an amino acid sequence which is at least 82% identical to the amino acid sequence of Fig. 8.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
GENE CONFERRING RESISTANCE TO PHYTO§§THERA INFESTANS (LATE-BLIGHT)<br>
IN SOLANACEAE<br>
Late blight, caused by the oomycete pathogen Phytophthora infestans is world-wide the most destructive disease for potato cultivation. The disease also threatens the tomato crop. The urgency of obtaining resistant cultivars has intensified as more virulent, crop-specialised and pesticide resistant strains of the pathogen are rapidly emerging.<br>
A way to prevent crop failures or reduced yields is the application of fungicides that prevent or cure an infection by P. infestans. However, the application of crop protectants is widely considered to be a burden for the environment. Thus, in several Western countries, legislation is becoming more restrictive and partly prohibitive to the application of specific fungicides, making chemical control of the disease more difficult. An alternative approach is the use of cultivars that harbour partial or complete resistance to late blight. Two types of resistance to late blight have been described and used in potato breeding. One kind is conferred by a series of major, dominant genes that render the host incompatible with specific races of the pathogeri (race specific resistance). Eleven such R genes (Rl-Rll) have been identified and are believed to have originated in the wild potato species Solanum demissum, which is native to Mexico, where the greatest genetic variation of the pathogen is found. Several of these B genes have been mapped on the genetic map of potato (reviewed in Gebhardt and Valkonen, 2001 Annu. Rev. Phytopathol. 39: 79-102). Rl and R2 are located on chromosomes 5 and 4, respectively. R3, R6 and R7 are located on chromosome 11. Unknown R genes conferring race specific resistance to late blight have also been described in S. tuberosum ssp. andigena and S. berlhaultii (Ewing et al., 2000 Mol. Breeding 6: 25-36). Because of the high level of resistance and ease of transfer, many cultivars contain S. demissum derived resistance. Unfortunately, 5. demissum derived race specific resistance, although nearly complete, is not durable. Once newly bred cultivars are grown on larger scale in commercial fields, new virulences emerge in P. infestans that render the pathogen able to overcome the introgressed resistance. The second type of resistance, termed field resistance and often quantitative in nature, is thought to be race non-specific and more durable. Field resistance to late blight can be found in several Mexican and Middle and South American Solanum species (Rossi et al., 1986 PNAS 95:9750-9754).<br><br>
piploid S. bulbocastanum from Mexico and Guatemala is one of the tuber bearing species that is known for its high levels of field resistance to late blight (Niederhauser and Mills, 1953 Phytopathology 43: 456-457). Despite differences in endosperm balance numbers, introgression of the S. bulbocastanum resistance trait has been successful. Ploidy manipulations and a series of tedious bridge crosses has resulted in S. bulbocastanum derived, P. infestans resistant germplasm (Hermsen and Ramanna, 1969 Euphytica 18:27-35;  1973 Euphytica 22:457-466; Ramanna and Hermsen, 1971 Euphytica 20:470-481; Hermsen and De Boer, 1971 Euphytica 20:171-180). However, almost 40 years after the first crosses and intense and continuous breeding efforts by potato breeders in the Netherlands with this germplasm, late blight resistant cultivars still remain to be introduced on the market. Successful production of somatic hybrids of S. bulbocastanum and S. tuberosum has also been reported (Thieme et al, 1997 Euphytica 97(2): 189-200; Helgeson et al., 1998 Theor Appl. Genet 96:738-742). Some of these hybrids and backcrossed germplasm were found to be highly resistant to late blight, even under extreme disease pressure. Despite reports of suppression of recombination, resistance in the backcrossed material appeared to be on chromosome 8 within an approximately 6 cM interval between the RFLP markers CP53 and CT64 (Naess et al, 2000 Theor. Appl Genet 101:697-704). A CAPS marker derived from the tomato RFLP probe CT88 cosegregated with resistance. Suppression of recombination between the S. bulbocastanum and S. tuberosum chromosomes forms a potential obstacle for successful reconstitution of the recurrent cultivated potato germplasm to a level that could meet the standards for newly bred potato cultivars. Isolation of the genes that code for resistance found in S. bulbocastanum and subsequent transformation of existing cultivars with these genes, would be a much more straight forward and quicker approach when compared to introgression breeding.<br>
The cloning and molecular characterisation of numerous plant R genes conferring disease resistance to bacteria, fungi, viruses, nematodes, and insects has identified several structural features characteristic to plant R genes (reviewed in Dangl and Jones, 2001 Nature 411, 826-833). The majority are members of tightly linked multigene families and all R genes characterised so far, with the exception of Pto, encode leucine-rich repeats (LRRs), structures shown to be involved in protein-protein interactions. LRR containing R genes can be divided into two classes based on the pi-esence of a putative tripartite nucleotide-binding site (NBS). R genes of the NBS-LRR class comprise motifs that are shared with animal apoptosis regulatory<br><br>
prtoteins <van der biezen et al. curr. biol. aravind trends biochem. sci. and can be subdivided into two subgroups based on their n-terminal domain which either exhibits sequence similarity to the drosophila toll protein mammalian interleukin-1 receptor or contains a potential leucine zipper coiled-coil pan genetics. lrri genes without an nbs encode transmembrane proteins whose extracellular region is composed of lrus adv. bot. res.24 these divided presence cytosolic serine kinase science four r have currently been cloned from potato. all including s. demissum derived rl gene conferring race specific resistance late blight belong cc-nbs-lrr class plant cell bendahmane j. van vossen journal ballvora></van>
The invention provides an isolated or recombinant nucleic acid comprising a nucleic acid coding for the amino acid sequence of fig. 8 or a functional fragment or a homologue thereof. The protein coded by said amino acid has been detected as being member of a cluster of genes identifiable by phylogenetic tree analysis, which thus far consists of the proteins Rpi-blb, RGCl-blb, RGC3-blb and RGC4-blb (herein also called the Rpi-blb gene cluster) of figure 9.<br>
Phylogenetic tree analysis is carried out as follows. First a multiple sequence alignment is made of the nucleic acid sequences and/or preferably of the deduced amino acid sequences of the genes to be analysed using CLUSTALW (http://www2.ebi.ac.uk/clustalw), which is in standard use in the art. ClustalW produces a .dnd file, which can be read by TREEVIEW<br>
(http://taxonomy.zoology.gla.ac.uk/rod/rod.html). The phylogenetic tree depicted in Figure 9A is a phylogram.<br>
Phylogenetic studies of the deduced amino acid sequences of Rpi-blb, RGCl-blb, RGC3-blb, RGC4-blb and those of the most similar genes from the art (as defined by the BLASTX) derived from diverse species, using the Neighbour-Joining method of Saitou and Nei (1987 Molecular Biology and Evolution 4, 406-425), shows that corresponding genes or functional fragments thereof of the Rpi-blb gene cluster can be placed in a separate branch (Figure 9A).<br>
f,<br><br>
Sequence comparisons between the four members of the Rpi-blb gene cluster identified on 8005-8 BAC clone SPB4 show that sequence homology within the Rpi-blb gene cluster varies between 70% and 81% at the amino acid sequence level. The deduced amino acid sequence of Rpi-blb shares the highest overall homology with RGC3-blb (Sl%&gt; amino-acid sequence identity; Table 4). When the different domains are compared it is clear that the effector domains present in the N-terminal halves of the proteins (coiled-coil and NBS-ARC domains) share a higher degree of homology (91% sequence identity) than the C-terminal halves of these proteins which are thought to contain the recognition domains (LRRs; 71% amino acid sequence identity). Comparison of all four amino-acid sequences revealed a total of 104 Rpi-blb specific amino acid residues (Figure 10). The majority of these are located in the LRR region (80/104). Within the latter region, these specific residues are concentrated in the LRR subdomain xxLxLxxxx. The relative frequency of these specific amino-acid residues within this LRR subdomain is more than two times higher (28.3%) than that observed in the rest of the LRR domain (12.3%). The residues positioned around the two conserved leucine residues in the consensus xxLxxLxxxx are thought to be solvent exposed and are therefore likely to be involved in creating/maintaining recognition specificity of the resistance protein.<br>
Sequences of additional members of the Rpi-blb gene cluster can be obtained by screening genomic DNA or insert libraries, e.g. BAC libraries with primers based on signature sequences of the Rpi-blb gene. Screening of various Solarium BAC libraries with primer sets A and/or B (Table 2 and Figure 7) identified numerous Rpi-blb homologues derived from different Solarium species. Alignment of these additional sequences with those presented in Figure 10 will help identify additional members of the Rpi-blb gene cluster and specific amino acid residues therein responsible for P. infestans resistance specificity. Furthermore, testing additional sequences in the above described phylogenetic tree analyses, e.g. using the Neighbour-Joining method of Saitou and Nei (1987 Molecular Biology and Evolution 4, 406-425), provides additional identification of genes belonging to the Rpi-blb gene cluster.<br>
The invention provides the development of an intraspecific mapping population of S. bulbocastanuni that segregated for race non-specific resistance to late blight. The resistance was mapped on chromosome 8, in a region located 0.3 cM distal from CT88. Due to the race non-specific nature of the resistance, S. bulbocastanuni late blight resistance has always been thought to be R gene<br><br>
in'de pen dent. However, with the current invention we demonstrate for the first time that S. bulbocastanum race non-specific resistance is in fact conferred by a gene bearing similarity to an R gene of the NBS-LRR type.<br>
The invention further provides the molecular analysis of this genomic region and the isolation by map based cloning of a DNA-fragment of the resistant parent that harbours an R gene, designated Rpi-blb. This DNA-fragment was subcloned from an approximately 80 kb bacterial artificial chromosome (BAC) clone which contained four complete R gene-like sequences in a cluster-like arrangement. Transformation of a susceptible potato cultivar by Agrobacterium tumefaciens revealed that one of the four R gene-like sequences corresponds to Rpi-blb that provides the race non-specific resistance to late blight. Characterisation of the Rpi-blb gene showed that it is a member of the NBS-LRR class of plant R genes. The closest functionally characterised sequences of the prior art are members of the 12 resistance gene family in tomato. These sequences have an overall amino acid sequence identity of approximately 32% with that of Rpi-blb.<br>
Thus, in a first embodiment, the invention provides an isolated or recombinant nucleic acid, said nucleic acid encoding a gene product having the sequence of i?pt-blb or a functional fragment thereof that is capable of providing a member of the Solanaceae family with race non-specific resistance against an oomycete pathogen.<br>
Isolation of the gene as provided here that codes for the desired resistance trait against late blight and subsequent transformation of existing potato and tomato cultivars with this gene now provides a much more straightforward and quicker approach when compared to introgression breeding. The results provided here offer possibilities to further study the molecular basis of the plant pathogen interaction, the ecological role of JR genes in a wild Mexican potato species and are useful for development of resistant potato or tomato cultivars by means of genetic modification.<br>
In contrast to the R genes cloned and described so far, the gene we provide here is the first isolated R gene from a Solarium species that provides race non¬specific resistance against an oomycete pathogen. Notably, the invention provides here a nucleic acid wherein said Solarium species that is provided with the desired resistance comprises S. tuberosum. In particular, it is the first gene that has been isolated from a phylogenetically distinct relative of cultivated potato, S. bulbocastanum, for which it was shown by complementation assays, that it is functional in 5. tuberosum. These data imply that the gene Rpi-blb can easily be<br>
£<br><br>
applied in potato production without a need for time-consuming and complex introgression breeding.<br>
The following definitions are provided for terms used in the description and examples<br>
that follow.<br>
Nucleic acid: a double or single stranded DNA or RNA molecule. Oligonucleotide: a short single-stranded nucleic acid molecule. Primer: the term primer refers to an oligonucleotide that can prime the synthesis of nucleic acid.<br>
Homology: homology is the term used for the similarity or identity of biological sequence information. Homology may be found at the nucleotide sequence and/or encoded amino acid sequence level. For calculation of precentage identity the BLAST algorithm can be used (Altschul el al., 1997 Nucl. Acids Res. 25:3389-3402) using default parameters or, alternatively, the GAP algorithm (Needleman and Wunsch, 1970 J. Mol. Biol. 48:443-453), using default parameters, which both are included in the Wisconsin Genetics Software Package, Genetics Computer . Group (GCG), 575 Science Dr., Madison, Wisconsin, USA. BLAST searches assume that proteins can be modelled as random sequences. However, many real proteins comprise regions of nonrandom sequences which may be homopolymeric tracts, short-period repeats, or regions enriched in one or more amino acids. Such low-complexity regions may be aligned between unrelated proteins even though other regions of the protein are entirely dissimilar. A number of low-complexity filter programs can be employed to reduce such low-complexity alignments. For example, the SEG (Wooten and Federhen, 1993 Comput. Chem. 17:149-163) and XNU (Claverie and States, 1993 Comput. Chem. 17:191-201) low-complexity filters can be employed alone or in combination.<br>
As used herein, 'sequence identity' or 'identity' in the context of two protein sequences (or nucleotide sequences) includes reference to the residues in the two sequences which are the same when aligned for maximum correspondence over a specified comparison window. When percentage of sequence identity is used in reference to proteins it is recognised that residue positions which axe not identical often differ by conservative amino acid substitutions, where amino acids are substituted for other amino acid residues with similar chemical properties (e.g. charge or hydrophobicity) and therefore do not change the functional properties of the molecule. Where sequences differ in conservative substitutions, the percentage .<br><br>
sequence identity may be adjusted upwards to correct for the conservative nature of the substitutions. Sequences, which differ by such conservative substitutions are said to have 'sequence similarity' or 'similarity'. Means for making these adjustments are well known to persons skilled in the art. Typically this involves scoring a conservative substitution as a partial rather than a full mismatch, thereby increasing the percentage sequence identity. Thus, for example, where an identical amino acid is given a score of 1 and a non-conservative substitution is give a score of zero, a conservative substitution is given a score between 0 and 1. The scoring of conservative substitutions is calculated, e.g.   according to the algorithm of Meyers and Miller (Computer Applic. Biol. Sci. 4:11-17, 1988). As used herein, 'percentage of sequence identity' means the value determined by comparing two optimally aligned sequences over a comparison window, wherein the portion of the amino acid sequence or nucleotide sequence in the comparison window  may comprise additions or deletions  (i.e.,  gaps)  as  compared to  the reference sequence for optimal alignment of the two sequences. The percentage is calculated by determining the number of positions at which the identical amino acid or nucleic acid base residue occurs in both sequences to yield the number of matched positions, dividing the number of matched positions by the total number of positions in the window of comparison and multiplying the result by 100 to yield the percentage of sequence identity. Preferably the amino acid sequence of the protein of the invention shares at least 82% or higher homology with the sequence as depicted in Fig. 8. As shown in Table 4, the closest functionally characterised sequence of the prior art (members of the 12 Fusarium resistance gene cluster in tomato) has a much lower level of amino acid, sequence identity than this (32% with respect to that of Rpi-blb). Homology within the gene cluster of the present invention varies between 70% and 81% at the amino acid sequence level.<br>
Homologous nucleic acid sequences are nucleic acid sequences coding for a homologous protein defined as above. One example of such a nucleic acid is the sequence as provided in figure 6A. However, there are many sequences which code for a protein which is 100% identical to the protein as depicted in fig. 8. This is due to the 'wobble' in the nucleotide triplets, where more than one triplet can code for one and the same amino acid. Thus, even without having an effect on the amino acid sequence of the protein the nucleotide sequence coding for this protein can be varied substantially. It is acknowledged that nucleotide sequences coding<br><br>
for amino acid sequences that are not 100% identical to said protein can contain even more variations. Therefore, the percentage identity on nucleic acid sequence level can vary within wider limits, without departing from the invention.<br>
Promoter, the term "promoter" is intended to mean a short DNA sequence to which RNA polymerase and/or other transcription initiation factors bind prior to transcription of the DNA to which the promoter is functionally connected, allowing transcription to take place. The promoter is usually situated upstream . (5') of the coding sequence. In its broader scope, the term "promoter" includes the RNA polymerase binding site as well as regulatory sequence elements located within several hundreds of base pairs, occasionally even further away, from the transcription start site. Such regulatory sequences are, e.g., sequences that are involved in the binding of protein factors that control the effectiveness of transcription initiation in response to physiological conditions. The promoter region should be functional in the host cell and preferably corresponds to the natural promoter region of the Rpi-blb resistance gene. However, any heterologous promoter region can be used as long as it is functional in the host cell where expression is desired. The heterologous promoter can be either constitutive or regulatable, tissue specific or not specific. A constitutive promoter such as the CaMV 35S promoter or T-DNA promoters, all well known to those skilled in the art, is a promoter which is subjected to substantially no regulation such as induction or repression, but which allows for a steady and substantially unchanged transcription of the DNA sequence to which it is functionally bound in all active cells of the organism provided that other requirements for the transcription to take place is fulfilled. It is possible to use a tissue-specific promoter, which is driving expression in those parts of the plant which are prone to pathogen infection. In the case of Phytophthora a promoter which drives expression in the leaves, such as the ferredoxin promoter, can be used. A regulatable promoter is a promoter of which the function is regulated by one or more factors. These factors may either be such which by their presence ensure expression of the relevant DNA sequence or may, alternatively, be such which suppress the expression of the DNA sequence so that their absence causes the DNA sequence to be expressed. Thus, the promoter and optionally its associated regulatory sequence may be activated by the presence or absence of one or more factors to affect transcription of the DNA sequences of the genetic construct of the<br><br>
invehtion. Suitable promoter sequences and means for obtaining an increased transcription and expression are known to those skilled in the art. Terminator, the transcription terminator serves to terminate the transcription of the DNA into RNA and is preferably selected from the group consisting of plant transcription terminator sequences, bacterial transcription terminator sequences and plant virus terminator sequences known to those skilled in the art. Gene: the term "gene" is used to indicate a DNA sequence which is involved in producing a polypeptide chain and which includes regions preceding and following the coding region (5'-upstream and 3'-downstream sequences) as well as intervening sequences, the so-called introns, which are placed between individual coding segments (so-called exons) or in the 5'-upstream or 3'-downstream region. The 5'-upstream region may comprise a regulatory sequence that controls the expression of the gene, typically a promoter. The 3'-downstream region may comprise sequences which are involved in termination of transcription of the gene and optionally sequences responsible for polyadenylation of the transcript and the 3' untranslated region. The term "resistance gene" is an isolated nucleic acid according to the invention said nucleic acid encoding a gene product that is capable of providing a plant with resistance against a pathogen, more specifically said plant being a member of the Solanaceae family, more preferably potato or tomato, said pathogen more specifically being an oomycete pathogen, more specifically Phytophthora, more specifically Phytophthora infestans, said nucleic acid preferably comprising a sequence as depicted in Fig. 8 or part thereof, or a homologous sequence with essentially similar functional and structural characteristics. A functionally equivalent fragment of such a resistance gene or nucleic acid as provided by the invention encodes a fragment of a polypeptide having an amino acid sequence as depicted in Fig. 8 or part thereof, or a homologous and/or functionally equivalent polypeptide, said fragment exhibiting the characteristic of providing at least partial resistance to an oomycete infection such as caused by P. infestans when incorporated and expressed in a plant or plant cell.<br>
Resistance gene product: a polypeptide having an amino acid sequence as depicted in Fig. 8 or part thereof, or a homologous and/or functionally equivalent polypeptide exhibiting the characteristic of providing at least partial resistance to an oomycete infection such as caused by P. infestans when incorporated and expressed in a plant or plant cell.<br><br>
Functionally equivalents of the protein of the invention are proteins that are homologous to and are obtained from the protein depicted in fig. 8 by replacing, adding and/or deleting one or more amino acids, while still retaining their pathogen resistance activity. Such equivalents can readily be made by protein engineering in vivo, e.g. by changing the open reading frame capable of encoding the protein so that the amino acid sequence is thereby affected. As long as the changes in the amino acid sequences do not altogether abolish the activity of the protein such equivalents are embraced in the present invention. Further, it should be understood that equivalents should be derivable from the protein depicted in fig. 8 while retaining biological activity, i.e. all, or a great part of the intermediates between the equivalent protein and the protein depicted in Eg. 8 should have pathogen resistance activity. A great part would mean 30% or more of the intermediates, preferably 40% or more, more preferably 50% or more, more preferably 60% or more, more preferably 70% or more, more preferably 80% or more, more preferably 90% or more, more preferably 95% or more, more preferably 99% or more.<br>
Preferred equivalents are equivalents in which the leucine rich repeat region is highly homologous to the LRR region as depicted in fig. 8. Other preferred equivalents are equivalents wherein the N-terminal effector domain is essential the same as the effector domain oiRpi-blb.<br>
The protein of the invention comprises a distinct N-terminal effector domain and a leucine rich repeat domain. It is believed that conservation of these regions is essential for the function of the protein, although some variation is allowable. However, the other parts of the protein are less important for the function and may be more susceptible to change.<br>
In order to provide a quick and simple test if the modified proteins and/or the gene constructs capable of expressing said modified proteins which are described here or any new constructs which are obvious to the person skilled in the art after reading this application indeed can yield a resistance response the person skilled in the art can perform a rapid transient expression test known under the name of ATT A (Agrobacterium tumefaciens Transient expression Assay). In this assay (of which a detailed description can be found in Van den Ackerveken, G., et al., Cell 87, 1307-1316, 1996) the nucleotide sequence coding for the modified protein which is to be tested is placed under<br><br>
control of the CaMV 35S promoter and introduced into an Agrobacterium strain which is also used in protocols for stable transformation. After incubation of the bacteria with acetosyringon or any other phenolic compound which is known to enhance Agrobacterium T-DNA transfer, 1 ml of the Agrobacterium culture is infiltrated into an in situ plant leaf (from e.g. a tobacco or potato or tomato plant) by injection after which the plants are placed in a greenhouse and infected with a pathogen, preferably P. infestans. After 2-5 days the leaves can be scored for occurrence of resistance symptoms.<br>
In the present invention we have identified and isolated the resistance gene Rpi-blb, which confers race non-specific resistance to Phytophthora infestans. The gene was cloned from a Solanum bulbocastanum genotype that is resistant to P. infestans. The isolated resistance gene according to the invention can be transferred to a susceptible host plant using Agrobacterium mediated transformation or any other known transformation method, and is involved in conferring the host plant resistant to plant pathogens, especially P. infestans. The host plant can be potato, tomato or any other plant, in particular a member of the Solanaceae family that may be infected by such a plant pathogen. The present invention provides also a nucleic acid sequence coding for this protein or a functional equivalent thereof, preferably comprising the Rpi-blb gene, which is depicted in Figure 6.<br>
With the Rpi-blb resistance protein or functionally equivalent fragment thereof according to the invention, one has an effective means of control against plant pathogens, since the gene coding for the protein can be used for transforming susceptible plant genotypes thereby producing genetically transformed plants having a reduced susceptibility or being preferably resistant to a plant pathogen. In particular, a plant genetically transformed with the Rpi-blb resistance gene according to the invention has a reduced susceptibility to P. infestans.<br>
In a preferred embodiment the Rpi-blb resistance gene comprises the coding sequence provided in Figure 6A or any homologous sequence or part thereof preceded by a promoter region and/or followed by a terminator region. The promoter region should be functional in plant cells, and preferably correspond to the native promoter region of the Rpi-blb gene. However, a heterologous promoter region that is functional in plant cells can be used in conjunction with the coding sequences.<br><br>
In addition the invention relates to the Rpi-blb resistance protein which is encoded by the Rpi-blb gene according to the invention and which has an amino acid sequence provided in Figure 8, or a functional equivalent thereof.<br>
The signal that triggers the expression of the resistance gene in the wild-type S. bulbocas:-anum or in the transgenic plants of the invention is probably caused by the presence of a pathogen, more specifically the pathogen P. infestans. Such systems are known for other pathogen-plant interactions (Klement, Z., In: Phytopathogenic Prokaryotes, Vol. 2, eds.: Mount, M.S. and Lacy, G.H., New York, Academic Press, 1982, pp. 149-177), and use of this system can be made to increase the applicability of the resistance protein resulting in a resistance to more pathogens (see EP 474 857). This system makes use of the elicitor compound derived from the pathogen and the corresponding resistance gene, wherein the resistance gene when activated by the presence of the elicitor would lead to local cell death (hypersensitive reaction). In case of the present resistance gene, the corresponding elicitor component has not yet been disclosed, but it is believed that this is achievable by a person skilled in the art. Once the elicitor component is isolated it will be possible to transform the gene coding for said elicitor together with the gene coding for the resistance protein into plant, whereby one of the genes is under control of a pathogen-inducible promoter. These promoters are well known in the art (e.g. prpl, Fisl, Bet v 1, Vstl, gstAl, and sesquiterpene cyclase, but any pathogen-inducible promoter which is switched on after pathogen infection can be used). If the transgenic plant contains such a system, then pathogen attack which is able to trigger the pathogen-inducible promoter will cause production of the component which is under control of said promoter, and this, in connection with the other component being expressed constitutively, will cause the resistance reaction to occur.<br>
It will also be possible to mutate the resistance protein causing it to be in an active state (see EP 1060257). Since this would permanently result in the resistance reaction to occur, which ultimately leads to local cell death, care should be taken not to constitutively express the resistance protein. This can be accomplished by placing the mutated resistance protein under control of a pathogen-inducible promoter, which not only would allow for expression of the active resistance protein only at times of pathogen attack, but would also allow a broader pathogen range to induce the hypersensitive reaction. Mutation of threonine and serine residues to aspartic acid and glutamic acid residues frequently leads to activation, as was shown in many proteins of which the activity is modulated by phosphorylation, e.g. in a MAPK-<br><br>
activated protein (Engel et al., 1995, J. Biol. Chem. 270, 27213-27221), and in a MAP-kinase-kinase protein (Huang et al.,1995 Mol. Biol.Cell 6, 237-245). Also C- and N-terminal as well as internal deletion mutants of these proteins can be tested for suitable mutants.<br>
A more undirected way of identifying interesting mutants of which constitutive activity is induced is through propagation of the protein-encoding DNA in so-called E. coli 'mutator' strains.<br>
A rapid way of testing all made mutants for their suitability to elicit a hypersensitive response is through a so-called ATTA assay (Van den Ackerveken, G., et al., Cell 87, 1307-1316, 1996). Many mutants can be screened with low effort to<br>
identify those that will elicit an HR upon expression.<br>
t<br>
The invention also provides a vector comprising a nucleic acid as provided herein, said nucleic acid encoding a gene product that is capable of providing a member of the Solanaceae family with resistance against an oomycete pathogen, or a functionally equivalent isolated or recombinant nucleic acid in particular wherein said member comprises S. tuberosum or Lycopersicon esculentum.<br>
The invention also provides a host cell comprising a nucleic acid or a vector according to the invention. An example of said host cell is provided in the detailed description herein. In a particular embodiment, said host cell comprises a plant cell. As a plant cell a cell derived from a member of the Solanaceae family is preferred, in particular wherein said member comprises S. tuberosum or Lycopersicon esculentum. From such a cell, or protoplast, a transgenic plant, such as transgenic potato plant or tomato plantjwith resistance against an oomycete infection can arise. The invention thus also provides a plant, or tuber root, fruit or seed or part or progeny derived thereof comprising a cell according to the invention.<br>
Furthermore, the invention provides a proteinaceous substance, exhibiting the characteristic of providing at least partial resistance to an oomycete infection such as caused by P. infestans when incorporated and expressed in a plant or plant cell. In particular such a proteinaceous substance is provided that is encoded by a nucleic acid according to the invention. In a preferred embodiment, the invention provides a proteinaceous substance comprising an amino acid sequence as depicted in figure 8 or a functional equivalent thereof. Preferably, such a functional equivalent will comprise one or more sequences which are relatively unique to Rpl-blb in comparison to<br><br>
RGC3-blb, RGC-blb and RGC4-blb. Such sequences can be spotted in the alignment (see fig. 10A) and would be the sequences RPLLGEM, AKMEKEKLIS, KHSYTHMM, FFYTLFPLEKFI, GDSTFNK, NLYGSGMRS, LQYCTKLC, GSQSLTCM, NNFGPHI, TSLKIYGFRGIH, IIHECPFLTLS, RICYNKVA, and KYLTISRCN. It is believed that one or more of these sequences provide the functional characteristics of the protein Rpl-blb.<br>
Furthermore, the invention provides a binding molecule directed at a nucleic acid according to the invention. For example, the Rpi-blb gene can be used for the design of oligonucleotides complementary to one strand of the DNA sequence as depicted in Figure 7 and Table 2. Such oligonucleotides as provided herein are useful as probes for library screening, hybridisation probes for Southern/Northern analysis, primers for PCR, for use in a diagnostic kit for the detection of disease resistance and so on. Such oligonucleotides are useful fragments of an isolated or recombinant nucleic acid as provided herein, said nucleic acid encoding a gene product that is capable of providing a member of the Solanaceae family with resistance against an oomycete fungus, or a functionally equivalent isolated or recombinant nucleic acid, in particular wherein said member comprises S. tuberosum, or Lycopersicon esculentum. They can be easily selected from a sequence as depicted in figure 6 or part thereof. A particular point of recognition comprises the LRR domain as identified herein. Such a binding molecule according to the invention is used as a probe or primer, for example provided with a label, in particular wherein said label comprises an excitable moiety which makes it useful to detect the presence of said binding molecule.<br>
The invention furthermore provides a method for selecting a plant or plant material or progeny thereof for its susceptibility or resistance to an oomycete infection comprising testing at least part of said plant or plant material or progeny thereof for the presence or absence of a nucleic acid, said nucleic acid encoding a gene product that is capable of providing a member of the Solanaceae family with resistance against an oomycete fungus, or for the presence of said gene product, said method preferably comprising contacting at least part of said plant or plant material or progeny thereof with a binding molecule according the invention and determining the binding of said molecule to said part. Said method is particularly useful wherein said oomycete comprises P. infestans, allowing to select plants or planting material for resistance against late blight, for example wherein said plant or material comprises S. tuberosum. It is believed that by the phylogenetic tree analysis as discussed above, proteins that are highly homologous to Rpi-blb and which would<br>
14--<br><br>
yield resistance against plant pathogens could be easily idientified. An example for<br>
i<br>
this is the detection of the three highly homologous proteins RGCl-blb, RGC3-blb and, RGC4-blb, which have not yet been shown to yield resistance to P. infestans, but j' which are nevertheless believed to be involved in pathogen resistance in plants.<br>
Also, the invention provides use of a nucleic acid or a vector or a cell or a substance or a binding molecule according to the invention in a method for providing a plant or its progeny with at least partial resistance against an oomycete infection, in particular wherein said oomycete comprises P. infestans especially wherein said plant comprises S. tuberosum, said method for providing a plant or its progeny with at least partial resistance against an oomycete infection comprising providing said plant or part thereof with a gene coding for a resistance protein or functional fragment thereof comprising a nucleic acid, said resistance protein being capable of providing a member of the Solanaceae family with resistance against an oomycete fungus, or providing said plant or part thereof with a nucleic acid or a vector or a cell or a substance according to the invention.<br>
Furthermore, the invention provides an isolated 5. bulbocastanurn, or part thereof, such as a tuber or seed, susceptible to an oomycete infection caused by P. infestans.<br>
The invention is further described in the detailed description below.<br>
H&gt;<br><br>
DESCRIPTION OF THE FIGURES<br>
Figure 1. Geographical map of Mexico indicating the origin of Solanum bulbocastanum accessions used to isolate the Rpi-blb gene. The letters a, b and c indicate the relative geographical origins of the used S. bulbocastanum accessions.<br>
Figure 2. Genetic linkage maps of the Rpi-blb locus on chromosome 8 of S. bulbocastanum. Horizontal lines indicate the relative positions of markers linked to late blight resistance. Distances between markers are indicated in centirhorgans. A. Genetic position of the Rpi-blb locus relative to markers TG513, CT88 and CT64 (n=508 genotypes). B. High density genetic linkage map of the Rpi-blb locus (n=2109 genotypes).<br>
Figure 3. Physical map of the Rpi-blb locus. A. Genetic and physical map of the S. bulbocastanum genomic region containing Rpi-blb. Vertical arrows indicate the relative positions of markers linked to resistance. Numbers above the horizontal line indicate the number of recombinants identified between the flanking markers in 2109 progeny plants. Rectangles represent bacterial artificial chromosome (BAC) clones. B. Relative positions of candidate genes for late blight resistance on BAC SPB4. C. Schematic representation of the Rpi-blb gene structure. Horizontal lines indicate exons. Open boxes represent coding sequence. Lines angled downwards indicate the position of a 678-nucleotide long intron sequence.<br>
Figure 4. Southern blot analysis of the BAC contig spanning the Rpi-blb locus. Names above each lane represent the names of BAC clones. The names of the restriction enzymes used to digest the BAC DNA prior to Southern blotting are indicated.<br>
Figure 5. Detached leaf disease assays. A. Resistant (left), intermediate (centre) and susceptible (right) phenotypes found in the 5. bulbocastanum mapping population B8 6 days post inoculation (d.p.i) with P. infestans sporangiospore droplets. B. Genetic complementation for late blight resistance in potato. Characteristic disease phenotypes of leaves derived from transgenic potato plants harbouring RGCl-blb, RGC2-blb, -bib or RGC4-blb 6 d.p.i. with P. infestans sporangiospore droplets. Genetic constructs harbouring the RGCs were transferred to the susceptible potato cultivar Impala through Agrobacterium mediated transformation. C. Genetic<br><br>
complementation for late blight resistance in tomato. Characteristic disease phenotype of a tomato leaf derived from transgenic tomat plants harbouring Rpi-blb 6 d.p.i. with P. infestans sporangiospore droplets (left panel). The genetic construct harbouring Rpi-blb was transferred to the susceptible tomato cultivar Moneymaker through Agrobccterium mediated transformation.<br>
Figure 6. Nucleic acid sequences of the Rpi-blb gene cluster members. A. Coding nucleic acid sequence of the Rpi-blb gene. B. Coding nucleic acid sequence of the Rpi-blb gene including the intron sequence (position 428-1106). C. Sequence of the 5.2 kb Seal genomic DNA fragment of S. bulbocastanum BAC SPB4 present in pRGC2-blb, the genetic construct used for genetic complementation for late blight resistance. The genomic fragment harbours the Rpi-blb gene including natural regulatory elements necessary for correct expression of the gene. The initiation codon (ATG position 1191-1193) and the termination codon (TAA position4781-4783) are underlined. D. Coding nucleic acid sequence ofRGCl-blb including the intron sequence (position 428-708). E. Coding nucleic acid sequence of RGC3-blb including the intron sequence (position 428-1458). F. Coding nucleic acid sequence oiRGC4-blb including intron sequences (positions 434-510, 543-618 and 743-1365).<br>
Figure 7. Relative primer positions. The horizontal bar represents the coding sequence of the Rpi-blb gene. Numbers represent nucleotide positions. Horizontal arrows indicate relative primer positions and orientations. GSPl and GSP2 represent nested gene specific primers used for 3' RACE experiments. GSP3 and GSP4 represent nested gene specific primers used for 5' RACE experiments. A(F), A(R), B(F) and B(R) are primers used to amplify Rpi-blb homologues. The position of the restriction site Nsil used to make domain swaps between Rpi-blb homologues is indicated.<br>
Figure 8. Deduced Rpi-blb protein sequence. The amino acid sequence deduced from the DNA sequence of Rpi-blb is divided into three domains (A-C), as described in Example 6. Hydrophobic residues in domain A that form the first and fourth residues of heptad repeats of potential coiled-coil domains are underlined. Conserved motifs in R proteins are written in lowercase and in italic in domain B. Residues matching the consensus of the cytoplasmic LRR are indicated in bold in domain C. Dots in the<br>
m<br><br>
sequence have been introduced to align the sequence to the consensus LRR sequence of cytoplasmic LRRs.<br>
Figure 9. Phylogenetic tree analysis. A. Phylogenetic tree of state of the art sequences which share some degree of homology to the deduced amino acid sequence ofRpi-blb and its gene cluster members RGCl-blb, RGC3-blb and RGC4-blb. The tree was made according to the Neighbour-Joining method of Saitou and Nei (1987 Molecular Biology and Evolution 4, 406-425). An asterix indicates that the gene has been assigned a function. The Rpi-blb gene cluster is boxed. B. Phylogenetic tree of state of the arc sequences which share some degree of homology to the deduced amino acid sequence of Rpi-blb. Included in this analysis are the Rpi-blb homologous sequences Bl49-blb, SHlO-tub, SH20-tub and Tll8-tar, sequences identified through PCR amplification using Rpi-blb gene cluster specific primers. C. Relative positions of state of the art DNA sequences which show significant homology to parts of the Rpi-blb gene sequence. Horizontal lines represent the relative positions of the homologous sequences. The degree of homology is indicated to the right of each line. The length of the homologous sequence is indicated above each line.<br>
Figure 10. Alignment of the predicted Rpi-blb gene product to the predicted protein sequences of Rpi-blb homologues A. Alignment of the deduced protein products encoded by Rpi-blb, RGCl-blb, RGC3-blb and RGC4-blb. The complete amino acid sequence ofRpi-blb is shown and amino acid residues from RGCl-blb, RGC3-blb and RGC4-blb that differ from the corresponding residue in Rpi-blb. Dashes indicate gaps inserted to maintain optimal alignment. Amino acid residues that are specific for Rpi-blb, when compared to those at corresponding positions in RGCl-blb, RGC3-blb and RGC4-blb, are highlighted in bold. The regions of the LRRs that correspond to the consensus L..L..L.L..C/N/S..a..aP are underlined. Conserved motifs in the NBS domain are indicated in lowercase. B. Alignment of the deduced protein products encoded by Rpi-blb, RGCl-blb, RGC3-blb, RGC4-blb, Bl49-blb, SHlO-tub, SH20-tub and T118-tar..<br>
Figure 11. Schematic overview of domain swaps made between Rpi-blb and homologues RGCl-blb and RGC3-blb. The vertical dotted line indicates the position of the Nsil site used to make the swaps. R and S indicate whether transgenic plants harbouring specific chimeric constructs are resistant or susceptible co late blight infection, respectively.<br><br>
Experimental part<br>
For the mapping of the Rpi-blb resistance gene an intraspecific mapping population of S. bulbocastanum was developed. A crucial step in this process was the identification of susceptible 5. bulbocastanum genotypes. For this purpose several S. bulbocastanum accessions originating from different clusters/areas in Mexico were analysed for P. infestans resistance or susceptibility in a detached leaf assay (Table 1 and Figure 1). The screened accessions BGRC 8008 and BGRC 7999 contained no susceptible genotypes. However in the accessions BGRC 8005, BGRC 8006 and BGRC 7997, susceptibility was found in 9%, 7% and 14 % of the analysed seedlings, respectively. A P. infestans susceptible clone of accession BGRC 8006 was subsequently selected and crossed with a resistant clone of accession BGRC 8005. The resulting Fl population was used to map the Rpi-blb locus and is hereafter referred to as the B8 population.<br>
Initial screening of 42 B8 genotypes for resistance to P. infestans in a detached leaf assay suggested that P. infestans resistance in S. bulbocastanum. accession 8005 could be caused by a single dominant R gene, or a tightly linked gene cluster. Of the 42 genotypes tested, 22 scored resistant and 16 susceptible in a repeated experiment. Resistance phenotypes of the remaining 4 seedlings remained unclear. In order to determine the chromosome position of this S. bulbocastanum resistance, B8 genotypes with an undoubted phenotype were used for marker analysis. The chromosome 8 specific marker TG330 (Table 2) was found to be linked in repulsion phase with the resistant phenotype, as only one recombinant was obtained between this marker and resistance in 12 B8 genotypes. Furthermore, chromosome 8 marker CT88 (Table 2) was found to be completely linked in repulsion phase to resistance, indicating that the locus responsible for resistance, designated Rpi-blb, was located in this region of chromosome 8. For this reason, tomato chromosome 8 specific markers that map proximal and distal to CT88 (TG513 and CT64; Tanksley et al., 1992 Genetics 132: 1141-1160; Table 2) were developed into CAPS markers and tested in 512 B8 genotypes with known resistance phenotypes. A total of five CT64-CT88 recombinant genotypes and 41 CT88-TG513 recombinant genotypes were identified in this screen (Figure 2A). The resistance locus Rpi-blb was mapped 1 recombination event distal to marker CT88 (Figure 2A).<br>
oh<br><br>
Fine mapping of the Rpi-blb locus was carried out with CAPS markers derived from left (L) and right (R) border sequences of BAC clones isolated from a BAC library prepared from the resistant S. bulbocastanum genotype BGRC 8005-8. The BAC library was initially screened with markers CT88 and CT64. BAC clones identified with these markers were used as seed BACs for a subsequent chromosome walk to the Rpi-blb locus. A total of 2109 B8 genotypes were screened for recombination between markers TG513 en CT64. All recombinant genotypes (219/2109) were subsequently screened with all available markers in the CT88-CT64 genetic interval. These data together with the disease resistance data of each recombinant, obtained through detached leaf assays, positioned the Rpi-blb locus between markers SPB33L and B149R, a 0.1 cM genetic interval (4/2109 recombinants) physically spanned by the overlapping BAC clones SPB4 and B49 (Figures 2b and 3). Within this interval resistance cosegregated with the BAC end marker SPB42L, the sequence of which was highly homologous to partial NBS fragments from tomato (e.g. Q194, Q137, Q152, Ql53;Pan et al., 2000 Genetics 155: 309-322). Southern analyses of BAC clones spanning the SP33L-B149R interval using a 32P-labeled PCR fragment of marker SPB42L as a probe revealed the presence of at least 4 copies of this R gene like sequence within the Rpi-blb interval (Figure 4). Moreover, all of these copies were present on BAC SPB4. Sequencing and annotation of the complete insert of this BAC clone indeed identified four complete R gene candidates (RGCl-blb, RGC2-blb, RGC3-blb and RGC4-blb) of the NBS-LRR class of plant R genes. A PCR-marker that was located in-between RGCl-blb and RGC4-blb revealed recombination between P. infestans resistance and RGC4-blb, ruling out the possibility of RGC4-blb being Rpi-blb. Despite this finding, all four RGCs were selected for complementation analysis.<br>
Genomic fragments of approximately 10 kb harbouring RGCl-blb, RGC2-blb, RGC3-blb or RGC4-blb were subcloned from BAC SPB4 into the binary plant transformation vector pBINPLUS (van Engelen et al., 1995 Trans. Res. 4, 288-290) and transferred to a susceptible potato cultivar using standard transformation methods. Primary transformants were tested for P. infestans resistance as described in Example 1. Only the genetic construct harbouring RGC2-blb was able to complement the susceptible phenotype; 86% of the primary transformants harbouring RGC2-blb were resistant (Table 3) whereas all RGCl-blb, RGC3-blb and RGC4-blb containing primary transformants were completely susceptible to P. infestans. The resistant RGC2-blb containing transformants showed similar resistance phenotypes<br>
Jl<br><br>
as the S. bulbocastanum resistant parent (Figure 5). RGC2-blb was therefore designated the Rpi-blb gene, the DNA sequence of which is provided in Figure 6.<br>
EXAMPLE l: DISEASE ASSAY<br>
t     The phenorype of S. bulbocastanum and transgenic S. tuberosum genotypes for<br>
resistance to P. infestans was determined by detached leaf assays. Leaves from plants grown for 6 to 12 weeks in the greenhouse were placed in pieces of water-saturated florists foam, approximately 35x4x4 cm, and put in a tray (40 cm width, 60 cm length and 6 cm height) with a perforated bottom. Each leaf was inoculated with two droplets or more (25 ul each) of sporangiospore solution on the abaxial side. Subsequently, the tray was placed in a plastic bag on top of a tray, in which a water-saturated filter paper was placed, and incubated in a climate room at 17°C and a 16h/8h day/night photoperiod with fluorescent light (Philips TLD50W784HF). After 6 days, the leaves were evaluated for the development of P. infestans disease symptoms. Plants with leaves that clearly showed sporulating lesions 6 days after inoculation were considered to have a susceptible phenotype whereas plants with leaves showing no visible symptoms or necrosis at the side of inoculation in the absence of clear sporulation were considered to be resistant. The assay was performed with P. infestans complex isolate 655-2A (race 1, 2, 3,4, 5, 6, 7, 8, 9, 10, 11), which was obtained from Plant Research International BV (Wageningen, The Netherlands).<br>
EXAMPLE 2: MAPPING OF THE Rpi-blb RESISTANCE LOCUS<br>
Plant material<br>
In order to produce an intraspecific mapping population that segregated for the P. infestans resistance gene present in S. bulbocastanum accession BGRC 8005 (CGN 17692, PI 275193), a susceptible 5. bulbocastanum genotype was required. Several S. bulbocastanum accessions originating from different clusters/areas in Mexico were analysed for P. infestans resistance or susceptibility in a.detached leaf assay (Table 1 and Figure 1). In accession BGRC 8008 and BGRC 7999 no susceptibility was detected. In accession BGRC 8005, BGRC 8006 and BGRC 7997 susceptibility was only present in 9%, 7% and 14 % of the analysed seedlings, respectively. Thus, only a few susceptible S. bulbocastanum genotypes were obtained.<br>
Jt<br><br>
The intraspecific mapping population of S. bulbocastanum (B8) was produced by crossing a P. infestans susceptible clone of accession BGRC 8006 with a resistant clone of accession BGRC 8005. DNA of 2109 progeny plants was extracted from young leaves according to Doyle and Doyle (1989 Focus 12, 13-15).<br>
CAPS marker analysis<br>
For PCR analysis, 15 ul reaction mixtures were prepared containing 0.5 ug DNA, 15 ng of each primer, 0.2 mM of each dNTP, 0.6 units Taq-polymerase (15 U/jil, SphaeroQ, Leiden, The Netherlands), 10 mM Tris-HCl pH 9, 1.5 mM MgCl2, 50 mM KC1, 0.1% Triton X-100 and 0.01% (w/v) gelatin. The PCRs were performed using the following cycle profile: 25 seconds DNA denaturation at 94°C, 30 seconds annealing (see Table 1) and 40 seconds elongation at 72°C. As a first step in PCR-amplification DNA was denatured for 5 min at 94°C and finalised by an extra 5 min elongation step at 72°C. The amplification reactions were performed in a Biometra® T-Gradient or Biometra* Uno-II thermocycler (Westburg, Leusden, The Netherlands). Depending on the marker, the PCR product was digested with an appropriate restriction enzyme. An overview of the markers including primer sequences, annealing temperature and restriction enzymes, is given in Table 2. Subsequently, the (digested) PCR products were analysed by electrophoresis in agarose or acrylamide gels. For acrylamide gel analysis, the CleanGel DNA Analysis Kit and DNA Silver Staining Kit (Amersham Pharmacia Biotech Benelux, Roosendaal, the Netherlands) were used.<br>
Genetic mapping of the Rpi-blb locus<br>
Initially a small group of 42 progeny plants of the B8 population was screened for resistance to P. infestans in a detached leaf assay. Plants with leaves that clearly showed sporulating lesions 6 days after inoculation were considered to have a susceptible phenotype whereas plants with leaves showing no visible symptoms or necrosis at the side of inoculation in the absence of clear sporulation were considered to be resistant. Of the 42 seedlings, 22 scored resistant and 16 susceptible. The phenotype of the remaining 4 seedlings remained unclear in this initial phase. These data indicated that resistance could be due to a single dominant gene or a tightly linked gene cluster. In order to determine the chromosome position, seedlings with a reliable phenotype were used for marker analysis. Chromosome 8 marker TG330 was found to be linked in repulsion with the resistant phenotype, as only one recombinant was obtained between this marker and resistance in 12 B8 seedlings. Furthermore,<br>
J3<br><br>
.chromosome 8 marker CT88 was found to be completely linked in repulsion phase to resistance, indicating that a resistance gene was located on chromosome 8.<br>
Subsequently, chromosome 8 specific markers that had been mapped proximal and distal to CT88 (Tanksley et al, 1992 Genetics 132: 1141-1160) were developed to CAPS markers. In order to map these markers more precisely, another 512 individuals of the B8 population were screened for late blight resistance using the detached leaf disease assay. Simultaneously, plants were scored for the markers CT64, CT8S and TG513. For 5 seedlings, recombination was detected between markers CT64 and CT88, while 41 seedlings were recombinant between markers CT88 and TG513 (Figure 2A). The resistance gene Rpi-blb was mapped in between markers CT64 and CT88. In this stage, the positioning of CT88 proximal to Rpi-blb was based on only one recombined seedling.<br>
In order to determine the position of Rpi-blb more precisely relative to the available markers, another 1555 seedlings of the B8 population were grown and analysed for recombination between the markers TG513 and CT64. Thus, a total of 2109 individual offspring clones of the B8 population were screened. Recombination between markers TG513 en CT64 was detected in 219 of these seedlings (10.4 cM). All of the recombinants were screened with marker CT88 and phenotyped for the resistance trait by making use of the detached leaf assay. In agreement with earlier results, the Rpi-blb gene was mapped in between markers CT88 and CT64 (Figure 2B).<br>
EXAMPLE 3: CONSTRUCTION OF A S. BULBOCASTANUMBAC LIBRARY AND CONSTRUCTION OF A CONTIGUOUS BAC CONTIG SPANNING THE Rpi-blb LOCUS<br>
BAC library construction<br>
A resistant clone of S. bulbocastanum (bib) accession BGRC 8005 (CGN 17692, PI 275193) heterozygous for the Rpi-blb locus, was used as source DNA for the construction of a genomic BAC library, hereafter referred to as the 8005-8 BAC library. High molecular weight DNA preparation and BAC library construction were carried out as described in Rouppe van der Voort et al. (1999 MPMI 12:197-206). Approximately 130.000 clones with an average insert size of 100 kb, which corresponds to 15 genome equivalents were finally obtained. A total of approximately<br><br>
83.000 individual clones were stored in 216 384-well microtiter plates (Invitrogen, The Netherlands) containing LB freezing buffer (36 mM K2HPO4, 13.2 mM KH2PO4, 1.7 mM citrate. 0.4 mM MgSCi, 6.8 mM (NfiUhSCU, 4.4 % V/V glycerol, 12.5 ug/ml chloramphenicol in LB medium) at -80°C. Another 50.000 clones were stored as bacterial pools containing -1000 white colonies. These were generated by scraping the colonies from the agar plates into LB medium containing 18% glycerol and 12.5 ug/ml chloramphenicol using a sterile glass spreader. These so-called super pools were also stored at -80°C.<br>
Screening of the BAC library and construction of a physical map of the Rpi-blb locus<br>
The 8005-8 BAC library was initially screened with CAPS markers CT88 and CT64. This was carried out as follows. For the first part of the library of approximately 83.000 clones stored in 384 well microtiter plates, plasmid DNA was isolated using the standard alkaline lysis protocol (Sambrook et al., 1989 in Molecular cloning: a laboratory manual 2nd edn, Cold Spring Harbor Press, Cold Spring Harbor, New York) from pooled bacteria of each plate to produce 216 plate pools. To identify individual BAC clones carrying the CAPS markers the plate pools were screened by PCR. Once an individual plate pool was identified as being positive for a particular CAPS marker the positive row and positive column were identified through a two dimensional PCR screening. For this purpose, the mother 384-well plate was replicated twice on LB medium containing chloramphenicol (12.5 ug/ml). After growing the colonies for 16 h at 37°C one plate was used to scrape the 24 colonies of each row together and the other plate was used to scrape the 16 colonies of each column together. Bacteria of each row or column were resuspended in 200 pi TE buffer. CAPS marker analysis on 5 pi of these bacterial suspensions was subsequently carried out leading to the identification of single positive BAC clones. For the second part of the library, stored as 50 pools of approximately 1000 clones, plasmid DNA was isolated from each pool of clones using the standard alkaline lysis protocol and PCR was carried out to identify positive pools. Bacteria corresponding to positive pools were diluted and plated on LB agar plates containing chloramphenicol (12.5 ug/ml). Individual white colonies were subsequently picked into 384-well microtiter plates and single positive BAC clones subsequently identified as described above. Names of BAC clones isolated from the super pools carry the prefix SP (e.g. SPB33).<br>
1^<br><br>
Insert sizes of BAC clones were estimated as follows. Positive BAC clones were analysed by isolating plasmid DNA from 2 ml overnight cultures (LB medium supplemented with 12.5 mg/ml chloramphenicol) using the standard alkaline lysis miniprep protocol and resuspended in 20 ul TE. Plasmid DNA (10 ul) was digested with 5 U Notl for 3 h at 37°C to free the genomic DNA from the pBeloBACll vector. The digested DNA was separated by CHEF electrophoresis in a 1% agarose gel in 0.5 X TBE at 4°C using a BIORAD CHEF DR II system (Bio-Rad Laboratories, USA) at 150 volts with a constant pulse time of 14 sec for 16 h.<br>
Screening of the 8005-8 BAC library with marker CT88 identified two positive BAC clones: B139 and B180, with potato DNA inserts of 130 and 120 kb, respectively (Figure 3A). Digestion of the CT88 PCR product generated from these BAC clones and several resistant and susceptible progeny plants of the B8 mapping population with Mbol revealed that BAC 139 carried the CT88 allele that was linked in cis to resistance. To identify the relative genome position of BAC B139, pairs of PCR primers were designed based on the sequence of the right (R) and left (L) ends of the insert. BAC end sequencing was carried out as described in Example 4 using 0.5 ng of BAC DNA as template. Polymorphic CAPS markers were developed by digesting the PCR products of the two parent genotypes of the B8 population and of two resistant and two susceptible progeny genotypes with several 4-base cutting restriction enzymes (Table 2). Screening of the 37 CT88-CT64 recombinant B8 genotypes mapped 5 of the 7 CT88-Rpi-blb recombinants between CT88 and B139R, indicating that marker B139R was relatively closer to the Rpi-blb locus than marker CT88. Screening of the 216 plate pools with B139R did not lead to the identification of a positive BAC clone. Screening of the 50 super pools identified the positive BAC clones SPB33 and SPB42 with DNA inserts of 85 and 75 kb, respectively (Figure 3A). Screening of the complete BAC library with SPB33L identified the positive BAC clones B149 and SPB4. BAC clone SPB4 contained the SPB33L allele that was linked in cis to resistance whereas BAC clone B149 did not. However, screening of the CT88-CT64 recombinant panel with B149R revealed that this BAC spanned the Rpi-blb locus. B149R was separated from the Rpi-blb locus by two recombination events (Figure 3A). Screening of the 8005-8 BAC library with B149R identified BAC clone B49 as having the B149R allele that was linked in cis to resistance. This BAC clone together with BAC clone SPB4 therefore formed a BAC contig that spanned the Rpi-blb locus (Figure 3).<br>
ri<br><br>
EXAMPLE 4: SEQUENCE ANALYSIS OF BAC SPB4 AND IDENTIFICATION OF RESISTANCE GENE CANDIDATES WITHIN THE Rpi-blb LOCUS<br>
Within the SPB33L-B149R interval resistance cosegregated with BAC end marker SPB42L, the sequence of which was highly homologous to partial NBS fragments from tomato (e.g. Q194, Q137, Q97, Q152, Q153; Pan et al., 2000 Genetics 155:309-22). Southern analyses of BAC clones spanning the SPB33L-B149R interval using a 32P-labeled PCR fragment of marker SPB42L as a probe revealed the presence of at least 4 copies of this R gene like sequence within the Rpi-blb interval (Figure 4). Moreover, all of these copies were present on BAC SPB4. The DNA sequence of BAC clone SPB4 was therefore determined by shotgun sequence analysis. A set of random subclones with an average insert size of 1.5 kb was generated. 10 ug of CsCl purified DNA was sheared for 6 seconds on ice at 6 amplitude microns in 200 ul TE using an MSE soniprep 150 sonicator. After ethanol precipitation and resuspension in 20 ul TE the ends of the DNA fragments were repaired by T4 DNA polymerase incubation at 11°C for 25 minutes in a 50 pi reaction mixture comprising lx T4 DNA polymerase buffer (New England BioLabs, USA), 1 mM DTT, 100 uM of all 4 dNTP's and 25 U T4 DNA polymerase (New England Biolabs, USA), followed by incubation at 65°C for 15 minutes. The sheared DNA was subsequently separated by electrophoresis on 1% SeaPlaque LMP agarose gel (FMC). The fraction with a size of 1.5-2.5 kb was excised from the gel and dialysed against 50 ml TE for 2 hr at 4°C. Dialysed agarose slices were then transferred to a 1.5 ml Eppendorf tube, melted at 70°C for 5 min, digested with 1 unit of GELASE (Epicentre Technologies, USA) per 100 mgof agarose gel for 1 hr at 45°C, and the DNA was subsequently precipitated. The 1.5-2.5 kb fragments were ligated at 16°C in a EcdRV restricted and dephosphorylated pBluescript SK+ vector (Stratagene Inc.). The ligation mixture was subsequently used to transform ElectroMAX E. coli DH10B competent cells (Life Technologies, UK) by electroporation using the BioRad Gene Pulser. Settings on the BioRad Gene Pulser were as recommended for E. coli by the manufacturer. The cells were spread on Luria broth (LB) agar plates containing ampicillin (100 ug/ml), 5-bromo-4-chloro-3-indolyl-B-D-galactoside (Xgal) (64 pg/ml) and isopropyl-1-thio-fl-D-galactoside (IPTG) (32 ug/ml). Plates were incubated at 37°C for 24 hours. Individual white colonies were grown in 96-well flat-bottom blocks (1.5 ml Terrific Broth medium containing 100 ug/ml ampicillin).<br>
-&gt;7<br><br>
Pla smid DNA was isolated using the QIAprep 96 Turbo Miniprep system in conjunction with the BioRobot™ 9600 (QIAGEN) according to the manufacturers instructions. Sequencing reactions were performed using ABI PRISM BigDye™ Terminator cycle sequencing kit (Stratagene) according to the manufacturer's instructions. All clones were sequenced bi-directionally using universal primers. Sequence products were separated by capillary electrophoresis on a Perkin Elmer ABI 3700 DNA Analyzer.<br>
The automated assembly of the shotgun reads was carried out using the Phred-Phrap programs (Ewing and Green, 1998 Genome Research 8, 186-194; Ewing et al., 1998 Genome Research 8, 175-185). A total of 835 reads provided an overall BAC sequence coverage equal to 5x. Gaps between contigs were closed by primer walking or through a combinatorial PCR approach. The sequence was finaUy edited at Phred quality 40 (1 error every 10,000 nt) by manual inspection of the assembly using the Gap4 contig editor and re-sequencing of all low-quality regions. The complete sequence of the insert of BAC SPB4 consisted of 77,283 nucleotides.<br>
Analysis of the contiguous sequence of BAC SPB4 using the computer programme GENSCAN (Burge and Karlin, 1997 J. Mol. Biol: 268, 78-94), GENEMARK (Lukashin and Borodovsky, 1998 NAR 26, 1107-1115) and BLASTX (Altschul et al., 1990 J. Mol. Biol. 215, 403-410) identified four complete R gene candidate sequences (RGCl-blb, RGC2-blb, RGC3-blb and RGC4-blb) belonging to the NBS-LRR class of plant R genes. A CAPS marker designed in between RGCl-blb and RGC4-blb, marker RGCl-4 revealed recombination between P. infestans resistance and RGC4-blb, ruling out the possibility of RGC4-blb being Rpi-blb (Figure 3A and B). Despite this finding, all four RGCs were selected for complementation analysis.<br>
EXAMPLE 5: COMPLEMENTATION ANALYSIS<br>
Subcloning of candidate genes and transformation to Agrobacterium tumefaciens<br>
Genomic fragments of approximately 10 kb harbouring RGCl-blb, RGC2-blb, RGC3-blb or RGC4-blb were subcloned from BAC clone SPB4 into the binary plant transformation vector pBINPLUS (van Engelen et al., 1995 Trans. Res. 4, 288-290). Restriction enzyme digestion of BAC clone SPB4 DNA and subsequent size selection was carried out as follows. Aliquots of ~1 ug DNA were digested with 1U, 0.1U or<br>
o»<br><br>
0,Q1U of Sau3Al restriction enzym for 30 min. The partially digested BAC DNA was subjected to CHEF electrophoresis at 4°C in 0.5 X TBE using a linear increasing pulse time of 1-10 sec and a field strength of 6 V/cm for 16 hr. After electrophoresis, the agarose gel was stained with ethidium bromide to locate the region of the gel containing DNA fragments of approximately lOkb in size. This region was excised from the gel using a glass coverslip and dialysed against 50 ml TE for 2 hr at 4°C. Dialysed agarose slices were then transferred to a 1.5 ml Eppendorf tube, melted at 70°C for 5 nun and digested with 1 unit of GELASE (Epicentre Technologies, USA) per 100 mgof agarose gel for 1 hr at 45°C. Ligation of the size selected DNA to BamHI-digested and dephosphorylated pBINPLUS and subsequent transformation of ElectroMAX E. coli DH10B competent cells (Life Technologies, UK) with the ligated DNA was carried as described in Example 5, using the BioRad Gene Pulser for electroporation. The cells were spread on Luria broth (LB) agar plates containing kanamycin (50 pg/ml), Xgal (64 pg/ml) and IPTG (32 ug/ml). Plates were incubated at 37°C for 24 hours. Individual white colonies were grown in 96-well plates (100 ul LB medium containing 50 pg/ml kanamycin). A total of 480 clones were PCR screened for the presence of RGCs using primers SPB42LF and SPB42LR or RGC4F and RGC4R (Table 2.). Positive clones were selected for plasmid isolation and further characterisation. Identification of clones harbouring RGCl-blb, RGC2-blb, RGC3-blb or RGC4-blb was carried out by sequencing the SPB42L PCR fragments derived from positive clones. The relative position of the RGCs within a subclone was determined by sequencing the ends of the clone and subsequent comparison of the sequences to the complete BAC insert sequence. Finally four binary plasmids, pRGCl-blb, pRGC2-blb, pRGC3-blb and pRGC4-blb were selected and transferred to Agrobacterium tumefaciens strains AGL0 (Lazo et al., 1991 Bio/Technology 9, 963-967), LBA4404 (Hoekema et al., 1983 Nature 303: 179-180) or UIA143 (Farrand et aL,1989 J. of Bacteriology 171, 5314-5321) either by electroporation using the BioRad Gene Pulser or by conjugation. Settings on the BioRad Gene Pulser were as recommended for A. tumefaciens by the manufacturer. Conjugation was carried out as described by Simon et al. (1983 Bio/Tech. 1, 784-791). The cells were spread on Luria broth (LB) agar plates containing kanamycin (100 mg/1) and rifampicin (50 mg/1). Plates were incubated at 28°C for 48 hours. Small-scale cultures from selected colonies were grown in LB medium containing kanamycin (100 mg/1) and rifampicin (50 mg/1). Plasmid DNA was isolated as described previously and the integrity of the plasmids was verified by restriction analysis upon reisolation from A. tumefaciens and<br><br>
subsequent transformation to E. coli. A tutnefaciens cultures harbouring a plasmid with the correct DNA pattern were used to transform a susceptible potato genotype.<br>
Transformation of susceptible potato cultivar<br>
A. tumefaciens strains were grown for 2 days at 28°C in 20 ml LB medium supplemented with 50 mg/1 rifampicine and 25 mg/1 kanamycin. Subsequently, 0.2 ml of A. tumefaciens culture was diluted in 10 ml LB medium containing the same antibiotics and grown overnight (28°C). The overnight culture was centrifuged (30 min, 2647 x g) and the pellet was resuspended in 50 ml MS medium (Murashige and Skoog, 1962 Physiol. Plant. 15, 473-497) supplemented with 30 g/1 sucrose (MS30). Certified seed potatoes of cultivar Impala were peeled and surface sterilised for 30 min. in a 1% sodium hypochlorate solution containing 0.1 % Tween-20. Tubers were then washed thoroughly in large volumes of sterile distilled water (4 times, 10 min). Discs of approximately 2 mm thicjcness and 7 mm in diameter, were sliced from cylinders of tuber tissue prepared with a corkborer. The tuber discs were transferred into liquid MS30 medium containing A. tumefaciens and incubated for 15 min. After removing the .4. tumefaciens solution, the tuber discs were transferred to regeneration medium containing MS30, 0.9 mg/1 IAA, 3.6 mg/1 zeatine riboside and 8 g/1 agar (Hoekema et al., 1989 Bio/Technology 7, 273-278). The plates were incubated at 24°C, 16 hour day-length (Philips TLD50W/84HF). After 48 hours of co-cultivation, the tuber discs were rinsed for 5 min in liquid MS medium including antibiotics, 200 mg/1 vancomycin, 250 mg/1 cefotaxim and 75 mg/1 kanamycin, and transferred to regeneration medium supplemented with the same antibiotics. The plates were incubated at 24°C, 16 hour day-length (Philips TLD50W/84HF). Every three weeks, the tuber discs were transferred to fresh medium. Regenerating shoots were transferred to MS30 medium containing 75 mg/1 kanamycin. Rooting shoots were propagated in vitro and tested for absence of A. tumefaciens cells by incubating a piece of stem in 3 ml LB medium (3 weeks, 37°C, 400 rpm). One plant of each transformed regenerant was transferred to the greenhouse.<br>
Complementation of the susceptible phenotype in potato<br>
Primary transformants were tested for P. infestans resistance as described in Example^l. Only the genetic construct harbouring RGC2-blb was able to complement the susceptible phenotype; 15 out of 18 RGC2-blb containing primary transformants were resistant (Table 3) whereas all RGCl-blb, RGC3-blb and RGC4-blb containing primary transformants were completely susceptible to P. infestans. The resistant<br>
on<br><br>
RGC2-blb transformants showed similar resistance phenotypes as the S. bulbocastanum resistant parent (Figure 5). RGC2-blb was therefore designated the RpL-blb gene, the DNA sequence of which is provided in Figure 6.<br>
Transformation of susceptible tomato<br>
Seeds of the susceptible tomato line Moneymaker were rinsed in 70% ethanol to dissolve the seed coat and washed with sterile water. Subsequently, the seeds were surface-sterilised in 1.5% sodium hypochlorite for 15 minutes, rinsed three times in sterile water and placed in containers containing 140 ml MS medium pH 6.0 (Murashige and Skoog, 1962 Physiol. Plant. 15, 473-497) supplemented with 10 g/1 sucrose (MS 10) and 160 ml vermiculite. The seeds were left to germinate for 8 days at 25°C and 0.5 W/M2 light.<br>
Eight day old cotyledon explants were pre-cultured for 24 hours in Petri dishes containing a two week old feeder layer of tobacco suspension cells plated on co-cultivation medium (MS30 pH 5.8 supplemented with Nitsch vitamines (Duchefa Biochemie BV, Haarlem, The Netherlands), 0.5 g/1 MES buffer and 8 g/1 Daichin agar).<br>
Overnight cultures of A. tumefaciens were centrifuged and the pellet was resuspended in cell suspension medium (MS30 pH 5.8 supplemented with Nitsch vitamines, 0.5 g/1 MES buffer, pH 5.8) containing 200 uM acetosyringone to a final O.D.GOO of 0.25. The explants were then infected with the diluted overnight culture of A. tumefaciens strain UIA143 (Farrand et al., 1989 J. of Bacteriology 171, 5314-5321) containing the helper plasmid pCH32 (Hamilton et al., 1996 PNAS 93, 9975-9979) and pRGC2-blb for 25 minutes, blotted dry on sterile filter paper and co-cultured for 48 hours on the original feeder layer plates. Culture conditions were as described above.<br>
Following the co-cultivation, the cotyledons explants were transferred to Petri dishes with selective shoot inducing medium (MS pH 5.8 supplemented with 10 g/1 glucose, including Nitsch vitamines, 0.5 g/1 MES buffer, 5 g/1 agargeL 2 mg/1 zeatine riboside, 400 mg/1 carbenicilline, 100 mg/1 kanamycine, 0.1 mg/1 LAA) and cultured at 25°C with 3-5 W/m2 light. The explants were sub-cultured every 3 weeks onto fresh medium. Emerging shoots were dissected from the underlying callus and transferred to containers with selective root inducing medium (MS10 pH 5.8 supplemented with Nitsch vitamines, 0.5 g/1 MES buffer, 5 g/1 agargel, 0.25 mgfl IBA, 200 mg/1 carbenicillin and 100 mg/1 kanamycine).<br>
1 J<br><br>
Complementation of the susceptible phenotype in tomato<br>
To investigate whether Rpi-blb could complement the susceptible phenotype in tomato, primary transformants of Moneymaker harbouring the Rpi-blb gene construct were initially challenged with the potato derived P. infestans isolates IP0655-2A and IP0428. Seven out of nine primary transformants were resistant (Table 3). In view of the observation that the tested potato P. infestans isolates were less virulent on tomato than on potato, the primary transformants were also tested with a P. infestans isolate collected from susceptible home garden tomato plants. Even though this isolate was significantly more virulent on Moneymaker than the previously tested ones, all 7 primary transformants remained resistant. These results illustrate the potential effectiveness of the Rpi-blb gene not only against complex isolates derived from potato but also to those specialised on tomato.<br>
Molecular analysis of primary transformants<br>
RT-PCR analysis<br>
In order to produce cDNA, a mix of 19 (jtl containing 1 \xg of total or polyA RNA, 0.25 mM of each dNTP, 50 raM Tris-HCl pH 8.3, 75 mM KC1, 3 mM MgCb, 10 mM DTT and 530 ng oligo d(T) primer,<br>
GCTGTCAACGATACGCTACGTAACGGCATGACAGTG(T)i8 was denatured (1 min 83°C). Subsequently, the mix was placed at 42°C and 1 u.1 reverse transcriptase (M-MLV reverse transcriptase, Promega Benelux b.v., Leiden, The Netherlands) was added. After 60 min, the mix was heated for 1 min at 99°C and transferred to ice. 2 ul cDNA was used for standard PCR.<br>
Rapid amplification ofcDNA ends<br>
The 5' and 3' ends of the Rpi-blb cDNA were determined by rapid amplification of cDNA ends (RACE) using the GeheRacer™ kit (Invitrogen™, The Netherlands). 3' RACE was carried out with the primers GSPl (5'-GAGGAATCCATCTCCCAGAG) and GSP2 (5'-GTGCTTGAAGAGATGATAATTCACGAG) in combination with the GeneRacer™ 3' primer and GeneRacer™ 3' nested primer. 5' RACE was carried out on cDNA synthesised with the primer GSP3 (S'-GTCCATCTCACCAAGTAGTGG) using primers GSP4 (5'-GAAATGCTCAGTAACTCTCTGG) and GSP5 (5'-<br>
%7<br><br>
GGAGGACTGAAAGGTGTTGG) in combination with the GeneRacer™ 5' primer and GeneRacer™  5' nested primer (Figure 7).<br>
EXAMPLE 6: STRUCTURE OF THE Rpi-blb GENE AND THE CORRESPONDING PROTEIN.<br>
The size and structure of the Rpi-blb gene was determined by comparing the genomic sequence derived from the insert of pRGC2-blb with cDNA fragments generated by 5' and 3' rapid amplification of cDNA ends. RACE identified 5' and 3' Rpi-blb specific cDNA fragments of a single species, respectively, suggesting that the genomic clone encodes a single Rpi-blb specific transcript. The coding sequence of the Rpi-blb transcript is 2913 nucleotides The putative Rpi-blb transcript is estimated to be 3138 nucleotides (at) and contains a 44 and 181 nt long 5'- and 3'-untranslated region (UTR), respectively. The Rpi-blb gene contains a single intron of 678 nt starting 428 nt after the translational ATG start codon of the gene (Figure 3C).<br>
The deduced open reading frame of the Rpi-blb gene encodes a predicted polypeptide of 970 amino acids with an estimated molecular weight of 110.3 kD (Figure 8). Several functional motifs present in R genes of the NBS-LRR class of plant R genes are apparent in the encoded protein which can be subdivided into 3 domains (A, B and C; Figure 8). The N-terminal part of the protein contains potential<br>
i<br>
coiled-coil domains, heptad repeats in which the first and fourth residues are generally hydrophobic (domain A). Domain B harbours the NBS and other motifs that constitute the NB-ARC domain (ARC for Apaf-1, R protein, and CED-4) of R proteins and cell death regulators in animals (van der Biezen and Jones, 1998). This domain includes the Ap-ATP„ase motifs present in proteins of eukaryotic and prokaryotic origin (Aravind et al., 1999 Trends Biochem. Sci. 24, 47-53). The C-terminal half of Rpi-blb comprises a series of 19-20 irregular LRRs (domain C). The LRRs can be aligned according to the consensus sequence LxxLxxLxLxxC/N/SxxLxxLPxxa, where x designates any residue and "a" designates the positions of aliphatic amino acids, followed by a region of varying length. This repeat format approximates the consensus for cytoplasmic LRRs (Jones and Jones, 1997 Adv. Bot. Res.24, 89-167).<br>
EXAMPLE 7: NATURAL HOMOLOGUES AND ARTIFICIAL VARIANTS OF THE Rpi-blb GENE<br>
33<br><br>
Natural homologues<br>
BLASTN homology searches with the coding DNA sequence of the Rpi-blb gene identified a number of sequences with significant homology to short stretches of the Rpi-blb gene (Figure 9C). Nucleotides 549-1245 of the coding sequence of the Rpi-blb gene share SI- 90% sequence identity to partial NBS fragments from tomato (e.g. Q194, Q137, Q198 and Q199; Pan et al, 2000 Genetics. 155:309-22). These homologous sequences vary in length between 525 and 708 nucleotides and are PCR fragments which were identified by systematically scanning the tomato genome using (degenerate) primer pairs based on ubiquitous NBS motifs (Pan et al., 2000 Genetics. 155:309-22: Leister et al., 1996 Nat Genet. 14:421-429). Another region of the Rpi-blb gene which shares significant homology to a state of the art sequence comprises nucleotides 76-805 of the coding sequence. This 729 nt long sequence shares 91% sequence identity to an EST from potato (EMBL database accession no. BG890602; Figure 9C). The Rpi-blb gene sequence downstream of nucleotide 1245, comprising the LRR region, shares no significant homology to any state of the art sequence. BLASTX homology searches with the coding sequence of the Rpi-blb gene revealed that amino acid sequence homology with various state of the art genes does not exceed 36% sequence identity (Table 4). The best BLASTX score was obtained with an NBS-LRR gene derived from Oryza sativa (36.5% amino acid sequence identity). NBS-LRR genes sharing an overall sequence homology of 27-36% amino-acid sequence identity with Rpi-blb can be found among others in Arabidopsis thaliana, Phaseolus vulgaris, Lycopersicon esculentum (Fusarium 12 gene cluster; Ori et al., 1997 Plant CelL 9, 521-532; Simons et al, 1998 Plant Cell 10, 1055-1O68), Zea mays, Hordeum vulgare and Lactuca sativa. Phylogenetic studies of the deduced amino acid sequences of Rpi-blb, RGCl-blb, RGC3-blb, RGC4-blb and those of the homologous state of the art genes (as defined by BLASTX) derived from diverse species, using the Neighbour-Joining method of Saitou and Nei (1987 Molecular Biology and Evolution 4, 406-425), shows that members of the Rpi-blb gene cluster can be placed in a separate branch (Figure 9).<br>
Sequence comparisons of the four RGCs of the Rpi-blb gene cluster identified on 8005-8 BAC clone SPB4 show that sequence homology within the Rpi-blb gene cluster varies between 70% and 81% at the amino acid level. The deduced amino acid sequence of Rpi-blb shares the highest overall homology with RGC3-blb (81% amino acid sequence identity; Table 4). When the different domains are compared it is clear<br><br>
that the N-terminal halves of the proteins (coiled-coil and NB-ARC domains) share a higher degree of homology (91% amino acid sequence identity) than the C-terminal halves oi these proteins (LRRs; 71% amino acid sequence identity). The N-terminus of NBS-LRR proteins influences the requirement for downstream signalling components and is therefore thought to be the putative effector domain (Feys and Parker, 2000 Trends Gene: 16:449-55). The C-terminal LRR region is implicated, by genetic studies, in elicitor recognition specificity (Ellis et al., 2000 Trends Plant Sci. 5:373-379; Dodds et al, 2001 Plant Cell 13:163-78).<br>
Comparison of all four amino acid sequences revealed a total of 104 Rpi-blb specific amino acid residues (Figure 10A). The majority of these are located in the LRR region (SO/104). Within the latter region, these specific residues are concentrated in the LRR subdomain xxLxLxxxx. The relative frequency of these specific amino-acid residues within this LRR subdomain is more than two times higher (28.3%) than that observed in the rest of the LRR domain (12.3%). The residues positioned around the two conserved leucine residues in the consensus xxLxxLxxxx are thought to be solvent exposed and are therefore likely to be involved in creating/maintaining recognition specificity of the resistance protein.<br>
Sequences of additional homologues of the Rpi-blb gene can be obtained by screening genomic DNA or insert libraries, e.g. BAC libraries with primers based on signature sequences of the Rpi-blb gene. Screening of various Solarium BAC libraries with primer sets A and/or B (Table 2 and Figure 7) identified other Rpi-blb homologues derived from Solarium bulbocastanum (B149-blb), S. tuberosum (SH10-tub and SH20-tub) and S. tarijense (Tll8-tar). Comparison of all 8 protein sequences reduces the number of Rpi-blb specific amino acid residues to 51 (51/970; 5.25%) (Figure 10B). The majority of these are located in the LRR region (42/51; 82%). The relative frequency of these specific amino-acid residues within the LRR subdomain xxLxlxxxx is 3.3 times higher than that observed in the rest of the LRR domain (18.8% versus 5.7%, respectively). These data clearly suggest that evolution of P. infestans resistance specificity within the Rpi-blb gene cluster has mainly evolved through shifts in Rpi-blb LRR specific residues.<br>
Inclusion of the additional Rpi-blb homologues in the above described phylogenetic tree analyses, using the Neighbour-Joining method of Saitou and Nei (1987 Molecular Biology and Evolution 4, 406-425), further justifies phylogenetic tree analysis as a method to define Rpi-blb homologous sequences (Figure 9B). Any functional R gene product which shares at least 70% sequence identity at the amino<br><br>
acid level will end up in the same branch as gene products of the the Rpi-blb gene cluster and can thus be defined as being a homologue of Rpi-blb. Artificial variants<br>
Domain swaps between the different homologues can be made to ascertain the role of the different sequences in P. infestans resistance. The restriction enzyme Nsil for example, which recognises the DNA sequence ATGCAT present in the conserved MHD motif can be used to swap the complete LRR domain of Rpi-blb with that of RGCl-blb or RGC3-blb using techniques known to those skilled in the art. Chimeric variants of the Rpi-blb gene were made which encode the N-terminal half of Rpi-blb and the C-terminal half of RGCl-blb or RGC3-blb and visa versa, i.e., the N-terminal half of RGCl-blb or RGC3-blb and the C-terminal half of Rpi-blb (Figure 11). These variants were transformed to the susceptible potato genotype Impala and tested for P. infestans resistance. Chimeric RGC3-blb genes containing the LRR domain of Rpi-blb were resistant to P. infestans indicating that the specificity of the Rpi-blb gene is encoded by this part of the gene.<br>
3*<br><br><br><br><br><br><br><br><br>
a Primary transformants obtained from transformation of the susceptible potato and tomato genotypes Impala (IMP) and Moneymaker (MM), respectively, with T-DNA constructs containing the Rpi-blb gene candidates RGCl-blb, RGC2-blb, RGC3-blb or RGC4-blb. Agrobacterium tumefaciens strains AGLOb, LBA4404C, or UIA143d were used for transformation. Resistance was tested in detached leaf assays using the complex isolates IP0655-2A and IP0428-2.<br><br><br>
a Percentage nucleotide (nt) and amino acid (aa) sequence identity. b Separate comparisons were made for the N-terminal (N) and C-terminal (C) halves of the protein sequences. The border between the two halves is the conserved Nsil restriction sice in the DNA sequence (position 1417 of the Rpi-blb coding sequence).<br><br>
WE CLAIM:<br>
1.	A recombinant nucleic acid comprising<br>
(a)	a nucleic acid as depicted in Fig. 6;<br>
(b)	a nucleic acid coding for the amino acid sequence of Fig. 8; or<br>
(c)	a homologous nucleic acid sequence of (a) and/or (b) that is capable of providing at least partial resistance against an oomycete infection when incorporated and expressed in a Solanaceae plant or plant cell, and that encodes an amino acid sequence which is at least 82% identical to the amino acid sequence of Fig. 8.<br><br>
2.	The nucleic acid sequence as claimed in claim 1, wherein said Solanaceae plant or plant cell is derived from S. tuberosum.<br>
3.	The nucleic acid as claimed in claim 1, wherein said resistance is race non¬specific.<br>
4.	The nucleic acid as claimed in any one of claims 1 to 3, at least comprising a LRR domain.<br>
5.	A vector comprising the nucleic acid as claimed in any one of claims 1 to 4.<br>
6.	A proteinaceous substance encoded by the nucleic acid as claimed in any one of claims 1 to 4.<br>
7.	A proteinaceous substance comprising an amino acid sequence as depicted in Fig. 8 or a functional equivalent thereof.<br><br>
8.	A method for providing a plant or its progeny with at least partial resistance<br>
against an oomycete infection comprising providing said plant or part thereof<br>
by providing said plant or part thereof with the nucleic acid as claimed in any<br>
one of claims 1 to 4 or the vector as claimed in claim 5 or the substance as<br>
claimed in claim 6 or 7.<br>
9.	A method for selecting a plant or plant material or progeny thereof for its<br>
susceptibility or resistance to an oomycete infection comprising testing at least<br>
part of said plant or plant material or progeny thereof for the presence or<br>
absence of the nucleic acid as claimed in any of claims 1 to 4 or the vector as<br>
claimed in claim 5 or the substance as claimed in claim 6 or 7.<br>
10.	The method as claimed in claim 9, wherein said oomycete comprises<br>
Phytophthora infestans.<br>
11.	The method as claimed in claim 9 or 10, wherein said plant comprises S.<br>
tuberosum.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2OC1jaGVucC0yMDA0IGFic3RyYWN0IGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">1968-chenp-2004 abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2OC1jaGVucC0yMDA0IGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1968-chenp-2004 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2OC1jaGVucC0yMDA0IGNsYWltcyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1968-chenp-2004 claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2OC1jaGVucC0yMDA0IGNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1968-chenp-2004 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2OC1jaGVucC0yMDA0IGNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1968-chenp-2004 correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2OC1jaGVucC0yMDA0IGNvcnJlc3BvbmRlbmNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">1968-chenp-2004 correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2OC1jaGVucC0yMDA0IGRlc2NyaXB0aW9uKGNvbXBsZXRlKSBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1968-chenp-2004 description(complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2OC1jaGVucC0yMDA0IGRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">1968-chenp-2004 description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2OC1jaGVucC0yMDA0IGRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1968-chenp-2004 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2OC1jaGVucC0yMDA0IGZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">1968-chenp-2004 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2OC1jaGVucC0yMDA0IGZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">1968-chenp-2004 form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2OC1jaGVucC0yMDA0IGZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">1968-chenp-2004 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2OC1jaGVucC0yMDA0IGZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">1968-chenp-2004 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2OC1jaGVucC0yMDA0IGZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">1968-chenp-2004 form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2OC1jaGVucC0yMDA0IHBjdCBzZWFyY2ggcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">1968-chenp-2004 pct search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2OC1jaGVucC0yMDA0IHBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1968-chenp-2004 pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2OC1jaGVucC0yMDA0IHBldGl0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">1968-chenp-2004 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="230235-a-hapten-carrier-conjugate-comprising-virus-like-particles.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="230237-a-method-for-bearer-independent-call-control-bicc-in-optimization-for-ip-bearer-support.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>230236</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1968/CHENP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>25-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>03-Sep-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>KWEEK-EN RESEARCHBEDRIJF AGRICO B.V.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>BURCHTWEG 17, 8314 PP BANT,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ALLEFS, JOSEPHUS, JACOBUS, HENDRICUS, MARIA</td>
											<td>NAARDENSTRAAT 8, NL-8304 EJ EMMELOORD,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>VAN DER VOSSEN, EDWIN, ANDRIES, GERARD</td>
											<td>OORTLAAN 20, NL-3572 ZM UTRECHT,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>CO7K14/145</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/NL03/00091</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-02-07</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>02075565.8</td>
									<td>2002-02-08</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/230236-gene-conferring-resistance-to-phytophthora-infestans-late-blight-in-solanacea by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:59:46 GMT -->
</html>
