<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/230433-methods-of-increasing-abiotic-stress-tolerance-and-or-biomass-in-plants by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:12:47 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 230433:METHODS OF INCREASING ABIOTIC STRESS TOLERANCE AND/OR BIOMASS IN PLANTS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHODS OF INCREASING ABIOTIC STRESS TOLERANCE AND/OR BIOMASS IN PLANTS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>No.3482/CHENP/2005 ABSTRACT &quot;METHODS OF INCREASING ABIOTIC STRESS TOLERANCE AND/OR BIOMASS IN PLANTS&quot; The present invention relates to a method of increasing tolerance of a plant to an abiotic stress or increasing biomass of a plant, comprising:(a)expressing within the plant an exogenous polynucleotide at least 90% homologous to a polynucleotide which comprises SEQ ID NO: 13, thereby increasing the tolerance of the plant to the abiotic stress or increasing the biomass of the plant. (figures 1 to 5)</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
METHODS OF INCREASING ABIOTIC STRESS TOLERANCE AND/OR BIOMASS IN PLANTS AND PLANTS GENERATED THEREBY<br>
FIELD AND BACKGROUND OF THE INVENTION<br>
The present invention relates to methods of increasing abiotic stress tolerance and/or biomass in plants and, more particularly, to plants expressing exogenous abiotic stress-tolerance genes.<br>
Abiotic stress (also referred to as "environmental stress") conditions such as salinity, drought, flood, suboptimal temperature and toxic chemical pollution, cause substantial damage to agricultural plants. Most plants have evolved strategies to protect themselves against these conditions. However, if the severity and duration of the stress conditions are too great, the effects on plant development, growth and yield of most crop plants are profound. Furthermore, most of the crop plants are very susceptible to abiotic stress (ABS) and thus necessitate optimal growth conditions for commercial crop yields. Continuous exposure to stress causes major alterations in the plant metabolism which ultimately lead to cell death and consequently yield losses. Thus, despite extensive research and the use of sophisticated and intensive crop-protection measures, losses due to abiotic stress conditions remain in the billions of dollars annually (1,2).<br>
Developing stress-tolerant plants is a strategy that has the potential to solve or mediate at least some of these problems. However, traditional plant breeding strategies used to develop new lines of plants that exhibit tolerance to ABS are relatively inefficient since they are tedious, time consuming and of unpredictable outcome. Furthermore, limited germplasm resources for stress tolerance and incompatibility in crosses between distantly related plant species represent significant problems encountered in conventional breeding. Additionally, the cellular processes leading to ABS tolerance are complex in nature and involve multiple mechanisms of cellular adaptation and numerous metabolic pathways (4-7).<br>
Genetic engineering efforts, aimed at conferring abiotic stress tolerance to transgenic crops, have been described in the prior art. Studies by Apse and Blumwald (Curr Opin Biotechnol. 13:146-150, 2002), Quesada et al. (Plant Physiol. 130:951-963, 2Q02), Holmstrom et al. (Nature 379: 683-684, 1996), Xu et al. (Plant Physiol 110: 249-257, 1996), Pilon-Smits and Ebskamp (Plant Physiol 107: 125-130, 1995)<br>
2-<br><br>
and Tarczynski et al. (Science 259: 508-510, 1993) have all attempted at generating stress tolerant plants.<br>
In addition, several U.S. patents and patent applications also describe polynucleotides associated with stress tolerance and their use in generating stress tolerant plants. U.S. Pat. Nos. 5,296,462 and 5,356,816 describe transforming plants with polynucleotides encoding proteins involved in cold adaptation in Arabidopsis thaliana , to thereby promote cold tolerance in the transformed plants.<br>
U.S. Pat. No. 6,670,528 describes transforming plants with polynucleotides encoding polypeptides binding to stress responsive elements, to thereby promote tolerance of the transformed plants to abiotic stress.<br>
U.S. Pat. No. 6,720,477 describes transforming plants with a polynucleotide encoding a signal transduction stress-related protein, capable of increasing tolerance of the transformed plants to abiotic stress.<br>
U.S Application Ser. Nos. 09/938842 and 10/342224 describe abiotic stress-related genes and their use to confer upon plants tolerance to abiotic stress.<br>
U.S. Application Ser. No. 10/231035 describes overexpressing a molybdenum cofactor sulfurase in plants to thereby increase their tolerance to abiotic stress.<br>
Although the above described studies were at least partially successful in generating stress tolerant plants, there remains a need for stress tolerant genes which can be utilized to generate plants tolerant of a wide range of abiotic stress conditions.<br>
While reducing the present invention to practice, the present inventors have identified tiirough bioinmformatic and laboratory studies several novel abiotic stress-tolerance genes, which can be utilized to increase tolerance to abiotic stress and/or biomass in plants.<br>
SUMMARY OF THE INVENTION<br>
According to one aspect of the present invention there is provided a mediod of increasing tolerance of a plant to an' abiotic stress. The method includes expressing within the plant an exogenous polynucleotide at least 90% homologous to a polynucleotide selected from the group consisting of SEQ ID NOs: 1-18.<br>
According to an additional aspect of the present invention there is provided a-method of increasing tolerance of a plant to an abiotic stress.   The method includes<br>
3<br><br>
expressing within the plant an exogenous polynpeptide including an amino acid sequence selected from the group consisting of SEQ ID NOs: 39-92.<br>
According to another aspect of the present invention there is provided a method of increasing biomass of a plant. The method includes expressing within the plant an exogenous polynucleotide at least 90% homologous to a polynucleotide selected from the group consisting of SEQ ID NOs: 1-18.<br>
According to still additional aspect of the present invention there is provided a method of increasing biomass of a plant. The method includes expressing witiiin the plant an exogenous polypeptide including an amino acid sequence selected from the group consisting of SEQ ID NOs: 39-92.<br>
According to yet another aspect of the present invention there is provided a plant cell comprising an exogenous polynucleotide at least 90% homologous to a polynucleotide selected from the group consisting of SEQ ID NOs: 1-18.<br>
According to yet another aspect of the present invention there is provided a plant cell comprising an exogenous polynucleotide encoding a polypeptide including an amino acid sequence selected from the group consisting of SEQ ID NOs: 39-92.<br>
According to still another aspect of the present invention there is provided a nucleic acid construct, including a polynucleotide at least 90% homologous to a nucleotide sequence selected from the group consisting of SEQ ID NOs: 1-18 and a promoter capable of directing transcription of the polynucleotide in a host cell.<br>
According to another aspect of the present invention there is provided a nucleic acid construct, including a polynucleotide encoding a polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NOs: 39-92 and a promoter capable of directing transcription of the polynucleotide in a host cell.<br>
According to further yet another aspect of the present invention there is provided an isolated polypeptide, including an amino acid sequence at least 90% homologous to the amino acid sequence encoded by a polynucleotide selected from the group consisting of SEQ ID NOs: 1-18.<br>
According to an additional aspect of the present invention there is provided an isolated polypeptide including an amino acid sequence selected from die group consisting of SEQ ID NOs: 39-92.<br>
According to further features in preferred embodiments of the invention described below, the expressing is effected by (i) transforming a cell of the plant with<br>
4<br><br>
the exogenous polynucleotide; (ii) generating a mature plant from the cell; and (iii) cultivating the mature plant under conditions suitable for expressing the exogenous polynucleotide within the mature plant.<br>
According to still further features in the described preferred embodiments the transforming is effected by introducing to the plant cell a nucleic acid construct including the exogenous polynucleotide and at least one promoter capable of directing transcription of the exogenous polynucleotide in the plant cell.<br>
According to still further features in the described preferred embodiments the at least one promoter is a constitutive promoter.<br>
According to still further features in the described preferred embodiments the constitutive promoter is CaMV 35S promoter.<br>
According to still further features in the described preferred embodiments the constitutive promoter is At6669 promoter.<br>
According to still further features in the described preferred embodiments the at least one promoter is an inducible promoter.<br>
According to still further features in the described preferred embodiments the inducible promoter is an abiotic stress inducible promoter.<br>
According to still further features in the described preferred embodiments the at least one promoter is a tissue-specific promoter.<br>
According to still further features in the described preferred embodiments the expressing is effected by infecting the plant with a virus including the exogenous polynucleotide.<br>
According to still further features in the described preferred embodiments the virus is an avirulent virus.<br>
According to still further features in the described preferred embodiments die abiotic stress is selected from the group consisting of salinity, water deprivation, low temperature, high temperature, heavy metal toxicity, anaerobiosis, nutrient deficiency, nutrient excess, atmospheric pollution and UV irradiation.<br>
According to still further features' in the described preferred embodiments the plant is a dicotyledonous plant.<br>
According to still further features in the described preferred embodiments the plant is a monocotyledonous plant.<br>
5~<br><br>
According to still further features in the described preferred embodiments the plant cell forms a part of a plant.<br>
The present invention successfully addresses the shortcomings of the presently known configurations by providing methods of utilizing novel abiotic stress-tolerance genes to increase plants tolerance to abiotic stress and/or biomass.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
The invention is herein described, by way of example only, with reference to the accompanying drawings. With specific reference now to the drawings in detail, it is stressed that the particulars shown are by way of example and for purposes of illustrative discussion of the preferred embodiments of the present invention only, and are presented in the cause of providing what is believed to be the most useful and readily understood description of the principles and conceptual aspects of the invention. In this regard, no attempt is made to show structural details of the invention in more detail than is necessary for a fundamental understanding of the invention, the description taken with the drawings making apparent to those skilled in the art how the several forms of the invention may be embodied in practice.<br>
FIG. 1 is a flow chart illustrating a process of identifying putative plant stress-tolerance genes from nucleic-acid sequence databases.<br>
FIGs. 2A-B are photographs illustrating a T2 transgenic Arabidopsis thaliana mature plant at flowering stage, expressing exogenous luciferase transgene from the At6669 promoter . The same plant is shown under normal light conditions (Figure 2A) and in the dark (Figure 2B). Strong illumination indicative of luciferase expression is observed in the flower and root tissues.<br>
FIG. 3 illustrate the mean fresh weights of transgenic Ti A. thaliana plants grown under normal or stress conditions (irrigated with 0 or 100 M NaCl solution, respectively). The plants were transformed with putative stress tolerance genes, or with luciferase reporter gene (control), positioned under the transcriptional control of the At6669 promoter. Means followed by the same letter are not significantly different according to a one way ANOVA T-Test.<br>
FIG. 4A illustrates the mean fresh weights of T2 A. thaliana plants grown under normal or stress conditions (irrigated with 0 or 100 M NaCl solution, respectively). The plants were transformed with the putative stress tolerance genes of<br>
6<br><br>
the present invention, or with luciferase reporter gene (control), positioned under the transcriptional control of die 35S promoter. Means followed by the same letter are not significantly different according to a one way ANOVA T-Test.<br>
FIG. 4B illustrates the mean fresh weights of T2 A. thaliana plants grown under normal or stress conditions (irrigated with 0 or 100 M NaCl solution, respectively). The plants were transformed with the putative stress tolerance genes of the present invention, or with luciferase reporter gene (control), positioned under the transcriptional control of the At6669 promoter. Means followed by the same letter are not significantly different according to a one way ANOVA T-Test.<br>
FIG. 5 illustrates me relative (percent) fresh-weight of transgenic Ti A. thaliana plants grown under salinity stress conditions (irrigated with 100 mM NaCl solution), as compared with similar plants grown under normal conditions (irrigated with water only). The plants were transformed with the putative stress tolerance genes of the present invention, or with luciferase reporter gene (control), positioned under the transcriptional control of the At6669 promoter.<br>
DESCRIPTION OF THE PREFERRED EMBODIMENTS<br>
The present invention is of methods of increasing plants tolerance to abiotic stress and/or biomass by utilizing novel abiotic stress tolerance genes and of plants exhibiting increased tolerance to stress conditions and/or increased capacity to accumulate biomass.<br>
The principles and operation of the present invention may be better understood with reference to the drawings and accompanying descriptions.<br>
Before explaining at least one embodiment of the invention in detail, it is to be understood that the invention is not limited in its application to the details of construction and the arrangement of the components set forth in the following description or illustrated in the drawings. The invention is capable of other embodiments or of being practiced or carried out in various ways. Also, it is to be understood mat die phraseology and terminology employed herein is for the purpose of description and should not be regarded as limiting.<br>
While reducing the present invention to practice, the present inventors while employing bioinformatic techniques, identified polynucleotide sequences which encode putative abiotic-stress tolerance (ABST) proteins (Example 1).    Selected<br>
7<br><br>
sequences were isolated (Example 2), cloned into expression vectors (Example 3-4) and introduced into Arabidopsis thaliana plants (Example 5). These plants, which were grown under salinity stress conditions, or under normal conditions, exhibited significantly higher biomass as compared with similar plants not carrying the exogenous ABST genes (Example 6).<br>
Thus, according to one aspect of the present invention, mere is provided a method of increasing tolerance of a plant to an abiotic stress and/or plant biomass. The method includes expressing within the plant an exogenous polynucleotide at least 70% homologous, preferably at least 80% homologous, more preferably at least 85% homologous, most preferably at least 90% homologous to a polynucleotide selected from the group consisting of SEQ ID NOs: 1-18. Alternatively, the exogenous polynucleotide of the present invention encodes a polypeptide having an amino acid sequence selected from the group consisting of SEQ ID NOs: 39-92.<br>
The phrase "abiotic stress" used herein refers to any adverse effect on metabolism, growth, reproduction and/or viability of a plant. Accordingly, abiotic stress can be induced by suboptimal environmental growth conditions such as, for example, salinity, water deprivation, flooding, low or high temperature, heavy metal toxicity, anaerobiosis, nutrient deficiency, atmospheric pollution or UV irradiation.<br>
The phrase "abiotic stress tolerance" as used herein refers to the ability of a plant to endure an abiotic stress without suffering a substantial alteration in metabolism, growth, productivity and/or viability.<br>
A suitable plant for use with the metliod of the present invention can be any monocotyledonous or dicotyledonous plant including, but not limited to, maize, wheat, _ barely, rye, oat, rice, soybean, peanut, pea, lentil and alfalfa, cotton, rapeseed, canola, pepper, sunflower, potato, tobacco, tomato, eggplant, eucalyptus, a tree, an ornamental plant, a perennial grass and a forage crop.<br>
As used herein, the term "exogenous polynucleotide" refers to a nucleic acid sequence which is not naturally expressed within the plant but which, when introduced into the plant either in a stable or transient manner, produces at least one polypeptide product.<br>
Expressing the exogenous polynucleotide of the present invention within the plant can be effected- by transforming one or more cells of the plant with the exogenous   polynucleotide,   followed   by   generating   a   mature   plant   from   the<br>
?<br><br>
transformed cells and cultivating the mature plant under conditions suitable for expressing the exogenous polynucleotide within the mature plant.<br>
Preferably, the transformation is effected by introducing to the plant cell a nucleic acid construct which includes the exogenous polynucleotide of the present invention and at least one promoter capable of directing transcription of the exogenous polynucleotide in the plant cell. Further details of suitable transformation approaches are provided hereinbelow.<br>
As used herein, the term "promoter" refers to a region of DNA which lies upstream of the transcriptional initiation site of a gene to which RNA polymerase binds to initiate transcription of RNA. The promoter controls where (e.g., which portion of a plant, which organ within an animal, etc.) and/or when (e.g., which stage or condition in the lifetime of an organism) die gene is expressed.<br>
Any suitable promoter sequence can be used by the nucleic acid construct of the present invention. Preferably the promoter is a constitutive promoter, a tissue-specific, or an abiotic stress-inducible promoter.<br>
Suitable constitutive promoters include, for example, CaMV 35S promoter (SEQ ID NO: 19; Odell et aL, Nature 313:810-812, 1985); Arabidopsis At6669 promoter (SEQ ID NO: 20); maize Ubi 1 (Christensen et aL, Plant Sol. Biol. 18:675-689, 1992); rice actin (McElroy et aL, Plant Cell 2:163-171, 1990); pEMU (Last et aL, Theor. Appl. Genet. 81:581-588, 1991); and Synthetic Super MAS (Ni et aL, The Plant Journal 7: 661-76, 1995). Other constitutive promoters include those in U.S. Pat. Nos. 5,659,026, 5,608,149; 5.608,144; 5,604,121; 5.569,597: 5.466,785; 5,399,680; 5,268,463; and 5,608,142.<br>
Suitable tissue-specific promoters include, but not limited to, leaf-specific promoters such as described, for example, by Yamamoto et aL, Plant J. 12:255-265, 1997; Kwon et aL, Plant Physiol. 105:357-67, 1994; Yamamoto et aL, Plant Cell Physiol. 35:773-778, 1994; Gotor et aL, Plant J. 3:509-18, 1993; Orozco et aL, Plant Mol. Biol. 23:1129-1138, 1993; and Matsuoka et al,, Proc. Natl. Acad. Sci. USA 90:9586-9590, 1993.<br>
Suitable abiotic stress-inducible promoters include, but not limited to, salt-inducible promoters such as RD29A (Yamaguchi-Shinozalei et aL, Mol. Ge.n. Genet. 236:331-340, 1993); drought-inducible promoters such as maize rabl7 gene promoter (Pla et. aL, Plant Mol. Biol. 21:259-266, 1993), maize rab28 gene promoter (Busk et.<br>
1<br><br>
ai, Plant J. 11:1285-1295, 1997) and maize Ivr2 gene promoter (Pelleschi et. ai, Plant Mol. Biol. 39:373-380, 1999); and heat-inducible promoters such as heat tomato hsp80-promoter from tomato (U.S. Pat. No. 5,187,267).<br>
The nucleic acid construct of the present invention preferably further includes an appropriate selectable marker and/or an origin of replication. Preferably, the nucleic acid construct utilized is a shuttle vector, which can propagate both in E. coli (wherein the construct comprises an appropriate selectable marker and origin of replication) and be compatible for propagation in cells. The construct according to the present invention can be, for example, a plasmid, a bacmid, a phagemid, a cosmid, a phage, a virus or an artificial chromosome.<br>
The nucleic acid construct of the present invention can be utilized to stably or transiently transform plant cells. In stable transformation, the exogenous polynucleotide of the present invention is integrated into the plant genome and as such it represents a stable and inherited trait. In transient transformation, the exogenous polynucleotide is expressed by the cell transformed but it is not integrated into the genome and as such it represents a transient trait.<br>
There are various methods of introducing foreign genes into both monocotyledonous and dicotyledonous plants (Potrykus, I., Annu. Rev. Plant. Physiol., Plant. Mol. Biol. (1991) 42:205-225; Shimamoto et ai, Nature (1989) 338:274-276).<br>
The principle methods of causing stable integration of exogenous DNA into plant genomic DNA include two main approaches:<br>
(i) Agrobacterium-mediated gene transfer: Klee et al. (1987) Annu. Rev. Plant Physiol. 38:467-486; Klee and Rogers in Cell Culture and Somatic Cell Genetics of Plants, Vol. 6, Molecular Biology of Plant Nuclear Genes, eds. Schell, J., and Vasil, L. K., Academic Publishers, San Diego, Calif. (1989) p. 2-25; Gatenby, in Plant Biotechnology, eds. Kung, S. and Arntzen, C. J., Butterworth Publishers, Boston, Mass. (1989) p. 93-112.<br>
(ii) Direct DNA uptake: Paszkowski et al., in Cell Culture and Somatic Cell Genetics of Plants, Vol. 6, Molecular Biology of Plant Nuclear Genes eds. Schell, J., and Vasil, L. K., Academic Publishers, San Diego, Calif. (1989) p. 52-68; including methods for direct uptake of DNA into protoplasts, Toriyama, K. et al. (1988) Bio/Technology 6:1072-1074.   DNA uptake induced by brief electric shock of plant<br>
(O<br><br>
cells: Zhang et al. Plant Cell Rep. (1988) 7:379-384. Fromm et al. Nature (1986) 319:791-793. DNA injection into plant cells or tissues by particle bombardment, Klein et al. Bio/Technology (1988) 6:559-563; McCabe et al. Bio/Technology (1988) 6:923-926; Sanford, Physiol. Plant. (1990) 79:206-209; by the use of micropipette systems: Neuhaus et al., Theor. Appl. Genet. (1987) 75:30-36; Neuhaus and Spangenberg, Physiol. Plant. (1990) 79:213-217; glass fibers or silicon carbide whisker transformation of cell cultures, embryos or callus tissue, U.S. Pat. No. 5,464,765 or by the direct incubation of DNA with germinating pollen, DeWet et al. in Experimental Manipulation of Ovule Tissue, eds. Chapman, G. P. and Mantell, S. H. and Daniels, W. Longman, London, (1985) p. 197-209; and Ohta, Proc. Natl. Acad. Sci. USA (1986) 83:715-719.<br>
The Agrobacterium system includes the use of plasmid vectors that contain defined DNA segments that integrate into the plant genomic DNA. Methods of inoculation of the plant tissue vary depending upon the plant species and the Agrobacterium delivery system. A widely used approach is the leaf disc procedure which can be performed with any tissue explant that provides a good source for initiation of whole plant differentiation. Horsch et al. in Plant Molecular Biology Manual A5, Kluwer Academic Publishers, Dordrecht (1988) p. 1-9. A supplementary approach employs the Agrobacterium delivery system in combination with vacuum infiltration. The Agrobacterium system is especially viable in the creation of transgenic dicotyledonous plants.<br>
There are various methods of direct DNA transfer into plant cells. In electroporation, the protoplasts are briefly exposed to a strong electric field. In microinjection, the DNA is mechanically injected directly into the cells using very small micropipettes. In microparticle bombardment, the DNA is adsorbed on microprojectiles such as magnesium sulfate crystals or tungsten particles, and the microprojectiles are physically accelerated into cells or plant tissues.<br>
Following stable transformation plant propagation is exercised. • The most common method of plant propagation is by seed. Regeneration by seed propagation, however, has the deficiency that due to heterozygosity there is a lack of uniformity in the crop, since seeds are produced by plants according to the genetic variances governed by Mendelian rules. Basically, each seed is genetically different and each will grow with its own specific traits.   Therefore, it is preferred that the transformed<br>
((<br><br>
plant be produced such that the regenerated plant has the identical traits and characteristics of the parent transgenic plant. Therefore, it is preferred that the transformed plant be regenerated by micropropagation which provides a rapid, consistent reproduction of the transformed plants.<br>
Micropropagation is a process of growing new generation plants from a single piece of tissue that has been excised from a selected parent plant or cultivar. This process permits the mass reproduction of plants having the preferred tissue expressing the fusion protein. The new generation plants which are produced are genetically identical to, and have all of the characteristics of, the original plant. Micropropagation allows mass production of quality plant material in a short period of time and offers a rapid multiplication of selected cultivars in the preservation of the characteristics of the original transgenic or transformed plant. The advantages of cloning plants are the speed of plant multiplication and the quality and uniformity of plants produced.<br>
Micropropagation is a multi-stage procedure that requires alteration of culture medium or growth conditions between stages. Thus, the micropropagation process involves four basic stages: Stage one, initial tissue culturing; stage two, tissue culture multiplication; stage three, differentiation and plant formation; and stage four, greenhouse culturing and hardening. During stage one, initial tissue culturing, the tissue culture is established and certified contaminant-free. During stage two, the initial tissue culture is multiplied until a sufficient number of tissue samples are produced to meet production goals. During stage three, the tissue samples grown in stage two are divided and grown into individual plantlets. At stage four, the transformed plantlets are transferred to a greenhouse for hardening where the plants' tolerance to light is gradually increased so that it can be grown in the natural environment.<br>
Preferably, mature transformed plants generated as described above are further selected for abiotic stress tolerance. Accordingly, transformed and non-transformed (wild type) plants are exposed to an abiotic stress condition, such as water depravation, suboptimal temperature, nutrient deficiency, or preferably a salt stress condition. Salt stress can be effected in many ways such as, for example, by irrigating the plants with a hyperosmotic solution, by cultivating the plants hydroponically in a hyperosmotic growth solution (e.g., Hoagland solution), or by culturing the plants in a hyperosmotic growth medium (e.g., MS medium).  Since different plants vary considerably in their<br>
[^<br><br>
tolerance to salinity, the salt concentration in the irrigation water, growth solution, or growth medium is preferably adjusted according to the specific characteristics of the specific plant cultivar or variety, so as to inflict a mild or moderate effect on the physiology and/or morphology of the plants (for guidelines as to appropriate concentration please see, Bernstein and Kafkafi, Root Growth Under Salinity Stress In: Plant Roots, The Hidden Half 3rd ed. Waisel Y, Eshel A and Kafkafi U. (editors) Marcel Dekker Inc., New York, 2002, and reference therein). Following exposure to the stress condition the plants are frequently monitored until substantial physiological and/or morphological effects appear in wild type plants. Subsequently, transformed plants not exhibiting substantial physiological and/or morphological effects, or exhibiting higher biomass than wild-type plants, are identified as abiotic stress tolerant plants.<br>
Although stable transformation is presently preferred, transient transformation of leaf cells, meristematic cells or the whole plant is also envisaged by the present invention.<br>
Transient transformation can be effected by any of the direct DNA transfer methods described above or by viral infection using modified plant viruses.<br>
Viruses that have been shown to be useful for the transformation of plant hosts include CaMV, TMV and BV. Transformation of plants using plant viruses is described in U.S. Pat. No. 4,855,237 (BGV), EP-A 67,553 (TMV), Japanese Published Application No. 63-14693 (TMV), EPA 194,809 (BV), EPA 278,667 (BV); and Gluzman, Y. et al., Communications in Molecular Biology: Viral Vectors, Cold Spring Harbor Laboratory, New York, pp. 172-189 (1988). Pseudovirus particles for use in expressing foreign DNA in many hosts, including plants, is described in WO 87/06261.<br>
Preferably, the virus of the present invention is avirulent and thus is incapable of causing severe symptoms such as reduced growth rate, mosaic, ring spots, leaf roll, yellowing, streaking, pox formation, tumor formation and pitting. A suitable avirulent virus may be a naturally occurring avirulent virus or an artificially attenuated virus. Virus attenuation may be effected by using methods well known in the art including, but not limited to, sub-lethal heating, chemical treatment or by directed mutagenesis techniques such as described, for example, by Kurihara and<br>
13,<br><br>
Watanabe (Molecular Plant Pathology 4:259-269, 2003), Gal-on et al. (1992), Atreya et al. (1992) and Huet et al. (1994).<br>
Suitable virus strains can be obtained from available sources such as, for example, the American Type culture Collection (ATCC) or by isolation from infected plants. Isolation of viruses from infected plant tissues can be effected by techniques well known in the art such as described, for example by Foster and Tatlor, Eds. "Plant Virology Protocols: From Virus Isolation to Transgenic Resistance (Methods in Molecular Biology (Humana Pr), Vol 81)", Humana Press, 1998. Briefly, tissues of an infected plant believed to contain a high concentration of a suitable virus, preferably young leaves and flower petals, are ground in a buffer solution (e.g., phosphate buffer solution) to produce a virus infected sap which can be used in subsequent inoculations.<br>
Construction of plant RNA viruses for the introduction and expression of non-viral exogenous polynucleotide sequences in plants is demonstrated by the above references as well as by Dawson, W. O. et al., Virology (1989) 172:285-292; Takamatsu et al. EMBO J. (1987) 6:307-311; French et al. Science (1986) 231:1294-1297; and Takamatsu et al. FEBS Letters (1990) 269:73-76.<br>
When the virus is a DNA virus, suitable modifications can be made to the virus itself. Alternatively, the virus can first be cloned into a bacterial plasmid for ease of constructing the desired viral vector with the foreign DNA. The virus can then be excised from the plasmid. If the virus is a DNA virus, a bacterial origin of replication can be attached to the viral DNA, which is then replicated by the bacteria. Transcription and translation of mis DNA will produce the coat protein which will encapsidate the viral DNA. If the virus is an RNA virus, the virus is generally cloned as a cDNA and inserted into a plasmid. The plasmid is then used to make all of the constructions. The RNA virus is then produced by transcribing the viral sequence of the plasmid and translation of the viral genes to produce the coat protein(s) which encapsidate the viral RNA.<br>
Construction of plant RNA viruses for the introduction and expression in plants of non-viral exogenous polynucleotide sequences such as those included in the construct of the present invention is demonstrated by the above references as well as in U.S. Pat. No. 5,316,931.<br>
'&amp;<br><br>
In one embodiment, a plant viral polynucleotide is provided in which the native coat protein coding sequence has been deleted from a viral polynucleotide, a non-native plant viral coat protein coding sequence and a non-native promoter, preferably the subgenomic promoter of the non-native coat protein coding sequence, capable of expression in the plant host, packaging of the recombinant plant viral polynucleotide, and ensuring a systemic infection of the host by the recombinant plant viral polynucleotide, has been inserted. Alternatively, the coat protein gene may be inactivated by insertion of the non-na,tive polynucleotide sequence within it, such that a protein is produced. The recombinant plant viral polynucleotide may contain one or more additional non-native subgenomic promoters. Each non-native subgenomic promoter is capable of transcribing or expressing adjacent genes or polynucleotide sequences in the plant host and incapable of recombination with each other and with native subgenomic promoters. Non-native (foreign) polynucleotide sequences may be inserted adjacent the native plant viral subgenomic promoter or the native and a non-native plant viral subgenomic promoters if more than one polynucleotide sequence is included. The non-native polynucleotide sequences are transcribed or expressed in the host plant under control of die subgenomic promoter to produce the desired products.<br>
In a second embodiment, a recombinant plant viral polynucleotide is provided as in the first embodiment except that the native coat protein coding sequence is placed adjacent one of the non-native coat protein subgenomic promoters instead of a non-native coat protein coding sequence.<br>
In a third embodiment, a recombinant plant viral polynucleotide is provided in which the native coat protein gene is adjacent its subgenomic promoter and one or more non-native subgenomic promoters have been inserted into the viral polynucleotide. The inserted non-native subgenomic promoters are capable of transcribing or expressing adjacent genes in a plant host and are incapable of recombination wiui each other and with native subgenomic promoters. Non-native polynucleotide sequences may be inserted adjacent the non-native subgenomic plant viral promoters such that the sequences are transcribed or expressed in the host plant under control of the subgenomic promoters to produce the desired product.<br>
In a fourth embodiment, a recombinant plant viral polynucleotide is provided as in the third embodiment except that- the native coat protein coding sequence is replaced by a non-native coat protein coding sequence.<br>
l£<br><br>
The viral vectors are encapsidated by the coat proteins encoded by the recombinant plant viral polynucleotide to produce a recombinant plant virus. The recombinant plant viral polynucleotide or recombinant plant virus is used to infect appropriate host plants. The recombinant plant viral polynucleotide is capable of replication in the host, systemic spread in the host, and transcription or expression of foreign gene(s) (exogenous polynucleotide) in the host to produce the desired protein.<br>
Techniques for inoculation of viruses to plants may be found in Foster and Taylor, eds. "Plant Virology Protocols: From Virus Isolation to Transgenic Resistance (Methods in Molecular Biology (Humana Pr), Vol 81)", Humana Press, 1998; Maramorosh and Koprowski, eds. "Methods in Virology" 7 vols, Academic Press, New York 1967-1984; Hill, S.A. "Methods in Plant Virology", Blackwell, Oxford, 1984; Walkey, D.G.A. "Applied Plant Virology", Wiley, New York, 1985; and Kado and Agrawa, eds. "Principles and Techniques in Plant Virology", Van Nostrand-Reinhold, New York.<br>
In addition to the above, the polynucleotide of the present invention can also be introduced into a chloroplast genome thereby enabling chloroplast expression.<br>
A technique for introducing exogenous polynucleotide sequences to the genome of the chloroplasts is known. This technique involves the following procedures. First, plant cells are chemically treated so as to reduce the number of chloroplasts per cell to about one. Then, the exogenous polynucleotide is introduced via particle bombardment into the cells with the aim of introducing at least one exogenous polynucleotide molecule into the chloroplasts. The exogenous polynucleotides selected such that it is integratable into the chloroplast's genome via homologous recombination which is readily effected by enzymes inherent to the chloroplast. To this end, the exogenous polynucleotide includes, in addition to a gene of interest, at least one polynucleotide stretch which is derived from the chloroplast's genome. In addition, the exogenous polynucleotide includes a selectable marker, which serves by sequential selection procedures to ascertain that all or substantially all of the copies of the chloroplast genomes following such selection will include the exogenous polynucleotide. Further details relating to this technique are found in U.S. Pat. Nos. 4,945,050; and 5,693,507 which are incorporated herein by reference. A polypeptide can thus be produced by the protein expression system of the chloroplast and become integrated into the chloroplast's inner membrane.<br>
K&gt;<br><br>
Since abiotic stress tolerance in plants can involve multiple genes acting additively or in synergy (see, for example, in Quesda et a/., Plant Physiol. 130:951-063, 2002), the present invention also envisages expressing a plurality of exogenous polynucleotides in a single host plant to thereby achieve superior abiotic stress tolerance.<br>
Expressing a plurality of exogenous polynucleotides in a single host plant can be effected by co-introducing multiple nucleic acid constructs, each including a different exogenous polynucleotide, into a single plant cell. The transformed cell can than be regenerated into a mature plant using the methods described hereinabove.<br>
Alternatively, expressing a plurality of exogenous polynucleotides in a single host plant can be effected by co-introducing into a single plant-cell a single nucleic-acid construct including a plurality of different exogenous polynucleotides. Such a construct can be designed with a single promoter sequence which can transcribe a polycistronic message including all the different exogenous polynucleotide sequences. To enable co-translation of the different polypeptides encoded by the polycistronic message, the polynucleotide sequences can be inter-linked via an internal ribosome entry site (IRES) sequence which facilitates translation of polynucleotide sequences positioned downstream of the IRES sequence. In this case, a transcribed polycistronic RNA molecule encoding the different polypeptides described above will be translated from both the capped 5' end and the two internal IRES sequences of the polycistronic RNA molecule to thereby produce in the cell all different polypeptides. Alternatively, the construct can include several promoter sequences each linked to a different exogenous polynucleotide sequence.<br>
The plant cell transformed with the construct including a plurality of different exogenous polynucleotides, can be regenerated into a mature plant, using the methods described hereinabove. •<br>
Alternatively, expressing a plurality of exogenous polynucleotides in a single host plant can be effected by introducing different nucleic acid constructs, including different exogenous polynucleotides, into a plurality of plants. The regenerated transformed plants can then be cross-bred and resultant progeny selected for superior abiotic stress tolerance and/or biomass traits, using conventional plant breeding techniques.<br>
//<br><br>
Hence, the present application provides methods of utilizing novel abiotic stress-tolerance genes to increase tolerance to abiotic stress and/or biomass in a wide range of economical plants, safely and cost effectively.<br>
Additional objects, advantages, and novel features of the present invention will become apparent to one ordinarily skilled in the art upon examination of the following examples, which are not intended to be limiting. Additionally, each of the various embodiments and aspects of the present invention as delineated hereinabove and as claimed in the claims section below finds experimental support in the following examples.<br>
EXAMPLES<br>
Reference is now made to the following examples, which together with the above descriptions, illustrate the invention in a non limiting fashion.<br>
Generally, the nomenclature used herein and the laboratory procedures utilized in the present invention include molecular, biochemical, microbiological and recombinant DNA techniques. Such techniques are thoroughly explained in the literature. See, for example, "Molecular Cloning: A laboratory Manual" Sambrook et al., (1989); "Current Protocols in Molecular Biology" Volumes I-III Ausubel, R. M., ed. (1994); Ausubel et al., "Current Protocols in Molecular Biology", John Wiley and Sons, Baltimore, Maryland (1989); Perbal, "A Practical Guide to Molecular Cloning", John Wiley &amp; Sons, New York (1988); Watson et al., "Recombinant DNA", Scientific American Books, New York; Birren et al. (eds) "Genome Analysis: A Laboratory Manual Series", Vols. 1-4, Cold Spring Harbor Laboratory Press, New York (1998); methodologies as set forth in U.S. Pat. Nos. 4,666,828; 4,683,202; 4,801,531; 5,192,659 and 5,272,057; "Cell Biology: ALaboratory Handbook", Volumes I-III Cellis, J. E., ed. (1994); "Current Protocols in Immunology" Volumes I-III Coligan J. E., ed. (1994); Stites et al. (eds), "Basic and Clinical Immunology" (8th Edition), Appleton &amp; Lange, Norwalk, CT (1994); Mishell and Shiigi (eds), "Selected Methods in Cellular Immunology", W. H. Freeman and Co., New York (1980); available immunoassays are extensively described in the patent and scientific literature, see, for example, U.S. Pat. Nos. 3,791,932; 3,839,153; 3,850,752; 3,850,578; 3,853,987; 3,867,517;   3,879,262;   3,901,654;   3,935,074;   3,984,533;   3,996,345;   4,034,074;<br>
te<br><br>
4,098,876; 4,879,219; 5,011,771 and 5,281,521; "Oligonucleotide Synthesis" Gait, M. J., ed. (1984); "Nucleic Acid Hybridization" Hames, B. D., and Higgins S. J., eds. (1985); "Transcription and Translation" Hames, B. D., and Higgins S. J., eds. (1984); "Animal Cell Culture" Freshney, R. I., ed. (1986); "Immobilized Cells and Enzymes" IRL Press, (1986); "A Practical Guide to Molecular Cloning" Perbal, B., (1984) and "Methods in Enzymology" Vol. 1-317, Academic Press; "PCR Protocols: A Guide To Methods And Applications", Academic Press, San Diego, CA (1990); Marshak et al., "Strategies for Protein Purification and Characterization - A Laboratory Course Manual" CSHL Press (1996); all of which are incorporated by reference as if fully set forth herein. Other general references are provided throughout this document. The procedures therein are believed to be well known in the art and are provided for the convenience of the reader.<br>
Unless otherwise defined, all technical and scientific terms used herein have the same meaning as commonly understood by one of ordinary skill in the art to which this invention belongs. Although methods and materials similar or equivalent to those described herein can be used in the practice or testing of the present invention, suitable methods and materials are described below.<br>
EXAMPLE 1 Identifying putative abiotic stress- tolerance genes<br>
Putative abiotic stress-tolerance (ABST) genes were selected from NCBI databases of tomato expressed sequence tags (ESTs) and cDNAs. The database sequences were clustered and assembled using the LEADS™ software (Compugen). Clustering resulted in more than 20,000 clusters, each representing a different gene. An expression profile summary was compiled for each cluster by pooling all keywords included in the sequence records comprising the cluster. The clusters were men screened to include polynucleotides originating from libraries identified by keywords relating to ABST. The selected clusters were further filtered to exclude any cluster which included more than 100 ESTs per cluster and/or any cluster in which less than 50% of tlie sequences are annotated by ABST-related keywords.<br>
Prior art ABST plant genes were identified from the publications of Quesada et al. (Plant Physiol. 130:951-963, 2002); Apse and Blumwald (Curr Opin Biotechnol. 13:146-150, 2002); Rontein et al. (Metab Eng 4:49-56, 2002); and references therein.<br>
tf<br><br>
Known plant ABST genes were aligned with the clustered tomato nucleic-acid sequences, using the BLAST program. The tomato sequences having an e-score value lower than 5 were identified as ABST orthologes. Additional prior art tomato ABST genes were identified by searching the clustered tomato sequence records using the keywords "root", "crown gall", "nutrient", "callus", "disease", "pathogen", "elicitor" and "pseudomonas".<br>
Finally, all identified prior art ABST genes were matched (by sequence alignment using the BLAST software) with the output set of tomato gene clusters, • selected as described above. Consequently, about 40% of the genes selected in the output set of clusters which matched with prior art ABST genes proved to be known ABST genes, indicating that the remaining genes of the selected clusters are potentially capable of increasing abiotic stress tolerance in plants.<br>
The selected polynucleotide sequences (Table la), were analyzed for presence of ORFs using Vector NTI suite (InforMax, U.K.) version 6 (Hasting Software, Inc: www.generunner.com/'). ORFs identified in each of these polynucleotide sequences were compared to Genbank database sequences, using Blast (www.ncbi.nlni.nih.uov/BLAST/'); the ORF displaying the highest homology to a GenBank sequence or sequences, was mapped in order to identify an ATG start codon. The position of the ATG start codon of this ORF was then compared with that of the identified polynucleotide sequence in order to verify that each of the eighteen sequences described herein includes a full length ORF and an ATG start codon (thus qualifies as a "putative ABST gene").<br>
Table la Putative ABST genes<br>
ABST No.	SEQ ID No.<br>
1	1<br>
3	2<br>
5	3<br>
6	4<br>
10	5    .<br>
11	6<br>
12	7<br>
19	8<br>
22	9<br>
24	10<br>
26	-   11<br>
27	12<br>
36	13<br>
37	14<br>
2°<br><br>
39 TO	15<br><br>
16<br>
39 Tl<br>
49 TO	17<br>
49 Tl	18<br><br>
ABST polypeptide homologues have been identified from the NCBI databases using BLAST software (Table lb).<br>
Table lb<br>
ABST homologues<br>
ABST Putative Gene SEQ ID No.	ABST Polypeptide<br>
Homologue NCBI Accession No.	ABST Polypeptide Homologue SEQ ID No.	Homology (%)<br>
	BAA96366	39	98<br>
	AAS47510	40	98<br>
	NP 567151	41	97<br>
	NP 567104	42	96<br>
	AAK55664	43	96<br>
	P46298	44	97<br>
	T01338	45	96<br>
	T47888	46	95<br>
	BAD09465	47	92<br>
	O05761	48	91<br>
	BAD09464	49	88<br>
	CAA79496	50	84<br>
	EAJ94592	51	85<br>
4	NP 188036	52	76<br>
4	NP 035977	53	70<br>
4	XP 342608	54	. 69<br>
4	T09295	55	60<br>
4	NP 564717	56	59<br>
4	AAM63624	57	59<br>
9	P37707	58	93<br>
9	CAD37200	59	81<br>
9	CAA04664	60	78<br>
9	AAM64572	61	77<br>
9	NP 189345	62	77<br>
9	NP 974979	63	60<br>
13	AAC49992	64	88<br>
13	T10804	65	87<br>
13	AAL38357	66	87<br>
13	NP 188245	67	87<br>
13	NP 193465	68	87<br>
13	AAG44945	69	86<br>
13	T07819	70	86<br>
13	T12632	71	86<br>
13	CAC39073	72	86<br>
13	TO 1648	73	86<br>
13	AAF90121	74	86<br>
13	S48U6	75	86<br>
13	' AAO86710	76    '	86<br>
13	T14002	77	85<br>
13	T14001	78	85<br>
13	T48886	79	85<br>
2(<br><br>
13	T14314	80	85<br>
13	P33560	81	85<br>
13	P21653	82	85<br>
13	T14000	83	85<br>
13	T48884	84	85<br>
13	P24422	85	85<br>
13	AAB53329	86	85<br>
14	NP 200279	87	67<br>
14	AAM64276	88	67<br>
14	AA072577	89	66<br>
14	NP 175518	90	64<br>
14	BAC78588	91	64<br>
14	BAD03011	92	62<br>
EX.4MPLE 2 Isolation of putative A BS tolerance genes<br>
RNA was extracted from 4 week-old tomato root and leaf tissues using Tri<br>
Reagent (Molecular Research Center, Inc), following the protocol provided by the<br>
manufacturer (w-ww.mrcgene.com/tri.htrn). Complementary DNA molecules were<br>
produced from the extracted mRNA using M-MuLV reverse-transcriptase (RT)<br>
enzyme (Roche) and T16NN DNA primer, according to the manufacturer's<br>
instructions. The cDNA sequences set forth in SEQ ID NOs: 1,4,8-9 and 12-14, were<br>
amplified by PCR using the primers described in Table 2 below, with PFU proof<br>
reading	DNA	polymerase	enzyme	(Promega-<br>
wmv.promega.com/pnotes/68/7381 07/738 l_07.bt7.nl). following the protocol provided by the manufacturer. Additional restriction endonuclease sites were added to the 5' prime end of each primer to facilitate cloning of the putative ABS tolerance genes in binary vectors.<br><br>
Table 2 PCR primers used for amplifying putative ABS tolerance (ABST) genes<br>
ABST gene SEQ ID No	Forward Primer SEQ ED No	Reverse Primer SEQ ED No	upstream restriction site	downstream restriction site<br>
1	21	22	BamHl	SacI<br>
4	23	24	BamHl	Sad<br>
8	25	26	BamHl	SacI<br>
9	27	28	Xbal	Smal<br>
12	29	30	BamHl	SacI<br>
13	31	32	BamHl	SacI<br>
14	33	34	BamHl	Smal<br>
X^<br><br>
EXAMPLE 3 Cloning the putative ABSTgenes<br>
The resulting PCR blunt ended products were purified using PCR Purification Kit (Qiagen, Germany), digested with the appropriate restriction enzymes (Roche) and then inserted into the binary plasmid vector pPI. The plasmid pPI was constructed by inserting a synthetic poly-(A) signal sequence, originating from pGL3 basic plasmid vector (Promega, Ace No U47295; bp 4658-4811) into the HindlU restriction site of the binary vector pBHOl.3 (Clontech, Ace. No. U12640).<br>
The resulting pPI plasmid was digested with restriction enzymes (BamlU. and Sad; MB1 Fermentas) and purified using PCR Purification Kit (Qiagen, Germany). The open pPI construct was then ligated with each of the seven PCR products described hereinabove. The ligation was effected using a ligation mixture containing T4 DNA ligase enzyme (Roche) and was performed according to the manufacturer's instructions.<br>
The pPI constructs harboring putative ABST genes were introduced to E. coli DH5 competent cells by electroporation, using a MicroPulser electroporator (Biorad), 0.2 cm cuvettes (Biorad) and EC-2 electroporation program (Biorad). The treated cells were cultured in LB liquid medium at 37°C for 1 hr, then plated over LB agar supplemented with kanamycin (50 mg/L; Sigma) and incubated at 37°C for 16 hrs. Colonies which developed on the selective medium were analyzed by PCR using the primers set forth in SEQ CD NOs: 35-36, which were designed to span the inserted sequence in the pPI plasmid. The resulting PCR products were separated on 1.5% agarose gels and the DNA fragment having the predicted size were isolated and sequenced using the ABI 377 sequencer (Amersham Biosciences Inc) in order to verify that the correct DNA sequences were properly introduced to the E. coli cells.<br>
EXAMPLE 4 Generating binary vectors comprising putative ABST genes and plant promoters<br>
operably linked thereto<br>
Generating binary vectors comprising the Cauliflower Mosaic Virus 35S<br>
promoter: The Cauliflower Mosaic Virus 35S promoter sequence (set forth in SEQ ED<br>
NO:  19) was inserted upstream of the putative ABST gene in each of the pPI<br>
constructs described above.   The promoter was isolated from the pBI121 plasmid<br>
23<br><br>
(Clontech, Accession No. AF485783) using the restriction endonucleases HindUI -and BamUl (Roche). The isolated promoter was ligated into the pPI constructs digested with the same enzymes. Altogether, seven pPI constructs were generated, each comprising the CaMV 35S promoter positioned upstream of a putative ABST gene having a sequence set forth in SEQ ID NO: 1,4,8,9,12,13 or 14.<br>
Generating binary vectors comprising the At6669 promoter: The At6669 promoter sequence (set forth in SEQ ID NO: 20) was inserted upstream of the putative ABST gene in each of the pPI binary constructs described above. The promoter was isolated from Arabidopsis thaliana var ColO genomic DNA by PCR amplification using the primers set forth in SEQ ID NOs: 37-38. The PCR product was purified (Qiagen, Germany) and digested with the restriction endonucleases HindlU and BamHL (Roche). The resulting promoter sequence was introduced into the open binary constructs digested with the same enzymes. Altogether, seven pPI constructs were generated, each comprising the At6669 promoter positioned upstream of a putative ABST gene having a sequence set forth in SEQ ID NO: 1,4,8,9,12,13 or 14.<br>
EXAMPLE 5 Confirming At6669promoter activity in transgenic Arabidopsis thaliana<br>
The capacity of At-6669 promoter to regulate transcription of genes carried by the pPI vector in plants was tested. Accordingly, the promoter At6669 was inserted into the pPI binary vector upstream of a Luciferase reporter gene. The binary vector was introduced to Arabidopsis thaliana plants using the procedure as described in Example 6 below. Mature transformed T2 Arabidopsis plants were assayed for bio-illumination in a darkroom using an ultra-low light detection camera (Princeton Instruments Inc., USA) using the procedure described by Meissner et al. (Plant J. 22:265, 2000). Illumination indicating positive Luciferase activity was observed in the flower and root meristem tissues of transformed plants (Figure 2).<br>
EXAMPLE 6 Transforming Agrobacterium tumefaciens cells with binary vectors harboring<br>
putative ABST genes<br>
Each of the binary vectors described in Example 4 above were used to transform Agrobacterium  cells.     Two  additional  binary  constructs,  having  the<br>
2 if<br><br>
Luciferase reporter gene replacing an ABST gene (positioned downstream of the 35S or At6669 promoter), were used as negative controls.<br>
The binary vectors were introduced to Agrobacterium tumefaciens GV301, or LB4404 competent cells (about 109 cells/mL) by electroporation. The electroporation was effected by using a MicroPulser electroporator (Biorad), 0.2 cm cuvettes (Biorad) and EC-2 electroporation program (Biorad). The treated cells were cultured in LB liquid medium at 28°C for 3 hr, then plated over LB agar supplemented with gentamycin (50 mg/L; for Agrobacterium strains GV301) or streptomycin (300 mg/L; for Agrobacterium strain LB4404) and kanamycin (50 mg/L) at 28°C for 48 hrs. Abrobacterium colonies which developed on the selective media were analyzed by PCR using the primers set forth in SEQ ID NOs: 35-36, which were designed to span the inserted sequence in die pPI plasmid. The resulting PCR products were isolated and sequenced as described in Example 4 above, to verify that the correct ABST sequences were properly introduced to the Agrobacterium cells.<br>
EXAMPLE 7 Transformation of Arabidopsis thaliana plants with putative ABST genes<br>
Arabidopsis thaliana Columbia plants (To plants) were transformed using die Floral Dip procedure described by Clough and Bent (10) and by Desfeux et al. (11), with minor modifications. Briefly, To Plants were sown in 250 ml pots filled with wet peat-based growth mix. The pots were covered with aluminum foil and a plastic dome, kept at 4°C for 3—4 days, then uncovered and incubated in a growth chamber at 18-24°C under 16/8 hr light/dark cycles. The To plants were ready for transformation six days before anmesis.<br>
Single colonies of Agrobacterium carrying the binary constructs, generated as described in Example 6 above, were cultured in LB medium supplemented with kanamycin (50 mg/L) and gentamycin (50 mg/L). The cultures were incubated at 28°C for 48 hrs under vigorous shaking and then centrifuged at 4000 rpm for 5 minutes. The pellets comprising Agrobacterium cells were re-suspended in a transformation medium containing half-strength (2.15 g/L) Murashig-Skoog (Duchefa); 0.044 uM benzylamino purine (Sigma); 112 ug/L B5 Gambourg vitamins (Sigma); 5% sucrose; and 0.2 ml/L Silwet L-77 (OSI Specialists, CT) in double-distilled water , at pH of 5.7.<br>
X^<br><br>
Transformation of To plants was effected by inverting each plant into an Agrobacterium suspension, such that the above ground plant tissue was submerged for 3-5 seconds. Each inoculated To plant was immediately placed in a plastic tray, then covered with clear plastic dome to maintain humidity and was kept in the dark at room temperature for 18 hrs, to facilitate infection and transformation. Transformed (transgenic) plants were then uncovered and transferred to a greenhouse for recovery and maturation. The transgenic To plants were grown in the greenhouse for 3-5 weeks until siliques were brown and dry. Seeds were harvested from plants and kept at room temperature until sowing.<br>
For generating Ti and T2 transgenic plants harboring the genes, seeds collected from transgenic To plants were surface-sterilized by soaking in 70% ethanol for 1 minute, followed by soaking in 5% sodium hypochloride and 0.05% triton for 5 minutes. The surface-sterilized seeds were thoroughly washed in sterile distilled water then placed on culture plates containing half-strength Murashig-Skoog (Duchefa); 2% sucrose; 0.8% plant agar; 50 mM kanamycin; and 200 mM carbenicylin (Duchefa). The culture plates were incubated at 4°C for 48 hours then transferred to a growth room at 25°C for an additional week of incubation. Vital Ti Arabidopsis plants were transferred to a fresh culture plates for another week of incubation. Following incubation the Ti plants were removed from culture plates and planted in growth mix contained in 250 ml pots. The transgenic plants were allowed to grow in a greenhouse to maturity. Seeds harvested from Ti plants were cultured and grown to maturity as T2 plants under the same conditions as used for culturing and growing the Ti plants.<br>
EXAMPLE 8 Evaluating growth of transgenic plants cultivated under abiotic stress conditions Methods:<br>
Ti or Ta transgenic plants generated as described above were individually transplanted into pots containing a growth mixture of peat and vermiculite (volume ratio 3:2, respectively). The pots were covered for 24 hr period for hardening, then placed in the greenhouse in complete random order and irrigated with tap water (provided from the pots' bottom every 3-5 days) for seven days. Thereafter, half of the plants were irrigated with a salt solution (100 mM NaCl and 5 mM CaCU) to induce salinity stress (stress conditions). The other half of the plants were continued<br>
Z4<br><br>
to be irrigated with tap water (normal conditions). All plants were grown in the greenhouse at 100% RH for 28 days, then harvested (the above ground tissue) and weighted (immediately or following drying in oven at 50°C for 24 hr).<br>
Results:<br>
No significant differences in plant fresh weights were observed between Ti plants transformed with 3 different ABST genes and plants transformed with the Luciferase reporter gene, grown either under normal or stress conditions (Figure 3 and Table 3 below). Yet, Ti plants, transformed with SEQ ED NO: 1 positioned under the regulatory control of die At6669 promoter maintained 71% of their fresh weight when exposed to stress conditions, while the control plants (carrying Luciferase gene positioned under the regulatory control of the AT6669 promoter) maintained only 61% of their fresh weight under similar stress conditions.<br>
Table 3<br>
Fresh weight	ofTi transgenic Arabidopsis plants irrigated with water or	salt solution<br>
Transgene<br>
(SEQID<br>
NO)	Promoter	N Rows1	Irrigation solution (mM NaCI)	Mean (g)	Std Error<br>
Luciferase	At6669	2	0	0.7925	0.0275<br>
Luciferase	At6669	2	100	0.485	0.045<br>
13	At6669	8	0	0.81625	0.020305<br>
13	At6669	8	100	0.4725	0.029246<br>
1	At6669	8	0	0.7875	0.026032<br>
1	At6669	8	100	0.55875	0.044699<br>
8	At6669	8	0	0.8575	0.023088<br>
8	At6669	8	100	0.440625	0.011198<br>
N Rows represent number of independent transformation event plants measured. For each transgene, 3-5 independent transformation events with 1-3 plants per a single transformation event were used.<br>
T2 plants transformed with SEQ ID NOs: 7 or 14 positioned under the regulatory control of die 35S promoter accumulated significantly higher biomass than control plants, regardless of growth conditions. As shown in Figure 4A and Table 4 below, the mean fresh weight of plants transformed with SEQ ID NOs: 7 and 14, grown under stress conditions, were 15% and 24%, respectively, higher than the mean fresh weight control plants grown under similar stress conditions. Similarly, the mean fresh weight of plants transformed wim SEQ ID NOs: 7 or 14, grown under normal conditions, were 21% and 27%, respectively, higher than the mean fresh weight control plants grown under similar normal conditions.<br>
2,7<br><br>
Similar phenomenon was observed with T2 plants transformed with SEQ ID NO: 4 positioned under the regulatory control of the 35S promoter. Accordingly, as shown in Figure 4A and Table 4 below, the mean fresh weight of plants tiansformed with SEQ ID NO: 4 was 14% and 7% was higher than the mean fresh weight of control plants grown under stress and normal conditions, respectively. Similarly, T2 plants transformed with SEQ ID NO: 4 positioned under the regulatory control of the At6669 promoter exhibited 1.3 and 5% higher biomass than control plants grown under stress and normal conditions, respectively. However, these differences wefe not found statistically different under the experimental conditions.<br>
Table 4<br><br>
Fresh weight of	T2 transgenic Arabidopsis plants irrigated with water or salt so tunc<br>
Transgcne<br>
(SEQ ID<br>
NO)	Promoter	N Rows	Irrigation<br>
solution (raM<br>
NaCl)	Mean (g)	Std Error<br>
Luciferase	CaMV-35S	11	0	0.352727	0.011208<br>
Luciferase	CaMV-35S	11	100	0.280909	0.010484<br>
9	CaMV-35S	11	0	0.426364	0.019599<br>
9	CaMV-35S	11	100	0.322727	0.027306<br>
12	CaMV-35S	11	0	0.374545	0.015746<br>
12	CaMV-35S	11	100	0.249091	0.020647<br>
1	CaMV-35S	8	0	0.36625	0.034171<br>
1	CaMV-35S	8	100	0.265	0.031225<br>
13	CaMV-35S	11	0	0.349091	0.013515<br>
13	CaMV-35S	11	100	0.293636	0.019921<br>
14	CaMV-35S	11	0	0.446364	0.025558<br>
14	CaMV-35S	11	100	0.348182	0.023772<br>
8	CaMV-35S	11	0	0.310909	0.015223<br>
8	CaMV-35S	11	100	0.253636	0.01539<br>
4	CaMV-35S	11	0	0.379091	0.010992<br>
4	CaMV-35S	11	100	0.318182	0.013336<br>
N Rows represent number of independent transformation event plants measured. For each transgene, 3-5 independent transformation events with 1-3 plants per a single transformation event were used.<br>
T2 plants transformed with SEQ ID NOs: 1 and 13 positioned under the regulatory control of the At6669 promoter and grown under stress conditions, exhibited significantly higher biomass than control plants grown under similar stress conditions. The mean fresh weight of T2 plants transformed with SEQ ID NOs: 1 and 13 positioned under the regulatory control of the'At6669 promoter, and grown under stress conditions, were 37% and 21%, respectively, higher than the mean fresh weight control plants grown under similar stress conditions (Figure 4B and Table 5 below). No significant increase in biomass over control was observed when these transgenic<br>
2-8<br><br>
plants (carrying SEQ ID NOs: 1 and 13 regulated under At6669 promoter) where grown under normal conditions.<br><br>
Table S<br>
Fresh weight ofT; transgenic Arabidopsis plants irrigated with water or salt solution<br>
™	"   "	Irrigation<br>
Irrigation<br>
solution<br>
(mM NaCl)<br>
Promoter<br>
N Rows<br>
Mean (g)<br>
Std Error<br>
Transgene (SEQ ID NO)<br>
0.3<br>
0.010328<br>
M6669<br>
0<br>
Luciferase<br>
100<br>
0.125<br>
0.009916<br>
At6669<br>
Luciferase<br>
0.286667<br>
At6669<br>
0.024449<br>
13<br>
100<br>
0.151667<br>
0.007032<br>
At6669<br>
13<br>
0.305<br>
•At6669<br>
0.03423<br>
100<br>
0.171667<br>
At6669<br>
0.012225<br>
0.315<br>
At6669<br>
0.049983<br>
0.126667<br>
100<br>
A16669<br>
0.005578<br>
0.263333<br>
12<br>
At6669<br>
0.012824<br>
100<br>
At6669<br>
0.098333<br>
12<br>
0.007923<br>
0.228333<br>
At6669<br>
0<br>
0.020235<br>
0.004014<br>
o	i     At6669   7       6      |	100        I   0.121667 
'N Rows represent number of independent transformation event plants measured. For each transgene, 3-5 independent transformation events with 1-3 plants per a single transformation event were used.<br><br>
The results illustrate that the isolated putative ABST genes, set forth in SEQ ID NOs: 1 and 13, are capable of increasing plant tolerance to abiotic stress, such as a salinity stress. In addition, the isolated putative ABST genes set forth in SEQ ID NOs: 7, 14 (and possibly also 4), are capable of substantially promoting biomass in plants grown under stress, as well as under normal conditions.<br>
Hence, the results clearly indicate that the putative abiotic stress tolerance genes described herein can be readily isolated and utilized to substantially increase tolerance to abiotic stress and/or biomass in plants.<br>
It is appreciated that certain features of the invention, which are, for clarity, described in the context of separate embodiments, may also be provided in combination in a single embodiment. Conversely, various features of the invention, which are, for brevity, described in the context of a single embodiment, may also be provided separately or in any suitable subcombination.<br>
Almough the invention has been described in conjunction with specific embodiments thereof, it is evident that many alternatives, modifications and variations will be apparent to those skilled in the art. Accordingly, it is intended to embrace all<br>
2?<br><br>
such alternatives, modifications and variations that fall within the spirit and broad scope of the appended claims. All publications, patents, patent applications and sequences identified by their accession numbers mentioned in this specification are herein incorporated in their entirety by reference into the specification, to the same extent as if each individual publication, patent or patent application or sequence identified by their accession number was specifically and individually indicated to be incorporated herein by reference. In addition, citation or identification of any reference in this application shall not be construed as an admission that such reference is available as prior art to the present invention.<br>
30<br><br>
REFERENCES CITED (Additional references are cited hereinabove)<br>
1.	www.fao.org/ag/agl/agll/spush/degrad.htm.<br>
2.	www.fao.org/ag/agl/aglw/watermanagement/introduc.strn<br>
3.	McCue KF, Hanson AD (1990).Drought and salt tolerance: towards understanding and application. Trends Biotechnol 8: 358-362.<br>
4.	Flowers TJ, Yeo Ar (1995). Breeding for salinity resistance in crop plants: where next? Aust J Plant Physiol 22:875-884.<br>
5.	Nguyen BD, Brar DS, Bui BC, Nguyen TV, Pham LN, Nguyen HT (2003). Identification and mapping of the QTL for aluminum tolerance introgressed from the new source, ORYZA RUFIPOGON Griff., into indica rice ( Oryza sativa L.). Theor Appl Genet. 106:583-93.<br>
6.	Sanchez AC, Subudhi PK, Rosenow DT, Nguyen HT (2002). Mapping QTLs associated with drought resistance in sorghum (Sorghum bicolor L. Moench). Plant Mol Biol. 48:713-26.<br>
7.	Quesada V, Garcia-Martinez S, Piqueras P, Ponce MR, Micol JL (2002). Genetic architecture of NaCl tolerance in Arabidopsis.<br>
Plant Physiol. 130:951-963.<br>
8.	Apse MP, Blumwald E (2002). Engineering salt tolerance in plants. Curr Opin Biotechnol. 13:146-150.<br>
9.	Rontein D, Basset G, Hanson AD (2002). Metabolic engineering of osmoprotectant accumulation in plants.<br>
Metab Eng 4:49-56<br>
10.	Clough SJ, Bent AF (1998). Floral dip: a simplified method for Agrobacterium-mediated transformation of Arabidopsis thaliana. Plant J 16:735-43.<br>
11.	Desfeux C, Clough SJ, Bent AF (2000). Female reproductive tissues are the primary target of Agrobacterium-mediated transformation by the Arabidopsis floral-dip method. Plant Physiol 123:895-904.<br>
3'<br><br><br><br><br><br>
WE CLAIM :<br>
1.	A method of increasing tolerance of a plant to an abiotic stress or increasing<br>
biomass of a plant, comprising:<br>
(a)      expressing  within  the  plant  an  exogenous  polynucleotide  at  least  90% homologous to a polynucleotide which comprises SEQ ID NO: 13, thereby increasing the tolerance of the plant to the abiotic stress or increasing the biomass of the plant.<br>
2.	The method as claimed in claim 1, wherein said expressing is effected by transforming a cell of said plant with said exogenous polynucleotide.<br>
3.	The method as claimed in claim 2, wherein said transforming is effected by introducing to said plant cell a nucleic acid construct including said exogenous polynucleotide and at least one promoter capable of directing transcription of said exogenous polynucleotide in said plant cell.<br>
4.	The method as claimed in claim 3, wherein said at least one promoter is a constitutive promoter.<br>
5.	The method as claimed in claim 3, wherein said constitutive promoter is CaMV 35S promoter.<br>
6.	The method as claimed in claim 3, wherein said constitutive promoter is<br>
At6669 promoter.<br>
7.	The method as claimed in claim 3, wherein said at least one promoter is an<br>
inducible promoter.<br>
32<br><br>
8.	The method as claimed in claim 7, wherein said inducible promoter is an abiotic stress inducible promoter.<br>
9.	The method as claimed in claim 1, wherein said expressing is effected by infecting said plant with a virus including said exogenous polynucleotide.<br>
10.	The method as claimed in claim 9, wherein said virus is an avirulent virus.<br>
11.	The method as claimed in claim 1, wherein said abiotic stress is selected from the group consisting of salinity, water deprivation, low temperature, high temperature, heavy metal toxicity, anaerobiosis, nutrient deficiency, nutrient excess, atmospheric pollution and UV irradiation.<br>
12.	The method as claimed in claim 1, wherein said plant is a dicotyledonous plant.<br>
13.	The method as claimed in claim 1, wherein said plant is a monocotyledonous plant.<br>
14.	A nucleic acid construct, comprising a polynucleotide at least 90% homologous to a nucleotide sequence which comprises SEQ ID NO: 13, and a promoter capable of directing transcription of the polynucleotide in a host cell.<br>
15.	The construct as claimed in claim 14, wherein said promoter is a constitutive promoter.<br>
16.	The construct as claimed in claim 15, wherein said constitutive promoter is CaMV 35S promoter.<br>
33<br><br>
17.	The construct as claimed in claim 15, wherein said constitutive promoter is<br>
At6669 promoter.<br>
18.	The construct as claimed in claim 14, wherein said promoter is an inducible promoter.<br>
19.	The construct as claimed in claim 18, wherein said inducible promoter is an abiotic stress inducible promoter.<br><br>
20.	The construct as claimed in claim 14, wherein said host cell is a plant cell.<br>
21.	An isolated polypeptide, comprising an amino acid sequence at least 90% homologous to the amino acid sequence encoded by a polynucleotide which comprises SEQ ID NO: 13.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ4Mi1jaGVucC0yMDA1IGFic3RyYWN0IGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">3482-chenp-2005 abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ4Mi1jaGVucC0yMDA1IGFic3RyYWN0LTEuanBn" target="_blank" style="word-wrap:break-word;">3482-chenp-2005 abstract-1.jpg</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ4Mi1jaGVucC0yMDA1IGFic3RyYWN0LTIuanBn" target="_blank" style="word-wrap:break-word;">3482-chenp-2005 abstract-2.jpg</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ4Mi1jaGVucC0yMDA1IGFic3RyYWN0LTMuanBn" target="_blank" style="word-wrap:break-word;">3482-chenp-2005 abstract-3.jpg</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ4Mi1jaGVucC0yMDA1IGFic3RyYWN0LmpwZw==" target="_blank" style="word-wrap:break-word;">3482-chenp-2005 abstract.jpg</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ4Mi1jaGVucC0yMDA1IGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">3482-chenp-2005 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ4Mi1jaGVucC0yMDA1IGNsYWltcyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">3482-chenp-2005 claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ4Mi1jaGVucC0yMDA1IGNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">3482-chenp-2005 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ4Mi1jaGVucC0yMDA1IGNvcnJlc3BvbmRhbmNlIG90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">3482-chenp-2005 correspondance others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ4Mi1jaGVucC0yMDA1IGNvcnJlc3BvbmRhbmNlIHBvLnBkZg==" target="_blank" style="word-wrap:break-word;">3482-chenp-2005 correspondance po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ4Mi1jaGVucC0yMDA1IGRlc2NyaXB0aW9uKGNvbXBsZXRlKSBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">3482-chenp-2005 description(complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ4Mi1jaGVucC0yMDA1IGRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">3482-chenp-2005 description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ4Mi1jaGVucC0yMDA1IGRyYXdpbmdzIGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">3482-chenp-2005 drawings duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ4Mi1jaGVucC0yMDA1IGRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">3482-chenp-2005 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ4Mi1jaGVucC0yMDA1IGZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">3482-chenp-2005 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ4Mi1jaGVucC0yMDA1IGZvcm0tMTMucGRm" target="_blank" style="word-wrap:break-word;">3482-chenp-2005 form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ4Mi1jaGVucC0yMDA1IGZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">3482-chenp-2005 form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ4Mi1jaGVucC0yMDA1IGZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">3482-chenp-2005 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ4Mi1jaGVucC0yMDA1IGZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">3482-chenp-2005 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ4Mi1jaGVucC0yMDA1IGZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">3482-chenp-2005 form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ4Mi1jaGVucC0yMDA1IHBjdCBzZWFyY2ggcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">3482-chenp-2005 pct search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ4Mi1jaGVucC0yMDA1IHBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">3482-chenp-2005 pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ4Mi1jaGVucC0yMDA1IHBldGl0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">3482-chenp-2005 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="230432-a-print-ink-and-a-method-of-recording-inkjet.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="230434-vanadium-titania-based-catalyst-for-removing-nitrogen-oxide-at-low-temperature-window.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>230433</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3482/CHENP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>26-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>22-Dec-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>EVOGENE LTD.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>13 Gad Finstein Street, 76121 Rehovot,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>RONEN, Gil</td>
											<td>20 MOSHAV OMETZ, 33870 EMEQ-HEFER,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>GOLAN, Ezekiel</td>
											<td>5 GIMMEL KEHILAT LEVOV STREET, 69703 TEL AVIV,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>KARCHI, Hagai</td>
											<td>249 Sapir street, Moshav Sitriya, 76834 Doar-Na Emek Soreq,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>MEISSNER, Rafael</td>
											<td>11 Halevona Street, 76350 Rehovot,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N015/82</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IL04/00431</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-05-20</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/472,433</td>
									<td>2003-05-22</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/230433-methods-of-increasing-abiotic-stress-tolerance-and-or-biomass-in-plants by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:12:48 GMT -->
</html>
