<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/230408-a-method-and-kit-for-identifying-elite-event-ee-gh1-in-biological-samples by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:11:15 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 230408:A METHOD AND KIT FOR IDENTIFYING ELITE EVENT EE-GH1 IN BIOLOGICAL SAMPLES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD AND KIT FOR IDENTIFYING ELITE EVENT EE-GH1 IN BIOLOGICAL SAMPLES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention pertains to transgenic cotton platns, plant material and seeds, characterized by harboring a specific transformation event, particularly by the presence of a gene encoding a [protein that confers herbicide tolerance, at a specific location in the cotton genome. The cotton plants of the invention combine the herbicide tolerant phenotype with optimal agronomic performance.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Title of the invention<br>
HERBICIDE TOLERANT COTTON PLANTS AND METHODS FOR PRODUCING AND IDENTIFYING SAME<br>
Field of the invention<br>
This invention pertains to transgenic cotton plants, plant material and seeds, characterized by harboring a specific transformation event, particularly by the presence of a gene encoding a protein that confers herbicide tolerance, at a specific location in the cotton genome. The cotton plants of the invention combine the herbicide tolerant phenotype with an agronomic performance, genetic stability and adaptability to different genetic backgrounds equivalent to the non-transformed cotton line in the absence of weed pressure.<br>
All documents cited herein are hereby incorporated herein by reference.<br>
Background of the invention<br>
The phenotypic expression of a transgene in a plant is determined both by the structure of the gene itself and by its location in the plant genome. At the same time the presence of the transgene at different locations in the genome will influence the overall phenotype of the plant in different ways. The agronomically or industrially successful introduction of a commercially interesting trait in a plant by genetic manipulation can be a lengthy procedure dependent on different factors. The actual transformation and regeneration of genetically transformed plants are only the first in a series of selection steps, which include extensive genetic characterization, breeding, and evaluation in field trials.<br>
Cotton fiber is the single most important textile worldwide. About 80 million acres of cotton are harvested annually across the globe. Cotton is the fifth largest crop in the U.S. in terms of acreage production, with OVQT 15 million acres planted in 2000. Primary weed<br><br>
species for cotton are Ipomoea sp. (morning glory), Amaranthus spp. (pigweed), Cyperus spp. (nutsedge), Xanthium spp. (cocklebur) and Sorghum spp. (johnsongrass). Before the introduction of broad-leaf herbicides that could be used on a growing cotton field, growers used directed, post-emergence applications of nonselective herbicides taking care not to contact the growing crop plants. As this requires a difference in height between the weeds and the crop, this is not always possible. Especially for small cotton, » this practice is time-consuming and potentially damaging to the crop.<br>
The bar gene (Thompson et al, 1987, EMBO J 6:2519-2523; Deblock et al 1987, EMBO J. 6:2513-2518) is a gene encoding the enzyme phosphinothricin acetyl transferase (PAT), which, when expressed in a plant, confers resistance to the herbicidal compounds phosphinothricin (also called glufosinate) or bialaphos (see also for example US patents, 5,646,024 and 5,561,236) and salts and optical isomers thereof. Phosphinothricin controls broadleaf weeds including morning glory and has a wide window of application.<br>
Successful genetic transformation of cotton has been obtained by a number of methods including Agrobacterium infection of cotton explants (Firoozabady et al. 1987, Plant Molecular Biology 10:105-116; Umbeck et al. 1987, Bio/Technology 5:263-266 and in WO 00/71733, US 5,004,863, and US 5,159,135), as well as direct gene transfer by microprojectile bombardment of meristematic cotton tissues (Finer and Mc Mullen, 1990, Plant Cell Reports, 5:586-589; McCabe and Martinell, 1993, Bio/Technology 11:596-598, W092/15675, EP0 531 506). Increased transformation efficiency for Agrobacterium transformation has been reported using the methods described by Hansen et al. (1994, Proc. Nat. Acad. Sci. 91:7603-7607) Veluthambi et al. (1989, Journal of Bacteriology 171:3696-3703) and WO 00/71733.<br>
Different methods for regeneration of cotton plants have also been described (WO 89/05344, US 5,244,802, US 5,583,036, WO89/12102, W098/15622, and W097/12512).<br>
However, the foregoing documents fail to teach or suggest the present invention.<br><br>
Reference seed comprising the elite event of the invention has been deposited at the ATCC under accession number PTA-3343. Thus, a preferred embodiment of the invention is the seed comprising elite event EE-GH1 deposited as ATTC accession number PTA-3343, which will grow into a cotton plant resistant to glufosinate. The seed of ATCC deposit number PTA-3343, which is a seed lot consisting of about 50% non- » transgenic kernels and 50% transgenic kernels hemizygous for the transgene, comprising the elite event of the invention, which will grow into glufosinate tolerant plants. The seed can be sown and the growing plants can be treated with PPT or Liberty as described herein to obtain 100% glufosinate tolerant plants, comprising the elite event of the invention. The invention further relates to cells, tissues, progeny, and descendants from a plant comprising the elite event of the invention grown from the seed deposited at the ATCC having accession number PTA-3343. The invention further relates to plants obtainable by propagation of and/or breeding with a cotton plant comprising the elite event of the invention grown from the seed deposited at the ATCC having accession number PTA-3343.<br>
The invention further relates to plants, seeds, cells or tissues comprising a foreign DNA sequence, preferably a herbicide tolerance gene as described herein, integrated into the chromosomal DNA in a region which comprises the plant DNA sequence of SEQ ID NO: 1 and/or SEQ ID NO: 4, more particularly which comprises the DNA sequence of SEQ ID NO: 5, or a sequence which hybridizes under stringent conditions to a sequence that is complementary to a sequence comprising the plant DNA sequence of SEQ ID NO: 1, SEQ ID NO: 4 and/or SEQ ID NO: 5.<br>
The invention further provides a process for producing a transgenic cell of a cotton plant, which comprises inserting a recombinant DNA molecule into a region of the chromosomal DNA of a cotton cell, tissue or callus which comprises the plant DNA sequence of SEQ ID NO: 1 and/or SEQ ID NO: 4, or which comprises a sequence which hybridizes under stringent conditions to a sequence that is complementary to a sequence comprising the plant DNA sequence of SEQ ID NO: 1 and/or SEQ ID NO: 4.<br><br>
Reference seed comprising the elite event of the invention has been deposited at the ATCC under accession number PTA-3343. Thus, a preferred embodiment of the invention is the seed comprising elite event EE-GH1 deposited as ATTC accession number PTA-3343, which will grow into a cotton plant resistant to glufosinate. The seed of ATCC deposit number PTA-3343, which is a seed lot consisting of about 50% non- ♦ transgenic kernels and 50% transgenic kernels hemizygous for the transgene, comprising the elite event of the invention, which will grow into glufosinate tolerant plants. The seed can be sown and the growing plants can be treated with PPT or Liberty as described herein to obtain 100% glufosinate tolerant plants, comprising the elite event of the invention. The invention further relates to cells, tissues, progeny, and descendants from a plant comprising the elite event of the invention grown from the seed deposited at the ATCC having accession number PTA-3343. The invention further relates to plants obtainable by propagation of and/or breeding with a cotton plant comprising the elite event of the invention grown from the seed deposited at the ATCC having accession number PTA-3343.<br>
The invention further relates to plants, seeds, cells or tissues comprising a foreign DNA sequence, preferably a herbicide tolerance gene as described herein, integrated into the chromosomal DNA in a region which comprises the plant DNA sequence of SEQ ID NO: 1 and/or SEQ ID NO: 4, more particularly which comprises the DNA sequence of SEQ ID NO: 5, or a sequence which hybridizes under stringent conditions to a sequence that is complementary to a sequence comprising the plant DNA sequence of SEQ ID NO: 1, SEQ ID NO: 4 and/or SEQ ID NO: 5.<br>
The invention further provides a process for producing a transgenic cell of a cotton plant, which comprises inserting a recombinant DNA molecule into a region of the chromosomal DNA of a cotton cell, tissue or callus which comprises the plant DNA sequence of SEQ ID NO: 1 and/or SEQ ID NO: 4, or which comprises a sequence which hybridizes under stringent conditions to a sequence that is complementary to a sequence comprising the plant DNA sequence of SEQ ID NO: 1 and/or SEQ ID NO: 4.<br><br>
The invention further relates to a method for identifying a transgenic plant, or cells or tissues thereof, comprising elite event EE-GH1 which method is based on identifying the presence of characterizing DNA sequences or amino acids encoded by such DNA sequences in the transgenic plant, cells or tissues. According to a preferred embodiment of the invention, such characterizing DNA sequences are sequences of 15bp, preferably ♦ 20bp, most preferably 30bp or more which comprise the insertion site of the event, i.e. both a part foreign DNA and a part of the cotton genome (either the 5' or 3' flanking region) contiguous therewith, allowing specific identification of the elite event.<br>
According to another aspect of the invention, DNA sequences are disclosed comprising the insertion site of the event and sufficient length of polynucleotides of both the cotton genomic DNA and the foreign DNA (transgene), so as to be useful as primer or probe for the detection of EE-GH1. Preferably, such sequences comprise at least 9 nucleotides of the cotton genomic DNA and a similar number of nucleotides of the foreign DNA (transgene) of EE-GH1 therewith at each side of the insertion site respectively. Most preferably, such DNA sequences comprise at least 9 nucleotides of the cotton genomic DNA and a similar number of nucleotides of the foreign DNA contiguous with the insertion site in SEQ ID NO: 1 or SEQ ID NO: 4.<br>
According to another preferred aspect of the invention, the method for identifying a transgenic plant, or cells or tissues thereof, comprising elite event EE-GH1, comprises amplifying a sequence of a nucleic acid present in biological samples, using a polymerase chain reaction, with at least two primers, one of which recognizes the plant DNA in the 5' or 3' flanking region of EE-GH1, the other which recognizes a sequence within the foreign DNA. Preferably, the genomic DNA is analyzed using primers which recognize a sequence within the plant 5' flanking region of EE-GH1, most preferably within the plant DNA sequence in SEQ ID NO: 1, and a sequence within the foreign DNA, respectively. Especially preferably, the genomic DNA is analyzed according to the PCR identification protocol described herein whereby the primer recognizing a sequence within the 5' flanking region comprises the nucleotide sequence of SEQ ID NO: 2. Particularly, the<br><br>
primer recognizing a sequence within the 5' flanking region comprises the nucleotide sequence of SEQ ID NO: 2 and the primer recognizing a sequence within the foreign DNA comprises the nucleotide sequence of SEQ ID NO: 3, so that the amplified fragment is a fragment preferably of between 250 and 290 bp, more preferably of between 260 and 270 bp, most preferably of about 269 bp.<br>
Accordingly, the present invention relates to the transgenic plant, cells or tissues thereof which can be identified according the above-described identification methods for EE-GH1.<br>
The present invention relates to methods for identifying elite event EE-GH1 in biological samples, which methods are based on primers or probes that specifically recognize the 5' and/or 3' flanking sequence of EE-GH1. In a preferred embodiment of the invention these methods are based on primers or probes which recognize a sequence within SEQ ID NO: 1 and/or SEQ ID NO: 4, more particularly primers or probes comprising the sequence of SEQ ID NO: 2. According to another embodiment of the invention such methods are based on primers or probes which recognize a characterizing sequence of EE-GH1, which is a DNA sequence comprising the insertion site of the event, including sufficient nucleotides of the 5' or 3' flanking DNA and the foreign DNA contiguous therewith so as to be diagnostic for the event. Most particularly, such methods are based on sequences comprising sufficient nucleotides of the 5* or 3' flanking DNA and the foreign DNA contiguous therewith of SEQ ID NO: 1 or SEQ ID NO: 4. Such methods are known in the art and include, but are not limited to, the method described by Lyamichev et al. (1999, Nature Biotechn 17:292-296).<br>
The present invention further relates to the specific flanking sequences of EE-GH1 described herein, which can be used to develop specific identification methods for EE-GH1 in biological samples. More particularly, the invention relates to the 5' and or 3' flanking regions of EE-GH1, which can be used for the development of specific primers and probes as well as to the specific primers and probes developed from the 5' and/or 3* flanking sequences of EE-GH1 and the insertion site adjacent thereto. The invention<br><br>
further relates to identification methods for the presence of EE-GH1 in biological samples based on the use of such specific primers or probes.<br>
The invention thus also relates to a kit for identifying elite event EE-GH1 in biological samples, the kit comprising at least one primer or probe which specifically recognizes the 5' or 3' flanking region of EE-GH1.<br>
Preferably the kit of the invention comprises, in addition to a primer which specifically recognizes the 5' or 3' flanking region of EE-GH1, a second primer which specifically recognizes a sequence within the foreign DNA of EE-GH1, for use in a PCR identification protocol. Preferably, the kit of the invention comprises two (or more) specific primers, one of which recognizes a sequence within the 5' flanking region of EE-GH1, most preferably a sequence within the plant DNA region of SEQ ID NO: 1, and an other which recognizes a sequence within the foreign DNA. Especially preferably, the primer recognizing the plant DNA sequence within 5' flanking region comprises the nucleotide sequence of SEQ ID NO: 2. Particularly, the primer recognizing the plant DNA sequence within 5' flanking region comprises the nucleotide sequence of SEQ ID NO: 2 and the primer recognizing the foreign DNA comprises the nucleotide sequence of SEQ ID NO: 1 described herein.<br>
According to a further embodiment, the invention relates to a kit for identifying elite event EE-GH1 in biological samples, which kit comprises at least one specific primer or probe having a sequence which corresponds (or is complementary to) a sequence that hybridizes under stringent conditions to a specific or characterizing region of EE-GH1. Preferably the sequence of the probe corresponds to a specific region comprising part of the foreign DNA and part of the 5' or 3' flanking region (plant DNA) of EE-GH1 contiguous therewith. Most preferably the specific probe comprises (or is complementary to) a sequence thai hybridizes under stringent conditions to a sequence of 15 to 30 nucleotides spanning the insertion site at the 5' or 3' region of the foreign DNA within SEQ ID NO: 1 or SEQ ID NO: 4. Especially preferably the specific probe comprises (or is complementary to) a sequence that hybridizes under stringent conditions to a sequence<br><br>
of 9 nucleotides of plant DNA and a similar number of nucleotides of the foreign DNA contiguous with the insertion site, within the sequences of SEQ ID NO: I or SEQ ID NO:4. According to a particular embodiment, such a primer comprises a sequence that is identical to a sequence of 9 nucleotides of plant DNA and a similar number of nucleotides of the foreign DNA contiguous with the insertion site, within the sequences of SEQ ID NO: 1 or SEQ ID NO: 4.<br>
The methods and kits encompassed by the present invention can be used for different purposes such as, but not limited to the following: to identify EE-GH1 in plants, plant material or in products such as, but not limited to food or feed products (fresh or processed) comprising or derived from plant material; additionally or alternatively, the methods and kits of the present invention can be used to identify transgenic plant material for purposes of segregation between transgenic and non-transgenic material; additionally or alternatively, the methods and kits of the present invention can be used to determine the quality (i.e. percentage pure material) of plant material comprising EE-GH1.<br>
The present invention further relates to a method for tracking plants comprising elite event EE-GH1 in their genome upon introduction into different cultivars.<br>
It will be understood that particular embodiments of the invention are described by the dependent claims cited herein.<br>
Brief description of the Figure<br>
The following detailed description, given by way of example, but not intended to limit the invention to specific embodiments described, may be understood in conjunction with the accompanying Figure, incorporated herein by reference, in which:<br><br>
Fig. 1. PCR analysis of other events and elite event EE-GH1 using the EE-GH1 PCR identification protocol. Loading sequence of the gel: lane 1, molecular weight marker (lOObp ladder), lane 2, DNA sample from a cotton plant comprising the transgenic event EE-GH1, lane 3, DNA samples from a cotton plant comprising another transgenic event, lane 4, DNA from wild-type cotton, lane 5, wild-type + 1 copy of the pGSV71-BamHI digest (positive control), lane 6, negative control ♦ (no template), lane 8, molecular weight marker (lOObp ladder).<br>
Detailed Description of Preferred Embodiments<br>
The term "gene" as used herein refers to any DNA sequence comprising several operably linked DNA fragments such as a promoter region, a 5' untranslated region (the 5'UTR), a coding region (which may or may not code for a protein), and an untranslated 3' region (3'UTR) comprising a polyadenylation site. Typically in plant cells, the 5'UTR, the coding region and the 3'UTR are transcribed into an RNA of which, in the case of a protein encoding gene, the coding region is translated into a protein. A gene may include additional DNA fragments such as, for example, introns. As used herein, a genetic locus is the position of a given gene in the genome of a plant.<br>
The term "chimeric" when referring to a gene or DNA sequence is used to refer to the fact that the gene or DNA sequence comprises at least two functionally relevant DNA fragments (such as promoter, 5'UTR, coding region, 3'UTR, intron) that are not naturally associated with each other and/or originate, for example, from different sources. "Foreign" referring to a gene or DNA sequence with respect to a plant species is used to indicate that the gene or DNA sequence is not naturally found in that plant species, or is not naturally found in that genetic locus in that plant species. The term "foreign DNA" will be used herein to refer to a DNA sequence as it has incorporated into the genome of a plant as a result of transformation. The "transforming DNA" as used herein refers to a recombinant DNA molecule used for transformation. The transforming DNA usually comprises at least one "gene of interest" (e.g. a chimeric gene) which is capable of<br><br>
conferring one or more specific characteristics to the transformed plant. The term "recombinant DNA molecule" is used to exemplify and thus can include an isolated nucleic acid molecule which can be DNA and which can be obtained through recombinant or other procedures.<br>
As used herein the term "transgene" refers to a gene of interest (or the whole of the * foreign DNA) as incorporated in the genome of a plant. A "transgenic plant" refers to a plant comprising at least one transgene in the genome of all of its cells.<br>
The foreign DNA present in the plants of the present invention will preferably comprise a herbicide tolerance gene, more specifically a 35S-bar gene as the gene of interest.<br>
A "herbicide tolerance" gene as used herein refers to a gene that renders the plant tolerant to a herbicide. An example of a herbicide tolerance gene is gene comprising a sequence encoding the enzyme phosphinothricin acetyl transferase, which detoxifies phosphinothricin, under the control of a constitutive promoter. More specifically, in the elite event of the present invention the herbicide tolerance gene comprises the coding sequence of the bialaphos resistance gene (bar) of Streptomyces hygroscopicus (Thompson et al (1987) EMBO J 6: 2519-2523) under control of the 35S promoter from Cauliflower Mosaic Virus (Odell et al., (1985), Nature 313: 810-812), also referred to as "35S-/&gt;ar" herein. The expression of the 25S-bar gene confers tolerance to herbicidal compounds phosphinothricin or bialaphos or glufosinate, or more generally, glutamine synthase inhibitors, or salts or optical isomers thereof which will generally be referred to as "glufosinate tolerance" herein.<br>
By hybridizing under "stringent conditions" is meant the conventional hybridizing conditions as described by Sambrook et al. (1989) (Molecular Cloning: A Laboratory Manual, Second Edition, Cold Spring Harbour Laboratory Press, NY) which for instance can comprise the following steps: 1) immobilizing plant genomic DNA fragments on a filter, 2) prehybridizing the filter for 1 to 2 hours at 42°C in 50% formamide, 5 X SSPE, 2 X Denhardt's reagent and 0.1% SDS, or for 1 to 2 hours at 68°C in 6 X SSC, 2 X Denhardt's reagent and 0.1% SDS, 3) adding the hybridization probe which has been<br><br>
labeled, 4) incubating for 16 to 24 hours, 5) washing the filter for 20 min. at room temperature in IX SSC, 0.1 %SDS, 6) washing the filter three times for 20 min. each at 68°C in 0.2 X SSC, 0.1 %SDS, and 7) exposing the filter for 24 to 48 hours to X-ray film at -70°C with an intensifying screen.<br>
The incorporation of a recombinant DNA molecule in the plant genome typically results • from transformation of a cell, tissue or callus (or from another genetic manipulation). The particular site of incorporation is either random or is at a predetermined location (if a process of targeted integration is used).<br>
The DNA introduced into the plant genome as a result of transformation of a plant cell or tissue with a recombinant DNA or "transforming DNA" is hereinafter referred to as "foreign DNA" comprising one or more "transgenes". Thus, foreign DNA may comprise both recombinant DNA as well as newly introduced, rearranged DNA of the plant. However, the term "plant DNA" in the context of the present invention will refer to DNA of the plant which is generally found in the same genetic locus in the corresponding wild-type plant.<br>
The foreign DNA can be characterized by the location and the configuration at the site of incorporation of the recombinant DNA molecule in the plant genome. The site in the plant genome where a recombinant DNA has been inserted is also referred to as the "insertion site" or "target site". Insertion of the recombinant DNA into the plant genome can be associated with a deletion of plant DNA, referred to as "target site deletion". A "flanking region" or "flanking sequence" as used herein refers to a sequence of at least 20 bp, preferably at least 50 bp, and up to 5000 bp of the plant genome which is located either immediately upstream of and contiguous with or immediately downstream of and contiguous with the foreign DNA. Transformation procedures leading to random integration of the foreign DNA will result in transformants with different flanking regions, which are characteristic and unique for each transformant. When the recombinant DNA present in a transgenic plant is introduced into a different plant through traditional crossing, its insertion site in the plant genome, or its flanking regions will generally not be changed (apart from occasional changes due to mutations or cross-<br><br>
over and transposons). An "insertion region" as used herein refers to the region corresponding to the region of at least 40 bp, preferably at least 100 bp, and up to more than 10000 bp, encompassed by the sequence which comprises the upstream and/or the downstream flanking region of a foreign DNA in the (untransformed) plant genome (and including the insertion site and possible target site deletion). Taking into consideration minor differences due to mutations within a species, an insertion region will retain at least • 85%, preferably 90%, more preferably 95%, and most preferably 100% sequence identity with the sequence comprising the upstream and downstream flanking regions of the foreign DNA in a given plant of that species.<br>
Expression of a gene of interest refers to the fact that the gene confers on the plant one or more phenotypic traits (e.g. herbicide tolerance) that were intended to be conferred by the introduction of the recombinant DNA molecule - the transforming DNA - used during transformation (on the basis of the structure and function of part or all of the gene(s) of interest).<br>
An "event" is defined as a (artificial) genetic locus that, as a result of genetic engineering, carries a foreign DNA comprising at least one copy of the gene(s) of interest (also referred to as a transformation event). An event is characterized phenotypically by the expression of the transgenes. At the genetic level, an event is part of the genetic makeup of a plant. At the molecular level, an event is characterized by the restriction map (e.g. as determined by Southern blotting) and/or by the upstream and/or downstream flanking sequences of the foreign DNA, and/or the molecular configuration of the foreign DNA comprising the transgenes. Usually when transforming a plant cell, tissue or callus with a transforming DNA, a multitude of events are generated, each of which is unique.<br>
An "elite event", as used herein, is an event which is selected from a group of events, obtained by transformation with the same transforming DNA or by back-crossing with plants obtained by such transformation, based on the phenotypic expression and stability of the transgenes and the absence of negative impact on the agronomic characteristics of the plant comprising it (i.e., selected transformation event). Thus the criteria for elite<br><br>
event selection are one or more, preferably two or more, advantageously all of the following:<br>
a)	That the presence of the foreign DNA in the plant does not compromise other desired characteristics of the plant, such as those relating to agronomic performance or commercial value;<br>
b)	That the event is characterized by a well defined molecular configuration which is » stably inherited and for which appropriate diagnostic tools for identity control can be developed;<br>
c)	That the gene(s) of interest show(s) an appropriate and stable spatial and temporal phenotypic expression in homozygous condition of the event, at a commercially acceptable level in a range of environmental conditions in which the plants carrying the event are likely to be exposed in normal agronomic use.<br>
It is preferred that the foreign DNA is associated with a position in the plant genome that allows introgression into desired commercial genetic backgrounds.<br>
The status of an event as an elite event is confirmed by introgression of the elite event in different relevant genetic backgrounds and observing compliance with one, two or all of the criteria e.g. a), b) and c) above.<br>
An "elite event" thus refers to a genetic locus comprising a foreign DNA, which answers to the above-described criteria. A plant, plant material or progeny such as seeds can comprise one or more elite events in its genome. Thus, when referring to a plant, seed cell or tissue comprising elite event EE-GH1 in its genome, a plant, seed cell or tissue is intended which comprises the foreign DNA described herein (comprising the 35S-bar gene) integrated in its genome at the integration site described herein.<br>
The tools developed to identify an elite event or the plant or plant material comprising an elite event, or products which comprise plant material comprising the elite event are based on the specific genomic characteristics of the elite event, such as, a specific restriction map of the genomic region comprising the foreign DNA, molecular markers or the sequence of the flanking region(s) of the foreign DNA.<br><br>
Once one or both of the flanking regions of the foreign DNA have been sequenced, primers and probes can be developed which specifically recognize this (these) sequence(s) in the nucleic acid (DNA or RNA) of a sample by way of a molecular biological technique. For instance a PCR method can be developed to identify the elite event in biological samples (such as samples of plants, plant material or products * comprising plant material). Such a PCR is based on at least two specific "primers" one recognizing a sequence within the 5' or 3' flanking region of the elite event and the other recognizing a sequence within the foreign DNA. The primers preferably have a sequence of between 15 and 35 nucleotides which under optimized PCR conditions "specifically recognize" a sequence within the 5' or 3' flanking region of the elite event and the foreign DNA of the elite event respectively, so that a specific fragment ("integration fragment") is amplified from a nucleic acid sample comprising the elite event. This means that only the targeted integration fragment, and no other sequence (of that size) in the plant genome or foreign DNA, is amplified under optimized PCR conditions. Preferably, the integration fragment has a length of between 50 and 500 nucleotides, most preferably of between 100 and 350 nucleotides. Preferably the specific primers have a sequence which is between 80 and 100% identical to a sequence within the 5' or 3' flanking region of the elite event and the foreign DNA of the elite event, respectively, provided the mismatches still allow specific identification of the elite event with these primers under optimized PCR conditions. The range of allowable mismatches however, can easily be determined experimentally and are known to a person skilled in the art. As the sequence of the primers and their recognized sequence in the genome are unique for the elite event, amplification of the integration fragment will occur only in biological samples comprising (the nucleic acid of) the elite event. Preferably when performing a PCR to identify the presence of EE-GH1 in unknown samples, a control is included of a set of primers with which a fragment within a "housekeeping gene" of the plant species of the event can be amplified. Housekeeping genes are genes that are expressed in most cell types and which are concerned with basic metabolic activities common to all cells. Preferably, the fragment amplified from the housekeeping gene is a fragment which is<br><br>
larger than the amplified integration fragment. Depending on the samples to be analyzed, other controls can be included.<br>
Standard PCR protocols are described in the art, such as in 'PCR Applications Manual" (Roche Molecular Biochemicals, 2nd Edition, 1999). The optimal conditions for the PCR, including the sequence of the specific primers, is specified in a "PCR identification * protocol" for each elite event. It is however understood that a number of parameters in the PCR identification protocol may need to be adjusted to specific laboratory conditions, and may be modified slightly to obtain similar results. For instance, use of a different method for preparation of DNA may require adjustment of, for instance, the amount of primers, polymerase and annealing conditions used. Similarly, the selection of other primers may dictate other optimal conditions for the PCR identification protocol. These adjustments will however be apparent to a person skilled in the art, and are furthermore detailed in current PCR application manuals such as the one cited above.<br>
Alternatively, specific primers can be used to amplify an integration fragment that can be used as a "specific probe" for identifying EE-GH1 in biological samples. Contacting nucleic acid of a biological sample, with the probe, under conditions which allow hybridization of the probe with its corresponding fragment in the nucleic acid, results in the formation of a nucleic acid/probe hybrid. The formation of this hybrid can be detected (e.g. labeling of the nucleic acid or probe), whereby the formation of this hybrid indicates the presence of EE-GH1. Such identification methods based on hybridization with a specific probe (either on a solid phase carrier or in solution) have been described in the art. The specific probe is preferably a sequence which, under optimized conditions, hybridizes specifically to a region comprising part of the 5' or 3' flanking region of the elite event and also comprising part of the foreign DNA contiguous therewith (hereinafter also referred to as a "specific region" of the event). Depending on the method used, such a specific probe can comprise a sequence of between 15 and 30 bp or of between 50 and 500 bp, preferably of 100 to 350 bp which hybridizes under stringent conditions to the nucleotide sequence (or the complement of such sequence) of a specific region. Preferably, the specific probe will comprise a sequence of about  15 to about 100<br><br>
contiguous nucleotides identical (or complementary) to a specific region of the elite event. Most preferably such,a probe comprises at least 9 nucleotides identical to (or complementary to the 3' or 5' flanking region and a similar number of nucleotides in the foreign DNA contiguous therewith. Most preferably, such a probe comprises at least 9 nucleotides identical to (or complementary to the 3' or 5' flanking region and a similar number of nucleotides in the foreign DNA contiguous therewith of SEQ ID NO: 1 or* SEQ fD NO: 4.<br>
A "restriction map" as used herein refers to a set of Southern blot patterns obtained after cleaving plant genomic DNA (and/or the foreign DNA comprised therein) with a particular restriction enzyme, or set of restriction enzymes and hybridization with a probe sharing sequence similarity with the foreign DNA under standard stringency conditions. Standard stringency conditions as used herein refers to the conditions for hybridization described herein or to the conventional hybridizing conditions as described by Sambrook et al (1989) (Molecular Cloning: A Laboratory Manual, Second Edition, Cold Spring Harbour Laboratory Press, NY) which for instance can comprise the following steps: 1) immobilizing plant genomic DNA fragments on a filter, 2) prehybridizing the filter for 1 to 2 hours at 42°C in 50% formamide, 5 X SSPE, 2 X Denhardt's reagent and 0.1% SDS, or for 1 to 2 hours at 68°C in 6 X SSC, 2 X Denhardt's reagent and 0.1% SDS, 3) adding the hybridization probe which has been labeled, 4) incubating for 16 to 24 hours, 5) washing the filter for 20 min. at room temperature in IX SSC, 0.1 %SDS, 6) washing the filter three times for 20 min. each at 68°C in 0.2 X SSC, 0.1 %SDS, and 7) exposing the filter for 24 to 48 hours to X-ray film at -70°C with an intensifying screen.<br>
Due to the (endogenous) restriction sites present in a plant genome prior to incorporation of the foreign DNA, insertion of a foreign DNA will alter the specific restriction map of that genome. Thus, a particular transformant or progeny derived thereof can be identified by one or more specific restriction patterns.<br>
Alternatively, plants or plant material comprising an elite event can be identified by testing according to a PCR identification protocol. This is a PCR which specifically<br><br>
recognizes the elite event. Essentially, a set of PCR primers is developed which recognizes a) a sequence within the 3' or 5' flanking sequence of the elite event and b) a sequence within the foreign DNA, which primers amplify a fragment (integration fragment) preferably of between 100 and 300 nucleotides. Preferably, a control is included of a set of primers which amplifies a fragment within a housekeeping gene of the plant species (preferably a fragment which is larger than the amplified integration * fragment). The optimal conditions for the PCR, including the sequence of the specific primers is specified in a PCR identification protocol.<br>
Other methods for identifying plants, plant material or products comprising plant material comprising elite event EE-GH1 are also envisaged. These methods include all methods based on the detection of the foreign DNA sequence and flanking sequence(s) of the elite event with a specific probe. More particularly, chip-based technologies, such as those described by Hacia et al 1996 (Nat Genet 14(4):441-447) and Shoemaker et al 1996 (Nat Genet 14(4):450-456) are envisaged. These methods allow segregation of target molecules as high-density arrays by using fixed probe arrays or by tagging of the genes with oligonucleotides, after which they can be screened by hybridization. Alternatively, third wave detection methods, such as those described by Lyamichev et al (1999, above) are envisaged.<br>
Identification of the protein(s) encoded by the foreign DNA of the elite event can be done by classical protein detection methods described in the art, such as those based on chromatographic or electromagnetic properties of the protein or the detection by specific monoclonal antibodies (as described in "Guide to protein purification, Murray P. Deutscher editor).<br>
A "kit" as used herein refers to a set of reagents for the purpose of performing the method of the invention, more particularly, the identification of the elite event EE-GH1 in biological samples. More particularly, a preferred embodiment of the kit of the invention comprises at least one or two specific primers, as described above. Optionally, the kit can further comprise any other reagent described herein in the PCR identification protocol.<br><br>
Alternatively, according to another embodiment of this invention, the kit can comprise a specific probe, as described above, which specifically hybridizes with nucleic acid of biological samples to identify the presence of EE-GH1 therein. Optionally, the kit can further comprise any other reagent (such as but not limited to hybridizing buffer, label) for identification of EE-GH1 in biological samples, using the specific probe.<br>
The kit of the invention can be used, and its components can be specifically adjusted, for purposes of quality control (e.g., purity of seed lots), detection of the elite event in plant material or material comprising or derived from plant material, such as but not limited to food or feed products.<br>
The present invention relates to the development of an elite event in cotton, EE-GH1, to the plants comprising this event, the progeny obtained from these plants and to the plant cells, or plant material derived from this event. Plants comprising elite event EE-GH1 were obtained through transformation with pGSWl as described in example 1.<br>
Cotton plants or plant material comprising EE-GH1 can be identified according to the PCR identification protocol described for EE-GH1 in Example 4 herein. Briefly, cotton genomic DNA is amplified by PCR using a primer which specifically recognizes a sequence within the 5' or 3' flanking sequence of EE-GH1, particularly the primer with the sequence of SEQ ID NO: 2, and a primer which recognizes a sequence in the foreign DNA, particularly the primer with the sequence of SEQ ID NO: 3. Endogenous cotton DNA primers are used as controls. If the plant material yields a fragment of between 250 and 290 bp, preferably of about 269 bp, the cotton plant is determined to harbor elite event EE-GH1.<br>
Plants harboring EE-GH1 are characterized by their glufosinate tolerance, which in the context of the present invention includes that plants are tolerant to the herbicide Liberty . Tolerance to Liberty can be tested in different ways. The leaf paint method as described herein, is most useful when you wish to identify both resistant and sensitive plants, but do not want to kill the sensitive ones. Alternatively, tolerance can be tested by<br><br>
Liberty spray application. Spray treatments should be made between the leaf stages V3 and V4 for best results. Tolerant plants are characterized by the fact that spraying of the plants with at least 200 grams active ingredient/hectare (g.a.i./ha), preferably 400 g.a.i./ha, and possibly up to 1600 g.a.i./ha (4X the normal field rate), does not kill the plants. A broadcast application should be applied at a rate of 28-34 oz Liberty™. It is best to apply at a volume of 20 gallons of water per acre using a flat fan type nozzle while * being careful not to direct spray applications directly into the whorl of the plants to avoid surfactant burn on the leaves. The herbicide effect should appear within 48 hours and be clearly visible within 5-7 days.<br>
Plants harboring EE-GH1 can further be characterized by the presence in their cells of phosphinothricin acetyl transferase as determined by a PAT assay (De Block et al, 1987, supra).<br>
Plants harboring EE-GH1 can, for example, be obtained from seeds deposited at the ATCC under accession number PTA-3343, which contain 50% kernels that are hemizigous for the elite event. Such plants can be further propagated to introduce the elite event of the invention into other cultivars of the same plant species. Selected seeds obtained from these plants contain the elite event stably incorporated into their genome. The invention further relates to plants derived from the ATCC accession number PTA-334, comprising EE-GH1. The term "derived from" herein indicates that the plants are related, i.e. they are both progeny (direct or of two or more generations) of the same transformant by crossing.<br>
Plants harboring EE-GH1 are also characterized by having agronomical characteristics that are comparable to commercially available varieties of cotton in the US, in the<br>
'V\A<br>
absence of weed pressure and use of Liberty for weed control. It has been observed that the presence of a foreign DNA in the insertion region of the cotton plant genome described herein, confers particularly interesting phenotypic and molecular characteristics to the plants comprising this event. More specifically, the presence of the foreign DNA in this particular region in the genome of these plants, results in plants which display a stable phenotypic expression of the gene of interest without significantly compromising<br><br>
any aspect of desired agronomic performance of the plants. Thus, the insertion region, corresponding to a sequence comprising the plant DNA of SEQ ID NO: 1 and/or SEQ ID NO: 4, more particularly a sequence corresponding to SEQ ID NO: 5, most particularly the insertion site of EE-GH1 therein, is shown to be particularly suited for the introduction of a gene(s) of interest. More particularly, the insertion region of EE-GH1 (corresponding to a DNA sequence of at least 40 bp in the cotton genome within SEQ ID * NO: 5), or a sequence of at least 40 bp which hybridizes under stringent conditions to the complement of the sequence of SEQ ID NO: 5, is particularly suited for the introduction of foreign DNA comprising a herbicide tolerance gene, ensuring expression of each of these genes in the plant without compromising agronomic performance.<br>
A recombinant DNA molecule can be specifically inserted in an insertion region by targeted insertion methods. Such methods are well known to those skilled in the art and comprise, for example, homologous recombination using a recombinase such as, but not limited to the FLP recombinase from Saccharomyces cerevisiae (published PCTP application WO 99/25821), the CRE recombinase from Escherichia colt phage PI (published PCT application WO 99/25840), the recombinase from pSRl of Saccharomyces rouxii (Araki et al. 1985, J Mol Biol 182:191-203), the Gin/gix system of phage Mu (Maeser and Kahlmann, 1991, Mol Gen Genetics 230:170-176) or the lambda phage recombination system (such as described in US Patent 4,673,640).<br>
As used herein, "sequence identity" with regard to nucleotide sequences (DNA or RNA), refers to the number of positions with identical nucleotides divided by the number of nucleotides in the shorter of the two sequences. The alignment of the two nucleotide sequences is performed by the Wilbur and Lipmann algorithm (Wilbur and Lipmann, 1983, Proc Nat Acad Sci USA 80:726) using a window-size of 20 nucleotides, a word length of 4 nucleotides, and a gap penalty of 4. Computer-assisted analysis and interpretation of sequence data, including sequence alignment as described above, can, e.g., be conveniently performed using the programs of the Wisconsin Package (from the Genetics Computer Group, Inc). Sequences are indicated as "essentially similar" when such sequences have a sequence identity of at least about 75%, particularly at least about<br><br>
80%, more particularly at least about 85%, quite particularly about 90%, especially about 95%, more especially about 100%, quite especially are identical. It is clear that when RNA sequences are said to be essentially similar or have a certain degree of sequence identity with DNA sequences, thymine (T) in the DNA sequence is considered equal to uracil (U) in the RNA sequence. "Complementary to" as used herein refers to the complementarity between the A and T (U), and G and C nucleotides in nucleotide * sequences.<br>
As used in herein, a "biological sample" is a sample of a plant, plant material or products comprising plant material. The term "plant" is intended to encompass cotton (such as but not limited to Gossypium hirsutum) plant tissues, at any stage of maturity, as well as any cells, tissues, or organs taken from or derived from any such plant, including without limitation, any seeds, leaves, stems, flowers, roots, single cells, gametes, cell cultures, tissue cultures or protoplasts. "Plant material", as used herein refers to material which is obtained or derived from a plant. Products comprising plant material relate to food, feed or other products which are produced using plant material or can be contaminated by plant material. It is understood that, in the context of the present invention, such biological samples are preferably tested for the presence of nucleic acids specific for EE-GH1, implying the presence of nucleic acids in the samples. Thus the methods referred to herein for identifying elite event EE-GH1 in biological samples, preferably relate to the identification in biological samples of nucleic acids which comprise the elite event.<br>
As used herein "comprising" is to be interpreted as specifying the presence of the stated features, integers, steps or components as referred to, but does not preclude the presence or addition of one or more features, integers, steps or components, or groups thereof. Thus, e.g., a nucleic acid or protein comprising a sequence of nucleotides or amino acids, may comprise more nucleotides or amino acids than the actually cited ones, i.e., be embedded in a larger nucleic acid or protein. A chimeric gene comprising a DNA sequence which is functionally or structurally defined, may comprise additional DNA sequences, etc.<br><br>
The following examples describe the development and characteristics of cotton plants harboring the elite events EE-GH1  as well as the development of tools  for the dentification of elite event EE-GH1 in biological samples.<br>
Unless otherwise stated, all recombinant DNA techniques arc carried out according to standard protocols as described in Sambrook et al. (1989) Molecular Cloning: A * Laboratory Manual, Second Edition, Cold Spring Harbour Laboratory Press, NY and in Volumes 1 and 2 of Ausubel et al. (1994) Current Protocols in Molecular Biology, Current Protocols, USA. Standard materials and methods for plant molecular work are described in Plant Molecular Biology Labfax (1993) by R.D.D. Croy published by BIOS Scientific Publications Ltd (UK) and Blackwell Scientific Publications, UK.<br>
In the description and examples, reference is made to the following sequences:<br>
SEQ ID NO: 1:	sequence comprising the 5' flanking region<br>
SEQ ID NO: 2:	primer GHI06<br>
SEQ ID NO: 3:	primer GHI05<br>
SEQ ID NO: 4:	sequence comprising the 3' flanking region<br>
SEQ ID NO: 5:	insertion region<br>
SEQ ID NO: 6:	plasmid pGSV71<br>
SEQ ID NO: 7:	plasmid pRVA44<br>
SEQ ID NO: 8	primer MDB327<br>
SEQ ID NO: 9:	primer MLDO15<br>
SEQ ID NO: 10:	primer MLD016<br>
SEQ ID NO: 11:	primer MDB612<br>
SEQ ID NO: 12:	primer MDB053<br>
SEQIDNO:13:	primerMDB356<br>
SEQ ID NO: 14:	pnmer DPA017<br>
SEQIDNO:15:	primerMLD019<br>
SEQ ID NO: 16:	sequence comprising target site deletion<br>
SEQTDNO-.17:	pnmerGHIOl<br>
SEQ ID NO: 18:	primer GHI02<br><br>
EXAMPLES Example 1:     Transformation of cotton with a herbicide tolerance gene<br>
1.1.	Construction of the chimeric DNA comprising the bar gene under the control of<br>
a constitutive promoter<br>
A plasmid pGSV71 was constructed following standard procedures. The sequence of the genetic elemennts of plasmid pGSV71 is given in table 1 (SEQ ID NO: 6):<br>
Table 1. Nucleotide positions of the genetic elements in pGSV71<br>
1.2.	Transformation of Gossypium hirsutum<br>
Cotton tissue from Coker312 plants was transformed with pGSV71 using Agrobacterium transformation (US 5,986,181) and regenerated to plants on appropriate media.<br><br>
The small plantlets initiated on the selective regeneration media were transferred to new medium for germination (all medium is hormone-free). Plantlets were then transferred to the growth chambers or to the greenhouses.<br>
Selection was done on phosphinothricin (PPT) at all stages except plantlet regeneration, which was done in the absence of PPT to accelerate growth. This resulted in a set of * primary transformants (plants of generation TO).<br>
Example 2:     Development of events<br>
2.1. Development of lines carrying the event trait<br>
TO shoots were transferred to greenhouse soil and plants were screened for glufosinate tolerance and for the presence of the PAT enzyme with a PAT ELISA (Steffens Biotechnische Analysen GmbH, Ebringen, Germany).<br>
Tl to T3 plants were grown in the greenhouse and tested for Liberty1 M tolerance at a 2x rate (by spraying 56 oz/ha). Positive plants were tested for expression of the bar gene using the Pat assay as described by Deblock et al. 1987 (EMBO J. 6:2513-2518).<br>
Presence of the foreign DNA and copy number was checked by Southern blot analysis. Total genomic DNA was isolated from Ig of leaf tissue according to the CETAB method of Doyle et al. (1987, Phytochem. Bull. 19:11) and digested with EcoRI restriction enzyme.<br>
Probes such as the following were used for Southern analysis : "bar' probe: 474 bp Kpnl-Bgll digest of plasmid pDEl 10 (WO 92/09696) "35S" probe: 892 bp NcoI-MunI digest of plasmid pRVA44 (SEQ ID NO: 7) T2 Plants were also evaluated for general phenotypic characteristics compared to the non-transgenic isogenic lines. In later generations, the lines for which no negative penalties on phenotype or agronomic performance was observed for the presence of the transgene<br><br>
either in hemizygous or in homozygous condition, as compared to wild-types were selected.<br>
T4 material was grown in the field and tested under field conditions for Liberty™ • tolerance according to different schedules.<br>
In later generations, plants were compared to commercial varieties for yield, fiber quality and plant mapping data. Agronomic characteristics, such as plant height, height to node, boll retention, stand, vigor, fiber length, fiber strength and lint yield were evaluated.<br>
It was determined that one event performed equally or better than the comparable checks and that for this event yield was dependent on background rather than on presence of the transgene.<br>
2.2.	Selection of an elite event<br>
This selection procedure, yielded one elite event which displayed optimal expression of the 35S-bar gene, i.e. tolerance to glufosinate ammonium, without penalty on agronomic performance and yield. This elite event was named EE-GH1.<br>
2.3.	Testing of EE-GH1 in cotton varieties with different genetic backgrounds and in<br>
different locations<br>
The selected event was introduced into different commercial genetic backgrounds, including FM989, FM 832, FM958, and FM966 and results of field trials of four different locations were compared. Plants were sprayed with 1600 g.a.i./ha, using different treatments (1x3-5 leaf stage, 4x, 3-5 leaf stage, lx+lx, 3-5 leaf stage, 4x+4x, 3-5 leaf stage, 0 as control). Seedling emergence and vigor rating for the elite event was very good.<br><br>
No visible damage as a result of herbicide application was ever observed after application regardless of rate or stage of development at the time of application. There were no detrimental effects on morphology or growth habit of plants by herbicide application<br>
Furthermore, the event had normal leaf, flower and boll morphology, excellent fertility, and showed no disease or abnormal insect susceptibility in multiple genetic backgrounds/ During  introgression  into  multiple  genetic backgrounds  no   aberrant problems  or abnormalities were observed over four generations.<br>
2.4. Genetic analysis of the locus<br>
The genetic stability of the insert for the EE-GH1 event was checked by molecular and<br>
phenotypic analysis in the progeny plants over several generations.<br>
Southern blot analyses of plants of the Tls T2 and T3 generation were compared for the<br>
EE-GH1 event. The patterns obtained were found to be identical in the different generations. This proves that the molecular configuration of the foreign DNA in EE-GH1 was stable.<br>
The EE-GH1 event displayed Mendelian segregation for the transgene as a single genetic locus in at least three subsequent generations indicating that the insert is stable.<br>
Example 3:     Characterization of elite event EE-GH1<br>
3.1 In-depth molecular and genetic analysis of the locus<br>
Once the EE-GH1 event was identified as the event in which expression of the transgene as well as overall agronomic performance were optimal, the locus of the transgene was analyzed in detail on a molecular level. This included sequencing of the flanking regions of the transgene.<br><br>
The sequence of the regions flanking the inserted transgene in the EE-GH1 event was determined using the TAIL-PCR protocol as described by Liu et al. (1995, Plant J. 8(3): 457-463).<br>
a)	Determination of the 5' flanking region<br>
The primers used were:<br><br>
The fragment amplified using MDB327-GHI05 was ca. 1200 bp which was sequenced (5'flank: SEQ ID NO: 1). The sequence between bp 1 and bp 677 comprised plant DNA, while the sequence between bp 678 and bp 850 corresponded to pGSV71 DNA.<br>
b)	Determination of the 3' flanking region<br>
The primers used were:<br><br><br>
The fragment amplified using MDB612-DPA017 was ca. 400 bp, the complete sequence of which was determined (SEQ ID NO: 4). The sequence between nucleotide 1 and 179 corresponds to T-DNA, while the sequence between nucleotide 180 and 426 corresponds to plant DNA.<br>
c) Identification of the target site deletion<br>
Using primers corresponding to sequences within the flanking regions of the transgene on the wildtype Gossypium hirsutum as a template, the insertion site of the transgene was identified.<br>
The following primers were used:<br>
Position in	Position in<br>
Sequence (5' -»3')	5'flank	3'flank<br>
(SEQ ID NO: 1)       (SEQ ID NO: 4)<br>
GHI06	TTg.CAC.CAT.CTA.gCT.CAC.TC	815 -» 795		<br>
(SEQ ID NO: 2)<br><br><br><br>
EE-GH1 Elite event Polymerase Chain Reaction identification protocol<br>
A test run, with all appropriate controls, has to be performed before attempting to screen unknowns. The presented protocol might require optimization for components that may differ between labs (template DNA preparation, Taq DNA polymerase, quality of the primers, dNTP's, thermocyler, etc.).<br>
Amplification of the endogenous sequence plays a key role in the protocol. One has to attain PCR and thermocycling conditions that amplify equimolar quantities of both the endogenous and transgenic sequence in a known transgenic genomic DNA template. Whenever the targeted endogenous fragment is not amplified or whenever the targeted sequences are not amplified with the same ethidium bromide staining intensities, as judged by agarose gel electrophoresis, optimization of the PCR conditions may be required.<br>
Template DNA<br>
Template DNA is prepared according to the CTAB method described by Doyle and Doyle (1987, Phytochem. Bull. 19: 11). When using DNA prepared with other methods, a test run utilizing different amounts of template should be done. Usually 50 ng of genomic template DNA yields the best results.<br>
Assigned positive and negative controls<br>
The following positive and negative controls should be included in a PCR run:<br>
-   Master Mix control (DNA negative control). This is a PCR in which no DNA is added to the reaction. When the expected result, no PCR products, is observed this indicates that the PCR cocktail was not contaminated with target DNA. &gt;<br><br>
-	A DNA positive control (genomic DNA sample known to contain the transgenic<br>
sequences). Successful amplification of this positive control demonstrates that the<br>
PCR was run under conditions which allow for the amplification of target sequences.<br>
-	A wildtype DNA control. This is a PCR in which the template DNA provided is<br>
genomic DNA prepared from a non-transgenic plant. When the expected result, no •<br>
amplification of the transgene PCR product but amplification of the endogenous PCR<br>
product, is observed this indicates that there is no detectable transgene background<br>
amplification in a genomic DNA sample.<br>
Primers<br>
The following primers, which specifically recognize the transgene and a flanking sequence of EE-GH1 are used:<br><br>
Primers targeting an endogenous sequence are always included in the PCR cocktail. These primers serve as an internal control in unknown samples and in the DNA positive control. A positive result with the endogenous primer-pair demonstrates that there is ample DNA of adequate quality in the genomic DNA preparation for a PCR product to be generated. The endogenous primers used are:<br>
GHI01:	5,-AAC.CTA.ggC.TgC.TgA.Agg.AgC-3'	(SEQ ID NO: 17)<br>
(Alcohol dehydrogenase gene Ace. NO: AF036569, 1070-&gt;1090)<br><br><br><br>
1 min. at 72°C For 25 cycles Followed by:    5 minutes at 72°C<br>
Agarose gel analysis -<br>
Between 10 and 20|il of the PCR samples should be applied on a 1.5% agarose gel (Tris-borate buffer) with an appropriate molecular weight marker (e.g. lOObp ladder PHARMACIA).<br>
Validation of the results<br>
Data from transgenic plant DNA samples within a single PCR run and a single PCR cocktail should not be acceptable unless 1) the DNA positive control shows the expected PCR products (transgenic and endogenous fragments), 2) the DNA negative control is negative for PCR amplification (no fragments) and 3) the wild-type DNA control shows the expected result (endogenous fragment amplification).<br>
Lanes showing visible amounts of the transgenic and endogenous PCR products of the expected sizes, indicate that the corresponding plant from which the genomic template DNA was prepared, has inherited the EE-GH1 elite event. Lanes not showing visible amounts of the transgenic PCR product and showing visible amounts of the endogenous PCR product, indicate that the corresponding plant from which the genomic template DNA was prepared, does not comprise the elite event. Lanes not showing visible amounts of the endogenous and transgenic PCR products, indicate that the quality and/or quantity of the genomic DNA didn't allow for a PCR product to be generated. These plants cannot be scored. The genomic DNA preparation should be repeated and a new PCR run, with the appropriate controls, has to be performed.<br><br>
Use of discriminating PCR protocol to identify EE-GH1<br>
Cotton leaf material from plants comprising different transgenic events (samples 1 to 4) was tested according to the above-described protocol. Samples from cotton wild-type were taken as negative.controls.<br>
The results of the PCR analysis are illustrated in Figure 1. Sample 1 is recognized as comprising elite event EE-GH1. All other tested lines do not comprise this elite event.<br>
Example 5:      Introgression of EE-GH1 into preferred cultivars<br>
Elite event EE-GH1 is introduced by repeated back-crossing into the following commercial cotton cultivars: FM5013, FM5015, FM5017, FM989, FM832, FM966 and FM958.<br>
It is observed that the introgression of the elite event into these cultivars does not significantly influence any of the desirable phenotypic or agronomic characteristics of these cultivars (no linkage drag) while expression of the transgene, as determined by glufosinate tolerance, meets commercially acceptable levels. This confirms the status of event EE-GH1 as an elite event.<br>
As used in the claims below, unless otherwise clearly indicated, the term "plant" is intended to encompass plant tissues, at any stage of maturity, as well as any cells, tissues, or organs taken from or derived from any such plant, including without limitation, any seeds, leaves, stems, flowers, roots, single cells, gametes, cell cultures, tissue cultures or protoplasts.<br>
Reference seed comprising elite event EE-GH1 was deposited as EE-GH1 at the ATCC (10801 University Blvd., Manassas, VA 20110-2209) on April 26, 2001, under ATCC accession number PTA-3343.<br><br>
As used in the claims below, unless otherwise clearly indicated, the term "plant" is intended to encompass plant tissues, at any stage of maturity, as well as any cells, tissues, or organs taken from or derived from any such plant, including without limitation, any seeds, leaves, stems, flowers, roots, single cells, gametes, cell cultures, tissue cultures or protoplasts.<br>
The above description of the invention is intended to be illustrative and not limiting. Various changes or modifications in the embodiments described may occur to those skilled in the art. These can be made without departing from the spirit or scope of the invention.<br><br><br><br>
WHAT IS CLAIMED IS:<br>
\.      A transgenic cotton plant or seed, cells or tissues thereof comprising event EE-GH1 in its genome.<br>
2.	The transgenic cotton plant, or seed, cells or tissues of claim 1, the genomic DNA of which, when analyzed using the Elite event identification protocol for EE-GH1 with two primers comprising the nucleotide sequence of SEQ ID NO: 2 and SEQ ID NO: 3 respectively, yields a DNA fragment of between 250 and 290 bp.<br>
3.	The cotton plant, or seed, cells or tissues thereof, according to claim 2, wherein said DNA fragment is a fragment of about 269 bp.<br>
4.	A cotton plant, or seed, cells or tissues thereof, obtained by propagation of and/or breeding with a cotton plant grown from the seed deposited at the ATCC under accession number PTA-3343.<br>
5.	A cotton plant, seed, cells or tissues thereof which is the progeny of the seed deposited at the ATCC under accession number PTA-3343.<br>
6.	A method for identifying elite event EE-GH1 in biological samples, which method comprises detecting an EE-GH1 specific region with a primer or probe which specifically recognizes the 5' flanking region of SEQ ID NO: 3 or the 3' flanking region of SEQ ID NO: 4 of EE-GH1.<br>
7.	The method of claim 6, said method comprising amplifying a DNA fragment of between 100 and 350 bp from a nucleic acid present in said biological samples using a polymerase chain reaction with at least two primers, one of which recognizes the 5' or 3' flanking region of EE-GH1 and the other which recognizes a sequence within the foreign DNA of EE-GH1.<br><br>
8.	The method of claim 7, wherein said one primer recognizes a sequence within the 5' flanking region of SEQ ID NO: 3 and said other primer recognizes a sequence within the foreign DNA of EE-GH1.<br>
9.	The method of claim 8, wherein said primer recognizing a sequence within the 5' flanking region of EE-GH1 comprises the sequence of SEQ ID NO: 2.<br>
10.	The method of any one of claims 6 to 9, wherein said primer recognizing a sequence within the foreign DNA comprises the sequence of SEQ ID NO: 1.<br>
11.	A method for identifying EE-GH1 in a biological sample, which method comprises detecting an EE-GH1 specific region with a specific primer or probe which hybridizes under stringent conditions to a sequence within the 5* of SEQ ID NO: 3 or within the 3' flanking sequence of SEQ ID NO: 4 of EE-GH1.<br>
12.	A method for identifying a transgenic plant, or cells or tissues thereof, comprising the elite event EE-GH I, which method comprises establishing that genomic DNA can be used, according to a PCR identification protocol, to amplify a DNA fragment of between 250 and 290 bp, using a polymerase chain reaction with two primers having the nucleotide sequence of SEQ ID NO: 1 and SEQ ID NO: 2, respectively.<br>
13.	A kit for identifying elite event EE-GH1 in biological samples, said kit comprising at least one PCR primer or probe, which recognizes a sequence within the 5' flanking region of SEQ ID NO: 3 or the 3' flanking region of SEQ ID NO: 4 of EE-GH1.<br>
14.	The kit of claim 13, wherein said at least one PCR primer recognizes a sequence within the plant DNA in SEQ ID NO: 3.<br><br>
15.	The kit of claim 14, wherein said primer recognizing a sequence within the plant DNA in SEQ ID NO: 3 comprises the sequence of SEQ ID NO: 2.<br>
16.	The kit of Claims 13 to 15, which further comprises at least a second PCR primer or probe which recognizes a sequence within the foreign DNA of EE-GH1.<br>
17.	The kit of claim 16, wherein said primer recognizing a sequence within the foreign DNA of EE-GH1 comprises the sequence of SEQ ID NO: 1.<br>
18.	A method for confirming seed purity, which method comprises detecting an EE-GH1 specific DNA sequence with a specific primer or probe which specifically recognizes a sequence within the 5' flanking region of SEQ ID NO: 3 or the 3' flanking region of SEQ ID NO: 4 of EE-GH1, in seed samples.<br>
19.	A method for screening seeds for the presence of EE-GH1, which method comprises detecting an EE-GH1 specific DNA sequence with a specific primer or probe which specifically recognizes a sequence within the 5' flanking region of SEQ ID NO: 3 or the 3' Hanking region of SEQ ID NO: 4 of EE-GH1, in samples of seed lots.<br>
20.	A seed deposited at the ATCC under accession number PTA-3343.<br>
21.	A cotton seed comprising elite event EE-GH1, reference seed comprising said event having been deposited at the ATCC under accession number PTA-3343.<br>
22.	A cotton plant, cell or tissue or plant material thereof comprising elite event EE-GH1, derived from the seed of claim 21.<br>
23.	Transgenic cotton plants, seeds, cells or tissues, the genomic DNA of which comprises a trar.sgene integrated into the chromosomal DNA in a region which<br><br>
»<br>
comprises a sequence of at least 40 bp which hybridizes under stringent conditions with a sequence which is complementary to the sequence of SEQ ID NO: 5.<br>
24.	A process for producing a transgenic cotton plant or cell or tissue of a cotton plant, said process comprising introducing a recombinant DNA molecule into a region of cotton chromosomal DNA corresponding to a sequence of at least 40 bp that * hybridizes under stringent conditions with a sequence that is complementary to the sequence of SEQ ID NO: 5, and, optionally, regenerating a cotton plant from the transformed cotton cell or tissue.<br>
25.	The process of claim 24, wherein said recombinant DNA molecule comprises an herbicide resistance gene.<br>
26.	The plant or cell or tissue of a cotton plant obtained by the process of claims 24 or<br>
25.<br>
27.	A transgenic cotton plant or seed, cells or tissues thereof comprising<br>
(i)        event EE-GH1 in its genome ;or<br>
(ii) event EE-GH1 with the proviso that the bar gene used in the event is substituted with a nucleic acid sequence that hybridizes to the complement of the bar gene under stringent conditions.<br>
28.	A DNA molecule which can be amplified from a nucleic acid sample of cotton, using a set of primers the first of which comprises a sequence of 15 to 30 nucleotides complementary to a sequence within SEQ ID NO:l or SEQ ID NO:4, the second of which comprises a sequence complementary to the foreign DNA present in said nucleic acid sample of GS». Cotfc§A.<br>
29.	The DNA molecule of claim 28, which can be amplified from a nucleic acid sample of cotton, using a set of primers, the first of which comprising SEQ ID NO:2, the second of which comprising SEQ ID NO:3.<br><br>
30.	A detection kit comprising the DNA of claim 28 or 29.<br>
31.	A DNA molecule isolated from corn tissue, comprising the sequence of SEQ ID NO:lorSEQH)NO:4<br>
32.	A DNA molecule comprising a sequence which is identical to or complementary to the sequence of 9 nucleotides of plant DNA and foreign DNA at each side contiguous with the insertion site contained in SEQ ID NO: 1 or SEQ ID NO: 4.<br><br>
33.	A transgenic cotton plant or seed substantially as herein described with reference to the<br>
accompanying drawings.<br>
34.	A DNA molecule substantially as herein described with reference to the accompanying<br>
drawings.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ4NS1jaGVucC0yMDA0IGNsYWltcy1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">0485-chenp-2004 claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ4NS1jaGVucC0yMDA0IGRlc2NyaXB0aW9uIChjb21wbGV0ZSktZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">0485-chenp-2004 description (complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ4NS1jaGVucC0yMDA0IGRyYXdpbmdzLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">0485-chenp-2004 drawings-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWNoZW5wLTIwMDQgIGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">485-chenp-2004  abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWNoZW5wLTIwMDQgIGNsYWltcyBncmFudGVkLnBkZg==" target="_blank" style="word-wrap:break-word;">485-chenp-2004  claims granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWNoZW5wLTIwMDQgIGNvcnJlc3BvbmRlbmNlIG90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">485-chenp-2004  correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWNoZW5wLTIwMDQgIGNvcnJlc3BvbmRlbmNlIHBvLnBkZg==" target="_blank" style="word-wrap:break-word;">485-chenp-2004  correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWNoZW5wLTIwMDQgIGZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">485-chenp-2004  form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWNoZW5wLTIwMDQgIGZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">485-chenp-2004  form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWNoZW5wLTIwMDQgIGZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">485-chenp-2004  form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWNoZW5wLTIwMDQgIGZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">485-chenp-2004  form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWNoZW5wLTIwMDQgIHBldGl0aW9ucy5wZGY=" target="_blank" style="word-wrap:break-word;">485-chenp-2004  petitions.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWNoZW5wLTIwMDQtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">485-chenp-2004-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWNoZW5wLTIwMDQtY29ycmVzcG9uZG5lY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">485-chenp-2004-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWNoZW5wLTIwMDQtY29ycmVzcG9uZG5lY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">485-chenp-2004-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWNoZW5wLTIwMDQtZGVzY3JpcHRpb24oY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">485-chenp-2004-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWNoZW5wLTIwMDQtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">485-chenp-2004-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWNoZW5wLTIwMDQtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">485-chenp-2004-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWNoZW5wLTIwMDQtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">485-chenp-2004-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWNoZW5wLTIwMDQtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">485-chenp-2004-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWNoZW5wLTIwMDQtb3RoZXIgZG9jdW1lbnRzLnBkZg==" target="_blank" style="word-wrap:break-word;">485-chenp-2004-other documents.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWNoZW5wLTIwMDQtcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">485-chenp-2004-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="230407-switching-device.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="230409-coating-composition-for-metal-substrates.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>230408</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>485/CHENP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>26-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>05-Mar-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>BAYER BIOSCIENCE N.V.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>JOZEF PLATEAUSTRAAT 22, B-9000 GENT,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>TROLINDER, LINDA</td>
											<td>1004 DOGWOOD ST., IDALOUT, TX 79329,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>JEFFERSON, GWYN</td>
											<td>2787 MARICO PLACE, GREEN VILLE, MS 38701,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>DE BEUCKELEER, MARC</td>
											<td>HEKERS 36, B-9052 ZWIJNAARDE,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A01H</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP02/08136</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-07-19</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>09/921,922</td>
									<td>2001-08-06</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/230408-a-method-and-kit-for-identifying-elite-event-ee-gh1-in-biological-samples by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:11:16 GMT -->
</html>
