<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/230544-dna-fragment-and-plasmid-and-partial-dna-fragment-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:19:56 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 230544:DNA FRAGMENT AND PLASMID AND PARTIAL DNA FRAGMENT THEREOF</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">DNA FRAGMENT AND PLASMID AND PARTIAL DNA FRAGMENT THEREOF</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A shuttle vector is constructed by preparing a DNA region replicable in bacteria belonging to the genus Rhodococcus from a Rhodococcus-derived plasmid having the nucleotide sequence set forth as SEQ ID NO: 73 and a plasmid or its DNA fragment having the nucleotide sequence set forth as SEQ ID NO: 74, and a DNA region replicable in E. coli from an E. coli-derived plasmid or its DNA fragment. An aminoketone asymmetric reductase gene is inserted into the shuttle vector, transformants containing the vector are created, and the aminoketone asymmetric reductase and optically active aminoalcohols are produced.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>I <br><br>
DESCRIPTION<br>
NOVEL PLASMIDS AND UTILIZATION THEREOF Technical Field<br>
[0001]	The present invention relates to novel plasmids derived<br>
from any of microorganisms belonging to the genus Rhodococcus (hereinafter referred to as "the genus Rhodococcus') and to utilization thereof More specifically, the invention relates to plasmids or their partial DNA fragments (hereinafter also referred to simply as "DNA fragments"), and to shuttle vectors, vectors, transformants, aminoketone asymmetric reductase production methods and optically active aminoalcohol production methods which utilize them. Background Art<br>
[0002]	The genus Rhodococcus is known to produce enzymes<br>
involved in nitrile metabolism and to produce enzymes which asymmetrically reduce aminoketones. In particular, Rhodococcus erythropolis is known to have very high aminoketone asymmetric reduction activity. Such microorganisms and enzymes act on a-aminoketones to high selectively produce optically active p-aminoalcohols at high yields (for example. Patent documents, 1 and 5). Thus, it has long been desired to develop a host-vector system intended for mass production of useftil enzymes and the like in the genus Rhodococcus. However, the development of vectors suitable for the genus Rhodococcus as hosts has lagged behind. Only a few strains of the genus Rhodococcus have been found with plasmids, namely Rhodococcus sp. H13-A (Non-patent document 1), Rhodococcus rhodochrous    ATCC4276    (Patent    document    2),    Rhodococcus<br><br>
rhodochrous   ATCC4001   (Patent   document   3)   and  Rhodococcus<br>
erythropolis IFO12320 (Patent document 4).<br>
[0003]	[Patent document 1] WOOl/73100<br>
[Patent   document   2]   Japanese   Unexamined   Patent Publication HEI No. 4-148685<br>
[Patent   document   3]   Japanese   Unexamined   Patent Publication HEI No. 4-330287<br>
[Patent   document   4]   Japanese   Unexamined   Patent Publication HEI No. 9-28379<br>
[Patent document 5] WO02/070714<br>
[Non-patent document 1] J. Bacteriol., 170, 638, 1988 Disclosure of the Invention Problems to be Solved by the Invention<br>
[0004]	As mentioned above, it has been desired to develop new<br>
vectors for breeding and improve to industrially useful strains (mutant<br>
strains) from the genus Rhodococcus. In particular, self-cloning<br>
systems are preferred from the standpoint of safety of the recombinant<br>
DNA microbes and their products which may be used as foods and<br>
additives. It is an object of the present invention to provide novel<br>
plasmids that can be used as vectors for such a host-vector system.<br>
[0005]	It is desirable to create recombinant microbes suitable for<br>
industrial application from among Rhodococcus erythropolis which has<br>
aminoketone asymmetric reduction activity.  In particular, it is a first<br>
object of the invention to provide novel plasmids or their partial DNA<br>
fragments which can be used to create such recombinant microbes.<br>
[0006]	If a plasmid such as described above can be ohtainprl  it<br><br>
would become easy to construct a shuttle vector that is replicable even<br>
in other microbes.   It is therefore a second object of the invention to<br>
provide   nucleotide   sequence   data   relating   to   DNA   replication<br>
(replication region, etc.) necessary for construction of such a shuttle<br>
vector.<br>
[0007]	It is a third object of the invention to provide shuttle<br>
vectors that are replicable in both the genus Rhodococcus and £■. coli.<br>
[0008]	It is a fourth object of the invention to apply the shuttle<br>
vectors to an aminoketone asymmetric reductase.<br>
Means for Solving the Problems<br>
[0009]	The present inventors carefully screened plasmids for<br>
vector construction from among Rhodococcus strains, and as a result<br>
discovered several^ovel plasmidyusable as vectors for host-vector<br>
sy sterns.<br>
[0010]	Furthermore, the present inventors found that it is possible<br>
to construct shuttle vectors by transferring into the aforementioned<br>
plasmids a drug resistance gene and a gene region that is replicable in E.<br>
coli.    As  a result there were obtained nucleotide sequence data,<br>
plasmids and shuttle vectors that achieve the objects stated above, and<br>
the present invention has thereupon been completed.<br>
[0011]	Specifically,   the   present   invention  provides   a  DNA<br>
fragment, a DNA, a plasmid, a shuttle vector, a vector, a transformant,<br>
a method for production of an aminoketone asymmetric reductase, and<br>
a method for production of an optically active aminoalcohol, according<br>
to following (1) to (39).<br>
(1) A DNA fragment having at least one nucleotide sequence selected<br><br>
from the group consisting of the nucleotide sequences set forth as SEQ ID NO: 35, SEQ ID NO: 36 and SEQ ID NO: 37.<br>
(2)	A plasmid or a partial DNA fragment thereof, characterized by comprising a DNA replication region having at least one nucleotide sequence selected from the group consisting of the nucleotide sequences set forth as SEQ ID NO: 35, SEQ ID NO: 36 and SEQ ID NO: 37.<br>
(3)	A DNA fragment having at least one nucleotide sequence selected from the group consisting of the nucleotide sequences set forth as SEQ ID NO: 1, SEQ ID NO: 4, SEQ ID NO: 14, SEQ ID NO: 17 and SEQ ID NO: 22.<br>
(4)	A plasmid or a partial DNA fragment thereof, characterized by comprising a coding region for a DNA replication-related protein having at least one nucleotide sequence selected from the group consisting of the nucleotide sequences set forth as SEQ ID NO: 1, SEQ ID NO: 4, SEQ ID NO: 14, SEQ ID NO: 17 and SEQ ID NO: 22.<br>
(5)	A plasmid or a partial DNA fragment thereof, characterized by comprising a coding region for a DNA replication-related protein having at least one nucleotide sequence selected from the group consisting of the nucleotide sequences set forth as SEQ ID NO: 1, SEQ ID NO: 4, SEQ ID NO: 14, SEQ ID NO: 17 and SEQ ID NO: 22 and comprising a DNA replication region having at least one nucleotide sequence selected from the group consisting of the nucleotide sequences set forth as SEQ ID NO: 35, SEQ ID NO: 36 and SEQ ID NO: 37.<br>
(6)	A DNA fragment having the nucleotide sequence set forth as SEQ<br><br>
ID NO: 76.<br>
(7)	A plasmid or a partial DNA fragment thereof, characterized by comprising a promoter region having the nucleotide sequence set forth as SEQ ID NO: 76.<br>
(8)	A plasmid or a partial DNA fragment thereof, characterized by comprising a coding region for a DNA replication-related protein having at least one nucleotide sequence selected from the group consisting of the nucleotide sequences set forth as SEQ ID NO: 1, SEQ ID NO: 4, SEQ ID NO: 14, SEQ ID NO: 17 and SEQ ID NO: 22, comprising a DNA replication region having at least one nucleotide sequence selected from the group consisting of the nucleotide sequences set forth as SEQ ID NO: 35, SEQ ID NO: 36 and SEQ ID NO: 37, and comprising a promoter region having the nucleotide sequence set forth as SEQ ID NO: 76.<br>
(9)	A circular plasmid characterized by comprising a plasmid or a partial DNA fragment according to any one of (1) to (8), wherein the numbers of restriction endonuclease cleavage sites are BamH I: 2, EcoR I: 2, Kpn I: 1, Pvu II: 1, Sac I: 1 and Sma I: 1, and the size is approximately 5.4 kbp.<br><br>
(10)	A plasmid having the nucleotide sequence set forth as SEQ ID NO: 73)<br>
(11)	A plasmid or a DNA fragment according to any one of (1) to (10), characterized by being derived from a bacterium belonging to the genus Rhodococcus.<br>
(12)	A DNA fragment having at least one nucleotide sequence selected from the group consisting of the nucleotide sequences set forth as SEQ<br><br>
ID NO: 70, SEQ ID NO: 71 and SEQ ID NO: 72.<br>
(13)	A plasmid or a partial DNA fragment thereof, characterized by comprising a DNA replication region having at least one nucleotide sequence selected from the group consisting of the nucleotide sequences set forth as SEQ ID NO: 70, SEQ ID NO: 71 and SEQ ID NO: 72.<br>
(14)	A DNA fragment having at least one nucleotide sequence selected from the group consisting of the nucleotide sequences set forth as SEQ ID NO: 40, SEQ ID NO: 42, SEQ ID NO: 44, SEQ ID NO: 45, SEQ ID NO: 53, SEQ ID NO: 55, SEQ ID NO: 56, SEQ ID NO: 61, SEQ ID NO: 62 and SEQ ID NO: 69.<br>
(15)	A plasmid or a partial DNA fragment thereof, characterized by comprising a coding region for a DNA replication-related protein<br>
having at least one nucleotide sequence selected from The group consisting of the nucleotide sequences set forth as SEQ ID NO: 40, SEQ ID NO: 42, SEQ ID NO: 44, SEQ ID NO: 45, SEQ ID NO: 53, SEQ ID NO: 55, SEQ ID NO: 56, SEQ ID NO: 61, SEQ ID NO: 62 and SEQ ID NO: 69.<br>
(16) A plasmid or a partial DNA fragment thereof, characterized by comprising a coding region for a DNA replication-related protein having at least one nucleotide sequence selected from the group consisting of the nucleotide sequences set forth as SEQ ID NO: 40, SEQ ID NO: 42, SEQ ID NO: 44, SEQ ID NO: 45, SEQ ID NO: 53, SEQ ID NO: 55, SEQ ID NO: 56, SEQ ID NO: 61, SEQ ID NO: 62 and SEQ ID NO: 69 and comprising a DNA replication region having at least one nucleotide sequence selected from the group consisting of<br><br>
the nucleotide sequences set forth as SEQ ID NO: 70, SEQ ID NO: 71 and SEQ ID NO: 72.<br>
(17)	A plasmid or a partial DNA fragment thereof, characterized by comprising a coding region for a DNA replication-related protein having at least one nucleotide sequence selected from the group consisting of the nucleotide sequences set forth as SEQ ID NO: 40, SEQ ID NO: 42, SEQ ID NO: 44, SEQ ID NO: 45, SEQ ID NO: 53, SEQ ID NO: 55, SEQ ID NO: 56, SEQ ID NO: 61, SEQ ID NO: 62 and SEQ ID NO: 69, comprising a DNA replication region having at least one nucleotide sequence selected from the group consisting of the nucleotide sequences set forth as SEQ ID NO: 70, SEQ ID NO: 71 and SEQ ID NO: 72, and comprising a promoter region having the nucleotide sequence set forth as SEQ ID NO: (76.;<br>
(18)	A DNA fragment having at least one nucleotide sequence selected from the group consisting of the nucleotide sequences set forth as SEQ ID NO: 67 and SEQ ID NO: 47.<br>
(19)	A plasmid or a partial DNA fragment thereof, characterized by comprising a mobilization protein region having at least one nucleotide sequence selected from the group consisting of the nucleotide sequences set forth as SEQ ID NO: 67 and SEQ ID NO: 47.<br>
(20)	A DNA fragment having the nucleotide sequence set forth as SEQ ID NO: 75.<br>
(21)	A plasmid or a partial DNA fragment thereof, characterized by comprising a mobilization-related region having the nucleotide sequence set forth as SEQ ID NO:(75.<br>
(22)	A circular plasmid characterized by comprising a plasmid or DNA<br><br>
fragment according to any one of (12) to (21), wherein the numbers of restriction endonuclease cleavage sites are BamH I: 2, Pvu II: 4, Sac I: 3 and Sma I: 4, and the size is approximately 5.8 kbp.<br>
(23)	A plasmid having the nucleotide sequence set forth as SEQ ID<br>
Nom)<br>
(24)	A plasmid or a DNA fragment according to any one of (12) to (23), characterized by being derived from a bacterium belonging to the genus Rhodococcus.<br>
(25)	A DNA fragment having the nucleotide sequence set forth as SEQ ID NO: 77.<br>
(26)	A DNA fragment characterized by comprising a piojnotstJisgion having the nucleotide sequence set forth as SEQ ID NO: 77.<br><br>
(27)	A shuttle vector replicable in bacteria belonging to the genus Rhodococcus and E. coli, and comprising a plasmid or partial DNA fragment thereof according to any one of (1) to (26) and a DNA region replicable in E. coli.<br>
(28)	A vector characterized by being constructed using a shuttle vector according to (27).<br>
(29)	A vector characterized by comprising a plasmid or DNA fragment according to any one of (6), (7), (25) or (26).<br>
(30)	A vector according to (28) or (29), characterized by having<br>
inserted therein an aminoketone asymmetric reductase gene.<br>
(31)	A vector according to (30), characterized in that the aminoketone<br>
asymmetric reductase gene is a nucleic acid coding for a protein<br>
consisting the amino acid sequence set forth as SEQ ID NO: 78, or a<br>
nucleic acid that codes for a protein having the amino acid sequence set<br><br>
forth as SEQ ID NO: 78 with a deletion, insertion, substitution or addition of one or a plurality of amino acids, and having aminoketone asymmetric reduction activity.<br>
(32)	A vector according to (30), characterized in that the aminoketone asymmetric reductase gene is a nucleic acid consisting the nucleotide sequence set forth as SEQ ID NO: 79, or a nucleic acid that hybridizes with nucleic acid having a nucleotide sequence complementary to the nucleotide set forth as SEQ ID NO: 79 under stringent conditions, and that codes for a protein having aminoketone asymmetric reduction activity.<br>
(33)	A transformant containing a vector according to (28) or (29).<br>
(34)	A transformant containing a vector according to any one of (30) to (32).<br>
(35)	A method for production of an aminoketone asymmetric reductase, which comprises a culturing step in which a transformant according to (34) is cultured in medium that allows growth of said transformant, and<br>
a purification step in which the aminoketone asymmetric reductase is purified from said transformant obtained in said culturing step.<br>
(36)	A method for production of an optically active aminoalcohol,<br>
wherein an aminoketone asymmetric reductase obtained by the<br>
production method of (35) is reacted with an enantiomeric mixture of<br>
an a-aminoketone compound represented by the following general<br>
formula (1):<br>
[Chemical Formula 1]<br><br><br><br>
and having the desired optical activity.<br>
(37) A method for production of an optically active aminoalcohol,<br>
wherein   a  transformant   according  to   (34)   is   reacted   with   an<br>
enantiomeric mixture of an a-aminoketone compound represented by<br>
the following general formula (1):<br>
[Chemical Formula 3]<br><br>
wherein X may be the same or different and represents at least one<br>
species selected from the group consisting of halogen, lower alkyl,<br>
hydroxyl optionally protected with a protecting group,  nitro  and<br>
sulfonyl;<br>
n represents an integer of 0 to 3;<br>
R^ represents lower alkyl;<br>
R   and R  may be the same or different and represent at least one<br>
species selected from the group consisting of hydrogen and lower<br>
alkyl; and<br>
* represents asymmetric carbon,<br>
or a salt thereof, to produce an optically active aminoalcohol compound<br>
represented by the following general formula (2):<br>
[Chemical Formula 4]<br><br><br><br><br>
o<br>
hydrocarbon ring which is bonded to R   or a 5- to 8-membered<br>
heterocyclic skeleton containing 1-3 heteroatoms which is bonded to R^<br>
orR^;<br>
R represents hydrogen or optionally substituted CI-6 alkyl;<br>
R^ represents hydrogen, carboxyl, optionally substituted CI-6 alkyl, a<br>
5- to 8-membered heterocyclic skeleton containing 1-3 heteroatoms<br><br>
which is bonded to R"* or a C5-10 hydrocarbon ring which is bonded to<br>
R';<br>
R^ represents hydrogen, optionally substituted Cl-6 alkyl, optionally substituted Cl-6 alkyloxycarbonyl, optionally substituted acyl or a 5-to 8-membered heterocyclic skeleton containing 1-3 heteroatoms which is bonded to R^ or R^; and<br>
R^° represents hydrogen or optionally substituted Cl-6 alkyl, or a pharmaceutically acceptable salt or solvate thereof, for production of an optically active aminoalcohol.<br>
(39) A shuttle vector according to (27), having a nucleotide sequence selected from the group consisting of the nucleotide sequences set forth as SEQ ID NO: 89 to SEQ ID NO: 100. Effect of the Invention<br>
[0012]	The   plasmids   of  the   invention   are   novel   plasmids<br>
unknown to the prior art, and are valuable as vectors for host-vector<br>
systems belonging to the industrially useful the genus Rhodococcus.<br>
They are of particular utility in the creation of recombinant microbes<br>
capable of industrial asymmetric reduction of aminoketones. An<br>
example of asymmetric reduction of an aminoketone to which such<br>
microbes may contribute is a reaction for production of d-(lS, 2S)-<br>
pseudoephedrine from 1-2-methylamino-l-phenyl-1-propanone.<br>
[0013]	The  plasmids  of the  invention  can  coexist  in  single<br>
Rhodococcus cell and therefore can be used not only alone for their replicating ftinction, but also as compatible plasmids. That is, by inserting different protein (for example, enzyme) genes into the different plasmids, it is possible to express the proteins simultaneously<br><br>
In the same cell.<br>
The shuttle vectors of the invention are useful for creation of industrially useful recombinant microbes of the genus Rhodococcus and Escherichia coli.<br>
The nucleotide sequence data relating to DNA replication obtained from the plasmids of the invention may serve as the basis for construction of the aforementioned shuttle vectors, and specifically they provide DNA fragments as constituent elements of the vectors.<br>
Brief Description of the accompanying Drawings<br>
Fig. 1 is a restriction enzyme cleavage map of plasmid PretllOO.<br>
Fig. 2 is a restriction enzyme cleavage map of plasmid PretlOOO.<br>
Fig. 3 is a summary illustration for construction of shuttle vector pRETllOl.<br>
Fig. 4 is a summary illustration for construction of shuttle vector pRET1102.<br>
Fig. 5 is a summary illustration for construction of shuttle vector pRETllOS.<br>
Best Mode for Carrying Out the Invention<br>
Preferred embodiments of the invention will now be explained.<br>
The first plasmid of the invention is a plasmid isolated from the genus Rhodococcus, or a derivative thereof. Specifically, it may be isolated from, for example, Rhodococcus erythropolis<br><br>
IAM1400, IAM1503, JCM2893 and JCM2894 strains, has a size of approximately 5.4 kbp and is a circular plasmid cleavable by the restriction enzymes shown in Table 1. The plasmids isolated from each of these strains are designated as pRETllOO, pIlET1300, pRETlSOO and pRET1700, respectively. Plasmids of the invention may be prepared from these sample strains by publicly known methods (for example, boiling, alkali dissolution, cesium chloride density gradient ultracentriftigation: Lab Manual Idenshi Kogaku, 3rd Edition, Chapter 10, pp.55-59, Maruzen).<br><br>
pRETl 100. This plasmid was sequenced by a publicly known method (using a fluorescent automatic sequencer, for example) and its full nucleotide sequence was revealed to be 5444 bp set forth as SEQ ID NO: 73 of the Sequence Listing.<br>
[0021]	The second plasmid of the invention is also a plasmid<br>
isolated from the genus Rhodococcus, or its derivative. Specifically, it may be isolated from, for example, Rhodococcus rhodnii JCM3203, has a size of approximately 5.8 kbp and is a circular plasmid cleavable by the restriction enzymes shown in Table 2.     This plasmid is<br><br><br>
pRETlOOO.   This plasmid was also sequenced by a publicly known<br>
method and its full nucleotide sequence was revealed to be 5813 bp set<br>
forth as SEQ ID NO: 74 of the Sequence Listing.<br>
[0024]	The plasmids of the invention (natural- or wild-types) are<br>
circular plasmids that can also be defined by the restriction enzyme cleavage patterns shown in Tables 1 and 2. Thus, the present invention encompasses the following two types of circular plasmids.<br>
(1)	A circular plasmid derived from a Rhodococcus strain,<br>
characterized by having a size of approximately 5.4 kbp and possessing<br>
the following restriction enzyme cleavage sites: BamYi 1:2, EcoK 1:2,<br>
Kpnl-A,PvulVA, Sacl'A 2CCi&amp;Smal-A.<br>
(2)	A circular plasmid derived from a Rhodococcus strain,<br>
characterized by having a size of approximately 5.8 kbp and possessing<br>
the following restriction enzyme cleavage sites: BarnR 1:2, Pvu 11:4,<br>
Sac 1:3 and Sma 1:4.<br>
[0025]	As a result of analysis of the nucleotide sequences of<br>
plasmids pRETllOO and pRETlOOO (i.e., SEQ ID NO: 73 and SEQ ID NO: 74), there is predicted the existence of a group of nucleotide sequences (open reading frames, hereinafter "orf) coding for proteins<br><br>
for DNA replication or other functions.<br>
[0026]	In the relevant technical field, "DNA replication" refers to<br>
using DNA itself as template to form two double-stranded DNA molecules exactly identical to existing double-stranded DNA (parent DNA). The replication mechanism consists of three stages: initiation from the starting point of replication (replication origin), DNA chain elongation and termination. During replication, a portion of the DNA double strand is unraveled and new DNA strands are synthesized complementary to each single strand. The double strand is unraveled by DNA helicase and helix destabilizing proteins (also known as single-strand DNA-binding protein), and the unraveled portion is referred to as the replication fork. The template DNA in the direction from 3' to 5' toward the replication fork is the "leading strand", and the one in the direction from 5' to 3' is the "lagging strand". DNA polymerase extends the DNA strand in the direction from 5' to 3'. Therefore when the leading strand is the template, DNA is synthesized in the direction of the replication fork. However when the opposite lagging strand is the template, the DNA strand must be extended in the opposite direction from the replication fork. Consequently, replication of the lagging strand is accomplished in fragments of about 200 bases, known as Okazaki fragments. Every approximately 200 bases, RNA primer is used with DNA as template to synthesize 10 bases of RNA in the direction from 5' to 3'. From this RNA as primer, DNA polymerase synthesizes a DNA strand in the direction from 5' to 3' on the lagging strand as template. The replicated DNA fragment of approximately 200 bases then binds to the single-stranded DNA from which RNA is<br><br>
removed. In this replication mechanism, several proteins including DNA helicase and helix-destabilizing protein work together to form the replicating machinery. Other proteins involved include DNA topoisomerase (which prevents twisting during the DNA replication), replication initiation proteins and replication termination proteins. The DNA replication mechanism is described in detail in, for example, "Saibou no Bunshiseibutsugaku [Molecular Biology of the Cell]", 3rd Edition, translated by Keiko Nakamura et al., pp.251-262, Kyoikusha, 1996.<br>
[0027]	Upon analysis of the nucleotide sequences of the plasmids<br>
pRETllOO and pRETlOOO, they were found to include sequences of AT-rich homologous or analogous repeats and a sequence thought to have a DNA secondary structure, i.e. a nucleotide sequence predicted to be a DNA replication region (a nucleotide sequence region recognized by proteins involved in DNA replication or a region including the DNA replication origin), in the vicinity of the aforementioned orf relating to DNA replication.<br>
[0028]	DNA replication requires a DNA replication region and a<br>
region coding for a protein involved in DNA replication (hereinafter referred to as "DNA replication-related protein"). According to the present invention it is possible to obtain data relating to the nucleotide sequences of these regions for both plasmids pRETllOO and pRETlOOO.<br>
[0029]	First, the nucleotide sequences set forth as SEQ ID NO:<br>
35-37 were identified as DNA replication regions for plasmid pRETl 100. As regions coding for proteins related to DNA replication<br><br>
there were identified the nucleotide sequences set forth as SEQ ID NO:<br>
1-3 (orfl), the nucleotide sequence set forth as SEQ ID NO: 4 (orfZ),<br>
the nucleotide sequences set forth as SEQ ID NO: 5-16 (orfi), the<br>
nucleotide sequences set forth as SEQ ID NO: 17-21 (orf4), the<br>
nucleotide sequences set forth as SEQ ID NO: 22-26 (orfS), the<br>
nucleotide sequence set forth as SEQ ID NO: 27 or 28 (orf6), the<br>
nucleotide sequence set forth as SEQ ID NO: 29 or 30 (orf7), the<br>
nucleotide sequence set forth as SEQ ID NO: 31 or 32 (orfS), and the<br>
nucleotide sequence set forth as SEQ ID NO: 33 or 34 (orf9).<br>
[0030]	Construction of a plasmid capable of DNA replication<br>
from pRETllOO requires that the recombinant plasmid have at least one DNA replication region and at least one coding region (orf) for a DNA replication-related protein. Thus, the (recombinant) plasmids of the invention are characterized by comprising at least one DNA replication region and at least one coding region for a DNA replication-related protein. The coding region for a DNA replication-related protein preferably has a nucleotide sequence selected from the group consisting of the nucleotide sequences set forth as SEQ ID NO: 1,4, 14, 17 and 22.<br>
[0031]	The region of the nucleotide sequence set forth as SEQ ID<br>
NO: 76 has been suggested as a promoter involved in expression of replication-related proteins, and the plasmids of the invention preferably comprise a promoter region having the nucleotide sequence set forth as SEQ ID NO: 76.<br>
[0032]	For   plasmid   construction,   the   DNA   fragments   are<br>
appropriately   selected   based   on   the   aforementioned   nucleotide<br><br>
sequence data. The present invention also encompasses derivatives or<br>
functional (DNA-replicating) fragments of the plasmids.<br>
[0033]	Next, the nucleotide sequences set forth as SEQ ID NO:<br>
70-72 were identified as DNA replication regions for plasmid pRETlOOO. As regions coding for proteins related to DNA replication there were identified the nucleotide sequences set forth as SEQ ID NO: 38-41 (orflO), the nucleotide sequence set forth as SEQ ID NO: 42 or 43 (orfl 1), the nucleotide sequence set forth as SEQ ID NO: 44 (orfl2), the nucleotide sequence set forth as SEQ ID NO: 45 or 46 (orfl3), the nucleotide sequences set forth as SEQ ID NO: 48-50 (orfl4), the nucleotide sequence set forth as SEQ ID NO: 51 or 52 (orfl5), the nucleotide sequence set forth as SEQ ID NO: 53 or 54 (orfl6), the nucleotide sequence set forth as SEQ ID NO: 55 (orfl 7), the nucleotide sequences set forth as SEQ ID NO: 56-60 (orfl8), the nucleotide sequence set forth as SEQ ID NO: 61 (orfl9), the nucleotide sequence set forth as SEQ ID NO: 62 (orfZO), and the nucleotide sequences set forth as SEQ ID NO: 63-69 (orfZl).<br>
[0034]	Construction of a plasmid capable of DNA replication<br>
from pRETlOOO requires that the recombinant plasmid have at least one DNA replication region and at least one coding region (orf) for a DNA replication-related protein. Thus, the (recombinant) plasmids of the invention are characterized by comprising at least one DNA replication region and at least one coding region for a DNA replication-related protein. The coding region for a DNA replication-related protein preferably has a nucleotide sequence selected from the group consisting of the nucleotide sequences set forth as SEQ ID NO: 40, 42,<br><br>
44,45, 53, 55, 56, 61, 62 and 69.<br>
[0035]	The regions with the nucleotide sequences set forth as<br>
SEQ ID NO: 67 and 47 are homologous with mobilization proteins, and have been implicated in mobilization. The region with the nucleotide sequence set forth as SEQ ID NO: 75 has been implicated in gene expression of mobilization protein and suggested as a recognition site for an expressed protein. Thus, the plasmids of the invention preferably include mobilization protein regions having the nucleotide sequences set forth as SEQ ID NO: 67 and 47, or include a region involved in mobilization having the nucleotide sequence set forth as SEQ ID NO: 75.<br>
[0036]	For   plasmid   construction,   the   DNA   fragments   are<br>
appropriately   selected   based   on   the   aforementioned   nucleotide<br>
sequence data.  The present invention also encompasses derivatives or<br>
functional (DNA-replicating) fragments of the plasmids.<br>
[0037]	The plasmids or DNA fragments of the invention may also<br>
contain nucleotide sequences with a substitution, deletion or insertion<br>
of one or a plurality of nucleotides in a DNA replication region, DNA<br>
replication-related protein coding region, promoter region, mobilization<br>
protein region or mobilization-related region, or a portion thereof, so<br>
long as the function of each region is not impaired.<br>
[0038]	The shuttle vectors of the invention may be any which<br>
:omprise a plasmid or DNA fragment having a DNA replication region, DNA   replication-related  protein   coding  region,  promoter  region, nobilization protein region or mobilization-related region, and a DNA region that is replicable in E. coli, and which are replicable in the genus<br><br>
Rhodococcus and E. coli, such as those having the nucleotide sequences set forth as SEQ ID NO; 89 to 100, The shuttle vectors of the invention may also have nucleotide sequences with one or a plurality of nucleotide substitutions, deletions or insertions in the aforementioned nucleotide sequences, so long as they are replicable in the genus Rhodococcus and E. coli.<br>
[0039]	The "plurality" referred to above will differ depending on<br>
the type of region, and specifically may be 2-1100, preferably 2-800, more preferably 2-300, even more preferably 2-100, yet more preferably 2-20 and most preferably 2-10.<br>
[0040]	As a plasmid or DNA fragment having substantially the<br>
same nucleotide sequence as the aforementioned DNA replication region, DNA replication-related protein coding region, promoter region, mobilization protein region or mobilization-related region, or a portion thereof, there may be mentioned specifically, a nucleotide sequence which hybridizes with a DNA replication region, DNA replication-related protein coding region, promoter region, mobilization protein region or mobilization-related region, or a portion thereof, under stringent conditions. Here "stringent conditions" are conditions under which specific hybrids are formed and non-specific hybrids are not formed. While it is difficult to precisely quantify the conditions, one example is a set of conditions that permit hybridization of DNA with high homology, such as 80% or greater, preferably 90% or greater or more preferably 95% or greater homology, while not permitting hybridization of DNA with lower homology. More specifically, there may be mentioned hybridization conditions with ordinary Southern<br><br>
hybridization washing at 60°C, IxSSC, 0.1% SDS or preferably<br>
Southern hybridization washing at O.lxSSC, 0.1% SDS corresponding<br>
salt concentration. When a DNA fragment with a length of<br>
approximately 300 bp is used as a portion of the DNA replication<br>
region, DNA replication-related protein coding region, promoter region,<br>
mobilization protein region or mobilization-related region, the<br>
hybridization washing conditions may be 50°C, 2xSSC, 0.1% SDS.<br>
[0041]	The aforementioned plasmid or DNA fragment having<br>
substantially the same nucleotide sequence as the aforementioned DNA replication region, DNA replication-related protein coding region, promoter region, mobilization protein region or mobilization-related region, or a portion thereof, may be obtained by, for example, modification of a DNA replication region, DNA replication-related protein coding region, promoter region, mobilization protein region or mobilization-related region, or a portion thereof, by site-directed mutagenesis so as to have a substitution, deletion or insertion of nucleotides at a specific site. Such modified DNA may also be obtained by mutation treatment known in the prior art. As mutation treatments there may be mentioned methods of in vitro treatment of DNA including a DNA replication region, DNA replication-related protein coding region, promoter region, mobilization protein region or mobilization-related region, or a portion thereof, with hydroxylamine or the like, and methods of treating a microbe possessing the DNA above, such as the genus Escherichia, with ultraviolet rays or with a mutagenic agent ordinarily used for mutagenesis such as N-methyl-N'-nitro-N-nitrosoguanidine (NTG) or EMS.<br><br>
[0042]	Nucleotide   substitutions,   deletions   or   insertions   as<br>
mentioned above include those found in naturally occurring mutants or<br>
variants due to differences in Rhodococcus strains.<br>
[0043]	A shuttle vector of the invention includes a DNA fragment<br>
(A) as the aforementioned plasmid or portion thereof, and a DNA region (B) which is replicable in E. coli. In some cases it is preferred for the shuttle vector to comprise a DNA region including a drug resistance gene. In the relevant technical field, a "shuttle vector" is a vector which comprises the DNA replication mechanism for two different cell types, and preferably also a drug resistance gene or the like as a selective marker, allowing its auto-replication in the two different cell types. The DNA fragment (A) as the aforementioned plasmid or portion thereof is a DNA region that is replicable in the genus Rhodococcus. The DNA region (B) which is replicable in E. coli may be a full plasmid or a portion thereof, so long as it can be replicated and amplified in E. coli. As such DNA regions that are replicable in E. coli there may be used, for example, the plasmids pUC18, pHSG299 and pHSG398.<br>
[0044]	When the shuttle vector of the invention includes a drug<br>
resistance gene, the preferred ones are ampicillin resistance gene, kanamycin resistance gene and chloramphenol resistance gene, but there are no particular restrictions on the manner of drug so long as the gene is expressed in the genus Rhodococcus and E. coli as hosts and confers drug resistance to the host cells, in order to allow verification of the presence of plasmids in the two genera based on resistance to the drug.   Also, a plurality of such drug resistance genes may be used in<br><br>
combination.<br>
[0045]	The shuttle vector preferably contains multiple cloning<br>
sites (multicloning sites), and the cloning sites and drug resistance gene may be induced from, for example, an E. coli plasmid. That is, a publicly known E. coli plasmid such as one listed above may be cleaved with an appropriate restriction endonuclease and a DNA region containing the cloning sites and drug resistance gene constructed and ligated with another DNA fragment (a DNA region which is replicable in the genus Rhodococcus).<br>
[0046]	As an illustration, outline of shuttle vector constructions is<br>
shown in Figs. 3 to 5. The shuttle vectors may be constructed by treating the aforementioned plasmids and E. coli plasmids with suitable restriction endonucleases and then ligating them. In this manner, the present inventors constructed 18 shuttle vectors (Table 5) using the Rhodococcus plasmids pRETlOOO, pRETl 100 or pRET1200, and the E. coli plasmids pUC18, pHSG299 or pHSG398.<br>
[0047]	The shuttle vectors of the invention are replicable in the<br>
genus Rhodococcus and E. coli as hosts, and are industrially usefiil. The Rhodococcus and E. coli strains transformed by the shuttle vectors of the invention, as well as other microbial transformants, are useful in this way and such transformants are also encompassed by the scope of the invention.<br>
[0048]	A vector of the  invention is characterized by being<br>
constructed using a shuttle vector of the invention. Specifically, it is a vector having target DNA inserted therein which is to be introduced into the shuttle vector of the invention. The DNA to be introduced and<br>
'^ &gt;'.    'irf<br><br>
the shuttle vector of the invention are treated with appropriate restriction endonucleases and then ligated them to construct the vector. The vector may then be used to obtain transformants having the desired DNA transferred therein.<br>
[0049]	As  examples  of DNA to be inserted there may be<br>
mentioned aminoketone asymmetric reductase genes and coenzyme-<br>
regenerating system enzyme genes. Aminoketone asymmetric<br>
reductase genes are genes coding for aminoketone asymmetric<br>
reductases as described in WO02/070714, and more specifically, DNA<br>
coding for a protein comprising the amino acid sequence set forth as<br>
SEQ ID NO: 78 (aminoketone asymmetric reductase derived from R.<br>
erythropolis MAK-34), and particularly DNA comprising the<br>
nucleotide sequence set forth as SEQ ID NO: 79. The entirety of the<br>
content described in WO02/070714 is incorporated herein by reference.<br>
[0050]	An aminoketone asymmetric reductase is any having the<br>
properties described in WO02/070714, and includes a protein having<br>
the amino acid sequence set forth as SEQ ID NO: 78 of the Sequence<br>
Listing, as well as proteins having amino acid sequences obtained by<br>
deletion, insertion, substitution or addition of one or more amino acids<br>
in the aforementioned amino acid sequence, and exhibiting<br>
aminoketone asymmetric reduction activity, Aminoketone asymmetric<br>
reduction activity is activity of producing an optically active<br>
aminoalcohol represented by general formula (2) above using an a-<br>
aminoketone represented by general formula (1) above as the substrate.<br>
[0051]	There are no particular restrictions on the methods of<br>
deletion, insertion, substitution and addition, and any publicly known<br><br>
methods may be employed. For example, there may be mentioned the<br>
methods described in "Zoku Seikagaku Jikken Kouza 1, Idenshi<br>
Kenkyuhou 11", edited by the Japanese Biochemical Society, pi05<br>
(Hirose, S.), Tokyo Kagaku Dojin (1986); "Shin Seikagaku Jikken<br>
Kouza 2, Kakusan III (Recombinant DNA Technology)", edited by the<br>
Japanese Biochemical Society, p.233 (Hirose, S.), Tokyo Kagaku Dojin<br>
(1992); R.Wu, L. Grossman ed., "Methods in Enzymology", Vol. 154,<br>
p.350 &amp; p.367, Academic Press, New York (1987); R.Wu, L.<br>
Grossman, ed., "Methods in Enzymology", Vol. 100, p.457 &amp; p.468.<br>
Academic Press, New York (1983); J.A. Wells et al., "Gene", Vol. 34,<br>
p.315 (1985); T. Grundstroem et al., "Nucleic Acids Res", Vol. 13,<br>
p.3305 (1985); J. Taylor et al., "Nucleic Acids Res.", Vol. 13, p.8765<br>
(1985); RWu, ed., "Methods in Enzymology", Vol. 155, p.568.<br>
Academic Press, New York (1987); and A.R. Oliphant et al., "Gene",<br>
Vol.44, p. 177 (1986). As specific examples, there may be mentioned<br>
the site-directed mutagenesis method (site-specific mutagenesis<br>
method) utilizing synthetic oligonucleotides, the Kunkel method, the<br>
dNTP[aS] method (Eckstein method), and the region-directed<br>
mutagenesis method using sulfurous acid or nitrous acid.<br>
[0052]	Sugar chains are attached to the majority of proteins, and<br>
substitution of one or a plurality of amino acids can modify the attachment of sugar chains. Thus, the aminoketone asymmetric reductases of the invention also include proteins having the amino acid sequence set forth as SEQ ID NO: 78 of the Sequence Listing and having modifications of sugar chains, so long as they exhibit the aforementioned aminoketone asymmetric reduction activity.<br><br>
[0053]	The aminoketone asymmetric reductases of the invention<br>
may also have modifications of their amino acid residues by chemical<br>
methods, or their derivatives may be enhanced by modification or<br>
partial   degradation   using   peptidase   enzymes   such   as   pepsin,<br>
chymotrypsin, papain, bromelain, endopeptidase and exopeptidase.<br>
[0054]	When the  aminoketone  asymmetric reductases  of the<br>
invention are produced by a gene recombinant method, a fusion protein may be expressed and then converted or processed into a protein having biological activity which is substantially equivalent to a natural aminoketone asymmetric reductase either in vivo or ex vivo. In this case, a fusion production method ordinarily employed for genetic engineering may be used, and the fusion protein may be purified by affinity chromatography or the like, utilizing the fused portion thereof Modification and enhancement of protein structures may be carried out with reference to "Shin Seikagaku Jikken Kouza 1, Tanpakushitsu VII, Tanpakushitsu Kogaku", edited by the Japanese Biochemical Society, Tokyo Kagaku Dojin (1993), by the methods described therein, the methods described in literature cited therein, or methods which are essentially equivalent thereto.<br>
[0055]	The aminoketone asymmetric reductase of the invention<br>
may also differ fi-om naturally occurring forms in the identities of one or more of the amino acid residues or in the positions of one or more of the amino acid residues. The present invention also encompasses deletion analogues with deletion of one or more (for example, 1-80, preferably 1-60, more preferably 1-40, even more preferably 1-20 and especially  1-10) amino  acid residues,  substitution analogues with<br><br>
substitution of one or more (for example, 1-80, preferably 1-60, more<br>
preferably 1-40, even more preferably 1-20 and especially 1-10) amino<br>
acid residues or addition analogues with addition of one or more (for<br>
example, 1-80, preferably 1-60, more preferably 1-40, even more<br>
preferably 1-20 and especially 1-10) amino acid residues peculiar to<br>
natural aminoketone asymmetric reductases. Also encompassed are<br>
enzymes having the domain structure characteristic of natural<br>
aminoketone asymmetric reductases. There may also be mentioned<br>
isomers of the aminoketone asymmetric reductases.<br>
[0056]	So long as the domain structure characteristic of natural<br>
aminoketone asymmetric reductases is maintained, all mutants above are also encompassed among the aminoketone asymmetric reductases of the invention. In addition, it is assumed that enzymes having a primary structural conformation substantially equivalent to natural aminoketone asymmetric reductases of the invention, or a portion thereof, as well as enzymes having biological activity substantially equivalent to natural aminoketone asymmetric reductases, may also be included. Naturally occurring mutants may also be mentioned. The aminoketone asymmetric reductases of the invention may be separated and purified in the manner explained below. The present invention encompasses DNA fragments coding for the aforementioned polypeptides, polypeptides of aminoketone asymmetric reductases having all or some of the natural features, and DNA fragments coding for analogues or derivatives thereof. The nucleotides of the aminoketone asymmetric reductases may be modified (for example, with addition, deletions or substitutions), and such modified forms are<br><br>
also encompassed by the invention.<br>
[0057]	An aminoketone asymmetric reductase gene according to<br>
the invention is a nucleic acid coding for any of the aforementioned<br>
aminoketone asymmetric reductases. As representative examples there<br>
may be mentioned nucleic acid coding for a protein having the amino<br>
acid sequence set forth as SEQ ID NO: 78 of the Sequence Listing, and<br>
especially nucleic acid having the nucleotide sequence set forth as SEQ<br>
ID NO: 79, but since several nucleotide sequences (codons) can code<br>
for each amino acid, there exist numerous nucleic acids coding for a<br>
protein having the amino acid sequence set forth as SEQ ID NO: 78.<br>
Thus, all such nucleic acids are also encompassed among the<br>
aminoketone asymmetric reductase genes of the invention. Here,<br>
"coding for a protein" means that, when the DNA consists of two<br>
strands, one of the two complementary strands has a nucleotide<br>
sequence coding for the protein, and therefore the nucleic acids of the<br>
invention include nucleic acids comprising nucleotide sequences<br>
directly coding for the amino acid sequence set forth as SEQ ID NO:<br>
78 and nucleic acids comprising nucleotide sequences which are<br>
complementary thereto. In addition, the aminoketone asymmetric<br>
reductase genes of the invention may be nucleic acids which hybridize<br>
with nucleic acid comprising a nucleotide sequence complementary to<br>
SEQ ID NO: 79 under stringent conditions, and which code for<br>
proteins with aminoketone asymmetric reduction activity. Here,<br>
"stringent conditions" has the same definition as explained above.<br>
[0058]	The coenzyme-regenerating system enzyme gene may be<br>
one for various dehydrogenases, specifically, glucose dehydrogenase,<br><br>
glucose-6-phosphate dehydrogenase, aldehyde dehydrogenases, alcohol<br>
dehydrogenases, organic acid dehydrogenases and amino acid<br>
dehydrogenases. More specifically, there may be suitably used<br>
acetaldehyde dehydrogenase, ethanol dehydrogenase, propanol<br>
dehydrogenase, glycerol dehydrogenase, formate dehydrogenase,<br>
acetate dehydrogenase, butyrate dehydrogenase, lactate dehydrogenase,<br>
maleate dehydrogenase and glutamate dehydrogenase.<br>
[0059]	A transformant according to the invention is characterized<br>
by comprising the aforementioned vector. The transformant is obtained by introducing the vector into host cells. The vector introduction method may be a publicly known method, such as the calcium phosphate method, lipofection, electroporation, microinjection or the like.<br>
[0060]	For example, a transformant of the invention comprising a<br>
vector having an aminoketone asymmetric reductase gene inserted<br>
therein has aminoketone asymmetric reduction activity, and may be<br>
applied for an aminoketone asymmetric reductase production method<br>
or optically active aminoalcohol production method as described below.<br>
[0061]	The method for production of an aminoketone asymmetric<br>
reductase of the invention is characterized by comprising a culturing<br>
step in which transformants containing a vector having an aminoketone<br>
asymmetric reductase gene inserted therein are cultured in medium<br>
which allows growth of the transformants, and a purification step in<br>
which the aminoketone asymmetric reductase is purified from the<br>
transformants obtained in the culturing step.<br>
[0062]	The  method for culturing may be  a publicly known<br><br>
method with no particular restrictions so long as it permits growth of<br>
the cells used, and ordinarily a liquid medium containing a carbon<br>
source, nitrogen source and other nutrients is used. As carbon sources<br>
for the medium there may be used any of those that can be utilized by<br>
the cells. Specifically, there may be mentioned sugars such as glucose,<br>
fructose, sucrose, dextrin, starch and sorbitol, alcohols such as<br>
methanol, ethanol and glycerol, organic acids such as fiimaric acid,<br>
citric acid, acetic acid and propionic acid, and their salts, hydrocarbons<br>
such as paraffin, and mixtures thereof. As nitrogen sources there may<br>
be used any of those that can be utilized by the cells. Specifically,<br>
there may be mentioned ammonium salts of inorganic acids such as<br>
ammonium chloride, ammonium sulfate and ammonium phosphate;<br>
ammonium salts of organic acids such as ammonium fumarate and<br>
ammonium citrate; nitric acid salts such as sodium nitrate and<br>
potassium nitrate; and inorganic or organic nitrogenous compounds<br>
such as meat extract, yeast extract, malt extract and peptone, as well as<br>
mixtures thereof. The medium may also contain appropriately added<br>
nutrient sources ordinarily used for culturing, such as inorganic salts,<br>
trace metal salts and vitamins. When necessary, there may also be<br>
added to the medium substances that promote cell growth and buffering<br>
substances effective for maintaining the pH of the medium.<br>
[0063]	The culturing of the cells may be carried out under<br>
conditions suitable for growth. Specifically, the medium pH may be 3-10, preferably 4-9, and the temperature may be 0-50°C, preferably 20-40°C. The cell culturing may be conducted either under aerobic or anaerobic conditions.   The culturing time is preferably 10-150 hours,<br><br>
but should be appropriately determined for the type of cells used.<br>
[0064]	The culture solution of the cells cultured in the manner<br>
described above is filtered or centrifuged and the cells are rinsed with water or buffer solution. The rinsed cells are suspended in a suitable amount of buffer solution for disruption of the cells. The method of disruption is not particularly restricted but as examples there may be mentioned mechanical disruption with a mortar, Dynomill, French press, ultrasonic cell disruptor or the like. The aminoketone asymmetric reductase in the cell-free extract obtained by filtration or centrifugation of the solid matter from the cell disruptate is recovered by an ordinary enzyme isolating method.<br>
[0065]	There are no particular restrictions on the method for<br>
isolation of the enzyme and any publicly known method may be<br>
employed, but as examples there may be mentioned purification by<br>
salting out such as ammonium sulfate precipitation; gel filtration<br>
methods using Sephadex and the like; ion-exchange chromatography<br>
methods using carriers with diethylaminoethyl groups or<br>
carboxymethyl groups; hydrophobic chromatography using carriers<br>
with hydrophobic groups such as butyl, octyl and phenyl; dye gel<br>
chromatography methods; electrophoresis methods; dialysis;<br>
ultrafiltration methods; affinity chromatography methods; high<br>
performance liquid chromatography methods and the like.<br>
[0066]	The enzyme may also be used as an immobilized enzyme.<br>
There are no particular restrictions on the method and any publicly known method may be employed, among which there may be mentioned immobilization of the enzyme or the enzyme-producing<br><br>
cells, and the immobilization may be accomplished by a carrier bonding method such as a covalent bonding method or adsorption method, a crosslinking method, entrapment method or the like. A condensing agent such as glutaraldehyde, hexamethylene diisocyanate or hexamethylene diisothiocyanate may also be used if necessary. Other immobilizing methods include: a monomer method in which a monomer is gelled by polymerizing reaction; a prepolymer method in which molecules larger than monomers are polymerized; a polymer method in which a polymer is gelled; immobilization using polyacrylamide; immobilization using natural polymers such as alginic acid, collagen, gelatin, agar and K-carrageenan; and immobilization using synthetic polymers such as photosetting resins and urethane polymers.<br>
[0067]	The enzyme purified in this manner is judged as having<br>
been adequately purified if a single band is confirmed in electrophoresis (SDS-PAGE, etc.).<br>
[0068]	A   method   for   production   of   an   optically   active<br>
aminoalcohol according to the invention is characterized to produce an optically active aminoalcohol compound represented by the following general formula (2), which compound exhibits the desired optical activity, by reacting an aminoketone asymmetric reductase obtained by the production method of the invention with an enantiomeric mixture of an a-aminoketone compound represented by the following general formula (1) or a salt thereof [Chemical Formula 8]<br><br><br><br>
[0070]	The substituent X is as follows.   As the aforementioned<br>
halogen there may be mentioned fluorine, chlorine, bromine and iodine.<br>
[0071]	As lower alkyl there are preferred CI-6 alkyl, among<br>
which there may be mentioned methyl, ethyl, propyl, isopropyl, butyl, isobutyl, s-butyl, t-butyl, pentyl, isopentyl, hexyl and the like. These may have straight-chain or branched structures. As substituents they may contain halogens such as fluorine or chlorine, or hydroxyl, alkyl, amino, alkoxy and the like.<br>
[0072]	As protecting groups for hydroxyl optionally protected<br>
with a protecting group there may be mentioned groups that can be removed by treatment with water, groups that can be removed by acid or weak base treatment, groups that can be removed by hydrogenation or groups that can be removed with Lewis acid catalysts and thiourea, and such protecting groups include optionally substituted acyl, optionally substituted silyl, alkoxyalkyl, optionally substituted lower alkyl, benzyl, p-methoxybenzyl, 2,2,2-trichloroethoxycarbonyl, allyloxycarbonyl, trityl and the like.<br>
[0073]	The    aforementioned    acyl    groups    include    acetyl,<br>
chloroacetyl, dichloroacetyl, pivaloyl, benzoyl, p-nitrobenzoyl and the like. They may contain hydroxyl, alkyl, alkoxy, nitro, halogen and the like as substituents. The aforementioned silyl groups include trimethylsilyl, t-butyldimethylsilyl, triarylsilyl and the like. They may contain alkyl, aryl, hydroxyl, alkoxy, nitro, halogen and the like as substituents. The aforementioned alkoxyalkyl groups include methoxymethyl, 2-methoxyethoxymethyl and the like. The aforementioned lower alkyl include CI-6 alkyl, among which there<br><br>
may be mentioned methyl, ethyl, propyl, isopropyl, butyl, isobutyl, s-butyl, t-butyl, pentyl, isopentyl, hexyl and the like. These may have straight-chain or branched structures. As substituents they may contain halogen such as fluorine or chlorine, or hydroxyl, alkyl, amino, alkoxy and the like.<br>
[0074]	X may be nitro or sulfonyl, and specifically there may be<br>
mentioned methylsulfonyl and the like.<br>
[0075]	The number "n"  for X is an integer of 0-3, and is<br>
preferably 0.<br>
[0076]	R' in general formula (1) above represents lower alkyl.<br>
As lower alkyl there are preferred CI-6 alkyl, among which there may be mentioned methyl, ethyl, propyl, isopropyl, butyl, isobutyl, s-butyl, t-butyl, pentyl, isopentyl, hexyl and the like. These may have straight-chain or branched structures.<br>
[0077]	Each of R andR represent hydrogen or lower alkyl. The<br>
lower alkyl include CI-6 alkyl, among which there may be mentioned methyl, ethyl, propyl, isopropyl, butyl, isobutyl, s-butyl, t-butyl, pentyl, isopentyl, hexyl and the like. These may have straight-chain or branched structures.<br>
[0078]	As salts of the aforementioned a-aminoketone compounds<br>
there may be mentioned salts of inorganic acids such as hydrochloride, sulfate, nitrate, phosphate and carbonate, and salts of organic acids such as acetic acid and citric acid.<br>
[0079]	The   a-aminoketone   can   be   easily   synthesized   by<br>
halogenation (for example, bromination) of the a-carbon of a corresponding 1-phenylketone derivative, followed by replacement of<br><br>
the halogen such as bromine with an amine (Ger. (East), 11, 332, Mar.l2, 1956).<br>
[0080]	The optically active aminoalcohol represented by general<br>
formula (2) above according to the invention will now be explained. In general formula (2), X, n, R , R , R and * have the same definitions as in general formula (1) above. As p-aminoalcohols having the desired optical activity there may be mentioned (IS, 2S)aminoalcohols. As specific examples of (IS, 2S)aminoalcohols there may be mentioned d-threo-2-methylamino-l-phenylpropanol (d-pseudoephedrine), d-threo-2-dimethylamino-l-phenylpropanol (d-methylpseudoephedrine), (IS, 2S)-a-(l-aminoethyl)-benzyl alcohol (d-norpseudoephedrine), (IS, 2S)-l-(p-hydroxyphenyl)-2-methylamino-l-propanol, (IS, 2S)-a-(l-aminoethyl)-2,5-dimethoxy-benzyl alcohol, (IS, 2S)-l-(m-hydroxyphenyl)-2-amino-1 -propanol, (IS, 2S)-1 -(p-hydroxyphenyl)-2-amino-1-propanol, (IS, 2S)-l-phenyl-2-ethylamino-l-propanol, (IS, 2S)-1 -phenyl-2-amino-1 -butanol, (IS, 2S)-1 -phenyl-2-methylamino-1 -butanol and the like.<br>
[0081]	The    conditions    for    reaction    of   the    aminoketone<br>
asymmetric reductase are not particularly restricted so long as an optically active aminoalcohol represented by general formula (2) having the desired optical activity is produced, but since the enzyme optimum pH is 8.1 and the optimum temperature is 55°C, the reaction is preferably carried out under conditions of pH 7-9 and 30-65°C temperature.<br>
[0082]	A   method   for   production   of   an   optically   active<br>
aminoalcohol  according to the  invention is  also  characterized to<br><br>
produce an optically active aminoalcohol compound represented by the following general formula (2), which compound exhibits the desired optical activity, by reacting a transformant of the invention with an enantiomeric mixture of an a-aminoketone compound represented by the following general formula (1) or a salt thereof. [Chemical Formula 10]<br><br>
[0083] As the reaction conditions for the reaction described above, for example, the transformants shake cultured in liquid medium may be collected, an aqueous aminoketone solution (0.1-10% concentration) added to the obtained cells, and reaction conducted at a temperature of 20-40°C for a period of several hours to one day while regulating the pH to between 6-8. Upon completion of the reaction, the cells may be separated and the product in the reaction solution isolated to obtain an<br><br>
optically active aminoalcohol. The reaction may be conducted in the same manner for treated transformant cells (dry cells or immobilized cells) or the enzyme or immobilized enzyme obtained from the transformants.<br>
[0084]	In   the   production   method   for   an   optically   active<br>
aminoalcohol of the invention, the reaction may be carried out with ftirther addition of a compound represented by the following general formula (3) or a pharmaceutically acceptable salt or solvate thereof, for more efficient production of the optically active aminoalcohol. [Chemical Formula 121<br><br><br>
(wherein R^ represents hydrogen, CI-3 alkyl or a 5- to 8-membered heterocyclic skeleton containing 1-3 heteroatoms which is bonded to R^ or R^, R^ represents hydrogen, optionally substituted CI-3 alkyl, a C5-<br>
Q<br>
10 hydrocarbon ring which is bonded to R or a 5- to 8-membered heterocyclic skeleton containing 1-3 heteroatoms which is bonded to R^ or R , and R represents hydrogen or optionally substituted CI-6 alkyl); R represents hydrogen, carboxyl, optionally substituted CI-6 alkyl, a 5- to 8-membered heterocyclic skeleton containing 1-3 heteroatoms which is bonded to R"* or a C5-10 hydrocarbon ring which is bonded to R^; R^ represents hydrogen, optionally substituted Cl-6 alkyl, optionally substituted Cl-6 alkyloxycarbonyl, optionally substituted acyl or a 5- to 8-membered heterocyclic skeleton containing 1-3 heteroatoms which is bonded to R^ or R^; and R'° represents hydrogen or optionally substituted Cl-6 alkyl)<br>
[0085]	In general formula (3) above, CI-3 alkyl may be straight-<br>
chain or branched, and specifically there may be mentioned methyl, ethyl, n-propyl, isopropyl and the like.   Cl-6 alkyl may be straight-<br><br>
chain or branched, and specifically there may be mentioned methyl, ethyl, n-propyl, isopropyl, n-butyl, i-butyl, s-butyl, t-butyl, pentyl^ hexyl and the like. As C5-10 hydrocarbon rings there may be mentioned cyclopentyl, cyclohexyl, cycloheptyl, cyclooctyl, cyclononyl, cyclodecanyl and the like.<br>
[0086]	As heteroatoms for the 5- to 8-membered heterocyclic<br>
skeleton containing 1-3 heteroatoms there may be mentioned nitrogen, oxygen, sulfur and the like, among which nitrogen and oxygen are particularly preferred, and as 5- to 8-membered heterocyclic skeletons there may be mentioned pyrrolidine, piperidine, imidazolidine, piperazine, tetrahydrofuran, tetrahydropyran, tetrahydrothiophene^ morpholine and the like.<br>
[0087]	As   CI-6   alkyloxycarbonyl   there   may   be  mentioned<br>
methyloxycarbonyl, ethyloxycarbonyl, isopropyloxycarbonyl, isobutyloxycarbonyl, t-butyloxycarbonyl and the like. As acyl there may be mentioned formyl, acetyl, propionyl, butyryl, isobutyryl, pivaloyl, benzoyl, valeryl and the like. When the aforementioned CI-3 or CI-6 alkyl, CI-6 alkyloxycarbonyl or acyl have substituents there are no particular restrictions on the types, positions and numbers of substituents, and as examples of substituents there may be mentioned halogen such as fluorine and chlorine, hydroxyl, alkyl, carboxyl, amino, alkoxy, nitro, aryl and the like. As pharmaceutically acceptable salts there may be mentioned salts of inorganic acids such as hydrochloric acid, sulfiiric acid, nitric acid and phosphoric acid, salts of organic acids such as acetic acid and citric acid, salts of inorganic bases such as Na, K, Mg, Ca and ammonia, and salts of organic bases such as<br><br>
triethylamine and cyclohexylamine.<br>
[0088]	As   examples   of  compounds   represented   by   general<br>
formula (3) above there may be mentioned l-acetylamino-2-<br>
hydroxypropane, l-methylamino-2-hydroxypropane, l-amino-2-<br>
oxopropane,	l-amino-2-hydroxycyclopentane,	l-amino-2,3-<br>
dihydroxypropane, L-threonine, 4-amino-3-hydroxybutanoic acid, 1-<br>
amino-2-oxocyclohexane, morpholine, 3-hydroxypyrrolidine, 3-<br>
hydroxypiperidine,	2-aminomethyl-tetrahydrofuran,	l-(2-<br>
hydroxypropyl)amino-2-hydroxypropane, 1 -t-butyloxycarbonylamino-<br>
2-hydroxypropane, 2-amino-3-hydroxybutane, DL-serine, l-amino-2-<br>
hydroxypropane,     l-amino-2-hydroxybutane     and	l-amino-2-<br>
hydroxycyclohexane. Compounds among these having asymmetric carbons may be optically active forms or racemic forms, unless otherwise specified.<br>
[0089]	Addition of such activity inducers to the medium can<br>
induce cellular activity and thus more efficiently promote production of the optically active p-aminoalcohol than when no such activity inducers are added. The activity inducers may be used alone, or several such activity inducers may be used in admixture. The amount of such activity inducers is preferably 0.01-10 wt% with respect to the medium.<br>
[0090]	The reaction method for production of the {3-aminoalcohol<br>
of the invention is not particularly restricted so long as it is a method in which the cells or the cell-produced enzyme is reacted with an enantiomeric mixture of an a-aminoketone compound represented by general formula (1) above or its salt, to produce the corresponding<br><br>
optically active P-aminoalcohol compound represented by general formula (2), and the reaction is initiated by mixing the cells rinsed with buffer solution or water with the a-aminoketone aqueous solution used as the starting material.<br>
[0091]	The reaction conditions may be selected within a range<br>
that does not impede production of the optically active p-aminoalcohol compound represented by general formula (2). The cell volume is preferably 1/100 to 1000-fold and more preferably 1/10 to 100-fold in terms of dry weight with respect to the racemic aminoketone. The concentration of the racemic aminoketone substrate is preferably 0.01-20% and more preferably 0.1-10%. The pH of the reaction solution is preferably 5-9 and more preferably 6-8, and the reaction temperature is preferably 10-50°C and more preferably 20-40°C. The reaction time is preferably 5-150 hours, but this may be appropriately determined depending on the cell type.<br>
[0092]	In order to more efficiently promote the reaction, there<br>
may be added sugars such as glucose, organic acids such as acetic acid and energy sources such as glycerol. These may be used alone or as mixtures. The amount of addition is preferably 1/100 to 10-fold with respect to the substrate. Coenzymes and the like may also be added. As coenzymes there may be used nicotinamide adenine dinucleotide (NAD), reduced nicotinamide adenine dinucleotide (NADH), nicotinamide adenine dinucleotide phosphate (NADP), reduced nicotinamide adenine dinucleotide phosphate (NADPH) and the like, either alone or in mixtures, added in amounts of preferably 1/1000 to 1/5 with respect to the racemic aminoketone.    In addition to such<br><br>
coenzymes, there may be added coenzyme-regenerating enzymes such as glucose dehydrogenase, in amounts of 1/1000 to 1/5 with respect to the racemic aminoketone. Also, substrates for coenzyme-regenerating enzymes, such as glucose, may be added, in amounts of 1/100 to 10-fold with respect to the racemic aminoketone. There may also be used combinations of sugars such as glucose, organic acids such as acetic acid, energy sources such as glycerol, coenzymes, coenzyme-regenerating enzymes and coenzyme-regenerating enzyme substrates. These usually accumulate in the cells but if necessary they may be added to increase the reaction speed or yield, and therefore may be added as appropriate.<br>
[0093]	If the reaction solution is reacted with addition of the<br>
specific salts described above under the aforementioned conditions, racemization of the unreacted a-aminoketone isomers will be aided, thus more efficiently promoting conversion to the enantiomer which will serve as the substrate of the cells or cell-produced enzyme. This will tend to yield the target aminoalcohol from the starting material at a high yield of 50% or greater.<br>
[0094]	As  salts  that promote  racemization  of unreacted  a-<br>
aminoketones there may be used weak acid salts such as acetate, tartarate, benzoate, citrate, malonate, phosphate, carbonate, paranitrophenol salt, sulfite and borate, but there are preferably used phosphate (for example, sodium dihydrogen phosphate, potassium dihydrogen phosphate, ammonium dihydrogen phosphate), carbonate (for example, sodium carbonate, sodium hydrogen carbonate, potassium carbonate, ammonium carbonate) and citrate (for example,<br><br>
sodium citrate, potassium citrate, ammonium citrate). Mixtures thereof<br>
may also be used, with a buffer solution with a pH of 6.0-8.0 added to a<br>
final concentration of preferably 0.01-1 M. In the case of a phosphate,<br>
for example, sodium dihydrogen phosphate and sodium monohydrogen<br>
phosphate may be mixed in a proportion of between 9:1 and 5:95.<br>
[0095]	The  optically active  a-aminoalcohol produced by the<br>
reaction may be purified by ordinary separation and purification means. For example, the optically active P-aminoalcohol may be obtained directly fi-om the reaction solution or after separation of the cells, by being subjected to a common purification process such as membrane separation, extraction with an organic solvent (for example, toluene, chloroform, etc.), column chromatography, vacuum concentration, distillation, crystallization, recrystallization or the like. The optical purity of the produced optically active P-aminoalcohol can be measured by high performance liquid chromatography (HPLC). ^xamples<br>
[0096]	The present invention will now be explained in greater<br>
detail through examples, with the understanding that these examples in<br>
no way limit the technical scope of the invention.<br>
[0097]	(Example 1) Isolation and purification of plasmids<br>
(1) Method<br>
Rhodococcus strains were inoculated to 5 mL of GPY medium (1% glucose, 0.5% bactopeptone, 0.3% yeast extract) and cultured with shaking at 25°C. After adding 250 pL of a 100 mg/mL ampicillin solution in the logarithmic growth phase, culturing was continued at 25°C   for 2  hours  with  shaking.     The  cells  were  harvested  by<br><br>
centrifligation (12 krpm, 5 min), and after removing off the supernatant,<br>
they were suspended in 1 mL of 50 mM Tris (pH 7.5), the cells were<br>
again harvested by centrifligation (12 krpm, 5 min) and the supernatant<br>
was removed off. They were then suspended in 250 |aL of a 10 mg/mL<br>
lysozyme solution dissolved in TE solution (10 mM Tris (pH 7.5), 1<br>
mM EDTA), and the suspension was allowed to stand at 37°C for 30<br>
minutes. Next, 100 p,L of 3 M sodium chloride and 25 JJL of 10% SDS<br>
were added and the mixture was allowed to stand at -20 °C overnight.<br>
To the supernatant from centrifligation (12 krpm, 5 min) there were<br>
added 0.5 |iL each of 50 ^g/mL Proteinase K and 50 ^g/mL RNase A,<br>
and the mixture was allowed to stand at 37°C for 15 minutes. An<br>
equivalent of phenol/chloroform/isoamyl alcohol solution was added<br>
and centrifligation was performed (12 krpm, 5 min). A 2.5-fold<br>
amount of ethanol was added to the supernatant, the mixture was<br>
centrifliged (12 krpm, 5 min), and the precipitate was dissolved in 50<br>
]xL of steriUzed water. Confirmation of plasmids was accomplished by<br>
electrophoresis with 0.8% agarose gel and staining with ethidium<br>
bromide, followed by UV irradiation.<br>
[0098]	(2) Test bacteria strains and results<br>
Throughout the examples, the presence or absence of plasmids was screened fi-om available strains belonging to the genus Rhodococcus and its related genus Mycobacterium followed the method described in (1) above.<br>
[0099]	Table 3 shows the screened strains confirmed to contain<br>
plasmids.        Specifically,    Rhodococcus    erythropolis    (IAM1400, IAM1503,  JCM2893,  JCM2894  and JCM2895)  and Rhodococcus<br><br>
rhodnii   (JCM3203)    were    confirmed   to    contain   plasmids    of<br>
approximately 5.4 kbp and 5.8 kbp, respectively. These plasmids were<br>
designated according to the names listed in Table 3: pRETllOO,<br>
pRET1200,     PRET1300,     pRETMOO,     pRET1500,     pRET1600,<br>
PRET1700, pRET1800, pRET0500, pRETlOOO (see Table 3).<br>
[0100]	R. erythropolis IAM1400 and IAM1503 are described in<br>
"lAM Catalogue of Strains, Third Edition, 2004" published by the Institute of Molecular and Cellular Biosciences, The University of Tokyo, and are available from the institute. Also, R. erythropolis JCM2893, JCM2894 and JCM2895 and R. rhodnii JCM3203 are described in "JCM Catalogue of Strains, Eighth Edition 2002" published by RIKEN, Japan, and are available from the institute.<br><br>
Various restriction endonucleases were used to determine restriction endonuclease sites, for classification of the plasmids shown in Table 3.   Each plasmid was isolated by the method described in<br><br>
Example 1, and then digested with EcoR I, Hind III, Pvu II, Sea I, Sph I, Sma I, Sac I, BamH I and Xpw I, and electrophoresed on 0,8% agarose gel for confirmation of the DNA fragments. The size marker used was Loading Quick DNA size Marker X/EcoR I+i/md III double digest (Toyobo). The numbers of sites cleaved by the restriction endonucleases and the sizes of the fragments were determined based on the size marker. The results are shown in Table 4.<br><br>
[0104]	Based on the analysis results shown above, the plasmids in<br>
Table 3 were classified into three types: plasmids possessing the same<br>
restriction endonuclease sites as pRETllOO, plasmids possessing the<br>
same restriction endonuclease sites as pRET1200, and pRETlOOO.<br>
[0105]	(Example 3) Plasmid sequencing and homology search<br>
As the plasmids were classified into three types, i.e. pRETlOOO, pRETllOO and pRET1200 based on the results of Example 2, it was attempted to sequence each of the plasmids.<br>
[0106]	First, the DNA fragments of the plasmids were cloned for<br>
determination   of  the   nucleotide   sequences.      For   Rhodococcus<br><br>
erythropolis (JAM 1400), the plasmids (pRETllOO, pRET1200) were isolated and digested with Sma I and Sac I. Upon electrophoresis on 0.8% agarose gel, DNA fragments with sizes of approximately 0.5 kbp, approximately 1.7 kbp, approximately 3.7 kbp and approximately 4.9 kbp were confirmed. The respective DNA fragments were recovered from the agarose gel using a GFX™ PCR DNA and Gel Band Purification Kit (Amersham Bioscience) and used as insert DNA. Separately, pBluescript II KS(-) was used after digesting with Sma I alone or with Sma I and Sac I, as vector DNA. The insert DNA and vector DNA were ligated with Ligation High (Toyobo) and used to transform E. coli JM109. The obtained transformants were screened using a GFX Micro Plasmid Prep Kit (Amersham Bioscience) to obtain different clones.<br>
[0107]	For   Rhodococcus   rhodnii   (JCM3203),   the   plasmid<br>
(pRETlOOO) was isolated and then digested with BamR I. Upon electrophoresis on 0.8% agarose gel, DNA fragments with sizes of approximately 2.0 kbp and approximately 3.8 kbp were confirmed. The respective DNA fragments were recovered from the gel using the aforementioned Kit and used as insert DNA. The vector DNA used was pBluescript II KS(-) digested with BamH I,<br>
[0108]	Determination of the nucleotide sequences of the plasmid<br>
inserts  was  accomplished  by  the  primer walking method.     The<br>
apparatus used was an ABI PRISM™ 310NT Genetic Analyzer, and<br>
the enzyme used was a BigDye Terminator v3.1 Cycle Sequencing Kit<br>
(ABI).<br>
[0109]	First, P7 (Ml3 forward, Toyobo) and P8 (Ml3 reverse,<br><br>
Toyobo) primers were used for partial decoding of the insert nucleotide<br>
sequences. Next, primers were designed within the decoded sequence<br>
(using the sequence analyzing software DNASIS Pro; Hitachi Software<br>
Corp.), and the designed primers (synthetic oligo DNA) were used for<br>
fiirther decoding of the nucleotide sequence. This procedure was<br>
repeated until decoding of the entirety of each insert nucleotide<br>
sequence. Upon completion of the insert nucleotide sequence decoding,<br>
primers were designed for reaction fi-om the ends of each insert to the<br>
vector direction in order to analyze how the inserts were linked, and<br>
PCR was conducted (using KOD -plus-), using the plasmid isolated<br>
fi-om Rhodococcus etythropolis (IAM1400) as template. The PCR<br>
product was purified using a GFX™ PCR DNA and Gel Band<br>
Purification Kit, and sequencing was carried out using the same<br>
primers used for PCR, to analyze the arrangement of the inserts.<br>
[0110]	The   results   of   sequencing   showed   that   pRETllOO<br>
consisted of 5444 bp, with a G+C content of 59%. The fiill determined<br>
nucleotide sequence is set forth as SEQ ID N(|; 73 )of the Sequence<br>
Listing. Plasmid pRET1200 consisted of 5421 bp and had a G+C<br>
content of 62%. Plasmid pRETlOOO consisted of 5813 bp and had a<br>
G+C content of 67%. The full determined nucleotide sequence is set<br>
forth as SEQ ID NO: 74 of the Sequence Listing.<br>
[0111]	A   homology   search   for   the   determined   nucleotide<br>
sequences using DNASIS Pro revealed that pRETlOOO and pRETllOO were novel plasmids. On the other hand, : pRET1200 had approximately 99.6% homology with pN30 (GenBank accession no. AF312210) (calculated based on pRET1200).<br><br>
[0112]	For pRETlOOO and pRETllOO, comparison was made<br>
with publicly known plasmids based on the determined nucleotide<br>
sequences, using DNASIS Pro.   As a result, neither of the plasmids<br>
were found to have completely matching restriction endonuclease sites<br>
with other plasmids.<br>
[0113]	(Example 4) Nucleotide sequence analysis<br>
The results of analysis of the nucleotide sequences of pRETl 100<br>
and pRETlOOO are shown below.<br>
[0114]	The following orfs were found in pRETl I GO:<br>
orfl (SEQ ID NO: 1, SEQ ID NO: 2 or SEQ ID NO: 3) consisting of the nucleotide sequence from bases 202, 238 or 337 to 480 of the nucleotide sequence set forth as SEQ ID NO: 73;<br>
orfZ (SEQ ID NO: 4) consisting of the nucleotide sequence from bases 477 to 758 of the nucleotide sequence set forth as SEQ ID NO: 73;<br>
orf3 (SEQ ID NO: 5, SEQ ID NO: 6, SEQ ID NO: 7, SEQ ID NO: 8, SEQ ID NO: 9, SEQ ID NO: 10, SEQ ID NO: 11, SEQ ID NO: 12, SEQ ID NO: 13, SEQ ID NO: 14, SEQ ID NO: 15 or SEQ ID NO: 16) consisting of the nucleotide sequence from bases 862, 1294, 1450, 1462, 1486, 1489, 1513, 1630, 1645, 1687, 2224 or 2227 to 2409 of the nucleotide sequence set forth as SEQ ID NO: 73;<br>
orf4 (SEQ ID NO: 17, SEQ ID NO: 18, SEQ ID NO: 19, SEQ ID NO: 20 or SEQ ID NO: 21) consisting of the nucleotide sequence complementary to the nucleotide sequence from bases 1875, 1734, 1701, 1674 or 1581 to 1444 of the nucleotide sequence set forth as SEQ ID NO: 73;<br><br>
orf5 (SEQ ID NO: 22, SEQ ID NO: 23, SEQ ID NO: 24, SEQ ID NO: 25 or SEQ ID NO: 26) consisting of the nucleotide sequence complementary to the nucleotide sequence from bases 2828, 2792, 2747, 2594 or 2540 to 2406 of the nucleotide sequence set forth as SEQ ID NO: 73;<br>
orf6 (SEQ ID NO: 27 or SEQ ID NO: 28) consisting of the nucleotide sequence from bases 2971 or 3049 to 3306 of the nucleotide sequence set forth as SEQ ID NO: 73;<br>
orf7 (SEQ ID NO: 29 or SEQ ID NO: 30) consisting of the nucleotide sequence complementary to the nucleotide sequence from bases 3577 or 3571 to 3053 of the nucleotide sequence set forth as SEQ ID NO: 73;<br>
orfS (SEQ ID NO: 31 or SEQ ID NO: 32) consisting of the nucleotide sequence from bases 3339 or 3648 to 3902 of the nucleotide sequence set forth as SEQ ID NO: 73; and<br>
orf9 (SEQ ID NO: 33 or SEQ ID NO: 34) consisting of the<br>
nucleotide sequence from bases 4366 or 4477 to 5034 of the nucleotide<br>
sequence set forth as SEQ ID NO: 73.<br>
[0115]	The following orfs were found in pRETl 000:<br>
orflO (SEQ ID NO: 38, SEQ ID NO: 39, SEQ ID NO: 40 or SEQ ID NO: 41) consisting of the nucleotide sequence complementary to the nucleotide sequence from bases 3350, 3251, 2945 or 2849 to 2412 of the nucleotide sequence set forth as SEQ ID NO: 74;<br>
orfll (SEQ ID NO: 42 or SEQ ID NO: 43) consisting of the nucleotide sequence complementary to the nucleotide sequence from bases 2365 or 2332 to 2159 of the nucleotide sequence set forth as SEQ<br><br>
ID NO: 74;<br>
orfl2 (SEQ ID NO: 44) consisting of the nucleotide sequence from bases 3197 to 3526 of the nucleotide sequence set forth as SEQ ID NO: 74;<br>
orfl3 (SEQ ID NO: 45 or SEQ ID NO: 46) consisting of the nucleotide sequence complementary to the nucleotide sequence from bases 4035 or 3996 to 3679 of the nucleotide sequence set forth as SEQ ID NO: 74;<br>
orfl4 (SEQ ID NO: 48, SEQ ID NO: 49 or SEQ ID NO: 50) consisting of the nucleotide sequence from bases 4621, 4654 or 4666 to 4830 of the nucleotide sequence set forth as SEQ ID NO: 74;<br>
orfl5 (SEQ ID NO: 51 or SEQ ID NO: 52) consisting of the nucleotide sequence complementary to the nucleotide sequence from bases 5161 or 5062 to 4709 of the nucleotide sequence set forth as SEQ ID NO: 74;<br>
orfl6 (SEQ ID NO: 53 or SEQ ID NO: 54) consisting of the nucleotide sequence from bases 2331 or 2334 to 2618 of the nucleotide sequence set forth as SEQ ID NO: 74;<br>
orfl7 (SEQ ID NO: 55) consisting of the nucleotide sequence from bases 2907 to 3242 of the nucleotide sequence set forth as SEQ ID NO: 74;<br>
orfl8 (SEQ ID NO: 56, SEQ ID NO: 57, SEQ ID NO: 58, SEQ ID NO: 59 or SEQ ID NO: 60) consisting of the nucleotide sequence from bases 1650, 1689, 1713, 1827 or 1875 to 2162 of the nucleotide sequence set forth as SEQ ID NO: 74;<br>
orfl9 (SEQ ID NO: 61) consisting of the nucleotide sequence<br><br>
from bases 1906 to 2169 of the nucleotide sequence set forth as SEQ ID NO: 74;<br>
orfZO (SEQ ID NO: 62) consisting of the nucleotide sequence complementary to the nucleotide sequence from bases 810 to 553 of the nucleotide sequence set forth as SEQ ID NO: 74;<br>
orf21 (SEQ ID NO: 63, SEQ ID NO: 64, SEQ ID NO: 65, SEQ ID NO: 66, SEQ ID NO: 67, SEQ ID NO: 68 or SEQ ID NO: 69) consisting of the nucleotide sequence from bases 117, 147, 306, 456, 5144, 5276 or 5534 to 656 of the nucleotide sequence set forth as SEQ ID NO: 74.<br>
[0116]	The DNA replication region of pRETllOO is the region<br>
represented by the nucleotide sequence set forth as SEQ ID NO: 35 (from bases 2410 to 3200), the nucleotide sequence set forth as SEQ ID NO: 36 (from bases 1000 to 1500) or the nucleotide sequence set forth as SEQ ID NO: 37 (from bases 5000 to 500). The DNA replication region of pRETlOOO is the region represented by the nucleotide sequence set forth as SEQ ID NO: 70 (from bases 3355 to 3507), the nucleotide sequence set forth as SEQ ID NO: 71 (from bases 4290 to 4350) or the nucleotide sequence set forth as SEQ ID NO: 72 (from bases 3570 to 3894).<br>
[0117]	The region of the nucleotide sequence from bases 5144 to<br>
656 (SEQ ID NO: 67) and the region of the nucleotide sequence from<br>
bases 4381 to 4830 (SEQ ID NO: 47) of the nucleotide sequence of<br>
pRETlOOO (SEQ ID NO:  74)  are homologous  with mobilization<br>
proteins, suggesting that they are involved in mobilization.<br>
[0118]	A DNA secondary structure is predicted for the region of<br><br>
the nucleotide sequence from bases 4260 to 4339 (SEQ ID NO: 75) of the nucleotide sequence of pRETlOOO (SEQ ID NO: 74), and it is presumably involved in expression of the mobilization protein gene or is the recognition site of the expressed protein.<br>
[Oil 9]	On the other hand, it was suggested that the region of the<br>
nucleotide sequence from bases 761 to 868 (SEQ ID NO: 76) of the<br>
nucleotide sequence of pRETUOO (SEQ ID NO: 73) is a promoter<br>
involved in expression of a protein related to replication.<br>
[0120]	(Example 5) Construction of shuttle vectors<br>
For construction of a shuttle vector    between Rhodococcus<br>
strains and E. coli, the Rhodococcus plasmids pRETlOOO, pRETllOO<br>
and pRET1200  and the E.   coli plasmids pUC18, pHSG299  and<br>
pHSG398 were used for the following experiment.<br>
[0121]	First, DNA fragments were prepared from R. erythropolis<br>
plasmids. Specifically, plasmids pRETllOO and pRET1200 were obtained from R. erythropolis (IAM1400), and then AlwAA I was used for digestion of pRETllOO at 37°C for 2 hours and Blunting High (Toyobo) was used for blunting of the ends, while BsplJJl 11 was used for digestion of pRET1200 at 48°C for 2 hours and Blunting High (Toyobo) was used for blunting of the ends, to obtain DNA fragments oiR. erythropolis plasmid. Each of the DNA fragments was dissolved in TE solution.<br>
[0122]	For pRETlOOO, plasmid pRETlOOO was obtained from R.<br>
rhodnii (JCM3203), and then Drd I was used for digestion of pRETlOOO at 37°C for 2 hours and Blunting High was used for blunting of the ends, to obtain pRETlOOO DNA fragments, which were<br><br>
dissolved in TE solution.<br>
[0123]	Next, DNA fragments were prepared from the E. coli<br>
plasmids. Specifically, pUC18 (containing the ampicillin-resistance<br>
gene (Amp"^)) was digested with Sma I at 30°C for 2 hours, and<br>
pHSG299 (containing the kanamycin-resistance gene (Km^)) and<br>
pHSG398 (containing the chloramphenicol-resistance gene (Cm"^))<br>
were digested with Hinc II at 37°C for 2 hours to obtain DNA<br>
fragments ofE. coli plasmid, which were dissolved in TE.<br>
[0124]	After ligating the DNA fragments from the Rhodococcus<br>
and E. coli plasmids prepared in the manner described above, they were used for transformation in E. coli DH5a, which were plated on LB (1% tryptophan, 0.5% yeast extract, 1% sodium chloride; pH 7.2) agar medium containing 100 )ig/mL kanamycin, 100 ^g/mL ampicillin or 30 |ig/mL chloramphenicol, coated with 30 (iL of 0.1 M IPTG (isopropyl-p-galactoside) and 4% X-gal (5-bromo-4-chloro-3-indole-P-D-galactopyranoside) and allowed to stand at 30°C for 60 hours. White colonies were selected from among the appearing colonies, and were cultured with shaking in LB liquid medium containing 100 |j,g/mL kanamycin, 100 |ig/mL ampicillin or 30 jig/mL chloramphenicol, at 30°C for 60 hours. The DNA was purified from the obtained culture solution using a GFX^'^ Micro Plasmid Prep Kit (Amersham Bioscience, with purification under the manufacturer's specified conditions). The obtained DNA was confirmed by electrophoresis on 0.8% agarose gel. The obtained shuttle vectors are shown in Table 5, and the methods for constructing each of the shuttle vectors using pRETl 100 are shown in Figs. 3 to 5.<br><br><br>
pUC18, pHSG299 or pHSG398 were designated respectively as<br>
pRETllOl (SEQ ID NO: 89), pRET1102 (SEQ ID NO: 90) or<br>
pRET1103 (SEQ ID NO: 91), respectively. Of the shuttle vectors, ^<br>
pRETllOl exhibits ampicillin resistance, pRET1102 exhibits<br>
kanamycin resistance and pRETllOS exhibits chloramphenicol<br>
resistance. Also, the shuttle vectors pRETl 101 to 1103 wherein the E.<br>
coli gene and pRETllOO were linked in reverse (Rv))were designated<br>
respectively as pRETllOlRv (SEQ ID NO: 92), pRET1102Rv (SEQ<br>
ID NO: 93) and pRETl 103Rv (SEQ ID NO: 94).<br>
[0127]	Similarly, the shuttle vectors constructed using pRETlOOO<br>
and pRET1200 were designated as pRET1001-pRET1003 (SEQ ID<br>
NO: 95-SEQ ID NO: 97) and pRET1001Rv-pRET1003Rv (SEQ ID<br>
NO:   98-SEQ  ID NO:   100),   and  as  pRET1201-pRET1203   and<br>
pRET1201Rv-pRET1203Rv (Table 5).<br>
[0128]	(Example 6) Examining method of transformation to R.<br>
erythropolis<br><br>
The Rhodococcus-E. coli shuttle vectors obtained in Example 5 were used for transformation oiR. erythropolis MAK-34 strain (MAK-34; deposited at the National Institute of Bioscience and Human-Technology, National Institute of Advanced Industrial Science and Technology, Ministry of Economy, Trade and Industry, (currently: International Patent Organism Depositary, National Institute of Advanced Industrial Science and Technology) on February 15, 2001 as PERM BP-7451).j Electroporation was investigated as the method of gene transfer.<br>
[0129]	First, R. erythropolis MAK-34 strain was inoculated to 5<br>
mL of GPY medium and cultured with shaking at 30°C for 36 hours.<br>
After seeding 1 mL of culture solution in 100 mL of LB medium,<br>
culturing was continued at 200 rpm at 30°C for 10 hours. The cultured<br>
cells were harvested by centrifugation (12 krpm, 5 min, 4°C) and the<br>
harvested cells were rinsed twice with ultrapurified water. The rinsed<br>
cells were harvested by centrifugation (12 krpm, 5 min, 4°C) and<br>
suspended in 2.4 mL of a 10% glycerol solution. The suspension was<br>
dispensed into 300 \x\ portions and frozen at -80°C as competent cells.<br>
[0130]	A 90 i^L portion of the prepared competent cells and a 5<br>
^L portion of the shuttle vector (pRETlOOl, pRET1002, pRET1003, pRETUOl, pRET1102, pRET1103, pRET1201, pRET1202 or pRET1203) were mixed on ice. The mixed solution was gently poured into a 0.1 cm cuvette which had been cooled on ice, and was set in a Gene Pulser II Electroporation System (BIO-RAD). After pulsing at 20 kV/cm, 400 Q, 25 ^iF, the mixed solution was added with 300 piL of LB medium immediately and was allowed to stand at 25°C for 3 hours.<br><br>
[0131]	A portion of the  cell  suspension was  plated on an<br>
antibiotic-containing LB plate (100 ^ig/mL kanamycin, 100 |j,g/mL ampicillin or 30 [ig/mL chloramphenicol). As a result, colonies were obtained when using pRET1002, pRETl 102 and pRET1202 containing the kanamycin resistance gene. In order to confirm that the obtained colonies contained the plasmids, the plasmids were isolated and all were verified to contain the shuttle vector.<br>
[0132]	This suggested that R. erythropolis can be transformed by<br>
electroporation and that pRET1002, pRET1102 and pRET1202 function as shuttle vectors.<br>
[0133]	(Example 7) Obtaining aminoketone asymmetric reductase<br>
gene {mak gene)<br>
The mak gene was isolated from R. erythropolis MAK-34 strain<br>
for insertion of the mak gene into the shuttle vector shown in Fig. 5.<br>
[0134]	First, genomic DNA was obtained from R. erythropolis<br>
MAK-34 strain. After inoculating R. erythropolis MAK-34 strain to 5 mL of GPY medium, culturing with shaking was performed at 30°C for 48 hours, and then the culture solution was seeded in 100 mL of GPY medium and subcultured at 200 rpm at 30°C for 10 hours. The genomic DNA was obtained using a Genomic DNA Buffer set and Genomic-tip 500/G (QIAGEN).<br>
[0135]	The obtained genomic DNA was used as template for PCR<br>
using KOD -plus-. The primers used were MAKFl (5'-GAATCTTCTCGTTGATGCAGATCAGGTC-3'; SEQ ID NO: 80) and MAKR2 (5'-CTGACTCCGTAGTGTTCTGCCAGTTC-3'; SEQ ID NO:  81), for PCR at an annealing temperature of 68°C and<br><br>
extension reaction for 1 minute and 50 seconds. The obtained PCR<br>
product was subjected to phenol/chloroform treatment and ethanol<br>
precipitation, and then mixed with pUC18 that had been digested with<br>
Sma I for 2 hours at 30°C, and ligated therewith using Ligation High.<br>
Competent High (Toyobo) was used for transformation of E. coli<br>
DH5a, which was then plated on LB agar medium (containing 100<br>
^g/mL ampicillin) that had been coated with 30 fiL of 0.1 MIPTG and<br>
4% X-gal, and was allowed to stand at 30°C for 60 hours. White<br>
colonies were selected from among the appearing colonies, and were<br>
cultured with shaking in LB liquid medium containing 100 ^.g/mL<br>
ampicillin at 30°C for 60 hours. The DNA was purified from the<br>
obtained culture solution using a GFX^^ Micro Plasmid Prep Kit. The<br>
obtained DNA was confirmed by electrophoresis on 0.8% agarose gel.<br>
The obtained clone was designated as pMAK-1.<br>
[0136]	(Example 8) Construction of expression vector - 1<br>
A promoter and aminoketone asymmetric reductase gene {mak<br>
gene) were inserted into the shuttle vector shown in Table 5.<br>
[0137]	First, an expression vector (without exogenous promoter)<br>
containing approximately 400 bp upstream from the mak gene was constructed.<br>
[0138]	pMAK-1 was digested with Sma I at 3 0°C for 2 hours, and<br>
then with Pst I at 37°C for 2 hours. The solution was supplied for 0.8% agarose gel electrophoresis. The DNA size marker used was Loading Quick DNA size Marker X/EcoR l+Hind III double digest. After electrophoresis, an approximately 1.4 kbp DNA fragment was purified using a GFX™ PCR DNA and Gel Band Purification Kit, and<br><br>
used as the insert DNA. On the other hand, the vector used was pRET1102 digested with Hinc II and Pst I at 37°C for 2 hours. The DNA fragments were Hgated with Ligation High and Competent High was used for transformation of E. coli DH5a. The cells were plated on LB agar medium containing 100 (ig/mL kanamycin and allowed to stand at 30°C for 60 hours.<br>
[0139]	The appearing colonies were cultured with shaking on LB<br>
liquid medium containing 100 |ig/mL kanamycin at 30°C for 60 hours. The DNA was purified from the obtained cultured medium using a Qp-j^TM jy^j(,j.Q piasmid Prep Kit. The obtained DNA was confirmed by 0.8% agarose gel electrophoresis.<br>
[0140]	For  screening,  the  obtained  DNA  without  restriction<br>
endonuclease treatment and the DNA after digestion with Pst I at 37°C<br>
for 2 hours were subjected to 0.8% agarose gel electrophoresis, and the<br>
target plasmid was obtained based on the size of the DNA. The size<br>
marker used was Loading Quick DNA size Marker X/EcoR l+Hind III<br>
double digest, pRET1102 and pRET1102 that had been digested with<br>
Pst I at 37°C for 2 hours. The plasmid obtained in this manner was<br>
designated as pRETl 104.<br>
[0141]	(Example 9) Construction of expression vector - 2<br>
The shuttle vectors were reduced, since reduction of shuttle vectors is effective for expression vector enhancement, gene modification, transformation efficiency improvement and replication in cells.<br>
[0142]	First,   shuttle  vector  pRET1102  was  reduced.     After<br>
digesting pRET1102 with BamH I and Hinc II for 2 hours, it was<br><br>
electrophoresed on 0.8% agarose gel and an approximately 2.7 kbp DNA fragment was recovered using a GFX™ PCR DNA and Gel Band Purification Kit to prepare a pRET1102 DNA fragment. The size marker used was Loading Quick DNA size Marker X/EcoR l+Hind III double digest.<br>
[0143]	Separately, a DNA fragment replicable in E. coli was<br>
prepared by digesting pHSG299 with BamH I and Hinc II for 2 hours, subjecting it to 0.8% agarose gel electrophoresis, and recovering an approximately 2.7 kbp DNA fragment using a GFX™ PCR DNA and Gel Band Purification Kit.<br>
[0144]	The DNA fragments were ligated with Ligation High and<br>
Competent High was used for transformation of £■. coli JM109 cells,<br>
which were then plated on LB agar medium, containing 100 )J,g/mL<br>
kanamycin, that had been coated with 30 ^iL of 0.1 MIPTG and 4% X-<br>
gal, and was allowed to stand at 30°C for 48 hours.<br>
[0145]	White colonies were selected from among the appearing<br>
colonies, and were cultured with shaking in LB liquid medium<br>
containing 100 |ig/mL kanamycin at 30°C for 48 hours. The DNA was<br>
purified from the obtained culture solution using a GFX Micro<br>
Plasmid Prep Kit. The reduced shuttle vector of pRETl 102 obtained in<br>
this manner was designated as pRETl 123 (approximately 5.3 kbp).<br>
[0146]	Next,   shuttle   vector   pRET1202   was   reduced.      The<br>
Rhodococcus-derived DNA fragment was prepared by digesting pRET1202 with EcoR I for 2 hours and then with Dra III for 2 hours, using Blunting High for blunting of the ends, performing 0.8% agarose gel electrophoresis, and then recovering an approximately 3.7 kbp<br><br>
DNA fragment using a GFX™ PCR DNA and Gel Band Purification<br>
Kit. The size marker used was Loading Quick DNA size Marker<br>
X/EcoR l+Hind III double digest. The DNA fragment was inserted at<br>
the Hinc II site of pHSG299. After ligation, Competent High was used<br>
for transformation of E. coli DH5a, which was then plated on LB agar<br>
medium, containing 100 \xglmL kanamycin, that had been coated with<br>
30 piL of 0.1 MIPTG and 4% X-gal, and was allowed to stand at 30°C<br>
for 72 hours. White colonies were selected from among the appearing<br>
colonies, and were cultured with shaking in LB liquid medium<br>
containing 100 |ig/mL kanamycin at 30°C for 72 hours. The DNA was<br>
purified from the obtained culture solution using a GFX^'^ Micro<br>
Plasmid Prep Kit. When the plasmid obtained by screening was<br>
digested with Sac I, BamH I, Pst I or EcoR I for 2 hours, all of the<br>
clones had approximately 500 bp clipped at the side of EcoR I site of<br>
the Rhodococcus-dQUwed region. The plasmid was designated as<br>
pRET1204 (approximately 5.9 kbp). It was not possible to obtain a<br>
clone with no clipping of the genus Rhodococcus replication region.<br>
[0147]	The shuttle vector pRET1002 was reduced in a similar<br>
manner to obtain pRET1006 (approximately 4.9 kbp).<br>
[0148]	R. erythropolis was transformed with these three reduced<br>
plasmids, pRET1006, pRETl 123 and pRET1204, and upon confirming<br>
the presence or absence of shuttle vector by the method described in<br>
Example 6, all the shuttle vectors were detected in the transformed<br>
cells. This suggested that the three reduced plasmids pRET1006,<br>
pRETl 123 and pRET1204 are replicated in R. erythropolis.<br>
[0149]	(Example 10) Construction of expression vector - 3<br><br>
An expression vector was constructed by having the mak gene<br>
inserted into the shuttle vector constructed in Example 9.<br>
[0150]	The Pst I site of pRETl 123 constructed in Example 9 was<br>
deleted for cloning of the promoter in the single step. After digesting pRETl 123 with Pst I for 2 hours, Blunting High was used for blunting of the ends and Ligation High was used for ligation. The solution was used to transform E. coli JM109 using Competent High, and culturing was performed on an LB plate containing 100 ng/mL kanamycin at 30°C for 36 hours. The formed colonies were inoculated on LB liquid medium containing 100 lig/mL kanamycin and cultured at 30°C for 24 hours, and then the DNA was purified using a GFX Micro Plasmid Prep Kit to obtain pRETl 132.<br>
[0151]	The obtained pRET1132 was digested with Pst I for 1<br>
hour and then electrophoresed on 0.8% agarose, which resulted in<br>
confirming lack of cleavage of pRETl 132 by Pst I.  As controls there<br>
were used pRET1123 and pRET1132 not digested with Pst I, and<br>
pRETl 123 digested with Pst L<br>
[0152]	(Example 11) Construction of expression vector - 4<br>
A clone was constructed having a promoter and the mak gene inserted in the aforementioned shuttle vector.<br>
[0153]	A clone was constructed having a P^n site upstream from<br>
the mak gene, for insertion of a promoter. The procedure was carried<br>
out in the following manner to obtain a clone having His-Tag added to<br>
the C-terminus of the aminoketone asymmetric reductase. PCR was<br>
conducted with KOD -plus- using the pMAK-1 obtained in Example 7<br>
as	template,	MAKPstF	(5'-<br><br>
GACCACTGCAGATCAATCAACTCTGATGAGGTCC-3'; SEQ ID<br>
NO:	82)	and	MAKHisBglllR	(5'-<br>
CGCTTAGATCTCAGTTCGCCGAGCGCCATCGCCG-3';   SEQ  ID<br>
NO: 83) as primers, with an annealing temperature of 68°C and<br>
extension reaction for 1 minute and 50 seconds. A PCR fragment<br>
(insert) produced by digesting the obtained PCR product with Bgl II at<br>
37°C for 2 hours was ligated with pQE70 (digested with Sph I at 37°C<br>
for 2 hours, blunted with Blunting High and digested with Bgl II at<br>
37°C for 2 hours) using Ligation High, and then Competent High was<br>
used for transformation ofE. coli DH5a cells, which were plated on LB<br>
agar medium containing 100 |ig/mL ampicillin and allowed to stand at<br>
30°C for 60 hours. The appearing colonies were cultured with shaking<br>
on LB liquid medium containing 100 jig/mL ampicillin at 30°C for 60<br>
hours. The DNA was purified using a GFX^^ Micro Plasmid Prep Kit.<br>
The obtained DNA was confirmed by 0.8% agarose gel electrophoresis.<br>
[0154]	For screening, the DNA without restriction endonuclease<br>
treatment and the DNA after digestion with Pst I and Bgl II at 37°C for 2 hours were subjected to 0.8% agarose gel electrophoresis, and the target plasmid was obtained based on the size of the DNA. The plasmid obtained in this manner was designated as pMAK-2. The size marker used was Loading Quick DNA size Marker X/EcoK l+Hind III double digest, pQE70, and pQE70 that had been digested with Bgl II at 37°C for 2 hours.<br>
[0155]	A clone was constructed by inserting the pRET1200 repA<br>
promoter (obtained by PCR amplification using as template a clone of pRET1204   wherein   the   orientation   of   repA   encoded   by   the<br><br>
Rhodococcus-denved DNA fragment was in the same orientation as the<br>
kanamycin resistance gene encoded by pHSG299, and using as primers<br>
P1200rep-Pst5195	(5'-<br>
AGCCGCTGCAGAAGCAACACCGCATCCGCCCATTG-3'; SEQ ID NO: 84) and P7 (5'-CGCCAGGGTTTTCCCAGTCACGAC-3'; SEQ ID NO: 85), with an annealing temperature of 60°C and extension reaction for 1 minute, followed by digestion with EcoR I and Pst I at 37°C for 2 hours) at the EcoR l-Pst I site of pMAK-2 (designated as pMAK-19).<br>
[0156]	Next, PCR was conducted with KOD -plus- using as<br>
template pMAK-19 and as primers pQE70Fl (5'-<br>
GGCGTATCACGAGGCCCTTTCGTCTTCACC-3'; SEQ ID NO: 86)<br>
and	pQE70R1135Bm	(5'-<br>
GGTTGGATCCGTCATCACCGAAACGCGCGAGGCAG-3'; SEQ ID NO: 87), with an annealing temperature of 60°C and extension reaction for 3 minutes. The PCR product was purified from the reaction solution by using a GFX™ PCR DNA and Gel Band Purification Kit and after digestion of the purified PCR product with EcoR I and BamH I for 2 hours, it was electrophoresed on 0.8% agarose gel and the DNA fragment was purified by using a GFX^'^ PCR DNA and Gel Band Purification Kit. The DNA fragment was used as an insert DNA.<br>
[0157]	Separately, a vector to be used as the expression shuttle<br>
vector was obtained by digesting pRETl 132 with EcoR I and BamH I for 2 hours, subjecting the DNA fragment to 0.8% agarose gel electrophoresis and purifying the DNA fragment by using a GFX<br><br>
PCR DNA and Gel Band Purification Kit. After mixing the insert DNA and vector, Ligation High was used for ligating them and Competent High was used for transformation of E. coli JM109 cells, which were plated on an LB plate containing 100 |j,g/mL kanamycin. The obtained colonies were cultured on LB liquid medium containing 100 |ig/mL kanamycin, and then the plasmid DNA was recovered by using a GFX^'^ Micro Plasmid Prep Kit and subjected to 0.8% agarose gel electrophoresis for screening. The size markers used were Loading Quick DNA size Marker X/EcoR l+Hind III double digest and pRET1132. The obtained expression vector was designated as pRET1133.<br>
[0158]	Also, pMAK-19 was digested with EcoR I and Hind III at<br>
37°C for 2 hours, blunted with Blunting High and subjected to 0.8%<br>
agarose gel electrophoresis, and the approximately 1,6 kbp DNA<br>
fragment was purified by using a GFX^^ PCR DNA and Gel Band<br>
Purification Kit. The clone having this fragment inserted at the Hinc II<br>
site of pRETl 102 was designated as pRETl 114.<br>
[0159]	The pRET1133 promoter was also modified.   The mak<br>
gene-expressing promoter encoded in pRET1133 is the repA gene<br>
promoter of pRET1200 and has a length of approximately 800 bp, and<br>
a plasmid was constructed by having approximately 200 bp clipped off<br>
from this promoter. The promoter used for the cloning was prepared<br>
by PCR. Plasmid pRET1200 was used as template, P1204rep-Ec2958<br>
(5'-CGCGGAATTCGACCACCACGCACGCACACCGCAC-3'; SEQ<br>
ID	NO:	88)	and	P1200rep-Pst5195	(5'-<br>
AGCCGCTGCAGAAGCAACACCGCATCCGCCCATTG-3';     SEQ<br><br>
ID NO: 84) were used as primers anH vnn    i<br>
^ ™^'' ^^ ^^O -Pius- was used as the PCR enzyme for PCR at an annealing temperature of 60°C and extension reaction for 50 seconds.  The PCR product was purified by using a GFX- PCR DNA and Gel Band Purification Kit, digested with the restriction endonucleases EcoR I and Pst I for 2 hours, and subjected to 1.6% agarose gel electrophoresis, and the DNA fragment was purified by using a GFX™ PCR DNA and Gel Band Purification Kit.  The DNA fragment was used as the insert DNA.  The nucleotide sequence of the promoter region in the DNA fragment is set forth as SEQ ID NO: 77.<br>
[0160]	Separately, for the vector, pRET1133 was digested with<br>
restriction endonucleases EcoR I and Pst I for 2 hours and subjected to 0.8%) agarose gel electrophoresis, and an approximately 7.2 kbp DNA fragment was purified by using a GFX^^ PCR DNA and Gel Band Purification Kit. The size marker used was Loading Quick DNA size Marker X/EcoR l+Hind III double digest.<br>
[0161]	The insert DNA and vector obtained in this manner were<br>
ligated by using Ligation High, and Competent High was used for transformation of ^. coli JM109 cells, which were plated on an LB plate containing 100 }ig/mL kanamycin. The obtained colonies were cultured on LB liquid medium containing 100 ^ig/mL kanamycin, and then the plasmid DNA was recovered by using a GFX™ Micro Plasmid Prep Kit and subjected to 0.8% agarose gel electrophoresis for screening.    The size markers used were Loading Quick DNA size<br>
Marker X/EcoR l+Hind III double digest and pRET1133.<br>
[0162]	Also, after digesting the obtained DNA with restriction<br><br>
endonucleases EcoRl and Pst I for 2 hours, it was subjected to 1.6% agarose gel electrophoresis and a DNA fragment corresponding to the approximately 600 bp insert DNA was confirmed. The size marker used was a 100 bp DNA Ladder. The expression vector obtained in this maimer was designated as pRETl 138.<br>
[0163]	(Example 12) Preparation of recombinant R. erythropolis<br>
and measurement of enzyme activity<br>
The aforementioned expression vectors pRET1102, pRET1104,<br>
pRETllH and pRET1138 were used for transformation of R.<br>
erythropolis MAK-34 strain and R. erythropolis JCM2895 (provided<br>
by RIKEN Japan), and the enzyme activity was measured. The<br>
aminoketone asymmetric reductase purified from MAK-34 strain has<br>
the abilities to react with 1-2-methylamino-l-phenyl-1-propanone as<br>
described in International Patent Publication WO02/070714, and to<br>
produce d-(lS, 2S)-pseudoephedrine. It was also reacted with 1-2-<br>
dimethylaminopropiophenone, l-amino-2-butanone, etc. and<br>
production of each corresponding P-aminoalcohol was confirmed.<br>
[0164]	The activity assay was conducted by preparing a reaction<br>
solution with a cell density O.D. = 5, 2% glucose and 0.2 M sodium phosphate buffer (pH 6.0), and 3% (IS, 2S)-2-(N-ethylamino)-l-phenyl-1-propanol (EAM) was contained in the reaction as substrate. A synthesis method for EAM is described in J. Am. Chem. Soc, Vol. 50, pp.2287-2292, 1928. The reaction solution was incubated with shaking at 30°C for 16 hours. Confirmation of (IS, 2S)-2-(N-ethylamino)-l-phenyl-1-propanol (EPE), which was P-aminoalcohol as the reaction product, was accomplished by HPLC.   The column used<br><br>
was an Inertsil Ph-3 3.0 x  75 mm, the eluent was aqueous 7%<br>
acetonitrile and 0.05 M sodium phosphate buffer (pH 6.0), and the<br>
detection was carried out with UV (220 nm).<br>
[0165]	The results of the activity assay carried out in this manner<br>
are shown in Table 6.   The pRET1104-introduced recombinant cells<br>
lacking the exogenous promoter region exhibited about the same<br>
activity as the pRETl 102-introduced recombinant cells lacking the mak<br>
gene used as the control, and no recombinant enzyme expression was<br>
found.<br>
[0166]	With transformation of pRETllH into MAK-34 strain,<br>
high specific activity was  found compared to pRET1104.     This<br>
indicated that the pRET1200 repA promoter region inserted into the<br>
vector functions as a promoter.<br>
[0167]	With transformation of pRET1138, the specific activity of<br>
the recombinant R. erythropolis MAK-34 strain was 37.7 |ig/h-mL/O.D.<br>
while the specific activity of the recombinant R. erythropolis JCM2895<br>
was 34.9 ^g/h-mL/O.D., and therefore expression of the enzyme in R.<br>
erythropolis strain was confirmed.<br><br><br>
30°C for 4 days in 100 mL of LB medium containing 100 jig/mL<br>
kanamycin, the cells were harvested by centrifugation at 12,000 rpm<br>
for 5 minutes and the protein having His-tag was purified with The<br>
QIAexpressionist Kit (Qiagen). Specifically, the cells were disrupted<br>
by ultrasonic treatment, the supernatant was obtained by centrifugation,<br>
and the protein was purified with a nickel chelate column. Upon<br>
applying the obtained protein to SDS-PAGE, a band of protein, which<br>
molecular weight is approximately 28,000, was observed. This<br>
molecular weight is roughly equivalent to the molecular weight of the<br>
aminoketone asymmetric reductase described in International Patent<br>
Publication WO02/070714, thus indicating that the aminoketone<br>
asymmetric reductase was produced in the recombinant Rhodococcus<br>
strains.<br>
[0170]	(Example 14) Enzymatic production of P-aminoalcohol<br>
A 0,5 mL portion of reaction solution containing the purified enzyme (0.5 ^ig/mL) obtained in Example 13,5 mM NADPH, 120 mM Tris-HCl (pH 7.5) and 5 mM EAM was reacted at 37°C for 16 hours. The substrate and product (EPE) were analyzed by HPLC. The column used was an Inertsil Ph-3 3,0 x 75 mm, the eluent was aqueous 7% acetonitrile and 0.05 M sodium phosphate buffer (pH 6.0), and the detection was carried out with UV (220 rmi). The results confirmed production of EPE.<br>
[0171]	Similarly, the purified enzyme or the crude enzyme extract<br>
obtained fi"om the recombinant cells cultured as described in Example 13 was reacted with 1-2-dimethylaminopropiophenone and l-amino-2-butanone, etc. and production of the corresponding P-aminoalcohols<br><br><br><br><br><br>
We Claim,<br>
1.	A DNA fragment having at least one nucleotide sequence selected from the group consisting of the nucleotide sequences set forth as SEQ ID NO: 35, SEQ ID NO: 36 and SEQ ID NO: 37.<br>
2.	A plasmid or a partial DNA fragment thereof, characterized by comprising a DNA replication region having at least one nucleotide sequence selected from the group consisting of the nucleotide sequences set forth as SEQ ID NO: 35, SEQ ID NO: 36 and SEQ ID NO: 37.<br>
3.	A DNA fragment having at least one nucleotide sequence selected from the group consisting of the nucleotide sequences set forth as SEQ ID NO: 1, SEQ ID NO: 4, SEQ ID NO: 14, SEQ ID NO: 17 and SEQ ID NO: 22.<br>
4.	A plasmid or a partial DNA fragment thereof, characterized by comprising a coding region for a DNA replication-related protein having at least one nucleotide sequence selected from the group consisting of the nucleotide sequences set forth as SEQ ID NO: 1, SEQ ID NO: 4, SEQ ID NO: 14, SEQ ID NO: 17 and SEQ ID NO: 22.<br>
5.	A plasmid or a partial DNA fragment thereof, characterized by comprising a coding region for a DNA replication-related protein having at least one nucleotide sequence selected from the group consisting of the nucleotide sequences set forth as SEQ ID NO: 1, SEQ ID NO: 4, SEQ ID NO: 14, SEQ ID NO: 17 and SEQ ID NO: 22 and comprising a DNA replication region having at least one nucleotide sequence selected from the group consisting of the nucleotide sequences set forth as SEQ ID NO: 35, SEQ ID NO: 36 and SEQ ID NO: 37.<br>
6.	A DNA fragment having the nucleotide sequence set forth as SEQ ID NO: 76.<br><br>
7.	A plasmid or a partial DNA fragment thereof, characterized by comprising a promoter region having the nucleotide sequence set forth as SEQ ID NO: 76.<br>
8.	A plasmid or a partial DNA fragment thereof, characterized by comprising a coding region for a DNA replication-related protein having at least one nucleotide sequence selected from the group consisting of the nucleotide sequences set forth as SEQ ID NO: 1, SEQ ID NO: 4, SEQ ID NO: 14, SEQ ID NO: 17 and SEQ ID NO: 22, comprising a DNA replication region having at least one nucleotide sequence selected from the group consisting of the nucleotide sequences set forth as SEQ ID NO: 35, SEQ ID NO: 36 and SEQ ID NO: 37, and comprising a promoter region having the nucleotide sequence set forth as SEQ ID NO: 76.<br>
9.	A circular plasmid characterized by comprising a plasmid or a partial DNA<br>
fragment according to any one of claims 1 to 8, wherein the numbers of restriction<br>
endonuclease cleavage sites are Bairk\ I: 2, EcOK I: 2, Kpn I: 1, Pvu II: 1, Sac I: 1<br>
and Smal\ 1, and the size is approximately 5.4 kbp.<br>
10.	A plasmid having the nucleotide sequence set forth as SEQ ID NO: 73.<br>
11.	A plasmid or a DNA fragment according to any one of claims 1 to 10,<br>
characterized by being derived from a bacterium belonging to the genus Rhodococcus.<br>
12.	A plasmid or a partial DNA fragment thereof, characterized by comprising<br>
a coding region for a DNA replication-related protein having at least one nucleotide<br>
sequence selected from the group consisting of the nucleotide sequences set forth as<br>
SEQ ID NO: 40, SEQ ID NO: 42, SEQ ID NO: 44, SEQ ID NO: 45, SEQ ID NO: 53, SEQ<br>
ID NO: 55, SEQ ID NO: 56, SEQ ID NO: 61, SEQ ID NO: 62 and SEQ ID NO: 69,<br>
comprising a DNA replication region having at least one nucleotide sequence selected<br>
from the group consisting of the nucleotide sequences set forth as SEQ ID NO: 70,<br>
SEQ ID NO; 71 and SEQ ID NO: 72, and comprising a promoter region having the<br>
nucleotide sequence set forth as SEQ ID NO: 76.<br><br>
13.	A plasmid or a DNA fragment according to claim 12, characterized by being derived from a bacterium belonging to the genus Rhodococcus.<br>
14.	A shuttle vector replicable in bacteria belonging to the genus Rhodococcus and E coll, and comprising a plasmid or partial DNA fragment thereof according to any one of claims 1 to 13 and a DNA region replicable in £. coli.<br><br>
15.	A vector characterized by being constructed using a shuttle vector according to claim 14.<br>
16.	A vector characterized by comprising a plasmid or DNA fragment according to any one of claims 6 or 7.<br>
17.	A vector according to claim 15 or 16, ch-sracterized by having inserted<br>
therein an aminoketone asymmetric reductase gene.<br>
18.	A vector according to claim 17, characterized in that the aminoketone<br>
asymmetric reductase gene is a nucleic acid coding for a protein consisting the amino<br>
acid sequence set forth as SEQ ID NO: 78, or a nucleic acid that codes for a protein<br>
having the amino acid sequence set forth as SEQ ID NO: 78 with a deletion, insertion,<br>
substitution or addition of one or a plurality of amino acids, and having aminoketone<br>
asymmetric reduction activity.<br>
19.	A vector according to claim 17, characterized in that the aminoketone<br>
asymmetric reductase gene is a nucleic acid consisting the nucleotide sequence set<br>
forth as SEQ ID NO: 79, or a nucleic acid that hybridizes with nucleic acid having a<br>
nucleotide sequence complementary to the nucleotide set forth as SEQ ID NO: 79<br>
under stringent conditions, and that codes for a protein having aminoketone<br>
asymmetric reduction activity.<br>
20.     A method for production of an aminoketone asymmetric reductase, which<br>
comprises a culturing step in which a transformant containing a vector according to any one of claims 17 to 19, is cultured in medium that allows growth of said transformant, and<br>
a purification step in which the aminoketone asymmetric reductase is purified from said transformant obtained in said culturing step.<br><br>
21.        A method for production of an optically active aminoalcohol, wherein an aminoketone asymmetric reductase obtained by the production method of claim 20 is reacted with an enantiomeric mixture of an a-aminoketone compound represented by the following general formula (1); [Chemical Formula 1]<br><br>
wherein X may be the same or different and represents at least one species selected<br>
from the group consisting of halogen, lower alkyl, hydroxyl optionally protected with<br>
a protecting group, nitro and sulfonyl;<br>
n represents an integer of 0 to 3;<br>
R^ represents lower alkyl;<br>
R^ and R^ may be the same or different and represent at least one species selected<br>
from the group consisting of hydrogen and lower alkyl; and<br>
* represents asymmetric carbon,<br>
or a salt thereof, to produce an optically active aminoalcohol compound represented<br>
by the following general formula (2):<br>
[Chemical Formula 2]<br><br>
wherein X, n, R^ R^ R^ and * have the same definitions as above, and having the desired optical activity.<br><br>
22.	A method for production of an opticeilly active aminoalcoliol, wlierein a<br>
transformant containing a vector according to any one of claims 17 to 19 is reacted with an enantiomeric mixture of an a-aminol<etone compound represented by the following general formula></etone>
[Chemical Formula 3]<br><br><br>
23.	A production method for an optically active aminoalcohol according to<br>
claim 22, wherein the production method for the optically active aminoalcohol is carried out with further addition of a compound represented by the following general formula (3): [Chemical Formula 5]<br><br><br>
R® represents hydrogen, optionally substituted Cl-3 alky I, a C5-10 hydrocarbon ring<br>
which is bonded to R^ or a 5- to 8-membered heterocyclic skeleton containing 1-3<br>
heteroatoms which is bonded to R^ or R^;<br>
R^ represents hydrogen or optionally substituted Cl-6 alkyl;<br>
R^ represents hydrogen, carboxyl, optionally substituted Cl-6 alkyl, a 5- to 8-<br>
membered heterocyclic skeleton containing 1-3 heteroatoms which is bonded to R"*<br>
or a C5-10 hydrocarbon ring which is bonded to R^;<br>
R' represents hydrogen, optionally substituted Cl-6 alkyl, optionally substituted Cl-6<br>
alkyloxycarbonyl, optionally substituted acyl or a 5- to 8-membered heterocyclic<br>
skeleton containing 1-3 heteroatoms which is bonded to R^ or R^; and<br>
RI° represents hydrogen or optionally substituted Cl-6 alkyl,<br>
or a pharmaceutically acceptable salt or solvate thereof, for production of an<br>
optically active aminoalcohol.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM0My1jaGVucC0yMDA2IGFic3RyYWN0LWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">1343-chenp-2006 abstract-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM0My1jaGVucC0yMDA2IGFic3RyYWN0LmpwZw==" target="_blank" style="word-wrap:break-word;">1343-chenp-2006 abstract.jpg</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM0My1jaGVucC0yMDA2IGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1343-chenp-2006 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM0My1jaGVucC0yMDA2IGNsYWltcy1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1343-chenp-2006 claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM0My1jaGVucC0yMDA2IGNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1343-chenp-2006 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM0My1jaGVucC0yMDA2IGNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1343-chenp-2006 correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM0My1jaGVucC0yMDA2IGNvcnJlc3BvbmRlbmNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">1343-chenp-2006 correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM0My1jaGVucC0yMDA2IGRlc2NyaXB0aW9uIChjb21wbGV0ZSktZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">1343-chenp-2006 description (complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM0My1jaGVucC0yMDA2IGRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">1343-chenp-2006 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM0My1jaGVucC0yMDA2IGRyYXdpbmdzLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">1343-chenp-2006 drawings-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM0My1jaGVucC0yMDA2IGRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1343-chenp-2006 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM0My1jaGVucC0yMDA2IGZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">1343-chenp-2006 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM0My1jaGVucC0yMDA2IGZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">1343-chenp-2006 form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM0My1jaGVucC0yMDA2IGZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">1343-chenp-2006 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM0My1jaGVucC0yMDA2IGZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">1343-chenp-2006 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM0My1jaGVucC0yMDA2IGZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">1343-chenp-2006 form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM0My1jaGVucC0yMDA2IHBldGl0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">1343-chenp-2006 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="230543-a-method-for-preparation-of-dihydrotagetone-alcohols-from-tagetes-minuta-oil.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="230545-benzothiazole-derivative-compounds-of-formula-i-and-a-process-for-preparing-the-same.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>230544</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1343/CHENP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>27-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>19-Apr-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>DAIICHI FINE CHEMICAL CO., LTD</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>530, Chokeiji, Takaoka-shi, Toyama 933 8511,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>YAMAMURA, Eitora</td>
											<td>c/o DAIICHI FINE CHEMICAL CO., LTD., 530, Chokeiji, Takaoka-shi, Toyama 933 8511,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>FUJIMOTO, Noboru</td>
											<td>c/o DAIICHI FINE CHEMICAL CO., LTD., 530, Chokeiji, Takaoka-shi, Toyama 933 8511,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/09</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/JP2004/016104</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-10-29</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>2003-373476</td>
									<td>2003-10-31</td>
								    <td>Japan</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/230544-dna-fragment-and-plasmid-and-partial-dna-fragment-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:19:57 GMT -->
</html>
