<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/230564-nucleotide-sequences-regulating-gene-expression-and-constructs-and-methods-utilizing-same by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:21:12 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 230564:NUCLEOTIDE SEQUENCES REGULATING GENE EXPRESSION AND CONSTRUCTS AND METHODS UTILIZING SAME</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">NUCLEOTIDE SEQUENCES REGULATING GENE EXPRESSION AND CONSTRUCTS AND METHODS UTILIZING SAME</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>ABSTRACT 2604/CHENP/2005 &quot;Nucleotide sequences regulating gene expression and constructs and methods utilizing same&quot; The present invention relates to an isolated polynucleotide comprising a nucleic acid sequence selected from the group consisting of SEQ ID NOS: 1, 6, 11, 16, 21, 26, 31,36,41,46,56,61,66,71,76,81,86,91,96, 101, 106, 111, 116, 121, 126, 131, 136, 141, 146, 151, 156, 161, 166, 171, 176, 181, 186, 191, 196, 201, 202, 203, 210 and 213 wherein the isolated polynucleotide is capable of regulating expression of at least one polynucleotide sequence operably linked thereto and whereas said nucleic acid sequence is no more than 3, 348 nucleic acids in length.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
NUCLEOTIDE   SEQUENCES   REGULATING   GENE    EXPRESSION   AND<br>
CONSTRUCTS AND METHODS UTILIZING SAME<br>
FIELD AND BACKGROUND OF THE INVENTION<br>
The present invention relates to isolated polynucleotides which are capable of<br>
regulating gene expression in an organism and more specifically, to novel nucleic acid<br>
sequences which include constitutive, inducible, tissue-specific and developmental<br>
stage-specific promoters which are capable of directing gene expression in plants.<br>
A promoter is a nucleic acid sequence approximately 200-1500 base pairs (bp)<br>
in length which is typically located upstream of coding sequences. A promoter<br>
functions in directing transcription of an adjacent coding sequence and thus acts as a<br>
switch for gene expression in an organism. Thus, all cellular processes are ultimately<br>
governed by the activity of promoters, making such regulatory elements important<br>
research and commercial tools.<br>
Promoters are routinely utilized for heterologous gene expression in<br>
commercial expression systems, gene therapy and a variety of research applications.<br>
The choice of the promoter sequence determines when, where and how<br>
strongly the heterologous gene of choice is expressed. Accordingly, when a<br>
constitutive expression throughout an organism is desired, a constitutive promoter is<br>
preferably utilized. On the other hand, when triggered gene expression is desired, an<br>
inductive promoter is preferred. Likewise, when an expression is to be confined to a<br>
particular tissue, or a particular physiological or developmental stage, a tissue specific<br>
or a stage specific promoter is respectively preferred.<br>
Constitutive promoters are active throughout the cell cycle and have been<br>
utilized to express heterologous genes in transgenic plants, such that the expression of<br>
traits encoded by the heterologous genes is effected throughout the plant at all time.<br>
Examples of known constitutive promoters often used for plant transformation include<br>
the cauliflower heat shock protein 80 (hsp80) promoter, 35S cauliflower mosaic virus<br>
promoter, nopal ine synthase (nos) promoter, octopine (ocs) Agrobacterium promoter<br>
and the mannopine synthase (mas) Agrobacterium promoter.<br>
Inducible promoters can be switched on by an inducing agent and are typically<br>
active as long as they are exposed to the inducing agent. The inducing agent can be a<br>
chemical agent, such as a metabolite, gro\\'th regulator, herbicide, or phenolic<br>
compound, or a physiological stress directly imposed upon the plant such as cold,<br><br>
/<br>
heat, salt, toxins, or through the action of a microbial pathogen or an insecticidal pest.<br>
Accordingly, inducible promoters can be utilized to regulate expression of desired<br>
traits, such as genes that control insect pests or microbial pathogens, whereby the<br>
protein is only produced shortly upon infection or first bites of the insect and<br>
transiently so as to decrease selective pressure for resistant insects. For example,<br>
plants can be transfoimed to express insecticidal or fungicidal traits such as the<br>
Bacillus thuringiensis (Bt) toxins, viruses coat proteins, glucanases, chitinases or<br>
phytoalexins. In another example, plants can be transformed to tolerate herbicides by<br>
overexpressing, upon exposure to a herbicide, the acetohydroxy acid synthease<br>
enzyme, which neutralizes multiple types of herbicide? [Hattori, J. et ah, Mol.<br>
General. Genet. 246:419 (1995)].<br>
Several fruit-specific promoters have been described, including an apple-<br>
isolated Thi promoter (U.S. Pat. No. 6,392,122); a strawberry-isolated promoter (U.S.<br>
Pat. No. 6,080,914); tomato-isolated E4 and E8 promoters (U.S. Pat. No. 5,859,330);<br>
a polygalacturonase promoter (U.S. Pat. No. 4,943,674); and the 2AII tomato gene<br>
â– promoter [Van Haaren et al.. Plant Mol. Biol. 21: 625-640 (1993)]. Such fruit<br>
specific promoters can be utilized, for example, to modify fruit ripening by regulating<br>
expression of ACC deaminase which inhibits biosynthesis of ethylene. Other gene<br>
products which may be desired to express in fruit tissue include genes encoding flavor<br>
or color traits, such as thaumatin, cyclase or sucrose phosphate synthase.<br>
3,483.862^<br>
Seed specific promoters have been described in U.S, Pat. Nos. 6,<br>
5,608,152 and 5,504,200; and in U.S. Patent Application Ser. Nos. 09/998059 and<br>
10/137964. Such seed specific promoters can be utilized, for example, to alter the<br>
levels of saturated or unsaturated fatty acids; to increase levels of lysine- or sulfur-<br>
containing amino acids, or to modify the amount of starch contained in seeds.<br>
Several promoters which regulate gene expression specifically during<br>
germination stage have been described, including the a-glucoronidase and the<br>
cystatin-1 barely-isolated promoters (U.S. Pat. No. 6',355*,19^, and the hydrolase<br>
promoter [Skriver et ai, Proc. Natl. Acad. Sci. USA, 8^7256-7270 (1991)].<br>
While reducing the present invention to practice, the present inventors have<br>
uncovered several regulatory sequences which exhibit a wide range of promoter<br>
activities in plants, as is fiirther described hereinunder, such regulatory sequences can<br>
be used in a variety' of commercial and research applications.<br>
^<br><br>
STJMMARY OF THE INVENTION<br>
According to one aspect of the present invention there is provided an isolated<br>
polynucleotide comprising a nucleic acid sequence selected from the group consisting<br>
of SEQIDNOS: 1, 6,11,16,21, 26, 31, 36, 41,46, 56,61,66,71,76, 81,86,91,96,<br>
101, 106, 111, 116, 121, 126, 131, 136, 141, 146, 151, 156, 161, 166, 171, 176, 181,<br>
I<br>
186, 191, 196, 201, 202, 211, 210 and 213, wherein the isolated polynucleotide is<br>
capable of regulating expression of at least one polynucleotide sequence operably<br>
linked thereto.<br>
According to another aspect of the present invention there is provided a<br>
nucleic acid construct which includes the isolated polynucleotide comprising the<br>
nucleic acid sequence selected from die group consisting of SEQ ID NOS: 1,6, 11,<br>
16, 21,26, 31, 36,41, 46, 56, 61, 66, 71, 76, 81. 86, 91, 96,101. 106, 111, 116,121,<br>
126, 131,136, 141, 146, 151, 156, 161, 166, 171. 176, 181, 186, 191, 196,201,202,<br>
203,210 and 213.<br>
According to yet another aspect of the present invention tiiere is provided a<br>
transgenic cell which includes the isolated polynucleotide comprising the nucleic acid<br>
sequence selected from the group consisting of SEQ ID NOS: 1, 6.11, 16, 21, 26, 31,<br>
36,41.46, 56, 61, 66, 71, 76, 81. 86, 91. 96.101, 106, 111, 116,121, 126.131,136,<br>
141. 146. 151, 156. 161, 166, 171, 176, 181, 186. 191, 196. 201.202, 203. 210 and<br>
213.<br>
According to still another aspect of the present invention there is provided a<br>
transgenic cell comprising the nucleic acid construct which includes the isolated<br>
polynucleotide comprising the nucleic acid sequence selected from the group<br>
consisting of SEQ ID NOS: 1,6,11.16,21,26, 31,36,41,46, 56, 61, 66, 71, 76, 81,<br>
86,91,96,101,106, 111, 116,121, 126,131,136, 141,146,151,156,161,166.171,<br>
176,181,186,191,196.201,202.203,210 and 213.<br>
According to an additional aspect of the present invention there is provided a<br>
transgenic organism which includes the isolated polynucleotide comprising the<br>
nucleic acid sequence selected from the group consisting of SEQ ID NOS: 1,6. 11,<br>
16. 21, 26, 31, 36.41,46, 56, 61, 66, 71, 76, 81, 86, 91, 96,101,106, 111, 116,121,<br>
126, 131, 136, 141, 146, 151, 156, 161, 166, 171, 176, 181,186,191. 196, 201, 202,<br>
203,210 and 213.<br><br>
According to yet an additional aspect of Ae present invention there is provided<br>
a transgenic organism comprising a nucleic acid construct which includes the isolated<br>
polynucleotide comprising the nucleic acid sequence selected firom the group<br>
consisting of SEQ ID NOS: 1, 6, 11, 16,21,26, 31, 36,41,46, 56, 61,66, 71, 76, 81,<br>
86, 91, 96,101, 106, 111, 116,121, 126, 131, 136, 141,146,151, 156, 161, 166, 171,<br>
176,181,186,191,196, 201,202,203,210 and 213.<br>
According to still an additional aspect of the present invention tho'e is<br>
provided a transgenic plant which includes the isolated polynucleotide comprising<br>
the nucleic acid sequence selected from the group consisting of SEQ ID NOS: 1, 6,<br>
11, 16, 21, 26, 31, 36,41, 46,.56. 61, 66, 71, 76, 81, 86,,91, 96, 101, 106, 111, 116,<br>
121, 126, 131,136, 141,146, 151, 156, 161, 166, 171, 176,181,186, 191, 196, 201,<br>
202,203,210 and 213.<br>
According to a further aspect of the present invention there is provided a<br>
transgenic plant comprising a nucleic acid construct which includes the isolated<br>
polynucleotide comprising the nucleic acid sequence selected from the group<br>
consisting of SEQ ID NOS: 1, 6,11, 16,21,26, 31, 36,41,46, 56,61,66, 71, 76, 81,<br>
86, 91, 96, 101, 106, HI, 116. 121, 126,131, 136, 141, 146,151,156, 161, 166, 171<br>
176,181,186,191,196,201,202,203,210 and 213.<br>
According to yet a further aspect of the present invention there is provided a<br>
method of producing a transgenic plant comprising transforming a plant with an<br>
isolated polynucleotide which includes a nucleic acid sequence selected from the<br>
group consisting of SEQ ID NOS: 1, 6, 11,16,21, 26,31, 36, 41, 46, 56, 61, 66, 71,<br>
76, 81, 86, 91, 96, 101, 106, 111, 116, 121, 126, 131, 136, 141, 146, 151, 156, 161,<br>
166,171,176,181,186,191,196,201,202,203,210 and 213.<br>
According to still a further aspect of the present invention there is provided a<br>
method of producmg a transgenic plant comprising transforming a plant with a<br>
nucleic acid construct which includes the isolated polynucleotide comprising the<br>
nucleic acid sequence selected from the group consisting of SEQ ID NOS: 1, 6, 11,<br>
16, 21,26, 31, 36, 41,46, 56, 61, 66, 71, 76, 81, 86, 91, 96,101, 106, 111, 116, 121,<br>
126,131, 136, 141, 146,151, 156, 161, 166, 171, 176, 181,186, 191,196. 201, 202,<br>
203,210 and 213.<br>
According to still a further aspect of the present invention there is provided a<br>
method of expressing a polypeptide of interest in a cell comprising transforming the<br>
^<br><br>
cell wifli a nucleic acid construct including a polynucleotide sequence encoding the<br>
polypeptide of interest operably linked to a regulatory nucleic acid sequence selected<br>
from the group consisting of SEQ ID NOS: 1, 6,11,16, 21,26, 31, 36,41,46, 56, 61,<br>
66,71, 76, 81, 86,91, 96,101,106, 111, 116, 121,126,131, 136,141,146,151,156,<br>
161, 166, 171, 176, 181, 186, 191, 196, 201, 202, 203, 210 and 213 thereby<br>
expressing the polypeptide of interest in the cell.<br>
According to still a further aspect of the present invention there is provided a<br>
method of co-expressing two polypeptides of interest in a cell comprising<br>
transforming the cell with a nucleic acid construct including two polynucleotide<br>
sequences encoding the two polypeptides of interest operably linked to a regulatory<br>
nucleic acid sequence selected from the group consisting of SEQ ID NOS: 1, 6, 11,<br>
16, 21, 26, 31, 36, 41, 46, 56, 6\, 66, 71, 76, 81, 86, 91, 96, 101, 106, 111, 116, 121,<br>
126, 131, 136, 141, 146, 151, 156, 161, 166, 171, 176, 181, 186, 191, 196, 201, 202,<br>
203, 210 and 213 such that said two polynucleotide sequences flank said regulatory<br>
nucleic acid sequence, thereby expressing the two polypeptides of interest in the cell.<br>
According to further features in preferred embodiments of the invention<br>
described below, the isolated polynucleotide includes at least one promoter region.<br>
According to still further features in the described preferred embodiments the<br>
nucleic acid sequence is selected from the group consisting of SEQ ID NOS: 1, 6,41,<br>
46, 51, 86, 121, 136, 171, 181 and 202, and whereas the at least one promoter region<br>
is capable of directing transcription of said at least one polynucleotide sequence in a<br>
constitutive manner.<br>
According to still furtha- features in the described preferred embodiments the<br>
nucleic acid sequence is selected from the group consisting of SEQ ID NOS: 1, 11,<br>
16, 21, 26, 31, 36, 56, 61, 66, 71, 76, 81, 91, 96, 101, 116, 126, 141, 146, 151, 156,<br>
161, 166, 176, 186, 191, 196, 201, 203, 210 and 213, and whereas the at least one<br>
promoter region is capable of directing transcription of said at least one<br>
polynucleotide sequence in an inductive manner.<br>
According to still further features in the described preferred embodiments the<br>
nucleic acid sequence is selected from the group consisting of SEQ ID NOS: 1, 11,<br>
16,21, 26, 31, 36, 56, 61, 66,71, 76, 91,116,126,141, 146,151,156, 161,166,176,<br>
186,191,196, 201, 203, 210 and 213, and whereas the at least one promoter region is<br>
(o		<br><br>
capable of directing transcription of said at least one polynucleotide sequence in a<br>
tissue specific manner.<br>
According to still fiirther features in the described preferred embodiments the<br>
nucleic acid sequence is selected from the group consisting of SEQ ID NOS: 81, 96,<br>
101,106 and 131, and whereas the at least one promoter region is capable of directing<br>
transcription of said at least one polynucleotide sequence in a developmental stage<br>
specific manner.<br>
According to still further features in the described preferred embodiments the<br>
nucleic acid construct fiirther includes at least one heterologous polynucleotide<br>
operably linked to the isolated polynucleotide.<br>
According to still further features in the described preferred embodiments the<br>
at least one heterologous polynucleotide is a reporter gene.<br>
According to still further features in the described preferred embodiments the<br>
nucleic acid construct further includes two heterologous polynucleotides each being<br>
operably linked to an end of the isolated polynucleotide such that the two<br>
heterologous polynucleotides flank the isolated polynucleotide.<br>
The present invention successfully addresses the shortcomings of the presently<br>
known configurations by providing a plurality of isolated polynucleotide sequences<br>
which exhibit a wide spectrum of promoter ftmction patterns. These polynucleotides<br>
can be used to generate nucleic acid constructs, such as expression vectors suitable for<br>
transforming an organism. Such nucleic acid constructs can be used to promote<br>
expression of desired traits or expression products in transgenic organisms, such as<br>
plants, in a constitutive, induced, tissue specific, or a developmental stage specific<br>
marmer.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
The invention is herein described, by way of example only, with reference to<br>
the accompanying drawings. With specific reference now to the drawings in detail, it<br>
is stressed that the particulars shown are by way of example and for purposes of<br>
illustrative discussion of the preferred embodiments of tiie present invention only, and<br>
are presented in the cause of providing what is believed to be the most useful and<br>
readily understood description of the principles and conceptual aspects of the<br>
invention.    In this regard, no attempt is made to show structural details of the<br>
4<br><br>
r<br>
invention in more detail than is necessary for a fundamental understanding of the<br>
invention, the description taken v^ith the drawings making apparent to those skilled in<br>
the art how the several forms of the invention may be embodied in practice.<br>
In the drawings:<br>
FIGs. la-b are photographs showing an Arabidopsis thaliana plant<br>
transformed with a nucleic acid construct comprising the nucleic acid sequence set<br>
forth in SEQ ID NO: 11 operably linked to a luciferase encoding sequence. Figure la<br>
shows the transgenic plant under normal light. Figure lb is an ultra-low light<br>
photograph of the same plant in the dark, illustrating a specific expression luciferase<br>
in flower tissue.<br>
FIGs. 2a-b are photographs showing an Arabidopsis thaliana plant<br>
transformed with a nucleic acid construct comprising the nucleic acid sequence set<br>
forth in SEQ ID NO: 21 operably linked to a luciferase encoding sequence. Figure 2a<br>
shows the transgenic plant under normal light. Figure 2b is an ultra-low light<br>
photograph of the same plant in the dark, illustrating a specific expression of<br>
luciferase in root tissue.<br>
FIGs. 3a-b are photographs showing an Arabidopsis thaliana plant<br>
transformed with a nucleic acid construct comprising the nucleic acid sequence set<br>
forth in SEQ ID NO: 36 operably linked to a luciferase encoding sequence. Figure 3a<br>
shows tlae transgenic plant under normal light. Figure 3b is an ultra-low light<br>
photograph of the same plant in the dark, illustrating a specific expression of<br>
luciferase in root and flower tissue.<br>
FIGs. 4a-b are photographs showing an Arabidopsis thaliana plant<br>
transformed with a nucleic acid construct comprising the nucleic acid sequence set<br>
forth in SEQ ID NO: 61 operably linked to a luciferase encoding sequence. Figure 4a<br>
shows the transgenic plant under normal light. Figure 4b is an ultra-low light<br>
photograph of the same plant in the dark, illustrating a specific expression of<br>
luciferase in young tissue.<br>
FIGs. 5a-b are photographs showing an Arabidopsis thaliana seedling<br>
transformed with a nucleic acid construct comprising the nucleic acid sequence set<br>
forth in SEQ ID NO: 66 operably linked to a luciferase encoding sequence. Figure 5a<br>
shows the transgenic plant under normal light.   Figure 5b is an ultra-low light<br><br>
photograph of the same plant in the dark, illustrating an expression of luciferase in<br>
leaf tissue.<br>
FIGs. 6a-b are photographs showing an Arabidopsis thaliana mature plant<br>
transformed with a nucleic acid construct comprising the nucleic acid sequence set<br>
forth in SEQ ID NO: 66 operably linked to a luciferase encoding sequence. Figure 6a<br>
shows the transgenic plant under normal light Figure 6b is an ultra-low light<br>
photograph of the same plant in the dark, illustrating an expression of luciferase in<br>
stem tissue.<br>
FIGs. 7a-b are photographs showing an Arabidopsis thaliana plant seedlings<br>
transformed with a nucleic acid construct comprising the nucleic acid sequence set<br>
forth in SEQ ED NO: 81 operably linked to a luciferase encoding sequence. Figure 7a<br>
shows the transgenic plant under normal light. Figure 7b is an ultra-low light<br>
photograph of the same plant in the dark, illustrating an expression of luciferase in<br>
above ground tissue.<br>
FIGs. 8a-b are photographs showing an Arabidopsis thaliana mature plant<br>
transformed with a nucleic acid construct comprising the nucleic acid sequence set<br>
forth in SEQ ID NO: 81 operably linked to a luciferase encoding sequence. Figure 8a<br>
shows the transgenic plant under normal light. Figure 8b is an ultra-low light<br>
photograph of the same plant in the dark, illustrating an expression of luciferase in<br>
flower tissue.<br>
FIGs. 9a-b are photographs showing an Arabidopsis thaliana plant<br>
transformed with a nucleic acid construct conqjrising the nucleic acid sequence set<br>
forth in SEQ ID NO: 91 operably linked to a luciferase encoding sequence. Figure 9a<br>
shows the transgenic plant under normal light Figure 9b is an ultra-low light<br>
photograph of the same plant in the dark, illustrating a specific expression of<br>
luciferase in root and flower tissue.<br>
FIGs. lOa-b are photographs showing an Arabidopsis thaliana seedling<br>
transformed with a nucleic acid construct comprising the nucleic acid sequence set<br>
forth in SEQ ID NO: 96 operably linked to a luciferase encoding sequence. Figure<br>
10a shows the transgenic plant under normal light. Figure 10b is an ultra-low light<br>
photograph of the same plant in the dark, illustrating a specific expression of<br>
luciferase in above ground tissue.<br><br>
FIGs. lla-b are photographs showing an Arabidopsis thaliana mature plant<br>
transformed with a nucleic acid construct comprising the nucleic acid sequence set<br>
forth in SEQ ID NO: 96 operably linked to a luciferase encoding sequence. Figure<br>
11a shows the transgenic plant imder normal light Figure lib is an ultra-low light<br>
photograph of the same plant in the dark, illustrating a specific expression of<br>
luciferase in above ground tissue.<br>
FIGs. 12a-b are photographs showing seeds of zn Arabidopsis thaliana plant<br>
transformed with a nucleic acid construct comprising the nucleic acid sequence set<br>
forth in SEQ ID NO: 111 operably linked to a luciferase encoding sequence. Figure<br>
12a shows the seeds under normal light. Figure 12b is an ultra-low light photograph<br>
of the same seeds in tine dark, illustrating a specific expression of luciferase in seeds.<br>
FIGs. 13a-b are photographs showing an Arabidopsis thaliana plant<br>
transformed with a nucleic acid construct comprising the nucleic acid sequence set<br>
forth in SEQ ID NO: 111 operably linked to a luciferase encoding sequence. Figure<br>
13a shows the transgenic plant under normal light. Figure 13b is an ultra-low light<br>
photograph of the same plant in the dark, illustrating a specific expression of<br>
luciferase in roots.<br>
FIGs. I4a-b are photographs showing an Arabidopsis thaliana plant<br>
transformed with a nucleic acid construct comprising the nucleic acid sequence set<br>
forth in SEQ ID NO: 121 operably linked to a luciferase encoding sequence. Figure<br>
14a shows the transgenic plant under normal light. Figure 14b is an ultra-low liglit<br>
photograph of the same plant in the dark, illustrating a specific expression of<br>
luciferase in meristematic tissue.<br>
FIGs. I5a-b are photographs showing an Arabidopsis thaliana seedling<br>
transformed with a nucleic acid construct comprising the nucleic acid sequence set<br>
forth in SEQ ID NO: 126 operably linked to a luciferase encoding sequence. Figure<br>
15a shows the transgenic plant under normal light. Figure 15b is an ultra-low light<br>
photograph of the same plant in the dark, illustrating a specific expression of<br>
luciferase in root meristematic tissue.<br>
FIGs. 16a-b are photographs showing an Arabidopsis thaliana plant<br>
transformed with a nucleic acid construct comprising the nucleic acid sequence set<br>
forth in SEQ ID NO: 126 operably linked to a luciferase encoding sequence. Figure<br>
16a shows the transgenic plant under normal light.  Figure 16b is an ultra-low light<br>
[O<br><br>
photograph of the same plant in the dark, illustrating a specific expression of<br>
luciferase in flower meristematic tissue.<br>
FIGs. 17a-b are photographs showing an Arabidopsis thaliana plant<br>
transformed with a nucleic acid construct comprising the nucleic acid sequence set<br>
forth in SEQ ID NO: 131 operably linked to a luciferase encoding sequence. Figure<br>
17a shows the transgenic plant under normal light. Figure 17b is an ultra-low light<br>
photograph of the same plant in the dark, illustrating a specific expression of<br>
luciferase in leaf tissue.<br>
FIGs. 18a-b are photographs showing an Arabidopsis thaliana plant<br>
transformed with a nucleic acid construct comprising the nucleic acid sequence set<br>
forth in SEQ ID NO: 136 operably linked to a luciferase encoding sequence. Figure<br>
18a shows the transgenic plant under normal light. Figure 18b is an ultra-low light<br>
photograph of the same plant in the dark, illustrating a non-specific constitutive<br>
expression of luciferase.<br>
FIGs. 19a-b are photographs showing an Arabidopsis thaliana seedling<br>
transformed with a nucleic acid construct comprising the nucleic acid sequence set<br>
forth in SEQ ID NO: 156 operably linked to a luciferase encoding sequence. Figure<br>
19a shows the transgenic plant under normal light. Figure 19b is an ultra-low ligjit<br>
photograph of the same plant in the dark, illustrating a specific expression of<br>
luciferase in above ground tissue.<br>
FIGs. 20a-b are photographs showing an Arabidopsis thaliana mature plant<br>
transformed with a nucleic acid construct comprising the nucleic acid sequence set<br>
forth in SEQ ID NO: 156 operably linked to a luciferase encoding sequence. Figure<br>
20a shows the transgenic plant xmder normal light. Figure 20b is an ultra-low light<br>
photograph of the same plant in the dark, illustrating a specific expression of<br>
luciferase in above ground tissue.<br>
FIGs. 21a-b are photographs showing seeds of zn Arabidopsis thaliana plant<br>
transformed with a nucleic acid construct comprising the nucleic acid sequence set<br>
forth in SEQ ID NO: 161 operably linked to a luciferase encoding sequence. Figure<br>
21a shows the seeds under normal light. Figure 21b is an ultra-low light photograph<br>
of liie same plant in the dark, illustrating a specific expression of luciferase in seed<br>
tissue.<br>
U<br><br>
FIGs. 22a-b are photographs showing an Arabidopsis thaliana plant<br>
transformed with a nucleic acid construct comprising the nucleic acid sequence set<br>
forth in SEQ ID NO: 186 operably linked to a luciferase encoding sequence. Figure<br>
22a shows the transgenic plant under normal light. Figure 22b is an ultra-low light<br>
photograph of the same plant in the dark, illustrating an expression of luciferase in<br>
stalk and stem tissue.<br>
FIGs. 23a-b are photographs showing an Arabidopsis thaliana plant<br>
transformed with a nucleic acid construct comprising the nucleic acid sequence set<br>
forth in SEQ ID NO: 191 operably linked to a luciferase encoding sequence. Figure<br>
23 a shows the transgenic plant under normal light. Figure 23b is an ultra-low light<br>
photograph of the same plant in the dark, illustrating a weak expression of luciferase<br>
in vegetative tissue.<br>
FIGs. 24a-b are photographs showing an Arabidopsis thaliana plant<br>
transformed with a nucleic acid construct comprising the nucleic acid sequence set<br>
forth in SEQ ID NO: 201 operably linked to a luciferase encoding sequence. Figure<br>
24a shows the transgenic plant under normal light. Figure 24b is an ultra-low light<br>
photograph of the same plant in the dark, illustrating an above ground tissue specific<br>
expression of luciferase.<br>
FIGs. 25a-b are photographs showing an Arabidopsis thaliana plant<br>
transformed with a nucleic acid construct comprising the nucleic acid sequence set<br>
forth in SEQ ID NO: 176 operably linked to a luciferase encoding sequence. Figure<br>
25a shows the transgenic plant under normal light. Figure 25b is an ultra-low light<br>
photograph of the same plant in the dark, illustrating a specific expression of<br>
luciferase in flower tissue.<br>
FIGs. 26a-b are photographs showing transformed Arabidopsis thaliana plants<br>
transformed with nucleic acid constructs including partial DREs operably each linked<br>
to a GUS encoding sequence. Figure 26a shows a plant transformed with a nucleic<br>
acid construct including the nucleic acid sequence set forth in SEQ ID NO: 210<br>
operably linked to a GUS encoding sequence. Figure 26b shows root tips of a plant,<br>
transformed with a nucleic acid construct comprising the nucleic acid sequence set<br>
forth in SEQ ID NO: 213 operably linked to a GUS encoding sequence.<br>
FIG. 27 is a nucleic acid sequence alignment between DRE 6669 (SEQ ED<br>
NO: 61, QUER\0 and a prior art sequence (SEQ ID NO: 214, SBJCT), revealing a<br>
(2-<br><br>
different 5' sequence which is important for constitutive expression, as is exemplified<br>
in the Examples section hereinbelow.<br>
DESCRIPTION OF THE PREFERRED EMBODIMENTS<br>
The present invention provides isolated polynucleotides capable of regulating<br>
the expression of operably linked heterologous polynucleotides, and more<br>
specifically, novel nucleic acid sequences which are capable of promoting gene<br>
expression in a constitutive, inductive, tissue specific and/or developmental stage<br>
specific manner. The present invention also provides nucleic acid constructs, as well<br>
transgenic organisms which carry the polynucleotides of the present invention and<br>
methods of producing thereof<br>
The principles and operation of the present invention may be better understood<br>
with reference to the accompanying descriptions.<br>
Before explaining at least one embodiment of the invention in detail, it is to be<br>
understood that the invention is not limited in its application to the details of<br>
construction and the arrangement of the components set forth in the following<br>
descriptions or illustrated in the Examples section. The invention is capable of other<br>
embodiments or of being practiced or carried out in various ways. Also, it is to be<br>
understood that the phraseology and terminology employed herein is for the purpose<br>
of description and should not be regarded as limiting.<br>
The term "polynucleotide" or the phrase "nucleic acid sequence" are used<br>
herein interchangeably and refer to a polymer of deoxyrebonucleotide (DNA) or<br>
ribonucleotide (RNA).<br>
The phrase "heterologous polynucleotide" refers to a polynucleotide sequence<br>
which originates from a heterologous organism or to a polynucleotide sequence which<br>
is linked to a regulatory sequence of the same organism which does not normally<br>
regulate expression of the polynucleotide sequence in the organism.<br>
PCT Publication WO 02/07989 describes a unique approach developed by the<br>
present inventors in order to uncover novel regulatory sequences in organisms such as<br>
plants. This approach combines molecular and bioinformatics techniques for high<br>
throughput isolation of DNA regulating elements (DREs), located within the non-<br>
transcribed (non-coding) regions of the genome and which include, for example,<br>
promoters, enhancers, suppressors, silencers, locus control regions and the like.<br>
[3<br><br>
Utilizing this approach, the present inventors have uncovered several novel<br>
polynucleotide sequences which, as illustrated in the Examples section which follows,<br>
exhibit regulatory activity in plants.<br>
Thus, according to one aspect of the present invention, there is provided<br>
isolated polynucleotides which are capable of regulating the expression of at least one<br>
polynucleotide operably linked thereto. As is further described in the Examples<br>
section which follows, these isolated polynucleotides are as set forth in SEQ ID NOS:<br>
1, 6, 11, 16, 21, 26, 31, 36, 41, 46, 56, 61, 66, 71, 76, 81, 86, 91, 96, 101, 106, 111,<br>
116, 121, 126, 131, 136, 141, 146, 151,156,161, 166, 171, 176. 181,186,191, 196,<br>
201, 202 and 203, or fragments (e.g., SEQ ID NOS:^210 and 213), variants or<br>
derivatives thereof<br>
A coding nucleic acid sequence is "operably linked" to a regulatory sequence<br>
if it is capable of exerting a regulatory effect on the coding sequence linked thereto.<br>
Preferably, the regulatory sequence is positioned 1-500 bp upstream of the ATG<br>
codon of the coding nucleic acid sequence, although it will be appreciated that<br>
regulatory sequences can also exert their effect when positioned elsewhere with<br>
respect to the coding nucleic acid sequence (e.g., within an intron).<br>
As is clearly illustrated in the Examples section which follows, the isolated<br>
polynucleotides of the present invention are capable of regulating expression of a<br>
coding nucleic acid sequence (e.g., luciferase) operably linked thereto (see Figures 1-<br>
25).<br>
The isolated polynucleotides of the present invention range in length from 174<br>
to 3,348 nucleotides and include one or more sequence regions which are capable of<br>
recognizing and binding RNA polymerase II and other proteins (trans-acting<br>
transcription factors) involved in transcription.<br>
Although most of the isolated polynucleotides described herein include one<br>
promoter region, some include two distinct promoter regions each positioned on a<br>
different strand of the same genorm'c sequence. Such bidirectional DREs are further<br>
described in the Examples section which follows (see for example. Tables 3-17).<br>
As is further illustrated by the Examples section which follows, the isolated<br>
polynucleotides of the present invention exhibit a range of activities and tissue<br>
specificities.<br><br>
Thus for example, the nucleic acid sequences set forth in SEQ ID N0S:1, 6,<br>
41,46,51, 86, 121,136, 171, 181 and 202 or fiagment, variants or derivatives thereof,<br>
are capable of directing transcription of coding nucleic acid sequences operably<br>
linked thereto in a constitutive manner and thus include a constitutive promoter<br>
region.<br>
In another example, the nucleic acid sequences set forth in SEQ ID NOS: 1,<br>
11, 16, 21, 26, 31, 36, 56, 61, 66, 71, 76, 81, 91, 96, 101, 116, 126, 141, 146, 151,<br>
156,161,166,176,186, 191,196,201 and 203, or fragments (e.g., SEQ ID NOS: 210<br>
and 213), variants or derivatives thereof, are capable of directing transcription of<br>
coding nucleic acid sequences operably linked thereto in an inductive manner and<br>
thus include an inductive promoter region.<br>
In yet another example, the nucleic acid sequences set forth in SEQ ID NOS:<br>
1, 11, 16, 21, 26, 31, 36, 56, 61, 66, 71, 76, 91, 116, 126, 141, 146, 151, 156, 161,<br>
166,176,186,191,196, 201 and 203, or fragments (e.g., SEQ ID NOS: 210 and 213),<br>
variants or derivatives thereof, are capable of directing transcription of coding nucleic<br>
acid sequences operably linked thereto in a tissue specific manner and thus include a<br>
tissue specific promoter region.<br>
In further yet another example, the nucleic acid sequences set forth in SEQ ID<br>
NOS: 81, 96, 101, 106 and 131, or fragment, variants or derivatives thereof, are<br>
capable of directing transcription of coding nucleic acid sequences operably linked<br>
thereto in a developmental stage specific manner and thus include a developmental<br>
stage specific promoter region.<br>
Preferably, the polynucleotide of the present invention are modified to create<br>
variations in the molecule sequences such as to enhance their promoting activities,<br>
using methods known in the art, such as PCR-based DNA modification, or standard<br>
DNA mutagenesis techniques, or by chemically synthesizing the modified<br>
polynucleotides.<br>
Accordingly, the sequences set forth in SEQ ID NOS: 1, 6, 11, 16, 21,26, 31,<br>
36,41,46,56,61,66,71,76,81,86,91,96, 101, 106, 11, 116, 121, 126,131, 136,<br>
141,146, 151, 156, 161, 166, 171, 176,181,186, 191, 196, 201, 202 and 203 may be<br>
truncated or deleted and still retain the capacity of directing the transcription of an<br>
operably linked DNA sequence (e.g., SEQ ID NOS: 210 and 213). The minimal<br>
length of a promoter region can be determined by systematically removing sequences<br>
15<br><br>
from the 5' and 3'-ends of the isolated polynucleotide by standard techniques known<br>
in the art, including but not limited to removal of restriction enzyme fragments or<br>
digestion with nucleases. Consequently, any sequence fragments, portions, or regions<br>
of the disclosed polypeptide sequences of the present invention can be used as<br>
regulatory sequences. It will be appreciated that modified sequences (mutated,<br>
truncated and the like) can acquire different transcriptional properties such as the<br>
direction of different pattern of gene expression as compared to the unmodified<br>
element (e.g., SEQ ID NO: 61 as compared to SEQ ED NO: 213, see the Examples<br>
section which follows).<br>
Optionally, the sequences set forth in SEQ ID NQS: 1, 6, 11, 16, 21, 26, 31,<br>
36, 41, 46, 56, 61, 66, 71, 76, 81, 86, 91, 96, 101, 106, 111, 116, 121, 126, 131, 136,<br>
141,146, 151, 156, 161,166,171,176,181, 186, 191,196, 201, 202 and 203 maybe<br>
modified, for example for expression in a range of plant systems. In another approach,<br>
novel hybrid promoters can be designed or engineered by a number of methods.<br>
Many promoters contain upstream sequences which activate, enhance or define the<br>
strength and/or specificity of the promoter, such as described, for example, by<br>
Atchison [Ann. Rev. Cell Biol. 4:127 (1988)]. T-DNA genes, for example contain<br>
"TATA" boxes defining the site of transcription initiation and other upstream<br>
elements located upstream of the transcription initiation site modulate transcription<br>
levels [Gelvin In: Transgenic Plants (Kung, S.-D. and Us,R., eds, San Diego:<br>
Academic Press, pp.49-87, (1988)]. Another chimeric promoter combined a trimer of<br>
the octopine synthase (ocs) activator to the mannopine synthase (mas) activator plus<br>
promoter and reported an increase in expression of a reporter gene [Min Ni et al.. The<br>
Plant Journal 7:661 (1995)]. The upstream regulatory sequences of the polynucleotide<br>
sequences of present invention can be used for the construction of such chimeric or<br>
hybrid promoters. Methods for construction of variant promoters include, but are not<br>
limited to, combining control elements of different promoters or duplicating portions<br>
or regions of a promoter (see for example, U.S. Pat. Nos. 5,110,732 and 5,097,025).<br>
Those of skill in the art are familiar with the specific conditions and procedures for<br>
the construction, manipulation and isolation of macromolecules (e.g., DNA<br>
molecules, plasmids, etc.), generation of recombinant organisms and the screening<br>
and isolation of genes, [see for example Sambrook et ah. Molecular Cloning: A<br>
Laboratory Manual, Cold Spring Harbor Press, (1989); Mailga et al.. Methods in<br><br>
Plant Molecular Biology, Cold Spring Harbor Press, (1995); Birren et al.. Genome<br>
Analysis: volume 1, Analyzing DNA, (1997); voltraie 2, Detecting Genes, (1998);<br>
volume 3, Cloning Systems, (1999); and volume 4, Mapping Genomes, (1999), Cold<br>
Spring Harbor, N.Y].<br>
The polynucleotides of the present invention, or fragment, variants or<br>
derivatives thereof,.can be incorporated into nucleic acid constructs, preferably<br>
expression constructs (i.e., expression vectors) which can be introduced and replicate<br>
in a host cell.<br>
Thus, according to another aspect of the present invention there is a provided a<br>
nucleic acid construct which includes at least one of the polynucleotides of the present<br>
invention, or fragments, variants or derivatives thereof.<br>
Preferably, the nucleic acid construct of the present invention includes at least<br>
one operably linked heterologous polynucleotide. More preferably, at least one<br>
operably linked reporter gene.<br>
The phrase "reporter gene" used herein refers to a gene encoding a selectable,<br>
screenable or detectable phenotype.<br>
Reporter genes which may be utilized in the present invention may include,<br>
but not limited to, LUX or LUC coding for luciferase, GUS coding for p-<br>
glucoronidase, GFP coding for green-fluorescent protein, or antibiotic or herbicide<br>
tolerance genes. A general review of suitable markers is foimd in Wilmink and Dons,<br>
Plant Mol. Biol. Reprt. 11:165-185 (1993).<br>
Further preferably, the nucleic acid construct of the present invention includes<br>
at least one heterologous polynucleotide encoding a desirable trait or an expression<br>
product<br>
A desirable trait which may be utilized in this invention may include, but not<br>
limited to, any phenotype associated with organism's morphology, physiology,<br>
growth and development, yield, produce quality, nutritional enhancement, disease or<br>
pest resistance, or stress tolerance.<br>
Alternatively, the heterologous polynucleotide can encode any naturally<br>
occurring or man-made recombinant protein, such as pharmaceutical proteins [e.g.,<br>
growth factors and antibodies Schillberg Naturwissenschaflen. (2003) Apr,90(4):145-<br>
55] and food additives. It will be appreciated that molecular farming is a well-proven<br>
way of producing a range of recombinant proteins, as described in details in Ma Nat<br><br>
Rev    Genet    2003    Oct;4( 10): 794-805;    Twyman   Trends    Biotechnol.    2003<br>
Dec;21(12):570-8.<br>
An expression product which may be utilized in this invention may include,<br>
but not limited to, pharmaceutical polypeptides, industrial enzymes, oils, dyes,<br>
flavors, biofuels, or industrial biopolymers.<br>
In cases of bidirectional DREs, the nucleic acid construct of this invention<br>
may include two heterologous polynucleotides each being operably linked to an end<br>
of the isolated polynucleotide of this invention, such that the two heterologous<br>
polynucleotides flank the isolated polynucleotide of this invention.<br>
The nucleic acid construct can be, for example, a plasmid, a bacmid, a<br>
phagemid, a cosmid, a phage, a virus or an artificial chromosome. Preferably, the<br>
nucleic acid construct of the present invention is a plasmid vector, more preferably a<br>
binary vector.<br>
The phrase "binary vector" refers to an expression vector which carries a<br>
modified T-region fi-om Ti plasmid, enable to be multiplied both in E. coli and in<br>
Agrobacterium cells, and usually comprising reporter gene(s) for plant transformation<br>
between the two boarder regions. A binary vector suitable for the present invention<br>
includes pBI2113, pBI121, pGA482, pGAH, pBIG, pBIlOl (Clonetech), or a<br>
modification thereof such as pVERl which is a modified pBIlOl plasmid, where the<br>
GUS gene was replaced by the LucII gene fi-om pGL3-Basic (Promega).<br>
The nucleic acid construct of the present invention can be utilized to transform<br>
a host cell. Thus, according to another aspect of the present invention there is<br>
provided a transgenic cell, a transgenic organism or a transgenic plant which is<br>
transformed with an isolated polynucleotide of the present invention. Preferably the<br>
transgenic cell, the transgenic organism or the transgenic plant is transforrned with the<br>
nucleic acid construct of the present invention.<br>
As used herein, the terms "transgenic" or "transfonned" are used<br>
interchangeably referring to a cell or an organism into which cloned genetic material<br>
has been transferred.<br>
Methods of introducing nucleic acid constructs into a cell, an organism or a<br>
plant are well known in the art. Accordingly, suitable methods for introducing<br>
nucleic acid sequences into plants include, but are not limited to, bacterial infection,<br>
direct     deliver}'     of     DNA     (e.g.,      via     PEG-mediated     transformation,<br>
(8-<br><br>
desiccation/inhibition-mediated DNA uptake, electroporation, agitation with silicon<br>
carbide fibers, and acceleration of DNA coated particles, such as described by<br>
Potiykus Ann. Rev. Plant Physiol. Plant Mol. Biol. 42:205-225 (1991).<br>
Methods for specifically transforming dicots primarily xise Agrobacterium<br>
tumefaciens. For example, transgenic plants reported include but are not limited to<br>
cotton (U.S. Pat. Nos. 5,004,863. 5,159,135, 5,518,908; and WO 97/43430), soybean<br>
[U.S. Pat. Nos. 5,569,834, 5,416,011; McCabe et al, Bio/Technology, 6:923 (1988);<br>
and Christou et al. Plant Physiol., 87:671, (1988)]; Brassica (U.S. Pat. No.<br>
5,463,174),   and  peanut  [Cheng  et  a/..  Plant   Cell   Rep.,   15:   653,  (1996)].<br>
Similar methods have been reported in the tr^sformation of monocots.<br>
Transformation and plant regeneration using these methods have been described for a<br>
number of crops including but not limited to asparagus [Asparagus officinalis;<br>
Bytebier et al, Proc. Natl. Acad. Sci. U.S.A., 84: 5345, (1987); barley (Hordeum<br>
vulgarae; Wan and Lemaux, Plant Physiol., 104: 37, (1994)]; maize [Zea mays;<br>
Rhodes, C. A., et a/.. Science, 240: 204, (1988); Gordon-Kamm, et al. Plant Cell, 2:<br>
603, (1990); Froram, et al, Bio/Technology, 8: 833, (1990); Koziel, et al.,<br>
Bio/Technology, 11: 194, (1993)]; oats [Avena sativa; Somers, et al.,<br>
Bio/Technology, 10: 1589, (1992)]; orchardgrass [Daciylis glomerata; Horn, et al..<br>
Plant Cell Rep., 7: 469, (1988); rice [Oryza sativa, including indica and japonica<br>
varieties, Toriyama, et al, Bio/Technology, 6: 10, (1988); Zhang, et al.. Plant Cell<br>
Rep., 7: 379, (1988); Luo and Wu, Plant Mol. Biol. Rep., 6: 165, (1988); Zhang and<br>
Wu, Theor. Appl. Genet, 76: 835, (1988); Christou, et al., Bio/Technology, 9: 957,<br>
(1991); sorghum [Sorghum bicolor, Casas, A. M., et al., Proc. Natl. Acad. Sci.<br>
U.S.A., 90: 11212, (1993)]; sugar cane [Saccharum spp.; Bower and Birch, Plant J., 2:<br>
409, (1992)]; tall fescue [Festuca arundinacea; Wang,.Z. Y, et al., Bio/Technology,<br>
10: 691, (1992)]; turfgrass [Agrostis paliistris; Zhong'ef al. Plant Cell Rep., 13: 1,<br>
(1993)]; wheat [Triticum aestivum; Vasil et al., Bio/Iechnology, 10: 667, (1992);<br>
Weeks T., et al. Plant Physiol., 102: 1077, (1993); Becker, et al. Plant, J. 5: 299,<br>
(1994)], and alfalfa [Masoud, S. A., et al, Transgen. Res., 5: 313, (1996)]. It is<br>
apparent to those of skill in the art that a number of transformation methodologies can<br>
be used and modified for production of stable transgenic plants from any number of<br>
target crops of interest.<br><br>
The transformed plants can be analyzed for the expression features conferred<br>
by the polynucleotides of the present invention, using methods known in the art for<br>
the analysis of transformed plants. A variety of methods are used to assess gene<br>
expression and determine if the introduced gene(s) is integrated, functioning properly,<br>
and inherited as expected. Preferably, the promoters can are evaluated by determining<br>
the expression levels and the expression features of genes to which the promoters are<br>
operatively linked. A preliminary assessment of promoter function can be determined<br>
by a transient assay method using reporter genes, but a more definitive promoter<br>
assessment can be determined fi-om the analysis of stable plants. Methods for plant<br>
analysis include but are not limited to Southern blots or northern blots, PCR-based<br>
approaches, biochemical analyses, phenotypic screening methods, field evaluations,<br>
and immunodiagnostic assays.<br>
Preferably, the capacity of isolated polynucleotides of this invention to<br>
promote gene expression in plants is evaluated according to phenotypic expression of<br>
reporter genes using procedures as described in the Examples section that follows.<br>
Briefly, the expression of luciferase in transgenic Arabidopsis is determined and<br>
consistently classified by quantitatively scoring certain features of expression, such as<br>
the intensity, specificity, development stage and positioning of expression.<br>
i\ccordingly, a luciferase gene that is expressed in a constitutive manner would<br>
indicate a putative constitutive promoter activity of the isolated polynucleotide.<br>
Likewise, a luciferase gene that is expressed in an inductive, tissue specific or a<br>
development-stage specific manner, would respectively indicate a putative inductive,<br>
a tissue specific or a stage specific promoter activity.<br>
Hence, the present invention provides a plurality of isolated polynucleotide<br>
sequences which exhibit a wide spectrum of promoter function patterns. These<br>
polynucleotides can be used to generate nucleic acid constructs, such as expression<br>
vectors suitable for transforming an organism. Such nucleic acid constructs can be<br>
used to promote expression of desired traits or expression products in transgenic<br>
organisms, such as plants, in a constitutive, induced, tissue specific, or a<br>
developmental stage specific manner.<br>
Additional objects, advantages, and novel features of the present invention<br>
will become apparent to one ordinarily skilled in the art upon examination of the<br>
following examples, which are not intended to be limiting. Additionally, each of the<br><br>
various embodiments and aspects of the present invention as delineated hereinabove<br>
and as claimed in the claims section below finds experimental support in the<br>
following examples.<br>
EXAMPLES<br>
Reference is now made to the following examples, which together with the<br>
above descriptions, illustrate the invention in a non limiting fashion.<br>
Generally, the nomenclature used herein and the laboratory procedures utilized<br>
in the present invention include molecular, biochemical, microbiological and<br>
recombinant DNA techniques. Such techniques are thoroughly explained in the<br>
literature. See, for example, "Molecular Cloning: A laboratory Manual" Sambrook et<br>
al, (1989); "Current Protocols in Molecular Biology" Volumes I-III Ausubel, R. M.,<br>
ed. (1994); Ausubel et al., "Current Protocols in Molecular Biology", John Wiley and<br>
Sons, Baltimore, Maryland (1989); Perbal, "A Practical Guide to Molecular Cloning",<br>
John Wiley &amp; Sons, New York (1988); Watson et al, "Recombinant DNA",<br>
Scientific American Books, New York; Birren et al. (eds) "Genome Analysis: A<br>
Laboratory Manual Series", Vols. 1-4, Cold Spring Harbor Laboratory Press, New<br>
York (1998); methodologies as set forth in U.S. Pat. Nos. 4,666,828; 4,683,202;<br>
4,801,531; 5,192,659 and 5,272,057; "Cell Biology: A Laboratory Handbook",<br>
Volumes I-III Cellis, J. E., ed. (1994); "Cunrent Protocols in Immunology" Volumes<br>
I-ni Coligan J. E., ed. (1994); Stites et al. (eds), "Basic and Clinical Immunology"<br>
(8th Edition), Appleton &amp; Lange, Norwalk, CT (1994); Mishell and Shiigi (eds),<br>
"Selected Methods in Cellular Immunology", W. H. Freeman and Co., New York<br>
(1980); available immunoassays are extensively described in the patent and scientific<br>
literature, see, for example, U.S. Pat. Nos. 3,791,932; 3,839,153; 3,850,752;<br>
3,850,578; 3,853,987; 3,867,517; 3,879,262; 3,901,654; 3,935,074; 3,984,533;<br>
3,996,345; 4,034,074; 4,098,876; 4,879,219; 5,011,771 and 5,281,521;<br>
"Oligonucleotide Synthesis" Gait, M. J., ed. (1984); "Nucleic Acid Hybridization"<br>
Hames, B. D., and Higgins S. J., eds. (1985); "Transcription and Trmislation" Hames,<br>
B. D., and Higgins S. J., eds. (1984); "Animal Cell Culture" Freshney, R. I., ed.<br>
(1986); "Immobilized Cells and Enzymes" IRL Press, (1986); "A Practical Guide to<br>
Molecular Cloning" Perbal, B., (1984) and "Methods in Enzymology" Vol. 1-317,<br>
Academic Press;  "PCR Protocols:  A  Guide To Methods  And  Applications",<br><br>
Academic Press, San Diego, CA (1990); Marshak et al, "Strategies for Protein<br>
Purification and Characterization - A Laboratory Course Manual" CSHL Press<br>
(1996); all of which are incorporated by reference as if fully set forth herein. Other<br>
general references are provided throughout Ihis document. The procedures therein are<br>
believed to be well known in the art and are provided for the convenience of the<br>
reader. All the information contained therein is incorporated herein by reference.<br>
IDENTIFICATION, ISOLATION AND CHARACTERIZATION OF DNA<br>
REGULA TING ELEMENTS (DREs)<br>
Novel DREs were identified by luciferase expression assay driven by<br>
bioinformatically identified DNA fhigments from Arabidopsis thaliana genomic DNA.<br>
Positive DREs were fused upstream a reporter gene in a vector which was used to<br>
transform Arabidopsis thaliana plants. The reporter gene expression driven by these<br>
DREs was characterized.<br>
MA TERIALS AND EXPERIMENTAL METHODS<br>
Isolation of DREs: A high throughput method of cloning DNA regulating<br>
elements (DREs) using a single reaction tube, referred to herein as the "one-tube"<br>
method, was utilized in order to enable large scale production of DRE transformed<br>
plants. Accordingly, genomic DNA (gDNA) was extracted from leaves of<br>
Arabidopsis thaliana Coll using DNAeasy Plant Mini Kit (Qiagen, Germany).<br>
Primers for PCR amplification of DREs were designed using PRIMERSÂ® software<br>
and modified to contain restriction sites absent from the DRE sequence, for PCR<br>
product insertion into pVERl binary plasmid, which is a pBIlOl (clontech) modified<br>
plasmid, where the GUS reporter gene was replaced by LucII gene from pGL3-Basic<br>
(promega). Briefly, GUS gene was cut out of pBIlOl using the blunt restriction<br>
enzymes Ecll36II and Smal. The pGL-Basic plasmid [after eliminating the Hindlll<br>
and BamHI sites, by digestion, fill-in using klenow fragment (Roche) and self ligating<br>
the plasmid, using T4 DNA ligase (Roche)] was cut Sad and Xbal and the LucII gene<br>
insert was inserted into pBluescript, digested with the same enzymes. The new<br>
plasmid was digested Smal, as a result a blunt ends LucII gene was cut out. The LucII<br>
gene was inserted into The pBI plasmid instead of the GUS gene. To eliminate all<br>
possible read-through of the Nos-promoter, which regulates Kanamycin resistance<br><br>
gene on pBIlOl, a poly-A signal was added between the Nos-terminator and the LucII<br>
gene. Poly-A signal was amplified fi-om pGL3-Basic using proof reading Taq<br>
polymerase PFU (Promega) and using primers 5'-aggtacttggagcggccgca-3' and 5'-<br>
tagagaaatgttctggcacctg-3'. The Product was inserted into Hindlll site on pVerl after<br>
filling the overhang 5' ends, using Klenow fragment (Roche).<br>
Polymerase chain reaction analyses were performed using Taq Expand Long<br>
Template PCR kit (Roche), according to the manufacturer's instructions, using as<br>
thermal cycle: 92 Â°C/2 min -* 10 x [94 Â°C/10 min -^ 55 Â°C/30 sec -* 68 Â°C/5 min]<br>
-* 18 X [94 Â°C/10 min -* 55 Â°C/30 sec -^ 68 Â°C/5 min (+ 20 sec each cycle)] -* 68<br>
"C/? min. PCR products were double-digested with- restriction endonucleases<br>
according to the protocols described in Table 1.<br><br>
		Table 1	.* DRE double digestion protocols		<br>
En2yme<br>
combination	First<br>
digest	Buffer<br>
(Roche)	Digest<br>
time<br>
(min)	Heat<br>
inactivation<br>
conditions	Second<br>
digest	Buffer	Digest<br>
time<br>
(min)	Heat<br>
inactivation<br>
conditions<br>
Hindlll, Sail	Hindm	M	90	20 min,<br>
VCC	Sail	M +<br>
NaCl +<br>
Tris	60	20 min,<br>
70 "C<br>
Hindlll,<br>
BamHI	Hindlll	B	30	No	BamHI	B	60	20 min,<br>
10 "C<br>
Sail, BamHI	BamHI	M	60	20 min,<br>
sec	Sail	M +<br>
NaCl +<br>
Tris	60	20 min,<br>
70 "C<br>
Hindlll,<br>
EcoRV	Hindlll	B	30	No	EcoRV	B	60	20 min,<br>
70 "C<br>
Sail, Seal	Sail, Seal	H	60	20 min,<br>
80 Â°C				<br>
BamHI, Smal	Smal	A	60<br>
(30 "O	20 min,<br>
10 "C	BamHI	A	60	20 min,<br>
sec<br>
Sail, PvuO	PvuU	M	60	20 min,<br>
80 Â°C	Sail	M +<br>
NaCl +<br>
Tris	60	20 min<br>
Hindni,<br>
PvuU	Hindni	M	30	No	PvuII	M	60	20 min,<br>
sec<br>
HJndlll, StuI	HindUl,<br>
StuI	B	90	20 min,<br>
sec				<br>
BamHI, StuI	StuI	B	30	No	BamHI	B	60	20 min,<br>
80 "C<br>
Cloning of DREs in luciferase reporter gene expression: PCR amplified<br>
DREs were cloned into a luciferase reporter gene expression vector pVERl, derived<br>
from the binary vector pBIlOl (Clontech), was double-digested using the same<br>
restriction endonucleases used to excise cloned DREs from vector, purified using<br>
^S<br><br>
PCR Purification Kit (Qiagen, Germany), treated with alkaline-phophatase (Roche)<br>
according to the manufacturer's instructions and re-purified using PCR Purification<br>
Kit (Qiagen, Germany). Insertion of DRE into vector pVERl was performed by<br>
adding to DRE digests: 500 ng of double digested pVerl plasmid, 1 |j.l of T4 DNA<br>
ligase (40 U/jil; Roche) and 6 p.1 of T4 buffer (Roche). Following overnight<br>
incubation of ligation mixes at 4 Â°C, Agrobacterium tumefaciens GV303 competent<br>
cells were transformed using 1-2 fxl of ligation reaction mixture by electroporation,<br>
using a MicroPulser electroporator (Biorad), 0.2 cm cuvettes (Biorad) and EC-2<br>
electroporation program (Biorad). Agrobacterium cells were grown on LB at 28 Â°C<br>
for 3 h and plated on LB-agar plates supplemented with the antibiotics gentamycin 50<br>
mg/L (Sigma) and kanaraycm 50 mg/L (Sigma). Plates were then incubated at 28 Â°C<br>
for 48 h. Cloned DREs were identified by PCR analysis of bacterial colony DNA<br>
using the vector specific, insert flanking upstream and downstream primers 5'-<br>
AGGTACTTGGAGCGGCCGCA-3' and 5'-CGAACACCACGGTAGGCTG-3',<br>
respectively and the thermal cycle: 92 Â°C/3 min -â™¦ 31 x [94 Â°C/30 sec -&gt; 54 Â°C/30<br>
sec -^ 72 Â°C/^min (^= length (kb) of longest PCR product expected)] -^ 72 Â°C/10<br>
min. Positive Agrobacterium colonies were siibsequently used for Arabidopsis plant<br>
transformation.<br>
Plant transformation and cultivation: Arabidopsis thaliana Columbia (To<br>
plants) were transformed using the Floral Dip procedure described by Clough SJ and<br>
Bent AF [The Plant J. 16:735-743 (1998)] and by Desfeux et al. [Plant Physiology<br>
123:895-904 (2000)] with minor modifications. Briefly, To Plants were sown in 250<br>
ml pots filled with wet peat-based growth mix. The pots were covered with<br>
aluminum foil and a plastic dome, kept at 4 Â°C for 3 - 4 days, then uncovered and<br>
incubated in a growth chamber at 18 - 24 Â°C under 16/8 hr light/dark cycle. The To<br>
plants were ready for transformation six days before anthesis.<br>
Single colonies of Agrobacterium carrying plant DREs were cultured in LB<br>
medium supplemented with kanamycin (50 mg/L) and gentamycin (50 mg/L). The<br>
cultures were incubated at 28Â°C for 48 hoiffs under vigorous shaking and centrifiiged<br>
at 4000 rpm for 5 minutes. The pellets comprising Agrobacterium cells were<br>
resuspended in a transformation medium which contained half-strength (2.15 g/L)<br>
Murashig-Skoog (Duchefa); 0.044 fAM benzylamino purine (Sigma); 112 ng/L B5<br>
c^^<br><br>
Gambourg vitamins (Sigma); 5% sucrose; and U.l ml/L biiwet L- / / {Ubl Specialists,<br>
CT) in double-distilled water, at pH of 5.7.<br>
Transformation of To plants was effected by inverting each plant into an<br>
Agrobacterium suspension such that the above grovmd plant tissue was submerged for<br>
3-5 seconds. Each inoculated To plant was immediately placed in a plastic tray, then<br>
covered with clear plastic dome to maintain humidity and kept in the dark at room<br>
temperature for eighteen hours to facilitate infection and transformation.<br>
Transformed (transgenic) plants were then imcovered and transferred to a greenhouse<br>
for recovery and maturation. The transgenic To plants were grown in the greenhouse<br>
for 3-5 weeks until siliques were brown and dry then seeds were harvested from<br>
plants and kept at room temperature until sowing<br>
Generating Tj and T2 transgenic plants harboring DREs: Seeds collected<br>
from transgenic TQ plants were surface-sterilized by soaking in 70% ethanol for 1<br>
minute, followed by soaking in 5% sodium hypochloride and 0.05% triton for 5<br>
minutes. The surface-sterilized seeds were thoroughly washed in sterile distilled<br>
water then placed on culture plates containing half-strength Murashig-Skoog<br>
(Duchefa); 2% sucrose; 0.8% plant agar, 50 mM kanamycin; and 200 mM<br>
carbenicylin (Duchefa). The culture plates were incubated at 4Â°C for 48 hours then<br>
transferred to a growth room at 25Â°C for an additional week of incubation. Vital Ti<br>
Arabidopsis plants were transferred to a fresh culture plates for another week of<br>
incubation. Following incubation the Ti plants were removed from culture plates and<br>
planted in growth mix contained in 250 ml pots. The transgenic were allowed to grow<br>
in a greenhouse to maturity. Seeds harvested from Ti plants were cultured and grown<br>
to maturity as T2 plants under the same conditions as used for culturing and growing<br>
the Ti plants.<br>
Evaluating DRE gene-promoting activity in transgenic plants: The ability of<br>
DREs to promote gene expression in plants was determined based on the expression<br>
of luciferase reporter gene. Accordingly, fransgenic i4ra6Wop5Â£s plantlets at a<br>
development stage of 2-3 true leaves were subjected to lummescence assays using the<br>
procedure described by Messinner R. [Plant J. 22:265 (2000)]. The imaging of<br>
luciferase was performed in a darkroom using ultra-low light detection camera<br>
(Princeton Instruments Inc., USA). Using the procedure described by Messinner R.<br>
[Plant J. 22:265 (2000)].<br><br>
Scoring promoter activity in triuisgenic plants: DREs promoting gene<br>
expression was characterized based luciferase expression in transgenic plants using<br>
quantitative values such as to enable consistent evaluations of a large volume of<br>
transgenic plants, as follows:<br>
Scoring distribution and intensity of expression: The distribution of reporter<br>
genes' expression in transgenic plants was presented in a three variables fimctions, as<br>
follows: (i) plant ID (X axis), (ii) plant organ (Y axis), and (iii) development stage (Z<br>
axis). The intensity of expression, relevant to any of these three variables, was<br>
measured by a distribution function value (DF), referred hereinbelow as<br>
f^y^(^T077Wter).   The DF received a value ranging from 0 to 5, representing no<br>
expression and the highest expression intensity, respectively.<br>
Scoring specificity of expression: The specificity of reporter genes'<br>
expression in transgenic plants was calculated by summing two independent addends:<br>
(a) the zero value/nonzero values ratio, as described in table 2 below and which<br>
further referred to as the Binary Function B{ ); and (b) the variance of the nonzero<br>
values only.<br>
Table 2<br><br>
	No. of zero values<br>
No. of non zero<br>
values	0	1	2	3	4<br>
0		0	0	0	0<br>
1	0	0.7	1.5	2	<br>
2	0	0.6	1		<br>
3	0	0.5			<br>
4	0				<br>
The Organ Specificity expression value (SpOr) was calculated according to<br>
the following equation:<br>
SpOr {promoter    )=Var ^ {AV ,, (/,,,., {promoter   ))l^,o )+ B{AV ,. {f,^. {promoter    )))<br>
Whereas Var is the variance, Av is the average and B is the Binary Function.<br>
The development Stage Specificity expression value (SpDs) was calculated<br>
according to the following equation:<br>
SpDS {promoter )=Var.(Av^y(f^y^{pro}noter ))l:&gt;o)+^Uv,j,(/,^_,(pro/;jofer )))<br>
Whereas Var is the variance, Av is the average and B is the Binary Function.<br>
Scoring position effect:   Similarly to the Binary Factor approach described<br>
above, position values  were also classified as either zero or nonzero values.<br><br>
Accordingly, the reporter genes' expression in a given organ in a given development<br>
stage was measured by a Local Position Effect value (LoPoEf). The Position Effect<br>
value (PoEf) was the average of all the Local Position Effects, calculate in three steps<br>
as follows:<br>
U J,    (	, \   P       f,^y^{promoteij = 0<br>
I-) L^.[promoted =i      ,    /	\   ,Â«,.-â€¢<br>
xj.U'	/   ij   /,^^^(promoH = 1.2,3.4,5<br>
2)<br>
LoPoEf {promoter , organ ,developmen   t _ stage )=<br>
(     no _of _0s_in[k^^,y^^z {promoter  )}       no _of _nan _0s_m{h^ â€žY^-Z {promoter ))<br>
l^no_of _nan _0s_in(/i,.r,y^.z{promoter ))'      no_of _Qs_in{h^^,,..^^{promoter ))<br>
3) PoEf {promoter )= Av{LoPoEf {promoter ,Y,Z^.<br>
Scoring expression level: The average expression level value (ExLe) and the<br>
EjcLe variance (VrExLe) were calculated per each DRE promoter x plant organ x<br>
plant development stage combination, according the following equations:<br>
ExLe{promoter, organ, development _ stage) =Av^ {fx,y^ {promoter))<br>
VrExLe{promoter, organ, de\&gt;elopment _ stage) = var, {f^^y^ {promoter)).<br>
Scoring evaluation reliability: The General Reliability value (Grel) was the<br>
number of independent plants that were used for evaluating a specific DRE promoter<br>
activity.   Hence,   GKel{promoter)=Count^{/^^^{promoter)).   The  Development<br>
Stage Reliability value (Rel(DS)) was the number of independent plants that were<br>
used for evaluating a specific DRE proraotCT activity in any given plant developing<br>
stage.   Rel{promoter, developmeiU _ stage) = Coimt^,^^^,,^^ _^^ {f^^^ {promoter)).<br>
Creation of partial fragments from vDREs 4209 and 6669: Genomic DNA<br>
derived from Arabidopsis thaliana var ColO was extracted and PCR-amplified using<br>
oligonucleotide primers complementary to sequences within vDRE 4209 (SEQ ID<br>
NO:36) [sense-primer 5'- GTTGGTTCGTCGACTAGAGAAGGT -3' (SEQ ID NO:<br>
208), antisense primer 5'- TTGGATCCGGGAGGCAATGATGCTTTAG - 3' (SEQ<br>
ID NO: 209)], and vDRE 6669 (SEQ ID N0:61) [sense primer 5'-<br>
TTGTAAGCTTGCAGGGATACGGATGGGTAG -3' (SEQ ID NO: 211), antisense<br>
primer 5'- AAATATTGGATCCTTTGGGGTTCTC - 3' (SEQ ID NO: 212)].<br><br>
The above PC31 amphfications resulted in a 470 bp firagment, containing bp<br>
76-548 of the original vDRE 4209 (SEQ ID NO:210) and a 1569 bp fi-agment,<br>
containing bp 748-2316 of the original vDRE 6669 (SEQ ID NO:213), respectively.<br>
PCR products were digested with Hindlll and BainHI and ligated into the<br>
binary vector, pBI121 (Clontech, accession niraiber: AF485783) upstream to the GUS<br>
gene, generating plasmids p4209short-GUS , and p6669short-GUS, respectively.<br>
Arabidopsis plants (var colO) were transformed with the binary constructs generated<br>
(p4209short-GUS and p6669short-GUS), and GUS activity was analyzed on 10<br>
independent Tl transformed plants using standard GUS staining protocol [Jefferson<br>
RA, Kavanagh TA, Bevan MW. 1987. GUS fusions: betaTglucuronidase as a sensitive<br>
and versatile gene fusion marker in higher plants. EMBO J. 6(13): 3901-7]. Genomic<br>
DNA extraction, PCR amplification, DNA restriction, ligation and transformation of<br>
Arabidopsis plant were preformed according to the protocols described above.<br>
EXPERIMENTAL RESULTS<br>
Characterization of DREs:<br>
Various features of the isolated DREs of the present invention are described in<br>
Tables 3-17 which follow. As is clearly evident firom the Table provided data, the<br>
DREs of the present invention exhibit a wide range of gene-promoting activities<br>
including: constitutive, inductive, tissue specific, and stage specific activities.<br>
J^<br><br>
Tables<br><br>
DRE number '	1345	1495	2176<br>
Cluster reference *	Z18125	Z17428	ATBIBBI<br>
Cluster position ^	Upstream	Upstream	Upstream<br>
DRE regulatory direction ^	Bidirectional	Unidirectional	Unidirectional<br>
DRE lengtii (bp)	1611	901	2192   â€¢<br>
DRE sequence	SEO ID NO: 1	SEQIDN0:6	SEO ID NO: 11<br>
Internal for&gt;vard primer<br>
sequence^	SEQIDNO:2	SEQIDN0:7	SEQ ID NO: 12<br>
External forward primer<br>
sequence^	SBQIDN0:3	SEQ ID NO: 8	SEQ ID NO: 13<br>
Internal reverse primer<br>
sequence^	SEQIDN0:4	SEQ ID NO: 9	SEQ ID NO: 14<br>
External reverse primer<br>
sequence^	SEQIDN0:5	SEQ ID NO: 10	SEQ ID NO: 15<br>
Position Effect Value *	0.37	0.21	8.33<br>
Development Stage<br>
Specificity Value *	1.09	0.32	0.62<br>
Organ Specificity Value *	1.56	0.38	2.60<br>
Number of transgenic<br>
plants	11	10	7<br>
Young roots score (No^<br>
Ave., Var) *	10,12,3.36	6,2333,1.555	4,0,0<br>
Mature roots score (No.,<br>
Ave., Var) *	7,1.571,3.387	7,3,2.285	5.0,0<br>
Young above-ground<br>
Tissue (TNo., Ave., Var) *	10,3.3,2.21	6,4.16,0.13	4,0.0<br>
Mature above-ground<br>
tissue (No., Ave., Var)'	7,3,2	7, 3.28, 1.06	5,0,0<br>
Siliques/Seed (No., Ave.,<br>
Var)*	3,4.33,0.88	3.2,2	3,1.67,5.56<br>
Flowers (No., Ave., Var)'	7,1.42,3.10	7,3.14,4.40	5,4.2,2.56<br>
Description	Constitutive.<br>
Strong in seeds.	Constitutive.	Specific to flower tissue.<br>
Strong in flower buds.<br>
Lower expression in open<br>
floweis.<br>
' ID number of the DRE as assigned by the present inventors.<br>
^ Internal reference assigned by the present inventors to a cluster of Arabidopsis genes (contig)<br>
downstream or upstream of the DRE.<br>
^ Unidirectional implies that only tiie sense strand of Ihe DRE is enable of regulating gene e;q}ression;<br>
Bidirectional implies that both the sense and antisense strands of tiie DRE are capable of regulating gene<br>
expression.<br>
* A PCR primer for isolating die DRE from Arabidopsis genomic DNA.<br>
^ Position Effect Values (PoEf), Development Stage-Specificity Values (SpDs) and Organ Specificity<br>
Values (SpOr) were calculated as described in the materials and methods section hereinabove.<br>
' No. = number: Ave. = average; Var. = variance.<br><br>
Table 4<br><br>
DRE number '	2524	3560	3583<br>
Cluster reference'	Z17778	Z17937	av558751<br>
Cluster position ^	Upstream	Upstream	Upstream<br>
DRE regulatory direction	Bidirectional	Bidirectional	Bidirectional<br>
DRE length (bp)	1975	3126	2501<br>
DRE sequence	SE0IDNO:16	SE0IDNO:21	SEOIDNO:26<br>
Internal forward primer<br>
sequence*	SEQIDN0:17	SEQIDNO:22	SEQ ID NO:27<br>
External forward primer<br>
sequence*	SEQIDN0:18	SEQIDNO:23	SEQIDNO:28<br>
Internal reverse primer<br>
sequence*	SEQIDN0:19	SEQ ID NO:24	SEQIDNO-29<br>
External reverse primer<br>
sequence*	SEQIDNO:20	SEQIDNO:25	SEQIDNO:30<br>
Position Effect Value ^	0.15	0.3	5.555<br>
Development Stage<br>
Speciflcity Value *	0.69	0.77	1.5<br>
Organ Specificity Value *	1.16	1.14	2<br>
Number of transgenic<br>
plants	8	11	6<br>
Young roots score (No.,<br>
Ave., Var)'	5,0,0	6,3.5, 1.92	5,0,0<br>
Mature roots score (No.,<br>
Ave., Var) *	5.0.0	7,3.71,1.63	4,0,0<br>
Young above-ground<br>
Tissue (No., Ave., Var) *	5, 0.4, 0.24	6,1.83,2.14	5,0,0<br>
Mature above-ground<br>
tissue (No., Ave., Var) '	5,2,0.8	7,1.43,1.10	4,0.25,0.19<br>
Siliques/Seed (No., Ave.,<br>
Var)'	3,0,0	3,0.67,0.89	3,0,0<br>
Flowers (No., Ave., Var) *	5, 0.4, 0.64	7,1.86,1.55	4,0,0<br>
Description	Specific to above ground<br>
tissue.	Specific to root tissue.<br>
Strong expression,<br>
mainly in root meristems.<br>
Weak expression in above<br>
ground tissues.	Weak in above ground<br>
tissue.<br>
[D number of the DRE as ass	igned by the present inve	ntors.	<br>
^ Internal reference assigned by the present inventors to a cluster of Arabidopsis genes (contig)<br>
downstream or upstream of the DRE.<br>
^ Unidirectional implies that only the sense strand of the DRE is capable of regulating gene expression;<br>
Bidirectional implies that both the sense and antisense strands of the DRE are capable of regulating gene<br>
expression.<br>
* A FOR primer for isolating the DRE from Arabidopsis genomic DNA.<br>
^ Position Effect Values (PoEO, Development Stage-Specificity Values (SpDs) and Organ Specificity<br>
Values (SpOr) were calculated as described in the materials and methods section hereinabove.<br>
' No. = number, Ave. = average; Var. = variance.<br>
$0<br><br>
Tables<br><br>
DRE number '	3714	4209	5095<br>
Cluster reference ^	Z25961	229176	AI996150<br>
Cluster position ^	Upstream	Downstream	Downstream<br>
DRE regulatory direction	Unidirectional	Bidirectional	Bidirectional<br>
DRE length (bp)	513	1022	1056<br>
DRE sequence	SE0IDN0:31	SEOIDNO:36	SE0IDNO:41<br>
Internal forward primer<br>
sequence^	SEQIDNO:32	SEQ ID NO:37	SEQ ID NO:42<br>
External forward primer<br>
sequence^	SBQ ro NO:33	SEQ ID NO:38	SEQIDNO:43<br>
Internal reverse primer<br>
sequence^	SEQIDNO:34	SEQIDNO:39	SEQ ID NO:44<br>
External reverse primer<br>
sequence^	SEQroNO:35	SEQIDNO:40	SEQIDNO:45<br>
Position Effect Value ^	0.3625	0.40	0.6<br>
Development Stage<br>
Speciflcity Value	0.11241	0.57	0<br>
Organ Specificity Value *	0.377	0.40	0.85<br>
Number of transgenic<br>
plants	11	18	3<br>
Young roots score (No.,<br>
Ave., Var) *	9,0.611,0.987	14,3.46,2.87	2, 0.5, 0.25<br>
Mature roots score (No.,<br>
Ave., Var)*	3,0,0	9,2.11,3.65	2 1  1<br>
Young above-ground<br>
Tissue (No., Ave., Var) *	9,2.38,1.20	14.2.89,2.36	2,1.5,1.25<br>
Mature above-ground<br>
tissue (No., Ave., Var)'	3,1,2	9,2.44,1.80	2,2.0<br>
Siliques/Seed (No., Ave.,<br>
Var)'	3,1.66,0.22	3, U3, 1.56	Not available<br>
Flowers (No., Ave., Var) *	3,1.33,3.55	9,2,3.78	2.0.0<br>
Description	Weak in above ground tisAie	Strong in roots, mainly roo<br>
tips, and flower buds.<br>
Lower expression in veins.<br>
Very low expression in<br>
seeds.	Constitutive, weak.<br>
' ID number of the DRE as assigned by the present inventors.<br>
^ Internal reference assigned by the present inventors to a cluster of Arabidopsis genes (contig)<br>
downstream or upstream of the DRE.<br>
â– 'Unidirectional implies that only the sense strand of the DRE is capable of regulating gene expression;<br>
Bidirectional implies that both the sense and antisense strands of the DRE are cq)able of regulating gene<br>
expression.<br>
* A PCR primer for isolating flie DRE from Arabidopsis genomic DNA.<br>
^ Position Effect Values (PoEQ, Development Stage-Specificity Values (SpDs) and Organ Specificity<br>
Values (SpOr) were calculated as described in the materials and methods section hereinabove.<br>
' No. = number; Ave. = average; Var. = variance.<br>
5<br><br>
Table 6<br><br>
DRE number '	5311	5532	5587<br>
Cluster reference ^	ATHC0L2A	ATASCO	Z26363<br>
Quster position ^	Upstream	Upstream	Upstream<br>
DSÂ£ regulatory direction	Bidu%ctional	Unidirectional	Unidirectional<br>
DRE length (bp)	435	3348	1331<br>
DRE sequence	SE0IDNO:46	SE0IDNO:51	SEQ ID NO:56<br>
Internal forward primer<br>
sequence^	SEQIDNO:47	SEQ ID NO:52	SEQ ID NO:57<br>
External forward primer<br>
sequence^	SEQ ID NO: (none)	SEQIDNO:53	SEQIDN0.58<br>
Internal reverse primer<br>
sequence*	SEQIDNO:49	SEQIDNO:54	SEQ ID NO:59<br>
External reverse primer<br>
sequence*	SEQ ID NO: (none)	SEQIDNO:55	SEQ ID NO:60<br>
Position Effect Value ^	0.36	0.25	8.33<br>
Development Stage<br>
Specificity Value	1.15	1.30932	1.5<br>
Organ Specificity Value *	0.332	1.246	2<br>
Number of transgenic<br>
plants	8	6	4<br>
Young roots score (No^<br>
Ave^Var)*	7, 0.36,0.48	5,2.2,1.36	4,0,0<br>
Mature roots score (No.,<br>
Ave^ Var) *	4, 0.5, 0.75	4,4.25,0.187	3,0,0<br>
Young above-ground<br>
Tissue (No., Ave., Var) *	7,1.57,1.74	5,3.6,1.84	4,0,0<br>
Mature above-ground<br>
tissue (No., Ave., Var)'	4, 1.5,2.25	4,3.5,0.25	3.0,0<br>
Siliques/Seed (No., Ave.,<br>
Var)'	Not available	3,0.67,0.22	3,1.33,3.55<br>
Flowers (No., Ave., Var) *	4,0.25,0.18	4,2,4.5	3,0,0<br>
Description	Constitutive, weak.	Constitutive, mainly in<br>
vegetative tissue.	Siliques specific<br>
High position effect<br>
ID number of the DRE as assigned by the present inventors.<br>
^ Internal reference assigned by the present inventors to a cluster of Arabidopsis genes (contig)<br>
downstream or upstream of the DRE.<br>
^ Unidirectional implies that only tlie sense strand of the DRE is capable of regulating gene expression;<br>
Bidirectional implies that both the sense and antisense strands of the DRE are capable of regulating gene<br>
expression.<br>
*	A PCR primer for isolating the DRE from Arabidopsis genomic DNA.<br>
*	Position Effect Values (PoEf), Development Stage-Specificity Values (SpDs) and Organ Specificity<br>
Values (SpOr) were calculated as described in the materials and methods section hereinabove.<br>
' No. == number, Ave. = average; Var, = variance.<br>
3s?<br><br>
Table 7<br><br>
DRE number '	6669	6762	7357<br>
Cluster reference ^	Z26440	Z17588	F13952<br>
Cluster position ^	Upstream	Upstream	Upstream<br>
DRE regulatory direction	Unidirectional	Bidirectional	Unidirectional<br>
DRE length (bp)	2316	379	979<br>
DRE sequence	SEO ID N0:61	SEO ID NO:66	SEQIDN0:71<br>
Internal forward primer<br>
sequence^	SEQ ID NO:62	SEQ ID NO:67	SEQ ED NO:72<br>
External forward primer<br>
sequence^	SEQ ID NO:63	SEQ ID NO:68	SEQ ID NO:73<br>
Internal reverse primer<br>
sequence*	SEQ ID NO:64	SEQ ID NO:69	SEQ ID NO:74<br>
External reverse primer<br>
sequence*	SEQIDNO:65	SEQ ID NO:70	SEQ ID NO:75<br>
Position Effect Value ^	0.28	9.72	0.32<br>
Development Stage<br>
Specificity Value *	1.18	0.16	0.6<br>
Organ Specificity Value'	1.32	1.42	0.64<br>
Number of transgenic<br>
plants	4	11	7<br>
Young roots score (No.,<br>
Ave., Var) *	3,2.67, 0.22	8,1.25,4.69	6, 0.5,0.58<br>
Mature roots score (No.,<br>
Ave Var)'	4,4.75,0.19	9,0.33, 0.89	5,0,0<br>
Young above-ground<br>
Tissue (No,, Ave., Var) *	3,3.67, 3.56	8,4.19,0.87	6, 0.43, 0.47<br>
Mature above-ground<br>
tissue (No., Ave., Var) *	4,1,3	9,3.61,1.10	5,0.8,0.16<br>
Siliques/Seed (No., Ave.,<br>
Var)*	4,0.75, 0.69	3,3 J3, 1.56	3.0,0<br>
Flowers (No., Ave., Var) *	4,3,4.5	9,3.11,2.57	5, 1.2,1.36<br>
Description	Specific to young and<br>
meristematic tissue.	Strong in above giound<br>
tissue.	Weak in above ground<br>
tissue.<br>
' ID numbo- of the DRE as assigned by the present inventors.<br>
^ Internal reference assigned by the present inventors to a cluster of Arabidopsis genes (contig)<br>
downstream or upstream of the DRE.<br>
^ Unidirectional implies that only the sense strand of the DRE is capable of regulating gene expression ;<br>
Bidirectional implies that both tiie sense and antisense strands of the DRE are capable of regulating gene<br>
expression.<br>
* A PCR primer for isolating the DRE from Arabidopsis genomic DNA.<br>
^ Position Effect Values (PoEf), Development Stage-Specificity Values (SpDs) and Organ Specificity<br>
Values (SpOr) were calculated as described in the materials and methods section hereinabove.<br>
' No. = number; Ave. = average; Var. = variance.<br>
33<br><br>
Tables<br><br>
DRE number '	7617	8463	9136<br>
Cluster reference ^	Z17636	Z26728	F15462<br>
Ouster position ^	Upstream	Downstream	Downstream<br>
DRE regulatory direction	Bidirectional	Unidirectional	Unidirectional<br>
DRE length (bp)	665	2834	486<br>
DRE sequence	SEO ID NO:76	SE0IDN0:81	SEO ID NO:86<br>
Internal forward primer<br>
sequence^	SEQIDNO:77	SEQIDNO:82	SEQIDNO:87<br>
External forward primer<br>
sequence^	SEQIDNO:78	SEQIDNO:83	SEQ ID NO:88<br>
Internal reverse primer<br>
sequence^	SEQ ID NO:79	SEQIDNO:84	SEQIDNO:89<br>
External reverse primer<br>
sequence^	SEQ ID NO:80	SEQIDNO:85	SEQIDNO:90<br>
Position Effect Value *	0.42	0.16	0.48<br>
Development Stage<br>
Specificity Value	0.16	0.68	0.60<br>
Organ Specificity Value *	0.41	2.02	0.53<br>
Number of transgenic<br>
plants	3	12	13<br>
Young roots score (No^<br>
Ave^ Var)'	3,0,0	6,0,0	9,0.778,2.617<br>
Mature roots score (No.,<br>
Ave., Var) *	3,0,0	7,1.14,3.55	11,0.73,1.107<br>
Young above-ground<br>
Tissue (No., Ave., Var)'	3,0.17,5.56	6,3.33,3.32	9,0.778, 0.84<br>
Mature above-ground<br>
tissue (No., Ave., Var) *	3,0.33, 0.22	7, 2, 2.57	11,1.18,2J1<br>
Siliques/Seed (No., Ave.,<br>
Var)*	2 1  1	4,0,0	3,0,0<br>
Flovfers (No., Ave., Var)'	3,0.33.0.22	7, 3.57,3.96	11,0.55,0.98<br>
Description	Very weak in above ground,<br>
tissue.	Strong in above ground<br>
tissue of seedlings.<br>
Strong in flower tissue of<br>
mature plants.	Constitutive, weak.<br>
' ID number of the DRE as assigned by the present inventors.<br>
^ Internal reference assigned by the present inventors to a cluster of Arabidopsis genes (contig)<br>
downstream or upstream of the DRE.<br>
^Unidirectional implies that only the sense strand of the DRE is capable of regulating gene expression;<br>
Bidirectional implies that both the sense and antisense strands of the DRE are capable of regulating gene<br>
expression.<br>
*	A PCR primer for isolating the DRE from Arabidopsis genomic DNA.<br>
*	Position Effect Values (PoEO, Development Stage-Specificity Values (SpDs) and Organ Specificity<br>
Values (SpOr) were calculated as described in the materials and methods section hereinabove.<br>
' No. = number; Ave. = average; Var. = variance.<br>
5V<br><br>
Table 9<br><br>
DRE number '	108'2T	12582	13257<br>
Cluster reference ^	Z30896	Z33953	Z17541<br>
Cluster position ^	Upstream	Downstream	Upstream<br>
DRE regulator)' direction	Bidirectional	Unidirectional	Bidirectional<br>
DRE length (bp)	1840	1665	807<br>
DRE sequence	SE0IDNO:91	SEOIDNO:96	SEQIDNO-.lOl<br>
Internal forward primer<br>
sequence^	SEQIDNO:92	SEQ ID NO:97	SEQ ID NO: 102<br>
External forward primer<br>
sequence^	SEQ ID NO:93	SEQIDNO:98	SEQ ID NO: 103<br>
Interna! reverse primer<br>
sequence^	SEQIDNO:94	SEQIDNO:99	SEQ ID NO: 104<br>
External reverse primer<br>
sequence^	SEQ ID NO:95	SEQ ID NO: 100	SEQ ID NO: 105<br>
Position Effect Value *	0.27	0.19	0<br>
Development Stage<br>
Specificity Value ^	0.50	0.32	1.5<br>
Organ Specificity Value *	8.19	1.38	1.14<br>
Number of transgenic<br>
plants	5	20	2<br>
Young roots score (No^<br>
Ave., Var) *	3,1.67,2.89	18,0.56,1.36	2,0,0<br>
Mature roots score (No.,<br>
Ave., Var) *	4,3.38,4.17	10,0.5,1.05	2,0,0<br>
Young above-ground<br>
Tissue (No., Ave., Var) '	3,2,2.67	18,2.39.3.90	2,0,0<br>
Mature above-ground<br>
tissue (Nc Ave., Var) *	4,3.12,1.55	10,3.2,0.36	2, 2.5,0.25<br>
Siliques/Seed (No., Ave^<br>
Var)*	Not available	3,1.0	2,0,0<br>
Flowers (No., Ave., Var) *	4,3.25, 3.06	10,4.4,1.84	2,2.1<br>
Description	Strong in root and flower<br>
tissue.	Strong in above ground<br>
tissue of seedlings.<br>
Lower expression in matur<br>
plants.	Specific toabove ground<br>
tissue of mature plants.<br>
ID number of the DRE as ass	igned by the present inve	ntors.	<br>
^ Internal reference assigned by the present inventors to a cluster of Arabidopsis genes (contig)<br>
downstream or upstream of the DRE.<br>
â– * Unidirectional implies that only the sense strand of the DRE is capable of regulating gene expression;<br>
Bidirectional implies that both tiie sense and antisense strands of the DRE are capable of regulating gene<br>
expression.<br>
*	A PCR primer for isolating the DRE from Arabidopsis genomic DNA.<br>
*	Position Effect Values (PoEf), Development Stage-Specificity Values (SpDs) and Organ Specificity<br>
Values (SpOr) were calculated as described in tlie materials and methods section hereinabove.<br>
' No. = number, Ave. = average; Var. = variance.<br>
35<br><br>
Table 10<br><br>
DRE number '	13277	15980     "	16665<br>
Cluster reference ^	Z18392	BE522497	T04806<br>
Cluster position ^	Upstream	Downstream	Downstream<br>
DRE regulatory direction	Bidirectional	Unidirectional	Bidirectional<br>
DRE length (bp)	3297	2183	1358<br>
DRE sequence	SEOIDNO:106	SEQIDNOrlll	SE0IDNO:116<br>
Internal forward primer<br>
sequence^	SEQIDNO:107	SEQIDNO:112	SEQIDN0:117<br>
External forward primer<br>
sequence^	SEQIDNO:108	SEQIDN0:H3	SEQIDNO-.llS<br>
Internal reverse primer<br>
sequence^	SEQIDNO:109	SEQIDN0:114	SEQIDN0:119<br>
External reverse primer<br>
sequence^	SEQIDNO:110	SEQIDNO:115	SEQIDNO:120<br>
Position Effect Value ^	0.22	0.38	0.33<br>
Development Stage<br>
Specificity Value *	1.5	1.18	4.44<br>
Organ Specificity Value *	1	1.45	1.5<br>
Number of transgenic<br>
plants	5	16	5<br>
Young roots score (No.,<br>
Avfc,Var)*	5,0.6, 0.24	10,2.1,1.49	5,0,0<br>
Mature roots score (No.,<br>
Ave., Var) *	3,0,0	13,2.46,0.86	2,0,0<br>
Young above-ground<br>
Tissue (No., Ave., Var) *	5,0.4, 0.24	10,12,1.76	5,0.6, 0.34<br>
Mature above-ground<br>
tissue (No., Ave., Var) *	3,0,0	13,0.46,0.86	2,0.5, 0.25<br>
Siliques/Seed (No., Ave.,<br>
Var)*	3,0,0	4,3.75,1.69	Not available<br>
Flowers (No-, Ave., Var)'	3,0,0	13,0.92,1.76	2,0,0<br>
Description	Weak in seedlings.	Root tissue, mainly root<br>
ups; and seeds.	Above ground vegetative<br>
tissue of mature plants.<br>
' ID number of the DRE as assigned by the present inventors.<br>
^ Internal reference assigned by Gie present inventors to a cluster of Arabidopsis genes (contig)<br>
downstream or upstream of the DRE.<br>
^ Um'directional implies that only the sense strand of the DRE is capable of regulating gene expression ;<br>
Bidirectional implies that both die sense and antisense strands of the DRE are capable of regulating gene<br>
expression.<br>
* A PCR primer for isolating the DRE from Arabidopsis genomic DNA.<br>
' Position Effect Values (PoEf), Development Stage-Specificity Values (SpDs) and Organ Specificity<br>
Values (SpOr) were calculated as described in the materials and methods section hereinabove.<br>
' No. = number, Ave. = average; Var. = variance.<br>
9^<br><br>
Table 11<br><br>
DRE number '	16900	17109	17809<br>
Cluster reference ^	Z25996	Z17897	Z18103<br>
Cluster position'	Upstream	Upstream	Upstream<br>
DRE regulatory direction	Bidirectional	Bidirectional	Bidirectional<br>
DRE length (bp)	824	2927	3165<br>
DRE sequence	SE0IDNO:121	SEOIDNO:126	SE0IDNO:131<br>
Internal forward primer<br>
sequence^	SEQK)NO:122	SEQIDNO:127	SEQ ID NO: 132<br>
External forward primer<br>
sequence^	SEQIDNO:123	SEQIDNO:128	SEQ ID NO: 133<br>
Internal reverse primer<br>
sequence^	SEQ ID NO: 124	SEQ ID NO: 129	SEQ ID NO: 134<br>
External reverse primer<br>
sequence^	SEQroNO:125	SEQ ID NO: 130	SEQ ID NO: 135<br>
Position Effect Value ^	4.17	026	0.21<br>
Development Stage<br>
Speciflcity Value	0.21	0.63	0.60<br>
Organ Specificity Value *	0.22	1.85	1.38<br>
Number of transgenic<br>
plants	5	10	10<br>
Young roots score (No^<br>
Ave., Var) *	4,3.5,0.75	6,4,1.25	5, 0.8,0.56<br>
Mature roots score (No.,<br>
Ave., Var) *	5,3.2,0.56	7.3.07,2.60	7,1.5,1.5<br>
Young above-ground<br>
Tissue (No., Ave., Var)'	4,4,0	6,0.42, 0.37	5, 4, 0.8<br>
Mature above-ground<br>
tissue (No., Ave Var) *	5, 3.8,0.56	7,1.05,1.07	7, 3.07, 1.03<br>
Siliques/Seed (No., Ave.,<br>
Var)*	3,4.67, 0.22	3.0,0	3,0,0<br>
Flowers (No., Ave., Var)'	5,4,4	7,1.07,2.89	7,2.71,2.99<br>
Description	Constitutive pattern.<br>
Strong in meristematic tissue<br>
and seeds.	Strong in root, flower and<br>
meristematic tissue.	Strong in leaf tissue of<br>
seedlings.<br>
Variable in mature plants.<br>
' ID number of the DRE as assigned by the present inventors.<br>
^ Internal reference assigned by the present inventors to a cluster of Arabidopsis genes (contig)<br>
downstream or upstream of the DRE.<br>
' Unidirectional implies that only the sense strand of the DRE is capable of regulating gene expression;<br>
Bidirectional implies that both the sense and antisense strands of the DRE are capable of regulating gene<br>
expression.<br>
*	A PCR primer for isolating the DRE from Arabidopsis genomic DNA.<br>
*	Position Effect Values (PoEf), Development Stage-Specificity Values (SpDs) and Organ Specificity<br>
Values (SpOr) were calculated as described in the materials and methods section hereinabove.<br>
' No. = number, Ave. = average; Var. = variance.<br>
3^<br><br>
Table 12<br><br>
DRE number '	19672	19678	19827<br>
Cluster reference ^	Z25683	BE523552	Z17577<br>
Cluster position ^	Upstream	Upstream	Upstream<br>
DRE refiulatory direction	Unidirectional	Bidirectional	Bidirectional<br>
DRE length (bp)	1155	2877	578<br>
DRE sequence	SE0roNO:136	SE0IDN0:141	SE0roNO:146<br>
Interna! forward primer<br>
sequence^	SEQIDNO:137	SEQIDNO:142	SEQIDNO:147<br>
External for&gt;vard primer<br>
sequence^	SEQIDNO:138	SEQIDNO:143	SEQIDNO:148<br>
Internal reverse primer<br>
sequence^	SEQIDNO:139	SEQIDNO:144	SEQIDNO:149<br>
External reverse primer<br>
sequence^	SEQ1DNO:140	SEQIDNO:145	SEQIDNO:150<br>
Position Effect Value*	0.03	5.55	0.37<br>
Development Stage<br>
Specificity Value *	9.78	1.5	0.60<br>
Organ Speciflcity Value *	3.99	2	0.64<br>
Number of transgenic<br>
plants	17	5	12<br>
Young roots score (No.,<br>
Ave., Var) *	15.4.33,0.76	5,0,0	10,0, 0<br>
Mature roots score (No.,<br>
Ave., Var) *	17,4,1.76	4,0,0	5,0.1,0.04<br>
Young above-ground<br>
Tissue (No., Ave., Var)'	15,4.53,0.38	5,0,0	10, 2.9,2.29<br>
Mature above-ground<br>
tissue (No., Ave., Var)'	17,3.12,2.22	4,0.25,0.18	5,0.8,1.36<br>
Siliques/Seed (No., Ave.,<br>
Var)Â«	3, 4.33,0.22	3,0,0	3,0,0<br>
Flowers (No_ Ave., Var)'	17,4.06,2.17	4,0,0	5,0.8,1.36<br>
Description	Strong, constitutive.<br>
Lower expression in mature<br>
leaftissue.	Very weak.<br>
High position efTecL	Above ground tissue.<br>
Weak.<br>
ID number of the DRE as assigned by the present inventors.<br>
â€¢	Internal reference assigned by the present inventors to a cluster oT Arabidopsis genes (contig)<br>
downstream or upstream of the DRE.<br>
^ Unidirectional implies that only the sense strand of the DRE is capable of regulating gene expression ;<br>
Bidirectional implies that both the sense and antisense strands of the DRE are capable of regulating gene<br>
expression.<br>
*	A PCR primer for isolating the DRE from Arabidopsis genomic DNA.<br>
^ Position Effect Values (PoEf), Development Stage-Specificity Values (SpDs) and Organ Specificity<br>
Values (SpOr) were calculated as described in the materials and methods section hereinabove.<br>
' No. = number, Ave. = average; Var. = variance.<br>
SB-<br><br>
Table 13<br><br>
1<br>
^ Internal reference assigned by the present inventors to a cluster of Arabidopsis genes (contig)<br>
downstream or upstream of the DRE.<br>
^ Unidirectional implies that only the sense strand of the DRE is capable of regulating gene expression;<br>
Bidirectional inqilies that both tiie sense and antisense strands of the DRE are capable of regulating gene<br>
expression.<br>
* A PCR primer for isolating the DRE from Arabidopsis genomic DNA.<br>
^ Position Effect Values (PoEf), Development Stage-Specificity Values (SpDs) and Organ Specificity<br>
Values (SpOr) were calculated as described in the materials and methods section hereinabove.<br>
' No. = number; Ave. = average; Var. = variance.<br><br>
DRE number *	20607	22397	22604<br>
Cluster reference ^	AI998130	ATHD12A	ATHFEDAA<br>
Cluster position ^	Upstream	Upstream	Downstream<br>
DRE regulatory direction	Bidirectional	Unidirectional	Bidirectional<br>
DRE length (bp)	2819	1313	2080<br>
DRE sequence	SE0IDN0:151	SEQ ID NO: 156	SE0IDNO:161<br>
Internal forward primer<br>
sequence^	SEQ ID NO: 152	SEQ ID NO: 157	SEQ ID NO: 162<br>
External for&gt;vard primer<br>
sequence^	SEQIDNO:153	SEQ ID NO: 158	SEQ ID NO: 163<br>
Internal reverse primer<br>
sequence^	SEQ ID NO: 154	SEQ ID NO: 159	SEQ ID NO: 164<br>
External reverse primer<br>
sequence^	SEQ ID NO: 155	SEQ ID NO: 160	SEQ ID NO: 165<br>
Position Effect Value *	0.25	0.38	9.72<br>
Development Stage<br>
Specificity Value	2.50	0.89	0.71<br>
Organ Specificity Value ^	0.916	133	1.10<br>
Number of transgenic<br>
plants	5	12	17<br>
Young roots score (No^<br>
Ave^ Var) *	5,0,0	12,1.13,2.09	15,0,0<br>
Mature roots score (No^<br>
Ave^ Var) *	3,0,0	12,1.67,4.22	13,0,0<br>
Young above-ground<br>
Tissue (No^ Ave^ Var)'	5,2.2,2.16	12,3.33,0.89	15,0.2,0.16<br>
Mature above-ground<br>
tissue (No^ Ave., Var) '	3,2,2	12,2.63,2.69	13,0,0<br>
Siliques/Seed (No., Ave.,<br>
Var)*	Not available	3,4,0.67	4,0.75,1.69<br>
Flowers (No., Ave&gt;, Var) '	3,1.2	12,1.21,3.06	13,0,0<br>
Description	Above ground tissue.	Above ground tissue and<br>
seed.	Above ground tissue and<br>
seed.<br>
High position effect<br>
Very weak.<br>
D number of the DRE as ass	igned by the present invc	ntors.	<br>
31<br><br>
Table 14<br><br>
DRE number '	24136	24291	24728<br>
Cluster reference ^	Z34788	Z17960	AV530349<br>
Cluster position ^	Downstream	Upstream	Upstream<br>
DRE regulatory direction	Unidirectional	Bidirectional	Unidirectional<br>
DRE length (bp)	174	2096	1617<br>
DRE sequence	SEOIDNO:166	SE0IDNO:171	SEOIDNO:176<br>
Internal forward primer<br>
sequence*	SEQIDNO:167	SEQ ID NO: 172	SEQ ID NO: 177<br>
External fonvard primer<br>
sequence*	SEQ ID NO: none	SEQ ID NO: 173	SEQ ID NO: 178<br>
Internal reverse primer<br>
sequence*	SEQIDNO:169	SEQ ID NO: 174	SEQ ID NO: 179<br>
External reverse primer<br>
sequence*	SEQ ID NO: none	SEQ ID NO: 175	SEQ ID NO: 180<br>
Position EiTect Value ^	0.17	0.56	5.71<br>
Development Stage<br>
Specificity Value	1.5	7.76	0.17<br>
Organ Speciflcity Value *	2	6.93	1.75<br>
Number of transgenic<br>
plants	5	12	9<br>
Young roots score (No.,<br>
Ave^Var)*	1,0,0	9,1.56,3.14	8,0,0<br>
Mature roots score (No.,<br>
Ave., Var) *	2,0,0	8,237,1.48	8,0.5,1.75<br>
Young above-ground<br>
Tissue (No., Ave., Var)'	1,0,0	9,2.33,3.11	8,2.63, 0.48<br>
Mature above-ground<br>
tissue (No., Ave., Var)'	2, 0.25, 0.06	8,1.62,2.33	8,2.5,1.5<br>
Siliques/Seed (No., Ave.,<br>
Var)*	3,0,0	4,2,1.5	Not available<br>
Flowers (No., Ave., Var) *	2,0,0	8,1.37, 1.98	8,3.38, 2.73<br>
Description	Above ground tissue.<br>
Very weak.	Constitutive.	Strong in flower tissue.<br>
Low expression in veins.<br>
' ID number of the DRE as assigned by the present inventors.<br>
^ Internal reference assigned by tlie present inventors to a cluster of Arabidopsis genes (contig)<br>
downstream or upstream of the DRE.<br>
' Unidirectional implies that only the sense strand of the DRE is enable of regulating gene expression ;<br>
Bidirectional implies that both the sense and antisense strands of the DRE are capable of regulating gene<br>
expression.<br>
*	A PCR primer for isolating the DRE from Arabidopsis genomic DNA.<br>
*	Position Effect Values (PoEf), Development Stage-Specificity Values (SpDs) and Organ Specificity<br>
Values (SpOr) were calculated as described in the materials and methods section hereinabove.<br>
' No. = number, Ave. = average; Var. = variance.<br>
/(C<br><br>
Table 15<br><br>
DRE number '	24811	4209	5095<br>
Cluster reference '	H36200	H36237	Z29720<br>
Cluster position'	Upstream	Upstream	Upstream<br>
DRE regulatory direction	Bidirectional	Bidirectional	Bidirectional<br>
DRE length (bp)	428	1022	1056<br>
DRE sequence	SE0IDN0:181	SBQIDNO:186	SEQIDN0:191<br>
Internal forward primer<br>
sequence^	SEQ1DN0:182	SEQIDNO:187	SEQIDNO:192<br>
External forward primer<br>
sequence^	SEQ ID NO: none	SEQ ID NO: 188	SEQ ID NO: 193<br>
Internal reverse primer<br>
sequence^	SEQIDNO:184	SEQIDNO:189	SEQ ID NO: 194<br>
External reverse primer<br>
sequence^	SEQ ID NO: none	SEQ ID NO: 190	SEQ ID NO: 195<br>
Position Effect Value *	0.53	0.60	0.33<br>
Development Stage<br>
Specificity Value	8.11	028	0.61<br>
Organ Specificity Value *	0.23	0.49	1.35<br>
Number of transgenic<br>
plants	4	5	3<br>
Young roots score (No^<br>
Ave., Var) *	4,0.75,1.69	5, 0.4, 0.34	3, 033,0.22<br>
Mature roots score (No^<br>
Ave^ Var) *	3,1,2	4,1.2,2.16	â– ^ 1 1<br>
Young above-ground<br>
Tissue (No., Ave., Var)'	4,1,3	5,2.4,1.84	3,1.33,1.56<br>
Mature above-ground<br>
tissue (No., Ave., Var) *	3,1.33,3.56	5,2.2.8	2,2,0<br>
Siliques/Seed (No., Ave^<br>
Var)*	Not available	5,0.4, 024	2,0.0<br>
Flowers (No., Ave., Var) *	3 "^ 4	5,1.6,3.44	2,0,0<br>
Description	Constitutive, weak.	Leaf-stalk and stem tissue.	Vegetative tissue, weak.<br>
' ID number of the DRE as assigned by the present inventors.<br>
^ Internal reference assigned by the present inventors to a cluster of Arabidopsis genes (contig)<br>
downstream or upstream of die DRE.<br>
^Unidirectional implies that only die sense strand of the DRE is capable of regulating gene e^qiression;<br>
Bidirectional implies that both the sense and antisense strands of the DRE are enable of regulating gene<br>
expression.<br>
*	A PCR primer for isolating the DRE from Arabidopsis genomic DNA.<br>
*	Position Effect Values (PoEf), Development Stage-Specificity Values (SpDs) and Oi^an Specificity<br>
Values (SpOr) were calculated as described in the materials and methods section hereinabove.<br>
' No. = number; Ave. = average; Var. = variance.<br>
i|l<br><br>
Table 16<br><br>
DRE number '	17109	20607	24811<br>
Cluster reference ^	R29912	R90407	T22055<br>
Cluster position ^	Downstream	Downstream	Downstream<br>
DRE regulatory direction	Bidirectional	Bidirectional	Bidirectional<br>
DRE length (bp)	2027	2834	428<br>
DRE sequence	SE0IDNO:196	SEOIDNO:201	SEO ID NO:202<br>
Internal forward primer<br>
sequence^	SEQIDNO:197	SEQ ID NO: 168	SEQ ID NO:48<br>
External forward primer<br>
sequence'	SEQrDNO:198	SEQ ID NO: 170	SEQ ID NO: none<br>
Internal reverse primer<br>
sequence'	SEQIDNO:199	SEQ ID NO: 183	SEQIDNO:50<br>
External reverse primer<br>
sequence'	SEQ ID NO:200	SEQ ID NO: 185	SEQ ID NO:none<br>
Position Effect Value ^	0.46	0.26	0.24<br>
Development Stage<br>
Specificity Value	0	0.60	0.61<br>
Organ Specificity Value *	0.49	1.16	0.51<br>
Number of transgenic<br>
plants	5	5	5<br>
Young roots score (No^<br>
Ave., Var) *	5, 0.6, 0.64	5.0,0	4,0.75,1.69<br>
Mature roots score (No.,<br>
Ave., Var)'	Not available	5,0,0	5,0.6, 1.44<br>
Young above-ground<br>
Tissue (No., Ave., Var) *	5,2,2	5,2.2,2.16	4,1,3<br>
Mature above-ground<br>
tissue (No., Ave^ Var)'	Not available	5,1.8,2.16	5, 0.8,2.56<br>
Siliques/Seed (No., Ave.,<br>
Var)'	Not available	4,0,0	3,0,0<br>
Flowers (No., Ave., Var) *	Not available	5,1.2,1.36	5, 0.8, 2.56<br>
Description	Above ground tissue, weak	Above ground tissue,<br>
mainly in leaves.	Constitutive, weak<br>
ID number of the DRE as assigned by the present inventors.<br>
^ Internal reference assigned by the present inventors to a cluster of Arabidopsis genes (contig)<br>
downstream or upstream of the DRE.<br>
^ Unidirectional implies that only the sense strand of tlie DRE is capable of regulating gene egression ;<br>
Bidirectional implies that both the sense and antisense strands of the DRE are capable of regulating gene<br>
expression.<br>
' A PCR primer for isolating tiie DRE from Arabidopsis genomic DNA.<br>
' Position Effect Values (PoEf), Development Stage-Specificity Values (SpDs) and Organ Specificity<br>
Values (SpOr) were calculated as described in the materials and methods section hereinabove.<br>
' No. = number, Ave. = average; Var. = variance.<br>
L|2-<br><br>
TabUn<br><br>
DRE number '	16665		<br>
Cluster reference ^	Z26101		<br>
Cluster position'	Upstream		<br>
DRE regulatory direction	Bidirectional		<br>
DRÂ£ length (bp)	1358		<br>
DRE sequence	SBO ID NO:203		<br>
Internal forward printer<br>
sequence'	SEQ ID NO:204		<br>
External forward primer<br>
sequence'	SEQ ID NO:206		<br>
Internal reverse primer<br>
sequence'	SEQ ID NO:206		<br>
External reverse primer<br>
sequence'	SEQIDNO:207		<br>
Position Effect Value ^	0.51		<br>
Development Stage<br>
Specificity Value	8.82		<br>
Organ Specificity Value *	0.403		<br>
Number of transgenic<br>
plants	12		<br>
Young roots score (No^<br>
Ave^ Var) *	10,0, 0		<br>
Mature roots score (No.,<br>
Ave., Var)*	5,0.6,0.64		<br>
Young above-ground<br>
Tissue (No., Ave., Var)'	10,1.5,3.05		<br>
Mature above-ground<br>
tissue (No., Ave., Var) *	5,2,2.08		<br>
Siliques/Seed (No., Ave.,<br>
Var)*	3,1,0.66		<br>
Flowers (No., Ave., Var) *	5,1.8,3.76		<br>
Description	Above ground tissue	1<br>
' ID number of the DRE as assigned by the present inventors.<br>
^ Internal reference assigned by the present inventors to a cluster of Arabidopsis genes (contig)<br>
downstream or upstream of the DRE.<br>
^ Unidirectional implies that only the sense strand of the DRE is capable of regulating gene expression ;<br>
Bidirectional implies that both die sense and antisense strands of the DRE are enable of regulating gene<br>
expression.<br>
'A PCR primer for isolating the DRE from Arabidopsis genomic DNA.<br>
^ Position Effect Values (PoEf), Development Stage-Specificity Values (SpDs) and Organ Specificity<br>
Values (SpOr) were calculated as described in the materials and methods section hereinabove.<br>
' No. = number, Ave. = averse; Var. = variance.<br>
U3<br><br>
Deletion analysis of DREs 4209 and 6669:<br>
The ability of partial DRE sequences to modify in vivo gene expression<br>
pattern, was tested by comparing reporter gene expression driven by unmodified<br>
DREs (SEQ ED NO:36 and 61) with that of deletion mutants thereof (SEQ ID NO:210<br>
and 213, respectively).<br>
GUS expression pattern in p4209short-GUS (including the DRE 4209 partial<br>
sequence set forth in SEQ ID NO:210) transformed plants was similar to that driven by<br>
the full length promoter sequence, DRE 4209 (SEQ ID NO:36). As is shown in Figure<br>
26a, expression was strong in roots, mainly root tips, as well as in flower buds.<br>
Insterstingly, p4209short-GUS transformed plants exhibited lower reporter gene<br>
expression in veins, while leaves exhibited higher expression. Note, expression in seeds<br>
was not examined.<br>
GUS expression pattern in the p6669short-GUS (comprising the DRE 6669<br>
partial sequence set forth in SEQ ID NO:213) transformed plants was restricted to the<br>
root tips (Figure 26b) while expression in other young or meristematic tissues, as was<br>
obtained by the full length DRE 6669 promoter (SEQ ID N0:61), was lost.<br>
These results demonstrate that the 5' nucleic acid sequence of SEQ ID NO: 61<br>
(e.g., nucleotide coordinates 1-747), is important for constitutive gene expression.<br>
Indeed, a DNA sequence (SEQ ID NO: 214, see Figure 27 WO 02/16655) which does<br>
not include the 5' first 400 nucleotides of SEQ ID NO: 61 has been implicated in<br>
stress regulated gene expression.<br>
These results indicate that the promoters of the present invention may be<br>
modified by partial deletions, to generate inductive or tissue specific expression<br>
pattern as demonstrated for DRE 6669 (SEQ ID N0:61).<br>
As is clearly illustrated by Tables 3-17 and Figures 1-26, the DREs isolated<br>
according to the teachings of the present invention exhibit a wide range of activities as<br>
well as a wide range of tissue and developmental stage specificities. The DREs of the<br>
present invention were classified according to function as determined using the<br>
.^rflWc/opsiy assay described hereinabove.<br>
The luciferase gene was expressed in a constitutive maimer in Arabidopsis when<br>
functionally linked to SEQ ID NOS: 1, 6, 41, 46, 51, 86, 121, 136, 171, 181 and 202<br>
(illustrated in Figure 18), thus the promoters of these DREs are putatively classified<br>
herein as constitutive promoters.<br><br>
The luciferase gene was expressed in an inductive manner in Arabidopsis<br>
when functionally linked to SEQ ID NOS: 1, 11, 16, 21, 26, 31, 36, 56, 61, 66, 71,<br>
76, 81, 91, 96, 101, 116, 126, 141, 146, 151, 156, 161, 166, 176 and 203, thus the<br>
promoters of these DREs are putatively classified herein as inductive promoters.<br>
The luciferase gene was expressed in a yoxmg or meristematic, tissue-specific<br>
manner in Arabidopsis when functionally linked to SEQ ID NOS: 61, 121, 126, 213<br>
(illustrated in Figures 4, 14, 15, 16 and 26b), thus the promoters of these DREs are<br>
putatively classified herein as young or meristematic, tissue-specific promoters.<br>
The luciferase gene was expressed in root tissue specific manner in<br>
Arabidopsis when functionally linked to SEQ ID NOSL 21, 36, 91, 111, and 126<br>
(illustrated in Figures 2,3,9, and 13), thus the promoters of these DREs are putatively<br>
classified herein as root tissue-specific promoters<br>
The luciferase gene was expressed in an above ground tissue-specific manner<br>
in Arabidopsis when fimctionally linked to SEQ ID NOS: 16, 26, 31, 66, 71, 76, 81,<br>
96, 106, 101, 116, 131, 146, 151, 156, 161, 166, 196, 201 and 203 (illustrate in<br>
Figures 10, 11, 17, 19 and 20), thus the promoters of these DREs are putatively<br>
classified herein as above ground tissue-specific promoters.<br>
The luciferase gene was expressed in a stem tissue specific manner in<br>
Arabidopsis when functionally linked to SEQ ID NO: 186 (illustrated in Figure 22),<br>
thus the promoter(s) of this DRE are putatively classified herein as stem tissue<br>
specific promoter(s).<br>
The luciferase gene was expressed in a flower tissue specific manner in<br>
Arabidopsis when functionally linked to SEQ ID NOS: 11, 36, 81, 91, 126, 176 and<br>
210 (illustrated in Figures 1, 3, 9 and 26a), thus the promoters of these DREs are<br>
putatively classified herein as flower tissue-specific promoters.<br>
The luciferase gene was expressed in a finit (silique) tissue specific manner in<br>
Arabidopsis when functionally linked to SEQ ID NO: 56, thus the promoter(s) of this<br>
DRE are putatively classified herein as fiiiit (silique) tissue specific promoter(s).<br>
The luciferase gene was expressed in a seed tissue specific manner in<br>
Arabidopsis when functionally linked to SEQ ID NOS: 1, 156, and 161 (illustrated in<br>
figures 12 and 21), thus the promoters of these DREs are putatively classified herein<br>
as seed tissue specific promoters.<br><br>
The luciferase gene was expressed in a developmental stage specific manner in<br>
Arabidopsis when functionally linked to SEQ ID NOS: 81, 96, 101, 106, and 131<br>
(illustrated comparatively in Figures 5-6,7-8, 10-11 and 15-16), thus the promoters of<br>
these DREs  are putatively classified herein  as. developmental  stage  specific<br>
promoters.<br>
The GUS gene was expressed in an inductive manner in Arabidopsis when<br>
fimctionally linked to SEQ ID NOS: 210 and 213 (illustrated in Figure 26b), thus the<br>
promoters of these partial DREs sequences are putatively classified herein as<br>
inductive promoters.<br>
The GUS gene was expressed in a root, as well as in a flower bud tissue<br>
specific manner in Arabidopsis when fimctionally linked to SEQ ID NO: 210<br>
(illustrated in Figure 26a), thus the promoter of this partial DRE sequence is<br>
putatively classified herein as a root as well as a flower tissue-specific promoter.<br>
The GUS gene was expressed in a root-tip tissue specific maimer in<br>
Arabidopsis when fimctionally linked to SEQ ID NO: 213 (illustrated in Figure 26b),<br>
thus this promoter is putatively classified herein as a root tissue-specific promoter.<br>
Although the invention has been described in conjunction with specific<br>
embodiments thereof, it is evident that many alternatives, modifications and variations<br>
will be apparent to those skilled in the art Accordingly, it is intended to embrace all<br>
such alternatives, modifications and variations that fall within the spirit and broad<br>
scope of the appended claims. All publications, patents, patent applications and<br>
sequences identified by their accession numbers mentioned in this specification are<br>
herein incorporated in their entirety by reference into the specification, to the same<br>
extent as if each individual publication, patent, patent application or sequence<br>
identified by their accession number was specifically and individually indicated to be<br>
incorporated herein by reference. In addition, citation or identification of any<br>
reference in this application shall not be construed as an admission that such reference<br>
is available as prior art to the present invention.<br>
^^<br><br><br>
WE CLAIM :<br>
1.	A polynucleotide comprising a nucleic acid sequence selected from the group<br>
consisting of SEQ IDNOS: 1, 6, 11, 16, 21, 26, 31, 36, 41, 46, 56, 61, 66, 71, 76,<br>
81, 86, 91, 96, 101, 106, 111, 116, 121, 126, 131, 136, 141, 146, 151, 156, 161,<br>
166, 171, 176, 181, 186, 191, 196, 201, 202, 203, 210 and 213 wherein the<br>
polynucleotide is capable of regulating expression of at least one polynucleotide<br>
sequence operably linked thereto and whereas said nucleic acid sequence is no<br>
more than 3,348 nucleic acids in length.<br>
2.	A nucleic acid construct, comprising the polynucleotide as claimed in claim 1.<br>
3.	The polynucleotide as claimed in claim 1 or the nucleic acid construct as claimed<br>
in claim 2, wherein said nucleic acid sequence comprises a promoter activity.<br>
4.	The nucleic acid construct as claimed in claim 2, optionally comprising at least<br>
one heterologous polynucleotide operably linked to the polynucleotide.<br>
5.	The nucleic acid construct as claimed in claim 4, wherein said at least one<br>
heterologous polynucleotide is a reporter gene.<br>
6.	The nucleic acid construct as claimed in claim 4,	optionally comprising two<br>
heterologous polynucleotides each being operably	linked to an end of the<br>
polynucleotide such that said two heterologous	polynucleotides flank the<br>
polynucleotide.<br>
7.	A method of producing a transgenic plant, comprising transforming a plant with<br>
the polynucleotide as claimed in claim 1 or 3.<br>
-47-<br><br>
8.	A method of producing a transgenic plant, comprising transforming a plant with<br>
the nucleic acid construct as claimed in claim 2, 3, 4, 5 or 6.<br>
9.	A method of expressing a polypeptide of interest in a cell comprising transforming<br>
the cell with a nucleic acid construct including a polynucleotide sequence<br>
encoding the polypeptide of interest operably linked to a regulatory nucleic acid<br>
sequence selected from the group consisting of SEQ ID NOS: 1,6, 11, 16, 21, 26,<br>
31, 36, 41, 46, 56, 61, 66, 71, 76, 81, 86, 91, 96, 101, 106, 111, 116, 121, 126,<br>
131, 136, 141, 146, 151, 156, 161, 166, 171, 176, 181, 186, 191, 196, 201, 202,<br>
203, 210 and 213 thereby expressing the polypeptide of interest in the cell.<br>
10.	A method of co-expressing two polypeptides of interest in a cell comprising<br>
transforming the cell with a nucleic acid construct including two polynucleotide<br>
sequences encoding the two polypeptides of interest operably linked to a<br>
regulatory nucleic acid sequence selected from the group consisting of SEQ ID<br>
NOS: 1, 6, 11, 16, 21, 26, 31, 36, 41, 46, 56, 61, 66, 71, 76, 81, 86, 91, 96, 101,<br>
106, 111, 116, 121, 126, 131, 136, 141, 146, 151, 156, 161, 166, 171, 176, 181,<br>
186, 191, 196, 201, 202, 203, 210 and 213 such that said two polynucleotide<br>
sequences flank said regulatory nucleic acid sequence, thereby expressing the two<br>
polypeptides of interest in the cell.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNC1jaGVucC0yMDA1IGFic3RyYWN0IGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">2604-chenp-2005 abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNC1jaGVucC0yMDA1IGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2604-chenp-2005 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNC1jaGVucC0yMDA1IGNsYWltcyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">2604-chenp-2005 claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNC1jaGVucC0yMDA1IGNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2604-chenp-2005 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNC1jaGVucC0yMDA1IGNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2604-chenp-2005 correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNC1jaGVucC0yMDA1IGNvcnJlc3BvbmRlbmNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">2604-chenp-2005 correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNC1jaGVucC0yMDA1IGRlc2NyaXB0aW9uIChjb21wZWxldCkgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">2604-chenp-2005 description (compelet) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNC1jaGVucC0yMDA1IGRlc2NyaXB0aW9uIChjb21wZWxldCkucGRm" target="_blank" style="word-wrap:break-word;">2604-chenp-2005 description (compelet).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNC1jaGVucC0yMDA1IGRyYXdpbmdzIGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">2604-chenp-2005 drawings duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNC1jaGVucC0yMDA1IGRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">2604-chenp-2005 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNC1jaGVucC0yMDA1IGZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">2604-chenp-2005 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNC1jaGVucC0yMDA1IGZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">2604-chenp-2005 form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNC1jaGVucC0yMDA1IGZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">2604-chenp-2005 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNC1jaGVucC0yMDA1IGZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">2604-chenp-2005 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNC1jaGVucC0yMDA1IGZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">2604-chenp-2005 form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNC1jaGVucC0yMDA1IHBjdCBzZWFyY2ggcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">2604-chenp-2005 pct search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNC1jaGVucC0yMDA1IHBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">2604-chenp-2005 pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNC1jaGVucC0yMDA1IHBldGl0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">2604-chenp-2005 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="230563-an-apparatus-for-the-manufacture-of-compost-from-biodegradable-wastes-and-method-therefor.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="230565-an-improved-process-for-the-preparation-of-ceftriaxone.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>230564</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2604/CHENP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>27-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>12-Oct-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>EVOGENE LTD</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>13 Gad Finstein Street, 76121 Rehovot,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>KARCHI, Hagai</td>
											<td>249 SAPIR STREET, MOSHAV SITRIYA, 76834 DOAR-NA EMEK SOREQ,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>MEISSNER, Rafael</td>
											<td>11 Halevona Street, Rehovot 76 350,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>RONEN, Gil</td>
											<td>20 MOSHAV OMETZ, 33870 EMEQ-HEFER,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>GOLAN, Ezekiel</td>
											<td>5 GIMMEL KEHILAT LEVOV STREET, 69703 TEL AVIV,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>RABINOVICH, Larisa</td>
											<td>13/13 Hanagid Street, Rishon Lezion 75 482,</td>
										</tr>
										<tr>
											<td>6</td>
											<td>ZELIGER, Naama</td>
											<td>Moshav Mechora 90 698,</td>
										</tr>
										<tr>
											<td>7</td>
											<td>SAVIR, Noa</td>
											<td>Kibbutz Givat-brener, Kibbutz Givat-brener 60948,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IL 04/00235</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-03-11</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/453,843</td>
									<td>2003-03-12</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/230564-nucleotide-sequences-regulating-gene-expression-and-constructs-and-methods-utilizing-same by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:21:13 GMT -->
</html>
