<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/230531-proteins-with-repetitive-bacterial-ig-like-big-domains-present-in-leptospira-species by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:19:09 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 230531:PROTEINS WITH REPETITIVE BACTERIAL-IG-LIKE (BIG) DOMAINS PRESENT IN LEPTOSPIRA SPECIES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">PROTEINS WITH REPETITIVE BACTERIAL-IG-LIKE (BIG) DOMAINS PRESENT IN LEPTOSPIRA SPECIES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>ABSTRACT 2867/CHENP/2004 &quot;PROTEINS WITH REPETITIVE BACTERIAL-IG-LIKE (BIG) DOMAINS PRESENT IN LEPTOSPIRA SPECIES. The invention relates to three isolated DNA molecules that encode for proteins BigLl, BigL2 and Big L3, in the Leptospira sp bacterium which have repetitive Bacterial-Ig-like (Big) domains and their use in diagnostic, therapeutic and vaccine application. According to the present invention, the isolated molecules encoding for BigLl, BigL2 and BigL3 proteins are used for the diagnosis and prevention of infection with Leptospira species that are capable of producing disease in humans and other mammals, including those of veterinary importance.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Title:  Proteins  with  repetitive  Bacterial-ig-like  (Big) domains present in Leptospira  species<br>
FIELD OF THE INVENTION<br>
The invention relates to three isolated DNA molecules that encode for proteins, BigLl, BigL2 and BigL3, in the Leptospira sp bacterium which have repetitive Bacterial-Ig-like (Big) domains and their use in diagnostic, therapeutic and vaccine applications. According to the present invention, the isolated molecules encoding for BigLl, BigL2 and BigL3 proteins are used for the diagnosis and prevention of infection with Leptospira species that are capable of producing disease in humans and other mammals, including those of veterinary importance.<br>
BACKGROUND OF THE INVENTION<br>
Spirochetes are motile, helically shaped bacteria and include three genuses, Leptospira, Borrelia and Treponema, which are pathogens of humans and other animals. Borrelia and Treponema are the causative agents of diseases that include Lyme disease, relapsing fever, syphilis and yaws. Leptospira consists of a genetically diverse group of eight pathogenic and four non-pathogenic, saprophytic species (1, 2). Leptospires are also classified according to serovar status - more than 200 pathogenic serovars have been identified. Structural heterogeneity in lipopolysaccharide moieties appears to be the basis for the large degree of antigenic variation observed among serovars (1, 2).<br>
Leptospirosis is a. zoonotic disease: transmission to humans occurs through contact with domestic or wild animal reservoirs or an environment contaminated by their urine. Infection produces a wide spectrum of clinical manifestations. The early-phase of illness is characterized by fever, chills, headache and severe myalgias. Disease progresses in 5 to 15% of the clinical infections to produce severe multisystem complications such as jaundice, renal insufficiency and hemorrhagic manifestations (1-4). Severe leptospirosis is associated with mortality rates of 5-40%.<br>
Leptospirosis has a world-wide distribution. Because of the large spectrum of animal species that serve as reservoirs, it is considered to be the most widespread zoonotic disease (1). Leptospirosis is traditionally an important occupational disease among risk groups such as military personnel, farmers, miners, sewage and refuse removal workers, veterinarians and abattoir workers (1-3) . However, new patterns of disease transmission have emerged recently  that  emphasize  the  growing  importance  of<br><br><br>
leptospirosis as a public health problem. In developed countries, leptospirosis has become the cause of outbreaks associated with recreational activities (1) and sporting events (1, 4, 5). In Brazil and other developing countries, underlying conditions of poverty have produced large urban epidemics of leptospirosis associated with high mortality (4, 5) .<br>
In addition to its public health impact, leptospirosis is a major economic burden as the cause of disease in livestock and domestic animals (2). Leptospirosis produces abortions, stillbirths, . infertility, failure to thrive, reduced milk production and death in animals such as cows, pigs, sheep, goats, horses and dogs and induces chronic infection and shedding of pathogenic leptospires in livestock (2) and therefore represents an additional source of economic loss for the animal husbandry industry because of current international and national quarantine regulations.<br>
The control of human and animal leptospirosis is hindered by the current lack of adequate diagnostic tools. The standard serologic test, the microscopic agglutination test (MAT), is inadequate for rapid case identification since it can only be performed in few reference laboratories and requires analyses of paired sera to achieve sufficient sensitivity (1, 2). Dependence upon the MAT results in delays in establishing the cause of outbreaks as seen in several investigations (1, 2). Enzyme-linked immunosorbent assays (ELISA), and other rapid serologic tests based on whole-cell leptospiral antigen preparations have been developed for use as an alternative method to screen for leptospiral infection, although the MAT is still required for case confirmation (1, 2). Recombinant antigen-based serologic tests are widely used in screening for spirochetal infections such as Lyme disease and syphilis, but the use of recombinant proteins for serodiagnosis of leptospirosis has not been widely investigated. Recently, a recombinant flagellar-antigen immuno-capture assay was described for serodiagnosis o'f bovine leptospirosis (6). A recombinant heat shock protein, Hsp58, showed a high degree of ELISA reactivity with serum samples from a small number of human cases (7). However, the utility of recombinant antigens for the serodiagnosis of leptospirosis has not been investigated in large validation studies.<br>
Furthermore, there are no effective interventions presently available, which control or prevent leptospirosis. Environmental control measures are difficult to implement because of the long-term survival of pathogenic leptospires in soil and water and the abundance of wild and domestic<br><br><br>
animal reservoirs (1, 3). Efforts have focused on developing protective immunization as an intervention against leptospirosis. Currently-available vaccines are based on inactivated whole cell or membrane preparations of pathogenic leptospires and appear to induce protective responses through induction of antibodies against leptospiral lipopolysaccharide (1, 3). However, these vaccines do not induce long-term protection against infection. Furthermore, they do not provide cross-protective immunity against leptospiral serovars that are not included in the vaccine preparation. The large number of pathogenic serovars (&gt;'200) and the cost of producing a multi-serovar vaccine have been major limitations in developing efficacious vaccines through strategies based on whole cell or membrane preparations.<br>
The mechanism of pathogenesis in leptospirosis, as in spirochetal disease such as Lyme disease and syphilis, relies on the pathogen's ability to widely disseminate within the host during the early stage of infection (2) . Membrane-associated leptospiral proteins are presumed to mediate interactions that enable entry and dissemination through host tissues. Putative surface-associated virulence factors serve as candidates for vaccine strategies that induce responses to these factors which block dissemination in the host. Furthermore, membrane-associated proteins would be accessible to the immune response during host infection and therefore, constitute targets for immune protection through mechanisms such antibody-dependent phagocytosis and complement-mediated killing. Production of these antigen targets as recombinant proteins offers a cost-effective approach for protective immunization for leptospirosis as a sub-unit based vaccine. In addition, selection of surface-associated targets that are conserved among pathogenic leptospires can avoid the limitations encountered with currently available whole-cell vaccine preparations.<br>
A major limitation in the field of leptospirosis has been identifying surface-associated and host-expressed proteins with conventional biochemical and molecular methods. From the genome sequence of the spirochete, Borrelia burgdorferi, more than 100 surface associated lipoproteins were identified. Based on genome size and the biology of its lifecycle, Leptospira are expected to have a significantly greater number of surface-associated targets. At present, less than 10 surface-associated proteins have been characterized though isolation of membrane extracts, purification and characterization of proteins in these extracts and molecular cloning of these protein targets (8-14)  (12) .    Immunization  with  recombinant  proteins  for<br><br><br>
several identified targets, LipL32, OmpLl and LipL41, induce partial, but not complete, protective responses (11, 12).<br>
To develop a more comprehensive understanding of leptospiral protein expression we have used the humoral immune response during human leptospirosis as a reporter of protein antigens expressed during infection. The identification of leptospiral antigens expressed during infection has potentially important implications for the development of new serodiagnostic and immunoprotective strategies. Sera from patients with leptospirosis was used to identify clones from a genomic Leptospira DNA phage library which express immunoreactive polypeptides. A proportion of these clones were found to encode a novel family of membrane-associated Leptospira proteins. The identification of these polynucleotides and polypeptides and their application for diagnosis of leptospirosis and inducing an immune response to pathogenic spirochetes is the basis for this invention.<br>
SUMMARY OF THE INVENTION<br>
The invention relates to DNA molecules in Leptospira and the polypeptides they encode which have repetitive bacterial Ig-like domains. The invention describes the isolation of three DNA molecules, originally derived from L. kirschneri and L. interrogans, which encode proteins, herein designated "BigLl", "BigL2" and "BigL3", that have molecular masses of approximately 110, 205 and 205 kDa, respectively, based on the predicted amino acid sequence of the polypeptides. The three proteins have 12-13 tandem repeat sequences of approximately 90 amino acids. Repeats sequence from. BigLl, BigL2 and BigL3 are highly related (&gt;90% amino acid sequence identify) .to each other and belong to the family of bacteria Ig-like (Big) domains, moieties which are found in virulence factors of bacterial pathogens.<br>
The DNA molecules that encode for Leptospira proteins with Big domains, herein called "bigLl", "bigL2" and "bigL3", can be inserted as heterologous DNA into an expression vector for producing peptides and polypeptides. Recombinant polypeptides can be purified from surrogate hosts transformed with such expression vectors. BigLl, BigL2 and BigL3-derived polypeptides are serological markers for active and past infection since sera from leptospirosis patients and animals infected or immunized with pathogenic Leptospira  recognize isolated polypeptides.<br>
Furthermore, BigLl, BigL2 and BigL3 polypeptides from recombinant or native antigen preparations are immunogenic. Antibodies obtained from experimental animals immunized with<br><br><br>
purified recombinant BigLl, BigL2 and BigL3 polypeptides recognize native antigen from Leptospira, and are useful for detecting pathogenic spirochetes in samples from subjects with a suspected infection.<br>
In addition, BigLl, BigL2 and BigL3 polypeptides induce an immune response against pathogenic spirochetes. BigLl, BigL2 and BigL3-derived polypeptides; antibodies to these polypeptides; and polynucleotides that encode for BigLl, BigL2 and BigL3 may be used alone or combined with pharmaceutically acceptable carrier to treat or prevent infection with Leptospira. Since Big domains are present in proteins associated with virulence in other bacterial pathogens, these moieties may be used to treat or prevent infections unrelated to those caused by Leptospira.<br><br>
In a third embodiment, the invention provides a method for identifying a compound which binds to BigLl, BigL2, BigL3 polypeptides or polypeptides with functionally equivalent sequences that includes incubating components comprising of the compound and BigLl, BigL2 or BigL3 polypeptide or polypeptides with functionally equivalent sequences under conditions sufficient to allow the components to interact and measuring the binding of the compound to the BigLl, BigL2 or BigL3 polypeptide or polypeptides with functionally equivalent sequences. Preferably, the inventive method is a serodiagnostic method utilizing sera from a subject with a suspected active or past infection with Leptospira or other related bacterial pathogen.<br>
In a fourth embodiment, the invention provides a method for detecting pathogens in a sample which includes contacting a sample suspected of containing a pathogenic<br><br>
spirochete with a reagent that binds to the pathogen-specific cell component and detecting binding of the reagent to the component. In one aspect, the reagent that binds to the pathogen-specific cell component is an oligonucleotide for the identification of bigLl, bigL2 and bigL3 polynucleotide. In another aspect, the reagent that binds to the pathogen-specific cell component is an antibody against the BigLl, BigL2 or BigL3 polypeptide or polypeptides with functionally equivalent sequences.<br>
A fifth embodiment, the invention provides a kit useful for the detection of BigLl, BigL2, and BigL3 polypeptides or polypeptides with functionally equivalent sequences; bigLl, bigL2 and bigL3 polynucleotides; or antibodies that bind to BigLl, BigL2, BigL3, polypeptides or polypeptides with functionally equivalent sequences.<br>
Other objects, features and advantages of the present invention will become apparent from the following detailed description. It should be understood, however, that the detailed description and the specific examples, while indicating preferred embodiments of the invention, are given by way of illustration only, since various changes and modifications within the spirit and scope of the invention will become apparent to those skilled in the art from this detailed description.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
FIGS. 1A and B show a Southern blot analysis of JbigL gene sequences in Leptospira. Genomic DNA (3mcg/lane) from L. interrogans strain Fiocruz Ll-130 (lanes 1), L. kirschneri strain Rm52 (lanes 2) and L. biflexi strain Patoc I (lanes 3) digested with Nsil and subject to agarose gel electrophoresis. After transfer to nitrocellulose membranes, blots were probed with DNA fragments that encode for BigL repetitive domains (4th-6th repetitive domain of BigL3, FIG. 1A) and C-terminal regions of bigLl, bigL2 and bigL3, which are unique to each of these genes, respectively (FIG.IB).<br>
FIG. 2 shows a schematic diagram of the genomic organization of the region encoding the BigLl and BigL3 proteins in L. kirschneri. The BigLl protein would contain a signal peptide (hatched box) and thirteen 90-amino-acid bacterial immunoglobulin-like domains (solid boxes). The BigL3 protein would contain a signal peptide, twelve 90-amino-acid bacterial immunoglobulin-like domains, and a 793 amino acidcarboxyterminal (C-terminal) domain. The locations of the 2156 bp region of 100% DNA sequence identity are shown. The organization of the region depicted was conserved in L.   interrogans  and L.   kirschneri.<br><br><br>
FIG. 3 shows the polymerase chain reaction (PCR) amplification of DNA fragments from strains of five pathogenic species of Leptospira. Degenerate primers were designed based on the L. kirschneri and L. interrogans sequence encoding for the BigL3 region corresponding to positions 46-65 aa. PCR reactions were performed from purified DNA from five pathogenic (I. kirschneri, borgpetersenii, interrogans, santarosai, and noguchi) and two non-pathogenic species (I. biflexi  and wolbachii).<br>
FIG. 4 shows amplified products from RT-PCR of RNA extracts of L. kirschneri with bigLl, bigL2 and bigL3 specific primers. Reverse transcription reactions (lanes "+") were performed on RNA extracts of cultured leptospires and then subject to a polymerase chain reaction (PCR) amplification step with primers that bind to unique sequences within bigLl, bigL2 and bigL3. Amplification with primers based on sequences within lipL45 was performed as a control reaction as was PCR reactions for which samples were not subjected to the reverse transcription step.<br>
FIG. 5 shows the immunoblot reactivity of pooled sera from patients and animal reservoirs infected with pathogenic Leptospira and laboratory animals immunized with whole L. interrogans antigen preparation to recombinant BigL3 protein (rBigL3). Western blot, analysis was performed with purified rBigL3 (lmcg per lane, lanes 3). Membranes were probed with sera from patients with leptospirosis (lane A) , healthy individuals (lane B) , captured rats that are colonized with L. interrogans (lane C) , captured rats that are not colonized with L. interrogans (lane D), laboratory rats immunized with whole antigen preparations of in vitro cultured L. interrogans (lane E) and pre-immune sera from the laboratory rats collected prior to immunization (lane F) . Reactivity to whole L. interrogans antigen preparation (lanes 1) and recombinant LipL32 protein (rLipL32, lanes 2) is shown for comparison. The numbers on the left indicate the positions and relative mobilities (kDa) for molecular mass standards (Invitrogen).<br>
FIG. 6 shows an ELISA evaluation of individual patient seroreactivity to rBigL3 during the acute (lanes A) and convalescent (lanes B) phase of illness with leptospirosis. Sera from 4 leptospirosis patients (unbroken lines) and 4 health individuals (broken lines), at dilutions of 1:50, 1:100 and 1:200, were incubated with RBigL3 (25-200ng/well). Mu and gamma chain specific antibodies conjugated to horse radish peroxidase was used to determine IgM and IgG seroreactivity, respectively.   Mean absorbance values (OD<br><br><br>
450nm)  and  standard  deviations  are  represented  in  the graphs.<br>
FIG. 7 shows the rBigL3 IgM (Column A) and igG (Column B) reactivity of sera from 29 individual patients with leptospirosis during the acute (lanes 2) and convalescent (lanes 3) phase of illness and 28 health individuals (lanes 1) . Sera (1:50 dilutions) and Mu and gamma chain specific antibodies conjugated to horse radish peroxidase were used to determine reactivity. Solid bars represent mean absorbance (OD 450nm) values.<br>
FIG. 8 shows the immunoblot reactivity of individual patients with  leptospirosis  to rBigL3  during  the  acute<br>
(lanes 6-9) and convalescent (lanes 10-13) phase of illness. Western blot analysis was performed with purified rBigL3<br>
(lmcg per lane, lanes 3) . Membranes were probed with sera diluted 1:100. A gamma chain-specific antibodies conjugated to alkaline phosphatase were used to determine reactivity to the recombinant 58 kD protein of region 1 of BigL3 (2nd to 6th Big repeat domains). Reactivity to rLipL32 (1 meg per lane) was performed as a comparison. The mobility of purified rBigL32 and rLipL32 (lane 14) and molecular mass standards<br>
(lane  15)  are  shown after  staining with  Ponceau-S  and coomassie blue, respectively.<br>
FIG. 9 shows the immunoblot reactivity of rat anti-rBigL3 antisera to rBigL3 and native angigen from L. interrogans lysates. Immunoblots were prepared with purified rBigL3 (lmg/lane; lanes 3, 5, 7, 9) and whole antigen preparations (108 leptospira per lane; lanes 2, 4, 6 and 8) from cultured leptospires. Membranes were probed with pooled sera (dilutions 1:500 [lanes 4 and 5], 1:100 [lanes 6 and 7], and 1:2500 [lanes [8 and 9]] from rats immunized with rBigL3 from E. coli ewpressing a cloned DNA fragment of bigL3 from L. interrogans. Pre-immune sera was obtained prior to the first immunization and used in the immunoblot analysis as a control (lanes 2 and 3) . The mobility (kDa) of molecular mass standards are shown on the left side of the figure<br>
FIG. 10 shows the immunoblot reactivity of rabbit anti-rBigL3 antisera to native antigen from Leptospira strain lysates. Immunoblots were prepared with whole antigen preparations (108 leptospira per lane) of the following cultured strains: lane 1, L interrogans sv pomona (type kennewicki) strain RM211, low-passage; lane 2, L. interrogans sv canicola strain CDC Nic 1808, low passage; lane 3, L. interrogans sv pomona strain PO-01, high passage; lane 4, L. interrogans sv bratislava strain AS-05, high passage; lane 5, L.   kirschneri   sv grippotyphosa strain RM52,<br><br><br>
low passage; lane 6, L. kirschneri sv grippotyphosa strain P8827-2, low passage; lane 7, L. kirschneri sv grippotyphosa strain 86-89, low passage; lane 8, L. kirschneri sv grippotyphosa strain Moskva V, high passage; lane 9, L. kirschneri sv mozdok strain 5621, high passage; lane 10, L. kirschneri sv grippotyphosa strain RM52, high passage. Membranes were probed with sera from rabbits immunized with rBigL3 from E. coli expressing a cloned DNA fragment of bigL3 from L. kircshneri and, as a control measure, sera from rabbits immunized with recombinant L. kirschneri GroEL protein. The positions of native antigens corresponding to BigL3 and GroEL and the mobility (kDa) of molecular mass standards are shown on the left and right sides, respectively, of the figure.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
For convenience, the meaning of certain terms and phrases employed in the specification, examples, and appended claims are provided below. Unless defined otherwise, all technical and scientific terms used herein have the same meaning as commonly understood by one of ordinary skill in the art to which this invention belongs.<br>
BigL - are polypeptides of Leptospira sp. having tandem repeat sequences each of which are similar, according to their sequence homology, to bacterial immunoglobulin-like (Big) domains. Big domains are present in bacterial proteins, expressed in bacterial pathogens such as E. coli, Yersinia and Bordetella, which have virulence functions such as host cell adhesion.<br>
Reference sequence - is a new sequence obtained by isolation from a natural organism or through genetic engineering and presents an accurate biological function, which is characteristic of the present invention.<br>
Functionally equivalent sequences - are the sequences, related to a reference sequence, that are the result of variability, i.e. all modification, spontaneous or induced, in a sequence, being substitution and/or deletion and/or insertion of nucleotides or amino acids, and/or extension and/or shortening of the sequence in one of their ends, without resulting in modification of the characteristic function of the reference sequence. Functionally equivalent sequences emcompass fragment and analog therof. In other words, sequences functionally equivalent are sequences that are "substantially the same" or "substantially identical" to the reference sequence, such as polypeptides or nucleic acids that have at least 80% homology in relation to the sequence of amino acids or reference nucleic acids.   The<br><br><br>
homology usually is measured by a software system that performs sequence analyses (Sequence Analysis Software Package of the Genetics Computer Group, University of Wisconsin Biotechnology Center, 1710, University Avenue, Madison, Wis, 53705) .<br>
As we mentioned before, Leptospira antigens expressed during the host infection are important in the identification of targets for diagnosis tests and vaccines. The LipL32 protein is one of these targets and was identified as immunodominant antigen by the immune humoral response during the natural infection. However the sensitivity of serologic tests based upon detection of antibodies against LipL32 in patient sera during acute-phase illness with leptospirosis detection is limited (see Flannery, B: "Evaluation of recombinant Leptospira antigen-based Enzyme-linked Immunosorbent Assays for the serodiagnosis of Leptospirosis" J.Clin. Microbiology 2001;39(9): 3303-3310; W09942478).<br>
The present invention is based on the identification	of<br>
the family of proteins BigL associated with species	of<br>
spirochetal bacteria, including those belonging	to<br>
Leptospira.<br>
According to the present invention, the BigL protein family was identified as targets of the host humoral immune response, generated during infection with pathogenic Leptospira or immunization with pathogenic Leptospira or recombinant BigL polypeptides. BigL polypeptides and polynucleotides that encode these polypeptides are useful as in diagnostic tests to identify naturally occurring infection in different species including humans and animal reservoirs. The diagnostic test based on those proteins presents improved sensitivity and specificity in relation to standard diagnostic tests or those that are have been used in the published literature. the identification of leptospirosis in the initial phase. In addition BigL polypeptides can induce immune responses when used in a pharmaceutical composition for immunization.<br>
In the present invention, the three BigL polypeptides are characterized with molecular weights 128.4 kD, 201.3 kD and 200.4kD, based on the deduced amino acid sequence of the isolated polynucleotides, bigLl, bigL2 and bigL3, which encode for these polypeptides. The amino acid sequence of the BigL polypeptides has a signal sequence and a putative signal peptidase cleavage site largely conforming to the spirochetal lipobox; therefore BigL polypeptides are membrane-associated lipoproteins.  The polypeptides of 128.4<br><br><br>
kD, 201.3 kD and 200.4 kD are designated "BigLl", "BigL2" and "BigL3", respectively..<br>
Although the BigL polypeptides of the present invention have been isolated originally of Leptospira sp, they are useful not just for induction of the immune response against the pathogenic organisms Leptospira sp. , but also against other spirochetes bacteria and pathogens that have factors with Big domains. Additionally, BigL polypeptides can be used for the diagnosis of infections due to Leptospira sp., other pathogenic spirochetes and bacterial pathogens.<br>
Several processes that incorporate state-of-the-art methodologies can be used to obtain polynucleotide sequences that encode for BigL polypeptides. These processes include, but they are not limited to, the isolation of DNA using hybridization of genomic libraries with probes to detect homologous sequences of nucleotides; screening of antibodies of expression libraries to detect fragments of cloned DNA with shared structural aspects; polymerase chain reaction (PCR) in genomic DNA using initiators able to recombine sequence of DNA of interest; and computer-based searches of sequence databases for similar sequences to that of the bigL polynucleotides.<br>
In the present invention the identification of the^<br>
antigens was based on knowledge that there is differential<br>
expression of Leptospira   antigens during culture (in   vitro)<br>
and  during  host  infection  (in      vivo).	Differential<br>
expression of Leptospira antigens is presumed to be important in host adaptation during infection. We used a strategy to identify immunoreactive antigens and therefore antigens expressed during host infection. Sera from patients infected with pathogenic Leptospira were used to select polynucleotide sequences from genomic Leptospira DNA library in lambda phage that encode for immunoreactive polypeptides.<br>
The present invention identified and isolated three polynucleotides with nucleotide sequences corresponding to SEQ ID No:l, SEQ ID No:3 and SEQ ID No:5, as well as the amino acid sequences of the respective polypeptides, , BigLl, BigL2 and BigL3, encoded by such nucleotides.<br>
Step 1 - The screening the positive clones consisted basically of the following steps:<br>
(a) The DNA of a pathogenic Leptospira was cut with an appropriate enzyme and ligated into a specific site in the lambda phage genome. Host bacteria were infected with phage and   the   resulting   clones,   expressing   recombinant<br><br><br>
polypeptides after induction with IPTG, were submitted to immunoblot protocol where a membrane of colony lysates was incubated with sera from patients with laboratory confirmed leptospirosis and then with a secondary antibody conjugated to horseradish peroxidase, which recognized human Ig. Positive clones were detected through an indicator reaction, for antigen-antibody complexes based on the production of color.<br>
(b)	The sequence of cloned and isolated polynucleotides was<br>
determined using phage vector-specific sequences as<br>
initiators of the sequencing reaction. Analysis of the<br>
clone sequences and the use of a primer walking strategy<br>
identified the complete nucleotide sequence for the genes<br>
that encode for BigLl, BigL2, and BigL3.<br>
(c)	Most of the obtained positive clones contains genes<br>
encoding proteins of thermal shock Hsp58 and DnaK and the<br>
protein of outer membrane LipL41. However, it was found<br>
clones containing genes encoding repetitions in tandem of 90<br>
amino acids compared by Database of proteins family (Pfam)<br>
as proteins of bacterium, type immunoglobulin (Big). With<br>
the analysis of the clone sequences, were identified 3 genes<br>
containing 12 tandem repeats for bigLl and 13 tandem repeats<br>
in bigL2 and bigL3.<br>
Step 2 - Subcloning expression and purification of the protein<br>
-	Drawing of two oligonucleotides with base in sequences of two proteins BigL<br>
-	Amplification by PCR of the initial BigL portion encoding for part of the repetitive region, from those oligonucleotides<br>
-	Sequencing of the product of the amplification<br>
-	Subcloning of the region-encoding by the product sequenced<br>
-	Expression of the recombinant protein.<br>
-	Purification of the recombinant protein.<br>
Immunoblot analyses demonstrate that sera from leptospirosis patient and rodent reservoirs infected with pathogenic Leptospira produce antibodies primarily to the BigL domain repeats of the BigL polypeptides, indicating that they are the main antigenic regions recognized during infection.<br><br><br>
In relation to the polypeptides of the present invention they consist of sequences of DNA, cDNA or RNA (and sequences of nucleic acids which are complementary), as well as their functionally equivalent sequence, i.e., those sequences that encode the whole or a part, of the polypeptides designated as BigLl, BigL2 and BigL3, but are non-identical due to variability.<br>
The polypeptides and polynucleotides in the present invention consist of BigLl, BigL2 and BigL3 and the polynucleotides that encode these polypeptides; however they include, in addition, polypeptides and polynucleotides that have functionally equivalent  sequence.<br>
In the present invention, both polynucleotides and polypeptides may be of natural, synthetic or recombinant origin, having the necessary purity degree to grant to their biological activities.<br>
The present invention also refers to the polynucleotides encoding for BigLl, BigL2 and BigL3 which are used in PCR reactions to obtain either complete or partial amplified DNA fragments of the bigL polynucleotides, for the purpose of detection of Leptospira in samples or expression of recombinant BigL polypeptides. In the case of initiators used for the polynucleotide amplification in the present invention, they are oligonucleotides made of two or more deoxyribonucleotides or ribonucleotides, natural or synthetic.<br>
Each initiator is preferably constructed in order to be substantially similar to a flanking region of the sequence strand that is the target for amplification. In this sense, an initiator can be designated functionally equivalent if corresponding polymers can produce the same process, without being identical, facing the utilization or application considered.<br>
Polynucleotide sequences of this invention can also be inserted in an expression vector, such as a plasmid, virus or other vehicle used for recombinant cloning, which is used by inserting or incorporating whole or partial nucleotide sequences that encode for BigLl, BigL2 and BigL3 or their functionally equivalent sequences. Such expression vectors contain a promoter sequence that facilitates the efficient transcription from genetic sequence in the host in which the vector is inserted. Such hosts can include prokaryotes or eukaryotes, including microorganisms such as yeast or insects and mammals. Such processes for the use of expression vectors construction and for the expression of<br><br><br>
recombinant sequences, properly so-called, are well known by experts in technique.<br>
The present invention provides for a method to produce antibodies that bind to complete or partial polypeptides of BigLl, BigL2 and BigL3 or their functionally equivalent sequences. Such antibodies are useful as research and diagnostic tools in the study and diagnosis of spirochete infections in general, and more specifically in the development of diagnostics and therapeutics whether treatment or prevention, for leptospirosis. Such antibodies can be administered alone or as part of a pharmaceutical composition that use these antibodies and a pharmaceutically acceptable carrier as part of anti-spirochetal therapeutic.<br>
The invention is relates to the use of pharmaceutical compositions of BigL polypeptides or the polynucleotides that encode for these polypeptides as vaccines, either as a vaccine for prevention of disease which induces an immunoprotective; response to infection or colonization with pathogenic spirochetes or as therapeutic vaccine that provides a beneficial impact in reducing the duration or severity of the clinical course of illness in an subject due infected with a pathogenic spirochete or in reducing the reservoir state of a carrier of pathogenic spirochete such as in pigs, cows, rats or dogs that harbor and excrete pathogenic spirochetes for prolonged periods of time. Such compositions may be prepared with an immunogenically effective quantity of an antibody against BigLl, BigL2 and BigL3 respectively, or with one or more of BigLl, BigL2 and BigL3 isolated from the leptospiral pathogen or recombinant BigL polypeptides, or its functionally equivalent sequences, in excipients and additives or auxiliaries.<br>
Another embodiment of present invention relates to the pharmaceutical composition used to induce an immune response to a pathogenic spirochete in an individual, particularly Leptospira sp. , including a immunologically effective quantity of BigLl, BigL2 and BigL3 or of their functionally equivalent sequence in a pharmaceutically acceptable vehicle. As "individual" we refer to any mammal, including humans, rodents, domesticated and laboratory animals and livestock. As "quantity immunologically effective" we refer to quantity of BigL polypeptide antigen necessary to induce, in an individual, an immunological response against Leptospira or any other pathogenic spirochete or bacterial pathogen.  The invention further provides a kit for:<br>
1 - detecting one   of polypeptides, BigLl, BigL2 and BigL3, or'their functionally equivalent sequences;<br><br><br>
2	-detecting nucleic acid encoding for BigLl,  BigL2 and<br>
BigL3 or their functionally equivalent sequences;<br>
3	- detecting antibodies for such polypeptides, BigLl, BigL2<br>
and BigL3, or their functionally equivalent sequences.<br>
The kit used for detection of BigL polypeptides includes those that use a vehicle containing one or more receptacles with a first receptacle containing a linking reagent to BigLl, BigL2 and BigL3 or to their functionally equivalent sequences.<br>
The kit used for detection of polynucleotides that encode BigL polypeptides includes those that use a vehicle containing one or more receptacles with a first receptacle containing a polynucleotide that hybridizes to the nucleic acid sequence that encodes BigLl, BigL2 and BigL3 or to their functionally equivalent sequences.<br>
The kit useful for detecting antibodies against BigL polypeptides includes those that use a vehicle containing one or more receptacles with a first receptacle containing a polypeptide of BigLl, BigL2 and BigL3 or of their functionally equivalent sequences.<br>
The present invention will be now described with reference to the Examples, which are should not be considered as limitative of the present invention.<br>
EXAMPLE 1:<br>
Example 1A: Bacterial strains, plasmids and media<br>
Leptospira kirschneri serovar grippotyphosa, strain RM52, was isolated during an outbreak of porcine abortion in 1983). L. interrogans serovar copenhageni, strain Fiocruz (Ll-130), was isolated from the bloodstream of a human leptospirosis patient. L. kirschneri serovar grippotyphosa strain RM52 and other leptospiral strains were obtained from the National Leptospirosis Reference Center (National Animal Disease Center, Agricultural Research Service, U.S. Department of Agriculture, Ames, Iowa). Leptospiral strains were cultivated at 30°C in Johnson-Harris bovine serum albumin-Tween80 medium (Bovuminar PLM-5 Microbiological Media, Intergen (2). Low-passage samples of the RM52 isolate were either stored in liquid nitrogen or passaged in liquid medium at least 200 times to generate a high-passage form. The high-passage strain was unable to produce a lethal infection in hamsters at any dose and was only able to infect hamsters at a dose of 10 by intraperitoneal inoculation.<br><br><br>
Escherichia coli XLl-Blue MRF'DmcrA)183QmcrCB-hsdSMR-mrr)173 endAl supE44 thi-1 recAl gyrA96 relAl lac [F'proAB lacIqZDM15 TnlO (Tetr)] (Stratagene) and E. coli PLK-F' {endAl gyrA96 hsdR17 lac' recAl relAl supE44 thi-1 [F' laci*ZUM15} ) were used as the host strains, for infection with the A,Zap II (Stratagene) and A.TriplEx (Clontech) vectors, respectively. E. coli SOLR (el4~[mcrA], Q[mcrCB-hsdSMR-mrr)171   sbcC recB recJ  umuC: :Tn5[Kanr] uvrC lac  gyrA96  relAl<br>
thi-1 endAl	kr, [F' proAB lacI^ZDMlS] , Su~ [non-<br>
suppressing]]	and E. coli BM25.8 (supE44 thi Ulac-proAB [F'<br>
traD36 proAB*	lacPZDMlS] \imm434 (kan1) Pl(ca/nr) hsdR(r*12'm*u-<br>
) ) were used	for in vivo excision of the pBluescript and<br>
pTriplEx phagemids, respectively. BLR(DE3)pLysS [F~ ompT hsdS% (rs-niB-) gal dcm _ (srl-recA) 306: :TnlO(TcR) (DE3) pLysS(CmR)] (Novagen) was used as the host strain for the pRSET expression vector (Invitrogen). E. coli strains were grown in LB supplemented with 100 ng/ml ampicillin, 100 fig/ml carbenicillin, or 25 ng/ml chloramphenicol where appropriate.  Antibiotics were purchased from Sigma.<br>
Example IB: Isolation and Characterization of bigL  Genes;<br>
This example illustrates the identification and isolation of the bigL genes. Genomic DNA was prepared from virulent, low-passage L. kirschneri, serovar grippotyphosa, strain RM52 by the method of Yelton and Charon (15). Genomic DNA was prepared from a clinical isolate of L. interrogans, serovar Copenhagen!, strain Fiocruz Ll-130, using a kit to genomic DNA (Qiagen) . The QIAquick PCR Purification Kit (Qiagen) was used to obtain purified DNA. The genomic DNA was partially digested with Tsp509I and<br>
ligated to X.TriplEx arms following the instructions provided (Clontech). The Gigapack III Gold Packaging Extract (Stratagene) was used to packaged ligated, digested genomic DNA into lambda heads. The phage titer of the library was determined by infection E.   coli  XLl-Blue.<br>
For screening of genomic library, approximately 103 pfu approximately were plated on a lawn of E. coli XLl-Blue, transferred to nitrocellulose membrane (Schleicher &amp; Schuell), sensibilized with IPTG and processed as recommended (Schleicher &amp; Schuell). The nitrocellulose filter was blocked with 5% skimmed milk in Tris-buffered saline (pH7.8) with 0.05% Tween 20 (TBST) or phosphate-buffered saline (pH 7.4) with 0.05% Tween 20 (PBST) , and incubated for 1 hour with pooled sera, diluted 1:50, from patients with laboratory-confirmed leptospirosis. Sera were collected from patients, identified in urban epidemics in<br><br><br>
Brazil between 1996 and 1999, during the convalescent-phase of illness, and were pre-absorbed with E. coli lysates prior to use to remove antibodies to E. coli. Membranes were washed three times with TBST or PBST, and incubated for more than 1 hour with rabbit or goat anti-human immunoglobin antibody conjugated with alkaline phosphatase (Sigma) in the dilution of 1:1000. Detection with NBT (0,3 mg/ml) and BCIP (0,15 mg/ml) or development with the ECL Western Blot Detection Reagents (Amersham) followed by exposure to Hyperfilm (Amersham) was used to identify plaques with antigen-antibody complexes.<br>
Each positive plaque was stored at 4CC in 1 ml SM (0,1 M NaCl, 8 mM MgSOi, 50 mM Tris-HCl pH 7,5; 0,01% in gelatin, with 1-2 drops of chloroform. The lambda plaque clones that reacted with pooled sera were subjected to two additional stages of purification. The genomic DNA fragments inserted into lambda bacteriophage were excised by infecting E. coli SOLR or BM25.8 strains with the lambda clones as described by the supplier (Stratagene and Clontech, respectively).<br>
The sequence of the first 500-700 nucleotides of the insert was obtained using a vector-specific primer that links adjacent to the insert. Nucleotide sequence analysis of 131 clones identified 13 that had DNA fragment inserts, found to encode tandem repeats approximately 90 amino acids in length. Each of the repeat sequences were subsequently identified in Pfam 6.6 (http://pfam.wustl.edu/) to belong to the Big2 family Big2 family of bacterial immunocjlobulin-like (Big) domains.<br>
To identify sequences that encode full-length proteins according to the predicted amino acid sequence, the nucleotide sequences of the clones were assembled from individual sequences obtained by a combination of primer walking and sequencing of nested deletions. The deletions were generated from the plasmid clones by removal of restriction fragments extending from inside the insert into the multicloning sites flanking the insert. Oligonucleotides were synthesized and obtained from GIBCO BRL or Operon. Inverse PCR (iPCR) was performed to obtain sequences containing the remainder of the genes and flanking DNA. The UCLA Core Sequencing Facility, the Yale/Keck Core DNA Sequencing Facility and the University of California at Berkeley Sequencing Facility performed the sequencing reactions.<br>
Two L. kirschneri clones and four L. interrogans clones<br>
were found to encode a gene which we designate bacterial<br>
ijnmunocjlobulin-like Leptospiral protein one, bigLl.	The<br>
complete nucleotide sequence of L.   kirschneri   bigLl   and the<br><br><br>
predicted amino acid sequence of the gene product is shown in SEQ ID NO: 1 and SEQ ID NO: 2. Six L. kirschneri clones were found to encode a second gene which we designated bigL2. The complete nucleotide sequence of L. kirschneri bigL2 is shown in SEQ ID NO: 3. L. kirschneri bigL2 appears to be a pseudogene, an extra adenine residue occurs at nucleotide 1011 resulting in a frameshift mutation and downstream TAG stop codon. However, the antibody screening with pooled patient sera was able to identify lamda clones with DNA fragments encoding bigL2 gene products, presumably since the cloned fragments did not have the frameshift mutation and were inserted in an orientation that allowed expression of a product that was recognized by patient sera. The predicted amino acid sequence of the L. kirschneri bigL2 gene product, without the frameshift mutation, is shown in SEQ ID NO: 4. A fifth L. interrogans clone was found to encode several Big ■ repeats initially thought to belong to BigLl. However the upstream DNA encoded by this fifth L. interrogans clone was found to differ from the sequence upstream of bigLl. Sequencing the regions flanking the bigLl gene revealed that the fifth L. interrogans clone corresponded to a third gene, designated bigL3, downstream of bigLl (FIG. 2). The complete nucleotide sequence for bigL3 was obtained from L. kirschneri DNA and is shown in SED ID NO: 5. The predicted amino acid sequence of the I. Jcirschneri bigL3  gene product is shown in SEQ ID NO: 6.<br>
All three bigL genes encode a signal peptide and putative signal peptidase cleavage site largely conforming to the spirochetal lipobox, as previously defined (Haake, D. A. 2000. Spirochetal lipoproteins and pathogenesis. Microbiology. 146:1491-1504). Comparison of the sequences of known spirochetal lipoproteins indicates that the spirochetal lipobox is much more loosely defined than the E. coli lipobox. For example, while most E. coli lipoproteins have Leu in the -3 position relative to Cys, spirochetal lipoproteins may also have a number of other hydrophobic amino acids in this position, including Val, Phe, and lie. E. coli experiments involving site-specific mutagenesis of amino acids following cysteine indicates that acidic residues cause sorting of lipoproteins to the cytoplasmic membrane. Sequence analysis of leptospiral lipoproteins indicates that a similar sorting signal is present in these bacteria. For example, LipL31 is the only lipoprotein having an unopposed negative charge in the first two amino acids following cysteine, and is also the only lipoprotein sorted exclusively to the cytoplasmic membrane. Like the outer membrane lipoproteins LipL32 and LipL41, the BigL proteins have uncharged amino acids in the +2 and +3 positions, indicating that they would be sorted to the outer membrane.<br><br><br>
Following their signal peptides, all three proteins would contain a series of tandem repeats, approximately 90-amino-acids in length. The mature BigLl protein would consist almost entirely of thirteen repeats, while in contrast BigL2 and BigL3 contain twelve repeats followed large carboxy-terminal domains. Though there is a high degree of sequence variation among the 31 unique repeats found in the three proteins, all of the repeats were identified by the Pfam database as bacterial immunoglobulin-like Big protein family with E-values as low as 4 x e"30.<br>
The L. interrogans and L. kirschneri versions of bigLl, bigL2, and bigL3 were highly related, with &gt;90% dna and amino acid sequence identity. In both species there is a region of DNA sequence identity involving the 5' ends of bigLl and bigL3 (FIG 2) . The region of sequence identity begins extends from the initial ATG start codon to position 1890 bp in both genes. The large region of DNA sequence identity between bigLl and bigL3 results in an identical amino acid sequence for the first 630 amino acids {positions 1-630) of BigLl (SEQ ID NO: 2) and BigL3 (SEQ ID NO: 6). This region of identity corresponds to the first six BigL domain repeats.<br>
EXAMPLE 2<br>
Example 2A: Characterization of the bigL  genes and Detection of bigL   DNA and RNA<br>
This example illustrates the distribution of multiple copies of bigL genes among Leptospira species and methods to detect bigL  DNA and RNA in samples.<br>
Southern Blot Analysis<br>
Southern blot analysis was performed to identify multiple copies of bigL genes in genomic DNA from L. interrogans strain Fiocruz Ll-130, L. kirschneri strain RM52, and L. Jbiflexi strain Patoc I. DNA restriction and modifying enzymes were purchased from New England Biolabs. Genomic DNA was extracted from from 500ml of 7-day cultures of Leptospira cells with the Blood and Cell Culture kit (Qiagen, Valencia, Calif.). Approximately 3mcg of DNA was digested with 5-20 units of Wsil overnight in a final volume of 50mcl. DNA was then purified with phenol:chloroform:isoamyl and precipitated with 100% cold ethanol and 3M sodium acetate pH and washed with 70% ethanol. Purified DNA was then re-digested with 5-20 units Pad overnight in a final volume of 25 mcl. The double digested DNA was separated in a 0.8% agarose gel at 20V<br><br><br>
overnight. The gel was then incubated twice for 30 minutes in denaturing buffer (1.5 M NaCl, 0.5 N NaOH) , and twice for 30 minutes in neutralization buffer (1M Tris (pH7.4) 1.5 M NaCl). Genomic DNA was transfered onto a positively charged nylon membrane (Roche Molecular Biochemicals, Indianapolis, Ind.) according to the method described by Southern.<br>
Probes were synthesized with the PCR Dig Probe Synthesis kit (Roche, Manheim, Germany) . Reactions were assembled according to the manufacturer in a final volume of 50mcl. Temperature cycles for the amplification were 94 °C for 5 min, 94°C for 30 sec, 57°C for 30 s min, and 72°C for 1 min, with a final extension time of 7 min after a total of 35 cycles. Probe sequences were as follows: to amplify the bigi DNA fragments that encodes for BigL repetitive domains, a bigL3 DNA sequence was selected that correspond to the region that encodes for BigL3 repetitive domains 4-6, BigL3_395 gat-ttt-aaa-gtt-aca-caa-gc and BigL3_573 aaa-ccg-gac-tac-tta-cct-ttc-c; and to amplify bigL DNA fragments that are specific for each of the bigL genes, sequences that encode for C-terminal regions of the BigL gene products were selected: BigL1.2078p, tta-cgg-cta-cag-gta-ttt-tta-cg and BigL1.2691p att-gga-aga-ttt-cca-agt-aac-c, BigL2.5121p tat-cta-cgc-tgc-aaa-tgg and BigL2.5865p ttg-ttg-gcg-ata-cgt-ccg, BigL3.5071p cat-aac-tct-cct-cat-aac-a and BigL3.5548p tat-gta-gag-ata-aga-tcc.<br>
The UV Crosslinked membrane was subject to prehybridization . at 42°C for 1 hour in Dig Easy Hybridization solution (Roche). Prior to hybridization, the Dig labeled probes were boiled for 10 minutes and rapidly transferred to ice for 5 minutes. The denatured probes were mixed with hybridization solution and incubated overnight with the membrane at 42°C. Following hybridization, the membranes were washed twice for 5 minute at room temperature with 2xSSC (NaCl, Sodium Citrate), 0.1%SDS. The membranes were then washed twice for 30 minutes at 42°C with 0.1 SSC, 0.1% SDS. Membranes were exposed for 1-3 minutes to Biomax ML film (Eastman Kodak, Rochester, N.Y.) for the detection of chemiluminescent products<br>
FIGS. 1A and B shows the results of the Southern blots.<br>
Probes corresponding to DNA sequences that encode BiGL<br>
repeats hybridized to multiple DNA fragments in L.<br>
kirschneri and interrogans (FIG. 1A) . In contrast,<br>
hybridization was not identified with digested genomic DNA<br>
from the  non-pathogenic  1. biflexi.	Probes  based  on<br>
sequences that encode for specific C-terminal regions for each of the L. interrogans bigL gene products hybridized to one unique fragment in digested L. interrogans genomic DNA, therefore confirming that there are one copy of each of the<br><br><br>
three bigL gene identified in Example 1 (FIGS. IB). These results illustrate a method of identifying specifically pathogenic Leptospira based on detection of DNA fragments not found in non-pathogenic Leptospira.<br>
Example 2B: PCR Detection of bigL gene sequences in Leptospira  genomic DNA<br>
This example illustrates the distribution of bigL gene in pathogenic Leptospira. In order to detect bigL genes in other Leptospira species, degenerate primers were designed based on an alignment for bigL genes from L. kirschneri strain RM52 and L. interrogans strain Fiocruz Ll-130, identified in Example 1. The sequence of the "upstream" primer, designated BigL-lup, is 5'-(GC)AAAGTTG(TC)(AG)(TC)G(TG)CTTGGCC-3' corresponding to positions 46-65 in JbigLl and bigL3 (SEQ ID NO: 1 and 5), relative to A of start c6don. The sequence of the "downstream" primer, designated BigL-2dn, is 5'-(GC)(AT)ACC(AG)TC(CT)GAAAA{AG)AT(AT)CC-3' corresponding to positions 506-487 in bigLl and bigL3 (SEQ ID NO: 1 and 5) , relative to A of the start codon. Each primer is 20 nucleotides long. These primers were designed to anneal to bigL2 at positions 97-116 and 590-571relative to the A in bigL2's start codon (SEQ ID NO: 3).<br>
PCR reactions were performed with purified genomic DNA from high and low-passage strains of Leptospira. In FIG 3., amplified DNA fragments were identified in PCR reactions with genomic DNA of strains in all four pathogenic species evaluated. Fragments had the predicted electrophoretic mobility based on the sequences of bigLl/bigL3 (461 bp) and bigL2 (494 bp). Amplified DNA fragments were not identified in the two non-pathogenic Leptospira species evaluated. Therefore this example illustrates the application of this PCR method for identifying specifically DNA from pathogenic Leptospira in samples.<br>
Example 2C: Reverse transcriptase-Polymerase Chain Reaction (RT-PCR) Detection of Leptospira bigL  RNA<br>
This example illustrates the detection of bigL RNA in samples. L. kirschneri strain RM52 was grown to late exponential phase, and total RNA was extracted from 1 x 1010 leptospiral cells using the hot-phenol method and resuspended in water following ethanol precipitation (ref) . ~2 Dg of leptospiral RNA was digested with 6 units of DNase I (Ambion) in 70 Dl DNase I buffer (10 mM Tris-HCl pH 7.5, 25 mM MgCl2, 1 mM CaCl2 in lx RNA secure from Ambion) for 30 min at 37D. To inactivate DNase I, 1.75 Dl of 25 mM EDTA was added to terminate the reaction, and the enzyme was heat<br><br><br>
killed for 5 min at 70D.  RT-PCR was	performed using ~200 ng<br>
leptospiral RNA and Omniscript RT	as described  (Qiagen).<br>
The  following  primers  were  used	to  prime  the  reverse<br>
transcriptase reaction:<br>
bigLl, 5'-CGCAGAAATTTTAGAGGAACCTACAG-3' bigL2,   5' -TTTGACTCCAAGACGCAGAGGATGAT-3'<br>
bigL3,   5' -ATTTTCAAGATTTGTTCTCCAGATTT-3'; lipL45,    5'-ATTACTTCTTGAACATCTGCTTGAT-3'.<br>
The RT reactions were subjected to DNA PCR using Taq polymerase (Qiagen). Prior to PCR, the following primers were added to the reactions:<br>
bigLl, 5' -CTGCTACGCTTGTTGACATAGAAGTA-3' bigL2, 5' -TAGAACCAACACGAAATGGCACAACA-3' bigL3, 5' -ATCCGAAGTGGCATAACTCTCCTCAT-3' lipL45,   5'-TGAAAAGAACATTACCAGCGTTGTA-3'.<br>
Along with the primers added for reverse transcription, PCR products of 500 bp, 479 bp, 440 bp, and 438 bp are expected. To perform PCR, the reaction mixtures were placed in a Techne Progene thermocycler. An initial denaturation step of 95D for 1 min was followed by 30 cycles of denaturation at 95D for 30 sec, annealing at 53D for 30 sec, and extension at 72D for 30 sec. A final 72D incubation for 30 sec was then performed.<br>
The results in FIG. 4 show that RT-PCR method can detect BigL3 transcripts and the control LipL46 transcripts. BigLl and BigL2 transcripts were not identified indicating that that whereas BigL3 is expressed in Leptospira, BigLl and BigL2 may not be. Furthermore, these results demonstrate the application of the RT-PCR method to identify specific BigL gene transcripts in samples.<br>
EXAMPLE 3<br>
Expression and Purification of Recombinant BigL Proteins<br>
This example illustrates the use of the DNA sequences of bigL genes to express and purify recombinant BigL polypeptides. Two pairs of oligonucleotides were designed for use in expressing two regions of L. interrogans BigL3. The first region was a region within BigL3 corresponding to the 2nd to 6th repetitive domains and corresponded to positions 131-649 of SEQ ID NO: 6 in the L. kirschneri BigL3DNA sequence. Oligonucleotides were designed based upon sequence of lambda L. interrogans BigL3 clones identified in Example land their sequence are:<br><br><br>
4 5B-1 5'-ATGGGACTCGAGATTACCGTTACACCAGCCATT-3' 45B-2 5'-ATTCCATGGTTATCCTGGAGTGAGTGTATTTGT-3'<br>
PCR amplification with oligonucleotides 45B-1 and 45B-2 and purified L. interrogans genomic DNA was performed to obtain DNA fragments. These fragments were digested with Xhol and Ncol Enzymes (New Biolabs) and then ligated into the pRSETA expression vector (Invitrogen) (16). The cloned product was sequenced using vector specific primers and primer walking and the sequence of the 1557 bp product is shown in SEQ ID NO: 7. The predicted sequence of the encoded 519 amino acid polypeptide, designated BigL3 region 1, is shown in SEQ ID NO: 8.<br>
A second region was selected for expression that contained the final 200 amino acids of the C-terminal region of L. interrogans BigL3. This region corresponded amino acid positions 1687-1886 of SEQ ID NO: 6 in L. kirschneri BigL3.  The oligonucleotides used to clone this region are:<br>
BIGLCTERM1 5' aac-ctc-gaq-cat-aac-tct-cct-cat-aac 3' BIGLCTERM2 5'  tto-gaa-ttc-tta-ttq-att-ctq-ttq-tct-q 3'<br>
PCR amplification with oligonucleotides BIGLCTERM1 and BIGLCTERM2 and purified L. interrogans genomic DNA was performed to obtain DNA fragments. These fragments were digested with Xhol and EcoRI enzymes (New Biolabs) and then were ligated into the pRSETA expression vector (Invitrogen) (16). The cloned product was sequenced using vector specific primers and primer walking and the nucleotide sequence of the 600 bp product is shown in SEQ ID NO: 9. The predicted sequence of the encoded 200 amino acid polypeptide, designated BigL3 region 2, is shown in SEQ ID NO: 10.<br>
Recombinant proteins, rBigL regions 1 and 2, were expressed in BL21(DE3) pLysogen (Invitrogen). Isopropyl-p1-D-thiogalactopyranoside (IPTG; 2mM final concentration, Life Technologies) was added to log-phase cultures of E. coli BLR(DE3)pLysS (Novagen) transformed with pRSET plasmids encoding leptospiral DNA fragments for expression of His6-fusion proteins. 6M guanidine hydrochloride was used to solubilize culture pellets and His6-fusion proteins were purified by affinity chromatography with Ni2+-nitrilotriacetic acid-agarose (Qiagen and Pharmacia). The purity of eluted His6 fusion proteins was assessed by gel electrophoresis and staining with Coomassie brilliant blue . Proteins were dialyzed against PBS, 10% (v/v) glycerol, 0.025% (w/v) sodium azide. After dialysis, the protein concentration was determined with bicinchoninic acid (42). A Ponceau-S (Sigma Chem Co)-stained nitrocellulose membrane<br><br><br>
after transfer of purified BigL3 region 1 is shown in FIG 7. The relative mobility of the purified BigL3 was similar to the estimated molecular mass of approximately 58 kD, which was calculated based on the predicted amino acid sequence of the recombinant protein.<br>
EXAMPLE 4<br>
Example 4A: Detection of Antibodies against Recombinant BiqL Proteins<br>
This example illustrates two among several methods that utilize BigL polypeptides to detect antibodies in subject samples. Furthermore, this example provides methods for a serodiagnostic kits for identifying infection in subjects suspected of harboring infection.<br>
Immunoblot Detecting of Antibodies to BiqL Polypeptides in Samples from Infected Subjects<br>
Purified recombinant BigL3 region 2 polypeptide (Imcg/lane) (Example 3) was subjected to sodium dodecylsulfate-polyacrylamide 12% gel eloectorphoresis (SDS-PAGE) using a discontinuous buffer system and transferred to nitrocelulose membranes (Osmomics), as previously described (17) . The nitrocellulose filter was blocked with TBST with 5% skimmed milk, incubated for more than 1 hour with pooled sera from patients with labortory confirmed leptospirosis, captured rat (Rattus norvegicus) reservoirs of Leptosprira which had urine and kidney cultures positive for pathogenic Leptospira, and experimental laboratory rats and rabbits, immunized with whole L. interrogans serovar copenhageni strain Fiocruz Ll-130 lysates. As control experiments, incubations were performed with sera from health individuals from Brazil, captured rats who had no culture or serologic evidence for a Leptospira infection and laboratory rats and rabbits prior to immunization. Sera were diluted 1:100 prior to use. After washing, membranes were incubated with goat anti-human gamma chain antibody conjugated to alkaline phosphatase (Sigma), diluted 1:1000, for more than 1 hour. Antigen-antibody complexes were detected by color reaction through with NBT (0,3 mg/ml) and BCIP (0,15 mg/ml) . Pooled sera from leptospirosis patients, captured rats who were infected with pathogenic leptospires strongly recognized purified recombinant BigL 3 region 1 protein. However, rats immunized with whole Leptospira lysates did not visibly bind to the BigL3 polypeptide, indicating that although BigL3 is expressed in cultured leptospires (Example 2, FIG. 4), there may be differential expression of the bigL3 gene. Sufficient quantities of native BigL3 protein may not be present  in  vitro  whereas,  during  natural  infection,<br><br><br>
leptospires in vivo produce sufficient quantities of BigL3 to induce a strong immune response. Furthermore, this example illustrates that a spectrum of animals produce an immune response to BigL3 during infection and detection of this immune response, and detection of antibodies to recombinant BigL3 polypeptide can be used as a method to identify infection in subjects.<br>
To further illustrate the use of a detection method for antibodies against recombinant BigL3 polypeptide, an immunoblot evaluation was performed with individual sera of patients with laboratory-confirmed leptospirosis, healthy individuals from Brazil and US and patients hospitalized or evaluated in ambulatory clinics with diagnoses other than leptospirosis. The microagglutination test and culture isolation was used to confirm the diagnosis of leptospirosis in patients with clinically-suspected disease (5). The collection of sera from leptospirosis patients was during five-year surveillance for leptospirosis in the city of Salvador, Brazil. The collection of sera from control individuals was obtained from pre-existing serum banks of hospitalized and clinic patients and healthy individuals from Salvador, Brazil and through donations from the Center for Disease Control and Prevention, USA. A list of the sera used is shown in TABLE 1. Sera diluted 1:100 were analyzed following the method described above. The finding of any visible colorization of the lmcg band of recombinant BigL3 region 1 polypeptide in the immunoblot was considered a positive reaction.<br>
FIG. 8 illustrates that sera from individual leptospirosis patients react with recombinant BigL3. Table 1 summarizes the findings that demonstrate that more than 90% of hospitalized patients and approximately 70% of outpatients with leptospirosis react to rBigL3 during active infection. All (100%) of the leptospirosis patients react to rBigL3 during the convalescent-phase of their illness. Table 2 compares seroreactivity to rBigL3 with standard diagnostic tests. RBigL3 seroreactivity was greater during the initial phase of illness to those observed for standard diagnostic tests. Healthy individuals from the US and 88% of the healthy individuals from Brazil do not react to rBigL3, demonstrating that this reaction to rBigL3 is specific. The specificity of the reaction increases to 100% when it is calculated based on the frequency of IgM seroreactivity among healthy Brazilian individuals. Together, these finding illustrate that the method has utility as a serological marker of active infection and is the basis for a kit that can be used for diagnosis with leptospirosis.<br><br><br>
Table 1 also summarizes findings for rBigL3 seroreactivity in endemic regions that have high risk for leptospirosis. 25% of the population that resides in these regions demonstrate rBigL3 IgG seropositivity, indicating that this reaction may be a useful marker to identify past infection. Among patients with confirmed leptospirosis, 56% were seroreactive agains rBigL3 during the period two years after their infection with leptospirosis (Table 2). In the period between 2 and 4 years after infection with leptospirosis, 18% demonstrated rBigL3 seroreactivity. Together, these findings -illustrate that a kit based on the immunoblot method can _ detect a past infection with leptospirosis.<br>
Example 4B:  ELISA-based Detection of Antibodies to BiqL Polypeptides in Samples from Infected Subjects<br>
This example illustrates that ELISA methods are useful in detecting antibodies to BigL polypeptides and in identifying patients with leptospirosis among those with suspected infection. Flat-bottomed polystyrene microtiter plates (Corning) were coated at 4° C overnight with HiS6-fusion rBigL3, 0.5 - 100 ng/well, suspended in 0.05 M sodium carbonate, pH 9.6 (16). The plates were washed twice with distilled water and three times with PBS, 0.05% (v/v) Tween 20 (PBST). Plates were incubated with blocking solution (PBST/1% . [w/v] bovine serum albumin) for 2 hours at room temperature and after four washes with PBST, were stored at<br>
-20°C until use. Wells were incubated with 50 pi of sera, diluted 50 to 200-fold in blocking solution, for 1 hour at room temperature with agitation. After four washes with PBST, wells were incubated with 50 pi of 5,000 to 20,000-fold dilutions of anti-human u or y-chain goat antibodies conjugated to horseradish peroxidase (Sigma) for 1 hour at room temperature with agitation. Afterwards, plates were washed twice with PBST and three times with PBS and incubated with 50 ul/well of 0.01% (w/v) 3,3',5,5'-tetramethylbenzidine in substrate buffer (0.03% [v/v] hydrogen peroxide, 25 mM citric acid, 50 mM Na2HP04, pH 5.0) for 20 minutes in the dark at room temperature. The color reaction was stopped by adding 25 uL 2 N H2SO4 and the absorbance at 450 nm was measured in an Emax microplate reader (Molecular Devices, Sunnyvale, CA).<br>
Initial assays were performed to determine the antigen concentration (mcgg/well) that best discriminated between ELISA reactions of serum samples from laboratory-confirmed leptospirosis cases (n = 4) and healthy individuals from an endemic area for leptospirosis in Brazil {n = 4) . Checkerboard titrations were performed with 50, 100 or 200-<br><br><br>
fold serum dilutions and antigen concentrations per well of 25, 50, 100 and 200 ng. FIG. 6 illustrates that significantly increased absorbance values were observed at all serum dilutions and rBigL3 polypeptide concentrations for leptospirosis patients than for control individuals.<br>
In subsequent assays to determine sensitivity and specificity, plates were coated with 50ng of rBigL3. Incubations were performed with 50 and 10,000-fold dilutions of primary sera and secondary antibody conjugate, respectively. Individual serum samples were tested in duplicate and the means' of the two measurements were calculated for analysis. Paired measurements that differed by greater than 10% were retested. One positive control serum sample which reacted with all recombinant antigens and one negative control serum sample were included, in duplicate, on each plate as a quality control measure. FIG. 7 illustrates that leptospirosis patients in the acute phase of illness had significantly increased absorbances than control individuals for IgM and IgG seroreactivity (FIG. 7). These differences increased when comparing absorbance values for patients in their convalescent-phase of illness. These experiments illustrate that an ELISA-based method for detecting antibodies against rBigL3 polypeptide is useful for identifying infection with leptospirosis and can be used as a kit for diagnosis.<br>
EXAMPLE 5<br>
Induction of an Immune Response against Leptospira in Subjects<br>
This example illustrates that an immune response against BigL proteins can be induced via immunization with recombinant BigL proteins. Purified recombinant BigL3 polypeptide derived from L. interrogans was obtained with the method described in Example 3. Laboratory rats (Wistar strain) were immunized with 40 megs of rBigL3 in Freund's adjuvant (Sigma), and inoculated subcutaneously. Additional immunizations were performed with 20 megs of rBigL3 at weeks 3 and 6. Blood was collected 7 weeks after primary immunization and process for serum. Immunoblots with rBigL3 (lmcg/lane) were prepared as in Example 4. FIG. 9 illustrates the seroreactivity of rBigL3-immunized rats. rBigL3 was an effective immunogen inducing immunoblot rBigL3 seroreactivity with titers of greater than 1:2500 after a total of three immunizations. Furthermore, antibodies raised to rBigL3 polypeptide recognized native antigens in whole Leptospira lysates (108 leptospires per lane) (FIG. 9). A band with relative mobility at 200kD is faintly stained in immunoblots as are more intensely staining bands with lower<br><br><br>
relative mobility, which may represent degradation of the 200kD or high molecular weight BigL proteins. Seroreactivity against these native antigens is specific since no reactions are observed in the pre-immune sera.<br>
Immunogenicity experiments were performed with purified recombinant BigL polypeptides derived L. kirschneri. Purified recombinant proteins were loaded onto a preparative 12% SDS-PAGE gel and allowed to migrate into the separating gel by electrophoresis. A band containing 100-200smcg of recombinant protein was excised from the gel, desiccated, ground to powder, dissolved in 1 ml of water, mixed with 1 ml complete Freund's adjuvant (Sigma), and inoculated subcutaneously and intramuscularly in New Zealand white rabbits (Harlan Sprague Dawley) that were free of leptospiral antibodies. Additional immunizations with similar amounts of fusion protein in powdered acrylamide gel mixed with incomplete Freund's adjuvant (Sigma) were administered at four and eight weeks after primary immunization. Blood was collected from the rabbits ten weeks after primary immunization and processed for serum (Harlow, 1988). Immunoblots were performed as previously described (Guerreiro et al Infect Immun 2001) with concentrations of 108 leptospires per lane.<br>
FIG 10. illustrates that immunization with rBigL3 derived from L. kirschneri induces high_ level antibody titers to native BigL3 polypeptides in L. kirschneri and other pathogenic Leptospira species such as L. interrogans. Together these findings illustrate that immunization with rBigL polypeptides induces an immune response against species of pathogenic spirochetes other than the species used to design the recombinant rBigL polypeptide. Furthermore, the antibodies produced by this method of immunization can be used to detect pathogenic spirochetes in samples.<br>
Finally, this example demonstrates that the presence of native BigL polypeptides is observed in virulent low culture passaged strains and not in avirulent attenuated high culture passaged strains (Fig 10) . Sera from rBigL3-immunized rabbits recognized a predicted 200kDa corresponding to BigL3 in whole Leptospira lysates of virulent and not avirulent attenuated strains. This example illustrates that BigL proteins are markers for virulence and that antibodies against BigL proteins can be used as a method to identify virultent strains. Since BigL may be itself a virulence factor, induction of an immune response to BigL proteins as demonstrated in the example will be useful for application as a vaccine.<br><br><br><br><br><br><br>
It will be apparent to those skilled in the art that various modifications and variations can be made to the compounds and processes of this invention. Thus, it is intended that the present invention cover such modifications and variations, provided they come within the scope of the appended claims and their equivalents. Accordingly, the invention is limited only by the following claims.<br><br><br>
References<br>
1.	Levett PN. Leptospirosis. Clin Microbiol Rev. 2001/14(2) :296-326.<br>
2.	Faine SB, Adlar B, Bolin C, Perolat P. Leptospira and leptospirosis. 2nd ed Melbourne, Australia: MediSci; 1999.<br>
3.	Farr RW. Leptospirosis. Clin Infect Dis. 1995;21(1):1-6; quiz 7-8.<br>
4.	Lomar AV, Diament D, Torres JR. Leptospirosis in Latin America. Infect Dis Clin North Am. 2000;14 (1) :23-39, vii-viii.<br>
5.	Ko AI, Galvao Reis M, Ribeiro Dour ado CM, Johnson WD, Jr., Riley LW. Urban epidemic of severe leptospirosis in Brazil. Salvador Leptospirosis Study Group. lancet. 1999;354(9181):820-5.<br>
6.	Bughio NI, Lin M, Surujballi OP. Use of recombinant flagellin protein as a tracer antigen in a fluorescence polarization assay for diagnosis of leptospirosis. Clin Diagn  Lab  Immunol.   1999;6 (4):599-605.<br>
7.	Park SH, Ann BY, Kim MJ. Expression and immunologic characterization of recombinant heat shock protein 58 of Leptospira species: a major target antigen of the humoral immune response. DNA	Cell	Biol. 1999;18(12):903-10.<br>
8.	Haake DA, Walker EM, Blanco DR, Bolin CA, Miller MN, Lovett MA. Changes in the surface of Leptospira interrogans serovar grippotyphosa during in vitro cultivation. Infect   Immun.   1991;59 (3) :1131-40.<br>
9.	Haake DA, Champion CI, Martinich C, et al. Molecular cloning and sequence analysis of the gene encoding OmpLl, a transmembrane outer membrane protein of pathogenic Leptospira spp. J Bacterid. 1993;175(13):4225-34.<br>
10.	Haake DA, Martinich C, Summers TA, et al. Characterization of leptospiral outer membrane lipoprotein LipL36: downregulation associated with late-log-phase growth and mammalian infection. Infect Immun.   1998;66(4):1579-87.<br>
11.	Haake DA, Mazel MK, McCoy AM, et al. Leptospiral outer membrane proteins OmpLl and LipL41 exhibit synergistic immunoprotection. Infect  Immun.   1999;67(12):6572-82 .<br>
12.	Haake DA, Chao 6, Zuerner RL, et al. The leptospiral major outer membrane protein LipL32 is a lipoprotein expressed during mammalian infection. Infect Immun. 2000;68 (4) :2276-85.<br>
13.	Shang ES, Exner MM, Summers TA, et al. The rare outer membrane protein, OmpLl, of pathogenic Leptospira species is a heat-modifiable porin. Infect Immun. 1995;63(8) : 3174-81.<br><br><br>
Shang ES, Summers TA, Haake DA. Molecular cloning and sequence analysis of the gene encoding LipL41, a surface-exposed lipoprotein of pathogenic Leptospira species. Infect Immun. 1996;64(6):2322-30. Xelton DB, Charon NW. Cloning of a gene required for tryptophan biosynthesis from Leptospira biflexa serovar patoc into Escherichia coli. Gene. 1984 ,-28 (2) :147-52. Flannery B, Costa D, Carvalho FP, et al. Evaluation of recombinant Leptospira antigen-based enzyme-linked immunosorbent assays for the serodiagnosis of leptospirosis. Journal of Clinical Microbiology. 2001;39(9):3303-3310. '<br>
Guerroiro H, Croda J, Flannery B, et al. Leptospiral<br>
proteins recognized during the humoral immune response<br>
to   leptospirosis   in   humans.    Infect	Immun.<br>
2001;69(8):4958-68.<br>
^<br><br><br><br><br><br><br>
WE CLAIM :<br>
1.	A purified polypeptide having an amino acid sequence as set forth in SEQ ID<br>
NO.2, 4, 6, 8, or 10.<br>
2.	A polynucleotide segment having a sequence selected from the group<br>
consisting of:<br>
c)	a polynucleotide sequence encoding an amino acid sequence as set forth in<br>
SEQ ID NO.2, 4, 6 ,8 or 10; and<br>
d)	a polynucleotide sequence SEQ ID NO.l, 3, 5, 7 or 9.<br>
3.	Antibodies that bind specifically to the substantially purified polypeptide<br>
defined in claim 1.<br>
4.	A method for detecting pathogens in a sample which comprises contacting a<br>
sample suspected of containing a pathogenic spirochete with a reagent that<br>
specifically binds to the pathogen-specific cell component and detecting<br>
binding of the reagent to the component, wherein the component is selected<br>
from SEQ ID NOS. 2, 4, 6, 8 and 10.<br>
5.	The method as claimed in claim 4, wherein the reagent that binds to the<br>
pathogen-specific cell component is an antibody against the BigLl, BigL2 or<br>
BigL3 polypeptide or polypeptides of SEQ ID NOS. 2, 4, 6, 8, 10.<br>
6.	A kit for the detecting BigLl, BigL2 and BigL3 polypeptides; or antibodies<br>
that bind to BigLl, BigL2, BigL3 polypeptides or polypeptides, wherein the<br>
proteins present in the kit are selected from SEQ ID NOS.2, 4, 6, 8 or 10 or<br>
antibodies specifically binding to said proteins.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Ny1jaGVucC0yMDA0IGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2867-chenp-2004 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Ny1jaGVucC0yMDA0IGNsYWltcyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">2867-chenp-2004 claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Ny1jaGVucC0yMDA0IGNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2867-chenp-2004 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Ny1jaGVucC0yMDA0IGNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2867-chenp-2004 correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Ny1jaGVucC0yMDA0IGNvcnJlc3BvbmRlbmNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">2867-chenp-2004 correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Ny1jaGVucC0yMDA0IGRlc2NyaXB0aW9uIChjb21wbGV0ZSkgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">2867-chenp-2004 description (complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Ny1jaGVucC0yMDA0IGRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">2867-chenp-2004 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Ny1jaGVucC0yMDA0IGRyYXdpbmdzIGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">2867-chenp-2004 drawings duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Ny1jaGVucC0yMDA0IGZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">2867-chenp-2004 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Ny1jaGVucC0yMDA0IGZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">2867-chenp-2004 form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Ny1jaGVucC0yMDA0IGZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">2867-chenp-2004 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Ny1jaGVucC0yMDA0IGZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">2867-chenp-2004 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Ny1jaGVucC0yMDA0IGZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">2867-chenp-2004 form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Ny1jaGVucC0yMDA0IHBjdCBzZWFyY2ggcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">2867-chenp-2004 pct search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Ny1jaGVucC0yMDA0IHBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">2867-chenp-2004 pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Ny1jaGVucC0yMDA0IHBldGl0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">2867-chenp-2004 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="230530-devices-for-treatment-of-sample-using-shock-waves-and-methods-thereof.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="230532-a-mechanical-self-for-vehicle.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>230531</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2867/CHENP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>27-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>16-Dec-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>FUNDACAO OSWALDO CRUZ-FIOCRUZ</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>AVENIDA BRASIL 4365, 21045-900 MANGUINHOS, RIO DE JANEIRO,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>HAAKE, DAVID A</td>
											<td>SCHOOL OF MEDICINE OF UNIVERSITY OF CALIFORNIA, LOS ANGELES,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>BAROCCHI, MICHELE</td>
											<td>DIVISION OF INFECTIONS DISEASES, UNIVERSITY OF CALIFORNIA, LOS ANGELES,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>REIS, MITERMAYER GALVAO</td>
											<td>RUA WALDEMAR FALCAO 121, 40295-001 SALVADOR, BAHIA,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>CRODA, JULIO HENRIQUE ROSA</td>
											<td>RUA WALDEMAR FALCAO 121, 40295-001 SALVADOR, BAHIA,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>SIQUEIRA, ISADORA CRISTINA</td>
											<td>RUA WALDEMAR FALCAO 121, 40295-001 SALVADOR, BAHIA,</td>
										</tr>
										<tr>
											<td>6</td>
											<td>MATSUNAGA, JAMES</td>
											<td>SCHOOL OF MEDICINE OF UNIVERSITY OF CALIFORNIA, LOS ANGELES,</td>
										</tr>
										<tr>
											<td>7</td>
											<td>RILEY, LEE W</td>
											<td>DIVISION OF PUBLIC HEALTH BIOLOGY AND EPIDEMIOLOGY, UNIVERSITY OF CALIFORNIA AT BERKELE,</td>
										</tr>
										<tr>
											<td>8</td>
											<td>YOUNG, TRACY, ANN</td>
											<td>426 STANLEY HALL, DEPARTMENT OF MOLECULAR AND CELL, UNIVERSITY OF CLAIFORNIA AT BERKELE,</td>
										</tr>
										<tr>
											<td>9</td>
											<td>KO, ALBERT I</td>
											<td>RUA WALDEMAR FALCAO 121, 40295-001 SALVADOR, BAHIA,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>G0IN33/53</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/BR02/00072</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-05-20</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>10/147,299</td>
									<td>2002-05-17</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/230531-proteins-with-repetitive-bacterial-ig-like-big-domains-present-in-leptospira-species by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:19:10 GMT -->
</html>
