<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/230631-array-having-substances-fixed-on-support-arranged-with-chromosomal-order-or-sequence-position-information-added-thereto by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:25:30 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 230631:ARRAY HAVING SUBSTANCES FIXED ON SUPPORT ARRANGED WITH CHROMOSOMAL ORDER OR SEQUENCE POSITION INFORMATION ADDED THERETO</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">ARRAY HAVING SUBSTANCES FIXED ON SUPPORT ARRANGED WITH CHROMOSOMAL ORDER OR SEQUENCE POSITION INFORMATION ADDED THERETO</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>ABSTRACT In fabricating various types of arrays such as a micro array, different kinds of biosubstances, or synthetic substances interacting with the biosubstances, are arranged and immobilized on a support such that the chromosomal order of base sequence blocks, corresponding to the biosubstances, is flticertainable. The biosubstances mny he niiclcic ni I&#x27;dH HVicii MS DNA, or polypeptides such as protein. The synthetic substances may be compounds that react with the biosubstances. By thus specifying the order of the biosubstances or synthetic substances immobilized on the support, the array can be used, for example, for screening in variety improvement of living organisms.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
TECHNICAL FIELD<br>
The present invention relates, for example, to a novel<br>
array and a fabrication method of the array, various analytical<br>
systems using the array, and representative methods of using<br>
these techniques.<br>
More specifically, the invention relates to (1) an array,<br>
such as a DNA micro array, in which biosubstances derived<br>
from a Hving organism, or synthetic substances that interact<br>
with the biosubstances, are immobilized on a support by being<br>
arranged in an orderly manner, (2) a system for analyzing a<br>
genotype of the organism of interest for display, and, in<br>
particular a genotype analyzing and display system that enables<br>
locations o( crossovers on the chromosomes to be visually<br>
recognized in hybrid individuals obtained by crossing, (3) a<br>
HVstfin  for analyzing quantitative iriiit  Inci fÂ»( llir'  riri'.nnism of<br><br>
inlrrrNl, and representative melhutU of UHltiK ll, iiiul In<br>
particular a quantitative loci analyzing system for analyzing<br>
QTL by effectively using the analysis result obtained from a<br>
nucleic acid array, and (4) a gene interaction analyzing system,<br>
and in particular a gene interaction analyzing system for<br>
effectively analyzing which genes or a group of genes are<br>
associated with the traits or genes being analyzed, by effectively<br>
using the analysis result obtained from a nucleic acid array.<br>
The invention also relates to representative methods of using<br>
such arrays and analyzing systems.<br>
BACKGROUND ART<br>
With the recent advance of the worldwide genome project,<br>
the entire genomes of many model organisms have been<br>
sequenced. Sequencing of the entire genomes of many other<br>
organisms are underway as in the sequencing ai the human<br>
genome in the Human Genome Project. As evidenced by these<br>
advances, research in molecular biology has entered the<br>
post-genome (post-sequence) era.<br>
In the post-genome era, a new approach has been used for<br>
the analysis of genome functions. Specifically, the emphasis of<br>
genome function analysis has shifted, rather drastically, from<br>
the conventional pinpoint approach whereby analysis is made<br>
by cloning individual genes associated with certain living<br>
phenomena, to a systematic and comprehensive approach<br>
whereby gene functions are analyzed on a genome scale.<br>
The genome information is also used for the analysis of<br>
transcripts and proteins. Specifically, ^transcriptome analysis<br>
and prote.c&gt;m&amp; analysis^ have won the recognition. The<br>
transcriptome analysis is used for the analysis of transcripts,<br>
whereby the expression of all transcripts in an organism or cells<br>
are  analyzed  both  systematically  and  comprehensively using<br><br>
genome information. The proteome analysis is a systematic and<br>
comprehensive method of analyzing proteins, in which the<br>
properties or expression of all proteins expressed at any given<br>
location and any given time in an organism or cells are analyzed<br>
using genome information.<br>
For the systematic and comprehensive analyses, various<br>
array techniques are often used. The array technique refers to a<br>
technique using an array, in which biosubstances, such as DNA<br>
or various proteins obtained from the organism of interest being<br>
analyzed, or synthetic substances (for example, compounds<br>
with hythophobic groups or ion exchniiRe KrcuipH) lliiil internet<br>
with such biosubstances urr immohillzeci on ti MVijiptut in tm<br>
orderly manner.<br>
With the array technique, the systematic and<br>
comprehensive analysis can be performed efficiently. For<br>
example, for the analysis of gene transcription control<br>
mechanism, it is required to measure transcription level of<br>
genes, which varies according to the state of the cell. For this<br>
purpose, use of a DNA micro array, one form of the array<br>
technique, allows for systematic measurement of transcription<br>
level of several thousand to several ten thousand of genes (see<br>
Non-Patent Documents 1-6, for example).<br>
Among such DNA micro array techniques, one that has<br>
been widely used is the DNA micro array technique developed<br>
by Affymetrix. In this technique, oligonucleotides are directly<br>
synthesized on a silica substrate using a microfabrication<br>
technique employed in the fabrication of semiconductors (see<br>
Patent Document 1, for example).<br>
For example, for the analysis of gene transcription control<br>
mechanism, it is required to measure transcription level of<br>
genes, which varies according to the state of the cell. For this<br>
purpose,   use  of a  DNA  micro  array,   one  form  of the  array<br>
H<br><br>
technique, allows for systematic measurement of transcription<br>
level of several thousand to several ten thousand of genes. Thus,<br>
through hybridization, the nucleic acid array such as the DNA<br>
micro array can produce a large amount of data concerning<br>
gene expression.<br>
However, it is practically impossible to manually process<br>
the gene expression data obtained from the nucleic acid array<br>
since the amount of data obtained in biotechnology is enormous.<br>
In view of this, there have been proposed various types of<br>
bioinformatics techniques, whereby a large volume of data is<br>
analyzed using computers. As a technique of analyzing gene<br>
expression data, it has been known to analyze gene expression<br>
patterns in clusters, as disclosed in Patent Document 2, or<br>
analyze gene expression data based on parameters and use it<br>
for clinical purposes, as disclosed in Patent Document 3.<br>
With the large data volume to be analyzed, the analysis<br>
may yield complex results. Therefore, the bioinformatics<br>
technique requires a technique of desirably displaying the<br>
analysis results. For example, as a technique concerning gene<br>
expression display, a technique for two-dimensionally<br>
displaying expression level has been known, as disclosed in<br>
Patent Document 4.<br>
Willi the recent tidvancc in Ihr unir itunllfit iidnn<br>
technique, alien genes have been introduced into vurinua pIuntB<br>
to confer new traits. Actual application of such plants aa crop<br>
plants is also underway. The development of genetically<br>
modified crops (GMO) was once believed to have a promising<br>
future in bio-industries. However, the GMO could not win<br>
customer acceptance, and, today, safety of processed foods is<br>
often promoted by not using GMO.<br>
It is therefore inconceivable that the traditional crossing<br>
or mutant induction will fade away in the variety improvement<br>
$"<br><br>
of crops. On the contrary, for' improving the market value of<br>
crops or processed foods using crops, crossing or other<br>
traditional methods are still favored as a primary method of<br>
variety improvement of crops.<br>
However, in actual variety improvement by crossing for<br>
example, a group of hybrid individuals, numbering several<br>
thousand to several tens of thousand, is screened for useful<br>
individuals by observing or analyzing traits of the hybrid<br>
individuals. As such, the efficiency of screening for superior<br>
individuals is considerably poor.<br>
The array technique and bioinformatics technique are<br>
believed to facilitate the variety improvement employing<br>
traditional crossing.<br>
One known technique of crossing is screening of a<br>
genotype using genetic markers. In variety improvement using<br>
genetic markers, it is important to recognize loci associated with<br>
target quantitative traits (QTL). The quantitative traits are<br>
governed by the polygene system, and therefore it is not<br>
possible to directly deal with the effects of expression of<br>
individual genes. This is where statistical analysis is important<br>
for the recognition of QTL. [Specifically, in order to recognize<br>
QTL, selected genetic markers are scattered along the entire<br>
chromosomes, and any linkage between the genetic markers<br>
and the quantitative traits is determined in order to map<br>
locations of QTL on a linkage map.<br>
The QTL analysis requires development of genetic markers<br>
or other materials such as hybrid lines (family lines), which are<br>
used to construct a linkage map. In addition, the QTL analysis<br>
produces a vast amount of information concerning analysis,<br>
such as measurement of traits, or typing of genetic markers<br>
(number of genetic markers B number of individuals). The array<br>
technique    and    bioinformatics   technique    are    considered    to<br><br>
facilitate the QTL analysis.<br>
In tlie analysis of ^rne rxpicMHion (inlii. llir iritii<br>
/^expression profile" is used to refer to puUcrna of ^f.nc<br>
expression or amount which vary depending on the cell type or<br>
cell stage. By measuring and analyzing the expression profile,<br>
important findings concerning gene functions or regulation<br>
mechanisms can be obtained. Such findings can be effectively<br>
used for the variety improvement of industrially useful species.<br>
In the case of humans, the analysis of expression profile can<br>
yield useful results for drug discovery, pharmacology, toxicology,<br>
and diagnosis.<br>
One technique of expression profile analysis is one that<br>
employs plustering, as disclosed in Patent Document 2 and<br>
Patent Document 5. In clustering, a group of genes that shows<br>
similar expression patterns under different measurement<br>
conditions is identified and sorted into clusters on a nucleic<br>
acid array. Another technique is one that Analyzes expression<br>
networks between genes, as disclosed in Patent Document 6.<br>
The expression level of a gene is directly or indirectly regulated<br>
by other genes, and therefore finding expression networks<br>
between genes provides important information in the expression<br>
profile analysis, as does clustering.<br>
For human applications, Patent Document 7 discloses an<br>
evaluation index estimation technique, in which genes for<br>
quantitatively estimating an evaluation index of interest are<br>
suitably selected from data obtained from, each sample. For<br>
example, in the score measuring changes in gene expression<br>
profile caused by human illness, the number of samples is<br>
considerably smaller than the number of genes on which<br>
changes in expression level are measured, owning to the<br>
difficulty in collecting a large number of samples. Thus, it is<br>
often difficult to analyze the correlation with the illness by a<br>
1<br><br>
common statistical method. In order to overcome such problems,<br>
the technique disclosed in Patent Document 7 extracts genes<br>
closely related to an evaluation index of interest and estimates<br>
evaluation index data.<br>
[Non-Patent Document 1]<br>
Genome Functions, Expression Profile and Transcriptome;<br>
Editor-in-Chief, Ken-ich Matsubara, Yoshiyuki Sakaki,<br>
Nakayama-Shoten Co., Ltd., published on September 13, 2000<br>
[Non-Patent Document 2]<br>
DNA   Micro   Array;    chief   translator,   Ikunoshinn   Kato,<br>
Maruzen, published on September 25, 2000<br>
[Non-Patent Document 3)<br>
DNA Micro Array Praclicnl Maniinl for S\iiirnsfui lUtta<br>
Acquisition, Basic Principle, from Chip Fabrication to<br>
Bioinformatics, Editor-in-Chief, Yoshihidc Hayashizaki,<br>
YODOSHA Co., Ltd., published on December ], 2000<br>
[Non-Patent Document 4]<br>
Concise and Practical Introductions to DNA Micro Array<br>
Data Analysis, YODOSHA Co., Ltd., published on November 20,<br>
2002<br>
[Non-Patent Document 5]<br>
DNA Microarrays Associate Editor: Kaaren Janssen, Cold<br>
Spring Harbor Laboratory Press, 2003<br>
[Non-Patent Document 6]<br>
Microarray Analysis, Mark Schena, John Wiley 86 Sons,<br>
Inc., 2003<br>
[Patent Document 1]<br>
Japanese Unexamined Patent PubUcation No.<br>
228999/2000 (Tokukai 2000-228999; published on August 22,<br>
2000)<br>
[Patent Document 2]<br>
Japanese        Unexamined        Patent        Publication        No.<br>
i<br><br>
342299/2000  {Tokukai 2000-34^299; published on December<br>
12, 2000)<br>
[Patent Document 3]<br>
Japanese PCT Laid-Open Publication No. 508853/2003<br>
(published on March 4, 2003; International Publication No.<br>
WOOl/016860, published on March 8, 2001)<br>
[Patent Document 4]<br>
Japanese Unexamined Patent Publication No.<br>
342000/1999 [Tokukaihei U-342000; published on December<br>
14, 1999)<br>
[Patent Document 5]<br>
Japanese Unexamined Patent Publication No. 30093/2004<br>
{Tokukai 2004-30093; published on January 29, 2004)<br>
[Patent Document 6]<br>
Japanese Unexamined Patent Publication No.<br>
175305/2002 (Tokukai 2002-175305; published on June 21,<br>
2002)<br>
[Patent Document 7]<br>
Japanese Unexamined Patent Publication No. 4739/2003<br>
(Tokukai 2003-4739; published on January 8, 2003)<br>
Conventionally, the array technique han been developed<br>
primarily for academic purposes centered on genome analysis,<br>
or for providing a research tool. As such, Ihrrr has been no<br>
Hcrrivf flrvclopment for morr prflcticid piniKiNfN. A intthlrni of<br>
the array technique then is that it is not oftrn HUitoblr for<br>
practical purposes such as identification of individuals, or<br>
genetic analysis.<br>
Specifically, in the array technique, the biosubstances or<br>
synthetic substances are immobilized on a support in an orderly<br>
fashion, but the order is not specific and the biosubstances or<br>
synthetic substances are randomly arranged in most cases. The<br>
random arrangement of biosubstances or synthetic substance,<br><br>
r<br>
however,  does  not cause  any  problem  as  long  as the  array<br>
technique   is   used   for   the    systematic   and   comprehensive<br>
analysis of genes, etc. That is, there was no special meaning in<br>
arranging   the   biosubstances   or   synthetic   substances   in   a<br>
predetermined order based on some criteria.<br>
However, the systematic and comprehensive analysis of<br>
genes, etc, has potential use in more practical applications such<br>
as variety improvement of plants, for example. In using the<br>
array technique for such purposes, it is desirable that the<br>
biosubstances and synthetic substances be analyzed with<br>
additional position information of chromosomes. In some cases,<br>
it may be required to use some kind of reference to set the order<br>
of arrangement.<br>
A problem of the conventional bioinformatics technique is<br>
that it cannot be used to efficiently perform crossing for variety<br>
improvement, QTL analysis, and the like.<br>
Specifically, in crossing, numerous numbers of<br>
individuals in the hybrid generations need to be screened for<br>
individuals in which target traits are expressed. Conventionally,<br>
it is been required to grow the hybrid generation for several<br>
years until the traits are confirmed. Further, depending on the<br>
type of trait, the traits cannot be easily recognized by simply<br>
growing the hybrid individuals. On the other hand, if the<br>
screening is performed with large gene expression data obtained<br>
from the nucleic acid array, whether target traits have been<br>
inherited or not can be efficiently confirmed with good<br>
reproducibility only by obtaining nucleic acids from the<br>
individuals of the hybrid generation.<br>
However, since the conventional bioinformatics technique<br>
concerning gene expression is not intended for such a purpose,<br>
the gene expression data obtained from the DNA micro array<br>
has not been effectively used for crossing,<br>
iO<br><br>
The QTL analysis involves statistical analysis. When only<br>
this aspect of QTL analysis is considered, the bioinformatics<br>
ir(:hni(|iir IH easily applicablt* lo the QTl. IUIHIVHIN. Ilnwrvrr, im<br>
technique is known that uses the array technique and the<br>
bioinformatics technique in combination for the QTL analysis,<br>
Further, as to the conventional bioinformatics technique<br>
concerning gene expression, it has not been passible to<br>
effectively use the technique in the QTL analysis.<br>
Further, while the conventional technique allows<br>
information concerning gene functions or regulating functions<br>
to be obtained by performing an expression profile analysis on<br>
cells of a particular type or particular stage, the technique<br>
cannot provide enough information concerning expression of<br>
genes associated with particular traits.<br>
More specifically, since the expression profile analysis<br>
analyzes expression profiles of cells of a particular type or<br>
particular stage, a comprehensive gene expression analysis can<br>
be carried out and expression patterns specific to a particular<br>
cell type or particular cell stage can be obtained. However, while<br>
the technique is useful in finding target genes or a target gene<br>
group, it is not sufficient to analy2e which genes or a group of<br>
genes are associated with predetermined specific traits or genes<br>
of interest.<br>
That is, the comprehensive gene expression analysis is<br>
useful in finding clusters or networks in a vast amount<br>
expression information, and obtaining therefrom specific genes<br>
or a group of genes. However, the technique is not effective in<br>
analyzing which genes or a group of genes are associated with<br>
specific traits or genes of interest, because the technique in<br>
which a vast amount of expression information is narrowed<br>
down to desired information involves unnecessary information<br>
processing and may cause difficulties in accurately narrowing<br>
W<br><br>
down the information.<br>
DISCLOSURE OF INVENTION<br>
The present invention was made in view of the foregoing<br>
problems, and an object of the invention is to provide an array<br>
technique in which the order of arrangement of biosubstances<br>
or synthetic substances immobihzed on a support is specified,<br>
and which is therefore applicable to, for example, screening in<br>
variety improvement of organisms.<br>
Another object of the invention is to provide {IJ a genotype<br>
analyzing and display system to be suitably used in effectively<br>
using gene expression data of a nucleic acid array in crossing<br>
for variety improvement, (2) a quantitative toci nniilyzing system<br>
to br miitulily used in effectively iminK tlnla of a inulric Htid<br>
arr&amp;y in QTL analysis, (3) a gene interaction iitm]yzmg Hyntrm<br>
for effectively analyzing, using the result of analysis obtained<br>
from the nucleic acid array, which genes or a group of genes are<br>
associated with target traits or genes that have been specified<br>
beforehand, and (4) representative methods of using such<br>
analyzing systems.<br>
The inventors of the present invention diligently worked to<br>
solve the foregoing problems, and accomplished the invention<br>
by finding that, for example, a DNA micro array can be used for<br>
screening in variety improvement of living organisms when DNA<br>
fragments immobilized on a glass substrate (support) are<br>
arranged in the order they are coded for on the chromosomes,<br>
or when information obtained from the array is analyzed with<br>
such order information.<br>
In order to achieve the foregoing objects, the present<br>
invention provides an array in which different kinds of<br>
biosubstances obtained from an organism of interest, or<br>
synthetic substances interacting with such biosubstances are<br>
\1J<br><br>
arranged and immobilized on 'a support in an orderly manner,<br>
the different kinds of biosubstances or the synthetic substances<br>
being arranged such that a chromosomal order of base<br>
sequence blocks corresponding to the biosubstances is<br>
ascertainable.<br>
In one specific example of the array in which the<br>
biosubstances or synthetic substances are arranged in such a<br>
manner that their chromosomal order is recognizable, different<br>
kinds of biosubstances or synthetic substances are arranged in<br>
the chromosomal order of respective base sequence blocks of<br>
the biosubstances. Such an arrangement will be called a "direct<br>
arrangement" (see First Embodiment).<br>
in the direct-arrangement array, it is not necessarily<br>
required that all of the biosubstances or synthetic substances<br>
are arranged in the chromosomal order of respective base<br>
sequence blocks of the biosubstances. As such, only some of<br>
the biosubstances or synthetic substances may he arranged in<br>
the chromosomal order of their respective base sequence blocks.<br>
The support may include labels that indicate the chromosomal<br>
order of the respective base sequence blocks of the<br>
biosubstances.<br>
In another example of the chromosomal order recognizable<br>
array, the biosubstances or synthetic substances immobiUzed<br>
on the support are each appended with sequence position<br>
information corresponding to the chnimoHomnl nttter of the<br>
respective sequence blocks of the biosubstunceH, udd. In UHr,<br>
data is acquired and the sequence position information is read<br>
out, so as to rearrange sequences of the data in the<br>
chromosomal order. Such an arrangement will be called an<br>
"indirect arrangement" (see First Embodiment).<br>
In a more specific example of the indirect-arrangement<br>
array, the support is realized by a collection of micro supports<br>
13<br><br>
individually immobilizing the biosubstances or synthetic<br>
substances, and each micro support is appended with sequence<br>
position information corresponding to the chromosomal order of<br>
the respective base sequence blocks of the biosubstances.<br>
Based on the sequence position information, the order of<br>
acquired data is rearranged in the chromosomal order.<br>
In the array, nucleic acids or polypeptides can be used as<br>
the biosubstances. The nucleic acid may be DNA, for example.<br>
The type of DNA is not particularly limited, but a genetic marker,<br>
genomic DNA, genomic DNA treated with restriction enzyme,<br>
cDNA, EST, and synthetic oligoDNA are preferably used, for<br>
example. It is preferable that a plurality of DNA molecules<br>
immobilized on the support be arranged based on a genetic map<br>
or physical map.<br>
As a rule, in order to quantify gene expression level, cDNA<br>
or cRNA derived from mRNA is generally used as a target<br>
sample. In addition to cDNA and cRNA, the target sample used<br>
in an array of the present invention may be genomic DNA<br>
treated with restriction en2yme, when the biosubstance is<br>
nucleic acid. Here, it is preferable that the target DNA have<br>
been fractionated by size after treated with restriction enzyme.<br>
When the biosubstance is polypeptide, proteins, fragments<br>
of proteins, or oligopeptides can be used as the biosubstances.<br>
The type of protein is not particularly limited. For example,<br>
enzymes, kinase, antibodies, receptors, and proteins with an<br>
SH3 region may be used. It is preferable that the proteins be<br>
arranged based on a genetic map or physical map {see Second<br>
Embodiment).<br>
In an array according to the present invention, the<br>
support or micro support may be an inorganic substrate, an<br>
organic membrane, or a bead. More specifically, an array<br>
according to the present invention may be  a  micro array,  a<br>
i)f<br><br>
macro array, a bead array, or a protein chip.<br>
A producing process of an array according to the present<br>
invention includes the step of orderly arranging and<br>
imniobiliziiiB on a support differrnl kintlH nf UluHuiiNltinrrH<br>
(iblftined from an organism of interest, or synlhrlic BubstnnneH<br>
interacting with such biosubstances, the step including<br>
arranging and immobilizing the biosubstances or the synthetic<br>
substances according to the order in which genes corresponding<br>
to the biosubstances are coded for on a chromosome of the<br>
organism. In the process, nucleic acids or polypeptides may be<br>
used as the biosubstances.<br>
Use of the present invention is not particularly limited.<br>
For example, the invention can be used for identification of a<br>
genotype, in which a chromosome fragment including a target<br>
trait is identified from hybrids obtained by crossing, with the<br>
use of an array using DNA as the biosubstance. The organism<br>
used for the identification of such a chromosome fragment is<br>
not particularly limited, and experimental animals and plants<br>
can be used, for example. Further, the organism used for this<br>
purpose may be a human. In this case, the genotype<br>
identification method can be used as a gene diagnosis method.<br>
The present invention can also be used, for example, for<br>
screening in variety improvement, whereby a variety including a<br>
target trait is selected, with the use of an array using DNA as<br>
the biosubstance, from hybrids obtained by crossing of<br>
organisms whose characteristics are to be improved. Here, the<br>
type of organism used for variety improvement is not<br>
particularly limited. For example, domestic animals or crops<br>
can be used. Specific examples of crops include cereals such as<br>
rice, wheat, corn, and barley.<br>
The inventors of the present invention diligently worked to<br>
achieve the foregoing objects, and accomplished the invention<br>
\
 <br>
based on the following finding. Namely, the inventors found that,<br>
in analyzing gene expression data obtained from hybrid<br>
individuals with the nucleic acid array, use of at least (1)<br>
genetic information of parents of the hybrid individuals and (2)<br>
a genetic map of the species to which these individuals belong<br>
allows the gene expression data to be analyzed based on<br>
graphical representation of locations of crossovers on the<br>
chromosomes, and thereby enables the gene expression data<br>
obtained with the nucleic acid array to be effectively used in<br>
crossing for variety improvement.<br>
Namely, a genotype analyzing and display system<br>
according to the present invention includes; a genotype origin<br>
detecting section for comparing (a) gene expression level<br>
information comprehensively obtained through n hybridization<br>
HMijlysiN of hybrid individnnlN wilh &gt;i nuilrii uriil itnuv witlt (b|<br>
genetic information of parents of the hybrid individuals, and a<br>
genetic map of a species to which the hybrid individuals belong,<br>
so as to determine whether a genotype of a hybrid individual of<br>
interest derives from which parent; and a display information<br>
generating section for gathering a plurality of results obtained<br>
from the genotype origin detecting section and, based on the<br>
results, generating display information used to display a<br>
plurality of genotypes altogether on a chromosome basis, so as<br>
to determine whether individual genotypes derives from which<br>
parent (see Fourth Embodiment).<br>
In the genotype analyzing and display system, it is highly<br>
preferable that the nucleic acid be a chromosomal location<br>
recognizable array in which a plurality of nucleic acid molecules<br>
immobilized thereon are arranged such that a chromosomal<br>
order of base sequence blocks corresponding to the nucleic acid<br>
molecules is ascertainable.<br>
It is preferable that the genotype analyzing and display<br>
\G<br><br>
system includes a genetic map constructing section for<br>
constructing, based on genetic map constructing information, a<br>
genetic map of a species to which the hybrid individuals belong.<br>
It is preferable that the genetic map constructing information<br>
includes names of genes and/or genetic markers known in the<br>
species, and chromosomal loci of the genes and/ or genetic<br>
markers.<br>
In the genotype analyzing and display system, it is<br>
preferable that the genotype origin detecting section determines<br>
a genotype as being homozygous for one of the parents,<br>
heterozygous, or unrecognizable to yield a result. Further, it is<br>
preferable that the genotype origin detecting section use<br>
genotype information and/or gene expression profile<br>
information of parents as genetic information of parents.<br>
In the genotype analyzing and display system, it is<br>
preferable that the display information generating section<br>
generate display information including at least one of<br>
recombination number and recombination frequency of<br>
individual chromosomes. Further, it is preferable that the<br>
display information generating section generate display<br>
information such that an origin of a genotype is identifiable<br>
based on different display colors or patterns.<br>
It is preferable that the genotype analyzing and display<br>
system include at least one of an input section and an output<br>
section. The input section preferably receives at least one of<br>
('()mprr))rnHJve expression Irve] infnrmntinn of ^rnrn of Ihr<br>
hybrid individuals, and genetic information of pHirnfs. J'Yirt.hrr,<br>
the input section preferably receives genetic map constructing<br>
information.<br>
The input section may be, for example, a scanner i'or<br>
enabling a hybridization result of the nucleic acid array to be<br>
read out as image information. Preferably, an image information<br>
1^<br><br>
processing section is also provided that analyzes an expression<br>
level of gene based on the image information and generating<br>
comprehensive expression level information of gene.<br>
It is preferable that the input section be a manual input<br>
section for modifying at least one of: the comprehensive<br>
expression level information of gene of the hybrid individuals;<br>
the genetic information of parents; and the genetic map<br>
constructing information.<br>
It is preferable that the output section include at least one<br>
of: a display for displaying the display information on a screen;<br>
and a printer for printing the display information. Preferably,<br>
the input section and output section are realized by an externa)<br>
communications section for sending and receiving information<br>
to and from an external device.<br>
In the genotype analyzing and display system, the nucleic<br>
acid array is generally, but not limited to, a DNA array on which<br>
DNA is immobilized. Specific examples of DNA immobilized on<br>
the DNA array include a genetic marker, genomic DNA, genomic<br>
DNA treated with a restriction enzyme, cDNA, EST, and<br>
synthetic oligoDNA. Specific examples of the nucleic acid array<br>
include a micro array, a macro array, and a bead array.<br>
Use of the present invention is not particularly limited.<br>
For example, the invention can be used for identifying a target<br>
trait-including chromosome fragment, using the genotype<br>
analyzing and display system, from hybrids obtained by<br>
crossing organisms. The organisms may be experimental<br>
animals and plants.<br>
The invention can also be used for screening for a target<br>
trait-carrying variety from hybrids obtained by crossing<br>
organisms whose characteristics are to be improved, using the<br>
genotype analyzing and display system. The organisms crossed<br>
for   variety   improvement   may   be   experimental   animals   and<br>
\^<br><br>
plants, domestic animals, or crops.<br>
The inventors of the present invention diliKf ntly worked to<br>
achieve the foregoing objects, and accomplished the invention<br>
by finding that the gene expression data ohf[iined with the<br>
niiclric lu'id array can be effccrtively used fnr (he *y\'\. IUIMIVHIH<br>
when the result of hybrid!2alion oblained from llir spoty of ihr<br>
nucleic acid array is used as genetic marker information.<br>
Namely, a quantitative loci analyzing system according to<br>
the present invention include: a genetic marker specifying<br>
section for comparing (a) com,prehensive presence information<br>
of genes of hybrid individuals, obtained by hybridizing a<br>
genomic sample of the hybrid individuals of a certain hybrid<br>
line with a nucleic acid array on which a genetic marker of a<br>
species of interest is immobilized (b) with a genetic map of a<br>
species to which the hybrid individuals belong, and genetic<br>
marker information known in the species, so as to specify a<br>
genetic marker that exists in the hybrid line; and a quantitative<br>
loci detecting section for detecting a quantitative locus of a<br>
phenotype of interest of the hybrid individual, by confirming<br>
whether a phenotypic value indicative of the phenotype is linked<br>
to the genetic marker (see Fifth Embodiment).<br>
In the quantitative loci analyzing system, it is highly<br>
preferable that the nucleic acid array be a chromosomal<br>
location recognizable array in which a plurality of nucleic acid<br>
molecules immobilized thereon are arranged such that a<br>
chromosomal order of base sequence blocks corresponding to<br>
the nucleic acid molecules is ascertainable.<br>
It is preferable that the quantitative loci analyzing system<br>
include a genetic map constructing section for constructing,<br>
based on genetic map constructing information, a genetic map<br>
of a species to which the hybrid individuals belong. The genetic<br>
map   constructing  information   preferably   includes   names   of<br>
\^<br><br>
genes and/or genetic markers known in the species, and<br>
chromosomal loci of the genes and/or genetic markers.<br>
In the quantitative loci analyzing system, it is preferable<br>
that the genetic marker information used by the genetic marker<br>
specifying section include a genetic marker with polymorphism.<br>
More specifically, the genetic marker is preferably SNP or RFLP.<br>
In the quantitative loci analyzing system, it is preferable<br>
that the quantitative loci detecting section detect a quantitative<br>
locus of phenotype by interval mapping.<br>
It is preferable that the quantitative loci analyzing system<br>
include: a scanner for enabling a hybridization result of the<br>
nucleic acid array to be read out as im.age information; and an<br>
image information processing section for analyzing an<br>
expression level of gene based on the image information and<br>
generating comprehensive expression level informntion of gene.<br>
il IH preferable that thr t|vian1ilii1ive loci inmlvziiig t*VMlrm<br>
include at least one of an input section and an outpvit section.<br>
Here, the scanner can be used as an input section. The input<br>
section preferably receives at least one of the genetic marker<br>
information and the phenotypic value. Further, the input<br>
section preferably receives at least one of the genetic map and<br>
the genetic map constructing information.<br>
Further, it is preferable that the input section be a<br>
manual input section for modifying at least one of: the<br>
comprehensive presence information of gene of the hybrid<br>
individuals; the genetic marker information, and the genetic<br>
map constructing information.<br>
It is preferable that the output section be at least one of a<br>
display for displaying an analysis result on a screen; and a<br>
printer for printing an analysis result. Preferably, the input<br>
section and output section be realized by an external<br>
communications section for sending and receiving information<br>
J-0<br><br>
to and from an external device.<br>
In the quantitative loci analyzing system, the nucleic acid<br>
array is generally, but not limited to, a DNA array on which<br>
DNA is immobilized. Specific examples of the nucleic acid array<br>
include a micro array, a macro array, and a bead array.<br>
Use oi the present invention is not particularly limited.<br>
For example, the invention can be used as a quantitative trait<br>
analyzing method for analyzing a quantitative trait of an<br>
organism, using the quantitative loci analyzing system, or a<br>
gene searching method for searching for a gene associated with<br>
expression of a trait of interest, using the quantitative loci<br>
analyzing system, or a variety improvement method for<br>
organisms, which uses the quantitative loci analyzing system.<br>
The organisms used for variety improvement are preferably<br>
laboratory animals and plants, domestic animals, or crops.<br>
The inventors of the present invention diligently worked to<br>
achieve the foregoing objects, and accomplished the invention<br>
by finding that an analysis of whether or not which gene or<br>
which group of genes is associated with a previously specified<br>
trait or gene of interest can be effectively performed when<br>
hereditary factors for regulating the expression level oi<br>
individual genes are described based on the hybridization<br>
results of the genetic markers immobilized on the nucleic acid<br>
array.<br>
Namely, a gene interaction analyzing system according to<br>
the present invention includes: a genetic marker specifying<br>
Hpftinn for comparing (a) comprehensivr ptcHcnir information<br>
of ^PiirB of hybrid Individ mils, nljliiijird by hybi idlzln^ n<br>
genomic sample of the hybrid individuals of u certain hybrid<br>
line with a nucleic acid array on which a genetic marker of H<br>
species oi interest is immobilized (b| with a genetic map oi a<br>
species  to which  the  hybrid  individuals  belong,  and  genetic<br>
Jl<br><br>
marker information known in the species, so as to specify a<br>
genetic marker that exists in the hybrid line; a spot marker<br>
information generating section for comparing the specified<br>
genetic marker with the genetic marker immobilized on the<br>
nucleic acid array, so as to generate spot marker information,<br>
being genetic marker information for use in analysis, from<br>
hybridization results obtained from individual spots on the<br>
nucleic acid array; and a hereditary factor specifying section for<br>
specifying, with regard to an arbitrarily selected phenotype and<br>
gene to be analyzed, a hereditary factor of the selected<br>
phenotype by determining whether the phenotypic value<br>
indicative of the phenotype, and an expressed gene included in<br>
expression profile information obtained from the hybrid<br>
individual are linked to a plurality of spot marker information<br>
(see Sixth Embodiment).<br>
In the gene interaction analyzing systena, it is highly<br>
preferable that the nucleic acid array be a chromosomal<br>
location recognizable array in which a plurality of nucleic acid<br>
molecules immobilized thereon are arranged such that a<br>
chromosomal order of base sequence blocks corresponding to<br>
the nucleic acid molecules is ascertainable.<br>
It is preferable that the gene interaction analyzing system<br>
include a genetic map constructing section for constructing,<br>
based on genetic map constructing information, a genetic map<br>
of a species to which the hybrid individuals belong. Further, it<br>
is preferable that the genetic map constructing information be<br>
names of genes and/or genetic markers known in the species,<br>
and chromosomal loci of the genes and/or genetic markers.<br>
In the gene interaction analyzing system, it is preferable<br>
that the genetic marker information used by the genetic marker<br>
specifying section be a genetic marker with polymorphism. More<br>
specifically, the genetic marker is preferably SNP or RFLP.<br>
W^<br><br>
In the gene interaction analyzing system, the spot marker<br>
information generating section generates spot marker<br>
information only for a genetic marker spot found by<br>
hybridization. Here, it is preferable that the spot marker<br>
information generating section generate spot marker<br>
iiiforniHlion by including position informntion of ii K'*"'^!^'^<br>
murkrr immobilized on the jjucleic ucJd arvuy.<br>
It is preferable that the gene interaction analyzing system<br>
include an expression profile information generating section for<br>
analyzing an expression profile in regard to a comprehensive<br>
gene expression level obtained from the hybrid individual, so as<br>
to generate expression profile information of the hybrid<br>
individual. The expression profile information generating<br>
section generates expression profile information of the hybrid<br>
individual by comprehensively measuring gene expression,<br>
using at least one of a micro array, a macro array, a bead array,<br>
and a differential display. Here, it is preferable that the<br>
expression profile information generating section generate<br>
expression profile information using a nucleic acid array used<br>
to obtain comprehensive presence information of gene of the<br>
hybrid individual, or a nucleic acid array on which the sample<br>
has been spotted.<br>
The DNA array on which DNA is immobilized can be<br>
suitably used as the nucleic acid array for obtaining the<br>
gene-presence-information, or the nucleic acid array for<br>
obtaining expression profiles. Specifically, the nucleic acid may<br>
be a micro array, a macro array, or a bead array.<br>
In the gene interaction analyzing system, the hereditary<br>
factor specifying section specifies a hereditary factor of a<br>
phenotype based on a quantitative trait locus (QTL) that exists<br>
among genetic markers obtained by interval mapping. Here, the<br>
hereditary  factor  specifying  section  may  uses  information   of<br><br>
expression level of a gene associated with the genetic marker, so<br>
as to specify a hereditary factor of the phenotype.<br>
The gene interaction analyzing system includes at least<br>
one of an input section and an output section. The input<br>
section receives at least one of: comprehensive presence<br>
information of gene of the hybrid individual; the genetic marker<br>
information; the phenotypic value; and the expression profile<br>
information. Preferably, the input section receives at least one<br>
of the genetic map and the genetic map constructing<br>
information.<br>
The input section is not limited to a particular structure.<br>
For example, the input section may be provided as a scanner for<br>
enabling a hybridization result of the nucleic acid array to be<br>
read out as image information. Here, it is preferable that an<br>
image information processing section be provided that analyzes<br>
an expression level of gene based on the image information and<br>
f^rnrriiting comprehensive expresaitui U'vel inrciniintifin of ^rne.<br>
Tlic HCinmcr may be used us an input HCCMHII lui entering thr<br>
expression profile information.<br>
Further, it is preferable that the input section be provided<br>
as a manual input section for modifying at least one of; the<br>
comprehensive presence information of gene of the hybrid<br>
individuals; the genetic marker information, and the genetic<br>
map constructing information.<br>
It is preferable that the output section be at least one of a<br>
display for displaying an analysis result on a screen; and a<br>
printer for printing an analysis result. Further, it is preferable<br>
that the input section and the output section be realized by an<br>
external communications section for sending and receiving<br>
information to and from an external device.<br>
Use of the present invention is not particularly limited.<br>
For example,   the present invention  may be  used  as  a  gene<br><br>
interaction analyzing method for analyzing interaction between<br>
genes, using the gene interaction analyzing system, or a gene<br>
searching method for searching for a gene associated with a<br>
trait of interest, using the gene interaction analyzing system, or<br>
a variety improvement method for organisms, which uses the<br>
gene interaction analyzing system. The organisms used for<br>
variety improvement may be laboratory animals and plants,<br>
domestic animals, or crops.<br>
For a fuller understanding of the nature and advantages<br>
of the invention, reference should be made to the ensuing<br>
detailed description taken in conjunction with the<br>
accompanying drawings.<br>
BRIEF DESCRIPTION OF DRAWINGS<br>
Fig. 1 is a schematic diagram showing a specific<br>
exemplary structure of an array according to the present<br>
invention, when the substance immobilized on a support<br>
(substrate) is DNA.<br>
Figs. 2(a) and 2(b) are plan views schematically<br>
illustrating expression of genes with particular characteristics<br>
in the array of Fig. 1.<br>
Fig. 3 is a schematic diagram showing expression of genes<br>
with particular characteristics, concerning a resulting<br>
segregating population of the cross between varieties<br>
respectively expressing genes as shown in Figs, 2(a) and 2(b),<br>
and a specific variety selected from the segregating population.<br>
Fig. 4 is a schematic diagram showing a specific<br>
exrmpliiry structure of an array arrordinK In I he present<br>
invention, when the subsliince iminohilizrd oii ii Hupjunl<br>
(aubatrtite) is protein.<br>
Fig. 5 is a schematic diagram showing a specific<br>
exemplary   structure   of   an   array   according   to   the   present<br>
J^<br><br>
invention, when the substance immobilized on a support<br>
(substance) is a compound (synthetic substance) which<br>
specifically interacts with protein.<br>
Fig. 5 is a schematic diagram showing a specific<br>
exemplary structure of a bead array as one example of an array<br>
according to the present invention.<br>
Fig. 7 is a block diagram illustrating an example of a<br>
genotype analyzing and display system according to the present<br>
invention.<br>
Fig. 8 is a view illustrating an example of display<br>
information displayed in the genotype analyzing and display<br>
system according to the present invention.<br>
Fig. 9 is a flowchart representing an example of an<br>
analysis method employed by the genotype analyzing and<br>
display system according to the present invention.<br>
Fig. 10 is a block diagram illustrating an example of a<br>
quantitative loci analyzing system according to the present<br>
invention.<br>
Fig. 11 is a flowchart representing an example of an<br>
analysis method employed in the quantitative loci analyzing<br>
system according to the present invention.<br>
Fig. 12 is a block diagram illustrating an example of a<br>
gene interaction analyzing system according to the present<br>
invention.<br>
Fig. 13 is a flowchart representing an example of an<br>
analysis method employed by the gene interaction analyzing<br>
system according to the present invention.<br>
BEST MODE FOR CARRYING OUT THE INVENTION<br>
[First Embodiment]<br>
The following will describe one embodiment of the present<br>
invention with reference to Fig.   1  through Fig.  3. It should be<br>
Â«<br><br>
appreciated that the present invention is not just limited to the<br>
particular embodiment described below.<br>
According to the present invention, there is provided an<br>
array in which substances are immobilized on a support by<br>
being arranged in a chromosomal order. The invention is<br>
applicable to a wide range of array techniques. As used herein,<br>
the "arniy techniques" refer to techniques conerMiiMH nrrnvs in<br>
wliich different kinds of subsLancctt arc ordrily uimiigcd unci<br>
immobiUzed on a support.<br>
An array according to the present invention can be<br>
classified according to the type of substance immobilized, the<br>
type of support, use, or the Uke. The invention, to a large extent,<br>
is characterized by the order of substances immobilized on a<br>
support, and therefore the following specifically describes<br>
representative examples of the invention based on different<br>
types of substances immobilized on a support. First, in the<br>
present embodiment, the invention will be described through<br>
the case where the substance immobilized on a support is<br>
nucleic acid.<br><basic structure of array><br>
The basic structure of an array used in the present<br>
invention is not particularly limited. As noted above, the<br>
invention provides an array in which a substance is immobilized<br>
on a support, Here, the support (substrate) is not particularly<br>
limited and may have any shape and may be made of any<br>
material as long as it can immobilize the substance.<br>
Examples of support materials include, generally,<br>
inorganic materials such as glass or silicon wafer; natural<br>
polymers such as paper; synthetic polymers such as<br>
nitrocellulose or nylon; and gels using synthetic polymer or<br>
natural polymer. The shape of the support is not particularly<br>
limited either as long as it has a sufficient area on which the<br>
^1<br><br>
substance can be immobilized. Generally, those with a two<br>
dimensional plane, for example, such as a substrate with little<br>
or no flexibility, a flexible membrane, or a flexible substrate<br>
with intermediate flexibility can be preferably used. The<br>
thickness of the substrate or membrane is not particularly<br>
limited either, and it can be suitably set according to the<br>
material or use of the substrate or membrane.<br>
The invention can also use bead arrays, as will be<br>
described later. As such, the support may be a collection of<br>
micro-supports on which biosubstances or synthetic substances<br>
are individually immobilized. As such micro-supports, various<br>
beads may be used, for example.<br>
Here, a collection (group) of micro-supports makes up a<br>
single support. Such a group of micro-supports is prepared and<br>
used as a dispersion liquid (or a solution} charged into a small<br>
container, in which m.icro-supports immobilizing biosubstances<br>
(nucleic acid, protein, etc.) are dispersed. In this way, data can<br>
br fredy acquired from the micro-supportH, I'^MCII rnictn HUpporl<br>
is iippended with an ID code, and dutu is ufqulicd fruni llir<br>
micro-support with the ID code. Thus, the order of substances<br>
immobilized on the micro-supports corresponds to the arranged<br>
order of data acquired from the micro-supports based on the ID<br>
codes.<br>
As used herein, the "substances immobilized on a<br>
support" refer to different kinds of biosubstances obtained from<br>
a living organism of interest, or synthetic substances which<br>
interact with such biosubstances. In other words, in an array<br>
according to the present invention, it is required that the<br>
substances immobilized on a support be at least substances<br>
associated with biosubstances derived from living organisms,<br>
Substances which are not associated with biosubstances cannot<br>
be used because, in this case, the coding order of chromosomes<br>
^H<br><br>
cannot be used as a basis of arranging these substances.<br>
Nucleic acids and polypeptides are specific examples of<br>
such biosubstances. As nucleic acids, DNA and RNA can be<br>
used. Use of polypeptides as the biosubstances will described in<br>
detail in the Second Embodiment. As to use of synthetic<br>
substances that interact with biosubstances, detailed<br>
description will be given in the Third Embodiment. Note that,<br>
the biosubstances may include sugar chains, etc.<br>
In an array according to the present invention, different<br>
kinds of biosubstances or synthetic substances are arranged in<br>
such a manner that the chromosomal order of respective base<br>
sequence blocks of these biosubstances is recognizable. Thus,<br>
for convenience of explanation, an array according to the<br>
present invention will be referred to as a chromosomal location<br>
recognizable array. In one specific implementation of such a<br>
chromosomal location recognizable array, different kinds of<br>
biosubstances are arranged in the chromosomal order. For<br>
convenience of explanation, such an arrangement will be called<br>
a "direct arrangement," because the order of the substances<br>
arranged on the array directly corresponds to the order in which<br>
these substances are sequenced on the chromosome.<br>
In another implementation of a chromosomal location<br>
recognizable array, the order of the substances arranged on the<br>
array indirectly corresponds to the chromosomal order. This will<br>
be called an "indirect arrangement."<br><br>
The present embodiment is described below in more detail<br>
based on an example (direct-arrangement array) in which DNA,<br>
as an example of nucleic acid, is arrnngfd on a support in a<br>
clirnrndsmiiul order.<br>
For example, it is assumed here lluit an array is<br>
fabricated    for   an   organism   Z   based    on    an    organism   Z<br>
c?-t<br><br>
chromosome in which 10 genes ABCl through ABC 10 are<br>
present that are lined up in this order on the chromosome, as<br>
schematically illustrated in Fig. 1. It is also assumed that the<br>
genes ABCl through ABC 10 respectively have corresponding<br>
DNA fragments (assuming that such DNA fragments are<br>
obtained). In this case, an array is fabricated by spotting these<br>
DNA fragments in an orderly manner on a substrate. Note that,<br>
in the following, the biosubstances immobilized on a substrate<br>
will be referred to as "spots" where appropriate.<br>
In spotting the DNA fragments on the substrate, a device<br>
called a spotter or arrayer is generally used. The operation of<br>
the spotter is controlled in such a manner that the DNA<br>
fragments are spotted in the order their corresponding genes<br>
are found on the chromosome. In this way, the DNA fragments<br>
are immobilized on the support by being arranged in the order<br>
"respective base sequence blocks of the biosubstances are<br>
sequenced on the chromosome."<br>
As used herein, the "base sequence block" refers to a<br>
region of a certain length in the base sequence of a chromosome.<br>
A typical example is a region corresponding to a gene that<br>
encodes a protein. It should be noted, however, that the "base<br>
sequence block" is not just limited to gene but may be a large<br>
DNA fragment like a BAC (Bacterial Artificial Chromosome)<br>
clone, or a region corresponding to only an exon. Further, the<br>
"base sequence block" may be a region, like EST, that does not<br>
necessarily include a coding region of a protein.<br>
Referring to the foregoing example, the chromosomal order<br>
may be simply the order of the genes ABCl, ABC2, ABC3, ... up<br>
to ABC 10, or the order of three different fragments of ABCl<br>
gene, three different fragments of ABC2 gene, and three<br>
different fragments of ABC3 gene, and so on. Here, the number<br>
of fragments may be three for ABCl gene, two for ABC2 gene.<br>
30<br><br>
and   five   for   ABC3   gene.   Namely,   the   order   of   substances<br>
immobilized on the support is not particularly limited as long as,<br>
when taken as a whole, it corresponds to the order in which<br>
these substances are sequenced on the chromosome.<br>
In the example illustrated in Fig. 1, a plurality of DNA<br>
fragments occurs on a single chromosome. However, the present<br>
invention is not just limited to this example, and the DNA<br>
fragments may occur in more than one clirdinoHnme. In this<br>
ciiHc, us with the foregoing, Ihr DNA IriiKiiirniN iitr niiimKrd oi)<br>
the array in the order they are sequenced on the chromosomes.<br>
Further, in the example illustrated in Fig. 1, a plurality of<br>
DNA fragments is arranged as they are sequenced on the<br>
chromosome. However, the present invention is not just limited<br>
to this example. For example, in order to meet different<br>
purposes, only some of the DNA fragments may be arranged in<br>
the chromosomal order. That is, an array according to the<br>
present invention may immobilize substances other than<br>
nucleic acids, and at least some of the different kinds of<br>
biosubstances or synthetic substances may be arranged in the<br>
order the respective base sequence blocks of the biosubstances<br>
are sequenced on the chromosome.<br>
Further, in the direct arrangement, the chromosomal<br>
order can be recognized by techniques other than arranging the<br>
substances in the chromosomal order. For example, labels<br>
indicative of the chromosomal order of respective base sequence<br>
blocks of the biosubstances may be appended on the support.<br>
As an example, labels may be provided that can<br>
distinguish between first and second rows of DNA fragments<br>
obtained from an organism of interest, wherein the first row<br>
includes 10 kinds of DNA fragments (spots) obtained from<br>
chromosome 1 and arranged in the chromosomal order, and the<br>
second row includes 10 kinds of DNA fragments (spots) obtained<br>
^ <br>
from chromosome 2 and arranged in the chromosomal order.<br>
Further, as in the indirect arrangement described below,<br>
information indicative of the type of DNA fragment immobilized<br>
on each spot may be appended as a label in the vicinity of each<br>
spot.<br><indirect-arrangement array><br>
The following describes the indirect-arrangement array. In<br>
the indirect-arrangement array, sequence position information<br>
corresponding to the chromosomal order of the base sequence<br>
blocks of the biosubstances is added to each of the<br>
biosubstances or synthetic substances immobilized on the<br>
support. This enables acquired data to be rearranged in the<br>
chromosomal order based on the sequence position information,<br>
irrespective of the order of the immobilized substances.<br>
A specific example of the in direct-arrangement array is a<br>
bead array, in which the support is a collection of<br>
micro-supports individually immobilizing biosubstances or<br>
synthetic substances (bead array will be described later). In this<br>
arrnnRemrnt, each micro-support is npppndrti witli sequence<br>
poMitidii information corrrHpondin^ lo thr- onlrr In wlilili<br>
respective base sequence blocks of the bioHubstuncea iirc<br>
sequenced on the chromosome.<br>
In use, data is acquired and the sequence position<br>
information is read out. Based on the sequence position<br>
information, the sequence of the acquired data is rearranged in<br>
the chromosomal order. By thus recognizing the chromosomal<br>
order, the substances immobilized on the micro-supports can<br>
be arranged in the chromosomal order.<br>
Note that, a specific form of the sequence position<br>
information is not particularly limited as long as it corresponds<br>
to the chromosomal order of the respective base sequence<br>
blocks of the DNA immobilized on the micro-supports.<br>
-5^<br><br><immobilized dna><br>
In the present embodiment, DNA is used as the<br>
biosubstance. The type of DNA (DNA fragment) is not<br>
particularly limited, but a genetic marker, genomic DNA,<br>
genomic DNA treated with restriction enzyme, cDNA, EST, and<br>
synthetic oligoDNA are preferably used, for example. It is<br>
preferable that the DNA be arranged based on a genetic map or<br>
physical map. For example, for a group of different kinds of<br>
genetic markers, it is preferable that these genetic markers<br>
make up a genetic map. Based on the genetic map, the DNA<br>
fragments can be arranged on a substrate.<br>
The genetic marker or a group of genetic markers are riot<br>
particularly limited as long as they can serve as genetic labels<br>
on the chromosome. Non-limiting examples include an EST<br>
marker using EST, a SNP marker including SNP (Single<br>
Nucleotide Polymorphism), a RFLP (Restriction Fragment Length<br>
Polymorphism) marker, and a micro satellite marker (SSR<br>
(simple sequence repeat) marker). Thus, the genetic marker or a<br>
group of genetic markers include genomic DNA treated with<br>
restriction enzyme, EST, synthetic oligoDNA, and the like, if<br>
they can be used as markers.<br>
The number of biosubstances immobilized on the support<br>
is not particularly limited, and it is generally on the order of<br>
several thousand (10^). The number of immobilized (or<br>
arranged) biosubstances varies greatly depending on the type of<br>
device, such as a spotter, used for the fabrication of the array,<br>
or the area of the support (substrate), for example.<br>
It should be noted that, in the DNA array, information<br>
concerning gene expression can only be obtained for genes<br>
corresponding to the immobilized DNA fragmmlH. It is therefore<br>
prrfrrtibk.' to increase the number oC iiiiiiHihl!l/cil blnHubMliiiii'fH<br>
(DNA fragments) as much as possilile, in order lo perform ^;enc<br>
i3<br><br>
expression analysis more systematically and comprehensively.<br><types of arrays used in the embodiment><br>
The type of array used in the present invention is not<br>
particularly limited and various conventional arrays can be<br>
used. Specifically, a micro array, a macro array, a bead array,<br>
or a protein chip can be used, for example. The present<br>
embodiment uses nucleic acid as the biosubstance, and<br>
therefore more specific examples include a DNA micro array and<br>
DNA macro array, for example.<br>
The DNA micro array is also known as a DNA chip, and<br>
the immobilized DNA is often referred to as a probe. The micro<br>
array is smaller in size than macro array and provides more<br>
density. This enables the number of genes (DNA fragments)<br>
immobilized as probes to be increased, allowing for more<br>
comprehensive gene expression analysis.<br>
The DNA micro array can be classified based on types of<br>
immobilized DNA. However, structural differences can be<br>
revealed more clearly if the DNA micro array is classified based<br>
on fabrication methods. Specifically, based on fabrication<br>
methods, the micro array can be broadly classified into the<br>
Stanford type and the Affymetrix type.<br>
A DNA micro array of the Stanford type is fabricated by<br>
spotting a DNA solution onto a substrate (support) with a<br>
spotter, wherein a slide glass for a microscope is used as the<br>
substrate. One advantage of a DNA micro array of a Stanford<br>
type is that it can always be fabricated with the use of a spotter.<br>
However, this comes with a drawback in that it requires<br>
expensive hardware (spotter, etc.}, or complex procedures for<br>
the preparation of biosubstances as necessitated by a large<br>
number of probes required for spotting.<br>
On the other hand, a DNA micro array of the Affymetrix<br>
type, as described in the BACKGROUND ART section, does not<br>
3.^<br><br>
employ the method of immobilizing DNA fragments on a<br>
substrate with a spotter, etc., but is fabricated by chemically<br>
synthesizing oligoDNA of about 25 mer on a substrate using a<br>
micro fabrication technique commonly used in the fabrication of<br>
semiconductors, namely, a photolithography technique.<br>
Specifically, for each gene, 11 to 20 oligos (25 mers) (for<br>
example, 11 oligos in the case of a barley DNA array) are set<br>
based on base sequence data, and a pair of oligo DNA: one with<br>
a perfect match to each 25 mer, and onr with a forced<br>
MinKlf-buse mismatch at thr 13th biiMr IN UHrd MN II [iiolir, 'I'hr<br>
array can be fabricated without using a spotter or other devices<br>
when it is designed with data of a known database. Further,<br>
since the probe (DNA fragment) has a constant length and the<br>
sequence is known, the CG content, which influences the<br>
strength of hybridization, can remain constant. It should be<br>
noted, however, that since the probe is synthesized based on<br>
information of a database, clones to be analyzed need to be<br>
separately isolated.<br>
As described above, the present invention is characterized<br>
by the order information of the sequence of the nucleic acids<br>
(biosubstances) immobiUzed on a substrate (support), and the<br>
invention can use various types of DNA, including synthetic<br>
oligoDNA, as the nucleic acids. This makes the techniques of<br>
the present invention suitable for both Stanford DNA micro<br>
array and Affymetrix DNA micro array.<br>
The following describes an exemplary method of using the<br>
DNA micro array. First, the DNA micro array is hybridized with<br>
fluorescent-labeled target DNA (hereinafter, "targets"). Here, the<br>
target molecules containing complementary sequences to the<br>
probes on the DNA micro array bind to (hybridize with) their<br>
complementary probe molecules, leaving other target molecules<br>
unbound. Then, these target molecules not bound to the probes<br>
^s-<br><br>
are washed and removed, leaving only the hybridized target<br>
molecules on the micro array. Since the target molecules are<br>
fluorescence-labeled, the fluorescence of the targets is<br>
measured as signal intensity and hybridized probes are<br>
identified.<br>
The fluorescent-labeled targets are generally prepared first<br>
by extracting mRNA from cells of two different states (first state<br>
and second state) to be compared, and then performing a<br>
reverse transcription reaction in the presence of fluorescent<br>
nucleotides. Here, two kinds of fluorescent dyes with different<br>
detection wavelengths are used for the first state and second<br>
state, respectively. The expression level of genes is greater for<br>
the cDNA contained in the targets, and the fluorescent signal<br>
intensity is in accord with the expression level of genes in each<br>
state. Thus, from the measured signal intensity, the expression<br>
level of a specific gene can be detected.<br>
The DNA macro array basically has the same structure as<br>
the DNA micro array, but differs from the DNA micro array in<br>
that it uses a common membrane filter like a nylon membrane<br>
as H subHtrate. An advantage nf the macro iirrny JH Ihnt it allows<br>
liir fin t^xpression profile itiitilysiM, nciinrnr- wu\r, mnirdliiK.', In<br>
methods based on conventional blotting nirlhoclH, Anollier<br>
advantage is that, unlike the micro array, the DNA does nol<br>
detach in washing, owning to the fact that Ihe spotted DNA is<br>
immobilized on a membrane filter after denatured by an alktili<br>
treatment. Therefore, the macro array and micro array should<br>
be suitably selected according to use.<br>
The following describes an exemplary method of using the<br>
macro array. The macro array is used basically in the same way<br>
as the micro array. Specifically, the macro array is hybridized<br>
with isotope {^^P, etc.)-labeled targets. Then, target molecules<br>
that did not bind to the array are washed and removed, leaving<br>
5^<br><br>
only hybridized target molecules on the macro array. Here,<br>
since the target molecules are isotope-labeled, the spots are<br>
exposed on an imaging plate and the expression level of the<br>
targets is determined by measuring signal intensity from the<br>
imaging plateâa procedure not performed in the micro array.<br>
The techniques of the present invention can also be<br>
applied to the mass array. In the mass array, genomic DNA<br>
fragments are arranged and immobilized in an orderly manner<br>
on a silicon substrate, and therefore the structure is basically<br>
the same as that of the micro array. The mass array was<br>
developed for SNP analysis, and as such it is used differently<br>
from the DNA micro array.<br>
Specifically, oligonucleotides corresponding to regions in<br>
the vicinity of target SNP are synthesized and hybridized with<br>
the mass array. Then, by using the oligonucleotides as primers,<br>
a DNA fragment having a SNP single base difference is<br>
synthesized through elongation catalyzed by DNA polymerase.<br>
The DNA fragment is eluted and then ionized with MALDI. The<br>
SNP type can be determined by detecting a single base mass<br>
difference using TOS-MS. Note that, as to the MALDI-TOS-MS,<br>
details will be described later in the Third Embodiment.<br>
The DNA micro array and macro array are both<br>
direct-arrangement arrays, whereas the bead array is classified<br>
as an indirect-arrangement array. The bead array is used in<br>
such a manner that, in a small container, a probe such as a<br>
nucleic acid or antibody is immobilized on a surface of each<br>
bead to which an ID code has been added, and that the probe<br>
immobilized on the probe surface is specified by reading the ID<br>
code of the bead. With use of a two-wavelength inser beam, 100<br>
kinds of beads can be quantified. That is, in an array according<br>
to the present invention, the support may br o collection of<br>
micro itrrtiys (beads,  for example) on wliicli  hldHiilmtHiiifM or<br>
?)-l<br><br>
synthetic substances are individually immobilized.<br>
In applying the invention to the bead array, each bead Is<br>
appended with an ID code containing sequence position<br>
information, as described above. In this way, measurement can<br>
be performed in the same manner as in the other techniques.<br>
Further, since the bead array allows for detection in a liquid<br>
phase, it is effective in efficiently quantifying proteins in<br>
particular. This will be described in detail in the Third<br>
Embodiment.<br><target dna><br>
The target DNA is not particularly limited. In quantifying<br>
the expression level of genes, cDNA or cRNA derived from mRNA<br>
is generally used as a target sample. In the present invention,<br>
genomic DNA treated with restriction enzyme can also be used,<br>
for example.<br>
A gene expression analysis with a common DNA array<br>
{represented by DNA micro array) is based on the principle of<br>
Northern blotting. This is effective in detecting genes having<br>
different expression patterns between two samples that differ<br>
from each other by the presence or absence of a particular<br>
disease, for example. However, if the purpose of the analysis is<br>
to detect genetic differences between the two samples, finding<br>
different gene expression is often not effective in meeting such a<br>
purpose because different gene expression does not necessarily<br>
mean that the samples are genetically different.<br>
For a comparative expression analysis of a large number<br>
of samples (lines) using known DNA micro array techniques, a<br>
strict coordination (synchronization) of growth stage is required<br>
between tested samples, or only specific tissues need to be<br>
collected. Further, since the mRNA (cDNA) used as target DNA<br>
is a collection of expressed genes, comparison can only be made<br>
for  the   information  of genes  whose  expression  is   specifically<br>
-^9<br><br>
activated or suppressed in a tested growth stage.<br>
Further, there have been many reports that suggest<br>
difficulties of a DNA micro array analysis in detecting a specific<br>
mutated gene in the genome even if it is present, owning to the<br>
fact that the expression level may not reflect the amount of<br>
transcripts, that the genes may be expressed only in limited<br>
tissues or stages, or that the amount of transcripts may be too<br>
small to be detected by the Northern blotting method.<br>
Meanwhile, diversity of genes is not necessarily governed<br>
by mutations in the coding regions of genes. For rxample, there<br>
have h^rn many reports that jiddrcHM ttir picuriicr ot (ibNriK r oC<br>
insprEion and/or deletion in the introns, or Btructural<br>
differences (for example, differences in promoter activities) in<br>
the expression regulating region like a promoter sequence.<br>
One applicable area of the present invention is variety<br>
improvement. In this application, cereals can be suitably used<br>
for variety improvement, for example. Among cereals, the<br>
genome size of barley for example is greater than that of rice by<br>
more than 10 fold. It is then highly likely that the non-coding<br>
regions, which account for the majority of the barley genome,<br>
contribute to the intraspecies diversity in barley.<br>
In a DNA array according to the present invention, the<br>
DNA fragments (biosubstances) immobilized on a support are<br>
arranged in the chromosomal order. Thus, with an array of the<br>
present invention, the location of chromosomal recombination<br>
can be grasped by a single round of testing. Thus, in an<br>
analysis using an array of the present invention, target DNA is<br>
prepared so as to allow for use of the Southern blotting method.<br>
In this way, structural mutations in the non-coding regions of<br>
genes can also be efficiently detected, in addition to solving the<br>
conventional problems associated with the Northern blotting<br>
method.<br>
^^<br><br>
The method by which target DNA is prepared for Southern<br>
blotting is not particularly limited, and genomic DNA is<br>
fragmented by known methods. Specifically, genomic DNA<br>
subjected to restriction enzyme is used as target DNA. In other<br>
words, RFLP analysis is performed with an array of the present<br>
invention.<br>
Digestion of genomic DNA with restriction enzymes<br>
produce probe DNA fragments of many different sizes as<br>
compared with using mRNA (cDNA). This can be a drawback<br>
where accurate detection of polymorphism, such as a length<br>
difference, for example, between 500 bp and 5 kbp is required<br>
on the array (detection sensitivity of imaging means for<br>
detecting image information of array is brought into question).<br>
In order to avoid such a problem, DNA fragments obtained<br>
by the treatment of genomic DNA with restriction enzymes are<br>
fractionated by size to be used as target DNA. In this way, a<br>
length difference can be effectively detected as a polymorphism,<br>
enabling an array of the present invention to be effectively used<br>
in the analysis employing the Southern blotting method.<br>
The method of size fractionation is not particularly limited,<br>
unci any technique can be used as long fis tlir inr(hod HUOWS<br>
Ihr ^ciinmic DNA treated with i rHliiitioti fiizviin'H 1
fractionated to required sizes. For example, ti coTnmerciidly<br>
available nucleic acid purification column kit usinf^ u<br>
centrifugal tube can be used. Further, size fractionation can be<br>
performed by setting PCR conditions such that DNA fragments<br>
of certain sizes are specifically amplified. The labeling method of<br>
genomic DNA is not particularly limited, and labeling can be<br>
made by a known method using PCR, for example.<br><fabrication method of array><br>
A fabrication method of array according to the present<br>
invention    at    least    includes    the     step    of    arranging    and<br>
/eO<br><br>
immobilizing on a support different kinds of biosubstances<br>
obtained from a living organism of interest, or synthetic<br>
substances interacting with such biosubstances. In the step,<br>
the biosubstances or synthetic substances immobilized on the<br>
support are arranged in the order genes of the organism are<br>
coded on the chromosome.<br>
When the biosubstances are nucleic acids as in the<br>
present embodiment, the step follows the following procedure,<br>
for example. After preparing genomic DNA, the genomic DNA is<br>
fragmented by restriction enzymes, and a solution of DNA<br>
fragments is spotted on the support using a spotter. Here, the<br>
DNA fragments are spotted with a spotter in such a manner<br>
that chromosome information of the corresponding genes can be<br>
identified, as described above.<br>
The spotter is not particularly limited and known<br>
instruments can be suitably used. Specifically, for example, an<br>
instrument that sputters a DNA solution onto a substrate<br>
through a capillary pen, or an ink jet device that plots a DNA<br>
solution on a substrate is available.<br>
In the case where the support is a collection of micro<br>
supports (beads) like a bead array {micro support group), DNA<br>
or other substances are individually immobilized on the beads,<br>
and sequence position information indicative of chromosomal<br>
locations of the immobilized DNA is added, together with an<br>
identification code, to each bead. The group of beads so<br>
obtained is dispersed in a known liquid to prepare a bead<br>
solution, which is then charged into a small container and used<br>
as a bead array.<br><use of an array according to the invention><br>
Use of an array according to the present invention is not<br>
particularly limited. For example, in the case of an array using<br>
DNA   as   biosubstances,   the   array   can   be   suitably   used   to<br>
h^<br><br>
icit-nlily chromosome fruj'mrnls Inrliidiii^i n itu^rt litiil<br>
(identification of genotype), from hybrids obtainrd by croMsin^<br>
living organisms. Further, the array can be suitably used to<br>
screen for a variety with a target trait, from hybrids obtained by<br>
crossing organisms for variety improvement.<br>
In conventional arrays, the DNA fragments immobilized on<br>
the support are randomly arranged. This enables the expression<br>
level or other profiles of the immobilized DNA fragments to be<br>
individually analyzed. In hybrids, individual genes are inherited<br>
in units of blocks, from a point of crossing over to the next<br>
point of crossing over, on the chromosomes. Therefore, for the<br>
genotype identification or selection in variety improvement, etc.,<br>
it is necessary to determine the location and extent of<br>
recombination and the presence or absence of unnecessary<br>
recombination, in addition to finding individual traits. Thus,<br>
conventional arrays with randomly arranged DNA fragments<br>
cannot be used efficiently for the screening in variety<br>
improvement, etc.<br>
On the other hand, in an array of the present invention,<br>
the DNA fragments (biosubstances) immobilized on the support<br>
are arranged in the chromosomal order. Thus, with an array of<br>
the present invention, the location of recombination on the<br>
chromosomes can be found, if any, with a single round of<br>
testing. This allows for accurate selection of individuals with<br>
desirable traits from a segregating population of hybrid<br>
individuals. Further, with an array according to the present<br>
invention, chromosomal recombinations in the hybrid<br>
generation can easily be estimated. This allows a group of genes<br>
to be introduced in units of blocks, or genes in the blocks to be<br>
modified.<br>
Further, with an array according to the present invention,<br>
the   recombination   patterns,   i.e.,   the   location   and   type   of<br>
i^<br><br>
recombination on the chromosomes can be accurately grasped.<br>
Thus, by identifying conserved regions of chromosomes where<br>
recombination frequency is smal] in the population of hybrids or<br>
natural population, recombination can be efficiently promoted<br>
only in these regions of the chromosomes.<br>
In an analysis using conventional arrays, the cause of<br>
signal failure at a particular spot, whether it is actually caused<br>
by unexpressed genes, or due to experimental error, cannot be<br>
accurately determined unless it is rechecked. In contrast, in an<br>
array according to the present invention, such an experimental<br>
error can easily be found because the chromosomal order of<br>
ONA fragments {biosubstanres) immohilizrd on tlir Hiipporl (-nn<br>
be rrcoynized from their arrangement.<br>
For example, consider the situation where signals are<br>
obtained from spots in front of and after the spot where signal<br>
failure has occurred. In an array according to the present<br>
invention, the spots are arranged in such a manner that their<br>
chromosomal order is recognizable. As a rule, in order for a<br>
gene flanked by another gene on the same chromosome to be<br>
recombined, two recombinations must occur in close proximity.<br>
Given a significantly low probability of such a phenomenon, the<br>
signal failure can be attributed to experimental error. Thus,<br>
with an array according to the present invention, whether signal<br>
failure that has occurred at a particular spot is due to<br>
experimental error or not can easily be found, with the result<br>
that analysis accuracy is improved.<br>
The following schematically describes an example of a<br>
screening method using an array according to the present<br>
invention. It is assumed here that a DNA micro array according<br>
to the present invention is fabricated using DNA fragments<br>
obtained from barley. In a DNA micro array according to the<br>
present invention, solid spots X in Fig. 2(a) indicate that genes<br>
Ui%<br><br>
that confer brewing characteristics are expressed, and hatched<br>
spots Y in Fig. 2(b) indicates expression of genes that confer<br>
disease-resistance.<br>
In a DNA micro array according to the present invention,<br>
the spots are arranged in a chromosomal order, and therefore<br>
the positions of spots X and Y are fixed, For example, in Fig.<br>
2(a), the spots X are fixed at the first, second, fifth, and sixth<br>
positions of the first row, and at the ninth and tenth positions<br>
of the bottom row. The spots Y are fixed at the third and fourth<br>
positions of the first row, as shown in Fig. 2(b),<br>
It is assumed here that segregating populations as<br>
represented by four micro arrays in the bottom of Fig. 3 were<br>
obtained from the cross between a variety expressing the<br>
brewing genes as indicated by spots X (corresponding to the<br>
upper left DNA micro array in Fig. 3) and a variety expressing<br>
the disease-resistant genes as indicated by spots Y<br>
(corresponding to the upper right DNA micro array in Fig. 3), for<br>
example. From the result of analysis using these DNA micro<br>
arrays, varieties expressing both the brewing genes and<br>
disease-resistant genes can be screened for from the segregating<br>
populations (variety corresponding to the upper left DNA miicro<br>
array circled by a dotted line in the lower porlioii of Fig. 3).<br>
I
from wliicli parent can easily be determincti MIHO for oilier'<br>
regions of the genome. Thus, a backcross, for example, hf.iwc.fin<br>
H hybrid and the variety shown in Fig. 2(a) eusily allows for<br>
selection and growth of varieties having all of the expressed<br>
spots as illustrated in Fig. 2(a), i.e., the first, second, fifth, and<br>
sixth spots of the first row, and the ninth and tenth spots of the<br>
bottom row, as well as the third and fourth spots of the first row<br>
as shown in Fig. 2(b).<br>
The  type  of organism  to which  an  array  of the  present<br>
k^<br><br>
invention is applicable is not particularly limited, and any of<br>
plants, animals, and microorganisms may be used. Particularly,<br>
an array of the present invention can be used in the foregoing<br>
screening method in organisms that include chromosomes and<br>
obey the laws of Mendelian genetics. Examples of such an<br>
organism are, but not limited to, those commercially available<br>
and for which need for variety improvement is high.<br>
In the case of plants, various crops (plant and farm<br>
products produced in agriculture, forestry, and fishery<br>
industries) can be used. Specific examples include: cereals such<br>
as rice, wheat, barley, rye, triticale, and corn; marine plants<br>
such as seaweed; various vegetables and flowers; and trees<br>
such as cedar or cypress. In the case of animals, various<br>
domestic animals can be used. Specific examples include:<br>
domestic mammals such as bovines, sheep, and pigs; domestic<br>
birds such as chickens and quails; fish such as yellowtail<br>
snapper, sea bream, carp, and sweetfish; insects such as honey<br>
bees, and silkworm; and shellfish such as oyster, ormer, and<br>
scallop. As microorganisms, bacteria such as Escherichia coli,<br>
yeasts, fungi, actinomycetes, and basidiomycetes can be used.<br>
Among these examples, the cereals include crops svtch as<br>
rice, wheat, corn, and barley, which are cultivated worldwide<br>
and are strategically important. Thus, by using the present<br>
invention for the variety improvement of these plants, varieties<br>
with desirable traits can be efficiently produced.<br>
An array according to the present invention can also be<br>
used for experimental animals and plants. Specific examples of<br>
experimental animals include mice, rats, D. melanog aster, and<br>
C. elegans. A specific example is Arabidopsis thaliana.<br>
Further, for the purpose of identifying genotypes with an<br>
array of the present invention, the invention can be applied to<br>
humans.   In  other words,  an  array according to  the  present<br>
k^<br><br>
invention can be preferably used for a gene diriKiinsis method,<br>
Hiiirp llif iirray allows for efricicnt idrniiricftlinii n( j'.r'MutvpcH,<br>
[Sf^cond Embodiment]<br>
Referring to Fig. 4, the following will descrilie another<br>
embodiment of an array according to the present invention. It<br>
should be appreciated that the invention is not limited by the<br>
following description.<br>
In the First Embodiment, the invention was described<br>
through the case where nucleic acids were used as the<br>
biosubstances. The present invention is not just limited to this<br>
example, and the biosubstances may be polypeptides.<br><polypeptides as biosubstances><br>
Polypeptides used in the present embodiment are not<br>
particularly limited as long as they are peptides of amino acids.<br>
Specific examples are proteins, fragments of proteins, and<br>
oligopeptides. As used herein, "fragments of protein" refers to<br>
polypeptides of partial amino acid sequences of a complete<br>
protein. The "oligopeptide" refers to an oligopeptide with a<br>
molecular weight of no more than 5000. The "protein" includes<br>
a protein complex forming multimers, as well as monomer<br>
proteins.<br>
In an array according to the present embodiment, as in<br>
the First Embodiment, the polypeptides immobilized on a<br>
support are arranged in the order respective base sequence<br>
blocks of the polypeptides are sequenced on a chromosome.<br>
For example, it is assumed here, as in the example of Fig.<br>
1, that an array is fabricated for an organism Z based on an<br>
organism Z chromosome in which 10 genes ABCl through<br>
ABC 10 are present that are lined up in this order on the<br>
chromosome, as schematically illustrated in Fig. 4. It is also<br>
assumed that 10 kinds of proteins are respectively transcribed<br>
and    translated    from   the   genes   ABCl    through   ABC 10   (as<br>
k^<br><br>
indicated by arrows). In this case, an array is fabricated by<br>
spotting these proteins on a substrate in the chromosomal<br>
order.<br>
The "base sequence blocks" may be regions corresponding<br>
to genes encoding the proteins, or regions corresponding to only<br>
polypeptides as fragments of protein.<br>
The type of protein used as the biosubstance is not<br>
particularly limited. For example, enzymes, kinase, antibodies,<br>
and proteins with an SH3 region may be used. It is preferable<br>
that the proteins, as with the DNA in the First Embodiment, be<br>
arranged based on a genetic map or physical map.<br><types of arrays used in the embodiment><br>
The type of array used in the prrHcnt rnilmdimrni IN not<br>
pHrticularly limited as long as the polypeptldcH hiuni)bili/-cd on<br>
the support are arranged in the order respective bawe sequence<br>
blocks of the polypeptides are sequenced on the chromosome.<br>
Specifically, for example, a peptide array, a kinase array, an<br>
enzyme array, an SH3 domain array, and a receptor array may<br>
be used, depending on the type of polypeptide immobilized on<br>
the support.<br>
In the peptide array, oligopeptides are immobilized on a<br>
support. The oligopeptides may be synthetic, or may be<br>
obtained by degrading or cutting proteins or other polypeptides<br>
by a known method.<br>
In the kinase array, different kinds of purified kinase<br>
proteins are arranged and immobilized on a support. By finding<br>
phosphorylation patterns exhibited by the kinase, the behaviors<br>
of proteins in the cell can be observed, or phosphorylation<br>
targets can be searched comprehensively.<br>
In the antibody array, different kinds of antibodies are<br>
arranged and immobilized on a support. The antibody array is<br>
also known as an antibody chip. By allowing the antibody array<br>
W)<br><br>
to bind to proteins, proteins that interact with the target<br>
antibodies can be detected.<br>
In the enzyme array, different kinds of enzymes are<br>
arranged and immobilized on a support. The enzyme array is<br>
used for the purpose of monitoring activities of different kinds<br>
of enzymes, for example.<br>
In the SH3 domain array, a group of proteins with an SH3<br>
region are arranged and immobilized on a support. A<br>
representative example is the array manufactured by Panoraics.<br>
The SH3 (Src Homology 3) domain is a relatively short<br>
conserved region that occurs in Src protein. Specifically, the<br>
SH3 domain has a beta-barrel structure of 50 to 70 amino acid<br>
residues with five to six anti-parallel beta strands packed<br>
together. The SH3 domain specifically binds to a target protein<br>
via a peptide region (SH3 ligand) with a common sequence of six<br>
to twelve residues. Two types of SH3 ligands are known, both of<br>
which contain prolines. The binding is made as the proline<br>
occupies a hydrophilic pocket.<br>
In humans, about 408 kinds of proteins with SH3<br>
domains are known. These proteins serve as mediators of<br>
various interactions, playing part in cell-cell communications,<br>
or signal transduction from a cell surface to the nucleus. Thus,<br>
with the SH3 domain array, it is possihif to recognize<br>
iiivnJvrim'iil of a specific- prowin in ji pju li" iiliu HJI^IIDI<br>
iransriuclion, or the number of proteins involved in a spct-iric<br>
reaction pathway.<br>
In tlie receptor array, receptor proteins ussocitited witli<br>
various cell responses are arranged and immobilized on a<br>
support. The receptors are not particularly limited and not<br>
necessarily limited to proteins as long as they can specifically<br>
recognize substances such as hormones, neurotransmitters, or<br>
foreign substances such as autacoid, or respond to physical or<br>
1^1<br><br>
chemical stimuli, when these substances or stimuli induce cell<br>
response. Generally, the receptors are proteins which are<br>
activated by specific substances or stimuli present in the cell<br>
membrane, organelle membrane, or cytoplasm.<br>
Some of the arrays described above are classified as<br>
so-called biological chips, which are particular type of protein<br>
chips.<br>
The protein chip is a small array (chip) on which various<br>
chemical properties suitable for protein analysis are spotted<br>
and immobilized. Depending on the purpose of analysis, the<br>
protein chip is broadly classified as a chemical chip and a<br>
biological chip. The biological chip is used for the analysis of<br>
specific binding (interaction) of proteins or other polypeptides.<br>
As the substances immobilized on the chip, substances, for<br>
example, such as antibody, receptor, or DNA are used that can<br>
interact with polypeptides. Thus, in terms of purpose (use), the<br>
nucleic acid-immobilized array described in the First<br>
Embodiment can also be classified as a biological chip. As to<br>
the chemical chip, details will be described later.<br>
In some types of biological chips, a carbonyldiimidazole<br>
group or epoxy group is immobilized on the surface. These<br>
functional groups (compounds) can easily immobilize<br>
biosubstances such as an antibody, receptor protein, or DNA,<br>
allowing an array to be easily fabricated according to the<br>
purpose of analysis. In other words, an array according to the<br>
present invention may be adapted so that biosubstances (or<br>
synthetic substances) are immobilized either directly on a<br>
surface of a support, or, as in the biological chip, with an<br>
intervening ligand compound that can desirably bind to the<br>
support surface and the biosubstances.<br>
fThird Embodiment]<br>
Referring to Fig. 5 and Fig. 5, the following wilt describe<br>
1/^<br><br>
yet another embodiment of an array according to the present<br>
invention. It should be appreciated that the invention is not<br>
limited l)y the fallowing desirlplion.<br>
In the foregoing First and Second EmbodimentB, the<br>
present invention was described through the case whrrp nucleic<br>
acids and polypeptides are used as biosubstances, respectively.<br>
However, the invention is not limited to these examples, and<br>
synthetic substances that can interact with the biosubstances<br>
can also be used.<br><examples of synthetic substances><br>
The synthetic substance is not particularly limited as long<br>
as it can interact with the biosubstance. Specific examples are<br>
compounds with a protein-interacting group, which may be a<br>
hydrophobic group, cation-exchange group, anion-exchange<br>
group, metal ion immobilized group, or normal phase group.<br>
The synthetic substances also include synthetic<br>
oligonucleotides and synthetic oligopeptides.<br>
In an array of the present embodiment, as in the First and<br>
Second Embodiments, the synthetic substances immobilized on<br>
a support are arranged in the order respective base sequence<br>
blocks of the biosubstances interacting with the synthetic<br>
substances are sequenced on the chromosome.<br>
For example, as in Fig. 1 and Fig. 3, it is assumed here<br>
that an array is fabricated for an organism Z based on an<br>
organism Z chromosome in which 10 genes ABCl through<br>
ABC 10 are present that are lined up in this order on the<br>
chromosome, as schematically illustrated in Fig. 5. It is also<br>
assumed that 10 kinds of proteins are respectively transcribed<br>
and translated from the genes ABC 1 through ABC 10 (as<br>
indicated by arrows), and that the proteins specifically interact<br>
with a certain compound. In this case, an array is fabricated by<br>
spotting the compound in the order genes encoding the proteins<br>
STO<br><br>
interacting with the compound are sequenced on the<br>
chromosome.<br>
A specific example of an array using the synthetic<br>
substances is a chemical chip as one type of the protein chip<br>
described in the foregoing embodiment. The chemical chip is<br>
generally used for the expression analysis, purification, and<br>
identification of proteins, whereas the biological chip is used for<br>
the evaluation of specific binding (interaction) of the proteins,<br>
for example.<br>
As described above, the protein chips include chemical<br>
chip and biological chip. In the chemical chip, a functional<br>
group (compound) such as a hydrophobic group,<br>
cation-exchange group, anion-exchange group, metal ion<br>
iniintibilized group, or normal phase Kf'&gt;iip if* imiinil)ili/rd on a<br>
chip surface. As with common chromatogruphy, the chcnilcnl<br>
chip is used such that, when brought into contact with a<br>
sample under certain reaction conditions, the functional group<br>
can capture the proteins in the sample. The sample is not<br>
particularly limited as long as it contains (or may contain)<br>
proteins. Specific examples include biological samples such as<br>
serum, urine, spinal fluid, synovial fluid, saliva, and tissue<br>
homogenate; and culture samples such as cultured-cell<br>
supernatant or cultured-cell crushed solution.<br><protein chip system><br>
The method of analyzing the protein chip, including both<br>
the chemical chip and the biological chips described in the<br>
Second Embodiment, is not particularly limited. Generally, a<br>
protein chip system is used. The protein chip system is not<br>
limited to a particular structure, and is generally realized by a<br>
computer including; a protein chip, a protein chip reader used<br>
for measurement, and software for measurement and analysis.<br>
The protein chip system may also include other components as<br>
SI<br><br>
well.<br>
The protein chip reader is not particularly limited as long<br>
as it can read out data of expression analysis or interaction<br>
evaluation of proteins from the protein chip. Generally, a<br>
Time-of-Flight Mass Spectrometry (TOS-MS) is used for this<br>
purpose. In the TOS-MS, an ionized sample is allowed to fly<br>
through a highly evacuated column by applying kinetic energy<br>
of a constant acceleration voltage. The time of flight of the<br>
sample reaching a detector is then measured to analyze the<br>
mass of the sample. In this way, data of expression analysis or<br>
interaction evaluation of proteins or the like is read out from<br>
the protein chip.<br>
The sample used in the TOS-MS may be polypeptides such<br>
as proteins, or nucleic acids such as DNA. The method of<br>
ionizing the biosubstance sample is not particularly limited,<br>
and generally, a MALDI (Matrix Assisted Laser<br>
Desorption/Ionization) method is used. In this method, a<br>
sample immobilized on a metal plate (support) is ionized by<br>
irradiation of a laser beam. The TOS-MS using the MALDI<br>
method is called MALDI-TOF-MS.<br>
The following describes an exemplary analysis method<br>
using the protein chip system. First, one to several hundred<br>
micro liters of sample is spotted on a protein chip. Then, the<br>
surface of the protein chip is washed under predetermined<br>
(â (HidilioiiH, HO as to remove siibstfincrs lliiil tin imi iiiiffiicl wilii<br>
ihc suliHttuiees immobilizctl on thr pi'oUnn v.Uii&gt; wuilbcc. In UUH<br>
woy, proteins captured on the spot under specific conditions are<br>
selected. Each spot is then ionized by MALDI, and the molecular<br>
weight is measured by TOS-MS. The data obttiined from each<br>
spot is analyzed by a computer.<br>
The protein chip system allows a large number of samples<br>
to   be  analyzed   both   quickly  and  quantitatively  from   a  small<br>
y^<br><br>
amount of sample and based on the mass number, without<br>
using any label or tag. Further, the system allows for<br>
measurement of a trace component in a crude sample without<br>
pre-treatment. Further, residual salts on the spots can be easily<br>
removed before measurement is performed. The system is<br>
therefore suitable for the search of marker proteins of various<br>
diseases, or evaluation of toxicity, or for screening molecules<br>
(candidate substances for drugs) that interact with specific<br>
molecules.<br>
Here, the advantages described in the First Embodiment<br>
can be obtained if the synthetic substances or biosubstancs<br>
immobihzed on the protein chip are arranged in the order they<br>
are coded on the chromosome. In this case, the protein chip<br>
system can improve the level of analysis it performs, or can be<br>
used in more practical applications.<br><bead array system><br>
In order to efficiently identify proteins, the bead array as<br>
described in the First Embodiment can be used. In the bead<br>
array, as shown in Fig. 6, a plurality of beads (10 beads in Fig.<br>
6) respectively appended with ID codes are charged in a small<br>
container formed by a cell of a micro titer plate. On the surface<br>
of each bead, a probe such as a biosubstance or synthetic<br>
substance (antibody in this example) is immobilized.<br>
With the sequence position information appended to the<br>
beads, the type of protein (which of the 10 proteins transcribed<br>
and translated (as indicated by the arrows) from the genes<br>
ABCl through ABCIO as shown in Fig. 6) corresponding to the<br>
probe immobilized on the surface can be specified. With a<br>
two-wavelength laser beam, 100 kinds of beads may be<br>
quantified.<br>
This technique can be used for detection in a liquid phase,<br>
and   therefore   is   useful   for   quantification   of   proteins   in<br>
5J<br><br>
particular. A representative example of the bead array system is<br>
the fluorescent micro bead array system Luminex, the product<br>
of Hitachi Software Engineering Co., Ltd.<br>
The head array systrm is not Hmilrd in n iitirlii'iilnr<br>
structur-e, and is generally realized by a slriictiirr including; n<br>
plute with probes, an analyzer used for fluorescence detei-:tion,<br>
and a computer equipped with software for measurement and<br>
analysis. The micro bead array may include other elements avS<br>
well.<br>
The analyzer is not particularly limited as long as it can<br>
read out the result of expression analysis or interaction<br>
evaluation of the proteins, in some form of data, from the bead<br>
array. Generally, a device equipped with a flowmetry<br>
mechanism and fluorescence detection capability using a laser<br>
beam can be used. The device can distinguish tones of bead<br>
colors. Thus, by immobilizing different antibodies on the beads<br>
and allowing the antibodies to bind to labeled samples, the level<br>
of sam.ple binding can be measured for each bead by<br>
flowcytometry. The samples can then be quantified from these<br>
reactions by gathering several hundred samples for each type of<br>
bead.<br>
The sample used in the bead array system may be<br>
polypeptides such as proteins, or nucleic acids such as DNA.<br>
The bead array system allows a large number of samples to be<br>
analyzed in a liquid phase both quickly and quantitatively from<br>
a small amount of sample. Here, the advantages described in<br>
the First Embodiment can be obtained if the synthetic<br>
substances or biosubstances immobilized on the bead array are<br>
arranged in the order they are coded on the chromosome.<br>
It should be appreciated here that the invention is not<br>
just limited to the foregoing embodiments and various<br>
modifications are possible within the scope of the invention as<br>
^/f<br><br>
defined in the appended claims. Embodiments obtained by<br>
suitably combining different technical means as disclosed in the<br>
embodiments also fall within the scope of the present invention.<br>
Thus, even though the foregoing Third Embodiment was<br>
described through the case of bead array and protein chip as<br>
arrays according to the present invention, an array analyzing<br>
system such as the protein chip system is also applicable, for<br>
example, to the DNA micro array described in the First<br>
Embodiment.<br>
As described above, in an array according to the present<br>
invention, biosubstances, or synthetic substances that interact<br>
with the biosubstances, are analyzed by arranging these<br>
substances in the chromosomal order of the genes that encode<br>
the biosubstances. This enables the array to be used in more<br>
practical applications such as screening in variety improvement,<br>
in udditian Lo improving reliabiUty of array unMlyHlh.<br>
Note that, an array according to the present invention<br>
may be provided as a kit according to intended use. For<br>
example, in the case where the DNA array described in the First<br>
Embodiment is used for variety improvement, the array may be<br>
provided as a kit including reagents or instruments for<br>
preparing target DNA.<br>
[Fourth Embodiment]<br>
Referring to Fig. 7 through Fig. 9, the following will<br>
describe one embodiment of a genotype analyzing and display<br>
system according to the present invention. It should be<br>
appreciated that the invention is not limited by the following<br>
description.<br>
In a genotype analyzing and display system according to<br>
the present invention, analysis is made for hybrid individuals<br>
derived from the cross between individual A and individual B (A<br>
H B) of an arbitrarily selected species of living organism, using<br>
Sr<br><br>
the result of hybridization performed with the nucleic acid array.<br>
From the result of analysis, the system provides a graphical<br>
representation of locations of the chromosomes of the hybrid<br>
individuals where crossovers have occurred.<br>
A genotype analyzing and display system according to the<br>
present invention is not limited to a particular structure.<br>
Specifically, as shown in Fig. 7, a genotype analyzing and<br>
display system includes, for example, an image information<br>
processing section (image information processing means) 11, a<br>
genetic map constructing section (genetic map constructing<br>
means) 12, a genotype origin detecting section (genotype origin<br>
detecting means) 13, a display information constructing section<br>
(display information constructing means) 14, a control section<br>
(control means) 15, a memory (storage means) 16, a scanner<br>
(image reading means, input means} 21, an external<br>
communications section (external information input and output<br>
means) 22, a storage medium reading and writing section<br>
(memory means, input means, output means) 23, a manual<br>
input section (manual input means} 24, a printer (image<br>
forming means, printing means, output means) 25, and a<br>
display (image display means, output means) 26. The genotype<br>
analyzing and display system of such a structure can be<br>
roughly divided into an input section, an output section, and an<br>
analyzing section (analyzing means) 10,<br>
(I) Nucleic acid array<br>
â ^Specific structure of nucleic iicid Jirniv -<br>
'I'he invention analyzes and displayh a gcnolypf (if ii<br>
desired species of living organism based on I he resiili of<br>
analysis performed with a nucleic acid array on tlie expression<br>
level of genes of hybrid individiials derived from the cross<br>
between individual A and individual B (A n B}. The nucleic acid<br>
array used in the present invention is not particularly lirnited,<br>
BG<br><br>
and conventional nucleic acid arrays can be suitably used.<br>
Specific examples include a micro array, a macro array, and a<br>
bead array. In the present embodiment, DNA is used as the<br>
nucleic acid, and therefore more specific examples of the<br>
nucleic acid array are DNA arrays such as a DNA micro array<br>
and a DNA macro array.<br>
The DNA micro array is also known as a DNA chip, and<br>
the immobilized DNA is often referred to as a probe. The micro<br>
array is smaller in size than macro array and provides more<br>
density. This enables the number of genes (DNA fragments)<br>
immobilized as probes to be increased, allowing for more<br>
comprehensive gene expression analysis.<br>
The DNA micro array can be classified based on types of<br>
immobilized DNA. However, structural differences can be<br>
revealed more clearly if the DNA micro array is classified based<br>
on fabrication methods. Specifically, based on fabrication<br>
methods, the micro array can be broadly classified into the<br>
Stanford type and the Affymetrix type.<br>
A DNA micro array of the Stanford type is fabricated by<br>
spotting a DNA solution onto a substrate (support) with a<br>
spotter, wherein a slide glass for a microscope is used as the<br>
substrate. One advantage of a DNA micro array of a Stanford<br>
type is that it can always be fabricated with the use of a spotter.<br>
However, this comes with a drawback in that it requires<br>
expensive hardware (spotter, etc.), or complex procedures for<br>
the preparation of biosubstances as necessitated by a large<br>
number of probes required for spotting.<br>
On the other hand, a DNA micro array of the Affymetrix<br>
type does not employ the method of immobilizing DNA<br>
fragments on a substrate with a spotter, etc., but is fabricated<br>
by chemically synthesizing oligoDNA of about 25 mer on a<br>
substrate using a micro fabrication technique commonly used<br>
57<br><br>
in the fabrication of semiconductors, namely, a<br>
photolithography technique.<br>
Specifically, for each gene, 11 to 20 oligos (25 mers) (for<br>
example, 11 oligos in the case of a barley DNA array) are set<br>
Im.sfd on b/me sequence dala, and a pair r&gt;f olij^o DNA; our vvilb<br>
a pfrfcct match to each 25 mer, unci <riir with u></riir>
single-base mismatch at the I 3th base is used as a probe. The<br>
array can be fabricated without using a spotter or other devices<br>
when it is designed with data of a known database. Further,<br>
since the probe (DNA fragment) has a constant length and the<br>
sequence is known, the GC content, which influences the<br>
strength of hybridization, can remain constant. It should be<br>
noted, however, that since the probe is synthesized based on<br>
information of a database, clones to be analyzed need to be<br>
separately isolated.<br>
In the present invention, either the Stanford DNA micro<br>
array or the Affymetrix DNA micro array can be used as the<br>
nucleic acid array.<br>
The following describes an exemplary method of using the<br>
DNA micro array. First, the DNA micro array is hybridized with<br>
fluorescent-labeled target DNA (hereinafter, "targets"). Here, the<br>
target molecules containing complementary sequences to the<br>
probes on the DNA micro array bind to (hybridize with) their<br>
complementary probe molecules, leaving other target molecules<br>
unbound. Then, these target molecules not bound to the probes<br>
are washed and removed, leaving only the hybridized target<br>
molecules on the micro array. Since the target molecules are<br>
fluorescence-labeled, the fluorescence of the targets is<br>
measured as signal intensity and hybridized probes are<br>
identified,<br>
The fluorescent-IabeJed targets are generally prepared first<br>
by extracting mRNA from cells of two different states [first state<br>
Â£?<br><br>
and second state) to be compared, and then performing a<br>
reverse transcription reaction in the presence of fluorescent<br>
nucleotides. Here, two kinds of fluorescent dyes with different<br>
detection wavelengths are used for the first state and second<br>
state, respectively. The expression level of genes is greater for<br>
the cDNA contained in the targets, and the fluorescent signal<br>
intensity is in accord with the expression level of genes in each<br>
state. Thus, from the measured signal intensity, the expression<br>
level of a specific gene can be detected.<br>
The DNA macro array basically has the same structure as<br>
the DNA micro array, but differs from the DNA micro array in<br>
that it uses a common membrane filter like a nylon membrane.<br>
An advantage of the macro array is that it allows for an<br>
expression profile analysis, genome wide, according to methods<br>
based on conventional blotting methods. Another advantage is<br>
that, unlike the micro array, the DNA fiorÂ» nn) detach in<br>
wushiny, owning to the fad thut the Npotted DNA IK itiinicihlll/.fd<br>
on a membrane filter after denatured by an ulkuli treatment.<br>
Therefore, the macro array and micro array should be suitably<br>
selected according to use.<br>
The following describes an exemplary method of using the<br>
macro array. The macro array is used basically in the same way<br>
as the micro array. Specifically, the macro array is hybridized<br>
with isotope (^^P, etc.)-labeled targets. Then, target molecules<br>
that did not bind to the array are washed and removed, leaving<br>
only hybridized target molecules on the macro array. Here,<br>
since the target molecules are isotope-labeled, the spots are<br>
exposed on an imaging plate and the expression level of the<br>
targets is determined by measuring signal intensity from the<br>
imaging plateâa procedure not performed in the micro array.<br>
The techniques of the present invention can also be<br>
applied  to   the  mass  array.  In  the  mass  array,   genomic  DNA<br>
Sf<br><br>
fragments are arranged and immobilized in an orderly manner<br>
on a silicon substrate, and therefore the structure is basically<br>
the same as that of the micro array. The mass array was<br>
developed for SNP analysis, and as such it is used differently<br>
from the DNA micro array.<br>
Specifically, oligonucleotides corresponding to regions in<br>
the vicinity of target SNP are synthesized and hybridized with<br>
the mass array. Then, by using the oligonucleotides as primers,<br>
a DNA fragment having a SNP single base difference is<br>
synthesized through elongation catalyzed by DNA polymerase.<br>
The DNA fragment is eluted and then ionized with MALDL The<br>
SNP type can be determined by detecting a single base mass<br>
difference using TOS-MS.<br>
The bead array can also be used as the nucleic acid array<br>
of the present invention. The bead array is used in such a<br>
manner that, in a small container, a probe such as a nucleic<br>
acid or antibody is immobilized on a surface of each bead to<br>
which an ID code has been added, and that the probe<br>
immobilized on the probe surface is specified by reading the ID<br>
code of the bead. With use of a two-wavelength laser beam, 100<br>
kinds of beads can be quantified. That is, in an array according<br>
to the present invention, the support may be a collection of<br>
micro arrays (beads, for example) on which biosubstances or<br>
synthetic substances are individually immobilized.<br><chromosomal location recognizable array><br>
The nucleic acid array used in the present invention is<br>
nios! prrft^rably the chromosoma) lofjiiinn rcrnjj))i/,;ib]f nrray<br>
descriticii in tlie First I'^nihodtnu'iil, In (he i in'oriinmnniil<br>
localion recognizable array, the probes (a plurnlily of nucleic<br>
acid molecules) immobilized on a support are arranged In such<br>
an orderly manner that the chromosomal ordrr of respertivf<br>
base    sequence    blocks    of    the    nucleic    acid    molecuies    is<br>
60<br><br>
recognizable. In the most typical chromosomal location<br>
recognizable array, the nucleic acid molecules (probes) are<br>
arranged in the chromosomal order of respective sequence<br>
blocks of the base sequences of the probes. That is, the probes<br>
are arranged in the chromosomal order. With the probes<br>
arranged in the chromosomal order, the chromosomal location<br>
recognizable array provided as a nucleic acid array is usable as<br>
a micro array, a macro array, a mass array, and the like.<br>
In the bead array, the beads immobilizing the probes are<br>
appended with sequence position information indicative of the<br>
chromosomal order of base sequence blocks corresponding to<br>
the base sequences of the probes. In use, the analysis result<br>
(result of hybridization) is acquired and the sequence position<br>
information is read out, so that the order obtained from the<br>
analysis result is rearranged in the chromosomal order.<br>
The nucleic acid molecules immobilized on the nucleic<br>
acid array may be DNA or RNA, and DNA is generally used as<br>
described above. The type of DNA used as the probes<br>
immobilized on the DNA array (nucleic acid array) is not<br>
particularly limited, but a genetic marker, genomic DNA,<br>
genomic DNA treated with restriction enzyme, cDNA, EST, and<br>
synthetic oligoDNA are preferably used, for example. It is<br>
preferable that the DNA be arranged based on a genetic map or<br>
physical map. For example, for a group of different kinds of<br>
genetic markers, it is preferable that these genetic markers<br>
make up a genetic map. Based on the genetic map, the DNA<br>
fragments can be arranged on a substrate.<br>
The genetic marker or a group of genetic markers are not<br>
particularly limited as long as they can serve as genetic labels<br>
on the chromosome. Non-limiting examples include an EST<br>
marker using EST, a SNP marker including SNP (Single<br>
Nucleotide Polymorphism), a RFLP (Restriction Fragment Length<br>
a<br><br>
Polymorphism) marker, and a micro satellite marker (SSR<br>
(simple sequence repeat) marker). Thus, the genetic marker or a<br>
group of genetic markers include genomic Dl^fA treated with<br>
restriction enzyme, EST, synthetic oligoDNA, and the like, if<br>
they can he used as markers.<br>
Thf number of DNA (probes) immohilizrd oti tlir support<br>
is imt juirticularly limited, mul it in grnrriijly nii Ihr ordrr ol<br>
several thousand (10'). The number of immobilized (or<br>
arranged) DNA varies greatly depending on the type of device,<br>
such as a spotter, used for the fabrication of the array, or the<br>
area of the support (substrate), for example.<br>
It should be noted that, in the DNA array, information<br>
concerning gene expression can only be obtained for genes<br>
corresponding to the immobilized DNA fragments. It is therefore<br>
preferable to increase the number of immobilized DNA (probes)<br>
as much as possible, in order to perform gene expression<br>
analysis more systematically and comprehensively.<br>
(11) Structure of genotype analyzing and display system<br><input section><br>
The invention analyzes and displays a genotype of a<br>
desired species of living organism based on the result of<br>
analysis performed with an nucleic acid array on the expression<br>
level of genes of hybrid individuals derived from the cross<br>
between individual A and individual B (A n B). To this end, a<br>
genotype analyzing and display system according to the present<br>
invention includes means (input section) for inputting the result<br>
of hybridization analysis performed with a nucleic acid array on<br>
the expression level of genes, i.e., comprehensive information<br>
concerning expression level of genes of the hybrid individuals<br>
being analyzed.<br>
In the structure shown in Fig. 7, the analysis result from<br>
the nucleic acid array is entered as image information through<br>
S'ii'<br><br>
the scanner 21. The image information processing section 11<br>
analyzes the image information and generates expression level<br>
information of genes from the analysis result. The scanner 21 is<br>
not particularly limited as long as it can serve as image reading<br>
means for reading the hybridization result as image information.<br>
Specifically, the fluorescence of the targets that hybridized with<br>
the probes is read out as image data from the nucleic acid array,<br>
and the expression level of genes is detected from the signal<br>
intensity of the image data. Thus, as the scanner 21, a<br>
conventional fluorescent scanner 21 can be suitably used, for<br>
example.<br>
Here, the image information obtained from the scanner 21<br>
is subjected to necessary information processing to generate<br>
gene-expression-level information. Thus, as shown in Fig. 7,<br>
the present invention preferably includes the image<br>
information processing section 11 for analyzing the expression<br>
level of genes based on the image information, nnd generating<br>
cnmprrliensive gene-expresHion-Irvrl Infornutlldii. Tlir iniuKP<br>
information processing section 11 is not Hmilcd to a particular<br>
structure, and conventional gene expression analyzing systems<br>
can be used.<br>
Further, in the present invention, the result of<br>
hybridization analysis from the nucleic acid array<br>
(gene-expression-level information of the hybrid individuals) is<br>
compared with the genetic information of the parents of the<br>
hybrid individuals, and with the genetic map of the species to<br>
which these individuals belong, as will be described later. To<br>
this end, a genotype analyzing and display system according to<br>
the present invention includes, in addition to the scanner 21<br>
(image reading means), means for inputting genetic<br>
information of the parents, and information of the genetic map.<br>
The   means   for   inputting   genetic   information   of   the<br>
63<br><br>
parents is not particularly limited. In the structure shown in<br>
Fig. 7, the external communications section 22, the storage<br>
medium reading and writing section 23, and the manual input<br>
section 24 correspond to such means. Generally, the genetic<br>
information of individuals of the parental generation is well<br>
known, and therefore genetic information of the parents may<br>
be obtained from database via networks with the external<br>
communications section 22, or genetic information stored in<br>
various storage media may be read out with the storage<br>
medium reading and writing section 23. Further, if the<br>
information can be manually entered, it may be entered<br>
through the manual input section 24.<br>
The external communications section 22 is not<br>
particularly limited as long as it allows for input and output of<br>
information to and from external devices, and conventional<br>
communications interfaces such as a LAN card, a LAN board, a<br>
LAN adapter, and a modem can be used. The storage medium<br>
reading and writing section 23 is not limited to a particular<br>
structure either. For example, known disk drives such as a<br>
hard disk drive, a flexible disk drive, a CD-ROM drive, and a<br>
DVD-ROM drive, or various memory cards or memory<br>
cartridges such as USB memory can be used. The manual<br>
input section 24 is not limited to a particular structure, and<br>
conventional input means such as a keyboard or a tablet can<br>
be suitably used.<br>
The genetic information of parents is not particularly<br>
limited, and may be genotypes of parents, or gene expression<br>
prnfile information, for example. Amonf?, tln-Ne examples,<br>
^^riidlyiie information of piirrnts IH iiioir liltfly In I"' Uimwii. In<br>
particular, genotypes of organisms usee! for cxprrimenl, or<br>
genotypes of important species in crops, domestic nnimals, or<br>
the like are widely known and some are available as a dataliase.<br>
dlf<br><br>
Thus, genotype information can be suitably used as genetic<br>
information of parents.<br>
The gene expression profile information is obtained<br>
through a comprehensive analysis of gene expression in the<br>
cell. Under ordinary or specific conditions, the expression<br>
pattern of genes may vary depending on the genotypes of<br>
individuals. Thus, the gene expression profile information can<br>
be used as genetic information of parents, instead of the<br>
genotype information. Further, the gene expression profile<br>
information may be used together with the genotype<br>
information, so as to enhance genetic information of parents.<br>
The means for entering genetic map information is not<br>
particularly limited, and the means for entering genetic<br>
information of parents may be used therefor. This is because a<br>
relatively large number of genetic maps are available as<br>
complete chromosome maps, as with the genetic information of<br>
parents. However, given that fact that sufficient genetic maps<br>
are not available for the majority of agricultural crops or<br>
domestic animals, the invention may include, for example, the<br>
genetic map constructing section 12 for constructing a<br>
necessary genetic map based on genetic map constructing<br>
information, as shown in Fig. 7, The genetic map constructing<br>
section 12 will be described in more detail later.<br>
Note that, the comprehensive gene-expression-level<br>
information of the hybrid individuals may be entered without<br>
being mediated by the scanner 21 or image information<br>
constructing section 11. For example, gene-expression-level<br>
information having been entered and analyzed through the<br>
scanner 21 or other gene expression analyzing systems may be<br>
entered through the external communications section 22 or<br>
storage medium reading and writing section 23,<br>
Further,  it is  preferable that a genotype analyzing and<br>
^r<br><br>
display system according to the present invention include<br>
means for correcting at least one of the comprehensive<br>
gene-expression-level information of hybrid individuals, genetic<br>
information of parents, and genetic map constructing<br>
information. Specifically, the manual input section 24 shown<br>
in Fig. 7 corresponds to such means.<br>
As will be described later, a genolypr iinnlyzinR and<br>
di.spliiy system according In llie juTNnit iiivi-nlloji prjfnrmH iin<br>
analysing steps in which an entry error is found, if any, wlien<br>
creating a genetic map or display information. In this way,<br>
reliability of final display information can be improved, It is<br>
therefore preferable that means be provided for correcting an<br>
entry error. Specifically, the manual input section 24 can be<br>
used to for this purpose. The means for correcting an entry<br>
error is not just limited to the manual input section 24, and<br>
other means may be used as well.<br><analyzing section><br>
The present invention analyzes and displays genotypes of<br>
hybrid individuals of the cross between individual A and<br>
individual B (A Bl B) in a desired species of living organism,<br>
using the result of analysis performed on the expression level<br>
of genes with a nucleic acid array. Thus, the analyzing section<br>
10 for analyzing the information entered through the input<br>
section is an essential component. The analyzing section 10 at<br>
least includes the genotype origin detecting section 13 and the<br>
display information generating section 14, as shown in Fig. 7.<br>
In the genotype origin detecting section 13, the<br>
gene-expression-level information or polymorphism information<br>
generated in the image information processing section 11<br>
through the scanner 21, or the gene-expression-level<br>
information entered through the external communications<br>
section 22 or the storage medium reading and writing section<br>
a<br><br>
23, etc., is compared with the genetic information of parents or<br>
the genetic map, so as to determine whether the genotype of a<br>
hybrid individual of interest derived from which parent. This is<br>
not limited to a particular process, and the genotype origin<br>
detecting section 13 can suitably perform the process<br>
according to the selected procedure of crossing, or the type of<br>
species of living organisms used. For example, a genotype may<br>
be determined as being signal parental, hetero, or<br>
unrecognizable to yield the result.<br>
The determination is made using the genetic information<br>
of the parents (genotype information, expression profile<br>
information) and a genetic map, by comparing these<br>
information with the gene-expression-level information and<br>
polymorphism information. The polymorphism information may<br>
be of SNP or RFLP, for example. The polymorphism information<br>
is generally used as a genetic marker, and therefore the<br>
presence or absence of a genetic marker distinct to the<br>
genotype being compared may be used as a criterion of the<br>
(Idermination. It should be apprr('iiitr<i liowrvrr lliiil llir></i>
prescnl invention is not just limited to this example, and the<br>
polymorphism information is not necessarily required in the<br>
comparison as long as the determination yields effective<br>
results.<br>
The display information generating section ]4 gathers the<br>
results of determination obtained in the genotype origin<br>
detecting section 13, and displays genotypes as a whole on the<br>
chromosome basis based on the results of determination, so<br>
that the parental type of the genotypes can be individually<br>
recognized. The display information is not particularly limited<br>
as long as the origin of each genotype can be recognized on the<br>
chromosome basis. For example, it is preferable that the<br>
display information be generated with statistics. The statistics<br>
61<br><br>
included in the display information is not particularly limited,<br>
and may be, for example, at least one of, or preferably both of a<br>
recombination number and recombination frequency of<br>
individual chromosomes. With such statistics included in the<br>
display information, information concerning crossovers can be<br>
recognized comprehensively for each chromosome, in addition<br>
to recognizing the origin of individual genotypes.<br>
The display information is adapted so that the origin of<br>
each genotype can be recognized on the chromosome basis.<br>
Specifically, as shown in Fig. 8, the origin of a genotype of<br>
interest can be recognized by different display colors or display<br>
patterns. In the example illustrated in Fig. 8, the stripe region<br>
indicates paternal origin (one of the parents), the dotted region<br>
indicates maternal origin (the other parent), and solid region<br>
indicate hetero origin. The blank region means that the origin<br>
is unrecognizable. The display is not limited to such display<br>
patterns, and different colors may be displayed to make<br>
distinctions.<br>
A map of genotypes of different individuals of a given<br>
species of living organism is generally referred to as a graphical<br>
genotype. From the graphical genotype, whether a particular<br>
individual includes a particular trait (locus) can be found if a<br>
marker linked to the trait is available. Thus, the present<br>
invention can be thought of as a technique of displaying<br>
genotypes of parents as graphical genotypes, based on the<br>
result of hybridization analysis performed on the expression<br>
level of genes with a nucleic acid array. As such, for the<br>
display information generated in the present invention, a<br>
display method used in the graphical genotype* cnn be suitably<br>
A Ecnntype analyzing and display system iiccording lo llie<br>
present  invention  may include  the  genetic  map  constructing<br>
M<br><br>
section 12, in addition to the genotype origin detecting section<br>
13 and the display information generating section 14. The<br>
genetic map constructing section 12 constructs a genetic map<br>
of a species to which the hybrid individuals belong, based on<br>
genetic map constructing information. As described earlier, the<br>
genetic map is constructed for only some of the species. It is<br>
therefore preferable to provide the genetic map constructing<br>
section 12.<br>
The genetic map constructing section 12 is not<br>
particularly limited as long as the genetic map is constructed<br>
on the chromosome basis based on various genetic map<br>
constructing information. As the genetic map constructing<br>
information, at least names of genes and/or genetic markers<br>
known in the species being analyzed, and the chromosomal<br>
loci of the genes and/or genetic markers are used, for example.<br>
The means for entering the genetic map constructing<br>
information is not particularly limited, and various input<br>
sections, for example, such as the external communications<br>
section 22, the storage medium reading and writing section 23,<br>
and the manual input section 24 shown in Fig. 7 can be used.<br>
Further, with the chromosomal location recognizable<br>
array, a genetic map can be constructed through mapping of<br>
genetic markers with unknown locations. Specifically, in order<br>
to construct a genetic map, targets obtained from Mendelian<br>
segregation population of the species being analyzed are<br>
hybridized with the chromosomal location recognizable array.<br>
Then, genetic markers with unknown locations are hybridi2:ed<br>
on the same chromosomal location recognizable array, so as to<br>
determine locations of the genetic markers. In this way, a high<br>
density genetic map can be constructed,<br>
Even though the foregoing example uses the same<br>
chromosomal   location   recognizable   array,    the   method   of<br>
6"^<br><br>
mapping the genetic markers of unknown location is not just<br>
limited to this example. For example, mapping can be made by<br>
processing the same targets with the genetic markers on<br>
different arrays. Here, mapping of genes is possible if the genes<br>
follow the rule of Mendelian segregation as in Single Feature<br>
Polymorphism (SFP), even if SNP or RFLP is not detected.<br>
Thus, a genotype analyzing and display system according<br>
to the present invention may be adapted HO thnt, in order to<br>
constrti(:t a genetic map in the f^fiirtic nmp i (itiMlriuiliin<br>
section 12, the hybridization result is analyzed and processed<br>
by reading it from the array with the scanner 21 and the image<br>
information processing section 11, before analyzing genotypes.<br>
TO this end, the image information processing section 11 is<br>
adapted to output information also to the genetic map<br>
constructing section 12, as shown in Fig. 7 (as indicated by<br>
arrow in the figure).<br>
Information such as the genetic map constructed by the<br>
genetic map constructing section 12, or the result of<br>
determination made by the genotype origin detecting section 13<br>
can be temporarily stored in the memory 16. The memory 15 is<br>
provided in the analyzing section 10 as shown in Fig. 7, and<br>
serves as a storage section for storing various information used<br>
or generated in a genotype analyzing and display system<br>
according to the present invention. The memory operation of<br>
the memory 15 is controlled by the control section 15. The<br>
memory 16 is not limited to a particular structure, and may be<br>
realized, for example, by a semiconductor memory, such as<br>
RAM or ROM. Note that, the storage medium reading and<br>
writing section 23 described as an input section can be used<br>
as a storage section of the present invention. This will be<br>
described later in more detail In conjunction with the output<br>
section.<br>
10<br><br>
The analyzing section of the structure shown in Fig. 7<br>
includes the control section 15 for controlling the entire<br>
operation of the analyzing section 10, and in turn the entire<br>
operation of the genotype analyzing and display system. In the<br>
structure shown in Fig. 7, the control section 15 outputs<br>
control information to the image information processing<br>
section 11, the genetic map constructing section 12, the<br>
genotype origin detecting section 13, the display information<br>
generating section 14, and the memory 16. These means<br>
operate based on the control information they receive, thereby<br>
operating the genotype analyzing and display system. It should<br>
be noted here that the control section 15 is also adapted to<br>
receive information from these means, and as such the flow of<br>
control information is indicated by the bidirectional arrow in<br>
Fig. 7.<br><output section><br>
The present invention analyzes and displays genotypes of<br>
hybrid individuals of the cross between individual A and<br>
individual B (A H B} in a desired species of living organism,<br>
iming thr result of analysis perforinrd (in tlir rxprrHHJdn Irvrl<br>
of genes with a nucleic acid array. To this end, a genotype<br>
analyzing and display system according to the present<br>
invention includes means, provided as an output section, for<br>
outputting display information.<br>
The output section for outputting the display information<br>
is not particularly limited, and at least one of, or preferably<br>
both of a display 26 for displaying display information on a<br>
display screen (soft copy), and a printer 25 for printing display<br>
information {hard copy) are provided. The display 26 is not<br>
limited to a particular structure, and various types of known<br>
displays such as a CRT, a liquid crystal display, and a plasma<br>
display   can   be   used.   The   printer   25   is   not   limited   to   a<br>
-^1<br><br>
particular structure, and known image forming devices such as<br>
an ink-jet printer and a laser printer can be used.<br>
It is preferable that the display 26 and the printer 25 are<br>
both adapted to output display information in colors. This<br>
enables the origin of a genotype to be displayed in different<br>
colors, thereby increasing display variations. Use of color is<br>
also preferable in displaying the origin of a genotype in the<br>
display patterns shown in Fig. 8, because colors offer graphical<br>
representations that are easier to read.<br>
The output section is not just limited to the display 26 or<br>
printer 25, and other means can be used as well. For example,<br>
the external communications section 22 can be used as an<br>
output section. Specifically, the external communications<br>
section 22 allows for input and output of information to and<br>
from external devices, by serving as both an input section and<br>
an output section. This enables display information to be<br>
transmitted to other devices via external networks, etc,<br>
enabling a genotype analyzing and display system according to<br>
the present invention to be used more efficiently.<br>
Specifically, when the genotype analyzing and display<br>
system is connected to externa) devices via LAN for example,<br>
the genotype analyzing and display system, installed in a<br>
research facility for example, can be shared with other<br>
researchers via information terminals such as personal<br>
computers. Further, the results of analysis obtained in the<br>
genotype analyzing and display system, may be accumulated in<br>
an external server via a communications network, allowing the<br>
analysis result to be used more efficiently.<br>
As the output section, the storage medium reading and<br>
writing section 23, described as an input Hfction, can be<br>
suitiilily used. Specifically, in a gciinly|K' i\\u\\\'/.\t\y_ MIKI (hft|iliiy<br>
sysirin according to the present invenlion, a chivr for rt-nding<br>
^^<br><br>
information from a storage medium can be used as an output<br>
section if the drive has a writing capability. The storage<br>
medium reading and writing section 23 is not limited to a<br>
particular structure, and known disk drives such as a hard<br>
disk drive, a flexible disk drive, a CD-ROM drive, and a<br>
DVD-ROM drive, or various memory cards or memory<br>
cartridges such as a USB memory can be suitably used for<br>
example, as described above in conjunction with the input<br>
section.<br>
Note that, in the exemplary structure shown in Fig. 7, the<br>
system is realized by the analyzing section 10 and<br>
independently provided input and output sections, wherein the<br>
analyzing section 10 includes the image information processing<br>
section 11, the genetic map constructing section 12, the<br>
genotype origin detecting section 13, the display information<br>
generating section 14, the control section 15, and the memory<br>
16. However, the present invention is not just limited to this<br>
structure. For example, all means may be provided as a single<br>
unit, or some of the input sections and/or output sections may<br>
be integrated with the analyzing section 10. Further, the<br>
system may include means other than those shown in Fig. 7.<br>
The analyzing section 10 is not just limited to a particular<br>
structure, and conventional arithmetic means, for exaniple,<br>
such as a central processing unit (CPU) of a computer may be<br>
used. The operation of the analyzing section 10 is executed by<br>
a computer program.<br>
(Ui) Analyzing method by the genotype analyzing and<br>
display system<br>
An analyzing method performed by a genotype analyzing<br>
and display system according to the present invention is not<br>
particularly limited. Specifically, the method may include 12<br>
steps as shown in Fig. 8.<br>
"13<br><br>
First, in step 101 (step will be denoted by "S" hereinafter),<br>
genetic map constructing information (names of chromosomes,<br>
genes, and genetic markers, and loci, etc.) is entered through<br>
input sections. In S102, the genetic map constructing section<br>
12 constructs a genetic map based on the genetic map<br>
constructing information, and the genetic map is supplied to<br>
the genotype origin detecting section 13. Here, the genetic map<br>
may be stored in the memory 16, or optionally displayed in the<br>
display 26. In S103, the presence or absence oT nn entry error<br>
IN found (need for correction \H detcnniiirdl. 1( thrtr IM MM cMliy<br>
error (YES), the genetic map constructing informfltitni is<br>
re-entered in SI04 through, for example, the manual input<br>
section 24, and the sequence returns to SlOl.<br>
On the other hand, if correction is not required (NO), the<br>
sequence goes to S105. In S105, genetic information of parents<br>
(genotypes of parents and/or gene expression profile<br>
information) is entered through input sections. In SI06,<br>
gene-expression-level information (DNA array analysis result)<br>
of hybrid individuals (target individuals in Fig. 9) being<br>
analyzed is entered through the scanner 21 and image<br>
information processing section 11. In S107, the genotype origin<br>
detecting section 13 compares the comprehensive<br>
gene-expression-level information of the hybrid individuals<br>
with the genetic information of the parents, and the genetic<br>
map of the species to which these individuals belong, so as to<br>
determine whether a genotype of a hybrid individual of interest<br>
derived from which parent. In SI08, it is decided whether the<br>
determination has been made for all necessary genotypes. If<br>
not (NO in Fig. 9), the sequence returns to S107 and the<br>
procedure is repeated.<br>
On the other hand, if the determination has been finished<br>
for all genotypes in  S108,  the display information generating<br>
iif<br><br>
section 14 in S109 gathers results of determination and, based<br>
on the results, generates display information, the display<br>
information being generated for displaying a plurality of<br>
genotypes together on the chromosome basis so that the<br>
parental type of individual genotypes can be recognized. Here,<br>
the display information may be stored in the memory 16, or<br>
optionally displayed in the display 26. Then, in SI 10, the<br>
presence or absence of an error in generating the display<br>
information is found (need for correction is determined). If<br>
there is an error (YES), correction information is entered in<br>
Sill through the manual input section 24, and the sequence<br>
returns to S107. If correction is not required (NO), the output<br>
section outputs the display information in S112. This<br>
completes the series of analysis procedures.<br>
(IV) Use of the present invention<br>
Use of a genotype analyzing and display system according<br>
to the present invention is not particularly limited, and the<br>
system is used for graphically displaying locations of<br>
crossovers that have occurred on the chromosomes of hybrid<br>
individuals of the cross between individual A and individual B<br>
(A n n) (if an arbitrary spenirB of living (irgruiiHiii, liiiHcd nti the<br>
result of analysis using the hybridization rcHiiltH ohlaincd wllh<br>
the use of a nucleic acid array.<br>
As a specific example, the invention can be suitably used<br>
for identifying a target trait-containing chromosome fragment<br>
from hybrids of organisms (identifying a genotype), or selecting<br>
a variety with a target trait from hybrids of organisms being<br>
crossed for variety improvenaent. With the present invention,<br>
the site of recombination on the chromosomes can easily be<br>
found. This allows for accurate selection of only those<br>
individuals with a target trait from a segregating population<br>
obtained   by  the  cross.   Further,   with   the   present  invention,<br>
^5-<br><br>
data of chromosomal recombinations in the hybrid generations<br>
can be accumulated, making it possible to readily estimate<br>
recombinations. This enables gene groups to be inserted in<br>
units of blocks, or genes in the blocks to be modified.<br>
The organisms to which the identification or screening of<br>
a genotype according to the present invention is applicable are<br>
not particularly limited, and may be any of plants, animals,<br>
and microorganisms. Particularly, the invention is applicable<br>
to organisms that have chromosomes and follow the laws of<br>
Mendelian genetics. The organisms that follow the laws of<br>
Mendelian genetics are not particularly limited, and those<br>
commercially available and for which need for variety<br>
improvement is high can be used.<br>
In the case of plants, various crops (plant and farm<br>
products produced in agriculture, forestry, and fishery<br>
industries) can be used. Specific examples include: cereals<br>
such as rice, wheat, barley, rye, triticale, and corn; marine<br>
plants such as seaweed; various vegetables and flowers; and<br>
trees such as cedar or cypress. In the case of animals, various<br>
domestic animals can be used. Specific examples include:<br>
domestic mammals such as bovines, sheep, and pigs; domestic<br>
birds such as chickens and quails; fish such as yellowtail<br>
snapper, sea bream, carp, and sweetfish; insects such as<br>
honey bees, and silkworm; and shellfish such as oyster, ormer,<br>
and scallop. As microorganisms, bacteria such as Escherichia<br>
coli, yeasts, fungi, actinomycetes, and basidiomycetes can be<br>
used.<br>
Among these examples, the cereals include crops such as<br>
rice, wheat, corn, and barley, which are cultivated worldwide<br>
and are strategically important. Thus, by using the present<br>
invention for the variety improvement of these plants, varieties<br>
with desirable traits can he rrfic-ieiillv prnclui fd.<br>
-76<br><br>
An array according to the present invention ctm QIHO be<br>
used for experimental animals and plants. Specific examples of<br>
experimental animals include mice, rats, D. melanogastef, and<br>
C. elegans. A specific example is Arabidopsis thaliana.<br>
Further, for the purpose of identifying genotypes by the<br>
present invention, the invention can be applied to humans.<br>
It should be appreciated that the present invention is not<br>
just limited to the foregoing embodiments. The foregoing<br>
examples are not intended to limit the invention to the<br>
particular forms disclosed, but on the contrary, the invention<br>
is to cover all modifications, equivalents, and alternatives<br>
falling within the scope of the invention as defined in the<br>
appended claims.<br>
As described, in a genotype analyzing and display system<br>
according to the present invention, comprehensive<br>
gene-expression-level information of hybrid individuals is<br>
compared with the genetic information of parents of the hybrid<br>
individuals, and a genetic map of the species to which these<br>
individuals belong, so as to determine and display the origin of<br>
the genotype of a hybrid individual of interest. Thus, for each<br>
chromosome of the hybrid individuals, locations of crossovers<br>
can be graphically displayed. That is, a genotype of a hybrid<br>
individual can be accurately determined or recognized only by<br>
acquiring nucleic acid from each individual of the hybrid<br>
generation and obtaining a hybridization result using the<br>
nucleic acid array.<br>
Thus, with the present invention, whether or not a<br>
genotype or trait of interest conferred by such a genotype has<br>
been inherited can be accurately determined for each<br>
individual of the hybrid generation. This enables individuals<br>
with a target trait to be selected from a large group of hybrid<br>
generation both easily and reliably and with good repeatability.<br>
"77<br><br>
That is, the invention allows gene expression  data obtained<br>
with the use of a nucleic acid array to be effectively used in<br>
crossing for variety improvement.<br>
[Fifth Embodiment]<br>
Referring to Fig. 10 and Fig. 11, the following will describe<br>
one embodiment of a quantitative loci analyzing system<br>
according to the present invention. It should be appreciated<br>
that the invention is not limited by the following description.<br>
A quantitative loci analyzing system according to the<br>
present invention examines various Menflrliun segregation<br>
popululionH, such as an F2 populnlinn, bnclfrnmN |ii&gt;|uilnl ion,<br>
and doubled haploid population, in regard (o u phenotypic<br>
value (for example, disease resistance quantified in scores) of<br>
each hybrid individual of these populations, and hybridizes<br>
extracted nucleic acid samples of the hybrid individuals with a<br>
nucleic acid array so as to analyze quantitative loci using the<br>
array spots as genetic markers.<br>
A quantitative loci analyzing system according to the<br>
present invention is not limited to a particular structure. For<br>
example, a quantitative loci analyzing system includes, as<br>
shown in Fig. 10, an image information processing section<br>
(image information processing means) 31, a genetic map<br>
constructing section (genetic map constructing means) 32, a<br>
genetic marker specifying section (genetic marker specifying<br>
means) 33, a quantitative loci detecting section (quantitative<br>
loci detecting means) 34, a control section (control means) 35, a<br>
memory (storage means) 36, a scanner (image reading means,<br>
input means) 21, an external communications section (external<br>
information input and output means) 22, a storage medium<br>
reading and writing section (storage means, input means,<br>
output means) 23, a manual input section (manual input<br>
means)   24,  a  printer  (image  forming  means,   printing  means,<br>
1^<br><br>
output means) 25, and a display (image display means, output<br>
means) 26. The quantitative loci analyzing system having such<br>
a structure can be roughly divided into an input section, an<br>
output section, and an analyzing section (analyzing means) 30.<br>
([) Nucleic acid array<br>
The invention prepares genomic samples from hybrid<br>
individuals obtained from each different Mendelian segregation<br>
population of a desired species of living organism, and<br>
hybridizes the genom.ic samples with a nucleic acid array so as<br>
to obtain comprehensive gene-presence-information of the<br>
hybrid individuals. The nucleic acid array used in the present<br>
invention is not particularly limited, and conventional nucleic<br>
acid arrays can be suitably used. Specific examples include a<br>
micro array, a macro array, and a bead array. In the present<br>
embodiment, DNA is used as the nucleic acid, and therefore<br>
more specific examples of the nucleic acid array are DNA arrays<br>
such as DNA micro array and DNA macro array.<br>
As to the specifics of the DNA micro array, DNA macro<br>
array, and other DNA arrays, no further description will be<br>
given since they were described in detail in the foregoing Fourth<br>
Embodiment. The chromosomal location recognizable array is<br>
preferably used also in the present embodimrnl.<br>
The following describes u metluKl of LIMJUK Ihc DNA urruy,<br>
taking the DNA micro array as an example. First, the DNA micro<br>
array is hybridized with fluorescent-labeled target DNA<br>
(hereinafter, "targets"). Here, the target molecules containing<br>
complementary sequences to the probes on the DNA micro array<br>
bind to (hybridize with) their complementary probe molecules,<br>
leaving other target molecules unbound. Then, these target<br>
molecules not bound to the probes are washed and removed,<br>
leaving only the hybridized target molecules on the micro array.<br>
Since    the    target    molecules    are    fluorescence-labeled,    the<br>
-^^i<br><br>
fluorescence oi the targets is measured as signal intensity, so<br>
as to identify hybridized probes.<br>
In	the	present	invention,	comprehensive<br>
gene-presence-information of hybrid individuals is obtained,<br>
and therefore the presence of individual genes can be<br>
recognized by the presence or absence of hybridization.<br>
Specifically, genomic DNA obtained from individuals are treated<br>
with restriction enzymes for example, and the resulting<br>
fragments are used as targets and hybridized with a DNA array<br>
so as to check for the presence or absence of complementary<br>
base sequences that hybridize with the probes on the DNA array.<br>
The resulting information is obtained as comprehensive<br>
gene-presence-information.<br>
The nucleic acid molecules immobilized on the nucleic<br>
acid array are not particularly limited and DNA is generally<br>
used, as described above. The type of DNA used as probes<br>
immobilized on the DNA array (nucleic acid array) is not<br>
particularly limited either, and a genetic marker or a group of<br>
genetic markers are used in the invention. The genetic marker<br>
or a group of genetic markers are arranged based on a genetic<br>
map or physical map.<br>
The genetic marker or a group of genetic markers are<br>
not particularly limited as long as they can serve as genetic<br>
labels on the chromosome. Non-limiting examples include an<br>
EST marker, a SNP marker including, a RFLP marker, and a<br>
micro satellite marker (SSR marker), as described above. Among<br>
these examples, a SNP marker and RFLP marker including<br>
polymorphism can be preferably used.<br>
(II) Structure of quantitative loci analyzing system<br><input sectidn><br>
In the present invention, genomic samples obtained from<br>
hybrid  individuals of each  different hybrid  line  are  hybridized<br>
^0<br><br>
witli a nuclric acid array on which Krnrtic iniirkriM of ii Nprciea<br>
have hren immobilized. With the rcsultiiiiLi cmiiprclieuHlvc<br>
gene-presence-information of the hybrid individuals, genetic<br>
markers are specified for each different hybrid line and QTL<br>
analysis is carried out. To this end, a quantitative loci analyzing<br>
system according to the present invention includes, as an input<br>
section,	means	for	entering	comprehensive<br>
gene-presence-information for specifying genetic markers.<br>
In the structure shown in Fig. 10, the analysis result from<br>
the nucleic acid array is entered as image information through<br>
the scanner 21. The image information processing section 31<br>
analyzes the image information and generates<br>
gene-presence-information from the analysis result. The<br>
scanner 21 is not particularly limited as long as it can serve as<br>
image reading means for reading the hybridization result as<br>
image information. Specifically, the fluorescence of the targets<br>
that hybridized with the probes is read out as image data from<br>
the nucleic acid array, and the expression level of genes is<br>
detected from the signal intensity of the image data. Thus, as<br>
the scanner 21, a conventional fluorescent scanner 21 can be<br>
suitably used, for example.<br>
Here, the image information obtained from the scanner 21<br>
is subjected to necessary information processing to generate<br>
gene-presence-information. Thus, as shown in Fig. 10, the<br>
present invention preferably includes the image information<br>
processing section 31 for analyzing the image information, and<br>
generating comprehensive gene-presence-information. The<br>
image information processing section 31 is not limited to a<br>
particular structure, and conventional analyzing systems can<br>
be used.<br>
Further, in the present invention, the result of<br>
hybridization     analysis     from     the      nucleic     acid     array<br>
1{<br><br>
(comprehensive gene-presence-information of the hybrid<br>
individuals) is compared with the genetic marker information<br>
of the hybrid individuals, and with the genetic map of the<br>
species to which these individuals belong, as will be described<br>
later. To this end, a quantitative loci analyzing system<br>
according to the present invention includes, in addition to the<br>
scanner 21 (image reading means), means for inputting at least<br>
one of the genetic marker information, and a phenotypic value<br>
representing a phenotype of interest. In addition, the<br>
quantitative loci analyzing system includes means for inputting<br>
at least one of a genetic map and genetic mnp constructing<br>
iiifnrmution.<br>
The means for inputting genetic mnrkcr informotion or<br>
phenotypic value is not particularly limited. In the structure<br>
shown in Fig. 10, the external communications section 22, the<br>
storage medium reading and writing section 23, and the<br>
manual input section 24 correspond to such means. These<br>
input means are also used for inputting a genetic map or<br>
genetic map constructing information.<br>
The external communications section 22 is not<br>
particularly limited as long as it allows for input and output of<br>
information to and from external devices, and conventional<br>
communications interfaces such as a LAN card, a LAN board, a<br>
LAN adapter, and a modem can be used. The storage medium<br>
reading and writing section 23 is not limited to a particular<br>
structure either. For example, known disk drives such as a<br>
hard disk drive, a flexible disk drive, a CD-ROM drive, and a<br>
DVD-ROM drive, or various memory cards or memory<br>
cartridges such as USB memory can be used. The manual<br>
input section 24 is not limited to a particular structure, and<br>
conventional input means such as a keyboard or a tablet can<br>
be suitably used.<br>
?^<br><br>
An example of the genetic marker information is position<br>
information immobilized on the nucleic acid array. Specifically,<br>
the hybridization detects spots if nucleic acid molecules having<br>
complementary base sequences are present. Thus, once the<br>
positions of immobilized spots on the nucleic acid array were<br>
found to correspond to which genetic markers, the information<br>
can be used as genetic marker information.<br>
The phenotypic value is not particularly limited as long<br>
as it represents a phenotype of interest. For example, the<br>
inventors of the present invention have evaluated resistance to<br>
Fusarium head bhght with the scores of 0 (resistance) to 10<br>
(diseased) by modifying a cut spike test (see Development of<br>
Fusarium head blight testing method, and a search for resistant<br>
varieties in barley, Japanese Journal of Variety Improvement,<br>
39, 1989, Kazuyoshi Takeda, Hideo Heta). In this manner,<br>
phenotypic values may be suitably selected depending on the<br>
type of species to be analyzed, or the type of desired trait.<br>
A relatively large number of genetic maps as chromosome<br>
maps are available for experimental animals and some of the<br>
crops and domestic animals. However, the selection of genetic<br>
maps is often not sufficient in most crops and domestic<br>
animals. Thus, the genetic map is directly entered if it is<br>
uvailalilr. If not, genetic map constnutitin inrortmitum is<br>
entered and a new genetic map \H cunHtrucird In the gcnrtlc<br>
map constructing section 32 shown in Fig. 10. Details of the<br>
genetic map constructing section 32 will be described later.<br>
A quantitative loci analyzing system according to the<br>
present invention preferably includes means for correcting at<br>
least one of the comprehensive gene-presence-information of<br>
hybrid individuals, genetic marker information, and genetic<br>
map constructing information. Specifically, the manual input<br>
section 24 in the structure shown in Fig.  10 corresponds to<br>
^S<br><br>
such means.<br>
As will be described later, a quantitative loci analyzing<br>
system according to the present invention performs the step of<br>
checking for the presence or absence of an entry error in the<br>
analysis process, particularly in specifying genetic markers.<br>
This improves reliability of final interval mapping performed in<br>
a subsequent stage. It is therefore preferable that the system<br>
include means for correcting entry error, i.e., the manual input<br>
section 24, for example. Note that, the means for correcting<br>
entry error is not just limited to the manual input section 24,<br>
and other means may be used as well.<br><analyzing section><br>
In the present invention, genomic samples obtained from<br>
hybrid individuals of each different hybrid line are hybridized<br>
with a nucleic acid array on which genetic markers of a species<br>
of interest have been immobilized. With the resulting<br>
comprehensive gene-presence-information of the hybrid<br>
individuals, genetic markers are specified for each different<br>
hybrid line and QTL analysis is carried out. To this end, a<br>
quantitative loci analyzing system according to the present<br>
invention includes, as an input section, means for entering<br>
comprehensive gene-presence-information for specifying<br>
genetic markers. Thus, the analyzing section 30 for analyzing<br>
the information entered through the input section is an<br>
essential component. The analyzing section 30 at least<br>
includes the genetic marker specifying section 33 and the<br>
quantitative loci detecting section 34, as shown in Fig. 10.<br>
In the genetic marker specifying section 33, the<br>
gene-presence-information generated in the image information<br>
processing section 31 through the scanner 21 is compared<br>
with the genetic map and genetic marker information, so as to<br>
specify    genetic    markers    for    each    different     hybrid    line.<br>
n<br><br>
Specifically, from the result of comparison between a genetic<br>
map and position information of thr K^"!""'"' mnrUrrH<br>
immobilized on the nucleic acid array, whethcj- oi MUL a hybrid<br>
individual of interest includes the genetic markers is<br>
determined. If the genetic markers are included, the genetic<br>
markers are specified as the genetic markers of the hybrid line<br>
to which the hybrid individuals belong.<br>
In obtaining the presence information of genes of the<br>
hybrid individuals, the use of the chromosomal location<br>
recognizable array as a nucleic array allows the order of<br>
immobilized spots to be used as the genetic marker<br>
information. In other words, the order of immobilized spots<br>
and the map distance (cM) of chromosomes, etc, can be used<br>
as genetic marker information. This is highly preferable as it<br>
makes it easier for the genetic marker specifying section 33 to<br>
perform the comparison. Further, as described above, the<br>
genetic marker information should preferably be polymorphic<br>
genetic markers (SNP or RFLP}, since it is easily recognizable<br>
as typical genetic markers of each different hybrid line.<br>
The quantitative loci detecting section 34 determines the<br>
quantitative loci of a phenotype of interest in the same hybrid<br>
individuals by checking whether a phenotypic value<br>
representing the phenotype is linked to the specific genetic<br>
markers. The quantitative loci are determined by interval<br>
mapping. The interval mapping is not particularly limited, and<br>
simple interval mapping (SIM), or composite interval mapping<br>
(CIM) may be used, for example. For a specific analysis of<br>
interval mapping, a known analyzing system may be used.<br>
Specific examples of such analyzing system include those<br>
using analyzing software such as MAPMARKER/QTL or QTL<br>
Cartographer.<br>
A   quantitative   loci   analyzing   system   according  to   the<br>
gs-<br><br>
present invention may include the genetic map constructing<br>
section 32 in addition to the genetic marker specifying section<br>
33 and the quantitative loci detecting section 34. Based on the<br>
genetic map constructing information, the genetic map<br>
constructing section 32 constructs a genetic map of a species<br>
to which the hybrid individuals belong. As described earlier,<br>
the genetic map is constructed for only some of the species. It<br>
is therefore preferable to provide the genetic map constructing<br>
section 32.<br>
The genetic map constructing section 32 is not<br>
particularly limited as long as the genetic map is constructed<br>
on the chromosome basis based on various genetic map<br>
c-onstnuMing information. AH thr RriiPtic iniip rntiHlniclinR<br>
information, at least names of genes and/or genetic markrra<br>
known in the species being analyzed, and chromosomal loci of<br>
the genes and/or genetic markers are used, for example.<br>
The means for entering the genetic map constructing<br>
information is not particularly limited, and various input<br>
sections, for example, such as the external communications<br>
section 22, the storage medium reading and writing section 23,<br>
and the manual input section 24 shown in Fig. 10 can be used.<br>
Further, with the chromosomal location recognizable<br>
array, a genetic map can be constructed through mapping of<br>
genetic markers with unknown locations. Specifically, in order<br>
to construct a genetic map, targets obtained from a Mendelian<br>
segregation population of the species being analyzed are<br>
hybridized with the chromosomal location recognizable array.<br>
Then, genetic markers with unknown locations are hybridized<br>
on the same chromosomal location recognizable array, so as to<br>
determine locations of the genetic markers. In this way, a<br>
highly dense genetic map can be constructed.<br>
Even    though    the    foregoing    example    uses    the    same<br>
'E'i<br><br>
chromosomaJ location recognizable array, the method of<br>
mapping the genetic markers of unknown locations is not just<br>
limited to this example. For example, mapping can be made by<br>
processing the same targets with the genetic markers on<br>
different arrays. Here, mapping of genes is possible if the genes<br>
follow the rule of Mendelian segregation as in Single Feature<br>
Polymorphism (SFP), even if SNP or RFLP is not detected.<br>
Thus, a quantitative loci analyzing system according to<br>
the present invention may be adapted so that, in order to<br>
construct a genetic map in the genetic map constructing<br>
section 32, the hybridization result is analyzed and processed<br>
by reading it from the array with the scanner 21 and the image<br>
information processing section 31, before analyzing genotypes.<br>
To this end, the image information processing section 31 is<br>
adapted to output information also to the genetic map<br>
constructing section 32, as shown in Fig. 10 (as indicated by<br>
arrow in the figure).<br>
Information such as the genetic map constructed by the<br>
genetic map constructing section 32, information concerning<br>
the genetic markers identified by the genetic marker specifying<br>
section 33, and the result of determination made by the<br>
genotype origin detecting section 34 can be temporarily stored<br>
in the memory 36. The memory 36 is provided in the analyzing<br>
aection 30 as shown in Fig. 10, and serves us n HUirnf^p Bpction<br>
for Mtoring various infoimution used or grnci ulcci in a<br>
quantitative loci analyzing system according to the present<br>
invention. The storage operation of the memory 36 is controlled<br>
by the control section 35. The memory 36 is not limited to a<br>
particular structure, and may be realized, for example, by a<br>
semiconductor memory, such as RAM or ROM. Note that, the<br>
storage medium reading and writing section 23 described as an<br>
input section can be used as a storage section of the present<br>
^â¢7<br><br>
invention.   This   will   be   described   later   in   more   detail   in<br>
conjunction with the output section.<br>
The analyzing section 30 of the structure shown in Fig. 10<br>
includes the control section 35 for controlling the entire<br>
operation of the analyzing section 30, and in turn the entire<br>
operation of the quantitative loci analyzing system. In the<br>
structure shown in Fig. 10, the control section 35 outputs<br>
control information to the image information processing<br>
section 31, the genetic map constructing section 32, the<br>
genetic marker specifying section 33, the display information<br>
generating section 34, and the memory 36. These means<br>
operate based on the control information they receive, thereby<br>
operating the quantitative loci analyzing system. It should be<br>
noted here that the control section 35 is also adapted to<br>
receive information from these means, and as such the flow of<br>
control information is indicated by the bidirectional arrow in<br>
Fig. 10.<br><output section><br>
In the present invention, a genomic sample of a hybrid<br>
individual obtained from each different hybrid line is<br>
hybridized with a nucleic acid array on which genetic markers<br>
of a species of interest are immobilized. In this way,<br>
comprehensive gene-presence-information of hybrid individuals<br>
are obtained, and by using the gene-presence-information,<br>
genetic markers are specified for each different hybrid line and<br>
QTL analysis is performed. To this end, a quantitative loci<br>
analyzing system according to the present invention includes<br>
means, provided as an output section, for outputting a result<br>
of QTL analysis.<br>
The output section is not particularly limited, and at least<br>
one of, or preferably both of a display 26 for displaying a result<br>
of QTL analysis on a display screen (soft copy), and a printer<br>
^-F<br><br>
25 for printing a result of QTL analysis (hard copy) are<br>
provided. The display 26 is not limited to a particular structure,<br>
Hiul various types of known displays such HH ii CRT, ii llrniid<br>
crystal display, and a plasma displtiy can be used. The prinU-f<br>
25 is not limited to a particular structure, and known image<br>
forming devices such as an ink-jet printer and laser printer<br>
can be used.<br>
The output section is not just limited to the display 26 or<br>
printer 25, and other means can be used as well. For example,<br>
the external communications section 22 can be used as an<br>
output section. Specifically, the external communications<br>
section 22 allows for input and output of information to and<br>
from external devices by serving as both an input section and<br>
an output section. This enables the result of QTL analysis to<br>
be transmitted to other devices via external networks, etc,<br>
enabling a quantitative loci analyzing system according to the<br>
present invention to be used more efficiently.<br>
Specifically, when the quantitative loci analyzing system is<br>
connected to external devices via LAN for example, the<br>
quantitative loci analyzing system, installed in a research<br>
facility for example, can be shared with other researchers via<br>
information terminals such as personal computers. Further,<br>
the results of analysis obtained in the quantitative loci<br>
analyzing system may be accumulated in an external server via<br>
a communications network, allowing the analysis result to be<br>
used more efficiently.<br>
As the output section, the storage medium reading and<br>
writing section 23, described as an input section, can be<br>
suitably used. Specifically, in a quantitative loci analyzing<br>
system according to the present invention, a drive for reading<br>
information from a storage medium can be used as an output<br>
section   if  the   drive   has   a   writing   capability.   The   storage<br>
?^<br><br>
medium reading and writing section 23 is not limited to a<br>
particular structure, and known disk drives such as a hard<br>
disk drive, a flexible disk drive, a CD-ROM drive, and a<br>
DVD-ROM drive, or various memory cards or memory<br>
cartridges such as a USB memory can be suitably used for<br>
example, as described above in conjunction with the input<br>
section.<br>
Note that, in the exemplary structure shown in Fig. 10,<br>
the system is realized by the analyzing section 30 and<br>
independently provided input and output sections, wherein the<br>
analyzing section 30 includes the image information processing<br>
section 31, the genetic map constructing section 32, the<br>
genetic marker detecting section 33, the quantitative loci<br>
{irtecting section 34, the control srction 35, nnd llir mrmory<br>
v36. However, the present invention in nol juÂ»t limited to Lhiw<br>
structure. For example, all means may be provided as a single<br>
unit, or some of the input sections and/or output sections may<br>
be integrated with the analyzing section 30. Further, the<br>
system may include means other than those shown in Fig. 10.<br>
The analyzing section 30 is not just limited to a particular<br>
structure, and conventional arithmetic means, for example,<br>
such as a central processing unit (CPU) of a computer may be<br>
used. The operation of the analyzing section 30 is executed by<br>
a computer program.<br>
(Ill) Analyzing method by the quantitative loci analyzing<br>
system<br>
An analyzing method performed by a quantitative loci<br>
analyzing system according to the present invention is not<br>
particularly limited. Specifically, the method may include 12<br>
steps as represented in Fig. 11.<br>
First, in step 201 (step will be denoted by "S" hereinafter),<br>
genetic map constructing information (names of chromosomes.<br>
^0<br><br>
genes, and genetic markers, and Joci, etc.) is entered through<br>
input sections. In S202, the genetic map constructing section<br>
32 constructs a genetic map based on the genetic map<br>
constructing information, and the genetic map is supplied to<br>
the genetic marker detecting section 33. In S203, the number<br>
of hybrid lines is entered through the input section. In S204,<br>
gene-presence-information (i.e., the result of DNA array<br>
analysis) of the hybrid individuals (targets individuals in Fig.<br>
11) being analyzed is entered through the scanner 21 and the<br>
image information processing section 31 for each different<br>
hybrid line. In S205, genetic marker information is entered.<br>
In S206, based on the entered information, the genetic<br>
marker detecting section 33 determines a genetic marker that<br>
is present in each different hybrid line. Here, the result of<br>
determination of a genetic marker may be stored in the<br>
memory 36, or optionally displayed in the display 25. In S207,<br>
the presence or absence of an entry error is found (need for<br>
correction is determined). If there is an entry error (YES), the<br>
information is re-entered in S208 through, for example, the<br>
manual input section 24, and the sequence returns to S201.<br>
On the other hand, if correction is not required (NO), the<br>
sequence goes to S209.<br>
In S209, a phenotypic value is entered through the input<br>
section. In S210, interval mapping (QTL analysis) is performed<br>
based on the result of determination of a genrlic marker and<br>
the phrnolypic value, so as lo detertninr n t|uiintilntiv(' IOCUH of<br>
the phenotype. In S211, based on the result of (^TL unulysis,<br>
the locations and functions of the genes associated with the<br>
quantitative traits are estimated. The result of analysis is<br>
outputted from the output section in S212. This completes the<br>
series of analysis procedures.<br>
fIV) Use of the present invention<br>
^ <br>
Use of a quantitative loci analyzing system according to<br>
the present invention is not particularly limited as long as QTL<br>
analysis is carried out on a species of interest. In one specific<br>
example, a search for a useful gene is made through an<br>
organism of interest, and such a useful gene is used for variety<br>
improvement. That is, in the present invention, the<br>
quantitative loci analyzing system can be suitably used in a<br>
quantitative trait analyzing method in which quantitative traits<br>
of organisms are analyzed, and in a gene search method in<br>
which genes associated with expression of traits of interest are<br>
searched.<br>
A QTL analysis is an analyzing method by which a genetic<br>
map distance between two loci is estimated based on<br>
recombination value. The genetic map distance refers to an<br>
expected value of a crossover between two loci that occurs in<br>
each round of meiosis. In the QTL analysis, a recombination<br>
value is estimated from the data of hybrid lines, and a genetic<br>
map distance is estimated from the recombination value.<br>
If it is assumed that crossing over occurs at equal<br>
probability anywhere on the chromosomes, then the genetic<br>
map distance is directly proportional to the physical distance.<br>
From this relationship, the physical distance can be estimated<br>
based on the genetic map distance. By thus finding the genetic<br>
map distance between a genetic marker of a known physical<br>
location and a gene associated with the expression of the trait<br>
of interest, the chromosomal location of the gene can be<br>
specified with fair accuracy.<br>
It should be noted, however, that the recombination value<br>
is equal to the genetic map distance only when the physical<br>
distance between two loci on the chromosome is close. That is,<br>
the recombination value does not indicate the genetic map<br>
distance when the two loci are far apart from each other. The<br>
'lil<br><br>
number of crossovers that might have occurred between two<br>
loci cannot be directly measured, and it is measurable only<br>
through recombination values. Thus, a genetic map distance is<br>
estimated from the recombination value, usin^ n Renetic map<br>
funrlion.<br>
A QTL analysis is generally performed in the following<br>
manner. First, genetically different two hybrid lines are crossed<br>
to produce hybrid generations Fl, F2 and subsequent<br>
generations. Then, typing of multiple genetic markers is<br>
performed for these individuals, and statistics are taken for the<br>
resulting data.<br>
As described thus far, the QTL analysis requires<br>
estimation of a genetic map distance, or processing of typing<br>
data of genetic markers, etc. The QTL analysis is therefore<br>
suitable for technical areas involving bioinformatics. To this<br>
date, no proposal has been made for applying QTL analysis to<br>
these technical areas. As described above, in the present<br>
invention, a genomic sample of hybrid individuals obtained<br>
from each different hybrid line is hybridized with a nucleic acid<br>
array on which genetic markers of the species of interest are<br>
immobilized, using a quantitative loci analyzing system. With<br>
the resulting comprehensive gene-presence-information of the<br>
hybrid individuals, genetic markers are specified for each<br>
different hybrid line and QTL analysis is carried out. In this<br>
way, information can be analyzed comprehensively in the QTL<br>
analysis, making it possible to efficiently perform the QTL<br>
analysis.<br>
Thus, a more specific application of the present invention<br>
is use of the quantitative loci analyzing system in variety<br>
improvement using genetic markers.<br>
The type of organism to which a quantitative trait<br>
analyzing method, gene search method, or variety improvement<br>
^3<br><br>
method of the present invention is applicable is not<br>
particularly limited, and any of plants, animals, and<br>
microorganisms may be used. Particularly, the present<br>
invention is applicable in organisms that include chromosomes<br>
and obey the lavys of MendeUan genetics. Examples of such an<br>
organism are, but not limited to, those commercially available<br>
and for which need for variety improvement is high.<br>
In the case of plants, various crops (plant and farm<br>
products produced in agriculture, forestry, and fishery<br>
industries) can be used. Specific examples include: cereals<br>
such as rice, wheat, barley, rye, triticale, and corn; marine<br>
plants such as seaweed; various vegetables and flowers; and<br>
trees such as cedar or cypress. In the case of animals, various<br>
domestic animals can be used. Specific examples include:<br>
domcMlic mammals such as bovinca, sherp, und piM"; domestic<br>
birds such as chickens and quailti; fish HLICII UB yclluwUiil<br>
snapper, sea bream, carp, and sweetfish; insects such as<br>
honey bees, and silkworm; and shellfish such as oyster, ormer,<br>
and scallop. As microorganisms, bacteria such as Escherichia<br>
coli, yeasts, fungi, actinomycetes, and basidiomycetes can be<br>
used.<br>
Among these examples, the cereals include crops such as<br>
rice, wheat, corn, and barley, which are cultivated worldwide<br>
and are strategically important. Thus, by using the present<br>
invention for the variety improvement of these plants, varieties<br>
with desirable traits can be efficiently produced. The invention<br>
is also applicable to experimental animals and plants. Specific<br>
examples of experimental animals include mice, rats, D.<br>
melanogaster, and C. elegans. A specific example is<br>
Arabidopsis thaliana. Further, for the purpose of identifying<br>
genotypes with the present invention, the invention can be<br>
applied to humans.<br>
tH<br><br>
It should be appreciated that the present invention is not<br>
just limited to the foregoing embodiments. The foregoing<br>
examples are not intended to limit the invention to the<br>
particular forms disclosed, but on the contrary, the invention<br>
is to cover all modifications, equivalents, and alternatives<br>
falling within the scope of the invention as defined in the<br>
appended claims.<br>
As described above, a quantitative loci analyzing system<br>
according to the present invention utilizes a hybridization<br>
analysis using a nucleic acid array for specifying genetic<br>
markers present in each hybrid line, and performs QTL<br>
analysis using the genetic markers so specified. In this way,<br>
information can be analyzed comprehensively in the QTL<br>
analysis, making it possible to efficiently perform QTL analysis.<br>
[Sixth Embodiment]<br>
Referring to Fig. 12 and Fig. 13, the following will describe<br>
another embodiment of the present invention. It should be<br>
appreciated that the invention is not limited by the following<br>
description.<br>
A gene interaction analyzing system according to the<br>
present invention individually performs interval mapping (QTL<br>
analysis) for specific genes or phenotypes based on the<br>
hybridization result of genetic markers immobilized on a<br>
nucleic acid array, and thereby estimates locations and<br>
functions of genes involved. In this way, hereditary factors of<br>
specific genes or phenotypes can be regulatrd, allowing for<br>
effirienl unulysis of gene interaction,<br>
A gene interaction analyzing system ficc^ording fo flic<br>
present invention is not limited to a particular structure. For<br>
example, a gene interaction analyzing system includes, as<br>
shown in Fig. 12, an image information processing section<br>
(image   information   processing   means)   41,    a   genetic   map<br>
R-S-<br><br>
constructing section (genetic map constructing means) 42, a<br>
genetic marker specifying section (genetic marker specifying<br>
means) 43, a spot marker information generating section (spot<br>
marker information generating means) 44, an expression profile<br>
information generating section (expression profile information<br>
generating means) 45, a hereditary factor regulating section<br>
(hereditary factor regulating means) 46, a control section<br>
(control means) 47, a memory (storage means) 48, a scanner<br>
(image reading means, input means) 21, an external<br>
communications section (external information input and output<br>
means) 22, a storage medium reading and writing section<br>
(storage means, input means, output means) 23, a manual<br>
input section (manual input means) 24, a printer (image<br>
forming means, printing means, output means) 25, and a<br>
display (image display means, output means) 25. The gene<br>
interaction analyzing system having such a structure can be<br>
roughly divided into an input section, an output section, and an<br>
analyzing section (analyzing means) 40.<br>
(I) Nucleic acid array<br>
The invention prepares genomic samples from hybrid<br>
individuals obtained from each different Mendelian segregation<br>
population of a desired species of living organism, and<br>
hybridizes the genomic samples with a nucleic acid array so as<br>
to obtain comprehensive gene-presence-information of the<br>
hybrid	individuals.	The	comprehensive<br>
gene-presence-information so obtained is used for the analysis<br>
of gene interaction. The nucleic acid array used in the present<br>
invention is not particularly limited, and conventional nucleic<br>
acid arrays can be suitably used. Specific examples include a<br>
micro array, a macro array, and a bead array. In the present<br>
embodiment, DNA is used as the nucleic acid, and therefore<br>
more specific examples of the nucleic acid array are DNA arrays<br>
'^i<br><br>
such as DNA micro array and DNA macro array.<br>
As to the specifics of the DNA arrays, no further<br>
description will be given since they are described in detail in the<br>
foregoing Fourth and Fifth Embodiments. The chromosomal<br>
location recognizable array is preferably used nlso in the<br>
present emVjodiment.<br>
The following describes a method of using tlie DNA array,<br>
taking the DNA micro array as an example. First, the DNA micro<br>
array is hybridized with fluorescent-labeled target DNA<br>
(hereinafter, "targets"). Here, the target molecules containing<br>
complementary sequences to the probes on the DNA micro array<br>
bind to (hybridize with) their complementary probe molecules,<br>
leaving other target molecules unbound. Then, these target<br>
molecules not bound to the probes are washed and removed,<br>
leaving only the hybridized target molecules on the micro array.<br>
Since the target molecules are fluorescence-labeled, the<br>
fluorescence of the targets is measured as signal intensity, so<br>
as to identify hybridized probes.<br>
In	the	present	invention,	comprehensive<br>
gene-presence-information of hybrid individuals is obtained,<br>
and therefore the presence of individual alleles can be<br>
recognized by the presence or absence of hybridization.<br>
Specifically, genomic DNA obtained from individuals are treated<br>
with restriction enzymes for example, and the resulting<br>
fragments are used as targets and hybridized with a DNA array<br>
so as to check for the presence or absence of complementary<br>
base sequences that hybridize with the probes on the DNA array.<br>
The resulting information is obtained as comprehensive<br>
gene-presence-information.<br>
(11) Structure of gene interaction analyzing system<br><input section><br>
In	the	present	invention,	comprehensive<br>
^7<br><br>
gene-presence-information of hybrid individuals is compared<br>
with a genetic map of the species to which the hybrid<br>
individuals belong, and with the genetic marker information<br>
known in the species, so as to specify genetic markers present<br>
in each hybrid line and thereby generate spot marker<br>
information (described later) based on the genetic markers so<br>
specified. Then, phenotj'pes and genes of interest to be analyzed<br>
are specified, and whether or not phenotypic values<br>
representing the phenotypes are linked to the expressed genes<br>
included in the expression profile information obtained from the<br>
same hybrid individuals is confirmed. To this end, a gene<br>
interaction analyzing system according to the present invention<br>
includes, as an input section, means for entering at least one of<br>
comprehensive gene-presence-information of hybrid individuals,<br>
the genetic marker information, the phenotypic values, and the<br>
expression profile information.<br>
The means for entering llir t(nni&gt;rrlipnHlve<br>
gene-presence-information is not particularly limited. For<br>
example, in the structure shown in Fig. 12, the analysis result<br>
from the nucleic acid array is entered as image information<br>
through the scanner 21. The image information processing<br>
section 41 analyzes the image information and generates<br>
gene-presence-information from the analysis result. In this<br>
manner, the scanner 21 can be used as the means for entering<br>
comprehensive gene-presence-information, for example.<br>
The scanner 21 is not particularly limited as long as it can<br>
serve as image reading means for reading the nucleic acid array<br>
hybridization result as image information. Specifically, the<br>
fluorescence of the targets that hybridized with the probes is<br>
read out as image data from the nucleic acid array, and the<br>
expression level of genes is detected from the signal intensity of<br>
the   image   data.   Thus,   as   the   scanner   21,   a   conventional<br>
1?<br><br>
fluorescent scanner 21 can be suitably used, for example.<br>
Here, the image information obtained from the scanner 21<br>
is subjected to necessary information processing to generate<br>
gene-presence-information. Thus, as shown in Fig. 12, the<br>
present invention preferably includes the image information<br>
processing section 41 for analyzing the image information, and<br>
generating comprehensive gene-presence-information. The<br>
image information processing section 41 is not limited to a<br>
particular structure, and conventional analyzing systems can<br>
be used.<br>
The means for entering the genetic marker information<br>
and phenotypic value is not particularly limited. In the<br>
structure shown in Fig. 12, for example, the external<br>
communications section 22, the storage medium reading and<br>
writing section 23, and the manual input section 24<br>
correspond to such means. The expression profile information<br>
can be generated by reading the result of nucleic acid array<br>
analysis, as will be described later. Alternatively, expression<br>
profile information that has been obtained by performing<br>
expression profile analysis beforehand may be entered as<br>
appropriate. Thus, the external communications section 22<br>
and the storage medium reading and writing section 23, etc.<br>
can also be used as the means for entering the expression<br>
profile information.<br>
Further, the present invention uses a genetic map for<br>
specifying genetic markers that exist in each hybrid line. To<br>
this end, it is preferable that means be provided for entering<br>
the griu'lic map or genetic map conHlrutliii^ iiifiH niMtinn, an<br>
well us the foregoing information. As the means for entering<br>
the genetic map or genetic map information, the external<br>
communications section 22, the storage medium reading and<br>
writing section 23,  and  the  manual input section  24  can  be<br>
^^<br><br>
suitably used, for example.<br>
The external communications section 22 is not<br>
particularly limited as long as it allows for input and output of<br>
information to and from external devices, and conventional<br>
communications interfaces such as a LAN card, a LAN board, a<br>
LAN adapter, and a modem can be used. The storage medium<br>
reading and writing section 23 is not limited to a particular<br>
structure either. For example, known disk drives such as a<br>
hard disk drive, a flexible disk drive, a CD-ROM drive, and a<br>
DVD-ROM drive, or various memory cards or memory<br>
cartridges such as USB memory can be used. The manual<br>
input section 24 is not limited to a particular structure, and<br>
conventional input means such as a keyboard or a tablet can<br>
be suitably used.<br>
An example of the genetic marker information is position<br>
information immobilized on the nucleic acid array. Specifically,<br>
hybridization detects spots if nucleic acid molecules having<br>
complementary base sequences are present. Thus, once the<br>
positions of immobilized spots on the nucleic acid array were<br>
found to correspond to which genetic markers, the information<br>
can be used as genetic marker information.<br>
The phenotypic value is not particularly limited as long as<br>
it represents a phenotype of interest. For example, the<br>
inventors of the present invention have evaluated resistance to<br>
Fusarium head blight with the scores of 0 (resistance) to 10<br>
(diseased) by modifying a cut spike test (see Development of<br>
Fusarium head blight testing method, and a search for resistant<br>
varieties in barley, Japanese Journal of Variety Improvement,<br>
39, 1989, Kazuyoshi Takeda, Hideo Heta). In this manner,<br>
phenotypic values may be suitably selected depending on the<br>
type of species to be analyzed, or the type of desired trait.<br>
A relatively large number of genetic maps as chromosome<br>
lOO<br><br>
maps are available for experimental animals and some of the<br>
crops and domestic animals. However, the selection of genetic<br>
maps is often not sufficient for most crops and domestic<br>
animals. Thus, the genetic map is directly entered if it is<br>
available. If not, genetic map constructing information is<br>
entered and a new genetic map is constructed in the genetic<br>
map cnnstructing section A2 shown hi Fi^, 1^. Drlnils of ihr<br>
genetic map constructing section 42 will be described later.<br>
The expression profile information is not particularly<br>
limited as long as it is obtained by comprehensively analyzing<br>
gene expression in the cell. As noted earlier, information that<br>
has been analyzed may be entered as the expression profile<br>
information; however, it is more preferable that expression<br>
profile analysis be performed as appropriate for the hybrid<br>
individuals being analyzed. Thus, the means for entering<br>
expression profile information can be used not only to enter<br>
analyzed information but to generate expression profile<br>
information by reading and analyzing the expression profile as<br>
appropriate.<br>
As the means for reading the expression profile, image<br>
reading means such as the scanner 21 can be used. The<br>
experiment system used for reading (experiment system for<br>
performing an expression profile experiment) is not particularly<br>
limited, and a nucleic acid array for obtaining comprehensive<br>
gene-presence-information may be used. Other than the<br>
nucleic acid array, other experiment systems can be used as<br>
well. Examples of the nucleic acid array include a m.icro array,<br>
a macro array, and a bead array, as described earlier. As an<br>
experiment system for performing an expression profile<br>
experiment, a differential display may be used, for example.<br>
A differential display is a technique whereby a difference<br>
in  gene  expression  level in the cells  placed under different<br>
01<br><br>
conditions is detected on a gel as a difference between band<br>
profiles, and the genes are collected and identified. Specifically,<br>
in the case of a fluorescent differential display for example, a<br>
PCR product of fluorescent-labeled cDNA is obtained from total<br>
RNA, and a fluorescent image is measured as signal intensity<br>
after separating the PCR product on a denatured<br>
polyacrylamide gel.<br>
The differential display is not a method for<br>
comprehensively analyzing total mRNA. However, since the<br>
differential display allows a large number of samples to be<br>
compared using a small amount of mRNA, it can be used to<br>
perform the expression profile experiment as with the nucleic<br>
acid array. Thus, as the input section, image reading means<br>
for detecting signal intensity from a gel plate of the<br>
electrophorased polyacrylamide gel is provided.<br>
A gene interaction analyzing system according to the<br>
present invention preferably includes meons for correcting at<br>
least one of the comprehensive griic-prrÂ»riH'r iiilfMriiHlidii ol'<br>
hybrid individuals, genetic marker information, and genetic<br>
map constructing information. Specifically, the manual input<br>
section 24 in the structure shown in Fig. 12 corresponds to<br>
such means.<br>
As will be described later, a gene interaction analyzing<br>
system according to the present invention performs the step of<br>
checking for the presence or absence of an entry error in the<br>
analysis process, particularly in specifying genetic markers.<br>
This improves reliability of final interval mapping performed in<br>
a subsequent stage. It is therefore preferable that the system<br>
include means for correcting entry error, i.e., the manual input<br>
section 24, for example. Note that, the means for correcting<br>
entry error is not just limited to the manual input section 24,<br>
and other means may be used as well.<br>
lOil.<br><br><analyzing section><br>
In the present invention, genetic markers are specified<br>
and spot marker information (described later) is generated<br>
from the genetic markers. Then, it is confirmed whether<br>
expressed genes in the expression profile information are<br>
linked to the spot marker information, so as to regulate<br>
hereditary factors of the phenotype of interest. To this end, a<br>
gene interaction analyzing system according to the present<br>
invention includes, as an essential component, the analyzing<br>
section 40 for analyzing the information entered through the<br>
input section. The analyzing section 40 includes at least the<br>
genetic marker specifying section 43, the spot marker<br>
information generating section 44, and the hereditary factor<br>
regulating section 46, as shown in Fig. 12.<br>
In the genetic marker specifying section 43, the<br>
gene-presence-information generated in the image information<br>
processing section 41 through the scanner 21 is compared<br>
with the genetic map and genetic marker information, so as to<br>
specify genetic markers for each different hybrid line.<br>
Specifically, from the result of comparison between a genetic<br>
map and position information of the genetic markers<br>
immobilized on the nucleic acid array, whether or not a hybrid<br>
individual of interest includes the genetic markers is<br>
determined. If the genetic markers are included, the genetic<br>
markers are specified as the genetic markers of the hybrid line<br>
to which the hybrid individuals belong.<br>
In obtaining the presence information of genes of the<br>
hybrid individuals, the use of the chromosomal location<br>
recoKnizable array as a nucleic firriiy IIIIOWH tlir order of<br>
immobilized spots to be used as the genetic marker<br>
information. In other words, the order of immobilized spots<br>
and the map distance of chromosomes, etc. can be used as<br>
f03<br><br>
genetic marker information. This is highly preferable as it<br>
makes it easier for the genetic marker specifying section 43 to<br>
perform the comparison. Further, as described above, the<br>
genetic marker information should preferably be polymorphic<br>
genetic markers (SNP or RFLP), since it is easily recognizable<br>
as typical genetic markers of each different hybrid line.<br>
In the spot marker information generating section 44, the<br>
genetic markers specified by the genetic marker specifying<br>
section 43 are compared with the genetic markers im.mobilized<br>
on the nucleic acid array, and the result of hybridization of<br>
individual spots on the nucleic acid array is generated as spot<br>
marker information and used as genetic marker information for<br>
analysis. More specifically, the specified genetic markers are<br>
compared with the genetic markers on the nucleic acid array,<br>
and only the genetic marker spots so found by the<br>
hybridization are generated as spot marker information. This<br>
enables the hybridized spots on the nucleic acid array to be<br>
used as the genetic markers that exist in each hybrid line.<br>
The chromosomal location recognizable array may be used<br>
as the nucleic acid array. In this case, the order of immobilized<br>
spots can be used as the genetic marker information.<br>
Specifically, in generating the spot marker information, certain<br>
immobilized genetic markers are obtained by reversing the<br>
order of immobilized spots, and the genetic markers so<br>
obtained are compared with the genetic markers previously<br>
specified. It is therefore preferable that the spot marker<br>
information include position information indicative of the<br>
positions of genetic markers immobilized on the nucleic acid<br>
array. In this way, when the expression profile information was<br>
obtained using the chromosomal location recognizable array,<br>
the position information can actually be used as spot marker<br>
information, and genetic markers can be readily specified.<br>
\ok<br><br>
In the hereditary factor regulating section 45, phenotypes<br>
and genes of interest to be analyzed are specified, and whether<br>
or not phenotypic values representing the phenotypes, and the<br>
expressed genes of interest included in the expression profile<br>
information obtained from the same hybrid individual are<br>
linked to plural pieces of spot marker information is confirmed.<br>
In this manner, the hereditary factors of the phenotypes of<br>
interest are regulated baaed on tlic cxprrHHcd M^'if*"' '1'IIUH,<br>
based on the specified phenotypes and genes anHociuted with<br>
the expression of the phenotypes, genes that are closely<br>
associated with the expression of the phenotype can be<br>
selected from the spot marker information, based on the<br>
presence or absence of a linkage.<br>
Specifically, for a gene pi known to be associated with a<br>
phenotype (trait) P, the gene pi and a phenotypic value Vp<br>
representing the phenotype P are specified, and the presence<br>
or absence of any linkage between the spot marker information<br>
and phenotypic value Vp, and gene pi is confirmed. For<br>
example, genetic markers mp2 and mp3 are found as genetic<br>
marker information linked to the phenotypic value Vp and gene<br>
pi. Here, if these genetic markers mp2 and mp3 are for genes<br>
p2 and p3, then the genes p2 and p3 are specified as genes<br>
interacting with the phenotype P and gene pi. Note that, the<br>
genetic markers can determine hereditary factors even when<br>
there is no linkage to known genes. Thus, as the information<br>
for regulating hereditary factors of the phenotypes, the<br>
hereditary factor regulating section 46 uses at least genetic<br>
markers, and preferably uses known genes.<br>
When no spot marker is found to be directly linked, a<br>
quantitative trait locus (QTL) is estimated between the most<br>
proximate genetic markers, and the hereditary factors may be<br>
regulated   based   on   the   QTL.   Thus,   provided   that   genetic<br>
(0^<br><br>
markers to be used as spot marker information are present<br>
with such a density that linkage is detectable on the genetic<br>
map, hereditary factors can be regulated based on QTL even<br>
when genetic markers that are linked at high resolution cannot<br>
be specified.<br>
By thus finding genetic markers (or genes, QTL) linked to<br>
the phenotypes, the hereditary factor regulating section 46,<br>
based on such genetic markers, can estimate locations or<br>
functions of genes associated with the expression of the<br>
phenotypes. Further, as the information for regulating<br>
hereditary factors of the phenotypes, the hereditary factor<br>
regulating section 46 can use the expression level of genes<br>
associated with the genetic markers.<br>
Whether the plural pieces of spot marker information is<br>
linked to the specified phenotypic values or genes can be found<br>
through analysis employing Interval mapping. The interval<br>
mapping is not particularly limited, and simple interval<br>
mapping (SIM), or composite interval mapping (CIM) may be<br>
used, for example. For a sprcific aniilyHiH uf inlnvnl ninppltig,<br>
a known analyzing system may be used. Specific- examples of<br>
such analyzing system include those using analyzing software<br>
such as MAPMARKER/QTL or QTL Cartographer.<br>
A gene interaction analyzing system according to the<br>
present invention may include the expression profile<br>
information generating section 45 in addition to the genetic<br>
marker specifying section 43, the spot marker information<br>
generating section 44, and the hereditary factor regulating<br>
section 46. As described earlier, the expression profile<br>
information generating section 45 is adapted to generate<br>
expression profile information of a hybrid individual by<br>
performing expression profile analysis with regard to a<br>
comprehensive  gene-expression level obtained  from  the  same<br>
(OG<br><br>
hybrid individual. Here, the expression profile information is<br>
generated through comprehensive measurement of gene<br>
expression using at least one of experiment systems using a<br>
micro array, a macro array, a bead array, and a differential<br>
display, as described above.<br>
The expression profile information generating section 45<br>
can also generate expression profile information using the<br>
nucleic acid array used for obtaining comprehensive<br>
gene-presence-information of hybrid individuals, or the nucleic<br>
acid array on which the same sample has been spotted.<br>
Namely, in the structure shown in Fig. 12, the scanner 21 first<br>
reads the result of expression profile experiment, and then the<br>
expression profile information generating section 45 generates<br>
expression profile information after images have been<br>
processed in the image processing section 11.<br>
Here, the analysis can be carried out more efficiently<br>
when analysis data for specifying genetic markers (and for<br>
constructing a genetic map) and analysis data for the<br>
expression profile are acquired simultaneously. For example,<br>
when four or more kinds of labeling is possible in the<br>
hybridization experiment using the nucleic acid array, two<br>
kind of labeling may be set for specifying genetic markers, and<br>
two kinds for gene expression. In this way, analysis data for<br>
specifying genetic markers (and for constructing a genetic map)<br>
and analysis data for the expression profile can be acquired<br>
simultaneously from a single nucleic acid array.<br>
Note that, in the structure shown in Fig. 12, the<br>
expression profile information generating section 45 and the<br>
image information processing section 41 may be provided in<br>
ont^ unjl. That is, the expression profiir infoniiMlion mtiy l&gt;e<br>
genrrafrcl by the image iniui'mation processing Hccliun 41.<br>
A   gene   interaction   analyzing   system   according   to   the<br>
lO"^<br><br>
present invention may include the genetic map constructing<br>
section 42 in addition to the genetic marker specifying section<br>
43, the spot marker information generating section 44, the<br>
expression profile information generating section 45, and the<br>
hereditary factor regulating section 46. Based on genetic map<br>
constructing information, the genetic map constructing section<br>
42 constructs a genetic map of a species to which the hybrid<br>
individual belongs. As described earlier, the genetic map is<br>
constructed for only some of the species. It is therefore<br>
preferable to provide the genetic map constructing section 42.<br>
The genetic map constructing section 42 is not<br>
particularly limited as long as the genetic map is constructed<br>
on the chromosome basis based on various genetic map<br>
constructing information. As the genetic map constructing<br>
information, at least names of genes and/or genetic markers<br>
known in the species being analyzed, and chromosomal loci of<br>
the genes and/or genetic markers are used, for example.<br>
The means for entering the genetic map constructing<br>
information is not particularly limited, and various input<br>
sections, for example, such as the external communications<br>
section 22, the storage medium reading and writing section 23,<br>
and the manual input section 24 shown in Fig. 12 can be used.<br>
Further, with the chromosomal location recognizable<br>
array, a genetic map can be constructed through mapping of<br>
genetic markers with unknown locations. Specifically, in order<br>
to construct a genetic map, targets obtained from a Mendelian<br>
segregation population of the species being analyzed are<br>
hybridized with the chromosomal location recognizable array.<br>
Then, genetic markers with unknown locations are hybridized<br>
on the same chromosomal location recognizable array, so as to<br>
determine locations of the genetic markers. In this way. a high<br>
density genetic map can be constructed.<br>
0?<br><br>
Even though the foregoing example uses the same<br>
chromosomal location recognizable array, the method of<br>
mapping the genetic markers of unknown location is not just<br>
limited to this example. For example, mapping can be made by<br>
processing the same targets with the genetic markers on<br>
different arrays. Here, mapping of genes is possible if the genes<br>
follow the rule of Mendelian segregation as in Single Feature<br>
Polymorphism (SFP), even if SNP or RFLP is not detected.<br>
Thus, a gene interaction analyzing HyHleni ticcdrrlinn to<br>
the present invention may be adapted ao Lhut, in order lo<br>
construct the genetic map in the genetic map constructing<br>
section 42, the hybridization result is analyzed and processed<br>
by reading it from the array with the scanner 21 and the image<br>
information processing section 41, before specifying genetic<br>
markers. To this end, the image information processing section<br>
41 is adapted to output information also to the genetic map<br>
constructing section 42, as shown in Fig. 12 (as indicated by<br>
arrow in the figure).<br>
Information such as the genetic map constructed by the<br>
genetic map constructing section 42, information concerning<br>
the genetic markers specified by the genetic marker specifying<br>
section 43, the result of determination made by the genotype<br>
origin detecting section 44, the expression profile information<br>
generated by the expression profile information generating<br>
section 45, or hereditary factor information generated by the<br>
hereditary factor regulating section 46 can be temporarily<br>
stored in the memory 48. The memory 48 is provided in the<br>
analyzing section 40 as shown in Fig. 12, and serves as a<br>
storage section for storing various information used or<br>
generated in a gene interaction analyzing system according to<br>
the present invention. The storage operation of the memory 48<br>
is controlled by the control section 47. The memory 48 is not<br>
(0^<br><br>
limited to a particular structure, and may be realized, for<br>
example, by a semiconductor memory, such as RAM or ROM.<br>
Note that, the storage medium reading and writing section 23<br>
described as an input section can be used as a storage section<br>
of the present invention. This will be described later in more<br>
detail in conjunction with the output section.<br>
The analyzing section 40 of the structure shown in Fig. 12<br>
includes the control section 47 for controlling the entire<br>
operation of the analyzing section 40, and in turn the entire<br>
operation of the gene interaction analyzing system. In the<br>
structure shown in Fig. 12, the control section 47 outputs<br>
control information to the image information processing<br>
section 41, the genetic map constructing section 42, the<br>
genetic marker specifying section 43, the spot marker<br>
information generating section 44, the expression profile<br>
information generating section 45, the hereditary factor<br>
regulating section 45, and the memory 48. These means<br>
operate based on the control information they receive, thereby<br>
operating the gene interaction analyzing syslem. It should be<br>
noted here that the control section A7 is IIIMO iidnplcd to<br>
rfcv'ivt information from these means, and as Much the How iiI<br>
control information is indicate by the bidirectional arrow in Fig.<br>
12.<br><output section><br>
In the present invention, hereditary factors of the<br>
phenotype of interest are regulated by finding whether the spot<br>
marker information is linked to the expressed genes included<br>
in the expression profile information. To this end, a gene<br>
interaction analyzing system according to the present<br>
invention includes means, provided as an output section, for<br>
outputting a regulation result of hereditary factors.<br>
The output section is not particularly limited, and at least<br><br>
one of, or preferably both of a display 26 for displaying a<br>
regulation result of hereditary factors on a display screen (soft<br>
copy), and a printer 25 for printing a regulation result of<br>
hereditary factors (hard copy} are provided. The display 26 is<br>
not limited to a particular structure, and various types of<br>
known displays such as a CRT, a liquid crystal display, and a<br>
plasma display can be used. The printer 25 is not limited to a<br>
particular structure, and known image forming devices such as<br>
an ink-jet printer and a laser printer can be used.<br>
The output section is not just limited to the display 26 or<br>
printer 25, and other means can be used as well. For example,<br>
the external communications section 22 can be used as an<br>
output section. Specifically, the external communications<br>
section 22 allows for input and output of information to and<br>
from external devices by serving as both an input section and<br>
an output section. This enables the result of QTL analysis to<br>
be transmitted to other devices via external networks, etc,<br>
enabling a gene interaction analyzing system according to the<br>
present invention to be used more efficiently.<br>
Specifically, when the gene interaction analyzing system is<br>
connected to external devices via LAN for example, the gene<br>
interaction analyzing system, installed in a research facility for<br>
example, can be shared with other researchers via information<br>
terminals such as personal computers. Further, the results of<br>
analysis obtained in the gene interaction analyzing system may<br>
be accumulated in an external server via a communications<br>
network, allowing the analysis result to be used more<br>
efficiently.<br>
As the output section, the storage medium reading and<br>
writing section 23, described as an input section, can be<br>
suitably used. Specifically, in a gene inlrrniiion analyzing<br>
Hyslcm ac(^ording to the present  iiivrnlion, ii diivp lor rpiidlii^<br>
in<br><br>
information from a storage medium can be used as an output<br>
section if the drive has a writing capability. The storage<br>
medium reading and writing section 23 is not limited to a<br>
particular structure, and known disk drives such as a hard<br>
disk drive, a flexible disk drive, a CD-ROM drive, and a<br>
DVD-ROM drive, or various memory cards or memory<br>
cartridges such as a USB memory can be suitably used for<br>
example, as described above in conjunction with the input<br>
section.<br>
Note that, in the exemplary structure shown in Fig. 12,<br>
the system is realized by the analyzing section 40 and<br>
independently provided input and output sections, wherein the<br>
analyzing section 40 includes the image information processing<br>
section 41, the genetic map constructing section 42, the<br>
genetic marker specifying section 43, the spot marker<br>
information generating section 44, the expression profile<br>
information generating section 45, the hereditary factor<br>
regulating section 46, the control section 47, and the memory<br>
48. However, the present invention is not just limited to this<br>
structure. For example, all means may be provided as a single<br>
unit, or some of the input sections and/or output sections may<br>
be integrated with the analyzing section 40. Further, the<br>
system may include means other than those shown in Fig. 12.<br>
The analyzing section 40 is not just limited to a particular<br>
structure, and conventional arithmetic means, for example,<br>
such as a central processing unit (CPU) of a computer may be<br>
used. The operation of the analyzing section 40 is executed by<br>
a computer program.<br>
iUl) Analyzing method by the gene interaction analyzing<br>
system<br>
An analyzing method performed by a gene interaction<br>
analyzing   system   according  to   the   present   invention   is   not<br>
n5/<br><br>
particularly limited. Specifically, the method may include 16<br>
steps as represented in Fig. 13.<br>
First, in step 301 (step will be denoted by "S" hereinafter),<br>
genetic map constructing information {names of chromosomes,<br>
genes, and genetic markers, and loci, etc.) is entered through<br>
input sections. In S302, the genetic map constructing section<br>
42 constructs a genetic map based on the genetic map<br>
constructing information, and the genetic map is supplied to<br>
the genetic marker specifying section 43. In S303, the number<br>
of hybrid lines is entered through the inpul srction. In R304,<br>
genc-preHence-information (i.e., the reuutl (iV UNA tinny<br>
analysis) of the hybrid individuals (targets individuals in Fig.<br>
13) being analyzed is entered through the scanner 21 and the<br>
image information processing section 41 for each different<br>
hybrid line. In S305, genetic marker information is entered.<br>
In S306, based on the entered gene-presence-information,<br>
genetic map, and genetic marker information, the genetic<br>
marker detecting section 43 determines a genetic marker that<br>
is present in each different hybrid line. Here, the result of<br>
determination of a genetic marker may be stored in the<br>
memory 48, or optionally displayed in the display 26. In S307,<br>
the genetic marker specified in S306 is compared with genetic<br>
markers immobilized on the DNA array, and spot marker<br>
information is generated from the individual spots of the DNA<br>
array. The spot marker information may be stored in the<br>
memory 48, or optionally displayed in the display 26. In S308,<br>
the presence or absence of an entry error is found (need for<br>
correction is determined). If there is an entry error (YES), the<br>
information is re-entered in S309 through, for example, the<br>
manual input section 24, and the sequence returns to S301.<br>
On the other hand, if correction is not required (NO), the<br>
sequence goes to S310.<br>
(15<br><br>
In S310, a phenotypic value is entered through the input<br>
section. In S311, expression profile information of the hybrid<br>
individual being analyzed is entered through the input section.<br>
In S312, based on the expression profile information entered in<br>
S311, the expression profile information generating section 45<br>
identifies genes with different expression levels. The identified<br>
genes may be stored in the memory 48, or optionally displayed<br>
in the display 26. In S313, a phenotype and gene of interest to<br>
be analyzed is entered through the input section.<br>
In S314, interval mapping (QTL analysis) is performed<br>
based on the phenotypic value entered in S310 and the<br>
expressed genes identified in S312, so as to determine whether<br>
the plural pieces of spot marker information is linked to the<br>
phenotypic value and the expressed genes. In S315, based on<br>
the result of QTL analysis, associated genes and/or genetic<br>
markers are estimated, and the hereditary factors of the<br>
specified phenotype and gene are regulated based on the result<br>
of estimation. The regulation result of hereditary factor may be<br>
stored in the memory 48, or optionally displayed in the display<br>
26. In S316, whether or not correction is required for the genes,<br>
flc beins analyzed is determined hiiscd oii the rrHiilt of<br>
aiiulysis. If correction is required (YES), llie stMiLieiit^e returns<br>
lo S313. If correction is not required (NO), the sec]Lience goes to<br>
S317. The result of analysis is outputted from the output<br>
section in S317. This completes the series of analysis<br>
procedures,<br>
(IV) Use of the present invention<br>
Use of a gene interaction analyzing system according to<br>
the present invention is not particularly limited as long as<br>
interaction of more than one gene associated with a desired<br>
phenot3'pe (trait) is analyzed in the species of interest.<br>
As   described   earlier,   a   technique   of   expression   profile<br>
Vk<br><br>
analysis is known in which a group of genes are analyzed in<br>
clusters, or a network of gene expression is analyzed. These<br>
techniques are useful for the non-exclusive or comprehensive<br>
analysis of gene expression. However, since the techniques are<br>
for extracting closely associated genes with no other given<br>
information, they cannot be used for extracting hereditary<br>
factors closely associated with previously specified traits or<br>
genes to be analyzed. However, with a gene interaction<br>
analyzing system according to the present invention, a trait or<br>
gene of interest can be specified before actually analyzing gene<br>
interaction. That is, gene interaction can be analyzed<br>
differently from the expression profile analysis.<br>
Thus, a gene interaction analyzing system according to<br>
the   present   invention   is   applicable   not   only   to   a   gene<br>
interaction analyzing method but also to a gene search method,<br>
in which a search is made for genes associated with previously<br>
specified traits or genes to be analyzed.<br>
In the expression profile analysis performed by the<br>
technique disclosed in Patent Document 6 as described in the<br>
BACKGROUND ART section, genes closely associated with a<br>
target evaluation index are extracted to estimate evaluation<br>
index data. The technique may appear to be similar to the<br>
present invention in the sense that the analysis is based on<br>
previously entered evaluation index data of interest. However,<br>
since the technique is for analyzing expression profiles based<br>
on the results obtained from a DNA array, it does not use the<br>
DNA array spots as genetic markers (spot marker information),<br>
nor does it perform a QTL analysis using such genetic<br>
markers.<br>
The present invention does not analyze expression profiles,<br>
i.e.,    comprehensive    analysis    of   gene    expression    is    not<br>
performed. Rather, the analysis focuses on inlrrnctlon between<br>
^<br><br>
genrs. In this way, a relationship betWf-rn grncs of interest c.iin<br>
be established more clearly as compared with the<br>
comprehensive analysis. Therefore, the present invention can<br>
be suitably used for purposes requiring detailed analysis of<br>
genes which have already been analyzed by the comprehensive<br>
analysis.<br>
Generally, in the cluster technique, a group of genes with<br>
different expression levels for a particular trait are detected in<br>
clusters. In order to find any relationship between these genes,<br>
each gene is annotated or a known pathway is analyzed, or, by<br>
an experimental biological approach, labeled genes are directly<br>
introduced into an organism. On the other hand, the present<br>
invention employs a genetic approach, whereby an analysis<br>
using a nucleic acid array is applied to a Mendelian<br>
segregation population, and interaction between individual<br>
genes contained in the cluster is identified in an exploratory<br>
manner. That is, interaction can be identified even for genes of<br>
unknown functions, and therefore any genetic association of a<br>
group of genes with a particular trait can be estimated. Since<br>
these genes are mapped, every gene can be introduced by<br>
variety improvement. Further, by the QTL analysis, the<br>
influence of the introduced genes on a trait can be statistically<br>
analyzed.<br>
Specific use of the present invention is not particularly<br>
limited. For example, the gene interaction analyzing system<br>
can be used in a variety improvement method using genetic<br>
markers. As an example, traits or genes of interest may be<br>
specified for variety improvement, and genes or genetic<br>
markers closely associated with these traits or genes may be<br>
regulated. In this way, variety improvement can be performed<br>
more efficiently.<br>
The  type  of organism to  which  an  array  of the  present<br><br>
invention is appHcable is not particularly limited, and any of<br>
plants, animals, and microorganisms may be used. Particularly,<br>
an array of the present invention can be used in the foregoing<br>
screening method in organisms that include chromosomes and<br>
obey the laws of Mendelian genetics. Examples of such an<br>
organism are, but not limited to, those commercially available<br>
and for which need for variety improvement is high.<br>
In the case of plants, various crops (plant and farm<br>
products produced in agriculture, forestry, and fishery<br>
industries] can be used. Specific examples include; cereals<br>
sxich as rice, wheat, barley, rye, triticale, and corn; marine<br>
plunta siicli as seaweed; vuricius vr^rtaljlrH uiid flowrrM; nnd<br>
trees such as cedar or cypress. In the case of animals, various<br>
domestic animals can be used. Specific examples include:<br>
domestic mammals such as bovines, sheep, and pigs; domestic<br>
birds such as chickens and quails; fish such as yellowtail<br>
snapper, sea bream, carp, and sweetfish; insects such as<br>
honey bees, and silkworm; and shellfish such as oyster, ormer,<br>
and scallop. As microorganisms, bacteria such as Escherichia<br>
coli, yeasts, fungi, actinomycetes, and basidiomycetes can be<br>
used.<br>
Among these examples, the cereals include crops such as<br>
rice, wheat, corn, and barley, which are cultivated worldwide<br>
and are strategically important. Thus, by using the present<br>
invention for the variety improvement of these plants, varieties<br>
with desirable traits can be efficiently produced.<br>
The present invention can also be used for experimental<br>
animals and plants. Specific examples of experimental animals<br>
include mice, rats, D. raelanogaster, and C. elegans. A specific<br>
example is Arabidopsis thaliana. Further, for the purpose of<br>
identifying genotypes with the present invention, the invention<br>
can be applied to humans.<br>
in<br><br>
As described above, in a gene interaction analyzing system<br>
according to the present invention, data of genetic markers for<br>
mapping is obtained together with gene expression data. With<br>
these data combined with phenotype data, the expression<br>
levels of respective genes are taken as target variables and are<br>
individually analyzed by a QTL analysis. That is, genes<br>
associated with traits or genes of interest are estimated after<br>
specifying traits or genes to be analyzed. It is therefore possible<br>
to efficiently estimate genes closely associated with traits or<br>
genes to be analyzed. Further, by interval mapping, hereditary-<br>
factors can be estimated as quantitative trait loci (QTL) that<br>
exist between genetic markers, even when the genetic markers<br>
themselves are not sufficient to regulate the hereditary factors.<br>
In sum, gene interaction can be efficiently analyzed under<br>
specific conditions, allowing for detailed analysis of gene<br>
interaction with the linkage information or QTL information,<br>
for example.<br>
The invention being thus described, it will be obvious that<br>
the same may be varied in many ways. Such variations are not<br>
to be regarded as a departure from the spirit and scope of the<br>
invention, and all such modifications as would be obvious to<br>
one skilled in the art are intendecf to he iruindrfl within the<br>
Ht'ope of the following claims.<br>
INDUSTRIAL APPLICABILITY<br>
As described above, in the present invention, the<br>
biosubstances or synthetic substances immobilized on an array<br>
are arranged based on the chromosomal order of genes<br>
corresponding to the biosubstances. The invention therefore has<br>
practical applications such as identification of a genotype, gene<br>
diagnosis, screening in variety improvement, or the like. Further,<br>
the invention improves reliability of an array analysis. Thus, the<br><br>
invention can be suitably used in the production of research<br>
reagents or samples using various types of arrays, or industries<br>
related to analytical techniques. Other applicable areas of the<br>
invention include crop production, animal production, and<br>
fisheries, in which identification of a genotype or variety<br>
improvement of organisms that follow the laws of Mendelian<br>
genetics is performed. The invention also has medical or<br>
pharmaceutical applications, such as gene diagnosis.<br>
With the present invention, the genotype of each<br>
individual of a hybrid generation can be accurately determined<br>
or confirmed only by acquiring a genome of each individual of<br>
the hybrid generation and obtaining a hybridization result of a<br>
nucleic acid array. Thus, the invention can be suitably used in<br>
the production of research reagents or samples using various<br>
types of arrays, or industries related to analytical techniques.<br>
Other appHcable areas of the invention include crop production,<br>
animal production, and fisheries, in which identification of a<br>
genotype or variety improvement of organisms that follow the<br>
laws of Mendelian genetics is performed. The invention also has<br>
medical or pharmaceutical applications, such as gene diagnosis.<br>
Further, with the present invention, a QTL analysis can be<br>
efficiently performed with the array technique. Thus, the<br>
invention can be suitably used in the production of research<br>
reagents or samples using various types of arrays, or industries<br>
related to analytical techniques. Other applicable areas of the<br>
invention include crop production, animal production, and<br>
fisheries, in which identification of a genotype or variety<br>
improvement of organisms that follow the laws of Mendelian<br>
genetics is performed. The invention also has medical or<br>
pharmaceutical applications, such as gene diagnosis.<br>
Further, with the present invention, gene interaction<br>
can be analyzed both efficiently and thoroughly under specific<br>
.\^<br><br><br>
animal production, and fisheries, in which identification of a<br>
genotype or variety improvement of organisms that follow the<br>
laws of Mendelian genetics is performed. The invention also<br>
has medical or pharmaceutical applications, such as gene<br>
diagnosis.<br>
\^10<br><br><br>
WE CLAIM:<br>
1.	An array in which different kinds of biosubstances obtained from an<br>
organism of interest, or synthetic substances interacting with such biosubstances<br>
are arranged and immobilized on a support in an orderly manner.<br>
the different kinds of biosubstances or the synthetic substances being<br>
arranged such that a chromosomal order of base sequence blocks corresponding<br>
to the biosubstances is ascertainable,<br>
said biosubstances being nucleic acids or polypeptides, wherein the<br>
nucleic acids comprise DNA, and the polypeptides comprise proteins, fragments<br>
thereof, or oligopeptides,<br>
said DNA or said proteins being arranged on the support based on a<br>
genetic map or physical map.<br>
2.	The array as claimed in claim I, wherein at least part of the different<br>
kinds of biosubstances or the synthetic substances are arranged in the<br>
chromosomal order of the base sequence blocks corresponding to the<br>
biosubstances.<br>
3.	The array as claimed in claim 1 or 2. wherein the support includes a<br>
label that indicates the chromosomal order of the base sequence blocks<br>
corresponding to the biosubstances,<br>
4.	'The array as claimed in claim 1, wherein the biosubstances or the<br>
synthetic substances immobilized on the support each include sequence position<br>
information corresponding to the chromosomal order of the base sequence blocks<br>
corresponding to the biosubstances, and<br>
wherein, in use, data is acquired and the sequence position information is<br>
read out. so as to rearrange sequences of the data in the chromosomal order.<br>
5.	An array as set forth in claim 1.<br><br><br>
wherein the support comprises a collection of micro supports on which the<br>
biosubstances or the synthetic substances are individually immobilized,<br>
wherein the micro supports each include sequence position information<br>
corresponding to the chromosomal order of the base sequence blocks<br>
corresponding to the biosubstances, and<br>
wherein sequences of acquired data are rearranged in the chromosomal<br>
order based on the sequence position information.<br>
6.	The array as claimed in claim 1, wherein the DNA comprises a genetic<br>
marker, genomic DNA, genomic DNA treated with a restriction enzyme, cDNA,<br>
EST. or synthetic oligoDNA,<br>
7.	The array as claimed in claim 1 or 6, wherein genomic DNA treated<br>
with a restriction enzyme is used as target DNA.<br>
8.	The array as claimed in claim 7. wherein the target DNA is fractionated<br>
by size after the treatment with a restriction enzyme.<br>
9.	The array as claimed in claim I, wherein the proteins comprise<br>
enzymes, kinase, antibodies, receptors, or proteins with SH3 region.<br>
10.	The array as claimed in any one of claims 1 through 9, wherein the<br>
support or micro support comprises an inorganic substrate, an organic film, or a<br>
bead.<br>
11.	The array as claimed in any one of claims 1 through 10, which<br>
comprises any one of a micro array, a macro array, a bead array, and a protein<br>
chip.<br>
12.	A producing process of an array, comprising the step of orderly<br>
arranging  and   immobilizing   on  a  support  different  kinds  of biosubstances<br>
122<br><br>
obtained from an organism of interest, or synthetic substances interacting with<br>
such biosubslances.<br>
said step comprising arranging and immobilizing the biosubstances or the<br>
synthetic substances according to the order in which genes corresponding to the<br>
biosubstances are coded for on a chromosome of the organism.<br>
13.	The producing process as claimed in claim 12, wherein the<br>
biosubstances comprise nucleic acids or polypeptides.<br>
14.	A genotype identification method, comprising identifying a target<br>
trait^including chromosome fragment, using the array of claim 1, from hybrids<br>
obtained by crossing organisms.<br>
15.	An identification method as claimed in claim 14, wherein the<br>
organisms comprise laboratory animals and plants.<br>
16.	A gene diagnosis method for identifying human genotypes, using the<br>
identification method of claim 15.<br>
17.	A screening method for screening for a target trait-carrying variety<br>
from hybrids obtained by crossing organisms whose characteristics are to be<br>
improved, using the array of claim 1.<br>
18.	The screening method as claimed in claim 17, wherein the living<br>
organisms crossed for variety improvement comprise laboratory animals and<br>
plants, domestic animals, or crops.<br>
19.	The screening method as claimed in claim 18, wherein the crops<br>
comprise cereals.<br><br><br>
20. The screening method as claimed in claim 19. wherein the cereals<br>
comprise rice, wheat, corn, or barley.<br>
2!. A genotype analyzing and display system, comprising:<br>
genotype origin detecting means for comparing (a) gene expression level<br>
information and polymorphism information comprehensively obtained through a<br>
hybridization analysis of hybrid individuals with the array of claim 1 with (b)<br>
genetic information of parents of the hybrid individuals, and a genetic map of a<br>
species to which the hybrid individuals belong, so as to determine whether a<br>
genotype of a hybrid individual of interest derives from which parent; and<br>
display information generating means for gathering a plurality of results<br>
obtained from the genotype origin detecting means and, based on the results,<br>
generating display information used to display a plurality of genotypes<br>
altogether on a chromosome basis, so as to determine whether individual<br>
genotypes derives from which parent.<br>
22. A quantitative loci analyzing system which uses the array of claim 1,<br>
and in which a genetic marker of a species of interest is immobilized on the<br>
array,<br>
said quantitative loci analyzing system comprising;<br>
genetic marker specifying means for comparing (a) comprehensive<br>
presence information of genes of hybrid individuals, obtained by hybridizing the<br>
array with a genomic sample obtained from the hybrid individuals of a certain<br>
hybrid line with (b) a genetic map of a species to which the hybrid individuals<br>
belong, and genetic marker information known in the species, so as to specify a<br>
genetic marker that exists in the hybrid line; and<br>
quantitative loci detecting means for detecting a quantitative locus of a<br>
phenotype of interest of the hybrid individual, by confirming whether a<br>
phenotypic value indicative of the phenotype is linked to the genetic marker.<br><br><br>
23.	A gene interaction analyzing system which uses the array of claim 1.<br>
and in which a genetic marker of a species of interest is immobilized on the<br>
array,<br>
said gene interaction analyzing system comprising:<br>
genetic marker specifying means for comparing (a) comprehensive<br>
presence information of genes of hybrid individuals, obtained by hybridizing the<br>
array with a genomic sample obtained from the hybrid individuals of a certain<br>
hybrid line (b) with a genetic map of species to which the hybrid individuals<br>
belong, and genetic marker information known in the species, so as to specify a<br>
genetic marker that exists in the hybrid line;<br>
spot marker inforn)ation generating means for comparing the specified<br>
genetic marker with the genetic marker immobilized on the support, so as to<br>
generate spot marker information, being genetic marker information for use in<br>
analysis, from hybridization results obtained from individual spots on the array;<br>
and<br>
hereditary factor specifying means for specifying, with regard to an<br>
arbitrarily selected phenotype and gene to be analyzed, a hereditary factor of the<br>
selected phenotype by determining whether the phenotypic value indicative of<br>
the phenotype, and an expressed gene included in expression profile information<br>
obtained from the hybrid individual are linked to a plurality of spot marker<br>
information.<br>
24,	A genotype analyzing and display system which uses the nucleic acid<br>
array of claim 1, comprising:<br>
genotype origin detecting means for comparing (a) gene expression level<br>
information and polymorphism information comprehensively obtained through a<br>
hybridization analysis of hybrid individuals using a nucleic acid array with (b)<br>
genetic information of parents of the hybrid individuals, and a genetic map ot<br>
species to which the hybrid individuals belong, so as to determine whether a<br>
genotype of a hybrid individual of interest derives from which parent; and<br>
125<br><br>
display information generating means for gathering a plurality of results<br>
obtained from the genotype origin detecting means and, based on the results,<br>
generating display information used to display a plurality of genotypes<br>
altogether on a chromosome basis, so as to determine whether individual<br>
genotypes derives from which parent.<br>
25.	The genotype analyzing and display system as claimed in claim 24,<br>
further comprising genetic map constructing means for constructing, based on<br>
genetic map constructing information, a genetic map of a species to which the<br>
hybrid individuals belong.<br>
26.	The genotype analyzing and display system as claimed in claim 25,<br>
wherein the genetic map constructing information comprises names of genes<br>
and/or genetic markers known in the species, and chromosomal loci of the genes<br>
and/or genetic markers.<br>
27.	The genotype analyzing and display system as claimed in any one of<br>
claims 24 through 26, wherein the genotype origin detecting means determines a<br>
genotype as being homozygous for one of the parents, heterozygous, or<br>
unrecognizable to yield a result.<br>
28.	The genotype analyzing and display system as claimed in any one of<br>
claims 24 through 26, wherein the genotype origin detecting means uses<br>
genotype information and/or gene expression profile information of parents as<br>
genetic information of parents.<br>
29.	The genotype analyzing and display system as claimed in any one of<br>
claims 24 through 28, wherein the display information generating means<br>
generates display information including at least one of recombination number<br>
and recombination frequency of individual chromosomes.<br><br><br>
30. The genotype analyzing and display system as claimed in any one of<br>
claims 24 through 29, wherein the display information generating means<br>
generates display information such that an origin of a genotype is identifiable<br>
based on different display colors or patterns.<br>
3 1. The genotype analyzing and display system as claimed in any one of<br>
claims 24 through 30, comprising at least one of input means and output means.<br>
32.	The genotype analyzing and display system as claimed in claim 31,<br>
wherein the input means receives at least one of comprehensive expression level<br>
information of genes of the hybrid individuals, and genetic information of<br>
parents.<br>
33.	The genotype analyzing and display system as claimed in claim 32,<br>
wherein the input means receives genetic map constructing information.<br>
34.	The genotype analyzing and display system as claimed in any one of<br>
claims 31 through 33, comprising:<br>
image reading means, provided as the input means, for enabling a<br>
hybridization result of the nucleic acid array to be read out as image information;<br>
and<br>
image information processing means for analyzing an expression level of<br>
gene based on the image information and generating comprehensive expression<br>
level information of gene.<br>
35.	The genotype analyzing and display system as claimed in any one of<br>
claims 31 through 34, comprising manual input means, provided as the input<br>
means, for modifying at least one of: the comprehensive expression level<br>
information of gene of the hybrid individuals; the genetic information of parents;<br>
and the genetic map constructing information.<br>
127<br><br>
36.	The genotype analyzing and display system as claimed in any one of<br>
claims 31 through 35, wherein the output means comprises at least one of: image<br>
display means for displaying the display information on a screen; and printing<br>
means for printing the display information.<br>
37.	The genotype analyzing and display system as claimed in any one of<br>
claims 31 through 35, wherein the input means and the output means comprise<br>
external information input-output means for sending and receiving information<br>
to and from an external device.<br>
38.	The genotype analyzing and display system as claimed in any one of<br>
claims 24 through 37, wherein the nucleic acid array comprises a DNA array on<br>
which DNA is immobilized.<br>
39.	The genotype analyzing and display system as claimed in claim 38,<br>
wherein the DNA immobilized on the DNA array comprises a genetic marker,<br>
genomic DNA, genomic DNA treated with a restriction enzyme, cDNA, EST, or<br>
synthetic oligoDNA.<br>
40.	The genotype analyzing and display system as claimed in any one of<br>
claims 24 through 39, which comprises any one of a micro array, a macro array,<br>
and a bead array.<br>
41.	A genotype identification method, comprising identifying a target<br>
trait-including chromosome fragment, using the genotype analyzing and display<br>
system of any one of claims 24 through 40, from hybrids obtained by crossing<br>
organisms.<br>
42.	An identification method as claimed in claim 41, wherein the<br>
organisms comprise laboratory animals and plants.<br>
128<br><br>
' 43. A screening method for screening for a target Ira it-carrying variety<br>
from hybrids obtained by crossing organisms whose characteristics are to be<br>
improved, using the genotype analyzing and display system of any one of claims<br>
24 through 40.<br>
44.	The screening method as claimed in claim 43, wherein the organisms<br>
crossed for variety improvement comprise laboratory animals and plants,<br>
domestic animals, or crops.<br>
45.	A quantitative loci analyzing system which uses the nucleic acid array<br>
of claim 1. comprising:<br>
genetic marker specifying means for comparing (a) comprehensive<br>
presence information of genes of hybrid individuals, obtained by hybridizing a<br>
genomic sample of the hybrid individuals of a certain hybrid line with a nucleic<br>
acid array on which a genetic marker of a species of interest is immobilized with<br>
(b) a genetic map of a species to which the hybrid individuals belong, and<br>
genetic marker information known in the species, so as to specify a genetic<br>
marker that exists in the hybrid line; and<br>
quantitative loci detecting means for detecting a quantitative locus of a<br>
phenotype of interest of the hybrid individual, by confirming whether a<br>
phenotypic value indicative of the phenotype is linked to the genetic marker.<br>
46.	The quantitative loci analyzing system as claimed in claim 45. further<br>
comprising genetic map constructing means for constructing, based on genetic<br>
map constructing information, a genetic map of a species to which the hybrid<br>
individuals belong.<br>
47.	The quantitative loci analyzing system as claimed in claim 46, wherein<br>
the genetic map constructing information comprises names of genes and/or<br>
genetic markers known in the species, and chromosomal loci of the genes and/or<br>
genetic markers.<br>
129<br><br>
' 84. The gene interaction analyzing system as claimed in any one of claims<br>
78 through 83, wherein the output means comprises at least one of image display<br>
means for displaying an analysis result on a screen; and printing means for<br>
printing an analysis result.<br>
85.	The gene interaction analyzing system as claimed in any one of claims<br>
78 through 84, wherein the input means and the output means comprise external<br>
information input-output means for sending and receiving information to and<br>
from an external device.<br>
86.	A gene interaction analyzing method for analyzing interaction between<br>
genes, using the gene interaction analyzing system of any one of claims 64<br>
through 85.<br>
87.	A gene searching method for searching for a gene associated with a<br>
trait of interest, using the gene interaction analyzing system of any one of claims<br>
64 through 85,<br>
88.	A variety improvement method for organisms, which uses the gene<br>
interaction analyzing system of any one of claims 64 through 85.<br>
89.	A variety improvement method as set forth in claim 88. wherein the<br>
organisms comprise laboratory animals and plants, domestic animals, or crops.<br><br><br>
48.	The quantitative loci analyzing system as claimed in any one of claims<br>
45 through 47. wherein the genetic marker information used by the genetic<br>
marker specifying means comprises a genetic marker with polymorphism.<br>
49.	The quantitative loci analyzing system as claimed in claim 48, wherein<br>
the genetic marker comprises SNP or RFLP.<br>
50.	The quantitative loci analyzing system as claimed in any one of claims<br>
45 through 49, wherein the quantitative loci detecting means detects a quantitative<br>
locus of phenotype by interval mapping,<br>
51.	The quantitative loci analyzing system as claimed in any one of cJaims<br>
45 through 50, comprising:<br>
image reading means for enabling a hybridization result of the nucleic<br>
acid array to be read out as image information; and<br>
image information processing means for analyzing the image information<br>
and generating comprehensive expression level information of gene.<br>
52.	The quantitative loci analyzing system as claimed in any one of claims<br>
45 through 50, comprising at least one of input means and output means.<br>
53.	The quantitative loci analyzing system as claimed in claim 52, wherein<br>
the input means receives at least one of the genetic marker information and the<br>
phenotypic value.<br>
54.	The quantitative loci analyzing system as claimed in claim 53, wherein<br>
the input means receives at least one of the genetic map and the genetic map<br>
constructing information.<br>
130<br><br>
55.	The quantitative loci analyzing system as claimed in any one of claims<br>
52 through 54, comprising manual input means, provided as the input means, for<br>
modifying at least one of: the comprehensive presence information of gene of the<br>
hybrid individuals; the genetic marker information, and the genetic map<br>
constructing information.<br>
56.	The quantitative loci analyzing system as claimed in any one of claims<br>
52 through 55, wherein the output means comprises at least one of image display<br>
means for displaying an analysis result on a screen; and printing means for<br>
printing an analysis result,<br>
57.	The quantitative loci analyzing system as claimed in any one of claims<br>
52 through 56, wherein the input means and the output means comprise external<br>
information input-output means for sending and receiving information to and<br>
from an external device.<br>
58.	The quantitative loci analyzing system as claimed in any one of claims<br>
45 through 57, wherein the nucleic acid array comprises a DNA array on which<br>
DNA is immobilized.<br>
59.	The quantitative loci analyzing system as claimed in any one of claims<br>
45 through 58, wherein the nucleic acid array comprises a micro array, a macro<br>
array, or a bead array.<br>
60.	A quantitative trait analyzing method for analyzing a quantitative trait<br>
of an organism, using the quantitative loci analyzing system of any one of claims<br>
45 through 59.<br>
61.	A gene searching method for searching for a gene associated with<br>
expression of a trail of interest, using the quantitative loci analyzing system of<br>
any one of claims 45 through 59.<br>
131<br><br>
62.	A variety improvement method for organisms, which uses the<br>
quantitative loci analyzing system of any one of claims 45 through 59.<br>
63.	A variety improvement method as claimed in claim 62, wherein the<br>
organisms comprise laboratory animals and plants, domestic animals, or crops.<br>
64.	A gene interaction analyzing system which uses the nucleic acid array<br>
of claim 1, comprising:<br>
genetic marker specifying means for comparing (a) comprehensive<br>
presence information of genes of hybrid individuals, obtained by hybridizing a<br>
genomic sample of the hybrid individuals of a certain hybrid line with a nucleic<br>
acid array on which a genetic marker of a species of interest is immobilized with<br>
(b) a genetic map of a species to which the hybrid individuals belong, and<br>
genetic marker information known in the species, so as to specify a genetic<br>
marker that exists in the hybrid line;<br>
spot marker information generating means for comparing the specified<br>
genetic marker with the genetic marker immobilized on the nucleic acid array, so<br>
as to generate spot marker information, being genetic marker information for use<br>
in analysis, from hybridization results obtained from individual spots on the<br>
nucleic acid array; and<br>
hereditary factor specifying means for specifying, with regard to an<br>
arbitrarily selected phenotype and gene to be analyzed, a hereditary factor of the<br>
selected phenotype by determining whether the phenotypic value indicative of<br>
the phenotype, and an expressed gene included in expression profile information<br>
obtained from the hybrid individual are linked to a plurality of spot marker<br>
information.<br>
132<br><br>
65.	The gene interaction analyzing system as claimed in claim 64, further<br>
comprising genetic map constructing means for constructing, based on genetic<br>
map constructing information, a genetic map of a species to which the hybrid<br>
individuals belong.<br>
66.	The gene interaction analyzing system as claimed in claim 65,<br>
wherein the genetic map constructing information comprises names of genes<br>
and/or genetic markers known in the species, and chromosomal loci of the genes<br>
and/or genetic markers.<br>
67.	The gene interaction analyzing system as claimed in any one of claims<br>
64 through 66, wherein the genetic marker information used by the genetic<br>
marker specifying means comprises a genetic marker with polymorphism.<br>
68.	The gene interaction anaiyzing system as claimed in claim 67, wherein<br>
the genetic marker comprises SNP or RFLP.<br>
69.	The gene interaction analyzing system as claimed in any one of claims<br>
64 through 68, wherein the spot marker information generating means generates<br>
spot marker information only for a genetic marker spot found by hybridization.<br>
70.	The gene interaction analyzing system as claimed in claim 69, wherein<br>
the spot marker information generating means generates spot marker information<br>
by including position information of a genetic marker immobilized on the<br>
nucleic acid array.<br>
71.	The gene interaction analyzing system as claimed in any one of claims<br>
64 through 70, comprising expression profile information generating means for<br>
analyzing an expression profile in regard to a comprehensive gene expression<br>
level obtained from the hybrid individual, so as to generate expression profile<br>
information of the hybrid individual.<br>
133<br><br>
72.	The gene interaction analyzing system as claimed in claim 71, wherein<br>
the expression profile information generating means generates expression profile<br>
information of the hybrid individual by comprehensively measuring gene<br>
expression, using at least one of a micro array, a macro array, a bead array, and a<br>
differential display.<br>
73.	The gene interaction analyzing system as claimed in claim 72, wherein<br>
the expression profile information generating means generates expression profile<br>
information using a nucleic acid array used to obtain comprehensive presence<br>
information of gene of the hybrid individual, or a nucleic acid array on which the<br>
same sample has been spotted.<br>
74.	The gene interaction analyzing system as claimed in any one of claims<br>
64 through 73, wherein the nucleic acid array comprises a DNA array on which<br>
DNA is immobilized.<br>
75.	The gene interaction analyzing system as claimed in any one of claims<br>
64 through 74, wherein the nucleic acid array comprises a micro array, a niacro<br>
array, or a bead array.<br>
76.	The gene interaction analyzing system as claimed in any one of claims<br>
64 through 75. wherein the hereditary factor specifying means specifies a<br>
hereditary factor of a phenotype based on a quantitative trait locus (QTL) that<br>
exists among genetic markers obtained by interval mapping.<br>
77.	The gene interaction analyzing system as claimed in claim 76, wherein<br>
the hereditary factor specifying means uses information of expression level of a<br>
gene associated with the genetic marker, so as to specify a hereditary factor of<br>
the phenotype.<br>
134<br><br>
78.	The gene interaction analyzing system as claimed in any one of claims<br>
64 through 77. comprising at least one of input means and output means.<br>
79.	The gene interaction analyzing system as claimed in claim 78, wherein<br>
the input means receives at least one of: comprehensive presence information of<br>
gene of the hybrid individual; the genetic marker information; the phenotypic<br>
value; and the expression profile information,<br>
80.	The gene interaction analyzing system as claimed in claim 79, wherein<br>
the input means receives at least one of the genetic map and the genetic map<br>
constructing information.<br>
81.	The gene interaction analyzing system as claimed in any one of<br>
claims 78 through 80, comprising:<br>
image reading means, provided as the input means, for enabling a<br>
hybridization result of the nucleic acid array to be read out as image information;<br>
and<br>
image information processing means for analyzing an expression level of<br>
gene based on the image information and generating comprehensive expression<br>
level information of gene.<br>
82.	The gene interaction analyzing system as claimed in claim 81, wherein<br>
the input means receiving the expression profile information comprises image<br>
information reading means.<br>
83.	The gene interaction analyzing system as claimed in any one of claims<br>
78 through 82, comprising manual input means, provided as the input means, for<br>
modifying at least one of: the comprehensive presence information of gene of the<br>
hybrid individuals; the genetic marker information, and the genetic map<br>
constructing information.<br>
135<br></output></analyzing></output></analyzing></output></analyzing></chromosomal></bead></protein></examples></types></polypeptides></use></fabrication></target></types></immobilized></indirect-arrangement></basic></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNC1jaGVucC0yMDA1IGFic3RyYWN0LWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">3224-chenp-2005 abstract-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNC1jaGVucC0yMDA1IGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">3224-chenp-2005 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNC1jaGVucC0yMDA1IGNsYWltcy1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">3224-chenp-2005 claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNC1jaGVucC0yMDA1IGNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">3224-chenp-2005 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNC1jaGVucC0yMDA1IGNvcnJlc3BvbmRlbmNlcyBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">3224-chenp-2005 correspondences others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNC1jaGVucC0yMDA1IGNvcnJlc3BvbmRlbmNlcyBwby5wZGY=" target="_blank" style="word-wrap:break-word;">3224-chenp-2005 correspondences po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNC1jaGVucC0yMDA1IGRlc2NyaXBpdGlvbihjb21wbGV0ZWQpLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">3224-chenp-2005 descripition(completed)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNC1jaGVucC0yMDA1IGRlc2NyaXBpdGlvbihjb21wbGV0ZWQpLnBkZg==" target="_blank" style="word-wrap:break-word;">3224-chenp-2005 descripition(completed).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNC1jaGVucC0yMDA1IGRyYXdpbmdzLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">3224-chenp-2005 drawings-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNC1jaGVucC0yMDA1IGRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">3224-chenp-2005 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNC1jaGVucC0yMDA1IGZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">3224-chenp-2005 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNC1jaGVucC0yMDA1IGZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">3224-chenp-2005 form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNC1jaGVucC0yMDA1IGZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">3224-chenp-2005 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNC1jaGVucC0yMDA1IGZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">3224-chenp-2005 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNC1jaGVucC0yMDA1IGZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">3224-chenp-2005 form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNC1jaGVucC0yMDA1IHBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">3224-chenp-2005 pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="230630-preparation-of-3-pentenenitrile-from-1-3-butadiene.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="230632-a-ceramic-component-and-a-method-for-forming-the-same.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>230631</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3224/CHENP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>27-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>01-Dec-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>JAPAN SCIENCE AND TECHNOLOGY AGENCY</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>4-1-8, HONCHO, KAWAGUCHI-SHI, SAITAMA 332-0012,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>TAKEDA KAZUHIRO</td>
											<td>1160-171, ARIKI KURASHIKI-SHI, OKYAMA 710-0031,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SATO, KAZUYOSHI</td>
											<td>1169-40, ARIKI KURASHIKI-SHI, OKAYAMA 710-0031,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/JP04/06284</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-04-30</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>2004-093826</td>
									<td>2004-03-26</td>
								    <td>Japan</td>
								</tr>
								<tr>
									<td>2</td>
									<td>2004-093824</td>
									<td>2004-03-26</td>
								    <td>Japan</td>
								</tr>
								<tr>
									<td>3</td>
									<td>2003-126667</td>
									<td>2003-05-01</td>
								    <td>Japan</td>
								</tr>
								<tr>
									<td>4</td>
									<td>2004-093825</td>
									<td>2004-03-26</td>
								    <td>Japan</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/230631-array-having-substances-fixed-on-support-arranged-with-chromosomal-order-or-sequence-position-information-added-thereto by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:25:31 GMT -->
</html>
