<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/230713-polypeptide-of-the-human-immunoglobulin-superfamily by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:30:57 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 230713:POLYPEPTIDE OF THE HUMAN IMMUNOGLOBULIN SUPERFAMILY</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">POLYPEPTIDE OF THE HUMAN IMMUNOGLOBULIN SUPERFAMILY</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A polypeptide in isolated form belonging to a subfamily of the human Immunoglobulin Superfamily selected from the group consisting of: a) a polypeptide comprising the amino acid sequence of murine Confluency Regulated Adhesion Molecule 1 (CRAM-1) as depicted in SEQ ID NO: 19; b) a polypeptide showing at least 70% sequence homology over the entire length to the polypeptide of (a); c) a polypeptide comprising the amino acid sequence of human Confluency Regulated Adhesion Molecule 1 (CRAM-1) as depicted in SEQ ID NO:23; d) a fragment of (a) comprising (i) the V domain of murine CRAM-1 as depicted in SEQ ID NO: 19 from amino acids 53 to 115; (ii) the V domain of murine CRAM-1 as depicted in SEQ ID NO: 19 from amino acids 53 to 115 and the C2 domain of murine CRAM-1 as depicted in SEQ ID NO: 19 from amino acids 160 to 219; (iii) amino acids 1 to 159 of murine CRAM-1 as depicted in SEQ ID NO: 19; or (iv) amino acids 1 to 238 of murine CRAM-1 as depicted in SEQ ID NO: 19; and e) a fragment of (c) comprising (i) the V domain of human CRAM-1 as depicted in SEQ ID NO:23 from amino acids 53 to 115; or (ii) the V domain of human CRAM-1 as depicted in SEQ ID NO:23 from amino acids 53 to 115 and the C2 domain of human CRAM-1 as depicted in SEQ ID NO:23 from amino acids 160 to 219.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>«<br>
c c<br>
c<br>
I \ <br><br>
The present invention relates to the identification of a new subfamily of vascular adhesion molecules and to the modulation of the function of these molecules for the treatment of various diseases.<br>
Throughout embryonic and early postnatal development, endothelial cells proliferate and differentiate to form new blood vessels via vasculogenesis and angiogenesis. In adult organisms the endothelium defines the blood-tissue barrier and consists of non-cycling quiescent cells. These polarized cells are linked to each other by tight junctions and adherens junctions to form a continuous layer of cells. The fimctions of the endothelial layer consist in the maintenance of tissue homeostasis, fibrinolysis, coagulation, vasotonus, and leukocyte transmigration. All these properties are controlled by a fine tuning of the expression and the function of adhesion molecules.<br>
Pathological situations such as inflammation, tumor growth, wounding or angiogenesis lead to a temporary change of the number and function of adhesion molecules on the vascular endothelium and this results in altered homeostasis of the vessel. As an example, tumors increase the local concentration of angiogenic factors which induces a switch from non-cycling quiescent endothelial cells to proliferating endothelium. The angiogenic switch is induced by several factors including IL-8, epidermal growth factor (EGF), vascular endothelial growth factor (VEGF), soluble VCAM-1, basic fibroblast growth factor (bFGF), and tumor necrosis factor (TNF). As a result, endothelial cells of existing vessels degrade the extracellular matrix (ECM) and invade the surrounding tissue, which leads to vascularization of tumors.<br>
During the angiogenic switch the pattern of endothelial gene expression is modified. For example, treatment of endothelial cells with bFGF or TNF results in a fourfold increase in. v^s integrin expression, an adhesion molecule implicated in endothelial cell migration. In addition, the angiogenic switch modifies the inflammatory response of endothelium leading to an abnormal migration of leukocytes toward the tumors. Normally, leukocytes extravasate fi"om the blood by adhering to and migrating through endothelium. These mechanisms occur in a multistep process that involves selectins, integrins and Immunoglobulin Superfamily adhesion molecules.<br><br>
In tumor associated endothelium VCAM, ICAM, and selectins have been shown to be downregulated. The downregulation of these adhesion molecules may represent a mechanism by which tumors avoid invasion by cytotoxic cells of the immune system.<br>
It is the object of the present invention to search for new adhesion proteins of the Immunoglobulin Superfamily (Ig Sf), which are transcriptionally regulated in endothelium under the influence of tumors.<br>
It is a further object of the invention to define molecules derived from the new adhesion proteins for use in the treatment of various indications, such as for example tumors and inflanamation.<br>
In the research that led to the present invention an experimental murine model was used for the identification of transcripts regulated during the co-culture of an endothelial cell line with melanoma cells. To restrict the screening strategy to adhesion molecules of the Ig Sf, a new approach of RNA display termed "Targeted Differential Display" was developed. The novelty of the modified display technique resides in the use of only one set of degenerated primers. As will be demonstrated in the examples, it was surprisingly found that this leads to sufficient specificity.<br>
More specifically, partially degenerated primers (in the present case the level of degeneracy is between 2048 and 4096 different forms of primers within one set), designed to target the conserved sequences found in the C2 domains of Ig Sf members were used to drive the Polymerase Chain Reaction (PCR) based Targeted Differential Display technique (Samaridis &amp; Colonna (1997) Eur. J. Immunol. 27, 660-665).<br>
Based on this finding the invention provides a	method   for  the   specific<br>
identification of differentially expressed DNA-sequences comprising the use of Differential Display Reverse Transcription PCR, in which one set of partially or completely degenerated primers specific for the target gene is used. One major limitation of the conventional RNA display strategy is the lack of specificity of the method. In the aim to increase this specificity, the inventors in their search for other adhesion molecules used degenerated primers targeting the sequences encoding molecules with C2 domains. This was achieved by the alignment of C2 domains of several Ig Sf adhesion molecules, and the identification of a linear amino-acid consenstis, surroimding the cysteine residue participating to the C2 domain structure: Y-(RQYS)-C-x-A-S-N-X2-G. In a more general sense, this approach can also be used in the search<br><br>
for othet sequences in which the reverse translation of one or more of the most frequent consensus sequences is used to design the degenerated primers used for differential display.<br>
The method allowed the identification of a transcript, downregulated in endothelial cells by confluency in the presence of melanoma or carcinoma cells. The cDNA coded for a new molecule of the Ig Sf with unusual structural features, and was named CRAM-1 for "Confluency Regulated Adhesion Molecule". The recent description of a structurally related molecule, JAM, implicated in leukocyte transmigration, suggested the existence of a new family of adhesion molecules in which JAM and CRAM-1 were the prototypes. Sequence comparison with EST databases furthermore allowed the cloning of CRAM-2, a third member of this molecular family. Fig 1 shows the murine cDNA sequences encoding CRAM-1 and CRAM-2 proteins. In this application the names JAM and JAM-1, CRAM-1 and JAM-2 as well as CRAM-2 and JAM-3 are used interchangeably.<br>
The comparative tissue distribution of the transcripts encoding JAM, CRAM-1 and CRAM-2 showed a preferential expression of these molecules in endothelial and epithelial compartments suggesting a role in the maintenance of cell-cell contacts. These cell-cell interactions of quiescent endothelial cells regulate the vascular permeability, the cell cycle, and the leukocyte transmigration across endothelial wall.<br>
To further elucidate the function and the interplay of the three molecules, a molecular approach was used. To this end, chimeric molecules were constructed consisting of Flag-tag and Enhanced Green Fluorescent Protein (EGFP) sequences fused to a soluble or a membrane bound form of CRAM-1, CRAM-2 or JAM (summarized in Fig 2). When transfected into cell lines, the EGFP fusion products of CRAM-1 and JAM localized in cell-cell contacts, confirming a role of these molecules in the cell-cell communication. In contrast, CRAM-2 was more widely distributed on the cell surface. Moreover, the soluble construct of CRAM-1 blocked transendothelial migration of leukocytes in vitro, whereas soluble JAM showed only marginal effect. Altogether, these results suggested a central role of this new subfamily of adhesion molecules in the maintenance of vascular integrity and the function of the endothelial layer.<br>
Based on these findings the present invention provides for new means of counteracting medical indications like chronic inflammation and tumor development with reagents based on CRAM polypeptides.<br><br>
More in particular, the present invention relates to a polypeptide in isolated form belonging to a subfamily of the human Immunoglobulin Superfamily, which polypeptide shows at least 70% sequence homology with the amino acid sequence of the murine Confluency Regulated Adhesion Molecules 1 or 2 (CRAM-1 or CRAM-2) as depicted in Fig 3 upper and second row, respectively. Figures 4 and 5 show the alignment on amino acid level between mouse JAM-2 (CRAM-1) and JAM-3 (CRAM-2), respectively.<br>
The CRAM polypeptides found in the human or animal body are markers for growing cells. CRAM expression is upregulated in cells that are growing.<br>
Disclosed herein are two new murine polypeptides that are members of this family. Based on the sequence information of these polypeptides other members of the family can be identified by well known means such as PCR, crosshybridization on DNA libraries, crossreactivity of antibodies.<br>
The sequence information can be either the amino acid sequence or the nucleotide sequence encoding the amino acid sequence.<br>
More in particular, the invention thus relates to a corresponding polypeptide in humans, comprising essentially the amino acid sequence as depicted in Fig 6B or an amino acid sequence that is at least 70% homologous thereto.<br>
In addition to using the sequence information of the two CRAM proteins disclosed herein for identifying other members of the family in other species, like humans, the two proteins and their corresponding family members can also be used for the preparation of derived molecules, such as antibodies directed against the (poly)peptides of the invention, or recombinant equivalents of the proteins, optionally in soluble form, or peptides comprising at least part of the amino acid sequence of the polypeptides. Suitable parts of the amino acid sequence are especially the extracellular domains: VC2, and the membrane proximal cytoplasmic sequence: A-[Y,Q]-[R,S]-[R,K]-G-[C,Y]-F.<br>
In addition to antibodies and (poly)peptide type derivatives, the invention also relates to poly- or oligonucleotides having a sequence that encodes a complete polypeptide or part thereof, which polypeptide has an amino acid sequence that is at least 70% homologous to the amino acid sequence of the CRAM-1 or CRAM-2 proteins as disclosed herein. More in particular, the invention relates to nucleotide sequences that are at least 70%, preferably at least<br><br>
80%, more preferably at least 90%, most preferably essentially 100% homologous to the human DNA CRAM-1 sequence as depicted in Fig. 6.<br>
Such poly- or oligonucleotides may for example be RNA or DNA and can be primers, probes, antisense RNA etc.<br>
All such molecules can be used for modulating the function of the original polypeptides found in the human or animal body or for diagnosis.<br>
Angiogenesis in for example tumors can be inhibited with antibodies. They can be used as targeting molecules for cells bearing the CRAM polypeptides. The antibodies can act on their o\yn or can be coupled to other molecules, such as toxins, radioactive labels, fluorescent labels, enzymatic labels, photo-activatable labels, but also to liposomes, cells, etc.<br>
The labeled antibodies are particularly suitable for the diagnostic use of the antibodies, i.e. they can be utilized to locate angiogenesis in a growing tumor. In addition, antibodies coupled to toxins or radioactive molecules can be used to specifically kill the tumor from within by targeting to the (growing) vessels in the tumor.<br>
It was found that CRAM-type molecules were not detected in the normal vasculature except for lymphatics and the high endothelial venules in lymphoid organs such as lymph nodes and Peyer's patches. The advantage thereof is that the targeting of for example anti-CRAM antibodies can be highly specific to for example tumor cells thus avoiding undesirable side-effects.<br>
Moreover, the (poly)peptides may also bind the molecule on angiogenic vessels and by that stimulate or inhibit angiogenesis.<br>
Soluble (poly)peptide having essentially the same amino acid sequence as the CRAM polypeptides can be used in the treatment of inflammation reactions of the vascular endothelium. It was found according to the invention that the transendothelial migration of leukocytes can be inhibited by sCRAM-l-IG2Do or monoclonal antibodies against CRAM-1. This and similar molecules can therefore be used to quench or stimulate an immunological reaction such as found in inflammation.<br>
The specific expression of the molecule on vascular cells of HEVs in vivo which are specialized in lymphocyte migration argues for a stimulating effect of CRAM on lymphocyte migration or vascular permeability. This effect can thereof be due to the modulation of molecules normally involved in the sealing of the vascular bed (CRAM-1, CRAM-2, JAM,<br><br>
PECAM, VE-Cadherin). This finding is the basis for other applications of the invention involving the regulation of interendothelial junctions by delivering recombinant CRAM molecules (poly)peptides of the invention, or monoclonal antibodies against CRAM-1.<br>
Anti-CRAM antibodies can also be used to block cell-cell interactions in growing cells. This leads to disorganization of intercellular contacts which are normally required for the barrier function of blood vessels. This finding may be used to increase the permeability of growing vessels to increase the delivery of drugs to sites, such as growing tumors, post-menstrual uterus, etc. The disorganization of intercellular contacts may therefore be used to block the development of tumor cells bearing the antigen, such as angiomas (timiors originating fi-om vascular endothelium) or some rapidly growing carcinomas.<br>
For diagnosis use can be made of labeled antibodies but also of labeled oligonucleotides that are complementary to the CRAM DNA or mRNA found in the endothelial cells expressing the CRAM protein(s).<br>
The present invention will be further illustrated in the following example in which reference is made to the accompanying drawing, which show:<br>
Fig 1: Murine cDNA sequence encoding the CRAM-1 and CRAM-2 proteins. muCRAM-1 was subcloned in pcDNA3 vector and sequenced using Sp6 and T7 primers. muCRAM-2 was obtained as IMAGE clone from EST library (Ac: AA690843 and W80145) and was sequenced in the pT7T3-DPac vector using T7 and T3 primers.<br>
Fig 2: Schematic representation of the molecular tools used in the example. The structure and important residues of the new family are depicted in the upper top panel. The stars represent the putative phosphorylation sites in the cytoplasmic part of the three molecules. The second canonical Cys residue of the C2 domain is missing in the JAM sequence. Different chimeric molecules are represented below with the position and the surrounding residues of the fusion sites. Part of the molecules originating from JAM, CRAM-1 or CRAM-2 sequences are shown in white background.<br>
Fig 3: Alignment of CRAM-1 and CRAM-1 amino acid sequences. Gaps are indicated as dashed.<br>
Fig 4: Alignment between murine and human CRAM-1 (JAM-2).<br>
Fig 5: Aligimient between murine and human<br><br>
CRAM-2 (JAM-3).<br>
Fig 6: Nucleic acid sequence of human CRAM-1 (upper panel), complete amino acid sequence of human CRAM-1 (middle panel), and partial amino acid sequence of human CRAM-2.<br>
Fig 7: Targeted differential display using degenerated primers. (A): Nucleotide sequences of PCR primers encoding the sequences present in C2 Ig domains are shown. Two primers encode the same sequence due to the codons encoding Ser residue. The level of degeneracy is 4096 different forms for the primers encoding YRCXAS and 2048 forms for the others. (B): The display of radioactive PCR products obtained with the YYCXAS1 primers is shown. The lanes correspond to the display of PCR product run on cDNA obtained from the t-end endothelial cell line (lane t-end), the B16 melanoma cell line (lane B16), or the co-culture between the two cell lines (central lane). The arrow indicates the PCR product of interest obtained from downregulated transcript CRAM-1 under co-culture condition.<br>
Fig 8: (A) nucleotide and deduced amino acid sequence of Confluency Regulated Adhesion Molecule 1 (CRAM-1) cDNA. The putative hydrophobic signal peptide (first) and transmembrane region (second) are underlined. Predicted N-glycosylation sites (strikeout), cysteines likely to form disulfide bonds (brackets) and Ser/Thr/Tyr residues of possible phosphorylation sites (bold) are indicated. (B) Structural model for murine CRAM-1 protein. Extracellular part showing a VH and a C2 like Ig domain with two putative N-linked glycosylation sites. The arrow points to the region targeted by the partially degenerated primers (YYCXAS 1) used in the Targeted Differential Display.<br>
Fig 9: JAM, CRAM-1 and CRAM-2 murine protein sequence alignment. The identical residues are boxed in black and the homologous residues are shaded in gray. The overall identity is 36% between CRAM-2 and CRAM-1, 31% between JAM and CRAM-1 and 33% between JAM and CRAM-2; the respective homologies are 52%, 52% and 49%. The gaps are shown by dashes in the sequences. The canonical conserved residues (Cys and Trp) of the V and C2 domains are marked by an asterix.<br>
Fig 10: Expression of transcripts encoding JAM, CRAM-1 and CRAM-2 detected by RT-PCR in different lines (A) or detected by Northern blot in various tissues (B). (A): RT-PCR is achieved on cDNA originating from endothelial cell line treated by TNF (lanes 2 and 11 correspond to TNF treated t-end) or not treated (lanes 3, 4, 6, 7, 9, 12 correspond to b-end.5, e-<br><br>
end.2, t-end V"^L", t-end v'^^L"^, TME and t-end, respectively). Lanes 5 and 10 correspond to the tumor cell lines B16 (melanoma) and KLN205 (carcinoma). Lane 8 corresponds to the non transformed thymic epithelial cell line MTE4-14. Lane 1 is the positive control for JAM, CRAM-1 and CRAM-2 amplifications on the plasmids containing the cloned cDNAs. (B): Autoradiograph of P^^ probe hybridization to mouse Northern blot. The probes used for each hybridization are indicated left. The hybridization signals for JAM and CRAM-1 are detected at the size of 2kb.<br>
Fig 11: JAM-2 and JAM-1 localization to established cell-cell contacts. A: hnmunocytochemistry was performed on paraformaldehyde fixed TME cells with anti-JAM-2 (a) or anti-JAM-1 (b) antibodies. Arrows indicate the specific localization of the proteins to cell-cell contacts. Bar, lO^im. B: JAM-2-EGFP (a) and<br>
JAM-1-EGFP (c) chimeric molecules were specifically localized to cell contacts between transfected cells. The enrichment in EGFP recombinant proteins was not observed between transfected and non-transfected cells (arrowhead). Bar, 20jmi. C: Immunoprecipitation of JAM-2 after surface biotinylation of TME endothelial cells. Anti-PECAM (lane 1) and anti-JAM-1 (lane 2) antibodies were used as negative and positive controls respectively for the immunoprecipitation with CRAM-XIXH36 antibody (lane 3). Molecular weights are indicated on the right. D: Immunoprecipitation of EGFP recombinant proteins from CHO transfected cells. Anti-JAM-2 (lanes 2, 3, 6), anti-JAM-1 (lanes 1, 4, 5) were used to immunoprecipitate the biotinylated lysates from untransfected (lanes 1 and 2), JAM-1-EGFP (lanes 3 and 4), or JAM-2-EGFP (lanes 5 and 6) transfected CHO cells. Molecular weights are indicated on the right.<br>
Fig 12: Migration of splenocytes across monolayers of TNF-activated endotheliomas, in the presence or absence of the chemokine SDF-1. Three endotheliomas were used: wild-type t.end.l, or t.end.l transfected with the cDNA encoding for CRAM-1 or CRAM-2. Two monoclonal antibodies were tested for their ability to affect transmigration, F-26 or H-26, both rat IgGl monoclonal antibodies directed against murine CRAM-1.<br>
Fig 13: CRAM-1 regulation in function of confluency. The semi-quantitative PCR is driven using a mix of primers specific for HPRT and the CRAM-1 cDNAs. The PCR reactions are run on a 1.2% agarose gel and stained with ethydium bromide. Lanes 1, 2 and 3, correspond to 100, 50 and 10% confluency respectively. A weaker signal for CRAM-1 in the<br><br>
100% confluency (lane 1) is observed. The culture condition of the endothelial cell lines (t-end. 1 and TME) on their own or mixed with the tumor cell line KLN 205 is indicated.<br>
Fig 14: Northern blot analysis of JAM-2 (a), JAM-1 (b) or.-actin (c) transcripts in mouse tissues. Results on embryonic post-coi'tum (pc) and adult mRNA preparations are shown. The sizes of the hybridization signals are indicated on the right.<br>
Fig 15: Immunohistological analysis of JAM-2, JAM-1, ZO-1 and PECAM expression. Serial sections of kidney (a-d) or sections from mesenteric lymph node (e-1) were stained with anti-JAM-2 (a, e, i), anti-JAM-1 (b, f, j), anti-ZO-1 (c, g, k) or anti-PECAM (d, h, 1) antibodies. Each series of pictures (a-d, e-h, and i-1) were acquired with identical settings for the CCD.<br>
Fig 16: JAM-2 expression on endothelial cells. A: Cytofluorimetric analysis of JAM-2, JAM-1, and PECAM expression on endothelial cell lines (tEnd.l, eEnd.2 and TME) or squamous carcinoma cell line (KLN 205). Dashed profiles represent the negative controls obtained with an antibody directed against CD4. B: Cytofluorimetric analysis of JAM-2 on freshly isolated endothelial cells. Indicated organs were dissociated by CoUagenase/dispase digestion, stained with DilAc-LDL, CDS 1 and anti-JAM-2 or anti-JAM-1 as indicated. Histogram profiles were obtained by gating endothelial cell population positive for<br>
DilAc-LDL (FL-2) and CDS 1 (FL-3). Negative controls were obtained by omitting the primary mAbs against JAM-1 or JAM-2.<br>
Fig 17: (A): JAM-2-EGFP localization during cell-cell contact formation. Single fluorescence pictures were collected every 3 min for 1 hour during the monolayer formation of CHO cells transfected with<br>
JAM-2-EGFP. Pictures obtained during the first 18 min are shown. At time 0, asterisks identify the three cells present on the field. At time 6, 12 and 18 min, arrows highlight the relocahsation of JAM-2-EGFP to the newly formed cell-cell contact. (B): JAM-2-EGFP localization after wounding. Arrows indicate the wounded side and arrowheads highlight the membrane processes rich in JAM-2-EGFP. Elapsed time is indicated on the pictures. Bar, lO^m.<br>
Fig 18: JAM-2 expression decreases paracellular permeability. (A): Paracellular permeability was evaluated by FITC-Dextran diffusion across non transfected CHO cell<br><br>
monolayers, CHO cells transfected with Tac (huIL2R) or with the indicated EGFP fusion protein (JAM-1 or JAM-2). Transfection of JAM-2-EGFP or JAM-1-EGFP in CHO cells led to a significant decrease in paracellular permeability (57.8%V-4.9 and 70.8%V-3.6 respectively, p
Fig 19: Targeting of JAM-2-EGFP (A) and JAM-1-EGFP (B) to preexisting tight junctions. Confluent MDCK cells, stably transfected with JAM-2-EGFP (A), or JAM-1-EGFP (B), were stained with anti-occludin and anti-rabbit-Texas/Red. Series of pictures every 0.9nm from basal to apical levels are shown for EGFP fluorescence (a) or occludin staining (b). The basal level on the left was arbitrary defined such as the serial pictures comprise the tight junctional level on focus at +3.6 and +4.5^m (fourth and fifth pictures to the right).<br>
Fig 20: Effect of soluble recombinant molecules on leukocyte transendothelial migration. (A): Transmigration is expressed as a relative index and normalized on the values obtained on the non-treated<br>
t-end cell line (dashed line Index 1). Results obtained in the presence of 1 fxg sJAM-Ig2do (open squares) or in the presence of 1 |xg sCRAM-l-Ig2do (filled circles) are shown. Index is calculated as a mean of five independent transmigration experiments. (B): Phenotype of transmigrated cells is expressed as cell numbers calculated from the percentages obtained by Facs analysis following staining with anti CD3-FITC and anti B220-PE. The stars indicate the experimental points with a significant difference to the control.<br>
In the examples the terms JAM and JAM-1, CRAM-1 and JAM-2, as well as CRAM-2 and JAM-3 may be used interchangeably.<br>
EXAMPLE<br>
MATERIALS AND METHODS<br>
Cell lines<br>
The thymic (tEnd.l), and embryonic (eEnd.2) endothelioma cell lines (Williams et al., 1989, Cell 57:1053-1063) were provided by Dr. W. Risau and Dr B. Engelhardt (Max Planck Institute, Bad-Nauheim, Germany). The SV40 transformed lymph node endothelial cell line TME was provided by Dr A.Hamann (Harder et al., 1991, Exp Cell Res. 197:259-267). The<br><br>
squamous cell carcinoma KLN 205, the CHO, the MDCK, and the myeloma cell line Sp2/0, were obtained from the American Type Tissue Culture Collection (ATCC). All cells, except CHO, were grown in DMEM (Gibco BRL, Paisley, Scotland), supplemented with 10% PCS (PAA Laboratories, Linz, Ausfria), 2 mM Glutamine, 100 U/ml Penicillin and 100 U/ml Streptomycin (all Gibco BRL). CHO cells were grown in Nut.Mix.F-12 (HAM) medium supplemented as above. Adherent cells were detached by washing with PBS/ 0.15 mM EDTA followed by 5 min incubation in trypsin/EDTA at 37°C. Display, cloning and sequence analysis<br>
For co-culture experiments, 5x10^ t.End.l cells were grown together with 2.5x10'* B16 FIO melanoma cells for 64 hours in 10 cm tissue culture dishes. As control, 5x10^ t.End.l and 2.5x10^ B16 FIO cells were grown separately under the same conditions resulting in confluent monolayers after 64 hours. Total RNA was directly extracted in pefri dishes with Trizol reagent following manufacturer's instructions (Gibco BRL, Paisley, Scotland). The cDNA was prepared from 5 |j,g of total RNA, employing oligo-dT (16-mer) primer and Superscript Reverse Transcriptase (Gibco BRL, Paisley, Scotland). The quality and the quantity of cDNA were checked by running 27 cycles of PCR on 1 i^l of cDNA diluted 1:5, using primers specific for the housekeeping HPRT cDNA. Then the differential PCR was performed with the following degenerated pruners: ^'TAYAGNTGYNNNGCYTCYAA^', ^'TAYCRGTGYNNNGCYTCYAA^', and ^TAYTAYTGYNNNGCYTCYAA^' encoding for the most frequent amino acid sequences encountered in C2 domains: YRCXAS, YQCXAS, and YYCXAS. The PCR conditions consisted in using: 2 ^il of diluted cDNA; 2.5 |il of lOX Goldstar PCR buffer; 2 ^il of MgCb; 2 ^1 of degenerated primers 0,3 mM; 0.5 [il of dNTP 0.1 mM; 0.1 |xl of P^^ dATP lOmCi/ml (Amersham Pharmacia Biotech, Diibendorf, Switzerland); 15.65 111 H2O; 0.25 fil Goldstar Taq polymerase (Eurogentech, Seraing, Belgium).<br>
The parameters for the PCR were as follows: 45 sec at 94°C, 90 sec at 50°C, and 45 sec at 72°C repeated 40 times. Formamide/EDTA loading buffer was added and samples were denatured for 2 min at 94°C. The PCR products were then separated on a 6% polyacrylamide gel, and autoradiographed using Kodak OM-Mat. The band intensities were compared.<br>
Differentially expressed bands were cut from the dried polyacrylamide gel and fragments were retrieved by boiling and ethanol precipitation as previously described (Liang<br><br>
and Pardee, 1992, Science. 257:967-970). The PCR products were then reamplified using increased concentrations of dNTPs (0.2 mM instead of 2 |am) without P^^-ATP. The products of re-amplification were cloned into pGem-T Easy Vector (Promega Corp, Wallisellen, Switzerland) as described previously (Sambrook, Fritsch, and Maniatis; Molecular cloning; 2"' ed; Cold Spring Harbor Laboratory Press; 1989).<br>
Nucleic acid sequences of two independent clones were determined using the Thermo Sequence Fluorescent Labeled Primer Cycle Sequencing Kit (Amersham Pharmacia Biotech, Dtibendorf, Switzerland) and the LI-COR DNA Analysis System (MWG-Biotech GmbH, Ebersberg, Germany).<br>
Identification of JAM-3<br>
Sequence analysis and comparison were performed via the applications available on the ExPASy Molecular Biology Server i.e. Blast, Prosite, Swiss-Prot. Three different ESTs homologous to CRAM-1 were identified (Accession No. AA726206, AA052463 and AA175925). None of them encoded for a full length transcript and comprised the initiating ATG sequence. Therefore, the 5' coding sequence was obtained using the 5'RACE-PCR System for Rapid Amplification of cDNA Ends, Version 2.0 according to manufacturer's instructions (Gibco BRL, Paisley, Scotland).<br>
The three primers used were designed based on the EST sequences as follows: 5'-GAGGTACTTGCATGTGCT-3' for synthesis of the first strand, 5'-CGACAGGTGTCAGATAACA-3' and 5'-CACCCTCCTCACTCGT-3' for the two nested PCRs. The 5'RACE-PCR product was cloned into pGem-T Vector. To obtain the full length coding sequence for CRAM-1, the cloned 5'RACE-PCR product and the EST (accession No. AA726206) were digested with Hpal and NotI restriction enzymes and ligated into pGem-t vector. Cloning of full length CRAM-2 was based on the same strategy of sequence comparison and 5'RACE technique. The full-length cDNA encoding CRAM-2 was finally obtained fi"om ESTs accession numbers: AA690843 and W80145. These two clones differ by the length of the 3' untranslated region.<br>
Northern blot<br><br>
Total mRNA from cells or tissues was extracted using Trizol (Life technologies AG, Basel, Switzerland) according to manufacturer's instructions. Poly-A"*" mRNA was extracted from 250 ng total RNA with the Oligotex mRNA Purification Kit (Qiagen, Zurich, Switzerland). Embryonic Poly-A northern blot was purchased from CLONTECH (P.H Stehelin and Cie AG, Basel, Switzerland). The riboprobes were prepared from pcDNA3 vector (Invitrogen, Leek, Netherlands), and comprised the sequences encoding for the immunoglobulin domains of JAM-1 and JAM-2, or the ftiU-length coding sequence for.-actin. Hybridization was performed at 62°C in buffer containing 50% formamide. The blots were then washed twice (O.SxSSC, 0.1%SDS, 67°C), and autoradiographed on Kodak X-Omat at -80°C.<br>
Confluencv experiment<br>
The effect of endothelial cell confluency on JAM-2 mRNA levels was investigated. 2x10^ TME endothelial cells were cultured in 6, 10, and 15 cm diameter culture dishes to reach different levels of confluency after 64 hours ranging from 10 to 100%. The number of cells after 64 hours, checked by trypan blue exclusion and counting, was the same in all cases, and was not related to the surface area of the petri dish.<br>
Semi-quantitative PCR reaction or northern blotting were used to determine relative amount of franscript in the various conditions. For the detection of the JAM-2 transcript, the '-GACTCACAGACAAGTGAC-^' and ^'-CACCCTCCTCACTCGT-^' primer pair was used, giving a 750 bp amplification product. As internal confrol, the following primers specific for Hprt cDNA were used to amplify a 350 bp long fragment: ^'-GTTGGATACAGGCCAGACTTTGTTG-^'and ^'-GAGGGTAGGCTGGCCTATAGGCT-^'.<br>
Construction of expression vectors<br>
The sequence encoding EGFP was subcloned from pEGFP-1 vector (CLONTECH, P.H Stehelin and Cie AG, Basel, Switzerland) into pcDNA3 using Hindlll and NotI sites, therefore named pcDNA3/EGFP. The 3' resfriction sites, Hpal and Seal, found in the sequence encoding respectively the cytoplasmic domain of JAM-2 and JAM-1, were used to fuse the two sequences at the N-terminus of the EGFP in pcDNA3 vector (Invifrogen, Leek,<br><br>
Netherlands). The inserts encoding JAM-2 or JAM-1 were excised from pGemt or pRc/CMV using SacII/Hpal or Hindlll/Scal digestions, respectively.<br>
The coding sequences were then cloned in pcDNA3/EGFP vector digested with Agel, blunted by fill-in and further digested with Hindlll or SacII enzymes. This resulted in fusion sites at amino-acid positions DGV291 for JAM-2 and QPS285 for JAM-1. The transfection of CHO cells was performed as previously described (Ballestrem et al., 1998, J Cell Sci. 111:1649-1658).<br>
Stable transfectants used for permeability assays were selected by growing transfected CHO cells for two weeks in medium containing 1 mg/ml of G418. Resistant colonies were isolated and checked for EGFP fluorescence intensity by flow cytometry (FACScalibur apparatus, Becton Dickinson, Mountain View, CA) and fluorescence localization by microscopy (Axiovert, Zeiss, Oberkochen, Germany).<br>
Time-lapse video microscopy was performed using an Axiovert fluorescence microscope and Openlab software for image acquisition.<br>
The mammalian expression vector pcDNA 3 (Invitrogen, Leek, Holland) was modified by integrating the Flag-Tag (G. Wiedle, Dep. of Pathology, CMU, Geneva) coding sequences. Flag-Tag constructs containing coding sequences for the soluble forms of the JAM, CRAM-1 and CRAM-2 proteins were prepared by PCR. In all cases, the forward primers were designed to fit ATG initiation region. The reverse primers were designed in the sequences encoding the hinge region for the one Ig soluble form or in the sequence encoding the region between the C2 and transmembrane domains for two Ig domains soluble molecules. All reverse primers had 3' extensions containing a Xbal restriction site for direct in-frame cloning in the Flag-tag modified vector. Pfu DNA polymerase was employed in the PCR to avoid frequent mutations (Stratagene, La JoUa, CA, USA). The PCR fragments were then digested with Xbal and cloned into the pcDNA-3 Flag-Tag vector, digested by EcoRI, filled by Klenow and followed by an Xbal digest.<br>
Reagents and immunofluorescence analysis<br>
The following monoclonal antibodies were used: anti-PECAM (GC51, rat IgG2a; EA-3, rat IgG,) and anti-JAM (H202.106.7.4, rat IgGi) (Malergue et al., 1998, Mol Immunol. 35:llll-1119.;Pialietal., 1993, Eur J Immunol. 23:2464-2471).<br><br>
The panel of CRAM antibodies against JAM-2 was generated in the laboratory using standard techniques, and recombinant soluble molecule as immunogen (Aurrand-Lions et al., 1996, Immunity. 5: 391-405). The selected hybridomas were screened by ELISA for the production of antibodies recognizing specifically the recombinant soluble JAM-2 molecule. Positive clones were further tested on CHO cells transfected with JAM-2 cDNA (not shown).<br>
All CRAM antibodies are of the IgGi or IgG2a isotype except CRAM-25F24, which is of the IgG2b subclass. Antibodies were purified on Protein G sepharose columns (Pharmacia Biotech Europe, Dubendorf, Switzerland) according to the manufacturer instructions. CRAM-19H36 mAb was used for immunoprecipitation, whereas CRAM-18F26 was the reagent used for immunohistochemistry. Similar results were obtained with CRAM-4H31 andCRAM-17D33 mAbs.<br>
Immunofluorescence analysis was performed using secondary reagents coupled to FITC or Texas Red (Jackson Immunoresearch, Milan AG, La Roche, Switzerland) for cytofluorimetry and immunohistochemistry, respectively.<br>
For immunohistochemistry, samples were fixed 5 min with cooled (-20°C) methanol. Samples were rehydrated in PBS, gelatine 0.2%o, Tween 20 0.05%, incubated overnight with the primary antibodies before washing, and revealed with the appropriate secondary reagent coupled to Texas Red. For the analysis of fresh endothelial cells, dissociation of freshly dissected tissues was performed using coUagenase/dispase digestion, according to established procedures (Kramer et al., 1984, / Cell Biol. 99:692-698). The dissociated cells were stained for 2 hours at 37^ with Dil-Acetylated LDL (Molecular Probe Europe BV, Leiden, Netherlands) before staining with anti CRAM-19 and goat anti rat-FITC probe. After three washes, cells were stained with biotinylated anti-CD31 (Pharmingen) and streptavidine Red 670 (Life technologies AG, Basel, Switzerland).<br>
JAM-1 or JAM-2 expression was analyzed on cells positive for the two endothelial cell markers: Acetylated-LDL (FL-2) and CD31 (FL-3). Negative controls were obtained by omitting primary antibody.<br>
Immunoprecipitations<br>
Immunoprecipitations were performed as previously described (Aurrand-Lions et al., 1997, Cellular Immunology. 176:173-179) using 10 mM Tris-HCl buffer pH 7.4, 150 mM<br><br>
NaCl, 0.5% Triton XlOO, 0.5% NP40, protease inhibitor cocktail (Roche Diagnostics Ltd, Rotkreuz, Switzerland) for lysis. After immunoprecipitation, SDS/PAGE, and transfer to nitrocellulose membrane, the biotinylated proteins were revealed using streptavidin coupled to peroxidase (Jackson Immunoresearch) and ECL (Amersham Pharmacia Biotech, UK).<br>
Permeabilitv assays<br>
Permeability was measured using Transwell chambers (6.5 mm diameter, PC filters, 0.4 |xm pore size, Costar Corp). In brief, 1x10'* transfected or non-transfected CHO cells were cultured to confluency on filters previously coated for 30 min with 0.2% gelatin. After 5 days, the medium was changed for prewarmed Nut/F12 medium without PCS (500(il in the lower chamber and 200 |j,l in the upper chamber), FITC-dextran (MW 38.900, Sigma Chemical Co) was added in the upper chamber at 1 mg/ml final concentration.<br>
After 1 hour, chambers were removed and fluorescence was read directly in the lower chamber using Cytofluor II. The mean fluorescence intensity of five independent chambers was calculated and compared using Statview software and t-test unpaired comparisons. To normalize experiments, the value of mean fluorescence intensity obtained with wild type CHO cells was taken as 100%.<br>
Transfection and purification of soluble molecules<br>
Transient transfection of 293 T, Bosc 23 or stable transfection of CHO cells with the soluble IglDo and Ig2Do Flagtag/pcDNA-3 constructs were carried out using Lipofectamine Reagent according to manufacturer's instructions (Gibco BRL, Paisley, Scotland). Following transfection supematants were collected every two days during a ten days period. M2 Beads (Kodak, New Haven, USA) covalently linked to anti-Flag antibody were washed twice with PBS containing a Protease Inhibitor Mix (Boehringer Mannheim, Germany). The beads were then incubated at 4'C for 3 hours with supernatant fi-om the transfected cells. After five washes with PBS containing protease inhibitors, a column was packed with the beads, and recombinant molecules were eluted with 10 mM glycine buffer pH 3.4 according to the manufacturer. The eluted fractions containing the recombinant proteins were then concentrated on Centricon-10 (Millipore) and dialyzed against PBS.<br><br>
Final protein concentration was determined using the Micro BCA assay (Biorad). The purified products were then submitted to a polyacrylamide SDS gel electrophoresis followed by coomassie blue staining to analyze their purity.<br>
Transmigration assays<br>
Leukocyte transmigration across endothelial cells was performed as previously described (Wheerasinghe et al., 1998, J. Cell Biology. 142: 595-607). Briefly, 1x10^ t-end cells were cultured for two days in transwell units (polycarbonate filters, 5 \im pore size, costar) in the presence of 1 ^ig of Ig soluble recombinant molecules: sJAM 2do or sCRAM-1 2do. After two days, 1x10^ leukocytes obtained fi"om lymph nodes and Peyer's patches were added to the upper compartment, and the number of transmigrated cells was monitored during the experiment every hour. After 4 hours, transmigrated cells obtained from five independent wells were pooled and submitted to cytofluorimetric analysis for B-and T-cell markers B220 and CD3. Results were obtained using a Facscalibur machine and the Cell-Quest analysis program (Becton-Dickinson).<br>
For a transmigration assay with splenocytes, 3x10^ endothelial cells were seeded in transwell units (polycarbonate filters, 8 ^im pore size, costar) allowing the cells to form a monolayer over 18 hours. Medium in the upper compartment was removed and 1x10^ leukocytes in 100 ^il, freshly prepared from spleen by FicoU centrifugation, were added to the upper compartment.<br>
SDF-1 was added to the medium (final concentration: 40 nM) in the lower compartment to establish a chemokine gradient between lower and upper compartments. For the experiment with antibodies, purified antibodies 18-F26 or 19-H36 were added at the concentration of 10 |xg/ml in the upper compartment with splenocytes. After 4 hours, transmigrated leukocytes (in the lower compartment) were harvested and counted. Results were expressed as % of input.<br>
RESULTS<br>
Targeted Differential Display<br>
The regulation of genes in endothelial cells depends on their environment. The present invention was directed to the identification of genes that undergo regulation upon the contact of endothelium with tumor cells. For this purpose, an in vitro assay was developed using<br><br>
the co-cUlture of melanoma tumor cells (B16) with an endothelioma cell line (t-end). Total RNA extracted from the mix was used as template to prepare cDNA submitted to a differential PCR screen. The cDNA obtained from the endothelial or melanoma cells cultured on their own were used as controls. The three different patterns were compared to identify the transcripts regulated by the co-culture condition. To limit the analysis to the sequences encoding for cell surface molecules of Ig superfamily, partially degenerated primers were used that target the sequence surroimding the C-terminal cysteine of C2 domains in Ig superfamily molecules. The most reproducible pattern of PCR products was obtained using primers that encode the sequence YYCxAS 1 (Fig 7A). This improved method of RNA display technique was named TDD for "Targeted Differential Display".<br>
In repeated experiments of TDD, sixteen differentially expressed genes were identified. Following cloning, nucleotide and amino acid sequence analysis, three of the sixteen PCR products were possible candidates encoding for unknown members of the Ig superfamily. One of the three candidates (CRAM-1) was chosen for further investigation. When grown separately, t-end. 1 endothelial and B16 melanoma cells expressed high levels of the CRAM-1 transcript. However, under co-culture conditions the level of CRAM-1 expression was down regulated (Fig 7B). Translation of a 350 bp long fragment corresponding to CRAM-1 showed the amino acid sequence YYCxAS indicating the endings of an Ig C2 domain followed by an open reading frame (ORF) containing an hydrophobic sfretch of 18 amino-acids that signed a fransmembrane region.<br>
CRAM-1, a member of the Immunoglobulin Superfamily<br>
Sequence comparison between the PCR product sequence and nucleotide databases revealed homologous and identical sequences in mouse ESTs databases. The presence of ESTs indicated that the PCR product corresponded to a sequence expressed in vivo. Three ESTs were found to contain a 300 bp long sequence at their 5' end, which was identical to 300 bp in the TDD product. The 3' ends of each EST contained a poly-A tail. In total the ESTs were 1270 bp in length and corresponded to the 3' end of the CRAM transcript. Since the 5' end of the transcript was missing in the EST cDNA clone, it was obtained by 5'RACE-PCR. The resulting 1980 nucleotide long full length coding sequence of the postulated CRAM-1 cDNA is shown in Fig 8A. There was a sfrong consensus site (GACATGG) for translation initiation 16<br><br>
bp dowiistream from the 5' end, followed by a single ORF predicting a protein of 310 amino acid. The 31 amino-acid region subsequent to the potential initiating methionine, was characteristic of a signal peptide. The cleavage site was predicted to be at Ala 31-Val 32.<br>
The putative structure of the murine CRAM-1 protein is shown in Fig 8B and consists of an extracellular region with a variable heavy chain and a constant type 2 like immunoglobulin domain (Pfam, The Sanger Centre and Blast) with two potential N-linked glycosylation sites (aa 104 and 192). The hydrophobicity analysis (Tmpred, ISREC) predicted a transmembrane region between positions 242-260. The postulated cytoplasmic domain consisted of 49 amino acids and contained a number of highly conserved Ser/Thr and Tyr phosphorylation sites (Fig 8A, residues in italic). The search of known patterns with the Prosite program identified the motifs SSK/SYK as protein kinase C, SKQD/TSEE as CK2 and KQDGESY/KHDGVNY as Tyrosine kinase phosphorylation signatures.<br>
JAM, CRAM-1 and CRAM-2 define a new subfamily<br>
Several proteins showed high homology to CRAM-1. Two members of the Ig Superfamily: Human A33 antigen and part of the mouse neural cell adhesion molecule, N-CAM were found to have 41% and 46% homology with CRAM-1 respectively. JAM, another member of the Ig Sf, had a similar structure as CRAM-1 with 34% amino acid sequence identity, and 54% homology. The significant identity between JAM and CRAM-1 was used to find a third closely related sequence in EST databases, namely CRAM-2. The identity between the three molecules suggested the existence of a new subfamily of molecules in the Ig superfamily (Fig 9). The homology concerned not only the overall structure of V and C2 domains (C54 to CI 18 and C147 to C235 in Fig 9) but also sequences inside the cytoplasmic domains. Interestingly, the most divergent regions between the three molecules were found at the beginning of the V domain (position 40 to 60) and in the proximal cytoplasmic part (position 280 to 300). The functions of these two regions correspond to sequences involved in ligand binding and signal transduction in other members of the Ig superfamily suggesting a role of JAM, CRAM-1 and CRAM-2 in cell-cell communication.<br>
Tissue Distribution of JAM. CRAM-1 and CRAM-2 mRNA<br><br>
Expression of the three transcripts in cells of different origin was detected using RT-PCR. All endothelial, epithelial and most tumor cell lines tested, were positive, although at varying degrees for the different transcripts (Fig lOA). The highest expression level for CRAM-1 was found in the SV40 transformed HEV cell line TME (lane 9), and in the embryonic endothelial cell line e-end 2 (lane 4). The CRAM-2 and JAM transcripts showed a more restricted distribution, and were found in adult endothelial cell lines together with the CRAM-1 transcript (lanes 3, 7, 9 and 12). Notably, JAM and CRAM-2 transcripts were strongly downregulated by TNF treatment of endothelial cells whereas the level of CRAM-1 transcript remained unchanged (lanes 2 and 11). Interestingly, an embryonic endothelial cell line (lane 4) or an adult endothelial cell line representing an angiogenic variant of t-end (lane 6, failed to express JAM or CRAM-2.<br>
The tissue distribution of JAM-2 transcript was explored by northern blotting and compared to JAM-1 (Fig 14). The JAM-2 transcript was 2 kb long, highly expressed in embryonic tissue, and in Peyer's patches, lymph nodes, kidney, and testis of aduU animals. A putative splice variant of 1.8 kb was detected in testis. Expression of JAM-2 transcript was low in lung, liver, spleen, and thymus. The relative abundance of JAM-1 and JAM-2 were compared during embryogenesis: the mRNA encoding JAM-2 was detectable as early as day 7.5 post coitum, whereas JAM-1 mRNA was not detected at all during embryogenesis.<br>
These results suggest that CRAM-1 is widely expressed during embryogenesis and shows a restricted expression to epithelial or endothelial compartments in adult tissues. This is in agreement with the idea that it plays a role in the establishment and the maintenance of the polarized organization of cells.<br>
JAM-2, a 45kD protein, depending on homophilic interactions for its localization to cell-cell contacts<br>
Since the HEV derived cell line TME expressed the highest level of JAM-2, this endothelial cell line was used to further study the subcellular localization of JAM-2, and to compare to that of JAM-1. The localization of the JAM-2 protein on the surface of the endothelial cells was restricted to cell-cell contacts (Fig 11 A, a). The staining for JAM-2 was<br><br>
weaker than that observed for JAM-1 and less prominent in the membrane extensions between cells.<br>
Then it was investigated whether JAM-2 present at cell-cell contacts interacted homophilically with<br>
JAM-2 or whether it interacted heterophilically with another molecule on the neighboring cell. For this purpose the JAM-2 protein was fused to green fluorescent protein (JAM-2-EGFP), and the construct transfected in CHO cells. When CHO cells transfected with JAM-2-EGFP cDNA reached confluency, JAM-2 was only observed in<br>
cell-cell contacts where both cells expressed the protein (Fig IIB), whereas the contacts between expressing and non-expressing cells were devoid of JAM-2 (Fig IIB, a, indicated by arrow heads). The same result was obtained when cells were transfected with the chimeric molecule JAM-1-EGFP (Fig IIB, b). This indicated that either JAM-2, or JAM-1, needed homophilic interactions to be localized at cell-cell contacts.<br>
To characterize JAM-2 biochemically, immunoprecipitations of JAM-2 or EGFP chimeric proteins, expressed by TME cell line or by transfected CHO cells, respectively (Fig lie and D) were performed. The anti-JAM-2 antibody, CRAM-19H36, immunoprecipitated a single band of 45kD from TME cells lysate (Fig IIC, lane 3).<br>
The apparent molecular weight was identical under reducing or non-reducing conditions (not shown) and corresponded to the predicted molecular weight deduced from the amino-acid sequence of JAM-2, each<br>
N-glycosylation site accoimting for 5 kDa. Immunoprecipitation of JAM-1 using H202-106 anti-JAM specific antibody resulted in a single band of lower molecular weight (~42kD, lane 2) that excluded a possible cross-reactivity between anti-JAM-2 and anti-JAM-1 antibodies.<br>
Immunoprecipitations of recombinant JAM-1-EGFP or JAM-2-EGFP proteins after surface biotinylation resulted in single broad bands of 70 and 73kD respectively, indicating that the molecules were expressed on the surface of CHO transfected cells (Fig IID, lanes 4 and 6). These molecular weights were expected since EGFP has a molecular weight of 28kD.<br>
Tighmess and leukocyte migration<br><br>
In order to understand how molecules influence the integrity of the endothelial cell monolayer and how they regulate the function of the vascular endothelium, leukocyte trans-endothelial migration assays were performed in the presence of recombinant soluble JAM or CRAM-1. Endothelial cells were cultured for two days in the presence of sJAM-Ig2Do or sCRAM-l-Ig2Do. The monolayer integrity was not affected during this period, probably due to the molecular redundancy of the mechanism of cell-cell contact formation. The transmigration assay was performed in the presence of 1 jj,g of recombinant soluble molecules. As shown in Fig 20A, the number of transmigrating cells was poorly affected by the presence of sJAM-Ig2Do (open squares) during the first three hours. After four hours, the number of transmigrated cells increased when compared to the control (dashed line). In contrast, the presence of sCRAM-l-Ig2Do (closed circles) strongly reduced the number of transmigrating cells.<br>
Since the leukocyte populations were heterogeneous, it was evaluated if sCRAM-l-Ig2Do acted on a specific leukocyte subpopulation or whether transmigration was blocked without specificity. For this purpose, the transmigrated cells were labeled for the lymphocyte markers CDS and B220 (Fig 20B).<br>
Remarkably, sCRAM-l-Ig2Do specifically blocked the transmigration of non-lymphoid leukocytes, i.e. myeloid lineage cells (central panel, dashed columns). In contrast, sJAM-Ig2Do poorly increased the number of transmigrating T cells (left panel, white column) without any effect on other cell subpopulations.<br>
Furthermore, when endothelioma cells transfected with CRAM-1 were used for transmigration assay, an increased transmigration was observed (Fig 12), whereas the transfection of CRAM-2 was without effect on the transmigration. When SDF-1 was added to the assay, the leukocyte transendothelial migration reached 20%. This was partially blocked by monoclonal antibodies against CRAM-1.<br>
These results indicate that the engagement of CRAM-1 between endothelial cells may regulate the function of the endothelial layer. It could be expected that the molecules of this family will become a barrier when endothelial cells reach confluency. To this end, the regulation of CRAM-1 transcripts was explored in endothelial cells under different culture conditions.<br>
CRAM is downregulated bv high confluency<br><br>
Since the transcript that encodes CRAM-1 is not regulated by TNF, but is downregulated when the endothelium was co-cultured with tumor cells, a confluency assay was used to further explore this regulation. Under low confluency, the cells were actively cycling and CRAM-1 interactions did not occur whereas under high confluency the cells divided less and CRAM-1 was engaged. The level of CRAM-1 mRNA expression was determined under various cell densities by semi-quantitative RT-PCR. As shown in Fig 13, the expression level of CRAM-1 transcripts decreased when confluency was reached (lanes 1, 2, 3 in Fig 13 correspond to 100, 50, and 10% confluency respectively). This effect was hardly detectable with the t-end cells but was more pronounced with the TME cell line which highly expressed CRAM-1, The downregulation of CRAM-1 in endothelia was also enhanced when the endothelial cells were co-cultured with KLN 205 carcinoma cells which themselves do not express CRAM-1. This confirmed the link between CRAM-1 expression and the cell cycle since tumor cells were described to increase the growth rate of endothelial cells upon contact. It is noteworthy that the results obtained with KLN 205 carcinoma cells was identical to the one used in our original screening strategy with the B16 melanoma tumor. This indicates a general mechanism by which tumors affect endothelial behavior.<br>
JAM-2 is highly expressed during embryogenesis, and restricted to HEVs and endothelial cells subpopulations in adult tissues<br>
To better define the tissue distribution of JAM-2, immunohistological analysis was performed on kidney and mesenteric lymph node sections (Fig 15), which expressed the highest levels of JAM-2 transcript. In the cortical region of the kidney, a specific staining of intertubular structures was detected with anti-PECAM (GC51) or anti-JAM-2 (CRAM-XVIIIF26) mAbs, whereas anti-ZO-1 or anti-JAM-1 stained predominantly the tubular epithelial cells (not shown).<br>
The inventors therefore focused their attention on vascular structures, which dip down into the medulla and correspond to radial veins or vasa recta endothelial structures. For this purpose, serial sections were performed and the vascular structures identified with the anti-PECAM staining (Fig 15d). On the equivalent region of neighboring sections, linear interendothelial stainings were detected with anti-JAM-2, anti-JAM-1 or anti-ZO-1 (Fig 15 a, b and c, respectively). On sections of mesenteric lymph node, typical staining of high endothelial<br><br>
venules'(HEVs) was obtained with anti-JAM-2 mAb (Fig 15e). The HEVs were also found to express JAM-1, ZO-1 or PECAM (Fig 15 f, g and h), with subtle differences in the subcellular localization of the stainings (Fig 15 e-h, insets). In the cortical area of the mesenteric lymph nodes, a typical staining of the subcapsular sinuses was observed with all antibodies (Fig 15 i-I), corresponding to the staining of afferent lymphatic vessels. Thus, the staining with the CRAM-18F26 anti-JAM-2 mAb is restricted to certain endothelial cells or to structures closely associated with vasculature.<br>
In order to clarify whether endothelial cells staining accounted for the pictures shown in Fig 15, cytofluorimetry analysis of JAM-2 expression was performed on various cell lines or freshly isolated endothelial cells from dissociated tissues. Endothelial cell lines (tEnd.l, eEnd.2 and TME) expressed low levels of JAM-2 on the cell surface and variable levels of JAM-1 (Fig 16A).<br>
Cytometric analysis of freshly isolated endothelial cells was performed by triple staining of cell suspensions obtained after coUagenase/dispase organ dissociation. Endothelial cells were identified by gating cells stained with both PECAM/CD31 and Acetylated-LDL (Voyta et al, 1984, J. Cell Biol. 99:2034). Staining for JAM-2 or JAM-1 on this double positive cell population is shown in Fig 16B. In kidney and Peyer's patches, all the isolated cells positive for CD31 and Acetylated-LDL were also stained for JAM-2, meaning that, at least in these organs, endothelial cells expressed JAM-2 in vivo. When the staining was performed on cells obtained from lymph node, JAM-2 expression was only found on a cell subpopulation, reflecting a possible heterogeneity of endothelial cell phenotypes within this tissue.<br>
Altogether, the results of cytometric and immunohistochemical analysis show that JAM-2 is co-expressed with JAM-1 by endothelial cells of kidney, Peyer's patches and lymph nodes.<br>
Dynamic localization of JAM-2 to cell-cell contacts<br>
To dissect the mechanism by which JAM-2 was specifically localized to cell-cell contacts; time lapse video microscopy was used. The CHO cells, stably transfected with the fluorescent chimeric molecule, were trypsinized and plated into chamber slides for imaging. After cell spreading, surface expression of JAM-2-EGFP was not uniform, but was rather clustered at cell-cell contacts (Fig 17A, cells depicted by asterisks). During the formation of<br><br>
new cell-cell contacts, relocalization of JAM-2-EGFP to cell junctions was observed and an intense fluorescence signal was detected at the novel contact point between the cells forming the new cell-cell contact (arrows). The chimeric protein was enriched in the membrane protrusions between contacting cells, leading to the "zipper like" pictures seen after 12 or 18 min. Interestingly, the localization of JAM-2 at the primary cell-cell contacts was not lost during the formation of the new membrane contact (see upper left comer cell contacts). This finding indicated that JAM-2-EGFP was specifically relocalized to the new cell contact, and that, upon engagement, its localization was stable.<br>
To further address the requirements for JAM-2 localization, time lapse video microscopy was performed after wounding the cell monolayer (Fig 17B). Cells at the wounded edge maintained JAM-2 at their intact contact sites (arrowhead), but lost JAM-2 localization at the wounded side (arrows), indicating that JAM-2 engagement by a ligand on the opposing cell was necessary to maintain its membrane localization. Over a period of 90 min following wounding, cells bordering the wound began to migrate into the wounded area. Interestingly, these cells maintained contacts with neighboring cells via membrane protrusions that were brightly fluorescent, i.e JAM-2 positive (arrowhead). These results supported the hypothesis that JAM-2 homophilic interactions may play a role in the establishment or maintenance of cell-cell contacts.<br>
JAM-2 increases monolayer tightness and participates to tight junctional complexes<br>
Since a number of molecules participating in cell-cell connection, have been shown to regulate the paracellular permeability of cell monolayers, it was tested whether JAM-2 could also affect this function. Transfection of JAM-2-EGFP reduced the paracellular permeability to FITC dextran and improved sealing of CHO cell monolayers by 42.5%; whereas transfection of the unrelated molecule Tac (IL2R), did not significantly reduce the paracellular permeability of CHO transfected cells (Fig 18). The transfection of JAM-1-EGFP also reduced the paracellular permeability of CHO transfected cells monolayer.<br>
These results raised the question whether the chimeric molecules were able to participate to a subcellular specialized compartment such as tight junctions in polarized epithelial cells.<br><br>
To answer this question, the EGFP chimeric proteins were transfected in MDCK cells, and their subcellular localization compared with that of the tight junctional marker: occludin. As shown in Fig 19A, when serial pictures every 0.9^m were analyzed for EGFP fluorescence and compared to occludin staining,<br>
JAM-2-EGFP was specifically enriched in cell-cell contacts at the level of tight junction. At the basal level (left), intracellular dots of EGFP fluorescence were observed. A similar analysis (Fig 19B) of MDCK cells transfected with JAM-1-EGFP showed a similar<br>
co-localization with occludin. Nevertheless, the distribution of JAM-1-EGFP fluorescence was less continuous than that observed for JAM-2-EGFP at the level of tight junctions.<br>
DISCUSSION<br>
This example reports the use of a new screening strategy to identify regulated transcripts encoding members of the Ig superfamily of adhesion molecules. Described here is the cloning with this method of the new molecule CRAM-1 as a regulated transcript. The regulation observed in endothelial cells grown in the presence of tumors is confirmed by semi¬quantitative RT-PCR and is shown to be dependant on the growth phase of the cells. Due to differential expression under changing cell confluency conditions, the name CRAM-1 for "Confluency Regulated Adhesion Molecule-1" was adopted.<br>
Also described herein is a closely related sequence to CRAM-1 named CRAM-2. CRAM-1 and -2 represent the prototypes of a new subfamily of adhesion molecules which also includes the recently described molecule JAM (Chretien et al., 1998, Eur. J. Immunol. 28, 4094-4104; Malergue et al., 1998, Mol. Immunol. 35, 1111-1119).<br>
CRAM-1 and JAM are preferentially expressed by endothelial and epithelial tissue at the cell-cell contacts and confer special properties to polarized layers. The effect of recombinant soluble molecules in a transendothelial migration assay and the regulation of JAM, CRAM-1 and CRAM-2 show that these three molecules play an important role in the maintenance of vascular physiology.<br>
The new screening strategy, named Targeted Differential Display (TDD), has proved to be an efficient technique in selectively amplifying cDNA of interest. TDD successfully exploited the use of partially degenerated primers to confer selective targeting to the conserved region, Y(Y/Q/R)CXAS, of C2 like Ig domains. Repeated experiments lead to<br><br>
reproducible display patterns. Out of 16 differentially expressed transcripts, three correspond to * " genes with significant homology to conserved Ig sequences. This increase in specificity manages to overcome the major difficulties in the already known techniques of classical RNA fingerprinting and differential display. RNA fingerprinting has long been used for the identification of differentially expressed genes. However, due to the sequence specific primers employed, this method detects only the transcripts of selected and already known proteins. On the other hand, RNA display employs random primers and involves the non-specific amplification of transcripts. The aim in this case is to pinpoint any differences in mRNA levels between two biological systems, which are submitted to comparison. TDD is an advanced screening method that combines the specificity of RNA fingerprinting with the degeneracy of Differential RNA Display resuhing in selectivity. Due to the targeting of related transcripts, this technique significantly reduced the time needed for screening. The identification of new members of specific protein families, therefore, becomes possible. This is a substantial improvement of the reported non-specific screening strategies.<br>
These common features were used to construct recombinant proteins in order to study the functions of JAM, CRAM-1 and CRAM-2. In the present example, effects of sJAM-Ig2Do and sCRAM-l-Ig2Do are described in an in vitro transmigration assay. Specific blocking effects on the migration of myeloid cells could be observed with sCRAM-l-Ig2Do whereas sJAM-Ig2Do showed only a small effect on lymphocytes.<br>
JAM and CRAM-2 transcripts showed a similar tissue distribution and regulation of expression under the influence of TNF, indicating that they act by similar physiological mechanisms. In contrast, CRAM-1 transcripts are not regulated by TNF but rather by the rate of proliferation or the density of endothelial cells. In fact, overexpression of CRAM-1 transcripts in cycling cells and its downregulation in quiescent cells indicate that this molecule participates in the establishment of a continuous monolayer. Its function in confluent monolayers of cells is the maintenance of the endothelial cell layer and the related properties. Since different leukocyte populations have to migrate to the site of immune response, it is thought that non-lymphoid cells migrate via a CRAM-1 dependant mechanism, whereas lymphoid cells migrate via JAM or CRAM-2. In this case the immune response can be modulated by using combinations of different soluble recombinant molecules.<br><br><br>
WE CLAIM :<br>
1.	A polypeptide in isolated form belonging to a subfamily of the human Immunoglobulin<br>
Superfamily selected from the group consisting of:<br>
a)	a polypeptide comprising the amino acid sequence of murine Confluency Regulated Adhesion Molecule 1 (CRAM-1) as depicted in SEQ ID NO: 19;<br>
b)	a polypeptide showing at least 70% sequence homology over the entire length to the polypeptide of (a);<br>
c)	a polypeptide comprising the amino acid sequence of human Confluency Regulated Adhesion Molecule 1 (CRAM-1) as depicted in SEQ ID NO:23;<br>
d)	a fragment of (a) comprising<br>
(i) the V domain of murine CRAM-1 as depicted in SEQ ID NO: 19 from amino acids 53 to 115;<br>
(ii) the V domain of murine CRAM-1 as depicted in SEQ ID NO: 19 from amino acids 53 to 115 and the C2 domain of murine CRAM-1 as depicted in SEQ ID NO: 19 from amino acids 160 to 219;<br>
(iii)    amino acids 1 to 159 of murine CRAM-1 as depicted in SEQ ID NO: 19; or (iv)    amino acids 1 to 238 of murine CRAM-1 as depicted in SEQ ID NO: 19; and<br>
e)	a fragment of (c) comprising<br>
(i) the V domain of human CRAM-1 as depicted in SEQ ID NO:23 from amino acids 53 to 115; or<br>
(ii) the V domain of human CRAM-1 as depicted in SEQ ID NO:23 from amino acids 53 to 115 and the C2 domain of human CRAM-1 as depicted in SEQ ID NO:23 from amino acids 160 to 219.<br>
2.	The antibody directed against the polypeptide as claimed in claim 1.<br><br>
3.	The antibody as claimed in claim 2 coupled to another molecule selected from toxins,<br>
radioactive labels, fluorescent labels, enzymatic labels, photo activatable labels, liposomes,<br>
drugs and cells.<br>
4.	The poly- or oligonucleotide encoding any of the polypeptide sequences as claimed in<br>
claim 1.<br>
5.	The pharmaceutical composition comprising as an active ingredient a polypeptide as<br>
claimed in claim 1 or an antibody as claimed in claim 2 or 3<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDEtMTIyNy1DSEUgIENPUlJFU1BPTkRFTkNFIE9USEVSUyAwMS0wOC0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">IN-PCT-2001-1227-CHE  CORRESPONDENCE OTHERS 01-08-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDEtMTIyNy1DSEUgIEZPUk0tNiAgMDEtMDgtMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">IN-PCT-2001-1227-CHE  FORM-6  01-08-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDEtMTIyNy1DSEUgIE9USEVSIERPQ1VNRU5UICAwMS0wOC0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">IN-PCT-2001-1227-CHE  OTHER DOCUMENT  01-08-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDEtMTIyNy1DSEUgIFBPV0VSIE9GIEFUVE9STkVZIDAxLTA4LTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2001-1227-CHE  POWER OF ATTORNEY 01-08-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTIyNy1jaGUgYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1227-che abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTIyNy1jaGUgY2xhaW1zLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1227-che claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTIyNy1jaGUgY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1227-che claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTIyNy1jaGUgY29ycmVzcG9uZGVuY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1227-che correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTIyNy1jaGUgY29ycmVzcG9uZGVuY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1227-che correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTIyNy1jaGUgZGVzY3JpcHRpb24gKGNvbXBsZXRlKS1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1227-che description (complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTIyNy1jaGUgZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1227-che description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTIyNy1jaGUgZHJhd2luZ3MtZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1227-che drawings-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTIyNy1jaGUgZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1227-che drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTIyNy1jaGUgZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1227-che form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTIyNy1jaGUgZm9ybS0xOS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1227-che form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTIyNy1jaGUgZm9ybS0yNi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1227-che form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTIyNy1jaGUgZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1227-che form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTIyNy1jaGUgZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1227-che form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTIyNy1jaGUgZm9ybS02LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1227-che form-6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTIyNy1jaGUgcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1227-che pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTIyNy1jaGUgcGV0aXRpb24ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1227-che petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="230712-synthetic-path-diversity-repeater-and-a-method-of-repeating-a-signal-in-a-wireless-communication-system.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="230714-a-compact-disk-case-base.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>230713</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/1227/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>27-Feb-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>04-Sep-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>LABORATOIRES SERONO S.A</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>Z.I.D 1&#x27;OURIETTAZ, 1170 AUBONNE,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>IMHOF, Beat, Albert</td>
											<td>28 chemin des Bougeries, CH-1233 Conches,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>AURRAND-LIONS, Michel</td>
											<td>8 rue Pierre Fatio, CH-1204 Geneva,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 14/705</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2000/002219</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-03-13</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>EP 99200746.8</td>
									<td>1999-03-11</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/230713-polypeptide-of-the-human-immunoglobulin-superfamily by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:30:58 GMT -->
</html>
