<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/231224-a-process-for-preparing-a-bacterial-consortium-useful-for-lignin-biodegradation by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:04:57 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 231224:A PROCESS FOR PREPARING A BACTERIAL CONSORTIUM USEFUL FOR LIGNIN BIODEGRADATION</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A PROCESS FOR PREPARING A BACTERIAL CONSORTIUM USEFUL FOR LIGNIN BIODEGRADATION</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention provides a novel process of lignin degradation using a consortium of bacteria. To date, biodegradation of lignin has been centered to fungi only. Degradation of lignin by bacteria confer a new understanding that may be of tremendous industrial significance. This invention also discloses the isolation and acclimatization of ligninolytic bacteria from a specific site</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>A PROCESS FOR THE ISOLATION AND ACCLIMATIZATION OF BACTERIA FOR LIGNIN DEGRADATION<br>
Field of the invention<br>
['resent invention relates to a novel, aerobic biological process for the degradation of lignin using the defined consortium of ligninolytic bacteria isolated from the specific<br>
site.<br>
Description of the prior art Fungi<br>
Lignin is the most abundant aromatic polymer in the biosphere. It is found in the cell wall of all vascular plants in association with cellulose and hemicellulose. Because inter-unit bonds in lignin are not hydrolysable, lignin is difficult to degrade either chemically or biologically. Lignin surrounds cellulose in the plant cell wall forming a matrix, which is itself resistant to degradation. Lignin biodegradation is responsible for much of the natural destruction of wood in use. and it may have an important role in plant pathogenesis. On the other hand, potential applications utilizing lignin-degrading organisms and their enzymes have become attractive, because they may provide environmentally friendly technologies for the pulp and paper industry .To date, only a few groups of organisms are capable of degrading complex lignin polymers, and they are best exemplified by the white rot fungi. Most of the research concerning biodegradation of lignin has been centered on some fungi only such as Phanerochaete chrysosporium, Streptomyces viridosporus, Pleurotus eryngii, Trameles irogii, Fusarium proliferation (Regaldo et al., 1997) etc. (1)<br>
Wood-rotting basidiomycetous fungi that cause white rot in wood are the most efficient lignin degraders in nature (Kirk and Farrell, 1987; Eriksson et al., 1990), and they are perhaps nature's major agents for recycling the carbon of lignified tissues. No other microorganisms as pure culture have been described to mineralize lignified tissues as efficiently (Kirk and Cullen, 1998). They are a group of taxonomically heterogeneous higher fungi, characterized by their unique ability to depolymerize and mineralize lignin using a set of extracellular lignnolytic enzymes. Lignin degradation by white-rot fungi has been intensively studied during the last thirty years in relation to biotechnical applications such as biopulping, biobleaching, treating of pulp mill<br><br>
effluents, and soil bioremediation (Akhtar et al., 1992. 1998: Lamar et al.. 1992; Messner and Srebotnik. 1994).<br>
The enzymology and molecular biology of lignin degradation has been mainly studied in Phanerochaete chrysosporium (Gold and Alic, 1993; Cullen, 1997; Kirk and Cullen. 1998). Many of the enzymes necessary for lignin degradation were not characterized before the beginning of the 1980s when virtually only laccase had been known. Since the discovery of two important peroxidases in the beginning of the 1980s, namely lignin peroxidases (LiPs) in 1983 and manganese peroxidases (MnPs) in 1984 (Kirk and Farrell. 1987), an array of enzymes have been isolated from fungi and characterized in detail.<br>
LiP (lignin peroxidase) is believed to be one of the key enzymes in lignin biodegradation by white rot fungi. DNA probes specific for the genes encoding major lignin peroxidases (LiP) isozymes of P. chrysosporium were constructed.. These probes were used to study the temporal expression of LiP enzymes in defined low nitrogen medium. (Boominathan et al. 1993).<br>
Aspergilli, the versatile ascomycetes are also found to transform at a rapid rate a wide spectrum of lignin related aromatic compounds. They are shown to overproduce high levels of hemicellulolytic enzymes. (4)<br>
Maria Teresa et al. have shown that Bjerkandera sp. Strain BOS55 is a white rot fungus that can bleach EDTA extracted eucalyptus oxygen delignified Kraft pulp (UKP) without any requirement for manganese. Furthermore, under manganese free conditions, addition of simple physiological organic acids (e.g. Glycolate, glyoxylate, oxalate and others) at 1-5 mM stimulated brightness gains and pulp delignification two to three fold compared to results not receiving acids. The stimulation was attributed to increase production of MnP and LiP as well as increased physiological concentrations of veratryl alcohol and oxalate. These factors contributed to greatly improved production of superoxide anion radicals, which may have been accounted for the more extensive biobleaching. (5)<br>
Till now, all the basics and applied research work has centered on fungi only. In case of biobleaching of raw pulp, the application of fungi is not feasible due to its<br><br>
structural hindrance caused by fungal filaments. Therefore, identification of bacteria having lignin oxidizing enzymes would be of significant importance.<br>
Bacteria<br>
The role of bacteria in lignin biodegradation is still a matter of conjecture. Some workers have demonstrated that either mixed (Sundman et al., 1968) or pure culture of bacteria (Sorensen, 1962) can grow on lignin as a carbon source. Pseudomonas spp. was claimed by Kawakami (1976) and Odier and Monties (1977) to degrade plant lignins. Odier and Montis also indicated several other bacterial strains that can use within seven days time more than 50% of the lignin supplied in a mineral medium containing glucose.<br>
Several Nocardia and Pseudomonas spp. as well as some unidentified bacteria, isolated from lake water containing high loads of waste lignin, were tested for their capacity to release 14CO2 from specifically C -labelled dehydropolymer of coniferyl alcohol (DHP) or corn stack lignins. However only some of them could release significant amount s of 14C02 from the labeled lignin. The tested Nocardia spp. was more active than the Pseudomonas spp. and the unidentified bacteria.(6)<br>
Actinomycetes are filamentous bacteria which are found in soil and composts where lignocellulose is decomposed. Several reports provide evidence that several species belonging to the genus Streptomyces are able to degrade lignin. Other lignin degrading Actinomycetes include Thermomonospora mesophila, Actinomadura, Micromonospora with Streptomyces exibiting the highest lignin degrading ability. In most of the studies, the lignin degrading enzyme was produced at higher levels in cultures containing lignocellulose which suggests that an induction mechanism was active.<br>
Ajit Verma et al.(1994) while working on symbiotic relationship between termites and their intestinal microbes concluded that both termite soil and termite gut bacteria play an important role in polymer depolymerizan^n. Gut bacteria have the capacity to<br>
-i<br>
degrade cellulosic and hemicellulosic mate, tis more efficiently. Several bacterial isolates which hydrolyze cellulose and hemicellulose have been obtained in pure culture from the termite gut. Some of these are Arthrobacter sp., Bacillus cereus, Clostridium sp., Micrococcus sp., Streptomyces sp., Serratia marcescens. Only a few<br><br>
xylan decomposing bacteria have been obtained from the termite gut (Micrococcus luteuns. Pseudomonas aeruginosa). The question of lignin degradation by termites is intriguing, since much of the termite gut is anaerobic and natural anaerobic mechanisms of lignin degradation are unknown.(7)<br>
Berrocal et al. (1997) have shown that cell free filtrates from streptomyces sp. Grown in solid state fermentation were capable of solubilising up to 20% of the [I4C] lignin. The activity of two enzymes, extracellular peroxidase and phenol oxidase (laccase) was found to correlate with both solubilisation and mineralisation rates of lignin.(8)<br>
The presence of bacteria in rotted wood often in association with fungi has been the subject of numerous reports. However, their exact role in degradation of wood components is still unclear. While the availability of nutrient nitrogen represses metabolism of synthetic C lignin to CCh by Phanerochaete Chrysosporium, high levels of organic nitrogen were optimal for lignin degradation by the bacterium Streptomyces badius. (9}<br>
Few bacterial isolates which exhibited a remarkable capability of bleaching the hardwood kraft pulp as reported in a previous pending patent application, have shown lignin degradation capability.<br>
Enzymes are the catalytic cornerstones of metabolism, and as such are the focus of intense worldwide research, not only in biological community, but also with process designers/engineers, chemical engineers, and researchers working in other scientific fields. Since ancient times, enzymes have played a central role in many manufacturing process, such as in the production of wine, cheese, bread etc. The latter half of the twentieth century saw an unprecedent expansion in our knowledge of the use of microorganisms, their metabolic products, and enzymes in a broad area of basic research and their potential industrial applications. Only in the past two decades, however have microbial enzymes been used commercially in the Pulp and Paper industry. (10)<br>
The most common application of enzymes in paper industry is to enhance bleaching. At least 15 patents or patent disclosures dealing with enzymatic treatments to enhance bleaching of Kraft pulps were submitted between 1988 and 1993.<br><br>
Lignin. correctly known as " nature's plastic", although resistant to microbial attack, certain filamentous fungi is capable of degrading it to the level of CCK Until 1981, it was not even known whether enzymes are involved in lignin depolymerization. Ming Tien et al. from university of Michigan have discovered enzymes that degrade lignin.<br>
The major enzymes involved in lignin biodegradation by fungi are two extracellular heme containing peroxidases: Lignin Peroxidase (LiP, EC 1.11.1.14) and Manganese Peroxidases (MnP, EC: 1.11.1.13) (Kirk et al., 1987), Gold et al. (1989); Hatakka (1994). The main difference between LiP and MnP is the nature of substrate that is oxidized. LiP is capable of oxidizing non phenolic or phenolic lignin structures directly to yield aryl cation radicals and phenoxy radicals, respectively. (Kirk, 1987). For MnP, the primary reducing substrate is divalent manganese ion Mn2". The catalytic cycle of MnP in the presence of appropriate chelators generates highly reactive Mn3+ chelate complexes that are able to oxidize various phenols and carbon centered radicals, respectively. (Wariishi et al., 1989; Hofrichter et al., 1998).<br>
Usually MnP is not able to oxidize or depolymerize the more recalcitrant non-phenolic lignin structures that make up about 90% of the lignin in wood. Interestingly, it seems that primary attack on lignin requires low molecular weight agents, because LiP and other enzymes are too large to penetrate lignocellulose. (Call et al., 1997). Because of these discrepancies, it has been proposed that there are mechanisms that enable MnP to cleave non-phenolic lignin structures via the action of small mediators such as thiyl or lipid radicals. (Wariishi et al., (1989), Bao et al., (1994)).<br>
The Mn (II) concentration of the growth medium strongly affects the secretion patterns of lignin peroxidase and laccase. Mn Peroxidase was not found in fast protein liquid chromatography profiles of extracellular proteins under either low (2.4 pM) or elevated (24 and 120 |JM) Mn (II) concentrations^ 16)<br>
The role of one more enzyme, extracellular phenol oxidases (laccases) in lignin degradation has been suggested and it has recently been demonstrated that laccase can take part in lignin degradation. (Bourbonnais and Paice 1990; 1992: Srinivasan et al. 1995). Laccase is a type of copper containing polyphenol oxidase known to catalyze the oxidation of a range of inorganic and aromatic substances by the removal of<br><br>
electrons with the concomitant reduction of Ch to water. But its application in Bioblecahing has to be proved yet.<br>
Cited References<br>
(1)	Etienne Odier and Isabelle Artaud /Degradation of Lignm'. Prof. Dr. Annele<br>
Hatakka, 'Biodegradation of Lignin'.<br>
(2)	Frederick S. Archibald,  ' Lignin Peroxidase activity is not important in<br>
biological bleaching and delignification of unbleached kraft pulp by Trametes<br>
versicolor' Appl. and Env. Microbiol, Sept. 1992 p. 3101-3109.<br>
(3)	Brian P.Roy and Frederick Archibald, 'Effects of Kraft pulp and lignin on<br>
Trametes versicolor carbon metabolism', Appl. and Environmental Microbiol.,<br>
June 1993, P. 1855-1863.<br>
(4)	Daurte JC, Costa-Ferreira M; 'Aspergilli and Lignocellulosics: Enzymology<br>
and biotechnological applications', FEMS Microbiol. Rev., Mar 1994; 13 (2-<br>
3):377-86.<br>
(5)	Maria Teresa. Gumersindo Feijoo, tuned mester, Pablo Mayorga, Reyes sierra-<br>
Alvarez and Jim A. Field.;  ' Role of Organic acids in the Manganese<br>
independent biobleaching system of Bjerkandera sp. Strain BOS55'^4ppl. and<br>
Env. Microbiol., July 1998, p. 2409-2417.<br>
(6)	K. Haider, J, Trojanowski, and V. Sundman, 'Screening for lignin degrading<br>
Bacteria by means of [I4C] labeled lignin' Arch. Microbiol.  119, 103-106<br>
(1978).<br>
(7)	Varma A.,  Koll's B.  K.,  Paul J.,  Saxena S., Koniig H;  Lignocellulose<br>
degradation by microorganisms from termite hills and termite guts: A survey<br>
on the present state of art. FEMS Microbiology Reviews 15 (1994) 9-28.<br>
(8)	M.       M.       Berrocal.       J.Rodriguez. A. S.BalLM.I.Perez-Lebric.M.E. Alias.<br>
'Solubilization and mineralization of [14C] lignocellulose from wheat straw by<br>
Streptomyces cyaneus CECT 3335 during growth in solid state fermentation',<br>
Appl. Microbiol. Biotechnol (1997) 48:379-384.<br>
(9)	Ian D. Reid, 'Effects of Nitrogen supplements on degradation of aspen wood<br>
lignin and carbohydrate components by P. chrysosporium', Appl. And Env.<br>
Micro. Mar 1983, p. 830-837.<br><br>
(10)	Q. K. Beg .M. Kapoor.L. Mahajan. G.S.Hoondal. Microbial xylanases and<br>
their industrial applications, A Review', Appl. Microbiol.  Biotech. (2001)<br>
56:326-328.<br>
(11)	J.    H.    Clarke.    K.    Davidson.    J.    E.    Rixon.    J.R.    Halstead,    M.P.<br>
Fransen.H.J.Gilbert.   G.P.Hazlewood..   'A   comparison   of  enzyme   aided<br>
bleaching of softwood paper pulp using combinations of xylanase. mannanase<br>
and 2-galactosidase.' Appl. Microbiol. Biotechnol. (2000) 53: 661-667.<br>
(12)	Thomas W. Jeffries. ' Enzymatic treatments of pulps: Opportunities for the<br>
Enzyme Industry in Pulp and Paper Manufacture'.<br>
(13)	N. Gupta, R. M. Vohra and G. S. Hoondal. 'A thermophillic extracellular<br>
xylanase from alkalophilic Bacillus sp. NG-27', Biotechnology Letters, vol. 14<br>
No. 11 (Nov. 1992) pp. 1045-1046.<br>
(14)	Michael J. Bailey. Peter Biely and Kaisa Poutanen, ' Inter laboratory testing, of<br>
methods for assay of xylanase activity', Journal of Biotechnology, 23 (1992)<br>
257-270.<br>
(15)	Toshiya susaki, Tsutomu Kajono, BoLi,  Hidehiko Sugiyama. and Harua<br>
Takatashi;   'New pulp biobleaching system involving manganese peroxidase<br>
immobilized in a silicon support with controlled pore sizes', Appl. and Env.<br>
Microbiol. May 2001, p 2208-2212.<br>
(16)	Tamara   Vares,   Outi   Niemenmaa   and   Annele   Hatakka,,   'Secretion   of<br>
Lignolytic enzymes and mineralization of   C-ring labeled synthetic lignin by<br>
three Phlebia tremellosa strains', Appl. and Environmental Microbiol. Feb.<br>
1994569-573.<br>
(17)	Ryuichiro   Kondo,   Koichi   Harazono,   and   Kokki   Sakai;   'Bleaching  of<br>
hardwood    Kraft    pulp    with    Manganese    Peroxidase    secreted    from<br>
Phanerochaete sordida YK-624'. Appl. and Env. Microbiol. Dec.  1994, p.<br>
4359-4363.<br>
(18)	J. C. Rols, G. Goma, C. Fonade. 'Biotechnology and the paper industry,<br>
Aerated lagoon for the wastewater treatment'.<br>
(19)	J. Sealey and A. J. Ragauskas, ' Resk
1<br>
Kraft pulps', Em. And Micro Tech. 23: -+_2-426, 1998. Objects of the invention<br><br>
The main object of the present invention is to provide a novel consortium of ligninolytic bacteria for degradation of lignin.<br>
Another object of the present invention is to provide a novel biological process for the degradation of lignin using the above-defined consortium of ligninolytic bacteria.<br>
Yet another object of the invention is to provide a process for preparing a consortium of ligninolytic bacteria capable of degrading lignin.<br>
Summary of the invention<br>
The present invention provides a novel consortium of ligninolytic bacteria for degradation of lignin and also a biological process for the degradation of lignin using said consortium of ligninolytic bacteria. Ligninolytic bacteria were isolated from a specific Indian site where sawdust continually accumulated over the long period. The said bacteria were acclimatized to improve their capability to degrade lignin and finally formulated in a consortium to degrade the lignin.<br>
Detailed description of the invention<br>
The present invention relates to a synergistic consortium of ligninolytic bacteria for degradation of lignin, said consortium comprising bacterial strains CBTCC/52-03, CBTCC/53-03 and CBTCC54-03 deposited at International Depository at IMTECH, Chandigarh, India, and having accession numbers MTCC 5094, MTCC 5095 and MTCC 5098 respectively in any possible combination.<br>
In an embodiment of the present invention, the consortium comprises bacterial strains MTCC 5094, MTCC 5095 and MTCC 5098.<br>
In another embodiment of the present invention, the consortium comprises bacterial strains MTCC 5094, MTCC 5095 and MTCC 5098 in the range of 20 to 40% by wt.<br>
each.<br>
In yet another embodiment of the present invention, the consortium comprises bacterial strains MTCC 5094, MTCC 5095 a'nd MTCC 5098 are in equal proportions.<br>
In still another embodiment of the present invention, wherein the bacteria are isolated from a mixture of sawdust and soil.<br><br>
In one more embodiment of the present invention, the bacteria are isolated from a mixture of sawdust and soil from Roorkhee, India.<br>
In one another embodiment of the present invention, the consortium exhibits lignin degradation of up to 0.8%.<br>
In a further embodiment of the present invention, the characteristics of MTCC 5094 are as follows: Gram - Negative. Shape - Small rods.<br>
In an embodiment of the present invention, the characteristics of MTCC 5095 are as follows: Gram - Negative. Shape - Cocci.<br>
In another embodiment of the present invention, the characteristics of MTCC 5098 are as follows: Gram - Negative. Shape - Long rods<br>
The bacterial isolates concerned with the present invention are being deposited at<br>
Institute of Genomics and	Integrative Biology (IGIB) as CBTCC, and their<br>
identification is underway.<br>
S.No.              Culture	Accession No.<br>
1.	CBTCC/52-03	MTCC 5094<br>
2.	CBTCC/53-03	MTCC 5095<br>
3.	CBTCC/54-03	MTCC 5098<br>
These bacterial isolates are exhibiting a remarkable capability to degrade the lignin under defined conditions.<br>
The bacterial isolates in the present invention have been isolated from a wood workshop situated at Roorkee, India, where sawdust continually accumulated over the period of 10-12 years.<br>
The invention further provides a process of isolation and acclimatization of bacterial isolates capable to degrade the lignin which cohiprises:<br>
a)   enriching the bacterial flora of the said   site using soil extract and particular inducers under defined conditions;<br><br>
b)	using different media ( 0.3 lignin + agar ;  soil extract + agar; 50% soil extract +<br>
agar; 0.3% lignin + 50% soil extract) to entrap the maximum bacterial flora from<br>
the said site;<br>
c)	culturing the said bacteria isolated from specific site under defined conditions<br>
such as media, temperature, pH, carbon source etc.;<br>
d)	checking  the   lignin   degrading   capability   of  isolated   bacterial   isolates   by<br>
inoculating them in 10ml 0.4% lignin;<br>
e)	lignin degradation was estimated by a known spectrophotometric method:<br>
i)   selecting the bacterial isolates which can decolorize the lignin effectively;<br>
g)  acclimatizing the short listed bacterial isolates for higher concentration of lignin to<br>
see the enhancement of their ligninolytic activity; h)  Further short listing and formulation of different bacterial isolates in order to see<br>
their synergistic effect for lignin degradation; i)   culturing the said bacteria under defined conditions for the detection of enzyme<br>
activity in crude as well as in concentrated sample. MSM having 1.0%   glucose<br>
was used to grow the culture in 2 litre flask having 1000ml culture. The culture<br>
flask was incubated at 30°C/120rpm for 3 days in order to obtain heavy growth; j)   centrifuging the resulting culture after attaing the heavy growth O.D. (1.00); k)  collecting  the   supernatent  and   concentrating  through   ammonium   sulphate<br>
precipitation followed by dialysis; 1)   assaying  the  lignin  peroxidase  in both the  samples,  crude  as  well  as  in<br>
concentrated      by      measuring      the      product      (veratryldehyde)formation<br>
spectrophotometrically at 31 Onm.<br>
In an embodiment of the present invention, the bacteria are isolated from specific sawdust site located in Roorkee, India.<br>
In another embodiment of the present invention, enrichement of the soil from siad site is done by taking 5g of fresh soil in the 500 ml autoclaved flask containing 100 ml soil extract, 0.3% lignin, ImM veratryl alcohol and 50 ul Candid B. Enrichment flask is kept at 120rpm for 96 hours at 30°C.<br>
In another embodiment of the present invention, four types of different media ( 0.3 lignin + agar ; soil extract + agar; 50% soil extract + agar ; 0.3% lignin + 50% soil extract) are used to entrap the maximum bacterial flora of the said site.<br><br>
In another embodiment of the present invention, isolated bacterial isolates are cultured under defined conditions such as media, temperature, pH. carbon source etc.<br>
In another embodiment of the present invention, all the bacterial isolates are tested for their lignin degrading capability by inoculating them in 10ml 0.4% lignin. Lignin degradation was estimated by a known spectrophotometric method.<br>
In another embodiment of the present invention, some selected bacterial isolates are acclimatized for higher concentration of lignin to see the enhancement of their ligninolytic activity. Bacteria are inoculated in lignin ranging from 0.5% to 1.0% and kept at 30°C for a period of three months. Lignin degradation was estimated by a known spectrophotometric method.<br>
In another embodiment of the present invention, further short listed bacterial isolates are formulated in a number of consortia to see their synergistic effect for lignin degradation.<br>
In another embodiment of the present invention, MSM having 1.0% glucose is used to grow the culture in 2 litre flask having 1000ml culture for the enzyme study. The culture flasks are incubated at 30°C/120rpm for 3 days in order to obtain heavy growth<br>
In another embodiment of the present invention, cultures are centrifuged and supernatent is collected followed by 80% ammonium sulphate precipitation and dialysis.<br>
In another embodiment of the present invention, lignin peroxidase is assayed in both the samples, crude as well as in concentrated by measuring the product (veratryldehyde )formation spectrophotometrically at 310nm.<br>
The present invention further provides a process of lignin degradation, said process comprising inoculating the bacterial ct 'hiium comprising bacterial strains CBTCC/52-03, CBTCC/53-03 and CBTCC54-03 deposited at International Depository at IMTECH, Chandigarh, India, and having accession numbers MTCC<br><br>
5094, MTCC 5095 and MTCC 5098 in an solution containing lignin for 1 to 5 days at temperature between 25 to 35°C.<br>
In an embodiment of the present invention, the bacterial consortium comprises whole cell bacterial isolates of bacterial strains MTCC 5094, MTCC 5095 and MTCC 5098.<br>
In another embodiment of the present invention, the consortium comprises bacterial strains MTCC 5094, MTCC 5095 and MTCC 5098.<br>
In yet another embodiment of the present invention, the consortium comprises bacterial strains MTCC 5094, MTCC 5095 and MTCC 5098 in the range of 20 to 40% by wt. each.<br>
In still another embodiment of the present invention, the consortium comprises bacterial strains MTCC 5094, MTCC 5095 and MTCC 5098 in equal proportions.<br>
In one more embodiment of the present invention, the bacteria are isolated from a mixture of sawdust and soil.<br>
In one another embodiment of the present invention, the bacteria are isolated from a mixture of sawdust and soil from Roorkhee, Uttar Pradesh, India.<br>
In a further embodiment of the present invention, the consortium exhibits lignin degradation of up to 0.8%.<br>
For the isolation of ligninolytic bacteria, proper enrichment was done .To improve the yield of desired bacteria, 5g of fresh soil from the said site was inoculated in the 500 ml autoclaved flask containing 100 ml soil extract, 0.3% lignin and 50 ul Candid B (antifungal). Enrichment flask was kept at 120 rpm for 96 hours at 30°C.<br>
For the preparation of soil extract, IKg soil was taken and dried at 50°C for 2 hours. 400g of dried soil was dissolved in 960 ml single distilled water and autoclaved at 151bs for 1 hour. After autoclaving, the sample was centrifuged at SOOOrpm for 10 minutes. The supernatant ( extract) was collected and stored in sterile bottle for the preparation of enrichment flask till further use.<br><br>
The enriched soil samples were serially diluted in 0.85% saline. 100 ul from each respective dilution was spread onto petriplates containing soil extract. 50% nutrient agar and 0.2% lignin for the the isolation of ligninolytic bacteria. The plates thus obtained were incubated at 30 ± 2 JC for 24 - 96 hrs in inverted position.<br>
On the basis of colony morphology and colour, total 36 isolates were selected to check their ligninolytic activity. The single isolated colonies were picked and streaked on fresh plates containing the same medium. The above step was repeated till pure colonies were obtained.<br>
To check the ligninolytic activity of the isolated bacteria, 10 ml of 0.4% lignin was taken in 30ml test tubes and all the bacterial isolates were inoculated individually. After 3 days, different bcatenal isolates showed different degree of decolorization of lignin. Lignin degradation was estimated by a known spectrophotometric method.<br>
On the basis of lignin decolorization, total 15 bacterial isolates out of 36 isolates were selected for further study. To enhance the ligninolytic activity of the selected bacterial isolates, acclimatization was done for higher concentration of lignin. All the 15 isolates were inoculated in 10 ml lignin for the range from 0.5% to 1.00%. All the tubes were kept at 30 C for three months. After the long acclimatization, 8 bacterial isolates were chosen for the further study. The selected isolates were able to decolorize the lignin upto 0.7%. Lignin degradation was estimated by a known spectrophotometric method<br>
In order to see the synergistic effect of these bacterial isolates, a number of consortia were designed and tested for their capability of degrading the lignin. One consortium comprising of three bacteria designated as MTCC 5094, MTCC 5095 and MTCC 5098 was found able to degrade the lignin upto 0.8%.<br>
To find out the mechanism of lignin degradation at enzyme level, enzyme assay for lignin peroxiadse, a key enzyme for lignin degradation, was carried out. All the three bacteria were inoculated in 1 litre minimal salt medium (MSM) having 1% glucose. For the induction of ligninolytic enzymes, ImM veratryl alcohol was added to the cultures. All the cultures were incubated at 30-35°C for three days under shaking conditions (T00-120rpm). After observing the heavy bacterial growth, all the cultures<br><br>
were centrifuged at lO.OOOrpm at 4UC. Supernatent was collected to check the enzyme activity.<br>
For the concentration of extracellular enzymes. 80% ammonium sulphate precipitation was carried out using 100% saturated ammonium sulphate solution followed by subsequent dialysis.<br>
Enzyme assay for lignin peroxidase was performed for both, crude cell free extract as well as concentrated sample. For the enzyme assay. lOmM veratryl alcohol, 5mM FbCh and 400ul enzyme solution was added in tartaric acid (pH 3) buffer. The formation of product ( veratryldehyde) was monitored at 31 Onm. Lignin peroxidase catalyzes the oxidation of veratryl alcohol by H2O2 to veratryldehyde. Veratry alcohol exhibits no absorbance at 310 nm wherease veratrlydehyde absorbs strongly (molar extinction coefficient = 9300 M-l cm-1). All the three bacterial isolates showed different degree of ligninolytic activity.<br>
The present invention is further described with reference to the accompanying examples, which are given by way of illustration and hence, should not be construed to limit the scope of the present invention in any manner.<br>
Example 1<br>
In the endeavor of exploring ligninolytic bacteria, strategic isolation was done to entrap the maximum ligninolytic bacterial flora from the specific site.<br>
Isolation point was a workshop situated at Roorkee, India, where sawdust continually accumulated over the period of 10-12 years.<br>
For the isolation of ligninolytic bacteria, proper enrichment was done .To improve the yield of desired bacteria, 5g of fresh soil from the said site was inoculated in the 500 ml autoclaved flask containing 100 ml soil extract, 0.3% lignin and 50 ul Candid B. Enrichment flask was kept at 120 rpm for 96 hours at 300C.<br>
For the preparation of soil extract, IKg soil was taken and dried at 500C for 2 hours. 400g of dried soil was dissolved in 960 ml single distilled water and autoclaved at 151bs for 1 hour. After autoclaving, the sample was centrifuged at SOOOrpm for 10<br><br>
minutes. The supernatant (extract) was collected and stored in sterile bottle for the preparation of enrichment flask and further use.<br>
The enriched soil samples were serially diluted in 0.85% saline. 100 ul from each respective dilution was spread onto petriplates containing soil extract, 50% nutrient agar and 0.2% lignin for the isolation of ligninolytic bacteria. The plates thus obtained were incubated at 30 ± 2 °C for 24 - 96 hrs in inverted position.<br>
On the basis of colony morphology and color, total 36 isolates were selected to check their ligninolytic activity. The single isolated colonies were picked and streaked on fresh plates containing the same medium. The above step was repeated till pure colonies were obtained.<br>
Example 2<br>
For the isolation of ligninolytic bacteria, proper enrichment was done .To improve the yield of desired bacteria, 5g of fresh soil from the said site was inoculated in the 500 ml autoclaved flask containing 100 ml soil extract, 0.3% lignin and 50 ul Candid B. Enrichment flask was kept at 120 rpm for 96 hours at 300C.<br>
For the preparation of soil extract, IKg soil was taken and dried at 500C for 2 hours. 400g of dried soil was dissolved in 960 ml single distilled water and autoclaved at 151bs for 1 hour. After autoclaving, the sample was centrifuged at 5000rpm for 10 minutes. The supernatant (extract) was collected and stored in sterile bottle for the preparation of enrichment flask and further use.<br>
The enriched soil samples were serially diluted in 0.85% saline. 100 ul from each respective dilution was spread onto petriplates containing soil extract, 50% nutrient agar and 0.2% lignin for the isolation of ligninolytic bacteria. The plates thus obtained were incubated at 30 ± 2 °C for 24 - 96 hrs in inverted position.<br>
On the basis of colony morphology and color, total 36 isolates were selected to check<br>
their ligninolytic activity. The single isolate ] •• !onies were picked and streaked on fresh plates containing the same medium, me above step was repeated till pure colonies were obtained.<br><br>
To check the ligninolytic activity of the isolated bacteria, 10 ml of 0.4% lignin was taken in 30 ml test tubes and all the bacterial isolates were inoculated individually. After 3 days, different bacterial isolates showed different degree of decolorization of lignin. Lignin degradation was estimated by a known spectrophotometric method.<br>
Table 1. Bacterial degradation of lignin (0.4%) as a sole carbon source<br>
(Table Removed)<br>
Example 3<br>
For the isolation of ligninolytic bacteria from specific site, proper enrichment was done. To improve the yield of desired bacteria. 5g of fresh soil from the said site was inoculated in the 500 ml autoclaved flask containing 100 ml soil extract. 0.3% lignin and 50ul Candid B. Enrichment flask was kept at 120 rpm for 96 hours at 300C.<br>
For the preparation of soil extract. IKg soil was taken and dried at 500C for 2 hours. 400g of dried soil was dissolved in 960 ml single distilled water and autoclaved at 151bs for 1 hour. After autoclaving, the sample was centrifuged at SOOOrpm for 10 minutes. The supernatant (extract) was collected and stored in sterile bottle for the preparation of enrichment flask and further use.<br>
The enriched soil samples were serially diluted in 0.85% saline. 100 ul from each respective dilution was spread onto petriplates containing soil extract, 50% nutrient agar and 0.2% lignin for the isolation of ligninolytic bacteria. The plates thus obtained were incubated at 30 ± 2 °C for 24 - 96 hrs in inverted position.<br>
On the basis of colony morphology and color, total 36 isolates were selected to check their ligninolytic activity. The single isolated colonies were picked and streaked on fresh plates containing the same medium. The above step was repeated till pure colonies were obtained.<br>
To check the ligninolytic activity of the isolated bacteria, 10 ml of 0.4% lignin was taken in 30ml test tubes and all the bacterial isolates were inoculated individually. After 3 days, different bacterial isolates showed different degree of decolorization of lignin.<br>
On the basis of lignin decolorization, total 15 bacterial isolates, namely, L4, L5, L8, L10, Lll, L13, L18, L20, L20, L22, L23, L27, L28, L29, L35 AND L36 from 36 were selected for further study. To enhance the ligninolytic activity of the selected bacterial isolates, acclimation was done for higher concentration of lignin. All the 15 isolates were inoculated in 10 ml lignin for the range from 0.5% to 1.00%. All the tubes were kept at 30°C for three months. After the long acclimatization, 8 bacterial isolates, namely, L4, L5, L10, L13, L28, L29, L35 and L36 were chosen for the further study. Lignin degradation was estimated by a known spectrophotometric<br>
method the selected isolates had been able to decolorize the lignin upto 0.7%. (Table<br>
2)<br>
Table: 2 Acclimatization of selected ligninolytic bacteria to higher concentration of<br>
(Table Removed)<br>
Example 4<br>
In order to see the synergistic effect of the acclimatized bacteria, a number of consortia were designed and tested for their capability of degrading the lignin. Consortia were designed in taking three bacterial isolates together. All the consortia were made from 8 bacterial isolates, namely, L4, L5, L10, LI3, L28, L29, L35 and L36, which have been able to degrade the lignin upto 0.7%. These consortia were tested for their ligninolytic activity in a synergistic manner.<br>
To make the inoculum from consortium for biodegradation study, a loop from agar plate of three different bacteria were inoculated individually in 10 ml MSM with 1% glucose. The cultures were incubated at 30°C for 16-24 hours in an incubator shaker at 100-120 rpm. After incubation, optical density was measured at 650 nm. Optical density of the culture was maintained to 1 0' ] her by diluting or concentrating the bacterial suspension. All the three cultures were mixed together in equal proportion in order to make the inoculum. Different concentrations of lignin (0.7%, 0.8%, 0.9%, 1.0%) were inoculated by the inoculum of different consortia. All the tubes were<br>
incubated for 3 days at 30 C. Lignin degradation was estimated by a known spectrophotometric method. Finally a consortium comprising the bacterial isolate, L35. L36 and L13, was selected as a potent lignin degrader which was able to degrade the lignin upto 0.8%<br>
Table 3. Synergistic effect of ligninolytic bacteria on lignin degradation.<br>
(Table Removed)<br>
Example 5<br>
To find out the mechanism of lignin degradation at enzyme level, enzyme assay for lignin peroxidase, a key enzyme for lignin degradation, was carried out. All the three bacteria. L35, L36 and L13 of the consortium 16 (Table 3) which was able to degrade the lignin upto 0.8%, were inoculated individually in 1 liter minimal salt medium (MSM) having 1% glucose. For the induction of ligninolytic enzymes, ImM veratryl alcohol was added to the cultures. All the cultures were incubated at 30-35°C for three days under shaking conditions (100-120rpm). After seeing the heavy bacterial growth,<br>
1	!	I<br>
all the cultures were centrifuged at 10,000rpm at 4 C. Supernatant was collected to check the enzyme activity.<br>
For the concentration of extracellular enzymes. 80% ammonium sulfate precipitation was carried out using 100% saturated ammonium sulfate solution followed by subsequent dialysis.<br>
Enzyme assay for lignin peroxidase was performed for both, crude cell free extracts as well as concentrated sample. For the enzyme assay, lOmM veratryl alcohol. 5mM 1K&gt; and 400ul enzyme solution was added in tartaric acid (pH 3) buffer. The formation of product (veratryldehyde) was monitored at 310nm. Lignin peroxidase catalyzes the oxidation of veratryl alcohol by H2O2 to veratryldehyde. Veratry alcohol exhibits no absorbance at 310 nm whereas veratrlydehyde absorbs strongly (molar extinction coefficient = 9300 M-l cm-1). All the three bacterial isolates showed different degree of ligninolytic activity (table 4).<br>
Table 4: Enzyme activity of L35, L36 and L13 in term of increase in O.D. after incubation of 15 minutes<br>
(Table Removed)<br>
Advantages<br>
1.	Degradation   of   lignin   by   bacteria   has   tremendous   significance   for   the<br>
bioremediation of pulp and paper industrial wastewater.<br>
2.	Bacterial degradation of lignin confers a new understanding of conversion of<br>
industrial lignin waste into useful commodities.<br><br><br><br><br>
We claim<br>
1. A process for preparing a bacterial consortium useful for lignin biodegradation characterized in that it comprises the bacterial isolates MTCC 5094, MTCC 5095 and MTCC 5095 wherein the said process comprising the steps of:<br>
a)	enriching bacterial flora of a site from where the bacteria is to be extracted wherein the soil enrichment is done in l00ml-soil extract and 0.3% lignin for the period of 96 hours;<br>
b)	isolating the ligninolytic bacteria using different media of the kind such as herein described;<br>
c)	culturing the bacteria isolated in step (b) under at pH 6.8 - 7.2 and at 30 - 35 degree C;<br>
d)	checking the lignin degrading capability of the isolated bacterial isolates by inoculating them in 10ml of 0.4% lignin;<br>
e)	selecting the bacterial isolates which can decolorize the lignin effectively;<br>
f)	acclimatizing the short listed bacterial isolates for higher concentration of lignin to see the enhancement of their ligninolytic activity;<br>
g)	further short listing and formulation of bacterial isolates in order to see their synergistic effect for lignin degradation;<br>
h) selecting from the short list of step (g), a microbial consortium comprising of<br>
three bacterial isolates exhibiting the lignin degradation upto 0.8%, and if<br>
desired; i)  culturing the bacterial isolates of the said consortium under defined conditions<br>
for the detection of enzyme activity in crude as well as in concentrated<br>
sample; j)  centrifuging the resulting culture after attaining the heavy growth O.D. (1.00 -<br>
1.20);<br><br>
k) collecting the  supernatant  and  concentrating through  ammonium  sulfate<br>
precipitation followed by dialysis, and 1)  assaying the lignin peroxidase in both the samples, crude as well as in<br>
concentrated sample by measuring the product (veratryldehyde) formation by<br>
a known spectrophotometric method at 310nm.<br>
2.	A process as claimed in claim 1, wherein the isolation of ligninolytic bacteria is carried out using four media: 0.3 lignin + agar ; soil extract + agar; 50% soil extract + agar ; 0.3% lignin + 50% soil extract.<br>
3.	A process as claimed in claim 1, wherein the lignin degradation is carried out by inoculating the isolated bacterial isolates in 10ml of 0.4% lignin.<br>
4.	A process as claimed in claim 1, wherein the short listed bacterial isolates are acclimatized to higher concentrations of lignin, having strength ranging from 0.5% - \% in 30ml test tubes.<br>
5.	A process as claimed in claim 1, wherein the said consortium comprising of three ligninolytic bacteria is obtained by testing different combinations of the isolated bacteria and selecting the consortium that has been able to degrade the lignin upto 0.8%).<br>
6.	A process as claimed in claim 1, wherein the bacterial isolates of the said consortium are cultured in 1000ml minimal salt medium having 1.0% glucose individually.<br>
7.	A process as claimed in claim 1, wherein the culture flask is incubated at 30 -34°C at 100 - 120rpm for 3 days to obtain heavy growth.<br><br>
8.	A process as claimed in claim 1, wherein the cultures are centrifuged at 10,000rpm for 30 minutes at 4°C.<br>
9.	A process as claimed in claim 1, wherein the collected supernatant is concentrated through 80% ammonium sulphate precipitation followed by dialysis.<br>
10.	A process as claimed in claim 1, wherein the lignin peroxidase is assayed in both the samples, crude as well as in concentrated sample by measuring the product (veratryldehyde ) formation spectrophotometrically at 310nm.<br>
11.	A process as claimed in claim 1, wherein the sawdust soil is enriched before extraction to get the desired ligninolytic bacterial flora.<br>
12.	A process as claimed in claim 1, wherein the sawdust soil is enriched by:<br><br>
a)	enriching the said soil in soil extract and 0.3% lignin;<br>
b)	incubating the enrichment flask at 100-120 rpm / 30 - 32°C.<br><br>
13.	A process as claimed in claim 1, wherein the 5 - 7gram of said soil is enriched in 100 - 120ml soil extract and 0.3% lignin.<br>
14.	A process as claimed in claim 1, wherein the enrichment is done at 100-120rpm / 30 - 32°C for a period of 96 hours.<br>
15.	A process as claimed in claim 1, wherein the ligninolytic bacteria is isolated from the saw-dust site using the following media:<br>
a)	0.3 lignin + agar;<br>
b)	soil extract + agar;<br>
c)	50% soil extract + agar;<br>
d)	0.3% lignin + 50% soil extract.<br><br>
16.	A process as claimed in claim 1, wherein the isolated ligninolytic bacterial<br>
stains are acclimatized to higher concentrations of lignin by:<br>
a)	inoculating the isolated bacterial strains in lignin ranging from 0.5% to 1.00%, and<br>
b)	incubating the tubes at 30-32°C for a long period.<br><br>
17.	A process as claimed in claim 1, wherein the ligninolytic bacterial isolates are inoculated in lignin ranging from 0.5% to 1.00%).<br>
18.	A process as claimed in claim 1, wherein the inoculated tubes of lignin are kept at 30-32°C for a period of 3 months.<br>
19.	A process of lignin degradation using the formulated microbial consortium as obtained in claim 1, wherein the said process comprising inoculating the said bacterial strains in an solution containing lignin for 1 to 5 days at temperature between 25 to 35°C.<br>
20.	A process as claimed in claim 30, wherein the consortium comprises bacterial strains MTCC 5094, MTCC 5095 and MTCC 5098 in equal proportions.<br>
21.	A process as claimed in claim 30, wherein the bacteria are isolated from a mixture of sawdust and soil.<br>
22.	A process as claimed in claim 30, wherein the bacteria are isolated from a mixture of sawdust and soil from Roorkhee, India.<br><br><br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
		<br>
		<div class="pull-left">
			<a href="231223-a-hygienic-disposable-absorbent-article.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="231225-an-adsorbent-bed-for-purification-of-a-gas-stream-and-process-thereof.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>231224</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>374/DEL/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>11/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>13-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>03-Mar-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>25-Mar-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>COUNCIL OF SCIENTIFIC AND INDUSTRIAL RESEARCH</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>RAFI MARG,NEW DELHI-110 001,INDIA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>RITA KUMAR</td>
											<td>INSTITUTE OF GENOMICS&amp; INTEGRATIVE BIOLOGY, MALL ROAD, DELHI-110007</td>
										</tr>
										<tr>
											<td>2</td>
											<td>ANIL KUMAR</td>
											<td>INSTITUTE OF GENOMICS&amp; INTEGRATIVE BIOLOGY, MALL ROAD, DELHI-110007</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 1/20</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/231224-a-process-for-preparing-a-bacterial-consortium-useful-for-lignin-biodegradation by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:04:58 GMT -->
</html>
