<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/231276-a-process-for-the-selection-of-a-microorganism-based-on-inactivation-of-an-essential-translocating-enzyme by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:08:25 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 231276:A PROCESS FOR THE SELECTION OF A MICROORGANISM BASED ON INACTIVATION OF AN ESSENTIAL TRANSLOCATING ENZYME</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A PROCESS FOR THE SELECTION OF A MICROORGANISM BASED ON INACTIVATION OF AN ESSENTIAL TRANSLOCATING ENZYME</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>ABSTRACT 2119/CHENP/2005 &quot;A process for the selection of a microorganism based on inactivation of an essential translocating enzyme&quot; The present invention relates to a process for the selection of a microorganism, characterized in that (a) an endogenously present gene coding for an essential translocation activity is inactivated and (b) the essential translocation activity inactivated according to (a) is cured by means of a vector.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Translocating enzyme used as a selection marker<br>
[0001] The present invention relates to a s_e_l_ect_ipn system for microorganisms, which is based on the inactivation of an essential translocating enzyme and the curing of this inactivation by means of an identically acting factor which is made available to the cells concerned by means of a vector.<br>
[0002] Proteins needed in a large amount and, among these, in particular enzymes for industrial fields of application, are usually obtained nowadays by fermentation of microorganisms. As regards those microorganisms which naturally form the proteins of interest, genetically modified producer strains are increasingly gaining importance. Genetic engineering processes of this type for protein production have long been established in the prior art. The principle for this consists in the fact that the genes for the,,, proteins of interest are incorporated into the host cells as transgenes, transcribed and translated by these, and optionally secreted through the membranes concerned into the periplasma, or the surrounding medium. They are then obtainable from the cells concerned or the culture supernatants.<br>
[0003] In industrial processes for protein production, the natural abilities of the microorganisms employed for production for the synthesis and optionally for the secretion of the proteins are first utilized. Basically, the bacterial systems selected for protein production are those which are inexpensive in the fermentation, which promise a high product formation rate from the start and which guarantee correct folding, modification etc. of the protein to be produced; the latter is all the more probable with<br>
A<br><br>
increasing relationship with the organism originally producing the protein of interest. Host cells particularly established for this purpose are gram-negative bacteria, such as, for example, Escherichia coli or Klebsiella, or gram-positive bacteria, such as, for example, species of the genera Staphylococcus or Bacillus.<br>
[0004] The economy of a biotechnological process depends crucially on the achievable yield of protein. This yield is determined, in addition to the expression system employed, by the process employed, in particular by means of the fermentation parameters and the substrates. By optimization of the expression system and of the fermentation process, the potential of a production organism and of the achievable yield can be markedly increased.<br>
[0005] Moreover, expression systems are essentially developed and developed further on the basis of two fundamentally different genetic constructions. On the one hand, the gene for the protein to be produced is integrated into the chromosome of the host organism. Constructs of this type are very stable to the presence of an additional marker gene without selection (see below) . The disadvantage is that only one copy of the gene is present in the host and the integration of further copies to increase the product formation rate by means of the gene dose effect is arranged methodically in a very complicated manner. This prior art may be briefly illustrated below.<br>
[0006] European patent EP 284126 Bl solves the problem of stable multiple integration in that a number of gene copies are incorporated into the cell, which contain the endogenous and essential chromosomal DNA sections lying in between.<br>
3<br><br>
[0007] Another solution to stable multiple integration is disclosed in the patent application WO 99/41358 Al. Accordingly, two copies of the gene of interest are integrated in opposite transcription directions and are separated from one another by a nonessential DNA section in order to prevent homologous recombination of the two copies in this way.<br>
[0008] From patent application DD 277467 Al, a process for the production of extracellular enzymes follows which is based on the stable, advantageously multiple, integration of the genes coding for the enzyme of interest into the bacterial chromosome. The integration takes place via homologous regions. For the control of successful integration events an erythromycin gene contained on the plasmid is used, which is inactivated on successful integration.<br>
[0009] According to specification DE 4231764 Al, the integration into the chromosome can take place via a single or double crossing-over via the gene for thymidylate synthetase. The latter allows the control of this process, since with a single crossing-over the thy activity is retained, whereas it is lost on a double crossing-over, that is to say auxotrophy is achieved thereby. With a single crossing-over, in this specific system a resistance to the antibiotic trimethoprim accompanies this, with a double crossing-over a corresponding sensitivity.<br>
[0010] In the application WO 96/23073 Al, a transposon-based system for the integration of multiple copies of the gene of interest into the bacterial chromosome is disclosed, which is characterized in that the marker gene of the plasmid is deleted by the integration and the strains contained are thus free_ of a resistance marker. Also, according to this specification, a marker is only needed for the control of the construction of<br>
1<br><br>
the bacterial strain concerned.<br>
[0011]  A system for	increasing	the copy number of /<br>
certain  transgenes	integrated	into  a  bacterial<br>
chromosome is also	disclosed	in the application<br>
WO 01/90393 Al.<br>
[0012] The second approach to the construction of producer strains consists in transferring the gene of interest to an autonomously replicating element, for example a plasmid, and in this manner transferring it to the host organism. The customarily high number of plasmid copies per cell has an advantageous effect in this case via the gene dose effect. The fact that over the entire culture period a selection pressure has to be exerted in order to keep the plasmids in the cells is disadvantageous. By default, this is carried out by the addition of antibiotics to the culture medium, whereas genes which impart resistance to the substances concerned are presented to the plasmids. Thus only the cells which possess the plasmids in adequate number are able to grow.<br>
[0013] The application of resistances to antibiotics as selection markers is recently increasingly running into criticism. On the one hand, the application of antibiotics is really expensive, in particular if the resistance is based on an enzyme degrading the antibiotic, and the substance concerned must therefore be added during the entire culturing. On the other hand, its widespread application, in particular in other technical fields, contributes to the spread of the resistance genes to other strains, even to pathological strains. This is already leading, for example, in medical hygiene and in particular in the treatment of infectious diseases to considerable difficulties due to 'multiresistant human-pathogenic strains'.<br>
r<br><br>
[0014] Therefore, to a great extent, regress is made to the systems illustrated above for the stable integration of genes into the chromosome of the producer cells, because these are stable without a permanent selection pressure. However, the strains concerned, as mentioned above, can only be prepared with great expense. It is quicker and more convenient in biotechnological practice to incorporate a, for example, newly found or modified gene on a plasmid with selection markers into host cells and to express it in this manner.<br>
[0015] In the prior art, antibiotic-free selection systems have meanwhile also been developed. For instance, in the publication "Transposon vectors containing non-antibiotic resistance selection markers for cloning and stable chromosomal insertion of foreign genes in gram-negative bacteria" by Herrero et al.<br>
(1990), in J. Bacteriol., Volume 172, pages 6557-6567, resistances to herbicides and heavy metals as selection markers are described. There are the same misgivings against the application of these compounds, however, as against antibiotics.<br>
[0016] Selection via auxotrophy, for example, that is to say via a specif.ic~:me5!lS^^<br>
cells ^concerned dependent on... the .....supply .of certain metabolic products, functions similarly in principle to an antibiotic selection. Auxotrophic strains then receive, coupled with the transgene of interest, one which cures this auxotrophy. In the case of loss, under appropriate culture conditions they would simultaneously lose their viability, such that the desired selection of the auxotrophic producer strains occurs. For instance, in the publication "Gene cloning in lactic streptococci" by de Vos in Netherlands Milk and Dairy Journal, Volume 4£, (1986), page 141-154, reference is made, for example, on p. 148 to various<br>
6<br><br>
selection markers developed from the metabolism of lacto-streptococci; among these are those from lactose metabolism, copper resistance and resistance genes to various bacteriocins of lacto-streptococci. The patent EP 284126 Bl, which looks into the problem of the stable integration of genes of interest into the ; bacterial chromosome (see above) summarizes the systems auxotrophy, resistance to biocides and resistance to virus infections possible for selection on p. 7 under the term "Survival selection". Examples of auxotrophy selection markers mentioned here are the metabolic genes leu, his, trp "or similar"; obviously those from J amino acid synthesis pathways are meant.<br>
[0017] In practice, the application of such auxo-trophies, however, has hitherto turned out to be very problematical, since in particular in industrial fermentation media almost all substrates necessary are available in adequate amounts and the cells concerned can compensate the shortage for the synthesis of a certain compound by means of the uptake of this same compound from the nutrient medium.<br>
[0018] An exception has hitherto only been the essential thymidine, which in industrial fermentation media only occurs in traces and therefore must be formed from the - proliferating and thus DNA-synthesizing - organisms by means of a thymidylate synthase. Thus the application EP 251579 A2 offers the<br>
j from Escherichia coli K12) and to cure the gene defect. If this vector additionally carries the gene for the protein of interest, an antibiotic-like selection of the producer cells occurs.<br>
7<br><br>
[0019] In summary, it must be realized that in the prior art although great experience with respect to the biotechnological production of proteins exists and the expression of genes of interest by means of chromosomal integration and antibiotic selection is systematically controllable, for these two systems up to now as good as no practical alternatives exist, in particular none which are less complicated than chromosomal integration and at the same time manage without selection by means of an expensive or ecologically questionable compound. In particular, the approach for the selection by means of auxotrophy markers has up to now led only to very scanty results with respect to the complex nutrient media generally customary in industry.<br>
[0020] The object was thus to develop a new selection system which is as comparatively simple to handle as selection via an antibiotic, but manages without expensive and, under certain circumstances, environmentally harmful substances. It should be utilizable on an industrial scale. It should not be based on an essential gene whose absence in industrial media can be compensated by contaminants.<br>
[0021] This object is achieved according to the invention by processes for the selection of a microorganism, which are characterized in that<br>
(a)	an endogenously present gene coding for an essential translocation activity is inactivated and<br>
(b)	the essential translocation activity inactivated according to (a) is cured by means of a vector.<br>
[0022] Surprisingly, it has been recognized that the essential factors involved in the__ translocation._are suitable for a selection. For the selection, according to the invention a gene is used whose derived protein<br>
8<br><br>
is involved in the protein translocation as a factor essential for the cell concerned. This means that the absence of this gene is lethal and thus an antibiotic¬like selection of microorganisms is possible.<br>
[0023] This selection system manages without additives (such as, for example, the antibiotics established in the prior art) and in principle functions independently of the composition of the nutrient media. For this, a molecular biological modification of the microorganisms concerned is necessary, which is described following the details for the translocation and is illustrated in the examples of the present application.<br>
[0024] The process of translocation involves the secretion of the proteins formed by bacteria into the periplasma (in the case of gram-negative bacteria), or the surrounding medium (both in the case of gram-negative and in the case of gram-positive bacteria). It is described, for example, in the paper of A.J. Driessen (1994): "How proteins cross the bacterial cytoplasmic membrane" in J". Membr. Biol., 142 (2) , pp. 145-59. The secretion apparatus necessary for this consists of a series of different, mainly membrane-associated proteins, which are shown in figure 1 of the present application. These include, in particular, the proteins SecA, SecD, SecF (together as the complex SecDF), E, G and Y well characterized, for example, for Bacillus subtilis (van Wely, K.H., Swaving, J., Freudl, R., Driessen, A.J. (2001): "Translocation of proteins across the cell envelope of Gram-positive bacteria", FEMS Microbiol Rev. 2001, 25 (4) , pp. 437-54) . Further factors to be considered part of this system are YajC, which likewise comes into direct contact with the Sec complex, and the factors Bdb (Dsb), SPase (for "signal peptidase"), PrsA and b-SRP (Ffh, Ffs/Scr, SRP-RNA) likewise to be recognized in figure 1.<br>
°l<br><br>
[0025] The last-mentioned factor is the bacterial factor, which in principle takes over the same function as the SRP (signal recognition particle) originally described in eukaryotes. A subunit of this is the factor Ffh, which is characterized both from B. Subtilis and from E. coli. The next subunit of b-SRP is called Scr in B. subtilis and Ffs in E. coli. Furthermore, an RNA (SRP-RNA) is part of the functional b-SRP complex. A further factor functionally associated herewith is called Srb in E. coli and FtsY in B. subtilis. It corresponds functionally to the eukaryotic docking protein.<br>
[0026] In a further context, for example, the factor PrfB (peptide chain release factor B; also RF2) is also to be included. It is known as part of the apparatus for protein synthesis, namely both in gram-positive and in gram-negative bacteria. It is responsible, in connection with the translation, in particular for the detachment of the ready-translated proteins of the ribosome, that is to say for the termination of the translation. The relationship to the translocation presented above is only indirectly afforded in that the gene prfB in many bacteria is transcribed simultaneously with the gene for the factor SecA. There is thus a regulatory relationship.<br>
[0027] The prerequisite for the translocation is that the proteins to be discharged have a signal peptide N-terminally (Park, S-, Liu, G. , Topping, T.B., Cover, W.H., Randall, L.L. (1988): "Modulation of folding pathways of exported proteins by the leader sequence", Science, 239, pp. 1033-5) . This applies both to extracellular proteins and to membrane proteins.<br>
[0028] After the translation has taken place on the ribosomes, the resulting peptide chain is kept in an unfolded  state  by  cytoplasmic  proteins  having  a<br>
to<br><br>
chaperone function and transported to the membrane. The transport of the peptide through the membrane is then catalyzed with consumption of ATP (Mitchell, C, Oliver, D. (1993) : Two distinct ATP-binding domains are needed to promote protein export by Escherichia coli SecA ATPase", Mol. Microbiol., 10 (3), pp. 483-97). SecA functions here as the energy-supplying component (ATPase) of the multienzyme complex translocase. After crossing the membrane, the signal peptide is cleaved by the activity of a signal peptidase and the extra¬cellular protein is detached from the membrane in this manner. In the case of gram-positive bacteria, the discharge of the exoproteins directly into the surrounding medium takes place in this manner. In the case of gram-negative bacteria, the proteins are subsequently found, as a rule, in the periplasma and further modifications are needed in order to achieve their release into the surrounding medium.<br>
[0029] A more recent investigation of B.v.d. Berg et al. (2004; Nature, Volume 427, pages 36 to 44) describes the X-ray structure of the translocase, that is to say the SecY complex from Methanococcus jannaschii, as a model for the corresponding complex in other organisms.<br>
[0030] The central molecule of this process is thus the preprotein translocase consisting of the subunits SecA, SecY, SecE, SecD, SecF (SecDF) and SecG. As the ATPase controlling this process, the factor SecA has a crucial function here for the translocation. The preferred embodiments of the present invention are characterized by these factors (see below).<br>
[0031] In table 1 below, the factors known up to now involved in the translocation are presented and classified as to whether they are essential in one of the  two  model  organisms Escherichia      coli      (gram-<br>
n<br><br>
negative) and Bacillus subtilis (gram-positive). Provided one of these is essential in the organism considered in each case, it is suitable for the selection according to the invention. Basically, it is to be assumed from this that a corresponding pattern also exists in other organisms, in particular gram-negative and gram-positive bacteria.<br>
Table 1: Protein factors known up to now involved in the translocation of gram-negative and gram-positive bacteria, classified according to whether they are essential in these organisms.<br><br>
	E.   coli	B.   subtilis<br>
SecA	essential	essential<br>
SecY	essential	essential<br>
SecE	essential	essential<br>
SecG	nonessential (cold-sensitive phenotype)	nonessential (cold-sensitive phenotype with overproduction of export proteins)<br>
SecD, SecF (SecDF)	essential	nonessential (cold-sensitive phenotype)<br>
Signal peptidase	essential	nonessential, since present in redundant form<br>
b-SRP (Ffh;<br>
Ffs/Scr;SRP-<br>
RNA)	essential	essential<br>
FtsY/Srb	essential	essential<br>
PrsA	not present	essential<br>
Bdb/Dsb	nonessential	nonessential<br>
YajC	essential	not known whether essential, but<br>
\y<br><br>
		present in redundant form<br>
[0032] According to the invention in gram-negative bacteria, in particular in coliform bacteria, very particularly in E. Coli, a selection can be made via the following translocating enzymes or their associated genes: SecA, SecY, SecE, SecD, SecF, signal peptidase, b-SRP (Ffh or FfS), Srb or YajC.<br>
[0033] According to the invention in gram-positive bacteria, in particular in Bacillus, very particularly in B. subtilis, a selection can be made via the following translocating enzymes or their associated genes: SecA, SecY, SecE, b-SRP (Ffh or Scr), FtsY or PrsA.<br>
[0034] The Or connection in these lists is not to be understood exclusively. Technically, it should be possible also to switch off a number of the associated genes simultaneously. According to the invention, however, it is sufficient to select only one for this.<br>
[0035]  In  each case,  individual  sequences  of  the associated genes are obtainable, for example, from the following  generally accessible  data banks:  GenBank (National Center For Biotechnology Information NCBI, National  Institutes of Health,  Bethesda,  MD,  USA; http://www3.ncbi.nlm.nih.gov);   EMBL   European  Bio-informatics institute (EBI) in Cambridge, Great Britain (http://www.ebi.ac.uk);   Swiss-Prot    (Geneva   Bio-informatics   (GeneBio)   S.A.,   Geneva,   Switzerland; http://www.genebio.com/sprot.html);    "Subtilist"   or "Colibri" of the Pasteur Institute,  25,  28 rue du Docteur Roux, 75724 Paris CEDEX 15, France for genes and   factors   from   B.        subtilis       or   E.        coli (http://genolist.pasteur.fr/SubtiList/   or   http:// genolist.pasteur.fr/Colibri/). Furthermore, resort can be made to the data banks which are reachable via<br>
tt<br><br>
cross-referencing in the data banks mentioned. According to the invention, it is in each case only necessary to identify and to use appropriately a single essential gene of the translocation apparatus in the strain intended for culturing.<br>
[0036] The sequences for the factor SecA from various microorganisms indicated in the sequence listing for the present application yield a further starting point. These can be used either directly (see below: preferred embodiments) or be employed in order to identify the homolog concerned in a gene bank which has been designed beforehand for the microorganism of interest.<br>
[0037] These translocating enzymes or factors are primarily to be understood as meaning the wild-type molecules. Provided it is possible, however, to prepare variants thereof which in principle have the same function as the wild-type enzyme in the translocation apparatus, selection systems constructed thereon are additionally included in the scope of protection.<br>
[0038] For the realization of the present invention, the strain intended for culturing, provided it does not belong to one of the species investigated in the present application, must first be checked as to whether at least one of these factors is essential. This is possible in a simple manner, for example by removing one of these known genes from a strain which is as closely related as possible (for example a likewise gram-negative or gram-positive bacterium) or by synthesizing it with the aid of an entry in a generally accessible data bank and incorporating it into a knock-out vector. A procedure of this type is generally known to the person skilled in the art. If the transformation with this vector and a subsequent -preferably initiated separately from the transformation - homologous recombination of this gene into the genome<br>
'1<br><br>
of the host cell has a lethal effect, the gene considered is to be regarded as essential. This gene which is recognized as essential can now be employed according to the invention as a selection marker and in particular according to the model of the examples of the present application.<br>
[0039] An inactivation necessary according to feature (a) is carried out, for example, by means of homologous recombination of an inactivated gene copy, which has been introduced into a cell of the microorganism strain of interest, for example by transformation with an appropriate vector. Methods for this are known per se. As a result of the recombination event, the chromosomal copy of the gene is completely or partially deleted and1 thus incapable of function. This can be carried out, for example, by means of the same gene with which the test for lethality has been carried out beforehand. Preferably, however, the endogenous homolog, provided it is known or can be isolated with justifiable expenditure, is employed in order to achieve a high success rate for the recombination. Whether the inactivation is successful is decisive for the accomplishment of the invention.<br>
[0040] This concept can be realized, for example, with plasmid vectors which possess a temperature-sensitive replication origin and into which the homologous DNA regions of the gene to be deleted coming into consideration have additionally been inserted (deletion vector). A reversible inactivation, for example, would also be conceivable, for example by means of integration of a mobile genetic element, for example a transposon, into the target gene.<br>
[0041] In this context, in each case feature (b) is to be taken into account, namely that even before this recombination or inactivation event, or at the latest<br>
fb<br><br>
simultaneously, an intact copy of the gene selected for the selection according to the invention is prepared in the cell concerned, because the cell would otherwise not  survive  the  inactivation.  According  to  the invention, the resulting defect is compensated by means of a vector,  that is to say the vector cures the inactivation. In this context, as mentioned above, the genes  endogenously present  in the  host  cells  and deleted according to (a) are preferably used. However, functionally  identical  genes  from other organisms, preferably  related  strains,  can  also  be  employed provided they are able to cure the defect concerned. Thus it is possible, for example, to cure the defect of SecA of a B.   subtilis  by provision of a secA  gene from Staphylococcus carnosus.<br>
[0042] It would also be conceivable that by means of the vector another genetic element abolishing the first defect is brought into the cell, for example the gene of a factor which is in principle identical functionally, but modified by mutation.<br>
[0043] In this cell, a situation thus prevails in which a lethal defect is compensated by means of a separate genetic element. A loss of this separate genetic element would in turn be lethal, so that such a cell is forced in the case of any cell division to pass on this element to the subsequent generation.<br>
[0044] Feature (c) takes advantage of this in a preferred embodiment (see below), according to which the vector compensating the gene defect carries the transgene, precisely the gene for the protein of interest which is to be prepared by means of the process according to the invention.<br>
[0045] An endogenous selection pressure to a certain extent  prevails,  without  the  addition  of  another<br>
Ih<br><br>
compound, for example of a heavy metal or of an antibiotic, being necessary from outside, that is to say via the nutrient medium, in order to prevent the loss of the vector having the transgene. On the other hand, the complicated modifications discussed at the outset in order to integrate the transgene itself into the chromosomal DNA are inapplicable. For instance, a once-produced microorganism strain, which is prepared for a defined inactivation of the translocation apparatus, can be used for ever new transformations using similarly constructed vectors, which each time make available the same function curing the gene defect, but in each case carry various transgenes. A selection system which is very practical and can be employed in a versatile manner is thus available.<br>
[0046] The purpose of the selection process according to the invention consists in obtaining a genetic element which is stable in the cell over a number of generations. This element is precisely the vector by means of which the inactivation of the essential translocation activity is compensated, that is to say is cured.<br>
[0047] In a preferred embodiment, selection processes according to the invention are characterized in that<br>
(c)  the  vector  according  to  (b)  carries  a transgene.<br>
[0048] This, then, is the technically most important field of application of selection systems. The genetic element which is stable over a number of generations is then one which carries a transgene, in particular one whose gene product is of commercial interest. Preferred embodiments thereof are carried out further below.<br>
[0049] In preferred embodiments, a selection process according to the invention is characterized in that the<br>
n<br><br>
essential  translocation  activity	is  one  of  the<br>
following factors: SecA, SecY, SecE,	SecD, SecF, signal<br>
peptidase, b-SRP (Ffh or Ffs/Scr),	FtsY/Srb, PrsA or<br>
YajC.<br>
[0050] As compiled in Table 1, these factors or the associated genes are then those of which it is known either from E. coli or from B. subtilis that they are essential. It is therefore obvious, in particular in these two organisms, but also in related or even less related species, to establish a selection system according to the invention by means of the homologs concerned. Since it is known that even individual members of these genes can substitute the function concerned in the other organisms in each case, that is to say over and beyond the limit gram-negative/gram-positive, at least individual members of the genes concerned even from only distantly related species should be employable according to the invention.<br>
[0051] Preferably, the essential translocation activity is one of the following subunits of the preprotein translocase: SecA, SecY, SecE, SecD or SecF, preferably the subunit SecA.<br>
[0052] These factors then to a certain extent represent, as shown in figure 1, the functional core of the translocation apparatus. For SecA, it has been explained further above that this factor occupies an important key position in the ATPase activity. For this reason, the invention has also been transposed in the examples to the present application with the aid of the gene secA.<br>
[0053] Preferably, selection processes according to the invention are characterized in that the curing according to (b) takes place by means of an activity acting  identically  to  the  inactivated  endogenously<br>
If<br><br>
present essential translocation activity, preferably by means of a genetically related activity, particularly preferably by means of the same activity.<br>
[0054] It is reflected therein that on account of the generally high homology values between the species for the factors concerned, the genes from less closely related species concerned can also be employed promisingly. However, of course those from more closely related species and very particularly from the same organisms are preferred, because these are the most promising with respect to the crossing-over necessary for inactivation. It may again be pointed out that only a single gene suitable for the inactivation suffices in order to realize a selection according to the invention.<br>
[0055] As already said above, the DNA and amino acid sequences concerned are obtainable from generally accessible data banks. For instance, the sequences for the protein SecA from B. subtilis from the data bank "Subtilist" of the Pasteur Institute (see above) indicated in the sequence listing under SEQ ID NO. 1 and 2 have been removed (date: 2. 3. 2003); they are identical with that of Swiss-Prot (see above) which are deposited there under the accession number P28366.<br>
[0056] The sequences indicated in the sequence protocol under SEQ ID NO. 3 and 4 for the protein SecA from E. coli originate from the data bank "Colibri" of the Pasteur Institute (see above; date: 2.3.2003); they are identical to that of Swiss-Prot (see above), which can be retrieved there under the accession number P10408.<br>
[0057] SEQ ID NO. 5 and 6 for B. licheniformis were obtained from the commercially obtainable strain B. licheniformis (DSM13) as described in example 1 of the present    application    (Deutsche    Sammlung    von<br>
'1<br><br>
Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg lb, 38124 Brunswick; http://www.dsmz.de).<br>
[0058] Because the species Bacillus subtilis, Escherichia coli and Bacillus licheniformis are the technically most frequently employed microorganisms, it is particularly important to make available a process according to the invention for these bacteria. Using the sequence listing for the present application, the homologous secA genes from these three most important organisms are made available without them having to be isolated for copying. By means of these genes, it is, for example, possible to identify the homologs concerned in other microorganisms, for example by means of preparation of a gene bank and screening using one of these genes as a probe. In particular in related species, it is, however, also possible to employ these genes themselves for an inactivation according to the invention.<br>
[0059] Preferred embodiments are thus characterized in that the curing according to (b) takes place by means of the regions of the gene secA from Bacillus subtilis, Escherichia coli and Bacillus licheniformis restoring the translocation activity, which are indicated in the sequence listing under SEQ ID NO. 1, SEQ ID NO. 3 and SEQ ID NO. 5 respectively.<br>
[0060] Preferred processes are moreover characterized in that the inactivation according to (a) takes place such that a recombination between the gene region inactivated according to (a) and the homologous region on the vector according to (b) is prevented, preferably with complete loss of a gene section contained in the chromosomal gene concerned.<br>
[0061] If the vector was integrated into the chromosome of  the  host  cell,  the  lethal  mutation  would be<br>
c&amp;o<br><br>
permanently cured without a selection pressure on the vector concerned existing simultaneously. By this means, the actually interesting transgene could be lost by means of the following cell divisions. Extensive deletion during the inactivation step (a) prevents this.<br>
[0062] In the prior art, in particular in the publication "Genetic manipulation of Bacillus amyloliquefaciens" by J. Vehmaanpera et al. (1991) in J. Biotechnol. , Volume 3^9, pages 221-240, processes for the inactivation of genes by means of a deletion vector are described. With the aid of this description, it was possible in example 3 to carry out the deletion of the gene secA from B. licheniformis successfully. The replication origin of this deletion vector is distinguished by its temperature dependence.<br>
[0063] It is particularly easily possible thereby first to select on a successful transformation at relatively low temperature and subsequently, by increasing the temperature, to exert a selection pressure on a successful integration, that is to say inactivation of the endogenous gene. Analogously, for example, a construct for control by means of the addition of low molecular weight compounds would also be possible.<br>
[0064] Preferred processes according to the invention are consequently characterized in that the inactivation according to (a) is carried out by means of a deletion vector, preferably by means of a deletion vector having an externally regulatable replication origin, particularly preferably by means of a deletion vector having a temperature-dependent replication origin.<br>
[0065] As explained above, in principle it is possible that the curing vector according to (b) , including the transgene, is integrated into the bacterial chromosome.<br>
o*l<br><br>
As here, however, the danger exists that the transgene in this case is permanently lost, preferred processes are characterized in that the vector according to (b) is a plasmid autonomously replicating in the microorganism. This then establishes itself in the derived cell line.<br>
[0066] It is particularly advantageous if the plasmid is a plasmid establishing itself in a plural copy number (for example 2 to 100 plasmids per cell), preferably in a multiple copy number (more than 100 plasmids per cell) . Then the more plasmid copies are present, the more successful the curing may be. Moreover, this increases, in the case in which the vector carries the transgene coding for a protein of interest, the achievable yield of this product by means of the gene dose effect.<br>
[0067] On account of the great importance of gram-negative strains of bacteria, in particular in the cloning and characterization of genes or gene products, preferred selection processes are characterized in that the microorganism is a gram-negative strain of bacteria.<br>
[0068] Among these, in particular, are to be understood<br>
processes which are characterized in that a gram-<br>
negative strain of bacteria of the genera E. coli or<br>
Klebsiella is concerned, in particular derivatives of<br>
Escherichia coli K12, of Escherichia coli B or<br>
Klebsiella        planticola,	and   very   particularly<br>
derivatives of the strains Escherichia coli BL21 (DE3), E. coli RV308, E. coli DH5a, E. coli JM109, E. coli XL-1 or Klebsiella planticola (Rf). These are the organisms most frequently employed in molecular biology.<br>
[0069]   In  particular  for  fermentative  protein<br>
Al<br><br>
production, gram-positive bacteria are of particular importance. This applies to a particular extent for secreted proteins. Preferred processes according to the invention are therefore characterized in that the microorganism is a gram-positive strain of bacteria.<br>
[0070] Among these, in particular in industry, gram-<br>
positive strains of bacteria of the genera<br>
Staphylococcus, Corynebacteria or Bacillus are<br>
established, in particular of the species<br>
Staphylococcus carnosus, Corynebacterium glutamicum,<br>
Bacillus	subtilis,	B.	licheniformis,	B.<br>
amyloliquefaciens, B. globigii or B. lentus, and very particularly derivatives of the strains B. licheniformis or B. amyloliquefaciens, which is why these characterize correspondingly preferred selection processes.<br>
[0071] Of very particular interest are processes according to the invention which are directed at certain products produced by the culturing of the microorganisms. Correspondingly preferred selection processes are thus those which are characterized in that the transgene according to (c) is one which codes for a nonenzyme protein, in particular for a pharmacologically relevant protein, very particularly for insulin or calcitonin.<br>
[0072] However, enzymes are also of great industrial importance. Thus, according to the invention those processes are also claimed which are characterized in that the transgene according to (c) is one which codes for an enzyme, preferably for a hydrolytic enzyme or an oxidoreductase, particularly preferably for a protease, amylase, hemicellulase, cellulase, lipase, cutinase, oxidase, peroxidase or laccase. Of these, representatives preferably produced for application in detergents and cleansers may be explained below.<br>
JS<br><br>
[0073] In principle, compositions of this type can be employed for increasing the detergent, or cleansing, power of all enzymes established in the prior art. Among the proteases, those of the subtilisin type are preferred, for example the alkaline protease of Bacillus lentus. From the protease of Bacillus lentus DSM 5483 (WO 91/02792 Al) are derived the variants used under the name BLAP®, which are described in particular in WO 92/21760 Al, WO 95/23221 Al, WO 02/088340 A2 and WO 03/038082 A2. Further proteases from various Bacillus sp. and B. gibsonii which can be prepared according to the invention are evident from the patent applications WO 03/054185 Al, WO 03/056017 A2, WO 03/055974 A2 and WO 03/054184 Al.<br>
[0074] Examples of amylases which can be prepared according to the invention are the a-amylases from JBacillus licheniformis, from B. amyloliquefaciens or from B. stearothermophilus, and their improved further developments, in particular for application in detergents and cleansers. In addition, the a-amylase from Bacillus sp. A 7-7 (DSM 12368) disclosed in the application WO 02/10356 A2 and the cyclodextrin glucanotransferase (CGTase) from B. agaradherens (DSM 9948) described in the application WO 02/44350 A2 are to be emphasized for this purpose. Furthermore, the amylolytic enzymes lie at the focus of the present application which belong to the sequence sector of a-amylases, which is defined in the application WO 03/002711 A2, and those which are described in the application WO 03/054177 A2. Likewise, fusion products of the molecules mentioned are meant, for example those from the application DE 10138753 Al.<br>
[0075] According to the invention, lipases or cutinases can also be produced, for example those originally obtainable  from Humicola      lanuginosa       (Thermomyces<br>
3x,<br><br>
lanuginosus), or further-developed lipases, in particular those having the amino acid exchange D96L or the lipases or cutinases whose starting enzymes have originally been isolated from Pseudomonas mendocina and Fusarium solanii.<br>
[0076] Furthermore, it is in particular intended for the industrial treatment of textiles, but also for detergents on cellulases, for example endoglucanase, and/or cellobiohydrolases. For example, cellulases which can also be produced by the natural producers are those from Bacillus sp. CBS 670.93 and CBS 669.93, such as are disclosed in WO 96/34092 A2. Furthermore, it is possible according to the invention to produce further enzymes which are summarized under the term hemicellulases. These include, for example, mannanases, xanthan lyases, pectin lyases (= pectinases), pectin esterases, pectate lyases, xyloglucanases (= xylan-ases) , pullulanases and (3-glucanases.<br>
[0077] The detergent and cleanser enzymes likewise include oxidoreductases, for example oxidases, oxygenases, catalases, peroxidases, such as halo-, chloro-, bromo-, lignin, glucose or manganese peroxidases, dioxygenases or laccases (phenol oxidases, polyphenol oxidases) or all other enzymes described in the prior art for this area of use.<br>
[0078] As already explained by way of introduction, the object underlying the present invention particularly clearly stood against the background of large-scale fermentation. Since precisely there the disadvantages appeared that, on the one hand, a selection by means of antibiotics is expensive and problematic from environmental points of view, on the other hand auxotrophy based on metabolic defects can be compensated on account of the complexity of industrial media.<br>
X?<br><br>
[0079] The conversion of a selection process according to the invention to a large-scale process is therefore of particular importance, for example for the production of low molecular weight compounds such as antibiotics or vitamins or very particularly for protein production.<br>
[0080] Processes for the production of a protein by culturing cells of a microorganism strain are generally known in the prior art. They are based on the fact that cells which produce the protein of interest naturally or after transformation with the gene concerned are taken into culture in a suitable manner and preferably stimulated for the formation of the protein of interest.<br>
[0081] Processes for the production of a protein by culturing cells of a microorganism strain thus form a further subject of the invention, and are characterized by a selection process described beforehand. On this, in particular the embodiments already presented further above stress that the curing vector itself contains a transgene and that this preferably codes for a non¬enzyme protein or for an enzyme. Among these, commercially important proteins are in particular to be understood. Thus, transgenically produced insulin is employed for the treatment of diabetes, and a broad spectrum of enzymes, of proteases, lipases and amylases up to oxidative enzymes is employed for the production of detergents and cleansers.<br>
[0082] On account of the simple industrial handleability, if the transgene of interest and the curing gene are made available on a vector, those protein production processes according to the invention are preferred which are characterized in that the transgene  according  to  (c)  codes  for  the  protein<br>
Ms<br><br>
produced by means of this process.<br>
[0083] In principle, bacteria can be used on a solid surface. This is in particular of importance for testing their metabolic properties or for permanent culture on the laboratory scale. For the production of proteins, on the other hand, processes are preferred which are characterized in that the culture of the microorganisms takes place in a liquid medium, preferably in a fermenter. Techniques of this type are widespread in the prior art and are developed further according to the invention precisely by the transformation to selection by means of essential translocation factors.<br>
[0084] Of particular importance are protein production processes which are characterized in that the protein of interest is secreted into the surrounding medium. By this means, the workup of the product obtained is significantly facilitated. A possible alternative according to the invention, however, also consists in breaking down the cells concerned producing the protein following the actual production and thereby obtaining the product.<br>
[0085] In principle, a new strain is produced by each molecular biological change to a microorganism. Thus, new microorganism strains are also produced by transformation of the present invention, precisely those which differ from the starting strain (to put it more precisely: from the starting cell) by the specific inactivation of an essential translocation activity and its curing by provision of an identically acting translocation factor. Novel microorganisms are thus produced by use of a selection process according to invention.<br>
[0086] Thus microorganisms which have been obtained by<br>
J7<br><br>
a selection process described beforehand fall within the scope of protection of the present application.<br>
[0087] A particularly advantageous aspect consists in the fact that a group-related microorganism is obtained by always carrying out the same type of inactivation and curing on the curing vector but each time preparing another transgene. A process, once used successfully, can in this way be transferred to innumerable other selection problems.<br>
[0088] For the realization, in particular, of the protein production processes explained above, it is necessary that the transgene is expressed. By means of this, microorganisms preferred according to the invention are characterized.<br>
[0089] Among these, according to what has been said above for protein production, the microorganisms are preferred which are characterized in that the transgene is secreted.<br>
[0090] According to what has been said above, it was only possible to find selection systems according to the invention in that essential genes coding for translocation activities were recognized as such, by which selection is possible. Use of these genes of this type has up to now still not been thought about, although numerous of these are known from a large number of microorganisms. Precisely this knowledge works to the advantage of selection systems according to the invention, since virtually for all microorganisms genes can thus be defined by means of which an appropriate selection is possible. For this, they only have to be inactivated as explained above and are substituted in the cell concerned by a functioning homolog.<br>
M<br><br>
[0091] One subject of the invention thus forms the use of a gene coding for an essential translocation activity for the selection of a microorganism, which is characterized in that<br>
(a)	this essential translocation activity which is endogenously present in the microorganism is inactivated and<br>
(b)	the essential translocation activity inactivated according to (a) is cured by means of a vector.<br>
[0092] The embodiments of this subject of the invention explained below are accordingly preferred corresponding to the previous presentations.<br>
[0093] It is of particular molecular genetic and technical interest if this use is characterized in that<br>
(c)	the vector according to (b) carries a<br>
transgene.<br>
[0094] A key point of the present invention forms any appropriate use which is characterized in that the essential translocation activity is one of the following factors: SecA, SecY, SecE, SecD, SecF, signal peptidase, b-SRP (Ffh or Ffs/Scr), FtsY/Srb, PrsA or YajC.<br>
[0095] Among these, any use is preferred which is based on the essential translocation activity of one of the following subunits of the preprotein translocase: SecA, SecY, SecE, SecD or SecF, preferably the subunit SecA.<br>
[0096] An advantageous use consists in carrying out the curing according to (b) by means of an activity acting identically to the inactivated endogenously present essential translocation activity, preferably by means of a genetically related activity, particularly preferably via the same activity.<br><br>
[0097] For this, the present application makes available some appropriately preferred starting points, namely the use possibilities mentioned, which in particular are characterized in that the curing according to (b) is carried out by means of the regions of the gene secA from Bacillus subtilis, Escherichia coli or Bacillus licheniformis restoring the translocation activity, which are indicated in the sequence listing under SEQ ID NO. 1, SEQ ID NO. 3 and SEQ ID NO. 5 respectively.<br>
[0098] Particularly advantageous uses are characterized in that the vector according to (b) is a plasmid autonomously replicating in the microorganism.<br>
[0099] Further preferred uses are characterized in that the plasmid is a plasmid establishing in a plural, preferably in a multiple, copy number.<br>
[0100] Finally the present invention is also realized by the provision of appropriate vectors. Vectors are intended hereby which carry a gene for an essential translocation activity and a transgene capable of expression which, however, if it is present as a single transgene, does not code for an antibiotic resistance.<br>
[0101] The exclusion of genes for an antibiotic resistance, in the case where it is here the only transgene in addition to the gene curing the essential translocation activity inactivated according to the invention, is for the purpose of the present application not technically limited. It takes into account only that in the prior art, in connection with the characterization of the translocation proteins which can be used according to the invention, precisely these are already described. This, of course, also includes that they have been sequenced and cloned,<br>
&gt;<br><br>
namely by means of the most widespread cloning vectors in the prior art, precisely those which contain, as markers, genes for an antibiotic resistance. Thus, vectors which contain -only a gene for an essential translocating enzyme and an antibiotic marker are known in the prior art.<br>
[0102] Corresponding to what has been said above, a vector according to the invention is preferably characterized in that the transgene contained on it, preferably intended for protein production, codes for a pharmacologically relevant nonenzyme protein or for a hydrolytic enzyme or for an oxidoreductase. These are characterized in that the gene to be expressed is provided with a functioning promoter. Here, of course, all such constructs are included in the scope of protection which also code for - possibly pharmacologically interesting - factors, which can mediate antibiotic resistance provided the presence of this vector is selected not by means of this property but by means of the essential translocation activity.<br>
[0103] According to the details of the selection system, certain vectors also represent preferred embodiments of the present invention.<br>
[0104] These include appropriate vectors which are characterized in that the translocation activity encoded by them is able to cure an inactivated essential translocation activity endogenously present in a microorganism strain, preferably by means of a genetically related activity, particularly preferably by means of the same activity.<br>
[0105] Further preferably, these include appropriate vectors which are characterized in that the essential translocation activity is one of the following factors: SecA, SecY, SecE, SecD, SecF, signal peptidase, b-SRP<br>
3)<br><br>
(Ffh or Ffs/Scr), FtsY/Srb, PrsA or YajC.<br>
[0106] Among these, the vectors are preferred which are characterized in that the essential translocation activity is one of the following subunits of the pre-protein translocase: SecA, SecY, SecE, secD or SecF, preferably the subunit SecA.<br>
[0107] According to the teaching of the present application, the vectors are furthermore preferred which are characterized in that the essential translocation activity is one of the genes secA from Bacillus subtilis, Escherichia coli or Bacillus licheniformis, which are indicated in the sequence listing under SEQ ID NO. 1, SEQ ID NO. 3 and SEQ ID NO. 5 respectively.<br>
[0108] Furthermore, it is a favorable property of the vectors according to the invention if they are plasmids replicating autonomously in the microorganism used.<br>
[0109] In this connection, in particular because of the gene dose effect, it is particularly advantageous if the plasmids are characterized in that they are in this case plasmids establishing in a plural, preferably in a multiple, copy number.<br>
Examples<br>
[0110] All molecular biological operations follow standard methods, such as are indicated, for example, in the handbook by Fritsch, Sambrook and Maniatis "Molecular cloning: a laboratory manual", Cold Spring Harbor Laboratory Press, New York, 1989, or comparable relevant works. Enzymes and kits were employed according to the details of the respective manufacturer.<br>
3V<br><br>
[0111] Example 1 Isolation of the gene secA from B.   licheniformis<br>
[0112]   Identification  of  the secA      locus  in B. licheniformis<br>
For the identification of the secA/prfB locus in B. licheniformis, a gene probe was derived by means of PCR with the aid of the known sequence of the prfB-secA gene locus of B. subtilis (databank "Subtilist" of the Pasteur Institute, 25, 28 rue du Docteur Roux, 75724 Paris CEDEX 15, France; http://genolist.pasteur.fr/ SubtiList/; date: 8.16.2002). This gene locus is also shown in figure 2. The probe obtained was 3113 bp long and additionally comprised the first 451 bp of the N-terminal region of the gene prfB. Subsequently, preparations of chromosomal DNA of B. licheniformis, which is obtainable, for example, from Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg lb, 38124 Brunswick (http:// www.dsmz.de) under the order number 13, and, for the control, chromosomal DNA of B. subtilis were digested using various restriction enzymes and subjected to a Southern hybridization using the probe mentioned. On the chromosomal DNA of B. licheniformis treated with the restriction enzyme Muni, a single fragment of a size of about 5.5 kB was identified, while the digestion of the chromosomal DNA of B. subtilis using Muni yielded the fragments expected for B.   subtilis.<br>
[0113]  Cloning  of  the  identified  region  from B. licheniformis  DSM 13<br>
Chromosomal DNA of the same strain B. licheniformis was isolated, digested preparatively using Muni and the DNA region around 5.5 kB was isolated by means of agarose gel electrophoresis, and the nucleic acids were extracted therefrom using commercially obtainable kits. The mixture of Muni-cleaved DNA fragments obtained was ligated in the Muni-compatible BcoRI cleavage site of<br>
3}<br><br>
the low-copy-number vector pHSG575 (described in: "High-copy-number and low-copy-number vectors for lacZ alpha-complementation and chloramphenicol- or kanamycin-resistance selection"; S. Takeshita; M. Sato; M. Toba; W. Masahashi; T. Hashimoto-Gotoh; Gene (1987), Volume 6_1, pages 63-74) and transformed in E. coli JM109 (obtainable from Promega, Mannheim, Germany).<br>
[0114] Selection of the resistance encoded by the vector was carried out. Additionally, the method of blue/white screening (selection plates contained 80 ug/ml of X-Gal) served for the identification of clones which had taken up a vector having an insert. In the process 2 00 clones were obtained, of which it was possible by means of colony hybridization to identify 5 clones which carried the B. licheniformis secA gene. These were checked by fresh Southern blot analysis using the probe described above and a vector derived from pHSG575 containing the Muni fragment carrying the secA gene of B. licheniformis and comprising 5.5 kB was carried on under the name pHMHl.<br>
[0115] Restriction analysis<br>
The cloned 5.5 kB region was first characterized by means of restriction mapping. For this, using various enzymes, individual and double digestions of pHMHl were carried out and by means of Southern blot analysis those fragments were identified which carry parts of the secA/prfB operon. The restriction map resulting therefrom was supplemented after complete sequencing of the 5.5 kB fragment (see below) and can be seen in figure 3.<br>
[0116] Sequence analysis<br>
The fragment 5.5 kB in size shown in figure 3 was sequenced into subsequences according to standard methods. The subsequences showed strong homologies with the following genes from B. subtilis: fliT   (codes for a<br>
2<f></f><br>
flagellar protein), orfl89/yvyD (unknown function) , secA (translocase-binding subunit; ATPase) and prfB (peptide chain release factor 2) , in exactly the same gene sequence as in B. subtilis. These are likewise shown in figure 3.<br>
[0117] On the basis of these values, it can also be assumed therefrom that the SecA from B. licheniformis exerts the same biochemical activity as, in particular, the SecA from B. subtilis and thereby takes over the same physiological function. It is thus to be considered as an essential enzyme included in the translocation.<br>
[0118] The DNA sequence and the amino acid sequence derived therefrom determined according to this example are given in the sequence listing under SEQ ID NO. 5 or 6. Accordingly, the translation start lies in the position 154 and the stop codon in the positions 2677 to 2679. The subsequence from the positions 60 to 65 or 77 to 82 is presumably to be regarded as a promoter region and the region from position 138 to 144 as a ribosome binding site.<br>
[0119] Example 2<br>
Preparation of a plasmid containing a secA gene and<br>
subtilisin gene<br>
[0120] The secA gene obtained according to example 1 was amplified using its own promoter by means of PCR starting from chromosomal DNA from B. licheniformis. For this, as shown in figure 4, with the aid of the DNA sequence of the gene of B. licheniformis primers were selected which at the respective 5'-end possess a BamH.1 restriction cleavage site. By means of this, the fragment amplified using these primers was cloned into the cleavage site of the plasmid pCB56C.  This  is<br>
3<r></r><br>
described in the application WO 91/02792 Al and contains the gene for the alkaline protease from B. lentus   (BLAP) .<br>
[0121] This cloning strategy, also shown in figure 4, yielded the vector pCB56CsecA 8319 bp in size which, in addition to the genes secA and BLAP, also contains one which codes for a tetracycline resistance.<br>
[0122] This vector pCB56CsecA and, for the control,	the<br>
starting  vector  pCB56C  were  transformed  in	B.<br>
licheniformis,    mainly in the case of pCB56C in	the<br>
wild-type strain B.   licheniformis   (secA)   capable of	the<br>
formation of SecA. In the case of pCB56CsecA,	the<br>
transformation was carried out such that the endogenous<br>
secA was simultaneously inactivated. The procedure	for<br>
this is described in example 3.<br>
[0123] In this manner, the two strains B.   licheniformis<br>
(ASecA) pCB56CsecA and B.    licheniformis    (secA)    pCB56C<br>
were obtained, which were both able to express the<br>
plasmid-encoded gene for the alkaline protease. They<br>
are further investigated in example 4.<br>
[0124] Example 3<br>
Preparation  of  the  strain B. licheniformis     (AsecA)<br>
pCB56CsecA<br>
[0125] The switching off of the gene secA was performed by means of a deletion vector. The procedure follows the description of J. Vehmaanpera et al. (1991) in J. Biotechnol.,   Volume ^9, pages 221-240.<br>
[0126] The vector selected for secA deletion was the plasmid pE194 described in the same publication. The advantage of this deletion vector consists in the fact that it possesses a temperature-dependent replication<br>
?4<br><br>
origin. At 33°C, pE194 can replicate in the cell, such that a successful transformation can first be selected at this temperature. Subsequently, the cells which contain the vector are incubated at 42°C. At this temperature, the deletion vector no longer replicates and a selection pressure is exerted on the integration of the plasmid into the chromosome by means of one of the two homologous regions (up- or downstream region of secA) . A further homologous recombination by means of the other (second) homologous region then leads to the deletion of secA. A repeated recombination of the first homologous region would also be possible. In this connection, the vector recombines again from the chromosome, such that the chromosomal secA is retained. The secA deletion must therefore be detected in the Southern blot after restriction of the chromosomal DNA using suitable enzymes or with the aid of the PCR technique by means of the size of the amplified region.<br>
[0127] For the construction of the deletion vector, the regions located up- and downstream of secA (figure 5) were amplified by means of PCR. The primers for the amplification and the restriction cleavage sites for subsequent cloning (Xbal and EcoRV) associated with these were selected with the aid of the DNA sequence of the secA/prfB locus of B. licheniformis determined according to example 1. In the case of the SecA deletion, it should be taken into consideration that the prfB located downstream of secA lies in one operon with secA,    that is possesses no promoter of its own<br>
(compare figure 2). The prfB codes for the protein RF2, which in connection with the protein biosynthesis ensures the detachment of the protein from the ribosome. In order to guarantee the transcription of the prfB, which is important for protein biosynthesis, even after SecA deletion, the orfl89 with its own terminator situated before the secA and the secA promoter located downstream was amplified such that the<br>
T7<br><br>
prfB  can be transcribed directly from the secA  promoter after secA  deletion (figure 5).<br>
[0128] The amplified regions (orf!89' and prfB') were intercloned into the E. coli vector pBBRMCS2 in a control step. The subsequent sequencing of the orf189'prfB' construct showed that the amplified fragments were cloned together correctly.<br>
[0129] The orf189'prfB' construct was recloned in the next step into the vector pE194 in B. subtilis DB104 selected for the deletion (figure 6) . In this context, using the method of protoplast transformation according to Chang &amp; Cohen, 1979, transformants were obtained which carried the deletion vector pEorfprfB. All operations were carried out at 33°C in order to guarantee replication of the vector.<br>
[0130] In a next step, the vector pCB56CsecA described in example 2 was likewise transformed into the host strain B. licheniformis carrying the plasmid pEorfprfB by means of the method of protoplast transformation. The transformants obtained in such a way and identified as positive using customary methods were subsequently selected for the presence of both plasmids at 42°C under selection pressure (tetracycline for pCB56CsecA and erythromycin for pEorfprfB). At this temperature, the deletion vector can no longer replicate and only those cells survive in which the vector is integrated into the chromosome, this integration taking place with the highest probability in homologous or identical regions. By culturing at 3 3°C without erythromycin selection pressure, the excision of the deletion vector :an subsequently be induced, the chromosomally encoded 3ene secA being removed from the chromosome completely.<br>
[0131] The plasmid pCB56CsecA, which mediates the ability  for  subtilisin  synthesis  and  also  makes<br>
3&gt;£<br><br>
available the essential translocation factor secA, remains in the cell. The strain obtained in this manner was designated by B.   licheniformis   (AsecA) pCB56CsecA.<br>
[0132] Example 4<br>
Investigation of the plasmid stability<br>
[0133] For the determination of the genetic stability of the secA-carrying subtilisin plasmid pCB56CsecA, the two strains B. licheniformis (secA) pCB56C and B. licheniformis (AsecA) pCB56CsecA obtained according to examples 2 and 3 were investigated in liquid medium in a shaker flask experiment without addition of antibiotic. For this, starting from one individual colony each, an overnight culture was grown and using this 14 ml of LB medium in each case (according to standard recipe) were inoculated to an optical density at 600 nm (OD600) of 0.05. Culturing was carried out in a 100 ml Erlenmeyer shaker flask. After 8 to 16 hours in each case, the cultures were inoculated into 14 ml of fresh medium and here in turn an OD6oo of 0.05 was set. The culturing was carried out over 8 days and aights; the cultures were inoculated altogether 16 times in this process. Every day, dilution series were plated out and a random selection of the clones sbtained was transferred to protease test plates. The result is shown in table 2 and in figure 7.<br>
Table 2: Plasmid stability in the transformants B. licheniformis {secA) pCB56C (control) and B. licheniformis (AsecA) pCB56CsecA, detectable with the aid of the respective fraction of the clones with protease activity<br><br><br><br>
[0134] For each culturing time, all clones of the strain B. licheniformis (AsecA) pCB56CsecA show protease activity, while with the strain S. licheniformis (secA) pCB56C individual clones no longer possess any protease activity. This is to be interpreted as a loss of the plasmid pCB56C; this loss was additionally checked by plasmid minipreparation.<br>
[0135] By means of these data, it is thus clear that on culturing in LB medium without antibiotic addition, in particular without tetracycline, for which the plasmid would impart resistance, the secA-carrying subtilisin plasmid pCB56CsecA is stable in the AsecA strain, while the subtilisin plasmid pCB56C in the strain without chromosomal secA deletion is lost in the course of culturing. The gene secA from B. licheniformis can thus cure the chromosomal secA deficiency on transfer to an expression vector and in this manner can be utilized for the selection of a bacterial culture which expresses a gene for another protein, in this case an alkaline protease.<br><br>
Description of the figures<br>
Figure 1: Schematic representation of the translation/translocation apparatus of gram-positive bacteria<br>
Analogously according to van Wely, K.H., Swaving, J., Freudl, R. , Driessen, A.J. (2001); "Translocation of proteins across the cell envelope of Gram-positive bacteria", FEMS Microbiol Rev. 2001, 25(4), pp. 437-54) .<br>
Figure 2:      Gene locus of secA  in B.   subtills<br>
It is recognized that the gene prfB also lies in the secA region and a continuous mRNA is formed, so that it is also possible to speak of a secA/prfB  operon.<br>
Figure 3: Restriction map' of the gene locus orfl89/secA/prfB  in B.   licheniformis<br>
As shown in example 1, in the immediate vicinity to secA are found the gene prfB and an orf on a fragment about 5.5 kB in size, which can be obtained by restriction with Muni from the genomic DNA of B. 1icheniformis.<br>
Figure 4: Preparation of a plasxnid having a secA gene and a subtilisin gene<br>
As described in example 2, the secA was amplified by means of PCR and cloned into a vector which contains as a transgene the alkaline protease from B.   lentus.<br>
Figure 5: Regions of secA (up- and downstream) amplified by means of PCR<br>
Amplification of the up- and downstream regions of secA using the restriction cleavage sites selected for cloning as described in example 3. The 3' end of orf189 is amplified using its own terminator and the secA promoter lying downstream, so that after secA  deletion<br><br><br>
the prfB can be transcribed directly from the secA promoter. The sections orfl89' and prfB' derived in each case comprise 502 bp or 546 bp.<br>
Figure 6:      Construction  of  the  deletion  plasmid pEorfprfB<br>
The regions amplified by means of PCR were intercloned into E. coli, excised again by means of Xbal and EcoKV and subsequently ligated into the restriction cleavage sites Xbal  and AccI  in the vector pE194.<br>
Figure 7: Plasmid stability in the transformants B. licheniformis (secA) pCB56C (control) and B.   licheniformis   (AsecA) pCB56Csec.A<br>
The fraction of the clones having protease activity is in each case applied, as described in example 4, after an appropriate number of days.<br>
Squares:       B.   licheniformis   (AsecA)   pCB56CsecA Triangles:    B.   licheniformis   (secA)   pCB56C (control)<br><br><br><br>
WE CLAIM:<br>
1.	A process for the selection of a microorganism, characterized in that<br>
(a)	an endogenously present gene coding for an essential translocation activity is inactivated and<br>
(b)	the essential translocation activity inactivated according to (a) is cured by means of a vector.<br>
2.	The process as claimed in claim 1, wherein<br>
(c)	the vector according to (b) carries a transgene.<br>
3.	The process as claimed in claim 1 or 2, wherein the essential translocation activity is one of the following factors: SecA, SecY, SecE, SecD, SecF, signal peptidase, b-SRP (Ffli or Ffs/Scr), FtsY/Srb, PrsA or YajC.<br>
4.	The process as claimed in claim 3, wherein the essential translocation activity is one of the following subunits of the preprotein translocase: SecA, SecY, SecE, SecD or SecF, preferably the subunit SecA.<br>
5.	The process as claimed in any one of claims 1 to 4, wherein the curing according to (b) takes place by means of an activity acting identically to the inactivated endogenously present essential translocation activity, preferably by means of a genetically related activity, particularly preferably by means of the same activity.<br>
S. The process as claimed in any one of claims 1 to 5, wherein the curing according to (b) takes place by means of the regions of the gene secA from Bacillus subtilis, Escherichia coli and Bacillus licheniformis restoring the translocation activity, which are indicated in the sequence listing under SEQ ID NO. 1, SEQ ID NO. 3 and SEQ ID NO. 5 respectively.<br>
-43-<br><br>
The process- as claimed in any one of claims 2 to 6, wherein the inactivation according to (a) is carried out such that a recombination between the gene region inactivated according to (a) and the homologous region on the vector according to (b) is prevented, preferably with complete loss of a gene section contained in the chromosomal gene concerned.<br>
The process as claimed in any one of claims 1 to 7, wherein the inactivation according to (a) takes place by means of a deletion vector, preferably by means of a deletion vector having an externally regulatable replication origin, particularly preferably by means of a deletion vector having a temperature-dependent replication origin.<br>
The process as claimed in any one of claims 1 to 8, wherein the vector according to (b) is a plasmid replicating autonomously in the microorganism.<br>
The process as claimed in claim 9, wherein the plasmid is a plasmid establishing in a plural, preferably in a multiple, copy number.<br>
The process as claimed in any one of claims 1 to 10, wherein the microorganism is a gram-negative strain of bacteria.<br>
The process as claimed in claim 11, wherein a gram-negative strain of bacteria of the genera E. coli or Klebsiella is concerned, in particular derivatives of Escherichia coli K12, of Escherichia coli B or Klebsiella planticola, and very particularly derivatives of the strains Escherichia coli BL21 (DE3), E. coli RV308, E. coli DH5a, E. coli JM109, E. coli XL-1 or Klebsiella planticola<br>
(Rf).<br>
-44-<br><br>
13.	The process as claimed in any one of claims 1 to 10, wherein the microorganism is a gram-positive strain of bacteria.<br>
14.	The process as claimed in claim 13, wherein it is a gram-positive strain of bacteria of the genera Staphylococcus, Corynebacteria or Bacillus, in particular of the species Staphylococcus carnosus, Corynebactehum glutamicum, Bacillus subtilis, B. licheniformis, B. amyloliquefaciens, B. globigii or B. lentus, and very particularly derivatives of the strains B. licheniformis or B. amyloliquefaciens.<br>
15.	The process as claimed in any one of claims 2 to 14, wherein the transgene according to (c) is one which codes for a nonenzyme protein, in particular for a pharmacologically relevant protein, very particularly for insulin or calcitonin.<br>
16.	The process as claimed in any one of claims 2 to 14, wherein the transgene according to (c) is one which codes for an enzyme, preferably for a hydrolytic enzyme or an oxidoreductase, particularly preferably for a protease, amylase, hemicellulase, cellulase, lipase, cutinase, oxidase, peroxidase or laccase.<br><br>
7.	A process for the preparation of a protein by culturing cells of a microorganism strain, wherein a selection process as claimed in any one of claims 1 to 16.<br>
8.	The   process   as   claimed   in   claim   17,   wherein   the   culturing   of the microorganisms takes place in a liquid medium, preferably in a fermenter.<br>
9.	The process as claimed in any one of claims 17 and 18, wherein the protein of interest is secreted into the surrounding medium.<br>
-45-<br><br>
20.	A microorganism, obtained by a selection process as claimed in any one of claims 1 to 16.<br>
21.	The microorganism as claimed in claim 20, wherein the transgene is expressed.<br>
22.	The microorganism as claimed in claim 21, wherein the transgene is secreted.<br>
23.	A vector which carries a gene for an essential translocation activity and a transgene which, if it is present as the only transgene, does not code for an antibiotic resistance.<br>
24.	The vector as claimed in claim 23, wherein the transgene codes for a pharmacologically relevant nonenzyme protein or for a hydrolytic enzyme or for an oxidoreductase.<br>
25.	The vector as claimed in claim 23 or 24, wherein the translocation activity encoded by it is able to cure an inactivated essential translocation activity endogenously present in a microorganism strain, preferably by means of a genetically related activity, particularly preferably by means of the same activity.<br>
26.	The vector as claimed in any one of claims 23 to 25, wherein the essential translocation activity is one of the following factors: SecA, SecY, SecE, SecD, SecF, signal peptidase, b-SRP (Ffh or Ffs/Scr), FtsY/Srb, PrsA or YajC.<br>
27.	The vector as claimed in claim 26, wherein the essential translocation activity is one of the following subunits of the preprotein translocase: SecA, SecY, SecE, SecD or SecF, preferably the subunit SecA.<br>
-46-<br><br>
28.	The vector as claimed in any one of claims 23 to 27, wherein the essential<br>
translocation activity is one of the genes secA from Bacillus subtilis,<br>
Escherichia coli or Bacillus licheniformis, which are indicated in the sequence<br>
listing under SEQ ID NO. 1, SEQ ID NO. 3 and SEQ ID NO. 5 respectively.<br>
29.	The vector as claimed in any one of claims 23 to 28, wherein a plasmid<br>
replicating autonomously in the microorganism is concerned.<br>
30.	The vector as claimed in claim 29, wherein the plasmid is one establishing in a<br>
plural, preferably in a multiple, copy number.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjExOS1jaGVucC0yMDA1IGFic3RyYWN0IGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">2119-chenp-2005 abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjExOS1jaGVucC0yMDA1IGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2119-chenp-2005 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjExOS1jaGVucC0yMDA1IGNsYWltcyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">2119-chenp-2005 claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjExOS1jaGVucC0yMDA1IGNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2119-chenp-2005 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjExOS1jaGVucC0yMDA1IGNvcnJlc3BvbmRlbmNlcyBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">2119-chenp-2005 correspondences others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjExOS1jaGVucC0yMDA1IGNvcnJlc3BvbmRlbmNlcyBwby5wZGY=" target="_blank" style="word-wrap:break-word;">2119-chenp-2005 correspondences po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjExOS1jaGVucC0yMDA1IGRlc2NycHRpb24gKGNvbXBsZXRlKSBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">2119-chenp-2005 descrption (complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjExOS1jaGVucC0yMDA1IGRlc2NycHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">2119-chenp-2005 descrption (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjExOS1jaGVucC0yMDA1IGRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">2119-chenp-2005 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjExOS1jaGVucC0yMDA1IGZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">2119-chenp-2005 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjExOS1jaGVucC0yMDA1IGZvcm0tMTMucGRm" target="_blank" style="word-wrap:break-word;">2119-chenp-2005 form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjExOS1jaGVucC0yMDA1IGZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">2119-chenp-2005 form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjExOS1jaGVucC0yMDA1IGZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">2119-chenp-2005 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjExOS1jaGVucC0yMDA1IGZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">2119-chenp-2005 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjExOS1jaGVucC0yMDA1IGZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">2119-chenp-2005 form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjExOS1jaGVucC0yMDA1IHBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">2119-chenp-2005 pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjExOS1jaGVucC0yMDA1IHBldGl0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">2119-chenp-2005 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="231275-a-tachometer-for-an-aircraft-wheel.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="231277-a-device-for-transporting-lengthy-structural-members.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>231276</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2119/CHENP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>04-Mar-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>02-Sep-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>HENKEL AG &amp; Co. KGaA</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>HENKELSTRASSE 67, D-40589 DUSSELDORF,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>HINTZ, MAREN</td>
											<td>ZIEGELSTRASSE 41, 40468 DUSSELDORF,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>FREUDL, ROLAND</td>
											<td>ISOLASTRASSE 1, 52353 DUREN,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>FEESCHIE, JORG</td>
											<td>GEORG-BUCHNER-STRASSE 23, 40699 ERKRATH,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>BREVES,ROLAND</td>
											<td>SCHWALBENWEG 5, 40822 METTMANN,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N1/21</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2004/001949</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-02-27</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>103 09 557.8</td>
									<td>2003-03-04</td>
								    <td>Germany</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/231276-a-process-for-the-selection-of-a-microorganism-based-on-inactivation-of-an-essential-translocating-enzyme by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:08:26 GMT -->
</html>
