<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/231303-variant-subtilisin-enzymes-subtilases by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:10:13 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 231303:VARIANT SUBTILISIN ENZYMES (SUBTILASES)</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">VARIANT SUBTILISIN ENZYMES (SUBTILASES)</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>This invention relates to a subtilase enzyme being selected from the group consisting of (a) a subtilase having an amino acid sequence that has at more than 99.26% identity with the amino acid sequence shown as amino acids 1 to 269 of SEQ ill NO:2; or (b) a subtilase encoded by the subtilase encoding part of the polynucleotide cloned into a plasmid fragment present in Escherichia coli MT173 DSM 15575, or a variant thereof having more than 99.26% identity to said subtilase.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br><br>
VARIANT SUBTILISIN ENZYMES (SUBTILASES)<br>
TECHNICAL FIELD<br>
The present invention relates to novel subtilases having an improved performance on soil, in<br>
particular  egg   stains.   These  subtilases  are   useful  exhibiting  excellent  or  improved<br>
performance on egg stains when used in e.g. cleaning or detergent compositions, such as<br>
laundry detergent compositions and dish wash compositions, including automatic dish wash<br>
compositions.<br>
The present invention also relates to isolated polynucleotides encoding the subtilases, nucleic acid constructs, recombinant expression vectors, host cells comprising the nucleic acid construct, and methods for producing and using the subtilases of the invention. Further, the present invention relates to cleaning and detergent compositions comprising the subtilase. enzymes of the invention as well as to use of such enzymes in detergent compositions and for removal of egg stains.<br>
BACKGROUND OF THE INVENTION<br>
In the detergent industry enzymes have for more than 30 years been implemented in washing formulations. Enzymes used in such formulations comprise proteases, lipases, amylases, cellulases, mannosidases as well as other enzymes or mixtures thereof. Commercially most important enzymes are proteases.<br>
An increasing number of commercially used proteases is protein engineered variants of naturally occurring wild type proteases, e.g. DURAZYM® (Novozymes A/S), RELASE® (Novozymes A/S), MAXAPEM® (Gist-Brocades N .V.), PURAFECT (Genencor International, Inc.).<br>
Further, a number of protease variants is described in the art, such as in EP 130756 (GENENTECH)(corresponding to US Reissue Patent No. 34,606 (GENENCOR)); EP 214435 (HENKEL); WO 87/04461 (AMGEN); WO 87/05050 (GENEX); EP 260105 (GENENCOR); Thomas, Russell, and Fersht (1985) Nature 318 375-376; Thomas, Russell, and Fersht (1987) J. Mol. Biol. 193 803-813; Russel and Fersht Nature 328 496-500 (1987); WO 88/08028 (Gehex); WO 88/08033 (Amgen); WO 95/27049 (SOLVAY S.A.); WO 95/30011 (PROCTER &amp; GAMBLE COMPANY); WO 95/30010 (PROCTER &amp; GAMBLE COMPANY); WO 95/29979 (PROCTER &amp; GAMBLE COMPANY); US 5.543.302 (SOLVAY S.A.); EP 251 446 (GENENCOR); WO 89/06279 (NOVOZYMES A/S); WO 91/00345 (NOVOZYMES A/S); EP 525 610 A1 (SOLVAY); WO 94/02618 (GIST-BROCADES N.V.).<br>
1<br><br>
Test method used for screening (AMSA) is described in WO 02/42740 (NOVOZYMES A/S).<br>
WO 01/75087 (MAXYGEN, INC. / NOVOZYMES A/S) describes subtilisin homoiogues that are improved for a variety of specific properties including thermal stability, activity at low temperature and alkaline stability.<br>
WO 01/68821 (NOVOZYMES A/S) describes subtilase enzymes, which are suitable for removal of egg stains from for example laundry and/or hard surfaces.<br>
However, even though a number of useful proteases and protease variants have been described, there is still a need for further improvement of proteases or protease variants for a -   number of industrial uses..<br>
In particular, the problem of removing egg stains from e.g. laundry or hard surfaces has been pronounced due to the fact that substances present in the egg white inhibit many serine proteases.<br>
Therefore, an object of the present invention is to provide improved subtilase enzymes, which are suitable for removal of egg stains from for example laundry and/or hard surfaces.<br>
SUMMARY OF THE INVENTION<br>
Thus, in a first aspect the present invention relates to a subtilase enzyme having improved wash performance on egg stains, the subtilase being selected from the group consisting of<br>
a subtilase having an amino acid sequence which has more than 99.26% identity with the amino acid sequence shown as amino acids 1 to 269 of SEQ ID NO;2; and<br>
a subtilase encoded by the subtilase encoding part of the polynucleotide cloned into a plasmid fragment present in Escherichia coll MT173 DSM 15575, or a variant thereof having at least 99.26% identity to said subtilase; and<br>
a subtilase having an amino acid sequence which has more than 97.40% identity with the amino acid sequence shown as amino acids 1 to 269 of SEQ ID NO:4; and<br>
a subtilase encoded by the subtilase encoding part of the polynucleotide cloned into a plasmid fragment present in Escherichia coli MT173 DSM 15574, or a variant thereof having at least 97.40% identity to said subtilase.<br><br>
In a second aspect the present invention relates to an isolated polynucleotide comprising a nucleic acid sequence that encodes for the subtilases according to the invention.<br>
in a third aspect the present invention relates to an isolated polynucleotide encoding a subtilase, selected from the group consisting of<br>
(a)	a polynucleotide having at least 88% identity with the nucleic acid sequence shown ■<br>
as nucleotides 1 to 807 SEQ ID NO:1; and<br>
(b)	the subtilase encoding part of the polynucleotide that has been cloned into a piasmid<br>
present in Escherichia coli MT173 DSM 15575, or a variant thereof having at least 88% identity to said nucleic acid sequence.<br>
(c)	a polynucleotide having at least 88% identity with the nucleic acid sequence shown as<br>
nucleotides 1 to 807 SEQ ID NO:3; and<br>
(d)	the subtilase encoding part of the polynucleotide that has been cloned into a piasmid<br>
present in Escherichia coli MT173 DSM 15574, or a variant thereof having at least 88% identity to said nucleic acid sequence.<br>
in a fourth aspect the present invention relates to a nucleic acid construct comprising the nucleic acid sequence according to the invention operably linked to one or more control sequences capable of directing the expression of the subtilase in a suitable host.<br>
In a fifth aspect the present invention relates to a recombinant expression vector comprising the nucleic acid construct according to the invention, a promoter, and transcriptional and trans-lational stop signals.<br>
In a sixth aspect the present invention relates to a recombinant host cell comprising the nucleic acid construct of the invention.<br>
In a seventh aspect the present invention relates to a method for producing the subtilase according to the invention, the method comprising:<br>
cultivating a recombinant host cell according to the invention under conditions conducive to the production of the subtilase; and<br>
recovering the subtilase.<br>
In an eight aspect the present invention relates to a cleaning or detergent composition,<br><br>
preferably a laundry or dish wash composition, comprising the subtilase according to the invention.<br>
Further aspects of the present invention relate to use of the subtilases according to the invention in a cleaning or detergent composition; use of the subtilases or the compositions according to the invention for removal of egg stains; a method for cleaning or washing, including a method for removal of egg stains from a hard surface or laundry comprising contacting the hard surface or the laundry with the composition of the invention.<br>
Concerning alignment and numbering reference is made to fig. 1 which shows alignments between subtilisin BPN' (a) (BASBPN) and the novel subtilases of the invention (b) and (c).<br>
These alignments are in this patent application used as a reference for numbering the residues,<br>
DEFINITONS<br>
Prior to discussing this invention in further detail, the following terms and conventions will first be defined.<br>
NOMENCLATURE OF AMINO ACIDS<br>
A	=      Ala	=	Alanine<br>
V	Val	Valine<br>
L	=.	Leu	=	Leucine<br>
I	=	lie	=	Isoleucine<br>
P	=	Pro	=	Proline<br>
F	=	Phe	=	Phenylalanine<br>
W	=	Trp	=	Tryptophan<br>
M	=	Met	=	Methionine<br>
G	=	Gly	=	Glycine<br>
S	=	Ser	=	Serine<br>
T	=	Thr	=	Threonine<br>
C	=	Cys	=	Cysteine<br>
Y	=	Tyr	=	Tyrosine<br>
N	=      Asn	=	Asparagine<br>
Q	=      Gin	=	Glutamine<br>
D	=      Asp	=	AsparticAcid<br><br>
E	=	Gfu	=	Glutamic Acid<br>
K	=	Lys	=	Lysine<br>
R	=	Arg	=	Arginine<br>
H	=	His	=	Histidine<br>
X	=	Xaa	=	Any amino acid<br>
NOMENCLATURE OF NUCLEIC ACIDS<br>
A	=      Adenine<br>
G	=      Guanine<br>
C	=      Cytosine<br>
. T	=      Thymine     (only in DNA)<br>
U	=      Uracil	(onlyinRNA)<br>
NOMENCLATURE AND CONVENTIONS FOR DESIGNATION OF VARIANTS<br>
In describing the various subtilase enzyme variants produced or contemplated according to<br>
the invention, the following nomenclatures and conventions have been adapted for ease of<br>
reference:<br>
A frame of reference is first defined by aligning the isolated or parent enzyme with subtilisin BPN' (BASBPN).<br>
"Homology" or "homologous to" is in the context of the present invention to be understood in its conventional meaning and the "homology" between two amino acid sequences should be determined by use of the "Similarity" defined by the GAP program from the University of Wisconsin Genetics Computer Group (GCG) package using default settings for alignment parameters, comparison matrix, gap and gap extension penalties. Default values for GAP penalties, i.e. GAP creation penalty of 3.0 and GAP extension penalty of 0.1 (Program Manual for the Wisconsin Package, Version 8, August 1994, Genetics Computer Group, 575 Science Drive, Madison, Wisconsin, USA 53711). The method is also described in S.B. Needleman and CD. Wunsch, Journal of Molecular Biology, 48, 443-445 (1970). Identities can be extracted from the same calculation. The homology between two amino acid sequences can also be determined by "identity* or "similarity" using the GAP routine of the GCG package version 9.1 with default setting for alignment parameters, comparison matrix, gap and gap extension penalties can also be applied using the following parameters: gap creation penalty = 8 and gap extension penalty = 8 and all other parameters kept at their default values. The output from the routine is besides the amino acid alignment the calculation of the "Percent Identity" and the "Similarity" between the two sequences. The numbers calculated using GCG package<br><br>
version 9.1 is slightly different from the version 8.<br>
. Another method is to use known recognized alignments between subtilases, such as the alignment indicated in WO 91/00345. In most cases the differences will not be of any importance.<br>
Such an alignment between subtilisin BPNT (BASBPN) and the novel subtilases of the invention is indicated in fig. 1.<br>
Thereby, a number of deletions and insertions will be defined in relation to BASBPN. In fig. 1, the novel subtilases according to the invention have 6 deletions in positions 36, 58, 158, 162, 163, and 164 in comparison to BASBPN. These deletions are in fig. 1 indicated by asterixes<br>
The various modifications performed in a parent enzyme are indicated in general using three .   elements as follows:<br>
Original amino acid position substituted amino acid<br>
The notation G195E means a substitution of a glycine in position 195 with a glutamic acid.<br>
Position substituted amino acid<br>
In the case where the original amino acid residue may be any amino acid residue, a short<br>
hand notation may at times be used indicating only the position and substituted amino acid:<br>
170Seror170S.<br>
Such a notation is particular relevant in connection with modification(s) in homologous subtilases {vide infra).<br>
Original amino acid position<br>
Such a notation is in particular relevant when the identity of the substituting amino acid residue(s) is immaterial. The substitution of any amino acid residue acid for glycine in position 195 is designated as: Gly195 or G195.<br>
Position<br>
When both the original amino acid(s) and substituted amino acid(s) may comprise any amino<br>
acid, then only the position is indicated, e.g.: 170.<br><br>
Original amino acid position (substituted amino acidl	substituted amino acidn)<br>
When the original amino acid(s) and/or substituted amino acid(s) may comprise more than one, but not all amino acid(s), then the selected amino acids are indicated inside brackets:{}.<br>
For specific variants the specific three or one letter codes are used, including the codes Xaa and X to indicate any amino acid residue.<br>
SUBSTITUTIONS-:<br>
The substitution of Glutamic acid for glycine in position 195 is designated as:<br>
Gly195Glu	or        G195E<br>
or the substitution of any amino acid residue acid-for glycine in position 195 is designated as:<br>
Gly195Xaa	or       G195X<br>
or<br>
Gly195	or        G195<br>
The substitution of serine for any amino acid residue in position 170 would thus be designated:<br>
Xaa170Ser	or       X170S.<br>
or<br>
170Ser	or        170S<br>
Such a notation is particular relevant in connection with modification(s) in homologous subtilases (vide infra). 170Ser is thus meant to comprise e.g. both a Lys170Ser modification in BASBPN and Arg170Ser modification in the subtilase according to the invention (cf. fig. 1).<br>
For a modification where the original amino acid(s) and/or substituted amino acid(s) may comprise more than one, but not all amino acid(s), the substitution of glycine, alanine, serine or threonine for arginine in position 170 would be indicated by:<br>
Arg170{Gly,Ala,Ser,Thr} or R170{GIAISI1}<br>
to indicate the variants<br>
R170G, R170A, R170S, and R170T.<br><br>
•   DELETIONS: A deletion of glycine In position 195 'wilI be indicated by:<br>
GIy195*	or        G195*<br>
Correspondingly, the deletion of more than one amino acid residue, such as the deletion of glycine and leucine in positions 195 and 196 will be designated:1<br>
Gly195*+Leu196*	or       G195*+L196*<br>
INSERTIONS:<br>
The insertion of an additional amino acid residue such as e.g. a lysine after G195 is indicated<br>
by:<br>
Gly195GlyLys	or       G195GK;<br>
or when more than one amino acid residue is inserted, such as e.g. a Lys, and Ala after G195 this will be indicated as:<br>
Giy195GlyLysAla	or       G195GKA<br>
In such cases, the inserted amino acid residue(s) are numbered by the addition of lower case letters to the position number of the amino acid residue preceding the inserted amino acid residue(s). In the above example, the sequences 194 to 196 would thus be:<br>
194195196<br>
BLSAVI      A - G -   L	.<br>
194 195 195a 195b 196 Variant      A - G - K -   A- -    L<br>
In cases where an amino acid residue identical to the existing amino acid residue is inserted it is clear that degeneracy in the nomenclature arises. If for example a glycine is inserted after the glycine in the above example this would be indicated by G195GG. The same actual change could just as well be indicated as A194AG for the change from<br>
194195196 BLSAVI      A - G - L<br><br>
to<br>
194.195 195a 196 Variant      A - G -  G - L<br>
194 194a 195 196<br>
Such instances will be apparent to the skilled person, and the indication G195GG and corresponding indications for this type of insertions are thus meant to comprise such<br>
equivalent degenerate indications.<br>
FILLING A GAP:<br>
Where a deletion in an enzyme exists in the reference comparison with the subtilisin BPN'<br>
sequence used for the numbering, an insertion in such a position is indicated as:<br>
*36Asp	or        *36D<br>
for the insertion of an aspartic acid in position 36<br>
MULTIPLE MODIFICATIONS:<br>
Variants comprising multiple modifications are separated by pluses, e.g.:<br>
Arg 170Tyr+Gly195Glu    or        R170Y+G195E<br>
representing modifications in positions 170 and 195 substituting tyrosine and glutamic acid for arginine and glycine, respectively.<br>
Thus, Tyr167{Gly,AlalSer,Thr}+Arg170{Gly,AIa,Ser,Thr} designates the following variants:<br><br><br>
This nomenclature is particular relevant relating to modifications aimed at substituting, replacing, inserting or deleting amino acid residues having specific common properties, such as residues of positive charge (K, R, H), negative charge (D, E), or conservative amino acid modificatbn(s) of e.g. Tyr167{GlylAia,Ser,Thr}+Arg170{GlyIAla,SerIThr}f which signifies substituting a small amino acid for another small amino acid. See the section "Detailed description of the invention* for further details.<br>
Proteases<br>
Enzymes cleaving the amide linkages in protein substrates are classified as proteases, or (interchangeably) peptidases (see Walsh, 1979, Enzymatic Reaction Mechanisms. W.H. Freeman and Company, San Francisco, Chapter 3).<br>
Numbering of amino acid positions/residues<br>
If nothing-else is mentioned, the amino acid numbering used herein corresponds to that of the subtilase BPN' (BASBPN) sequence. For further description of the BPN' sequence, see fig. 1 or Siezen et a/., Protein Engng. 4 (1991) 719-737.<br>
Serine proteases<br>
A serine protease is an enzyme which catalyzes the hydrolysis of peptide bonds, and in which there is an essential serine residue at the active site (White, Handier and Smith, 1973 'Principles of Biochemistry," Fifth Edition, McGraw-Hill Book Company, NY, pp. 271-272).<br>
The bacterial serine proteases have molecular weights in the 20,000 to 45,000 Dalton range. They are inhibited by diisopropyl fluorophosphate. They hydrolyze simple terminal esters and are similar in activity to eukaryotic chymotrypsin, also a serine protease. A more narrow term, alkaline protease, covering a sub-group, reflects the high pH optimum of some of the serine proteases, from pH 9.0 to 11.0 (for review, see Priest (1977) Bacteriological Rev. 41 711-753).<br>
Subtilases<br>
Siezen et al have proposed a sub-group of the serine proteases tentatively designated subtilases, Protein Engng, 4 (1991) 719-737 and Siezen et al. Protein Science 6 (1997) 501-523. They are defined by homology analysis of more than 170 amino acid sequences of serine proteases previously referred to as subtilisin-like proteases. A subtilisin was previously often defined as a serine protease produced by Gram-positive bacteria or fungi, and according to Siezen et al. now is a subgroup of the subtilases. A wide variety of subtilases have been identified, and the amino acid sequence of a number of subtilases has been determined. For a more detailed description of such subtilases and their amino acid sequences reference is<br><br>
made to Siezen et a/. (1997).<br>
One subgroup of the subtilases, I-S1 or "true" subtilisins, comprises the "classical" subtilisins, such as subtilisin 168 (BSS168), subtilisin BPN' (BASBPN), subtilisin Carlsberg (BLSCAR)(ALCALASE®, NOVOZYMES A/S), and subtilisin DY (BSSDY).<br>
A further subgroup of the subtilases, I-S2 or high alkaline subtilisins, is recognized by Siezen et at. (supra). Subgroup I-S2 proteases are described as highly alkaline subtilisins and comprises enzymes such as subtilisin PB92 (BAALKP) (MAXACAL®, Gist-Brocades NV), subtilisin 309 (BLSAVI)(SAV!NASE® NOVOZYMES A/S), subtilisin 147 (BLS147) (ESPERASE®, NOVOZYMES A/S), and alkaline elastase YaB (BSEYAB).<br>
Parent subtilase<br>
The term "parent subtilase" describes a subtilase defined according to Siezen et al. (1991 and 1997). For further details, see description of "SUBTILASES" immediately above. A parent subtilase may also be a subtilase isolated from a natural source/wherein subsequent modifications have been.made while retaining the characteristic of a subtilase. Furthermore, a parent subtilase may also be a subtilase which has been prepared by the DNA shuffling technique, such as described by J.E. Ness et a/., Nature Biotechnology, 17, 893-896 (1999).<br>
Alternatively the term "parent subtilase" may be termed "wild type subtilase".<br>
Modifications) of a subtilase<br>
The term Hmodification(s)H used herein is defined to include chemical modification of a subtilase as well as genetic manipulation of the DNA encoding a subtilase. The modification(s) can be replacement(s) of the amino acid side chain(s), substitution(s), deletion(s) and/or insertions in or at the amino acid(s) of interest.<br>
Subtilase variant<br>
In the context of this invention, the term subtilase variant or mutated subtilase means a subtilase that has been produced by an organism which is expressing a mutant gene derived from a parent micro organism which possessed an original or parent gene and which produced a corresponding parent enzyme, the parent gene having been mutated in order to produce the mutant gene from which said mutated subtilase protease is produced when expressed in a suitable host. Analogously, the mutant gene may also be derived from a parent gene produced by DNA shuffling technique.<br><br>
Homologous subtilase sequences<br>
In the present context, the homology between two amino acid sequences is described by the<br>
parameter "identity".<br>
In order to determine the degree of identity between two subtilases, the GAP routine of the GCG package version 9.1 can be applied (infra) using the same settings. The output from the routine is besides the amino acid alignment the calculation of the "Percent Identity" between the two sequences.<br>
Based on this description it is routine for a person skilled in the art to identify suitable homologous subtilases and corresponding homologous active site loop regions, which can be modified according to the invention.<br>
Isolated polynucleotide<br>
The term "isolated polynucleotide" as used herein refers to a polynucleotide, which has been isolated and purified and is thus in a form suitable for use within genetically engineered protein production systems. Such isolated molecules may be those that are separated from their natural environment and include cDNA and genomic clones as well as polynucleotides derived from DNA shuffling experiments or from site-directed autogenesis experiments. Isolated polynucleotides of the present invention are free of other genes with which they are ordinarily associated, but may include 51 and 3' untranslated regions such as promoters and terminators. The identification of associated regions will be evident to one of ordinary skill in the art (see for example Dynan and Tijan, Nature 316:774*78, 1985). The term "isolated nucleic acid sequence" may alternatively be termed "isolated DNA sequence", "cloned nucleic acid sequence" or "cloned DNA sequence".<br>
Isolated protein<br>
When applied to a protein, the term "isolated1' indicates that the.protein has been removed from its native environment.<br>
In a preferred form, the isolated protein is substantially free of other proteins, particularly other homologous proteins (i.e. "homologous impurities" (see below)).<br>
An isolated protein is more than 10% pure, preferably more than 20% pure, more preferably more than 30% pure, as determined by SDS-PAGE. Further, it is preferred to provide the protein in a highly purified form, i.e. more than 40% pure, more than 60% pure, more than 80% pure, more preferably more than 95% pure, and most preferably more than 99% pure, as<br><br>
determined by SDS-PAGE.<br>
The term 'isolated protein" may alternatively be termed "purified protein".<br>
Homologous impurities<br>
The term "homologous impurities" means any impurity (e.g. another polypeptide than the sub-tilase of the invention), which originate from the homologous cell where the subtilase of the invention is originally obtained from.<br>
Obtained from<br>
The term "obtained from" as used herein in connection with a specific microbial source means that the polynucleotide and/or subtilase produced by the specific source, or by a cell in which a gene from the source has been inserted.<br>
Substrate<br>
The term "substrate" used in connection with a substrate for a protease should be interpreted in. its broadest form as comprising a compound containing at least one peptide bond susceptible to hydrolysis by a subtilisin protease.<br>
Product<br>
The term "product" used in connection with a product derived from a protease enzymatic reaction should in the context of the present invention be interpreted to include the products of a hydrolysis reaction involving a subtilase protease. A product may be the substrate in a subsequent hydrolysis reaction.<br>
Wash Performance<br>
In the present context, the term "wash performance" is used as an enzyme's ability to remove soil, in particular egg stains present on the object to the cleaned during e.g. wash or hard surface cleaning. See also the "Model Detergent Wash Performance Test" in Example 2.<br>
% Removed Protein Film<br>
In the present context, the term "% Removed Protein Film" is used as an enzyme's ability to remove soil, in particular egg stains present on the object to the cleaned during automatic dish wash. Performance data originate from gravimetric measurements of the steel plates as clean, soiled and washed. The performance is calculated as:<br><br><br><br><br><br><br><br>
1)BLAP {Bacillus lentus Alkaline Protease) has been described in US 5,352,604<br>
It is well-known in the art that a so-called conservative substitution of one amino acid residue to a similar amino acid residue is expected to produce only a minor change in the characteristic of the enzyme.<br>
Table I below lists groups of conservative amino acid substitutions.<br>
Table I<br>
Conservative amino acid substitutions<br>
Common Property	Amino Acid<br>
Basic (positive charge)	R = arginine<br>
K = lysine<br>
H = histidine<br>
Acidic (negative charge)	E = glutamic acid<br>
D = aspartic acid<br>
Polar	Q = glutamine   .<br>
N = asparagine<br>
Hydrophobic	L = leucine<br>
I = isoleucine<br>
V	= valine<br>
M = methionines<br>
Aromatic	F = phenylalanine<br>
W = tryptophane<br>
Y	= tyrosine<br>
Small                                                        G = glycine<br>
A = alanine S = serine T = threonine<br>
Therefore, in a further interesting embodiment of the invention, the subtilase having the amino acid sequence of SEQ ID NO:2 and SEQ ID NO:4 is combined with a substitution, deletion and/or insertion of one or more amino acid residues.<br>
Especially, combinations with other modifications known in the art to provide improved properties to the enzyme are envisaged. The art describes a number of subtilase variants with differ-<br><br>
ent improved properties and a number of those is mentioned in the "Background of the invention" section herein (vide supra).<br>
Such combinations comprise the positions: 222 (improves oxidation stability), 218 (improves thermal stability), substitutions in the Ca2+-binding sites stabilizing the enzyme, e.g. position 76, and many other apparent from the prior art.<br>
In further embodiments, a subtilase variant described herein may advantageously be combined with one or more modification(s) in any of the positions:<br>
27, 36, 56, 76, 87, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 120, 123, 159, 167, 170, 206, 218, 222, 224, 232, 235, 236, 245, 248, 252 and 274 (BPN' numbering).<br>
Specifically, the following BLSAVI, BLSUBL, BSKSMK, and BAALKP modifications are considered appropriate for combination:<br><br>
A particular interesting variant is a variant, which in addition to modifications according to the invention contains the following substitutions: S101G+S103A+V104I+G159D+A232V+Q236H+Q245R+N248D+N252K.<br>
Moreover, subtilase variants of the main aspect(s) of the invention are preferably combined with one or more modification(s) in any of the positions 129,131 and 194, preferably as 129K, 131H and 194P modifications, and most preferably as P129K, P131H and A194P modifications. Any of those modification(s) are expected to provide a higher expression level of the subtilase variant in the production thereof.<br>
Furthermore, it is contemplated that insertion of at least one additional amino acid residue in<br><br>
the active site (b) loop region, corresponding to insertion of at least one additional amino acid residue from position 95 to position 103 (BASBPN numbering), will confer additional wash performance to the subtilase of the invention. In particular, it is preferred to insert at least one additional amino acid residue, such as one additional amino acid residue, in the following posi-tions: between positions 98 and 99, and between positions 99 and 100.<br>
Moreover, isolated subtilases, preferably in a purified form, having immunochemical identity or partial immunochemical identity to the subtilase having the amino acid sequence of SEQ ID NO:2 or SEQ ID NO:4 are also considered as being within the scope of the present invention. The immunochemical properties are determined by immunological cross-reaction identity tests by the well-known Ouchterlony double immunodiffusion procedure. Specifically, an antiserum containing polyclonal antibodies which are immunoreactive or bind to epitopes of the subtilase having the amino acid sequence of SEQ ID NO:2 or SEQ ID NO:4 are prepared by immunizing rabbits (or other rodents) according to the procedure described by Harboe and Ingild, In N.H. Axelsen, J. Krall, and B. Weeks, editors, A Manual of Quantitative Immunoelectrophoresis, Blackwell Scientific Publications, 1973, Chapter 23, or Johnstone and Thorpe, Immuno-chemistry in Practice, Blackwell Scientific Publications, 1982 (more specifically pages 27-31). A subtilase having immunochemical identity is a subtilase, which reacts with the antiserum in an identical fashion such as total fusion of precipitates, identical precipitate morphology, and/or identical electrophoretic mobility using a specific immunochemical technique. Axelsen, Bock, and krall describe a further explanation of immunochemical identity in N.H, Axelsen, J. Krall, and B. Weeks, editors, A Manual of Quantitative Immunoelectrophoresis, Blackwell Scientific Publications, 1973, Chapter 10. A subtilase having partial immunochemical identity is a subtilase, which reacts with the antiserum in a partially identical fashion such as partial fusion of precipitates, partially identical precipitate morphology, and/or partially identical electrophoretic mobility using a specific immunochemical technique. Bock and Axelsen describe partial immunochemical identity in N.H. Axelsen, J. Krall, and B. Weeks, editors, A Manual of Quantitative Immunoelectrophoresis, Blackwell Scientific Publications, 1973, Chapter 11.<br>
The a ntibody m ay a Iso be a monoclonal antibody. Monoclonal antibodies may be prepared and used, e.g. according to the methods of E. Harlow and D. Lane, editors, 1988, Antibodies, A Laboratory Manual, Cold Spring Harbor Press, Cold Spring Harbor, New York.<br>
The present inventors have isolated the gene encoding the subtilase having the amino acid sequence shown in SEQ ID NO:2 and inserted it into E. coli MT173. The E co// MT173 strain harbouring the gene was deposited according to the Budapest Treaty on the International Recognition of the Deposits of Microorganisms for the Purpose of Patent Procedures on 8<br><br>
February 2000 at the Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 1 B, D-38124 Braunschweig, Germany, and designated the accession No. DSM 15575.<br>
The present inventors have isolated the gene encoding the subtilase having the amino acid sequence shown in SEQ ID NO:4 and inserted it into £ coli MT173. The £ coli MT173 strain harbouring the gene was deposited according to the Budapest Treaty on the International Recognition of the D eposits of M icroorganisms for the P urpose of Patent P rocedures on 8 February 2000 at the Deutsche Sammlung von- Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 1 B, D-38124 Braunschweig, Germany, and designated the accession No. DSM 15574.<br>
In an interesting embodiment of the invention, the subtilase has more than 99.26% or more than 99.63% identity with the subtilase encoded by the subtilase encoding part of the polynucleotide cloned into a plasmid fragment present in £ coli MT173 deposited under the accession No. DSM 15575.<br>
In another interesting embodiment of the invention, the subtilase has more than 97.40%, or more than 97.77%, or more than 98.14%, or more than 98.51%, or more than 98.89%, or more than 99.26%, or more than 99.63% identity with the subtilase encoded by the subtilase encoding part of the polynucleotide cloned into a plasmid fragment present in £. coli MT173 deposited under the accession No. DSM 15574.<br>
As mentioned above, the subtilase of the invention exhibits excellent wash performance on . egg stains. Therefore, in order to enable the skilled person - at an early stage of his development work - to select effective and preferred subtilases for this purpose, the present inventors have provided a number of suitable preliminary tests, which can easily be carried out by the skilled person in order to initially assess the performance of the subtilase in question.<br>
Thus, the "Model Detergent Wash Performance Test" disclosed in Example 2, herein, may be employed to assess the efficiency of the selected subtilase. In other words, the "Model Detergent Wash Performance Test" may be employed to assess the ability of a subtilase, when incorporated in a standard detergent composition, to remove egg stains from a textile surface as compared to a reference system (incorporated in the same model detergent system and tested under identical conditions). Using this test, the suitability of a selected subtilase to remove egg stains can be initially investigated, the rationale being that if a selected subtilase does not show a significant improvement in the test compared to the<br><br>
reference enzyme, it is normally not necessary to carry out further test experiments.<br>
Therefore, subtilases which are particular interesting for laundry wash purposes, are such<br>
subtilases which when tested in a model detergent composition comprising<br>
6.2%       LAS (Nansa 80S)<br>
2%'        Sodium salt of C16-C18 fatty acid    ,<br>
4%	Non-ionic surfactant (Plurafax LF404)<br>
22%        Zeolite P<br>
10.5%     Na2C03<br>
4%	Na2Si205<br>
2%	Carboxymethylcellulose (CMC)<br>
6.8%      Acrylate liquid CP5 40%<br>
20%        Sodium perborate (empirical formula NaB02.H202) 0.2%       EDTA 21%        Na2S04<br>
Water (balance)<br>
as described in the "Model Detergent Wash Performance Test" herein, shows an improved wash performance on egg stains as compared to a reference enzyme tested under identical<br>
o<br>
conditions.<br>
The improvement in the wash performance may be quantified by employing the so-called "Performance factor" defined in Example 2, herein.<br>
In a very interesting embodiment of the invention, the subtilase of the invention, when tested in the "Wash Performance Test", has a Performance Factor of at least 1, such as at least 1.5, e.g. at least 2, preferably at least 2.5, such as at least 3, e.g. at least 3.5, in particular at least 4, such as at least 4.5, e.g. at least 5.<br>
Evidently, it is preferred that the subtilase of the invention fulfils the above criteria on at least the stated lowest level, more preferably at the stated intermediate level and most preferably on the stated highest level.<br>
"The full scale Automatic Dish Washing (ADW) test" disclosed in Example 3 or the "Mini scale Automatic Dish Washing (ADW mini wash)" disclosed in Example 4 or the "Automated Mechanical Stress Assay (AMSA)" disclosed in Example 5, all herein, may be employed to assess the efficiency of the selected subtilase in Automatic Dish Washing. In other words, the<br><br>
tests may be employed to assess the ability of a subtilase, when incorporated in a detergent composition (commercial or standard), to remove egg stains from a hard surface as compared to a reference system (incorporated in the same detergent system and tested under identical conditions). Using this test, the suitability of a selected subtilase to remove egg stains can be initially investigated, the rationale being that if a selected subtilase does not show a significant improvement in the test compared to the reference enzyme, it is normally not necessary to carry out further test experiments.<br>
Therefore, subtilases which are particular interesting for automatic dish washing purposes are such subtilases which, when tested in a model detergent composition comprising:<br>
Sodium Tripolyphosphate	23.0%<br>
Sodium Citrate Dihydrate	22.3%<br>
Sodium Perborate Monohydrate	6.0%<br>
Tetraacetyl Ethylendiamine	2.0%<br>
Sodium Disiiicate (noncrystaline)	5.0%<br>
Linear Fatty Alcohol Ethoxylate	2.0%<br>
(non-ionic surfactant, low foaming)<br>
Maleic acid/Acrylic acid copolymer	4.0%<br>
(Sodium salt, 50% active on Sodium Carbonate)<br>
Sodium Carbonate, anhydrous	add to 100%<br>
as described in the tests in Examples 3, 4 or 5 herein, shows an improved performance on egg stains as compared to a reference enzyme tested under identical conditions.<br>
The subtilase of the invention may be constructed by standard techniques for artificial creation of diversity, such as by DNA shuffling of different subtilase genes (see WO 95/22625 and J.E. Ness etal.t Nature Biotechnology, 17, 893-896 (1999)).<br>
Obviously, the subtilase of the invention may also be isolated from a natural source, i.e. the subtilase of the invention may, for example, be a bacterial subtilase, e.g. a gram positive bacterial subtilase such as a Bacjjlus polypeptide, e.g., a Bacillus alkalophilus, Bacillus amyloliquefaciens, Bacillus brevis, Bacillus circulans, Bacillus coagulans, Bacillus lautus, Bacillus lentus, Bacillus licheniformis, Bacillus megaterium, Bacillus stearothermophilus, Bacillus subtilis, or Bacillus thuringiensis subtilase; or a Streptomyces subtilase, e.g., a Streptomyces lividans or Streptomyces murinus subtilase; or a gram negative bacterial subtilase, e.g., an E. coll or a Pseudomonas sp. subtilase.<br><br>
The subtilase of the present invention may also be a fungal polypeptide, and more preferably a yeast subtilase such as a Candida, Kluyveromyces, Pichia, Saccharomyces, Schizosac-charomyces, or Yarrowia subtilase; or more preferably a filamentous fungal subtilase such as an Acremonium, Aspergillus, Aureobasidium, Cryptococcus, Filibasidium, Fusarium, Humicola, Magnaporthe, Mucor, Myceliophthora, Neocallimastix, Neurospora, Paecilomyces, Penicillium, Piromyces, Schizophyllum, Talaromyces, Thermoascus, Thielavia, Tolypocladium, or Tricho-derma subtilase.<br>
In an interesting embodiment, the subtilase is a Saccharomyces carlsbergensis, Saccharomyces cerevisiae, Saccharomyces diastaticus, Saccharomyces dougiasii, Saccharomyces kluy-veri, Saccharomyces norbensis or Saccharomyces oviformis subtilase.<br>
In another interesting embodiment, the subtilase is an Aspergillus aculeatus, Aspergillus awamori, A spergi/lus foetidus, A spergillus j aponicus, A spergi/lus n idulans, A spergillus niger, Aspergillus oryzae, Fusarium bactridioides, Fusarium cerealis, Fusarium crookwellense, Fusa-rium culmorum, Fusarium graminearum, Fusarium graminum, Fusarium heterosporum, Fusarium negundi, Fusarium oxysporum, Fusarium reticulatum, Fusarium roseum, Fusarium sam-bucinum, Fusarium sarcochroum, Fusarium sporotrichioides, Fusarium sulphureum, Fusarium torulosum, Fusarium trichothecioides, Fusarium venenatum, Humicola insolens, Humicola lanuginosa, Mucor miehei, Myceliophthora thermophila, Neurospora crassa, Penicillium pur-purogenumt Trichoderma harzianum, Trichoderma koningi'u Trichoderma longibrachiatumt Trichoderma reesei, or Trichoderma viride subtilase.<br>
It will be understood that for the aforementioned species, the invention encompasses both the perfect and imperfect states, and other taxonomic equivalents, e.g., anamorphs, regardless of the species name by which they are known. Those skilled in the art will readily recognize the identity of appropriate equivalents.<br>
Strains of these species are readily accessible to the public in a number of culture collections, such as the American Type Culture Collection (ATCC), Deutsche Sammlung von Mikroorgan-ismen und Zelikuituren GmbH (DSM), Centraaibureau Voor Schimmelcuitures (CBS), and Agricultural Research Service Patent Culture Collection, Northern Regional Research Center (NRRL).<br>
Furthermore, such subtilases may be identified-and obtained from other sources including microorganisms isolated from nature (e.o.. soil, composts, water, etc.i usino the above-<br><br>
mentioned probes. Techniques for isolating microorganisms from natural habitats are well known in the art. Similarly screening a genomic or cDNA library of another microorganism may then derive the polynucleotide. Once a polynucleotide encoding a subtilase has been detected with the probe(s), the sequence may be isolated or cloned by utilizing techniques which are known to those of ordinary skill in the art (see, e.g., Sambrook et a/., 1989, supra).<br>
Many methods for cloning a subtilase of the invention and for introducing insertions into genes (e.g. subtilase genes) are well-known in the art, cf. the references cited in the "BACKGROUND OF THE INVENTION" section.<br>
In general standard procedures for cloning of genes and introducing insertions (random and/or site directed) into said genes may be used in order to obtain a subtilase enzyme of the invention. For further description of suitable techniques reference is made to Examples herein (vide infra) and (Sambrook et a!. (1989) Molecular cloning: A laboratory manual, Cold Spring Harbor lab., Cold Spring Harbor, NY; Ausubel, F. M. et al. (eds.) "Current protocols in Molecular Biology". John Wiley and Sons, 1995; Harwood, C. R., and Cutting, S. M. (eds.) "Molecular Biological Methods for Bacillus". John Wiley and Sons, 1990); and WO 96/34946.<br>
Further, a subtilase enzyme of the invention may be constructed by standard techniques for artificial creation of diversity, such as by DNA shuffling of different subtilase genes (WO 95/22625; Stemmer WPC, Nature 370:389-91 (1994)). DNA shuffling of e.g. the gene encoding Savinase® with one or more partial subtilase sequences identified in nature will, after subsequent screening for improved wash performance, provide subtilases according to the invention.<br>
POLYNUCLEOTIDES<br>
The present invention also relates to an isolated polynucleotide, which encodes a subtilase of<br>
the present invention.<br>
In one interesting embodiment, the polynucleotide has at least 88%, or at least 89%, at least 90%, at least 91%, at least 92%, at least 93%, at least 94%, at least 95%, at least 96%, at least 97%, at least 98%, or at least 99% identity with the polynucleotide shown as nucleotides 1 to 807 of SEQ ID NO;1; or at least 88%, or at least 89%, at least 90%, at least 91%, at least 92%, at least 93%, at least 94%, at least 95%, at least 96%, at least 97%, at least 98%, or at least 99% identity with the polynucleotide shown as nucleotides 1 to 807 of SEQ ID NO:3. In another interesting embodiment of the invention, the polynucleotide comprises the polynucleotide shown as nucleotides 1 to 807 of SEQ ID NO:1 or of SEQ ID NO;3, an allelic variant<br><br>
thereof, or a fragment thereof capable of encoding subtilases according to the invention. Ob-  viously, the polynucleotide may consist of the polynucleotide shown as nucleotides 1 to 807 of SEQ ID NO:1 or SEQ ID NO:3.<br>
The p resent i nvention a Iso e ncompasses p olynucleotides t hat e ncode a p olypeptide having the amino acid sequence of SEQ ID NO:2, which differ from SEQ ID NO:2 by virtue of the degeneracy of the genetic code; the present invention further encompasses polynucleotides that encode a polypeptide having the amino acid sequence of SEQ ID NO:4, which differ from SEQ ID NO:4 by virtue of the degeneracy of the genetic code. The present invention also relates to subsequences of SEQ ID NO:1 that encode fragments of SEQ ID NO:2 that have proteolytic activity, and to subsequences of SEQ ID NO:3 that encode fragments of SEQ ID NO:4 that have proteolytic activity.<br>
A subsequence of SEQ ID NO:1 is a polynucleotide encompassed by nucleotides 1 to 807<br>
SEQ ID NO:1 except that one or more nucleotides from the 5' and/or 3' end have been de<br>
leted; a subsequence of SEQ ID NO:3 is a polynucleotide encompassed by nucleotides 1 to<br>
807 SEQ ID NO:3 except that one or more nucleotides from the 5' and/or 3' end have been<br>
deleted.	0<br>
The techniques used to isolate or clone a polynucleotide encoding a polypeptide are known in the art and include isolation from genomic DNA, preparation from c DNA, or a combination thereof. The cloning of the polynucleotides of the present invention from such genomic DNA can be effected, e.g., by using the well-known polymerase chain reaction (PCR) or antibody screening of expression libraries to detect cloned DNA fragments with shared structural features. See, e.g., Innis et a/.t 1990, PCR: A Guide to Methods and Application, Academic Press, New York. Other nucleic acid amplification procedures such as ligase chain reaction (LCR), ligated activated transcription (LAT) and nucleic acid sequence-based amplification (NASBA) may be used.<br>
An. isolated polynucleotide can for example be obtained by standard cloning procedures used in genetic engineering to relocate the polynucleotide from its natural location to a different site where it will be reproduced. The cloning procedures may involve excision and isolation of a desired nucleic acid fragment comprising the polynucleotide encoding the subtilase, insertion of the fragment into a vector molecule, and incorporation of the recombinant vector into a host cell where multiple copies or clones of the polynucleotide will be replicated. The polynucleotide may be of genomic, cDNA, RNA, semi-synthetic, synthetic origin, or any combinations thereof.<br><br>
For purposes of the present invention, the degree of identity between two polynucleotides is determined is described above.<br>
Modification of a polynucleotide encoding a subtilase of the present invention may be necessary for the synthesis of subtilases substantially similar to the subtilase. The term "substantially similar" to the subtilase refers to non-naturally occurring forms of the subtilase. These subtilases may differ in some engineered way from the subtilase isolated from its native source, e.g., variants that differ in specific activity, thermostability, pH optimum, or the like. The variant sequence may be constructed on the basis of the polynucleotide presented as the polypeptide encoding part of SEQ ID NO:1, or on the basis of the polynucleotide presented as the polypeptide encoding part of SEQ ID NO:3, e.g., a subsequence thereof, and/or by introduction of nucleotide substitutions which do not give rise to another amino acid sequence of the subtilase encoded by the nucleic acid sequence, but which correspond to the codon usage of the host organism intended for production of the enzyme, or by introduction of nucleotide substitutions which may give rise to a different amino acid sequence. For a general description of nucleotide substitution see, e.g., Ford et a/., 1991, Protein Expression and Purification 2: 95-107.<br>
It will be apparent to those skilled in the art that such substitutions can be made outside the regions critical to the function of the molecule and still result in an active subtilase. Amino acid residues essential to the activity of the polypeptide encoded by the isolated polynucleotide of the invention, and therefore preferably not subject to substitution, may be identified according to procedures known in the art, such as site-directed mutagenesis or alanine-scanning mutagenesis (see, e.g., Cunningham and Wells, 1989, Science 244: 1081-1085). In the latter technique, mutations are introduced at every positively charged residue in the molecule, and the resultant mutant molecules are tested for proteolytic activity to identify amino acid residues that are critical to the activity of the molecule. Sites of substrate-enzyme interaction can also be determined by analysis of the three-dimensional structure as determined by such techniques as nuclear magnetic resonance analysis, crystallography or photoaffinity labelling (see, e.g., de Vos et a/., 1992, Science 255: 306-312; Smith et a/., 1992, Journal of Molecular Biology 224: 899-904; Wiodaver et al., 1992, FEBS Letters 309: 59-64).<br>
NUCLEIC ACID CONSTRUCTS<br>
The present invention also relates to nucleic acid constructs comprising a polynucleotide of the present invention operably linked to one or more control sequences capable of directing the expression of the polypeptide in a suitable host cell.<br><br>
An isolated polynucleotide encoding a subtilase of the present invention may be manipulated in a variety of ways to provide for expression of the subtilase. Manipulation of the polynucleotide prior to its insertion into a vector may be desirable or necessary depending on the expression vector. The techniques for modifying polynucleotides utilizing recombinant DNA methods are well-known in the art.<br>
The control sequences include ail components that are necessary or advantageous for the expression of a subtilase of the present invention. Each control sequence may be native or foreign to the polynucleotide encoding the subtilase. Such control sequences include, but are not limited to, a leader, polyadenylation sequence, propeptide sequence, prompter, signal peptide sequence, and transcription terminator. At a minimum, the control sequences include a promoter, and transcriptional and translational stop signals. The control sequences may be provided with linkers for the purpose of introducing specific restriction sites facilitating ligation of the control sequences with the coding region of the polynucleotide encoding a subtilase. <br>
The control sequence may be an appropriate promoter sequence, a polynucleotide that is recognized by a host cell for expression of the nucleic acid sequence. The promoter sequence contains transcriptional control sequences that mediate the expression of the subtilase. The promoter may be any polynucleotide that shows transcriptional activity in the host cell of choice including mutant, truncated, and hybrid promoters, and may be obtained from genes encoding extracellular or intracellular subtilases either homologous or heterologous to the host cell.<br>
Examples of suitable promoters for directing the transcription of the nucleic acid constructs of the present invention, especially in.a bacterial host cell, are the promoters obtained from the £. coli lac operon, Streptomyces coelicolor agarase gene (dagA), Bacillus subtilis levansu-crase gene (sacB), Bacillus licheniformis alpha-amylase gene {amyL)t Bacillus stearothermo-philus maltogenic amylase gene {amyM), Bacillus amyloliquefaciens alpha-amylase gene {amyQ\ Bacillus licheniformis penicillinase gene (penP), Bacillus subtilis xylA and xylB genes, and prokaryotic beta-lactamase gene (Villa-Kamaroff et a/M 1978, Proceedings of the National Academy of Sciences USA 75: 3727-3731), as well as the tac promoter (DeBoer et a/M 1983, Proceedings of the National Academy of Sciences USA 80: 21-25). Further promoters are described in "Useful proteins from recombinant bacteria" in Scientific American, 1980, 242: 74-94; and in Sambrook ef a/., 1989, supra.<br>
Examples of suitable promoters for directing the transcription of the nucleic acid constructs of the present invention in a filamentous fungal host cell are promoters obtained from the genes<br><br>
for Aspergillus oryzae TAKA amylase, Rhizomucor miehei aspartic proteinase, Aspergillus ni~ ger neutral alpha-amylase, Aspergillus niger acid stable alpha-amylase, Aspergillus niger or Aspergillus awamorl glucoamylase (glaA), Rhizomucor miehei lipase, Aspergillus oryzae alkaline protease, Aspergillus oryzae triose phosphate isomerase, Aspergillus niduians acetami-dase, and Fusarium oxysporum trypsin-like protease (WO 96/00787), as well as the NA2-tpi promoter (a hybrid of the promoters from the genes for Aspergillus niger neutral alpha-amylase and Aspergillus oryzae triose phosphate isomerase), and mutant, truncated, and hybrid promoters thereof.<br>
In a yeast host, useful promoters are obtained from the genes for Saccharomyces cerevisiae enolase (ENO-1), Saccharomyces cerevisiae galactokinase (GAL1), Saccharomyces cerevisiae alcohol dehydrogenase/glyceraldehyde-3-phosphate dehydrogenase (ADH2/GAP), and Saccharomyces cerevisiae 3-phosphoglycerate kinase. Other useful promoters for yeast host cells are described by Romanes etal.t 1992, Yeast 8:423-488.<br>
The control sequence may also be a suitable transcription terminator sequence, a sequence recognized by a host cell to terminate transcription. The terminator sequence is operably linked to the 3' terminus of the polynucleotide encoding the subtilase. Any terminator that is functional in the host cell of choice may be used in the present invention.<br>
Preferred terminators for filamentous fungal host cells are obtained from the genes for Aspergillus oryzae TAKA amylase, Aspergillus niger glucoamylase, Aspergillus niduians anthranilate synthase, Aspergillus niger alpha-glucosidase, and Fusarium oxysporum trypsin-like protease.<br>
Preferred terminators for yeast host cells are obtained from the genes for Saccharomyces cerevisiae enolase, Saccharomyces cerevisiae cytochrome C (CYC1), and Saccharomyces cerevisiae glyceraldehyde-3-phosphate dehydrogenase. Romanos et ah, 1992, supra, describe other useful terminators for yeast host cells.<br>
The control sequence may also be a suitable leader sequence, a non-translated region of an mRNA that is important for translation by the host cell. The leader sequence is operably linked to the 5' terminus of the polynucleotide encoding the polypeptide. Any leader sequence that is functional in the host cell of choice may be used in the present invention.<br>
Preferred leaders for filamentous fungal host cells are obtained from the genes for Aspergillus oryzae TAKA amylase and Aspergillus niduians triose phosphate isomerase.<br><br>
Suitable leaders for yeast host cells are obtained from the genes for Saccharomyces cere-visiae enolase (ENO-1), Saccharomyces cerevisiae 3-phosphogIycerate kinase, Saccharomyces cerevisiae alpha-factor, and Saccharomyces cerevisiae alcohol dehydro-genase/glyceraldehyde-3-phosphate dehydrogenase (ADH2/GAP).<br>
The control sequence may also be a polyadenylation sequence, a sequence operably linked to the 3* terminus of the polynucleotide and which, when transcribed, is recognized by the host cell as a signal to add polyadenosine residues to transcribed mRNA. Any polyadenylation sequence that is functional in the host cell of choice may be used in the present invention.<br>
Preferred polyadenylation sequences for filamentous fungal host cells are obtained from the genes for Aspergillus oryzae TAKA amylase, Aspergillus /?/ger glucoamylase, Aspergillus nidu-lans anthranilate synthase, Fusarium oxysporum trypsin-like protease, and Aspergillus niger alpha-glucosidase.<br>
Useful polyadenylation sequences for yeast host cells are described by Guo and Sherman, 1995, Molecular Cellular Biology 15: 5983-5990,<br>
The control sequence may also be a signal peptide coding region that codes for an amino acid sequence linked to the amino terminus of a subtilase and directs the encoded subtilase into the cell's secretory pathway. The 5* end of the coding sequence of the polynucleotide may inherently contain a signal peptide coding region naturally linked in translation reading frame with the segment of the coding region that encodes the secreted subtilase. Alternatively, the 5' end of the coding sequence may contain a signal peptide coding region that is foreign to the coding sequence. The foreign signal peptide coding region may be required where the coding sequence does not naturally contain a signal peptide coding region. Alternatively, the foreign signal peptide coding region may simply replace the natural signal peptide coding region in order to enhance secretion of the subtilase. However, any signal peptide coding region that directs the expressed subtilase into the secretory pathway of a host cell of choice may be used in the present invention.<br>
Effective signal peptide coding regions for bacterial host cells are the signal peptide coding regions obtained from the genes for Bacillus NCIB 11837 maltogenic amylase, Bacillus stearothermophilus alpha-amylase, Bacillus licheniformis subtilisin, Bacillus licheniformis beta-lactamase, Bacillus stearothermophilus neutral proteases {nprT, nprS, nprM), and Bacillus subtilis prsA. Further signal peptides are described by Simonen and Palva, 1993, Microbiological Reviews 57:109-137.<br><br>
Effective signal peptide ceding regions for filamentous fungal host cells are the signal peptide coding regions obtained from the genes for Aspergillus oryzae TAKA amylase, Aspergillus /?/-ger neutral amylase, Aspergillus niger glucoamylase, Rhizomucor miehei aspartic proteinase, Humicola insolens cellulase, and Humicola lanuginosa lipase.<br>
Useful signal peptides for yeast host cells are obtained from the genes for Saccharomyces cerevisiae alpha-factor and Saccharomyces cerevisiae invertase. Romanos et a/., 1992, supra, describe other useful signal peptide coding regions.<br>
The control sequence may also be a propeptide coding region that codes for an amino acid sequence positioned at the amino terminus of a subtilase. The resultant polypeptide is known as a proenzyme or propolypeptide (or a zymogen in some cases). A prppolypeptide is generally inactive and can be converted to a mature active polypeptide by catalytic or autocatalytic cleavage of the propeptide from the propolypeptide. The propeptide coding region may be obtained from the genes for Bacillus subtilis alkaline protease (aprf), Bacillus subtilis neutral protease (nprT), Saccharomyces cerevisiae alpha-factor, Rhizomucor miehei aspartic proteinase, and Myceliophthora thermophila laccase (WO 95/33836).<br>
Where both signal peptide and propeptide regions are present at the amino terminus of a subtilase, the propeptide region is positioned next to the amino terminus of a subtilase and the signal peptide region is positioned next to the amino terminus of the propeptide region.<br>
It may also be desirable to add regulatory "sequences that allow the regulation of the expression of the polypeptide relative to the growth of the host cell. Examples of regulatory systems are those which cause the expression of the gene to be turned on or off in response to a chemical or physical stimulus, including the presence of a regulatory compound. Regulatory systems in prokaryotic systems include the lac, tec, and trp operator systems. In yeast, the ADH2 system or GAL1 system may be used. In filamentous fungi, the TAKA alpha-amylase promoter, Aspergillus niger glucoamylase promoter, and Aspergillus oryzae glucoamylase promoter may be used as regulatory sequences. Other examples of regulatory sequences are those that allow for gene amplification. In eukaryotic systems, these include the dihydrofolate reductase gene that is amplified in the presence of methotrexate, and the metallothionein genes that are amplified with heavy metals. In these cases, the polynucleotide encoding the polypeptide would be operably linked with the regulatory sequence.<br>
EXPRESSION VECTORS<br><br>
The present invention also relates to a recombinant expression vector comprising the nucleic acid construct of the invention, a promoter, and transcriptional and translational stop signals.<br>
The recombinant expression vector comprising the nucleic acid construct encoding the enzyme of the invention may be any vector that may conveniently be subjected to recombinant DNA procedures.<br>
The choice of vector will often depend on the host cell into which it is to be introduced. Thus, the vector may be an autonomously replicating vector, i.e. a vector that exists as an extrachromosomal entity, the replication of which is independent of chromosomal replication, e.g. a plasmid. Alternatively, the vector may be one that on introduction into a host cell is integrated into the host cell genome in part or in its entirety and replicated together with the chromosome(s) into which it has been integrated.<br>
The vector is preferably an expression vector in which the DNA sequence encoding the enzyme of the invention is operably linked to additional segments required for transcription of the DNA. In general, the expression vector is derived from plasmid or viral DNA, or may contain elements of both. The term "operably linked" indicates that the segments are arranged so that they function in concert for their intended purposes, e.g. transcription initiates in a promoter and proceeds through the DNA sequence coding for the enzyme.<br>
The promoter may be any DNA sequence that shows transcriptional activity in the host cell of choice and may be derived from genes encoding proteins either homologous or heterologous to the host cell<br>
Examples of suitable promoters for use in bacterial host cells include the promoter of the Bacillus stearothermophilus maltogenic amylase gene, the Bacillus licheniformis alpha-amylase gene, the Bacillus amyloliquefaciens alpha-amylase gene,' the Bacillus subtilis alkaline protease gene, or the Bacillus pumilus xylosidase gene, or the phage Lambda PR or PL promoters or the E. coli lac, trg or tec promoters.<br>
The DNA sequence encoding the enzyme of the invention may also, if necessary, be operably connected to a suitable terminator.<br>
The recombinant vector of the invention may further comprise a DNA sequence enabling the vector to replicate in the host cell in question.<br><br>
The vector may also comprise a selectable marker, e.g. a gene the product of which complements a defect in the host cell, or a gene encoding resistance to e.g. antibiotics like kanamycin, chloramphenicol, erythromycin, tetracycline, spectinomycine, or the like, or resistance to heavy metals or herbicides.<br>
To direct an enzyme of the present invention into the secretory pathway of the host cells, a secretory signal sequence (also known as a leader sequence, prepro sequence or pre sequence) may be provided in the recombinant vector. The secretory signal sequence is joined to the DNA sequence encoding the enzyme in the correct reading frame. Secretory signal sequences are commonly positioned 5' to the DNA sequence encoding the enzyme. The secretory signal sequence may be that normally associated with the enzyme or may be from a gene encoding another secreted protein.<br>
The procedures used to ligate the DNA sequences coding for the present enzyme, the promoter and optionally the terminator and/or secretory signal sequence, respectively, or to assemble these sequences by suitable PCR amplification schemes, and to insert them into suitable vectors containing the information necessary for replication or integration, are well known to persons skilled in the art (cf., for instance, Sambrook et al., op.cit).<br>
HOST CELL<br>
The present invention also relates to a recombinant host cell comprising the nucleic acid<br>
construct of the invention.<br>
The DNA sequence encoding the present enzyme introduced into the host cell may be either homologous or heterologous to the host in question, If homologous to the host cell, i.e. produced by the host ceil in nature, it will typically be operably connected to another promoter sequence or, if applicable, another secretory signal sequence and/or terminator sequence than in its natural environment. The term "homologous" is intended to include a DNA sequence encoding an enzyme native to the host organism in question. The term "heterologous" is intended to include a DNA sequence not expressed by the host cell in nature. Thus, the DNA sequence may be from another organism, or it may be a synthetic sequence.<br>
The host cell into which the DNA construct or the recombinant vector of the invention is introduced may be any cell that is capable of producing the present enzyme and includes bacteria, yeast, fungi and higher eukaryotic cells including plants.<br><br>
Examples of bacterial host cells which on cultivation are capable of producing the enzyme of the invention are gram-positive bacteria such as strains of Bacillus, such as strains of B. subtilis, B. licheniformis, B. lentus, B. brevis, B. stearothermophilus, B. alkalophllus, B. amyloliquefaciens, B. coagulans, B. circulans, B. lautus, S. megaterium or B. thuringiensis, in particular B. lentus, or strains of Streptomyces, such as S. lividans or S. murinus, or gram-negative bacteria such as Escherichia coli.<br>
The transformation of the bacteria may be effected by protoplast transformation, electroporation, conjugation, or by using competent cells in a manner known per se (cf. Sambrooket al., supra).<br>
When expressing the enzyme in bacteria such as £ coli, the enzyme may be retained in the cytoplasm, typically as insoluble granules (known as inclusion bodies), or may be directed to the periplasmic space by a bacterial secretion sequence. In the former case, the cells are lysed and the granules are recovered and denatured after which the enzyme is refolded by diluting the denaturing agent. In the latter case, the enzyme may be recovered from the periplasmic space by disrupting the cells, e.g. by sonication or osmotic shock, to release the contents of the periplasmic space and recovering the enzyme.<br>
When expressing the enzyme in gram-positive bacteria such as Bacillus or Streptomyces strains, the enzyme may be retained in the cytoplasm, or may be directed to the extracellular medium by a bacterial secretion sequence, in the latter case, the enzyme may be recovered from the medium as described below.<br>
In another embodiment of the invention, the fungal host cell is a yeast cell. "Yeast" as used herein includes ascosporogenous yeast (Endomycetales), basidiosporogenous yeast, and yeast belonging to the Fungi Imperfecti (Blastomycetes). Since the classification of yeast may change in the future, for the purposes of this invention, yeast shall be defined as described in Biology and Activities of Yeast{ Skinner, FA, Passmore, S.M., and Davenport, R.R., edsM Soc. App. Bacteriol. Symposium Series No. 9,1980).<br>
In a preferred embodiment, the yeast host cell is a Candida, Hansenula, Kluyveromyces, Pichiat Saccharomyces, Schizosaccharomyces, orYarrowia cell.<br>
In a most preferred embodiment, the yeast host cell is a Saccharomyces carlsbergensis, Saccharomyces cerevisiae, Saccharomyces diastaticus, Saccharomyces douglasii, Saccharomyces kluyveri, Saccharomyces norbensis or Saccharomyces oviformis cell.   In another most<br><br>
preferred embodiment, the yeast host cell is a Kluyveromyces lactis cell. In another most preferred embodiment, the yeast host cell is a Yarrowia lipolytics cell.<br>
In another preferred embodiment, the fungal host cell is a filamentous fungal cell. "Filamentous fungi" include all filamentous forms of the subdivision Eumycota and Oomycota (as defined by Hawksworth et a/., 1995, supra). The filamentous fungi are characterized by a mycelial wall composed of chitin, cellulose, glucan, chitosanf mannan, and other complex polysaccharides. Vegetative growth is by hyphal elongation and carbon catabolism is obligately aerobic. In contrast, vegetative growth by yeasts such as Saccharomyces cerevisiae is by budding of a unicellular thallus and carbon catabolism may be fermentative.<br>
In an even more preferred embodiment, the filamentous fungal host cell is a cell of a species of, but not limited to, Acremonium, Aspergillus, Fusarium, Humicola, Mucor, Myceliophthora, Neurospora, Penicillium, Thielavia, Tolypocladium,'or Trichoderma.<br>
In a most preferred embodiment, the filamentous fungal host cell is an Aspergillus awamori, Aspergillus foetidus, Aspergillus japonicus, Aspergillus nidulans, Aspergillus niger or Aspergillus oryzae cell. In another most preferred embodiment, the filamentous fungal host cell is a Fusarium bactridioides, Fusarium "terealis, Fusarium crookwellense, Fusarium culmorum, Fusarium graminearum, Fusarium graminum, Fusarium heterosporum, Fusarium negundi, Fusarium oxysporum, Fusarium reticulatum, Fusarium roseum, Fusarium sambucinum, Fusarium sarcochroum, Fusarium sporotrichioides, Fusarium sulphureum, Fusarium torulosum, Fusarium trichothecioides, or Fusarium venenatum cell. In an even most preferred embodiment, the filamentous fungal parent cell is a Fusarium venenatum (Nirenberg sp. nov.) cell. In another most preferred embodiment, the filamentous fungal host cell is a Humicola insolens, Humicola lanuginosa, Mucor miehei} Myceliophthora thermophilat Neurospora crassat Penicillium purpurogenum, Thielavia terrestris, Trichoderma harzianum, Trichoderma koningii, Trichoderma longibrachiatum, Trichoderma reesei, or Trichoderma viride cell.<br>
Fungal cells may be transformed by a process involving protoplast formation, transformation of the protoplasts, and regeneration of the cell wall in a manner known per se. Suitable procedures for transformation of Aspergillus host cells are described in EP 238 023 and Yelton et al.t 1984, Proceedings of the National Academy of Sciences USA 81: 1470-1474. Suitable methods for transforming Fusarium species are described by Malardier et a/., 1989, Gene 78; 147-156 and WO 96/00787. Yeast may be transformed using the procedures described by Becker and Guarente, In Abelson, J.N. and Simon, M.I., editors, Guide to Yeast Genetics and Molecular Biology, Methods in Enzymology, Volume 194, pp 182-187, Academic Press, Inc.,<br><br>
New York; Ito etal., 1983, Journal of Bacteriology 153:163; and Hinnen et a/., 1978, Proceedings of the National Academy of Sciences USA 75:1920.<br>
METHOD OF PRODUCING A SUBTILASE OF THE INVENTION<br>
The present invention further relates to a method for producing a subtilase of the invention,<br>
the method comprising:<br>
cultivating a recombinant host cell of the invention under conditions conducive to the production of the subtilase; and<br>
recovering the subtilase.<br>
When an expression vector comprising a DNA sequence encoding the enzyme is transformed into a heterologous host cell, it is possible to enable heterologous recombinant production of the enzyme of the invention.<br>
Thereby it is possible to make a highly purified subtilase composition, characterized in being free from homologous impurities.<br>
In this context, homologous impurities mean any impurities (e.g. other polypeptides than the enzyme of the invention) that originate from the homologous cell where the enzyme of the invention .is originally obtained from.<br>
The medium used to culture the transformed host cells may be any conventional medium suitable for growing the host cells in question. The expressed subtilase may conveniently be secreted into the culture medium and may be recovered there from by well-known procedures including separating the cells from the medium by centrifugation or filtration, precipitating proteinaceous components of the medium by means of a salt such as ammonium sulfate, followed by chromatographic procedures such as ion exchange chromatography, affinity chromatography, or the like.<br>
USE OF A SUBTILASE OF THE INVENTION<br>
A subtilase enzyme of the invention may be used for a number of industrial applications, in particular within the detergent industry. Thus, the present invention also relates to a cleaning or detergent composition, preferably a laundry or dish washing composition, comprising the subtilase enzyme of the invention.   :1<br><br>
DETERGENT   COMPOSITIONS   COMPRISING   THE   SUBTILASE   ENZYME   OF   THE<br>
INVENTION:<br>
In general, cleaning and detergent compositions are well described in the art and reference is<br>
made to WO 96/34946; WO 97/07202; WO 95/30011 for further description of suitable<br>
cleaning and detergent compositions.<br>
Furthermore the examples herein demonstrate the improvements in performance on egg stains for the subtilases of the invention.<br>
- Detergent Compositions<br>
The enzyme of the invention may be added to and thus become a component of a cleaning or detergent composition.<br>
The detergent composition of the invention may for example be formulated as a hand or machine laundry detergent composition including a laundry additive composition suitable for pre-treatment of stained fabrics and a rinse added fabric softener composition, or be formulated as a detergent composition for use in general household hard surface cleaning operations, or be formulated for hand or machine dishwashing operations.<br>
In a specific aspect, the invention provides a detergent additive comprising the subtilase enzyme of the invention. The detergent additive as well as the detergent composition may comprise one or more other enzymes such as another protease, a lipase, a cutinase, an amylase, a carbohydrase, a cellulase, a pectinase, a mannanase, an arabinase, a galactanase, a xy-lanase, an oxidase, e.g., a laccase, and/or a peroxidase.<br>
In general the properties of the chosen enzyme(s) should be compatible with the selected detergent, (i.e. pH-optimum, compatibility with other enzymatic and non-enzymatic ingredients, etc.), and the enzyme(s) should be present in effective amounts.<br>
Proteases: Suitable proteases include those of animal, vegetable or microbial origin. Microbial origin is preferred. Chemically modified or protein engineered mutants are included. The protease may be a serine protease or a metallo protease, preferably an alkaline microbial protease or a trypsin-like protease. Examples of alkaline proteases are subtilisins, especially those derived from Bacillus, e.g., subtilisin Novo, subtilisin Carlsberg, subtilisin 309, subtilisin 147andsubtilisin 168 (described in WO 89/06279). Examples of trypsin-like proteases are trypsin (e.g. of porcine or bovine origin) and the Fusarium protease described in WO 89/06270 and WO 94/25583.<br><br>
Examples of useful proteases are the variants described in WO 92/19729, WO 98/20115, WO 98/20116, and WO 98/34946, especially the variants with substitutions in one or more of the following positions; 27, 36, 56, 76, 87, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 120, 123, 159, 167, 170, 206, 218, 222, 224, 232, 235, 236, 245, 248, 252 and 274 (BPN' numbering).<br>
Preferred   commercially   available   protease   enzymes   include   Alcalase™,	Savinase™,<br>
Primase™,   Duralase™,   Esperase™,   and   Kannase™   (Novozymes   A/S),	Maxatase™,<br>
Maxacal™, Maxapem™, Properase™, Purafect™, Purafect OxP™, FN2™,	and FN3™<br>
(Genencor International Inc.).<br>
Lipases: Suitable lipases include those of bacterial or fungal origin. Chemically modified or protein engineered .mutants are included. Examples of useful lipases include lipases from Humicola (synonym Thermomyces), e.g. from H. lanuginosa (T. lanuginosus) as described in EP 258 068 and EP 305 216 or from H. insolens as described in WO 96/13580, a Pseudomonas lipase, e.g. from P. alcaligenes or P. pseudoalcaligenes (EP 218 272), P. cepacia (EP 331 376), P. stutzeri (GB 1,372,034), P. fluorescens, Pseudomonas sp. strain SD 705 (WO 95/06720 and WO 96/27002), P. wisconsinensis (WO 96/12012), a Bacillus lipase, e.g. from S. subtilis (Dartois et al. (1993), Biochemica et Biophysica Acta, 1131, 253-360), B. stearothermophilus (JP 64/744992) or B. pumilus (WO 91/16422).<br>
Other examples are lipase variants such as those described in WO 92/05249, WO 94/01541, EP 407 225, EP 260 105, WO 95/35381, WO 96/00292, WO 95/30744, WO 94/25578, WO 95/14783, WO 95/22615, WO 97/04079 and WO 97/07202.<br>
Preferred commercially available lipase enzymes include Upolase™ and Lipolase Ultra™ (Novozymes A/S).<br>
Amylases: Suitable amylases (a and/or p) include those of bacterial or fungal origin. Chemically modified or protein engineered mutants are included. Amylases include, for example, a-amylases obtained from Bacillus, e.g. a special strain of B. licheniformis, described in more detail in GB 1,296,839.<br>
Examples of useful amylases are the variants described in WO 94/02597, WO 94/18314, WO 96/23873, and WO 97/43424, especially the variants with substitutions in one or more of the following positions: 15, 23, 105, 106, 124, 128, 133, 154, 156, 181, 188, 190, 19/ 202, 208, 209, 243, 264, 304, 305, 391, 408, and 444.<br><br>
Commercially available amylases are Duramyl™, Termamyl™, Fungamyl™ and BAN™ (Novozymes A/S), Rapidase™ and Purastar™ (from Genencor International Inc.).<br>
Cellulases: Suitable cellulases include those of bacterial or fungal origin. Chemically modified or protein engineered mutants are included. Suitable cellulases include cellulases from the genera Bacillus, Pseudomonas, Humicola, Fusarium, Thielavia, Acremonium, e.g. the fungal cellulases produced from Humicola insolens, Myceliophthora thermophila and Fusarium oxysporum disclosed in US 4,435,307, US 5,648,263, US 5,691,178, US 5,776,757 and WO 89/09259.<br>
Especially suitable cellulases are the alkaline or neutral cellulases having colour care benefits. Examples of such cellulases are cellulases described in EP 0 495 257, EP 0 531 372, WO 96/11262, WO 96/29397, WO 98/08940. Other examples are cellulase variants such as those described in WO 94/07998, EP 0 531 315, US 5,457,046, US 5,686,593, US 5,763,254, WO 95/24471, WO 98/12307 and PCT/DK98/00299.<br>
Commercially available cellulases include Celiuzyme™, and Carezyme™ (Novozymes A/S), Clazinase™, and Puradax HA™ (Genencor International Inc.), and KAC-500(B)™ (Kao Corporation).<br>
Peroxidases/Oxidases: Suitable peroxidases/oxidases include those of plant, bacterial or fungal origin. Chemically modified or protein engineered mutants are included. Examples of useful peroxidases include peroxidases from Coprinus, e.g. from C. cinereus, and variants thereof as those described in WO 93/24618, WO 95/10602, and WO 98/15257.<br>
The detergent enzyme(s) may be included in a detergent composition by adding separate additives containing one or more enzymes, or by adding a combined additive comprising all of these enzymes. A detergent additive of the invention, i.e. a separate additive or a combined additive, can be formulated e.g. as a granulate, a liquid, a slurry, etc. Preferred detergent additive formulations are granulates, in particular non-dusting granulates, liquids, in particular stabilized liquids, or slurries.<br>
Non-dusting granulates may be produced, e.g. as disclosed in US 4,106,991 and 4,661,452 and may optionally be coated by methods known in the art. Examples of waxy coating materials are polyethylene oxide) products (polyethylene glycol, PEG) with mean molar weights of 1000 to 20000; ethoxylated nonylphenols having from 16 to 50 ethylene oxide units; ethoxy-<br><br>
lated fatty alcohols in which the alcohol contains from 12 to 20 carbon atoms and in which there are 15 to 80 ethylene oxide anits; fatty alcohols; fatty acids; and mono-anddi-and triglycerides of fatty acids. Examples of film-forming coating materials suitable for application by fluid bed techniques are given in GB 1483591. Liquid enzyme preparations may, for instance, be stabilized by adding a polyol such as propylene glycol, a sugar or sugar alcohol, lactic acid or boric acid according to established methods. Protected enzymes may be prepared according to the method disclosed in EP 238,216.<br>
The detergent composition of the invention may be in any convenient form, e.g. a bar, a tablet, a powder, a granule, a paste or a liquid. A liquid detergent may be aqueous, typically containing up to 70% water and 0-30% organic solvent, or non-aqueous.<br>
The detergent composition typically comprises one or more surfactants, which may be non-ionic including semi-polar and/or anionic and/or catlonic and/or zwitterionic. The surfactants are typically present at a level of from 0.1% to 60% by weight.<br>
When included therein the detergent will usually contain from about 1% to about 40% of an anionic surfactant such as linear alkylbenzenesulfonate, alpha-olefinsulfonate, alkyl sulfate (fatty alcohol sulfate), alcohol ethoxysulfate, secondary alkanesulfonate, alpha-sulfo fatty acid methyl ester, alkyl- or alkenylsuccinic acid or soap.<br>
When included therein the detergent will usually contain from about 0.2% to about 40% of a non-ionic surfactant such as alcohol ethoxylate, nonylphenol ethoxylate, alkylpolyglycoside, alkyldimethylamineoxide, ethoxylated fatty acid monoethanolamide, fatty acid monoethanola-mide, polyhydroxy alkyl fatty acid amide, or N-acyl N-alkyl derivatives of glucosamine ("glucamides").<br>
The detergent may contain 0-65% of a detergent builder or complexing agent such as zeolite, diphosphate, triphosphate, phosphonate, carbonate, citrate, nitrilotriacetic acid, ethylenedia-minetetraacetic acid, diethylenetriaminepentaacetic acid, alkyl- or alkenylsuccinic acid, soluble silicates or layered silicates (e.g. SKS-6 from Hoechst).<br>
The detergent may comprise one or more polymers. Examples are carboxymethylcellulose, poly(vinylpyrrolidone), poly (ethylene glycol), polyvinyl alcohol), poly(vinylpyridine-N-oxide),<br><br>
poly(vinylimidazole), polycarboxylates such as polyacrylates, maleic/acrylic acid copolymers and lauryl methacrylate/acrylic acid copolymers.<br><br>
The detergent may contain a bleaching system that may comprise a H202 source such as perborate or percarbonate that may be combined with a peracid-forming bleach activator such as tetraacetylethylenediamine or nonanoyloxybenzenesulfonate. Alternatively, the bleaching system may comprise peroxyacids of e.g. the amide, imide, or sulfone type.<br>
The enzyme(s) of the detergent composition of the invention may be stabilized using conventional stabilizing agents, e.g. a polyol such as propylene glycol or glycerol, a sugar or sugar alcohol, lactic acid, boric acid, or a boric acid derivative, e.g. an aromatic borate ester, or a phenyl boronic acid derivative such as 4-formyIphenyl boronic acid, and the composition may be formulated as described in e.g. WO 92/19709 and WO 92/19708.<br>
The detergent may also contain other conventional detergent ingredients such as e.g. fabric conditioners including clays, foam boosters, suds suppressors, anti-corrosion agents, soil-suspending agents, anti-soil redeposition agents, dyes, bactericides, optical brighteners, hy-drotropes, tarnish inhibitors, or perfumes.<br>
It is at present contemplated that in the detergent compositions any enzyme, in particular the enzyme of the invention, may be added in an amount corresponding to 0.01-100 mg of enzyme protein per litre of wash liquor, preferably 0.05-5 mg of enzyme protein per litre of wash liquor, in particular 0.1-1 mg of enzyr^e protein per litre of wash liquor.<br>
The enzyme of the invention may additionally be incorporated in the detergent formulations disclosed in WO 97/07202, which is hereby incorporated as reference.<br>
The invention is described in further detail in the following examples that are not in any way intended to limit the scope of the invention as claimed.<br>
In the detergent compositions, the abbreviated component identifications have the following meanings:<br>
LAS:	Sodium linear C12 alky! benzene sulfonate<br>
TAS:	Sodium tallow alkyi sulphate<br>
XYAS:	Sodium C1x- C-IY alkyl sulfate<br>
SS:	Secondary soap surfactant of formula 2-butyl octanoic acid<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
The subtilisins having the amino acid sequence shown in SEQ ID NO:2 was located in vector pJRoC112, which is very similar to plasmid pKH400 (previously described in WO 98/41623). The plasmids are identical outside the regions encoding the mature subtilisin, i.e. the origin of replication; the cat gene conferring resistance towards chloramphenicol, the promoter directing the initiation of transcription of the subtilisin and the pre/pro regions from Savinase® are identical in these plasmids. Differences are only found within the part of the gene encoding the mature subtilisin.<br>
This plasmid replicates both in £ coli and in Bacillus subtilis. In Bacillus subtilis the subtilisin according to the invention was expressed from this plasmid. Fermentation and purification of<br>
the protease is described below.<br>
PKH400 was constructed from pJS3 (£ coli - B. subtilis shuttle vector containing a synthetic gene encoding for subtilase 309 (Savinase®) as described by J. Schi0dt et al. in Protein and Peptide Letters, 3, 39-44 (1996)) by introduction of two BamHI sites at positions 1841 and 3730.<br>
The mature gene has been subcloned into plasmid pzero-2 (Invitrogen, Groningen, The Netherlands). An approximately 1240 bp Pmel-BamHI fragment containing the complete mature region of the subtilase having the amino acid sequence shown in SEQ ID NO:2 was ligated with vector pZero-2 and digested with restriction endonucleases BamHI-EcoRV. The ligation mixture was transformed into competent £ coli cells. Transformants were analysed by PCR to verify the presence of the inserted fragment and the part of this fragment encoding the mature subtilisin was sequenced. The resulting plasmid, denoted pTVB364, was deposited on 10 February 2000 at DSMZ and was given the accession number DSM 13306.<br>
Fermentation<br>
Fermentations for the production of subtilase enzymes were performed at 30°C on a rotary shaking table (300 r.p.m.) in 500 ml baffled Erlenmeyer flasks containing 100 ml BPX medium for 5 days.<br>
Consequently, in order to make e.g. a 2 litre broth 20 Erlenmeyer flasks were fermented<br>
simultaneously.<br>
Media:<br>
BPX Medium Composition (per litre)<br>
Potato starch	100g<br><br>
Ground barley	50 g<br>
Soybean flour	20 g<br>
Na2HP04x12H20	9g<br>
Pluronic	0.1 g<br>
Sodium caseinate	10 g<br>
The starch in the medium was liquefied with a-amylase and the medium was sterilized by heating at 120°C for 45 minutes. After sterilization the pH of the medium was adjusted to 9 by addition of NaHC03 to 0.1 M.<br>
Purification<br>
This procedure relates to purification of a 2 litre scale fermentation for the production of the<br>
subtilases of the invention in a Bacillus host cell.<br>
Approximately 1.6 litres of fermentation broth was centrifuged at 5000 rpm for 35 minutes in 1 litre beakers. The supernatants were adjusted to pH 6.5 using 10% acetic acid and filtered on Seitz Supra S100 filter plates.<br>
The filtrates were concentrated to approximately 400 ml using an Amicon CH2A UF unit equipped with an Amicon S1Y10 UF cartridge. The UF concentrate was centrifuged and filtered at room temperature prior to absorption on a Bacitracin affinity column at pH 7. The subtilase was eluted from the Bacitracin column at room temperature using 25% 2-propanol and 1 M sodium chloride in a buffer solution with 0.01 dimethylglutaric acid, 0.1 M boric acid and 0.002 M calcium chloride adjusted to pH 7.<br>
The fractions with protease activity from the Bacitracin purification step were combined and applied to a 750 ml Sephadex G25 column (5 cm diameter) equilibrated with a buffer containing 0.01 dimethylglutaric acid, 0.2 M boric acid and 0.002 m calcium chloride adjusted to pH 6.5.<br>
Fractions with proteolytic activity from the Sephadex G25 column were combined and applied to a 150 ml CM Sepharose CL 6B cation exchange column (5 cm diameter) equilibrated with a buffer containing 0.01 M dimethylglutaric acid, 0.2 M boric acid, and 0.002 M calcium chloride<br>
adjusted to pH 6.5.<br>
The protease was eluted using a linear gradient of 0-0.1 M sodium chloride in 2 litres of the same buffer.<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
CLAIMS<br>
1.	A subtilase enzyme being selected from the group consisting of<br>
a)	a subtilase having an amino acid sequence that has at more than 99.26% identity with the amino acid sequence shown as amino acids 1 to 269 of SEQ ID NO:2; or<br>
b)	a subtilase encoded by the subtilase encoding part of the polynucleotide cloned into a plasmid fragment present in Escherichia coli MT173 DSM 15575, or a variant thereof having more than 99.26% identity to said subtilase; or<br>
c)	a subtilase having an amino acid sequence that has more than 97.40% identity with the amino acid sequence shown as amino acids 1 to 269 of SEQ ID NO:4; or<br>
d)	a subtilase encoded by the subtilase encoding part of the polynucleotide cloned into a plasmid fragment present in Escherichia coli MT173 DSM 15574, or a variant thereof having more than 97.40% identity to said subtilase.<br>
2.	A subtilase according to claim 1, having<br>
I)	an amino acid sequence which has more than 99.26%, or more than 99.63% identity with the amino acid sequence described in 1a) or 1b); or<br>
II)	an amino acid sequence which has more than 97.40%, or more than 97.77%, or more than 98.14%, or more than 98.51%, or more than 98.89%, or more than 99.26%, or more than 99.63% identity with the amino acid sequence described in 1c) or 1d).<br><br>
3.	A subtilase according to claim 2, which consists of either the amino acid sequence shown as amino acids 1 to 269 of SEQ ID NO:2, or of the amino acid sequence shown as amino acids 1 to 269 of SEQ ID NO:4.<br>
4.	A subtilase according to claim 1  having<br>
I) an amino acid sequence which has more than 99.26%, or more than 99.63% identity with the subtilase encoded by the subtilase encoding part of the polynucleotide cloned into a plasmid fragment present in Escherichia coli MT173 DSM 15575; or<br>
II) an amino acid sequence which has more than 97.40%, or more than 97.77%, or more than 98.14%, or more than 98.51%, dT more than 98.89%, or more than 99.26%F or more than 99.63% identity with the subtilase encoded by the subtilase encoding part of the polynucleotide cloned into a plasmid fragment present in Escherichia coli MT173 DSM 15574.<br>
5.	A subtilase according to claim 4, which consists of<br>
I) the subtilase encoded by the subtilase encoding part of the polynucleotide cloned into a plasmid fragment present in Escherichia coli MT173 DSM 15575; or<br><br>
II) the subtilase encoded by the subtilase encoding part of the polynucleotide cloned into a plasmid fragment present in Escherichia coli MT173 DSM 15574.<br>
6.	A subtilase according to any of the preceding claims, wherein the subtilase is a variant of a<br>
subtilase having<br>
I)	the amino acid sequence shown as amino acids 1 to 269 of SEQ ID. NO:2 comprising a substitution, deletion, and/or insertion of one or more amino acid residues; or<br>
II)	the amino acid sequence shown as amino acids 1 to 269 of SEQ ID NO:4 comprising a substitution, deletion, and/or insertion of one or more amino acid residues.<br><br>
7.	A subtilase according to claim 6, comprising at least one modification in one of the positions 27, 36, 56, 76, 87, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 120, 123, 159, 167, 170, 206, 218, 222, 224, 232, 235, 236, 245, 248, 252 and 274 (BASBPN numbering).<br>
8.	A subtilase according to claim 7, wherein the modification is selected from the group consisting of K27R, *36D, S56P, N76D, S87N, G97N, S101G, S103A, V104A, V104I, V104N, V104Y, H120D, N123S, G159D, Y167, R170, Q206E, N218S, M222S, M222A, T224S, A232V, K235L, Q236H, Q245R, N248D, N252K and T274A (BASBPN numbering).<br>
9.	An isolated polynucleotide comprising a polynucleotide that encodes for the subtilase defined in any of claims 1-8.<br>
10.	An isolated polynucleotide encoding a subtilase, selected from the group consisting of<br>
(a)	a polynucleotide having at (east 88% identity with the polynucleotide shown as nucleotides 1 to 807 SEQ ID NO:1; or<br>
(b)	the subtilase encoding part of the polynucleotide that has been cloned into a plasmid present in Escherichia coli MT173 DSM 15575, or a variant thereof having at least 88% identity to said nucleic acid sequence; or<br>
(c)	a polynucleotide having at least 88% identiy with the polynucleotide shown as nucleotides 1 to 807 SEQ ID NO:1; or<br>
(d)	the subtilase encoding part of the polynucleotide that has been cloned into a plasmid present in Escherichia coli MT173 DSM 15574, or a variant thereof having at least 88% identity to said nucleic acid sequence.<br>
11.	A polynucleotide according to claim 10 having<br>
I) a nucleic acid sequence which has at least 89%, at least 90%, at least 91%, at least 92%, at least 93%, at least 94%, at least 95%, at least 96%, at least 97%, at least 98%, or at least<br><br>
99% identity with the polynucleotide shown as nucleotides 1 to 807 of SEQ ID NO:1; or II) a nucleic acid sequence which has at least 89%, at least 90%, at least 91%, at least 92%, at least 93%, at least 94%, at least 95%, at least 96%, at least 97%, at least 98%, or at least 99% identity with the polynucleotide shown as nucleotides 1 to 807 of SEQ ID NO:3.<br>
12.	A polynucleotide according to claim 10, having<br>
I)	a nucleic acid sequence which has at least 89%, at least 90%, at least 91%, at least 92%, at<br>
least 93%, at least 94%, at least 95%, at least 96%, at least 97%, at least 98%, or at least<br>
99% identity with the subtilase encoding part of the.polynucleotide which has been cloned into<br>
a plasmid present in Escherichia coli MT173 DSM 15575; or<br>
II)	a nucleic acid sequence which has at least 89%, at least 90%, at least 91%, at least 92%,<br>
at least 93%, at least 94%, at least 95%, at least 96%, at least 97%, at least 98%, or at least<br>
99% identity with the subtilase encoding part of the polynucleotide which has been cloned into<br>
a plasmid present in Escherichia coli MT173 DSM 15574.<br>
13.	A nucleic acid construct comprising the nucleic acid sequence of any of claims 9 -12 op-erably linked to one or more control sequences capable of directing the expression of the subtilase in a suitable host.<br>
14.	A recombinant expression vector comprising the nucleic acid construct of claim 13, a promoter, and transcriptional and translational stop signals.<br>
15.	A recombinant host cell comprising the nucleic acid construct of claim 13.<br>
16.	A host cell according to claim 15, which is a bacterium, preferably a Bacillus, especially a Bacillus lentus.<br>
17.	A host cell according to claim 15, which is a fungus or yeast, preferably a filamentous fungus, especially an Aspergillus.<br>
18.	A method for producing the subtilase according to any of claims 1-8, the method comprising:<br><br>
(a)	cultivating a recombinant host cell as defined in any of claims 15-17 under conditions conducive to the production of the subtilase; and<br>
(b)	recovering the subtilase.<br><br><br><br>
C <br><br><br>
VARIANT SUBTILISIN ENZYMES (SUBTILASES)<br>
TECHNICAL FIELD<br>
The present invention relates to novel subtilases having an improved performance on soil, in<br>
particular  egg   stains.   These  subtilases  are   useful  exhibiting  excellent  or  improved<br>
performance on egg stains when used in e.g. cleaning or detergent compositions, such as<br>
laundry detergent compositions and dish wash compositions, including automatic dish wash<br>
compositions.<br>
The present invention also relates to isolated poiynucleotides encoding the subtilases, nucleic acid constructs, recombinant expression vectors, host cells comprising the nucleic acid construct, and methods for producing and using the subtilases of the invention. Further, the present invention relates to cleaning and detergent compositions comprising the subtilase. enzymes of the invention as well as to use of such enzymes in detergent compositions and for removal of egg stains.<br>
BACKGROUND OF THE INVENTION<br>
In the detergent industry enzymes have for more than 30 years been implemented in washing formulations. Enzymes used in such formulations comprise proteases, lipases, amylases, cellulases, mannosidases as well as other enzymes or mixtures thereof. Commercially most important enzymes are proteases.<br>
An increasing number of commercially used proteases is protein engineered variants of naturally occurring wild type proteases, e.g. DURAZYM® (Novozymes A/S), RELASE® (Novozymes A/S), MAXAPEM® (Gist-Brocades N .V.), PURAFECT* (Genencor International, Inc.).<br>
Further, a number of protease variants is described in the art, such as in EP 130756 (GENENTECH)(corresponding to US Reissue Patent No. 34,606 (GENENCOR)); EP 214435 (HENKEL); WO 87/04461 (AMGEN); WO 87/05050 (GENEX); EP 260105 (GENENCOR); Thomas, Russell, and Fersht (1985) Nature 318 375-376; Thomas, Russell, and Fersht (1987) J. Mol. Biol. 193 803-813; Russel and Fersht Nature 328 496-500 (1987); WO 88/08028 (Gefiex); WO 88/08033 (Amgen); WO 95/27049 (SOLVAY S.A.); WO 95/30011 (PROCTER &amp; GAMBLE COMPANY); WO 95/30010 (PROCTER &amp; GAMBLE COMPANY); WO 95/29979 (PROCTER &amp; GAMBLE COMPANY); US 5.543.302 (SOLVAY S.A.); EP 251 446 (GENENCOR); WO 89/06279 (NOVOZYMES A/S); WO 91/00345 (NOVOZYMES A/S); EP 525 610 A1 (SOLVAY); WO 94/02618 (GIST-BROCADES N.V.).<br>
1<br>
Test method used for screening (AMSA) is described in WO 02/42740 (NOVOZYMES A/S).<br>
WO 01/75087 (MAXYGEN, INC. / NOVOZYMES A/S) describes subtilisin homoiogues that are improved for a variety of specific properties including thermal stability, activity at low temperature and alkaline stability.<br>
WO 01/68821 (NOVOZYMES A/S) describes subtilase enzymes, which are suitable for removal of egg stains from for example laundry and/or hard surfaces.<br>
However, even though a number of useful proteases and protease variants have been described, there is still a need for further improvement of proteases or protease variants for a -   number of industrial uses..<br>
In particular, the problem of removing egg stains from e.g. laundry or hard surfaces has been pronounced due to the fact that substances present in the egg white inhibit many serine proteases.<br>
Therefore, an object of the present invention is to provide improved subtilase enzymes, which are suitable for removal of egg stains from for example laundry and/or hard surfaces.<br>
SUMMARY OF THE INVENTION<br>
Thus, in a first aspect the present invention relates to a subtilase enzyme having improved wash performance on egg stains, the subtilase being selected from the group consisting of<br>
(a)	a subtilase having an amino acid sequence which has more than 99.26% identity<br>
with the amino acid sequence shown as amino acids 1 to 269 of SEQ ID NO;2; and<br>
(b)	a subtilase encoded by the subtilase encoding part of the polynucleotide cloned into<br>
a plasmid fragment present in Escherichia coli MT173 DSM 15575, or a variant<br>
thereof having at least 99.26% identity to said subtilase; and<br>
(c)	a subtilase having an amino acid sequence which has more than 97.40% identity<br>
with the amino acid sequence shown as amino acids 1 to 269 of SEQ ID NO:4; and<br>
(d)	a subtiiase encoded by the subtilase encoding part of the polynucleotide cloned into<br>
a piasmid fragment present in Escherichia coli MT173 DSM 15574, or a variant<br>
thereof having at least 97.40% identity to said subtilase.<br>
In a second aspect the present invention relates to an isolated polynucleotide comprising a nucleic acid sequence that encodes for the subtilases according to the invention.<br>
in a third aspect the present invention relates to an isolated polynucleotide encoding a subtilase, selected from the group consisting of<br>
(a)	a polynucleotide having at least 88% identity with the nucleic acid sequence shown ■<br>
as nucleotides 1 to 807 SEQ ID NO:1; and<br>
(b)	the subtilase encoding part of the polynucleotide that has been cloned into a plasmid<br>
present in Escherichia coil MT173 DSM 15575, or a variant thereof having at least 88% identity to said nucleic acid sequence.<br>
(c)	a polynucleotide having at least 88% identity with the nucleic acid sequence shown as<br>
nucleotides 1 to 807 SEQ ID NO:3; and<br>
(d)	the subtilase encoding part of the polynucleotide that has been cloned into a plasmid<br>
present in Escherichia coli MT173 DSM 15574, or a variant thereof having at least 88% identity to said nucleic acid sequence.<br>
In a fourth aspect the present invention relates to a nucleic acid construct comprising the nucleic acid sequence according to the invention operably linked to one or more control sequences capable of directing the expression of the subtilase in a suitable host.<br>
In a fifth aspect the present invention relates to a recombinant expression vector comprising the nucleic acid construct according to the invention, a promoter, and transcriptional and trans-lational stop signals.<br>
In a sixth aspect the present invention relates to a recombinant host cell comprising the nucleic acid construct of the invention.<br>
In a seventh aspect the present invention relates to a method for producing the subtilase according to the invention, the method comprising:<br>
(a)	cultivating a recombinant host cell according to the invention under conditions condu<br>
cive to the production of the subtilase; and<br>
(b)	recovering the subtilase.<br>
In an eight aspect the present invention relates to a cleaning or detergent composition,<br>
preferably a laundry or dish wash composition, comprising the subtilase according to the invention.<br>
Further aspects of the present invention relate to use of the subtilases according to the invention in a cleaning or detergent composition; use of the subtilases or the compositions according to the invention for removal of egg stains; a method for cleaning or washing, including a method for removal of egg stains from a hard surface or laundry comprising contacting the hard surface or the laundry with the composition of the invention.<br>
Concerning alignment and numbering reference is made to fig. 1 which shows alignments between subtiiisin BPN1 (a) (BASBPN) and the novel subtilases of the invention (b) and (c).<br>
These alignments are in this patent application used as a reference for numbering the residues,<br>
DEF1NITONS<br>
Prior to discussing this invention in further detail, the following terms and conventions will first be defined.<br>
NOMENCLATURE OF AMINO ACIDS<br>
A	=      Ala	=	Alanine<br>
V	Val	Valine<br>
L	=.	Leu	=	Leucine<br>
I	=	lie	=	Isoleucine<br>
P	=	Pro	=	Praline<br>
F	=	Phe	=	Phenylalanine<br>
W	=	Trp	=	Tryptophan<br>
M	=	Met	=	Methionine<br>
G	=	Gly	=	Glycine<br>
S	=	Ser	=	Serine<br>
T	=	Thr	=	Threonine<br>
C	=	Cys	=	Cysteine<br>
Y	=	Tyr	=	Tyrosine<br>
N	=      Asn	=	Asparagine<br>
Q	=      Gin	=	Glutamine<br>
D	=      Asp	=	AsparticAcid<br>
E	=	Gfu	=	Glutamic Acid<br>
K	.         =	Lys	=	Lysine<br>
R	=	Arg	=	Arginine<br>
H	=	His	=	Histidine<br>
X	=	Xaa	=	Any amino acid<br>
NOMENCLATURE OF NUCLEIC ACIDS<br>
A	=      Adenine<br>
G	=      Guanine<br>
C	=      Cytosine<br>
. T	=      Thymine     (only in DNA)<br>
U	=      Uracil         (onlyinRNA)<br>
NOMENCLATURE AND CONVENTIONS FOR DESIGNATION OF VARIANTS<br>
In describing the various subtiiase enzyme variants produced or contemplated according to<br>
the invention, the following nomenclatures and conventions have been adapted for ease of<br>
reference:<br>
A frame of reference is first defined by aligning the isolated or parent enzyme with subtilisin BPN' (BASBPN).<br>
"Homology" or "homologous to" is in the context of the present invention to be understood in its conventional meaning and the "homology" between two amino acid sequences should be determined by use of the "Similarity" defined by the GAP program from the University of Wisconsin Genetics Computer Group (GCG) package using default settings for alignment parameters, comparison matrix, gap and gap extension penalties. Default values for GAP penalties, i.e. GAP creation penalty of 3.0 and GAP extension penalty of 0.1 (Program Manual for the Wisconsin Package, Version 8, August 1994, Genetics Computer Group, 575 Science Drive, Madison, Wisconsin, USA 53711). The method is also described in S.B. Needleman and CD. Wunsch, Journal of Molecular Biology, 48, 443-445 (1970). Identities can be extracted from the same calculation. The homology between two amino acid sequences can also be determined by "identity* or "similarity" using the GAP routine of the GCG package version 9.1 with default setting for alignment parameters, comparison matrix, gap and gap extension penalties can also be applied using the following parameters: gap creation penalty = 8 and gap extension penalty = 8 and all other parameters kept at their default values. The output from the routine is besides the amino acid alignment the calculation of the "Percent Identity" and the "Similarity" between the two sequences. The numbers calculated using GCG package<br>
version 9.1 is slightly different from the version 8.<br>
. Another method is to use known recognized alignments between subtilases, such as the alignment indicated in WO 91/00345. In most cases the differences will not be of any importance.<br>
Such an alignment between subtilisin BPNT (BASBPN) and the novel subtilases of the invention is indicated in fig. 1.<br>
Thereby, a number of deletions and insertions will be defined in relation to BASBPN. In fig. 1, the novel subtilases according to the invention have 6 deletions in positions 36, 58, 158, 162, 163, and 164 in comparison to BASBPN. These deletions are in fig. 1 indicated by asterixes<br>
The various modifications performed in a parent enzyme are indicated in general using three .   elements as follows:<br>
Original amino acid position substituted amino acid<br>
The notation G195E means a substitution of a glycine in position 195 with a glutamic acid.<br>
«<br>
Position substituted amino acid<br>
In the case where the original amino acid residue may be any amino acid residue, a short<br>
hand notation may at times be used indicating only the position and substituted amino acid:<br>
170SeroM70S.<br>
Such a notation is particular relevant in connection with modification(s) in homologous subtilases {vide infra).<br>
Original amino acid position<br>
Such a notation is in particular relevant when the identity of the substituting amino acid residue(s) is immaterial. The substitution of any amino acid residue acid for glycine in position 195 is designated as: Gly195 or G195.<br>
Position<br>
When both the original amino acid(s) and substituted amino acid(s) may comprise any amino<br>
acid, then only the position is indicated, e.g.: 170.<br>
Original amino acid position (substituted amino acidi	substituted amino acidn)<br>
When the original amino acid(s) and/or substituted amino acid(s) may comprise more than one, but not all amino acid(s), then the selected amino acids are indicated inside brackets:{}.<br>
For specific variants the specific three or one letter codes are used, including the codes Xaa and X to indicate any amino acid residue.<br>
SUBSTITUTIONS-:<br>
The substitution of Glutamic acid for glycine in position 195 is designated as:<br><br>
or the substitution of any amino acid residue add-for glycine in position 195 is designated as:<br><br>
The substitution of serine for any amino acid residue in position 170 would thus be designated:<br><br>
Such a notation is particular relevant in connection with modification(s) in homologous subtilases (vide infra). 170Ser is thus meant to comprise e.g. both a Lys170Ser modification in BASBPN and Arg170Ser modification in the subtitase according to the invention (cf. fig. 1).<br>
For a modification where the original amino acid(s) and/or substituted amino acid(s) may comprise more than one, but not all amino acid(s), the substitution of glycine, alanine, serine or threonine for arginine in position 170 would be indicated by:<br>
Arg170{Gly,Ala,Ser,Thr} or R170{GlAlSl1} to indicate the variants<br>
R170G, R170A, R170S, and R170T.<br>
•   DELETIONS: A deletion of glycine in position 195'wHI be indicated by:<br>
Correspondingly, the deletion of more than one amino acid residue, such as the deletion of glycine and leucine in positions 195 and 196 will be designated:1<br><br>
INSERTIONS:<br>
The insertion of an additional amino acid residue such as e.g. a lysine after G195 is indicated<br>
by:<br><br>
or when more than one amino acid residue is inserted, such as e.g. a Lys, and Ala after G195 this will be indicated as:<br><br>
In such cases, the inserted amino acid residue(s) are numbered by the addition of lower case letters to the position number of the amino acid residue preceding the inserted amino acid residue(s). In the above example, the sequences 194 to 196 would thus be:<br><br>
In cases where an amino acid residue identical to the existing amino acid residue is inserted it is clear that degeneracy in the nomenclature arises. If for example a glycine is inserted after the glycine in the above example this would be indicated by G195GG. The same actual change could just as well be indicated as A194AG for the change from<br>
194195196 BLSAVI      A - G - L<br>
to<br>
194.195 195a 196 Variant      A - G -  G - L<br>
194 194a 195 196<br>
Such instances will be apparent to the skilled person, and the indication G195GG and corresponding indications for this type of insertions are thus meant to comprise such<br>
equivalent degenerate indications.<br>
FILLING A GAP:<br>
Where a deletion in an enzyme exists in the reference comparison with the subtilisin BPN1<br>
sequence used for the numbering, an insertion in such a position is indicated as:<br><br>
for the insertion of an aspartic acid in position 36<br>
MULTIPLE MODIFICATIONS:<br>
Variants comprising multiple modifications are separated by pluses, e.g.:<br><br>
representing modifications in positions 170 and 195 substituting tyrosine and glutamic acid for arginine and glycine, respectively.<br>
Thus, Tyr167{Gly,AlalSer,Thr}+Arg170{Gly,Ala,Ser,Thr} designates the following variants:<br><br>
This nomenclature is particular relevant relating to modifications aimed at substituting, replacing, inserting or deleting amino acid residues having specific common properties, such as residues of positive charge (K, R, H), negative charge (D, E), or conservative amino acid modificatbn(s) of e.g. Tyr167{GlylAia,Ser,Thr}+Arg170{GlyIAla,SerIThr}f which signifies substituting a small amino acid for another small amino acid. See the section "Detailed description of the invention* for further details.<br>
Proteases<br>
Enzymes cleaving the amide linkages in protein substrates are classified as proteases, or (interchangeably) peptidases (see Walsh, 1979, Enzymatic Reaction Mechanisms. W.H. Freeman and Company, San Francisco, Chapter 3).<br>
Numbering of amino acid positions/residues<br>
If nothing-else is mentioned, the amino acid numbering used herein corresponds to that of the subtilase BPN' (BASBPN) sequence. For further description of the BPN1 sequence, see fig. 1 or Siezen et a/., Protein Engng. 4 (1991) 719-737.<br>
Serine proteases<br>
A serine protease is an enzyme which catalyzes the hydrolysis of peptide bonds, and in which there is an essential serine residue at the active site (White, Handier and Smith, 1973 'Principles of Biochemistry," Fifth Edition, McGraw-Hill Book Company, NY, pp. 271-272).<br>
The bacterial serine proteases have molecular weights in the 20,000 to 45,000 Dalton range. They are inhibited by diisopropyl fluorophosphate. They hydrolyze simple terminal esters and are similar in activity to eukaryotic chymotrypsin, also a serine protease. A more narrow term, alkaline protease, covering a sub-group, reflects the high pH optimum of some of the serine proteases, from pH 9.0 to 11.0 (for review, see Priest (1977) Bacteriological Rev. 41 711-753).<br>
Subtilases<br>
Siezen et al have proposed a sub-group of the serine proteases tentatively designated subtilases, Protein Engng, 4 (1991) 719-737 and Siezen et al. Protein Science 6 (1997) 501-523. They are defined by homology analysis of more than 170 amino acid sequences of serine proteases previously referred to as subtilisin-like proteases. A subtilisin was previously often defined as a serine protease produced by Gram-positive bacteria or fungi, and according to Siezen et al. now is a subgroup of the subtilases. A wide variety of subtiiases have been identified, and the amino acid sequence of a number of subtilases has been determined. For a more detailed description of such subtilases and their amino acid sequences reference is<br>
made to Siezen et a/. (1997).<br>
One subgroup of the subtilases, I-S1 or "true" subtilisins, comprises the "classical" subtilisins, such as subtilisin 168 (BSS168), subtilisin BPN! (BASBPN), subtilisin Carlsberg (BLSCAR)(ALCALASE®, NOVOZYMES A/S), and subtilisin DY (BSSDY).<br>
A further subgroup of the subtilases, I-S2 or high alkaline subtilisins, is recognized by Siezen et at. (supra). Subgroup I-S2 proteases are described as highly alkaline subtilisins and comprises enzymes such as subtilisin PB92 (BAALKP) (MAXACAL®, Gist-Brocades NV), subtilisin 309 (BLSAVI)(SAV!NASE® NOVOZYMES A/S), subtilisin 147 (BLS147) (ESPERASE®, NOVOZYMES A/S), and alkaline elastase YaB (BSEYAB).<br>
Parent subtilase<br>
The term "parent subtilase" describes a subtilase defined according to Siezen et al. (1991 and 1997). For further details, see description of "SUBTILASES" immediately above. A parent subtilase may also be a subtilase isolated from a natural source/wherein subsequent modifications have been.made while retaining the characteristic of a subtilase. Furthermore, a parent subtilase may also be a subtilase which has been prepared by the DNA shuffling technique, such as described by J.E. Ness et a/., Nature Biotechnology, 17, 893-896 (1999).<br>
Alternatively the term "parent subtilase" may be termed "wild type subtilase".<br>
Modifications) of a subtilase<br>
The term Hmodification(s)H used herein is defined to include chemical modification of a subtilase as well as genetic manipulation of the DNA encoding a subtilase. The modification(s) can be replacement(s) of the amino acid side chain(s), substitution(s), deletion(s) and/or insertions in or at the amino acid(s) of interest.<br>
Subtilase variant<br>
In the context of this invention, the term subtilase variant or mutated subtilase means a subtilase that has been produced by an organism which is expressing a mutant gene derived from a parent micro organism which possessed an original or parent gene and which produced a corresponding parent enzyme, the parent gene having been mutated in order to produce the mutant gene from which said mutated subtilase protease is produced when expressed in a suitable host. Analogously, the mutant gene may also be derived from a parent gene produced by DNA shuffling technique.<br>
Homologous subtilase sequences<br>
In the present context, the homology between two amino acid sequences is described by the<br>
parameter "identity".<br>
In order to determine the degree of identity between two subtilases, the GAP routine of the GCG package version 9.1 can be applied (infra) using the same settings. The output from the routine is besides the amino acid alignment the calculation of the "Percent Identity" between the two sequences.<br>
Based on this description it is routine for a person skilled in the art to identify suitable homologous subtilases and corresponding homologous active site loop regions, which can be modified according to the invention.<br>
Isolated polvnucleotide<br>
The term "isolated polynucleotide" as used herein refers to a polynucleotide, which has been isolated and purified and is thus in a form suitable for use within genetically engineered protein production systems. Such isolated molecules may be those that are separated from their natural environment and include cDNA and genomic clones as well as polynucleotides derived from DNA shuffling experiments or from site-directed autogenesis experiments. Isolated polynucleotides of the present invention are free of other genes with which they are ordinarily associated, but may include 51 and 31 untranslated regions such as promoters and terminators. The identification of associated regions will be evident to one of ordinary skill in the art (see for example Dynan and Tijan, Nature 316:774*78, 1985). The term "isolated nucleic acid sequence" may alternatively be termed "isolated DNA sequence", "cloned nucleic acid sequence" or "cloned DNA sequence".<br>
Isolated protein<br>
When applied to a protein, the term "isolated11 indicates that the.protein has been removed from its native environment.<br>
In a preferred form, the isolated protein is substantially free of other proteins, particularly other homologous proteins (i.e. "homologous impurities" (see below)).<br>
An isolated protein is more than 10% pure, preferably more than 20% pure, more preferably more than 30% pure, as determined by SDS-PAGE. Further, it is preferred to provide the protein in a highly purified form, i.e. more than 40% pure, more than 60% pure, more than 80% pure, more preferably more than 95% pure, and most preferably more than 99% pure, as<br>
determined by SDS-PAGE.<br>
The term 'isolated protein" may alternatively be termed "purified protein".<br>
Homologous impurities<br>
The term "homologous impurities" means any impurity (e.g. another polypeptide than the sub-tilase of the invention), which originate from the homologous cell where the subtilase of the invention is originally obtained from.<br>
Obtained from<br>
The term "obtained from" as used herein in connection with a specific microbial source means that the poiynucleotide and/or subtilase produced by the specific source, or by a cell in which a gene from the source has been inserted.<br>
Substrate<br>
The term "substrate" used in connection with a substrate for a protease should be interpreted in. its broadest form as comprising a compound containing at least one peptide bond susceptible to hydrolysis by a subtilisin protease.<br>
Product<br>
The term "product" used in connection with a product derived from a protease enzymatic reaction should in the context of the present invention be interpreted to include the products of a hydrolysis reaction involving a subtilase protease. A product may be the substrate in a subsequent hydrolysis reaction.<br>
Wash Performance<br>
In the present context, the term "wash performance" is used as an enzyme's ability to remove soil, in particular egg stains present on the object to the cleaned during e.g. wash or hard surface cleaning. See also the "Model Detergent Wash Performance Test11 in Example 2.<br>
% Removed Protein Film<br>
In the present context, the term "% Removed Protein Film" is used as an enzyme's ability to remove soil, in particular egg stains present on the object to the cleaned during automatic dish wash. Performance data originate from gravimetric measurements of the steel plates as clean, soiled and washed. The performance is calculated as:<br><br>
The data are fitted to the four-parameter logistic model that can be written as:<br><br>
Where F(z) is the response calculated from Yo as the intercept, Yo+Vma* being maximum response, C the enzyme dosage and ks being the half-saturation value. A is the steepness parameter that in a Michaelis-Menten model is equal 1, but here it is equal or different from one as we are allowing S-shaped curves to be fitted. Each curve fit is compared to the performance of the reference enzyme.  '<br>
For further details, see the "Mini scale Automatic Dish Washing (ADW mini wash)" in Example 4, herein.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figure 1 shows an alignment between subtilisin BPN1 (a) and the amino acid sequence of the novel s'ubtilases of the invention (b) and (c) using the GAP routine mentioned above.<br>
Figure 2 schematically depicts the set up of the ADW mini wash as described in example 4. Figure 3 shows the results of the ADW mini wash test method on egg yolk soil.<br>
Figure 4 shows the results of the AMSA test method (example 5) in combination with a commercially available.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
In a first interesting aspect of the present invention, the subtilase enzyme having improved wash performance on egg stains is an isolated subtilase which has more than 99.26% identity with the amino acid sequence shown as amino acids 1 to 269 of SEQ ID NO:2 (i.e. the mature subtilase). In an interesting embodiment of the invention, the subtilase has more than 99.26% or more than 99.63% identity with the amino acid sequence shown as amino acids 1 to 269 of SEQ ID NO:2 (hereinafter "homologous subtilases"). In another interesting embodiment of the invention, the isolated subtilase consists of the amino acid sequence shown as amino acids 1 to 269 of SEQ ID NO:2.<br>
In a further interesting aspect of the present invention," the subtilase enzyme having improved wash performance on egg stains is an isolated subtilase which has more than 97.40% identity with the amino acid sequence shown as amino acids 1 to 269 of SEQ ID NO:4 (i.e. the mature subtilase). In an interesting embodiment of the invention, the subtilase has more than 97.40%, or more than 97.77%, or more than 98.14%, or more than 98.51%, or more than 98.89%, or more than 99.26%, or more than 99.63% identity with the amino acid sequence shown as amino acids 1 to 269 of SEQ ID NO:4 (hereinafter "homologous subtilases"). In another interesting embodiment of the invention, the isolated subtilase consists of the amino acid sequence shown as amino acids 1 to 269 of SEQ ID NO:4.<br>
Alignments of sequences and calculation of identity scores can be done using a full Smith-Waterman alignment, useful for both protein and DNA alignments. The default scoring matrices BLOSUM50 and the identity matrix are used for protein and DNA alignments respectively. The penalty for the first residue in a gap is -12 for proteins and -16 for DNA, while the penalty for additional residues in a gap is -2 for proteins and -4 for DNA. Align is from the Fasta package version v3.1t11 (W. R. Pearson and D. J. Lipman (1988), "Improved Tools for Biological Sequence Analysis", PNAS 85:2444-2448, and W. R. Pearson (1990) "Rapid and Sensitive Sequence Comparison with FASTP and FASTA" Methods in Enzymology 183:63-98).<br>
By performing such alignments, the following identities (in percentage) between the amino acid sequences of the subtilase having the amino acid sequence of SEQ ID NO:2 and SEQ ID    . NO:4, and various known subtilases were found:<br><br>
1)BLAP {Bacillus lentus Alkaline Protease) has been described in US 5,352,604<br>
It is well-known in the art that a so-called conservative substitution of one amino acid residue to a similar amino acid residue is expected to produce only a minor change in the characteristic of the enzyme.<br>
Table I below lists groups of conservative amino acid substitutions.<br>
Table I<br>
Conservative amino acid substitutions<br>
Common Property	Amino Acid<br>
Basic (positive charge)     •	R = arginine<br>
K = lysine<br>
H = histidine<br>
Acidic (negative charge)	E = glutamic acid<br>
D = aspartic acid<br>
Polar	Q = glutamine   .<br>
N = asparagine<br>
Hydrophobic	L = leucirie<br>
I = isoleucine<br>
V	= valine<br>
M = methionines<br>
Aromatic	F = phenylaianine<br>
W = tryptophane<br>
Y	= tyrosine<br>
Small                                                        G = glyclne<br>
A = alanine S = serine T = threonine<br>
Therefore, in a further interesting embodiment of the invention, the subtilase having the amino acid sequence of SEQ ID NO:2 and SEQ ID NO:4 is combined with a substitution, deletion and/or insertion of one or more amino acid residues.<br>
Especially, combinations with other modifications known in the art to provide improved properties to the enzyme are envisaged. The art describes a number of subtilase variants with differ-<br>
ent improved properties and a number of those is mentioned in the "Background of the invention" section herein (vide supra).<br>
Such combinations comprise the positions: 222 (improves oxidation stability), 218 (improves thermal stability), substitutions in the Ca2+-binding sites stabilizing the enzyme, e.g. position 76, and many other apparent from the prior art.<br>
In further embodiments, a subtilase variant described herein may advantageously be combined with one or more modification(s) in any of the positions:<br>
27, 36, 56, 76, 87, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 120, 123, 159, 167, 170, 206, 218, 222, 224, 232, 235, 236, 245, 248, 252 and 274 (BPN! numbering).<br>
Specifically, the following BLSAVI, BLSUBL, BSKSMK, and BAALKP modifications are considered appropriate for combination:<br>
K27R, *36D, S56P, N76D, S87N, G97N, S101G, S103A, V104A, V104I, V104N, V104Y, H120D, N123S, G159D, Y167, R170, Q206E, N218S, M222S, M222A, T224S, A232V, K235L, Q236H, Q245R, N248D, N252K and T274A.<br>
Furthermore, variants comprising any of the modifications S101G+V104N, S87N+S101G+V104N, K27R+V104Y+N123S+T274A, N76D+S103A+V104I or N76D+V104A, or other combinations of the modifications K27R, N76D, S101G,-S103A, V104N, V104Y, V104I, V104A, N123S, G159D, A232V, Q236H, Q245R, N248D, N252K, T274A in combination with any one or more of the modification(s) mentioned above exhibit improved properties.<br>
A particular interesting variant is a variant, which in addition to modifications according to the invention contains the following substitutions: S101G+S103A+V104I+G159D+A232V+Q236H+Q245R+N248D+N252K.<br>
Moreover, subtilase variants of the main aspect(s) of the invention are preferably combined with one or more modification(s) in any of the positions 129,131 and 194, preferably as 129K, 131H and 194P modifications, and most preferably as P129K, P131H and A194P modifications. Any of those modification(s) are expected to provide a higher expression level of the subtilase variant in the production thereof.<br>
Furthermore, it is contemplated that insertion of at least one additional amino acid residue in<br>
the active site (b) loop region, corresponding to insertion of at least one additional amino acid residue from position 95 to position 103 (BASBPN numbering), will confer additional wash performance to the subtilase of the invention. In particular, it is preferred to insert at least one additional amino acid residue, such as one additional amino acid residue, in the following posi-tions: between positions 98 and 99, and between positions 99 and 100.<br>
Moreover, isolated subtilases, preferably in a purified form, having immunochemical identity or partial immunochemical identity to the subtilase having the amino acid sequence of SEQ ID NO:2 or SEQ ID NO:4 are also considered as being within the scope of the present invention. The immunochemical properties are determined by immunological cross-reaction identity tests by the well-known Ouchterlony double immunodiffusion procedure. Specifically, an antiserum containing polyclonal antibodies which are immunoreactive or bind to epitopes of the subtilase having the amino acid sequence of SEQ ID NO:2 or SEQ ID NO:4 are prepared by immunizing rabbits (or other rodents) according to the procedure described by Harboe and Ingild, In N.H. Axelsen, J. Krall, and B. Weeks, editors, A Manual of Quantitative Immunoelectrophore-sls, Blackwell Scientific Publications, 1973, Chapter 23, or Johnstone and Thorpe, Immuno-chemistry in Practice, Blackwell Scientific Publications, 1982 (more specifically pages 27-31). A subtilase having immunochemical identity is a subtilase, which reacts with the antiserum in an identical fashion such as total fusion of precipitates, identical precipitate morphology, and/or identical electrophoretic mobility using a specific immunochemical technique. Axelsen, Bock, and krall describe a further explanation of immunochemical identity in N.H. Axelsen, J. Krall, and B. Weeks, editors, A Manual of Quantitative Immunoelectrophoresis, Blackwell Scientific Publications, 1973, Chapter 10. A subtilase having partial immunochemical identity is a subtilase, which reacts with the antiserum in a partially identical fashion such as partial fusion of precipitates, partially identical precipitate morphology, and/or partially identical electrophoretic mobility using a specific immunochemical technique. Bock and Axelsen describe partial immunochemical identity in N.H. Axelsen, J. Krall, and B. Weeks, editors, A Manual of Quantitative Immunoelectrophoresis, Blackwell Scientific Publications, 1973, Chapter 11.<br>
The a ntibody m ay a Iso be a monoclonal antibody. Monoclonal antibodies may be prepared and used, e.g. according to the methods of E. Harlow and D. Lane, editors, 1988, Antibodies, A Laboratory Manual, Cold Spring Harbor Press, Cold Spring Harbor, New York.<br>
The present inventors have isolated the gene encoding the subtilase having the amino acid sequence shown in SEQ ID NO:2 and inserted it into E. coli MT173. The E co// MT173 strain harbouring the gene was deposited according to the Budapest Treaty on the International Recognition of the Deposits of Microorganisms for the Purpose of Patent Procedures on 8<br>
February 2000 at the Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 1 B, D-38124 Braunschweig, Germany, and designated the accession No. DSM 15575.<br>
The present inventors have isolated the gene encoding the subtilase having the amino acid sequence shown in SEQ ID NO:4 and inserted it into £ coli MT173. The £ coli MT173 strain harbouring the gene was deposited according to the Budapest Treaty on the International Recognition of the D eposits of M icroorganisms for the P urpose of Patent P rocedures on 8 February 2000 at the Deutsche Sammlung von- Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 1 B, D-38124 Braunschweig, Germany, and designated the accession No. DSM 15574.<br>
In an interesting embodiment of the invention, the subtilase has more than 99.26% or more than 99.63% identity with the subtilase encoded by the subtilase encoding part of the polynu-cleotide cloned into a plasmid fragment present in £ coli MT173 deposited under the accession No. DSM 15575.<br>
In another interesting embodiment of the invention, the subtilase has more than 97.40%, or more than 97.77%, or more than 98.14%, or more than 98.51%, or more than 98.89%, or more than 99.26%, or more than 99.63% identity with the subtilase encoded by the subtilase encoding part of the polynucleotide cloned into a plasmid fragment present in £. coli MT173 deposited under the accession No. DSM 15574.<br>
As mentioned above, the subtilase of the invention exhibits excellent wash performance on . egg stains. Therefore, in order to enable the skilled person - at an early stage of his development work - to select effective and preferred subtilases for this purpose, the present inventors have provided a number of suitable preliminary tests, which can easily be carried out by the skilled person in order to initially assess the performance of the subtilase in question.<br>
Thus, the "Model Detergent Wash Performance Test" disclosed in Example 2, herein, may be employed to assess the efficiency of the selected subtilase. In other words, the "Model Detergent Wash Performance Test" may be employed to assess the ability of a subtilase, when incorporated in a standard detergent composition, to remove egg stains from a textile surface as compared to a reference system (incorporated in the same model detergent system and tested under identical conditions). Using this test, the suitability of a selected subtilase to remove egg stains can be initially investigated, the rationale being that if a selected subtilase does not show a significant improvement in the test compared to the<br>
reference enzyme, it is normally not necessary to carry out further test experiments.<br>
Therefore, subtilases which are particular interesting for laundry wash purposes, are such subtilases which when tested in a model detergent composition comprising 6.2%       LAS (Nansa 80S) 2%'        Sodium salt of C16-Cia fatty acid    , 4%         Non-ionic surfactant (Plurafax LF404) 22%        Zeolite P 10.5%     Na2CO3 4%         Na2Si205<br>
2%         Carboxymethylcellulose (CMC) 6.8%      Acrylate liquid CP5 40%<br>
20%        Sodium perborate (empirical formula NaBO2.H2O2) 0.2%       EDTA 21%        Na2SO4<br>
Water (balance)<br>
as described in the "Model Detergent Wash Performance Test" herein, shows an improved wash performance on egg stains as compared to a reference enzyme tested under identical<br>
o<br>
conditions.<br>
The improvement in the wash performance may be quantified by employing the so-called "Performance factor11 defined in Example 2, herein.<br>
In a very interesting embodiment of the invention, the subtilase of the invention, when tested in the "Wash Performance Test", has a Performance Factor of at least 1, such as at least 1.5, e.g. at least 2, preferably at least 2.5, such as at least 3, e.g. at least 3.5, in particular at least 4, such as at least 4.5, e.g. at least 5.<br>
Evidently, it is preferred that the subtilase of the invention fulfils the above criteria on at least the stated lowest level, more preferably at the stated intermediate level and most preferably on the stated highest level.<br>
"The full scale Automatic Dish Washing (ADW) test" disclosed in Example 3 or the "Mini scale Automatic Dish Washing (ADW mini wash)" disclosed in Example 4 or th"e "Automated Mechanical Stress Assay (AMSA)" disclosed in Example 5, all herein, may be employed to assess the efficiency of the selected subtilase in Automatic Dish Washing. In other words, the<br>
tests may be employed to assess the ability of a subtilase, when incorporated in a detergent composition (commercial or standard), to remove egg stains from a hard surface as compared to a reference system (incorporated in the same detergent system and tested under identical conditions). Using this test, the suitability of a selected subtilase to remove egg stains can be initially investigated, the rationale being that if a selected subtilase does not show a significant improvement in the test compared to the reference enzyme, it is normally not necessary to carry out further test experiments.<br>
Therefore, subtilases which are particular interesting for automatic dish washing purposes are such subtilases which, when tested in a model detergent composition comprising:<br>
Sodium Tripolyphosphate	23.0%<br>
Sodium Citrate Dihydrate	.            22.3%<br>
Sodium Perborate Monohydrate	6.0%<br>
Tetraacetyl Ethylendiamine	2.0%<br>
Sodium Disiiicate (noncrystaline)	5.0%<br>
Linear Fatty Alcohol Ethoxylate	'                    2.0%<br>
(non-ionic surfactant, low foaming)<br>
Maleic acid/Aciylic acid copolymer	,                4.0%<br>
(Sodium salt, 50% active on Sodium Carbonate)<br>
Sodium Carbonate, anhydrous	add to 100%<br>
as described in the tests in Examples 3, 4 or 5 herein, shows an improved performance on egg stains as compared to a reference enzyme tested under identical conditions.<br>
The subtilase of the invention may be constructed by standard techniques for artificial creation of diversity, such as by DNA shuffling of different subtilase genes (see WO 95/22625 and J.E. Ness etal.t Nature Biotechnology, 17, 893-896 (1999)).<br>
Obviously, the subtilase of the invention may also be isolated from a natural source, i.e. the subtilase of the invention may, for example, be a bacterial subtilase, e.g. a gram positive bacterial subtilase such as a Bacjjlus polypeptide, e.g., a Bacillus alkalophilus, Bacillus amyloliquefaciens, Bacillus brevis, Bacillus circulans, Bacillus coagulans, Bacillus lautus, Bacillus lentus. Bacillus licheniformis, Bacillus megaterium, Bacillus stearothermophilus, Bacillus subtilis, or Bacillus thuringiensis subtilase; or a Streptomyces subtilase, e.g., a Streptomyces lividans or Streptomyces murinus subtilase; or a gram negative bacterial subtilase, e.g., an E. coli or a Pseudomonas sp. subtilase.<br>
The subtilase of the present invention may also be a fungal polypeptide, and more preferably a yeast subtilase such as a Candida, Kluyveromyces, Pichia, Saccharomyces, Schizosac-charomyces, or Yarrowia subtilase; or more preferably a filamentous fungal subtilase such as an Acremonium, Aspergillus, Aureobasidium, Cryptococcus, Filibasidium, Fusarium, Humicola, Magnaporthe, Mucor, Myceliophthora&gt; Neocallimastix, Neurospora, Paecilomyces, Penicillium, Piromyces, Schizophyllum, Talaromyces, Thermoascus, Thielavia, Tolypocladium, or Tricho-derma subtilase.<br>
In an interesting embodiment, the subtiiase is a Saccharomyces carlsbergensis, Saccharomyces cerevisiae, Saccharomyces diastaticus, Saccharomyces douglasii, Saccharomyces kluy-veri, Saccharomyces norbensis or Saccharomyces oviformis subtilase.<br>
In another interesting embodiment, the subtilase is an Aspergillus aculeatus, Aspergillus awamori, A spergi/lus foetidus, A spergillus j aponicus, A spergi/lus n idulans, A spergillus niger, Aspergillus oryzae, Fusarium bactridioides, Fusarium cerealis, Fusarium crookwellense, Fusa-rium culmorum, Fusarium graminearum, Fusarium graminum, Fusarium heterosporum, Fusarium negundi, Fusarium oxysporum, Fusarium reticulatum, Fusarium roseum, Fusarium sam-bucinum, Fusarium sarcochroum, Fusarium sporotrichioides, Fusarium sulphureum, Fusarium torulosum, Fusarium trichothecioides, Fusarium venenatum, Humicola insolens, Humicola la-nuginosa, Mucor miehei, Myceliophthora thermophila, Neurospora crassa, Penicillium pur-purogenumt Trichoderma harzianum, Trichoderma koningii, Trichoderma longibrachiatum, Trichoderma reesei, or Trichoderma viride subtilase.<br>
It will be understood that for the aforementioned species, the invention encompasses both the perfect and imperfect states, and other taxonomic equivalents, e.g., anamorphs, regardless of the species name by which they are known. Those skilled in the art will readily recognize the identity of appropriate equivalents.<br>
Strains of these species are readily accessible to the public in a number of culture collections, such as the American Type Culture Collection (ATCC), Deutsche Sammlung von Mikroorgan-ismen und Zellkulturen GmbH (DSM), Centraalbureau Voor Schimmelcuitures (CBS), and Agricultural Research Service Patent Culture Collection, Northern Regional Research Center (NRRL).<br>
Furthermore, such subtilases may be identified-and obtained from other sources including microorganisms isolated from nature (e.g., soil, composts, water, etc.) using the above-<br>
mentioned probes. Techniques for isolating microorganisms from natural habitats are well known in the art. Similarly screening a genomic or cDNA library of another microorganism may then derive the polynucleotide. Once a polynucleotide encoding a subtilase has been detected with the probe(s), the sequence may be isolated or cloned by utilizing techniques which are known to those of ordinary skill in the art (see, e.g., Sambrook et a/., 1989, supra).<br>
Many methods for cloning a subtilase of the invention and for introducing insertions into genes (e.g. subtilase genes) are well-known in the art, cf. the references cited in the "BACKGROUND OF THE INVENTION" section.<br>
In general standard procedures for cloning of genes and introducing insertions (random and/or site directed) into said genes may be used in order to obtain a subtilase enzyme of the invention. For further description of suitable techniques reference is made to Examples herein (vide infra) and (Sambrook et a!. (1989) Molecular cloning: A laboratory manual, Cold Spring Harbor lab., Cold Spring Harbor, NY; Ausubel, F. M. et al. (eds.) "Current protocols in Molecular Biology". John Wiley and Sons, 1995; Harwood, C. R., and Cutting, S. M. (eds.) "Molecular Biological Methods for Bacillus". John Wiley and Sons, 1990); and WO 96/34946.<br>
Further, a subtilase enzyme of the invention may be constructed by standard techniques for artificial creation of diversity, such as by DNA shuffling of different subtilase genes (WO 95/22625; Stemmer WPC, Nature 370:389-91 (1994)). DNA shuffling of e.g. the gene encoding Savinase® with one or more partial subtilase sequences identified in nature will, after subsequent screening for improved wash performance, provide subtilases according to the invention.<br>
POLYNUCLEOTIDES<br>
The present invention also relates to an isolated polynucleotide, which encodes a subtilase of<br>
the present invention.<br>
In one interesting embodiment, the polynucleotide has at least 88%, or at least 89%, at least 90%, at least 91%, at least 92%, at least 93%, at least 94%, at least 95%, at least 96%, at least 97%, at least 98%, or at least 99% identity with the polynucleotide shown as nucleotides 1 to 807 of SEQ ID NO:1; or at least 88%, or at least 89%, at least 90%, at least 91%, at least 92%, at least 93%, at least 94%, at least 95%, at least 96%, at least 97%, at least 98%, or at least 99% identity with the polynucleotide shown as nucleotides 1 to 807 of SEQ ID NO:3. In another interesting embodiment of the invention, the polynucleotide comprises the polynucleotide shown as nucleotides 1 to 807 of SEQ ID NO:1 or of SEQ ID NO;3, an allelic variant<br>
thereof, or a fragment thereof capable of encoding subtilases according to the invention. Ob-■  viously, the polynucleotide may consist of the polynucleotide shown as nucleotides 1 to 807 of SEQ ID NO:1 or SEQ ID NO:3.<br>
The p resent i nvention a Iso e ncompasses p olynucleotides t hat e ncode a p olypeptide having the amino acid sequence of SEQ ID NO:2, which differ from SEQ ID NO:2 by virtue of the degeneracy of the genetic code; the present invention further encompasses polynudeotides that encode a polypeptide having the amino acid sequence of SEQ ID NO:4, which differ from SEQ ID NO:4 by virtue of the degeneracy of the genetic code. The present invention also relates to subsequences of SEQ ID NO:1 that encode fragments of SEQ ID NO:2 that have proteolytic activity, and to subsequences of SEQ ID NO:3 that encode fragments of SEQ ID NO:4 that have proteolytic activity.<br>
A subsequence of SEQ ID NO:1 is a polynucleotide encompassed by nucleotides 1 to 807<br>
SEQ ID NO:1 except that one or more nucleotides from the 51 and/or 31 end have been de<br>
leted; a subsequence of SEQ ID NO:3 is a polynucleotide encompassed by nucleotides 1 to<br>
807 SEQ ID NO:3 except that one or more nucleotides from the 51 and/or 3' end have been<br>
deleted.	.	0<br>
The techniques used to isolate or clone a polynucleotide encoding a polypeptide are known in the art and include isolation from genomic DNA, preparation from c DNA, or a combination thereof. The cloning of the polynudeotides of the present invention from such genomic DNA can be effected, e.g., by using the well-known polymerase chain reaction (PCR) or antibody screening of expression libraries to detect cloned DNA fragments with shared structural features. See, e.g., Innis et a/., 1990, PCR: A Guide to Methods and Application, Academic Press, New York. Other nucleic acid amplification procedures such as ligase chain reaction (LCR), ligated activated transcription (LAT) and nucleic acid sequence-based amplification (NASBA) may be used.<br>
An. isolated polynucleotide can for example be obtained by standard doning procedures used in genetic engineering to relocate the polynucleotide from its natural location to a different site where it will be reproduced. The cloning procedures may involve excision and isolation of a desired nucleic acid fragment comprising the polynucleotide encoding the subtilase, insertion of the fragment into a vector molecule, and incorporation of the recombinant vector into a host ceil where multiple copies or clones of the polynucleotide will be replicated. The polynucleotide may be of genomic, cDNA, RNA, semi-synthetic, synthetic origin, or any combinations thereof.<br>
For purposes of the present invention, the degree of identity between two polynucleotides is determined is described above.<br>
Modification of a polynucleotide encoding a subtilase of the present invention may be necessary for the synthesis of subtilases substantially similar to the subtilase. The term "substantially similar1' to the subtilase refers to rion-naturally occurring forms of the subtilase. These subtilases may differ in some engineered way from the subtilase isolated from its native source, e.g., variants that differ in specific activity, thermostability, pH optimum, or the like. The variant sequence may be constructed on the basis of the polynucleotide presented as the polypeptide encoding part of SEQ ID NO:1, or on the basis of the polynucleotide presented as the polypeptide encoding part of SEQ ID NO:3, e.g., a subsequence thereof, and/or by introduction of nucleotide substitutions which do not give rise to another amino acid sequence of the subtilase encoded by the nucleic acid sequence, but which correspond to the codon usage of the host organism intended for production of the enzyme, or by introduction of nucleotide substitutions which may give rise to a different amino acid sequence. For a general description of nucleotide substitution see, e.g., Ford et a/., 1991, Protein Expression and Purification 2: 95-107.<br>
It will be apparent to those skilled in the art that such substitutions can be made outside the regions critical to the function of the molecule and still result in an active subtilase. Amino acid residues essential to the activity of the polypeptide encoded by the isolated polynucleotide of the invention, and therefore preferably not subject to substitution, may be identified according to procedures known in the art, such as site-directed mutagenesis or alanine-scanning mutagenesis (see, e.g., Cunningham and Wells, 1989, Science 244: 1081-1085). In the latter technique, mutations are introduced at every positively charged residue in the molecule, and the resultant mutant molecules are tested for proteolytic activity to identify amino acid residues that are critical to the activity of the molecule. Sites of substrate-enzyme interaction can also be determined by analysis of the three-dimensional structure as determined by such techniques as nuclear magnetic resonance analysis, crystallography or photoaffinity labelling (see, e.g., de Vos et a/., 1992, Science 255: 306-312; Smith et a/., 1992, Journal of Molecular Biology 224: 899-904; Wlodaver et al., 1992, FEBS Letters 309: 59-64).<br>
NUCLEIC ACID CONSTRUCTS<br>
The present invention also relates to nucleic acid constructs comprising a poiynucleotide of the present invention operably linked to one or more control sequences capable of directing the expression of the polypeptide in a suitable host cell.<br>
An isolated polynucleotide encoding a subtilase of the present invention may be manipulated in a variety of ways to provide for expression of the subtilase. Manipulation of the polynucleotide prior to its insertion into a vector may be desirable or necessary depending on the expression vector. The techniques for modifying polynucleotides utilizing recombinant DNA methods are well-known in the art.<br>
The control sequences include all components that are necessary or advantageous for the ex-pression of a subtilase of the present invention. Each control sequence may be native or foreign to the polynucleotide encoding the subtilase. Such control sequences include, but are not limited to, a leader, polyadenylation sequence, propeptide sequence, prompter, signal peptide sequence, and transcription terminator. At a minimum, the control sequences include a promoter, and transcriptional and translational stop signals. The control sequences may be provided with linkers for the purpose of introducing specific restriction sites facilitating ligation of the control sequences with the coding region of the polynucleotide encoding a subtilase. ■<br>
The control sequence may be an appropriate promoter sequence, a polynucleotide that is recognized by a host cell for expression of the nucleic acid sequence. The promoter sequence contains transcriptional control sequences that mediate the expression of the subtilase. The promoter may be any polynucleotide that shows transcriptional activity in the host cell of choice including mutant, truncated, and hybrid promoters, and may be obtained from genes encoding extracellular or intracellular subtilases either homologous or heterologous to the host cell.<br>
Examples of suitable promoters for directing the transcription of the nucleic acid constructs of the present invention, especially in.a bacterial host cell, are the promoters obtained from the £ coli lac operon, Streptomyces coelicolor agarase gene (dagA), Bacillus subtilis levansu-crase gene (sacB), Bacillus licheniformis alpha-amylase gene {amyL)t Bacillus stearothermo-philus maltogenic amylase gene {amyM), Bacillus amyloliquefaciens alpha-amylase gene {amyQ\ Bacillus licheniformis penicfilinase gene (penP), Bacillus subtilis xylA and xylB genes, and prokaryotic beta-lactamase gene (Villa-Kamaroff et a/M 1978, Proceedings of the National Academy of Sciences USA 75: 3727-3731), as well as the tac promoter (DeBoer et a/M 1983, Proceedings of the National Academy of Sciences USA 80: 21-25). Further promoters are described in "Useful proteins from recombinant bacteria" in Scientific American, 1980, 242: 74-94; and in Sambrook ef a/., 1989, supra.<br>
Examples of suitable promoters for directing the transcription of the nucleic acid constructs of the present invention in a filamentous fungal host cell are promoters obtained from the genes<br>
for Aspergillus oryzae TAKA amylase, Rhizomucor miehei aspartic proteinase, Aspergittus ni~ ger neutral alpha-amylase, Aspergillus niger acid stable alpha-amylase, Aspergillus niger or Aspergillus awamorl glucoamylase (glaA), Rhizomucor miehei lipase, Aspergillus oryzae alkaline protease, Aspergillus oryzae triose phosphate isomerase, Aspergillus nidulans acetami-dase, and Fusarium oxysporum trypsin-like protease (WO 96/00787), as well as the NA2-tpi promoter (a hybrid of the promoters from the genes for Aspergillus niger neutral alpha-amylase and Aspergillus oryzae triose phosphate isomerase), and mutant, truncated, and hybrid promoters thereof.<br>
In a yeast host, useful promoters are obtained from the genes for Saccharomyces cerevisiae enolase (ENO-1), Saccharomyces cerevisiae galactokinase (GAL1), Saccharomyces cerevisiae alcohol dehydrogenase/glyceraldehyde-3-phosphate dehydrogenase (ADH2/GAP), and Saccharomyces cerevisiae 3-phosphoglycerate kinase. Other useful promoters for yeast host cells are described by Romanes etal.t 1992, Yeast 8:423-488.<br>
The control sequence may also be a suitable transcription terminator sequence, a sequence recognized by a host cell to terminate transcription. The terminator sequence is operably linked to the 3' terminus of the polynucleotide encoding the subtilase. Any terminator that is functional in the host cell of choice may be used in the present invention.<br>
Preferred terminators for filamentous fungal host cells are obtained from the genes for Aspergillus oryzae TAKA amylase, Aspergillus niger glucoamylase, Aspergillus nidulans anthranilate synthase, Aspergillus niger alpha-glucosidase, and Fusarium oxysporum trypsin-like protease.<br>
Preferred terminators for yeast host cells are obtained from the genes for Saccharomyces cerevisiae enolase, Saccharomyces cerevisiae cytochrome C (CYC1), and Saccharomyces cerevisiae glyceraldehyde-3-phosphate dehydrogenase. Romanos et al., 1992, supra, describe other useful terminators for yeast host cells.<br>
The control sequence may also be a suitable leader sequence, a non-translated region of an mRNA that is important for translation by the host cell. The leader sequence is operably linked to the 5} terminus of the polynucleotide encoding the polypeptide. Any leader sequence that is functional in the host cell of choice may be used in the present invention.<br>
Preferred leaders for filamentous fungal host cells are obtained from the genes for Aspergillus oryzae TAKA amylase and Aspergillus nidulans triose phosphate isomerase.<br>
Suitable leaders for yeast host cells are obtained from the genes for Saccharomyces cere-visiae enolase (ENO-1), Saccharomyces cerevisiae 3-phosphogIycerate kinase, Saccharomyces cerevisiae alpha-factor, and Saccharomyces cerevisiae alcohol dehydro-genase/glyceraldehyde-3-phosphate dehydrogenase (ADH2/GAP).<br>
The control sequence may also be a polyadenylation sequence, a sequence operably linked to the 31 terminus of the polynucleotide and which, when transcribed, is recognized by the host cell as a signal to add polyadenosine residues to transcribed mRNA. Any polyadenylation sequence that is functional in the host cell of choice may be used in the present invention.<br>
Preferred polyadenylation sequences for filamentous fungal host cells are obtained from the genes for Aspergillus oryzae TAKA amylase, Aspergillus /?/ger glucoamylase, Aspergillus nidu-lans anthranilate synthase, Fusarium oxysporum trypsin-like protease, and Aspergillus niger alpha-glucosidase.<br>
Useful polyadenylation sequences for yeast host cells are described by Guo and Sherman, 1995, Molecular Cellular Biology 15: 5983-5990,<br>
The control sequence may also be a signal peptide coding region that codes for an amino acid sequence linked to the amino terminus of a subtilase and directs the encoded subtilase into the cell's secretory pathway. The 51 end of the coding sequence of the polynucleotide may inherently contain a signal peptide coding region naturally linked in translation reading frame with the segment of the coding region that encodes the secreted subtilase. Alternatively, the 5' end of the coding sequence may contain a signal peptide coding region that is foreign to the coding sequence. The foreign signal peptide coding region may be required where the coding sequence does not naturally contain a signal peptide coding region. Alternatively, the foreign signal peptide coding region may simply replace the natural signal peptide coding region in order to enhance secretion of the subtilase. However, any signal peptide coding region that directs the expressed subtilase into the secretory pathway of a host cell of choice may be used in the present invention.<br>
Effective signal peptide coding regions for bacterial host cells are the signal peptide coding regions obtained from the genes for Bacillus NCIB 11837 maltogenic amylase, Bacillus stearothermophilus alpha-amylase, Bacillus licheniformis subtilisin, Bacillus licheniformis beta-lactamase, Bacillus stearothermophilus neutral proteases {nprT, nprS, nprM), and Bacillus subtilis prsA. Further signal peptides are described by Simonen and Palva, 1993, Microbiological Reviews 57:109-137.<br>
Effective signal peptide ceding regions for filamentous fungal host cells are the signal peptide coding regions obtained from the genes for Aspergillus oryzae TAKA amylase, Aspergillus niger neutral amylase, Aspergillus niger glucoamylase, Rhizomucor miehei aspartic proteinase, Humicola insolens celiulase, and Humicola lanuginosa lipase.<br>
Useful signal peptides for yeast host cells are obtained from the genes for Saccharomyces cerevisiae alpha-factor and Saccharomyces cerevisiae invertase. Romanos et a/., 1992, supra, describe other useful signal peptide coding regions.<br>
The control sequence may also be a propeptide coding region that codes for an amino acid sequence positioned at the amino terminus of a subtilase. The resultant polypeptide is known as a proenzyme or propolypeptide (or a zymogen in some cases). A prppolypeptide is generally inactive and can be converted to a mature active polypeptide by catalytic or autocatalytic cleavage of the propeptide from the propolypeptide. The propeptide coding region may be obtained from the genes for Bacillus subtilis alkaline protease (aprf), Bacillus subtilis neutral protease (nprT), Saccharomyces cerevisiae alpha-factor, Rhizomucor miehei aspartic proteinase, and Myceliophthora thermophila laccase (WO 95/33836).<br>
Where both signal peptide and propeptide regions are present at the amino terminus of a subtilase, the propeptide region is positioned next to the amino terminus of a subtilase and the signal peptide region is positioned next to the amino terminus of the propeptide region.<br>
It may also be desirable to add regulatory "sequences that allow the regulation of the expression of the polypeptide relative to the growth of the host cell. Examples of regulatory systems are those which cause the expression of the gene to be turned on or off in response to a chemical or physical stimulus, including the presence of a regulatory compound. Regulatory systems in prokaryotic systems include the lac, tac, and trp operator systems. In yeast, the ADH2 system or GAL1 system may be used. In filamentous fungi, the TAKA alpha-amylase promoter, Aspergillus niger glucoamylase promoter, and Aspergillus oryzae glucoamylase promoter may be used as regulatory sequences. Other examples of regulatory sequences are those that allow for gene amplification. In eukaryotic systems, these include the dihydrofolate reductase gene that is amplified in the presence of methotrexate, and the metallothionein genes that are amplified with heavy metals. In these cases, the polynucleotide encoding the polypeptide would be operably linked with the regulatory sequence.<br>
EXPRESSION VECTORS<br>
The present invention also relates to a recombinant expression vector comprising the nucleic acid construct of the invention, a promoter, and transcriptional and translational stop signals.<br>
The recombinant expression vector comprising the nucleic acid construct encoding the enzyme of the invention may be any vector that may conveniently be subjected to recombinant DNA procedures.<br>
The choice of vector will often depend on the host cell into which it is to be introduced. Thus, the vector may be an autonomously replicating vector, i.e. a vector that exists as an extrachromosomal entity, the replication of which is independent of chromosomal replication, e.g. a plasmid. Alternatively, the vector may be one that on introduction into a host cell is integrated into the host cell genome in part or in its entirety and replicated together with the chromosome(s) into which it has been integrated.<br>
The vector is preferably an expression vector in which the DNA sequence encoding the enzyme of the invention is operably linked to additional segments required for transcription of the DNA. In general, the expression vector is derived from plasmid or viral DNA, or may contain elements of both. The term "operably linked" indicates that the segments are arranged so that they function in concert for their intended purposes, e.g. transcription initiates in a promoter and proceeds through the DNA sequence coding for the enzyme.<br>
The promoter may be any DNA sequence that shows transcriptional activity in the host cell of choice and may be derived from genes encoding proteins either homologous or heterologous to the host cell<br>
Examples of suitable promoters for use in bacterial host cells include the promoter of the Bacillus stearothermophilus maltogenic amylase gene, the Bacillus licheniformis alpha-amylase gene, the Bacillus amyloliquefaciens alpha-amylase gene, * the Bacillus subtilis alkaline protease gene, or the Bacillus pumilus xylosidase gene, or the phage Lambda PR or PL promoters or the E. coli lac, trg or tec promoters.<br>
The DNA sequence encoding the enzyme of the invention may also, if necessary, be operabty connected to a suitable terminator.<br>
The recombinant vector of the invention may further comprise a DNA sequence enabling the vector to replicate in the host cell in question,<br>
The vector may also comprise a selectable marker, e.g. a gene the product of which complements a defect in the host cell, or a gene encoding resistance to e.g. antibiotics like kanamycin, chloramphenicol, erythromycin, tetracyciine, spectinomycine, or the like, or resistance to heavy metals or herbicides.<br>
To direct an enzyme of the present invention into the secretory pathway of the host cells, a secretory signal sequence (also known as a leader sequence, prepro sequence or pre sequence) may be provided in the recombinant vector. The secretory signal sequence is joined to the-DNA sequence encoding the enzyme in the correct reading frame. Secretory signal sequences are commonly positioned 51 to the DNA sequence encoding the enzyme. The secretory signal sequence may be that normally associated with the enzyme or may be from a gene encoding another secreted protein.<br>
The procedures used to ligate the DNA sequences coding for the present enzyme, the promoter and optionally the terminator and/or secretory signal sequence, respectively, or to assemble these sequences by suitable PCR amplification schemes, and to insert them into suitable vectors containing the information necessary for replication or integration, are well known to persons skilled in the art (cf., for instance, Sambrook et al., op.cit).<br>
HOST CELL<br>
The present invention also relates to a recombinant host cell comprising the nucleic acid<br>
construct of the invention.<br>
The DNA sequence encoding the present enzyme introduced into the host cell may be either homologous or heterologous to the host in question, If homologous to the host cell, i.e. produced by the host ceil in nature, it will typically be operably connected to another promoter sequence or, if applicable, another secretory signal sequence and/or terminator sequence than in its natural environment. The term "homologous" is intended to include a DNA sequence encoding an enzyme native to the host organism in question. The term "heterologous" is intended to include a DNA sequence not expressed by the host cell in nature. Thus, the DNA sequence may be from another organism, or it may be a synthetic sequence.<br>
The host cell into which the DNA construct or the recombinant vector of the invention is introduced may be any cell that is capable of producing the present enzyme and includes bacteria, yeast, fungi and higher eukaryotic cells including plants.<br>
Examples of bacterial host cells which on cultivation are capable of producing the enzyme of the invention are gram-positive bacteria such as strains of Bacillus, such as strains of B. subtilis, B. licheniformis, B. lentus, B. brevis, B. stearothermophilus, B. alkalophllus, B. amyloliquefaciens, B. coagulans, B. circulans, B. lautus, S. megaterium or B. thuringiensis, in particular B. lentus, or strains of Streptomyces, such as S. lividans or S. murinus, or gram-negative bacteria such as Escherichia coli.<br>
The transformation of the bacteria may be effected by protoplast transformation, electroporation, conjugation, or by using competent cells in a manner known per se (cf. Sambrooket al., supra).<br>
When expressing the enzyme in bacteria such as £ coli, the enzyme may be retained in the cytoplasm, typically as insoluble granules (known as inclusion bodies), or may be directed to the periplasmic space by a bacterial secretion sequence. In the former case, the cells are lysed and the granules are recovered and denatured after which the enzyme is refolded by diluting the denaturing agent. In the latter case, the enzyme may be recovered from the periplasmic space by disrupting the cells, e.g. by sonication or osmotic shock, to release the contents of the periplasmic space and recovering the enzyme.<br>
When expressing the enzyme in gram-positive bacteria such as Bacillus or Streptomyces strains, the enzyme may be retained in the cytoplasm, or may be directed to the extracellular medium by a bacterial secretion sequence. In the latter case, the enzyme may be recovered from the medium as described below.<br>
In another embodiment of the invention, the fungal host cell is a yeast cell. "Yeast" as used herein includes ascosporogenous yeast (Endomycetales), basidiosporogenous yeast, and yeast belonging to the Fungi Imperfecti (Blastomycetes). Since the classification of yeast may change in the future, for the purposes of this invention, yeast shall be defined as described in Biology and Activities of Yeast{ Skinner, FA, Passmore, S.M., and Davenport, R.R., edsM Soc. App. Bacteriol. Symposium Series No. 9,1980).<br>
In a preferred embodiment, the yeast host cell is a Candida, Hansenula, Kluyveromyces, Pichia, Saccharomyces, Schizosaccharomyces, orYarrowia cell.<br>
In a most preferred embodiment, the yeast host cell is a Saccharomyces carlsbergensis, Sac-charomyces cerevisiae, Saccharomyces diastaticus, Saccharomyces douglasii, Saccharomyces kluyveri, Saccharomyces norbensis or Saccharomyces oviformis cell,   in another most<br>
preferred embodiment, the yeast host cell is a Kluyveromyces lactis cell. In another most preferred embodiment, the yeast host cell is a Yarrowia lipolytica cell.<br>
In another preferred embodiment, the fungal host cell is a filamentous fungal cell. "Filamentous fungi" include all filamentous forms of the subdivision Eumycota and Oomycota (as defined by Hawksworth et a/., 1995, supra). The filamentous fungi are characterized by a myce-lial wall composed of chitin, cellulose, glucan, chitosanf mannan, and other complex polysac-charides. Vegetative growth is by hyphal elongation and carbon catabolism is obligately aerobic. In contrast, vegetative growth by yeasts such as Saccharomyces cerevisiae is by budding of a unicellular thallus and carbon catabolism may be fermentative.<br>
In an even more preferred embodiment, the filamentous fungal host cell is a cell of a species of, but not limited to, Acremonium, Aspergillus, Fusarium, Humicola, Mucor, Myceliophthora, Neurospora, Penicillium, Thielavia, Tolypocladium,'or Trichoderma.<br>
In a most preferred embodiment, the filamentous fungal host cell is an Aspergillus awamori, Aspergillus foetidus, Aspergillus japonicus, Aspergillus nidulans, Aspergillus niger or Aspergillus oryzae cell. In another most preferred embodiment, the filamentous fungal host cell is a Fusarium bactridioldes, Fusarium "terealis, Fusarium crookwellense, Fusarium culmorum, Fusarium graminearum, Fusarium graminum, Fusarium heterosporum, Fusarium negundi, Fusarium oxysporum, Fusarium reticuiatum, Fusarium roseum, Fusarium sambucinum, Fusarium sarcochroum, Fusarium sporotrichioides, Fusarium sulphureum, Fusarium torulosum, Fusarium trichothecioides, or Fusarium venenatum cell. In an even most preferred embodiment, the filamentous fungal parent cell is a Fusarium venenatum (Nirenberg sp. nov.) cell. In another most preferred embodiment, the filamentous fungal host cell is a Humicola insolens, Humicola lanuginosa, Mucor miehei} Myceliophthora thermophila, Neurospora crassat Penicillium purpurogenumt Thielavia terrestris, Trichoderma harzianum, Trichoderma koningii, Trichoderma longibrachiatum, Trichoderma reesei, or Trichoderma viride cell.<br>
Fungal cells may be transformed by a process involving protoplast formation, transformation of the protoplasts, and regeneration of the cell wall in a manner known per se. Suitable procedures for transformation of Aspergillus host cells are described in EP 238 023 and Yelton et al.t 1984, Proceedings of the National Academy of Sciences USA 81: 1470-1474. Suitable methods for transforming Fusarium species are described by Malardier et a/., 1989, Gene 78; 147-156 and WO 96/00787. Yeast may be transformed using the procedures described by Becker and Guarente, In Abelson, J.N. and Simon, M.I., editors, Guide to Yeast Genetics and Molecular Biology, Methods in Enzymology, Volume 194, pp 182-187, Academic Press, Inc.,<br>
New York; ito etal., 1983, Journal of Bacteriology 153:163; and Hinnen et a/., 1978, Proceedings of the National Academy of Sciences USA 75:1920.<br>
METHOD OF PRODUCING A SUBTILASE OF THE INVENTION<br>
The present invention further relates to a method for producing a subtilase of the invention,<br>
the method comprising:<br>
a)	cultivating a recombinant host cell of the invention under conditions conducive to the<br>
production of the subtilase; and<br>
b)	recovering the subtilase.<br>
When an expression vector comprising a DNA sequence encoding the enzyme is transformed into a heterologous host cell, it is possible to enable heterologous recombinant production of the enzyme of the invention.<br>
Thereby it is possible to make a highly purified subtilase composition, characterized in being free from homologous impurities.<br>
In this context, homologous impurities mean any impurities (e.g. other polypeptides than the enzyme of the invention) that originate from the homologous cell where the enzyme of the invention .is originally obtained from.<br>
The medium used to culture the transformed host cells may be any conventional medium suitable for growing the host cells in question. The expressed subtilase may conveniently be secreted into the culture medium and may be recovered there from by well-known procedures including separating the cells from the medium by centrifugation or filtration, precipitating proteinaceous components of the medium by means of a salt such as ammonium sulfate, followed by chromatographic procedures such as ion exchange chromatography, affinity chromatography, or the like.<br>
USE OF A SUBTILASE OF THE INVENTION<br>
A subtilase enzyme of the invention may be used for a number of industrial applications, in particular within the detergent industry. Thus, the present invention also relates to a cleaning or detergent composition, preferably a laundry or dish washing composition, comprising the subtilase enzyme of the invention.   :1<br>
DETERGENT   COMPOSITIONS   COMPRISING   THE   SUBTILASE   ENZYME   OF   THE<br>
INVENTION:<br>
In general, cleaning and detergent compositions are well described in the art and reference is<br>
made to WO 96/34946; WO 97/07202; WO 95/30011 for further description of suitable<br>
cleaning and detergent compositions.<br>
Furthermore the examples herein demonstrate the improvements in performance on egg stains for the subtilases of the invention.<br>
- Detergent Compositions<br>
The enzyme of the invention may be added to and thus become a component of a cleaning or detergent composition.<br>
The detergent composition of the invention may for example be formulated as a hand or machine laundry detergent composition including a laundry additive composition suitable for pre-treatment of stained fabrics and a rinse added fabric softener composition, or be formulated as a detergent composition for use in general household hard surface cleaning operations, or be formulated for hand or machine dishwashing operations.<br>
In a specific aspect, the invention provides a detergent additive comprising the subtilase enzyme of the invention. The detergent additive as well as the detergent composition may comprise one or more other enzymes such as another protease, a lipase, a cutinase, an amylase, a carbohydrase, a cellulase, a pectinase, a mannanase, an arabinase, a galactanase, a xy-lanase, an oxidase, e.g., a laccase, and/or a peroxidase.<br>
In general the properties of the chosen enzyme(s) should be compatible with the selected detergent, (i.e. pH-optimum, compatibility with other enzymatic and non-enzymatic ingredients, etc.), and the enzyme(s) should be present in effective amounts.<br>
Proteases: Suitable proteases include those of animal, vegetable or microbial origin. Microbial origin is preferred. Chemically modified or protein engineered mutants are included. The protease may be a serine protease or a metallo protease, preferably an alkaline microbial protease or a trypsin-like protease. Examples of alkaline proteases are subtilisins, especially those derived from Bacillus, e.g., subtilisin Novo, subtilisin Carlsberg, subtilisin 309, subtilisin 147andsubtilisin 168 (described in WO 89/06279). Examples of trypsin-like proteases are trypsin (e.g. of porcine or bovine origin) and the Fusarium protease described in WO 89/06270 and WO 94/25583.<br>
Examples of useful proteases are the variants described in WO 92/19729, WO 98/20115, WO 98/20116, and WO 98/34946, especially the variants with substitutions in one or more of the following positions; 27, 36, 56, 76, 87, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 120, 123, 159, 167, 170, 206, 218, 222, 224, 232, 235, 236, 245, 248, 252 and 274 (BPN1 numbering).<br>
Preferred commercially available protease enzymes include -Alcalase™, Savinase™, Primase™, Duralase™, Esperase™, and Kannase™ (Novozymes A/S), Maxatase™, Maxacal™, Maxapem™, Properase™, Purafect™, Purafect OxP™, FN2™, and FN3™ (Genencor Internationa] Inc.).<br>
Upases: Suitable I ipases include those of bacterial or fungal origin. Chemically modified or protein engineered .mutants are included. Examples of useful Upases include lipases from Humicola (synonym Thermomyces), e.g. from H. lanuginosa (T. lanuginosus) as described in EP 258 068 and EP 305 216 or from H. insolens as described in WO 96/13580, a Pseudomonas lipase, e.g. from P. alcaligenes or P. pseudoalcaligenes (EP 218 272), P. cepacia (EP 331 376), P. stutzeri (GB 1,372,034), P. fluorescens, Pseudomonas sp. strain SD 705 (WO 95/06720 and WO 96/27002), P. wisconsinensis (WO 96/12012), a Bacillus lipase, e.g. from S. subtilis (Dartois et al. (1993), Biochemica et Biophysica Acta, 1131, 253-360), 8. stearothermophilus (JP 64/744992) or B. pumilus (WO 91/16422).<br>
Other examples are lipase variants such as those described in WO 92/05249, WO 94/01541, EP 407 225, EP 260 105, WO 95/35381, WO 96/00292, WO 95/30744, WO 94/25578, WO 95/14783, WO 95/22615, WO 97/04079 and WO 97/07202.<br>
Preferred commercially available lipase enzymes include Upolase™ and Lipolase Ultra™ (Novozymes A/S).<br>
Amvlases: Suitable amylases (a and/or p) include those of bacterial or fungal origin. Chemically modified or protein engineered mutants are included. Amylases include, for example, a-amylases obtained from Bacillus, e.g. a special strain of S. licheniformis, described in more detail in GB 1,296,839.<br>
Examples of useful amylases are the variants described in WO 94/02597, WO 94/18314, WO 96/23873, and WO 97/43424, especially the variants with substitutions in one or more of the following positions: 15, 23, 105, 106, 124, 128, 133, 154, 156, 181, 188, 190, 19/ 202, 208, 209, 243, 264, 304, 305, 391, 408, and 444.<br>
Commercially available amylases are Duramyl™, Termamyl™, Fungamyl™ and BAN™ (Novozymes A/S), Rapidase™ and Purastar™ (from Genencor International Inc.).<br>
Cellulases: Suitable cellulases include those of bacterial or fungal origin. Chemically modified or protein engineered mutants are included. Suitable cellulases include cellulases from the genera Bacillus, Pseudomonas, Humicola, Fusarium, Thielavia, Acremonium, e.g. the fungal cellulases produced from Humicola insolens, Myceliophthora thermophila and Fusarium oxysporum disclosed in US 4,435,307, US 5,648,263, US 5,691,178, US 5,776,757 and WO 89/09259.<br>
Especially suitable cellulases are the alkaline or neutral cellulases having colour care benefits. Examples of such cellulases are cellulases described in EP 0 495 257, EP 0 531 372, WO 96/11262, WO 96/29397, WO 98/08940. Other examples are cellulase variants such as those described in WO 94/07998, EP 0 531 315, US 5,457,046, US 5,686,593, US 5,763,254, WO 95/24471, WO 98/12307 and PCT/DK98/00299.<br>
Commercially available cellulases include Celluzyme™, and Carezyme™ (Novozymes A/S), Clazinase™, and Puradax HA™ (Genencor International Inc.), and KAC-500(B)™ (Kao Corporation).<br>
Peroxidases/Oxidases: Suitable perpxidases/oxidases include those of plant, bacterial or fungal origin. Chemically modified or protein engineered mutants are included. Examples of useful peroxidases include peroxidases from Coprinus, e.g. from C. cinereus, and variants thereof as those described in WO 93/24618, WO 95/10602, and WO 98/15257.<br>
The detergent enzyme(s) may be included in a detergent composition by adding separate additives containing one or more enzymes, or by adding a combined additive comprising all of these enzymes. A detergent additive of the invention, i.e. a separate additive or a combined additive, can be formulated e.g. as a granulate, a liquid, a slurry, etc. Preferred detergent additive formulations are granulates, in particular non-dusting granulates, liquids, in particular stabilized liquids, or slurries.<br>
Non-dusting granulates may be produced, e.g. as disclosed in US 4,106,991 and 4,661,452 and may optionally be coated by methods known in the art. Examples of waxy coating materials are polyethylene oxide) products (polyethylene glycol, PEG) with mean molar weights of 1000 to 20000; ethoxylated nonylphenols having from 16 to 50 ethylene oxide units; ethoxy-<br>
lated fatty alcohols in which the alcohol contains from 12 to 20 carbon atoms and in which there are 15 to 80 ethylene oxide enits; fatty alcohols; fatty acids; and mono-anddi-and triglycerides of fatty acids. Examples of film-forming coating materials suitable for application by fluid bed techniques are given in GB 1483591. Liquid enzyme preparations may, for instance, be stabilized by adding a polyol such as propylene glycol, a sugar or sugar alcohol, lactic acid or boric acid according to established methods. Protected enzymes may be prepared according to the method disclosed in EP 238,216.<br>
The detergent composition of the invention may be in any convenient form, e.g. a bar, a tablet, a powder, a granule, a paste or a liquid. A liquid detergent may be aqueous, typically containing up to 70% water and 0-30% organic solvent, or non-aqueous.<br>
The detergent composition typically comprises one or more surfactants, which may be non-ionic including semi-polar and/or anionic and/or catlonic and/or zwitterionic. The surfactants are typically present at a level of from 0.1% to 60% by weight.<br>
When included therein the detergent will usually contain from about 1% to about 40% of an anionic surfactant such as linear alkylbenzenesulfonate, alpha-olefinsulfonate, alky! sulfate (fatty alcohol sulfate), alcohol ethoxysulfate, secondary alkanesulfonate, alpha-sulfo fatty acid methyl ester, alkyl- or alkenylsuccinic acid or soap.<br>
When included therein the detergent will usually contain from about 0.2% to about 40% of a non-ionic surfactant such as alcohol ethoxylate, nonylphenol ethoxylate, alkylpolyglycoside, alkyldimethylamineoxide, ethoxylated fatty acid monoethanolamide, fatty acid monoethanola-mide, polyhydroxy alkyl fatty acid amide, or N-acyl N-alkyl derivatives of glucosamine ("glucamides").<br>
The detergent may contain 0-65% of a detergent builder or complexing agent such as zeolite, diphosphate, triphosphate, phosphonate, carbonate, citrate, nitrilotriacetic acid, ethylenedia-minetetraacetic acid, diethylenetriaminepentaacetic acid, alkyl- or alkenylsuccinic acid, soluble silicates or layered silicates (e.g. SKS-6 from Hoechst).<br>
The detergent may comprise one or more polymers. Examples are carboxymethylcellulose, poiy(vinylpyrrolidone), poly (ethylene glycol), poly(vinyl alcohol), poly(vinylpyridine-N-oxide),<br>
o<br>
poly(vinylimidazole), polycarboxylates such as polyacrylates, maleic/acrylic acid copolymers and lauryl methacrylate/acrylic acid copolymers.<br>
The detergent may contain a bleaching system that may comprise a H2O2 source such as perborate or percarbonate that may be combined with a peracid-forming bleach activator such as tetraacetylethylenediamine or nonanoyloxybenzenesulfonate. Alternatively, the bleaching system may comprise peroxyacids of e.g. the amide, imide, or sulfone type.<br>
The enzyme(s) of the detergent composition of the invention may be stabilized using conventional stabilizing agents, e.g. a polyol such as propylene glycol or glycerol, a sugar or sugar alcohol, lactic acid, boric acid, or a boric acid derivative, e.g. an aromatic borate ester, or a phenyl boronic acid derivative such as 4-formylphenyl boronic acid, and the composition may be formulated as described in e.g. WO 92/19709 and WO 92/19708.<br>
The detergent may also contain other conventional detergent ingredients such as e.g. fabric conditioners including clays, foam boosters, suds suppressors, anti-corrosion agents, soil-suspending agents, anti-soil redeposition agents, dyes, bactericides, optical brighteners, hy-drotropes, tarnish inhibitors, or perfumes.<br>
It is at present contemplated that in the detergent compositions any enzyme, in particular the enzyme of the invention, may be added in an amount corresponding to 0.01-100 mg of enzyme protein per litre of wash liquor, preferably 0.05-5 mg of enzyme protein per litre of wash liquor, in particular 0.1-1 mg of enzyr^e protein per litre of wash liquor.<br>
The enzyme of the invention may additionally be incorporated in the detergent formulations disclosed in WO 97/07202, which is hereby incorporated as reference.<br>
The invention is described in further detail in the following examples that are not in any way intended to limit the scope of the invention as claimed.<br>
In the detergent compositions, the abbreviated component identifications have the following meanings:<br>
LAS:	Sodium linear C12 alkyl benzene sulforiate<br>
TAS:	Sodium tallow alkyi sulphate<br>
XYAS:	Sodium C1X - C1Y alkyl sulfate<br>
SS:	Secondary soap surfactant of formula 2-butyl octanoic acid<br>
25EY:	A C12-C15 predominantly linear primary alcohol condensed with an average of<br>
Y	moles of ethylene oxide<br>
45EY:	A C14-C15 predominantly linear primary alcohol condensed with an average of<br>
Y	moles of ethylene oxide<br>
XYEZS:	CIX-G|Y sodium alkyl sulfate condensed with an average of Z moles of ethyl-<br>
ene oxide per mole<br>
Non-ionic:	C13-C16 mixed ethoxylated/propoxylated fatty alcohol with an average degree<br>
of ethoxylation of 3.8 and an average degree.of propoxylation of 4.5 sold under the trade name Plurafax LF404 by BASF GmbH<br>
CFAA:	C12-C14 alkyl N-methyl glucamide<br>
TFAA:	C16-C18 alkyl N-methyl glucamide<br>
Silicate:	Amorphous Sodium Silicate (SiO2:Na2O ratio = 2.0)<br>
NaSKS-6:	Crystalline layered silicate of formula 8-Na2Si205<br>
Carbonate:	Anhydrous sodium carbonate<br>
Phosphate:	Sodium tripolyphosphate<br>
MA/AA:	Copolymer of 1:4 maleic/acrylic acid, average molecular weight about 80,000<br>
Polyacrylate:      Polyacrylate homopolymer with an average molecular weight of         8,000<br>
sold under the trade name PA30 by BASF GmbH<br>
Zeolite A:	Hydrated Sodium Aluminosilicate of formula Na12(AIO2SiO2)i2.27H2O having<br>
a primary particle size in the range from 1 to 10 micrometers<br>
Citrate:	Tri-sodium citrate dihydrate<br>
Citric:	Citric Acid<br>
Perborate:	Anhydrous   sodium   perborate   monohydrate   bleach,   empirical  formula<br>
NaBO2.H2O2<br>
PB4:	Anhydrous sodium perborate tetrahydrate<br>
Percarbonate:	Anhydrous sodium percarbonate bleach of empirical formula 2Na2CO3.3H2O2<br>
TAED:	Tetra-acetyl ethylene diamine<br>
CMC:	Sodium carboxymethyl cellulose<br>
DETPMP:	Diethylene triamine penta (methylene phosphonic acid), marketed by Mon-<br>
santo under the trade name Dequest 2060<br>
PVP:	Polyvinylpyrrolidone polymer<br>
EDDS:	Ethylene diamine-N, N'-disuccinic acid, [S,S] isomer in the form of the so-<br>
dium salt<br>
Suds	25% paraffin wax, Mpt 50°C, 17% hydrophobic silica,<br>
Suppressor:       58% paraffin oil<br>
Granular Suds    12% Silicone/silica, 18% stearyl alcohol, 70% Suppressor:       starch in granular form<br>
Sulphate:	Anhydrous sodium sulphate<br>
HMWPEO:	High molecular weight polyethylene oxide<br>
TAE 25:	Tallow alcohol ethoxylate (25)<br>
Detergent Example I<br>
A granular fabric cleaning composition in accordance with the invention may be prepared as<br>
follows:<br>
Sodium linear C12 alky!	6.5<br>
benzene sulfonate<br>
Sodium sulphate	"       15.0<br>
Zeolite A	26.0<br>
Sodium nitrilotriacetate	5.0<br>
Enzyme	0.1<br>
PVP	0.5<br>
TAED	3.0      •<br>
Boric acid	4.0<br>
Perborate	18.0<br>
Phenol sulfonate	0.1<br>
Minors	up to 100%<br>
Detergent Example II<br>
A compact granular fabric cleaning composition (density 800 g/l) in accord with the invention<br>
may be prepared as follows:<br>
o	.        	  	      .  	..	.	<br>
45AS	8.0<br>
25E3S	2.0<br>
25E5	3.0<br>
25E3	3.0<br>
TFAA	2.5<br>
Zeolite A	17.0<br>
NaSKS-6	12.0<br>
Citric acid	3.0<br>
Carbonate	7.0<br>
MA/AA	5.0<br>
CMC	0.4<br>
Enzyme	0.1<br>
TAED	6.0<br>
Percarbonate	22.0<br>
EDDS	0.3<br>
Granular suds suppressor	3.5<br>
water/minors	Up to 100%<br>
Detergent Example III<br>
Granular fabric cleaning compositions in accordance with the invention that are especially useful in the laundering of coloured fabrics were prepared as follows:<br>
LAS	10.7<br>
TAS	2.4<br>
TFAA	-                         4.0<br>
45AS	3.1                     10.0<br>
45E7	4.0<br>
25E3S	3.0<br>
68E11                                       .	1.8                      -<br>
25E5	-                          8.0<br>
Citrate	15.0                   7.0.                .<br>
Carbonate	-                       10.0<br>
Citric acid	2.5                     3.0<br>
Zeolite A	32.1                   25.0<br>
Na-SKS-6                                            .       -	9.0<br>
MA/AA	5.0                     5.0<br>
DETPMP	0.2                      0.8<br>
Enzyme	0.10                   0.05<br>
Silicate	2.5<br>
Sulphate	5.2                    3.0<br>
PVP	0.5<br>
Poly (4-vinylpyridine)-N-	-                      0.2<br>
Oxide/copolymer of vinyl-<br>
imidazole and vinyl-<br>
pyrrolidone<br>
Perborate	1.0<br>
Phenol sulfonate	0.2<br>
Water/Minors	• Up to 100%<br>
Detergent Example IV<br>
Granular fabric cleaning compositions in accordance with the invention which provide "Softening through the wash" capability may be prepared as follows:<br>
45AS	-                        10.0<br>
LAS	7.6<br>
68AS	1.3<br>
45E7	4.0<br>
25E3	.	-	5.0<br>
Coco-aikyl-dimethyl hydroxy-	1.4	1.0<br>
ethyl ammonium chloride<br>
Citrate	5.0	3.0<br>
Na-SKS-6	-	11.0<br>
Zeolite A	15.0     -	15.0<br>
MA/AA	■	4.0	4.0<br>
DETPMP	0.4	0.4<br>
Perborate	15.0<br>
Percarbonate	-	15.0<br>
TAED	5.0	5.0<br>
Smectite clay	10.0	10.0	.<br>
HMWPEO	-	0.1<br>
Enzyme	0.10	0.05<br>
Silicate	.	3.0	5.0<br>
Carbonate	10.0   .	10.0<br>
Granular suds suppressor	1.0	4.0<br>
CMC	0.2	0.1<br>
Water/Minors	Up to 100%<br>
Detergent Example V<br>
Heavy duty liquid fabric cleaning compositions in accordance with the invention may be prepared as follows:<br>
LAS acid form	-	25.0<br>
Citric acid	5.0	2.0<br>
25AS acid form	8.0<br>
25AE2S acid form	3.0<br>
25AE7	8.0<br>
CFAA	•	5<br>
DETPMP    •	1.0	1.0<br>
Fatty acid	8<br>
Oleicacid	.	-	1.0<br>
Ethanol	4.0	6.0<br>
Propanediol	.	2.0	6.0<br>
Enzyme	0.10	0.05<br>
Coco-alkyl dimethyl	-	3.0<br>
Smectite clay	-	5.0<br>
PVP	2.0	-<br>
Water/Minors	Up to 100%<br>
Powder automatic dish	wash composition I<br><br>
Powder automatic dish	wash composition II<br><br><br><br><br><br><br><br><br><br><br><br>
XII: Automatic dishwashing compositions as described in I, II, HI, IV, VI and X, wherein perborate is replaced by percarbonate.<br>
XIII: Automatic dishwashing compositions as described in 1-VI, which additionally contain a manganese catalyst. The manganese catalyst may, e.g., be one of the compounds described in "Efficient manganese catalysts for low-temperature bleaching", Nature, (1994), 369, 637-639.<br>
MATERIALS AND METHODS<br>
PROTEOLYTIC ACTIVITY<br>
In the context of this invention, proteolytic activity is expressed in Kilo NOVO Protease Units (KNPU). The activity is determined relatively to an enzyme standard (SAVINASE®), and the determination is based on the digestion of a dimethyl casein (DMC) solution by the proteolytic enzyme at standard conditions, i.e. 50°C, pH 8.3, 9 min. reaction time, 3 min. measuring time. A folder AF 220/1 is available upon request to Novozymes A/S, Denmark, which folder is hereby included by reference.<br>
A GU is a Giycine Unit, defined as the proteolytic enzyme activity that under standard conditions during a 15 minutes1 incubation at 40°C, with N-acetyl casein as substrate produces an amount of NH2-group equivalent to 1 mmole of giycine.<br>
Enzyme activity can also be measured using the PNA assay, according to reaction with the soluble substrate succinyl-alanine-alanine-proline-phenyl-alanine-para-nitro-phenol, which is described in the Journal of American Oil Chemists Society, Rothgeb, T.M,, Goodlander, B.D., Garrison, P.H., and Smith, LA, (1988).<br>
EXAMPLE 1 - Construction and expression of subtilases according to the invention<br>
Example 1 covers both SEQ ID NO:2 and SEQ ID NO:4. It is to be understood that the term SEQ ID NO:2 at any time can be replaced by the term SEQ ID NO:4.<br>
The subtilisins having the amino acid sequence shown in SEQ ID NO:2 was located in vector pJRoC112, which is very similar to plasmid pKH400 (previously described in WO 98/41623). The plasmids are identical outside the regions encoding the mature subtilisin, i.e. the origin of replication; the cat gene conferring resistance towards chloramphenicol, the promoter directing the initiation of transcription of the subtilisin and the pre/pro regions from Savinase® are identical in these plasmids. Differences are only found within the part of the gene encoding the mature subtilisin.<br>
This plasmid replicates both in £ coli and in Bacillus subtilis. In Bacillus subtilis the subtilisin according to the invention was expressed from this plasmid. Fermentation and purification of<br>
the protease is described below.<br>
PKH400 was constructed from pJS3 (£ coli - B. subtilis shuttle vector containing a synthetic gene encoding for subtilase 309 (Savinase®) as described by J. Schi0dt et al. in Protein and Peptide Letters, 3, 39-44 (1996)) by introduction of two BamHI sites at positions 1841 and 3730.<br>
The mature gene has been subdoned into plasmid pzero-2 (Invitrogen, Groningen, The Netherlands). An approximately 1240 bp Pmel-BamHI fragment containing the complete mature region of the subtilase having the amino acid sequence shown in SEQ ID NO:2 was ligated with vector pZero-2 and digested with restriction endonucleases BamHI-EcoRV. The ligation mixture was transformed into competent £ coli cells. Transformants were analysed by PCR to verify the presence of the inserted fragment and the part of this fragment encoding the mature subtilisin was sequenced. The resulting plasmid, denoted pTVB364, was deposited on 10 February 2000 at DSMZ and was given the accession number DSM 13306.<br>
Fermentation<br>
Fermentations for the production of subtilase enzymes were performed at 30°C on a rotary shaking table (300 r.p.m.) in 500 ml baffled Erlenmeyer flasks containing 100 mi BPX medium for 5 days.<br>
Consequently, in order to make e.g. a 2 litre broth 20 Erlenmeyer flasks were fermented<br>
simultaneously.<br>
Media:<br>
BPX Medium Composition (per litre)<br>
Potato starch	100g<br>
Ground barley	50 g<br>
Soybean flour	20 g<br>
Na2HPO4x12H2O	9g<br>
Pluronic	0.1 g<br>
Sodium caseinate	10 g<br>
The starch in the medium was liquefied with a-amylase and the medium was sterilized by heating at 12O°C for 45 minutes. After sterilization the pH of the medium was adjusted to 9 by addition of NaHCO3 to 0.1 M.<br>
Purification<br>
This procedure relates to purification of a 2 litre scale fermentation for the production of the<br>
subtilases of the invention in a Bacillus host cell.<br>
Approximately 1.6 litres of fermentation broth was centrifuged at 5000 rpm for 35 minutes in 1 litre beakers. The supernatants were adjusted to pH 6.5 using 10% acetic acid and filtered on Seitz Supra S100 filter plates.<br>
The filtrates were concentrated to approximately 400 ml using an Amicon CH2A UF unit equipped with an Amicon S1Y10 UF cartridge. The UF concentrate was centrifuged and filtered at room temperature prior to absorption on a Bacitracin affinity column at pH 7. The subtilase was eluted from the Bacitracin column at room temperature using 25% 2-propanol and 1 M sodium chloride in a buffer solution with 0.01 dimethylglutaric acid, 0.1 M boric acid and 0.002 M calcium chloride adjusted to pH 7.<br>
The fractions with protease activity from the Bacitracin purification step were combined and applied to a 750 ml Sephadex G25 column (5 cm diameter) equilibrated with a buffer containing 0.01 dimethylglutaric acid, 0.2 M boric acid and 0.002 m calcium chloride adjusted to pH 6.5.<br>
Fractions with proteolytic activity from the Sephadex G25 column were combined and applied to a 150 ml CM Sepharose CL 6B cation exchange column (5 cm diameter) equilibrated with a buffer containing 0.01 M dimethylgiutaric acid, 0.2 M boric acid, and 0.002 M calcium chloride<br>
adjusted to pH 6.5.<br>
The protease was eluted using a linear gradient of 0-0.1 M sodium chloride in 2 litres of the same buffer.<br>
In a final purification step, protease-containing fractions from the CM Sepharose column were combined and concentrated in an Amicon ultra filtration cell equipped with a GR81PP membrane (from the Danish Sugar Factories Inc.).<br>
By using the techniques mentioned above for the construction and fermentation, and the above isolation procedure, the novel subtilase having the amino acid sequence set forth in SEQ ID NO:2 was produced and isolated.<br>
EXAMPLE 2 -The "Model Detergent Wash Performance Test"<br>
In order to asses the wash performance of subtilases in a standard detergent composition, standard washing experiments may be performed using the below experimental conditions:<br>
Detergent:	Model detergent<br>
Detergent dosage	4.0 g/l<br>
pH	10,1<br>
Wash time	20 min<br>
Temperature:	30°C<br>
Water hardness:	15°dH<br>
Enzyme concentration:	10 nm (in the detergent solution)<br>
Test system:	10 ml beakers with a stirring rod<br>
Textile/volume:	5 textile pieces (0 2.5 cm)/50 ml<br>
detergent solution<br>
Test material:	WFK10N (egg stains)<br>
The composition of the model detergent is as follows;<br>
6.2%	LAS (Nansa 80S)<br>
2%	Sodium salt of C10-CiB fatty acid<br>
4%	Non-ionic surfactant (Plurafax LF404)<br>
22%	Zeolite P<br>
10.5%	Na2CO3<br>
■    4%	Na2Si205<br>
2%	Carboxymethylcellulose (CMC)<br>
6.8%	Acrylate liquid CP5 40%<br>
20%	Sodium perborate (empirical formula NaBO2.H2O2)<br>
0.2%       EDTA 21%        Na2SO4<br>
Water (balance)<br>
pH of the detergent solution is adjusted to 10.1 by addition of HCI or NaOH. Water hardness is adjusted to 15cdH by addition of CaCI2 and MgCI2 (Ca2+:Mg24 = 4:1) to the test system. After washing the textile pieces are flushed in tap water and air-dried.<br>
Measurement of the reflectance (RSUbtiiase) on the test material is performed at 460 nm using a Macbeth ColorEye 7000 photometer (Macbeth, Division of Kollmorgen Instruments Corporation, Germany). The measurements are performed in accordance with the manufacturer's protocol.<br>
in order to determine a blank value, a similar wash experiment is performed without addition of enzyme. The subsequent measurement of the reflectance (RbianO is performed as described right above.<br>
A reference experiment is then performed as described above, wherein the wash performance of Savinase® is tested. The subsequent measurement of the reflectance (RsaVinas4) is performed as described right above.<br>
The wash performance is evaluated by means of the Performance Factor (P) which is defined in accordance with the below formula:<br><br>
EXAMPLE 3 - The full scale Automatic Dish Washing (ADW) test<br>
The performance of the subtilase of the invention in full scale ADW is tested in a household<br>
dish wash composition using standard conditions. The soil used is an egg/milk mixture coated<br>
on a steel piate. Further, a ballast soil containing various foodstuffs is added.	. ■<br>
Example:<br>
Detergent:	Commercial or model detergent.<br>
Detergent dosage	5.0 g/l<br>
pH	As is.<br>
Water hardness:	As is.<br>
Temperature:	50°C or 55°C ■<br>
Enzyme concentration:	10 nM to 230 nM, based on the total volume of wash water<br>
in the machine.<br>
Test method;	Egg/milk or egg yolk soiling on steel plates as described<br>
below.<br>
Machine:	Bosch or other commercially available.<br>
Wash program:<br><br>
1)	Heating of tap water takes place during the indicated time interval.<br>
2)	Final temperature upon heating of tap water.<br>
Egg/milk soiling for full scale ADW test<br>
Materials:<br>
220 ml full cream milk 15 eggs, medium size Steel plates, diameter 18 cm<br>
The dish wash composition is heated at 85°C for 5 minutes in a microwave oven in order to inactivate enzyme activity in the composition.<br>
Soiling of steel plates;<br>
220 ml full cream milk is mixed with 15 raw eggs in a Braun UK 20 kitchen, machine for 2<br>
minutes. After sieving, stainless steel plates are soiled in the mixture by immersion.<br>
The plates are dried overnight at room temperature in an upright position. The dried plates are then heated at 120°C for 45 minutes in order to denature the proteins on the surface.<br>
Egg yolk soiling for full scale ADW test.<br>
Materials;<br>
3 cJL pasteurized egg yolk.<br>
Steel plates, diameter 18 cm<br>
The dish wash composition is heated at 85°C for 5 minutes in a microwave oven in order to inactivate enzyme activity in the composition.<br>
Soiling of steel plates:<br>
The steel plates are weighed on a balance giving 3 decimals.<br>
Approx. 3 dL pasteurized egg yolk is mixed thoroughly and sieved through a kitchen screen.<br>
The egg yolk broth is rolled onto the plates in a thin layer, e.g. using a paint roller. This is done<br>
twice (without drying in between and with the roller dipped in egg yolk also the second time).<br>
The resulting layer of egg yolk should be around 1 g.<br>
The plates are left to dry for minimum 4 hours at room temperature.<br>
The soiled plates and the racks are then lowered into boiling demineralised water for precisely<br>
30 seconds.<br>
The plates are left to dry for 30 minutes at room temperature.<br>
After drying at room temperature the plates are dried in an oven at 80°C for 30 minutes.<br>
The plates are left to cool at room temperature for 30-60 minutes after which they are weighed<br>
again.<br>
Upon washing and drying at room temperature the plates are dried in the oven at 80°C for 30<br>
min.<br>
Again after cooling at room temperature for 30-60 minutes the plates are weighed.<br>
ADW experiments<br>
For each experiment, 10 soiled plates are washed in accordance with conditions listed above. In addition to the soiled plates, the machine is filled up with 10 porcelain plates, 4 glasses, 4 cups and 16 pieces of cutlery.<br>
Furthermore, 50 g of ballast slurry is added to the machine. The composition of the slurry is as<br>
follows:<br><br><br>
1.	Margarine, lard and deep-fry oil are melted at low temperature. Afterwards sieved<br>
gravy powder is added - under good stirring - and is cooled down to 40 °C.<br>
2.	Rapeseed oil and egg are mixed.<br>
3.	Ketchup and mustard are added into the oil/egg mass followed by 5 minutes mixing.<br>
4.	The under 1) produced fat/gravy (cooled) is slowly added to the mixture produced in 3)<br>
and mixed for further 5 minutes.<br>
5.	Double cream and full-cream milk are added the mixture and mixed for 5 minutes.<br>
6.	The last flours and powders (step 5 in the table) is added. The ballast slurry is mixed to<br>
a smooth mass.<br>
7.	The Ballast slurry is weighed out in portions of 50 g.<br>
Measurements and calculations for egg/milk<br>
The light reflection values (R-values) are measured at six different locations on the plates<br>
using a Minolta Chroma Meter (Type: CR-300). Measurements are made on clean plates<br>
(Rciean). on soiled plates after heating (R^ied) and on plates after wash (Rafterwash).<br>
The removed protein film (%RPF) is calculated according to the below formula:<br>
%RPF = 1 00% X (Rafter wash " RsoHedV(Rclean " Rsoited)<br>
Measurements and calculations for egg yolk<br>
Performance data originate from gravimetric measurements of the steel plates as clean, soiled and washed. The performance is calculated as:<br><br>
Data analysis<br>
%RPF is fitted as a function of mg enzyme protein added.<br>
The data are fitted by means of a four-parameter logistic model that can be written as:<br><br>
Where F(z) is the response calculated from Yo as the intercept, Y0+Vmax being maximum response, C the enzyme dosage and ks being the half-saturation value. A is the steepness parameter that in a Michaelis-Menten model is equal 1, but.here it is equal or different from one as we are allowing S-shaped curves to be fitted.<br>
Each curve fit is compared to the performance of the reference enzyme. EXAMPLE 4 - Mini scale Automatic Dish Washing (ADW mini wash)<br>
Description of ADW min?-wash<br>
The mini-wash is developed as a computerized robot. Each robot carries a frame with 8 racks. Each rack contains 6 pieces of 35 x 45 mm steel plates to be soiled and washed. If needed the 6 positions can represent the six points in the dosage range, e.g. 0 - 20 -40 - 75 - 100 -160 nM enzyme as shown below. One wash unit consists of two plates, egg yolk and egg/milk soiled respectively, and a thermostated vessel with 150 ml wash float. A robot operates on 24 vessels at a time<br>
As mentioned two racks are needed for each enzyme to be tested, one being soiled with egg yolk the other egg/milk. The reference enzyme is included at each run and each enzyme dosage is repeated twice.<br>
The setup is schematically depicted in figure 2.<br>
The egg/milk soil is prepared as follows:<br>
10 eggs + 167 ml of milk are blended 2 min at low speed in a food processor. The mixture is sieved through a disposable cloth before use. Racks are mounted with mini plates and dipped into the sieved soil. The plates are placed upright on an absorbing table cover. They are left to<br>
dry for 4 hours or until next day where they are denatured by heat treatment in hot air ovens at 120 °C for 35 minutes.<br>
The egg yolk soil is prepared as follows:<br>
20 yolks -400 ml pasteurized caterfhg yolk (-420 g) + 167 ml deionised water is blended 2 minutes at lowest speed in a food processor then sieved through a disposable cloth before use. Racks are mounted with mini plates and dipped into the sieved soil. The plates are placed " upright on an absorbing table cover. They are left to dry for 4 hours or until next day where they are denatured 30 sec. in boiling water. After denaturing the plates are heat treated in hot air ovens at 80°C for 30 minutes.<br>
Detergents<br>
Detergents for wash performance tests of the shuffled proteases of the invention can be obtained by purchasing fully formulated commercial detergents at the market and subsequently inactivate the enzymatic components by heat treatment (5 minutes at 85°C in aqueous solution). Moreover a commercial detergent base without enzymes can be purchased directly from the manufacturer. Further a suitable model detergent can be purchased and used for wash performance tests.<br>
The proteases may be tested in a model detergent composition comprising<br>
Sodium Tripoiyphosphat     o	23.0%<br>
Sodium Citrate Dihydrate	22,3%<br>
Sodium Perborate Monohydrate	6.0%<br>
Tetraacetyl Ethylendiamine	2.0%<br>
Sodium Disilicate (noncrystaline)	'      5.0%<br>
Linear Fatty Alcohol Ethoxylate	2.0%<br>
(non-ionic surfactant low foaming)<br>
Maieic add/Acrylic acid copolymer	4.0%<br>
(Sodium salt, 50% active on Sodium Carbonate)<br>
Sodium Carbonate, anhydrous add to	100%<br>
The wash floats are prepared by mixing CaCI2, MgSO4 and NaHCO3 deionised water to make up the various water hardness values. Detergent is added: 18-25 g/4L<br>
Evaluation:<br>
Performance data originate from gravimetric measurements of the steel plates as clean, soiled<br>
and washed. The performance is calculated as:<br>
% Removed Protein Film (%RPF) = <br>
Data analysis<br>
%RPF is fitted as a function of mg enzyme protein added.<br>
The data are fitted to the four-parameter logistic model that can be written as:<br><br>
Where F(z) is the response calculated from Yo as the intercept, Y0+Vmax being maximum response, C the enzyme dosage and k5 being the half-saturation value, h is the steepness parameter that in a Michaelis-Menten model is equal 1, but here it is equal or different from one as we are allowing S-shaped curves to be fitted. Each curve fit is compared to the performance of the reference enzyme.<br>
Using the above test method on egg yolk soil the results shown in figure 3 were obtained (%RPF as a function of mg enzyme protein). As it appears, the subtilases according to the invention exhibits improved wash performance on egg stains in comparison to SEQ ID NO:6 (named PH004).<br>
EXAMPLE 5 - Automatic Mechanical Stress Assay (AMSA).<br>
Description of AMSA-test method:<br>
Washing experiments are performed in order to asses the wash performance of selected shuffled protease variants in detergent compositions. The proteases of the present application are tested using the Automatic Mechanical Stress Assay (AMSA). With the AMSA, the wash performance of a large quantity of small volume enzyme-detergent solutions can be examined. The AMSA plate has a number of slots for test solutions and a lid firmly squeezing the textile swatch to be washed against all the slot openings. During the washing time, the plate, test solutions, textile and lid are vigorously shaken to bring the test solution in contact with the textile and apply mechanical stress in a regular, periodic oscillating manner. For further description see WO 02/42740 especially the paragraph "Special method embodiments" at page 23-24.<br>
The experiment was conducted under the experimental conditions specified below:<br><br>
Water hardness was adjusted to 25°dH by addition of CaCI2r MgCI2, and NaHCO3.(Ca2+:Mg2+ = 4:1) to the test system. After washing the textile pieces were flushed in tap water and dried.<br>
The performance of the enzyme variant is measured as the brightness of the colour of the textile samples washed with that specific protease. Brightness can also be expressed as the intensity of the light reflected from the textile sample when illuminated with white light When the textile is stained the intensity of the reflected light is lower, than that of a dean textile. Therefore the intensity of the reflected light can be used to measure wash performance of a shuffled protease.<br>
Colour measurements are made with a professional flatbed scanner (PFU DL2400pro, obtainable from: J.M. Thomsen, Dorfgade 2, Dorf, Dronninglund, DK-9330), which is used to capture an image of the washed textile samples. The scans are made with a resolution of 200 dpi and with an output colour dept of 24 bits. In order to get accurate results, the scanner is frequently calibrated with a Kodak reflective IT8 target.<br>
To extract a value for the light intensity from the scanned images, a special designed software application is used (Novozymes Color Vector Analyzer). The program retrieves the 24 bit pixel values from the image and converts them into values for red, green and blue (RGB). The intensity value (Int) is calculated by adding the RGB values together as vectors and then taking the length of the resulting vector:<br><br>
Detergents	•  -<br>
Detergents for wash performance tests of the shuffled proteases of the invention can be obtained by purchasing fully formulated commercial detergents at the market and subsequently inactivate the enzymatic components by heat treatment (5 minutes at 85°C in aqueous solution). Moreover a commercial detergent base without enzymes can be purchased directly from the manufacturer. Further a suitable model detergent can be purchased and used for wash performance tests.<br>
The proteases may be tested in a model detergent composition comprising<br>
Sodium Tripoiyphosphate	23.0%<br>
Sodium Citrate Dihydrate	22.3%<br>
Sodium Perborate Monohydrate	6.0%<br>
Tetraacetyl Ethylendiamine	2.0%<br>
Sodium Disilicate (noncrystaline)	5.0%<br>
Linear Fatty Alcohol Ethoxylate	2.0%<br>
(non-ionic surfactant, low foaming)<br>
Maleic acid/Acrylic acid copolymer	• 4.0%<br>
(Sodium salt, 50% active on Sodium Carbonate)<br>
Sodium Carbonate, anhydrous add to	100%<br>
Textiles:<br>
Standard textile pieces are obtained from wfk-Cleaning Technology Research Institute, Christenfeld 10, D-41379 Bruggen-Bracht,'Germany.- Especially type wfk10N (cotton textile stained with egg/pigment), wfkiOeggEG (cotton textile stained with egg yolk). Denaturation of wfkiON occurs in an autoclave.<br>
Using the above test method in combination with a commercially available detergent the results shown in figure 4 were obtained. As it appears, the subtilases according to the invention exhibits improved wash performance on egg stains in comparison to the protease with SEQ ID NO:6 (named PH004).<br>
DEPOSIT OF BIOLOGICAL MATERIAL<br>
The following biological material has been deposited under the terms of the Budapest Treaty with the Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 1 B, D-38124 Braunschweig, Germany, and given the following accession numbers:<br>
S<br><br><br><br>
CLAIMS<br>
1.	A subtilase enzyme being selected from the group consisting of<br>
a)	a subtilase having an amino acid sequence that has at more than 99.26% identity with the<br>
amino add sequence shown ae amino acids 1 to 269 of SEQ ID NO:2; or<br>
b)	a subtilase encoded by the subtilase encoding part of the polynucleotide cloned into a<br>
plasmid fragment present in Escherichia coli MT173 DSM 15575, or a variant thereof hav<br>
ing more than 99.26% identity to said subtilase; or<br>
c)	a subtilase having an amino acid sequence that has more than 97.40% identity with the<br>
amino acid sequence shown as amino acids 1 to 269 of SEQ ID NO:4; or<br>
d)	a subtilase encoded by the subtilase encoding part of the poiynucleotide cloned into a<br>
plasmid fragment present in Escherichia coli MT173 DSM 15574, or a variant thereof hav<br>
ing more than 97.40% identity to said subtilase.<br>
2.	A subtilase according to claim 1, having<br>
I)	an amino acid sequence which has more than 99.26%, or more than 99.63% identity with<br>
the amino acid sequence described in 1a) or 1b); or<br>
II)	an amino acid sequence which has more than 97.40%, or more than 97.77%, or more than<br>
98.14%, or more than 98.51%, or more than 98.89%, or more than 99.26%f or more than<br>
99.63% identity with the amino acid sequence described in 1c) or 1d).<br><br>
3.	A subtilase according to claim 2, which consists of either the amino acid sequence shown<br>
as amino acids 1 to 269 of SEQ ID NO:2, or of the amino acid sequence shown as amino<br>
acids 1 to 269 of SEQ ID NO:4.<br>
4.	A subtilase according to claim 1 ( having<br><br>
I)	an amino acid sequence which has more than 99.26%, or more than 99.63% identity with<br>
the subtilase encoded by the subtilase encoding part of the polynucleotide cloned into a<br>
plasmid fragment present in Escherichia coli MT173 DSM 15575; or<br>
II)	an amino acid sequence which has more than 97.40%, or more than 97.77%, or more than<br>
98.14%, or more than 98.51%, dT more than 98.89%, or more than 99.26%F or more than<br>
99.63%   identity with  the  subtilase  encoded  by  the  subtilase  encoding  part  of the<br>
polynucleotide cloned into a plasmid fragment present in Escherichia coli MT173 DSM 15574.<br>
5.	A subtilase according to claim 4, which consists of<br>
I) the subtilase encoded by the subtilase encoding part of the polynucleotide cloned into a plasmid fragment present in Escherichia coli MT173 DSM 15575; or<br><br>
II) the subtilase encoded by the subtilase encoding part of the polynucleotide cloned into a plasmid fragment present in Escherichia coli MT173 DSM 15574.<br>
6.	A subtilase according to any of the preceding claims, wherein the subtilase is a variant of a<br>
subtilase having<br>
I)	the amino acid sequence shown as amino acids 1 to 269 of SEQ ID. NO:2 comprising a<br>
substitution, deletion, and/or insertion of one or more amino acid residues; or<br>
II)	the amino acid sequence shown as amino acids 1 to 269 of SEQ ID NO:4 comprising a<br>
substitution, deletion, and/or insertion of one or more amino acid residues.<br><br>
7.	A subtilase according to claim 6, comprising at least one modification in one of the positions<br>
27, 36, 56, 76, 87, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 120, 123, 159, 167, 170, 206,<br>
218, 222, 224, 232, 235, 236, 245, 248, 252 and 274 (BASBPN numbering).<br>
8.	A subtilase according to claim 7, wherein the modification is selected from the group<br>
consisting of K27R, *36D, S56P, N76D, S87N, G97N, S101G, S103A, V104A, V104I, V104N,<br>
V104Y,  H120D, N123S,  G159D, Y167, R170, Q206E, N218S, M222S, M222A, T224S,<br>
A232V, K235L, Q236H, Q245R, N248D, N252K and T274A (BASBPN numbering).<br>
9.	An isolated poiynudeotide comprising a polynucleotide that encodes for the subtilase defined in<br>
any of claims 1-8.<br>
10.	An isolated polynucleotide encoding a subtilase, selected from the group consisting of<br>
(a)	a polynucieotide having at (east 88% identity with the polynucleotide shown as<br>
nucleotides 1 to 807 SEQ ID NO:1; or<br>
(b)	the subtilase encoding part of the polynucleotide that has been cloned into a plasmid<br>
present in Escherichia coli MT173 DSM 15575, or a variant thereof having at least 88%<br>
identity to said nucleic acid sequence; or<br>
(c)	a polynucleotide having at least 88% identiy with the polynucleotide shown as<br>
nucleotides 1 to 807 SEQ ID NO:1; or<br>
(d)	the subtilase encoding part of the polynucleotide that has been cloned into a plasmid<br>
present in Escherichia coli MT173 DSM 15574, or a variant thereof having at least 88%<br>
identity to said nucleic acid sequence.<br>
11.	A polynucleotide according to claim 10 having<br>
I) a nucleic acid sequence which has at least 89%, at least 90%, at least 91%, at least 92%, at least 93%, at least 94%, at least 95%, at least 96%, at least 97%, at least 98%, or at least<br><br>
99% identity with the polynucleotide shown as nucleotides 1 to 807 of SEQ ID NO:1; or II) a nucleic acid sequence which has at least 89%, at least 90%, at least 91%, at least 92%, at least 93%, at least 94%, at least 95%, at least 96%, at least 97%, at least 98%, or at least 99% identity with the polynucleotide shown as nucleotides 1 to 807 of SEQ ID NO:3.<br>
12.	A polynucieotide according to claim 10, having<br>
I)	a nucleic acid sequence which has at least 89%, at least 90%, at least 91%, at least 92%, at<br>
least 93%, at least 94%, at least 95%, at least 96%, at least 97%, at least 98%, or at least<br>
99% identity with the subtilase encoding part of the.polynucleotide which has been cloned into<br>
a plasmid present in Escherichia coli MT173 DSM 15575; or<br>
II)	a nucleic acid sequence which has at least 89%, at least 90%, at least 91%, at least 92%,<br>
at least 93%, at least 94%, at least 95%, at least 96%, at least 97%, at least 98%, or at least<br>
99% identity with the subtilase encoding part of the polynucleotide which has been cloned into<br>
a plasmid present in Escherichia coli MT173 DSM 15574.<br>
13.	A nucleic acid construct comprising the nucleic acid sequence of any of claims 9 -12 op-<br>
erably linked to one or more control sequences capable of directing the expression of the sub<br>
tilase in a suitable host.<br>
14.	A recombinant expression vector comprising the nucleic acid construct of claim 13, a pro<br>
moter, and transcriptional and translational stop signals.<br>
15.	A recombinant host cell comprising the nucleic acid construct of claim 13.<br>
16.	A host cell according to claim 15, which is a bacterium, preferably a Bacillus, especially a<br>
Bacillus lentus.<br>
17.	A host cell according to claim 15, which is a fungus or yeast, preferably a filamentous fun<br>
gus, especially an Aspergillus.<br>
18.	A method for producing the subtilase according to any of claims 1-8, the method compris<br>
ing:<br><br>
(a)	cultivating a recombinant host cell as defined in any of claims 15-17 under conditions<br>
conducive to the production of the subtilase; and<br>
(b)	recovering the subtilase.<br><br>
19.	A cleaning or detergent composition, preferably a laundry or dish wash composition,<br>
comprising the subtilase according to any of claims 1-8.<br>
20.	A composition according to claim 19, which additionally comprises a cellulase, lipase,<br>
cutinase, oxidoreductase, another protease, an amylase or a mixture thereof.<br>
21.	Use of a subtilase as defined in any of claims 1-8 in a cleaning or detergent composition.<br>
22.	Use of a subtilase as defined in any of claims 1-8 for removal of egg stains.<br>
23.	Use of a cleaning or detergent composition as defined in any of claims 19-20 for removal<br>
of egg stains.<br>
24.	A method for cleaning or dish washing, washing a hard surface or laundry, the method<br>
comprising contacting the hard surface or the laundry with the composition defined in claims<br>
19-20.<br>
25.	A method for removal of egg stains from a hard surface or from laundry, the method<br>
comprising contacting the egg stain-containing hard surface or the egg stain-containing<br>
laundry with the composition defined in claims 19-20.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg5OC1jaGVucC0yMDA1IGFic3RyYWN0IGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">2898-chenp-2005 abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg5OC1jaGVucC0yMDA1IGNsYWltcyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">2898-chenp-2005 claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg5OC1jaGVucC0yMDA1IGRlc2NyaXB0aW9uIChjb21wbGV0ZSkgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">2898-chenp-2005 description (complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg5OC1jaGVucC0yMDA1IGRyYXdpbmdzIGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">2898-chenp-2005 drawings duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg5OC1jaGVucC0yMDA1LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2898-chenp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg5OC1jaGVucC0yMDA1LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2898-chenp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg5OC1jaGVucC0yMDA1LWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2898-chenp-2005-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg5OC1jaGVucC0yMDA1LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">2898-chenp-2005-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg5OC1jaGVucC0yMDA1LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">2898-chenp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg5OC1jaGVucC0yMDA1LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">2898-chenp-2005-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg5OC1jaGVucC0yMDA1LWZvcm0gMjYucGRm" target="_blank" style="word-wrap:break-word;">2898-chenp-2005-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg5OC1jaGVucC0yMDA1LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">2898-chenp-2005-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg5OC1jaGVucC0yMDA1LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">2898-chenp-2005-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg5OC1jaGVucC0yMDA1LXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">2898-chenp-2005-pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIzNi1jaGVucC0yMDA1LWZvcm0gMTgucGRm" target="_blank" style="word-wrap:break-word;">3236-chenp-2005-form 18.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="231302-wind-energy-system.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="231304-peelable-bonded-ribbon-matrix-material-optical-fiber-bonded-ribbon-arrays-containing-same-and-process-for-preparing-said-optical-fiber-bonded-ribbon-arrays.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>231303</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2898/CHENP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>04-Mar-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>07-Nov-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NOVOZYMES A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>KROGSHOEJVEJ 36, DK-2880 BAGSVAERD,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>MINSHULL, JEREMY, S</td>
											<td>679 LOS NINOS WAY, LOS ALTOS, CA 94022,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>MINNING, STEFAN</td>
											<td>LYKKESHOLMS ALLE 3C, 2. TV., DK-1902 FREDERIKSBERG C,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>KNOTZEL, JURGEN, CARSTEN, FRANZ</td>
											<td>OSTERBROGADE 226, 2, TH., DK-2100 COPENHAGEN,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>SORENSEN, NIELS, HENRIK</td>
											<td>DYRELUNDEN 1, DK-3320 SKAVINGE,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>NESS, JON, E</td>
											<td>710 BAIR ISLAND ROAD, #405, REDWOOD CITY, CA 94063,</td>
										</tr>
										<tr>
											<td>6</td>
											<td>WELCH, MARK, D</td>
											<td>25 MONTALBAN DRIVE, FREMONT, CALIFORNIA 94536,</td>
										</tr>
										<tr>
											<td>7</td>
											<td>GIVER, LORRAINE, J</td>
											<td>101 W. FERNDALE DRIVE, SUNNYVALE, CA 94085,</td>
										</tr>
										<tr>
											<td>8</td>
											<td>CHERRY, JOEL</td>
											<td>3314 CHESAPEAKE BAY AVENUE, DAVIS, CALIFORNIA 95616,</td>
										</tr>
										<tr>
											<td>9</td>
											<td>VEDEL BORCHERT, TORBEN</td>
											<td>BISKOP SVANES VEJ 65A, 1TH, DK-3460 BIRKEROD,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N9/54</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/DK04/00312</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-05-06</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>PA 2003 00690</td>
									<td>2003-05-07</td>
								    <td>Denmark</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/231303-variant-subtilisin-enzymes-subtilases by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:10:14 GMT -->
</html>
