<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/231750-ingap-regulatory-sequences-and-methods-of-use-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:39:42 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 231750:&quot;INGAP REGULATORY SEQUENCES AND METHODS OF USE THEREOF&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;INGAP REGULATORY SEQUENCES AND METHODS OF USE THEREOF&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A reporter construct contains mammalian INGAP 5&#x27;-regulatory region or a fragment thereof, a minimal promoter element from mammalian INGAP or a heterologous promoter, and a reporter gene. The reporter construct can be used to screen for agents which alone or in combination up-regulate or down-regulate reporter gene expression. Alternatively, the reporter construct can be used to screen for agents that bind to the hamster INGAP 5&#x27;-regulatory region or a fragment thereof.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FIELD OF THE INVENTION<br>
The invention relates to reporter construct and methods for identifying agents that modulate the expression of the INGAP gene. The invention also provides assays for the detection of factors that modulate gene expression by employing the reporter construct.<br>
BACKGROUND OF THE INVENTION<br>
Islet neogenesis gene associated protein (INGAP protein) has been identified as a pancreatic acinar cell protein that can induce islet cell neogenesis from progenitor cells resident in the pancreas in a manner that recapitulates islet development during normal embryogenesis. INGAP is unique in its ability to stimulate growth and differentiation of islets of Langerhans from precursor cells associated with pancreas. These islets evolve a mature insulin secretory profile capable of responding to perturbations in blood glucose in a physiologic manner. This potential anti-diabetic therapeutic has been shown to demonstrate homology across several species and to exert a biological response.<br>
Pancreatic islet cell mass is lost in type I diabetes mellitus, a disease in which a progressive autoimmune reaction results in the selective destruction of insulin-producing ß - cells. In type 2 diabetes mellitus, so-called adult-onset disease, but also increasingly a condition in young overweight people, the ß - cell mass may be reduced by as much as 60% of nornlal. The number of functioning ß - cells in the<br>
pancreas is of critical significance for the development, course, and outcome of diabetes. In type I diabetes, there is a reduction of ß-cell mass to less than 2% of normal Even in the face of severe insulin resistance as occurs in type II diabetes, the development of diabetes only occurs if there is inadequate compensatory increase in ß-cell mass. Thus, the development of either of the major forms of diabetes can be regarded as a failure of adaptive P-cell growth and a subsequent deficiency in insulin secretion. Stimulating the growth of islets and p-cells from precursor cells, known as islet neogenesis, is an attractive approach to the amelioration of diabetes. There is need in the art for methods to identify agents that can modulate the expression of INGAP, whether in animals or in cultured cells.<br>
BRIEF SUMMARY OF THE INVENTION<br>
It is an object of the invention to provide a reporter construct containing the 5'-regulatory region from mammalian INGAP gene.<br>
It is another object of the invention to provide methods for identifying agents which modulate INGAP expression.<br>
It is another object of the invention to provide a nucleic acid or fragment of INGAP 5'-regulatory region.<br>
It is another object of the invention to provide methods for increasing ENGAP expression.<br>
It is another object of the invention to provide a kit for modulating INGAP expression.<br>
These and other objects of the invention are provided by one or more of the embodiments described below.<br>
In one aspect of the invention a reporter construct is provided. The reporter construct comprises a regulatory region nucleotide sequence and a nucleotide<br>
sequence encoding a detectable product. In one aspect of the invention, the reporter construct is provided in a vector. The regulatory region nucleotide sequence is linked to the nucleotide sequence encoding a detectable product The regulatory region nucleotide sequence may comprise one or more fragments of 5' regulatory region of the INGAP genomic sequence, SEQ ID NO: 23, or it may comprise the entire length of the 5' regulatory region. In one embodiment of the reporter construct, a promoter element is interposed between the regulatory region nucleotide sequence and the nucleotide sequence encoding a detectable product. The promoter element may be selected from the promoter elements present in the INGAP regulatory sequence. Alternatively, the promoter element present in the vector comprising the reporter construct may be used. The detectable product encoded by the said nucleotide sequence encoding a detectable product could be either a nucleic acid or a protein. The detectable product need not be the INGAP gene nucleic acid or protein.<br>
In another embodiment of the invention, a method identifying agents that modulate INGAP expression is provided. The method comprises contacting a cell with a test agent, wherein the cell comprises a reporter construct of the present invention. Expression of the detectable nucleic acid or protein product in the cell is determined. A test agent is identified as a modulator of INGAP expression if the test agent modulates expression of the detectable product in the cell.<br>
In another embodiment of the invention, an isolated nucleic acid comprising the genomic sequence of the hamster INGAP gene (SEQ ID NO: 2), or a fragment thereof is provided.<br>
According to another embodiment of the invention, an in vitro method for identifying agents that modulate INGAP expression is provided. The method comprises contacting a test agent with a reporter construct of the present invention in a cell-free system that allows for transcription and translation of a nucleotide sequence.   Expression of the detectable product is determined.  The substance is<br>
identified as a modulator of INGAP expression if the test substance modulates expression of the detectable product.<br>
According to another embodiment of the invention, an in vitro method for identifying an agent that modulate INGAP expression is provided The method comprises contacting a test agent with a nucleic acid of the invention. Binding of the test agent to the nucleic acid is determined. The test agent is identified as a modulator of INGAP expression if the test agent binds to the nucleic acid.<br>
According to another embodiment of the invention a method for increasing INGAP expression is provided. An effective amount of a factor that stimulates INGAP expression directly or indirectly, for example cytokines, chemokines, growth factors, or pharmacological agents, is administered to a mammal in need of increased INGAP expression.<br>
According to another embodiment of the invention a kit for modulating INGAP expression is provided. The kit comprises a modulator of INGAP expression ard instructions for using the modulator of INGAP expression to modulate INGAP expression.<br>
According to another embodiment of the invention a method for modulating INGAP expression in a mammal to treat a disease state related to reduced islet cell function is provided. The method compnses the step of administering to the mammal an effective amount of a modulator of INGAP expression whereby the level of INGAP expression in the mammal is modified.<br>
All documents cited are, in relevant part, incorporated herein by reference; the citation of any document is not to be construed as an admission that it is prior art with respect to the present invention<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figure 1 shows the annotation of the hamster INGAP gene structure. The boundaries of introns 1 - 5 are listed in Table 1.<br>
Figure 2 shows an overview of the 5'-regulatory region of the hamster INGAP gene (nucleotides 1-3137 of SEQ ID NO: 2) showing many well known and well-characterized transcription factor binding sites. The minimal promoter element contains the regions noted with an underline (CAAT-box, TATA-box, and GC-box).<br>
Figure 3 shows a schematic of many well known and well-characterized transcription factor-binding sites for nucleotides 1-3123 of the 5'-regulatory region (SEQ ID NO: I) of the hamster INGAP gene. Table 3 further describes these transcription factor-binding sites.<br>
Figure 4 shows the predicted transcription start sites within the 5'-regulatory region (SEQ ID NO: 1) of the hamster INGAP gene (SEQ ED NO: 2). The predicted start site is indicated by a boldface nucleotide. The start and end nucleotide numbers are indicated for the promoter sequence. The numbers refer to nucleotide numbers of the hamster INGAP gene (SEQ ID NO: 2)<br>
Figure 5 shows the adapter primer structure and sequence used in gene walking. Adapter primer 1 (API) and adapter primer 2 (AP2) are shown.<br>
Figures 6 and 7 show the strategy for reconstructing the hamster INGAP gene. The hamster INGAP gene was reconstructed using the technique of gene walking. Shown are the fragments and the gene specific primers (GSP1 and GSP2) used in PCR amplification for gene walking. Fragments were joined together using unique restriction enzyme sites within each fragment. The nucleotide sequences of the individual primers are listed in Table 2.<br>
Figure 8 shows the fragments of INGAP 5'-regulatory region, which were cloned into pßGal-basic upstream of a P-galactosidase reporter gene. The labels on the left refer to the nucleotide fragments of SEQ ID NO: 23 which were cloned upstream of pßGal-basic.<br>
Figures 9A shows reporter activity in human embryonic kidney cells (293T) transfected with a reporter construct that contains various fragments of the 5'-<br>
regulatory region (SEQ ID NO: 23) of hamster INGAP DNA cloned upstream of a P-galactosidase reporter gene (pPGal-basic), or in a reporter construct which contains no INGAP DNA The cells are stimulated with phorbol myristate acetate. Promoter activity is assessed by determining the level of p-galactosidase present in the cell using a p-galactosidase luminescent assay.<br>
Figures 9B shows reporter activity in human embryonic kidney cells (293T) transfected with a reporter construct that contains nucleotides 2030 to 3137 of the 5'-regulatory region (SEQ ID NO: 23) of hamster INGAP cloned upstream of a ß-galactosidase reporter gene, or in a reporter construct which contains no INGAP DNA. The cells are stimulated with leukemia inhibitory factor. Promoter activity is assessed by determining the level of P-galactosidase present in the cell using a p-galactosidase luminescent assay.<br>
Figure 10 shows the reporter activity in human embryonic kidney cells (293T) transfected with a reporter construct that contains different fragments (see Figure 8) of the 5'-regulatory region of hamster INGAP cloned upstream of a p-galactosidase reporter gene. The cells are stimulated with phorbol myristate acetate. Concentrations of PMA used are 6 ng/ml, 17 ng/ml, 50 ng/ml, 100 ng/ml, or 300 ng/ml. Promoter activity is assessed by determining the level of P-galactosidase present in the cell using a p-galactosidase luminescent assay.<br>
Figure 11 shows reporter activity in human embryonic kidney cells (293T) transfected with a reporter construct that contains different fragments (see Figure 8) of the 5'-regulatory region of hamster INGAP cloned upstream of a p-galactosidase reporter gene. The cells are stimulated with human leukemia inhibitory factor (hLIF). Concentrations of hLIF used are 1 ng/ml, 10 ng/ml, or 30 ng/ml. Promoter activity was assessed by determining the level of P-galactosidase present in the cell using a P-galactosidase luminescent assay.<br>
Figure 12 shows RNA analysis for INGAP gene upregulation in rat amphicrine pancreatic cells, AR42J, treated with cytokine IL-6 or untreated. Total RNA is probed by Northern analysis for INGAP gene.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
Definitions<br>
It must be noted that as used herein and in the appended claims, the singular forms "a", "an", and "the" include plural references unless the context clearly dictates otherwise.<br>
The term "promoter" is used to define the region of a gene at which initiation and rate of transcription are controlled. It contains the site at which RNA polymerase binds and also sites for the binding of regulatory proteins, e.g. transcription factors, repressors, etc. In order to differentiate between the transcription initiation site and other sites that modulate rate of transcription, promoter region is generally subdivided into "minimal promoter element" and "regulatory region". The term "minimal promoter element" or sometimes simply referred to as "promoter" therefore may include TATA box, GC-rich sequence and CAAT box; while "regulatory region" is usually a long stretch of nucleotide sequence where transcription factors and other factors bind. Most eukaryotic genes have long regulatory regions where many different transcription factors bind. The expression or the lack of expression of a given gene in a given cell type, tissue, organ, or an organism is governed by the interactions that take place on its regulatory region.<br>
The term "transcription factor" is used to describe the proteins that bind short stretches of DNA in the regulatory regions of a gene. Transcription factors may interact with each other as well as RNA polymerase. Thus, transcription factors may bind hormones or second messengers, DNA, RNA, other transcription factors, or other proteins. They may activate or inhibit transcription of a given gene. Transcription factors are also sometimes referred to as "enhancers" or "repressors". Transcription factor binding sites can be used to identify agents that bind to the 5'-regulatory region of the gene and modulate the gene's expression.<br>
The term "reporter" is used to describe a coding sequence attached to a heterologous promoter or enhancer elements and whose product, either nucleic acid or protein, is easily detected and is quantifiable. Some common reporter genes include p-galactosidase (lacZ), chloramphenicol acetyltransferase (cat), p-glucuronidase (GUS), and green fluorescent protein (GFP).<br>
A "reporter construct" is a piece of nucleic acid that includes a promoter element and a reporter gene housed in a suitable vector plasmid DNA. Regulatory region nucleotide sequences may be cloned 5' of the promoter element to determine if they contain transcription factor binding sites. The reporter construct-containing vector is introduced into a cell that contains many transcription factors. Activation of the reporter gene by transcription factors may be monitored by detection and quantification of the product of the reporter gene.<br>
The term "agent" is used here to essentially describe any means to modulate INGAP expression. Agent may be a chemical compound, a biological agent, or a physical force, a mechanical contraption, 01 any combinations thereof.<br>
INGAP Promoter and Regulatory Region<br>
It is a discovery of the present inventors that INGAP gene is regulated by a 5'-regulatory region that is susceptible to modulation by many known transcription factors, including PMA and LIF.<br>
It is a further discovery of the present invention that the 5'-regulatory region nucleotide sequence of the INGAP gene may be used in screening assays to identify agents capable of modulating the INGAP gene expression. These modulating agents have potential as therapeutic agents for treating pathological conditions including, but not limited to, diabetes mellirus, both type 1 and type 2,<br>
endocrine and non-endocrine hypoplasia, hypertrophy, adenoma, neoplasia, and nesidioblastosis.<br>
Mammalian INGAP, like most genes, has a 5'-regulatory region followed by introns and exons. The sequence of a mammalian (Hamster  sp.) IN GAT gene is provided as SEQ ID NO: 2. Figure 1 details the relative location of the 5'-regulatory region, the introns and the exons of the hamster INGAP gene. The boundaries of introns 1-5 and the location of the TATA-box and the poly-A signal are listed in Table 1.<br>
Table 1<br><br>
(Table Removed) <br>
The nucleotide sequence of the 5'-regulatory region including the promoter elements of mammalian INGAP, is shown partially in SEQ ED NO: 1, and completely in SEQ ID NO: 2 and 23 (nucleotides 1-3137 of SEQ ID NO: 2). Nucleotides 1-3120 of SEQ ED NO: 1 are identical to nucleotides 1-3120 of SEQ ID NO: 2 and SEQ ID NO: 23. An overview of the 5'-regulatory region is shown in Figure 2. Representative transcription enhancer/repressor binding sites are shown also in Figure 2. Predicted transcription enhancer/repressor binding sites for nucleotides 1-3123 of the 5'-regulatory region are shown in Figure 3. Table 3 at the end of the specification details these transcription factors and their binding sites, and their locations in the regulatory region. Potential transcription factor binding analysis was done using Matlnspector professional™, which is a bioinformatics software that utilizes a library of matrix descriptions for transcription  factor binding sites to  locate matches in  sequences of unlimited<br>
length (Quandt, K., Freeh, K., Karas, H., Wingender, E., Werner, T. (1995) Nucleic Acids Res. 23, 4878-4884).<br>
Table 3 lists predicted binding proteins (Further Information) based upon their classification into functionally similar matrix families (Family/matrix). The DNA sequence predicted to bind the protein (Sequence), whether sense or antisense DNA (Str) and location of the sequence in SEQ ID NO: 2, (Position) are listed. Further the similarity to the consecutive highest conserved nucleotides of a matrix (Core sim.) and similarity to all nucleotides in that matrix (Matrix sim.) along with the optimized value (Opt) defined in a .way that a minimum number of matches is found in non-regulatory test sequences are also listed. Details to the algorithms used in Mailnspector profess ionaFM is referenced:<br>
OPT: This matrix similarity is the optimized value defined in a way that a minimum number of matches are found in non-regulatory test sequences (i.e. with this matrix similarity the number of false positive matches is minimized). This matrix similarity is used when the user checks "Optimized" as the matrix similarity threshold for MzMnspzciov professional™.<br>
Family: Each matrix belongs to a so-called matrix family, where functionally<br>
similar matrices are grouped together, eliminating redundant matches by<br>
M at Inspector professional™ professional (if the family option was selected). E.g.<br>
the matrix family V$NFKB includes 5 similar matrices for NFkappaB<br>
(VSNFKAPPAB.Ol,	VSNFKAPPAB.02,	VSNPKAPPAB.03,<br>
VSNFKAPPAB50.01, VSNFKAPPAB65.01) as well as 1 matrix for the NFkappaB related factor c-Rel (VSCREL.Ol).<br>
Matrix: The Matlnspectorpro/ew/ona/™ matrices have an identifier that indicates one of the following seven groups- vertebrates (V$), insects (1$), plants (P$), fungi (F$), nematodes (N$), bacteria (B$), and other functional elements (0$); followed by an acronym for the factor the matrix refers to, and a consecutive number discriminating between different matrices for the same factor. Thus, V$OCT1.02 indicates the second matrix for vertebral Oct-1 factor.<br>
Core Sim: The "core sequence" of a matrix is defined as the (usually 4) consecutive highest conserved positions of the matrix. The core similarity is calculated as described here. The maximum core similarity of 1 0 is only reached when the highest conserved bases of a matrix match exactly in the sequence. More important than the core similarity is the matrix similarity which takes into account all bases over the whole matrix length.<br>
Matrix Sim: The matrix similarity is calculated as described here. A perfect match to the matrix gets a score of 1.00 (each sequence position corresponds to the highest conserved nucleotide at that position in the matrix), a "good" match to the matrix usually has a similarity of &gt;0.80. Mismatches in highly conserved positions of the matrix decrease the matrix similarity more than mismatches in less conserved regions.<br>
Another aspect of the invention provides for a reporter construct. Reporter constructs contain a 5' regulatory region nucleotide sequence fragment of SEQ ID NO: 23 (e.g., an enhancer and/or repressor binding site containing region), a promoter element (which may or may not be from INGAP regulatory region nucleotide sequence, SEQ ID NO: 23), and a reporter gene. The 5'-regulatory region nucleotide sequence is positioned upstream of the reporter gene. In order to determine the identity of various transcription factors that bind the 5' regulatory region nucleotide sequence and to elucidate their binding locations within the 5' regulatory nucleotide sequence of the INGAP gene, the region may be mapped using deletion analysis. One or more fragments of the regulatory region nucleotide sequence may be initially analyzed for their responses to various transcription factor activators. Once, a region of interest is determined, further fine mapping may be carried out where DNA from different locations within the regulatory region could be combined to make a more robust, and responsive reporter construct. DNA sequences, such as INGAP 5'-regulatory region DNA or a fragment thereof, can be manipulated by methods well known in the art. Examples of such techniques include, but are not limited to, polymerase chain<br>
reaction (PCR), restriction enzyme endonuclease digestion, ligation, and gene walking. Cloning fragments of DNA, such as 5'-regulatory regions is well known in the art.<br>
Another approach to quantify the expression levels of a gene is to measure transcription of the gene. PCR-ELISA may be used to capture transcripts onto a solid phase using biotin or digoxigenin-labelled primers, oligonucleotide probes (oligoprobes) or directly after incorporation of the digoxigenin into the transcripts (Watzinger, F. and Lion, T. (2001) Nucleic Acids Res., 29, e52). Once captured, the transcripts can be detected using an enzyme-labeled avidin or anti-digoxigenin reporter molecule similar to a standard ELISA format. Another approach is to employ real-time PCR to detect the transcript of the reporter gene (Mackay, I. M. and Nitsche, A., Nucleic Acids Res. 2002 Mar 15; 30(6), 1292-305). In real-time PCR fluorogenic nucleotides are used and progress of the transcript is monitored in real-time as the polymerase transcribes the reporter gene.<br>
The promoter element in the reporter construct may or may not be from the same gene as the 5'-regulatory region. As an example, the enhancer/repressor region from the DSIGAP 5'-regulatory region, or a fragment of the enhancer/repressor region from the 1NGAP 5'-regulatory region, may be cloned upstream of a heterologous minimal promoter element, e.g., the minimal CMV promoter (Boshart et al, 1985) and the promoters for TK (Nordeen, 1988), IL-2, and MMTV.<br>
Transcription of a gene begins around the minimal promoter. Figure 4 shows the predicted transcription start sites for mammalian INGAP gene (SEQ ID NO: 2). SEQ ID NO: 2 was analyzed using "Neural Network Promoter Prediction" program designed by Martin Reese to identify eukaryotic promoter recognition elements such as TATA-box, GC-box, CAAT-box, and the transcription start site. These promoter elements are present in various combinations separated by various distances in sequence. The program is available on the Internet and is located at http://www.fruitfly.org/seq_tools/promoter.html.<br>
The reporter construct can be used to identify agents that modulate, either alone or in combination, the expression of INGAP. Some such agents may modulate expression of INGAP by binding to the regulatory region directly while others may regulate expression of transcription factors that bind to the INGAP regulatory region.<br>
The reporter construct can be transfected into a host cell in vitro, or in vivo through the pancreatic duct, either transiently or stably, and a test agent introduced to the assay system. Examples of test agents include, but are not limited to organic and inorganic chemical agents, carbohydrates, proteins, oligonucleotides, cholecystokinin, mechanically induced pressure, and agents which cause a pancreatic duct obstruction. Expression of the reporter gene product can be determined by an assay appropriate for the reporter gene employed. Examples of such assays include, but are not limited to a luminescent assay for p-galactosidase or luciferase, an enzymatic assay for chloramphenicol acetyl transferase, and fluorescence detection for fluorescent proteins. Such assays are well known in the art, and a skilled artisan will be able to select an appropriate assay for the chosen reporter. A test agent is identified as a modulator of INGAP expression if the test agent modulates expression of the reporter gene product. Preferably the level of increase or decrease is at least 50%, 100%, 200%, 500%, or 1000%, but any statistically significant change can be an indicator of modulatory activity. A skilled artisan may also determine reporter gene product expression in untreated cells, and in treated and untreated cells transfected with a promoter-less reporter gene only. Such determinations can be used to determine background levels of expression.<br>
Test agents can also be obtained by fractionating pancreatic secretion fluids. A pancreatic duct obstruction can be used as an exemplary method of harvesting pancreatic secretion fluids. The pancreatic secretion fluids can be fractionated by methods well known in the art. Examples include high-pressure liquid chromatography (HPLC), size exclusion chromatography, hydrophobic interacting columns, and density gradient centrifugation.   Individual fractions can be tested<br>
for agents that modulate reporter gene expression using a method described herein. The individual fractions can be further fractionated to identify agents that modulate reporter gene expression. The identified test agents can be used to modulate the expression of INGAP.<br>
A host cell can be any cell suitable for transfection and maintenance in a suitable assay system. Examples of suitable cells include, but are not limited to, mammalian cells, human cells, mouse cells, rat cells, monkey cells, dog cells, bovine cells, and porcine cells. Preferably the cells used will be human cells. The cells could be either transformed cells line or primary cells. Whole organ explants may also be used where the regulation may be monitored over many different cell types. Many methods exist in the art for transfecting or infecting cells with reporter construct DNA. Such methods include, but are not limited to, lipofection, electroporation, calcium phosphate precipitation, DEAE dextran, gene guns, and modified viral techniques (e.g., recombinant adenovirus or recombinant retrovirus). The skilled artisan can readily choose a method suitable for use with a given cell type and assay system.<br>
The reporter construct can also be introduced in vivo directly into cells of the pancreas. Examples of methods to introduce the reporter construct into pancreatic cells in vivo include pancreatic duct retrograde perfusion and in vivo electroporation (Mir, 2001). The reporter construct encodes a reporter gene product that is readily measured in vivo. A test agent can be administered systemically or locally, and expression of the reporter gene in vivo can be determined by an assay appropriate for the particular reporter employed. Examples of such include a fluorescence assay for green fluorescent protein.<br>
Methods for identifying agents that modulate INGAP expression can also be accomplished in vitro. The reporter construct can be contacted with a test agent in vitro under conditions sufficient for transcription and/or translation of the reporter gene. Components such as rabbit reticulocyte lysates or wheat germ extracts can be utilized for such a method.  Subsequently, the expression level of the reporter<br>
gene can be determined as described above utilizing an appropriate assay for a given reporter gene. A test agent is identified as a modulator of INGAP expression if the test agent modulates expression of the reporter gene. Threshold levels of change can be set by the practitioner as discussed above.<br>
A test agent can alternatively be contacted with an isolated and purified INGAP 5'-regulatory DNA molecule and one can determine if the test agent binds to the DNA molecule. Test agents can be a chemical agent, a protein, or a nucleic acid. Appropriate INGAP 5'-regulatory DNA molecules would include nucleotides 1-6586 of SEQ ID NO: 2, the 5'-regulatory region DNA (SEQ ID NO: 1, or SEQ ID NO: 23), or any fragment of the 5'-regulatory region, preferably a fragment which contains one or more enhancer/repressor binding sites. Methods to determine binding of the test agent to the fragment of DNA are well known in the art, e.g., electrophoretic mobility shift assay (EMSA). See for example Sambrook et al., MOLECULAR CLONING: A LABORATORY MANUAL, 2d ed., 1989, at pages 9.50-9.51. Fragments of the 5'-regulatory region can be obtained by methods well known in the art using the disclosed sequence (SEQ ID NO: 2). Examples of such methods include, PCR, restriction enzyme digestion, and chemical synthesis. Any fragment of DNA within the 5'-regulatory region (SEQ ID NO: 1, or 23) can be used. The exact location that an agent binds can be determined for example by utilizing smaller fragments to map precisely the binding site for the test agent. Test agents that bind in the assay can be further tested in other assays that require modulatory activity.<br>
An agent that causes an increase or decrease in reporter gene expression can be used as a modulator of INGAP expression. The modulator can be administered to a mammal in need of such modulation. Examples of mammals that may need INGAP expression modulation are those with reduced pancreatic function, in particular reduced islet cell function. Such mammals include those who have diabetes mellitus, impaired glucose tolerance, impaired fasting glucose, hyperglycemia, obesity, and pancreatic insufficiency.<br>
An agent that is identified as a modulator of INGAP expression can be supplied in a kit to treat diseases associated with reduced islet cell function. The kit would comprise in single or divided containers, in single or divided doses a modulator of INGAP expression. Written instructions may be included for using the modulator of INGAP expression. The instructions may simply refer a reader to another location such as a website or other information source.<br>
Agents that cause an increase in reporter gene expression can be used to increase INGAP expression to treat a disease state related to reduced islet cell function. Agents that cause a decrease in reporter gene expression can be used to decrease INGAP expression to treat a disease state related to hyperactivity of islet cells or a disease where reduced INGAP expression is desirable. Examples of such agents include, but are not limited to, PMA, LEF, interleukin-6, Oncostatin M, and ciliary neurotropic factor. Agents can be administered by any number of routes including, but not limited to, oral, intravenous, intramuscular, intra-arterial, intramedullary, intrathecal, intraventricular, transdermal, subcutaneous, intraperitoneal, intranasal, parenteral, topical, sublingual, rectal, or pancreatic duct retrograde perfusion. Agents for oral administration can be formulated using pharmaceutically acceptable carriers well known in the art in dosages suitable for oral administration. Such carriers enable the pharmaceutical compositions to be formulated as tablets, pills, dragees, capsules, liquids, gels, syrups, slurries, suspensions, and the like, for ingestion by the mammal. Agents for intravenous, intramuscular, intra-arterial, transdermal, and subcutaneous injections can be formulated using pharmaceutically acceptable carriers well known in the art in dosages suitable for injection into the mammal. Agents for intranasal, topical, and rectal administration can be formulated using pharmaceutically acceptable carriers well known in the art in dosages suitable for surface administration to the mammal. Mammals in need of an increase in INGAP expression include for example, mammals with diabetes mellitus, impaired glucose tolerance, impaired fasting glucose, hyperglycemia, obesity, and pancreatic insufficiency. Mammals in need of a decrease in INGAP expression include for example, mammals with hypoglycemia.<br>
The following examples are offered by way of illustration and do not limit the invention disclosed herein.<br>
Examples<br>
Example 1<br>
Hamster INGAP Genomic Sequence and Structure<br>
The hamster INGAP genomic sequence and structure was determined by gene walking (Clontech) and DNA sequencing. Gene walking is a method for walking upstream toward a promoter or downstream in genomic DNA from a known sequence, such as cDNA. This method utilizes four uncloned, adapter-ligated genomic fragment libraries. The manufacturer's recommended protocol is followed with one notable exception; hamster genomic DNA was used to create the uncloned, adapter-ligated genomic fragment libraries.<br>
To create uncloned, adapter ligated genomic fragment libraries, genomic DNA was purified from hamster cells. Four separate aliquots were thoroughly digested with PvuU, Stul, Dral, or EcoKV. Following digestion, inactivation of the restriction enzymes, and dephosphorylation, each separate pool of DNA fragments was ligated to an adapter, see figure 5. The adapter was phosphorylated to provide the requisite phosphate group for a ligation reaction. Also note that the 3-prime side of the short adapter contains an amine group to prevent the adapters from forming concatamers.<br>
Two gene specific primers (GSPl and GSP2) were designed for each region of known sequence (i.e., the exons of the INGAP gene). See Figure 6 for fragment location and GSPl and GSP2 location. The gene specific primers were designed as reverse PCR primers for all fragments except fragments 12 and 14_5. The gene specific primers for fragments 1_2 and 145 were designed as forward primers. Adapter primer 1 (API) and adapter primer 2 (AP2) (Figure 5) were forward PCR primers for all fragments except fragments 12 and 145, which were reverse PCR primers.  The outer gene specific primer (GSPl) was used with<br>
adapter primer 1 in a PCR reaction. To increase specificity, a second, nested PCR was set up using the inner gene specific primer (GSP2) and adapter primer 2. A small aliquot of the first reaction served as template for the second reaction. Gene specific PCR primers utilized for gene walking are listed in Table 2 and the strategy used to build the INGAP genomic sequence is shown in Figures 6 and 7. The arrowheads in Figure 6 represent the adapter primers (API and AP2), while the circles represent the gene specific primers (GSP1 and GSP2).<br>
Table 2<br><br>
(Table Removed) <br>
[66] The PCR fragments were sequenced to determine the nucleotide sequence of the INGAP 5'-regulatory region, the introns, the intron/exon junctions, and the 3-prime polyadenylation regions. The nucleotide sequence of hamster INGAP genomic DNA is shown in SEQ ID NO: 2.<br>
Example 2<br>
Cloning Hamster INGAP 5 '-Regulatory Region Fragment into a Reporter Construct<br>
To construct the INGAP 5'-regulatory region, individual PCR fragments were joined together at unique restriction sites located within two adjoining fragments. Figures 6 and 7 detail the strategy used to piece the INGAP 5'-regulatory region together. Fragments 83 and 23 were joined at a unique Sphl site; 143 and 83 were joined at a unique Bbsl site; 163 and 143 were joined at a unique Pstl site. The nucleotide sequence of hamster INGAP 5'-regulatory region DNA is shown in SEQ ID NO: 1 and 23 in the sequence listing.<br>
The hamster INGAP 5'-regulatory region or a fragment of the 5'-regulatory region was cloned into a reporter plasmid, pfJGal-Basic (Clontech). The 5'-regulatory region or fragments were cloned utilizing the unique Xmal site from the gene walking adapter primer and a unique BglII site located at the 3-prime side of the regulatory region. Figure 8 details the fragments cloned into pPGal-Basic. The sizes of the fragments are indicated to the right of the fragments and are expressed as the number of nucleotides of the fragment.<br>
Example 3<br>
Assay System to Screen for Factors that Modulate the Expression of INGAP<br>
Promoter analysis of INGAP identified a number of potential promoter-proximal regulatory sites including the consensus transcription factor binding sites; cAMP response element (CRE), AP-1 and STAT. Promoter-fragment reporter-gene constructs were transiently transfected into 293T cells and co-transfection of secretory alkaline phosphatase was used to normalize for transfection efficiency.<br>
Reporter constructs containing INGAP 5'-regulatory region fragments 2_3sP (SEQ ID NO: 37), 2_3dP (SEQ ID NO: 38), 2_3pP (SEQ ID NO: 36), 14_3P (SEQ ID NO: 34), 16_3P (SEQ ID NO: 31), or 19_3P (SEQ ID NO: 23) were transfected into human cells.    The pPGal-Basic plasmid without the hamster<br>
INGAP DNA was also transfected into human cells as a control to measure the level of endogenous reporter activity. Two days following transfection, the cells were treated with PMA for 24 hours or were untreated. To determine the level of promoter activity, the amount of 3-galactosidase gene product was determined using a luminescent assay for P-galactosidase. Figure 9A shows that construct 143P activated the INGAP expression the most, followed by 2_3pP, and 16_3P.<br>
Reporter construct containing INGAP 5'-regulatory region DNA nucleotides 2030 to 3120 was transfected into human cells. The pPGal-Basic plasmid without the hamster INGAP DNA was also transfected into human cells as a control to measure the level of endogenous reporter activity. Two days following transfection, the cells were treated with LIF for 24 hours or were untreated. To determine the level of promoter activity, the amount of P-galactosidase gene product was determined using a luminescent assay for p-galactosidase. Figure 9B shows the results. LIF was determined to increase the activity of the 5'-regulatory region of mammalian INGAP. Forskolin (an activator of cAMP/CREB/CRE) did not modulate gene expression (data not shown).<br>
It is important to note that when present in human cells, the hamster INGAP 5'-regulatory region is transacrivated by the human transcription factors. Thus, linked to a reporter gene, the 5'-regulatory region of hamster INGAP creates a sensitive assay system to screen for factors that modulate the expression of INGAP.<br>
Example 4<br>
Determination of Approximate Location of PMA and LIF-mediated Transcription Factor Binding in the 5 '-Regulatory Region<br>
To map the approximate location of PMA-initiated or LIF-initiated transcription factor binding different fragments of the hamster INGAP 5'-regulatory region were cloned into pPGal-Basic. See Figure 8. The fragments cloned into the reporter construct were 2_3sP (SEQ ID NO: 37), 2_3dP (SEQ ID NO: 38), 2_3pP<br>
 (SEQ ID NO: 36), 14_3P (SEQ ID NO: 34), 16_3P (SEQ ID NO: 31), or 19_3P (SEQ ID NO: 23). The reporter constructs were transfected into human cells. Two days following transfection, the cells were treated with different concentrations of PMA or LIF for 24 hours. The concentrations of PMA used were 6 ng/ml, 17 ng/ml, 50 ng/ml, 100 ng/ml, or 300 ng/ml. The concentrations of LIF used were 1 ng/ml, 10 ng/ml, or 30 ng/ml. To determine the level of promoter activity, the amount of p-galactosidase gene product was determined using a luminescent assay for P-galactosidase. Figure 10 and 11 show the results for PMA and LIF treatment, respectively. Both PMA and LIF activated the cell reporter constructs. The exact location of the DNA contact sites can be narrowed further by cloning smaller fragments of the hamster INGAP 5'-regulatory region and by site directed mutations or deletions.<br>
Example 5<br>
RNA Analysis of INGAP gene upregulation<br>
To determine if INGAP RNA levels increase after stimulation with a cytokine that signals through STAT, rat amphocrine pancreatic cells, AR42J were treated with IL-6 (1000 U/ml) for 24 hours. Total RNA was extracted from the treated and untreated cells using techniques well known in the art, e.g., using TRIZOL® reagent.<br>
Equal amounts of total RNA (l0µg) were loaded in 2.5% formaldehyde gel and electrophoresed for 4 hours at 70V with a constant circulation of the buffer using a circulating pump. The gel was photographed and washed with water twice at room temperature and soaked in 20X SSC. The gel was transferred to a nylon membrane (Amersham) in 20X SSC overnight following a standard procedure. The membrane was washed with 20X SSC to remove any agar that might have attached to the membrane and baked for 4 hours at 80°C.<br>
One hundred nanograms of hamster INGAP cDNA was labeled using Random Prime Labeling kit (Roche-BMB) and alpha-P32 dCTP (ICN).  Approximately 20<br>
million counts were used for hybridization in 20 ml hybridization buffer following the standard procedure at 42°C for overnight. The blot was washed as follows: 2-times at room temperature with ?X SSC for 10 minutes each: 2-times at 42°C with 2X SSC for 10 minutes each; 2-times at 55°C with IX SSC for 10 minutes each. The membrane was exposed to the film (XOMAT-Kodak) and kept at -80°C overnight before developing.<br>
Treatment with IL-6 caused an increase in INGAP gene expression (Figure 12). These data demonstrate that extracellular factors that elevate AP-1 -binding transcription factors and STAT-binding transcription factors are involved in the regulation of INGAP gene expression. These studies suggest that it is feasible to enhance INGAP expression as a means of inducing islet neogenesis.<br>
While particular embodiments of the present invention have been illustrated and described, it would be obvious to those skilled in the art that various other changes and modifications can be made without departing from the spirit and scope of the invention. It is therefore intended to cover in the appended claims all such changes and modifications that are within the scope of this invention. <br>
Table 3<br>
(Table Removed) <br><br><br><br><br><br>
We Claim:<br>
1.	A reporter construct, comprising:<br>
a.	a nucleotide sequence which encodes a detectable product;<br>
b.	a regulatory region nucleotide sequence linked to the 5' end of<br>
nucleotide sequence that encodes a detectable product, wherein the regulatory region<br>
nucleotide sequence comprises of one or more regions chosen from nucleotides 1-<br>
3137 of SEQ ID NO:2;and<br>
c.	optionally a promoter slement interposed between the regulatory<br>
region nucleotide sequence and the nucleotide sequence encoding the detectable<br>
product.<br>
2.	The reporter construct of Claim 1 wherein the regulatory region nucleotide<br>
sequence is selected from the group consisting of SEQ ID NO: 1, 2, 23, 32, 35, 37, 28, 24, 25,<br>
26, 27, 29, 30, 31, 33, 34, 38, and 36.<br>
3.	The reporter construct of claim 1 wherein the promoter element is selected from SEQ lD NO:2.<br>
4.	An in vitro method for identifying agents which modulate INGAP expression, comprising:<br>
a.	contacting a reporter construct according to claim 1 or 2 with a test<br>
substance under conditions sufficier for transcription and translation of said<br>
nucleotide sequence;<br>
b.	determining expression of the detectable protein or nucleic acid<br>
product; and<br>
c.	identifying the test agent as a modulator of INGAP expression if the<br>
test substance modulates expression of the detectable product.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcyNy1kZWxucC0yMDA0LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1727-delnp-2004-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcyNy1kZWxucC0yMDA0LWFzc2lnbm1lbnQucGRm" target="_blank" style="word-wrap:break-word;">1727-delnp-2004-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcyNy1kZWxucC0yMDA0LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1727-delnp-2004-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcyNy1kZWxucC0yMDA0LWNvbXBsZXRlIHNwZWNpZmljYXRpb24gKGFzIGZpbGVkKS5wZGY=" target="_blank" style="word-wrap:break-word;">1727-delnp-2004-complete specification (as filed).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcyNy1kZWxucC0yMDA0LWNvbXBsZXRlIHNwZWNpZmljYXRpb24gKGdyYW50ZWQpLnBkZg==" target="_blank" style="word-wrap:break-word;">1727-delnp-2004-complete specification (granted).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcyNy1kZWxucC0yMDA0LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1727-delnp-2004-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcyNy1kZWxucC0yMDA0LWNvcnJlc3BvbmRlbmNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">1727-delnp-2004-correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcyNy1kZWxucC0yMDA0LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">1727-delnp-2004-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcyNy1kZWxucC0yMDA0LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1727-delnp-2004-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcyNy1kZWxucC0yMDA0LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">1727-delnp-2004-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcyNy1kZWxucC0yMDA0LWZvcm0tMTkucGRm" target="_blank" style="word-wrap:break-word;">1727-delnp-2004-form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcyNy1kZWxucC0yMDA0LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">1727-delnp-2004-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcyNy1kZWxucC0yMDA0LWZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">1727-delnp-2004-form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcyNy1kZWxucC0yMDA0LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">1727-delnp-2004-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcyNy1kZWxucC0yMDA0LWZvcm0tNC5wZGY=" target="_blank" style="word-wrap:break-word;">1727-delnp-2004-form-4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcyNy1kZWxucC0yMDA0LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">1727-delnp-2004-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcyNy1kZWxucC0yMDA0LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">1727-delnp-2004-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcyNy1kZWxucC0yMDA0LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">1727-delnp-2004-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcyNy1kZWxucC0yMDA0LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">1727-delnp-2004-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcyNy1kZWxucC0yMDA0LXBjdC00MDkucGRm" target="_blank" style="word-wrap:break-word;">1727-delnp-2004-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcyNy1kZWxucC0yMDA0LXBjdC00MTYucGRm" target="_blank" style="word-wrap:break-word;">1727-delnp-2004-pct-416.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="231749-dual-input-ac-dc-battery-assisted-power-convertor.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="231751-a-laundry-detergent-bar-composition.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>231750</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1727/DELNP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>09-Mar-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>17-Jun-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>GMP ENDOTHERAPEUTICS, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>ONE EAST BROWARD BOULEVARD, SUITE 1701, FT. LAUDERDALE, FL 33301 U.S.A.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>TAYLOR-FISHWICK, DAVID</td>
											<td>6117 ROLFE AVENUE NORFOLK, VA23510 U.S.A.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>VINIK, AARON I</td>
											<td>40 RADAR STREET, #603, NORFOLK, VA 23510 USA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US03/00707</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-01-10</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/388,315</td>
									<td>2002-06-14</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/346,898</td>
									<td>2002-01-11</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>60/361,073</td>
									<td>2002-03-01</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/231750-ingap-regulatory-sequences-and-methods-of-use-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:39:43 GMT -->
</html>
