<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/231863-method-for-increasing-abiotic-stress-tolerance-of-plants by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:47:11 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 231863:&quot;METHOD FOR INCREASING ABIOTIC STRESS TOLERANCE OF PLANTS&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;METHOD FOR INCREASING ABIOTIC STRESS TOLERANCE OF PLANTS&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Method for increasing abiotic stress tolerance of plants, comprising the steps of introducing and expressing an isolated nucleic acid encoding a CHMP protein in said plants and selecting for increased expression is said plants of said nucleic acid sequence and wherein said CHMP protein (a) comprises the sequence as given in SEQ ID NO. 2, 4, or 6: (b) comprises a sequence having at least 50%, alternatively 60%, 70%, 80%, preferable 90%, more preferably 95%, 96%, 98% or 99% sequence identity to the full length sequence as given in SEQ ID NO 2, 4, or 6; (c) comprises a substitution variant or insertion variant of (a); (d) according to any (a) to (c), comprises substitutions with corresponding naturally or non-naturally altered amino acids; (e) comprises a functional fragment of any of (a) to (d).</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Stress Tolerance<br>
Field of the invention<br>
The present invention concerns a method for identifying and obtaining nucleic adds capable of modifying stress tolerance, particularly cold tolerance, in plants. The invention also concerns isolated nucleic acids so obtained. The invention further concerns a method for obtaining plants having modified stress tolerance and to plants obtained by the methods according to the invention. The invention also relates to a yeast strain having modified tolerance to cold stress.<br>
Background<br>
Environmental stress conditions, such as shortage or excess of solar energy, water or nutrients, high salinity and pollution (e.g., heavy-metal pollution), can have a major impact on plant growth and can significantly reduce plant yield. Osmotic stress, a type of environmental stress, may be induced by conditions of excess salinity, drought, excessive heat, cold or freezing.<br>
Cold stress may be induced by temperatures below the range which allow optimal growth for a particular plant species. Each plant species or variety has an optimal growth temperature at which the growth rate is maximal; the further the deviation from this optimal growth temperature, the greater the stress on the plants. Many plant species, especially from tropical or subtropical regions, are sensitive to cold. For example. It has been estimated that the worldwide rice production would decrease by 40% if the worldwide mean temperature dropped only between 0.5 to 1.0'C (Salisbury and Ross, Plant Physiology. 4" ed. Wadsworth Publishing Company, Belmont, CA, 1992). Plants from temperate regions however have the ability to adapt their metabolism and to survive freezing temperatures after undergoing a process of adaptation to low but non-freezing temperatures, a process called cold acclimation. For instance non-acclimated rye typically does not survive temperatures below-5* C, but after cold acclimation it can withstand temperatures as low as -30"C. The process of cold acclimation involves altered expression of many genes. Plants may differ in their ability to withstand cold, which could lead to periodic but significant losses in plant productivity. As a consequence, the areas in which crops or horticultural plants can be cultivated is determined by assessing the risk of lower temperatures, relative to typical growth temperatures for any given plant.<br>
The most prominent changes during cold acclimation include a reduction or cessation of growth, reduction of tissue water content (Levitt; Responses of Plants to Environmental Stresses, Vol. 1.2nd edn. Academic Press. New York, NY 1980), transient increase in abscisic<br>
acid (ABA) levels (Chen et at.. Plant Physiology 71,362 - 365.1983), changes in membrane lipid composition (Lynch and Steponkus. Plant Physiology 83, 761 - 767,1987; Uemura and Steponkus, Plant Physiology 104.479 - 496,1994), the accumulation of compatible osmolytes such as praline, betaine. polyols and soluble sugars, and increased levels of antioxidants (Koster and Lynch, Plant Physiology 98, 108 - 113, 1992; Kishitani et al.. Plant, Cell and Environment 17, 89 - 95, 1994; Murelli et al., Physiologia Plantarum 94, 87 - 93 1995; Nomura et al.. Euphytica 83, 247 • 250,1995; Dorffling et al., Plant Molecular Biology 23. 221 - 225.1997; Tao otal., Cryobiology 37,38 • 45,1998).<br>
Various methods for the identification and isolation of genes or proteins differentially expressed during cold stress are known. For example, mapping techniques allow determination of chromosome locations of genes involved in cold tolerance (Pan et al., Theoretical and Applied Genetics 89,900-910,1994; Galiba et al., Theoretical and Applied Genetics 90.1174 - 1179, 1995). Another approach involves mutational analysis in which mutants that have an altered response to cold tolerance are isolated and characterized. For example, eskimol, conferring improved freezing tolerance of 20C over acclimated wild-type plants, was isolated from a collection of 800000 Ethyl Methyl Sulphonate (EMS)- mutagenised Arabidopsis lines that were screened for constitutively freezing-tolerant mutants (Xin and Browse, PNAS 95,7799 - 7804, 1998). Conversely, plant lines were screened for mutants defective in cold acclimation (Warren et al., Plant Physiology 111.1011 - 1019,1996; Knight et al., Plant Cell 8,489 - 503, 1996). cos-, los- and hos-mutants (for respectively constitutive, low and high expression of osmotically responsive genes) were isolated using a combination of mutagenesis and reporter gene activation (Ishitani et al., Plant Cell 9. 1935 -1949, 1997; Ishilani e(e/.. Plant Cell 10, 1151 - 1161. 1998; Lee et al., Plant Journal 17. 301 - 308, 1999). One of the drawbacks of mapping and the mutant analysis strategy is that they do not directly result in the isolation of nucleic acids coding for cold-induced genes. Another strategy, using differential screening of cDNA libraries and related techniques, has in the past yielded several cold induced genes from different plant species (reviewed in Xin and Browse, Plant, Cell and environment 23, 893 -902,2000). Many of those genes have known functions and can be grouped as being involved in drought stress, in signal transaction pathways, or as being related to heat shock proteins, molecular chaperones, 'antifreeze proteins" or regulatory proteins. Several of the genes are highly expressed during cold stress and are commonly referred to as COR (COId Regulated) genes (Tomashow, Annual Review of Plant Physiology and Plant Molecular Biology 50, 571 -599,1999).<br>
Strategies used to engineer cold resistant plants include accumulation of osmoprotectants such as mannitol (US 6,416,985), praline (US 6,239,332), trehalose (US 6,323,001) or glycine-<br>
betaine (Hayashi et a/.. Plant Journal 12,133 - 142,1997; US 6,281.411). Other approaches involve manipulating the signal transduction pathway controlling the stress response (WO 01/77355), including use of transcription factors (WO 01/77311, US 6,417,428, WO 02/44389, US 5,891,859). Furthermore a number of genes have been used to enhance cold resistance. Examples are members of the COR group (COR15a: US 5.296,462, US 5,356,816), a cell cycle related gene (WO 01 777354), protein kinase related proteins (WO 01/77356). the LEA-like protein CAP85 (US 5,837,545) and use of a phospholipid binding protein (WO 02/00697). Nevertheless, signal transduction pathways leading to cold acclimation and the identity of the genes that confer resistance to cold stress in plants remain largely unknown.<br>
Yeast has been used for screening plant genes that confer resistance to salt stress. For example, a salt-sensitive yeast strain (JM26) has previously been transformed with a cDNA library from salt-stressed sugar beet and used to screen for clones having increased salt tolerance (WO 02/52012). The transformed yeast cells were grown on a rich medium (YPD) or on a synthetic medium plus methionine and leucine (SO}, "Supplemented with 0.15 M NaCI or with 20 mM LiCI. Putative positive clones showing better growth on the selective media compared to the non-transformed yeast strain were isolated and further characterised. However, the use of yeast for identifying plant genes involved in cold stress has not been used before. A recent study in haploid yeast by de Jesus Ferreira et al. (2001), in which transposon mutagenesis was employed, identified 10 different yeast genes responsive to cold tolerance, which upon mutation caused a growth stop at 15 *C. The identified genes Include a gene coding for a glutamate synthase (YDL171C), a GTP binding protein (YML121W). a GSK-3 Ser/Thr protein kinase (YNL307C) and a component of TFIID (YLR399C). Three of the genes were previously described as cold responsive (YLR399C, YML121W. YNL307C) and four of the isolated genes were also involved in resistance to salt stress.<br>
Summary of the invention<br>
The present invention provides a novel screening method for nucleic acids involved in stress responses in a plant, which method involves screening in diploid yeast for plant genes involved in modifying tolerance/resistance to temperature stress. The present invention also provides new plant genes identified by this screen and polypeptides encoded by these genes. Also provided are methods for producing plants having modified tolerance or resistance to environmental stress conditions, comprising introduction of the above-mentioned genes into plants. Also provided are plants having modified tolerance or resistance to environmental stress conditions, which plants are transformed with the gene according to the invention.<br>
Detailed description of the Invention<br>
According to a first embodiment of the present invention, there is provided a screening method for identifying nucleic acids capable of modifying tolerance or resistance to cold stress conditions in plants or yeast, which method comprises the steps of:<br>
(i)       providing a cONA library of coding sequences from an organism;<br>
(ii)      introducing these coding sequences in an expressible format into wild type yeast cells;<br>
(iii)      growing the yeast cells of (ii) under conditions of cold stress;<br>
(iv)      identifying differences between transgenic yeast cells and wild type yeast cells, preferably identifying differences in growth rate;<br>
(v)      isolating nucleic acids from the transgenic yeast cells that differ from the wild type<br>
yeast cells.<br>
Preferably the wild type yeast cells are wild type diploid Saccharomyces cerevisiae yeast cells, more preferably wild type Sacchammyces carovisiaa W303 yeast cells. Furthermore preferably the organism is a plant, which plant .preferably is a salt treated plant, more preferably a salt treated halophytic plant or a part thereof, most preferably salt treated Beta vulgaris or a part thereof.<br>
The use of yeast cells for identifying genes involved in salt or osmotic stress is known in the art. Yeast is a good model organism for testing genes conferring tolerance to osmotic or salt stress because suitable mutants are known that allow for complementation. WO 02/052012 teaches a screening method wherein a mutant salt sensitive yeast strain is transformed with cONA isolated from salt stresses sugar beet. The method resulted in the identification of genes that may contribute to increased tolerance to salt, drought or osmotic stress in plants. Here, for the first time yeast was used for screening a plant cONA library involving the application of cold stress. In contrast to WO 02/052012, where a mutation causing salt sensitivity is to be complemented by an introduced plant cDNA, wild type yeast was used in the present invention instead of a mutant; the wild type yeast was diploid to avoid any effects of recessive chromosomal mutations that would eventually result in cold tolerance of the yeast host. The present invention demonstrates that, since no suitable cold sensitive mutants exist, wild type yeast cells transformed with cDNA from salt stressed plants can be used to isolate genes capable of conferring tolerance against cold stress in plants. The terms "tolerance" and "resistance* are used interchangeably herein.<br>
The first step of the screening method involves providing a cONA library of coding sequences from any organism, such as plants animals or fungi. According to a preferred feature of the present invention, the cDNA library is made from a plant, preferably a salt treated plant, further<br>
preferably a salt treated halophyUc plant or a part thereof, more preferably from a salt treated sugar beet plant or a part thereof, most preferably from leaves of salt treated Beta vulgaris plants. Sugar beet (Beta vulgaris), a relatively halophytic crop plant, provides a potentially good source of cold tolerance genes. Although the present invention is exemplified by use of a sugar beet cDNA library, it is to be understood that other hatophytic plants could equally serve the same purpose. The preparation of cDNA libraries is a routine technique well known In the art The cDNA library preferably comprises copies of essentially all mRNA of the plant call. Advantageously, coding sequences alone are sufficient.<br>
The second step of the screening method involves introducing the coding sequences into yeast cells. Methods for transformation of yeast, such as electroporation or treatment with Lithium Acetate, and for expressing genes in yeast, including yeast vectors, such as pYES, are well known in the art (see e.g. Current Protocols in Molecular Biology. Unit 13 (Ausubel ef a/., 1994) and the Guide to Yeast Genetics and Molecular Biology (Guthrie and STink, 1991)). Advantageously, coding sequences may be introduced and expressed in yeast' using any of several known methods, with the aim of testing tolerance or resistance to stress conditions. According to a preferred feature of the present invention, a vector based on the A phage is employed, more preferably XPG15 is used for introducing and expressing coding sequences in yeast. Phage XPG15 comprises the excisable expression plasmid pYPGEIS which may be used directly for both Escherichia coll and yeast complementation (Brunelli and Pail, Yeast 9, 1309 - 1318. 1993). A plasmid cONA library can be recovered from XPG15 using the cre-tor recombinase system (Brunelli and Pall, Yeast 9, 1309 - 1318, 1993). Preferably, the yeast cells are Saccharomyces cerevisiae, more preferably the diploid wild type strain W303 and its diploid mutant deficient for glycerol phosphate dehydrogenase (gpdl). The yeast strain W303 has the genotype MATa/MATa, AOE2/ade2, CAN1/can1-100. CYH2/cyh2, his3-11,15/his3-11,15, LEU1/leu1-c, LEU2/leu2-3,112, trp1-1:URA3:trp1-3'D rtrp1-1, ura3-1/ura3-1, and originates from the parent strains W303-1A and W303-1B (Primig et al., Nat. Genet 26, 415 -423, 2000). The W303 gpdl mutant was unexpectedly more cold tolerant than the W303 wild type strain (see Figure 1). For this reason the wild type strain was used in the screening, while the gpdl mutant strain served as a standard for comparison. It was thus expected that nucleic acids conferring cold tolerance would enhance the growth of the wild type yeast cells to a comparable or better level to that of the gpdl mutant.<br>
Advantageously, the gpdl gene can be used for enhancing cold tolerance of yeast, for example baker's yeast. Yeast is known to be sensitive to cold stress. Freezing stress in particular has a negative impact on the quality of yeast as a leaven. Yeast cells that have been mutated or engineered such that the glycerol phosphate dehydrogenase (gpdf) gene is<br>
inactivated (using techniques known in the art) are surprisingly more tolerant to cold and/or freezing stress than wild type yeast This trait can be of benefit in, for example, the baking or brewing industries. The present invention thus also provides a method for increasing cold tolerance of yeast cells, comprising downregulating expression in yeast of a nucleic acid encoding a glycerol phosphate dehydrogenase and/or inhibiting activity of a glycerol phosphate dehydrogenase. The invention furthermore provides for the use of a gpdl gene for altering the stress tolerance of yeast by downregulating its expression. The stress-tolerant yeast cells thus obtained can be used in purified form (for example leaven) or in compositions (for example dough).<br>
The third step of the screen involves growing the yeast cells under stressed conditions. Yeast cells transformed with cONA of the salt stressed sugar beet were plated onto a suitable medium and grown under cold stress. A temperature of 10*C was chosen, as this still allowed a minimal growth of the yeast strain, but a person skilled in the art may choose any other temperature below the optimal growth temperature. After a certain period of time colonies that were able to grow under these conditions of cold were selected and their cold tolerance was retested by growing the transgenic cells again under cold stress conditions Advantageously, the cDNA from salt treated plants may also be a suitable basis for finding genes capable for conferring tolerance against other stresses. This may be achieved simply by growing the yeast cells In step (iii) above in conditions of stress determined by the type of gene sought For example, in order to identify genes conferring tolerance or resistance to heat stress, the yeast cells would be grown in conditions of heat According to a preferred feature of the invention, the stress is preferably cold stress. It was then determined whether the stress tolerance originated from the transgene and not from a mutation in the host genome. To this end, the plasmid comprising the transgene was cured from a transgenic cold tolerant yeast done and it was verified whether the cold tolerance had disappeared too; secondly the plasmid comprising the transgene was isolated from a transgenic cold tolerant yeast done and reintroduced into a non-transgenic yeast strain, whereafter the cold tolerance of the newly transformed yeast strain was compared to the non-transformed yeast strain.<br>
The fourth step in the screening method is the identification of fast growing yeast cells. Yeast cells transformed with a plant nudeic acid conferring stress resistance were identified based on their ability to grow faster under stress conditions than yeast cells not transformed with such a nucleic acid, although other selection criteria may also be used, depending on the type of stress that is applied.<br>
Finally, in the last step of the screening method nucleic acids conferring stress tolerance are isolated from the yeast host and characterised. Methods for isolating nucleic adds from yeast and sequencing these nucleic acids are known to those skilled in the art.<br>
The present invention also encompasses the use of the screening method described above for identifying nucleic acids encoding proteins capable of conferring cold stress tolerance to plant cells or yeast cells.<br>
The screening method described above yielded several nucleic acids encoding proteins that increase cold stress tolerance of the yeast strain, hereafter named CRYO genes and CRYO proteins. The proteins encoded by these nucleic adds all relate to vesicle trafficking to the vacuole or plasma membrane. Response to stress requires an adaptation of metabolism, induding transport of proteins and other components between different organelles, in particular between the Golgi apparatus and the vacuole, but also between the plasma membrane and the vacuole. Plant vacuoles perform different functions, depending on the cell type in which they occur. They play an important role in cell growth or function as storage organelles for proteins, ions, secondary metabolites and metabolic waste products. In this last aspect, vacuoles also resemble lysosomes. They contain many hydrolytic enzymes for degradation of damaged or redundant cell material. Adaptation to changing environmental conditions or to stress involves not only synthesis of new cellular components, but also degradation of cellular material. These degradation processes require an extensive trafficking of material via membrane bound vesicles such as endosomes. Also hydrolytic enzymes are delivered to the vacuole via endosomes.<br>
CRYO4 (SEQ 10 NO 8) is a protein with homology to At1g72160, a cytosolic factor in Arabidopsis thaliana, and ft further has significant homology with yeast SEC14 (=YMR079W). This yeast protein is a cytosolic phosphatidylinositol/phosphatidylcholine transfer protein and is required for the transport of secretory proteins from the Golgi complex and for protein secretion (Bankaitis et al., (1990) Nature 347,561-562). In yeast it is associated with the Golgi complex as a peripheral membrane protein and forms a link between phosphollpkJ metabolism and vesicle trafficking (Li et al.. (2000) Mol. Biol. Cell 11,1989-2005). It catalyses the transfer of phosphatidylinosKol and phosphatidylcholine between membranes in vitto and is essential few-viability and secretion (Tschopp et al.. (1984) J. Bacteriol 160,966-970).<br>
CRYO5 (SEQ ID N010) is a protein with a RING-domain. RING-domain proteins are known to be involved in biological processes such as transcriptional and translation^ regulation, and in targeted proteolysls. The RING-domain mediates protein-protein interactions and is a C3HC4<br>
type zinc-finger domain of 40 to 60 amino adds long. Various proteins with a RING finger domain exhibit binding to E2 ubiquitin-conjugating enzymes (Ubc's) (Freemont (2000) Curr Biol. 10, R84-67). The above domain Zf-RING finger is different from that found in yeast CLASS E vacuolar sorting protein VPS27. However, there may be some functional conservation since VPS27 has also been linked to ubiqu'rtination processes and protein turnover, and the Zf-RING finger domain present in CRIO5 is usually found in proteins that are also involved in ubiquttination and proteasome protein degradation.<br>
The other proteins of the present invention belong to the group of Class-E vacuolar trafficking mutants. Certain mutants in yeast, known as "Class-E" mutants (Jones of a/., In: Yeast III, Cold Spring Harbor Laboratory Press, p363 - 470,1997), are unable to perform a correct sorting of proteins to the vacuole. Microscopical analysis reveals that these mutants contain large aberrant endosomal structures (Raymond et a/., Molecular Biology of the Cell 3,1389,1992), filled with proteins that are normally transited to the vacuole.<br>
CRYO1, CRY02 and CRYO3 all have a SNF7 domain (Pfam PF03357/IPR005024; Pfam<br>
database, Bateman et a)., (2004) Nucleic Acids Research Database Issue 32, D138-D141). Structurally, the three proteins belong to the family of CHMP proteins (Howard et al., (2001) J. Cell Sci. 114, 2395-2404). CRYO1 (SEQ ID NO 2) and CRY2 (SEQ ID NO 4) are isoforms of each other. CRYO1 and its plant homologues have not yet been functionally characterised, but they are related to yeast SNF7 (=DID1 =VPS32 =YLR025W). SNF7 mutants belong to the group of class E vacuolar trafficking mutants (Jones ot al., In: Yeast III, Cold Spring Harbor Laboratory Press, p363 - 470, 1997). The SNF7 mutant accumulates a prominent organelle distinct from the vacuole, containing large amounts of enzymes which are normally present in the vacuole such as the hydrdases CpY, PrA &amp; PrB. The protein is involved in derepression of SUC2 in response to glucose limitation. SNF7 mutants show a decrease in invertase derepression, a growth defect on raffinose, temperature-sensitive growth on glucose, and a spoliation defect in homozygous diploids. The SNF7 sugar related phenotype could be due to an altered turnover of a glucose sensor. These and other data suggest that the protein transport from the Golgi network and from the plasma membrane to the vacuole is interfered with. SNF7 forms a family of coiled-coil-forming proteins with vps20 and moslO. The proteins are involved in same trafficking step, endosome-to-vacuole transport, but probably participate in different cargo-specific events (Kranz et al., 2001).<br>
SEQ ID NO 6 (CRY03) is the plant homologue of yeast DID2 (=FT11 =YKR035W-A), another member of the class E vacuolar trafficking proteins. DID2 is related to SNF7; it has similar structural features, it may have a comparable function and possibly belongs to the same protein complex in yeast (Amerik at al.. Molecular Biology of the Cell, 11,3365 - 3380,2000). A<br>
human orthologue. CHMP1, is reported to be involved in membrane trafficking and localises to early endosomes (Howard et al., 2001). CHMP1 also localises to the nuclear matrix, thereby affecting chromatin structure and cell-cycle progression and furthermore interacts with the PcG protein Polycombllke (Pel) (Stauffer et al., 2001J Cell Set. 114,2363-93).<br>
Besides modifying tolerance to cold stress, the proteins may also be involved in protein transport and sorting (CRYO1 [SEQ ID NO 1/2], CRYO2 [SEQ ID NO 3/4], CRY03 [SEQ ID NO 5/6] and CRY04 [SEQ ID NO 7/8]), vacude formation, development or functioning (CRY01, CRYO2, CRYO3), in transcription and translation (CRYO3. CRYO5), in membrane fluidity (CRYO4) and in protein turnover (CRYO5).<br>
The proteins encoded by the nucleic acids identified by the screening method according to the<br>
present invention were hitherto unknown. Therefore, the invention also provides an isolated<br>
CRYO protein,	<br>
(a)	comprising the sequence as given in SEQ ID NO 2,4, 6,8 or 10;<br>
(b)	comprising a sequence having at least<br>
i.        76 %, alternatively 80%, preferably 90%, more preferably 95%, 96%, 97%, 98%<br>
or 99% sequence identity to the full length sequence as given in SEQ ID NO 2; ii.        55 %. alternatively 60%, 70%, 80%, preferably 90%, more preferably 95%.<br>
96%. 97%, 98% or 99% sequence identity to the full length sequence as given<br>
in SEQ ID NO 4; iii.       90.5 %, alternatively 90.6%, 90.7%, 90.8%, 90.9%, preferably 91%, 92%, 93%,<br>
94% more preferably 95%, 96%, 97%, 98% or 99% sequence identity to the full<br>
length sequence as given in SEQ ID NO 6; iv.       50 %, alternatively 60%, 70%, 60%, preferably 90%, more preferably 95%,<br>
96%. 97%, 98% or 99% sequence identity to the full length sequence as given<br>
in SEQ ID NO 8; v.       50 %, alternatively 60%, 70%, 80%, preferably 90%, more preferably 95%,<br>
96%, 97%, 98% or 99% sequence identity to the full length sequence as given<br>
in SEQ ID NO 10;<br>
(c)	comprising a substitution variant or insertion variant of (a);<br>
(d)	according to any of (a) to (c), comprising substitutions with corresponding naturally or<br>
non-naturally altered amino acids;<br>
A CRYO gene according to the present invention is any nucleic acid encoding a CRYO protein as defined above.<br>
"Homologues" of a protein encompass peptides, oligopeptides, polypeptidas, proteins and enzymes having amino acid substitutions, deletions and/or insertions relative to the unmodified protein in question and having similar biological and functional activity as the unmodified protein from which they are derived. To produce such homologues, amino acids of the protein may be replaced by other amino  acids  having  similar properties (such  as  similar hydrophobicity, hydrophilicity, antigenicity, propensity to form or break a-helical structures or 13-sheet structures). Conservative substitution tables are well known in the art (see for example Creighton (1984) Proteins. W.H. Freeman and Company). The homologues useful in the methods according to the invention have in the case of CRYO4 or CRYO5 at least 50% sequence identity or similarity (functional identity) to the unmodified protein, alternatively at least 60% sequence identity or similarity to an unmodified protein, or alternatively at least 70% sequence identity or similarity to an unmodified protein. Typically, the homologues of CRYO4 or CRY05 have at least 80% sequence identity or similarity to an unmodified protein, preferably at least 85% sequence identity or similarity, further preferably at least 90% sequence identity or similarity to an unmodified protein, most preferably at least 95% sequence identity or similarity to an unmodified protein. In the case of CRYO2, the homologues useful in the methods according to the invention have at least 55% sequence' identity or similarity (functional identity) to the unmodified protein, alternatively at least 60% sequence identity or similarity to an unmodified protein, or alternatively at least 70% sequence identity or similarity to an unmodified protein. Typically, the homologues of CRYO2 have at least 80% sequence identity or similarity to an unmodified protein, preferably at least 85% sequence identity or similarity, further preferably at least 90% sequence identity or similarity to an unmodified protein, most preferably at least 95% sequence identity or similarity to an unmodified protein. In the case of CRYO1, the homologues useful in the methods according to the invention have at least 76% sequence identity or similarity (functional identity) to the unmodified protein. Typically, the homologues of CRYO1 have at least 80% sequence identity or similarity to an unmodified protein, preferably at least 85% sequence identity or similarity, further preferably at least 90% sequence identity or similarity to an unmodified protein, most preferably at least 95% sequence identity or similarity to an unmodified protein.   In the case of CRYO3, the homologues useful in the methods according to the invention have at least 90.5% sequence identity or similarity (functional identity) to the unmodified protein. Typically, the homologues of CRYO3 have at least 91% sequence identity or similarity to an unmodified protein, preferably at least 92% sequence identity or similarity, further preferably at least 93% sequence identity or similarity to an unmodified protein, most preferably at least 95% sequence identity or similarity to an unmodified protein. Furthermore, homologues of a CRYO protein according to the present invention are capable of increasing the cold stress tolerance of yeast in an assay as outlined above.<br>
Two special forms of homology, orthologous and paralogous, are evolutionary concepts used to describe ancestral relationships of genes. The term "paralogous" relates to gene-duplications within the genome of a species leading to paralogous genes. The term "orthologous* relates to homologous genes in different organisms due to ancestral relationship. The term 'homologues' as used herein also encompasses paraloguea and orthologues of the proteins useful in the methods according to the invention.<br>
Orthologues in other plant species may easily be found by performing a so-called reciprocal blast search. This may be done by a first blast involving blasting the sequence in question (any one of SEQ ID NO 1 to 10) against any sequence databases, such as the publicly available NCBI database which may be found at: http://www.ncbi.nlm.nih.gov. If orthologues in rice are sought, the sequence in quesb'on would be blasted against, for example, the 28,469 full-length cDNA clones from Oryza saliva Nipponbare available at NCBI. BLASTn Aiay be used when starting from nudeotides or TBLASTX when starting from the protein, with standard default values. The blast results may be filtered. The full-length sequences of either the filtered results or the non-filtered results are then blasted back (second blast) against the sequence in question. The results of the first and second blasts are then compared. True orthologues are those that match again with the query gene. In the case of large families, ClustalW and the neighbour joining method is used to construct a phylogenetic tree to help visualize the clustering.<br>
Homologous proteins can be grouped in "protein families'. A protein family can be defined by functional and sequence similarity analysis, such as, for example, Clustal W, A neighbour-joining tree of the proteins homologous to a protein of interest can be generated by the Clustal W program and gives a good overview of their structural and ancestral relationships.<br>
The CRY01, CRY02 or CRY3 proteins comprise a SNF7 domain and can be regarded as members of the same protein famly as human CHMP1 and yeast SNF7. The SNF7 domain (Ram PF03357) occurs in a group of proteins involved in protein sorting and transport from the endosome to the vacuole/lysosome in eukaryotic cells. In the Interpro database, the SNF7 family is described as a family of eukaryotic proteins which are variously described as either hypothetical protein, developmental protein or related to yeast SNF7. The family contains human CHMP1. CHMP1 (CHromatin Modifying Protein; CHarged Multivesicular body Protein) is encoded by an alternative open reading frame in the PRSM1 gene and is conserved in both complex and simple eukaryotes. CHMP1 contains a predicted bipartite nuclear localisation signal and distributes as distinct forms to the cytoplasm and the nuclear matrix in all cell lines<br>
tested. Human CHMP1 is strongly implicated in multivesicular body formation. A multivesicular body is a vesicle-filled endosome that targets proteins to the interior of rysosomes. Immunocytochemistry and biochemical fractionatton localise CHMP1 to early endosomes and CHMP1 physically interacts with SKD1/VPS4, a highly conserved protein directly linked to multivesicular body sorting in yeast Similar to the action of a mutant SKD1 protein, over expression of a fusion derivative of human CHMP1 dilates endosomal compartments and disrupts the normal distribution of several endosomal markers. Genetic studies in Saccharomycos coravisiao further support a conserved role of CHMP1 in vesicle trafficking. Deletion of CHM1, the budding yeast homolog of CHMP1, results in defective sorting of carboxypeptidases S and Y and produces abnormal, multi-lamellar prevacuolar compartments. This phenotype classifies CHM1 as a member of the class E vacuolar protein sorting genes. In this CHMP family other CHMP proteins can be found, such as the yeast proteins Chmf p, ChmZp, Vps24p, ChmSp and Chm6p, or the human proteins AF281064, AF042384, AF151842. AF19226, AF161483, AF132968 and AW965590 (Howard et al., 2001). All these proteins constitute a family of structurally related proteins with a similar size and charge distribution (basic N-terminus and acidic C-terminus) and sequence conservation throughout the complete protein sequence (Howard et at., 2001). This structural conservation indicates also a functional conservation: CHM gene products are involved in correct sorting of carboxypeptidase Y and can be assayed in a pulse chase experiment as described by Howard etal. (2001).<br>
CRYO4 belongs to the same family as yeast SEC14. CRYO4 comprises a SEC14 domain as defined in the SMART database (Schultz et al. (1998) Proc. Natl. Acad. Sci. USA 95, 5857-5864; Letunic et al. (2004) Nucleic Acids Res 32, D142-D144): the SEC14 domain is (bund in homologues of a S. cerevisiae phosphatkJylinositol transfer protein (Sec14p) and in RhoGAPs, RhoGEFs and the RasGAP, neurofibromin (NF1). It is also reported to be a lipid-binding domain. The SEC14 domain of Dbl is known to associate with G protein beta/gamma subunits. This domain is also described in the Interpro database (Mulder et al., (2003) Mud. Acids. Res. 31,315-318) (IPR001251), where the proteins comprising this domain are grouped as a family of various retinaldehyde/retlnal-binding proteins that may be functional components of the visual cycle in animals. Cellular retinaldehyde-binding protein (CRALBP) carries 11-cis-retinol or 11-cis-retinaldehyde as endogenous ligands and may function as a substrate carrier protein that modulates interaction of these retinoids with visual cycle enzymes. The mullidomain protein Trio binds the LAR transmembrane tyrosine phosphatase, contains a protein kinase domain, and has separate rac-specific and rho-specific guanine nucleotide exchange factor domains. Trio is a multifunctional protein that integrates and amplifies signals involved in coordinating actin remodeling, which is necessary for cell migration and growth. Other<br>
members of the family are transfer proteins that include, guanine nudeotide exchange factor that may function as an effector of RAC1, phosphatidylinosttol/phosphatidytcholine transfer protein that is required for the transport of secretory proteins from the Golgi complex and alpha-tocopherol transfer protein that enhances the transfer of the ligand between separate membranes. HomcHogues useful in the present invention comprise a SEC14 domain and exhibit tipid transfer activity (a suitable assay Is described by Jouannic et al., Eur. J. Biochem. 258, 402410 (1998)) and include for example the Arabidopsis proteins At1g72160, At4g09160, At1g22530, At1g72150, At3g51670, At1g30690, the rice protein BAB86220 and the maize protein encoded by AY107978.<br>
CRYO5 comprises a RING type zinc finger domain in the carboxy-terminal part of the protein, in addition there is a conserved sequence spanning amino acid (AA) 15 to 305 of SEQ ID NO 10 that corresponds to a Pfam-B_23829 domain (which domain is postulated to be associated with the C3HC4 type Zn-finger domain) and a sequence from AA 425 to 473, corresponding to Pfam-B_2377. The CRYO5 sequence corresponding to this Pfam-B_2377 domain comprises a conserved stretch of amino acids that is also found in other homologous plant proteins and includes of the sequence HDQHRDMRLDIDNMSYEELLALEERIG, in which less than 7 mismatches may occur among the various homologues. Plant homologues of CRYO5 constitute a new class of proteins (CRYO5-like proteins) characterised by the presence of a serine rich region in the N-terminal half of the protein, an acidic region that comprises the conserved sequence signature hereabove and the RING finger domain. Exemplary homologues include sequences NPJ96626. NP_974832, NP_568462.2, and the protein encoded by AK066069.<br>
Advantageously, homologous proteins belonging to these families and/or comprising one or more of these domains may be useful in the methods of the present invention for conferring abiotic stress tolerance and in particular cold stress tolerance to plant cells or yeast.<br>
Two polypeptides or nucleic acids are said to be "identical" if the sequence of amino add residues or nudeotides, respectively, in the two sequences is the same when optimally aligned. Sequence comparisons between two (or more) polypeptide or nucleic acids are typically performed by comparing sequences of the two sequences over a "comparison window" to identify and compare local regions of sequence similarity. A "comparison window", as used herein, refers to a segment of at least about 20 contiguous positions, usually about 50 to about 200, more usually about 100 to about 150, in which a sequence may be compared to a reference sequence of the same number of contiguous positions after the two sequences are optimally aligned. Optimal alignment of sequences for comparison may be conducted by the<br>
local homology algorithm of Smith and Waterman (Adv. Appl. Math. 2, 482. 1981), by the homology alignment algorithm of Needleman and Wunsch (J. Mol. Biol. 48 443,1970), by the search for similarity method of Pearson and Llpman (Proc. Nat Acad. Sci. 85,2444,1988), by computerized Implementations of these algorithms (GAP, BESTFIT, BLAST, FASTA, and TFASTA in the Wisconsin Genetics Software Package, Genetics Computer Group (GCG), 575 Science Dr., Madison, Wl), or by inspection. When the Needleman-Wunsch algorithm is used, a gap opening penalty of 10 or 11, a gap extension penalty of 0.5 or 1 is used and where possible, full-length sequences are compared to each other.<br>
The term 'derivatives' refers to peptides, oligopeptides, polypeptkJes, proteins and enzymes which may comprise substitutions, deletions or additions of naturally and non-naturally occurring amino add residues compared to the amino acid of a naturally-occurring form of the proteins as presented in SEQ ID NO 2. 4, 6, 8 or 10. 'Derivatives' of a protein comprise proteins in which amino acid residues are substituted by corresponding naturally or non-naturally altered amino acids. "Derivatives" of a protein encompass peptides, oligopeptides, polypeptides, proteins and enzymes which may comprise naturally occurring altered, (such as glycosylated, acylated, myristoylated or phosphorylated amino acids) or non-naturally occurring amino acid residues (such as biotinylated amino acids, or amino acids modified after CNBr treatment) compared to the amino acid of a naturally-occurring form of the polypeptide. 'Derivatives' of a protein also encompass proteins carrying post-translational modifications. A derivative may also comprise one or more non-amino acid substttuents compared to the amino acid from which it is derived, for example a reporter molecule or other ligand, covalently or non-covalently bound to the amino acid such as, for example, a reporter molecule which is bound to facilitate its detection, and non-naturally occurring amino acid residues relative to the amino acid of a naturally-occurring protein. "Substitutional variants' of a protein are those in which at least one residue in an amino acid has been removed and a different residue inserted in its place. Amino acid substitutions are typically of single residues, but may be clustered depending upon functional constraints placed upon the polypeptide; insertions will usually be of the order of about 1-10 amino acid residues, and deletions will range from about 1-20 residues. Preferably, amino acid substitutions comprise conservative amino acid substitutions. "Insertional variants* of a protein are those in which one or more amino acid residues are introduced into a predetermined site in said protein. Insertions can comprise amino-terminal and/or carboxy-terminal fusions as well as intra-sequence insertions of single or multiple amino acids. Generally, insertions within the amino acid will be smaller than amino- or carboxy-terminal fusions, in the order of about 1 to 10 residues. Examples of amino- or carboxy-terminal fusion proteins or peptides include the binding domain or activation domain of a transcriptions! activator as used in the yeast two-hybrid system, phage coat proteins,<br>
(histidineV-tag. glutathione S-transferase-tag, protein A, maltose-binding protein, dihydrofdate reductase, Tag-100 epitope, c-myc epKope. FLAG'-epitope, lacZ, CMP (calmodulin-binding peptide), HA epitope, protein C epitope and VSV epitope. "Functional fragments" or "deletion variants" of a protein are characterised by the removal of one or more amino acids from the protein, such that the remaining fragment still retains the biological activity of the unmodified protein, for example the capacity of conferring cold stress to yeast in an assay detailed in examples 2 and 3. Such "functional fragments' of a protein encompasses at least fifteen contiguous amino add residues of a protein, in case of a functional fragment the minimum size being a sequence of sufficient size to provide this sequence with at least a comparable function and/or activity to the original sequence which was tmncated, while the maximum size is not critical. Typically, the truncated amino acid will range from about 5 to about 60 amino acids in length. "Immunologically active" refers to molecules or specific fragments thereof, such as specific epitopes or haptens, that are recognised by (i.e. that bind to) antibodies. Specific epitopes may be determined using, for example, peptide-scanning techniques as described in Geysen et al., (Chem Bid. 3, 679-688, 1996). Functional .fragments can also include those comprising an epitope which is specific for the proteins according to the invention.<br>
Preferably, the derivatives, functional fragments, substitution, deletion or insertion variants of a CRYO protein have at least the same or better functional activity than the unmodified protein, such as the capability of increasing the cold stress tolerance of yeast The functional activity can be tested with for example the screening method as described above or with methods described for related proteins.<br>
Amino acid variants of a protein may readily be made using peptide synthetic techniques well known in the art, such as solid phase peptide synthesis and the like, or by recombinant ONA manipulations. The manipulation of ONA sequences to produce substitution, insertion or deletion variants of a protein are well known in the art. For example, techniques for making substitution mutations at predetermined sites in DMA are well known to those skilled in the art and include M13 mutagenesis, T7-Gen in vitro mutagenesis (USB, Cleveland, OH), QuickChange Site Directed mutagenesis (Stratagene, San Diego, CA), PCR-mediated site-directed mutagenesis or other site-directed mutagenesis protocols.<br>
Another embodiment of the present invention provides nucleic acids obtainable by the screening method according to the present invention, which nucleic acids can be used to modify stress tolerance or resistance in plants and/or yeast. The screening method according to the invention identified several nucleic acids hitherto unknown. The present invention<br>
therefore also provides an isolated nucleic acid encoding a protein as defined above, the complement thereof or a part thereof.<br>
The terms "nucleic acid(s)*, "nucleotide sequence(s)', "gene(s)". "polynudeoUde(s)' and "nucleic acid molecule(s)" are used herein interchangeably to refer to ribonudeotides or deoxyribonucteotides or a combination of both, in a polymeric form of any length. The terms also include double-stranded and single-stranded DMA and RNA. Also included are known nucleotide modifications such as methylation, cydization and 'caps' and substitution of one or more naturally occurring nudeotides with an analogue such as inosine. The terms also encompass peptide nucleic adds (PNAs).<br>
Advantageously, the nucleic acids according to the invention may be produced using recombinant or synthetic means, such as, for example, PCR cloning mechanisms. Generally, such techniques as defined herein are well known in the art, for example as described in Sambrook et al. (Molecular Cloning: a Laboratory Manual, 2001). Polynucleotidas may also be synthesized by well-known techniques as described in the technical literature. See, e.g., Caruthers et al., Cold Spring Harbor Symp. Quant Biol. 47,411-418 (1982), and Adarns et al.. J. Am. Chem. Soc. 105, 661 (1963). Double stranded DMA fragments may then be obtained either by synthesizing the complementary strand and annealing the strands together under appropriate conditions, or by adding the complementary strand using DNA polymerase with an appropriate primer sequence.<br>
A nucleotide sequence encoding a protein (gene, coding sequence, open reading frame or ORF) is a nucleotide sequence that can be transcribed into mRNA and/or translated into a polypeptide when present in an expressible format, i.e. when the coding sequence or ORF is placed under the control of appropriate control sequences or regulatory sequences. A coding sequence or ORF is bounded by a 5' translation start codon and a 3' translation stop codon. A coding sequence or ORF can include, but is not limited to RNA, mRNA, cDNA, recombinant nucleotide sequences, synthetically manufactured nucleotide sequences or genomic DNA. The coding sequence or ORF can be interrupted by Intervening nucleic acids.<br>
By "expressible format" is meant that the isolated nucleic acid molecule is in a form suitable for being transcribed into mRNA and/or translated to produce a protein, either constitutively or following induction by an intracellular or extracellular signal, such as an environmental stimulus or stress {mitogens, anoxia, hypoxia, temperature, salt, light, dehydration, etc) or a chemical compound such as IPTG (isopropyl-8-D-thiogalactopyranoside), or such as an antibiotic (tetracydine, ampicillin, rifampfcin, kanamycin), hormone (e.g. gibberellin, auxin, cytokinin,<br>
glucocorticoid, brassinosteroid, ethylene, abscisic acid etc), hormone analogue (indolacetic acid (IAA), 2,4-D, etc), metal (zinc, copper, iron, etc), or dexamethasone, amongst others. As will be known to those skilled in the art, expression of a functional protein may also require one or more post-translational modifications, such as glycosylation, phosphorylation, dephosphorylation, or one or more protein-protein interactions, amongst others. All such processes are Included within the scope of the term "expressible format".<br>
Genes and coding sequences essentially encoding the same protein but isolated from different sources can consist of substantially divergent nucleic acids. Reciprocally, substantially divergent nudeic acids can be designed to effect expression of essentially the same protein. These nucleic acids are the result of e.g. the existence of different alleles of a given gene, or of the degeneracy of the genetic code or of differences in codon usage. Differences in preferred codon usage are illustrated in http://www.kazusa.or.jp/codon. Allelic variants are further defined  as  to  comprise  single  nudeotide  polymorphisms (SNPs)  as  well  as  small insertion/deletion polymorphisms (INDELs, having a size of usually less than 100 bp). SNPs and INDELs form the largest set of sequence variants in naturally occurring polymorphic strains of most organisms. Additionally or alternatively, in particular conventional breeding programs, such as for example marker assisted breeding, it is sometimes practical to introduce allelic variation in the plants by mutagenic treatment of a plant   One suitable mutagenic method is EMS mutagenesis. Identification of allelic variants then takes place by, for example, PCR.   This is followed by a selection step for selection of superior allelic variants of the sequence in question and which give rise to altered growth characteristics.   Selection is typically carried out by monitoring growth performance of plants containing different allelic variants of the sequence in question (for example SEQ ID NO 1, 3, 5, 7 or 9).   Monitoring growth performance can be done in a greenhouse or in the field.   Further optional steps include crossing plants in which the superior allelic variant was identified with another plant. This could be used, for example, to make a combination of interesting phenotypic features. According to another aspect of the present invention, advantage may be taken of the nudeotide sequence capable of modulating expression of a nucleic acid encoding a CRYO protein (such as SEQ ID NO 2, 4, 6, 8 or 10} in breeding programs. For example, in such a program, a DNA marker is identified which may be genetically linked to the gene capable of modulating the activity of a protein of interest (for example SEQ ID NO 2, 4, 6, 8 or 10) in a plant (which gene can be the gene encoding a protein of interest or another gene capable of influencing the activity of a protein of interest). This DMA marker is then used in breeding programs to select plants having altered growth characteristics. Many techniques are nowadays available to identify SNPs and/or INDELs.<br>
Also within the scope of the present Invention are nucleic acids which are alternative splice variants of a CRYO protein encoded by any one of SEQ 10 NO 1, 3, 5, 7 or 9. The term "alternative splice variant" as used herein encompasses variants of a nucleic acid encoding a CRYO protein in which selected introns and exons have been excised, replaced or added, optionally in response to specific signals. Methods for the determination of intron and exon positions in a genomic sequence of a gene are known in the art Splice variants all originate from one and the same pre-mRNA. The splicing occurs after transcription of the gene but before mRNA translation, and is usually regulated in a tissue specific or temporal way (for example, such that the mRNA has tissue-specific expression, see for example Burge et si., (1999). Splicing of precursors to mRNAs by the spliceosomes. In The RNA World II, Gesteland. Cech, and Atkins, eds. (Cold Spring Harbor, NY. Cold Spring Harbor Laboratory Press), pp. 525-560). Preferred variants will be ones in which the biological activity of the protein remains unaffected, which can be achieved by selectively retaining functional segments of the protein. Methods for making such splice variants are well known in the art, for example by RNAi (Celotto and Graveley (2002) RNA 8, 718-724), with ribozymes, by introducing mutations in the gene, by modifying the spliceosome or by modifying the signal transduction pathways inducing alternative splicing.<br>
The invention furthermore encompasses nucleic acids that are capable of hybridising with a nucleic acid encoding a protein as represented by SEQ ID NO 2, 4, 6, 8 or 10. The term "hybridisation" as used herein is the process wherein substantially homologous complementary nudeotide sequences anneal to each other. The hybridisation process can occur entirely in solution, i.e. both complementary nucleic acids are in solution. Tools in molecular biology relying on such a process include the polymerase chain reaction (PCR; and all methods based thereon), subtractive hybridisation, random primer extension, nudease S1 mapping, primer extension, reverse transcription, cONA synthesis, differential display of RNAs, and DNA sequence determination. The hybridisation process can also occur with one of the complementary nucleic acids immobilised to a matrix such as magnetic beads, Sepharose beads or any other resin. Tools in molecular biology relying on such a process include the isolation of poly (A+) mRNA. The hybridisation process can furthermore occur with one of the complementary nucleic acids immobilised to a solid support such as a nitro-cellulose or nylon membrane or immobilised by e.g. photolithography to e.g. a siliceous glass support (the latter known as nucleic acid arrays or microarrays or as nucleic acid chips). Tools in molecular biology relying on such a process include RNA and DNA gel blot analysis, colony hybridisation, plaque hybridisation, in sttu hybridisation and microarray hybridisation. In order to allow hybridisation to occur, the nucleic acid molecules are generally thermally or chemically denatured to melt a double strand into two single strands and/or to remove hairpins or other<br>
secondary structures from single stranded nucleic acids. The stringency of hybridisation is influenced by conditions such as temperature, salt concentration and hybridisation buffer composition. High stringency conditions for hybridisation include high temperature and/or low salt concentration (salts include NaCI and Na3-citrate) and/or the inclusion of formamida in the hybridisation buffer and/or lowering the concentration of compounds such as SOS (detergent) in the hybridisation buffer and/or exclusion of compounds such as dextran sulphate or polyethylene glycol (promoting molecular crowding) from the hybridisation buffer. Conventional hybridisation conditions are described in, for example, Sambrook (2001) Molecular Cloning: a laboratory manual (3rd Edition Cold Spring Harbor Laboratory Press, CSH, New York), but the skilled craftsman will appreciate that numerous different hybridisation conditions can be designed in function of the known or the expected homology and/or length of the nucleic acid. Typical conditions for "stringent hybridisation" are for example hybridising at a temperature of 60"C followed by washes in 2XSSC, 0.1XSDS, and 1X SSC, 0.1X SDS.<br>
.Advantageously, the method according to the present invention may also be practised using portions of a DNA or nucleic acid, which portions encode polypeptides that retain CRYO activity, i.e. a similar biological function to those encoding proteins represented in SEQ ID NO: 2,4, 6, 8 or 10. Portions of a DNA sequence refer to a piece of DNA derived or prepared from an original (larger) DNA molecule, which DNA portion, when expressed in a plant, gives rise to plants having modified growth characteristics. The portion may comprise many genes, with or without additional control elements, or may contain just spacer sequences etc.<br>
The term "part of a sequence" means a truncated sequence of the original sequence referred to. The truncated nucleic acid sequence can vary widely in length; the minimum size being a sequence of sufficient size to provide a sequence with at least a comparable function and/or activity or the original sequence referred to, while the maximum size is not critical. In some applications, the maximum size usually is not substantially greater than that required to provide the desired activity and/or functton(s) of the original sequence. Typically, the truncated nudeotide sequence will range from about 15 to about 180 nudeotides in length. More typically, however, the sequence will be a maximum of about 150 nudeotides in length, preferably a maximum of about 180 nudeotides. It is usually desirable to select sequences of at least about 30, 36 or 45 nudeotides, up to a maximum of about 60 or 75 nudeotides.<br>
DNA sequences as defined in the current invention can also be interrupted by intervening sequences. With "intervening sequences" is meant any nucleic acid which disrupts a coding sequence in the DNA sequence of interest or which disrupts the expressible format of a DNA sequence comprising the DNA sequence of interest. Removal of intervening sequences<br>
restores the coding sequence or said expressible formal Examples of intervening sequences include introns and mobilisable ONA sequences such as transposons. With "mobilisable DMA sequence' is meant any DNA sequence that can be mobilized as the result of a recombination event.<br>
The present invention also relates to a recombinant genetic construct comprising a nucleic acid according to the invention.   The genetic constructs facilitate the introduction and/or expression and/or maintenance of a nudeotide sequence as defined above into a plant cell, tissue or organ. Preferably, the genetic construct comprises (i)       an isolated nucleic acid encoding a plant protein<br>
(a)	comprising the sequence as given in SEQ ID NO 2,4,6,8 or 10;<br>
(b)	comprising a sequence having at least 50 %, alternatively 60%, 70%, 80%,<br>
preferably 90%, more preferably 95%, 96%, 97%, 98% or 99% sequence identity to<br>
the full length sequence as given in SEQ ID NO 2,4,6,8 or 10;<br>
(c)	comprising a substitution variant or insertion variant of (a);<br>
(d)	according to any of (a) to (c), comprising substitutions with corresponding naturally<br>
or non-naturally altered amino acids;<br>
(i!)      a regulatory element operably linked to the nucleic acid of (i), which regulatory element<br>
is a plant and/or yeast expressible promoter; and optionally (iii)      a transcription termination sequence.<br>
The nucleic acid construct can be an expression vector wherein the nucleic acid is operably linked to one or more regulatory elements allowing expression in proKaryotic and/or eukaryotic host cells. The vector may be commercially available, suitable for transforming into plants and suitable for expression of the gene of interest in the transformed cells.<br>
Advantageously, any nucleic acid obtainable by the screening method according to the present invention can be used in the construct; preferably a nucleic acid as defined in any of (a) to (d) above is used.<br>
The term "operably linked" as used herein refers to a functional linkage between the regulatory element and the gene of Interest, such that the regulatory element is able to initiate transcription of the gene of interest.<br>
As used herein, the term "plant-expressible promoter" refers to a promoter that is capable of driving transcription in a plant cell. This not only includes any promoter of plant origin, such as the natural promoter of the transcribed DNA sequence, but also any promoter of non-plant<br>
origin which is capable of directing transcription in a plant cell. The promoter may also be an artificial or synthetic promoter. The term "plant-expressible promoter* includes, but is not restricted to, constitutive, inducibte, organ-, tissue- or cell-specific and/or developmental^ regulated promoters. The terms 'regulatory element", 'control sequence*, "promoter* are all used herein interchangeably and, taken in a broad context, refer to regulatory nucleic acids capable of effecting expression of the sequences to which they are ligated.<br>
Advantageously, any type of promoter may be used to drive expression of the nucleic acid encoding a CRYO protein in a plant More specifically, a constitutive promoter can be, but is not restricted to, one of the following: a 35S promoter (Odell et a/.. Nature 313, 482 -493, 1985), a 35S'3 promoter (Hull and Howell, Virology 86, 482 - 493, 1987), the promoter of the nopaline synthase gene ("PNOS") of the Ti-plasmid (Herrera-Estrella, Nature 303. 209-213, 1983) or the promoter of the octopine synthase gene ("POCS*, De Grave et al., J. Mot. Appl. Genet. 1,499 - 511,1982). It is dear that other constitutive promoters can be used to obtain similar effects. A meristenvspecific promoter, such as the mr (ribonudeotide reductase), cdc2a promoter and the cyc07 promoter, could be used to effect expression in all growing parts of the plant, thereby increasing cell proliferation, which in turn would increase yield or biomass. If the desired outcome would be to influence seed characteristics, such as the storage capacity, seed size, seed number, biomass etc., then a seed-specific promoter, such as p2S2, pPROLAMIN. pOLEOSIN could be selected. An aleurone-specific promoter may be selected in order to increase growth at the moment of germination, thereby increasing the transport of sugars to the embryo. An inflorescence-specific promoter, such as pLEAFY, may be utilised if the desired outcome would be to modify the number of flower organs. To produce male-sterile plants one would need an anther specific promoter. To impact on flower architecture for example petal size, one could choose a petal-specific promoter. If the desired outcome would be to modify growth and/or developmental characteristics in particular organs, then the choice of the promoter would depend on the organ to be modified. For example, use of a root-specific promoter would lead to increased growth and/or increased biomass or yield of the root and/or phenotypic alteration of the root. This would be particularly important where it is the root itself that is the desired end product, such crops include sugar beet, turnip, carrot, and potato. A fruit-specific promoter may be used to modify, for example, the strength of the outer skin of the fruit or to increase the size of the fruit. A green tissue-specific promoter may be used to increase leaf size. A cell wall-specific promoter may be used to increase the rigidity of the cell wall, thereby increasing pathogen resistance. An anther-specific promoter may be used to produce male-sterile plants. A vascular-specific promoter may be used to increase transport from leaves to seeds. A nodule-specific promoter may be used to increase the nitrogen fixing capabilities of a plant, thereby increasing the nutrient levels in a plant A stress-<br>
inducible promoter may also be used to drive expression of a nucleic acid to increase membrane integrity during conditions of stress. A stress inducible promoter such as the water stress induced promoter WSI18, the drought stress induced Trg-31 promoter, the ABA related promoter rab21 or any other promoter which is induced under a particular stress condition such as temperature stress (cold, freezing, heat) or osmotic stress, or drought stress or oxidative stress or btotic stress can be used to drive expression of a CRYO gene.<br>
If the desired outcome would be to influence the cold tolerance of a plant under adverse conditions, then a cold-inducible promoter such as, for example, prd29, pws18 or pcr 15 could be selected.<br>
Similarly, the term "yeast-expressible promoter* refers to a promoter that is capable of driving transcription in a yeast cell and encompasses natural yeast promoters as well as other promoter sequences capable of driving expression in yeast cells. Suitable promoters for expression in yeast are known in the art, see for example Currant Protocols in Molecular Biology. Unit 13 (Ausubel ef a/., 1994) and the Guide to Yeast Genetics and Molecular Biology (Guthrie and Fink, 1991).<br>
The reoombinant genetic construct according to the present invention may include further regulatory or other sequences from other genes. Encompassed are transcriptional regulatory sequences derived from a classical eukaryotic genomlc gene (including the TATA box which is required for accurate transcription initiation, with or without a CCAAT box sequence) and additional regulatory elements (i.e. upstream activating sequences, enhancers and silencers) which alter gene expression in response to developmental and/or external stimuli, or in a tissue-specific manner. Also included is a transcriptional regulatory sequence of a classical pro karyotic gene, in which case it may include a -35 box sequence and/or -10 box transcriptional regulatory sequences. Regulatory elements also encompass a synthetic fusion molecule or derivative which confers, activates or enhances expression of a nucleic acid molecule in a cell, tissue or organ.<br>
Optionally, one or more terminator sequences may also be used in the construct introduced into a plant The term "terminator" encompasses a control sequence which is a DNA sequence, at the end of a transcriptional unit, which signals 3' processing and polyadenylation of a primary transcript and termination of transcription. Additional regulatory elements may include transcriptional as well as translational enhancers. Those skilled in the art will be aware of terminator and enhancer sequences which may be suitable for use in performing the invention.<br>
Furthermore, the recombinant nucleic acid can be constructed and employed to target the gene product of the nucleic acid of the invention to a specific intracellular compartment within a plant cell or to direct a protein to the extracellular environment This can generally be obtained by operably joining a ONA sequence encoding a transit or signal peptide to the recombinant nucleic acid.<br>
The genetic constructs of the invention may further Include an origin of replication sequence which is required for maintenance and/or replication in a specific cell type, for example a bacterial cell, when the genetic construct is required to be maintained as an episomal genetic element (e.g. plasmid or cosmid molecule) in a cell. Preferred origins of replication include, but are not limited to, the f1 -ori and colE1 origins of replication.<br>
The genetic construct may optionally comprise a selectable marker gene. As used herein, the term "selectable marker gene' includes any gene which confers a phenotype on a cell in which it is expressed to facilitate the identification and/or selection of cells which are transfected or transformed with a genetic construct of the invention or a derivative thereof. Suitable markers may be selected from markers that confer antibiotic or herbicide resistance. Cells containing the recombinant DNA will thus be able to survive in the presence of antibiotic or herbicide concentrations that kill untransformed cells. Examples of selectable marker genes include the bar gene which provides resistance to the herbicide Basta; the ampfcillln resistance gene (Amp'), the tetracycline resistance gene (Tcf), the bacterial kanamycin resistance gene (Kan'), the phosphinothricin resistance gene, the neomycin phosphotransferase gene (nptll), the hygromycin resistance gene, and the chloramphenicol acetyltransferase (CAT) gene. Visual markers, such as the Green Fluorescent Protein (GFP, Haseloff et al., Nature 334, 585-591, 1997), β-glucuronidase (GUS), and luciferase, may also be used as selectable markers.<br>
According to another embodiment, the present invention relates to the use of the nucleic acids encoding a protein of the present invention or the use of such a protein as selectable marker gene in plants or other organisms. More preferably, the present invention also relates to the use of a gene coding for a CRYO protein as defined above as selectable marker gene, selection taking place by treating with a stress condition such as a sub-optimal growth temperature.<br>
The nucleic acids obtainable by the screening method as described herein encode proteins that support faster growth of yeast under stress conditions, relative to wild type yeast, therefore it is likely, since these nucleic acids originate from plants, that modulation of expression of<br>
these nucleic acids upon introduction into plants, will also support fester growth of plants under stress conditions when compared to corresponding wild type plants. Therefore the present invention provides a method for increasing abiotic stress tolerance of plants, comprising the steps of Introducing a genetic modification in these plants and selecting for modulated expression in these plants of a nucleic acid sequence encoding a protein<br>
(a)	comprising the sequence as given in SEQ ID NO 2,4,6,8 or 10;<br>
(b)	comprising a sequence having at least 50 %, alternatively 60%, 70%, 80%, preferably<br>
90%, more preferably 95%, 96%, 97%, 98% or 99% sequence identity to the full<br>
length sequence as given in SEQ ID NO 2,4,6,8 or 10;<br>
(c)	comprising a substitution variant or insertion variant of (a);<br>
(d)	according to any of (a) to (c), comprising substitutions with corresponding naturally or<br>
non-naturally altered amino adds;<br>
(e)	comprising a functional fragment of any of (a) to (d).<br>
Preferably this protein increases cold stress tolerance of yeast. Similarly, the present invention provides a method for increasing stress tolerance of yeast, preferably to cold stress, comprising modulating expression in plants of a nucleic acid sequence encoding a CRYO protein and/or modulating activity of a CRYO protein.<br>
Advantageously, proteins homologous to a CRYO protein can also be used in the methods of the present invention.   The invention thus provides a method for increasing abiotic stress tolerance of plants, comprising the steps of introducing a genetic modification in these plants and selecting for modulated expression in these plants of a nucleic acid sequence encoding a protein chosen from the group of (i)       proteins belonging to the family of CHMP proteins (ii)      proteins comprising a SEC14 domain and exhibiting lipid transfer activity (iii)      CRY05 like plant proteins comprising a RING finger domain, a serine rich domain and       an       acid       domain       which       comprises       the       signature "HDQHRDMRLDIDNMSYFELLALEERIG", in which no more than 6 substitutions may occur.<br>
'Increased stress tolerance" as used herein comprises, for any given stress, increasing tolerance in plants or yeast to that particular stress, whether those plants or yeast already have some degree of tolerance to the particular stress or whether that plant or yeast is being provided with tolerance to that stress anew.<br>
Preferably, the increased tolerance is to at least one of temperature stress, osmotic stress, drought stress, salt stress or oxidative stress, more preferably cold stress.<br>
The terms tolerance" and "resistance" as used herein encompass protection against stress ranging from a delay to substantially a complete inhibition of alteration in cellular metabolism, reduced cell growth and/or cell death caused by environmental stress conditions. Advantageously, transgenic plants or yeasts obtained by the methods of the present invention are tolerant or resistant to environmental stress conditions.<br>
The term 'environmental stress" as used herein encompasses stress factors such as drought stress (water, dehydration), osmotic stress, salt stress, temperature stress (due to for example heat or frost). Temperature stress" which includes "cold stress", "chilling stress', freezing stress" or "heat stress* is a stress induced by sub-optimal or supra-optimal growth temperatures for a particular organism. Optimal growth temperature ranges may be readily determined or would be known to those skilled in the art. "Osmotic stress" is any stress associated with for induced by loss of water, reduced turgor or reduced water content of a cell, tissue, organ or whole plant. "Drought stress" refers to any stress which is induced by or associated with the deprivation of water or reduced supply of water to a cell, tissue, organ or organism. The term "salt-stress" refers to any stress which is associated with or induced by elevated concentrations of salt or ions in general and which result in a perturbation in the osmotic potential of the intracellular or extracellular environment of a cell. "Oxidative stress" occurs in situations of cold stress combined with intensive light, in situations of ozone stress, in cases of necrosis as a result of pathogen infection or wounding, in cases of senescence and due to application of certain herbicides (like atrazine or paraquat).<br>
According to a preferred feature of the invention, the stress is cold stress. Advantageously the results of testing for tolerance or resistance to environmental conditions in the yeast cells give a reliable measure of the capability of the inserted coding sequence or gene to induce tolerance or resistance to environmental stress in plants. The capacity of an isolated nudeic acid to confer tolerance or resistance to environmental stress tolerance to plants can be tested according to methods well-known in the art, see for example, Physical Stresses in Plants: Genes and Their Products for Tolerance. S. Grillo (Editor), A. Leone (Editor) (June 1996), Springer Veriag; ISBN: 3540613471; Handbook of Plant and Crop Stress. Mohammad Peassarakli (Editor), Marcel Dekker, ISBN: 0824789873; The Physiology of Plants Under Stress; Abiotic Factors. Erik T. Nilsen, David M. Orcutt (Contributor), Eric T. Nilsen. 2nd edition (October 1996), John Wiley &amp; Sons; ISBN: 047131526; Drought, Salt, Cold and Heat Stress: Molecular Responses in Higher Plants (Biotechnology Intelligence Unit). Kazuo Shinozaki (Editor), Kazuko Yamaguchi-Shinozaki (Editor) (1999). R G Landes Co; ISBN: 1570595631; Plants Under Stress: Biochemistry, Physiology and Ecology and Their Application to Plant<br>
Improvement (Society for Experimental Biology Seminar Serie). Hamlyn G. Jones, T.J. Flowers. M.B. Jones (Editor). (September 1989). Cambridge Univ. Pr. (Short); ISBN: 0521344239; Plant Adaptation to Environmental Stress. Leslie Fowden, Terry Mansfield, John Stoddart (Editor) (October 1993) Chapman &amp; Hall; ISBN: 0412490005; or the appended examples. Similar methods exist for yeast; see for example: The molecular and cellular biology of the yeast Saccharomyces cerevisiae. Pringle, Jones, Broach and Strathern, Cols Spring Harbor laboratory press, 1992 (New York); Guide to yeast Genetics and Mollecular and Cell Biology (Volum 350 and 351 of Methods in enzymology) (Guthrie and Finf Eds), Academic Press (2002) San Diego; Yeast Gene Analysis (Brown and Tuite) (Volume 26 of Methods in Microbiology) Academic press (San Diego); Yeast Stress Responses (Ed. Hohmann and Mager) Springer Verlag, Heidelberg 1997.<br>
The methods of the present invention encompass a genetic modification of a plant or a plant cell. The term 'genetic modification" refers to a change by human intervention in the genetic<br>
content of a cell compared to a wild type cell and includes techniques like genetic engineering,<br>
breeding or mutagenesis.   The change in genetic content comprises modifications of the<br>
genome and includes addition, deletion and substitution of genetic material in the chromosomes of a plant cell as well as in episomes. The term also encompasses the addition of extrachromosomal information to a plant cell. Preferably, the genetic modification results in modulated expression of a nucleic acid. The methods of the present invention also encompass a subsequent step of selection, during which plants with the desired characteristics are selected. The selection step may be based on monitoring the presence or absence of modified growth characteristics, or on monitoring the presence or absence of selectable or screenable marker genes linked an introduced nucleic acid of interest<br>
Modulation (enhancing or decreasing) of expression of a nucleic acid encoding a CRYO protein or modulation of a CRYO protein itself encompasses altered expression of a gene or altered levels of a gene product, namely a polypeptkte, in specific cells or tissues, which gene or gene-product influences CRYO gene expression or protein activity.<br>
The nucleic acids obtained by the screening method according to the invention will have the capacity to modify tolerance to cold stress in plants or yeast. This effect may also be obtained by applying the proteins encoded by the nucleic acids as denned above, directly to the plants or yeast.<br>
Preferably, modulation of expression of a nucleic acid encoding a CRYO protein and/or modulation of activity of the CRYO protein itself is effected by recombinant means. Such<br>
recombinant means may comprise a direct and/or indirect approach for modulation of expression of a nucleic acid and/or for modulation of the activity of a protein.<br>
For example, an indirect approach may comprise introducing, into a plant, a nucleic add capable of modulating activity of the protein in question (a CRYO protein) and/or expression of the gene in question (a gene encoding a CRYO protein). The CRYO gene or the CRYO protein may be wild type, i.e. the native or endogenous nucleic acid or polypeptide. Alternatively, it may be a nucleic acid derived from the same or another species, which gene is introduced as a transgene, for example by transformation. This transgene may be substantially modified from its native form in composition and/or genomic environment through deliberate human manipulation. Also encompassed by an indirect approach for modulating activity of a CRYO protein and/or expression of a CRYO gene is the inhibition or stimulation of regulatory sequences that drive expression of the native gene or transgene. Such regulatory sequences may be introduced into a plant.<br>
A direct and more preferred approach comprises introducing into a plant or yeast a nucleic acid encoding a protein, or a functional fragment thereof, as defined above. The nucleic acid may be introduced by, for example, transformation. The nucleic acid may be derived (either directly or indirectly (if subsequently modified)) from any source provided that the sequence, when expressed in a plant or yeast, leads to modulated expression of a CRYO nucleic acid/gene or modulated activity of a CRYO protein.<br>
Preferably, the nucleic acid is isolated from a halophytic plant, more preferably from Beta vulgaris. Most preferably, the nucleic acid capable of modulating expression of a CRYO gene or activity of a CRYO protein in a plant is a nucleic acid as represented by SEQ ID NO 1,3, 5, 7,9, or homologues, derivatives or functional fragments thereof, or a nucleic acid encoding a protein represented by SEQ ID NO 2, 4, 6, 8 or 10, or homologues, derivatives or functional fragments thereof.<br>
However, it should be dear that the applicability of the invention is not limited to use of a nucleic acid represented by SEQ ID NO 1, 3, 5. 7, 9 nor to the nucleic acid encoding the protein of SEQ ID NO 2, 4, 6, 8 or 10, but that other nucleic acids encoding homologues, derivatives or functional fragments of SEQ ID NO 1 to 10 may be useful in the methods of the present invention. Advantageously, the method according to the present invention serves to confer tolerance or resistance to environmental stress conditions in plants and parts thereof, or in yeast.<br>
Modulating the activity of a nucleic acid/gene can be achieved for example by inhibiting or stimulating control elements that drive expression of a native gene or of a transgene, such regulatory sequences may be introduced into a plant or yeast. The 'nucleic acid" or "protein* may be wild type, i.e. the native or endogenous nucleic acid or polypeptide. Alternatively, the gene may be a heterologous nucleic acid derived from the same or another species, which gene is introduced as a transgene, for example, by transformation. This transgene may be substantially modified from its native form in composition and/or genomic environment through deliberate human manipulation. Modulating gene expression also encompasses altered transcript level of a gene, which can be sufficient to induce certain phenotypic effects.<br>
According to a preferred feature of the present invention, enhanced or increased expression of a nucleic acid is envisaged. Methods for obtaining enhanced or increased expression of genes or gene products are well documented in the art and include, for example, overexpression driven by a strong promoter, the use of transcription enhancers or translation enhancers.<br>
However downregulation of the expression of a nucleic acid may also give rise to modified stress tolerance in a plant or yeast. Advantageously, plants having modified stress tolerance may be obtained by expressing a nucleic acid encoding a CRYO protein in either sense or antisense orientation. Techniques for downregulation are well known in the art. Similar and other approaches for downregulation expression in yeast are known in the art (for example interruption of the ORF with a gene complementing a metabolic defect of the host strain or with a gene from bacteria conferring tolerance to the antibiotics Kanamycin or Geneticin).<br>
Another embodiment of the invention provides host cells comprising a nucleic acid molecule encoding a CRYO protein. Preferred host cells are plant cells or yeast. The polypeptides of the present invention may also be produced by recombinant expression in prokaryotte and eukaryotic engineered cells other than plant cells, such as bacteria, fungi, or animal cells. Suitable expression systems are known to those skilled in the art.<br>
The invention extends to plants or yeast tolerant to abiotic stress, preferably cold stress, which plants or yeast have elevated levels of a protein as defined above compared to corresponding wild type plants or yeast. The present invention thus also encompasses plants obtainable by the methods according to the present invention. The present invention therefore provides plants obtainable by the method according to the present invention, which plants have increased stress tolerance and which plants have altered CRYO protein activity and/or altered expression of a nucleic acid encoding a CRYO protein. In particular, the present invention provides plants with increased tolerance to abiotic stress, preferably cold stress, which plants<br>
have increased expression of a nucleic acid encoding a protein, or a functional fragment<br>
thereof, chosen from the group of<br>
0)       proteins belonging to the family of CHMP proteins;<br>
(ii)      proteins comprising a SEC14 domain and exhibiting lipid transfer activity;<br>
(iii)      CRYO5 like plant proteins comprising a RING finger domain, a serine rich domain and<br>
an acid domain which comprises the signature "HDQHRDMRLDIDNMSYEELLALEERIG".<br>
in which no more than 6 substitutions may occur; which increased tolerance is relative to corresponding wild type plants.<br>
Furthermore, the present invention provides plants with increased tolerance to abiotic stress, preferably cold stress, which plants have increased expression of a nudeic acid encoding a protein<br>
(a)	comprising the sequence as given in SEQ ID NO 2,4,6,8 or 10;<br>
(b)	comprising a sequence having at least 50 %, alternatively 60%, 70%, 80%, preferably<br>
90%, more preferably 95%, 96%, 97%, 98% or 99% sequence identity to the full length<br>
sequence as given in SEQ ID NO 2,4,6,8 or 10;<br>
(c)	comprising a substitution variant or insertion variant of (a);<br>
(d)	according to any of (a) to (c), comprising substitutions with corresponding naturally or<br>
non-naturally altered amino acids;<br>
(e)	comprising a functional fragment of any of (a) to (d);<br>
when compared to corresponding wild type plants.<br>
The present invention also relates to a method for the production of transgenic plants, plant<br>
cells or plant tissues, comprising introduction of a nucleic acid molecule of the invention in an<br>
expressible format or a genetic construct as defined above into a plant, plant cell or plant<br>
tissue. Therefore, according to a further embodiment of the present invention there is provided<br>
a method for producing transgenic plants having modified tolerance to stress, relative to<br>
corresponding wild type plants, which method comprises:<br>
(i)       introducing into a plant cell a nucleic acid encoding a CRYO protein or a family member<br>
of a CRYO protein or a functional fragment thereof; and<br>
(ii)      regenerating and/or growing a mature plant from said plant cell.<br>
Preferably, the stress is at least one of cold stress, salt stress, osmotic stress, drought stress<br>
or oxidative stress. More preferably, the stress is cold stress.<br>
The present invention extends to any plant cell, plant or plant part or yeast cell obtained by any of the methods described herein, and to all plant parts, including harvestable parts of a plant, seeds and propagules thereof. The present invention also encompasses a plant or a part<br>
thereof comprising a plant cell transformed with a nucleic acid according to the invention. The present invention extends further to encompass the progeny of a primary transformed or transfected cell, tissue, organ or whole plant that has been produced by any of the aforementioned methods, the only requirement being that progeny exhibit the same genotyplc and/or phenotypic characteristics) as those produced in the parent by the methods according to the invention.<br>
The term "plant* as used herein encompasses whole plants, ancestors and progeny of the plants, plant parts, plant cells, tissues and organs.    The term 'plant" also therefore encompasses suspension cultures, embryos, meristematic regions, callus tissue, leaves, flowers, fruits, seeds, rhizomes, bulbs, roots (including tubers), shoots (including stem cultures), gametophytes, sporophytes, pollen, and mtcrospores.   Plants that are particularly useful in the methods of the invention include all plants which belong to the superfamHy Viridiplantae, in particular monocotyledonous and dicotyledonous plants including a fodder or forage legume, ornamental plant, food crop, tree, or shrub selected from the list comprising Acanthaceae, Aceraceae, Acoraceae. Adiantaceae, Agavaceae, Aizoaceae, Alismataceae, Altiaceae, Aloeaceae, Alstroemeriaceae, Amaranthaceae, Amaryllidaceae, Anacardiaceae, Anemiaceae,     Angiopteridaceae,     Annonaceae,     Apocynaceae,     Aponogetonaceae, Aquifoliaceae,    Araceae,    Araliaceae,    Araucariaceaa,    Arecaceae,    Aristolochiaceae, Asparagaceae,   Aspleniaceae,   Asteliaceae,   Asteraceae.   Balsaminaceae,   Basellaceae, Bataceae, Begoniaceae, Berberidaceae, Betulaceae, Bignoniaceae, Bixaceae, Blechnaceas, Bombacaceae, Boraginaceae, Brassicaceae: Alliaria potiolata, Arabidopsis thaliana, Arabia pettolaris, Arabis pumila, Arabis sp., Berteroa incana. Biscutella laevigata. Brassicajunceae, Brassica napus, Brassica napus var. napus, Brassica nigra, Brassica oleracea, Brassica oteracea var. gongylo, Capsalla bursa-pastoris, Cardamino pratensis, Cochloaria officinafis, Dontaria laciniata, Dascurainia pinnata, Draba aspretla. Draba vema, Draba, Eiysimum asporum. Erysimum asperum, Eiysimum capttatum, Lepidiumdavum, Lepidium virginicum, Losqueralla argyraoa,  Lesquorolla donsiftora,  Lesquerslla rubicundula,  Lesquerella sp., Lobularia maritima,   Lunaria annua,   Lunaria rediviva,  Neobeckia  aquatica,   Nerisyrenia camporum,   Physaria  chambers/I,   Raphanus sativus,   Sinapis alba.   Stanlgya  pinnata, Slnptanthus cordatus, Th/aspi arvense, Thlaspi rotundifolium, Bromeliaceae, Buddlejaceae, Burseraceae,   Buxaceae,   Cabombaceae,   Cactaceae,   Caesalpiniaceae,   Callitrichaceaa, Calochortaceae, Calyceraceae, Campanulaceae, Cannabaceae, Cannaceae, Capparaceae, Caprifoliaceae,       Caricaceae,       Caryophyllaceae,       Casuarinaceae,      Celastraceae, Chenopodiaceae, Cistaceaa, Clusiaceae, Cneoraceae, Cochlospermaceae, Combretaceae, Commelinaceae, Convallariaceae, Convolvulaceae, Comaceae, Corylaceaa, Crassulaceae, Crossosomataceae,     Cucurbitaceae,      Cunoniaceae,     Cupressaceae,     Cuscutaceae,<br>
Cyatheaceae, Cycadaceae,  Cyperaceae,  Cyrillaceae,  Dennstaedtiaceae,  Oicksoniaceae, Dldiereaceae, Dilleniaceae, Dioscoreaceae, Dipsacaceae, Dipterocarpaceae, Droseraceae, Dryopteridaceae, Ebenaceae, Ehretiaceae, Elaeagnaceae, Elaeocarpaceae,  Elatinaceae, Empetraceae,   Epacridaceae,   Ephedraceae,   Equisetaceae,   Ericaceae,   Eriocaulaceae, Erythroxylaceae, Escalloniaceae, Euphorbiaceae, Eupomatiaceae, Fabaceae, Fagaceae, Flacourtiaceae, Fouquieriaceae, Frankenlaceae, Fumariaceae, Gentianaceae, Geraniaceae, Gesneriaceae, Ginkgoaceae, Globulariaceae, Goodeniaceae, Grossulariaceae, Gunneraceae, Haemodoraceae,   Haloragaceae,   Hamamelldaceae,    Heliconiaceae,   Hippocastanaceae, Hyatinthaceae,  Hydrangeaceae,  Hydrophyllaceae, Hypericaceae,   Iridaceae,  Isoetaceae, Juglandaceae,   Juncaceae,    Koeberiiniaceae,    Krameriaceae,    Lamiaceae,    Lauraceae, Lecythidaceae, Lemnaceae, Lentibulariaceae, Liliaceae, Limnanthaceae, Umnocharitaceae, Linaceae,   Loasaceae,   Lobeliaceae,   Loganiaceae,   Lomandraceae,   Lomariopsidaceae, Loranthaceae,   Lycopodiaceae,   Lythraceae,   Magnoliaceae,   Malpighiaceae,   Malvaceae, Maranlaceae, MarcgraWaceae, Marsileaceae, Martyniaceae, Mayacaceae, Melanthiaceae, Melastomataceae,     Meliaceae,     Melianthaceae,     Menispermacaae,     Manyanthaceae. Mimosaceae,    Monimiaceae,    Monotropaceae,    Moraceae,    Musaceae,    Myoporaceae, Myricaceae,   Myristicaceae,   Myrsinaceae,   Myrtaosae,   Nelumbonaceae,   Nyctaginaoaae, Nymphaeaceae,     Nyssaceae,    Ochnaceae,     Oenotheraceae,    Oleaceae,     Oliniaceae. Onagraceae, Ophioglossaceae, Orchidaceae, Orabanchaceae, Osmundaceae, Oxalidaceae, Paeoniaceae,   Pandanaceae,   Papaveraceae.Passifloraceae,   Pedaliaceae,   Philydraceae, Phormlaceae,   Phytolaccaceae,   Pinaceae,   Piperaceae,   Pittosporaceae,   plantaginaceae, Platanaceae, Plumbaglnaceae, Poaceae, Podocarpaceae, Podophyllaceae. Polemoniaceae, Polygalaceae, Polygonaceae, Polypodiaceae, Pontederiaceae, Portulacaceae, Primulaceae, Proteaceae,    Pteridaceae,    Punicaceae,    Pyrolaceae,    Rafflesiaceae,    Ranunculaceae, Resedaceae, Restionaceae, Rhamnaceae, Rosaceae, Rubiaceae, Ruscaceaa, Rutaceae, Salicaceae,    Salviniaceae,   Santalaceae,    Sapindaceae,   Sapotaceae,   Sarraceniaceae, Saururaceae,     Saxifragaceae,     Scrophulariaceae,     Selaginsllaceae,     Simaroubaceae, Smilacaceae,   Solanaceae,   Sparganiaceae,   Sterculiaceae,   Strelitziaceae,   Styracaceae, Taccaceae,    Tamaricaceae,    Taxaceae,    Taxodiaceae,    Theaceae,    Thelypteridaceae, Thymelaeaceae,  Tlliaceae,  Trapaceae, Tremandraceae,  Trilllaceae,  Trochodendraceae, Tropaeolaceae,    Tumeraceae,    Typhaceae,    Ulmaceae,    Urticaceae,    Valerianaceae, Verbenaceae, Veronicaceae, Vidaceae, Viscaceae, Vitaceae, Welwitschiaceae, Winteraceae, Xanlhorrhoeaceae,    Xerophyllaceae,     Xyridaceae,    Zamiaceae,     Zingiberaceae,     and Zygophyllaceae. According to a preferred feature of the present invention, the plant is a monocotyledonous or dicotyledonous plant, such as a crop plant selected from rice, maize, wheat, barley, soybean, sunflower, canola, alfalfa, millet, barley, rapeseed and cotton. Additional species such as amaranth, artichoke, asparagus, broccoli, Brussels sprouts,<br>
cabbage, carrot, cauliflower, celery, collard greens, flax, kale, lentil, oilseed rape, okra, onion, potato, sugar beet, sugar cane, tomato, squash, and tea, trees and algae are not excluded. Further advantageously, plants obtained by the methods according to the invention enable crops to be grown with Improved yield, growth, development and productivity under stress conditions, preferably under conditions of cold stress. The present invention also enables crops to be grown in areas which would otherwise not be possible.<br>
The gene of interest Is preferably introduced into a plant by transformation. The term "transformation" as referred to herein encompasses the transfer of an exogenous polynucleotide into a host cell, irrespective of the method used for transfer. The polynudeotkte may be transiently or stably introduced into a host cell and may be maintained non -integrated, for example, as a plasmid, or alternatively, may be integrated into the host genome. The resulting transformed plant cell can then be used to regenerate a transformed plant in a manner known to persons skilled in the art. Transformation of a plant species is now a fairly routine technique. Advantageously, any of several transformation methods may be used to introduce the gene of interest into a suitable ancestor cell. Transformation methods include the use of liposomes, electroporation, chemicals that increase free DNA uptake, injection of the DNA directly into the plant, particle gun bombardment, transformation using viruses or pollen and microprqection. Methods may be selected from the calcium/polyethylene glycol method for protoplasts (Krens et al., Nature 296, 72 - 74. 1982; Negrutiu I. et aL., Plant Mot. Biol. 8, 363 - 373. 1987); electroporation of protoplasts (Shillito er al.. Bio/Technol. 3, 1099 -1102, 1985); microinjection into plant material (Crossway et al., Mol. Gen. Genet. 202, 179 -185, 1986); DMA or RNA-coated particle bombardment (Klein et a/.. Nature 327. 70 1987) infection with (non-integrative) viruses and the like, Ag/ooacre/fum-mediated transformation (Cheng et al. 1997 - WO 97/48814; Hansen 1998 - WO 96/54961, Hiei ef al. 1994 -WO 94/00977; Hiei et al. 1998 - WO 98/17813; Rikiishi et al. 1999 - WO 99/04618; Saito et al. 1995 - WO 95/06722), including the flower dip' transformation method (Bechtold and Pelletier, Methods Mol. Biol. 82,259 - 266,1998; Trieu et al.. Plant J. 22,531 - 541,2000).<br>
Generally after transformation, plant cells or cell groupings are selected for the presence of one or more markers which are encoded by plant-expressible genes co-transferred with the gene of interest, following which the transformed material is regenerated into a whole plant. A whole organism may be regenerated from a single transformed or transfected cell, using methods known in the art Plant tissue capable of subsequent donal propagation, whether by organogenesis or embryogenesis, may be transformed with a genetic construct of the present invention and a whole plant regenerated therefrom. The particular tissue chosen will vary depending on the donal propagation systems available for, and best suited to, the particular<br>
species being transformed. Exemplary tissue targets include leaf disks, pollen, embryos, cotyledons, hypocotyts, megagametophytes, callus tissue, existing meristematic tissue (e.g., apical meristem, axillary buds, and root meristems), and induced meristem tissue (e.g., cotyledon meristem and hypocotyl meristem).<br>
Following DNA transfer and regeneration, putatively transformed plants may be evaluated, for instance using Southern analysis, for the presence of the gene of interest, copy number and/or genomic organisation. Alternatively or additionally, expression levels of the newly introduced ONA may be undertaken using Northern and/or Western analysis, both techniques being well known to persons having ordinary skill in the art.<br>
The generated transformed plants may be propagated by a variety of means, such as by donal propagation or classical breeding techniques. For example, a first generation (or T1) transformed plant may be selfed to give homozygous second generation (or T2) transformany, and the T2 plants further propagated through classical breeding techniques.<br>
The generated transformed organisms may take a variety of forms. For example, they may be chimeras of transformed cells and non-transformed cells; donal transformants (e.g., all cells transformed to contain the expression cassette); grafts of transformed and untransformed tissues (e.g., in plants, a transformed rootstock grafted to an untransformed scion).<br>
The invention also provides a method for producing a transgenic yeast cell having modified<br>
tolerance to abiotic stress relative to corresponding wild type yeast cells, which method<br>
comprises introducing into a yeast cell a nucleic acid encoding a CRYO protein as defined<br>
above. In particular, the present invention provides a method for producing transgenic yeast<br>
with increased tolerance to abiotic stress, preferably cold stress, relative to corresponding wild<br>
type yeast, which method comprises introducing into a yeast cell a nucleic acid encoding a<br>
protein, or a functional fragment thereof, chosen from the group of<br>
(i)        proteins belonging to the family of CHMP proteins;<br>
(ii)       proteins comprising a SEC14 domain and exhibiting lipid transfer activity;<br>
(iii)      CRYO5 like plant proteins comprising a RING finger domain, a serine rich domain and<br>
an acid domain which comprises the signature "HDQHRDMRLDIDNMSYEELLALEERIG",<br>
in which no more than 6 substitutions may occur.<br>
Furthermore, the present invention provides a method for producing transgenic yeast with increased tolerance to abiotic stress, preferably cold stress, relative to corresponding wild type<br>
yeast, which method comprises introducing into a yeast cell a nucleic acid encoding a protein, or a functional fragment thereof<br>
(a)	comprising the sequence as given in SEQ ID NO 2,4,6,6 or 10;<br>
(b)	comprising a sequence having at least 50 %, alternatively 60%, 70%, 60%, preferably<br>
90%, more preferably 95%, 96%, 97%, 98% or 99% sequence identity to the full length<br>
sequence as given in SEQ ID NO 2,4, 6,8 or 10;<br>
(c)	comprising a substitution variant or insertion variant of (a);<br>
(d)	according to any of (a) to (c), comprising substitutions with corresponding naturally or<br>
non-naturally altered amino adds;<br>
(e)	comprising a functional fragment of any of (a) to (d).<br>
The invention furthermore provides a Iransgenic yeast cell obtained by the methods of the invention. Preferably, the transgenic yeast cell with increased expression of a nucleic ackJ encoding a protein as defined here above, or a functional fragment thereof, is tolerant to abiotic stress, preferably cold stress when compared to corresponding wild type yeast cells.<br>
Furthermore, the invention also relates to the use of a nucleic acid encoding a CRYO protein as defined above or of a CRYO protein itself, to modify stress tolerance of plants or parts thereof or of plant cells. The sequences as depicted in SEQ ID NO 1 to SEQ ID NO 10 are revealed to be involved in important processes leading to stress tolerance, as exemplified by plants having altered stress tolerance, which plants have been transformed with a nucleic acid sequence encoding a CRYO protein as defined above. Similarly, the invention also relates to the use of a nucleic acid encoding a CRYO protein or of a CRYO protein itself, to modify stress tolerance of yeast. Preferably, the stress is at least one of temperature stress, osmotic stress, drought stress salt stress or oxidative stress.<br>
The present invention encompasses also the use of homologues of a CRYO protein. Therefore the invention relates to the use of a protein or an active fragment thereof, or of a nucleic acid encoding that protein or active fragment thereof, which protein is chosen from (i)       proteins belonging to the family of CHMP proteins; (ii)       proteins comprising a SEC14 domain and exhibiting lipid transfer activity; (iii)      CRY05 like plant proteins comprising a RING finger domain, a serine rich domain and        an        acid        domain        which        comprises        the        signature "HDQHRDMRLDIDNMSYEELLALEERIG", in which no more than 6 substitutions may occur.<br>
to modify stress tolerance of plants or parts thereof or of plant cells. Preferably, the stress is at least one of temperature stress, osmotic stress, drought stress salt stress or oxidative stress.<br>
The present invention encompasses also the use of homologues of a CRYO protein. Therefore the Invention relates to the use of a protein or of a nucleic acid encoding that protein, which protein<br>
(a)	comprises the sequence as given in SEQ ID NO 2.4,6,8 or 10;<br>
(b)	comprises a sequence having at least 50 %, alternatively 60%. 70%. 80%. preferably<br>
90%. more preferably 95%. 96%. 97%. 98% or 99% sequence identity to the full length<br>
sequence as given in SEQ ID NO 2.4,6,8 or 10;<br>
(c)	comprises a substitution variant or insertion variant of (a);<br>
(d)	according to any of (a) to (c), comprises substitutions with corresponding naturally or<br>
non-naturally altered amino acids;<br>
(e)	comprises a functional fragment of any of (a) to (d);<br>
Furthermore, the characteristic of the transgenic plants of the present invention to display tolerance to cold stress conditions can be combined with other approaches to confer cold stress tolerance to plants, e.g., use of osmotic protectants such as mannitol, proline; glycine-betaine. water-channeling proteins, etc. Thus, the approach of the present invention to confer tolerance to environmental stress conditions to plants can be combined with known approaches which include introduction of various stress tolerance genes. Combination of these approaches may have additive and/or synergistic effects in enhancing tolerance or resistance to environmental stress.<br>
The methods of the present invention to create plants with enhanced tolerance to stress can also be combined with other traits of interest, for example:<br>
(i)        herbicide tolerance (DE-A 3701623; Stalker, Science 242 (1988), 419),<br>
(ii)      insect resistance (Vaek, Plant Cell 5 (1987), 159-169),<br>
(iii)      virus resistance (Powell, Science 232 (1966), 738-743; Pappu, World Journal of<br>
Microbiology &amp; Biotechnology 11 (1995), 426437; Lawson, Phytopathology 86 (1996) 56<br>
suppl.),<br>
(iv)      ozone resistance (Van Camp, Biotechnol. 12 (1994), 165-168),<br>
(v)       improving the preserving of fruits (Oeller, Science 254 (1991), 437-439).<br>
(vi)      improvement of starch composition and/or production (Stark, Science 242 (1992),<br>
419; Visser, Mol. Gen. Genet 225 (1991), 289-296),<br>
(vii)     altering lipid composition (Voelker, Science 257 (1992), 72-74),<br>
(viii)    production of (bio)polymers (Poirer, Science 256 (1992). 520-523),<br>
(ix)      alteration of the flower colour, e.g., by manipulating the anthocyanin and flavonoid<br>
biosynthetic pathway (Meyer, Nature 330 (1987), 667-678. W090/12084),<br>
(x)      resistance to bacteria, insects and fungi (Duering, Molecular Breeding 2 (1996),<br>
297-305;    Strittmattar.    Bio/Technology    13   (1995),    1085-1089;    Estruch,    Nature<br>
Biotechnology 15(1997). 137-141).<br>
(xl)     alteration of al kaloid and/or cardia glyooside composition,<br>
(xfi)     inducing maintaining male and/or female sterility (EP-A1 0 412 006; EP-A1 0 223<br>
399;WO93/25695);<br>
(xiii)    higher longevity of the inflorescences/flowers, and<br>
(xiv)    abiotic stress resistance, other than temperature stress<br>
Description of the figures:<br>
The present invention will now be illustrated with reference to the following figures:<br>
Fig.1: Cold sensitivity of the wild type (wt) yeast strain compared to the gpd1 mutant. The yeast cells were grown on YPD (top row) or on SD medium (bottom row) at 30*C (control, left column), at 10°C (middle column) or at 15 °C (top right). The WT strain showed reduced growth compared to the gpd1 strain.<br>
Fig.2: Cold tolerance of the wild type yeast strain transformed with the CRYO1. CRYO2, CRYO3, CRY04 or CRYO5 gene, compared to the wt yeast strain transformed with an empty vector (pYPGE@). (a) Enhanced growth after 10 days at 10'C of the yeast cells transformed with the CRYO1, CRYO2 or CRYO3 genes, or after 14 days for yeast cells transformed with the CRYO4 gene, (b) Enhanced growth of the yeast cells transformed with the CRYO5 gene compared to wild type yeast transformed with an empty vector (pYPGE). The two left panels are controls grown at 30*C on YPD medium or SD medium. The two right panels show growth of the same yeast strains grown at 10°C on YPD medium or SD medium.<br>
Fig.3: Alignment between sequences of CRY01 and CRY02 from sugar beet and their homologues In Arabidopsis and yeast At= Arabidopsis thatiana, Bv= Beta vulgaris and Sc= Saccrtaromyces cerevlsiao.<br>
Fig.4: Alignment between the sequences of CRY03 and homologous proteins from various organisms, showing a high degree of conservation among the different species. At= Arabidopsis thaliana; Bv= Beta vulgaris; Mm= Mus musculus; Hs= Homo sapiens; Sp= Schizosaccharomyces pombe.<br>
Fig. 5: Southern blot with a CRY01 and CR/O2 probe on genomic sugar beet DNA. Enzymes used were SamHI, H/'ndlll and EcoRI.<br>
Fig. 6: a) Northern blot with a CRYO2 probe. Different timepoints (in hrs) after treating the sugar beet plants with 250 mM NaCI are indicated. α3-tubulin was used as control, b) Northern blot with a CRYO2 probe. Different timepoints (in hrs) after treating the sugar beet plants with 100 µM ABA are indicated. tubulin was used as control.<br>
Fig.7: Growth of wild type yeast (upper row) and yeast transformed with the CRYO5 gene (bottom row) on YPD (left panel) and YPD supplemented with 1 mM tert-butyl hydroperoxide (right panel).<br>
Examples<br>
Unless stated otherwise in the Examples, all recombinant DNA techniques are carried out according to standard protocols as described in Sambrook et al. (2001) Molecular Cloning: A Laboratory Manual, Third Edition, Cold Spring Harbor Laboratory Press, NY and in volumes 1 and 2 of Ausubel et al. (1994) Current Protocols in Molecular Biology, Current Protocols, USA. Standard materials and methods for plant,molecular work are described in Plant Molecular Biology Labfax (1993) by R.D.D. Croy, jointly published by BIOS Scientific Publications Ltd. (UK) and Blackwell Scientific Publications, UK.<br>
Example 1: Construction of a sugar beet cDNA library induced by salt stress:<br>
Sugar beet seeds (Beta vulgaris cv. Dlta) were sown in pots containing a mixture of sand and vermicuIHe (1:1 w/w). The plants were grown under greenhouse conditions (8 h at 20*C, 16 h at 25°C with supplementary lighting to ensure a minimum of 12 h photoperiod). The plants were periodically irrigated with a nutrient solution (2.4 g/l Ca(NO3)24H2O. 1 g/l KNO3, 1 g/l MgSO47H2O.0.3 g/l KH2PO,. 5.6 mg/l Fequelate (Kelantren. Bayer), 1.1 mg/l ZnSO47H2O, 3.3 mg/l MnO4H2O, 0.3 mg/l CuSO,5H20. 3.8 mg/l H3BO3, 0.18 mg/l (NH4)6Mo74H2O). For the construction of the cDNA library. 3-week old plants were irrigated with 200 mM NaCI for 24 h before harvesting.<br>
Directional cDNAs were synthesized (cDNA synthesis kit, Stratagene) with poly(A)+ RNA prepared from leaves of salt-treated sugar beet plants. cDNAs were ligated into phage λPG15 vector and packaged with Gigapack III gold packaging extract (Stratagene). A plasmid cDNA library was recovered from λPG15 by the cre-lox  reoombinasesystem (Brunelli and Pall, 1993).<br>
Example 2: Setup of a screening assay:<br>
The yeast strains used in this work were the wild type diploid strain W303/W303 (can1-100,his 3-11,15,leu2-3,112, trp1-1,ura3-1,GAL+) (WT) and a mutant deficient for glycerol phosphate<br>
dehydrogenase (gpdf). A diploid strain from two gpd 1 mutant strains (YRA111 (W303-1A mat a gpd1::TRP1) and YRA114 (W303-1A gpd1::TRP1 mat a)) was constructed. The diploid strains were used because these prevent the isolation of recessive chromosomal mutations which might give tolerance to cold stress. The strains were grown on YPD medium (2% glucose, 2% peptone, and 1% of yeast extract) or on SD medium (2% glucose, 0.7% yeast nitrogen base (Difco) without amino acids, 50 mM MES [2-(N-morpholino)-ethanesulfonic acid] adjusted to pH 5.5 with Tris, and the required amino acids, purine and pyrimidine bases). In a first step, the sensitivity to cold of the WT diploid strain was compared with that of the gpdl mutant strain. It was assumed that the production of glycerol could be a response against cold stress. Growth was monitored under cold stress conditions. Yeast strains were grown until stationary phase and 20 ul of 1/10,1/100 and 1/1000 dilutions of the culture were spot on both YPD and on SD medium at a temperature of 15 or 10'C for 10 to 14 days, and at 30*C for 2 days as a control. 10'C was the lowest temperature measured that allowed growth of the WT strain. Surprisingly, the gpd 1strain was shown to be cold tolerant whereas the wild type was cold sensitive (Fig. 1). This allowed the use of the WT strain for screening genes that could confer cold tolerance, while the gpd 1 strain could serve as a standard cold tolerant yeast strain for comparative studies.<br>
In a second step, the best conditions for transformation were determined. At the end the best protocol was: 300 ml YPD medium was inoculated with 30 jd of a saturated preculture of WT cells. The culture was grown overnight until an OD680=O.8, and centrifuged at 2000 rpm. The cells were subsequently washed with water and AcLiTE solution (0.1 M lithium acetate, 10mM Tris-HCI pH 7.6 and 1 mM EDTA (Ethylene diaminetetraacetic acid, disodium salt)). Next, the pellet of cells was resuspended in 2 ml of AcLiTE solution, incubated during 15 minutes at 30*C while shaking, whereafter 200 ul of ssDNA (10 mg/ml) was added. The cell suspension was divided in 110 µl aliquots in an Eppendorf tubes, and 200 ng of cDNA library was added. The heat shock transformation according to Gietz et al. (Nucleic Acids Res. 20, 1425, 1992) was used: in brief, 500 ul of PEG-AcLiTE solution (AcLiTE solution with 40% w/w of PEG (Polyethyleneglycol) 4000) was added to each aliquot. After mixing, the aliquots were incubated for 30 minutes at 30'C and next for twenty minutes at 42'C; then the cells were harvested and resuspended in 200 ul of 1M sorbitol. Two aliquots were plated in 14 cm Petri dishes with SD agar and all the necessary supplements except tryptophan (marker for the gpd1mutation), and uracil (marker for the plasmid). To quantify the efficiency of the transformation, four 55 ul aliquots were kept separately from the original cell pellet and were inoculated with 0,10, 50 and 100 ng of cDNA library. Then the same transformation protocol was applied, at the end the cells were resuspended in 100 µl of sorbitol and plated in 7 cm<br>
Petri dish containing the same SO medium. The average yield was about 60 colonies per ng of cDNA. In addition it was observed that transformation with competent cells that had been frozen, or transformation in one large-scale reaction instead of many small-scale reactions dramatically decreased the yield of the transformation.<br>
Example 3: Isolation of CRYO genes:<br>
The cDNA library constructed in pYPGE15 was used to transform the yeast WT strain W303 by the LiClmethod (Nucleic Acids Res. 20, 1425,1992). Transformants were selected on SD plates with leucine and adenine by uracil prototrophy. Three days after transformation colonies appeared in the Petri dishes. The colonies were harvested in sterile water and the number of cells quantified by plating different dilutions. On average a 10-fold higher concentration of cells than recovered from the transformation plates was plated on YPO and SO medium. Then the plates were left in a 10*C incubator and colonies able to grow after eight days were selected. Next the tolerance of the colonies isolated in the first round was re-checked and those not giving significant tolerance were discarded. From the remaining colonies, the plasm id was eliminated by selection in minimal medium for analysing whether the tolerance was dependent on the plasmid. As a final confirmation, the plasmid was recovered from the colonies that were able to pass the previous controls, transformed into a wild type strain and again a selection for those clones giving tolerance was performed. The results obtained are summarised in Table 1:<br>
Table 1: Summary of the screening procedure for the selection of cold tolerant yeast transformants on YPD or SO medium.<br><br>
(Table Removed1)<br>
The reconfirmed positive clones were sequenced and it was shown that they encoded different genes, among those were the genes named CRY01, CRYO2. CRY03, CRY04 and CRYO5 (for cryo-tolerant). An overview is given in Table 2:<br>
Table 2:<br><br>
(Table Removed2)<br>
The genes encoding CRYO1 to CRYO5 proteins conferred cold stress tolerance when<br>
transferred into yeast (Fig. 2).<br>
CRY01 was found to be homologous to the yeast DID1 protein and, upon further analysis of<br>
the homology data, was shown to have significant homology (
thaliana putative proteins gi/15233464 and gi/15224854 (shown in Fig. 3). According to the<br>
invention, these  putative Arabidopsis proteins were  named AtCRYOt  and AICRYO2<br>
respectively.<br>
CRYO3 is also conserved in Arabidopsis, with three putative proteins annotated in the database as At1g73030, At1g17730 and At4g17680, sharing more than 90% of homology. According to the invention, these putative Arabidopsis proteins were named AtCRYO3, AtCRY03.2 and AtCRY03.3 respectively. CRY03 is also conserved in humans and mice, as shown in the pile-up of Fig. 4.<br>
Example 4: Southern blotting reveals more than one Isoform In sugar beet<br>
In order to confirm the presence of CRYO1 and CRYO2 in the sugar beet genome and to estimate the number of genes encoding the haemoglobin in this plant species, a Southern blot analysis was performed. Genomic DMA was prepared from leaves of 3-week old sugar beet leaves (Rogers SO and Bendich AJ, Extraction of total cellular DNA from plants, algae and fungi (Eds) Plant molecular biology manual, Kluwer Academic Publishers, Dordrecht, Netherlands, 1994). 5 mg of DNA were digested with SamHI, Hindlll or EcoRI, electrophoresed in 0.8% agarose gel and blotted onto a nylon membrane filter (Hybond N-t-, Amersham Life Science). The membrane filter was hybridised with "P-labelled probes for CRYOf and CRYO2. Hybridisation and washes were carried out under high stringency conditions (65°C) (Church GM and Gilbert W., Proc. Nat. Acad. Sci. USA 81, 1991-1995, 1984). The presence of several hybridisation fragments in all lanes, independent of the restriction endonucleases used to digest the genomic DNA, suggests that there are several isoforms in the genome, especially for CRY02 (Figure 5).<br>
Example 5: CRYO2 is not induced by NaCI and ABA In sugar beet<br>
In order to investigate whether CRY02 also participates in the response of sugar beet plants to salt stress, the accumulation of CRY02 mRNA in response to various exposure times to NaCI was analysed using northern blot analysis. Total RNA was isolated from control, 250 mM Na* or 100 mM ABA -treated sugar beet leaves as described by Davis et at. (Basic methods in<br>
Molecular Biology. Elsevter. Amsterdam pp.143-146 1986). 30 mg of total RNA were separated on a 1% agarose gel containing 2.2% formaldehyde and blotted onto a nylon membrane filter (Hybond N, Amersham Life Science). Hybridization using the above described probe. The CRYO2 specific probe showed only one band that corresponded to the size of the CRYO2 cDNA. The filter was washed twice with 4X SSC buffer (0.6 M NaCI, 0.06 M trisodium citrate adjusted to pH 7 with HCI), 0.1% SDS for 5 minutes and twice with 0.4X SSC, 0.1% SDS for five minutes at 65°C. The same filter was re-hybridized with a 1.9 kb EcoRI fragment comprising the dj-tubulin gene of Arabidopsis thaliana (Ludwig et al., PNAS 84, 5833-5837, 1987). As shown in Figure 6a the CRY02 mRNA did not accumulate with time upon NaCI treatment. Similarly, no induction of CRYO2 after 3 hours of ABA treatment was observed (Figure 6b).<br>
Example S: CRYO5 gives also tolerance to oxldatlva stress.<br>
dilution series of W303 pYPGECRYOS and wt yeast (control) was plated on YPD medium with 1 mM tert-butyl hydroperoxyde (t-BOOH) and tested for tolerance to oxidative stress after 2 and 4 days.<br>
The yeast done with CRYO5 had a strong t-BOOH tolerance phenotype and the phenotype was very reproducible: at a concentration of 1 mM t-BOOH, control yeast cells did not grow at all, whilst yeast cells overexpressing CRYO5 did (Fig. 7).<br>
The definition of a strong phenolype is based on drop test experiments. Different dilutions of saturated cultures (1:10,1:100,1:1000) were made and these were grown on selective media (YPD with 1 mM t-BOOH). "Strong phenotypes* were those clones that grew well in all the dilutions assayed. With "no strong phenotypes" is meant that the done does not grow in all dilutions. The control cells expressing the empty plasmld did not grow at all in the selective media.<br>
Example 7: construction of cold tolerant plants:<br>
Plants are transformed with at least one of the CRYO genes in an expressible format under control of a constitutive or inducible promoter, using standard techniques.<br>
Example 8: Testing of cold tolerant plants:<br>
Transformed plants are tested by subjecting the plants to cold stress during a sufficiently long time period. When compared to the same untransformed plant line, the transformed lines show a better growth during stress conditions and/or better recovery after stress conditions and/or higher yield (biomass and/or harvestable parts).<br><br><br><br><br><br>
WE CLAIM:<br>
1. Method for increasing abiotic stress tolerance of plants, comprising the steps of introducing and expressing an isolated nucleic acid encoding a CHMP protein in said plants and selecting for increased expression is said plants of said nucleic acid sequence and wherein said CHMP protein<br>
(a)	comprises the sequence as given in SEQ ID NO. 2, 4, or 6:<br>
(b)	comprises a sequence having at least 50%, alternatively 60%, 70%, 80%, preferable 90%, more preferably 95%, 96%, 98% or 99% sequence identity to the full length sequence as given in SEQ ID NO 2, 4, or 6;<br>
(c)	comprises a substitution variant or insertion variant of (a);<br>
(d)	according to any (a) to (c), comprises substitutions with corresponding naturally or non-naturally altered amino acids;<br>
(e)	comprises a functional fragment of any of (a) to (d).<br><br>
2.	Method as claimed in claim 1, wherein said abiotic stress is at least one of temperature stress, osmotic stress, drought stress, salt stress or oxidative stress, preferably said abiotic stress is cold stress.<br>
3.	Method for selecting plants using a nucleic acid encoding a protein as claimed in claim 1 as selectable marker.<br>
4.	Method as claimed in claim 1, wherein it involves a genetic construct comprising:<br>
(i)        an isolated nucleic acid encoding a plant protein<br>
(a) comprising the sequence as given in SEQ ID NO. 2, 4 or 6;<br><br>
(b)	comprising a sequence having at least 50%, alternatively 60%, 70%,<br>
80%, preferably 90%, more preferably 95%, 96%, 97%, 98% or 99%<br>
sequence identity to the full length sequence as given in SEQ ID NO 2, 4<br>
or 6;<br>
(c)	comprising a substitution variant or insertion variant of (a);<br>
(d)	according to any of (a) to (c), comprising substitutions with<br>
corresponding naturally or non-naturally altered amino acids;<br>
(ii) a regulatory element operably linked to the nucleic acid of (i), which regulatory element is a plant and/or yeast expressible promoter and optionally<br>
(iii)      a transcription termination sequence.<br>
5. A host cell comprising an isolated nucleic acid encoding a protein as claimed in claim 1, wherein said host cell is a bacterial or fungal.<br><br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDE2MS1ERUxOUC0yMDA1LUFic3RyYWN0LSgyMy0xMC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">4161-DELNP-2005-Abstract-(23-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDE2MS1kZWxucC0yMDA1LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">4161-delnp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDE2MS1ERUxOUC0yMDA1LUNsYWltcy0oMjMtMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">4161-DELNP-2005-Claims-(23-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDE2MS1kZWxucC0yMDA1LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">4161-delnp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDE2MS1kZWxucC0yMDA1LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjYtMDQtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">4161-delnp-2005-Correspondence Others-(26-04-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDE2MS1ERUxOUC0yMDA1LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjMtMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">4161-DELNP-2005-Correspondence-Others-(23-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDE2MS1kZWxucC0yMDA1LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">4161-delnp-2005-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDE2MS1kZWxucC0yMDA1LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">4161-delnp-2005-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDE2MS1ERUxOUC0yMDA1LURyYXdpbmdzLSgyMy0xMC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">4161-DELNP-2005-Drawings-(23-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDE2MS1kZWxucC0yMDA1LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">4161-delnp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDE2MS1ERUxOUC0yMDA1LUZvcm0tMS0oMjMtMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">4161-DELNP-2005-Form-1-(23-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDE2MS1kZWxucC0yMDA1LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">4161-delnp-2005-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDE2MS1kZWxucC0yMDA1LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">4161-delnp-2005-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDE2MS1ERUxOUC0yMDA1LUZvcm0tMi0oMjMtMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">4161-DELNP-2005-Form-2-(23-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDE2MS1kZWxucC0yMDA1LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">4161-delnp-2005-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDE2MS1ERUxOUC0yMDA1LUZvcm0tMy0oMjMtMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">4161-DELNP-2005-Form-3-(23-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDE2MS1kZWxucC0yMDA1LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">4161-delnp-2005-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDE2MS1kZWxucC0yMDA1LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">4161-delnp-2005-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDE2MS1ERUxOUC0yMDA1LUdQQS0oMjMtMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">4161-DELNP-2005-GPA-(23-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDE2MS1kZWxucC0yMDA1LUdQQS0oMjYtMDQtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">4161-delnp-2005-GPA-(26-04-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDE2MS1kZWxucC0yMDA1LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">4161-delnp-2005-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDE2MS1kZWxucC0yMDA1LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">4161-delnp-2005-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDE2MS1kZWxucC0yMDA1LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">4161-delnp-2005-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDE2MS1kZWxucC0yMDA1LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">4161-delnp-2005-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDE2MS1kZWxucC0yMDA1LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">4161-delnp-2005-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDE2MS1kZWxucC0yMDA1LXBjdC0zMDgucGRm" target="_blank" style="word-wrap:break-word;">4161-delnp-2005-pct-308.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDE2MS1kZWxucC0yMDA1LVBldGl0aW9uIE90aGVycy0oMjYtMDQtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">4161-delnp-2005-Petition Others-(26-04-2012).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="231862-a-process-for-the-preservation-of-coconut-sap-neera.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="231864-an-antitumor-effect-potentiator.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>231863</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>4161/DELNP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>12-Mar-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>15-Sep-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CROPDESIGN N.V.,</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>TECHNOLOGIEPARK 3, B-9052 ZWIJNAARDE, BELGIUM</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>RAMON SERRANO SALOM</td>
											<td>CALLE ASTURIAS 22, PUERTA 2, E-46023 VALENCIA, SPAIN</td>
										</tr>
										<tr>
											<td>2</td>
											<td>JOSE MIGUEL MULET SALORT</td>
											<td>HASINGERSTRASSE 33, CH-4055 BASEL, SWITZERLAND</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/82</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2004/050513</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-04-13</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>03076064.9</td>
									<td>2003-04-11</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/231863-method-for-increasing-abiotic-stress-tolerance-of-plants by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:47:12 GMT -->
</html>
