<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/231903-a-method-of-sequentially-assembling-several-dna-units by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:49:51 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 231903:&#x27;&#x27;A METHOD OF SEQUENTIALLY ASSEMBLING SEVERAL DNA UNITS&#x27;&#x27;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&#x27;&#x27;A METHOD OF SEQUENTIALLY ASSEMBLING SEVERAL DNA UNITS&#x27;&#x27;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method of sequentially assembling several DNA units to form a DNA construct, such that the junctions so formed are protected from cleavage by the subsequent application of the chosen restriction enzyme, permitting the repeated use of the chosen restriction enzyme in assembling the DNA construct.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The present invention relates to a   method of sequentially assembling several DNA units.<br>
Background<br>
Polyketides, including the valuable drugs avermectin, erythromycin and rapamycin, are natural products that are synthesised by stepwise condensation of acetate, propionate and occasionally butyrate units. The enzymes that take part in the biosynthesis of polyketide chains are collectively known as the polyketide synthase (PKS). PKSs include examples of both type 1 (multifunctional enzyme) and type II (dissociable complex) organisation. The sequencing of the gene clusters encoding the erythromycin- (ery) and rapamycin- (rap) producing polyketide syntheses has shown that each cycle of polyketide chain extension is catalysed by a different set or 'module' of enzyme activities, housed in a few very large multienzyme polypeptides. The basic building blocks of modules are enzymatic 'domains' that are covalently linked .together. The ability of these domains to act upon the carbon chain and remove/add functionalities is reminiscent of a molecule being acted upon by chemical reagents in a<br>
s   •**"	'	-	'<br>
chemical synthesis. The aim is therefore to assemble these domains or . even modules in a manner as desired, so that the linked enzymes can .. carry out efficient synthesis of any target molecule. Until now, it has ..however not been possible to find a versatile methodology to assemble these PKS units.<br>
The whole area of polyketide research is at a stage where the flexibility of the whole enzymatic machinery is understood, despite the lack of any X-ray crystal structure data on these giant enzymes, but it remains difficult to "re-assemble" the enzymes de novo. A de novo synthesis is desirable for two reasons. Firstly, one does not need to change the structure of, for example, an antibiotic using tedious chemical methodologies that are time-consuming and expensive. Engineering an<br>
synthetic enzyme at the genetic level is much easier, faster and cheaper. As more and more antibiotics are rendered useless, simply because the bacteria they were active against have developed ways in which to become resistant to these drugs, there is an urgency to keep developing altered drug structures. Secondly, there is an ever-growing need for new drugs, more potent in their action than their predecessors. Whilst nature provides a large proportion of the new molecules that are, for example, antibiotic, anticholesterol, antifungal, or anti-cancer, the complicated structures of these drugs (for example the anti-cancer Taxol) makes it increasingly difficult for chemists to carry out conventional syntheses. The problem is made more difficult by the fact that the genes that make these drugs cannot always be isolated.<br>
The isolation of the genes coding for the proteins that make the highly potent anti-cancer compound Taxol, has not as yet been reported. The resulting choice for obtaining Taxol is either to cut down 200 Pacific Yew trees to obtain enough taxol for one chemotherapy session, or to make the drug chemically using one of the many exceedingly expensive and long chemical routes that have appeared recently in the literature.<br>
With the isolation, cloning and sequencing of the genes coding for the erythromycin polyketide synthases, a model for the functioning of modular type I PKSs began to emerge. It was clear that such a system is genetically programmed to carry out the necessary catalytic activities needed for processing of the polyketide chain. It is hypothesised that each domain acts independently on the progressing carbon skeleton and there is a correlation between the structure of the growing chain and the enzymatic activities carried out by the enzymes.<br>
The first conclusive proof of such an arrangement came from experiments done by Donadio era/. (1991, 1993). One such experiment (1991) involved an in-frame deletion in the ORF3 segment of erythromycin chromosome. This deletion eliminated the entire 183 amino acids of the ketoreductase domain of ery PKS module 5, along with some of the<br>
flanking region (a total of 271 amino acids) and resulted in the production of 5,6-dideoxy-3-a-mycarosyl-5-oxo-erythronolide B, the structure of which was confirmed by X-ray crystallography. Replacement of two amino acids in the putative NAD(P)H-binding motif of the enoylreductase domain encoded by ORF2 resulted in a new macrolide 6'7-anhydroerythromycin C being produced albeit in low yield. These results demonstrated that erythromycin PKS can be genetically reprogrammed to produce novel macrolides that would otherwise be difficult to get via chemical means. During the analysis of the fermentation products produced by a strain of S. erythraea that was genetically engineered to produce an analogue of 6dEB, it was found that a minor component of the fermentation was 3,5-dihydroxy-2,4-dimethyl-n-heptanoic acid 8-lactone (Donadio et al., 1991). This product was predicted to result from premature release of the chain from either the ACP of module 2 or the KS of module 3. A greater yield of this triketide product was obtained by heterolpgous over-expression of ORF1 in Streptomyces coelicolor (Kao et al., 1994), which also showed that DEBS1 can function autonomously. More recently (Cortes et al., 1995), a six-membered lactone was produced through genetically engineering the PKS. By repositioning the TE (cyclase) domain from module 6 to the C-terminus of module 2 (end of DEBS1), it was found that the yield of the lactone is increased by five-fold to 10-15 mg/L as compared to 1-3 mg/L obtained by Kao et al.<br>
The relocation of the thioesterase domain at the end of DEBS1 was the first example demonstrating the efficacy of repositioning domains in type I modular systems. Since then, numerous such experiments have been carried out in order to probe further the efficacy of these multienzymes. The TE domain has been relocated at the end of module 5 as well as module 3 respectively (Kao et al., 1995, 1996). In both cases, the predicted compounds were produced that resulted from truncation of the progressing polyketide chain. Release of the 12-membered product in the former case showed that the thioesterase domain can indeed catalyse<br>
ring closure even for less energetically favourable reactions. In the second experiment, two products were produced, one of them thought to be resulting from spontaneous decarboxylation.<br>
The first example of a chimaeric polyketide synthase constructed from a domain taken from a second PKS was demonstrated by Oliynyk et al. (1996). An acyltransferase domain (AT) from module 2 of the rapamycin polyketide synthase was used to replace the AT of module 1 in the DEBS1-TE system. The resulting triketide lactone had a methyl group missing at position 5 of the six-membered ring. This was expected since the AT of module 2 of rap PKS (unlike the AT of module 1 of DEBS1) incorporates a malonyl-CoA extender unit, instead of a methylmalonyl-CoA unit.<br>
Thus, it has been shown thaT not only can domains residing within a particular PKS be interchanged or destroyed, analogous domains can be derived from other synthases for the same purpose or for achieving the required synthetic goal. Such a strategy immediately provides a glimpse of the manner in which "designer" polyketides can be constructed through using "off-the-shelf gene products.<br>
More recently, another hybrid system has been constructed (Marsden et al., 1998) wherein a complete loading module from the avermectin PKS has been swapped with the erythromycin loading module, while keeping the rest of the DEBS modules intact. As expected, incorporation of butyryl-CoA as well as 2-methylisobutyryl-CoA was seen and in both cases, the end products contained the above mentioned residues. A closely-related experiment has been reported by Kuhstoss et a!. (1996) in which the loading module from the platenolide PKS was replaced with the loading module from tylactone PKS to yield the expected polyketide product.<br>
It is very clear from the various engineering efforts outlined above that the aim must now be to exploit the potential for genetic manipulation of type I (modular) polyketide synthases (PKS) to produce hybrid synthases<br>
that might catalyse the formation of novel secondary metabolites in a<br>
predictable way.<br>
What might be a giant step towards the realisation of this aim, would<br>
be to investigate whether these enzymes might be constructed de novo, as<br>
an essential step in developing a truly combinatorial biosynthesis of<br>
complex polyketides.<br>
The 'assembly line' nature of type I polyketide syntheses (PKS) that contain sets (called modules) of structurally similar but functionally different enzymatic activities (domains) suggests their potential as a source of "off-the-shelf enzymatic reagents which can be used to synthesise new and complex polyketide molecules. Outlined below are methodologies for the rapid assembly of DNA units encoding such enzyme domains or modules of enzyme domains.<br>
There are over 40 gene sequences for polyketides that are available from various databases. In addition there are numerous domains known from other synthetic enzymes such as, for example, fatty acid synthase (Joshi and Smith, 1993), peptide synthetases (Eisner et al., 1997) and hybrid polyketide/peptide synthesising enzymes (Paitan et al., 1999; Shen et al., 1999). This amounts to a vast library of domains and modules that cater for a chemical reaction (e.g. stereospecific condensation, dehydration, etc), or in the case of a module, a set of chemical reactions. In order to obtain analogues of a bio-active molecule, research efforts till now have been focused on strategies that involve either chromosomally altering the PKS genes that make the particular molecule (McDaniel et al., 1999) or feeding synthetic intermediates to the PKS (Jacobsen et al., 1997) Because of the simplified nature of such experiments, these strategies will remain a fast route towards obtaining a wide variety of drug analogues. However, in the case of compounds like the highly potent anti-cancer discodermolide (TerHaar et al., 1996) the only possible means of obtaining sufficient quantities of the drug is through chemical synthesis. This is because in such cases, the genes responsible for making these bio-active<br>
molecules have not been isolated. The chemical synthesis of large molecules having numerous chiral centres like for example discodermolide, howsoever elegant, is tedious and expensive to scale-up (Marshall and Johns, 1998).<br>
Abbreviations<br>
In addition to those listed in Biochem. J. (1986) 233, 1-24, the following abbreviations have been used:<br>
6-dEB	6-deoxyerythronolide B<br>
6-MSA	6-methylsalicylic acid<br>
6-MSAS	6-methylsalicylic acid synthase<br>
ACP	acyl carrier protein<br>
AT	. (3-keto acyl transferase<br>
bp	base pair(s) of DNA<br>
DEBS	6-deoxyerythronolide B synthase<br>
DH	p-hydroxyacyl-ACP dehydratase (dehydratase)<br>
ER	enoyl reductase<br>
FAS	fatty acid synthase<br>
kbp	kilobase pair(s)<br>
KR	ß-ketoacyl reductase<br>
KS	ß-ketoacyl synthase<br>
ORF	open reading frame<br>
PKS	polyketide synthase<br>
RAPS	rapamycin synthase<br>
TE	thioesterase<br>
Accordingly, there is provided a method of sequentially assembling several DNA units to form a DNA construct, such that the junctions so formed are<br>
protected from cleavage by the subsequenta application of the chosen restriction <br>
enzyme, permitting the repeated use of the chosen restriction enzyme in <br>
assembling the DNA construct.<br><br>
In one aspect the invention provides a method of assembling several<br>
DNA units in sequence in a.DNA construct. This method comprises the<br>
steps of:.	•         .<br>
a)	providing each DNA unit with a restriction enzyme recognition sequence<br>
at it's 5' end and with a recognition sequence for the same restriction<br>
enzyme at its 3' end that is combined with a recognition site for a DNA<br>
modification enzyme,	b)	providing a starting DNA construct having an accessible restriction site<br>
for the same or a compatible restriction enzyme and cleaving the starting<br>
DNA construct with such a restriction enzyme,<br>
c)	inserting the desired DNA unit and bringing the ligated product into<br>
contact with a DNA modification enzyme such that the restriction site at the<br>
3' end of the inserted DNA unit is abolished,<br>
d)	cleaving the ligated product at an accessible unmodified recognition site<br>
for the same or a compatible restriction enzyme,<br>
e)	repeating steps c) and d) to introduce each desired DNA unit to give a<br>
DNA construct containing all the desired units in sequence.<br>
DNA units can be any desired DNA sequence, though usually they encode enzyme domains or modules of two or more enzyme domains. The recognition sequences are usually positioned at the ends of the DNA unit once the DNA unit has been cut with the relevant enzyme, by this it is meant that the recognition sequences are adjacent to the coding sequence, or that they flank the said .sequence. An accessible restriction site is herein defined as a restriction site which is unmodified, such that it can be cleaved by a restriction enzyme that normally recognises the sequence of the site. The. accessible restriction site is preferably a unique site in the DNA unit or ligated product. Where there is more than one accessible site present, it is possible to perform a partial digest, as known in the art, to obtain digested products in which only the required site is cleaved in the DNA unit The<br>
DNA modification enzyme employed in the method can be a methylase for example the dam methylase of Escherichia coll. Other methylases such as dcm are also envisaged.<br>
A particular method comprises the steps of<br>
a)	providing each DNA unit with an Xba\ recognition sequence<br>
5'XXTCTAGA3' (where XX is not GA) at it's 5' end and with an Xbarecognition sequence 5'GATCTAGA3' at its 3' end.<br>
b)	providing a starting DNA construct having an accessible Xba\ site and<br>
cleaving the starting DNA construct with Xba\,<br>
c)	inserting the desired DNA unit and using a resulting ligated product to<br>
transform a dam+ strain of £ coll,<br>
d)	recovering a resulting plasmid and cleaving the plasmid at an accessible<br>
Xba\ site with Xba\,<br>
e)	repeating steps c) and d) to introduce each desired DNA unit to give a<br>
DNA construct containing all the desired units in sequence.<br>
The recognition sequences for the restriction enzyme and the DNA modification enzyme employed in the method can be created in the DNA units prior to cutting with the restriction enzyme, for example by means of a primer extension reaction. The preferred DNA construct made by the method can be an expression vector capable of facilitating expression of the protein encoded by the desired DNA units.<br>
It is also envisaged that the DNA modification can be removed and the restriction site re-established by replicating the ligated product in a dam- strain of E. coli by means of suitable vectors as known in the art.<br>
The invention also encompasses DNA unit assemblies where any given restriction enzyme recognition site can be modified by addition of a certain combination of nucleotide bases in order for it to be protected.<br>
In a further aspect, the invention provides a method of making an<br>
assembly of several DNA units in sequence which method comprises the<br>
steps of:	<br>
providing a first DNA unit with a recognition sequence for a first<br>
restriction enzyme at its 3' end, and cleaving the said first DNA unit with<br>
said first restriction enzyme,<br>
a)	providing each other DNA unit with a recognition sequence at its 5' end<br>
for a second restriction enzyme which has a. compatible ligation sequence<br>
with that of the first restriction enzyme, and an upstream recognition<br>
sequence for said first restriction enzyme and a downstream recognition<br>
sequence for a third restriction enzyme at its 3' end, and cleaving each said<br>
other DNA unit with the second and third restriction enzymes,<br><br>
c)	ligating the said first DNA unit with a desired other DNA unit to form a<br>
ligated product such that the ligation of the two units abolishes the<br>
recognition site for the first restriction enzyme at the ligation junction, and<br>
cleaving the ligated product with said first restriction enzyme,<br>
d)	ligating the product from c) with a desired DNA unit from b) to form a<br>
ligated product and cleaving the ligated product with said first restriction<br>
enzyme<br>
e)	repeating step d) with each other DNA unit in turn so as to assemble the<br>
DNA units in sequence.<br>
A.particular method comprises the steps of:<br>
a)	providing a first DNA unit with an Xba\ recognition sequence<br>
5TCTAGA31 at its 3' end, and cleaving the said first DNA unit with Xbal,<br>
b)	providing each other DNA unit with a Spel recognition sequence<br>
5'ACTAGT3' at its 5' end, and a downstream Xba\ recognition sequence<br>
5TCTAGA3' followed by a downstream Smal recognition sequence<br>
5'CCCGGG3' at its 3' end, cleaving each said other DNA unit with Spel and<br>
Smal, and dephosphorylating the 5' end of the cleaved DNA unit,<br>
c)	ligating the said first DNA unit with a desired other DNA unit to form a<br>
ligated product and cleaving the ligated product with Xbal,<br>
d)	ligating the product from c) with a desired DNA unit from b) to form a<br>
ligated product and cleaving the ligated product with Xbal<br>
e) repeating step d) with each other DNA unit in turn so as to assemble the DNA units in sequence.<br>
In one embodiment the assembly can occur via stepwise addition of fragments to a vector.<br>
In an alternative embodiment the first DNA unit can be attached to the solid phase for use in step c). This permits the solid phase to be split and mixed between steps c), d), and e) to make several different assemblies. Methods of attaching DNA units to the solid phase are well know in the art. Preferred solid phase elements are beads attached to the DNA units via a biotinylated nucleotide, as known in the art.<br>
The recognition sequences in one or more of the DNA units are preferably introduced by means of extension primers, as known in the art, though other methods such as the ligation of the required sequences or in vitro mutagenesis can also be employed.<br>
The assembly of several DNA units can be inserted into an expression vector and thus used to transform a host capable of expressing the protein encoded by the insert of the vector.<br>
The method is particularly useful where one or more of the DNA units encodes a catalytic or transport protein domain for example a ketoreductase domain from a PKS enzyme or an ACP domain from a hybrid polyketide/peptide synthesising enzyme. Such domains can be derived from enzyme domain DNA sequences from, for example, polyketide synthesising enzymes, peptide synthesising enzymes, hybrid peptide polyketide synthesising enzymes, fatty acid synthesising enzymes or other enzyme domains known in the art.<br>
The DNA units used in the methods of the invention can encode modules comprising one or more catalytic or transport domains. Usually a module contains all of the domains required to complete one condensation step in the synthesis of a target molecule.<br>
Alternative aspects of the invention resulting from the methods of the invention include: DNA constructs or vectors incorporating a DNA assembly<br>
encoding synthetic enzymes, synthetic enzymes encoded by such DNA assemblies, hosts expressing synthetic enzymes, hybrids of transformed hosts expressing synthetic enzymes, and compounds produced by the synthetic enzymes.<br>
Where the product produced by the synthetic enzyme exhibits toxicity to a host stain, this can be worked around e.g. by means of choosing a different strain or mutating the original strain to provide mutants which are more tolerant. The diversity of compounds produced by hosts transformed • with the synthetic enzymes of the invention can be further increased by using known methods of using different feedstocks in the fermentation to provide different starter units for the desired product. Where yield of desired synthetic enzyme product is low, routine steps e.g. mutation and selection, can be taken to improve this,<br>
The synthetic enzymes of the invention can also be used in cell-free systems to produce the desired target molecule in vitro as known in the art, for example, see Carreras and Khosla (1998).'<br>
In a further aspect, the invention provides a method of synthesising a target molecule comprising the steps of<br>
a)	examining the composition and stereochemistry of a target molecule,<br>
b)	determining which catalytic and transport domains need to be present in<br>
a synthetic enzyme in order to catalyse the synthesis of the target<br>
molecule,<br>
c)	using any one of the methods of the invention to assemble the required<br>
DNA units encoding the catalytic and transport domains into a DNA<br>
assembly that encodes said synthetic enzyme which is capable of<br>
synthesising the target molecule.<br>
d)	placing the DNA assembly into a vector to allow expression of the<br>
synthetic enzyme in a host capable of synthesising the target molecule<br>
after transformation with said vector.<br>
Target molecules are generally bio-active molecules, usually having a predominantly carbon based backbone and usually are macromolecules<br>
comprised of condensed units. The transformed host can be tested for the presence of the target molecule after step d).  If yields of the desired compound are low then conventional methods of improving product yield from, for example Streptomycetes, can be employed. Transformed hosts -  which result from the methods of the invention and their use in producing target molecules are also aspects of the invention. Hosts suitable for transformation with the DNA assemblies of the invention are known in the art and include insect or mammalian cells, though more usually suitable are bacterial cells, for example, the improved host strains described by Ziermann and Betlach (1999).<br>
As stated previously, it is also envisaged that the synthetic enzyme can be used in a cell-free system to produce the target molecule in vitro.<br>
A further aspect of the invention is a method of making a synthetic<br>
enzyme to catalyse the synthesis of a target molecule comprising the steps<br>
of	<br>
a)	examining the composition and stereochemistry of a target molecule,<br>
b)	determining which catalytic and transport domains need to be present in<br>
the synthetic enzyme in order to catalyse synthesis of the target molecule,<br>
c)	using any one of the methods of the invention to assemble the required<br>
DNA units encoding the catalytic and transport domains into a DNA<br>
assembly that encodes an enzyme which is capable of synthesizing the<br>
target molecule.<br>
d)	expressing the DNA assembly in a suitable host to produce the enzyme.<br>
In a further aspect the invention provides a library of DNA units encoding catalytic or transport protein domains, wherein each DNA unit has a recognition sequence for a restriction enzyme at it's 5-end and a second recognition sequence for the same or a compatible enzyme at it's 3'-end which incorporates a recognition sequence for a DNA modifying enzyme.<br>
In a particular embodiment of such a library, each DNA unit has an Xbal recognition sequence 5'XXTCTAGA3' (where XX is not GA) at it's 5'-end and an Xbal recognition sequence 5'GATCTAGA3' at it's 3'-end<br>
Also provided by the invention is a library of DNA units encoding catalytic or transport protein domains, wherein each DNA unit has a recognition sequence at its 5' end for a first restriction enzyme, and a downstream recognition sequence for a second restriction enzyme followed by a downstream recognition sequence for a third restriction enzyme at its 3' end, such that the DNA units, once restricted by the first and second restriction enzymes can be ligated together to abolish the restriction sites at the ligation junction. In one embodiment of this aspect of the invention each DNA unit has a Spel recognition sequence 5'ACTAGT3' at its 5'-end, and a downstream Xbal recognition sequence 5TCTAGA3' followed by a downstream Smal recognition sequence 5'CCCGGG31 at it's 3'-end<br>
Catalytic or transport protein domains can be derived from any enzyme, for example those listed above. Particularly'envisaged are libraries in which the DNA units encode polyketide synthetic domains, comprising two KS domains, at least two AT domains, two KR domains, two DH domains, two ER domains, an ACP domain and a TE domain.<br>
Also provided by the invention are modules comprising a DNA   . sequence encoding a functional set of polyketide synthetic domains wherein the module has a recognition sequence for a restriction enzyme at it's 5'-end and a second recognition sequence for the same or a compatible enzyme at it's 3'-end which incorporates a recognition sequence for a DNA modifying enzyme. An envisaged module has an Xbal recognition sequence 5'XXTCTAGA3' (where XX is not GA} at it's 5'-end and an Xbal recognition sequence 5'GATCTAGA3' at it's 3'-end<br>
Alternatively a module comprising a DNA sequence encoding a functional set of polyketide synthetic domains can have a recognition sequence at its 5' end for a first restriction enzyme, and a downstream recognition sequence for a second restriction enzyme followed by a<br>
downstream recognition sequence for a third restriction enzyme at its 3' end, such that the DNA units, once restricted by the first and second restriction enzymes can be ligated together to abolish the restriction sites at the ligation junction. In one particular example, the module has a Spe\ recognition sequence 5'ACTAGT3' at its 5'-end, and an upstream Xbal recognition sequence 5TCTAGA3' and a downstream Sma\ recognition sequence 5'CCCGGG3' at it's 3'-end.<br>
Particularly envisaged are modules wherein the DNA units encode polyketide synthetic domains, comprising two KS domains, at least two AT domains, two KR domains, two DH domains, two ER domains, an ACP domain and a TE domain. It is also envisaged that other non-polyketide enzyme domains can be included in the modules provided.by the invention.<br>
Also provided by the invention are vectors containing one or more modules. Particularly useful are vectors in which a non-functional recA gene is also present. Such vectors prevent unwanted homologous recombination occurring between domains within the vector upon integration into a suitable host by abolishing the recA gene activity in that host. Thus the invention also provides a method of transforming a host with one or more synthetic DNA "assemblies encoding enzyme domains which comprises the steps of:<br>
a)	Inserting said DNA assembly into a vector containing a mutated internal<br>
fragment of a recA gene sequence such that the vector is capable of<br>
undergoing homologous recombination with the recA gene of the host,<br>
b)	bringing said vector into contact with a host chromosome under<br>
conditions which permit homologous recombination to take place,<br>
c)	disrupting the host recA gene by the integration of the DNA of said<br>
vector into the chromosome. The expression vector can be used to<br>
transform a Steptomyces host. The DNA assemblies contained in the<br>
vector can be modules as described herein.<br>
Also envisaged are transformed hosts which prior to transformation with a vector containing one or more modules according to the invention, were already lacking a recA function.<br>
i<br>
In a further aspect the invention provides kits containing DNA units, DNA modules, vectors, DNA manipulation hosts, DNA modification hosts, expression hosts, or solid phase elements for use in the methods of the invention. For example, one such kit might contain a first DNA unit which is a vector suitable for transforming a suitable host, a library of modules for insertion into that vector, both the first DNA unit and the library having the necessary recognition sites for use in the methods of the invention, together with host strains suitable for the manipulation and expression of the DNA assemblies of the invention.<br>
A de novo "domain-by-domain" reconstruction of a hybrid rnultienzyme from the erythromycin-producing PKS has been achieved by the inventors by assembling DNA units corresponding to the constituent domains. The assembled gene was expressed in S. erythraea and the expected compounds were isolated from the bacterial broth. Application of this methodology, or variations of this methodology for making combinatorial assemblies of complex and aromatic PKSs allows for the rapid generation of novel or altered PKS or other synthetic multienzymes and paves the way for a quick and inexpensive synthesis of potentially bio-active molecules.<br>
One alternative to chemical syntheses is to carry out a 'retrobiosynthetic analysis' of the desired molecule, by pinpointing the exact number and type of synthetic enzyme domains that are required for every chemical step, and then assembling the DNA units that encode these enzymes in order to make a hybrid synthetic enzyme. The aim is therefore, to assemble these domains or even modules in a manner as desired, so that the linked enzymes can carry out a progressive synthesis of a desired target molecule. Until now, it has not been possible to find a methodology to assemble these PKS DNA units using restriction enzymes and DNA<br>
ligase to cut and join the DNA pieces together - one of the limiting factors being the non-availability of appropriate restriction enzyme sites in the DNA sequence of the enzymes which synthesise these polyketide drugs. There exist very few unique restriction enzyme sites and even fewer restriction enzymes that do not cut in the polyketide DNA sequence (i.e. are "non-cutters"). However, the restriction enzyme Xba\, because of its TA-rich recognition sequence (5'TCTAGA3'), does not cleave the majority of GC-rich polyketide gene clusters. Thus, flanking both ends of the DNA of the desired DNA unit (domain or module) with a recognition sequence that is cleaved on one end by Xba\, and on the other end by a restriction enzyme that is compatible with Xba\ (e.g. Spe\) is possible. A vectorial assembly, where such units are progressively joined, leaves one end of the unit that has been constructed by the ligation of Xba\ and Spel-cut DNA ends, not recognisable by either of the two enzymes, thus making further addition of units possible at only one of the two ends.<br>
This strategy makes use of selective recognition of the restriction enzyme site by the restriction enzyme Xbal, depending upon the sequence adjacent to the restriction enzyme site and upon the strain used (dam"1" or dam") during the assembly process. The method has been shown to be successful, and by using this methodology to assemble modules, the complete erythromycin-producing PKS (comprising of six modules coded by three large open reading frames) can be built in under 10 days. Even though this time-period is small compared to what it would take to assemble the ery PKS genes using conventional methodologies, using a variation of the above mentioned methodology, complete gene-clusters, like the 33 kbp erythromycin PKS, can be built within a matter of hours.<br>
Also described herein, is an approach wherein the assembly of the units itself can also be carried out in vitro without the need for an in vivo DNA modification step. Furthermore, employing the in vitro assembly methodology described below, one is now able to not only construct predetermined PKS genes, but also a randomly constructed combinatorial<br>
library of shuffled domains from one or more known synthetic enzymes. This has immediate and important implications for drug-discovery.<br>
The methodology thus outlined requires DNA units to be modified so that they contain the appropriate 5'and 3' ends (X and Xd respectively). These units are then progressively assembled to achieve the desired gene length. The vector containing the assembled or reconstructed gene is then used to transform an expression system to achieve protein expression. This methodology has been shown to work effectively - the hybrid muitienzyme DEBS1-TE was reconstructed by assembling de now the ten constituent domains. The assembled gene, when expressed in S. erythraea gave the expected six-membered triketide lactones.<br>
However, in the case of larger molecules like discodermolide, one would require a vectorial assembly of some 50 or so PKS units (if domains). A hypothetical PKS that would make a molecule as large as discodermolide would require 12 modules, each possessing the appropriate KS, AT, ACP and a set of reductive domains (e.g. KR, DH or ER). One would find that some of the domains in this group of 50 would be required to carry out the same catalytic function. For example, if all the hydroxy groups resulting from the ketoreductase activity from all 12 modules are of the same configuration, in effect 12 KRs that function in an identical fashion are required. Also, all 12 ACPs would, of course have the same catalytic function. It would therefore logically be more convenient, and less time-consuming if, to achieve ketoreduction from every one of the 12 modules, one used only one KR domain instead of 12 different ones in all the modules, or one ACP instead of 12 different ACPs. In fact, one can calculate that for every possible chemical reaction that can be carried out using PKS domains, one requires a set of only 12 domains, that in theory can be used repeatedly (Figure 1).<br>
It is possible that inter-modular recombination events within the reconstituted PKS or other synthetic enzyme gene, may preclude the use of identical PKS or other enzyme domain DNA units in a set of modules. It<br>
might be expected that, for example (Figure 2) the ACP* DNA in module 1 to recombine with the identical ACP* DNA in module 3. This event can take place, for example, when the expression vector that possesses the assembled gene containing numerous identical PKS DNA units is used to transform a streptomyces host for polyketide production.<br>
The inventors have developed a strategy that can circumvent this problem, therefore making it possible to construct large synthetic enzyme gene clusters using identical domains or modules repeatedly. This translates into a less expensive route towards synthetic enzyme gene construction (one would not require to have a start-up library of 200 or so to cover all possibilities), as the set of 12 domains, or similar functional arrangements of domains, are true "off-the-shelf" components for the assembly of PKS genes or genes for other hybrid synthetic enzymes.<br>
The inventors provide methods of DNA assembly that pave the way for a cheap and fast synthesis of a host of bio-active molecules, e.g. the anti-cancer drug Discodermolide.<br>
The examples that follow are better described with reference to the following figures:<br>
Figure 1 shows the chemical/stereochemical choices that each PKS domain can make. A total of 12 domains are required for every conceivable polyketide reaction.<br>
Figure 2 shows integration of a plasmid containing more than one identical DNA unit (ACP*). After the plasmid has integrated in the streptomyces host through homologous recombination with TE, internal recombination can occur to yield truncated PKS genes. This is because the host is recA*.<br>
Figures 3A and 3B show a schematic representation of the assembly process. The de novo construction of DEBS1-TE. DNA fragments (units) encoding for the constituent domains of the multienzyme DEBS1-TE were inserted sequentially into the expression plasmid pCJR24.<br>
The final plasmid pAR10 was then expressed in S. erythraea/JC2 to yield the expected triketide lactone products that are synthesised by the schematically shown re-assembied DEBS1-TE synthase.   The amino acid changes made within the linker regions between domains are shown below the actual amino acid sequence. Construction of the expression plasmid pAR10 and structural characterisation of the two triketide lactones shown in the above figure is described in the methods section.   X - Xbal restriction enzyme recognition sequence   (5TCTAGA3'),   Xd   -  Xbal   and   Dam   methylase   recognition sequence (5'GATCTAGA3')<br>
Figure 4 shows the methodology of the assembly of DNA units using Xbal/dam methylase technology. During the second last stage of assembly, indicated as transform and cut in the figure, transformation of a Dam'strain with plasmid (as it is a dam'strain, even Xd would be cleaved by Xjbaf) is effected. Cutting is achieved by Xbal and the DNA unit purified on a gel.<br>
Figure 5 shows the procedure for the assembly of DNA units using Xbal/dam methylase technology.<br>
Figure 6 shows how an Xbal site can be made sensitive to methylation. The RE cuts at the sites shown by arrows. The boxed sequence is methylated in a dam*strain thereby altering the Xba\ recognition site. The sequence however is not methylated in a dam strain, and so can still be cleaved by Xbal. The Xbal recognition sequence (5TCTAGA3') can therefore be selectively cleaved by Xbal. Assembly of DNA units uses only one restriction enzyme - Xbal.<br>
Figure 7 shows the methodology of the in vitro assembly of DNA units - I using solid phase beads with the enzymes Xbal, Spel and Smal (other Xbal -compatible REs may be used).<br>
Figures 8 and 9 show how the methodology of the in vitro assembly of DNA units - II would proceed to the point of placing the DNA assembly into an expression vector for transforming and appropriate host. In vitro assembly of DNA units (domains) from the first multienzym'e of erythromycin - producing PKS.<br>
Figure 10 shows how in one single ligation, 16 ongoing assemblies are generated. This cascade can obtain exponential proportions. The gene library can be increased by increasing the diversity of the incoming unit.<br>
Figure 11 shows the integration of an expression plasmid into a streptomyces host, using a mutated internal fragment of the recA gene as the region for homologous recombination. The resulting PKS gene can now contain more than one identical DNA units as the strain has been made recA minus.<br>
Figure 12 shows the assembled PKS recADEBS1-TE. The second module is composed of domains that normally belong to the first module.<br>
Figure 13 shows the amino acid sequence alignment of the recA protein of S. lividans (S.I.) and S. ambofaciens (S.a).   Percent similarity:  96.496, percent identity: 95.418. Match display thresholds for the alignment(s): I = identity : = 2 . = 1<br>
Figures 14A and 14B show a DNA sequence alignment of the recA gene S. lividans (S.I) and S. ambofaciens (S.a). Start of the gene is from 'ATG' and stopis 'TGA'. Percent similarity: 94.713, percent identity: 94.713.<br>
Figure 15 shows how an Xoal/Spel system might be used instead of an Xoal/dam methylase system to assemble DNA units, a strategy involving compatible restriction enzymes flanking either end of a DNA unit. An example<br>
of compatible REs would be Xba\ and Spet. The recognition sequence of Xbal is - 5TCTAGA3' and that for Spel is 5'ACTAGT3'. After Xbal and Spe! have cleaved the DNA at their respective sites, the DNA unit can be ligated together as the overhanging is complementary. The junction where any two units are joined is now recognised by either Xbal or Spel.<br>
Figure 16 is a schematic representation of the compatibility of Xba\- and Spel-digested DNA overhangs.    It shows the compatibility of the sticky ends produced by Xbal and Spel and how re-Jigation abolishes both sites. Figure 17 shows a schematic representation of the erythromycin-producing polyketide   synthase;   primary   organisation   of   the   genes   and   their corresponding protein domains,    The multienzymes deoxyerythronolide B synthase 1 (DEBS1), DEBS2 and DEBS3 each have two modules, each of which processes one cycle of polyketide chain extension.   Each of the six modules is constituted by covafently-linked enzymatic domains.   Exploitation of such an  enzymatic hierarchy as  "of-the-shelf. reagents can  lead to synthesis of important chemical compounds.<br>
Figure 18 shows the structure of the anticancer drug discodermolide (top) and the 'retrobiosynthetic approach' towards synthesising a target molecule (a discodermolide). Such an approach would involve opening up the structure (a.), identifying the number and type of polyketide carbon units that would make the discodermolide carbon skeleton (b.), and choosing the PKS DNA units (modules/domains) responsibte for the uptake and subsequent processing of the carbon units (c.).<br>
Figure 19 shows the anti-tumour compound octalactin and the strategy behind the retrobiosynthetic approach towards synthesising bio-active molecules. The strategy comprises the steps of:<br>
Identify polyketide units - e.g. whether acetate, propionate, etc,<br>
Break-up and identify - break up the carbon skeleton and identify how many such carbon units are present. Eight units would mean-one requires eight modules to make a PKS.<br>
Choose - choose the modules or domains that would be required, form an existing library of such PKS modules and domains.<br>
Assemble - assemble the DNA units (modules/domains/using the invention.<br>
Express - express the assembled gene in a host and check for compound production.<br>
Figure 20 shows a schematic representation of they hypothetical polyketide synthase for synthesising octalactin B, assembled from enzyme units that belong to various PKSs in the public domain.<br>
Figure 21 shows a schematic representation of the hypothetical decarestrictine polyketide synthase for synthesising the anti-cholesterol compound decarestrictine J, assembled from enzyme units that belong to various PKSs in the public domain.<br>
Examples<br>
Example 1: Vectorial assembly of DNA units<br>
DNA units that are to be assembled contain the Xba\ recognition sequence at either end of the unit. At one of the ends, two nucfeotides (GA) are arranged at the 5' end of the Xbal recognition sequence (thus making it 5'GATCTAGA3'). This is achieved by first incorporating the Xbal recognition sequences in the oligonucleotide primers and then amplifying the desired DNA unit by PCR. The PCR products are then ligated to a pUC-18 vector, used to transform a dam+ strain of E. coli, and the clones isolated and sequenced for possible errors in the PCR products. A dam4 strain of £ coli - like DH10B™ - methylate the nucleotide A in the sequence GATCTAGA, as 5'GATCS' is a sequence that is recognised by the product of the Dam methylase gene (Fujimoto et al., 1965; Geier era/., 1979). This makes only one end of the DNA unit cleavable by Xbal. The vector is then used to transform a dam" strain 'Of E. coli (e.g. ET12567 -MacNeil et at. (1992)) and the plasrnid DNA isolated. This DNA is now cleavable at both ends of the DNA unit by Xba\. When a library of units has been constructed using this strategy, and both ends of these units have been cleaved by Xbal, they are progressively inserted into a vector that has a unique Xbal site and the ligated products are used always to transform a dam* strain of E. coli, thereby making sure that one end of the DNA unit is always protected from cleavage by Xbal through methylation. When the assembly of such units is completed, the final plasmid is integrated into a streptomyces strain for the production of the desired polyketide.<br>
Using this methodology, the polyketide synthase DEBS1-TE, a multienzyme that has the first of the three bimodular erythromycin DEBS enzymes (DEBS1), fused with the erythromycin thioesterase (Cortes er al., 1995) was constructed in a de novo fashion. The ten inherent PKS domains in DEBS1-TE, namely, loading module (itself composed of an AT and an ACP), KS1 (ketosynthase of module 1), AT1, KR1, ACP1, KS2<br>
(ketosynthase of module 2), AT2, KR2, ACP2 and TE function in conjunction to catalyse the synthesis of (2R,3S,4S,5R)~2,4-dimethyl-3,5-dihydroxy-n-hexanoic acid 8-lactone (2), figure 3.<br>
The DNA for all ten domains was amplified by PCR to incorporate the two aforementioned recognition sequences forXbal (5TCTAGA3' and 5'GATCTAGA3') at the 51 and 3' ends of the DNA unit respectively. The PCR products were cloned in pUC18 vector, sequenced, and then used to transform the dam" E. coli ET12567 strain. To initiate the assembly process, the DNA unit forTE was inserted into S. erythraea expression vector pCJR24 (Rowe et al., 1998) which has a unique Xba\ site. This vector also contains a thiostrepton-resistance gene as a marker for identifying successful integrands. The ligated products were used to transform the dam* E. co//DH10B™ strain and the plasmid DNA isolated. This plasmid (pAR1) can only be singly cleaved with Xbal, despite possessing two Xba\ recognition sequences, as one of the sites (situated at the 3' end of the TE unit) has been methylated by the E. coli Dam methylase. The next DNA unit (ACP2 from module 2 of DEBS1) was then ligated to the Xdal-cut pAR1, the ligation mixture used to transform DH10B cells and the plasmid DNA isolated. Likewise, the other eight DNA units were successively added to pAR1 to finally yield the expression plasmid pAR10 containing the reconstituted DEBS1-TE gene (Figure 3). The junctions where these domains were joined were chosen in the linker regions that lie between these domains, so as to cause minimum disturbance of the structural features of these domains, that might in turn affect the proficiency of the domains themselves (Figure 3). Plasmid pAR10 was then used to transform S. erythraealJC2 - a mutant strain of the wild-type S. erythraea NRRL2338 thai lacks the DEBS genes except for the TE DNA fragment (Rowe etal., 1998). Thiostrepton-resistant colonies were selected upon integration of the vector into the S. erythraea chromosome. Single transformants were grown on selective media, as described in the methods section. The fermentation broth was extracted with ethyl acetate<br>
and a sample of the organic extract was analysed by gas chromatography-mass spectroscopy (GC-MS). Two peaks were observed, corresponding to molecular massess 158 and 172, indicating the presence of the expected acetate- and propionate- derived polyketides (2R,3S)4S,5R}-2,4-dirnethyl-3,5-dihydroxy-n-pentanoic acid d-lactone (1) and (2R,3S,4S15R)-2,4-dimethyl-3,5-dihydroxy-n-hexanoic acid d-lactone (2). Both compounds were isolated and fully characterised by high-pressure liquid chromatography (HPLC), 1H 1D and 2D NMR, 13C NMR, FT-ICR spectrometry, and by comparison with a synthetic standard of (2) (Brown ef al., 1995). One litre of fermentation broth produces 24 mg of (1) and 56 mg of (2) - yields that are comparable to those reported elsewhere (Lau et aL, 1999). It can therefore be asserted that the ten newly constructed inter-domain junctions have not in any way dimmed the catalytic proficiency of the DEBS1 -TE synthase.<br>
!n the absence of any crystal-structure data on PKS domains, all genetic engineering efforts known in the art have been based on trial-and-error methods of experimenting with where to join two such domains, As a result, the yield of the synthesised polyketide products have varied depending upon the position in the polypeptide chain at which the domains or modules have been linked (McDaniel et aL, 1999; Ruan ef al,, 1997). The successful functioning of the reconstructed polyketide synthase described above has supplied new information about the inter-domain junction sites. Using this information, and the described methodology for the rapid assembly of these enzyme units, it is now possible to carry out a 'retrobiosynthetic analysis' of target molecules and then to use polyketide and other biosynthelic enzyme domains as truly 'off-the-shelf reagents to achieve a stereospecific synthesis. There is also the possibility of using this methodology for randomly combining DNA units that encode catalytic e.g. DH or transport e.g. ACP protein domains to generate combinatorial libraries of hybrid synthases. By using a suitable assay system to test for biological activity of the compounds that are generated by such means, it is<br>
possible to go back and isolate the hybrid synthetic gene resposible for the production of these compounds.<br>
From 6-methylsalicilic acid to maitotoxin, nature displays a staggering diversity in compounds that are synthesised by means of 'combinatorial gene-shuffling'. This methodology, or variations of this methodology can be used as effective tools towards harnessing the combinatorial potential of discrete enzymatic units or their sets that are the feature of multi-functional PKS and other systems.<br>
A similar system to the Xba\/dam system described above, uses the restriction enzyme Fokl which has the recognition site:<br>
5'GGATG(N)9↑3' 3'CCTAC(N)13↑5'<br>
with the dcm methylase of E.coli. Adding CCA or CCT to the 5' end of the Fokl recognition site would make the site dcm sensitive. Furthermore, if the sequence TCTAGA were inserted into the redundant section of the Fokl restriction site, then the enzyme could be used to generate 'Xbal-cut ends'. Methods<br>
E. cotf dam* DH10B™ strain was purchased from Gibco BRL, USA.. Pfu DNA polymerase was purchased from Boeringer, Germany.<br>
Construction of the final expression plasmid pAR10 was carried out in several steps, as follows. The ten PKS DNA units were amplified by PCR using pfu DNA polymerase. The respective regions of eryAl gene, as well as the oligonucleotides used for each PCR are outlined: LM - segment of eryAlgene (Bevitt er al., 1992) extending from nucleotide (N) 588 to N 2389;<br>
5'GGCATATGGCGGACCTGTCAAAGCTCTCCGACAGT3'and 5'GGTCTAGATCCCAGCCGCGGTCGGTCGGCAGTCCCG3'i KS1 - segment of eryAI gene extending from N 2384 to N 3769; 5'GGTCTAGACTCGCTGTTCCACCCCGACCCCACGCGCTCGGGCACC GCGCACCA31 and<br>
5'GGTCTAGATCGCGCAGCGCGGCGGACTCGTCGACGGGGGCGAAG<br>
GCGG3',<br>
AT1 - segment of eryAlgene extending from N 3764 to N 4813;<br>
5'GGTCTAGACGGTCTCGCGACGGGAAACGCCGACGGTGCCGCCGTT<br>
GGAA3'<br>
and<br>
5'GGTCTAGATCCACCGCGACACCGGCGGCGAACGCGCGGGAGAGC<br>
GCTTCGC31,<br>
KR1 - segment of eryAi gene extending from N 4808 to N 6316;<br>
5'GGTCTAGAGTCGGTGCACCTGGGCACCGGAGCACGCCGGGTGCCC<br>
TT3'<br>
and<br>
5'GGTCTAGATeGTCGAAGAGCCTGGTCGGGCGCTGCGCGGTGTA3',<br>
ACP1 - segment of eryAlgene extending from N 6311 to N 6679;<br>
5'GGTCTAGACGACGCGCGGCGGGCTGCGCCGCAGGCGCCGGCCGA<br>
ACCGCGGG3'<br>
and<br>
5'GGTCTAGATCGGCCGTGG-TCGCCGGTGCCGCCTGCTCGGCT3',V<br>
KS2 - segment of eryAlgene extending from N 6674 to N 8200;<br>
5'GGTCTAGACGAGCCGATCGCGATCGTCGGCATGGCGTGC-<br>
CGGCTGC3'<br>
and<br>
5'GGTCTAGATCGTGCACGGCCTCGGCGGTGTCGGCGGCGAGC-<br>
ACCGCGGCCCGCTCCTC3',<br>
AT2 - segment of eryAlgene extending from N 8195 to N 9340;<br>
5'GGTCTAGAGGCGGTGGCCGACGGCGCGGTGGTT3'<br>
and<br>
5'GGTCTAGATCGTCACGAGGGGTGGTGCGGTCCGGCAGCAGCCAGA<br>
A3',<br>
KR2 - segment of eryAlgene extending from N 9335 to N 10639;<br>
5'GGTCTAGACGGCTGGTTCTACC-GGGTCGACTGGACCGAG3'<br>
and<br>
5'GGTCTAGATCCGGCCGGGGCCGGGCGGCGG-TGTAGGACT3',<br>
ACP2 - segment of eryAlgene extending from N 10634 to N 10966;<br>
5'GGTCTAGACCGCATCGTCACGACCGCGCCGAGCGA3'<br>
and<br>
5'GGTCTAGATCG-GCGTCGAGGAAA3',<br>
TE - segment of eryAIII gene (Donadio et al. 1991) extending from N 8753<br>
to N 9602; 5'GGTCTAGACAGCGGGACTCCCGCCCGGGAAGCG3'<br>
and<br>
5'GGGCTAGCTCTAGATCATGAATTCCCTCCGCCCAGCCAGGCGTC3'.<br>
All PCR products were 5' phosphorylated and ligated to Smal-cut, dephosphorylated pUC18 vector and used to transform E. coliDH10B electrocompetent cells. The desired plasmids - containing the amplified DNA fragments were isolated and sequenced using standard pUC forward and reverse primers. No mistakes in the amplified products were detected. All ten plasmids were then used to transform the E.coli ET12567 dam" strain. Isolated DNA was digested with Xbal restriction enzyme and desired fragments isolated and purified. The TE unit was then ligated to Xbal-cut pCJR24 vector and the ligation products used to transform E. coli DH10B electrocompetent cells. Plasmid pAR1 was isolated, digested with Xbal, and ligated to the ACP2 fragment, and ligation products treated as mentioned above. The other DNA fragments, namely, KR2, AT2, KS2, ACP1, KR1, AT1 and KS1 were sequentially added to finally yield plasmid pAR10. This plasmid was then digested with Nde\ and Xba\ restriction enzymes and ligated with the LM fragment previously digested with the same two enzymes. The ligated products were used to transform E. coli DH10B electrocompetent cells and the final expression plasmid pAR10 isolated. Plasmid pAR10 was then used to transform S. erythraea/JC2 strain and colonies carrying the expression plasmid were selected through resistance to thiostrepton upon integration of the plasmid into the S.    . erythraea chromosome. Single transformants were picked and grown on<br>
tap-water medium plates supplemented with thiostrepton, following which single transformants were grown in 5X200ml of SM3 liquid media supplemented with 5 ug/ml of thiostrepton for seven days (Rowe et al., 1998). Cells were removed by centrifugation, the supernatant was saturated with NaCI and extracted three times with equal volumes of ethyl acetate at pH 4.0. The solvent was evaporated to yield 1.12 g of crude product. A sample of this crude product was analysed by GC-MS. Two peaks were observed, corresponding to molecular masses 158 and 172, indicating the presence of the expected acetate- and propionate- derived polyketides (2R,3S,4S,5R)-2,4-dimethyl-3,5-dihydroxy-n-pentanoic acid 5-lactone (1) and (2R,3S)4S,5R)-2,4-dimethyl-315-dihydroxy-n-hexanoic acid 8-lactone (2). Compounds (1) and (2) were found to be structurally identical to those reported previously (Cortes et al,1995). Characterisation of(2R.3S,4S,5R)-2,4-dimethyl~3,5-dihydroxy-n~pentanoic acid 8-lactone (1)<br>
1H NMR (CDCI3, 500 MHz) δH 4.45-4.35 (1H,dq, J = 6.56 and 1.62 Hz, C5-H), 3.8 (1H, dd, J = 10.15 and 4.17 Hz C3-H), 2.45-2.70 (1H, br, O-H), 2.42 (1H, dq, J = 10.0 and 6.97 Hz C2-H), 2.05 (1H, m, C4-H), 1.37 (3H, d, J = 7.17 Hz, C2-CH3), 1.32 (3H, d, J = 6.74 Hz, C5-CH3), 0.95 (3H, d, J = 7.20 Hz, C4-CH3) ppm. 13C NMR (CDCI3, 250 MHz) 6 174.20, 76.15, 73.62, 39.42,38.14,18.11,14.24,4.48.<br>
Characterisation of(2R,3S,4S,5R)-2,4-dimethyl-3,5-dihydroxy-n-hexanoic acid 8-lactone (2)<br>
1H NMR (CDCI3, 500 MHz) dH 4.13 (1H, ddd, J = 8.12, 5.93 and 2.19 Hz, C5-H), 3.82 (1H, m, C3-H), 2.42-2.50 (1H, dq, J = 10.17 and 7.08 Hz, C2-H), 2.12-2.19(1H, m.OrH), 1.77-1.86(1 H, m, one of C6-H2), 1.52-1.61 (1H,m, one of C6-H2), 1 -4 (3H, d, J = 7.09 Hz, C2-CH3), 1.0 (3H, t, J = 7.42 Hz, C6-CH3), 0.97 (3H, d, J = 6.96 Hz, C4-CH3) ppm. 13C NMR (CDCI3, 250 MHz) d 173.56, 81.34, 73.96, 40.08, 36.76, 25.27, 14.27, 9.88, 4.37.<br>
[Example 2: in vitro assembly of DNA units<br>
Figure 7 outlines the strategy for the in vitro assembly of PKS DNA units. The inventors have constructed the multienzyme DEBS1-TE. The in vivo construction of the gene for DEBS1-TE, it should be noted, took 12 days to complete. The in vitro assembly on the other hand was completed in 2 days.<br>
All ten domains, namely, LM, KS1, KR1, AT1, ACP1, KS2, AT2, KR2, ACP2 and TE were amplified by means of PCR. The forward primer in all cases, except the LM contained the Spe\ recognition sequence 5'ACTAGT3' while the reverse primer was engineered in such a way that it contained the Xbal recognition sequence 5' TCTAGA3' and Sma\ recognition sequence 5'CCCGGG3' downstream of the Xbal site (Figure 7). The amplification of the LM was carried out using a biotinylated forward primer and a reverse primer that contained the Xbal recognition sequence (5TCTAGA31), All the PCR products were cloned in pUC-18 vector and the resulting plasmids sequenced to detect possible errors introduced by PCR. All plasmids, except the one containing the LM unit were then digested with Spe\ and Sma\, dephosphorylated in order to remove the 5' phosphate group and the appropriate fragments isolated and eluted. The LM unit was cleaved with Xba] and attached to a bead that was coated with streptavidin (following the manufacturer's instructions) as shown in figure 7.<br>
The assembly process was initiated by adding DNA ligase to the tube containing a large excess of the first unit (KS1) and LM-bead. The reason for having a large excess of the KS1 unit compared to the LM-bead unit is to favour the LM-bead ligating to the incoming unit, as opposed to the self-ligation of the LM-bead (see figure 7). The ligation of the two DNA fragments is unidirectional as only the Spel-cut end of KS1 complements the Xba\-cut end of the LM-bead. After the ligation was complete, the desired product of the ligation reaction, namely 'bead-LM-KS1' was separated from the reaction mixture and washed. This product was then cleaved with Xbal, in order to activate the 3' end of KS1. The beads were washed again to remove the small Xba\-Sma\ DNA fragment that was<br>
released from the 3' end of KS1 as a result of RE cleavage. The 'activated' bead-LM-KS1 unit was then ligated with Spel, Sma\-cui and 5' dephosphorylated AT1. The Spel-cut 5' end of AT1 complemented the Xba\-cu\ 3' end of KS1 to give bead-LM-KS1-AT1 as shown in figure 8. This product was separated from the reaction mixture and washed as before. The 3' end of AT1 in this product was then 'activated' through cleavage by Xba\, and the assembly process continued.<br>
Finally, Spel, Smal-cut and 5' dephosphorylated TE unit was ligated with the DNA fragment that was now bead-LM-KS1-AT1-KR1-ACP1-KS2-AT2-KR2-ACP2 as shown in figure 9. The 3' end of the latter fragment was 'activated' by digesting it with Xbal. The assembled DEBS1-TE gene was then inserted in the expression plasmid pCJR24 and the resulting plasmid used to transform a streptomyces strain. The expected triketide lactone products were isolated and structurally characterised.<br>
Use of the in vitro technology described above drastically reduces the time it takes to assemble predetermined or- randomly shuffled genes. Also, the possibility of continuing with the assembly process while having numerous different assembly arrays attached to the beads, and splitting and mixing the beads between each unit/module addition from a library of units/modules, results finally in the generation of a cascade of different assemblies (Figure 10). These assembled genes can then be cloned simultaneously and expressed in a suitable host. An assay system can then be used to identify those assembled genes that yield bio-active compounds.<br>
Example 3: Retrobjosynthetic synthesis of a target molecule<br>
A strategy employing the invention in order to construct the highly potent anti-breast cancer drug discodermolide, the anticholesterol compound decarestrictine, and the antitumour compound octalacin using polyketide synthase domains/modules is outlined below.<br>
Discodermolide<br>
The drug discodermolide (Figure 18), isolated from the marine sponge 'Discodermia disoluta', has been identified as a highly potent anti-cancer compound and 80 times more effective than the well known anticancer drug Taxol (TerHarr et al., 1996). It has the same mechanism of action as Taxol, even though it is structurally different from the latter.<br>
One can infer from its structure (Figure 18) that discodermolide is a polyketide and can therefore be constructed from a system that has the basic enzymatic building blocks (domains and modules) that make other polyketides   like   erythromycin   and   rapamycin.   Having   predicted   that approximately 45 domains housed in 12 modules would be required in order to carry out the chemistry that accounts for the functionalities on the carbon skeleton of discodermolide, one can now begin to construct such a system. All one has to do is to identify the type and nature of the domains/modules    that    one    requires    to    generate    the    observed functionalities, and then assemble these units in the desired order (Figure 18). The resulting DNA assembly can then be put into a bacterial strain that makes a functional polyketide synthase.<br>
Until now, it would have been exceedingly difficult, if not impossible to assemble 45 or so pieces of DNA in the wanted order, for several reasons. Firstly, one would have to look for two different restriction enzymes every time one needed to assemble two DNA segments. This is because if one uses just one restriction enzyme at either end of the<br>
domain, the already-assembled piece/pieces of DNA would be cleaved from the  assembly every time one decided to insert a new domain. Secondly,   in   GC-rich   DNA   like   the   polyketide   synthase   producing Streptomyces strain,  unique  restriction enzyme sites are few and far between. To a molecular biologist, the task of assembling 40 pieces of DNA with the limitations mentioned above, would seem an insurmountable one. One would rather attempt to isolate the genes that make the drug at the first place than consider carrying out "step-by-step" reconstruction of the gene itself. In the case of discodermolide, even the last possibility is in the realms of fantasy. The organism within the marine sponge that makes the drug has not been identified. The only way discodermolide can be made available is through chemical synthesis - there have been a few chemical routes reported in literature recently (Marshall and Johns, 1998 and references therein). However, as is the case with most other complex molecules, large scale production of discodermolide, using the chemical route would turn out to be outrageously expensive. Chemists have been using the  retrosynthetic analysis approach towards total  synthesis of important bioactive molecules. This approach breaks the target compound into many smaller pieces -  easily synthesised  - which  are then reassembled.<br>
The type of polyketide or other synthetic enzyme domains required in order to construct the target molecule from the starting units are identified using a "retrobiosynthetic analysis" approach for discodermolide,<br>
by matching which molecules need to be condensed to form the macromolecule with the enzyme domains that carry out the required catalysis to build the macromolecule.<br>
Having identified the enzyme units that are required, the unit-DNA segments are amplified using the polymerase-chain-reaction (PCR) - from the library of existing polyketide synthase unit-DNA, and the appropriate recognition sequences are attached to each unit-DNA fragment. All of the unit fragments are then replicated in a dam" strain whereby both the unmodified and modified sequences (5'TCTAGA3' and 5'GATCTAGA3' respectively) are cleaved by the restriction enzyme Xba\. Having constructed this library of appropriate PKS or other synthetic enzyme units, the corresponding DNA units are then assembled. The assembled DNA piece is then placed in a vector, so that it can be inserted in a bacterial strain to yield the desired synthetic protein. Suitable vectors have an antibiotic resistance marker (for selection of this vector on an<br>
/<br>
antibiotic-rich media) and an "origin-of -replication" (ori). Ori is essential for the independent growth of the vector in any strain. Particularly suitable vectors for the expression of the synthetic enzymes of the invention are the actinomycete vectors described by Rowe ef a/. (1998).<br>
The strain is then grown in a media that is supplemented with the antibiotic, the resistance gene for which is present in the vector.<br>
Figures 4 and 5 show how the assembly proceeds. The first domain is inserted into a vector that is cut by cleavage with Xba\. After the ligation<br>
of the domain has taken place with the vector, the DNA is put in a bacterial-    . strain that is dam* and grown. Finally, bacterial colonies that have the desired vector-domain DNA are identified and DNA isolated from them. The whole procedure is cheap and fast. Only one restriction enzyme (Xba\) is made use of, routine cloning technology is employed, the desired DNA fragment is obtained, which can then be expressed in a Streptomyces strain to yield the polyketide synthase.<br>
The in vivo "domain-by domain" construction of the discodermolide producing polyketide synthase would take approximately 55 days via this method. In comparison, assembly of modules would take less time, as one would need to assemble fewer pieces. Most importantly, once the synthase is shown to be functionally active, a large fermentation of the bacterial strain can be carried out, and the drug isolated in however much quantity one requires - unlike the chemical route where the starting materials have to be freshly synthesised every time one requires the target compound. Employing such a strategy would lead to a quick and inexpensive synthesis of important bioactive molecules like discodermolide. Retrobiosynthetic analysis<br>
The whole approach (retrobiosynthetic analysis followed by identification of PKS units, followed by assembly of PKS units) is made clearer in the following two examples.<br>
Octalactin<br>
A new addition to the rare class of eight-membered lactone natural products is the family of Octalactin. Octalactin A and B (Figure 20) are natural products isolated from the marine gorgonian octocoral 'Pacifigorgia sp.' (Tapiolas et. a!., 1991). Octalactin A shows very strong cytotoxicity toward B-16-F-10 murine melanoma and HCT-116 human colon tumour cell lines and is a promising drug candidate, while octalactine B displayed no such activity (Tapiolas et. al., 1991). Total syntheses of both octalactin A and B have been reported in literature. One such synthesis (Buszek, et. al., 1994) typically involves more than 12 chemical steps in leading to the target  molecules.  Clearly,   large-scale  production  of  octalactins  using chemical synthesis is industrially not viable. On the other hand, the genes that code for the enzymes that make octalactins have not be identified or isolated. This means that at present, modified octalactins can only be made using chemical synthesis, A gene is constructed from the available PKS spare parts - that would code for the enzymes that would make octalactin B. Octalactin B can then be converted into the cytotoxic octalactin A by one-step stereospecific epoxidation. Also, once the gene for octalactin B is constructed and shown to make the octalactin PKS, genetic engineering on this  gene  would  yield  modified   octalactin   PKSs  that  in  turn  would synthesise octalactin analogues.<br>
Clearly, a polyketide, the carbon skeleton of octalactin B (Figure 19} can be seen to be assembled by acetate and propionate units. The uptake<br>
and assembly of these units in the prescribed sequence, as well as the functionalities that decorate the carbon chain of octalactin can be assigned to various PKS modules (see figure 19). Once a decision has been made regarding the type and nature of PKS modules, they can be strung together to make a gene using the invention. This gene can then be expressed in a suitable   host   in   order   to   look   for   octalactin   B   production.   The retrobiosynthetic approach towards octalactin is shown in detail in figure 19. A choice of what modules to select from the PKS module library is followed   by   amplification   of  the   modular  DNA   fragments   using  the oligonucleotides such that the 5' and the 3' ends of every DNA fragment have the restriction enzyme recognition sites stated under the description of the invention. The choice of modules that, when assembled, would make the 'octalactin gene' is displayed as a schematic representation in figure 20. Decarestrictine J<br>
The molecule decarestrictine J can be synthesised using the retrebiosynthetic approach. Decarestrictine J is a ten-membered lactone that comes from the family of decarestrictines, shown to display strong anti-cholesterol activity (Grabley et. al,, 1992). The total synthesis of Decarestrictina J has been reported and involves numerous chemical steps (Yamada et. al., 1995). The target molecule (figure 21) can be conceived to be formed by assembly of five acetate polyketide units. Using the retrobiosynthetic approach, one can identify the PKS domains/modules that<br>
would be required for the carbon skeleton of decarestrictine J. A hypothetical decarestrcitine PKS is shown in figure 21. The loading module, as well as the four internal modules along with the TE domains can be conveniently assembled using the invention. The assembled 'decarestrictine gene1 can then be expressed in a suitable host in order to check for the production of decarestrictine J.<br>
In summary, the retrobiosynthetic approach involves the following<br>
steps;<br>
a). Identification of the number and nature of carbon units that make up the<br>
target molecule<br>
b).     Identification     of     the     modules/domains     from     libraries     of<br>
polyketide/peptide synthetase/fatty   acid/etc,    encoding   units   that   are<br>
responsible for the uptake of the said carbon units and the nature and<br>
degree of functionalisation of the carbon chain<br>
c). Assembly of the said modules/domains using the methods of the<br>
invention<br>
d). Expression of the assembled gene in a suitable expression host.<br>
Example 4: Transforming strains with DNA encoding similar synthetic enzyme domains<br>
A method for transforming expression strains with DNA encoding similar synthetic enzyme domains has been devised. Instead of using the TE PKS DNA fragment as a region of integration from the assembled gene into a streptomyces host (S. erythraeaJJC2, Rowe etal., 1998), a mutated recA gene fragment from streptomyces is used. The assembly process is carried<br>
out in a recA' E. constrain (e.g. DH10B) as previously described. As this strain is recA", one can assemble any number of identical DNA units. The vector, into which the assembled gene is being constructed, contains a portion of a streptomyces recA gene. This recA fragment carries a mutation. After the synthetic enzyme gene has been assembled, the vector is used to transform a streptomyces host (e.g. S. lividans or S. eryihraea). The fragment of recA gene carrying a mutation recombines with the recA gene of the streptomyces host, abolishing the functional recA gene and making the strain recombination minus (Figure 11). This means that an event, such as the one described in figure 2 is now not possible. The strain s then grown to look for the encoded enzyme product. This strategy is cested by assembling a functional PKS gene having more than one type of dentical DNA units (Figure 12).<br>
Construction of the PKS multienzyme recDEBS1-TE RecA protein has been characterised as a multifunctional enzyme that is essential for homologous recombination, DNA repair, SOS response and DNA rearrangements (Miller and Kokjohn, 1990). Most of the routinely used strains of E. coli are recA'. The gene for recA has been identified from many streptomyces strains. The first streptomyces recA gene to be characterised and isolated was from S. lividans (NuBbaumer and Wohlleben, 1994) RecA mutants have since been generated in S. ambofaciens (Aigle el al., 1997). The streptomyces recA protein has approximately 372 amino acid residues (Figure 13). DNA sequence analysis suggests a coding region of 1122 bp, and is found to be highly conserved within streptomyces (Figure 14). In fact the recA mutants of S. ambofaciens were generated by integrating a mutated portion of the S. lividans recA gene into the S. ambofaciens host. It was found that a recA mutant lacking 30 aa from the C-terminus of the protein inhibited recombination events in S. ambofaciens (Aigle et al., 1997).<br>
A recA mutant of the streptomyces host that is used for expression of the assembled gene was generated.<br>
The oligonucleotides:<br>
5'- GGTCT/4GMTTCGGCAAGGGCGCCGGTCATGCGCAT-3' and .   5'-GGTCLAGATCTGCGGCGTCGGCCGGGGCGGCGGAGGCG-3' were used as the forward and reverse primers respectively and the 1000 bp internal region of S. lividans recA gene (NuBbaumer and Wohlleben, 1994) was amplified using pfu polymerase. An additional nucleotide (C) was incorporated into the forward primer to generate a frame shift in the amplified recA gene fragment. The PCR product was cloned-in pUC-18 vector and sequenced to detect for possible errors during PCR. The 1.0 kbp recA fragment, flanked at both ends by an Xba\ site was then inserted in the expression vector pCJR24 that has a unique Xba\ site. The ligation mixture was used to transform E. co/i DH10B cells and the desired plasmid DNA   isolated.   The   resulting   plasmid   (pARecA24)   contains   a   non-methylated Xba\ site at the 5' end of the recA gene fragment. The ten PKS DNA units, namely, TE, two each of ACP1, KR1, AT1 &amp; KS1, and LM were inserted into the plasmid phRecA24 to finally yield the expression plasmid pRecAD1TE. This plasmid was used to transform wild-type S. lividans protoplasts, and thiostrepton resistant colonies were grown in defined liquid media as described above. The compound (Figure 12) was isolated from the bacterial broth and chemically characterised.<br>
Thus, it has been shown that a gene carrying interspaced DNA units that are identical in structure as well as function does not lead to internal recombination events, as the native recA gene of the streptomyces host has been disrupted. Furthermore, it has been shown that it is possible to use identical domains to reach the objective of generating hybrid synthetic enzyme systems. This strategy will greatly reduce the number of domains that otherwise have to be employed for the purposes of de novo PKS gene assembly that yields the desired chemical compounds. The inventors have established a set of 12 domains that are capable of functioning robustly and are independent of flexibility and spacial constraints - problems that beset the choice of domains and modules previously.<br>
References<br>
Aigle, B., Hoil, A-C., Angulo, J.F., Leblond, P. and Decaris, B. (1997) Characterization of two Streplomyces ambofadens recA mutants: identification of the recA protein by immunoblotting. FEMS Microbiol. left., 149, 181-187.<br>
Bevitt, D.J., Cortes, J., Haydock, S.F. and Leadlay, P.F.-(1992) 6-Deoxyerythronolide B synthase 2 from Saccharopolyspora erythraea. Cloning of the structural gene, sequence analysis and inferred domain structure of the multifunctional enzyme. Eur. J. Biochem., 204, 38-49.<br>
Brown, M.J.B., Cortes, J., Cutter, A.L, Leadlay, P.P. and Staunton, J. (1995) A mutant generated by expression of an engineered DEBS1 protein from the erythromycin-producing polyketide synthase (PKS) in Streptomyces coelico/or produces the triketide as a lactone, but the major product is the nor-analogue derived from acetate as starter acid. J. Chem. Soc,, Chem. Commun., 1517-1518.<br>
Buszek, KR., Sato, N. and Jeong, Y.M. (1994) Total synthesis of octalactin-A and octalactin-B. J. Amer. Chem. Soc. 116, 5511-5512.<br>
Carreras C. and Khosla C. (1998) Purification and in vitro reconstitution of the essential protein components of an aromatic polyketide synthase. Biochemistry 37,2084-2088.<br>
Cortes, J., Wiesmann, K.E.H., Roberts, G.A., Brown, M.J.B., Staunton, J. and Leadlay, P.F. (1995) Repositioning of a domain in a modular polyketide synthase to promote specific chain cleavage. Science, 268, 1487-1489.<br>
Donadio, S., McAlpine, J.B., Sheldon, P.J., Jackson, M. and Katz, L. (1993) Proc. A/a//. Acad. Sci. USA, 90, 7119-7123.<br>
Donadio, S., Staver, M.J., Mcalpine, J.B., Swanson, S.J. and Katz, L (1991) Modular organization of genes required for complex polyketide biosynthesis. Science, 252, 675-679<br>
Eisner, A., Engert, H., Saenger, W., Hamoen, L., Venema, G. and Bernhard, F. (1997) Substrate specificity of hybrid molecules from peptide synthetases. J. Biol. Chem. 272, 4814-4819.<br>
Fujimoto, D., Srinivasan, P.R. and Borek, E. (1965) On the nature of the deoxyribonucleic acid methylases. Biological evidence for the multiple nature of the enzymes. Biochemistry 4, 2849-2855.<br>
Geier, G. E. and Modrich, P. (1979) Recognition sequence of the dam methylase of Escherichia coll K12 and mode of cleavage of Dpn I endonuciease. J. Biol. Chem, 254, 1408-1413.<br>
Grabley, S., Granzer, E., Mutter, K.t Ludwig, D., Mayer, M., Thiericke, R., Till, G., Wink, J., Phillips, S. and Zeeck, A. (1992) J. Antibiot 45, 56-65.<br>
Jacobsen, J.R., Hutchinson, C.R., Cane, D.E. and Khosla, C. Precursor-directed biosynthesis of erythromycin analogs by an engineered polyketide synthase. Science 277, 367-369 (1997)<br>
Joshi, A.K. and Smith S. (1993) Construction of a cDNA encoding the . multifunctional animal fatty acid synthase and expression in Spodoptera frugiperda cells using baculoviral vectors. Biochem J.,296, 143-149.<br>
Kao, C.M., Luo, G.L, Katz, L, Cane, D.E. and Khosla, C. (1995) J. Am. Chem. Soc., 117, 9105-9106.<br>
Kao, C.M., Luo, G.L, Katz, L, Cane, D.E. and Khosla, C. (1996) J. Am. Chem. Soc., 118, 9184-9185.<br>
Kao, C.M., Luo, G.L, Katz, L., Cane, D.E. and Khosla, C., (1994) J.Am. Chem. Soc., 116, 11612-11613.<br>
Kuhstoss, S., Huber, M., Turner, J.R., Paschal, J.W. and Rao, R.N. (1996) Gene, 183,231-236.<br>
Lau, J., Fu, H., Cane, D. E. and Khosla, C. (1999) Dissecting the role of Acyltransferase domains of modular polyketide synthases in the choice and stereochemical fate of extender units. Biochemistry,.38, 1643-1651.<br>
MacNeii, D J., Gewain, K.M., Ruby, C.L., Dezeny, G., Gibbons, P.H. and MacNeil, T. (1992) Analysis of Streptomyces avermitflis genes required for avermectin biosynthesisutilizing a novel integration vector. Gene 111, 61-68.<br>
Marsden, A.F.A., Wilkinson, B.( Cortes, J., Dunster, N.J., Staunton, J. and Leadlay, P.P. (1998) Science, 279, 199-202.<br>
Marshall, J.A. and Johns, B.A. (1998) Total synthesis of (-*-)-discodermolide. J. Org. Chem. 63, 7885-7892.<br>
McDaniel, R. et al. (1999) and references therein. Multiple genetic modifications of the erythromycin polyketide synthase to produce a library of novel "unnatural" natural products. Proc, Natl. Acad. Sci. USA, 96, 1846-1851.<br>
Miller, R.V. and Kokjohn, T.A. (1990) General microbiology of recA: Environmental and evolutionary significance. Annu. Rev. Microbio!., 44, 365-394.<br>
Nufibaumer, B. and Wohlleben, W. (1994) Identification, isolation and sequencing of the recA gene of Streptomyces lividans TK24. FEMS Microbioi. lett, 118, 51-56.<br>
Oliynyk, M., Brown, M.J.B., Cortes, J., Staunton, J. and Leadlay, P.P. (1996) Chem. BioL, 3, 833-839.<br>
Paitan, Y., Alon, G., Orr, E., Ron, E.2., and Rosenberg, E. (1999) The first gene in the biosynthesis of the polyketide antibiotic TA of Myxococcus xanthus codes for a unique PKS module coupled to a peptide synthetase. J. Mol. Bioi. 286,465-474.<br>
Rowe, C.J., Cortes, J., Gaisser, S., Staunton, J., Leadlay, P.P. (1998) Construction of new vectors for regulated high-level expression in actinomycetes. Gene, 216, 215-223.<br>
Ruan, X., Pereda, A,, Stassi, D.L, Zeidner, D., Summers, R.G., Jackson, M., Shivakumar, A,, Kakavas, S., Staver, M.J., Donadio, S. and Katz, L (1997) Acyltransferase domain substitutions in erythromycin polyketide synthase yield novel erythromycin derivatives. J. Bacteriol. 179, 6416-6425.<br>
Shen, B., Du, L., Sanchez, C., Chen, M. and Edwards, D.J. (1999)<br>
Bleomycin biosynthesis in Streptomyces verticillus ATCC15003: A model of<br>
hybrid peptide and polyketide biosynthesis. Bioorganic Chemistry 27, 123-<br>
129.<br>
Tapiolas, D.M., Roman, M., Fenical, W., Stout, T.J. and Clardy, J. (1991)<br>
Octalactin-A and Octalactin-B - cytotoxic 8-membered-ring lactones from a marine bacterium, Streptomyces sp. J. Amer. Chem. Soc. 113, 4682-4683.<br>
TerHaar, E., Kowalski, R.J., Hamel, E., Lin, C.M., Longley, R.E., Gunasekera, S.P., Rosenkranz, H.S. and Day, B.W. (1996) Discodermolide, a cytotoxic marine agent that stabilizes microtubules more potently than taxol. Biochemistry 35, 243-250.<br>
Yamada, S., Tanaka, A. and Oritani, T. (1995) Total synthesis of Decarestrictine-J. Biosci. Biotech. &amp; Biochem. 59, 1657-1660<br>
Ziermann, R. and Betlach, M.C. (1999) Recombinant Polyketide Synthesis in Streptomyces: Engineering of improved host strains. BioTechniques 26, 106-110.<br><br><br><br><br><br><br><br><br>
WE CLAIM:<br>
1.	A method of sequentially assembling several DNA units to form<br>
a DNA construct, such that the junctions so formed are protected<br>
from cleavage by the subsequent application of the chosen restriction<br>
enzyme, permitting the repeated use of the chosen restriction enzyme<br>
in assembling the DNA construct.<br>
2.	The method as claimed in  Claim   1, wherein the junctions<br>
formed   are   protected   from   subsequent   cleavage   by   the   chosen<br>
restriction enzyme by modifying the restriction site at each junction<br>
formed  in  the  sequential assembly process using a modification<br>
enzyme, and comprises the steps of<br><br>
a)	providing each DNA unit with a restriction enzyme-recognition<br>
sequence at it's 5'end and with a recognition sequence for the same<br>
restriction enzyme at its 3'end that is combined with a recognition<br>
site for a DNA modification enzyme.<br>
b)	providing   a   starting   DNA   construct   having   an   accessible<br>
restriction site for the same or a compatible restriction enzyme and<br>
cleaving the starting DNA construct with such a restriction enzyme,<br>
c)	inserting the desired DNA unit and bringing the ligated product<br>
into contact with a DNA modification enzyme such that the restriction<br>
site at the 3'end of the inserted DNA unit is abolished d) cleaving the<br>
ligated product at an accessible unmodified recognition site for the<br>
same or a compatible restriction enzyme,<br>
e)       repeating steps c) and d) to introduce each desired DNA unit to give a DNA construct containing all the desired units in sequence.<br>
3.	The   method   as   claimed   in   claim   2   wherein   the   DNA<br>
modification enzyme is a methylase.<br>
4.	The method as claimed in claim 3 wherein the METHYLASE is<br>
the dam METHYLASE of Escherichia coli as herein described.<br>
5.	The method as claimed in claims 2 to 4 wherein the junctions<br>
formed   are   protected   from   subsequent   cleavage   by  the   chosen<br>
restriction enzyme by modifying the restriction site at each junction<br>
formed in the sequential assembly process using the modification<br>
enzyme and comprises the steps of:<br>
a)	providing  each  DNA unit with  an  Xbal recognition   sequence<br>
5'XXTCTAGA3' (where XX is not GA) at it's 5'end and with an Xbal<br>
recognition sequence 5'GATCTAGA3'at its 3'end.<br>
b)	providing a starting DNA construct having an accessible Xbal site<br>
and cleaving the starting DNA construct with Xbal,<br>
c)	inserting the  desired  DNA unit  and using a resulting ligated<br>
product to transform a dam+strain of E. coli,<br>
d)	recovering a resulting plasmid and cleaving the plasmid at an<br>
accessible Xbal site with Xbal,<br>
e)	repeating steps c) and d) to introduce each desired DNA unit to give<br>
a DNA construct containing all the desired units in sequence.<br>
6.	The method as claimed in any one of claims 2 to 5, wherein the<br>
recognition   sequences   for   the   restriction   enzyme   and   the   DNA<br>
modification enzyme are created in the DNA units prior to cutting<br>
with the restriction enzyme.<br>
7.	The method as claimed in claim 6 wherein the restriction sites<br>
are created in the fragment by means of a primer extension reaction.<br>
8.	The method as claimed in any one of claims 2 to 7, wherein the<br>
DNA   construct   is   an   expression   vector   capable   of   facilitating<br>
expression of the protein encoded by the desired DNA units.<br>
9.	The method as claimed in claim 4 or claim 5, wherein the DNA<br>
modification is removed and the restriction site re-established by<br>
replicating the ligated product in a dam-strain of E. coli by means of a<br>
suitable vector.<br>
10.	The method as claimed in Claim   1  in which the junctions<br>
formed   are   protected   from   subsequent   cleavage   by   the   chosen<br>
restriction enzyme, by the use of a complementary restriction enzyme<br>
to prepare the end of each DNA unit to be ligated to the DNA<br>
assembly, such that the ends so created are compatible and may be<br>
ligated, but the junction so formed is not recognized by the chosen<br>
restriction enzyme used.<br>
the method comprises the steps of:<br>
a)	providing a first DNA unit with a recognition sequence for a first<br>
restriction enzyme at its 3'end, and cleaving the said first DNA unit<br>
with said first restriction enzyme,<br>
b)	providing each other DNA unit with a recognition sequence at its<br>
5'end for a second restriction enzyme which has a compatible ligation<br>
sequence with that of the first restriction enzyme, and a downstream<br>
recognition sequence for said first restriction enzyme followed by a<br>
downstream recognition sequence for a third restriction enzyme at its<br>
3' end, and cleaving each said other DNA unit with the second and<br>
third restriction enzymes,<br>
c)	ligating the said first DNA unit with a desired other DNA unit to<br>
form  a ligated  product  such  that the  ligation  of the  two  units<br>
abolishes the recognition site for the first restriction enzyme at the<br>
ligation junction,  and cleaving the ligated product with said first<br>
restriction enzyme,<br>
d)	ligating the product from c) with a desired DNA unit from b) to form<br>
a ligated product and cleaving the ligated product with said first<br>
restriction enzyme<br>
e)	repeating step d) with each other DNA unit in turn so as to<br>
assemble the DNA units in sequence.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDQtZGVsLWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-01144-del-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDQtZGVsLWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-01144-del-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDQtZGVsLWNvbXBsZXRlIHNwZWNpZmljYXRpb24gKGdyYW50ZWQpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-01144-del-complete specification (granted).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDQtZGVsLWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-01144-del-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDQtZGVsLWNvcnJlc3BvbmRlbmNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-01144-del-correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDQtZGVsLWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-01144-del-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDQtZGVsLWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-01144-del-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDQtZGVsLWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-01144-del-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDQtZGVsLWZvcm0tMTkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-01144-del-form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDQtZGVsLWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-01144-del-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDQtZGVsLWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-01144-del-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDQtZGVsLWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-01144-del-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDQtZGVsLWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-01144-del-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDQtZGVsLXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-01144-del-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDQtZGVsLXBjdC00MDkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-01144-del-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDQtZGVsLXBjdC00MTYucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-01144-del-pct-416.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="231902-a-method-for-multi-dimentional-orthogonal-resource-hopping-multiplexing-communication-in-a-digital-communiacation-system-and-apparatus-thereof.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="231904-housing-box-supporting-structure-for-a-motorcycle.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>231903</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/01144/DEL</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>13-Mar-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>10-Dec-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>QXYZ LIMITED</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>MITCHELLS ROBERTON SOLICITORS, GEORGE HOUSE, 36 NORTH HANOVER STREET, GLASGOW G1 2AD, ENGLAND.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ANAND RANGANATHAN</td>
											<td>INTERNATIONAL CENTRE FOR GENETIC ENFINEERING AND BIOTECHNOLOGY,(ICGEB), ARUNA ASAF ALI MARG, P.O. BOX 10504, NEW DELHI-110067, INDIA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 9/10</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/GB00/02286</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-06-12</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>9913694.7</td>
									<td>1999-06-11</td>
								    <td>U.K.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/231903-a-method-of-sequentially-assembling-several-dna-units by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:49:52 GMT -->
</html>
