<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/231926-a-method-for-assaying-the-activity-of-transcriptional-control-elements by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:51:21 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 231926:&quot;A METHOD FOR ASSAYING THE ACTIVITY OF TRANSCRIPTIONAL CONTROL ELEMENTS&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A METHOD FOR ASSAYING THE ACTIVITY OF TRANSCRIPTIONAL CONTROL ELEMENTS&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method for assaying the activity of a first transcriptional control element in a cell, the method comprising expressing a first polynucleotide in the cell from a first construct comprising, in operable linkage; the first transcriptional control element, the coding sequence of a first polypeptide and a nucleic acid sequence encoding a RNA destabilizing element that reduces the stability of a transcript corresponding to the first polynucleotide in the cell; measuring the level and/or functional activity of the first polypeptide produced from the first construct; and determining the activity of the first transcriptional control element based on the measured level and/or functional activity of the first polypeptide.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>NOVEL EXPRESSION VECTORS FIELD OF THE INVENTION<br>
The present invention relates generally to constructs and their use in gene expression or gene regulation assays. More particularly, the present invention provides expression vectors and/or reporter vectors providing kinetics of protein expression with improved temporal correlation to promoter activity. The present invention provides, inter alia, novel vectors and cell lines useful for modulating gene expression, identifying and analysing regulatory sequences, new targets and reagents for therapeutic intervention in human diseases and for drug-screening.<br>
BACKGROUND OF THE INVENTION<br>
Bibliographic details of the publications referred to by author in this specification are collected at the end of the description.<br>
Reference herein to prior art, including any one or more prior art documents, is not to be taken as an acknowledgment, or suggestion, that said prior art is part of the state of the art.<br>
The rapidly increasing sophistication of recombinant DNA technology is greatly facilitating research and development in the medical and allied health fields. A particularly important area of research is the use of expression vectors to study gene expression. However, until now, a real-time analysis of gene expression has been limited by the lack of suitably designed vectors.<br>
Reporter assays permit an understanding of what controls the expression of a gene of interest e.g., DNA sequences, transcription factors, RNA sequences, RNA-binding proteins, signal transduction pathways and specific stimuli.<br>
Furthermore, reporter assays can be used to identify aspects of gene regulation that serve as new targets for therapeutic intervention in human disease. Reporter assays can potentially be used to screen drugs for their ability to modify gene expression. However, the cost and time<br><br>
required for current reporter assay systems, together with the lengthy response times, has limited this application.<br>
Genomic sequences have promoter sequences, generally upstream of the coding region, which dictate the cell specificity and inducibility of transcription and thereby affect the level of expression of protein products.<br>
Specific sequence elements, typically rich in the nucleotide buses A and U and often located in the 3'-UTR of a gene, affect the stability of the mRNA and thereby affect the level of expression of the protein product. RNA-binding proteins bind cerium mRNA sequences and thereby regulate mRNA stability and protein expression. Other sequences modulate translational efficiency.<br>
A common application of gene reporter assays is the study of DNA sequences that regulate transcription. Typically, these sequences are located in the promoter region, 5' of the transcription start site. Such DNA elements are tested by cloning them into a similar site within a reporter plasmid, such that they drive and/or regulate transcription and therefore, expression of reporter protein. The reporter protein should be distinguishable from endogenous proteins and easily quantified. Various reporter proteins are used, the most common being luciferase. chloramphenicol transferase (CAT) and β galactosidase (β-gal).<br>
The reporter protein is quantified in an appropriate assay and often expressed relative to the level of a control reporter driven by a ubiquitous promoter such as for example the promoter SV40. The control reporter must be distinguishable from the test reporter and is contained on a separate vector that is co-transfected with the test vector and used to control for transaction efficiency. Such assays are based on the premise that cells take up proportionally equal amounts of both vectors. Transient transfections of plasmid vectors are most commonly used.<br>
The assays described above are used to identify a promoter region or the specific elements within a promoter. Alternatively, they are used to study the response to various stimuli of a<br>
promoter or regulatory element. In some applications, the reporter constructs, or the transfected cells, are placed into an organism to study promoter function in vivo.<br>
Another application of these reporter assays is the study or measurement of signal transduction pathways upstream of a specific promoter. For example, a promoter dependent on mitogen activated protein kinase (MAPK) for transcription can be linked to a reporter construct and used to measure the level of MAPK activation (or MAPK-dependent transcription) in cells. This technique can be utilized with a variety of informative promoters or enhancers and can he applied to cells or living organisms such as transgenic mice. For example, a photon camera can be used to measure luciferase reporter activity in whole mice containing a luciferase reporter linked to a promoter of interest (Contag, et al, 1997).<br>
Luciferase is by far the most commonly used reporter assay for in vitro systems. The Dual Luciferase assay (DLA; Promega, Madison, WI, USA), is an improvement over other luciferase based systems in that both test and control reporter can essentially be measured in the same assay. As an example of current use, a typical DLA protocol is provided as follows:<br>
The putative promoter element is cloned upstream of a firefly luciferase reporter gene such that it drives its expression. This plasmid is transiently transfected into a cell line, along with a control plasmid containing the Renilla luciferase gene driven by the SV40 promoter. -2-50% of cells take up plasmid and express the reporters for ~3 days. The kinetics of expression involve an increase during the first -24 h as luciferase protein accumulates, followed by a decrease from -48 h as the number of plasmids maintained within the cells declines. 24-48 h after transfection, cells are harvested and lysed. Cell lysates are incubated with substrates specific to firefly luciferase and activity (light emission) is measured using a luminometer (96 well plate or individual samples). Additional substrates are then added, which inactivate firefly luciferase but allow Renilla luciferase to generate light. Renilla luciferase activity can then be measured.<br>
The level of firefly luciferase activity is dependent, not only on promoter activity, but also on transfection efficiency. This varies greatly, depending on the amount of DNA, the quality of<br>
the DNA preparation and the condition of the cells. The co-transfected control plasmid (Renilla luciferase driven by the SV40 promoter) is used to correct for these vanables, based on the premise that Renilla luciferase activity is proportional to the amount of firefly luciferase plasmid taken up by the cells. Data are expressed as firefly luciferase activity / Renilla luciferase activity.<br>
The disadvantages of the Dual Luciferase assay are as follows:<br>
(i)        Reagents are expensive and perishable and must be freshly prepared.<br>
(ii) Generally this assay involves the preparation of cell lysates, which is time consuming and adds inaccuracy, e.g., loss of cells during lysis, pipetting errors, residual buffer/medium altering volumes.<br>
(iii) Each sample yields only one datum point being the total activity of the cell population. No information is gained concerning the percentage of cells that express the reporter, nor the amount of expression per cell.<br>
(iv) The transfection control (Renilla) does not always correct for huge variation in transfection efficiencies because:<br>
a.	Certain DNA preparations trans feet/express poorly (perhaps due to reduced<br>
proportion of supercoiled DNA), but do not cause a corresponding decrease in<br>
the amount of co-transfected control plasmid.<br>
b.	There is evidence of cross-talk between the promoters of the two plasrnids, such<br>
that control reporter activity is dependent on the construct with which it is co-<br>
transfected, e.g., expression of Renilla luciferase seems highest when co-<br>
transfected with a plasmid containing a strong promoter. Interference between<br>
promoters has also limited, if not prevented, the use of single plasmids<br>
expressing both test and control reporters.<br>
c. A common application of both transcriptional and post-transcriptional studies is to measure activation/suppression by various stimuli (e.g., PMA, EGF, hormones). Unfortunately, SV40, RSV, TK and probably many other ubiquitously expressed promoters are activated by a variety of stimuli. Since these promoters are used to drive expression of the transfection control reporter (Renilla), these reporters do not give a true reflection of transfection efficiency following such treatments. (Ibrahim el al, 2000).<br>
d.	Differences in the half-lives of firefly vs Renilla luciferase proteins and perhaps<br>
mRNAs make the whole system very time-sensitive.<br>
e.	Rapidly diminishing light emission, particularly for Reni.Ua luciferase, require<br>
absolute precision in the timing of measurement.<br>
f.	The relatively long half-lives of luciferase proteins and mRNAs, effectively<br>
mask temporal changes in transcription (e.g., following various stimuli or<br>
treatments).<br>
In existing post-transcriptional/mRNA stability reporter assays, candidate elements, thought to affect mRNA stability are cloned into the corresponding region of a reporter vector (e.g., firefly luciferase) driven by a constitutive promoter such as SV40 or RSV. Changes in expression relative to the empty vector (same vector without element of interest) are assumed to be the result of altered mRNA stability. As with the preliminary described transfection assays, a transfection control plasmid (e.g., Renilla luciferase driven by a constitutive promoter such as SV40 or RSV) is co-transfected to allow correction for transfection efficiency. These assays suffer from the following additional disadvantages:<br>
(i).       Existing vectors were not designed for post-transcriptional studies and have no means for switching off transcription.<br>
(ii).     The purpose of these protocols is to study the post-transcriptional effects of candidate mRNA elements. However, these elements can also affect transcription of the reporter at the level of DNA. Furthermore, since the endogenous promoter of the gene of interest is not used, any transcriptional effects seen, may have little physiological relevance.<br>
Systems for studying mRNA stability exist but involve direct measurement of the mRNA rather than a protein reporter. Due to the labor-intensive nature of protocols for quantifying mRNA, such systems are far more time consuming.<br>
One system, for example, utilizes the  c-fos promoter, which responds to serum induction with a brief burst of transcription. Putative instability elements are cloned into the 3-UTR of a Beta Globin (BBB) construct, which expresses a very stable mRNA under the control of a serum-inducible (c-fos) promoter. Transfected cells (generally NTH 3T3 cells) are first serum starved and then exposed to medium containing serum. The brief nature of the transcriptional response allows the kinetics of reporter mRNA degradation to be followed in a time course. These assays suffer from the following disadvantages:<br>
(i).       This assay is very time consuming and is therefore not applicable to rapid screening.<br>
(ii). Can only be used in cells that support serum inducibility of the c-fos promoter. For example, many tumor cell lines maintain c-fos promoter activity in the absence of serum.<br>
(iii). In cells such as NTH 3T3 cells, which do have the desired serum response, serum deprivation causes a cell cycle block and subsequent addition of serum, releases the cells from this block in a synchronous manner. Therefore, mRNA stability can only he measured in specific stages of the cell cycle.<br>
 (iv). In addition to activating the c-fos promoter, serum activates a multitude of other pathways, which introduce unwanted variables and prevent the study of more specific stimuli.<br>
in another assay, cells are treated with drugs, such as Actinomycin D that inhibit transcription from all genes. The mRNA levels are measured in a time course to determine mRNA degradation rates. This system is used to study endogenous genes and suffer from the following disadvantages:<br>
(i). Transcriptional inhibitors are extremely toxic at doses required such thai mRNA stability is often being measured in stressed or dying cells.<br>
(ii). Transcription inhibitors possess numerous unwanted activities including stabilization of certain mRNAs.<br>
(iii). The process blocks transcription from all genes such that many signal transduction cascades are blocked, whereas others are activated. Therefore, results may not be physiologically relevant.<br>
(iv).     The technique is extremely labor intensive.<br>
(v).      The technique is highly variable within and between assays.<br>
(vi). The technique is often not sensitive enough for transient transfection reporter assays, particularly in cells with a low transfection efficiency.<br>
There is a need therefore to develop improved vectors and systems for conducting gene expression assays and in particular post-translational and post-transcriptional assays as well as assays that permit a more real-time determination of changes in gene expression.<br>
SUMMARY OF THE INVENTION<br>
Throughout this specification, unless the context requires otherwise the word "comprise", and variations such as "comprises" and "comprising", will be understood to imply the inclusion of a stated integer or step or group of integers or steps but not the exclusion of any other integer or step or group of integers or steps. The present invention is not to be limited in scope by the specific embodiments described herein, which are intended for the purposes of exemplification only. Functionally-equivalent products, compositions and methods are clearly within the scope of the invention, as described herein.<br>
In accordance with the present invention, the inventor has developed a series of vectors and methods which permit inter alia modulation and determination of transcript stability and/or improved real-time determination of gene expression.<br>
Nucleotide sequences are referred to by sequence identifier numbers (SEQ ID NO:). The SEQ ID NOs: correspond numerically to the sequence identifiers 1, 2, etc. A sequence listing is provided after the claims.<br>
One aspect of the present invention is directed to a construct comprising in operable linkage: a polynucleotide which encodes a polypeptide whose intracellular half-life is less than 3 hours and a nucleic acid sequence which encodes an RNA element that modulates the stability of a transcript encoded by the polynucleotide. As used herein the stability of a transcript may correspond to the half-life of the transcript.<br>
Another aspect of the present invention contemplates a construct comprising in operable linkage: a polynucleotide which encodes a polypeptide whose intracellular half-life is less than 3 hours and a nucleic acid sequence which encodes an RNA element that modulates the stability of a transcript encoded by the polynucleotide; wherein the RNA clement is a destabilising element that reduces the stability of the transcript.<br>
In a related embodiment the present invention contemplates a construct comprising in operable<br>
linkage: a polynucleotide which encodes a polypepticle whose intracellular half-life is less than 3 hours and a nucleic acid sequence which encodes an RNA element that modulates the stability of a transcript encoded by the polynucleotide; wherein the RNA element is a stabilising element that increases the stability of the transcript.<br>
Yet another aspect of the present invention contemplates a construct comprising in operable linkage: a polynucleotide which encodes a reporter polypeplide whose intracellular half-life is less than 3 hours and a nucleic acid sequence which encodes an RNA destabilising element that reduces the stability of a transcript encoded by the polynucleotide.<br>
Still another aspect of the present invention is directed to a construct comprising in operable linkage a polynucleotide which encodes a polypeptide whose intracellular half-life is less than 3 hours and a nucleic acid sequence which encodes an RNA destabilising element that modulates the stability of a transcript encoded by the polynucleotide, and wherein the construct comprises one or more members selected from the group consisting of:<br>
(i)	a multiple cloning site for introducing a sequence of nucleotides;<br>
(ii)	a reporter gene;<br>
(iii)	a promoter for regulating expression of said transcribable polynucleotide;<br>
(iv)	a transcriptional enhancer for enhancing transcription of the polynucleotide;<br>
(v)       a translalional enhancer for enhancing translation of the transcript encoded by the polynucleotide;<br>
(vi)       a polyadenylation sequence; (vii)     a selectable marker gene; and<br>
 (viii)    an origin of replication.<br>
Even still another aspect of the present invention contemplates a cell containing a construct according to the present invention comprising in operable linkagef a polynucleotide which encodes a polypeptide whose intracellular half-life is less than 3 hours and a nucleic acid sequence which-encodes an RNA element that modulates the stability of a transcript encoded by the polynucleotide.<br>
Still another aspect of the present invention contemplates a cell containing a construct according to the present invention comprising in operable linkage: a polynucleotide which encodes a polypeptide whose intracellular half-life is less than 3 hours and a a nucleic acid sequence which encodes an RNA destabilising element that reduces the stability of a transcript encoded by the polynucleotide.<br>
A related aspect of the instant invention considers a genetically modified non-human organism comprising a polynucleotide which encodes a polypeptide whose intracellular half-life is less than 3 hours and a nucleic acid sequence which encodes an RNA element that modulates the stability of a transcript encoded by the polynucleotide.<br>
Yet a further embodiment of the present invention contemplates a method for assaying the activity of a transcriptional control element, the method comprising:<br>
-	expressing from the transcriptional control element a polynucleotide which encodes<br>
a polypeptide whose intracellular half-life is less than 3 hours and which is operably connected<br>
to a nucleic acid sequence which encodes an RNA element that modulates the stability of a<br>
transcript encoded by the polynucleotide; and<br>
-	measuring the polypeptide produced from the polynucleotide.<br>
In still yet a further embodiment, the present invention contemplates a method for assaying the activity of a post-transcriptional control element, the method comprising:<br>
- expressing from a transcriptional control element a polynucleoticle which encodes a polypeptide whose intracellular  half-life is less than 3 hours and which is operahly linked to a nucleic acid sequence which encodes the post transcriptional control element; and<br>
- measuring the level and/or functional activity of the polypeptide produced from ihc polynucleotide.<br>
In another aspect, the invention provides a method for identiiying a nucleotide sequence which encodes a post-transcriptional control element thai modulates the expression of an RNA transcript from a first polynucleotide that encodes a polypeptide whose intraccliular half-life is less than 3 hours, the method comprising:<br>
-	expressing from a first transcriptional control element in a first construct the first<br>
polynucleotide which is operably connected to a test nucleotide sequence that is suspected of<br>
encoding the post-transeriptional control element;<br>
-	expressing from a second transcriptional control element in a second construct a<br>
second polynucleotide, which encodes a second polypeptide whose intracellular half-life is less<br>
than 3 hours, but which is not operably connected to the test nucleotide sequence, wherein the<br>
second polypeptide is the same as, or different to, the first polypeptide and wherein the second<br>
transcriptional control element is the same as, or different to, the first transcriptional control<br>
element;<br>
-	comparing the level and/or functional activity of the polypeptides from the first and<br>
second constructs, wherein a difference between the level and/or functional activity of the first<br>
poiypcptide and the level and/or functional activity of the second polypeptide indicates that the<br>
test nucleotide sequence encodes a said post-transcriptional control element.<br>
In yet another aspect, the invention embraces a method for identifying an agent which modulates the activity of an post-transcriptional control element that modulates the expression of an RNA transcript from a polynucleoticle which encodes a polypeptide whose intracellular half-life is less than 3 hours, the method comprising:<br>
-	expressing from a transcriptional control element the polynucleotide which is operably<br>
connected to a nucleic acid sequence which encodes the post-transcriptional control element,<br>
wherein the expression of the polynucleotide is carried out in the presence and absence of a test<br>
agent:<br>
measuring  the  level   and  or  functional  activity  of the  polypeptide   from   the polynucleotide in the presence and absence of the test agent; and<br>
- comparing those levels and/or functional activities, wherein a difference between the level and/or functional activity of the polypeptide in the presence and absence of the lest agent indicates that the test agent modulates the activity of the post-transcriptional control element.<br>
In still another aspect,  the invention contemplates the use of a construct in the preparation of a kit for identifying or assaying the activity of a post-transcriptional control element that modulates the expression of a transcript encoded by a polynucleotide which encodes a polypeptide whose intracellular half-life is less than 3 hours, the construct comprising a transcriptional control element that is operably connected to the polynucleotide and a cloning site for introducing a nucleotide sequence which encodes the post-transcriptional control element.<br>
In a further aspect, the invention encompasses a method for identifying a transcriptional control element, the method comprising:<br>
-	subjecting a construct to conditions sufficient for RNA and protein synthesis to occur,<br>
wherein the construct comprises in operable linkage: a nucleotide sequence suspected of<br>
having transcriptional control activity; a polynucleotide which encodes a polypeptide whose<br>
intracellular half-life is less than 3 hours and a nucleic acid sequence that encodes an RNA<br>
element that modulates the stability of a transcript encoded by the polynucleotide; and<br>
-	detecting production of the polypeptide from the construct.<br>
In still a further aspect, the invention provides a method for identifying a post-transcriptional control element, the method comprising:<br>
-	expressing from a first transcriptional control element in a first construct a first<br>
polynucleotide, which encodes a first polypeptide whose intracellular half-life is less than 3<br>
hours	and which is operably linked to a test nucleic acid sequence which is suspected to encode a post-<br>
transcriptional control element; and<br>
-   expressing from a second transcriptional control element in a second construct a second polynucleotide, which encodes a second polypeptide whose intracellular half-life is less than 3 hours, but which is not operably connected to the test nucleic acid sequence, wherein the second polypeptide is the same as, or different to, the first polypeptide, and wherein the second transcriptional control element is the same as, or different to, the first transcriptional control element; and<br>
-	detecting a difference between the level and/or functional activity of the first<br>
polypeptide produced from the first construct and the level and/or functional activity of the<br>
second polypeptide produced from the second construct, wherein the difference indicates that<br>
the test nucleic acid sequence is a post-transcriptional control element.<br>
Still a further aspect of the invention contemplates a method for assaying the activity of a transcriptional control element, the method comprising:<br>
-	expressing from the transcriptional control element a polynucleotide which encodes<br>
a polypeptide comprising a protein-destabilising element and which is operably connected to<br>
a nucleic acid sequence which encodes an RNA element that modulates the stability of a<br>
transcript encoded by the polynucleotide; and<br>
-	measuring the level and/or functional activity of the polypeptide produced from the<br>
construct. Yet another aspect of the invention embraces a construct comprising in operable linkage: a polynucleotide which encodes a polypeptide comprising a protein-destabilising element, and a nucleic acid sequence which encodes an RNA element that modulates the stability of a transcript encoded by the polynucleotide.<br>
Still another aspect of the invention features a method for assaying the activity of a transcriptional control element, the method comprising:<br>
-	expressing from the transcriptional control element a polynucleotide which encodes<br>
a polypeptide and which is operably connected to a nucleic acid sequence which encodes an<br>
RNA element that modulates the stability of a transcript encoded by the polynucleotide; and - measuring the level and/or functional activity of the polypeptidc produced from the construct.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figure 1 is a schematic representation of expression vectors encoding a destabilising mJRNA.<br>
Figure 2 is a schematic representation of transcription reporter vectors; Figure 2a shows vector series 2; Figure 2b shows vector series 3 and Figure 2c shows vector series 4,<br>
Figure 3 is a schematic representation of Bi-directional transcription reporter vectors; Figure 3a shows vector series 5 and Figure 3b shows vector series 6.<br>
Figure 4 is a schematic representation of reporter vectors for studying post-transcriptional regulation; Figure 4a shows vector series 7 and Figure 4b shows vector series 8.<br>
Figure 5 is a graphical representation showing reporter activity as a measure against the amount of DNA transfected. A single DNA preparation of a plasmid encoding firefly luciferase was mixed at a 30:1 ratio with a separate plasmid encoding Renilla luciferase. Both DNA preparations appeared normal in spectrophotometry (OD260/280) and on ethidium bromide stained agarose gels (data not shown). Different volumes of this mixture were transfected into cells such that the total quantity of DNA was 1, 2 or 3 micrograms but the ratio of firefly to Renilla plasmids remained the same.<br>
Fig. 5A is a graphical representation showing that Renilla luciferase activity was dependent on the amount of DNA transfected. However, firefly luciferase activity (Fig. 5B) did not increase with increasing amounts of DNA, perhaps because the firefly DNA preparation was of poor quality. Consequently, the firefly/ Renilla ratio (Fig. 5C), which would typically be used as a measure of the firefly promoter activity, varied considerably depending on the amount of DNA used. These data demonstrate that co-transfections with Renilla plasmids do not adequately control for the transfection efficiency of the firefly plasmid<br>
Figure 6 is a graphical representation showing reporter activity for various promoter systems using the Dual Luciferase Assay. Six different promoter fragments (numbered 1 -6) were cloned<br>
into pGL3 firefly luciferase plasmids. One microgram of each clone was co-transfected with 30ng of Renilla (transfection control) plasmid, driven by an SV40 promoter. Firefly and Renilla luciferase activities were measured using the Dual Luciferase Assay (Promega, Madison, WI, USA). Results are expressed as Renilla luciferase activity (A), Firefly luciferase activity (B) and firefly divided by Renilla activity (C). Similar results were seen in multiple experiments using at least 2 different preparations of each construct.<br>
Renilla luciferase activity (Fig. 6A) is intended as a transfection control and analysis of this result alone would suggest an unusually high variation in transfection efficiency. For example, Renilla luciferase activity is 3.5 fold higher when co-transfected with construct 4 compared to co-transfection with construct 3. Variations in DNA quality or errors in the quantification of DNA seem unlikely as sources of error since the same pattern was seen with a separate set of DNA preparations (data not shown).<br>
Firefly luciferase activity (Fig. 6B) is influenced by both transfection efficiency and differences between promoters 1-6. The pattern of differences is similar to that seen with Renilla (Fig. 6A). For example, 3 and 6 are low whilst 4 and 5 are high. However these differences between constructs are more marked with firefly (e.g., construct 4 is 12 fold higher than construct 3), suggesting that the activity of promoters 1-6 is somehow affecting expression of Renilla (or vice versa).<br>
Firefly/Renilla (Fig. 6C) is considered to be a measure of true firefly promoter activity (1-6) after correction for transfection efficiency (Renilla). Again a similar pattern is seen, suggesting that indeed 3 and 6 are the weakest promoters whilst 4 and 5 are the strongest. Whilst it is possible that promoter activity (Fig. 6C) coincidentally correlated with transfection efficiency (Fig. 6A), this possibility seems extremely unlikely given that similar results were obtained with numerous different constructs and multiple different preparations of the same construct. It seems more likely that the level of expression of Renilla luciferase is affected by the strength of the promoter construct with which it is co-transfectcd. Consequently, apparent differences between promoters 1-6 are likely to be an underestimation of the true differences.<br><br>
Figure 7 is a graphical representation showing different reporter levels for BTL, BTG2, BTG1 and BTG1N4 expression vectors on a time course after blocking transcription. let-Off HeLa cells were transfected with the following reporter plasmids, each containing a TRE promoter linked to a reporter gene;       BTL (luciferase), BTG2 (d2EGFP), BTG1  (dlEGFP) and BTG1N4 (same as BTG1 but with 4 copies of the nonamer UUAUUUAUU present in the 3'UTR-encoding region). Ten hrs after transfection, each flask of cells was split into multiple small plates. Doxycycline (10g/ml) was added at 24 hrs after transfection (time zero) to block transcription of the reporter genes. Reporter levels (fluorescence or luminescence) were measured at this and subsequent time points, as described in Exmaple 14, and presented as the percentage of time zero. No decrease in luciferase activity (BTL) was seen during the 10 hr time-course. The 2 hr half-life EGFP construct (BTG2) showed a moderate response to the doxycycline-induced block in transcription and a faster response was seen with the 1 hr half-life EGFP (BTG1). The construct containing the nonamers (BTG1N4), however, showed by far the fastest response to this block in transcription.<br>
Figure 8 is a graphical representation showing the data used for Fig. 7 displayed on a linear scale. The doxycycline-induced block in transcription is detectable as a 50% block in reoorter levels after approximately 6.5 hrs with BTG1. However, this is reduced to less than 3 hrs by inclusion of the nonamers (BTG1N4).<br>
Figure 9 is a graphical representation showing the effect of different numbers (1, 2 or 4) of nonamer RNA destabilising elements. A time-course was performed as described in Fig. 7, except with time zero defined as 4 hrs after addition of doxycycline to eliminate the effect of the delay in the action of this drug. The presence of a single nonamer (BTG1N1) was sufficient to increase the "effective rate of decay," whereas progressively stronger effects were seen with 2 nonamers (BTG1N2) and 4 nonamers (BTG1N4). The latter construct showed an "effective half-life" of ~1 hr 20 mins, which is little more than the 1 hr half-life of the protein alone.<br>
Figure 10 is a graphical representation showing changing reporter levels over time in the absence of a transcriptional block. A time-course was performed as described in Fig. 7. However, the data presented represent samples not treated with doxycycline and measured at<br><br>
24 hrs after transfection (start) or 34 hrs after transfection (finish). Consistent expression levels were seen only with BTG1N4.<br>
Figure 11 is a graphical representation showing changes in reporter levels over time in the absence of a transcriptional block. A time-course was performed as described in Fig. 7. BTGlfos contains the c-fos ARE, These data demonstrate that different types of mRNA destabilising elements can be used to achieve the same effect.<br>
Figure 12 is a graphical representation showing that RNA destabilising elements are useful in determining expression when a Luciferase reporter protein is used. A further enhancement would be expected using a luciferase reporter protein with protein destabilising elements. A time-course was performed as described in Fig. 7, using two luciferase-expressing constructs. BTL contains the standard Firefly luciferase-coding region and 3'UTR (derived from pGL3-Basic; Promega), whereas BTLN6 contains 6 copies of the nonamer UUAUUUAUU in the 3'UTR.<br>
Figure 13 is a graphical representation showing reporter levels over time using DsRed destabilised by RNA destabilising elements and protein destabilising elements. A time course was performed as described in Fig. 7 and Example 14. The constructs used were DsRed2 (BTR), DsRed-MODC (BTRl) and DsRed-MODC containing 4 UUAUUUAUU nonamers in the 3'UTR (BTR1N4). After blocking transcription with doxycycline, red fluorescence continues to increase with all constructs. This is substantially reduced by the protein destabilising element and further reduced by the rnRNA destabilising element.<br>
Figure 14 is a graphical representation showing a time-course was performed as described in Fig. 7. All of the mRNA destabilising elements tested were very effective at increasing the rate of decay compared to controls (BTG1). These data show that the c-myc ARE is an effective destabilising element (BTGlmyc) and that a modest increase in destabilising activity can be obtained by combining the myc ARE with 4 nonamers (BTGlN4myc). Six nonamers (BTG1N6) also appeared to destabilise somewhat more than 4 nonamers (BTG1N4).<br>
DETAILED DESCRIPTION OF THE PREFERRED EMBODIMENTS<br>
Definitions<br>
By "3 'UTR" is meant	the region of a polynucleotide downstream of the<br>
termination codon of a protein-encoding region of that polynucleotide, which is not translated to produce protein.<br>
By "5 'UTR" is meant	5' (upstream) untranslated region of an mRNA. Also<br>
used to refer to the DNA region encoding the 5'UTR of the mRNA.<br>
By "About" is meant	a quantity, level, value, dimension, size, or amount<br>
that varies by as much as 30%, preferably by as much as 20%, and more preferably by as much as 10% to a reference quantity, level, value, dimension, size, or amount.<br>
By "ARE" is meant	an AU-rich element in mRNA i.e., a sequence that<br>
contains a high proportion of adenine and uracil nucleotides. Also used to refer to the DNA region encoding such an mRNA element.<br>
By "Biologically active fragment" is<br>
meant	a fragment of a full-length reference polynucleotide or<br>
polypeptide which fragment retains the activity of the reference polynucleotide or polypeptide, respectively.<br>
By "c-fos" is meant	an immediate early gene, briefly induced by mitogenic<br>
signals<br>
By "CAT:" is meant	Chloramphenicol acetyltransferase. A bacterial enzyme<br>
often used as a reporter.<br>
By "dlEQFP" is meant	a variant of EGFP that is fused to a mutated PEST<br>
sequence and consequently has a half-life of only about 1 hour. Similarly, dlECFP and dlEYFP are also available. A destabilised variant of DsRed could be made in the same way. Henceforth referred to as dlDsRed.<br><br><br>
By "d2EGFP" is meant<br>
By "dEGFP" is meant<br>
By "DNA" is meant<br>
By "Derivative" is meant<br>
By "DsRed" is meant By "ECFP" is meant<br>
By "EGF' is meant BY "EGFP" is meant<br>
By "ELISA" is meant By "ErbI12" is meant<br>
By "Exon" is meant<br>
"Expression Vector" is meant<br>
By "EYFP" is meant<br><br>
a mutant form of EGFP variants that is fused to a PEST sequence and consequently has a half-life of only 2 hours. Similarly, d2ECFP (cyan) and d2EYFP (yellow) are also available. A destabilised variant of DsRed could possibly be made in the same way. Henceforth referred to as d2DsRed.<br>
a general term for all destabilised variants of EGFP (including all colors) formed. (Li et al).<br>
deoxyribonucleic acid.<br>
a polynucleotide or polypeptide that has been derived from a reference polynucleotide or polypeptide, respectively, for example by conjugation or complexing with other chemical moieties or by post-traascriptional or post-translational modification techniques as would be understood in the art.<br>
the red fluorescent protein isolated from the IndoPacific sea anemone relative Discosoma species.<br>
the mutant form of EGFP with altered excitation/emission spectra that fluoresces cyan colored light.<br>
Epidermal growth factor<br>
the enhanced green fluorescent protein. A mutant form of GFP with enhanced fluorescence. (Cormack et al).<br>
the enzyme-linked immunosorbent assay<br>
the second member of the epidermal growth factor receptor family. Also known as HER-2<br>
the sequences of an RNA primary transcript that are part of a messenger RNA molecule, or the DNA that encodes such sequences. In the primary transcript neighbouring exons are separated by introns.<br>
a vector that allows a cloned segment of DNA to be expressed inside a cell.<br>
a mutant form of EGFP with altered excitation/emission spectra that fluoresces yellow colored light.<br><br><br>
By "Firefly Luciferase" is meant<br>
By "Flow Cytometry" is meant<br>
By "Gene" is meant<br>
By "GFP" is meant<br>
By "Half-life" is meant By "Intron" is meant<br><br>
the enzyme derived from the luc gene, cloned from the firefly. Catalyzes a reaction using D-luciferin and ATP in the presence of oxygen and Mg- resulting in light emission. Often used as a reporter.<br>
a method, in which live or fixed cell suspensions are applied to a flow cytometer that individually measures an activity or property of a detectable label associated with the cells of the suspension. Labelling of cells can occur, for example, via fluorescent compounds or by antibodies covalently attached to a specific fluorescent compound. Several different excitation/emission wavelengths can be tested simultaneously to measure different types of fluorescence. Sub-populations of cells with desired characteristics (fluorescence, cell size) can be gated such that further statistical analyses apply only to the gated cells. Flow cytometers equipped with a cell sorting option can physically separate cells with the desired fluorescence and retrieve those (live) cells in a tube separate from the remainder of the initial cell population. Also referred to as FACS (fluorescence activated cell sorting).<br>
the segment of DNA that encodes a RNA molecule. The term "gene" sometimes but not always includes the promoter region.<br>
a fluorescent protein (Tsien et a/), which is isolatable from the jellyfish Aequoria victoria, and which can be used as a reporter protein. DNA constructs encoding GFP can be expressed in mammalian cells and cause the cells to fluoresce green light when excited with specific wavelengths. The term "GFP" is used herein to refer to all homologues and analogues, including colour variants and fluorescent proteins derived from organisms other than Aequoria victoria (e.g., DSRed, Clonetech; hrGFP, Stratagene).<br>
the time taken for half of the activity, amount or number of molecules to be eliminated.<br>
a non-coding sequence within a gene, or its primary transcript, that is removed from the primary transcript and is not present in a corresponding messenger RNA molecule.<br><br><br>
By "Luciferase" is meant<br>
By "MAPK" is meant<br>
By "MCMV" is meant<br>
By "MCS" is meant<br>
By "mRNA" is meant<br>
By "MODC" is meant<br><br>
the commonly used reporter enzyme that catalyses a reaction, which leads to light emission. Exogenous substrates are added and the reaction is quantified using a luminometer. The substrate requirements for firefly and Renilla luciferases are different, allowing the two to be distinguished in the Dual Luciferase Assay (Promega, Madison, WI, USA).<br>
Mitogen Activated Protein Kinase. Includes several different kinases involved in intracellular signal transduction pathways that lead to growth or apoptosis (cell death). The term "MAPK" is sometimes used in reference to two specific MAPKs, Erkl and Erk2 (extracellular regulated kinases 1 and 2).<br>
Minimal CMV promoter. Does not activate transcription on its own but can be linked to a TRE to provide tetracycline (and doxycycline)-dependent transcription.<br>
Multiple Cloning Site. The region of a DNA vector that contains unique restriction enzyme recognition sites into which a DNA fragment can be inserted. The term "MCS" as used herein, also includes any other site that assists the insertion of DNA fragments into the vector. For example, a T overhang (Promega, Madison, WI, USA), which allows direct insertion of fragments generated by polymerase chain reaction (PCR).<br>
Messenger RNA. A "transcript" produced in a cell using DNA as a template, which itself encodes a protein. mRNA is typically comprised of a 5'UTR, a protein encoding region and a 3'UTR. mRNA has a limited half-life in cells, which is determined, in part, by stability elements, particularly within the 3'UTR but also in the 5'UTR and protein encoding region.<br>
Mouse ornithine decarboxylase or the portion and/or derivative thereof containing a PEST sequence.<br><br><br><br>
By "Modulating" is meant<br><br>
increasing or decreasing, either directly or indirectly, the stability of a molecule of interest.<br><br>
By "operably connected" or "operably<br>
linked" and the like is meant	a linkage of polynucleotide elements in a functional<br>
relationship. A nucleic acid is "operably linked" when it is placed into a functional relationship with another nucleic acid sequence. For instance, a promoter or enhancer is operably linked to a coding sequence if it affects the transcription of the coding sequence. Operably linked means that the nucleic acid sequences being linked are typically contiguous and, where necessary to join two protein coding regions, contiguous and in reading frame. A coding sequence is "operably linked to" another coding sequence when RNA polymerase will transcribe the two coding sequences into a single mRNA, which is then translated into a single polypeptide having amino acids derived from both coding sequences. The coding sequences need not be contiguous to one another so long as the expressed sequences are ultimately processed to produce the desired protein, "Operably connecting" a promoter to a transcribable polynucleotide is meant placing the transcribable polynucleotide (e.g., protein encoding polynucleotide or other transcript) under the regulatory control of a promoter, which then controls the transcription and optionally translation of that polynucleotide. In the construction of heterologous promoter/structural gene combinations, it is generally preferred to position a promoter or variant thereof at a distance from the transcription start site of the transcribable polynucleotide, which is approximately the same as the distance between that promoter and the gene it controls in its natural setting; i.e.: the gene from which the promoter is derived. As is known in the art, some variation in this distance can be accommodated without loss of function. Similarly, the preferred positioning of a regulatory sequence element (e.g., an operator, enhancer etc) with respect to a transcribable polynucleotide to be placed under its control is defined by the positioning of the element in its natural setting; i.e. the genes from which it is derived.<br><br><br>
The term "pA" is meant<br>
The term "PEST1 refers<br>
By "Plasmid" is meant<br><br>
used in diagrams here to indicate a poly adenylation site. A DNA sequence that serves as the site to stop transcription and add to the immature mRNA, a polyA tail. Various pA sequences from SV40 virus genes or the β galactosidase gene or other sources, including synthetic polyadenylation sites can be used in expression vectors for this purpose.<br>
to an amino acid sequence that is enriched with the amino acids proline (P), glutamic acid (E), serine (S) and threonine (T). Proteins containing PEST sequences have shortened half-lives.<br>
a Circular DNA vector. Plasmids contain an origin of replication that allows many copies of the plasmid to be produced in a bacterial (or sometimes eukaryotic) cell without integration of the plasmid into the host cell DNA.<br><br><br><br>
By "Polynucleotide" or "Nucleic acid" is meant<br><br>
linear sequences of nucleotides, including DNA or RNA, which may be double-stranded or single-stranded..<br><br><br><br>
By    "Polypeptide",    "Peptide"    or "Protein" is meant<br>
By "Promoter" is meant<br>
By "PMA" is meant<br>
By "Renilla Luciferase" is meant<br>
By "Reporter Vector" is meant<br><br>
a polymer of amino acids joined by peptide bonds in a specific sequence.<br>
a region of DNA, generally upstream (5') of the mRNA encoding region, which controls the initiation and level of transcription. This term also includes within its scope inducible, repressible and constitutive promoters.<br>
Phorbol myristoloic acid<br>
that derived from sea pansy (Renilla reniformis), utilizes oxygen and coelenterate luciferin (coelenterazine) to generate light emission<br>
a expression vector containing a "Reporter Gene" that encodes a protein or polypeptide (or mRNA) that can be easily assayed. Typically, the reporter gene is linked to regulatory sequences, the function or activity of which, is being tested.<br><br><br>
By "Reporter" is meant<br>
By "RNA" is meant By "rtTA" is meant<br>
By "SEAP" is meant By "SKBR3" is meant<br>
By "Stringent conditions" is meant<br><br>
a molecule, typically a protein or polypeptide, that is encoded by a reporter gene and measured in a reporter assay. Current systems generally utilize an enzymatic reporter and measure reporter activity.<br>
Ribonucleic Acid.<br>
Reverse tTA (see below), which binds the TRE and activates transcription only in the presence of tetracycline or doxycycline.<br>
Secreted alkaline phosphatase reporter gene.<br>
the human breast cancer cell line that overexpresses ErbB2.<br>
temperature and ionic conditions under which only nucleotide sequences having a high frequency of complementary bases will hybridise. The stringency required is nucleotide sequence dependent and depends upon the various components present during hybridisation and subsequent washes, and the time allowed for these processes. Generally, in order to maximise the hybridisation rate, non-stringent hybridisation conditions are selected; about 20 to 25° C lower than the thermal melting point (Tm). The Tm is the temperature at which 50% of specific target sequence hybridises to a perfectly complementary probe in solution at a defined ionic strength and pH. Generally, in order to require at least about 85% nucleotide complementarity of hybridised sequences, highly stringent washing conditions are selected to be about 5 to 15" C lower than the Tm. In order to require at least about 70% nucleotide complementarity of hybridised sequences, moderately stringent washing conditions are selected to be about 15 to 30° C lower than the Tm. Highly permissive (low stringency) washing conditions may be as low as 50° C below the Tm, allowing a high level of mis-matching between hybridised sequences. Those skilled in the art will recognise that other physical and chemical parameters in the hybridisation and wash stages can also be altered to affect the outcome of a detectable hybridisation signal from a specific level of homology between target and probe sequences.<br><br><br>
"SV40/CMV/RSV"<br>
By "TetO" is meant<br>
By "Tet-Off Cell Lines" is meant<br>
By "Tet-On Cell Lines" is meant<br><br>
to refer to promoter elements derived from simian virus, cytomegalovirus and rous sarcoma virus respectively. Generally, these promoters are thought to be constitutively active in mammalian cells.<br>
the Tet operator DNA sequence derived from the E. coli tetracycline-resistance operon.<br>
cell lines stably expressing tTA such that tetracycline or doxycycline will shut off transcription from TRE promoters.<br>
cell lines stably expressing rtTA such that tetracycline or doxycycline will turn on transcription from TRE promoters.<br><br><br><br>
By "Transcription" is meant<br><br>
the   process   of   synthesizing   a   RNA complementary to the DNA template.<br><br>
molecule<br><br><br><br>
By "Transfection" is meant<br>
By "Translation" is meant<br>
By"TRE"is meant<br>
By "tTA" is meant<br><br>
the process during which a plasmid or DNA fragment is inserted into a eukaryotic cell. Typically, 2-50% of cells take up the plasmid and express the protein product for ~3 days without incorporating the plasmid DNA into the cell's chromosomes (= transient transfection). A small proportion of these cells will eventually incorporate the plasmid DNA into their gemone and permanently express the protein product (= stable transfection).<br>
the process whereby an mRNA molecule is used as a template for protein synthesis.<br>
here to define any Tetracycline Responsive Element (Gossen et al), generally combined with a minimal promoter such that transcription occurs only via the binding of exogenous factors (e.g., tTA or rtTA) to the TRE. Preferred embodiments of this invention utilize a TRE comprised of 7 repeats of the tetO sequence linked to a minimal CMV promoter (mCMV) (Clontech Laboratories Inc., Palo Alto, CA, USA).<br>
Tetracycline-controlled transactivator, which is comprised of the Tet represser protein (TetR) and the VP16 activation domain, such that it binds the TRE and activates transcription, only in the absence of tetracycline or doxycycline.<br><br>
By "TS" is meant By "Variant" is meant<br>
By "Vector" is meant<br><br>
Thromboxane synthase promoter.<br>
a polynucleotide or po]ypeptide displaying substantial sequence identity with a reference polynucleotide or polypeptide, respectively. Variant polynucleotides also include polynucleotides that hybridise with a reference sequence under stringent conditions. These terms also encompasses  polynucleotides   which   differ  from   a reference polynucleotide by the addition, deletion or substitution of at least one nucleotide. In this regard, it is well understood in the art that certain alterations inclusive   of   mutations,    additions,    deletions    and substitutions can be made to a reference polynucleotide whereby the altered polynucleotide retains the biological function or activity of the reference polynucleotide. The terms   "polynucleotide  variant"  and   "variant" also include naturally occurring allelic variants. With regard to variant polypeptides, it is well understood in the art for example that some ammo acids may be changed to others with broadly similar properties without changing the    nature   of   the    activity   of   the    polypeptide (conservative substitutions).<br>
a vehicle for inserting a foreign DNA sequence into a host cell and/or amplifying the DNA sequence in cells that support replication of the vector. Most commonly a plasmid but can also be a phagemid, bacteriophage, adenovirus or retrovirus.<br><br>
By  "vEGFP"   "EGFP  variants"  or   the different color variants of EGFP and/or the different<br>
"variant of EGFP" is meant	half-life variants.<br><br>
By "vGFP" is meant<br><br>
all variants of GFP; including homologues and analogues such as DsRed, also EGFP variants or destabilised GFP variants.<br><br>
The present invention provides inter alia expression constructs which modulate the stability of transcripts and consequently, the amount of protein produced by the vector. Although constructs which increase the stability of a transcript are clearly encompassed by the present invention, a particularly preferred embodiment focuses on destabilising transcripts. Here transcript stability can be reduced by the addition of one or more destabilising elements to, or by the removal of one or more stability elements (e.g., a poly A tail) from, a transcribable polynucleotide. Compared to existing expression constructs, the construct of the present<br><br>
invention provides kinetics of protein expression with improved temporal correlation to the promoter activity, e.g., by reducing the time lag between decreased promoter activity and decreased levels of a corresponding expression product.<br>
Accordingly, one aspect of the present invention is directed to a construct comprising in operable linkage: a polynucleotide which encodes a polypeptide and a nucleic aci;l sequence which encodes an RNA element that modulates the stability of a transcript encoded by the polynucleotide.<br>
The term "modulates" in the context of transcript stability refers to increasing or decreasing the stability of a transcript and optimal amounts of modulation depends upon the particular application. Without limiting the present invention to any one particular theory or mode of operation, where the RNA element is a sequence of nucleotides which destablilises the transcript, it is envisaged that the element directly or indirectly targets the transcript for degradation.<br>
As used herein the term "destabilising element" refers to a sequence of amino acids or nucleotides which reduces the half-life of a protein or transcript, respectively, inside a cell. Accordingly, an "RNA destabilising element" comprises a sequence of nucleotides which reduces the intracellular half-life of an RNA transcript and a "protein destabilising element" comprises a sequence of amino acids which reduces the intracellular half-life of a protein.<br>
The extent of the reduction sought depends upon the, particular application. In a preferred embodiment the extent of RNA destabilisation significantly improves the temporal correlation between promoter activity and reporter levels or activity in expression vectors compared to vectors without destabilisation elements. In relation to increasing transcript stability, optimum levels of stability will again depend upon the application.<br>
An "RNA stabilising element" is a sequence of nucleotides which increases the intracellular half-life.<br><br>
"RNA" molecules include all RNA molecules such as mRNA, heterogenous nuclear RNA (hnRNA), small nuclear RNA (snRNA), small nucleolar RNA (snoRNA), small cytoplasmic RNA (scRNA), ribosomal RNA (rRNA), translational control RNA (tcRNA), transfer RNA (tRNA), eRNA, messenger-RNA-interfering complementary RNA (micRNA) or interference RNA (iRNA) and mitochondrial RNA (mtRNA).<br>
Messenger RNA (mRNA) is a preferred form of RNA.<br>
In the context of reducing the intracellular half-life of a molecule selected from an RNA transcript or an encoded protein of interest, (a) one or more destabilising elements and/or (b) one or more stabilising elements are typically chosen to confer a level of enhanced degradation on the molecule, which thereby reduce(s) the intracellular half-life of the molecule to a half-life that is suitably less than about 24 hours, more preferably less than about 10 hours, even more preferably less than about 5 hours, even more preferably less than about 3 hours, even more preferably less than about 1 hour, even more preferably less than about 30 minutes, even more preferably less than about 15 minutes, even more preferably less than about 10 minutes, even more preferably less than about 5 minutes, and still even more preferably less than about 3 minutes. The half-life of an RNA transcript or an encoded protein of interest preferably corresponds to the lowest half-life that provides a steady-state expression level of at least 10 fold the minimum detectable level of the transcript or encoded protein.<br>
The intracellular or intracellular-like conditions are preferably physiological for the cell type. The temperature of the intracellular or intracellular-like conditions is preferably physiological for the cell type. Exemplary temperatures for mammalian cells range suitably from about 30° C to about 42° C, and preferably from about 35° C to about 37° C.<br>
At a minimum, enhanced ribonucleic or proteolytic degradation of an RNA transcript or polypeptide, respectively, refers to a level of ribonucleic or proteolytic degradation that is at least about 5%, preferably at least about 10%, more preferably at least about 20%, even more preferably at least about 40%, even more preferably at least about 50%, even more preferably at least about 60%, even more preferably at least about 70%, even more preferably at least<br><br>
about 80%, even more preferably at least about 90%, even more preferably at least about 100%, even more preferably at least about 150%, even more preferably at least about 200%, even more preferably at least about 400%, even more preferably at least about 600%, even more preferably at least about 1,000%, even more preferably at least about 2,000%, even more preferably at least about 4,000%, even more preferably at least about 6,000%, even more preferably at least about 8,000%, preferably at least about 10,000%, still even more preferably at least about 12,000%, greater than that of the RNA transcript or polypeptide in the absence of the destabilising element(s) or in the presence of a stabilising element(s). Assays for measuring RNA degradation are known to those of skill in the art. For example, RNA degradation can be measured using a range of assays disclosed for example by Ross, J (1995) or by Liu, J et al. (JBC 2000), which are based on the use of transcriptional inhibitors (Actinomycin D, DRB, cordycepin, alpha-amanitin), pulse labelling (radioactive nucleosides), cell-free decay methods (polysomes, cytosol or reticulocytes), or short-term promoter activation (fos promoter, see below). Assays for measuring degradation of proteins are also known to persons of skill in the art. For example, proteolytic degradation may be measured in vitro using a mammalian cell lysate assay including, but not restricted to, the reticulocyte lysate assay of Bachmair et al in U.S. Patent Serial No. 5,646,017. Alternatively, proteolytic degradation may be measured in vivo using cycloheximide or pulse-chase protocols as for example disclosed by Vazhappilly, R and Sucher, N (2002) or by Saito, T et al. (1998).<br>
The RNA destabilising elements can be derived from any source and in particular the 3' UTR or 5' UTR regions of short-lived mRNAs often contain destabilising sequences. As used herein, the term "derived from" shall be taken to indicate that a particular integer or group of integers has originated from the species specified, but has not necessarily been obtained directly from the specified source.<br>
The RNA destabilising sequences may be cloned from short-lived RNAs such as, for example; c-fos, c-jun, c-myc, GM-CSF, IL-3, TNF-alpha, IL-2, IL-6, IL-8, IL-10, Urokinase. bcl-2, SGLTI (Na(+)-coupled glucose transporter), Cox-2 (cyclooxygenase 2), IL8, PAI-2 (plasminogen activator inhibitor type 2), betal-adrenergic receptor, GAP43 (5'UTR and 3'UTR).<br><br>
AU-rich elements (AREs) and/or U-rich elements (UREs), including but not limited to single, tandem or multiple or overlapping copies of the nonamer UUAUUUA(U/A)(U/A) (where U/A is either an A or a U) (Lagnado et al 1994) and/or the pentamer AUUUA (Xu et al 997) and/or the tetramer AUUU (Zubiaga et al. 1995).<br>
RNA destabilising elements have also been described for example from phosphoenolpynivatc carboxy kinase mRNA (PEPCK), the Drosophila Bicoid gene, the human thioredoxin gene, heat stable antigen and soybean 10A5 gene.<br>
Iron responsive elements and iron regulatory protein binding sites may also advantageously be incorporated into the instant vectors to modulate RNA stability and particularly, translational efficiency. Histone RNAs, particularly their 3'UTRs, are especially useful for modulating RNA stability in a cell-cycle dependent fashion.<br>
Also contemplated are modifications to or permutations of the elements listed above. The term "tandern copies" allows for both duplication and/or non-duplication of one or more of the outer nucleotides. For example, tandem copies of the pentamer AUUUA, includes sequences such as AUUU AUUU AUUUA as well as AUUUA AUUU AAUUUA.<br>
RNA destabilising elements may be identified and or modifications made thereto using a computational approach and database analysis (Dandekar T et al).<br>
Accordingly, biologically active fragments as well as variants and derivatives of reference destabilising elements are encompassed by the present invention.<br>
Eukaryotic expression vectors are contemplated.<br>
In a related embodiment the present invention contemplates a construct comprising in operable linkage: a polynucleotide which encodes a polypeptide and a nucleic acid sequence which encodes an stabilising RNA element that enhances the stability of a transcript encoded by<br><br>
polynucleotide.<br>
In another related embodiment the present invention contemplates a construct comprising in operable linkage: a polynucleotide which encodes a polypeptide and a nucleic acid sequence which encodes an RNA stabilising element that enhances the stability of a transcript encoded by the polynucleotide, wherein the stabilising element is, or is derived from, a gene selected from alpha2 ftlobin, alpha] glohin. beta glohin, or growth hormone, which are examples of long-lived rnRNAs. As used herein, underscoring or italicising the name of a gene shall indicate the gene, in contrast to its protein product, which is indicated bv the name of the gene in the absence of any underscoring or italicising. For example, "alpha? x/ohin" shall mean the alpha? globin gene, whereas "alpha2 globin" shall indicate the protein product of the "alpha?.<br>
globin^ gene.<br>
The ability to destabilise a transcript and reduce the amount of protein produced by a cell will clearly be useful for a wide range of applications.<br>
Another aspect of the present invention contemplates a construct comprising in operable linkage a polynucleotide which encodes a polypeptide and a nucleic acid sequence which encodes an RNA destabilising element that reduces the stability of a transcript encoded by the polynucleotide.<br>
In another aspect, the present invention contemplates a construct comprising in operable linkage a polynucleotide which encodes a polypeptide and a nucleic acid sequence which encodes an RNA destabilising element that reduces the stability of a transcript encoded by the polynucleotide, wherein the nucleic acid sequence is, or is derived from, a gene selected from c-fos. c-jun, c-myc, GM-CSF, IL-3, TNF-aipha, IL-2, IL-6, IL-S, IL-10, Urokinase, bcl-2. SGI-TI (Na(+)-coupled glucose transporter), Cox-2 (cyclooxygenase 2). . fL-ti, PAl-2 en activator inhibitor type 2), betal-adrencrgic receptor or, d'AJ'43.<br>
In one particular embodiment the nucleic acid sequence sequences encoding the RNA destabilising elements are linked to sequences encoding a protein of interest, which in turn is<br><br>
linked to a promoter of interest that is preferably modulatable (i.e., inducible or repressive) such that expression is turned on and then off modulation . In this application, the RNA destabilising elements typically serve to shorten the period of expression of a functional mRNA or protein. This may be applied in vitro or in vivo. For example, a cell cycle-specific promoter could be combined with the RNA instability elements to express a protein of interest, exclusively in certain stages of the cell cycle. The protein of interest may be a functional protein or a reporter protein. In the latter example, reporter levels can be used as an indicator of cell-cycle stage or cell proliferation.<br>
Yet another aspect of the present invention contemplates a construct comprising in operable linkage: a polynucleotide which encodes a polypeptide and a nucleic acid sequence which encodes an RNA destabilising element that reduces the stability of a transcript encoded by the polynucleotide, wherein the nucleic acid sequence is selected from any one of SEQ ID NOS 1 to 23, or biologically active fragments thereof, or variants or derivatives of these.<br>
Even another aspect of the present invention contemplates a construct comprising in operable linkage: a polynucleotide which encodes a polypeptide and a nucleic acid sequence which<br>
encodes an RNA destabilising element that reduces the stability Ma transcript encoded by the<br>
polynucleotide, wherein the nucleic acid sequence is set forth in SEQ ID NO: 1, 2 or 22 or<br>
biologically active fragments, variants or derivatives thereof.<br>
One particular application is in the area of determining gene expression. Specifically, by reducing the amount of transcript produced in a cell it is possible to more accurately determine promoter or enhancer activity. In this application a reporter gene is used to determine promoter activity either directly, or indirectly as a fusion protein with another polypeptide whose expression has been modulated by regulatory elements within the vector.<br>
In one embodiment the RNA destabilising sequences are incorporated into the region encoding the 3'-UTR of the reporter mRNA. Alternatively or in addition, destabilising elements are incorporated into the 5'-UTR and/or protein coding region, which is preferably not essential to, or does not interfere with, the selected activity of the encoded protein.<br><br>
fn a related embodiment the RNA destabilising sequences are used to destabilise a gene of interest when for example there is a need to accurately monitor or reduce its expression.<br>
Typically for this application, RNA destabilising elements are used in conjunction with<br>
reporter protein destabilising elements.<br>
The subject expression vectors have applications in a variety of gene expression systems where it is preferable to have a brief period of mRNA or protein expression or where it is preferable to minimise the time lag between changes in promoter activity and the resultant changes in. mRNA/protein levels.<br>
Even yet another aspect of the present invention contemplates a construct comprising in operable linkage: a polynucleotide which encodes a reporter polypeptide and a nucleic acid sequence which encodes an RNA destabilising element that reduces the stability of a transcript encoded by the polynucleotide.<br>
The expression vectors are designed for use in eukaryotic cell systems. It should be noted however that the RNA destabilising elements may be used in a wide range of eukaryotic and/or plants systems including cells, tissues or whole organisms defined as yeast, insect, nematode, fish, bird or mammal. For use in plants, different promoters and possibly different reporters and RNA destabilising elements (e.g., DST sequences) may be used.<br>
It is contemplated that the expression vectors of the present invention will incorporate standard protein reporter molecules or destabilised reporter protein molecules. Standard reporter molecules arc well known in the art.<br>
Another aspect of the present invention contemplates the combination of a protein destabilising, element (e.g., a DNA/RNA sequence encoding an intracellular protein degradation signal or degron which may be selected from a destabilising amino acid at the amino-terminus of a polypeptide of interest, a PEST region or a ubiquitin) and an mRNA destabilising element (e.g., multiple copies of the nonamer UIJAUIJUAUU), such that both mRNA and protein are<br><br>
destabilised. For example, one such embodiment incorporates into an expression vector, a PEST sequence immediately upstream of the translation stop codon and 4 nonamers located downstream of the stop codon (preferably 20nt or more from stop codon).<br>
In this way, reporter protein may be destabilised both at the protein level and the mRNA level.<br>
The destabilised reporter protein may be any suitable protein. For example, destabilised GFP proteins are suitable, such as for example dl EGFP, dlEYFP and dlECFP comprising the d 1<br>
mutant of MODC. The destabilised luciferase protein has been described by Leclerc G. et al. The MODC PEST sequence was used. The MODC from dlEGFP is also contemplated.<br>
Any method of destabilising a polypeptide of interest is contemplated by the present invention. For example, a polypeptide of interest can be modified to include a destabilising amino acid at its amino-terminus so that the protein so modified is subject to the N-end rule pathway as disclosed, for example, by Bachmair et al in U.S. Patent Serial No. 5,093,242 and by Varshavsky et al. in U.S. Patent Serial No. 5,122,463. In a preferred embodiment of this type, the destabilising amino acid is selected from isoleucine and glutamic acid, more preferably from histidine tyrosine and glutamine, and even more preferably from aspartic acid, asparagine, phenylalanine, leucine, tryptophan and lysine. In an especially preferred embodiment, the destabilising amino acid is arginine. In some proteins, the amino-terminal end is obscured as a result of the protein's conformation (i.e., its tertiary or quaternary structure). In these cases, more extensive alteration of the amino-terminus may be necessary to make the protein subject to the N-end rule pathway. For example, where simple addition or replacement of the single amino-terminal residue is insufficient because of an inaccessible amino-terminus, several amino acids (including lysine, the site of ubiquitin joining to substrate proteins) may be added to the original amino-terminus to increase the accessibility and/or segmental mobility of the engineered amino terminus.<br>
Modification or design of the amino-terminus of a protein can be accomplished at the genetic level. Conventional techniques of site-directed mutagenesis for addition or substitution of appropriate codons to the 5 end of an isolated or synthesised antigen-encoding polynucleotide can be employed to provide a desired amino-terminal structure for the encoded protein. For example, so that the protein expressed has the desired amino acid at its amino-terminus the appropriate codon for a destabilising amino acid can be inserted or built into the amino-terminus of the protein-encoding sequence. Where necessary, a nucleic acid sequence encoding the amino-terminal region of a protein can be modified to introduce a lysine residue in an appropriate context. This can be achieved most conveniently by employing DNA constructs encoding "universal destabilising segments". A universal destabilising segment comprises a nucleic acid construct which encodes a polypeptide structure, preferably segmentally mobile,<br>
containing one or more lysine residues, the codons for lysine residues being positioned within the construct such that when the construct is inserted into the coding sequence of the antigen-encoding polynucleotide, the lysine residues are sufficiently spatially proximate to the amino-terminus of the encoded protein to serve as the second determinant of the complete amino-terrninal degradation signal. The insertion of such constructs into the 5' portion of a antigen-encoding polynucleotide would provide the encoded protein with a lysine residue (or residues) in an appropriate context for destabilisation.<br>
In another embodiment, the polypeptide of interest is modified to contain a PEST region, which is rich in an amino acid selected from proline, glutamic acid, serine and threoninc, which region is optionally flanked by amino acids comprising electropositive side chains. In this regard, it is known that amino acid sequences of proteins with intracellular half-lives less than 2 hours contain one or more regions rich in proline (P), glutamic acid (E), serine (S), and threonine (T) as for example shown by Rogers et al. (1986, Science 234 (4774): 364-368). In yet another embodiment, the polypeptide of interest is conjugated to a ubiquitin or a biologically active fragment thereof, to produce a modified polypeptide whose rate of intracellular proteolytic degradation is increased, enhanced or otherwise elevated relative to the unmodified polypeptide.<br>
Still another aspect of the present invention contemplates an expression vector comprising a transcribable polynucleotide which comprises a sequence of nucleotides encoding an RNA element which modulates the stability of a transcript corresponding to said transcribable polynucleotide; wherein said RNA clement is a destabilising clement which reduces the stability of said transcript, wherein said reporter protein is selected from Luci ferase, Green Fluorescent Protein, Red Fluorescent Protein, SEAP, CAT, or biologically active fragments thereof, or variants or derivatives of these.<br>
Such vectors can be used to screen for drugs or treatments that alter the activity of that promoter. Compared to existing reporter vectors, a near "real-time" measurement of drug action can be obtained.<br>
Still another aspect of the present invention is directed to a construct and preferably an expression vector comprising in operable linkage: a polynucleotide which encodes it polypeptide and a nucleic acid sequence which encodes an RNA element that modulates the stability of a transcript encoded by the polynucleotide, wherein the construct comprises one or more members in any order selected from the group consisting of:<br>
(i) a multiple cloning site for introducing a sequence of nucleotidcs, which site is preferably cleavable enzymatically or otherwise biochemically to provide a linearised vector into which PCR amplification products are clonable directly (e.g., an EclHKl site);<br>
(ii)       a reporter gene;<br>
(iii) a promoter and/or enhancer for regulating expression of said transcribable polynucleotide;<br>
(iv)      a polyadenylation sequence; (v)       a selectable marker gene; and (vi)      an origin of replication.<br>
Another aspect of the present invention contemplates vectors or sets of vectors, particularly but not exclusively, plasmids, with applications in the study or measurement or monitoring of gene expression (e.g., promoter activity). Many other vectors could also be used such as for example viruses, artificial chromosomes and other non-plasmid vectors.<br>
One embodiment involves pairs or sets of plasmids, each containing one or more of the mRNA destabilising sequences described above incorporated into a construct encoding a destabilised reporter protein such as, for example, dlEGFP, dlEYFP, dlECFP or dlDsRed. One plasmid<br>
 (the control) from each pair or set contains a promoter 5' of the reporter encoding region. The promoter is comprised of elements which is modulatable (i.e., inducible or repressible) by exogenous treatments (e.g., the TRE combined with a minimal promoter such as mCMV; see Fig, 2c). Alternatively, a constitutively active promoter such as TS, SV40, CMV, TK or RSV is used (see Fig. 2b). In plant systems the Top-ten promoter could replace TRE, and (he 35S promoter of cauliflower mosaic virus can replace SV40 etc. Agrobacterium tumefaciens can be used in plants to facilitate gene transfer. The other plasmid(s) in the pair or set are identical to the control plasmid, except that a cloning site (MCS) replaces the promoter, and the reporter encoding region encodes a reporter similar to but distinguishable from the control reporter (see Fig. 2a). In a preferred embodiment, the control plasmid encodes a destabilised variant of EGFP (e.g., dl EGFP, dlEYFP or dlECFP) and the other vectors (test vectors) each encode a different colour variant from the same list (same protein half-life).<br>
In other embodiments, a control and one of the test reporters are incorporated into a single vector, such as for example a bi-directional plasmid (see Fig. 3).<br>
In the above embodiments, both control and test plasmids encode a destabilised raRNA, which in turn encodes a destabilised protein. Thus the time lag between decreased promoter activity and decreased reporter protein levels, is significantly reduced compared to the time lag with existing constructs. Similarly, increased promoter activity is more readily and quickly detectable due to the reduced levels of pre-existing mRNA and protein. Other differences between the control and test constructs, which can lead to errors, arc minimised by using fluorescent proteins that differ from each other by only a few small mutations. Compared to luciferase or other enzyme based assays, the fluorescent reporters described here, offer several other advantages including:<br>
•	Several different reporters can he measured in the same cells/samples.<br>
•	Live cells can be measured, allowing multiple time points of the same samples or<br>
further manipulation post-measurement e.g., measurement of the same cells before and<br>
after treatment with a drug.<br><br>
• Successfully transfected cells can be visualised by fluorescent microscopy. Therefore poor transfections can be identified simply by looking at the cells under a microscope, without further investment of resources.<br>
•	No substrates are required, therefore the method is less technically demanding, faster,<br>
less expensive and more accurate.<br>
•	Both control and test reporter expression can be measured simultaneously by flow<br>
cytometry (see advantages of flow cytometry below).<br>
•	Embodiments utilising TREs as the control promoter can only be used in Tct-On or<br>
Tet-Off cell lines, but as compared to other control promoters, exhibit less interference<br>
from or to the test promoter and are less affected by various stimuli used to examine<br>
inducibility of the test promoter. Thus, they provide a more accurate measurement of<br>
transfection efficiency and relative test promoter activity. Control reporter expression<br>
can be switched on or off as required and used to confirm the lack of promoter cross<br>
talk or compensate for it if present.<br>
• In another embodiment, the control and one of the test reporters described above are both incorporated into a single vector, preferably a bi-directional plasmid. Interference between the two promoters, which is a major drawback of previous dual promoter vectors, is minimised by using TREs in the cqntrol promoter. Such a single vector system prevents the inaccuracies of co-transfection studies.<br>
The invention also provides vectors in which informative promoters or promoter fragments, are placed upstream of the reporter-encoding region. The present invention provides a simpler, quicker and more cost-effective reporter system for such assays when using utilising EGFP variants as opposed to luciferase or other enzymes as the reporter. Furthermore, the inclusion of mRNA instability elements allows a near real-time analysis.<br>
Informative promoters include, but are not restricted to, cell cycle-dependent promoters (e.g., cyclin A, B, or Dl, histone or topoisomerase I promoters), promoters activated by apoptotic (cell death) pathways and promoters/fragments linked to mitogenic signals (Table 1). Examples of informative enhancers that can be used include any of those used in Clontech's Mercury Pathway Profiling Systems. Clontech's Mercury In Vivo Kinase Assay Kits represent another example of how the present invention can be used. In this example the promoter element is a TRE that is combined in cells with a chimeric TetR-transactivator protein that permits transcription from the TRE only when a specific kinase is active and can phosphor/late the transactivator domain of the fusion protein. Thus, the present invention can be used to provide a more real-time measurement of specific kinase activity.<br>
Even still another aspect of the present invention contemplates a cell transfected or transduced with a vector according to the present invention comprising a transcribable polynucleotide which comprises a sequence of nucleotides encoding an RNA element which modulates the stability of a transcript corresponding to said transcribable polynucleotide.<br>
In some applications the expression vectors or cells expressing the reporter constructs are inserted into an organism to allow measurement of reporter activity in vivo. In some of these applications, destabilised luciferase rather than destabilised EGFP variants may be the preferred reporter. For example, transgenic mice expressing destabilised luciferase under the control of an informative promoter, can be used to measure the activity of that promoter in the tissues of a live mouse, using a photon camera (photon camera analysis is described by Contag, et al, 1997). The mRNA destabilising sequences serve to improve the temporal correlation between promoter activity and reporter levels, thus providing a significant improvement to applications such as drug screening, which benefit from a near real-time measurement of promoter activity.<br>
In some applications it is desirable to express, either in vitro in cell-based systems or in vivo in mammalian systems, both a reporter molecule and a functional gene product. This may involve two separate mRNAs, each containing an mRNA destabilising element. Alternatively, an mRNA destabilising elements may be incorporated into a single destabilised transcript that gives rise to two separate proteins (e.g., using an internal ribosome entry site; IRES) or a fusion<br>
-protein comprised of the reporter and the functional gene product.<br>
The invention also provides cell lines stably expressing these vectors (with or without a control). Such cells have applications in areas such as drug screening. For example, cells containing a MAPK-dependent reporter vector provide a rapid and inexpensive method for testing the efficacy of drugs designed to inhibit MAPK or any pathway upstream of MAPK-dependent transcription in those cells. In SKBR3 human breast cancer cells, for example, MAPK activity is dependent on signalling from the overexpressed ErbB2 protein. Therefore, drugs that inhibit ErbB2, would cause a decrease in the fluorescence of SKBR3colls containing such a construct but not in cells lacking ErbB2. Alternatively, cells could be tested ± drug and ± a specific ligand or treatment that leads to MAPK activation via a different pathway, in order to monitor inhibition of that pathway. Cell lines (or organisms) stably expressing a vector linked to a cell-cycle-regulated promoter can be used as very fast, simple and inexpensive means for measuring cell-cycle progression or cell proliferation. Such cell lines have obvious utility in drug screening and are contemplated in the present invention. Examples of cell-cycle regulated promoters are readily available, for example, (Lee, H et al. 1995), (Stein, J et al. 1996) and (Huet, X et al. 1996).<br>
Another embodiment of the present invention includes vectors, for the study of post-transcriptional regulation, particularly mRNA stability. The reporter is, for example, a destabilised variant of EGFP (e.g., dlEGFP, dlEYFP, dlECFP), with a different color variant in each separate vector, The TRE (linked to a minimal promoter such as mCMV) is 5' of the reporter encoding region and drives transcription in a tetracycline (or doxycycline) dependent fashion. Other inducible promoter systems can also be used.<br>
In one embodiment, the mRNA instability elements described above are not included and in their place, MCSs are located, primarily in the 3'-UTR (see Fig. 4a) but also in the 5'-UTR and/or coding region in some specific embodiments. Sequences thought to affect mRNA stability can be tested by cloning them into the appropriate cloning site of a vector containing one color variant and measuring the rate of decrease in reporter levels after blocking transcription with tetracycline or doxycycline (see Fig. 7). If desired, the rate of decay can be-<br>
compared between the "test vector" and the "control vector," (which encodes a different colour EGFP variant and does not contain the sequence being tested) in the same cells.<br>
The MCS may usefully comprise or work in conjunction with restriction endonuclease sites which allow direct cloning of PCR products having overhangs (see below).<br>
In another related embodiment of the invention, one or more mRNA instability element(s) are included to assist scientists specifically searching for mRNA stabilising elements. Similarly, other embodiments include mRNA stabilising element(s) to assist scientists specifically searching for mRNA destabilising elements,<br>
In other embodiments, the control and one of the test reporters are both incorporated into a single vector, preferably a bi-directional plasmid (see Fig. 4b).<br>
Stabilising elements are useful for increasing levels of expressed protein for example during protein purification where high levels or protein are required or when a promoter is weak.<br>
Interference between the two promoters and moreover, transcription effects of the element or various stimuli tested, is circumvented by using a TRE or similar element to drive both reporters and by measuring reporter levels after addition of doxycycline (or tetracycline), which shuts off transcription from the vector.<br>
Still another aspect of the present invention contemplates a cell transfected or transduced with a construct according to the present invention comprising in operable linkage a polynucleotide which encodes a polypeptide and a nucleic acid sequence which encodes an RNA element that modulates the stability of a transcript encoded by the polynucleotide.<br>
A related aspect of the instant invention considers a genetically modified non-human organism comprising in operable linkage: a polynucleotide which encodes a polypeptide and a nucleic acid sequence which encodes an RNA element that modulates the stability of a transcript encoded by the polynucleotide.<br>
Yet a further embodiment of the present invention contemplates a method for determining expression  of a polynucleotide of interest,  said  method  comprising expressing  said polynucleotide of interest from a reporter expression vector for a time and under conditions sufficient for RNA and protein synthesis to occur, said vector comprising a transcribable polynucleotide which comprises a sequence of nucleotides encoding a transcribed element and said polynucleotide of interest wherein said transcribed element modulates the stability of a transcript corresponding to said transcribable polynucleotide; and wherein said expression vector comprises one or more members in any order selected from the group consisting of:<br>
(i)        a multiple cloning site for introducing said polynucleotide of interest; (ii)       a reporter gene;<br>
(iii) a promoter for regulating expression of said polynucleotide of interest and/or a reporter gene, which promoter is preferably modulatable (e.g., using a tetracycline responsive element (TRE));<br>
(iv)      a polyadenylation sequence; (v)       a selectable marker gene; and (vi)      an origin of replication;<br>
and measuring the level or activity of the reporter over time compared to a control wherein said destabilising clement enhances the temporal correlation between promoter or enhancer activity and reporter level or activity.<br>
A combination of different RNA destabilising elements acting in combination is contemplated herein.<br>
The present invention is further described by the further non-limiting Examples.<br>
EXAMPLE 1 Cloning DNA Elements into Vectors<br>
Cloning is carried out according to existing methods, using restriction enzyme sites in the MCS or direct ligation of PCR products in the case of vectors with a "T overhang" in the MCS. With respect to post-transcriptional reporter vectors, however, the inclusion of a MCS in the 3'-UTR or other regions is a significant improvement over current vectors, which were designed for transcriptional or other studies and do not contain convenient cloning sites in these locations.<br>
EXAMPLE 2 Transfection<br>
Co-transfection of control and test vectors is performed as per existing methods (e.g., Fugene [Boehringer Mannheim, Mannheim, Germany] or electroporation), except in the case of the single (e.g., bi-directional) vector systems described above, which require only one vector and thus eliminate inaccuracies associated with co-transfection.<br>
EXAMPLE 3 Measurement of Reporter Expression<br>
An immediate advantage of the vGFP system is that reporter expression can be visualized directly in living cells, simply by viewing the tissue culture plate or flask under a fluorescent microscope. Therefore, poor transfections can be identified and discarded before any additional time is wasted. Quantitative measurement can be performed using a fluorometer (e.g., 96 well<br>
plate format) and since live cells can be measured, the same samples can be measured repeatedly e.g., in a time course.<br>
A further advantage compared to luciferase and other enzyme based assays is that flow cytometry can also be used to measure reporter levels.<br>
EXAMPLE 4 Advantages of Using Flow Cytometry to Measure Reporter Levels<br>
i. Two or more reporters (control and test) as well as additional parameters, can be measured individually in every cell at a rate of &gt;2,000 cells per second. Therefore, in this application, the method yields thousands to hundreds of thousands of data points per sample versus one datum point for existing luciferase assays.<br>
ii. Accurate measurement of transfection efficiency: This is useful for optimising transfection protocols. In addition to allowing comparison of different methods, it is also possible to measure both expression per cell and the proportion of cells expressing. This helps the investigator to determine the cause of any problems,<br>
iii. Identification of co-transfection errors: Co-transfection studies are based on the premise that cells will take up and express an amount of control reporters, which is proportional to the amount of test plasmid taken up by the same cells. This is not always the case. By using the flow cytometry method described here, it is possible to correlate test versus control expression levels,.in different cells of the same sample. Invalid samples can be identified by the lack of a good linear relationship between test and control reporter levels. Such errors go unnoticed in current methods.<br>
iv. Simultaneous measurement of additional parameters: Fluorescent labelled antibodies can be used to quantify specific proteins on a cell by cell basis and this can be correlated with reporter levels to determine whether that protein affects gene expression via the element cloned into the reporter construct. Alternatively, the protein of interest can be expressed as a vGFP-fusion protein (the protein of interest fused to a GFP<br>
variant) via transfection of an appropriate expression vector (inducible or non-inducible). Levels of the specific protein can then be correlated with the expression of a different GFP variant linked to a regulatory element of interest (co-transfected or transfected at a different time). In a third application, the vGFP reporter is linked to regulatory elements (e.g., promoters) thought to be cell cycle specific. Transfected cells are stained with a fluorescent DNA dye such as propidium iodide to measure DNA content, which is then correlated with reporter expression. In principle, several of the DNA constructs described herein, each containing a different vGFP, could he co-expressed and independently measured. Furthermore, other fluorescent markers could be used in conjunction with these vectors (singly or in multiples).<br>
v- Cell Sorting: Using a cell sorter, it is possible to isolate viable vGFP expressing cells from the non-expressors. This technique can be used to select stably expressing cells or to remove non-expressors prior to assay initiation. Similarly, it is possible to remove cells expressing very low and/or very high levels of vGFP. This can be used to generate a more homogeneous population and/or to remove cells expressing levels so high that they may not be physiological relevant or may perturb normal cellular function and/or may otherwise adversely affect the data obtained from the DNA vectors described herein.<br>
It is important to note that transient and stable transfections of expression vectors result in a cell population with very heterogeneous levels of expression. In general a thousand fold difference between the highest and lowest expressor is not unusual. The present invention not only offers a method for selecting homogeneous populations when required (see v above), but can also utilise heterogeneity to the benefit of the scientist. For example, identifying co-transfection errors. Another example of this relates to (iv) above. To determine whether protein X affects transcription from promoter Y, then cells are transfected with a reporter construct expressing dlEGFP under the control of promoter Y. If required, cell sorting can be used to isolate cells transiently or stably expressing appropriate levels of dlEGFP. These cells are in turn transiently transfected with a vector expressing a protein X-EYFP fusion protein. During How cytometry, EGFP is plotted on one axis and EYFP on the other. A positive correlation<br>
would indicate that protein X increases transcription from promoter Y and a negative correlation would indicate that protein X inhibits transcription from promoter Y.<br>
Currently, scientists attempting to establish such a correlation would select several different clones of high versus low expressors of protein X. Each clone would then be separately transtected with a promoter Y-luciferase construct and the luciferase activities measured. The use of cell clones requires months of preparation and introduces many variables including pre-existing heterogeneity amongst the host cells and variable sites of vector integration (vector DNA may interfere with a specific gene at the integration site and this site is different for every clone). Furthermore, such a method yields very few data points, with each datum point obtained from a different transfection of a different clone. Thus, the new system is not only more versatile but is quicker and more accurate than existing methods.<br>
EXAMPLE 5 Laser scanning cytometry (LSC)<br>
Unlike flow cytometry, LSC measures multi-colour fluorescence and light scatter of cells on slides, and records the position and time of measurement for each cell analysed. This technique provides data equivalent to flow cytometry but has the advantage of being microscope slide based (Darzynkiewicz et al., 1999; Kamentsky et al., 1997). Owing to the fluorescence of GFP and its variants, the techniques described for flow cytometry are also applicable to LSC,<br>
EXAMPLE 6 Specific Methods for Post-Transcriptional Assays<br>
These are best summarised by using the example of a study aimed at determining whether a specific 3'-UTR fragment affects mRNA stability. Although this example is one of transient expression, stable transfection could also be used<br>
(i). The 3'-UTR fragment is ligated into the 3'-UTR cloning site of the test vector and co-transfected with the control vector into a Tet-Off cell. line. In the case of the bi-directional vector, no control vector is required. Indeed, the typical application does not<br>
require a control reporter or vector since rate of decay can be measured in samples from within the same transfection, 5'-UTR fragments can be tested by inserting them into vectors with a 5'-UTR cloning site.<br>
(ii). The cells are grown in the absence of doxycycline (or tetracycline) for 6-48 h to allow expression of both vectors. Alternatively, cells are grown with low doses of doxycycline (or tetracycline), for 6-48 h to block transcription and then switched to medium without doxycycline (or tetracycline) for 2-12 h to provide a brief burst of transcription.<br>
(iii).     High doses of doxycycline (or tetracycline) are then applied to shut off transcription from both vectors.<br>
(iv).     The fluorescence of both reporters is measured (by flow cytometry, fluorometry or LSC) in a time course following addition of doxycycline (or tetracycline).<br>
If the cloned element confers mRNA instability, a more rapid decrease in "test" fluorescence will be seen compared to "control" fluorescence of the same cells or sample. Similar studies can be used to test an mRNA element's response to certain stimuli or its effect in different cells or cells expressing different amounts of a specific protein, such as an RNA-binding protein. Applying the stimulus after doxycycline will determine whether pre-existing transcripts are affected by the stimulus. Inserting the element in different locations (e.g., 5'-UTR, 3'-UTR) will determine whether its function is dependent on position. Inserting a protein/polypeptide coding sequence (in frame) within the reporter-coding region of the vector, can be used to determine the effect of that sequence on mRNA and protein stability.<br>
RNA can be extracted from transfected cells and used to measure reporter mRNA directly.<br>
EXAMPLE 7 Transcription Reporter Vectors<br>
The vectors are plasmids suitable for expansion in E, coli and expression of a fluorescent reporter in eukaryotic cells. The plasmids may be used in sets. Each set is comprised of one or more "control" vectors and one or more "test" vectors. Every vector within a set expresses a similarly destabilised mRNA and a similarly destabilised fluorescent reporter protein. In addition to the standard features of such plasmids (ampicillin resistance, origin of replication etc.), each plasmid contains the following construct (see also Figs 2 and 3):<br>
5'— MCS/prornoter-—transcription start site—5' UTR—ATG--vEGFP encoding region --stop codon—3'UTR with mRNA destabilising element—polyadenylation signal<br>
Where:<br>
MCS/promoter denotes either a multiple cloning site (test vectors; see Fig. 2a) or a constitutively active promoter such as SV40 (control vectors; see Fig. 2b) or an inducible promoter such as TRE-mCMV (control vector; see Fig2c).<br>
ATG denotes a translation start codon. Stop codon denotes a translation stop codon. 5'UTR denotes a 5' untranslated region.<br>
3'UTR with mRNA destabilising element denotes a 3' untranslated region containing one or more of the mRNA destabilising elements outlined.<br>
vEGFP denotes a destabilised variant of EGFP. One set of plasmids is provided for each type of destabilising modification (e.g., 1 hr half-life, 2 hr half-life), Within each set of plasmids,<br>
one vector is provided for each different colour variant. For example, one set contains vectors expressing dlEGFP, dlEYFP, dlECFP whereas another set expresses the d2 variants.<br>
In other examples, the control and one of the test reporters described above are both incorporated into a single vector, preferably a bi-directional plasmid (see Fig. 3).<br>
EXAMPLE 8 Post-transcription Reporter Vectors<br>
Similar to the transcription reporter "control" vectors that contain a TRE-mCMV promoter, except that the mRNA destabilising element in the 3'-UTR is replaced with a MCS (sec Fig. 4a). In some embodiments, MCS are also located in the 5' UTR and/or coding region.<br>
Such a construct can be used as a "test" or a "control" vector for the post-transcriptional assays outlined herein.<br>
In other examples, the control and one of the test reporters described above are both incorporated into a single vector, preferably a bi-directional plasmid (see Fig. 4b).<br>
EXAMPLE 9 Reporter Vectors for Assaying Specific Pathways<br>
Vectors similar to those described herein, into which a regulatory element has been inserted into the MCS for the purpose of studying or measuring the function of said regulatory element. For example, plasmids similar to the transcription reporter plasmids outlined herein, except that they contain within the MCS, a promoter or promoter element(s) or enhancer(s) that are responsive to pathways such as those referred to in Table 1 and/or contain any of the following cis-acting enhancer elements as described in Clontech's Mercury Pathway Profiling Systems: API, CRE, E2F, ORE, HSE, ISRE, Myc, NFAT, NFKB, p53, Rb, SRE. The reporter is preferably a destabilised version of GFP, luciferase or SEAP.<br>
Cell lines and Mice for Assaying Specific Pathways<br>
Cell lines or genetically modified mice stably expressing one or more of the vectors described herein.<br>
EXAMPLE 10<br>
Method of Use<br>
The vectors described in this invention are used for experimentation in essentially the same manner as the existing vectors that they replace, with the exception of the new methods described herein,<br>
Method of Construction<br>
The vectors and DNA constructs outlined here are assembled using standard cloning techniques. The SV40 and TRE-mCMV promoters described here as well as the more standard components of plasmid vectors (e.g., origin of replication, antibiotic resistance or another selection gene) are readily available in a variety of common vectors. DNA sequences encoding the destabilised variants of EGFP (e.g., dlEGFP, dlEYFP, dlECFP and d2EGFP, d2EYFP, d2ECFP) are available from Clontech (Clontech Laboratories Inc., Palo Alto, CA, USA). DNA sequences encoding destabilised DsRed variants are constructed by fusing to the 3' end of the DsRed encoding region, sequences encoding the degradation domains (or mutants thereof) from short-lived proteins. For example, amino acids 422-461 from mouse ornithine<br>
*<br>
decarboxylase, which contains a PEST sequence. Such sequences could potentially be derived from existing dEGFP variants.<br>
EXAMPLE11 Summary<br>
In summary the present vectors and methods are now available:<br>
• Expression vectors or parts thereof that incorporate one or more mRNA instability elements in order to provide a relatively short-lived mRNA. Compared to existing expression vectors, the vectors claimed here provide kinetics of protein expression that correlate more closely with promoter activity. For example, the time lag between decreased promoter activity and decreased mRNA and protein levels is substantially reduced.<br>
•	Expression vectors or parts thereof encoding a destabilised mRNA that in turn, encodes<br>
a destabilised protein. Compared to existing vectors, the vectors claimed here provide<br>
kinetics of protein expression that correlate more closely with promoter activity,<br>
•	Expression vectors or parts thereof in which the mRNA destabilising elements are<br>
comprised of sequences cloned from short-lived mRNAs such as c-fos, examples of<br>
short-lived mRNAs include; c-fos, c-myc, GM-CSF, DL-3, TNF-alpha, FL-2, IL-6,IL-8,<br>
Urokinase, bcl-2, SGLT1 (Na(+)-coupled glucose transporter), Cox-2 (cyclooxygenase<br>
2), 1L8, PAI-2 (plasminogen activator inhibitor type 2), betal-adrenergic receptor,<br>
GAP43 (5'UTR and 3'UTR) AU-rich elements (AREs) and/or U-rich elements,<br>
including but not limited to single, tandem or multiple or overlapping copies of the<br>
nonamer UUAUUUA(U/A)(U/A) (where U/A is either an A or a U) (Lagnado et al<br>
1994) and/or the pentamer AUUUA (Xu et al 997) and/or the tetramer AUUU (Zubiaga<br>
et al. 1995). Also included are minor modifications to or pennutations of the elements<br>
listed above. The term "tandem copies," allows for both duplication and/or non-<br>
duplication of one or more of the outer nucleotides. For example, tandem copies of the<br>
pentamer AUUUA, includes sequences such as AUUUAUUUAUUUA as well as<br>
AUUUAAUUUAAUUUA. The 3' UTR or 5' UTR regions of short-lived mRNAs often<br>
contain destabilising sequences.<br>
Expression vectors or parts thereof in which the mRNA destabilising elements were identified or validated using the vectors described herein, which provide substantially improved methods for identifying such elements.<br>
Expression vectors or parts thereof, in which the destabilised mRNA encodes a shortlived reporter protein such as a destabilised variant of EGFP or luciferase. Compared to existing reporter vectors, the vectors claimed here provide kinetics of reporter expression that correlate more closely with promoter activity. For example, the time lag between decreased promoter activity and decreased mRNA and protein levels is substantially reduced.<br>
Sets of reporter vectors or parts thereof that encode similarly destabilised mRNAs (similar to other vectors in the same set), which in turn, encode similarly (similar to other vectors in the same set) destabilised variants of EGFP or DsRed or other fluorescent markers. One or more vectors (control vectors) within each set contain a constitutive promoter (e.g., SV40, CMV, RSV, TK, TS; see Fig. 2b) or an inducible promoter (e.g., TRE-mCMV; see Fig. 2c), whereas the other vectors (test vectors) within each set contain a cloning site (e.g., MCS) in place of the promoter (e.g., see Fig. 2a). Applications of these vectors include but are not limited to the study or measurement of promoter activity. For example, a promoter element of interest can be cloned into the MCS of a test vector encoding dlEGFP and reporter expression measured relative to that of a control vector expressing dlEYFP. Also claimed is each individual vector described well as bi-directional vectors or other single vector systems that incorporate one test and one control reporter construct within the same vector (e.g., Fig. 3a and Fig. 3b). Compared to existing sets of reporter vectors, the vector sets claimed here offer the following advantages:<br>
a).        A measurement of promoter activity that is closer to real-time.<br>
b).       Decreased errors due to the closer similarity between control and test constructs,<br>
c). Decreased errors resulting from cross talk between test promoters and the control promoters. By utilising inducible promoters in the control vectors, such cross talk is minimised and/or identified and corrected for via measurement with and without induction.<br>
d).        Can be used in conjunction with the flow cytometry/LSC methods described.<br>
Reporter vectors or sets of reporter vectors or parts thereof that utilise an inducible promoter, preferably but not exclusively the tetracycline responsive element (TRE), to drive expression of a destabilised fluorescent reporter protein (preferably but not exclusively destabilised EGFP variants). Such vectors contain cloning sites in the 3'-UTR (e.g., Fig. 4a) and/or 5'-UTR and/or reporter coding region, such that regulatory elements or putative regulatory elements can be cloned into a vector expressing one color fluorescent reporter and, if required, compared to a control vector which expresses a different color reporter and does not contain the element of interest. Such vectors have applications in the study or measurement of post-transcriptional regulation, since transcription can be shut off as desired via the inducible promoter. The advantages offered by these vectors include those listed in b-d, the ability to separate post-transcriptional effects from transcriptional effects and also: a). incorporation of convenient cloning sites, not present in other vectors; and<br>
b).       the technique is more rapid than any existing method.<br>
Single vector systems that essentially link one test and one control construct and described (e.g., Fig. 4b). Both test and control reporters are driven by an inducible promoter and the cloning sites allow ligation of regulatory elements into the test construct only. In addition to the advantages of vectors outlined, the single vector<br><br>
systems eliminate problems and inaccuracies associated with co-transfection of separate test and control vectors.<br>
The use of flow cytometry or LSC to measure the levels of 2 or more fluorescent reporters expressed via the vectors outlined. In this application, the method yields thousands to hundreds of thousands of data points per sample versus one datum point for existing enzyme-based assays. Two or more reporters (control and test) as well us additional parameters (e.g., DNA content, levels of other proteins) can be measured individually in every cell. Also encompassed is the use of flow cytometry to correlate the levels of 2 or more reporters in multiple cells within the same sample and the utilisation of such data to optimise transfection protocols and/or identify problems associated with co-transfection. For example, invalid samples can be identified by the lack of a good linear relationship between test and control reporter levels. Such errors go unnoticed in current methods.<br>
Methods for utilising the post-transcriptional reporter vectors claimed. These methods are best summarised by using the example of a study aimed at determining whether a specific 3'-UTR fragment affects mRNA stability. Although this example is one of transient expression, stable transfection could also be used.<br>
(i). The 3'-UTR fragment is ligated into the 3'-UTR cloning site of the test vector and co-transfected with the control vector into a Tet-Off cell line. In the case of the single vector system, no control vector is required. 5'-UTR fragments can be tested by inserting them into vectors with a 5'-UTR cloning site.<br>
(ii). The cells are grown in the absence of doxycycline (or tetracycline) for 6-48 h to allow expression of both vectors. Alternatively, cells are grown with low doses of doxycycline (or tetracycline), for 6-48 h to block transcription and then switched to medium without doxycycline (or tetracycline) for 2-12 h to provide a brief burst of transcription.<br><br>
 (iii).    High doses of doxycycline (or tetracycline) are then applied to shut off transcription from both vectors.<br>
(iv).     The fluorescence of both reporters is measured (by flow cytometry, fluorometry or LSC) in a time course following addition of doxycycline (or tetracycline).<br>
If the cloned element confers mRNA instability, a more rapid decrease in "test" fluorescence will be seen compared to "control" fluorescence of the same cells or sample. Similar studies can be used to test an rnRNA element's response to certain stimuli or its effect in different cells or cells expressing different amounts of a specific protein, such as an RNA-binding protein. Applying the stimulus after doxycycline will determine whether pre-existing transcripts are affected by the stimulus. Inserting the element in different locations (e.g., 5'-UTR, 3'-UTR) will determine whether its function is dependent on position. Inserting a protein/polypeptide coding sequence (in frame) within the reporter protein-coding region of the vector can be used to determine the effect of that sequence on rnRNA and protein stability.<br>
RNA can be extracted from transfected cells and used to measure reporter mRNA directly.<br>
 Cell lines transiently or stably expressing one or more of the expression constructs or parts thereof claimed.<br>
 Cell lines transiently or stably expressing one or more of the expression constructs or parts thereof claimed, wherein the expression construct contains a regulatory element that serves as a marker for the activation of signal transduction pathways associated with human disease and/or response to drug treatment. Such pathways include, but are not restricted to the list in Table 1 and those indicated elsewhere in this document (e.g., CRE, SRE, API, cyclin A, B and Dl promoters).<br><br>
Transgenic mice, knock-in mice or other genetically modified mice expressing one or more of the expression constructs or parts thereof claimed.<br>
Transgenic mice, knock-in mice or other genetically modified mice expressing one or more of the expression constructs or parts thereof claimed, wherein the expression construct contains a regulatory element that serves as a marker for the activation of signal transduction pathways associated with human disease and/or response to drug treatment. Such pathways include, but are not restricted to the list in Table 1.<br>
Destabilised variants of DsRed or the mutant DsRedl-E5. These can be constructed by fusing to the C-terminus of DsRed, degradation domains (or mutants thereof) from various unstable proteins. For example, amino acids 422-461 of mouse ornithine decarboxylase, which contains a PEST sequence (Li el al 1998). Additional destabilising elements can also be added. Also contemplated are DNA constructs encoding destabilised variants of DsRed.<br>
Vectors encoding destabilised variants of DsRed outlined, including such vectors also containing the mRNA instability elements outlined.<br>
The following method for creating Tet-Off or Tet-On cell lines: The tTA or rtTA expression vector, preferably a retrovirus, adenovirus or plasmid, is stably expressed in the cell line of interest using standard techniques and expressing cells are isolated via a drug resistance marker. These cells are then transiently transfected with a TRE-vGFP construct and subjected to several rounds of cell sorting by flow cytometry. For example, good Tet-Off cells would show no fluorescence in the presence of doxycycline and are sorted as such. After a further 5-48 hr without doxycycline, green cells are sorted. Finally, the cells are grown for a week or more without doxycycline and sorted a final time to eliminate stably transfected (green) cells.<br><br>
EX AMPLE 12 Vectors Incorporating mRNA and Protein Destabilising Elements<br>
The coding region of interest (e.g., a reporter such as EGFP or luciferase) could include combined sequence of a protein destabilising element (e.g., dl mutant of MODC; Clontech, but also including other PEST sequences or other protein destabilising elements such as ubiquitination sites) and an mRNA destabilising element (e.g., AU-rich element).<br>
For example, the stop codon of luciferase and DsRed is replaced with a Hind3 site (AAGCTT) to allow the addition of the sequence:<br>
AAGCTTAGCCATGGCTTCCCGCCGGCGGTGGCGGCGCAGGATGATGGCACGCTGC CCATGTCTTGTGCCCAGGAGAGCGGGATGGACCGTCACCCTGCAGCCTGTGCTTCT GCTAGGATCAATGTGTAG which is Clontech's dl mutant of MODC that confers a 1 hr half life to EGFP. This is followed by a linker (which becomes part of the 3'UTR and then: UUAUUUAUU    GGCGG    UUAUUUAUU    CGGCG    UUAUUUAUU     GCGCG UUAUUUAUU ACTAG which contains 4 nonamers and connects to the Xbal site of the parent vector (pGL3; Promega) also in the 3'UTR but further downstream.<br>
EXAMPLE 13 Direct Ligation of PCR Products<br>
Inclusion into the MCS of a vector of two separate but nearby RE recognition sites, which, when cut with that/those RE(s), leave a 3' overhang of a single T nucleotide at both ends of the remaining vector. For example, the recognition-sequence for EclHKl is GACNNN, NNGTC (cuts between 3rd and 4th N from 5' leaving a 3' overhang of a single N at each end). Two of these sites are incorporating into the MCS, such that the short region between them is released by digestion with EclHKl, leaving a linearised vector with a 3' overhang of a single N at each end. In this example, the upstream recognition sequence should be 5'GACNNTNNGTC3' and the downstream sequence 5'GACNNANNGTC3'. After cutting with EclHKl, the large vector fragment will contain a single 3' T overhang at both ends (similar to Promega's pGEM-T Easy vector). This facilitates the direct ligation of PCR<br><br>
products that are produced with a polymerase such as Taq, that yields a 5' A overhang. This constitutes a significant improvement over standard MCSs, which do not support direct ligation of PCR products without inclusion of RE sites into PCR primers and subsequent digestion of PCR product, This is also a significant improvement over the pGEM-T Easy vector, which cannot be amplified (supplied as linear) and is useful only for subcloning (i.e., PCR products are typically ligated into pGEM-T Easy, amplified and then removed by RE digestion and subsequently cloned into the expression vector of interest). Thus, the present MCS permits direct ligation of PCR products without the need for digesting them with a RE (which is often problematic) or subcloning them into an intermediate vector,<br>
EXAMPLE 14 Destabilised Reporter Model Shows Improved Real-Time Analysis<br>
Plasmid reporter vectors were assembled in a pGL3-Basic (Promega) backbone (ampicillin resistance gene etc.) using standard cloning techniques. A tetracycline-responsive element (TRE), derived from Clontech's pTRE-d2EGFP vector was inserted into the MCS. In some constructs the luciferase-coding region was replaced with the dlEGFP- or d2EGFP- coding sequence (including Kozak sequence) as defined by Clontech. This was achieved by PCR using appropriate primers with convenient 5' flanking RE sites. In some constructs, specific examples of mRNA destabilising elements were cloned into the 3'UTR-encoding region. Typically, these sequences were prepared by synthesising and then hybridising the sense and antisense sequences. Flanking sequences provided overhanging "sticky ends" that are compatible with those generated when the 3'UTR-encoding region is cut with specific restriction enzymes. Following digestion of the vector with these enzymes and subsequent purification, the hybridised oligomers were ligated into the vector using standard techniques. PCR of genomic DNA or cDNA from an appropriate source was used as an alternative method for obtaining the larger destabilising elements such as c-myc-ARE. Very small elements (e.g., 1 or 2 nonamers) were incorporated into a reverse PCR primer that contained a 5' flanking RE site and a 3' flanking region complementary to the pre-existing 3'UTR in the vector template. Following PCR with an appropriate forward primer (complementary to the protein-coding region and overlapping an endogenous RE site), the PCR product was digested with the<br><br>
appropriate RE sites and ligated into the original vector.<br>
Nomenclature;<br>
B = Vector backbone derived from Promega's pGL3-Basic<br>
T = Tetracycline-responsive element (TRE), derived from Clontech's pTRE-d2EGFP vector<br>
and used as a promoter to drive transcription of the reporter.<br>
Gl = GFP with 1 hr half-life used as reporter i.e., dlEGFP protein encoding sequence as defined by Clontech. G2 = GFP with 2 hr half-life used as reporter i.e., d2EGFP protein encoding sequence as<br>
defined by Clontech. L = Luciferase used as reporter i.e., The Firefly luciferase encoding sequence from pGL3-Basic<br>
(Promega).<br>
R = DsRed2 used as the reporter<br>
Rl = DsRed fused at the carboxy-end to the same MODC mutant as present in dlEGFP N6 = 6 copies of the nonamer TTATTTATT (SEQ ED NO: 1) inserted into the 3'UTR-<br>
encoding region,<br>
N4 = 4 copies of the nonamer TTATTTATT inserted into the 3'UTR-encoding region. N2 = 2 copies of the nonamer TTATTTATT inserted into the 3'UTR-encoding region. Nl = 1 copy of the nonamer TTATTTATT inserted into the 3'UTR-encoding region, fos = The c-fos ARE as defined by Shyu et al (1989) inserted into the 3'UTR-encoding region<br>
i.e., 5'AAAACGTTTTATTGTGTTTTTAATTTATTTATTAAGATGGATTCTCAGATATTTAT<br>
ATTTTTATTTTATTTTTTT3'. (SEQ ED NO: 2). myc = the myc ARE defined as follows 5'ATGCATGATCAAATGCAACCTCACAACCTTGGCTGAGTCTTGAGACTGAAAGAT<br>
TTAGCCATAATGTAAACTGCCTCAAATTGGACTTTGGGCATAAAAGAACTTT<br>
TTTATGCTTACCATCTTTTTTTTTTCTTTAACAGATTTGTATTTAAGAATTGT'rr<br>
TTAAAAAATTTTAAGATTTACACAATGTTTCTCTGTAAATATTGCCATTAAAT<br>
GTAAATAACTTT3' (SEQIDNO:2n<br><br>
Method:<br>
Five micrograms of maxiprep quality DNA was transfected into -50% confluent 10cm flasks of HeLa let-Off cells (Clontech) using Fugene reagent (Roche). -Ten hours later, the flasks of cells were each split into -12 small (6cm) dishes and then incubated overnight (-12-14 hrs). At this time point (typically designated time zero or To), doxycycline was added to the culture media of most plates at a final concentration of 1 microgram per ml. Cells were trypsinised and collected at this and subsequent time points. For constructs expressing GFP, these samples were analysed by flow cytometry using standard FITC filters. Total GFP fluorescence was measured by gating out non-transfected cells (background fluorescence only) and then multiplying the mean fluorescence per cell (with background fluorescence subtracted) by the number of positive cells. RFP fluorescence (DsRed) was measured similarly using appropriate filters. Cells transfected with luciferase-encoding vectors were lysed and measured in a luminometer using Promega's Dual Luciferase Assay methods and reagents.<br>
Data are typically expressed as the percentage of reporter (fluorescence or luminescence) remaining, relative to time zero.<br>
Since the doxycycline added at time zero causes a block in transcription of the reporter, the rate of decrease in reporter levels indicates the time lag between altered transcription and altered reporter/protein levels. A prime purpose of the invention is to reduce this time lag and Figs 7, 8, 9 and 11-14 demonstrate that this is achieved.<br>
As an example of the utility of this invention, a pharmaceutical company may wish to screen for drugs that reduce transcription of a gene involved in disease. The tetracycline/doxycycline-induced block in transcription from the TRE promoter is a model of such a system. Figs 7 and 8 show that with the standard luciferase reporter vectors, even a total block in transcription (with doxycycline) is not detectable as a decrease in luciferase activity within 10 hrs. The destabilised EGFP mutants represent an improvement in that the total block in transcription is detectable as a 50% decrease in EGFP fluorescence within 11 hrs (d2EGFP; BTG2) or 7 hrs (dlEGFP; BTG1). However, when the latter reporter is combined with an mRNA destabilising element such as 4 copies of the nonamer UUAUUUAUU (BTG1N4), a 50% decrease in<br><br>
reporter levels is detectable within 3 hrs. It follows that an increase in a transcription would also be detected sooner with constructs containing the destabilising elements (Roth, 1995).<br>
Of course the action of doxycycline is not immediate so that part of the time Jag is due to the time required for this drug to induce a 100% transcriptional block. Therefore, the "Effective rate of decay" was measured by plotting data points subsequent to and relative to the time point at 4 hrs after addition of doxycycline (Fig. 9). The effective rate of decay therefore excludes the delay in drug action and is a combined effect of protein and mRNA half-lives. Fig. 9 shows the effective rate of decay with constructs containing 1, 2 or 4 nonamers. These data show that 4 nonamers arc more efficient than 2, which is more efficient than 1. Furthermore, these data show that by combining a 1 hr half-life protein (dl EGFP) with 4 nonamers, an effective rate of decay of approximately 1 hr 20 mins can be achieved. This is very close to the 1 hr half-life of the protein and demonstrates an extremely short mRNA half-life. Further reduction could be achieved by combining 2 or more different mRNA instability elements (Fig. 13). However, this is unlikely to be required for most applications. Applications that require a more moderate destabilising effect could utilise 1 or 2 nonamers, rather than 4.<br>
With the standard luciferase reporter, luminescence actually increased after the addition of doxycycline. This is most apparent when the data is expressed on a linear scale (Fig. 8) and can be explained, in part, by the delay in the action of doxycycline. However, even from 4 hrs onwards, no decay is evident, demonstrating the inadequacy of this reporter for measuring changes in transcription over time. A further problem of this vector is revealed in Fig. 10. These data relate to changes in reporter levels over time (24-34 hrs post transfection), in the absence of any treatment or drug. Reporter levels generally increase during the first 24 hrs post transfection as the plasmids enter the cells and begin to be expressed. A decrease is generally seen from about 48 hrs as the plasmids are expelled from the cells. Therefore, measurements are typically taken between 24 and 48 hrs. In the absence of drugs or treatment, the new vector (BTG1N4), containing the instability elements, shows excellent stability of reporter levels. In contrast, the luciferase vector is clearly still ramping up expression levels. Constructs with moderate stability (e.g., BTG1) showed intermediate results. Clearly reporters with longer mRNA and protein half-lives will undergo a more lengthy ramping up phase as indicated in<br><br>
Fig. 10. The more stable expression levels seen with the new construct during the critical period of 24-34 hrs will facilitate accurate measurement and represent another advantage of the invention.<br>
The rate of decrease in reporter levels can be compared between two or more constructs, which differ in their reporter mRNA sequence (e.g., in 3'UTR) but encode the same protein or different proteins with the same half-life (e.g., d2EGFP, d2EYFP). In this context, differences in the rate of decay indicate an effect of the altered mRNA sequence on mRNA stability. For example, the presence of 4 UUAUUUAUU nonamers as DNA TTATTTATT (SEQ ID NO: 1) (Figs 7-9) or the c-fos ARE (Fig. 11) (SEQ ID NO: 2), within the 3'UTR significantly increased the rate of mRNA decay. In addition to demonstrating the effectiveness of these elements, the methods and vectors used also represent a substantially improved system for detecting other cis-acting mRNA stability/instability elements and this process is also encompassed herein.<br>
As shown in Figures 12 to 14 mRNA destabilising elements work with Luciferase, GFP and DsRed not withstanding the low level of homology between these reporters. DsRed has only 23% homology with EGFP. As shown in Figure 14 myc ARE (SEQ ID NO: 21) are effective and are also effective in combination with different destabilising elements.<br>
EXAMPLE 15 mRNA Destabilising Elements<br>
RNA destabilising elements in accordance with the present invention can be derived inter alia from the 3'UTR of the following genes. In most cases, the full-length 3'UTR can be used. However, the U-rich and/or AU-rich elements can often be used alone.<br>
a)        Phosphoenolpyruvate carboxykinasc (PEPCK) mRNA destabilising elements<br>
described by Laterza OF et al. Regions within 3' half of 3'UTR referred to as JW6 and JW7 i.e., GTATGTTTAAATTATTTTTATACACTGCC CTTTCTTACCTTTCTTTACATAATTGAAATAGGTATCCTGACCA (SEQ ID<br><br>
NO: 4).<br>
b) The Bicoid gene from Drosophila melanogaster comprises an mRNA destabilising element in first 43 nt of 3 'UTR (Surdej P. et al) such an element can be used inter alia to destabilise mRNA in insect cells.<br>
c) The Human Thioredoxin reductase gene (Gasdaska, JR et al). The entire 3'UTR, Nucleotide 1933-3690 (contains 6 AU-rich elements). Segment containing 3 upstream AU repeats (nucleotide 1975-3360). There is also as Non-AU-rich destabilising element at nt 1933-2014.<br>
d) Heat Stable Antigen (HSA) Gene described in Zhou, Q et al. For example, nuclcotides 1465-1625 in the 3'UTR.<br>
e)        Granulocyte-macrophage colony stimulating factor (GM-CSF) ARE described by Chyi-Ying, A et al. AGUAAUAUUUAUAUAUUUAUAUUUUUAA AAUAUUU AUUUAUUUAUUUAUUUAA ie as DNA: AGTAAUATTTATATA TTTATATTTTTAAAATATTTATTTATTTATTTATTTAA (SEQ ID NO: 5).<br>
0         c-fos full length 3'UTR or part thereof or ARE as defined by Shyu et al 5'AA<br>
AACGTTTTATTGTGTTTTTAATTTATTTATTAAGATGGATTCTCAGATATT TATATTTTTATTTTATTTTTTT3' (SEQ ID NO: 2).<br>
Or by Peng, S et al. S'TTTTATTGTGTTTTTAATTTATTTATTAAGATGGAT TCTCAGATATTTATATTTTTATTTTATTTTTTTT3' (SEQ ID NO: 6).<br>
g)        c-iunAR£as described by Peng, S et al. 5 'UUUCGUUAACUGUGUAUGUA<br>
CAUAUAUAUAUUUUUUAAUUUGAUUAAAGCUGAUUACUGUGAAUAAA CAGCUUCAUGCCUUUGUAAGUU3' Sequence as DNA: 5'TTTCGTTAACT GTGTATGTACATATATATATTTTTTAATTTGATTAAAGCTGATTACTGTG AATAAACAGCTTCATGCCTTTGTAAGTT31 (SEQ ID NO: 7).<br><br>
or the mutant thereof which does not contain a polyadenylation (AAUAAA) signal i.e., 5'UUUCGUUAACUGUGUAUGUACAUAUAUAUAUUUUUUAAUU UGAUUAAAGCUGAUUACUGUGgAUccACAGCUUCAUGCCUUUGUAAGU U3' or as DNA 5'TTTCGTTAACTGTGTATGTACATATATATATTTTTTAA TTTGATTAAAGCTGATTACTGTGgATccACAGCTTCATGCCTTTGTAAGTT 3'(SEQIDNO:8).<br>
h)        Sequences from the following genes, that include their respective ARE components as described by Henics, T. et a/.;<br>
IFN- γARE<br>
5'UCUAUUUAUUAAUAUUUAACAUUAUUUAUAUAUGGG3' or as DNA 5TCTATTTATTAATATTTAACATTATTTATATATGGG31 (SEQ ID NO: 9).<br>
i)        IL-2ARE<br>
5'CUCUAUUUAUUUAAAUAUUUAACUUUAAUUUAUUUUUGGAUGUAU<br>
UGUUUACUAACUUUUAGUGCUUCCCACUUAAAACAUAUCAGGCUUCU<br>
AUUUAUUUAAAUAUUUAAAUUUUAUAUUUAUU3' or as DNA<br>
5'CTCTATTT<br>
ATTTAAATATTTAACTTTAATTTATTTTTGGATGTATTGTTTACTAACTTT<br>
TAGTGCTTCCCACTTAAAACATATCAGGCTTCTATTTATTTAAATATTTA<br>
AATTTTATATTTATT3' (SEQ ID NO: 10).<br>
j)        c-mvc ARE (see also SEO ED NQ:21)<br>
5' AUAAACCCUAAUUUUUUUUAUUU AAGU ACAUUUUGCUUUUA AAGUU3' or as DNA 5'ATAAACCCTAATTTTTTTTATTTAAGTACATTTTGCTrTTAAAG TT3'(SEQIDNO: II).<br>
k)        IL-10<br>
5'UAGAAUAUUUAUUACCUCUGAUACCUCAACCCCCAUUUCUAUUUAU<br><br>
UUACUGAGCUUCUCUGUGAACGAUUUAGAAAGAAGCCCAAUAUUAUA AUUUUUUUCAAUAUUUAUUAUUUUCA3' or as DNA 5'TAGAATATTTATT ACCTCTGATACCTCAACCCCCATTTCTATTTATTTACTGAGCTTCTCTGTG AACGATTTAGAAAGAAGCCCAATATTATAATTTTTTTCAATATTTATTAT TTTCA3'(SEQIDNO: 12).<br>
1)         bcl-2<br>
Sequences from the bcl-2 3'UTR that include all or part of the bcl-2 ARE as defined by Schiavone, N et al. 5'UCAGCUAUUUACUGCCAAAGGGAAAUA UCAUUUAULUUUUUACAUUAUUAAGAAAAAAGAUUUALUUAUUUAAGA CAGUCCCAUCAAAACUCCGUCUUUGGAAAUC31 (Ml3994 from nt 2371-2475) or as DNA 5'TCAGCTATTTACTGCCAAAGGGAAATATCATTTATTT TTTACATTATTAAGAAAAAAGATTTATTTATTTAAGACAGTCCCATCAAA ACTCCGTCTTTGGAAATC3' (SEQIDNO: 13).<br>
m)       TNFARE as described by Xu, N et al. 5' AUUAUUUAUUAUUUAUUUAUUA<br>
UUUAUUUAUUUA3' or as DNA 5'ATTATTTATTATTTATTTATTATTTATTT ATTTA3'(SEQIDNO: 14).<br>
n)       IL3.ARE as described by Xu, N et al. 5'UAUUUUAUUCCAUUAAGGCUAUU UAUUUAUGUAUUUAUGUAUUUAUUUAUUUAUU3' or as DNA 5'TATTTT ATTCCATTAAGGCTATTTATTTATGTATTTATGTATTTATTTATTTATT3' (SEQIDNO: 15).<br>
o)        The nonamer UUAUUUAUU as DNA TT ATTT ATT (SEQ ID NO: 1) As described by Zubiaga, A et al.<br>
p)        The nonamer UUAUUUA(U/A)(U/A) as DNA TTATTTA(T/A)(T/A) (SEQ ID NO: 3) as described by Lagnado, C et al<br>
q)        The pentamer AULUUA as described by Xu, N et al or as DNA ATTTA (SEQ ID<br><br>
NO: 16)<br>
r)        The tetramer AUUU or as DNA ATTT (SEQ ID NO: 17).<br>
AU-rich elements (AREs) in general of both class J and class II as described by Chen, C and Shyu, A.<br>
Plants have DST (downstream sequences) which act as destabilising elements. DST sequence are defined in: Newnan, T at al. A proposed consensus DST sequence is:<br>
GGAgN2-9cATAGATTaN3-8(A/C)(T/A)(A/T)TUGTA(T/C)<br>
s)        This is based on comparison of 9 different DST sequences. Bold = conserved in 9/9 genes. Capital = conserved in at least 7/9 genes N2-9 = variable length region of 2-9 nucleotides; average = 5. N3-8 = variable length region of 3-8 nucleotides; average = 6. Distance from stop codon = 19-83 nt.<br>
Further examples of DST sequences include the:<br>
Soybean .10A5 gene; 5'GGAGN5CATAGATTAN8AAATTTGTAC3' (SEQ ID NO: 18).<br>
Arabidopsis SAURAC1 gene; 5'GGAAN9CATAGATCGN8CAATGCGTAT3' (SEQTDNO: 19).<br>
DST sequences are an alternative to AU-rich elements for use in plants. Both AU-rich elements and DST sequences destabilise transcripts in plants.<br>
t)         Iron Responsive Element (IRE) Thomson, A el al. 1999.<br><br>
IREs contain consensus CAGUG in a hairpin-loop.<br>
Examples:<br>
Ferritin IRE;<br>
GUUCUUGCUUCAACAGUGUUUGAACGGAAC or as DNA GTTCTTGCTTCA<br>
ACAGTGTTTGAACGGAAC (SEQ ID NO: 20).<br>
Transfernn Receptor IRE;<br>
GAUUAUCGGGAGCAGUGUCUUCCAUAAUC or as DNA GATTATCGGGAG<br>
CAGTGTCTTCCATAATC (SEQ ID NO: 21),<br>
Iron Regulatory Proteins (IRPs; e.g., LRP1 and 2) bind IREs in an iron-dependent fashion. Binding is also modulated by various other stimuli and treatments (e.g., oxidative stress, nitric oxide, erythropoietin, thyroid hormone or phosphoryiation by PKCs.<br>
IREs can modulate both translational efficiency and mRNA stability. For example, the 5 'UTR IRE in Ferritin mRNA blocks translation only when bound to an IRP. The IREs in the 3'UTR of Transfernn receptor mRNA inhibit mRNA decay when bound by an IRP. Therefore, IREs can be inserted into 5'UTR or 3'UTR of expression vectors to provide expression that can be controlled by modulating iron levels or other stimuli.<br>
Destabilising elements can be used with Clontech's Mercury Pathway Profiling vectors and in vivo kinase assay kits. Clontech produce 3 different protein destabilising elements, all containing a PEST sequence and all derived from the MODC gene. Different mutant MODCs placed at the carboxy-end of EGFP provide protein half-lives of 1 hr, 2 hr and 4 hr. mRNA destabilising elements in accordance with the present invention can be used in conjunction with these and any other protein destabilising element (e.g., ubiquitination signals).<br>
u)        c-myc ARE may also be defined as: 5' ATGCATGATCAAATGCAACCTCAC<br>
AACCTTGGCTGAGTCTTGAGACTGAAAGATTTAGCCATAATGTAAACTG<br><br>
CCTCAAATTGGACTTTGGGCATAAAAGAACTTTTTTATGCTTACCATCTT TTTTTTTTCTTTAACAGATTTGTATTTAAGAATTGTTTTTAAAAAATTTTA AGATTTACACAATGTTTCTCTGTAAATATTGCCATTAAATGTAAATAACT TT3'(SEQIDNO:22).<br>
Another useful mRNA element can be obtained from histone mRNA. Specifically, 3'UTR sequences including a consensus stem loop structure are described by Gallic, D et al:<br>
TGA-N20-40-CCAAAGGYYYUUYUNARRRCCACCCA, where Y=pyrimidine, R=purine, N= any nucleotide or as DNA TGA-N2o-4o-CCAAAGGYYYTTYTN ARRRCCACCCA (SEQ ID NO: 23).<br>
Such sequences can increase translational efficiency. Moreover, they are capable of directing mRNA decay specifically outside of S phase. Reporter constructs containing a cell-cycle-specific promoter, together with mRNA destabilising elements are contemplated in this invention as a tool for directing cell-cycle specific expression (e.g., of a reporter). The histone 3'UTR element offers an alternative for use with an S-phase or late Gl specific promoter, since it will direct increased mRNA decay in G2 relative to S-phase, thus further restricting protein expression to S phase.<br>
Yet another use of 3'UTR elements in expression vectors is for the purpose of specifically localising the chimeric mRNA. For example, the utrophin 3'UTR is capable of directing reporter mRNA to the cytoskeletal-bound polysomes. mRNA stabilising elements are also contained in this 3'UTR (Gramolini, A, et al)<br>
EXAMPLE 16 mRNA Stabilising Elements and Expression Vectors encoding a stabilised mRNA<br>
Stabilising sequences may contain CT-rich elements and/or sequences derived from long-lived mRNAs (particularly 3'UTR regions)<br><br>
CT-rich elements may contain (C/U)CCANxCCC(U/A)PyxUC(C/U)CC as described by Holcik and Liebhaber, 1997.<br>
CT-rich elements may contain the following element<br>
CCTCCTGCCCGCTGGGCCTCCCAACGGGCCCTCCTCCCCTCCTTGC or parts thereof, including CCTCC, CCTCCTGCC or CCCTCCTCCCCTGG<br>
A 14 nt pyrimidine-rich region from the 3'UTR of human beta-globin described by Yu and Russell is also contemplated for use as a stabilising element.<br>
Examples of long-lived mRNAs from which stabilising elements may be derived include; Alpha2 globin, Alphal globin, beta globin. From human, mouse, rabbit or other species, bovine growth hormone 3'UTR.<br>
The mRNA instability elements described herein generally act in a dominant-fashion to destabilise chitneric genes. It follows, therefore that mRNA stabilising elements are often recessive-acting. For example, insertion of a c-fos ARE into the rabbit beta-globin gene, results in a destabilised transcript despite the continued presence of mRNA stability elements (Shyu, A et al. 1989). Both alpha- and beta-globin mRNAs contain stability elements that have been mapped to their respective 3'UTRs, whereas zeta-globin mRNA lacks these elements and is less stable. Replacing the zeta-globin 3'UTR with that of alpha globin mRNA nearly doubles mRNA stability (Russell, J et al. 1998). However, such elements do not stabilise all transcripts. Therefore, the requirements for generating an expression vector that expresses a stable mRNA differ, dependent on the original mRNA that is to be stabilised. To create such a vector it is generally preferable to include large segments from a stable gene such as alpha- or beta-globin. With these examples, such segments should preferably include the entire globin 3'UTR, replacing the endogenous 3'UTR. As exemplified with zeta-globin, this is sometimes sufficient. However, the further incorporation of protein-coding and/or 5'UTR sequences is often required. Generally, it is preferable to replace any endogenous AU- or U-rich regions, which may act as dominant destabilising elements (these can be identified using the techniques described herein). Such regions in the 5'UTR or 3'UTR are simply replaced with alpha- or<br><br>
beta-globin sequences from the same relative position. Instability elements from the coding region can be rendered non-functional by mutation to synonymous codons. The globin protein-coding region can be incorporated into the coding region of the gene of interest to create an N-or C-terminal fusion protein. However this is often not desirable and it is generally sufficient to localise the globin protein-coding region (and 3'UTR) into the 3'UTR of the chimeric gene. This allows expression of the desired protein from a more stable transcript, thus markedly increasing levels of the protein. When the desired protein is a reporter or is fused to a reporter or can be easily distinguished from endogenous protein, the TRE vector system described herein (see Fig. 7) greatly facilitates the testing of chimeric constructs for mRNA stability.<br>
Those skilled in the art will be aware that the invention described herein is subject to variations and modifications other than those specifically described. It is to be understood that the invention described herein includes all such variations and modifications. The invention also includes all such steps, features, compositions and compounds referred to or indicated in this specification, individually or collectively, and any and all combinations of any two or more of said steps or features.<br>
TABLE 1<br>
Signal transducers that could be used in the present invention<br>
Signal transducer <br>
(Table Removed 1)<br>
BIBLIOGRAPHY<br>
Chen, C and Shyu, A. Trends Biochem. Sci. 1995: vol 20 465-470. Chyi-Ying, A etal. MCB 1995: vol 15 (10) 5777-5788.<br>
Contag, et al. Non-invasive localization of a light-emitting conjugate in a mammal United States Patent 5,650,135. July 22, 1997.<br>
Cormack, et al. FACS-optimized mutants of the green fluorescent protein (GFP) United Stales Patent 5,804,387. September 8, 1998.<br>
Dandekar T et al Bioinformation 1998: 74(3):271-278.<br>
Darzynkiewicz. Laser-scanning cytometry: A new instrumentation with many applications. ExpCellResl999;249:l-12.<br>
Gallic, D et al. Nucleic Acids Res. 1996: vo!24 (10); 1954-1962. Gasdaska, JR et al. JBC 1999:vol 274 (36); 25379-25385.<br>
Gossen, et al. Tight control of gene expression in eucaryotic cells by tetracycline-responsive promoters. United States Patent 5,464,758. November 7, 1995.<br>
Gramolini, A, et al. JBC 2001: vol 154 (6) 1173-1183. Henics, T. et al JBC 1999: vol 274 (24) 17318-17324.<br>
Holcik and Liebhaber, 1997, PNAS vol 94 pp2410-2414.<br>
-Huet, X et al. Mol Cell Biol, 1996; vol 16 (7): 3789-3798.<br>
Ibrahim, et al. Pitfall of an internal control plasmid: Response oiRenilla luciferase (pRL-TK) plasmid to dihydrotestosterone and dexamethasone. Biotechniques 2000;29:682-684.<br>
Kamentsky et al Slide-based laser scanning cytometry. Acta Cytol 1997;41:123-43.<br>
Li, et al. Generation of destabilised green fluorescent protein as a transcription reporter. J Biol<br>
Chem 1998;273':34970-34975.<br>
Lagnado C et al, MCB, 14:1984-1995, 1994.<br>
Lalcrza, OF et al Am J Physiol Renal Physioi. 2000: vol 279(5) F866-F873,<br>
Leclerc G et al, Biotechniques 2000, 29(3) 590-595.<br>
Lee, H et al Gene Expr 1995; vol 4(3): 95-109.<br>
Li. et al. Rapidly degrading GFP-fusion proteins. United States Patent 6,130,313 October 10,<br>
2000,<br>
Liu, J ei al. JBC 2000; vol 275 (16): 11846-11851.<br>
Newnan, T et al The Plant Cell 1993: vol 5; 701-714.<br>
Peng, S et al. MCB 1996: vol 6 (4) 1490-1499.<br>
Ross, J. Microbiological Reviews. 1995: vol 59 (3):423-450.<br>
Saito, T et al. Biochern Biophys Res Commun. 1998; vol 252 (3): 775-778.<br>
Schiavone, N etal. FASEB J. 2000: vol 14 174-184.<br>
Shyu et al 1989:Genes &amp; Dev. Vol 3, pp60-72.<br>
Stein, J et al, Int J Obes Relat Metab Disord. 1996 Mar;20 Suppl 3: S84-90.<br>
Surdej, P and Jacobs-Lorena, M. MCB 1998 vol 18 (5) 2892-2900.<br>
Thomson, A et al. Int. J Biochem Cell Biol. 1999: vo! 31; 1139-1152.<br>
Tsien, et al. Modified green fluorescent proteins. United States Patent 5,625,048. April 29, 1997.<br>
Tsien, et al. Modified green fluorescent proteins United States Patent 5,777,079. July 7, 1998.<br>
Vazhappilly, R and Sucher, N. Neurosci Lett. 2002; vol 318 (3): 153-157. Xu W et al, MCB, 77(8):4611-4621, 1997. Yu and Russell MCB Sept 2001 5879-88. Zhou, Q et al. MCB 1998: vol 18 (2) 815-826. Zubiaga et al. MCB 1995;15:2219-30.<br><br><br><br><br><br>
WE CLAIM:<br>
1.	A method for assaying the activity of a first transcriptional control element in a cell, the<br>
method comprising:<br>
(a)	expressing a first polynucleotide in the cell from a first construct comprising, in operable linkage; the first transcriptional control element the coding sequence of a first polypeptide and a nucleic acid sequence encoding a RNA destabilising element that reduces the stability of a transcript corresponding to the first polynucleotide in the cell;<br>
(b)	measuring the level and/or functional activity of the first polypeptide produced from the first construct; and<br>
(c)	determining the activity of the first transcriptional control element based on the measured level and/or functional activity of the first polypeptide.<br>
wherein the nucleic acid sequence encoding the RNA destabilising element is selected from any one of SEQ ID NOS 1-23, or biologically active fragments thereof.<br>
2.	A method as claimed in claim 1, wherein the expression of the first polynucleotide is carried out in the presence and absence of a test agent such as herein described.<br>
3.	A method as claimed in claim 2, further comprising:<br>
comprising the  level  and/or  functional  activity  of the  first  polypeptide produced in the presence and absence of the test agent.<br>
4.	A method as claimed in any one of claims 1 to 3, further comprising:<br>
-	expressing a second polynucleotide from a second construct comprising, in operable linkage; a second transcriptional control element and the coding sequence of a second polypeptide that is detectably distinguishable from the first polypeptide wherein the second transcriptional control element is different from the first transcriptional control element;<br>
-	measuring the level and/or functional activity of the second polypeptide produced from the expression vector; and<br>
-	comparing the level and/or functional activity of the second polypeptide with the level and/or functional activity of the first polypeptide.<br><br>
5.	A method as claimed in claim 4, wherein the coding sequence of the second polypeptide<br>
is operably linked to a nucleic acid sequence encoding a RNA destabilising element<br>
reduces the stability of a transcript corresponding to the second polynucleotide in the cell.<br>
6.	A method as claimed in claim 4 or 5, wherein the first construct and the second construct are present on the same vector.<br>
7.	A method as claimed in claim 4 or 5, wherein the first construct and the second construct are present on separate vectors.<br>
8.	A method as claimed in claim 4 or 5, wherein the first construct and the second construct are contained within a single cell.<br>
9.	A method as claimed in claim 4 or 5, wherein the first construct and the second construct are contained within different cells.<br>
10.	A method as claimed in claim 2, wherein the first construct is contained within a first cell type and within a second cell type, wherein a difference in the level and/or functional activity of the first polypepetide in the presence of the test agent between the cell types indicates that the test agent has a higher activity in one of those cell types than in the other of those cell types.<br>
11.	A method according to any one of claims 1 to 10, wherein the nucleic acid sequence encoding the RNA destabilising element is, or is derived from, a gene selected fro c-fos, c-jun, c-myc, GM-CSF, IL-3, TNF-alpha, IL-2, IL-6, IL-8, IL-10, Urokinase, bcl-2, SGLT1 (NA(+)-coupled glucose transporter), Cox-2 (cyclooxygenase 2), IL-8, PAI-2 (plasminogen activator inhibitor type 2), betal-adrenergic receptor or GAP43.<br>
12.	A method as claimed in any one of claims 1 to 12, wherein the first polypeptide further comprises a protein destabilising element.<br><br>
13. A method as claimed in claim 13, wherein said protein destabilizing element is encoded by a sequence of nucleotides encoding a PEST sequence, a ubiquitin, or a biologically active fragment thereof.<br><br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1ERUxOUC0yMDAzLUFic3RyYWN0LSgyOC0wMS0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">1424-DELNP-2003-Abstract-(28-01-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1kZWxucC0yMDAzLWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1424-delnp-2003-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1kZWxucC0yMDAzLWFzc2lnbm1lbnQucGRm" target="_blank" style="word-wrap:break-word;">1424-delnp-2003-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1ERUxOUC0yMDAzLUNsYWltcy0oMjQtMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1424-DELNP-2003-Claims-(24-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1ERUxOUC0yMDAzLUNsYWltcy0oMjYtMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1424-DELNP-2003-Claims-(26-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1ERUxOUC0yMDAzLUNsYWltcy0oMjgtMDEtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1424-DELNP-2003-Claims-(28-01-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1kZWxucC0yMDAzLWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1424-delnp-2003-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1ERUxOUC0yMDAzLUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDgtMDktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1424-DELNP-2003-Correspondence-Others-(08-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1ERUxOUC0yMDAzLUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTAtMTItMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1424-DELNP-2003-Correspondence-Others-(10-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1ERUxOUC0yMDAzLUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTktMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1424-DELNP-2003-Correspondence-Others-(19-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1ERUxOUC0yMDAzLUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjQtMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1424-DELNP-2003-Correspondence-Others-(24-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1ERUxOUC0yMDAzLUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjYtMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1424-DELNP-2003-Correspondence-Others-(26-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1ERUxOUC0yMDAzLUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjgtMDEtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1424-DELNP-2003-Correspondence-Others-(28-01-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1kZWxucC0yMDAzLWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1424-delnp-2003-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1kZWxucC0yMDAzLWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">1424-delnp-2003-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1ERUxOUC0yMDAzLURyYXdpbmdzLSgyOC0wMS0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">1424-DELNP-2003-Drawings-(28-01-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1kZWxucC0yMDAzLWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1424-delnp-2003-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1ERUxOUC0yMDAzLUZvcm0tMS0oMjQtMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1424-DELNP-2003-Form-1-(24-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1ERUxOUC0yMDAzLUZvcm0tMS0oMjYtMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1424-DELNP-2003-Form-1-(26-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1ERUxOUC0yMDAzLUZvcm0tMS0oMjgtMDEtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1424-DELNP-2003-Form-1-(28-01-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1kZWxucC0yMDAzLWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">1424-delnp-2003-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1kZWxucC0yMDAzLWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">1424-delnp-2003-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1ERUxOUC0yMDAzLUZvcm0tMi0oMjQtMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1424-DELNP-2003-Form-2-(24-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1ERUxOUC0yMDAzLUZvcm0tMi0oMjYtMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1424-DELNP-2003-Form-2-(26-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1ERUxOUC0yMDAzLUZvcm0tMi0oMjgtMDEtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1424-DELNP-2003-Form-2-(28-01-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1ERUxOUC0yMDAzLUZvcm0tMy0oMTAtMTItMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1424-DELNP-2003-Form-3-(10-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1ERUxOUC0yMDAzLUZvcm0tMy0oMjgtMDEtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1424-DELNP-2003-Form-3-(28-01-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1kZWxucC0yMDAzLWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">1424-delnp-2003-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1ERUxOUC0yMDAzLUZvcm0tNS0oMjgtMDEtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1424-DELNP-2003-Form-5-(28-01-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1kZWxucC0yMDAzLWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">1424-delnp-2003-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1kZWxucC0yMDAzLWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">1424-delnp-2003-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1ERUxOUC0yMDAzLU90aGVycy1Eb2N1bWVudC0oMDgtMDktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1424-DELNP-2003-Others-Document-(08-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1ERUxOUC0yMDAzLU90aGVycy1Eb2N1bWVudC0oMjgtMDEtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1424-DELNP-2003-Others-Document-(28-01-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1kZWxucC0yMDAzLXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">1424-delnp-2003-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1kZWxucC0yMDAzLXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">1424-delnp-2003-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1kZWxucC0yMDAzLXBjdC0zMDEtKDA4LTA5LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">1424-delnp-2003-pct-301-(08-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1ERUxOUC0yMDAzLVBDVC0zMDQtKDA4LTA5LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">1424-DELNP-2003-PCT-304-(08-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1kZWxucC0yMDAzLXBjdC00MDEucGRm" target="_blank" style="word-wrap:break-word;">1424-delnp-2003-pct-401.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1kZWxucC0yMDAzLXBjdC00MDkucGRm" target="_blank" style="word-wrap:break-word;">1424-delnp-2003-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1kZWxucC0yMDAzLXBjdC00MTYucGRm" target="_blank" style="word-wrap:break-word;">1424-delnp-2003-pct-416.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQyNC1kZWxucC0yMDAzLXBldGl0aW9uLTEzOC5wZGY=" target="_blank" style="word-wrap:break-word;">1424-delnp-2003-petition-138.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="231925-a-peelable-heat-seal-pouch-and-method-and-apparatus-for-manufacturing-the-same.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="231927-a-pharmaceutical-composition-containing-insulin-as-the-active-agent-and-the-delivery-agent-being-the-monosodium-salt-of-4-cnab-sodium-n-4-4-chloro-2hydroxybenzoyl-amino1-butanoic-acid.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>231926</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1424/DELNP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>13-Mar-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>05-Sep-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>GENE STREAM PTY LTD.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>96 CHIPPING ROAD, CITY BEACH, WESTERN AUSTRALIA 6015, AUSTRALIA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>DALY, JOHN</td>
											<td>96 CHIPPING ROAD, CITY BEACH, WESTERN AUSTRALIA 6015, AUSTRALIA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/67</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/AU02/00351</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-03-08</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/274,770</td>
									<td>2001-03-09</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/231926-a-method-for-assaying-the-activity-of-transcriptional-control-elements by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:51:22 GMT -->
</html>
