<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/232232-method-for-screening-genomic-dna-fragments by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:02:41 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 232232:METHOD FOR SCREENING GENOMIC DNA FRAGMENTS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHOD FOR SCREENING GENOMIC DNA FRAGMENTS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention provides a method for selecting genomic DNA fragments which are useful for providing a plant with an agriculturally advantageous improvement. The method of the present invention comprises the steps of: 1) preparing genomic DNA from a plant, which is then cloned into a cloning vector to form a genomic DNA library; 2) introducing a genomic fragment from each of the genomic clones constituting the genomic DNA library separately into a plant to produce transgenic plants; 3) cultivating the transgenic plants or progeny thereof to select a plant exhibiting an agriculturally advantageous phenotypic variation; and 4) selecting the genomic DNA fragment, which was introduced in step (2) into the plant selected in step (3), as a purposed genomic DNA fragment.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
SPECIFICATION<br>
METHOD FOR SCREENING GENOMIC DNA FRAGMENTS TECHNICAL FIELD<br>
[0001]  This invention relates to a method for efficient screening of genomic DNA fragments capable of providing plants with an agriculturally advantageous phenotypic variation. BACKGROUND ART<br>
[0002]  With a view to breeding agriculturally advantageous new plant varieties, cross breeding in which two plants are crossed and progeny is selected, mutation breeding in which mutation is induced in a plant, and other methods have conventionally been practiced.  Recently, with the progress of biotechnology, genetically modified plants have been bred by introducing a useful gene and causing its function to be expressed. Breeding a new variety by introducing an individual gene<br>
In order to breed a new variety by genetic engineering, it is usually required at a first step to isolate a gene and analyze its function.  Recent years have seen a dramatic increase in molecular biological findings about plant genes and the genomic DNA sequences of many species have been determined with many partial- as well as full-length cDNA clones being isolated and determined for their sequences.  However, many of the heretofore cloned putative gene functions are simply based on the information that the nucleotide sequences of genetic coding regions or the amino acid sequences deduced therefrom of the proteins<br><br>
are similar to the sequences of already discovered enzyme genes and the like and in order to verify the function of a particular gene, one must make sure that the expression of the gene and its phenotype agree in a transformant.  As a result, considerable time and labor is required to unravel the functions of individual genes and little progress has been made in this aspect.  An attempt is being made to verify the functions of a gene by isolating a full-length cDNA clone, linking it to a suitable promoter and terminator and transforming it.  As an improved version of this attempt, a technique has been developed that comprises introducing a library of full-length cDNAs into a plant and making an exhaustive analysis of functions of the genes (WO 03/018808A).  However, in those approaches, the promoter is not what is inherently associated with the gene and introns as well as other gene expression regulating functions have been removed, so it is not expected for the genes to be expressed to exhibit the inherent functions.  As a further problem, splicing of some genes is shown to be alternative (Jordan et al. Trends in Plant Sciences 7:392-398, 2002), so the cDNA clones obtained may have lost their inherent functions.  As a matter of fact, the phenotypic variations observed in such transgenic plants do not have much utility for the purpose of breeding a new variety. [0003]  In recent years, techniques in bioinformatics are employed to deduce coding region that is translated into the protein, and promoter, intron and other regions of a gene.  Modes of gene expression are investigated by the<br><br>
microarray technology using DNA fragments.  A number of function-deficient variants have been prepared by the gene knockout technique and are used in function analysis of genes.  In addition, transformants having enhanced gene expression are prepared by activation tagging and used in function analysis of genes.  To unravel the interrelationship between proteins encoded by genes, two-hybrid system is employed. [0004]  In the deduction of gene function by bioinformatics, the finding obtained from the relation between the function and structure of a known protein and the sequence of the gene encoding it are often employed to search for the yet to be known function of a gene. However, recent studies have shown that there are many cDNAs that are not translated into proteins, or which permit transcription of a mRNA-like RNA but not produce a protein.  There are also many genes that function as low-molecular weight RNAs after transcription.  Therefore, in the bioinformatics techniques proposed to date, there are many genes on genomic DNA that present difficulty in unraveling their functions.  Therefore, deducing gene functions is not easy even if such latest techniques are fully exploited.<br>
[0005]  As noted above, analysis of gene functions is not easy even today.  And even if a gene function is specified, it is difficult for the above-mentioned methods involving the transformation of individual genes to breed a new variety that is improved in traits whose expression will be<br><br>
improved in so-called heterosis or in quantitative traits. [0006]  In order to capture a gene in a certain organism that brings about a known phenotype possessed by said organism, an attempt is being widely made that comprises constructing genomic libraries from said organism, introducing the libraries with a plasmid into a microorganism such as yeast or bacterium to prepare transformed cells, selecting a particular transformed cell on the basis of the information known for said known phenotype, for example, information such as the transcript of said gene, and employing the selected transformant to clone the desired gene (shotgun cloning) (Dairi et al. Mol Gen Genet 262:957-964, 2000).<br>
[0007]  In one application of shotgun cloning, a plant genomic library was transformed by introducing it into a plant, rather than a microorganism (Klee et al. Mol Gen Genet 210:282-287, 1987).  In this experiment, a genomic library was constructed from an Arabidopsis transformant prepared by introducing a kanamycin resistance gene from a microorganism.  Petunia leaf discs were infected with mixed strains of Agrobacterlum containing the genomic clones in order to select kanamycin resistant petunia cells, namely, petunia cells harboring the kanamycin resistance gene derived from the Arabidopsis transformant.  As a result, it was shown that the microorganism derived kanamycin resistance gene in the Arabidopsis genome could be captured after introduction into petunia by transformation. [0008]  Further disclosed in connection with Arabidopsis<br><br>
was a case in which a genomic library was constructed from a mutant showing chlorosulfuron resistance due to mutation in the acetohydroxy acid synthase (AHAS) gene and three genomic clones harboring the mutant AHAS gene were isolated and introduced into tobacco, producing chlorosulfuron resistant transformants (Olszewski et al. Nucleic Acid Res. 16:10765-10782, 1988).<br>
[0009]  These studies disclose techniques in which genomic libraries are used to transform plant cells and the gene cloning is performed.  However, they have not succeeded in capturing any unknown gene of the donor plants of the genomic libraries, nor in improving the plants by the introduction of the unknown genes.  Given those techniques, it is still difficult to breed a new variety that is improved in agriculturally useful traits, particularly in traits whose expression will be improved in so-called heterosis or in quantitative traits. Heterosis<br>
Heterosis is a phenomenon in which the F1 generation of a cross between inbred lines is superior to the parental lines.  In heterosis, various trait improvements are recognized, such as higher vigor of the entire plant, larger plant and organs, higher yield, rapid growth, greater resistance to diseases and pests, greater resistance to various environmental stresses including drought, high temperature and cold temperature, increase or decrease in a specified component, and increase or decrease in a specified enzyme activity, and many of these traits<br><br>
are extremely advantageous in agriculture.  A heterosis based breeding method that has been employed from old times in order to improve cultivated plants is F1 hybrid breeding in which different parents are crossed to create a new variety and this has made great contribution to breeding superior varieties of many crops including maize.  However, Fi hybrid breeding requires a large number of steps such as development and improvement of the breeding population, the development of inbred lines, examination of general combining ability, examination of specific combining ability, and the selection of F1 variety.  In addition, each of these steps requires a lot of time and labor.  What is more, while heterosis often produces great efficacy in the crossing of genetically distant parents, in the case where the relation between the parents is remote, crossing often does not produce fertility, thus limiting the range of species that can be crossed.<br>
[0010]  The molecular mechanism for heterosis is yet to be unraveled.  Even the latest textbook on thremmatology states as follows: "the causal factors (in heterosis) at the physiological, biochemical, and molecular levels are today almost as obscure as they were at the time of the conference on heterosis held in 1952" (Genetics and Exploitation of Heterosis in Crops, p. 173, ed. Coors and Pandey, 199 9, American Society of Agronomy, Inc. and Crop Science Society of America, Inc., Madison, WI, U.S.A.) [0011]  Interesting reports on heterosis in maize were recently made.  They are Fu and Dooner, Proc. Natl. Acad<br><br>
Sci USA 99:9573-9578, 2002 and Song and Messing, Proc Natl Acad Sci USA 100:9055-9060, 2003.  In both reports, the authors investigated intervarietal differences in nucleotide sequence noting specific loci in maize, and consequently showed that the intervarietal differences were considerably greater than in self-fertilizing crops such as rice.<br>
[0012]  These findings are interesting because they show that in cross-fertilizing crops such as maize which tend to develop heterosis, the sequences of genomic DNA have greater intervarietal differences than in self-fertilizing crops; yet, it cannot be said they have reasonably unraveled the molecular mechanism for heterosis. [0013]  Thus, no insight has yet been gained into the mechanism for heterosis at the molecular level.  However, at the level of classical genetics, it has been suggested that the following various genetic interactions are involved in heterosis. [0014]  A) Dominance effect<br>
Traits for which heterosis is observed are governed by a large number of loci in various linkage groups, and in each locus, an allele advantageous for survival and productivity is often considered to be dominant whereas a disadvantageous allele is recessive.  Since there are many loci in linkage, it is almost impossible to obtain a plant line in which advantageous alleles are homozygous for all of such loci.  However, F1 plants can possess all the advantageous alleles from the parents so that heterosis is<br><br>
induced.<br>
[0015]  B) Over-dominance effect<br>
In a large number of loci, the case where two alleles are heterozygous is sometimes more advantageous in survival and productivity than the case where the locus is homozygous, and the sum of such effects brings about heterosis.<br>
[0016]  If over-dominance effect exists in a locus having particularly great effect, one can observe heterosis due to the over-dominance effect of that single locus.  This phenomenon is called single-gene heterosis or single-locus heterosis.  Although not contributing to any particular phenotype in the original plant, this is a gene or locus that brings about a useful phenotypic variation by the interaction between genes in another plant.  Known examples of genes or loci that exhibit such property are the alcohol dehydrogenase gene in maize (Schwartz, Theor Appl Genet 43:117-120, 1973) and the purple plant locus in maize (Hollick and Chandler, Genetics 150:891-897, 1998). [0017]  C) Interaction of non-allelic genes<br>
Traits advantageous for survival and productivity are sometimes brought about in F1 hybrids as synergism between different genes.  The sum of the effects of a large number of genes exhibiting such property brings about heterosis. The interaction between non-alleles is also called epistasis.<br>
[0018]  D) Interaction between nuclear genes and cytoplasmic genes<br><br>
Through the interaction between nuclear genes and cytoplasmic genes, traits advantageous for survival and productivity are sometimes expressed in F1 hybrids. [0019]  The various types of interaction between multiple genes is considered to induce heterosis.  Stuber (Plant Breeding Reviews 12:227-251, 1994) reviews a large number of references that show examples of the involvement of those types of interaction of genes and emphasizes that heterosis is governed by a large number of genetic factors. Li and Yuan (Plant Breeding Reviews 17:15-158, 2000) also consider that heterosis is caused by the combination of the above-mentioned various effects.<br>
[0020]  Thus, heterosis is governed by a large number of genetic factors, so it has been difficult for the prior art technology to breed a new variety that is further improved in traits whose expression is known to be higher in heterosis. Quantitative traits<br>
Traits that can be improved in expression by heterosis are often "quantitative traits", and it is not easy to genetically analyze quantitative trait loci (QTL) which govern heterosis-  Nevertheless, with the recent advances in molecular biological techniques, it has become possible to perform genetic analysis of QTL using DNA markers.  As a matter of fact, there are cases for successful identification of chromosomal sites containing QTL that govern certain quantitative traits.  In addition, studies are being made to clone agriculturally useful genes by<br><br>
molecular biological techniques using genetic maps. [0021]  In some organisms, many molecular markers have been identified on chromosomes to help construct genetic maps based on the linkage analysis of the markers.  Their physical relative positions have also become clear by linking long cloned genomic DNAs.<br>
[0022]  In organisms for which genetic maps have been constructed, attempts to unravel the physical positions of genes that govern those traits and isolate such genes have been made by linkage analysis of traits that exhibit specified phenotypes and their markers, and subsequent chromosome walking.  As a matter of fact, several genes have been isolated by this technique (map-based cloning). [0023]  However, in standard QTL analysis, a QTL-containing site can only be identified in an approximate manner and only DNA fragments theoretically harboring a large number of genes can be identified as QTL-containing DNA fragments. It is not easy to identify such fragments as those capable of being cloned or as those that can be introduced into a plant by transformation.  In addition, the task of constructing a detailed genetic map, specifying a gene of interest on the basis of the map information and cloning the gene requires a considerable amount of time and labor. In fact, there are only few cases in which DNA fragments that could increase quantitative traits were cloned on the basis of QTL analysis.<br>
Constructing genomic DNA libraries and the technology of transformation with genomic fragments<br><br>
The technology of constructing libraries of plant genomic fragments is known.  Using transformation vectors that can be used to transform plants in the process is also known.  For example, vectors are known that can be used for cloning large (40-80 kb) DNA fragments and which permit gene transfer into plants (Liu et al. Proc. Natl. Acad. Sci. USA 96:6535-6540, 1999).  Experimental attempts have also been made to introduce plant genomic fragments as individual clones into higher plants.  However, no one has ever made an attempt in such a way that a large number of genomic fragments that constitute a genomic DNA library are individually introduced into plants when the functions of these fragments are unknown.<br>
[0024]  It is also known that the use of a genomic clone sometimes results in a higher gene expression than when the corresponding cDNA clone is used.  As a matter of fact, when a genomic fragment harboring a certain gene (maize phosphoenolpyruvate carboxylase) was introduced into a plant (rice), an extremely high-level expression of the foreign gene was observed (Ku et al. Nature Biotechnol. 17:76-80, 1999).  Other reports relate to experiments in which three 40-80 kb genomic clones from Arabidopsis were individually transferred back into Arabidopsis (Liu et al. Proc Natl Acad Sci US   A 96:6535-6540, 1999; Shibata and Liu Trends in Plant Sci 5:354-357, 2000).  Two of the clones were introduced into an Arabidopsis line that had lost gravitropism due to mutation at the locus contained in those clones and the recovery of the normal gravitropic<br><br>
response was confirmed. [0025]<br>
The above findings suggest that genes in organisms, in particular, genes in multi-cell organisms are controlled with regard to their expression level in a complex way by temporal and spatial distributions of the genes in the organisms and environmental conditions such as external stimuli; in other words, the importance of a particular gene is determined by the time and extent of its expression as well as by the tissue and cell in which it is expressed, and the timing of its expression.  Thus, in order to unravel gene functions including these sophisticated gene regulations, the promoter, intron, enhancer, structural gene, splicing site and all other extensive gene expression regulating factors that are contained in the genomic fragment of the particular gene must be clarified. However, this task requires a considerable amount of labor and time, making it difficult to unravel the interaction between many genetic factors Patent document<br>
1: WO 03/018808 A Non-patent document<br>
1: Jordan et al. Trends in Plant Sciences 17:392-<br>
398,2002<br>
2: Dairi et al. Mol Gen Genet 262: 957-964, 2000 3: Klee et al. Mol Gen Genet 210:282-287, 1987 4: Olszewski et al. Nucleic Acid Res. 16:10765-10782, 1988<br><br>
5: Genetics and Exploitation of Heterosis in Crops, p. 173, ed. Coors and Pandey, 1999, American Society of Agronomy, Inc. and Crop Science Society of America, Inc., Madison, WI, U.S.A. 6: Fu and Dooner, Proc. Natl Acad Sci USA 99:9573-<br>
9578, 2002 7: Song and Messing, Proc Natl Acad Sci USA 100:9055-<br>
9060, 2003<br>
8: Schwartz, Theor Appl Genet 43:117-120, 1973 9: Hollick and Chandler, Genetics 150:891-897, 1998 10: Stuber, Plant Breeding Reviews 12:227-251, 1994 11: Li and Yuan, Plant Breeding Reviews 17:15-158,<br>
2000 12: Liu et al. Proc Natl Acad Sci USA 96:6535-6540,<br>
1999<br>
13: Ku et al. Nature Biotechnol. 17:76-80, 1999 14: Shibata and Liu Trends in Plant Sci 5:354-357, 2000<br>
DISCLOSURE OF THE INVENTION<br>
PROBLEMS TO BE SOLVED BY THE INVENTION<br>
[0026]  The present invention provides a method by which a<br>
large number of genomic DNA fragments capable of providing<br>
plants with an agriculturally advantageous phenotypic<br>
variation are efficiently screened and prepared as cloned<br>
DNA fragments.<br>
[0027]  The present invention also provides a method by<br>
which genomic DNA fragments that improve the expression of<br><br>
a trait whose expression is improved by more than one genetic factor are screened and prepared with high efficiency.<br>
[0028]  The present invention also provides a method for efficiently screening and preparing a large number of genomic DNA fragments, that can improve traits that are expressed in heterosis or quantitative traits, as cloned DNA fragments.<br>
[0029]  The present invention also provides a method for efficiently screening and preparing genomic DNA fragments capable of providing plants with a potentially agriculturally advantageous phenotypic variation, wherein the method does not require the great number of steps that are unavoidable in conventional techniques such as F1 hybrid breeding, as exemplified by breeding and improving the breeding population, breeding of a inbred line, performance test on general combining ability, performance test on specific combining ability and the selection of F1 variety and each of which requires a great length of time. [0030]  The present invention also provides a method for efficiently screening and preparing genomic DNA fragments capable of providing plants with a potentially agriculturally advantageous phenotypic variation by selecting superior individuals solely on the basis of the phenotype of the recipient plant into which the genomic DNA fragments have been introduced even if there is little information available about the mechanism of trait expression or about the individual of genes expressing the<br><br>
trait.<br>
[0031]  The present invention also provides a method for efficiently screening and preparing genomic DNA fragments enabling expression of a trait similar to that of an improved trait which occurs in heterosis (hereunder referred to as "heterosis-like expression") not only in plant varieties of the same species but also in plant varieties of different species.<br>
[0032]  The present invention also provides a method for efficiently screening and preparing a large number of genomic DNA fragments enabling heterosis-like expression in a short period without requiring a great amount of time and labor.<br>
[0033]  The present invention also provides a process for producing a plant having a potentially agriculturally advantageous phenotypic variation by transforming a plant with a genomic DNA fragment capable of providing it with a potentially agriculturally advantageous phenotypic variation or a genomic DNA fragment capable of inducing heterosis-like expression, both being prepared by the method of the present invention, as well as the plant produced by that process.<br>
[0034]  The present invention also provides a method for breeding a plant having a potentially agriculturally advantageous phenotypic variation by using as a marker either all or part of genomic DNA fragment capable of providing a plant with a potentially agriculturally advantageous phenotypic variation or genomic DNA fragment<br><br>
capable of inducing heterosis-like expression, both being prepared by the method of the present invention, as well as the plant produced by that method. [Description of Terms]<br>
In the present specification, "an agriculturally advantageous phenotypic variation" is "a phenotypic variation which causes a quantitative increase or decrease of a plant or a part of a plant, or an increase or decrease in growth rate of a plant or a part of a plant, in particular, in a cultivated plant species and/or an ornamental plant species under conditions of cultivation that are normal or favorable to the plant or under conditions which are somewhat stressful to the plant".  The conditions that are stressful include the salinity of the growing site, high temperature, cold temperature, drought, diseases, pests, etc.<br>
[0035]  This is because such phenotypic variation gives rise to the trait of high yield if fruit, foliage, etc. increase under normal conditions of cultivation, while if the plant does not die under conditions where stress as from diseases and pests is imposed and its fruit, foliage, etc. increase as compared to the control plant, the phenotypic variation means resistance to the stress as from diseases and pests.  The constituents of the plant, the enzymes contained in the plant, etc, are of course encompassed by the term "part of the plant".  A decrease in the size of the entire plant or a part of the plant is often agriculturally beneficiary since dwarf plants are<br><br>
bred actively and cultivated widely.<br>
[0036]  Therefore, the concept of "a phenotypic variation which causes a quantitative increase or decrease of a plant or a part of a plant or an increase or decrease in growth rate of a plant or a part of a plant under certain conditions of cultivation" encompasses agriculturally advantageous phenotypic variations including, for example, higher vigor of the entire plant, larger plant and organs, higher yield, rapid growth, greater resistance to diseases and pests, greater resistance to various environmental stresses including drought, high temperature and cold temperature, increase or decrease in a specific component, increase or decrease in a specific enzymatic activity, and dwarfing, and so on.<br>
[0037]  In addition, the production of disease-free seedlings using tissue culture technology is also an important technology in agriculture, so it goes without saying that an improvement in the growth of cells in tissue culture is also an agriculturally advantageous phenotypic variation.<br>
[0038]  In the present invention, the term "screening or selection" shall also include the case where, after being subjected to a certain process of selection, the number of individual plants  in a given population becomes zero, namely the case where the population proves to include no plant that complies with the criterion for selection.  This is because, after such a selection, one can avoid inputting further labor and resources to the investigation of that<br><br>
population by knowing that a given population does not include any plants that comply with the criterion for selection.<br>
Means for Solving the Problems<br>
[0039]  The present invention provides a method for screening genomic DNA fragments capable of providing plants with a potentially agriculturally advantageous phenotypic variation by the following steps (1) to (4) and optionally step (5),<br>
[0040]  (1) First, genomic DNA fragments are isolated from a plant by a conventional method, subjected to partial restrictive degradation and, after size fractionation, a genomic DNA library is constructed in the usual manner. [0041]  The plant as a donor of genomic DNA fragments is not limited in any particular way and preferred examples are plants that can produce heterosis by crossing with the plant as a recipient of the genomic DNA fragments.  For example, if the recipient plant is Japonica rice, Oryza rufigopon, which is a wild rice, and Indica rice are preferred.  If the recipient plant is a variety of maize, another variety of maize and teosinte, a wild species, are preferred examples of the donor plant.  Generally, a greater extent of heterosis is observed in plants of more distant relationship.  Heretofore, distant relationship impedes crossing, so it has been impossible to use heterosis from the combination with a plant of distant relationship; on the other hand, in the method of the present invention, genomic DNA fragments from a donor plant<br><br>
that cannot be used for crossing can easily be used, so even plants of distant relationship can be used as preferred donor plants.<br>
[0042]  Various vectors can be used as cloning vectors in constructing genomic libraries.  Preferably, vectors that can be directly used in transformation of the recipient plant may be employed.  For example, in order to transform rice, tobacco, Arabidopsis, etc. pSB200 and pCLD04541 (Tao and Zhang Nucleic Acid Res 26:4901-4909, 1998) may be used, and to transform maize, pSB25UNpHm may be used. [0043]  In the case of single-locus heterosis, the DNA fragments to be cloned may be of a sizes sufficient to contain at least one gene; however, in order to include the individual genes present in the genome and the regions required for regulating their expression, sizes of at least 1 kb are preferred, with sizes of at least 10 kb being more preferred, at least 20 kb being particularly preferred, and 30-40 kb being most preferred.  Whichever the case, there is no particular upper limit on the size of the DNA fragments as long as they can be introduced into cloning vectors.  Methods of partial restrictive degradation for obtaining such DNA fragments are known.  The total number of clones that constitute a genomic DNA library, namely, the size of the library is preferably large enough to include as many genes in the plant genome as possible. Various enzymes may be used to effect partial restrictive degradation.  For performing less deviated degradation, the use of 4-base recognizing restriction enzymes, such as.<br><br>
Mbol and TaqI, is desirable.  Methods for determining appropriate conditions for degradation are known and detailed disclosures may be found in Molecular Cloning, A Laboratory Manual, Third Edition, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York, 2001. [0044]  Theoretically, the total number of clones for ensuring that a given genomic fragment is contained in a genomic DNA library of interest with a certain probability is calculated by the following formula: [0045]     N = ln(l-P)/ln(l-f)<br>
(where<br>
P is the probability that the given genomic fragment is contained in the genomic DNA library of interest;<br>
f is the ratio of "the average length of the genomic fragments contained in the clones" to "the genome size of the original plant";<br>
N is the total number of the genomic clones). [0046]  For example, in the case of a rice genomic DNA library, if the probability that a given genomic fragment is contained in the genomic DNA library is 70% and the average fragment length of the genomic DNA library is 40 kb, the formula goes like this:<br>
N = ln(l-0.7)/ln[l-(40 x 103/430 x 106)] = 1.3 x 104 Hence, 13,000 clones are required.<br>
[0047]  The above calculation is simply intended to illustrate the significance of handling a large number of clones and it is in no way intended to show that a similar<br><br>
number of genomic clones must invariably be handled. Hence, it is possible that a smaller number of clones are investigated and yet at least one of the fragments under search may be found in those clones with considerably high probability, if more than one genomic DNA fragments in the genomic DNA library can provide plants with a potentially agriculturally advantageous phenotypic variation. [0048]  If the size of genomic fragments contained in individual clones is large, a smaller number of genomic fragments need to be investigated to acquire the fragments under search.  On the other hand, if the genomic fragment size is small, subsequent handling steps such as cloning are easy to perform and the efficiency of transfer into the plant through transformation is also high.  Factors such as the genome size of the plant to be handled and the scale of the experiment are determined after comprehensive review. [0049]  When handling a plant of larger genome size, one may advantageously use a technique that involves excluding methylated DNA fragments to increase the probability of DNA fragments containing expressible genes.  Plants of larger genome size are believed to contain many unwanted DNAs that do not function as a gene and it is said that such DNAs are often methylated.  Methods for biochemically removing methylated DNAs are known and removal of methylated DNAs is also possible if cloning is performed using E. coli having an ability to remove methylated DNAs (WO 00/50587). [0050]  After constructing the genomic DNA library, part of the clones that constitute the library are incorporated<br><br>
into E. coli for cultivation.  The number of colonies that appear (in the case of a plasmid or cosmid vector) or plaques (in the case of a phage vector) is counted and on the basis of their counts, the total number of clones in the library is estimated.  In addition, DNA is prepared from part of the colonies or phages that appeared, and the size of the cloned DNA fragments is measured to estimate the average fragment length.<br>
[0051]  (2) Genomic DNA that is contained in each of the clones that constitute the library is individually introduced into a plant.<br>
[0052]  If vectors that can be directly used in transformation of plants, such as pSB200, pCLD04541 and pSB25UNpHm, are employed, the individual clones may directly be subjected to an experiment of transformation. Otherwise, all or part of the DNA fragments contained in each clone may be transferred to a transformation vector before an experiment of transformation.<br>
[0053]  The donor plant to be used in transformation may be of a different species from the plant from which the genomic DNA is derived; alternatively, it may be a different variety of the same species or the same variety of the same species.  Preferred examples of plants cover a substantially unlimited wide range including cereal plants such as rice, barley, wheat and maize, plants for producing luxuries such as coffee, cocoa, tea and tobacco, vegetables, fruits and ornamental plants such as flowers. [0054]  Transformation may be effected by any existing<br><br>
methods.  Known examples are biological transfer such as Agrobacterium-mediated method, physical transfer such as microinjection, electroporation, particle bombardment, silicon carbide method and air injection, and chemical transfer such as polyethylene glycol method.  By transformation, the genomic DNA is incorporated into the genome of the recipient plant.<br>
[0055]  According to the present invention, it has been found that it is not just one genomic DNA fragment that is obtained from a single plant genomic library and which can induce heterosis-like expression.  Hence, in order to select a greater number of genomic fragments, it is desirable that as many genomic fragments as possible are individually introduced into a plant.  In the screening method of the present invention, a certain preliminary step of selection may be included; however, in order to eliminate any deviation to occur in the chosen candidate fragments, it is desirable that the genomic DNA fragments to be introduced into the plant are not subjected to such a preliminary selection step before they are introduced into the plant.<br>
[0056]  Note that in the present invention, there is no need to have information about the genomic DNA fragments to be introduced, in particular, the phenotype with which said genomic DNA fragments are associated in the original plant. This is because it is not until the phenotype of the transgenic plant is selected that a useful genomic DNA fragment is specified.<br><br>
[0057]  Referring to each of the genomic clones to be introduced into the plant, at least part of them are amplified and/or stored.  Storage may be performed by conventional methods in the form of purified DNA or bacterium (e.g. E. coli), yeast, etc. that contain the genomic clones.<br>
[0058]  (3) The transgenic plant into which the genomic fragments have been introduced are regenerated to a complete plant and cultivated.<br>
[0059]  The regenerated transgenic plants and their progeny plants are evaluated for various agriculturally advantageous traits including, for example, the vigor of the overall plant, the size and weight of the plant and individual organs, yield, growth rate, resistance to diseases and pests, resistance to various environmental stresses including drought, high temperature and cold temperature, increase or decrease in a specified component, and increase or decrease in a specified enzyme activity. Vigor means the activity of the plant taken as a whole or its ability to grow healthily.<br>
[0060]  In the present invention, the traits to be evaluated are independent of the characteristics of the plant which is a donor of the genomic DNA fragments and the genomic DNA fragments that are introduced and there is no limitation at all as long as they are agriculturally useful traits.  Preferred are quantitative traits and traits that can be improved by heterosis, and more preferred are traits which prove agriculturally useful when the donor plant is<br><br>
regarded as the object to be bred.<br>
[0061]  After the evaluation test, those plants which are found to have exhibited a phenotypic variation compared with the plant into which none of the genomic fragments were introduced are selected.  For example, one can select plants that show higher vigor of the overall plant, larger and heavier plant and individual organs, higher yield, rapid growth, greater resistance to diseases and pests, greater resistance to various environmental stresses including drought, high temperature and cold temperature, increase or decrease in a specified component, increase or decrease in a specified enzyme activity, etc. as compared with the plant into which none of the genomic fragments were introduced.  The phenotypic variations to be selected for each trait are not limited to one direction.  Consider, for example, the trait of dwarfness; it is an important agricultural trait that serves as a goal of breeding various crops, so as for the size of the plant and individual organs, plants that have become smaller than the plant into which none of the genomic fragments were introduced can be selected.  This is also true with other traits.<br>
[0062]  Many of these traits are so-called quantitative traits and are greatly affected not only by genetic factors but also by environmental factors.  Even in the case of the plant into which genomic fragments were not introduced, values of measurement show a distribution with some dispersion due, for example, to environmental factors.<br><br>
According to the present invention, in a population of plants into which genomic DNA fragments were randomly introduced, the distribution of measured values is expected to become broader if there exist genomic DNA fragments that bring about a phenotypic variation.  By selecting the plants that present the values of measurement located at one or both ends of the distribution, one can obtain a smaller population including plants that contain genomic DNA fragments that bring about a phenotypic variation. [0063]  The thus obtained plant or each of the plants in a smaller population may be subjected to the evaluation of progeny plants and, further, to a repeated investigation of various traits, thereby evaluating the characteristics of the trait or traits that were found to have been expressed as a phenotypic variation, the mode of inheritance and the correlation with other traits; in addition, detailed evaluation can be made from the viewpoints of molecular biology, genetics, biochemistry and plant physiology. After various evaluations, the plant or plants may be agriculturally used as a novel variety.  If a plant showing greater superiority in traits is obtained from those plants, the genomic DNA fragment introduced into that plant may be chosen as more valuable genomic DNA fragment. [0064]  In that case, the introduced genomic DNA fragment can be easily analyzed and acquired by conventional cloning methods.<br>
[0065]  (4) As mentioned before, the genomic DNA fragment introduced into the selected plant is separately stored as<br><br>
a genomic clone and the required amount can be produced by amplification in E. coli using a cloning vector or by biochemical amplification methods such as PCR and LAMP. Using such an amplified genomic DNA fragment, determination of the nucleotide sequence, analysis of the contained genes, intron and other genetic elements, etc. can be performed in detail.  Since the genomic fragment can be introduced into any plants using known transformation techniques, the fragment can be utilized in variety improvement of a plant of dissimilar species from the plant of origin of the genomic DNA fragment, in improvement of a different variety of a plant of the same species, and in the breeding of the same variety of a plant of the same species.<br>
[0066]  (5) If necessary, all or part of the thus selected genomic DNA fragment may be re-introduced into a plant of the same or dissimilar species and subjected to similar evaluations, thereby effecting a step of secondary screening.  In this case, transformation may be performed using the same cloning vector as employed in step (2) or by using a different cloning vector.  If a different cloning vector is employed, the genomic DNA fragments selected in step (4) will be subcloned into the vector.  The restriction sites to be used for cloning in the cloning vector differ from one cloning vector to another, so depending on what restriction enzymes are to be used, it is sometimes appropriate to perform subcloning of only part of the genomic DNA fragment selected in step (4).  Further,<br><br>
the size of DNA fragments that can be cloned varies with the cloning vector or the method of cloning and for this reason, too, it is sometimes appropriate to perform subcloning of only part of the genomic DNA fragment.  One of the advantages that result from using only part of the DNA fragment to perform the step of secondary screening is that if the transformation using only certain part of the DNA fragment shows that the result is the same as what was obtained by using all of the DNA fragment, it becomes clear that a certain part of the genomic DNA fragment selected in step (4) is unwanted.  The transgenic plants obtained by the secondary transformation and their progeny plants are evaluated for various agriculturally advantageous traits including, for example, the vigor of the overall plant, the size and weight of the plant and individual organs, yield, growth rate, resistance to diseases and pests, resistance to various environmental stresses including drought, high temperature and cold temperature, increase or decrease in a specified component, and increase or decrease in a specified enzyme activity.<br>
[0067]  After the evaluation test, the genomic DNA fragments that gave rise to plants which exhibited a phenotypic variation compared with the plant into which none of the genomic fragments were introduced can create a preferred phenotypic variation in plants irrespective of the conditions for cultivation in the step of primary screening, the plant species, etc. and can be selected as a particularly preferred genomic DNA fragment.  Again, as in<br><br>
the step of primary screening, one can select in the steps of secondary and subsequent screenings those plants which show higher vigor of the overall plant, larger and heavier plant and individual organs, higher yield, rapid growth, greater resistance to diseases and pests, greater resistance to various environmental stresses including drought, high temperature and cold temperature, increase or decrease in a specified component, increase or decrease in a specified enzyme activity, etc. as compared with the plant into which none of the genomic fragments were introduced,<br>
[0068]  The selected plants may be subjected to the evaluation of progeny plants and, further, to a repeated investigation of various traits, thereby evaluating the characteristics of the trait or traits that were found to have been expressed as a phenotypic variation, the mode of inheritance and the correlation with other traits.  After various evaluations, the plants may be agriculturally used as a novel variety.<br>
[0069]  Secondary screening provides a genomic DNA fragment that has been verified to be capable of providing a plant with an agriculturally advantageous phenotypic variation even if it is introduced again into the plant, as well as a genomic DNA fragment that has been also verified to be capable of providing another plant with an agriculturally advantageous phenotypic variation.  Hence, more valuable genomic DNA fragments will be chosen than when only the primary screening is applied.<br><br>
[0070]  This step of selection can be repeated as many times as one likes, to yield even more valuable genomic DNA fragments.<br>
[0071]  The transcripts of the genes contained in the selected genomic DNA fragment and cDNAs derived from the fragments may be analyzed and the characteristics of the genes deduced from the nucleotide sequences of the genomic DNA fragment may be analyzed in detail and comprehensively, as a result of which one can obtain findings that are useful in deducing the genetic functions contained in the genomic DNA fragment and unraveling the mechanism of inducing heterosis.<br>
[0072]  While the foregoing description of the present invention centers on the method of screening genomic DNA fragments, it also provides the thus selected genomic DNA fragments which are capable of providing plants with an agriculturally advantageous phenotypic variation, as well as a plant conferred with the agriculturally advantageous phenotypic variation by transformation with said genomic DNA fragment.  Various methods are already known for introducing a specified DNA fragment into plant cells or a plant tissue, forming calli from the cells or tissue, cultivating the calli and causing them to regenerate into a complete plant.  See, for example, Hiei et al. Plant J. 6:271-282, 1994.  In some plants, the plant may be regenerated from transgenic cells without passing through noticeable callus formation and the present invention is also effective in that case.  The regenerated plant may be<br><br>
fixed as a variety in accordance with the method described in Maruta et al. Molecular Breeding 8:273-284, 2001. [0073]  The present invention also relates to a method of using the genomic DNA fragment of the invention as a marker in the improvement of a plant variety.  In other words, a plant having the genomic DNA fragment of the invention can be used to enhance the efficiency of improvement of a plant variety.  The use of the plant for the improvement of a plant variety may be embodied in such a way that said plant serves as a donor plant introducing the genomic DNA fragment of the invention into another plant or as a parental plant for performing variety improvement through breeding by crossing.  For instance, a plant known to have the genomic DNA fragment of the invention may be crossed with a plant variety to be improved and a genomic DNA preparation is prepared from the individual progeny plant; and subsequent steps of selecting an individual progeny plant that contains the genomic DNA fragment of the invention in said genomic DNA preparation and using a specific sequence information in the genomic DNA fragment to employ it as a marker are known procedures, as typically described in Komori et al. Euphytica 129:241-247, 2003. Speaking further of use as a marker, the whole genomic DNA fragment may be employed as a marker or, alternatively, if part of said genomic DNA fragment contains a characteristic sequence, the sequence of that part may be used as a marker. [0074]  The genomic DNA fragments according to the present<br><br>
invention include isolated, biologically active DNA and RNA that hybridize under low or high stringency conditions to the nucleotide sequences of the genomic fragments obtained by the methods disclosed in the present specification. Conditions for hybridization under high stringency may be exemplified by the following which are described in Molecular Cloning, etc.: hybridization in 0.5 M sodium phosphate (pH 7.2), 1 mM EDTA, 7% SDS and 1% BSA at 65C, followed by washing in 40 mM sodium phosphate buffer (pH 7.2), 1 mM EDTA, 5% SDS and 0.5% BSA at 65C, then washing in 40 mM sodium phosphate buffer (pH 7.2), 1 mM EDTA and 1% SDS at 65C.  Conditions for hybridization under moderate stringency may be exemplified by the following: hybridization in 0.5 M sodium phosphate (pH 7.2), 1 mM EDTA, 7% SDS and 1% BSA at 55C# followed by two washings in 40 mM sodium phosphate buffer (pH 7.2), 1 mM EDTA, 5% SDS and 0.5% BSA at 55C for 15 minutes, then two washings in 40 mM sodium phosphate buffer (pH 7.2), 1 mM EDTA and 1% SDS at 55C for 15 minutes.  Alternatively, as described in Molecular Cloning, hybridization in 30% deionized formamide, 0.6 M NaCl, 40 mM sodium phosphate (pH 7.4), 2.5 mM EDTA and 1% SDS at 42C may be followed by two washings in 2XSSC and 0.1% SDS at room temperature for 10 minutes and one-hour washing in the same buffer at 55C.  These, however, are not the sole conditions that can be adopted in the present invention.<br>
ADVANTAGEOUS EFFECT OF THE INVENTION [0075]  In the methods of the present invention for<br><br>
screening genomic DNA fragments, it is possible to search for DNA fragments involved in heterosis by a new approach that requires no previous unraveling of the functions of plant genes, while such unraveling has been necessary in conventional methods for genome analysis as exemplified by a comparison with known nucleotide sequences and production of the cDNA function.<br>
[0076]  In the methods of the present invention for screening genomic DNA fragments, DNA fragments associated with agriculturally useful traits are searched for by a new approach that does not need any information about the genomic DNA fragments to be introduced, in particular, the phenotype with which said genomic DNA fragments are associated in the original plant.<br>
[0077]  In the methods of the present invention for screening genomic DNA fragments, there is no limit on the traits to be selected and choice can be made from a wide range of agriculturally useful traits. [0078]  In the methods of the present invention for screening genomic DNA fragments, genomic DNA fragments are selected by the phenotype of the recipient plant, so the genomic DNA fragments that are chosen can be directly used in the breeding of the recipient plant. [0079]  In the methods of the present invention for screening genomic DNA fragments, genomic DNA fragments that induce an effect similar to heterosis can be obtained as DNA fragments that are cloned and which can be easily introduced into a plant by transformation, so the great<br><br>
amount of time and labor that has been required in the use of heterosis in classical methods of plant breeding can be dispensed with.<br>
[0080]  In the methods of the present invention for screening genomic DNA fragments, unlike the use of heterosis in classical methods of plant breeding, a genomic fragment from one variety is introduced into another variety, so the reproductive barriers between the parental varieties do not take effect and the combination of plants that have impeded the conventional F1 hybrid breeding becomes practicable.  Hence, the DNA fragments of the present invention can be easily introduced by transformation into various plants for use in breeding, irrespective of whether they are of the same or different species than the plant from which the DNA fragments originate; as a result, the advantages of heterosis can be utilized in a short period with high efficiency. [0081]  Furthermore, unlike QTL analysis, the methods of the present invention for screening genomic DNA fragments have no need to search for loci that are involved in agricultural traits, so genomic DNA fragments that increase or decrease quantitative traits can be selected efficiently without requiring a considerable amount of time and labor. [0082]  As a further advantage, the genomic DNA fragments of the present invention are capable of inducing heterosis-like expression, so if they are used as a marker in conventional methods of breeding by crossing, the selection efficiency on the progeny of crossing can be significantly<br><br>
increased.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
[0083]  [Fig. 1]  Fig. 1 is a genetic map of cloning vector<br>
pSB200.<br>
[Fig. 2]  Fig. 2 is a genetic map of cloning vector pSB25UNpHm.<br>
[Fig. 3]  Fig. 3 is a set of photographs showing some transgenic plant specimens selected on the basis of the external observation of panicle size, the number of grains in one panicle and the vigor of the plant, as compared with a control plant (generation: TO, number of screened lines: 5310) .<br>
[Fig. 4]  Fig, 4 is a graph showing the results of a blast resistance test of transgenic rice specimens selected after introducing genomic DNA fragments of Oryza rufipogon.<br>
[Fig. 5]  Fig. 5 is a graph showing the growth of leaves in transgenic rice specimens selected after introducing genomic DNA fragments of Oryza rufipogon extended under stress.<br>
[Fig. 6]  Fig. 6 is a graph showing the effect of the introduction of genomic DNA fragments on the growth of callus in transgenic tobacco specimens selected after introducing genomic DNA fragments of Oryza rufipogon.<br>
[Fig. 7]  Fig. 7 is a photograph showing transgenic rice specimens selected after introducing genomic DNA fragments of teosinte, as compared with a control plant.<br>
[Fig. 8]  Fig. 8 shows sites of PCR amplification on a genomic DNA fragment of Oryza rufipogon.<br><br>
[Fig. 9]  Fig. 9 is a set of photographs showing some examples of PCR amplification as performed on a genomic DNA fragment of Oryza rufipogon.<br>
[Fig. 10]  Fig. 10 is a photograph showing electrophoretic patterns of restriction enzyme-cleaved fragments from a genomic DNA fragment of Oryza rufipogon.<br>
[Fig. 11]  Fig. 11 is a photograph showing electrophoretic patterns of restriction enzyme-cleaved fragments from a transformation vector containing a genomic DNA fragment of Oryza rufipogon. EXAMPLES<br>
[0084]  In the following examples, details of experimental procedures are, unless otherwise indicated, described in Current Protocols in Molecular Biology, John Wiley &amp; Sons, Inc. (Supplements up to No. 59, July 2002, are included) or Molecular Cloning, A Laboratory Manual, Third Edition, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York, 2001.<br>
Example 1. Extraction of Genomic DNA from Oryza rufipogon and Construction of Genomic DNA Library<br>
Seeds of Oryza rufipogon, an allied species of rice, were obtained from National Institute of Agrobiological Sciences and planted for cultivation in a greenhouse.  From leaves of the plants, genomic DNA was extracted in the usual manner.  The extracted genomic DNA was subjected to partial restrictive degradation with restriction enzyme TaqI and, thereafter, fractions of 30 kb to 50 kb were prepared by sucrose density gradient centrifugation.  Using<br><br>
those fractions, cloning was made at the site of cleavage in cosmid vector pSB200 by Nsp(7524)V (hereunder sometimes designated simply as NspV) to construct a genomic DNA library,<br>
[0085]  The vector pSB200 was a cloning vector constructed from the pSBll described in Komari et al. (Plant J. 10:165-174, 1996).  To be specific, a maize ubiquitin promoter was connected before a hygromycin resistance gene and the 3' terminal signal of NOS gene.  A Nsp(7524)V cleavage site was added to the construct, which was then inserted into pSBll thereby to construct pSB200.  Using pSB200, one can construct a genomic DNA library having an average fragment length of about 40 kb.  Speaking further of pSB200, it is also a transformation vector for higher plants and can be used for gene transfer into various plants with the hygromycin resistance gene used as a selection marker. [0086]  Most of the DNA fragments cloned in the library had sizes from about 30 kb to about 50 kb and the total number of clones was about 80,000.  The E. coli strains used were DH5a and GeneHogs.<br>
Example 2. Transformation of Japonica Rice with the Clones Constituting the Oryza rufipogon Derived Genomic DNA Library<br>
The clones constituting the genomic DNA library derived from Oryza rufipogon were individually transferred into the Agrobacterium strain LBA4404(pSBl) (Komari et al. 1996).  The method used for transfer was triparental mating (Ditta et al. Proc Natl Acad Sci. U.S.A. 77:7347-7351,<br><br>
1980).  The resulting Agrobacterium carrying the clones were individually introduced into rice (variety: Yukihikari).  The method of transformation was in accordance with Hiei et al. (1994) and based on inoculation of immature embryos with Agrobacterium.  The immature embryos of the variety Yukihikari were obtained from plants cultivated in a greenhouse after sowing grains of unpolished rice marketed for food, or from their progeny plants cultivated in the greenhouse.<br>
[0087]  As a result, transgenic plants were obtained into which a total of 5310 genomic DNA fragments from the Oryza rufipogon derived genomic DNA library had been individually introduced.  For each genomic DNA fragment, 1-5 individuals of independent transformant plants were obtained. Hereinafter, the transgenic plants of the current generation will be referred to as TO generation plants and their progeny as Tl generation, T2 generation and so on, according to the general rule.<br>
[0088]  If these results are substituted into the formula set forth above, 5310 = ln(l-P)/ln[l-(40 x 103/430 x 106)] and, hence P=0.39.  Therefore, the probability that a given Oryza rufipogon derived genomic DNA fragment is contained in those 5310 genomic DNA fragments is 39%.<br>
Example 3. Evaluation of Japonica Rice Transformed with the Genomic DNA Fragments from the Oryza rufipogon Derived Genomic DNA Library and Selection of Plants that exhibited Phenotypic Variation<br>
The transgenic plants were cultivated in a greenhouse<br><br>
and the respective individuals were investigated for the vigor of the plant taken as a whole, plant length, relative growth rate, number of panicles, shoot weight, panicle weight, panicle length, number of fertile grains, and yield.  In the present specification, relative growth rate refers to the amount of daily growth per unit plant length and is determined by the formula: ((plant length at the day the investigation was completed minus plant length at the day the investigation was started)/the period of investigation in days)/plant length at the day the investigation was started.  After the investigation, the plants that were found to have exhibited a phenotypic variation in any trait as compared with the control plant were selected.  Tables 1 to 6 and Fig. 3 show the selected plant specimens and the names conferred to the genomic DNA fragments that were introduced.  In those specimens, more than one of the transformants into which the same genomic fragment had been introduced showed a similar phenotypic variation and hence were selected.  For each genomic fragment, the average of the measured values for the selected plants is listed.  The plants that were selected on the basis of the external observation of the vigor of the plant taken as a whole also had, in many cases, exhibited a variation in a certain numeric value of measurement.  Each of the specimens shown below was selected on the basis of the vigor of the plant taken as a whole and a certain numeric value of measurement. [0089]  In the following cases, the distribution of<br><br>
measured values for the control plant was fit to a normal distribution.  The control plant was rice (Yukihikari) that was transformed with GUS gene.  In accordance with the normal distribution and on the hypothesis that the introduced fragments were not effective, the probability that a line showing the values of measurement for the selected transgenic plant lines would appear was calculated.  In each case, the probability of appearance among the screened lines was extremely small and the expected value for the appearance of those selected lines was by far smaller than 1.0.  Therefore, the hypothesis for the non-effectiveness of the introduced fragments was rejected and it was statistically demonstrated that the selected lines showed a significant phenotypic variation. [0090]  [Table 1]<br>
Transgenic plant specimens selected on the basis of plant length and the overall vigor of the plant at day 14 after transplantation, as compared with the control plant (generation: TO, number of screened lines: 846)<br><br>
* Probability for the occurrence of the line of interest on the hypothesis for non-effectiveness of introduced fragments<br><br>
[0091]      [Table  2]<br>
Transgenic plant specimens selected on the basis of plant length and the overall vigor of the plant at day 21 after transplantation, as compared with the control plant (generation: TO, number of screened lines: 931)<br><br>
*	Probability for the occurrence of the line of interest on the hypothesis for<br>
non-effectiveness of introduced fragments<br>
[0092]      [Table  3]<br>
Transgenic plant specimens selected on the basis of relative growth rate and the overall vigor of the plant from day 14 to day 21 after transplantation, as compared with the control plant (generation: T0t number of screened lines: 841<br><br>
*	Probability for the occurrence of the line of interest on the hypothesis for<br>
non-effectiveness of introduced fragments<br><br>
[0093]  [Table 4]<br>
Transgenic plant specimens selected on the basis of shoot weight and the overall vigor of the plant in the stage of maturity, as compared with the control plant (generation: TO, number of screened lines: 1464)<br><br>
* Probability for the occurrence of the line of interest on the hypothesis for non-effectiveness of introduced fragments<br>
[0094]  [Table 5]<br>
Transgenic plant specimens selected on the basis of ear weight and the overall vigor of the plant at day 14 after transplantation, as compared with the control plant (generation: T0f number of screened lines: 1464<br><br>
* Probability for the occurrence of the line of interest on the hypothesis for non-effectiveness of introduced fragments<br><br>
[0095]      [Table  6]<br>
Transgenic plant specimens selected on the basis of ear length and the overall vigor of the plant, as compared with the control plant (generation: TO, number of screened lines: 1464)<br><br>
[0096]  The plants as the progeny of the transgenic plants were also cultivated and evaluated as described above.  In the Tl generation, the individuals containing the introduced genomic DNA fragments were anticipated to segregate from the individuals containing no such fragments according to Mendel's first law, so investigation was made to check for the presence or absence of an introduced fragment by polymerase chain reaction (PCR). [0097]  Tables 7 to 9 that follow show the selected plant specimens and the names conferred to the genomic DNA fragments that were introduced.  In those specimens, the plants that were the progeny derived from the same transgenic plant and in which the presence of an introduced fragment was verified by PCR all showed a similar phenotypic variation.  The plants in which no introduced<br><br>
fragment was detected by PCR all failed to show a comparable phenotypic variation.  Each of the specimens shown below was selected on the basis of the vigor of the plant taken as a whole and a certain numeric value of measurement.<br>
[0098]  In the following cases, the distribution of measured values for the control plant was fit to a normal distribution.  The control plant was yet to be transformed Yukihikari.  In accordance with the normal distribution and on the hypothesis that the introduced fragments were not effective, the probability that a line showing the values of measurement for the selected transgenic plant lines would appear was calculated.  In each case, the probability of appearance among the lines to be selected was extremely small and the expected value for the appearance of those selected lines was by far smaller than 1.0.  Therefore, the hypothesis for the non-effectiveness of the introduced fragments was rejected and it was statistically demonstrated that the selected lines showed a significant phenotypic variation.<br><br>
[0099]  [Table 7]<br>
Transgenic plant specimens selected on the basis of plant length and the overall vigor of the plant at day 14 after transplantation, as compared with the control plant (generation: T1, number of screened lines: 114)<br><br>
* Probability for the occurrence of the line of interest on the hypothesis for non-effectiveness of introduced fragments<br>
[0100]  [Table 8]<br>
Transgenic plant specimens selected on the basis of plant length and the overall vigor of the plant at day 21 after transplantation, as compared with the control plant (generation: Tl, number of screened lines: 114)<br><br>
* Probability for the occurrence of the line of interest on the hypothesis for non-effectiveness of introduced fragments<br><br>
[0101]  [Table 9]<br>
Transgenic plant specimens selected on the basis of relative growth rate and the overall vigor of the plant from day 14 to day 21 after transplantation, as compared with the control plant (generation: T1, number of screened lines: 114)<br><br>
* Probability for the occurrence of the line of interest on the hypothesis for non-effectiveness of introduced fragments<br>
[0102]  Example 4. Screening of Transgenic Plants On the Basis of Evaluation of Disease Resistance<br>
The T1 generation plants of transgenic lines that were created in Example 2 were evaluated for blast resistance in comparison with two control plants, untransformed Yukihikari and Koshihikari, and the plants that were found to have a phenotypic variation in a blast resistance related trait were selected.  In addition, the genomic DNA fragments introduced into those plants were selected as genomic DNA fragments capable of providing plants with a potentially agriculturally advantageous phenotypic variation.<br>
[0103]  Each line was sown in a closed-system greenhouse and seedlings at day 12 after sowing were transplanted for cultivation in a growth chamber, KOITOTRON (KOITO MFG. CO., LTD.)  Nine individuals of each control variety and 5-9 individuals per transgenic line were inoculated with<br><br>
Magnaporthe grlsea, a rice blast causing pathogen, and the degree of the disease was evaluated to select resistant lines.<br>
[0104]  Inoculum was prepared in the following way. A colony of Magnaporthe grisea strain TSU-01 was inoculated to an oatmeal-agar medium (Difco) containing 10 g/1 of sucrose and cultivated at 26C in the dark for 3 weeks. For conidium formation, sterile distilled water (10 ml) was added to the plate and after cutting hyphae with a sterilized painting brush, cultivation was continued at 25C for 3 days under illumination.  Eight milliliters of an LB liquid medium (Difco) diluted to one half the initial concentration with sterile distilled water was put into the plate and conidia were suspended with a sterile painting brush.  After filtering the suspension through dual-layered gauze, the concentration of conidia was adjusted to about 2 x 106 conidia/ml.  Immediately before inoculation, Silwet L-77 was added to the inoculum to give a final concentration of 0.01%.<br>
[0105]  Inoculation was performed by applying the inoculum, with a painting brush, to the topmost expanded leaves of the plant at day 19 after transplantation.  Immediately after the application, the inoculated leaves were passed through plastic tubes, each of which was plugged with cotton swab at both top and bottom openings which was then fully wetted with distilled water.  Cultivation was performed for one week, with light on for 14 hours at 25C during the day and at 20C during the night.  In the<br><br>
process, the cotton swab at the openings of each tube were wetted with distilled water once a day.  The inoculated leaves were cut off and the number and area of lesions were assessed by the following criteria for the degree of the disease: 0 (no symptom) to 3 (lesion spread to cover most of the leaves). [0106]<br>
In susceptible variety "Koshihikari", a large number of lesions formed on many leaves.  The degree of the disease was 1.5 on average.  Compared to "Koshihikari11, moderately blast resistant "Yukihikari" had formed fewer lesions and the degree of the disease was 0.8 on average. [0107]  In Example 4, check for the presence or absence of an introduced fragment was made on the basis of the hygromycin sensitivity of laminas sampled from each individual.  Individuals having no hygromycin resistance, or individuals having no introduced fragment, were removed as segregated individuals and comparison was made for the average degree of the disease on the transgenic individuals.  Many of the transgenic lines showed a comparable degree of the disease to the untransformed control "Yukihikari".  Data is shown below for 13 transgenic lines.  A014D1201, A020E0401, A023F0303 and A078C0102 showed significantly lower degrees of the disease than the control "Yukihikari" (Fig. 4).  In particular, the degree of the disease on A078C0102 was zero on average, showing the possibility that the introduction of Oryza ruflpogon genomic fragments had conferred a high level of<br><br>
blast resistance.<br>
Example 5. Screening of Transgenic Plants On the Basis of<br>
Evaluation of Drought Resistance<br>
Of the T1 generation plants of transgenic lines that were created in Example 2, 4872 lines were evaluated for the resistance to drying stress in comparison with two control plants, untransformed Yukihikari and Sue won 287 which had been reported as an drought resistant variety (and obtained from the National Institute of Agrobiological Sciences), and plants that were found to have a phenotypic variation in a trait related to the resistance towards drying stress were selected.  Each line was sowed on seedling raising boxes in a closed-system greenhouse and, after 10 days, hygromycin resistance testing was done by leaf detachment to check for the presence or absence of an introduced fragment in each individual.  Individuals that showed hygromycin resistance, namely, those individuals which were estimated to contain introduced fragments were planted on pots 12 cm in diameter and 10 cm high (POLYPOT of Tokai Kasei Co., Ltd.).  For ridging, a paddy rice seedling raising soil (INGS) was used and a total of 8 individuals were planted per pot, 6 of them were transformants and 2 were control plants.  The number of individuals tested was between 12 and 18 (on 2-3 pots) per variety or line.  Two weeks after planting (4 weeks after sowing), the supply of water was stopped to start the application of drying stress. [0108]  After one week, the transgenic plants were<br><br>
evaluated for their resistance to drought.  Evaluation was made visually on each individual by 5 scores [ranging from 0 (died) to 5 (complete recovery)].  In order to correct any scattering in data between pots, the score of each individual under test was adjusted by subtracting the score of Yukihikari on the same pot.  On the basis of the thus obtained results of evaluation, the top 10% of the transformants were selected as plants containing genomic fragments highly likely to confer drought resistance to crops.<br>
[0109]  Of the above-described transgenic plants, four potentially promising lines were measured for the length of the topmost leaf on each individual immediately before the drying treatment and one week after the treatment.  The leaf length measured one week after the treatment minus the value measured immediately before the treatment was used as the amount of leaf extension of each individual under drying stress.  After the leaf length measurement, water supply was resumed and after four days, the degree of recovery was investigated to evaluate the amount of leaf extension under drying stress.  Evaluation was made in the same manner as described above, except that Yukihikari (T2) having only GUS gene transferred to it was added as a control plant.  To check to see if the distribution of scores on each line would differ from the distribution of scores on Yukihikari, the Kolmogorov-Smirnov test was applied in analysis.  On the basis of the results thus obtained, one of the four lines of transformants under test<br><br>
was selected as a line that showed a significantly greater<br>
leaf extension than the control plants under drying stress<br>
(Fig. 5).<br>
Example 6. Evaluation of Maize Transformed with the Genomic<br>
DNA Fragments Contained in Oryza rufipogon Derived Genomic<br>
DNA Library and Selection of Plants Having Phenotypic<br>
Variation<br>
Transformation of maize was performed using Agrobacterium containing the Oryza rufipogon derived genomic fragments that were created in Example 1. Transformation procedures were in accordance with Ishida et al (2003, Plant Biotechnol. 20:57-66).  The recipient variety was inbred A188 (available from the National Institute of Agrobiological Sciences).  As in Example 1, a genomic DNA library was constructed using pSB25UNpHrn as a vector; also created was Agrobacterium containing the Oryza rufipogon derived genomic DNA fragments.  Using the library and Agrobacterium, transformation of maize was performed in the same manner as described above.  As a result, transgenic plants were obtained into which a total of 108 genomic DNA fragments contained in the Oryza rufipogon derived genomic DNA library had been individually introduced.  The vector pSB25UNpHm was the same as the pSB25 described in Ishida et al. Nature Biotech 14:745-750, 1996, except that the promoter of bar gene was replaced by a maize derived ubiquitin promoter and that three additional cleavage sites of Nsp(7524)V, I-Scel and I-Ceul were conferred.  The vector pSB25UNpHm has a comparable<br><br>
cloning ability to pSB200 and can be used for gene transfer into maize and various other plants with the bar gene used as a selection marker.<br>
[0110]  Transgenic plants of the current generation (TO plants) were cultivated in a greenhouse as in the case of rice and the effectiveness of the introduced genomic DNA fragments was assessed by investigating the plant length at days 28 and 35 after transplantation, relative growth rate (((plant length at day 35 after transplantation minus plant length at day 28 after transplantation)/7)/plant length at day 28 after transplantation), lamina length at ear bearing nodes, largest ear's weight, number of grains in largest ear, total grain weight in largest ear, and single grain weight (total grain weight in largest ear divided by the number of grains in largest ear).  In order to correct the seasonal unevenness in growth, the average was calculated for all individuals that were potted on the same day and analysis was made after normalizing the data by the formula (value of each individual - average)/average.  A maize variety (A188) transformed with GUS gene was used as a control plant.  Among the traits of the TO plants, the total grain weight in largest ear and the single grain weight had significantly greater variances than those of the control plant.  This showed that the population of the plants into which the genomic DNA fragments had been introduced had a greater spread in distribution of measured values. [0111]  In the following cases (Tables 10-13), the<br><br>
distribution of measured values for the control plant was fit to a normal distribution.  In accordance with the normal distribution and on the hypothesis that the introduced fragments were not effective, the probability that a line showing the values of measurement for the selected transgenic plant lines would appear was calculated.  In each case, the probability of appearance among the screened lines was extremely small and the expected value for the appearance of those selected lines was by far smaller than 1.0.  Of the 108 lines that were investigated for the total grain weight in the largest ear, 5 lines had expected values smaller than 1.0.  Therefore, the hypothesis for the non-effectiveness of the introduced fragments was rejected and it was statistically demonstrated that the selected lines showed a significant phenotypic variation.<br>
[0112]  The obtained transgenic maize was cultivated in a greenhouse, pollinated with the pollen of the maize variety A188 grown in a separate greenhouse, and seeds were obtained.  For the purpose of the present invention, the generation derived from those seeds is designated T1 generation.  For this generation T1, 5 to 8 individuals of each line were cultivated in a greenhouse and investigated for their traits.  In the T1 generation, the individuals containing the introduced genomic DNA fragments were anticipated to segregate from the individuals containing no such fragments according to Mendel's first law, so investigation was made to check for the presence or absence<br><br>
of an introduced fragment by polymerase chain reaction (PCR).  Using PCR, one can check for the presence or absence of an introduced fragment (for details, see Example 16).  As the result of comparing the presence or absence of an improved fragment with the values of the traits investigated, the inventors could identify lines in each of which the averages of plant length and relative growth rate for the individuals containing the introduced genomic fragments were higher than those for all individuals in that line (Table 14).<br>
[0113]  On the basis of the results thus obtained, the transgenic plants that were found to have a phenotypic variation in one or more traits and their progeny plants were selected.  In addition, the genomic DNA fragments introduced into those plants were selected as genomic DNA fragments capable of providing maize with a potentially agriculturally advantageous phenotypic variation. [0114]   [Table 10]<br>
Transgenic plant (maize) specimens selected on the basis of relative growth rate (generation: TO, number of screened lines: 108, data normalized due to data integration over more than two potting days)<br><br>
* Probability for the occurrence of the line of interest on the hypothesis for non-effectiveness of introduced fragments<br><br>
[0115]   [Table 11]<br>
Transgenic plant (maize) specimens selected on the basis of largest ears weight (generation: TO, number of screened lines: 108, data normalized due to data integration over more than two potting days)<br><br>
*	Probability for the occurrence of the line of interest on the hypothesis for<br>
non-effectiveness of introduced fragments<br>
[0116]   [Table 12]<br>
Transgenic plant (maize) specimens selected on the basis of total grain weight in largest ear (generation: TO, number of screened lines: 108, data normalized due to data integration over more than two potting days)<br><br>
*	Probability for the occurrence of the line of interest on the hypothesis for<br>
non-effectiveness of introduced fragments<br><br>
[0117]        [Table   13]<br>
Transgenic plant (maize) specimens selected on the basis of the number of grains in largest ear (generation: TO, number of screened lines: 108, data normalized due to data integration over more than two potting days)<br><br>
* Probability for the occurrence of the line of interest on the hypothesis for non-effectiveness of introduced fragments<br><br><br><br>
[0119]  Example 7. Evaluation of Tobacco Transformed with the Genomic DNA Fragments Contained in Oryza rufipogon Derived Genomic DNA Library and Selection of Plants Having Phenotypic Variation<br>
Transformation of tobacco was performed using Agrobacterium containing the Oryza rufipogon derived genomic DNA fragments that were created in Example 1. Transformation procedures were in accordance with Komari (Theor Appl Genet. 80:167-171, 1990).  The recipient variety was SRI (Kodama et al. Plant Physiol 105:601-605, 1994).<br>
[0120]  The weight of calli produced from the transformed tobacco cells was measured to investigate how the growth of calli would be influenced by the Oryza rufipogon derived genomic DNA fragments.  Tobacco leaves were bored through with a cork borer to prepare lamina disks, which were used in transformation.  The control was disks of tobacco lamina that had been transformed with vector pSB134 having only the hygromycin resistance gene and the GUS gene. Agrobacterium was inoculated with those genomic DNA fragments, cultivated on a medium for 20 days, and the weight of each lamina disk with callus formation was measured.  As Fig. 6 shows, genomic DNA fractions could be selected that showed more active callus growth than the control.<br>
[0121]  The transgenic plants and their progeny were cultivated in a greenhouse as in the case of rice and maize and the effectiveness of the introduced genomic DNA<br><br>
fragments was assessed by investigating the vigor of the plant taken as a whole, plant length, relative growth rate, number of leaves, leaf length, leaf width, leaf weight, shoot weight, yield, drought resistance, salt tolerance, and disease resistance.  The transgenic plants that were found to have a phenotypic variation in one or more of the traits listed above and their progeny were selected.  In addition, the genomic DNA fragments introduced into those plants were selected as genomic DNA fragments capable of providing tobacco with a potentially agriculturally advantageous phenotypic variation.<br>
[0122]  The thus selected 14 lines of transgenic plant were acclimatized and potted on 4-inch pots, then cultivated in a greenhouse as in the case of rice and maize, and investigated for the length of largest leaf (2 and 3 weeks after potting), natural plant height (3 weeks after potting), and culm length.  The tobacco variety SRI transformed with GUS gene was used as a control plant.  As it turned out, many of the transgenic individuals were superior to the control plant in the length of largest leaf at week 2 after potting (x2 test, P=0.00012).<br><br>
[0123]   [Table 15]<br>
Frequency distribution of lamina length of largest transgenic plant (tobacco) leaf at week 2 after potting<br><br>
[0124]  In the following cases (Tables 16-18), the distribution of measured values for the control plant was fit to a normal distribution.  In accordance with the normal distribution and on the hypothesis that the introduced fragments were not effective, the probability that an individual showing the values of measurement for the selected transgenic plant individuals would appear was calculated.  In each case, the probability of appearance among the individuals to be screened was extremely small and the expected value for the appearance of those selected individuals was by far smaller than 1,0,  Therefore, the hypothesis for the non-effectiveness of the introduced fragments was rejected and it was statistically demonstrated that the selected individuals showed a significant phenotypic variation.<br><br>
[0125]        [Table   16]<br>
Specimens of transgenic plant (tobacco) selected on the basis of the length of largest leaf at week 2 after transplantation as compared with control plant (generation: TO, No. of individuals to be screened: 172, calculation made on the basis of the upper 70% distribution of control plant) (individuals of low rooting ability would distort the distribution, so the lower 30% was excluded)<br><br>
[0126]        [Table   17]<br>
Specimens of transgenic plant (tobacco) selected on the basis of the length of largest leaf at week 3 after transplantation as compared with control plant (generation: TO, No. of individuals to be screened: 172, calculation made on the basis of the upper 70% distribution of control plant) (individuals of low rooting ability would distort the distribution, so the lower 30% was excluded)<br><br>
[0127]   [Table 18]<br>
Specimens of transgenic plant (tobacco) selected on the basis of natural plant height at week 3 after transplantation as compared with control plant (generation: TO, No. of individuals to be screened: 172, calculation made on the basis of the upper 70% distribution of control plant) (individuals of low rooting ability would distort the distribution, so the lower 30% was excluded)<br><br>
* Probability for the occurrence of the line of interest on the hypothesis for non-effectiveness of introduced fragments<br><br>
[0128]  Example 8. Evaluation of Rice Transformed with the Genomic DNA Fragments Contained in Arabidopsls Derived Genomic DNA Library and Selection of Plants Having Phenotypic Variation<br>
As in the case of Oryza rufipogon, genomic DNA was isolated from Arabidopsis {Arabldopsis thaliana), a genomic DNA library was constructed, and the genomic clones constituting the library were individually introduced into rice (Yukihikari) by transformation.  As a result, transgenic plants were obtained into which a total of 1477 genomic DNA fragments in the genomic DNA library had been introduced individually.  The ecotype of the Arabidopsis used in Example 8 was Columbia and its seeds are available from an international Arabidopsis gene resource bank (say, RIKEN Bioresource Center).  The transgenic plants and their progeny were cultivated in a greenhouse as in the case of the Oryza rufipogon derived genomic DNA library and the effectiveness of the introduced genomic DNA fragments was assessed by investigating the vigor of the plant taken as a whole, plant length, relative growth rate, number of panicles, shoot weight, panicle weight, panicle length, number of fertile grains, yield, number of leaves, leaf length, leaf width, leaf weight, drought resistance, salt tolerance, and disease resistance.  The rice variety Yukihikari transformed with GUS gene was used as a control plant.  The transgenic plants that were found to have a phenotypic variation in one or more of the traits listed above and their progeny were selected.  In addition, the<br><br>
genomic DNA fragments introduced into the selected plants were selected as Arabidopsis derived genomic DNA fragments capable of providing crops with a potentially agriculturally advantageous phenotypic variation. [0129]  In the following case (Table 19), the distribution of measured values for the control plant was fit to a normal distribution.  In accordance with the normal distribution and on the hypothesis that the introduced fragments were not effective, the probability that a line showing the values of measurement for the selected transgenic plant lines would appear was calculated.  In each case, the probability of appearance among the screened lines was extremely small and the expected value for the appearance of those selected lines was by far smaller than 1.0.  Of the 1477 lines that were investigated for the total panicle weight, 13 lines had expected values smaller than 1.0.  Therefore, the hypothesis for the non-effectiveness of the introduced fragments was rejected and it was statistically demonstrated that the selected lines showed a significant phenotypic variation.<br><br>
[0130]        [Table  19]<br>
Transgenic plant (rice) specimens selected on the basis of total panicle weight as compared with control plant (generation: TO, number of screened lines: 1477)<br><br>
* Probability for the occurrence of the line of interest on the hypothesis for non-effectiveness of introduced fragments<br>
[0131]  Example 9. Evaluation of Rice Transformed with the Genomic DNA Fragments Contained in Rhodes Grass Derived Genomic DNA Library and Selection of Plants Having Phenotypic Variation<br>
As in the case of Oryza rufipogon, genomic DNA was isolated from Rhodes grass (Chloris gayana), a genomic DNA library was constructed, and the genomic clones constituting the library were individually introduced into rice (Yukihikari) by transformation.  As a result, transgenic plants were obtained into which a total of 1450 genomic DNA fragments in the genomic DNA library had been introduced individually.  The variety of Rhodes grass used in Example 9 is commercially available in the name of Callide.  The transgenic plants and their progeny were cultivated in a greenhouse as in the case of the Oryza rufipogon derived genomic DNA library and the effectiveness of the introduced genomic DNA fragments was assessed by investigating the vigor of the plant taken as a whole.<br><br>
plant length, relative growth rate, number of panicles, shoot weight, panicle weight, panicle length, number of fertile grains, yield, number of leaves, leaf length, leaf width, leaf weight, drought resistance, salt tolerance, and disease resistance.  The rice variety Yukihikari transformed with GUS gene was used as a control plant.  The transgenic plants that were found to have a phenotypic variation in one or more of the traits listed above and their progeny were selected.  In addition, the genomic DNA fragments introduced into the selected plants were selected as Rhodes grass derived genomic DNA fragments capable of providing crops with a potentially agriculturally advantageous phenotypic variation.<br>
[0132]  In the following case (Table 20), the distribution of measured values for the control plant was fit to a normal distribution.  In accordance with the normal distribution and on the hypothesis that the introduced fragments were not effective, the probability that a line showing the values of measurement for the selected transgenic plant lines would appear was calculated.  In each case, the probability of appearance among the screened lines was extremely small and the expected value for the appearance of those selected lines was by far smaller than 1.0.  Of the 905 lines that were investigated for the total panicle weight, 23 lines had expected values smaller than 1.0.  Therefore, the hypothesis for the non-effectiveness of the introduced fragments was rejected and it was statistically demonstrated that the selected lines showed a<br><br>
significant phenotypic variation.<br>
[0133]   [Table 20]<br>
Transgenic plant (rice) specimens selected on the basis of total panicle weight as compared with control plant (generation: TO, number of screened lines: 905)<br><br>
* Probability for the occurrence of the line of interest on the hypothesis for non-effectiveness of introduced fragments<br>
[0134]  Example 10. Evaluation of Rice and Maize Transformed with the Genomic DNA Fragments Contained in Sorghum Derived Genomic DNA Library and Selection of Plants Having Phenotypic Variation<br>
As in the case of Oryza rufipogon, genomic DNA was isolated from sorghum (Sorghum bicolor), a genomic DNA library was constructed, and the genomic clones constituting the library were individually introduced into rice (Yukihikari) and maize (A188) by transformation.  As a result, transgenic plants were obtained in two groups, into which 2560 and 200 genomic DNA fragments in the genomic DNA library had been respectively introduced individually.  The variety of sorghum used in Example 10 is commercially available in the name of gold sorgho.  The transgenic plants and their progeny were cultivated in a greenhouse as<br><br>
in the case of the Oryza rufipogon derived genomic DNA library and the effectiveness of the introduced genomic DNA fragments was assessed by investigating the vigor of the plant taken as a whole, plant length, relative growth rate, number of ears, shoot weight, ear weight, ear length, number of fertile grains, yield, number of leaves, leaf length, leaf width, leaf weight, drought resistance, salt tolerance, and disease resistance.  The transgenic plants that were found to have a phenotypic variation in one or more of the traits listed above and their progeny were selected.  In addition, the genomic DNA fragments introduced into the selected plants were selected as sorghum derived genomic DNA fragments capable of providing crops with a potentially agriculturally advantageous phenotypic variation.<br>
[0135]  Shown below is a case for the total panicle weight of rice (Table 21).  In the following case, the distribution of measured values for the control plant was fit to a normal distribution.  The rice variety Yukihikari transformed with GUS gene was used as a control plant.  In accordance with the normal distribution and on the hypothesis that the introduced fragments were not effective, the probability that a line showing the values of measurement for the selected transgenic plant lines would appear was calculated.  In each case, the probability of appearance among the lines to be selected was extremely small and the expected value for the appearance of those selected lines was by far smaller than 1.0.  Of the 2504<br><br>
lines that were investigated for the total panicle weight, 43 lines had expected values smaller than 1.0.  Therefore, the hypothesis for the non-effectiveness of the introduced fragments was rejected and it was statistically demonstrated that the selected lines showed a significant phenotypic variation.<br>
[0136]   [Table 21]<br>
Transgenic plant (rice) specimens selected on the basis of total panicle weight as compared with control plant (generation: TO, number of screened lines: 2504)<br><br>
 Probability for the occurrence of the line of interest on the hypothesis for non-effectiveness of introduced fragments<br>
[0137]  After introducing sorghum fractions, transgenic maize plants of the current generation were subjected to analysis of variance on the following traits: the plant length at days 28 and 35 after transplantation, relative growth rate (((plant length at day 35 after transplantation minus plant length at day 28 after transplantation)/?)/plant length at day 28 after transplantation), lamina length at ear bearing nodes, largestear's weight, number of grains in largest ear, total grain weight in largest ear, and single grain weight (total<br><br>
grain weight in largest ear divided by the number of grains in largest ear).  The relative growth rate, number of grains in largest ear and single grain weight had significantly greater variances than those of the control plant.  In order to correct the seasonal unevenness in growth, the average was calculated for all individuals that were potted on the same day and analysis was made after normalizing the data by the formula (value of each individual - average)/average.<br>
[0138]  In the following cases (Tables 22-27), the distribution of measured values for the control plant was fit to a normal distribution.  A maize variety (A188) transformed with GUS gene was used as a control plant.  In accordance with the normal distribution and on the hypothesis that the introduced fragments were not effective, the probability that a line showing the values of measurement for the selected transgenic plant lines would appear was calculated.  In each case, the probability of appearance among the screened lines was extremely small and the expected value for the appearance of those selected lines was by far smaller than 1.0.  Of the 150 lines investigated, 7 lines had expected values smaller than 1.0 for relative growth rate and 8 lines did so for the number of grains in the largest ear.  Therefore, the hypothesis for the non-effectiveness of the introduced fragments was rejected and it was statistically demonstrated that the selected lines showed a significant phenotypic variation. [0139]  On the basis of those results, the genomic DNA<br><br>
fragments derived from sorghum were selected as genomic DNA fragments capable of providing maize with a potentially agriculturally advantageous phenotypic variation. [0140]   [Table 22]<br>
Transgenic plant (maize) specimens selected on the basis of plant length at day 28 after transplantation (generation: TO, number of screened lines: 150, data normalized due to data integration over more than two potting days)<br><br>
[0141]        [Table  23]<br>
Transgenic plant (maize) specimens selected on the basis of relative growth rate (generation: TO, number of screened lines: 150, data normalized due to data integration over more than two potting days)<br><br><br>
[0142]   [Table 24]<br>
Transgenic plant (maize) specimens selected on the basis of largest ear's weight (generation: TO, number of screened lines: 150, data normalized due to data integration over more than two potting days)<br><br>
[0143]        [Table  25]<br>
Transgenic plant (maize) specimens selected on the basis of total grain weight in largest ear (generation: T0f number of screened lines: 150, data normalized due to data integration over more than two potting days)<br><br>
[0144]   [Table 26]<br>
Transgenic plant (maize) specimens selected on the basis of the number of grains in largest ear (generation:T0f number of screened lines: 150, data normalized due to data integration over more than two potting days)<br><br><br>
[0145]   [Table 27]<br>
Transgenic plant (maize) specimens selected on the basis of lamina length (generation: TO, number of screened lines: 150, data normalized due to data integration over more than two potting days)<br><br>
[0146]  Example 11. Evaluation of Rice Transformed with the Genomic DNA Fragments Contained in Teosinte Derived Genomic DNA Library and Selection of Plants Having Phenotypic Variation<br>
As in the case of Oryza rufipogon, genomic DNA was isolated from teosinte (Zea diploperenis), a genomic DNA library was constructed, and the genomic clones constituting the library were individually introduced into rice (Yukihikari) by transformation.  As a result, transgenic plants were obtained into which a total of 1608 genomic DNA fragments in the genomic DNA library had been introduced individually.  The variety of teosinte used in Example 11 is commercially available as teosinte for pasture.  The transgenic plants and their progeny were cultivated in a greenhouse as in the case of the Oryza ruflpogon derived genomic DNA library and the effectiveness of the introduced genomic DNA fragments was assessed by investigating the vigor of the plant taken as a whole, plant length, relative growth rate, number of panicles.<br><br>
shoot weight, panicle weight, panicle length, number of fertile grains, yield, number of leaves, leaf length, leaf width, leaf weight, drought resistance, salt tolerance, and disease resistance.  The transgenic plants that were found to have a phenotypic variation in one or more of the traits listed above and their progeny were selected.  In addition, the genomic DNA fragments introduced into the selected plants were selected as teosine derived genomic DNA fragments capable of providing crops with a potentially agriculturally advantageous phenotypic variation.  The rice variety Yukihikari transformed with GUS gene was used as a control plant.  The growth of rice cultivated after treatment with teosinte genomic fragments is shown in Fig. 7.<br>
Example 12. Evaluation of Rice Transformed with the Genomic DNA Fragments Contained in Sudan Grass Derived Genomic DNA Library and Selection of Plants Having Phenotypic Variation<br>
As in the case of Oryza rufipogon, genomic DNA was isolated from Sudan grass (Sorghum Sudanese), a genomic DNA library was constructed, and the genomic clones constituting the library were individually introduced into rice (Yukihikari) by transformation.  As a result, transgenic plants were obtained into which a total of 2644 genomic DNA fragments in the genomic DNA library had been introduced individually.  The variety of Sudan grass used in Example 12 is commercially available for pasture.  The transgenic plants and their progeny were cultivated in a greenhouse as in the case of the Oryza rufipogon derived<br><br>
genomic DNA library and the effectiveness of the introduced genomic DNA fragments was assessed by investigating the vigor of the plant taken as a whole, plant length, relative growth rate, number of panicles, shoot weight, panicle weight, panicle length, number of fertile grains, yield, number of leaves, leaf length, leaf width, leaf weight, drought resistance, salt tolerance, and disease resistance. The transgenic plants that were found to have a phenotypic variation in one or more of the traits listed above and their progeny were selected.  In addition, the genomic DNA fragments introduced into the selected plants were selected as Sudan grass derived genomic DNA fragments capable of providing crops with a potentially agriculturally advantageous phenotypic variation.<br>
[0147]  In the following case (Table 28), the distribution of measured values for the control plant was fit to a normal distribution.  The rice variety Yukihikari transformed with GUS gene was used as a control plant.  In accordance with the normal distribution and on the hypothesis that the introduced fragments were not effective, the probability that a line showing the values of measurement for the selected transgenic plant lines would appear was calculated.  In each case, the probability of appearance among the screened lines was extremely small and the expected value for the appearance of those selected lines was by far smaller than 1.0.  Of the 2 644 lines that were investigated for the total panicle weight, 21 lines had expected values smaller than 1.0.  Therefore, the<br><br>
hypothesis for the non-effectiveness of the introduced fragments was rejected and it was statistically demonstrated that the selected lines showed a significant phenotypic variation. [0148]   [Table 28]<br>
Transgenic plant (rice) specimens selected on the basis of total panicle weight as compared with control plant (generation: TO, number of screened lines: 2644)<br><br>
* Probability for the occurrence of the line of interest on the hypothesis for non-effectiveness of introduced fragments<br>
[0149]  Example 13. Evaluation of Rice Transformed with the Genomic DNA Fragments Contained in Millet Derived Genomic DNA Library and Selection of Plants Having Phenotypic Variation<br>
As in the case of Oryza rufipogon, genomic DNA was isolated from millet {Seteria italica), a genomic DNA library was constructed, and the genomic clones constituting the library were individually introduced into rice (Yukihikari) by transformation.  As a result, transgenic plants were obtained into which a total of 2952 genomic DNA fragments in the genomic DNA library had been introduced individually.  The variety of millet used in<br><br>
Example 13 is extremely early Italian millet R which is commercially available for pasture.  The transgenic plants and their progeny were cultivated in a greenhouse as in the case of the Oryza rufipogon derived genomic DNA library and the effectiveness of the introduced genomic DNA fragments was assessed by investigating the vigor of the plant taken as a whole, plant length, relative growth rate, number of panicles, shoot weight, panicle weight, panicle length, number of fertile grains, yield, number of leaves, leaf length, leaf width, leaf weight, drought resistance, and disease resistance.  The transgenic plants that were found to have a phenotypic variation in one or more of the traits listed above and their progeny were selected.  In addition, the genomic DNA fragments introduced into the selected plants were selected as millet derived genomic DNA fragments capable of providing crops with a potentially agriculturally advantageous phenotypic variation. [0150]  In the following case (Table 29), the distribution of measured values for the control plant was fit to a normal distribution.  The rice variety Yukihikari transformed with GUS gene was used as a control plant.  In accordance with the normal distribution and on the hypothesis that the introduced fragments were not effective, the probability that a line showing the values of measurement for the screened transgenic plant lines would appear was calculated.  In each case, the probability of appearance of the selected lines was extremely small and the expected value for the appearance of those selected<br><br>
lines was by far smaller than 1.0.  Of the 1126 lines that were investigated for the total panicle weight, 15 lines had expected values smaller than 1.0.  Therefore, the hypothesis for the non-effectiveness of the introduced fragments was rejected and it was statistically demonstrated that the selected lines showed a significant phenotypic variation. [0151]   [Table 29]<br>
Transgenic plant (rice) specimens selected on the basis of total panicle weight as compared with control plant (generation: TO, number of screened lines: 1126)<br><br>
[0152]  Example 14. Evaluation of Plants Transformed with the Genomic DNA Fragments Contained in Guinea Grass Derived Genomic DNA Library and Selection of Plants Having Phenotypic Variation<br>
As in the case of Oryza rufipogon, genomic DNA was isolated from Guinea grass (Panlcum maximum), a genomic DNA library was constructed, and the genomic clones constituting the library were individually introduced into<br><br>
rice, maize and tobacco by transformation.  The variety of Guinea grass used in Example 14 is colored Guinea grass commercially available for pasture.  The transgenic plants and their progeny were cultivated in a greenhouse as in the case of the Oryza rufipogon derived genomic DNA library and the effectiveness of the introduced genomic DNA fragments was assessed by investigating the vigor of the plant taken as a whole, plant length, relative growth rate, number of panicles, shoot weight, panicle weight, panicle length, number of fertile grains, yield, number of leaves, leaf length, leaf width, leaf weight, drought resistance, salt tolerance, and disease resistance.  The transgenic plants that were found to have a phenotypic variation in one or more of the traits listed above and their progeny were selected.  In addition, the genomic DNA fragments introduced into the selected plants were selected as Guinea grass derived genomic DNA fragments capable of providing rice, maize or tobacco with a potentially agriculturally advantageous phenotypic variation.<br>
Example 15  Production of Genomic DNA Fragments by PCR Using Selected Genomic DNA Fragments as Templates<br>
The selected genomic DNA fragments, AS4(A011D07), AS8(A014E08), AS19(A010B03), AS20(A011C02), AS22(A014D12), AS27(A012D12), AS28(A015C06) and AS30(A016D02), were investigated for their terminal nucleotide sequences. Primers were designed for the mapped sequences and vector sequences and, using plasmid DNA isolated from E. coli as a template, PCR was conducted to verify the presence of both<br><br>
terminal ends of the T-DNA region (as indicated by PCR1 and PCR3 in Fig. 8).<br>
[0153]  The vector region primers used in PCR1 and PCR3 respectively had the following sequences:<br>
5'-CTGAAGGCGGGAAACGACAATCTG-3'; and<br>
5'-AACTGCACTTCAAACAAGTGTGAC-3'.<br>
[0154]  The genomic DNA fragment specific primers used in PCR1 had the following sequences: AS4 : 5' - GATTCCGACCTCTACACGAACAAC - 3' AS8 : 5' - AGAAACCCTAGCCGTCACTTCCCT - 3' AS19 : 5' - TCAAGTCATTTCACAAAGTCGGAC - 3' AS20 : 5' - GCTTAGAGGTGAAAATGGTAACGG - 3' AS22 : 5' - TTCTGTCCTTGTTCGATTTGTCAG - 3' AS27 : 5' - CCGGATTCACCGTGGTACGAAAGG - 3' AS28 : 5' - TTCCAATTACCAGACACTAAAGCG - 3' AS30 : 5' - TGGCACCAGACTTGCCCTCCAATG - 3'<br>
[0155]  The genomic DNA fragment specific primers used in<br>
PCR3 had the following sequences:<br>
AS4 : 5' - GTACGGCCTGGGTCACTCACTGTC - 3'<br>
AS8 : 5' - TCATCATCCTGTTATCTAGACTCC - 3'<br>
AS19 : 5' - TACTTATTCCGTGAGTCGGAAGCG - 3'<br>
AS20 : 5' - TCCAGTGTTATGATGTTTGGGCTG - 3'<br>
AS22 : 5' - AACTCATCTTTAATCCCAGTTTGC - 3'<br>
AS27 : 5' - TAACGCCATAAACAAGTGTCACTC - 3'<br>
AS28 : 5' - GAACTGTGAAACTGCGAATGGCTC - 3'<br>
AS30 : 5' - AAATCCACACGACTCTCGGCAACG - 3'<br><br>
[0156]  The genomic DNA fragments AS4, AS8 and AS22 were also subjected to PCR to verify the presence of central portions of fragment (as indicated by PCR2 in Fig. 8).  The primers used in PCR2 had the following sequences: AS4: S'-TGGGCTCCAGCAGAAACGAACCCT-S1 and<br>
5'-CTTATATTTAGGAACGGAGTGAGT-3' AS8: S'-AAGCGAAGGCACCCCTTCACAT-3' and<br>
5'-ACGAGGAGCCCGACAAGGAGAC-3' AS22: 5'-TGAAATACCACTCATGAACTTCCG-3' and 5'-ATTATCTGTTGTGTCCGAAATGTG-3'.<br>
In each case of PCR, Takara ExTaq (TAKARA) or Takara LA Taq (TAKARA) was used and the cycle of thermal denaturation (94C x 30 sec), annealing (58C x 30 sec) and extension (72C x 30 sec) was repeated 30 or 35 times.  The PCR products were analyzed by agarose gel electrophoresis. [0157]  The results of PCR analysis are shown in Fig. 9. In each of PCR1, PCR2 and PCR3, PCR products of the desired sizes were observed when plasmid DNA for the respective genomic DNA fragments were used as templates.  However, no such PCR products were observed when pSB200 was used as a template.  These results show that the rufipogon fragments introduced into pSB200 can be produced by PCR (Fig. 9). Example 16 Use of the Selected Fragments as Markers The genomic DNA assessed and transferred into rice to cause a phenotypic variation in Example 3 was further transferred into maize for transformation in Example 6.  The transgenic maize was cultivated in a greenhouse and pollinated with pollens of the maize variety A188 grown in a separate<br><br>
greenhouse.  It is anticipated that the resulting progeny seeds will segregate into two types of individuals, one having Oryza rufipogon genomic DNA fragments and the other having not.  Hence, using their T-DNA border sequences as a marker, the introduced genomic DNA fragments were amplified by PCR to check for the presence of Oryza rufipogon derived genomic DNA fragments.  As a result, the plant individuals having the marker were found to be suitable for variety breeding and, hence, applicable in further steps of breeding whereas the plant individuals having no such marker were found to be unsuitable for variety improvement. [0158]  Table 30 shows the relation between the presence or absence of introduced genomic DNA fragments and the measured values of traits.<br><br><br><br>
[0160]  Example 17  E. coli Based Method of Producing Selected Genomic DNA Fragments<br>
E. coli having a cloning vector (pSB200) containing transformed genomic DNA fragments were individually stored at -80C in glycerol stocks.  From those stocks, E. coli were picked up and cultured on an LA(Sp50) plate at 28C for 3 days until a single colony grew.  An LB(Sp50) liquid medium (2 ml) was inoculated with the grown single colony and shaken overnight at 28C; thereafter, the liquid culture was transferred into a 1.5-ml microtube and centrifuged at 15000 rpm for 2 minutes to collect cells. The supernatant was discarded and plasmid was isolated from the cell pellet in the usual manner (alkali method); plasmid DNA was dissolved in 40 l of TE to get the genomic DNA fragments cloned in the cloning vector.<br>
Example 18 Method of Producing DNA Fragments Including the Step of Restrictive Degradation of Prepared Genomic DNA Fragments<br>
Using the plasmid DNA prepared in Example 17 (clone designations: AS88, AS90, AS95-AS102, AS104-AS106), reaction solutions were prepared according to the following recipe.<br>
[0161]  The respective plasmid DNA clones contained the following genomic DNA fragments (A018D06, A047C01, A082B03, A082B06, A083A01, A083A02, A084H04, A084H05, A088A12, A091E11, A049B03, A080C09 and A088C09).<br>
Plasmid DNA	4 l<br>
10x M buffer (Takara)	2l<br><br><br>
After reaction at 37C for 1 hour, the reaction solution was mixed with 4 l of 6x dye and 6 l of the mixture was subjected to electrophoresis on 0.7% agarose gel (100 V, 1 hr).  After the end of electrophoresis, the gel was stained with Ethidium Bromide to yield restriction enzyme cleaved fragments from the respective genomic clones.  An example of this result is shown in Fig. 10. Example 19  Incorporating E. coli Produced Genomic DNA Fragments into Transformation Vector (pSB200)<br>
Using the plasmid DNA isolated and purified from E. coli (genomic DNA fragment G001A03), BP reaction was performed (25C overnight) in accordance with the protocol attached to a GATEWAY Technology (Invitrogen) kit and, after treatment with Proteinase K, ethanol was added to precipitate the plasmid DNA.  After centrifugal (15000 rpm) treatment, the pellet was washed with 70% ethanol and redissolved in 10 l of TE.  A portion (2 l) of the solution was used to introduce the plasmid into E. coli DB3.1 by electroporation and the bacteria was plated on LA(Sp50Cm30) and cultivated at 28C for 3 days.  The growing single colony was cultivated on 2 ml of LA(Sp50Cm30) and the plasmid DNA was isolated in the usual manner (alkali method) and fragment analysis was performed<br><br>
using HindIII and SacI to select the desired plasmid.  By BP reaction, the aaBl-HPT-aaB2 fragment is replaced by the aaRl-ccdB-Cm-aaR2 fragment and the vector size changes from 9.8 kb to 10.4 kb; this change was used as a marker for the selection of the recombinant plasmid (G001A03DEST) (lane 2 in Fig. 11) .<br>
[0162]  Using the thus selected plasmid G001A03DEST, LR reaction was performed under the same conditions as the BP reaction and G001A03bar was selected; G001A03bar was a clone substituted with a drug selection marker gene.  By LR reaction, the aaRl-ccdB-Cm-aaR2 fragment is replaced by the aaBlbar-aaB2 fragment and the vector size changes from 10.4 kb to 9.3 kb; this change was used as a marker for the selection of G001A03bar (lane 3 in Fig. 11).  As a result, the selected fragment of interest could successfully be incorporated into a vector capable of plant transformation. Example 20.  Analysis of the Selected Genomic DNA Clones<br>
The selected genomic DNA fragments were investigated for the nucleotide sequences of 280-500 bases at both terminals.  The results are indicated by the sequence numbers associated with the respective genomic DNA fragments shown in Tables 1-9 and in Fig. 3.  The sequences of PCR primer pairs that can be used to detect those fragments by PCR are shown in the following Table 31. [0163]   [0165]<br><br>
Table 31<br>
Examples of the selected genomic DNA fragments derived from Oryza rufipogon's genome and the PCR primer pairs that can detect them<br><br><br><br><br><br>
[0166]  Example 21.  Introducing the Selected Genomic DNA Fragments into Plants<br><br>
The genomic DNA fragments selected in Examples 3-14 were introduced into rice, maize and tobacco by the method described in Example 2, 6 or 7.  The obtained transgenic plants and their progeny were assessed as in Examples 3-14. Further selected were those transgenic plants which got a phenotypic variation in one or more of the traits assessed, and progeny plants thereof.  The genomic DNA fragments introduced into the thus selected plants were screened as genomic DNA fragments capable of providing plants with a potentially agriculturally advantageous phenotypic variation.<br>
[0167]  Selected as the result of secondary screening were the genomic DNA fragments that were verified to be capable of providing the same plants with a potentially agriculturally advantageous phenotypic variation upon retransfer, as well as the genomic DNA fragments that were verified to be capable of providing other plants with an agriculturally advantageous phenotypic variation.  This means a successful selection of genomic DNA fragments having higher value than when only primary screening was applied.<br>
[0168]  Examples of the genomic DNA fragments thus selected are shown in Table 32.<br><br><br><br><br><br>
 CLAIMS<br>
[1]  A method for screening genomic DNA fragments capable of providing plants with an agriculturally advantageous phenotypic variation, comprising the steps of:<br>
1)	preparing genomic DNA from a plant, which is then cloned into a cloning vector to form a genomic DNA library; introducing the genomic fragment from each of the<br>
2)	genomic clones constituting the genomic DNA library<br>
3)	separately into a plant to produce transgenic plants;<br>
4)	cultivating the transgenic plants or progeny<br>
5)	thereof to select a plant exhibiting an agriculturally<br>
6)	advantageous phenotypic variation;<br>
7)	selecting the genomic DNA fragment, which was<br>
8)	introduced in step (2) into the plant selected in step<br>
9)	(3), as a purposed genomic DNA fragment; and optionally introducing the genomic DNA fragment<br>
10)	selected in step (4) or a part thereof into a plant to repeat steps (3) and (4), and selecting a genomic DNA<br>
11)	fragment which produces a plant exhibiting an<br>
12)	agriculturally advantageous phenotypic variation as a<br>
13)	purposed genomic DNA fragment in each repetition.<br>
[2]  A method according to claim 1, wherein the size of the selected genomic DNA fragment is 1 kb or greater provided that the DNA fragment can be introduced into the cloning vector. [3]  A method for screening according to claim 1 or 2,<br><br>
wherein step (2) comprises the sub-steps of: introducing the genomic fragment into the genome of a cell or tissue of the plant; regenerating a complete plant from the plant cell; and cultivating the regenerated plant. [4]  A screening method according to claim 3, wherein the introduction of the genomic DNA fragment into a plant cell or tissue is conducted by a method selected from the group consisting of biological introduction methods, physical introduction methods and chemical introduction methods. [5]  A screening method according to any one of claims 1 to 4, wherein the agriculturally advantageous phenotypic mutation in a plant gives rise to an increase or decrease of the size or the weight of at least a part of the plant or of at least a constituent thereof, an increase of growth rate or an excellent resistance against diseases or pests, under normal cultivation conditions, as compared with a case where the plant does not have the phenotypic variation.<br>
[6]  A screening method according to any one of claims 1 to 4, wherein the agriculturally advantageous phenotypic variation in a plant gives rise to an increase or decrease of the size or the weight of at least a part of the plant or of at least a constituent thereof, an increase of growth rate or an excellent resistance against diseases or pests, under conditions which are more stressful for the plant than normal conditions, as compared with a case where the plant does not have the phenotypic variation. [7]  A screening method according to claims 5 or 6,<br><br>
wherein the plant transformed in step (2) is of the same<br>
species as that of the plant which supplied the genomic<br>
DNA in step (1).<br>
[8]  A screening method according to claim 5 or 6, wherein<br>
the plant transformed in step (2) is of a different<br>
species from that of the plant which supplied the genomic<br>
DNA in step (1).<br>
[9]  A screening method according to claim 7 or 8, wherein<br>
the optional introduction of the genomic DNA fragment in<br>
step (5) is made into a plant of the same species as that<br>
of the plant which was transformed in step (2).<br>
[10] A screening method according to claim 7 or 8, wherein<br>
the optional introduction of the genomic DNA fragment in<br>
step (5) is made into a plant of a different species from<br>
that of the plant which was transformed in step (2).<br>
[11] A screening method according to claim 9 or 10,<br>
wherein the plant subjected to the optional introduction of<br>
the genomic DNA fragment in step (5) is cultivated under<br>
the same conditions as in the plant transformed in step<br>
(2).<br>
[12] A screening method according to claim 9 or 10,<br>
wherein the plant subjected to the optional introduction of<br>
the genomic DNA fragment in step (5) is cultivated under<br>
different conditions from those in the case of the plant<br>
transformed in step (2).<br>
[13] A method for producing a genomic DNA fragment capable<br>
of bringing about an agriculturally advantageous<br>
phenotypic variation in plants comprising the steps of:<br><br>
culturing E.coli cells containing a cloning vector carrying a genomic DNA fragment selected by the method according to any one of claims 1 to 12; and preparing the cloning vector amplified in the E. coli cells along with the genomic DNA fragment.<br>
[14] A method for producing a genomic DNA fragment wherein the genomic DNA fragment selected by the method according to any one of claims 1 to 12 is used as a template and the amplification of the fragment is conducted by a biochemical amplification method.<br>
[15] A method for producing a DNA fragment wherein the genomic DNA fragment obtained by the method of claims 13 or 14 is digested with restriction enzyme(s). [16] A DNA fragment produced by the method according to any one of claims 13 to 15.<br>
[17] A method for producing a plant having an agriculturally advantageous phenotypic variation comprising the step of introducing a genomic DNA fragment capable of bringing about an agriculturally advantageous phenotypic variation in plants, wherein the genomic DNA fragment is produced by a method comprising the steps of: culturing E. coli cells containing a cloning vector carrying the genomic DNA fragment, and preparing the cloning vector amplified in the E. coli cells along with the genomic DNA fragment.<br>
[18] A method for producing a plant having an agriculturally advantageous phenotypic variation according to claim 17 wherein the step of introducing a genomic DNA<br><br>
fragment capable of bringing about an agriculturally advantageous phenotypic variation in plants comprises the steps of: introducing the genomic fragment into a plant cell or tissue; regenerating a complete plant from the plant cell; and cultivating the regenerated plant. [19] A method for producing a plant according to claim 18 wherein the introduction of the genomic DNA fragment into a plant cell or tissue is conducted by a method selected from the group consisting of biological introduction methods, physical introduction methods and chemical introduction methods.<br>
[20] A method for producing a plant according to any one of claims 17 to 19, wherein the genomic DNA fragment capable of bringing about an agriculturally advantageous phenotypic variation is introduced in a plant of the same species as that of the plant from which the genomic DNA fragment was derived.<br>
[21] A method for producing a plant according to any one of claims 17 to 19, wherein genomic DNA fragment capable of introducing an agriculturally beneficial phenotypic variation is introduced in a plant of a different species from the plant from which the genomic DNA fragment was derived.<br>
[22] A plant produced by a method according to any one of claims 17 to 21.<br>
[23] A method for analyzing a genomic DNA fragment capable of bringing about an agriculturally advantageous phenotypic variation comprising the steps of: culturing<br><br>
E. coli cells containing a cloning vector carrying a genomic DNA fragment selected by the method according to any one of claims 1 to 12; and preparing the cloning vectors amplified in the E. coli cells along with the genomic DNA fragment, and reading the nucleotide sequence of the plant genomic DNA fragment in the cloning vector. [24] A method for analyzing a DNA fragment comprising the step of restricting the genomic DNA fragment selected according to any one of claims 1 to 12. [25] A method for analyzing a DNA fragment wherein the genomic DNA fragment selected by the method according to any one of claims 1 to 12 is used as a template and the amplification is conducted by a biochemical amplification method.<br>
[26] A method according to claim 24 or 25 wherein the analysis comprises the step of reading the nucleotide sequence of the restriction product of the genomic DNA fragment or the biochemically amplified product. [27] A method of using a genomic DNA fragment selected by the method according to any one of claims 1 to 12 as a marker to be used in the improvement of a plant variety. [28] A method according to claim 27 wherein a genomic DNA fragment selected in accordance with any one of claims 1 to 12 is used as a marker, wherein presence of the marker in the genomic DNA from a plant indicates that the plant is useful for improving a plant variety whereas a plant not having the marker is not useful for improving a plant variety.<br><br>
[29] A method according to claim 28 wherein a genomic DNA preparation is prepared from a progeny plant which is obtained by crossing a plant known to contain a genomic DNA fragment selected according to the method of any one of claims 1 to 12 with a plant of a variety to be improved, and if a progeny plant contains the same genomic DNA fragment, said progeny plant is useful for a further step of improvement of the plant variety.<br>
[30] A method according to claim 29 wherein the plant known to contain a genomic DNA fragment selected according to the method of any one of claims 1 to 12 is a plant as claimed in claim 22.<br>
Dated this 24 day of May 2006<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgyNC1jaGVucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1824-chenp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgyNC1jaGVucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1824-chenp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgyNC1jaGVucC0yMDA2LWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1824-chenp-2006-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgyNC1jaGVucC0yMDA2LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">1824-chenp-2006-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgyNC1jaGVucC0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1824-chenp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgyNC1jaGVucC0yMDA2LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">1824-chenp-2006-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgyNC1jaGVucC0yMDA2LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">1824-chenp-2006-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgyNC1jaGVucC0yMDA2LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">1824-chenp-2006-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgyNC1jaGVucC0yMDA2LW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1824-chenp-2006-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgyNC1jaGVucC0yMDA2LXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1824-chenp-2006-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="232231-1-d-glucopyranosyl-3-2-thienylmethyl-benzene-derivatives.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="232233-information-input-and-output-method-with-dot-pattern.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>232232</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1824/CHENP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>16-Mar-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>24-May-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>JAPAN TOBACCO, INC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>2-1, Toranomon 2-chome, Minato-ku, Tokyo 105-8422,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>KUBO, Tomoaki</td>
											<td>c/o Japan Tobacco Inc., Plant Innovation Center, 700, Higashibara, Toyoda-cho, Iwata-gun, Shizuoka, 438-0802,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>KOMARI, Toshihiko</td>
											<td>c/o Japan Tobacco Inc., Plant Innovation Center, 700, Higashibara, Toyoda-cho, Iwata-gun, Shizuoka, 438-0802,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>USAMI, Satoru</td>
											<td>c/o Japan Tobacco Inc., Plant Innovation Center, 700, Higashibara, Toyoda-cho, Iwata-gun, Shizuoka, 438-0802,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>TAKAKURA, Yoshimitsu</td>
											<td>c/o Japan Tobacco Inc., Plant Innovation Center, 700, Higashibara, Toyoda-cho, Iwata-gun, Shizuoka, 438-0802,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>HIEI, Yukoh</td>
											<td>c/o Japan Tobacco Inc., Plant Innovation Center, 700, Higashibara, Toyoda-cho, Iwata-gun, Shizuoka, 438-0802,</td>
										</tr>
										<tr>
											<td>6</td>
											<td>ISHIDA, Yuji</td>
											<td>c/o Japan Tobacco Inc., Plant Innovation Center, 700, Higashibara, Toyoda-cho, Iwata-gun, Shizuoka, 438-0802,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/09</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/JP2004/015743</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-10-22</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>2003-364682</td>
									<td>2003-10-24</td>
								    <td>Japan</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/232232-method-for-screening-genomic-dna-fragments by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:02:42 GMT -->
</html>
