<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/232256-method-for-detecting-low-levels-of-a-fusion-protein by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:03:29 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 232256:METHOD FOR DETECTING LOW LEVELS OF A FUSION PROTEIN</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHOD FOR DETECTING LOW LEVELS OF A FUSION PROTEIN</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention relates to the detection of among others tumor-specific fusion proteins. Provided is a method for detecting a fusion protein in a sample, said fusion protein comprising an ammo-terminal fragment and a carboxy-terminal fragment which are each corresponding to a native protein, said method comprising contacting said sample with at least one binding molecule specifically reactive with a part of the native protein that is not present in the fusion protein, under conditions that allow for the formation of a complex between said at least one binding molecule and said native protein, removing said complex from said sample to deplete said sample of said native protein but not of said fusion protein; and detecting said fusion protein in said sample using at least one antibody probe directed against said fusion protein. Also provided is a diagnostic kit for carrying out a method of the invention.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Title: Method for detecting low levels of a fusion protein.<br>
The invention relates to the detection of, among others, tumor-specific fusion proteins. More specifically, the invention relates to techniques that indicate the presence of chromosomal translocations by detecting the presence of a fusion protein in a biological sample.<br>
In the diagnosis of various types of cancer, such as leukemias, lymphomas and solid tumours, chromosomal aberrations are frequently used for classification into prognostically relevant subgroups. Many of these chromosomal aberrations result in fusion genes, i.e. aberrantly coupled genes coupled via the upstream part of one gene to the downstream part of the other gene, or vice versa. Fusion genes can be transcribed into fusion gene transcripts and translated into fusion proteins. Generally, fusion proteins play an important role in the oncogenetic process. For example, approximately 35% of adult patients with acute lymphoblastic leukemia (ALL) and approximately 95% of patients with chronic myeloid leukemia (CML) have a specific chromosomal defect in their leukemic cells, i.e. a translocation between chromosomess 9 and 22 that creates the Philadelphia (Ph) chromosome. This translocation occurs at the site in the genome of a protein tyrosine kinase named ABL, creating the abnormal BCR-ABL fusion protein, a gene product of the in-frame fusion of the ABL gene with another gene called BCR. Generally, fusion proteins play an important role in the oncogenetic process. For example, the kinase activity of ABL in the BCR-ABL fusion protein is activated and deregulated, driving the uncontrolled cell growth observed in ALL and in CML.<br>
When ALL is diagnosed in a patient, the total number of leukemia cells typically lies within the range of 10n to 1012. Cytostatic or cytotoxic treatment induces complete remission in the majority of patients with lymphoid<br><br>
malignancies. Nevertheless, many of these patients relapse. Complete remission does not mean that the leukemic cells are totally eradicated from the body but that their level is beyond the sensitivity level of classical cytomorphologic methods. Since the detection limit of cytomorphological techniques is not lower than 1-5% malignant cells, this means that up to 1010 malignant cells can still remain in the patient, also indicated as the "minimal residual disease" (MRD). Apparently the current treatment protocols are not capable of killing all malignant cells in these relapsing patients, although they reached so-called complete remission according to cytomorphological criteria. Thus, it is obvious that cytomorphological techniques can only provide superficial information about the effectiveness of treatment.<br>
Detection of low numbers of residual malignant cells during early treatment response allows a precise assessment of the kinetics of tumour decrease during chemotherapy. It is now established that MRD information represents a powerful prognostic factor for final outcome. Detection of an increase of the MRD level enables to anticipate impending relapse. Techniques with a high sensitivity to detect MRD are therefore crucial to obtain a better insight in the reduction of tumour mass during induction treatment and further eradication of the malignant cells during maintenance treatment.<br>
Existing techniques for detecting chromosomal aberrations include traditional techniques such as cytogenetic analyses and biomolecular technology, such as Southern blotting or Fluorescent in situ hybridisation (FISH) analysis. However, the detection sensitivity of cytogenetics, FISH analysis and Southern blotting is typically not lower than 1-5%, which makes them unsuitable for the detection of MRD,<br>
PCR, although in essence well-suited for rapid and massive diagnostic testing or even screening, allows only 0.1 to 4-5 kb of nucleic acid (DNA or RNA) to be analysed routinely per PCR analysis, which greatly hampers rapid screening of vast stretches of chromosomes and breakpoint clusters or fusion regions<br><br>
within the chromosomes or their gene-products. An additional disadvantage of PCR is its inherent sensitivity to mismatched primers. Small alterations which can always be present in the nucleic acid sequence of the gene fragment complementary to the primer will make it impossible to operate the PCR with the wanted effect and may result in misdiagnosis and false-negative results. Especially false-negative results render a PCR-based diagnostic test, albeit very specific, unsuitable for reliable diagnosis, and it goes without saying that only a reliable diagnosis of malignancies can contribute to an insight into the prognosis and to the design of an adequate therapy. As indicated above, the majority of currently available techniques aim at detecting specific chromosomal aberrations involve analysis at the chromosomal or nucleic acid (DNA or RNA) level. It is however also possible to detect a chromosomal aberration at the protein level.<br>
Considerable effort has been undertaken in the field of leukaemia diagnostics to develop a suitable technology for the immunological detection of a fusion protein resulting from a chromosomal aberration. Many of those studies focussed on finding immunological reagents (antibodies) that are exclusively reactive with tumour-specific proteins and which do not at the same time give immunological detection of non-fusion proteins that are normally also produced by the body (see for example Nagasaki et al., J.Imm. Methods 162, 235-245, 1993 and Van Denderen et al., J Exp Med. 1989 169(l):87-98). Usually, such antibodies cross-react with normal cellular proteins. Only when specific fusion points are known, it may be possible to select specific immunological reagents that react exclusively with the tumour-specific protein, by selective binding to the fusion point epitope. However, the variation in fusion points is so large between patients (different breakpoints at the DNA level) and within patients (mRNA splice variants) that specific immunological detection only works in a few occasions, often solely on a patient-by-patient basis (see Van Dongen et al. Leukemia 1999; 13:1901-1928). Furthermore, diagnostic tests involving immunodetection of the fusion point of<br><br>
a fusion protein have the great inherent disadvantage that they require specialised and well-equipped laboratories and trained and highly skilled personal. Also, the above tests are typically only used in suspected cases of malignancies, and are not suitable for large scale screening of leukaemia patients for the presence of chromosomal aberrations.<br>
The development of a rapid and more convenient immunological method allowing the exclusive detection of a fusion protein A-B was a major step forward in the area of diagnostic tests aimed at detecting a fusion protein (see PCT/NL98/00289). The reported method involves the application of a so-called catching antibody which recognizes one part of a fusion protein (part A) and a labelled detection antibody which recognizes an other part of a fusion protein (part B). In such a system, a catching antibody is bound to a solid support layer, such as an.ELISA plate or a dipstick (P. Berendes, "Recognition of tumor-specific proteins in human cancer. PhD Thesis Erasmus University Rotterdam, ISBN 90-73436-36-2). A catching antibody can also be immobilized onto beads that can be analysed by flow cytometry (see for example PCT/NLO1/00945). Following incubation of a bead-bound catching antibody with a cellular lysate suspected of containing a fusion protein, bound fusion protein is detected by a labelled detection antibody.<br>
A (bead-based) catching/detection antibody system permits the exclusive detection of a fusion protein and is elegant and easy to perform, especially when using flow cytometry. Such a detection system also allows high-throughput diagnosis of many clinical samples. At the time of diagnosis of a malignancy involving a fusion protein, a diagnostic sample typically contains a relatively high number of malignant cells with a fusion protein compared to the number of normal cells which might contain native (non-fused) proteins. When ALL is diagnosed in a patient, the total number of leukemia cells is approximated to 1011 to 1012, of which approximately generally 25% to 98% contain a fusion gene encoding a fusion protein. A catching/detection antibody system is therefore suitable for the rapid screening for chromosomal<br><br>
aberrations which give rise to a fusion protein. Unfortunately however, the sensitivity of a catching/detection antibody system often appears insufficient in detecting low frequencies (10%). Taken together, there is a specific need for a method that allows to exclusively and conveniently detect a fusion protein in a sample, even if the sample contains only minimal amounts of the fusion protein.<br>
It has now been recognized that, in current methods, a fusion protein and a non-fused native (i.e. wild-type) protein typically compete for specific binding to a probe that is used for the detection of the fusion protein. For example, a fusion protein A-B and a non-fused native protein A will compete for binding to a catching antibody, or other type of probe, specifically reactive with part A of the fusion protein. Likewise, a fusion protein A-B and a non-fused native protein B compete for specific binding to a catching antibody that is reactive with part B of the fusion protein. Thus, the sensitivity of a catching/detection antibody system for detecting a fusion protein comprising parts of at least two native proteins is largely determined by the proportion of a fusion protein present in a sample relative to said native non-fused proteins. When a native protein is in large excess of a fusion protein of interest, the native protein is likely to occupy most, if not all, binding sites of a catching antibody, thereby essentially preventing binding of a fusion protein. Furthermore, even if some fusion protein is efficiently caught by a catching antibody, the signal intensity indicative of binding of a labelled detection antibody may not be sufficient to yield a reliable test result. The invention provides the insight that the sensitivity of a method for detecting a fusion protein in a sample can be increased by depleting said<br><br>
sample of one or more native proteins prior to detecting the presence of a fusion protein, to enrich said sample for said fusion protein relative to one or more 'competing1 native proteins. Provided is a method to detect a fusion protein (e.g. resulting from a chromosomal translocation) in a sample via the detection of a fusion protein comprising an ammo-terminal fragment and a carboxy-terminal fragment which are each corresponding to a different native protein, said method comprising (i) contacting said sample with at least one binding molecule specifically reactive with a part of the native protein that is not present in the fusion protein, under conditions that allow the formation of a complex between said at least one binding molecule and said native protein; (ii) removing said complex from said sample to essentially deplete said sample of said native protein but not the fusion protein and (iii) detecting said fusion protein in said sample using at least one antibody directed against said fusion protein. The term native protein or relevant native protein as used herein refers to a non-fused, wild-type protein which can compete with a fusion protein of interest for binding to a probe or reagent that is used for the detection of a fusion protein, such as for instance a catching antibody. A native protein of the invention can adopt a native conformation (the three dimensional structure of a protein when in a physiological environment) or a denatured conformation.<br>
According to the invention, a fusion protein is derived from a fusion gene of a malignant cell or from an artificially produced fusion gene. In a fusion protein comprising an amino-terminal fragment and a carboxy-terminal fragment which are each corresponding to a native protein, said fragments are typically directly fused or connected to each other via a breakpoint fusion or fusion region such that these fragments together represent the whole fusion protein. The length or the size of a fragment, whether amino-terminal or carboxy-terminal, can vary. If the fusion region is located approximately halfway a fusion protein, the fusion protein comprises an amino (N)-terminal fragment and a carboxy (C) - terminal fragment of approximately the same size. In other<br><br>
cases, the fusion region is located more closely towards either the C-terminal or N-terminal end of the fusion protein, such that the fusion protein consists of a large N-terminal fragment and a small C-terminal fragment or a large C-terminal fragment and a small N-terminal fragment, respectively. A fusion protein of the invention contains sufficient amino acids upstream and downstream from the fusion region to allow for (epitope) recognition by an antibody probe specifically reactive with parts of the fusion protein located at opposite sides of the fusion region or a binding fragment functionally equivalent thereto.<br>
In a method of the invention, a sample is enriched for a fusion protein relative to one or more 'competing1 native proteins, resulting in increased binding of a fusion protein to a probe directed against said fusion protein. A method provided is particularly suitable to increase the sensitivity of a catching/detection antibody or sandwich system for the detection of a fusion protein. Thus, the invention provides a solution to a major problem encountered using existing detection methods wherein a native protein saturates or "consumes" a probe that is used to detect a fusion protein, such as a catching antibody, thereby preventing the binding of a fusion protein to said probe. Obviously, probe saturation with native protein reduces the sensitivity of detecting a fusion protein present in the same sample. By depleting a sample of a competing native protein according to the invention, the ratio fusion protein/native protein is strongly increased, thus favouring binding of the detection probe, and detection of the fusion protein. With a method provided herein, it is now possible to specifically detect the presence of a fusion protein in relatively rapid and simple fashion, even if said fusion protein is present in a sample in only a limited amount. For example, it allows to apply a catching/detection antibody system to detect a tumour-specific fusion protein in a sample comprising only a small population of fusion protein-positive malignant cells in a large background of normal cells, which contain a large number of competing, native proteins.<br><br>
Furthermore, with a method according to the invention it is now also possible to calculate the amount of native protein (as a measure for the amount of normal cells) relative to the amount of fusion protein (as a measure of the amount of malignant cells. The obtained ration of normal protein versus fusion protein can be used to determine the relative frequency of malignant cells, i.e. the level of MRD.<br>
The invention provides the insight that the sensitivity of detecting the presence of a fusion protein in a cell, e.g. a chimeric oncoprotein, in a malignant cell, can be increased by depleting such a cell of at least one relevant native protein. Of course, it is possible to deplete more than one native protein prior to detection of a fusion protein. Needless to say, use of a method provided herein is most beneficial when detecting a fusion protein in a sample that also-contains a significant amount of native proteins which can compete in binding to a probe that is used to catch a fusion protein. In one example, a method is provided for the detection of MRD by determining the presence of malignant cells in a sample, said method comprising detection of a tumour-specific fusion protein comprising parts of at least two native, non-tumour specific proteins in a sample, said detection step being preceded by a "preclear" step wherein said sample is depleted of at least one of said native proteins, which would otherwise compete with the detection of the fusion protein.<br>
In a method of the invention at least one binding molecule is used to remove one or more native proteins from a sample. A sample comprises a biological sample such as a blood sample, serum sample, tissue sample, bone marrow, cerebrospinal fluid sample, biopsies and other samples that may contain one or more cells expressing a xusion protein encoded by a fusion gene. A sample is treated in such a manner that a binding molecule has sufficient access to a native protein. Because many cellular proteins are localized intracellularly, this is typically achieved by disrupting the membrane integrity of a cell. For example, a sample is treated in such a manner that it yields a cell<br><br>
lysate or a cell homogenate, for instance obtained by mechanical rupture of cells or by exposing cells to a cell lysis buffer containing a detergent. A suitable binding molecule preferably comprises a protein or a polypeptide, like an antibody or an antibody fragment. However, other types of binding molecules are also instrumental in practicing a method provided. A specific antibody or fragment thereof can be obtained using various standard procedures known in the art, including methods to raise polyclonal antibodies in a laboratory animal, methods to obtain monoclonal antibodies using conventional hydridoma technology, and single-chain Fv antibody fragments (scFvs) using a phage display selection process. Such an antibody or antibody fragment is herein further referred to as "depleting antibody".<br>
For the design or development of a binding molecule that is reactive with a native protein but not with a fusion protein comprising a part of said native protein (such as a depleting antibody), it is of course crucial to know which part or fragment of said native protein is part of the fusion protein and which part is not. In other words, it is preferred that the fusion point or fusion region of a fusion protein to be detected is established. A method of the invention is advantageously used to detect a fusion protein that is the result of a known chromosomal aberration. In these cases, the resulting fusion genes and fusion proteins have often been characterized into detail. Well known examples are the translocations resulting in BCR-ABL fusion genes found in &gt;95% of cases of CML and in 30% of cases of adult ALL and the TEL-AML1 fusion genes which are found in 25-30% of childhood ALL cases. However, many more fusion genes and encoded fusion proteins are known to be involved in leukemia, such as E2A-PBX1, ETO-AML1 and PML-RARa.<br>
It is to be understood that application of the present invention is not limited to the detection of leukemia; several other, non-leukemic diseases are characterized by the occurrence of fusion genes and fusion proteins. For example, the following ALK fusion genes can be observed in anaplastic large cell lymphoma (ALCL): the fusion gene NPM-ALK resulting from chromosome<br><br>
aberration t(2;5)(p23;q35); fusion gene TPM3-ALK resulting from chromosome aberration t(l;2)(q25;p23); fusion gene TFG-ALK resulting from t(2;3)(p23;q21) and fusion gene ATIC-ALK from inv(2)(p23q35). Depletion of native ALK protein from a sample according to the invention using a depleting antibody reactive with native ALK but not with the fusion protein will increase the sensitivity of a subsequent catching/detection assay. In another embodiment, the invention is advantageously applied to detect a chromosomal aberration observed Ewing's sarcoma (EWS), such as t(ll;22)(q24;ql2) resulting in the EWS-FLI1 fusion protein, t(21;22)(q22;ql2) resulting in the EWS-ERG fusion protein and t(7;22)(p22;ql2) giving rise to the EWS-ETV1 fusion protein. Ewing's sarcoma (EWS) / Peripheral Primitive Neuroectodermal Tumours (PNET) of bone is a type of cancer usually found in children and young adults. The peak incidence is between ages 10 and 20, it is less common in children under 5 or in adults over 30. Ewing's sarcoma can occur in any bone in the body; the most common sites are the pelvis, thigh, lower leg, upper arm, and rib. Depletion of normal cellular EWS protein (and/or optionally depletion of the fusion partner gene such as FLU, ERG or ETV1) in a preclear step according to the invention will increase the sensitivity of detecting a fusion protein related to Ewing's sarcoma.<br>
Thus, the invention allows a person skilled in the art to apply and perform the invention without undue burden to the diagnosis or classification of any type of disease that is associated with the occurrence of a fusion gene that is translated into a fusion protein comprising an amino-terminal fragment and a carboxy-terminal fragment which are each corresponding to a native protein.<br>
Based on the structure of a fusion protein, one can easily determine which part of the native protein is not present in the fusion protein. A binding molecule specifically reactive with this part, or a (polypeptide) stretch thereof, can be used to essentially remove the native protein, but not the fusion protein, from a sample. A person skilled in the art will recognize which steps to<br><br>
follow to obtain a binding molecule specifically reactive with the native (wild-type) protein but not with the fusion protein. In a preferred embodiment, such a specific binding molecule is an antibody or a binding fragment functionally equivalent thereto.<br>
Methods of producing an antibody are known to those skilled in the art. For example, to obtain a polyclonal antibody, a laboratory animal is immunized with an immunogen. In a method of the invention, said immunogen preferably is a recombinant protein fragment or a synthetic pep tide corresponding to the selected stretch or domain of the native protein. The animal's immune response is monitored by taking test bleeds and determining the titer of the reactivity. When appropriately high titers are obtained, blood is collected from the animal and antisera are prepared. Further fractionation of the antisera to enrich for antibodies reactive with the native protein can be done if desired. See e.g. Harlow et al. Antibodies. A Laboratory Manual, Cold Spring Harbor Publications, New York (1988). Monoclonal antibodies can be obtained by various techniques known in the art, for example by fusing spleen cells of immunized mice with a myeloma cell line by the addition of polyethylene glycol (PEG). Fused cells are cultured in a selection medium, e.g. medium containing a mixture of hypoxanthine, aminopterin and thymidine. Fused cells which survive in this selection medium are tested for the production of the desired antibody (for instance by a solid-phase immunoassay such as ELISA) and, if positive, the cultures are cloned so that there is only one cell in each culture well. This produces a clone of cells from a single progenitor which is both immortal and a producer of monoclonal antibody. Antibodies obtained can be characterised using conventional immunodiagnostic techniques e.g. by Western blotting using lysates of cells expressing either the (recombinant) native protein or the (recombinant) fusion protein. Only those antibodies reactive with the native protein A but not with the fusion protein A-B are suitable for use as depleting antibody in a method according to the invention.<br><br>
According to the invention, a complex between a depleting antibody and a native protein is removed from a sample prior to detecting a fusion protein. In one embodiment, a method provided entails rapid and selective removal or precipitation of a native protein out of a cell lysate or homogenate, via for example the formation of an antibody/antigen complex. Hereto, a sample is contacted with at least one binding molecule specifically reactive with a native protein under conditions that allow the formation of a complex between said binding molecule and said native protein. Various separation techniques can be used in a method according to the invention to remove a complex from a sample. Commonly used methods to precipitate an antibody/antigen complex out of a crude mixture include solid phase immunoprecipitation. In one embodiment, said complex is removed from said sample via the addition of solid beads or a microspheres capable of binding said complex, to form a suspension of beads in an essentially liquid sample. According to the invention, a suitable bead comprises a Sepharose bead, or a polystyrene bead, or any kind of solid particle that allows the simple removal of a bead-bound complex from a sample. The advantage of solid beads is that they can be simply pelleted from a sample, typically by centrifugation of the suspension. Alternatively, a pre-cleared sample is obtained by filtration of the suspension. In one embodiment of the invention, a sample is contacted with a depleting antibody to form a complex with a native protein and an immunoglobulin binding protein immobilized onto a bead is used to remove said complex from said sample. Various types of immunoglobulin binding proteins exist which can suitably be used (Antibodies, a Laboratory Manual, E. Harlow &amp; D. Lane, (1988)). These include protein A, protein G and bacterial cell ghosts of Staphylococcus aureus. S. aureus has a special protein on its surface called Protein A which is a versatile, broad range IgG-binding reagent. Protein A binds the Fc portion of antibodies (IgG class) without disturbing their binding of antigen. Antibodies from many mammalian species will generally react with Protein A. In another embodiment, GammaBind G Type 2 or Protein G is used to remove a complex<br><br>
of a depletion antibody and a native protein from a sample. GammaBindâ„¢ G Type 2 and Protein G are recombinant engineered forms of streptococcal protein G. Both proteins can be produced in Escherichia colL They bind to the constant region of many types of immunoglobulin G, and are widely used to detect, quantify and purify IgG antibodies and antigen/antibody complexes. GammaBind G Type 2 and Protein G are the most universally applicable antibody binding proteins available. Compared with Protein A, they bind tightly and specifically to antibodies from many different species. For example, GammaBind G Type 2 binds better to mouse and rat IgG, GammaBind G Type 2 and Protein G bind selectively to all subclasses from human IgG. Bacterial ghosts, immunoglobulin binding proteins and beads coated with immunoglobulin binding proteins can be commercially obtained from various suppliers, such as Pharmacia Biotech AB, Sigma Aldrich or Pierce. In a method provided herein, following the formation of a complex between a native protein and a depleting antibody, bacterial ghosts or coated beads are mixed with the cell sample or cell homogenate, during which time the immunoglobulin binding protein attaches to the depleting antibody. A brief and simple centrifugation or filtration step can separate the bacterial ghosts or beads with attached depleting antibody / native protein complex from the sample.<br>
In a preferred embodiment however, a binding molecule (preferably a depleting antibody) is directly coupled to or immobilized onto a bead or another solid particle. Display of biochemical reagents on synthetic microbeads is a prevailing trend in the development of bioanalytical assays. Commercial sources (e.g. Spherotech, IL; Bangs, IN; Polysciences; PA); Molecular Probes, OR) for bead-based display systems (e.g., covalent coupling, biotin-streptavidin, His-tag) are currently available. Use of a depleting antibody which is directly conjugated to a bead does not require the intermittent use of an immunoglobulin binding protein and will in general result in increased specificity. Following incubation for a sufficient time period, beads with<br><br>
attached depleting antibody complexed to native protein can be removed from a sample, such as a lysate or homogenate. Removal of beads from a sample is easily and rapidly achieved by for instance centrifugation, resulting in a sample that is essentially depleted of a native protein which would otherwise compete with a fusion protein for binding to a probe directed against said fusion protein. Because this depletion step results in a sample that is enriched for a fusion protein relative to the "competing" native, non-fused protein, the probability that a fusion protein will bind to such a probe, e.g. a catching antibody, is significantly increased. For example, a method provided now allows to detect a fusion protein using a catching/detection system with increased sensitivity.<br>
In another embodiment of the invention, a depletion antibody or any other type of suitable binding molecule, is conjugated to a magnetic bead. This allows the removal of a bead-bound complex from a sample using immunomagnetic separation (IMS). IMS is a technique that involves adding magnetic beads coated with a binding molecule, which is specifically reactive with an antigen of interest, to a cellular lysate. The beads and lysate are mixed, during which time a complex is formed between a depleting antibody or other binding molecule and a native protein to form a (antigen/antibody) complex. After a certain incubation period, a magnetic field is applied to separate the magnetic beads or magnetic particles with attached complex from other substances, leaving behind a depleted or 'pre-cleared1 sample. Suitable magnetic beads are for example those marketed by Spherotec Inc. (www.spherotech.com) under the brandname SPHERO magnetic particles. They are prepared by coating a layer of magnetite and polystyrene onto monodispersed (i.e.. uniform sized) polystyrene core particles (see US Patent No. 5,091,206). As a result, the particles are spherical in shape, and paramagnetic in nature. They are also very uniform in size. The magnetite contents of these magnetic particles can be adjusted but in general it represents about 10% to 15%. The magnetic particles can be easily separated<br><br>
from a suspension magnetically. These particles become non-magnetic when removed from a magnet, and do not retain any detectable magnetism even after repeated exposure to strong magnetic field. The magnetic particles can be used for cell separation, affinity purification, DNA probe assays, magnetic particle EIA, etc. Various sizes of beads, be it Sepharose beads, polystyrene beads, latex beads, magnetic beads or the like, are commercially available. In a preferred embodiment, beads with a relatively small diameter are used, because for a certain volume of beads, small beads have a larger surface area compared to large beads. The larger the bead surface, the more binding molecules can be immobilized onto a bead and the more efficient a native protein can be removed from a sample per volume of beads. This is especially advantageous when removing a native protein from a sample with a small sample volume.<br>
As said, the invention solves the problem that a catching/detection antibody system, although rapid and easy to perform, is often not sensitive enough to detect the presence of low amounts of a fusion protein in a sample. A method is provided to detect a fusion protein comprising parts of two different native proteins with increased sensitivity by including a pre-treatment step to obtain a sample that is enriched in a fusion protein of interest relative to one or both of the native proteins. This enriched, or precleared, sample, is processed further to detect the presence of a fusion protein. In theory, a fusion protein A-B present in a sample that is completely depleted from native protein A using a method provided, can be detected using a single antibody (or functional equivalent thereof) directed against part A of the A-B fusion protein. However, as it is very difficult in practice to obtain a precleared sample which does not contain any native protein A, use of a single A-specific antibody may not always give the desired detection specificity. Thus, when aiming for the exclusive detection of an A-B fusion protein, it is preferred to use at least one additional antibody that is directed against part B of the fusion protein. For example, a fusion protein is detected using a set of at least<br><br>
a first and a second antibody, each antibody capable of recognizing a binding site positioned at opposite sides of the fusion region of said fusion protein. In a preferred embodiment of the invention, the invention provides a method wherein a fusion protein derived from a fusion gene is detected by flow cytometric detection or by any other convenient way to measure bead-associated fluorescence using at least two antibody probes, whereby at least one is directed against a protein fragment comprising the ammo-terminal fragment of the fusion protein, and at least one other one is directed against a protein fragment comprising the carboxy-terminal fragment of the fusion protein.<br>
The use of microspheres, beads, or other particles as solid supports for antigen-antibody reactions in order to detect antigens or antibodies in a sample is particularly attractive when linked to flow cytometry (see e.g. US 6,159,748). In a preferred embodiment, an antibody probe is coupled to a bead that allows the detection of a fusion protein via flow cytometry. Flow cytometers have the capacity to detect particle size differences and are highly sensitive fluorescence detectors. Microspheres can be sized by forward angle light scatter (FALS) or electronic volume. Used in conjunction with right angle light scatter (RALS), a flow cytometer can distinguish between single and aggregated particles.<br>
A method according to the invention is very attractive for the detection of a tumour-specific protein comprising an amino-terminal fragment and a carboxy-terminal fragment which are each corresponding to a different non-tumour-specific protein. Said tumour-specific fusion protein is for example a result of a Philadelphia chromosome aberration, creating the abnormal BCR-ABL protein. Prior to detecting, one or both of the native proteins BCR and ABL are depleted from a sample to be analysed by using at least one binding molecule specifically reactive with (a fragment of) the ABL or BCR protein, but not with the BCR-ABL fusion protein. Following removal of the complex comprising BCR and/or ABL, the BCR-ABL fusion protein is detected,<br><br>
preferably using a set of at least a first and a second antibody, whereby each antibody recognizes a binding site positioned at opposite sides of the fusion region of the fusion protein. The fusion protein can be detected using a catching/detection system, for instance using flow cytometry, ELISA, or a dipstick method (see also the Example below).<br>
A method of the invention is especially suitable to detect very low numbers of cells expressing a fusion protein (
 <br><br><br>
breakpoints. An essential part of a method according to the invention for the detection of a one or more fusion proteins, be it using a catching/detection system or any other kind of detection procedure, lies in the fact that fusion protein detection is preceded by the depletion of one or more relevant native, non-fused proteins. This can be achieved using one or more binding molecules, such as depletion antibodies, that can specifically recognize and bind to a native protein to form a complex. Different binding proteins may be attached to different single solid support surfaces, like a bead. Alternatively, multiple binding partners specifically reactive with different native proteins are coupled to the same bead such that only a single bead can be used to simultaneously deplete a sample of multiple native proteins. This is especially of interest when a method according to the invention is used to investigate the occurrence of different fusion proteins simultaneously in one tube, e.g. for the diagnosis of a malignancy within one disease category, particularly when using patient samples with a low tumour load (e.g. less than 5%) at diagnosis. Thus, a method of the invention allows for the simultaneous detection of a first fusion protein A-B, comprising the N-terminal fragment of the native protein A and the C-terminal fragment of the native protein B, and a second fusion protein C-D, comprising the N-terminal fragment of the native protein C and the C-terminal fragment of the native protein D, in a single tube assay with a high specificity and improved sensitivity compared to existing detection methods. In this specific example, a patient sample with a low tumour load is for example contacted with a depletion antibody specifically reactive with the C-terminal fragment of protein A (anti-A) and with a depletion antibody specifically reactive with N-terminal fragment of protein D (anti-D), to deplete the sample from native proteins A and D, respectively. The anti-A and anti-D depletion antibodies are preferably immobilized onto a (small) bead, even more preferred onto the same bead for reasons of efficiency and ease of use. A simple centrifugation step is in general sufficient to separate the complex, in this specific example consisting of beads/anti-A/protein A/anti-D/protein D, from<br><br>
the sample. As a result, a sample is obtained which is essentially depleted of native proteins A and D. This will improve the likelihood that the fusion proteins A-B and C-D will be recognized by their specific probes, e.g. a catching antibody specifically reactive with the N-terminal fragment of fusion protein A-B and the C-terminal fragment of fusion protein C-D.<br>
In addition to the limited sensitivity, current methods for detecting a fusion protein in a sample face the common problem that the possibility to quantify the amount of fusion protein detected are limited. Especially when applying these methods as diagnostic tools, it is highly desirable to quantify, or at least semi-quantify, a fusion protein. If possible at all, a detection signal, such as fluorescence intensity, which indicates the presence of a fusion protein can be expressed relative to an internal control factor assumed to be constant irrespective of whether or not a fusion protein is present. Such a factor can for instance be the number of cells analysed or the total protein content of a sample. This relative value can be used for the purpose of comparison between different samples, e.g. a positive and a negative control sample. Obviously, it would be far more informative to directly relate the presence of a fusion protein comprising parts of two native proteins to one, or even both, non-fused native protein(s).<br>
Thus far, none of the existing detection methods allows for such a quantification. Importantly, the present invention now also provides a simple and straightforward solution to this problem. A complex containing one or more depleted native protein(s) that is removed from a sample in a method of the invention is preferably analysed further to indicate the amount of native protein(s) depleted from said sample. In a preferred embodiment, this analysis comprises the use of at least one reagent capable of specifically binding to one or more components of the depleted or isolated complex, followed by determining the binding of said reagent to said complex as an indication of the amount of native protein that was depleted from said sample. A reagent preferably comprises a binding partner, like an antibody or a functionally<br><br>
equivalent fragment thereof, capable of binding a depleted native protein. In a preferred embodiment, a depleted complex is detected using a reagent directed against a depleted native protein, at a region which is distinct from the site of interaction between the native protein and a depletion antibody. More preferred, a reagent for detecting a depleted native protein reacts with that part of the native protein which is also present in the fusion protein. For example, a depleted protein A is advantageously detected using the '"catching antibody" used for the detection of the A-B fusion protein as a reagent. A reagent is preferably provided with a reporter molecule or label, like a fluorochrome, allowing detection of a complex via the binding of a labeled-reagent to a depleted protein.<br>
Detection of a complex to determine the amount of native protein depleted form a sample is advantageously carried out using flow cytometry. In one embodiment, a depletion antibody is conjugated to a bead. Suitable beads comprise those discussed above for detection of a fusion protein, including magnetic beads compatible with flow cytometry.<br>
When the amount of depleted native protein is known, this information can be used as an indication of the amount of cell material or total protein content. In other words, the depleted protein can serve as a kind of "internal control" in the diagnostic test. When relating the presence (expression) of a protein in an isolated complex to the amount of cell material or to the number of cells it is of course preferred that the isolated complex contains a sufficient amount of said native protein to be detected by a reagent. Thus, a native protein to be depleted and analysed further as a measure of the amount of cell material preferably comprises a protein that is relatively ubiquitous in said cells. In addition, it is important that the expression of said native protein is not subject to large variations, e.g. between individual cells or under different cellular conditions. In a preferred embodiment, said native protein to be depleted and analysed further is an abundant protein with a stable expression, such as a housekeeping protein. For example, when detecting the BCR-ABL<br><br>
fusion protein in a sample it is advantageous to deplete and subsequently detect the native ABL protein because ABL is an abundant protein. However, a fusion protein does not always comprise a fragment of an abundant native protein. For example, in the E2A-PBX1 fusion protein the N-terminal transactivation domain of the basic helix-loop-helix (bHLH) transcription factor, E2A, is joined to the majority of the pre-B-cell leukemia transcription factor 1(PBX1), Transcription factors are typically expressed at a low level and in a cell-type specific fashion. Thus, in these cases it is not preferred to use any of the depleted native proteins as an internal control. Instead, it is advantageous to use an additional (antibody) probe which simultaneously depletes a sample of an abundant protein. To illustrate this further, in the situation depicted above, a sample e.g. a cell lysate is contacted with a bead that is coated with two antibodies: one antibody reactive with native E2A but not with the E2A-PBX1 fusion protein to deplete the sample of competing E2A protein, and one antibody reactive with a housekeeping protein (such as ABL) to deplete the sample at the same time of a housekeeping protein that can serve as internal control.<br>
According to the invention, (semi)-quantification of a depleted native protein can be performed by constructing a standard curve relating a measured quantity (e.g. fluorescence) to "known" amounts of native protein. Preferably, "known" samples contain the native protein in amounts chosen to span the concentration range of native proteins that are expected in the "unknown" samples (e.g. using (serial) dilutions of normal cells). Such a standard curve can then be used to determine concentrations of the depleted native protein substance in an "unknown" sample (or a dilution thereof) containing the depleted complex.<br>
The invention provides a method as discussed above using probes that can be labelled or conjugated with reporter molecules, such as biotin, digyoxigenin, enzymes such as peroxidase, alkaline phosphatase, or other reporter molecules or reporter particles, such as beads, known in the art. The<br><br>
invention further provides a diagnostic kit comprising all the means, such as binding molecules, (conjugated) beads, (labelled) probes or reagents or substrate or instructions, necessary to carry out the method according to the invention. Methods or diagnostic kits provided by the invention are preferably used to detect chromosomal aberrations found with certain types of cancer, for example with leukaemia, be it in the detection of (residual) cancer in patients or the screening for cancer in larger populations as a whole.<br>
Strengths of a method according to the invention are: 1) the ability to simultaneously, but discretely, analyse multiple fusion proteins (most relevant at the time of diagnosis); 2) the simplicity of binding proteins to microspheres; 3) the ability of flow cytometry to detect small particle size differences; and 4) the exquisite sensitivity of flow cytometry as a detector of different wavelengths of fluorescence, simultaneously. Available auto-sampling systems make it even more appealing in this regard. A method of the invention is advantageously used in diagnostic testing of biological samples such as blood samples, serum samples, samples of cells, tissue samples, cerebrospinal fluid, bone marrow, biopsies, for chromosomal aberrations. The invention provides a method to be used in diagnostic testing where both a high sensitivity as well as a high specificity is required. A method provided now allows to both diagnose a malignancy and closely monitor MRD during follow-up of patients with various types of cancer which involve chromosomal translocations, inversions or deletions that give rise to a fusion gene. Use of a method of the invention is provided before, during and after treatment of a disease to evaluate the effectiveness of said treatment.<br><br>
EXAMPLE<br>
Detection of BCR-ABL fusion protein in precleared cell lysates<br>
This example illustrates how normal (i.e. native) ABL can be removed from a sample prior to detection in a catching/detection antibody assay to improve the sensitivity of detecting the BCR-ABL fusion protein in an MRD assay. Such a "pre-clear" step can be performed with an anti-ABL antibody against the N-terminal part of ABL. All normal ABL expressed in the cells will be cleared, and only the ABL fragment that is present in the BCR-ABL fusion protein will bind to the detection beads carrying anti-ABL catching antibody against the C-terminal part of ABL. BCR-ABL bound to the beads is then detected using an anti-BCR detection antibody conjugated to a fluorescent label.<br>
Materials<br>
1)	Leukemic cells obtained from a MRD patient cultured in medium<br>
2)	Phosphate-buffered saline (PBS) 12% foetal calf serum (FCS): add 2 ml FCS<br>
(heat-inactivated) to 100 mi PBS pK 7.8; Store at 4 "C.<br>
3)	Radioimmunoprecipitation (RIPA)-buffer (without sodium deoxycholate):<br>
Preparation of 50 ml of RIPA-buffer:<br>
50 mM Tris-HCl: use 2.5 ml of a 1 M stock Tris-HCl<br>
150 mM NaCl: use 1.5 ml of a 5 M stock NaCl<br>
1% NP40: use 0.5 ml of undiluted Nonidet P40 (BDH, #56009 2L)<br>
0.1% SDS: use 0.5 ml of a 10% stock SDS<br>
Add 45 ml milli-Q.<br>
Store at 4 Â°C<br>
4)	Protease inhibitor cocktail, Sigma, #P2714:<br>
Dissolve the powder in 2 ml milli-Q to prepare a 50x concentrated solution. Store at 4 Â°C.<br><br>
5)	Preclear beads: Polystyrene beads (&gt;10 pm) coated with depleting antibody<br>
directed against the N-terminus of ABL.<br>
6)	Detection beads: Luminex, PolyScience, or Molecular Probe beads, coated<br>
with catching antibody directed against the C-terminus of ABL.<br>
5     7) Detection antibody: either FITC- or biotin-conjugated antibody directed<br>
against the N-terminus of BCR. 8) Streptavidin (SA)-PE, Caltag laboratories, #SA1004-l<br>
Methods 10     Preparation of cell Ivsates<br>
1)	Transfer the number of cells needed and exactly calculated to a fresh tube<br>
and wash cells twice with PBS. Minimally l*106 cells are typically required<br>
for the analysis of a fusion protein. However, more cells e.g. up to 50* 106<br>
15	cells may be needed if the MRD level (and therefore the levels of aberrant<br>
fusion protein) is expected to be very low.<br>
2)	In the meantime, calculate the volume of RIPA-buffer needed to make cell<br>
lysates: 50 \d RIPA-buffer per 500.000 cells. Transfer the RIPA-buffer to a<br>
15 ml tube and add 50x concentrated protease inhibitor cocktail (20 \i20	protease inhibitor cocktail per 1 ml RIPA-buffer).<br>
3)	Resuspend the cell pellets in the appropriate amount of RIPA-buffer (5 x<br>
106/ml) containing protease inhibitors and transfer the solutions to fresh<br>
eppendorf-tubes.<br>
4)	Incubate on ice for 30 minutes.<br>
25     5) Centrifuge 10 minutes at 10.000 xg, 4 Â°C.<br>
6) Transfer the supernatant (cell lysate) to a fresh eppendorf-tube.<br>
Preclear of cell lvsate<br><br>
7)	Add lxl06 preclear beads to 100 id of cell lysate sample.<br>
8)	Incubate on ice for 30 minutes.<br>
9)	Spin at 10.000 x g for 3 minutes and transfer the supernatant to a new<br>
eppendorf tube.<br>
10)	Repeat step 9.<br>
Detection of fusion protein<br>
11)	Add to one test-tube:<br>
â€¢	100 jd of precleared cell lysate sample<br>
â€¢	10.000 catching beads in 10 pi PBS + 2% FCS<br>
â€¢	1-2 ug of conjugated detection antibody in 25ul PBS + 2% FCS<br><br>
12)	Incubate on ice for 30 minutes.<br>
13)	Wash the beads with 500 ul PBS/2% FCS and spin for 1 minute at 14.000 x<br>
g. Remove supernatant and repeat this step.<br>
When a biotin-labeled detection antibody is used, continue with steps 14 -18. When a FITC-labeled detection antibody is used, continue with steps 17 and 18.<br>
14)	Dilute SA-PE 200 times in PBS/2% FCS and resuspend the beads in 20<br>
of the diluted SA-PE per tube.<br>
15)	Incubate for 10 minutes at room temperature.<br>
16)	Wash the beads with 500 ul PBS/2% FCS and spin for 1 minute at<br>
14.000xÂ£. Remove supernatant and repeat this step.<br>
17)	After the second wash, resuspend the beads in 100-150 ^1 PBS/2% FCS per<br>
tube and transfer to FACS-tubes.<br><br>
18) Measure the amount of label on the beads with a flow cytometer. The ratio of the amount of label and the initial number of cells is a measure of MRD level.<br><br>
Claims<br>
1.         A method for detecting a fusion protein in a sample, said fusion protein comprising an amino-terminal fragment and a carboxy-terminal fragment which are each corresponding to a native protein, said method comprising:<br>
-	contacting said sample with at least one binding molecule specifically<br>
reactive with a part of the native protein that is not present in the fusion<br>
protein, under conditions that allow for the formation of a complex between<br>
said at least one binding molecule and said native protein;<br>
-	removing said complex from said sample to deplete said sample of said native<br>
protein; and<br>
-	detecting said fusion protein in said sample using at least one antibody<br>
directed against said fusion protein.<br><br>
2.	A method according to claim 1 wherein said at least one binding<br>
molecule is an antibody or a binding fragment functionally equivalent thereto.<br>
3.	A method according to claims 1 or 2 wherein said at least one<br>
binding molecule is conjugated to a solid particle, preferably a bead.<br>
4.	A method according to claim 3, wherein said complex is removed<br>
from said sample by centrifugation or by magnetic separation.<br>
5.	A method according to any one of claims 1 to 4, wherein said fusion<br>
protein is detected using a set of at least a first and a second antibody, each<br>
antibody capable of recognizing a binding site positioned at opposite sides of<br>
the fusion region of said fusion protein.<br>
6.	A method according to any one of claims 1 to 5, whereby at least one<br>
antibody is labelled with a fluorochrome.<br><br>
7.	A method according to any one of claims 1 to 6, whereby at least one<br>
antibody is coupled to a bead.<br>
8.	A method according to any one of claims 1 to 7, wherein said fusion<br>
protein is detected via flow cytometry.<br>
9.	A method according to any one of the preceding claims further<br>
comprising after the step of removing said complex from said sample, detecting<br>
said complex using at least one reagent capable of specifically binding to said<br>
complex and determining the binding of said at least one reagent to said<br>
complex as an indication of the amount of native protein depleted from said<br>
sample.<br>
10.	A method according to claim 9 wherein said native protein is an<br>
abundant or housekeeping protein.<br>
11.	A method according to any one of claims 1 to 10, wherein said fusion<br>
protein comprises an amino-terminal fragment and a carboxy-terminal<br>
fragment which are each corresponding to a different non-tumour-specific<br>
protein.<br>
12.	A method according to claim 11 wherein said fusion protein is a<br>
result of a Philadelphia chromosome aberration.<br>
13.	A method according to claim 12 whereby the amino-terminal<br>
fragment of said fusion protein corresponds to the ABL or BCR protein<br>
whereas the carboxy-terminal fragment of said fusion protein corresponds to<br>
the BCR or ABL protein, respectively.<br><br>
14.	A method according to claim 13 whereby said at least one binding<br>
molecule, preferably an antibody, is specifically reactive with a fragment of the<br>
ABL or BCR protein, but not with said fusion protein.<br>
15.	A diagnostic kit comprising a binding molecule and at least one<br>
probe for carrying out a method according to any of claims 1 to 14.<br>
16.	Use of a method according to any of claims 1 to 14 or of a diagnostic<br>
kit according to claim 15 in the diagnosis and / or classification of a disease.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDg2Mi1jaGVucC0yMDA2IGFic3RyYWN0LWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">0862-chenp-2006 abstract-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDg2Mi1jaGVucC0yMDA2IGNsYWltcy1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">0862-chenp-2006 claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDg2Mi1jaGVucC0yMDA2IGRlc2NyaXB0aW9uIChjb21wbGV0ZSktZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">0862-chenp-2006 description (complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYyLUNIRU5QLTIwMDYgQ0xBSU1TIEdSQU5URUQucGRm" target="_blank" style="word-wrap:break-word;">862-CHENP-2006 CLAIMS GRANTED.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYyLUNIRU5QLTIwMDYgQ09SUkVTUE9OREVOQ0UgT1RIRVJTLnBkZg==" target="_blank" style="word-wrap:break-word;">862-CHENP-2006 CORRESPONDENCE OTHERS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYyLUNIRU5QLTIwMDYgQ09SUkVTUE9OREVOQ0UgUE8ucGRm" target="_blank" style="word-wrap:break-word;">862-CHENP-2006 CORRESPONDENCE PO.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYyLUNIRU5QLTIwMDYgRk9STSAxOC5wZGY=" target="_blank" style="word-wrap:break-word;">862-CHENP-2006 FORM 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYyLUNIRU5QLTIwMDYgUE9XRVIgT0YgQVRUT1JORVkucGRm" target="_blank" style="word-wrap:break-word;">862-CHENP-2006 POWER OF ATTORNEY.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYyLWNoZW5wLTIwMDYtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">862-chenp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYyLWNoZW5wLTIwMDYtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">862-chenp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYyLWNoZW5wLTIwMDYtY29ycmVzcG9uZG5lY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">862-chenp-2006-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYyLWNoZW5wLTIwMDYtZGVzY3JpcHRpb24oY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">862-chenp-2006-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYyLWNoZW5wLTIwMDYtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">862-chenp-2006-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYyLWNoZW5wLTIwMDYtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">862-chenp-2006-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYyLWNoZW5wLTIwMDYtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">862-chenp-2006-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYyLWNoZW5wLTIwMDYtcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">862-chenp-2006-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="232255-6-halogeno-1-2-4-triazolo-1-5-a-pyrimidines-for-combating-animal-pests.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="232257-a-method-and-apparatus-for-presenting-at-least-a-part-of-an-object.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>232256</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>862/CHENP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>16-Mar-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>10-Mar-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>ERASMUS UNIVERSITY MEDICAL CENTER ROTTERDAM</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>Dr. Molewaterplein 50, NL-3015 GD Rotterdam,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>STAAL, Frank, Jakob, Theodor</td>
											<td>Oosteinde 167, NL-2611 VD Delft,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>VAN DONGEN, Jacobus, Johannes, Maria</td>
											<td>Parallelweg Zuid 147, 2914 LE Nieuwerkerk aan den IJssel,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>G01N 33/574</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/NL2004/000562</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-08-11</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>03077529.0</td>
									<td>2003-08-12</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/232256-method-for-detecting-low-levels-of-a-fusion-protein by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:03:30 GMT -->
</html>
