<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/232658-a-seminested-polymerase-chain-reaction-pcr-method-for-rapid-detection-and-specific-identification-of-mycobacterium-fortuitum by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:16:34 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 232658:A SEMINESTED POLYMERASE CHAIN REACTION (PCR) METHOD FOR RAPID DETECTION AND SPECIFIC IDENTIFICATION OF MYCOBACTERIUM FORTUITUM</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A SEMINESTED POLYMERASE CHAIN REACTION (PCR) METHOD FOR RAPID DETECTION AND SPECIFIC IDENTIFICATION OF MYCOBACTERIUM FORTUITUM</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A seminested Polymerase Chain Reaction (PCR) method for rapid detection and specific identification of Mycobacterium fortuitum (rapid grower of Mycobacteria) using species-specific primers derived from the internal transcribed spacer (ITS) sequences between 16S and 23 rRNA, of the said mycobacterium, the said primer set comprising the oligonucleotides VRFI 5&#x27; AGTGAGGCAACAACACGCCG - 3&#x27; (Complement of 178-197) designed from the available DNA sequence of M. fortuitum accession number AF 144326--of the ITS region for detection of M. , fortuitum , the said primers being designed such that PCR products obtained using the primers have two definite sizes of 222 bp (base pair) at the end of the I round of amplification and 153 bp (base pair) at the end of the lI round of amplification for easy discrimination thereof by gel electrophoresis.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>This invention relates to a seminested polymerase chain reaction (PCR) method and oligonucleotides for rapid detection and specific identification of Mycobacterium fortuitum (M.fortuitum).<br>
Culture in liquid and solid media remains the gold standard, but it needs three to eight weeks to obtain a result, and the microscopic detection of acid fast bacilli requires large number of bacteria (106/mL) in the sample.<br>
These conventional biochemical methods and phenotypic tests for species differentiation are laborious and time consuming and frequently require specialized testing that is beyond the capacity of clinical laboratories In addition it may take several weeks after reception of specimens and the tests sometimes fail to produce a precise identification and also reproducible results.<br>
The rise in incidence of nontuberculous mycobacteria (NTM) including newly described species or subspecific phylogenetic lineages are of potential clinical significance and me crucial role of the laboratory demand methods that provide accurate results in a more timely fashion..<br><br>
Several techniques have been used to replace time-consuming biochemical tests. Hybridization with specific probes is a rapid and reliable method, but has been applicable only to a limited range of mycobacterial species. High performance liquid chromatography (HPLC) of mycolic acid or Thin Layer Chromatography (TLC) of mycobacterial lipids can be used to differentiate most species of mycobacteria, but requires special equipment and standardization of growth and working condition, and remain limited to the reference laboratories.<br>
To meet the necessity for more rapid species identification and to improve the accuracy of identification of mycobacteria, methods utilizing the amplification of DNA by PCR coupled with restriction enzyme digestion, nucleic acid sequence determination have been developed. Molecular methods (genotypic methods) have been developed for differentiation of mycobacteria and they are rapid with unequivocal results. It has more rapid turnaround time and improved accuracy of identification.<br>
The Internal Transcribed Spacer (ITS) Region is a sequence located between 3' end of the 16S and 5' end of the 23S coding region inside the mycobacterial ribosomal operon ITS between the 16S and 23S rRNA gene is approximately 270-360 base pairs but varies in size from species to species. The spacer sequence of slowly growing species is approximately 75 nucleotides shorter than those of rapid<br><br>
growers. The genes coding for the rRNA are arranged in the order 5'- 16S - 23S - 5S - 3', and they are separated by 2 non coding spacer regions.The ITS region is considered to be less prone to selective pressure and consequently be expected to have accumulated a higher percentage of mutations than the corresponding rDNA. Sequencing of the ITS region of diverse bacteria indicates that considerable length and primary sequence variation occurs and this variability has been successfully used to distinguish between closely related mycobacteria.<br>
Many of the PCR assays used for detecting mycobacteria involve species-specific primers targeting the 16S rRNA, hsp65, 32-kDa protein genes, or the ITS between the 16S and23S rRNA genes is approximately 270 to 360 bp but varies in size from species to species. It is considered to be a suitable target for probes with which additional phylogenetic information can be derived. Furthermore, the ITS is suitable for differentiating species of mycobacteria and potentially can be used to distinguish clinically relevant subspecies. With respect to mycobacteria, both the high level of spacer sequence variation and the good reproducibility of ITS sequencing suggest the applicability of this approach. Rapid identification of species of mycobacteria is an important factor.<br><br>
However, it is not easy to identify species of mycobacteria, especially NTM.<br>
The novel primers that Parker et al ( JCM 2000 38 4080-4085) designed could be used to identify mycobacteria and the protocol facilitated the early and accurate detection of mycobacteriosis. This study demonstrated that an ITS-based PCR method has a high degree of sensitivity and specificity for the detection and identification of important species of mycobacteria. The ITS sequence between the 16S rRNA and 23S rRNA genes, which is more variable than the 16S rRNA gene itself, has been shown to be species specific in many microorganisms<br>
Thus when we evaluate the molecular methods of detection with<br>
that of the conventional techniques, molecular techniques are<br>
superior to conventional techniques in terms of rapidity sensitivity<br>
and reliability.<br>
There are no rapid sensitive PCR techniques available to directly<br>
detect the genome of M.fortuitum<br>
Thus most conventional methods for identifying and classifying bacteria are based on the morphological, biochemical, and growth characteristics of bacteria. These conventional methods are very tedious and complicated to conduct and take much time. Due to<br><br>
these problems, use of a simple and rapid identification method using a gene as a target sequence becomes more common. For an emerging gene-based identification method, genus-specific or species-specific PCR primers or nucleotide probes are applied to a gene of interest.<br>
To the best of our knowledge, till date there is no seminested PCR approach mentioned in literature to detect the genomes of fortuitum directly from clinical specimens using primers targeting ITS region.<br>
We have established a new approach by exploiting the seminested PCR approach by using  a new sequence of primer within the ITS region to increase the sensitivity of PCR VRF1 VRFi5' AGTGAGGCAACAACACGCCG -3' (complement of 178-197) designed from the available DNA sequence of M. fortuitum accession number: AF 144326-of the ITS region for detection of M. fortuitum<br>
The present invention relates to a new seminested Polymerase Chain Reaction (PCR) method  and oligonucleotides for rapid and specific identification of rapid growers of mycobacteria -fortuitum using species-specific primers derived from the ITS sequences of mycobacteria.<br>
The ITS region is considered to be less prone to selective<br><br>
pressure and consequently be expected to have accumulated a higher percentage of mutations than the corresponding ribosomal DNA. Sequencing of ITS region of diverse bacteria indicates that considerable length and primary sequence variation occurs and this variability has been successfully used to distinguish between closely related mycobacteria.<br>
It is an object of the present invention to develop rapid, reliable sensitive seminested PCR using specific new sequences of primers coding for the ITS between the 16S and 23 rRNA for the rapid detection and also identification of M. fortuitum species of rapid growers of mycobacteria.<br>
It is another object of this invention to provide a new sequence of oligonucleotides for a seminested PCR method capable of amplifying M. fortuitum<br>
It is yet another object of this invention to provide the optimum reaction mixture and thermal profile for the seminested PCR method capable of amplifying M. fortuitum.<br>
The seminested PCR primer set comprises the oligonucleotide having      the       new       species      specific primer<br><br>
VRF1     VRF^'AGTGAGGCAACAACACGCCG-S'<br>
(Complement of 178-197) been designed from the available DNA sequence of M. fortuitum accession number: AF 144326-of the ITS region for detection of M. fortuitum.<br>
The  seminested PCR method  and oligonucleotides for rapid detection and specific identification of rapid growers of M. fortuitum species, according to this invention, uses species-specific primers derived from the internal transcribed spacer (ITS) sequences between 16S and 23SrRNA, of the said mycobacterium, the said primer set comprising the oligonucleotides   VRF1 VRFj 5' AGTGAGGCAACAACACGCCG - 3' (Complement of 178-197) designed from the available DNA sequence of M. fortuitum accession number : AF 144326-of the ITS region for detection of M fortuitum, the said primers being designed such that PCR products obtained using the primers have two definite sizes of 222 bp (base pair) at the end of the I round of amplification and 153 bp (base pair) at the end of the II round of amplification for easy discrimination thereof by gel electrophoresis.<br>
The seminested PCR method proposed herein is developed for identifying rapid growers of mycobacteria in which nucleotide sequences of an ITS region between the 16S r RNA and 23 S rRNA genes, which has genus-and species-specific sequences for mycobacteria, are used as a target sequence.<br><br>
In other words, from the ITS region between the 16S rRNA and 23S rRNA genes, which contain both conserved and polymorphic sequences for mycobacteria, a genus-specific primer having the conserved sequence and species-specific primers having the polymorphic sequences are derived. One conserved sequence region is used as a common reverse primer, and four species-specific primers combined with the one genus-specific primer are used as forward primers to obtain PCR products having different sizes. As a result, non-tuberculosis mycobacteria (NTM) species can be identified. These primers are designed such that PCR products obtained using the primers have different sizes for easy discrimination thereof by gel electrophoresis. The new primer sequence is  applied to a seminested PCR to detect M.fortuitum .<br>
The PCR primers according to the present invention are synthesized from an ITS sequence of mycobacteria. The seminested PCR method according to the present invention for detecting the M. fortuitum species through a seminested PCR reaction uses three primers including species-specific forward primers and one common species -specific reverse primer. For the seminested PCR method according to the present invention for detecting the M. fortuitum in the seminested PCR a combination of appropriate primers, the reaction conditions should be further restricted when designing the<br><br>
primers. In addition, the primers should be designed such that the amplification products have different sizes and can be distinguished from one another on a gel after the PCR.<br>
The two new primer sequences by seminested PCR can detect M. fortuitum species through a seminested PCR with high sensitivity.<br>
Preparation of Primers M. fortuitum : In this semi nested PCR a<br>
new       set       of	primer       sequence       VRFi       5'<br>
AGTGAGGCAACAACACGCCG - 3' (Complement of 178-197) has been designed from the available DNA sequence of M. fortuitum accession number : AF 144326--of the ITS region. The specificity of the sequence was confirmed by the 'Blast search" that this sequence is specific for M. fortuitum. The forward primer used for the uniplex PCR was combined with the newly designed primers to modify the uniplex PCR to seminested PCR. The thermal cycling profile also has been modified for optimizing the sensitivity of the PCR amplification and by including Tetra Methyl Ammonium Chloride (TMAC) mispriming or nonspecific amplification also has been reduced.<br>
The primer set for the I round of amplification for M. fortuitum.<br>
Forward primer 5' CCGTGAGGAACCGGTTGCCT3' (residues 45 -64)<br>
Reverse primer 5' CCA CACGATTCTGCGTCGTA3'(complement of residues 247-266)<br><br>
The expected amplified product at the end of I round : 222 base Pair.<br>
The constituents of PCR mixture were 0.2 mM each deoxy nucleotide tri phosphate (d ATP, d TTP, d GTP, d CTP) 10 pM of each primer. 10 x assay buffer IU Taq DNA Polymerse 5uM TMAC Milli Q water. Template DNA. The thermal cycle profile for the I round of amplification was with the initial denaturation at 94°C for 5 minutes followed by 25 cycles of denaturation at 94°C for 1 minute, annealing at 60°C for 1 minute and extension at 72°C for 1 minute and final extension at 72°C for 10 minutes. To increase the sensitivity VRFj primer sequence was designed by targeting the ITS region between 16S-23SrRNA of M.fortuitum. 5'AGTGAGGCAACAACACGCCG 3M (complement of 178-197) The constituents of PCR mixture were be as for the first round : 0.2mM each deoxy nucleotide tri phosphate (d ATP, d TTP, d GTP, d CTP) 10 pM of each primer 10 x assay buffer IU Taq DNA Polymerase 5uM TMAC Milli Q water Template DNA.<br>
II round of Semi nested PCR for M. fortuitum<br>
To reamplify mycobacterial DNA, 5uM of the initial amplification products were transferred into sterile vial containing the same cocktail described above except that of reverse primer. Forward primer was same for first and second round PCR. The samples were then amplified for an additional 35 cycles of<br><br>
denaturation at 94°C for 30 seconds, annealing 60°C for 1 minute<br>
and extension at 72°C for 30 seconds minute and final extension at<br>
72°C for 10 minutes.<br>
Forward primers:	5' CCGTGAGGAACCGGTTGCCT 3'<br>
(residues 45 -64)<br>
Reverse primer VRFi 5'AGTGAGGCAACAACACGCCG 3'<br>
( complement of residues 178-197)<br>
The expected amplified product at the end of IT round : 153 base pair.<br>
Detection of Amplified Product:<br>
Agarose gel Electrophoresis:<br>
2% Agarose gel in lx TBE Buffer was used for the detection of<br>
amplified product. 2mg/ml of Ethidium Bromide was added to the molten agarose in a final concentration of 0.5uM/ml. The molecular weight marker, negative and positive controls were also be loaded along with the product Electrophoresis was carried out at 100V for 30-45 min.  The gel was visualized on UV transilluminator at 305 nm wavelength<br><br>
Example 1<br>
Sensitivity of the Semi nested PCR for M. fortuitum :<br>
The sensitivity of amplification was determined by amplifying DNA from Mycobacterium fortuitum ATCC strain (15ug- 150fg) of template DNA. 10 fold  dilutions of template DNA was made and PCR was done.  Amplicon were analyzed by 2% agarose gel electrophoresis. For the detection of sensitivity lul of template DNA was diluted to 1 ml with Milli Q water and reading was taken at 260 nm spectrophotometrically. One OD corresponds to 50 ug/ml of DNA.  The concentration of DNA was calculated from the corresponding OD value. The sensitivity of the I round primers was50ng of M. fortuitum DNA and at the end of II round of amplification it was 10 pg of DNA.<br>
The constituents of PCR mixture were 0.2 mM each deoxy nucleotide tri phosphate (d ATP, d TTP, d GTP, d CTP) 10 pM of each primer. 10 x assay buffer IU Taq DNA Polymerse 5uM TMAC Milli Q water Template DNA. The thermal cycle profile for the I round of amplification was with the initial denaturation at 94°C for 5 minute, followed by 25 cycles of denaturation at 94°C for 1 minute, annealing 52°C for 1 minute and extension at 72°C for 1 minute and final extension at 72°C for 10 minutes.<br><br>
II round of seminested PCR : To reamplify mycobacterial DNA, 5uM of the initial amplification products were transferred into sterile vial containing the same cocktail described above except that of reverse primer. Forward primer was same for first and second round PCR. The samples were then be reamplifled an additional 35 cycles of denaturation at 94°C for 30 seconds, annealing 52°C for 1 minute and extension at 72°C for 30 seconds minute and final extension at 72°C for 10 minutes.<br>
Industrial Applicability<br>
As. described above, in the seminested PCR method according to the present invention, the species can be identified. The seminested PCR method according to the present invention ensures rapid and accurate identification at low cost without conducting biochemical and other tests for definite identification to species level.<br>
The Seminested PCR method according to the present invention can be efficiently applied directly on the clinical specimens for the detection of the rapid grower of mycobacteria at low cost.<br><br><br><br><br><br><br>
We Claim:<br>
1.	A seminested Polymerase Chain Reaction (PCR) method  for rapid<br>
detection and specific identification of Mycobacterium fortuitum (rapid<br>
grower of Mycobacteria) using species-specific primers derived from the<br>
internal transcribed spacer (ITS) sequences between 16S and 23 rRNA, of the<br>
said mycobacterium, the said primer set comprising the oligonucleotides<br>
VRFj 5' AGTGAGGCAACAACACGCCG - 3' (Complement of 178-197)<br>
designed from the available DNA sequence of M fortuitum accession number<br>
: AF 144326-of the ITS region for detection of M. fortuitum, the said primers being designed such that PCR products obtained using the primers have two definite sizes of 222 bp (base pair) at the end of the I round of amplification and 153 bp (base pair) at the end of the II round of amplification for easy discrimination thereof by gel electrophoresis.<br>
2.	A method as claimed in Claim 1 wherein three primers are used including species-specific forward primers and one common species-specific reverse primer, the primers being designed such that the amplification products have the said two definite sizes and are distinguishable from one another on a gel.<br>
3.	A method as claimed in Claim 1 or Claim 2 wherein the specificity of the DNA sequence for M. fortuitum was confirmed by a blast search.<br><br>
4.	A method as claimed in any one of the preceding Claims wherein the forward primer used for the uniplex PCR is combined with the said species-specific primers derived from the ITS sequences of the said mycobacterium to modify the uniplex PCR to seminested PCR.<br>
5.	A method as claimed in any one of the preceding Claims wherein the thermal cycling profile is modified for optimizing the sensitivity of the PCR amplification.<br>
6.	A method as claimed in any on of die preceding Claims wherein Tetra Methyl Ammonium Chloride is used for reducing mispriming or nonspecific amplification.<br>
7.	A method as claimed in any one of the preceding Claims wherein the<br>
primer set for the I round of amplification:<br>
Forward primer: 5'CCGTGAGGAACCGGTTGCCT3' (residues 45-64) Reverse primer:  5' CCA CACGATTCTGCGTCGTA3'(complement of residues 247-266), the amplified product at the end of I round being 222<br>
base pair<br>
8.	A method as claimed in any one of the preceding Claims wherein the<br>
thermal cycle profile for the I round of amplification is with initial<br>
denaturation at 94 deg. C for 5 minutes followed by 25 cycles of<br><br>
denaturation at 94 deg C for 1 minute, annealing at 60 deg C for 1 minute and extension at 72 deg C for 1 minute and final extension at 72 deg C for 10 minutes.<br>
9 A method as claimed in any one of the preceding Claims wherein to reamplify mycobacterial DNA 5uM of the initial amplification products is transferred into a sterile vial containing the same cocktail as given above except that of the reverse primer, the forward primer being the same for the first and second round PCR, the samples being then amplified for an additional 35 cycles of denaturation 94 deg C for 30 seconds, annealing 60 deg C for 1 minute and extension at 72 deg C for 30 seconds and final extension at 72 deg C for 10 minutes<br>
10. A method as claimed in any one of the preceding Claims wherein the<br>
primer set for the II round of amplification is<br>
Forward primer: 5' CCGTGAGGAACCGGTTGCCT 3' (residues 45 -64)<br>
Reverse primer VRFj 5'AGTGAGGCAACAACACGCCG 3'<br>
(complement of residues 178-197)<br>
the amplified product at the end of the II round being 153 base pair<br><br>
11.	A composition comprising a combination of oligonucleotides as set out<br>
in the preceding Claims 1 to 10.<br>
12.	A kit comprising a composition as claimed in Claim 11.<br>
13.	A seminested Polymerase Chain Reaction (PCR) method for rapid<br>
detection and specific identification of Mycobacterium fortuitum (rapid<br>
grower of mycobacteria) substantially as herein described and illustrated by<br><br><br>
A method of producing a recombinant Chrysosporium strain comprising the steps of encoding a desired polypeptide sequence in the nucleic acid sequence of a Chrysosporium strain, said nucleic acid sequence being operably linked to expression-regulating region and optionally to a secretion sequence, to produce a recombinant strain expressing said desired polypeptide at a higher level than the corresponding non-recombinant strain under the same conditions.<br>
The method as claimed in claim 1, wherein at least one heterologous nucleic acid sequence selected from heterologous polypeptide-encoding nucleic acid sequences, heterologous signal sequences and heterologous expression-regulating sequences is introduced Lito said strain.<br>
The method as claimed in claim 1, wherein said desired polypeptide is a heterologous polypeptide of plant, animal (including human), algal, bacterial, archaebacterial, viral or fungal origin.<br>
The method as claimed in claim 1, wherein said desired polypeptide is a homologous polypeptide which is expressed at a higher level than in the corresponding non-recombinant strain under the same conditions.<br>
The method as claimed in claim 1, wherein said desired polypeptide is selected from carbohydrate-degrading enzymes, proteases, lipases, esterases, other hydrolases, oxidoreductases and transferases.<br>
The method as claimed in claim 1, wherein said desired polypeptide is selected from fungal enzymes allowing (overproduction of primary metabolites, organic acids, secondary metabolites, and antibiotics.<br><br>
The method as claimed in claim 1, wherein said desired polypeptide exhibits optimal activity and/or stability at a pH above 6, and/or has more than 50% of its activity and/or stability at a pH above 6.<br>
The method as claimed in claim 1, wherein the recombinant Chrysosporium strain produced has a heterologous or homologous signal sequence.<br>
The method as claimed in claim 1, wherein the recombinant Chrysosporium strain has a fungal, e.g. ascomycete, signal sequence.<br>
The method as claimed in claim 1, wherein the recombinant Chrysosporium strain has a fungal signal sequence is a signal sequence of a cellulase, 6-galactosidase, xylanase, pectinase, esterase, protease, amylase, polygalacturonase or hydrophobin.<br>
The method as claimed in claim 1, wherein a selectable marker, such as a marker conferring resistance to a drug or relieving a nutritional defect is introduced in the strain in a known manner.<br>
The method as claimed in claim 1, wherein a heterologous or homologous expression-regulating region, preferably a fungal expression-regulating sequence is introduced.<br>
The method as claimed in claim 12, wherein the expression-regulating region is an inducible or a constitutive promoter.<br><br>
The method as claimed in claim 12 or 13, wherein the expression-regulating region is a fungal cellobiohydrolase, gluco-amylase, glyceraldehyde phosphate dehydrogenase, alcohol dehydrogenase A, alcohol dehydrogenase R, phosphoglycerate, aspartic proteinase, lipase, beta-galactosidase, hydrophobin, protease, amylase, xylanase, pectinase, esterase, endo-glucanase or polygalacturonase promoter.<br>
The method as claimed in claim 1, wherein said recombinant strain is obtained by transformation of a mutant strain, said mutant strain being obtained by mutagenesis steps including at least one of UV irradiation and chemical mutagenesis, a first UV irradiation step, a N-methyl-N'-nitro-N-nitrosoguanidine treatment step, and a second UV irradiation step.<br>
The method as claimed in claim 15, wherein said mutant strain is derived from Chrysosporium lucknowense, especially from C. lucknowense strain CI (VKM F-3500 D).<br>
The method as claimed in claim 16, said mutant corresponding to or being derived from one of Chrysosporium lucknowense mutant strains UV13-6 (VKM F-3632 D), NG7C-19 (VKM F-3633 D), and UV18-25 (VKM F-3631 D).<br>
The method as claimed in any of the preceding claims, wherein said strain exhibits a biomass less than half that of Trichoderma reesei, said Trichoderma in culture exhibits a viscosity of 200-600 cP when cultured under equivalent conditions.<br><br>
The method as claimed in any of the preceding claims, wherein said strain is capable of producing at least the amount of cellulase in moles per liter as produced by any of the Chrysosporium lucknowense mutant strains CI (VKM F-3500 D), UV13-6 (VKM F-3632 D), NG7C-19 (VKM F-3633 D), and UV18-25 (VKM F-3631 D).<br>
The method as claimed in any of the preceding claims, wherein said strain is capable of producing less protease than produced by the Chrysosporium lucknowense strain CI (VKM F-3500 D), preferably less than half the amount produced by said CI strain.<br>
A method of producing a polypeptide comprising culturing a strain produced as claimed in any one of claims 1 to 20 under known conditions permitting expression and preferably secretion of the protein or polypeptide and recovering the subsequently produced polypeptide by known means.<br>
The method as claimed in claim 21, wherein a cleavage step of a precursor of said polypeptide into the polypeptide or precursor of interest, a cleavage with a Kex-2 like protease, any basic amino acid paired protease or Kex-2 is carried out.<br>
The method as claimed in claim 21 or 22, wherein the cultivation is effected at pH in the range 4-9, and/or at a temperature between 25 and 43°C.<br>
The method for producing a recombinant Chrysosporium strain as claimed in any of claims 1 to 20, wherein a nucleic acid sequence encoding a heterologous or homologous polypeptide is stably introduced into a Chrysosporium strain, said nucleic acid sequence being operably linked to an expression regulating region.<br><br>
The method as claimed in claim 24, wherein the transformation method is the protoplast transformation method.<br>
The method as claimed in claims 1 to 20, 24 and 25, wherein a Chrysosporium xylanase of the xylanase F family, having a pi of 9,1, an MW of about 30 kD by SDS PAGE, and having at least 75% amino acid identity over a stretch of 120 amino acids with the amino acid sequence depicted in SEQ ID No. 5 is produced.<br>
The method as claimed in claim 26, wherein a mutant microbial strain corresponding to or being derived from one of mutant strains UV13-6 (VKM F-3632 D), NG7C-19 (VKM F-3633 D), and UV18-25 (VKM F-3631 D) is produced.<br>
The method as claimed in claim 27, wherein said strain exhibiting a biomass less than half that of Trichoderma reesei, with said Trichoderma in culture exhibiting a viscosity of 200-600 cP when cultured under equivalent conditions is produced.<br>
The method as claimed in claim 27 or 28, wherein a strain producing less than half the amount of protease produced by the parent strain from which said mutant was derived is obtained.<br>
The method as claimed in claims 1 to 20 and 27 to 29, wherein the recombinant microbial strain has a nucleic acid sequence encoding a polypeptide of interest, said nucleic acid sequence being operable linked to an expression regulating region and optionally a secretion signal sequence.<br><br>
A method of producing a recombinant Chrysosporium strain, substantially as hereinbefore described with reference to the Examples.<br>
A method of producing a polypeptide, substantially as hereinbefore described with reference to the Examples.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDkzNC1jaGUtMjAwNSBhYnN0cmFjdC1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">0934-che-2005 abstract-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDkzNC1jaGUtMjAwNSBjbGFpbXMtZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">0934-che-2005 claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDkzNC1jaGUtMjAwNSBkZXNjcmlwdGlvbiAoY29tcGxldGUpLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">0934-che-2005 description (complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTM0LWNoZS0yMDA1LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">934-che-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTM0LWNoZS0yMDA1LWNvcnJlc3BvbmRlbmNlIG90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">934-che-2005-correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTM0LWNoZS0yMDA1LWRpc2NyaXB0aW9uIGNvbXBsZXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">934-che-2005-discription complete.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTM0LWNoZS0yMDA1LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">934-che-2005-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTM0LWNoZS0yMDA1LWZvcm0gMjYucGRm" target="_blank" style="word-wrap:break-word;">934-che-2005-form 26.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="232657-a-method-for-producing-a-cutting-plate-made-from-pcbn-or-cbn-composite-material.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="232659-a-polymerization-process.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>232658</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>934/CHE/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>20-Mar-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>12-Jul-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>VISION RESEARCH FOUNDATION</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>SANKARA NETHRALAYA, 18 COLLEGE ROAD, CHENNAI 600 006,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>KULANDAI LILY THERESE</td>
											<td>SANKARA NETHRALAYA, 18 COLLEGE ROAD, CHENNAI 600 006,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>JASMIN THERESE</td>
											<td>SANKARA NETHRALAYA, 18 COLLEGE ROAD, CHENNAI 600 006,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>HAJIB NARAHARIRAO MADHAVAN</td>
											<td>SANKARA NETHRALAYA, 18 COLLEGE ROAD, CHENNAI 600 006,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q01/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/232658-a-seminested-polymerase-chain-reaction-pcr-method-for-rapid-detection-and-specific-identification-of-mycobacterium-fortuitum by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:16:35 GMT -->
</html>
