<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/233093-a-substantially-purified-nuclelc-acid-molecule-involved-in-elevation-of-oil-levels-in-plants by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:31:07 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 233093:&quot;A SUBSTANTIALLY PURIFIED NUCLELC ACID MOLECULE INVOLVED IN ELEVATION OF OIL LEVELS IN PLANTS&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A SUBSTANTIALLY PURIFIED NUCLELC ACID MOLECULE INVOLVED IN ELEVATION OF OIL LEVELS IN PLANTS&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A substantially purified nucleic acid molecule used interalia for making transgenic maize plants and seeds, selected from the group consisting of: a) a nucleic acid molecule comprising SEQ ID NO: 1 or the complement thereof. b) a nucleic acid molecule comprising SEQ ID NO: 11 of the complement thereof; c) a nucleic acid molecule which encodes a polypeptide having at least 95% amino acid identity with SEQ ID No.: 3.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>ELEVATION OF OIL LEVELS IN PLANTS<br>
This application claims the benefit of the filing date of the Provisional Application<br>
U.S. Serial Number 60/483,491, filed June 27, 2003, which is incorporated herein by<br>
reference.<br>
The present invention relates to the fields of nucleic acid chemistry and agricultural<br>
biotechnology. In particular, the present invention is directed at the identification of nucleic<br>
acids that encode proteins useful for increasing oil levels in maize plants and creating maize<br>
plants that include such nucleic acids.<br>
Plants are a major source of oils for feed, food, and industrial uses. While tissues of<br>
most plant species contain little oil, the cultivation of certain plant types, over many acres,<br>
permit large quantities of plant oils to be produced. If the oil content of these plants could be<br>
increased, then plant oils could be produced more efficiently. For example, the normal oil<br>
content of yellow #2, dent com is about 4%. If the oil content of corn could be increased to<br>
8% or even 12%, without significantly affecting yield, the same amount of oil could be<br>
produced from half or even one-third the number of acres.<br>
Currently, levels of oil in oilseed crops have increased incrementally by traditional<br>
breeding and selection methods. There exist few references to transgenic plants with<br>
increased levels of oil. In contrast, increases in the proportions of some strategic fatty acids<br>
have been achieved by the introduction or manipulation of various plant fatty acid<br>
biosynthesis genes in oilseeds. For instance, Voelker el al., Science, 257:72-74 (1992),<br>
demonstrated that expression in Brassicaceae of a medium chain fatty acyl-ACP thioesterase<br>
from California Bay, increased the lauric acid (12:0) content. Hitz el al., Proc. 9th<br>
International Cambridge Rapeseed Congress UK, pp 470-472 (1995) increased proportions of<br>
oleic acid in Glycine max by co-suppression using a sense construct encoding a plant<br>
microsomal FAD-2 (A 12) desaturase. Although the use of these plant transgenes resulted in<br>
an increased production of lauric acid in canola and altered proportions of oleic acid in soy,<br>
there was no evidence of increased total fatty acid content, or increased oil yield in these<br>
transgenics.<br>
Certain workers have attempted to increase or modulate the oil content of plants by<br>
manipulation of oil biosynthetic pathway genes. For example, U.S. Patent 6,268,550 to<br>
Gengenbach et al. provides maize acetyl CoA carboxylase nucleic acids for altering the oil<br>
content of plants. Additionally, U.S. Patent 5,925,805 to Ohlrogge el al. provides an<br>
Arabidopsis acetyl CoA carboxylase gene that can be used to increase the oil content of<br>
plants. However, the synthesis of fatty acids requires the coordinated activity of many<br>
enzymes, none of which when solely upregulated has been found to substantially increase oil<br>
content.<br>
A need therefore exists for an improved method to alter the oil content of plants, and<br>
in particular to increase the oil content of plants and seeds.<br>
In addition to oil, starch from maize is also agriculturally and commercially<br>
significant. Starch comprises a major component of animal feed and human food. Starch is<br>
also used industrially in the production of paper, textiles, plastics, and adhesives, as well as<br>
providing the raw material for some bioreactors.<br>
In higher plants, the starch consists of linear chain and branched chain glucans known<br>
as amylose and amylopectin, respectively. Starch with various amounts of amylose and<br>
amylopectin are found in different plants. Typically, maize starch contains approximately<br>
25% amylose, the remainder being amylopectin. Amylopectin contains short chains and long<br>
chains, the short chains ranging from 5-30 glucose units and the long chains ranging from<br>
30-100 glucose units, or more. The ratio of amylose to amylopectin, as well as the<br>
distribution of short to long chains in the amylopectin fraction, affect the physical properties<br>
of starch, (e.g., thermal stabilization, retrogradation, and viscosity).<br>
The WAXY locus of maize determines the amylose content in pollen and in kernel<br>
endosperm, (Shure et al., Cell, 35(l):225-233 (1983)), resulting in starch having unique<br>
properties. Most mutations in the WAXY locus of maize, which encodes granule bound<br>
starch synthase (GBSS), result in an opaque endosperm of smooth, firm non-corneous starch<br>
comprising mostly amylopectin and a reduced amount of amylose in the endosperm, pollen<br>
and embryo sac ("WAXY phenotype") (see, Okagaki and Wessler, Genetics,<br>
120(4): 1137-1143 (1988)). When no functioning GBSS is synthesized in the homozygous<br>
WAXY mutant, it also lacks amylose (Echt and Schwartz, Genetics, 99:275-284 (1981)).<br>
Additionally, classic, recessive WAXY has a small (approximately 0.5% increase)<br>
effect on percent oil in the kernel when compared to yellow #2 com (Pfahler and Linskens,<br>
Theoretical and Applied Genetics, 41(l):2-4 (1971)). In comparison, the inbred line HOI001,<br>
a dominant WAXY mutant inbred described in U.S. Patent Publication No. 20030172416,<br>
herein incorporated by reference, has whole kernel oil concentrations greater than four times<br>
that of yellow #2 com.<br>
SUMMARY OF THE INVENTION<br>
The present invention describes and provides isolated nucleic acid molecules encoding<br>
an HOI001 GBSS polypeptide. In addition, this invention relates to nucleic acid molecules<br>
that are complementary to the nucleic acid molecule encoding an HOI001 GBSS polypeptide.<br>
In addition, this invention relates to expression cassettes comprising these nucleic acid<br>
molecules. In addition, this invention relates to transgenic maize plants containing these<br>
expression cassettes. In addition, this invention relates to the seeds of these transgenic maize<br>
plants. This invention further relates to the oil and animal feed obtained from the seeds of<br>
these transgenic maize plants.<br>
In another embodiment, the present invention relates to a recombinant DNA construct,<br>
associated with increased oil production in plants, comprising a nucleic acid molecule<br>
encoding an HOI001 GBSS polypeptide operably linked to a promoter, which is functional in<br>
a plant cell.<br>
The present invention describes and provides a method of increasing oil in a maize<br>
plant by expression of an HOI001 GBSS gene. This invention further provides a method of<br>
altering the kernel composition in a corn plant by expression of an HOI001 GBSS gene. This<br>
invention further describes and provides sequences of an HOI001 GBSS gene from Zea mays.<br>
This invention further provides vector constructs for plant transformation and tissue-specific<br>
expression of an HOI001 GBSS gene. This invention further provides maize plants<br>
transformed with the GBSS gene with higher oil levels when compared to plants with the<br>
same or similar genetic background, but not containing the inserted HOI001 GBSS gene.<br>
This invention further provides seeds from these maize plants. This invention further<br>
provides for kernels from maize plants transformed with the HOI001 GBSS gene containing a<br>
higher level of oil when compared to kernels from corn plants with the same or similar genetic<br>
background, but not containing the inserted HOI001 GBSS gene. This invention also<br>
provides oil and animal feed produced from these seeds and kernels.<br>
The present invention further provides a method of marker-assisted breeding useful in<br>
breeding higher oil levels in maize.<br>
BRIEF DESCRIPTION OF THE FIGURES<br>
Figure 1 shows the nucleic acid sequence alignment of the granule bound starch<br>
synthase gene isolated from HOI001 (HOI001 GBSS, pMON72506) [SEQ ID NO: 1]<br>
compared to the granule bound starch synthase (GBSS) gene from inbred LH59<br>
(pMON72510), and published sequence of the GBSS gene described in Shure et al., supra,<br>
(X03935). For additional comparison, the coding sequence for the published GBSS gene is<br>
given (CDS22509).<br>
Figure 2 shows the alignment of the corresponding predicted aniino acid sequences<br>
from the GBSS gene isolated from HOI001 (HOI001 GBSS from pMON72506) [SEQ ID<br>
NO: 3], and the GBSS gene described in Shure et al, supra, [SEQ ID NO: 4], respectively.<br>
Figure 3 shows the alignment of the corresponding predicted amino acid sequences<br>
from the Zea mays GBSS gene isolated from inbred LH59 [SEQ ID NO: 10], and the Zea<br>
mays granule bound starch synthase gene described in Shure et al., supra, respectively.<br>
Figure 4 depicts a plasmid map of pMON72506.<br>
Figure 5 depicts a plasmid map of pMON72510.<br>
Figures 6A and 6B graphically depict the difference in oil levels from kernels of plants<br>
transformed with pMON72506 containing the GBSS from HOI001 (SEQ ID NO: 1, 6A) and<br>
pMON72510 containing the GBSS from LH59 (SEQ ID NO: 8, 6B). Gene positive and gene<br>
negative kernels are compared from each event. Only events with statistically significant<br>
changes in oil (14 of 29) are shown in 6A.<br>
Figure 7 depicts a plasmid map of pMON81464.<br>
Figure 8 depicts a plasmid map of pMON68298.<br>
Figure 9 depicts a plasmid map of pMON81465.<br>
BRIEF DESCRIPTION OF THE SEQUENCES<br>
SEQ ID NO: 1 is the nucleic acid sequence of the granule bound starch synthase from<br>
HOI001 (HOI001 GBSS from pMON72506).<br>
SEQ ID NO: 2 is the published nucleic acid sequence of Zea mays GBSS from Shure<br>
et al., supra.<br>
SEQ ID NO: 3 sets forth the predicted amino acid sequence of HOI001 GBSS from<br>
pMON72506.<br>
SEQ ID NO: 4 sets forth the predicted amino acid sequence from the Zea mays GBSS<br>
as published by Shure et al., supra.<br>
SEQ ID NO: 5 is a primer sequence for Primer number 14543.<br>
SEQ ID NO: 6 is a primer sequence for Primer number 14547.<br>
SEQ ID NO: 7 sets forth a nucleic acid sequence of a DNA molecule that encodes a<br>
GBSS from corn line LH59.<br>
SEQ ID NO: 8 sets forth the predicted amino acid sequence of GBSS from corn line<br>
LH59.<br>
SEQ ID NO: 9 is a primer sequence for Primer number 20095.<br>
SEQ ID NO: 10 is a primer sequence for Primer number 20092.<br>
SEQ ID NO: 11 sets forth the coding region of the GBSS cDNA of HOI001.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The following definitions are provided as an aid to understanding the detailed<br>
description of the present invention.<br>
The phrases "coding sequence," "coding region," "structural sequence," and<br>
"structural nucleic acid sequence" refer to a physical structure comprising an orderly<br>
arrangement of nucleotides. The nucleotides are arranged in a series of triplets that each form<br>
a codon. Each codon encodes a specific amino acid. Thus, the coding sequence, structural<br>
sequence, and structural nucleic acid sequence encode a series of amino acids forming a<br>
protein, polypeptide, or peptide sequence. The coding sequence, structural sequence, and<br>
structural nucleic acid sequence may be contained within a larger nucleic acid molecule,<br>
vector, or the like. In addition, the orderly arrangement of nucleotides in these sequences may<br>
be depicted in the form of a sequence listing, figure, table, electronic medium, or the like.<br>
The phrase "codon degeneracy" refers to divergence in the genetic code permitting<br>
variation of the nucleotide sequence without affecting the amino acid sequence of an encoded<br>
polypeptide. Accordingly, the instant invention relates to any nucleic acid fragment<br>
comprising a nucleotide sequence that encodes all or a substantial portion of the amino acid<br>
sequences set forth herein. The skilled artisan is well aware of the "codon-bias" exhibited by<br>
a specific host cell in usage of nucleotide codons to specify a given amino acid. Therefore,<br>
when synthesizing a nucleic acid fragment for improved expression in a host cell, it is<br>
desirable to design the nucleic acid fragment such that its frequency of codon usage<br>
approaches the frequency of preferred codon usage of the host cell.<br>
The term "cDNA" refers to a double-stranded DNA that is complementary to and<br>
derived from mRNA.<br>
The phrases "DNA sequence," "nucleic acid sequence," and "nucleic acid molecule"<br>
refer to a physical structure comprising an orderly arrangement of nucleotides. The DNA<br>
sequence or nucleotide sequence may be contained within a larger nucleotide molecule,<br>
vector, or the like. In addition, the orderly arrangement of nucleic acids in these sequences<br>
may be depicted in the form of a sequence listing, figure, table, electronic medium, or the like.<br>
"Expression" refers to the transcription of a gene to produce the corresponding mRNA<br>
and translation of this mRNA to produce the corresponding gene product (i.e., a peptide,<br>
polypeptide, or protein).<br>
"Expression of antisense RNA" refers to the transcription of a DNA to produce a first<br>
RNA molecule capable of hybridizing to a second RNA molecule, which second RNA<br>
molecule encodes a gene product that is desirably down-regulated.<br>
As used herein, "gene" refers to a nucleic acid fragment that expresses a specific<br>
protein, including regulatory sequences preceding (5' non-coding sequences) and following (3'<br>
non-coding sequences) the coding sequence. "Native gene" refers to a gene as found in nature<br>
with its own regulatory sequences. "Chimeric gene" refers to any gene that is not a native<br>
gene, comprising regulatory and coding sequences that are not found together in nature.<br>
Accordingly, a chimeric gene may comprise regulatory sequences and coding sequences that<br>
are derived from different sources, or regulatory sequences and coding sequences derived<br>
from the same source, but arranged in a manner different than that found in nature.<br>
"Endogenous gene" refers to a native gene in its natural location in the genome of an<br>
organism. An "exogenous gene" or "transgene" refer to a non-native gene that has been<br>
introduced into the genome by a transformation procedure.<br>
"Hemizygous" refers to a diploid individual having only one copy of a particular gene<br>
(for example, because a chromosome has been lost). "Homozygous" refers to a gene pair<br>
having identical alleles in two homologous chromosomes.<br>
"Heterologous" refers to the relationship between two or more nucleic acid or protein<br>
sequences that are derived from different sources. For example, a promoter is heterologous<br>
with respect to a coding sequence if such a combination is not normally found in nature. In<br>
addition, a particular sequence may be "heterologous" with respect to a cell or organism into<br>
which it is inserted (i.e., does not naturally occur in that particular cell or organism).<br>
"Homology" refers to the level of similarity between two or more nucleic acid or<br>
amino acid sequences in terms of percent of positional identity (i.e., sequence similarity or<br>
identity). Homology also refers to the concept of similar functional properties among<br>
different nucleic acids or proteins.<br>
"Hybridization" refers to the ability of a first strand of nucleic acid to join with a<br>
second strand via hydrogen bond base pairing when the two nucleic acid strands have<br>
sufficient sequence complementarity. As used herein, a nucleic acid molecule is said to be the<br>
"complement" of another nucleic acid molecule if they exhibit complete complementarity. As<br>
used herein, molecules are said to exhibit "complete complementarity" when every nucleotide<br>
of one of the molecules is complementary to a nucleotide of the other. Thus two nucleic acid<br>
strands are said to have sufficient complementarity when they can hybridize to one another<br>
with sufficient stability to permit them to remain annealed to one another under appropriate<br>
conditions.<br>
The phrases "marker-assisted selection" or "marker-assisted breeding" refer to the use<br>
of genetic markers to identify and select plants with superior phenotypic potential. Genetic<br>
markers determined previously to be associated with a trait locus or trait loci are used to<br>
uncover the genotype at trait loci by virtue of linkage between the marker locus and the trait<br>
locus. Plants containing desired trait alleles are chosen based upon their genotypes at linked<br>
marker loci.<br>
The phrase "breeding population" refers to a genetically heterogeneous collection of<br>
plants created for the purpose of identifying one or more individuals with desired phenotypic<br>
characteristics. The term "phenotype" refers to the observed expression of one or more plant<br>
characteristics.<br>
A "genetic marker" is any morphological, biochemical, or nucleic acid based<br>
phenotypic difference which reveals a DNA polymorphism. Examples of genetic markers<br>
include but are not limited to RFLPs, RAPDs, allozymes, SSRs, and AFLPs.<br>
The phrase "marker locus" refers to the genetically defined location of DNA<br>
polymorphisms as revealed by a genetic marker. A "trait locus" refers to a genetically defined<br>
location for a collection of one or more genes (alleles) which contribute to an observed<br>
characteristic.<br>
The phrase "restriction fragment length polymorphism" (RFLP) refers to a DNAbased<br>
genetic marker in which size differences in restriction endonuclease generated DNA<br>
fragments are observed via hybridization (Botstein et al., Am. J. Hum. Genet., 32:314-331<br>
(1980)).<br>
The phrase "random amplified polymorphic DNA" (RAPD) refers to a DNA<br>
amplification based genetic marker in which short, sequence arbitrary primers are used and<br>
the resulting amplification products are size separated and differences in amplification<br>
patterns observed (Williams et al., Nucleic Acids Res., 18:6531-6535 (1990)).<br>
The phrase "simple sequence repeat" (SSR) refers to a DNA amplification-based<br>
genetic marker in which short stretches of tandemly repeated sequence motifs are amplified<br>
and the resulting amplification products are size separated and differences in length of the<br>
nucleotide repeat are observed (Tautz, Nucleic Acids Res., 112:4127-4138 (1989)).<br>
The term "AFLP" refers to a DNA amplification-based genetic marker in which<br>
restriction endonuclease generated DNA fragments are ligated to short DNA fragments which<br>
facilitate the amplification of the restricted DNA fragments (Vos et al., Nucleic Acids Res.,<br>
23:4407-4414 (1995)). The amplified fragments are size separated and differences in<br>
amplification patterns observed.<br>
The phrase "operably linked" refers to the functional spatial arrangement of two or<br>
more nucleic acid regions or nucleic acid sequences. For example, a promoter region may be<br>
positioned relative to a nucleic acid sequence such that transcription of the nucleic acid<br>
sequence is directed by the promoter region. Thus, a promoter region is "operably linked" to<br>
the nucleic acid sequence.<br>
The terms "promoter" or "promoter region" refer to a nucleic acid sequence, usually<br>
found upstream (5') to a coding sequence that is capable of directing transcription of a nucleic<br>
acid sequence into mRNA. The promoter or promoter region typically provides a recognition<br>
site for RNA polymerase and the other factors necessary for proper initiation of transcription.<br>
As contemplated herein, a promoter or promoter region includes variations of promoters<br>
derived by inserting or deleting regulatory regions, subjecting the promoter to random or sitedirected<br>
mutagenesis, and the like. The activity or strength of a promoter may be measured in<br>
terms of the amounts of RNA it produces, or the amount of protein accumulation in a cell or<br>
tissue, relative to a second promoter that is similarly measured.<br>
The phrase "3" non-coding sequences" refers to nucleotide sequences located<br>
downstream of a coding sequence and include polyadenylation recognition sequences and<br>
other sequences encoding regulatory signals capable of affecting mRNA processing or gene<br>
expression. The polyadenylation signal is usually characterized by affecting the addition of polyadenylic acid tracts to the 3' end of the mRNA precursor. The use of different 3' noncoding<br>
sequences is exemplified by Ingelbrecht et al, Plant Cell, 1:671-680 (1989).<br>
"Translation leader sequence" or "5" untranslated region" or "5'-UTR" all refer to a<br>
nucleotide sequence located between the promoter sequence of a gene and the coding<br>
sequence. The 5'-UTR is present in the fully processed mRNA upstream of the translation<br>
start sequence. The 5'-UTR may affect processing of the primary transcript to mRNA, mRNA<br>
stability, or translation efficiency. Examples of translation leader sequences have been<br>
described (Turner and Foster, Molecular Biotechnology, 3:225(1995)).<br>
"RNA transcript" refers to the product resulting from RNA polymerase-catalyzed<br>
transcription of a DNA sequence. When the RNA transcript is a perfect complementary copy<br>
of the DNA sequence, it is referred to as the primary transcript or it may be a RNA sequence<br>
derived from posttranscriptional processing of the primary transcript and is referred to as the<br>
mature RNA. "Messenger RNA" (mRNA) refers to the RNA that is without introns and that<br>
can be translated into polypeptide by the cell. "Sense RNA" refers to an RNA transcript that<br>
includes the mRNA and so can be translated into a polypeptide by the cell. "Antisense RNA"<br>
refers to an RNA transcript that is complementary to a target mRNA, resulting in specific<br>
RNA:RNA duplexes being formed by base pairing between the antisense RNA substrate and<br>
the target mRNA.<br>
"Recombinant vector" refers to any agent by or in which a nucleic acid of interest is<br>
amplified, expressed, or stored, such as a plasmid, cosmid, virus, autonomously replicating<br>
sequence, phage, or linear single-stranded, circular single-stranded, linear double-stranded, or<br>
circular double-stranded DNA or RNA nucleotide sequence. The recombinant vector may be<br>
derived from any source and is capable of genomic integration or autonomous replication.<br>
"Regulatory sequence" refers to a nucleotide sequence located upstream (51), within, or<br>
downstream (3') with respect to a coding sequence. Additionally, introns may have regulatory<br>
activity. Transcription and expression of the coding sequence is typically impacted by the<br>
presence or absence of the regulatory sequence.<br>
"Substantially homologous" refers to two sequences that are at least about 90%<br>
identical in sequence, as measured by the CLUSTAL W method in the Omiga program, using<br>
default parameters (Version 2.0; Accelrys, San Diego, CA).<br>
"Substantially purified" refers to a molecule separated from substantially all other<br>
molecules normally associated with it in its native state. More preferably, a substantially<br>
purified molecule is the predominant species present in a preparation. A substantially purified<br>
molecule may be greater than about 60% free, preferably about 75% free, more preferably<br>
about 90% free, and most preferably about 95% free from the other molecules (exclusive of<br>
solvent) present in the natural mixture. The phrase "substantially purified" is not intended to<br>
encompass molecules present in their native state.<br>
The term "transformation" refers to the introduction of nucleic acid into a recipient<br>
host. The term "host" refers to bacteria cells, fungi, animals or animal cells, plants or seeds,<br>
or any plant parts or tissues including plant cells, protoplasts, calli, roots, tubers, seeds, stems,<br>
leaves, seedlings, embryos, and pollen.<br>
As used herein, a "transgenic plant" is a plant having stably introduced into its<br>
genome, for example, the nuclear or plastid genomes, a nucleic acid.<br>
The terms "seeds" and "kernels" are understood to be equivalent in meaning. The<br>
term kernel is frequently used in describing the seed of a corn or rice plant. In all plants the<br>
seed is the mature ovule consisting of a seed coat, embryo, and in plants of the present<br>
invention, an endosperm.<br>
HOIQ01 GBSS Nucleic Acids<br>
The present invention provides nucleic acids that encode polypeptides substantially<br>
homologous to a granule bound starch synthase isolated from the inbred plant HOI001<br>
(HOI001 GBSS). In one embodiment, these nucleic acid molecules are used in the context of<br>
the present invention for increasing the oil content of plant tissues. In one embodiment, the<br>
present invention provides an isolated nucleic acid that encodes a HOI001 GBSS protein,<br>
which nucleic acid is selected from the group consisting of SEQ ID NO: 1 and complements<br>
thereof, and nucleic acids which encode polypeptides having at least about 94% sequence<br>
identity with SEQ ID NO: 3. The percent sequence identity of the polypeptides encoded by<br>
nucleic acids of this invention is preferably at least about 95%; and most preferably at least<br>
about 98%.<br>
The present invention also provides vectors containing such HOI001 GBSS nucleic<br>
acids. As set forth in further detail hereinbelow, preferred nucleic acids include appropriate<br>
regulatory elements operably linked thereto that facilitate efficient expression of the inventive<br>
nucleic acids in a host, including without limitation bacterial, fungal, or plant hosts. Vectors<br>
useful in the context of the present invention can include such regulatory elements.<br>
The nucleic acids and vectors encompassed by the present invention need not have the<br>
exact nucleic acid sequences described herein. Instead, the sequences of these nucleic acids<br>
and vectors can vary, so long as the nucleic acid either performs the function for which it is<br>
intended or has some other utility, for example, as a nucleic acid probe for complementary<br>
nucleic acids. For example, some sequence variability in any part of a HOI001 GBSS nucleic<br>
acid is permitted so long as transformation of a plant with the mutant or variant polypeptide or<br>
polypeptides result in a phenotype substantially similar to that of HOI001 GBSS. Most<br>
preferably, the aforementioned sequence variability results in increased oil accumulation in<br>
plant tissues, as compared to plants of the same or similar genotype, but without the<br>
transgene.<br>
Fragment and variant nucleic acids of SEQ ID NO: 1, are also encompassed by the<br>
present invention. Nucleic acid fragments encompassed by the present invention are of three<br>
general types. First, fragment nucleic acids that are not full length but do perform their<br>
intended function are encompassed within the present invention. Second, fragments of<br>
nucleic acids identified herein that are useful as hybridization probes, are also included in the<br>
invention. And, third, fragments of nucleic acids identified herein can be used in suppression<br>
technologies known in the art, such as, for example, anti-sense technology or RNA inhibition<br>
(RNAi), which provides for reducing carbon flow in a plant into oil, making more carbon<br>
available for protein or starch accumulation, for example. Thus, fragments of a nucleotide<br>
sequence, such as SEQ ID NO: 1 may range from at least about 15 nucleotides, about <br>
nucleotides, about 18 nucleotides, about 20 nucleotides, about 50 nucleotides, about 100<br>
nucleotides or more. In general, a fragment nucleic acid of the present invention can have any<br>
upper size limit so long as it is related in sequence to the nucleic acids of the present invention<br>
but does not include the full length.<br>
As used herein, "variants" have substantially similar or substantially homologous<br>
sequences when compared to reference or wild type sequence. For nucleotide sequences that<br>
encode proteins, variants also include those sequences that, because of the degeneracy of the<br>
genetic code, encode the identical amino acid sequence of the reference protein. Variant<br>
nucleic acids also include those that encode polypeptides that do not have amino acid<br>
sequences identical to that of the proteins identified herein, but which encode an active protein<br>
with conservative changes in the amino acid sequence.<br>
The present invention is not limited to silent changes in the present nucleotide<br>
sequences but also includes variant nucleic acid sequences that conservatively alter the amino<br>
acid sequence of a polypeptide of the present invention. Because it is the interactive capacity<br>
and nature of a protein that defines that protein's biological functional activity, certain amino<br>
acid sequence substitutions can be made in a protein sequence and, of course, its underlying<br>
DNA coding sequence and, nevertheless, a protein with like properties can still be obtained.<br>
It is thus contemplated by the inventors that various changes may be made in the peptide<br>
sequences of the proteins or fragments of the present invention, or corresponding DNA<br>
sequences that encode the peptides, without appreciable loss of their biological utility or<br>
activity. According to the present invention, then, variant and reference nucleic acids of the<br>
present invention may differ in the encoded amino acid sequence by one or more<br>
substitutions, additions, insertions, deletions, fusions, and truncations, which may be present<br>
in any combination, so long as an active HOI001 GBSS protein is encoded by the variant<br>
nucleic acid. Such variant nucleic acids will not encode exactly the same amino acid<br>
sequence as the reference nucleic acid, but have conservative sequence changes. Codons<br>
capable of coding for such conservative amino acid substitutions are well known in the art.<br>
Another approach to identifying conservative amino acid substitutions require analysis<br>
of the hydropathic index of amino acids may be considered. The importance of the<br>
hydropathic amino acid index in conferring interactive biological function on a protein is<br>
generally understood in the art (Kyte and Doolittle, /. Mol. Biol, 157:105-132 (1982)). It is<br>
accepted that the relative hydropathic character of the amino acid contributes to the secondary<br>
structure of the resultant polypeptide, which in turn defines the interaction of the protein with<br>
other molecules, for example, enzymes, substrates, receptors, DNA, antibodies, antigens, and<br>
the like.<br>
Each amino acid has been assigned a hydropathic index on the basis of its<br>
hydrophobicity and charge characteristics (Kyte and Doolittle, J. Mol. Biol, 157:105-132<br>
(1982)); these are isoleucine (+4.5), valine (+4.2), leucine (+3.8), phenylalanine (+2.8),<br>
cysteine/cystine (+2.5), methionine (+1.9), alanine (+1.8), glycine (-0.4), threonine (-0.7),<br>
serine (-0.8), tryptophan (-0.9), tyrosine (-1.3), proline (-1.6), histidine (-3.2), glutamate<br>
(-3.5), glutamine (-3.5), aspartate (-3.5), asparagine (-3.5), lysine (-3.9), and arginine (-4.5).<br>
In making such changes, the substitution of amino acids whose hydropathic indices are<br>
within ±2 is preferred, those that are within ±1 are particularly preferred, and those within<br>
+0.5 are even more particularly preferred.<br>
It is also understood in the art that the substitution of like amino acids can be made<br>
effectively on the basis of hydrophilicity. U.S. Patent 4,554,101, states that the greatest local<br>
average hydrophilicity of a protein, as governed by the hydrophilicity of its adjacent amino<br>
acids, correlates with a biological property of the protein.<br>
As detailed in U.S. Patent 4,554,101, the following hydrophilicity values have been<br>
assigned to amino acid residues: arginine (+3.0), lysine (+3.0), aspartate (+3.0±1), glutamate<br>
(+3.0±1), serine (+0.3), asparagine (+0.2), glutamine (+0.2), glycine (0), threonine (-0.4),<br>
proline (-0.5+1), alanine (-0.5), histidine (-0.5), cysteine (-1.0), methionine (-1.3), valine<br>
(-1.5), leucine (-1.8), isoleucine (-1.8), tyrosine (-2.3), phenylalanine (-2.5), and tryptophan<br>
(-3.4).<br>
In making such changes, the substitution of amino acids whose hydrophilicity values<br>
are within ±2 is preferred, those that are within +1 are particularly preferred, and those within<br>
±0.5 are even more particularly preferred.<br>
Variant nucleic acids with silent and conservative changes can be defined and<br>
characterized by the degree of homology to the reference nucleic acid. Preferred variant<br>
nucleic acids are substantially homologous to the reference nucleic acids of the present<br>
invention. As recognized by one of skill in the art, such substantially similar nucleic acids can<br>
hybridize under stringent conditions with the reference nucleic acids identified by SEQ ID<br>
NO: 1, herein. These types of substantially homologous nucleic acids are encompassed by<br>
this invention.<br>
Variant nucleic acids can be detected and isolated by standard hybridization<br>
procedures. Hybridization to detect or isolate such sequences is generally carried out under<br>
"moderately stringent" and preferably under "stringent" conditions. Moderately stringent<br>
hybridization conditions and associated moderately stringent and stringent hybridization wash<br>
conditions used in the context of nucleic acid hybridization experiments, such as Southern and<br>
northern hybridization, are sequence dependent, and are different under different<br>
environmental parameters. Longer sequences hybridize specifically at higher temperatures.<br>
An extensive guide to the hybridization of nucleic acids is found in Tijssen, Laboratory<br>
Techniques in Biochemistry and Molecular Biology-Hybridization with Nucleic Acid Probes,<br>
page 1, Chapter 2 "Overview of principles of hybridization and the strategy of nucleic acid<br>
probe assays" Elsevier, NY (1993). See also, J. Sambrook et al, Molecular Cloning: A<br>
Laboratory Manual, Cold Spring Harbor Press, NY, pp 9.31-9.58 (1989); J. Sambrook et al,<br>
Molecular Cloning: A Laboratory Manual, Cold Spring Harbor Press, NY (3rd ed. 2001).<br>
The present invention also provides methods for detection and isolation of derivative<br>
or variant nucleic acids encoding the proteins provided herein. The methods involve<br>
hybridizing at least a portion of a nucleic acid comprising any part of SEQ ID NO: 1 with<br>
respect to HOI001 GBSS-related sequences, to a sample nucleic acid, thereby forming a<br>
hybridization complex; and detecting the hybridization complex. The presence of the<br>
complex correlates with the presence of a derivative or variant nucleic acid that can be further<br>
characterized by nucleic acid sequencing, expression of RNA and/or protein and testing to<br>
determine whether the derivative or variant retains the ability to increase oil levels in plant<br>
tissue when transformed into that plant. In general, the portion of a nucleic acid comprising<br>
any part of SEQ ID NO: 1 used for hybridization is preferably at least about fifteen<br>
nucleotides, and hybridization is under hybridization conditions that are sufficiently stringent<br>
to permit detection and isolation of substantially homologous nucleic acids; preferably, the<br>
hybridization conditions are "moderately stringent", more preferably the hybridization<br>
conditions are "stringent", as defined herein and in the context of conventional molecular<br>
biological techniques well known in the art.<br>
Generally, highly stringent hybridization and wash conditions are selected to be about<br>
5°C lower than the thermal melting point (Tm) for the specific double-stranded sequence at a<br>
defined ionic strength and pH. For example, under "highly stringent conditions" or "highly<br>
stringent hybridization conditions" a nucleic acid will hybridize to its complement to a<br>
detectably greater degree than to other sequences (e.g., at least 2-fold over background). By<br>
controlling the stringency of the hybridization and/or the washing conditions, nucleic acids<br>
having 100% complementary can be identified and isolated.<br>
Typically, stringent conditions will be those in which the salt concentration is less than<br>
about 1.5 M Na ion, typically about 0.01 to 1.0 M Na ion concentration (or other salts) at pH<br>
7.0 to 8.3 and the temperature is at least about 30°C for short probes (e.g., 10 to 50<br>
nucleotides) and at least about 60°C for long probes (e.g., greater than 50 nucleotides).<br>
Stringent conditions may also be achieved with the addition of destabilizing agents such as<br>
formamide, in which case hybridization temperatures can be decreased. Dextran sulfate<br>
and/or Denhardt's solution (50X Denhardt's is 5% Ficoll, 5% polyvinylpyrrolidone, 5% BSA)<br>
can also be included in the hybridization reactions.<br>
Exemplary low stringency conditions include hybridization with a buffer solution of<br>
30 to 50% formamide, 5X SSC (20X SSC is 3M NaCl, 0.3 M trisodium citrate), 50mM<br>
sodium phosphate, pH7, 5mM EDTA, 0.1% SDS (sodium dodecyl sulfate), 5X Denhardt's<br>
with 100 ug/ml denatured salmon sperm DNA at 37°C, and a wash in IX to 5X SSC (20X<br>
SSC defined as 3.0 M NaCl and 0.3 M trisodium citrate), 0.1% SDS at 37°C. Exemplary<br>
moderate stringency conditions include hybridization in 40 to 50% formamide, 5X SSC<br>
50mM sodium phosphate, pH 7, 5mM EDTA, 0.1% SDS, 5X Denhardt's with 100 ug/ml<br>
denatured salmon sperm DNA at 42°C, and a wash in 0.1X to 2X SSC, 0.1% SDS at 42 to<br>
55°C. Exemplary high stringency conditions include hybridization in 50% formamide, 5X<br>
SSC, 50mM sodium phosphate, pH 7.0, 5mM EDTA, 0.1% SDS, 5X Denhardt's with<br>
100 u.g/ml denatured salmon sperm DNA at 42°C, and a wash in 0. IX SSC, 0.1% SDS at 60<br>
to 65°C.<br>
In another embodiment of the present invention, the inventive nucleic acids are<br>
defined by the percent identity relationship between particular nucleic acids and other<br>
members of the class using analytic protocols well known in the art. Such analytic protocols<br>
include, but are not limited to: CLUSTAL in the PC/Gene program (available from<br>
Intelligenetics, Mountain View, CA or in the Omiga program version 2.0 Accelrys Inc., San<br>
Diego, CA); the ALIGN program (Version 2.0); and GAP, BESTFIT, BLAST, FASTA, and<br>
TFASTA in the Wisconsin Genetics Software Package, Version 8 (available from Genetics<br>
Computer Group (GCG), 575 Science Drive, Madison, WI). Alignments using these<br>
programs can be performed using the default parameters. The CLUSTAL program is well<br>
described by Higgins et al, Gene, 73:237-244 (1988); Higgins et al, CABIOS, 5:151-153<br>
(1989); Corpet et al., Nucleic Acids Res., 16:10881-10890 (1988); Huang et al., CABIOS,<br>
8:155-165 (1992); and Pearson et al., Meth.Mol. Biol., 24:307-331 (1994). The ALIGN<br>
program is based on the algorithm of Meyers and Miller, Computer Applic. Biol. Sci., 4:11-17<br>
(1988). The BLAST programs of Altschul et al, J. Mol. Biol., 215:403 (1990), are based on<br>
the algorithm of Karlin and Altschul, Proc. Natl. Acad. Sci. U.S.A., 87:2264-2268 (1990). To<br>
obtain gapped alignments for comparison puiposes, Gapped BLAST (in BLAST 2.0) can be<br>
utilized as described in Altschul et al., Nucleic Acids Res., 25:3389 (1997). Alternatively,<br>
PSI-BLAST (in BLAST 2.0) can be used to perform an iterated search that detects distant<br>
relationships between molecules. See, Altschul et al., supra. When utilizing BLAST, Gapped<br>
BLAST, PSI-BLAST, the default parameters of the respective programs (e.g., BLASTN for<br>
nucleotide sequences, BLASTP for proteins) can be used. The BLASTN program (for<br>
nucleotide sequences) uses as defaults a word length (W) of 11, an expectation (E) of 10, a<br>
cutoff of 100, M=5, N=-4, and a comparison of both strands. For amino acid sequences, the<br>
BLASTP program uses as defaults a word length (W) of 3, an expectation (E) of 10, and the<br>
BLOSUM62 scoring matrix (see, Henikoff &amp; Henikoff, Proc. Natl. Acad. Sci. U.S.A.,<br>
89:10915 (1989)) (see, http://www.ncbi.nlm.nih.gov.). Alignment may also be performed<br>
manually by inspection.<br>
For purposes of the present invention, comparison of nucleotide sequences for<br>
determination of percent sequence identity to the nucleic acid sequences disclosed herein is<br>
preferably made using the BLASTN program (version 1.4.7 or later) with its default<br>
parameters or any equivalent program. By "equivalent program" is intended any sequence<br>
comparison program that, for any two sequences in question, generates an alignment having<br>
identical nucleotide or amino acid residue matches and an identical percent sequence identity<br>
when compared to the corresponding alignment generated by the preferred program.<br>
Expression Vectors and Cassettes<br>
The expression vectors and cassettes of the present invention include nucleic acids<br>
encoding HOIOOl GBSS. A transgene comprising a HOIOOl GBSS can be subcloned into an<br>
expression vector or cassette, and HOIOOl GBSS expression can be detected and/or<br>
quantified. This method of screening is useful to identify transgenes providing for an<br>
expression of a HOIOOl GBSS, and expression of a HOIOOl GBSS in a transformed plant cell.<br>
Plasmid vectors that provide for easy selection, amplification, and transformation of<br>
the transgene in prokaryotic and eukaryotic cells include, for example, pUC-derived vectors,<br>
pSK-derived vectors, pGEM-derived vectors, pSP-derived vectors, pBS-derived vectors,<br>
pFastBac (Invitrogen Corporation, Carlsbad, CA) for baculovirus expression and pYES2<br>
(Invitrogen) for yeast expression. Additional elements may be present in such vectors,<br>
including origins of replication to provide for autonomous replication of the vector, selectable<br>
marker genes, preferably encoding antibiotic or herbicide resistance, unique multiple cloning<br>
sites providing for multiple sites to insert DNA sequences or genes encoded in the transgene,<br>
and sequences that enhance transformation of prokaryotic and eukaryotic cells. One vector<br>
that is useful for expression in both plant and prokaryotic cells is the binary Ti plasmid (as<br>
disclosed in Schilperoot et al., U.S. Patent 4,940,838), as exemplified by vector pGA582.<br>
This binary Ti plasmid vector has been previously characterized by An, Methods in<br>
Enzymology, 153:292 (1987). This binary Ti vector can be replicated in prokaryotic bacteria,<br>
such as E. coli and Agrobacterium. The Agrobacterium plasmid vectors can also be used to<br>
transfer the transgene to plant cells. The binary Ti vectors preferably include the T DNA right<br>
and left borders to provide for efficient plant cell transformation, a selectable marker gene,<br>
unique multiple cloning sites in the T border regions, the co/El replication of origin and a<br>
wide host range replicon. The binary Ti vectors carrying a transgene of the present invention<br>
can be used to transform both prokaryotic and eukaryotic cells, but is preferably used to<br>
transform plant cells, (see, Classman et al, U.S. Patent 5,258,300). Examples of plant<br>
expression vectors include the commercial vectors pBHOl, pBI101.2, pBI101.3, and pBIN19<br>
(Clontech, Palo Alto, CA).<br>
In general, the expression vectors and cassettes of the present invention contain at least<br>
a promoter capable of expressing RNA in a plant cell and a terminator, in addition to a nucleic<br>
acid encoding a HOI001 GBSS. Other elements may also be present in the expression<br>
cassettes of the present invention. For example, expression cassettes can also contain<br>
enhancers, introns, untranslated leader sequences, cloning sites, matrix attachment regions for<br>
silencing the effects of chromosomal control elements, and other elements known to one of<br>
skill in the art.<br>
Expression cassettes have promoters that can regulate gene expression. Promoter<br>
regions are typically found in the flanking DNA sequence upstream from coding regions in<br>
both prokaryotic and eukaryotic cells. A promoter sequence provides for regulation of<br>
transcription of the downstream gene sequence and typically includes from about 50 to about<br>
2,000 nucleotide base pairs. Promoter sequences also contain regulatory sequences, such as<br>
enhancer sequences that can influence the level of gene expression. Some isolated promoter<br>
sequences can provide for gene expression of heterologous genes, that is, a gene different<br>
from the native or homologous gene. Promoter sequences are also known to be strong or<br>
weak or inducible. A strong promoter provides for a high level of gene expression, whereas a<br>
weak promoter provides for a very low level of gene expression. An inducible promoter is a<br>
promoter that provides for turning on and off of gene expression in response to an<br>
exogenously added agent or to an environmental or developmental stimulus. Promoters can<br>
also provide for tissue specific or developmental regulation. An isolated promoter sequence<br>
that is a strong promoter for heterologous genes is advantageous because it provides for a<br>
sufficient level of gene expression to allow for easy detection and selection of transformed<br>
cells and provides for a high level of gene expression when desired. Transcription initiation<br>
egions that are preferentially expressed in seed tissue, and that are undetectable in other plant<br>
parts, are considered desirable for seed oil modifications in order to minimize any disruptive<br>
or adverse effects of the gene product.<br>
Promoters of the present invention will generally include, but are not limited to,<br>
promoters that function in bacteria, plant cells, or plastids. Useful promoters for bacterial<br>
expression are the lacZ, T7, T5, or E. coli gig C promoters. Useful promoters for plant cells<br>
include wheat high molecular weight glutenin promoter (bp 2647-3895 of Genbank Accession<br>
X12928, version X12928.3, originally described in Anderson et al, Nucleic Acids Res.,<br>
17:461-462 (1989)), the globulin promoter (see, Belanger and Kriz, Genet., 129:863-872,<br>
(1991)), gamma zein Z27 promoter (see, U.S. Serial Number 08/763,705; also, Lopes et al,<br>
Mol Gen Genet., 247:603-613 (1995)), L3 oleosin promoter (U.S. Patent 6,433,252), CaMV<br>
35S promoter (Odell et al, Nature, 313:810 (1985)), the CaMV 19S (Lawton et al, Plant<br>
Mol Biol, 9:3 IF (1987)), nos (Ebert et al, Proc. Natl Acad. Sci. U.S.A., 84:5745 (1987)),<br>
Adh (Walker et al, Proc. Natl Acad. Sci. U.S.A., 84:6624 (1987)), sucrose synthase (Yang et<br>
al, Proc. Natl Acad. Sci. U.S.A., 87:4144 (1990)), tubulin, actin (Wang et al, Mol Cell<br>
Biol, 12:3399 (1992)), cab (Sullivan et al, Mol Gen. Genet., 215:431 (1989)), PEPCase<br>
promoter (Hudspeth et al, Plant Mol Biol, 12:579 (1989)), or those associated with the R<br>
gene complex (Chandler et al, The Plant Cell, 1:1175 (1989)).<br>
Indeed in a preferred embodiment the promoter used is highly-expressed in the<br>
endosperm. Exemplary promoters include those from the zeins which are a group of storage<br>
proteins found in maize endosperm. Genomic clones for zein genes have been isolated<br>
(Pedersen et al, Cell, 29:1015-1026 (1982) and Russell et al, Transgenic Res., 6(2):157-168<br>
(1997)) and the promoters from these clones, including the 15 kD, 16 kD, 19 kD, 22 kD, and<br>
27 kD genes (Z27, U.S. Serial Number 08/763,705; also, Reina et al, Nucl Acids Res.,<br>
18:6426 (1990), Lopes et al, Mol Gen. Genet., 247:603-613 (1995)), can also be used. Other<br>
preferred promoters, known to function in maize, and in other plants, include the promoters<br>
for the following genes: WAXY (granule bound starch synthase; Shure et al, Cell,<br>
35:225-233 (1983); Russell et al, Transgenic Res., 6(2):157-168 (1997)), Brittle 2 and<br>
Shrunken 2 (ADP glucose pryophosphorylase, Anderson et al, Gene, 97:199-205 (1991),<br>
Russell et al, Transgenic Res., 6(2):157-168 (1997)), Shrunken /(sucrose synthase, Yang and<br>
Russell, Proc. Natl Acad. Sci. U.S.A., 87:4144-4148 (1990)), branching enzymes I and II,<br>
WAXY promoter from rice (Terada et al, Plant Cell Physiology, 41(7):881-888 (2000)),<br>
debranching enzymes, glutelins (Zheng et al, Plant J., 4:357-366 (1993), Russell et al,<br>
Transgenic Res., 6(2):157-168 (1997)), and Belli (basal endosperm transfer layer; Hueros et<br>
al. Plant Physiol., 121:1143-1152 (1999)). Other promoters useful in the practice of the<br>
present invention that are known by one of skill in the art are also contemplated by the<br>
invention.<br>
Moreover, transcription enhancers or duplications of enhancers can be used to increase<br>
expression from a particular promoter. Examples of such enhancers include, but are not<br>
limited to, elements from the CaMV 35S promoter and octopine synthase genes (Last et al.,<br>
U.S. Patent 5,290,924). As the DNA sequence between the transcription initiation site and the<br>
start of the coding sequence, i.e., the untranslated leader sequence, can influence gene<br>
expression, one may also wish to employ a particular leader sequence. Any leader sequence<br>
available to one of skill in the art may be employed. Preferred leader sequences direct<br>
optimum levels of expression of the attached gene, for example, by increasing or maintaining<br>
mRNA stability and/or by preventing inappropriate initiation of translation (Joshi, Nucl. Acid<br>
Res., 15:6643 (1987)). The choice of such sequences is at the discretion of those of skill in<br>
the art. Sequences that are derived from genes that are highly expressed in higher plants, and<br>
in soybean, corn, and canola in particular, are contemplated.<br>
Expression cassettes of the present invention will also include a sequence near the 3'<br>
end of the cassette that acts as a signal to terminate transcription from a heterologous nucleic<br>
acid and that directs polyadenylation of the resultant mRNA. These are commonly referred to<br>
as 3' untranslated regions or 3' UTRs. Some 3' elements that can act as transcription<br>
termination signals include the wheat HSP17 3' UTR (bp532-741 of GenBank X13431,<br>
version XI3431.1, McElvain and Spiker, Nucleic Acids Res., 17:1764 (1989)), those from the<br>
nopaline synthase gene of Agrobacterium tumefaciens (Bevan et al., Nucl. Acid Res., 11:369<br>
(1983)), a napin 3' UTR (Kridl et al., Seed Sci Res., 1:209-219 (1991)), a globulin 3' UTR<br>
(Belanger and Kriz, Genetics, 129:863-872 (1991)), or one from a zein gene, such as Z27<br>
(Lopes et al, Mol Gen Genet., 247:603-613 (1995)). Other 3' elements known by one of skill<br>
in the art also can be used in the vectors of the present invention.<br>
Regulatory elements, such as Adh intron 1 (Callis et al., Genes Develop., 1:1183<br>
(1987)), a rice actin intron (McElroy et al., Mol. Gen. Genet., 231(1):150-160 (1991)), sucrose<br>
synthase intron (Vasil et al., Plant Physiol., 91:5175 (1989)), the maize HSP70 intron<br>
(Rochester et al, EMBO J., 5:451-458 (1986)), or TMV omega element (Gallic et al, The<br>
Plant Cell 1:301 (1989)) may further be included where desired. These 3' nontranslated<br>
regulatory sequences can be obtained as described in An, Methods in Enzymology, 153:292<br>
(1987) or are already present in plasmids available from commercial sources, such as<br>
Clontech, Palo Alto, CA. The 3' nontranslated regulatory sequences can be operably linked to<br>
-18-<br>
the 3' terminus of any heterologous nucleic acid to be expressed by the expression cassettes<br>
contained within the present vectors. Other such regulatory elements useful in the practice of<br>
the present invention are known by one of skill in the art and can also be placed in the vectors<br>
of the invention.<br>
The vectors of the present invention, as well as the coding regions claimed herein, can<br>
be optimized for expression in plants by having one or more codons replaced by other codons<br>
encoding the same amino acids so that the polypeptide is optimally translated by the<br>
translation machinery of the plant species in which the vector is used.<br>
Selectable Markers<br>
Selectable marker genes or reporter genes are also useful in the present invention.<br>
Such genes can impart a distinct phenotype to cells expressing the marker gene and thus allow<br>
such transformed cells to be distinguished from cells that do not have the marker. Selectable<br>
marker genes confer a trait that one can "select" for by chemical means, i.e., through the use<br>
of a selective agent (e.g., a herbicide, antibiotic, or the like). Reporter genes, or screenable<br>
genes, confer a trait that one can identify through observation or testing, i.e., by "screening"<br>
(e.g., the R-locus trait). Of course, many examples of suitable marker genes are known to the<br>
art and can be employed in the practice of the present invention.<br>
A number of selectable marker genes are known in the art and can be used in the<br>
present invention. A preferred selectable marker gene for use in the present invention would<br>
include genes that confer resistance to herbicides like glyphosate, such as EPSP (Della-Cioppa<br>
et al., Bio/Technology, 5(6):579-84 (1987)). A particularly preferred selectable marker would<br>
include a gene that encodes an altered EPSP synthase protein (Hinchee et al., Biotech., 6:915<br>
(1988)). Other possible selectable markers for use in connection with the present invention<br>
include, but are not limited to, a neo gene (Potrykus et al., Mol. Gen. Genet., 199:183 (1985))<br>
which codes for kanamycin resistance and can be selected for by applying kanamycin, a<br>
kanamycin analog such as geneticin (Sigma Chemical Company, St. Louis, MO), and the like;<br>
a bar gene that codes for bialaphos resistance; a nitrilase gene, such as bxn from Klebsiella<br>
ozaenae, which confers resistance to bromoxynil (Stalker et al., Science, 242:419 (1988)); a<br>
mutant acetolactate synthase gene (ALS) that confers resistance to imidazolinone,<br>
sulfonylurea or other ALS-inhibiting chemicals (EP 154 204A1 (1985)); a methotrexateresistant<br>
DHFR gene (Thillet et al., J. Biol. Ghent., 263:12500 (1988)); a dalapon<br>
dehalogenase gene that confers resistance to the herbicide dalapon. Where a mutant EPSP<br>
synthase gene is employed, additional benefit may be realized through the incorporation of a<br>
suitable plastid transit peptide (CTP).<br>
Screenable markers that may be employed include, but are not limited to, a<br>
/3-glucuronidase or uidA gene (GUS), which encodes an enzyme for which various<br>
chromogenic substrates are known; an R-locus gene, which encodes a product that regulates<br>
the production of anthocyanin pigments (red color) in plant tissues (Dellaporta et al, In<br>
Chromosome Structure and Function, pp. 263-282 (1988)); a /3-lactamase gene (Sutcliffe,<br>
Proc. Natl. Acad. Sci. U.S.A., 75:3737 (1978)), which encodes an enzyme for which various<br>
chromogenic substrates are known (e.g., PAD AC, a chromogenic cephalosporin); axylE gene<br>
(Zukowsky et al., Proc. Natl. Acad. Sci. U.S.A., 80:1101 (1983)) that encodes a catechol<br>
dioxygenase that can convert chromogenic catechols; an a-amylase gene (Ikuta et al.,<br>
Biotech., 8:241 (1990)); atyrosinase gene (Katzetal., J. Gen. Microbiol., 129:2703 (1983))<br>
that encodes an enzyme capable of oxidizing tyrosine to DOPA and dopaquinone, which in<br>
turn condenses to form the easily detectable compound melanin; a /3-galactosidase gene,<br>
which encodes an enzyme for which there are chromogenic substrates; a luciferase (lux) gene<br>
(Ow et al., Science, 234:856 (1986)), which allows for bioluminescence detection; or an<br>
aequorin gene (Prasher et al., Biochem. Biophys. Res. Comm., 126:1259 (1985)), which may<br>
be employed in calcium-sensitive bioluminescence detection, or a green fluorescent protein<br>
gene (Niedz et al., Plant Cell Reports, 14:403 (1995)). In a preferred embodiment, the<br>
screenable marker gene is operably linked to an aleurone-specific promoter as described by<br>
Kriz et al., in U.S. Patent 6,307,123.<br>
In addition to nuclear plant transformation, the present invention also extends to direct<br>
transformation of the plastid genome of plants. Hence, targeting of the gene product to an<br>
intracellular compartment within plant cells may also be achieved by direct delivery of a gene<br>
to the intracellular compartment. In some embodiments, direct transformation of plastid<br>
genome may provide additional benefits over nuclear transformation. For example, direct<br>
plastid transformation of HOI001 GBSS eliminates the requirement for a plastid targeting<br>
peptide and post-translational transport and processing of the pre-protein derived from the<br>
corresponding nuclear transformants. Plastid transformation of plants has been described by<br>
P. Maliga, Current Opinion in Plant Biology, 5:164-172 (2002), Heifetz, Biochimie,<br>
82:655-666 (2000), Bock, J. Mol Biol, 312:425-438 (2001), and Daniell et al., Trends in<br>
Plant Science, 7:84-91 (2002), and references cited therein.<br>
After constructing a transgene containing an HOI001 GBSS, the expression vector or<br>
cassette can then be introduced into a plant cell. Depending on the type of plant cell, the level<br>
of gene expression, and the activity of the enzyme encoded by the gene, introduction of DNA<br>
encoding an HOI001 GBSS into the plant cell can lead to increased oil content in plant<br>
tissues.<br>
Plant Transformation<br>
Techniques for transforming a plant cell, a plant tissue, a plant organ, or a plant with a<br>
nucleic acid construct, such as a vector are known in the art. Such methods involve plant<br>
tissue culture techniques, for example. As used herein, "transforming" refers to the<br>
introduction of nucleic acid into a recipient host and the expression therein.<br>
The plant cell, plant tissue, plant organ, or plant can be contacted with the vector by<br>
any suitable means as known in the art. Preferably, a transgenic plant expressing the desired<br>
protein is to be produced. Various methods for the introduction of a desired polynucleotide<br>
sequence encoding the desired protein into plant cells include, but are not limited to: (1)<br>
physical methods such as microinjection (Capecchi, Cell, 22(2):479-488 (1980)),<br>
electroporation (Fromm et al, Proc. Nat. Acad. Sci. U.S.A., 82(17):5824-5828 (1985); U.S.<br>
Patent 5,384,253), and microprojectile bombardment mediated delivery (Christou et al.,<br>
Bio/Technology, 9:957 (1991); Fynan et al., Proc. Nat. Acad. Sci. U.S.A.,<br>
90(24):! 1478-11482 (1993)); (2) virus mediated delivery methods (Clapp, Clin. Perinatal,<br>
20(1):155-168 (1993); Lu et al., J. Exp. Med., 178(6):2089-2096 (1993); Eglitis and<br>
Anderson, Biotechniques, 6(7):608-614 (1988); and (3) Agrobacterium-mediaied<br>
transformation methods.<br>
The most commonly used methods for transformation of plant cells are the<br>
Agrobacterium-mediated DNA transfer process (Fraley et al., Proc. Nat. Acad. Sci. U.S.A.,<br>
80:4803 (1983)) and the microprojectile bombardment mediated process. Typically, nuclear<br>
transformation is desired but where it is desirable to specifically transform plastids, such as<br>
chloroplasts or amyloplasts, plant plastids may be transformed utilizing a microprojectile<br>
bombardment mediated delivery of the desired polynucleotide for certain plant species such as<br>
tobacco, Arabidopsis, potato, and Brassica species.<br>
Agrobacterium-mediated transformation is achieved through the use of a genetically<br>
engineered soil bacterium belonging to the genus Agrobacterium. Several Agrobacterium<br>
species mediate the transfer of a specific DNA known as "T-DNA," which can be genetically<br>
engineered to carry any desired piece of DNA into many plant species. The major events<br>
marking the process of T-DNA mediated pathogenesis are: induction of virulence genes,<br>
processing, and transfer of T-DNA. This process is the subject of many reviews (Ream, Ann.<br>
Rev. Phytopathol, 27:583-618 (1989); Howard and Citovsky, Bioassays, 12:103-108 (1990);<br>
Kado, Crit. Rev. Plant Sci., 10:1-32 (1991); Zambryski, Annual Rev. Plant Physiol. Plant<br>
Mol. BioL, 43:465-490 (1992); Gelvin, In Transgenic Plants, Kung and Wu, (eds.), Academic<br>
Press, San Diego, CA, pp. 49-87 (1993); Binns and Howitz, In Bacterial Pathogenesis of<br>
Plants and Animals, Dang, (ed.). Berlin: Springer Verlag, pp. 119-138 (1994); Hooykaas and<br>
Beijersbergen,/4««. Rev. Phytopathol, 32:157-179 (1994); Lessl and Lanka, Cell, 77:321-324<br>
(1994); Zupan and Zambryski, Annual Rev. Phytopathol, 27:583-618 (1995)).<br>
Agrobacterium-mediated genetic transformation of plants involves several steps. The<br>
first step, in which the virulent Agrobacterium and plant cells are first brought into contact<br>
with each other, is generally called "inoculation." The Agrobacterium containing solution is<br>
then removed from contact with the explant by draining or aspiration. Following the<br>
inoculation, the Agrobacterium and plant cells/tissues are permitted to be grown together for a<br>
period of several hours to several days or more under conditions suitable for growth and<br>
T-DNA transfer. This step is termed "co-culture." Following co-culture and T-DNA<br>
delivery, the plant cells are treated with bactericidal or bacteriostatic agents to kill the<br>
Agrobacterium remaining in contact with the explant and/or in the vessel containing the<br>
explant. If this is done in the absence of any selective agents to promote preferential growth<br>
of transgenic versus non-transgenic plant cells, then this is typically referred to as the "delay"<br>
step. If done in the presence of selective pressure favoring transgenic plant cells, then it is<br>
referred to as a "selection" step. When a "delay" is used, it is typically followed by one or<br>
more "selection" steps. Both the "delay" and "selection" steps typically include bactericidal<br>
or bacteriostatic agents to kill any remaining Agrobacterium cells because the growth of<br>
Agrobacterium cells is undesirable after the infection (inoculation and co-culture) process.<br>
A number of wild-type and disarmed strains of Agrobacterium tumefaciens and<br>
Agrobacterium rhizogenes harboring Ti or Ri plasmids can be used for gene transfer into<br>
plants. The Agrobacterium hosts contain disarmed Ti and Ri plasmids that do not contain the<br>
oncogenes that cause tumorigenesis or rhizogenesis, respectfully, which are used as the<br>
vectors and contain the genes of interest that are subsequently introduced into plants.<br>
Preferred strains would include but are not limited to Agrobacterium tumefaciens strain C58, a<br>
nopaline-type strain that is used to mediate the transfer of DNA into a plant cell, octopinetype<br>
strains such as LBA4404 or succinamopine-type strains, e.g., EHA101 or EHA105. The<br>
nucleic acid molecule, prepared as a DNA composition in vitro, is introduced into a suitable<br>
host such as E. coli and mated into the Agrobacterium, or directly transformed into competent<br>
Agrobacterium. These techniques are well-known to those of skill in the art.<br>
The Agrobacterium can be prepared either by inoculating a liquid such as Luria<br>
Burtani (LB) media directly from a glycerol stock or streaking the Agrobacterium onto a<br>
solidified media from a glycerol stock, allowing the bacteria to grow under the appropriate<br>
selective conditions, generally from about 26°C-30°C, or about 28°C, and taking a single<br>
colony or a small loop of Agrobacterium from the plate and inoculating a liquid culture<br>
medium containing the selective agents. Those of skill in the art are familiar with procedures<br>
for growth and suitable culture conditions for Agrobacterium as well as subsequent<br>
inoculation procedures. The density of the Agrobacterium culture used for inoculation and<br>
the ratio of Agrobacterium cells to explant can vary from one system to the next, and therefore<br>
optimization of these parameters for any transformation method is expected.<br>
Typically, an Agrobacterium culture is inoculated from a streaked plate or glycerol<br>
stock and is grown overnight and the bacterial cells are washed and resuspended in a culture<br>
medium suitable for inoculation of the explant.<br>
With respect to microprojectile bombardment (U.S. Patents 5,550,318; 5,538,880; and<br>
5,610,042; and PCT Publication WO 95/06128; each of which is specifically incorporated<br>
herein by reference in its entirety), particles are coated with nucleic acids and delivered into<br>
cells by a propelling force. Exemplary particles include those comprised of tungsten,<br>
platinum, and preferably, gold. It is contemplated that in some instances DNA precipitation<br>
onto metal particles would not be necessary for DNA delivery to a recipient cell using<br>
microprojectile bombardment. However, it is contemplated that particles may contain DNA<br>
rather than be coated with DNA. Hence, it is proposed that DNA-coated particles may<br>
increase the level of DNA delivery via particle bombardment but are not, in and of<br>
themselves, necessary.<br>
For the bombardment, cells in suspension are concentrated on filters or solid culture<br>
medium. Alternatively, immature embryos or other target cells may be arranged on solid<br>
culture medium. The cells to be bombarded are positioned at an appropriate distance below<br>
the microprojectile stopping plate.<br>
An illustrative embodiment of a method for delivering DNA into plant cells by<br>
microprojectile bombardment is the Biolistics Particle Delivery System (BioRad, Hercules,<br>
CA), which can be used to propel particles coated with DNA or cells through a screen, such as<br>
a stainless steel or Nytex screen, onto a filter surface covered with monocot plant cells<br>
cultured in suspension. The screen disperses the particles so that they are not delivered to the<br>
recipient cells in large aggregates. It is believed that a screen intervening between the<br>
projectile apparatus and the cells to be bombarded reduces the size of projectile aggregates<br>
and may contribute to a higher frequency of transformation by reducing the damage inflicted<br>
on the recipient cells by projectiles that are too large.<br>
For microprojectile bombardment, one will attach (i.e., "coat") DNA to the<br>
microprojectiles such that it is delivered to recipient cells in a form suitable for transformation<br>
thereof. In this respect, at least some of the transforming DNA must be available to the target<br>
cell for transformation to occur, while at the same time during delivery the DNA must be<br>
attached to the microprojectile. Therefore, availability of the transforming DNA from the<br>
microprojectile may comprise the physical reversal of bonds between transforming DNA and<br>
the microprojectile following delivery of the microprojectile to the target cell. This need not<br>
be the case, however, as availability to a target cell may occur as a result of breakage of<br>
unbound segments of DNA or of other molecules which comprise the physical attachment to<br>
the microprojectile. Availability may further occur as a result of breakage of bonds between<br>
the transforming DNA and other molecules, which are either directly or indirectly attached to<br>
the microprojectile. It further is contemplated that transformation of a target cell may occur<br>
by way of direct recombination between the transforming DNA and the genomic DNA of the<br>
recipient cell. Therefore, as used herein, a "coated" microprojectile will be one which is<br>
capable of being used to transform a target cell, in that the transforming DNA will be<br>
delivered to the target cell, yet will be accessible to the target cell such that transformation<br>
may occur.<br>
Any technique for coating microprojectiles, which allows for delivery of transforming<br>
DNA to the target cells, may be used. Methods for coating microprojectiles, which have been<br>
demonstrated to work well with the present invention, have been specifically disclosed herein.<br>
DNA may be bound to microprojectile particles using alternative techniques, however. For<br>
example, particles may be coated with streptavidin and DNA end labeled with long chain thiol<br>
cleavable biotinylated nucleotide chains. The DNA adheres to the particles due to the<br>
streptavidin-biotin interaction, but is released in the cell by reduction of the thiol linkage<br>
through reducing agents present in the cell.<br>
Alternatively, particles may be prepared by functionalizing the surface of a gold oxide<br>
particle, providing free amine groups. DNA, having a strong negative charge, binds to the<br>
functionalized particles. Furthermore, charged particles may be deposited in controlled arrays<br>
on the surface of mylar flyer disks used in the PDS-1000 Biolistics device, thereby facilitating<br>
controlled distribution of particles delivered to target tissue.<br>
As disclosed above, it further is proposed, that the concentration of DNA used to coat<br>
microprojectiles may influence the recovery of transformants containing a single copy of the<br>
transgene. For example, a lower concentration of DNA may not necessarily change the<br>
efficiency of the transformation, but may instead increase the proportion of single copy<br>
insertion events. In this regard, approximately 1 ng to 2000 ng of transforming DNA may be<br>
used per each 1.8 nig of starting microprojectiles. In other embodiments of the present<br>
invention, approximately 2.5 ng to 1000 ng, 2.5 ng to 750 ng, 2.5 ng to 500 ng, 2.5 ng to<br>
250 ng, 2.5 ng to 100 ng, or 2.5 ng to 50 ng of transforming DNA may be used per each<br>
1.8 mg of starting microprojectiles.<br>
Microprojectile bombardment techniques are widely applicable, and may be used to<br>
transform virtually any plant species. Examples of species that have been transformed by<br>
microprojectile bombardment include monocot species such as maize (PCX Publication WO<br>
95/06128), barley, wheat (U.S. Patent 5,563,055, specifically incorporated herein by reference<br>
in its entirety), rice, oat, rye, sugarcane, and sorghum; as well as a number of dicots including<br>
tobacco, soybean (U.S. Patent 5,322,783, specifically incorporated herein by reference in its<br>
entirety), sunflower, peanut, cotton, tomato, and legumes in general (U.S. Patent 5,563,055,<br>
specifically incorporated herein by reference in its entirety).<br>
For microprojectile bombardment transformation in accordance with the present<br>
invention, both physical and biological parameters may be optimized. Physical factors are<br>
those that involve manipulating the DNA/microprojectile precipitate or those that affect the<br>
flight and velocity of either the macro- or microprojectiles. Biological factors include all<br>
steps involved in manipulation of cells before and immediately after bombardment, such as<br>
the osmotic adjustment of target cells to help alleviate the trauma associated with<br>
bombardment, the orientation of an immature embryo or other target tissue relative to the<br>
particle trajectory, and also the nature of the transforming DNA, such as linearized DNA or<br>
intact supercoiled plasmids. It is believed that pre-bombardment manipulations are especially<br>
important for successful transformation of immature embryos.<br>
Accordingly, it is contemplated that one may wish to adjust various bombardment<br>
parameters in small scale studies to fully optimize the conditions. One may particularly wish<br>
to adjust physical parameters such as DNA concentration, gap distance, flight distance, tissue<br>
distance, and helium pressure. It further is contemplated that the grade of helium may affect<br>
transformation efficiency. One also may optimize the trauma reduction factors (TRFs) by<br>
modifying conditions which influence the physiological state of the recipient cells and which<br>
may therefore influence transformation and integration efficiencies. For example, the osmotic<br>
state, tissue hydration, and the subculture stage or cell cycle of the recipient cells may be<br>
adjusted for optimum transformation.<br>
Other methods of cell transformation can also be used and include but are not limited<br>
to introduction of DNA into plants by direct DNA transfer into pollen (Hess et al, Intern Rev.<br>
Cytol., 107:367 (1987); Luo etal, Plant Mol Biol. Reporter, 6:165 (1988)), by direct injection<br>
of DNA into reproductive organs of a plant (Pena et al, Nature, 325:274 (1987)), or by direct<br>
injection of DNA into the cells of immature embryos followed by the rehydration of<br>
desiccated embryos (Neuhaus et al., Theor. Appl. Genet., 75:30 (1987)).<br>
The regeneration, development, and cultivation of plants from single plant protoplast<br>
transformants or from various transformed explants is well known in the art (Weissbach and<br>
Weissbach, In: Methods for Plant Molecular Biology, Academic Press, San Diego, CA,<br>
(1988)). This regeneration and growth process typically includes the steps of selection of<br>
transformed cells, culturing those individualized cells through the usual stages of embryonic<br>
development through the rooted plantlet stage. The resulting transgenic rooted shoots are<br>
thereafter planted in an appropriate plant growth medium such as soil.<br>
The development or regeneration of plants containing the exogenous gene that encodes<br>
a protein of interest is well known in the art. Preferably, the regenerated plants are selfpollinated<br>
to provide homozygous transgenic plants. Otherwise, pollen obtained from the<br>
regenerated plants is crossed to seed-grown plants of agronomically important lines.<br>
Conversely, pollen from plants of these important lines is used to pollinate regenerated plants.<br>
A transgenic plant of the present invention containing a desired polypeptide is cultivated using<br>
methods well known to one skilled in the art.<br>
There are a variety of methods for the regeneration of plants from plant tissue. The<br>
particular method of regeneration will depend on the starting plant tissue and the particular<br>
plant species to be regenerated.<br>
Assays for gene expression based on the transient expression of cloned nucleic acid<br>
constructs have been developed by introducing the nucleic acid molecules into plant cells by<br>
polyethylene glycol treatment, electroporation, or particle bombardment (Marcotte et al,<br>
Nature, 335:454-457 (1988); Marcotte et al, Plant Cell, 1:523-532 (1989); McCarty et al,<br>
Cell, 66:895-905 (1991); Hattori etal, GenesDev., 6:609-618 (1992); G o f f e t a l , EMBO J.,<br>
9:2517-2522 (1990)). Transient expression systems may be used to functionally dissect gene<br>
constructs (see generally, Maliga et al, Methods in Plant Molecular Biology, Cold Spring<br>
Harbor Press (1995)).<br>
Any of the nucleic acid molecules of the present invention may be introduced into a<br>
plant cell in a permanent or transient manner in combination with other genetic elements such<br>
as vectors, promoters, enhancers, etc. Further, any of the nucleic acid molecules of the<br>
present invention may be introduced into a plant cell in a manner that allows for expression or<br>
overexpression of the protein or fragment thereof encoded by the nucleic acid molecule.<br>
Transgenic plants may find use in the commercial manufacture of proteins or other<br>
molecules, such as oils, where the molecules of interest are extracted or purified from plant<br>
parts, seeds, and the like. Cells or tissue from the plants may also be cultured, grown in vitro,<br>
or fermented to manufacture such molecules.<br>
Improvements encoded by the recombinant DNA may be transferred, e.g., from cells<br>
of one species to cells of other species, e.g., by protoplast fusion. The transgenic plants may<br>
also be used in commercial breeding programs, or may be crossed or bred to plants of related<br>
crop species. For example, a nucleic acid of the present invention, operably linked to a<br>
promoter, can be introduced into a particular plant variety by crossing, without the need for<br>
ever directly transforming a plant of that given variety. Therefore the present invention not<br>
only encompasses a plant directly regenerated from cells that have been transformed in<br>
accordance with the present invention, but also the progeny of such plants.<br>
The present invention also provides for a method of stably expressing an HOI001<br>
GBSS of interest in a plant, which includes, contacting the plant cell with a vector of the<br>
present invention that has a nucleic acid encoding the HOI001 GBSS of interest, under<br>
conditions effective to transfer and integrate the vector into the nuclear genome of the cell. A<br>
promoter within the expression cassette can be any of the promoters provided herein, for<br>
example, a constitutive promoter, an inducible promoter, a tissue-specific promoter, or a seed<br>
specific promoter. Such promoters can provide expression of an encoded HOI001 GBSS at a<br>
desired time, or at a desired developmental stage, or in a desired tissue. The vector can also<br>
include a selectable marker gene. When using the vector with Agrobacterium tumefaciens,<br>
the vector can have an Agrobacterium tumefaciens origin of replication.<br>
Plants<br>
Plants for use with the vectors of the present invention preferably include monocots,<br>
especially oil producing species, most preferably corn (Zea mays). Other species<br>
contemplated by the present invention include alfalfa (Medicago saliva), rice (Oryza saliva),<br>
barley (Hordeum vulgare), millet (Panicum miliaceum), rye (Secale cereale), wheat (Triticwn<br>
aestivum), and sorghum (Sorghum bicolor).<br>
Any of the plants or parts thereof of the present invention may be processed to produce<br>
a feed, meal, protein, or oil preparation. A particularly preferred plant part for this purpose is<br>
a seed. Methods to produce feed, meal, protein, and oil preparations are known in the art.<br>
See, for example, U.S. Patents 4,957,748; 5,100,679; 5,219,596; 5,936,069; 6,005,076;<br>
6,146,669; and 6,156,227.<br>
Characterization of Transformed Plants<br>
To confirm the presence of the transgene in the regenerated plant, a variety of<br>
techniques, which are well known in the art, are available. Examples of these techniques<br>
include but are not limited to: (a) molecular assays of DNA integration or RNA expression<br>
such as Southern or northern blotting, TAQMAN® technology (Applied Biosystems, Foster<br>
City, CA) and PCR; (b) biochemical assays detecting the presence of the protein product such<br>
as ELISA, western blotting, or by enzymatic function; or (c) chemical analysis of the targeted<br>
plant part, such as seed tissue, for qualitative and quantitative determination of oil, protein, or<br>
starch.<br>
The following examples are provided to illustrate the present invention and are not<br>
intended to limit the invention in any way.<br>
Example 1<br>
This example describes the isolation and sequencing of the HOI001 GBSS gene from<br>
com line HOI001. HOI001 is an inbred plant derived from MGSC 915E (Maize Genetic<br>
Stock Center, Urbana, IL), and is more fully described in U.S. Patent Publication Nos.<br>
20030172416 and 20030154524, both of which are incorporated herein by reference.<br>
Genomic DNA was extracted from corn germ tissue from HOI001, 22 days after<br>
pollination, using the following procedure. Between 50-100 mg dissected genn tissue was<br>
placed in a BiolOl Multimix tube (Qiagen, Carlsbad, CA, Cat. No. 657-601) with extraction<br>
buffer and glass beads. The extraction buffer consisted of 100 mM Tris-HCl (pH 8.0), 50 mM<br>
EDTA, 100 mM NaCl, 5 mM DTT, and 1% SDS. The tissue was then disrupted using the Bio<br>
101 FASTPREP® machine (Qiagen) with 3 pulses of 20 seconds each. Following a 15 minute<br>
incubation at 65°C, 330 ul of 5M potassium acetate was added to each tube. The tubes were<br>
then incubated at 0°C for 20 minutes to precipitate the SDS, followed by centrifugation at<br>
12,000 rpm (Eppendorf Model 54172) for 10 minutes. The supernatant was transferred to a<br>
new tube and 100 ul of 5M ammonium acetate (pH 7.0) and 700 |al of isopropanol was added<br>
to precipitate the DNA. The tubes were mixed by inversion and centrifuged at 14,000 rpm for<br>
10 minutes. After discarding the supernatant, the pellet was resuspended in 500 joJ of 70%<br>
ethanol and recovered by centrifugation at 14,000 rpm for 5 minutes. The recovered pellet<br>
containing the DNA was resuspended in 50 ul of TE buffer and stored at 4°C.<br>
The HOI001 GBSS gene was isolated from the extracted genomic DNA using PCR<br>
methodology that was adapted from Advantage GC (BD Biosciences Clontech, Palo Alto,<br>
CA). The following primers were designed based on the published sequence of Zea mays<br>
GBSS from Shure et al, Cell, 35(l):225-233 (1983) [SEQ ID NO: 2]:<br>
5' primer (Primer number 14543)<br>
5'-TCAGCCGTTCGTGTGGCAAGATTCATCTGTTGTCTC-3' [SEQ ID NO: 5]<br>
3' primer (Primer number 14547)<br>
5'-TCAGCGGGATTATTTACTCCACCACTACAGGTCCATTT-3' [SEQ ID NO: 6].<br>
The following PCR reaction was assembled for a total volume of 50^1;<br>
37 ul PCR grade water<br>
5 ul 5X Advantage GC PCR buffer<br>
1 ul SOX dNTP Mix (1 OmM each)<br>
1 ul SOX Advantage GC Polymerase Mix<br>
2.5  primer 14543<br>
2.5  primer 14547<br>
1 ul genomic DNA<br>
The cycle parameters were: 95°C for 1 minute, 35 cycles of 95°C for 30 seconds and<br>
68°C for 3 minutes.<br>
The PCR products were separated by electrophoresis in agarose and a 4.7 kB fragment<br>
containing the gene of interest was observed. Five microliters of the original PCR reaction<br>
was utilized as template for additional amplification using the same primers and conditions<br>
described above. The 4.7 kB amplification products from independent amplification reactions<br>
were isolated by agarose gel electrophoresis, cloned into the PCR 2.1 cloning vector using the<br>
TOPO TA cloning kit (Invitrogen), then transformed into an E. coli host. Plasmid DNA was<br>
prepared from cultures grown from each colony, and then the inserts from 3 separate plasmid<br>
preparations were sequenced. Alignment of these sequences generated a consensus sequence<br>
highly homologous but not identical to the published GBSS gene, although no specific insert<br>
sequence was equivalent to the consensus. One clone (designated pCGN9480-2) had an insert<br>
sequence with the fewest sequence changes relative to the consensus. A clone containing the<br>
consensus was then generated by restriction enzyme-mediated excision of non-consensus<br>
sequence and religation with fragments containing the consensus sequence, obtained by<br>
digestion of the other clones or by PCR amplification from HOI001 genomic DNA. The<br>
consensus sequence, including 1.5 kB upstream of the transcription start site and<br>
approximately 300 base pairs downstream of the stop codon, is listed as SEQ ID NO: 1.<br>
The GBSS gene from elite com inbred line LH59 [SEQ ID NO: 7] was isolated using<br>
the procedures and primers described above, and cloned into the binary vector pMON68203.<br>
The resulting plasmid containing the LH59 GBSS is named pMON72510 (Figure 5).<br>
Figure 1 shows the nucleic acid sequence alignment of the HOI001 GBSS [SEQ ID<br>
NO: 1] compared to the published sequence of Shure et al., supra [SEQ ID NO: 2] and the<br>
GBSS from LH59 [SEQ ID NO: 7], using the Omiga software package 2.0, (Accelrys Inc.,<br>
San Diego, CA). The alignment shows there are the following polymorphisms unique to the<br>
HOI001 GBSS sequence, that is not found in either the LH59 GBSS sequence or the<br>
published sequence of Shure et al., supra:<br>
1. Single nucleotide polymorphisms:<br>
a. T&gt;C at position 158<br>
b. G&gt;A at position 337<br>
c. OA at position 343<br>
d. OA at position 349<br>
e. G&gt;A at position 441<br>
f. OT at position 666<br>
g. G&gt;C at position 777<br>
h. T&gt;A at position 878<br>
i. OT at position 980<br>
j. T&gt;A at position 1210<br>
k. OT at position 1216<br>
1. A&gt;T at position 1450<br>
m. T&gt;C at position 1709<br>
n. A&gt;G at position 1720<br>
o. T&gt;A at position 1721<br>
p. G&gt;C at position 1722<br>
q. OT at position 1761<br>
r. G&gt;A at position 1836<br>
s. OT at position 1852<br>
t. G&gt;A at position 195 3<br>
u. OT at position 2043<br>
v. OT at position 2109<br>
w. OG at position 2110<br>
x. G&gt;C at position 2115<br>
y. A&gt;T at position 2448<br>
z. OT at position 2454<br>
aa. T&gt;G at position 2609<br>
bb. A&gt;G at position 2929<br>
cc. G&gt;T at position 2933<br>
dd. OT at position 2946<br>
ee. G&gt;T at position 3875<br>
ff. T&gt;A at position 4008<br>
gg. T&gt;C at position 4018<br>
hh. T&gt;G at position 4023<br>
ii. OA at position 4025<br>
jj. OT at position 4169<br>
kk. A&gt;T at position 4225<br>
11. OA at position 4562<br>
2. Insertions:<br>
a. Sequence g at position 632<br>
b. Sequence atgc at position 1185-1189<br>
c. Sequence tgcaccagcagc at position 1456-1467<br>
d. Sequence atgca at position 1746-1750<br>
e. Sequence catcaca at position 1868-1874<br>
f. Sequence ct at position 2100-2101<br>
g. Sequence ccat at position 2488-2491<br>
h. Sequence tat at position 3810-3812<br>
3. Deletions:<br>
a. Sequence cgt at position 288-290<br>
b. Sequence aa at position 704-705<br>
c. Sequence c at position 882<br>
d. Sequence atccg at position 1139-1143<br>
e. Sequence ctctctg at position 1256-1262<br>
f. Sequence tc at position 1714-1715<br>
g. Sequence tgcaactgcaaatgca at position 1917-1932<br>
h. Sequence g or a at position 3790<br>
i. Sequence cgagccaggggt(t or c)gaaggcgaggagatcgcgccgctcgccaagg<br>
agaacgtggccgcgccctgaagagttcggcct at position 4393-4467<br>
Figure 2 shows the alignment of the corresponding predicted amino acid sequences<br>
from the GBSS gene isolated from HOIOOl [SEQ ID NO: 3], and the GBSS gene described in<br>
Sliure et al, supra [SEQ ID NO: 4], respectively. The results indicate that there is a sequence<br>
of additional amino acid residues on the carboxy terminus of the HOIOOl GBSS starting at<br>
approximately position 1441 and an area of non-alignment in the region of amino acid residue<br>
55-60.<br>
Figure 3 shows the alignment of the corresponding predicted amino acid sequences<br>
from the Zea mays GBSS gene isolated from inbred LH59 [SEQ ID NO: 8], and the Zea mays<br>
granule bound starch synthase gene described in Shure et al., supra, respectively. The results<br>
indicate that there is a sequence of additional amino acid residues on the carboxy terminus of<br>
the HOIOOl GBSS starting at approximately position 1441 and an area of non-alignment in<br>
the region of amino acid residue 55-60.<br>
Example 2<br>
This example sets forth the construction of plant transformation vectors containing the<br>
sequences of the HOIOOl GBSS and the GBSS from inbred line LH59, [SEQ ID NOs: 1 and<br>
7, respectively].<br>
The HOI001 GBSS sequence was cut from the consensus-corrected version of<br>
pMON9480-2 using the restriction enzyme EcoRl. The resulting 4.7kb fragment was purified<br>
following the manufacturer's protocol for the Qiagen miniprep kit (Qiagen, Inc., Valencia,<br>
CA). The ends of the fragment were blunted following manufacturer's protocol in the<br>
Stratagene PCR polishing kit (Stratagene, Inc., La Jolla, CA). The fragment was then gel<br>
purified using the Qiagen Gel Extraction kit (Qiagen), and cloned into pMON68203, a binary<br>
vector for plant transformation. The binary vector, pMON68203, contains left and right<br>
borders for T-DNA transfer, a CaMV 35S promoter::nptll::nos 3' UTR plant selectable<br>
marker element (described in U.S. Patent 6,255,560), and plant expression cassette sequences<br>
which include a 1.1 kb Z27 promoter (bp 19-1117 of Accession #878780, Lopes et at., Mol.<br>
Gen. Genet., 247(5):603-613 (1995)) for endosperm expression, a com hsp70 intron (base<br>
pairs 4-153 of the maize gene for heat shock 70 exon 2, Accession #X03679, Rochester et al.,<br>
EMBOJ., 5:451-458 (1986)), and a nos 3' UTR, (base pairs 2924-2671 of the Agrobacterim<br>
tumefaciens strain C58 Ti plasmid, Accession #AE009420, Wood et al., Science,<br>
294:2317-2323 (2001)). The binary vector pMON68203 was digested with Stul,<br>
dephosphorylated by incubating with shrimp alkaline phosphatase (Roche Applied Science,<br>
Indianapolis, IN) at 37°C for 60 minutes and ligated with the 4.7 kb gel purified fragment of<br>
the HOI001 GBSS, described above. The resulting plasmid was named pMON72506 (Figure<br>
4).<br>
The GBSS from corn line LH59, [SEQ ID NO: 7], was similarly cloned into the binary<br>
vector pMON68203, to form pMON72510.<br>
Example 3<br>
This example describes the transformation of corn with the HOI001 GBSS and the<br>
GBSS from corn line LH59, using the vectors described in Example 2.<br>
The transformation vectors pMON72506 and pMON72510 were used to transform<br>
maize plants using the following procedure.<br>
Corn plants are grown in a greenhouse under standard practices. Controlled<br>
pollinations were made. The ears of the plants are harvested when the resulting hybrid<br>
embryos were 1.5 to 2.0 mm in length, usually 10-15 days after pollination. After removing<br>
the husks, the kernels on the ears were surface-sterilized by spraying with or soaking in 80%<br>
ethanol.<br>
The Agrobacterium strain ABI, and an Agrobacterium tumefaciens binary vector<br>
system were used for the transformations. Plasmids pMON72506 and pMON72510 were<br>
transformed into Agrobacterium tumefaciens according to methods well known in the art.<br>
Prior to inoculation of corn cells the Agrobacterium cells are grown overnight at room<br>
temperature in AB medium (Chilton et al., Proc. Nat. Acad. Sci. U.S.A., 71:3672-3676<br>
(1974)) comprising appropriate antibiotics for plasmid maintenance and 200 jjM<br>
acetosyringone. Immediately prior to inoculation the Agrobacterium cells were pelleted by<br>
centrifugation, and resuspended in either CRN122 medium (2.2 g/L MS (Murashige and<br>
Skoog, Physiol Plant, 15:473-497 (1962)) basal salts, 2 mg/L glycine, 0.5 g/L niacin, 0.5 g/1<br>
L-pyridoxine-HCl, 0.1 g/L thiamine, 115 mg/L L-proline, 36 g/L glucose, and 68.5 g/L<br>
sucrose, pH 5.4) or CRN347 medium (CRN122 medium except with 0.44 g/L MS salts,<br>
10 g/L glucose, 20 g/L sucrose, and 100 mg/L ascorbic acid) containing 200 uM<br>
acetosyringone and 20 jaM Ag NOs.<br>
The immature maize embryos were excised from individual kernels, immersed in an<br>
Agrobacterium suspension, and incubated at room temperature for 5-15 minutes. The<br>
Agrobacterium solution is then removed, and the inoculated immature embryos were<br>
transferred scutellum-side up from inoculation CRN122 medium to co-cultivation CRN123<br>
medium (CRN122 medium except with 0.5 mg/L additional thiamine-HCl, 20 g/L sucrose,<br>
10 g/L glucose and 3 mg/L 2,4 D) containing 200 fiM acetosyringone and 20 faM silver nitrate<br>
and incubated at 23°C for 1 day. Alternatively, excised embryos were cultured for 8-11 days<br>
in 21IV medium (3.98 g/L Chu N6 salts (Chu, C.C., The N6 medium and its application to<br>
anther culture of cereal crops, in Plant Tissue Culture Plant Tissue Culture. Proceedings of<br>
the Peking Symposium, Boston, MA (1981), 43-50), 0.5 mg/L thiamine HC1, 0.5 mg/L<br>
nicotinic acid; 1.0 mg/L 2,4 D, 20 g/L sucrose, 0.69 g/L L-proline, 0.91 g/L L-asparagine<br>
monohydrate, 1.6 g/L MgCl2 hexahydrate, 0.1 g/L casein hydrolysate, 0.5 g/L MES, 0.1 g/L<br>
myo-inositol, and 16.9 mg/L silver nitrate, pH 5.8 solidified with 2 g/L Gelgro) and calli were<br>
inoculated with Agrobacterium CRN347 medium suspensions at 23°C for 3 days without<br>
adding additional media.<br>
The embryos were then transferred to CRN220 selection medium (4.4 g/L MS salts,<br>
1.3 mg/L nicotinic acid, 0.25 mg/L pyridoxine HC1, 0.25 mg/L thiamine HC1, 0.25 mg/L<br>
calcium pantothenate, 30 g/L sucrose, 12 mM proline, 0.05 g/L casamino acids, 500 mg/L<br>
carbenicillin, 200 mg/L paromomycin, 2.2 mg/L picloram, 0.5 mg/L 2,4 D and 3.4 mg/L<br>
silver nitrate, pH 5.6 solidified with 7 g/L Phytagar), or calli are transferred to CRN344<br>
selection medium (3.98 g/L Chu N6 salts, 1.0 mg/L thiamine HC1, 0.5 mg/L nicotinic acid;<br>
1.0 mg/L 2,4 D, 20 g/L sucrose, 0.69 g/L L-proline, 0.91 g/L L-asparagine monohydrate,<br>
1.6 g/L MgCL hcxahydrate, O.I g/L casein hydrolysale, 0.5 g/L ML'S, 0.1 g/L niyo-inosilol,<br>
500 nig/L carbenicillin, 200 nig/L paromomyciri and 16.9 mg/L silver nitrate, pi I 5.8<br>
solidified with 6 g/L Phytugar). After 2-3 weeks at 27°C in the dark, surviving tissues were<br>
transferred to the same selection medium and cultured for up to an additional 2 weeks or<br>
transferred to regeneration medium as described below.<br>
Plant regeneration is achieved by transferring the putative Iransgcnic callus from<br>
CRN220 to CRN232 medium (CRN220 medium lacking picloram, 2,4-D, and silver nilr.de,<br>
and containing 3.52 mg/L ben/ylaminopurine (BAP) and 250 mg/L carbenicillin) or from<br>
CRN344 medium to 217A medium (21IRTTV lacking sils-er nitrate, 2,4 D, and paromomycin<br>
and containing 3.5? mg/L BAP and 250 mg/L, carbenicillin) and incubating lor 5-7 days at<br>
27°C. Tissue is then transferred from CRN232 medium to CRN264 medium (4.4 g/L MS<br>
salts, 1.3 g/L nicotinie acid, 0.25 mg/L pyridoxine MCI, 0,25 mg/L thiaminc MCI, 0.25 mg/L<br>
calcium pantolhcnate, 10 g/L glucose, 20 g/L maltose, 1 mM L-asparagine, O.I g/L<br>
inyo-inosilol, 250 mg/L carbenicillin and 100 mg/L paromomycin, pll 5.8 solidified \vilh<br>
6 g/L Phylagar) or from 2I7A medium to CRN346 medium (4.4 g/L MS salts, MS vitamins,<br>
60 g/L sucrose, 0,05 g/L rnyo-inositol, 250 mg-'L. carbenicillin, 75 mg/L paromomycin, pll 5.8<br>
solidified with 6 g/L KOI I) in Phytalrays, and incubated in the light at 2S°C until shoots with<br>
well-developed roots \vere produced (typically 2-3 weeks). These developing planilcts were<br>
then transferred (o soil, hardened off in a growth chamber at 27°C, 80% humidity, ami low<br>
light intensity for approximately 1 week, and then transferred lo a greenhouse and the RO<br>
plants were grown under standard greenhouse conditions. The RO plants were reciprocally<br>
crossed and both immature/developing kernels and mature kernels were collected from each<br>
of the resulting plants for subsequent analyses. The results of the analyses are described<br>
below in Example 6.<br>
These developing planilcts were then transferred to soil, hardened off in a growth<br>
chamber at 27"C, 80% humidity, and low light intensity for approximately 1 week, and then<br>
transferred to a greenhouse. The RO plants were then grown under standard greenhouse<br>
conditions. Fertile RO plants were crossed to a non-lransgcnic recurrent inbred, with the RO<br>
plant serving as either the female or male (or occasionally both) in the cross. Both developing<br>
and mature Fl kernels were collected and analy/cd, from each of the resulting ears as<br>
described in Example 4. The results of the analyses are reported below in Example 5.<br>
Example 4<br>
This example provides the analytical procedures to determine oil, protein, and starch<br>
levels in kernels from transgenic plants containing the HOI001 GBSS gene or the LH59<br>
GBSS gene.<br>
Oil Content Analysis: Oil levels (on a mass basis and as a percent of tissue weight)<br>
of first generation single corn kernels and dissected germ and endosperm are determined by<br>
low-resolution H nuclear magnetic resonance (NMR) (Tiwari et al, JAOCS, 51:104-109<br>
(1974); or Rubel, JAOCS, 71:1057-1062 (1994)), whereby NMR relaxation times of single<br>
kernel samples are measured, and oil levels are calculated based on regression analysis using a<br>
standard curve generated from analysis of corn kernels with varying oil levels as determined<br>
gravimetrically following accelerated solvent extraction.<br>
To compare oil analyses of transgenic and non-transgenic kernels, the presence or<br>
absence of the transgene is determined by detection (or lack thereof) of a transgene-specific<br>
517 bp PCR product, using a sequence within the Hsp70 intron as a 5' primer, and a sequence<br>
within the HOI001 GBSS gene as a 3' primer:<br>
5' primer (primer number 19056):<br>
5'-ATCTTGCTCGATGCCTTCTC-3' [SEQ ID NO: 16],<br>
3' primer (primer number 18986):<br>
5'-GCCTTCGCTTGTCGTGGGT-3' [SEQ ID NO: 17].<br>
Oil levels in advanced generation seed are determined by NIT spectroscopy, whereby<br>
NIT spectra of pooled seed samples harvested from individual plants are measured, and oil<br>
levels are calculated based on regression analysis using a standard curve generated from<br>
analysis of com kernels with varying oil levels, as determined gravimetrically following<br>
accelerated solvent extraction or elemental (%N) analysis, respectively.<br>
One-way analysis of variance and the Student's T-test are performed to identify<br>
significant differences in oil (% kernel weight) between seed from marker positive and marker<br>
negative plants.<br>
Alternatively, oil levels of pooled kernels from single ears are determined by lowresolution<br>
'H nuclear magnetic resonance (NMR) (Tiwari et al., JAOCS, 51:104-109 (1974);<br>
or Rubel, JAOCS, 71:1057-1062 (1994)), whereby NMR relaxation times of pools of kernels<br>
are measured, and oil levels are calculated based on regression analysis using a standard curve<br>
generated from analysis of com kernels with varying oil levels as determined gravimetrically<br>
following accelerated solvent extraction.<br>
Protein Analyses: For kernel protein analysis, small bulk samples consisting of<br>
50-100 kernels for each treatment are measured using near infrared reflectance spectroscopy<br>
(InfraTec model 1221, Teccator, Hogannas Sweden). This procedure is based upon the<br>
observation that a linear relation exists between the absorption of near infrared radiation and<br>
the quantity of chemical constituents comprised in a typical grain sample. Prior to analyzing<br>
unknown samples, spectral data is collected with calibration samples that are subsequently<br>
analyzed using a nitrogen combustion analysis technique (Murray, I., and P.C. Williams,<br>
1987, Chemical Principles of Near-infrared Technology, In Near-Infrared Technology in the<br>
Agricultural and Food Industries, P. Williams and K. Norris eds.). A multivariate model is<br>
developed using the spectral data from the spectrometer and the primary data. In the present<br>
case a PLS-1 (Partial Least Squares Regression Type I) multivariate model is constructed<br>
using 152 calibration samples. Each unknown sample is scanned on the spectrometer at least<br>
5 times and its protein content predicted with each scan. Each time the sample is scanned it is<br>
added back to the sample cuvette to minimize multiplicative scattering effects, which are not<br>
correlated to chemical property of interest. The predicted starch is averaged for the multiple<br>
scans and then reported for each sample.<br>
Starch analyses: For kernel starch analysis, small bulk samples consisting of 50-100<br>
kernels for each treatment are measured using near infrared reflectance spectroscopy<br>
(InfraTec model 1221, Teccator, Hogannas Sweden). This procedure is based upon the<br>
observation that a linear relation exists between the absorption of near infrared radiation and<br>
the quantity of chemical constituents comprised in a typical grain sample. Prior to analyzing<br>
unknown samples, spectral data is collected with calibration samples that are subsequently<br>
analyzed using standard wet chemistry analytical techniques (Murray, I., and P.C. Williams,<br>
1987, Chemical Principles of Near-infrared Technology, In Near-Infrared Technology in the<br>
Agricultural and Food Industries, P. Williams and K. Norris eds.). A multivariate model is<br>
developed using the spectral data from the spectrometer and the primary data. Each unknown<br>
sample is scanned on the spectrometer at least 5 times and its starch content predicted with<br>
each scan. Each time the sample is scanned it is added back to the sample cuvette to minimize<br>
multiplicative scattering effects, which are not correlated to chemical property of interest.<br>
The predicted starch is averaged for the multiple scans and then reported for each sample.<br>
Example 5<br>
This example describes the analysis of kernels from plants transformed with the<br>
HOI001 GBSS and the GBSS from LH59.<br>
Kernels from a total of 54 Iransgcnic events expressing the HOI001 GBSS Iransgenic<br>
allele were analyzed using the procedures set forth in Example 4. Table 1 shows whole kernel<br>
oil levels of Iransgcnic (positive) and nontransgenic (negative) Fl kernels from ears of 20<br>
transgenic events analyzed by the single kernel NMR procedure described in Example 4.<br>
Only results from events with a statistically significant increase in oil (p
The results demonstrate that transgenic kernels from ears of 20 of the 54 events had<br>
statistically significant increases in whole kernel oil content (% dry weight) relative to<br>
nontransgenic kernels on the same ear. No events had a statistically significant decrease in<br>
oil.<br>
(Table Removed)<br>
Transgenic kernels from RO plants pollinated by non-lransgenic inbred pollen (for<br>
example, pedigree ZM_S67336/LHl72, positive) had both a higher frequency of a significant<br>
oil increase (15/29 plants analyzed) and a higher average significant oil increase (0,61%)<br>
relative to kernels from non-transgenic inbred plants pollinated by Iransgenic pollen (for<br>
example, pedigree LH172/ZM_S66817, negative) from an RO male parent (5/37 plants<br>
analyzed, 0.34% significant oil increase). These results suggest that the greater transgene<br>
dosage found in the endosperm of kernels from the RO plants, due to maternal inheritance<br>
effects, may result in a greater increase in oil.<br>
- 3 7 -<br>
Similarly, kernels from a total of 15 transgenic events containing the LH59 GBSS<br>
transgenic allele were analyzed. None of the kernels from ears of any of the events had<br>
statistically significant increases in whole kernel oil content (% dry weight) relative to<br>
nontransgenic kernels on the same ear, indicating that the increase in oil was unique to the<br>
HOI001 GBSS allele.<br>
Example 6<br>
This example describes the increase in oil levels obtained in transgenic F2 kernels<br>
from field-grown plants.<br>
To ascertain the impact of the HOI001 GBSS gene on kernel oil levels of field-grown<br>
plants, 24-48 segregating Fl seed from each of 40 events were planted in a field nursery.<br>
Developing plants were screened for the presence of the transgenic cassette by a non-lethal<br>
kanamycin resistance assay, whereby an antibiotic solution (0.1%(w/v) kanamycin and 0.1%<br>
(w/v) paromomycin) is applied to the leaf surface and scored for the presence (nontransgenic)<br>
or absence (transgenic) of necrotic lesions 1 week after antibiotic application. Kernels were<br>
isolated from the ears of both transgenic plants and non-transgenic plants, and then were<br>
assayed for kernel oil, protein, and starch by Near-Infrared Transmittance Spectroscopy.<br>
Table 2 shows the mean whole kernel oil levels and the increase in whole kernel oil<br>
levels (Delta) in ears from plants containing (positive) and lacking (negative) the transgenic<br>
cassette containing the selectable marker and the HOI001 GBSS gene. Oil levels were<br>
determined by the NIT procedure described in Example 4, and only events with a statistically<br>
significant increase in oil (p
(Table Removed)<br>
The results show whole kernel oil level was increased in transgenic ears relative to<br>
nontransgenic ears (p
Table 3 shows the mean kernel starch levels (%) and the change in kernel starch levels<br>
in ears from plants containing (positive) and lacking (negative) the transgenic cassette<br>
containing the selectable marker and the HOIOOl GBSS gene. Only events with a statistically<br>
significant increase in oil (p
Table 4 shows mean kernel protein levels (%) and the change in kernel protein levels<br>
in ears from plants containing (positive) and lacking (negative) the transgenic cassette<br>
containing the selectable marker and the HOIOOl GBSS gene. Only events with a statistically<br>
significant increase in oil (p
Based on NIT analysis, starch levels in events with increases in oil were lowered<br>
slightly (Table 3), and protein levels were mostly unchanged (Table 4).<br>
(Table Removed)<br>
Example 7<br>
This example describes the increase in oil levels obtained in transgenic F2 hybrid<br>
kernels from field-grown plants.<br>
To ascertain the impact of the HOI001 GBSS gene on kernel oil levels of hybrid fieldgrown<br>
plants, 24-48 segregating Fl seed from each of 14 events, having sufficient seed, were<br>
planted in a field nursery. Developing plants were screened for the presence of the transgenic<br>
cassette by the non-lethal Kanamycin resistance assay, described above in Example 6. Pollen<br>
from transgenic plants was used to pollinate the stiff-stalk inbred LH244. The segregating Fl<br>
transgenic seed generated was then planted and the resultant plants were screened for the<br>
presence of the transgene by the non-lethal Kanamycin resistance assay. F2 hybrid kernels<br>
were isolated from ears from transgenic plants and non-transgenic plants, and assayed for<br>
kernel oil by Nuclear Magnetic Resonance Spectroscopy, as described in Example 4.<br>
Table 5 shows mean whole kernel oil levels and the increase (Delta) in whole kernel<br>
oil levels in ears from hybrid plants containing (positive) and lacking (negative) the transgenic<br>
cassette containing the selectable marker and the HOI001 GBSS gene. Oil levels were<br>
determined by the bulk set NMR procedure described in Example 4, and only events with a<br>
statistically significant increase in oil (p
oil level was increased in transgenic ears relative to nontransgenic ears (p
events analyzed.<br>
Example 8<br>
This example describes the elevation of GBSS activity in corn endosperm tissue<br>
expressing the HOI001 GBSS gene.<br>
Developing ears from Fl plants screened for the presence of the transgenic cassette by<br>
the non-lethal Kanamycin resistance assay were harvested and immediately frozen at 24 days<br>
post pollination. Segregating F2 kernels were removed from the ear, then dissected into germ<br>
and endosperm fractions. Individual dissected kernels were identified as transgenic or<br>
nontransgenic by screening for the ability to PCR-amplify a portion of the transgenic cassette<br>
from genomic DNA isolated from individual germs using transgene-specific primers, as<br>
described in Example 4. For each of six events, approximately 10 endosperms from the<br>
corresponding transgenic and nontransgenic kernels were pooled separately.<br>
Each endosperm pool was ground to a fine powder with a mortar and pestle under<br>
liquid nitrogen, and starch granules were isolated in triplicate according to the procedure of<br>
Shure et al., Cell, 35(l):225-233 (1983). Granule-bound starch synthase activity was assayed<br>
on the isolated granules using the method of Vos-Scheperkeuter et al., Plant Physiol., 82:411-<br>
416(1986).<br>
Table 6 shows the granule-bound starch synthase activity (pmol/min/mg starch) in<br>
developing F2 endosperm containing or lacking the HOI001 GBSS transgenic cassette.<br>
Values shown are means and standard errors of triplicate assays. The data indicates that<br>
starch granules from transgenic kernels generally had elevated GBSS activity, indicating that<br>
the effect of the HOI001 allele on oil levels is not a function of reducing overall GBSS<br>
activity, but functions by the addition of an activity uniquely encoded by the HOI001 GBSS<br>
gene.<br>
Example 9<br>
This example describes the isolation and sequencing of the coding region of the GBSS<br>
cDNA from com line HOI001.<br>
mRNA was extracted from developing corn endosperm tissue from HOI001, 22 days<br>
after pollination, using a procedure adapted from Opsahl-Ferstad et aL, Plant J., 12(10):235-<br>
246 (1997). Briefly, developing endosperm from 3 separate kernels was pooled, frozen in<br>
liquid nitrogen, and then pulverized with a mortar and pestle. Approximately 50 mg frozen<br>
powdered endosperm was extracted with 0.5 mL buffer (0.5 M LiCl, 10 mM EDTA, 5 mM<br>
dithiotlireitol.100 mM Tris-HCl, pH 8.0, l%(w/v) SDS). This aqueous extract was then<br>
extracted with phenol:chloroform:isoamyl alcohol (25:24:21), and the organic fraction was<br>
discarded. Nucleic acids were precipitated from the aqueous fraction by addition of an equal<br>
volume of isopropyl alcohol followed by centrifugation. The resulting supernatant was<br>
discarded. The pellet containing the mRNA was washed twice with 70% ethanol, dried, and<br>
then resuspended in 50 jaL water containing 0.1% (v/v) diethylpyrocarbonate.<br>
First-strand cDNA was synthesized from the isolated mRNA using the Clontech<br>
SMART™ cDNA synthesis system (BD Biosciences). This first-strand cDNA was used as<br>
template to amplify HOI001 GBSS cDNA sequences using primers containing an EcoRV<br>
restriction site followed by 18 bp of the predicted translational start site (5 primer) and a<br>
Sse83871 restriction site followed by 17 bp of the predicted 3' end up to the translation stop<br>
site (3 primer):<br>
5' Primer (primer number 20095):<br>
5'-GGATATCACCATGGCGGCTCTGGCCACG-3' [SEQ ID NO: 9],<br>
3' Primer (primer number 20092):<br>
5'-GTCCTGCAGGCTACACATACTTGTCCA-3( [SEQ ID NO: 10].<br>
The resulting 1.8 kB amplification products from independent amplification reactions<br>
were isolated by agarose gel electrophoresis, cloned independently into the PCR 2.1 cloning<br>
vector using the TOPO TA cloning kit (Invitrogen), and then transformed into an E. coli host.<br>
Multiple colonies were isolated from each transformation, plasmid DNA was prepared from<br>
cultures grown from each colony, and then the insert in each plasmid preparation was<br>
sequenced. Alignment of these sequences generated a consensus sequence containing an open<br>
reading frame equivalent to that predicted to be encoded by the HOI001 GBSS gene, although<br>
no specific insert sequence was equivalent to the consensus. One clone (designated 7345705-<br>
10) had an insert sequence with single base pair deletion relative to the consensus. This clone<br>
was used to generate a plasmid (designated pMON81463) containing the consensus sequence<br>
by inserting the additional nucleotide using the Quick-Change mutagenesis kit (Stratagene).<br>
This sequence, representing the coding region of the HOI001 GBSS cDNA, is listed in SEQ<br>
ID NO: 11.<br>
Example 10<br>
This example sets forth the construction of plant transformation vectors containing the<br>
HOI001 GBSS cDNA coding region [SEQ ID NO: 11], designed to obtain different levels,<br>
timing and spatial patterns of expression, and the subsequent transformation of corn.<br>
A plant transformation vector containing the HOI001 GBSS coding region driven by a<br>
Z27 promoter was constructed. The HOI001 GBSS coding region was isolated from<br>
pMON81463 by restriction digest with EcoRV and Sse83871, and cloned into the binary<br>
vector pMON71274. This binary vector contains left and right borders for T DNA transfer; a<br>
rice Actin promoter: :rice Actin intron::CP4::nos 3' UTR, plant selectable marker element; and<br>
plant expression cassette sequences which include a 1.1 kb Z27 promoter (bp 19-1117 of<br>
Genbank Accession #878780, Lopes et al., Mol. Gen. Genet., 247(5):603 613 (1995)) for<br>
endosperm expression; a corn hsp70 intron (base pairs 4-153 of the maize gene for heat shock<br>
70 exon 2, Genbank Accession #X03679, Rochester et al., EMBOJ., 5:451-458 (1986)), and a<br>
globulin 3' UTR. The resulting plasmid was named pMON81464 (Figure 7).<br>
A second plant expression binary vector containing the wheat high molecular weight<br>
glutenin promoter (bp 2647-3895 of Genbank Accession X12928, version X12928.3,<br>
originally described in Anderson et al, Nucleic Acids Res., 17:461-462 (1989)) and the corn<br>
hsp70 intron, fused to the GBSS coding region, fused to the wheat HSP17n3' UTR (bp532-<br>
741 of Gen Bank Accession X13431, version X13431.1, McElvain and Spiker, Nucleic Acids<br>
Res., 17:1764 (1989)), was constructed. A sequence containing the wheat high molecular<br>
weight glutenin promoter fused to the corn hsp70 intron was amplified from an intermediary<br>
vector using 5' and 3" primers containing AscI and NotI restriction sites, respectively:<br>
5' Primer (primer number 21084):<br>
5'-GGCGCGCCGTCGACGGTATCGATAAGCTTGC-3' [SEQ ID NO: 12],<br>
3' Primer (primer number 21085):<br>
5'-GCGGCCGCCCGCTTGGTATCTGCATTACAATG-3' [SEQ ID NO: 13].<br>
The amplification product, containing the promoter and intron fragment with the<br>
introduced restriction sites, was purified by agarose gel electrophoresis and cloned into<br>
pCR2.1 TOPO (hivitrogen) to generate a plasmid vector for E. coli transformation (pOP28).<br>
After transformation into an E. coli vector, plasmid DNA was isolated, digested with Asci and<br>
NotI, and the purified fragment was cloned into the binary vector pMON71274 to generate a<br>
vector (pOP29) containing a cassette with the wheat high molecular weight glutenin promoter<br>
fused to the com HSP70 intron fused to the globulin 3' UTR. The HOI001 GBSS coding<br>
region was isolated by digestion of pMON81464 with NotI/Sse83871 and cloned into pOP29<br>
to generate the binary vector pOP31 containing an expression cassette with the wheat high<br>
molecular weight glutenin promoter fused to the com HSP70 intron fused to the HOI001<br>
GBSS coding region fused to the globulin 3' UTR. The promoter/intron/HOIOOl GBSS<br>
coding region fragment was then isolated from pOP31 by digestion with AscI/Sse83871 and<br>
then cloned into the plant binary vector pMON71290 containing a gene of interest cassette<br>
with the TR7 3' UTR to generate pOP35, containing an expression cassette with the wheat<br>
high molecular weight glutenin promoter fused to the corn HSP70 intron fused to the HOI001<br>
GBSS coding region fused to the TR7 3' UTR. The promoter/intron/HOIOOl GBSS coding<br>
region fragment was then isolated from pOP35 by digestion with AscI/Sse83871 and then<br>
cloned into the plant binary vector pMON67647, containing a gene of interest cassette with<br>
the wheat HSP17 3' UTR. The resulting plasmid contained an expression cassette with the<br>
wheat high molecular weight glutenin promoter fused to the com HSP70 intron fused to the<br>
HOI001 GBSS coding region fused to the wheat HSP17 3' UTR. This plasmid, was named<br>
pMON68298, is shown in Figure 8.<br>
A third plant expression binary vector containing the promoter and 5' UTR of the<br>
HOI001 GBSS gene fused to the HOI001 GBSS coding region, fused to the corn globulin<br>
3' UTR, was constructed. The HOI001 GBSS promoter and 5' UTR (which also contained the<br>
first predicted intron) was isolated by PCR amplification from pMON72506, using a 5' primer<br>
that contains the restriction site for Pmel:<br>
5' Primer (primer number 20362):<br>
5'-GATCGTTTAAACGTTCGTGTGGCAGATTCATC-3' [SEQ ID NO: 14],<br>
3" Primer (primer number 20363):<br>
-44-<br>
5'-GACGTGGCCAGAGCCGCCATGCCGATTAATCCACTGCATAG-3' [SEQ ID<br>
NO: 15].<br>
The amplification product, a fragment containing 1125 bp upstream of the predicted<br>
HOI001 GBSS translational start site and 20 bp of the predicted coding sequence from<br>
pMON72506 (corresponding to bp 17-1162 of SEQ ID NO: 1), was purified by agarose gel<br>
electrophoresis and cloned into pCR2.1 TOPO (Invitrogen) to generate pMON81466.<br>
The HOI001 GBSS coding region was removed from pMON81463 and cloned into the<br>
vector pMONS 1466 to generate pMON81468, containing the HOI001 GBSS promoter/51<br>
UTR fused to the HOI001 GBSS coding region, with 45 bp extraneous polylinker sequence<br>
between the promoter/UTR and coding region elements. This extraneous sequence was then<br>
deleted by digestion of pMON81468 with Mlul to remove a 780 bp fragment spanning the<br>
extraneous sequence, then reannealing with the analogous 735 bp fragment (lacking the<br>
extraneous sequence), generating pMON81469. This 735 bp fragment was generated by<br>
digestion of pMON72506 with Mlul and isolating the resulting fragment. This entire<br>
promoter/UTR/coding region sequence was then isolated from pMON81469 by digestion with<br>
Pmel and Sse83871, and cloned into the binary vector pMON71274 to generate the binary<br>
vector pMON81465. This vector contained an expression cassette with the promoter and<br>
5' UTR of the HOI001 GBSS gene fused to the GBSS coding region fused to the com<br>
globulin 3' UTR (Figure 9).<br>
These three plant transformation vectors are transformed into an elite corn inbred<br>
(LH244) (Corn States Hybrid Serv., LLC, Des MoinesJA). Briefly, ears containing immature<br>
embryos are harvested approximately 10 days after pollination and kept refrigerated at 4°C<br>
until use (up to 5 days post-harvest). The preferred embryo size for this method of<br>
transformation is -1.0-2.0 mm. This size is usually achieved 10 days after pollination inside<br>
the greenhouse with the growth conditions of an average temperature of 87°F, day length of<br>
14 hours with supplemental lighting supplied by GE 1000 Watt High Pressure Sodium lamps.<br>
Immature embryos are isolated from surface sterilized ears and directly dropped into<br>
the prepared Agrobacterium cell suspension in a 1.5-mL microcentrifuge tube. The isolation<br>
lasts continuously for 15 minutes. The tube is then set aside for 5 minutes, resulting in a<br>
total inoculation time for individual embryos from 5 to 20 minutes. After the Agrobacterium<br>
cell suspension is removed using a fine tipped sterile transfer pipette, the immature embryos<br>
are transferred onto a co-culture medium (Table 7). The embryos are then placed on the<br>
medium with the scutellum side facing up. The embryos are cultured in a dark incubator<br>
(23°C) for approximately 24 hours.<br>
-45-<br>
The embryos are then transferred onto a modified MS medium (MSW50, Table 7)<br>
supplemented with 0.1 or 0.25 mM glyphosate and 250 mg/L carbenicillin to inhibit<br>
Agrobacterium in Petri dishes (100 mm x 25 mm). The cultures are incubated in a dark<br>
culture room at 27°C for 2-3 weeks. All the callus pieces are then transferred individually<br>
onto the first regeneration medium (MS/6BA, Table 7) supplemented with the same levels of<br>
glyphosate. The cultures are grown on this medium and in a culture room with 16 hours<br>
light/8 hours dark photoperiod and 27°C for 5-7 days. They are then transferred onto the<br>
second 15 regeneration medium (MSOD, Table 7) in Petri dishes (100 mm x 25 mm) for<br>
approximately 2 weeks. All the callus pieces with regenerating shoots and living tissue are<br>
transferred onto the same medium contained in phytatrays for shoots to grow further prior to<br>
being transferred to soil (approximately 2-4 weeks). The regeneration media (MS6BA and<br>
MSOD) are all supplemented with 250 mg/L carbenicillin and 0.1 or 0.25 mM glyphosate.<br>
These developing plantlets are then transferred to soil, hardened off in a growth<br>
chamber at 27°C, 80% humidity, and low light intensity for approximately 1 week,<br>
and then transferred to a greenhouse and grown under standard greenhouse conditions. The<br>
resulting kernels are collected and analyzed as described in Example 4. The results indicate<br>
that the different promoters have different impacts on oil accumulation based upon the<br>
strength and timing of the expression of the HOI001 GBSS coding region.<br>
(Table Removed)<br>
Co-culture medium was solidified with 5.5 mg/1 low EEO agarose. All other media<br>
were solidified with 7 g/1 Phytagar for NPTII selection and with 3 g/1 phytagel for glyphosate<br>
selection.<br>
Example 11<br>
This example sets forth the use of the polymorphisms in the HOI001 GBSS gene as<br>
molecular markers to accelerate incorporation of HOI001 GBSS sequence polymorphisms<br>
into other corn germplasm with the result of increasing oil in the kernel.<br>
The present invention provides a com plant with increased kernel oil selected for by<br>
use of marker assisted breeding wherein a population of plants are selected for the presence of<br>
a polymorphism sequence unique to the HOI001 GBSS gene (SEQ ID NO: 1). Example 1,<br>
above, lists polymorphisms unique to the HOI001 GBSS sequence, that is not found in either<br>
the LH59 GBSS sequence or the published sequence (Shure et al, supra).<br>
The selection of plants having the HOI001 GBSS gene for high oil comprises probing<br>
genomic DNA of the resulting plants, through the selection process, for the presence of the<br>
molecular marker for the HOI001 GBSS gene. The molecular marker is a DNA molecule<br>
representing a unique polymorphism in the HOI001 GBSS gene that functions as a probe or<br>
primer to a target HOI001 GBSS in a plant genome. The selected polymorphism may or may<br>
not be from a coding region of the gene. The plants containing the HOI001 GBSS gene are<br>
continued in the breeding and selection process.<br><br><br><br><br><br>
WE CLAIM:<br>
1.A substantially purified nucleic acid molecule used interalia for making transgenic maize plants and seeds, selected from the group consisting of:<br>
a)	a nucleic acid molecule comprising SEQ ID NO: 1 or the complement thereof.<br>
b)	a nucleic acid molecule comprising SEQ ID NO: 11 of the complement thereof;<br>
c)	a nucleic acid molecule which encodes a polypeptide having at least 95% amino acid identity with SEQ ID No.: 3.<br>
2.An expression cassette comprising a nucleic acid as claimed in claim 1, wherein said nucleic acid molecule is operably linked to a promoter, which is functional in a plant cell.<br>
3.	A method of producing a plant having increase levels of oil production,<br>
wherein the method comprises:<br>
(a)	transforming a plant with an expression cassette comprising a nucleic acid<br>
molecule as claimed in claim 1 selected from the group consisting of;<br>
i) a nucleic acid molecule comprising SEQ ID NO: 1 or the complement thereof;<br>
ii) a nucleic acid molecule comprising SEQ ID NO 11 or the complement thereof; and<br>
iii) a nucleic acid molecule which encodes a polypeptide having at least 95% amino acid identity with SEQ ID NO: 3;<br>
wherein said expression cassette further comprises a promoter region functional in a plant cell, operably linked to said nucleic acid molecule, and<br>
(b)	growing the transformed plant.<br>
4.	The method as claimed in claim 3, wherein the promoter region is an<br>
endosperm promoter region.<br><br>
' 5. The method as claimed in claim 4, wherein the promoter region in the Z27 promoter.<br>
6. A method of selecting com germplasm, comprising the steps of:<br>
a)	identifying at least one polymorphism unique to the  HOIOOl   GBSS sequence represented in SEQ ID NO: 1 as claimed in claim 1;<br>
b)	selecting a fragment of SEQ ID NO: 1 containing at least part of one of the identified polymorphisms to be used as a molecular marker;<br>
c)	assaying com plants for the presence of the marker; and<br>
d)	selecting plants that contain the marker.<br><br><br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtREVMTlAtMjAwNi1BYnN0cmFjdC0oMDktMDMtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">98-DELNP-2006-Abstract-(09-03-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtREVMTlAtMjAwNi1BYnN0cmFjdC0oMTAtMDktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">98-DELNP-2006-Abstract-(10-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtZGVsbnAtMjAwNi1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">98-delnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtREVMTlAtMjAwNi1Bc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">98-DELNP-2006-Assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtREVMTlAtMjAwNi1DbGFpbXMtKDA5LTAzLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">98-DELNP-2006-Claims-(09-03-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtREVMTlAtMjAwNi1DbGFpbXMtKDEwLTA5LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">98-DELNP-2006-Claims-(10-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtZGVsbnAtMjAwNi1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">98-delnp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtREVMTlAtMjAwNi1Db3JyZXNwb25kZW5jZS1PdGhlcnMtKDA5LTAzLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">98-DELNP-2006-Correspondence-Others-(09-03-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtREVMTlAtMjAwNi1Db3JyZXNwb25kZW5jZS1PdGhlcnMtKDEwLTA5LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">98-DELNP-2006-Correspondence-Others-(10-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtREVMTlAtMjAwNi1Db3JyZXNwb25kZW5jZS1PdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">98-DELNP-2006-Correspondence-Others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtZGVsbnAtMjAwNi1kZXNjcmlwdGlvbiAoY29tcGxldGUpLTEwLTA5LTIwMDgucGRm" target="_blank" style="word-wrap:break-word;">98-delnp-2006-description (complete)-10-09-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtZGVsbnAtMjAwNi1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">98-delnp-2006-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtZGVsbnAtMjAwNi1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">98-delnp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtREVMTlAtMjAwNi1Gb3JtLTEtKDEwLTA5LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">98-DELNP-2006-Form-1-(10-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtZGVsbnAtMjAwNi1mb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">98-delnp-2006-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtREVMTlAtMjAwNi1Gb3JtLTE4LnBkZg==" target="_blank" style="word-wrap:break-word;">98-DELNP-2006-Form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtREVMTlAtMjAwNi1Gb3JtLTItKDEwLTA5LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">98-DELNP-2006-Form-2-(10-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtZGVsbnAtMjAwNi1mb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">98-delnp-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtREVMTlAtMjAwNi1Gb3JtLTMtKDA5LTAzLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">98-DELNP-2006-Form-3-(09-03-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtREVMTlAtMjAwNi1Gb3JtLTMtKDEwLTA5LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">98-DELNP-2006-Form-3-(10-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtZGVsbnAtMjAwNi1mb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">98-delnp-2006-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtZGVsbnAtMjAwNi1mb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">98-delnp-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtREVMTlAtMjAwNi1HUEEtKDEwLTA5LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">98-DELNP-2006-GPA-(10-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtZGVsbnAtMjAwNi1ncGEucGRm" target="_blank" style="word-wrap:break-word;">98-delnp-2006-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtREVMTlAtMjAwNi1PdGhlcnMtRG9jdW1lbnQtKDA5LTAzLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">98-DELNP-2006-Others-Document-(09-03-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtREVMTlAtMjAwNi1PdGhlcnMtRG9jdW1lbnQtKDEwLTA5LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">98-DELNP-2006-Others-Document-(10-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtZGVsbnAtMjAwNi1wY3QtMTAxLnBkZg==" target="_blank" style="word-wrap:break-word;">98-delnp-2006-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtZGVsbnAtMjAwNi1wY3QtMjEwLnBkZg==" target="_blank" style="word-wrap:break-word;">98-delnp-2006-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtZGVsbnAtMjAwNi1wY3QtMzA0LnBkZg==" target="_blank" style="word-wrap:break-word;">98-delnp-2006-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtZGVsbnAtMjAwNi1wY3QtNDAxLnBkZg==" target="_blank" style="word-wrap:break-word;">98-delnp-2006-pct-401.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtZGVsbnAtMjAwNi1wY3QtNDAyLnBkZg==" target="_blank" style="word-wrap:break-word;">98-delnp-2006-pct-402.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtZGVsbnAtMjAwNi1wY3QtNDA5LnBkZg==" target="_blank" style="word-wrap:break-word;">98-delnp-2006-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtZGVsbnAtMjAwNi1wY3QtNDE2LnBkZg==" target="_blank" style="word-wrap:break-word;">98-delnp-2006-pct-416.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtREVMTlAtMjAwNi1QZXRpdGlvbi0xMzgtKDA5LTAzLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">98-DELNP-2006-Petition-138-(09-03-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtREVMTlAtMjAwNi1QZXRpdG9uLTEzNy0oMTAtMDktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">98-DELNP-2006-Petiton-137-(10-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTgtREVMTlAtMjAwNi1QZXRpdG9uLTEzOC0oMTAtMDktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">98-DELNP-2006-Petiton-138-(10-09-2008).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="233092-a-pyrrolo-3-4-c-pyrazole-compound-of-formula-i.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="233094-method-for-treating-seeds-and-seeds-obtainable-with-the-method.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>233093</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>98/DELNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>26-Mar-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>05-Jan-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MONSANTO TECHNOLOGY, LLC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>800 NORTH LINDBERGH BLVD.,ST. LOUIS,MO 63167, U.S.A</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>TERRY J FOLEY</td>
											<td>800 NORTH LINDBERGH BLVD.ST, LOUIS, MO 63167, U.S.A.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>JOHN R. LEDEAUX</td>
											<td>800 NORTH LINDBERGH BLVD.ST, LOUIS, MO 63167, U.S.A.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>ANNETTE E. WYRICK</td>
											<td>800 NORTH LINDBERGH BLVD.ST, LOUIS, MO 63167, U.S.A.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>THOMAS J .SAVAGE</td>
											<td>800 NORTH LINDBERGH BLVD.ST, LOUIS, MO 63167, U.S.A.</td>
										</tr>
										<tr>
											<td>5</td>
											<td>MONICA P. RAVANELLO</td>
											<td>800 NORTH LINDBERGH BLVD.ST, LOUIS, MO 63167, U.S.A.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 5/10,</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2004/020582</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-06-25</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/483,491</td>
									<td>2003-06-27</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>10/877,645</td>
									<td>2004-06-25</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/233093-a-substantially-purified-nuclelc-acid-molecule-involved-in-elevation-of-oil-levels-in-plants by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:31:08 GMT -->
</html>
