<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/233162-method-for-stimulating-root-growth-comprising-overexpressing-cytokinin-oxidase by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:33:23 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 233162:&quot;METHOD FOR STIMULATING ROOT GROWTH COMPRISING OVEREXPRESSING CYTOKININ OXIDASE&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;METHOD FOR STIMULATING ROOT GROWTH COMPRISING OVEREXPRESSING CYTOKININ OXIDASE&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method for stimulating root growth or for enhancing the formation of lateral or adventitious roots or for altering root geotropism, characterized in that the said method comprises overexpression of a nucleic acid encoding a plant cytokinin oxidase wherein said nucleic acid encoding a plant cytokinin oxidase is selected from the group consisting of: (a) nucleic acids comprising a DNA sequence as given in any of SEQ ID Nos. 27,1,3,5,7,9,11,25,26,28 to 31,33 or 34, or the complement thereof, (b) nucleic acids comprising the RNA sequences corresponding to any of SEQ ID Nos. 27,1,3,5,7,9,11,25,26,28 to 31,33 or 34, or the complement thereof, (c) nucleic acids specifically hybridizing to any of SEQ ID Nos. 27,1,3,5,7,9,11,25, 26, 28 to 31, 33 or 34, or to the complement thereof, (d) nucleic acids encoding a protein comprising the amino acid sequence as given in any of SEQ ID Nos. 2, 4, 6, 8, 10, 12, 32 or 35, or the complement thereof, (e) nucleic acids as defined in any of (a) to (d) characterized in that said nucleic acid is DNA, genomic DNA, cDNA, synthetic DNA or RNA wherein T is replaced by U, (f) a nucleic acid which is degenerated to a nucleic acid as given in any of SEQ ID Nos. 27,1,3,5,7,9,11,25,26,28 to 31,33 or 34, or which is degenerated to a nucleic acid as defined in any of (a) to (e) as a result of the genetic code, (g) nucleic acids which are diverging from a nucleic acid encoding a protein as given in any of SEQ ID Nos. 2,4,6,8,10,12 or 35 or which is diverging from a nucleic acid as defined in any of (a) to (e), due to the differences in codon usage between the organisms, (h) nucleic acids encoding a protein as given in SEQ ID Nos. 2, 4, 6, 8, 10, 12 or 35 or nucleic acids as defined in (a) to (e) which are diverging due to the differences between alleles, (i) nucleic acids encoding a protein as given in any of SEQ ID Nos. 2,4,6,8, 10,12 or 35, (j) functional fragments of nucleic acids as defined in any of (a) to (i) having the biological activity of a cytokinin oxidase, and (k) nucleic acids encoding a plant cytokinin oxidase, or comprising expression, preferably in roots, of a nucleic acid encoding a protein that reduces the level of active cytokinins in plants or plant parts.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FIELD OF THE INVENTION<br>
The present invention generally relates to a method for modifying plant morphological, biochemical and physiological properties or characteristics, such as one or more developmental processes and/or environmental adaptive processes, including but not limited to the modification of initiation or stimulation or enhancement of root growth, and/or adventitious root formation, and/or lateral root formation, and/or root geotropism, and/or shoot growth, and/or apical dominance, and/or branching, and/or timing of senescence, and/or timing of flowering, and/or flower formation, and/or seed development, and/or seed yield, said method comprising expressing a cytokinin degradation control protein, in particular cytokinin oxidase} in the plant, operably under the control of a regulatable promoter sequence such as a cell-specific promoter, tissue-specific promoter, or organ-specific promoter sequence. Preferably, the characteristics modified by the present invention are cytokinin-mediated and/or auxin-mediated characteristics. The present invention extends to genetic constructs which are useful for performing the inventive method and to transgenic plants produced therewith having altered morphological and/or biochemical and/or physiological properties compared to their otherwise isogenic counterparts.<br>
BACKGROUND OF THE INVENTION<br>
Roots are an important organ of higher plants. Their main functions are anchoring of the plant in the soil and uptake of water and nutrients (N-nutrition, minerals, etc.). Thus, root growth has a direct or indirect influence on growth and yield of aerial organs, particularly under conditions of nutrient limitation. Roots are also relevant for the production of secondary plant products, such as defense compounds and plant hormones. Roots are also storage organs in a number of important staple crops. Sugar beet is the most important plant for sugar production in Europe (260 Mill t/year; 38 % of world production). Manioc (cassava), yams and sweet potato (batate) are important starch producers (app. 150 Mill t/year each). Their content in starch can be twice as high as that of potato. Roots are also the relevant organ for consumption in a number of vegetables (e.g. carrots, radish), herbs (e.g. ginger, kukuma) and medicinal plants (e.g.<br>
ginseng). In addition, some of the secondary plant products found in roots are of economic importance for the chemical and pharmaceutical industry. An example is yams, which contain basic molecules for the synthesis of steroid hormones. Another example is shikonin, which is produced by the roots of Lithospermum erythrorhizon in hairy root cultures. Shikonin is used for its anti-inflammatory, anti-tumour and wound-healing properties.<br>
Moreover, improved root growth of crop plants will also enhance competitiveness with weedy plants and will improve growth m arid areas, by increasing water accessibility and uptake.<br>
Improved root growth is also relevant for ecological purposes, such as bioremediation and prevention/arrest of soil erosion.<br>
Root architecture is an area that has remained largely unexplored through classical breeding, because of difficulties with assessing this trait in the field. Thus, biotechnology could have significant impact on the improvement of this trait, because it does not rely on large-scale screenings in the field. Rather, biotechnological approaches require a basic understanding of the molecular components that determine a specific characteristic of the  plant.   Today,  this   knowledge  is  only fragmentary,   and  as   a  consequence, biotechnology was so far unable to realize a break-through in this area. A well-established regulator of root growth is auxin. Application of indole-3-acetic acid (IAA) to growing plants stimulates lateral root development and lateral root elongation (Torrey, Am J Bot 37: 257-264, 1950; Blakely et al., Bot Gaz 143: 341-352, 1982; Muday and Haworth, Plant Physiol Biochem 32: 193-203, 1994). Roots exposed to a range of concentrations of IAA initiated increasing numbers of lateral roots (Kerk ef a/., Plant Physiol, 122: 925-932, 2000). Furthermore, when roots that had produced laterals in response to a particular concentration of exogenous auxin were subsequently exposed to a higher concentration of IAA, numerous supernumerary lateral roots spaced between existing ones were formed (Kerk et at., Plant Physiol, 122: 925-932, 2000). Conversely, growth of roots on agar containing auxin-transport inhibitors, including NPA, decreases the number of lateral roots (Muday and Haworth, Plant Physiol Biochem 32: 193-203, 1994).<br>
Arabidopsis mutants containing increased levels of endogenous IAA have been isolated (Boerjan et al„ Plant Cell 7: 1405-141, 1995; Celenza et al., Gene Dev 9: 2131-2142, 1995; King et al., Plant Cell 7; 2023-2037, 1995; Lehman et al., Cell 85: 183-194, 1996).<br>
They are now known to be alleles of a single locus located on chromosome 2. These mutant seedlings have excess adventitious and lateral roots, which is in accordance with the above-described effects of external auxin application.<br>
The stimulatory effect of auxins on adventitious and lateral root formation suggests that overproduction of auxins in transgenic plants is a valid strategy for increasing root growth. Yet, it is also questionable whether this would yield a commercial product with improved characteristics. Apart from its stimulatory effect on adventitious and lateral root formation, auxin overproduction triggers other effects, such as reduction in leaf number, abnormal leaf morphology (narrow, curled leaves), aborted inflorescences, increased apical dominance, adventitios root formation on the stem, most of which are undesirable from an agronomic perspective (K)ee et al., Genes Devel 1: 86-96, 1987; Kares et al.. Plant Mol Biol 15: 225-236, 1990). Therefore, the major problem with approaches that rely on increased auxin synthesis is a problem of containment, namely to confine the effects of auxin to the root. This problem of containment is not likely overcome by using tissue-specific promoters:  auxins are transported  in the plant and their action  is consequently not confined to the site of synthesis. Another issue is whether auxins will always enhance the total root biomass. For agar-grown plants, it has been noticed that increasing concentrations progressively stimulated lateral root formation but concurrently inhibited the outgrowth of these roots (Kerk et al., Plant Physiol, 122: 925-932, 2000). The  above-mentioned   problems   related  to  containment  of  auxin   effects  and   to maintenance of root outgrowth are solved by the embodiments in the patent claim.<br>
SUMMARY OF THE INVENTION<br>
The present invention relates to a genetic construct comprising a gene encoding a protein with cytokinin oxidase activity from Arabidopsis thaliana. This gene is expressed under control of a regulated promoter. This promoter may be regulated by endogenous tissue-specific or environment-specific factors or, alternatively, it may be induced by application of specific chemicals.<br>
The present invention also relates to a cell or plant containing the genetic construct. The present invention also relates to a method to modify root architecture and biomass by expression of a cytokinin oxidase gene under control of a promoter that is specific to the root or to certain tissues or cell types of the root.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
To by-pass above-mentioned problems associated with increasing auxin biosynthesis, it was decided to follow an alternative approach. We reasoned that down-regulation of biological antagonists of auxins could evoke similar or even superior effects on root growth as compared to increasing auxin levels. Hormone actions and interactions are extremely complex, but we hypothesized that cytokinins could function as auxin antagonists with respect to root growth. Hormone studies on plant tissue cultures have shown that the ratio of auxin versus cytokinin is more important for organogenesis than the absolute levels of each of these hormones, which indeed indicates that these hormones function as antagonists - at least in certain biological processes. Furthermore, lateral root formation is inhibited by exogenous application of cytokinins. Interestingly, also root elongation is negatively affected by cytokinin treatment, which suggests that cytokinins control both root branching and root outgrowth.<br>
Together, current literature data indicate that increasing cytokinin levels negatively affects root growth, but the mechanisms underlying this process are not understood. The sites of cytokinin synthesis in the plant are root tips and young tissues of the shoot. Endogenous concentrations of cytokinins are in the nM range. However, as their quantification is difficult, rather large tissue amounts need to be extracted and actual local concentrations are not known. Also the subcellular compartmentation of cytokinins is not known. It is generally thought that the free base and ribosides are localized in the cytoplasm and nucleus, while glucosides are localized in the vacuole. There exist also different cytokinins with slightly different chemical structure. As a consequence, it is not known whether the effects of exogenous cytokinins should be ascribed to a raise in total cytokinin concentration or rather to the competing out of other forms of plant-borne cytokinins (which differ either in structure, cellular or subcellular location) for receptors, translocators, transporters, modifying enzymes ...<br>
In order to test the hypothesis that cytokinin levels in the root indeed exceed the level optimal for root growth, novel genes encoding cytokinin oxidases (which are cytokinin metabolizing enzymes) were cloned from Arabidopsis thaliana (designated AtCKX) and were subsequently expressed under a strong constitutive promoter in transgenic tobacco and Arabidopsis. Transformants showing AtCKX mRNA expression and increased cytokinin oxidase activity also manifested enhanced formation and growth of roots.<br><br>
Negative effects on shoot growth were also observed. The latter is in accordance with the constitutive expression of the cytokinin oxidase gene in these plants, illustrating the importance of confined expression of the cytokinin oxidase gene for general plant growth properties. Containment of cytokinin oxidase activity can be achieved by using cell-, tissue- or organ-specific promoters, since cytokinin degradation is a process limited to the tissues or cells that express the CKX protein, this in contrast to approaches relying on hormone synthesis, as explained above.<br>
The observed negative effects of cytokinin oxidase expression on shoot growth demonstrate that cytokinin oxidases are interesting targets for the design of or screening for growth-promoting chemicals. Such chemicals should inhibit cytokinin oxidase activity, should preferably not be transported to the root and should be rapidly degraded in soil, so that application of these chemicals will not inhibit root growth. Cytokinins also delay leaf senescence, which means that positive effects will include both growth and maintenance of photosynthet/c tissues. In addition, the observation that cytokinins delay senescence, enhance greening (chlorophyll content) of leaves and reduce shoot apical dominance shows that strategies based on suppressing CKX activity (such as antisense, ribozyme, and cosuppression technology) in the aerial parts of the plant could result in delayed senescence, enhanced leaf greening and increased branching. Similarly, the observed positive effects of cytokinin oxidase expression on root growth demonstrate that cytokinin oxidases are interesting targets for the design of or screening for herbicides. Such herbicides should inhibit cytokinin oxidase activity, should preferably not be transported to the shoot, and should be soluble and relatively stable in a solvent that can be administered to the root through the soil.<br>
These effects of cytokinin oxidase overexpression on plant development and architecture were hitherto unknown and, as a consequence, the presented invention and its embodiments could not be envisaged.<br>
The observed negative effects on shoot growth demonstrate that manipulation of cytokinin oxidases can also be used for obtaining dwarfing phenotypes. Dwarfing phenotypes are particularly useful in commercial crops such as cereals and fruit trees for example.<br>
Preferable embodiments of the invention relate to the positive effect of cytokinin oxidase expression on plant growth and architecture, and in particular on root growth and architecture.  The cytokinin oxidase gene family contains at least six members in<br>
Arabidopsis (see examples below) and the present inventors have shown that there are quantitative differences in the effects achieved with some of these genes in transgenic plants. It is anticipated that functional homologs of the described Arabidopsis cytokinin oxidases can be isolated from other organisms, given the evidence for the presence of cytokinin oxidase activity in many green plants (Hare and van Staden, Physiol Plant 91:128-136, 1994; Jones and Schreiber, Plant Growth Reg 23:123-134, 1997), as well as in other organisms (Armstrong, in Cytokinins; Chemistry, Activity and Function. Eds Mok and Mok, CRC Press, pp139-154, 1994). Therefore, the sequence of the cytokinin oxidase, functional in the invention, need not to be identical to those described herein. This invention is particularly useful for cereal crops and monocot crops in general and cytokinin oxidase genes from for example wheat or maize may be used as well (Morris et al., 1999; Rinaidi and Comandini, 1999).  It is envisaged that other genes with cytokinin oxidase activity or with any other cytokinin metabolizing activity (see Zazlmalova et al., Biochemistry and Molecular Biology of Plant Hormones, Hooykaas, Hall and Libbenga (Eds.), Elsevier Science, pp141-160, 1997) can also be used for the purpose of this invention. Similarly, genes encoding proteins that would increase endogenous cytokinin metabolizing activity can also be used for the purpose of this invention. In principle, similar phenotypes could also be obtained by interfering with genes that function downstream of cytokinin such as receptors or proteins involved in signal transduction pathways of cytokinin.<br>
For the purpose of this invention, it should be understood that the term 'root growth' encompasses all aspects of growth of the different parts that make up the root system at different stages of its development, both in monocotyledonous and dicotyledonous plants. It is to be understood that enhanced growth of the root can result from enhanced growth of one or more of its parts including the primary root, lateral roots, adventitious roots, etc. all of which fall within the scope of this invention.<br>
According to a first embodiment, the present invention relates to a method for stimulating root growth and/or enhancing the formation of lateral and/or adventitious roots and/or altering root geotropism comprising expression of a plant cytokinin oxidase or comprising expression of another protein that reduces the level of active cytokinins in plants or plant parts.<br>
In the context of the present invention it should be understood that the term "expression" and/or 'overexpression' are used interchangeably and both relate to an "enhanced<br>
and/or ectopic expression" of a plant cytokinin oxidase or any other protein that reduces the level of active cytokinins in plants. It should be clear that herewith an enhanced expression of the plant cytokinin oxidase as well as "de novo1' expression of plant cytokinin oxidases or of said other proteins is meant. Alternatively, said other protein enhances the cytokinin metabolizing activity of a plant cytokinin oxidase. It futher should be understood that in the context of the present invention the expression "lateral and/or adventitious roots" can mean "lateral and adventitious roots" but also "lateral or adventitious roots". The enhancement can exist in the formation of lateral roots or in the formation of adventitious roots as well as in the formation of both types of non-primary roots, but not necessarily.<br>
According to a further embodiment, the present invention relates to a method for stimulating root growth and/or enhancing the formation of lateral or adventitious roots and/or altering root geotropism and/or increasing yield and/or enhancing early vigor and/or modifying root/shoot ratio and/or improving resistance to lodging and/or increasing drought tolerance and/or promoting in vitro propagation of explants, comprising expression of a plant cytokinin oxidase or comprising expression of another protein that reduces the level of active cytokinins in plants or plant parts. According to a preferred embodiment, the present invention relates to a method for-stimulating root growth resulting in an increase of root mass by overexpression of a cytokinin oxidase, preferably a cytokinin oxidase according to the invention, or another protein that reduces the level of active cytokinins in plants or plant parts, preferably in roots.<br>
Higher root biomass production due to overexpression of growth promoting sequences has a direct effect on the yield and an indirect effect of production of compounds produced by root cells or transgenic root cells or cell cultures of said transgenic root cells. One example of an interesting compound produced in root cultures is shikonin, the yield of which can be advantageously enhanced by said methods. According to a more specific embodiment, the present invention relates to a method for stimulating root growth or for enhancing the formation of lateral and adventitious roots or for altering root geotropism comprising expression of a nucleic acid encoding a plant cytokinin oxidase selected from the group consisting of:<br>
(a) nucleic acids comprising a DNA sequence as given in any of SEQ ID NOs 27, 1, 3, 5, 7, 9, 11, 25, 26, 28 to 31, 33 or 34, or the complement thereof,<br>
(b)	nucleic acids comprising the RNA sequences corresponding to any of SEQ ID NOs 27, 1, 3, 5, 7, 9,11, 25, 26, 28 to 31, 33 or 34, or the complement thereof,<br>
(c)	nucleic acids specifically hybridizing to any of SEQ ID NOs 27, 1, 3, 5, 7, 9, 11, 25, 26, 28 to 31, 33 or 34, or to the complement thereof,<br>
(d)	nucleic acids encoding a protein comprising the amino acid sequence as given in any of SEQ ID NOs 2,4, 6, 8,10,12, 32 or 35, or the complement thereof,<br>
(e)	nucleic acids as defined in any of (a) to (d) characterized in that said nucleic acid is DNA, genomic DNA, cDNA, synthetic DNA or RNA wherein T is replaced by U,<br>
(f)	nucleic acids which are degenerated to a nucleic acid as given in any of SEQ ID NOs 27, 1, 3, 5, 7, 9,11, 25, 26, 28 to 31, 33 or 34, or which are degenerated to a nucleic acid as defined in any of (a) to (e) as a result of the genetic code,<br>
(g)	nucleic acids which are diverging from a nucleic acid encoding a protein as given in any of SEQ ID NOs 2, 4, 6, 8,10,12 or 35 or which are diverging from a nucleic acid as defined in any of (a) to (e), due to the differences in codon usage between the organisms,<br>
(h) nucleic acids encoding a protein as given in SEQ ID NOs 2, 4, 6,8, 10, 12 or 35 or nucleic acids as defined in (a) to (e) which are diverging due to the differences between alleles, (i)  nucleic acids encoding a protein as given in any of SEQ ID NOs 2, 4, 6, 8, 10,12<br>
or 35, (j)  functional fragments of nucleic acids as defined in any of (a) to (i) having the<br>
biological activity of a cytokinin oxidase, and (k) nucleic acids encoding a plant cytokinin oxidase, or comprising expression, preferably in roots, of a nucleic acid encoding a protein that reduces the level of active cytokinins in plants or plant parts.<br>
In the present invention, nucleic acids encoding novel Arabidopsis thaliana cytokinine oxidases have been isolated and for the first time, the present inventors suprisingly couid show that the expression of cytokinin oxidases in transgenic plants or in transgenic plant parts resulted in the above-mentioned root-related features. Preferably, the expression of the cytokinine oxidase(s) should take place in roots, preferably under the control of a root-specific promoter. One example of such a root-specific promoter is provided in SEQ ID NO 36.<br>
It should be clear that, although the invention is supported in the examples section by several new AtCKX genes and proteins, the inventive concept also relates to the use of other cytokinin oxidases isolated from and expressed in other plants, preferably in the roots of said other plants to obtain similar effects in plants as desribed in the examples section.<br>
Therefore, the present invention more generally relates to the use of a nucleic acid encoding a plant cytokinin oxidase or encoding a protein that reduces the level of active cytokinins in plants or plant parts for stimulating root growth or for enhancing the formation of lateral or adventitious roots or for altering root geotropism. Preferred cytokinin oxidases to be used are encoded by the nucleic acids encoding the cytokinin oxidases as defined above and are encoded by the novel nucleic acids of the invention as defined hereunder.<br>
The invention relates to an isolated nucleic acid encoding a novel plant protein having cytokinin oxidase activity selected from the group consisting of:<br>
(a)	a nucleic acid comprising a DNA sequence as given in any of SEQ ID NOs 29,<br>
3, 5, 9, 26, 27, 31, 33 or 34, or the complement thereof,<br>
(b)	a nucleic acid comprising the RNA sequences corresponding to any of SEQ ID<br>
. NOs 29, 3, 5, 9, 26, 27, 31, 33 or 34, or the complement thereof,<br>
(c)	a nucleic acid specifically hybridizing to a nucleic acid as given in any of SEQ ID NOs 29, 3, 5, 9, 26, 27, 31, 33 or 34, or the complement thereof,<br>
(d)	a nucleic acid encoding a protein with an amino acid sequence comprising the polypeptide as given in SEQ ID NO 32 and which is at least 70% similar, preferably at least 75%, 80% or 85%, more preferably at ieast 90% or 95%, most preferably at least 99% similar to the amino acid sequence as given in SEQ ID NO 4,<br>
(e)	a nucleic acid encoding a protein with an amino acid sequence which is at least 35% simiiar, preferably 37%, 40%, 45%, 47% or 50%, simitar, more preferably 55%, 60%, 65%, 70%, 75% or 80% similar, most preferably 85%, 90% or 95% simiiar to the amino acid sequence as given in SEQ ID NO 6,<br>
(f)	a nucleic acid encoding a protein with an amino acid sequence which is at least 35% similar, preferably 37%, 40%, 45%, 47% or 50%, similar, more preferably 55%, 60%, 65%, 70%, 75% or 80% similar, most preferably 85%, 90% or 95% similar to the amino acid sequence as given in SEQ ID NO 10 or 35,<br>
(g) a nucleic acid encoding a protein comprising the amino acid sequence as given<br>
in any of SEQ ID NOs 4, 6, 10, 32 or 35, (h) a nucleic acid which is degenerated to a nucleic acid as given in any of SEQ ID<br>
NOs 29, 3, 5, 9, 26, 27, 33 or 34 or which is degenerated to a nucleic acid as<br>
defined in any of (a) to (g) as a result of the genetic code, (i)   a nucleic acid which is diverging from a nucleic acid encoding a protein as given<br>
in any of SEQ ID NOs 4, 6,10 or 35 or which is diverging from a nucleic acid as<br>
defined in any of (a) to (g) due to the differences in codon usage between the<br>
organisms, (j)   a nucleic acid encoding a protein as given in SEQ ID NOs 4, 6, 10 or 35, or a<br>
nucleic acid as defined in (a) to (g)   which is diverging due to the differences<br>
between alleles, (k) a nucleic acid    encoding an immunologically active fragment of a cytokinin<br>
oxidase encoded by a nucleic acid as given in any of SEQ ID NOs 29, 3, 5, 9,<br>
26, 27, 31, 33 or 34, or an immunologically active fragment of a nucleic acid as<br>
defined in any of {a) to (j), (I)   a nucleic acid encoding a functional fragment of a cytokinin oxidase encoded by<br>
a nucleic acid as given in any of SEQ ID NOs 29, 3, 5, 9, 26, 27, 31, 33 or 34, or<br>
a functional fragment of a nucleic acid as defined in any of (a) to (j), wherein said fragment has the biological activity of a cytokinin oxidase, and<br>
(m)a nucleic acid encoding a protein as defined in SEQ ID NO 4, 6, 10 or 35, provided that said nucleic acid is not the nucleic acid as deposited under any of the following Genbank accession numbers: AC005917, AB024035, and AC023754 The invention also relates to an isolated nucleic acid of the invention which is DNA, cDNA, genomic DNA or synthetic DNA, or RNA wherein T is replaced by U. The invention also relates to a nucleic acid molecule of at least 15 nucleotides in length hybridizing specifically with or specifically amplifying a nucleic acid of the invention. According to another embodiment, the invention also relates to a vector comprising a nucleic acid of the invention. In a preferred embodiment, said vector is an expression vector wherein the nucleic acid is operably linked to one or more control sequences allowing the expression of said sequence in prokaryotic ano/or eukaryotic host cells. It should be understood that for expression of the cytokinin oxidase genes of the invention in monocots, a nucleic acid sequence corresponding to the cDNA sequence<br>
should be used to avoid mis-spiicing of introns in monocots. Preferred cDNA sequences<br>
to be expressed in monocots have a nucleic acid sequence as represented in any of<br>
SEQ ID NOs 25 to 30 and 34.<br>
The invention also relates to a host cell containing any of the nucleic acid molecules or<br>
vectors of the invention. Said host cell is chosen from the group comprising bacterial,<br>
insect, fungal, plant or animal cells.<br>
Another embodiment of the invention relates to an isolated polypeptide encodable by a<br>
nucleic acid  of  the  invention,  or a  homologue  or  a derivative  thereof,   or  an<br>
immunologically active or a functional fragment thereof. Preferred polypeptides of the<br>
invention comprise the amino acid sequences as represented in any of SEQ ID NOs 2,<br>
4, 6, 8, 10,12, 32 and 35, or a homologue or a derivative thereof, or an immunologically<br>
active and/or functional fragment thereof. In an even more preferred embodiment, the<br>
invention relates to a polypeptide which has an amino acid sequence as given in SEQ ID<br>
NO 2, 4, 6, 8, 10,12 or 35, or a homologue or a derivative thereof, or an immunologically<br>
active and/or functional fragment thereof. Preferred functional fragments thereof are<br>
those fragments which are devoid of their signal peptide.<br>
According to yet another embodiment, the invention relates to a method for producing a<br>
polypeptide of the invention comprising culturing a host cell of the invention under<br>
conditions allowing the expression of the polypeptide and recovering the produced<br>
polypeptide from the culture.<br>
The invention also relates to an antibody specifically recognizing a polypeptide of the<br>
invention or a specific epitope thereof.<br>
The invention further relates to a method for the production of transgenic plants, plant<br>
cells or plant tissues comprising the introduction of a nucleic acid molecule of the<br>
invention in an expressible format or a vector of the invention in said plant, plant cell or<br>
plant tissue.<br>
The invention also relates to a method for the production of altered plants, p\ant cells or<br>
plant tissues comprising the introduction of a polypeptide of the invention directly into a<br>
cell, a tissue or an organ of said plant.<br>
According to another embodiment, the invention relates to a method for effecting the<br>
expression of a polypeptide of the invention comprising the introduction of a nucleic acid<br>
molecule of the invention operably linked to one or more control sequences or a vector<br>
of the invention stably into the genome of a plant cell. The invention further relates to the method as described above further comprising regenerating a plant from said plant cell. The invention also relates to a transgenic plant cell comprising a nucleic acid sequence of the invention which is operably linked to regulatory elements allowing transcription and/or expression of said nucleic acid in plant cells or obtainable by a method as explained above.<br>
According to another preferred embodiment, the invention relates to a transgenic plant cell as described here above wherein the nucleic acid of the invention is stably integrated into the genome of said plant cell.<br>
The invention further relates to a transgenic plant or plant tissue comprising plant cells as herein described and also to a harvestable part of said transgenic plant, preferably selected from the group consisting of seeds, leaves, fruits, stem cultures, roots, tubers, rhizomes and bulbs. The invention also relates to the progeny derived from any of said transgenic plants or plant parts.<br>
According to another embodiment, the invention relates to a method for stimulating root growth comprising expression of a nucleic acid of the invention or comprising expression of another protein that reduces the level of active cytokinins in plants or plant parts. A plant cell or tissue culture is an artificially produced culture of plants cells or plant tissues that is grown in a special medium, either liquid or solid, which provides these plant cells or tissues with all requirements necessary for growth and/or production of certain compounds. Plant cell and/or tissue cultures can be used for the rapid propagation of plants and for the production of transgenic plant to name a few examples. Root formation can be difficult for some explants or under some conditions in said cultures and expression of a cytokinin oxidase gene in said cultured plant cells or tissue(s) can be used to enhance root formation. Plant cell and/or tissue culture can also be used for the industrial production of valuable compounds. Possible production compounds are pharmaceuticals, pesticides, pigments, cosmetics, perfumes, food additives, etc. An example of such a product is shikonin, which is produced by the roots of the plant Lithospermum erythrorhizon. An example of a plant tissue culture is a hairy root culture, which is an artificially produced mass of hairy roots. Roots of L. erythrorhizon are difficult to collect in large numbers and by preparing hairy root cultures, the end product shikonin could be industrially prepared at a faster rate than would normally occur. As disclosed herein, expression of cytokinin oxidases enhances root<br>
growth and development and can therefore be used advantageously in said plant cell<br>
and tissue culture procedures. Therefore, according to another embodiment of this<br>
invention, a method is provided for stimulating root growth and development comprising<br>
expression of a nucleic acid encoding a plant cytokinin oxidase, preferably a cytokinin<br>
oxidase of the invention, in a transgenic plant cell or tissue culture comprising said<br>
transgenic plant cells.<br>
The invention further relates to a method for enhancing the formation of lateral or<br>
adventitious roots comprising expression of a nucleic acid of the invention or comprising<br>
expression of another protein that reduces the level of active cytokinins in plants or plant<br>
parts.<br>
The invention also relates to method for altering root geotropism comprising altering the<br>
expression of a nucleic acid of the invention or comprising expression of another protein<br>
that that reduces the level of active cytokinins in plants or plant parts.<br>
The invention also relates to methods for enhancing early vigor and/or for modifying<br>
root/shoot ratio and/or for improving resistance to lodging and/or for increasing drought<br>
tolerance and/or for promoting in vitro propagation of explants comprising expression of<br>
a nucleic acid of the invention comprising expression of another protein that reduces the<br>
level of active cytokinins in plants or plant parts.<br>
The invention further relates to methods for increasing the root size or the size of the<br>
root meristem comprising expression of a nucleic acid of the invention or comprising<br>
expression of another protein that reduces the level of active cytokinins in plants or plant<br>
parts, preferably in roots.<br>
According to yet another embodiment, the invention relates to a method for increasing<br>
the size of the shoot meristem comprising downregulation of expression of a nucleic acid<br>
of the invention, preferably in shoots.<br>
According to a preferred embodiment the invention relates to a method for delaying leaf<br>
senescence comprising downregulation of expression of any of the cytokinin oxidases of<br>
the invention in leaves, preferably in senescing leaves. Also the invention relates to a<br>
method for altering leaf senescence comprising expression of one of the cytokinin<br>
oxidases in senescing leaves.<br>
The   invention   also   relates   to   methods  for  increasing   leaf  thickness  comprising<br>
expression of a nucleic acid of the invention or comprising expression of another protein<br>
that reduces the level of active cytokinins in plants or plant parts, preferably in leaves.<br>
The invention  also  relates to a method for reducing the vessel  size  comprising<br>
expression of a nucleic acid of the invention or comprising expression of another protein<br>
that reduces the level of active cytokinins in plants or plant parts, preferably in vessels.<br>
The invention further relates to a method for increasing the vessel size comprising<br>
downregulation of expression of a nucleic acid of the invention in plants or plant parts.<br>
According to another embodiment, the invention relates to a method for improving<br>
standability of seedlings comprising expression of a nucleic acid of the invention or<br>
comprising expression of another protein that reduces the level of active cytokinins in<br>
seedlings.<br>
Furthermore, the invention relates to any of the above described methods, said method<br>
leading to an increase in yield.<br>
The invention further relates to any of the methods of the invention wherein said<br>
expression of said nucleic acid occurs under the control of a strong constitutive<br>
promoter. In a preferred embodiment the invention relates to any of the methods of the<br>
invention wherein said expression of said nucleic acid occurs under the control of a<br>
promoter that is preferentially expressed in roots. In Table 5 a non-exhaustive list of root<br>
specific promoters is included. A preferred promoter to be used in the methods of the<br>
invention is the root clavata homolog promoter, having a sequence as given in SEQ ID<br>
NO 36.<br>
According to yet another embodiment, the invention relates to a method for modifying<br>
cell fate and/or modifying plant development and/or modifying plant morphology and/or<br>
modifying plant biochemistry and/or modifying plant physiology and/or modifying the cell<br>
cycle progression rate comprising the modification of expression in particular cells,<br>
tissues or organs of a plant, of a nucleic acid of the invention.<br>
The invention also relates to a method for obtaining enhanced growth, and/or increased<br>
yield and/or altered senescence of a plant cell, tissue and/or organ and/or increased<br>
frequence of formation of lateral organs in a plant, comprising the ectopic expression of<br>
a nucleic acid of the invention.<br>
The invention also relates to a method for promoting and extending cell division activity<br>
in cells in adverse growth conditions and/or in stress, comprising the ectopic expression<br>
of a nucleic acid sequence of the invention.<br>
According to yet another embodiment, the invention relates to a method for identifying<br>
and obtaining proteins interacting with a polypeptide of the invention comprising a<br>
screening assay wherein a polypeptide of the invention is used.<br>
In a more preferred embodiment, the invention relates to a method for identifying and<br>
obtaining proteins interacting with a polypeptide of the invention comprising a two-hybrid<br>
screening assay wherein a polypeptide of the invention as a bait and a cDNA library as<br>
pray are used.<br>
The invention further relates to a method for modulating the interaction between a<br>
polypeptide of the invention and interacting protein partners obtainable by a method as<br>
described above.<br>
In a further embodiment, the invention relates to a method for identifying and obtaining<br>
compounds interacting with a polypeptide of the invention comprising the steps of:<br>
a)	providing a two-hybrid system wherein a polypeptide of the invention and an interacting protein partner obtainable by a method as described above,<br>
b)	interacting said compound with the complex formed by the expressed polypeptides as defined in a), and,<br>
c)	performing (real-time) measurement of interaction of said compound with said polypeptide or the complex formed by the expressed polypeptides as defined in a).<br>
The invention further relates to a method for identifying compounds or mixtures of compounds which specifically bind to a polypeptide of the invention, comprising:<br>
a)	combining a polypeptide of the invention with said compound or mixtures of compounds under conditions suitable to allow complex formation, and,<br>
b)	detecting complex formation, wherein the presence of a complex identifies a compound or mixture which specifically binds said polypeptide.<br>
The invention also relates to a method as described above wherein said compound or<br>
mixture inhibits the activity of said polypeptide of the invention and can be used for the<br>
rational design of chemicals.<br>
According to another embodiment, the invention relates to the use of a compound or<br>
mixture identified by means of a method as described above as a plant growth regulator<br>
or herbicide.<br>
The invention also relates to a method tor production of a plant growth regulator or<br>
herbicide  composition  comprising  the steps  of the compound  screening  methods<br>
described above and formulating the compounds obtained from said steps in a suitable form for the application in agriculture or plant cell or tissue culture. The invention also relates to a method for increasing branching comprising expression of a nucleic acid of the invention in plants or plant parts, preferably in stems or axillary buds.<br>
The invention also relates to a method for improving lodging resistance comprising expression of a nucleic acid of the invention in piants or plant parts, preferably in stems or axillary buds.<br>
The invention also relates to a method for the design of or screening for growth-promoting chemicals or herbicides comprising the use of a nucleic acid of the invention or a vector of the invention.<br>
According to another embodiment, the invention relates to the use of a nucleic acid molecule of the invention, a vector of of the invention or a polypeptide of the invention for increasing yield.<br>
The invention also relates to the use of a nucleic acid molecule of of the invention, a vector of the invention or a polypeptide of the invention for stimulating root growth. The invention also relates to the use of a nucleic acid molecule of the invention, a vector of the invention or a polypeptide of the invention for enhancing the formation of lateral or adventitious roots.<br>
The invention also relates to the use of a nucleic acid molecule of the invention, a vector of of the invention or a polypeptide of the invention for altering root geotropism. The invention further relates to the use of a nucleic acid molecule of of the invention, a vector of the invention or a polypeptide of the invention for enhancing early vigor and/or for modifying root/shoot ratio and/or for improving resistance to lodging and/or for increasing drought tolerance and/or for promoting in vitro propagation of explants. The invention also relates to the use of a nucleic acid molecule of the invention, a recombinant vector of the invention or a polypeptide of the invention for modifying plant development and/or for modifying plant morphology and/or for modifying plant biochemistry and/or for modifying plant physiology.<br>
According to yet another embodiment, the invention relates to a diagnostic composition comprising at least a nucleic acid molecule of the invention, a vector of the invention, a polypeptide of the invention or an antibody of the invention.<br><br>
Another embodiment of the current invention relates to the use of a transgenic rootstock that has an enhanced root growth and development due to expression of a cytokinin oxidase in grafting procedures with a scion to produce a plant or tree with improved agricultural  or  horticultural  characteristics.   The scion   may  be  transgenic  or  non-transgenic. Specific characteristics envisaged by this embodiment are those conferred by root systems and include improved anchoring of the plant/tree in the soil and/or improved uptake of water resulting for example in improved drought tolerance, and/or improved nutrient uptake from the soil and/or improved transport of organic substances throughout the plant and/or enhanced secretion of substances into the soil such as for example  phytosiderophores,  and/or  improved  respiration  and/or  improved  disease resistance and/or enhanced yield. An advantage of using AtCKX transformed rootstocks for grafting, in addition to their enhanced root system, is the delayed senescence of leaves on the graft, as disclosed herein (see Figure 12 A). Preferred plants or trees for this particular embodiment include plants or trees that do not grow well on their own roots and are grafted in cultivated settings such as commercially profitable varieties of grapevines, citrus, apricot, almond, plum, peach, apple, pear, cherry, walnut, fig, hazel and loquat.<br>
As; mentioned supra, auxins and cytokinins act as antagonists in certain biological processes. For example, the cytokinin/auxin ratio regulates the production of roots and shoots with a high concentration of auxin resulting in organized roots and a high concentration of cytokinins resulting in shoot production. As disclosed in this invention, expression of cytokinin oxidases in tobacco and Arabidopsis results in enhanced root development consistent with enhanced auxin effects. Auxins are also involved in the development of fruit. Treatment of female flower parts with auxin results in the development of parthenocarpic fruit in some plant species. Parthenocarpic fruit development has been genetically engineered in several horticultural crop plants through increased biosynthesis of auxins in the female reproductive organs (WO0105985). Therefore, according to another embodiment, this invention relates to a method for inducing the parthenocarpic trait in plants, said method consisting of downregulating the expression of one or more cytokinin oxidases or of another protein that reduces the level of active cytokinins in plants or plant parts, preferably in the female reproductive organs such as the placenta, ovules and tissues derived therefrom. The DefH9 promoter region<br>
from Antirrhinum majus or one of its homologues, which confer high expression specificity in placenta and ovules, can be used for this purpose.<br>
DEFINITIONS AND ELABORATIONS TO THE EMBODIMENTS<br>
Those skilled in the art will be aware that the invention described herein is subject to variations and modifications other than those specifically described. It is to be understood that the invention described herein includes all such variations and modifications. The invention also includes all such steps, features, compositions and compounds referred to or indicated in this specification, individually or collectively, and any and all combinations of any or more of said steps or features.<br>
The present invention is applicable to any plant, in particular a monocotyledonous plants and dicotyledonous plants including a fodder OT forage legume, ornamental plant, food crop, tree, or shrub selected from the list comprising Acacia spp., Acer spp., Actinidia spp.,Aesculus spp., Agathis australis, Albizia amara, Alsophila tricolor, Andropogon spp., Arachts spp, Areca catechu, Astelia fragrans, Astragalus cicer, Baikiaea plurijuga, Betula spp., Brassica spp., Bruguiera gymnorrhiza, Burkea africana, Butea frondosa, Cadaba farinosa, Calliandra spp', Camellia sinensis, Canna indica, Capsicum spp., Cassia spp., Centroema  pubescens,   Chaenomeles  spp.,Cinnamomum   cassia,   Coffea   arabica, Colophospermum mopane,  Coronillia  varia,  Cotoneaster serotina,   Crataegus spp., Cucumis spp.,   Cupressus spp.,  Cyathea  dealbata,   Cydonia  oblonga,   Cryptomeria japonica, Cymbopogon spp., Cynthea dealbata, Cydonia oblonga, Dalbergia monetaria, Davallia divaricata, Desmodium spp., Dicksonia squarosa, Diheteropogon amplectens, Dioclea spp, Dolichos spp., Dorycnium rectum, Echinochloa pyramidalis, Ehrartia spp., Eleusine coracana, Eragrestis spp., Erythrina spp., Eucalyptus spp., Euclea schimperi, Eulalia  villosa,   Fagopymm spp.,  Feijoa sellowiana,   Fragaria  spp.,  Flemingia spp, Freycinetia banksii, Geranium thunbergii, Ginkgo biloba, Glycine javanica, Gliricidia spp, Gossypium   hirsutum,   Grevillea   spp.,   Guibourtia   coleosperma,   Hedysarum   spp., Hemarthia   aliissima,   Heteropogon  contortus,   Hordeum   vulgare,   Hyparrhenia   tufa, Hypericum  erectum,   Hyperthelia dissoluta,   Indigo incarnata,   Iris spp.,   Leptarrhena pyrolifolia,  Lespediza spp., Lettuca spp., Leucaena leucocephala,  Loudetia simplex, Lotonus bainesii,  Lotus spp., Macrotyloma axi/lare,  Malus spp.,  Manihot esculenta, Medicago sativa,  Metasequoia glyptostmboides,  Musa sapientum,  Nicotianum spp.,<br>
Onobrychis spp., Omithopus spp., Oryza spp., Peltophorum africanum, Pennisetum spp., Persea gratissima, Petunia spp., Phaseolus spp., Phoenix canadensis, Phormium cookianum, Photinia spp., Picea glauca, Pinus spp., Pisum sativum, Podocarpus totara, Pogonarthria fleckii, Pogonarthria squamosa, Populus spp., Prosopis cineraria, Pseudotsuga menziesii, Pterolobium stelfatum, Pyrus communis, Quercus spp., Rhaphiolepsis umbellata, Rhopaiostylis sapida, Rhus natalensis, Ribes grossularia, Ribes spp., Robinia pseudoacacia, Rosa spp., Rubus spp., Salix spp., Schyzachyrium sanguineum, Sciadopitys verticillata, Sequoia sempervirens, Sequoiadendron giganteum, Sorghum bicolor, Spinacia spp., Sporobolus fimbriatus, Stiburus alopecuroides, Stylosanthos humilis, Tadehagi spp, Taxodium distichum, Themeda triandra, Trifolium spp., Triticum spp., Tsuga heterophylla, Vaccinium spp., Vicia spp. Vitis vinifera, Watsonia pyramidata, Zantedeschia aethiopica, Zea mays, amaranth, artichoke, asparagus, broccoli, brussel sprout, cabbage, canola, carrot, cauliflower, celery, collard greens, flax, kale, lentil, oilseed rape, okra, onion, potato, rice, soybean, straw, sugarbeet, sugar cane, sunflower, tomato, squash, and tea, amongst others, or the seeds of any plant specifically named above or a tissue, cell or organ culture of any of the above species.<br>
Throughout this specification, unless the context requires otherwise the word "comprise", and variations such as "comprises" and "comprising", will be understood to imply the inclusion of a stated integer or step or group of integers or steps but not the exclusion of any other integer or step or group of integers or steps.<br>
As used herein, the term "derived from" shall be taken to indicate that a particular integer or group of integers has originated from the species specified, but has not necessarily been obtained directly from the specified source.<br>
The terms "protein(s)", "peptide(s)" or "oligopeptide(s)", when used herein refer to amino acids in a polymeric form of any length. Said terms also include known amino acid modifications such as disulphide bond formation, cysteinylation, oxidation, glutathionylation, methylation, acetylation, farnesylation, biotinylation, stearoylation, formylation, lipoic acid addition, phosphorylation, sulphation, ubiquitination, myristoylation, palmitoylation, geranylgeranylation, cyclization (e.g. pyrogiutamic acid formation), oxidation, deamidation, dehydration, glycosylation (e.g. pentoses, hexcsamines,   N-acetylhexosamines,  deoxyhexoses,  hexoses,   sialic  acid   etc.)  and<br>
acylation as well as non-naturally occurring amino acid residues, L-amino acid residues and D-arnino acid residues.<br>
"Homologues" of a protein of the invention are those peptides, oligopeptides, polypeptides, proteins and enzymes which contain amino acid substitutions, deletions and/or additions relative to the said protein with respect to which they are a homoiogue, without altering one or more of its functional properties, in particular without reducing the activity of the resulting. For example, a homoiogue of said protein will consist of a bioactive amino acid sequence variant of said protein. To produce such homologues, amino acids present in the said protein can be replaced by other amino acids having similar properties, for example hydrophobicity, hydrophilicity, hydrophobic moment, antigenicity, propensity to form or break a-he!ical structures or (3-sheet structures, and so on. An overview of physical and chemical properties of amino acids is given in Table 1. Substitutional variants of a protein of the invention are those in which at least one residue in said protein amino acid sequence has been removed and a different residue inserted in its place. Amino acid substitutions are typically of single residues, but may be clustered depending upon functional constraints placed upon the polypeptide; insertions will usually be of the order of about 1-10 amino acid residues, and deletions wiil range from about 1-20 residues. Preferably, amino acid substitutions will comprise conservative amino acid substitutions, such as those described supra.<br>
Table 1. Properties of naturally occurring amino acids. (Table Removed)<br>
Insertional amino acid sequence variants of a protein of the invention are those in which one or more amino acid residues are introduced into a predetermined site in said protein. Insertions can comprise amino-terminal and/or carboxy-terminal fusions as well as intra-sequence insertions of single or multiple amino acids. Generally, insertions within the amino acid sequence will be smaller than amino or carboxyl terminal fusions, of the order of about 1 to 10 residues. Examples of amino- or carboxy-terminal fusion proteins or peptides include the binding domain or activation domain of a transcriptional activator as used in a two-hybrid system, phage coat proteins, (histidine)6-tag, glutathione S-transferase,   protein  A,   maltose-binding  protein,  dihydrofolate  reductase,  Tag«100 epitope     (EETARFQPGYRS),     c-myc     epitope     (EQKLISEEDL),     FLAG®-epitope (DYKDDDK),  lacZ,  CMP  (calmodulin-binding  peptide),  HA epitope  (YPYDVPDYA), protein C epitope (EDQVDPRLIDGK) and VSV epitope (YTDIEMNRLGK). Deletional variants of a protein of the invention are characterised by the removal of one or more amino acids from the amino acid sequence oi said protein. Amino acid variants of a protein of the invention may readily be made using peptide synthetic techniques well known in the art, such as solid phase peptide synthesis and the like, or by recombinant DNA manipulations. The manipulation of DNA sequences to produce variant proteins which manifest as substitutional,  insertional or deletional variants are well known in the art. For example, techniques for making substitution mutations at predetermined sites in DNA having known sequence are well known to those skilled in the art, such as by M13 mutagenesis, T7-Gen in vitro mutagenesis kit (USB, Cleveland, OH), QuickChange Site Directed mutagenesis kit (Stratagene, San Diego, CA), PCR-mediated site-directed mutagenesis or other site-directed mutagenesis protocols.<br>
In   the   current   invention   "identity"   and/or  "similarity"   percentages   between   DNA sequences and/or proteins are calculated using computer programs known in the art such as the DNAstar/MegAlign programs in combination with the Clustal method. "Derivatives"   of   a   protein   of   the   invention   are   those   peptides,   oligopeptides, polypeptides, proteins and enzymes which comprise at least about five contiguous<br>
amino acid residues of said polypeptide but which retain the biological activity of said protein. A "derivative" may further comprise additional naturally-occurring, altered glycosylated, acylated or non-naturally occurring amino acid residues compared to the amino acid sequence of a naturally-occurring form of said polypeptide. Alternatively or in addition, a derivative may comprise one or more non-amino acid substituents compared to the amino acid sequence of a naturally-occurring form of said polypeptide, for example a reporter molecule or other ligand, covalently or non-covalently bound to the amino acid sequence such as, for example, a reporter molecule which is bound thereto to facilitate its detection.<br>
With "immunologically active" is meant that a molecule or specific fragments thereof such as specific epitopes or haptens are recognized by, i.e. bind to antibodies. Specific epitopes may be determined using, for example, peptide scanning techniques as described in Geysen et al. (1996) (Geysen, H.M., Rodda, S.J. and Mason, T.J. (1986). A priori delineation of a peptide which mimics a discontinuous antigenic determinant. Mol. Immunol. 23, 709-715.).<br>
The term "fragment of a sequence" or "part of a sequence" means a truncated sequence of the original sequence referred to. The truncated sequence (nucleic acid or protein sequence) can vary widely in length; the minimum size being a sequence of sufficient size to provide a sequence with at least a comparable function and/or activity or the original sequence referred to (e. g. "functional fragment"), while the maximum size is not critical. In some applications, the maximum size usually is not substantially greater than that required to provide the desired activity and/or function(s) of the original sequence. Typically, the truncated amino acid sequence will range from about 5 to about 60 amino acids in length. More typically, however, the sequence will be a maximum of about 50 amino acids in lenght, preferably a maximum of about 60 amino acids. It is usually desirable to select sequences of at least about 10, 12 or 15 amino acids, up to a maximum of about 20 or 25 amino acids.<br>
Functional fragments can also include those comprising an epitope which is specific for the proteins according to the invention. Preferred functional fragments have a length of at least, for example, 5,10, 25,100,150 or 200 amino acids.<br>
It should thus be understood that functional fragments can also be immunologically active fragments or not.<br>
In the context of the current invention are embodied homologues, derivatives and/or (mmunologicaHy active and/or functional fragments of the cytokinin oxidases as defined supra. Particularly preferred homologues, derivatives and/or immunologically active and/or functional fragments of the cytokinin oxidase proteins which are contemplated for use in the current invention are derived from plants, more specifically from Arabidopsis thadana, even more specifically said cytokinin oxidases are the Arabidopsis thaliana (At)CKX, or are capable of being expressed therein. The present invention clearly contemplates the use of functional homologues or derivatives and/or immunologically active fragments of the AtCKX proteins and is not to be limited in application to the use of a nucleotide sequence encoding one of said AtCKX proteins.<br>
Any of said proteins, polypeptides, peptides and fragments thereof can be produced in a biological system, e.g. a cell culture. Alternatively any of said proteins, polypeptides, peptides and fragments thereof can be chemically manufactured e.g. by solid phase peptide synthesis. Said proteins or fragments thereof can be part of a fusion protein as is the case in e.g. a two-hybrid assay which enables e.g. the identification of proteins interacting with a cytokinin oxidase according to the invention.<br>
The proteins or fragments thereof are furthermore useful e.g. to modulate the interaction between a cytokinin oxidase according to the invention and interacting protein partners obtained by a method of the invention. Chemically synthesi2ed peptides are particularly useful e.g. as a source of antigens for the production of antisera and/or antibodies. "Antibodies" include monoclonal, polyclonal, synthetic or heavy chain camel antibodies as well as fragments of antibodies such as Fab, Fv or scFv fragments. Monoclonal antibodies can be prepared by the techniques as described in e.g. Liddle and Cryer (1991) which comprise the fusion of mouse myeloma cells to spleen cells derived from immunized animals. Furthermore, antibodies or fragments thereof to a molecule or fragments thereof can be obtained by using methods as described in e.g. Harlow and Lane (1988). In the case of antibodies directed against small peptides such as fragments of a protein of the invention, said peptides are generally coupled to a carrier protein before immunization of animals. Such protein carriers include keyhole limpet hemocyanin (KLH), bovine serum albumin (BSA), ovalbumin and Tetanus toxoid. The carrier protein enhances the immune response of the animal and provides epitopes for T-cell receptor binding sites. The term "antibodies" furthermore includes derivatives thereof such  as labelled antibodies. Antibody labels include alkaline phosphatase,<br>
PKH2, PKH26, PKH67, fluorescein (FITC), Hoechst 33258, R-phycoerythrin (PE), rhodamine (TRITC), Quantum Red, Texas Red, Cy3, biotin, agarose, peroxidase and gold spheres. Tools in molecular biology relying on antibodies against a protein include protein gel blot analysis, screening of expression libraries allowing gene identification, protein quantitative methods including ELISA and RIA, immunoaffinity purification of proteins, immunoprecipitation of proteins (see e.g. Example 6) and immunolocalization. Other uses of antibodies and especially of peptide antibodies include the study of proteolytic processing (Loffler et al. 1994, Woulfe et al. 1994), determination of protein active sites (Lerner 1982), the study of precursor and post-translaticnal processing (Baron and Baltimore 1982, Lerner et al. 1981, Semier et al. 1982), identification of protein domains involved in protein-protein interactions (Murakami et al. 1992) and the study of exon usage in gene expression (Tamura et ai. 1991).<br>
Embodied in the current invention are antibodies specifically recognizing a cytokinin oxidase or homologue, derivative or fragment thereof as defined supra. Preferably said cytokinin oxidase is a plant cytokinin oxidase, more specifically one of the Arabidopsis thaliana cytokinin oxidases (AtCKX).<br>
The terms "gene(s)", "polynucleotide(s)", "nucleic acid(s)", "nucleic acid sequence(s)", "nucleotide sequence(s)", or "nucleic acid molecule(s)", when used herein refer to nucleotides, either ribonucleotides or deoxyribonucleotides or a combination of both, in a polymeric form of any length. Said terms furthermore include double-stranded and single-stranded DNA and RNA. Said terms also include known nucleotide modifications such as methylation, cyclization and 'caps' and substitution of one or more of the naturally occurring nucleotides with an analog such as inosine. Modifications of nucleotides include the addition of acridine, amine, biotin, cascade blue, cholesterol, Cy3®, Cy5®, Cy5.5® Dabcyl, digoxigenin, dinitrophenyl, Edans, 6-FAM, fluorescein, 3'-glyceryl, HEX, IRD-700, IRD-800, JOE, phosphate psoralen, rhodamine, ROX, thiol (SH), spacers, TAMRA, TET, AMCA-S®, SE, BODIPY®, Marina Blue®, Pacific Blue®, Oregon Green®, Rhodamine Green®, Rhodamine Red®, Rhodol Green® and Texas Red®. Polynucleotide backbone modifications include methylphosphonate, 2'-OMe-methylphosphonate RNA, phosphorothiorate, RNA, 2'-OMeRNA. Base modifications include 2-amino-dA, 2-aminopurine, 3'-(ddA), 3'dA(cordycepin), 7-deaza-dA, 8-Br-dA, 8-oxo-dA, N6-Me-dA, abasic site (dSpacer), biotin dT, 2'-OMe-5Me-C, 2'-OMe-propynyl-C, 3'-(5-Me-dC), 3'-(ddC), 5-Br-dC, 5-l-dC, 5-Me-dC, 5-F-dC, carboxy-dT, convertible dA,<br>
convertible dC, convertible dG, convertible dT, convertible dU, 7-deaza-dG, 8-Br-dG, 8-oxo-dG, Oe-Me-dG, S6-DNP-dG, 4-methyl-indole, 5-nitroindole, 2'-OMe-inosine, 2'-dl, 0B-phenyl-dl, 4-methyl-indole, 2'-deoxynebularine, 5-nitroindole, 2-aminopurine, dP(purine analogue),   dK(pyrimidine   analogue),   3-nitropyrrole,   2-thio-dT,   4-thio-dT,   biotin-dT, carboxy-dT, 04-Me-dT, 04-triazoi dT, 2'-OMe-propynyl-U, 5-Br-dU, 2,-dU, 5-F-dU, 5-l-dU, 04-triazol  dU.  Said terms  also  encompass peptide  nucleic acids  (PNAs),  a  DNA analogue in which the backbone is a pseudopeptide consisting of N-(2-aminoethyl)-glycine units rather than  a sugar.  PNAs mimic the behaviour of DNA  and  bind compiementary nucleic acid strands. The neutral backbone of PNA results in stronger binding and greater specificity than normally achieved. In addition, the unique chemical, physical and biological properties of PNA have been exploited to produce powerful biomolecular tools, antisense and antigene agents, molecular probes and biosensors. The present invention also advantageously provides nucleic acid sequences of at least approximately 15 contiguous nucleotides of a nucleic acid according to the invention and preferably from 15 to 50 nucleotides. These sequences may, advantageously be used as probes to specifically hybridise to sequences of the invention as defined above or primers to initiate specific amplification or replication of sequences of the invention as defined above, or the like. Such nucleic acid sequences may be produced according to techniques well known in the art, such as by recombinant or synthetic means. They may also be used in diagnostic kits or the like for detecting the presence of a nucleic acid according to the invention. These tests generally comprise contacting the probe with the sample under hybridising conditions and detecting the presence of any duplex or triplex formation between the probe and any nucleic acid in the sample.<br>
Advantageously, the nucleic acid sequences, according to the invention may be produced using such recombinant or synthetic means, such as for example using PCR cloning mechanisms which generally involve making a pair of primers, which may be from approximately 15 to 50 nucleotides to a region of the gene which is desired to be cloned, bringing the primers into contact with mRNA, cDNA or genomic DNA from a cell, performing a polymerase chain reaction under conditions which bring about amplification of the desired region, isolating the amplified region or fragment and recovering the amplified DNA. Generally, such techniques as defined herein are well known in the art, such as described in Sambrook et al. (Molecular Cloning: a Laboratory Manual, 1989).<br>
A "coding sequence" or "open reading frame" or "ORF" is defined as a nucleotide sequence that can be transcribed into mRNA and/or translated into a polypeptide when placed under the control of appropriate control sequences or regulatory sequences, i.e. when said coding sequence or ORF is present in an expressible format. Said coding sequence of ORF is bounded by a 5' translation start codon and a 3" translation stop codon. A coding sequence or ORF can include, but is not limited to RNA, mRNA, cDNA, recombinant nucleotide sequences, synthetically manufactured nucleotide sequences or genomic DNA. Said coding sequence or ORF can be interrupted by intervening nucleic acid sequences.<br>
Genes and coding sequences essentially encoding the same protein but isolated from different sources can consist of substantially divergent nucleic acid sequences. Reciprocally, substantially divergent nucleic acid sequences can be designed to effect expression of essentially the same protein. Said nucleic acid sequences are the result of e.g. the existence of different alleies of a given gene, of the degeneracy of the genetic code or of differences in codon usage. Thus, as indicated in Table 2, amino acids such as methionine and tryptophan are encoded by a single codon whereas other amino acids such as arginine, leucine and serine can each be translated from up to six different codons. Differences in preferred codon usage are illustrated in Table 3 for Agrobacterium tumefaciens (a bacterium), A. thaliana, M. sativa (two dicotyledonous plants) and Oryza sativa (a monocotyledonous plant). To extract one example, the codon GGC (for glycine) is the most frequently used codon in A. tumefaciens (36.2 %0), is the second most frequently used codon in O. sativa but is used at much lower frequencies in A. thaliana and M. sativa (9 %o and 8.4 %o, respectively). Of the four possible codons encoding glycine (see Table 2), said GGC codon is most preferably used in A. tumefaciens and O. sativa. However, in A. thaliana this is the GGA (and GGU) codon whereas in M. sativa this is the GGU (and GGA) codon.<br>
DNA sequences as defined in the current invention can be interrupted by intervening sequences. With "intervening sequences" is meant any nucleic acid sequence which disrupts a coding sequence comprising said inventive DNA sequence or which disrupts the expressible format of a DNA sequence comprising said inventive DNA sequence. Removal of the intervening sequence restores said coding sequence or said expressible format.   Examples  of  intervening  sequences  include  introns  and  mobilizable  DNA<br>
sequences such as transposons. With "mobilizable DNA sequence" is meant any DNA sequence that can be mobilized as the result of a recombination event.<br>
Table 2. Degeneracy of the genetic code.  (Table Removed)<br>
Table 3. Usage of the indicated codons in the different organisms given as frequency per thousand codons (http://www.kazusa.or.ip/codon).<br>
(Table Removed)<br>
"Hybridization"   is   the   process   wherein   substantially   homologous   complementary nucleotide sequences anneal to each other. The hybridization process can occur entirely in solution, i.e. both complementary nucleic acids are in solution. Tools in molecular biology relying on such a process include PCR, subtractive hybridization and DNA sequence determination. The hybridization process can also occur with one of the complementary  nucleic  acids  immobilized  to  a  matrix such  as  magnetic   beads, Sepharose beads or any other resin. Tools in molecular biology relying on such a process include the isolation of poly (A+) mRNA. The hybridization process can furthermore occur with one of the complementary nucleic acids immobilized to a solid support  such   as   a   nitrocellulose   or   nylon   membrane   or   immobilized   by   e.g. photolitography to e.g. a silicious glass support (the latter known as nucleic acid arrays or microarrays or as nucleic acid chips). Tools in molecular biology relying on such a process  include   RNA   and   DNA   gel   blot  analysis,   colony  hybridization,   plaque hybridization and microarray hybridization. In order to allow hybridization to occur, the nucleic acid molecules are generally thermally or chemically (e.g. by NaOH) denatured to melt a double strand into two single strands and/or to remove hairpins or other secondary structures from single stranded nucleic acids. The stringency of hybridization is intluenced by conditions such as temperature, salt concentration and hybridization buffer composition. High stringency conditions for hybridization include high temperature and/or low salt concentration (salts include NaCI and Na3-citrate) and/or the inclusion of formamide in the hybridization buffer and/or lowering the concentration of compounds such as SDS (detergent) in the hybridization buffer and/or exclusion of compounds such as dextran sulfate or polyethylene glycol (promoting molecular crowding) from the<br>
hybridization buffer. Conventional hybridization conditions are described in e.g. Sambrook et al. (1989) but the skilled craftsman will appreciate that numerous different hybridization conditions can be designed in function of the known or the expected homology and/or length of the nucleic acid sequence. Sufficiently low stringency hybridization conditions are particularly preferred to isolate nucleic acids heterologous to the DNA sequences of the invention defined supra. Elements contributing to said heterology include allelism, degeneration of the genetic code and differences in preferred codon usage as discussed supra.<br>
Clearly, the current invention embodies the use of the inventive DNA sequences encoding a cytokinin oxidase, homologue, derivative or immunologically active and/or functional fragment thereof as defined higher in any method of hybridization. The current invention furthermore also relates to DNA sequences hybridizing to said inventive DNA sequences. Preferably said cytokinin oxidase is a plant cytokinin oxidase, more specifically the Arabidopsis thaliana (At)CKX.<br>
To effect expression of a protein in a ceil, tissue or organ, preferably of plant origin, either the protein may be introduced directly to said cefi, such as by microinjection or ballistic means or alternatively, an isolated nucleic acid molecule encoding said protein may be introduced into said cell, tissue or organ in an expressible format. Preferably, the DNA sequence of the invention comprises a coding sequence or open reading frame (ORF) encoding a cytokinin oxidase protein or a homologue or derivative thereof or an immunologically active and/or functional fragment thereof as defined supra. The preferred protein of the invention comprises the amino acid sequence of said cytokinin oxidase. Preferably said cytokinin oxidase is a plant cytokinin oxidase and more specifically a Arabidopsis thaliana (At)CKX.<br>
With "vector" or "vector sequence" is meant a DNA sequence which can be introduced in an organism by transformation and can be stably maintained in said organism. Vector maintenance is possible in e.g. cultures of Escherichia coli, A. tumefaciens, Saccharomyces cerevisiae or Schizosaccharomyces pombe. Other vectors such as phagemids and cosmid vectors can be maintained and multiplied in bacteria and/or viruses. Vector sequences generally comprise a set of unique sites recognized by restriction enzymes, the multiple cloning site (MCS), wherein one or more non-vector sequence(s) can be inserted.<br>
With "non-vector sequence" is accordingly meant a DNA sequence which is integrated in one or more of the sites of the MCS comprised within a vector.<br>
"Expression vectors" form a subset of vectors which, by virtue of comprising the appropriate regulatory or control sequences enable the creation of an expressible format for the inserted non-vector sequence(s), thus allowing expression of the protein encoded by said non-vector sequence(s). Expression vectors are known in the art enabling protein expression in organisms including bacteria (e.g. E. coli), fungi (e.g. S. cerevisiae, S. pombe, Pichia pastoris), insect cells (e.g. baculoviral expression vectors), animal cells (e.g. COS or CHO cel\s) and plant cells (e.g. potato virus X-based expression vectors). The current invention clearly includes any cytokinin oxidase, homologue, derivative and/or immunologically active and/or functional fragment thereof as defined supra. Preferably said cytokinin oxidase is a plant cytokinin oxidase, more specifically a Arabidopsis thaliana (At)CKX.<br>
As an alternative to expression vector-mediated protein production in biological systems, chemical protein synthesis can be applied. Synthetic peptides can be manufactured in solution phase or in solid phase. Solid phase peptide synthesis (Merrifield 1963) is, however, the most common way and involves the sequential addition of amino acids to create a linear peptide chain. Solid phase peptide synthesis includes cycles consisting of three steps: (i) immobilization of the carboxy-terminal amino acid of the growing peptide chain to a solid support or resin; (ii) chain assembly, a process consisting of activation, coupling and deprotection of the amino acid to be added to the growing peptide chain; and (iii) cleavage involving removal of the completed peptide chain from the resin and removal of the protecting groups from the amino acid side chains. Common approaches in solid phase peptide synthesis include Fmoc/tBu (9-fluorenylmethyloxycarbonyl/t-butyl) and Boc (t-butyloxycarbonyl) as the amino-terminal protecting groups of amino acids. Amino acid side chain protecting groups include methyl (Me), formyl (CHO), ethyl (Et), acetyl (Ac), t-butyl (t-Bu), anisyl, benzyl (Bzl), trifluroacetyl (Tfa), N-hydroxysuccinimide (ONSu, OSu), benzoyl (Bz), 4-methylbenzyl (Meb), thioanizyl, thiocresyl, benzyloxymethy) (Bom), 4-nitrophenyl (ONp), benzyloxycarbonyl (Z), 2-nitrobenzoyl (NBz), 2-nitrophenylsulphenyl (Nps), 4-toluenesulphonyl (Tosyl.Tos), pentafluorophenyl (Pfp), diphenylmethyl (Dpm), 2-chlorobenzyloxycarbonyl (Cl-Z), 2,4,5-trichloropheny), 2-bromobenzyloxycarbonyl (Br-Z), tripheylmethyl (Trityl, Trt), and 2,5,7,8-pentamethyl-chroman-6-sulphonyl (Pmc). During chain assembly, Fmoc or Boc are removed resulting<br>
in an activated amino-terminus of the amino acid residue bound to the growing chain. The carboxy-terminus of the incoming amino acid is activated by conversion into a highly reactive ester, e.g. by HBTU. With current technologies (e.g. PerSeptive Biosystems 9050 synthesizer, Applied Biosystems Model 431A Peptide Synthesizer), linear peptides of up to 50 residues can be manufactured. A number of guidelines is available to produce peptides that are suitable for use in biological systems including (i) limiting the use of difficult amino acids such as cys, met, trp (easily oxidized and/or degraded during peptide synthesis) or arg; (ii) minimize hydrophobic amino acids (can impair peptide solubility); and (iii) prevent an amino-terminal glutamic acid (can cyclize to pyroglutamate).<br>
By "expressible format" is meant that the isolated nucleic acid molecule is in a form suitable for being transcribed into mRNA and/or translated to produce a protein, either constitutively or following induction by an intracellular or extracellular signal, such as an environmental stimulus or stress (mitogens, anoxia, hypoxia, temperature, salt, light, dehydration,    etc)    or   a   chemical    compound   such    as    IPTG    (isopropyl-p-D-thiogalactopyranoside)  or such as  an  antibiotic (tetracycline,  ampicillin,   rifampicin, kanamycin), hormone (e.g. gibberellin, auxin, cytokinin, glucocorticoid, brassinosteroid, ethylene, abscisic acid etc), hormone analogue (indolacetic acid (IAA), 2,4-D, etc) , metal (zinc, copper, iron, etc), or dexamethasone, amongst others. As will be known to those skilled in the art, expression of a functional protein may also require one or more post-translational      modifications,      such      as      glycosylation,       phosphorylation, dephosphorylation, or one or more protein-protein interactions, amongst others. All such processes are included within the scope of the term "expressible format". Preferably, expression of a protein in a specific cell, tissue, or organ, preferably of plant origin, is effected by introducing and expressing an isolated nucleic acid molecule encoding   said   protein,   such   as   a   cDNA   molecule,   genomic   gene,   synthetic oligonucleotide molecule, mRNA molecule or open reading frame, to said cell, tissue or organ, wherein said nucleic acid molecule is placed operably in connection with suitable regulatory or control sequences including a promoter, preferably a plant-expressible promoter, and a terminator sequence.<br>
Reference herein to a "promoter" is to be taken in its broadest context and includes the transcriptional regulatory sequences derived from a classical eukaryotic genomic gene, including the TATA box which is required for accurate transcription initiation, with or<br>
without a CCAAT box sequence and additional regulatory or control elements (i.e.<br>
upstream activating sequences, enhancers and silencers) which alter gene expression in<br>
response to developmental and/or external stimuli, or in a tissue-specific manner.<br>
The term "promoter" also includes the transcriptional regulatory sequences of a classical<br>
prokaryotic gene, in which case it may include a -35 box sequence and/or a -10 box<br>
transcriptional regulatory sequences.<br>
The term "promoter" is also used to describe a synthetic or fusion molecule, or derivative<br>
which confers, activates or enhances expression of a nucleic acid molecule in a cell,<br>
tissue or organ.<br>
Promoters may contain additional copies of one or more specific regulatory elements, to<br>
further enhance  expression and/or to alter the spatial expression and/or temporal<br>
expression of a nucleic acid molecule to which it is operably connected. Such regulatory<br>
elements may be placed adjacent to a heterologous promoter sequence to drive<br>
expression of a nucleic acid molecule in response to e.g. copper, glucocorticoids,<br>
dexamethasone,   tetracycline,   gibberellin,   cAMP,   abscisic   acid,   auxin,   wounding,<br>
ethylene, jasmonate or salicylic acid or to confer expression of a nucleic acid molecule to<br>
specific cells, tissues or organs such as meristems, leaves, roots, embryo, flowers,<br>
seeds or fruits.<br>
In the context of the present invention, the promoter preferably is a plant-expressible<br>
promoter sequence. Promoters that also function or solely function in non-plant cells<br>
such as bacteria, yeast cells, insect cells and animal cells are not excluded from the<br>
invention. By "plant-expressible" is meant that the promoter sequence, including any<br>
additional regulatory elements added thereto or contained therein, is at least capable of<br>
inducing, conferring, activating or enhancing expression in a plant cell, tissue or organ,<br>
preferably a monocotyledonous or dicotyledonous plant cell, tissue, or organ.<br>
The terms "plant-operable" and "operable in a plant" when used herein, in respect of a<br>
promoter sequence, shall be taken to be equivalent to a plant-expressible promoter<br>
sequence.<br>
Regulatable promoters as part of a binary viral plant expression system are also known<br>
to the skilled artisan (Yadav 1999 - WO9922003; Yadav 2000 - WO0017365).<br>
In the present context, a "regulatable promoter sequence" is a promoter that is capable<br>
of conferring expression on a structural gene in a particular cell, tissue, or organ or group<br>
of cells, tissues or organs of a plant, optionally under specific conditions, however does<br>
generally not confer expression throughout the plant under all conditions. Accordingly, a<br>
regulatable promoter sequence may be a promoter sequence that confers expression on<br>
a gene to which it is operably connected in a particular location within the plant or<br>
alternatively, throughout the plant under a specific set of conditions, such as following<br>
induction of gene expression by a chemical compound or other elicitor.<br>
Preferably, the regulatable promoter used in the performance of the present invention<br>
confers expression in a specific location within the plant, either constitutively or following<br>
induction, however not in the whole plant under any circumstances. Included within the<br>
scope of such promoters are cell-specific promoter sequences, tissue-specific promoter<br>
sequences,  organ-specific promoter sequences,  cell  cycle  specific gene promoter<br>
sequences, inducible promoter sequences and constitutive promoter sequences that<br>
have been modified to confer expression in a particular part of the plant at any one time,<br>
such as by integration of said constitutive promoter within a transposable genetic<br>
element {Ac, Ds, Spm, En, or other transposon).<br>
Similarly,  the term  "tissue-specific"  shall  be taken to  indicate  that expression  is<br>
predominantly in a particular tissue or tissue-type, preferably of plant origin, albeit not<br>
necessarily exclusively in said tissue or tissue-type.<br>
Similarly,  the  term  "organ-specific"  shall  be taken  to  indicate  that expression  is<br>
predominantly in a particular organ, preferably of plant origin, albeit not necessarily<br>
exclusively in said organ.<br>
Similarly, the term "cell cycle specific" shall be taken to indicate that expression is<br>
predominantly cyclic and occurring in one or more, not necessarily consecutive phases<br>
of the cell cycle albeit not necessarily exclusively in cycling cells, preferably of plant<br>
origin.<br>
Those skilled in the art will be aware that an "inducible promoter" is a promoter the<br>
transcriptional activity of which is increased or induced in response to a developmental,<br>
chemical,  environmental, or physical stimulus. Similarly, the skilled craftsman will<br>
understand that a "constitutive promoter" is a promoter that is transcriptionally active<br>
throughout most, but not necessarily all parts of an organism, preferably a plant, during<br>
most, but not neccessarily all phases of its growth and development.<br>
Those skilled in  the art will  readily be capable of selecting appropriate promoter<br>
sequences for use in regulating appropriate expression of the cytokinin oxidase protein<br>
from publicly-available or readily-available sources, without undue experimentation.<br>
Placing a nucleic acid molecule under the regulatory control of a promoter sequence, or in operable connection with a promoter sequence, means positioning said nucleic acid molecule such that expression is controlled by the promoter sequence. A promoter is usually, but not necessarily, positioned upstream, or at the 5'-end, and within 2 kb of the start site of transcription,  of the nucleic acid  molecule which  it regulates.   In the construction  of  heterologous  promoter/structural  gene  combinations it is  generally preferred to position the promoter at a distance from the gene transcription start site that is approximately the same as the distance between that promoter and the gene it controls in its natural setting (i.e., the gene from which the promoter is derived). As is known in the art, some variation in this distance can be accommodated without loss of promoter function. Similarly, the preferred positioning of a regulatory sequence element with respect to a heterologous gene to be placed under its control is defined by the positioning of the element in its natural setting (i.e., the gene from which it is derived). Again, as is known in the art, some variation in this distance can also occur. Examples of promoters suitable for use in gene constructs of the present invention include those listed in Table 4, amongst others. The promoters listed in Table 4 are provided for the purposes of exemplification only and the present invention is not to be limited by the list provided therein. Those skilled in the art will readily be in a position to provide additional promoters that are useful in performing the present invention. In the case of constitutive promoters or promoters that induce expression throughout the entire plant,   it  is preferred that such  sequences are  modified  by the addition  of nucleotide sequences derived from one or more of the tissue-specific promoters listed in Table 4, or alternatively, nucleotide sequences derived from one or more of the above-mentioned tissue-specific inducible promoters, to confer tissue-specificity thereon. For example, the CaMV 35S promoter may be modified by the addition of maize Adh1 promoter sequence, to confer anaerobically-regulated root-specific expression thereon, as described previously (Ellis ef al„ 1987). Another example describes conferring root specific or root abundant gene expression by fusing the CaMV35S promoter to elements of the maize glycine-rich protein GRP3 gene (Feix and Wulff 2000 - WO0015662). Such modifications can be achieved by routine experimentation by those skilled in the art. The term "terminator" refers to a DNA sequence at the end of a transcriptional unit which signals termination of transcription. Terminators are 3'-non-translated DNA sequences containing  a  polyadenylation signal, which facilitates the addition  of polyadenylate<br>
sequences to the 3'-end of a primary transcript. Terminators active in cells derived from viruses, yeasts, moulds, bacteria, insects, birds, mammals and plants are known and described in the literature. They may be isolated from bacteria, fungi, viruses, animals and/or plants.<br>
Table 4. Exemplary plant-expressible promoters for use in the performance of the present invention    (Table Removed)<br>
Table 4 (continued). Exemplary plant-expressibfe promoters for use in the performance of the present invention (Table Removed)  <br>
Table 4 (continued). Exemplary plant-expressible promoters for use in the performance of the present invention  (Table Removed)   <br>
Table 4 (continued). Exemplary plant-expressible promoters for use in the performance of the present invention (Table Removed)<br><br><br>
Examples of terminators particularly suitable for use in the gene constructs of the present invention include the Agrobacterium tumefaciens nopaline synthase (NOS) gene terminator, the Agrobacterium tumefaciens octopine synthase (OCS) gene terminator sequence, the Cauliflower mosaic virus (CaMV) 35S gene terminator sequence, the Oryza sativa ADP-giucose pyrophosphorylase terminator sequence (13'Bt2), the Zea mays zein gene terminator sequence, the rbcs-1A gene terminator, and the rbcs-3A gene terminator sequences, amongst others.<br>
Preferred promoter sequences of the invention include root specific promoters such as but not limited to the ones listed in Table 5 and as outlined in the Examples.<br>
Table 5. Exemplary root specific promoters for use in the performance of the present Invention    (Table Removed)<br>
Those skilled in the art will be aware of additional promoter sequences and terminator sequences which may be suitable for use in performing the invention. Such sequences may readily be used without any undue experimentation.<br>
In the context of the current invention, "ectopic expression" or "ectopic overexpression" of a gene or a protein are conferring to expression patterns and/or expression levels of said gene or protein normally not occurring under natural conditions, more specifically is meant increased expression and/or increased expression levels. Ectopic expression can be achieved in a number of ways including operably linking of a coding sequence encoding said protein to an isolated homologous or heterologous promoter in order to create a chimeric gene and/or operably linking said coding sequence to its own isolated promoter (i.e. the unisoiated promoter naturally driving expression of said protein) in order to create a recombinant gene duplication or gene multiplication effect. With "ectopic co-expression" is meant the ectopic expression or ectopic overexpression of two or more genes or proteins. The same or, more preferably, different promoters are used to confer ectopic expression of said genes or proteins.<br>
Preferably, the promoter sequence used in the context of the present invention is operably [inked to a coding sequence or open reading frame (ORF) encoding a cytokinin oxidase protein or a homologue, derivative or an immunologically active and/or functional fragment thereof as defined supra.<br>
"Downregulation of expression" as used herein means lowering levels of gene expression and/or levels of active gene product and/or levels of gene product activity. Decreases in expression may be accomplished by e.g. the addition of coding sequences or parts thereof in a sense orientation (if resulting in co-suppression) or in an antisense orientation relative to a promoter sequence and furthermore by e.g. insertion mutagenesis (e.g. T-DMA insertion or transposon insertion) or by gene silencing strategies as described by e.g. Angell and Baulcombe (1998 - WO9836083), Lowe et al. (1989 - WO9853083), Lederer et al. (1999 - W09915682) or Wang et al. (1999 -WO9953050). Genetic constructs aimed at silencing gene expression may have the nucleotide sequence of said gene (or one or more parts thereof) contained therein in a sense and/or antisense orientation relative to the promoter sequence. Another method to downregulate gene expression comprises the use of ribozymes.<br>
Modulating, including lowering, the level of active gene products or of gene product activity can be achieved by administering or exposing cells, tissues, organs or organisms to said gene product, a homologue, derivative and/or immunologically active fragment thereof. Immunomodulation is another example of a technique capable of downregulation levels of active gene product and/or of gene product activity and comprises administration of or exposing to or expressing antibodies to said gene product to or in cells, tissues, organs or organisms wherein levels of said gene product and/or gene product activity are to be modulated. Such antibodies comprise "plantibodies", single chain antibodies, IgG antibodies and heavy chain camel antibodies as well as fragments thereof.<br>
Modulating, including lowering, the level of active gene products or of gene product activity can futhermore be achieved by administering or exposing cells, tissues, organs or organisms to an agonist of said gene product or the activity thereof. Such agonists include proteins (comprising e.g. kinases and proteinases) and chemical compounds identified according to the current invention as described supra.<br>
in the context of the current invention is envisaged the downregulation of the expression of a cytokinin oxidase gene as defined higher. Preferably said cytokinin oxidase gene is<br>
a plant cytokinin oxidase gene, more specifically an AtCKX. The invention further comprises downregulation of levels of a cytokinin oxidase protein or of a cytokinin oxidase activity whereby said cytokinin oxidase protein has been defined supra. Preferably said cytokinin oxidase protein is a plant cytokinin oxidase, more specifically an AtCKX.<br>
By "modifying cell fate and/or plant development and/or plant morphology and/or biochemistry and/or physiology" is meant that one or more developmental and/or morphological and/or biochemical and/or physiological characteristics of a plant is altered by the performance of one or more steps pertaining to the invention described herein.<br>
"Cell fate" refers to the cell-type or cellular characteristics of a particular cell that are produced during plant development or a cellular process therefor, in particular during the cell cycle or as a consequence of a cell cycle process.<br>
"Plant development" or the term "plant developmental characteristic" or similar term shall, when used herein, be taken to mean any cellular process of a plant that is involved in determining the developmental fate of a plant cell, in particular the specific tissue or organ type into which a progenitor cell will develop. Cellular processes relevant to plant development will be known to those skilled in the art. Such processes include, for example, morphogenesis, photomorphogenesis, shoot development, root development, vegetative development, reproductive development, stem elongation, flowering, and regulatory mechanisms involved in determining cell fate, in particular a process or regulatory process involving the cell cycle.<br>
"Plant morphology" or the term "plant morphological characteristic" or similar term will, when used herein, be understood by those skilled in the art to refer to the external appearance of a plant, including any one or more structural features or combination of structural features thereof. Such structural features include the shape, size, number, position, colour, texture, arrangement, and patternation of any cell, tissue or organ or groups of cells, tissues or organs of a plant, including the root, stem, leaf, shoot, petiole, trichome, flower, petal, stigma, style, stamen, pollen, ovule, seed, embryo, endosperm, seed coat, aleurone, fibre, fruit, cambium, wood, heartwood, parenchyma, aerenchyma, sieve element, phloem or vascular tissue, amongst others.<br>
"Plant biochemistry" or the term "plant biochemical characteristic" or similar term will, when used herein, be understood by those skilled in the art to refer to the metabolic and<br><br>
catalytic processes of a plant, including primary and secondary metabolism and the products thereof, including any small molecules, macromolecules or chemical compounds, such as but not limited to starches, sugars, proteins, peptides, enzymes, hormones, growth factors, nucleic acid molecules, celluloses, hemicelluloses, calloses, lectins, fibres, pigments such as anthocyanins, vitamins, minerals, micronutrients, or macronutrients, that are produced by plants.<br>
"Plant physiology" or the term "plant physiological characteristic" or similar term will, when used herein, be understood to refer to the functional processes of a plant, including developmental processes such as growth, expansion and differentiation, sexual development, sexual reproduction, seed set, seed development, grain filling, asexual reproduction, cell division, dormancy, germination, light adaptation, photosynthesis, leaf expansion, fibre production, secondary growth or wood production, amongst others; responses of a plant to externally-applied factors such as metals, chemicals, hormones, growth factors, environment and environmental stress factors (eg. anoxia, hypoxia, high temperature, low temperature, dehydration, light, dayiength, flooding, salt, heavy metals, amongst others), including adaptive responses of plants to said externally-applied factors.<br>
Means for introducing recombinant DNA into plant tissue or cells include, but are not limited to, transformation using CaCI2 and variations thereof, in particular the method described by Hanahan (1983), direct DNA uptake into protoplasts (Krens et al, 1982; Paszkowski et al, 1984), PEG-mediated uptake to protoplasts (Armstrong et a/, 1990) microparticle bombardment, electroporation (Fromm et al., 1985), microinjection of DNA (Crossway et al., 1986), microparticle bombardment of tissue explants or cells (Christou et al, 1988; Sanford, 1988), vacuum-infiltration of tissue with nucleic acid, or in the case of plants, T-DNA-mediated transfer from Agrobacterium to the plant tissue as described essentially by An era/.(1985), Dodds et al., (1985), Herrera-Estrelia et al. (1983a, 1983b, 1985). Methods for transformation of monocotyledonous plants are well known in the art and include /Ac/robacfer/um-mediated transformation (Cheng et al., 1997 - W09748814; Hansen 1998 - W09854961; Hiei et al., 1994 - WO9400977; Hiei et al., 1998 -W09817813; Rikiishi et al., 1999 - WO9904618; Saito et al., 1995 - WO9506722), microprojectile bombardment (Adams er al., 1999 - US5969213; Bowen et al., 1998 -US5736369; Chang et at., 1994 - W09413822; Lundquist ef al., 1999 -US5874265/US5990390; Vasii and Vasil,  1995 - US5405765. Walker er al.,  1999 -<br>
US5955362), DNA uptake (Eyal ef al., 1993 - W09318168), microinjection of Agrobacterium cells (von Holt, 1994 - DE4309203) and sonication (Finer et al., 1997 -US5693512).<br>
For microparticle bombardment of cells, a microparticle is propelled into a cell to produce a transformed cell. Any suitable ballistic cell transformation methodology and apparatus can be used in performing the present invention. Exemplary apparatus and procedures are disclosed by Stomp ef at. (U.S. Patent No. 5,122,466) and Sanford and Wolf (U.S. Patent No. 4,945,050). When using ballistic transformation procedures, the gene construct may incorporate a plasmid capable of replicating in the cell to be transformed. Examples of microparticles suitable for use in such systems include 1 to 5 um gold spheres. The DNA construct may be deposited on the microparticle by any suitable technique, such as by precipitation.<br>
A whole plant may be regenerated from the transformed or transfected cell, in accordance with procedures well known in the art. Plant tissue capable of subsequent clonal propagation, whether by organogenesis or embryogenesis, may be transformed with a gene construct of the present invention and a whole plant regenerated therefrom. The particular tissue chosen will vary depending on the clonal propagation systems available for, and best suited to, the particular species being transformed. Exemplary tissue targets include leaf disks, pollen, embryos, cotyledons, hypocotyls, megagametophytes, callus tissue, existing meristernatic tissue (e.g., apical meristem, axillary buds, and root meristems), and induced meristem tissue (e.g., cotyledon meristem and hypocoty! meristem).<br>
The term "organogenesis", as used herein, means a process by which shoots and roots are developed sequentially from meristernatic centres.<br>
The term "embryogenesis", as used herein, means a process by which shoots and roots develop together in a concerted fashion (not sequentially), whether from somatic cells or gametes.<br>
Preferably, the plant is produced according to the inventive method is transfected or transformed with a genetic sequence, or amenable to the introduction of a protein, by any art-recognized means, such as microprojectile bombardment, microinjection, Agrobacterium-mediated transformation (including in planta transformation), protoplast fusion, or electroporation, amongst others. Most preferably said plant is produced by Agrobacterium-mediated transformation.<br>
Agrobactrium-mediated transformation or agrolistic transformation of plants, yeast, moulds or filamentous fungi is based on the transfer of part of the transformation vector sequences, called the T-DNA, to the nucleus and on integration of said T-DNA in the genome of said eukaryote.<br>
With "Agrobacterium" is meant a member of the Agrobacteriaceae, more preferably Agrobacterium or Rhizobacterium and most preferably Agrobacterium tumefaciens. With "T-DNA", or transferred DNA, is meant that part of the transformation vector flanked by T-DNA borders which is, after activation of the Agrobacterium vir genes, nicked at the T-DNA borders and is transferred as a single stranded DNA to the nucleus of an eukaryotic cell.<br>
When used herein, with "T-DNA borders", "T-DNA border region", or "border region" are meant either right T-DNA border (RB) or left T-DNA border (LB). Such a border comprises a core sequence flanked by a border inner region as part of the T-DNA flanking the border and/or a border outer region as part of the vector backbone flanking the border. The core sequences comprise 22 bp in case of octopine-type vectors and 25 bp in case of nopaline-type vectors. The core sequences in the right border region and left border region form imperfect repeats. Border core sequences are indispensable for recognition and processing by the Agrobacterium nicking complex consisting of at least VirD1 and VirD2. Core sequences flanking a T-DNA are sufficient to promote transfer of said T-DNA. However, efficiency of transformation using transformation vectors carrying said T-DNA solely flanked by said core sequences is low. Border inner and outer regions are known to modulate efficiency of T-DNA transfer (Wang et al. 1987). One element enhancing T-DNA transfer has been characterized and resides in the right border outer region and is called overdrive (Peralta et al. 1986, van Haaren et al. 1987). With "T-DNA transformation vector" or "T-DNA vector" is meant any vector encompassing a T-DNA sequence flanked by a right and left T-DNA border consisting of at least the right and left border core sequences, respectively, and used for transformation of any eukaryotic cell.<br>
With "T-DNA vector backbone sequence" or "T-DNA vector backbone sequences" is meant all DNA of a T-DNA containing vector that lies outside of the T-DNA borders and, more specifically, outside the nicking sites of the border core imperfect repeats. The current invention includes optimized T-DNA vectors such thai vector backbone integration in the genome of a eukaryotic cell is minimized or absent. With "optimized T-<br>
DNA vector" is meant a T-DNA vector designed either to decrease or abolish transfer of vector backbone sequences to the genome of a eukaryotic cell. Such T-DNA vectors are known to the one familiar with the art and include those described by Hanson et al. (1999) and by Stuiver et al. (1999 - WO9901563).<br>
The current invention clearly considers the inclusion of a DNA sequence encoding a cytokinin oxidase, homologue, derivative or immunologically active and/or functional fragment thereof as defined supra, in any T-DNA vector comprising binary transformation vectors, super-binary transformation vectors, co-integrate transformation vectors, Ri-derived transformation vectors as well as in T-DNA carrying vectors used in agrolistic transformation. Preferably, said cytokinin oxidase is a plant cytokinin oxidase, more specifically an Arabidopsis thaliana (At)CKX. With "binary transformation vector" is meant a T-DNA transformation vector comprising:<br>
(a)	a T-DNA region comprising at least one gene of interest and/or at least one selectable marker active in the eukaryotic cell to be transformed; and<br>
(b)	a vector backbone region comprising at least origins of replication active in E. coli and Agrobacterium and markers for selection in E. coli and Agrobacterium.<br>
The T-DNA borders of a binary transformation vector can be derived from octopine-type or nopaline-type Ti plasmids or from both. The T-DNA of a binary vector is only transferred to a eukaryotic cell in conjunction with a helper plasmid. With "helper plasmid" is meant a plasmid that is stably maintained in Agrobacterium and is at least carrying the set of vir genes necessary for enabling transfer of the T-DNA. Said set of vir genes can be derived from either octopine-type or nopaline-type Ti plasmids or from both.<br>
With "super-binary transformation vector" is meant a binary transformation vector additionally carrying in the vector backbone region a  vir region of the Ti plasmid pTiBo542 of the super-virulent A. tumefaciens strain A281  (EP0604662, EP0687730). Super-binary transformation vectors are used in conjunction with a helper plasmid. With "co-integrate transformation vector" is meant a T-DNA vector at least comprising:<br>
(a)	a T-DNA region comprising at [east one gene of interest and/or at least one selectable marker active in plants; and<br>
(b)	a vector backbone region comprising at least origins of replication active in Escherichia coli and Agrobacterium, and markers for selection in E. coli and<br>
Agrobacterium, and a set of vir genes necessary for enabling transfer of the T-<br>
DNA. The T-DNA borders and said set of vir genes of a said T-DNA vector can be derived from either octopine-type or nopaline-type Ti plasmids or from both. With "Ri-derived plant transformation vector" is meant a binary transformation vector in which the T-DNA borders are derived from a Ti plasmid and said binary transformation vector being used in conjunction with a 'helper' Ri-plasmid carrying the necessary set of vir genes.<br>
As used herein, the term "selectable marker gene" or "selectable marker" or "marker for selection" includes any gene which confers a phenotype on a cell in which it is expressed to facilitate the identification and/or selection of cells which are transfected or transformed with a gene construct of the invention or a derivative thereof. Suitable selectable marker genes contemplated herein include the ampicillin resistance (Amp1"), tetracycline resistance gene (Tcr), bacterial kanamycin resistance gene (Kanr), phosphinothricin resistance gene, neomycin phosphotransferase gene (npfll), hygromycin resistance gene, ß-glucuronidase (GUS) gene, chloramphenicol acetyltransferase (CAT) gene, green fluorescent protein (gfp) gene (Haseloff et al, 1997), and luciferase gene, amongst others.<br>
With "agrolistics", "agrolistic transformation" or "agrolistic transfer" is meant here a transformation method combining features of Agrobacrerium-rnediated transformation and of biolistic DNA delivery. As such, a T-DNA containing target plasmid is co-delivered with DNA/RNA enabling in planta production of VirD1 and VirD2 with or without VirE2 (Hansen and Chilton 1996; Hansen et al. 1997; Hansen and Chilton 1997 - WO9712046). With "foreign DNA" is meant any DNA sequence that is introduced in the host's genome by recombinant techniques. Said foreign DNA includes e.g. a T-DNA sequence or a part thereof such as the T-DNA sequence comprising the selectable marker in an expressible format. Foreign DNA furthermore include intervening DNA sequences as defined supra. With "recombination event" is meant either a site-specific recombination event or a recombination event effected by transposon 'jumping'.<br>
With "recombinase" is meant either a site-specific recombinase or a transposase. With "recombination site" is meant either site-specific recombination sites or transposon border sequences.<br>
With "site specific recombination event" is meant an event catalyzed by a system generally consisting of three elements: a pair of DNA sequences (the site-specific recombination   sequences   or   sites)   and   a   specific   enzyme   (the   site-specific recombinase). The site-specific recombinase catalyzes a recombination reaction only between two site-specific recombination sequences depending on the orientation of the site-specific recombination sequences. Sequences intervening between two site-specific recombination sites will be inverted in the presence of the site-specific recombinase when the site-specific recombination sequences are oriented in opposite directions relative  to   one   another  (i.e.   inverted   repeats).   If  the  site-specific  recombination sequences are oriented in the same direction relative to one another (i.e. direct repeats), then any intervening sequences will be deleted upon interaction with the site-specific recombinase. Thus, if the site-specific recombination sequences are present as direct repeats at both ends of a foreign DNA sequence integrated into a eukaryotic genome, such integration of said sequences can subsequently be reversed by interaction of the site-specific recombination sequences with the corresponding site specific recombinase. A number of different site specific recombinase systems can be used including but not limited to the Cre/lox system of bacteriophage P1, the FLP/FRT system of yeast, the Gin recombinase of phage Mu, the Pin recombinase of E. coli, the PinB, PinD and PinF from Shigella, and the R/RS system of the pSR1   plasmid. Recombinases generally are integrases, resolvases or flippases. Also dual-specific recombinases can be used in conjunction with direct or indirect repeats of two different site-specific recombination sites corresponding to the dual-specific recombinase (WO99/25840). The two preferred site-specific recombinase systems are the bacteriophage P1   Cre/lox and the yeast FLP/FRT systems. In these systems a recombinase (Cre or FLP) interact specifically with its respective site-specific recombination sequence (!ox or FRT respectively) to invert or excise the intervening sequences. The site-specific recombination sequences for each of these two systems are relatively short (34 bp for lox and 47 bp for FRT). Some of these systems have already been used with high efficiency in plants such as tobacco  (Dale  et  al.   1990)  and  Arabidopsis  (Osborne  et  al.   1995).   Site-specific recombination systems have many applications in plant molecular biology including methods  for  control  of  homologous  recombination  (e.g.  US5527695),  for targeted insertion,  gene stacking,  etc.  (W099/25821)  and for  resolution  of complex T-DNA integration patterns or for excision of a selectable marker (WO99/23202).<br>
Although the site-specific recombination sequences must be linked to the ends of the DNA to be excised or to be inverted, the gene encoding the site specific recombinase may be located elsewhere. For example, the  recombinase gene could already be present in the eukaryote's DNA or could be supplied by a later introduced DNA fragment either  introduced  directly into  cells,  through  crossing  or through   cross-pollination. Alternatively, a substantially purified recombinase protein could be introduced directly into the eukaryotic cell, e.g. by micro-injection or particle bombardment. Typically, the site-specific recombinase coding region will be operably linked to regulatory sequences enabling expression of the site-specific recombinase in the eukaryotic cell. With "recombination event effected by transposon jumping" or "transposase-mediated recombination" is meant a recombination event catalyzed by a system consisting of three elements: a pair of DNA sequences (the transposon border sequences) and a specific enzyme (the transposase). The transposase catalyzes a recombination reaction only between two transposon border sequences which are arranged as inverted repeats. A number of different transposon/transposase systems can be used including but not limited to the Ds/Ac system, the Spm system and the Mu system. These systems originate from corn but it has been shown that at least the Ds/Ac and the Spm system also function in other plants (Fedoroff et al. 1993, Schlappi et al. 1993, Van Sluys et al. 1987). Preferred are the Ds- and the Spm-type transposons which are delineated by 11 bp- and 13 bp- border sequences, respectively.<br>
Although the transposon border sequences must be linked to the ends of the DNA to be excised, the gene encoding the transposase may be located elsewhere. For example, the recombinase gene could already be present in the eukaryote's DNA or could be supplied by a later introduced DNA fragment either introduced directly into cells, through crossing or through cross-pollination. Alternatively, a substantially purified transposase protein could be introduced directly into cells, e.g. by microinjection or by particle bombardment.<br>
As part of the current invention, transposon border sequences are included in a foreign DNA sequence such that they lie outside said DNA sequence and transform said DNA into a transposon-like entity that can move by the action of a transposase. As transposons often reintegrate at another locus of the host's genome, segregation of the progeny of the hosts in which the transposase was  allowed to act might be<br>
necessary to separate transformed hosts containing e.g. only the transposon footprint and transformed hosts still containing the foreign DNA.<br>
In performing the present invention, the genetic element is preferably induced to mobilize, such as, for example, by the expression of a recombinase protein in the cell which contacts the integration site of the genetic element and facilitates a recombination event therein,  excising the genetic element completely,  or alternatively,  leaving  a "footprint", generally of about 20 nucleotides in length or greater, at the original integration site. Those hosts and host parts that have been produced according to the inventive  method  can  be  identified  by standard  nucleic  acid  hybridization  and/or amplification techniques to detect the presence of the mobilizable genetic element or a gene construct comprising the same. Alternatively, in the case of transformed host cells, tissues, and hosts wherein the mobilizable genetic element has been excised, it is possible to detect a footprint in the genome of the host which has been left following the excision event, using such techniques. As used herein, the term "footprint" shall be taken to refer to any derivative of a mobilizable genetic element or gene construct comprising the same as described herein which is produced by excision, deletion or other removal of the mobilizable genetic element from the genome of a cell transformed previously with said gene construct. A footprint generally comprises at least a single copy of the recombination loci or transposon used to promote excision. However, a footprint may comprise additional sequences derived from the gene construct, for example nucleotide sequences derived from the left border sequence, right border sequence, origin of replication, recombinase-encoding or transposase-encoding sequence if used, or other vector-derived nucleotide sequences. Accordingly, a footprint is identifiable according to the nucleotide sequence of the recombination locus or transposon of the gene construct used, such as, for example, a sequence of nucleotides corresponding or complementary to a /ox site or frf site.<br>
The term "cell cycle" means the cyclic biochemical and structural events associated with growth and with division of cells, and in particular with the regulation of the replication of DNA and mitosis. Cell cycle includes phases called: GO, Gap1 (G1), DNA synthesis (S), Gap2 (G2), and mitosis (M). Normally these four phases occur sequentially, however, the cell cycle also includes modified cycles wherein one or more phases are absent resulting in modified cell cycle such as endomitosis, acytokinesis, polyploidy, polyteny, and endoreduplication.<br><br>
The term "cell cycle progression" refers to the process of passing through the different cell cycle phases. The term "cell cycle progression rate" accordingly refers to the speed at which said cell cycle phases are run through or the time spans required to complete said cell cycle phases.<br>
With "two-hybrid assay" is meant an assay that is based on the observation that many eukaryotic transcription factors comprise two domains, a DNA-binding domain (DB) and an activation domain (AD) which, when physically separated (i.e. disruption of the covalent linkage) do not effectuate target gene expression. Two proteins able to interact physically with one of said proteins fused to DB and the other of said proteins fused to AD will re-unite the DB and AD domains of the transcription factor resulting in target gene expression. The target gene in the yeast two-hybrid assay is usually a reporter gene such as the p-galactosidase gene. Interaction between protein partners in the yeast two-hybrid assay can thus be quantified by measuring the activity of the reporter gene product (Bartel and Fields 1997). Alternatively, a mammalian two-hybrid system can be used which includes e.g. a chimeric green fluorescent protein encoding reporter gene (Shioda et al., 2000).<br>
Furthermore, folding simulations and computer redesign of structural motifs of the protein of the invention can be performed using appropriate computer programs (Olszewski, Proteins 25 (1996), 286-299; Hoffman, Comput. Appl. Biosci. 1 (1995), 675-679). Computer modeling of protein folding can be used for the conformational and energetic analysis of detailed peptide and protein models (Monge, J. Mol. Biol. 247 (1995), 995-1012; Renouf, Adv. Exp. Med. Biol. 376 (1995), 37-45). In particular, the appropriate programs can be used for the identification of interactive sites of the cytokinin oxidases, its ligands or other interacting proteins by computer assistant searches for complementary peptide sequences (Fassina, Immunomethods 5 (1994), 114-120). Further appropriate computer systems for the design of protein and peptides are described in the prior art, for example in Berry, Biochem. Soc. Trans. 22 (1994), 1033-1036; Wodak, Ann, N. Y. Acac. Sci. 501 (1987), 1-13; Pabo, Biochemistry 25 (1986), 5987-5991. The results obtained form the above-described computer analysis can be used for, e.g. the preparation of peptidomimetics of the protein of the invention or fragments thereof. Such pseudopeptide analogues of the natural amino acid sequence of the protein ma"y very efficiently mimic the parent protein (Benkirane, J. Biol. Chem. 271  (1996), 33218-33224). For example, incorporation of easily available achiral CI-<br>
amino acid residues into a protein of the invention or a fragment thereof results in the substitution of amino bonds by polymethylene units of an aliphatic chain, thereby providing a convenient strategy for constructing a peptidomimetic (Banerjee, Biopolymers 39 (1996), 769-777). Superactive peptidomimetic analogues of small peptide hormones in other systems are described in the prior art (Zhang, Biochem. Biophys. Res. Commun. 224 (1996). 327-331). Appropriate peptidomimetics of the protein of the present invention can also be identified by the synthesis of peptidomimetic combinatorial libraries through successive amine alkylation and testing the resulting compounds, e.g., for their binding, kinase inhibitory and/or immunlogical properties. Methods for the generation and use of peptidomimetic combinatorial libraries are described in the prior art, for example in Ostresh, Methods in Enzymology 267 (1996), 220-234 and Dorner, Bioorg. Med. Chem. 4 (1996), 709-715.<br>
Furthermore, a three-dimensional and/or crystallographic structure of the protein of the invention can be used for the design of peptidomimetic inhibitors of the biological activity of the protein of the invention (Rose, Biochemistry 35 (1996), 12933-12944; Ruterber, Bioorg. Med. Chem. 4 (1996), 1545-1558).<br>
The compounds to be obtained or identified in the methods of the invention can be compounds that are able to bind to any of the nucleic acids, peptides or proteins of the invention. Other interesting compounds to be identified are compounds that modulate the expression of the genes or the proteins of the invention in such a way that either the expression of said gene or protein is enhanced or decreased by the action of said compound. Alternatively the compound can exert his action by enhancing or decreasing the activity of any of the proteins of the invention. Herein, preferred proteins are novel cytokinin oxidases.<br>
Said compound or plurality of compounds may be comprised in, for example, samples, e.g., cell extracts from, e.g., plants, animals or microorganisms. Furthermore, said compound(s) may be known in the art but hitherto not known to be capable of suppressing or activating cytokinin oxidase interacting proteins. The reaction mixture may be a cell free extract of may comprise a cell or tissue culture. Suitable set ups for the method of the invention are known to the person skilled in the art and are, for example, generally described in Alberts et al., Molecular Biology of the Cell, third edition (1994), in particular Chapter 17. The plurality of compounds may be, e.g., added to the reaction mixture, culture medium or injected into the ceil.<br>
If a sample containing a compound or a plurality of compounds is identified in the method of the invention, then it is either possible to isolate the compound form the original sample identified as containing the compound capable of acting as an agonist, or one can further subdivide the original sample, for example, if it consists of a plurality of different compounds, so as to reduce the number of different substances per sample and repeat the method with the subdivisions of the original sample. Depending on the complexity of the samples, the steps described above can be performed several times, preferably until the sample identified according to the method of the invention only comprises a limited number of or only one substance(s). Preferably said sample comprises substances or similar chemical and/or physical properties, and most preferably said substances are identical. Preferably, the compound identified according to the above-described method or its derivative is further formulated in a form suitable for the application in plant breeding or plant cell and tissue culture.<br>
The term "early vigor" refers to the ability of a plant to grow rapidly during early development, and relates to the successful establishment, after germination, of a well-developed root system and a well-developed photosynthetic apparatus. The term "resistance to lodging" or "standability" refers to the ability of a plant to fix itself to the soil. For plants with an erect or semi-erect growth habit this term also refers to the ability to maintain an upright position under adverse (environmental) conditions. This trait relates to the size, depth and morphology of the root system.<br>
The term 'grafting' as used herein, refers to the joining together of the parts of two different plants so that they bind together and the sap can flow, thus forming a single new plant that can grow and develop. A graft therefore consists of two parts: (i) the lower part is the rootstock as referred to herein and essentially consists of the root system and a portion of the stem, and (ii) the upper part, the scion or graft, which gives rise to the aerial parts of the plant.<br>
As used herein, tblastn refers to an alignment tool that is part of the BLAST (Basic Local Alignment Search Toof) family of programs (http://www.ncbi.nlm.nih.gov/BLAST/). BLAST aims to identify regions of optimal local alignment, i.e. the alignment of some portion of two nucleic acid or protein sequences, to detect relationships among sequences which share only isolated regions of similarity (Altschul et al., 1990). In the present invention, tblastn of the BLAST 2.0 suite of programs was used to compare the maize cytokinin oxidase protein sequence against a nucleotide sequence database<br>
dynamically translated in all reading frames (Altschul et al., Nucleic Acids Res. 25: 3389-3402 (1997)).<br>
The following examples and figures are given by means of illustration of the presentinvention and are in no way limiting. The contents of all references included in this application are incorporated by reference.<br>
BRIEF DESCRIPTION OF THE FIGURES<br>
Figure 1. Schematic representation of plant cytokinin oxidase genes.<br>
Shown are the structures of different cytokinin oxidase genes isolated from maize {ZmCKXI, accession number AF044603, Biochem. Biophys. Res. Com. 255:328-333, 1999) and Arabidopsis (MCKX1 to AiCKX4). Exons are denominated with !E' and represented by shaded boxes, lntrons are represented by white boxes. Further indicated are the gene sizes (in kb, on top of each structure), the gene accession numbers (under the names) and a size bar representing 0.5 kb.<br>
Figure 2. Alignment of plant cytokinin oxidase amino acid sequences.<br>
The amino acid sequences from cytokinin oxidases from maize (ZmCKXI) and Arabidopsis (AtCKXI to AtCKX4) are aligned. Identical amino acid residues are marked by a black box, similar amino acid residues are in a grey box. Amino acid similarity groups: (M,I,L,V), (F,W.Y), (G,A), (S,T), (R.K.H), (E,D), (N,Q),<br>
Figure 3. Northern blot analysis of AtCKKX1-expressing tobacco and Arabidopsis plants.<br>
(A)	Northern blot analysis of constitutively expressing tobacco plants (lanes 1-8)<br>
compared to wild type SNN tobacco (lane 9)<br>
(B)	Comparison of tetracycline-induced gene expression in leaves after 12h of induction with a constitutively expressing clone. Lanes 2-9, leaves of four different AtCKXI-W38TetR clones (+,-, with or without tetracycline treatment), lane 1, constitutively expressing 35S:: AtCKXI clone.<br>
(C)	Northern blot analysis of Arabidopsis plants constitutively expressing AtCKXI gene. Lanes 2-4, three different constitutively expressing 35S::AtCKX1 clones compared to wild type Arabidopsis plant (lane 1).<br>
Figure 4: Growth characteristics of 35S::AtCKX1 transgenic Arabidopsis plants.<br>
(A) Two wild type seedlings (left) compared to two 35S::AtCKX1 expressing seedlings (right). Note the increased formation of adventitious roots and increased root branching in the trangenic seedlings. Pictures were taken 14 days after germination. Plants were grown in vitro on MS medium in petri dishes in a vertical position.<br>
(B)	Like A, but roots stained with toiuidine blue.<br>
(C)	Top view of a petridish with 35S::AtCKX1 transgenic seedlings three weeks after germination.<br>
(D)	A 35S::AtCKX1 transgenic plants grown in liquid culture. Roots of wild type seedlings grow poorly under these conditions (not shown).<br>
(E)	Transformants (TO) that express the 35S::AtCKX1 gene (three plants on the right), a<br>
wild type plant is shown on the left.<br>
(F)	Phenotype of T1   plants grown in soil. Wild type plant (left)  compared to two<br>
35S::AtCKX1 trangenic plants.<br>
Figure 5: Phenotype of AtCKX2 overexpressing Arabidopsis plants.<br>
T1 generation of 35S::AtCKX2 expressing Arabidopsis plants (two plants on the right) compared to wild type (plant on the left).<br>
Figure 6. Northern blot analysis of ,4tCKX2-expressing tobacco and Arabidopsis plants.<br>
(A)	Northern blot analysis of constitutively expressing tobacco plants (lanes 1-7)<br>
compared to wild type SNN tobacco (lane 8)<br>
(B)	Northern blot analysis of Arabidopsis plants constitutively expressing AtCKX2 gene.<br>
Lanes 2-8, seven different consitutively expressing 35S::AtCKX2 clones compared to<br>
wild type Arabidopsis plant (lane 1).<br>
Figure 7. Shoot phenotype of AtCKXI and AtCKX2 expressing tobacco plants.<br>
(A)	Top view of six week old plants.<br>
(B)	Tobacco plants at the flowering stage.<br>
(C)	Kinetics of stem elongation. Arrows mark the onset of flowering. Age of plants (days after germination) and leaf number at that stage are indicated above the arrows. Bars indicate SD; n = 12.<br>
(D)	Number of leaves (n = 12) formed between day 68 and day 100 after germination and final surface area of these leaves (100% of wild type is 3646 ±144 cm2; n = 3).<br>
(E)	Comparison of leaf size and senescence. Leaves were from nodes number 4, 9, 12,<br>
16 and 20 from the top (from left to right).<br>
 Figure 8. Root phenotype of AtCKX expressing transgenic tobacco plants.<br>
(A)	Seedlings 17 days after germination.<br>
(B)	Root system of soil grown plants at the flowering stage.<br>
(C)	Root length, number of lateral roots (LR) and adventitious roots (AR) on day 10 after germination.<br>
(D)	Dose-response curve of root growth inhibition by exogenous cytokinin. Bars indicate ±SD; n = 30.<br>
Figure 9: Growth of axillary shoot meristems in 35S::AtCKX1 expressing tobacco plants.<br>
Figure 10: Histology of shoot meristems, leaves and root meristems of AtCKXI overexpressing tobacco plants versus wild type (WT) tobacco.<br>
(A)	Longitudinal median section through the vegetative shoot apical meristem. P, leaf primordia.<br>
(B)	Vascular tissue in second order veins of leaves. X, xylem, PH, a phloem bundle.<br>
(C)	Cross sections of fully developed leaves.<br>
(D)	Scanning electron microscopy of the upper leaf epidermis.<br>
(E)	Root apices stained with DAPI. RM, root meristem.<br>
(F)	Longitudinal median sections of root meristems ten days after germination. RC, root<br>
cap; PM, promeristem.<br>
(G)	Transverse root sections 10 mm from the apex. E, epidermis, C1-C4, cortical cell<br>
layer, X, xylem, PH, phloem. Bars are 100 µm.<br>
Figure 11: Northern blot analysis of AtCKX3 and 4tCKX4-expressing tobacco plants.<br>
(A) Northern blot analysis of constitutively expressing AtCKX3 tobacco plants. Lane designations indicate individual transgenic plant numbers, WT is wild type SNN tobacco. The blot on top was probed with a AtCKX3 specific probe, the lower blot with a probe specific for the 25S rRNA and serves as a control for RNA loading.<br>
(B) Northern blot analysis of constitutively expressing AtCKX4 tobacco plants. Lane designations indicate individual transgenic plant numbers, WT is wild type SNN tobacco. The blot on top was probed with an AtCKX4 specific probe, the lower blot with a probe specific for the 25S rRNA and serves as a control for RNA loading. Figure 12: Recipocal grafts of AtCKX2 transgenic tobacco plants and wild type plants.<br>
(A)	Two plants on the left: Control (WT scion grafted on a WT rootstock).<br>
Two plants on the right: WT scion grafted on a AtCKX2-38 transgenic rootstock.<br>
(B)	Left: Control (WT scion grafted on a WT rootstock). Right: Scion of AtCKX2-38 plant grafted on WT rootstock.<br>
(C)	Magnification of root area.<br>
Left: Control (WT scion grafted on a WT rootstock).<br>
Right: WT scion grafted on an AXCKX2-38 transgenic rootstock.<br>
(D)	Formation of adventitious roots.<br>
Left: Control (WT scion grafted on an WT rootstock).<br>
Right: WT scion grafted on an AtCKX2-38 transgenic rootstock.<br>
EXAMPLES<br>
Example 1. Brief descnption of the sequences of the invention<br><br>
(Table Removed)<br>
Example 2. identification of candidate cytokinin oxidase encoding genes from Arabidopsis thaliana<br>
Six different  genes were identified  from  Arabidopsis  thaliana that  bear  sequence similarity to a cytokinin oxidase gene from maize (Morris et al., Biochem Biophys Res Comm 255:328-333, 1999; Houda-Herin et al. Plant J 17:615-626; WO 99/06571). These genes were found by screening 6-frame translations of nucleotide sequences from public genomic databases with the maize protein sequence, employing tblastn program. These sequences were designated as Arabidopsis thaliana cytokinin oxidase-like genes or AtCKX. They were arbitrarily numbered as AtCKXI to AtCKX6. The below list summarizes the information on these genes. The predicted ORF borders and protein sequences are indicative, in order to illustrate by approximation the protein sequence divergence between the Arabidopsis and maize cytokinin oxidases, as well as amongst the different Arabidopsis cytokinin oxidases. The ORF borders and protein sequences shown should not be taken as conclusive evidence for the mode of action of these AtCKX genes. For DNA and protein sequence comparisons the program MegAlign from DNAstar was used. This program uses the Clustal method for alignments. For multiple alignments of protein and cDNA sequences the gap penalty and gap length penalty was set at 10 each. For pairwise alignments of proteins the parameters were as follows: Ktuple at   1;  Gap penalty at 3; window at 5;  diagonals saved at 5.  For pairwise alignments of cDNA's the parameters were as follows: Ktuple at 2; Gap penalty at 5; window at 4; diagonals saved at 4. The similarity groups for protein alignments was: (M,I,L,V), (F.W.Y), (G,A), (S,T), (R.K.H), (E,D), (N,Q).    The values that are indicated amongst the Arabidopsis cDNA and protein sequences represent the lowest and highest values found with all combinations.<br>
A. Gene name: AtCKXI (Arabidopsis thaliana cytokinin oxidase-like protein 1, SEQ ID N01)<br>
Location in database (accession number, location on bac): AC002510, Arabidopsis thaliana chromosome II section 225 of 255 of the complete sequence. Sequence from clones T32G6.<br>
ORF predicted in the database:<br>
15517..16183, 16415..16542, 1 6631 ..16891, 16995..17257, 17344..17752<br>
The AtCKXI cDNA sequence is listed as SEQ ID NO 25 Predicted protein sequence: SEQ ID NO 2<br>
Homologies<br>
% identity with Z. mays cDNA:<br>
31,5% (Dnastar/MegAlign - Clustal method) % similarity with Z. mays protein:<br>
32,2% (Dnastar/MegAlign - Clustal method)<br>
% identity with other Arabidopsis cDNA's (range):<br>
38,2% {AtCKX2) - 54,1% (AtCKX6) (Dnastar/MegAlign - Clustal method)<br>
% similarity with other Arabidopsis proteins (range):<br>
37,1% (AtCKX2) - 58,1 % {AtCKX6) (Dnastar/MegAlign - Clustal method)<br>
B. Gene name: AtCKX2 (Arabidopsis thaliana cytokinin oxidase-like protein 2, SEQ ID N03)<br>
Location in database (accession number, location on bac): AC005917, Arabidopsis thaliana chromosome II section 113 of 255 of the complete sequence. Sequence from clones F27F23, F3P11.<br>
ORF predicted in the database:<br>
complement, 40721..41012, 41054..41364, 41513-41770, 42535.-42662, 43153..43711<br>
Please note: The cDNA sequence identified by the inventor using the gene prediction program NetPlantGene (http://www.cbs.dtu.dk/services/NetGene2/) was different than the one annotated in the database. Based on the new cDNA sequence the ORF predicted in the database was revised:<br>
complement, 40721 ..41012, 41095..41364, 41513..41770, 42535.-42662, 43153..43711 The protein sequence encoded by this cDNA is listed as SEQ ID NO 4. The cDNA of AtCKX2 was cloned by RT-PCR from total RNA of AtCKX2 transgenic plant tissue with the one-step RT-PCR kit (Qiagen, Hilden, Germany) and sequenced using an ABI PRISM Big Dye Terminator cycle sequencing reaction kit (Perkin Elmer Applied Biosystems Division). This confirmed that the cDNA sequence identified and predicted<br>
by the inventor was correct. The new AtCKX2 cDNA sequence is listed as SEQ ID NO 26. An 84-bp fragment corresponding to nucleotides 1171 through1254 of the AtCKX2 cDNA is listed as SEQ ID NO 31. The corresponding peptide sequence of this 84-bp cDNA sequence is listed as SEQ ID NO 32,<br>
Homologies<br>
% identity with Z mays cDNA:<br>
38,4% (Dnastar/MegAlign - Clustal method)<br>
% similarity with Z. mays protein:<br>
37,5% (Dnastar/MegAlign - Clustal method)<br>
% identity with other Arabidopsis cDHA's (range):<br>
34,9% (AtCKX6) - 64,5% {AtCKX4) (Dnastar/MegAlign - Clustal method)<br>
% similarity with other Arabidopsis proteins (range):<br>
36,5% [AtCKXG) - 66,1 % (AtCKX4) (Dnastar/MegAlign - Clustal method)<br>
C. Gene name: AtCKX3 (Arabidopsis thaliana cytokinin oxidase-like protein 3, SEQ ID NO 5)<br>
Location in database (accession number, location on bac): AB024035, Arabidopsis thaliana genomic DNA, chromosome 5, P1 clone: MHM17, complete sequence.<br>
No prediction of the ORF in the database.<br>
The gene was identified by the inventor using several gene prediction programs<br>
including GRAIL (ftp: //arthur.epm. ornl.gov/pub/xgrail), Genscan (http://CCR-<br>
081.mit.edu/GENSCAN	.html)	and	NetPlantGene<br>
(http://www.cbs.dtu.dk/services/NetGene2/):<br>
complement, 29415..29718, 29813..30081, 30183..30443, 30529..30656, 32107..32716 The new AtCKX3 cDNA sequence identified by the inventor is listed as SEQ ID NO 27<br>
Predicted protein sequence, based on own ORF prediction: SEQ ID NO 6<br>
Homologies<br>
% identity with Z. mays cDNA:<br>
38,7% (Dnastar/MegAlign - Clustal method)<br>
% similarity with Z mays protein:<br>
39,2% (Dnastar/MegAlign - Clustal method)<br>
% identity with other Arabidopsis cDNA's (range):<br>
38.8% (AtCKX6) - 51,0% {AtCKX2) (Dnastar/MegAlign - Clustal method)<br>
% similarity with other Arabidopsis proteins (range):<br>
39,9% (AtCKX6) - 46,7% {AtCKX2) (Dnastar/MegAlign - Clustal method)<br>
D. Gene name: AtCKX4 (Arabidopsis thaliana cytokinin oxidase-Iike protein 4, SEQ ID NO 7)<br>
Location in database (accession number, location on bac):<br>
1)	AL079344, Arabidopsis thaliana DNA chromosome 4, BAC clone T16L4 (ESSA project)<br>
2)	AL161575, Arabidopsis thaliana DNA chromosome 4, contig fragment No. 71.<br>
ORF predicted in the database:<br>
1)	76187..76814, 77189..77316, 77823..78080, 78318..78586, 78677..78968<br>
2)	101002..101 629, 102004..102131, 102638..102895, 103133..103401, 103492..103783<br>
The AtCKX4 cDNA sequence is listed as SEQ ID NO 28<br>
Predicted protein sequence: SEQ ID NO 8<br>
Homologies<br>
% identity with Z. mays cDNA:<br>
41,0% (Dnastar/MegAlign - Clustal method)<br>
% similarity with Z mays protein:<br>
41,0% (Dnastar/MegAlign - Clustal method)<br>
% identity with other Arabidopsis cDNA's (range):<br>
35,2% (AtCKXS) - 64,5% (AtCKX2) (Dnastar/M eg Align - Clustal method)<br>
% similarity with other Arabidopsis proteins (range):<br>
35,1% (AtCKX6) - 66,1% (AtCKX2) (Dnastar/MegAlign - Clustal method)<br>
E. Gene name: AtCKXS (Arabidopsis thaliana cytokinin oxidase-like protein 5, SEQ ID NO 9)<br>
Location in database (accession number, location on bac): AC023754, F1B16, complete sequence, chromosome 1<br>
No prediction of the ORF in the database.<br>
The gene was identified by the inventors using several gene prediction programs including      GRAIL      (ftp://arthur.epm.ornl.gov/pub/xqrail),      Genscan      (http://CCR:<br>
081.mit.edu/GEN	SCAN.html)	and	NetPlantGene<br>
(http://www.cbs.dtu.dk/services/NetGene2/).<br>
43756.-44347, 44435.-44562, 44700-44966,  45493.-45755, 46200..46560 The new AtCKX5 cDNA sequence identified and predicted by the inventor is listed as SEQ ID NO 29. The predicted protein sequence for this cDNA is listed as 5EQ ID NO 10. A second potential ATG startcodon is present 9 nucleotides more upstream in the genomic sequence. It is unclear which of these 2 startcodons encodes the first amino acid of the protein. Therefore, a second potential AtCKX5 cDNA starting at this upstream startcodon is also listed in this invention as SEQ ID NO 34. The corresponding genomic sequence is listed as SEQ ID NO 33 and the encoded protein as SEQ ID NO 35.<br>
Homologies<br>
% identity with Z mays cDNA:<br>
39,1% (Dnastar/MegAlign - Clustal method)<br>
% similarity with Z mays protein:<br>
36,6% (Dnastar/MegAlign - Clustal method)<br>
% identity with other Arabidopsis cDNA's (range):<br>
40,1 % (AtCKX2) - 44,0% {AtCKX3) (Dnastar/MegAlign - Clustal method)<br>
% similarity with other Arabidapsis proteins (range):<br>
41,6% (AtCKX4) - 46,4% {AtCKX6) (Dnastar/MegAlign - Clustal method)<br>
F. Gene name: AtCKX6 (Arabidapsis thaliana cytokinin oxidase-like protein 6, SEQ ID NO 11)<br>
Location in database (accession number, location on bac): AL163818, Arabidopsis thaliana DMA chromosome 3, P1 clone MAA21 (ESSA project).<br>
ORF predicted in the database:<br>
46630..47215, 47.343..47470, 47591 ..47806, 47899..48161, 48244..48565 The 4rCkkX6 cDNA sequence is listed as SEQ ID NO 30<br>
Predicted protein sequence: SEQ ID NO 12<br>
Homologies<br>
% identity with Z. mays cDNA:<br>
37,3% (Dnastar/MegAlign - Clustal method)<br>
% similarity with Z. mays protein:<br>
36,1% (Dnastar/MegAlign - Clustal method)<br>
% identity with other Arabidopsis cDNA's (range):<br>
34,9% {AtCKX2) - 54,1% {AtCKXI) (Dnastar/MegAlign - Clustal method)<br>
% similarity with other Arabidopsis proteins (range):<br>
35,1 % {AtCKX4) - 58,1 % (AtCKXI) (Dnastar/MegAlign - Clustal method)<br>
Genes AtCKX3 and AtCKX5 were not annotated as putative cytokinin oxidases in the database and ORFs for these genes were not given. Furthermore, the ORF (and consequently the protein structures) predicted for AtCKX2 was different from our own prediction and our prediction was confirmed by sequencing the AtCKX2 cDNA.<br>
A comparison of the gene structure of the Arabidopsis AtCKX genes 1 to 4 and the maize CKX gene is shown in Fig 1.<br>
The predicted proteins encoded by the Arabidopsis AtCKX genes show between 32% and 41% sequence similarity with the maize protein, while they show between 35%) and 66% sequence similarity to each other. Because of this reduced sequence conservation, it is not clear a priori whether the Arabidopsis AtCKX genes encode proteins with cytokinin oxidase activity. An alignment of the Arabidopsis AtCKX predicted proteins 1 to 4 and the maize CKX gene is shown in Fig 2.<br>
Example 3. Transgenic plants overexpressing AtCKXI showed increased cytokinin oxidase activity and altered plant morphology<br>
1.	Description of the cloning process<br>
The following primers were used to PCR amplify the AtCKXI gene from Arabidopsis<br>
thaliana, accession Columbia (non-homologous sequences used for cloning are in lower<br>
case):<br>
Sequence of 5" primer: cggtcgacATGGGATTGACCTCATCCTTACG (SEQ ID NO:13)<br>
Sequence of 3' primer: gcgtcgacTTATACAGTTCTAGGTTTCGGCAGTAT (SEQ ID NO:<br>
14)<br>
A 2235-bp PCR fragment, amplified by these primers, was inserted in the Sal I site of pUC19. The insert was sequenced and confirmed that the PCR amplification product did not contain any mutations. The Sail/Sail fragment of this vector was subcloned in the Sail site downstream of a modified CaMV 35S promoter (carrying three tetracycline operator sequences) in the binary vector pBinHyg-Tx (Gatz er a/., 1992). The resulting construct was introduced into tobacco and Arabidopsis thaliana through Agrobacterium-mediated transformation, using standard transformation protocols.<br>
2.	Molecular analysis of the transgenic lines<br>
Several transgenic lines were identified that synthesize the AtCKXI transcript at high levels (Fig 3). Transgenic lines expressing AtCKXI transcript also showed increased cytokinin oxidase activity as determined by a standard assay for cytokinin oxidase activity based on conversion of [2-3HliP to adenine as described (Motyka et aL, 1996).<br>
This is exemplified for 2 tobacco and 2 krabidopsis lines in Table 6. This result proves that the AtCKXI gene encodes a protein with cytokinin oxidase activity.<br>
Table 6. Cytokinin oxidase activity in AtCKXI transgenic plant tissues  <br>
(Table Removed)<br>
3. Phenotypic description of the transgenic lines<br>
3.1 In tobacco:<br>
The plants had a dwarfed phenotype with reduced apical dominance (Figure 7 A, B and<br>
C) and increased root production (Figure 8).<br>
Five categories of phenotype:<br>
1)	strong - 2 clones<br>
2)	intermediate - 3 clones<br>
3)	weak - 4 clones<br>
4)	tall plants (as WT) with large inflorescence - 5 clones<br>
5)	similar to WT, 9 clones<br>
Height fsee Fig. 7 B and C)<br>
WT: between 100-150 cm weak: approximately 75 cm<br>
intermediate: appr. 40-45 cm (main stem app. 25 cm but overgrown by side branches. -    strong: appr. 10 cm<br>
The transgenics AtCKX1-48 and AtCKXI-50 displayed a strong phenotype. Below are measurements for stem elongation as compared to WT plants:<br>
(Table Removed)<br>
Experimental Plants were grown in soil in a greenhouse. Data were collected from at least ten plants per line.<br>
Leaves (see Figure 7 D and El<br>
The shape of leaves of AtCKXI transgenic expressors was lanceolate (longer and narrow): the width-to-length ratio of mature leaves was reduced from 1:2 in wild type plants to 1:3 in AtCKXI transgenics (Figure 7 E). The number of leaves and leaf surface was reduced compared to WT (see Figure 7 D). A prominent difference was also noted for progression of leaf senescence. In WT tobacco, leaf senescence starts in the most basal leaves and leads to a uniform reduction of leaf pigment (Figure 7 E). By contrast, ageing leaves of strongly expressing AtCKXI plants stayed green along the leaf veins<br>
and turned yellow in the intercostal regions, indicating altered leaf senescence. The texture of older leaves was more rigid.<br>
Roots<br>
In vitro grown plants highly expressing the gene were easily distinguishable from the WT<br>
by their ability to form more roots which are thicker (stronger) (Figure 8 A), as well as by<br>
forming aerial roots along the stem.<br>
The primary root was longer and the number of lateral and adventitious roots was higher<br>
as illustrated in Figure 8 C for AtCKX1-50 overexpressing seedlings (see also Example<br>
9).<br>
The dose-response curve of root growth inhibition by exogenous cytokinin showed that roots of transgenic seedllings are more cytokinin resistant than WT roots (Figure 8 D). The resistance of AtCKXI transgenics to iPR was less marked than for AtCKX2, which is consistent with the smaller changes in iP-type cytokinins in the latter (see Table 10). A large increase in root biomass was observed for adult plants grown in soil (see Figure 8 B for a plant grown in soil for 4 to 5 months) despite the fact that growth of the aerial plant parts was highly reduced.<br>
Internode distance<br>
•	intermediate phenotype: the 5lf1 internode below inflorescence is about 2.5 cm long and 9th internode was about 0,5 cm long compared to 5 cm and 2 cm for the length of the 5m and 9th internode respectively, in WT plants.<br>
•	strong phenotype: plant AtCKX1-50 The length of the 20th internode from the bottom measured at day 131 after germination was 1,3 ± 0,4 mm compared to 39,2 ±3,8 mm for WT<br>
Apical dominance and branching<br>
More side branches were formed indicating reduced apical dominance compared to WT plants during vegetative growth (see Figure 9). The side branches overgrew the main stem, reaching a height of 40-45 cm for intermediate AtCKXI expressors. Even secondary branches appeared. However, the buds were not completely released from apical dominance, i.e. lateral shoots did not really continue to develop. The reduced<br>
apical dominance might be due to reduced auxin production by the smaller shoot apical meristem (see Example 10). Reproductive development<br>
The onset of flowering in AtCKXI transgenics was delayed, the number of flowers and the seed yield per capsule was reduced. The size of flowers was not altered in transgenic plants and the weight of the individual seeds was comparable to the weight of seeds from wild type plants. Data for two representative AtCKXI transgenics is summarized below:<br>
Onset of flowering         (Table Removed)<br>
Experimental: Data collected for at least ten plants per line. The full elongation of the first flower was defined as onset of flowering. DAG = days after germination.<br>
B. Number of seed capsules per plant<br>
(Table Removed)<br>
Experimental: Number of seed capsules was determined at least from 5 different plants. Please note that these plants were grown under greenhouse conditions during winter time. This affects negatively the number of flowers that are formed, in particular in the transgenic clones. However, the general picture that they form a reduced number of flowers is correct, n.d., not determined<br>
C. Seed yield / capsule (mq)   <br><br>
(Table Removed)<br>
Experimental: Seed yield was determined for at least 12 seed capsules. The size of seed capsules was very variable, hence the large standard deviations, n.d., not determined<br>
P. Weight of 100 seeds (mq)<br>
(Table Removed)<br>
Experimental: The seed biomass was determined as the weight of 100 seed from at least 5 different seed capsules, n.d., not determined<br>
3.2 In Arabidopsis<br>
onset of germination was same as for WT<br>
the total root system was enlarged and the number of side roots and adventitious<br>
roots was enhanced (see Figure 4 A through D)<br>
the growth of aerial organs was reduced resulting in a dwarfed phenotype (see<br>
Figure 4 E and F) and the leaf biomass was reduced. Leaf and flower formation<br>
is delayed.<br>
the life cycle was longer compared to WT and the seed yield  was  lower<br>
compared to WT<br>
The following morphometric data illustrate these phenotypes: Root development<br>
Total length of the root system   (Table Removed)<br>
B. Primary root length  (Table Removed)  C. Lateral roots (LR) length <br>
(Table Removed)  D. Adventitious roots length   (Table Removed)<br>
E. Number of lateral roots (LR) (Table Removed) <br>
F. Number of adventitious roots CAR)  <br>
(Table Removed)<br>
Experimental: Measurements were carried out on plants 8 days after germination in vitro on MS medium. At least 17 plants per line were scored.<br>
Shoot development<br>
Leaf surface    (Table Removed)<br>
Experimental: Leaf surface area of main rosette leaves formed after 30 days after germination was measured. 3 plants per clone were analysed. Reproductive development<br>
Onset of flowering   <br>
(Table Removed)<br>
Experimental: Plants were grown under greenhouse condition. At least 13 plants per clone were analysed. DAG = days after germination<br>
Conclusion: The analysis of AtCKXI transgenic Arabidopsis plants confirmed largely the results obtained from tobacco and indicates the general nature of the consequences of a reduced cytokinin content. The total root system was enlarged (the total root length was increased app. 110-140% in AtCKXI transgenics), the shoot developed more slowly (retarded flowering) and the leaf biomass was reduced. The seed yield was lower in the transgenics as well (data not shown).<br>
Example   4.    Transgenic   plants   overexpressinq   AtCKX2   showed    increased cytokinin oxidase activity and altered plant morphology<br>
1. Description of the cloning process<br>
The following primers were used to PCR amplify the AtCKX2 gene from Arabidopsis<br>
thaliana, accession Columbia (non-homologous sequences used for cloning are in lower<br>
case):<br>
Sequence of 5' primer: gcggtaccAGAGAGAGAAACATAAACAAATGGC (SEQ ID<br>
NO: 15)<br>
Sequence of 3' primer: gcggtaccCAATTTTACTTCCACCAAAATGC (SEQ ID NO: 16)<br>
A 3104-bp PCR fragment, amplified by these primers, was inserted in the Kpni site of pUCl9. The insert was sequenced to check that no differences to the published sequence were introduced by the PCR procedure. The Kpnl/Kpnl fragment of this vector was subcloned in the Kpnl site downstream of a modified CaMV 35S promoter (carrying three tetracycline operator sequences) in the binary vector pBinHyg-Tx (Gatz et a/., 1992). The resulting construct was introduced into tobacco and Arabidopsis thaliana through Agrobacterium-mediated transformation, using standard transformation protocols.<br>
2. Molecular analysis of the transgenic lines<br>
Several transgenic lines were identified that synthesize the AtCKX2 transcript at high levels (Fig 6). Transgenic lines expressing AtCKX2 transcript also showed increased cytokinin oxidase activity. This is exemplified for 2 tobacco and 3 Arabidopsis lines in Table 7. This result proves that the AtCKX2 gene encodes a protein with cytokinin oxidase activity.<br>
Table 7. Cytokinin oxidase activity in AtCKX2 transgenic plant tissues <br><br>
(Table Removed)<br>
3. Phenotypic description of the transgenic lines<br>
3.1 In tobacco (see Fig 7 to 10): Three categories of phenotype:<br>
1)	strong -15 clones (similar to intermediate phenotype of AtCKXI)<br>
2)	weak - 6 clones<br>
3)	others - similar to WT plants, 7 clones<br>
Aerial plant parts<br>
The observations concerning plant height, internode distance, branching, leaf form and yellowing were similar as for AtCKXI transgenics with some generally minor quantitative differences in that the dwarfing characteristics were more severe in AtCKXI transgenics than in AtCKX2 trangenics (compare AtCKXI plants with AtCKX2 plants in Figure 7 A and 8). This is illustrated below for stem elongation and internode distance measurements of clones with a strong phenotype AtCKX2-38 and AtCKX2-40:<br>
Stem elongation   <br><br>
(Table Removed)<br>
Experimental: Plants were grown in soil in a green house. Data were collected from at least ten plants per line.<br>
Intemode distance<br>
(Table Removed)<br>
Experimental: The length of the 20   intemode from the bottom was measured at<br>
day 131 after germination. Roots<br>
In vitro grown plants highly expressing the gene were easily distinguishable from WT plants by their ability to form more roots which are thicker (stronger) as well as by forming aerial roots along the stem.<br>
The primary root was longer and the number of lateral and adventitious roots was higher as illustrated in Figure 8 C for AtCKX2-38 overexpressing seedlings (see also Example 9).<br>
The dose-response curve of root growth inhibition by exogenous cytokinin showed that roots of transgenic seedllings were more cytokinin resistant than WT roots (Figure 8 D). The resistance of AtCKX1-28 transgenics to iPR was less marked than for AtCKX2-38, which is consistent with the smaller changes in iP-type cytokinins in the latter (see Table 10).<br>
An increase in fresh and dry weight of the root biomass of TO lines of AtCKX2 transgenic plants compared to WT was observed for plant grown in soil, as illustrated in the following table: <br><br>
(Table Removed)<br>
Experimental: Six WT plants and six independent TO lines of 35S::AtCKX2 clone<br>
were grown on soil. After flowering the root system was washed with water, the<br>
soil was removed as far as possible and the fresh weight and dry weight was<br>
measured.<br>
An increase in fresh and dry weight of the root biomass was also observed for F1<br>
progeny of AtCKX2 transgenics grown in hydroponics as compared to WT, as<br>
illustrated in the following table:<br>
(Table Removed)<br>
Experimental: Soil grown plants were transferred 60 days after germination to a<br>
hydroponic system (Hoagland's solution) and grown for additional 60 days. The<br>
hydroponic solution was aerated continuously and replaced by fresh solution<br>
every third day.<br>
In summary, transgenic plants grown in hydroponic solution formed approximately 65-<br>
150% more root biomass (fresh weight) than wild type plants. The increase in dry weight<br>
was 10-50%. This difference is possibly in part due to the larger cell volume of the<br>
transgenics. This reduces the relative portion of cell walls, which forms the bulk of dry<br>
matter material. The shoot biomass was reduced to 20%-70% of wild type shoots. The<br>
difference in fresh weight leads to a shift in the shoot/root ratio, which was approximately 8 in wild type but approximately 1 in the transgenic clones.<br>
Conclusion:<br>
An increase in root growth and biomass was observed for AtCKX2 transgenic seedlings and adult plants grown under different conditions compared to WT controls despite the fact that growth of the aerial plant parts is reduced. Quantitative differences were observed between different transgenic plants: higher increases in root biomass were observed for the strongest expressing clones.<br>
Reproductive development<br>
The onset of flowering in AtCKX2 transgenics was delayed, the number of flowers and the seed yield per capsule was reduced. These effects were very similar to those observed in the AtCKXI transgenic plants but they were less prominent in the AtCKX2 transgenics, as indicated in the tables below. The size of flowers was not altered in transgenic plants and the weight of the individual seeds was comparable to the weight of seeds from wild type plants.<br>
A.	Onset of flowering <br>
(Table Removed)<br>
Experimental: Data collected for at least ten plants per line. The full elongation of the first flower was defined as onset of flowering. DAG = days after germination.<br>
Number of seed capsules per plant    (Table Removed)<br>
Experimental: Number of seed capsules was determined at least from 5 different plants. Please note that these plants were grown under green house conditions during winter time. This affects negatively the number of flowers that are formed,<br>
in particular in the transgenic clones. However, the general picture that they form a reduced number of flowers is correct, n.d., not determined<br>
B.	Seed yield / capsule (mq)<br>
(Table Removed)<br>
Experimenter. Seed yield was determined for at least 12 seed capsules. The size of seed capsules was very variable, hence the large standard deviations, n.d., not determined<br>
C.	Weight of 100 seeds (mq)<br>
(Table Removed)<br>
Experimental: The seed biomass was determined as the weight of 100 seed from at least 5 different seed capsules, n.d., not determined<br>
3.2 In Arabidopsis:<br>
The following morphornetric data were obtained for AtCKX2 transgenics:<br>
Root development<br>
A.	Total length of the root system<br>
(Table Removed)<br>
B. Primary root length(Table Removed)<br>
C. Lateral roots length (Table Removed)<br><br>
P. Adventitious roots length (Table Removed)<br><br><br>
D.	Number of lateral roots (LR) (Table Removed)<br>
F. Number of adventitious roots (AR) (Table Removed)<br>
Experimental: Measurements were carried out on plants 8 d.a.g. in vitro on MS medium. At least 17 plants per line were scored.<br>
Shoot development<br>
Leaf surface  (Table Removed)<br>
Experimental: Leaf surface area of main rosette leaves formed after 30 days after germination was measured. 3 plants per clone were analysed.<br>
Reproductive development<br>
Onset of flowering<br>
(Table Removed)<br>
Experimental: Plants were grown under greenhouse condition. At least 13 plants per clone were analysed. DAG = days after germination.<br>
Conclusion: Arabidopsis AtCKX2 transgenics had reduced leaf biomass and a dwarfing phenotype similar to AtCKXI transgenics (compare Figure 5 with Figure 4 F). The total root system was also enlarged in AtCKX2 transgenic Arabidopsis. The total root length is increased approximately 50% in AtCKX2 transgenics. The AtCKXI transgenics have longer primary roots, more side roots and form more adventitious roots. AtCKX2 transgenics lack the enhanced growth of the primary root but form more side roots and lateral roots than WT.<br>
Summary.<br>
The phenotypes observed for AtCKX2 transgenics were very similar but not identical to the AtCKXI transgenics, which in turn were very similar but not identical to the results obtained for the tobacco trangenics. This confirms the general nature of the consequences of a reduced cytokinin content in these two plant species and therefore, similar phenotypes can be expected in other plant species as well. The main difference between tobacco and Arabidopsis is the lack of enhanced primary root growth in AtCKX2 overexpressing plants (data not shown).<br>
Example 5. Transgenic plants overexpressing AtCKX3 showed increased cytokinin oxidase activity and altered plant morphology<br>
1. Description of the cloning process<br>
The following primers were used to PCR amplify the AtCKX3 gene from Arabidopsis<br>
thaliana, accession Columbia (non-homologous sequences used for cloning are in lower<br>
case):<br>
Sequence of 5' primer: gcggtaccTTCATTGATAAGAATCAAGCTATTCA (SEQ ID<br>
NO:17)<br>
Sequence of 3' primer: gcggtaccCAAAGTGGTGAGAACGACTAACA (SEQ ID NO:18)<br>
A 3397-bp PCR fragment, produced by this PCR amplification, was inserted in the Kpnl site of pBluescript. The insert was sequenced to confirm that the PCR product has no sequence changes as compared to the gene. The Kpnl/Kpnl fragment of this vector was subcloned in the Kpnl site downstream of a modified CaMV 35S promoter (carrying three<br>
tetracycline operator sequences) in the binary vector pBinHyg-Tx (Gatz et al., 1992). The resulting construct was introduced into tobacco and Arabidopsis thai/ana through Agrobacterium-mediated transformation, using standard transformation protocols.<br>
2. Molecular analysis of the transgenic lines<br>
Several transgenic tobacco lines were identified that synthesize the AtCKX3 transcript at high levels (Fig 11 A.). Transgenic tobacco lines expressing AtCKX3 transcript also showed increased cytokinin oxidase activity. This is exemplified for three plants in Table 8. This proves that the AtCKX3 gene encodes a protein with cytokinin oxidase activity.<br>
Table 8. Cytokinin oxidase activity in AtCKX4 transgenic plant tissues  (Table Removed)<br>
3, Plant phenotypic analysis<br>
The phenotypes generated by overexpression of the AtCKX3 gene in tobacco and Arabidopsis were basically similar as those of AtCKXl and AtCKX2 expressing plants, i.e. enhanced rooting and dwarfing. However, overexpression of the AtCKX3 gene in tobacco resulted in a stronger phenotype compared to AtCKX2. In this sense AtCKX3 overexpression was more similar to AtCKXl overexpression.<br>
Example   6.   Transgenic   plants   overexpressinq   AtCKX4   showed    increased cytokinin oxidase activity and altered plant morphology<br>
1.	Description of the cloning process<br>
The following primers were used to PCR amplify the AtCKX4 gene from Arabidopsis<br>
thaliana, accession Columbia (non-homologous sequences used for cloning are in lower<br>
case):<br>
Sequence of 5' primer: gcggtaccCCCATTAACCTACCCGTTTG (SEQ ID NO:19)<br>
Sequence of 3' primer:-gcggtaccAGACGATGAACGTACTTGTCTGTA (SEQ ID NO:20)<br>
A 2890-bp PCR fragment, produced by this PCR amplification, was inserted in the Kpni site of pBluescript. The insert was sequenced to confirm that the PCR product has no sequence changes as compared to the gene. The Kpnl/Kpnl fragment of this vector was subcloned in the Kpni site downstream of a modified CaMV 35S promoter (carrying three tetracycline operator sequences) in the binary vector pBinHyg-Tx (Gatz et al., 1992). The resulting construct was introduced into tobacco and Arabidopsis thaliana through Agrobacterium-medlated transformation, using standard transformation protocols.<br>
2.	Molecular analysis of the transgenic lines<br>
Several transgenic tobacco lines synthesized the AtCKX4 transcript at high levels (Fig 11 B.). Transgenic lines expressing AtCKX4 transcript also showed increased cytokinin oxidase activity. This is exemplified for 3 Arabidopsis and 3 tobacco lines in Table 9. This result proves that the AtCKX4 gene encodes a protein with cytokinin oxidase activity.<br>
Table 9. Cytokinin oxidase activity in AtCKX4 transgenic plant tissues<br><br>
(Table Removed)<br>
Overall, the data showed that the apparent Km values tor the four cytokinin oxidases were in the range of 0.2 to 9.5 µM with iP as substrate, which further demonstrates that the proteins encoded by AtCKXI through 4 are indeed cytokinin oxidase enzymes as disclosed herein.<br>
3. Plant phenotypic analysis<br>
The phenotypes generated by overexpression of the AtCKX4 gene in tobacco and Arabidopsis were basically similar as those of AtCKXI and AtCKX2 expressing plants, i.e. enhanced rooting, reduced apical dominance, dwarfing and yellowing of intercostal regions in older leaves of tobacco. An additional phenotype in tobacco was lanceolate feaves (altered (ength-to-wfdth ratio).<br>
General observations of AtCKX overexpressinq tobacco plants<br>
Overall, the phenotypic analysis demonstrated that AtCKX gene overexpression caused drastic developmental alterations in the plant shoot and root system in tobacco, including enhanced development of the root system and dwarfing of the aerial plant part. Other effects such as altered leaf senescence, formation of adventitious root on stems, and others were also observed as disclosed herein. The alterations were very similar, but not<br>
identical, for the different genes. In tobacco, AtCKXI and AtCKX3 overexpressors were alike as were AtCKX2 and AtCKX4. Generally, the two former showed higher expression of the traits, particularly in the shoot. Therefore, a particular cytokinin oxidase gene may be preferred for achieving the phenotypes that are described in the embodiments of this invention.<br>
Example 7. Cloning ot the AtCKX5 gene<br>
The following primers were used to PCR amplify the AtCKX5 gene from Arabidopsis<br>
thaliana, accession Columbia (non-homologous sequences used for cloning are in lower<br>
case):<br>
Sequence of 5' primer: gqqgtaccTTGATGAATCGTGAAATGAC (SEQ ID NO:21)<br>
Sequence of 3' primer: ggggtaccCTTTCCTCTTGGTTTTGTCCTGT (SEQ ID NO:22)<br>
The sequence of the 5' primer includes the two potential startcodons of the AtCKX5 protein, the most 5' startcodon is underlined and a second ATG is indicated in italics. A 2843-bp PCR fragment, produced by this PCR amplification, was inserted as a blunt-end product in pCR-Blunt ll-TOPO cloning vector (Invitrogen).<br>
Example 8. Cloning of the AtCKX6 gene<br>
The following primers were used to PCR amplify the AtCKX6 gene from Arabidopsis<br>
thaliana, accession Columbia (non-homologous sequences used for cloning are in lower<br>
case):<br>
Sequence of 5" primer: gctctagaTCAGGAAAAGAACCATGCTTATAG (SEQ ID NO:23)<br>
Sequence of 3' primer: gctctagaTCATGAGTATGAGACTGCCTTTTG (SEQ ID NO:24)<br>
A 1949-bp PCR fragment, produced by this PCR amplification, was inserted as a blunt-end product in pCR-Blunt ll-TOPO cloning vector (Invitrogen).<br>
Example 9. Tobacco seedling growth test demonstrated early vigor of AtCKX transgenics<br>
Seeds of AtCKX1-50 and AtCKX2-38 overexpressing transgenics and WT tobacco were sown in vitro on MS medium, brought to culture room 4 days after cold treatment and<br>
germinated after 6 days. Observations on seedling growth were made 10 days after germination (see also Figure 8C) and are summarized below. At least 20 individuals were scored per clone. Similar data have been obtained in two other experiments.<br>
A.	Total length of the root system<br>
(Table Removed)<br>
AtCKXI and AtCKX2 plants, general observations:<br>
Seedlings of AtCKXI and AtCKX2 overexpressing tobacco plants had 60% more adventitious roots and three times more lateral roots than untransformed control plants 10 days after germination. The length of the primary root was increased by about 70%. This - together with more and longer side roots and secondary roots - resulted in a 70-<br><br>
100% increase in total root length. These results showed that overexpression of cytokinin oxidase enhances the growth and development of both the main root and the adventitious roots, resulting in early vigor.<br>
Example 10. Histological analysis of altered plant morphology in AtCKXI overexpressing tobacco plants<br>
Microscopic analysis of different tissues revealed that the morphological changes in AtCKX transgenics are reflected by distinct changes in cell number and rate of cell formation (see Figure 10). The shoot apical meristem (SAM) of AtCKXI transgenics was smaller than in wild type and fewer cells occupy the space between the central zone and the peripheral zone of lateral organ fromation, but the cells were of the same size (Figure 10 A). The reduced cell number and size of the SAM as a consequence of a reduced cytokinin content indicates that cytokinins have a role in the control of SAM proliferation. No obvious changes in the differentiation pattern occurred, suggesting that the spatial organization of the differentiation zones in the SAM is largely independent from cell number and from the local cytokinin concentration. The overall tissue pattern of leaves in cytokinin oxidase overexpressers was unchanged. However, the size of the phloem and xylem was significantly reduced (Figure 10 B). By contrast, the average cell size of leaf parenchyma and epidermal ceils was increased four- to fivefold (Figure 10 C, D). New cells of AtCKXI transgenics are formed at 3-4% of the rate of wild type leaves and final leaf cell number was estimated to be in the range of 5-6% of wild type. This indicates an absolute requirement for cytokinins in leaves to maintain the cell division cycle. Neither cell size nor cell form of floral organs was altered and seed yield per capsule was similar in wild type and AtCKX transgenic plants. The cell population of root meristems of AtCKXI transgenic plants was enlarged approximately 4-fold and the cell numbers in both the central and lateral columnella were enhanced (Figure 10 E, F). The final root diameter was increased by 60% due to an increased diameter of all types of root cells. The radial root patterns was identical in wild type and transgenics, with the exception that frequently a fourth layer of cortex cells was noted in transgenic roots (Figure 10 G). The increased ceil number and the slightly reduced cell length indicates that the enhanced root growth is due to an increased number of cycling cells rather than increased cell growth. In the presence of lowered cytokinin content, root meristem cells<br>
must undergo additional rounds of mitosis before they leave the meristem and start to elongate. The exit from the meristem is therefore regulated by a mechanism that is sensitive to cytokinins. Apparently, cytokinins have a negative regulatory rote in the root meristem and wild type cytokinin concentrations are inhibitory to the development of a maximal root system. Therefore, reducing the level of active cytokinins by overexpressing cytokinin oxidases stimulates root development, which results in an increase in the size of the root with more lateral and adventitious roots as compared to WT plants.<br>
Example 11. AtCKXI and AtCKX2- overexpressing tobacco plants had a reduced cytokinin content.<br>
Among the 16 different cytokinin metabolites that were measured, the greatest change occurred in the iP-type cytokinins in AtCKX2 overexpressers (Table 10): the overall decrease in the content of iP-type cytokinins is more pronounced In AtCKX2 expressing plants than in AtCKXI transgenics. AtCKXI transgenics showed a stronger phenotype in the shoot. It is not known which cytokinin metabolite is relevant for the different traits that were analyzed. It may be that different cytokinin forms play different roles in the various development processes. Smaller alterations were noted for Z-type cytokinins, which could be due to a different accessibility of the substrate or a lower substrate specificity of the protein. The total content of iP and Z metabolites in individual transgenic clones was between 31% and 63% of wild type. The cytokinin reserve pool of O-glucosides was also lowered in the transgenics (Table 10). The concentration of N-g!ucosides and DHZ-type cytokinins was very low and was not or only marginally, altered in transgenic seedlings (data not shown).<br>
Table 10. Cytokinin content of AtCKX transgenic plants. Cytokinin extraction, immunopurification, HPLC separation and quantification by ELISA methods was carried out as described by Faiss et al., 1997. Three independently pooled samples of approximately 100 two week old seedlings (2.5 g per sample) were analyzed for each clone. Concentrations are in pmol x g fresh weight"1. Abbreviations: iP, N5-(A2isopentenyl)adenine; iPR, N6-(2isopentenyl)adenine riboside; iPRP, N6-(A2isopentenyl)adenine riboside 5'-monophosphate; Z, trans-zeatfn; ZR, zeatin riboside; ZRP, zeatin riboside 5'-monophosphate; ZOG, zeatin O-glucoside; ZROG, zeatin riboside O-glucoside.<br>
(Table Removed)<br>
Example 12. Grafting experiments showed that dwarfing and enhanced root development due to AtCKX overexpression is confined to transgenic tissues<br>
To investigate which phenotypic effects of cytokinin oxidase overexpression are restricted to expressing tissues, i.e. are cell- or organ-autonmous traits, grafting experiments were performed. Reciprocal grafts were made between an AtCKX2 transgenic tobacco plant and a WT tobacco. The transgenic plant used in this experiment was AtCKX2-38, which displayed a strong phenotype characterized by enhanced root growth and reduced development of the aerial plant parts. As described in Example 3 through 6, these were two important phenotypes that resulted from cytokinin oxidase overexpression in tobacco and arabidopsis.<br>
Plants were about 15 cm tall when grafted and the graft junction was about 10 cm above the soil. Figure 12 shows plants 15 weeks after grafting. The main results were that: (i) the aerial phenotype of a WT scion grafted on a transgenic rootstock was similar to the WT control graft (= WT scion on WT rootstock). Importantly, this showed that overexpression of the AtCKX2 transgene in the rootstock did not induce dwarfing of the non-transgenic aerial parts of the plant (see Figure 12 A), improved root growth of the transgenic rootstock was maintained, indicating that improved root growth of AtCKX transgenics is autonomous and does not depend on an AtCKX transgenic shoot (Figure 12 C). Interestingly, the WT scions grafted on the transgenic rootstocks looked healthier and were better developed. Notably, senescence of the basai leaves was retarded in these plants (see Figure 12 A); (ii) the transgenic scion grafted on the WT rootstock looked similar to the aerial part of the transgenic plant from which it was derived, i.e. the shoot dwarfing phenotype is also autonomous and not dependent on the improved root growth (see Figure 12 B).<br>
In addition to the above-mentioned better appearance of WT shoots grafted on a transgenic rootstock, the formation of adventitious roots on the basal part of WT shoots was noted (Figure 12 D, right plant). Formation of adventitious roots also occurred on the stem of ,4tCKXtransgenics but not on stems of WT control grafts (Figure 12 D, left plant) and therefore seems to be a non-autonomous trait.<br>
In summary, it is disclosed in this invention that enhanced root formation and dwarfing of the shoot in AtCKX'overexpressing tobacco are autonomous traits and can be uncoupled by grafting procedures. Surprisingly, grafting of a WT scion on an AtCKX<br>
transgenic rootstock resulted in more vigourosly growing plants and retardation of leaf<br>
senescence.<br>
As an alternative to grafting, tissue-specific promoters could be used for uncoupling the<br>
autonomous phenotypic effects of cytokinin overexpression. Therefore, it is disclosed in<br>
this invention that cytokinin oxidase overexpression in a tissue specific manner can be<br>
used to alter the morphology of a plant such as the shoot or root system.<br>
Example 13.. Expression of an AtCKX gene under a root-specific promoter in transgenic plants leads to increased root production<br>
An AtCKX gene (see example 4) is cloned under control of the root clavata homolog promoter of Arabidopsis (SEQ ID NO 36), which is a promoter that drives root-specific expression. Other root-specific promoters may also be used for the purpose of this invention. See Table 5 for exemplary root-specific promoters.<br>
Transgenic plants expressing the AtCKX gene specifically in the roots show increased root production without negatively affecting growth and development of the aerial parts of the plant. Positive effects on leaf senescence and growth of aerial plant parts are observed.<br>
Example 14. Suppression of an AtCKX gene under a senescence-induced promoter in transgenic plants leads to delayed leaf senescence and enhanced seed yield.<br>
A chimeric gene construct derived from an AtCKXgene and designed to suppress expression of endogenous cytokinin oxidase gene(s) is cloned under control of a senescence-induced promoter. For example, promoters derived from senescence-associated genes (SAG) such as the SAG12 promoter can be used (Quirino et al., 2000). Transgenic plants suppressing endogenous cytokinin oxidase gene(s) specifically in senescing leaves show delayed leaf senescence and higher seed yield without negatively affecting the morphology and growth and development of the plant<br>
Example 15. Overexpression of an AtCKX gene in the female reproductive organs leads to parthenocarpic fruit development<br>
The open reading frame of an AtCKX gene is cloned under control of a promoter that confers overexpression in the female reproductive organs such as for example the DefH9 promoter from Antirrhinum majus or one of its homologues, which have high expression specificity in the placenta and ovules. Transgenic plants with enhanced cytokinin oxidase activity in these tissues show parthenocarpic fruit development.<br>
REFERENCES<br>
WO0105985. Method to modulate the expression of genes inducing the parthenocarpic trait in plants.<br>
Alberts, B., Bray, D., Lewis, J., Raff, M., Roberts, K., and Watson, J. D. (1994). "Molecular Biology of the Cell." Garland Publishing Inc.<br>
Altschul, S.F., Madden, T.L., Schaffer, A.A., Zhang, J., Zhang, Z., Miller, W. &amp; Lipman, D.J. (1997) "Gapped BLAST and PSI-BLAST: a new generation of protein database search programs." Nucl. Acids Res. 25, 3389-3402.<br>
An, G., Watson, B. D., Stachel, S., Gordon, M. P., and Nester, E. W. (1985). New cloning vehicles for transformation of higher plants. EMBO J. 4, 277-284.<br>
Armstrong, C. L, Petersen, W. P., Buchholz, W. G., Bowen, B. A., and Sulc, S. L. (1990). Factors affecting PEG-mediated stable transformation of maize protoplasts. Plant Ceil Reports 9, 335-339.<br>
Banerjee, A., Pramanik, A., Bhattacharjya, S., and Balaram, P. (1996). Omega amino acids in peptide design: incorporation into helices. Biopolymers39, 769-777.<br>
Baron, M. H. and Baltimore, D. (1982). Antibodies against the chemically synthesized genome-linked protein of poliovirus react with native virus-specific proteins. Ce//28, 395-404.<br>
Bartel, P. L. and Fields, S. (1997). 'The Yeast Two-Hybrid System." Oxford University Press.<br>
Benkirane, N., Guichard, G., Briand, J. P., and Muller, S. (1996). Exploration of requirements for peptidomimetic immune recognition. Antigenic and immunogenic oroperties of reduced peptide bond pseudopeptide analogues of a histone hexapeptide. J.Biol Chem. 271, 33218-33224.<br>
Berry, A. and Brenner, S. E. (1994). A prototype computer system for de novo protein design. Biochem.Soc.Trans. 22, 1033-1036.<br>
Christou, P., McCabe, D. E., and Swain, W. F. (1988). Stable transformation of soybean callus by DNA-coated gold particles. Plant Physiol. 87, 671-674.<br>
rossway, A., Oakes, J. V., Irvine, J. M., Ward, B., Knauf, V. C, and Shewmaker, C. K. (1986). Integration of foreign DIMA following microinjection of tobacco mesophyd protoplasts. Mol.Gen.Genet. 202, 179-185.<br>
Dale, E. C. and Ow, D. W. (1990). Intra- and intermolecular site-specific recombination in plant cells mediated by bacteriophage P1 recombinase. Gene 91, 79-85.<br>
Dodds, J. H. (1985). "Plant genetic engineering." Cambridge University Press.<br>
Doerner, P., Jorgensen, J. E., You, R., Steppuhn, J., and Lamb, C. (1996). Control of root growth and development by cyclin expression. Nature 380, 520-523.<br>
Dorner, B., Husar, G. M„ Ostresh, J. M., and Houghten, R. A. (1996). The synthesis of peptidomirnetic combinatorial libraries through successive amide alkylations. Bioorg.Med.Chem. 4, 709-715.<br>
Ellis, J. G., Llewellyn, D. J., Dennis, E. S., and Peacock, W. J. (1987). Maize Adh-1 promoter sequences control anaerobic regulation: addition of upstream promoter elements from constitutive genes is necessary for expression in tobacco. EMBO J. 6, 11-16.<br>
Faiss, M., Zalubilova, J., Strnad, M., Schmulling, T. (1997). Conditional transgenic expression of the ipt gene indicates a function for cytokinins in paracrine signaling in whole tobacco plants. Plant J. 12, 401-415.<br>
Fassina, G. and Meili, M. (1994). Identification of interactive sites of proteins and protein receptors by computer-assisted searches for complementary peptide sequences. Immunomethods. 5, 114-120.<br>
Fedoroff, N. V. and Smith, D. L. (1993). A versatile system for detecting transposition in Arabidopsis. Plant J. 3, 273-289.<br>
Hanahan, D. (1983). Studies on transformation of Escherichia coli with plasmids. J.Mol.Biol 166, 557-580.<br>
Hansen, G.&gt; and Chilton, M. D. (1996). "Agrolistic" transformation of plant cells: integration of T-strands generated in planta. Proc.Natl.Acad.Sci.U.S.A 93, 14978-14983.<br>
Hansen, G., Shiliito, R. D., and Chilton, M. D. (1997). T-strand integration in maize protoplasts after codelivery of a T-DNA substrate and virulence genes. Proc.Natl.Acad.Sci.U.S.A 94,11726-11730.<br>
Hanson, 8., Engler, D., Moy, Y., Newman, B., Ralston, E., and Gutterson, N. (1999). A simple method to enrich an Agrobacterium-transformed population for plants containing only T-DNA sequences. Plant J. 19, 727-734.<br>
Harlow, E. and Lane, D. (1988). "Antibodies: A Laboratory Manual." Cold Spring Harbor Laboratory Press,<br>
Herrera-Estrella, L., De Block, M., Messens, E. H. J. P., Van Montagu, M., and Schell, J. (1983). Chimeric genes as dominant selectable markers in plant cells. EMBO J. 2, 987-995.<br>
Hoffman, D. L., Laiter, S., Singh, R. K., Vaisman, I. I., and Tropsha, A. (1995). Rapid protein structure classification using one-dimensional structure profiles on the bioSCAN parallel computer. Comput.Appl.Biosci. 11, 675-679.<br>
Krens, F. A., Molendijk, L., Wullems, G. J., and Schilperoort, R. A. (1982). In vitro transformation of plant protoplasts with Ti-plasmid DNA. Nature 296, 72-74.<br>
Lerner, R. A. (1982). Tapping the immunological repertoire to produce antibodies of predetermined specificity. Nature299, 593-596.<br>
Lerner, R. A., Green, N_, Alexander, H., Liu, F. T., Sutcliffe, J. G., and Shinnick, T. M. (1981). Chemically synthesized peptides predicted from the nucleotide sequence of the hepatitis B virus genome elicit antibodies reactive with the native envelope protein of Dane particles. Proc.Natl.Acad.Sci.U.S.A 78, 3403-3407.<br>
Liddle, J. E. and Cryer, A. (1991). "A Practical Guide to Monoclonal Antibodies." Wiley New York.<br>
Loffler, J., Langui, D., Probst, A., and Huber, G. (1994). Accumulation of a 50 kDa N-terminal fragment of beta-APP695 in Alzheimer's disease hippocampus and neocortex. Neurochem.lnt. 24, 281-288.<br>
Monge, A., Lathrop, E. J., Gunn, J. R., Shenkin, P. S., and Friesner, R. A. (1995). Computer modeling of protein folding: conformational and energetic analysis of reduced and detailed protein models. J.Mol.Biol 247, 995-1012.<br>
Morris, R.O. et ai. (1999). isolation of a gene encoding a glycosylated cytokinin oxidase from maize. Bioechem. Biophys. Res. Commun. 255, 328-333<br>
Motyka, V., Faiss, M„ Strnad, M., Kaminek, M. and Schmuelling, T. (1996). Changes in cytokinin content and cytokinin oxidase activity in response to derepression of ipt gene transcription in transgenic tobacco calli and plants. Plant Physiol. 112, 1035-1043.<br>
Murakami, T., Simonds, W. F., and Spiegel, A. M. (1992). Site-specific antibodies directed against G protein beta and gamma subunits: effects on alpha and beta gamma subunit, interaction. Biochemistry 31, 2905-2911.<br>
Olszewski, K. A., Kolinski, A., and Skolnick, J. (1996). Folding simulations and computer redesign of protein A three-helix bundle motifs. Proteins 25, 286-299.<br>
Osborne, B. I., Wirtz, U., and Baker, B. (1995). A system for insertional mutagenesis and chromosomal rearrangement using the Ds transposon and Cre-lox. Plant J. 7, 687-701.<br>
Ostresh, J. M., Blondelle, S. E., Dorner, B., and Houghten, R. A. (1996). Generation and use of nonsupport-bound peptide and peptidomimetic combinatorial libraries. Methods Enzymol. 267, 220-234.<br>
Pabo, C. O. and Suchanek, E. G. (1986). Computer-aided model-building strategies for protein design. Biochemistry 25, 5987-5991.<br>
Paszkowski, J., Shillito, R. D., Saul, M., Mandak, V., and Hohn, T. H. B. P. I. (1984). Direct gene transfer to plants. EMBO J. 3, 2717-2722.<br>
Peralta, E. G., Hellmiss, R., and Ream, W. (1986). Overdrive, a T-DNA transmission enhancer on the A. tumefaciens tumour-inducing plasmid. EMBO J. 5, 1137-1142.<br>
Quirino, B.F., Noh, Y.-S., Himelbau, E., and Amasino, R.M. (2000). Molecular aspects of leaf senescence. Trends in Plant Science 5, 278-282.<br>
Renouf, D. V. and Hounsell, E. F. (1995). Molecular modelling of glycoproteins by homology with non-glycosylated protein domains, computer simulated glycosylation and molecular dynamics. Adv.Exp.Med.Biol 376, 37-45.<br>
Rinaldi, A.C. and Comandini, O. (1999). Cytokinin oxidase strikes again. Trends in Plant Sc. 4, 300.<br>
Rose, R. B., Craik, C. S., Douglas, N. L, and Stroud, R. M. (1996). Three-dimensional structures of HIV-1 and SIV protease product complexes. Biochemistry 35, 12933-12944.<br>
Rutenber, E. E., McPhee, F., Kaplan, A. P., Gallion, S. L, Hogan, J. C, Jr., Craik, C. S., and Stroud, R. M. (1996). A new class of HIV-1 protease inhibitor: the crystallographic structure, inhibition and chemical synthesis of an aminimide peptide isostere. Bioorg.Med.Chem. 4,1545-1558.<br>
Sambrook, J., Fritsch, E. F., and Maniatis, T. (1989). "Molecular Cloning: A Laboratory Manual." Cold Spring Harbor Laboratory Press.<br>
Schlappi, M., Smith, D., and Fedoroff, N. (1993). TnpA trans-activates methylated maize Suppressor-mutator transposable elements in transgenic tobacco. Genetics 133, 1009-1021.<br>
Shioda, T., Andriole, S., Yahata, T., and Isselbacher, K. J. (2000). A green fluorescent protein-reporter mammalian two-hybrid system with extrachromosomal maintenance of a prey expression plasmid: Application to interaction screening. Proc.Natl.Acad.Sci. U.S.A 97 , 5220-5224.<br>
Tamura, R. N., Cooper, H. M., Collo, G., and Quaranta, V. (1991). Cell type-specific integrin variants with alternative alpha chain cytoplasmic domains. Proc.Natl.Acad.Sci.U.S.A 88, 10183-10187.<br>
Van Haaren, M. J., Sedee, N. J., Schilperoort, R. A., and Hooykaas. P. J. (1987). Overdrive is a T-region transfer enhancer which stimulates T-strand production in Agrobacterium tumefaciens. Nucleic Acids Res. 15, 8983-8997.<br>
Van Sluys, M. A., Tempe, J., and Fedoroff, N. (1987). Studies on the introduction and mobility of the maize Activator element in Arabidopsis thaliana and Daucus carota. EMBOJ. 6, 3881-3889.<br>
Wang, K., Genetello, C, Van Montagu, M., and Zambryski, P. C. (1987).. Sequence context of the T-DNA border repeat element determines its relative activity during T-DNA transfer to plant cells. MoLGen.Genet. 210, 338-346.<br>
Woulfe, J., Lafortune, L, de Nadai, F., Kitabgi, P., and Beaudet, A. (1994). Post-translational  processing  of the  neurotensin/neuromedin  N  precursor in  the  central<br>
nervous system of the rat—li. immunohistochemical localization of maturation products. Neuroscience 60, 167-181.<br>
Zhang, Y. L, Dawe, A. L, Jiang, Y., Becker, J. M., and Naider, F. (1996). A superactive peptidomimetic analog of a farnesylated dodecapeptide yeast pheromone. Biochem.Biaphys.Res.Commun. 224, 327-331.<br><br><br><br><br>
WE CLAIM:<br>
1. A method for stimulating root growth or for enhancing the formation of lateral or adventitious roots or for altering root geotropism, characterized in that the said method comprises overexpression of a nucleic acid encoding a plant cytokinin oxidase wherein said nucleic acid encoding a plant cytokinin oxidase is selected from the group consisting of:<br>
(a)	nucleic acids comprising a DNA sequence as given in any of SEQ ID Nos. 27,1,3,5,7,9,11,25,26,28 to 31,33 or 34, or the complement thereof,<br>
(b)	nucleic acids comprising the RNA sequences corresponding to any of SEQ ID Nos. 27,1,3,5,7,9,11,25,26,28 to 31,33 or 34, or the complement thereof,<br>
(c)	nucleic acids specifically hybridizing to any of SEQ ID Nos.<br>
27,1,3,5,7,9,11,25, 26, 28 to 31, 33 or 34, or to the complement thereof,<br>
(d)	nucleic acids encoding a protein comprising the amino acid sequence as given in any of SEQ ID Nos. 2, 4, 6, 8, 10, 12, 32 or 35, or the complement thereof,<br>
(e)	nucleic acids as defined in any of (a) to (d) characterized in that said nucleic acid is DNA, genomic DNA, cDNA, synthetic DNA or RNA wherein T is replaced by U,<br>
(f)	a nucleic acid which is degenerated to a nucleic acid as given in any of<br>
SEQ ID Nos. 27,1,3,5,7,9,11,25,26,28 to 31,33 or 34, or which is<br>
degenerated to a nucleic acid as defined in any of (a) to (e) as a result of<br>
the genetic code,<br>
(c)	a nucleic acid specifically hybridizing to a nucleic acid as given in any<br>
of SEQ ID Nos. 29, 3, 5, 9, 26, 27, 31,33 or 34, or the complement<br>
thereof,<br>
(d)	a nucleic acid encoding a protein with an amino acid sequence<br>
comprising the polypeptide as given in SEQ ID NO 32 and which is at<br>
least 70% similar to the amino acid sequence as given in SEQ ID NO 4,<br>
(e)	a nucleic acid encoding a protein with an amino acid sequence which is at least 47% similar to the amino acid sequence as given in SEQ ID NO 6,<br>
(f)	a nucleic acid encoding a protein with an amino acid sequence which is at least 47% similar to the amino acid sequence as given in SEQ ID NO 10 or 35,<br>
(g)	a nucleic acid encoding a protein comprising the amino acid sequence as given in any of SEQ ID Nos. 4,6,10,32 or 35,<br>
(h) a nucleic acid which is degenerated to a nucleic acid as given in any of SEQ ID Nos. 29, 3, 5, 9, 26, 27, 33 or 34 or which is degenerated to a nucleic acid as defined in any of (a) to (g) as a result of the genetic code,<br>
(i) a nucleic acid which is diverging from a nucleic acid encoding a protein as given in any of SEQ ID Nos. 4,6, 10 or 35 or which is diverging from a nucleic acid as defined in any of (a) to (g) due to the differences in codon usage between the organisms;<br>
j) a nucleic acid encoding a protein as given in SEQ ID Nos 4, 6, 10 or 35, or a nucleic acid as defined in (a) to (g) which is diverging due to the differences between alleles,<br>
(k) a nucleic acid encoding an immunologically active fragment of a cytokinin oxidase encoded by a nucleic acid as given in any of SEQ ID Nos. 29, 3, 5, 9, 26, 27, 31, 33 or 34, or an immunologically active fragment of a nucleic acid as defined in any of (a) to (j),<br>
(1) a nucleic acid encoding a functional fragment of a cytokinin oxidase encoded by a nucleic acid as given in any of SEQ ID Nos. 29,3,5, 9, 26,27,31, 33 or 34, or a functional fragment of a nucleic acid as defined in any of (a) to (j), wherein said fragment has the biological activity of a cytokinin oxidase, and<br>
(m) a nucleic acid encoding a protein as defined in SEQ ID NO 4, 6, 10 or<br>
35,<br>
provided that said nucleic acid is not the nucleic acid as deposited under<br>
any of the following Genbank accession numbers: AC005917, AB024035,<br>
and AC023754.<br>
3.	An isolated nucleic acid as claimed in claim 2 which is DNA, cDNA, genomic DNA or synthetic DNA, or RNA wherein T is replaced by U.<br>
4.	A nucleic acid molecule of at least 15 nucleotides in length hybridizing specifically with a nucleic acid as claimed in claims 2 or 3.<br>
5.	A nucleic acid molecule of at least 15 nucleotides in length specifically amplifying a nucleic acid as claimed in claims 2 or 3.<br>
6.	A vector comprising a nucleic acid as claimed in claims 2 or 3.<br>
7.	A vector as claimed in claim 6 which is an expression vector wherein the  nucleic acid is  operably linked  to  one  or more  control<br>
sequences allowing the expression of said nucleic acid in prokaryotic and/or eukaryotic host cells.<br>
8.	A bacterial cell or fungal cell containing a nucleic acid as claimed in claims 2 or 3 or a vector as claimed in claims 6 or 7.<br>
9.	Diagnostic  kit  comprising  at least  a  nucleic acid  molecule  as claimed in any of claims 2 to 5, the vector of claims 6 or 7.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDEyNDMtZGVsLWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-01243-del-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDEyNDMtZGVsLWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-01243-del-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDEyNDMtZGVsLWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-01243-del-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDEyNDMtZGVsLWNvcnJlc3BvbmRlbmNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-01243-del-correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDEyNDMtZGVsLWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-01243-del-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDEyNDMtZGVsLWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-01243-del-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDEyNDMtREVMLUZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-01243-DEL-Form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDEyNDMtZGVsLWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-01243-del-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDEyNDMtREVMLUZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-01243-DEL-Form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDEyNDMtZGVsLWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-01243-del-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDEyNDMtZGVsLWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-01243-del-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDEyNDMtZGVsLWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-01243-del-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDEyNDMtZGVsLXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-01243-del-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDEyNDMtZGVsLXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-01243-del-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDEyNDMtZGVsLXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-01243-del-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDEyNDMtZGVsLXBjdC00MDgucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-01243-del-pct-408.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDEyNDMtZGVsLXBjdC00MDkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-01243-del-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDEyNDMtZGVsLXBldGl0aW9uLTEzNy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-01243-del-petition-137.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDEyNDMtZGVsLXBldGl0aW9uLTEzOC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-01243-del-petition-138.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMTI0My1ERUwtQ2xhaW1zLSgxMi0wNC0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-1243-DEL-Claims-(12-04-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMTI0My1ERUwtQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgxMi0wNC0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-1243-DEL-Correspondence Others-(12-04-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMTI0My1ERUwtR1BBLSgxMi0wNC0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-1243-DEL-GPA-(12-04-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMTI0My1ERUwtUGV0aXRpb24gMTM3LSgxMy0xMC0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-1243-DEL-Petition 137-(13-10-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMTI0My1ERUwtUGV0aXRpb24gT3RoZXJzLSgxMi0wNC0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-1243-DEL-Petition Others-(12-04-2012).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="233161-3-2z-1-3-4-dimethylphenyl-1-5-dihydro-3-methyl-5-oxo-4h-pyrazol-4-ylidene-hydrazino-2-hydroxy-1-1-biphenyl-3-carboxylic-acid-bis-monoethanolamine.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="233163-method-for-transmitting-a-digital-message-and-system-for-carrying-out-said-method.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>233162</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2002/01243/DEL</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>16-Dec-2002</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>THOMAS SCHMULLING</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>GERTRUD-BAUMER-STRASSE 8, 72074 TUBINGEN, GERMANY.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>THOMAS SCHMULLING</td>
											<td>GERTRUD-BAUMER-STRASSE 8, 72074 TUBINGEN, GERMANY</td>
										</tr>
										<tr>
											<td>2</td>
											<td>TOMAS WERNER</td>
											<td>HALDENSTRASSE 2, 72074 TUBINGEN, GERMANY</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/82</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EPO1/06833</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-06-18</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>00870132.8</td>
									<td>2000-06-16</td>
								    <td>EPO</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/258.415</td>
									<td>2000-12-27</td>
								    <td>EPO</td>
								</tr>
								<tr>
									<td>3</td>
									<td>01870053.4</td>
									<td>2001-03-16</td>
								    <td>EPO</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/233162-method-for-stimulating-root-growth-comprising-overexpressing-cytokinin-oxidase by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:33:24 GMT -->
</html>
