<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/233266-a-method-of-quality-control-using-an-internal-control-nucleic-acid-sequence-encasulated-within-liposome-particles by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:36:51 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 233266:&quot;A METHOD OF QUALITY CONTROL USING AN INTERNAL CONTROL NUCLEIC ACID SEQUENCE ENCASULATED WITHIN LIPOSOME PARTICLES&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A METHOD OF QUALITY CONTROL USING AN INTERNAL CONTROL NUCLEIC ACID SEQUENCE ENCASULATED WITHIN LIPOSOME PARTICLES&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to the use of non-viable particles (and in particular liposome, particles, particles which are in the form of a viral protein coat, nonÂ¬viable genetically modified organisms or particles made of synthetic polymers), comprising an internal control (IC) nucleic acid sequence as an internal control in nucleic acid-based analysis. The present invention further relates to non-viable particles comprising an IC nucleic acid and kits for carrying out the methods and uses of the invention.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The present invention relates to a method of quality control using an internal control nucleic acid sequence encasulated within liposome particles. More particularly, it relates to the use of non-viable particles containing an internal control nucleic acid sequence in nucleic acid-based analysis. The present invention further relates to methods of nucleic acid-based analysis using these non-viable particles and kits for carrying out said methods.<br>
The use of nucleic acid-based analysis has become extremely widespread during the last few years, particularly in the field of diagnostic testing. For example, such analysis has been used in the diagnosis of microbiological pathogens and genetic disorders and has also contributed to the discovery of unknown infectious agents and improved diagnostic tools. Such nucleic acid-based analysis can be either qualitative or quantitative and may or may not involve nucleic acid-based amplification techniques. The nucleic acid-based amplification assays, for example the polymerase chain reaction (PCR), ligase chain reaction (LCR), gap-filling LCR (Gap-LCR), nucleic acid sequence based amplification (NASBA) and transcription mediated amplification (TMA) have the advantage over more traditional micobiological tests of being very highly sensitive and specific. The major disadvantage of such assays and indeed other nucleic acid based analysis is however the obtaining of false positive and false negative results, which may in turn result in an erroneous interpretation of the data and inadequate or incorrect treatment of patients. In a worst case scenario, a false negative result might even result in a situation where a patient would be given the "all clear" for a particular disease and not be treated at all. The elimination or reduction of such false negative results is clearly desirable.<br>
False positive results can be caused by     contamination   between   different    sample    or   bycontamination of the sample with previously made amplification products, also called "carry-over" contamination. Much research and development has been undertaken to find ways to reduce the risk for "carryover" contamination.  Some commercial tests have incorporated quality assurance measures which use chemical procedures to reduce the "carry-over" contamination.  The risk of obtaining false positive results is thereby virtually eliminated.<br>
False negative results can be caused for example by the presence of inhibitory substances as impurities in the nucleic acid preparations.  Many biological sample materials such as blood, saliva, urine and faeces contain such inhibitory substances that might interfere with the enzymes used in for example, amplification reactions causing partial or complete reduction of the enzyme activities or may cause the digestion or degradation of the nucleic acids to be tested.  False negative results can also be due to erroneous execution of the assay procedure.<br>
A current solution to try and minimise the number of false negative results which is used in both "in house" assays as well as in commercially available kits, involves the use of internal control (1C) nucleic acid sequences, or to which for example probes can bind in order to facilitate detection.  Such 1C sequences are designed for the particular assay in question and in general are nucleic acid sequences which contain regions to which particular primers can bind and initiate amplification of the 1C sequence or to which for example probes or other entities can bind in order to facilitate detection.  Such 1C sequences can be so called "ideal", "pseudo-ideal" or "non-ideal" 1C sequences.  An "ideal" 1C sequence generally comprises a binding region, for example primer or probe binding regions which are substantially identical to the equivalent regions to which the primers or probes bind in the target nucleic<br><br>
acid to be detected.  In addition, such "ideal" 1C sequences generally have an identical sequence and are thus the same length as the target sequence.  Thus, in this case the same primers or amplification probes can be used to amplify both the 1C sequence and the target sequence and the efficiency of amplification should be the same in both cases.  Alternatively, the use of probes (e.g. labelled probes) or other entities which bind to common sequences present in the 1C and the target sequences similarly may provide quantitative information.  However, for many applications "non-ideal" 1C sequences can be used (see Cleland et al., Vox Sanguinis, Vol. 76: 170-174, 1999).  Such "non-ideal" 1C sequences generally comprise binding regions, for example primer or probe binding regions which do not bind to the primers which are used to amplify the target nucleic acid or probes which bind to such target nucleic acids.  Assays involving such non-ideal"1C sequences thus generally involve the use of a set of primers (or amplification probes) which can amplify the target sequence, together with a further set of primers (or amplification probes) which can amplify the 1C nucleic acid.  Alternatively such non-ideal 1C sequences can contain a binding region, e.g. a probe or other entity binding region which is unique to the 1C nucleic acid and not found in the target sequences.<br>
In the case of nucleic acid-based assays which involve amplification, an amplification of the 1C sequence in the absence of amplification of the target sequence will then be evidence of a correct negative result and the amplification of both sequences a correct positive result. In nucleic acid-based assays where amplification is not involved, detection may occur by binding a probe, and in which case the binding of a probe to the 1C sequence in the absence of the binding of an appropriate probe to the target sequence will be evidence of a correct negative result and the binding of<br><br>
the respective probes to both the 1C nucleic acid and the target sequence a correct or positive result.<br>
Such 1C sequences can have a variety of forms, e.g. they can be DNA molecules in the form of plasmids (Rosenstraus et al., J. Clin. Microbiol. 1998. vol. 36: 191-197) or RNA molecules (see W093/23573 of New England Deaconess Hospital) .  In addition, in cases where the assay has been designed for the detection of a microorganism, a genetically modified organism has been used as an internal control (Kolk et al./ J. Clin. Microbiology, 1994, vol. 32, 1354-1356).<br>
An important feature of any 1C sequence for use in nucleic acid-based assays is that it can be distinguished from the target sequence in the subsequent analysis or detection of the nucleic acid molecules produced.  Methods of distinguishing 1C sequences over target sequences often involve the design of an 1C sequence so that it is a different size to the target sequences.  Alternatively, the 1C sequence can be engineered so that it contains a unique "probe binding region" that differentiates the 1C from the target nucleic acid.  In this way 1C sequences (which may or may not have been amplified, depending on the assay in which they have been used) can be separated or distinguished from target sequences using an oligonucleotide to which the probe will interact.  Such methods can be used in conjunction with "non-ideal" 1C sequences as discussed above.  However, the introduction of such distinguishing features into an "ideal" 1C nucleic acid, where the 1C has exactly the same sequence as the target sequence (i.e. the same primer or probe binding sites and the same sequence between the primer binding sites) would mean that strictly speaking such a modified 1C sequence could no longer be regarded as an "ideal" 1C sequence.  In other words when the sequence of an "ideal" 1C is modified, even if only by one base pair, such an 1C is arguably no longer an "ideal" 1C.<br><br>
Such 1C nucleic acid sequences which can still bind to the same primers or probes as the target nucleic acid but contain modifications so that they can be distinguished from the target nucleic acid by methods such as those outlined above are referred to herein as "pseudo-ideal" 1C sequences.<br>
Advantageously, it has now been found that 1C nucleic acids for use in nucleic acid-based assays can be encapsulated and contained in non-viable particles. Examples of non-viable particles which can be used in this way are liposomes, protein coats and non-viable genetically modified organisms.<br>
The use of 1C nucleic acids contained or encapsulated within non-viable particles in this way and in particular those contained within liposomes or protein coats have advantages over the use of viable genetically modified organisms, which have been used previously as internal controls.  For example, such liposomes or protein coats containing 1C nucleic acids are non-expensive to design and adapt to any nucleic acid amplification/detection system or any other non-amplification based nucleic acid assay and it is less labourious (and therefore less expensive) to make different kinds of liposomes or protein coats with regard to sequence and particle properties.  In addition and perhaps most importantly, the non-viable particles in the form of liposomes, protein coats or non-viable genetically modified organisms are biologically safe, politically non-controversial, and contain no potential endogenous or exogenous hazardous sequences (e.g. antibiotic resistance genes).<br>
The use of 1C nucleic acids contained or encapsulated within such non-viable particles also has advantages over the use of naked DNA and RNA types of 1C sequences discussed above.  In this regard, it is important to realise that the idea behind an 1C sequence is that it follows as precisely as possible as many as<br><br>
possible of the series of treatment steps that the target sequence undergoes.  Most amplification based assays will comprise all or some of the steps of transportation of the sample to the analysis laboratory, storage prior to sample preparation, sample preparation (involving e.g. centrifugation or sedimentation), release and purification of nucleic acids, enzymatic amplification and detection of any amplification products.  (Similarly non-amplification based assays will comprise all or some of those steps, with the exception of the amplification step).  In the present "in house" and commercial assays that include an internal control, the internal control nucleic acid is added to the sample at the nucleic acid release/purification step or just before the amplification stage (see for example Rosenstraus et al. 1998, supra., which describes the technique behind the Roche commercial PCR assay) .  This has the effect that the steps which precede the addition of the 1C have no quality assurance to ensure for example proper transportation and storage of the sample, efficient sample preparation (e.g. efficient centrifugation or sedimentation) and in some cases efficient release of nucleic acid.  This is a serious drawback.<br>
However, by using an 1C nucleic acid which has been encapsulated in a non-viable particle in accordance with the present invention, the 1C nucleic acid can preferentially be added to the samples at a very early time point in the process, i.e. such particles can even be added to the urine, blood etc. directly after the sample has been derived from the patient (i.e. at the collection step) and before transportation and/or subsequent processing to release nucleic acid takes place.  (Of course the stage at which the particular type of non-viable particle is added to the sample will necessarily depend on the intention of the use of the internal control, e.g. as reference standard in<br><br>
quantitative analysis, as a quality control for the lysis of the cells in the sample, as a quality control for the detection step, etc). A conventional 1C which was in the form of DNA or RNA which was not encapsulated would not necessarily be added at such an early time point as there is a significant risk that the samples will contain enzymes or other impurities (e.g. RNAase or DNAase enzymes) which will degrade the 1C DNA/RNA while not degrading the target nucleic acid because it is not exposed to these elements (it is still within the cells present in the collected sample).  In such a case, the 1C sequence would fail to work as a quality control. The non-viable particles on the other hand would protect the 1C nucleic acid at this early stage (like a cell membrane) with the 1C nucleic acid only being released when the target nucleic acid was also released.  In addition, it is not uncommon that the sample processing involves a centrifugation step or other steps for whole-cell-isolation/purification, meaning that any "naked" nucleic acid which had previously been added to the sample, would be discarded with the supernatant.  This is another drawback of the use of a non-encapsulated 1C.<br>
Thus, it can be seen that a non-viable particle encapsulated 1C nucleic acid shows much improved properties over the prior art 1C sequences and will, if required, allow the quality control of each step of the assay procedure.<br>
Thus viewed from one aspect the present invention provides the use of non-viable particles comprising an internal control nucleic acid sequence as an internal control in nucleic acid-based analysis.<br>
Viewed from another aspect the invention provides a method of nucleic acid-based analysis comprising the step of bringing the sample being analysed into contact with non-viable particles comprising an internal control (1C) nucleic acid sequence.<br>
Generally and preferably the 1C nucleic acid<br><br>
sequences are encapsulated within the non-viable particles.<br>
"Encapsulated" or "encapsulation" as used herein in relation to the relationship between the 1C nucleic acid and the non-viable particles refers to situations where all or part of the 1C nucleic acid molecule is located/ entrapped within the central core of the particle. Thus, the 1C nucleic acids may be located entirely within the central core or pool of the particle, i.e. no part of said 1C nucleic acid molecule is present at or on the external surface of said particle. Alternatively, all or part of the 1C nucleic acid may be embedded or entrapped within or otherwise bound to the inner or internal surface of the particle.  It is preferred that the 1C nucleic acid (or at least a significant proportion of the 1C nucleic acid) be encapsulated within the particles as described above as any nucleic acid located on the external surface of the particle will be at risk of degradation, e.g. by nuclease enzymes or contamination by inhibitory impurities which may be present in the sample.  Thus, the terms "encapsulation" or "encapsulate" as used herein include any means or interaction between the non-viable particle and the 1C nucleic acid, by which the non-viable particle protects the 1C nucleic acid from the external environment surrounding the non-viable particle.<br>
"Nucleic acid-based analysis or assays" as used herein refers to any analysis technique or assay, or one or more steps thereof, which is based on the quantitative or qualitative detection of nucleic acids. The main proviso is that the analysis technique has to be one which allows the use of an internal control sequence (in ELISA assays for example, the use of internal control sequences is not possible).  Preferred nucleic acid based analysis techniques will be those which involve amplification of the nucleic acid in<br><br>
question (i.e. the target nucleic acid), e.g PCR, LCR, Gap-LCR, NASBA and TMA.  In such assays it is also preferred that the 1C nucleic acid be amplified alongside the target nucleic acid.  There may be occasions however where the target nucleic acid is amplified in the nucleic acid based assay but the 1C nucleic acid is not amplified (if for example a sufficient number of 1C nucleic acid sequences are encapsulated within the non-viable particles such that amplification of said 1C sequences is not necessary to obtain sufficient 1C nucleic acid sequences to be detected, and/or for example the 1C nucleic acid comprises PNA (peptide nucleic acid) which cannot be amplified) .  It will be appreciated however that in such cases the 1C nucleic acid would act as a control for steps other than the amplification steps, i.e act as a control for all steps of the procedure in which it had been taken through the same process steps as the target sequences, e.g the detection steps.<br>
It is important to note however that the nucleic acid-based analysis or assays discussed herein may not involve amplification of the target nucleic acid and/or 1C nucleic acid.  For example the assay may involve the binding of a probe or some other reagent to the target nucleic acid after which the binding of this entity can be detected.  The branched DNA (bDNA) technique of Chiron might be mentioned in this regard (Urdea et al., 1991, Nucleic Acid Symposium Series, vol 24:197-200). This involves the hybridisation of a series of probes to a target nucleic acid.  Each of these probes have a branched DNA attached which have signal moieties attached and which can give rise to an amplified signal. Thus, the probes can be detected in order to detect the target nucleic acid. Again in such assays it is preferred that the 1C nucleic acid is also not amplified and is assayed in the same way as the target nucleic acid, e.g. by the binding of an entity to the 1C nucleic<br><br>
acid and the subsequent detection thereof.  If probes are used for example, then the 1C nucleic acid may be designed to bind the same or different probe to the target nucleic acid.  Either way the 1C nucleic acid can still be used as an internal control.<br>
"Internal control nucleic acid sequence" as used herein refers to any nucleic acid sequence which can function as an internal control in a nucleic acid based analysis.  The 1C nucleic acid can be any type of nucleic acid.  For example it may be single stranded or double stranded DNA in a linear or circular form, for example in the form of a circular plasmid or a double stranded or single stranded oligonucleotide or PCR product.  Alternatively, it may be RNA (for example sense or antisense RNA molecules or double stranded RNA molecules) or DNA/RNA hybrids.  Alternatively, the 1C nucleic acid may be PNA or a mixture or hybrid of PNA with other types of nucleic acid molecules.  In some assays a mixture of different types of 1C nucleic acid may be used e.g. a PNA 1C may be used in addition to an RNA or DNA 1C.<br>
As is discussed further herein, depending on the type of nucleic acid-based assay being carried out, the 1C nucleic acids comprise primer or probe binding sites or regions, or other sites or regions which can interact with appropriate assay reagents, such as capture probe hybridisation sites or probe detection sequences. Alternatively 1C nucleic acids may carry or contain other distinctive information, e.g. have a particular length or detectable composition.<br>
As outlined above the 1C nucleic acid can be a "pseudo-ideal" 1C nucleic acid sequence which may be amplified in essentially the same way and at the same rate as the target nucleic acid in the nucleic acid-based assay (or for nucleic acid based assays which do not involve amplification said pseudo-ideal 1C nucleic acid is assayed or treated in essentially the same way<br><br>
as the target nucleic acid, e.g. is reacted and can bind to the same probe or entity as the target nucleic acid or the probes may be used after the assay for identification purposes only) , but can be distinguished from said target nucleic acid by appropriate techniques. In this embodiment of the invention therefore, said 1C nucleic acid can be amplified or e.g. probed using the same primers as are used to amplify the target nucleic acid or the same probes which are used to assay the target nucleic acid.  In other words such 1C nucleic acids comprise binding regions, e.g. primer or probe binding regions, or other sites or regions which can interact with appropriate assay reagents, such as capture probe hybridisation sites or probe detection sequences, or carry other distinctive information, e.g. have a particular length or detectable composition, which are capable of binding to the same primers or amplification probes as are used to amplify the target nucleic acid or are capable of binding to the same probes or entities which are used to assay the target nucleic acid, or have the same assessable informational content as the target nucleic acid.  It will be appreciated that although such binding regions, e.g. primer or probe binding regions can be identical in sequence to the equivalent primer or probe binding regions in the target nucleic acid, complete identity is not absolutely necessary and what is required is that the binding regions, e.g. the primer or probe binding regions are substantially identical or sufficiently similar to the primer or probe binding regions of the target nucleic acid so that the primers or probes which bind to the primer or probe binding regions of the target nucleic acid can also bind to the primer or probe binding regions of the 1C nucleic acid and function as primers, e.g. in extension reactions or, in the case of probes or other binding agents, function as entities in the nucleic acid based assay which can subsequently be<br><br>
detected.  "Pseudo-ideal" 1C nucleic acids with such binding regions, e.g. primer or probe binding regions are sometimes referred to herein as "near-ideal" 1C nucleic acids.<br>
In an alternative embodiment of the invention the 1C nucleic acid can be a "non-ideal" 1C nucleic acid. Such "non-ideal" 1C nucleic acids generally comprise binding regions e.g. primer or probe binding regions, or other sites or regions which can interact with appropriate assay reagents, such as capture probe hybridisation sites or probe detection sequences, or carry other distinctive information, e.g. have a particular length or detectable composition, which are distinct/different from the primer or probe binding regions (or informational content) in the target nucleic acid and which cannot bind to the primers or probes (or do not have the same informational content) which are used to amplify or otherwise assay (e.g. by probe binding or assessing informational content) the target nucleic acid.  Thus, in assays involving a non-ideal 1C nucleic acid a different set of primers or probes or other entities is used to amplify or otherwise assay (e.g. by probe or other entity binding or recognition or assessment of informational content) the 1C nucleic acid to the set of primers or probes which is used to amplify or otherwise assay the target nucleic acid.<br>
Thus it can be seen that for the embodiments of the invention where a "pseudo-ideal" or "near-ideal" 1C nucleic acid is used, where the nucleic acid-based assay involves amplification, the same set of primers (or amplification probes in the case of LCR) may be used to amplify both the target and the 1C nucleic acid.  In a PCR based amplification approach this set of primers will comprise a standard 2 primer PCR system (one primer designed to interact with the coding strand and the other with the complementary strand in the standard way).  In a ligase chain reaction (LCR) based<br><br>
amplification system (e.g. LCR or Gap-LCR assays) amplification involves the use of 4 oligonucleotide amplification probes, 2 of which hybridise to the coding strand forming a nick between them and 2 of which hybridise to the complementary strand forming a nick between them.  These two nicks are sealed by DNA ligase. In Gap LCR the pairs of amplification probes hybridise so that they are almost adjacent so that a DNA polymerase has to incorporate a few (1-3) nucleotides to fill the gap.  Thereafter the ligase can seal the nick. This gives Gap-LCR a lower background since blunt end ligation of probes hybridised to its complementary probe can not occur.  Thus, in the cases where "pseudo-ideal" or "near-ideal" 1C nucleic acids are used, both the 1C nucleic acid and the target nucleic acid will be amplified using the same four LCR amplification probes. On the other hand, in the alternative embodiments of the invention where a "non-ideal" 1C nucleic acid is used, in a PCR based amplification 4 primers will be used, 2 of which will interact with the coding and non-coding strands of the target nucleic acid to allow amplification thereof and 2 of which interact with the two complementary strands of the 1C nucleic acid to allow amplification thereof. Non-ideal 1C nucleic acids can also be used in amplification based assays based on the ligase chain reaction.  Here four amplification probes will be used to amplify the target nucleic acid and two additional amplification probes will be used to amplify the 1C nucleic acid.  The two products will then be identical on one side of the nick position and different on the other side.  If the 1C is completely different from the wild type target, the reaction would need eight amplification probes, four amplification probes to amplify the target nucleic acid and four additional amplification probes to amplify the 1C nucleic acid.  Further information with regard to the design of appropriate 1C nucleic acids for use in LCR<br><br>
based amplification systems (and in particular Gap-LCR) .can be found in W097/04128, the teaching of which is incorporated herein by reference.<br>
As discussed further below an encapsulated, entrapped or embedded 1C nucleic acid according to the present invention may also be used in the nucleic acid-based amplification reactions NASBA and TM&amp;.  These are both assays based on the use of two primers and thus, as for the standard PCR system as described above, where a pseudo-ideal 1C nucleic acid is used only two primers are required, whereas for assays where a non-ideal 1C nucleic acid is used four primers are required.<br>
In general, for nucleic-based assays where quantitative results are required, it is necessary to use a "pseudo-ideal" or "near-ideal" rather than a "non-ideal" 1C nucleic acid.  On the other hand in assays where only qualitative results are required either a "pseudo-ideal", "near-ideal" or "non-ideal" 1C nucleic acid may be used.<br>
Where the nucleic acid based assay involves amplification and the 1C nucleic acid is designed to be amplified, the primers or amplification probes for inducing the amplification of the 1C nucleic acid may optionally be encapsulated inside the non-viable particles with the 1C nucleic acid.  In such embodiments, amplification of the 1C nucleic acid would of course not occur in the non-viable particles and would only occur once the non-viable particle was lysed and the 1C nucleic acid and primers released into an appropriate environment for inducing amplification. Similarly in non-amplification based assays, the probes or other entities which are used to assay the 1C nucleic acid may be encapsulated inside the non-viable particles with the 1C nucleic acid.<br>
A further important feature of the 1C nucleic acid is that it can be distinguished from the target nucleic acid by any suitable method. Depending on the nucleic<br><br>
acid-based assay for which the 1C nucleic acid is acting as a control, the 1C nucleic acid may or may not have been amplified before said distinguishing takes place. Thus, if the nucleic acid based assay involves amplification then detection may take place straight away by any suitable technique (e.g. using a specific probe or on the basis of size, see below), or a further amplification of the 1C nucleic acid and/or the target nucleic acid may take place before detection. Similarly, in nucleic acid-based assays which do not involve amplification, detection may take place directly or after an amplification step.<br>
Methods of distinguishing 1C nucleic acids from the target nucleic acid are well known and documented in the art (see for example the article by Zimmerman et al., Biotechniques, 1996 vol. 21: 268-279) and are discussed briefly above.  A common method involves the distinguishing of the target and 1C nucleic acid sequences on the basis of size.  For example, the 1C sequence might be designed so that it corresponds to the target sequence but contains a region of deletion or insertion located somewhere in the molecule, for example in between the primer binding regions which are used to amplify the sequence.  This has the effect of allowing the target and the 1C sequence to be amplified using the same primers but then enables the two species to be separated on the basis of their different sizes using for example any one of a variety of known chromatographic methods such as agarose gel electrophoresis, acrylamide gel electrophoresis, chroraatographic size separation etc.<br>
Distinguishing on the basis of size can also be used in assays involving a "non-ideal" 1C nucleic acid. In such a case the 1C nucleic acid is simply designed so that it is a different size to the target nucleic acid thus enabling the two to be distinguished by appropriate techniques.  Alternative techniques which could be used<br><br>
to distinguish the target nucleic acid from the 1C nucleic acid are well known and documented in the art and include the use of selective probe hybridisation or the use of primers, one or more of which have different labels incorporated therein and which thereby result in the 1C nucleic acid and target nucleic acid (or amplification products thereof) being labelled differently.<br>
An 1C nucleic acid for use in the present invention can thus be designed for any assay and will generally be designed based on the particular target nucleic acid that is to be analysed and the nature of the nucleic acid-based assay to be used.  The 1C nucleic acid may (in the case of nucleic acid assays which involve amplification) be designed to contain sequences that can interact with the primers/LCR amplification probes in the amplification reaction mixture, often preferably with the same prirners/amplification probes as the corresponding sequences in the target nucleic acid and may be further designed to contain a distinguishing feature such as an insertion or deletion region or a marker which will interact with a specific probe. Alternatively, the 1C nucleic acid may be designed to contain sequences that can react with other entities which are used in the non-amplification based assay concerned, e.g. with probes, often preferably with the same probes/entities that bind to the corresponding sequences in the target nucleic acid. Again such 1C nucleic acids may be further designed to contain a distinguishing feature (as described above).<br>
Although in the case of a "pseudo-ideal" or "near ideal" 1C nucleic acid it will be necessary to design the 1C nucleic acid based on the sequence of the particular target nucleic acid that is to be analysed in order to ensure that the same binding reagents, e.g. primers or LCR amplification probes (in the case of an assay which involves amplification) or other probes or<br><br>
binding entities (in the case of assays which do not involve amplification) can be used to amplify or otherwise assay both the target and the 1C nucleic acid, such "assay specific" 1C nucleic acids are generally not required for the assays wherein a "non-ideal" 1C nucleic acid is used.  Thus, once designed a "non-ideal" 1C nucleic acid could be used in any number of nucleic acid based assays for any number of different target nucleic acids provided the appropriate 1C nucleic acid specific primer or probe set was introduced into the assay at the appropriate time and providing that the 1C nucleic acid could be distinguished from the target nucleic acid. Such "non-ideal" 1C nucleic acids are thus more general reagents.<br>
Once designed, the construction of such 1C nucleic acids can be carried out using techniques which are standard or conventional in the art, for example standard genetic engineering techniques (see the discussion in Zimmerman et al., supra) .  Furthermore, many examples of 1C nucleic acids with such an appropriate design are well known and documented in the art and any of these 1C nucleic acids may be used in the present invention.  The document by Zimmerman et al., supra gives some examples of 1C nucleic acids. Examples of "pseudo-ideal" IC's designed to be longer than the wild type target nucleic acid can be found in Ursi et al. (1992), APMIS 100(7): 635-9; Siebert and Larrick (1993), Biotechniques, 14(2): 244-9; Rosenstraus et al. (1998), Journal of Clinical Microbiology, 36(1): 191-7; Bretagne et al. (1993), Journal of Infectious Diseases, 168(6), 1585-8; Gilliland et al. (1990), PNAS USA, 87(7): 2725-9; Wang et al. (1989), PNAS USA, 86(24): 9717-21.  Examples of "pseudo-ideal" ICs designed to be shorter than the wild type target can be found in Galea and Feinstein (1992) , PCR Methods and applications, 2:66-69.  Examples of "non-ideal" ICs can be found in Cleland et al. (1999) Vox Sanguinis 76(3):170-4;<br><br>
Rosenstraus et al. (1998), supra; WO 93/02215 and WO 97/04128.  Ideally, in order for the 1C nucleic acid to display similar kinetics, e.g. amplification kinetics, to the target nucleic acid, the 1C nucleic acid is designed to be approximately the same length as the target nucleic acid.  Thus, the length of the 1C will generally be dependent on the length of the target nucleic acid.  Preferably ICs are isolated nucleic acid molecules of less than 1000 bases.  More preferably said ICs are less than 600 bases in length but greater than 40 bases in length, e.g. 50 to 500 bases or 100 to 500 bases in length.<br>
The term "non-viable particle" as used herein refers to any entity which is capable of encapsulating, entrapping or embedding an internal control nucleic acid but which is not capable of propagation either alone (i.e. by self propagation) or by culture in a biological system which would normally allow the propagation of the entity in question.  Such particles may never have been capable of being propagated, e.g. liposomes, protein particles, or synthetic particles or other particles which consist solely of an encapsulating shell and do not contain any genetic material which enables replication and propagation of the particle in a biological system, for example particles which are made up of viral coat proteins or viral capsid proteins. Other non-viable particles included herein are those which were capable of being propagated, e.g. virus particles or other pathogenic organisms, but which have been altered in such a way that replication and/or propagation of the particles are no longer possible. For example such particles could include genetically modified organisms (GMOs) which have been further modified so that they are no longer viable in terms of propagation and/or replication, described herein as "dead" or non-viable GMOs.<br>
Such particles may thus be made of any material<br><br>
which is capable of encapsulating, entrapping or embedding a nucleic acid.  Such material may for example include lipids or modified lipids (e.g. as part of a liposome or liposome type particle) or may include proteins (e.g. in the form of a protein coat such as the protein coat or "capsid" of a virus) or a combination of lipid and protein (e.g. where proteins are embedded in a lipid vesicle in a way which will mimic the normal protein embedded lipid bilayer of a cell) .  Such particles may also be made of synthetic material, e.g. a synthetic polymer.  Exemplary synthetic materials/polymers for use in the encapsulation of 1C nucleic acids in accordance with the present invention include cationic polymers such as those described in Wolfert et al., 1999, Bioconjugate Chemistry, 10: 993-1004, which entrap nucleic acids in a cationic polymer-nucleic acid complex and polylysine based complexes such as those described in Wagner et al., 1992, PNAS USA, 89: 7934-7938.<br>
Another important feature of the "non-viable particles" which encapsulate, entrap or embed the 1C nucleic acid is that the structure of the particles will lyse, collapse, leak, i.e. generally be disrupted, under the same conditions which will "disrupt" the target entity e.g. cells or viruses which contain the target nucleic acid which is being analysed.  Whilst preferably the target nucleic acid is contained within cells or viruses and subsequent discussion refers to such target cells or viruses, the methods of the invention may be used in assays in which the target nucleic acid is contained within an entity other than a cell or virus, e.g. a non-naturally occurring particulate structure, such as a liposome.  Target "cells" include cells derived from multicellular organisms or unicellular organisms (e.g. yeast, protozoa and bacteria).  Target "viruses" include any viruses, including bacteriophage. The term "disrupt" as used herein thus includes any<br><br>
disruption (e.g. lysis etc. as mentioned above) which will result in release of the contents of the non-viable particle/cell/virus, i.e. the release of at least the 1C molecule (from the non-viable particle) and the target nucleic acid (from the target entity e.g. target cell or virus).  Such conditions generally involve the exposure of the target cells or viruses to appropriate lysis reagents which often contain detergents, and/or reagents such as strong acids, strong bases or organic compounds such as phenol or guanidine thiocyanate.  Preferably such particles will also be of the same or similar stability under assay and sample conditions (such as temperature, salt concentration, etc.) as the target cell or virus.  The ability of the non-viable encapsulating, entrapping or embedding particle to "mimic" as far as possible the target cell is a central theme of the present invention and generally any modification which can be made to the non-viable particles in order that they more closely resemble the target cells or viruses which contain, e.g. encapsulates the target nucleic acid is an advantage and such modified particles are included in the above definition.<br>
For this reason, for some applications particularly preferred particles are liposome particles or protein particles in which all or some of the lipids or proteins have been modified (e.g. so that they do not correspond to naturally occurring or native proteins or lipids) so that they exhibit an advantageous property such as increased stability.  Known modifications which can be used to increase the stability of liposomes for example are discussed further below.<br>
Additionally or alternatively the non-viable particles can be designed to include proteins which are naturally found in the membrane of the target cells. The inclusion of such proteins will enable the particles to more closely mimic the target cell but may also be used in order to target the delivery of a liposome.<br><br>
Preferred non-viable particles for use in the present invention are liposome particles, particles which are in the form of a viral protein coat, "dead"/non-viable GMOs or particles made of synthetic polymers.  More preferred non-viable particles are liposome particles, "dead"/non-viable GMOs or particles made of synthetic polymers.  Other preferred non-viable particles are those comprising modified proteins or lipids as described above. Most preferred non-viable particles are liposome particles.<br>
As mentioned above it is important that the non-viable particles used for a particular assay mimic as far as possible the target cells or viruses which contain the target: nucleic acid.  Thus, the appropriate encapsulation (or entrapment, embedding) vehicle, e.g. a liposome, a viral protein coat, a "dead" GMO or a synthetic particle as described above, for a particular assay will be selected depending on the nature and characteristics of the target cell or virus and also the conditions under which the target cells or viruses will lyse.  The appropriate encapsulation, entrapment or embedding vehicle will be one which will allow the encapsulated, entrapped or embedded 1C nucleic acid to undergo the same treatment steps as the target cell or virus and target nucleic acid under assay.  Thus the appropriate encapsulation, entrapment or embedding vehicle will for example be one which can be separated from the sample with the target cells in the same step (e.g. by centrifugation or sedimentation) and which can be lysed, disrupted etc. under the same conditions which lyse the target cells or viruses.<br>
As the separation step is often carried out by centrifugation or sedimentation, in order to ensure that the non-viable particles can be separated from the sample with the target cells or viruses, it is often useful to select non-viable particles with similar weights and/or densities as the target cells or viruses.<br><br>
Methods for altering the density/weight of liposomes are well known and documented in the art.  For example the liposome density can be increased by filling the central core of the liposome with a more dense solution, such as cesium chloride (or other heavy compounds) or a high salt solution. Alternatively and preferably, density of the liposomes can be increased by the incorporation of polysaccharides, for example Blue Dextran or Dextran sulphate into the liposomes.  Said polysaccharides may be incorporated into the central core and/or the lipid membrane of the liposome.  "Dead" GMOs which are produced by modifying the live GMO so that it can no longer replicate and propagate should automatically have the same density as the target entities, e.g. living target cells.  The viral protein particles will also have practically the same density as the living target cells.  A way of making such particles has been described by Pear et al., (1993), PNAS USA 90: 8392-8396.<br>
If the sample preparation to obtain the target nucleic acid from the target cell or virus includes a step of selective cell capture, for example an immunoseparation step or other type of affinity separation step to bind cell surface proteins (or other molecules), then such surface proteins or molecules may need to be introduced into or onto the non-viable particles in order that the non-viable particles can be separated by the same steps as the target cells.  The introduction of proteins and other markers into the lipid membrane of a liposome particle can be carried out using techniques which are documented in the art (see for example Rongen et al., 1997, J. of Immunol. Methods, 204(2) : 105-33) and thus when a selective cell capture step is involved, appropriate liposome particles can be adapted for such use.  The dead GMOs and the protein coats will often automatically have the particular marker entrapped on the surface.  This is particularly<br><br>
the case where said protein coats or dead GMOs are prepared based on, e.g. by modifying, the target cells in question.  For example where the target cell is a type of microorganism then an appropriate encapsulation, entrapment or embedding vehicle would be an equivalent "dead" microorganism which would automatically express the same marker proteins on its surface.<br>
Generally, the composition of the non-viable particles should be as simple as possible.  Thus, if an unmodified simple non-viable particle has a similar stability to the target cells, can be separated with the target cells in the same step and can be lysed with the target cells under the same conditions, then none of the above discussed modifications such as the inclusion of proteins in the particle surface should be necessary or desired.  For example, in an assay for the microorganism Chlamydia trachomatis a simple centrifugation step can be used to separate the cells of the microorganism from the sample and thus a simple liposome (or other simple non-viable particle) with no modifications (such as proteins in the lipid membrane) but which has a density greater than the density of the fluid making up the sample (so that it can be centrifuged to the bottom of a vessel with the Chlamydia) and which will lyse under the same conditions as Chlamydia will be suitable for encapsulating, entrapping or embedding an 1C nucleic acid.<br>
The term "liposome particle" as used herein includes all types of liposomes and liposome-type vesicles known in the art.  Thus, at its most general, a "liposome particle" may be any lipid-based vesicular structure.  The 1C nucleic acid once designed and produced as described above can be introduced into liposome particles by methods which are well known and standard in the art.  In this regard, the nucleic acid might either be encapsulated within the internal aqueous pool or core of the liposome and/or be bound to the<br><br>
inner surface of the liposotne, for example by electrostatic forces or entrapped or embedded within the vesicular forming lipid-based layer, preferably within the inner surface.  In the methods and uses of the present invention it is preferred that the nucleic acid be encapsulated within the liposome (or other type of non-viable particle) and/or be bound to or entrapped or embedded within the inner surface of the liposome (or other type of non-viable particle), as any nucleic acid on the external surface of the particle will be at risk of degradation by nuclease enzymes or contamination by inhibitory impurities which may be present in the sample.<br>
Any method of encapsulation of the nucleic acid may be used.  However, there are three main methods described in the literature for the encapsulation of nucleic acids, (i) the reverse phase evaporation method, (ii) the dehydration/rehydration method and (iii) freeze/thawing (F. Szoka Jr., et al., Proc. Natl. Acad. Sci. USA 75 (1978) 4194-4198; D.W. Deamer, et al., J. Mol. Evol. 18 (1982), 203-206; U. Pick, Arch. Biochem. Biophys. 212 (1981), 186-194; M.J. Hope, et al. Biochim. Biophys. Acta 812 (1985) 55-65; C.J. Chapman et al, Chem. Phys. Lipids 55 (1990) 73-83; and Monnard et al., Biochim. Biophys. Acta 1329 (1997) 39-50) and the use of one of these methods is preferred.  Indeed the freeze/ thawing method has been shown to be particularly efficient in the encapsulation of nucleic acids (Monnard et al., supra) and this method is preferred.<br>
The efficiency of entrapment/encapsulation tends to vary depending on the conditions used during the encapsulation process and the lipid composition of the liposomes themselves. Appropriate encapsulation conditions for a particular 1C nucleic acid and liposome formulation can of course be derived by routine trial and error.  Some preferred liposome formulations for use in the present invention are discussed further below.<br><br>
In general, the efficiency of entrapment appears to be enhanced if the method of preparing the liposomes and encapsulation of the 1C nucleic acid involves extrusion of the liposome dispersion, for example by forcing the dispersion through one or more filters with appropriate pore sizes.  Liposomes which have been prepared using extrusion are thus preferred for aspects of the invention where high encapsulation efficiency is desired.  Methods of extrusion and the selection of appropriate pore sizes are well known and documented in the art.<br>
In general, the size of the 1C nucleic acid which is to be encapsulated or embedded into or entrapped in the liposome does not represent a problem, with small nucleic acid fragments (e.g. fragments of the order of tens of base pairs, e.g. fragments of 10 to 100 base pairs and fragments of the order of a few hundred base pairs) and larger molecules (of the order of a few kilobases) being encapsulated with similar efficiency. Again however adjusting the conditions may be used to improve encapsulation, embedding or entrapment if it is not adequate.<br>
Other methods may be used to facilitate uptake of nucleic acids into liposomes.  For example a nucleic acid in aqueous solution might be used to hydrate a dried mixture of lipids making up the liposomes, in which case the nucleic acid will be incorporated into the fluid filled core of the liposomes during liposome formation.  Alternatively methods wherein cationic lipid-DNA complexes can be made to form spontaneously when lipids are mixed with DNA can be used to facilitate the entrapment of the nucleic acids into liposomes (see for example Feigner et al., 1987, PNAS USA, 84: 7413-7417 and Hofland et al., 1996, PNAS USA, 93, 7305-7309).<br>
Liposomes are microscopic spherical particles in which membranes, consisting of one or more lipid bilayers, encapsulate a fraction of the solvent in which<br><br>
they are suspended into their interior.  Liposornes and methods of preparation thereof are well known and documented in the art.  Thus, the properties of the liposome particles into which the 1C nucleic acid is to be introduced can be manipulated and selected by methods well known and documented in the art.  Liposomes are generally composed of phospho(lipids) and may be composed of one or a mixture of lipids with varying properties.  Modified lipids, such as lipids modified with polyethylene glycol can also be used, as can liposomes coated with inert hydrophilic polymers (so-called "stealth" liposomes, Lasic, 1995, CRC Press) or liposomes which have been modified to include stabilising proteins such as S-layer protein (Mader et al., 1999, Biochimica et Biophysica Acta. 1418: 106-116) .<br>
The appropriate amount of each type of lipid to be incorporated into the liposomes will be selected based on the particular use to which the liposomes will be put.  The individual lipids making up the liposomes can have an overall positive, negative or neutral charge. Thus, depending on the particular combination and quantities of lipids used, the liposome particles themselves can have an overall positive (cationic), negative (anionic), or neutral charge.<br>
Preferred liposomes for the most efficient encapsulation, embedding or entrapment of nucleic acids will comprise an overall positive charge, i.e. will be "cationic11 liposomes containing a proportion of positively charged lipids.  Examples of such cationic (positively charged) lipids are DOTAP (1,2-dioleoyloxy-3-(trimethylammonium)propane), DOGS (N,N-dioctadecylamidoglycylspermine), DDAB<br>
(dimethyldioctadecylammonium bromide), DOTM&amp; (N-[l-(2,3-dioleyloxy)propyl] -N,N,N-trimethylammonium chloride) , DOSPA (2,3-dioleyloxy-N-[2(spermine-carboxamido)ethyl] -N,N~dimethyl-1-propanaminiumtrifluoroacetate) and DMRIE<br><br>
(N- [1- (2,3-dirayristyloxy)propyl] -N,N-dimethyl-N- (2-hydroxyethyl)ammonium bromide).  Particularly preferred liposomes for use in the present invention comprise one or more neutral lipids such as POPC (l-palmitoyl-2-oleoyl-sn-glycero-3-phosphocholine) or DOPE {1,2-dioleoyl-3-sn-phosphatidylethanolamine), and one or more of the positively charged lipids DDAB, DOTAP or DOSPA. The particular proportions of lipid which are optimum for encapsulation, embedding or entrapment can be determined by routine trial and error, but an exemplary liposome composition might comprise from between 1% to 50% positively charged lipid, for example from between 1% to 10% DDAB (or another positively charged lipid) or from between 20% to 50% DOTAP (or another positively charged lipid), or a ratio of positively charged lipid to neutral lipid of from approximately 0.5:1 to approximately 5:1, for example a ratio of DOSPA (or another positively charged lipid) to a neutral lipid (e.g. DOPE) of approximately 3:1 or a ratio of DOTAP (or another positively charged lipid) to a neutral lipid (e.g. DOPE) of approximately 1:1.<br>
A further type of particularly preferred liposomes are those which comprise a proportion of phospholipid which are phospholipid derivatives of polyethylene glycol, for example PEG-PE (N- (co-methoxypoly-(oxyethylene)oxycarbonyl)-DSPE).  (DSPE = 1,2-distearoyl-3-sn-phosphatidylethanolamine).  Such liposomes are also preferably cationic liposomes.<br>
A yet further type of preferred liposomes are those in which a polysaccharide (e.g. Blue Dextran or Dextran sulphate) has been incorporated, for example to increase the density of the liposomes.  Said polysaccharide can be incorporated into the central core and/or the lipid membrane of the liposome.<br>
Most particularly preferred liposomes comprise the lipids POPC and DDAB, preferably in a ratio of 97.5:2.5, or the lipids DOTAP, DOPE and PEG-PE, preferably in a<br><br>
molar ratio of 25:25:3.  Optionally, these liposomes may also comprise polysaccharides such as Blue Dextran or Dextran sulphate.<br>
Other examples of lipids that can be used to form the liposome molecules in accordance with the present invention are well known and documented in the art. Some examples of neutral lipids include cholesterol, DPPC (dipalmitoylphosphatidylcholine), egg phosphatidylcholine and soybean phosphatidylcholine. Examples of negatively charged lipids include PS (phosphatidyl serine), PI (phosphatidyl inositol), ganglioside GM1, PG (phosphatidyl glycerol) and PA (phosphatidic acid).  Further examples of positive lipids include HDA (hexadecylamine).<br>
Liposomes can be produced in various sizes from small (often unilamellar) vesicles of 50-150 nm to large (often multilamellar) vesicles of a few fims.  The size range can be chosen as appropriate and is a compromise between loading efficiency of liposomes (increases with increasing size) and liposome stability (decreases with increasing size above an optimal 80-200 nm range).  The choice of liposome size can be determined by routine trial and error.  However, a preferred size for the liposomes might be in the range of 80-200 nm.<br>
Preferred liposomes for use in accordance with the present invention are liposomes which are stable in biological media and fluids such as for example blood, serum, faeces, urine etc.  The stability of liposomes can be increased by for example coating liposomes with inert hydrophilic polymers such as PEG or by coating them with proteins derived from thermally stable bacteria (for example coating with S-layer protein - see Mader et al., Bioch. Biophys. Acta. 1418 (1999) 106-116).  Alternatively the use of lipids derived from thermophilic bacteria could be considered (see e.g. Chang. Bioch. Biophys. Research Communications, vol. 202, 1994, 673-679).  As discussed above the stability<br><br>
of the liposome is designed to mimic the stability of the target cells.<br>
Once prepared the liposomes can either be used immediately in the methods and uses of the present invention or stored for future use.  If the liposomes are to be used immediately, conveniently they are used in the form of an aqueous suspension.  If the liposomes are to be kept for an extended period of time before use then preferably the preparations will be freeze dried or lyophilised for storage using methods well known and documented in the art.  Such dried preparations can then be rehydrated for use at the appropriate time.<br>
For the embodiment where the 1C nucleic acid is encapsulated within or embedded or entrapped in a viral protein coat, the particular 1C nucleic acid can be encapsulated using methods which are well known and documented in the art.  Such methods will generally involve the mixture of the various proteins making up the protein coat in appropriate portions, together with the particular 1C nucleic acid which is to be encapsulated and subjecting said mixture to conditions which induce the assembly of the viral proteins to form a viral coat particle.  In this way the 1C nucleic acid will automatically become trapped/encapsulated within the viral coat particle.  Assembly of virus capsids might for example be carried out in vitro, either after e.g. disruption of the capsid by chelating agents and reassembly by addition of calcium (e.g. Brady et al. 1979, J. Virol. 32: 640-647, Paintsil et al. 1998, J. General Virol. 79; 1133-1141) or after in vivo expression of the capsid proteins (Sternberg 1990, PNAS USA, 87: 103-7; Hwang et al. 1994, PNAS USA, 91: 9067-71; Tellinghuisen et al. 1999, J. Virol, 73: 5309-19).<br>
For the embodiments where the 1C nucleic acid is encapsulated in a "dead" or non-viable GMO, the 1C nucleic acid could first be cloned into the appropriate GMO using standard and well known techniques, after<br><br>
which the GMO could be inactivated or "killed" in such a way that the cell surface/cell membrane of the GMO and the 1C nucleic acid remained unaffected or intact, thereby leaving the 1C nucleic acid trapped inside the cell membrane of a non-viable GMO.  Suitable reagents which could be used to inactivate the genetic and cellular machinery of the GMO without affecting the cell membrane or the 1C nucleic acid are well known in the art and include for example antibiotics, antimicrobial agents or other agents that target and disrupt protein synthesis, for example by disrupting or altering ribosome functions.  Examples of antibiotic or antimicrobial agents that could be used to inactivate the GMOs are tetracyclin which inhibits protein synthesis, rifamycin that inhibits transcription and nalidixic acid that inhibits DNA gyrase and blocks replication (Davies and Smith 1978, Annual Review of Microbiology, 32: 469-518).<br>
Although it is known to introduce nucleic acids into non-viable particles such as those discussed above, it is nowhere disclosed in the prior art that the nucleic acid to be introduced can be an 1C nucleic acid. Thus, non-viable particles, for example liposome particles, synthetic particles, viral coat protein particles or "dead" GMOs comprising an 1C nucleic acid form further embodiments of the present invention.  Non-viable particles such as those described herein for use in the methods and uses of the invention described herein form yet further aspects of the invention.<br>
Especially preferred nucleic acid-based assays in which the 1C nucleic acids are used are those in which there is a risk of false negative results occurring, for example in diagnostic assays, especially those carried out on samples derived directly from the patient. Alternatively or additionally, the liposomes (or other non-viable particles) and internal controls of the present invention may be used simply to monitor the<br><br>
amplification of nucleic acids in any qualitative nucleic acid based test, for example in a PCR or LCR based test.  "Pseudo-ideal" or "near-ideal" or "non-ideal" 1C nucleic acids can be used for qualitative nucleic acid based tests.<br>
Furthermore, as will be described in more detail below, whatever the source of the sample, the non-viable particles and the 1C nucleic acids of the present invention can be used in any assay where quantitation of the nucleic acids produced is required.  Where quant itat ion is required, the use of a "pseudo- ideal" or "near-ideal" 1C nucleic acid is necessary whereby the 1C nucleic acid and the target nucleic acid are amplified and/or detected or otherwise assayed by the same probes or primers (or informational content, etc).<br>
A preferred method of nucleic acid-based analysis according to the present invention will comprise the steps of:<br>
(i)   obtaining a sample to be analysed;<br>
(ii)  bringing said sample into contact with non-viable particles comprising an appropriate internal control nucleic acid;<br>
(iii) inducing the release of the nucleic acid to be analysed from within the sample and the release of the internal control nucleic acid from within the non-viable particles; and<br>
(iv)  analysing the released nucleic acids.<br>
In step (i) the sample to be analysed may be any sample on which it is desired to carry out a qualitative or quantitative nucleic acid-based assay.<br>
Said samples may thus be derived from in vitro sources (such as cultured cells, bacteria or viral particles) or in vivo sources (such as samples derived from human, plant or animal sources) or synthesized in the case of samples containing non-naturally occurring target entities.  Other samples might be those that are tested for detection of food pathogens.  If the sample<br><br>
is derived from a patient or an animal, the appropriate type of biological sample to obtain will vary depending on the nature and source of the nucleic acid being analysed, but examples include blood, serum, plasma, saliva, faeces, urine, milk and organ, tissue or cellular extracts or secretions, e.g. mucosal secretions etc.  Such samples are likely to be fairly impure and contain all manner of enzymes and other compounds which will inhibit or impair the enzymes involved in DNA amplification (or inhibit or impair other enzymes or entities involved in non-amplification based nucleic acid assays) , or will degrade any nucleic acid {e.g. the target nucleic acid or any 1C nucleic acid) which is itself present.  Methods for obtaining appropriate samples from patients or animals are well known and described in the art.<br>
The addition of the non-viable particles to the sample (i.e. step (ii)) can be carried out in any convenient way.  The sample to be analysed is likely to be in the form of a solution or can conveniently be made in the form of a solution e.g. by adding fluid and/or tissue disrupting or dissolving agents.  Appropriate preparations and properties of non-viable particles for use in accordance with the present invention are defined and described above.  Thus, the non-viable particles when bought into contact with the sample may be in the form of an aqueous suspension of said particles or may be in a dried form which are then rehydrated and reconstituted when added to the fluid sample.<br>
In a preferred embodiment of the invention the time between step (i) (obtaining the sample) and step (ii) adding the non-viable particles is kept to a minimum. Thus, the particles are added to the samples as soon as practicable and ideally are added immediately after sample is obtained in vitro or collected from the patient or animal.  It should be borne in mind however, that for the methods of the invention to work, the non-<br><br>
viable particles can be added to the sample at any convenient time before, at the same time, or during the time that the inducing of the release of the nucleic acid from the sample (i.e. step (iii)) takes place.  The particles should not be added after this time because then the 1C nucleic acid will not be released from the particles at the same time point that the target nucleic acid is released from the target cells and therefore will not be taken through the procedure with the target nucleic acid.<br>
Step (iii) of the above discussed method involves inducing the release of the nucleic acid to be analysed from within the sample and the 1C nucleic acid from within the non-viable particles.  Conveniently the nucleic acids from the sample (e.g. from the cells or viruses of the sample) and the particles are jointly released, i.e. are released at the same time and under the same conditions.  Accordingly, an appropriate lysis buffer or other disruptive agent or conditions which induce the disruption of both the membranes of the target cells (or other target entities) and the structure of the non-viable particles used in the assay should be selected.  This is of course not difficult to do when the non-viable particles are liposomes as the nucleic acid from the sample will normally be contained in cells and the liposomes by their very nature resemble cell membranes.  Thus, conditions (for example lysis buffers), which will disrupt/lyse the membranes of the cells or other target entities in the sample and release the nucleic acid contained therein, should in general also disrupt/lyse the liposome particles.  Similar considerations apply when the non-viable particles used are "dead" GMOs.  "Dead" GMOs will generally be appropriate non-viable particles in assays where the target cells are the equivalent or similar "live" wild type organisms and thus again it is clear that conditions under which the wild type cells are lysed<br><br>
will also result in lysis of the "dead" GMOs.<br>
Appropriate lysis medium/buffers are well known and standard in the art and may for example contain detergents or proteinases or protein denaturing agents such as guanidine thiocyanate which will disrupt the membranes of the target cells or viruses (and the liposome membranes) and also the particles which comprise a. viral protein coat.  Other standard ingredients such as protease inhibitors, DNase and RNase inhibitors, preservatives, chelating agents etc. can also be added to the lysis buffer if desired or necessary.<br>
It will be usual practice that before inducing the release of the nucleic acids in step (iii) the cells (or other target entities) and particles will be separated from the other materials in the samples.  Such separation can conveniently be carried out for example by centrifugation of the target cells or viruses and liposomes (non-viable particles) into a pellet and carefully removing the unwanted supernatant before the lysis medium is added.  Thus, such a separation step is an optional step in the above discussed methods.<br>
As the samples are generally obtained from patients the above method may optionally involve the steps of transporting the sample to the analysis laboratory and storage of the sample prior to sample preparation.  Such transportation and/or storage steps might occur after step (i), step (ii) or step (iii).<br>
Once the nucleic acids have been released these can be analysed (step iv) . Generally said analysis step will involve carrying out an appropriate nucleic acid-based analysis or assay as described above, which may or may not involve amplification, followed by some kind of examination or detection step. Analysis may be carried out on the crude cell (and non-viable particle) lysate directly (generally after amplification of the nucleic acids to be analysed has been carried out, see below),<br><br>
or on DNA or RNA or PNA molecules which have been further purified from the crude cell (and non-viable particle) lysate by standard methods.  In general, if the nucleic acid which is being analysed is a DNA molecule, then the analysis of the crude cell (and non-viable particle) lysate is not problematic and the same reproducibility can be obtained as with purified DNA. Moreover, the use of crude cell lysates entails less labour and avoids the potential loss of specific sample which may occur during the DNA purification.  Where the nucleic acid to be analysed is RNA, generally a further purification step to isolate the RNA from the crude lysate is preferred.<br>
Usually, in order to aid analyses of the nucleic acids an amplification step will be carried out at some stage.  Amplification will normally be required as the amount of target nucleic acid present and/or 1C nucleic acid present in a sample is likely to be at a concentration below that which is easily detectable. Thus, if the nucleic acid-based assay involves amplification, this amplification may be followed by a further amplification step before examination or detection occurs. Alternatively, if the nucleic acid-based assay does not involve amplification then an amplification step may be carried out before examination or detection of the nucleic acid occurs.  Alternatively, no amplification step may be required before examination or detection takes place.<br>
Amplification can be carried out by any of the techniques well known and documented in the art, e.g. PCR, LCR, Gap LCR, NASBA or TMA.  For the DNA based amplification reactions to occur a DNA rather than an RNA molecule should be present as a template.  Thus, it should be borne in mind that if the 1C nucleic acid and/or target nucleic acid are RNA molecules then these should first be reverse transcribed into DNA molecules which can then be amplified by the usual techniques. As<br><br>
discussed above, the primers or amplification probes used for such amplification may be specific for either the 1C nucleic acid or the target nucleic acid or may be designed so that they amplify not only the target nucleic acid (if present) but also the 1C nucleic acid.<br>
Regardless of whether the analysis step involves amplification, where that is performed, the examination and detection part of the analysis step will generally depend on the distinguishing feature of the 1C nucleic acid over the target nucleic acid.  If for example the 1C nucleic acid is of a different size to the target then the analysis can conveniently be carried out by separating the nucleic acids on an agarose gel and visualising the nucleic acids in the sample by e.g. ethidium bromide staining or by e.g. separating the nucleic acids by capillary gel electrophoresis and detecting the nucleic acids in the sample by virtue of some kind of incorporated label, e.g. a fluorescent label incorporated into one or more of the primers or probes used.  In embodiments of the invention where different sets of primers or probes, or other entities are used to analyse the target nucleic acid and the 1C nucleic acid (i.e. in embodiments where non-ideal 1C nucleic acid sequences are used), preferably different labels, e.g. different fluorescent labels, may be incorporated into one or more of the primers or amplification probes of each primer or amplification probe set so that the amplification products deriving from the 1C nucleic acid or the target nucleic acid (if present) can be distinguished using appropriate apparatus and/or software. Alternatively, the amplification reaction may be carried out using a proportion of labelled nucleotides (e.g. fluorescent or radiolabelled nucleotides) , in which case the separated and labelled nucleic acids can be visualised by appropriate detection means well known in the art.  In methods of nucleic acid-based analysis which do not<br><br>
involve amplification the probes or entities which may be used to assay the 1C nucleic acid and/or the target nucleic acid can also be engineered to carry appropriate labels other entities which can be detected and distinguished.  Alternatively, in such non-amplification based methods it may be possible to distinguish the target nucleic acids from the 1C nucleic acids by other methods, e.g. by virtue of size or other informational content.<br>
The analysis of the nucleic acids in step (iv) can either be qualitative e.g. an observation as to whether the band corresponding to the target nucleic acid is present or absent, or may be quantitative in that the concentration of the target nucleic acid present can be determined.  In either a quantitative or a qualitative method, it is important to first ascertain that the quality control of the assay is ensured by the observation of the presence of the 1C sequence which has been taken through at least some of the same steps as the target nucleic acid.  For example, in nucleic acid-based assay methods which involve amplification, an amplification of the 1C sequence in the absence of amplification of the target sequence will then be evidence of a correct negative result and the amplification of both sequences a correct positive result.  No amplification of either the 1C sequence or the target sequence might well indicate technical failure or some problem with the assay conditions, e.g. the presence of inhibitors.<br>
In a qualitative nucleic acid-based assay involving amplification the amplification of the target sequence in the absence of amplification of the 1C nucleic acid will not normally be regarded as a concern and such a sample would be noted as a correct positive result. However, it should be noted that in a quantitative assay involving the use of a "pseudo-ideal" or "near-ideal" 1C sequence, the amplification of the target sequence in<br><br>
the absence of the amplification of the 1C sequence is likely to indicate that the target sequence was present in such high quantities in the sample that the amplification of the 1C sequence was, in effect, competed out.  Competition of the 1C nucleic acid and target nucleic acid for the same primers in fact provides the basis of one of the best methods of quantitation of the assay, i.e. by so-called "competitive PCR".<br>
The methods of the present invention as described above involving a "pseudo-ideal" or a "near-ideal" 1C sequence are well suited to competitive PCR as the 1C nucleic acid and the target nucleic acid may be amplified by the same primers.  Thus the amount of amplification which will occur will be a function of the concentration of nucleic acid molecules present in the original sample.  If the 1C nucleic acid is present at a higher concentration than the target nucleic acid then this will be subject to more amplification and vice versa.  When the ratio of 1C nucleic acid to target nucleic acid is equal, i.e. 1:1, then one can expect the amount of amplification of both species to also be equal.<br>
This provides a convenient way of quantitating the results.  For example a series of samples can be set up containing various dilutions of a known amount of the 1C nucleic acid and unknown amounts of the target nucleic acid.  Analysis of the amplification products should reveal the concentration of 1C nucleic acid at which the amplification of both species is roughly equal and the quantity of target nucleic acid present in the original sample can be extrapolated by analysing the amount of amplification products present at the various dilutions and drawing up standard curves.  Quantitative competitive PCR is a standard procedure and is described for example in the review by Zimmerman et al., supra.<br>
Relative quantitation can also be carried out using<br><br>
a series of dilutions containing constant unknown amounts of 1C nucleic acid and varying known concentrations of the target nucleic acid.  Clearly however the most likely scenario in any assay will be that it is the concentration of 1C nucleic acid rather than target nucleic acid which is known and thus the first described method is likely to be more appropriate.<br>
Any other suitable method of quantitation can be used to analyse the products of the nucleic acid-based assay, e.g. the amplification products.  For example, the TaqMan system can be used where the accumulation of PCR products are measured in "real time" by release of two fluorescent dyes, one for the wild type and one for the 1C targets (Heid et al., 1996, 6: 986-94, Tremmel et. al., 1999, Tissue Antigens, 54: 508-16).<br>
The methods and uses as described above can be used in a number of different technical fields, for example in the fields of analysis and diagnosis,. Most preferably the methods and uses can be used for qualitative or quantitative analysis of nucleic acids or for diagnosis.<br>
In a yet further aspect, the present invention provides a kit for carrying out the methods or uses of the invention which comprises non-viable particles containing or comprising an appropriate 1C nucleic acid as described and defined above.<br>
Optionally said kits may further comprise one or more reagents selected from reagents suitable for nucleic acid amplification, such as appropriate primers or amplification probes (in the case of LCR) designed to be compatible with the 1C nucleic acid in question and/or the target nucleic acid to be analysed and which may optionally be labelled, nucleotides (a proportion of which may be labelled), DNA polymerases, probes or other entities which can hybridise to the target nucleic acid and/or the 1C nucleic acid and give rise directly or indirectly to a detectable signal, e.g. are optionally<br><br>
labelled, etc.<br>
The invention will now be described in more detail in the following non-limiting Examples with reference to the following figures in which:<br>
Figure 1 shows an agarose gel with PCR products from nucleic acid encapsulated in POPC/DDAB liposomes. Lanes M, molecular size standard (123-bp ladder, Life Technologies); lane 1, liposomes with 1C prepared without extrusion; lane 2, liposomes with 1C prepared with extrusion,- lane 3, procedure contamination control<br>
-	liposomes prepared without DNA; lane 4, nuclease<br>
control - 1C nucleic acid solution treated with DNase I<br>
and Exonuclease III; lane 5, procedure positive control<br>
-	1C nucleic acid solution; lane 6, non-template PCR<br>
control;<br>
Figure 2 shows the electropherogram after capillary gel electrophoresis of C. trachomatis PCR product (207 bp) and 1C PCR product (216 bp) from nucleic acid from cultured C. trachomatis and/or from POPC/DDAB liposome/ 1C DNA complex.  The size of the PCR products is interpolated from the internal size standard (GenScan-500 TAMRA, Applied Biosysterns).<br>
A.	Cell culture of C. trachomatis spiked with<br>
POPC/DDAB liposomes containing 1C nucleic acid.<br>
B.	POPC/DDAB liposomes containing 1C nucleic acid<br>
without C. trachomatis.<br>
C.	Cell culture of C. trachomatis without spiking.<br>
D.	Non-template PCR control;<br>
Figure 3 shows agarose gel with PCR products from nucleic acid encapsulated in DOTAP/DOPE liposomes coated with polyethylene glycol.  Lanes M, molecular size standard (123-bp ladder, Life Technologies); lane 1, liposomes with 1C prepared without extrusion; lane 2, liposomes with 1C prepared with extrusion; lane 3, procedure contamination control - liposomes prepared without DNA: lane 4, nuclease control - 1C nucleic acid solution treated with DNase I and Exonuclease III; lane<br><br>
5, procedure positive control - 1C nucleic acid solution; lane 6, non-template PCR control.<br>
Figure 4 shows the electropherogram after capillary gel electrophoresis of C. trachomatis PCR products (207 bp) and 1C PCR products (216 bp) from nucleic acid from a urine specimen spiked with cultured C. trachomatiB and/ or POPC/DDAB liposome/IC DNA/Blue Dextran complex. The size of the PCR products is interpolated from the internal size standard (GenScan-500 TAMRA, Applied Biosystems).<br>
A.	Nucleic acid prepared from urine specimens spiked<br>
with cultured C. trachomatis and POPC/DDAB<br>
liposome/IC DNA/Blue Dextran complex.<br>
B.	Nucleic acid prepared from urine specimens spiked<br>
with cultured C. trachomatis.<br>
C.	Nucleic acid prepared from urine specimens spiked<br>
with POPC/DDAB liposome/IC DNA/Blue Dextran<br>
complex.<br>
D.	Negative control.  Urine specimens without spiking.<br>
EXAMPLE 1<br>
Detection of 1Cjnucleic acid entrapped in liposomes<br>
Production of the 1C nucleic acid<br>
A 216 bp segment of the phage M13 genome was chosen as<br>
an 1C sequence and was amplified by use of published<br>
primers (Berg and Olaisen 1994, Biotechniques, 17: 896-<br>
901) :<br>
LacL: 5'-GGCGAAAGGGGGATGTGC-3'<br>
LacH: 5 ' - (FAM) - CGGCTCGTATGTTGTGTGGAAT-3 '<br>
In the PCR production of the 1C nucleic acid 1 ng<br>
M13mpl8 DNA (Pharmacia) was used as template DNA<br>
followed by a 30 cycle PCR using the same conditions as<br>
published for the LacL/LacH primers.  The resulting PCR<br>
product was purified by gelfiltration (Sephacryl S300,<br><br>
Pharmacia) .  Due to the FAM fluorescent modification of the LacH primer, the PCR product can be detected in an ABI Prizm 310 Genetic Analyzer by capillary gel electrophoresie and analysis by Genescan software.<br>
Endpoint titration with subsequent PCR revealed the amount of the DNA in the purified 1C solution.<br>
Preparation of the liposomes and encapsulation of the 1C nucleic acid<br>
Liposomes containing the lipids POPC and DDAB in a ratio of 97.5:2.5 are prepared by the freezing/thawing procedure as described by Monnard et al. (BBA, 1329: 39-50, 1997) .  The lipids are dissolved in chloroform and the solvent subsequently removed by evaporation followed by an overnight drying under high vacuum.  The dried lipids are dispersed in a buffer solution (50 mM Tris, pH 8.0) and sonicated for 10 minutes in a bath sonicator.  Then 10 fig of the 1C nucleic acid to be encapsulated is added and the final concentration of lipids adjusted to 120 mM.  The dispersion is treated by freeze/thawing 10 times. After freezing in liquid nitrogen, the samples are thawed for 15 minutes at room temperature.  Before extrusion, the liposome dispersion is diluted to a lipid concentration of 40 mM using 50 mM Tris (pH 8.0) and then forced 10 times through two stacked polycarbonate filters with pore sizes of 400 nm in diameter (for extrusion a Liposofast from Avestin Inc. is used) .  The extruded liposomes are loaded on a 'spin column1 (Bio-Gel A-15 m, previously equilibrated with the appropriate buffer, pH 8.0) and centrifuged at 165 x g for 2 minutes.  Usually 22-24 eluates of about 50 /il each are collected: the fractions 2-7 are usually turbid, the others showed no clearly visible turbidity, indicating that they contain no significant number of liposomes.<br><br>
Lysis of the liposome/DNA complex<br>
Isolation of DNA from the suspension of liposomes with entrapped 1C was done by use of a Qiagen DNA mini kit according to a procedure described by the manufacturer. Briefly, after lysis of the liposomes in the presence of detergents, nucleic acids were bound to a silica-gel membrane, washed and finally eluted in TE buffer.<br>
Analysis of isolated 1C DNA<br>
The 1C DNA isolated from the liposomes was analysed in a PCR reaction that included the LacL/LacH primers.  The resulting PCR product was analysed after agarose gel electrophoresis and EtBr staining with visualisation under UV light.<br>
EXAMPLE 2<br><br><br>
The following PCR primers were used in an assay for<br>
detection of Chlamydia trachoma tis, such primers being<br>
slightly modified at the 5" ends compared to the<br>
published primers (Loeffelholz et al. 1992, J. Clinical<br>
Microbiology, 30, 2847-51):<br>
CP24 5 ' -GGGATTCCTGTAACAACAAGTCAGG-3 '<br>
CP27 5 ' - (ROX) -CCTCTTCCCCAGAACAATAAGAACAC-3 '<br>
These primers define an amplification product containing 207 bp of the C. trachomatis cryptic plasmid.<br>
The production of the 1C and the entrapment of it into liposomes was done as described in the above example 1.<br><br>
Isolation of DNA from an i-n vivo sample containing C. trachomabis spiked wit;h entrapped 1C<br>
A cell suspension of a cultured C. trachomatie L2 strain was spiked with liposomes/IC DNA complex prepared as described above.  Chlamydia and 1C DNA was prepared from the cells/liposomes by using a Qiagen DNA mini kit according to a procedure described by the manufacturer.<br>
PCR analysis of the DNA solution<br>
The purified Chlamydia and 1C DNA was analysed in a multiplex PCR including both of the above primer sets. The 35 cycles PCR was performed by use of Amplitaq Gold (Roche) as described by the manufacturer of the enzyme. Due to different fluorescence labelling of the Chlamydia and 1C PCR products they were distinguishable in an ABI Prizm 310 Genetic Analyzer after capillary gel electrophoresis and analysis by Genescan software.<br>
EXAMPLE 3<br>
Preparation of cationic  lioosomes coated with<br>
The  liposomes encapsulating an 1C nucleic acid are prepared as described in Example 1.     In this case however the liposomes contain the cationic lipid DOTAP [1,2-dioleoyloxy-3- (trimethylammonium) propane] ,   the neutral  lipid DOPE   [l,2,dioleoyl-3-sn-phosphatidylethanolamine]   and PEG-PE   [N- (a-methoxypoly (oxyethylene)oxycarbonyl)-DSPE]   at a molar ratio 25:25:3 and extrusion was carried out  through 50  nm pore  filters [DSPE = l,2-distearoyl-3-sn-phosphatidylethanolamine].<br><br>
EXAMPLE 4<br>
Detection of 1C nucleic acid entrapped in liposomes<br>
Production of the 1C nucleic acid<br>
A 216 bp segment of the phage M13 genome was chosen as<br>
an 1C sequence and was amplified by use of published<br>
primers (Berg and Olaisen 1994, Biotechniques, 17: 896-<br>
901) :<br>
La cL: 5' -GGCGAAAGGGGGATGTGC-3'<br>
LacH: 5'-(FAM)-CGGCTCGTATGTTGTGTGGAAT-3'<br>
In the PCR production of the 1C nucleic acid 1 ng<br>
M13mpl8 DNA (Pharmacia) was used as template DNA<br>
followed by a 30 cycle PCR using the same conditions as<br>
published for the LacL/LacH primers.  The resulting PCR<br>
product was purified by gelfiltration (Sephacryl S300,<br>
Pharmacia).  The resulting PCR product was purified by<br>
gelfiltration (Sephacryl S300, Pharmacia).  Endpoint<br>
titration with subsequent PCR revealed the amount of the<br>
DNA in the purified 1C solution.<br>
Due to the FAM fluorescent modification of the LacH primer, the Lac PCR product can be detected in an ABI Prizm 310 Genetic Analyzer by capillary gel electrophoresis and analysis by Genescan software.<br>
Preparation of the liposomes and encapsulation of the 1C nucleic acjd<br>
Liposomes containing the lipids POPC (l-palmitoyl-2-oleoyl-si2-glycero-3-phosphocholine) and DDAB (didodecyldimethy 1 ammonium bromide) in a ratio of 97.5:2.5 were prepared by the freezing/thawing procedure as described by Monnard et al. (BBA, 1997, 1329: 39-50). The lipids were dissolved in chloroform and the solvent subsequently removed by evaporation followed by an<br><br>
overnight drying under high vacuum.  The dried lipids were dispersed in a buffer solution (50 mM Tris, pH 8.0) and sonicated for 10 minutes in a bath sonicator.  Then the 1C nucleic acid to be encapsulated was added and the final concentration of the nucleic acid and lipids adjusted to 2.3 pM and 120 mM, respectively.  The dispersion was treated by freeze/thawing 10 times. After freezing in liquid nitrogen, the sample was thawed for 15 minutes at room temperature.  Before extrusion, the liposome dispersion was diluted to a lipid concentration of 40 mM using 50 tnM Tris (pH 8.0) . Extrusion was then carried out by forcing the liposome dispersion 10 times through two stacked polycarbonate filters with pore sizes of 400 nm in diameter (for extrusion a Liposofast from Avestin Inc. was used) . After extrusion, the non-encapsulated nucleic acid was digested by a combined DNasel and Exonucleaselll treatment as described in Monnard et al., supra.<br>
Lysis, of the liposome/DNA complex<br>
/<br>
Isolation of DNA from the suspension of liposotnes with entrapped 1C was done by use of a QIAamp DNA mini kit (Qiagen GmbH) according to a procedure described by the manufacturer. Briefly, after lysis of the liposomes in the presence of detergent, nucleic acid was bound to silica-gel membrane, washed and finally eluted in TE-buffer.<br>
Analysis of isolated DNA<br>
The 1C DNA isolated from the liposomes was amplified in a PCR that included the LacL/LacH primers.  The resulting PCR product was analysed after agarose gel electrophoresis and Ethidium bromide staining with visualisation under UV light (figure 1) .<br><br>
Figure 1 shows an agarose gel with PCR products from nucleic acid encapsulated in POPC/DDAB liposomes. Seemingly, the extrusion through two polycarbonate filters enhances the entrapment efficiency of nucleic acid in the particles.  Further, the results of the control experiments demonstrate that the 1C nucleic acid was present in the core region of the liposomes and was protected from the nuclease digestion.  Lanes M, molecular size standard (123-bp ladder, Life Technologies); lane I, liposomes with 1C prepared without extrusion; lane 2, liposomes with 1C prepared with extrusion; lane 3, procedure contamination control<br>
-	liposomes prepared without DNA; lane 4, nuclease<br>
control - 1C nucleic acid solution treated with DNase I<br>
and Exonuclease III; lane 5, procedure positive control<br>
-	1C nucleic acid solution; lane 6, non-template PCR<br>
control<br>
EXAMPLE 5<br>
Use of the j-iposome/IC complex for quality assurance in an assay for detection of Chlarpydia trachomatis<br>
The following PCR primers were used in an assay for detection of Chlamydia trachomatis, such primers being slightly modified with fluorescent dyes compared to the published primers (Loeffelholz et al. 1992, J. Clin. Microbiol., 30: 2847-51):<br>
CP24 5 ' - (FAM) -GGGATTCC- (T-ROX) -GTAACAACAAGTCAGG-3 ' CP2 7 5 ' - CCTCTTCCCCAGAACAATAAGAACAC - 3 '<br>
(FAM and ROX are different fluorophores, green and red, respectively).<br>
These primers define an amplification product containing 207 bp of the C. trachoma tie cryptic plasmid.<br><br>
The production of the 1C and the entrapment of it into liposomes was done as described in the above Example 4.<br>
Isolation of DNA from an in vivo sample containing C. trachomatig spiked with entrapped 1C<br>
A" cell suspension of a cultured C. trachomatis L2 strain was spiked with liposomes/IC DNA. complex prepared as described above.  Chlamydia and 1C DNA was prepared from the cells/liposomes by using a QIAamp DNA mini kit (Qiagen GmbH) according to a procedure described by the manufacturer.<br>
PCR analysis of. the DNA solution<br>
The purified Chlamydia and 1C DNA was analysed in a multiplex PCR including both of the above primer sets. The 25 cycles PCR was performed by use of Amplitaq Gold<br>
(Roche) as described by the manufacturer of the enzyme. Due to the different fluorescent labelling of the Chlamydia and Lac PCR products they were distinguishable in an ABI Prizm 310 Genetic Analyzer after capillary gel electrophoresis and analysis by Genescan software<br>
(figure 2).<br>
EXAMPLE 6<br>
Preparation of cationic liposomes coated with polyethylene glycol and encapsulation of an 1C nucleic acjd<br>
The liposomes encapsulating an 1C nucleic acid was prepared as described in Example 4.  In this case however the liposomes contained the cationic lipid DOTAP [1, 2-dioleoyloxy-3-(trimethylammonium)propane], the neutral lipid DOPE (l,2-dioleoyl-3-sn-phosphatidyl-ethanolamine) and PEG-PE [N-(<a-methoxypoly></a-methoxypoly><br>
oxycarbonyl)DSPE] at a molar ratio 25:25:3 [DSPE = 1,2-distearoyl-3-sn-phosphatidylethanolamine] as described by Meyer et al, (JBC 1998, 273(25):15621-27).  The PCR product from the released 1C nucleic acid was analysed after agarose gel electrophoresis (figure 3).<br>
Figure 3 shows agarose gel with PCR products from nucleic acid encapsulated in DOTAP/DOPE liposomes coated with polyethylene glycol.  As for the case with the POPC/DDAB liposomes described in Example 4, extrusion seems to enhance the entrapment efficiency of the nucleic acid into the particles.  However, on comparing the results of Example 4 with the present example, no differences in the performance of the two liposome systems were observed.  Lanes M, molecular size standard (123-bp ladder, Life Technologies) ; lane 1, liposomes with 1C prepared without extrusion; lane 2, liposomes with 1C prepared with extrusion; lane 3, procedure contamination control - liposomes prepared without DMA: lane 4, nuclease control - 1C nucleic acid solution treated with DNase I and Exonuclease III; lane 5, procedure positive control - 1C nucleic acid solution/lane 6, non-template PCR control.<br>
EXAMPLE 7<br>
Generation of lir&gt;osome/IC DNA/Blue Dextran Comdex with.<br>
and, their use in a Chlamydla tjrachojnafcls PCR assay<br>
The Lac PCR product used as 1C nucleic acid was made as described in Example 4.<br>
The preparation of the liposomes/IC complex was done as described in Example 4 with the following modification; Blue Dextran (Pharmacia) was added to the POPC/DDAB liposome dispersion together with the 1C subsequent to<br><br>
the sonication.  The concentration of the polysaccharide and the nucleic acid was adjusted to 75 nM and 23 pM, respectively.  The subsequent freeze/thawing, extrusion and nuclease treatment was done as described in Example 4.<br>
Isolation of DNA from a urine sample containing C. trachoma.tis spiked with liposQtne/JC DNA/Bli|e Dextran complex<br>
A urine sample collected from a healthy donor was spiked with a cell suspension of a cultured C. trachomatis L2 strain as well as with the liposome/IC DNA/Blue Dextran complex.  Generation of a crude DMA lysate from the sample was done by use of the reagents provided in the commercially available COBAS AMPLICORâ¢ CT/NG assay (Roche Molecular Systems Inc. Diagnostics, Brachburg, NJ, USA) kit according to a procedure described by the manufacturer.  Briefly, 500 /il of the urine specimen was diluted by 500 /il of the CT/NG Urine Wash solution, incubated at 37Â°C followed by centrifugation at 12.500 x g for 5 minutes.  The cell/liposome pellet was re-suspended in 250 /zl of the CT/NG LYS solution followed by 15 minutes incubation at room temperature.  After addition of 250 /*! of the CT/NG DIL solution, mixing and centrifugation at 12.500 x g for 10 minutes, the crude DNA lysate was ready for PCR.<br>
PCR analysis of the DNA solution<br>
An aliquot of the crude DNA lysate was added to a multiplex PCR reaction mixture including the two primer pairs listed in Example 4. After 30 cycles PCR that was performed as described, the PCR product was subjected to capillary gel electrophoresis and Genescan software analysis (figure 4).<br><br><br><br>
WE CLAIM:<br>
1. A method of quality control using an internal control nucleic acid sequence encapsulated within liposome particles comprising the step of bringing a sample such as herein described in which a target nucleic acid is contained within a target entity into contact with said liposomes before, at the same time or during induction of release of nucleic acids, wherein the internal control is being used in a nucleic acid based analysis of said sample, wherein the internal control nucleic acid is capable of being distinguished from the target nucleic acid and wherein said liposome particles are modified to have a similar density, weight and/or size to the target entity or are modified to introduce into or onto said liposome particles, surface proteins or molecules found on said target entities, allowing said particles to be separated from the sample together with the target entities using a separation step and using the IC nucleic acid as an internal control in an analysis of the target nucleic acid.<br>
2.	Method as claimed in claim 1 wherein said modification comprises modification of all or some of the lipids comprising said liposome particles.<br>
3.	Method as claimed in claim 1 wherein the liposome particles have been modified so that the density of said particles is increased.<br>
4.	Method as claimed in claim 3 wherein the liposome particles have been modified to incorporate polysaccharides or to fill the central core with a dense solution.<br>
5.	Method as claimed in claim 4 wherein said polysaccharides are Blue Dextran or Dextran sulphate.<br>
6.	Method as claimed in claim  1  wherein said proteins or molecules allow said particles to be selectively captured.<br>
7.	Method as claimed in claim 6 wherein said particles are selectively captured by affinity separation or immunoseparation.<br>
8.	Method as claimed in any one of claims 1 to 7 wherein said liposome particles are disrupted under the same conditions which will disrupt the target entity.<br><br>
9.	Method as claimed in any one of claims 1 to 8 wherein said liposome particles are<br>
selected or modified such that said particles have a stability under assay and sample<br>
conditions which is the same as or similar to the target entities.<br>
10.	Method as claimed in claim 9 wherein the stability of the liposome particles is modified by modifying some or all of the proteins or lipids.<br>
11.	Method as claimed in claim 10 wherein the stability of the liposome particles is increased relative to the liposome particles prior to modification.<br>
12.	Method as claimed in claim 11 wherein the liposomes are coated with inert hydrophilic polymers or proteins derived from thermally stable bacteria or are modified using lipids derived from thermophilic bacteria.<br>
13.	Method as claimed in claim 12 wherein the hydrophilic polymer is PEG or the thermally stable protein is S-layer protein.<br>
14.	Method as claimed in any one of claims 1 to 13 wherein the liposome particles comprise a proportion of phospholipids which are phospholipid derivatives of polyethylene	glycol,	for	example	PEG-PE	(N-(co-methoxypoly-(oxyethylene)oxycarbonyl)-DSPE).<br>
15.	Method as claimed in any one of claims 1 to 14 wherein the liposome particles are cationic liposomes.<br>
16.	Method as claimed in any one of claims 1 to 15 wherein the liposome particles comprise one or more of the neutral lipids POPC (l-palmitoyl-2-oleoyl-sw-glycero-3-phosphocholine), and DOPE (l,2-dioleoyl-3-sn-phosphatidylethanolamine), and one or more of the positivelycharged lipids DDAB (dimethyldioctadecylammonium bromide), DOTAP (l,2-dioleoyloxy-3-(trimethylammonium)propane), or DOSPA (2,3-dioleyloxy-N- [2(spermine-carboxamido)ethyl]-N,N-dimethyl-l-propanaminiumtrifluoroacetate).<br>
17.	Method as claimed in any one of claims 1 to 16 wherein said target entity is a cell, preferably a cell derived from multicellular or unicellular organisms.<br><br>
18.	Method as claimed in any one of claims 1 to 16 wherein said target entity is a virus or bacteriophage.<br>
19.	Method as claimed in any one of claims 1 to 18 wherein said IC nucleic acid sequence is from 50 to 500 bases in length.<br>
20.	Method as claimed in any one of claims 1 to 19 wherein said nucleic acid-based analysis is a technique which involves amplification of a target nucleic acid.<br>
21.	Method as claimed in claim 20 wherein the nucleic acid-based analysis is PCR, LCR, Gap-LCR, NASBA or TMA.<br>
22.	Method as claimed in any one of claims 1 to 21 wherein the analysis is quantitative.<br>
23.	Method as claimed in any one of claims 1 to 22 wherein said particles are separated from the sample together with the target entities using centrifugation or sedimentation.<br>
24.	Method as claimed in claim 1 wherein an internal control nucleic acid sequence is used as an internal control, comprising obtaining a sample to be analysed, bringing the sample into contact with the non-viable liposome particles comprising the internal control (IC) nucleic acid sequence, inducing the release of the nucleic acid to be analysed from within the sample and release of the internal control nucleic acid from within the nonÂ¬viable particles and analysing the target nucleic acid and the IC nucleic acid, such that analysis of the IC nucleic acid functions as an internal control for analysis of the target nucleic acid.<br>
25.	Non-viable liposome particles comprising an IC nucleic acid for use in the method as claimed in claim 1 of an internal control in a nucleic acid based analysis of a sample in which a target nucleic acid is contained within a target entity, wherein the internal control nucleic acid encapsulated within said non-viable liposome particles is capable of being distinguished from the target nucleic acid and wherein said particles are modified to have a similar density, weight and/or size to the target entity filling with a density-increasing solution or wherein all or some of the lipids in said particles have been modified, to more closely resemble the target entity to allow said liposome particles to be separated from a sample together with the target entity using a separation step.<br><br>
26.       A kit comprising the non-viable liposome particles as claimed in claim 25.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTEtZGVsbnAtMjAwMy1hYnN0cmFjdC0oMzAtMDUtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">00451-delnp-2003-abstract-(30-05-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTEtZGVsbnAtMjAwMy1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">00451-delnp-2003-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTEtZGVsbnAtMjAwMy1jbGFpbXMtKDMwLTA1LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">00451-delnp-2003-claims-(30-05-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTEtZGVsbnAtMjAwMy1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">00451-delnp-2003-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTEtZGVsbnAtMjAwMy1jb3JyZXNwb25kZW5jZS1vdGhlcnMtKDMwLTA1LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">00451-delnp-2003-correspondence-others-(30-05-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTEtREVMTlAtMjAwMy1Db3JyZXNwb25kZW5jZS1PdGhlcnMtMjgtMDQtMjAwOC5wZGY=" target="_blank" style="word-wrap:break-word;">00451-DELNP-2003-Correspondence-Others-28-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTEtZGVsbnAtMjAwMy1jb3JyZXNwb25kZW5jZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">00451-delnp-2003-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTEtZGVsbnAtMjAwMy1kZXNjcmlwdGlvbiAoY29tcGxldGUpLSgzMC0wNS0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">00451-delnp-2003-description (complete)-(30-05-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTEtZGVsbnAtMjAwMy1kZXNjcmlwdGlvbiAoY29tcGxldGUpLTMwLTA1LTIwMDgucGRm" target="_blank" style="word-wrap:break-word;">00451-delnp-2003-description (complete)-30-05-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTEtZGVsbnAtMjAwMy1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">00451-delnp-2003-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTEtZGVsbnAtMjAwMy1kcmF3aW5ncy0oMzAtMDUtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">00451-delnp-2003-drawings-(30-05-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTEtZGVsbnAtMjAwMy1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">00451-delnp-2003-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTEtZGVsbnAtMjAwMy1mb3JtLTEtKDMwLTA1LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">00451-delnp-2003-form-1-(30-05-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTEtREVMTlAtMjAwMy1Gb3JtLTEtMjgtMDQtMjAwOC5wZGY=" target="_blank" style="word-wrap:break-word;">00451-DELNP-2003-Form-1-28-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTEtZGVsbnAtMjAwMy1mb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">00451-delnp-2003-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTEtZGVsbnAtMjAwMy1mb3JtLTEzLnBkZg==" target="_blank" style="word-wrap:break-word;">00451-delnp-2003-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTEtZGVsbnAtMjAwMy1mb3JtLTItKDMwLTA1LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">00451-delnp-2003-form-2-(30-05-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTEtZGVsbnAtMjAwMy1mb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">00451-delnp-2003-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTEtZGVsbnAtMjAwMy1mb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">00451-delnp-2003-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTEtZGVsbnAtMjAwMy1ncGEucGRm" target="_blank" style="word-wrap:break-word;">00451-delnp-2003-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTEtZGVsbnAtMjAwMy1vdGhlcnMtZG9jdW1lbnQtKDMwLTA1LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">00451-delnp-2003-others-document-(30-05-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTEtZGVsbnAtMjAwMy1wY3QtMjIwLnBkZg==" target="_blank" style="word-wrap:break-word;">00451-delnp-2003-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTEtZGVsbnAtMjAwMy1wY3QtNDA5LnBkZg==" target="_blank" style="word-wrap:break-word;">00451-delnp-2003-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTEtZGVsbnAtMjAwMy1wY3QtNDE2LnBkZg==" target="_blank" style="word-wrap:break-word;">00451-delnp-2003-pct-416.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTEtZGVsbnAtMjAwMy1wY3Qtc2VhcmNoIHJlcG9ydC5wZGY=" target="_blank" style="word-wrap:break-word;">00451-delnp-2003-pct-search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTEtREVMTlAtMjAwMy1QZXRpdGlvbi0xMzgtMjgtMDQtMjAwOC5wZGY=" target="_blank" style="word-wrap:break-word;">00451-DELNP-2003-Petition-138-28-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUxLURFTE5QLTIwMDMtQWJzdHJhY3QtKDAzLTAzLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">451-DELNP-2003-Abstract-(03-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUxLURFTE5QLTIwMDMtQXNzaWdubWVudC0oMDUtMDMtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">451-DELNP-2003-Assignment-(05-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUxLURFTE5QLTIwMDMtQ2xhaW1zLSgwMy0wMy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">451-DELNP-2003-Claims-(03-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUxLURFTE5QLTIwMDMtQ2xhaW1zLTIyLTA1LTIwMDgucGRm" target="_blank" style="word-wrap:break-word;">451-DELNP-2003-Claims-22-05-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUxLURFTE5QLTIwMDMtQ29ycmVzcG9uZGVuY2UtT3RoZXJzLSgwMy0wMy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">451-DELNP-2003-Correspondence-Others-(03-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUxLURFTE5QLTIwMDMtQ29ycmVzcG9uZGVuY2UtT3RoZXJzLSgwNS0wMy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">451-DELNP-2003-Correspondence-Others-(05-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUxLURFTE5QLTIwMDMtQ29ycmVzcG9uZGVuY2UtT3RoZXJzLSgyNC0wNy0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">451-DELNP-2003-Correspondence-Others-(24-07-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUxLURFTE5QLTIwMDMtQ29ycmVzcG9uZGVuY2UtT3RoZXJzLTIyLTA1LTIwMDgucGRm" target="_blank" style="word-wrap:break-word;">451-DELNP-2003-Correspondence-Others-22-05-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUxLURFTE5QLTIwMDMtRGVzY3JpcHRpb24gKENvbXBsZXRlKS0oMDMtMDMtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">451-DELNP-2003-Description (Complete)-(03-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUxLURFTE5QLTIwMDMtRGVzY3JpcHRpb24gKENvbXBsZXRlKS0yMi0wNS0yMDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">451-DELNP-2003-Description (Complete)-22-05-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUxLURFTE5QLTIwMDMtRHJhd2luZ3MtKDAzLTAzLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">451-DELNP-2003-Drawings-(03-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUxLURFTE5QLTIwMDMtRm9ybS0yLSgwMy0wMy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">451-DELNP-2003-Form-2-(03-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUxLURFTE5QLTIwMDMtRm9ybS0zLSgwMy0wMy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">451-DELNP-2003-Form-3-(03-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUxLURFTE5QLTIwMDMtRm9ybS0zLSgwNS0wMy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">451-DELNP-2003-Form-3-(05-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUxLURFTE5QLTIwMDMtRm9ybS02LSgyNC0wNy0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">451-DELNP-2003-Form-6-(24-07-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUxLURFTE5QLTIwMDMtR1BBLSgwNS0wMy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">451-DELNP-2003-GPA-(05-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUxLURFTE5QLTIwMDMtUENULTMwNC0oMDMtMDMtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">451-DELNP-2003-PCT-304-(03-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUxLURFTE5QLTIwMDMtUGV0aXRpb24tMTM3LTIyLTA1LTIwMDgucGRm" target="_blank" style="word-wrap:break-word;">451-DELNP-2003-Petition-137-22-05-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Zm9ybS0xOC50aWY=" target="_blank" style="word-wrap:break-word;">form-18.tif</a></p>
		<br>
		<div class="pull-left">
			<a href="233265-a-polypeptide-comprising-an-immunogenic-portion-of-a-prostate-protein.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="233267-a-stick-kit-or-elisa-kit-for-evaluating-the-physical-state-of-a-subject-suffering-from-a-respiratory-bacterial-infection.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>233266</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>00451/DELNP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>26-Mar-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>IC PARTICLES AS.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>SIMENSBRATVEIEN 38, 1182 OSLO, NORWAY</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>BERG, EINAR SVERRE</td>
											<td>VIMV, VIRUS DEPARTMENT, NATIONAL INSTITUTE OF PUBLIC OF HEALTH, P.O.BOX 4404 NYDALEN, 0403 OSLO, NORWAY</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SKAUG, KJELL</td>
											<td>VIMV, VIRUS DEPARTMENT, NATIONAL INSTITUTE OF PUBLIC OF HEALTH, P.O.BOX 4404 NYDALEN, 0403 OSLO, NORWAY</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/GB01/03879</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-08-30</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>0021303.3</td>
									<td>2000-08-30</td>
								    <td>U.K.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/233266-a-method-of-quality-control-using-an-internal-control-nucleic-acid-sequence-encasulated-within-liposome-particles by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:36:52 GMT -->
</html>
