<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/233399-a-recombinant-e-coli by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:41:11 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 233399:A RECOMBINANT E.COLI</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A RECOMBINANT E.COLI</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention provides an improved method for the biological production of 1,3-propanediol from a fermentable carbon source in a single microorganism. In one aspect of the present invention, an improved process for the conversion of glucose to 1,3-propanediol is achieved by the use of an E. coli transformed with the Klebsiella pneumoniae dha regulon genes dhaR, orfY, dhaT, orfX, orfW, dhaB1, dhaB2, dhaB3, and orfZ, all these genes arranged in the same genetic organization as found in wild type Klebsiella pneumoniae. In another aspect of the present invention, an improved process for the production of 1,3-propanediol from glucose using a recombinant E. coli containing genes encoding a G3PDH, a G3P phosphatase, a dehydratase, and a dehydratase reactivation factor compared to an identical process using a recombinant E. coli containing genes encoding a G3PDH, a G3P phosphatase, a dehydratase, a dehydratase reactivation factor and a 1,3-propanediol oxidoreductase (dhaT). The dramatically improved process relies on the presence in E. Coli of a gene encoding a non-specific catalytic activity sufficient to convert 3-hydroxypropionaldehyde to 1,3-propanediol.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM 2<br>
THE PATENTS ACT,   1970<br>
[39 OF 1970]<br>
&amp;<br>
THE PATENTS RULES, 2003<br>
COMPLETE SPECIFICATION<br>
[See Section 10; rule 13]<br>
" A MICROORGANISM TRANSFORMED WITH A CHIMERIC GENE"<br>
E.I. DU PONT DE NEMOURS AND COMPANY, a Delaware corporation, of 1007 Market Street, Wilmington, Delaware 19898 , United States of America, and GENENCOR INTERNATIONAL INC., of 925 Page Mill Road, Palo Alto, CA 94303, United States of America,<br>
The following specification particularly describes the nature of the invention and the manner in which it is to be performed:-<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
TITLE<br>
PROCESS FOR THE BIOLOGICAL PRODUCTION<br>
OF U-PROPANEDIOL WITH HIGH TITER<br>
FIELD OF INVENTION<br>
5	This invention comprises process for the bioconversion of a fermentable<br>
carbon source to 1,3-propanediol by a single microorganism.<br>
BACKGROUND<br>
1,3-Propanediol is a monomer having potential utility in the production of<br>
polyester fibers and the manufacture of polyurethanes and cyclic compounds.<br>
10	A variety of chemical routes to 1,3-propanediol are known. For example<br>
ethylene oxide may be converted to 1,3-propanediol over a catalyst in the presence<br>
of phosphine, water, carbon monoxide, hydrogen and an acid, by the catalytic<br>
solution phase hydration of acrolein followed by reduction, or from compounds<br>
such as glycerol, reacted in the presence of carbon monoxide and hydrogen over 15     catalysts having atoms from group VIII of the periodic table. Although it is<br>
possible to generate 1,3-propanediol by these methods, they are expensive and<br>
generate waste streams containing environmental pollutants.<br>
It has been known for over a century that 1,3-propanediol can be produced<br>
from the fermentation of glycerol. Bacterial strains able to produce <br>
20      1,3-propanediol have been found, for example, in the groups Citrobacter,<br>
Clostridium, Enterobacter, Ilyobacter, Klebsiella, Lactobacillus, and Pelobacter.<br>
In each case studied, glycerol is converted to 1,3-propanediol in a two step,<br>
enzyme catalyzed reaction sequence. In the first step, a dehydratase catalyzes the<br>
conversion of glycerol to 3-hydroxypropionaldehyde (3-HPA) and water, <br>
25      Equation 1. In the second step, 3-HPA is reduced to 1,3-propanediol by a<br>
NAD+-linked oxidoreductase, Equation 2. The 1,3-propanediol is not<br>
metabolized further and, as a result,<br>
Glycerol -► 3-HPA + H20	(Equation 1)<br>
30	3-HPA + NADH + H+ -&gt; 1,3-Propanediol + NAD+	(Equation 2)<br>
       accumulates in the media. The overall reaction consumes a reducing equivalent in<br>
        the form of a cofactor, reduced β-nicotinamide adenine dinucleotide (NADH),<br>
          which is oxidized to nicotinamide adenine dinucleotide (NAD+).<br>
35	In Klebsiella pneumonia, Citrobacter freundii, and Clostridium<br>
pasteurianum, the genes encoding the three structural subunits of glycerol dehydratase {dhaBl-3 or dhaB, C andE) are located adjacent to a gene encoding a specific 1,3-propanediol oxidoreductase (dhaT) (see Figure 1). Although the<br>
1<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
genetic organization differs somewhat among these microorganisms, these genes are clustered in a group which also comprises orfX and or/Z (genes encoding a<br>
dehydratase reactivation factor for glycerol dehydratase), as well as orfYand orfw<br>
(genes of unknown function). The specific 1,3-propanediol oxidoreductases <br>
5     	 (dhaTs) of these microorganisms are known to belong to the family of type III alcohol dehydrogenases; each exhibits a conserved iron-binding motif and has a preference for the NAD+/NADH linked interconversion of 1,3-propandiol and 3-HPA. However, the NAD+/NADH linked interconversion of 1,3-propandiol and 3-HPA is also catalyzed by alcohol dehydrogenases which are not specifically<br>
10      linked to dehydratase enzymes (for example, horse liver and baker's yeast alcohol dehydrogenases (B.C. 1.1.1.1)), albeit with less efficient kinetic parameters. Glycerol dehydratase (E.C. 4.2.1.30) and diol [1,2-propanediol] dehydratase (E.C. 4.2.1.28) are related but distinct enzymes that are encoded by distinct genes. Diol dehydratase genes from Klebsiella oxytoca and Salmonella typhimurium are<br>
15     similar to glycerol dehydratase genes and are clustered in a group which<br>
comprises genes analogous to orpC and or/Z (Daniel et al., FEMS Microbiol. Rev. 22, 553 (1999); Toraya and Mori, J. Biol. Chem. 274, 3372 (1999); GenBank AF026270).<br>
The production of 1,3-propanediol from glycerol is generally performed <br>
20	under anaerobic conditions using glycerol as the sole carbon source and in the absence of other exogenous reducing equivalent acceptors. Under these conditions, in e.g., strains of Citrobacter, Clostridium, and Klebsiella, a parallel pathway for glycerol operates which first involves oxidation of glycerol to dihydroxyacetone (DHA) by a NAD+- (or NADP+-) linked glycerol <br>
25	      dehydrogenase, Equation 3. The DHA, following phosphorylation to dihydroxyacetone phosphate (DHAP) by a DHA kinase (Equation 4),<br>
Glycerol + NAD+ -&gt; DHA + NADH + H+	(Equation 3)<br>
DHA + ATP -* DHAP + ADP	(Equation 4)<br>
30<br>
becomes available for biosynthesis and for supporting ATP generation via e.g.,<br>
glycolysis. In contrast to the 1,3-propanediol pathway, this pathway may provide carbon and energy to the cell and produces rather than consumes NADH.<br>
In Klebsiella pneumoniae and Citrobacter freundii, the genes encoding the<br>
3 5     functionally linked activities of glycerol dehydratase {dhaB), 1,3-propanediol oxidoreductase (dhaT), glycerol dehydrogenase (dhaD), and dihydroxyacetone kinase (dhaK) are encompassed by the dha regulon. The dha regulon, in Klebsiella pneumoniae and Citrobacter freundii, also encompasses a gene<br>
2<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
encoding a transcriptional activator protein (dhaR). The dha regulons from Citrobacter and Klebsiella have been expressed in Escherichia coli and have been<br>
shown to convert glycerol to 1,3-propanediol.<br>
Neither the chemical nor biological methods described above for the <br>
5 	    production of 1,3-propanediol are well suited for industrial scale production since the chemical processes are energy intensive and the biological processes are limited to relatively low titer from the expensive starting material, glycerol. These drawbacks could be overcome with a method requiring low energy input and an inexpensive starting material such as carbohydrates or sugars, or by<br>
10   	  increasing the metabolic efficiency of a glycerol process. Development of either method will require the ability to manipulate the genetic machinery responsible for the conversion of sugars to glycerol and glycerol to 1,3-propanediol.<br>
Biological processes for the preparation of glycerol are known. The overwhelming majority of glycerol producers are yeasts but some bacteria, other<br>
15      fungi and algae are also known. Both bacteria and yeasts produce glycerol by converting glucose or other carbohydrates through the fructose-1,6-bisphosphate pathway in glycolysis or the Embden Meyerhof Parnas pathway, whereas, certain algae convert dissolved carbon dioxide or bicarbonate in the chloroplasts into the 3-carbon intermediates of the Calvin cycle. In a series of steps, the 3-carbon<br>
20      intermediate, phosphoglyceric acid, is converted to glyceraldehyde 3-phosphate which can be readily interconverted to its keto isomer dihydroxyacetone phosphate and ultimately to glycerol.<br>
Specifically, the bacteria Bacillus licheniformis and Lactobacillus lycopersica synthesize glycerol, and glycerol production is found in the<br>
25      halotolerant algae Dunaliella sp. and Asteromonas gracilis for protection against high external salt concentrations. Similarly, various osmotolerant yeasts synthesize glycerol as a protective measure. Most strains of Saccharomyces produce some glycerol during alcoholic fermentation, and this can be increased physiologically by the application of osmotic stress. Earlier this century<br>
30     commercial glycerol production was achieved by the use of Saccharomyces cultures to which "steering reagents" were added such as sulfites or alkalis. Through the formation of an inactive complex, the steering agents block or inhibit the conversion of acetaldehyde to ethanol; thus, excess reducing equivalents (NADH) are available to or "steered" towards DHAP for reduction to produce<br>
35    glycerol. This method is limited by the partial inhibition of yeast growth that is due to the sulfites. This limitation can be partially overcome by the use of alkalis that create" excess NADH equivalents by a different mechanism. In this practice,<br>
3<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
the alkalis initiated a Cannizarro disproportionation to yield ethanol and acetic acid from two equivalents of acetaldehyde.<br>
The gene encoding glycerophosphate dehydrogenase (DARl, QPD1)<br>
has been cloned and sequenced from S. diastaticus (Wang et al., J. Bact. 176,<br>
5     	 7091-7095 (1994)). The DARl gene was cloned into a shuttle vector and used to transform E. coli where expression produced active enzyme. Wang et al. {supra) recognize that DARl is regulated by the cellular osmotic environment but do not suggest how the gene might be used to enhance 1,3-propanediol production in a recombinant microorganism.<br>
10	Other glycerol-3-phosphate dehydrogenase enzymes have been isolated:<br>
for example, sn-glycerol-3-phosphate dehydrogenase has been cloned and sequenced from Saccharomyces cerevisiae (Larason et al., Mol. Microbiol. 10, 1101 (1993)) and Albertyn et al. (Mo/. Cell. Biol. 14, 4135 (1994)) teach the cloning of GPD1 encoding a glycerol-3-phosphate dehydrogenase from<br>
15     Saccharomyces cerevisiae. Like Wang et al. {supra), both Albertyn et al. and Larason et al. recognize the osmo-sensitivity of the regulation of this gene but do not suggest how the gene might be used in the production of 1,3-propanediol in a recombinant microorganism.<br>
As with G3PDH, glycerol-3-phosphatase has been isolated from<br>
20     Saccharomyces cerevisiae and the protein identified as being encoded by the<br>
GPP1 and GPP2 genes (Norbeck et al., J. Biol. Chem. 271, 13875 (1996)). Like<br>
the genes encoding G3PDH, it appears that GPP2 is osmosensitive.<br>
Although a single microorganism conversion of fermentable carbon source other than glycerol or dihydroxyacetone to 1,3-propanediol is desirable, it has<br>
25      been documented that there are significant difficulties to overcome in such an endeavor. For example, Gottschalk et al. (EP 373 230) teach that the growth of most strains useful for the production of 1,3-propanediol, including Citrobacter freundii, Clostridium autobutylicum, Clostridium butylicum, and Klebsiella pneumoniae, is disturbed by the presence of a hydrogen donor such as fructose or<br>
30      glucose. Strains of Lactobacillus brevis and Lactobacillus buchner, which<br>
produce 1,3-propanediol in co-fermentations of glycerol and fructose or glucose, do not grow when glycerol is provided as the sole carbon source, and, although it has been shown that resting cells can metabolize glucose or fructose, they do not produce 1,3-propanediol (Veiga DA Cunha et al., J. Bacteriol, 174, 1013 (1992)).<br>
35    Similarly, it has been shown that a strain of Ilyobacter polytropus, which produces<br>
1,3-propanediol when glycerol and acetate are provided, will not produce 1,3-propanediol from carbon substrates other than glycerol, including fructose and<br>
4<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
glucose (Steib et al., Arch. Microbiol. 140,139 (1984)). Finally, Tong et al. (AppL Biochem. Biotech. 34, 149 (1992)) taught that recombinant Escherichia coli<br>
transformed with the dha regulon encoding glycerol dehydratase does not produce<br>
1,3-propanediol from either glucose or xylose in the absence of exogenous<br>
 5     glycerol.<br>
Attempts to improve the yield of 1,3-propanediol from glycerol have been reported where co-substrates capable of providing reducing equivalents, typically fermentable sugars, are included in the process. Improvements in yield have been claimed for resting cells of Citrobacter freundii and Klebsiella pneumoniae DSM<br>
10     4270 co-fermenting glycerol and glucose (Gottschalk et al., supra.; and Tran-Dinh et al., DE 3734 764); but not for growing cells of Klebsiella pneumoniae ATCC 25955 co-fermenting glycerol and glucose, which produced no 1,3-propanediol (I-T. Tong, Ph.D. Thesis, University of Wisconsin-Madison (1992)). Increased yields have been reported for the cofermentation of glycerol<br>
15      and glucose or fructose by a recombinant Escherichia coli; however, no<br>
1,3-propanediol is produced in the absence of glycerol (Tong et al., supra.). In these systems, single microorganisms use the carbohydrate as a source of generating NADH while providing energy and carbon for cell maintenance or growth. These disclosures suggest that sugars do not enter the carbon stream that<br>
20     produces 1,3-propanediol.<br>
Recently, however, the conversion of carbon substrates, other than glycerol or dihydroxyacetone, to 1,3-propanediol by a single microorganism that expresses a dehydratase enzyme has been described (U.S. 5,686,276; WO 9821339; WO 9928480; and WO 9821341 (US 6013494)). A specific<br>
25      deficiency in the biological processes leading to the production of 1,3-propanediol from either glycerol or glucose has been the low titer of the product achieved via fermentation; thus, an energy-intensive separation process to obtain 1,3-propanediol from the aqueous fermentation broth is required. Fed batch or batch fermentations of glycerol to 1,3-propanediol have led to final titers of 65 g/L<br>
30     by Clostridium butyricum (Saint-Amans et al., Biotechnology Letters 7
(1994)), 71 g/L by Clostridium butyricum mutants (Abbad-Andaloussi et a\.,Appl. Environ. Microbiol. 61,4413 (1995)), 61 g/L by Klebsiella pneumoniae (Homann et al., AppL Bicrobiol. Biotechnol. 33,121 (1990)), and 35 g/L by Citrobacter freundii (Homann et al., supra). Fermentations of glucose to 1,3-propanediol that<br>
3 5    exceed the titer obtained from glycerol fermentations have not yet been disclosed. The problem that remains to be solved is how to biologically produce 1,3-propanediol, with high titer and by a single microorganism, from an<br>
5<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
inexpensive carbon substrate such as glucose or other sugars. The biological production of 1,3-propanediol requires glycerol as a substrate for a two-step<br>
sequential reaction in which a dehydratase enzyme (typically a coenzyme<br>
B12-dependent dehydratase) converts glycerol to an intermediate, <br>
5       3-hydroxypropionaldehyde, which is then reduced to 1,3-propanediol by a<br>
NADH- (or NADPH) dependent oxidoreductase. The complexity of the cofactor requirements necessitates the use of a whole cell catalyst for an industrial process that utilizes this reaction sequence for the production of 1,3-propanediol.<br>
SUMMARY OF THE INVENTION<br>
10	Applicants have solved the stated problem and the present invention<br>
provides for bioconverting a fermentable carbon source directly to 1,3-propanediol at significantly higher titer than previously obtained and with the use of a single microorganism. Glucose is used as a model substrate and E. coli is used as the model host. In one aspect of this invention, recombinant E. coli<br>
15      expressing a group of genes (comprising genes that encode a dehydratase activity, a dehydratase reactivation factor, a 1,3-propanediol oxidoreductase [dhaT), a glycerol-3-phosphate dehydrogenase, and a glycerol-3-phosphatase) convert glucose to 1,3-propanediol at titer that approaches that of glycerol to 1,3-propanediol fermentations.<br>
20	In another aspect of this invention, the elimination of the functional dhaT<br>
gene in this recombinant E. coli results in a significantly higher titer of 1,3-propanediol from glucose. This unexpected increase in titer results in improved economics, and thus, an improved process for the production of 1,3-propanediol from glucose.<br>
25	Furthermore, the present invention may be generally applied to include<br>
any carbon substrate that is readily converted to 1) glycerol, 2) dihydroxyacetone,<br>
3)	C3 compounds at the oxidation state of glycerol (e.g., glycerol 3-phosphate), or<br>
4)	C3 compounds at the oxidation state of dihydroxyacetone (e.g.,<br>
dihydroxyacetone phosphate or glyceraldehyde 3-phosphate). The production of <br>
30	1,3-propanediol in the dhaT minus strain requires a non-specific catalytic activity that converts 3-HPA to 1,3-propanediol. Identification of the enzyme(s) and/or gene(s) responsible for the non-specific catalytic activity that converts 3-HPA to 1,3-propanediol will lead to production of 1,3-propanediol in a wide range of host microorganisms with substrates from a wide range of carbon-containing <br>
35	      substrates. It is also anticipated that the use of this non-specific catalytic activity that converts 3-HPA to 1,3-propanediol will lead to an improved process for the<br>
6<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
production of 1,3-propanediol from glycerol or dihydroxyacetone, by virtue of an improved titer and the resulting improved economics.<br>
This activity has been isolated from E. coli as a nucleic acid fragment<br>
encoding a non-specific catalytic activity for the conversion of 3-<br>
5    	 hydroxypropionaldehyde to 1,3-propanediol, as set out in SEQ ID NO:58 or as selected from the group consisting of:<br>
(a)	an isolated nucleic acid fragment encoding all or a substantial<br>
portion of the amino acid sequence of SEQ ID NO:57;<br>
(b)	an isolated nucleic acid fragment that is substantially similar to<br>
10	an isolated nucleic acid fragment encoding all or a substantial<br>
portion of the amino acid sequence of SEQ ID NO:57;<br>
(c)	an isolated nucleic acid fragment encoding a polypeptide of at<br>
least 387 amino acids having at least 80% with the amino acid<br>
sequence of SEQ ID NO:57;<br>
15	(d)   an isolated nucleic acid fragment that hybridizes with (a) under<br>
hybridization conditions of 0.1X SSC, 0.1% SDS, 65 °C and washed with 2X SSC, 0.1% SDS followed by 0.1X SSC, 0.1% SDS; and<br>
(d)	an isolated nucleic acid fragment that is complementary to (a),<br>
20	(b), (c), or (d). Alternatively, the nonspecific catalytic acitivity<br>
is embodieed in the polypeptide as set out in SEQ ID NO:57. A chimeric gene may be constructed comprising the isolated nucleic acid fragment described above operably linked to suitable regulatory sequences. This chimeric gene can be used to transform miciroorganisms selected from the group<br>
25      consisting of Citrobacter, Enterobacter, Clostridium, Klebsiella, Aerobacter, Lactobacillus, Aspergillus, Saccharomyces, Schizosaccharomyces, Zygosaccharomyces, Pichia, Kluyveromyces, Candida, Hansenula, Debaryomyces, Mucor, Torulopsis, Methylobacter, Salmonella, Bacillus, Aerobacter, Streptomyces, Escherichia, and Pseudomonas. E. coli is the<br>
30      preferred host.<br>
Accordingly, the present invention provides a recombinant microorganism, useful for the production of 1,3-propanediol comprising: (a) at least one gene encoding a polypeptide having glycerol-3-phosphate dehydrogenase activity; (b) at least one gene encoding a polypeptide having glycerol-3-phosphatase<br>
3 5     activity; (c) at least one gene encoding a polypeptide having a dehydratase<br>
activity; (d) at least one gene encoding a dehydratase reactivation factor, (e) at least one endogenous gene encoding an non-specific catalytic activity sufficient to<br>
7<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
convert 3-hydroxypropionaldehyde to 1,3-propanediol, wherein no functional dhaT gene encoding a 1,3-propanediol oxidoreductase is present. The preferred<br>
embodiment is a recombinant microorganism (preferably E. coli) where no dhaT<br>
gene is present. Optionally, the recombinant microorganism may comprise <br>
5    	 mutations (e.g., deletion mutations or point mutations) in endogenous genes selected from the group consisting of: (a) a gene encoding a polypeptide having glycerol kinase activity; (b) a gene encoding a polypeptide having glycerol dehydrogenase activity; and (c) gene encoding a polypeptide having triosephosphate isomerase activity.<br>
10	In another embodiment the invention includes a process for the production<br>
of 1,3-propanediol comprising:(a) contacting, under suitable conditions, a recombinant E. coli comprising a dha regulon and lacking a functional dhaT gene encoding a 1,3-propanediol oxidoreductase activity with at least one carbon source, wherein the carbon source is selected from the group consisting of<br>
15     monosaccharides, oligosaccharides, polysaccharides, and single-carbon substrates; and (b) optionally recovering the 1,3-propanediol produced in (a). The invention also provides a process for the production of 1,3-propanediol from a recombinant microorganism comprising: (a) contacting the recombinant microorganism of the present invention with at least one carbon<br>
20     source selected from the group consistmg of monosaccharides, oligosaccharides, polysaccharides, and single-carbon substrates whereby 1,3-propanediol is produced; and (b) optionally recovering the 1,3-propanediol produced in (a).<br>
Similarly the invention intends to provide a process for the production of 1,3-propanediol from a recombinant microorganism comprising:<br>
25	(a) contacting a recombinant microorganism with at least one carbon<br>
source, said recombinant microorganism comprising:<br>
(i)      at least one gene encoding a polypeptide having a dehydratase<br>
activity;<br>
(ii)       at least one gene encoding a dehydratase reactivation factor;<br>
30	(iii)     at least one endogenous gene encoding a non-specific catalytic<br>
activity sufficient to convert 3-hydroxypropionaldehyde to 1,3-propanediol; wherein no functional dhaT gene encoding a 1,3-propanediol oxidoreductase is present;<br>
said carbon source selected from the group consisting of glycerol and<br>
35	dihydroxyacetone, wherein 1,3-propanediol is produced and;<br>
(b) optionally recovering the 1,3-propanediol produced in (a).<br>
8<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
Yet another aspect of the invention provides for the co-feeding of the carbon substrate. In this embodiment for the production of 1,3-propanediol, the<br>
steps are: (a) contacting a recombinant E. coli with a first source of carbon and<br>
with a second source of carbon, said recombinant E. coli comprising: (i) at least <br>
5    	 one exogenous gene encoding a polypeptide having a dehydratase activity; (ii) at least one exogenous gene encoding a dehydratase reactivation factor; (iii) at least one exogenous gene encoding a non-specific catalytic activity sufficient to convert 3-hydroxypropionaldehyde to 1,3-propanediol, wherein no functional dhaTgene encoding a 1,3-propanediol oxidoreductase activity is present in the recombinant<br>
10     E. coli and wherein said first carbon source is selected from the group consisting of glycerol and dihydroxyacetone, and said second carbon source is selected from the group consisting of monosaccharides, oligosaccharides, polysaccharides, and single-carbon substrates, and (b) the 1,3-propanediol produced in (a) is optionally recovered. The co-feed may be sequential or simultaneous. The recombinant<br>
15     E. coli used in a co-feeding embodiemtn may further comprise: (a) a set of<br>
exogenous genes consisting of (i) at least one gene encoding a polypeptide having glycerol-3-phosphate dehydrogenase activity; (ii) at least one gene encoding a polypeptide having glycerol-3-phosphatase activity; and (iii) at least one subset of genes encoding the gene products of dhaR, orfY, or/X, orJW, dhaBl, dhaB2,<br>
20      dhaB3 and orfZ, and (b) a set of endogenous genes, each gene having a mutation inactivating the gene, the set consisting of: (i) a gene encoding a polypeptide having glycerol kinase activity; (ii) a gene encoding a polypeptide having glycerol dehydrogenase activity; and (iii) a gene encoding a polypeptide having triosephosphate isomerase activity.<br>
25	Useful recombinant E. coli strains include recombinant E. coli strain<br>
KLP23 comprising: (a) a set of two endogenous genes, each gene having a mutation inactivating the gene, the set consisting of: (i) a gene encoding a polypeptide having a glycerol kinase activity; and (ii) a gene encoding a polypeptide having a glycerol dehydrogenase activity; (b) at least one exogenous<br>
30     gene encoding a polypeptide having glycerol-3-phosphate dehydrogenase activity; (c) at least one exogenous gene encoding a polypeptide having glycerol73-phosphatase activity; and (d) a plasmid pKP32 and a recombinant E. coli strain RJ8 comprising: (a) set of three endogenous genes, each gene having a mutation inactivating the gene, the set consisting of: (i) a gene encoding a polypeptide<br>
35     having a glycerol kinase activity; (ii) a gene encoding a polypeptide having a<br>
glycerol dehydrogenase activity; and (iii) a gene encoding a polypeptide having a triosephosphate isomerase activity.<br>
9<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
Other useful embodiments include recombinant E. coli comprising: (a) a set of exogenous genes consisting of: (i) at least one gene encoding a polypeptide having a dehydratase activity; (ii) at least one gene encoding a polypeptide having<br>
glycerol-3-phosphate dehydrogenase activity; (iii) at least one gene encoding a <br>
5    	 polypeptide having glycerol-3-phosphatase activity; and (iv) at least one gene encoding a dehydratase reactivation factor; and (b) at least one endogenous gene encoding a non-specific catalytic activity to convert 3-hydroxypropionaldehyde to 1,3-propanediol; wherein no functional dhaT gene encoding a 1,3-propanediol oxidoreductase activity is present in the recombinant E. coli.<br>
10	Another embodiemtn is a recombinant E. coli comprising: (a) a set of<br>
exogenous genes consisting of (i) at least one gene encoding a polypeptide having glycerol-3-phosphate dehydrogenase activity; (ii) at least one gene encoding a polypeptide having glycerol-3-phosphatase activity; and (iii) at least one subset of genes encoding the gene products of dhaR, orfY, orjX, orJW, dhaBl, dhaB2,<br>
15      dhaB3 and orfZ, and (b) at least one endogenous gene encoding a non-specific catalytic activity to convert 3-hydroxypropionaldehyde to 1,3-propanediol, wherein no functional dhaT gene encoding a 1,3-propanediol oxidoreductase<br>
activity is present in the recombinant E. coli. This embodiment also includes a<br>
process using a   recombinant E. coli further comprising a set of endogenous<br>
20     genes, each gene having a mutation inactivating the gene, the set consisting of: (a) a gene encoding a polypeptide having glycerol kinase activity; (b) a gene encoding a polypeptide having glycerol dehydrogenase activity; and (c) a gene encoding a polypeptide having triosephosphate isomerase activity.<br>
This embodiment still further includes a process for the bioproduction of<br>
25      1,3-propanediol comprising: (a) contacting under suitable conditions the immediately disclosed recombinant E. coli with at least one carbon source selected from the group consisting of monosaccharides, oligosaccharides, polysaccharides, and single-carbon substrates whereby 1,3-propanediol is produced; and (b) optionally recovering the 1,3-propanediol produced in (a).<br>
30	And also includes a further process for the bioproduction of<br>
1,3-propanediol comprising: (a) contacting the recombinant E. coli of the immediately disclosed embodiments that further comprise: (i) at least one exogenous gene encoding a polypeptide having a dehydratase activity; (ii) at least one exogenous gene encoding a dehydratase reactivation factor; (iii) at least one<br>
35     endogenous gene encoding a non-specific catalytic activity to convert 3-hydroxypropionaldehyde to 1,3-propanediol, with at least one carbon source selected from<br>
10<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
the group consisting of glycerol and dihydroxyacetone, and (b) optionally recovering the 1,3-propanediol produced in (a).<br>
BRIEF DESCRIPTION OF THE DRAWINGS.<br>
SEQUENCE DESCRIPTIONS, AND BIOLOGICAL DEPOSITS<br>
5	The invention can be more fully understood from the following detailed<br>
description, Figures, the accompanying sequence descriptions, and biological deposits that form parts of this application.<br>
Figure 1 presents the gene organization within the sequence of the dha regulon subclone pHK28-26.<br>
10	Figure 2 presents a graph of the extracellular soluble protein (g/L)<br>
compared between two fermentations runs essentially as described in Example 7 using a constant feed of vitamin B^. In one case, solid lines, the strain used was KLP23/pAH48/pKP32. In the other case, dashed lines, the strain used was KLP23/pAH48/pDT29.<br>
15	Figure 3 presents a graph of the cell viability [(viable cells/mL)/OD550]<br>
compared between two fermentations runs essentially as described in Example 7 using a constant feed of vitamin B12. In one case (solid lines), the strain used was KLP23/pAH48/pKP32. In the other case (dashed lines), the strain used was KLP23/pAH48/pDT29.<br>
20	Figure 4 presents a graph of the yield of glycerol from glucose compared<br>
between two fermentations runs essentially as described in Example 7, but in the absence of vitamin B!2 or coenzyme B12. In one case (solid lines), the strain used was KLP23/pAH48/pKP32. In the other case (dashed lines), the strain used was KLP23/pAH48/pDT29.<br>
25	Figure 5 is a flow diagram illustrating the metabolic conversion of glucose<br>
to 1,3-propanediol.<br>
Figure 6 is a 2D-PAGE membrane blot with the soluble protein fraction extracted from a band showing endogenous E. coli oxidoreductase activity (non-specific catalytic activity) on a native gel.<br>
30	The 68 sequence descriptions and the sequence listing attached hereto will<br>
comply with the rules governing nucleotide and/or amino acid sequence disclosures in patent applications as set forth in 37 C.F.R. §1.821-1.825 ("Requirements for Patent Applications Containing Nucleotide Sequences and/or<br>
Amino Acid Sequence Disclosures - the Sequence Rules") and will be consistent <br>
35     	with World Intellectual Property Organization (WIPO) Standard ST2.5 (1998) and the sequence listing requirements of the EPO and PCT (Rules 5.2 and 49.5(a-bis), and Section 208 and Annex C of the Administration Instructions). The Sequence<br>
11<br><br>
10<br><br>
Descriptions contain the one letter code for nucleotide sequence characters and the<br>
three letter codes for amino acids as defined in conformity with the IUPAC-IYUB standards described in Nucleic Acids Res. 13, 3021-3030 (1985) and in the BiochemicalJournal 219, 345-373 (1984) which are herein incorporated by reference.<br>
SEQ ID NO:l contains the nucleotide sequence determined from a 12.1 kb EcoRI-Sall fragment from pKPl (cosmid containing DNA from Klebsiella pneumoniae), subcloned into pIBI31 (IBI Biosystem, New Haven, CT), and termed pHK28-26. Table 1 further details genes, corresponding base pairs identified within SEQ ID NO: 1, and associated functionality. See also Example 1.<br><br>
-SEQ ID NO:57 contains the amino acid sequence determined for YqhD<br>
SEQ ID NO:58 contains the nucleotide sequence determined for yqhD<br>
Applicants have made the following biological under the terms of the Budapest Treaty on the International Recognition of the Deposit of Micro-organisms for the Purposes of Patent Procedure:-<br><br>
20<br>
23<br><br>
Depositor Identification<br>
	Reference	<br>
Transformed E. coli DH5cc containing a portion of the Klebsiella genome encoding the glycerol dehydratase enzyme<br>
transformed E. coli DH5α containing cosmit pKP4 containing a portion of Klebsiella genome encoding a diol dehydratase enzyme<br>
E. coli MSP33.6<br><br><br>
Date of Deposit<br>
Int'l Depository Designation<br>
ATCC 69789 ,   18 April 1995<br>
ATCC 69790        18 April 1995<br>
ATCC 98598       25 November 1997<br><br><br><br>
)      glpK mutant E. coli RJF1 0m<br><br>
ATCC 98597      25 November 1997<br><br><br><br>
35<br>
40<br><br>
The deposit(s) will be maintained in the indicated international depository for at least 30 years and will be made available to the public upon the grant of a patent disclosing it. The availability of a deposit does not constitute a license to practice the subject invention in derogation of patent rights granted by government action.<br>
As used herein, "ATCC" refers to the American Type Culture Collection international depository located 10801 University Blvd., Manassas, VA 20110-2209 U.S.A. The "ATCC No." is the accession number to cultures on deposit with the ATCC<br><br><br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
DETAILED DESCRIPTION OF THE INVENTION <br>
The present invention provides for an improved process for bioconverting<br>
a fermentable carbon source directly to 1,3-propanediol using a single<br>
microorganism. The method is characterized by improved titer, yield, and cell <br>
5   	  viability as well as a decrease in cell lysis during fermentation.<br>
The present invention is based, in part, upon the observation that 1,3-propanediol fermentation processes comprising 1,3-propanediol oxidoreductase (dhaT) are characterized by high levels of 3HPA and other aldehydes and ketones in the medium, which is correlated to a decrease in cell<br>
10   	  viability. The present invention is also based, in part, upon the unexpected finding that the model host, E. coli, is capable of converting 3-HPA to 1,3-propanediol by an endogenous non-specific catalytic activity capable of converting 3-hydroxypropionaldehyde to 1,3-propanediol. The present invention is further based, in part, upon the unexpected finding that an E. coli fermentation<br>
15     process comprising this non-specific catalytic activity and lacking a functional<br>
dhaT results in increased cell viability during fermentation and provides for higher titers and/or yields of 1,3-propanediol than a fermentation process comprising a functional dhaT.<br>
In one aspect, glycerol is a model substrate, the host microorganism has a<br>
20     mutation in wild-type dhaT such that there is no 1,3-propanediol oxidoreductase activity and comprises a non-specific catalytic activity sufficient to convert 3-hydroxypropionaldehyde to 1,3-propanediol. In another aspect, glucose is a model substrate and recombinant E. coli is a model host. In this aspect, E. coli comprises an endogenous non-specific catalytic activity sufficient to convert<br>
25     3-hydroxypropionaldehyde to 1,3-propanediol. In one embodiment, the nonspecific catalytic activity is an alcohol dehydrogenase.<br>
In one aspect, the present invention provides a recombinant E. coli expressing a group of genes comprising (a) at least one gene encoding a polypeptide having glycerol-3-phosphate dehydrogenase activity; (b) at least one<br>
30     gene encoding a polypeptide having glycerol-3-phosphatase activity; (c) at least one gene encoding a polypeptide having a dehydratase activity; (d) at least one gene encoding a dehydratase reactivation factor; and (e) at least one endogenous gene encoding an non-specific catalytic activity sufficient to convert 3-hydroxypropionaldehyde to 1,3-propanediol; use of this microorganism converts glucose<br>
35     to 1,3-propanediol at a high titer. In another aspect of this invention, the<br>
elimination of the functional dhaT gene in this recombinant E. coli provides an<br>
13<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
unexpectedly higher titer of 1,3-propanediol from glucose than previously attained.<br>
The present invention provides an improved method for the biological production of 1,3-propanediol from a fermentable carbon source in a single <br>
5   	   microorganism. In one aspect of the present invention, an improved process for the conversion of glucose to 1,3-propanediol is achieved by the use of a recombinant microorganism comprising a host E. coli transformed with the Klebsiella pneumoniae dha regulon genes dhaR, or/Y, dhaT, orfX, orfW, dhaBl, dhaB2, dhaB3, and orfZ, all these genes arranged in the same genetic organization<br>
10  	   as found in wild type Klebsiella pneumoniae. The titer obtained for the<br>
fermentation process is significantly higher than any titer previously reported for a similar fermentation. This improvement relies on the use of the plasmid pDT29 as described in Example 6 and Example 7.<br>
In another aspect of the present invention, a further improved process for<br>
15     the production of 1,3-propanediol from glucose is achieved using a recombinant E. coli containing genes encoding a G3PDH, a G3P phosphtase, a dehydratase, and a dehydratase reactivation factor compared to a process using a recombinant E. coli containing genes encoding a G3PDH, a G3P phosphatase, a dehydratase, a dehydratase reactivation factor, and also a functional dhaT. The dramatically<br>
20     improved process relies on an endogenous gene encoding a non-specific catalytic activity, expected to be an alcohol dehydrogenase, which is present in E. coli. The dramatic improvement in the process is evident as an increase in<br>
1,3-propanediol titer as illustrated in Examples 7 and 9. The improvement in the process is also evident as a decrease in cell lysis as determined by the extracellular<br>
25     soluble protein concentration in the fermentation broth. This aspect of the<br>
invention is illustrated in Figure 2. Additionally, the improvement in the process is evident as prolonged cell viability over the course of the fermentation. This aspect of the invention is illustrated in Figure 3. Furthermore, the improvement in the process is also evident as an increase in yield. In E. coli expressing a<br>
30      1,3-propanediol oxidoreductase (dhaT) (for example, E. coli KLP23 transformed with the plasmid pDT29), glycerol can be metabolized to a product other than 3-HPA. In direct contrast, in E. coli not expressing a 1,3-propanediol oxidoreductase (dhaT) (for example, E. coli KLP23 transformed with the plasmid pKP32), glycerol is not metabolized to a product other than 3-HPA. That this<br>
3 5     cryptic pathway is attributable to the presence or absence of a functional dhaT is demonstrated by the lower yield of glycerol from glucose as illustrated in Figure 4.<br>
14<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
As used herein the following terms may be used for interpretation of the claims and specification.<br>
The terms "glycerol-3-phosphate dehydrogenase" and "G3PDH" refer to a<br>
polypeptide responsible for an enzyme activity that catalyzes the conversion of <br>
5     	dihydroxyacetone phosphate (DHAP) to glycerol-3-phosphate (G3P). In vivo G3PDH may be NADH; NADPH; or FAD-dependent. When specifically referring to a cofactor specific glycerol-3-phosphate dehydrogenase, the terms "NADH-dependent glycerol-3-phosphate dehydrogenase", "NADPH-dependent glycerol-3-phosphate dehydrogenase" and "FAD-dependent glycerol-3-phosphate<br>
10     dehydrogenase" will be used. As it is generally the case that NADH-dependent and NADPH-dependent glycerol-3-phosphate dehydrogenases are able to use NADH and NADPH interchangeably (for example by the gene encoded by gpsA), the terms NADH-dependent and NADPH-dependent glycerol-3-phosphate dehydrogenase will be used interchangeably. The NADH-dependent enzyme<br>
15     (EC 1.1.1.8) is encoded, for example, by several genes including GPDl (GenBank Z74071x2), or GPD2 (GenBank Z35169xl), or GPD3 (GenBank G984182), or DAR1 (GenBank Z74071 x2). The NADPH-dependent enzyme (EC 1.1.1.94) is encoded by gpsA (GenBank U321643, (cds 197911-196892) G466746 and L45246). The FAD-dependent enzyme (EC 1.1.99.5) is encoded by GUT2<br>
20      (GenBank Z47047x23), or glpD (GenBank G147838), or glpABC (GenBank<br>
M20938) (see WO 9928480 and references therein, which are herein incorporated by reference).<br>
The terms "glycerol-3-phosphatase", "sn-glycerol-3-phosphatase", or "d,l-glycerol phosphatase", and "G3P phosphatase" refer to a polypeptide<br>
25      responsible for an enzyme activity that catalyzes the conversion of glycerol-3-phosphate and water to glycerol and inorganic phosphate. G3P phosphatase is encoded, for example, by GPP1 (GenBank Z47047xl25), or GPP2 (GenBank Ul 8813x11) (see WO 9928480 and references therein, which are herein incorporated by reference).<br>
30	The term "glycerol kinase" refers to a polypeptide responsible for an<br>
enzyme activity that catalyzes the conversion of glycerol and ATP to glycerol-3-phosphate and ADP. The high-energy phosphate donor ATP may be replaced by physiological substitutes (e.g., phosphoenolpyruvate). Glycerol kinase is encoded, for example, by GUT1 (GenBank Ul 1583x19) and glpK (GenBank<br>
35     L19201) (see WO 9928480 and references therein, which are herein incorporated by reference).<br>
15<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
The term "glycerol dehydrogenase" refers to a polypeptide responsible for an enzyme activity that catalyzes the conversion of glycerol to dihydroxyacetone (EC. 1.1.1.6) or glycerol to glyceraldehyde (E.C. 1.1.1.72). A polypeptide<br>
          responsible for an enzyme activity that catalyzes the conversion of glycerol to<br>
5         dihydroxyacetone is also referred to as a "dihydroxyacetone reductase". Glycerol<br>
                 dehydrogenase may be dependent uponNADH (E.C. 1.1.1.6), NADPH<br>
                (E.C. 1.1.1.72), or other cofactors (e.g., E.C. 1.1.99.22). A NADH-dependent<br>
         glycerol dehydrogenase is encoded, for example, by gldA (GenBank U00006) (see<br>
         WO 9928480 and references therein, which are herein incorporated by reference).<br>
10	The term "dehydratase enzyme" or "dehydratase" will refer to any enzyme<br>
activity that catalyzes the conversion of a glycerol molecule to the product 3-hydroxypropionaldehyde. For the purposes of the present invention the dehydratase enzymes include a glycerol dehydratase (E.C. 4.2.1.30) and a diol<br>
dehydratase (E.C. 4.2.1.28) having preferred substrates of glycerol and <br>
15	1,2-propanediol, respectively. Genes for dehydratase enzymes have been identified in Klebsiella pneumoniae, Citrobacter freundii, Clostridium pasteurianum, Salmonella typhimurium, and Klebsiella oxytoca^ In each case, the dehydratase is composed of three subunits: the large or "a" subunit, the medium or "B" subunit, and the small or "β" subunit. Due to the wide variation in gene <br>
20	      nomenclature used in the literature, a comparative chart is given in Table 1 to<br>
facilitate identification. The genes are also described in, for example, Daniel et al. (FEMS Microbiol. Rev. 22, 553 (1999)) and Toraya and Mori (J. Biol. Chem. 274, 3372 (1999)). Referring to Table 1, genes encoding the large or "a" subunit of glycerol dehydratase include dhaBl, gldA and dhaB; genes encoding the medium <br>
25    	  or "β" subunit include dhaB2, gldB, and dhaC; genes encoding the small or "y" subunit include dhaB3, gldC, and dhaE. Also referring to Table 1, genes encoding the large or "α" subunit of diol dehydratase include pduC and pddA; genes encoding the medium or "β" subunit include pduD and pddB; genes encoding the small or "y" subunit include pduE and pddC.<br><br><br><br><br><br><br>
16<br><br>
Table 1: Comparative chart of gene names and GenBank references for dehydratase and GENE FUNCTION:	dehydratase linked functions.	<br>
	regulatory	unknown	reactivation	1,3-PD dehydrogenase	unknown<br>
	gene	base pairs	gene	base pairs	geme	base pairs	gene	base pairs	gene	base pairs<br>
ORGANISM (GenBank Reference)										<br>
AC pneumoniae (SEQ ID NO.l)	dhaR	2209-4134	or/W	4112-4642	orJX	4643-4996	dhaT	5017-6108	orfY	6202-6630<br>
K. pneumoniae (U30903)			orflc	7116-7646	orfib	6762-7115	dhaT	5578-6741	orJ2a	5125-5556<br>
K. pneumoniae (U60992)					gdrB					<br>
C.)hjwK/H(U09771)	dhaR	3746-5671	or/W	5649-6179	orJX	6180-6533	dhaT	6550-7713	orJY	7736-8164<br>
C. pasteurianum (AF051373)										<br>
O pasteurianum (AF006034)			or/W	210-731	orJX	1-196	dhaT	1232-2389	orfY	746-1177<br>
S: typhimurium (AF026270)					pduH	8274-8645				<br>
Koxytoca (AFO17781)					ddrB	2063-2440				<br>
K. oxytoco {AF051313)										<br>
|    *	GENE FUNCTION:						<br>
	dehydratase, a	dehydr	atase, (i	dehydratase, y	reactivation<br>
!	gene	base pairs	iRene	base pairs	gene	base Dairs	gene	base pairs<br>
ORGANISM (GenBank Reference)								<br>
K, pneumoniae (SEQ ID NO:l)	dhaBl	7044-8711	dhaBl	8724-9308	dhaBl	9311-9736	or/Z	9749-11572<br>
K. pneumoniae (U3O903)	dhaBl	3047-4714	dhaBl	2450-2890	dhaBl	2022-2447	dhaB4	186-2009<br>
K. pneumoniae (U60992)	gldA	121-1788	: gldB	1801-2385	gldC	2388-2813	gdrA	<br>
C.freundiiQJ09n\)	dhaB	8556-10223	dhaC	10235-10819	dhaE	10822-11250	or/Z	11261-13072<br>
C. pasteurianum (AF051373)	dhaB	84-1748	IdhaC	1779-2318	dhaE	2333-2773	orfZ	2790-4598<br>
C. pasteurianum (AF006034)								<br>
S. typhimurium (AF026270)	pduC	3557-5221	pduD	5232-5906	pduE	5921-6442	pduG	6452-8284<br>
K. oxytoca (AF017781)							ddrA	241-2073<br>
K. OJtyroco(AF051373)	pddi	121-1785	pddB	1796-2470	pddC	2485-3006		<br><br>
00<br>
3<br>
©<br>
§<br>
00<br>
sj<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
Glycerol and diol dehydratases are subject to mechanism-based suicide inactivation by glycerol and some other substrates (Daniel et al., FEMS Microbiol. Rev. 22,553 (1999)). The terra "dehydratase reactivation factor" refers to those<br>
proteins responsible for reactivating the dehydratase activity. The terms <br>
5     	"dehydratase reactivating activity", "reactivating the dehydratase activity" or "regenerating the dehydratase activity" refers to the phenomenon of converting a dehydratase not capable of catalysis of a substrate to one capable of catalysis of a substrate or to the phenomenon of inhibiting the inactivation of a dehydratase or the phenomenon of extending the useful half-life of the dehydratase enzyme in<br>
10      vivo. Two proteins have been identified as being involved as the dehydratase<br>
reactivation factor (see WO 9821341 (US 6013494) and references therein, which are herein incorporated by reference; Daniel et al., supra; Toraya and Mori, J. Biol. Chem. 274, 3372 (1999); and Tobimatsu et al., J. Bacteriol. 181,4110 (1999)). Referring to Table 1, genes encoding one of the proteins include orfZ,<br>
15      dhaB4, gdrA,pduG and ddrA. Also referring to Table 1, genes encoding the second of the two proteins include orfX, orflb, gdrB,pduH and ddrB. The terms "1,3-propanediol oxidoreductase", "1,3-propanediol dehydrogenase" or "DhaT" refer to the polypeptide(s) responsible for an enzyme activity that is capable of catalyzing the interconversion of 3-HPA and<br>
20      1,3-propanediol provided the gene(s) encoding such activity is found to be<br>
physically or transcriptionally linked to a dehydratase enzyme in its natural (i.e., wild type) setting; for example, the gene is found within a dha regulon as is the case with dhaT&amp;om Klebsiella pneumonia. Referring to Table 1, genes encoding a 1,3-propanediol oxidoreductase include dhaT from Klebsiella pneumoniae,<br>
25      Citrobacterfreundii, and Clostridium pasteurianum. Each of these genes encode a polypeptide belonging to the family of type III alcohol dehydrogenases, exhibits a conserved iron-binding motif, and has a preference for the NAD+/NADH linked intercoversion of 3-HPA and 1,3-propanediol (Johnson and Lin, J. Bacteriol. 169, 2050 (1987); Daniel et al., J. Bacteriol 177, 2151 (1995); and Leurs et al., FEMS<br>
30      Microbiol. Lett. 154, 337 (1997)). Enzymes with similar physical properties have been isolated from Lactobacillus brevis and Lactobacillus buchneri (Veiga da Dunha and Foster, Appl. Environ. Microbiol. 58, 2005 (1992)).<br>
The term "dha regulon" refers to a set of associated genes or open reading frames encoding various biological activities, including but not limited to a<br>
3 5     dehydratase activity, a reactivation activity, and a 1,3-propanediol oxidoreductase. Typically a dha regulon comprises the open reading frames dhaR, orjY, dhaT, orfX, orjW, dhaBl, dhaB2, dhaB3, and or/Z as described herein.<br>
18<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
The term "non-specific catalytic activity" refers to the polypeptide(s) responsible for an enzyme activity that is sufficient to catalyze the interconversion<br>
of 3-HPA and 1,3 -propanediol and specifically excludes 1,3-propanediol oxidoreductase(s). Typically these enzymes are alcohol dehydrogenases. Such <br>
5      	enzymes may utilize cofactors other than NAD+/NADH, including but not limited to flavins such as FAD or FMN. A gene(s) for a non-specific alcohol dehydrogenase(s) is found, for example, to be endogenously encoded and functionally expressed within the microorganism E. coli KLP23.<br>
The terms "function" or "enzyme function" refer to the catalytic activity of<br>
10     an enzyme in altering the energy required to perform a specific chemical reaction. It is understood that such an activity may apply to a reaction in equilibrium where the production of either product or substrate may be accomplished under suitable conditions.<br>
The terms "polypeptide" and "protein" are used interchangeably.<br>
15	The terms "carbon substrate" and "carbon source" refer to a carbon source<br>
capable of being metabolized by host microorganisms of the present invention and particularly carbon sources selected from the group consisting of monosaccharides, oligosaccharides, polysaccharides, and one-carbon substrates or mixtures thereof.<br>
20	The terms "host cell" or "host microorganism" refer to a microorganism<br>
capable of receiving foreign or heterologous genes and of expressing those genes to produce an active gene product.<br>
The terms "foreign gene", "foreign DNA", "heterologous gene" and "heterologous DNA" refer to genetic material native to one organism that has<br>
25     been placed within a host microorganism by various means. The gene of interest may be a naturally occurring gene, a mutated gene, or a synthetic gene.<br>
The terms "transformation" and "transfection" refer to the acquisition of new genes in a cell after the incorporation of nucleic acid. The acquired genes may be integrated into chromosomal DNA or introduced as extrachromosomal<br>
30      replicating sequences. The term "transformant" refers to the product of a transformation.<br>
The term "genetically altered" refers to the process of changing hereditary material by transformation or mutation.<br>
The terms "recombinant microorganism" and "transformed host" refer to<br>
35     any microorganism having been transformed with heterologous or foreign genes or extra copies of homologous genes. The recombinant microorganisms of the present invention express foreign genes encoding glycerol-3-phosphate<br>
19<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
dehydrogenase (GPD1), glycerol-3-phosphatase (GPP2), glycerol dehydratase (dhaBl, dhaB2 anddhaB3), dehydratase reactivation factor (orfZ and orfiC), and<br>
	optionally 1 ,3-propanediol oxidoreductase (dhaT) for the production of<br>
1,3-propanediol from suitable carbon substrates. A preferred embodiment is an<br>
5	E. coli transformed with these genes but lacking a functional dhaT. A host<br>
microorganism, other than E. coli, may also be transformed to contain the<br>
disclosed genes and the gene for the non-specific catalytic activity for the<br>
interconversion of 3-HPA and 1,3-propanediol, specifically excluding 1,3-<br>
propanediol oxidoreductase(s) (dhaT).<br>
10	"Gene" refers to a nucleic acid fragment that expresses a specific protein,<br>
including regulatory sequences preceding (5' non-coding) and following (3' non-coding) the coding region. The terms "native" and "wild-type" refer to a gene as found in nature with its own regulatory sequences.<br>
The terms "encoding" and "coding" refer to the process by which a gene,<br>
15      through the mechanisms of transcription and translation, produces an amino acid sequence. It is understood that the process of encoding a specific amino acid sequence includes DNA sequences that may involve base changes that do not cause a change in the encoded amino acid, or which involve base changes which may alter one or more amino acids, but do not affect the functional properties of<br>
20      the protein encoded by the DNA sequence. It is therefore understood that the invention encompasses more than the specific exemplary sequences.<br>
The term "isolated" refers to a protein or DNA sequence that is removed from at least one component with which it is naturally associated.<br>
An "isolated nucleic acid molecule" is a polymer of RNA or DNA that is <br>
25	      single- or double-stranded, optionally containing synthetic, non-natural or altered nucleotide bases. An isolated nucleic acid molecule in the form of a polymer of DNA may be comprised of one or more segments of cDNA, genomic DNA or synthetic DNA.<br>
"Substantially similar" refers to nucleic acid molecules wherein changes in <br>
30	one or more nucleotide bases result in substitution of one or more amino acids, but do not affect the functional properties of the protein encoded by the DNA sequence. "Substantially similar" also refers to nucleic acid molecules wherein changes in one or more nucleotide bases do not affect the ability of the nucleic acid molecule to mediate alteration of gene expression by antisense or <br>
3 5    co-suppression technology. "Substantially similar" also refers to modifications of<br>
the nucleic acid molecules of the instant invention (such as deletion or insertion of one or more nucleotide bases) that do not substantially affect the functional<br>
20<br><br>
WO 01/12833<br><br>
PCT/USOO/22874<br><br>
properties of the resulting transcript vis-a-vis the ability to mediate alteration of gene expression by antisense or co-suppression technology or alteration of the<br>
functional properties of the resulting protein molecule. The invention<br>
encompasses more than the specific exemplary sequences.<br>
5	For example, it is well known in the art that alterations in a gene which<br>
          result in the production of a chemically equivalent amino acid at a given site, but<br>
       do not effect the functional properties of the encoded protein are common. For the<br>
              purposes of the present invention substitutions are defined as exchanges within<br>
            one of the following five groups:<br>
10	1.    Small aliphatic, nonpolar or slightly polar residues: Ala, Ser, Thr<br>
(Pro,Gly);<br>
2.	Polar, negatively charged residues and their amides: Asp, Asn,<br>
Glu, Gin;<br>
3.	Polar, positively charged residues: His, Arg, Lys;<br>
15	4.   Large aliphatic, nonpolar residues: Met, Leu, lie, Val (Cys); and<br>
5.   Large aromatic residues: Phe, Tyr,Trp. Thus, a codon for the amino acid alanine, a hydrophobic amino acid, may be substituted by a codon encoding another less hydrophobic residue (such as glycine) or a more hydrophobic residue (such as valine, leucine, or isoleucine).<br>
20     Similarly, changes which result in substitution of one negatively charged residue for another (such as aspartic acid for glutamic acid) or one positively charged residue for another (such as lysine for arginine) can also be expected to produce a functionally equivalent product.<br>
In many cases, nucleotide changes which result in alteration of the<br>
25     N-terminal and C-terminal portions of the protein molecule would also not be expected to alter the activity of the protein.<br>
Each of the proposed modifications is well within the routine skill in the art, as is determination of retention of biological activity of the encoded products. Moreover, the skilled artisan recognizes that substantially similar sequences<br>
30     encompassed by this invention are also defined by their ability to hybridize, under stringent conditions (0.1X SSC, 0.1% SDS, 65 °C and washed with 2X SSC, 0.1% SDS followed by 0.1 X SSC, 0.1% SDS), with the sequences exemplified herein. Preferred substantially similar nucleic acid fragments of the instant invention are those nucleic acid fragments whose DNA sequences are at least 80% identical to<br>
35     the DNA sequence of the nucleic acid fragments reported herein. More preferred nucleic acid fragments are at least 90% identical to the DNA sequence of the nucleic acid fragments reported herein. Most preferred are nucleic acid fragments<br>
21<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
that are at least 95% identical to the DNA sequence of the nucleic acid fragments reported herein.<br>
A nucleic acid fragment is "hybridizable" to another nucleic acid fragment,<br>
such as a cDNA, genomic DNA, or RNA, when a single stranded form of the <br>
5    	 nucleic acid fragment can anneal to the other nucleic acid fragment under the appropriate conditions of temperature and solution ionic strength. Hybridization and washing conditions are well known and exemplified in Sambrook, J., Fritsch, E. F. and Maniatis, T. Molecular Cloning: A Laboratory Manual. Second Edition, Cold Spring Harbor Laboratory Press, Cold Spring Harbor (1989), particularly<br>
10     Chapter 11 and Table 11.1 therein (entirely incorporated herein by reference).<br>
The conditions of temperature and ionic strength determine the "stringency" of the hybridization. For preliminary screening for homologous nucleic acids, low stringency hybridization conditions, corresponding to a Tm of 55 °, can be used, e.g., 5X SSC, 0.1% SDS, 0.25% milk, and no formamide; or 30% formamide, 5X<br>
15      SSC, 0.5% SDS. Moderate stringency hybridization conditions correspond to a higher Tm, e.g., 40% formamidet with 5X or 6X SSC. Hybridization requires that the two nucleic acids contain complementary sequences, although depending on the stringency of the hybridization, mismatches between bases are possible. The appropriate stringency for hybridizing nucleic acids depends on the length of the<br>
20     nucleic acids and the degree of complementation, variables well known in the art. The greater the degree of similarity or homology between two nucleotide sequences, the greater the value of Tm for hybrids of nucleic acids having those sequences. The relative stability (corresponding to higher Tm) of nucleic acid<br>
hybridization decreases in the following order: RNA-.RNA, DNA:RNA, <br>
25	DNA:DNA. For hybrids of greater than 100 nucleotides in length, equations for calculating Tm have been derived (see Sambrook et al., supra, 9.50-9.51). For hybridization with shorter nucleic acids, i.e., oligonucleotides, the position of mismatches becomes more important, and the length of the oligonucleotide determines its specificity (see Sambrook et al., supra, 11.7-11.8). In one <br>
30     embodiment the length for a hybridizable nucleic acid is at least about 10<br>
nucleotides. Preferable a minimum length for a hybridizable nucleic acid is at least about 15 nucleotides; more preferably at least about 20 nucleotides; and most preferably the length is at least 30 nucleotides. Furthermore, the skilled artisan<br>
will recognize that the temperature and wash solution salt concentration may be <br>
35     adjusted as necessary according to factors such as length of the probe.<br>
A "substantial portion" refers to an amino acid or nucleotide sequence<br>
which comprises enough of the amino acid sequence of a polypeptide or the<br>
22<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
nucleotide sequence of a gene to afford putative identification of that polypeptide or gene, either by manual evaluation of the sequence by one skilled in the art, or<br>
by computer-automated sequence comparison and identification using algorithms<br>
such as BLAST (Basic Local Alignment Search Tool; Altschul et al., J. Mol.Biol. <br>
5    	 215:403-410 (1993); see also www.ncbi.nlm.nih.gov/BLAST/). In general, a sequence often or more contiguous amino acids or thirty or more nucleotides is necessary in order to putatively identify a polypeptide or nucleic acid sequence as homologous to a known protein or gene. Moreover, with respect to nucleotide sequences, gene-specific oligonucleotide probes comprising 20-30 contiguous<br>
10     nucleotides may be used in sequence-dependent methods of gene identification (e.g., Southern hybridization) and isolation (e.g., in situ hybridization of bacterial colonies or bacteriophage plaques). In addition, short oligonucleotides of 12-15 bases may be used as amplification primers in PCR in order to obtain a particular nucleic acid molecule comprising the primers. Accordingly, a "substantial<br>
15     portion" of a nucleotide sequence comprises enough of the sequence to afford specific identification and/or isolation of a nucleic acid molecule comprising the sequence. The instant specification teaches partial or complete amino acid and nucleotide sequences encoding one or more particular proteins. The skilled artisan, having the benefit of the sequences as reported herein, may now use all or<br>
20     a substantial portion of the disclosed sequences for the purpose known to those skilled in the art. Accordingly, the instant invention comprises the complete sequences as reported in the accompanying Sequence Listing, as well as substantial portions of those sequences as defined above.<br>
The term "complementary" describes the relationship between nucleotide<br>
25      bases that are capable to hybridizing to one another. For example, with respect to DNA, adenosine is complementary to thymine and cytosine is complementary to guanine. Accordingly, the instant invention also includes isolated nucleic acid molecules that are complementary to the complete sequences as reported in the accompanying Sequence Listing as well as those substantially similar nucleic acid<br>
30      sequences.<br>
The term "percent identity", as known in the art, is a relationship between two or more polypeptide sequences or two or more polynucleotide sequences, as determined by comparing the sequences. In the art, "identity" also means the degree of sequence relatedness between polypeptide or polynucleotide sequences,<br>
35     as the case may be, as determined by the match between strings of such sequences. "Identity" and "similarity" can be readily calculated by known methods, including but not limited to those described in: Computational<br>
23<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
Molecular Biology; Lesk, A. M., Ed.; Oxford University Press: New York, 1988; Biocomputing: Informatics and Genome Projects; Smith, D. W., Ed.; Academic<br>
Press: New York, 1993; Computer Analysis of Sequence Data, Part I; Griffin, A.<br>
M. and Griffin, H. G., Eds.; Humana Press: New Jersey, 1994; Sequence Analysis 5      in Molecular Biology; von Heinje, G., Ed.; Academic Press: New York, 1987; and<br>
Sequence Analysis Primer; Gribskov, M. and Devereux, J., Eds.; Stockton Press: New York, 1991. Preferred methods to determine identity are designed to give the largest match between the sequences tested.<br>
Methods to determine identity and. similarity are codified in publicly<br>
10     available computer programs. Preferred computer program methods to determine identity and similarity between two sequences include, but are not limited to, the GCG Pileup program found in the GCG program package, using the Needleman and Wunsch algorithm with their standard default values of gap creation penalty=12 and gap extension penalty=4 (Devereux et al., Nucleic Acids Res.<br>
15      12:387-395 (1984)), BLASTP, BLASTN, and FASTA (Pearson et al., Proc. Natl. Acad. Sci. USA 85:2444-2448 (1988). The BLASTX program is publicly available from NCBI and other sources (BLAST Manual. Altschul et al., Natl. Cent. Biotechnol. Inf., Natl. Library Med. (NCBI NLM) NIH, Bethesda, Md. 20894; Altschul et al., J. Mol. Biol. 215:403-410 (1990); Altschul et al, "Gapped<br>
20      BLAST and PSI-BLAST: a new generation of protein database search programs", Nucleic Acids Res. 25:3389-3402 (1997)). Another preferred method to determine percent identity, is by the method of DNASTAR protein alignment protocol using the Jotun-Hein algorithm (Hein et al. Methods Enzymol. 183:626-645 (1990)). Default parameters for the Jotun-Hein method for alignments are: for multiple<br>
25      alignments, gap penalty=l 1, gap length penalty=3; for pairwise alignments ktuple=6. As an illustration, by a polynucleotide having a nucleotide sequence having at least, for example, 95% "identity" to a reference nucleotide sequence it is intended that the nucleotide sequence of the polynucleotide is identical to the reference sequence except that the polynucleotide sequence may include up to five<br>
30     point mutations per each 100 nucleotides of the reference nucleotide sequence. In other words, to obtain a polynucleotide having a nucleotide sequence at least 95% identical to a reference nucleotide sequence, up to 5% of the nucleotides in the reference sequence may be deleted or substituted with another nucleotide, or a number of nucleotides up to 5% of the total nucleotides in the reference sequence<br>
35     may be inserted into the reference sequence. These mutations of the reference sequence may occur at the 5* or 3' terminal positions of the reference nucleotide sequence or anywhere between those terminal position, interspersed either<br>
24<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
individually among nucleotides in the reference sequence or in one or more contiguous groups within the reference sequence. Analogously, by a polypeptide<br>
having an amino acid sequence havingat least, for example, 95% identity to a<br>
reference amino acid sequence is intended that the amino acid sequence of the <br>
5    	 polypeptide is identical to the reference sequence except that the polypeptide sequence may include up to five amino acid alterations per each 100 amino acids of the reference amino acid. In other words, to obtain a polypeptide having an amino acid sequence at least 95% identical to a reference amino acid sequence, up to 5% of the amino acid residues in the reference sequence may be deleted or<br>
10     substituted with another amino acid, or a number of amino acids up to 5% of the total amino acid residues in the reference sequence may be inserted into the reference sequence. These alterations of the reference sequence may occur at the amino or carboxy terminal positions of the reference amino acid sequence or anywhere between those terminal positions, interspersed either individually<br>
15     among residues in the reference sequence or in one or more contiguous groups within the reference sequence.<br>
The term "homologous" refers to a protein or polypeptide native or naturally occurring in a given host cell. The invention includes microorganisms producing homologous proteins via recombinant DNA technology.<br>
20	The term "percent homology" refers to the extent of amino acid sequence<br>
identity between polypeptides. When a first amino acid sequence is identical to a second amino acid sequence, then the first and second amino acid sequences exhibit 100% homology. The homology between any two polypeptides is a direct function of<br>
the total number of matching amino acids at a given position in either sequence, e.g., <br>
25    	 if half of the total number of amino acids in either of the two sequences are the same then the two sequences are said to exhibit 50% homology.<br>
"Codon degeneracy" refers to divergence in the genetic code permitting variation of the nucleotide sequence without effecting the amino acid sequence of an encoded polypeptide. Accordingly, the instant invention relates to any nucleic <br>
30	acid molecule that encodes all or a substantial portion of the amino acid sequence as set forth in SEQ ID NO:57. The skilled artisan is well aware of the "codon-bias" exhibited by a specific host cell in usage of nucleotide codons to specify a given amino acid. Therefore, when synthesizing a gene for improved expression in a host cell, it is desirable to design the gene such that its frequency of codon <br>
3 5	    usage approaches the frequency of preferred codon usage of the host cell. Modifications to the sequence, such as deletions, insertions, or substitutions in the sequence which produce silent changes that do not<br>
25<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
substantially affect the functional properties of the resulting protein molecule are also contemplated. For example, alteration in the gene sequence which reflect the<br>
degeneracy of the genetic code, or which result in the production of a chemically<br>
equivalent amino acid at a given site, are contemplated. Thus, a codon for the <br>
5      	amino acid alanine, a hydrophobic amino acid, may be substituted by a codon encoding another less hydrophobic residue, such as glycine, or a more hydrophobic residue, such as valine, leucine, or isoleucine. Similarly, changes which result in substitution of one negatively charged residue for another, such as aspartic acid for glutamic acid, or one positively charged residue for another, such<br>
10     as lysine for arginine, can also be expected to produce a biologically equivalent product. Nucleotide changes which result in alteration of the N-terminal and C-terminal portions of the protein molecule would also not be expected to alter the activity of the protein. In some cases, it may in fact be desirable to make mutants of the sequence in order to study the effect of alteration on the biological activity<br>
15      of the protein. Each of the proposed modifications is well within the routine skill in the art, as is determination of retention of biological activity in the encoded products. Moreover, the skilled artisan recognizes that sequences encompassed by this invention are also defined by their ability to hybridize, under stringent conditions (0.1X SSC, 0.1% SDS, 65 °C), with the sequences exemplified herein.<br>
20	The term "expression" refers to the transcription and translation to gene<br>
product from a gene coding for the sequence of the gene product.<br>
The terms "plasmid", "vector", and "cassette" refer to an extra chromosomal element often carrying genes which are not part of the central metabolism of the cell, and usually in the form of circular double-stranded DNA<br>
25      molecules. Such elements may be autonomously replicating sequences, genome integrating sequences, phage or nucleotide sequences, linear or circular, of a single- or double-stranded DNA or RNA, derived from any source, in which a number of nucleotide sequences have been joined or recombined into a unique construction which is capable of introducing a promoter fragment and DNA<br>
30      sequence for a selected gene product along with appropriate 3' untranslated<br>
sequence into a cell. "Transformation cassette" refers to a specific vector<br>
containing a foreign gene and having elements in addition to the foreign gene that facilitates transformation of a particular host cell. "Expression cassette" refers to a specific vector containing a foreign gene and having elements in addition to the<br>
3 5     foreign gene that allow for enhanced expression of that gene in a foreign host.<br>
26<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
Construction of Recombinant Organisms<br>
Recombinant organisms containing the necessary genes that will encode the enzymatic pathway for the conversion of a carbon substrate to 1,3-propanediol<br>
may be constructed using techniques well known in the art. Genes encoding <br>
5     	glycerol-3-phosphate dehydrogenase (GPD1), glycerol-3-phosphatase (GPP2), glycerol dehydratase (dhaBJ, dhaB2, and dhaB3), dehydratase reactivation factor (prfZ and or/X) and 1,3-propanediol oxidoreductase (dhaT) were isolated from a native host such as Klebsiellas* Saccharomyces and used to transform host strains such as £jg/EDH_5a7 ECL707, AA20Q, or KLP23.<br>
10      Isolation of Genes<br>
Methods of obtaining desired genes from a bacterial genome are common and well known in the art of molecular biology. For example, if the sequence of the gene is known, suitable genomic libraries may be created by restriction endonuclease digestion and may be screened with probes complementary to the<br>
15     desired gene sequence. Once the sequence is isolated, the DNA may be amplified using standard primer directed amplification methods such as polymerase chain reaction (PCR) (U.S. 4,683,202) to obtain amounts of DNA suitable for transformation using appropriate vectors.<br>
Alternatively, cosmid libraries may be created where large segments of<br>
20      genomic DNA (35-45kb) may be packaged into vectors and used to transform<br>
appropriate hosts. Cosmid vectors are unique in being able to accommodate large quantities of DNA. Generally cosmid vectors have at least one copy of the cos DNA sequence which is needed for packaging and subsequent circularization of the foreign DNA. In addition to the cos sequence these vectors will also contain<br>
25     an origin of replication such as ColEl and drug resistance markers such as a gene resistant to ampicillin or neomycin. Methods of using cosmid vectors for the transformation of suitable bacterial hosts are well described in Sambrook, J. et al., Molecular Cloning: A Laboratory Manual. Second Edition (1989) Cold Spring Harbor Laboratory Press, herein incorporated by reference.<br>
30	Typically to clone cosmids, foreign DNA is isolated and ligated, using the<br>
appropriate restriction endonucleases, adjacent to the cos region of the cosmid vector. Cosmid vectors containing the linearized foreign DNA are then reacted with a DNA packaging vehicle such as bacteriophage. During the packaging process the cos sites are cleaved and the foreign DNA is packaged into the head<br>
35 portion of the bacterial viral particle. These particles are then used to transfect suitable host cells such as E. coli. Once injected into the cell, the foreign DNA circularizes under the influence of the cos sticky ends. In this manner large<br>
27<br><br>
WO 01/12833	PCT/US00/2287-<br>
segments of foreign DNA can be introduced and expressed in recombinant host cells.<br>
Isolation And Cloning of Genes Encoding Glycerol Dehydratase (dhaBL dhaBL<br>
and dhaB3) Dehydratase Reactivating Factors (orfZ and orfX). and 5      1.3-propanediol dehydrogenase (dhaT)<br>
Cosmid vectors and cosmid transformation methods were used within the context of the present invention to clone large segments of genomic DNA from bacterial genera known to possess genes capable of processing glycerol to 1,3-propanediol. Specifically, genomic DNA from K. pneumoniae was isolated<br>
10      by methods well known in the art and digested with the restriction enzyme Sau3 A for insertion into a cosmid vector Supercos 1 and packaged using Gigapackll packaging extracts. Following construction of the vector E. coli XL 1-Blue MR cells were transformed with the cosmid DNA. Transformants were screened for the ability to convert glycerol to 1,3-propanediol by growing the cells in the<br>
15      presence of glycerol and analyzing the media for 1,3-propanediol formation.<br>
Two of the 1,3-propanediol positive transformants were analyzed and the cosmids were named pKPl and pKP2. DNA sequencing revealed extensive homology to the glycerol dehydratase gene from C.freundii, demonstrating that these transformants contained DNA encoding the glycerol dehydratase gene.<br>
20     Other 1,3-propanediol positive transformants were analyzed and the cosmids were named pKP4 and pKP5. DNA sequencing revealed that these cosmids carried DNA encoding a diol dehydratase gene.<br>
Although the instant invention utilizes the isolated genes from within a Klebsiella cosmid, alternate sources of dehydratase genes and dehydratase<br>
25	reactivation factor genes include, but are not limited to, Citrobacter, Clostridia and Salmonella (see Table 1). <br>
26	Genes Encoding G3PDH and G3P Phosphatase<br>
The present invention provides genes suitable for the expression of G3PDH and G3P phosphatase activities in a host cell.<br>
30	Genes encoding G3PDH are known. For example, GPD1 has been<br>
isolated from Saccharomyces and has the base sequence given by SEQ ID NO:53, encoding the amino acid sequence given in SEQ ID NO:54 (Wang et al., supra). Similarly, G3PDH activity has also been isolated from Saccharomyces encoded by GPD2 (Eriksson et al., Mol Microbiol. 17,95 (1995)).<br>
35	For the purposes of the present invention it is contemplated that any gene<br>
encoding a polypeptide responsible for NADH-dependent G3PDH activity is suitable wherein that activity is capable of catalyzing the conversion of<br>
28<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
dihydroxyacetone phosphate (DHAP) to glycerol-3-phosphate (G3P). Further, it is contemplated that any gene encoding the amino acid sequence of NADH-<br>
dependent G3PDH's corresponding to the genes DAR1, GPD1, GPD2, GPD3, and<br>
gpsA will be functional in the present invention wherein that amino acid sequence <br>
5     	 may encompass amino acid substitutions, deletions or additions that do not alter the function of the enzyme. The skilled person will appreciate that genes<br>
encoding G3PDH isolated from other sources will also be suitable for use in the<br>
present invention. Genes encoding G3P phosphatase are known. For example, GPP2 has been isolated from Saccharomyces cerevisiae and has the base sequence<br>
10     given by SEQ ID NO:55, which encodes the amino acid sequence given in SEQ ID NO:56 (Norbeck et al., J. Biol. Chem. 271, 13875 (1996)).<br>
For the purposes of the present invention, any gene encoding a G3P phosphatase activity is suitable for use in the method wherein that activity is capable of catalyzing the conversion of glycerol-3-phosphate plus H20 to glycero1<br>
15      plus inorganic phosphate. Further, any gene encoding the amino acid sequence o G3P phosphatase corresponding to the genes GPP2 and GPP1 will be functional in the present invention including any amino acid sequence that encompasses amino acid substitutions, deletions or additions that do not alter the function of th G3P phosphatase enzyme. The skilled person will appreciate that genes encoding<br>
20      G3P phosphatase isolated from other sources will also be suitable for use in the<br>
        present invention	-<br>
Host Cells<br>
Suitable host cells for the recombinant production of 1,3-propanediol may be either prokaryotic or eukaryotic and will be limited only by the host cell ability tc<br>
25      express the active enzymes for the 1,3-propanediol pathway. Suitable host cells will be bacteria such as Citrobacter, Enterobacter, Clostridium, Klebsiella, Aerobacter, Lactobacillus, Aspergillus, Saccharomyces, Schizosaccharomyces, Zygosaccharomyces, Pichia, Kluyveromyces, Candida, Hansenula, Debaryomyces, Mucor, Toruiopsis, Methylobacter, Escherichia, Salmonella,<br>
30      Bacillus, Streptomyces, and Pseudomonas. Preferred in the present invention are E. coli, E. blattae, Klebsiella, Citrobacter, and Aerobacter.<br>
Microorganisms can be converted to a high titer 1,3-propanediol produce] by using the following general protocol.'*-<br>
1.  Determine the presence in a potential host organism of an endogenous<br>
35	dhaTAikt activity that allows for the steady state concentration of a toxic<br>
or inhibitory level of 3-HPA in the presence of 1-2 M 1,3-propanediol.<br>
29<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
2. If such an activity exists in the potential host organism, perform suitable mutagenisis to delete or inactivate this activity. Confirmation of a nonfunctional or deleted dhaTAM activity can be detected by the lack of 3-<br>
HPA accumulation in the presence of 1-2 M 1,3-propanediol.<br>
5	 3.  Express appropriate genes for a) glycerol production, if glycerol is not the<br>
carbon source, b) glycerol dehydratase and the associated maintenance system, and c) yqhD. Considerations which would need to be taken with respect to certain microorganisms concern the expression or repression of endogenous dhaT-likc <br>
10	enzymes under the conditions for 1,3-propanediol production. These might also include the presence of glycerol, glucose or anaerobisis. <br>
11	Vectors and Expression Cassettes<br>
The present invention provides a variety of vectors and transformation and expression cassettes suitable for the cloning, transformation and expression of <br>
15      	G3PDH, G3P phosphatase, dehydratase, and dehydratase reactivation factor into a suitable host cell. Suitable vectors will be those which are compatible with the microorganism employed. Suitable vectors can be derived, for example, from a bacteria, a virus (such as bacteriophage T7 or a M-13 derived phage), a cosmid, a yeast or a plant. Protocols for obtaining and using such vectors are known to<br>
 20      those in the art (Sambrook et al., Molecular Cloning: A Laboratory Manual -volumes 1,2, 3 (Cold Spring Harbor Laboratory: Cold Spring Harbor, NY, 1989)).<br>
Typically, the vector or cassette contains sequences directing transcription and translation of the appropriate gene, a selectable marker, and sequences<br>
25	allowing autonomous replication or chromosomal integration. Suitable vectors comprise a region 5' of the gene, which harbors transcriptional initiation controls, and a region 3' of the DNA fragment which controls transcriptional termination. It is most preferred when both control regions are derived from genes homologous to the transformed host cell. Such control regions need not be derived from the <br>
30  	   genes native to the specific species chosen as a production host.<br>
Initiation control regions, or promoters, which are useful to drive expression of the G3PDH and G3P phosphatase genes (DAR1 and GPP2, respectively) in the desired host cell are numerous and familiar to those skilled in the art. Virtually any promoter capable of driving these genes is suitable for the<br>
35     present invention including but not limited to CYC1, HIS3, GAL1, GAL10,<br>
ADHl, PGK, PH05, GAPDH, ADC1, TRP1, URA3, LEU2, ENO, and TPI<br>
30<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
(useful for expression in Saccharomyces); AOX1 (useful for expression in Pichia); and lac, trp, X?L, XPR, T7, tac, and trc (useful for expression in E. coli).<br>
Termination control regions may also be derived from various genes native to the preferred hosts. Optionally, a termination site may be unnecessary;<br>
5     however, it is most preferred if included.<br>
For effective expression of the instant enzymes, DNA encoding the .<br>
enzymes are linked operably through initiation codons to selected expression<br>
control regions such that expression results in the formation of the appropriate<br>
messenger RNA.<br>
10	Particularly useful in the present invention are the vectors pDT29 and<br>
pKP32 which are designed to be used in conjunction with pAH48. The essential elements of pDT29 and pKP32 are derived from the dha regulon isolated from Klebsiella pneumoniae. pDT29 contains the open reading frames dhaR, orJY, dhaT, orfiC, or/W, dhaBl, dhaB2, and dhaB3, nucleotide the sequences of which<br>
15     are contained within SEQ ID NO: 1. pKP32 contains the same set of open reading frames as found on pDT29, from the same source, with the difference that pKP32 lacks the dhaT. pAH48 is the vehicle used for the introduction of DAR1 and GPP2 genes into the host cell and more specifically comprises the DAR1 and GPP2 genes isolated from Saccharomyces cerevisiae.<br>
20      Transformation of Suitable Hosts and Expression of Genes for the Production of 1,3 -propanediol<br>
Once suitable cassettes are constructed they are used to transform appropriate host cells. Introduction of the cassette containing the genes encoding G3PDH, G3P phosphatase, dehydratase, and dehydratase reactivation factor into<br>
25     the host cell may be accomplished by known procedures such as by<br>
transformation (e.g., using calcium-permeabilized cells, electroporation), or by transfection using a recombinant phage virus (Sambrook et al., supra).<br>
In the present invention cassettes were used to transform the E. coli as fully described in the GENERAL METHODS and EXAMPLES.<br>
30      Mutants<br>
In addition to the cells exemplified, it is contemplated that the present method will be able to make use of cells having single or multiple mutations specifically designed to enhance the production of 1,3-propanediol. Cells that<br>
normally divert a carbon feed stock into non-productive pathways, or that exhibit<br>
35    	 significant catabolite repression could be mutated to avoid these phenotypic deficiencies. For example, many wild type cells are subject to catabolite repression from glucose and by-products in the media and it is contemplated that<br>
31<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
mutant strains of these wild type organisms, capable of 1,3-propanediol production that are resistant to glucose repression, would be particularly useful in<br>
the present invention.<br>
Methods of creating mutants are common and well known in the art. For <br>
5   	   example, wild type cells may be exposed to a variety of agents such as radiation or chemical mutagens and then screened for the desired phenotype. When creating mutations through radiation either ultraviolet (UV) or ionizing radiation may be used. Suitable short wave UV wavelengths for genetic mutations will fall within the range of 200 nm to 300 run where 254 nm is preferred. UV radiation in this<br>
10   	  wavelength principally causes changes within nucleic acid sequence from<br>
guanidine and cytosine to adenine and thymidine. Since all cells have DNA repair mechanisms that would repair most UV induced mutations, agents such as caffeine and other inhibitors may be added to interrupt the repair process and maximize the number of effective mutations. Long wave UV mutations using<br>
15  	    light in the 300 nm to 400 nm range are also possible but are generally not as effective as the short wave UV light unless used in conjunction with various activators such as psoralen dyes that interact with the DNA.<br>
Mutagenesis with chemical agents is also effective for generating mutants and commonly used substances include chemicals that affect nonreplicating DNA<br>
20 	    such as HN02 and NH2OH, as well as agents that affect replicating DNA such as acridine dyes, notable for causing frameshift mutations. Specific methods for creating mutants using radiation or chemical agents are well documented in the art. See for example Thomas D. Brock in Biotechnology: A Textbook of Industrial Microbiology, Second Edition (1989) Sinauer Associates, Inc.,<br>
25      Sunderland, MA., or Deshpande, Mukund V., Appl. Biochem. Biotechnol. 36, 227 (1992), herein incorporated by reference.<br>
After mutagenesis has occurred, mutants having the desired phenotype may be selected by a variety of methods. Random screening is most common where the mutagenized cells are selected for the ability to produce the desired<br>
30     product or intermediate. Alternatively, selective isolation of mutants can be<br>
performed by growing a mutagenized population on selective media where only resistant colonies can develop. Methods of mutant selection are highly developed and well known in the art of industrial microbiology. See for example Brock, Supra; DeMancilha et aL, Food Chem. 14,313 (1984).<br>
3 5	The elimination of an undesired enzyme activity may be also<br>
accomplished by disruption of the gene encoding the enzyme. Such methods are known to those skilled in tneart and are exemplified in Example 4 and Example 8.<br>
32<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
Alterations in the 1.3-propanediol Production Pathway<br>
Representative Enzyme Pathway. The production of 1,3-propanediol from<br>
glucose can be accomplished by the following series of steps. This series is representative of a number of pathways known to those skilled in the art and is <br>
5    	 illustrated in Figure 5. Glucose is converted in a series of steps by enzymes of the glycolytic pathway to dihydroxyacetone phosphate (DHAP) and 3-phospho-glyceraldehyde (3-PG). Glycerol is then formed by either hydrolysis of DHAP to dihydroxyacetone (DHA) followed by reduction, or reduction of DHAP to glycerol 3-phosphate (G3P) followed by hydrolysis. The hydrolysis step can be<br>
10     catalyzed by any number of cellular phosphatases, which are known to be nonspecific with respect to their substrates, or the activity can be introduced into the host by recombination. The reduction step can be catalyzed by a NAD+ (or NADP+) linked host enzyme or the activity can be introduced into the host by recombination. It is notable that the dha regulon contains a glycerol<br>
15      dehydrogenase (E.C. 1.1.1.6) that catalyzes the reversible reaction of Equation 3.<br>
Glycerol -» 3-HPA + H20	(Equation 1)<br>
3-HPA + NADH + H+ -+ 1,3-PropanedioI + NAD+	(Equation 2)<br>
Glycerol + NAD+ -&gt; DHA + NADH + H+	(Equation 3)<br>
20<br>
Glycerol is converted to 1,3-propanediol via the intermediate 3-hydroxy-propionaldehye (3-HPA) as has been described in detail above. The intermediate 3-HPA is produced from glycerol, Equation 1, by a dehydratase enzyme that can be encoded by the host or can be introduced into the host by recombination. This<br>
25     dehydratase can be glycerol dehydratase (E.C. 4.2.1.30), diol dehydratase<br>
(E.C. 4.2.1.28) or any other enzyme able to catalyze this transformation. Glycerol dehydratase, but not diol dehydratase, is encoded by the dha regulon. 1,3-Propanediol is produced from 3-HPA, Equation 2, by aNAD+- (or NADP+) linked host enzyme or the activity can be introduced into the host by<br>
30      recombination. This final reaction in the production of 1,3-propanediol can be catalyzed by 1,3-propanediol dehydrogenase (E.C. 1.1.1.202) or other alcohol dehydrogenases.<br>
Mutations and transformations that affect carbon channeling. A variety of mutant microorganisms comprising variations in the 1,3-propanediol production pathway<br>
3 5    will be useful in the present invention. For example the introduction of a<br>
triosephosphate isomerase mutation (tpi-) into the microorganism of the present invention is an example of the use of a mutation to improve the performance by carbon channeling. Triosephosphate isomerase is the enzyme responsible for the<br>
33<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
conversion of DAHP to 3-phosphoglyceraldehyde, and as such allows the<br>
diversion of carbon flow from the main pathway form glucose to glycerol and 1,3-propanediol (Figure 5). Thus, the deletion mutation {tpi-) enhances the overall<br>
metabolic efficiency of the desired pathway over that described in the art. <br>
5       Similarly, mutations which block alternate pathways for intermediates of the<br>
1,3-propanediol production pathway would also be useful to the present invention. For example, the elimination of glycerol kinase prevents glycerol, formed from G3P by the action of G3P phosphatase, from being re-converted to G3P at the expense of ATP (Figure 5). Also, the elimination of glycerol dehydrogenase (for <br>
10    	 example, gldA) prevents glycerol, formed from DHAP by the action of NADH-dependent glycerol-3-phosphate dehydrogenase, from being converted to dihydroxyacetone (Figure 5). Mutations can be directed toward a structural gene so as to impair or improve the activity of an enzymatic activity or can be directed<br>
toward a regulatory gene, including promoter regions and ribosome binding sites,<br>
15       so as to modulate the expression level of an enzymatic activity.<br>
It is thus contemplated that transformations and mutations can be combined so as to control particular enzyme activities for the enhancement of 1,3-propanediol production. Thus, it is within the scope of the present invention to anticipate modifications of a whole cell catalyst which lead to an increased<br>
20  	   production of 1,3-propanediol.<br>
The present invention utilizes a preferred pathway for the production of 1,3-propanediol from a sugar substrate where the carbon flow moves from glucose to DHAP, G3P, Glycerol, 3-HPA and finally to 1,3-propanediol. The present production strains have been engineered to maximize the metabolic efficiency of<br>
25     the pathway by incorporating various deletion mutations that prevent the diversion of carbon to non-productive compounds. Glycerol may be diverted from conversion to 3HPA by transformation to either DHA or G3P via glycerol dehydrogenase or glycerol kinase as discussed above (Figure 5). Accordingly, the<br>
present production strains contain deletion mutations in the gldA and glpK genes. 30      Similarly DHAP may be diverted to 3-PG by triosephosphate isomerase, thus the<br>
present production microorganism also contains a deletion mutation in this gene. The present method additionally incorporates a dehydratase enzyme for the conversion of glycerol to 3 HP A, which functions in concert with the reactivation factor, encoded by orjX and orfZ of the dha regulon (Figure 5). Although <br>
3 5     	conversion of the 3HP A to 1,3-propanediol is typically accomplished via a 1,3-propanediol oxidoreductase, the present method utilizes a non-specific catalytic activity that produces greater titers and yields of the final product,<br>
34<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
1,3-propanediol (Figure 5). In such a process, titers of 1,3-propanediol of at least 10 g/L are achieved, where titers of 200 g/L are expected.<br>
Alternatively, an improved process for 1,3-propanediol production may<br>
utilize glycerol or dihydroxyacetone as a. substrate where the pathway comprises <br>
5       only the last three substrates, glycerol —&gt; 3HPA —&gt; 1,3-propanediol. In such a<br>
process, the oxidoreductase is again eliminated in favor of the non-specific<br>
catalytic activity, (expected to be an alcohol dehydrogenase), however the need<br>
for deletion mutations are nullified by the energy considerations of adding glycerol to the culture. In such as process titers of 1,3-propanediol of at least<br>
10      71 g/L are achieved where titers of 200 g/L are expected.<br>
Similarly it is within the scope of the invention to provide mutants of wildtype microorganisms that have been modified by the deletion or mutation of the dhaT activity to create improved 1,3-propandiol producers. For example, microorganisms, which naturally contain all the elements of the dha regulon, may<br>
15     be manipulated so as to inactivate the dhaT gene encoding the 1,3-propandiol oxidoreductase activity. These microorganisms will be expected to produce higher yields and titers of 1,3-propanediol, mediated by the presence of an endogenous catalytic activity, expected to be an alcohol dehydrogenase. Examples of such microorganisms include but are not limited to Klebsiella sp.,<br>
20      Citrobacter sp., and Clostridium sp. Media and Carbon Substrates<br>
Fermentation media in the present invention must contain suitable carbon substrates. Suitable substrates may include but are not limited to monosaccharides such as glucose and fructose, oligosaccharides such as lactose or sucrose,<br>
25      polysaccharides such as starch or cellulose or mixtures thereof and unpurified mixtures from renewable feedstocks such as cheese whey permeate, cornsteep liquor, sugar beet molasses, and barley malt. Additionally the carbon substrate may also be one-carbon substrates such as carbon dioxide, or methanol for which metabolic conversion into key biochemical intermediates has been demonstrated.<br>
30      Glycerol production from single carbon sources (e.g., methanol, formaldehyde or formate) has been reported in methylotrophic yeasts (K. Yamada et al., Agric. Biol. Chem. 53(2), 541-543 (1989)) and in bacteria (Hunter et. al., Biochemistry 24,4148-4155 (1985)). These microorganisms can assimilate single carbon compounds, ranging in oxidation state from methane to formate, and produce<br>
35    glycerol. The pathway of carbon assimilation can be through ribulose<br>
monophosphate, through serine, or through xylulose-momophosphate (Gottschalk,<br>
Bacterial Metabolism. Second Edition, Springer-Verlag: New York (1986)). The<br>
35<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
ribulose monophosphate pathway involves the condensation of formate with ribulose-5-phosphate to form a 6-carbon sugar that becomes fructose and<br>
eventually the three-carbon product glyceraldehyde-3-phosphate. Likewise, the<br>
serine pathway assimilates the one-carbon compound into the glycolytic pathway <br>
5      via methylenetetrahydrofolate.<br>
In addition to one and two carbon substrates, methylotrophic microorganisms are also known to utilize a number of other carbon-containing compounds such as methylamine, glucosamine and a variety of amino acids for metabolic activity. For example, methylotrophic yeast are known to utilize the<br>
10     carbon from methylamine to form trehalose or glycerol (Bellion et al., Microb. Growth CI Compd, [Int. Symp.], 7th (1993), 415-32. Editor(s): Murrell, J. Collin; Kelly, Don P. Publisher: Intercept, Andover, UK). Similarly, various species of Candida will metabolize alanine or oleic acid (Suiter et al., Arch. Microbiol. 153(5), 485-489 (1990)). Hence, it is contemplated that the source of<br>
15 carbon utilized in the present invention may encompass a wide variety of carbon-containing substrates and will only be limited by the choice of microorganism or process.<br>
Although it is contemplated that all of the above mentioned carbon substrates and mixtures (co-feed) thereof are suitable in the present invention,<br>
20     preferred carbon substrates are glucose, fructose, sucrose, or methanol where the process intends to produce an endogenous glycerol, and glycerol or dihydroxyacetone where the process anticipates a glycerol or dihydroxyacetone feed.<br>
In addition to an appropriate carbon source, fermentation media must<br>
25 contain suitable minerals, salts, cofactors, buffers and other components, known to those skilled in the art, suitable for the growth of the cultures and promotion of the enzymatic pathway necessary for 1,3-propanediol production. Particular attention is given to Co(II) salts and/or vitamin Bj2 or precursors thereof.<br>
Adenosyi-cobalamin (coenzyme B12) is an essential cofactor for<br>
30     dehydratase activity. Synthesis of coenzyme B12 is found in prokaryotes, some of which are able to synthesize the compound de novo, for example, Escherichia blattae, Klebsiella species, Citrobacter species, and Clostridium species, while others can perform partial reactions. E. coli, for example, cannot fabricate the corrin ring structure, but is able to catalyze the conversion of cobinamide to<br>
35     corrinoid and can introduce the 5'-deoxyadenosyl group. Thus, it is known in the art that a coenzyme Bj2 precursor, such as vitarnin Bj2, need be provided in E. coli fermentations.-<br>
36<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
Vitamin B12 additions to E. coli fermentations may be added continuously, at a constant rate or staged as to coincide with the generation of cell mass, or may be added in single or multiple bolus additions. Preferred ratios of vitamin BI2 (mg) fed to cell mass (OD550) are from 0.06 to 0.60. Most preferred ratios of <br>
5   	  vitamin B12 (mg) fed to cell mass (OD550) are from 0.12 to 0.48.<br>
Although vitamin B 12 is added to the transformed E. coli of the present invention it is contemplated that other microorganisms, capable of de novo Bj2 biosynthesis will also be suitable production cells and the addition of B12 to these microorganisms will be unnecessary. <br>
10      	Culture Conditions:<br>
Typically cells are grown at 35 °C in appropriate media. Preferred growth media in the present invention are common commercially prepared media such as Luria Bertani (LB) broth, Sabouraud Dextrose (SD) broth or Yeast medium (YM) broth. Other defined or synthetic growth media may also be used and the <br>
15	appropriate medium for growth of the particular microorganism will be known by someone skilled in the art of microbiology or fermentation science. The use of agents known to modulate catabolite repression directly or indirectly, e.g., cyclic adenosine 2':3'-monophosphate, may also be incorporated into the reaction media. Similarly, the use of agents known to modulate enzymatic activities (e.g., methyl <br>
20  	   viologen) that lead to enhancement of 1,3-propanediol production may be used in conjunction with or as an alternative to genetic manipulations.<br>
Suitable pH ranges for the fermentation are between pH 5.0 to pH 9.0, where pH 6.0 to pH 8.0 is preferred as the initial condition.<br>
Reactions may be performed under aerobic or anaerobic conditions where 25     anaerobic or microaerobic conditions are preferred.<br>
Fed-batch fermentations may be performed with carbon feed, for example, glucose, limited or excess. Batch and Continuous Fermentations:<br>
The present process employs a batch method of fermentation. Classical <br>
30	batch fermentation is a closed system where the composition of the media is set at the beginning of the fermentation and is not subject to artificial alterations during the fermentation. Thus, at the beginning of the fermentation the media is inoculated with the desired microorganism or microorganisms and fermentation is permitted to occur adding nothing to the system. Typically, however,"batch" <br>
3 5  	   fermentation is batch with respect to the addition of carbon source and attempts are often made at controlling factors such as pH and oxygen concentration. In batch systems the metabolite and biomass compositions of the system change<br>
37<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
constantly up to the time the fermentation is stopped. Within batch cultures cells moderate through a static lag phase to a high growth log phase and finally to a stationary phase where growth rate is diminished or halted. If untreated, cells in the stationary phase will eventually die. Cells in log phase generally are <br>
5 	     responsible for the bulk of production of end product or intermediate.<br>
A variation on the standard batch system is the Fed-Batch system. Fed-Batch fermentation processes are also suitable in the present invention and comprise a typical batch system with the exception that the substrate is added in increments as the fermentation progresses. Fed-Batch systems are useful when<br>
10  	    catabolite repression is apt to inhibit the metabolism of the cells and where it is desirable to have limited amounts of substrate in the media. Measurement of the actual substrate concentration in Fed-Batch systems is difficult and is therefore estimated on the basis of the changes of measurable factors such as pH, dissolved oxygen and the partial pressure of waste gases such as CO2. Batch and Fed-Batch<br>
15      fermentations are common and well known in the art and examples may be found in Brock, supra.<br>
Although the present invention is performed in batch mode it is contemplated that the method would be adaptable to continuous fermentation methods. Continuous fermentation is an open system where a defined<br>
20     fermentation media is added continuously to a bioreactor and an equal amount of conditioned media is removed simultaneously for processing. Continuous<br>
fermentation generally maintains the cultures at a constant high density where<br>
cells are primarily in log phase growth.<br>
Continuous fermentation allows for the modulation of one factor or any<br>
25     number of factors that affect cell growth or end product concentration. For<br>
example, one method will maintain a limiting nutrient such as the carbon source or nitrogen level at a fixed rate and allow all other parameters to moderate. In other systems a number of factors affecting growth can be altered continuously while the cell concentration, measured by media turbidity, is kept constant.<br>
30      Continuous systems strive to maintain steady state growth conditions and thus the cell loss due to media being drawn off must be balanced against the cell growth rate in the fermentation. Methods of modulating nutrients and growth factors for continuous fermentation processes as well as techniques for maximizing the rate of product formation are well known in the art of industrial microbiology and a<br>
35     variety of methods are detailed by Brock, supra.<br>
It is contemplated that the present invention may be practiced using batch, fed-batch or continuous processes and that any known mode of fermentation<br>
38<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
would be suitable. Additionally, it is contemplated that cells may be immobilized on a substrate as whole cell catalysts and subjected to fermentation conditions for<br>
	1,3-propanediol production.<br>
Identification and purification of 1.3-propanediol:<br>
5	Methods for the purification of 1,3-propanediol from fermentation media<br>
are known in the art. For example, propanediols can be obtained from cell media by subjecting the reaction mixture to extraction with an organic solvent, distillation, and column chromatography (U.S. 5,356,812). A particularly good organic solvent for this process is cyclohexane (U.S. 5,008,473).<br>
10	1,3-Propanediol may be identified directly by submitting the media to high<br>
pressure liquid chromatography (HPLC) analysis. Preferred in the present invention is a method where fermentation media is analyzed on an analytical ion exchange column using a mobile phase ofO.OlN sulfuric acid in an isocratic fashion.<br>
15	EXAMPLES<br>
GENERAL METHODS<br>
Procedures for phosphorylations, ligations and transformations are well known in the art. Techniques suitable for use in the following examples may be found in Sambrook, J. et al., Molecular Cloning: A Laboratory Manual. Second<br>
20     Edition, Cold Spring Harbor Laboratory Press (1989).<br>
Materials and methods suitable for the maintenance and growth of bacterial cultures are well known in the art. Techniques suitable for use in the following examples may be found in Manual of Methods for General Bacteriology (Phillipp Gerhardt, R. G. E. Murray, Ralph N. Costilow, Eugene W. Nester, Willis<br>
25     A. Wood, Noel R. Krieg and G. Briggs Phillips, eds), American Society for<br>
Microbiology, Washington, D.C. (1994) or Thomas D. Brock in Biotechnoloev: A Textbook of Industrial Microbiology. Second Edition (1989) Sinauer Associates, Inc., Sunderland, MA. All reagents and materials used for the growth and maintenance of bacterial cells were obtained from Aldrich Chemicals<br>
30      (Milwaukee, WI), DIFCO Laboratories (Detroit, MI), GIBCO/BRL (Gaithersburg, MD), or Sigma Chemical Company (St. Louis, MO) unless otherwise specified.<br>
The meaning of abbreviations is as follows: "h" means hour(s), "min" means minute(s), "sec" means second(s), "d" means day(s), "mL" means<br>
milliliters, "L" means liters, 50 amp is 50 jig/mL ampicillin, and LB-50 amp is <br>
35     Luria-Bertani broth containing 50 μg/mL ampicillin.<br>
Within the tables the following abbreviations are used. "Con." is conversion, "Sel." is selectivity based on carbon, and "nd" is not detected.<br>
39<br><br>
WO 01/12833<br><br>
PCT/USOO/22874<br><br>
Strains and vectors used and constructed in the following examples are listed in the chart below:<br><br>
STRAIN/PLASMID	DELETION	ORF/GENE<br>
KLP23	gldA glpK	<br>
RJ8m	gldA glpK Tpi	<br>
pAH48		GPP2 DARl<br>
pDT29		dhaRorfYdhaTorfXorfWdhaBldhaB2dhaB3orfZ<br>
pKP32		dhaR orJY orpC orJWdhaBl dhaB2 dhaB3 orfl<br>
5	ENZYME ASSAYS<br>
Assays for dehydratase enzymes:<br>
Dehydratase activity in cell-free extracts was determined using either glycerol or 1,2-propanediol as substrate. Typically, cell-free extracts were prepared by cell disruption using a french press followed by centrifugation of the <br>
10      	cellular debris. The assay, based on the reaction of aldehydes with<br>
methylbenzo-2-thiazolone hydrazone, has been described by Forage and Foster {Biochim. Biophys. Acta 569, 249 (1979)).<br>
Honda et al. (J. Bacteriol. 143, 1458 (1980)) disclose an assay that measures the reactivation of dehydratases. Dehydratase activity was determined <br>
15     	in toluenized whole cells, with and without ATP, using either glycerol or<br>
1,2-propanediol as substrate. Reactivation was determined by the ratio of product formation with versus without the ATP addition. Product formation (3-HPA or<br>
40<br><br>
WO 01/12833<br><br>
PCT/USOO/22874<br><br>
propionaldehyde when glycerol or 1,2-propanediol is used as substrate,<br>
respectively) was measured directly, using HPLC, or indirectly, using the methyIbenzo-2-thiazolone hydrazone reagent. Alternatively, product formation was determined by coupling the conversion of the aldehyde to its respective <br>
5     	 alcohol using a NADH linked alcohol dehydrogenase and monitoring the disappearance of NADH. Assays for 1.3-propanediol oxidoreductase:<br>
The activity of 1,3-propanediol oxidoreductase, sometimes referred to as 1,3-propanediol dehydrogenase, was determined for cell-free extracts in solution<br>
10     or in slab gels using 1,3-propanediol and NAD+ as substrates has been described (Johnson and Lin, J. Bacterial. 169, 2050 (1987)). Alternatively, the conversion of 3-HPA and NADH to 1,3-propanediol and NAD+ was determined by the disappearance of NADH. The slab gel assay has the potential advantage of separating the activity of 1,3-propanediol oxidoreductase (dhaT) from that of non-<br>
15      specific alcohol dehydrogenases by virtue of size separation. The native<br>
molecular weights of 1,3-propanediol oxidoreductases (dhaT) from Citrobacter frendii, Klebsiella pneumoniae, and Clostridium pasteurianum are unusually large, on the order of 330,000 to 440,000 daltons. Lactobacillus brevis and Lactobacillus buchneri contain dehydratase associated 1,3-propanediol<br>
20     oxidoreductases with properties similar to those of known 1,3-propanediol oxidoreductases (dhaT). Assays for glycerol 3-phosphate dehydrogenase activity:<br>
A procedure was used asrhodified below" from a method published by Bell et al. (J. Biol Chem. 250, 7153 (1975)). This method involved incubating a cell-<br>
25      free extract sample in a cuvette that contained 0.2 mM NADH, 2.0 mM<br>
dihydroxyacetone phosphate (DHAP), and enzyme in 0.1 M Tris/HCl, pH 7.5 buffer with 5 mM DTT, in a total volume of 1.0 mL at 30 °C. A background rate of the reaction of enzyme and NADH was first determined at 340 nm for at least 3 min. The second substrate, DHAP, was subsequently added and the absorbance<br>
30     change over time was further monitored for at least 3 min. G3PDH activity was defined by subtracting the background rate from the gross rate. Assay for glvcerol-3-phosphatase activity.<br>
The assay for enzyme activity was performed by incubating the extract with an organic phosphate substrate in a bis-Tris or MES and magnesium buffer,<br>
35      pH 6.5. The substrate used was either 1-a-glycerol phosphate, or d,l-cc-glycerol phosphate. The final concentrations of the reagents in the assay are: buffer (20 mM, bis-Tris or 50 mM MES); MgCl2 (10 mM); and substrate (20 mM). If<br>
41<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
the total protein in the sample was low and no visible precipitation occurs with an acid quench, the sample was conveniently assayed in the cuvette. This method involved incubating an enzyme sample in a cuvette that contained 20 mM substrate (50 uL, 200 mM), 50 mM MES, 10 mM MgCl2, pH 6.5 buffer. The <br>
5    	 final phosphatase assay volume was 0.5 mL. The enzyme-containing sample was added to the reaction mixture; the contents of the cuvette were mixed and then the cuvette was placed in a circulating water bath at T = 37 °C for 5 to 120 min, the length of time depending on whether the phosphatase activity in the enzyme sample ranged from 2 to 0.02 U/mL. The enzymatic reaction was quenched by<br>
10     the addition of the acid molybdate reagent (0.4 mL). After the Fiske SubbaRow reagent (0.1 mL) and distilled water (1.5 mL) were added, the solution was mixed and allowed to develop. After 10 min, to allow full color development, the absorbance of the samples was read at 660 nm using a Cary 219 UV/vis spectrophotometer. The amount of inorganic phosphate released was compared to<br>
15      a standard curve that was prepared by using a stock inorganic phosphate solution (0.65 mM) and preparing 6 standards with final inorganic phosphate concentrations ranging from 0.026 to 0.130 (j.mol/mL. Assay for glycerol kinase activity:<br>
An appropriate amount of enzyme, typically a cell-free crude extract, was<br>
20     added to a reaction mixture containing 40 mM ATP, 20 mM MgS04,21 mM<br>
uniformly 13C labelled glycerol (99%, Cambridge Isotope Laboratories), and 0.1 M Tris-HCl, pH 9 for 75 min at 25 °C. The conversion of glycerol to glycerol 3-phosphate was detected by 13C-NMR (125 MHz): glycerol (63.11 ppm, 6, J = 41 Hz and 72.66 ppm, t, J= 41 Hz); glycerol 3-phosphate (62.93 ppm, 5,<br>
25      J = 41 Hz; 65.31 ppm, br d, J = 43 Hz; and 72.66 ppm, dt, J= 6, 41 Hz). NADH-linked glycerol dehydrogenase assay:<br>
NADH -linked glycerol dehydrogenase activity igldA) in cell-free extracts from E. coli strains was determined after protein separation by non-denaturing polyacrylamide gel electrophoresis. The conversion of glycerol plus NAD+ to<br>
30      dihydroxyacetone plus NADH was coupled with the conversion of 3-[4,5-<br>
dimethylthiazol-2-yl]-2,5-diphenyltetrazolium bromide (MTT) to a deeply colored formazan, using phenazine methosulfate (PMS) as mediator (Tang et al., J.Bacteriol.]40,m(\997)).<br>
Electrophoresis was performed in duplicate by standard procedures using<br>
3 5      native gels (8-16% TG, 1.5 mm, 15 lane gels from Novex, San Diego, CA).<br>
Residual glycerol was removed from the gels by washing 3x with 50 mM Tris or potassium carbonate buffer, pH 9 for 10 min. The duplicate gels were developed,<br>
42<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
with and without glycerol (approximately 0.16 M final concentration), in 15 mL of assay solution containing 50 mM Tris or potassium carbonate, pH 9,60 mg<br>
ammonium sulfate, 75 mg NAD+, 1.5 mg MTT, and 0.5 mg PMS.<br>
The presence or absence of NADH -linked glycerol dehydrogenase activity <br>
5    	 in E. coli strains (gldA) was also determined, following polyacrylamide gel electrophoresis, by reaction with polyclonal antibodies raised to purified K. pneumoniae glycerol dehydrogenase (dhaD). Isolation and identification of 1.3-propanediol:<br>
The conversion of glycerol to 1,3-propanediol was monitored by HPLC.<br>
10     Analyses were performed using standard techniques and materials available to one of skill in the art of chromatography. One suitable method utilized a Waters Maxima 820 HPLC system using UV (210 nm) and RJ detection. Samples were injected onto a Shodex SH-1011 column (8 mm x 300 mm, purchased from Waters, Milford, MA) equipped with a Shodex SH-101 IP precolumn (6 mm x<br>
15      50 mm), temperature controlled at 50 °C, using 0.01 N H2S04 as mobile phase at a flow rate of 0.5 mL/min. When quantitative analysis was desired, samples were prepared with a known amount of trimethylacetic acid as external standard. Typically, the retention times of glucose (RI detection), glycerol, 1,3-propanediol (RI detection), and trimethylacetic acid (UV and RI detection) were 15.27 min,<br>
20     20.67 min, 26.08 min, and 35.03 min, respectively.<br>
Production of 1,3-propanediol was confirmed by GC/MS. Analyses were performed using standard techniques and materials available to one of skill in the art of GC/MS. One suitable method utilized a Hewlett Packard 5890 Series II gas chromatograph coupled to a Hewlett Packard 5971 Series mass selective detector<br>
25 (EI) and a HP-INNOWax column (30 m length, 0.25 mm i.d., 0.25 micron film thickness). The retention time and mass spectrum of 1,3-propanediol generated were compared to that of authentic 1,3-propanediol (m/e: 57,58).<br>
An alternative method for GC/MS involved derivatization of the sample. To 1.0 mL of sample (e.g., culture supernatant) was added 30 uL of concentrated<br>
30      (70% v/v) perchloric acid. After mixing, the sample was frozen and lyophilized. A 1:1 mixture of bis(trimethylsilyl)trifluoroacetamide:pyridine (300 uL) was added to the lyophilized material, mixed vigorously and placed at 65 °C for one h. The sample was clarified of insoluble material by centrifugation. The resulting liquid partitioned into two phases, the. upper of which was used for analysis. The<br>
35     sample was chromatographed on a DB-5 column (48 m, 0.25 mm I.D., 0.25 urn film thickness; from J&amp;W Scientific) and the retention time and mass spectrum of the 1,3-propanediol derivative obtained from culture supernatants were compared<br>
43<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
to that obtained from authentic standards. The mass spectra of TMS-derivatized 1,3-propanediol contains the characteristic ions of 205,177,130 and 115 AMU. Cell Lvsis:<br>
Cell lysis was estimated by measuring the extracellular soluble protein <br>
5	concentration in the fermentation broth. Fermenter samples were centrifuged in a desktop centrifuge (typically, 3-5 min at 12,000 rpm in an Eppendorf, Model 5415C micro centrifuge) in order to separate cells. The resulting supernatant was analyzed for protein concentration by the Bradford method using a commercially available reagent (Bio-Rad Protein Assay, Bio-Rad, Hercules, CA). <br>
10     Viability:<br>
Cell viability was determined by plating, at appropriate dilutions, cells<br>
obtained from the fermenter on non-selective LB agar plates. Cell viability<br>
between fermenter experiments is compared by using the ratio of viable cells per<br>
mL of fermenter broth divided by OD550 (AU).<br>
15	EXAMPLE 1<br>
CLONING AND TRANSFORMATION OF E. COLI HOST CELLS WITH COSMID DNA FOR THE EXPRESSION OF 1.3-PROPANEDIOL Media:<br>
Synthetic S12 medium was used in the screening of bacterial transformants 20     for the ability to make 1,3-propanediol. S12 medium contains: l0mM<br>
ammonium sulfate, 50 mM potassium phosphate buffer, pH 7.0, 2 mM MgCl2, 0.7 mM CaCl2, 50 μM MnCl2, 1 uM FeCl3,1 μM ZnCl, 1.7 μM CuS04,2.5 μM CoCl2, 2.4 p.M Na2Mo04, and 2 μM thiamine hydrochloride.<br>
Medium A used for growth and fermentation consisted of: 10 mM <br>
25    	 ammonium sulfate; 50 mM MOPS/KOH buffer, pH 7.5; 5 mM potassium<br>
phosphate buffer, pH 7.5; 2 mM MgCl2; 0.7 mM CaCl2; 50 uM MnCl2; 1 uM FeCl3; 1 μM ZnCl; 1.72 μM CuS04; 2.53 μM CoCl2; 2:42 p.M Na2Mo04; 2 μM thiamine hydrochloride; 0.01% yeast extract; 0.01% casamino acids; 0.8 ug/rnL vitamin Bj2; and 50 μg/mL amp. Medium A was supplemented with either 0.2% <br>
30      	glycerol or 0.2% glycerol plus 0.2% D-glucose as required. Cells:<br>
Klebsiella pneumoniae ECL2106 (Ruch et al., J. Bacteriol. 124, 348<br>
(1975)), also known in the literature as K. aerogenes oiAerobacter aerogenes, was obtained from E. C. C. Lin (Harvard Medical School, Cambridge, MA) and<br>
35     was maintained as a laboratory culture.<br>
Klebsielh^newnomae ATCC 25955 was purchased from American Type " Culture Collection (Manassas, VA).<br>
44<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
E. coli DH5cc was purchased from Gibco/BRL and was transformed with the cosmid DNA isolated from Klebsiella pneumoniae ATCC 25955 containing a<br>
gene coding for either a glycerol or diol dehydratase enzyme. Cosmids containing the glycerol dehydratase were identified as pKPl and pKP2 and cosmid <br>
5  	    containing the diol dehydratase enzyme were identified as pKP4. Transformed DH5α cells were identified as DH5α-pKPl, DH5α-pKP2, and DH5α-pKP4.<br>
&amp; coli ECL707 (Sprenger et al.s J. Gen. Microbiol. 135,1255 (1989)) was obtained from E. C. C. Lin (Harvard Medical School, Cambridge, MA) and was similarly transformed with cosmid DNA from Klebsiella pneumoniae. These<br>
10     transformants were identified as ECL707-pKP 1 and ECL707-pKP2, containing the glycerol dehydratase gene and ECL707-pKP4 containing the diol dehydratase gene.<br>
E. coli AA200 containing a mutation in the tpi gene (Anderson et al., J. Gen. Microbiol. 62,329 (1970)) was purchased from the E. coli Genetic Stock<br>
15      Center, Yale University (New Haven, CT) and was transformed with Klebsiella cosmid DNA to give the recombinant microorganisms AA200-pKPl and AA200-pKP2, containing the glycerol dehydratase gene, and AA200-pKP4, containing the diol dehydratase gene. DH5α:<br>
20	Six transformation plates containing approximately 1,000 colonies of<br>
E. coli XL 1-Blue MR transfected with K pneumoniae DNA were washed with 5 mL LB medium and centrifuged. The bacteria were pelleted and resuspended in 5 mL LB medium + glycerol. An aliquot (50 uL) was inoculated into a 15 mL tube containing S12 synthetic medium with 0.2% glycerol + 400 ng per mL of<br>
25      vitamin B12 + 0.001% yeast extract + 50 amp. The tube was filled with the medium to the top and wrapped with parafilm and incubated at 30 °C. A slight turbidity was observed after 48 h. Aliquots, analyzed for product distribution as described above at 78 h and 132 h, were positive for 1,3-propanediol, the later time points containing increased amounts of 1,3-propanediol.<br>
30	The bacteria, testing positive for 1,3-propanediol production, were serially<br>
diluted and plated onto LB-50 amp plates in order to isolate single colonies. Forty-eight single colonies were isolated and checked again for the production of 1,3-propanediol. Cosmid DNA was isolated from 6 independent clones and transformed into E. coli strain DH5a. The transformants were again checked for<br>
35      the production of 1,3-propanediol. Two transformants were characterized further and designated as DH5ct-pKPl and DH5α-pKP2<br>
45<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
A 12.1 kb EcoRI-Sall fragment from pKPl, subcloned into pIBDl (BI Biosystem, New Haven, CT), was sequenced and termed pHK28-26 (SEQ ID<br>
NO: 1). Sequencing revealed the loci of the relevant open readmg frames of the dha operon encoding glycerol dehydratase and genes necessary for regulation. <br>
5  	   Referring to SEQ ID NO: 1, a fragment of the open reading frame for dhaKl encoding dihydroxyacetone kinase is found at bases 1-399 (complement); the open reading frame dhaD encoding glycerol dehydrogenase is found at bases 1010-2107; the open reading frame dhaR encoding the repressor is found at bases 2209-4134; the open reading frame orJW, encoding a protein of unknown function<br>
10     is found at bases 4112-4642 (complement); the open reading frame orflC encoding a dehydratase reactivation protein is found at bases 4643-4996 (complement); the<br>
open reading frame dhaT encoding 1,3-propanediol oxidoreductase is found at bases 5017-6180 (complement); the open reading frame or/Y, encoding a protein of unknown function is found at bases 6202-6630 (complement); the open reading<br>
15     frame dhaBl encoding the alpha subunit glycerol dehydratase is found at bases 7044-8711; the open reading frame dhaB2 encoding the beta subunit glycerol dehydratase is found at bases 8724-9308; the open reading frame dhaB3 encoding the gamma subunit glycerol dehydratase is found at bases 9311-9736; the open reading frame dhaBX, encoding a dehydratase reactivation protein is found at<br>
20     bases 9749-11572; and a fragment of the open reading frame for glpF encoding a glycerol uptake facilitator protein is found at bases 11626-12145.<br>
Single colonies of E. coli XLl-Blue MR transfected with packaged cosmid DNA from K. pneumoniae were inoculated into microtiter wells containing 200 μL of S15 medium (ammonium sulfate, 10 mM; potassium phosphate buffer,<br>
25     pH 7.0, 1 mM; MOPS/KOH buffer, pH 7.0, 50 mM; MgCl2,2 mM; CaCl2,<br>
0.7 mM; MnCl2, 50 μM; FeCl3, 1 μM; ZnCl, 1 μM; CuS04, 1.72 μM; CoCl2, 2.53 μM; Na2Mo04,2.42 μM; and thiamine hydrochloride, 2 μM) + 0.2% glycerol + 400 ng/mL of vitamin Bj2 + 0.001% yeast extract + 50 μg/mL ampicillin. In addition to the microtiter wells, a master plate containing<br>
30     LB-50 amp was also inoculated. After 96 h, 100 μL was withdrawn and<br>
centrifuged in a Rainin microfuge tube containing a 0.2 micron nylon membrane filter. Bacteria were retained and the filtrate was processed for HPLC analysis. Positive clones demonstrating 1,3-propanedioLproduction were identified after<br>
screening approximately 240 colonies. Three positive clones were identified, two <br>
35    	 of which had grown on LB-50 amp and one of which had not. A single colony, isolated from one of the two positive clones grown on LB-50 amp and verified for the production of 1,3-propanediol, was designated as pKP4. Cosmid DNA was<br>
46<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
   isolated from E. coli strains containing pKP4 and E. coli strain DH5a was<br>
   transformed. An independent transformant, designated as DH5a-pKP4, was<br>
    verified for the production of 1,3-propanediol.<br>
     ECL707:<br>
5	E. coli strain ECL707 was transformed with cosmid K. pneumoniae DNA<br>
corresponding to one of pKPl, pKP2, pKP4 or the Supercos vector alone and named ECL707-pKPl, ECL707-pKP2, ECL707-pKP4, and ECL707-SC, respectively. ECL707 is defective in glpK, gld, andptsD which encode the ATP-dependent glycerol kinase, NAD+-linked glycerol dehydrogenase, and<br>
10     enzyme II for dihydroxyacetone of the phosphoenolpyruvate-dependent phosphotransferase system, respectively.<br>
Twenty single colonies of each cosmid transformation and five of the Supercos vector alone (negative control) transformation, isolated from LB-50 amp plates, were transferred to a master LB-50 amp plate. These isolates were also<br>
15      tested for their ability to convert glycerol to 1,3-propanediol in order to determine if they contained dehydratase activity. The transformants were transferred with a sterile toothpick to microtiter plates containing 200 uL of Medium A supplemented with either 0.2% glycerol or 0.2% glycerol plus 0.2% D-glucose. After incubation for 48 h at 30 °C, the contents of the microtiter plate wells were<br>
20     filtered through a 0.45 micron nylon filter and chromatographed by HPLC. The results of these tests are given in Table 2.<br>
	TABLE 2      ...<br>
	Conversion of glycerol to 1,3-propanediol by transformed ECL707	<br>
transformant	glycerol*	glycerol plus glucose*<br>
ECL707-pKPl	19/20	19/20<br>
ECL707-pKP2	18/20	20/20<br>
ECL707-pKP4	0/20	20/20<br>
ECL707-sc	0/5	0/5<br>
* (Number of positive isolates/number of isolates tested)<br>
AA200:<br>
25	E. coli strain AA200 was transformed with cosmid K. pneumoniae DNA<br>
corresponding to one of pKPl, pKP2, pKP4 and the Supercos vector alone and<br>
named AA200-pKPl, AA200-pKP2, AA200-pKP4, and AA200-sc, respectively. Strain AA200 is defective in triosephosphate isomerase (tpf).<br>
Twenty single colonies of each cosmid transformation and five of the 30     empty vector transformation were isolated and tested for their ability to convert<br>
47<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
glycerol to 1,3-propanediol as described for E. coli strain ECL707. The results of these tests are given in Table 3.<br>
TABLE 3<br>
	Conversion of glycerol to 1,3-propanediol by transformed AA2Q0	<br>
transformant	glycerol*	glycerol plus glucose*<br>
AA200-pKPl	17/20	17/20<br>
AA200-pKP2	17/20	17/20<br>
AA200-pKP4	2/20	16/20<br>
AA200-sc	0/5	0/5<br>
*(Number of positive isolates/number of isolates tested)<br>
5	EXAMPLE 2<br>
ENGINEERING OF GLYCEROL KINASE MUTANTS OF E. COLI FM5 FOR PRODUCTION OF GLYCEROL FROM GLUCOSE Construction of integration plasmid for glycerol kinase gene replacement in E. coli FM5:<br>
10	E. coli FM5 (ATCC 53911) genomic DNA was prepared using the<br>
Puregene DNA Isolation Kit (Gentra Systems, Minneapolis, MN). A 1.0 kb DNA fragment containing partial glpF and glycerol kinase (glpK) genes was amplified by PCR (Mullis and Faloona, Methods Enzymol. 155, 335 (1987)) from FM5 genomic DNA using primers SEQ ID NO:2 and SEQ ID NO:3. A 1.1 kb DNA<br>
15      fragment containing partial glpK and glpX genes was amplified by PCR from<br>
FM5 genomic DNA using primers SEQ ID NO:4 and SEQ ID NO:5. A Muni site was incorporated into primer SEQ ID NO:4. The 5' end of primer SEQ ID NO:4 was the reverse complement of primer SEQ ID NO:3 to enable subsequent overlap extension PCR. The gene splicing by overlap extension technique (Horton et al.,<br>
20     BioTechniques 8, 528 (1990)) was used to generate a 2.1 kb fragment by PCR using the above two PCR fragments as templates anu prim ers SEQ ID NO:2 and SEQ ID NO:5. This fragment represented a deletion of 0.8 kb from the central region of the 1.5 kb glpK gene. Overall, this fragment had 1.0 kb and 1.1 kb flanking regions on either side of the Muni cloning site (within the partial glpK) to<br>
25      allow for chromosomal gene replacement by homologous recombination. The above 2.1 kb PCR fragment was blunt-ended (using mung bean nuclease) and cloned into the pCR-Blunt vector using the Zero Blunt PCR Cloning Kit (Invitrogen, San Diego, CA) to yield the 5.6 kb plasmid pRNlOO containing karamycin and Zeocin resistance genes. The 1.2 kb Hindi fragment<br>
30     from pLoxCatl (unpublished results), containing a chloramphenicol-resistance<br>
48<br><br>
WO 01/12833<br><br>
PCT/USOO/22874<br><br>
gene flanked by bacteriophage PI IoxP sites (Snaith et al., Gene 166,173 (1995)), was used to interrupt the glpK fragment in plasmid pRNlOO by ligating it to<br>
Munldkested (and blunt-ended) plasmid pKNIOO to yield the 6.9 kb plasmid<br>
pRN101-l. A 376 bp fragment containing the R6K origin was amplified by PCR <br>
5    	 from the vector pGP704 (Miller and Mekalanos, J. Bacteriol. 170,2575-25 83 (1988)) using primers SEQ ID NO:6 and SEQ ID NO:7, blunt-ended, and ligated to the 5.3 kbAspllS-Aatll fragment (which was blunt-ended) from pRN101-l to yield the 5.7 kb plasmid pRN102-l containing kanamycin and chloramphenicol resistance genes. Substitution of the ColEl origin region in pRN101-l with the<br>
10     R6K origin to generate pRN 102-1 also involved deletion of most of the Zeocin resistance gene. The host for pRN102-l replication was E. coli SY327 (Miller and Mekalanos, J. Bacteriol. 170,2575-2583 (1988)) which contains thepir gene necessary for the function of the R6K origin. Engineering of glycerol kinase mutant RJFl0m with chloramphenicol resistance<br>
15     gene interrupt:<br>
E.coli FM5 was electrotransformed with the non-replicative integration plasmid pRN 102-1 and transformants that were chloramphenicol-resistant (12.5 μg/mL) and kanamycin-sensitive (30 μg/mL) were further screened for glycerol non-utilization on M9 minimal medium containing 1 mM glycerol. An<br>
20     EcoRI digest of genomic DNA from one such mutant, RJFl 0m, when probed with the intact glpK gene via Southern analysis (Southern, J. Mol. Biol. 98, 503-517 (1975)) indicated that it was a double-crossover integrant (glpK gene replacement) since the two expected 7.9 kb and 2.0 kb bands were observed, owing to the presence of an additional EcoRI site within the chloramphenicol resistance gene.<br>
25     The wild-type control yielded the single expected 9.4 kb band. A 13C NMR analysis of mutant RJFlOm confirmed that it was incapable of converting 13C-labeled glycerol and ATP to glycerol-3-phosphate. This glpK mutant was further analyzed by genomic PCR using primer combinations SEQ ID NO:8 and SEQ ID NO:9, SEQ ID NO: 10 and SEQ ID NO: 11, and SEQ ID NO:8 and SEQ<br>
30     ID NO: 11 which yielded the expected 2.3 kb, 2.4 kb, and 4.0 kb PCR fragments respectively. The wild-type control yielded the expected 3.5 kb band with primers SEQ ID NO:8 and SEQ ID NO:l 1. The glpK mutant RJFlOm was electrotransformed with plasmid pAH48 to allow glycerol production from glucose. The glpK mutant E. coli RJFlOm has been deposited with ATCC under<br>
35     the terms of the Budapest Treaty on 24 November 1997.<br>
49<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
Engineering of glycerol kinase mutant RJF10 with chloramphenicol resistance gene interrupt removed:<br>
After overnight growth on YENB medium (0.75% yeast extract, 0.8%<br>
nutrient broth) at 37 °C, E. coli RJFlOm in a water suspension was<br>
5 	    electrotransformed with plasmid pJW 168 (unpublished results), which contained the bacteriophage PI Cre recombinase gene under the control of the IPTG-inducible lac\JV5 promoter, a temperature-sensitive pSClOl replicon, and an ampicillin resistance gene. Upon outgrowth in SOC medium at 30 °C, transformants were selected at 30 °C (permissive temperature for pJW168<br>
10 	    replication) on LB agar medium supplemented with carbenicillin (50 ng/mL) and IPTG (1 mM). Two serial overnight transfers of pooled colonies were carried out at 30 °C on fresh LB agar medium supplemented with carbenicillin and IPTG in order to allow excision of the chromosomal chloramphenicol resistance gene via recombination at the loxP sites mediated by the Cre recombinase (Hoess and<br>
15   	   Abremski, J. Mol. Biol. 181, 351-362 (1985)). Resultant colonies were replica-plated on to LB agar medium supplemented with carbenicillin and IPTG and LB agar supplemented with chloramphenicol (12.5 μg/mL) to identify colonies that were carbenicillin-resistant and chloramphenicol-sensitive indicating marker gene removal. An overnight 30 °C culture of one such colony was used to inoculate<br>
20     10 mL of LB medium. Upon growth at 30 °C to OD (600 nm) of 0.6 AU, the culture was incubated at 37 °C overnight. Several dilutions were plated on prewarmed LB agar medium and the plates incubated overnight at 42 °C (the non-permissive temperature for pJW168 replication). Resultant colonies were replica-plated on to LB agar medium and LB agar medium supplemented with<br>
25       carbenicillin (75 μg/mL) to identify colonies that were carbenicillin-sensitive indicating loss of plasmid pJWl 68. One such glpK mutant, RJFIO, was further analyzed by genomic PCR using primers SEQ ID NO:8 and SEQ ID NO:l 1 and yielded the expected 3.0 kb band confirming marker gene excision. Glycerol non-utilization by mutant RJF10 was confirmed by lack of growth on M9 minimal<br>
30     medium containing 1 mM glycerol. The glpK mutant RJF10 was<br>
electrotransformed with plasmid pAH48 to allow glycerol production from glucose.<br>
EXAMPLE 3<br>
 CONSTRUCTION OF E. COLI STRAIN WITH gldA GENE KNOCKOUT<br>
35	The gldA gene was isolated from E. coli by PCR (K. B. Mullis and F. A.<br>
Faloona, Meth. Enzymol. 155,335-350 (1987)) using primers SEQ ID NO: 12 and SEQ ID NO: 13, which incorporate terminal Sphl and Xbal sites, respectively,<br>
50<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
and cloned (T. Maniatis (1982) Molecular Cloning: A Laboratory Manual. Cold Spring Harbor, Cold Spring Harbor, NY) between the Sphl and Xbal sites in pUCl8, to generate pKP8. pKP8 was cut at the unique Sail and Ncol sites within the gldA gene, the ends flushed with Klenow and religated, resulting in a 109 bp 5     deletion in the middle of gldA and regeneration of a unique Sail site, to generate pKP9. A 1.4 kb DNA fragment containing the gene conferring kanamycin resistance (kan), and including about 400 bps of DNA upstream of the translational start codon and about 100 bps of DNA downstream of the translational stop codon, was isolated from pET-28a(+) (Novagen, Madison, Wis)<br>
10     by PCR using primers SEQ ID NO: 14 and SEQ ID NO: 15, which incorporate terminal Sail sites, and subcloned into the unique Sail site of pKP9, to generate pKP13. A 2.1 kb DNA fragment beginning 204 bps downstream of the gldA translational start codon and ending 178 bps upstream of the gldA translational stop codon, and containing the kan insertion, was isolated from pKP13 by PCR<br>
15      using primers SEQ ID NO: 16 and SEQ ID NO: 17, which incorporate terminal Sphl and Xbal sites, respectively, was subcloned between the Sphl and Xbal sites in pMAK705 (Genencor International, Palo Alto, CA), to generate pMP33. E. coli FM5 was transformed with pMP33 and selected on 20 ng/mL kan at 30 °C, which is the permissive temperature for pMAK705 replication. One colony was<br>
20     expanded overnight at 30 °C in liquid media supplemented with 20 p,g/mL kan. Approximately 32,000 cells were plated on 20 \xgfmL kan and incubated for 16 h at 44 °C, which is the restrictive temperature for pMAK705 replication, Transformants growing at 44 °C have plasmid integrated into the chromosome, occurring at a frequency of approximately 0.0001. PCR and Southern blot (E.M.<br>
25      Southern, J. Mol. Biol. 98, 503-517 (1975)) analyses were Used to determine the nature of the chromosomal integration events in the transformants. Western blot analysis (Towbin et al., Proc. Natl. Acad. Sci. 76,4350 (1979)) was used to determine whether glycerol dehydrogenase protein, the product of gldA, is produced in the transformants. An activity assay was used to determine whether<br>
30      glycerol dehydrogenase activity remained in the transformants. Activity in glycerol dehydrogenase bands on native gels was determined by coupling the conversion of glycerol plus NAD+ to dihydroxyacetone plus NADH to the conversion of a tetrazolium dye, MTT [3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide] to a deeply colored formazan, with phenazine<br>
35     methosulfate as mediator. Glycerol dehydrogenase also requires the presence of 30 mM ammonium sulfate and 100 mM Tris, pH 9 (Tang et al., J. Bacteriol. 140, 182 (1997)). Of 8 transformants analyzed, 6 were determined to be gldA<br>
51<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
knockouts. E. coli MSP33.6 has been deposited with ATCC under the terms of the Budapest Treaty on 24 November 1997.<br>
EXAMPLE 4<br>
CONSTRUCTION OF AN E. COLI STRAIN<br>
5	WITH eloK AND eldA GENE KNOCKOUTS<br>
A 1.6 kb DNA fragment containing the gldA gene and including 228 bps of DNA upstream of the translational start codon and 220 bps of DNA downstream of the translational stop codon was isolated from E. coli by PCR using primers SEQ ID NO: 18 and SEQ ID NO: 19, which incorporate terminal<br>
10     Sphl and Xbal sites, respectively, and cloned between the Sphl and Xbal sites of pUC18, to generate pQN2. pQN2 was cut at the unique Sail and Ncol sites within the gldA gene, the ends flushed with Klenow and religated, resulting in a 109 bps deletion in the middle of gldA and regeneration of a unique Sail site, to generate pQN4. A 1.2 kb DNA fragment containing the gene conferring<br>
15     kanamycin resistance (kan), and flanked by loxP sites was isolated from<br>
pLoxKan2 (Genencor International, Palo Alto, CA) as a Stul/Xhol fragment, the ends flushed with Klenow, and subcloned into pQN4 at the Sail site after flushing with Klenow, to generate pQN8. A 0.4 kb DNA fragment containing the R6K origin of replication was isolated from pGP704 (Miller and Mekalanos,<br>
20      J. Bacteriol. 170, 2575-2583 (1988)) by PCR using primers SEQ ID NO:20 and SEQ ID NO:21, which incorporate terminal Sphl and Xbal sites, respectively, and ligated to the 2.8 kb Sphl/Xbal DNA fragment containing the gldA::kan cassette from pQN8, to generate pKP22. A 1.0 kb DNA fragment containing the gene conferring chloramphenicol resistance (cam), and flanked by loxP sites was<br>
25     isolated from pLoxCat2 (Genencor International, Palo Alto, CA) as an Xbal<br>
fragment, and subcloned into pKP22 at the Xbal site, to generate pKP23. E. col strain RJF10 (see Example 2), which is glpK-, was transformed with pKP23 and transformants with the phenotype kanRcamS were isolated, indicating double crossover integration, which was confirmed by southern blot analysis. Glycerol<br>
30     dehydrogenase gel activity assays (as described in Example 3) demonstrated thai active glycerol dehydrogenase was not present in these transformants. The kan marker was removed from the chromosome using the Cre-producing plasmid pJW168, as described in Example 2, to produce strain KLP23. Several isolates<br>
with the phenotype kanS demonstrated no glycerol dehydrogenase activity, and<br>
35      southern blot analysis confirmed loss of the kan marker.<br>
52<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
EXAMPLE 5 <br>
PLASMID CONSTRUCTION AND STRAIN CONSTRUCTION FOR THE EXPRESSION OF GLYCEROL 3-PHOSPHATE DEHYDROGENASE (DAR1) AND/OR GLYCEROL 3-PHOSPHATASE CGPP2) <br>
5     	Construction of expression cassettes for glycerol 3-phosphatase (gpp2): The Saccharomyces cerevisiae chromosomeV lamda clone 6592 (GenBank, accession # U18813xl 1) was obtained from ATCC. The glycerol 3-phosphate phosphatase gene (GPP2) was cloned by cloning from the lamda clone as target DNA using synthetic primers (SEQ ID NO:22 with SEQ ID<br>
10     NO:23) incorporating an BamHI-RBS-Xbal site at the 51 end and a Smal site at the 3' end. The product was subcloned into pCR-Script (Stratagene, Madison, WI) at the Srfl site to generate the plasmid pAH15 containing GPP2. The plasmid pAH15 contains the GPP2 gene in the inactive orientation for expression from the lac promoter in pCR-Script SK+. The BamHI-Smal fragment from pAH15<br>
15     containing the GPP2 gene was inserted into pBlueScriptll SK+ to generate<br>
plasmid pAH19. The pAH19 contains the GPP2 gene in the correct orientation for expression from the lac promoter. The Xbal-PstI fragment from pAH19 containing the GPP2 gene was inserted into pPHOX2 to create plasmid pAH21. The pAH21/DH5a is the expression plasmid.<br>
20     Construction of expression cassettes for glycerol 3-phosphate dehydrogenase (DAR1):<br>
DAR1 was isolated by PCR cloning from genomic S. cerevisiae DNA using synthetic primers (SEQ ID NO:24 with SEQ ID NO:25). Successful PCR cloning places an Ncol site at the 5' end of DAR1 where the ATG within Ncol is<br>
25     the DAR1 initiator methionine. At the 3' end of DAR1 a BamHI site is introduced following the translation terminator. The PCR fragments were digested with Ncol + BamHI and cloned into the same sites within the expression plasmid pTrc99A (Pharmacia, Piscataway, NJ) to give pDARlA.<br>
In order to create a better ribosome binding site at the 5' end of DARi, an<br>
30      Spel-RBS-NcoI linker obtained by annealing synthetic primers (SEQ ID NO:26 with SEQ ID NO:27) was inserted into the Ncol site of pDARIA to create pAH40. Plasmid pAH40 contains the new RBS and DAR1 gene in the correct orientation for expression from the trc promoter of pTrc99A (Pharmacia, Piscataway, NJ). The NcoI-BamHI fragment from pDARl A and an second set of<br>
35      Spel-RBS-NcoI linker obtained by annealing synthetic primers (SEQ ID NO:28 with SEQ ID NO:29) was inserted into the Spel-BamHI site of pBC-SK+<br>
53<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
(Stratagene, Madison, WI) to create plasmid pAH42. The plasmid pAH42 contains a chloramphenicol resistant gene.<br>
Construction of expression cassettes for darl and gpp2:<br>
Expression cassettes for DARl and GPP2 were assembled from the <br>
5      	individual DARl and GPP2 subclones described above using standard molecular biology methods. The BamHI-PstI fragment from pAH19 containing the<br>
ribosomal binding site (RBS) and GPP2 gene was inserted into pAH40 to create<br>
pAH43. The BamHI-PstI fragment from pAH19 containing the RBS and GPP2<br>
gene was inserted into pAH42 to create pAH45.<br>
10	The ribosome binding site at the 5' end of GPP2 was modified as follows.<br>
A BamHI-RBS-Spel linker, obtained by annealing synthetic primers<br>
GATCCAGGAAACAGA (SEQ ID NO:30) with CTAGTCTGTTTCCTG (SEQ<br>
ID NO:31) to the Xbal-PstI fragment from pAH19 containing the GPP2 gene, was<br>
inserted into the BamHI-PstI site of pAH40 to create pAH48. Plasmid pAH48 <br>
15     contains the DARl gene, the modified RBS, and the GPP2 gene in the correct<br>
orientation for expression from the trc promoter of pTrc99A (Pharmacia,<br>
Piscataway, NJ). <br>
Transformation of E. coli:<br>
The plasmids described here were transformed into E. coli DH5oc, FM5 <br>
20   	  and KLP23 using standard molecular biology techniques. The transformants were<br>
verified by their DNA RFLP pattern.<br>
EXAMPLE 6<br>
CONSTRUCTION OF EXPRESSION PLASMIDS FOR USE IN<br>
TRANSFORMATION OF ESCHERICHIA COLI WITH GENES FROM THE<br>
25	KLEBSIELLA PNEUMONIAE dha REGULON<br>
Construction of the expression vector pTacIO:<br>
The E. coli expression vector pTacIQ was prepared by inserting laclq gene (Farabaugh, Nature 274(5613), 765-769 (1978)) and tac promoter (Amann et al., Gene 25, 167-178 (1983)) into the restriction endonuclease site EcoRI of pBR322 <br>
30     	(Sutcliffe, Cold Spring Harb. Symp. Quant. Biol. 43, 77-90 (1979)). A multiple cloning site and terminator sequence (SEQ ID NO:32) replaces the pBR322 sequence from EcoRI to Sphl. Subcloning the glycerol dehydratase genes (dhaBJ-,2,3, X):<br>
The open reading frame for the dhaB3 gene was amplified from pHK28-26 <br>
35	     by PCR using primers (SEQ ID NO:33 and SEQ ID NO:34) incorporating an EcoRI site at the 5' end and a Xbal site at the 3' end. The product was subcloned<br>
54<br><br>
into PLitmus29 (New England Biolab, Inc., Beverly, MA) to generate the plasmid pDHAB3 containing dhaB3.<br>
The region containing the entire coding region for dhaBl, dhaB2, dhaB3 and dhaBX of the dhaB operon from pHK28-26 was cloned into pBluescriptIIKS+ (Stratagene, La Jolla, CA) using the restriction enzymes Kpnl and EcoRI to create the plasmid pM7.<br>
The dhaBX gene was removed by digesting plasmid pM7 with Apal and Xbal, purifying the 5.9 kb fragment and ligating it with the 325-bp Apal-Xbal fragment from plasmid pDHAB3 to create pMl 1 containing dhaBl, dhaB2 and <br>
10     	dhaB3,<br>
The open reading frame for the dhaBl gene was amplified from pHK28-26 by PCR using primers (SEQ ID NO:35 and SEQ ID NO:36) incorporating a Hindlll site and a consensus ribosome binding site at the 5' end and a Xbal site at the 31 end. The product was subcloned into pLitmus28 (New England Biolab, Inc., Beverly, MA) to generate the plasmid pDTl containing dhaBl.<br>
A Notl-Xbal fragment from pMl 1 containing part of the dhaBl gene, the dhaB2 gene and the dhaB3 gene was inserted into pDTl to create the dhaB expression plasmid, pDT2. The Hindlll-Xbal fragment containing the dhaB(l,2,3) genes from pDT2 was inserted into pTacIQ to create pDT3. <br>
20     	Subcloning the U3-propanediol dehydrogenase gene (dhaT):<br>
The KpnI-SacI fragment of pHK28-26, containing the 1,3-propanediol dehydrogenase (dhaT) gene, was subcloned into pBluescriptll KS+ creating plasmid pAHl. The dhaT gene was amplified by PCR from pAHl as template DNA and synthetic primers (SEQ ID NO:37 with SEQ ID NO:38) incorporating an Xbal site at the 5' end and a BamHI site at \ht 3' end. The product was subcloned into pCR-Script (Stratagene) at the Srfl site to generate the plasmids pAH4 and pAH5 containing dhaT. The plasmid pAH4 contains the dhaT gene in \       the right orientation for expression from the lac promoter in pCR-Script and pAH5<br>
contains dhaT gene in the opposite orientation. The Xbal-BamHI fragment from<br>
30		      pAH4 containing the dhaT gene was inserted into pTacIQ to generate       plasmid pAH8. The Hindll-BamHI fragment from pAH8 containing the RBS and dhaT gene was inserted into pBluescriptIIKS+ to create pAHl 1. <br>
Construction of an expression cassette for dhaT and dhaBn.2.3):<br>
56<br><br>
—An expression cassette for dhaTand dhaB(l,2,3) was assembled from the individual dhaB(l,2,3) and dhaT subclones described previously using standard molecular biology methods. A Spel-SacI fragment containing the dhaB(l,2,3) genes from pDT3 was inserted into pAHl 1 at the Spel-SacI sites to create pAH24. A Sall-Xbal linker (SEQ ID NO:39 and SEQ ID NO:40) was inserted into pAH5 that was digested with the restriction enzymes Sall-Xbal to create pDT16. The linker destroys the Xbal site. The 1 kb Sall-Mlul fragment from pDT16 was then inserted into pAH24 replacing the existing Sall-Mlul fragment to create pDT18. pDT21 was constructed by inserting the Sall-NotI fragment from pDT18 and the Notl-Xbal fragment from pM7 into pCL1290 (SEQ ID NO:41). The glucose isomerase promoter sequence from Streptomyces (SEQ ID NO:42) was cloned by PCR and inserted into EcoRI-HinDIII sites of pLitmus28 to construct pDT5. pCL1925 was constructed by inserting EcoRI-PvuII fragment of pDT5 into the EcoRI-PvuII site of pCL1920. pDT24 was constructed by cloning the HinDIII-MluII fragment of pDT21 and the Mlul-Xbal fragment of pDT21 into the HinDIII-Xbal sites of PCL1925.--<br>
Construction of an expression cassette for dhaT and dhaB(l,2.3,X):<br>
pDT21 was constructed by inserting the Sall-NotI fragment from pDT18 151    and the Notl-Xbal fragment from pM7 into pCL1920 (SEQ ID NO:41), The<br>
glucose isomerase promoter sequence from Streptomyces (SEQ ID NO:42) was<br>
cloned by PCR and inserted into EcoRI-HinDIII sites of pLitmus28 to construct<br>
pDT5. pCL1925 was constructed by inserting EcoRI-PvuII fragment of pDT5<br>
into the EcoRI-PvuI site of pCL1920. pDT24 was constructed by cloning the <br>
20     HinDIII-MluII fragment of pDT21 and the Mlul-Xbal fragment of pDT21 into the<br>
HinDIII-Xbal sites of pCL1925.<br>
Construction of an expression cassette for dhaR, orfY, dhaT, orfX, orfW and<br>
dhaBa.2.3.X):<br>
pDT29 was constructed by inserting the SacI-EcoRI fragment of<br>
pHK28-26 into SacI-EcoRI sites of pCL1925.<br>
Construction of an expression cassette for dhaR, orfY, orfX, orffV and<br>
dhaB(L2S,X):<br>
A derivative of plasmid pDT29 was constructed in which all except the<br>
first 5 and the last 5 codons (plus stop codon) of the gene dhaTwere deleted by a 301    technique known as PCR-mediated overlap extension. Using pDT29 as template,<br>
2 primary PCR products were generated using the following primers: •<br>
SEQ ID NO:43 = 5'GAC GCA ACA GTA TTC CGT CGC3';<br>
SEQ ID NO:44 = 5'ATG AGC TAT CGT ATG TTC CGC CAG GCA TTC TGA<br>
GTGTTAACG3';<br>
SEQ ID NO:45 = 5'GCC TGG CGG AAC ATA CGA TAG CTC ATA ATA<br>
TAC3';<br>
SEQ ID NO:46 = 5'CGG GGC GCT GGG CCA GTA CTG3'.<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
SEQ ID NO:45 was paired with SEQ ID NO:46 to generate a product of 931 bps and encompassing nucleic acid including 51 dhaBl (to unique Seal site), all of orfY, and the first five codons of dhaT. SEQ ID NO:43 was paired with SEQ ID NO:44 to generate a product of 1348 bps and encompassing nucleic acid <br>
5	including the last five codons (plus stop codon) of dhaT, all of orfiC, all of orffV, and 5' dhaR (to unique Sapl site). The 15 bases at the 5' end of SEQ ID NO:44 constitute a tail that is the inverse complement of a 15 base portion of SEQ ID NO:45. Similarly, the 11 bases at the 5' end of SEQ ID NO:45 constitute a tail that is the inverse complement of an 11 base portion of SEQ ID NO:44. Thus, the <br>
10	2 primary PCR products were joined together after annealing (via 26 bp tail overlap) and extending by PCR, to generate a third nucleic acid product of 2253 bps. This third PCR product was digested with Sapl and Seal and ligated into pDT29 which was also digested with Sapl and Seal, to generate the plasmid pKP32, which is identical to pDT29, except for the large, in-frame deletion within <br>
15     dhaT.<br>
EXAMPLE 7<br>
CONVERSION OF GLUCOSE TO 1,3-PROPANEDIOL USING<br>
E. COLI STRAIN KLP23/pAH48/pDT29 AND<br>
THE IMPROVED PROCESS USING KLP23/DAH48/DKP32<br>
20      Pre-Culture:<br>
KLP23/pAH48/pDT29 and KLP23/pAH48/pKP32 were pre-cultured for seeding a fermenter in 2YT medium (10 g/L yeast extract, 16 g/L tryptone, and 10 g/L NaCl) containing 200 mg/L carbenicillin (or ampicillin) and 50 mg/L spectinomycin. KLP23/pAH48/pKP32 is identical to KLP23/pAH48/pDT29 <br>
25     	 except that dhaT is deleted.<br>
Cultures were started from frozen stocks (10% DMSO as cryoprotectant) in 500 mL of medium in a 2-L Erlenmeyer flask, grown at 35 °C in a shaker at 250 rpm until an OD550 of approximately 1.0 AU was reached and used to seed the fermenter. <br>
30      	Fermenter medium:<br>
The following components were sterilized together in the fermenter vessel 45 g KH2P04,12 g citric acid, 12 g MgS04-7H20, 30 g yeast extract, 2.0 g ferric ammonium citrate, 5 mL Mazu DF204 as antifoam, 1.2 g CaCl2-2H20, and 7.3 mL sulfuric acid. The pH was raised to 6.8 with 20-28% NH4OH and the <br>
35      	following components were added: 1.2 g carbenicillin or ampicillin, 0.30 g spectinomycin, 60 mL of a solution of trace elements and glucose (from a 60-67 weight % feed). After inoculation, the volume was 6.0 L and the glucose<br>
57<br><br>
WO 01/12833<br><br>
PCTYUS00/22874<br><br>
concentration was 10 g/L. The solution of trace elements contained (g/L): citric acid. H20 (4.0), MnS04-H20 (3.0), NaCl (1.0), FeS04-7H20 (0.10), CoCl2-6H20 (0.10), ZnS04-7H20 (0.10), Cu_S04-5H20 (0.010), H3B03 (0.010), and<br>
Na2Mo04-2H20 (0.010).<br>
 5     Fermentation growth:<br>
A 15 L stirred tank fermenter was prepared with the medium described above. The temperature was controlled at 35 °C and aqueous ammonia (20-28 weight %) was used to control pH at 6.8. Initial values for air flow rate (set to minimum values of between 6 and 12 standard liters per min) and agitator <br>
10     	speed (set to minimum values of between 350 and 690 rpm) were set so that dissolved oxygen (DO) control was initiated when OUR values reached<br>
approximately 140 mmol/L/h. Back pressure was controlled at 0.5 bar. DO control was set at 10%. Except for minor excursions, glucose was maintained at between 0 g/L and 10 g/L with a 60% or 67% (wt) feed. Vitamin B12 or <br>
15	coenzyme B 12 was added as noted below. <br>
16	Fermentation with KLP23/PAH48/DDT29:<br>
A representative fermentation summary of the conversion of glucose to 1,3-propanediol (1,3-PD) using E. coli strain KLP23/pAH48/pDT29 is given in Table 4. Vitamin B12 (0.075 g/L, 500 mL) was fed, starting 3 h after inoculation,<br>
20      at a rate of 16 mL/h. The yield of 1,3-propanediol was 24 wt % (g<br>
1,3-propanediol/g glucose consumed) and a titer of 68 g/L 1,3-propanediol was obtained.<br>
TABLE 4<br>
Representative fermentation summary of the conversion of glucose to 1,3-propanediol (1,3-PD) using E. coli strain KLP23/pAH48/pDT29<br>
Time (h)    OD550 (AU)     DO (%)     Glucose (g/L)   Glycerol (g/L)   1,3-PD (g/I<br><br>
0	0	150	12.9	0.0	0<br>
6	17	80	8.3	3.i	i<br>
12	42	53	2.8	12.5	9<br>
18	98	9	5.7	12.6	32<br>
24	136	11	32.8	12.0	51<br>
30	148	10	12.3	13.3	62<br>
32	152	11	12.5	14.3	65<br>
38	159	11	1.5	17.2	68<br>
58<br><br>
WO 01/12833	PCT/US00/22874<br>
Similar results were obtained with an identical vitamin B12 feed at twice the concentration or bolus additions of vitamin B12 across the time course of the<br>
	fermentation. The highest titer obtained Was 77 g/L.<br>
Improved fermentation with KLP23/PAH48/PKP32:<br>
5		A representative fermentation summary of the conversion of glucose to<br>
1,3-propanediol (1,3-PD) using E. coli strain KLP23/pAH48/pKP32 is given in Table 5. Vitamin B12 (0.150 g/L, 500 mL) was fed, starting 3 h after inoculation, at a rate of 16 mL/h. After 36 h, approximately 2 L of fermentation broth was purged in order to allow for the continued addition of glucose feed. The yield of <br>
10    	 1,3-propanediol was 26 wt % (g 1,3-propanediol/g glucose consumed) and a titer of 112 g/L 1,3-propanediol was obtained.<br>
TABLE 5<br>
Representative fermentation summary of the improved conversion of glucose to<br>
	1,3-propanediol (1,3-PD) using E. coli strain KLP23/pAH48/pKP32<br>
Time(h)     OD550(AU)    DO (%)   Glucose (g/L)   Glycerol (g/L)   1,3-PD (g/L)<br><br>
0	0	148	12.8	0.0	0<br>
6	22	84	6.9	3.3	0<br>
12	34	90	9.7	10.4	7<br>
18	66	43	9.3	5.9	24<br>
24	161	9	0.2	2.5	46<br>
30	200	10	0.2	6.0	67<br>
36	212	10	1.2	9.7	88<br>
42	202	2	0.1	15.5	98<br>
48	197	12	1.2	23.8	112<br>
Similar results were obtained with an identical vitamin B12 feed at half the <br>
15    	  concentration or bolus additions of vitamin B \ 2 across the time course of the fermentation. The highest titer obtained was 114 g/L.<br>
EXAMPLE 8<br>
ENGINEERING OF TRIOSEPHOSPHATEISOMERASE MUTANT OF<br>
E. COLI KLP23 FOR ENHANCED YIELD OF<br>
20	1.3-PROPANEDIOL FROM GLUCOSE<br>
Construction of plasmid for triosephosphate isomerase gene replacement in E. coli KLP23:<br>
E. coli KLP23 genomic DNA was prepared using the Puregene DNA Isolation Kit (Gentra Systems, Minneapolis, .MN).. A 1.0 kb DNA fragment <br>
25     	containing cdh and the 3' end of triosephosphate isomerase (tpiA) genes was<br>
59<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
amplified by PCR (Mullis and Faloona, Methods Enzymol. 155,335-350 (1987)) from KLP23 genomic DNA using primers SEQ ID NO:47 and SEQ ID NO:48. A<br>
1.0 kb DNA fragment containing the 5' end of tpUjiiQ, and the 5' end of yilR<br>
genes was amplified by PCR from KLP23 genomic DNA using primers SEQ ID <br>
5    	 NO:49 and SEQ ID NO:50. A Seal site was incorporated into primer SEQ ID NO:49. The 5' end of primer SEQ ID NO:49 was the reverse complement of primer SEQ ID NO:48 to enable subsequent overlap extension PCR. The gene splicing by overlap extension technique (Horton et al., BioTechniques 8, 528-535 (1990)) was used to generate a 2.0 kb fragment by PCR using the above two PCR<br>
10     fragments as templates and primers SEQ ID NO:47 and SEQ ID NO:50. This fragment represented a deletion of 73% of the 768 bp tpiA structural gene. Overall, this fragment had 1.0 kb flanking regions on either side of the Seal cloning site (within the partial tpiA) to allow for chromosomal gene replacement by homologous recombination.<br>
15	The above blunt-ended 2.0 kb PCR fragment was cloned into the<br>
pCR-Blunt vector using the Zero Blunt PCR Cloning Kit (Invitrogen, San Diego, CA) to yield the 5.5 kb plasmid pRN 106-2 containing kanamycin and Zeocin resistance genes. The 1.2 kb Hindi fragment from pLoxCatl (unpublished results), containing a chloramphenicol-resistance gene flanked by bacteriophage<br>
20      PI lox? sites (Snaith et al., Gene 166, 173-174 (1995)), was used to interrupt the tpiA fragment in plasmid pRN 106-2 by ligating it to Sorf-digested plasmid pRN 106-2 to yield the 6.8 kb plasmid pRN 107-1. Engineering of triosephosphate isomerase mutant RJ8m by linear DNA transformation:<br>
2 5	Using pRN 107-1 as template and primers SEQ ID NO :47 and SEQ ID<br>
NO:50, the 3.2 kb fragment containing tpiA flanking regions and the /oxP-CmR-ZoxP cassette was PCR amplified and gel-extracted. E. coli KLP23 was electrotransformed with up to 1 ug of this 3.2 kb linear DNA fragment and transformants that were chloramphenicol-resistant (12.5 ug/mL) and kanamycin-<br>
30      sensitive (30 ug/mL) were further screened on M9 minimal media for poor<br>
glucose utilization on 1 mM glucose, for normal gluconate utilization on 1 mM gluconate, and to ensure the glycerol non-utilization phenotype of host KLP23 on .1 mM glycerol. An £coRI digest of genomic DNA from one such mutant, RJ8m, when probed with the intact tpiA gene via Southern analysis (Southern, J. Mol.<br>
35     Biol. 98, 503-517 (1975)) indicated that it was a double-crossover integrant (tpiA gene replacement) since the two expected 6.6 kb and 3.0 kb bands were observed, owing to the presence of an additional jEcoRI site within the chloramphenicol<br>
60<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
resistance gene. As expected, the host KLP23 and wild-type FM5 controls yielded single 8.9 kb and 9.4 kb bands respectively. This tpiA mutant was further<br>
analyzed by genomic PCR using primers SEQ ID NO:51 and SEQ ID NO:52,<br>
which yielded the expected 4.6 kb PCR fragment while for the same primer pair<br>
5	the host KLP23 and wild-type FM5 strains both yielded the expected 3.9 kb PCR fragment. When cell-free extracts from tpiA mutant RJ8m and host KLP23 were tested for tpiA activity using glyceraldehyde 3-phosphate as substrate, no activity was observed with RJ8m. The tpiA mutant RJ8m was electrotransformed with plasmid pAH48 to allow glycerol production from glucose and also with both <br>
10  	   plasmids pAH48 and pDT29 or pKP32 to allow 1,3-propanediol production from glucose. The chloramphenicol resistance marker was eliminated from RJ8m to give RJ8.<br>
EXAMPLE 9<br>
CONVERSION OF GLUCOSE TO 1,3-PROPANEDIOL USING E. COLI<br>
15	STRAIN RJ8/pAH48/pDT29 AND THE IMPROVED<br>
PROCESS USING RJ8/PAH48/DKP32 Pre-Culture:<br>
RJ8/pAH48/pDT29 and RJ8/pAH48/pKP32 were pre-cultured for seeding a fermenter as described in Example 7. RJ8/pAH48/pKP32 is identical to <br>
20     	RJ8/pAH48/pDT29 except that dhaT is deleted. Fermenter medium:<br>
Fermenter medium was as described in Example 7. Fermentation growth:<br>
Fermenter growth was as described in Example 7 except that initial values <br>
25	for air flow rate (set to minimum values of between 5 and 6 standard liters per min) and agitator speed (set to minimum values of between 300 and 690 rpm) were set so that dissolved oxygen (DO) control was initiated when OUR values reached between 60 and 100 mmol/L/h. Vitamin B12 or coenzyme B12 was added as noted below. <br>
30      Fermentation with RJ8/pAH48/pDT29:<br>
A representative fermentation summary of the conversion of glucose to 1,3-propanediol (1,3-PD) using E. coli strain RJ8/pAH48/pDT29 is given in Table 6. Vitamin B12 was provided as bolus additions of 2,16 and 16 mg at 2, 8, and 26 h, respectively. The yield of 1,3-propanediol was 35 wt % (g <br>
3 5     	1,3-propanediol/g glucose consumed) and a titer of 50.1 g/L 1,3-propanediol was obtained.<br>
61<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
TABLE 6 <br>
Representative fermentation summary of the conversion of glucose to<br>
1,3-propanediol (1,3-PD) using R coli strain RJ8/pAH48/pDT29'<br>
TimeQi)    OD550 (AU)    DO (%)  "Glucose (g/L)   Glycerol (g/L)   1,3-PD (g/L)<br><br>
0	0	140	10.6	0.1	0.0<br>
6	5	107	11.1	0.5	0.4<br>
10	16	90	8.5	1.7	1.3<br>
14	25	86	1.8	2.4	5.9<br>
19	38	53	3.5	5.9	15.4<br>
25	53	38	0.1	9.2	26.7<br>
31	54	10	4.5	7.4	39.0<br>
37	37	23	17.2	6.0	45.0<br>
43	21	13	9.9	7.7	50.1<br>
Improved fermentation with RJ8/pAH48/pKP32:<br>
A representative fermentation summary of the conversion of glucose to 1,3-propanediol (1,3-PD) using E. coli strain RJ8/pAH48/pKP32 is given in <br>
5     	Table 7. Vitamin B^ was provided as bolus additions of 48 and 16 mg at approximately 26 and 44 hr, respectively. The yield of 1,3-propanediol was 34 wt % (g 1,3-propanediol/g glucose consumed) and a titer of 129 g/L 1,3-propanediol was obtained.<br>
TABLE 7<br>
Representative fermentation summary of the improved conversion of glucose<br>
to 1,3-propanediol (1,3-PD) using E. coli strain RJ8/pAH48/pKP32.<br>
Time(h) OD550 (AU)  DQ(%)   Glucose (g/L)   Glycerol (g/L)  1,3-PD (g/L)<br><br>
0	0	150	12.6	0.1	0<br>
6	12	113	6.0	2.6	0<br>
12	24	99	0.0	10.6	0<br>
18	51	76	2.4	28.9	0<br>
24	78	82	2.4	44.2	5<br>
30	114	70	3.8	26.9	33<br>
36	111	72	0.0	20.0	57<br>
42	139	65	0.1	21.9	69<br>
48	157	36	0.1	- 22.4	79<br>
55	158	25	0.2	21.4	94<br>
64	169	14	0.1	15.8	113<br>
72	169	12	0.1	13.4	119<br>
74	162	14	0.1	14.8	129<br>
10<br>
62<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
EXAMPLE 10<br>
 IDENTIFICATION OF THE E. COLI NON-SPECIFIC CATALYTIC<br>
ACTIVITY (vahD) IN THE IMPROVED 1.3-PROPANEDIOL PROCESS Demonstration of non-specific catalytic activity in 1.3-propanediol-producing <br>
5      	fermentations with the improved catalyst:<br>
A whole cell assay for 1,3-propanediol dehydrogenase activity was used to demonstrate that the non-specific catalytic activity in E. coli is present under fermentative conditions after the addition of vitamin B12 and the production of 3-hydroxypropionaldehyde (3-HPA), but not before. A recombinant E. coli strain<br>
10     containing the glycerol-production and 1,3-propanediol-production plasmids, pAH48 and pKP32, respectively, was grown in 10 L fermenters, essentially as described in Example 7, but in the absence of vitamin B12. A vitamin B12 bolus (48 mg) was added when the tanks reached approximately 100 OD550. Aliquots of cells were taken from the tanks immediately before and 2 h post-vitamin B12<br>
15      addition. The cells were recovered by centrifugation and resuspended to their original volume in PBS buffer containing 150 μg/mL chloramphenicol to inhibit new protein synthesis. An appropriate volume of the chloramphenicol treated cells was added to 250 mL baffled flasks containing a reaction mixture (PBS buffer containing 10 g/L glucose, 10 g/L glycerol, 1 mg/L coenzyme B12, and 150<br>
20      μg/mL chloramphenicol) so that the final volume was 50 mL at an OD550 of<br>
approximately 10. The flasks, protected from light, were shaken at 250 rpm at 35 °C. Aliquots for HPLC analysis were taken over time. Time-dependent production of 3-HPA was observed in flasks containing cells recovered from the fermenter either pre- or post-vitamin B12 addition. In direct contrast, significant<br>
25      levels of 1,3-propanediol were observed only in those flasks containing cells recovered from the fermenter post-vitamin B12 addition. Detection of non-specific catalytic activity in cell-free extracts:<br>
A native gel activity stain assay was used to demonstrate non-specific catalytic activity in cell-free extracts. Cells were recovered, pre- and post-vitamin<br>
30      B12 addition, from representative 10-L fermentations employing recombinant<br>
E. coli strains containing the glycerol-production and 1,3-propanediol-production plasmids, pAH48 and pKP32, respectively; and cell-free extracts were prepared by cell disruption using a French press. The cell-free extracts, a preparation of pure<br>
Klebsiella pneumoniae 1,3-propanediol dehydrogenase (dhaT), and molecular <br>
3 5     	weight standards were applied to and run out on native gradient polyacrylamide gels. The gels were then exposed to either the substrates 1,3-propanediol and NAD+ or ethanol and NAD+. As expected in the gels where 1,3-propanediol was<br>
63<br><br>
WO 01/12833<br><br>
PCT/USOO/22874<br><br>
the substrate, an activity stain for DhaT was observed which migrated on the native gel at approximately 340 Kdal. This activity was observed only in lanes<br>
where pure. Klebsiella pneumoniae 1,3 -propanediol dehydrogenase was applied. In contrast, "where 1,3-propanediol was the substrate and post-vitamin B12 cell-<br>
5      free extracts were applied, a non-specific catalytic activity was observed at<br>
approximately 90 Kdal. When ethanol was used as a substrate, neither the DhaT band nor the non-specific catalytic activity band were visible, but a separate band was found pre- and post-vitamin B12 addition at approximately 120 Kdal. This new band most likely represents an alcohol dehydrogenase with specificity <br>
10     	towards ethanol as substrate as is typically found in all organisms.<br>
This native gel assay, where proteins are separated by molecular weight<br>
prior to the enzymatic assay step, offered greater sensitivity and accuracy in<br>
measuring the reduction of 1,3-propanediol in those constructs with low activity and where the activity is likely to be distinct from the alcohol dehydrogenases<br>
15      with specificity towards ethanol as substrate that have been well characterized for E. coli and found in all organisms. The dehydrogenase assay works on the principle that dehydrogenase catalyzes the transfer of electrons from 1,3-propanediol (or other alcohols) to NAD+. PMS (phenazine methosulfate) then couples electron transfer between NADH and a tetrazolium bromide dye (MTT, 3-<br>
20      [4,5-dimethylthiazol-2-y}]-2,5-diphenyltetrazoIium bromide) which forms a<br>
precipitate in the gel. After a few hours to overnight soaking in the substrates, the gels are washed to remove reagents and soluble dye. At bands on the gel where there is an active dehydrogenase, an insoluble blue dye forms. Various aspects of the assay have been described by Johnson and Lin (J. Bacteriol 169:2050 (1987)).<br>
25      Purification and identification of the non-specific catalytic activity in E. coli: A large scale, partial purification of non-specific catalytic activity was performed on cells harvested from the end of a typical 1,3-propanediol production run as described in the improved process using KLP23/pAH48/pKP32 of Example 7. The cell pellet (16 g) was washed and resuspended three times in 20 mL of 50<br>
30     mM Hepes buffer, pH 7.5. The cells in the suspension were lysed by sonication. The cell-free extract was obtained by centrifugation (15 min, 20,000 x g, 10 °C) and the supernatant was further clarified by addition of 250 mg of protamine sulfate with stirring on ice. The supernatant obtained by centrifugation (20 min,<br>
20,000 x g, 10 °C) was fractionated by passage through a Superdex® 200 <br>
35     	preparative grade column (6 x 60 cm) equilibrated with Hepes buffer. Fractions of 10 mL each were collected and an aliquot of each was concentrated twentyfive-fold using 10,000 MW cutoff Centricon® membranes prior to assay by the native<br>
64<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
gel activity stain. The non-specific catalytic activity was identified in fractions 107-112, and the peak activity in fractions 108-109. A larger aliquot (7 mL each)<br>
effractions 108 and 109 were concentrated fifty-fold and loaded on all lanes of a 12-lane native gel. The gel was cut in half and one half was stained for <br>
5     	 dehydrogenase activity where a dark blue band appeared that represented the nonspecific catalytic activity. The unstained gel was aligned top to bottom with the stained gel and a band was cut on the unstained gel that corresponded to the band of non-specific catalytic activity. The gel strip was pulverized and soluble protein was extracted by immersing the pulverized particles in 0.5 mL of 2D-loading<br>
10 	    buffer, heating to 95 °C for 5 min, and centrifugation to remove the gel particles. The supernatant was loaded onto an isoelectricfocusing (IEF) strip for 2-dimension polyacrylamide gel electrophoresis (2D-PAGE) using conditions described for 2D-PAGE of E. coli extracts in the Swiss 2D database (http://www.expasy.ch/ch2d/; Tonella et al. Electrophoresis 19:1960-1971<br>
15     (1998)). The gel was transferred to a PVDF membrane by electroblotting. The membrane was stained for proteins using the Colloidal blue gel stain. The stained blot used to obtain the identity of the non-specific catalytic activity is shown in Figure 6. Spots were identified using standard techniques for amino terminus peptide sequencing. Only a single spot (Spot A) encoded for an oxidoreductase<br>
20      activity. Nineteen cycles of Spot A (Figure 6) yielded a 100% identity match by the FASTA search tool with the amino-terminus of yqhD, an E. coli open reading frame with putative oxidoreductase activity. Complete amino acid sequence for the protein encoded by yqhD is given in SEQ ID NO:57; the corresponding DNA sequence is given in SED ID NO:58. The yqhD gene has 40% identity to the gene<br>
25     adhB in Clostridium, a probable NADH-dependent butanol dehydrogenase 2. Gene Disruption of yqhD in E. coli KLP23:<br>
Biochemical assays and amino-terminal amino acid sequencing suggested that non-specific catalytic activity may be encoded by the E. coli yqhD gene. This<br>
gene of unknown function encodes a hypothetical oxidoreductase and contains <br>
30      	two alcohol dehydrogenase signatures also found in the Citrobacter freundii and Klebsiella pneumoniae 1,3-propanediol dehydrogenase encoded by the dhaT gene.<br>
To disrupt this gene, yqhD and 830 bp of 5'-flanking DNA sequence and 906 bp of 3'-flanking DNA sequence were amplified from E. coli KLP23 (Example 4) genomic DNA in a PCR using Taq polymerase and the following<br>
35     primers:<br>
(SEQ ID NO:59) S'-GCGGTACCGTTGCTCGACGCTCAGGTTTTCGG-S' (SEQ IDNOrbO) 5’-GCGAGCTCGACGCTTGCCCTGATGATCGAGTTTTGC-3’<br>
65<br><br>
WO 01/12833<br><br>
PCTYUS00/22874<br><br>
The reaction was run at 94 °C for 1 min, 50 °C for 1 min, and 72 °C for 3 min for 35 cycles followed by a final extension at 72 °C for 5 min. The resulting 3.7 Kb<br>
DNA fragment was purified, digested with Sad and Kpnl and ligated to similarly<br>
digested pBluescriptll KS(+) (Strategene) for 16 h at 16 °C. The ligated DNA <br>
5    	 was used to transform E. coli DH5α (Gibco/BRL) and the expected plasmid, pJSP29, was isolated from a transformant demonstrating white colony color on LB agar (Difco) containing X-gal (40 μg/mL) and ampicillin (100 μg/mL). Plasmid pJSP29 was digested with Aflll and Ndel to liberate a 409 bp DNA fragment comprising 363 bp of the yqhD gene and 46 bp of 3'-flanking DNA<br>
10     sequence. The remaining 5,350 bp DNA fragment was purified and ligated to the 1,374 bp AflHINdel DNA fragment containing the kanamycin resistance gene from pLoxKan2 (Genencor International, Palo Alto, CA) for 16 h at 16 °C. The ligated DNA was used to transform E. coli DH5a and the expected plasmid, pJSP32-Blue, was isolated from a transformant selected on LB agar media<br>
15      containing kanamycin (50 μg/mL). Plasmid pJSP32-Blue was digested with Kpnl and Sad and the 3,865 bp yqhD disruption cassette was purified and ligated to similarly digested pGP704 (Miller and Mekalanos, J. Bacteriol. 170:2575-2583 (1988)) for 16 h at 16 °C. The ligated DNA was usH tn transform E. coli SY327 (Miller and Mekalanos, J. Bacteriol. 170:2575-258:        18)) and the expected<br>
20     plasmid, pJSP32, was isolated from a transformant:        ed on LB agar media containing kanamycin (50 μg/mL). Plasmid pJSP3;        transformed into E. coli KLP23 and transformants were selected on LB agar containing kanamycin (50 μg/mL). Of the 200 kanamycin-resistant transformants screened, two demonstrated the ampicillin-sensitive phenotype expected for a double-crossover<br>
25     recombination event resulting in replacement of the yqhD gene with the yqhD disruption cassette.<br>
The disruption of the yqhD gene was confirmed by PCR using genomic DNA isolated from these two transformants as the template and the following set! of primer pairs:<br>
30     Set#l:<br>
(SEQ IDNO:61) S'-GCGAGCTCGACGCTTGCCCTGATCGAGTTTTGC-S'<br>
(SEQ ID NO:62) S'-CAGCTGGCAATTCCGGTTCGO'<br>
Set #2:<br>
(SEQ ID NO:63) S'-CCCAGCTGGCAATTCCGGTTCGCTTGCTGTO'<br>
35     (SEQ ID NO:64) 5'-GGCGACCCGACGCTCCAGACGGAAGCTGGT-3' Set#3: (SEQ ID NO:65) 5'-CCGCAAGATrCACGGATG€ATCGTGAAGGG-3'<br>
66<br><br>
WO 01/12833<br><br>
PCT7US00/22874<br><br>
(SEQ ID NO:66) 5'-CGCCTTCnGACGAGTTCTGAGCGGGA-3' Set #4:<br>
(SEQ ID NO:67) S'-GGAATTCATGAACAACTTTAATCTGCACACO' (SEQ ID NO:68) S'-GmGAGGCGTAAAAAGCTTAGCGGGCGGC-S' <br>
5	The reactions were run using either Expand High Fidelity Polymerase (Boehringer Manheim) or Platinum PCR Supermix containing Taq polymerase (Gibco/BRL) at 94 °C for 1 min, 50 °C for 1 min, and 72 °C for 2 min for 35 cycles followed by a final extension at 72 °C for 5 min. The resulting PCR products were analyzed by gel electrophoresis in 1.0 % (w/v) agarose. The results summarized in Table 8 <br>
10     confirmed disruption of the yqhD gene in both transformants.<br>
TABLE 8<br><br>
Primer Set	Expected Size (bp)	Observed Size (bp)<br>
	yqhD disruption	yqhD wild-type	<br>
1	1,200	no product	-1,200<br>
2	1,266	no product	-1,266<br>
3	2,594	no product	-2,594<br>
4	no product	1,189	-900<br>
The yqhD disruption deletes the 3' end of yqhD, including 46 bp of 3'-flanking<br>
intergenic DNA sequence. The deletion removes 363 bp of 2'yqhD coding 15      sequence corresponding to 121 amino acids. A stop codon is present 15 bp<br>
downstream of the remaining yqhD coding sequence in the kanamycin resistance<br>
cassette.<br>
Plasmids pAH48 and pKP32 were co-transformed into E. coli KLP23<br>
(yqhD-) and transformants containing both plasmids were selected on LB agar <br>
20     containing ampicillin (100 μg/mL) and spectinomycin (50 μg/mL). A<br>
representative transformant was tested for its ability to covert glucose to 1,3-propanediol in 10 L fermentations either in the presence or absence of vitamin<br>
B12.<br>
Demonstration that yqhD is required for significant L3-propanediol production in <br>
25     E. coli strain KLP23/DAH48/PKP32:<br>
Fermentations for the production of 1,3-propanediol were performed, essentially as described in Example 7, with the E. coli strain KLP23 (yqhD-)/pAH48/pKP32 in order to test for the effect of the yqhD disruption on<br>
1,3-propanediol production.<br>
30	A representative 10-L fermentation using the knockout of the non-specific<br>
catalytic activity, E. coli strain KLP23 (yqhD-)/pAH48/pKP32, is shown in Table<br>
67<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
9. The organism steadily accumulated cell mass and glycerol until the addition of vitamin B12 when the OD550 exceeded 30 A (10.4 h). Vitamin B12 was added as<br>
a bolus addition of 8 mg at 10.4 h and thereafter vitamin B12 was continuously fed<br>
at a rate of 1.32 mg/h. In the 4 h that followed B12 addition, glucose consumption <br>
5     	slowed, the oxygen utilization rate dropped and there was no further increase in optical density. Fermentation of glucose ceased and the glucose concentration in the tank accumulated. The highest titer of 1,3-propanediol obtained was 0.41 g/L. The organism was checked for its viability by plating a dilution series of the cells on agar plates containing ampicillin and spectinomycin. The plates were<br>
10     incubated for 24 h in a 30 °C incubator. There were no viable colonies on the plate from the fermentation of E. coli KLP23 0?/iZ)-J/pAH48/pKP32, Table 11. By contrast, the cell suspension from a control tank to which no vitamin B12 was added continued to accrue cell mass and glycerol until the 10-L tank was full due to the complete addition of the glucose feed solution (Table 10). An agar<br>
15      plate viability determination by dilution series of the cell suspension at the end of<br>
this fermentation showed a viable cell count that was consistent with the total cell number estimated by the optical density value (Table 11).<br>
TABLE 9<br>
Representative fermentation summary of the failed conversion of glucose to 1,3-<br>
20	propanediol (1,3-PD) using E. coli strain KLP23 (y^AZ)-;/pAH48/pKP32.<br>
time (h)	OD550 (AU)	DO (%)	glucose (g/L)	glycerol (g/L)	1,3-PD (g/L)<br>
0	0.4	150	11.3	0.05	0<br>
2.3	3.0	134	10.7	0.13	0<br>
4.3	10.8	85.0	8.2	1.41	0<br>
8.3	23.1	81.8	0.9	10.0	0<br>
16.3	37.2	149	13.1	21.4	0.41<br>
18.3	47.6	149	18.9	21.6	0.39<br>
20.3	39.6	149	24.4	22.3	0.42<br>
23.8	33.6	149	25.4	22.0	0.41<br>
68<br><br>
WO 01/12833<br><br>
PCT/US00/22874<br><br>
TABLE 10 <br>
Representative fermentation summary of the conversion of glucose to glycerol<br><br>
	using E. coli strin	y		<br>
Time (h)	OD55o(AU)	DO (%)	glucose (g/L)	glycerol (g/L)<br>
0	0.2	148	9.5	0.06<br>
2.2	2.8	128	8.9	0.13<br>
4.2	10.4	58.5	7.0	1-4<br>
8.2	21.6	57,6	2.7	11.2<br>
16.2	76.8	10.7	0	40.5<br>
20.2	117	10.2	0	52.9<br>
23.7	154	8.5	0	63.9<br>
36.2	239	10.1	0.1	122<br>
TABLE 11 <br>
Representative summary of viability plate counts from endpoints of fermentations of glucose using E. coli strain KLP23(yqhD-,)/pAH48/pKP32 in the absence and<br>
presence of vitamin B12-<br><br>
vitamin B12	time (h) at endpoint	OD550 (AU)	viable counts (cfu/mL)<br>
no	36.2	239	2.1E11<br>
yes	23.8	33.6	0<br>
yes	23.8	41.2	0<br>
69<br><br>
We claim<br>
1.       A microorganism transformed with a chimeric gene comprising an isolated       nucleic acid fragment encoding a non-specific catalytic<br>
activity for the conversion of 3-hydroxypropionaldehyde to 1, 3-propanediol operably linked to suitable regulatory sequences, wherein the isolated nucleic acid fragment is selected from the group consisting of:<br>
(a)	an isolated nucleic acid fragment encoding the amino acid sequence of SEQ ID NO:57;<br>
(b)	an isolated nucleic acid fragment encoding a polypeptide of at least 387 amino acids having at least 80% identity with the amino acid sequence of SEQ ID NO:57;<br>
(c)	an isolated nucleic acid fragment that hybridizes with (a) or (b) under hybridization conditions of 0.1X SSC, 0.1% SDS, 65 °C and washed with 2X SSC, 0.1% SDS followed by 0.1X SSC, 0.1% SDS; and<br>
(d)	an isolated nucleic acid fragment that is complementary to (a) (b), or (c):<br>
wherein the microorganism is selected from the group consisting of Citrobacter, Enterobacter, Clostridium, Klebsiella, Aerobacter, Lactobacillus, Aspergillus, Saccharomnyces, Schizosaccharomyces, Zygosaccharomyces, Pichia, Kluyveromyces, Candida, Debaryomyces, Mucor, Methylobacter, Salmonella, Bacillus, Streptomyces and Pseudomonas.<br>
2. A recombinant- microorganism, useful for the production of 1,3-propanediol, comprising:<br>
(a) at least one gene encoding a polypeptide having a dehydratase<br>
activity;<br>
75<br><br>
(b).at least one gene encoding a dehydratase reactivation factor;<br>
and<br>
(c) at least one exogenous gene encoding an non-specific catalytic activity to convert 3 -hydroxypropionaldehyde to 1 ,3-propanediol; wherein no functional dhaT gene encoding a 1,3-propanediol oxidoreductase activity is present in the recombinant microorganism and the microorganism is selected from, the group consisting of Citrobacter, Enterobacter, Clostridium. kliebsiella, Aerobacter, Lacto bacillus, Aspergillus, Saccharomyces, Schizosaccharomyces, Zygosaccharomyces, Pichia, Kluyveromyces, Candida, Debaryomyces, Mucor, Methylobacter, Salmonella, Bacillus, Streptomyces and Pseudomonas.<br>
3.	The recombinant microorganism as claimed in Claim 2 wherein it<br>
comprises:<br>
. (a) at least one gene encoding a polypeptide having glycerol-3-phosphate dehydrogenase activity; and<br>
(b) at least one gene encoding a polypeptide having glycerol-3-phsophatase activity.<br>
4<br>
4.	The recombinant microorganism as claimed in Claim 2 or 3 wherein the dehydratase reactivation factor is encoded by orfX and orfZ, isolated from a dha regulon.<br>
5.	The recombinant microorganism as claimed in Claim 4 wherein orjX and orfZ are independently isolated from Klebsiella sp., Citrobacter sp.. or Clostridium sp.<br>
6. The recombinant microorganism as claimed in Claim 3, wherein it<br>
comprises a set of endogenous genes, each having a mutation inactivating the gene, the set consisting of:<br>
(a)      a first gene encoding a polypeptide having glycerol kinase<br>
76<br><br>
activity;<br>
(b)    a second gene encoding a polypeptide having glycerol<br>
dehydrogenase	activity;<br>
and (e)      a third gene encoding a polypeptide having triosephosphate isomerase activity.<br>
7.	' The recombinant microorganism as claimed in Claims 3 or 6 wherein<br>
the recombinant microorganism converts a carbon source selected from the<br>
group consisting of monosaccharides,  oligosaccharides, polysaccharides,<br>
and single-carbon substrates to 1,3-propanediol.<br>
*<br>
8.	The recombinant microorganism as claimed in Claim 2 wherein the<br>
recombinant microorganism converts a carbon source selected from the<br>
group consisting of glycerol and dihydroxyacet;one to 1,3-propanediol.<br>
9.	The recombinant microorganism 4 as claimed in Claims 3 or 6<br>
wherein the gene encoding a polypeptide having glyercol-3-phosphate<br>
dehydrogenase activity is selected from the group consisting of GPD1,<br>
GPD2, GPD3, DAR1, gpsA, GUT2, glpD, and glpA BC. .<br>
10 The recombinant microorganism as claimed in Claims 3 or 6 wherein the gene encoding a polypeptide having glycerol-3-phosphatase activity is selected from the group consisting of GPP 1 and GPP2.<br>
11. The recombinant microorganism claimed in Claims 2, 3 or 6 wherein the gene encoding a polypeptide having a dehydratase activity is selected from the group consisting of a glycerol dehydratase and a diol dehydratase.<br>
12 The recombinant microorganism as claimed in Claims 2, 3 and 6 wherein the gene encoding a polypeptide having a dehydratase activity is isolated from Klebsiella sp, Citrobacter sp.,. or Clostridiun sp.<br>
77<br><br>
13     A recombinant E. coli pomprising:<br>
(a)	a set of exogenous genes consisting of:<br>
(i) at least one gene encoding a polypeptide having a glycerol or diol dehydratase activity;<br>
(ii) at least one gene encoding a polypeptide having glycerol-3-phosphate dehydrogenase activity;<br>
(iii) at least one gene encoding a polypeptide having glycerols-phosphatase activity; and<br>
(iv)   at least  one  gene  encoding a dehydratase  reactivation<br>
factor; and<br>
(b)	at least one endogenous gene encoding a non-specific catalytic<br>
activity to convert 3-hydroxypropionaldehyde to 1 ,3-propanediol:<br>
wherein    no    functional    dkaT   gene    encoding    a    1    ,3-propanediol oxidoreductase activity is present in the recombinant E. coli.<br>
14. A recombinant E. coli comprising:<br>
(a) a set of exogenous genes consisting of<br>
(i)       at least one copy of dhaR. nucleotides 2209-4134 of SEQ<br>
ID NO:l; (ii)       at least one copy of orfK complementary to nucleotides<br>
6202-6630 of SEQ ID NO:l; (iii)       at least one copy of orfK complementary to nucleotides 4643-4996 of<br>
SEQ ID NO: 1; (iv)      at least one copy of orfW. complementary to nucleotides<br>
4112-4642 of SEQ ID NO:l; (v)       at   least   one   copy   of   dhaBl,   dhaB2,   and   dhaB3,<br>
nucleotides 7044-8711  of SEQ ID NO:l,  nucleotides<br>
8724-9308 of SEQ ID NO:l, and nucleotides 931 1-9736<br>
of SEQ ID NO:l, respectively, and (vi)     at least on copy of orfZ, nucleotides 9749-11572 of SE<br>
78<br><br>
IDN0:1 and (b) at least one endogenous gene encoding a non-specific catalytic activity to convert 3-hydroxypropionaldehyde to 1 ,3 -propanediol, wherein    no    functional    dhaT    gene    encoding    a     1     ,3-propanediol oxidoreductase activity is present in the recombinant E. coli.<br>
15.	The recombinant E. coli as claimed in claim 14 wherein it comprises a<br>
set of endogenous genes, each gene having a mutation inactivating the gene,<br>
the set of endogenous genes consisting of:<br>
(a)	a gene encoding a polypeptide having glycerol kinase activity:;<br>
(b)	a gene encoding a polypeptide having glycerol dehydrogenase activity; and<br>
(c)	a gene encoding a polypeptide having triosephosphate isomerase activity.<br>
16.	A process for the bioproduction of 1,3-propanediol comprising:<br>
(a)	contacting under suitable conditions the recombinant E. coli as claimed in Claims 13, 14, or 15 with at least one carbon source selected from the group consisting of monosaccharides, oligosaccharides, polysaccharides, and single-carbon substrates whereby 1,3-propanediol is produced; and<br>
(b)	optionally recovering the 1,3-propanediol produced in (a).<br>
17.	A process for the bioproduction of 1,3-propanediol comprising:<br>
(a) contacting under suitable conditions 1) a recombinant F. coli comprising (a)    a set of exogenous genes consisting of<br>
(i)       at least  one  exogenous  gene  encoding a polypeptide    having    a    glycerol    or    diol<br>
dehydratase activity; and<br>
(ii)        at   least   one   exogenous   gene   encoding   a dehydratase reactivation factor; and<br>
79<br><br>
(b)  at least one endogenous gene encoding a nonspecific     catalytic     activity     to     convert     3-<br>
hydroxypropionaldehyde to I ,3-propanediol,<br>
wherein no functional c/haT gene encoding a 1 ,3-propanediol<br>
oxidoreductase   activity is  present  in  the  recombinant E.coli or (2) the recombinant E. coli of Claim 14 with  at  least  one   carbon   source   selected  from  the  group consisting of glycerol and dihydroxyactetone, and (b) optionally recovering the 1 ,3-propanediol produced in (a).<br>
A process for the production of 1 ,3-propanediol comprising:<br>
(a)	contacting a recombinant E. coli with a first source of carbon<br>
and with a second source of carbon, the recombinant E. coli<br>
comprising:<br>
(i)        at  least  one  exogenous  gene  encoding  a  polypeptide<br>
having a glycerol or diol dehydratase activity; (ii)  at least  one  exogenous  gene  encoding  a dehydratase<br>
reactivation factor;<br>
(iii)  at least one endogenous gene encoding a non-specific<br>
catalytic activity<br>
sufficient to convert 3-hydroxypropionaldehyde to I ,3 -propanediol, wherein no functional dhaT gene encoding a 1 ,3-propanediol oxidoreductase activity is present in the recombinant E .coli and wherein the first carbon source is selected from the group consisting of glycerol and dehydroxyacetone, and the second carbon source is selected from the group consisting of monosaccharides, ol igosaceharides, polysaccharides, and single-carbon substrates;<br>
and<br>
(b)	optionally recovering the 1 ,3-propanediol produced in (a).<br>
80<br><br>
19.    .The process as claimed in Claim 18 wherein the recombinant F. co/i<br>
comprises<br>
a set of exogenous genes consisting of<br>
(a)	at least one gene encoding a polypeptide having glycerol-3-<br>
phosphate dehydrogenase activity; and<br>
(b)	at least one gene encoding a polypeptide having glycerol-3-<br>
phosphatase activity and<br>
20. The process of as claimed in Claim 19 wherein the recombinant E.coli further comprises<br>
(i)       a gene encoding a polypeptide having glycerol kinase activity<br>
(ii)       a    gene     encoding    a    polypeptide    having    glycerol<br>
dehydrogenase activity; and (ii)      a gene encoding a polypeptide having triosephosphate<br>
isomerase activity.<br>
21.	A  microorganism transformed with a chimeric gene substantially as herein described with reference to the foregoing examples.<br>
22.	A recombinant microorganism useful for the production of 1,  3-propanediol substantially as herein described with reference to the<br>
foregoing examples. 23      A process for the bioproduction of 1, 3-propanediol substantially as herein described with reference to the foregoing examples.<br><br><br><br><br>
N/A</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LUFCU1RSQUNUKDI5LTEwLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-ABSTRACT(29-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LUFCU1RSQUNUKEFNRU5ERUQpLSgyOS0xMC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-ABSTRACT(AMENDED)-(29-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LUFCU1RSQUNUKEdSQU5URUQpLSgzMC0zLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-ABSTRACT(GRANTED)-(30-3-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LUNBTkNFTExFRCBQQUdFUygyOS0xMC0yMDA4KS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-CANCELLED PAGES(29-10-2008)-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LUNBTkNFTExFRCBQQUdFUygyOS0xMC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-CANCELLED PAGES(29-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LUNMQUlNUygyMS0xMS0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-CLAIMS(21-11-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LUNMQUlNUygyOS0xMC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-CLAIMS(29-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LUNMQUlNUyhHUkFOVEVEKS0oMzAtMy0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-CLAIMS(GRANTED)-(30-3-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1tdW1ucC0yMDA1LWNsYWltcy5kb2M=" target="_blank" style="word-wrap:break-word;">1280-mumnp-2005-claims.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1tdW1ucC0yMDA1LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1280-mumnp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LUNPUlJFU1BPTkRFTkNFKDE3LTUtMjAwNikucGRm" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-CORRESPONDENCE(17-5-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LUNPUlJFU1BPTkRFTkNFKDItNC0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-CORRESPONDENCE(2-4-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LUNPUlJFU1BPTkRFTkNFKDItNi0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-CORRESPONDENCE(2-6-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LUNPUlJFU1BPTkRFTkNFKDI5LTEwLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-CORRESPONDENCE(29-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LUNPUlJFU1BPTkRFTkNFKElQTyktKDYtNC0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-CORRESPONDENCE(IPO)-(6-4-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1tdW1ucC0yMDA1LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1280-mumnp-2005-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1tdW1ucC0yMDA1LWNvcnJlc3BvbmRlbmNlLXJlY2VpdmVkLXZlci0xODExMDUucGRm" target="_blank" style="word-wrap:break-word;">1280-mumnp-2005-correspondence-received-ver-181105.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1tdW1ucC0yMDA1LWNvcnJlc3BvbmRlbmNlLXJlY2VpdmVkLnBkZg==" target="_blank" style="word-wrap:break-word;">1280-mumnp-2005-correspondence-received.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1tdW1ucC0yMDA1LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">1280-mumnp-2005-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LURFU0NSSVBUSU9OKENPTVBMRVRFKS0oMjEtMTEtMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-DESCRIPTION(COMPLETE)-(21-11-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LURFU0NSSVBUSU9OKENPTVBMRVRFKS0oMjktMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-DESCRIPTION(COMPLETE)-(29-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LURFU0NSSVBUSU9OKEdSQU5URUQpLSgzMC0zLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-DESCRIPTION(GRANTED)-(30-3-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LURSQVdJTkcoMjEtMTEtMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-DRAWING(21-11-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LURSQVdJTkcoMjktMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-DRAWING(29-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LURSQVdJTkcoR1JBTlRFRCktKDMwLTMtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-DRAWING(GRANTED)-(30-3-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1tdW1ucC0yMDA1LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1280-mumnp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LUZPUk0gMSgyMS0xMS0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-FORM 1(21-11-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LUZPUk0gMSgyOS0xMC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-FORM 1(29-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1tdW1ucC0yMDA1LWZvcm0gMTMoMi02LTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">1280-mumnp-2005-form 13(2-6-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1tdW1ucC0yMDA1LWZvcm0gMTMoMjktMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1280-mumnp-2005-form 13(29-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LUZPUk0gMTgoMTctNS0yMDA2KS5wZGY=" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-FORM 18(17-5-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1tdW1ucC0yMDA1LWZvcm0gMigyOS0xMC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1280-mumnp-2005-form 2(29-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LUZPUk0gMihDT01QTEVURSktKDIxLTExLTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-FORM 2(COMPLETE)-(21-11-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LUZPUk0gMihHUkFOVEVEKS0oMzAtMy0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-FORM 2(GRANTED)-(30-3-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LUZPUk0gMihUSVRMRSBQQUdFKS0oMjEtMTEtMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-FORM 2(TITLE PAGE)-(21-11-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LUZPUk0gMihUSVRMRSBQQUdFKS0oMjktMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-FORM 2(TITLE PAGE)-(29-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LUZPUk0gMihUSVRMRSBQQUdFKS0oR1JBTlRFRCktKDMwLTMtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-FORM 2(TITLE PAGE)-(GRANTED)-(30-3-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LUZPUk0gMygyMS0xMS0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-FORM 3(21-11-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LUZPUk0gMygyOS0xMC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-FORM 3(29-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LUZPUk0gNSgyMS0xMS0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-FORM 5(21-11-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LUZPUk0gNSgyOS0xMC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-FORM 5(29-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1tdW1ucC0yMDA1LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">1280-mumnp-2005-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1tdW1ucC0yMDA1LWZvcm0tMi5kb2M=" target="_blank" style="word-wrap:break-word;">1280-mumnp-2005-form-2.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1tdW1ucC0yMDA1LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">1280-mumnp-2005-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1tdW1ucC0yMDA1LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">1280-mumnp-2005-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1tdW1ucC0yMDA1LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">1280-mumnp-2005-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LU9USEVSIERPQ1VNRU5UKDI5LTEwLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-OTHER DOCUMENT(29-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LVBFVElUSU9OIFVOREVSIFJVTEUtMTM4KDItNC0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-PETITION UNDER RULE-138(2-4-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LVBPV0VSIE9GIEFVVEhPUklUWSgyOS0xMC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-POWER OF AUTHORITY(29-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LVNFUVVFTkNFIExJU1RJTkcoMjktMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-SEQUENCE LISTING(29-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LVNQRUNJRklDQVRJT04oQU1FTkRFRCktKDI5LTEwLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-SPECIFICATION(AMENDED)-(29-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI4MC1NVU1OUC0yMDA1LVdPIElOVEVSTkFUSU9OQUwgUFVCTElDQVRJT04gUkVQT1JUKDIxLTExLTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">1280-MUMNP-2005-WO INTERNATIONAL PUBLICATION REPORT(21-11-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=QkMxMDIwIFBDVCBTRVEuVFhU" target="_blank" style="word-wrap:break-word;">BC1020 PCT SEQ.TXT</a></p>
		<br>
		<div class="pull-left">
			<a href="233398-a-process-for-the-preperation-of-photostabilizer.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="233400-loose-tube-optical-cable.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>233399</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1280/MUMNP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>14/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>03-Apr-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>30-Mar-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>21-Nov-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>E.I.DU PONT DE NEMOURS AND COMPANY</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1007 MARKET STREET, WILMINGTON, DELAWARE 19898</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>MARK EMPTAGE</td>
											<td>2822 LONDON DRIVE, WILMINGTON, DE 19810</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SHARON HAYNIE</td>
											<td>529 SPRUCE STREET, PHILADELPHIA, PA 19106</td>
										</tr>
										<tr>
											<td>3</td>
											<td>JEFF PUCCI</td>
											<td>925 PAGE MILL ROAD, PALO ALTO, CALIFORNIA 94303</td>
										</tr>
										<tr>
											<td>4</td>
											<td>LISA LAFFEND</td>
											<td>2 MOUNT VERNON DRIVE, CLAYMONT, DE 19703</td>
										</tr>
										<tr>
											<td>5</td>
											<td>GREG WHITED</td>
											<td>304 SOUTH ROAD, BELMONT, CALIFORNIA 94002</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12P7/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US00/22874</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-08-18</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/149,534</td>
									<td>1999-08-18</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/233399-a-recombinant-e-coli by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:41:12 GMT -->
</html>
