<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/233584-a-method-for-expression-or-a-nucleic-acid-sequence by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:47:19 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 233584:&quot;A METHOD FOR EXPRESSION OR A NUCLEIC ACID SEQUENCE&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A METHOD FOR EXPRESSION OR A NUCLEIC ACID SEQUENCE&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to clean synthetic vectors intended to facilitate and improve the insertion of one or more genes of interest into a eukaryotic cell, and notably into a plant cell. The invention also relates to a procedure for obtaining these vectors as well as transgenic plants containing them.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The present invention relates as a method for enpresion of a<br>
nerlate need .  <br>
The present invention relates to clean synthetic vectors, intended notably for use for genetic transformation in the field of plant biotechnology.<br>
In general, the vectors are known in the field of biotechnology and genetic manipulation. The vectors which are currently used most commonly for genetic transformation, and in particular in the field of plant biotechnology, present several disadvantages in their use, however. In actual fact, and notably for plant transgenesis, use has often been made of a vector called pBin!9 (Frisch et al., 1995). The nucleotide sequence of this binary plasmid pBinl9 is entirely known. The problem, however, is that this plasiuid is of large size (11.8 kbp) and that it contains useless elements (more than half of p8in!9) which are intolerable from a regulatory point of view, or even detrimental to good replication. Moreover, the selection cassette of this plasmid is located near the right border of the T-DNA. Thus, when the T-DNA is broken after the selection cassette, it will not be possible to make any selection in order to retain solely the plants possessing the expression cassette of the desired gene.<br>
The expressions used in the description and the claims have the following meaning :<br>
- "vector" means an expression system, for example DNA-coated projectiles, nucleic-acid-based transit vehicles, nucleic acid molecules adapted to deliver nucleic acid, and autonomous self-replicating circular DNA, for example plasmids, cosmids, phagemids, etc. If a micro-organism or a recombinant cell culture is described as host of an "expression vector", this can also include extrachromosomal circular DNA 'such as for example mitochondria! or chloroplast DNA}, DNA which has been integrated into the host chromosome(s), where the vector can be either replicated in a stable manner by the cells during mitosis as an<br>
autonomous   structure,   integrated  into  the  genome   of  the  host,   or htained  in  the  nucleus or cytoplasm of  the host.<br>
usually the vectors used for the genetic transformation exist in the form of plasmids. In this case, the "plasmid" is a molecule of autonomous circular DNA capable of replication in a cell. If a micro-organism or recombinant cell calture is described as the host of. an "expression" plasmid, this comprises both extrachromosomal circular DNA molecules and DNA which has been integrated into the host chromosome's). If the plasmid is maintained by a host cell, the plasmid is either replicated in a stable manner by the cells during mitosis as an autonomous structure, or integrated into the genome of the host;<br>
-	"clean"  means  that  the  vector  comprises  only  sequences<br>
that    are    indispensable    for    its    functionality    and    carries    a<br>
nucleic   acid    sequence   which   comprises    only   elements    that    are<br>
indispensable   for the  expression  of the  host  cell;<br>
-	"nucleic  acid" means  DNA or RNA ;<br>
-	"nucleic    acid   sequence"   means   a   single-    or   double-<br>
stranded   oligomer   or   polymer   of   nucleotide   bases   read   from   the<br>
5'     end    towards     the     3'     end,     and    comprises     self-replicating<br>
plasmids,    genes,   DNA   or   RNA   polymers,    which   may   or   may   not   be<br>
infectious,      and     DNA,      or     RNA,      either     functional      or      non¬<br>
functional.     In    the    nucleotide    notation    used    in    the    present<br>
application,     unless     specifically    stated,     the    left    end    of    a<br>
single-stranded  nucleotide  sequence  is  the  5'   end;<br>
"derived nucleic acid sequence" means that the sequence derives directly or indirectly from the sequence referred to, for example by substitution, deletion, addition, mutation, fragmentation, and/or synthesis of one or more nucleotides;<br>
"promoter" means a nucleic acid region which is upstream of the translation initiation codon and which is involved in the recognition and binding of RNA polymerase and ether transcription proteins;<br>
-	"plant   promoter"   is   a   promoter   capable   of   initiating<br>
transcription  in plant  cells;<br>
"constitutive promoter" is a promoter capable of expressing nucleic acid sequences operationally bound to said promoter, in all or practically all the tissues of the host organism/ during the whole development of said organism ;<br>
-	"tissue specific promoter" is a promoter capable of<br>
selectively  expressing  nucleic  acid  sequences  operationally<br>
bound to said promoter, in certain specific tissues of the host<br>
organism;<br>
"operationally bound" means the binding of a functional or regulatory element, for example a promoter, to the nucleic acid sequence, or gene, to be expressed which codes for  protein to be produced, in such a way that this element influences the transcription of the bound nucleic acid sequence;<br>
"expression cassette" means nucleotide sequences capable of directing the expression of a nucleic a:id sequence, or of a gene, coding for a polypeptide to be produced in a host organism compatible with such sequences. Such cassettes include at least one promoter and a transcription termination signal, and optionally other factors necessary or useful for the expression ;<br>
-	"heterologous sequence" or "heterologous nucleic acid<br>
sequence" means a sequence originating from a source, or from a<br>
species, foreign to the environment thereof, or if it originates<br>
from the same environment, which has oeen modified relative to<br>
its original form. The modification of the nucleic acid sequence<br>
can take place for example by treatment of the nucleic acid with<br>
a restriction enzyme in order to generate a nucleic acid<br>
fragment which can be operationally bound to a promoter. The<br>
modification can also take place by means of techniques such as<br>
directed mutagenesis;<br>
"box" means a nucleic acid seqrence to which  a regulatory function is attributed;<br>
-	"like" meant that the box, and/or the nucleic acid<br>
sequence with which this term is associated, involves a certain<br>
sequence identity or consensus with a box and/or a known nucleic<br>
acid  sequence,  called  a  reference  sequence,  preferably  a<br>
sequence identity of at least 50%, more preferably a sequence identity of at least 75%, and more particularly a sequence identity of at least 90% with the reference sequence. The percentage of sequence identity is calculated on the basis of a comparison window of at least 6 r.ucieotide bases. The determination of a comparison window can be carried out using sequence alignment algorithms to determine a homology with a reference sequence, for example the local homology algorithm, the homology alignment algorithm, and the similarity search algorithm, these algorithms existing also in computerized form, known by the names GAP, BESTFIT, FASTA and TFASTA. The percentage of sequence identity is obtained by comparing the reference sequence with the box and/or the nucleic acid sequence;<br>
"situated" means the position on a nucleic acid sequence of an identified element, such as a "box", a restriction site, or a codon having a particular function. The position which is given by a figure refers to the position of the start of the element in the nucleic acid sequence, in the direction of reading of the latter, i.e. in the direction 5'-&gt;3';<br>
"transgenic plant" means a plant which has been obtained by genetic manipulation techniques, and °Cvers the whole plants obtained, their progeny, and the plant organs, for example the roots, stems and leaves, obtained by these techniques. The transgenic plants according to the present invention can have different levels of ploidy, and can notably be polyploid, dipioid, and haploid;<br>
- "propagule" means a cluster or association of plant cells, which may or may not be structured, allowing the regeneration of a whole plant, for example explants, call, stems, leaves, roots, cuttings, and even seeds.<br>
The applicant of the present invention has su°Ceeded, surprisingly, in producing clean synthetic vectors, in particular binary plasmids, of completely known nucleotide sequence, of small size, allowing the aforesaid disadvantages to<br>
be alleviated, anc notably presenting a high replication rate iative to the existing vectors most commonly used. Furthermore, the applicant has su°Ceeded at the same time in producing a range of vectors in such a way as to be able to choose the one which it is convenient to use according to the application envisaged and the environment of its use, and thus in such a way as to be able to better control the rate of expression of a gene to be expressed, coding for a polypeptide to be produced.<br>
In addition, in some of the vectors according to the invention, each of the functional elements or components can be isolated by simple enzymatic digestion.<br>
Ar. object of the present invention is therefore a clean synthetic vector containing only the elements indispensable to its functionality and to the transgenesis of a cell, and notably of a plant cell.<br>
according to a preferred embodiment, the vector comprises, as elements which are indispensable to its functionality and to the transgenesis of a ceil, and which are operationally bound:<br>
-	at least one nucleic acid sequence coding for at least<br>
one first origin of replication, preferably an ori RK2, and more<br>
preferably at least one ori V of pRK2 of Escherichia °Cli with a<br>
broad host range;<br>
at least one nucleic acid sequence coding for a selection agent, preferably an antibiotic resistance gene, more preferably the npt III gene conferring resistance to kanamycin in bacteria;<br>
-	a trfA locus coding for at least one protein allowing<br>
an increase in the replication rate of the plasmid, preferably<br>
originating  from pRK2,  and more  preferably  coding  for the<br>
proteins P285 and P382.<br>
according to a more preferred embodiment the vector comprises the nucleic acid sequence identified by the number SEQ.IL01. Still more preferably, the vector consists of a single<br>
plasmid pMRTilOS whose nucleic acid sequence is identified by  number SEQ. IDOL<br>
according to another preferred embodiment, the vector includes at least one nucleic acid sequence coding for a second origin of replication, preferably an ori of Escherichia coli, and more preferably an ori ColEI. More preferably, the vector comprises the nucleic acid sequence identified by the number EEQ.ID02, and still more preferably, the vector consists of a single plasmid pHRTHOb whose nucleic acid sequence is identified by the number SEQ.ID02.<br>
Preferably, the synthetic vector according to the invention comprises a region made up of at least one nucleic acid sequence containing several unique enzymatic restriction sites called collectively «multiple cloning sites» (MCS) .<br>
according to a preferred embodiment, the vector also includes a nucleic acid sequence coding for a T-DNA comprising a right border RB and a left border LB, allowing the vector to act as a binary plasmid.<br>
Preferably, the 'MCS is situated near the right border RB cf the T-DNA.<br>
according to another preferred embodiment, the vector also includes a nucleic acid sequence coding for at least one expression promoter and at least one transcription terminator situated between the left borc'er LB and the right border RB of the T-DNA. More preferably, the expression promoter is chosen from the group consisting of the constitutive promoters, the inducible promoters, the specific promoters, and preferably chosen from the plant expression promoters. Still more preferably, the expression promoter is chosen from the group consisting of the 35S CaMV promoter, the ep35S of CaMV, the pea plastocyanin gene promoter, its "enhancer" zones and derivatives, the wheat "High Molecular Weight Glutenin" (HMWG) promoter, the "Cassava Vein Mosaic Virus" CsVMV promoter, the "'Commeiina Yellow Mottle Virus" CoYMV promoter, the chimeric promoters of the CsVMV and CoYMV promoters, and their derivatives .<br>
Preferably, the expression terminator is chosen from the functional terminators in a plant cell, and is preferably a 35S nos terminator.<br>
according to yet another preferred embodiment, the vector comprises at least one nucleic acid sequence coding for a selection agent that is functional in a plant cell, preferably at least one acid sequence coding for an antibiotic resistance gene, end/or a herbicide resistance gene.<br>
Preferably, the sequence coding for a selection agent is a sequence coding for the bar («bialaphos resistance») resistance gene or pat («phosphinothricin acetyltransferase») gene, or else a sequence coding for the mutant or wild-type resistance gene nptll. More preferably still, the nucleic acid sequence coding for the selection agent is situated near the left border of the T-DNA.<br>
according to another preferred embodiment of the present invention, the vector includes at least one expression cassette comprising an expression-promoting nucleic acid sequence operationally bound to a nucleic acid sequence to be expressed, coding for a polypeptide to be produced, itself bound to a transcription termination nucleic acid sequence. Preferably, the polypeptide to be produced is an enzyme or protein or derivative of the latter having activity in vitro and/or in man and/or in animals, said activity comprising digestive, pancreatic, biliary, antiviral, anti-inflammatory, pulmonary, anti¬microbial, nutritional, cosmetic, structural, blood, cardiovascular, ophthalmic, antigenic, immunostimulatory and cerebral activity. Examples cf such proteins are for example the insulins, interferons, gastric, pancreatic or biliary lipases, elastases, antiproteases such as alpha-1 antitrypsin, structural proteins such as collagen, transferrins such as lactoferrin, proteins derived from blood, such as haemoglobin, human albumin and the blood cofactors, and antioxidants such as superoxide dismutase,<br>
according to a particularly preferred embodiment of the invention, the vector is presented in the form of a binary,<br>
linear or circular piasmid, chosen from the group consisting of<br>
 nucleic acid sequences identified bv the numbers SEQ-ID03,<br>
SEQ.ID04,  SEQ.ID05,  SEQ.IL06,  SEQ.ID07,  SEQ.ID08,  SEQ.ID09,<br>
3EQ.ID10, SEQ.IDli, SEQ.ID12, SEQ.ID13, SEQ.IDI4, SEQ.ID15, SEQ.ID16, SEQ.ID17, SEQ.1318, SEQ.ID19, SEQ.ID2C, SEQ.ID21 and SEQ.ID22.<br>
according to a particularly advantageous embodiment, each functional component of the vector can be cleaved independently of the other components. Preferably, each functional component can be cieaveo independently of the other components by enzymatic digestion at the level of a first unique restriction site and a second unique restriction site which are present in 1 vector.<br>
Another object of the present invention is an isolated nucleic acid sequence, characterized in that it corresponds to a nucleic acid sequence chosen from the group consisting of the nucleic acid sequences identified by the numbers SEQ.ID01, SEQ.ID02, SEQ.ID03, SEQ.ID04, SEQ.ID05, SEQ.ID06, SEQ.ID07, SEQ.ID08, SEQ.ID09, SEQ.ID10, SEQ.ID11, SEQ.ID12, SEQ.ID13, SEQ.ID14, SEQ.ID15, SEQ.ID16, SEQ.ID17, SEQ.ID18, SEQ.ID19, SSQ.ID20, SEQ.ID21 and SEQ.ID22.<br>
Yet another object of the present invention is a ceil containing a vector or a nucleic acid sequence such as described earlier. The cell is preferably a plant cell.<br>
Yet another object of the present invention is a transgenic plant having stably integrated in its genome a vector or a nucleic acid sequence such as described earlier. The plant is preferably chosen from the dicotyledon species, such as potato, tobacco, cotton, lettuce, "omato, melon, cucumber, pea, rape, beetroot or sunflower, or the monocotyledon species, such as wheat, barley, oats, rice cr maize.<br>
Yet another object of the present invention is a propagule of a transgenic plant such as described earlier. The propaguie is preferably a seed.<br>
Yet another object of the present invention is a method for expression of a nucleic acid sequence, or gene, coding fcr a<br>
to be produced, in a cell, characterized in that it comprises the stages consisting of:<br>
-	transforming the cell with a vector or a nucleic acid<br>
sequence such as described earlier;<br>
-	making a culture of the cell under conditions allowing<br>
the expression of a nucleic acid sequence, or gene, coding for<br>
the polypeptide  to be produced.  The  cell  is  preferably  a<br>
prokaryotic or eukaryotic cell. More preferably, the cell is a<br>
cell chosen from the group consisting of microbial cells, fungal<br>
cells, insect cells, animal ceils and plant cells. Still more<br>
preferably, the cell is a plant cell.<br>
Yet another object according to the present invention is a method for obtaining a transgenic plant or a propagule such as described earlier, characterized in that it comprises the stages consisting of:<br>
-	transforming a plant cell with a vector comprising or<br>
a nucleic acid sequence according to any one of Claims 1 to 24 ;<br>
-	selecting the plant ceil having the vector or nucleic<br>
acid sequence integrated therein;<br>
-	propagating the transformed and selected plant cell,<br>
either by culture or by regeneration of whole chimeric or<br>
transgenic plants.<br>
Thus, the new synthetic vectors, and preferably, binary plasmids, according to the invention have only regions indispensable for the functionality of the vector and for transgenesis. The applicant has found that they have a high replication rate. In addition, they can have a selection cassette near the left border, as well as rare restriction sites in their «polylinkers» (multiple cloning sites). DESCRIPTION OF THE FIGURES<br>
The invention will be understood better from the detailed description of the different embodiments which is given below in the form of examples, which are not limiting, and with reference to the drawing in the annex, in which:<br>
- Figure 1 schematically represents a preferred minimal vector according to the present invention in the form of a plasmid<br>
identified by the reference pMRTlIOS, and with a length of 508 base pairs;<br>
-	Figure 2 represents a schematic chart of a variant of the<br>
vector of Figure I in the form of a plasmid identified by t: e<br>
reference pMRTHOo, and with a longth of 4098 base pairs ;<br>
-	Figure 3 represents a schematic chart of a preferred vector<br>
according to the present invention, in the form of a binary<br>
plasmid identified by the reference pMRTlllS, with a length of<br>
5971 base pairs, and including a T-DNA comprising a selection<br>
cassette  coding  for  resistance  to  an  antibiotic,  and  a<br>
multiple cloning site (MCS);<br>
-	Figure 4 represents a preferred embodiment of the vector<br>
according  to  Figure  3,  the  multiple  cloning  site  (MCS)<br>
including yet other unique sites, the vector being in the form<br>
of a binary plasmid identified by the reference pMRT1119, and<br>
with a length of 6016 base p = irs ;<br>
-	Figures 5 to 22 represent other preferred embodiments of the<br>
vector according to the invention, in the form of binary<br>
plasmids identified respectively by the references pMRT1121<br>
PMRT1176 PMRT1I92 DMRT1195 PMRT1201 pMRTi/03<br>
'7503 base oairs'<br><br>
(6017 	base 	pairs) , 	p.MRT1122 	(6016 	base 	pairs) , <br>
(6017 	base 	pairs) , 	pMRT1175 	(6767 	base 	pairs) , <br>
( 6767 	base 	pairs) , 	pMRT 1191 	(4805 	base 	pairs) , <br>
(8654 	base 	pairs) , 	pMRT1193 	(9143 	base 	pairs) , <br>
(6865 	base 	pairs) , 	pMRT1196 	(8654 	base 	pairs) , <br>
(7943 	base 	pairs) , 	PMRT1202 	(5614 	base 	pairs) , <br>
PMRT1204 (9390 base pairs), pMRT1205 (7503 base pairs), pMRT1206 (9390 base pairs), pMRT1210 (10003 base pairs) and pHRT1212 (8987 base pairs) ;<br>
- Figure 23 graphically represents a comparison between the protein expression capacities of the sythetic vectors according to the present invention in comparison to a known vector system ;<br>
-- Figures 24 to 28 represent further embodiments of vectors according to the present invention, in the form of binary plasmids identified by the references pMRTl334 (9688 bp),<br>
pMRT1335 (15208 bp) , pMRTI336 (92t5 bp) , pM.KT1337 (3289 bp) , 'pMRT1341 (14108 bp), pMRT1342 (15077 bp) .<br>
In the figures, the abbreviations have the following meanings :<br>
-	ori RK2 : ori V of pRK2 of Esche. ichia °Cli with a broad<br>
host range, unstable RK2 replicon producing a low number of<br>
copies;<br>
-	ori ColEI : the origin of replication ColEl of Escherichia<br>
coli;<br>
npt III gene: codinc for necmycin phospnotransferase conferring resistance to kanamyciri<br>
npt II gene: codinc for neomycin phospnotransferase conferring resistance to kanamycin ;<br>
-	trfA : trfA locus (1481 bp; originating from pRK2 allowing<br>
the production of 2 proteins, P285 and P3S2, which enhance the<br>
replication of  the  plasmid by binding  to  the  origin of<br>
replication ;<br>
-	Tnos : nos (nopaline synthetase) terminator;<br>
-	Pnos : nos \nopaline synthetase) promoter;<br>
-	LB : left border of a T-DNA ;<br>
-	RB : right border of a T-DNA ;<br>
-	MCS : multiple cloning site, S:XXX indicating the number of<br>
the starting base, ErXXX indicating the number of the end base<br>
of the MCS, the list starting with the first restriction site<br>
at the top and ending with the last restriction site at the<br>
bottom, in the direction 5'&gt;3' ;<br>
polyA 35S : transcription termination (pclyadenylation) signal ;<br>
-	gus gene: gene coding for beta-glucuronidase ;<br>
-	IA : the actin ir.tron of rice ;<br>
-	PA : actin promoter of rice ;<br>
-	Phmwg : «high molecular weight glutenin» promoter of wheat ;<br>
bar («bialaphos resistance))) gene: gene cooing for the enzyme phosphinothricin acetylrransferase conferring resistance to glufosinate ;<br>
-	ep35S : «enhanced promoter)) of the 25S rj.bosorr.e.<br>
In the detailed description which follows/ all the enzymatic digestions by restriction enzymes were performed in accordance with the recommendations of the supplier New England Biolabs. The purifications with the aid of the «QIAquick Gel Extraction:* and «QIAquick PCR Purif ication» kits were carried out in accordance with the recommendations of the supplier QIAGEN. The «Concert Rapid PCR Purification System» kit was used in accordance with the instructions of the supplier GIBCO BRL Life Technologies. The «GeneAmp FCR System 3700» thermocycler used is marketed by Perkin Elmer Applied Biosystems.<br>
Example 1<br>
1. Synthesis of pMRTHOS<br>
A clean synthetic vector according to the invention is presented preferably in the form of a minimal plasrnid pMRTllOS (3508 bp) and is made up of the following elements:<br>
-	ori RK2 : ori V of pRK2 of Escherichia coli with a<br>
broad host range, unstable RK2 replicon producing a small number<br>
of copies (643 bp).<br>
npt  III  gene .:  confers  resistance  to  kanamycin (bacterial selection marker, 1337 bp).<br>
-	TrfA : trfA locus (1481 bp) originating from pFK2,<br>
allowing the production of 2 proteins,  P285 and P382, which<br>
enhance the replication of the plasmid by binding to the origin<br>
of replication.<br>
The plasmid pMRTllOS results from the assembling of the fragments obtained by splicing overlap extension for «ori RK2» and a «part of npt III», of the fragment corresponding to a «part of trfA» produced by PCR («polymerase chain reaction») amplification and of «parts of trfA and npt III» isolated by enzymatic digestion.<br>
1.1. Synthesis of the fragment carrying «ori RK2 and part, of npt III»<br>
The AvrII-Sr.uI  fragment  (654 bp)  carrying «cri  RK2»<br>
( c 4 3 bp) was amplified by PCR from 5 ng of pEJnl9 matrix DMA<br>
with   the   aid   of   20   pmoles   of   each   of   the   2<br>
cligodeoxynucieotides, 5' AaccTAGGAAAAGaccGAGCG°CTTTGC 3' SEQ.ID23} containing the Avrll restriction site and 5' CGGATTAATGGTAGAAGG°CTTTCACGC-GAGGGTTCGAGAAGG 3 ' (SEQ. ID24 ) possessing the StuI restriction site, in the presence of 200 µM of each of the dNTPs, 60 mM Tris-SO,, pH 9.1, 18 mM (NH,)2 SO,, 1 .8 mM MgSO* and 2 U of ELONGASE enzyme (GIB°C BRL Life Technologies) in a final reaction medium of 50 µl  The PCR amplification reaction was carried out in the «GeneAmp PCR System C700» thermocycler. After denaturaticn at 94°C for 5 min., the DNA was subjected to 15 cycles each made up of the stages of denaturation at 94°C for 30 sec., hybridization at 55 °C for 30 sec. and elongation at 68°C for 45 sec. Then, in the last cycle, the elongation was continued at 68°C for 3 min. Forty µl of the PCR reaction medium were then subjected to the action of 12.5 U of the Klenow fragment (New England Biolabs) in the presence of 2 µl of each of the dNTPs at 10 mM. The reaction was carried out at 37°C for 10 min. The PCR product so treated was then isolated by 2% agarose gel electrophoresis in TBE buffer (90 mM Tris-HCi, 2 mM Na2~EDTA, 90 mM boric acid, pH 8.0) and purified with the aid of the «QIAquick Gel Extraction» kit. The DNA was recovered in 30 µl of H2O.<br>
The StuI-BstXI fragment (363 bp) carrying «part of npt<br>
III» (344 bp) was amplified and treated in the same way as the<br>
«ori RK2» fragment except that the 2 oligodeoxynucleotides used<br>
are 5' TGAAAGG°CTTCTaccATTAAT°CGCGATAAAC°CAGCGAacc 3'  (SEQ.ID25)<br>
containing    an    StuI    restriction    site    and    5'<br>
ATGCAT°CAAAATTTTGGTAGAATTTACAAGCTATAAGGTTATTGT°CTGGG	3'<br>
(SEQ.ID26) possessing the BstXI restriction site.<br>
The 2 fragments carrying «ori RK2» and «part of npt III» Vvere then assembled by splicing overlap extension in order ~ obtain the fragment «cri RK2 and part of npt III». To do this, a PCR amplification was performed from 7.5 µl of each of the treated PCR products corresponding to «ori RK2» ana «part of npt III» with the aid of 20 pmoles of each of the oligodeoxynucleotides 5' AaccTAGGAAAAGaccGAGCG°CTTTGC 3' {SEQ.ID23} :,nd 5' ATGCAT°CAAMTTTTGGTAGAATTTACAAGCTATAAGGTTATTGT°CTGGG 3'<br>
(SEQ.ID26) in the presence of 200 µM of each of the dNTPs, 60 mM .s-SO4 pH 9.1, 18 mM (NHJz SO,, 1.8 mM MgSO4 and 2 U of zLONGASE enzyme in a final reaction medium of 50 µl  The PCR amplification reaction was carried out in the «GeneAmp PCR System 9700» thermocycler. After denaturation at 94 °C for 3 min., the DNA was subjected to 15 cycles each made up of the stages of denaturation at 94°C for 30 sec., hybridization at 62°C for 30 sec. and elongation at 68°C for 45 sec. Then, in the last cycle, the elongation was continued at 68°C for 3 min. Forty µl of the FCR reaction medium were then subjected to 1.5% agarcse gel electrophoresis in TBE buffer and purified with the aid of the «QIAquick Gel Extraction» kit. The DNA, corresponding to the fragment carrying «ori RK2» and «part of npt III», was recovered in 30 µl of H2O. Next, 27 µl of this DNA were hydrolysed by BstXI followed by Avrll, then purified with the aid of the «QIAquick PCR Purif ication» kit and recovered in 30 µl of H2O.<br>
1.2.	Synthesis of the fragment carrying a «part of trfA»<br>
The Ndel-Avrll fragment (295 bp) carrying «part of trfA»<br>
(295 bp) was amplified and treated in the same way as the fragment «ori RK2» except that the 2 oligcdeoxynucleotides used are 5' ATCGACGAGGAAATCGTCGTGCTGTTTGC 3' (SEQ.ID27) situated upstream of the Ndel site and 5' AAaccTAGGAAATG°CAGTAAAGCGCTGGC 3' (SEQ.ID28) possessing the Avrll restriction site. The DNA fragments corresponding to a «part of trfA» originating from the PCR amplification are then purified with the aid of the «QIAquick PCR Purification)) kit, recovered in 30pl of H2O, hydrolysed by Avrll and Ndel, repurified with the aid of the «QIAquick PCR Purification* kit and recovered in 30 µl of H2O.<br>
1.3.	Synthesis of the fragment carrying the «parts of<br>
trfA and npt III»<br>
The BstXI-Ndel fragment '2196 bp) carrying the «parts of trfA ar:d npt III» (2196 bp) was isolated from 9 µg of pBin!9 DNA by enzymatic digestion by BstXI, purified with the aid of the «QIAquick PCR Purification)) kit, then hydrolysed by Ndel. The DNA fragment was then isolated by 0.9% agarose gel<br>
ctrophoresis    in    TEE   buffer,    purified   with    the    aid   of   the<br>
«QIAquick Gel Extraction» kit and recovered in 30 µl of H2O.<br>
1.4. Obtaining pMRTHOS<br>
The 3 fragments, «ori RK2 and part of npt III», «parts of trfA and npt III» and «part of trfA», were ligated. To do this, the reaction mixture containing 7.5 µl of the digested fragments corresponding to «ori RK2 and part of npt III» and «parts of trfA and npt III», and 11 µl of the digested fragment corresponding to «part of trfA», was concentrated with the SpeeaVac AES1000 (SAVANT) in order to achieve a volume of 8 ul. After that, 1 µl of T4 DNA ligase x 10 buffer (New England Biolabs) and 400 units of T4 DNA ligase enzyme (New England Biolabs) were added. The ligation was carried out by a PCR reaction made up of 200 cycles each consisting of 2 stages, one at 30°C for 30 sec. and the other at 10°C for 30 sec., in the «GeneAmp PCR System 9700» thermocycler. The bacteria,<br>
Escherichia coli DH5α, which had previously been made competent, were transformed (Hanahan, 1983). The plasmid DNA of the clones obtained, selected on Luria-Bertani medium (LB, bactotryptone 10 g/1, yeast extract 5 g/1, NaCl 10 g/1, Agar 15 g/1, pH 7.5) supplemented with kanamycin (50 mg/1), was extracted by the alkaline lysis method (Birnboim and Doly, 1979} and verified by enzymatic digestions and by sequencing. The resulting plasmid selected was called pMRTHOS (3508 bp) and is represented in Figure 1. Its complete sequence SEQ.ID01 is given in the sequence listing.<br>
2 Synthesis of pMRT1106<br>
The plasmid pMRT1106 (4098 bp) differs from pMRTHOS by the addition of tne origin of replication ColEl of Escherichia coli («ori ColEl»).<br>
The fragment, carrying «o.ri ColEl» (590 bp) , was isolated from the plasmid pBR322 marketed by New England Biclabs. The plasmid (5 ug) was digested by Hole I, purifieo with the aid of the «QIAquick PCR Purification» kit and recovered in 30 µl of TE buffer (10 mM Tris-HCl, 1 mM Na2-EPTA, pH 3.0). The<br>
plasmid thus linearized was subjected to the action of 20 units Infant he Klenow fragment (New England Biolabs) in the presence of 12 µl of 500 mM Tris-HCl, pH 7.5 - 500 mM MgCl2, 6 µl of 1M dithiothrextol, 6 µl of each of the 10 mM dNTPs in a reaction volume of 120 µl at 37°C for 30 min. The linearized plasmid so treated was purified with the aid of the «QIAquick PCR Purification» kit and recovered in 30 µl of H2O. It was then digested by AlwNI in order to isolate the fragment carrying «ori ColEl», purified with the aid of the «QIAquick PCR Purification» kit and subjected to the action of 6 units of T4 DNA polymerase (New England Eiolabs) in a reaction medium of 120 µl in the presence of 12 µl of T4 DNA polymerase X 1C buffer, 4 µl of 10 mM dNT? and 6 µl of B3A 1 mg/ml. The reaction was carried out at 37°c for 30 min. The DNA fragment was then isolated by 1.2% agarose gel electrophoresis in TEE buffer, purified with the aid of the «QlAquick Gel Extraction» kit and recovered in 50 µl of H2O. This DNA fragment was then inserted into the plasmid pMRTHOS (2 ug) digested by StuI, purified with the aid of the «QIAquick PCR Purification» kit, dephosphcrylated by 50 units of calf intestinal alkaline phosphatase (New England Biclabs) in a final reaction medium of 120 µl in the presence of 12 µl of 3 x 1C buffer (New England Biolabs) at 37°C for 1 hour, and purified with the aid of the «QIAquick PCR Purif ication» kit. The ligation by PCR reaction was carried out with 10 ng of digested dephosphorylated pMRTHOS plasmid and 100 ng of DNA fragments carrying «ori ColEl» in a reaction medium of 10 µl in the presence of 1 µl of T4 DNA ligase x 10 buffer (New England Biolabs} and 400 units of T4 DNA ligase enzyme (New England Biolabs) . It is made up of 180 cycles consisting of 2 stages, c-ne of 10°C for 30 sec. and the other of 30°C for 30 sec. in the «GeneAmp PCR System 9700» thermocycler. The bacteria,<br>
Eschericnia °Cli LH5a, which had previously been mads competent, were transformed (Kanahan, 1983). The validity of the clones obtained was verified by a PCR test on bacterial colonies using 10 pmol of each of the 2 oligodeoxynucleotides, 5' AaccTAGGAAAAGaccGAGCG°CTTTGC 3' (SEQ.ID23) and 5'<br>
ATT°CAAAATTTTGGTAGAATTTACAAGCTATAAGGTT'ATTGT°CTGGG 3! (SEQ. ID2 6) , in a reaction medium of 24 µl in the presence of 22 µl of SuperMix PCR (GIBCO ERL Life Technologies) . The PCR amplification reaction was carried out in the «GeneAmp PCR System 9700» thermocycier. After denaturation at 94 °C for 5 min., the DNA was subjected to 30 cycles each made up of the stages of denaturation at 94°C for 30 sec., hybridization at 60°C for 30 sec. and elongation at 72°C for 30 sec. Then, in the last cycle, the -longation was continued at 72°C for 7 min. The PCR reaction medium was then subjected to 1% agarose gel electrophoresis in TEE buffer. The insertion of the sequence «ori ColEl» was visualized by the presence of a fragment of about 1.6 kbp. The plasmid DNA of the clones obtained, selected on LB medium supplemented with kanamycin (50 mg/1), was extracted by the alkaline lysis method (Birnboim and Doly, 1979) and verified by enzymatic digestions and by sequencing. The resultant plasmid selected was called pMRT1106 (4098 bp) and is represented in Figure 2. Its complete sequence SEQ.ID02 is given in the sequence listing.<br>
3. Synthesis of pMRTUlB<br>
The plasmid pMRTlllS (5971 bp) differs from pMRT1106 by the introduction of a clean transfer DNA (T-DNA) in pMRT1106. This clean T-DNA is mace up of the expression cassette of the mutant nptll gene (Frisch et al., 1995) under the control of the promoter and terminator of the nopaline synthase gene (nos, Depicker et al. , 1982) of Agrobacterium tumefaciens placed between the right (RB) and left border (LB) of the plasmid pTiT37 of Agrobacterium tumefaciens nopalire strain.<br>
3.1 Synthesis of the fragment carrying «end nptll - Tnos - LB»<br>
3.1.1. Obtaining the fragment carrying LB<br>
The Avrll-Smal fragment (173 bp) carrying LB (151 bp)<br>
was amplified by PCR frcrn 5 ng of pBinI9 matrix DNA with the aid<br>
of  20  pinoles  of  each  of  the  2  oliaodeoxynucleot ides,  5'<br>
TT°CTAGGTTGACGTCTTCTGATGGGCTG°CTGTATCG 3'  (SEQ.ID29)  containing<br>
the   Avrll    and   Aatll   restriction   sites,    and    5'<br>
°CTATGGATATC°C°CGGGGGATAG°C°CAGTACATTAAAAACGT°C 3 ' (SEQ. IDS0 ) •Spoisessing the Smal restriction site, in the presence of 200 µM of each of the dNTPs, 60 mM Tris-S0
3.1.2.	Obtaining  the  fragment  carrying  the  nos<br>
terminator (Tnos)<br>
The Smal-BspEI fragment (288 bp) carrying the nos terminator (256 bp) was amplified and created in the same way as the fragment carrying LB except that the 2 oligodeoxynucleotides used are 5' CTATC°C°CGGGGGATAT°CATAGGC°CGATCTAGTAACATAGATGAC 3' (SEQ.ID31) containing a Smal restriction site and 5' GCGCACTTGGGC°CATAGCTCGACGAACGATCGTTCAAACATTTGGC 3 ' (SEQ. ID32 ) , possessing the Bspl20I restriction site.<br>
3.1.3.	Obtaining the fragment  carrying «er.d of  the<br>
nptll» gene («end nptll»)<br>
The fragment (221 bp) carrying «end nptll» (221 bp) was amplified and treated in the same way as the fragment carrying LB except that the 2 oligodeoxynucleotides used are 5r TTCGTCGAGCTATGGGC°CAAGTGCGCATC°CGTGGGCGAAGAACTC 3 ' (SEQ. ID33 ) containing a Bspi20I restriction site and 5' TTCTTGACGAGTTCTTCTGAGCGGG 3' (SEQ.ID34) downstream of EstBI.<br>
3.1.4.	Obtaining the fragment «end nptll - Trios - LB»<br>
The fragments, «LB», «Tnos» and «enci nptll», were assembled by splicing overlap extension in 2 stages :<br>
-	assembly of the fragments «LB» and Tnos» resulting in<br>
a fragment «Tncs - LB»,<br>
-	assembly of the fragment «Tnos - LB» and of the<br>
fragment «end nptll» resulting in a fragment «end nptll - Tnos<br>
-LB».<br>
To do this, a first PCR amplification was carried out from 10 µl of each of the PCR products treated corresponding to «L3» and «Tnos», with the aid of 20 pmcles of each of the oligodeoxynucleotides, 5' TT°CTAGGTTGACGTCTTCTGATGGGCTG°CTGTATCG 3' (SEQ.ID29) and 5' GCGCACTTGGGC°CATAGCTCGAO3AA:GA.TCGTTCAAACATTTGGC 3' (SEQ.ID32), in the presence of 200 µM of each of the dNTPs, 60 mM Tris-S04 pH 9.1, 18 mM (NHOa S04 1.8 mM MgSC4 and 2 U of ELONGASE enzyme (GIBCO BRL Life Technologies) in a final reaction medium of 50 µl.  The PCR amplification reaction was carried out in the «GeneAmp PCR System 9700» thermocycler. After denaturation at 94°C for 3 min., the DNA was subjected to 15 cycles each made up of the stages of denaturation at 94 "C for 45 sec., hybridization at 62°C for 45 sec. and elongation at 68°C for 1 min. Ten, in the last cycle, the elongation was continued at 68°C for 3 min. Forty µl of the PCR reaction medium were then subjected to the action of 12,5 U of the Klenow fragment (New England Biolabs) in the presence of 2 yl of each of the dNTPs at 10 mM. The reaction was carried out at 37°C for 10 min. The PCR product so treated, corresponding to the fragment «Tnos - LB» (477 bp) , was then isolated by 2% agarose gel electrophoresis in TBE buffer and purified with the aid of the «QIAquick Gel Extraction» kit. The DNA was recovered in<br>
30	ul of H?0.<br>
A second FCR amplification Was then carried out from 7 µl of each of vhe PCR products treated, corresponding to «end nptll» and «Tnos - LB», with the aid of 20 pmoles of each of the OJ igcaeoxynucleot ices , 5' TT°CTAGGTTGACGTCTTCTGATGGGCTG°CTGTATCG<br>
31	(SEQ.ID29) and 5' TTCTTGACGAGTTCTTCTGAGCGGG 3' (SEQ.ID34), in<br>
the presence of 200 uK of each of the dNTFs, 60 mM Tris-S04 pH<br>
9.1, 18 mM (NK4)2 S04, 1.8 mM MgS04 and 2 U of cLONGASE enzyme MGiBCO ERL Life Technologies) in a final reaction medium of 50 µl.  The PCR amplification reaction was carried out in the «GeneAmp PCR System 3700» therm°Cycler. After denaturation at 94 °C for 3 min ., the DNA was subjected to 15 cycles each made up  of the  stages  of  denaturation  at  94 °C  for 45  sec., hybridization at 60°C for 45 sec. and elongation at 68°C for 1 min. Then, in the last cycle, the elongation was continued at 68 °C for 3 min. This reaction is repeated 3 times for the PCR products treated, corresponding to «end nptll» and «Tncs - LB» . The PCR reaction medium was then subjected to 2% agarose gel electrophoresis in TBE buffer and purified with the aid of the «QIAquick Gel  Extraction»  kit.  The  DNA fragment  (672 bp), corresponding to «end nptll - Tnos - LB», was recovered in 100 µl of H2O. Next, 95 µl of this DNA were hydrolysed by Avrll, purified with the aid of the «QIAquick PCR Purification» kit, digested by BstBI, purified with the aid of the «QIAquick PCR Purification» kit and recovered in 50 µl of H2O. The 631 bp Avril-BstBI fragment carries the sequence «enc nptll - Tnos -LB» .<br>
3.2.	Synthesis of the fragment carrying «end of the nos<br>
promoter (Pnos) and start of the nptll gene» («end Pnos - st .<br>
nptll»)<br>
The fragment (1023 bp} carrying «end Pnos - st. nptll» was amplified and treated in the same way as the fragment carrying LB except that the 2 olioodeoxynucleotldes used are 5' GGAATCGAAATCTCGTGATGGCAGG 3' (SEQ.ID39) and 5' ATTATTGCGCGTTCAAAAGTCG°C 3' (SEQ.ID40), and that the DNA was subjected to 15 cycles each made up of the stages of denaturation at 94 °C for 45 sec., hybridization at 55°C for 45 sec. and elongation at 68 °C for 1 min. The amplification was repeated for 2 samples of pBin'i9 DNA.<br>
3.3.	Synthesis of the fragment «end Pnos - nptll - Tnos - LB»<br>
The fragments, «end Pnos - st. nptll» and «end nptll -<br>
Tnos - LB», were assembled by splicing overlap extension in order to produce a fragment «end Pnos - nptll - Tnos - LB».<br>
io do this, a PCF; amplification was carried out from of each cf the PCR products treated, corresponding to «end Pnos - st. nptll» anc «end nptll - Tncs - LB», with the aid of 20 pmoies of each of the oligodeoxynucleotides, 5! TT°CTAGGTTGACGTCTTCTGATGGGCTG°CTGTATCG 3' (SEQ.ID29) and 5' ATTATTGCGCGTTCAAAAGTCG°C 3' (SEQ.ID40), in the presence of 200 µM of each of the dNTPs, 60 mM Tris-SO4 pH 9.1, 18 mM (NH6)2 SO4, 1.8 mM MgSO4 and 2 U of iLONGAsc enzyme (GIBCO BRL Life Technologies) in a final reaction medium of 50 µl.  The PCR amplification reaction was carried out in the «GeneAmp PCR System 9700» thermocycler. After denaturation at 94°C for 3 min., the DNA was subjected to 15 cycles each made up of the stages of denaturation at 94°C for 45 sec., hybridization at 55°C for 45 sec. and elongation at 68°C for 1 min. Then, in the last cycle, the elongation was continued at 68°C for 3 min. This reaction was repeated 5 times for each of the PCR products treated, corresponding to «end Pnos - st. nptll» and «end nptll - Tnos - LB». The PCR reaction medium was then subjected to 0.7% agarose gel electrophoresis in TBE buffer and purified with the aid of the «QIAquick Gel Extraction» kit. The DNA fragment (1608 bp) corresponding to «end Pnos - nptll - Tnos - LB», was recovered in 50 µl cf H2O. Next, this DNA was subjected to the action of 62.5 U of the Klenow fragment (New England Biolabs) in the presence of 15 µl of the Klenow fragment x 10 buffer (New England Biolabs) and 10 µl of each of the dNTPs at 10 mM. The reaction was carried out at 37°C for 10 min. The DNA so treated was then purified with the aid of the «QIAquick PCR Purification» kit and recovered in 50 µl of H2O.<br>
3.4. Synthesis of the fragment carrying «RB - MCS - st. Pnos»<br>
3.4.1. Obtaining the fragment carrying «RB and multiple cloning site (MC3)» («RB - MCS»)<br>
The fragment (210 bp) carrying «RB - MCS» was amplified and treated i:i the same way as the fragment carrying LB except that    the    2    oligodeoxynucleotides    used    are    5' CGGTaccGAA3CTTTGAATTCACTCGAGCAGATTGTCGTTTC°CG°C   3'   (SEQ.ID35) possessing the restriction sites Kpnl, Hindlll, EcoRI and Xhcl,<br>
and   5'   TAT°C7AGGAaccGGIAAAC°CTGTGGT7GGCATGC   3'   (SEQ.ID36) sessing the restriction sites Avrll and Agel.<br>
3.4.2.	Obtaining the fragment carrying «multiple cloning<br>
site (MCS) and start of the nos promoter (Pnos)» («MCS - st.<br>
Pnos»)<br>
The fragment (209 bp) carrying «MCS - st. Pnos» was amplified and treated in the same way as the fragment carrying LB except that the 2 oligodeoxynucleotides used are 5' ATATGAGACTCTAATTGGATaccGAGGGG 3' (SEQ.ID37) and 5' GCTCGAGTGAA7TCAAAGCT7CGGTaccGTIGAAGGAG°CACTCAG°CG 3 ' (SEQ. IDS8 } possessing the restriction sites Xhcl, EcoRI, Hindlll and KpnI.<br>
3.4.3.	Obtaining the fragment carrying «RB - MCS - st.<br>
Pnos»<br>
The fragments, «RB - MCS» and «MCS - st. Pnos», were assembled by splicing overlap extension in order to produce a rragment «R3 - MCS - st. ?nos».<br>
To do this, a PCR amplification was carried out from 12 µl of each of the PCR products treated, corresponding to «RB - MCS» and «MCS - st. Pnos», with the aid of 20 pmoles of each of the oligodeoxynucleotides, 5' ATATGAGACTCTAATTGGATaccGAGGGG 3' (SEQ.ID37) and 5' TAT°CTAGGAaccGGTAAAC°CTGTGGTTGGCATGC 3' i,5EQ.ID36), in the presence of 200 µM of each of the dNTPs, 60 mM Tris-S04 pH9.1, 18 mM (NH412 S04, 1.8 mM MgS04 and 2 U of ELONGASE enzyme (GIBCO BRL Life Technologies) in a final reaction medium of 50 µl.  The PCR amplification reaction was carried out in the «GeneAmp PCR System 970C» thermocycler. After cenaturation at 94°C for 3 min., the DNA was subjected to 15 cycles each made up of the stages of denaturation at 94 °C for 30 sec., hybridization at 60°C for 30 sec. and elongation at 66 °C for 45 sec. Then, in the last cycle, the elongation was continued at 68°C for 3 min. This reaction was repeated for the remaining 3 x 12 µl of each of the PCR products treated, corresponding to «R3 - MCS» and «MCS - st. Pncs». The FOR reaction medium was then subjected to 2% agarcse gel electrophoresis in TBE buffer and purified with the aid of the «QIAquick Gel Extraction» kit (QIAGEN). The DNA fragment<br>
(390 bp) corresponding to «F.B - MCS - st. Fnos», was recovered<br>
in 100 µl of H2O. Next, 95 µl of this LNA was hydrolysed by Avrll, purified with the aid of the «QIAquick PCR Furification» kit, digested by Bsu36I, purified with the aid of the «QIAquick PCR P-jrification» kit and recovered in 50 µl of K;0. The 295 bp AvrII-Bsu36l fragment carries the sequence «RB - MCS - st. Fnos».<br>
3.5 Synthesis of the T-DNA<br>
The fragments, «RB - MCS - st. F;.os» and «end Pnos -nptll - Tnos - LE», were assembled by splicing overlap extension in order to produce a fragment «RE - MCS - Pnos - nptll - Tnos -LB» which corresponds to the T-DNA (1883 bp).<br>
To do this, a PCR amplification was carried out from 4 µl of the PCR product treated corresponding to «R3 - MCS - st. Pnos» and from 5 µl of the PCR product treated corresponding to «:end Pnos - nptll - Tnos - LB», with the aid of 20 pmoles of each of the oligodeoxynucleotides, 5r T1°CTAGGTTGACGTCTTCTGATGGGCTG°CTGTATCG 3' (SEQ.ID29) and 5' TAT°CTAGGAaccGGTAAAC°CTGTGGTTGGCATGC 3' (SEQ.ID36), in the presence of 200 µM of each of the dNTPs, 60 mM Tris-S04 pH 9.1, 13 mM (NH4)2 S04, 1.8 mM MgS04 and 2 U of ELONGASE enzyme (GIBCO BRL Life Technologies) in a final reaction medium of 50 µl.  The FCR amplification reaction was carried out in the «GeneAmp FCR System 9700» thermocycler. After denaturation at 94 °C for 3 min., the DNA was subjected to 15 cycles each made up of the stages of denaturation at 94°C for 45 sec., hybridization at 55°C for 45 sec. and elongation at 68°C for i min. Then, in the last cycle, the elongation was continued at 68°C for 3 min. This reaction was repeated 10 times for each oi the PCR products treated, corresponding to «R3 - MCS - st. Pnos» and «end Pnos -nptll - Tnos - LB:&gt;&gt;. The PCR reaction medium was then subjected to 0.8% agarose gel electrophoresis in TBE buffer ana purified with the aid of the «QIAquick Gel Extraction» kit. The DNA fragment (2883 bp) corresponding to the T-DNA was recovered in<br>
100 µl of H2O. Next, this T-DNA was hycirolyseci by Avrll (1873 bp gment),  purified  with  the  aid  of  the  «QIAquick  PCR Purifications kit and recovered in 100 µl cf H2O.<br>
3.6. Obtaining pMRTlilB<br>
The binary plasmid pMRTIUS (5971 bp) results from the introduction of the T-DNA fragment digested by Avrll into the Avrll site of the dephosphcrylated pMRTHOS plasmid.<br>
To do this, the pMRTI106 plasmid DNA (5 µg) was digested by Avrll, purified with the aid of the «QIAquick PCR Purification kit, then dephosphcrylated by 50 units of calf intestinal alkaline phosphatase (New England Biolabs) in a final reaction medium of 120 µl in the presence cf 12 µl of 3 x 10 buffer (New England Biolabs) at 37 °C for 1 hour, isolated by 0.6% agarose gel electrophoresis in TBE buffer, purified with the aid of the «QIAguick Gel Extractions kit, dephosphorylated a second time with the calf intestinal alkaline phosphatase enzyme under the aforesaid conditions, and finally purified with the aid of the «QIAquick PCR Purification» kit and recovered in 50 µl of H2O.<br>
The ligation by PCR reaction was carried out with 32.5 ng of digested dephosphorylated pMRT1106 plasmid and 50 ng of digestea T-DNA fragments in a reaction medium of 10 µl in the presence of 1 µl of T4 DNA ligase x 10 buffer (New England Biolabs) and 400 units of T4 DNA ligase enzyme (New England Biolabs) . It is made up of 180 cycles consisting of 2 stages, one of 10°C for 30 sec. and the other of 30°C for 30 sec., in the «GeneAmp PCR System 9700» thermocycler.<br>
The bacteria, Escherichia coli DH5α, which hao previously been made competent, were transformed (Hanahan, 1983) . The plasmid DNA of the clones obtained, selected on LB medium supplemented with kanamycin (50 mg/1), was extracted by the alkaline lysis method (Birnbcim and Doly, 1979) and verified by enzymatic digestions and by sequencing. The resultant plasmid selected was called pMRTlllS (Figure 3). Its complete sequence SEQ.ID03 is given in the sequence listing.<br>
4.   Synthesis  of pMRTI119<br>
The plasmid pMRT1119 (6016 bp) differs from pMRTlllS by the addition of additional unique restriction sites in the multiple cloning site MCS of pMRTillB.<br>
4.1.	Obtaining  the additional  unique  restriction  sites<br>
The    additional    unique    restriction    sites     (Xbal,     Sail,<br>
Pad, BamKI, Mlul, Kpal and Fsel) were created by hybridization<br>
between	the	2	oligodeoxynucleotides,<br>
5GCTTG°CGGXGTTtACACGCGTG3AT°CTTAA.TT.GTO3ACTCTAGAG 3' (SEQ.ID41) and<br>
5-' AATTC1CTAGAGTCGACTTAATTAAGGAT°CACG°CT	3' (SEQ.ID42;. To<br>
do this, 5 µg of each of the 2 oligodeoxynucleotides were mixed and held at 85°C for 1 min., followed by a progressive reduction of the temperature to 80°C for 5 mir., then a slow reduction of the temperature to 60°C in a water bath, and finally a rapid reduction of the temperature to the ambient temperature outside the water bath.<br>
4.2.	Obtaining pMRT1119<br>
The binary plasmid pMRT1119 (6016 bp) results from the introduction of the sequence carrying the unique restriction sites into the Hindlll and EcoRI sites of the pMRTlllS plasmid.<br>
To do this, the pMRTlllS plasmid DNA (5 pg) was doubly digested by Hindlll and EcoRI, purified with the aid of the «QIAquick PCR Purification» kit, and recovered in 50 µl of H2O.<br>
The ligation by PCR reaction was carried out with 75 ng cf digested pMRTlllS plasmid and 500 ng of fragments carrying the unique restriction sites (described in 4.1.) in a reaction medium of 10 µl in the presence of 1 µl cf T4 DNA ligase x 10 buffer (New England Biolabs) and 400 units of T4 DNA ligase enzyme (New England Biolabs). It is made up of 180 cycles consisting of 2 stages, one cf 10°C for 30 sec. and the other of 30°C for 30 sec. in the «GeneAmp PCR System 9700» thermocycler.<br>
The bacteria, Escherichia coli DH5α, which had previously been made competent, were transformed (Hanahan, 1983). The plasnid DNA of the clones obtained, selected on LB medium supplemented with kanamycin (50 mg/1), was extracted by the alkaline lysis method (Eirnboim and Doly, 1979) and verified<br>
by enzymatic digestions and by sequencing. The resultant plasmid selected was called pMRT1119 (Figure 4). Its complete sequence EQ.ID04 is giver, in the sequence listing.<br>
5. Synthesis of pMRT112i<br>
The plasmid pMRT1121 (6017 bp) results from the insertion into pMRT1119 of a unique restriction site, BspEI, between the nopaline synthase promoter of Agrobacterium tumefaciens (Pnos) and the mutant nptll gene.<br>
The BspEI site was inserted by the assembly by splicing overlap extension of two fragments obtained by PCR amplification.<br>
5.1.	Synthesis of the fragment carrying "part of nptll<br>
and BspEI site"<br>
The 892 bp fragment carrying "part of nptll and BspEI site" was amplified by PCR from 5 ng of pBin!9 matrix DNA with the aid of 20 pmoles of each of the 2 oligodeoxynucleotides, 5' GGAATCGAAATCTCGTGATGGCAGG 3' (SEQ.ID39) and 5' TAATCTGCAT°CGGATCTGGATCGTTTCGC 3' (SEQ.ID43) carrying the BspEI site, in the presence of 200 µM of each of the dNTPs, 60 mM Iris-S04 pH 9.1, 18 mM (NH4}2 S04, 1.8 mM MgS04 and 2 U of eLONGase enzyme (GI3CO BRL Life Technologies) in a final reaction medium of 50 µl.  This reaction was repeated. The PCR amplification reaction was carried out in a "GeneAmp PCR System 9700" thermocycler. After denaturation at 94°C for 3 min., the DNA was subjected to 15 cycles each made up of the stages of denaturation at 94°C for 45 sec., hybridization at 55°C for 45 sec. and elongation at 68°C for 1 min. Then, in the last cycle, the elongation was continued at 68°C for 3 min. The PCR product so obtained was isolated by 2% agarose gel electrophoresis in TBE buffer and purified with the aid of the <v gel extraction kit. the dna was recovered in of h></v>
5.2.	Synthesis of the fragment carrying "BspEI site<br>
MCS"<br>
The fragment (250 bp) carrying "BspEI site - MCS" v.-as amplified by PCR from 5 ng of pMRTliiS matrix DNA with the aid of 20 pmoles of each of the 2 oligodeoxynucleotides, 5' GCTCGAGTGAATTCAAAGCTTCGGTaccGTTGAAGGAG°CACTCAG°CG 3 ' ( SEQ . ID36 ) and 5' ACGAT°CAGAT°CGGATGCAGATTATTTGG 3' '3EQ.ID44) carrying the BspEI site. The conditions of PCR amplification and treatment cf the PCR fragment obtained are the same as described in 5.1.<br>
5.3. Synthesis of the fragment carrying "part nptll -MCS"<br>
The 1117 bp fragment carrying "part nptll - MCS" results from the assembly of the 2 PCR fragments, "part nptll - BspEI site" and "BspEI site - MCS", by splicing overlap extension.<br>
To do this, a PCR amplification was carried out from 7.5 µl of each of the PCR products treated, corresponding to "part nptll - BspEI site" and "BspEI site - MCS", with the aid of 20 pmoles of each of the 2 oligodeoxynucleotidesr 5' GGAATCGAAATCTCGTGATGGCAGG 3' (SEQ.ID39) and 5T GCTCGAGTGAATTCAAAGCTTCGGTaccGTTGAAGGAG°CACTCAG°CG 3 ' (SEQ. ID38 ) , in the presence of 200 µM of each of the dNTPs, 6Q mM Tris-SO4 pH 9.1, 18 mM (NH4J2 S04, 1.8 mM MgSO4 and 2 U of eLONGase enzyme (GIBCO BRL Life Technologies) in a final reaction medium of 50 µl.  This reaction was repeated 4 times. The PCR amplification reaction was carried out in a "Gene Amp PCR System 9700" thermocycler. After denaturation at 94°C for 3 min., the DNA was subjected to 15 cycles each made up of the stages of denaturation at 94=C for 45 sec., hybridization at 62°C for 45 sec. and elongation at 68°C for 1 min. Then, in the last cycle, the elongation was continued at 68°C for 7 min. The PCR product so obtained was isolated by 1% agarose gel electrophoresis in TBE buffer and purified with the aid of the "QIAquick Gel Extraction" kit. The DNA was recovered in 50 µl of H2O, digested by Bsu36I and PstI, and subjected to 2% agarose jel electroohoresis in TEB buffer. The 315 bo DNA fraament was<br><br>
isolated and purified with  the aid  of  the  "QIAquick  Gel fWfraction" kit.<br>
5.4. Obtaining pMRT1121<br>
The 315 bp DNA fragment Bsu36I - PstI, which carries the BspEI site, was ligated to the Bsu36I and PstI sites of th~ binary plasmid pMRT1119.<br>
The plasmid pMRT1119 was previously digested by 5su36I and PstI, subjected to 1% agarose gel electrophoresis in TEB buffer and purified with the aid of the «QIAquick Gel E-Lxtraction» kit. The DNA was recovered in 50 µl cf H2G, dephosphorylated with 50 units of calf intestinal alkaline phosphatase (New England Biolabs) in a final reaction medium of 120 µl in the presence of 12 µl of 3 x 10 buffer (New England Biolabs) at 37°C for 1 hour, and purified with the aid of the "QIAquick PCR Purification" kit.<br>
The ligation by PCR reaction was carriea out with 100 ng of digested dephosphorylated pMRT1119 plasmid and 30 ng of digested DNA fragments (315 bp) in a reaction medium of 10 µl in the presence of 1 µl of T4 DNA ligase x 10 buffer (New England Biolabs) and 400 units of T4 DNA ligase enzyme (New England Biolabs) . It is made up of 180 cycles consisting of 2 stages, one of 10°C for 30 sec. and the other of 30°C for 30 sec., in a "Gene Amp PCR System 9700" thermocycler. The bacteria,<br>
Escherichia coli DH5α or SCS110 (deficient in Dam and Dem methylases), which had previously been made competent, were transformed (Hanahan, 1983). The plasmid DNA of the clones obtained, selected on LB medium supplemented with kanamycin (50 mg/1), was extracted by the alkaline lysis method and verified by enzymatic digestions and sequencing. The resultant plasmid was called pMRT1121 (Figure 5) . Its complete sequence SEQ.ID05 is given in the sequence listing.<br>
6. Synthesis of pMRTI122<br>
The plasmid pMRT1122 (6016 bp) results from the introduction into pMRT1119 of a point mutation in the mutant<br>
 gene in order to restore the Bglll restriction site and thus lead to the obtaining of the wild-type nptll gene.<br>
To do this, the Bglll restriction site was restored in the mutant nptll gene of pMRTlllS by directed mutagenesis in accordance with the Michael procedure (1994).<br>
The cligodeoxynucleotide 5' ATGGGTCACGAC2AGATCTTCG°CGTCGGG 3' (SEQ.ID45) was previously phcsphorylated by subjecting 600 pmoles of the oliaodeoxynucleotide to the action of 90 units of T4 kinise (Arnersham) in the presence of 30 µl of T4 kinase x 10 buffer (Arnersham) and 3 µl Of 100 mM ATP in a final reaction medium of 300 µl at 37 °C for 30 min. The oliaodeoxynucleotide so treated was purified with the aid of the «Qiaquick Removal Nucleotide» (QIAGEN) kit in accordance with the supplier's recommendations .<br>
The ligation by PCR was then carried out from 10 ng of pBIN19 matrix with the aid of 200 pmoles of each of the oligodeoxynucleotides, 5' GGAATCGAAATCTCGTGATGGCAGG 3' (SEQ.ID39), 5' ATGGGTCACGACGAGATCTTCG°CGTCGGG 3' (SEQ.ID45), which was phosphorylated, including the Bglll restriction site, and 5' ATTATTGCGCGTTCAAAAGTCG°C 31 (SEQ.ID40), in the presence of 400 µM of each of the dNTPs, 10 µl of Taq DNA ligase x 10 buffer (New England BioLabs), 5 units of Vent DNA Polymerase (New England BioLabs) and 40 units of Taq DNA ligase (New England BioLabs) in a final reaction medium of 100 µl.  The PCR ligation reaction was carried out in the « Gene Amp PCR System 9700» thermocycler by carrying out the following three su°Cessive phases: the first phase consisted of a cycle made up of the stages of denaturation at 94 =C for 5 min., hybridisation at 50°C for 1 min. and elongation at 65°C for 4 min.; the second phase was made up of 28 cycles each comprising the stages of denaturation at 94 °C for 30 sec., hybridization at 50°C for 1 min. and elongation at 65 °C f"Nr 4 min. ; and, finally, the last phase consisted of a cycle made up of the stages of denaturation at 94 °C for 30 sec., hybridization at 50°C for 1 min. and elongation at 65 °C for 15 min. The PCR reaction medium was then subjected to 0.8% agarose gel electrophoresis in<br>
TBE buffer and purified with the aid of the «QIAquick Gel raction&gt;&gt; kit. The DNA fragments (1023 bp) so treated were hydrolysed by Ncol and PstI and subjected to 2% agarose gel electrophoresis. The 383 bp DNA fragments were isolated, purified with the aid of the «QIAquick Gel Extracticn» kit, recovered in 50 µl of H2O and ligated to the Ncol and PstI sites of the pMRT1119 plasmid. To do this, the pMRT1119 plasmid DNA was digested by Ncol and PstI, purified on 0.8% agarose gel. The fragment corresponding to the plasmid was isolated , purified with the aid of the «QIAquick Gel Extract ion» kit, then dephosphorylated with 50 units of calf intestinal alkaline phcsphatase (New England Biolabs) in a final reaction medium of 120 µl in the presence of 12 µl of 3 x 10 buffer (New England Biolabs) at 37°C for 1 hour, and, finally, recovered in 50 µl of H2O.<br>
The ligation by PCR was carried out with 50 ng of digested dephosphorylated pMRT1119 plasmid and all the DNA fragments digested and treated in a reaction medium of 10 µl in the presence of 1 µl of T4 DNA ligase x 10 buffer (New England Biolabs) and 400 units of T4 DNA ligase enzyme (New England Biolabs) . It is made up of 180 cycles consisting of 2 stages, one of 10°C for 30 sec. and the other of 30°C for 30 sec., in the «GeneAmp PCR System 9700» thermocycler.<br>
The bacteria, Escherichia coli DH5α, which had previously been made competent, were transformed (Hanahan, 1983) . The plasmid DNA of the clones obtained, selected on LB medium supplemented with kanamycin (50 mg/1), was extracted by the alkaline lysis method (Birnboim and Doly, 1979) and screened by digestion by Bglll. The plasmid DNA of the clone selected was verified by enzymatic digestions and by sequencing. The resultant plasmid selected was called pMRT1122 (6016 bp). It is represented in Figure 6 and its complete sequence SEQ.ID06 is given in the sequence listing.<br>
7. Synthesis of pMRTI155<br>
The plasmid pMRT1155 (6017 rp) differs from pMRTiI22 by the presence of the BspEI site.<br>
To do this, the 315 bp insert f ragme:" :; containing the BspEI site was obtained by digestion of pMRT112I by Esu36I and FstI, 1.2% agarose gel electrophoresis in TEB buffer, purification with the aid of the "QIAquick Gel Extraction" kit and recovery in 50 µl of H2O.<br>
The pMRT1122 vector f ragme: t was obtained by digestion of pMRT112 by Bsu36l and PstI, 1.2% agarose gel electrcphcresis in TEB buffer, purification with the aid of the "QIAquick Gel Extraction" kit and recovery in 50 µl of H2O. After that, digested pMRT1122 was dephosphorylated with 50 units of calf intestinal alkaline phosphatase (New England Biclabs) in a final reaction medium of 120 µl in the presence of 12 µl of 3 x 10 buffer (New England Biolabs) at 37 °C for 1 hour, and purified with the aid of the "QIAquick PCR Purification" kit.<br>
The ligation by PCR reaction was carried cut with 100 ng of dephosphorylated digested pMRT1122 plasmid and 50 ng of digested DNA fragments (315 bp) .in a reaction medium of 10 µl in the presence of 1 µl of T4 DNA ligase x 10 buffer (New England Eiolabs) and 400 units of T4 DNA ligase enzyme (New England Biolabs}. It is made up of 180 cycles consisting of 2 stages, one of 10°C for 30 sec. and the other of 30°C for 30 sec. in a "GeneAmp PCR System 9700" thermocycler. The bacteria,<br>
Escherichia coli DH5α or SCS1IO (deficient in Dam and Dem methylases), which had previously been made competent, were transformed (Hanahan, 1983) . The plasmid DNA of the clones obtained, selected on LB medium supplemented with kanamycin (50 mg/1), was extracted by the alkaline lysis method and verified by enzymatic digestion? and sequencing. The resultant plasmid was called pMRTI155 (Figure 7). Its complete sequence SEQ.ID07 is given in the sequence listing.<br>
8. Obtaining basic binary plssrnids comprising the .sequence "eP35S - T35S"<br>
8.1. Obtaining pMRT:.205<br>
The piasmid pMRTI205 (7503 bp) includes the expression bcUfsette of the mutant nptll gene and a "double 35S promoter -35S terminator" sequence (eP35S - T35S) for the cloning of genes of interest. It results from the cloning of the eP35S promoter of the cauliflower mosaic virus (CaMV) into pMP.T1175. The CaMV eP35S promoter corresponds to a duplication of the transcription-activating sequences situated upstream of the TATA element of the 35S promoter (Kay et al., 1987). As for the piasmid pMRT1175 (6767 bp), this results from the cloning of T35S of the cauliflower mosaic virus into pMRT1121. The transcription-terminating sequence, CaMV T35S, corresponds to the non-coding region in 3' of the circular double-stranded DNA sequence of the cauliflower mosaic virus producing the 35S transcript (Franck et al., 1980).<br>
8.1.1. Obtaining pMRT1175<br>
The piasmid pMRT1175 results from the cloning of the EcoRI - Xhol insert DNA fragments corresponding to the T35S into the EcoRI and Xhol sites of the vector pMRT1121 produced from Escherichia coli strain SCS11C.<br>
The DMRT1121 vector fragment was obtained by digestion of 7 ug of pMRT1121 by EcoRI and Xhol, purification with the aid of the "QIAquick PCR Purification" kit and recovery in 50 ui of H2O. After that, digested pMRTl!21 was dephosphorylated with 50 units of calf intestinal alkaline phosphatase (New England Eiolabs) in a final reaction medium of 120 ui in the presence cf 12 µl of 3 x 10 buffer (New England Biolabs) at 37°C for 1 hour, and purified with the aid of the "QIAquick PCR Purification" kit and recovered in 50 µl of H2O.<br>
The EcoRI - Xhol insert DNA fragments (757 bp) corresponding to the T35S (750 bp) were obtained by digestion by EcoRI and Xhol of the piasmid pJIT163, which derives from the piasmid pJIT60 (Guerineau and Mullineaux, 1993). They were then subjected to 1% agarose gel electrophoresis in TEB buffer, purified with the aid of the «QIAquick Gel Extraction» kit and recovered in 30 µl of H2O.<br>
The ligation by PCR reaction was carried out as describee earlier in 1, with 30 ng of dephosphorylated digested pMRT1121 plasmid and 80 ng of digested insert DNA fragments. The<br>
bacteria, Esc'nerichia coii DH5α, which had previously been made competent, were transformed. The plasmid DNA of the clones obtained, selected on LB medium supplemented with kanamycin (50 mg/i), was extracted by the alkaline lysis method and verified by enzymatic digestions and sequencing. The resultant plasmid was called pMRT1175 (6767 bp) . It is represented in Figure 8 and its complete sequence SEQ.ID08 is given in the seauence listina.<br>
8.1.2. Obtaining pMKT1205<br>
The plasmid pMRT1205 results from the cloning of the Kpnl - Hindlll insert DNA fragments corresponding to eP35S into the Kpnl and Kindlll sites of the vector pMRT1175.<br>
The pMRT1175 vector fragment was obtained by digestion of 4.1 ug of pMRT1175 by Kpnl and HindiII, purification with the aid of the «Concert Rapid PCR Purification System» kit and recovery in 100 µl of H2O. After that, digested pMRT1175 was dephosphorylated with 50 units of calf intestinal alkaline phosphatase (New England Bioiabs) in a final reaction medium of 120 µl in the presence of 12 µl of 3 x 10 buffer (New England Eiolabs) at 37°C for 1 hour, purified with the aid of the «Concert Rapid PCR Purification System» kit, and recovered in 50 µl of H?0.<br>
The Kpn! - Hindlll insert DNA fragments (743 bp) corresponding to eP353 (735 bp) were obtained by digestion by Kpnl and Kind! 11 of the plasmid pJIT163, which derives from the plasmid pJITSO (Guerineau and Mullineaux, 1993). They were then subjected to  agarose gel electrophoresis in TEB buffer, purified with tne aid of the "QIAquick Gel Extraction" kit and recovered ;.n 30 µl of H2O.<br>
The ligation by PCR reaction was carried out as described earlier in 7, with 48 ng of dephosphorylated digested<br>
pMRTI175 plasmid and 30 ng of digested insert DNA fragments. The bffcteris, Eschsrichia coli DH5α, which had previously been mace competent, were transformed. The plasmid DNA of the clones obtained, selected on LB medium supplemented with kanamycin (50 mg/1), was extracted by the alkaline lysis method and verified by enzymatic digestions and sequencing. The resultant plasmid was called pMRT1205 (7503 bp) . It is represented in Figure 19 and its complete sequence SEQ.ID19 is given in the sequence listing.<br>
8.2. Obtaining pMRT1203<br>
The plasmid pMRT12G3 (7503 bp) includes the expression cassette cf the wild-type nptll gene and a "double 35S promoter - 35S terminator" sequence (eP35S - T35S) for the cloning of genes of interest. It results from the cloning of the eP35S promoter of the cauliflower mosaic virus into pMRT1176. As for the plasmid pMRT1176 (6767 bp), this results from the cloning of T35S of the cauliflower mosaic virus into pMRT1155. The eP35S promoter and the T35S terminator are described in 8.1.<br>
8.2.1.	Obtaining pMRT1176<br>
The plasmid pHRT1176 results from the cloning of the EcoRI - Xhcl Insert DNA fragments (757 bp) corresponding to the T35S (750 bp) into the EcoRI and Xhol sites of the vector pMRTII55 produced from Escherichia coli strain SCS1IO.<br>
The plasmid pMRT1176 was obtained in accordance with the methodologies described in 8.1.1., except that 6.3 µg of the vector pMRTI155, which constitutes the cloning vector, were digested and treated. The plasmid DMRT1176 (6767 bp) is represented in Figure 9. Its complete sequence SEQ.ID09 is given in the sequence listing.<br>
8.2.2.	Obtaining pMRT1203<br>
The plasmid pMRT1203 results from the cloning of the Kpnl - HindiII insert DNA fragments (743 bp) corresponding to eP35S (735 bp) into the Kpnl and Hindlll sites of the vector PMRT1176.<br>
The plasmid pMRT1203 was obtained in accordance with the methodologies described in 8.1.2., except that 3.9 ug cf the<br>
vector pMR'iiI76, which constitutes the cloning vector, were digested and treated. The plasmid pMRT1203 (7503 bp) is represented in Figure 17. Its complete sequence SEQ.ID1" is given in the sequence listing.<br>
9. Obtaining binary plasmids comprising the sequence "eP35S - uidA - T35S"<br>
These vectors were employed in transformation of tobacco in order to evaluate them, and determine the incidence ci the mutation of the nptll gene.<br>
9.1. Obtaining pMRT1206<br>
The plasmid pMRT1206 (S390 bp) includes the expression cassette of the mutant nptll gene and the expression cassette of the uidA gene (gus) . It results from the cloning of the eP35S promoter of the cauliflower mosaic virus into pMRT1196. As for the plasmid pMRT1196 (8654 bp), this results from the cloning of the uidA gene (Jefferson RA et al., 1986) into pMRT1175.<br>
9.1.1. Obtaining pMRTH96<br>
The plasmid pMRTH96 results from the cloning of the insert DNA fragments (SmaJ - "Sad -f T4 DMA polymerase") , corresponding to the uidA gene, into the "Xbal + Klenow" site of the vector pMRT1175.<br>
The pMRT1175 vector fragment was obtained by digestion of 10 µg of pMRT1175 by Xbal, purified with the aid of the "QIAquick PCR Purification" kit, recovered in 50 µl of H2O and subjected to the action of 20 units of the Klencw fragment (New England Biolabs) in the presence of 12 µl of 500 mM Tris-HCl pH 7,5, 500 mM MgCl2, 6 µl of 1M dithiothreitol, 6 µl of each of the 10 mM dNTPs in a reaction volume of 120 µl at 37°C for 30 min. It was then purified with the did of the "QIAquick PCR Purification" kit and recovered in 50 µl of H2O. The plasmid pMPT]175 so digested and treated was dephosphorylated with 50 units of calf intestinal alkaline phosphatase (New England Biolabs) in a final reaction medium of 120 µl in the presence of 12 µl r-f 3 x 10 buffer (New England Biolebs) at 3~°C for 1 hour,<br>
and purified with the aid of the "QIAquick PCR Purification" kit and recovered in 50 µl of H2O.<br>
The insert DNA fragments (2 pg) corresponding to the uidA gens (1.8 kbp) were obtained by digestion of pB1221 (marketed by Clontech) by SacI, purification with the aid of the "QIAquick PCR Purification" kit and recovery in 50 µl of H2O. After that, digested pB!221 was subjected to the action of 6 units of T4 DNA polymerase (New England Eiolabs) in a reaction medium of 120 µl in the presence of 12 µl of T4 DNA polymerase x 10 buffer, 4 µl of 10 mM dWTPs and 5 µl of BSA 1 mg/mi. The reaction was carried out at- 37 °C for 30 Min. The piasmid pBI221 so treated was purified with the aid of the "QIAquick PCR Purification" kit and recovered in 50 µl of H2O. Finally, pBI221 so treated was digested by Smal. The «[SacI + T4 DNA polymerase] - Smal» fragment (1882 bp) was isolated by 0.8% agarose gel electrophoresis, purified with the aid of the "QIAquick Gel Extraction" kit and recovered in 50 µl of H2O.<br>
The ligation by PCR reaction was carried out in accordance with the methodologies described in 7, with 15 ng of dephosphorylated digested pMRT1175 piasmid and 100 ng of digested insert DNA fragments. The bacteria, Escherichia coli<br>
DH5α, which had previously been made competent, were transformed. The piasmid DNA of the clones obtained, selected on LB medium supplemented with kanamycin (50 mg/1), was extracted by the alkaline lysis method and verified by enzymatic digestions and sequencing. The resultant piasmid was called pMRT1196 (S654 bp) . It is represented in Figure 14 and its complete sequence 5EQ.ID14 is given in the sequence listing.<br>
9.1.2. Obtaining pMRT1206<br>
The piasmid pMRT1206 (9390 bp) results from the cloning of the Kpnl - Hindlll insert DNA fragments (743 bp) , corresponding to eF35S (735 bp) , into the Kpnl and Hindlll sites of the vector pMRT1136.<br>
The piasmid pMRT1206 was obtained in accordance with the methodologies described in 8.1.2., except that 2 µg of the vector pMRT1196, which constitutes the cloning vector, were<br>
and treated. It is represented in Figure 24 and its complete sequence SEC-ID24 is given in the sequence listing.<br>
The plasmid pMRT1206 was introduced into Agrobacterium tumefacien strain LBA4404 by direct transformation in accordance with the procedure of Holsters et al. (1978).<br>
9.2. Obtaining pMRT1204<br>
The plasmid pMRT1204 (9390 bp) includes the expression cassette of the wild-type nptll gene and the expression cassette of the uidA gene. It results from the cloning of the eP35S promoter of the cauliflower mosaic virus into pMRT1192. As for the plasmid pMRT1192 (8654 bp), this results from the cloning of the uidA gene into pMRT1176.<br>
9.2.1.	Obtaining pMnTHS2<br>
The plasmid pMRT1192 results from the cloning of the insert DNA fragments (Smal - "Sad + T4 DNA polymerase") , corresponding to the uidA gene, into the "Xbal + Kienow" site of the vector DMRT1176.<br>
The plasmid pMRT1192 (8654 bp) was obtained in accordance with the methodologies described in 9.1.2., except that the cloning vector is pMRT1176. It is represented in Figure 11 and its sequence SEQ.ID11 is given in the sequence listing.<br>
9.2.2.	Obtaining pMRT1204<br>
The plasmid pMRT1204 (9390 bp) results from the cloning of the Kpnl - Hindlll insert DNA fragments (743 bp), corresponding to e?35S (735 bp), into the Kpnl and Hindlll sites of the vector pMRT1192.<br>
The plasmid pMRT1204 was obtained in accordance with the methodologies described in 8.1.2., except that 2 yg of the vector pMRT1192, which constitutes the cloning vector, vere digested and treated. It is represented in Figure 18 and its complete sequence SEQ.IDI8 is given in the sequence listing.<br>
The plasmid pMRT1204" was introduced into Agrobacterium umefaciens strain LBA4404 by direct transformation in accordance with the procedure of Holsters et al. (1976).<br>
10. Obtaining binary plasmids containing the bar gene<br>
The final plasmids were employed in transformation' of maize and/or tobacco in order to evaluate them.<br>
10.1. Creation of pHRT1210<br>
The plasmid pMRT1210 (10003 bp) includes the expression cassette of the bar gene and the expression cassette of the uidA gene. It results from the cloning of the expression cassette of the uidA gene into pMRT1195. As for the plasmid pMP.T1195 (6865 bp), this results from the cloning of the sequence "promoter followed by intron 1 of the rice actin gene" (McElrcy I et al., 1991} into pMRT1191. The plasmid DMRT1191 (4805 bp) derives from pMRT1119.<br>
10.1.1. Obtaining pMRT1191<br>
The plasmid pMRT1191 (4805 bp) differs from pMRT1119 by the deletion of the nos promoter and mutant nptll gene sequences.<br>
It was obtained by digestion by Bspl20l of 10 µg of pMRT1119, followed by purification with the aid of the "QIAquick PCR Purification" kit and recovery in 50 µl of H2O. After that, digested pMRTH19 and [sic] was hydrolysed by Kpnl, purified with the aid of the "QIAquick PCR Purification" kit, recovered in 50 µl of H2O, subjected to the action of 6 units of T4 DNA pclymerase (New England Biolabs) in a reaction medium of 120 µl in the presence of 12 µl of T4 DNA polymerase x 10 buffer, 4 µl of 10 mM dNTPs and 6 µl of BSA at 1 mg/ml. The reaction was carried out at 31°C for 30 min. The vector fragment was then isolated by 0.8% agarose gel electrophoresis, purified with the aid of the "QIAquick Gel Extraction" kit and recovered in 50 µl of H,0.<br>
Thirty nancgrams cf the plasmid so treated were re 1 doited by ligation by ?CR reaction (procedure described in<br>
The resultant piasmid was called pMRTI191. It is represented in Figure 10 and its complete sequence SEQ.ID10 is given in the sequence listing.<br>
10.1.2. Obtaining p.MRT1201<br>
Tha piasmid pMRT1201 (19- -3 bp) differs from pMRT1191 by the insertion of the expression cassette of the uidA gene isolated from the piasmid pUC19-IA-uidA-Tnos which comprises the uidA expression cassette inserted into the EcoRI and HirdIII sites of pUC19 (marketed by Mew England Biolabs) . This piasmid results from su°Cessive clonings. The fragment corresponding to the sequence «uidA gene - nos terminator» was isolated by su°Cessive digestions by «EcoRI followed by the<br>
action of the Klenow fragment» and Smal from pBI221 A SacI ( = PBI221 (Clontech) whose Seal site was eliminated by SacI digestion followed by the action of T4 DNA polymerase, then ligation). The isolated and purified fragment was inserted into the «SmaI - [SphI + T4 DNA polymerase]» sites of pUC19 (Clontech) in order to obtain the resultant piasmid pUC.l 9-uidA-Tnos. After that, the EcoRI site recreated on the 3' side of Tnos was eliminated by replacement of the EstBI-Hindlll fragment of pUC19-uida-Tnos with the BstBI-Hindlll fragment obtained by digestion of the fragment amplified by FCR (1054 bp) from the pUC19-uidA-Tnos matrix with the aid of the 2 oligodeoxynuclec-ides 5' AGGCATTGGTTTCGAAICG 3' (SEQ.ID46) containing the BstEI site and 5' TACG°CAAGCTTGGCAATT°C 3' (SEQ.ID47) . The resultant piasmid was called pUC19-uidA-Tnos<br>
A EcoRI. Then, in order to introduce a Ncol site at the level of the initiator codon of the uidA gene, a 440 bp fragment was<br>
amplified by PCR from the pUC19-uidA-Tnos4EcoRI matrix with the aid of the 2 oligodeoxynucleotides 5' AATAC°CGGGaccATGGT°CGT°CTGTAG 3' (SEQ.ID48) containing the Ncol and Smal sites and 5' ATAGTCTG°CAGTTCAGTTCGTTG 3' (SEQ.ID49) situated downstream of SnaBI. The PCR fragment digested by Smal<br>
and SnaBI was then inserted into pUC19-uiciA-Tnos A EcoRI in order to replace the existing Smal-SnaBI fragment. The resultant<br>
piasniid was called pUC19-NcoI-uidA-Tnos. After that, the Phmwg tooter [High Molecular Weight Glutenin, Anderson et al. (1989;], carried by the EcoRI - Smal fragment, was introduced into the EcoRI and Smal sites of pUC19-NcoI-uidA-Tnos in order to produce the plasmid pUC19-Phmwg-uidA-Tnos. Finally, the intron 1 of the rice actin gene (IA, EcoRV - Smal fragment) comes from the plasmid pActl-F6 (HcElroy D et al., 1991) which 15 itself modified by the addition of restriction sites notably on both sides of the EcoRV-Smal fragment containing IA. The Smal - Ncol fragment carrying IA was isolated from pACtl-F6 modified and inserted into the Smal and Ncol sites of pUC19-Phmwg-uidA-Tnos in oraer to produce the plasmid pUC19-Phmwg-IA-uidA-Tnos. The techniques used are the ones already described in this patent.<br>
The pMRT1191 vector fragment was obtained by digestion of 2 µg of pMRTI191 by EcoRI, purification with the aid of the «Concert Rapid PCR Purification System» kit, followed by the action of 20 units of the Klenow fragment (New England Biolabs) in the presence of 12 µl of 500 m!vl Tris-HCl pH 7.5, 500 mM MgCl2, 6 µl of 1M dithiothreitol, 6 µl of each of the dNTPs at 10 mM in a reaction volume of 120 µl at 37°C for 30 min., purification with the aid of the «Concert Rapid PCR Purification System» kit, and recovery in 50 µl of H2O. After that, digested and treated pMRT1191 was dephosphorylated with 50 units of calf intestinal alkaline phosphatase (New England Biolabs) in a final reaction medium of 120 µl in the presence of 12 µl of 3 x 10 buffer (New England Biolabs) at 37 °C for 1 hour, purified with the aid of the «Concert Rapid PCR Purification System» kit, and recovered in 50 µl of H2O.<br>
The insert DMA fragments (2 pg) corresponding to the expression cassette of the uicA gene (3 kbp) , viz. the uidA gene placed under the control of the wheat "high molecular weight gutenin" promoter followed by the intron 1 of the rice actin gene and the nopaline synthase terminator of Acrobacterium tumefaciens, were obtained by digestion of pUC19-Phmwg-IA-uidA-Tr.os by Seal (site present in pUC19) , purification with the aid<br>
of the "QIAquick PCK purification" kit, recovery in 50 µl of H2O, Digestion by Hindlll, purification with the aid of the "QIAquick r. CR Purification" kit, recovery in 50 µl of H2O, treatment by 20 units of the Klenow fragment (New England Biolabs) in the presence of 12 µl of 500 mM Tris-HCl, pK 7.5, 500 mM MgCl2, 6 µl of 1M dithiothreitol, 6 µl of each of the 10 mM dNIPs in a reaction volume of 120 µl at 37 °C for 30 min., purification with the aid of the "QIAquick PCR Purification" kit, recovery in 50 µl of H2O and digestion by EcoP.I. After that, the insert DNA fragments were isolated by 0.8% agsrose gel electrcphoresis, purified with the aid of the "QIAquick Gel Extraction" kit, subjected to the action of the Klenow fragment as described above, and recovered in 50 µl of H2O.<br>
The ligation by PCR reaction was carried out as described earlier in 7, with 10 ng of dephosphorylated digested pMRTUSI plasmid and 100 ng of digested and treated insert DNA<br>
previously been made competent, were transformed. The plasmid LNA of the clones obtained, selected on LB medium supplemented with kanamycin (50 mg/1), was extracted by the alkaline lysis method and verified by enzymatic digestions and sequencing. The resultant plasmid was called pMRT1201 (7943 bp) . It is represented in Figure 15 and its complete sequence SEQ.ID15 is given in the sequence listing.<br>
10.1.3. Obtaining pMRT1210<br>
The plasmid pMRTI210 (10003 bp} differs from pMRT120I by tne insertion of the sequence "promoter followed by the intron 1 of the rice actin gene - bar gene" (Pact-IA-bar} isolated from p3Bl2-Pact-IA-ber-Tnos which derives from pS312 described by Komari et al. (1996). The plasmid pSB12-Pact-IA-bar-Tnos results from the cloning of the expression cassette «Pact-IA-bar-Tnos» (BspDI - «XhoI + Klenow» fragment), isolated from pDM302, into r.he Smal and BspDI sites of pSE12, whose Xhol site on the LB side was deleted. The plasmid pDM302, constructed in the boratory of Wu R., comprises the expression cassette «Fect-IA-cir-Tnos» in the plasmid pSP72 marketed by Promega.<br>
The pMRTI20i vector fragment was obtained by digestion otp-2 µg of pMRT1201 by Hpal, purification with the aid of the «Concert Rapid PCR Purification System» kit, and recovery in 98 µl of H2O. After that, digested pMRT120i was dephosphorylated with 50 units of calf intestinal alkaline phosphatase (New England Eiolabs) in a final reaction medium of 120 µl in the presence of 12 µl of 3 x 10 buffer (New England Biolabs) at 31°C for 1 hour, .purified with the aid of the «Concert Rapid PCR Purification System» kit, and recovered in 50 µl of H2O.<br>
The insert DNA fragments (2 pg) corresponding to "Pact-TA-bar" (2.1 kbp) were obtained by Xbal digestion of pBICS273, purification with the aid of the "QIAquick PCR Purification" kit, recovery in 50 µl of H2O, treatment with 20 units of the Klenow fragment (New England Biolabs) in the presence of 12 µl of 500 mM Tris-HCl, pH 7.5 - 500 mM MgCl2, 6 µl of • 1M dithiothreitol and 6 µl of each of the 10 mM dNTPs in a reaction volume of 120 µl at 37°C for 30 min. The insert DNA fragments so treated were then isolated by 0.8% agarose gel electrophoresis, purified with the aid of the "QIAquick PCR Purification" kit and recovered in 50 µl of H2O.<br>
The ligation by PCR reaction was carried out as 'described earlier in 7, with 60 ng of dephosphorylated digested pMRT1201 plasmid and 100 ng of digested and treated insert DNA<br>
fragments. The bacteria, Escherichi£ coli DHSct, which had previously been made competent, were transformed. The plasmid DNA of the clones obtained, selectee on LB medium supplemented with kanamycin (50 mg./l), was extracted by the alkaline lysis method and verified by enzymatic digestions and sequencing. The resultant plasmid was called pMRT1210. It is represented in Figure 21 and its complete sequence SEQ.ID21 is given in the sequence listing.<br>
This plasmid MRTl210 was then introduced into Aurobacterium tumefaciens strain LBA4404 (pSEl) [Komari T et al., 1996] and strain LBA4404 by direct transformation in accordance with the procedure of Holsters et al. (1978).<br>
10.1.4. Obtaining pHRT1193<br>
The binary plasraid pMRTI193 was- obtained by cloning, into the EcoRI and [Xhol 4 Klenow fragment action] sites of pMRT1119, of the «EcoRI - [Hir.dTII + Klenow fragment action]» fragment carrying the sequence «Phmwg-IA-uidA-Tnos» isolated from pUC19-Phmwg-IA-uidA-Tnos, described in 10.1.2.<br>
To do this, the pMRT1119 vector fragment was obtained by digestion of 10 µg of pMRTlilS by Xhol, purification wit': the aid of the "QIAquick PCR Purification" kit, recovery in 50 µl of H2O, treatment with 20 units of the Klenow fragment (New England Eiolabs) in the presence of 12 µl of 500 mM Tris-HCl, pH 7.5 -500 mM MgCl2, 6 µl of 1M dithiothreitol and 6 µl of each of the 10 mM dNTPs in a reaction volume of 120 µl at 37 °C for 30 min. Then purification with the aid of the "QIAquick PCR Purification" kit and digestion by EcoRI. The treated vector fragment was isolated by 0.8% agarose gel electrophoresis, purified with the aid of the "QIAquick PCR Purification" kit and recovered in 50 µl of H2O. After that, digested pMRT1119 was dephosphorylated with 50 units of calf intestinal alkaline phcsphatase (New England Biolabs) in a final reaction medium of 120 µl in the presence of 12 µl of 3 x 10 buffer (New England Biolabs) at 37°C for 1 hour, purified with the aid of the "QIAquick PCR Purification" kit and recovered in 50 µl of H2O.<br>
The insert DNA fragments (2 pg) corresponding to the expression cassette of the uidA gene (3 kbp) , viz. the uictA gene placed under the control of the wheat "high molecular weight glutenin" promoter followed by the intron 1 of the rice actin gene and the nopaline synthase terminator of Agrobacterium tumefaciens, were obtained by digestion of pUC19-Phmwg-IA-uidA-Tnos by Seal (site present in pUC19), purification with the aid of the "QIAquick PCR Purification" kit, recovery in 50 µl of H2O, digestion by Hindlll, purification with the aid of the "QIAquick PCR Purification" kit, recovery in 50 µl of H2O, treatment with .20 units of the Klenow fragment (New England Biolabs) in the presence of 12 µl of 500 mM Tris-HCl, pH 7.5, 500 mM MgCl2, 6 µl of 1M dithiothreitol and 6 µl of each of the 10 mM c'NTFs in a reaction volume of 120 µl at 3"°C for 30 min., purification with<br>
the aid of the "QIAquick I-CR Purification" kit, recovery in 4pl of H2O, and digestion by EcoRI. The insert DNA fragments were then isolated by 0.8% agarose gel electrophcresis, purified with the aid of the "QIAquick Gel Extraction" kit and recovered in 50 µl of H2O.<br>
The ligation by PCR reaction was carriec out as described earlier in 7, with 20 ng of depnosphorylated digested pMRT1119 plasmid and 100 ng of digested and treated insert DNA<br>
fragments. The bacteria, Escherichia coli DH5α, which had previously been made competent, were transformed. The plasmid DNA of the clones obtained, selected on LB medium supplemented with kanamycin (50 mg/1), was extracted by the alkaline lysis method and verified by enzymatic digestions and sequencing. The resultant plasmid was called pMRTUSS (9143 bp) . It is represented in Figure 12 and its complete sequence SEQ.ID12 is given in the sequence listing.<br>
10.2. Obtaining pMRT1195<br>
The plasmid pMRT1195 (6857 bp) differs from pMRT1191 by the insertion of the sequence "promoter followed by the intron 1 of the rice actin gene - bar gene" (Pact-IA-bar) . The vector pMRT1191 was digested by Bspl20I and Kpnl followed by the action of the enzyme T4 DNA polymerase and dephcsphorylatea but not religated was obtained as described in 10.1.1.<br>
The insert DNA fragments corresponding to "Pact-IA-bar" (2.1 kbp) were obtained as described in 10.1.3.<br>
The ligation by PCR reaction was carried out as described earlier in 7, with 20 ng of dephosphorylatea digested pMRT1191 plasmid and 100 ng of digested and treated insert DNA fragments. The bacteria, Escherichia coli DH5α, which had previously been made competent, were transformed. The plasmid DNA of the clones obtained, selected on LB medium supplemented with kanamycin (50 mg/1), was extracted by the alkaline lysis method and verified by enzymatic digestions and sequencing. The resultant plasmid was called pMRT1195. It is represented in<br>
Fioure 13 and its complete sequence SEQ. ID13 is given in the<br>
equence listing.<br>
10.3. Obtsiring pMRT1202<br>
The plasmid pMRTI202 (5614 bp} differs from pMRT1195 by the replacement of the sequence "Pact-IA" by the nopaline synthase (Pnos) promoter of Agrobacterium tumefaciens isolated from pMRT'1121 produced in Escherichia coli strain 3CS110.<br>
The pMRT1195 vector fragment (2 µg) was digested by Fstl, purified with the aid of the «Concert Rapid PCR Purification System» kit, recovered in 98 µl of H2O, treated by the action of 6 units of T4 DNA polymerase (New England Biolabs) in a reaction medium of 120 µl in the presence of 12 µl of T4 DNA polymerase x 10 buffer, 4 µl of 10 mM dNTPs and 6 µl of BSA at 1 mg/ml. The reaction was carried out at 37°C for 30 min. The vector fragment was then purified with the aid of the «Concert Rapid PCR Purification System» kit, recovered in 50 µl of H2O, digested by Hindlll, isolated by 0.8% agarose gel electrophoresis, purified with the aid of the «Concert Rapid PCR Purification System» kit, and recovered in 98 µl of H2O. After that, digested and treated pMRTH95 was dephosphorylated with 50 units of calf intestinal alkaline phospnatase (New England Biolabs) in a final reaction medium of 120 µl in the presence of 12 µl of 3 x 10 buffer (New England Biolabs) at 37°C for 1 hour, purified with the aid of the «Concert Rapid PCR Purification System* kit, and recovered in 50 µl of H2O.<br>
The insert DNA fragments corresponding to "Pnos" were obtained by BspEI digestion of pMRT1121, purification with the aid of the «Concert Rapid PCR Purification System» kit, recovery in 98 µl of H2O, treatment with 20 units of the Klenow fragment (New England Biolabs) in the presence of 12 µl of 500 mM Tris-HC1, pH 7.5 - 500 mM HgC12, 6 µl of 1M dithiothreitol and 6 µl of each of the 10 mM dNTPs in a reaction volume of 120 µl at 37 °C for 30 min., purification with the aid of the «°Cncert Rapid FCR Purification System» kit, recovery in 50 µl of H2O, and digestion by Hindlll. The insert DNA fragments (219 bp) carrying Pncs (209 bp) were then isolated by 2% aqarose gel<br>
electrophoresis, purified with the aid of the «Concert Rapid PCR pacification System» kit, and recovered in 50 µl of H2O.<br>
The ligation by PCR reaction was carried out as described earlier in 1, with 48 ng of dephosphorylated digested P-MRT1195 plasmid and 56 ng of digested and treated insert DMA<br>
fragments. The bacteria, Escherichia coli DH5α, which had previously been made competent, were transformed. The plasmid DMA of the clones obtained, selected on LE medium supplemented with kanamycin (50 mg/1), was extracted by the alkaline lysis method and verified by enzymatic digestions and sequencing. The resultant plasmid was called pMRT1202. It is represented in Figure 16 and its complete sequence SEQ.ID16 is given in the sequence listing.<br>
10.4. Obtaining pMRT1212<br>
The plasmid pMRT1212 (8987 bp) differs from pMRT1206 by the replacement of the nptll gene by the bar gene.<br>
To do this, the pMRT1206 vector fragment was obtained by digestion by Bsu36I of 2 µg of pMRT1206, purification with the aid of the «Concert Rapid PCR Purification System» kit, recovery in 50 µl of H2O and digestion by Aatll. The digestion by 5su36I and Aatll allowed deletion of the vector of the fragment corresponding to «part Pnos nptll Tnos LE». The digested vector fragment was then subjected to 1% agarose gel electrophoresis, purified with the aid of the «Concert Rapid FCR Purification System» kit and recovered in 98 µl of H2O. This digested and treated vector fragment was then dephosphorylated with 50 units of calf intestinal alkaline phosphatase (New England Biolabs) in a final reaction medium of 120 µl in the presence of 12 µl of 3 x 10 buffer (New England Biolabs) at 37°C for 1 hour, purified with the aid of the «Concert Rapid PCR Purification System kit and recovered in 50 µl of H2O.<br>
The insert DNA fragments corresponding to «par Pnos bar Tnos LB» (1.2 kbp) were obtained by digestion by Bsu36I of 2 µg of pMRT!202, purification with the aid of the Concert Rapid PCR Purification System kit, recovery in 40 µl of H2O, digestion by Aatll, isolation by 1% agarose gel electrophoresis-., purification<br>
the aid of the Concert Rapid PCR Purification System kit and recovery in 50 µl of H2O.<br>
The ligation by PCR reaction was carried out as described earlier in 7, with 100 ng of dephosphorylated digested pMRT1206 plasmid and 50 ng of digested insert DNA fragments. The<br>
bacteria, Escherichia coli DK5a, which had previously been made competent, were transformed. The plasmid DNA of the clones obtained, selected on LB medium supplemented with kanamycin ('50 mg/1), was extracted by the alkaline lysis method ana verified by enzymatic digestions and sequencing. The resultant plasmid was called pMRT1212. It is represented in Figure 22 and its complete sequence SEQ.ID22 is given in the sequence listing.<br>
11. Evaluation of the synthetic vectors produced in the bacteria The yield of plasmids obtained from the bacteria Escherichia<br>
coli strain DH5α was determined<br>
I Non-binary synthetic plasmitis<br>
Primary cultures of the recombinant bacteria possessing the plasmids pMRTHOS, pMRT1106 or pMRTli05-ori ColEl were prepared from 500 ml of each of the respective "glycerol-stocks" in 10 ml of LB medium supplemented with kanarnycin at 50 mg/1, at 31 °C for 16 hours. After that, 500 ml of each of the primary cultures was cultured in 100 ml of LB supplemented with kanamycin at 50 mg/1 at 37 °C for 23 hours. For pMRT1106, two cultures were prepared, one without addition of chloramphenicol and the other with addition of chloramphenicol at 40 mg/1 after 7 hours of culture. The cultures were then centrifuged at 5000 g for 10 min. The plasmid DNAs were extracted with the aid of the "QIAFilter Plasmid Midi kit" (QIAGEN) , in accordance with the manufacturer's recommendations, and quantitatively determined with the spectrophotometer. The quantities of plasrrids obtained were determined as 10 mg, 50.6 mg, 86.2 mg and 9.8 mg respectively for pMRTHOS, pMRT1106, pMRT1106 treated with chloramphenicol, and pMRTHOS ori ColEl (plasmid pMRTHOS in which ori ColEl was inserted in the reverse orientation to thai: of pMRTHOG) . These results show that the synthetic plasmids are replicated in Escherichia coli, the presence of the origin of replication "ori ColEl" in the right orientation (Ndel site of "ori ColEl" near "ori RK2") allows the yield to be increased by a factor of 5, whereas no effect is shown for the reverse orientation of "ori ColEi", treatment with chloramphenicol contributes an additional factor of 1.6.<br>
11.2. Synthetic binary plasmids.<br>
The synthetic binary plasmids pMRTlliS and pMRTlllS are evaluated relative to the controls pBin!9 and pBIOC4 . The plasmid pBIOC4 differs from pGA492 (An, 1986) by the deletion of virtually all the coding sequence of the pGA492 car gene and by the conversion of the Hindlll site into an EcoRI site. It was obtained by double digestion by SacI and Seal followed by the action of the enzyme T4 DNA poiymerase (New England Biolabs; and ligation of the modified plasmid (20 no) in a reaction medium of 10 µl in the presence of 1 µl of T4 DNA ligase  10 buffer<br>
16 hours. The bacteria, Escherichia coli strain DH5α, which had previously been made competent, were transformed (Hanahan, 1983). The plasmid DNA of the clones obtained, selected on tetracycline at 12 mg/1, was extracted by the alkaline lysis method (Birnboim and Doly, 1979) and analysed by enzymatic digestions. The Hindlll site of the plasmid DNA of the clone selected were then modified into an EcoRI site with the aid of a phosphorylated Hindlll-E°CRI adaptor (Stratagene). To do this, 500 ng of plasmid DNA of the clone selected were digested by Hindlll, depnosphoryiated by the enzyme calf intestinal alkaline phosphatase (Boehringer Mannheim) in accordance with the manufacturer's recommendations and coprecipitated in the presence of 1500 ng of Hindlll-EcoRI adaptor DNA, 0.1 volume of 3M sodium acetate, pH 4.8, and 2.5 volumes of absolute ethanol at -80°C for 30 min. After centrifugation at 1200C g for 30 min., the precipitated DNA was washed with 70% ethanol, dried, recovered in 8 µl of H2O, held at 65°C for 10 min., then ligated as described earlier. After inactivation of the T4 DNA ligase at 65°C for 10 min., the ligation reaction mixture was digested by EcoRI, purified by 0.8% agarose gel electrophoresis, electroeluted, precipitated with absolute ethanol, centrifuged at 12000 g for 30 min., washed with 70% ethanol, dried, then<br>
iigatsd and introduced into Escherichia °Cli strain DH5α, and treated as described earlier.<br>
Primary cultures of the recombinant bacteria possessing the plasmids mentioned above were prepared from 500 ml of each of the respective "glycerol-stocks" in 10 ml of LB medium supplemented with kanamycin at 50 mg/1, except for pBI~&gt;C4, which was selected on tetracycline at 12 mg/1, at 37°c for 16 hours. After that, 250 ml of each of the primary cultures were cultured in 100 ml of LB supplemented with the appropriate antibiotic at 37°C for 23 hours. The cultures were then centrifuged at 5000 g for 10 min. The plasmid DNAs were extracted with the aid of the "QIAFilter Plasmid Midi kit" (QIAGEN) in accordance with the manufacturer's recommendationr and quantitatively determined<br>
with the spectropnotometer. The quantities of plasmids obtained determined as  94.2 mg,  113 mg,  37.2 mg and 45.6 mg respectively for pMRTlllS, pMRT1119, pBin!9 and pBIOC4.<br>
These results show that the synthetic binary plasmids are replicated in Escherichia coli, and that the yield of pMRT1119 is increased by the factors 3 and 2.5 respectively, relative to the pBin!9 and pB10C4 controls.<br>
Furthermore, these synthetic plasmids are replicated in Agrobacterium tumefaciens strain LBA4404.<br>
12. Evaluation of the synthetic plasmids by transgenesis<br>
12.1. Transformation of tobacco<br>
12.1.1. Stable transformation of tobacco<br>
The transformation of tobacco (Nicotiana tabacum L. var. PBD6) was carried out by infecting foliar discs isolated from 6-week-old tobacco seedlings in vitro with recombinant agrobacteria in accordance with the method described by Horsch et al. (1985).<br>
All the in vitro cultures are prepared in a controlled environment chamber in which the light intensity is 200 uE.m-2.s-l, the photoperiod 16 hours light/8 hours darkness, and the temperature 25°C.<br>
Except for the initial coculture stage, the regeneration, development and rooting stages were performed on different selective media supplemented with a bacteriostat, namely augmentin at 400 mg/1, and a selective agent, namely kanamycin at 200 or 100 mg/1.<br>
The different stages and the media used are as follows:<br>
After primary culture of the agrobacteria in 5 ml of 2YT medium (bacto-tryptone 10 g/1, yeast extract 5 g/1, NaCl 5 g/1, pH 7.2) supplemented with CaCl2, with a final concentration of 6 mM, and with appropriate antibiotics, at 28°C for 48 hours, a culture is performed in 10 ml of 2YT medium, supplemented with CaCl2 and antibiotics, at 28°C for a period of one night. The culture is then centrif uged at 3000 g for 10 min. and the bacteria are resuspended in 10 ml of liquid -MS30 (M0404 at<br>
4 .A g/i, marketed by SIGMA, supplemented with sucrose, 30 g/i,<br>
The coculture is performed by placing the foliar explants cut from the leaves of the seedlings in vitro, about 1 cm2, in contact with the suspension of agrobacteria, diluted 1/10, in liquid MS30 for 20 min. The explants so treated are then rapidly dried on filter paper ar.d placed or. a solid coculture medium (CM) (MS30, Benzyl Amino Purine (BAF) 1 mg/1, Indole-3 Acetic Acid (ANA) at- 0.1 ng/1, agar at 8 g/1) for 48 hours in the controlled environment chamber.<br>
The treated explants are then placed on a solid regeneration medium (solid CM, augmentin 400 at mg/1, kanamycin at 200 mg/1) . The expiants are pricked out on the same medium after 2 weeks.<br>
After 2 weeks the shoots are pricked out on a solid development medium (M0404 at 4.4 g/i, marketed by SIGMA, supplemented with sucrose 20 g/1, pK 5.7 (liquid MS20), augmentin at 400 mg/1, kanamycin at 100 mg/1, agar at 8 g/1).<br>
After 2 weeks, the transformed seedlings are pricked out on a solid rooting medium identical to the development medium. Rooting takes 2 to 3 weeks, after which the seedlings are removed to the growth chamber in Giffy pots for 10 days (photoperiod 16 hours light/8 hours darkness, 23°C, 70% relative humidity), then placed in the greenhouse.<br>
12.1.2. Evaluation of the synthetic plasmids used in transformation of tobacco<br>
The recombinant agrobacteria (Agrobacterium tumefaciens strain LBA4404) containing the synthetic binary plasmid pMRTlllS or the control binary plasmid pBin19 were used to transform the tobacco Nicotiana tabacum L. var. PBD6.<br>
The results showed thar the number and growth of the<br>
seedlings which develop in the presence of the selective agent<br>
(kanamycin}  are similar for the two binary plasmids tested.<br>
These observations indicate that the synthetic binary plasmid is<br>
perfectly functional in transgenesis of tobacco.<br>
12.2. Transformation of maize<br>
12.2.1.	Genetic transformation of maize calli<br>
The genetic transformation of maize, whatever method is employed (electropcration, Agrobacterium, microfibres, particle gun) , generally requires the use of cells that are ur.differentiated in rapid divisions, having retained the ability to regenerate whole plants. Cells of this type make up the friable embryogenic callus (called Type II) of maize.<br>
These calli were obtained from immature embryos of genotype HI, II or (A188 x B73) in accordance with the method and on the media described by Armstrong (Maize handbook, 1994, Freeling M and Walbot V (eds), pp 665-671). The calli so obtained were multiplied and maintained by su°Cessive subcultures every 2 weeks on the starting medium.<br>
Seedlings were regenerated from these calli by modifying the hormonal and osmotic equilibrium of the cells in accordance with the method described by Vain et al. (1989). These plants were then hardened off in the greenhouse, where they, can be crossed or selfed.<br>
12.2.2.	Genetic  transformation  of  maize  with  the<br>
particle gun<br>
The production and regeneration of the cell lines needed for transformation are described in 12.2.1.<br>
The procedure which uses the particle gun for genetic transformation is described by Finer et al. (1992). The target cell3 are callus fragments with a surface area of 10 to 20 mm2. These fragments were placed on a starting medium supplemented with 0.2 M mannitol and 0.2 M scrbitol for 4 hours before bombardment.<br>
The tungsten particles (M10) and transformation plasmids were coprecipitated in accordance with the protocol described by-Klein (1987). The particles thus coated were projected towards the target cells by means of the particle gun in accordance with trie protocol of Finer et al. (1992).<br>
The bombarded calli were placed in darkness at 27°C. After 24 hours, the first subculturing took place, followed by subculturing every 2 weeks for 3 months on a starting medium to<br>
the appropriate selective agent to select only the transformed calli had been added. These calli were then grown in the presence of the selective agent in order to regenerate transformed seedlings, as described in 12.2.1. The seedlings obtained were hardened off and transferred to the greenhouse, where they can be crossed or selfed.<br>
12.2.3. Genetic transformation of maize by Agrobacterium tumefaciens<br>
The technique used is described by Ishida et al. (1996). Immature embryos 1.0 to 1.2 mm in length (9 to 14 days after pollination) were washed in the LS-inf medium, then immersed in the suspension of agrobacteria, prepared as described by Ishida et al. (1S96), vortexed for 30 sec., and incubated at ambient temperature for 5 min. The immature embryos so treated were cultured on LS-AS medium in darkness at 25°C for 3 days, then transferred to LSD 1.5 medium supplemented with phosphinothricin at 5 mg/1 and cefotaxime at 250 mg/1, in darkness at 25°C for 2 weeks, and finally placed on LSD 1.5 medium supplemented with phosphinothricin at 10 mg/1 and cefotaxime at 250 mg/1, in darkness at 25°C for 3 weeks. The Type I calli thus generated were isolated, fragmented and transferred to LSD 1.5 medium supplemented with phosphinothricin at 10 mg/1 and cefotaxime at 250 mg/1, in darkness at 25°C for 3 weeks. The Type I calli which proliferated were then isolated and placed on LSZ medium supplemented with phosphinothricin at 5 mg/1 and cefotaxime at 250 mg/1, subjected to a photoperiod of 16 hours light/8 hours darKness at 25°C for 2 to 3 weeks. The regenerated seedlings were then transferred to 2 LSF medium subjected to a photoperiod of 16 hours light/8 hours darkness at 25°C for 1 to 2 weeks, and then transferred to the growth chamber and greenhouse.<br>
13. Influence of the vector backbone on gene expression Stably transformed tobacco plants were obtained for the synthetic binary plasmids pMRT!204 bearing the wild-type nptll gene (16 plants) and pMRT1206 bearing the mutated nptll gene (12 plants), and also for the control vector pBI121 sold by Clontech which corresponds to plasmid pBIN19 bearing the wild-type nptll<br>
gene and containing the expression cassette "P35S - uidA - poiyA  (11 plants) .<br>
The influence of the synthetic vector backbone and that of the control was evaluated by ELISA analysis of protein extracts using the NPTII ELISA kit sold by 5 Prime -&gt; 3 Prime. The analysis enabled a quantification of the NPTII protein produced fin micrograms, pg) with respect to tne amount of soluble extracted protein(in milligrams, mg).<br>
The proteins were extracted in "Tris-HCl 25 mM pH7.8 ; Fhenylmethylsulfonylfiuoride 1 mM" buffer from tobacco leaves ground in liquid nitrogen. After centrifugation for 10 minutes at 10000 g and 4°C, the extracted soluble proteins contained in the supernatant were measured according to Bradford's method (A rapid and sensitive method for the detection of microgram quantities of proteins utilizing the principle of protein-dye binding. Anal. Biochem. (1976)72, 248-254).<br>
The antibodies used for ELISA quantification were rabbit polyclonal antibodies specifically recognizing the coating NPTII protein and biotinylated anti-KFTII anitbodies for the detection of the NPTII protein.<br>
The results of the analysis shown in Figure 23 indicate that :<br>
•	for the wild type nptll gene cassette and different plasmid<br>
backbones, the quantities of NPTII obtained with the synthetic<br>
binary plasmid pMRT1204 are greater by a factor of 6.7 than<br>
those obtained with the original pBIN19 binary plasmid, which<br>
demonstrates the efficiency of the synthetic vector.<br>
	for an  identical plasmid backbone,  i.e.  synthetic binary<br>
plasmids pMRT1204  and DMRT1206,  but with different nptll<br>
genes, i.e. Wild type and mutated nptll, the quantities of<br>
NPTII  obtained  in  the wild type gene differ from those<br>
obtained for the mutated nptll gene by a factor 1.2.<br>
14. Synthesis of pMRT1334.<br>
The plasmid pMRTI334 (9688 bp) was obtained by replacing the nptll expression cassette of pMRT1206 by the nptll expression cassette of pBIN19.<br>
The vector fragment derived from pMRT1206 was obtained by digestion of 10 µg of pMRT1206 with Kpnl, purified using a "Concert Rapid PCR Purification System" kit, and taken up in 50 µl of water. Then the digested pMRT1206 was subjected to the action of 6 units of T4 DNA polymerase (New England Eiolabs) in a reaction mixture of 120 µl in the presence of 12 µl of T4 lOx DNA polymerase buffer, 4 µl of 10 mM dNTP and 6 µl of 1 mg/mi BSA. The reaction was carried out- at- 37°C for 30 minutes. The digested and thus treated pMRT!206 vector was purified with a "Concert Rapid PCR Purification System" kit, taken up in 50 µl of water, then digested with Aflll. The vector fragment derived from pMRT1206 was isolated by electrophoresis on a 1% agarose gel, purified with a " Concert Rapid PCR Purification System" and taken up in 50 µl of water.<br>
The DNA insert fragment Oral - Aflll (1.5 kbp), corresponding to the nptll expression cassette was obtained by digestion of 9 µg of pBIN19 plasmid with Oral and Aflll. Then, the fragment was subjected to electrophoresis on a 1% agarose gel in TE3 buffer, purified using a "Concert Rapid PCF Purification System", and taken up in 50 µl of water.<br>
The PCR ligation reaction was carried out using 100 ng of the plamsid fragment derived from pMRT1206 ana 100 ng of the PNA insert fragment Oral - Aflll in a reaction mixture cf 20 µl in the presence of 2 µl of T4 lOx DNA ligase buffer (Epicentre Technologies), 2 µl 2.5 mM ATP and 4 units of T4 DNA iigase (Epicentre Technologies) . The reaction consists of 180 cycles each including 2 steps, the first at 10°C for 30 seconds and the second at 302C for 30 seconds in a "GeneAmp PCR System 9700" thermocycler.<br>
Previously prepared competent Escherichia coli DH5α bacteria, were transformed (Hanahan, 1983) . The plamsid DNA of trie obtained clones, selecteo on LB media supplemented with kanamycin (50 mg/1). was extracted according to the alakline lysis method and verified by enzymatic digestion and sequencing. The resulting plasmic was designated pMRT1334 (Figure 24). Its complete sequence is given under SEQ.ID50 in the sequence<br>
listing .<br>
15. Synthesis of pMRT1335<br>
The plasmid pMRT1335 (15208 bp) is a control vector and results from the insertion of the expression cassette "ep35S-gus (uidA) -polyA35S" isolated from pMRT1206 into pBIN19.<br>
13 µg of pBIN19 plasmid were digested with EcoRI, purified using a "Concert Rapid PCR Purification System", and taken up in 50 µl water. Then the previously digested pEIN19 was subjected to the action of 20 units of Klenow fragment (New England Biolabs) in the presence of 12 µl 500 mM pH7 . 5 Tris-HCl 500 mM MgC12, 6 µl of 1M dithiothreitol , 6 µl each of 10 mM dNTP in a reaction volume of 120 µl at 37 °C fcr 60 minutes. The digested pBIN19 vector was purified using a "Concert Rapid PCR Purification System", taken up in 50 µl water, then digested with KpnI. The vector fragment thus produced was isolated by electrophoresis on a 1% agarose gel, and purified using a "Concert Rapid PCR Purification System" and taken up in 50 µl of water.<br>
The DNA insert fragment corresponding to the "ep35S-gus-polyA35S" (3.5 kbp ;• expression cassette was obtained by digestion of 10 µg of plasmid pMRT1206 with Xhol, purified using a "Concert Rapid PCR Purification System", and taken up in 50 µl water, followed by the action of 20 units of Klenow fragment (New England Eiolabs) in the presence of 12 µl of 500 mM pH7 . 5 Tris-HCl 500 mM MgC12, 6 µl lixi dithicthreitoi, 6 µl each of 10 mM dNTP in a reaction volume of 120 µl at 37°C for 60 minutes, then purified using a "Concert Rapid PCR Purification System", taken up in 50 µl water, an.: finally digested with KpnI. The DNA insert fragment thus produced was isolated by electrophoresis on a 1% agaros gel in TEB buffer, and purified using a "Concert Rapid PCR Purification System", and taken up in 50 µl water.<br>
The PCR ligation reaction was carried out with 100 ng of pBTN19 treated and 100 ng of DNA insert fragment as prepared previously in a reaction volume of 20 µl in the presence of 2 µl of T4 lOx DNA ligase buffer (Epicentre Technologies}, 2 µl 2.5 mM ATP and 4 units T4 DNA ligase (Epicentre Technologies) . The<br>
Detection consists of 180 cycles each including 2 steps, the first at 10°C for 30 seconds and the second at 30°C for 30 seconds in a "GeneAmp PCR System 9700" thermocycler.<br>
Previously prepared competent  Escherichia  coli  DH5α bacteria, were transformed (Hananan, 1983). The plamsid DNA of the obtained clones,  selected on LE media supplemented with kanamycin  (50 mg/1), was extracted according to the alakline lysis method and verified by enzymatic digestion a:.d sequencing, The resulting plasmid was designated pMRT1335 (Figure 25) , Its complete  sequence  is  given  under  SEQ.ID51  in  the  sequence 1isting. 16. Synthesis of pMRT1336<br>
The plamsid pMRT1336 (9285 bp) results from the insertion into DMRT1196 of the promoter MPrll65 (610 bp) isolated from  lasmid pMRT1322 as described in PCT patent application PCT/IEOO/00370, and incorporated into the present description by reference thereto for that relevant part.<br>
10 µg of pMRT1196 plasmid were digested with Kpnl, purified using a "Concert Rapid PCF Purification System", takn up in in 50 µl water, and redigested with Hpal. The thus treated vector fragment was subjected to electrophoresis on a 1% agarose gel, and purified using a "Concert Rapid PCR Purification System" and then taken up in 50 µl water.<br>
The DNA insert fragment bearing the promoter MPrll65 (0.5 kbp) was obtained from 10 µg of pMRT1322 plasmid digested with Kpnl, purified using a "Concert Rapid PCR Purification System" and taken up in 50 µl water, and then redigested with Hpal. The DNA insert fragment thus produced was subjected to gel electrophoresis on a 1% agarose gel in TEB buffer, purified using a "Concert Rapid PCR Purification System", and taken up in 50 µl water.<br>
The PCR ligation reaction was carried out using 100 ng of vector fragment and 100 ng DNA insert fragment bearing the promoter HPrll65 as prepared above in a reaction volume of 20 µl in the presence of de 2 µl T4 lOx DNA ligase buffer (Epicentre Technologies;, 2 µl 2.5 mis ATP and 4 units T4 DNA ligase<br>
(Epicentre Technologies). The reaction consists of 180 cycles Ch including 2 steps, the first at 10°C for 30 seconds and the second at 30°C for 30 seconds in a "GeneAmp PCR System 9700" thermocycler.<br>
Previously prepared competent Escherichia coli DH5α bacteria, were transformed (Hanahan, 1983). The plamsid DMA of the obtained clones, selected on LB media supplemented with kanamycin (50 mg/1), was extracted according to the a kline lysis method and verified by enzymatic digestion and sequencing. The resulting plasmid was designated pMRT1336 (Figure 26). Its complete sequence is given in the sequence listing under SEQ.ID52.<br>
Plasmid pMRT1322 was obtained by the insertion of promoter MPrll65 into pMRTl!76. 10 µg of pMRT1176 plasmid was digested with Xbal and EcoRI, purified using a "Concert Rapid PCR Purification System" kit, taken up in 50 µl water, dephosphorylated with 50 units of calf intestine alkaline phosphatase (New England Biolabs) in a final reaction volume of 120 ui in the presence 12 µl of 3x10 buffer (New England Biolabs) at 37 °C for 1 hour, and purified using a "Concert Rapid PCR Purification System" kit, and then taken up in 50 µl water. The DNA insert fragment corresponding to promoter MPrll65 was obtained by PCR amplification from 10 ng of pMRT1240 matrix DNA usingn 20 pmoles each of two oligodesoxynucieotides, 5' AGCTCTAGAGCTG°CTGCAGCACTAGTAT°C 3'(SEQ.ID53) bearing the site Xbal and 5' CGGAATTCGG°CTCTAGGTTGTTGTGTTG 3'(SEQ.ID54) bearing the site EcoRI, using the enzyme Platinum Taq polymerase High Fidelity (GIBCO 3RL Life Technologies) and following the suppliers instructions. The PCR amplification reaction was carried out in a "GeneAmp PCR System S700" thermocycler. After denaturation at 94°C for 2 minutes, the DNA was subjected to 25 cycles, each including denaturation steps at 94 °C for 45 seconds, hybridisation at 55°C for 45 seconds and elongation at 66 =C for 45 seconds. On the final cycle the elongation was continued at 68 °C for 3 minutes. The PCR product obtained was isolated by electrophoresis on a 1.5% agarose gel in TBE buffer<br>
 purified using a "Concert Rapid PCR Purification System"<br>
kit, and then taken up in 50 µl water. The PCR product was then digested with Xbal and EcoRI, purified using a "Concert Rapid PCR Purification System" kit, and taken up in 50 µl water. The PCR ligation reaction was carried out as described previously with 80 ng of pMRT1176 vector fragment and 110 ng of DNA insert fragment bearing the MPrilcS promoter. Previously prepared<br>
competent Escherichia coli DH5α bacteria, were transformed 'jHanshan, 1983). The plamsid DNA of the obtained clones, selected on LB media supplemented with kanamycin (50 mg/1), was extracted according to the alakline lysis method and verified by enzymatic digestion and sequencing. The resulting plasmid was designated pMRT1322.<br>
Plasmid pMRT1240 was obtained by insertion of a DNA fragment into plasmid pMRT1234, by treatment with Klenow fragment (New England Biolabs) according to the supplier's instructions of the Hindlll sites and BamHl. The DNA insert fragment was the L5 promoter fragment obtained by double PstI digestion, and BamHI digestion, followed by the action of T4 DNA polymerase (New England Biolabs) according to the supplier's instructions, of plasmid pMRT1165, as described in PCT patent application PCT/IBOO/00370, and incorporated into the present description by reference thereto for that relevant part.<br>
Plasmid pMRT1234 was obtained by the insertion at the EcoRI site of pMRT1175 of a DNA insert fragment EcoRI corresponding to the cDNA coding for human lacotferrin isolated from the plamsid pHMWG-IA-PSSp-Lf, as described in previously published patent application W098/50543.<br>
17. Synthesis of pMRT1337<br>
Plasmid pMRT1337 (8289 bp) was obtained from the insertion, into pMRT1205, of the gfp gene isolated from pEINm-gfp5-ER and described in Haseloff J et Siemering KR. IS38, "The uses of green fluorescent protein in plants", in Green fluorescent protein : Strategies, applications nd protocols (Chalfie M et Kain S, eds, Wiley, pp 191-220).<br>
10 µg of plasmid pMRT1205  was digested with  EcoRI,<br>
purified using a "Concert Rapid PCR Purification System » kit, dprtfen up in 50 µl water, subjected to the action of 20 units of Klenow fragment (New England Biolabs) in the presence of 12 µl of 500 mM pH7.5 Tris-KCl 500 mM MgC12, 6 µl 1M dithiothreitoi, 6 µl each of 10 mM dNTP in a reaction volume of 120 µl at 37 °C for 60 minutes, purified using a "Concert Rapid PCR Purification System" kit, taken up in 50 µl water, and then digested with BamHI . The thus treated vector fragment was subjected to electrophoresis on 1% agsrose gel, purified using a "Concert Rapid PCR Purification System" kit and taken up in 50 µl water.<br>
The DNA insert fragment bearing the gfp gene (0.7 kbp) was obtained from 10 µg of plasmid pBINm-gfp5-ER digested with SacI, purified using a "Concert Rapid PCR Purification System" kit, taken up in 50 µl water, and subjected to the action of 6 units of T4 DN£ polymerase (New England Eiolabs) in a reaction volume of 120 µl in the presence of 12 µl T4 lOx DNA polymerase buffer, 4 µl 10 mM dNTP and 6 µl 1 mg/ml BSA. The reaction was carried out at 37°C for 30 minutes. The thus treated plasmid p5INm-gfp5-ER was then purified using a "Concert Rapid PCR Purification System" kit, taken up in 50 µl water, and digested with BamHI. The DNA insert fragment thus produced was subjected to gel electrophoresis on 1% gel agarose in TEB buffer, purified using a "Concert Rapid PCR Purification System" kit and taken up in 50 µl in water.<br>
The PCR ligation reaction was carried out with 100 ng of pMRT!205 vector fragment and 100 ng DNA insert fragment bearing the gfp gene as prepared above in a reaction volume of 20 µl in the presence of 2 µl T4 l0x DNA ligase buffer (Epicentre Technologies), 2 µl 2.5 mM ATP and 4 units T4 DNA 1-igase (Epicentre Technologies) . The reaction comprised 180 cycles each including two steps, the first at 10°C for 30 seconds, the second at 30°C for 30 seconds, in a "GeneAmp FCR System 9700" thermocycler .<br>
Previously prepared competent Escherichia coli DH5α bacteria, were transformed (Kanahan, 1983). The plamsid DNA of the obtained clones, selected on LB media supplemented with<br>
(50 mg/i), was extracted according to the aiakiine iysis method and verified by enzymatic digestion and sequencing. The resulting plasmid was designated pMRT1337 (Figure 27). Its complete sequence is given in the sequence listing under SEC.ID55. 18. Synthesis of pMRT1341<br>
The piasmid pMRT1341 (14106 bp) results from the replacement of the "ep35S-gus-polyA35S" expression cassette from p.MRT1335 by the expression cassette "ep35S-gfp-polyA35S" isolated from pMRT1337.<br>
10 µg of plasmid pMRT1335 was digested by Agel and KpnI. 1 he thus digested vector fragment was isolated by eiectrophoresis on a 0.8% agarose gel, purified using a "Concert Rapid PCR Purification System" kit and then take up in 50 µl water.<br>
The DNA insert fragment corresponding to the expression cassette "ep35S-gfp-polyA35S" (2.4 kbp) was obtained by digesting with Xhol of 10 µg plasmid pMRT1337, purifiying using a "Concert Rapid PCR Purification System" kit, taking up in 50 µl water, and the action of 20 units of Klenow fragment (New Errjland Biolabs) in the presence of 12 µl 500 mM pH7.5 Tris-HCl 500 mM MgC12, 6 µl 1M dithiothreitol, 6 µl each of 10 mM dNTP in a reaction volume of 120 µl at 37 °C for 30 minutes, then, purifying using a "Concert Rapid PCR Purification System" kit, taking up in 50 µl water, and finally digesting with KpnI. The thus preapred DNA insert fragment was isolated by eiectrophoresis on 0.8% agarose gel in TEB buffer, purified using a "Concert Rapid PCR Purification System" kit, taken up in 50 µl water, digested with Agel, purified using a "Concert Rapid PCR Purification System" and taken up again in 50 µl water.<br>
The PCR ligation reaction was carried out on 100 ng vector fragment derived from pMRT1335 et 100 ng DNA insert fragment as produced above in a reaction volume of 20 µl in the presence of 2 µl T4 lOx DNA ligase buffer (Epicentre Technologies), 2 µl 2,5 mM ATP and 4 units of T4 DNA ligase (Epicentre Technologies). The reaction comprised 180 cycles each<br>
including 2 steps, the first at 10°C for 30 seconds, the second ff 30°C for 30 seconds, in a "GeneAmp PCR System 9"700" thermocycler.<br>
Previously prepared competent Escherichia coli DH5α bacteria, were transformed (Kanahan, 1983}. The plamsid DMA of the obtained clones, selected on LB media supplemented with kanamycin (50 mg/1), was extracted according to the alakline lysis method and verified by enzymatic digestion and sequencing. The resulting plasmid was designated pMRT1341 (Figure 26). Its complete sequence is given in the sequence listing under SEQ.ID56. 19. Synthese de pMRT1342<br>
Plasmid pMRT1342 (15077 bp) results from the replacement of the expression cassette "ep35S-gus-pclyA35S" from pMRT.1335 by the expression cassette "L5-gus-polyA35S" isolated from pMRT1336.<br>
10 µg of plasmid pMRT1335 was digested with Agel and Kpnl. The thus digested vector fragment was isolated by electropnoresis on 0.6% agarcse gel, purified using a "Concert Rapid PCR Purification System" kit and taken up in 50 µl water.<br>
The DNA insert fragment corresponding to the expression casssette "L5-gus-poiyA35S" (3.4 kbp) was obtained by digestion with Xhol of 10 µg plasmid pMRT1336, purified using a "Concert Rapid PCR Purification System" kit, taken up in 50 µl water, subjecting this to the action of 20 units Klenow fragment (New England Biolabs) in the presence of 12 µl 500 mM pH7.5 Tris-HCl 500 mM MgC12, 6 µl 1M dithiothreitol, 6 µl each of 10 mM dNTP in a reaction volume of 120 µl at 37°C for 30 minutes, then, purified using a "Concert Rapid PCR Purification System" kit, and taken up in 50 µl water, and finally digested with Kpnl. The thus prepared DNA insert fragment was isolated by electrophoresis on a 0?8% agarose gel in TEB buffer, purified using a "Concert Rapid PCR Purification System" kit, taken up in 50 µl water, digested with Agel, purified using a kit as above and take nup again in 50 µl water.<br>
The PCR ligation reaction was carried out with 100 ng of<br>
vector fragment derived fron. pMRT1325 and 100 ng of DNA insert agment prepared as described above in reaction mixture of 20 µl in the presence of 2 µl T4 lOx DNA ligase buffer (Epicentre Technologies), 2 µl 2.5 mM ATP and 4 untis T4 DNA ligase (Epicentre Technologies). The reaction is comprised of 180 cycles eaca including 2 steps, the first at 10°C for 30 seconds and the second at 30°C for 30 seconds in a "GeneAmp PCR System 9700" thermocycler.<br>
Previously prepared competent. Escherichia coli DH5α bacteria, were transformed (Hanahan, 1983). The plamsid DNA of the obtained clones, selected on LB media supplemented with kanamycin (50 mg/1), was extracted according to the alakline lysis method and verified by enzymatic digestion and sequencing. The resulting plasmid was designated pMRT1342 (Figure 29). Its complete sequence is given in the sequence listing under SEQ.ID57. 20. Production of recombinant agrobacterie<br>
The binary plamsids were transferred into the Ag.obacterium tumefaciens LBA4404 strain according to the technique described by Holsters et al. [Holsters M., Dewaele D., Depicker A., Messenf E., Van Montagu M. et Schell J. (1978). Trahsfection and transformation of Agrobacterium tumefaciens. Mel. Gen. Genet. 136, 181-187.]. The plasmid DNA of the obtained clones, selected on LB media supplemented with rifampicine (50 mg/1), was extracted according to the alkaline lysis method, modified by the addition of lysozyme (25 mg/mi) in the cell lesuspension buffer. The plasmid DNA obtained was analysed by enzymatic digestion. The agrobacteria clones obtained were used for plant genetic transformation.<br>
References<br>
An G. 1985. Development of plant promoter expression vector and their use for analysis of differential activity of nopaline synthase promoter in transformed tobacco cells. Plant P.hysiol. 81 :86-91.<br>
Anderson OD, Green FC, Yip RE, Kalford KG, Shewry PR and<br>
Malpica-Romero JM. 1989. Nucleotide sequences of the two high molecular-weight glutenin genes from the D-genome of a hexaploid bread wheat, Triticum aestivum L. Cheyenne. Nucl. Acids Res. oA-17 :461-462.<br>
Eirnboim HC and Doly J. 1979. A rapid alkaline<br>
extraction procedure for screening recombinant plasmid DNA.<br>
Nucl. Acids Res. 7 :1513. —	 	<br>
Depicker A, Stachel S, Dhaese P, Zambryski P and Goodman HM. 1982. Nopaline synthase : transcript mapping and DNA sequence. J. Mol. Appl. Genet. 1 :561-573.<br>
Finer JJ, Vain P, Jones MW and McMullen MD. 1992. Development of the particle inflow gun for DNA delivery to plant cells. Plant Cell Rep. 11 :323-328.<br>
Franck A, Guiliey H, Jonard G, Richards K and Hirth L. 1980. Nucleotide sequence of cauliflower mosaic virus DNA. Cell 21 :285-294.<br>
Frisch DA, Harris-Keller LW, iokubaitis NT, Thomas TL, Hardin SH and Kail TC. 1995. Complete sequence of the binary vector Binl9. Plant Mol. Biol. 27 : 405-409.<br>
Guerineau F and Mullineaux P. 1993. Plant transformation and expression vectors. In Plant Molecular Biology Labfax. Croy RRD (ed). BIOS Scientific Publishers, Blackwell Scieptific Publications, pp 121-147.<br>
Hanahan D. 1983. Studies on transformation of Escherichia coli with plasmids. J. Mol. Biol. 166 :557.<br>
Holsters M, Dewaele D, Depicker A, Messenf E, Van Montagu M and Schell J. 1978. Transfection and transformation of Agrobacterium tumefaciens. Mol. Gen. Genet. 136 :181-187.<br>
Horsch R5, Fry JE, Hoffmann NL, Eiholtz D, Rogers SG and fraley RT. 1S85. A simple and general method for transferring genes intc plants. Science 227 :1229-1231.<br>
Ishida Y, Saito H, Ohta S, Hiei Y, Komari T and Kumashiro T. 1996. High efficiency transformation of maize (Zea mays L.) mediated by Agrobacterium tumefaciens. Nature Eiotechnology 14 :745-750.<br>
Jefferson RA. Burgess SM and Hirsh D. 1986. b-glucuronicase from Escherichia coli as a gene-fusion marker. Proc. Natl. Acad. Sci. (USA) 83 :8447-8451.<br>
Kay R, Chan A, Daly M and McPherson J. 1987. Duplication of CaMV 35S promoter sequences creates a strong enhancer for plant genes. Science 236 :1299-1302.<br>
Klein TM, Wolf ED, Wu R and Sanford JC. 1987. High velocity microprojectiles for delivering nucleic acids into living cells. Nature 327 :70-73.<br>
Komari T, Hiei Y, Yasuhito S, Murai N and Kumashiro T. 1955. Vectors carrying two separate T-DNAs for co-transformation of higher plants mediated by Agrobacterium tumefaciens and segregation of transformants free from selection markers. Plant J. 10 .-165-174.<br>
McElroy D, Blowers AD, Jenes B and Wu R. 1991. Construction of expression vectors based on the rice actin 1 (Actl) 5' region for the use in monocot transformation. Mol. Gen. Genet. 231 :150-i60.<br>
Michael SF. 1994. Mutagenesis by incorporation of a phosphorylated oligofor PCR amplification. BioTechniques 16 : 411-412.<br>
Vain P, Yean H and Fliment P. 1989. Enhancement of production and regeneration of embryogenic type II callus in Zea mays L. by AgN03. Plant Cell Tissue and Organ Culture 18 :143-151.<br><br><br><br><br><br>
We Claim;<br>
1.	A method for expression of a nucleic acid sequence, or gene, coding<br>
for a polypeptide to be produced, in a cell, characterized in that it<br>
comprises the stages consisting of:<br>
transforming the cell with a vector of the kind such as herein described or a nucleic acid sequence identified by the number SEQ.ID01 toSEQ.ID22<br>
making a culture of the cell under conditions allowing the expression of a nucleic acid sequence, or gene, coding for the polypeptide to be produced.<br>
2.	The method as claimed in claim 1 wherein the cell is a prokaryotic or<br>
eukaryotic tell.<br>
3.	The method as claimed in either of the preceding claims  1  or 2,<br>
wherein  the  cell is  a cell  chosen  from  the  group  consisting of<br>
microbial cells, fungal cells, insect cells, animal cells and plant cells.<br>
4.	The method as claimed in any of the preceding claims 1 to 3, wherein<br>
the cell is a plant cell.<br><br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAzODUtZGVsLWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-00385-del-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAzODUtZGVsLWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-00385-del-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAzODUtZGVsLWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-00385-del-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAzODUtZGVsLWNvcnJlc3BvbmRlbmNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-00385-del-correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAzODUtZGVsLWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-00385-del-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAzODUtZGVsLWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-00385-del-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAzODUtZGVsLWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-00385-del-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAzODUtZGVsLWZvcm0tMTMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-00385-del-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAzODUtZGVsLWZvcm0tMTkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-00385-del-form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAzODUtZGVsLWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-00385-del-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAzODUtZGVsLWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-00385-del-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAzODUtZGVsLWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-00385-del-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAzODUtZGVsLWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-00385-del-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAzODUtZGVsLXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-00385-del-pct-210.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="233583-an-intervertebral-implant-device-for-holding-a-spacer-between-two-vertebrae.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="233587-system-for-providing-flexible-charging-in-a-network.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>233584</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/00385/DEL</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>14/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>31-Mar-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>03-May-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MERISTEM THERAPEUTICS</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>8 RUE DES FRERES LUMIERE, F-63100 CLEMONT FERRAND, FRANCE</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>VERONIQUE GRUBER</td>
											<td>44C AVENUE JEAN JAURES, F-63400 CHAMALIERES, FRANCE</td>
										</tr>
										<tr>
											<td>2</td>
											<td>DAVID COMEAU</td>
											<td>19 RUE DES CHARTRES, F-63670 LA ROCHE BLANCHE, FRANCE</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IB00/01243</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-09-04</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>9911112</td>
									<td>1999-09-03</td>
								    <td>France</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/233584-a-method-for-expression-or-a-nucleic-acid-sequence by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:47:20 GMT -->
</html>
