<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/233600-ligand by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:47:47 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 233600:&quot;LIGAND&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;LIGAND&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a ligand comprising a first immunoglobulin single variable domain having a first antigen or epitope binding specificity and a second immunoglobulin single variable domain having a second antigen or epitope binding specificity wherein one or both of said first and second variable domains bind to an antigen or epitope found in blood which increases the half-life of the ligand in vivo, and either (i) the first and the second immunoglobulin variable domains are heavy chain variable domains; or (ii) the first and the second immunoglobulin variable domains are light chain variable domains, provided that when the first and second immunoglobulin variable domains are Camelid VHH domains, the VHH domain which is specific for an antigen which increases the half-life of the ligand in vivo does not bind hen egg white lysozyme (HEL), porcine pancreatic alpha-amylase; NmC-A, hcg, BSA-linked RR6 azo dye or S. mutants HG982 cells. The present invention also relates to method of producing the said ligand, a nucleic acid encoding the same.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>SMALL FUNCTIONAL UNITS OF ANTIBODY HEAVY CHAIN VARIABLE<br>
REGIONS<br>
FIELD OF THE INVENTION<br>
The present invention relates to functional single-domains of antibody heavy chain variable regions, processes for the preparation and use of phage display libraries for identification and isolation of functional antibody single-domain molecules which bind to a desired constituent, and to pharmaceutical compositions containing the selected binding molecules.<br>
BACKGROUND OF THE INVENTION<br>
The specificity of the immune system is dictated by a very large repertoire of molecular surfaces that are clustered within two homologous families of proteins: antibodies and the T cell receptors. The two families share structural homology and have similar function, i.e. to confer specificity in antigen recognition (reviewed by Padlan, Mol. Immunol., 31, 169-217, 1994).<br>
The intact native antibody molecule generally contains heterodimeric structures of heavy and light chains, interconnected by disulfide bridges. Antigen recognition is conferred on the antibody by a limited number of hypervariable surface loops, differing in sequence and in length between different antibodies, and which are connected to a conserved framework structure. Both heavy and light chain variable regions each contain three hypervariable loop domains also referred to as Complementarity Determining Regions (CDRs). The three CDRs are designated as CDR1-CDR3 and are encoded by the recombined variable region gene segments.<br>
Understanding recognition of antigen targets at the molecular level is both of fundamental and applied importance, since the ability to mimic these loops using small molecules is of important therapeutic value. In addition to the design of novel reagents that are based on antibody hypervariable loops, small and recombinant versions of antibodies are of fundamental importance in the field of targeted therapy and imaging (reviewed by Reiter and Pastan, Trends in Biotech 16, 513-520, 1998).<br>
Recent developments in antibody engineering and recombinant DNA technology have made it possible to generate recombinant antibodies with high specificity and affinity for theoretically any antigen by employing phage display technology and constructing very large repertoires of antibodies that are displayed on the surface of filamentous phage (Winter et. al., Ann. Rev. Immunol. 12,433-455, 1994). International patent application WO 92/18619 describes methods for producing a library of DNA molecules capable of expressing a fusion polypeptide on the surface of a filamentous phage particle (phagemids) and producing heterodimeric receptors such as antibodies, and T-cell receptors.<br>
These large repertoires of naive, immune, or synthetic antibody fragments are fused to a minor phage coat protein; they are integrated into the DNA of the filamentous phage and displayed on the phage surface. Panning and selection of individual phage clones can screen the phage population containing tens of millions of individual clones through binding to an immobilized antigen (Barbas, Nature Medicine 1, 837-839,1995).<br>
After selection, antibody genes rescued from the phage genome can be expressed very efficiently in bacteria for the production of soluble functional recombinant antibody fragments (Ward et. al.. Nature 341, 544-546, 1989).<br>
Several forms of recombinant antibody fragments can be designed to substitute for large intact immunoglobulin molecules. These options include Fab fragments or Fv fragments that are stabilized and/or covalently linked utilizing various strategies (Bird et. al., Science 242.423-426.1988).<br>
Fv fragments of antibodies are the smallest modules of antibodies that contain the functional antigen-binding moiety without significant loss in antigen affinity and specificity. US patent 4.946.778 describes single chain molecules with the characteristics of antibody. These molecules are produced by convening two naturally aggregated but chemically separate light and heavy polypeptide chains from an antibody variable region into a single polypeptide chain which will fold into a three dimensional structure very similar to the original native structure. Furthermore, the single chain molecules in that<br>
disclosure may have binding specificity and affinity substantially similar to the binding<br>
specificity and affinity of the light and heavy chain aggregate variable region of an<br>
antibody.<br>
Smaller fragments of antibodies are advantageous for pharmaceutical applications for<br>
cancer targeting and imaging for example when small antigen binding molecules are<br>
needed to penetrate into large solid tumors.<br>
The Fv fragments of antibodies consist of the heavy chain and light chain variable domains and typically the hypervariable loops (CDRs) of both chains contribute to antigen binding. However, there are examples in which heavy chains alone retain a significant binding ability in the absence of light chain. It is also well established, from structural studies, that CDR3 of the heavy chain contributes the most to antigen binding because CDR3 residues are responsible for most of the surface contact area and molecular interaction with the antigen (Harber and Richards Proc. R. Soc. London Ser B.. 166, 176-187, 1966). Little, if any, binding activity was observed for isolated light chains.<br>
In view of these data, attempts were made to isolate single VH domains. For example VH domains were isolated from expression libraries derived from immunized mice (Ward et. al.. ibid). PCT application WO 94/18219 discloses methods for producing phage display antibody libraries and for increasing antibody library diversity by inducing mutagenesis within the CDR regions. Furthermore, methods for producing binding sites within the CDR regions of immunoglobulin heavy or light chains that are displayed on the surface of filamentous phage particles are disclosed in PCT application WO 94/18221<br>
More recently, it has been shown that camels make functional immunoglobulins that naturally lack light chains (Hamers-Casterman et. al., Nature 363,446-448,1993). As a result of these findings, antigen-binding VH domains were rescued from a human phage-displayed VH library (Davies and Reichmann, Biotechnology 13, 475-479,1995). In this case a human VH/VL interface of camelid immunoglobulin heavy chain was mimicked to prevent non-specific binding of the VH through its interface for the light chain variable domain. This was achieved through three mutations in the VH/VL interface that mimic camel heavy chains naturally devoid of light chain partners.<br>
US patent 5,702.892 (to Mulligan-Kehoe) discloses phage-display libraries of immunoglobulin single-domain heavy chains. The library disclosed is constructed in an M13-derived expression vector. The nucleotides encoding either CDR1 or CDR3 comprise a plurality of synthetically produced random nucleotides. A fusion protein that includes amino acid sequences encoded by the vector insert is expressed on the outer surface of the recombinant phage, which make up the library. The fusion proteins of the library are advantageously capable of binding a ligand. The second aspect of this disclosure relates to a method of inhibiting an activity of an intracellular constituent.<br>
The VH sequence disclosed in US 5,702,982 did not contain a reading frame and could not be translated into VH protein. No biochemical characterization of the produced proteins, nor data on the stability of the VH fragment were disclosed. The relevant protein was expressed only intracellularly, and there were no teachings regarding cloned proteins or peptides which are expressed without the phage.<br>
SUMMARY OF THE INVENTION<br>
It is an object of the present invention to provide small antibody-derived recognition units for experimental, medical, and drug design purposes.<br>
It is yet another object of the present invention to provide "microbodies" which are herein defined as single-domain antibody-like polypeptides or proteins which are soluble and stable and capable of binding a specific antigen of interest. These microbodies are encoded by selected clones and are produced as proteins by any of the methods known in the prior art including but not limited to production in E. Coli as insoluble inclusion bodies and in-vitro refolding. Alternative suitable production hosts include but are not limited to additional unicellular organisms, whether prokaryotic or eukaryotic, or cell lines from multicellular organisms, whether plant or animal, the latter ranging from insect to mammalian cells.<br>
One aspect of the present invention involves a phage-display library of a single-domain o'f the variable region of the heavy chain of an antibody molecule (VH). The phage display library according to the present invention is based on a natural framework scaffold of a monoclonal antibody, without any induced mutations or modifications in the original VH/VL interface framework residues, having a unique VH/VL interface comprising at least one charged residue and a randomized CDR3. A VH library according to the present invention is a valuable source for the isolation of recombinant antibody fragments of minimal size against antigens of interest.<br>
Another aspect of the present invention is a method for the preparation of a single-domain VH phage-display library that is based on a natural framework scaffold of a monoclonal antibody with a unique VH/VL interface and a randomized CDR3. The monoclonal antibody scaffold can be from any suitable mammal, including human or humanized monoclonal antibodies. In a currently preferred embodiment, the antibody scaffold can be obtained conveniently from murine monoclonals, as exemplified herein. Whereas in most VH families residue 44 is a Glvcine. cloned VH genes were screened initially for families in which position 44 is other than Glvcine and a VH clone was selected that belongs to mouse VH group I (A). It was found that this VH clone has a basic lysine residue instead of the highly conserved glycine commonly found in position 44. Thus a crucial scaffold element representing the VH/VL interface in this exemplary library comprises the sequence Lysine-44, Leucine-45, and Tryptophan-47.<br>
It is another object of the present invention to provide clones, which bind selectively to a specific antigen of interest, such clones being selected from the above libraries.<br>
One preferred embodiment according to the present invention, includes polypeptides derived from the VH libraries comprising the randomized sequence in the CDR3. These polypeptides. denoted herein as microbodies, are stable monomeric single-domain antibody-like molecules, which are capable of binding a specific constituent.<br>
Shorter peptides derived from the randomized sequence in the CDR3 represent another preferred embodiment of the invention. These peptides are stable antibody-like peptides. capable of binding a specific constituent of interest.<br>
The microbodies according to the present invention are antibody-like molecules representing a functional monomeric single domain having a molecular weight in the range of 10-15 kD on average. Shorter peptides derived from the CDR3 loop, which retain the binding attributes of interest, are between 4-20 amino acids in length, preferably 7-15 amino acids in length.<br>
One currently most preferred embodiment of the present invention comprises immunoglobulin-binding molecules which are either microbodies or shorter peptides.<br>
Another most preferred embodiment of the present invention comprises microbodies or shorter peptides which are capable of binding tumor necrosis factor (TNF) which is absorbed or linked to a solid support. Yet more preferred embodiments comprises microbodies or peptides which are capable of binding membrane or cell-bound TNF. A most preferred embodiment according to the present invention provides microbodies or peptides which are capable of binding soluble TNF.<br>
Another aspect of the present invention is directed to pharmaceutical compositions comprising as an active ingredient microbodies or peptides isolated according to the principles of the present invention.<br>
Another aspect of the present invention is directed to the use of pharmaceutical compositions comprising these microbodies or peptides for production of medicaments useful for the treatment or diagnosis of diseases and disorders. The present invention discloses methods of treatment of disorders wherein TNF is involved including but not limited to inflammatory bowel disease, rheumatoid arthritis, septic shock, multiple sclerosis, chronic inflammation, and allograft rejection.<br>
Therefore, further objects of the present invention are directed to pharmaceutical compositions comprising pharmacologically or diagnostically active microbodies prepared according to the methods disclosed herein and a pharmaceutically acceptable carrier or diluent. Further objects of the present invention are directed to methods for the treatment of diseases comprising administering a pharmaceutical composition comprising a therapeutically effective amount or a microbody or peptide prepared according to the principles of the present invention. Further objects of the present invention are directed to methods for the diagnosis of diseases comprising administering a pharmaceutical composition comprising a diagnostically effective amount or a microbody or peptide prepared according to the principles of the present invention.<br>
Essentially all of the uses that the prior art has envisioned for monoclonal or polyclonal antibodies, or for fragments thereof, can be considered for the molecules of the present invention. These uses include research, diagnostic techniques and therapy.<br>
By way of exemplification, a heavy chain variable region (VH) single-domain phage-display library was designed and constructed. The scaffold that was used for library construction was a native sequence of a monoclonal murine antibody with unique VH/VL interface. In contrast to any previously known libraries, there was no need to modify any residues in the VL interface residues to avoid non-specific binding of the VH domain. The library repertoire was generated by the randomization of 9 amino acids in CDR3, to yield a large repertoire of independent clones. The library was selected through binding to protein antigens and individual clones were isolated. Isolated polypeptides were recovered having binding affinities were obtained in the nanomolar range.<br>
The VH genes encoding for specific binding clones were rescued and expressed in large amounts in E. coli. Large amounts of soluble and stable single-domain VH protein were made from insoluble inclusion bodies by in-vitro refolding and purification. Biochemical and biophysical characterization of the VH protein revealed a highly specific, correctly folded, and stable monomeric molecule. The properties of these molecules make them useful for clinical, industrial, and research applications as well as toward the improvement in the design of small molecules that are based on the hypervariable loops of antibodies.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figure 1. Composition of single-domain VH library: nucleotide and amino acid sequence. Figure 2. Construction of single-domain VH library: schematic presentation.<br>
Figure 3. Binding and specificity of phage clones to antigens: A: Binding titration of isolated phage clones to TNF. B: Binding titration of isolated phage clones to Ig.  C: Binding specificity of Ig reactive phage clones.<br>
Figure 4. Binding specificity of microbody clone number 7 and control to immobilized TNF and non-relevant antigens.<br>
Figure 5. Production and purification of single-domain VH protein.<br>
A: SDS-PAGE analysis of purified single-domain VH protein.<br>
B: Molecular homogeneity (by FPLC) of the refolded purified VH single-domain<br>
molecule.<br>
Figure 6. Biochemical characterization of single-domain VH protein. CD spectra of refolded single-domain VH protein.<br>
Figure 7. Characterization of the binding properties of Ig-specific VH single-domain. A: ELISA binding assay of purified Ig-specific single-domain VH protein to human IgG. B: Competition binding analysis of VH single-domain protein to human IgG using iodinated Ig-specific VH protein and increasing concentrations of cold VH protein.<br>
Figure 8. Characterization of the binding properties of Ig-specific VH single-domain. A: Binding specificity of purified Ig-specific VH protein to Ig subtypes and Ig fragments. B: Binding analysis of Ig-specific VH single-domain protein to IgG by real-time surface plasmon resonance technology.<br>
Figure 9. Specific binding of single-domain VH clones to Streptavidin and other proteins as tested by ELISA.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The antigen binding site of antibodies is formed by the hypervariable loops of the variable domains of light and heavy chains. Residues present in all six loops, three in each domain, may be actively involved in molecular interaction with the antigen. It is well established now from structural studies involving crystallographic analysis of antigen-antibody interactions, that residues in the CDR3 of the heavy chain contribute the most to antigen binding by making most of the contacts with the antigen. It is also known in the art that camelid immunoglobulin heavy chains can occur naturally without light chains but still bind antigen.<br>
It is now disclosed that according to the present invention it is possible to obtain native heavy chain variable region sequences, including but not limited to murine sequences, without '"camelid" mutations or any other induced mutations. Mutations induced in the sequence for stabilization, as is common in the prior art, were not necessary according to the present invention.<br>
The disclosed protein sequences derived are as stable as native intact immunoglobulins and furthermore, they retain the binding attributes of intact immunoglobulins. The sequence in the "camelid mutation" region, namely in the region which corresponds to the VH/VL interface in a native heavy chain, is unique and stable in the present invention. These and further advantages over the background art will become apparent from the description of the currently preferred embodiments of the present invention.<br>
In principle, the present invention provides for the first time a small monomeric functional unit derived from an antibody, which can be obtained in substantially purified form as a polypeptide which is soluble and stable and retains the binding capacity to any antigen of interest. These miniature antibodies are herein denoted as microbodies.<br>
In the specification and in the claims the term "microbodies" refers to single-domain functional modules of antibodies as described above.<br>
In the specification and in the claims the term "polypeptide" refers to a single chain of amino acids and may also be referred to as a protein.<br>
In the specification and in the claims the term 'soluble' refers to a molecule which is present in a substantially non-aggregated, non-precipitated form in solution in aqueous medium.<br>
In the specification and in the claims the term "stable" refers to a compound that is sufficiently robust to survive isolation to a useful degree of purity, and formulation into an efficacious therapeutic agent.<br>
As used herein and in the claims the term "antigen" defines any given molecular entity of interest including but not limited to: protein, polypeptide, peptide, glycoprotein, carbohydrate, polysaccharide, oligosaccharide, disaccharide, lipid, lipoprotein, or any organic molecule for which it is desired to obtain a binding molecule according to the principles of the present invention.<br>
Certain abbreviations are used herein to describe this invention and the manner of making and using it. For instance, BSA refers to bovine serum albumin; CDR refers to Complementary Determining Region; CPU refers to colony forming unit; CH refers to constant heavy chain region of antibodies;; CL refers to constant light chain region of antibodies; CPM refers to counts per minute; ELISA refers to enzyme linked immuno-sorbent assay; Fab refers the portion of the immunoglobulin molecule which contains the antigen binding site, containing the VH, CHI, VL and CL domains of antibodies; FPLC refers to fast performance liquid chromatography; Fc refers to the constant portion of the immunoglobulin molecule which contains the CH2 and CHS constant domains of antibodies; HRP refers to horseradish peroxidase; Ig refers to immunoglobulin; IgG refers to immunoglobulin gamma; PCR refers to polymerase chain<br>
reaction; SAV refers to Streptavidin; scFv refers to single chain variable region containing the VH and VL domains of antibodies; SDS-PAGE refers to sodium dodecyl sulfate polyacrylamide gel electrophoresis; SPR refers to surface plasmon resonance; TMB refers to 3'.3\5\5\- tetramethylbenzidine; TNF refers to Tumor necrosis factor; VH refers to heavy chain variable domain and VL refers to light chain variable domain.<br>
Natural coded amino acids are represented by three-letter codes or by single-letter code, according to IUPAC conventions. When there is no indication, the L isomer is used.<br>
The nomenclature used for the residues in the VH sequence is according to the Kabat system (Kabat. et al, 1991 Sequences of proteins of immunological interest. US Public Health Services. NIH, Publication no. 91-3242).<br>
Preferred Embodiments According to the Present Invention:<br>
A phage display library of VH single domain proteins was generated and used to isolate binding molecules against antigens to which the library was selected. An unmodified naturally occurring VH scaffold sequence was used as a framework for the construction of the VH library in which the CDR3 loop was randomized to create the VH repertoire.<br>
This particular VH sequence was chosen by computer sequence analysis using information about the molecular properties and interactions that compose the VL interface. This is in contrast to previously made VH libraries in which the VL interface of the VH domain was mutated to mimic camelid heavy chain variable domains that are naturally devoid of light chain partners.<br>
On the structural level, the interface interactions of the VH and VL canonical structures of a typical VL interface (44G. 45L. 46E, 47W) [PDB access code 2fbj and 2fb4] was compared with our VL interface (44K, 45L. 46E, 47W) [PDB access Ibafj. Significant differences in their contact residues where found, which may indicate the structural and functional differences of the two interfaces and their contribution to VH domain stability (Padlan, Mol. Immunol., 31, 169-217, 1994). The modifications in the VL interface are required to abolish non-specific binding of the single VH domains and to reduce the<br>
tendency for the formations of dimers because of the hydrophobic nature of the residues that compose the VH/VL interface.<br>
VH domain proteins specific for the antigens TNF, Ig and Streptavidin were rescued from the VH library after expression as fusion proteins to the minor coat protein on the surface of filamentous phage. The selection process consisted of panning on polystyrene immobilized antigen.<br>
The binding of phage clones and purified VH protein was specific. They recognized only the antigen against which they were selected. The VH domains do not exhibit non-specific binding to other ligands.<br>
VH domains are expressed very efficiently in bacteria, they are made as insoluble intracellular inclusion bodies. More than 30 mg/liter of highly purified active VH can be produced by in-vitro refolding from shake flask cultures that are harvested at OD600nm=2.5-3.0. The VH can be efficiently purified by standard chromatography techniques of ion exchange and size-exclusion.<br>
The secondary structure of the VH domain is identical to that of VH domains found as part of Fv, scFv, or Fab (or generally Ig V domains). The VH domains are very stable molecules that can be kept at high concentrations for structural analysis. Their stability is similar to that obtain with other stable Ig-based recombinant molecules such as scFv and Fab fragments.<br>
Biophysical analysis of the VH protein using analytical ultracentrifugation revealed that the VH could be maintained predominantly as a monomer. Although it has a very weak tendency to dimerize, with a dissociation equilibrium constant for dimer formation of 1.1 mM. no indications of higher oligomers were found. This suggests that high concentrations of VH protein can be achieved without stability problems that occur due to aggregation. This is the first demonstration of such analysis on VH protein.<br>
Recently, similar results were obtained for another Ig-like domain; a unique Va domain of a recombinant T cell receptor that was produced as a very stable monomeric protein (Plaksin et. al., J. Exp. Med. 184, 1251-1258, 1996).<br>
Previous studies indicated the VH proteins are very unstable and mutations in the VL interface were required to maintain the VH protein in a soluble and stable form. Dissociation constants that were detected for an Ig-binding VH protein were 20-100 nM. These values are similar to those determined for antibody fragments selected from synthetic scFv or Fab phage displayed naive repertoires of comparable size (Barbas ibid). The properties of the VH domain that are described here make them attractive for clinical applications.<br>
It was demonstrated previously that small antibody fragments perform better in vivo than whole antibodies or Fab's(Colcheret. al. J. Natl. Cancer Inst. 82. 1191-1197. 1990). When used for pharmaceutical applications such as imaging or drug carriers for targeted therapy, the reduced size of the molecule enable them to penetrate faster and better into tissues. The bio-distribution and renal clearance is faster.<br>
VH domains can be used for in-vitro and in vivo studies in the same way that other antibody fragments are being used. VH domains can be labeled with radioisotopes, fluorescent probes, or other detection markers in the same way that antibody fragments are being labeled.<br>
Fusion proteins can be constructed with VH domains (.with reporter proteins, fluorescent proteins, toxins, etc) as well as coupling to various agents.<br>
The affinity of the selected VH domain is high enough to perform these tasks without further improvement. However, it is tempting to try and improve the affinity of the VH domain and generate second generation of improved molecules. This can be achieved by further randomization of selected residues followed by further selection. More efficient is the direct isolation of high affinity binders from the original repertoire by improvement of the library complexity.<br>
Most desirable is the construction of large libraries, which cover more than the theoretical repertoire. These large libraries were constructed for scFv fragments of antibodies by recombination between large VH and VL repertoires (Griffiths et. al., EMBQ J. 13,<br>
3245-3260. 1994). The size of the CDR3 loop in the library can be enlarged combined with codon-based mutagenic oligonucleotides that can be used to avoid stop codons or cysteines within the CDR. Randomization of other CDRs is also desirable to construct larger repertoires. The generation of such libraries will enable also structure-function relationship studies on the relations between certain CDR3 loop sizes and the type of ligand or the relative contribution of each CDR loop for binding.<br>
The use of VH domains for structure-function studies can be also an important tool for drug discovery. As shown hereinbelow, it is possible to select single-domain VH phage clones that bind a specific consensus sequence with specificity to polystyrene that was defined previously by a peptide phage library (Adey et. al. Gene 156. 27-31, 1995). This suggests that the main and randomized CDR3 loop in the VH single-domain phage library can replace screening of peptide libraries and be an improved alternative because the leads discovered with the single-domain library have a significantly better binding affinity compared with leads isolated from peptide libraries.<br>
The hypervariable loops of antibodies and in particular CDR3 are sequential stretches of conformationaly constrained random amino acids. Understanding recognition at the molecular level by structural studies such as NMR and crystallography combined with molecular modeling is both of fundamental and applied importance since the ability to mimic these loops using small molecules is of broad therapeutic use. Synthetic and computerized tools can be combined for the design and synthesis of peptides designed to structurally resemble, and mimic antibody hypervariable loops. Success in this goal will lend credence to the idea that these loops can, in certain cases, mimic biological molecules and moreover, provide a powerful generic tool for the design of novel drugs (Sheriff and Constantine Nature Struct. Biol. 3. 733-736,1996).<br>
The functional VH domains that can be isolated from the VH library are the molecular leads that will enable the design and synthesis of novel peptides that are based on the hypervariable loop of the isolated VH sequence. Once the design principles and synthesis analysis are laid out. additional activities such as catalytic abilities, can be incorporated in the peptide molecule while conserving recognition, thus broadening the scope of the<br>
process developed. All this can be implanted into an efficient generic process leading to novel drug discovery.<br>
Initial examples for such a process were recently demonstrated by the generation of potent enzyme inhibitors that were derived from dromedary heavy chain domains (Ghahroudi et. al. FEBS Lett. 414. 521-526, 1997). Competitive inhibitors against bovine erythrocyte carbonic anhydrase and porcine pancreatic a-amylase were isolated. Crystallographic analysis of recombinant dromedary heavy chain single domain with lysozyme revealed the amino acids that are primarily involved in antigen recognition. They form an internal image of the lysozyme active site cavity and can inhibit the enzymatic activity in a competitive manner.<br>
Seven consecutive amino acids of the CDR3 loop form a structural mimic of the natural carbohydrate substrate of the enzyme. Therefore, the exposed CDR3 loop of the dromedary VH in that disclosure, or generally VH domains that will be isolated from libraries, might be good candidates to serve as a lead compound for new drugs. Another example is the selection of a camelized VH domain that acts as an inhibitor of hepatitis C virus NS3 protease (Martin et al. Protein Eng. 10, 607-614. 1997). Another example arises from our findings herein that we were able to isolate a VH protein that can bind specifically Ig of different types and species. This product can be further developed as a specific reagent for detection, purification, and analysis of antibodies. This can be performed on the intact VH protein or alternatively using the CDR3 encoded peptide that is responsible for the unique binding specificity.<br>
According to the principles of the present invention, the lower complexity of the antigen-binding site in isolated predominantly monomeric VH domains, being composed of only one randomized loop and two conserved loops versus six random loops in the Fv, reduces the complexity of choosing the optimal amino acid sequence from which to develop small molecules.<br>
The current strategy for choosing a particular VH domain as a starting point for the construction of the VH library was to use a naturally occurring VH domain that can be used as a template without the need for modifications by mutagenesis. Several observations related to VH-VL interactions that were reported in the literature were used.<br>
First, that the relative stability of Fvs depend on the strength of the VH and VL interactions in a region known to be the VH-VL interface. This region is very highly conserved among VH and VL family groups and lies around position 44 in the heavy chain (Kabat numbering system) and position 105 of the light chain. These positions are located in a stretch of the p4-strand of VH which is next to. and runs antiparallel to, a stretch of the p9-strand of VL (HP4-LP9). In the model structure of Fvs as well as in a crystal structure of a disulfide-stabilized Fv, the closest interchain (VH-VL) contacts in the framework region occur within these stretches.<br>
There are variations regarding these types of interactions with some Fvs being very stable to such a degree that the strong VH-VL interactions can hold the two domains together without the need for covalent stabilization such as a peptide linker or disulfide bonds (Bird et. al. ibid). In other cases the interchain interactions are not sufficient to stabilize the Fv and they need a covalent bridge or linker. It is also found in many Fvs the VH-VL interactions are so weak that even a peptide linker or other forms of stabilization does not enable the formation of a stable Fv because the VH and VL domains dissociate from each other rapidly.<br>
Second, the use of unmodified VH as a template for the generation of small stable VH domains hypothetically was a concern because the exposed hydrophobic VL interface was likely to generate instability or result in non-specific binding during antigen selection of the phage-displayed VH library (Davies and Reichmann, ibid). Non-specific interactions through the VL interface also hamper structural studies on isolated VH domains in solution.<br>
These observations suggest that VL interface residues which form weak VH-VL interactions will be better candidates to be used as a scaffold for the generation of a VH library because their weak interacting VL interface will abolish problems of non-specific binding during selection. These can be less hydrophobic or charged residues.<br>
The sequence diversity of the VH-VL interface was therefore analyzed, using computer databases (Kabat database of sequences of proteins of immunological interest). According to this analysis a hybridoma clone was selected from our collection that is different in the<br>
interface sequence from the frequently found residues of both in vivo rearranged and in-vitro selected antibodies.<br>
It was previously shown that so-called "camelid" mutations of positions 44 from Gly to Glu, 45 from Leu to Arg and 47 from Trp to Gly or to He, in the VH-VL interface enabled the construction of a VH library. The mutations that were introduced are based on sequence comparison with camelid immunoglobulin heavy chains, which occur naturally without light chain partners. These three residues comprise the VL interface and are known to be critical for VH-VL interactions (Davies and Reichmann ibid). The three residues at these positions are highly conserved through different human and mouse VH families. Residues 45 and 47 are more conserved however than residue 44 in which some variability among VH families is being observed. In most VH families residue 44 is a Glycine. Initially, cloned VH genes were screened for families in which position 44 is other than Glycine and selected a VH that belongs to mouse VH group I (A). It was found that this VH clone has a basic lysine residue instead of the naturally occurring highly conserved glycine commonly found in position 44. Thus the crucial scaffold element representing the VH/VL interface in the currently most preferred VH library comprises the sequence Lysine-44, Leucine-45, and Tryptophan-47.<br>
The library repertoire was generated by randomization of the third hypervariable loop (CDR3) of the VH. This loop typically makes most antigen contacts in antibody combining sites. The VH gene was produced by PCR using an oligonucleotide, which degenerate and randomize 9 residues in CDR3 between and inclusive residues 95 and 100C. The last 2 residues of the CDR3 (101 and 102), were not randomized because of their high level of conservancy and their known structural role at the base of the loop (Chothia and Lesk JL Mol. Biol. 196. 904-917,1987). Cloning sites were introduced by a second PCR which facilitated the cloning of the VH library into the phagemid vector pCANTABSE as a fusion to the phage minor coat protein encoded by geneS. A repertoire of 4x108 independent clones of VH domains was obtained following 3 ligation reactions and 30 electroporations.<br>
The skilled artisan will appreciate that the following examples are merely illustrative and serve as non limitative exemplification of the principles of the present invention and that<br>
many variations and modifications are possible within the scope of the currently claimed invention as defined i   ;he claims which follow.<br>
EXAMPLES<br>
Example 1. Design and construction of single-domain vh library.<br>
Method:<br>
The gene encoding the VH domain scaffold [family I(A)] originated from a mouse<br>
hybridoma specific for H-2Dd + RGPGRAFVTI peptide. The 5' region of the VH gene was<br>
amplified by PCR using oligonucleotides 5/7/5'<br>
[5'-AAGGAAAAAAGGCCCAGCCGGCCGATGTCCAGCTGCAGGAGTCA<br>
GGACCGGC-3'] which introduced the SfiJ cloning site and the 3' region with NotIT<br>
oligonucleotide[5'-TATCAAATGCGGCCGCGACGGTGACAGTGGTCCCTTGGCCCC<br>
AGTAGTCMNNMNNMNNMNNMNNMNNM<br>
GTGGCTGT-3'] that randomized 9 amino acids in CDR3 and introduced a NotI cloning<br>
site.<br>
One ug of PCR product was re-amplified (10 cycles) with the following oligonucleotides<br>
Sy//5'short[5'-AAGGAAAAAAGGCCCAGCCGGCCGAT<br>
GTCC-3'] and NotWshort [5'-TATCAAATGCGGCCGCGACGGTGACA<br>
GTGG-3'] to avoid non-symmetric pairing of strands due to primer exhaustion. The final<br>
PCR product product was digested with Sfil and NotI and ligated into the phagemid vector<br>
pCANTAB 5 E (Amersham Pharamcia Biotech). Ligated DNA was electroporated into the<br>
£ coli strain TGI (Gibco BRL).<br>
Results:<br>
In most VH families residue 44 is a Glycine, we screened initially our cloned VH genes for<br>
families in which position 44 is other than Glycine and selected a VH that was cloned from<br>
a mouse hybridoma generated against an HIV peptide in complex with H-2Dd. The VH<br>
belongs to mouse VH group I (A), the nucleotide and amino acid sequence are presented in<br>
Figure 1.<br>
The library repertoire was generated by randomization of the third hypervariable loop<br>
(CDR3) of the VH. This loop typically makes most antigen contacts in antibody combining<br>
sites. The VH gene was produced by PCR using an oligonucleotide, which degenerate and randomize 9 residues in CDR3 between and inclusive residues 95 and 100C. The last 2 residues of the CDR3 (101 and 102) were not randomized because of their high level of conservancy and their known structural role at the base of the loop. Cloning sites were introduced by a second PCR which facilitated the cloning of the VH library into the phagemid vector pCANTABSE as a fusion to the phage minor coat protein encoded by<br><br>
gene3 (Figure 2). A repertoire of 4x10 independent clones of VH domains was obtained following 3 ligation reactions and 30 electroporations. Test screening of individual randomly picked library clones by DNA sequencing revealed an intact insert that contained the cloned VH gene and random CDR3 sequences of expected length.<br>
Example 2. Panning of library and selection of specific phagc clones.<br>
Method:<br>
Phage library (5xlOl! cfu) was selected against antigens by panning 4 rounds on polystyrene sulfated latex beads (Interfacial Dynamics Corporation) coated with soluble TNF (R&amp;D) or with magnetic-streptavidin-coated polystyrene beads (DYNAL) to which biotinylated Goat Immunoglobulin was immobilized. Beads were coated overnight at room temperature with 1-5 jag of protein in 50-200 ul of PBS. Following antigen immobilization the beads were blocked with PBS containing 0,05% Tween. and 5% low fat milk. Phage pool was incubated for Ihr in blocking buffer and washed with PBS 0.05% tween. Bound phage were eluted with 500ul of 0.2M glycine pH 2.2 and neutralized with 75ul of 1M tris pH9.1. Results:<br>
Examples for panning on two antigens. Tumor necrosis factor alpha (TNF) and Ig are shown in Table 1. Soluble recombinant TNF immobilized to sulfated polystyrene latex beads was subjected to four rounds of panning. The number of phage captured on the antigen-coated beads increased by more than 80-fold with the fourth round of panning. Forty individual phage clones from the fourth round of panning were tested in a phage ELISA assay for binding to immobilized TNF. Positive clones were sequenced. Example for phage ELISA results using individual clones are presented in Figure 3A. We were able to isolate clones which were strongly positive and specific for TNF.<br>
The VH library was also used in a panning experiment in which biotinylated IgG was immobilized on Streptavidine-coated magnetic beads and Ig binding phage clones were isolated. As shown in Table IB. after four rounds of panning a 150-fold enrichment in the number of phage captured by antigen was observed. Phage ELISA of individual clones revealed strong and specific binding of the antigen compared to control phage (Figure 3B). The genes encoding the VH protein were rescued from positive phage clones and their sequences were analyzed. As shown in Table 2. all clones exhibited an intact VH insert that contained a random 9 amino acid stretch at the expected location of CDR3. Sequence analysis revealed also consensus residues between positive clones that were isolated after the forth round of panning. For example, clones number 1 and 4 which recognize Ig show a consensus sequence of GLY-X-SER-PRO-GLN. It may be noted that in these cases X is a hydrophilic residue though this may not be a straight requirement. The difference is the location of the consensus within the CDR. For a detailed characterization of a VH single domain protein we choose to use clone 4. Consensus sequences were also obtained in several independent screenings in which antigens were immobilized on polystyrene latex beads and binding phage clones were characterized and found to be specific for plastic (polystyrene). This phenomena is characterized in the literature using peptide phage display libraries and consensus sequences rich in Trp and Tyr, which bind plastic (Adey et al. ibid). Several VH phage clones with such consensus sequences were isolated as shown in Table 2. These results demonstrate that individual antigen-binding phage clones can be isolated from the VH library. These phage clones are highly reactive in phage ELISA assays and are specific for the antigen. DNA sequence analysis of the clones isolated after the fourth round of panning revealed that the enrichment was specific for individual clones, thus, 50-60% of the sequences obtained were identical at the expected region of CDR3.<br>
(Table Remove)  : Panning of single-domain vh library on the antigens TNF and Ig A. TNFB. Ig<br><br>
Amino acid composition of CDR3 region of selected phage clones.<br><br>
Example 3. Expression and production of soluble vh single-domain molecules.<br>
Methods:<br>
For large scale protein production plasmid DNA from positive binding clones was re-amplified with the following oligonucleotides pET-21aVH5'NdeI [5--GGGAATTCCATATGGATGTCCAGCTGCAGGAGTC-3'] and PET-21aVH3'XhoI [5'GGGAATTCCTCGAGCTATGCGGCACGCGG TTCCA-3'].These inserted cloning sites that enabled subcloning into the T7 promoter-based pET-21a expression vector (Novagen). Protein was expressed at high levels in BL21 (DEA.3) cells upon IPTG induction and accumulated in intracellular inclusion bodies. Inclusion bodies were isolated and purified from the induced BL21 cells and solubilized in Guanidine HC1. Following reduction inclusion bodies were refolded in a redox-shuffling buffer system and Arginine. After refolding the protein was dialyzed and concentrated by Minisette 5K (Filtron), and purified by MonoQ (Amersham Pharmacia Biotech) ion-exchange and TSK300 gel filtration chromatography.<br>
Results:<br>
To produce soluble VH protein we have rescued the VH gene from the isolated phage genome by PCR and subcloned the gene into a pET system expression vector in which expression is driven by the T7 promoter. Expression of the VH genes in E Coli BL21 cells was very efficient and recombinant protein accumulated as insoluble intracellular inclusion bodies. The VH could be detected as the major band on SDS/PAGE of solubilized whole cell as well as isolated purified inclusion bodies. Purified inclusion bodies contained &gt;90% recombinant VH protein. Although expression and production of VH was very efficient for all VH genes that were isolated (from TNF and Ig phage clones), we choose to focus on the characterization of one VH protein from phage clone #1 that recognize Ig. Inclusion bodies were purified, solubilized in 6M guanidine HC1, and refolded by in-vitro redox-shuffling buffer system.<br>
Refolded protein was purified by sequential Q-Sepharose and MonoQ ion-exchange chromatography. followed by size exclusion chromatography on TSK3000 column. The yield of refolded VH domain was 25-30%; i.e.. 25-30 mg of purified soluble VH protein was obtained from 100 mg of refolded inclusion bodies. These results indicate that the VH single-domain protein can be refolded in-vitro to high yields and purity. The purified VH protein migrated as a single band with an apparent size of Mr 19300 (±5% error) on SDS/PAGE as calculated according to the relative migration against a set of molecular weight markers (Figure 5A). Similar results were obtained when a VH protein was produced from the phage clone that binds TNF.<br>
Example 4. Characterization of binding specificity of phage clones.<br>
Method:<br>
Single phage clones were screened by EL1SA assays using PEG precipitated phages. Phage or soluble VH were analyzed by ELISA in 96-well microtitre plates (Maxisorb NUNC) coated with different concentrations of antigen (0.1-lp.g/ml). Blocking of plate was performed with PBS 0.05% Tween and 5% low fat milk. Phage or protein at different concentration was added in blocking buffer. Detection second antibody was anti-Mi 3-HRP (Amersham Pharmacia Biotech) for phage analysis and anti-E-tag-HRP (Amersham Pharmacia Biotech) for soluble protein analysis. ELISAs were developed with 3',3',5',5tetramethvlbenzidine (TMB).<br>
Results:<br>
The major concern with VH single domain phage is the non specific binding due to their exposed VL interface. The reduced area of the potential antigen binding site in an isolated VH, compared with a combination of VH and VL, might also compromise specificity. We therefore analyzed by phage ELISA assays the binding of the isolated phage clones to various control antigens. As shown in Figure 3C the isolated VH single domain phage that recognize IgG proved to be highly specific. No binding to any antigen other than that selected on. was detected. Similar specificity studies were performed on the phage clones that recognize TNF and similar results were obtained (Figure 4). The phage clones that were isolated by panning on IgG recognized specifically a range of immunoglobulins from different species including hamster, human, mouse, and rabbit IgG. They also recognize different Ig isotypes such as IgM, IgGl, IgG2a and IgG2b. Similar results were obtained when soluble VH single-domain protein was made from the periplasm of phage clones 1 and 4. These results suggest phage clones isolated form the VH library can bind very specifically to the antigen to which they were selected for and they are not sticky. Analysis of the binding characteristics of soluble, purified VH proteins that were generated from the isolated phage clones further indicate the specificity results that were obtained with the parental phage clones.<br>
Example 5. Biochemical and biophysical characterization of vh single-domain molecules.<br>
Methods:<br>
Mass Spectroscopy and Circular Dichroism (CD): CD spectra of VH domain was<br>
measured in a spectropolarimeter (JASCO 500) with sensitivity of 0.5mdeg/cm and scan<br>
speed of lOnm/min at room temperature. Protein concentration was 1.8mg/ml.<br>
Secondary structure calculations were measure using CONTIN, K2d and a published<br>
program.<br>
Analytical ultracentrifugation: Sedimentation equilibrium experiments were conducted<br>
with a Beckman Optima XL-A analytical ultracentrifuge equipped with absorbance optical<br>
system. The protein was dissolved in PBS. and epon double-sector centerpieces equipped<br>
with quartz windows were filled with 180 |j.l of protein at several different loading<br>
concentrations, ranging from 1.84 ^M to 91 |aM. Prior to the ultracentrifugation.<br>
aggregation state of the samples was assessed using a DynaPro dynamic light scattering instrument (Protein Solutions, Charlottesville, VA). Using an An50-Ti rotor. The samples were then centrifuged at rotor speeds of 20.000 rpm, 25.000 and 30.000 rpm at a temperature of 4°C. Absorbance distributions were recorded at wavelength of 230 nm, 250 nm, and 280 nrn in radial increments of 0.001 cm, taking 50 measurements per step. Sedimentation velocity experiments were performed with the same sample after the equilibrium experiment, using a rotor speed of 50,000 rpm at 4°C, scans were taken at 250 nm or 230 nm. Sedimentation equilibrium analysis was performed by global analysis of several data sets obtained at different loading concentrations and different rotor speeds. The partial specific volume of 0.716 ml/g at 4°C, and the molar extinction coefficient of 42400 was calculated on the basis of the amino acid composition. The extinction coefficients at 230 nm and at 250 nm were determined by analysis of the signal ratios 230/280 and 250/280. Each data set was decomposed into a sum of the well-known sedimentation equilibrium exponentials for a monomer and dimer in reversible self-association equilibrium. The absence of thermodynamic non-idealities and pressure effects was assumed. Results:<br>
As shown in Figure 5 A the VH protein was highly pure and homogeneous as judged by SDS/PAGE. Size exclusion chromatography on a calibrated TSK3000 column showed that the purified VH preparations eluted as monomers with a molecular mass of-19 kDa (Figure 5B). Mass spectrometry analysis revealed that the purified VH preparation has the expected mass. When purified protein was concentrated by ultrafiltration to levels as high as - 2 mM (25 mg/ml) and analyzed for molecular from on size exclusion TSK3000 column in PBS, no indication of dimer formation or other stability problems ,such as aggregation, were observed.<br>
To test the stability of the VH protein, aliquots at a concentration of 0.1 mg/ml were incubated at 37°C for up to 24 hours and subsequently analyzed for the molecular form of the VH by size-exclusion on TSK3000 column. The VH was very stable and no indication for dimerization or aggregation was observed after 24 hours of incubation at 37°C. To evaluate the nature of the secondary structure of the refolded VH domain, protein purified through both ion-exchange and size-exclusion chromatography was examined by CD spectroscopy (Figure 6). Spectra of the VH protein showed a characteristic minima at<br>
215 nm consistent with a largely p sheet structure. The spectra was analyzed for secondary structural calculations and found to give a specific pattern of secondary structure. The VH showed 56% p sheet and 39% p turn, no a helix. The estimated error is 6%. This spectra is similar to those reported for a single Vot domain of the T Cell receptor (Plaksin, et. al. ibid) and to those described for single-chain Fv. These all have similarities to the spectra of Ig V domains and Fab fragments. These observations suggest that the VH protein is folded correctly after the in-vitro renaturation process of the bacterial inclusion bodies. The VH protein exhibits a CD spectrum consistent with being a single-Ig-domain. The results suggest that the VH domain is folded in the correct conformation and is similar to other Ig domains with known CD spectra. To clearly demonstrate that the VH protein is a predominantly monomeric we performed analytical ultracentrifugation. The sedimentation equilibrium profiles of the VH protein were measured over a 50-fold range of protein loading concentrations and at 3 different rotor speeds. These profiles could not be satisfactory globally fitted on the basis of a single monomeric species with the molar mass and partial specific volume predicted by the amino acid composition. Independent analysis of the sedimentation profiles at the highest and lowest loading concentrations with the floating molar mass led to a best-fit molar mass increasing with concentration. This indicated that the protein exhibited a weak self-association. Global analysis of all the equilibrium data using the predicted buoyant molar mass and considering reversible dimerization resulted in a fit of high quality, with a best-fit association constant (Ka) for dimer formation of 900 ± 100 M'1. This result indicates that the VH protein has a very low weak tendency of dimerization. Sedimentation velocity experiments were well-described by the Lamm-equation model for rapid monomer-dimer equilibria, with a best-fit monomer sedimentation coefficient of 1.7S. These results also correlate well with dynamic light scattering data, which lead to a hydrodynamic radius of 2.0 nm. No indications of higher aggregates were found in sedimentation equilibrium, sedimentation velocity, and light scattering experiments. The association constant obtained corresponds to a dissociation equilibrium constant 1.1 mM. This would suggest that at a concentration of 1 mM (-17 mg/ml) 50% of the VH protein is in the form of a dimer and 50% in the form of a monomer. These results are the first demonstration of such biophysical analysis of a VH protein using analytical ultracentrifugation.<br>
Example 6. Characterization of the binding properties of vh single-domain molecules.<br>
Methods:<br>
Binding assays: For competition binding assays VH protein was iodinated using the Chloramine-T method. 96-well microtiter plates were coated with 1 u.g/ml human IgG (overnight, 4°C). Plates were blocked for 1 hr at room temperature with PBS containing PBS 0.05% Tween and 5% low fat milk. Increasing concentrations of cold VH protein were added (competitor) with 2x10" CPM of iodinated VH protein. Each experimental point was performed in triplicates. Binding was for 1 hr at room temperature. Plates were than washed 4 times with PBS containing 0.05% Tween. Bound Labeled VH protein was eluted from the plate by 1% SDS and 100 mM phosphoric acid. Bound and unbound (wash) were counted in a y-counter. Non specific binding was determined by using 30-fold molar excess of cold competing VH protein. Maximal binding was determined by using iodinated VH protein without competitor.<br>
Surface plasmon resonance (SPR) analysis: The direct binding of the expressed, purified microbody to an lgG2a murine monoclonal antibody was measured by surface plasmon resonance using the BlAcore 2000. Purified monoclonal antibody 34-2-12 (an IgG2a, k mAb with specificity for the MHC class I molecule, H-2Dd) was covalently coupled to a CM-5 carboxymethylated dextran chip using standard coupling procedures. The microbody was passed over the chip in standard HBST buffer at a flow rate of 1 Oul/min, and data was collected and analyzed with the global curve fitting programs of BIEvaluation 3.0 (Biacore AB). Results:<br>
To determine the binding properties of the purified VH protein we performed several studies which assay the binding to antigen directly or by a secondary reagent in an indirect test. First, an ELISA assay was performed to titrate the binding of the VH protein to human IgG which is immobilized onto maxisorb ELISA plates. This is an indirect assay due to the fact that binding is being monitored by a secondary peroxidase-labeled antibody directed to the E-tag sequence at the carboxy terminus of the VH protein. As shown in Figure 7 A the VH binds human IgG in a dose dependent manner and VH protein concentrations as low as 1.7 ng/ml (100 pM) could be detected. When tested for specificity, the purified VH protein recognized a large variety of Ig's from different species and different isotypes. The results<br>
demonstrate that the VH recognizes specifically Ig's and that this recognition lies in the CHI or CL domains since the VH protein recognized a Fab fragment but not an Fv fragment (Figure 8A). No binding was detected on control antigens. To determine the binding affinity of the purified VH protein to its antigen we performed two types of binding assays. First, we performed a competition binding analysis with radiolabeled VH protein and second using real-time surface plasmon resonance (SPR) technology. In the competition binding analysis the iodinated VH protein was used as a tracer with increasing concentrations of competing unlabeled purified VH and tested for binding to immobilized human. Apparent binding affinity was -100 nM at which 50% inhibition of binding of iodinated VH protein had occurred (Figure 7B). For the real-time surface plasmon resonance measurements we efficiently coupled a mouse IgG monoclonal antibody to the dextran matrix of the biosensor by random coupling through free amino groups. To estimate the kinetic association and dissociation rate constants of the VH protein for the IgG. we injected homogeneously loaded, highly purified VH single-domain protein at different concentrations over an IgG surface (Figure 8B). By curve fitting to the dissociation (washout) phase of the binding curve, we determined the kinetic dissociation rate constant to be d = 4.13 xlO"3 s'1 corresponding to a tl/2 of 168 s. The association rate constant was determined to be ka =2.14xlO Ms"'. These values give a calculated equilibrium constant for dissociation, A'd = Ad/ka of 1.9 x 10"8 M. These results correlate with the results obtained in the competition binding assay and indicate that the VH protein has a good binding affinity to its IgG antigen.<br>
Example 7. Selection of phage clones specific for Streptavidin and characterization of their binding specificity.<br>
Method:<br>
Panning was performed as described in example 2, with magnetic-streptavidin-coated<br>
polystyrene beads (DYNAL)<br>
Results:<br>
After three cycles of panning, several phage clones were selected for binding to<br>
Streptavidin. Two clones were positive by ELISA as demonstrated in Figure 9, which<br>
shows that these phages bind Streptavidin and no other protein tested. The entire VH<br>
region of these phage clones was sequenced. The nine random amino acids of CDR3 that are encoded by the DNA of these clones are described in Table 3.<br>
: Amino acid composition of CDR3 region of phage clones specific to Streptavidin.<br><br>
(Table Remove)Interestingly, three amino acids of the sequence of clone No. 8 (His-Pro-Gln) are identical to the consensus sequence previously published which is specific to Streptavidin (Devlin et al. Science 249, 404-406. 1990). Various improvements of the binding affinity of the His-Pro-Gln sequence were made by several investigators by elongation of the binding sequence (Schmidt and Skerra Protein Engineering 6, 109-122, 1993), and peptide cyclization (Giebel et al. Biochemistry 34. 15430-15435,1995). Identification of the His-Pro-Gln consensus sequence as one of the Streptavidin binding motifs from our libraries of single-domains antibody heavy chain variable regions prove that these libraries are a powerful tool for identification of functional antibody single-domain molecules which bind to a desired constituent.<br>
Based on preliminary experiments the binding affinity of the clones identified by us is higher that the previous described sequences. This may be a result of the antibody frame in which the sequences are located or the sequences themselves.<br><br><br><br><br>
WE CLAIM:<br>
1.	A ligand comprising a first immunoglobulin single variable domiain<br>
having a first antigen or epitope binding specificity and a second<br>
immunoglobulin single variable domain having a second antigen or epitope<br>
binding specificity wherein one or both of said first and second variable<br>
domains bind to an antigen or epitope found in blood which increases the<br>
half-life of the ligand in vivo, and either<br>
(i)        the first and the second immunoglobulin variable domains are heavy<br>
chain variable domains; or (ii)        the first and the second immunoglobulin variable domains are light<br>
chain variable domains, provided that when the first and second immunoglobulin variable domains are Camelid VHH domains, the VHH domain which is specific for an antigen which increases the half-life of the ligand in vivo does not bind hen egg white lysozyme (HEL), porcine pancreatic alpha-amylase; NmC-A, hcg, BSA-linked RR6 azo dye or S.mutants HG982 cells.<br>
2.	A ligand as claimed in claim 1, wherein the ligand comprises first and second Camelid VHH domains.<br>
3.	A ligand as claimed in claim 1, wherein the first and/or second variable domain is human.<br>
4.	A ligand as claimed in claim 2 or 3, wherein the single variable domains are identical.<br>
5.	A ligand as claimed in any preceding claim, wherein the first and second domains bind independently, such that the dual specific ligand may simultaneously bind both the first and second epitopes or antigens.<br>
6.	A ligand as claimed in claim 5, wherein the dual specific ligand comprises a first form and a second form in equilibrium in solution, wherein<br><br>
both epitopes or antigens bind to the first form independently but compete for binding to the second form.<br>
7.	A ligand as claimed in any preceding claim, wherein said first and second epitopes are present on separate antigens.<br>
8.	A ligand as claimed in any one of claims 1 to 5, wherein said first and second epitopes are present on the same antigen.<br>
9.	A ligand as claimed in any preceding claim, wherein the variable domains are non-covalently associated.<br>
10.	A ligand as claimed in any one of claims 1 to 8, wherein the variable regions are covalently associated.<br>
11.	A ligand as claimed in claim 10, wherein the covalent association is mediated by disulphide bonds.<br>
12.	A ligand as claimed in any preceding claim, comprising a single variable domain specific for TNFα, which has a dissociation constant (Kd) of 1x10-8 M or less for human TNFα, and a Koff rate constant of 1x10-3s-1 or less, as determined by surface plasmon resonance.<br>
13.	A ligand as claimed in claim 12, wherein the domain specific for TNFa is a Vk.<br>
14.	A ligand as claimed in any one of claims 1 to 11, comprising a single variable domain specific for TNF receptor 1 (p55), which has a dissociation constant (Kd) of 1x10-8 M or less for human TNF receptor 1, and a Koff rate constant of 1x10-3s-1 or less, as determined by surface plasmon resonance.<br>
15.	A ligand as claimed in any one of claims 12 to 14, wherein the TNFa-or p55-  specific domain is capable of neutralizing human TNFα or TNF<br><br>
receptor 1 in a standard cell assay with a ND50 or 1C50 respectively of 50nM or less.<br>
16.	A ligand as claimed in any one of claims 1 to 11, comprising a single variable domain specific for TNF receptor 1 (p55), wherein the dAb is capable of antagonizing the activity of the TNF receptor 1 in a standard cell assay with an NDso of
17.	A ligand as claimed in any preceding claim, comprising a single variable domain specific for serum albumin (SA) which has a dissociation constant (Kd) of 1nM to 500µM for SA, as determined by surface plasmon resonance.<br>
18.	A ligand as claimed in claim 17, where the SA-specific domain binds SA in a standard ligand binding assay with an 1C50 of InM to 500µM.<br>
19.	A ligand as claimed in any one of claims 1 to 11, comprising a single variable domain specific for TNFa, and which comprises the amino acid sequence of TARl-5-19 (SEQ ID NO. 6) or a sequence that is at least 80% homologous thereto.<br>
20.	A ligand as claimed in any one of claims 1 to 11, comprising a single variable domain specific for TNFa, and which comprises the amino acid sequence of TAR1-5 (SEQ ID NO. 14) or a sequence that is at least 80% homologous thereto.<br>
21.	A ligand as claimed in any one of claims 1 to 11, comprising a single variable domain specific for TNFa, and which comprises the amino acid sequence of TAR1-27 (SEQ ID NO. 27) or a sequence that is at least 80% homologous thereto.<br>
22.	A ligand as claimed in any one of claims 1 to 11, comprising a single variable domain specific for TNF receptor 1, and which comprises the amino<br><br>
acid sequence of TAR2-10 (SEQ ID NO. 9) or a sequence that is at least 80% homologous thereto.<br>
23.	A ligand as claimed in any one of claims 1 to 11, comprising a single variable domain specific for TNF receptor 1, and which comprises the amino acid sequence of TAR2-10 (SEQ ID NO. 9) or a sequence that is at least 90% homologous thereto.<br>
24.	A ligand as claimed in any one of claims 1 to 11, comprising a single variable domain specific for TNF receptor 1, and which comprises the amino acid sequence of TAR2-5 (SEQ ID NO. 4) or a sequence that is at least 80% homologous thereto.<br>
25.	A ligand as claimed in any one of claims 1 to 11, comprising a single variable is domain specific for TNF receptor 1, and which comprises the amino acid sequence of TAR2-5 (SEQ ID NO. 4) or a sequence that is at least 90% homologous thereto.<br>
26.	A ligand as claimed in any one of claims 1 to 11, comprising a single variable domain specific for SA, and which comprises the amino acid sequence of MSA-16 identified by its sequence in figure 16 or a sequence that is at least 80% homologous thereto.<br>
27.	A ligand as claimed in any one of claims 1 to 11, comprising a single variable domain specific for SA, and which comprises the amino acid sequence of MSA-26 identified by its sequence in figure 16 or a sequence that is at last 80% homologous thereto.<br>
28.	A ligand as claimed in any one of claims 19 to 25, comprising an SA specific single variable domain as defined in claim 17 or 18.<br>
29.	A ligand as claimed in any one of claims 12 to 28, wherein the TNFα, TNF receptor 1 or SA is in human form.<br><br>
30.	A ligand comprising an anti-human TNF alpha dAb and an anti-SA dAb.<br>
31.	The ligand as claimed in claim 30, wherein the dAbs are Camelid VHH domains.<br>
32.	The ligand as claimed in claim 31, wherein the anti-TNF alpha dAb is a TNFa specific domain as defined in any one of claims 12, 13, 15 and 19 to 21.<br>
33.	The ligand as claimed in claim 31, wherein the anti-SA dAb is an SA-specific domain as defined in any one of claims 17, 18, 26 and 27.<br>
34.	A ligand as claimed in any preceding claim, which comprises a universal framework.<br>
35.	A ligand as claimed in any preceding claim comprising a VH framework selected from the group consisting of DP47, DP45 and DP38; and/or a VL framework which is DPK9.<br>
36.	A ligand as claimed in any preceding claim which comprises a binding site for a generic ligand.<br>
37.	The ligand as claimed in claim 36, wherein the generic ligand binding site is selected from the group consisting of protein A, protein L and protein G.<br>
38.	A ligand as claimed in any preceding claim wherein the ligand comprises a variable domain having one or more framework regions comprising an amino acid sequence that is the same as the amino acid sequence of a corresponding framework region encoded by a human germline antibody gene segment, or the amino acid sequences of one or more of said framework regions collectively comprises up to 5 amino acid<br><br>
differences relative to the amino acid sequence of said corresponding framework region encoded by a human germline antibody gene segment.<br>
39.	A ligand as claimed in any one of claims 1 to 37, wherein the ligand comprises a variable domain, wherein the amino acid sequences of FWl, FW2, FW3 and FW4 are the same as the amino acid sequences of corresponding framework regions encoded by a human germline antibody gene segment, or the amino acid sequences of FW 1, FW2, FW3 and FW4 collectively contain up to 10 amino acid differences relative to the amino acid sequences of corresponding framework regions encoded by said human germline antibody gene segment.<br>
40.	The ligand as claimed in claim 38 or claim 49, which comprises an antibody variable domain comprising FWl, FW2 and FW3 regions, and the amino acid sequences of said FWl, FW2 and FW3 are the same as the amino acid sequences of corresponding framework regions encoded by human germline antibody gene segments.<br>
41.	The ligand as claimed in any of claims 38 to 40, wherein said human germline antibody gene segment is selected from the group consisting of DP47, DP45, DP48 and DPK9.<br>
42.	A ligand as claimed in claim 1, wherein the or each heavy chain variable domain is not a Camelid immunoglobulin variable domain.<br>
43.	The ligand as claimed in claim 1, comprising at least one heavy chain variable domain that does not contain one or more amino acids that are specific to Camelid immunoglobulin variable domains as compared to human VH domains.<br>
44.	A ligand as claimed in any preceding claim, wherein the antigen or epitope found in blood is serum albumin.<br><br>
45.	A ligand as claimed in claim 1, wherein the first immunoglobulin<br>
single variable domain has binding specificity for serum albumin (SA), and<br>
the second immunoglobulin single variable domain has binding specificity<br>
for:<br>
(i) an antigen selected from the group consisting of EPO receptor, ApoE, Apo-SAA, BDNF, Cardiotrophin-1, EGF, EGF receptor, ENA-78, Eotaxin, Eotaxin-2, Exodus-2, EpoR, FGF-acidic, FGF-basic, fibroblast growth factor-10, FLT3 ligand, Fractalkine (CX3C), GDNF, G-CSF, GM-CSF, GF-βl, insulin, IFN-γ, IGF-I, IGF-II, IL-lα, IL-1β, IL-2, IL-3, IL-4, IL-5, IL-6, IL-7, IL-8 (72 a.a.), IL-8 (77 a.a.), IL-9, IL-10, IL-11, IL-12, IL-13, IL-15, IL-16, IL-17, IL-18 (IGIF), Inhibin a, Inhibin β, IP-10, keratinocyte growth factor-2 (KGF-2), KGF, Leptin, LIF, L5anphotactin, MuUerian inhibitory substance, monocyte colony inhibitory factor, monocyte attractant protein, M-CSF, MDC (67 a.a.), MDC (69 a.a.), MCP-1 (MCAF), MCP-2, MCP-3, MCP-4, MDC (67 a.a.), MDC (69 a.a), MIG, MlP-lα, MIP-1β, MIP-3α, MIP-3β, MIP-4, myeloid progenitor inhibitor factor-1 (MPIF-1), NAP-2, Neurturin, Nerve growth factor, p-NGF, NT-3, NT-4, Oncostatin M, PDGF-AA, PDGF-AB, PDGF-BB, PF-4, RANTES, SDFlα, SDF1β, SCF, SCGF, stem cell factor (SCF), TARC, TGF-α, TGF-β, TGF-β2, TGF-β3, tumour necrosis factor (TNF), TNF-α, TNF-β, TNF receptor I, TNF receptor II, TNIL-1, TPO, VEGF, VEGF receptor 1, VEGF receptor 2, VEGF receptor 3, GCP-2, GRO/MGSA, GRO-β, GRO-Y, HCCl, 1-309, HER 1, HER 2, HER 3 and HER 4.<br>
46.	A ligand as claimed in claim 1, wherein the first immunoglobulin single variable domain has binding specificity for serum albumin (SA), and the second immunoglobulin single variable domain has binding specificity for a cytokine.<br>
47.	A ligand as claimed in claim 1, wherein the first immunoglobulin single variable domain has binding specificity for serum albumin (SA), and the second immunoglobulin single variable domain has binding specificity for a cytokine receptor.<br><br>
48.	A ligand as claimed in claim 1, wherein the first immunoglobulin single variable domain has binding specificity for serum albumin (SA), and the second immunoglobulin single variable domain has binding specificity for a growth factor.<br>
49.	A ligand as claimed in claim 1, wherein the first immunoglobulin single variable domain has binding specificity for serum albumin (SA), and the second single immunoglobulin variable domain has binding specificity for a receptor for a cytokine listed as claimed in claim 62.<br>
50.	The ligand as claimed in any one of claims 45 to 49, wherein the anti-SA domain is as defined in any one of claims 17, 18, 26 and 27.<br>
51.	A ligand as claimed in claim 14 or 16, wherein the TNF receptor 1 is in human form.<br>
52.	A ligand as claimed in claim 17, wherein the SA is in human form.<br>
53.	The ligand as claimed in claim 39, wherein the said human germline antibody gene segment is selected from the group consisting of DP47, DP45, DP48 and DPK9.<br>
54.	A ligand as claimed in any preceding claim for use in therapy.<br>
55.	A ligand as claimed in any of the preceding claims as and when used in a pharmaceutical composition.<br>
56.	A method for producing a ligand as claimed in claim 1, comprising a first immunoglobulin single variable domain having a first binding specificity and a second single immunoglobulin single variable domain having a second binding specificity, one or both of the binding specificities being specific for a protein found in blood which increases the half-life of the ligand in vivo, the method comprising the steps of:<br>
selecting a first variable domain by its ability to bind to a first epitope,<br><br>
(a)	selecting a second variable region by its ability to bind to a second epitope,<br>
(b)	combining the variable regions; and<br>
(c)	selecting the ligand by its ability to bind to said first and second epitopes.<br><br>
57.	A method as claimed in claim 56, wherein said first variable domain is selected for binding to said first epitope in absence of a complementary variable domain.<br>
58.	A method as claimed in claim 56, wherein said first variable domain is selected for binding to said first epitope in the presence of a third complementary variable domain in which said third variable domain is different from said second variable domain.<br>
59.	Nucleic acid encoding at least a ligand as claimed in any one of claims 1 to 53.<br>
60.	A nucleic acid as claimed in claim 59, comprising the nucleic acid sequence of TARl-5-19 (SEQ ID NO. 5) or a sequence that is at least 70% homologous thereto.<br>
61.	A nucleic acid as claimed in claim 59, comprising the nucleic acid sequence of TAR 1-5 (SEQ ID NO. 13) or a sequence that is at least 70% homologous thereto.<br>
62.	A nucleic acid as claimed in claim 59, comprising the nucleic acid sequence of TAR1-27 (SEQ ID NO. 26) or a sequence that is at least 70% homologous thereto.<br>
63.	A nucleic acid as claimed in claim 59, comprising the nucleic acid sequence of TAR2-10 (SEQ ID NO. 8) or a sequence that is at least 70% homologous thereto.<br><br>
64.	A nucleic acid as claimed in claim 59, comprising the nucleic acid sequence of TAR2-10 (SEQ ID NO. 8) or a sequence that is at least 80% homologous thereto.<br>
65.	A nucleic acid as claimed in claim 59, comprising the nucleic acid sequence of TAR2-5 (SEQ ID NO. 3) or a sequence that is at least 70% homologous thereto.<br>
66.	A nucleic acid as claimed in claim 59, comprising the nucleic acid sequence of TAR2-5 (SEQ ID NO. 3) or a sequence that is at least 80% homologous thereto.<br>
67.	A nucleic acid as claimed in claim 59, comprising the nucleic acid sequence of MSA-16 or a sequence that is at least 70% homologous thereto.<br>
68.	A nucleic acid as claimed in claim 59, comprising the nucleic acid sequence of MSA-26 identified by its sequence in figure 16 or a sequence that is at least 70% homologous thereto.<br>
69.	A vector comprising nucleic acid as claimed in any one of claims 59 to 68.<br>
70.	A vector as claimed in claim 69, further comprising com.ponents necessary for the expression of a dual-specific ligand.<br>
71.	A prokaryotic host cell or COS cell comprising a vector as claimed in claim 69 or 70.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1ERUxOUC0yMDA0LUFic3RyYWN0LSgxMi0wMy0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">4057-DELNP-2004-Abstract-(12-03-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1ERUxOUC0yMDA0LUFic3RyYWN0LSgyNS0wOC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">4057-DELNP-2004-Abstract-(25-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA0LWFic3RyYWN0LSgzMS0wMy0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">4057-delnp-2004-abstract-(31-03-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA0LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">4057-delnp-2004-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1ERUxOUC0yMDA0LUNsYWltcy0oMTItMDMtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">4057-DELNP-2004-Claims-(12-03-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1ERUxOUC0yMDA0LUNsYWltcy0oMjUtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">4057-DELNP-2004-Claims-(25-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA0LWNsYWltcy0oMzEtMDMtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">4057-delnp-2004-claims-(31-03-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA0LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">4057-delnp-2004-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA0LWNvbXBsZXRlIHNwZWNpZmljYXRpb24gKGdyYW50ZWQpLnBkZg==" target="_blank" style="word-wrap:break-word;">4057-delnp-2004-complete specification (granted).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTItMDMtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">4057-DELNP-2004-Correspondence-Others-(12-03-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjUtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">4057-DELNP-2004-Correspondence-Others-(25-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA0LWNvcnJlc3BvbmRlbmNlLW90aGVycy0oMzEtMDMtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">4057-delnp-2004-correspondence-others-(31-03-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA0LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">4057-delnp-2004-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA0LWRlc2NyaXB0aW9uIChjb21wbGV0ZSktMjUtMDgtMjAwOC5wZGY=" target="_blank" style="word-wrap:break-word;">4057-delnp-2004-description (complete)-25-08-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA0LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">4057-delnp-2004-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA0LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">4057-delnp-2004-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA0LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">4057-delnp-2004-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA0LWZvcm0tMTMucGRm" target="_blank" style="word-wrap:break-word;">4057-delnp-2004-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA0LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">4057-delnp-2004-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA0LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">4057-delnp-2004-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1ERUxOUC0yMDA0LUZvcm0tMy0oMjUtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">4057-DELNP-2004-Form-3-(25-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA0LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">4057-delnp-2004-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA0LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">4057-delnp-2004-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1ERUxOUC0yMDA0LUdQQS0oMjUtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">4057-DELNP-2004-GPA-(25-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA0LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">4057-delnp-2004-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA0LXBjdC0zMDYucGRm" target="_blank" style="word-wrap:break-word;">4057-delnp-2004-pct-306.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA0LXBjdC00MDkucGRm" target="_blank" style="word-wrap:break-word;">4057-delnp-2004-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA0LXBjdC00MTYucGRm" target="_blank" style="word-wrap:break-word;">4057-delnp-2004-pct-416.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA0LXBjdC1zZWFyY2ggcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">4057-delnp-2004-pct-search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1ERUxOUC0yMDA0LVBldGl0aW9uLTEzNy0oMjUtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">4057-DELNP-2004-Petition-137-(25-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1ERUxOUC0yMDA0LVBldGl0aW9uLTEzOC0oMjUtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">4057-DELNP-2004-Petition-138-(25-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QuanBn" target="_blank" style="word-wrap:break-word;">abstract.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="233599-crystalline-carvedilol-dihydrogen-phosphate-hemihydrate-and-pharmaceutical-composition-comprising-the-same.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="233601-method-and-system-for-compositing-three-dimensional-graphics-images-using-associative-decision-mechanism.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>233600</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>4057/DELNP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>14/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>31-Mar-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>20-Dec-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>DOMANTIS LIMITED</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>315 CAMBRIDGE SCIENCE PARK, CAMBRIDGE CB4 0WG, U.K.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>GREG WINTER</td>
											<td>MRC LABORATORY OF MOLECULAR BIOLOGY, HILLS ROAD, CAMBRIDGE CB2 2QH, ENGLAND</td>
										</tr>
										<tr>
											<td>2</td>
											<td>LUCY HOLT</td>
											<td>DOMANTIS LIMITED GRANTA PARK, ABINGTON, CAMBRIDGE CB1 6GS</td>
										</tr>
										<tr>
											<td>3</td>
											<td>ELENA DE ANGELIS</td>
											<td>DOMANTIS LIMITED GRANTA PARK, ABINGTON, CAMBRIDGE CB1 6GS ENGLAND</td>
										</tr>
										<tr>
											<td>4</td>
											<td>IAN TOMLINSON</td>
											<td>DOMANTIS LIMITED, GRANTA PARK, ABINGTON, CAMBRIDGE CB1 6GS, ENGLAND.</td>
										</tr>
										<tr>
											<td>5</td>
											<td>OLGA IGNATOVICH</td>
											<td>DOMANTIS LIMITED GRANTA PARK, ABINGTON, CAMBRIDGE CB1 6GS ENGLAND</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 16/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/GB03//02804</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-06-30</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>PCT/GB03/03014</td>
									<td>2002-06-28</td>
								    <td>U.K.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>0230202.4</td>
									<td>2002-12-27</td>
								    <td>U.K.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/233600-ligand by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:47:48 GMT -->
</html>
