<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/233693-method-for-quality-control-of-an-attenuated-varicella-live-vaccine by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:50:50 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 233693:METHOD FOR QUALITY CONTROL OF AN ATTENUATED VARICELLA LIVE VACCINE .</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHOD FOR QUALITY CONTROL OF AN ATTENUATED VARICELLA LIVE VACCINE .</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Disclosed is a method for quality control of an attenuated varicella live vaccine, which comprises analyzing the genomic DNA of a sample varicella vaccine virus, wherein the sample varicella vaccine virus is a virus for use as an active ingredient of an attenuated varicella live vaccine; and confirming that the genomic DNA of the sample varicella vaccine virus conserves the 5,745th G, the 105,356th C, the 105,544th G, the 106,262nd C and the 107,252nd C without suffering mutation, wherein the nucleotide numbers are in accordance with the nucleotide numbering system of the nucleotide sequence of the genomic DNA of the varicella virus Dumas strain of SEQ ID NO: 1.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>TITLE OF THE INVENTION<br>
Method for quality control of an attenuated<br>
varicella live vaccine<br>
BACKGROUND OF THE INVENTION<br>
Field of the Invention<br>
The present invention relates to a method for<br>
quality control of an attenuated varicella live vaccine.<br>
More particularly, the present invention relates to a<br>
method for quality control of an attenuated varicella<br>
live vaccine, which comprises subjecting the genomic<br>
DNA of a sample varicella vaccine virus to sequence<br>
analysis and confirming that the genomic DNA of the<br>
sample varicella vaccine virus conserves the specific<br>
nucleotides without suffering mutation. By the use of<br>
the method of the present invention, it has become pos-<br>
sible to determine very accurately the qualification of<br>
an attenuated varicella virus as an active ingredient<br>
of an attenuated varicella live vaccine and, conse-<br>
quently, to conduct an exact quality control of the<br>
vaccines.<br>
Prior Art<br>
As is well known, attenuated varicella live vac-<br>
cines used today are produced from a seed strain of<br>
varicella virus which is derived from the attenuated<br>
varicella virus Oka strain (see Examined Japanese Pat-<br>
ent Application Publication No. 53-41202 and U.S. Pat-<br>
ent No. 3,985,615), and the attenuated live vaccines<br>
are used widely throughout the world (Requirements for<br>
Varicella Vaccine (Live) Adopted 1984; Revised 1993:<br>
WHO Technical Report Series, No. 848, pp. 22-38, 1994).<br>
To ensure the safety and effectiveness of the vaccine,<br>
the number of passages of a virus used for producing<br>
the vaccine is restricted under the control of a seed<br>
lot system, taking into consideration the potential ge-<br>
netic mutation which is likely to occur during the pas-<br>
sage. That is, the manufacturers are under an obliga-<br>
tion to produce varicella vaccines only from the virus<br>
derived from the approved seed virus for the live vari-<br>
cella vaccine, wherein the number of passages of the<br>
virus is not more than 10 as counted from the approved<br>
seed virus which is counted as 0 passage. In other<br>
words, the quality control and quality assurance of the<br>
attenuated varicella live vaccine rely upon the ful-<br>
fillment of the seed lot system by the manufacturers,<br>
and such a method for the quality control and quality<br>
assurance is not a method which can be traced and ana-<br>
lyzed by a person skilled in the art.<br>
Further, from the viewpoint of epidemiology which<br>
involves a tracing of the effects of the varicella vac-<br>
cine and a post-market surveillance (PMS), the vi-<br>
rological difference between the fresh wild-type<br>
strains isolated from the naturally infected varicella<br>
patients and the vaccine virus strains derived from the<br>
above-mentioned Oka strain needs to be determined, and<br>
various analyses, such as those utilizing immunological<br>
techniques and genetic engineering techniques, have<br>
been attempted for determination of the virological<br>
difference. For example, the following analyses have<br>
been reported: the difference in DNA sequence between<br>
the various VZV strains (Journal of Virology, 59, 660-<br>
668, 1986; and <slittoa the differ-></slittoa>
ence in the absence or presence of a restriction enzyme<br>
Pst I cleavage site (Japanese Journal of Experimental<br>
Medicine, 59, 233-237, 1989), the difference in RFLP<br>
(restriction fragment length polymorphism) of the PCR<br>
(polymerase chain reaction) product (Journal of Virol-<br>
ogy, 66, 1016-1020, 1992), and the difference in the<br>
absence or presence of a restriction enzyme Pst I re-<br>
striction site which is taken in combination with the<br>
difference in RFLP of the PCR product (Journal of<br>
Clinical Microbiology, 33, 658-660, 1995). However,<br>
all of these analyses only propose criteria which can<br>
be used for differentiating a fresh wild-type strain<br>
from a vaccine strain derived from the Oka strain, and<br>
such analyses lack reliability and exactness. In addi-<br>
tion, a method for identifying the attenuated varicella<br>
virus Oka strain by using gene 14 region (U.S. Patent<br>
No. 6,093,535) and a method for identifying the attenu-<br>
ated varicella live vaccine virus by using gene 62 re-<br>
gion (International Patent Application Publication No.<br>
WO 00/50603) have been known. Both of these methods<br>
enabled a determination of the differences among the<br>
varicella virus Oka strain (virulent parental strain),<br>
a vaccine strain derived therefrom (attenuated Oka<br>
strain) and a varicella virus strain other than the Oka<br>
strain, but neither of these methods was satisfactory<br>
as a standard for the quality control and quality as-<br>
surance of the attenuated varicella live vaccine.<br>
As mentioned above, at present, the quality of the<br>
attenuated varicella virus used as an active ingredient<br>
of an attenuated varicella live vaccine is controlled<br>
by the fulfillment of the seed lot system by the manu-<br>
facturers . In other words, a method which can be<br>
traced and analyzed by a third party for evaluating and<br>
confirming the effectiveness of the vaccine, such as a<br>
method utilizing a direct and quantitative genetic ana-<br>
lysis of the genomic DNA of a seed virus or a vaccine<br>
virus, has not been used for the quality control of the<br>
vaccine and, thus, the exactness of the quality control<br>
is incomputable and vague. Therefore, an improvement<br>
in the exactness of the quality control and quality as-<br>
surance is critically important for assuring the effec-<br>
tiveness, safety and uniformity of the attenuated vari-<br>
cella live vaccine. However, as mentioned above, a re-<br>
liable method for the quality control has not been es-<br>
tablished, and a development of such a method has been<br>
earnestly desired in the art.<br>
SUMMARY OF THE INVENTION<br>
In the above situation, the present inventors have<br>
made extensive and intensive studies with a view toward<br>
developing a novel method for accurately and quantita-<br>
tively conducting the quality control of an attenuated<br>
varicella live vaccine. Specifically, the present in-<br>
ventors determined the whole genomic nucleotide se-<br>
quence of the attenuated varicella virus Oka strain<br>
containing more than 120,000 nucleotides, conducted a<br>
comparative analysis between the determined nucleotide<br>
sequence of the attenuated Oka strain and the whole ge-<br>
nomic nucleotide sequences of the virulent strain and<br>
the parental Oka strain (virulent strain), and identi-<br>
fied the genetic mutations of the attenuated varicella<br>
virus Oka strain. As a result, they have found that,<br>
by evaluating and determining whether or not a varicel-<br>
la virus strain conserves the below-mentioned specific<br>
nucleotides, a virus strain conserving the specific nu-<br>
cleotides can be determined accurately as a virus<br>
strain capable of functioning as an attenuated varicel-<br>
la vaccine virus. The present invention has been com-<br>
pleted, based on this novel finding.<br>
Therefore, it is an object of the present inven-<br>
tion to provide a novel method for the quality control<br>
of an attenuated varicella live vaccine.<br>
It is another object of the present invention to<br>
provide an attenuated varicella live vaccine which is<br>
quality-controlled by the above-mentioned method.<br>
It is a further object of the present invention to<br>
provide a vaccine strain capable of functioning as an<br>
attenuated varicella vaccine virus, which is identified<br>
by a method used in the above-mentioned method.<br>
The foregoing and other objects, features and ad-<br>
vantages of the present invention will be apparent to<br>
those skilled in the art from the following detailed<br>
description and the appended claims taken in connection<br>
with the accompanying sequence listing and drawings.<br>
SEQUENCE LISTING FREE TEXT<br>
SEQ ID NOs: 3 and 4 are of PCR primers used for<br>
detecting a mutation of the 560th nucleotide of a vari-<br>
cella vaccine virus.<br>
SEQ ID NOs: 5 and 6 are of PCR primers used for<br>
detecting a mutation of the 5,745th nucleotide of a<br>
varicella vaccine virus.<br>
SEQ ID NOs: 7 and 8 are of PCR primers used for<br>
detecting a mutation of the 26,125th nucleotide of a<br>
varicella vaccine virus.<br>
SEQ ID NOs: 9 and 10 are of PCR primers used for<br>
detecting a mutation of the 94,167th nucleotide of a<br>
varicella vaccine virus.<br>
SEQ ID NOs: 11 and 12 are of PCR primers used for<br>
detecting mutations of the 105,356th, 105,544th,<br>
124,353rd and 124,541st nucleotides of a varicella vac-<br>
cine virus.<br>
SEQ ID NOs: 13 and 14 are of PCR primers used for<br>
detecting mutations of the 105,705th, 106,262nd,<br>
123,635th and 124,192nd nucleotides of a varicella vac-<br>
cine virus.<br>
SEQ ID NOs: 15 and 16 are of PCR primers used for<br>
detecting mutations of the 107,136th, 107,252nd,<br>
122,645th and 122,761st nucleotides of a varicella vac-<br>
cine virus.<br>
SEQ ID NOs: 17 and 18 are of PCR primers used for<br>
detecting mutations of the 108,111st and 121,786th nu-<br>
cleotides of a varicella vaccine virus.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
In the drawings:<br>
Fig. 1 is a genetic map of the varicella virus Oka<br>
strain showing the number and direction of each gene,<br>
wherein, s represents a synonymous substitution, ? rep-<br>
resents a nonsynonymous substitution, s represents a<br>
mutation in a noncoding region, O represents deletion<br>
or insertion, the genome length is shown every 20 kb,<br>
R1 to R4 represent repetitive sequences, Ori represents<br>
an origin of replication, TRL represents a "Terminal<br>
Repeat Long", UL represents a "Unique Long", IRL repre-<br>
sents an "Internal Repeat Long", IRS represents an "In-<br>
ternal Repeat Short", US represents a "Unique Short",<br>
and TRS represents a "Terminal Repeat Short"; and<br>
wherein the nucleotide sequence of gene 62 to gene 64<br>
and the nucleotide sequence of gene 69 to gene 71 are<br>
symmetrical to each other (i.e., the two nucleotide se-<br>
quences are inverted repeats); and<br>
Fig. 2 shows the electropherograms which are the<br>
results of the RFLP analyses conducted in Example 4,<br>
wherein, the restriction enzymes used for treating the<br>
PCR products are as follows: Fig. 2(a) was obtained us-<br>
ing Nla III, Fig. 2(b) was obtained using Alu I, Fig.<br>
2(c) was obtained using BstX I, Fig. 2(d) was obtained<br>
using SfaN I, Fig. 2(e) was obtained using Ace II, Fig.<br>
2(f) was obtained using Sac II, Fig. 2(g) was obtained<br>
using Sma I, Fig. 2(h) was obtained using BssH II and<br>
Nae I in combination, and Fig. 2(h) was obtained using<br>
Bsr I; and wherein, V represents the attenuated Oka<br>
strain, P represents the parental Oka strain, and K<br>
represents the Kawaguchi strain.<br>
The terminologies used in the present specifica-<br>
tion are defined in the following items (a) to (g).<br>
(a) VZV: A virus which causes varicella and her-<br>
pes zoster. "VZV" is an abbreviation for "varicella-<br>
zoster virus" which is frequently referred to simply as<br>
"varicella virus".<br>
(b) Varicella vaccine virus and varicella vaccine:<br>
A varicella vaccine virus is an active ingredient of a<br>
vaccine and it is an attenuated virus. A varicella<br>
vaccine is a vaccine effective for preventing the in-<br>
fection with a VZV or the onset of the disease after<br>
the infection.<br>
(c) Attenuated Oka strain: Attenuated Oka strain<br>
is the attenuated varicella virus Oka strain (see Exam-<br>
ined Japanese Patent Application Publication 53-41202<br>
and U.S. Patent No. 3,985,615) or an attenuated vari-<br>
cella virus derived therefrom. The attenuated Oka<br>
strain is deposited under the deposition number VR-795<br>
on March 14, 1975 with ATCC.<br>
(d) Parental Oka strain: Parental Oka strain is<br>
the originally isolated, wild-type (virulent) varicella<br>
virus Oka strain.<br>
(e) Quality control: For assuring the effective-<br>
ness, safety and uniformity of a vaccine, raw materials<br>
for a vaccine, intermediates obtained during the pro-<br>
duction of a vaccine, and final products are subjected<br>
to various tests or analyses for confirming and assur-<br>
ing their qualification as a vaccine. With respect to<br>
an attenuated varicella live vaccine, at present, the<br>
quality control of the vaccine is conducted in accor-<br>
dance with Pharmaceutical Affairs Law (the Law No. 145<br>
established in 1960), Article 42, Item 1 and a provi-<br>
sion entitled "Dried Attenuated Varicella Virus Live<br>
Vaccine" in the Notification No. 217 of the Japanese<br>
Ministry of Health and Welfare: Seibutsugakuteki Seizai<br>
Kijun (Minimum Requirements for Biological Products) or<br>
the above-mentioned "Requirements for Varicella Vaccine<br>
(Live)" of WHO.<br>
(f) Nucleotide number of a DNA sequence: In the<br>
present invention, all nucleotide numbers of the vari-<br>
cella viruses are in accordance with the nucleotide<br>
numbering system of the nucleotide sequence of the ge-<br>
nomic DNA of the varicella virus Dumas strain (Journal<br>
of General Virology, 67, 1759-1816, 1986 and GenBank<br>
(National Center for Biotechnology Information, Nation-<br>
al Library of Medicine, Building 38A, Room 8N805, Be-<br>
thesda, MD 20894, USA), Accession No. X04370) which is<br>
shown in SEQ ID NO: 1. Further, in the present inven-<br>
tion, the nucleotide sequences are the sequences of a<br>
sense strand unless otherwise specified.<br>
(g) DNA mutation: Mutations in the genomic DNA<br>
of the attenuated Oka strain were identified by con-<br>
ducting homology searches among the nucleotide se-<br>
quences of the attenuated Oka strain, the Dumas strain<br>
and the virulent parental Oka strain. For example, the<br>
DNA mutation is described as follows: "The nucleotide A<br>
which is the 5,745th nucleotide of the Dumas strain and<br>
a nucleotide at a corresponding site of the parental<br>
Oka strain has been mutated to G in the attenuated Oka<br>
strain. This nucleotide mutation is a nonsynonymous<br>
substitution in which Ser is replaced with Pro."<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
In an aspect of the present invention, there is<br>
provided an accurate method for the quality control of<br>
an attenuated varicella live vaccine.<br>
For easy understanding of the present invention,<br>
the essential features and various embodiments of the<br>
present invention are enumerated below.<br>
1. A method for the quality control of an attenuated<br>
varicella live vaccine, which comprises subjecting the<br>
genomic DNA of a sample varicella vaccine virus to se-<br>
quence analysis and confirming that the genomic DNA of<br>
the sample varicella vaccine virus conserves without<br>
suffering mutation the following 5 nucleotides:<br>
the 5,745th G, the 105,356th C, the 105,544th G,<br>
the 106,262nd C and the 107,252nd C,<br>
wherein the nucleotide numbers are in accor-<br>
dance with the nucleotide numbering system of<br>
the nucleotide sequence of the genomic DNA of<br>
the varicella virus Dumas strain of SEQ ID NO:<br>
1.<br>
2. The method according to item 1 above, wherein the<br>
conservation of the 5 nucleotides combination is con-<br>
firmed by an RFLP analysis using the following primers:<br>
a pair of primers of SEQ ID NOs: 5 and 6 with<br>
respect to the confirmation of the 5,745th G;<br>
a pair of primers of SEQ ID NOs: 11 and 12 with<br>
respect to the confirmation of the 105,356th C<br>
and the 105,544th G;<br>
a pair of primers of SEQ ID NOs: 13 and 14 with<br>
respect to the confirmation of the 106,262nd C;<br>
and<br>
a pair of primers of SEQ ID NOs: 15 and 16 with<br>
respect to the confirmation of the 107,252nd C.<br>
3. The method according to item 1 or 2 above, which<br>
further comprises confirming that the the genomic DNA<br>
of sample varicella vaccine virus conserves without<br>
suffering mutation the following 4 nucleotides:<br>
the 122,645th G, the 123,635th G, the 124,353rd<br>
C and the 124,541st G,<br>
wherein the nucleotide numbers are in accor-<br>
dance with the nucleotide numbering system of<br>
the nucleotide sequence of the genomic DNA of<br>
the varicella virus Dumas strain of SEQ ID NO:<br>
1.<br>
4. The method according to item 3 above, wherein the<br>
conservation of the 4 nucleotides is confirmed by an<br>
RFLP analysis using the following primers:<br>
a pair of primers of SEQ ID NOs: 11 and 12 with<br>
respect to the confirmation of the 124,353rd C<br>
and the 124,541st G;<br>
a pair of primers of SEQ ID NOs: 13 and 14 with<br>
respect to the confirmation of the 123,635th G;<br>
and<br>
a pair of primers of SEQ ID NOs: 15 and 16 with<br>
respect to the confirmation of the 122,645th G.<br>
5. The method according to any one of items 1 to 4<br>
above, which further comprises confirming that the ge-<br>
nomic DNA of the sample varicella vaccine virus con-<br>
serves without suffering mutation the following 49 nu-<br>
cleotides:<br>
the 560th C, the 703rd Y, the 763rd Y, the<br>
2,515th Y, the 10,900th Y, the 12,779th Y, the<br>
19,431st Y, the 26,125th G, the 31,732nd Y, the<br>
38,036th Y, the 39,227th K, the 58,595th R, the<br>
59,287th R, the 64,067th R, the 71,252nd Y, the<br>
82,225th R, the 84,091st R, the 87,280th R, the<br>
87,306th Y, the 89,734th R, the 90,535th R, the<br>
94,167th C, the 97,748th R, the 97,796th Y, the<br>
101,089th R, the 105,169th R, the 105,310th R,<br>
the 105,705th C, the 106,710th R, the 107,136th<br>
C, the 107,599th R, the 107,797th R, the<br>
108,111st C, the 108,838th R, the 109,137th R,<br>
the 109,200th R, the 111,650th R, the 118,247th<br>
Y, the 120,697th Y, the 120,760th Y, the<br>
121,059th Y, the 121,786th G, the 122,100th Y,<br>
the 122,298th Y, the 122,761st G, the 123,187th<br>
Y, the 124,192nd G, the 124,587th Y and the<br>
124,728th Y,<br>
wherein:<br>
the nucleotide numbers are in accordance with<br>
the nucleotide numbering system of the nucleo-<br>
tide sequence of the genomic DNA of the vari-<br>
cella virus Dumas strain of SEQ ID NO: 1,<br>
R represents A or G,<br>
Y represents C or T, and<br>
K represents G or T.<br>
6. The method according to item 5 above, wherein the<br>
conservation of the 560th C, the 26,125th G, the<br>
94,167th C, the 105,705th C, the 107,136th C, the<br>
108,111st C, the 121,786th G, the 122,761st G and the<br>
124,192nd G among the 49 nucleotides is confirmed by an<br>
RFLP analysis using the following primers:<br>
a pair of primers of SEQ ID NOs: 3 and 4 with<br>
respect to the confirmation of the 560th C;<br>
a pair of primers of SEQ ID NOs: 7 and 8 with<br>
respect to the confirmation of the 26,125th G;<br>
a pair of primers of SEQ ID NOs: 9 and 10 with<br>
respect to the confirmation of the 94,167th C;<br>
a pair of primers of SEQ ID NOs: 13 and 14 with<br>
respect to the confirmation of the 105,705th C<br>
and the 124,192nd G;<br>
a pair of primers of SEQ ID NOs: 15 and 16 with<br>
respect to the confirmation of the 107,136th C<br>
and the 122,761st G; and<br>
a pair of primers of SEQ ID NOs: 17 and 18 with<br>
respect to the confirmation of the 108,111st C<br>
and the 121.786th G.<br>
7. The method according to any one of items 1 to 6<br>
above, which further comprises confirming deletion mu-<br>
tations in two origins of replication of the genomic<br>
DNA of the sample varicella vaccine virus,<br>
wherein the two origins of replication are a re-<br>
gion corresponding to the 110,087th to 110,350th nu-<br>
cleotides of the sense strand of the genomic DNA of the<br>
varicella virus Dumas strain of SEQ ID NO: 1 and a re-<br>
gion corresponding to the 119,547th to 119,810th nu-<br>
cleotides of the genomic DNA of the antisense strand of<br>
the Dumas strain, and<br>
wherein the deletion mutations occur with respect<br>
to segments each having a nucleotide sequence of<br>
ATATATATA arranged in the direction of from the 5' end<br>
to the 3' end, the segments being a segment correspond-<br>
ing to the 110,219th to 110,227th nucleotides of the<br>
genomic DNA of the sense strand of the Dumas strain and<br>
a segment corresponding to the 119,670th to 119,678th<br>
nucleotides of the antisense strand of the genomic DNA<br>
of the Dumas strain.<br>
8. The method according to any one of items 1 to 7<br>
above, which further comprises confirming that the re-<br>
petitive sequence of one whole Rl region of the genomic<br>
DNA of the sample varicella vaccine virus is a nucleo-<br>
tide sequence of abbabba'bbb'abababx arranged in the<br>
direction of from the 5' end to the 3' end,<br>
wherein:<br>
a represents a nucleotide sequence of<br>
GGACGCGATCGACGACGA;<br>
a' represents a nucleotide sequence of<br>
GGACGCGATTGACGACGA;<br>
b represents a nucleotide sequence of<br>
GGGAGAGGCGGAGGA;<br>
b' represents a nucleotide sequence of<br>
GGACGCGGCGGAGGA; and<br>
x represents a nucleotide sequence of GGA,<br>
wherein the whole Rl region is a region corre-<br>
sponding to the 13,937th to 14,242nd nucleotides of the<br>
genomic DNA of the varicella virus Dumas strain of SEQ<br>
ID NO: 1.<br>
9. The method according to any one of items 1 to 8<br>
above, which further comprises confirming that the re-<br>
petitive sequence of each of two whole R4 regions of<br>
the genomic DNA of the sample varicella vaccine virus<br>
is a nucleotide sequence of aaaaaaaaaaaax arranged in<br>
the direction of from the 5' end to the 3' end,<br>
wherein:<br>
a represents a nucleotide sequence of<br>
CCCCGCCGATGGGGAGGGGGCGCGGTA; and<br>
x represents a nucleotide sequence of<br>
CCCCGCCGATG,<br>
wherein the two whole R4 regions are a region cor-<br>
responding to the 109,762nd to 109,907th nucleotides of<br>
the sense strand of the genomic DNA of the varicella<br>
virus Dumas strain of SEQ ID NO: 1 and a region corre-<br>
sponding to the 119,990th to 120,135th nucleotides of<br>
the genomic DNA of the antisense strand of the Dumas<br>
strain.<br>
10. An attenuated varicella live vaccine which is<br>
quality-controlled by the method of any one of items 1<br>
to 9 above.<br>
11. A virus strain capable of functioning as an at-<br>
tenuated varicella vaccine virus, which is identified<br>
by a method used in the method of any one of items 1 to<br>
9 above.<br>
The present invention is described in detail below.<br>
During the course of studies for completing the<br>
method of the present invention, the present inventors<br>
determined for the first time the whole nucleotide se-<br>
quence of the genomic DNA of the attenuated Oka strain<br>
(deposited under the deposition number VR-795 on March<br>
14, 1975 with ATCC (American Type Culture Collection;<br>
10801 University Boulevard, Manassas, VA 20110-2209,<br>
USA)). This sequence is shown in SEQ ID NO:2. Further,<br>
using the determined whole genomic DNA sequence of the<br>
attenuated Oka strain, the present inventors conducted<br>
a homology search among the whole genomic DNA sequences<br>
of the Dumas strain, the parental Oka strain and the<br>
attenuated Oka strain. As a result, the present inven-<br>
tors disclosed the nucleotide mutations of the attenu-<br>
ated Oka strain (i.e., the nucleotides of the attenuat-<br>
ed Oka strain which are different from the nucleotides<br>
of the Dumas strain and/or the parental Oka strain at<br>
the corresponding sites) shown in Table 1 below. The<br>
present inventors made further analyses of the nucleo-<br>
tide mutations and identified the synonymous substitu-<br>
tions (no amino acid replacement resulting from the nu-<br>
cleotide mutations) and nonsynonymous substitutions<br>
(amino acid replacements resulting from the nucleotide<br>
mutations); mutations in noncoding regions (ncr muta-<br>
tion); stop codon mutations (ochre/amber mutation);<br>
number of repetitions and the sequence size of the re-<br>
petitive sequences and the differences in the order of<br>
the repetitions; and mutations in the origins of repli-<br>
cation (inverted repeats; see Fig. 1). As explained in<br>
detail below, the mutations of the attenuated Oka<br>
strain which have been disclosed by the present inven-<br>
tors are useful for differentiating the attenuated Oka<br>
strain from other varicella virus strains, especially<br>
from the virulent strains and, therefore, theses muta-<br>
tions can be used for the quality control of the at-<br>
tenuated varicella live vaccine.<br>
Among the nucleotide mutations of the attenuated<br>
Oka strain shown in Table 1, the important mutations<br>
show the XXY pattern or the XX(X/Y) pattern. The muta-<br>
tion showing the XXY pattern is a mutation wherein a<br>
nucleotide of the parental Oka strain is identical to a<br>
corresponding nucleotide of the Dumas strain (that is.<br>
both nucleotides are "X"), but the corresponding nu-<br>
cleotide of the attenuated Oka strain is a mutated nu-<br>
cleotide (that is, the nucleotide is mutated to "Y").<br>
Such a mutation is unique to the attenuated Oka strain.<br>
The mutation showing the XX(X/Y) pattern is a mutation<br>
wherein a nucleotide of the parental Oka strain is<br>
identical to a corresponding nucleotide of the Dumas<br>
strain (that is, both nucleotides are "X"), but a cor-<br>
responding nucleotide of the attenuated Oka strain is a<br>
mixture of a nucleotide which is identical to that of<br>
the Dumas strain and a mutated nucleotide (that is, the<br>
mixture of the identical nucleotide "X" and the mutated<br>
nucleotide "Y"). In the genome of the attenuated Oka<br>
strain, there are 18 nucleotide mutations showing the<br>
XXY pattern and 40 nucleotide mutations showing the<br>
XX(X/Y) pattern. Among the total of 58 nucleotide mu-<br>
tations, 49 nucleotide mutations are found in the cod-<br>
ing regions, 8 nucleotide mutations are found in the<br>
noncoding regions, and 1 nucleotide mutation is found<br>
in a stop codon. Further, among the 49 nucleotide mu-<br>
tations in the coding regions, 29 nucleotide mutations<br>
are nonsynonymous substitutions, and 20 nucleotide mu-<br>
tations are synonymous substitutions. Further detailed<br>
analyses revealed that among the 18 nucleotide muta-<br>
tions showing the XXY pattern, 9 nucleotide mutations<br>
are nonsynonymous substitutions, 8 nucleotide mutations<br>
are synonymous substitutions, and 1 nucleotide mutation<br>
is found in a noncoding region. The following nucleo-<br>
tide mutations which show the XXY pattern and are non-<br>
synonymous substitutions are unique to the attenuated<br>
Oka strain: the 5,745th G of gene 6; the 105,356th C,<br>
105,544th G, 106,262nd C and 107,252nd C of gene 62;<br>
and the 122,645th G, 123,635th G, 124,353rd C and<br>
124,541st G of gene 71. These unique nucleotides of<br>
the attenuated Oka strain are considered to be closely<br>
related to the attenuation and safety of a varicella<br>
virus and, thus these nucleotide are very important.<br>
Among the above-mentioned 9 nucleotides, 4 nucleotides<br>
are found in gene 62 and 4 nucleotides are found in<br>
gene 71. Since gene 62 and gene 71 are contained in<br>
the inverted repeats (see Fig. 1), in the present in-<br>
vention, the quality control of an attenuated varicella<br>
live vaccine is conducted by subjecting the genomic DNA<br>
of a sample varicella vaccine virus to sequence analy-<br>
sis and confirming that the above-mentioned 1 nucleo-<br>
tide of gene 6 and 4 nucleotides of gene 62 are con-<br>
served without suffering mutation. For improving the<br>
exactness of the quality control, it is preferred that<br>
the sample virus is further confirmed to conserve the<br>
above-mentioned 4 nucleotides of gene 71 without suf-<br>
fering mutation.<br>
Further in the present invention, it is preferred<br>
to confirm that the sample varicella vaccine virus con-<br>
serves, without suffering mutation, all 58 nucleotides<br>
which are unique to the attenuated Oka strain. Spe-<br>
cifically, together with the above-mentioned unique nu-<br>
cleotides of the attenuated Oka strain which show the<br>
XXY pattern and are nonsynonymous substitutions, the<br>
conservation of the following 49 nucleotides without<br>
suffering mutation is confirmed in the present inven-<br>
tion:<br>
the 560th C, the 703rd Y, the 763rd Y, the 2,515th Y,<br>
the 10,900th Y, the 12,779th Y, the 19,431st Y, the<br>
26,125th G, the 31,732nd Y, the 38,036th Y, the<br>
39,227th K, the 58,595th R, the 59,287th R, the<br>
64,067th R, the 71,252nd Y, the 82,225th R, the<br>
84,091st R, the 87,280th R, the 87,306th Y, the<br>
89,734th R, the 90,535th R, the 94,167th C, the<br>
97,748th R, the 97,796th Y, the 101,089th R, the<br>
105,169th R, the 105,310th R, the 105,705th C, the<br>
106,710th R, the 107,136th C, the 107,599th R, the<br>
107,797th R, the 108,111st C, the 108,838th R, the<br>
109,137th R, the 109,200th R, the 111,650th R, the<br>
118,247th Y, the 120,697th Y, the 120,760th Y, the<br>
121,059th Y, the 121,786th G, the 122,100th Y, the<br>
122,298th Y, the 122,761st G, the 123,187th Y, the<br>
124,192nd G, the 124,587th Y and the 124,728th Y,<br>
wherein, R represents A or G, Y represents C<br>
or T, and K represents G or T.<br>
In addition to the above-mentioned 58 nucleotide<br>
mutations, the following unique mutations of the at-<br>
tenuated Oka strain were found by the homology search<br>
conducted among the whole genomic DNA sequences of the<br>
Dumas strain, the parental Oka strain and the attenuat-<br>
ed Oka strain: a deletion mutation in the origin of<br>
replication, a mutation in the repetitive region Rl of<br>
gene 11 and a mutation in the repetitive region R4 of<br>
the noncoding regions.<br>
In a varicella virus genome, there are two origins<br>
of replication which are contained in the inverted re-<br>
peats (see Fig. 1). The origins of replication are a<br>
region corresponding to the 110,087th to 110,350th nu-<br>
cleotides of the sense strand of the genomic DNA of the<br>
Dumas strain and a region corresponding to the<br>
119,547th to 119,810th nucleotides of the antisense<br>
strand of the genomic DNA of the Dumas strain. The nu-<br>
cleotide sequence of the sense strand is shown in Table<br>
4. As is apparent from Table 4, the deletion in the<br>
origins of replication of the attenuated Oka strain oc-<br>
cur with respect to segments each having a nucleotide<br>
sequence of TATATATATATATA arranged in the direction of<br>
from the 5' end to the 3' end, and the segments are a<br>
segment corresponding to the 110,214th to 110,227th nu-<br>
cleotides of the sense strand and a segment correspond-<br>
ing to the 119,670th to 119,683rd nucleotides of the<br>
antisense strand. In the present invention, it is<br>
preferred that the presence of this deletion is further<br>
confirmed. Specifically, this deletion can be con-<br>
firmed by determining the presence or absence of the<br>
segments each having a nucleotide sequence of ATATATATA<br>
which correspond to the 110,219th to 110,227th nucleo-<br>
tides of the sense strand and the 119,670th to<br>
119,678th nucleotides of the antisense strand.<br>
The repetitive region Rl of gene 11 is a region<br>
corresponding to the 13,937th to 14,242nd nucleotides<br>
of the genomic DNA of the Dumas strain, and the nucleo-<br>
tide sequence of the Rl region is shown in Table 5. As<br>
is apparent from Table 5, the nucleotide sequence of<br>
the Rl region of the attenuated Oka strain is different<br>
from that of not only the.Dumas strain, but also the<br>
parental Oka strain. Therefore, for the quality con-<br>
trol of the vaccine, it is preferred that the Rl region<br>
of the sample varicella vaccine virus is confirmed to<br>
be identical with the Rl region of the attenuated Oka<br>
strain. Specifically, it is confirmed that the repeti-<br>
tive sequence of one whole Rl region of the genomic DNA<br>
of the sample varicella vaccine virus is a nucleotide<br>
sequence of abbabba'bbb'abababx arranged in the direc-<br>
tion of from the 5' end to the 3' end (wherein, a rep-<br>
resents a nucleotide sequence of GGACGCGATCGACGACGA; a'<br>
represents a nucleotide sequence of GGACGCGATTGACGACGA;<br>
b represents a nucleotide sequence of GGGAGAGGCGGAGGA;<br>
b' represents a nucleotide sequence of GGACGCGGCGGAGGA;<br>
and x represents a nucleotide sequence of GGA).<br>
In a varicella virus genome, two repetitive re-<br>
gions R4 which are contained in the inverted repeats<br>
are found in the noncoding regions (see Fig. 1). The<br>
R4 regions are a region corresponding to the 109,762nd<br>
to 109,907th nucleotides of the sense strand of the ge-<br>
nomic DNA of the Dumas strain and a region correspond-<br>
ing to the 119,990th to 120,135th nucleotides of the<br>
antisense strand of the genomic DNA of the Dumas strain.<br>
The nucleotide sequence of the R4 region in the direc-<br>
tion of the 5' end to the 3' end is shown in Table 7.<br>
As is apparent from Table 7, the repetitive sequence of<br>
the R4 region of the attenuated Oka strain is different<br>
from that of not only the Dumas strain, but also the<br>
parental Oka strain. Therefore, for the quality con-<br>
trol of the vaccine, it is preferred that the R4 region<br>
of the sample varicella vaccine virus is confirmed to<br>
be identical with the R4 region of the attenuated Oka<br>
strain. Specifically, it is confirmed that the repeti-<br>
tive sequence of each of two whole R4 regions of the<br>
genomic DNA of the sample varicella vaccine virus is a<br>
nucleotide sequence of aaaaaaaaaaaax arranged in the<br>
direction of from the 5' end to the 3' end (wherein, a<br>
represents a nucleotide sequence of<br>
CCCCGCCGATGGGGAGGGGGCGCGGTA; and x represents a nucleo-<br>
tide sequence of CCCCGCCGATG).<br>
In addition, the mutations shown in Table 6 have<br>
been found in the repetitive region R3 of gene 22.<br>
However, as is apparent from Table 6, there is a large<br>
diversity among the repetitive sequences of the R3 re-<br>
gion of the attenuated Oka strain and the parental Oka<br>
strain.<br>
The method of the present invention has been com-<br>
pleted based on the above-mentioned nucleotide muta-<br>
tions which are unique to the attenuated Oka strain.<br>
Therefore, the methods used in the methods for the<br>
quality control of the present invention can be used<br>
not only for the quality control of an attenuated vari-<br>
cella virus live vaccine (that is, determining whether<br>
a seed virus for a vaccine, an attenuated varicella vi-<br>
rus as a raw material of a vaccine or a live vaccine<br>
has been mutated or not), but also for the identifica-<br>
tion of a virus strain capable of functioning as an at-<br>
tenuated varicella vaccine virus (a virus which can be<br>
used as an active ingredient of a varicella vaccine)<br>
and the analysis of a virulent strain (the parental Oka<br>
strain or a natural wild-type strain). In addition,<br>
the method of the present invention provides an exact<br>
and advantageous techniques to be used for tracing the<br>
effects of the vaccination and for the researches in<br>
the field of epidemiology of varicella and zoster, and<br>
also provides an exact measure for preventing varicella<br>
and zoster.<br>
The specific methods for conducting the quality<br>
control of the present invention will be described in<br>
detail below.<br>
Preparation of a genomic DNA of a sample varicella<br>
virus: In the method of the present invention, a virus<br>
suspension or a bulk vaccine solution for use as an ac-<br>
tive ingredient of an attenuated varicella live vaccine,<br>
a virus suspension obtained by propagating a desired<br>
VZV, and vesicle fluid and the like obtained from a<br>
naturally infected varicella patient can be used as a<br>
sample varicella virus. The genomic DNA can be ex-<br>
tracted and purified directly from the sample viruses<br>
by a conventional method. Alternatively, cells can be<br>
infected with VZV to be used as the sample virus, and<br>
the genomic DNA of the virus can be extracted and puri-<br>
fied from the infected cells by a conventional method<br>
(with respect to the methods for extracting and purify-<br>
ing a DNA, reference can be made to "Current Protocol<br>
in Molecular Biology", Volume 1, Chapter 2, 2.0.1-<br>
2.6.12, John Wiley &amp; Sons, Inc., 1987-2000 (the loose-<br>
leaf system)). For propagating the VZV, WI-38 cells<br>
and MRC-5 cells can be used. It is preferred that the<br>
vesicle fluid used as a material for isolating, propa-<br>
gating and preparing a fresh wild-type strain or an<br>
epidemic strain is obtained from a naturally infected<br>
patient within 3 days after the onset of varicella.<br>
Preparation of PCR primers: A desired nucleotide<br>
sequence of a VZV genomic DNA can be amplified by a PCR<br>
method. First, polynucleotide strands consisting of<br>
contiguous sequences of about 15 to 30 nucleotides<br>
which correspond to the 5'-terminal sequence of the<br>
sense and antisense sequences of the desired region are<br>
prepared by a DNA synthesizer. The prepared polynu-<br>
cleotide strands are used as a pair of primers. The<br>
whole genomic DNA sequence of the attenuated Oka strain<br>
shown in SEQ ID NO:2 and the patent documents mentioned<br>
under "Prior Art" of the specification (U.S. Patent No.<br>
6,093,535 and International Application Publication WO<br>
00/50603) can be reffered when designing the PCR prim-<br>
ers .<br>
Determination of the nucleotide sequence of the<br>
PCR products: From the view point of saving labor for<br>
conducting the experiments, it is preferred that the<br>
PCR products are analyzed by a direct DNA sequencing<br>
method without the preparation of a genomic DNA library<br>
(this method is described in "Current Protocol in Mo-<br>
lecular Biology", Volume 3, Chapter 15, 15.2.1-15.2.11,<br>
ditto). In this method, the sequencing of a nucleotide<br>
sequence can be determined by conventional methods, for<br>
example, dideoxy method, a method using Cycle Sequence<br>
Kit (manufactured and sold by TAKARA SHUZO Co. Ltd.,<br>
Japan), and a method using DNA Sequencing Kit (manufac-<br>
tured and sold by Perkin Elmer Applied Biosystems, USA)<br>
Homology search of DNA sequences: Homology search<br>
of DNA sequences can be performed using commercially<br>
available computer software for gene analysis. For ex-<br>
ample, GENETYX-WIN (ver. 3.1) (manufactured and sold by<br>
Software Development Co., Ltd., Japan), DNASIS (ver.<br>
3.7) (manufactured and sold by Hitachi Software Engi-<br>
neering Co., Ltd., Japan), FASTA (http://www.ddjb.<br>
nig.ac.jp/), and BLAST (http://www.ncbi.nlm.nih.gov/)<br>
can be used. Whether or not a sample varicella vaccine<br>
virus has a specific nucleotide sequence of the attenu-<br>
ated varicella Oka strain disclosed in the present<br>
specification can be determined by a homology search.<br>
Confirmation of a nucleotide mutation by an RFLP<br>
analysis: In addition to the homology search conducted<br>
after determining the (whole or partial) nucleotide se-<br>
quence of the genomic DNA of a sample virus, an RFLP<br>
(Restriction Fragment Length Polymorphism) analysis can<br>
be conducted to confirm that the unique nucleotides of<br>
the attenuated Oka strain are conserved by the sample<br>
virus without suffering mutation. Specifically, poly-<br>
nucleotide strands consisting of contiguous sequences<br>
of about 15 to 30 nucleotides which correspond to the<br>
5'-terminal sequence of the sense and antisense se-<br>
quences of the desired region are prepared by a DNA<br>
synthesizer, and the prepared polynucleotide strands<br>
are used as a pair of PCR primers. A pair of PCR<br>
primers are simultaneously used for amplifying the de-<br>
sired region to be used as a sample DNA. Thus obtained<br>
sample DNA is digested with a restriction enzyme and<br>
applied to a gel electrophoresis. The presence of a<br>
mutation can be determined by the difference in the<br>
size of the detected DNA fragments. The RFLP analysis<br>
which is easier to perform than the homology search is<br>
preferably used in the method for the quality control<br>
of the present invention. With respect to the 9 speci-<br>
fie nucleotide mutations of the attenuated Oka strain<br>
which show the XXY pattern and are nonsynonymous sub-<br>
stitutions (the 5,745th G, the 105,356th C, the<br>
105,544th G, the 106,262nd C, the 107,252nd C, the<br>
122,645th G, the 123,635th G, the 124,353rd C and the<br>
124,541st G), and the 560th C, the 26,125th G, the<br>
94,167th C, the 105,705th C, the 107,136th C, the<br>
108,111st C, the 121,786th G, the 122,761st G and the<br>
124,192nd G among the remaining 49 nucleotide mutations<br>
of the attenuated Oka strain, the presence or the ab-<br>
sence of the mutations can be determined by the RFLP<br>
analysis using the eight primer pairs shown in Table 8<br>
(SEQ ID NOs: 3 to 18). The restriction sites of the<br>
PCR products obtained by using the PCR primers shown in<br>
Table 8 are summarized in Table 9, together with the<br>
sizes of the restriction fragments. For example, the<br>
mutation of gene 6 (the 5,745th nucleotide G found in<br>
gene 6 of the attenuated Oka strain) can be detected by<br>
the absence or presence of the restriction enzyme Alu I<br>
site. Specifically, 763 bp DNA fragment corresponding<br>
to the 5,372nd to 6,134th nucleotides of the genomic<br>
DNA of a sample varicella vaccine virus is amplified<br>
using the primers 01-N12 and 01-R13 shown in Table 8.<br>
The amplified fragment is digested with Alu I and ap-<br>
plied to an agarose gel electrophoresis. In the case<br>
of a virulent strain, the PCR product is cleaved into<br>
three fragments (170 bp, 205 bp and 388 bp). On the<br>
other hand, the PCR product of the attenuated Oka<br>
strain is cleaved into two fragments (170 bp and 593<br>
bp) . Therefore, whether or not a sample varicella vac-<br>
cine virus is a virus capable of functioning as a vac-<br>
cine strain can be determined from the restriction-<br>
fragment pattern.<br>
In the present invention, the 9 unique nucleotide<br>
mutations of the attenuated Oka strain which show the<br>
XXY pattern and are nonsynonymous substitutions are<br>
preferably confirmed by the RFLP analysis using the<br>
following primers: a pair of primers of SEQ ID NOs: 5<br>
and 6 with respect to the confirmation of the 5,745th<br>
G; a pair of primers of SEQ ID NOs: 11 and 12 with re-<br>
spect to the confirmation of the 105,356th C, the<br>
105,544th G, the 124,353rd C and the 124,541st G; a<br>
pair of primers of SEQ ID NOs: 13 and 14 with respect<br>
to the confirmation of the 106,262nd C and the<br>
123,635th G; and a pair of primers of SEQ ID NOs: 15<br>
and 16 with respect to the confirmation of the<br>
107,252nd C and the 122,645th G. Since gene 62 and<br>
gene 71 are inverted repeats (see Fig. 1), the quality<br>
control of an attenuated varicella live vaccine can be<br>
conducted by confirming the conservation of at least 5<br>
nucleotides (namely 1 nucleotide of gene 6 and 4 nu-<br>
cleotides of gene 62) by the RFLP analysis.<br>
As mentioned above, the whole genomic DNA sequence<br>
of the attenuated Oka strain (SEQ ID NO:2) and the nu-<br>
cleotide mutations which are unique to the attenuated<br>
Oka strain have been disclosed for the first time by<br>
the present inventors. Since the unique nucleotide mu-<br>
tations of the attenuated Oka strain are considered to<br>
be very important for the attenuation of a varicella<br>
viruses, an attenuated strain can be constructed by in-<br>
troducing a nucleotide substitution to a genomic DNA of<br>
a virulent strain (for example a wild type VZV strain<br>
or an epidemic strain), or by inducing an amino acid<br>
mutation in a virulent strain. The method described in<br>
Proc. Natl. Acad. Sci., USA, 90(15), 7376-7380, 1998<br>
can be used as a genetic engineering technique for in-<br>
ducing a nucleotide or amino acid mutation. Specifi-<br>
cally, the 58 nucleotide mutations which are unique to<br>
the attenuated Oka strain can be used as an index for<br>
inducing a mutation to a virulent strain, and the 9 nu-<br>
cleotide substitutions which are nonsynonymous substi-<br>
tutions are especially useful. In addition, taking in-<br>
to account the fact that gene 62 and gene 71 are con-<br>
tained in the inverted repeats (see Fig. 1), among the<br>
above-mentioned nucleotide substitutions, at least 5<br>
nucleotide mutations (namely a mutation found in gene 6<br>
and the mutations found in either gene 62 or gene 71)<br>
are considered to be very important.<br>
BEST MODE FOR CARRYING OUT THE INVENTION<br>
Hereinbelow, the present invention will be de-<br>
scribed in more detail with reference to the following<br>
Examples, but they should not be construed as limiting<br>
the scope of the present invention.<br>
Example 1<br>
The attenuated Oka strain (attenuated vaccine<br>
strain) and its parental strain (parental Oka strain; a<br>
virulent strain which is not attenuated) were individu-<br>
ally inoculated into MRC-5 cells to thereby obtain in-<br>
fected cells. The genomic DNAs of the attenuated Oka<br>
strain and the parental Oka strain were extracted indi-<br>
vidually from the infected cells by phenol extraction<br>
and chloroform/isoamyl alcohol extraction, and purified<br>
by ethanol precipitation, thereby obtaining DNA. PCR<br>
products covering the entire genome of each strain were<br>
prepared using the obtained DNA as a template and 88<br>
synthetic primers (44 primer pairs). Subsequently, the<br>
nucleotide sequences of the PCR products were deter-<br>
mined by the direct DNA sequencing method using 520<br>
synthetic primers and the DNA Sequencing Kit (manufac-<br>
tured and sold by Perkin Elmer Applied Biosystems, USA).<br>
Using the whole genomic sequence of the Dumas strain<br>
(virulent strain) shown in SEQ ID NO: 1 as a standard,<br>
the homology search was conducted with respect to the<br>
obtained whole genomic DNA sequences of the attenuated<br>
Oka strain and the parental Oka strain. DNASIS (ver-<br>
sion 3.7) (manufactured and sold by Hitachi Software<br>
Engineering Co., Ltd., Japan) was used for conducting<br>
the homology search. The characteristics of the at-<br>
tenuated Oka strain which became apparent from the ho-<br>
mology search are summarized in Tables 1 to 7.<br>
The nucleotide mutations which were detected by<br>
comparing the sequences among the three varicella<br>
strains (Dumas strain, parental Oka strain and attenu-<br>
ated Oka strain) are listed in Table 1. Specifically,<br>
the nucleotide number, the gene number, the mutated nu-<br>
cleotide and the amino acid mutation caused by the nu-<br>
cleotide mutation are described for each nucleotide mu-<br>
tation. In Table 1, Y represents a pyrimidine base<br>
(i.e., C or T), R represents a purine base (i.e., A or<br>
G), K represents G or T, (ncr) represents a noncoding<br>
region, an alphabet letter in parentheses (for example<br>
"(W)") is a one-letter abbreviation of an amino acid,<br>
(och) represents ochre codon, (amb) represents amber<br>
codon, (W/R) represents tryptophan (W) or arginine (R),<br>
and (del) represents deletion.<br>
Among the mutations listed in Table 1, the muta-<br>
tions which were detected by the sequence alignment be-<br>
tween the attenuated Oka strain and the parental Oka<br>
strain are listed in Table 2. Specifically, the nu-<br>
cleotide number, the gene number, the mutated nucleo-<br>
tide and the amino acid mutation caused by the nucleo-<br>
tide mutation are described for each nucleotide muta-<br>
tion. In Table 2, X/Y represents X or Y, and a three-<br>
letter abbreviation of an amino acid encoded by a nu-<br>
cleotide is shown in parentheses following the nucleo-<br>
tide (when a nucleotide is located in a noncoding re-<br>
gion, the nucleotide is followed by "(ncr)"). All<br>
other abbreviations used in Table 2 are the same as<br>
those used in Table 1.<br>
The nucleotide mutations described in Tables 1 and<br>
2 are summarized in Table 3, based on the mutation pat-<br>
terns . The mutation patterns and abundance thereof are<br>
listed together with the details of the mutation pat-<br>
terns, that is, the specific types of mutation (a stop<br>
codon (och/amb) mutation, a synonymous or nonsynonymous<br>
substitution, and deletion (or addition)) and the abun-<br>
dance of each type of mutation.<br>
The findings based on Tables 1 to 3 are explained<br>
in detail below.<br>
Among the major nucleotide and amino acid muta-<br>
tions which were determined by the comparison between<br>
the whole genomic DNA sequences of the attenuated Oka<br>
strain and the parental Oka strain, the mutations de-<br>
scribed in the following items (a) to (f) were found to<br>
be especially useful and important for the quality con-<br>
trol of an attenuated varicella live vaccine.<br>
(a) There were 58 important nucleotide mutations<br>
of the attenuated Oka strain, namely 18 nucleotide mu-<br>
tations showing the XXY pattern and 40 nucleotide muta-<br>
tions showing the XX(X/Y) pattern.<br>
(b) Among the above-mentioned 58 mutations, 49<br>
mutations were found in the coding regions, 8 mutations<br>
were found in the noncoding regions and 1 mutation was<br>
found in a stop codon.<br>
(c) Among the 49 mutations found in the coding<br>
regions, 29 mutations were nonsynonymous substitutions<br>
and 20 mutations were synonymous substitutions.<br>
(d) Among the 18 mutations showing the XXY pat-<br>
tern, 9 mutations were nonsynonymous substitutions, 8<br>
mutations were synonymous substitutions, and 1 mutation<br>
was found in a noncoding region.<br>
(e) The above-mentioned 9 mutations showing the<br>
XXY pattern which are nonsynonymous substitutions<br>
(namely the 5,745th nucleotide G of gene 6; the<br>
105,356th nucleotide C, the 105,544th nucleotide G, the<br>
106,262nd nucleotide C and the 107,252nd nucleotide C<br>
of gene 62; and the 122,645th nucleotide G, the<br>
123,635th nucleotide G, the 124,353rd nucleotide C and<br>
the 124,541st nucleotide G of gene 71) can be used as<br>
markers for the attenuation or safety of a virus strain<br>
capable of functioning as an active ingredient of a<br>
live vaccine. Therefore, these nucleotide mutations<br>
are useful and important for the quality control of a<br>
vaccine. It should be noted that gene 62 and gene 71<br>
are contained in the inverted repeats (see Fig. 1).<br>
(f) 40 nucleotides of the vaccine strain showed<br>
the XX(X/Y) pattern (that is, a mutation pattern where-<br>
in the virus strain is a mixture of a virus having nu-<br>
cleotide X and a virus having nucleotide Y). When the<br>
attenuated Oka strain was subcultured experimentally<br>
(i.e., the virus was passaged 5 times, 10 times, 17<br>
times and the like), all the nucleotides of the XX(X/Y)<br>
pattern, except for the 106,710th nucleotide, showed<br>
the following tendency. The detection frequency of nu-<br>
cleotide Y increased in accordance with the number of<br>
passages (that is, the nucleotide changed from X/Y to<br>
Y) and the mutation pattern of the nucleotide converged<br>
to the XXY pattern. In other words, the ratio of X to<br>
Y (x/y) decreased in accordance with the number of pas-<br>
sages. Based on the above-mentioned phenomenon, it is<br>
considered that the number of passages of a seed virus<br>
used in a seed lot system can be estimated by measuring<br>
the x/y value. It should be noted that among the<br>
above-mentioned mutations, 20 mutations converged to<br>
nonsynonymous substitutions. With respect to the re-<br>
mainder of the mutations, 12 mutations were synonymous<br>
substitutions, 1 mutation was found in a stop codon<br>
(specifically, an ochre codon/amber codon mixture con-<br>
verged to an amber mutation), and 7 mutations were<br>
found in the noncoding regions.<br>
Table 4 shows the sequence alignment of the nu-<br>
cleotide sequences of the sense strand of the origin of<br>
replication of the Dumas strain, the parental Oka<br>
strain and the attenuated Oka strain. In this table,<br>
"-" represents a deletion.<br>
In the attenuated Oka strain, deletions occur with<br>
respect to segments each having a nucleotide sequence<br>
of TATATATATATATA arranged in the direction of from the<br>
5' end to the 3' end, which correspond to the 110,214th<br>
to 110,227th nucleotides of the sense strand of the ge-<br>
nomic DNA of the Dumas strain of SEQ ID NO:l and a seg-<br>
ment corresponding to the 119,670th to 119,683rd nu-<br>
cleotides of the antisense strand of the genomic DNA of<br>
the Dumas strain. Therefore, taking into consideration<br>
the difference between the parental Oka strain and the<br>
attenuated Oka strain, it became apparent that the de-<br>
letion with respect to segments ATATATATA at the 3' end<br>
is useful for the quality control of a vaccine.<br>
Table 5 shows the sequence alignment of the re-<br>
petitive region Rl (in the direction of from the 5' end<br>
to the 3' end) of gene 11 of the Dumas strain, the par-<br>
ental Oka strain and the attenuated Oka strain.<br>
Table 6 shows the sequence alignment of the re-<br>
petitive region R3 (in the direction of from the 5' end<br>
to the 3' end) of gene 22 of the Dumas strain, the par-<br>
ental Oka strain and the attenuated Oka strain.<br>
Table 7 shows the sequence alignment of the re-<br>
petitive region R4 (in the direction of from the 5' end<br>
to the 3' end) of the Dumas strain, the parental Oka<br>
strain and the attenuated Oka strain.<br>
As shown in Table 5, the repetitive sequences of<br>
whole Rl region of all three strains, namely the at-<br>
tenuated Oka strain, the parental Oka strain and the<br>
Dumas strain, are different from each other. Similarly,<br>
the repetitive sequences of whole R4 region of all<br>
three strains, namely the attenuated Oka strain, the<br>
parental Oka strain and the Dumas strain, are different<br>
from each other. Therefore, the repetitive sequence<br>
abbabba'bbb'abababx of Rl region (wherein, a represents<br>
a nucleotide sequence of GGACGCGATCGACGACGA; a' repre-<br>
sents a nucleotide sequence of GGACGCGATTGACGACGA; b<br>
represents a nucleotide sequence of GGGAGAGGCGGAGGA; b'<br>
represents a nucleotide sequence of GGACGCGGCGGAGGA;<br>
and x represents a nucleotide sequence of GGA) and the<br>
repetitive sequence aaaaaaaaaaaax of R4 region (wherein,<br>
a represents a nucleotide sequence of<br>
CCCCGCCGATGGGGAGGGGGCGCGGTA; and x represents a nucleo-<br>
tide sequence of CCCCGCCGATG) are unique to the attenu-<br>
ated varicella virus, and these sequences are useful<br>
for the quality control of an attenuated varicella live<br>
vaccine.<br>
With respect to the sequences of the repetitive<br>
region R3 of gene 22 which are shown in Table 6, the<br>
sequences were diverse among the clones of the attenu-<br>
ated Oka strain and the parental Oka strain. Therefore,<br>
no unique sequence was found in the attenuated Oka<br>
strain.<br>
Example 2<br>
The genomic DNA of each of the attenuated Oka<br>
strain, the parental Oka strain and the Kawaguchi<br>
strain (wild-type strain of a varicella virus) was in-<br>
dividually prepared in the same manner as in Example 1.<br>
Using the PCR primers 01-N12 (SEQ ID NO:5) and 01-<br>
R13 (SEQ ID NO:6) shown in Table 8, a region corre-<br>
sponding to a part of gene 6 (a region corresponding to<br>
the 5,372nd to 6,134th nucleotides of the Dumas strain)<br>
was amplified by PCR, thereby obtaining a PCR product.<br>
The obtained PCR product (763 bp) was digested with the<br>
restriction enzyme Alu I to thereby cleave the DNA into<br>
fragments, and the restriction-fragment pattern was de-<br>
termined by an RFLP analysis. Specifically, each of<br>
the PCR products of the attenuated Oka strain, the par-<br>
ental Oka strain and the Kawaguchi strain was digested<br>
with the restriction enzyme Alu I, thereby obtaining a<br>
DNA fragment mixture, and the obtained DNA fragment<br>
mixture was applied to 4.0 % (w/v) agarose gel electro-<br>
phoresis to determine the size of each DNA fragment.<br>
Two fragments individually having a size of 170 bp<br>
and 593 bp were detected for the attenuated Oka strain.<br>
On the other hand, three fragments individually having<br>
a size of 170 bp, 205 bp and 388 bp were detected for<br>
the parental Oka strain and the Kawaguchi strain. The-<br>
se results show that the parental Oka strain and the<br>
Kawaguchi strain have the Alu I site located between<br>
the 205 bp fragment and the 388 bp fragment, but the<br>
attenuated Oka strain does not have this restriction<br>
site. It was confirmed from these results that the mu-<br>
tation of the 5,745th nucleotide A in gene 6 can be<br>
confirmed by detecting the absence of the Alu I site.<br>
Example 3<br>
54 epidemic varicella strains derived from the<br>
varicella patients and the zoster patients were indi-<br>
vidually subjected to an RFLP analysis. Specifically,<br>
the difference in a restriction-fragment pattern ob-<br>
tained by digesting a PCR product with the restriction<br>
enzyme Alu I was determined by an RFLP analysis in the<br>
same manner as in Example 2. As a result, it was found<br>
that the PCR products of all epidemic strains were<br>
cleaved into three fragments individually having a size<br>
of 170 bp, 205 bp and 388 bp. Such a restriction-<br>
fragment pattern was the same as that of the parental<br>
Oka strain obtained in Example 2 above.<br>
Example 4<br>
The primers shown in Table 8 (SEQ ID NOs: 3 to 6<br>
and 9 to 18) and the restriction enzymes Nla III, Alu I,<br>
BstX I, SfaN I, Ace II, Sac II, Sma I, a combination of<br>
BssH II and Nae I, or Bsr I were used to conduct an<br>
RFLP analysis in the same manner as in Example 2. Spe-<br>
cifically, the genomic DNAs of each of the attenuated<br>
Oka strain, the parental Oka strain and the Kawaguchi<br>
strain were individually prepared in the same manner as<br>
in Example 1. Next, a specific region of the genomic<br>
DNA was amplified using the PCR primers shown in Table<br>
8 in a specific combination shown in Table 9. The re-<br>
sultant PCR product was digested with a restriction en-<br>
zyme and applied to an agarose gel electrophoresis.<br>
The results are shown in Fig. 2. In addition, the re-<br>
striction enzymes used for the RFLP analysis and the<br>
sizes of the restriction fragments are summarized in<br>
Table 9.<br>
As is shown in Fig. 2 and Table 9, it became ap-<br>
parent that the numbers and sizes of the fragments re-<br>
sulting from the digestion of a PCR product of the at-<br>
tenuated Oka strain were different from those of the<br>
parental Oka strain and the Kawaguchi strain under all<br>
of the specific conditions employed for the RFLP analy-<br>
ses. Therefore, with respect to the 560th C, the<br>
5,745th G, the 94,167th C, the 105,356th C (the<br>
124,541st G), the 105,544th G (the 124,353rd C), the<br>
105,705th C (the 124,192nd G), the 106,262nd C (the<br>
123,635th G) , the 107,136th C (the 122,761st G), the<br>
107,252nd C (the 122,645th G), and the 108,111st C (the<br>
121,786th G), the nucleotide mutations can be confirmed<br>
by an RFLP analysis without determining the nucleotide<br>
sequence of the genome.<br>
INDUSTRIAL APPLICABILITY<br>
According to the method for quality control of the<br>
present invention, it has become possible to conduct an<br>
exact quality control and quality assurance of an at-<br>
tenuated varicella live vaccines, particularly with re-<br>
spect to the safety, effectiveness and uniformity of<br>
the vaccine. Further, the present invention provides<br>
exact and advantageous techniques which can be used for<br>
research in the field of epidemiology of varicella and<br>
zoster, including a tracing of the effects of vaccina-<br>
tion, and these techniques may expedite and enhance the<br>
research. Consequently, the present invention provides<br>
an exact and very effective measure for preventing<br>
varicella and zoster, which contributes to the health<br>
of human beings.<br>
WE CLAIM:<br>
1. A method for the quality control of an attenuated<br>
varicella live vaccine, which comprises:<br>
analyzing the genomic DNA of a sample varicella vac-<br>
cine virus, wherein the sample varicella vaccine virus<br>
is a virus for use as an active ingredient of an attenu-<br>
ated varicella live vaccine; and<br>
confirming that the genomic DNA of said sample<br>
varicella vaccine virus conserves without suffering mu-<br>
tation the following 5 nucleotides:<br>
the 5,745th G, the 105,356th C, the 105,544th G,<br>
the 106,262nd C and the 107,252nd C,<br>
wherein the nucleotide numbers are in accor-<br>
dance with the nucleotide numbering system of<br>
the nucleotide sequence of the genomic DNA of<br>
the varicella virus Dumas strain of SEQ ID NO:<br>
1.<br>
2. The method as claimed in claim 1, wherein the con-<br>
servation of said 5 nucleotides combination is confirmed<br>
by an RFLP analysis using the following primers:<br>
a pair of primers of SEQ ID NOs: '5 and 6 with re-<br>
spect to the confirmation of the 5,745th G;<br>
a pair of primers of SEQ ID NOs: 11 and 12 with<br>
respect to the confirmation of the 105,356th C<br>
and the 105,544th G;<br>
a pair of primers of SEQ ID NOs: 13 and 14 with<br>
respect to the confirmation of the 106,262nd C;<br>
and<br>
a pair of primers of SEQ ID NOs: 15 and 16 with<br>
respect to the confirmation of the 107,252nd C.<br>
3. The method as claimed in claim 1 or 2, which further<br>
comprises confirming that the genomic DNA of said sample<br>
varicella vaccine virus conserves without suffering mu-<br>
tation the following 4 nucleotides:<br>
the 122,645th G, the 123,635th G, the 124,353rd C<br>
and the 124,541st G,<br>
wherein the nucleotide numbers are in accor-<br>
dance with the nucleotide numbering system of<br>
the nucleotide sequence of the genomic DNA of<br>
the varicella virus Dumas strain of SEQ ID NO:<br>
1.<br>
4. The method as claimed in claim 3, wherein the con-<br>
servation of said 4 nucleotides is confirmed by an RFLP<br>
analysis using the following primers:<br>
a pair of primers of SEQ ID NOs: 11 and 12 with<br>
respect to the confirmation of the 124,353rd C<br>
and the 124,541st G;<br>
a pair of primers of SEQ ID NOs: 13 and 14 with<br>
respect to the confirmation of the 123,635th G;<br>
and<br>
a pair of primers of SEQ ID NOs: 15 and 16 with<br>
respect to the confirmation of the 122,645th G.<br>
5. The method as claimed in any one of claims 1 to 4,<br>
which further comprises confirming that the genomic DNA<br>
of said sample varicella vaccine virus conserves without<br>
suffering mutation the following 49 nucleotides:<br>
the 560th C, the 703rd Y, the 763rd Y, the 2,515th<br>
Y, the 10,900th Y, the 12,779th Y, the 19,431st Y,<br>
the 26,125th G, the 31,732nd Y, the 38,036th Y,<br>
the 39,227th K, the 58,595th R, the 59,287th R,<br>
the 64,067th R, the 71,252nd Y, the 82,225th R,<br>
the 84,091st R, the 87,280th R, the 87,306th Y,<br>
the 89,734th R, the 90,535th R, the 94,167th C,<br>
the 97,748th R, the 97,796th Y, the 101,089th R,<br>
the 105,169th R, the 105,310th R, the 105,705th C,<br>
the 106,710th R, the 107,136th C, the 107,599th R,<br>
the 107,797th R, the 108,111st C, the 108,838th R,<br>
the 109,137th R, the 109,200th R, the 111,650th R,<br>
the 118,247th Y, the 120,697th Y, the 120,760th Y,<br>
the 121,059th Y, the 121,786th G, the 122,100th Y,<br>
the 122,298th Y, the 122,761st G, the 123,187th Y,<br>
the 124,192nd G, the 124,587th Y and the 124,728th<br>
Y,<br>
wherein:<br>
the nucleotide numbers are in accordance with the<br>
nucleotide numbering system of the nucleotide se-<br>
quence of the genomic DNA of the varicella virus<br>
Dumas strain of SEQ ID NO: 1,<br>
R represents A or G,<br>
Y represents C or T, and<br>
K represents G or T.<br>
6. The method as claimed in claim 5, wherein the con-<br>
servation of the 560th C, the 26,125th G, the 94,167th C,<br>
the 105,705th C, the 107,136th C, the 108,111st C, the<br>
121,786th G, the 122,761st G and the 124,192nd G among<br>
said 49 nucleotides is confirmed by an RFLP analysis us-<br>
ing the following primers:<br>
a pair of primers of SEQ ID NOs: 3 and 4 with re-<br>
spect to the confirmation of the 560th C;<br>
a pair of primers of SEQ ID NOs: 7 and 8 with re-<br>
spect to the confirmation of the 26,125th G;<br>
a pair of primers of SEQ ID NOs: 9 and 10 with<br>
respect to the confirmation of the 94,167th C;<br>
a pair of primers of SEQ ID NOs: 13 and 14 with<br>
respect to the confirmation of the 105,705th C<br>
and the 124,192nd G;<br>
a pair of primers of SEQ ID NOs: 15 and 16 with<br>
respect to the confirmation of the 107,136th C<br>
and the 122,761st G; and<br>
a pair of primers of SEQ ID NOs: 17 and 18 with<br>
respect to the confirmation of the 108,111st C<br>
and the 121,786th G.<br>
7. The method as claimed in any one of claims 1 to 6,<br>
which further comprises confirming deletion mutations in<br>
two origins of replication of the genomic DNA of said<br>
sample varicella vaccine virus,<br>
wherein said two origins of replication are a region<br>
corresponding to the 110,087th to 110,350th nucleotides<br>
of the sense strand of the genomic DNA of the varicella<br>
virus Dumas strain of SEQ ID NO: 1 and a region corre-<br>
sponding to the 119,547th to 119,810th nucleotides of<br>
the genomic DNA of the antisense strand of said Dumas<br>
strain, and<br>
wherein said deletion mutations occur with respect<br>
to segments each having a nucleotide sequence of<br>
ATATATATA arranged in the direction of from the 5' end<br>
to the 3' end, said segments being a segment correspond-<br>
ing to the 110,219th to 110,227th nucleotides of the<br>
sense strand of the genomic DNA of said Dumas strain and<br>
a segment corresponding to the 119,670th to 119,678th<br>
nucleotides of the antisense strand of the genomic DNA<br>
of said Dumas strain.<br>
8. The method as claimed in any one of claims 1 to 7,<br>
which further comprises confirming that the repetitive<br>
sequence of one whole Rl region of the genomic DNA of<br>
said sample varicella vaccine virus is a nucleotide se-<br>
quence of abbabba'bbb'abababx arranged in the direction<br>
of from the 5' end to the 3' end,<br>
wherein:<br>
a represents a nucleotide sequence of<br>
GGACGCGATCGACGACGA;<br>
a' represents a nucleotide sequence of<br>
GGACGCGATTGACGACGA;<br>
b represents a nucleotide sequence of<br>
GGGAGAGGCGGAGGA;<br>
b' represents a nucleotide sequence of<br>
GGACGCGGCGGAGGA; and<br>
x represents a nucleotide sequence of GGA,<br>
wherein said whole Rl region is a region corre-<br>
sponding to the 13,937th to 14,242nd nucleotides of<br>
the genomic DNA of the varicella virus Dumas strain<br>
of SEQ ID NO: 1.<br>
9. The method as claimed in any one of claims 1 to 8,<br>
which further comprises confirming that the repetitive<br>
sequence of each of two whole R4 regions of the genomic<br>
DNA of said sample varicella vaccine virus is a nucleo-<br>
tide sequence of aaaaaaaaaaaax arranged in the direction<br>
of from the 5' end to the 3' end,<br>
wherein:<br>
a represents a nucleotide sequence of<br>
CCCCGCCGATGGGGAGGGGGCGCGGTA; and<br>
x represents a nucleotide sequence of CCCCGCCGATG,<br>
wherein said two whole R4 regions are a region cor-<br>
responding to the 109,762nd to 109,907th nucleotides of<br>
the sense strand of the genomic DNA of the varicella vi-<br>
rus Dumas strain of SEQ ID NO: 1 and a region corre-<br>
sponding to the 119,990th to 120,135th nucleotides of<br>
the antisense strand of the genomic DNA of said Dumas<br>
strain.<br><br>
Disclosed is a method for quality control of an attenuated<br>
varicella live vaccine, which comprises analyzing<br>
the genomic DNA of a sample varicella vaccine virus,<br>
wherein the sample varicella vaccine virus is a virus<br>
for use as an active ingredient of an attenuated<br>
varicella live vaccine; and confirming that the genomic<br>
DNA of the sample varicella vaccine virus conserves the<br>
5,745th G, the 105,356th C, the 105,544th G, the<br>
106,262nd C and the 107,252nd C without suffering mutation,<br>
wherein the nucleotide numbers are in accordance<br>
with the nucleotide numbering system of the nucleotide<br>
sequence of the genomic DNA of the varicella virus Dumas<br>
strain of SEQ ID NO: 1.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtODA4LWdyYW50ZWQtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-808-granted-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtODA4LWdyYW50ZWQtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-808-granted-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtODA4LWdyYW50ZWQtY29ycmVzcG9uZGVuY2UucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-808-granted-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtODA4LWdyYW50ZWQtZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-808-granted-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtODA4LWdyYW50ZWQtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-808-granted-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtODA4LWdyYW50ZWQtZXhhbWluYXRpb24gcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-808-granted-examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtODA4LWdyYW50ZWQtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-808-granted-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtODA4LWdyYW50ZWQtZm9ybSAxOC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-808-granted-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtODA4LWdyYW50ZWQtZm9ybSAyLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-808-granted-form 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtODA4LWdyYW50ZWQtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-808-granted-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtODA4LWdyYW50ZWQtcGEucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-808-granted-pa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtODA4LWdyYW50ZWQtcmVwbHkgdG8gZXhhbWluYXRpb24gcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-808-granted-reply to examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtODA4LWdyYW50ZWQtc3BlY2lmaWNhdGlvbi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-808-granted-specification.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtODA4LWdyYW50ZWQtdHJhbnNsYXRlZCBjb3B5IG9mIHByaW9yaXR5IGRvY3VtZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-808-granted-translated copy of priority document.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDEtODA4LUtPTC0oMDMtMTEtMjAxMSktRk9STSAyNy5wZGY=" target="_blank" style="word-wrap:break-word;">IN-PCT-2001-808-KOL-(03-11-2011)-FORM 27.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDEtODA4LUtPTC0oMDgtMDgtMjAxMiktRk9STS0yNy5wZGY=" target="_blank" style="word-wrap:break-word;">IN-PCT-2001-808-KOL-(08-08-2012)-FORM-27.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="233692-process-for-refining-impure-silver-bullion.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="233694-tunable-fiber-optic-connector-and-method-for-assembling.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>233693</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/808/KOL</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>14/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>03-Apr-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>01-Apr-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>07-Aug-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>THE RESEARCH FOUNDATION FOR MICROBIAL DISEASE OF OSAKA UNIVERSITY .</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>C/O OSAKA UNIVERSITY, 3-1, YAMADAOKA SUITA-SHI, OSAKA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>GOMI YASUYUKI</td>
											<td>854 TAKAYA-CHO, KANONJI-SHI, KAGAWA-KEN 768-0002</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SUNAMACHI HIROKI</td>
											<td>1152-3 MUROMOTO-CHO KANONJI-SHI KAGAWA-KEN 768-0001</td>
										</tr>
										<tr>
											<td>3</td>
											<td>TAKAHASHI MICHIAKI</td>
											<td>18-5, AOYAMADAI 3-CHOME SUITA-SHI OSAKA 565-0075</td>
										</tr>
										<tr>
											<td>4</td>
											<td>YAMANISHI KOICHI</td>
											<td>2-1-21 KOFUDAI, TOYONO-CHO, TOYONO GUN, OSAKA 563-0104</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A61K 39/2005</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/JP2001/00678</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-01-31</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>2000-62734</td>
									<td>2000-01-31</td>
								    <td>Japan</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/233693-method-for-quality-control-of-an-attenuated-varicella-live-vaccine by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:50:51 GMT -->
</html>
