<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/233641-prostate-stem-cell-antigen-psca-variants by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:49:06 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 233641:PROSTATE STEM CELL ANTIGEN (PSCA) VARIANTS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">PROSTATE STEM CELL ANTIGEN (PSCA) VARIANTS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>PSCA and its encoded protein, and variants thereof, are described wherein PSCA exhibits tissue specific expression in normal adult tissue, and is aberrantly expressed in the cancers listed in Table 1. Consequently, PSCA provides a diagnostic, prognostic, propylactic and/or therapeutic target for cancer. The PSCA gene or fragment thereof, or its encoded protein, or variants thereof, or a fragment thereof, can be used to elicit a humoral or cellular immune response; antibodies or T cells reactive with PSCA can he used in active or passive immunization.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>PROSTATE STEM CELL ANTIGEN (PSCA) VARIANTS AND SUBSEQUENCES THEREOF<br>
Technical Field<br>
[0001] The invention described herein relates to genes and their encoded proteins, termed PSCA, expressed in<br>
certain cancers, and to diagnostic and therapeutic methods and compositions useful in the management of cancers that<br>
express PSCA.<br>
Background Art<br>
[0002] Cancer is the second leading cause of human death next to coronary disease. Worldwide, millions of people<br>
die from cancer every year. In the United States alone, as reported by the American Cancer Society, cancer causes the<br>
death of well over a half-million people annually, with over 1.2 million new cases diagnosed per year. While deaths from<br>
heart disease have been declining significantly, those resulting from cancer generally are on the rise. In the early part of<br>
the next century, cancer is predicted to become the leading cause of death.<br>
[0003] Worldwide, several cancers stand out as the leading killers. In particular, carcinomas of the lung, prostate,<br>
breast, colon, pancreas, and ovary represent the primary causes of cancer death. These and virtually all other carcinomas<br>
share a common lethal future. With very few exceptions, metastatic disease from a carcinoma is fatal. Moreover, even for<br>
those cancer patients who initially survive their primary cancers, common experience has shown that their lives are<br>
dramatically altered. Many cancer patients experience strong anxieties driven by the awareness of the potential for<br>
recurrence or treatment failure. Many cancer patients experience physical debilitations following treatment. Furthermore,<br>
many cancer patients experience a recurrence.<br>
[0004] Worldwide, prostate cancer is the fourth most prevalent cancer in men. In North America and Northern<br>
Europe, it is by far the most common cancer in males and is the second leading cause of cancer death in men. In the<br>
United States alone, well over 30,000 men die annually of this disease - second only to lung cancer. Despite the<br>
magnitude of these figures, there is still no effective treatment for metastatic prostate cancer. Surgical prostatectomy,<br>
radiation therapy, hormone ablation therapy, surgical castration and chemotherapy continue to be the main treatment<br>
modalities. Unfortunately, these treatments are ineffective for many and are often associated with undesirable<br>
consequences.<br>
[0005] On the diagnostic front, the lack of a prostate tumor marker that can accurately detect early-stage, localized<br>
tumors remains a significant limitation in the diagnosis and management of this disease. Although the serum prostate<br>
specific antigen (PSA) assay has been a very useful tool, however its specificity and general utility is widely regarded as<br>
lacking in several important respects.<br>
[0006] Progress in identifying additional specific markers for prostate cancer has been improved by the generation of<br>
prostate cancer xenografts that can recapitulate different stages of the disease in mice. The LAPC (Los Angeles Prostate<br>
Cancer) xenografts are prostate cancer xenografts that have survived passage in severe combined immune deficient<br>
(SCID) mice and have exhibited the capacity to mimic the transition from androgen dependence to androgen independence<br>
(Klein etal., 1997, Nat. Med. 3:402). More recently identified prostate cancer markers include PCTA-1 (Su et al., 1996,<br>
Proc. Natl. Acad. Sci. USA 93:7252), prostate-specific membrane (PSM) antigen (Pinto et al., Clin Cancer Res 1996 Sep 2<br>
(9): 1445-51), STEAP (Hubert, et al., Proc Natl Acad Sci USA. 1999 Dec 7; 96(25): 14523-8) and prostate stem cell<br>
antigen (PSCA) (Reiter et al., 1998, Proc. Natl. Acad. Sci. USA 95:1735).<br><br>
[0007] While previously identified markers such as PSA, PSM, PCTA and PSCA have facilitated efforts to diagnose<br>
and at prostate cancer, there is need for the identification of additional markers and therapeutic targets for prostate and<br>
related cancers in order to further improve diagnosis and therapy.<br>
[0008] Renal cell carcinoma (RCC) accounts for approximately 3 percent of adult malignancies. Once adenomas<br>
reach a diameter of 2 to 3 cm, malignant potential exists. In the adult, the two principal malignant renal tumors are renal<br>
cell adenocarcinoma and transitional cell carcinoma of the renal pelvis or ureter. The incidence of renal cell<br>
adenocarcinoma is estimated at more than 29,000 cases in the United States, and more than 11,600 patients died of this<br>
disease in 1998. Transitional cell carcinoma is less frequent, with an incidence of approximately 500 cases per year in the<br>
United States.<br>
[0009] Surgery has been the primary therapy for renal cell adenocarcinoma for many decades. Until recently,<br>
metastatic disease has been refractory to any systemic therapy. With recent developments in systemic therapies,<br>
particularly immunotherapies, metastatic renal cell carcinoma may be approached aggressively in appropriate patients with<br>
a possibility of durable responses. Nevertheless, there is a remaining need for effective therapies for these patients.<br>
[0010] Of all new cases of cancer in the United States, bladder cancer represents approximately 5 percent in men<br>
(fifth most common neoplasm) and 3 percent in women (eighth most common neoplasm). The incidence is increasing<br>
slowly, concurrent with an increasing older population. In 1998, there was an estimated 54,500 cases, including 39,500 in<br>
men and 15,000 in women. The age-adjusted incidence in the United States is 32 per 100,000 for men and eight per<br>
100,000 in women. The historic male/female ratio of 3:1 may be decreasing related to smoking patterns in women. There<br>
were an estimated 11,000 deaths from bladder cancer in 1998 (7,800 in men and 3,900 in women). Bladder cancer<br>
incidence and mortality strongly increase with age and will be an increasing problem as the population becomes more<br>
elderly.<br>
[0011 ] Most bladder cancers recur in the bladder. Bladder cancer is managed with a combination of transurethral<br>
resection of the bladder (TUR) and intravesical chemotherapy or immunotherapy. The multifocal and recurrent nature of<br>
bladder cancer points out the limitations of TUR. Most muscle-invasive cancers are not cured by TUR alone. Radical<br>
cystectomy and urinary diversion is the most effective means to eliminate the cancer but carry an undeniable impact on<br>
urinary and sexual function. There continues to be a significant need for treatment modalities that are beneficial for bladder<br>
cancer patients.<br>
[0012] An estimated 130,200 cases of colorectal cancer occurred in 2000 in the United States, including 93,800<br>
cases of colon cancer and 36,400 of rectal cancer. Colorectal cancers are the third most common cancers in men and<br>
women. Incidence rates declined significantly during 1992-1996 (-2.1 % per year). Research suggests that these declines<br>
have been due to increased screening and polyp removal, preventing progression of polyps to invasive cancers. There<br>
were an estimated 56,300 deaths (47,700 from colon cancer, 8,600 from rectal cancer) in 2000, accounting for about 11%<br>
of all U.S. cancer deaths.<br>
[0013] At present, surgery is the most common form of therapy for colorectal cancer, and for cancers that have not<br>
spread, it is frequently curative. Chemotherapy, or chemotherapy plus radiation, is given before or after surgery to most<br>
patients whose cancer has deeply perforated the bowel wall or has spread to the lymph nodes. A permanent colostomy<br>
(creation of an abdominal opening for elimination of body wastes) is occasionally needed for colon cancer and is<br>
infrequently required for rectal cancer. There continues to be a need for effective diagnostic and treatment modalities for<br>
colorectal cancer.<br>
[0014] There were an estimated 164,100 new cases of lung and bronchial cancer in 2000, accounting for 14% of all<br>
U.S. cancer diagnoses. The incidence rate of lung and bronchial cancer is declining significantly in men, from a high of<br>
86.5 per 100,000 in 1984 to 70.0 in 1996. In the 1990s, the rate of increase among women began to slow. In 1996, the<br>
incidence rate in women was 42.3 per 100,000.<br>
[0015] Lung and bronchial cancer caused an estimated 156,900 deaths in 2000, accounting for 28% of all cancer<br>
deaths. During 1992-1996, mortality from lung cancer declined significantly among men (-1.7% per year) while rates for<br>
women were still significantly increasing (0.9% per year). Since 1987, more women have died each year of lung cancer<br>
than breast cancer, which, for over 40 years, was the major cause of cancer death in women. Decreasing lung cancer<br>
incidence and mortality rates most likely resulted from decreased smoking rates over the previous 30 years; however,<br>
decreasing smoking patterns among women lag behind those of men. Of concern, although the declines in adult tobacco<br>
use have slowed, tobacco use in youth is increasing again.<br>
[0016] Treatment options for lung and bronchial cancer are determined by the type and stage of the cancer and<br>
include surgery, radiation therapy, and chemotherapy. For many localized cancers, surgery is usually the treatment of<br>
choice. Because the disease has usually spread by the time it is discovered, radiation therapy and chemotherapy are often<br>
needed in combination with surgery. Chemotherapy alone or combined with radiation is the treatment of choice for small<br>
cell lung cancer; on this regimen, a large percentage of patients experience remission, which in some cases is long lasting.<br>
There is however, an ongoing need for effective treatment and diagnostic approaches for lung and bronchial cancers.<br>
[0017] An estimated 182,800 new invasive cases of breast cancer were expected to occur among women in the<br>
United States during 2000. Additionally, about 1,400 new cases of breast cancer were expected to be diagnosed in men in<br>
2000. After increasing about 4% per year in the 1980s, breast cancer incidence rates in women have leveled off in the<br>
1990s to about 110.6 cases per 100,000.<br>
[0018] In the U.S. alone, there were an estimated 41,200 deaths (40,800 women, 400 men) in 2000 due to breast<br>
cancer. Breast cancer ranks second among cancer deaths in women. According to the most recent data, mortality rates<br>
declined significantly during 1992-1996 with the largest decreases in younger women, both white and black. These<br>
decreases were probably the result of earlier detection and improved treatment.<br>
[0019] Taking into account the medical circumstances and the patient's preferences, treatment of breast cancer may<br>
involve lumpectomy (local removal of the tumor) and removal of the lymph nodes under the arm; mastectomy (surgical<br>
removal of the breast) and removal of the lymph nodes under the arm; radiation therapy; chemotherapy; or hormone<br>
therapy. Often, two or more methods are used in combination. Numerous studies have shown that, for early stage<br>
disease, long-term survival rates after lumpectomy plus radiotherapy are similar to survival rates after modified radical<br>
mastectomy. Significant advances in reconstruction techniques provide several options for breast reconstruction after<br>
mastectomy. Recently, such reconstruction has been done at the same time as the mastectomy.<br>
[0020] Local excision of ductal carcinoma in situ (DCIS) with adequate amounts of surrounding normal breast tissue<br>
may prevent the local recurrence of the DCIS. Radiation to the breast and/or tamoxifen may reduce the chance of DCIS<br>
occurring in the remaining breast tissue. This is important because DCIS, if left untreated, may develop into invasive breast<br>
cancer. Nevertheless, there are serious side effects or sequelae to these treatments. There is, therefore, a need for<br>
efficacious breast cancer treatments.<br>
[0021] There were an estimated 23,100 new cases of ovarian cancer in the United States in 2000. It accounts for<br>
4% of all cancers among women and ranks second among gynecologic cancers. During 1992-1996, ovarian cancer<br>
incidence rates were significantly declining. Consequent to ovarian cancer, there were an estimated 14,000 deaths in<br>
20%. Ovarian cancer causes more deaths than any other cancer of the female reproductive system.<br>
[0022] Surgery, radiation therapy, and chemotherapy are treatment options for ovarian cancer. Surgery usually<br>
includes the removal of one or both ovaries, the fallopian tubes (salpingo-oophorectomy), and the uterus (hysterectomy). In<br>
some very early tumors, only the involved ovary will be removed, especially in young women who wish to have children. In<br>
advanced disease, an attempt is made to remove all intra-abdominal disease to enhance the effect of chemotherapy.<br>
There continues to be an important need for effective treatment options for ovarian cancer.<br>
[0023] There were an estimated 28,300 new cases of pancreatic cancer in the United States in 2000. Over the past<br>
20 years, rates of pancreatic cancer have declined in men. Rates among women have remained approximately constant<br>
but may be beginning to decline. Pancreatic cancer caused an estimated 28,200 deaths in 2000 in the United States. Over<br>
the past 20 years, there has been a slight but significant decrease in mortality rates among men (about -0.9% per year)<br>
while rates have increased slightly among women.<br>
[0024] Surgery, radiation therapy, and chemotherapy are treatment options for pancreatic cancer. These treatment<br>
options can extend survival and/or relieve symptoms in many patients but are not likely to produce a cure for most. There<br>
is a significant need for additional therapeutic and diagnostic options for pancreatic cancer.<br>
Disclosure of the Invention<br>
[0025] The present invention relates to a gene, designated PSCA, that has now been found to be over-expressed in<br>
the cancer(s) listed in Table I. Northern blot expression analysis of PSCA gene expression in normal tissues shows a<br>
restricted expression pattern in adult tissues. The nucleotide (Figure 2) and amino acid (Figure 2, and Figure 3) sequences<br>
of PSCA are provided. The tissue-related profile of PSCA in normal adult tissues, combined with the over-expression<br>
observed in the tissues listed in Table I, shows that PSCA is aberrantly over-expressed in at least some cancers, and thus<br>
serves as a useful diagnostic, prophylactic, prognostic, and/or therapeutic target for cancers of the tissue(s) such as those<br>
listed in Table I.<br>
[0026] The invention provides polynucleotides corresponding or complementary to all or part of the PSCA genes,<br>
mRNAs, and/or coding sequences, preferably in isolated form, including polynucleotides encoding PSCA-related proteins<br>
and fragments of 4, 5, 6, 7, 8, 9,10,11,12,13,14,15,16,17,18,19,20,21,22,23, 24,25, or more than 25 contiguous<br>
amino acids; at least 30, 35,40,45, 50, 55, 60, 65,70, 80, 85, 90, 95,100 or more than 100 contiguous amino acids of a<br>
PSCA-related protein, as well as the peptides/proteins themselves; DNA, RNA, DNA/RNA hybrids, and related molecules,<br>
polynucleotides or oligonucleotides complementary or having at least a 90% homology to the PSCA genes or mRNA<br>
sequences or parts thereof, and polynucleotides or oligonucleotides that hybridize to the PSCA genes, mRNAs, or to<br>
PSCA-encoding polynucleotides. Also provided are means for isolating cDNAs and the genes encoding PSCA. Recombinant<br>
DNA molecules containing PSCA polynucleotides, cells transformed or transduced with such molecules, and host-vector<br>
systems for the expression of PSCA gene products are also provided. The invention further provides antibodies that bind to<br>
PSCA proteins and polypeptide fragments thereof, including polyclonal and monoclonal antibodies, murine and other<br>
mammalian antibodies, chimeric antibodies, humanized and fully human antibodies, and antibodies labeled with a<br>
detectable marker or therapeutic agent. In certain embodiments, there is a proviso that the entire nucleic acid sequence of<br>
Figure 2 is not encoded and/or the entire amino acid sequence of Figure 2 is not prepared. In certain embodiments, the<br>
entire nucleic acid sequence of Figure 2 is encoded and/or the entire amino acid sequence of Figure 2 is prepared, either of<br>
which are in respective human unit dose forms.<br><br>
[0027] The invention further provides methods for detecting the presence and status of PSCA polynucleotides and<br>
proper in various biological samples, as well as methods for identifying cells that express PSCA. A typical embodiment of<br>
this invention provides methods for monitoring PSCA gene products in a tissue or hematology sample having or suspected<br>
of having some form of growth dysregulation such as cancer.<br>
[0028] The invention further provides various immunogenic or therapeutic compositions and strategies for treating<br>
cancers that express PSCA such as cancers of tissues listed in Table I, including therapies aimed at inhibiting the<br>
transcription, translation, processing or function of PSCA as well as cancer vaccines. In one aspect, the invention provides<br>
compositions, and methods comprising them, for treating a cancer that expresses PSCA in a human subject wherein the<br>
composition comprises a carrier suitable for human use and a human unit dose of one or more than one agent that inhibits<br>
the production or function of PSCA. Preferably, the carrier is a uniquely human carrier. In another aspect of the invention,<br>
the agent is a moiety that is immunoreactive with PSCA protein. Non-limiting examples of such moieties include, but are<br>
not limited to, antibodies (such as single chain, monoclonal, polyclonal, humanized, chimeric, or human antibodies),<br>
functional equivalents thereof (whether naturally occurring or synthetic), and combinations thereof. The antibodies can be<br>
conjugated to a diagnostic or therapeutic moiety. In another aspect, the agent is a small molecule as defined herein.<br>
[0029] In another aspect, the agent comprises one or more than one peptide which comprises a cytotoxic T<br>
lymphocyte (CTL) epitope that binds an HLA class I molecule in a human to elicit a CTL response to PSCA and/or one or<br>
more than one peptide which comprises a helper T lymphocyte (HTL) epitope which binds an HLA class II molecule in a<br>
human to elicit an HTL response. The peptides of the invention may be on the same or on one or more separate<br>
polypeptide molecules. In a further aspect of the invention, the agent comprises one or more than one nucleic acid<br>
molecule that expresses one or more than one of the CTL or HTL response stimulating peptides as described above. In yet<br>
another aspect of the invention, the one or more than one nucleic acid molecule may express a moiety that is<br>
immunologically reactive with PSCA as described above. The one or more than one nucleic acid molecule may also be, or<br>
encodes, a molecule that inhibits production of PSCA. Non-limiting examples of such molecules include, but are not limited<br>
to, those complementary to a nucleotide sequence essential for production of PSCA (e.g. antisense sequences or<br>
molecules that form a triple helix with a nucleotide double helix essential for PSCA production) or a ribozyme effective to<br>
lyse PSCA mRNA.<br>
[0030] Note that to determine the starting position of any peptide set forth in Tables VIII-XXI and XXII to XL1X<br>
(collectively HLA Peptide Tables) respective to its parental protein, e.g., variant 1, variant 2, etc., reference is made to three<br>
factors: the particular variant, the length of the peptide in an HLA Peptide Table, and the Search Peptides in Table VII.<br>
Generally, a unique Search Peptide is used to obtain HLA peptides of a particular for a particular variant. The position of<br>
each Search Peptide relative to its respective parent molecule is listed in Table VII. Accordingly, if a Search Peptide begins<br>
at position "X", one must add the value "X -1" to each position in Tables VIII-XXI and XXII to XLIX to obtain the actual<br>
position of the HLA peptides in their parental molecule. For example, if a particular Search Peptide begins at position 150<br>
of its parental molecule, one must add 150 -1, i.e., 149 to each HLA peptide amino acid position to calculate the position of<br>
that amino acid in the parent molecule.<br>
[0031] One embodiment of the invention comprises an HLA peptide, that occurs at least twice in Tables VIII-XXI and<br>
XXII to XLIX collectively, or an oligonucleotide that encodes the HLA peptide. Another embodiment of the invention<br>
comprises an HLA peptide that occurs at least once in Tables VIII-XXI and at least once in tables XXII to XLIX, or an<br>
oligonucleotide that encodes the HLA peptide.<br>
[0032] Another embodiment of the invention is antibody epitopes, which comprise a peptide regions, or an<br>
oligonucleotide encoding the peptide region, that has one two, three, four, or five of the following characteristics:<br>
i) a peptide region of at least 5 amino acids of a particular peptide of Figure 3, in any whole number increment up<br>
to the full length of that protein in Figure 3, that includes an amino acid position having a value equal to or greater than 0.5,<br>
0.6,0.7,0.8,0.9, or having a value equal to 1.0, in the Hydrophilicity profile of Figure 5;<br>
ii) a peptide region of at least 5 amino acids of a particular peptide of Figure 3, in any whole number increment<br>
up to the full length of that protein in Figure 3, that includes an amino acid position having a value equal to or less than 0.5,<br>
0.4,0.3,0.2,0.1, or having a value equal to 0.0, in the Hydropathicity profile of Figure 6;<br>
iii) a peptide region of at least 5 amino acids of a particular peptide of Figure 3, in any whole number increment<br>
up to the full length of that protein in Figure 3, that includes an amino acid position having a value equal to or greater than<br>
0.5,0.6,0.7,0.8,0,9, or having a value equal to 1.0, in the Percent Accessible Residues profile of Figure 7;<br>
iv) a peptide region of at least 5 amino acids of a particular peptide of Figure 3, in any whole number increment<br>
up to the full length of that protein in Figure 3, that includes an amino acid position having a value equal to or greater than<br>
0.5, 0.6,0.7,0.8,0.9, or having a value equal to 1.0, in the Average Flexibility profile of Figure 8; or<br>
v) a peptide region of at least 5 amino acids of a particular peptide of Figure 3, in any whole number increment<br>
up to the full length of that protein in Figure 3, that includes an amino acid position having a value equal to or greater than<br>
0.5,0.6,0.7,0.8,0.9, or having a value equal to 1.0, in the Beta-turn profile of Figure 9.<br>
Brief Description of the Drawings<br>
[0033] Figure 1. Intentionally Omitted.<br>
[0034] Figure 2. A) The cDNA and amino acid sequence of PSCA variant 1 (also called "PSCA v.1" or "PSCA<br>
variant 1") is shown in Figure 2A. The start methionine is underlined. The open reading frame extends from nucleic acid<br>
18-389 including the stop codon.<br>
B) The cDNA and amino acid sequence of PSCA variant 2 (also called "PSCA v.2") is shown in Figure 2B. The<br>
codon for the start methionine is underlined. The open reading frame extends from nucleic acid 56-427 including the stop<br>
codon.<br>
C) The cDNA and amino acid sequence of PSCA variant 3 (also called "PSCA v.3") is shown in Figure 2C. The<br>
codon for the start methionine is underlined. The open reading frame extends from nucleic acid 423-707 including the stop<br>
codon.<br>
D) The cDNA and amino acid sequence of PSCA variant 4 (also called "PSCA v.4") is shown in Figure 2D. The<br>
codon for the start methionine is underlined. The open reading frame extends from nucleic acid 424-993 including the stop<br>
codon.<br>
E) The cDNA and amino acid sequence of PSCA variant 5 (also called "PSCA v.5") is shown in Figure 2E. The<br>
codon for the start methionine is underlined. The open reading frame extends from nucleic acid 910-1479 including the<br>
stop codon.<br>
F) The cDNA and amino acid sequence of PSCA variant 6 (also called "PSCA v.6") is shown in Figure 2F. The<br>
codon for the start methionine is underlined. The open reading frame extends from nucleic acid 83-427 including the stop<br>
codon.<br>
G) SNP variants of PSCA v.2, PSCA v.7 through v.18. The PSCA v.7 through v.18 proteins have 123 amino<br>
acids. Variants PSCA v.7 through v. 18 are variants with single nucleotide difference from PSCA v.2, and code for the same<br><br>
protein as v.2. Though these SNP variants are shown separately, they can also occur in any combinations and in any of<br>
transcript variants listed above in Figures 2A through 2F.<br>
H) SNP variants of PSCA v.4, PSCA v.19 through v.30. The PSCA v.19 through v.30 proteins have 189<br>
amino acids. Variants PSCA v.19 through v.30 are variants with single nucleotide difference from PSCA v.4. PSCA v.9,<br>
v.10, v.11, v.24 and v.25 proteins differ from PSCA v.1 by one amino acid. PSCA v.23, v.28, v.29 and v.30 code for the<br>
same protein as v.4. Though these SNP variants are shown separately, they can also occur in any combinations and in<br>
any of the transcript variants v.3 and v.4.<br>
[0035] Figure 3.<br>
A) The amino acid sequence of PSCA v.1 is shown in Figure 3A; it has 123 amino acids.<br>
3j The amino acid sequence of PSCA v.3 is shown in Figure 3B; it has 94 amino acids.<br>
C) The amino acid sequence of PSCA v.4 is shown in Figure 3C; it has 189 amino acids.<br>
D) The amino acid sequence of PSCA v.6 is shown in Figure 3D; it has 114 amino acids.<br>
E) The amino acid sequence of PSCA v.19 is shown in Figure 3E; it has 189 amino acids.<br>
F) The amino acid sequence of PSCA v.20 is shown in Figure 3F; it has 189 amino acids.<br>
G) The amino acid sequence of PSCA v.21 is shown in Figure 3G; it has 189 amino acids.<br>
H) The amino acid sequence of PSCA v.22 is shown in Figure 3H; it has 189 amino acids.<br>
I) The amino acid sequence of PSCA v.24 is shown in Figure 31; it has 189 amino acids.<br>
J) The amino acid sequence of PSCA v.25 is shown in Figure 3J; it has 189 amino acids.<br>
K) The amino acid sequence of PSCA v.26 is shown in Figure 3k; it has 189 amino acids.<br>
L) The amino acid sequence of PSCA v.27 is shown in Figure 3L; it has 189 amino acids.<br>
[0036] As used herein, a reference to PSCA includes all variants thereof, including those shown in Figures 2,3,10,<br>
11, and 12 unless the context clearly indicates otherwise.<br>
[0037] Figure 4. Alignment of PSCA v.4 with human Prostate Stem Cell Antigen (gi 27482160).<br>
[0038] Figure 5. Figures 5(a)-(c): Hydrophilicity amino acid profile of PSCAv.1, v.3, and v.4 determined by computer<br>
algorithm sequence analysis using the method of Hopp and Woods (Hopp T.P., Woods K.R., 1981. Proc. Natl. Acad. Sci.<br>
U.S.A. 78:3824-3828) accessed on the Protscale website located on the World Wide Web at (expasy.ch/cgi-<br>
bin/protscale.pl) through the ExPasy molecular biology server.<br>
[0039] Figure 6. Figures 6(a)-(c): Hydropathicity amino acid profile of PSCAv.1, v.3, and v.4 determined by<br>
computer algorithm sequence analysis using the method of Kyte and Doolittle (Kyte J., Doolittle R.F., 1982. J. Mol. Biol.<br>
157:105-132) accessed on the ProtScale website located on the World Wide Web at (.expasy.ch/cgi-bin/protscale.pl)<br>
through the ExPasy molecular biology server.<br>
[0040] Figure 7. Figures 7(a)-(c): Percent accessible residues amino acid profile of PSCAv.1, v.3, and v.4<br>
determined by computer algorithm sequence analysis using the method of Janin (Janin J., 1979 Nature 277:491-492)<br>
accessed on the ProtScale website located on the World Wide Web at (.expasy.ch/cgi-bin/protscale.pl) through the ExPasy<br>
molecular biology server.<br>
[0041] Figure 8. Figures 8(a)-(c): Average flexibility amino acid profile of PSCAv.1, v.3, and v.4 determined by<br>
computer algorithm sequence analysis using the method of Bhaskaran and Ponnuswamy (Bhaskaran R., and Ponnuswamy<br>
P.K., 1988. Int. J. Pept. Protein Res. 32:242-255) accessed on the ProtScale website located on the World Wide Web at<br>
(expasy.ch/cgi-bin/protscale.pl) through the ExPasy molecular biology server.<br>
[0042] Figure 9. Figures 9(a)-(c): Beta-turn amino acid profile of PSCAv.1, v.3, and v.4 determined by computer<br>
algorithm sequence analysis using the method of Deleage and Roux (Deleage, G., Roux B. 1987 Protein Engineering<br>
1 :260; 294) accessed on the ProtScale website located on the World Wide Web at (expasy.ch/cgi-bin/protscale.pl) through<br>
the ExPasy molecular biology server.<br>
[0043] Figure 10. Exon compositions of transcript variants of PSCA. Variant PSCA v.2, v.3, v.4 and v.5 are<br>
transcript variants of v.1. Variant v.2 started transcription 47 bp further to the 5' end than v.1. Variant v.3 had a shorter exon<br>
2 as compared to v.2. Variants v.4 and v.5 had an alternative first exon. Variant 5 kept the second intron as compared to<br>
v.4. The order of the potential exons on the human genome is shown at the bottom. Poly A tails were not shown in the<br>
figure. Ends of exons are shown above the boxes. Numbers in "()" underneath the boxes correspond to those of PSCA v.2.<br>
Lengths of introns and exons are not proportional.<br>
[0044] Figure 11. Figure 11(a): Schematic alignment of protein variants of PSCA. Protein variants correspond to<br>
nucleotide variants. Nucleotide variants PSCA v.2, v.7 through v.18 coded the same protein as v.1. Variant v.5 coded the<br>
same protein as v.4 and protein v.3 was part of v.4. Nucleotide variants PSCA v.2, v.3, v.4 and v.5 were transcript variants<br>
of v.1, as shown in Figure 10. The SNP in v.2 that did not cause codon change in v.2 caused a codon change in v.3, v.4,<br>
and v.5. Single amino acid differences were indicated above the boxes. Black boxes represent the same sequence as<br>
PSCA v.1 Numbers underneath the box correspond to PSCA v.1. Figure11(b): Schematic alignment of protein variants<br>
translated from SNP variants of PSCA v.4. Protein variants correspond to nucleotide variants. Nucleotide variants PSCA<br>
v.23, v.28, v.29 and v.30 coded the same protein as v.4. SNP in v.4 that resulted in an amino acid change in v.4 and also<br>
resulted in an amino acid change in v.5 and if occurring between aa 96-189, also in v.3. Single amino acid differences were<br>
indicated above the boxes. Black boxes represent the same sequence as PSCA v.4. Numbers underneath the box<br>
correspond to PSCA v.4.<br>
[0045] Figure 12. Figure 12(a): Schematic alignment of SNP variants of PSCA v.2. Variants PSCA v.6 through<br>
v.18 are variants with single nucleotide differences as compared to variant v.2. Variant v.6 changed the ORF from 56-427<br>
to 83-427. Though these SNP variants were shown separately, they could also occur in any combinations and in any<br>
transcript variants, such as v.4 shown in Fig. 12, that contained the base pairs. Numbers correspond to those of PSCA v.2.<br>
Black box shows the same sequence as PSCA v.2. SNPs are indicated above the box. Figure 12(b): Schematic<br>
alignment of SNP variants of PSCA v.4. Variants PSCA v.19 through v.30 are variants with single nucleotide differences as<br>
compared to variant v.4 (ORF.424-993). Though these SNP variants were shown separately, they could also occur in any<br>
combinations and in any transcript variants that contained the base pairs, such as v.5 shown in Fig. 10,. Numbers<br>
correspond to those of PSCA v.4. Black box shows the same sequence as PSCA v.4. SNPs are indicated above the box.<br>
[0046] Figure 13. Secondary structure and transmembrane domains prediction for PSCA protein variants.<br>
Figures 13A, 13B, 13C, and 13D: The secondary structure of PSCA protein variant 1 (SEQ ID NO:6532), variant 3 (SEQ<br>
ID NO:6536), variant 4 (SEQ ID NO:6540), and variant 6 (SEQ ID NO:6546) (Figures A-D, respectively) were predicted<br>
using the HNN - Hierarchical Neural Network method (NPS@: Network Protein Sequence Analysis TIBS 2000 March Vol.<br>
25, No 3 [291]:147-150 Combet C, Blanchet C, Geourjon C. and Deleage G., http://pbil.ibcp.fr/cgi-<br>
bin/npsa_automat.pl?page=npsa_nn.html), accessed from the ExPasy molecular biology server located on the World Wide<br>
Web at (.expasy.ch/fools/). This method predicts the presence and location of alpha helices, extended strands, and<br>
random coils from the primary protein sequence. The percent of each protein in a given secondary structure is also listed.<br>
Figures 13E, 13G, 131, and 13K: Schematic representation of the probability of existence of transmembrane regions of<br>
PSCA variants 1,3,4, and 6, respectively based on the TMpred algorithm of Hofmann and Stoffel which utilizes TMBASE<br><br>
(K. Hofmann, W. Stoffel. TMBASE - A database of membrane spanning protein segments Biol. Chem. Hoppe-Seyler<br>
374-386,1993). Figures 13F, 13H, 13J, and 13L: Schematic representation of the probability of the existence of<br>
transmembrane regions of PSCA variant 1, based on the TMHMM algorithm of Sonnhammer, von Heijne, and Krogh (Erik<br>
LL. Sonnhammer, Gunnar von Heijne, and Anders Krogh: A hidden Markov model for predicting transmembrane helices in<br>
protein sequences. In Proc. of Sixth Int. Conf. on Intelligent Systems for Molecular Biology, p 175-182 Ed J. Glasgow, T.<br>
Littlejohn, F. Major, R. Lathrop, D. Sankoff, and C. Sensen Menlo Park, CA: AMI Press, 1998). The TMpred and TMHMM<br>
algorithms are accessed from the ExPasy molecular biology server (http://www.expasy.ch/tools/).<br>
[0047] Figure 14. Expression of PSCA variants. Figure 14(A): Primers were designed to differentiate between<br>
PSCA v.1/v.2/v.4, PSCA v.3 and PSCA v.5. PSCA v.1/v.2/v.4 lead to a PCR product of 425 bp, PSCA v.3 leads to a PCR<br>
product of 300 bp, whereas PSCA v.5 leads to a PCR product of 910 bp in size. Figure 14(B): First strand cDNA was<br>
prepared from normal bladder, brain, heart, kidney, liver, lung, prostate, spleen, skeletal muscle, testis, pancreas, colon,<br>
stomach, pools of prostate cancer, bladder cancer, kidney cancer, colon cancer, lung cancer, ovary cancer, breast cancer,<br>
cancer metastasis, and pancreas cancer. Normalization was performed by PCR using primers to actin. Semi-quantitative<br>
PCR, using the variant specific primers was performed at 30 cycles of amplification. Results show expression of PSCA v.5<br>
mainly in breast cancer, cancer metastasis, and pancreas cancer, and at lower level in colon cancer and lung cancer.<br>
PSCA v.1/v.2/v.4 PCR product was detected in prostate cancer, bladder cancer, kidney cancer, colon cancer, lung cancer,<br>
ovary cancer, breast cancer, cancer metastasis, and pancreas cancer. Amongst normal tissues, PSCA v.1/v.2/v.4 PCR<br>
product was detected only in prostate, stomach and at lower level in kidney and lung, whereas PSCA v. 5 was not detected<br>
in any normal tissue. PSCA v.3 PCR detected product was not detected in any of the samples tested.<br>
[0048] Figure 15. Expression of PSCA v.4 and PSCA v.5. Figure 15(A): Primers were designed to differentiate<br>
between PSCA v.4 and PSCA v.5. PSCA v.4 lead to a PCR product of 460 bp, whereas PSCA v.5 leads to a PCR product<br>
of 945 bp in size. Figure 15(B): First strand cDNA was prepared from normal bladder, brain, heart, kidney, liver, lung,<br>
prostate, spleen, skel. muscle, testis, pancreas, colon, stomach, pools of prostate cancer, bladder cancer, and multi-<br>
xenograft pool (prostate cancer, kidney cancer and bladder cancer xenografts). Normalization was performed by PCR<br>
using primers to actin. Semi-quantitative PCR, using the variant specific primers was performed at 30 cycles of<br>
amplification. Results show expression of PSCA v.4 in prostate cancer, bladder cancer, and multi-xenograft pool, normal<br>
kidney and prostate. PSCA v.5 was detected only in normal prostate and bladder cancer.<br>
Modes of Carrying Out the Invention<br>
Outline of Sections<br>
I.) Definitions<br>
II.) PSCA Polynucleotides<br>
II A) Uses of PSCA Polynucleotides<br>
II.A.1.) Monitoring of Genetic Abnormalities<br>
II.A.2.) Antisense Embodiments<br>
II.A.3.) Primers and Primer Pairs<br>
II.A.4.) Isolation of PSCA-Encoding Nucleic Acid Molecules<br>
II.A.5.) Recombinant Nucleic Acid Molecules and Host-Vector Systems<br>
IIII.) PSCA-related Proteins<br><br>
IlI.A.) Motif-bearing Protein Embodiments<br>
III.B.) Expression of PSCA-related Proteins<br><br>
III.C.) Modifications of PSCA-related Proteins<br>
1I1.D.) Uses of PSCA-related Proteins<br>
IV.) PSCA Antibodies<br>
V.) PSCA Cellular Immune Responses<br>
VI.) PSCA Transgenic Animals<br>
VII.) Methods for the Detection of PSCA<br>
VIII.) Methods for Monitoring the Status of PSCA-related Genes and Their Products<br>
IX) Identification of Molecules That Interact With PSCA<br>
X.) Therapeutic Methods and Compositions<br>
X.A.) Anti-Cancer Vaccines<br>
X.B.) PSCA as a Target for Antibody-Based Therapy<br>
X.C.) PSCA as a Target for Cellular Immune Responses<br>
X.C.1. Minigene Vaccines<br>
X.C.2. Combinations of CTL Peptides with Helper Peptides<br>
X.C.3. Combinations of CTL Peptides with T Cell Priming Agents<br>
X.C.4. Vaccine Compositions Comprising DC Pulsed with CTL and/or HTL Peptides<br>
X.D.) Adoptive Immunotherapy<br>
X.E.) Administration of Vaccines for Therapeutic or Prophylactic Purposes<br>
XI.) Diagnostic and Prognostic Embodiments of PSCA.<br>
XII.) Inhibition of PSCA Protein Function<br>
XII.A.) Inhibition of PSCA With Intracellular Antibodies<br>
XII.B.) Inhibition of PSCA with Recombinant Proteins<br>
XII.C.) Inhibition of PSCA Transcription or Translation<br>
XII.D.) General Considerations for Therapeutic Strategies<br>
XIII.) Identification, Characterization and Use of Modulators of PSCA<br>
XIV.) KITS/Articles of Manufacture<br>
I)______Definitions:<br>
[0049] Unless otherwise defined, all terms of art, notations and other scientific terms or terminology used herein are<br>
intended to have the meanings commonly understood by those of skill in the art to which this invention pertains. In some<br>
cases, terms with commonly understood meanings are defined herein for clarity and/or for ready reference, and the<br>
inclusion of such definitions herein should not necessarily be construed to represent a substantial difference over what is<br>
generally understood in the art. Many of the techniques and procedures described or referenced herein are well<br>
understood and commonly employed using conventional methodology by those skilled in the art, such as, for example, the<br>
widely utilized molecular cloning methodologies described in Sambrook et al., Molecular Cloning: A Laboratory Manual 2nd.<br>
edition (1989) Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y. As appropriate, procedures involving the<br>
use of commercially available kits and reagents are generally carried out in accordance with manufacturer defined protocols<br>
and/or parameters unless otherwise noted.<br>
[0050] The terms "advanced prostate cancer", "locally advanced prostate cancer", "advanced disease" and "locally<br>
advanced disease" mean prostate cancers that have extended through the prostate capsule, and are meant to include<br>
stage C disease under the American Urological Association (AUA) system, stage C1 - C2 disease under the Whitmore-<br>
Jewett system, and stage T3 - T4 and N+ disease under the TNM (tumor, node, metastasis) system. In general, surgery is<br>
not recommended for patients with locally advanced disease, and these patients have substantially less favorable<br>
outcomes compared to patients having clinically localized (organ-confined) prostate cancer. Locally advanced disease is<br>
clinically identified by palpable evidence of induration beyond the lateral border of the prostate, or asymmetry or induration<br>
above the prostate base. Locally advanced prostate cancer is presently diagnosed pathologically following radical<br>
prostatectomy if the tumor invades or penetrates the prostatic capsule, extends into the surgical margin, or invades the<br>
seminal vesicles.<br>
[0051] "Altering the native glycosylate pattern" is intended for purposes herein to mean deleting one or more<br>
carbohydrate moieties found in native sequence PSCA (either by removing the underlying glycosylate site or by deleting<br>
the glycosylate by chemical and/or enzymatic means), and/or adding one or more glycosylate sites that are not present<br>
in the native sequence PSCA. In addition, the phrase includes qualitative changes in the glycosylate of the native<br>
proteins, involving a change in the nature and proportions of the various carbohydrate moieties present.<br>
[0052] The term "analog" refers to a molecule which is structurally similar or shares similar or corresponding attributes<br>
with another molecule (e.g. a PSCA-related protein). For example, an analog of a PSCA protein can be specifically bound by an<br>
antibody or T cell that specifically binds to PSCA.<br>
[0053] The term "antibody" is used in the broadest sense. Therefore, an "antibody" can be naturally occurring or man-<br>
made such as monoclonal antibodies produced by conventional hybridoma technology. Anti-PSCA antibodies comprise<br>
monoclonal and polyclonal antibodies as well as fragments containing the antigen-binding domain and/or one or more<br>
complementarity determining regions of these antibodies.<br>
[0054] An "antibody fragment" is defined as at least a portion of the variable region of the immunoglobulin molecule<br>
that binds to its target, i.e., the antigen-binding region. In one embodiment it specifically covers single anti-PSCA antibodies<br>
and clones thereof (including agonist, antagonist and neutralizing antibodies) and anti-PSCA antibody compositions with<br>
polyepitopic specificity.<br>
t0055] The term "codon optimized sequences" refers to nucleotide sequences that have been optimized for a<br>
particular host species by replacing any codons having a usage frequency of less than about 20%. Nucleotide sequences<br>
that have been optimized for expression in a given host species by elimination of spurious polyadenylation sequences,<br>
elimination of exon/intron splicing signals, elimination of transposon-like repeats and/or optimization of GC content in<br>
addition to codon optimization are referred to herein as an "expression enhanced sequences."<br>
[0056] A "combinatorial library" is a collection of diverse chemical compounds generated by either chemical<br>
synthesis or biological synthesis by combining a number of chemical "building blocks" such as reagents. For example, a<br>
linear combinatorial chemical library, such as a polypeptide (e.g., mutein) library, is formed by combining a set of chemical<br>
building blocks called amino acids in every possible way for a given compound length (i.e., the number of amino acids in a<br>
polypeptide compound). Numerous chemical compounds are synthesized through such combinatorial mixing of chemical<br>
building blocks (Gallop etal., J. Med. Chem. 37(9): 1233-1251 (1994)).<br>
[0057] Preparation and screening of combinatorial libraries is well known to those of skill in the art. Such<br>
combinatorial chemical libraries include, but are not limited to, peptide libraries (see, e.g., U.S. Patent No. 5,010,175,<br>
Fuka,Pept. Prot Res. 37:487-493 (1991), Houghton etal., Nature, 354:84-88 (1991)), peptoids (PCT Publication No WO<br>
91/19735), encoded peptides (PCT Publication WO 93/20242), random bio- oligomers (PCT Publication WO 92/00091),<br>
benzodiazepines (U.S. Pat. No. 5,288,514), diversomers such as hydantoins, benzodiazepines and dipeptides (Hobbs et<br>
al., Proc. Nat. Acad. Sci. USA 90:6909-6913 (1993)), vinylogous polypeptides (Hagihara et al„ J. Amer. Chem. Soc.<br>
114:6568 (1992)), nonpeptidal peptidomimetics with a Beta-D-Glucose scaffolding (Hirschmann et al., J. Amer. Chem. Soc.<br>
114:9217-9218 (1992)), analogous organic syntheses of small compound libraries (Chen et al., J. Amer. Chem. Soc.<br>
116:2661 (1994)), oligocarbarnates (Cho, et al., Science 261:1303 (1993)), and/or peptidyl phosphonates (Campbell et al.,<br>
J. Org. Chem. 59:658 (1994)). See, generally, Gordon et al., J. Med. Chem. 37:1385 (1994), nucleic acid libraries (see,<br>
e.g., Stratagene, Corp.), peptide nucleic acid libraries (see, e.g., U.S. Patent 5,539,083), antibody libraries (see, e.g.,<br>
Vaughn et al., Nature Biotechnology 14(3): 309-314 (1996), and PCT/US96/10287), carbohydrate libraries (see, e.g., Liang<br>
et al., Science 274:1520-1522 (1996), and U.S. Patent No. 5,593,853), and small organic molecule libraries (see, e.g.,<br>
benzodiazepines, Baum, C&amp;EN, Jan 18, page 33 (1993); isoprenoids, U.S. Patent No. 5,569,588; thiazolidinones and<br>
metathiazanones, U.S. Patent No. 5,549,974; pyrrolidines, U.S. Patent Nos. 5,525,735 and 5,519,134; morpholino<br>
compounds, U.S. Patent No. 5,506,337; benzodiazepines, U.S. Patent No. 5,288,514; and the like).<br>
[0058] Devices for the preparation of combinatorial libraries are commercially available (see, e.g., 357 NIPS,<br>
390 NIPS, Advanced Chem Tech, Louisville KY; Symphony, Rainin, Woburn, MA; 433A, Applied Biosystems, Foster City,<br>
CA; 9050, Plus, Millipore, Bedford, NIA). A number of well-known robotic systems have also been developed for solution<br>
phase chemistries. These systems include automated workstations such as the automated synthesis apparatus developed<br>
by Takeda Chemical Industries, LTD. (Osaka, Japan) and many robotic systems utilizing robotic arms (Zymate H, Zymark<br>
Corporation, Hopkinton, Mass.; Orca, Hewlett-Packard, Palo Alto, Calif.), which mimic the manual synthetic operations<br>
performed by a chemist. Any of the above devices are suitable for use with the present invention. The nature and<br>
implementation of modifications to these devices (if any) so that they can operate as discussed herein will be apparent to<br>
persons skilled in the relevant art. In addition, numerous combinatorial libraries are themselves commercially available<br>
(see, e.g., ComGenex, Princeton, NJ; Asinex, Moscow, RU; Tripos, Inc., St. Louis, MO; ChemStar, Ltd, Moscow, RU; 3D<br>
Pharmaceuticals, Exton, PA; Martek Biosciences, Columbia, MD; etc.).<br>
[0059] The term "cytotoxic agent" refers to a substance that inhibits or prevents the expression activity of cells,<br>
function of cells and/or causes destruction of cells. The term is intended to include radioactive isotopes chemotherapeutic<br>
agents, and toxins such as small molecule toxins or enzymatically active toxins of bacterial, fungal, plant or animal origin,<br>
including fragments and/or variants thereof. Examples of cytotoxic agents include, but are not limited to auristatins,<br>
auromycins, maytansinoids, yttrium, bismuth, ricin, ricin A-chain, combrestatin, duocarmycins, dolostatins, doxorubicin,<br>
daunorubicin, taxol, cisplatin, cc1065, ethidium bromide, mitomycin, etoposide, tenoposide, vincristine, vinblastine,<br>
colchicine, dihydroxy anthracin dione, actinomycin, diphtheria toxin, Pseudomonas exotoxin (PE) A, PE40, abrin, abrin A<br>
chain, modeccin A chain, alpha-sarcin, gelonin, mitogellin, retstrictocin, phenomycin, enomycin, curicin, crotin,<br>
calicheamicin, Sapaonaria officinalis inhibitor, and glucocorticoid and other chemotherapeutic agents, as well as<br>
radioisotopes such as At211,l131,l125, Y90, Re186, Re188, Sm153, Bj212 or 213, P32 and radioactive isotopes of Lu including Lu177.<br>
Antibodies may also be conjugated to an anti-cancer pro-drug activating enzyme capable of converting the pro-drug to its<br>
active form.<br>
[0060] The "gene product" is sometimes referred to herein as a protein or mRNA. For example, a "gene<br>
protect of the invention" is sometimes referred to herein as a "cancer amino acid sequence", "cancer protein", "protein of a<br>
cancer listed in Table I", a "cancer mRNA", "mRNA of a cancer listed in Table I", etc. In one embodiment, the cancer<br>
protein is encoded by a nucleic acid of Figure 2. The cancer protein can be a fragment, or alternatively, be the full-length<br>
protein to the fragment encoded by the nucleic acids of Figure 2. In one embodiment, a cancer amino acid sequence is<br>
used to determine sequence identity or similarity. In another embodiment, the sequences are naturally occurring allelic<br>
variants of a protein encoded by a nucleic acid of Figure 2. In another embodiment, the sequences are sequence variants<br>
as further described herein.<br>
[0061] "High throughput screening" assays for the presence, absence, quantification, or other properties of<br>
particular nucleic acids or protein products are well known to those of skill in the art. Similarly, binding assays and reporter<br>
gene assays are similarly well known. Thus, e.g., U.S. Patent No. 5,559,410 discloses high throughput screening methods<br>
for proteins; U.S. Patent No. 5,585,639 discloses high throughput screening methods for nucleic acid binding (i.e., in<br>
arrays); while U.S. Patent Nos. 5,576,220 and 5,541,061 disclose high throughput methods of screening for ligand/antibody<br>
binding.<br>
[0062] In addition, high throughput screening systems are commercially available (see, e.g., Amersham Biosciences,<br>
Piscataway, NJ; Zymark Corp., Hopkinton, MA; Air Technical Industries, Mentor, OH; Beckman Instruments, Inc. Fullerton,<br>
CA; Precision Systems, Inc., Natick, MA; etc.). These systems typically automate entire procedures, including all sample<br>
and reagent pipetting, liquid dispensing, timed incubations, and final readings of the microplate in detector(s) appropriate<br>
for the assay. These configurable systems provide high throughput and rapid start up as well as a high degree of flexibility<br>
and customization. The manufacturers of such systems provide detailed protocols for various high throughput systems.<br>
Thus, e.g., Zymark Corp. provides technical bulletins describing screening systems for detecting the modulation of gene<br>
transcription, ligand binding, and the like.<br>
[0063] The term "homolog" refers to a molecule which exhibits homology to another molecule, by for example, having<br>
sequences of chemical residues that are the same or similar at corresponding positions.<br>
[0064] "Human Leukocyte Antigen" or "HLA" is a human class I or class II Major Histocompatibility Complex (MHC)<br>
protein (see, e.g., Stites, ef a/., Immunology, 8th Ed., Lange Publishing, Los Altos, CA (1994).<br>
[0065] The terms "hybridize", "hybridizing", "hybridizes" and the like, used in the context of polynucleotides, are<br>
meant to refer to conventional hybridization conditions, preferably such as hybridization in 50% formamide/6XSSC/0.1 %<br>
SDS/100 ng/ml ssDNA, in which temperatures for hybridization are above 37 degrees C and temperatures for washing in<br>
0.1XSSC/0.1% SDS are above 55 degrees C.<br>
[0066] The phrases "isolated" or "biologically pure" refer to material which is substantially or essentially free from<br>
components which normally accompany the material as it is found in its native state. Thus, isolated peptides in accordance<br>
with the invention preferably do not contain materials normally associated with the peptides in their in situ environment. For<br>
example, a polynucleotide is said to be "isolated" when it is substantially separated from contaminant polynucleotides that<br>
correspond or are complementary to genes other than the PSCA genes or that encode polypeptides other than PSCA gene<br>
product or fragments thereof. A skilled artisan can readily employ nucleic acid isolation procedures to obtain an isolated PSCA<br>
polynucleotide. A protein is said to be "isolated," for example, when physical, mechanical or chemical methods are employed to<br>
remove the PSCA proteins from cellular constituents that are normally associated with the protein. A skilled artisan can readily<br>
employ standard purification methods to obtain an isolated PSCA protein. Alternatively, an isolated protein can be prepared by<br>
chemical means.<br><br>
[0067] The term "mammal" refers to any organism classified as a mammal, including mice, rats, rabbits, dogs, cats, cows,<br>
horses and humans. In one embodiment of the invention, the mammal is a mouse. In another embodiment of the invention, the<br>
mc inlet is a human.<br>
[0068] The terms 'metastatic prostate cancer" and "metastatic disease" mean prostate cancers that have spread to<br>
regional lymph nodes or to distant sites, and are meant to include stage D disease under the AUA system and stage<br>
TxNxM+ under the TNM system. As is the case with locally advanced prostate cancer, surgery is generally not indicated<br>
for patients with metastatic disease, and hormonal (androgen ablation) therapy is a preferred treatment modality. Patients<br>
with metastatic prostate cancer eventually develop an androgen-refractory state within 12 to 18 months of treatment<br>
initiation. Approximately half of these androgen-refractory patients die within 6 months after developing that status. The<br>
most common site for prostate cancer metastasis is bone. Prostate cancer bone metastases are often osteoblastic rather<br>
than osteolytic (i.e., resulting in net bone formation). Bone metastases are found most frequently in the spine, followed by<br>
the femur, pelvis, rib cage, skull and humerus. Other common sites for metastasis include lymphmodes, lung, liver and<br>
brain. Metastatic prostate cancer is typically diagnosed by open or laparoscopic pelvic lymphadenectomy, whole body<br>
radionuclide scans, skeletal radiography, and/or bone lesion biopsy.<br>
[0069] The term "modulator" or "test compound" or "drug candidate" or grammatical equivalents as used<br>
herein describe any molecule, e.g., protein, oligopeptide, small organic molecule, polysaccharide, polynucleotide, etc., to be<br>
tested for the capacity to directly or indirectly alter the cancer phenotype or the expression of a cancer sequence, e.g., a<br>
nucleic acid or protein sequences, or effects of cancer sequences (e.g., signaling, gene expression, protein interaction,<br>
etc.) In one aspect, a modulator will neutralize the effect of a cancer protein of the invention. By "neutralize" is meant that<br>
an activity of a protein is inhibited or blocked, along with the consequent effect on the cell. In another aspect, a modulator<br>
will neutralize the effect of a gene, and its corresponding protein, of the invention by normalizing levels of said protein, in<br>
preferred embodiments, modulators alter expression profiles, or expression profile nucleic acids or proteins provided<br>
herein, or downstream effector pathways. In one embodiment, the modulator suppresses a cancer phenotype, e.g. to a<br>
normal tissue fingerprint. In another embodiment, a modulator induced a cancer phenotype. Generally, a plurality of assay<br>
mixtures is run in parallel with different agent concentrations to obtain a differential response to the various concentrations.<br>
Typically, one of these concentrations serves as a negative control, i.e., at zero concentration or below the level of<br>
detection.<br>
[0070] Modulators, drug candidates or test compounds encompass numerous chemical classes, though<br>
typically they are organic molecules, preferably small organic compounds having a molecular weight of more than 100 and<br>
less than about 2,500 Daltons. Preferred small molecules are less than 2000, or less than 1500 or less than 1000 or less<br>
than 500 D. Candidate agents comprise functional groups necessary for structural interaction with proteins, particularly<br>
hydrogen bonding, and typically include at least an amine, carbonyl, hydroxyl or carboxyl group, preferably at least two of<br>
the functional chemical groups. The candidate agents often comprise cyclical carbon or heterocyclic structures and/or<br>
aromatic or polyaromatic structures substituted with one or more of the above functional groups. Modulators also comprise<br>
biomolecules such as peptides, saccharides, fatty acids, steroids, purines, pyrimidines, derivatives, structural analogs or<br>
combinations thereof. Particularly preferred are peptides. One class of modulators are peptides, for example of from about<br>
five to about 35 amino acids, with from about five to about 20 amino acids being preferred, and from about 7 to about 15<br>
being particularly preferred. Preferably, the cancer modulatory protein is soluble, includes a non-transmembrane region,<br>
and/or, has an N-terminal Cys to aid in solubility. In one embodiment, the C-terminus of the fragment is kept as a free acid<br>
and the N-terminus is a free amine to aid in coupling, i.e., to cysteine. In one embodiment, a cancer protein of the invention<br><br>
is conjugated to an immunogenic agent as discussed herein. In one embodiment, the cancer protein is conjugated to BSA.<br>
The pertides of the invention, e.g., of preferred lengths, can be linked to each other or to other amino acids to create a<br>
longer peptide/protein. The modulatory peptides can be digests of naturally occurring proteins as is outlined above,<br>
random peptides, or "biased" random peptides. In a preferred embodiment, peptide/protein-based modulators are<br>
antibodies, and fragments thereof, as defined herein.<br>
[0071] Modulators of cancer can also be nucleic acids. Nucleic acid modulating agents can be naturally occurring<br>
nucleic acids, random nucleic acids, or "biased" random nucleic acids. For example, digests of prokaryotic or eukaryotic<br>
genomes can be used in an approach analogous to that outlined above for proteins.<br>
[0072] The term "monoclonal antibody" refers to an antibody obtained from a population of substantially homogeneous<br>
antibodies, i.e., the antibodies comprising the population are identical except for possible naturally occurring mutations that are<br>
present in minor amounts.<br>
[0073] A "motif, as in biological motif of a PSCA-reiated protein, refers to any pattern of amino acids forming part of<br>
the primary sequence of a protein, that is associated with a particular function (e.g. protein-protein interaction, protein-DNA<br>
interaction, etc) or modification (e.g. that is phosphorylated, glycosylated or amidated), or localization (e.g. secretory<br>
sequence, nuclear localization sequence, etc.) or a sequence that is correlated with being immunogenic, either humorally or<br>
cellularly. A motif can be either contiguous or capable of being aligned to certain positions that are generally correlated<br>
with a certain function or property. In the context of HLA motifs, "motif refers to the pattern of residues in a peptide of<br>
defined length, usually a peptide of from about 8 to about 13 amino acids for a class I HLA motif and from about 6 to about<br>
25 amino acids for a class II HLA motif, which is recognized by a particular HLA molecule. Peptide motifs for HLA binding<br>
are typically different for each protein encoded by each human HLA allele and differ in the pattern of the primary and<br>
secondary anchor residues.<br>
[0074] A "pharmaceutical excipient" comprises a material such as an adjuvant, a carrier, pH-adjusting and buffering<br>
agents, tonicity adjusting agents, wetting agents, preservative, and the like.<br>
[0075] "Pharmaceutically acceptable" refers to a non-toxic, inert, and/or composition that is physiologically<br>
compatible with humans or other mammals.<br>
[0076] The term "polynucleotide" means a polymeric form of nucleotides of at least 10 bases or base pairs in length,<br>
either ribonucleotides or deoxynucleotides or a modified form of either type of nucleotide, and is meant to include single<br>
and double stranded forms of DNA and/or RNA. In the art, this term if often used interchangeably with "oligonucleotide". A<br>
polynucleotide can comprise a nucleotide sequence disclosed herein wherein thymidine (T), as shown for example in<br>
Figure 2, can also be uracil (U); this definition pertains to the differences between the chemical structures of DNA and RNA,<br>
in particular the observation that one of the four major bases in RNA is uracil (U) instead of thymidine (T).<br>
[0077] The term "polypeptide" means a polymer of at least about 4, 5, 6, 7, or 8 amino acids. Throughout the<br>
specification, standard three letter or single letter designations for amino acids are used. In the art, this term is often used<br>
interchangeably with "peptide" or "protein".<br>
[0078] An HLA "primary anchor residue" is an amino acid at a specific position along a peptide sequence which is<br>
understood to provide a contact point between the immunogenic peptide and the HLA molecule. One to three, usually two,<br>
primary anchor residues within a peptide of defined length generally defines a "motif for an immunogenic peptide. These<br>
residues are understood to fit in close contact with peptide binding groove of an HLA molecule, with their side chains buried<br>
in specific pockets of the binding groove. In one embodiment, for example, the primary anchor residues for an HLA class I<br>
molecule are located at position 2 (from the amino terminal position) and at the carboxyl terminal position of a 8, 9,10,11,<br><br>
or 12 residue peptide epitope in accordance with the invention. Alternatively, in another embodiment, the primary anchor<br>
residues of a peptide binds an HLA class II molecule are spaced relative to each other, rather than to the termini of a<br>
peptide; where the peptide is generally of at least 9 amino acids in length. The primary anchor positions for each motif and<br>
supermotif are set forth in Table IV. For example, analog peptides can be created by altering the presence or absence of<br>
particular residues in the primary and/or secondary anchor positions shown in Table IV. Such analogs are used to<br>
modulate the binding affinity and/or population coverage of a peptide comprising a particular HLA motif or supermotif.<br>
[0079] "Radioisotopes" include, but are not limited to the following (non-limiting exemplary uses are also set forth):<br>
"Examples of Medical Isotopes:<br>
Isotope Description of use<br><br>
Aclinium-225<br>
(AC-225) See Thorium-229 (Th-229)<br><br>
Actinium-227 Parent of Radium-223 (Ra-223) which is an alpha emitter used to treat metastases in the skeleton<br>
(AC-227) resulting from cancer (i.e., breast and prostate cancers), and cancer radioimmunotherapy<br><br>
Bismuth-212<br>
(Bi-212) See Thorium-228 (Th-228)<br><br>
Bismuth-212<br>
(Bi-212) See Thorium-228 (Th-228)<br><br>
Cadmum-109 Cancer detection<br>
(Cd-60)<br><br>
Cobalt-60 Radiation source for radiotherapy of cancer, for food irradiators, and for sterilization of medical<br>
(Co-60) supplies<br><br>
Copper -64<br>
(Cu-64) A Postion emitter used for cancer therapy and SPECT imaging<br><br>
Copper-67 Beta/gamma emitter used in cancer radioimmunotherapy and diagnostic studies (i.e., breast and<br>
(Cu-67) colon cancers, and lymphoma)<br><br>
Dysprosium-166<br>
(Dy-166) Cancer radioimmunotherapy<br><br>
Erbium-169 Rheumatoid arthritis treatment, particularly for the small joints associated with fingers and toes<br>
(Eu-152)<br><br>
Erbium-152<br>
(Eu-152) Radiation source for food irradiation and for sterilization of medical supplies<br><br>
Erbium-154<br>
(Eu-154) Radiation source for food irradiation and for sterilization of medical supplies<br><br>
Gadolinium-153 Osteoporosis detection and nuclear medical quality assurance devices<br>
(Gd-153)<br><br>
Gold-198 Implant and intracavity therapy of ovarian, prostate, and brain cancers<br>
(Au-198)<br><br>
Holmium-166 Multiple myeloma treatment in targeted skeletal therapy, cancer radioimmunotherapy, bone<br>
(Ho-166) marrow ablation, and rheumatoid arthritis treatment<br>
Osteoporosis detection, diagnostic imaging, tracer drugs, brain cancer treatment, radiolabeiing,<br>
lodine-125 tumor imaging, mapping of receptors in the brain, interstitial radiation therapy, brachytherapy for<br>
(1-125) treatment of prostate cancer, determination of glomerular filtration rate (GFR), determination of<br>
plasma volume, detection of deep vein thrombosis of the legs<br>
lodine-131 Tnyroicl function evaluation, thyroid disease detection, treatment of thyroid cancer as well as other<br>
(I-131) non-malignant thyroid diseases (i.e., Graves disease, goiters, and hyperthyroidism), treatment of<br>
 leukemia, lymphoma, and other forms of cancer (e.g., breast cancer) using radioimmunotherapy<br>
lridium-192 Brachytherapy, brain and spinal cord tumor treatment, treatment of blocked arteries (i.e.,<br>
(lr-192) arteriosclerosis and restenosis), and implants for breast and prostate tumors<br>
Lutetium-177 Cancer radioimmunotherapy and treatment of blocked arteries (i.e., arteriosclerosis and<br>
(Lu-177) restenosis)<br>
Parent of Technetium-99m (Tc-99m) which is used for imaging the brain, liver, lungs, heart, and<br>
Molybdenum-99 other organs. Currently, Tc-99m is the most widely used radioisotope used for diagnostic imaging<br>
(Mo-99) of various cancers and diseases involving the brain, heart, liver, lungs; also used in detection of<br>
deep vein thrombosis of the legs<br>
Osmium-194<br>
(Os-194) Cancer radioimmunotherapy<br>
Palladiunm-103<br>
(Pd-103) Prostate cancer trtment<br>
Palladiunm-193m(Pt-195m) Studies on biodistribution and metabolism of cisplatin, a chemotherapeutic drug<br>
Polycythemia rubra vera (blood cell disease) and leukemia treatment, bone cancer<br>
Phosphorus-32 diagnosis/treatment; colon, pancreatic, and liver cancer treatment; radiolabeling nucleic acids for<br>
(P-32) in vitro research, diagnosis of superficial tumors, treatment of blocked arteries (i.e.,<br>
arteriosclerosis and restenosis), and intracavity therapy<br>
Phosphorus-33 Leukemia treatment, bone disease diagnosis/treatment, radiolabeling, and treatment of blocked<br>
(P-33) arteries (i.e., arteriosclerosis and restenosis)<br>
Radium-223<br>
(Ra-223) See Actinium-227 (Ac-227)<br>
Rhenium-186 Bone cancer pain relief, rheumatoid arthritis treatment, and diagnosis and treatment of lymphoma<br>
(Re-186) and bone, breast, colon, and liver cancers using radioimmunotherapy<br>
Rhenium-188 Cancer diagnosis and treatment using radioimmunotherapy, bone cancer pain relief, treatment of<br>
(Re-188) rheumatoid arthritis, and treatment of prostate cancer<br>
Rhodium-105<br>
(Rh 105) Cancer radioimmunotherapy<br>
Samarium-145<br>
(Sm 145) Ocular cancer treatment<br>
(Sm-153) Cancer radioimmunotherapy and bone cancer pain relief<br>
(Sc-47) Cancer radioimmunotherapy and bone cancer pain relief<br>
Selenium-75 Radiotracer used in brain studies, imaging of adrenal cortex by gamma-scintigraphy, lateral<br>
(Se-75) locations of steroid secreting tumors, pancreatic scanning, detection of hyperactive parathyroid<br>
glands, measure rate of bile acid loss from the endogenous pool<br>
Strontium-85 „ , , ,. ...<br>
(Sr-85) e cancer detection and brain scans<br>
Strontium-89<br>
(Sr-89) cancer pain relief, multiple myeloma treatment, and osteoblastic therapy<br>
Technetium-99m<br>
(Tc-99m) Molybdenum-99 (Mo-99)<br>
(Th-228) Parent of Bismuth-212 (Bi-212)which is an alPna emitter used in cancer radioimmunotherapy<br>
Thoriurn-229 Parent of Actinium:225 (Ac-225) and grandparent of Bismuth-213 (Bi-213) which are alpha<br>
(Th-229) emitters used in cancer radioimmunotherapy<br>
(Tm-170) Gamma source for blood irradiators, energy source for implanted medical devices<br>
(Sn-117m) Cancer immunotherapy and bone cancer pain relief<br><br>
Parent for Rhenium-188 (Re-188) which is used for cancer diagnostics/treatment, bone cancer<br>
Tungsten-188 pain relief rneurnatoid arthritis treatment, and treatment of blocked arteries (i.e., arteriosclerosis<br>
(W-188) and restenosis)<br>
Xenon-127 Neuroimaging of brain disorders, high resolution SPECT studies, pulmonary function tests, and<br>
(Xe-127) cerebral blood flow studies<br>
Ytterbium-175<br>
(Yb-75) Cancer radioimmunotherapy<br>
Ytterbium-90 Microseeds obtained from irradiating Yttrium-89 (Y-89) for liver cancer treatment<br>
(Y -90)<br>
Yttrium-91 A gamma-emitting label for Yttrium-90 (Y-90) which is used for cancer radioimmunotherapy (i.e.,<br>
(Y-91) lymphoma, breast, colon, kidney, lung, ovarian, prostate, pancreatic, and inoperable liver cancers)<br>
[0080] By "randomized" or grammatical equivalents as herein applied to nucleic acids and proteins is meant that<br>
each nucleic acid and peptide consists of essentially random nucleotides and amino acids, respectively. These random<br>
peptides (or nucleic acids, discussed herein) can incorporate any nucleotide or amino acid at any position. The synthetic<br>
process can be designed to generate randomized proteins or nucleic acids, to allow the formation of all or most of the<br>
possible combinations over the length of the sequence, thus forming a library of randomized candidate bioactive<br>
proteinaceous agents.<br>
[0081] In one embodiment, a library is "fully randomized," with no sequence preferences or constants at any<br>
position. In another embodiment, the library is a "biased random" library. That is, some positions within the sequence<br>
either are held constant, or are selected from a limited number of possibilities. For example, the nucleotides or amino acid<br>
residues are randomized within a defined class, e.g., of hydrophobic amino acids, hydrophilic residues, sterically biased<br>
(either small or large) residues, towards the creation of nucleic acid binding domains, the creation of cysteines, for cross-<br>
linking, prolines for SH-3 domains, serines, threonines, tyrosines or histidines for phosphorylation sites, etc., or to purines,<br>
etc.<br>
[0082] A "recombinant" DNA or RNA molecule is a DNA or RNA molecule that has been subjected to molecular<br>
manipulation in vitro.<br>
[0083] Non-limiting examples of small molecules include compounds that bind or interact with PSCA, ligands<br>
including hormones, neuropeptides, chemokines, odorants, phospholipids, and functional equivalents thereof that bind and<br>
preferably inhibit PSCA protein function. Such non-limiting small molecules preferably have a molecular weight of less than<br>
about 10 kDa, more preferably below about 9, about 8, about 7, about 6, about 5 or about 4 kDa. In certain embodiments,<br>
small molecules physically associate with, or bind, PSCA protein; are not found in naturally occurring metabolic pathways;<br>
and/or are more soluble in aqueous than non-aqueous solutions<br>
[0084] "Stringency" of hybridization reactions is readily determinable by one of ordinary skill in the art, and generally<br>
is an empirical calculation dependent upon probe length, washing temperature, and salt concentration. In general, longer<br>
probes require higher temperatures for proper annealing, while shorter probes need lower temperatures. Hybridization<br>
generally depends on the ability of denatured nucleic acid sequences to reanneal when complementary strands are present<br>
in an environment below their melting temperature. The higher the degree of desired homology between the probe and<br>
hybridizable sequence, the higher the relative temperature that can be used. As a result, it follows that higher relative<br>
temperatures would tend to make the reaction conditions more stringent, while lower temperatures less so. For additional<br>
details and explanation of stringency of hybridization reactions, see Ausubel ef al., Current Protocols in Molecular Biology,<br>
Wiley Interscience Publishers, (1995).<br><br>
[0085] "Stringent conditions" or "high stringency conditions", as defined herein, are identified by, but not limited to,<br>
those that: (1) employ low ionic strength and high temperature for washing, for example 0.015 M sodium chloride/0.0015 M<br>
sodium citrate/0.1% sodium dodecyl sulfate at 50°C; (2) employ during hybridization a denaturing agent, such as<br>
formamide, for example, 50% (v/v) formamide with 0.1 % bovine serum albumin/0.1 % Ficoll/0.1 % polyvinylpyrrolidone/50<br>
mM sodium phosphate buffer at pH 6.5 with 750 mM sodium chloride, 75 mM sodium citrate at 42 °C; or (3) employ 50%<br>
formamide, 5 x SSC (0.75 M NaCI, 0.075 M sodium citrate), 50 mM sodium phosphate (pH 6.8), 0.1% sodium<br>
pyrophosphate, 5 x Denhardt's solution, sonicated salmon sperm DNA (50 µg/ml), 0.1% SDS, and 10% dextran sulfate at<br>
42 °C, with washes at 42°C in 0.2 x SSC (sodium chloride/sodium, citrate) and 50% formamide at 55 °C, followed by a high-<br>
stringency wash consisting of 0.1 x SSC containing EDTA at 55 °C. "Moderately stringent conditions" are described by, but<br>
not limited to, those in Sambrook et al., Molecular Cloning: A Laboratory Manual, New York: Cold Spring Harbor Press,<br>
1989, and include the use of washing solution and hybridization conditions (e.g., temperature, ionic strength and %SDS)<br>
less stringent than those described above. An example of moderately stringent conditions is overnight incubation at 37°C<br>
in a solution comprising: 20% formamide, 5 x SSC (150 mM NaCI, 15 mM trisodium citrate), 50 mM sodium phosphate (pH<br>
7.6), 5 x Denhardt's solution, 10% dextran sulfate, and 20 mg/mL denatured sheared salmon sperm DNA, followed by<br>
washing the filters in 1 x SSC at about 37-50°C. The skilled artisan will recognize how to adjust the temperature, ionic<br>
strength, etc. as necessary to accommodate factors such as probe length and the like.<br>
[0086] An HLA "supermotif is a peptide binding specificity shared by HLA molecules encoded by two or more HLA<br>
alleles. Overall phenotypic frequencies of HLA-supertypes in different ethnic populations are set forth in Table IV (F). The<br>
non-limiting constituents of various supetypes are as follows:<br>
A2: A*0201, A*0202, A*0203, A*0204, A* 0205, A*0206, A*6802, A*6901, A*0207<br>
A3: A3, A11, A31, A*3301, A*6801, A*0301, A*1101, A*3101<br>
B7: B7, B*3501-03, B*51, B*5301, B*5401, B*5501, B*5502, B*5601, B*6701, B*7801, B*0702, B*5101, B*5602<br>
B44: B*3701, B*4402, B*4403, B*60 (B*4001), B61 (B*4006)<br>
A1: A*0102, A*2604, A*3601, A*4301, A*8001<br>
A24: A*24, A*30, A*2403, A*2404, A*3002, A*3003<br>
B27: B*1401-02, B*1503, B*1509, B*1510, B*1518, B*3801-02, B*3901, B*3902, B*3903-04, B*4801-02,<br>
B*7301,B*2701-08<br>
B58: B*1516, B*1517, B*5701, B*5702, B58<br>
B62: B*4601, B52, B*1501 (B62), B*1502 (B75), B*1513 (B77)<br>
[0087] Calculated population coverage afforded by different HLA-supertype combinations are set forth in Table IV<br>
(G).<br>
[0088] As used herein "to treat" or "therapeutic" and grammatically related terms, refer to any improvement of any<br>
consequence of disease, such as prolonged survival, less morbidity, and/or a lessening of side effects which are the<br>
byproducts of an alternative therapeutic modality; full eradication of disease is not required.<br>
[0089] A "transgenic animal" (e.g., a mouse or rat) is an animal having cells that contain a transgene, which<br>
transgene was introduced into the animal or an ancestor of the animal at a prenatal, e.g., an embryonic stage. A<br>
"transgene" is a DNA that is integrated into the genome of a cell from which a transgenic animal develops.<br>
[0090] As used herein, an HLA or cellular immune response "vaccine" is a composition that contains or encodes<br>
one or more peptides of the invention. There are numerous embodiments of such vaccines, such as a cocktail of one or<br>
more individual peptides; one or more peptides of the invention comprised by a polyepitopic peptide; or nucleic acids that<br>
encode such individual peptides or polypeptides, e.g., a minigene that encodes a polyepitopic peptide. The "one or more<br>
peptides" can include any whole unit integer from 1-150 or more, e.g., at least 2,3,4,5,6,7, 8,9,10,11,12,13,14,15,<br>
16,17,18,19,20, 21, 22, 23, 24,25,26, 27, 28, 29, 30,31, 32, 33, 34, 35, 36, 37,38,39,40,41,42,43, 44,45,46,47,<br>
48,49, 50,55,60,65,70,75, 80, 85,90, 95,100,105,110,115,120,125,130,135,140,145, or 150 or more peptides of<br>
the invention. The peptides or polypeptides can optionally be modified, such as by lipidation, addition of targeting or other<br>
sequences. HLA class I peptides of the invention can be admixed with, or linked to, HLA class II peptides, to facilitate<br>
activation of both cytotoxic T lymphocytes and helper T lymphocytes. HLA vaccines can also comprise peptide-pulsed<br>
antigen presenting cells, e.g., dendritic ceils.<br>
[0091] The term "variant" refers to a molecule that exhibits a variation from a described type or norm, such as a protein<br>
that has one or more different amino acid residues in the corresponding positions) of a specifically described protein (e.g. the<br>
PSCA protein shown in Figure 2 or Figure 3. An analog is an example of a variant protein. Splice isoforms and single<br>
nucleotides polymorphisms (SNPs) are further examples of variants.<br>
[0092] The "PSCA-related proteins" of the invention include those specifically identified herein, as well as allelic variants,<br>
conservative substitution variants, analogs and homologs that can be isolated/generated and characterized without undue<br>
experimentation following the methods outlined herein or readily available in the art. Fusion proteins that combine parts of<br>
different PSCA proteins or fragments thereof, as well as fusion proteins of a PSCA protein and a heterologous polypeptide are<br>
also included. Such PSCA proteins are collectively referred to as the PSCA-related proteins, the proteins of the invention, or<br>
PSCA. The term "PSCA-related protein" refers to a polypeptide fragment or a PSCA protein sequence of 4,5,6,7,8,9,10,11,<br>
12,13,14,15,16,17,18,19,20,21,22,23,24,25, or more than 25 amino acids; or, at least 30,35,40,45,50,55,60,65,70,<br>
80, 85, 90, 95,100,105,110,115,120,125,130,135,140,145,150,155,160,165,170,175,180,185,190,195,200,<br>
225, 250, 275, 300, 325,350, 375,400,425,450,475,500,525,550, 575, or 576 or more amino acids.<br>
II.) PSCA Polynucleotides<br>
[0093] One aspect of the invention provides polynucleotides corresponding or complementary to all or part of a<br>
PSCA gene, mRNA, and/or coding sequence, preferably in isolated form, including polynucleotides encoding a PSCA-<br>
re/afed protein and fragments thereof, DNA, RNA, DNA/RNA hybrid, and related molecules, polynucleotides or<br>
oligonucleotides complementary to a PSCA gene or mRNA sequence or a part thereof, and polynucleotides or<br>
oligonucleotides that hybridize to a PSCA gene, mRNA, or to a PSCA encoding polynucleotide (collectively, "PSCA<br>
polynucleotides"). In all instances when referred to in this section, T can also be U in Figure 2.<br>
[0094] Embodiments of a PSCA polynucleotide include: a PSCA polynucleotide having the sequence shown in<br>
Figure 2, the nucleotide sequence of PSCA as shown in Figure 2 wherein T is U; at least 10 contiguous nucleotides of a<br>
polynucleotide having the sequence as shown in Figure 2; or, at least 10 contiguous nucleotides of a polynucleotide having<br>
the sequence as shown in Figure 2 where T is U. For example, embodiments of PSCA nucleotides comprise, without<br>
limitation:<br>
(I) a polynucleotide comprising, consisting essentially of, or consisting of a sequence as shown in<br>
Figure 2, wherein T can also be U;<br>
(H) a polynucleotide comprising, consisting essentially of, or consisting of the sequence as shown in<br>
Figure 2A, from nucleotide residue number 18 through nucleotide residue number 389, including the stop codon, wherein T<br>
can also be U;<br><br>
(III) a polynucleotide comprising, consisting essentially of, or consisting of the sequence as shown in<br>
Figs. 2B, from nucleotide residue number 56 through nucleotide residue number 427, including the stop codon, wherein T<br>
can also be U;<br>
(IV) a polynucleotide comprising, consisting essentially of, or consisting of the sequence as shown in<br>
Figure 2C, from nucleotide residue number 423 through nucleotide residue number 707, including the a stop codon,<br>
wherein T can also be U;<br>
(V) a polynucleotide comprising, consisting essentially of, or consisting of the sequence as shown in<br>
Figure 23 from nucleotide residue number 424 through nucleotide residue number 993, including the stop codon, wherein<br>
T cm also be U;<br>
(vII) a polynucleotide comprising, consisting essentially of, or consisting of the sequence as shown in<br>
Figure 2E from nucleotide residue number 910 through nucleotide residue number 1479, including the stop codon, wherein<br>
T can also be U;<br>
(VII) a polynucleotide comprising, consisting essentially of, or consisting of the sequence as shown in<br>
Figure 2F rom nucleotide residue number 83 through nucleotide residue number 427, including the stop codon, wherein T<br>
can also be U;<br>
(VIII) a polynucleotide comprising, consisting essentially of, or consisting of the sequence as shown in<br>
Figure 2G. from nucleotide residue number 56 through nucleotide residue number 427, including the stop codon, wherein T<br>
can also be U;<br>
(IX) a polynucleotide comprising, consisting essentially of, or consisting of (he sequence as shown in<br>
Figure 2H, from nucleotide residue number 424 through nucleotide residue number 993, including the stop codon, wherein<br>
T can also be (J;<br>
{X) a polynucleotide that encodes a PSCA-related protein thai is at least 90,91,92,93,94,95,96,97,98,<br>
S9 or 100% homologous to an entire amino acid sequence shown in Figure 2A-H;<br>
(XI) a polynucleotide that encodes a PSCA-related protein that is at least 90,91,92, 93,94, 95,96,97,98,<br>
99 or 100% identical to an entire amino acid sequence shown in Figure 2A-H;<br>
(XII) polynucleotide that encodes at least one peptide set forth in Tables VIII-XXI and XXII-XLIX;<br>
(XIII) a polynucleotide that encodes a peptide region of at least 5, 6,7, 8, 9,10,11,12,13,14,15,16,17,18,<br>
19, 20, 21,22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32,33, 34, 35 amino acids of a peptide of Figures 3A in any whole number<br>
increment up to 123 that includes at least 1, 2, 3, 4, 5, 6, 7, 8, 9,10,11,12,13,14, 15,16,17,18,19, 20, 21,22, 23, 24,<br>
25,26,27,28,29,30,31, 32, 33,34, 35 amino acid position(s) having a value greater than 0.5 in the Hydrophiiicity profile<br>
of Figure 5;<br>
(XIV) a polynucleotide that encodes a peptide region of at least 5,6,7, 8, 9,10,11,12,13,14,15,16,17,18,<br>
19, 20, 21, 22,23, 24, 25, 26, 27,28, 29, 30, 31,32,33, 34, 35 amino acids of a peptide of Figure 3A in any whole number<br>
increment up to 123 that includes 1, 2,3,4, 5, 6, 7,8, 9,10,11,12,13,14,15,16,17,18,19, 20,21,22, 23, 24, 25,26,<br>
27,28,29,30,31, 32,33, 34, 35 amino acid position(s) having a value less than 0.5 in the Hydropathicity profile of<br>
Figure 6;<br>
(XV) a polynucleotide that encodes a peptide region of at least 5,6,7,8,9,10,11,12,13,14,15,16,17,18,<br>
19,20,21,22,23, 24,25, 26,27,28,29,30,31,32,33,34,35 amino acids of a peptide of Figure 3A in any whole number<br>
increment up to 123 that includes 1,2, 3,4, 5, 6, 7,8, 9,10,11,12,13,14,15, 16,17,18,19,20,21, 22,23, 24,25,26,<br>
27,28,29,30,31,32,33,34,35 amino acid position(s) having a value greater than 0.5 in the Percent Accessible Residues<br>
profile of Figure 7;<br>
 (XVI) a polynucleotide that encodes a peptide region of at least 5, 6, 7, 8, 9,10,11,12,13,14,15,16,17,18,<br>
19, 20, 21, 22, 23, 24, 25, 26, 27,28,29, 30, 31, 32, 33,34, 35 amino acids of a peptide of Figure 3A in any whole number<br>
increment up to 123 that includes 1, 2, 3,4, 5,6, 7, 8,9,10,11,12,13,14,15,16,17,18,19, 20,21,22, 23, 24, 25, 26,<br>
27,28,29,30,31,32,33, 34,35 amino acid position(s) having a value greater than 0.5 in the Average Flexibility profile of<br>
Figure 8;<br>
(XVII) a polynucleotide that encodes a peptide region of at least 5,6,7,8,9,10,11,12,13,14,15,16,17,18,<br>
19, 20, 21, 22,23, 24, 25, 26, 27,28, 29, 30, 31, 32,33, 34, 35 amino acids of a peptide of Figure 3A in any whole number<br>
increment up to 123 that includes 1,2, 3, 4, 5, 6,7, 8,9,10,11,12,13,14,15,16,17,18,19,20,21,22, 23, 24, 25, 26,<br>
27,28,29,30,31,32,33,34,35 amino acid position(s) having a value greater than 0.5 in the Beta-turn profile of Figure 9;<br>
(XVIII) a polynucleotide that encodes a peptide region of at least 5,6,7, 8, 9,10,11,12,13,14,15,16,17,18,<br>
19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35 amino acids of a peptide of Figure 3B in any whole number<br>
increment up to 94 that includes 1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,<br>
28,29,30,31,32,33,34,35 amino acid position(s) having a value greater than 0.5 in the Hydrophilicity profile of Figure 5;<br>
(XIX) a polynucleotide that encodes a peptide region of at least 5,6,7,8, 9,10,11,12,13,14,15,16,17,18,<br>
19,20,21,22,23,24, 25,26,27, 28,29,30, 31,32,33,34,35 amino acids of a peptide of Figure 3B in any whole number<br>
increment up to 94 that includes 1,2, 3,4, 5, 6,7, 8, 9,10,11,12,13,14,15,16,17,18,19, 20, 21, 22, 23, 24, 25, 26, 27,<br>
28, 29,30,31,32, 33, 34,35 amino acid position(s) having a value less than 0.5 in the Hydropathicity profile of Figure 6;<br>
(XX) a polynucleotide that encodes a peptide region of at least 5,6,7, 8, 9,10,11,12,13,14,15,16,17,18,<br>
19, 20,21,22,23,24, 25,26, 27,28,29, 30,31,32,33,34,35 amino acids of a peptide of Figure 3B in any whole number<br>
increment up to 94 that includes 1,2, 3,4, 5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20, 21, 22,23,24, 25, 26, 27,<br>
28,29,30,31,32, 33, 34, 35 amino acid position(s) having a value greater than 0.5 in the Percent Accessible Residues<br>
profile of Figure 7;<br>
(XXI) a polynucleotide that encodes a peptide region of at least 5, 6,7,8, 9,10,11,12,13,14,15,16,17,18,<br>
19, 20, 21, 22, 23, 24, 25, 26, 27, 28,29, 30, 31, 32,33, 34, 35 amino acids of a peptide of Figure 3B in any whole number<br>
increment up to 94 that includes 1,2, 3,4,5,6,7, 8,9,10,11,12,13,14,15,16,17,18,19, 20,21,22, 23,24, 25,26, 27,<br>
28, 29, 30, 31, 32, 33,34, 35 amino acid position(s) having a value greater than 0.5 in the Average Flexibility profile of<br>
Figure 8;<br>
(XXII) a polynucleotide that encodes a peptide region of at least 5, 6,7,8,9,10,11,12,13,14,15,16,17,18,<br>
19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31,32,33,34,35 amino acids of a peptide of Figure 3B in any whole number<br>
increment up to 94 that includes 1,2, 3,4, 5,6, 7, 8, 9,10,11,12,13,14,15,16,17,18,19, 20, 21, 22, 23, 24, 25, 26, 27,<br>
28, 29, 30, 31, 32, 33, 34, 35 amino acid position(s) having a value greater than 0.5 in the Beta-turn profile of Figure 9;<br>
(XXII!) a polynucleotide that encodes a peptide region of at least 5, 6, 7, 8, 9,10,11,12,13,14,15,16,17,18,<br>
19,20,21, 22,23,24, 25,26,27,28,29,30,31,32,33,34,35 amino acids of a peptide of Figures 3C, 3E-3L in any whole<br>
number increment up to 189 that includes 1, 2, 3,4,5,6,7, 8, 9,10,11,12,13,14,15,16,17,18,19,20, 21, 22,23, 24,<br>
25, 26,27,28, 29,30, 31,32,33,34,35 amino acid position(s) having a value greater than 0.5 in the Hydrophilicity profile<br>
of Figure 5<br>
(XXIV) a polynucleotide that encodes a peptide region of at least 5,6,7,8,9,10,11,12,13,14,15,16,17,18,<br>
19,20,21, 22, 23,24, 25,26,27,28,29, 30,31,32,33,34,35 amino acids of a peptide of Figures 3C, 3E-3L in any whole<br><br>
number increment up to 189 that includes 1,2,3,4,5, 6, 7, 8, 9,10,11,12,13,14,15,16,17,18,19,20, 21, 22, 23, 24,<br>
25,26, 27, 28,29, 30, 31, 32, 33, 34,35 amino acid position(s) having a value less than 0.5 in the Hydropathicity profile of<br>
Figurb 6;<br>
(XXV) a polynucleotide that encodes a peptide region of at least 5, 6,7, 8, 9,10,11,12,13,14,15,16,17,18,<br>
19,20, 21, 22, 23,24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35 amino acids of a peptide of Figures 3C, 3E-3L in any whole<br>
number increment up to 189 that includes 1, 2, 3,4, 5, 6,7, 8, 9,10,11,12,13,14,15,16,17,18,19, 20, 21, 22, 23, 24,<br>
25, 26, 27, 28, 29, 30, 31,32, 33,34,35 amino acid position(s) having a value greater than 0.5 in the Percent Accessible<br>
Residues profile of Figure 7;<br>
(XXVI) a polynucleotide that encodes a peptide region of at least 5,6,7,8, 9,10,11,12,13,14,15,16,17,18,<br>
19, 20, 21,22,23, 24, 25, 26,27,28,29, 30, 31, 32, 33, 34, 35 amino acids of a peptide of Figures 3C, 3E-3L in any whole<br>
number increment up to 189 that includes 1,2, 3,4, 5,6,7,8, 9,10,11,12,13,14,15,16,17,18,19, 20, 21, 22, 23, 24,<br>
25,26,27,28,29,30,31,32,33,34,35 amino acid position(s) having a value greater than 0.5 in the Average Flexibility<br>
profile of Figure 8;<br>
(XXVII) a polynucleotide that encodes a peptide region of at least 5, 6, 7, 8, 9,10,11,12,13,14,15,16,17,18,<br>
19, 20, 21, 22,23, 24, 25, 26,27, 28, 29, 30, 31, 32, 33, 34, 35 amino acids of a peptide of Figures 3C, 3E-3L in any whole<br>
number increment up to 189 that includes 1,2,3,4, 5, 6,7, 8, 9,10,11,12,13,14,15,16,17,18,19,20,21, 22, 23, 24,<br>
25,26,27,28,29,30,31,32,33,34,35 amino acid position(s) having a value greater than 0.5 in the Beta-turn profile of<br>
Figure 9;<br>
(XXVIII) a polynucleotide that encodes a peptide region of at least 5, 6, 7, 8, 9,10,11,12,13,14,15,16,17,18,<br>
19, 20, 21, 22, 23, 24,25, 26, 27, 28, 29, 30, 31, 32, 33,34, 35 amino acids of a peptide of Figure 3D in any whole number<br>
increment up to 114 that includes 1, 2,3, 4,5, 6, 7,8, 9,10,11,12,13,14,15,16,17,18,19, 20, 21,22, 23, 24, 25, 26,<br>
27,28,29,30,31,32,33,34,35 amino acid position(s) having a value greater than 0.5 in the Hydrophiiicity profile of<br>
Figure 5<br>
(XXIX) a polynucleotide that encodes a peptide region of at least 5, 6, 7, 8, 9,10,11,12,13,14,15,16,17,18,<br>
19,20, 21, 22, 23, 24, 25, 26,27,28, 29, 30, 31, 32, 33, 34, 35 amino acids of a peptide of Figure 3D in any whole number<br>
increment up to 114 that includes 1, 2, 3,4, 5, 6, 7, 8, 9,10,11,12,13,14,15,16,17,18,19,20, 21, 22, 23, 24,25, 26,<br>
27,28,29, 30,31,32,33, 34,35 amino acid position(s) having a value less than 0.5 in the Hydropathicity profile of<br>
Figure 6;<br>
(XXX) a polynucleotide that encodes a peptide region of at least 5, 6, 7, 8, 9,10,11,12,13,14,15,16,17,18,<br>
19,20, 21, 22, 23,24, 25, 26, 27,28, 29, 30,31,32, 33, 34, 35 amino acids of a peptide of Figure 3D in any whole number<br>
increment up to 114 that includes 1, 2, 3,4, 5, 6, 7, 8, 9,10,11,12,13,14,15,16,17,18,19, 20, 21, 22,23, 24, 25, 26,<br>
27, 28, 29, 30, 31,32, 33,34,35 amino acid position(s) having a value greater than 0.5 in the Percent Accessible Residues<br>
profile of Figure 7;<br>
(XXXI) a polynucleotide that encodes a peptide region of at least 5, 6, 7, 8, 9,10,11,12,13,14,15,16,17,18,<br>
19,20, 21,22, 23,24, 25, 26, 27,28, 29, 30,31, 32,33,34, 35 amino acids of a peptide of Figure 3D in any whole number<br>
increment up to 114 that includes 1, 2, 3,4, 5, 6, 7,8, 9,10,11,12,13,14,15,16,17,18,19, 20, 21, 22,23, 24, 25, 26,<br>
27, 28, 29, 30,31, 32,33,34,35 amino acid position(s) having a value greater than 0.5 in the Average Flexibility profile of<br>
Figure 8;<br>
(XXXII) a polynucleotide that encodes a peptide region of at least 5, 6,7, 8, 9,10,11,12,13,14,15,16,17,18,<br>
19,20, 21,22,23,24,25, 26, 27,28, 29, 30, 31, 32, 33, 34, 35 amino acids of a peptide of Figure 3D in any whole number<br><br>
increment up to 114 that includes 1,2, 3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,<br>
27, 28^29, 30, 31,32,33,34,35 amino acid position(s) having a value greater than 0.5 in the Beta-turn profile of Figure 9;<br>
(XXXIII) a polynucleotide that is fully complementary to a polynucleotide of any one of (l)-(XXXII).<br>
(XXXIV) a peptide that is encoded by any of (I) to (XXXIII); and;<br>
(XXXV) a composition comprising a polynucleotide of any of (l)-(XXXlll) or peptide of (XXXIV) together with a<br>
pharmaceutical excipient and/or in a human unit dose form;<br>
(XXXVI) a method of using a polynucleotide of any (l)-(XXXIII) or peptide of (XXXIV) or a composition of (XXXV)<br>
in a method to modulate a cell expressing PSCA,;<br>
(XXXVII) a method of using a polynucleotide of any (l)-(XXXIII) or peptide of (XXXIV) or a composition of (XXXV)<br>
in a method to diagnose, prophylax, prognose, or treat an individual who bears a cell expressing PSCA;<br>
(XXXVIII) a method of using a polynucleotide of any (l)-(XXXIIi) or peptide of (XXXIV) or a composition of<br>
(XXXV) in a method to diagnose, prophylax, prognose, or treat an individual who bears a cell expressing PSCA, said cell<br>
from a cancer of a tissue listed in Table I<br>
(XXXIX) a method of using a polynucleotide of any (l)-(XXXIII) or peptide of (XXXIV) or a composition of (XXXV)<br>
in a method to diagnose, prophylax, prognose, or treat a a cancer<br>
(XL) a method of using a polynucleotide of any (l)-(XXXlll) or peptide of (XXXIV) or a composition of (XXXV)<br>
in a method to diagnose, prophylax, prognose, or treat a a cancer of a tissue listed in Table I; and,;<br>
(XLI) a method of using a polynucleotide of any (l)-(XXXIII) or peptide of (XXXIV) or a composition of (XXXV)<br>
in a method to identify or characterize a modulator of a cell expressing PSCA.;<br>
[0095] As used herein, a range is understood to disclose specifically all whole unit positions thereof.<br>
[0096] Typical embodiments of the invention disclosed herein include PSCA polynucleotides that encode specific<br>
portions of PSCA mRNA sequences (and those which are complementary to such sequences) such as those that encode<br>
the proteins and/or fragments thereof, for example: .<br>
(a) 4, 5, 6,7, 8, 9,10,11,12,13,14,15,16,17,18,19,20,21, 22,23,24, 25, 30, 35,40,45,50, 55, 60, 65, 70,<br>
75, 80, 85,90,95,100,105,110,115,120, and 123 or more contiguous amino acids of PSCA variant 1; the maximal<br>
lengths relevant for other variants are: variant 3,94 amino acids; variant 4,189 amino acids, variant 6,114 amino acids,<br>
variant 19,189 amino acids, variant 20,189 amoni acids, variant 21,189 amino acids, variant 22,189 amino acids, variant<br>
24,189 amino acids, variant 25,189 amino acids, variant 26,189 amino acids, and variant 27,189 amino acids.<br>
[0097] For example, representative embodiments of the invention disclosed herein include: polynucleotides and their<br>
encoded peptides themselves encoding about amino acid 1 to about amino acid 10 of the PSCA protein shown in Figure 2<br>
or Figure 3, polynucleotides encoding about amino acid 10 to about amino acid 20 of the PSCA protein shown in Figure 2<br>
or Figure 3, polynucleotides encoding about amino acid 20 to about amino acid 30 of the PSCA protein shown in Figure 2<br>
or Figure 3, polynucleotides encoding about amino acid 30 to about amino acid 40 of the PSCA protein shown in Figure 2<br>
or Figure 3, polynucleotides encoding about amino acid 40 to about amino acid 50 of the PSCA protein shown in Figure 2<br>
or Figure 3, polynucleotides encoding about amino acid 50 to about amino acid 60 of the PSCA protein shown in Figure 2<br>
or Figure 3, polynucleotides encoding about amino acid 60 to about amino acid 70 of the PSCA protein shown in Figure 2<br>
or Figure 3, polynucleotides encoding about amino acid 70 to about amino acid 80 of the PSCA protein shown in Figure 2<br>
or Figure 3, polynucleotides encoding about amino acid 80 to about amino acid 90 of the PSCA protein shown in Figure 2<br>
or Figure 3, polynucleotides encoding about amino acid 90 to about amino acid 100 of the PSCA protein shown in Figure 2<br><br>
or Figure 3, in increments of about 10 amino acids, ending at the carboxyl terminal amino acid set forth in Figure 2 or<br>
Figure 3. Accordingly, polynucleotides encoding portions of the amino acid sequence (of about 10 amino acids), of amino<br>
acids, 100 through the carboxyl terminal amino acid of the PSCA protein are embodiments of the invention. Wherein it is<br>
understood that each particular amino acid position discloses that position plus or minus five amino acid residues.<br>
[0098] Polynucleotides encoding relatively long portions of a PSCA protein are also within the scope of the invention.<br>
For example, polynucleotides encoding from about amino acid 1 (or 20 or 30 or 40 etc.) to about amino acid 20, (or 30, or<br>
40 or 50 etc.) of the PSCA protein "or variant" shown in Figure 2 or Figure 3 can be generated by a variety of techniques<br>
well known in the art. These polynucleotide fragments can include any portion of the PSCA sequence as shown in<br>
Figure 2.<br>
[0100] Additional illustrative embodiments of the invention disclosed herein include PSCA polynucleotide fragments<br>
encoding one or more of the biological motifs contained within a PSCA protein "or variant" sequence, including one or more<br>
of the motif-bearing subsequences of a PSCA protein "or variant" set forth in Tables VIII-XXI and XXII-XLIX. In another<br>
embodiment, typical polynucleotide fragments of the invention encode one or more of the regions of PSCA protein or<br>
variant that exhibit homology to a known molecule. In another embodiment of the invention, typical polynucleotide<br>
fragments can encode one or more of the PSCA protein or variant N-glycosylation sites, cAMP and cGMP-dependent<br>
protein kinase phosphorylation sites, casein kinase II phosphorylation sites or N-myristoylation site and amidation sites.<br>
[0101 ] Note that to determine the starting position of any peptide set forth in Tables VIII-XXI and Tables XXII to XLIX<br>
(collectively HLA Peptide Tables) respective to its parental protein, e.g., variant 1, variant 2, etc., reference is made to three<br>
factors: the particular variant, the length of the peptide in an HLA Peptide Table, and the Search Peptides listed in Table<br>
VII. Generally, a unique Search Peptide is used to obtain HLA peptides for a particular variant. The position of each<br>
Search Peptide relative to its respective parent molecule is listed in Table VII. Accordingly, if a Search Peptide begins at<br>
position "X", one must add the value "X minus 1" to each position in Tables VIII-XXI and Tables XXII-IL to obtain the actual<br>
position of the HLA peptides in their parental molecule. For example if a particular Search Peptide begins at position 150 of<br>
its parental molecule, one must add 150 -1, i.e., 149 to each HLA peptide amino acid position to calculate the position of<br>
that amino acid in the parent molecule.<br>
II.A.) Uses of PSCA Polynucleotides<br>
II.A.1.) Monitoring of Genetic Abnormalities<br>
[0102] The polynucleotides of the preceding paragraphs have a number of different specific uses. The human<br>
PSCA gene maps to the chromosomal location set forth in the Example entitled "Chromosomal Mapping of PSCA." For<br>
example, because the PSCA gene maps to this chromosome, polynucleotides that encode different regions of the PSCA<br>
proteins are used to characterize cytogenetic abnormalities of this chromosomal locale, such as abnormalities that are<br>
identified as being associated with various cancers. In certain genes, a variety of chromosomal abnormalities including<br>
rearrangements have been identified as frequent cytogenetic abnormalities in a number of different cancers (see e.g.<br>
Krajinovic et ai, Mutat Res. 382(3-4): 81-83 (1998); Johansson etal., Blood 86(10): 3905-3914 (1995) and Finger er a/.,<br>
P.N.A.S. 85(23): 9158-9162 (1988)). Thus, polynucleotides encoding specific regions of the PSCA proteins provide new<br>
tools that can be used to delineate, with greater precision than previously possible, cytogenetic abnormalities in the<br>
chromosomal region that encodes PSCA that may contribute to the malignant phenotype. In this context, these<br>
polynucleotides satisfy a need in the art for expanding the sensitivity of chromosomal screening in order to identify more<br><br>
subtle and less common chromosomal abnormalities (see e.g. Evans et al., Am. J. Obstet. Gynecol 171(4): 1055-1057<br>
[0103] Furthermore, as PSCA was shown to be highly expressed in prostate and other cancers, PSCA<br>
polynucleotides are used in methods assessing the status of PSCA gene products in normal versus cancerous tissues.<br>
Typically, polynucleotides that encode specific regions of the PSCA proteins are used to assess the presence of<br>
perturbations (such as deletions, insertions, point mutations, or alterations resulting in a loss of an antigen etc.) in specific<br>
regions of the PSCA gene, such as regions containing one or more motifs. Exemplary assays include both RT-PCR assays<br>
as well as single-strand conformation polymorphism (SSCP) analysis (see, e.g., Marrogi et al., J. Cutan. Pathol. 26(8): 369-<br>
378 (1999), both of which utilize polynucleotides encoding specific regions of a protein to examine these regions within the<br>
protein.<br>
(II.A.2.) Antisense Embodiments<br>
[0104] Other specifically contemplated nucleic acid related embodiments of the invention disclosed herein are<br>
genomic DNA, cDNAs, ribozymes, and antisense molecules, as well as nucleic acid molecules based on an alternative<br>
backbone, or including alternative bases, whether derived from natural sources or synthesized, and include molecules<br>
capable of inhibiting the RNA or protein expression of PSCA. For example, antisense molecules can be RNAs or other<br>
molecules, including peptide nucleic acids (PNAs) or non-nucleic acid molecules such as phosphorothioate derivatives that<br>
specifically bind DNA or RNA in a base pair-dependent manner. A skilled artisan can readily obtain these classes of<br>
nucleic acid molecules using the PSCA polynucleotides and polynucleotide sequences disclosed herein.<br>
[0105] Antisense technology entails the administration of exogenous oligonucleotides that bind to a target<br>
polynucleotide located within the cells. The term "antisense" refers to the fact that such oligonucleotides are<br>
complementary to their intracellular targets, e.g., PSCA. See for example, Jack Cohen, Oligodeoxynucleotides, Antisense<br>
Inhibitors of Gene Expression, CRC Press, 1989; and Synthesis 1:1-5 (1988). The PSCA antisense oligonucleotides of the<br>
present invention include derivatives such as S-oligonucleotides (phosphorothioate derivatives or S-oligos, see, Jack<br>
Cohen, supra), which exhibit enhanced cancer cell growth inhibitory action. S-oligos (nucleoside phosphorothioates) are<br>
isoelectronic analogs of an oligonucleotide (O-oligo) in which a nonbridging oxygen atom of the phosphate group is<br>
replaced by a sulfur atom. The S-oligos of the present invention can be prepared by treatment of the corresponding 0-<br>
oligos with 3H-1,2-benzodithiol-3-one-1,1-dioxide, which is a sulfur transfer reagent. See, e.g., Iyer, R. P. et al., J. Org.<br>
Chem. 55:4693-4698 (1990); and Iyer, R. P. et al., J. Am. Chem. Soc. 112:1253-1254 (1990). Additional PSCA antisense<br>
oligonucleotides of the present invention include morpholino antisense oligonucleotides known in the art (see, e.g.,<br>
Partridge et al., 1996, Antisense &amp; Nucleic Acid Drug Development 6:169-175).<br>
[0106] The PSCA antisense oligonucleotides of the present invention typically can be RNA or DNA that is<br>
complementary to and stably hybridizes with the first 100 5' codons or last 100 3' codons of a PSCA genomic sequence or<br>
the corresponding mRNA. Absolute complementarity is not required, although high degrees of complementarity are<br>
preferred. Use of an oligonucleotide complementary to this region allows for the selective hybridization to PSCA mRNA<br>
and not to mRNA specifying other regulatory subunits of protein kinase. In one embodiment, PSCA antisense<br>
oligonucleotides of the present invention are 15 to 30-mer fragments of the antisense DNA molecule that have a sequence<br>
that hybridizes to PSCA mRNA. Optionally, PSCA antisense oligonucleotide is a 30-mer oligonucleotide that is<br>
complementary to a region in the first 10 5' codons or last 10 3' codons of PSCA. Alternatively, the antisense molecules<br>
are modified to employ ribozymes in the inhibition of PSCA expression, see, e.g., L. A. Couture &amp; D. T. Stinchcomb; Trends<br>
Gaol 12:510-515 (1996).<br>
II.A.3.) Primers and Primer Pairs<br>
[0107] Further specific embodiments of these nucleotides of the invention include primers and primer pairs, which<br>
allow the specific amplification of polynucleotides of the invention or of any specific parts thereof, and probes that<br>
selectively or specifically hybridize to nucleic acid molecules of the invention or to any part thereof. Probes can be labeled<br>
with a detectable marker, such as, for example, a radioisotope, fluorescent compound, bioluminescent compound, a<br>
chemiluminescent compound, metal chelator or enzyme. Such probes and primers are used to detect the presence of a<br>
PSCA polynucleotide in a sample and as a means for detecting a cell expressing a PSCA protein.<br>
[0108] Examples of such probes include polypeptides comprising all or part of the human PSCA cDNA sequence<br>
shown in Figure 2. Examples of primer pairs capable of specifically amplifying PSCA mRNAs are also described in the<br>
Examples. As will be understood by the skilled artisan, a great many different primers and probes can be prepared based<br>
on the sequences provided herein and used effectively to amplify and/or detect a PSCA mRNA.<br>
[0109] The PSCA polynucleotides of the invention are useful for a variety of purposes, including but not limited to<br>
their use as probes and primers for the amplification and/or detection of the PSCA gene(s), mRNA(s), or fragments thereof;<br>
as reagents for the diagnosis and/or prognosis of prostate cancer and other cancers; as coding sequences capable of<br>
directing the expression of PSCA polypeptides; as tools for modulating or inhibiting the expression of the PSCA gene(s)<br>
and/or translation of the PSCA transcript(s); and as therapeutic agents.<br>
[0110] The present invention includes the use of any probe as described herein to identify and isolate a PSCA or<br>
PSCA related nucleic acid sequence from a naturally occurring source, such as humans or other mammals, as well as the<br>
isolated nucleic acid sequence perse, which would comprise all or most of the sequences found in the probe used.<br>
II.A.4.) Isolation of PSCA-Encodinq Nucleic Acid Molecules<br>
[0111] The PSCA cDNA sequences described herein enable the isolation of other polynucleotides encoding PSCA<br>
gene product(s), as well as the isolation of polynucleotides encoding PSCA gene product homologs, alternatively spliced<br>
isoforms, allelic variants, and mutant forms of a PSCA gene product as well as polynucleotides that encode analogs of<br>
PSCA-related proteins. Various molecular cloning methods that can be employed to isolate full length cDNAs encoding a<br>
PSCA gene are well known (see, for example, Sambrook, J. et al., Molecular Cloning; A Laboratory Manual, 2d edition,<br>
Cold Spring Harbor Press, New York, 1989; Current Protocols in Molecular Biology. Ausubel et al, Eds, Wiley and Sons,<br>
1995). For example, lambda phage cloning methodologies can be conveniently employed, using commercially available<br>
cloning systems (e.g., Lambda ZAP Express, Stratagene). Phage clones containing PSCA gene cDNAs can be identified<br>
by probing with a labeled PSCA cDNA or a fragment thereof. For example, in one embodiment, a PSCA cDNA (e.g.,<br>
Figure 2) or a portion thereof can be synthesized and used as a probe to retrieve overlapping and full-length cDNAs<br>
corresponding to a PSCA gene. A PSCA gene itself can be isolated by screening genomic DNA libraries, bacterial artificial<br>
chromosome libraries (BACs), yeast artificial chromosome libraries (YACs), and the like, with PSCA DNA probes or<br>
primers.<br>
II.A. 5) Recombinant Nucleic Acid Molecules and Host-Vector Systems<br>
[0112] The invention also provides recombinant DNA or RNA molecules containing a PSCA polynucleotide, a<br>
fragment, analog or homologue thereof, including but not limited to phages, plasmids, phagemids, cosmids, YACs, BACs,<br>
as well as various viral and non-viral vectors well known in the art, and cells transformed or transfected with such<br>
recombinant DNA or RNA molecules. Methods for generating such molecules are well known (see, for example, Sambrook<br>
et al., 1989, supra).<br>
[0113] The invention further provides a host-vector system comprising a recombinant DNA molecule containing a<br>
PSCA polynucleotide, fragment, analog or homologue thereof within a suitable prokaryotic or eukaryotic host cell.<br>
Examples of suitable eukaryotic host cells include a yeast ceil, a plant cell, or an animal cell, such as a mammalian ceil or<br>
an insect cell (e.g., a baculovirus-infectible cell such as an Sf9 or HighFive cell). Examples of suitable mammalian cells<br>
include various prostate cancer cell lines such as DU145 and TsuPr1, other transfectable or transducible prostate cancer<br>
cell lines, primary cells (PrEC), as well as a number of mammalian cells routinely used for the expression of recombinant<br>
proteins (e.g., COS, CHO, 293,293T cells). More particularly, a polynucleotide comprising the coding sequence of PSCA<br>
or a fragment, analog or homolog thereof can be used to generate PSCA proteins or fragments thereof using any number<br>
of host-vector systems routinely used and widely known in the art.<br>
[0114] A wide range of host-vector systems suitable for the expression of PSCA proteins or fragments thereof are<br>
available, see for example, Sambrook etal., 1989, supra; Current Protocols in Molecular Biology, 1995, supra). Preferred<br>
vectors for mammalian expression include but are not limited to pcDNA 3.1 myc-His-tag (Invitrogen) and the retroviral<br>
vector pSRatkneo (Muller et al., 1991, MCB 11:1785). Using these expression vectors, PSCA can be expressed in several<br>
prostate cancer and non-prostate cell lines, including for example 293,293T, rat-1, NIH 3T3 and TsuPrl The host-vector<br>
systems of the invention are useful for the production of a PSCA protein or fragment thereof. Such host-vector systems<br>
can be employed to study the functional properties of PSCA and PSCA mutations or analogs.<br>
[0115] Recombinant human PSCA protein or an analog or homolog or fragment thereof can be produced by<br>
mammalian cells transfected with a construct encoding a PSCA-related nucleotide. For example, 293T cells can be<br>
transfected with an expression plasmid encoding PSCA or fragment, analog or homolog thereof, a PSCA-related protein is<br>
expressed in the 293T cells, and the recombinant PSCA protein is isolated using standard purification methods (e.g.,<br>
affinity purification using anti-PSCA antibodies). In another embodiment, a PSCA coding sequence is subcloned into the<br>
retroviral vector pSRaMSVtkneo and used to infect various mammalian cell lines, such as NIH 3T3, TsuPrl, 293 and rat-1<br>
in order to establish PSCA expressing cell lines. Various other expression systems well known in the art can also be<br>
employed. Expression constructs encoding a leader peptide joined in frame to a PSCA coding sequence can be used for<br>
the generation of a secreted form of recombinant PSCA protein.<br>
[0116] As discussed herein, redundancy in the genetic code permits variation in PSCA gene sequences. In<br>
particular, it is known in the art that specific host species often have specific codon preferences, and thus one can adapt<br>
the disclosed sequence as preferred for a desired host. For example, preferred analog codon sequences typically have<br>
rare codons (i.e., codons having a usage frequency of less than about 20% in known sequences of the desired host)<br>
replaced with higher frequency codons. Codon preferences for a specific species are calculated, for example, by utilizing<br>
codon usage tables available on the INTERNET such as at URL dna.affrc.go.jp/~nakamura/codon.html.<br>
[0117] Additional sequence modifications are known to enhance protein expression in a cellular host. These include<br>
elimination of sequences encoding spurious polyadenylation signals, exon/intron splice site signals, transposon-like<br>
repeats, and/or other such well-characterized sequences that are deleterious to gene expression. The GG content of the<br>
sequcse is adjusted to levels average for a given cellular host, as calculated by reference to known genes expressed in<br>
the host cell. Where possible, the sequence is modified to avoid predicted hairpin secondary mRNA structures. Other<br>
useful modifications include the addition of a translational initiation consensus sequence at the start of the open reading<br>
frame, as described in Kozak, Mol. Cell Biol., 9:5073-5080 (1989). Skilled artisans understand that the general rule that<br>
eukaryotic ribosomes initiate translation exclusively at the 5' proximal AUG codon is abrogated only under rare conditions<br>
(see, e.g., Kozak PNAS 92(7): 2662-2666, (1995) and Kozak IMAR 15(20): 8125-8148 (1987)).<br>
111.) PSCA-related Proteins<br>
[0118] Another aspect of the present invention provides PSCA-related proteins. Specific embodiments of PSCA<br>
proteins comprise a polypeptide having all or part of the amino acid sequence of human PSCA as shown in Figure 2 or<br>
Figure 3. Alternatively, embodiments of PSCA proteins comprise variant, homolog or analog polypeptides that have<br>
alterations in the amino acid sequence of PSCA shown in Figure 2 or Figure 3.<br>
[0119] Embodiments of a PSCA polypeptide include: a PSCA polypeptide having a sequence shown in Figure 2, a<br>
peptide sequence of a PSCA as shown in Figure 2 wherein T is U; at least 10 contiguous nucleotides of a polypeptide<br>
having the sequence as shown in Figure 2; or, at least 10 contiguous peptides of a polypeptide having the sequence as<br>
shown in Figure 2 where T is U. For example, embodiments of PSCA peptides comprise, without limitation:(1) a<br>
protein comprising, consisting essentially of, or consisting of an amino acid sequence as shown in Figure 2A-H or<br>
Figure 3A-L;<br>
(II) a PSCA-related protein that is at least 90,91,92,93,94,95,96,97,98,99 or 100% homologous to an<br>
entire amino acid sequence shown in Figure 2A-H or 3A-L;<br>
(III) a PSCA-related protein that is at least 90,91,92,93,94,95,96,97,98,99 or 100% identical to an<br>
entire amino acid sequence shown in Figure 2A-H or 3A-L;<br>
(IV) a protein that comprises at least one peptide set forth in Tables VIII to XLIX, optionally with a proviso<br>
that it is not an entire protein of Figure 2;<br>
(V) a protein that comprises at least one peptide set forth in Tables VIII-XXI, collectively, which peptide is<br>
also set forth in Tables XXII to XLIX, collectively, optionally with a proviso that it is not an entire protein of Figure 2;<br>
(VI) a protein that comprises at least two peptides selected from the peptides set forth in Tables VIII-XLJX,<br>
optionally with a proviso that it is not an entire protein of Figure 2;<br>
(VII) a protein that comprises at least two peptides selected from the peptides set forth in Tables VIII to XLIX<br>
collectively, with a proviso that the protein is not a contiguous sequence from an amino acid sequence of Figure 2;<br>
(VIII) a protein that comprises at least one peptide selected from the peptides set forth in Tables VIII-XXI; and<br>
at least one peptide selected from the peptides set forth in Tables XXII to XLIX, with a proviso that the protein is not a<br>
contiguous sequence from an amino acid sequence of Figure 2;<br>
(IX) a polypeptide comprising at least 5, 6,7, 8, 9,10,11,12,13,14,15,16,17,18,19,20,21,22, 23,24,<br>
25, 26,27,28,29,30,31,32,33,34,35 amino acids of a protein of Figure 3A in any whole number increment up to 123<br>
respectively that includes at least 1, 2, 3,4,5,6,7, 8,9,10,11,12,13,14,15,16,17,18,19, 20, 21, 22, 23,24, 25, 26,<br>
27,28,29,30,31,32, 33, 34,35 amino acid position(s) having a value greater than 0.5 in the Hydrophilicity profile of<br>
Figure 5;<br>
(X) a polypeptide comprising at least 5, 6, 7, 8, 9,10,11,12,13,14,15,16,17,18,19, 20, 21,22,23,24,<br>
25,26 27,28,29,30, 31, 32,33,34,35 amino acids of a protein of Figure 3A, in any whole number increment up to 123<br>
respectively that includes at least at least 1, 2, 3,4, 5, 6, 7, 8, 9,10,11,12,13,14,15,16,17,18,19, 20, 21, 22, 23, 24,<br>
25,26,27, 28,29,30, 31, 32, 33, 34,35 amino acid position(s) having a value less than 0.5 in the Hydropathicity profile of<br>
Figure 6;<br>
(XI) a polypeptide comprising at least 5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,<br>
25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35 amino acids of a protein of Figure 3A, in any whole number increment to 123<br>
respectively that includes at least at least 1, 2, 3,4, 5, 6, 7, 8,9,10,11,12,13,14,15,16,17,18,19,20, 21, 22, 23, 24,<br>
25,26,27,28,29, 30,31, 32, 33, 34,35 amino acid position(s) having a value greater than 0.5 in the Percent Accessible<br>
Residues profile of Figure 7;<br>
(XII) a polypeptide comprising at least 5, 6, 7, 8, 9,10,11,12,13,14,15,16,17,18,19, 20,21,22, 23, 24,<br>
25, 26,27,28,29, 30,31, 32,33,34,35 amino acids of a protein of Figure 3A, in any whole number increment up to<br>
123respectively that includes at least at least 1,2,3,4, 5, 6,7, 8, 9,10,11,12,13,14,15,16,17,18,19, 20,21, 22, 23,<br>
24,25,26,27,28,29,30,31,32,33,34,35 amino acid position(s) having a value greater than 0.5 in the Average Flexibility<br>
profile of Figure 8;<br>
(XIII) a polypeptide comprising at least 5, 6, 7, 8, 9,10,11,12,13,14,15,16,17,18,19, 20, 21, 22,23, 24,<br>
25,26,27,28,29,30,31,32,33,34, amino acids of a protein of Figure 3A in any whole number increment up to 123<br>
respectively that includes at least at least 1, 2, 3,4, 5, 6, 7,8, 9,10,11,12,13,14,15,16,17,18,19,20, 21,22,23, 24,<br>
25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35 amino acid position(s) having a value greater than 0.5 in the Beta-turn profile of<br>
Figure 9;<br>
(XIV) a polypeptide comprising at least 5, 6, 7, 8,9,10,11,12,13,14,15,16,17,18,19, 20,21,22,23, 24,<br>
25,26,27,28,29, 30, 31, 32, 33, 34,35 amino acids of a protein of Figure 3B, in any whole number increment up to 94<br>
respectively that includes at least 1, 2, 3, 4, 5, 6, 7, 8, 9,10,11,12,13,14,15,16,17,18,19, 20, 21,22,23, 24,25,26,<br>
27, 28, 29, 30, 31, 32, 33, 34, 35 amino acid position(s) having a value greater than 0.5 in the Hydrophilicity profile of<br>
Figure 5;<br>
(XV) a polypeptide comprising at least 5, 6, 7,8, 9,10,11,12,13,14,15,16,17,18,19, 20,21,22, 23, 24,<br>
25, 26,27,28,29, 30, 31, 32, 33, 34, 35 amino acids of a protein of Figure 3B, in any whole number increment up to 94<br>
respectively that includes at least at least 1, 2, 3,4,5, 6, 7, 8,9,10,1-1,12,13,14,15,16,17,18,19,20,21, 22, 23,24,<br>
25, 26, 27, 28,29, 30, 31, 32, 33, 34, 35 amino acid position(s) having a value less than 0.5 in the Hydropathicity profile of<br>
Figure 6;<br>
(XVI) a polypeptide comprising at least 5, 6, 7, 8, 9,10,11,12,13,14,15,16,17,18,19, 20, 21,22, 23, 24,<br>
25, 26, 27,28,29,30,31,32,33,34,35 amino acids of a protein of Figure 3B, in any whole number increment up to 94<br>
respectively that includes at least at least 1, 2, 3,4, 5,6, 7, 8, 9,10,11,12,13,14,15,16,17,18,19,20,21, 22,23,24,<br>
25,26,27,28,29,30,31,32, 33,34, 35 amino acid position(s) having a value greater than 0.5 in the Percent Accessible<br>
Residues profile of Figure 7;<br>
(XVII) a polypeptide comprising at least 5, 6, 7,8, 9,10,11,12,13,14,15,16,17,18,19, 20, 21,22, 23,24,<br>
25,26,27,28,29,30, 31, 32,33,34,35 amino acids of a protein of Figure 3B, in any whole number increment up to 94<br>
respectively that includes at least at least 1, 2, 3,4, 5,6, 7, 8, 9,10,11,12,13,14,15,16,17,18,19, 20, 21,22, 23, 24,<br>
25,26,27,28,29,30, 31,32, 33, 34,35 amino acid position(s) having a value greater than 0.5 in the Average Flexibility<br>
profile of Figure 8;<br><br>
(XVIII) a polypeptide comprising at least 5, 6, 7, 8, 9,10,11,12,13,14,15,16,17,18,19, 20, 21, 22, 23, 24,<br>
25,26, 27, 28,29,30,31,32,33,34, amino acids of a protein of Figure 3B in any whole number increment up to 94<br>
respectively that includes at least at least 1,2,3,4,5,6,7, 8, 9,10,11,12,13,14,15,16,17,18,19,20, 21, 22, 23, 24,<br>
25,26,27,28,29,30,31, 32, 33,34,35 amino acid position(s) having a value greater than 0.5 in the Beta-turn profile of<br>
Figure 9;<br>
(XIX) a polypeptide comprising atleast 5,6,7, 8,9,10,11,12,13,14,15,16,17,18,19, 20, 21,22, 23, 24,<br>
25, 26, 27, 28,29, 30, 31,32,33, 34,35 amino acids of a protein of Figure 3C, 3E-3L in any whole number increment up to<br>
189 respectively that includes at least 1, 2,3,4,5, 6,7, 8,9,10,11,12,13,14,15,16,17,18,19, 20, 21, 22, 23, 24, 25,<br>
26,27,28,29,30,31, 32,33,34,35 amino acid position(s) having a value greater than 0.5 in the Hydrophilicity profile of<br>
Figure 5;<br>
(XX) a polypeptide comprising at least 5, 6, 7, 8,9,10,11,12,13,14,15,16,17,18,19,20, 21, 22,23,24,<br>
25,26,27,28,29,30,31,32,33,34,35 amino acids of a protein of Figure 3C, 3E-3L in any whole number increment up to<br>
189 respectively that includes atleast atleast 1,2, 3,4,5,6,7,8, 9,10,11,12,13,14,15,16,17,18,19, 20, 21, 22,23,<br>
24,25,26,27,28,29,30,31,32, 33,34,35 amino acid position(s) having a value less than 0.5 in the Hydropathicity<br>
profile of Figure 6;<br>
(XXI) a polypeptide comprising at least 5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20, 21, 22,23,24,<br>
25,26, 27, 28, 29,30,31,32,33,34,35 amino acids of a protein of Figure 3C, 3E-3L in any whole number increment up to<br>
189 respectively that includes at least at least 1, 2,3,4, 5, 6,7, 8, 9,10,11,12,13,14,15,16,17,18,19, 20, 21, 22, 23,<br>
24, 25,26,27, 28, 29, 30, 31, 32,33,34, 35 amino acid position(s) having a value greater than 0.5 in the Percent<br>
Accessible Residues profile of Figure 7;<br>
(XXII) a polypeptide comprising at least 5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20, 21,22,23,24,<br>
25, 26,27,28,29,30, 31, 32,33,34,35 amino acids of a protein of Figure 3C, 3E-3L in any whole number increment up to<br>
189 respectively that includes at least at least 1,2,3,4,5, 6,7, 8,9,10,11,12,13,14,15,16,17,18,19, 20, 21, 22,23,<br>
24, 25,26, 27, 28, 29, 30, 31, 32, 33, 34,35 amino acid position(s) having a value greater than 0.5 in the Average Flexibility<br>
profile of Figure 8;<br>
(XXIII) a polypeptide comprising at least 5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,<br>
25, 26,27,28, 29, 30,31, 32, 33,34, amino acids of a protein of Figure 3C, 3E-3L in any whole number increment up to<br>
189 respectively that includes at least at least 1,2,3,4, 5, 6,7, 8, 9,10,11,12,13,14,15,16,17,18,19, 20, 21,22, 23,<br>
24,25,26,27,28,29,30,31,32,33,34,35 amino acid position(s) having a value greater than 0.5 in the Beta-turn profile<br>
of Figure 9;<br>
(XXIV) a polypeptide comprising at least 5, 6, 7,8, 9,10,11,12,13,14,15,16,17,18,19, 20,21, 22, 23, 24,<br>
25, 26,27,28, 29,30, 31, 32,33,34,35 amino acids of a protein of Figure 3D, in any whole number increment up to 114<br>
respectively that includes at least 1, 2,3,4, 5,6,7, 8,9,10,11,12,13,14,15,16,17,18,19,20,21,22, 23, 24, 25, 26,<br>
27,28,29,30, 31, 32, 33, 34, 35 amino acid position(s) having a value greater than 0.5 in the Hydrophilicity profile of<br>
Figure 5;<br>
(XXV) a polypeptide comprising at least 5,6,7, 8,9,10,11,12,13,14,15,16,17,18,19,20,21, 22,23, 24,<br>
25,26,27,28,29,30,31,32,33,34,35 amino acids of a protein of Figure 3D, in any whole number increment up to 114<br>
respectively that includes at least at least 1, 2,3,4, 5, 6,7, 8,9,10,11,12,13,14,15,16,17,18,19, 20, 21, 22, 23, 24,<br>
25,26,27,28,29,30, 31, 32,33,34,35 amino acid position(s) having a value less than 0.5 in the Hydropathicity profile of<br>
Figure 6;<br><br>
(XXVI) a polypeptide comprising at least 5,6,7,8, 9,10,11,12,13,14,15,16,17,18,19,20, 21, 22, 23,24,<br>
25, 26, 27, 28, 29,30, 31, 32, 33, 34, 35 amino acids of a protein of Figure 3D, in any whole number increment up to 114<br>
respectively that includes at least at least 1, 2, 3,4, 5,6,7, 8, 9,10,11,12,13,14,15,16,17,18,19,20, 21, 22,23, 24,<br>
25,26,27,28,29, 30,31,32, 33,34,35 amino acid position(s) having a value greater than 0.5 in the Percent Accessible<br>
Residues profile of Figure 7;<br>
(XXVII) a polypeptide comprising at least 5,6, 7, 8, 9,10,11,12,13,14,15,16,17,18,19, 20, 21, 22,23, 24,<br>
25, 26, 27,28, 29, 30, 31, 32, 33, 34, 35 amino acids of a protein of Figure 3D, in any whole number increment up to 114<br>
respectively that includes at least at least 1,2,3,4,5, 6, 7, 8, 9,10,11,12,13,14,15,16,17,18,19,20, 21, 22,23,24,<br>
25,26, 27, 28, 29, 30, 31, 32, 33, 34, 35 amino acid position(s) having a value greater than 0.5 in the Average Flexibility<br>
profile of Figure 8;<br>
(XXVill) a polypeptide comprising at least 5,6,7,8, 9,10,11,12,13,14,15,16,17,18,19,20,21,22,23, 24,<br>
25,26,27, 28,29, 30,31, 32, 33,34, amino acids of a protein of Figure 3D in any whole number increment up to 114<br>
respectively that includes at least at least 1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,<br>
25, 26, 27,28, 29, 30, 31, 32, 33, 34,35 amino acid position(s) having a value greater than 0.5 in the Beta-turn profile of<br>
Figure 9;<br>
(XXIX) a peptide that occurs at least twice in Tables VIII-XXI and XXII to XLIX, collectively;<br>
(XXX) a peptide that occurs at least three times in Tables VIII-XXI and XXII to XLIX, collectively;<br>
(XXXI) a peptide that occurs at least four times in Tables VIII-XXI and XXII to XLIX, collectively;<br>
(XXXII) a peptide that occurs at least five times in Tables VIII-XXI and XXII to XLIX, collectively;<br>
(XXXIII) a peptide that occurs at least once in Tables VIII-XXI, and at least once in tables XXII to XLIX;<br>
(XXXIV) a peptide that occurs at least once in Tables VIII-XXI, and at least twice in tables XXII to XLIX;<br>
(XXXV) a peptide that occurs at least twice in Tables VIII-XXI, and at least once in tables XXII to XLIX;<br>
(XXXVI) a peptide that occurs at least twice in Tables VIII-XXI, and at least twice in tables XXII to XLIX;<br>
(XXXVII) a peptide which comprises one two, three, four, or five of the following characteristics, or an<br>
oligonucleotide encoding such peptide:<br>
i) a region of at least 5 amino acids of a particular peptide of Figure 3, in any whole number increment<br>
up to the full length of that protein in Figure 3, that includes an amino acid position having a value equal to or greater than<br>
0.5, 0.6,0.7,0.8,0.9, or having a value equal to 1.0, in the Hydrophilicity profile of Figure 5;<br>
ii) a region of at least 5 amino acids of a particular peptide of Figure 3, in any whole number increment<br>
up to the full length of that protein in Figure 3, that includes an amino acid position having a value equal to or'less than 0.5,<br>
0.4,0.3,0.2,0.1, or having a value equal to 0.0, in the Hydropathicity profile of Figure 6;<br>
iii) a region of at least 5 amino acids of a particular peptide of Figure 3, in any whole number increment<br>
up to the full length of that protein in Figure 3, that includes an amino acid position having a value equal to or greater than<br>
0.5,0.6,0.7,0.8,0.9, or having a value equal to 1.0, in the Percent Accessible Residues profile of Figure 7;<br>
iv) a region of at least 5 amino acids of a particular peptide of Figure 3, in any whole number increment<br>
up to the full length of that protein in Figure 3, that includes an amino acid position having a value equal to or greater than<br>
0.5,0.6,0.7,0.8,0.9, or having a value equal to 1.0, in the Average Flexibility profile of Figure 8; or,<br>
v) a region of at least 5 amino acids of a particular peptide of Figure 3, in any whole number increment<br>
up to the full length of that protein in Figure 3, that includes an amino acid position having a value equal to or greater than<br>
0.5, 0.6, 0.7, 0.8, 0.9, or having a value equal to 1.0, in the Beta-turn profile of Figure 9;<br><br>
(XXXVIII) a composition comprising a peptide of (l)-(XXXVII) or an antibody or binding region thereof together<br>
with a pharmaceutical excipient and/or in a human unit dose form;<br>
(XXXIX) a method of using a peptide of (l)-(XXXVil), or an antibody or binding region thereof or a composition of<br>
(XXXVIII) in a method to modulate a cell expressing PSCA;<br>
(XL) a method of using a peptide of (l)-(XXXVII) or an antibody or binding region thereof or a composition of<br>
(XXXVIII) m a method to diagnose, prophylax, prognose, or treat an individual who bears a cell expressing PSCA;<br>
(XLI) a method of using a peptide of (l)-(XXXVII) or an antibody or binding region thereof or a composition<br>
(XXXVIII) In a method to diagnose, prophylax, prognose, or treat an individual who bears a cell expressing PSCA, said cell<br>
from a cancer ot a tissue listed in Table 1;<br>
(XLII) a method of using a peptide of (l)-(XXXVII) or an antibody or binding region thereof or a composition of<br>
(XXXVIX) In a method to diagnose, prophylax, prognose, or treat a a cancer;<br>
(XLlll) a method of using a peptide of (l)-(XXXVIl) or an antibody or binding region thereof or a composition of<br>
(XXXX) In a method to diagnose, prophylax, prognose, or treat a a cancer of a tissue listed in Table I; and;<br>
( XLIV) a method of using a a peptide of {l)-(XXXVII) or an antibody or binding region thereof or a composition<br>
(XXXXi) In a method to identify or characterize a modulator of a cell expressing PSCA.<br>
[0120] As used herein, a range is understood to specifically disclose all whole unit positions thereof.<br>
[01211 Typical embodiments of the invention disclosed herein include PSCA polynucleotides that encode specific<br>
portions ot PSCA mRNA sequences (and those which are complementary to such sequences) such as those that encode<br>
the proteins and/or fragments thereof, for example:<br>
(a) 4, 5. 6, 7, 8,9,10,11,12,13,14,15,16,17,18,19, 20, 21, 22, 23, 24, 25, 30, 35,40,45, 50, 55, 60,65,70,<br>
75, 80, 85, 90, 95,100,105,110,115,120, and 123 or more contiguous amino acids of PSCA variant 1; the maximal<br>
lengths relevant for other variants are: variant 3,94 amino acids; variant 4,189 amino acids, variant 6,114 amino acids,<br>
variant 19,189 amino acids, variant 20,189 amino acids, variant 21,189 amino acids, variant 22,189 amino acids, variant<br>
24,189 amino acids, variant 25,189 amino acids, variant 26,189 amino acids, and variant 27,189 amino acids..<br>
[0122] In general, naturally occurring allelic variants of human PSCA share a high degree of structural identity and<br>
homology (e.g., 90% or more homology). Typically, allelic variants of a PSCA protein contain conservative amino acid<br>
substitutions within the PSCA sequences described herein or contain a substitution of an amino acid from a corresponding<br>
position in a homologue of PSCA. One class of PSCA allelic variants are proteins that share a high degree of homology<br>
with at least a small region of a particular PSCA amino acid sequence, but further contain a radical departure from the<br>
sequence, such as a non-conservative substitution, truncation, insertion or frame shift. In comparisons of protein<br>
sequences, the terms, similarity, identity, and homology each have a distinct meaning as appreciated in the field of<br>
genetics. Moreover, orthology and paralogy can be important concepts describing the relationship of members of a given<br>
protein family in one organism to the members of the same family in other organisms.<br>
[0123] Amino acid abbreviations are provided in Table II. Conservative amino acid substitutions can frequently be<br>
made in a protein without altering either the conformation or the function of the protein. Proteins of the invention can<br>
comprise 1,2, 3,4,5,6,7,8,9,10,11,12,13,14,15 conservative substitutions. Such changes include substituting any of<br>
isoleucine (I), valine (V), and leucine (L) for any other of these hydrophobic amino acids; aspartic acid (D) for glutamic acid<br>
(E) and vice versa; glutamine (Q) for asparagine (N) and vice versa; and serine (S) for threonine (T) and vice versa. Other<br>
substitutions can also be considered conservative, depending on the environment of the particular amino acid and its role in<br>
the three-dimensional structure of the protein. For example, glycine (G) and alanine (A) can frequently be interchangeable,<br><br>
as can alanine (A) and valine (V). Methionine (M), which is relatively hydrophobic, can frequently be interchanged with<br>
leucineand isoleucine, and sometimes with valine. Lysine (K) and arginine (R) are frequently interchangeable in locations<br>
in which the significant feature of the amino acid residue is its charge and the differing pK's of these two amino acid<br>
residues are not significant. Still other changes can be considered "conservative" in particular environments (see, e.g.<br>
Table 111 herein; pages 13-15 "Biochemistry" 2nd ED. Lubert Stryer ed (Stanford University); Henikoff et al., PNAS1992 Vol<br>
89 10915-10919; Lei et al., J Biol Chem 1995 May 19; 270(20):11882-6).<br>
[0124] Embodiments of the invention disclosed herein include a wide variety of art-accepted variants or analogs of<br>
PSCA proteins such as polypeptides having amino acid insertions, deletions and substitutions. PSCA variants can be<br>
made using methods known in the art such as site-directed mutagenesis, alanine scanning, and PCR mutagenesis. Site-<br>
directed mutagenesis (Carter et al., Nucl. Acids Res., 73:4331 (1986); Zoller et al., Nuci. Acids Res., 10:6487 (1987)),<br>
cassette mutagenesis (Wells et al., Gene, 34:315 (1985)), restriction selection mutagenesis (Wells et al., Philos. Trans. R.<br>
Soc. London SerA, 317:415 (1986)) or other known techniques can be performed on the cloned DNA to produce the PSCA<br>
variant DNA.<br>
[0125] Scanning amino acid analysis can also be employed to identify one or more amino acids along a contiguous<br>
sequence that is involved in a specific biological activity such as a protein-protein interaction. Among the preferred<br>
scanning amino acids are relatively small, neutral amino acids. Such amino acids include alanine, glycine, serine, and<br>
cysteine. Alanine is typically a preferred scanning amino acid among this group because it eliminates the side-chain<br>
beyond the beta-carbon and is less likely to alter the main-chain conformation of the variant. Alanine is also typically<br>
preferred because it is the most common amino acid. Further, it is frequently found in both buried and exposed positions<br>
(Creighton, The Proteins, (W.H. Freeman &amp; Co., N.Y.); Chothia, J. Mol. Biol., 150:1 (1976)). If alanine substitution does not<br>
yield adequate amounts of variant, an isosteric amino acid can be used.<br>
[0126] As defined herein, PSCA variants, analogs or homologs, have the distinguishing attribute of having at least<br>
one epitope that is "cross reactive" with a PSCA protein having an amino acid sequence of Figure 3. As used in this<br>
sentence, "cross reactive" means that an antibody or T cell that specifically binds to a PSCA variant also specifically binds<br>
to a PSCA protein having an amino acid sequence set forth in Figure 3. A polypeptide ceases to be a variant of a protein<br>
shown in Figure 3, when it no longer contains any epitope capable of being recognized by an antibody or T cell that<br>
specifically binds to the starting PSCA protein. Those skilled in the art understand that antibodies that recognize proteins<br>
bind to epitopes of varying size, and a grouping of the order of about four or five amino acids, contiguous or not, is regarded<br>
as a typical number of amino acids in a minimal epitope. See, e.g., Nair et al., J. Immunol 2000165(12): 6949-6955;<br>
Hebbes et al., Mol Immunol (1989) 26(9):865-73; Schwartz et al., J Immunol (1985) 135(4):2598-608.<br>
[0127] Other classes of PSCA-related protein variants share 70%, 75%, 80%, 85% or 90% or more similarity with an<br>
amino acid sequence of Figure 3, or a fragment thereof. Another specific class of PSCA protein variants or analogs<br>
comprises one or more of the PSCA biological motifs described herein or presently known in the art. Thus, encompassed<br>
by the present invention are analogs of PSCA fragments (nucleic or amino acid) that have altered functional (e.g.<br>
immunogenic) properties relative to the starting fragment. It is to be appreciated that motifs now or which become part of<br>
the art are to be applied to the nucleic or amino acid sequences of Figure 2 or Figure 3.<br>
[0128] As discussed herein, embodiments of the claimed invention include polypeptides containing less than the full<br>
amino acid sequence of a PSCA protein shown in Figure 2 or Figure 3. For example, representative embodiments of the<br>
invention comprise peptides/proteins having any 4,5,6,7,8,9,10,11,12,13,14,15 or more contiguous amino acids of a<br>
PSCA protein shown in Figure 2 or Figure 3.<br><br>
[0129] Moreover, representative embodiments of the invention disclosed herein include polypeptides consisting of<br>
about amino acid 1 to about amino acid 10 of a PSCA protein shown in Figure 2 or Figure 3, polypeptides consisting of<br>
about amino acid 10 to about amino acid 20 of a PSCA protein shown in Figure 2 or Figure 3, polypeptides consisting of<br>
about amino acid 20 to about amino acid 30 of a PSCA protein shown in Figure 2 or Figure 3, polypeptides consisting of<br>
about amino acid 30 to about amino acid 40 of a PSCA protein shown in Figure 2 or Figure 3, polypeptides consisting of<br>
about amino acid 40 to about amino acid 50 of a PSCA protein shown in Figure 2 or Figure 3, polypeptides consisting of<br>
about amino acid 50 to about amino acid 60 of a PSCA protein shown in Figure 2 or Figure 3, polypeptides consisting of<br>
about amino acid 60 to about amino acid 70 of a PSCA protein shown in Figure 2 or Figure 3, polypeptides consisting of<br>
about amino acid 70 to about amino acid 80 of a PSCA protein shown in Figure 2 or Figure 3, polypeptides consisting of<br>
about amino acid 80 to about amino acid 90 of a PSCA protein shown in Figure 2 or Figure 3, polypeptides consisting of<br>
about amino acid 90 to about amino acid 100 of a PSCA protein shown in Figure 2 or Figure 3, etc. throughout the entirety<br>
of a PSCA amino acid sequence. Moreover, polypeptides consisting of about amino acid 1 (or 20 or 30 or 40 etc.) to about<br>
amino acid 20, (or 130, or 140 or 150 etc.) of a PSCA protein shown in Figure 2 or Figure 3 are embodiments of the<br>
invention. It is to be appreciated that the starting and stopping positions in this paragraph refer to the specified position as<br>
well as that position plus or minus 5 residues.<br>
[0130] PSCA-related proteins are generated using standard peptide synthesis technology or using chemical<br>
cleavage methods well known in the art. Alternatively, recombinant methods can be used to generate nucleic acid<br>
molecules that encode a PSCA-related protein. In one embodiment, nucleic acid molecules provide a means to generate<br>
defined fragments of a PSCA protein (or variants, homologs or analogs thereof).<br>
III.A.) Motif-bearing Protein Embodiments<br>
[0131] Additional illustrative embodiments of the invention disclosed herein include PSCA polypeptides comprising<br>
the amino acid residues of one or more of the biological motifs contained within a PSCA polypeptide sequence set forth in<br>
Figure 2 or Figure 3. Various motifs are known in the art, and a protein can be evaluated for the presence of such motifs by<br>
a number of publicly available Internet sites (see, e.g., URL addresses: pfam.wusfl.edu/; searchlauncher.bcm.tmc.edu/seq-<br>
search/struc-predict.html; psort.ims.u-tokyo.ac.jp/; cbs.dtu.dk/; ebi.ac.uk/interpro/scan.html; expasy.ch/tools/scnpsitl .html;<br>
Epimatrix™ and Epimer™, Brown University, brown.edu/Research/TB-HlV_Lab/epimatrix/epimatrix.html; and BIMAS,<br>
bimas.dcrt.nih.gov/.).<br>
[0132] Motif bearing subsequences of all PSCA variant proteins are set forth and identified in Tables VIII-XXI and<br>
XXII-XLIX.<br>
[0133] Table V sets forth several frequently occurring motifs based on pfam searches (see URL address<br>
pfam.wustl.edu/). The columns of Table V list (1) motif name abbreviation, (2) percent identity found amongst the different<br>
member of the motif family, (3) motif name or description and (4) most common function; location information is included if<br>
the motif is relevant for location.<br>
[0134] Polypeptides comprising one or more of the PSCA motifs discussed above are useful in elucidating the<br>
specific characteristics of a malignant phenotype in view of the observation that the PSCA motifs discussed above are<br>
associated with growth dysregulation and because PSCA is overexpressed in certain cancers (See, e.g., Table I). Casein<br>
kinase il, cAMP and camp-dependent protein kinase, and Protein Kinase C, for example, are enzymes known to be<br>
associated with the development of the malignant phenotype (see e.g. Chen et a/, Lab Invest., 78(2): 165-174 (1998);<br>
Gaiddon et a/., Endocrinology 136(10); 4331-4338 (1995); Hall et al, Nucleic Acids Research 24(6): 1119-1126 (1996);<br><br>
Peterziel et al., Oncogene 18(46): 6322-6329 (1999) and O'Brian, Oncol. Rep. 5(2): 305-309 (1998)). Moreover, both<br>
glycosylate and myristoylation are protein modifications also associated with cancer and cancer progression (see e.g.<br>
Dermis et al., Biochem. Biophys. Acta 1473(1):21-34 (1999); Raju et al., Exp. Cell Res. 235(1): 145-154 (1997)). Amidation<br>
is another protein modification also associated with cancer and cancer progression (see e.g. Treston et al., J. Natl. Cancer<br>
Inst. Monogr. (13): 169-175 (1992)).<br>
[0135] In another embodiment, proteins of the invention comprise one or more of the immunoreactive epitopes<br>
identified in accordance with art-accepted methods, such as the peptides set forth in Tables VIII-XXI and XXII-XLIX. CTL<br>
epitopes can be determined using specific algorithms to identify peptides within a PSCA protein that are capable of optimally<br>
bonding to specified HLA alleles (e.g., Table IV; Epimatrix™ and Epimer™, Brown University, URL brown.edu/Research/TB-<br>
H'v' Lab Lpimatrix/epimatrix.html; and BIMAS, URL bimas.dcrt.nih.gov/.) Moreover, processes for identifying peptides that<br>
have sufficient binding affinity for HLA molecules and which are correlated with being immunogenic epitopes, are well<br>
know in the art, and are carried out without undue experimentation. In addition, processes for identifying peptides that are<br>
immunogenic epitopes, are well known in the art, and are carried out without undue experimentation either in vitro or in<br>
VIVO<br>
[0136] Also known in the art are principles for creating analogs of such epitopes in order to modulate<br>
immunogenicity. For example, one begins with an epitope that bears a CTL or HTL motif (see, e.g., the HLA Class I and<br>
HLA Class II motifs/supermotifs of Table IV). The epitope is analoged by substituting out an amino acid at one of the<br>
specified positions, and replacing it with another amino acid specified for that position. For example, on the basis of<br>
residues defined in Table IV, one can substitute out a deleterious residue in favor of any other residue, such as a preferred<br>
residue, substitute a less-preferred residue with a preferred residue; or substitute an originally-occurring preferred residue<br>
with another preferred residue. Substitutions can occur at primary anchor positions or at other positions in a peptide; see,<br>
e.g., Table IV.<br>
[0137] A variety of references reflect the art regarding the identification and generation of epitopes in a protein of<br>
interest as well as analogs thereof. See, for example, WO 97/33602 to Chesnut et al:, Sette, Immunogenetics 1999 50(3-<br>
4): 201-212; Sette et al., J. Immunol. 2001 166(2): 1389-1397; Sidney et al., Hum. Immunol. 1997 58(1): 12-20; Kondo et<br>
al., Immunogenetics 199745(4): 249-258; Sidney et al., J. Immunol. 1996 157(8): 3480-90; and Falk et al., Nature 351:<br>
290-6 (1991); Hunt et al., Science 255:1261-3 (1992); Parker et al., J. Immunol. 149:3580-7 (1992); Parker et al., J.<br>
Immunol. 152:163-75 (1994)); Kast et al., 1994152(8): 3904-12; Borras-Cuesta et al., Hum. Immunol. 2000 61(3): 266-278;<br>
Alexander et a/., J. Immunol. 2000 164(3); 164(3): 1625-1633; Alexander et al., PMID: 7895164, Ul: 95202582; O'Sullivan<br>
et al., J. Immunol. 1991 147(8): 2663-2669; Alexander et al., Immunity 1994 1(9): 751-761 and Alexander et al., Immunol.<br>
Res. 1998 18(2): 79-92.<br>
[0138] Related embodiments of the invention include polypeptides comprising combinations of the different motifs<br>
set forth in Table VI, and/or, one or more of the predicted CTL epitopes of Tables VIII-XXI and XXII-XLIX, and/or, one or<br>
more of the predicted HTL epitopes of Tables XLVI-XLIX, and/or, one or more of the T cell binding motifs known in the art.<br>
Preferred embodiments contain no insertions, deletions or substitutions either within the motifs or within the intervening<br>
sequences of the polypeptides. In addition, embodiments which include a number of either N-terminal and/or C-terminal<br>
amino acid residues on either side of these motifs may be desirable (to, for example, include a greater portion of the<br>
polypeptide architecture in which the motif is located). Typically, the number of N-terminal and/or C-terminal amino acid<br>
residues on either side of a motif is between about 1 to about 100 amino acid residues, preferably 5 to about 50 amino acid<br>
residues.<br><br>
[0139] PSCA-related proteins are embodied in many forms, preferably in isolated form. A purified PSCA protein<br>
molequle will be substantially free of other proteins or molecules that impair the binding of PSCA to antibody, T cell or other<br>
ligand. The nature and degree of isolation and purification will depend on the intended use. Embodiments of a PSCA-related<br>
proteins include purified PSCA-related proteins and functional, soluble PSCA-related proteins. In one embodiment, a<br>
functional, soluble PSCA protein or fragment thereof ret alns the ability to be bound by antibody, T cell or other ligand.<br>
[0140] The invention also provides PSCA proteins comprising biologically active fragments of a PSCA amino acid<br>
sequence shown in Figure 2 or Figure 3. Such proteins exhibit properties of the starting PSCA protein, such as the ability<br>
to elicit the generation of antibodies that specifically bind an epitope associated with the starting PSCA protein; to be bound<br>
by such antibodies; to elicit the activation of HTL or CTL; and/or, to be recognized by HTL or CTL that also specifically bind<br>
to the starting protein.<br>
[0141] PSCA-related polypeptides that contain particularly interesting structures can be predicted and/or identified<br>
using various analytical techniques well known in the art, including, for example, the methods of Chou-Fasman, Garnier-<br>
Robson, Kyte-Doolittle, Eisenberg, Karplus-Schultz or Jameson-Wolf analysis, or based on immunogenicity. Fragments<br>
that contain such structures are particularly useful in generating subunit-specific anti-PSCA antibodies or T cells or in<br>
identifying cellular factors that bind to PSCA. For example, hydrophiiicity profiles can be generated, and immunogenic<br>
peptide fragments identified, using the method of Hopp, T.P. and Woods, K.R., 1981, Proc. Natl. Acad. Sci. U.S.A.<br>
78:3824-3828. Hydropathicity profiles can be generated, and immunogenic peptide fragments identified, using the method<br>
of Kyte, J. and Doolittle, R.F., 1982, J. Mol. Biol. 157:105-132. Percent (%) Accessible Residues profiles can be generated,<br>
and immunogenic peptide fragments identified, using the method of Janin J., 1979, Nature 277:491-492. Average<br>
Flexibility profiles can be generated, and immunogenic peptide fragments identified, using the method of Bhaskaran R.,<br>
Ponnuswamy P.K., 1988, Int. J. Pept. Protein Res. 32:242-255. Beta-turn profiles can be generated, and immunogenic<br>
peptide fragments identified, using the method of Deleage, G., Roux B., 1987, Protein Engineering 1:289-294.<br>
[0142] CTL epitopes can be determined using specific algorithms to identify peptides within a PSCA protein that are<br>
capable of optimally binding to specified HLA alleles (e.g., by using the SYFPEITHI site at World Wide Web URL<br>
syfpeithi.bmi-heidelberg.com/; the listings in Table IV(A)-(E); Epimatrix™ and Epimer™, Brown University, URL<br>
(brown.edu/Research/TB-HIV_Lab/epimatrix/epimatrix.html); and BIMAS, URL bimas.dcrt.nih.gov/). Illustrating this,<br>
peptide epitopes from PSCA that are presented in the context of human MHC Class I molecules, e.g., HLA-A1, A2, A3,<br>
A11, A24, B7 and B35 were predicted (see, e.g., Tables VIII-XXI, XXII-XLIX). Specifically, the complete amino acid<br>
sequence of the PSCA protein and relevant portions of other variants, i.e., for HLA Class I predictions 9 flanking residues<br>
on either side of a point mutation or exon juction, and for HLA Class II predictions 14 flanking residues on either side of a<br>
point mutation or exon junction corresponding to that variant, were entered into the HLA Peptide Motif Search algorithm<br>
found in the Bioinformatics and Molecular Analysis Section (BIMAS) web site listed above; in addition to the site<br>
SYFPEITHI, at URL syfpeithi.bmi-heidelberg.com/.<br>
[0143] The HLA peptide motif search algorithm was developed by Dr. Ken Parker based on binding of specific<br>
peptide sequences in the groove of HLA Class I molecules, in particular HLA-A2 (see, e.g., Falk et a/., Nature 351:290-6<br>
(1991); Huntefa/., Science 255:1261-3 (1992); Parker et al., J. Immunol. 149:3580-7(1992); Parker et al., J. Immunol.<br>
152:163-75 (1994)). This algorithm allows location and ranking of 8-mer, 9-mer, and 10-mer peptides from a complete<br>
protein sequence for predicted binding to HLA-A2 as well as numerous other HLA Class I molecules. Many HLA class I<br>
binding peptides are 8-, 9-, 10 or 11-mers. For example, for Class IHLA-A2, the epitopes preferably contain a leucine (L)<br>
or methionine (M) at position 2 and a valine (V) or leucine (L) at the C-terminus (see, e.g., Parker et a/., J. Immunol.<br><br>
149:3580-7 (1992)). Selected results of PSCA predicted binding peptides are shown in Tables Vlll-XXi and XXII-XLIX<br>
herein. In Tables Vlll-XXi and XXll-XLVII, selected candidates, 9-mers and 10-mers, for each family member are shown<br>
along with their location, the amino acid sequence of each specific peptide, and an estimated binding score. In Tables<br>
XLVl-XLIX, selected candidates, 15-mers, for each family member are shown along with their location, the amino acid<br>
sequence of each specific peptide, and an estimated binding score. The binding score corresponds to the estimated half<br>
time of dissociation of complexes containing the peptide at 37°C at pH 6.5. Peptides with the highest binding score are<br>
predicted to be the most tightly bound to HLA Class I on the cell surface for the greatest period of time and thus represent<br>
the best immunogenic targets for T-cell recognition.<br>
[0144] Actual binding of peptides to an HLA allele can be evaluated by stabilization of HLA expression on the<br>
antigen-processing defective cell line T2 (see, e.g., Xue et al., Prostate 30:73-8 (1997) and Peshwa era/., Prostate 36:129-<br>
38 (1998)). Immunogenicity of specific peptides can be evaluated in vitro by stimulation of CD8+ cytotoxic T lymphocytes<br>
(CTL) in the presence of antigen presenting cells such as dendritic cells.<br>
[0145] It is to be appreciated that every epitope predicted by the BIMAS site, Epimer™ and Epimatrix™ sites, or<br>
specified by the HLA class I or class II motifs available in the art or which become part of the art such as set forth in Table<br>
IV (or determined using World Wide Web site URL syfpeithi.bmi-heidelberg.com/, or BIMAS, bimas.dcrt.nih.gov/) are to be<br>
"applied" to a PSCA protein in accordance with the invention. As used in this context "applied" means that a PSCA protein<br>
is evaluated, e.g., visually or by computer-based patterns finding methods, as appreciated by those of skill in the relevant<br>
art. Every subsequence of a PSCA protein of 8, 9,10, or 11 amino acid residues that bears an HLA Class I motif, or a<br>
subsequence of 9 or more amino acid residues that bear an HLA Class II motif are within the scope of the invention.<br>
III.B.) Expression of PSCA-related Proteins<br>
[0146] In an embodiment described in the examples that follow, PSCA can be conveniently expressed in cells (such<br>
as 293T cells) transfected with a commercially available expression vector such as a CMV-driven expression vector<br>
encoding PSCA with a C-terminal 6XHis and MYC tag (pcDNA3.1/mycHIS, Invitrogen or Tag5, GenHunter Corporation,<br>
Nashville TN). The Tag5 vector provides an IgGK secretion signal that can be used to facilitate the production of a<br>
secreted PSCA protein in transfected cells. The secreted HIS-tagged PSCA in the culture media can be purified, e.g.,<br>
using a nickel column using standard techniques.<br>
III.C.) Modifications of PSCA-related Proteins<br>
[0147] Modifications of PSCA-related proteins such as covalent modifications are included within the scope of this<br>
invention. One type of covalent modification includes reacting targeted amino acid residues of a PSCA polypeptide with an<br>
organ,c derivatizing agent that is capable of reacting with selected side chains or the N- or C- terminal residues of a PSCA<br>
protem. Another type of covalent modification of a PSCA polypeptide included within the scope of this invention comprises<br>
altermg the native glycosylate pattern of a protein of the invention. Another type of covalent modification of PSCA<br>
comprises linking a PSCA polypeptide to one of a variety of nonproteinaceous polymers, e.g., polyethylene glycol (PEG)<br>
polypropylene glycol, or polyoxyalkylenes, in the manner set forth in U.S. Patent Nos. 4,640,835' 4 496 689- 4 301144- '<br>
4,670,417; 4,791,192 or 4,179,337.<br>
[0148] The PSCA-related proteins of the present invention can also be modified to form a chimeric molecule<br>
compr(S,ng PSCA fused to another, heterologous polypeptide or amino acid sequence. Such a chimeric molecule can be<br>
synthesized chemically or recombinantly. A chimeric molecule can have a protein of the invention fused to another tumor-<br>
associated antigen or fragment thereof. Alternatively, a protein in accordance with the invention can comprise a fusion of<br>
fragments of a PSCA sequence (amino or nucleic acid) such that a molecule is created that is not, through its length,<br>
directly homologous to the amino or nucleic acid sequences shown in Figure 2 or Figure 3. Such a chimeric molecule can<br>
comprise multiples of the same subsequence of PSCA. A chimeric molecule can comprise a fusion of a PSCA-related<br>
protein with a polyhistidine epitope tag, which provides an epitope to which immobilized nickel can selectively bind, with<br>
cytokines or with growth factors. The epitope tag is generally placed at the amino- or carboxyl- terminus of a PSCA protein.<br>
In an alternative embodiment, the chimeric molecule can comprise a fusion of a PSCA-related protein with an<br>
immunoglobulin or a particular region of an immunoglobulin. For a bivalent form of the chimeric molecule (also referred to<br>
as an "immunoadhesin"), such a fusion could be to the Fc region of an IgG molecule. The Ig fusions preferably include the<br>
substitution of a soluble (transmembrane domain deleted or inactivated) form of a PSCA polypeptide in place of at least one<br>
variable region within an Ig molecule. In a preferred embodiment, the immunoglobulin fusion includes the hinge, CH2 and<br>
CH3, or the hinge, CHI, CH2 and CH3 regions of an IgGI molecule. For the production of immunoglobulin fusions see, e.g.,<br>
U.S. Patent No. 5,428,130 issued June 27,1995.<br>
lll.D.) Uses of PSCA-related Proteins<br>
[0149] The proteins of the invention have a number of different specific uses. As PSCA is highly expressed in<br>
prostate and other cancers, PSCA-related proteins are used in methods that assess the status of PSCA gene products in<br>
normal versus cancerous tissues, thereby elucidating the malignant phenofype. Typically, polypeptides from specific<br>
regions of a PSCA protein are used to assess the presence of perturbations (such as deletions, insertions, point mutations<br>
etc.) in those regions (such as regions containing one or more motifs). Exemplary assays utilize antibodies or T cells<br>
targeting PSCA-related proteins comprising the amino acid residues of one or more of the biological motifs contained within<br>
a PSCA polypeptide sequence in order to evaluate the characteristics of this region in normal versus cancerous tissues or<br>
to elicit an immune response to the epitope. Alternatively, PSCA-related proteins that contain the amino acid residues of<br>
one or more of the biological motifs in a PSCA protein are used to screen for factors that interact with that region of PSCA.<br>
[0150] PSCA protein fragments/subsequences are particularly useful in generating and characterizing domain-<br>
specific antibodies (e.g., antibodies recognizing an extracellular or intracellular epitope of a PSCA protein), for identifying<br>
agents or cellular factors that bind to PSCA or a particular structural domain thereof, and in various therapeutic and<br>
diagnostic contexts, including but not limited to diagnostic assays, cancer vaccines and methods of preparing such<br>
vaccines.<br>
[0151] Proteins encoded by the PSCA genes, or by analogs, homologs or fragments thereof, have a variety of uses,<br>
including but not limited to generating antibodies and in methods for identifying ligands and other agents and cellular<br>
constituents that bind to a PSCA gene product. Antibodies raised against a PSCA protein or fragment thereof are useful in<br>
diagnostic and prognostic assays, and imaging methodologies in the management of human cancers characterized by<br>
expression of PSCA protein, such as those listed in Table I. Such antibodies can be expressed intracellularly and used in<br>
methods of treating patients with such cancers. PSCA-related nucleic acids or proteins are also used in generating HTL or<br>
CTL responses.<br>
[0152] Various immunological assays useful for the detection of PSCA proteins are used, including but not limited to<br>
various types of radioimmunoassays, enzyme-linked immunosorbent assays (ELISA), enzyme-linked immunofluorescent<br>
assays (ELIFA), immunocytochemical methods, and the like. Antibodies can be labeled and used as immunological<br><br>
imaging reagents capable of detecting PSCA-expressing cells (e.g., in radioscintigraphic imaging methods). PSCA proteins<br>
are atso particularly useful in generating cancer vaccines, as further described herein.<br>
IV.) PSCA Antibodies<br>
[0153] Another aspect of the invention provides antibodies that bind to PSCA-related proteins. Preferred antibodies<br>
specifically bind to a PSCA-related protein and do not bind (or bind weakly) to peptides or proteins that are not PSCA-<br>
related proteins under physiological conditions. In this context, examples of physiological conditions include: 1) phosphate<br>
buffered saline; 2) Tris-buffered saline containing 25mM Tris and 150 mM NaCI; or normal saline (0.9% NaCI); 4) animal<br>
serum such as human serum; or, 5) a combination of any of 1) through 4); these reactions preferably taking place at pH<br>
7.5, alternatively in a range of pH 7.0 to 8.0, or alternatively in a range of pH 6.5 to 8.5; also, these reactions taking place<br>
at a temperature between 4°C to 37°C. For example, antibodies that bind PSCA can bind PSCA-related proteins such as<br>
the homologs or analogs thereof.<br>
[0154] PSCA antibodies of the invention are particularly useful in cancer (see, e.g., Table I) diagnostic and<br>
prognostic assays, and imaging methodologies. Similarly, such antibodies are useful in the treatment, diagnosis, and/or<br>
prognosis of other cancers, to the extent PSCA is also expressed or overexpressed in these other cancers. Moreover,<br>
intracellularly expressed antibodies (e.g., single chain antibodies) are therapeutically useful in treating cancers in which the<br>
expression of PSCA is involved, such as advanced or metastatic prostate cancers.<br>
[0155] The invention also provides various immunological assays useful for the detection and quantification of PSCA<br>
and mutant PSCA-related proteins. Such assays can comprise one or more PSCA antibodies capable of recognizing and<br>
binding a PSCA-related protein, as appropriate. These assays are performed within various immunological assay formats<br>
well known in the art, including but not limited to various types of radioimmunoassays, enzyme-linked immunosorbent<br>
assays (ELISA), enzyme-linked immunofluorescent assays (ELIFA), and the like.<br>
[0156] Immunological non-antibody assays of the invention also comprise T cell immunogenicity assays (inhibitory or<br>
stimulatory) as well as major histocompatibility complex (MHC) binding assays.<br>
[0157] In addition, immunological imaging methods capable of detecting prostate cancer and other cancers<br>
expressing PSCA are also provided by the invention, including but not limited to radioscintigraphic imaging methods using<br>
labeled PSCA antibodies. Such assays are clinically useful in the detection, monitoring, and prognosis of PSCA expressing<br>
cancers such as prostate cancer.<br>
[0158] PSCA antibodies are also used in methods for purifying a PSCA-related protein and for isolating PSCA<br>
homologues and related molecules. For example, a method of purifying a PSCA-related protein comprises incubating a<br>
PSCA antibody, which has been coupled to a solid matrix, with a lysate or other solution containing a PSCA-related protein<br>
under conditions that permit the PSCA antibody to bind to the PSCA-related protein; washing the solid matrix to eliminate<br>
impurities; and eluting the PSCA-related protein from the coupled antibody. Other uses of PSCA antibodies in accordance<br>
with the invention include generating anti-idiotypic antibodies that mimic a PSCA protein.<br>
[0159] Various methods for the preparation of antibodies are well known in the art. For example, antibodies can be<br>
prepared by immunizing a suitable mammalian host using a PSCA-related protein, peptide, or fragment, in isolated or<br>
immunoconjugated form (Antibodies: A Laboratory Manual, CSH Press, Eds., Harlow, and Lane (1988); Harlow, Antibodies,<br>
Cold Spring Harbor Press, NY (1989)). In addition, fusion proteins of PSCA can also be used, such as a PSCA GST-fusion<br>
protein. In a particular embodiment, a GST fusion protein comprising all or most of the amino acid sequence of Figure 2 or<br>
Figure 3 is produced, then used as an immunogen to generate appropriate antibodies. In another embodiment, a PSCA-<br>
relatad protein is synthesized and used as an immunogen.<br>
[0160] In addition, naked DNA immunization techniques known in the art are used (with or without purified PSCA-<br>
related protein or PSCA expressing cells) to generate an immune response to the encoded immunogen (for review, see<br>
Donnelly et al., 1997, Ann. Rev. Immunol. 15:617-648).<br>
[0161] The amino acid sequence of a PSCA protein as shown in Figure 2 or Figure 3 can be analyzed to select<br>
specific regions of the PSCA protein for generating antibodies. For example, hydrophobicity and hydrophilicity analyses of<br>
a PSCA amino acid sequence are used to identify hydrophilic regions in the PSCA structure. Regions of a PSCA protein<br>
that show immunogenic structure, as well as other regions and domains, can readily be identified using various other<br>
methods known in the art, such as Chou-Fasman, Garnier-Robson, Kyte-Doolittle, Eisenberg, Karplus-Schultz or Jameson-<br>
Wolf analysis. Hydrophilicity profiles can be generated using the method of Hopp, T.P. and Woods, K.R., 1981, Proc. Natl.<br>
Acad. Sci. U.S.A. 78:3824-3828. Hydropathicity profiles can be generated using the method of Kyte, J. and Doolittle, R.F.,<br>
1982, J. Mol. Biol. 157:105-132. Percent (%) Accessible Residues profiles can be generated using the method of Janin J.,<br>
1979, Nature 277:491-492. Average Flexibility profiles can be generated using the method of Bhaskaran R., Ponnuswamy<br>
P.K., 1988, Int. J. Pept. Protein Res. 32:242-255. Beta-turn profiles can be generated using the method of Deleage, G.,<br>
Roux B., 1987, Protein Engineering 1:289-294. Thus, each region identified by any of these programs or methods is within<br>
the scope of the present invention. Methods for the generation of PSCA antibodies are further illustrated by way of the<br>
examples provided herein. Methods for preparing a protein or polypeptide for use as an immunogen are well known in the<br>
art. Also well known in the art are methods for preparing immunogenic conjugates of a protein with a carrier, such as BSA,<br>
KLH or other carrier protein. In some circumstances, direct conjugation using, for example, carbodiimide reagents are<br>
used; in other instances linking reagents such as those supplied by Pierce Chemical Co., Rockford, IL, are effective.<br>
Administration of a PSCA immunogen is often conducted by injection over a suitable time period and with use of a suitable<br>
adjuvant, as is understood in the art. During the immunization schedule, titers of antibodies can be taken to determine<br>
adequacy of antibody formation.<br>
[0162] PSCA monoclonal antibodies can be produced by various means well known in the art. For example,<br>
immortalized cell lines that secrete a desired monoclonal antibody are prepared using the standard hybridoma technology<br>
of Kohler and Milstein or modifications that immortalize antibody-producing B cells, as is generally known. Immortalized<br>
ceil lines that secrete the desired antibodies are screened by immunoassay in which the antigen is a PSCA-related protein.<br>
When the appropriate immortalized cell culture is identified, the cells can be expanded and antibodies produced either from<br>
in vitro cultures or from ascites fluid.<br>
[0163] The antibodies or fragments of the invention can also be produced, by recombinant means. Regions that<br>
bind specifically to the desired regions of a PSCA protein can also be produced in the context of chimeric or<br>
complementarity-determining region (CDR) grafted antibodies of multiple species origin. Humanized or human PSCA<br>
antibodies can also be produced, and are preferred for use in therapeutic contexts. Methods for humanizing murine and<br>
other non-human antibodies, by substituting one or more of the non-human antibody CDRs for corresponding human<br>
antibody sequences, are well known (see for example, Jones et al., 1986, Nature 321:522-525; Riechmann et al., 1988,<br>
Nature 332:323-327; Verhoeyen et al, 1988, Science 239:1534-1536). See also, Carter et al., 1993, Proc. Natl. Acad.<br>
Sci. USA 89:4285 and Sims et al., 1993, J. Immunol. 151: 2296.<br>
[0164] Methods for producing fully human monoclonal antibodies include phage display and transgenic methods (for<br>
review, see Vaughan et al., 1998, Nature Biotechnology 16:535-539). Fully human PSCA monoclonal antibodies can be<br><br>
generated using cloning technologies employing large human Ig gene combinatorial libraries (i.e., phage display) (Griffiths<br>
and Heogenboom, Building an in vitro immune system: human antibodies from phage display libraries. In: Protein<br>
Engineering of Antibody Molecules for Prophylactic and Therapeutic Applications in Man, Clark, M. (Ed.), Nottingham<br>
Academic, pp 45-64 (1993); Burton and Barbas, Human Antibodies from combinatorial libraries, jd., pp 65-82). Fully<br>
human PSCA monoclonal antibodies can also be produced using transgenic mice engineered to contain human<br>
immunoglobulin gene loci as described in PCT Patent Application W098/24893, Kucherlapati and Jakobovits et al.,<br>
published December 3,1997 (see also, Jakobovits, 1998, Exp. Opin. Invest. Drugs 7(4): 607-614; U.S. patents 6,162,963<br>
issued 19 December 2000; 6,150,584 issued 12 November 2000; and, 6,114598 issued 5 September 2000). This method<br>
avoids the in vitro manipulation required with phage display technology and efficiently produces high affinity authentic<br>
human antibodies.<br>
[0165] Reactivity of PSCA antibodies with a PSCA-related protein can be established by a number of well known<br>
means, including Western blot, immunoprecipitation, ELISA, and FACS analyses using, as appropriate, PSCA-related<br>
proteins, PSCA-expressing cells or extracts thereof. A PSCA antibody or fragment thereof can be labeled with a detectable<br>
marker or conjugated to a second molecule. Suitable detectable markers include, but are not limited to, a radioisotope, a<br>
fluorescent compound, a bioluminescent compound, chemiluminescent compound, a met al chelator or an enzyme.<br>
Further, bi-specific antibodies specific for two or more PSCA epitopes are generated using methods generally known in the<br>
art. Homodimeric antibodies can also be generated by cross-linking techniques known in the art (e.g., Wolff et al., Cancer<br>
Res. 53:2560-2565).<br>
V.) PSCA Cellular Immune Responses<br>
[0166] The mechanism by which T cells recognize antigens has been delineated. Efficacious peptide epitope<br>
vaccine compositions of the invention induce a therapeutic or prophylactic immune responses in very broad segments of<br>
the world-wide population. For an understanding of the value and efficacy of compositions of the invention that induce<br>
cellular immune responses, a brief review of immunology-related technology is provided.<br>
[0167] A complex of an HLA molecule and a peptidic antigen acts as the ligand recognized by HLA-restricted T cells<br>
(Buus, S. et al, Cell 47:1071,1986; Babbitt, B. P. et al., Nafure 317:359,1985; Townsend, A. and Bodrner, H., Annu. Rev.<br>
Immunol. 7:601,1989; Germain, R. N., Annu. Rev. Immunol. 11:403,1993). Through the study of single amino acid<br>
substituted antigen analogs and the sequencing of endogenously bound, naturally processed peptides, critical residues that<br>
correspond to motifs required for specific binding to HLA antigen molecules have been identified and are set forth in Table<br>
IV (see also, e.g., Southwood, et al., J. Immunol. 160:3363,1998; Rammensee, et al., Immunogenetics41:178,1995;<br>
Rammensee et al., SYFPEITHI, access via World Wide Web at URL (134.2.96.221/scripts.hlaserver.dll/home.htm); Sette,<br>
A. and Sidney, J. Curr. Opin. Immunol. 10:478,1998; Engelhard, V. H., Curr. Opin. Immunol. 6:13,1994; Sette, A. and<br>
Grey, H. M., Curr. Opin. Immunol. 4:79,1992; Sinigaglia, F. and Hammer, J. Curr. Biol. 6:52,1994; Ruppert et al., Cell<br>
74:929-937,1993; Kondo et a/., J. Immunol. 155:4307-4312,1995; Sidney et al., J. Immunol. 157:3480-3490,1996; Sidney<br>
et al., Human Immunol. 45:79-93,1996; Sette, A. and Sidney, J. Immunogenetics 1999Nov; 50(3-4)201-12, Review).<br>
[0168] Furthermore, x-ray crystallographic analyses of HLA-peptide complexes have revealed pockets within the<br>
peptide binding cleft/groove of HLA molecules which accommodate, in an allele-specific mode, residues borne by peptide<br>
ligands; these residues in turn determine the HLA binding capacity of the peptides in which they are present. (See, e.g.,<br>
Madden, D.R. Annu. Rev. Immunol. 13:587,1995; Smith, et al, Immunity 4:203,1996; Fremont et al., Immunity 8:305,<br>
1998; Stern et al., Structure 2:245,1994; Jones, E.Y. Curr. Opin. Immunol. 9:75,1997; Brown, J. H. et at., Nature 364:33,<br><br>
1993; Guo, H. C. et al., Proc. Natl. Acad. Sci. USA 90:8053,1993; Guo, H. C. et al., Nature 360:364,1992; Silver, M. L et<br>
al., Nature 360:367,1992; Matsumura, M. et al., Science 257:927,1992; Madden era/., Ce/I70:1035,1992; Fremont, D. H.<br>
et al., Science 257:919,1992; Saper, M. A., Bjorkman, P. J. and Wiley, D. C, J. Mol. Biol. 219:277,1991.)<br>
[0169] Accordingly, the definition of class I and class II allele-specific HLA binding motifs, or class I or class II<br>
supermotifs allows identification of regions within a protein that are correlated with binding to particular HLA antigen(s).<br>
[0170] Thus, by a process of HLA motif identification, candidates for epitope-based vaccines have been identified;<br>
such candidates can be further evaluated by HLA-peptide binding assays to determine binding affinity and/or the time<br>
period of association of the epitope and its corresponding HLA molecule. Additional confirmatory work can be performed to<br>
select, amongst these vaccine candidates, epitopes with preferred characteristics in terms of population coverage, and/or<br>
immunogenicity.<br>
[0171] Various strategies can be utilized to evaluate cellular immunogenicity, including:<br>
1) Evaluation of primary T cell cultures from normal individuals (see, e.g., Wentworth, P. A. et al., Mol. Immunol.<br>
32:603,1995; Celis, E. et al., Proc. Natl. Acad. Sci. USA 91:2105,1994; Tsai, V. et al., J. Immunol. 158:1796,1997;<br>
Kawashima, I. et al., Human Immunol. 59:1,1998). This procedure involves the stimulation of peripheral blood<br>
lymphocytes (PBL) from normal subjects with a test peptide in the presence of antigen presenting cells in vitro over a period<br>
of several weeks. T cells specific for the peptide become activated during this time and are detected using, e.g., a<br>
lymphokine- or 51Cr-release assay involving peptide sensitized target cells.<br>
2) Immunization of HLA transgenic mice (see, e.g., Wentworth, P. A. et al., J. Immunol. 26:97,1996; Wentworth,<br>
P. A. et al., Int. Immunol. 8:651,1996; Alexander, J. et al., J. Immunol. 159:4753,1997). For example, in such methods<br>
peptides in incomplete Freund's adjuvant are administered subcutaneously to HLA transgenic mice. Several weeks<br>
following immunization, splenocytes are removed and cultured in vitro in the presence of test peptide for approximately one<br>
week. Peptide-specific T cells are detected using, e.g., a 51Cr-release assay involving peptide sensitized target cells and<br>
target cells expressing endogenously generated antigen.<br>
3) Demonstration of recall T cell responses from immune individuals who have been either effectively vaccinated<br>
and/or from chronically ill patients (see, e.g., Rehermann, B.et al., J. Exp. Med. 181:1047,1995; Doolan, D. L.et al.,<br>
Immunity7:97,1997; Bertoni, R. et al., J. Clin. Invest. 100:503,1997; Threlkeld, S. C. et al., J. Immunol. 159:1648,1997;<br>
Diepolder, H. M. et al., J. Virol. 71:6011,1997). Accordingly, recall responses are detected by culturing PBL from subjects<br>
that have been exposed to the antigen due to disease and thus have generated an immune response "naturally", or from<br>
patients who were vaccinated against the antigen. PBL from subjects are cultured in vitro for 1-2 weeks in the presence of<br>
test peptide plus antigen presenting celis (APC) to allow activation of "memory" T cells, as compared to "naive" T cells. At<br>
the end of the culture period, T cell activity is detected using assays including 51Cr release involving peptide-sensitized<br>
targets, T cell proliferation, or lymphokine release.<br>
VI.) PSCA Transgenic Animals<br>
[0172] Nucleic acids that encode a PSCA-related protein can also be used to generate either transgenic animals or<br>
"knock out" animals that, in turn, are useful in the development and screening of therapeutically useful reagents. In<br>
accordance with established techniques, cDNA encoding PSCA can be used to clone genomic DNA that encodes PSCA.<br>
The cloned genomic sequences can then be used to generate transgenic animals containing cells that express DNA that<br>
encode PSCA. Methods for generating transgenic animals, particularly animals such as mice or rats, have become<br>
conventional in the art and are described, for example, in U.S. Patent Nos. 4,736,866 issued 12 April 1988, and 4,870,009<br>
issued 26 September 1989. Typically, particular cells would be targeted for PSCA transgene incorporation with tissue-<br>
specie enhancers.<br>
[0173] Transgenic animals that include a copy of a transgene encoding PSCA can be used to examine the effect of<br>
increased expression of DNA that encodes PSCA. Such animals can be used as tester animals for reagents thought to<br>
confer protection from, for example, pathological conditions associated with its overexpression. In accordance with this<br>
aspect of the invention, an animal is treated with a reagent and a reduced incidence of a pathological condition, compared<br>
to untreated animals that bear the transgene, would indicate a potential therapeutic intervention for the pathological<br>
condition.<br>
[0174] Alternatively, non-human homologues of PSCA can be used to construct a PSCA "knock out" animal that has<br>
a defective or altered gene encoding PSCA as a result of homologous recombination between the endogenous gene<br>
encoding PSCA and altered genomic DNA encoding PSCA introduced into an embryonic cell of the animal. For example,<br>
cDNA that encodes PSCA can be used to clone genomic DNA encoding PSCA in accordance with established techniques.<br>
A portion of tN genomic DNA encoding PSCA can be deleted or replaced with another gene, such as a gene encoding a<br>
selectable matter that can be used to monitor integration. Typically, several kilobases of unaltered flanking DNA (both at<br>
the 5' and 3' ends) are included in the vector (see, e.g., Thomas and Capecchi, Cell, 51:503 (1987) for a description of<br>
homologous recombination vectors). The vector is introduced into an embryonic stem cell line (e.g., by electroporation) and<br>
cells in which the introduced DNA has homologously recombined with the endogenous DNA are selected (see, e.g., Li et<br>
at., Cell, 69:915 (1992)). The selected cells are then injected into a blastocyst of an animal (e.g., a mouse or rat) to form<br>
aggregation chimeras (see, e.g., Bradley, in Teratocarcinomas and Embryonic Stem Cells: A Practical Approach, E. J.<br>
Robertson, ed. (IRL, Oxford, 1987), pp. 113-152). A chimeric embryo can then be implanted into a suitable<br>
pseudopregnant female foster animal, and the embryo brought to term to create a "knock out" animal. Progeny harboring<br>
the homologously recombined DNA in their germ cells can be identified by standard techniques and used to breed animals<br>
in which all cells of the animal contain the homologously recombined DNA. Knock out animals can be characterized, for<br>
example, for their ability to defend against certain pathological conditions or for their development of pathological conditions<br>
due to absence of a PSCA polypeptide.<br>
VII.) Methods for the Detection of PSCA<br>
[0175] Another aspect of the present invention relates to methods for detecting PSCA polynucleotides and PSCA-<br>
related proteins, as well as methods for identifying a cell that expresses PSCA. The expression profile of PSCA makes it a<br>
diagnostic marker for metastasized disease. Accordingly, the status of PSCA gene products provides information useful for<br>
predicting a variety of factors including susceptibility to advanced stage disease, rate of progression, and/or tumor<br>
aggressiveness. As discussed in det all herein, the status of PSCA gene products in patient samples can be analyzed by a<br>
variety protocols that are well known, in the art including immunohistochemical analysis, the variety of Northern blotting<br>
techniques including in situ hybridization, RT-PCR analysis (for example on laser capture micro-dissected samples),<br>
Western blot analysis and tissue array analysis.<br>
[0176] More particularly, the invention provides assays for the detection of PSCA polynucleotides in a biological<br>
sample, such as serum, bone, prostate, and other tissues, urine, semen, cell preparations, and the like. Detectable PSCA<br>
polynucleotides include, for example, a PSCA gene or fragment thereof, PSCA mRNA, alternative splice variant PSCA<br>
mRNAs, and recombinant DNA or RNA molecules that contain a PSCA polynucleotide. A number of methods for<br><br>
amplifying and/or detecting the presence of PSCA polynucleotides are well known in the art and can be employed in the<br>
praptide of this aspect of the invention.<br>
[0177] In one embodiment, a method for detecting a PSCA mRNA in a biological sample comprises producing cDNA<br>
from the sample by reverse transcription using at least one primer; amplifying the cDNA so produced using a PSCA<br>
polynucleotides as sense and antisense primers to amplify PSCA cDNAs therein; and detecting the presence of the<br>
amplified PSCA cDNA. Optionally, the sequence of the amplified PSCA cDNA can be determined.<br>
[0178] In another embodiment, a method of detecting a PSCA gene in a biological sample comprises first isolating<br>
genomic DMA from the sample; amplifying the isolated genomic DNA using PSCA polynucleotides as sense and antisense<br>
pampers and detecting the presence of the amplified PSCA gene. Any number of appropriate sense and antisense probe<br>
combustion can be designed from a PSCA nucleotide sequence (see, e.g., Figure 2) and used for this purpose.<br>
[0179] The invention also provides assays for detecting the presence of a PSCA protein in a tissue or other<br>
biological sample such as serum, semen, bone, prostate, urine, cell preparations, and the like. Methods for detecting a<br>
PSCA-related protein are also well known and include, for example, immunoprecipitation, immunohistochemical analysis,<br>
western blot analysis, molecular binding assays, ELISA, ELIFA and the like. For example, a method of detecting the<br>
presence of a PSCA-related protein in a biological sample comprises first contacting the sample with a PSCA antibody, a<br>
PSCA-reactive fragment thereof, or a recombinant protein containing an antigen-binding region of a PSCA antibody; and<br>
then detecting the binding of PSCA-related protein in the sample.<br>
[0180] Methods for identifying a cell that expresses PSCA are also within the scope of the invention. In one<br>
embodiment, an assay for identifying a cell that expresses a PSCA gene comprises detecting the presence of PSCA mRNA<br>
m the celi. Methods for the detection of particular mRNAs in cells are well known and include, for example, hybridization<br>
assays using complementary DNA probes (such as in situ hybridization using labeled PSCA riboprobes, Northern blot and<br>
related techniques) and various nucleic acid amplification assays (such as RT-PCR using complementary primers specific<br>
for PSCA, and other amplification type detection methods, such as, for example, branched DNA, SISBA, TMA and the like).<br>
Alternatively, an assay for identifying a cell that expresses a PSCA gene comprises detecting the presence of PSCA-<br>
related protein in the cell or secreted by the cell. Various methods for the detection of proteins are well known in the art<br>
and are employed for the detection of PSCA-related proteins and cells that express PSCA-related proteins.<br>
[0181] PSCA expression analysis is also useful as a tool for identifying and evaluating agents that modulate PSCA<br>
gene expression. For example, PSCA expression is significantly upregulated in prostate cancer, and is expressed in<br>
cancers of the tissues listed in Table I. Identification of a molecule or biological agent that inhibits PSCA expression or<br>
over-expression in cancer cells is of therapeutic value. For example, such an agent can be identified by using a screen that<br>
quantifies PSCA expression by RT-PCR, nucleic acid hybridization or antibody binding.<br>
VIII.) Methods for Monitoring the Status of PSCA-related Genes and Their Products<br>
[0182] Oncogenesis is known to be a multistep process where cellular growth becomes progressively dysregulated<br>
and cells progress from a normal physiological state to precancerous and then cancerous states (see, e.g., Alers et al., Lab<br>
Invest. 77(5): 437-438 (1997) and Isaacs et al., Cancer Surv. 23:19-32 (1995)). In this context, examining a biological<br>
sample for evidence of dysregulated cell growth (such as aberrant PSCA expression in cancers) allows for early detection<br>
of such aberrant physiology, before a pathologic state such as cancer has progressed to a stage that therapeutic options<br>
are more limited and or the prognosis is worse. In such examinations, the status of PSCA in a biological sample of interest<br>
can be compared, for example, to the status of PSCA in a corresponding normal sample (e.g. a sample from that individual<br><br>
or alternatively another individual that is not affected by a pathology). An alteration in the status of PSCA in the biological<br>
sample (as compared to the normal sample) provides evidence of dysregulated cellular growth. In addition to using a<br>
biological sample that is not affected by a pathology as a normal sample, one can also use a predetermined normative<br>
value such as a predetermined normal level of mRNA expression (see, e.g., Grever et al., J. Comp. Neurol. 1996 Dec 9;<br>
376(2): 306-14 and U.S. Patent No. 5,837,501) to compare PSCA status in a sample.<br>
[0183] The term "status" in this context is used according to its art accepted meaning and refers to the condition or<br>
state of a gene and its products, Typically, skilled artisans use a number of parameters to evaluate the condition or state of<br>
a gene and its products. These include, but are not limited to the location of expressed gene products (including the<br>
location of PSCA expressing cells) as well as the level, and biological activity of expressed gene products (such as PSCA<br>
mRNA, polynucleotides and polypeptides). Typically, an alteration in the status of PSCA comprises a change in the location<br>
of PSCA and/or PSCA expressing cells and/or an increase in PSCA mRNA and/or protein expression.<br>
[0184] PSCA status in a sample can be analyzed by a number of means well known in the art, including without<br>
limitation, immunohistochemical analysis, in situ hybridization, RT-PCR analysis on laser capture micro-dissected samples,<br>
Western blot analysis, and tissue array analysis. Typical protocols for evaluating the status of a PSCA gene and gene<br>
products are found, for example in Ausubel et al. eds„ 1995, Current Protocols In Molecular Biology, Units 2 (Northern<br>
Blotting), 4 (Southern Blotting), 15 (Immunoblotting) and 18 (PCR Analysis). Thus, the status of PSCA in a biological<br>
sample is evaluated by various methods utilized by skilled artisans including, but not hmited to genomic Southern analysis<br>
(to examine, for example perturbations in a PSCA gene), Northern analysis and/o: PCR analysis of PSCA mRNA (to<br>
examine, for example alterations in the polynucleotide sequences or expression levels of PSCA mRNAs), and, Western<br>
and/or immunohistochemical analysis (to examine, for example alterations in polypeptide sequences, alterations in<br>
polypeptide localization within a sample, alterations in expression levels of PSCA proteins and/or associations of PSCA<br>
proteins with polypeptide binding partners). Detectable PSCA polynucleotides include, for example, a PSCA gene or<br>
fragment thereof, PSCA mRNA, alternative splice variants, PSCA mRNAs, and recombinant DNA or RNA molecules<br>
containing a PSCA polynucleotide.<br>
[0185] The expression profile of PSCA makes it a diagnostic marker for local and/or metastasized disease, and<br>
provides information on the growth or oncogenic potential of a biological sample. In particular, the status of PSCA provides<br>
information useful for predicting susceptibility to particular disease stages, progression, and/or tumor aggressiveness. The<br>
invention provides methods and assays for determining PSCA status and diagnosing cancers that express PSCA, such as<br>
cancers of the tissues listed in Table I. For example, because PSCA mRNA is so highly expressed in prostate and other<br>
cancers relative to normal prostate tissue, assays that evaluate the levels of PSCA mRNA transcripts or proteins in a<br>
biological sample can be used to diagnose a disease associated with PSCA dysregulation, and can provide prognostic<br>
information useful in defining appropriate therapeutic options.<br>
[0186] The expression status of PSCA provides information including the presence, stage and location of dysplastic,<br>
precancerous and cancerous cells, predicting susceptibility to various stages of disease, and/or for gauging tumor<br>
aggressiveness. Moreover, the expression profile makes it useful as an imaging reagent for metastasized disease.<br>
Consequently, an aspect of the invention is directed to the various molecular prognostic and diagnostic methods for<br>
examining the status of PSCA in biological samples such as those from individuals suffering from, or suspected of suffering<br>
from a pathology characterized by dysregulated cellular growth, such as cancer.<br>
[0187] As described above, the status of PSCA in a biological sample can be examined by a number of well-known<br>
procedures in the art. For example, the status of PSCA in a biological sample taken from a specific location in the body can<br><br>
be examined by evaluating the sample for the presence or absence of PSCA expressing cells (e.g. those that express<br>
PSGA mRNAs or proteins). This examination can provide evidence of dysreguiated cellular growth, for example, when<br>
PSCA-expressing cells are found in a biological sample that does not normally contain such cells (such as a lymph node),<br>
because such alterations in the status of PSCA in a biological sample are often associated with dysreguiated cellular<br>
growth. Specifically, one indicator of dysreguiated cellular growth is the metastases of cancer cells from an organ of origin<br>
(such as the prostate) to a different area of the body (such as a lymph node). In this context, evidence of dysreguiated<br>
cellular growth is important for example because occult lymph node metastases can be detected in a substantial proportion<br>
of patients with prostate cancer, and such metastases are associated with known predictors of disease progression (see,<br>
e.g., Murphy et al., Prostate 42(4): 315-317 (2000);Su et al., Semin. Surg. Oncol. 18(1): 17-28 (2000) and Freeman et al., J<br>
Urol 1995 Aug 154(2 Pt 1):474-8).<br>
[0188] In one aspect, the invention provides methods for monitoring PSCA gene products by determining the status<br>
of PSCA gene products expressed by cells from an individual suspected of having a disease associated with dysreguiated<br>
cell growth (such as hyperplasia or cancer) and then comparing the status so determined to the status of PSCA gene<br>
products in a corresponding normal sample. The presence of aberrant PSCA gene products in the test sampie relative to<br>
the normal sample provides an indication of the presence of dysreguiated cell growth within the cells of the individual.<br>
[0189] In another aspect, the invention provides assays useful in determining the presence of cancer in an<br>
individual, comprising detecting a significant increase in PSCA mRNA or protein expression in a test cell or tissue sample<br>
relative to expression levels in the corresponding normal cell or tissue. The presence of PSCA mRNA can, for example, be<br>
evaluated in tissues including but not limited to those listed in Table I. The presence of significant PSCA expression in any<br>
of these tissues is useful to indicate the emergence, presence and/or severity of a cancer, since the corresponding normal<br>
tissues do not express PSCA mRNA or express it at lower levels.<br>
[0190] In a related embodiment, PSCA status is determined at the protein level rather than at the nucleic acid level.<br>
For example, such a method comprises determining the level of PSCA protein expressed by cells in a test tissue sample<br>
and comparing the level so determined to the level of PSCA expressed in a corresponding normal sample. In one<br>
embodiment, the presence of PSCA protein is evaluated, for example, using immunohistochemical methods. PSCA<br>
antibodies or binding partners capable of detecting PSCA protein expression are used in a variety of assay formats well<br>
known in the art for this purpose.<br>
[0191] In a further embodiment, one can evaluate the status of PSCA nucleotide and amino acid sequences in a<br>
biological sample in order to identify perturbations in the structure of these molecules. These perturbations can include<br>
insertions, deletions, substitutions and the like. Such evaluations are useful because perturbations in the nucleotide and<br>
amino acid sequences are observed in a large number of proteins associated with a growth dysreguiated phenotype (see,<br>
e.g., Marrogi et al., 1999,, J. Cutan. Pathol. 26(8):369-378). For example, a mutation in the sequence of PSCA may be<br>
indicative of the presence or promotion of a tumor. Such assays therefore have diagnostic and predictive value where a<br>
mutation in PSCA indicates a potential loss of function or increase in tumor growth.<br>
[0192] A wide variety of assays for observing perturbations in nucleotide and amino acid sequences are well known<br>
in the art. For example, the size and structure of nucleic acid or amino acid sequences of PSCA gene products are<br>
observed by the Northern, Southern, Western, PCR and DNA sequencing protocols discussed herein. In addition, other<br>
methods for observing perturbations in nucleotide and amino acid sequences such as single strand conformation<br>
polymorphism analysis are well known in the art (see, e.g., U.S. Patent Nos. 5,382,510 issued 7 September 1999, and<br>
5,952,170 issued 17 January 1995).<br><br>
[0193] Additionally, one can examine the methylation status of a PSCA gene in a biological sample. Aberrant<br>
demethylation and/or hypermethylation of CpG islands in gene 5' regulatory regions frequently occurs in immortalized and<br>
transformed cells, and can result in altered expression of various genes. For example, promoter hypermethylation of the pi-<br>
class glutathione S-transferase (a protein expressed in normal prostate but not expressed in &gt;90% of prostate carcinomas)<br>
appears to permanently silence transcription of this gene and is the most frequently detected genomic alteration in prostate<br>
carcinomas (De Marzo et al., Am. J. Pathol. 155(6): 1985-1992 (1999)). In addition, this alteration is present in at least<br>
70st of cases of high-grade prostatic intraepithelial neoplasia (PIN) (Brooks et al., Cancer Epidemiol. Biomarkers Prev.,<br>
1998, 7.531 -536). In another example, expression of the LAGE-I tumor specific gene (which is not expressed in normal<br>
prostate but is expressed in 25-50% of prostate cancers) is induced by deoxy-azacytidine in lymphoblastoid cells,<br>
suggesting tnat tumoral expression is due to demethylation (Lethe et al., Int. J. Cancer 76(6): 903-908 (1998)). A variety of<br>
assays for examining methylation status of a gene are well known in the art. For example, one can utilize, in Southern<br>
hybroaction approaches, methylation-sensitive restriction enzymes that cannot cleave sequences that contain methylated<br>
CpG sfes to assess the methylation status of CpG islands. In addition, MSP (methylation specific PCR) can rapidly profile<br>
the moin lation status of all the CpG sites present in a CpG island of a given gene. This procedure involves initial<br>
modification of DNA by sodium bisulfite (which will convert all unmethylated cytosines to uracil) followed by amplification<br>
us:ng primers specific for methylated versus unmethylated DNA. Protocols involving methylation interference can also be<br>
found lor example in Current Protocols In Molecular Biology, Unit 12, Frederick M. Ausubel et al. eds., 1995.<br>
[0194] Gene amplification is an additional method for assessing the status of PSCA. Gene amplification is<br>
measured in a sample directly, for example, by conventional Southern blotting or Northern blotting to quantitate the<br>
transcription of mRNA (Thomas, 1980, Proc. Natl. Acad. Sci. USA, 77:5201-5205), dot blotting (DNA analysis), or in situ<br>
hybridization, using an appropriately labeled probe, based on the sequences provided herein. Alternatively, antibodies are<br>
employed that recognize specific duplexes, including DNA duplexes, RNA duplexes, and DNA-RNA hybrid duplexes or<br>
DNA-protein duplexes. The antibodies in turn are labeled and the assay carried out where the duplex is bound to a<br>
surface, so that upon the formation of duplex on the surface, the presence of antibody bound to the duplex can be detected.<br>
[0195] Biopsied tissue or peripheral blood can be conveniently assayed for the presence of cancer cells using for<br>
example, Northern, dot blot or RT-PCR analysis to detect PSCA expression. The presence of RT-PCR amplifiable PSCA<br>
mRNA provides an indication of the presence of cancer. RT-PCR assays are well known in the art. RT-PCR detection<br>
assays for tumor cells in peripheral blood are currently being evaluated for use in the diagnosis and management of a<br>
number of human solid tumors. In the prostate cancer field, these include RT-PCR assays for the detection of cells<br>
expressing PSA and PSM (Verkaik et al., 1997, Urol. Res. 25:373-384; Ghossein et al., 1995, J. Clin. Oncol. 13:1195-2000;<br>
Heston et al., 1995, Clin. Chem. 41:1687-1688).<br>
[0196] A further aspect of the invention is an assessment of the susceptibility that an individual has for developing<br>
cancer. In one embodiment, a method for predicting susceptibility to cancer comprises detecting PSCA mRNA or PSCA<br>
protein in a tissue sample, its presence indicating susceptibility to cancer, wherein the degree of PSCA mRNA expression<br>
correlates to the degree of susceptibility. In a specific embodiment, the presence of PSCA in prostate or other tissue is<br>
examined, with the presence of PSCA in the sample providing an indication of prostate cancer susceptibility (or the<br>
emergence or existence of a prostate tumor). Similarly, one can evaluate the integrity PSCA nucleotide and amino acid<br>
sequences in a biological sample, in order to identify perturbations in the structure of these molecules such as insertions,<br>
deletions, substitutions and the like. The presence of one or more perturbations in PSCA gene products in the sample is an<br>
indication of cancer susceptibility (or the emergence or existence of a tumor).<br><br>
[0197] The invention also comprises methods for gauging tumor aggressiveness. In one embodiment, a method for<br>
gauging aggressiveness of a tumor comprises determining the level of PSCA mRNA or PSCA protein expressed by tumor<br>
celts, comparing the level so determined to the level of PSCA mRNA or PSCA protein expressed in a corresponding normal<br>
tissue taken from the same individual or a normal tissue reference sample, wherein the degree of PSCA mRNA or PSCA<br>
protein expression in the tumor sample relative to the normal sample indicates the degree of aggressiveness. In a specific<br>
embodiment, aggressiveness of a tumor is evaluated by determining the extent to which PSCA is expressed in the tumor<br>
cells, with higher expression levels indicating more aggressive tumors. Another embodiment is the evaluation of the<br>
integrity of PSCA nucleotide and amino acid sequences in a biological sample, in order to identify perturbations in the<br>
structure of these molecules such as insertions, deletions, substitutions and the like. The presence of one or more<br>
perturbations indicates more aggressive tumors.<br>
[0198] Another embodiment of the invention is directed to methods for observing the progression of a malignancy in<br>
an individual over time. In one embodiment, methods for observing the progression of a malignancy in an individual over<br>
time comprise determining the level of PSCA mRNA or PSCA protein expressed by cells in a sample of the tumor,<br>
comparing the level so determined to the level of PSCA mRNA or PSCA protein expressed in an equivalent tissue sample<br>
taken from the same individual at a different time, wherein the degree of PSCA mRNA or PSCA protein expression in the<br>
tumor sample over time provides information on the progression of the cancer. In a specific embodiment, the progression<br>
of a cancer is evaluated by determining PSCA expression in the tumor cells over time, where increased expression over<br>
time indicates a progression of the cancer. Also, one can evaluate the integrity PSCA nucleotide and amino acid<br>
sequences in a biological sample in order to identify perturbations in the structure of these molecules such as insertions,<br>
deletions, substitutions and the like, where the presence of one or more perturbations indicates a progression of the cancer.<br>
[0199] The above diagnostic approaches can be combined with any one of a wide variety of prognostic and<br>
diagnostic protocols known in the art. For example, another embodiment of the invention is directed to methods for<br>
observing a coincidence between the expression of PSCA gene and PSCA gene products (or perturbations in PSCA gene<br>
and PSCA gene products) and a factor that is associated with malignancy, as a means for diagnosing and prognosticating<br>
the status of a tissue sample. A wide variety of factors associated with malignancy can be utilized, such as the expression<br>
of genes associated with malignancy (e.g. PSA, PSCA and PSM expression for prostate cancer etc.) as well as gross<br>
cytological observations (see, e.g., Booking et al., 1984, Anal. Quant. Cytol. 6(2):74-88; Epstein, 1995, Hum. Pathol.<br>
26(2):223-9; Thorson et al, 1998, Mod. Pathol. 11(6):543-51; Baisden et al., 1999, Am. J. Surg. Pathol. 23(8):918-24).<br>
Methods for observing a coincidence between the expression of PSCA gene and PSCA gene products (or perturbations in<br>
PSCA gene and PSCA gene products) and another factor that is associated with malignancy are useful, for example,<br>
because the presence of a set of specific factors that coincide with disease provides information crucial for diagnosing and<br>
prognosticating the status of a tissue sample.<br>
[0200] In one embodiment, methods for observing a coincidence between the expression of PSCA gene and PSCA<br>
gene products (or perturbations in PSCA gene and PSCA gene products) and another factor associated with malignancy<br>
entails detecting the overexpression of PSCA mRNA or protein in a tissue sample, detecting the overexpression of PSA<br>
mRNA or protein in a tissue sample (or PSCA or PSM expression), and observing a coincidence of PSCA mRNA or protein<br>
and PSA mRNA or protein overexpression (or PSCA or PSM expression). In a specific embodiment, the expression of<br>
PSCA and PSA mRNA in prostate tissue is examined, where the coincidence of PSCA and PSA mRNA overexpression in<br>
the sample indicates the existence of prostate cancer, prostate cancer susceptibility or the emergence or status of a<br>
prostate tumor.<br><br>
[0201] Methods for detecting and quantifying the expression of PSCA mRNA or protein are described herein, and<br>
standard nucleic acid and protein detection and quantification technologies are well known in the art. Standard methods for<br>
the detection and quantification of PSCA mRNA include in situ hybridization using labeled PSCA riboprobes, Northern blot<br>
and related techniques using PSCA polynucleotide probes, RT-PCR analysis using primers specific for PSCA, and other<br>
amplification type detection methods, such as, for example, branched DNA, SISBA, TMA and the like. In a specific<br>
embodiment, semi-quantitative RT-PCR is used to detect and quantify PSCA mRNA expression. Any number of primers<br>
capable of amplifying PSCA can be used for this purpose, including but not limited to the various primer sets specifically<br>
described herein. In a specific embodiment, polyclonal or monoclonal antibodies specifically reactive with the wild-type<br>
PSCA protein can be used in an immunohistochemical assay of biopsied tissue.<br>
IX.) Identification of Molecules That Interact With PSCA<br>
[0202] The PSCA protein and nucleic acid sequences disclosed herein allow a skilled artisan to identify proteins,<br>
small molecules and other agents that interact with PSCA, as well as pathways activated by PSCA via any one of a variety<br>
of art accepted protocols. For example, one can utilize one of the so-called interaction trap systems (also referred to as the<br>
"two-hybrid assay"). In such systems, molecules interact and reconstitute a transcription factor which directs expression of<br>
a reporter gene, whereupon the expression of the reporter gene is assayed. Other systems identify protein-protein<br>
interactions in vivo through reconstitution of a eukaryotic transcriptional activator, see, e.g., U.S. Patent Nos. 5,955,280<br>
issued 21 September 1999, 5,925,523 issued 20 July 1999, 5,846,722 issued 8 December 1998 and 6,004,746 issued 21<br>
December 1999. Algorithms are also available in the art for genome-based predictions of protein function (see, e.g.,<br>
Marcotte, et al., Nature 402:4 November 1999, 83-86).<br>
[0203] Alternatively one can screen peptide libraries to identify molecules that interact with PSCA protein sequences.<br>
In such methods, peptides that bind to PSCA are identified by screening libraries that encode a random or controlled<br>
collection of amino acids. Peptides encoded by the libraries are expressed as fusion proteins of bacteriophage coat<br>
proteins, the bacteriophage particles are then screened against the PSCA protein(s).<br>
[0204] Accordingly, peptides having a wide variety of uses, such as therapeutic, prognostic or diagnostic reagents,<br>
are thus identified without any prior information on the structure of the expected ligand or receptor molecule. Typical<br>
peptide libraries and screening methods that can be used to identify molecules that interact with PSCA protein sequences<br>
are disclosed for example in U.S. Patent Nos. 5,723,286 issued 3 March 1998 and 5,733,731 issued 31 March 1998.<br>
[0205] Alternatively, cell lines that express PSCA are used to identify protein-protein interactions mediated by PSCA.<br>
Such interactions can be examined using immunoprecipitation techniques (see, e.g., Hamilton B.J., et al. Biochem.<br>
Biophys. Res. Commun. 1999,261:646-51). PSCA protein can be immunoprecipitated from PSCA-expressing cell lines<br>
using anti-PSCA antibodies. Alternatively, antibodies against His-tag can be used in a cell line engineered to express<br>
fusions of PSCA and a His-tag (vectors mentioned above). The immunoprecipitated complex can be examined for protein<br>
association by procedures such as Western blotting, 35S-methionine labeling of proteins, protein microsequencing, silver<br>
staining and two-dimensional gel electrophoresis.<br>
[0206] Small molecules and ligands that interact with PSCA can be identified through related embodiments of such<br>
screening assays. For example, small molecules can be identified that interfere with protein function, including molecules<br>
that interfere with PSCA's ability to mediate phosphorylation and de-phosphorylation, interaction with DNA or RNA<br>
molecules as an indication of regulation of cell cycles, second messenger signaling or tumorigenesis. Similarly, small<br>
molecules that modulate PSCA-related ion channel, protein pump, or cell communication functions are identified and used<br><br>
to treat patients that have a cancer that expresses PSCA (see, e.g., Hilie, B., Ionic Channels of Excitable Membranes 2"d<br>
Ed. Siauer Assoc, Sunderland, MA, 1992). Moreover, ligands that regulate PSCA function can be identified based on<br>
their ability to bind PSCA and activate a reporter construct. Typical methods are discussed for example in U.S. Patent No.<br>
5,928,868 issued 27 July 1999, and include methods for forming hybrid ligands in which at least one ligand is a small<br>
molecule. In an illustrative embodiment, cells engineered to express a fusion protein of PSCA and a DNA-binding protein<br>
are used to co-express a fusion protein of a hybrid ligand/small molecule and a cDNA library transcriptional activator<br>
protein. The cells further contain a reporter gene, the expression of which is conditioned on the proximity of the first and<br>
second fusion proteins to each other, an event that occurs only if the hybrid ligand binds to target sites on both hybrid<br>
proteins. Those cells that express the reporter gene are selected and the unknown small molecule or the unknown ligand<br>
is identified. This method provides a means of identifying modulators, which activate or inhibit PSCA.<br>
[0207] An embodiment of this invention comprises a method of screening for a molecule that interacts with a PSCA<br>
amino acid sequence shown in Figure 2 or Figure 3, comprising the steps of contacting a population of molecules with a<br>
PSCA amino acid sequence, allowing the population of molecules and the PSCA amino acid sequence to interact under<br>
conditions that facilitate an interaction, determining the presence of a molecule that interacts with the PSCA amino acid<br>
sequence, and then separating molecules that do not interact with the PSCA amino acid sequence from molecules that do.<br>
In a specific embodiment, the method further comprises purifying, characterizing and identifying a molecule that interacts<br>
with the PSCA amino acid sequence. The identified molecule can be used to modulate a function performed by PSCA. In<br>
a preferred embodiment, the PSCA amino acid sequence is contacted with a library of peptides.<br>
X.) Therapeutic Methods and Compositions<br>
[0208] The identification of PSCA as a protein that is normally expressed in a restricted set of tissues, but which is<br>
also expressed in cancers such as those listed in Table I, opens a number of therapeutic approaches to the treatment of<br>
such cancers.<br>
[0209] Of note, targeted antitumor therapies have been useful even when the targeted protein is expressed on<br>
normal tissues, even vital normal organ tissues. A vital organ is one that is necessary to sustain life, such as the heart or<br>
colon. A non-vital organ is one that can be removed whereupon the individual is still able to survive. Examples of non-vital<br>
organs are ovary, breast, and prostate.<br>
[0210] For example, Herceptin® is an FDA approved pharmaceutical that has as its active ingredient an antibody<br>
which is immunoreactive with the protein variously known as HER2, HER2/neu, and erb-b-2. It is marketed by Genentecft<br>
and has been a commercially successful antitumor agent. Herceptin sales reached almost $400 million in 2002. Herceptin<br>
is a treatment for HER2 positive metastatic breast cancer. However, the expression of HER2 is not limited to such tumors.<br>
The same protein is expressed in a number of normal tissues. In particular, it is known that HER2/neu is present in normal<br>
kidney and heart, thus these tissues are present in all human recipients of Herceptin. The presence of HER2/neu in normal<br>
kidney is also confirmed by Latif, Z., et al., B.J.U. International (2002) 89:5-9. As shown in this article (which evaluated<br>
whether renal cell carcinoma should be a preferred indication for anti-HER2 antibodies such as Herceptin) both protein and<br>
mRNA are produced in benign renal tissues. Notably, HER2/neu protein was strongly overexpressed in benign renal<br>
tissue.<br>
[0211] Despite the fact that HER2/neu is expressed in such vital tissues as heart and kidney, Herceptin is a very<br>
useful, FDA approved, and commercially successful drug. The effect of Herceptin on cardiac tissue, i.e., "cardiotoxicity,"<br>
has merely been a side effect to treatment. When patients were treated with Herceptin alone, significant cardiotoxicity<br>
occurred in a very low percentage of patients.<br>
[0212] Of particular note, although kidney tissue is indicated to exhibit normal expression, possibly even higher<br>
expression than cardiac tissue, kidney has no appreciable Herceptin side effect whatsoever. Moreover, of the diverse array<br>
of normal tissues in which HER2 is expressed, there is very little occurrence of any side effect. Only cardiac tissue has<br>
manifested any appreciable side effect at all. A tissue such as kidney, where HER2/neu expression is especially notable,<br>
has not been the basis for any side effect.<br>
[0213] Furthermore, favorable therapeutic effects have been found for antitumor therapies that target<br>
epidermal growth factor receptor (EGFR). EGFR is also expressed in numerous normal tissues. There have been very<br>
limited side effects in normal tissues following use of anti-EGFR therapeutics.<br>
[0214] Thus, expression of a target protein in normal tissue, even vital normal tissue, does not defeat the utility<br>
of a targeting agent for the protein as a therapeutic for certain tumors in which the protein is also overexpressed.<br>
[0215] Accordingly, therapeutic approaches that inhibit the activity of a PSCA protein are useful for patients suffering<br>
from a cancer that expresses PSCA. These therapeutic approaches generally fall into two classes. One class comprises<br>
various methods for inhibiting the binding or association of a PSCA protein with its binding partner or with other proteins.<br>
Another class comprises a variety of methods for inhibiting the transcription of a PSCA gene or translation of PSCA mRNA.<br>
X.A.) Anti-Cancer Vaccines<br>
[0216] The invention provides cancer vaccines comprising a PSCA-related protein or PSCA-related nucleic acid. In<br>
view of the expression of PSCA, cancer vaccines prevent and/or treat PSCA-expressing cancers with minimal or no effects<br>
on non-target tissues. The use of a tumor antigen in a vaccine that generates humoral and/or cell-mediated immune<br>
responses as anti-cancer therapy is well known in the art and has been employed in prostate cancer using human PSMA<br>
and rodent PAP immunogens (Hodge et al., 1995, Int. J. Cancer 63:231-237; Fong et al., 1997, J. Immunol. 159:3113-<br>
3117).<br>
[0217] Such methods can be readily practiced by employing a PSCA-related protein, or a PSCA-encoding nucleic<br>
acid molecule and recombinant vectors capable of expressing and presenting the PSCA immunogen (which typically<br>
comprises a number of antibody or T cell epitopes). Skilled artisans understand that a wide variety of vaccine systems for<br>
delivery of immunoreactive epitopes are known in the art (see, e.g., Heryln et al., Ann Med 1999 Feb 31(1):66-78;<br>
Maruyama et al., Cancer Immunol Immunother 2000 Jun 49(3):123-32) Briefly, such methods of generating an immune<br>
response (e.g. humoral and/or cell-mediated) in a mammal, comprise the steps of: exposing the mammal's immune system<br>
to an immunoreactive epitope (e.g. an epitope present in a PSCA protein shown in Figure 3 or analog or homolog thereof)<br>
so that the mammal generates an immune response that is specific for that epitope (e.g. generates antibodies that<br>
specifically recognize that epitope). In a preferred method, a PSCA immunogen contains a biological motif, see e.g.,<br>
Tables VIII-XXI and XXII-XLIX, or a peptide of a size range from PSCA indicated in Figure 5, Figure 6, Figure 7, Figure 8,<br>
and Figure 9.<br>
[0218] The entire PSCA protein, immunogenic regions or epitopes thereof can be combined and delivered by<br>
various means. Such vaccine compositions can include, for example, lipopeptides (e.g.,Vitiello, A. et al., J. Clin. Invest.<br>
95:341,1995), peptide compositions encapsulated in poly(DL-lactide-co-glycolide) ("PLG") microspheres (see, e.g.,<br>
Eldridge, et al., Molec. Immunol. 28:287-294,1991: Alonso et al., Vaccine 12:299-306,1994; Jones et al., Vaccine 13:675-<br>
681,1995), peptide compositions contained in immune stimulating complexes (ISCOMS) (see, e.g., Takahashi et al.,<br><br>
Nature 344:873-875,1990; Hu et al., Clin Exp Immunol. 113:235-243,1998), multiple antigen peptide systems (MAPs) (see<br>
Tarn, J. P., Proc. Natl. Acad. Sci. U.S.A. 85:5409-5413,1988; Tarn, J.P., J. Immunol. Methods 196:17-32,1996),<br>
peptides formulated as multivalent peptides; peptides for use in ballistic delivery systems, typically crystallized peptides,<br>
viral delivery vectors (Perkus, M. E. et al., In: Concepts in vaccine development, Kaufmann, S. H. E., ed., p. 379,1996;<br>
Chakrabarti, S. et al., Nature 320:535,1986; Hu, S. L et al., Nature 320:537,1986; Kieny, M.-P. et al., AIDS<br>
Bio/Technology 4:790,1986; Top, F. H. et al., J. Infect. Dis. 124:148,1971; Chanda, P. K. et al., Virology 175:535,1990),<br>
particles of viral or synthetic origin (e.g., Kofler, N. et al., J. Immunol. Methods. 192:25,1996; Eldridge, J. H. et al., Sem.<br>
Hematol. 30:16,1993; Falo, L. D, Jr. et al., Nature Med. 7:649,1995), adjuvants (Warren, H. S., Vogel, F. R., and Chedid,<br>
L A. Annu. Rev. Immunol. 4:369,1986; Gupta, R. K. et al., Vaccine 11:293,1993), liposomes (Reddy, R. et al., J.<br>
Immunol. 148:1585,1992; Rock, K. L, Immunol. Today 17:131,1996), or, naked or particle absorbed cDNA (Ulmer, J. B. et<br>
al., Science 259:1745,1993; Robinson, H. L, Hunt, L A., and Webster, R. G., Vaccine 11:957,1993; Shiver, J. W. et al.,<br>
In: Concepts in vaccine development, Kaufmann, S. H. E., ed., p. 423,1996; Cease, K. B., and Berzofsky, J. A., Annu.<br>
Rev. Immunol. 12:923,1994 and Eldridge, J. H. et al., Sem. Hematol. 30:16,1993). Toxin-targeted delivery technologies,<br>
also known as receptor mediated targeting, such as those of Avant Immunotherapeutics, Inc. (Needham, Massachusetts)<br>
may also be used.<br>
[0219] In patients with PSCA-associated cancer, the vaccine compositions of the invention can also be used in<br>
conjunction with other treatments used for cancer, e.g., surgery, chemotherapy, drug therapies, radiation therapies, etc.<br>
including use in combination with immune adjuvants such as IL-2, IL-12, GM-CSF, and the like.<br>
Cellular Vaccines:<br>
[0220] CTL epitopes can be determined using specific algorithms to identify peptides within PSCA protein that bind<br>
corresponding HLA alleles (see e.g., Table IV; Epimer™ and Epimatrix™, Brown University (URL brown.edu/Research/TB-<br>
HIV_Lab/epimatrix/epimatrix.html); and, BIMAS, (URL bimas.dcrt.nih.gov/; SYFPEITHI at URL syfpeithi.bmi-<br>
heidelberg.com/). In a preferred embodiment, a PSCA immunogen contains one or more amino acid sequences identified<br>
using techniques well known in the art, such as the sequences shown in Tables VIII-XXI and XXII-XLIX or a peptide of 8, 9,<br>
10 or 11 amino acids specified by an HLA Class I motif/supermotif (e.g., Table IV (A), Table IV (D), or Table IV (E)) and/or a<br>
peptide of at least 9 amino acids that comprises an HLA Class II motif/supermotif (e.g., Table IV (B) or Table IV (C)). As is<br>
appreciated in the art, the HLA Class I binding groove is essentially closed ended so that peptides of only a particular size<br>
range can fit into the groove and be bound, generally HLA Class I epitopes are 8, 9,10, or 11 amino acids long. In<br>
contrast, the HLA Class II binding groove is essentially open ended; therefore a peptide of about 9 or more amino acids can<br>
be bound by an HLA Class II molecule. Due to the binding groove differences between HLA Class I and II, HLA Class I<br>
motifs are length specific, i.e., position two of a Class I motif is the second amino acid in an amino to carboxyl direction of<br>
the peptide. The amino acid positions in a Class II motif are relative only to each other, not the overall peptide, i.e.,<br>
additional amino acids can be attached to the amino and/or carboxyl termini of a motif-bearing sequence. HLA Class II<br>
epitopes are often 9,10,11,12,13,14,15,16,17,18,19,20,21, 22, 23, 24, or 25 amino acids long, or longer than 25<br>
amino acids.<br>
Antibody-based Vaccines<br>
[0221 ] A wide variety of methods for generating an immune response in a mammal are known in the art (for example<br>
as the first step in the generation of hybridomas). Methods of generating an immune response in a mammal comprise<br>
exposing the mammal's immune system to an immunogenic epitope on a protein (e.g. a PSCA protein) so that an immune<br>
response is generated. A typical embodiment consists of a method for generating an immune response to PSCA in a host,<br>
by contacting the host with a sufficient amount of at least one PSCA B cell or cytotoxic T-cell epitope or analog thereof; and<br>
at least one periodic interval thereafter re-contacting the host with the PSCA B cell or cytotoxic T-cell epitope or analog<br>
thereof. A specific embodiment consists of a method of generating an immune response against a PSCA-related protein or<br>
a man-made multiepitopic peptide comprising: administering PSCA immunogen (e.g. a PSCA protein or a peptide fragment<br>
thereof, a PSCA fusion protein or analog etc.) in a vaccine preparation to a human or another mammal. Typically, such<br>
vaccine preparations further contain a suitable adjuvant (see, e.g., U.S. Patent No. 6,146,635) or a universal helper epitope<br>
such as a PADRE™ peptide (Epimmune Inc., San Diego, CA; see, e.g., Alexander et al, J. Immunol. 2000164(3); 164(3):<br>
1625-1633; Alexander et al, Immunity 19941(9): 751-761 and Alexander et al, Immunol. Res. 199818(2): 79-92). An<br>
alternative method comprises generating an immune response in an individual against a PSCA immunogen by:<br>
administering in vivo to muscle or skin of the individual's body a DNA molecule that comprises a DNA sequence that<br>
encodes a PSCA immunogen, the DNA sequence operatively linked to regulatory sequences which control the expression<br>
of the DNA sequence; wherein the DNA molecule is taken up by cells, the DNA sequence is expressed in the cells and an<br>
immune response is generated against the immunogen (see, e.g., U.S. Patent No. 5,962,428). Optionally a genetic<br>
vaccine facilitator such as anionic lipids; saponins; lectins; estrogenic compounds; hydroxylated lowet alkyls; dimethyl<br>
sulfoxide; and urea is also administered. In addition, an antiidiotypic antibody can be administered that mimics PSCA, in<br>
order to generate a response to the target antigen.<br>
Nucleic Acid Vaccines:<br>
[0222] Vaccine compositions of the invention include nucleic acid-mediated modalities. DNA or RNA that encode<br>
protein(s) of the invention can be administered to a patient. Genetic immunization methods can be employed to generate<br>
prophylactic or therapeutic humoral and cellular immune responses directed against cancer cells expressing PSCA.<br>
Constructs comprising DNA encoding a PSCA-related protein/immunogen and appropriate regulatory sequences can be<br>
injected directly into muscle or skin of an individual, such that the cells of the muscle or skin take-up the construct and<br>
express the encoded PSCA protein/immunogen. Alternatively, a vaccine comprises a PSCA-related protein. Expression of<br>
the PSCA-related protein immunogen results in the generation of prophylactic or therapeutic humoral and cellular immunity<br>
against cells that bear a PSCA protein. Various prophylactic and therapeutic genetic immunization techniques known in the<br>
art can be used (for review, see information and references published at Internet address genvreb.com). Nucleic acid-<br>
based delivery is described, for instance, in Wolff ef. a/., Science 247:1465 (1990) as well as U.S. Patent Nos. 5,580,859;<br>
5,589,466; 5,804,566; 5,739,118; 5,736,524; 5,679,647; WO 98/04720. Examples of DNA-based delivery technologies<br>
include "naked DNA", facilitated (bupivicaine, polymers, peptide-mediated) delivery, cationic lipid complexes, and particle-<br>
mediated ("gene gun") or pressure-mediated delivery (see, e.g., U.S. Patent No. 5,922,687).<br>
[0223] For therapeutic or prophylactic immunization purposes, proteins of the invention can be expressed via viral or<br>
bacterial vectors. Various viral gene delivery systems that can be used in the practice of the invention include, but are not<br>
limited to, vaccinia, fowlpox, canarypox, adenovirus, influenza, poliovirus, adeno-associated virus, lentivirus, and sindbis virus<br>
(see, e.g., Restifo, 1996, Curr. Opin. Immunol. 8:658-663; Tsang et a/. J. Natl. Cancer Inst. 87:982-990 (1995)). Non-viral<br>
delivery systems can also be employed by introducing naked DNA encoding a PSCA-related protein into the patient (e.g.,<br>
intramuscularly or intradermally) to induce an anti-tumor response.<br>
[0224] Vaccinia virus is used, for example, as a vector to express nucleotide sequences that encode the peptides of<br>
the invention. Upon introduction into a host, the recombinant vaccinia virus expresses the protein immunogenic peptide,<br>
and thereby elicits a host immune response. Vaccinia vectors and methods useful in immunization protocols are described<br>
in, e.g., U.S. Patent No. 4,722,848. Another vector is BCG (Bacille Calmette Guerin). BCG vectors are described in Stover<br>
et al., Nature 351 -.456-460 (1991). A wide variety of other vectors useful for therapeutic administration or immunization of<br>
the peptides of the invention, e.g. adeno and adeno-associated virus vectors, retroviral vectors, Salmonella typhi vectors,<br>
detoxified anthrax toxin vectors, and the like, will be apparent to those skilled in the art from the description herein.<br>
[0225] Thus, gene delivery systems are used to deliver a PSCA-related nucleic acid molecule, in one embodiment,<br>
the full-length human PSCA cDNA is employed. In another embodiment, PSCA nucleic acid molecules encoding specific<br>
cytotoxic T lymphocyte (CTL) and/or antibody epitopes are employed.<br>
Ex Vivo Vaccines<br>
[0226] Various ex vivo strategies can also be employed to generate an immune response. One approach involves<br>
the use of antigen presenting cells (APCs) such as dendritic cells (DC) to present PSCA antigen to a patient's immune<br>
system. Dendritic cells express MHC class I and II molecules, B7 co-stimulator, and IL-12, and are thus highly specialized<br>
antigen presenting cells. In prostate cancer, autologous dendritic cells pulsed with peptides of the prostate-specific<br>
membrane antigen (PSMA) are being used in a Phase I clinical trial to stimulate prostate cancer patients' immune systems<br>
(Tjoa et al., 1996, Prostate 28:65-69; Murphy et al., 1996, Prostate 29:371-380). Thus, dendritic cells can be used to<br>
present PSCA peptides to T cells in the context of MHC class I or II molecules. In one embodiment, autologous dendritic<br>
cells are pulsed with PSCA peptides capable of binding to MHC class I and/or class II molecules. In another embodiment,<br>
dendritic cells are pulsed with the complete PSCA protein. Yet another embodiment involves engineering the<br>
overexpression of a PSCA gene in dendritic cells using various implementing vectors known in the art, such as adenovirus<br>
(Arthur et al., 1997, Cancer Gene Ther. 4:17-25), retrovirus (Henderson et al., 1996, Cancer Res. 56:3763-3770), lentivirus,<br>
adeno-associated virus, DNA transfection (Ribas et al., 1997, Cancer Res. 57:2865-2869), or tumor-derived RNA<br>
transfection (Ashley et al., 1997, J. Exp. Med. 186:1177-1182). Cells that express PSCA can also be engineered to<br>
express immune modulators, such as GM-CSF, and used as immunizing agents.<br>
X.B.) PSCA as a Target for Antibody-based Therapy<br>
[0227] 511582008800able the use of reduced dosages of concomitant chemotherapy, particularly for patients who<br>
do not tolerate the toxicity of the chemotherapeutic agent very well.<br>
[0228] Cancer patients can be evaluated for the presence and level of PSCA expression, preferably using<br>
immunohistochemical assessments of tumor tissue, quantitative PSCA imaging, or other techniques that reliably indicate<br>
the presence and degree of PSCA expression. Immunohistochemical analysis of tumor biopsies or surgical specimens is<br>
preferred for this purpose. Methods for immunohistochemical analysis of tumor tissues are well known in the art.<br>
[0229] Anti-PSCA monoclonal antibodies that treat prostate and other cancers include those that initiate a potent<br>
immune response against the tumor or those that are directly cytotoxic. In this regard, anti-PSCA monoclonal antibodies<br>
(mAbs) can elicit tumor cell lysis by either complement-mediated or antibody-dependent cell cytotoxicity (ADCC)<br><br>
mechanisms, both of which require an intact Fc portion of the immunoglobulin molecule for interaction with effector cell Fc<br>
receptor sites on complement proteins. In addition, anti-PSCA mAbs that exert a direct biological effect on tumor growth<br>
are useful to treat cancers that express PSCA. Mechanisms by which directly cytotoxic mAbs act include: inhibition of cell<br>
growth, modulation of cellular differentiation, modulation of tumor angiogenesis factor profiles, and the induction of<br>
apoptosis. The mechanism(s) by which a particular anti-PSCA mAb exerts an anti-tumor effect is evaluated using any<br>
number of in vitro assays that evaluate cell death such as ADCC, ADMMC, complement-mediated cell lysis, and so forth,<br>
as is generally known in the art.<br>
[0230] In some patients, the use of murine or other non-human monoclonal antibodies, or human/mouse chimeric<br>
mAbs can induce moderate to strong immune responses against the non-human antibody. This can result in clearance of<br>
the antibody from circulation and reduced efficacy. In the most severe cases, such an immune response can lead to the<br>
extensive formation of immune complexes which, potentially, can cause renal failure. Accordingly, preferred monoclonal<br>
antibodies used in the therapeutic methods of the invention are those that are either fully human or humanized and that<br>
bind specifically to the target PSCA antigen with high affinity but exhibit low or no antigenicity in the patient.<br>
[0231] Therapeutic methods of the invention contemplate the administration of single anti-PSCA mAbs as well as<br>
combinations, or cocktails, of different mAbs. Such mAb cocktails can have certain advantages inasmuch as they contain<br>
mAbs that target different epitopes, exploit different effector mechanisms or combine directly cytotoxic mAbs with mAbs that<br>
rely on immune effector functionality. Such mAbs in combination can exhibit synergistic therapeutic effects. In addition,<br>
anti-PSCA mAbs can be administered concomitantly with other therapeutic modalities, including but not limited to various<br>
chemotherapeutic agents, androgen-blockers, immune modulators (e.g., IL-2, GM-CSF), surgery or radiation. The anti-<br>
PSCA mAbs are administered in their "naked" or unconjugated form, or can have a therapeutic agent(s) conjugated to<br>
them.<br>
[0232] Anti-PSCA antibody formulations are administered via any route capable of delivering the antibodies to a<br>
tumor cell. Routes of administration include, but are not limited to, intravenous, intraperitoneal, intramuscular, intratumor,<br>
intradermal, and the like. Treatment generally involves repeated administration of the anti-PSCA antibody preparation, via<br>
an acceptable route of administration such as intravenous injection (IV), typically at a dose in the range of about 0.1, .2, .3,<br>
.4, .5, .6, .7, .8, .9., 1,2,3,4,5,6,7,8,9,10,15,20, or 25 mg/kg body weight. In general, doses in the range of 10-1000<br>
mg mAb per week are effective and well tolerated.<br>
[0233] Based on clinical experience with the Herceptin™ mAb in the treatment of metastatic breast cancer, an initial<br>
loading dose of approximately 4 mg/kg patient body weight IV, followed by weekly doses of about 2 mg/kg IV of the anti-<br>
PSCA mAb preparation represents an acceptable dosing regimen. Preferably, the initial loading dose is administered as a<br>
90-minute or longer infusion. The periodic maintenance dose is administered as a 30 minute or longer infusion, provided<br>
the initial dose was well tolerated. As appreciated by those of skill in the art, various factors can influence the ideal dose<br>
regimen in a particular case. Such factors include, for example, the binding affinity and half life of the Ab or mAbs used, the<br>
degree of PSCA expression in the patient, the extent of circulating shed PSCA antigen, the desired steady-state antibody<br>
concentration level, frequency of treatment, and the influence of chemotherapeutic or other agents used in combination with<br>
the treatment method of the invention, as well as the health status of a particular patient.<br>
[0234] Optionally, patients should be evaluated for the levels of PSCA in a given sample (e.g. the levels of<br>
circulating PSCA antigen and/or PSCA expressing cells) in order to assist in the determination of the most effective dosing<br>
regimen, etc. Such evaluations are also used for monitoring purposes throughout therapy, and are useful to gauge<br>
therapeutic success in combination with the evaluation of other parameters (for example, urine cytology and/or ImmunoCyt<br>
levels in bladder cancer therapy, or by analogy, serum PSA levels in prostate cancer therapy).<br>
[0235] Anti-idiotypic anti-PSCA antibodies can also be used in anti-cancer therapy as a vaccine for inducing an<br>
immune response to cells expressing a PSCA-related protein. In particular, the generation of anti-idiotypic antibodies is<br>
well known in the art; this methodology can readily be adapted to generate anti-idiotypic anti-PSCA antibodies that mimic<br>
an epitope on a PSCA-related protein (see, for example, Wagner et al., 1997, Hybridoma 16:33-40; Foon et al., 1995, J.<br>
Clin. invest. 96:334-342; Herlyn et al., 1996, Cancer Immunol. Immunother. 43:65-76). Such an anti-idiotypic antibody can<br>
be used in cancer vaccine strategies.<br>
X C.) PSCA as a Target for Cellular Immune Responses<br>
[0236] Vaccines and methods of preparing vaccines that contain an immunogenically effective amount of one or<br>
more HLA-binding peptides as described herein are further embodiments of the invention. Furthermore, vaccines in<br>
accordance with the invention encompass compositions of one or more of the claimed peptides. A peptide can be present<br>
in a vaccine individually. Alternatively, the peptide can exist as a homopolymer comprising multiple copies of the same<br>
peptide or as a heteropolymer of various peptides. Polymers have the advantage of increased immunological reaction<br>
and where different peptide epitopes are used to make up the polymer, the additional ability to induce antibodies and/or<br>
CTLs that react with different antigenic determinants of the pathogenic organism or tumor-related peptide targeted for an<br>
immune response. The composition can be a naturally occurring region of an antigen or can be prepared, e.g.,<br>
recombinantly or by chemical synthesis.<br>
[0237] Carriers that can be used with vaccines of the invention are well known in the art, and include, e.g.,<br>
thyroglobulin, albumins such as human serum albumin, tetanus toxoid, polyamino acids such as poly L-lysine, poly l-<br>
glutamic acid, influenza, hepatitis B virus core protein, and the like. The vaccines can contain a physiologically tolerable<br>
(i.e. acceptable) diluent such as water, or saline, preferably phosphate buffered saline. The vaccines also typically include<br>
an adjuvant. Adjuvants such as incomplete Freund's adjuvant, aluminum phosphate, aluminum hydroxide, or alum are<br>
examples of materials well known in the art. Additionally, as disclosed herein, CTL responses can be primed by<br>
conjugating peptides of the invention to lipids, such as tripalmitoyl-S-glycerylcysteinlyseryl- serine (P3CSS). Moreover, an<br>
adjuvant such as a synthetic cytosine-phosphorothiolated-guanine-containing (CpG) oligonucleotides has been found to<br>
increase CTL responses 10- to 100-fold, (see, e.g. Davila and Celis, J. Immunol. 165:539-547 (2000))<br>
[0238] Upon immunization with a peptide composition in accordance with the invention, via injection, aerosol, oral,<br>
transdermal, transmucosal, intrapleural, intrathecal, or other suitable routes, the immune system of the host responds to the<br>
vaccine by producing large amounts of CTLs and/or HTLs specific for the desired antigen. Consequently, the host<br>
becomes at least partially immune to later development of cells that express or overexpress PSCA antigen, or derives at<br>
least some therapeutic benefit when the antigen was tumor-associated.<br>
[0239] In some embodiments, it may be desirable to combine the class I peptide components with components that<br>
induce or facilitate neutralizing antibody and or helper T cell responses directed to the target antigen. A preferred<br>
embodiment of such a composition comprises class I and class II epitopes in accordance with the invention. An alternative<br>
embodiment of such a composition comprises a class I and/or class II epitope in accordance with the invention, along with<br>
a cross reactive HTL epitope such as PADRE™ (Epimmune, San Diego, CA) molecule (described e.g., in U.S. Patent<br>
Number 5,736,142).<br>
[0240] A vaccine of the invention can also include antigen-presenting cells (APC), such as dendritic cells (DC), as a<br>
vehicle to present peptides of the invention. Vaccine compositions can be created in vitro, following dendritic cell<br>
mobilization and harvesting, whereby loading of dendritic cells occurs in vitro. For example, dendritic cells are transfected,<br>
e.g., with a minigene in accordance with the invention, or are pulsed with peptides. The dendritic cell can then be<br>
administered to a patient to elicit immune responses in vivo. Vaccine compositions, either DNA- or peptide-based, can also<br>
be administered in vivo in combination with dendritic cell mobilization whereby loading of dendritic cells occurs in vivo.<br>
[0241 ] Preferably, the following principles are utilized when selecting an array of epitopes for inclusion in a<br>
polyepitopic composition for use in a vaccine, or for selecting discrete epitopes to be included in a vaccine and/or to be<br>
encoded by nucleic acids such as a minigene. It is preferred that each of the following principles be balanced in order to<br>
make the selection. The multiple epitopes to be incorporated in a given vaccine composition may be, but need not be,<br>
contiguous in sequence in the native antigen from which the epitopes are derived.<br>
1.) Epitopes are selected which, upon administration, mimic immune responses that have been observed<br>
to be correlated with tumor clearance. For HLA Class I this includes 3-4 epitopes that come from at least one tumor<br>
associated antigen (TAA). For HLA Class II a similar rationale is employed; again 3-4 epitopes are selected from at least<br>
one TAA (see, e.g., Rosenberg et at., Science 278:1447-1450). Epitopes from one TAA may be used in combination with<br>
epitopes from one or more additional TMs to produce a vaccine that targets tumors with varying expression patterns of<br>
frequently-expressed TAAs.<br>
2.) Epitopes are selected that have the requisite binding affinity established to be correlated with<br>
immunogenicity: for HLA Class I an IC50 of 500 nM or less, often 200 nM or less; and for Class II an IC50 of 1000 nM or<br>
less.<br>
3.) Sufficient supermotif bearing-peptides, or a sufficient array of allele-specific motif-bearing peptides, are<br>
selected to give broad population coverage. For example, it is preferable to have at least 80% population coverage. A<br>
Monte Carlo analysis, a statistical evaluation known in the art, can be employed to assess the breadth, or redundancy of,<br>
population coverage.<br>
4.) When selecting epitopes from cancer-related antigens it is often useful to select analogs because the<br>
patient may have developed tolerance to the native epitope.<br>
5.) Of particular relevance are epitopes referred to as "nested epitopes." Nested epitopes occur where at<br>
least two epitopes overlap in a given peptide sequence. A nested peptide sequence can comprise B cell, HLA class I<br>
and/or HLA class II epitopes. When providing nested epitopes, a general objective is to provide the greatest number of<br>
epitopes per sequence. Thus, an aspect is to avoid providing a peptide that is any longer than the amino terminus of the<br>
amino terminal epitope and the carboxyl terminus of the carboxyl terminal epitope in the peptide. When providing a muiti-<br>
epitopic sequence, such as a sequence comprising nested epitopes, it is generally important to screen the sequence in<br>
order to insure that it does not have pathological or other deleterious biological properties.<br>
6.) If a polyepitopic protein is created, or when creating a minigene, an objective is to generate the smallest<br>
peptide that encompasses the epitopes of interest. This principle is similar, if not the same as that employed when<br>
selecting a peptide comprising nested epitopes. However, with an artificial polyepitopic peptide, the size minimization<br>
objective is balanced against the need to integrate any spacer sequences between epitopes in the polyepitopic protein.<br>
Spacer amino acid residues can, for example, be introduced to avoid junctional epitopes (an epitope recognized by the<br>
immune system, not present in the target antigen, and only created by the man-made juxtaposition of epitopes), or to<br>
facilitate cleavage between epitopes and thereby enhance epitope presentation. Junctional epitopes are generally to be<br><br>
avoided because the recipient may generate an immune response to that non-native epitope. Of particular concern is a<br>
functional epitope that is a "dominant epitope." A dominant epitope may lead to such a zealous response that immune<br>
responses to other epitopes are diminished or suppressed.<br>
7.) Where the sequences of multiple variants of the same target protein are present, potential peptide<br>
epitopes can also be selected on the basis of their conservancy. For example, a criterion for conservancy may define that<br>
the entire sequence of an HLA class I binding peptide or the entire 9-mer core of a class II binding peptide be conserved in<br>
a designated percentage of the sequences evaluated for a specific protein antigen.<br>
X.C.1. Miniqene Vaccines<br>
[0242] A number of different approaches are available which allow simultaneous delivery of multiple epitopes.<br>
Nucleic acids encoding the peptides of the invention are a particularly useful embodiment of the invention. Epitopes for<br>
inclusion in a minigene are preferably selected according to the guidelines set forth in the previous section. A preferred<br>
means of administering nucleic acids encoding the peptides of the invention uses minigene constructs encoding a peptide<br>
comprising one or multiple epitopes of the invention.<br>
[0243] The use of multi-epitope minigenes is described below and in, Ishioka et al., J. Immunol. 162:3915-3925,<br>
1999; An, L and Whitton, J. L, J. Virol. 71:2292,1997; Thomson, S. A. et al., J. Immunol. 157:822,1996; Whitton, J. L. et al., J. Virol. 67:348,1993; Hanke, R. et al., Vaccine 16:426,1998. For example, a multi-epitope DNA plasmid encoding<br>
supermotif- and/or motif-bearing epitopes derived PSCA, the PADRE® universal helper T cell epitope or multiple HTL<br>
epitopes from PSCA (see e.g., Tables VIII-XXI and XXII to XLIX), and an endoplasmic reticulum-translocating signal<br>
sequence can be engineered. A vaccine may also comprise epitopes that are derived from other TAAs.<br>
[0244] The immunogenicity of a multi-epitopic minigene can be confirmed in transgenic mice to evaluate the<br>
magnitude of CTL induction responses against the epitopes tested. Further, the immunogenicity of DNA-encoded epitopes<br>
in vivo can be correlated with the in vitro responses of specific CTL lines against target cells transfected with the DNA<br>
plasmid. Thus, these experiments can show that the minigene serves to both: 1.) generate a CTL response and 2.) that the<br>
induced CTLs recognized cells expressing the encoded epitopes.<br>
[0245] For example, to create a DNA sequence encoding the selected epitopes (minigene) for expression in human<br>
cells, the amino acid sequences of the epitopes may be reverse translated. A human codon usage table can be used to<br>
guide the codon choice for each amino acid. These epitope-encoding DNA sequences may be directly adjoined, so that<br>
when translated, a continuous polypeptide sequence is created. To optimize expression and/or immunogenicity, additional<br>
elements can be incorporated into the minigene design. Examples of amino acid sequences that can be reverse translated<br>
and included in the minigene sequence include: HLA class I epitopes, HLA class II epitopes, antibody epitopes, a<br>
ubiquitination signal sequence, and/or an endoplasmic reticulum targeting signal. In addition, HLA presentation of CTL and<br>
HTL epitopes may be improved by including synthetic (e.g. poly-alanine) or naturally-occurring flanking sequences adjacent<br>
to the CTL or HTL epitopes; these larger peptides comprising the epitope(s) are within the scope of the invention.<br>
[0246] The minigene sequence may be converted to DNA by assembling oligonucleotides that encode the plus and<br>
minus strands of the minigene. Overlapping oligonucleotides (30-100 bases long) may be synthesized, phosphorylated,<br>
purified and annealed under appropriate conditions using well known techniques. The ends of the oligonucleotides can be<br>
joined, for example, using T4 DNA ligase. This synthetic minigene, encoding the epitope polypeptide, can then be cloned<br>
into a desired expression vector.<br>
[0247] Standard regulatory sequences well known to those of skill in the art are preferably included in the vector to<br>
ensure expression in the target cells. Several vector elements are desirable: a promoter with a down-stream cloning site<br>
for minigene insertion; a polyadenylation signal for efficient transcription termination; an E. coli origin of replication; and an<br>
E. coli selectable marker (e.g. ampicillin or kanamycin resistance). Numerous promoters can be used for this purpose, e.g.,<br>
the human cytomegalovirus (hCMV) promoter. See, e.g., U.S. Patent Nos. 5,580,859 and 5,589,466 for other suitable<br>
promoter sequences.<br>
[0248] Additional vector modifications may be desired to optimize minigene expression and immunogenicity In<br>
some cases, introns are required for efficient gene expression, and one or more synthetic or naturally-occurring nitrons<br>
could be incorporated into the transcribed region of the minigene. The inclusion of mRNA stabilization sequences and<br>
sequences lor replication in mammalian cells may also be considered for increasing minigene expression.<br>
[0249] Once an expression vector is selected, the minigene is cloned into the polylinker region downstream of the<br>
promote r This plasmid is transformed into an appropriate £ coli strain, and DNA is prepared using standard techniques.<br>
The orientation and DNA sequence of the minigene, as well as all other elements included in the vector, are confirmed<br>
using restriction mapping and DNA sequence analysis. Bacterial cells harboring the correct plasmid can be stored as a<br>
master cell bank and a working cell bank.<br>
[0250] In addition, immunostimulatory sequences (ISSs or CpGs) appear to play a role in the immunogenicity of<br>
DNA vaccines. These sequences may be included in the vector, outside the minigene coding sequence, if desired to<br>
enhance immunogenicity.<br>
[0251 ] In some embodiments, a bi-cistronic expression vector which allows production of both the minigene-encoded<br>
epitopes and a second protein (included to enhance or decrease immunogenicity) can be used. Examples of proteins or<br>
polypeptides that could beneficially enhance the immune response if co-expressed include cytokines (e.g., L-2, IL-12, GM-<br>
CSF), cytokine-inducing molecules (e.g., LeIF), costimulatory molecules, or for HTL responses, pan-DR binding proteins<br>
(PADRE™, Epimmune, San Diego, CA). Helper (HTL) epitopes can be joined to intracellular targeting signals and<br>
expressed separately from expressed CTL epitopes; this allows direction of the HTL epitopes to a cell compartment<br>
different than that of the CTL epitopes. If required, this could facilitate more efficient entry of HTL epitopes into the HLA<br>
class II pathway, thereby improving HTL induction. In contrast to HTL or CTL induction, specifically decreasing the immune<br>
response by co-expression of immunosuppressive molecules (e.g. TGF-ß) may be beneficial in certain diseases.<br>
[0252] Therapeutic quantities of plasmid DNA can be produced for example, by fermentation in £ coli, followed by<br>
purification. Aliquots from the working cell bank are used to inoculate growth medium, and grown to saturation in shaker<br>
flasks or a bioreactor according to well-known techniques. Plasmid DNA can be purified using standard bioseparation<br>
technologies such as solid phase anion-exchange resins supplied by QIAGEN, Inc. (Valencia, California). If required,<br>
supercoiled DNA can be isolated from the open circular and linear forms using gel electrophoresis or other methods.<br>
[0253] Purified plasmid DNA can be prepared for injection using a variety of formulations. The simplest of these is<br>
reconstitution of lyophilized DNA in sterile phosphate-buffer saline (PBS). This approach, known as "naked DNA," is<br>
currently being used for intramuscular (IM) administration in clinical trials. To maximize the immunotherapeutic effects of<br>
minigene DNA vaccines, an alternative method for formulating purified plasmid DNA may be desirable. A variety of<br>
methods have been described, and new techniques may become available. Cationic lipids, glycolipids, and fusogenic<br>
liposomes can also be used in the formulation (see, e.g., as described by WO 93/24640; Mannino &amp; Gould-Fogerite,<br>
BioTechniques 6(7): 682 (1988); U.S. Pat No. 5,279,833; WO 91/06309; and Feigner, et al, Proc. Nat'l Acad. Sci. USA<br>
84:7413 (1987). In addition, peptides and compounds referred to collectively as protective, interactive, non-condensing<br>
compounds (PINC) could also be complexed to purified plasmid DNA to influence variables such as stability, intramuscular<br>
dispersion, or trafficking to specific organs or cell types.<br>
[0254] Target cell sensitization can be used as a functional assay for expression and HLA class I presentation of<br>
minigene-encoded CTL epitopes. For example, the plasmid DNA is introduced into a mammalian cell line that is suitable as<br>
a target for standard CTL chromium release assays. The transfection method used will be dependent on the final<br>
formulation. Electroporation can be used for "naked" DNA, whereas cationic lipids allow direct in vitro transfection. A<br>
plasmid expressing green fluorescent protein (GFP) can be co-transfected to allow enrichment of transfected cells using<br>
fluorescence activated cell sorting (FACS). These cells are then chromium-51 (51Cr) labeled and used as target cells for<br>
epitope-specific CTL lines; cytolysis, detected by 51Cr release, indicates both production of, and HLA presentation of,<br>
minigene-encoded CTL epitopes. Expression of HTL epitopes may be evaluated in an analogous manner using assays to<br>
assess HTL activity.<br>
[0255] In vivo immunogenicity is a second approach for functional testing of minigene DNA formulations. Transgenic<br>
mice expressing appropriate human HLA proteins are immunized with the DNA product. The dose and route of<br>
administration are formulation dependent (e.g., lM for DNA in PBS, intraperitoneal (i.p.) for lipid-complexed DNA). Twenty-<br>
one days after immunization, splenocytes are harvested and restimulated for one week in the presence of peptides<br>
encoding each epitope being tested. Thereafter, for CTL effector cells, assays are conducted for cytolysis of peptide-<br>
loaded, 51Cr-labeled target cells using standard techniques. Lysis of target cells that were sensitized by HLA loaded with<br>
peptide epitopes, corresponding to minigene-encoded epitopes, demonstrates DNA vaccine function for in vivo induction of<br>
CTLs. Immunogenicity of HTL epitopes is confirmed in transgenic mice in an analogous manner.<br>
[0256] Alternatively, the nucleic acids can be administered using ballistic delivery as described, for instance, in U.S.<br>
Patent No. 5,204,253. Using this technique, particles comprised solely of DNA are administered. In a furthet alternative<br>
embodiment, DNA can be adhered to particles, such as gold particles.<br>
[0257] Minigenes can also be delivered using other bacterial or viral delivery systems well known in the art, e.g., an<br>
expression construct encoding epitopes of the invention can be incorporated into a viral vector such as vaccinia.<br>
X.C.2. Combinations of CTL Peptides with Helper Peptides<br>
[0258] Vaccine compositions comprising CTL peptides of the invention can be modified, e.g., analoged, to provide<br>
desired attributes, such as improved serum half life, broadened population coverage or enhanced immunogenicity.<br>
[0259] For instance, the ability of a peptide to induce CTL activity can be enhanced by linking the peptide to a<br>
sequence which contains at least one epitope that is capable of inducing a T helper cell response. Although a CTL peptide<br>
can be directly linked to a T helper peptide, often CTL epitope/HTL epitope conjugates are linked by a spacer molecule.<br>
The spacer is typically comprised of relatively small, neutral molecules, such as amino acids or amino acid mimetics, which<br>
are substantially uncharged under physiological conditions. The spacers are typically selected from, e.g., Ala, Gly, or other<br>
neutral spacers of nonpolar amino acids or neutral polar amino acids. It will be understood that the optionally present<br>
spacer need not be comprised of the same residues and thus may be a hetero- or homo-oligomer. When present, the<br>
spacer will usually be at least one or two residues, more usually three to six residues and sometimes 10 or more residues.<br>
The CTL peptide epitope can be linked to the T helper peptide epitope either directly or via a spacer either at the amino or<br>
carboxy terminus of the CTL peptide. The amino terminus of either the immunogenic peptide or the T helper peptide may<br>
be acylated.<br><br>
[0260] In certain embodiments, the T helper peptide is one that is recognized by T helper cells present in a majority<br>
of a genetically diverse population. This can be accomplished by selecting peptides that bind to many, most, or all of the<br>
HLA class II molecules. Examples of such amino acid bind many HLA Class II molecules include sequences from antigens<br>
such as tetanus toxoid at positions 830-843 (QY1KANSKFIGITE; SEQ ID NO:1), Plasmodium falciparum circumsporozoite<br>
(CS) protein at positions 378-398 (DIEKKIAKMEKASSVFNWNS; SEQ ID NO:2), and Streptococcus 18kD protein at<br>
positions 116-131 (GAVDSILGGVATYGAA; SEQ ID NO:3). Other examples include peptides bearing a DR 1-4-7<br>
supermotif, or either of the DR3 motifs.<br>
[0261] Alternatively, it is possible to prepare synthetic peptides capable of stimulating T helper lymphocytes, in a<br>
loosely HLA-restricted fashion, using amino acid sequences not found in nature (see, e.g., PCT publication WO 95/07707).<br>
These synthetic compounds called Pan-DR-binding epitopes (e.g., PADRE™, Epimmune, Inc., San Diego, CA) are<br>
designed, most preferably, to bind most HLA-DR (human HLA class II) molecules. For instance, a pan-DR-binding epitope<br>
peptide having the formula: aKXVMWTLKAa (SEQ ID NO:4), where "X" is either cyclohexylalanine, phenylalanine, or<br>
tyrosine, and a is either D-alanine or L-alanine, has been found to bind to most HLA-DR alleles, and to stimulate the<br>
response of T helper lymphocytes from most individuals, regardless of their HLA type. An alternative of a pan-DR binding'<br>
epitope comprises all "L" natural amino acids and can be provided in the form of nucleic acids that encode the epitope.<br>
[0262] HTL peptide epitopes can also be modified to alter their biological properties. For example, they can be<br>
modified to include D-amino acids to increase their resistance to proteases and thus extend their serum half life, or they can<br>
be conjugated to other molecules such as lipids, proteins, carbohydrates, and the like to increase their biological activity.<br>
For example, a T helper peptide can be conjugated to one or more palmitic acid chains at either the amino or carboxyl<br>
termini.<br>
X.C.3. Combinations of CTL Peptides with T Cell Priming Agents<br>
[0263] In some embodiments it may be desirable to include in the pharmaceutical compositions of the invention at<br>
least one component which primes B lymphocytes or T lymphocytes. Lipids have been identified as agents capable of<br>
priming CTL in vivo. For example, palmitic acid residues can be attached to the s-and a- amino groups of a lysine residue<br>
and then linked, e.g., via one or more linking residues such as Gly, Gly-Gly-, Ser, Ser-Ser, or the like, to an immunogenic<br>
peptide. The lipidated peptide can then be administered either directly in a micelle or particle, incorporated into a liposome,<br>
or emulsified in an adjuvant, e.g., incomplete Freund's adjuvant. In a preferred embodiment, a particularly effective<br>
immunogenic composition comprises palmitic acid attached to e- and a- amino groups of Lys, which is attached via<br>
linkage, e.g., Ser-Ser, to the amino terminus of the immunogenic peptide.<br>
[0264] As another example of lipid priming of CTL responses, E. coli lipoproteins, such as tripalmitoyl-S-<br>
glycerylcysteinlyseryl- serine (P3CSS) can be used to prime virus specific CTL when covalently attached to an appropriate<br>
peptide (see, e.g., Deres, et a/., Nature 342:561,1989). Peptides of the invention can be coupled to P3CSS, for example,<br>
and the lipopeptide administered to an individual to prime specifically an immune response to the target antigen. Moreover,<br>
because the induction of neutralizing antibodies can also be primed with P3CSS-conjugated epitopes, two such<br>
compositions can be combined to more effectively elicit both humoral and cell-mediated responses.<br>
X.C.4. Vaccine Compositions Comprising DC Pulsed with CTL and/or HTL Peptides<br>
[0265] An embodiment of a vaccine composition in accordance with the invention comprises ex vivo administration<br>
of a cocktail of epitope-bearing peptides to PBMC, or isolated DC therefrom, from the patient's blood. A pharmaceutical to<br>
facilitate harvesting of DC can be used, such as Progenipoietin™ (Pharmacia-Monsanto, St. Louis, MO) or GM-CSF/IL-4.<br>
After pulsing the DC with peptides and prior to reinfusion into patients, the DC are washed to remove unbound peptides. In<br>
this embodiment, a vaccine comprises peptide-pulsed DCs which present the pulsed peptide epitopes complexed with HLA<br>
molecules on their surfaces.<br>
[0266] The DC can be pulsed ex vivo with a cocktail of peptides, some of which stimulate CTL responses to PSCA.<br>
Optionally, a helper T cell (HTL) peptide, such as a natural or artificial loosely restricted HLA Class II peptide, can be<br>
included to facilitate the CTL response. Thus, a vaccine in accordance with the invention is used to treat a cancer which<br>
expresses or overexpresses PSCA.<br>
X.D. Adoptive Immunotherapy<br>
[0267] Antigenic PSCA-related peptides are used to elicit a CTL and/or HTL response ex vivo, as well. The resulting<br>
CTL or HTL cells, can be used to treat tumors in patients that do not respond to other conventional forms of therapy, or will<br>
not respond to a therapeutic vaccine peptide or nucleic acid in accordance with the invention. Ex vivo CTL or HTL<br>
responses to a particular antigen are induced by incubating in tissue culture the patient's, or genetically compatible, CTL or<br>
HTL precursor cells together with a source of antigen-presenting cells (APC), such as dendritic cells, and the appropriate<br>
immunogenic peptide. After an appropriate incubation time (typically about 7-28 days), in which the precursor cells are<br>
activated and expanded into effector cells, the cells are infused back into the patient, where they will destroy (CTL) or<br>
facilitate destruction (HTL) of their specific target cell (e.g., a tumor cell). Transfected dendritic cells may also be used as<br>
antigen presenting cells.<br>
X.E. Administration of Vaccines for Therapeutic or Prophylactic Purposes<br>
[0268] Pharmaceutical and vaccine compositions of the invention are typically used to treat and/or prevent a cancer<br>
that expresses or overexpresses PSCA. In therapeutic applications, peptide and/or nucleic acid compositions are<br>
administered to a patient in an amount sufficient to elicit an effective B cell, CTL and/or HTL response to the antigen and to<br>
cure or at least partially arrest or slow symptoms and/or complications. An amount adequate to accomplish this is defined<br>
as "therapeutically effective dose." Amounts effective for this use will depend on, e.g., the particular composition<br>
administered, the manner of administration, the stage and severity of the disease being treated, the weight and general<br>
state of health of the patient, and the judgment of the prescribing physician.<br>
[0269] For pharmaceutical compositions, the immunogenic peptides of the invention, or DNA encoding them, are<br>
generally administered to an individual already bearing a tumor that expresses PSCA. The peptides or DNA encoding them<br>
can be administered individually or as fusions of one or more peptide sequences. Patients can be treated with the<br>
immunogenic peptides separately or in conjunction with other treatments, such as surgery, as appropriate.<br>
[0270] For therapeutic use, administration should generally begin at the first diagnosis of PSCA-associated cancer.<br>
This is followed by boosting doses until at least symptoms are substantially abated and for a period thereafter. The<br>
embodiment of the vaccine composition (i.e., including, but not limited to embodiments such as peptide cocktails,<br>
polyepitopic polypeptides, minigenes, or TAA-specific CTLs or pulsed dendritic cells) delivered to the patient may vary<br>
according to the stage ot the disease or the patient's health status. For example, in a patient with a tumor that expresses<br>
PSCA, a vaccine comprising PSCA-specific CTL may be more efficacious in killing tumor cells in patient with advanced<br>
disease than alternative embodiments.<br>
[0271 ] It is generally important to provide an amount of the peptide epitope delivered by a mode of administration<br>
sufficient to stimulate effectively a cytotoxic T cell response; compositions which stimulate helper T cell responses can also<br>
be given in accordance with this embodiment of the invention.<br>
[0272] The dosage for an initial therapeutic immunization generally occurs in a unit dosage range where the lower<br>
value is about 1,5,50, 500, or 1,000 µg and the higher value is about 10,000; 20,000; 30,000; or 50,000 µg. Dosage<br>
values for a human typically range from about 500 µg to about 50,000 µg per 70 kilogram patient. Boosting dosages of<br>
between about 1.0 µg to about 50,000 µg of peptide pursuant to a boosting regimen over weeks to months may be<br>
administered depending upon the patient's response and condition as determined by measuring the specific activity of CTL<br>
and HTL obtained from the patient's blood. Administration should continue until at least clinical symptoms or laboratory<br>
tests indicate that the neoplasia, has been eliminated or reduced and for a period thereafter. The dosages, routes of<br>
administration, and dose schedules are adjusted in accordance with methodologies known in the art.<br>
[0273] In certain embodiments, the peptides and compositions of the present invention are employed in serious<br>
disease states, that is, life-threatening or potentially life threatening situations, in such cases, as a result of the minimal<br>
amounts of extraneous substances and the relative nontoxic nature of the peptides in preferred compositions of the<br>
invention, it is possible and may be felt desirable by the treating physician to administer substantial excesses of these<br>
peptide compositions relative to these stated dosage amounts.<br>
[0274] The vaccine compositions of the invention can also be used purely as prophylactic agents. Generally the<br>
dosage for an initial prophylactic immunization generally occurs in a unit dosage range where the lower value is about 1,5,<br>
50,500, or 1000 µg and the higher value is about 10,000; 20,000; 30,000; or 50,000 pg. Dosage values for a human<br>
typically range from about 500 µg to about 50,000 µg per 70 kilogram patient. This is followed by boosting dosages of<br>
between about 1.0 µg to about 50,000 µg of peptide administered at defined intervals from about four weeks to six months<br>
after the initial administration of vaccine. The immunogenicity of the vaccine can be assessed by measuring the specific<br>
activity of CTL and HTL obtained from a sample of the patient's blood.<br>
[0275] The pharmaceutical compositions for therapeutic treatment are intended for parenteral, topical, oral, nasal,<br>
intrathecal, or local (e.g. as a cream or topical ointment) administration. Preferably, the pharmaceutical compositions are<br>
administered parentally, e.g., intravenously, subcutaneously, intradermally, or intramuscularly. Thus, the invention provides<br>
compositions for parenteral administration which comprise a solution of the immunogenic peptides dissolved or suspended<br>
in an acceptable carrier, preferably an aqueous carrier.<br>
[0276] A variety of aqueous carriers may be used, e.g., water, buffered water, 0.8% saline, 0.3% glycine, hyaluronic<br>
acid and the like. These compositions may be sterilized by conventional, well-known sterilization techniques, or may be<br>
sterile filtered. The resulting aqueous solutions may be packaged for use as is, or lyophilized, the lyophilized preparation<br>
being combined with a sterile solution prior to administration.<br>
[0277] The compositions may contain pharmaceuticaily acceptable auxiliary substances as required to approximate<br>
physiological conditions, such as pH-adjusting and buffering agents, tonicity adjusting agents, wetting agents,<br>
preservatives, and the like, for example, sodium acetate, sodium lactate, sodium chloride, potassium chloride, calcium<br>
chloride, sorbitan monolaurate, triethanolamine oleate, etc.<br>
[0278] The concentration of peptides of the invention in the pharmaceutical formulations can vary widely, i.e., from<br>
tote than about 0.1 %, usually at or at least about 2% to as much as 20% to 50% or more by weight, and will be selected<br>
primarily by fluid volumes, viscosities, etc., in accordance with the particular mode of administration selected.<br>
[0279] A human unit dose form of a composition is typically included in a pharmaceutical composition that comprises<br>
a human unit dose of an acceptable carrier, in one embodiment an aqueous carrier, and is administered in a<br>
volume/quantity that is known by those of skill in the art to be used for administration of such compositions to humans (see,<br>
e.g., Remington's Pharmaceutical Sciences, 17th Edition, A. Gennaro, Editor, Mack Publishing Co., Easton, Pennsylvania,<br>
1985). For example a peptide dose for initial immunization can be from about 1 to about 50,000 \xg, generally 100-5,000<br>
µg, for a 70 kg patient. For example, for nucleic acids an initial immunization may be performed using an expression vector<br>
in the form of naked nucleic acid administered IM (or SC or ID) in the amounts of 0.5-5 mg at multiple sites. The nucleic<br>
acid (0.1 to 1000 µg) can also be administered using a gene gun. Following an incubation period of 3-4 weeks, a booster<br>
dose is then administered. The booster can be recombinant fowlpox virus administered at a dose of 5-107 to 5x109 pfu.<br>
[0280] For antibodies, a treatment generally involves repeated administration of the anti-PSCA antibody preparation,<br>
via an acceptable route of administration such as intravenous injection (IV), typically at a dose in the range of about 0.1 to<br>
about 10 mg/kg body weight. In general, doses in the range of 10-500 mg mAb per week are effective and well tolerated.<br>
Moreover, an initial loading dose of approximately 4 mg/kg patient body weight IV, followed by weekly doses of about 2<br>
mg/kg IV of the anti- PSCA mAb preparation represents an acceptable dosing regimen. As appreciated by those of skill in<br>
the art, various factors can influence the ideal dose in a particular case. Such factors include, for example, half life of a<br>
composition, the binding affinity of an Ab, the immunogenicity of a substance, the degree of PSCA expression in the<br>
patient, the extent of circulating shed PSCA antigen, the desired steady-state concentration level, frequency of treatment,<br>
and the influence of chemotherapeutic or other agents used in combination with the treatment method of the invention, as<br>
well as the health status of a particular patient. Non-limiting preferred human unit doses are, for example, 500ug -1 mg,<br>
1mg - 50mg, 50mg - 100mg, 100mg - 200mg, 200mg - 300mg, 400mg - 500mg, 500mg - 600mg, 600mg - 700mg, 700mg -<br>
800mg, 800mg - 900mg, 900mg - 1g, or 1mg - 700mg. In certain embodiments, the dose is in a range of 2-5 mg/kg body<br>
weight, e.g., with follow on weekly doses of 1-3 mg/kg; 0.5mg, 1, 2,3,4,5,6,7,8,9,10mg/kg body weight followed, e.g., in<br>
two, three or four weeks by weekly doses; 0.5 - 10mg/kg body weight, e.g., followed in two, three or four weeks by weekly<br>
doses; 225, 250, 275, 300, 325, 350, 375,400mg m2 of body area weekly; 1-600mg m2 of body area weekly; 225-400mg<br>
m2 of body area weekly; these does can be followed by weekly doses for 2, 3,4,5,6, 7, 8, 9,19,11,12 or more weeks.<br>
[0281] In one embodiment, human unit dose forms of polynucleotides comprise a suitable dosage range or effective<br>
amount that provides any therapeutic effect. As appreciated by one of ordinary skill in the art a therapeutic effect depends<br>
on a number of factors, including the sequence of the polynucleotide, molecular weight of the polynucleotide and route of<br>
administration. Dosages are generally selected by the physician or other health care professional in accordance with a<br>
variety of parameters known in the art, such as severity of symptoms, history of the patient and the like. Generally, for a<br>
polynucleotide of about 20 bases, a dosage range may be selected from, for example, an independently selected lower limit<br>
such as about 0.1, 0.25, 0.5,1,2, 5,10, 20, 30,40,50, 60,70,80,90,100, 200, 300,400 or 500 mg/kg up to an<br>
independently selected upper limit, greater than the lower limit, of about 60,80,100,200,300,400,500,750,1000,1500,<br>
2000,3000,4000, 5000,6000, 7000,8000, 9000 or 10,000 mg/kg. For example, a dose may be about any of the following:<br>
0.1 to 100 mg/kg, 0.1 to 50 mg/kg, 0.1 to 25 mg/kg, 0.1 to 10 mg/kg, 1 to 500 mg/kg, 100 to 400 mg/kg, 200 to 300 mg/kg, 1<br>
to 100 mg/kg, 100 to 200 mg/kg, 300 to 400 mg/kg, 400 to 500 mg/kg, 500 to 1000 mg/kg, 500 to 5000 mg/kg, or 500 to<br>
10,000 mg/kg. Generally, parenteral routes of administration may require higher doses of polynucleotide compared to more<br>
direct application to the nucleotide to diseased tissue, as do polynucleotides of increasing length.<br>
[0282] In one embodiment, human unit dose forms of T-cells comprise a suitable dosage range or effective amount<br>
that provides any therapeutic effect. As appreciated by one of ordinary skill in the art, a therapeutic effect depends on a<br>
number of factors. Dosages are generally selected by the physician or other health care professional in accordance with a<br>
variety of parameters known in the art, such as severity of symptoms, history of the patient and the like. A dose may be<br>
about 104 cells to about 106 cells, about 106 ceils to about 108 cells, about 108 to about 1011 cells, or about 108 to about 5 x<br>
1010 cells. A dose may also about 106 cells/m2 to about 1010 cells/m2, or about 106 cells/m2 to about 108 cells/m2.<br>
[0283] Proteins(s) of the invention, and/or nucleic acids encoding the protein(s), can also be administered via<br>
liposomes, which may also serve to: 1) target the proteins(s) to a particular tissue, such as lymphoid tissue; 2) to target<br>
selectively to diseases cells; or, 3) to increase the half-life of the peptide composition. Liposomes include emulsions,<br>
foams, micelles, insoluble monolayers, liquid crystals, phospholipid dispersions, lamellar iayers and the like. In these<br>
preparations, the peptide to be delivered is incorporated as part of a liposome, alone or in conjunction with a molecule<br>
which binds to a receptor prevalent among lymphoid cells, such as monoclonal antibodies which bind to the CD45 antigen,<br>
or with other therapeutic or immunogenic compositions. Thus, liposomes either filled or decorated with a desired peptide of<br>
the invention can be directed to the site of lymphoid cells, where the liposomes then deliver the peptide compositions.<br>
Liposomes for use in accordance with the invention are formed from standard vesicle-forming lipids, which generally<br>
include neutral and negatively charged phospholipids and a sterol, such as cholesterol. The selection of lipids is generally<br>
guided by consideration of, e.g., liposome size, acid lability and stability of the liposomes in the blood stream. A variety of<br>
methods are available for preparing liposomes, as described in, e.g., Szoka, et al, Ann. Rev. Biophys. Bioeng. 9:467<br>
(1980), and U.S. Patent Nos. 4,235,871,4,501,728,4,837,028, and 5,019,369.<br>
[0284] For targeting cells of the immune system, a ligand to be incorporated into the liposome can include, e.g.,<br>
antibodies or fragments thereof specific for cell surface determinants of the desired immune system cells. A liposome<br>
suspension containing a peptide may be administered intravenously, locally, topically, etc. in a dose which varies according<br>
to, intet alia, the manner of administration, the peptide being delivered, and the stage of the disease being treated.<br>
[0285] For solid compositions, conventional nontoxic solid carriers may be used which include, for example,<br>
pharmaceutical grades of mannitol, lactose, starch, magnesium stearate, sodium saccharin, talcum, cellulose, glucose,<br>
sucrose, magnesium carbonate, and the like. For oral administration, a pharmaceutically acceptable nontoxic composition<br>
is formed by incorporating any of the normally employed excipients, such as those carriers previously listed, and generally<br>
10-95% of active ingredient, that is, one or more peptides of the invention, and more preferably at a concentration of 25%-<br>
75%.<br>
[0286] For aerosol administration, immunogenic peptides are preferably supplied in finely divided form along with a<br>
surfactant and propellant. Typical percentages of peptides are about 0.01 %-20% by weight, preferably about 1%-10%.<br>
The surfactant must, of course, be nontoxic, and preferably soluble in the propellant. Representative of such agents are<br>
the esters or partial esters of fatty acids containing from about 6 to 22 carbon atoms, such as caproic, octanoic, lauric,<br>
palmitic, stearic, linoleic, linolenic, olesteric and oleic acids with an aliphatic polyhydric alcohol or its cyclic anhydride.<br>
Mixed esters, such as mixed or natural glycerides may be employed. The surfactant may constitute about 0.1%-20% by<br>
weight of the composition, preferably about 0.25-5%. The balance of the composition is ordinarily propellant. A carrier can<br>
also be included, as desired, as with, e.g., lecithin for intranasal delivery.<br>
XI.) Diagnostic and Prognostic Embodiments of PSCA.<br>
[0287] As disclosed herein, PSCA polynucleotides, polypeptides, reactive cytotoxic T cells (CTL), reactive helper T<br>
cells (HTL) and anti-polypeptide antibodies are used in well known diagnostic, prognostic and therapeutic assays that<br>
examine conditions associated with dysregulated cell growth such as cancer, in particular the cancers listed in Table I (see,<br>
e.g., both its specific pattern of tissue expression as well as its overexpression in certain cancers as described for example<br>
in the Example entitled "Expression analysis of PSCA in normal tissues, and patient specimens").<br>
[0288] PSCA can be analogized to a prostate associated antigen PSA, the archetypal marker that has been used by<br>
medical practitioners for years to identify and monitor the presence of prostate cancer (see, e.g., Merrill et al., J. Urol.<br>
163(2): 503-5120 (2000); Polascik et al, J. Urol. Aug; 162(2):293-306 (1999) and Fortier et al, J. Nat. Cancer Inst. 91(19):<br>
1635-1640(1999)). A variety of other diagnostic markers are also used in similar contexts including p53 and K-ras (see,<br>
e.g., Tulchinsky et al, Int J Mol Med 1999 Jul 4(1):99-102 and Minimoto et al, Cancer Detect Prev 2000;24(1):1-12).<br>
Therefore, this disclosure of PSCA polynucleotides and polypeptides (as well as PSCA polynucleotide probes and anti-<br>
PSCA antibodies used to identify the presence of these molecules) and their properties allows skilled artisans to utilize<br>
these molecules in methods that are analogous to those used, for example, in a variety of diagnostic assays directed to<br>
examining conditions associated with cancer.<br>
[0289] Typical embodiments of diagnostic methods which utilize the PSCA polynucleotides, polypeptides, reactive T<br>
cells and antibodies are analogous to those methods from well-established diagnostic assays, which employ, e.g., PSA<br>
polynucleotides, polypeptides, reactive T cells and antibodies. For example, just as PSA polynucleotides are used as<br>
probes (for example in Northern analysis, see, e.g., Sharief et al., Biochem. Mol. Biol. Int. 33(3):567-74(1994)) and primers<br>
(for example in PCR analysis, see, e.g., Okegawa et al., J. Urol. 163(4): 1189-1190 (2000)) to observe the presence and/or<br>
the level of PSA mRNAs in methods of monitoring PSA overexpression or the metastasis of prostate cancers, the PSCA<br>
polynucleotides described herein can be utilized in the same way to detect PSCA overexpression or the metastasis of<br>
prostate and other cancers expressing this gene. Alternatively, just as PSA polypeptides are used to generate antibodies<br>
specific for PSA which can then be used to observe the presence and/or the level of PSA proteins in methods to monitor<br>
PSA protein overexpression (see, e.g., Stephan et al., Urology 55(4):560-3 (2000)) or the metastasis of prostate cells (see,<br>
e.g., Alanen et al., Pathol. Res. Pract. 192(3):233-7 (1996)), the PSCA polypeptides described herein can be utilized to<br>
generate antibodies for use in detecting PSCA overexpression or the metastasis of prostate cells and cells of other cancers<br>
expressing this gene.<br>
[0290] Specifically, because metastases involves the movement of cancer cells from an organ of origin (such as the<br>
lung or prostate gland etc.) to a different area of the body (such as a lymph node), assays which examine a biological<br>
sample for the presence of cells expressing PSCA polynucleotides and/or polypeptides can be used to provide evidence of<br>
metastasis. For example, when a biological sample from tissue that does not normally contain PSCA-expressing cells<br>
(lymph node) is found to contain PSCA-expressing cells such as the PSCA expression seen in LAPC4 and LAPC9,<br>
xenografts isolated from lymph node and bone metastasis, respectively, this finding is indicative of metastasis.<br>
[0291] Alternatively PSCA polynucleotides and/or polypeptides can be used to provide evidence of cancer, for<br>
example, when cells in a biological sample that do not normally express PSCA or express PSCA at a different level are<br>
found to express PSCA or have an increased expression of PSCA (see, e.g., the PSCA expression in the cancers listed in<br>
Table I and in patient samples etc. shown in the accompanying Figures). In such assays, artisans may further wish to<br>
generate supplementary evidence of metastasis by testing the biological sample for the presence of a second tissue<br>
restricted marker (in addition to PSCA) such as PSA, PSCA etc. (see, e.g., Alanen et a/., Pathol. Res. Pract. 192(3): 233-<br>
237(1996)).<br>
[0292] The use of immunohistochemistry to identify the presence of a PSCA polypeptide within a tissue section can<br>
indicate an altered state of certain cells within that tissue. It is well understood in the art that the ability of an antibody to<br>
localize to a polypeptide that is expressed in cancer cells is a way of diagnosing presence of disease, disease stage,<br>
progression and/or tumor aggressiveness. Such an antibody can also detect an altered distribution of the polypeptide<br>
within the cancer cells, as compared to corresponding non-malignant tissue.<br>
[0293] The PSCA polypeptide and immunogenic compositions are also useful in view of the phenomena of altered<br>
subcellular protein localization in disease states. Alteration of cells from normal to diseased state causes changes in<br>
cellular morphology and is often associated with changes in subcellular protein localization/distribution. For example, cell<br>
membrane proteins that are expressed in a polarized manner in normal cells can be altered in disease, resulting in<br>
distribution of the protein in a non-polar manner over the whole cell surface.<br>
[0294] The phenomenon of altered subcellular protein localization in a disease state has been demonstrated with<br>
MUC1 and Her2 protein expression by use of immunohistochemical means. Normal epithelial cells have a typical apical<br>
distribution of MUC1, in addition to some supranuclear localization of the glycoprotein, whereas malignant lesions often<br>
demonstrate an apolar staining pattern (Diaz et al, The Breast Journal, 7; 40-45 (2001); Zhang et al, Clinical Cancer<br>
Research, 4; 2669-2676 (1998): Cao, et al, The Journal of Histochemistry and Cytochemistry, 45:1547-1557 (1997)). In<br>
addition, normal breast epithelium is either negative for Her2 protein or exhibits only a basolateral distribution whereas<br>
malignant cells can express the protein over the whole cell surface (De Potter, et al, International Journal of Cancer, 44;<br>
969-974 (1989): McCormick, et al, 117; 935-943 (2002)). Alternatively, distribution of the protein may be altered from a<br>
surface only localization to include diffuse cytoplasmic expression in the diseased state. Such an example can be seen with<br>
MUC1 (Diaz, et al, The Breast Journal, 7:40-45 (2001)).<br>
[0295] Alteration in the localization/distribution of a protein in the cell, as detected by immunohistochemical methods,<br>
can also provide valuable information concerning the favorability of certain treatment modalities. This last point is illustrated<br>
by a situation where a protein may be intracellular in normal tissue, but cell surface in malignant cells; the cell surface<br>
location makes the cells favorably amenable to antibody-based diagnostic and treatment regimens. When such an<br>
alteration of protein localization occurs for PSCA, the PSCA protein and immune responses related thereto are very useful.<br>
Accordingly, the ability to determine whethet alteration of subcellular protein localization occurred for 24P4C12 make the<br>
PSCA protein and immune responses related thereto very useful. Use of the PSCA compositions allows those skilled in the<br>
art to make important diagnostic and therapeutic decisions.<br>
[0296] Immunohistochemical reagents specific to PSCA are also useful to detect metastases of tumors expressing<br>
PSCA when the polypeptide appears in tissues where PSCA is not normally produced.<br>
[0297] Thus, PSCA polypeptides and antibodies resulting from immune responses thereto are useful in a variety of<br>
important contexts such as diagnostic, prognostic, preventative and/or therapeutic purposes known to those skilled in the<br>
art.<br>
[0298] Just as PSA polynucleotide fragments and polynucleotide variants are employed by skilled artisans for use in<br>
methods of monitoring PSA, PSCA polynucleotide fragments and polynucleotide variants are used in an analogous<br>
manner, in particular, typical PSA polynucleotides used in methods of monitoring PSA are probes or primers which consist<br>
of fragments of the PSA cDNA sequence. Illustrating this, primers used to PCR amplify a PSA polynucleotide must include<br>
less than the whole PSA sequence to function in the polymerase chain reaction. In the context of such PCR reactions,<br><br>
skilled artisans generally create a variety of different polynucleotide fragments that can be used as primers in order to<br>
simplify different portions of a polynucleotide of interest or to optimize amplification reactions (see, e.g., Caetano-Anolles, G.<br>
Biotechniques 25(3): 472-476,478-480 (1998); Robertson et al, Methods Mol. Biol. 98:121-154 (1998)). An additional<br>
illustration of the use of such fragments is provided in the Example entitled "Expression analysis of PSCA in normal tissues,<br>
and patient specimens," where a PSCA polynucleotide fragment is used as a probe to show the expression of PSCA RNAs<br>
in cancer cells. In addition, variant polynucleotide sequences are typically used as primers and probes for the<br>
corresponding mRNAs in PCR and Northern analyses (see, e.g., Sawai et al., Fet al Diagn. Ther. 1996 Nov-Dec 11 (6):407-<br>
13 and Current Protocols In Molecular Biology, Volume 2, Unit 2, Frederick M. Ausubel et al. eds., 1995)). Polynucleotide<br>
fragments and variants are useful in this context where they are capable of binding to a target polynucleotide sequence<br>
(e.g., a PSCA polynucleotide shown in Figure 2 or variant thereof) under conditions of high stringency.<br>
[0299] Furthermore, PSA polypeptides which contain an epitope that can be recognized by an antibody or T cell that<br>
specifically binds to that epitope are used in methods of monitoring PSA. PSCA polypeptide fragments and polypeptide<br>
analogs or variants can also be used in an analogous manner. This practice of using polypeptide fragments or polypeptide<br>
variants to generate antibodies (such as anti-PSA antibodies or T cells) is typical in the art with a wide variety of systems<br>
such as fusion proteins being used by practitioners (see, e.g., Current Protocols In Molecular Biology, Volume 2, Unit 16,<br>
Frederick M. Ausubel et al. eds., 1995). In this context, each epitope(s) functions to provide the architecture with which an<br>
antibody or T cell is reactive. Typically, skilled artisans create a variety of different polypeptide fragments that can be used<br>
in order to generate immune responses specific for different portions of a polypeptide of interest (see, e.g., U.S. Patent No.<br>
5,840,501 and U.S. Patent No. 5,939,533). For example it may be preferable to utilize a polypeptide comprising one of the<br>
PSCA biological motifs discussed herein or a motif-bearing subsequence which is readily identified by one of skill in the art<br>
based on motifs available in the art. Polypeptide fragments, variants or analogs are typically useful in this context as long<br>
as they comprise an epitope capable of generating an antibody or T cell specific for a target polypeptide sequence (e.g. a<br>
PSCA polypeptide shown in Figure 3).<br>
[0300] As shown herein, the PSCA polynucleotides and polypeptides (as well as the PSCA polynucleotide probes<br>
and anti-PSCA antibodies or T cells used to identify the presence of these molecules) exhibit specific properties that make<br>
them useful in diagnosing cancers such as those listed in Table I. Diagnostic assays that measure the presence of PSCA<br>
gene products, in order to evaluate the presence or onset of a disease condition described herein, such as prostate cancer,<br>
are used to identify patients for preventive measures or further monitoring, as has been done so successfully with PSA.<br>
Moreover, these materials satisfy a need in the art for molecules having similar or complementary characteristics to PSA in<br>
situations where, for example, a definite diagnosis of metastasis of prostatic origin cannot be made on the basis of a test for<br>
PSA alone (see, e.g., Alanen et al., Pathol. Res. Pract. 192(3): 233-237 (1996)), and consequently, materials such as<br>
PSCA polynucleotides and polypeptides (as well as the PSCA polynucleotide probes and anti-PSCA antibodies used to<br>
identify the presence of these molecules) need to be employed to confirm a metastases of prostatic origin.<br>
[0301] Finally, in addition to their use in diagnostic assays, the PSCA polynucleotides disclosed herein have a<br>
number of other utilities such as their use in the identification of oncogenetic associated chromosomal abnormalities in the<br>
chromosomal region to which the PSCA gene maps (see the Example entitled "Chromosomal Mapping of PSCA" below).<br>
Moreover, in addition to their use in diagnostic assays, the PSCA-related proteins and polynucleotides disclosed herein<br>
have other utilities such as their use in the forensic analysis of tissues of unknown origin (see, e.g., Takahama K Forensic<br>
Sci Int 1996 Jun 28;80(1-2): 63-9).<br>
[0302] Additionally, PSCA-related proteins or polynucleotides of the invention can be used to treat a pathologic<br>
condition characterized by the over-expression of PSCA. For example, the amino acid or nucleic acid sequence of Figure 2<br>
or Figure 3, or fragments of either, can be used to generate an immune response to a PSCA antigen. Antibodies or other<br>
molecules that react with PSCA can be used to modulate the function of this molecule, and thereby provide a therapeutic<br>
benefit.<br>
XII.) Inhibition of PSCA Protein Function<br>
[0303] The invention includes various methods and compositions for inhibiting the binding of PSCA to its binding<br>
partner or its association with other protein(s) as well as methods for inhibiting PSCA function.<br>
XIIA.) Inhibition of PSCA With Intracellular Antibodies<br>
[0304] In one approach, a recombinant vector that encodes single chain antibodies that specifically bind to PSCA<br>
are introduced into PSCA expressing cells via gene transfer technologies. Accordingly, the encoded single chain anti-<br>
PSCA antibody is expressed intracellularly, binds to PSCA protein, and thereby inhibits its function. Methods for<br>
engineering such intracellular single chain antibodies are well known. Such intracellular antibodies, also known as<br>
"intrabodies", are specifically targeted to a particular compartment within the cell, providing control over where the inhibitory<br>
activity of the treatment is focused. This technology has been successfully applied in the art (for review, see Richardson<br>
and Marasco, 1995, TIBTECH vol. 13). Intrabodies have been shown to virtually eliminate the expression of otherwise<br>
abundant cell surface receptors (see, e.g., Richardson et al., 1995, Proc. Natl. Acad. Sci. USA 92:3137-3141; Beerli et a/.,<br>
1994, J. Biol. Chem. 289: 23931-23936; Deshane et al., 1994, Gene Ther. 1:332-337).<br>
[0305] Single chain antibodies comprise the variable domains of the heavy and light chain joined by a flexible linker<br>
polypeptide, and are expressed as a single polypeptide. Optionally, single chain antibodies are expressed as a single<br>
chain variable region fragment joined to the light chain constant region. Well-known intracellular trafficking signals are<br>
engineered into recombinant polynucleotide vectors encoding such single chain antibodies in order to target precisely the<br>
intrabody to the desired intracellular compartment. For example, intrabodies targeted to the endoplasmic reticulum (ER)<br>
are engineered to incorporate a leader peptide and, optionally, a C-terminal ER retention signal, such as the KDEL amino<br>
acid motif. Intrabodies intended to exert activity in the nucleus are engineered to include a nuclear localization signal. Lipid<br>
moieties are joined to intrabodies in order to tether the intrabody to the cytosolic side of the plasma membrane. Intrabodies<br>
can also be targeted to exert function in the cytosol. For example, cytosolic intrabodies are used to sequester factors within<br>
the cytosol, thereby preventing them from being transported to their natural cellular destination.<br>
[0306] In one embodiment, intrabodies are used to capture PSCA in the nucleus, thereby preventing its activity<br>
within the nucleus. Nuclear targeting signals are engineered into such PSCA intrabodies in order to achieve the desired<br>
targeting. Such PSCA intrabodies are designed to bind specifically to a particular PSCA domain. In another embodiment,<br>
cytosolic intrabodies that specifically bind to a PSCA protein are used to prevent PSCA from gaining access to the nucleus,<br>
thereby preventing it from exerting any biological activity within the nucleus (e.g., preventing PSCA from forming<br>
transcription complexes with other factors).<br>
[0307] In order to specifically direct the expression of such intrabodies to particular cells, the transcription of the<br>
intrabody is placed under the regulatory control of an appropriate tumor-specific promoter and/or enhancer. In order to<br>
target intrabody expression specifically to prostate, for example, the PSA promoter and/or promoter/enhancer can be<br>
utilized (See, for example, U.S. Patent No. 5,919,652 issued 6 July 1999).<br><br>
XII.B.) Inhibition of PSCA with Recombinant Proteins<br>
[0308] In another approach, recombinant molecules bind to PSCA and thereby inhibit PSCA function. For example,<br>
these recombinant molecules prevent or inhibit PSCA from accessing/binding to its binding partner(s) or associating with<br>
other protein(s). Such recombinant molecules can, for example, contain the reactive part(s) of a PSCA specific antibody<br>
molecule. In a particular embodiment, the PSCA binding domain of a PSCA binding partner is engineered into a dimeric fusion<br>
protein, whereby the fusion protein comprises two PSCA ligand binding domains linked to the Fc portion of a human IgG, such<br>
as human lgG1. Such IgG portion can contain, for example, the Ch2 and Ch3 domains and the hinge region, but not the Ch1<br>
domain. Such dimeric fusion proteins are administered in soluble form to patients suffering from a cancer associated with the<br>
expression of PSCA, whereby the dimeric fusion protein specifically binds to PSCA and blocks PSCA interaction with a binding<br>
partner. Such dimeric fusion proteins are further combined into multimeric proteins using known antibody linking technologies.<br>
XII.C) Inhibition of PSCA Transcription or Translation<br>
[0309] The present invention also comprises various methods and compositions for inhibiting the transcription of the<br>
PSCA gene. Similarly, the invention also provides methods and compositions for inhibiting the translation of PSCA mRNA<br>
into protein.<br>
[0310] In one approach, a method of inhibiting the transcription of the PSCA gene comprises contacting the PSCA<br>
gene with a PSCA antisense polynucleotide. In another approach, a method of inhibiting PSCA mRNA translation<br>
comprises contacting a PSCA mRNA with an antisense polynucleotide. In another approach, a PSCA specific ribozyme is<br>
used to cleave a PSCA message, thereby inhibiting translation. Such antisense and ribozyme based methods can also be<br>
directed to the regulatory regions of the PSCA gene, such as PSCA promoter and/or enhancer elements. Similarly,<br>
proteins capable of inhibiting a PSCA gene transcription factor are used to inhibit PSCA mRNA transcription. The various<br>
polynucleotides and compositions useful in the aforementioned methods have been described above. The use of antisense<br>
and ribozyme molecules to inhibit transcription and translation is well known in the art.<br>
[0311 ] Other factors that inhibit the transcription of PSCA by interfering with PSCA transcriptional activation are also<br>
useful to treat cancers expressing PSCA. Similarly, factors that interfere with PSCA processing are useful to treat cancers<br>
that express PSCA. Cancer treatment methods utilizing such factors are also within the scope of the invention.<br>
XII.D.) General Considerations for Therapeutic Strategies<br>
[0312] Gene transfer and gene therapy technologies can be used to deliver therapeutic polynucleotide molecules to<br>
tumor cells synthesizing PSCA (i.e., antisense, ribozyme, polynucleotides encoding intrabodies and other PSCA inhibitory<br>
molecules). A number of gene therapy approaches are known in the art. Recombinant vectors encoding PSCA antisense<br>
polynucleotides, ribozymes, factors capable of interfering with PSCA transcription, and so forth, can be delivered to target<br>
tumor cells using such gene therapy approaches.<br>
[0313] The above therapeutic approaches can be combined with any one of a wide variety of surgical, chemotherapy<br>
or radiation therapy regimens. The therapeutic approaches of the invention can enable the use of reduced dosages of<br>
chemotherapy (or other therapies) and/or less frequent administration, an advantage for all patients and particularly for<br>
those that do not tolerate the toxicity of the chemotherapeutic agent well.<br>
[0314] The anti-tumor activity of a particular composition (e.g., antisense, ribozyme, intrabody), or a combination of<br>
such compositions, can be evaluated using various in vitro and in vivo assay systems. In vitro assays that evaluate<br>
therapeutic activity include cell growth assays, soft agar assays and other assays indicative of tumor promoting activity,<br>
bleeding assays capable of determining the extent to which a therapeutic composition will inhibit the binding of PSCA to a<br>
binding partner, etc.<br>
[0315] In vivo, the effect of a PSCA therapeutic composition can be evaluated in a suitable animal model. For<br>
example, xenogenic prostate cancer models can be used, wherein human prostate cancer explants or passaged xenograft<br>
tissues are introduced into immune compromised animals, such as nude or SCID mice (Klein et al., 1997, Nature Medicine<br>
3. 402-408). For example, PCT Patent Application W098/16628 and U.S. Patent 6,107,540 describe various xenograft<br>
models of human prostate cancer capable of recapitulating the development of primary tumors, micrometastasis, and the<br>
formation of osteoblastic metastases characteristic of late stage disease. Efficacy can be predicted using assays that<br>
pressure combition of tumor formation, tumor regression or metastasis, and the like.<br>
[0316] In vivo assays that evaluate the promotion of apoptosis are useful in evaluating therapeutic compositions. In<br>
one embodiment, xenografts from tumor bearing mice treated with the therapeutic composition can be examined for the<br>
presser of apoptotic foci and compared to untreated control xenograft-bearing mice. The extent to which apoptotic foci<br>
are ong in the tumors of the treated mice provides an indication of the therapeutic efficacy of the composition.<br>
[0317] The therapeutic compositions used in the practice of the foregoing methods can be formulated into<br>
pharmaceutical compositions comprising a carrier suitable for the desired delivery method. Suitable carriers include any<br>
material that when combined with the therapeutic composition ret alns the anti-tumor function of the therapeutic composition<br>
and is generally non-reactive with the patient's immune system. Examples include, but are not limited to, any of a number<br>
of standard pharmaceutical carriers such as sterile phosphate buffered saline solutions, bacteriostatic water, and the like<br>
(see, generally, Remington's Pharmaceutical Sciences 16th Edition, A. Osal., Ed., 1980).<br>
[0318] Therapeutic formulations can be solubilized and administered via any route capable of delivering the<br>
therapeutic composition to the tumor site. Potentially effective routes of administration include, but are not limited to,<br>
intravenous, parenteral, intraperitoneal, intramuscular, intratumor, intradermal, intraorgan, orthotopic, and the like. A<br>
preferred formulation for intravenous injection comprises the therapeutic composition in a solution of preserved<br>
bacteriostatic water, sterile unpreserved water, and/or diluted in polyvinylchloride or polyethylene bags containing 0.9%<br>
sterile Sodium Chloride for Injection, USP. Therapeutic protein preparations can be lyophilized and stored as sterile<br>
powders, preferably under vacuum, and then reconstituted in bacteriostatic water (containing for example, benzyl alcohol<br>
preservative) or in sterile water prior to injection.<br>
[0319] Dosages and administration protocols for the treatment of cancers using the foregoing methods will vary with<br>
the method and the target cancer, and will generally depend on a number of other factors appreciated in the art.<br>
XIII.) Identification, Characterization and Use of Modulators of PSCA<br>
Methods to Identify and Use Modulators<br>
[0320] In one embodiment, screening is performed to identify modulators that induce or suppress a particular<br>
expression profile, suppress or induce specific pathways, preferably generating the associated phenotype thereby. In<br>
another embodiment, having identified differentially expressed genes important in a particular state; screens are performed<br>
to identify modulators that alter expression of individual genes, either increase or decrease. In another embodiment,<br>
screening is performed to identify modulators that alter a biological function of the expression product of a differentially<br>
expressed gene. Again, having identified the importance of a gene in a particular state, screens are performed to identify<br>
coments that bind and/or modulate the biological activity of the gene product.<br>
[0321] In addition, screens are done for genes that are induced in response to a candidate agent. After identifying a<br>
modulator (one that suppresses a cancer expression pattern leading to a normal expression pattern, or a modulator of a<br>
cancer gene that leads to expression of the gene as in normal tissue) a screen is performed to identify genes that are<br>
specifically modulated in response to the agent. Comparing expression profiles between normal tissue and agent-treated<br>
cancer tissue reveals genes that are not expressed in normal tissue or cancer tissue, but are expressed in agent treated<br>
tissue, and vice versa. These agent-specific sequences are identified and used by methods described herein for cancer<br>
genes or proteins. In particular these sequences and the proteins they encode are used in marking or identifying agent-<br>
treated cells. In addition, antibodies are raised against the agent-induced proteins and used to target novel therapeutics to<br>
the treated cancer tissue sample.<br>
Modulator-related Identification and Screening Assays:<br>
Gene Expression-related Assays<br>
[0322] Proteins, nucleic acids, and antibodies of the invention are used in screening assays. The cancer-associated<br>
proteins, antibodies, nucleic acids, modified proteins and cells containing these sequences are used in screening assays,<br>
such as evaluating the effect of drug candidates on a "gene expression profile," expression profile of polypeptides or<br>
alteration of biological function. In one embodiment, the expression profiles are used, preferably in conjunction with high<br>
throughput screening techniques to allow monitoring for expression profile genes after treatment with a candidate agent<br>
(e.g., Davis, GF, et al, J Biol Screen 7:69 (2002); Zlokarnik, et al., Science 279:84-8 (1998); Heid, Genome Res 6:986-<br>
94,1996).<br>
[0323] The cancer proteins, antibodies, nucleic acids, modified proteins and cells containing the native or modified<br>
cancer proteins or genes are used in screening assays. That is, the present invention comprises methods for screening for<br>
compositions which modulate the cancer phenotype or a physiological function of a cancer protein of the invention. This is<br>
done on a gene itself or by evaluating the effect of drug candidates on a "gene expression profile" or biological function. In<br>
one embodiment, expression profiles are used, preferably in conjunction with high throughput screening techniques to allow<br>
monitoring after treatment with a candidate agent, see Zlokarnik, supra.<br>
[0324] A variety of assays are executed directed to the genes and proteins of the invention. Assays are run on an<br>
individual nucleic acid or protein level. That is, having identified a particular gene as up regulated in cancer, test<br>
compounds are screened for the ability to modulate gene expression or for binding to the cancer protein of the invention.<br>
"Modulation" in this context includes an increase or a decrease in gene expression. The preferred amount of modulation<br>
will depend on the original change of the gene expression in normal versus tissue undergoing cancer, with changes of at<br>
least 10%, preferably 50%, more preferably 100-300%, and in some embodiments 300-1000% or greater. Thus, if a gene<br>
exhibits a 4-fold increase in cancer tissue compared to normal tissue, a decrease of about four-fold is often desired;<br>
similarly, a 10-fold decrease in cancer tissue compared to normal tissue a target value of a 10-fold increase in expression<br>
by the test compound is often desired. Modulators that exacerbate the type of gene expression seen in cancer are also<br>
useful, e.g., as an upregulated target in further analyses.<br>
[0325] The amount of gene expression is monitored using nucleic acid probes and the quantification of gene<br>
expression levels, or, alternatively, a gene product itself is monitored, e.g., through the use of antibodies to the cancer<br>
protein and standard immunoassays. Proteomics and separation techniques also allow for quantification of expression.<br>
Expression Monitoring to Identify Compounds that Modify Gene Expression<br>
[0326] In one embodiment, gene expression monitoring, i.e., an expression profile, is monitored simultaneously for a<br>
number of entities. Such profiles will typically involve one or more of the genes of Figure 2. In this embodiment, e.g.,<br>
cancer nucleic acid probes are attached to biochips to detect and quantify cancer sequences in a particular cell.<br>
Alternatively, PCR can be used. Thus, a series, e.g., wells of a microtiter plate, can be used with dispensed primers in<br>
desired wells. A PCR reaction can then be performed and analyzed for each well.<br>
[0327] Expression monitoring is performed to identify compounds that modify the expression of one or more cancer-<br>
associated sequences, e.g., a polynucleotide sequence set out in Figure 2. Generally, a test modulator is added to the<br>
cells prior to analysis. Moreover, screens are also provided to identify agents that modulate cancer, modulate cancer<br>
proteins of the invention, bind to a cancer protein of the invention, or interfere with the binding of a cancer protein of the<br>
invention and an antibody or other binding partner.<br>
[0328] In one embodiment, high throughput screening methods involve providing a library containing a large number<br>
of potential therapeutic compounds (candidate compounds). Such "combinatorial chemical libraries" are then screened in<br>
one or more assays to identify those library members (particular chemical species or subclasses) that display a desired<br>
characteristic activity. The compounds thus identified can serve as conventional "lead compounds," as compounds for<br>
screening, or as therapeutics.<br>
[0329] In certain embodiments, combinatorial libraries of potential modulators are screened for an ability to bind to a<br>
cancer polypeptide or to modulate activity. Conventionally, new chemical entities with useful properties are generated by<br>
identifying a chemical-compound (called a "lead compound") with some desirable property or activity, e.g., inhibiting activity,<br>
creating variants of the lead compound, and evaluating the property and activity of those variant compounds. Often, high<br>
throughput screening (HTS) methods are employed for such an analysis.<br>
[0330] As noted above, gene expression monitoring is conveniently used to test candidate modulators (e.g., protein,<br>
nucleic acid or small molecule). After the candidate agent has been added and the cells allowed to incubate for a period,<br>
the sample containing a target sequence to be analyzed is, e.g., added to a biochip.<br>
[0331] If required, the target sequence is prepared using known techniques. For example, a sample is treated to<br>
lyse the cells, using known lysis buffers, electroporation, etc., with purification and/or amplification such as PCR performed<br>
as appropriate. For example, an in vitro transcription with labels covalently attached to the nucleotides is performed.<br>
Generally, the nucleic acids are labeled with biotin-FITC or PE, or with cy3 or cy5.<br>
[0332] The target sequence can be labeled with, e.g., a fluorescent, a chemiluminescent, a chemical, or a<br>
radioactive signal, to provide a means of detecting the target sequence's specific binding to a probe. The label also can be<br>
an enzyme, such as alkaline phosphatase or horseradish peroxidase, which when provided with an appropriate substrate<br>
produces a product that is detected. Alternatively, the label is a labeled compound or small molecule, such as an enzyme<br>
inhibitor, that binds but is not catalyzed or altered by the enzyme. The label also can be a moiety or compound, such as,<br>
an epitope tag or biotin which specifically binds to streptavidin. For the example of biotin, the streptavidin is labeled as<br>
described above, thereby, providing a detectable signal for the bound target sequence. Unbound labeled streptavidin is<br>
typically removed prior to analysis.<br><br>
[0333] As will be appreciated by those in the art, these assays can be direct hybridization assays or can comprise<br>
andwich assays", which include the use of multiple probes, as is generally outlined in U.S. Patent Nos. 5, 681,702;<br>
5,597,909; 5,545,730; 5,594,117; 5,591,584; 5,571,670; 5,580,731; 5,571,670; 5,591,584; 5,624,802; 5,635,352;<br>
5,594,118; 5,359,100; 5,124, 246; and 5,681,697. In this embodiment, in general, the target nucleic acid is prepared as<br>
outlined above, and then added to the biochip comprising a plurality of nucleic acid probes, under conditions that allow the<br>
formation of a hybridization complex.<br>
[0334] A variety of hybridization conditions are used in the present invention, including high, moderate and low<br>
stringency conditions as outlined above. The assays are generally run under stringency conditions which allow formation of<br>
the label probe hybridization complex only in the presence of target. Stringency can be controlled by altering a step<br>
parameter that is a thermodynamic variable, including, but not limited to, temperature, formamide concentration, salt<br>
concentration, chaotropic salt concentration pH, organic solvent concentration, etc. These parameters may also be used to<br>
control non-specific binding, as is generally outlined in U.S. Patent No. 5,681,697. Thus, it can be desirable to perform<br>
certain steps at higher stringency conditions to reduce non-specific binding.<br>
[0335] The reactions outlined herein can be accomplished in a variety of ways. Components of the reaction can be<br>
added simultaneously, or sequentially, in different orders, with preferred embodiments outlined below. In addition, the<br>
reaction may include a variety of other reagents. These include salts, buffers, neutral proteins, e.g. albumin, detergents,<br>
etc. which can be used to facilitate optimal hybridization and detection, and/or reduce nonspecific or background<br>
interactions. Reagents that otherwise improve the efficiency of the assay, such as protease inhibitors, nuclease inhibitors,<br>
anti-microbial agents, etc., may also be used as appropriate, depending on the sample preparation methods and purity of<br>
the target. The assay data are analyzed to determine the expression levels of individual genes, and changes in expression<br>
levels as between states, forming a gene expression profile.<br>
Biological Activity-related Assays<br>
[0336] The invention provides methods identify or screen for a compound that modulates the activity of a cancer-<br>
related gene or protein of the invention. The methods comprise adding a test compound, as defined above, to a cell<br>
comprising a cancer protein of the invention. The cells contain a recombinant nucleic acid that encodes a cancer protein of<br>
the invention. In another embodiment, a library of candidate agents is tested on a plurality of cells.<br>
[0337] In one aspect, the assays are evaluated in the presence or absence or previous or subsequent exposure of<br>
physiological signals, e.g. hormones, antibodies, peptides, antigens, cytokines, growth factors, action potentials,<br>
pharmacological agents including chemotherapeutics, radiation, carcinogenics, or other cells (i.e., cell-cell contacts). In<br>
another example, the determinations are made at different stages of the cell cycle process. In this way, compounds that<br>
modulate genes or proteins of the invention are identified. Compounds with pharmacological activity are able to enhance or<br>
interfere with the activity of the cancer protein of the invention. Once identified, similar structures are evaluated to identify<br>
critical structural features of the compound.<br>
[0338] In one embodiment, a method of modulating (e.g., inhibiting) cancer cell division is provided; the method<br>
comprises administration of a cancer modulator. In another embodiment, a method of modulating (e.g., inhibiting) cancer<br>
is provided; the method comprises administration of a cancer modulator. In a further embodiment, methods of treating cells<br>
or individuals with cancer are provided; the method comprises administration of a cancer modulator.<br>
[0339] In one embodiment, a method for modulating the status of a cell that expresses a gene of the invention is<br>
provided. As used herein status comprises such art-accepted parameters such as growth, proliferation, survival, function,<br><br>
apoptosis, senescence, location, enzymatic activity, signal transduction, etc. of a cell. In one embodiment, a cancer<br>
inhibitor is an antibody as discussed above. In another embodiment, the cancer inhibitor is an antisense molecule. A<br>
variety of cell growth, proliferation, and metastasis assays are known to those of skill in the art, as described herein.<br>
High Throughput Screening to Identify Modulators<br>
[0340] The assays to identify suitable modulators are amenable to high throughput screening. Preferred assays<br>
thus detect enhancement or inhibition of cancer gene transcription, inhibition or enhancement of polypeptide expression,<br>
and inhibition or enhancement of polypeptide activity.<br>
[0341] In one embodiment, modulators evaluated in high throughput screening methods are proteins, often naturally<br>
occurring proteins or fragments of naturally occurring proteins. Thus, e.g., cellular extracts containing proteins, or random<br>
or directed digests of proteinaceous cellular extracts, are used. In this way, libraries of proteins are made for screening in<br>
the methods of the invention. Particularly preferred in this embodiment are libraries of bacterial, fungal, viral, and<br>
mammalian proteins, with the latter being preferred, and human proteins being especially preferred. Particularly useful test<br>
compound will be directed to the class of proteins to which the target belongs, e.g., substrates for enzymes, or ligands and<br>
receptors.<br>
Use of Soft Agar Growth and Colony Formation to Identify and Characterize Modulators<br>
[0342] Normal cells require a solid substrate to attach and grow. When cells are transformed, they lose this<br>
phenotype and grow detached from the substrate. For example, transformed cells can grow in stirred suspension culture or<br>
suspended in semi-solid media, such as semi-solid or soft agar. The transformed cells, when transfected with tumor<br>
suppressor genes, can regenerate normal phenotype and once again require a solid substrate to attach to and grow. Soft<br>
agar growth or colony formation in assays are used to identify modulators of cancer sequences, which when expressed in<br>
host cells, inhibit abnormal cellular proliferation and transformation. A modulator reduces or eliminates the host ceils' ability<br>
to grow suspended in solid or semisolid media, such as agar.<br>
[0343] Techniques for soft agar growth or colony formation in suspension assays are described in Freshney, Culture<br>
of Animal Cells a Manual of Basic Technique (3rd ed., 1994). See also, the methods section of Garkavtsev et al. (1996),<br>
supra.<br>
Evaluation of Contact Inhibition and Growth Density Limitation to Identify and Characterize<br>
Modulators<br>
[0344] Normal cells typically grow in a flat and organized pattern in cell culture until they touch other cells. When the<br>
cells touch one another, they are contact inhibited and stop growing. Transformed cells, however, are not contact inhibited<br>
and continue to grow to high densities in disorganized foci. Thus, transformed cells grow to a higher saturation density than<br>
corresponding normal cells. This is detected morphologically by the formation of a disoriented monolayer of cells or cells in<br>
foci. Alternatively, labeling index with (3H)-thymidine at saturation density is used to measure density limitation of growth,<br>
similarly an MTT or Alamar blue assay will reveal proliferation capacity of cells and the the ability of modulators to affect<br>
same. See Freshney (1994), supra. Transformed cells, when transfected with tumor suppressor genes, can regenerate a<br>
normal phenotype and become contact inhibited and would grow to a lower density.<br>
[0345] In this assay, labeling index with 3H)-thymidine at saturation density is a preferred method of measuring<br>
density limitation of growth. Transformed host cells are transfected with a cancer-associated sequence and are grown for<br><br>
24 hours at saturation density in non-limiting medium conditions. The percentage of cells labeling with (3H)-thymidine is<br>
ditermined by incorporated cpm.<br>
[0346] Contact independent growth is used to identify modulators of cancer sequences, which had led to abnormal<br>
cellular proliferation and transformation. A modulator reduces or eliminates contact independent growth, and returns the<br>
cells to a normal phenotype.<br>
Evaluation of Growth Factor or Serum Dependence to Identify and Characterize Modulators<br>
[0347] Transformed cells have lower serum dependence than their normal counterparts (see, e.g., Temin, J. Natl.<br>
Cancer Inst. 37:167-175 (1966); Eagle et al., J. Exp. Med 131:836-879 (1970)); Freshney, supra. This is in part due to<br>
release of various growth factors by the transformed cells. The degree of growth factor or serum dependence of<br>
transformed host cells can be compared with that of control. For example, growth factor or serum dependence of a cell is<br>
monitored in methods to identify and characterize compounds that modulate cancer-associated sequences of the invention.<br>
Use of Tumor-specific Marker Levels to Identify and Characterize Modulators<br>
[0348] Tumor cells release an increased amount of certain factors (hereinafter "tumor specific markers") than their<br>
normal counterparts. For example, plasminogen activator (PA) is released from human glioma at a higher level than from<br>
normal brain cells (see, e.g., Gullino, Angiogenesis, Tumor Vascularization, and Potential Interference with Tumor Growth,<br>
in Biological Responses in Cancer, pp. 178-184 (Mihich (ed.) 1985)). Similarly, Tumor Angiogenesis Factor (TAF) is<br>
released at a higher level in tumor cells than their normal counterparts. See, e.g., Folkman, Angiogenesis and Cancer,<br>
Sem Cancer Biol. (1992)), while bFGF is released from endothelial tumors (Ensoli, B et al).<br>
[0349] Various techniques which measure the release of these factors are described in Freshney (1994), supra.<br>
Also, see, Unkless et al., J. Biol. Chem. 249:4295-4305 (1974); Strickland &amp; Beers, J. Biol. Chem. 251:5694-5702 (1976);<br>
Whur et al., Br. J. Cancer 42:305 312 (1980); Gullino, Angiogenesis, Tumor Vascularization, and Potential Interference with<br>
Tumor Growth, in Biological Responses in Cancer, pp. 178-184 (Mihich (ed.) 1985); Freshney, Anticancer Res. 5:111-130<br>
(1985). For example, tumor specific marker levels are monitored in methods to identify and characterize compounds that<br>
modulate cancer-associated sequences of the invention.<br>
Invasiveness into Matriqel to Identify and Characterize Modulators<br>
[0350] The degree of invasiveness into Matrigel or an extracellular matrix constituent can be used as an assay to<br>
identify and characterize compounds that modulate cancer associated sequences. Tumor cells exhibit a positive<br>
correlation between malignancy and invasiveness of cells into Matrigel or some other extracellular matrix constituent. In<br>
this assay, tumorigenic cells are typically used as host cells. Expression of a tumor suppressor gene in these host cells<br>
would decrease invasiveness of the host cells. Techniques described in Cancer Res. 1999; 59:6010; Freshney (1994),<br>
supra, can be used. Briefly, the level of invasion of host cells is measured by using filters coated with Matrigel or some<br>
other extracellular matrix constituent. Penetration into the gel, or through to the distal side of the filter, is rated as<br>
invasiveness, and rated histologically by number of cells and distance moved, or by prelabeling the cells with 1251 and<br>
counting the radioactivity on the distal side of the filter or bottom of the dish. See, e.g., Freshney (1984), supra.<br>
Evaluation of Tumor Growth In Vivo to Identify and Characterize Modulators<br>
[0351] Effects of cancer-associated sequences on cell growth are tested in transgenic or immune-suppressed<br>
organisms. Transgenic organisms are prepared in a variety of art-accepted ways. For example, knock-out transgenic<br>
organisms, e.g., mammals such as mice, are made, in which a cancer gene is disrupted or in which a cancer gene is<br>
inserted. Knock-out transgenic mice are made by insertion of a marker gene or other heterologous gene into the<br>
endogenous cancer gene site in the mouse genome via homologous recombination. Such mice can also be made by<br>
substituting the endogenous cancer gene with a mutated version of the cancer gene, or by mutating the endogenous<br>
cancer gene, e.g., by exposure to carcinogens.<br>
[0352] To prepare transgenic chimeric animals, e.g., mice, a DNA construct is introduced into the nuclei of<br>
embryonic stem cells. Cells containing the newly engineered genetic lesion are injected into a host mouse embryo, which<br>
is re-implanted into a recipient female. Some of these embryos develop into chimeric mice that possess germ cells some of<br>
which are derived from the mutant cell line. Therefore, by breeding the chimeric mice it is possible to obtain a new line of<br>
mice containing the introduced genetic lesion (see, e.g., Capecchi et al., Science 244:1288 (1989)). Chimeric mice can be<br>
derived according to US Patent 6,365,797, issued 2 April 2002; US Patent 6,107,540 issued 22 August 2000; Hogan et al.,<br>
Manipulating the Mouse Embryo: A laboratory Manual, Cold Spring Harbor Laboratory (1988) and Teratocarcinomas and<br>
Embryonic Stem Cells: A Practical Approach, Robertson, ed., IRL Press, Washington, D.C., (1987).<br>
[0353] Alternatively, various immune-suppressed or immune-deficient host animals can be used. For example, a<br>
genetically athymic "nude" mouse (see, e.g., Giovanelia et al., J. Natl. Cancer Inst. 52:921 (1974)), a SCID mouse, a<br>
thymectornized mouse, or an irradiated mouse (see, e.g., Bradley et al., Br. J. Cancer 38:263 (1978); Selby et al., Br. J.<br>
Cancer 41:52 (1980)) can be used as a host. Transplantable tumor cells (typically about 106 cells) injected into isogenic<br>
hosts produce invasive tumors in a high proportion of cases, while normal cells of similar origin will not. In hosts which<br>
developed invasive tumors, cells expressing cancer-associated sequences are injected subcutaneously or orthotopically.<br>
Mice are then separated into groups, including control groups and treated experimental groups) e.g. treated with a<br>
modulator). After a suitable length of time, preferably 4-8 weeks, tumor growth is measured (e.g., by volume or by its two<br>
largest dimensions, or weight) and compared to the control. Tumors that have statistically significant reduction (using, e.g.,<br>
Student's T test) are said to have inhibited growth.<br>
In Vitro Assays to Identify and Characterize Modulators<br>
[0354] Assays to identify compounds with modulating activity can be performed in vitro. For example, a cancer<br>
polypeptide is first contacted with a potential modulator and incubated for a suitable amount of time, e.g., from 0.5 to 48<br>
hours. In one embodiment, the cancer polypeptide levels are determined in vitro by measuring the level of protein or<br>
mRNA. The level of protein is measured using immunoassays such as Western blotting, ELISA and the like with an<br>
antibody that selectively binds to the cancer polypeptide or a fragment thereof. For measurement of mRNA, amplification,<br>
e.g., using PCR, LCR, or hybridization assays, e. g. Northern hybridization, RNAse protection, dot blotting, are preferred.<br>
The level of protein or mRNA is detected using directly or indirectly labeled detection agents, e.g., fluorescently or<br>
radioactively labeled nucleic acids, radioactively or enzymatically labeled antibodies, and the like, as described herein.<br>
[0355] Alternatively, a reporter gene system can be devised using a cancer protein promoter operably linked to a<br>
reporter gene such as luciferase, green fluorescent protein, CAT, or P-gal. The reporter construct is typically transfected<br>
into a cell. After treatment with a potential modulator, the amount of reporter gene transcription, translation, or activity is<br>
measured according to standard techniques known to those of skill in the art (Davis GF, supra; Gonzalez, J. &amp; Negulescu,<br>
JT. Curr. Opin. Biotechnol, 1998: 9:624).<br>
[0356] As outlined above, in vitro screens are done on individual genes and gene products. That is, having identified<br>
a particular differentially expressed gene as important in a particular state, screening of modulators of the expression of the<br>
gene or the gene product itself is performed.<br>
[0357] In one embodiment, screening for modulators of expression of specific gene(s) is performed. Typically, the<br>
expression of only one or a few genes is evaluated. In another embodiment, screens are designed to first find compounds<br>
that bind to differentially expressed proteins. These compounds are then evaluated for the ability to modulate differentially<br>
expressed activity. Moreover, once initial candidate compounds are identified, variants can be further screened to better<br>
evaluate structure activity relationships.<br>
Binding Assays to Identify and Characterize Modulators<br>
[0358] In binding assays in accordance with the invention, a purified or isolated gene product of the invention is<br>
generally used. For example, antibodies are generated to a protein of the invention, and immunoassays are run to<br>
determine the amount and/or location of protein. Alternatively, cells comprising the cancer proteins are used in the assays.<br>
[0359] Thus, the methods comprise combining a cancer protein of the invention and a candidate compound such as<br>
a Iigand, and determining the binding of the compound to the cancer protein of the invention. Preferred embodiments<br>
utilize the human cancer protein; animal models of human disease of can also be developed and used. Also, other<br>
analogous mammalian proteins also can be used as appreciated by those of skill in the art. Moreover, in some<br>
embodiments variant or derivative cancer proteins are used.<br>
[0360] Generally, the cancer protein of the invention, orthe'ligand, is non-diffusibly bound to an insoluble support.<br>
The support can, e.g., be one having isolated sample receiving areas (a microtiter plate, an array, etc.). The insoluble<br>
supports can be made of any composition to which the compositions can be bound, is readily separated from soluble<br>
material, and is otherwise compatible with the overall method of screening. The surface of such supports can be solid or<br>
porous and of any convenient shape.<br>
[0361] Examples of suitable insoluble supports include microtiter plates, arrays, membranes and beads. These are<br>
typically made of glass, plastic (e.g., polystyrene), polysaccharide, nylon, nitrocellulose, or Teflon™, etc. Microtiter plates<br>
and arrays are especially convenient because a large number of assays can be carried out simultaneously, using small<br>
amounts of reagents and samples. The particular manner of binding of the composition to the support is not crucial so long<br>
as it is compatible with the reagents and overall methods of the invention, maintains the activity of the composition and is<br>
nondiffusable. Preferred methods of binding include the use of antibodies which do not sterically block either the Iigand<br>
binding site or activation sequence when attaching the protein to the support, direct binding to "sticky" or ionic supports,<br>
chemical crosslinking, the synthesis of the protein or agent on the surface, etc. Following binding of the protein or<br>
ligand/binding agent to the support, excess unbound material is removed by washing. The sample receiving areas may<br>
then be blocked through incubation with bovine serum albumin (BSA), casein or other innocuous protein or other moiety.<br>
[0362] Once a cancer protein of the invention is bound to the support, and a test compound is added to the assay.<br>
Alternatively, the candidate binding agent is bound to the support and the cancer protein of the invention is then added.<br>
Binding agents include specific antibodies, non-natural binding agents identified in screens of chemical libraries, peptide<br>
analogs, etc.<br>
[0363] Of particular interest are assays to identify agents that have a low toxicity for human cells. A wide variety of<br>
assays can be used for this purpose, including proliferation assays, cAMP assays, labeled in vitro protein-protein binding<br>
assays, electrophoretic mobility shift assays, immunoassays for protein binding, functional assays (phosphorylation assays,<br>
etc.) and the like.<br>
[0364] A determination of binding of the test compound (ligand, binding agent, modulator, etc.) to a cancer protein of<br>
the invention can be done in a number of ways. The test compound can be labeled, and binding determined directly, e.g.,<br>
by attaching all or a portion of the cancer protein of the invention to a solid support, adding a labeled candidate compound<br>
(e.g., a fluorescent label), washing off excess reagent, and determining whether the label is present on the solid support.<br>
Various blocking and washing steps can be utilized as appropriate.<br>
[0365] In certain embodiments, only one of the components is labeled, e.g., a protein of the invention or ligands<br>
labeled. Alternatively, more than one component is labeled with different labels, e.g., I125, for the proteins and a fluorophor<br>
for the compound. Proximity reagents, e.g., quenching or energy transfer reagents are also useful.<br>
Competitive Binding to Identify and Characterize Modulators<br>
[0366] In one embodiment, the binding of the "test compound" is determined by competitive binding assay with a<br>
"competitor." The competitor is a binding moiety that binds to the target molecule (e.g., a cancer protein of the invention).<br>
Competitors include compounds such as antibodies, peptides, binding partners, ligands, etc. Under certain circumstances,<br>
the competitive binding between the test compound and the competitor displaces the test compound. In one embodiment,<br>
the test compound is labeled. Either the test compound, the competitor, or both, is added to the protein for a time sufficient<br>
to allow binding. Incubations are performed at a temperature that facilitates optimal activity, typically between four and<br>
40°C. Incubation periods are typically optimized, e.g., to facilitate rapid high throughput screening; typically between zero<br>
and one hour will be sufficient. Excess reagent is generally removed or washed away. The second component is then<br>
added, and the presence or absence of the labeled component is followed, to indicate binding.<br>
[0367] In one embodiment, the competitor is added first, followed by the test compound. Displacement of the<br>
competitor is an indication that the test compound is binding to the cancer protein and thus is capable of binding to, and<br>
potentially modulating, the activity of the cancer protein. In this embodiment, either component can be labeled. Thus, e.g.,<br>
if the competitor is labeled, the presence of label in the post-test compound wash solution indicates displacement by the<br>
test compound. Alternatively, if the test compound is labeled, the presence of the label on the support indicates<br>
displacement.<br>
[0368] in an alternative embodiment, the test compound is added first, with incubation and washing, followed by the<br>
competitor. The absence of binding by the competitor indicates that the test compound binds to the cancer protein with<br>
higher affinity than the competitor. Thus, if the test compound is labeled, fhe presence of the label on the support, coupled<br>
with a lack of competitor binding, indicates that the test compound binds to and thus potentially modulates the cancer<br>
protein of the invention.<br>
[0369] Accordingly, the competitive binding methods comprise differential screening to identity agents that are<br>
capable of modulating the activity of the cancer proteins of the invention. In this embodiment, the methods comprise<br>
combining a cancer protein and a competitor in a first sample. A second sample comprises a test compound, the cancer<br>
protein, and a competitor. The binding of the competitor is determined for both samples, and a change, or difference in<br>
binding between the two samples indicates the presence of an agent capable of binding to the cancer protein and<br>
potentially modulating its activity. That is, if the binding of the competitor is different in the second sample relative to the<br>
sample, the agent is capable of binding to the cancer protein.<br>
[0370] Alternatively, differential screening is used to identify drug candidates that bind to the native cancer protein,<br>
but cannot bind to modified cancer proteins. For example the structure of the cancer protein is modeled and used in<br>
rational drug design to synthesize agents that interact with that site, agents which generally do not bind to site-modified<br>
proteins. Moreover, such drug candidates that affect the activity of a native cancer protein are also identified by screening<br>
drugs for the ability to either enhance or reduce the activity of such proteins.<br>
[0371] Positive controls and negative controls can be used in the assays. Preferably control and test samples are<br>
performed in at least triplicate to obtain statistically significant results. Incubation of all samples occurs for a time sufficient<br>
to allow for the binding of the agent to the protein. Following incubation, samples are washed free of non-specifically bound<br>
material and the amount of bound, generally labeled agent determined. For example, where a radiolabel is employed, the<br>
samples can be counted in a scintillation counter to determine the amount of bound compound.<br>
[0372] A variety of other reagents can be included in the screening assays. These include reagents like salts,<br>
neutral proteins, e.g. albumin, detergents, etc. which are used to facilitate optimal protein-protein binding and/or reduce<br>
non-specific or background interactions. Also reagents that otherwise improve the efficiency of the assay, such as protease<br>
inhibitors, nuclease inhibitors, anti-microbial agents, etc., can be used. The mixture of components is added in an order<br>
that provides for the requisite binding.<br>
Use of Polynucleotides to Down-regulate or Inhibit a Protein of the Invention.<br>
[0373] Polynucleotide modulators of cancer can be introduced into a cell containing the target nucleotide sequence<br>
by formation of a conjugate with a ligand-binding molecule, as described in WO 91/04753. Suitable ligand-binding<br>
molecules include, but are not limited to, cell surface receptors, growth factors, other cytokines, or other ligands that bind to<br>
cell surface receptors. Preferably, conjugation of the ligand binding molecule does not substantially interfere with the ability<br>
of the ligand binding molecule to bind to its corresponding molecule or receptor, or block entry of the sense or antisense<br>
oligonucleotide or its conjugated version into the cell. Alternatively, a polynucleotide modulator of cancer can be introduced<br>
into a cell containing the target nucleic acid sequence, e.g., by formation of a polynucleotide-lipid complex, as described in<br>
WO 90/10448. It is understood that the use of antisense molecules or knock out and knock in models may also be used in<br>
screening assays as discussed above, in addition to methods of treatment.<br>
Inhibitory and Antisense Nucleotides<br>
[0374] In certain embodiments, the activity of a cancer-associated protein is down-regulated, or entirely inhibited, by<br>
the use of antisense polynucleotide or inhibitory small nuclear RNA (snRNA), i.e., a nucleic acid complementary to, and .<br>
which can preferably hybridize specifically to, a coding mRNA nucleic acid sequence, e.g., a cancer protein of the invention,<br>
mRNA, or a subsequence thereof. Binding of the antisense polynucleotide to the mRNA reduces the translation and/or<br>
stability of the mRNA.<br>
[0375] In the context of this invention, antisense polynucleotides can comprise naturally occurring nucleotides, or<br>
synthetic species formed from naturally occurring subunits or their close homologs. Antisense polynucleotides may also<br>
have altered sugar moieties or inter-sugar linkages. Exemplary among these are the phosphorothioate and other sulfur<br>
containing species which are known for use in the art. Analogs are comprised by this invention so long as they function<br>
effectively to hybridize with nucleotides of the invention. See, e.g., Isis Pharmaceuticals, Carlsbad, CA; Sequitor, Inc.,<br>
Natick, MA.<br>
[0376] Such antisense polynucleotides can readily be synthesized using recombinant means, or can be synthesized<br>
in vitro. Equipment for such synthesis is sold by several vendors, including Applied Biosystems. The preparation of other<br>
oligonucleotides such as phosphorothioates and alkylated derivatives is also well known to those of skill in the art.<br>
[0377] Antisense molecules as used herein include antisense or sense oligonucleotides. Sense oligonucleotides<br>
can, e.g., be employed to block transcription by binding to the anti-sense strand. The antisense and sense oligonucleotide<br>
comprise a single stranded nucleic acid sequence (either RNA or DNA) capable of binding to target mRNA (sense) or DNA<br>
(antisense) sequences for cancer molecules. Antisense or sense oligonucleotides, according to the present invention,<br>
comprise a fragment generally at least about 12 nucleotides, preferably from about 12 to 30 nucleotides. The ability to<br>
derive an antisense or a sense oligonucleotide, based upon a cDNA sequence encoding a given protein is described in,<br>
e.g., Stein &amp;Cohen (Cancer Res. 48:2659 (1988 and van der Krol et al. (BioTechniques 6:958 (1988)).<br>
Ribozvmes<br>
[0378] In addition to antisense polynucleotides, ribozymes can be used to target and inhibit transcription of cancer-<br>
associated nucleotide sequences. A ribozyme is an RNA molecule that catalytically cleaves other RNA molecules.<br>
Different kinds of ribozymes have been described, including group I ribozymes, hammerhead ribozymes, hairpin ribozymes,<br>
RNase P, and axhead ribozymes (see, e.g., Castanotto et al., Adv. in Pharmacology 25: 289-317 (1994) for a general<br>
review of the properties of different ribozymes).<br>
[0379] The general features of hairpin ribozymes are described, e.g., in Hampel et al., Nucl. Acids Res. 18:299-304<br>
(1990); European Patent Publication No. 0360257; U.S. Patent No. 5,254,678. Methods of preparing are well known to<br>
those of skill in the art (see, e.g., WO 94/26877; Ojwang et al., Proc. Natl. Acad. Sci. USA 90:6340-6344 (1993); Yamada et<br>
al., Human Gene Therapy 1:39-45 (1994); Leavitt et al., Proc. Natl. Acad Sci. USA 92:699-703 (1995); Leavittet al.,<br>
Human Gene Therapy 5:1151-120 (1994); and Yamada et al., Virology 205:121-126 (1994)).<br>
Use of Modulators in Phenotypic Screening<br>
[0380] In one embodiment, a test compound is administered to a population of cancer cells, which have an<br>
associated cancer expression profile. By "administration" or "contacting" herein is meant that the modulator is added to the<br>
cells in such a manner as to allow the modulator to act upon the cell, whether by uptake and intracellular action, or by<br>
action at the cell surface. In some embodiments, a nucleic acid encoding a proteinaceous agent (i.e., a peptide) is put into<br>
a viral construct such as an adenoviral or retroviral construct, and added to the cell, such that expression of the peptide<br>
agent is accomplished, e.g., PCT US97/01019. Regulatable gene therapy systems can also be used. Once the modulator<br>
has been administered to the cells, the cells are washed if desired and are allowed to incubate under preferably<br>
physiological conditions for some period. The cells are then harvested and a new gene expression profile is generated.<br>
Thus, e.g., cancer tissue is screened for agents that modulate, e.g., induce or suppress, the cancer phenotype. A change<br>
in at least one gene, preferably many, of the expression profile indicates that the agent has an effect on cancer activity.<br>
Similarly, altering a biological function or a signaling pathway is indicative of modulator activity. By defining such a<br>
signature for the cancer phenotype, screens for new drugs that alter the phenotype are devised. With this approach, the<br>
drug target need not be known and need not be represented in the original gene/protein expression screening platform, nor<br>
does the level of transcript for the target protein need to change. The modulator inhibiting function will serve as a surrogate<br>
Marker<br>
[0381] As outlined above, screens are done to assess genes or gene products. That is, having identified a particular<br>
differentially expressed gene as important in a particular state, screening of modulators of either the expression of the gene<br>
or the gene product itself is performed.<br>
Use of Modulators to Affect Peptides of the Invention<br>
[0382] Measurements of cancer polypeptide activity, or of the cancer phenotype are performed using a variety of<br>
assays For example, the effects of modulators upon the function of a cancer polypeptide(s) are measured by examining<br>
codeTele's described above. A physiological change that affects activity is used to assess the influence of a test<br>
compound on the polypeptides of this invention. When the functional outcomes are determined using intact cells or<br>
animals a variety of effects can be assesses such as, in the case of a cancer associated with solid tumors, tumor growth,<br>
total metastasis, neovascularization, hormone release, transcriptional changes to both known and uncharacterized<br>
generic markers (e.g., by Northern blots), changes in cell metabolism such as cell growth or pH changes, and changes in<br>
intracellular second messengers such as cGNIP.<br>
Methods of Identifying Characterizing Cancer-associated Sequences<br>
[0383] Expression of various gene sequences is correlated with cancer. Accordingly, disorders based on mutant or<br>
variant cancer genes are determined. In one embodiment, the invention provides methods for identifying cells containing<br>
variant cancer genes, e.g., determining the presence of, all or part, the sequence of at least one endogenous cancer gene<br>
in a cell. This is accomplished using any number of sequencing techniques. The invention comprises methods of<br>
identifying the cancer genotype of an individual, e.g., determining all or part of the sequence of at least one gene of the<br>
invention in the individual. This is generally done in at least one tissue of the individual, e.g., a tissue set forth in Table I,<br>
and may include the evaluation of a number of tissues or different samples of the same tissue. The method may include<br>
comparing the sequence of the sequenced gene to a known cancer gene, i.e., a wild-type gene to determine the presence<br>
of family members, homologies, mutations or variants. The sequence of all or part of the gene can then be compared to<br>
the sequence of a known cancer gene to determine if any differences exist. This is done using any number of known<br>
homology programs, such as BLAST, Bestfit, etc. The presence of a difference in the sequence between the cancer gene<br>
of the patient and the known cancer gene correlates with a disease state or a propensity for a disease state, as outlined<br>
herein.<br>
In a preferred embodiment, the cancer genes are used as probes to determine the number of copies of the cancer gene in<br>
the genome. The cancer genes are used as probes to determine the chromosomal localization of the cancer genes.<br>
Information such as chromosomal localization finds use in providing a diagnosis or prognosis in particular when<br>
chromosomal abnormalities such as translocations, and the like are identified in the cancer gene locus.<br>
XIV.') Kits/Articles of Manufacture<br>
[0384] For use in the laboratory, prognostic, prophylactic, diagnostic and therapeutic applications described herein,<br>
kits are within the scope of the invention. Such kits can comprise a carrier, package, or container that is compartmentalized<br>
to receive one or more containers such as vials, tubes, and the like, each of the container(s) comprising one of the separate<br>
elements to be used in the method, along with a label or insert comprising instructions for use, such as a use described<br><br>
herein. For example, the container(s) can comprise a probe that is or can be detectably labeled. Such probe can be an<br>
antobody or polynucleotide specific for a protein or a gene or message of the invention, respectively. Where the method<br>
utilizes nucleic acid hybridization to detect the target nucleic acid, the kit can also have containers containing nucleotide(s)<br>
for amplification of the target nucleic acid sequence. Kits can comprise a container comprising a reporter, such as a biotin-<br>
binding protein, such as avidin or streptavidin, bound to a reporter molecule, such as an enzymatic, fluorescent, or<br>
radioisotope label; such a reporter can be used with, e.g., a nucleic acid or antibody. The kit can include all or part of the<br>
amino acid sequences in Figure 2 or Figure 3 or analogs thereof, or a nucleic acid molecule that encodes such amino acid<br>
sequences.<br>
[0385] The kit of the invention will typically comprise the container described above and one or more other<br>
containers associated therewith that comprise materials desirable from a commercial and user standpoint, including buffers,<br>
diluents, filters, needles, syringes; carrier, package, container, vial and/or tube labels listing contents and/or instructions for<br>
use, and package inserts with instructions for use.<br>
[0386] A label can be present on or with the container to indicate that the composition is used for a specific therapy<br>
or non-therapeutic application, such as a prognostic, prophylactic, diagnostic or laboratory application, and can also<br>
indicate directions for either in vivo or in vitro use, such as those described herein. Directions and or other information can<br>
also be included on an insert(s) or label(s) which is included with or on the kit. The label can be on or associated with the<br>
container. A label a can be on a container when letters, numbers or other characters forming the label are molded or<br>
etched into the container itself; a label can be associated with a container when it is present within a receptacle or carrier<br>
that also holds the container, e.g., as a package insert. The label can indicate that the composition is used for diagnosing,<br>
treating, prophylaxing or prognosing a condition, such as a neoplasia of a tissue set forth in Table I.<br>
[0387] The terms "kit" and "article of manufacture" can be used as synonyms.<br>
[0388] In another embodiment of the invention, an artide(s) of manufacture containing compositions, such as amino<br>
acid sequence(s), small molecule(s), nucleic acid sequence(s), and/or antibody(s), e.g., materials useful for the diagnosis,<br>
prognosis, prophylaxis and/or treatment of neoplasias of tissues such as those set forth in Table I is provided. The article<br>
of manufacture typically comprises at least one container and at least one label. Suitable containers include, for example,<br>
bottles, vials, syringes, and test tubes. The containers can be formed from a variety of materials such as glass, met al or<br>
plastic. The container can hold amino acid sequence(s), small molecule(s), nucleic acid sequence(s), cell popuiation(s)<br>
and/or antibody(s). In one embodiment, the container holds a polynucleotide for use in examining the mRNA expression<br>
profile of a cell, together with reagents used for this purpose. In another embodiment a container comprises an antibody,<br>
binding fragment thereof or specific binding protein for use in evaluating protein expression of282P1G3 in cells and tissues,<br>
or for relevant laboratory, prognostic, diagnostic, prophylactic and therapeutic purposes; indications and/or directions for<br>
such uses can be included on or with such container, as can reagents and other compositions or tools used for these<br>
purposes. In another embodiment, a container comprises materials for eliciting a cellular or humoral immune response,<br>
together with associated indications and/or directions. In another embodiment, a container comprises materials for<br>
adoptive immunotherapy, such as cytotoxic T cells (CTL) or helper T cells (HTL), together with associated indications<br>
and/or directions; reagents and other compositions or tools used for such purpose can also be included.<br>
[0389] The container can alternatively hold a composition that is effective for treating, diagnosis, prognosing or<br>
prophylaxing a condition and can have a sterile access port (for example the container can be an intravenous solution bag<br>
or a vial having a stopper pierceable by a hypodermic injection needle). The active agents in the composition can be an<br>
antibody capable of specifically binding 282P1G3 and modulating the function of 282P1G3.<br><br>
[0390] The article of manufacture can further comprise a second container comprising a pharmaceutically-<br>
acceptable buffer, such as phosphate-buffered saline, Ringer's solution and/or dextrose solution. It can further include<br>
other materials desirable from a commercial and user standpoint, including other buffers, diluents, filters, stirrers, needles,<br>
syringes, and/or package inserts with indications and/or instructions for use.<br>
EXAMPLES<br>
[0391] Various aspects of the invention are further described and illustrated by way of the several examples that<br>
follow, none of which is intended to limit the scope of the invention.<br>
Example 1: SSH-Generated Isolation of cDNA Fragment of the PSCA Gene<br>
[0392] Intentionally Omitted<br>
Example 2: Isolation of Full Length PSCA Encoding cDNA<br>
[0393] Intentionally Omitted<br>
Example 3: Chromosomal Mapping of PSCA<br>
[0394] Intentionally Omitted<br>
Example 4<br>
Expression Analysis of PSCA Variants in Normal Tissues and Patient Specimens<br>
[0395] Previously, PSCA, herein referred to as PSCA v.1, was identified as an antigen expressed in prostate cancer.<br>
Its expression was detected in greater than 80% of primary prostate cancers and in the majority of prostate metastasis. It<br>
has also been shown to be expressed in bladder cancer, ovary cancer, and pancreatic cancer; these cancers are listed in<br>
Table I. By immunohistochemical analysis, PSCA has been shown to be overexpressed on the cell surface of most<br>
urothelial transitional carcinoma, and in 60% of primary pancreatic adenocarcinomas. The PSCA expression data has<br>
been reported in patent publications (PCT/US98/04664, PCT/US/28883, PCT/US00/19967) and in peer-reviewed articles<br>
(Saffran et al., Proc Natl Acad Sci USA. 2001 Feb 27; 98(5): 2658-2663; Amara et al., Cancer Res. 2001 Jun 15; 61(12):<br>
4660-65; Reiter et al., Proc Natl Acad Sci USA. 1998 Feb 17; 95(4): 1735-40; Argani et al., Cancer Res. 2001 Jun 1;<br>
61(11): 4320-24).<br>
[0396] Specific expression of different PSCA variants was studied in normal and cancer patient specimens<br>
(Figure 14 and Figure 15). Primers were designed to differentiate between PSCA v.1/v.2/v.4, PSCA v.3 and PSCA v.5.<br>
PSCA v.1/v.2/v.4 lead to a PCR product of 425 bp, PSCA v.3 leads to a PCR product of 300 bp, whereas PSCA v.5 leads<br>
to a PCR product of 910 bp in size (Figure 14A).<br>
[0397] First strand cDNA was prepared from normal bladder, brain, heart, kidney, liver, lung, prostate, spleen,<br>
skelet al muscle, testis, pancreas, colon, stomach, pools of prostate cancer, bladder cancer, kidney cancer, colon cancer,<br>
lung cancer, ovary cancer, breast cancer, cancer metastasis, and pancreas cancer (Figure 14B). Normalization was<br>
performed by PCR using primers to actin. Semi-quantitative PCR, using the variant specific primers was performed at 30<br>
cycles of amplification.<br>
[0398] Results show expression of PSCA v.5 mainly in breast cancer, cancer metastasis, and pancreas cancer, and<br>
at lower level in colon cancer and lung cancer. PSCA v.1/v.2/v.4 PCR product was detected in prostate cancer, bladder<br>
cancer, kidney cancer, colon cancer, lung cancer, ovary cancer, breast cancer, cancer metastasis, and pancreas cancer.<br>
Amongst normal tissues, PSCA v.1/v.2/v.4 PCR product was detected only in prostate, stomach and at lower level in kidney<br>
and lung, whereas PSCA v. 5 was not detected in any normal tissue. PSCA v.3 PCR detected product was not detected in<br>
any of the samples tested.<br>
[0399] Primers were designed to differentiate between PSCA v.4 and PSCA v.5 (Figure 15A). PSCA v.4 lead to a<br>
PCR product of 460 bp, whereas PSCA v.5 lead to a PCR product of 945 bp in size.<br>
[0400] First strand cDNA was prepared from normal bladder, brain, heart, kidney, liver, lung, prostate, spleen,<br>
stelet muscle, testis, pancreas, colon, stomach, pools of prostate cancer, bladder cancer, and multi-xenograft pool<br>
(prostate cancer, kidney cancer and bladder cancer xenografts) (Figure 15B). Normalization was performed by PCR using<br>
printers to actin. Semi-quantitative PCR, using the variant specific primers was performed at 30 cycles of amplification.<br>
[0401 ] Results show expression of PSCA v.4 in prostate cancer, bladder cancer, and multi-xenograft pool, normal<br>
kidney and prostate. PSCA v.5 was detected only in normal prostate and bladder cancer.<br>
[0402] The restricted expression of PSCA variants in normal tissues and the expression detected in cancer patient<br>
specimens indicate that PSCA variants are therapeutic, prognostic, laboratory, prophylactic, and diagnostic targets for<br>
hurr.3n cancers.<br>
Example 5: Transcript Variants of PSCA<br>
[0403] As used herein, the term variant includes transcript variants and single nucleotide polymorphisms (SNPs).<br>
Transcript variants are variants of mature mRNA from the same gene which arise by alternative transcription or alternative<br>
splicing. Alternative transcripts are transcripts from the same gene but start transcription at different points. Splice variants<br>
are mRNA variants spliced differently from the same transcript. In eukaryotes, when a multi-exon gene is transcribed from<br>
genomic DNA, the initial RNA is spliced to produce functional mRNA, which has only exons and is used for translation into<br>
an amino acid sequence. Accordingly, a given gene can have zero to many alternative transcripts and each transcript can<br>
have zero to many splice variants. Each transcript variant has a unique exon makeup, and can have different coding<br>
and/or non-coding (5' or 3' end) portions, from the original transcript. Transcript variants can code for the same, similar or<br>
different proteins, such proteins having the same or a similar function or a different function. The variant proteins can be<br>
expressed in the same tissue at the same time, in a different tissue at the same time, or in the same tissue at different<br>
times, or in a different tissue at a different time. Proteins encoded by a transcript variant can have similar or different<br>
subcellular or extracellular localizations (e.g., secreted versus intracellular).<br>
[0404] Transcript variants are identified by a variety of art-accepted methods. For example, alternative transcripts<br>
and splice variants are identified by full-length cloning, or by use of full-length transcript and EST sequences. First, all<br>
human ESTs were grouped into clusters which show direct or indirect identity with each other. Second, ESTs in the same<br>
cluster were further grouped into sub-clusters and assembled into a consensus sequence. The original gene sequence is<br>
compared to the consensus sequence(s) or other full-length sequences. Each consensus sequence is a potential splice<br>
variant for that gene. Several confirmation modalities are known in the art, such as identification of the variant by Northern<br>
analysis, full length cloning or by use of probe libraries, etc.. Even when a variant is identified that is not yet a full-length<br>
clone, that portion of the variant is very useful as a research tool, e.g., for antigen generation or for further cloning of the<br>
full-length splice variant, using techniques known in the art.<br><br>
[0405] Moreover, computer programs are available in the art that identify transcript variants based on genomic<br>
sequence. Genomic-based transcript variant identification programs include FgenesH (A. Salamov and V. Solovyev, "Ab<br>
initio gene finding in Drosophila genomic DNA," Genome Research. 2000 April; 10(4):516-22); Grail (URL<br>
compbio.ornl.gov/Grail-bin/EmptyGrailForm) and GenScan (URL genes.mit.edu/GENSCAN.html). For a general discussion<br>
of splice variant identification protocols see, e.g., Southan, C, A genomic perspective on human proteases, FEBS Lett.<br>
2001 Jun 8; 498(2-3):214-8; de Souza, S.J., et al., Identification of human chromosome 22 transcribed sequences with<br>
ORF expressed sequence tags, Proc. Natl Acad Sci USA. 2000 Nov 7; 97(23):12690-3.<br>
[0406] To further confirm the parameters of a transcript variant, a variety of techniques are available in the art, such<br>
as full-length cloning, proteomic validation, PCR-based validation, and 5' RACE validation, etc. (see e.g., Proteomic<br>
Validation: Brennan, S.O., et al., Albumin banks peninsula: a new termination variant characterized by electrospray mass<br>
spectrometry, Biochem Biophys Acta. 1999 Aug 17;1433(1-2):321-6; Ferranti P, et al., Differential splicing of pre-<br>
messenger RNA produces multiple forms of mature caprine alpha(s1)-casein, Eur J Biochem. 1997 Oct 1 ;249(1):1-7. For<br>
PCR-based Validation: Wellmann S, et al., Specific reverse transcription-PCR quantification of vascular endothelial growth<br>
factor (VEGF) splice variants by LightCycler technology, Clin Chem. 2001 Apr;47(4):654-60; Jia, H.P., et al., Discovery of<br>
new human beta-defensins using a genomics-based approach, Gene. 2001 Jan 24; 263(1-2):211-8. For PCR-based and 5'<br>
RACE Validation: Brigle, K.Ej, et al., Organization of the murine reduced folate carrier gene and identification of variant<br>
splice forms, Biochem Biophys Acta. 1997 Aug 7; 1353(2): 191-8).<br>
[0407] It is known in the art that genomic regions are modulated in cancers. When the genomic region to which a<br>
gene maps is modulated in a particular cancer, the alternative transcripts or splice variants of the gene are modulated as<br>
well. Disclosed herein is that PSCA has a particular expression profile related to cancer (see, e.g., Table I). Alternative<br>
transcripts and splice variants of PSCA are also involved in cancers, for example in one or more of these tissues and in<br>
certain additional tissues as well. The variants thus serve as tumor-associated markers/antigens.<br>
[0408] Using the full-length PSCA gene together with EST sequences, four additional transcript variants were<br>
identified, designated as PSCA v.2, v.3, v.4, and v.5. The boundaries of exons in the original transcript, PSCA v.1 were<br>
shown in Table LI. Schematic structures of the transcript variant nucleic acid sequences are shown in Figure 10. In<br>
Figure 10, bars with the same graphic pattern depict stretches of contiguous genetic material, e.g., the black bars designate<br>
genomic sequence found in variant 1.<br>
[0409] Tables Lll(a) - (d) through LV(a) - (d) are set forth on a variant-by-variant basis. Lll(a) - (d) shows the<br>
nucleotide sequences of the transcript variants. Table Llll(a) - (d) shows the alignment of the transcript variants with<br>
nucleic acid sequence of PSCA v.1 (for v.2 only) or with PSCAv.2 (for all other variants). Table LIV(a) - (d) present the<br>
amino acid translation of the transcript variants for the identified reading frame orientation. Table LV(a) - (d) displays<br>
alignments of the amino acid sequence encoded by the splice variant with that of PSCA v.1.<br>
Example 6: Single Nucleotide Polymorphisms of PSCA<br>
[0410] A Single Nucleotide Polymorphism (SNP) is a single base pair variation in a nucleotide sequence at a specific<br>
location. At any given point of the genome, there are four possible nucleotide base pairs: A/T, C/G, G/C, and T/A. As used<br>
herein, an allele is one of a series of alternative forms of a given gene, differing in DNA sequence, and affecting a product<br>
(RNA and/or protein).<br>
[0411] A SNP that occurs on a cDNA is called a cSNP. This cSNP may change amino acids of the protein encoded<br>
by the gene and thus change the function of the protein. Some SNPs cause inherited diseases; others contribute to<br><br>
quantitative variations in phenotype and reactions to environmental factors including diet and drugs among individuals.<br>
Therefore, the existence of a SNP and/or combinations of alleles (called haplotypes) have many useful applications, such<br>
a.i diagnosis of inherited diseases, determination of drug reactions and dosage, identification of genes responsible for<br>
diseases, and analysis of the genetic relationship between individuals (P. Nowotny, J. M. Kwon and A. M. Goate," SNP<br>
analysis to dissect human traits," Curr. Opin. Neurobiol. 2001 Oct; 11(5):637-641; M. Pirmohamed and B. K. Park, "Genetic<br>
susceptibility to adverse drug reactions," Trends Pharmacol. Sci. 2001 Jun; 22(6):298-305; J. H. Riley, C. J. Allan, E. Lai<br>
and A. Roses, "The use of single nucleotide polymorphisms in the isolation of common disease genes,"<br>
Pharmacogenomics. 2000 Feb; 1(1):39-47; R. Judson, J. C. Stephens and A. Windemuth, "The predictive power of<br>
haplotypes in clinical response," Pharmacogenomics. 2000 Feb; 1{1):15-26).<br>
[0412] SNPs are identified by a variety of art-accepted methods (P. Bean, "The promising voyage of SNP target<br>
discovery," Am. Clin. Lab. 2001 Oct-Nov; 20(9):18-20; K. M. Weiss, "In search of human variation," Genome Res. 1998 Jul;<br>
8(7):691-697; M. M. She, "Enabling large-scale pharmacogenetic studies by high-throughput mutation detection and<br>
genotyping technologies," Clin. Chem. 2001 Feb; 47(2):164-172). For example, SNPs are identified by sequencing DNA<br>
fragments that show polymorphism by gel-based methods such as restriction fragment length polymorphism (RFLP) and<br>
denaturing gradient gel electrophoresis (DGGE). They are also discovered by direct sequencing of DNA samples pooled<br>
from different individuals or by comparing sequences from different DNA samples. With the rapid accumulation of sequence<br>
data in public and private databases, one also discovers SNPs by comparing sequences using computer programs (Z. Gu,<br>
L. Hillier and P. Y. Kwok, "Single nucleotide polymorphism hunting in cyberspace," Hum. Mutat. 1998; 12(4):221-225).<br>
SNPs can be verified and the genotype or haplotype of an individual can be determined by a variety of methods including<br>
direct sequencing and high throughput microarrays (P. Y. Kwok, "Methods for genotyping single nucleotide<br>
polymorphisms," Annu. Rev. Genomics Hum. Genet. 2001; 2:235-258; M. Kokoris, K. Dix, K. Moynihan, J. Mathis, B. Erwin,<br>
P. Grass, B. Hines and A. Duesterhoeft, "High-throughput SNP genotyping with the Masscode system," Mol. Diagn. 2000<br>
Dec; 5(4):329-340).<br>
[0413] Using the methods described above, thirteen SNP were identified in the transcript for PSCA v.2. Variant 2<br>
was used, rather than for example variant 1, as it had fewer ambiguous bases than variant 1. Accordingly, SNPs were<br>
identified in PSCA v.2, at positions 57 (t/c), 367 (c/t), 424 (a/c), 495 (c/g), 499 (c/t), 563 (c/t), 567 (g/a), 627 (g/a), 634 (t/g),<br>
835 (g/a), 847 (g/a), 878 (g/a), and 978 (c/g). The transcripts or proteins with alternative alleles were designated as variant<br>
PSCA v.6 through v.18, as shown in Table LVI and Figure 12a.<br>
[0414] The nucleotide change in v.6 changed the start codon of v.1 and thus, the translation would not start until the<br>
next ATG (AUG in mRNA), resulting in a protein 9 AA shorter than v.1 protein (Figure 11 a). The nucleotide changes for<br>
v.7 and v.8 were silent at the protein level.<br>
[0415] Twelve of these 13 SNPs were also present in variant4 as set forth in Figure 12b and table LVI. The 12 SNP<br>
variants relative to PSCA v. 4 are designated PSCA v. 19 through v.30. Variants 19 through 27 encode alternative amino<br>
acids. (Figure 11b and Table LVI).<br>
[0416] Table LVI also shows the amino acid changes of protein sequence. These SNP, though shown individually<br>
separately here, can occur in different combinations and in any one of the transcript variants that contains the site of the<br>
SNP.<br>
Example 7: Production of Recombinant PSCA in Prokarvotic Systems<br>
[0417] To express recombinant PSCA and PSCA variants in prokaryotic cells, the full or partial length PSCA and<br>
PSCA variant cDNA sequences are cloned into any one of a variety of expression vectors known in the art. One or more of<br>
the following regions of PSCA variants are expressed: the full length sequence presented in Figures 2 and 3, or any 8,9,<br>
10,11,12,13,14,15,16,17,18,19,20, 21, 22,23,24,25, 26, 27,28, 29,30 or more contiguous amino acids from PSCA,<br>
variants, or analogs thereof.<br>
A.______In vitro transcription and translation constructs:<br>
[0418] pCRII: To generate PSCA sense and anti-sense RNA probes for RNA in situ investigations, pCRH constructs<br>
(Invitrogen, Carlsbad CA) are generated encoding eithet all or fragments of the PSCA cDNA. The pCRII vector has Sp6<br>
and T7 promoters flanking the insert to drive the transcription of PSCA RNA for use as probes in RNA in situ hybridization<br>
experiments. These probes are used to analyze the cell and tissue expression of PSCA at the RNA level. Transcribed.<br>
PSCA RNA representing the cDNA amino acid coding region of the PSCA gene is used in in vitro translation systems such<br>
as the TnT™ Coupled Reticulolysate System (Promega, Corp., Madison, Wl) to synthesize PSCA protein.<br>
B. Bacterial Constructs:<br>
[0419] pGEX Constructs: To generate recombinant PSCA proteins in bacteria that are fused to the Glutathione S-<br>
transferase (GST) protein, all or parts of the PSCA cDNA protein coding sequence are cloned into the pGEX family of GST-<br>
fusion vectors (Amersham Pharmacia Biotech, Piscataway, NJ). These constructs allow controlled expression of<br>
recombinant PSCA protein sequences with GST fused at the 'amino-terminus and a six histidine epitope (6X His) at the<br>
carboxyl-terminus. The GST and 6X His tags permit purification of the recombinant fusion protein from induced bacteria<br>
with the appropriate affinity matrix and allow recognition of the fusion protein with anti-GST and anti-His antibodies. The 6X<br>
His tag is generated by adding 6 histidine codons to the cloning primer at the 3' end, e.g., of the open reading frame (ORF).<br>
A proteolytic cleavage site, such as the PreScission™ recognition site in pGEX-6P-l, may be employed such that it permits<br>
cleavage of the GST tag from PSCA-related protein. The ampiciliin resistance gene and pBR322 origin permits selection<br>
and maintenance of the pGEX plasmids in E. coli.<br>
[0420] pMAL Constructs: To generate, in bacteria, recombinant PSCA proteins that are fused to maltose-binding<br>
protein (MBP), all or parts of the PSCA cDNA protein coding sequence are fused to the MBP gene by cloning into the<br>
pMAL-c2X and pMAL-p2X vectors (New England Biolabs, Beverly, MA). These constructs allow controlled expression of<br>
recombinant PSCA protein sequences with MBP fused at the amino-terminus and a 6X His epitope tag at the carboxyl-<br>
terminus. The MBP and 6X His tags permit purification of the recombinant protein from induced bacteria with the<br>
appropriate affinity matrix and allow recognition of the fusion protein with anti-MBP and anti-His antibodies. The 6X His<br>
epitope tag is generated by adding 6 histidine codons to the 3' cloning primer. A Factor Xa recognition site permits<br>
cleavage of the pMAL tag from PSCA. The pMAL-c2X and pMAL-p2X vectors are optimized to express the recombinant<br>
protein in the cytoplasm or periplasm respectively. Periplasm expression enhances folding of proteins with disulfide bonds.<br>
[0421] pET Constructs: To express PSCA in bacterial cells, all or parts of the PSCA cDNA protein coding sequence<br>
are cloned into the pET family of vectors (Novagen, Madison, Wl). These vectors allow tightly controlled expression of<br>
recombinant PSCA protein in bacteria with and without fusion to proteins that enhance solubility, such as NusA and<br>
thioredoxin (Trx), and epitope tags, such as 6X His and S-Tag ™ that aid purification and detection of the recombinant<br>
protein. For example, constructs are made utilizing pET NusA fusion system 43.1 such that regions of the PSCA protein<br>
are expressed as amino-terminai fusions to NusA.<br>
C. Yeast Constructs:<br>
[0422] pESC Constructs: To express PSCA in the yeast species Saccharomyces cerevisiae for generation of<br>
recombinant protein and functional studies, all or parts of the PSCA cDNA protein coding sequence are cloned into the<br>
pESC family of vectors each of which contain 1 of 4 selectable markers, HIS3, TRP1, LEU2, and URA3 (Stratagene, La<br>
Jolla, CA). These vectors allow controlled expression from the same plasmid of up to 2 different genes or cloned<br>
sequences containing either Flag™ or Myc epitope tags in the same yeast cell. This system is useful to confirm protein-<br>
protein interactions of PSCA. In addition, expression in yeast yields similar post-translational modifications, such as<br>
glycosylations and phosphorylations, that are found when expressed in eukaryotic cells.<br>
[0423] pESP Constructs: To express PSCA in the yeast species Saccharomyces pombe, all or parts of the PSCA<br>
cDNA protein coding sequence are cloned into the pESP family of vectors. These vectors allow controlled high level of<br>
expression of a PSCA protein sequence that is fused at either the amino terminus or at the carboxyl terminus to GST which<br>
aids purification of the recombinant protein. A Flag™ epitope tag allows detection of the recombinant protein with anti-<br>
Flag™ antibody.<br>
Example 8: Production of Recombinant PSCA in Higher Eukaryotic Systems<br>
[0424] Intentionally Omitted<br>
Example 9: Antigenicity Profiles and Secondary Structure<br>
[0425] Figure 5A-C, Figure 6A-C, Figure 7A-C, Figure 8A-C, and Figure 9A-C depict graphically five amino acid<br>
profiles of PSCA variants 1, 3, and 4, each assessment available by accessing the ProtScale website (URL<br>
www.expasy.ch/cgi-bin/protscale.pl) on the ExPasy molecular biology server.<br>
[0426] These profiles: Figure 5, Hydrophilicity, (Hopp T.P., Woods K.R., 1981. Proc. Natl. Acad. Sci. U.S.A. 78:3824-<br>
3828); Figure 6, Hydropathicity, (Kyte J., Doolittle R.F., 1982. J. Mol. Biol. 157:105-132); Figure 7, Percentage Accessible<br>
Residues (Janin J., 1979 Nature 277:491-492); Figure 8, Average Flexibility, (Bhaskaran R., and Ponnuswamy P.K., 1988.<br>
Int. J. Pept. Protein Res. 32:242-255); Figure 9, Beta-turn (Deleage, G., Roux B. 1987 Protein Engineering 1:289-294); and<br>
optionally others available in the art, such as on the ProtScale website, were used to identify antigenic regions of each of<br>
the PSCA variant proteins. Each of the above amino acid profiles of PSCA variants were generated using the following<br>
ProtScale parameters for analysis: 1) A window size of 9; 2) 100% weight of the window edges compared to the window<br>
center; and, 3) amino acid profile values normalized to lie between 0 and 1.<br>
[0427] Hydrophilicity (Figure 5), Hydropathicity (Figure 6) and Percentage Accessible Residues (Figure 7) profiles<br>
were used to determine stretches of hydrophilic amino acids (i.e., values greater than 0.5 on the Hydrophilicity and<br>
Percentage Accessible Residues profile, and values less than 0.5 on the Hydropathicity profile). Such regions are likely to<br>
be exposed to the aqueous environment, be present on the surface of the protein, and thus available for immune<br>
recognition, such as by antibodies.<br>
[0428] Average Flexibility (Figure 8) and Beta-turn (Figure 9) profiles determine stretches of amino acids (i.e., values<br>
greater than 0.5 on the Beta-turn profile and the Average Flexibility profile) that are not constrained in secondary structures<br>
such as beta sheets and alpha helices. Such regions are also more likely to be exposed on the protein and thus accessible<br>
immune recognition, such as by antibodies.<br>
[0429] Antigenic sequences of the PSCA variant proteins indicated, e.g., by the profiles set forth in Figure 5A-C,<br>
Figure 6A-C, Figure 7A-C, Figure 8A-C, and/or Figure 9A-C are used to prepare immunogens, either peptides or nucleic<br>
acids that encode them, to generate therapeutic and diagnostic anti-PSCA antibodies. The immunogen can be any 5, 6,7,<br>
8,9,10,11,12,13,14,15,16,17,18,19,20, 21, 22, 23, 24,25, 30,35, 40,45, 50 or more than 50 contiguous amino<br>
acids, or the corresponding nucleic acids that encode them, from the PSCA protein variants listed in Figures 2 and 3 of<br>
which the amino acid profiles are shown in Figure 9, or can be inferred because the variant contains sequence that is the<br>
same as a variant depicted in figure 9. In particular, peptide immunogens of the invention can comprise, a peptide region of<br>
at least 5 amino acids of Figures 2 and 3 in any whole number increment that includes an amino acid position having a<br>
value greater than 0.5 in the Hydrophilicity profiles of Figure 5; a peptide region of at least 5 amino acids of Figures 2 and 3<br>
in any whole number increment that includes an amino acid position having a value less than 0.5 in the Hydropathicity<br>
profile of Figures 6; a peptide region of at least 5 amino acids of Figures 2 and 3 in any whole number increment that<br>
includes an amino acid position having a value greater than 0.5 in the Percent Accessible Residues profiles of Figure 7; a<br>
peptide region of at least 5 amino acids of Figures 2 and 3 in any whole number increment that includes an amino acid<br>
position having a value greater than 0.5 in the Average Flexibility profiles on Figure 8; and, a peptide region of at least 5<br>
amino acids of Figures 2 and 3 in any whole number increment that includes an amino acid position having a value greater<br>
than 0.5 in the Beta-turn profile of Figures 9. Peptide immunogens of the invention can also comprise nucleic acids that<br>
encode any of the forgoing.<br>
[0430] All immunogens of the invention, peptide or nucleic acid, can be embodied in human unit dose form, or<br>
comprised by a composition that includes a pharmaceutical excipient compatible with human physiology.<br>
[0431] The secondary structure of PSCA protein variants 1,3,4, and 6, namely the predicted presence and location<br>
of alpha helices, extended strands, and random coils, is predicted from the primary amino acid sequence using the HNN -<br>
Hierarchical Neural Network method (NPS@: Network Protein Sequence Analysis TIBS 2000 March Vol. 25, No 3<br>
[291]:147-150 Combet C, Blanchet C, Geourjon C. and Deleage G., http://pbil.ibcp.fr/cgi-<br>
bin/npsa_automat.pl?page=npsa_nn.html), accessed from the ExPasy molecular biology server<br>
(http://www.expasy.ch/tools/). The analysis indicates that PSCA variant 1 is composed of 30.89% alpha helix, 21.95%<br>
extended strand, and 47.15% random coil (Figure 13A). PSCA protein variant 3 is composed of 14.89% alpha helix, 8.51%<br>
extended strand, and 76.60% random coil (Figure 13B). PSCA protein variant 4 is composed of 9.52% alpha helix, 8.99%<br>
extended strand, and 81.48% random coil (Figure 13C). PSCA protein variant 6 is composed of 24.56% alpha helix,<br>
21.93% extended strand, and 53.51% random coil (Figure 13D).<br>
[0432] Analysis for the potential presence of transmembrane domains in the PSCA variant proteins was carried out<br>
using a variety of transmembrane prediction algorithms accessed from the ExPasy molecular biology server<br>
(http://www.expasy.ch/tools/). Shown graphically in figure 13E, G, I, and K are the results of analyses of variants 1, 3,4,<br>
and 6, respectively, using the TMpred program. Shown graphically in figure 13F, H, J, and L are the results of analyses of<br>
variants 1,3,4, and 6, respectively using the TMHMM program. PSCA variant 1 and variant 6 proteins are likely to encode<br>
GPI-linked proteins. Variants 3 and 4 are likely to encode soluble proteins since they do not contain significant predictions<br>
for transmembrane domains. The results of structural analysis programs are summarized in Table VI.<br>
Example 10: Generation of PSCA Polyclonal Antibodies<br>
[0433] Polyclonal antibodies can be raised in a mammal, for example, by one or more injections of an immunizing<br>
a'genl and, if desired, an adjuvant. Typically, the immunizing agent and/or adjuvant will be injected in the mammal by<br>
multiple subcutaneous or intraperitoneal injections. In addition to immunizing with a full length PSCA protein variant,<br>
computet algorithms are employed in design of immunogens that, based on amino acid sequence analysis contain<br>
characteristics of being antigenic and available for recognition by the immune system of the immunized host (see the<br>
Example entitled "Antigenicity Profiles and Secondary Structure"). Such regions would be predicted to be hydrophilic,<br>
flexible, in beta-turn conformations, and be exposed on the surface of the protein (see, e.g., Figure 5A-C, Figure 6A-C,<br>
Figure 7A-C, Figure 8A-C, or Figure 9A-C for amino acid profiles that indicate such regions of PSCA protein variant 1).<br>
[0434] For example, recombinant bacterial fusion proteins or peptides containing hydrophilic, flexible, beta-turn<br>
regions of PSCA protein variants are used as antigens to generate polyclonal antibodies in New Zealand White rabbits or<br>
monoclonal antibodies as described in Example 11. For example, in PSCA variant 1, such regions include, but are not<br>
limited to, amino acids 28-56 and amino acids 66-94. For variant 3, such regions include, but are not limited to, amino<br>
acids 7-39 and amino acids 70-94. For variant 4 such regions include, but are not limited to, amino acids 6-18, amino acids<br>
27-39, amino acids 103-133, and 177-189. For variant 6, such regions include, but are not limited to, amino acids 19-35<br>
and amino acids 57-85. It is useful to conjugate the immunizing agent to a protein known to be immunogenic in the<br>
mammal being immunized. Examples of such immunogenic proteins include, but are not limited to, keyhole limpet<br>
hemocyanin (KLH), serum albumin, bovine thyroglobulin, and soybean trypsin inhibitor. In one embodiment, a peptide<br>
encoding amino acids 103-133 of PSCA variant 4 is conjugated to KLH and used to immunize a rabbit. Alternatively the<br>
immunizing agent may include all or portions of the PSCA variant proteins, analogs or fusion proteins thereof. For<br>
example, the PSCA variants amino acid sequences can be fused using recombinant DNA techniques to any one of a<br>
variety of fusion protein partners that are well known in the art, such as glutathione-S-transferase (GST) and HIS tagged<br>
fusion proteins. In one embodiment, the PSCA variant 1 sequence, amino acids 18-98 was fused to GST using<br>
recombinant techniques in the pGEX expression vector, expressed, purified and used to immunize both rabbits and mice to<br>
generate polyclonal and monoclonal antibodies respectively. Such fusion proteins are purified from induced bacteria using<br>
the appropriate affinity matrix.<br>
[0435] Other recombinant bacterial fusion proteins that may be employed include maltose binding protein, LacZ,<br>
thioredoxin, NusA, or an immunoglobulin constant region (see the section entitled "Production of PSCA in Prokaryotic<br>
Systems" and Current Protocols In Molecular Biology, Volume 2, Unit 16, Frederick M. Ausubul et al. eds., 1995; Linsley,<br>
P.S., Brady, W., Urnes, M., Grosmaire, L, Damle, N., and Ledbetter, L.(1991) J.Exp. Med. 174, 561-566).<br>
[0436] In addition to bacterial derived fusion proteins, mammalian expressed protein antigens are also used. These<br>
antigens are expressed from mammalian expression vectors such as the Tag5 and Fc-fusion vectors (see the section<br>
entitled "Production of Recombinant PSCA in Eukaryotic Systems"), and ret aln post-translational modifications such as<br>
glycosylates found in native protein. In one embodiment, the cDNA of PSCA variant 1, minus the N-terminal leader<br>
peptide and C-terminal GPI anchor was cloned into the Tag5 mammalian secretion vector, and expressed in 293T cells.<br>
The recombinant protein was purified by met al chelate chromatography from tissue culture supematants of 293T cells<br>
stably expressing the recombinant vector. The purified Tag5 PSCA protein was then used as immunogen.<br>
[0437] During the immunization protocol, it is useful to mix or emulsify the antigen in adjuvants that enhance the<br>
immune response of the host animal. Examples of adjuvants include, but are not limited to, complete Freund's adjuvant<br>
(CFA) and MPL-TDM adjuvant (monophosphoryl Lipid A, synthetic trehalose dicorynomycolate).<br><br>
[0438] In a typical protocol, rabbits are initially immunized subcutaneously with up to 200 µg , typically 100-200 µg ,<br>
passion protein or peptide conjugated to KLH mixed in complete Freund's adjuvant (CFA). Rabbits are then injected<br>
subcutaneously every two weeks with up to 200µg, typically 100-200µg, of the immunogen in incomplete Freund's<br>
adjuvant (IFA). Test bleeds are taken approximately 7-10 days following each immunization and used to monitor the titer<br>
of the antiserum by ELISA.<br>
[0439] To test reactivity and specificity of immune serum, such as rabbit serum derived from immunization with a<br>
GST-fusion of PSCA variant 3 or 4 protein, the respective full-length PSGA variant cDNA is cloned into pCDNA 3.1 myc-his<br>
expression vector (Invitrogen, see the Example entitled "Production of Recombinant PSCA in Eukaryotic Systems"). After<br>
transfection of the constructs into 293T cells, cell lysates are probed with the anti-variant serum and with anti-His antibody<br>
(Santa Cruz Biotechnologies, Santa Cruz, CA) to determine specific reactivity to denatured variant protein using the<br>
Western blot technique. In addition, the immune serum is tested by fluorescence microscopy, flow cytometry and<br>
immunoprecipitation against 293T and other recombinant PSCA variant-expressing cells to determine specific recognition<br>
of native protein. Western blot, immunoprecipitation, fluorescent microscopy, and flow cytometric techniques using cells<br>
that endogenously express PSCA are also carried out to test reactivity and specificity.<br>
[0440] Anti-serum from rabbits immunized with PSCA variant fusion proteins, such as GST and MBP fusion proteins,<br>
are purified by depletion of antibodies reactive to the fusion partner sequence by passage over an affinity column<br>
containing the fusion partner eithet alone or in the context of an irrelevant fusion protein. For example, antiserum derived<br>
from a GST-PSCA variant 1 fusion protein is first purified by passage over a column of GST protein covalently coupled to<br>
AffiGel matrix (BioRad, Hercules, Calif.). The antiserum is then affinity purified by passage over a column composed of a<br>
MBP-PSCA fusion protein covalently coupled to Affigel matrix. The serum is then further purified by protein G affinity<br>
chromatography to isolate the lgG fraction. Sera from other His-tagged antigens and peptide immunized rabbits as well as<br>
fusion partner depleted sera are affinity purified by passage over a column matrix composed of the original protein<br>
immunogen or free peptide.<br>
Example 11: Generation of PSCA Monoclonal Antibodies (mAbs)<br>
[0441] In one embodiment, therapeutic mAbs to PSCA variants comprise those that react with epitopes specific for<br>
each variant protein or specific to sequences in common between the variants that would disrupt or modulate the biological<br>
function of the PSCA variants, for example those that would disrupt the interaction with ligands and binding partners,<br>
immunogens for generation of such mAbs include those designed to encode or contain the entire PSCA protein variant<br>
sequence, regions of the PSCA protein variants predicted to be antigenic from computer analysis of the amino acid<br>
sequence (see, e.g., Figure 5A-C, Figure 6A-C, Figure 7A-C, Figure 8A-C, or Figure 9A-C, and the Example entitled<br>
"Antigenicity Profiles"). Immunogens include peptides, recombinant bacterial proteins, and mammalian expressed Tag 5<br>
proteins and human and murine lgG FC fusion proteins. In addition, cells engineered to express high levels of a respective<br>
PSCA variant, such as 293T-PSCA variant 4 or 300.19-PSCA variant 4 murine Pre-B cells, are used to immunize mice.<br>
[0442] To generate mAbs to a PSCA variant, mice are first immunized intraperitoneally (IP) with, typically, 10-50 ug<br>
of protein immunogen or W PSCA-expressing cells mixed in complete Freund's adjuvant. Mice are then subsequently<br>
immunized IP every 2-4 weeks with, typically, 10-50 ug of protein immunogen or 107 cells mixed in incomplete Freund's<br>
adjuvant. Alternatively, MPL-TDM adjuvant is used in immunizations. In addition to the above protein and cell-based<br>
immunization strategies, a DNA-based immunization protocol is employed in which a mammalian expression vector<br>
encoding a PSCA variant sequence is used to immunize mice by direct injection of the plasmid DNA. For example, the<br>
complete cDNA of PSCA of variant 4 is cloned into the Tag5 mammalian secretion vector and the recombinant vector will<br>
th3n be used as immunogen. In another example the same amino acids are cloned into an Fc-fusion secretion vector in<br>
which the PSCA variant 4 sequence is fused at the amino-terminus to an IgK leader sequence and at the carboxyl-terminus<br>
to the coding sequence of the human or murine IgG Fc region. This recombinant vector is then used as immunogen. The<br>
plasmid immunization protocols are used in combination with purified proteins expressed from the same vector and with<br>
cells expressing the respective PSCA variant.<br>
[0443] During the immunization protocol, test bleeds are taken 7-10 days following an injection to monitor titer and<br>
specificity of the immune response. Once appropriate reactivity and specificity is obtained as determined by ELISA,<br>
Western blotting, immunoprecipitation, fluorescence microscopy, and flow cytometric analyses, fusion and hybridoma<br>
generation is then carried out with established procedures well known in the art (see, e.g., Harlow and Lane, 1988).<br>
[0444] In one embodiment for generating PSCA monoclonal antibodies, a GST-fusion of variant 4 antigen encoding<br>
amino acids 1-189 is expressed and then purified from stably transfected 293T cells. Balb C mice are initially immunized<br>
intraperitoneally with 25µg of the GST-PSCA variant 4 protein mixed in complete Freund's adjuvant. Mice are<br>
subsequently immunized every two weeks with 25 µg of the antigen mixed in incomplete Freund's adjuvant for a total of<br>
three immunizations. ELISA using the GST-fusion antigen and a cleavage product from which the GST portion is removed<br>
determines the titer of serum from immunized mice. Reactivity and specificity of serum to full length PSCA variant 4 protein<br>
is monitored by Western blotting, immunoprecipitation and flow cytometry using 293T cells transfected with an expression<br>
vector encoding the PSCA variant 1 cDNA (see e.g., the Example entitled "Production of Recombinant PSCA in Eukaryotic<br>
Systems"). Other recombinant PSCA variant 4-expressing cells or cells endogenously expressing PSCA variant 4 are also<br>
used. Mice showing the strongest reactivity are rested and given a final injection of Tag5 antigen in PBS and then<br>
sacrificed four days later. The spleens of the sacrificed mice are harvested and fused to SPO/2 myeloma cells using<br>
standard procedures (Harlow and Lane, 1988). Supernatants from HAT selected growth wells are screened by ELISA,<br>
Western blot, immunoprecipitation, fluorescent microscopy, and flow cytometry to identify PSCA specific antibody-<br>
producing clones.<br>
[0445] To generate monoclonal antibodies that are specific for PSCA variant 4 protein, immunogens are designed to<br>
encode the sequence unique to that variant. For example, a peptide encoding amino acids 6-18 of PSCA variant 4 is<br>
synthesized, conjugated to KLH and used as immunogen. Hybridoma supernatants are then screened on the peptide<br>
antigen and then further screened on cells expressing the PSCA variant 4 protein and cross-screened on cells expressing<br>
the other PSCA variants to derive variant 4-specific monoclonal antibodies.<br>
[0446] The binding affinity of a PSCA variant monoclonal antibody is determined using standard technologies.<br>
Affinity measurements quantify the strength of antibody to epitope binding and are used to help define which PSCA variant<br>
monoclonal antibodies preferred for diagnostic or therapeutic use, as appreciated by one of skill in the art. The BIAcore<br>
system (Uppsala, Sweden) is a preferred method for determining binding affinity. The BIAcore system uses surface<br>
plasmon resonance (SPR, Welford K. 1991, Opt. Quant. Elect. 23:1; Morton and Myszka, 1998, Methods in Enzymology<br>
295:268) to monitor biomolecular interactions in real time. BIAcore analysis conveniently generates association rate<br>
constants, dissociation rate constants, equilibrium dissociation constants, and affinity constants.<br>
Example 12: HLA Class I and Class II Binding Assays<br>
HLA class I and class II binding assays using purified HLA molecules are performed in accordance with disclosed<br>
protocols (e.g., PCT publications WO 94/20127 and WO 94/03205; Sidney et al., Current Protocols in Immunology 18.3.1<br>
(1998); Sidney, et al., J. Immunol. 154:247 (1995); Sette, et al., Mol. Immunol. 31:813 (1994)). Briefly, purified MHC<br>
molecules (5 to 500 nM) are incubated with various unlabeled peptide inhibitors and 1-10 nM 125l-radiolabeled probe<br>
peptides as described. Following incubation, MHC-peptide complexes are separated from free peptide by gel filtration and<br>
the fraction of peptide bound is determined. Typically, in preliminary experiments, each MHC preparation is titered in the<br>
presence of fixed amounts of radiolabeled peptides to determine the concentration of HLA molecules necessary to bind 10-<br>
202 of the total radioactivity. All subsequent inhibition and direct binding assays are performed using these HLA<br>
concentrations.<br>
Since under these conditions [label][HLA], the measured ICso values are reasonable<br>
pop concations of the true Kd values. Peptide inhibitors are typically tested at concentrations ranging from 120 µg/ml to 1.2<br>
ng ml and are tested in two to four completely independent experiments. To allow comparison of the data obtained in<br>
different experiments, a relative binding figure is calculated for each peptide by dividing the IC50 of a positive control for<br>
inhibition by the IC50 for each tested peptide (typically unlabeled versions of the radiolabeled probe peptide). For database<br>
purposes, and inter-experiment comparisons, relative binding values are compiled. These values can subsequently be<br>
converted back into IC50 nM values by dividing the ICso nM of the positive controls for inhibition by the relative binding of the<br>
peptide of interest. This method of data compilation is accurate and consistent for comparing peptides that have been<br>
tested on different days, or with different lots of purified MHC.<br>
Binding assays as outlined above may be used to analyze HLA supermotif and/or HLA motif-bearing peptides<br>
(see Table IV).<br>
Example 13: Identification of HLA Supermotif- and Motif-Bearing CTL Candidate<br>
Epitopes<br>
[0447] HLA vaccine compositions of the invention can include multiple epitopes. The multiple epitopes can comprise<br>
multiple HLA supermotifs or motifs to achieve broad population coverage. This example illustrates the identification and<br>
confirmation of supermotif- and motif-bearing epitopes for the inclusion in such a vaccine composition. Calculation of<br>
population coverage is performed using the strategy described below.<br>
Computer searches and algorithms for identification of supermotif and/or motif-bearing<br>
epitopes<br>
[0448] The searches performed to identify the motif-bearing peptide sequences in the Example entitled "Antigenicity<br>
Profiles" and Tables VIII-XXI and XXII-XLIX employ the protein sequence data from the gene product of PSCA set forth in<br>
Figures 2 and 3, the specific search peptides used to generate the tables are listed in Table VII.<br>
[0449] Computer searches for epitopes bearing HLA Class I or Class II supermotifs or motifs are performed as<br>
follows. All translated PSCA protein sequences are analyzed using a text string search software program to identify<br>
potential peptide sequences containing appropriate HLA binding motifs; such programs are readily produced in accordance<br>
with information in the art in view of known motif/supermotif disclosures. Furthermore, such calculations can be made<br>
mentally.<br>
[0450] Identified A2-, A3-, and DR-supermotif sequences are scored using polynomial algorithms to predict their<br>
capacity to bind to specific HLA-Class I or Class II molecules. These polynomial algorithms account for the impact of<br>
different amino acids at different positions, and are essentially based on the premise that the overall affinity (or AG) of<br>
peptide-HLA molecule interactions can be approximated as a linear polynomial function of the type:<br><br>
where a;, is a coefficient which represents the effect of the presence of a given amino acid {I) at a given position<br>
(I) along the sequence of a peptide of n amino acids. The crucial assumption of this method is that the effects at each<br>
position are essentially independent of each other (i.e., independent binding of individual side-chains). When residue j<br>
occurs at position / in the peptide, it is assumed to contribute a constant amount// to the free energy of binding of the<br>
peptide irrespective of the sequence of the rest of the peptide.<br>
[0451] The method of derivation of specific algorithm coefficients has been described in Gulukota ef a/., J. Mol. Biol.<br>
267:1258-126,1997; (see also Sidney ef at., Human Immunol. 45:79-93,1996; and Southwood ef a/., J. Immunol.<br>
160:3363-3373,1998). Briefly, for all i positions, anchor and non-anchor alike, the geometric mean of the average relative<br>
binding (ARB) of all peptides carrying / is calculated relative to the remainder of the group, and used as the estimate of j.<br>
For Class II peptides, if multiple alignments are possible, only the highest scoring alignment is utilized, following an iterative<br>
procedure. To calculate an algorithm score of a given peptide in a test set, the ARB values corresponding to the sequence<br>
of the peptide are multiplied. If this product exceeds a chosen threshold, the peptide is predicted to bind. Appropriate<br>
thresholds are chosen as a function of the degree of stringency of prediction desired.<br>
Selection of HLA-A2 supertvpe cross-reactive peptides<br>
[0452] Protein sequences from PSCA are scanned utilizing motif identification software, to identify 8-, 9-10- and 11-<br>
mer sequences containing the HLA-A2-supermotif main anchor specificity. Typically, these sequences are then scored<br>
using the protocol described above and the peptides corresponding to the positive-scoring sequences are synthesized and<br>
tested for their capacity to bind purified HLA-A*0201 molecules in vitro (HLA-A*0201 is considered a prototype A2<br>
supertype molecule).<br>
[0453] These peptides are then tested for the capacity to bind to additional A2-supertype molecules (A*0202,<br>
A*0203, A*0206, and A*6802). Peptides that bind to at least three of the five A2-supertype alleles tested are typically<br>
deemed A2-supertype cross-reactive binders. Preferred peptides bind at an affinity equal to or less than 500 nM to three or<br>
more HLA-A2 supertype molecules.<br>
Selection of HLA-A3 supermotif-bearina epitopes<br>
[0454] The PSCA protein sequence(s) scanned above is also examined for the presence of peptides with the HLA-<br>
A3-supermotif primary anchors. P-.tides corresponding to the HLA A3 supermotif-bearing sequences are then<br>
synthesized and tested for binding to HLA-A*0301 and HLA-A*1101 molecules, the molecules encoded by the two most<br>
prevalent A3-supertype alleles. The peptides that bind at least one of the two alleles with binding affinities of 
often 
A*3301, and A*6801) to identify those that can bind at least three of the five HLA-A3-supertype molecules tested.<br>
Selection of HLA-B7 supermotif bearing epitopes<br>
[0455] The PSCA protein(s) scanned above is also analyzed for the presence of 8-, 9-10-, or 11-mer peptides with<br>
the HLA-B7-supermotif. Corresponding peptides are synthesized and tested for binding to HLA-B*0702, the molecule<br>
encoded by the most common B7-supertype allele (i.e., the prototype B7 supertype allele). Peptides binding B*0702 with<br>
IC50 of 
supertype molecules (e.g., ET3501, B*5101, B*5301, and B*5401). Peptides capable of binding to three or more of the five<br>
B7-supertype alleles tested are thereby identified.<br>
Selection of A1 and A24 motif-bearing epitopes<br>
[0456] To further increase population coverage, HLA-A1 and -A24 epitopes can also be incorporated into vaccine<br>
compositions. An analysis of the PSCA protein can also be performed to identify HLA-A1- and A24-motif-containing<br>
sequences.<br>
[0457] High affinity and/or cross-reactive binding epitopes that bear other motif and/or supermotifs are identified<br>
using analogous methodology.<br>
Example 14: Confirmation of Immunogenicity<br>
[0458] Cross-reactive candidate CTL A2-supermotif-bearing peptides that are identified as described herein are<br>
selected to confirm in vitro immunogenicity. Confirmation is performed using the following methodology:<br>
Target Cell Lines for Cellular Screening:<br>
[0459] The .221A2.1 cell line, produced by transferring the HLA-A2.1 gene into the HLA-A, -B, -C null mutant human<br>
B-lymphoblastoid cell line 721.221, is used as the peptide-loaded target to measure activity of HLA-A2.1-restricted CTL.<br>
This cell line is grown in RPMI-1640 medium supplemented with antibiotics, sodium pyruvate, nonessential amino acids<br>
and 10% (v/v) heat inactivated FCS. Cells that express an antigen of interest, or transfectants comprising the gene<br>
encoding the antigen of interest, can be used as target cells to confirm the ability of peptide-specific CTLs to recognize<br>
endogenous antigen.<br>
Primary CTL Induction Cultures:<br>
[0460] Generation of Dendritic Cells (DC): PBMCs are thawed in RPMI with 30 µg/ml DNAse, washed twice and<br>
resuspended in complete medium (RPMI-1640 plus 5% AB human serum, non-essential amino acids, sodium pyruvate, L-<br>
glutamine and penicillin/streptomycin). The monocytes are purified by plating 10x 10s PBMC/well in a 6-well plate. After 2<br>
hours at 37°C, the non-adherent cells are removed by gently shaking the plates and aspirating the supernatants. The wells<br>
are washed a total of three times with 3 ml RPMI to remove most of the non-adherent and loosely adherent cells. Three ml<br>
of complete medium containing 50 ng/ml of GM-CSF and 1,000 U/ml of IL-4 are then added to each well. TNFa is added<br>
to the DCs on day 6 at 75 ng/ml and the cells are used for CTL induction cultures on day 7.<br>
[0461] Induction of CTL with DC and Peptide: CD8+ T-cells are isolated by positive selection with Dynal<br>
immunomagnetic beads (Dynabeads® M-450) and the detacha-bead® reagent. Typically about 200-250x106 PBMC are<br>
processed to obtain 24x106 CD8+T-cells (enough for a 48-well plate culture). Briefly, the PBMCs are thawed in RPMI with<br>
30fjg/ml DNAse, washed once with PBS containing 1% human AB serum and resuspended in PBS/1% AB serum at a<br>
97<br>
concentration of 20x106cells/ml. The magnetic beads are washed 3 times with PBS/AB serum, added to the cells (140pl<br>
beads/20x106 cells) and incubated for 1 hour at 4°C with continuous mixing. The beads and cells are washed 4x with<br>
PBS/AB serum to remove the nonadherent cells and resuspended at 100x106 cells/ml (based on the original cell number) in<br>
PBS/AB serum containing 100µl/ml detacha-bead® reagent and 30 µg/ml DNAse. The mixture is incubated for 1 hour at<br>
room temperature with continuous mixing. The beads are washed again with PBS/AB/DNAse to collect the CD8+ T-cells.<br>
The DC are collected and centrifuged at 1300 rpm for 5-7 minutes, washed once with PBS with 1% BSA, counted and<br>
pulsed with 40pg/ml of peptide at a cell concentration of 1-2x106/ml in the presence of 3|jg/ml ß2- microglobulin for 4 hours<br>
at 20°C The DC are then irradiated (4,200 rads), washed 1 time with medium and counted again.<br>
[0462] Setting up induction cultures: 0.25 ml cytokine-generated DC (at 1x105 cells/ml) are co-cultured with 0.25ml<br>
cf CDS* T-cells (at 2x106 cell/ml) in each well of a 48-well plate in the presence of 10 µg/ml of IL-7. Recombinant human<br>
L-1C is added the next day at a final concentration of 10 ng/ml and rhuman IL-2 is added 48 hours later at 10 lU/ml.<br>
[0463] Restimulation of the induction cultures with peptide-pulsed adherent cells: Seven and fourteen days after the<br>
primary induction, the cells are restimulated with peptide-pulsed adherent cells. The PBMCs are thawed and washed twice<br>
with RPMI and DNAse. The cells are resuspended at 5x10s cells/ml and irradiated at -4200 rads. The PBMCs are plated<br>
at 2x10 m 0.5 ml complete medium per well and incubated for 2 hours at 37°C. The plates are washed twice with RPMI by<br>
tapping the plate gently to remove the nonadherent cells and the adherent cells pulsed with 10|jg/ml of peptide in the<br>
presence of 3 µg/ml ß2 microglobulin in 0.25ml RPMI/5%AB per well for 2 hours at 37°C. Peptide solution from each well is<br>
aspirated and the wells are washed once with RPMI. Most of the media is aspirated from the induction cultures (CD8+<br>
cells) and brought to 0.5 ml with fresh media. The cells are then transferred to the wells containing the peptide-puised<br>
adherent cells. Twenty four hours later recombinant human IL-10 is added at a final concentration of 10 ng/ml and<br>
recombinant human IL2 is added the next day and again 2-3 days later at 50IU/ml (Tsai et al., Critical Reviews in<br>
Immunology 18(1-2):65-75,1998). Seven days later, the cultures are assayed for CTL activity in a 51Cr release assay. In<br>
some experiments the cultures are assayed for peptide-specific recognition in the in situ IFNy ELISA at the time of the<br>
second restimulation followed by assay of endogenous recognition 7 days later. After expansion, activity is measured in<br>
both assays for a side-by-side comparison.<br>
Measurement of CTL lytic activity by 51Cr release.<br>
[0464] Seven days after the second restimulation, cytotoxicity is determined in a standard (5 hr) 61Cr release assay<br>
by assaying individual wells at a single E:T. Peptide-pulsed targets are prepared by incubating the cells with 10ug/ml<br>
peptide overnight at 37°C.<br>
[0465] Adherent target cells are removed from culture flasks with trypsin-EDTA. Target cells are labeled with<br>
200µCi of 51Cr sodium chromate (Dupont, Wilmington, DE) for 1 hour at 37°C. Labeled target cells are resuspended at 106<br>
per ml and diluted 1:10 with K562 cells at a concentration of 3.3x106/ml (an NK-sensitive erythroblastoma cell line used to<br>
reduce non-specific lysis). Target cells (100 µl) and effectors (100µl) are plated in 96 well round-bottom plates and<br>
incubated for 5 hours at 37°C. At that time, 100 µl of supernatant are collected from each well and percent lysis is<br>
determined according to the formula:<br>
[(cpm of the test sample- cpm of the spontaneous 51Cr release sample)/(cpm of the maximal 51Cr release sample-<br>
cpm of the spontaneous 51Cr release sample)] x 100.<br>
[0466] Maximum and spontaneous release are determined by incubating the labeled targets with 1 % Triton X-100<br>
and media alone, respectively. A positive culture is defined as one in which the specific lysis (sample- background) is 10%<br><br>
or higher in the case of individual wells and is 15% or more at the two highest E:T ratios when expanded cultures are<br>
asstayed.<br>
In situ Measurement of Human IFNy Production as an Indicator of Peptide-specific and<br>
Endogenous Recognition<br>
[0467] Immulon 2 plates are coated with mouse anti-human IFNy monoclonal antibody (4 µg /ml 0.1 M NaHCCO3,<br>
pH8.2) overnight at 4°C. The plates are washed with Ca2+, Mg2+-free PBS/0.05% Tween 20 and blocked with PBS/10%<br>
FCS for two hours, after which the CTLs (100 µl/well) and targets (100 µl/well) are added to each well, leaving empty wells<br>
for the standards and blanks (which received media only). The target cells, either peptide-pulsed or endogenous targets,<br>
are used at a concentration of 1x106 cells/ml. The plates are incubated for 48 hours at 37°C with 5% CO2.<br>
[0468] Recombinant human IFN-gamma is added to the standard wells starting at 400 µg or 1200µg/100<br>
microliter/well and the plate incubated for two hours at 37°C. The plates are washed and 100 µl of biotinylated mouse anti-<br>
human IFN-gamma monoclonal antibody (2 microgram/ml in PBS/3%FCS/0.05% Tween 20) are added and incubated for 2<br>
hours at room temperature. After washing again, 100 microliter HRP-streptavidin (1:4000) are added and the plates<br>
incubated for one hour at room temperature. The plates are then washed 6x with wash buffer, 100 microliter/well<br>
developing solution (TMB 1:1) are added, and the plates allowed to develop for 5-15 minutes. The reaction is stopped with<br>
50 microliter/well 1M H3PO4 and read at OD450. A culture is considered positive if it measured at least 50 µg of IFN-<br>
gamma/well above background and is twice the background level of expression.<br>
CTL Expansion.<br>
[0469] Those cultures that demonstrate specific lytic activity against peptide-pulsed targets and/or tumor targets are<br>
expanded over a two week period with anti-CD3. Briefly, 5x104 CD8+ cells are added to a T25 flask containing the<br>
following: 1x10s6 irradiated (4,200 rad) PBMC (autologous or allogeneic) per ml, 2x10s irradiated (8,000 rad) EBV-<br>
transformed cells per ml, and OKT3 (anti-CD3) at 30ng per ml in RPMI-1640 containing 10% (v/v) human AB serum, non-<br>
essential amino acids, sodium pyruvate, 25µM 2-mercaptoethanol, L-glutamine and penicillin/streptomycin. Recombinant<br>
human IL2 is added 24 hours later at a final concentration of 200IU/ml and every three days thereafter with fresh media at<br>
50IU/ml. The cells are split if the cell concentration exceeds 1x106/ml and the cultures are assayed between days 13 and<br>
15 at E:T ratios of 30,10, 3 and 1:1 in the 51Cr release assay or at 1x106/ml in the in situ IFNy assay using the same targets<br>
as before the expansion.<br>
[0470] Cultures are expanded in the absence of anti-CD3* as follows. Those cultures that demonstrate specific lytic<br>
activity against peptide and endogenous targets are selected and 5x104 CD8+ cells are added to a T25 flask containing the<br>
following: 1x105 autologous PBMC per ml which have been peptide-pulsed with 10 µg/ml peptide for two hours at 37°C and<br>
irradiated (4,200 rad); 2x10s irradiated (8,000 rad) EBV-transformed cells per ml RPMI-1640 containing 10%(v/v) human AB<br>
serum, non-essential AA, sodium pyruvate, 25mM 2-ME, L-glutamine and gentamicin.<br>
Immunoaenicity of A2 supermotif-bearinq peptides<br>
[0471 ] A2-supermotif cross-reactive binding peptides are tested in the cellular assay for the ability to induce peptide-<br>
specific CTL in normal individuals. In this analysis, a peptide is typically considered to be an epitope if it induces peptide-<br>
specific CTLs in at least individuals, and preferably, also recognizes the endogenously expressed peptide.<br><br>
[0472] immunogenicity can also be confirmed using PBMCs isolated from patients bearing a tumor that expresses<br>
PSCA. Briefly, PBMCs are isolated from patients, re-stimulated with peptide-pulsed monocytes and assayed for the ability<br>
to recognize peptide-pulsed target cells as well as transfected cells endogenously expressing the antigen.<br>
Evaluation of A*03/A11 immunogenicity<br>
[0473] HLA-A3 supermotif-bearing cross-reactive binding peptides are also evaluated for immunogenicity using<br>
methodology analogous for that used to evaluate the immunogenicity of the HLA-A2 supermotif peptides.<br>
Evaluation of B7 immunogenicity<br>
[0474] Immunogenicity screening of the B7-supertype cross-reactive binding peptides identified as set forth herein<br>
are confirmed in a manner analogous to the confirmation of A2-and A3-supermotif-bearing peptides.<br>
[0475] Peptides bearing other supermotifs/motifs, e.g., HLA-A1, HLA-A24 efc. are also confirmed using similar<br>
methodology.<br>
Example 15: Implementation of the Extended Supermotif to Improve the Binding<br>
Capacity of Native Epitopes by Creating Analogs<br>
[0476] HLA motifs and supermotifs (comprising primary and/or secondary residues) are useful in the identification<br>
and preparation of highly cross-reactive native peptides, as demonstrated herein. Moreover, the definition of HLA motifs<br>
and supermotifs also allows one to engineer highly cross-reactive epitopes by identifying residues within a native peptide<br>
sequence which can be analoged to confer upon the peptide certain characteristics, e.g. greater cross-reactivity within the<br>
group of HLA molecules that comprise a supertype, and/or greater binding affinity for some or all of those HLA molecules.<br>
Examples of analoging peptides to exhibit modulated binding affinity are set forth in this example.<br>
Analoging at Primary Anchor Residues<br>
[0477] Peptide engineering strategies are implemented to further increase the cross-reactivity of the epitopes. For<br>
example, the main anchors of A2-supermotif-bearing peptides are altered, for example, to introduce a preferred L, I, V, or M<br>
at position 2, and I or V at the C-terminus.<br>
[0478] To analyze the cross-reactivity of the analog peptides, each engineered analog is initially tested for binding to<br>
the prototype A2 supertype allele A*0201, then, if A*0201 binding capacity is maintained, for A2-supertype cross-reactivity.<br>
[0479] Alternatively, a peptide is confirmed as binding one or all supertype members and then analoged to modulate<br>
binding affinity to any one (or more) of the supertype members to add population coverage.<br>
[0480] The selection of analogs for immunogenicity in a cellular screening analysis is typically further restricted by<br>
the capacity of the parent wild type (WT) peptide to bind at least weakly, i.e., bind at an ICso of 5000nM or less, to three of<br>
more A2 supertype alleles. The rationale for this requirement is that the WT peptides must be present endogenously in<br>
sufficient quantity to be biologically relevant. Analoged peptides have been shown to have increased immunogenicity and<br>
cross-reactivity by T cells specific for the parent epitope (see, e.g., Parkhurst et al., J. Immunol. 157:2539,1996; and Pogue<br>
et al., Proc. Natl. Acad. Set USA 92:8166,1995).<br>
[0481] In the cellular screening of these peptide analogs, it is important to confirm that analog-specific CTLs are also<br>
able to recognize the wild-type peptide and, when possible, target cells that endogenously express the epitope.<br>
Analoging of HLA-A3 and B7-supermotif-bearinq peptides<br>
[0482] Analogs of HLA-A3 supermotif-bearing epitopes are generated using strategies similar to those employed in<br>
analoging HLA-A2 supermotif-bearing peptides. For example, peptides binding to 3/5 of the A3-supertype molecules are<br>
engineered at primary anchor residues to possess a preferred residue (V, S, M, or A) at position 2.<br>
[0483] The analog peptides are then tested for the ability to bind A*03 and A*11 (prototype A3 supertype alleles).<br>
Those peptides that demonstrate 
[0484] Similarly to the A2- and A3- motif bearing peptides, peptides binding 3 or more B7-supertype alleles can be<br>
improved, where possible, to achieve increased cross-reactive binding or greater binding affinity or binding half life. B7-<br>
supermotif-bearing peptides are, for example, engineered to possess a preferred residue (V, I, L, or F) at the C-terminal<br>
primary anchor position, as demonstrated by Sidney et al. (J. Immunol. 157:3480-3490,1996).<br>
[0485] Analoging at primary anchor residues of other motif and/or supermotif-bearing epitopes is performed in a like<br>
manner.<br>
[0486] The analog peptides are then be confirmed for immunogenicity, typically in a cellular screening assay. Again,<br>
it is generally important to demonstrate that analog-specific CTLs are also able to recognize the wild-type peptide and,<br>
when possible, targets that endogenously express the epitope.<br>
Analoqinq at Secondary Anchor Residues<br>
[0487] Moreover, HLA supermotifs are of value in engineering highly cross-reactive peptides and/or peptides that<br>
bind HLA molecules with increased affinity by identifying particular residues at secondary anchor positions that are<br>
associated with such properties. For example, the binding capacity of a B7 supermotif-bearing peptide with an F residue at<br>
position 1 is analyzed. The peptide is then analoged to, for example, substitute L for F at position 1. The analoged peptide<br>
is evaluated for increased binding affinity, binding half life and/or increased cross-reactivity. Such a procedure identifies<br>
analoged peptides with enhanced properties.<br>
[0488] Engineered analogs with sufficiently improved binding capacity or cross-reactivity can also be tested for<br>
immunogenicity in HLA-B7-transgenic mice, following for example, IFA immunization or lipopeptide immunization.<br>
Analoged peptides are additionally tested for the ability to stimulate a recall response using PBMC from patients with<br>
PSCA-expressing tumors.<br>
Other analoqinq strategies<br>
[0489] Another form of peptide analoging, unrelated to anchor positions, involves the substitution of a cysteine with<br>
a-amino butyric acid. Due to its chemical nature, cysteine has the propensity to form disulfide bridges and sufficiently alter<br>
the peptide structurally so as to reduce binding capacity. Substitution of a-amino butyric acid for cysteine not only<br>
alleviates this problem, but has been shown to improve binding and crossbinding capabilities in some instances (see, e.g.,<br>
the review by Sette et al., In: Persistent Viral Infections, Eds. R. Ahmed and I. Chen, John Wiley &amp; Sons, England, 1999).<br>
[0490] Thus, by the use of single amino acid substitutions, the binding properties and/or cross-reactivity of peptide<br>
ligands for HLA supertype molecules can be modulated.<br>
Example 16: Identification and confirmation of PSCA-derived sequences with HLA-DR<br>
binding motifs<br>
[0491 ] Peptide epitopes bearing an HLA class II supermotif or motif are identified and confirmed as outlined below<br>
using methodology similar to that described for HLA Class I peptides.<br>
Selection of HLA-DR-supermotif-bearinq epitopes.<br>
[0492] To identify PSCA-derived, HLA class IIHTL epitopes, a PSCA antigen is analyzed for the presence of<br>
sequences bearing an HLA-DR-motif or supermotif. Specifically, 15-mer sequences are selected comprising a DR-<br>
supermotif, comprising a 9-mer core, and three-residue N- and C-terminal flanking regions (15 amino acids total).<br>
[0493] Protocols for predicting peptide binding to OR molecules have been developed (Southwood et al., J.<br>
Immunol. 160:3363-3373,1998). These protocols, specific for individual DR molecules, allow the scoring, and ranking, of<br>
9-mer core regions. Each protocol not only scores peptide sequences for the presence of DR-supermotif primary anchors<br>
(i.e., at position 1 and position 6) within a 9-mer core, but additionally evaluates sequences for the presence of secondary<br>
anchors. Using allele-specific selection tables (see, e.g., Southwood et al., ibid.), it has been found that these protocols<br>
efficiently select peptide sequences with a high probability of binding a particular DR molecule. Additionally, it has been<br>
found that performing these protocols in tandem, specifically those for DR1, DR4w4, and DR7, can efficiently select DR<br>
cross-reactive peptides.<br>
[0494] The PSCA-derived peptides identified above are tested for their binding capacity for various common HLA-<br>
DR molecules. All peptides are initially tested for binding to the DR molecules in the primary panel: DR1, DR4w4, and<br>
DR7. Peptides binding at least two of these three DR molecules are then tested for binding to DR2w2 ß1, DR2w2 ß2,<br>
DR6w19, and DR9 molecules in secondary assays. Finally, peptides binding at least two of the four secondary panel DR<br>
molecules, and thus cumulatively at least four of seven different DR molecules, are screened for binding to DR4w15,<br>
DR5w11, and DR8w2 molecules in tertiary assays. Peptides binding at least seven of the ten DR molecules comprising the<br>
primary, secondary, and tertiary screening assays are considered cross-reactive DR binders. PSCA-derived peptides<br>
found to bind common HLA-DR alleles are of particular interest.<br>
Selection of DR3 motif peptides<br>
[0495] Because HLA-DR3 is an allele that is prevalent in Caucasian, Black, and Hispanic populations, DR3 binding<br>
capacity is a relevant criterion in the selection of HTL epitopes. Thus, peptides shown to be candidates may also be<br>
assayed for their DR3 binding capacity. However, in view of the binding specificity of the DR3 motif, peptides binding only<br>
to DR3 can also be considered as candidates for inclusion in a vaccine formulation.<br>
[0496] To efficiently identify peptides that bind DR3, target PSCA antigens are analyzed for sequences carrying one<br>
of the two DR3-specific binding motifs reported by Geluk et al. [J. Immunol. 152:5742-5748,1994). The corresponding<br>
peptides are then synthesized and confirmed as having the ability to bind DR3 with an affinity of 1 µM or better, i.e., less<br>
than 1 µM. Peptides are found that meet this binding criterion and qualify as HLA class II high affinity binders.<br>
[0497] DR3 binding epitopes identified in this manner are included in vaccine compositions with DR supermotif-<br>
bearing peptide epitopes.<br>
[0498] Similarly to the case of HLA class I motif-bearing peptides, the class II motif-bearing peptides are analoged to<br>
cave affinity or cross-reactivity. For example, aspartic acid at position 4 of the 9-mer core sequence is an optimal<br>
residue for DR3 binding, and substitution for that residue often improves DR 3 binding.<br>
Example 17: Immunoqenicitv of PSCA-derived HTL epitopes<br>
[0499] This example determines immunogenic DR supermotif- and DR3 motif-bearing epitopes among those<br>
identified using the methodology set forth herein.<br>
[0500] Immunogenicity of HTL epitopes are confirmed in a manner analogous to the determination of<br>
immunogenicity of CTL epitopes, by assessing the ability to stimulate HTL responses and/or by using appropriate<br>
transgenic mouse models. Immunogenicity is determined by screening for: 1.) in vitro primary induction using normal<br>
PBMC or 2.) recall responses from patients who have PSCA-expressing tumors.<br>
Example 18: Calculation of phenotvpic frequencies of HLA-supertvpes in various ethnic<br>
backgrounds to determine breadth of population coverage<br>
[0501] This example illustrates the assessment of the breadth of population coverage of a vaccine composition<br>
comprised of multiple epitopes comprising multiple supermotifs and/or motifs.<br>
[0502] In order to analyze population coverage, gene frequencies of HLA alleles are determined. Gene frequencies<br>
for each HLA allele are calculated from antigen or allele frequencies utilizing the binomial distribution formulae gf=1-<br>
(SQRT(1-af)) (see, e.g., Sidney et al., Human Immunol. 45:79-93,1996). To obtain overall phenotypic frequencies,<br>
cumulative gene frequencies are calculated, and the cumulative antigen frequencies derived by the use of the inverse<br>
formula [af=1-(1-Cgf)2].<br>
[0503] Where frequency data is not available at the level of DNA typing, correspondence to the serologically defined<br>
antigen frequencies is assumed. To obtain total potential supertype population coverage no linkage disequilibrium is<br>
assumed, and only alleles confirmed to belong to each of the supertypes are included (minimal estimates). Estimates of<br>
total potential coverage achieved by inter-loci combinations are made by adding to the A coverage the proportion of the<br>
non-A covered population that could be expected to be covered by the B alleles considered (e.g., total=A+B*(1-A)).<br>
Confirmed members of the A3-like supertype are A3, A11, A31, A*3301, and A*6801. Although the A3-like supertype may<br>
also include A34, A66, and A*7401, these alleles were not included in overall frequency calculations. Likewise, confirmed<br>
members of the A2-like supertype family are A*0201, A*0202, A*0203, A*0204, A*0205, A*0206, A*0207, A*6802, and<br>
A*6901. Finally, the B7-like supertype-confirmed alleles are: B7, B*3501-03, B51, B*5301, B*5401, B*5501-2, B*5601,<br>
B*6701, and B*7801 (potentially also B*1401, B*3504-06, B*4201, and B*5602).<br>
[0504] Population coverage achieved by combining the A2-, A3- and B7-supertypes is approximately 86% in five<br>
major ethnic groups. Coverage may be extended by including peptides bearing the A1 and A24 motifs. On average, A1 is<br>
present in 12% and A24 in 29% of the population across five different major ethnic groups (Caucasian, North American<br>
Black, Chinese, Japanese, and Hispanic). Together, these alleles are represented with an average frequency of 39% in<br>
these same ethnic populations. The total coverage across the major ethnicities when A1 and A24 are combined with the<br>
coverage of the A2-, A3- and B7-supertype alleles is &gt;95%, see, e.g., Table IV (G). An analogous approach can be used to<br>
estimate population coverage achieved with combinations of class II motif-bearing epitopes.<br>
[0505] Immunogenicity studies in humans (e.g., Bertoni et al., J. Clin. Invest. 100:503,1997; Doolan et al., Immunity<br>
7:97,1997; and Threlkeld et al., J. Immunol. 159:1648,1997) have shown that highly cross-reactive binding peptides are<br><br>
almost always recognized as epitopes. The use of highly cross-reactive binding peptides is an important selection criterion<br>
iydentifying candidate epitopes for inclusion in a vaccine that is immunogenic in a diverse population.<br>
[05061 With a sufficient number of epitopes (as disclosed herein and from the art), an average population coverage<br>
is predicted to be greater than 95% in each of five major ethnic populations. The game theory Monte Carlo simulation<br>
analysis, which is known in the art (see e.g., Osborne, M.J. and Rubinstein, A. "A course in game theory" MIT Press, 1994),<br>
can be used to estimate what percentage of the individuals in a population comprised of the Caucasian, North American<br>
Black, Japanese, Chinese, and Hispanic ethnic groups would recognize the vaccine epitopes described herein. A preferred<br>
percentage is 90%. A more preferred percentage is 95%.<br>
Example 19: CTL Recognition Of Endoqenously Processed Antigens After Priming<br>
[0507] This example confirms that CTL induced by native or analoged peptide epitopes identified and selected as<br>
described herein recognize endogenously synthesized, i.e., native antigens.<br>
[0508] Effector cells isolated from transgenic mice that are immunized with peptide epitopes, for example HLA-A2<br>
supermotif-bearing epitopes, are re-stimulated in vitro using peptide-coated stimulator cells. Six days later, effector cells<br>
are assayed for cytotoxicity and the cell lines that contain peptide-specific cytotoxic activity are further re-stimulated. An<br>
additional six days later, these cell lines are tested for cytotoxic activity on 51Cr labeled Jurkat-A2.1/Kb target cells in the<br>
absence or presence of peptide, and also tested on 51Cr labeled target cells bearing the endogenously synthesized antigen,<br>
i.e. cells that are stably transfected with PSCA expression vectors.<br>
[0509] The results demonstrate that CTL lines obtained from animals primed with peptide epitope recognize<br>
endogenously synthesized PSCA antigen. The choice of transgenic mouse model to be used for such an analysis depends<br>
upon the epitope(s) that are being evaluated. In addition to HLA-A*0201/Kb transgenic mice, several other transgenic<br>
mouse models including mice with human A11, which may also be used to evaluate A3 epitopes, and B7 alleles have been<br>
characterized and others (e.g., transgenic mice for HLA-A1 and A24) are being developed. HLA-DR1 and HLA-DR3 mouse<br>
models have also been developed, which may be used to evaluate HTL epitopes.<br>
Example 20: Activity Of CTL-HTL Conjugated Epitopes In Transgenic Mice<br>
[0510] This example illustrates the induction of CTLs and HTLs in transgenic mice, by use of a PSCA-derived CTL<br>
and HTL peptide vaccine compositions. The vaccine composition used herein comprise peptides to be administered to a<br>
patient with a PSCA-expressing tumor. The peptide composition can comprise multiple CTL and/or HTL epitopes. The<br>
epitopes are identified using methodology as described herein. This example also illustrates that enhanced<br>
immunogenicity can be achieved by inclusion of one or more HTL epitopes in a CTL vaccine composition; such a peptide<br>
composition can comprise an HTL epitope conjugated to a CTL epitope. The CTL epitope can be one that binds to multiple<br>
HLA family members at an affinity of 500 nM or less, or analogs of that epitope. The peptides may be lipidated, if desired.<br>
[0511 ] Immunization procedures: Immunization of transgenic mice is performed as described (Alexander et al., J.<br>
Immunol. 159:4753-4761,1997). For example, A2/K" mice, which are transgenic for the human HLA A2.1 allele and are<br>
used to confirm the immunogenicity of HLA-A*0201 motif- or HLA-A2 supermotif-bearing epitopes, and are primed<br>
subcutaneously (base of the tail) with a 0.1 ml of peptide in Incomplete Freund's Adjuvant, or if the peptide composition is a<br>
lipidated CTL/HTL conjugate, in DMSO/saline, or if the peptide composition is a polypeptide, in PBS or Incomplete Freund's<br>
Adjuvant. Seven days after priming, splenocytes obtained from these animals are restimulated with syngenic irradiated<br>
LPS-activated lymphoblasts coated with peptide.<br><br>
[0512] Cell lines: Target cells for peptide-specific cytotoxicity assays are Jurkat cells transfected with the HLA-<br>
Kb chimeric gene (e.g., Vitiello et al., J. Exp. Med. 173:1007,1991)<br>
[0513] In vitro CTL activation: One week after priming, spleen cells (30x106 cells/flask) are co-cultured at 37°C with<br>
syngeneic, irradiated (3000 rads), peptide coated lymphoblasts (10x106 cells/flask) in 10 ml of culture medium/T25 flask.<br>
After six days, effector cells are harvested and assayed for cytotoxic activity.<br>
[0514] ,4ssay for cytotoxic activity: Target cells (1.0 to 1.5x106) are incubated at 37° C in the presence of 200 µl of<br>
51Cr. After 60 minutes, cells are washed three times and resuspended in R10 medium. Peptide is added where required at<br>
a concentration of 1 µg/ml. For the assay, 104 51Cr-labeled target cells are added to different concentrations of effector<br>
cells (final volume of 200 pi) in U-bottom 96-well plates. After a six hour incubation period at 37°C, a 0.1 ml aliquot of<br>
supernatant is removed from each well and radioactivity is determined in a Micromedic automatic gamma counter. The<br>
percent specific lysis is determined by the formula: percent specific release = 100 x (experimental release - spontaneous<br>
release)/(maximum release - spontaneous release). To facilitate comparison between separate CTL assays run under the<br>
same conditions, % 51Cr release data is expressed as lytic units/106 cells. One lytic unit is arbitrarily defined as the number<br>
of effector cells required to achieve 30% lysis of 10,000 target cells in a six hour 51Cr release assay. To obtain specific lytic<br>
units/106, the lytic units/106 obtained in the absence of peptide is subtracted from the lytic units/106 obtained in the<br>
presence of peptide. For example, if 30% 51Cr release is obtained at the effector (E): target (T) ratio of 50:1 (i.e., 5x105<br>
effector cells for 10,000 targets) in the absence of peptide and 5:1 (i.e., 5x104 effector cells for 10,000 targets) in the<br>
presence of peptide, the specific lytic units would be: [(1/50,000)-(1/500,000)] x 106 = 18 LU.<br>
[0515] The results are analyzed to assess the magnitude of the CTL responses of animals injected with the<br>
immunogenic CTL/HTL conjugate vaccine preparation and are compared to the magnitude of the CTL response achieved<br>
using, for example, CTL epitopes as outlined above in the Example entitled "Confirmation of Immunogenicity." Analyses<br>
similar to this may be performed to confirm the immunogenicity of peptide conjugates containing multiple CTL epitopes<br>
and/or multiple HTL epitopes. In accordance with these procedures, it is found that a CTL response is induced, and<br>
concomitantly that an HTL response is induced upon administration of such compositions.<br>
Example 21: Selection of CTL and HTL epitopes for inclusion in a PSCA-specific<br>
vaccine.<br>
[0516] This example illustrates a procedure for selecting peptide epitopes for vaccine compositions of the invention.<br>
The peptides in the composition can be in the form of a nucleic acid sequence, either single or one or more sequences (i.e.,<br>
minigene) that encodes peptide(s), or can be single and/or polyepitopic peptides.<br>
[0517] The following principles are utilized when selecting a plurality of epitopes for inclusion in a vaccine<br>
composition. Each of the following principles is balanced in order to make the selection.<br>
[0518] Epitopes are selected which, upon administration, mimic immune responses that are correlated with PSCA<br>
clearance. The number of epitopes used depends on observations of patients who spontaneously clear PSCA. For<br>
example, if it has been observed that patients who spontaneously clear PSCA-expressing cells generate an immune<br>
response to at least three (3) epitopes from PSCA antigen, then at least three epitopes should be included for HLA class I.<br>
A similar rationale is used to determine HLA class II epitopes.<br>
[0519] Epitopes are often selected that have a binding affinity of an IC50 of 500 nM or less for an HLA class I<br>
molecule, or for class II, an IC50 of 1000 nM or less; or HLA Class I peptides with high binding scores from the BIMAS web<br>
site, at URL bimas.dcrt.nih.gov/.<br><br>
[0520] In order to achieve broad coverage of the vaccine through out a diverse population, sufficient supermotif<br>
bearing peptides, or a sufficient array of allele-specific motif bearing peptides, are selected to give broad population<br>
coverage. In one embodiment, epitopes are selected to provide at least 80% population coverage. A Monte Carlo<br>
analysis, a statistical evaluation known in the art, can be employed to assess breadth, or redundancy, of population<br>
coverage.<br>
[0521] When creating polyepitopic compositions, or a minigene that encodes same, it is typically desirable to<br>
generate the smallest peptide possible that encompasses the epitopes of interest. The principles employed are similar, if<br>
not the same, as those employed when selecting a peptide comprising nested epitopes. For example, a protein sequence<br>
for the vaccine composition is selected because it has maximal number of epitopes contained within the sequence, i.e., it<br>
has a high concentration of epitopes. Epitopes may be nested or overlapping (i.e., frame shifted relative to one another).<br>
For example, with overlapping epitopes, two 9-mer epitopes and one 10-mer epitope can be present in a 10 amino acid<br>
peptide. Each epitope can be exposed and bound by an HLA molecule upon administration of such a peptide. A multi-<br>
epitopic, peptide can be generated synthetically, recombinantly, or via cleavage from the native source. Alternatively, an<br>
analog can be made of this native sequence, whereby one or more of the epitopes comprise substitutions that alter the<br>
cross-reactivity and/or binding affinity properties of the polyepitopic peptide. Such a vaccine composition is administered<br>
for therapeutic or prophylactic purposes. This embodiment provides for the possibility that an as yet undiscovered aspect<br>
of immune system processing will apply to the native nested sequence and thereby facilitate the production of therapeutic<br>
or prophylactic immune response-inducing vaccine compositions. Additionally such an embodiment provides for the<br>
possibility of motif-bearing epitopes for an HLA makeup that is presently unknown. Furthermore, this embodiment (absent<br>
the creating of any analogs) directs the immune response to multiple peptide sequences that are actually present in PSCA,<br>
thus avoiding the need to evaluate any junctional epitopes. Lastly, the embodiment provides an economy of scale when<br>
producing nucleic acid vaccine compositions. Related to this embodiment, computer programs can be derived in<br>
accordance with principles in the art, which identify in a target sequence, the greatest number of epitopes per sequence<br>
length.<br>
[0522] A vaccine composition comprised of selected peptides, when administered, is safe, efficacious, and elicits an<br>
immune response similar in magnitude to an immune response that controls or clears cells that bear or overexpress PSCA.<br>
Example 22: Construction of "Minigene" Multi-Epitope DNA Plasmids<br>
[0523] This example discusses the construction of a minigene expression plasmid. Minigene plasmids may, of<br>
course, contain various configurations of B cell, CTL and/or HTL epitopes or epitope analogs as described herein.<br>
[0524] A minigene expression plasmid typically includes multiple CTL and HTL peptide epitopes. In the present<br>
example, HLA-A2, -A3, -B7 supermotif-bearing peptide epitopes and HLA-A1 and -A24 motif-bearing peptide epitopes are<br>
used in conjunction with DR supermotif-bearing epitopes and/or DR3 epitopes. HLA class I supermotif or motif-bearing<br>
peptide epitopes derived PSCA, are selected such that multiple supermotifs/motifs are represented to ensure broad<br>
population coverage. Similarly, HLA class II epitopes are selected from PSCA to provide broad population coverage, i.e.<br>
both HLA DR-1-4-7 supermotif-bearing epitopes and HLA DR-3 motif-bearing epitopes are selected for inclusion in the<br>
minigene construct. The selected CTL and HTL epitopes are then incorporated into a minigene for expression in an<br>
expression vector.<br>
[0525] Such a construct may additionally include sequences that direct the HTL epitopes to the endoplasmic<br>
reticulum. For example, the li protein may be fused to one or more HTL epitopes as described in the art, wherein the CLIP<br><br>
sequence of the li protein is removed and replaced with an HLA class li epitope sequence so that HLA class II epitope is<br>
ditected to the endoplasmic reticulum, where the epitope binds to an HLA class II molecules.<br>
[0526] This example illustrates the methods to be used for construction of a minigene-bearing expression plasmid.<br>
Other expression vectors that may be used for minigene compositions are available and known to those of skill in the art.<br>
[0527] The minigene DNA plasmid of this example contains a consensus Kozak sequence and a consensus murine<br>
kappa Ig-light chain signal sequence followed by CTL and/or HTL epitopes selected in accordance with principles disclosed<br>
herein. The sequence encodes an open reading frame fused to the Myc and His antibody epitope tag coded for by the<br>
pcDNA 3 1 Myc-His vector.<br>
[0528] Overlapping oligonucleotides that can, for example, average about 70 nucleotides in length with 15 nucleotide<br>
option are synthesized and HPLC-purified. The oligonucleotides encode the selected peptide epitopes as well as<br>
appropiate imker nucleotides, Kozak sequence, and signal sequence. The final multiepitope minigene is assembled by<br>
evending the overlapping oligonucleotides in three sets of reactions using PCR. A Perkin/Elmer 9600 PCR machine is<br>
used and a total of 30 cycles are performed using the following conditions: 95°C for 15 sec, annealing temperature (5°<br>
below the lowest calculated Tm of each primer pair) for 30 sec, and 72°C for 1 min.<br>
[0529] For example, a minigene is prepared as follows. For a first PCR reaction, 5 µg of each of two<br>
oponucleotides are annealed and extended: In an example using eight oligonucleotides, i.e., four pairs of primers,<br>
olgonucleotides 1+2, 3+4,5+6, and 7+8 are combined in 100 uJ reactions containing Pfu polymerase buffer (1x= 10 mM<br>
KCL 10 mM (NH4)2SO4, 20 mM Tris-chloride, pH 8.75, 2 mM MgSO4, 0.1% Triton X-100,100 ng/ml BSA), 0.25 mM each<br>
dtlTP. and 2.5 U of Pfu polymerase. The full-length dimer products are gel-purified, and two reactions containing the<br>
product of 1 +2 and 3+4, and the product of 5+6 and 7+8 are mixed, annealed, and extended for 10 cycles. Half of the two<br>
reactions are then mixed, and 5 cycles of annealing and extension carried out before flanking primers are added to amplify<br>
the full length product. The full-length product is gel-purified and cloned into pCR-blunt (Invitrogen) and individual clones<br>
are screened by sequencing.<br>
Example 23: The Plasmid Construct and the Degree to Which It Induces<br>
Immunogenicity.<br>
[0530] The degree to which a plasmid construct, for example a plasmid constructed in accordance with the previous<br>
Example, is able to induce immunogenicity is confirmed in vitro by determining epitope presentation by APC following<br>
transduction or transfection of the APC with an epitope-expressing nucleic acid construct. Such a study determines<br>
"antigenicity" and allows the use of human APC. The assay determines the ability of the epitope to be presented by the<br>
APC in a context that is recognized by a T cell by quantifying the density of epitope-HLA class I complexes on the cell<br>
surface. Quantitation can be performed by directly measuring the amount of peptide eluted from the APC (see, e.g., Sijts et<br>
a!., J. Immunol. 156:683-692,1996; Demotzet al., Nature 342:682-684,1989); or the number of peptide-HLA class I<br>
complexes can be estimated by measuring the amount of lysis or lymphokine release induced by diseased or transfected<br>
target cells, and then determining the concentration of peptide necessary to obtain equivalent levels of lysis or lymphokine<br>
release (see, e.g., Kageyamaet al., J. Immunol. 154:567-576,1995).<br>
[0531] Alternatively, immunogenicity is confirmed through in vivo injections into mice and subsequent in vitro<br>
assessment of CTL and HTL activity, which are analyzed using cytotoxicity and proliferation assays, respectively, as<br>
det alled e.g., in Alexander et al., Immunity 1:751-761,1994.<br>
[0532] For example, to confirm the capacity of a DNA minigene construct containing at least one HLA-A2 supermotif<br>
peptide to induce CTLs in vivo, HLA-A2.1/Kb transgenic mice, for example, are immunized intramuscularly with 100 µg of<br>
naked cDNA. As a means of comparing the level of CTLs induced by cDNA immunization, a control group of animals is<br>
also immunized with an actual peptide composition that comprises multiple epitopes synthesized as a single polypeptide as<br>
they would be encoded by the minigene.<br>
[0533] Splenocytes from immunized animals are stimulated twice with each of the respective compositions (peptide<br>
epitopes encoded in the minigene or the polyepitopic peptide), then assayed for peptide-specific cytotoxic activity in a 51Cr<br>
release assay. The results indicate the magnitude of the CTL response directed against the A2-restricted epitope, thus<br>
indicating the in vivo immunogenicity of the minigene vaccine and polyepitopic vaccine.<br>
[0534] It is, therefore, found that the minigene elicits immune responses directed toward the HLA-A2 supermotif<br>
peptide epitopes as does the polyepitopic peptide vaccine. A similar analysis is also performed using other HLA-A3 and<br>
HLA-B7 transgenic mouse models to assess CTL induction by HLA-A3 and HLA-B7 motif or supermotif epitopes, whereby<br>
it is also found that the minigene elicits appropriate immune responses directed toward the provided epitopes.<br>
[0535] To confirm the capacity of a class II epitope-encoding minigene to induce HTLs in vivo, DR transgenic mice,<br>
or for those epitopes that cross react with the appropriate mouse MHC molecule, l-Ab-restricted mice, for example, are<br>
immunized intramuscularly with 100 µg of plasmid DNA. As a means of comparing the level of HTLs induced by DNA<br>
immunization, a group of control animals is also immunized with an actual peptide composition emulsified in complete<br>
Freund's adjuvant. CD4+ T cells, i.e. HTLs, are purified from splenocytes of immunized animals and stimulated with each<br>
of the respective compositions (peptides encoded in the minigene). The HTL response is measured using a 3H-thymidine<br>
incorporation proliferation assay, (see, e.g., Alexander et al. Immunity 1:751-761,1994). The results indicate the<br>
magnitude of the HTL response, thus demonstrating the in vivo immunogenicity of the minigene.<br>
[0536] DNA minigenes, constructed as described in the previous Example, can also be confirmed as a vaccine in<br>
combination with a boosting agent using a prime boost protocol. The boosting agent can consist of recombinant protein<br>
(e.g., Barnett et al., Aids Res. and Human Retroviruses 14, Supplement 3:S299-S309,1998) or recombinant vaccinia, for<br>
example, expressing a minigene or DNA encoding the complete protein of interest (see, e.g., Hanke et al., Vaccine 16:439-<br>
445,1998; Sedegah et al., Proc. Natl. Acad. Sci USA 95:7648-53,1998; Hanke and McMichael, Immunol. Letters 66:177-<br>
181,1999; and Robinson et al., Nature Med. 5:526-34,1999).<br>
[0537] For example, the efficacy of the DNA minigene used in a prime boost protocol is initially evaluated in<br>
transgenic mice. In this example, A2.1/Kb transgenic mice are immunized IM with 100 µg of a DNA minigene encoding the<br>
immunogenic peptides including at least one HLA-A2 supermotif-bearing peptide. After an incubation period (ranging from<br>
3-9 weeks), the mice are boosted IP with 107 pfu/mouse of a recombinant vaccinia virus expressing the same sequence<br>
encoded by the DNA minigene. Control mice are immunized with 100 µg of DNA or recombinant vaccinia without the<br>
minigene sequence, or with DNA encoding the minigene, but without the vaccinia boost. After an additional incubation<br>
period of two weeks, splenocytes from the mice are immediately assayed for peptide-specific activity in an ELISPOT assay.<br>
Additionally, splenocytes are stimulated in vitro with the A2-restricted peptide epitopes encoded in the minigene and<br>
recombinant vaccinia, then assayed for peptide-specific activity in an alpha, beta and/or gamma IFN ELISA.<br>
[0538] It is found that the minigene utilized in a prime-boost protocol elicits greater immune responses toward the<br>
HLA-A2 supermotif peptides than with DNA alone. Such an analysis can also be performed using HLA-A11 or HLA-B7<br>
transgenic mouse models to assess CTL induction by HLA-A3 or HLA-B7 motif or supermotif epitopes. The use of prime<br>
boost protocols in humans is described below in the Example entitled "Induction of CTL Responses Using a Prime Boost<br>
protocol."<br>
Example 24: Peptide Compositions for Prophylactic Uses<br>
[0539] Vaccine compositions of the present invention can be used to prevent PSCA expression in persons who are<br>
at risk for tumors that bear this antigen. For example, a polyepitopic peptide epitope composition (or a nucleic acid<br>
comprising the same) containing multiple CTL and HTL epitopes such as those selected in the above Examples, which are<br>
also selected to target greater than 80% of the population, is administered to individuals at risk for a PSCA-associated<br>
tumor.<br>
[0540] For example, a peptide-based composition is provided as a single polypeptide that encompasses multiple<br>
epitopes. The vaccine is typically administered in a physiological solution that comprises an adjuvant, such as Incomplete<br>
Freunds Adjuvant. The dose of peptide for the initial immunization is from about 1 to about 50,000 µg , generally 100-5,000<br>
µg , for a 70 kg patient. The initial administration of vaccine is followed by booster dosages at 4 weeks followed by<br>
evaluation of the magnitude of the immune response in the patient, by techniques that determine the presence of epitope-<br>
specific CTL populations in a PBMC sample. Additional booster doses are administered as required. The composition is<br>
found to be both safe and efficacious as a prophylaxis against PSCA-associated disease.<br>
[0541 ] Alternatively, a composition typically comprising transfecting agents is used for the administration of a nucleic<br>
acid-based vaccine in accordance with methodologies known in the art and disclosed herein.<br>
Example 25: Polyepitopic Vaccine Compositions Derived from Native PSCA<br>
Sequences<br>
[0542] A native PSCA polyprotein sequence is analyzed, preferably using computet algorithms defined for each<br>
class I and/or class II supermotif or motif, to identify "relatively short" regions of the polyprotein that comprise multiple<br>
epitopes. The "relatively short" regions are preferably less in length than an entire native antigen. This relatively short<br>
sequence that contains multiple distinct or overlapping, "nested" epitopes can be used to generate a minigene construct.<br>
The construct is engineered to express the peptide, which corresponds to the native protein sequence. The "relatively<br>
short" peptide is generally less than 250 amino acids in length, often less than 100 amino acids in length, preferably less<br>
than 75 amino acids in length, and more preferably less than 50 amino acids in length. The protein sequence of the<br>
vaccine composition is selected because it has maximal number of epitopes contained within the sequence, i.e., it has a<br>
high concentration of epitopes. As noted herein, epitope motifs may be nested or overlapping {i.e., frame shifted relative to<br>
one another). For example, with overlapping epitopes, two 9-mer epitopes and one 10-mer epitope can be present in a 10<br>
amino acid peptide. Such a vaccine composition is administered for therapeutic or prophylactic purposes.<br>
[0543] The vaccine composition will include, for example, multiple CTL epitopes from PSCA antigen and at least one<br>
HTL epitope. This polyepitopic native sequence is administered either as a peptide or as a nucleic acid sequence which<br>
encodes the peptide. Alternatively, an analog can be made of this native sequence, whereby one or more of the epitopes<br>
comprise substitutions that alter the cross-reactivity and/or binding affinity properties of the polyepitopic peptide.<br>
[0544] The embodiment of this example provides for the possibility that an as yet undiscovered aspect of immune<br>
system processing will apply to the native nested sequence and thereby facilitate the production of therapeutic or<br>
prophylactic immune response-inducing vaccine compositions. Additionally, such an embodiment provides for the<br>
possibility of motif-bearing epitopes for an HLA makeup(s) that is presently unknown. Furthermore, this embodiment<br>
(feuding an analoged embodiment) directs the immune response to multiple peptide sequences that are actually present<br>
in native PSCA, thus avoiding the need to evaluate any junctional epitopes. Lastly, the embodiment provides an economy<br>
of scale when producing peptide or nucleic acid vaccine compositions.<br>
[0545] Related to this embodiment, computer programs are available in the art which can be used to identify in a<br>
target sequence, the greatest number of epitopes per sequence length.<br>
Example 26: Polyepitopic Vaccine Compositions from Multiple Antigens<br>
[0546] The PSCA peptide epitopes of the present invention are used in conjunction with epitopes from other target<br>
turner-associated antigens, to create a vaccine composition that is useful for the prevention or treatment of cancer that<br>
expresses PSCA and such other antigens. For example, a vaccine composition can be provided as a single polypeptide<br>
that incorporates multiple epitopes from PSCA as well as tumor-associated antigens that are often expressed with a target<br>
cancer associated with PSCA expression, or can be administered as a composition comprising a cocktail of one or more<br>
discrete epitopes. Alternatively, the vaccine can be administered as a minigene construct or as dendritic cells which have<br>
been leased with the peptide epitopes in vitro.<br>
Example 27: Use of peptides to evaluate an immune response<br>
[0547] Peptides of the invention may be used to analyze an immune response for the presence of specific<br>
antibodies, CTL or HTL directed to PSCA. Such an analysis can be performed in a manner described by Ogg et al.,<br>
Science 279:2103-2106,1998. In this Example, peptides in accordance with the invention are used as a reagent for<br>
diagnostic or prognostic purposes, not as an immunogen.<br>
[0548] In this example highly sensitive human leukocyte antigen tetrameric complexes ("tetramers") are used for a<br>
cross-sectional analysis of, for example, PSCA HLA-A*0201-specific CTL frequencies from HLA A*0201-positive individuals<br>
at different stages of disease or following immunization comprising a PSCA peptide containing an A*0201 motif. Tetrameric<br>
complexes are synthesized as described (Musey et a/., N. Engl. J. Med. 337:1267,1997). Briefly, purified HLA heavy chain<br>
(A*0201 in this example) and p2-microglobulin are synthesized by means of a prokaryotic expression system. The heavy<br>
chain is modified by deletion of the transmembrane-cytosolic tail and COOH-terminal addition of a sequence containing a<br>
BirA enzymatic biotinylation site. The heavy chain, ß2-microglobulin, and peptide are refolded by dilution. The 45-kD<br>
refolded product is isolated by fast protein liquid chromatography and then biotinylated by BirA in the presence of biotin<br>
(Sigma, St. Louis, Missouri), adenosine 5' triphosphate and magnesium. Streptavidin-phycoerythrin conjugate is added in a<br>
1:4 molar ratio, and the tetrameric product is concentrated to 1 mg/ml. The resulting product is referred to as tetramer-<br>
phycoerythrin.<br>
[0549] For the analysis of patient blood samples, approximately one million PBMCs are centrifuged at 300g for 5<br>
minutes and resuspended in 50 µl of cold phosphate-buffered saline. Tri-color analysis is performed with the tetramer-<br>
phycoerythrin, along with anti-CD8-Tricolor, and anti-CD38. The PBMCs are incubated with tetramer and antibodies on ice<br>
for 30 to 60 min and then washed twice before formaldehyde fixation. Gates are applied to contain &gt;99.98% of control<br>
samples. Controls for the tetramers include both A*0201-negative individuals and A*0201-positive non-diseased donors.<br>
The percentage of cells stained with the tetramer is then determined by flow cytometry. The results indicate the number of<br>
cells in the PBMC sample that contain epitope-restricted CTLs, thereby readily indicating the extent of immune response to<br>
the PSCA epitope, and thus the status of exposure to PSCA, or exposure to a vaccine that elicits a protective or therapeutic<br>
response.<br>
Example 28: Use of Peptide Epitopes to Evaluate Recall Responses<br>
[0550] The peptide epitopes of the invention are used as reagents to evaluate T cell responses, such as acute or<br>
recall responses, in patients. Such an analysis may be performed on patients who have recovered from PSCA-associated<br>
disease or who have been vaccinated with a PSCA vaccine.<br>
[0551] For example, the class I restricted CTL response of persons who have been vaccinated may be analyzed.<br>
The vaccine may be any PSCA vaccine. PBMC are collected from vaccinated individuals and HLA typed. Appropriate<br>
peptide epitopes of the invention that, optimally, bear supermotifs to provide cross-reactivity with multiple HLA supertype<br>
family members, are then used for analysis of samples derived from individuals who bear that HLA type.<br>
[0552] PBMC from vaccinated individuals are separated on Ficoll-Histopaque density gradients (Sigma Chemical<br>
Co., St. Louis, MO), washed three times in HBSS (GIBCO Laboratories), resuspended in RPMI-1640 (GIBCO Laboratories)<br>
supplemented with L-glutamine (2mM), penicillin (50U/ml), streptomycin (50 µg /ml), and Hepes (10mM) containing 10%<br>
heat-inactivated human AB serum (complete RPMI) and plated using microculture formats. A synthetic peptide comprising<br>
an epitope of the invention is added at 10 µg /ml to each well and HBV core 128-140 epitope is added at 1 µg /ml to each<br>
well as a source of T cell help during the first week of stimulation.<br>
[0553] in the microculture format, 4 x 105 PBMC are stimulated with peptide in 8 replicate cultures in 96-well round<br>
bottom plate in 100 ^I/well of complete RPMI. On days 3 and 10,100 pi of complete RPMI and 20 U/ml final concentration<br>
of rlL-2 are added to each well. On day 7 the cultures are transferred into a 96-well flat-bottom plate and restimulated with<br>
peptide, rlL-2 and 105 irradiated (3,000 rad) autologous feeder cells. The cultures are tested for cytotoxic activity on day<br>
14. A positive CTL response requires two or more of the eight replicate cultures to display greater than 10% specific 51Cr<br>
release, based on comparison with non-diseased control subjects as previously described (Rehermann, et al., Nature Med.<br>
2:1104,1108,1996; Rehermann et al, J. Clin. Invest 97:1655-1665,1996; and Rehermann et al. J. Clin. Invest. 98:1432-<br>
1440,1996).<br>
[0554] Target cell lines are autologous and allogeneic EBV-transformed B-LCL that are either purchased from the<br>
American Society for Histocompatibility and Immunogenetics (ASHI, Boston, MA) or established from the pool of patients<br>
as described (Guilhot, et al. J. Virol. 66:2670-2678,1992).<br>
[0555] Cytotoxicity assays are performed in the following manner. Target cells consist of eithet allogeneic HLA-<br>
matched or autologous EBV-transformed B lymphoblastoid cell line that are incubated overnight with the synthetic peptide<br>
epitope of the invention at 10 i^M, and labeled with 100 jaCi of 51Cr (Amersham Corp., Arlington Heights, IL) for 1 hour after<br>
which they are washed four times with HBSS.<br>
[0556] Cytolytic activity is determined in a standard 4-h, split well 51Cr release assay using U-bottomed 96 well<br>
plates containing 3,000 targets/well. Stimulated PBMC are tested at effector/target (E/T) ratios of 20-50:1 on day 14.<br>
Percent cytotoxicity is determined from the formula: 100 x [(experimental release-spontaneous release)/maximum release-<br>
spontaneous release)]. Maximum release is determined by lysis of targets by detergent (2% Triton X-100; Sigma Chemical<br>
Co., St. Louis, MO). Spontaneous release is 
[0557] The results of such an analysis indicate the extent to which HLA-restricted CTL populations have been<br>
stimulated by previous exposure to PSCA or a PSCA vaccine.<br>
[0558] Similarly, Class II restricted HTL responses may also be analyzed. Purified PBMC are cultured in a 96-well<br>
flat bottom plate at a density of 1.5x105 cells/well and are stimulated with 10 µg/ml synthetic peptide of the invention, whole<br>
PSCA antigen, or PHA. Cells are routinely plated in replicates of 4-6 wells for each condition. After seven days of culture,<br>
the medium is removed and replaced with fresh medium containing 10U/ml IL-2. Two days later, 1 pCi 3H-thymidine is<br>
added to each well and incubation is continued for an additional 18 hours. Cellular DNA is then harvested on glass fiber<br>
mats and analyzed for 3H-thymidine incorporation. Antigen-specific T cell proliferation is calculated as the ratio of 3H-<br>
thymidine incorporation in the presence of antigen divided by the 3H-thymidine incorporation in the absence of antigen.<br>
Example 29: Induction Of Specific CTL Response In Humans<br>
[0559] A human clinical trial for an immunogenic composition comprising CTL and HTL epitopes of the invention is<br>
set up as an IND Phase I, dose escalation study and carried out as a randomized, double-blind, placebo-controlled trial.<br>
Such a trial is designed, for example, as follows:<br>
[0560] A total of about 27 individuals are enrolled and divided into 3 groups:<br>
[0561] Group I: 3 subjects are injected with placebo and 6 subjects are injected with 5 µg of peptide composition;<br>
[0562] Group II: 3 subjects are injected with placebo and 6 subjects are injected with 50 µg peptide composition;<br>
[0563] Group 111: 3 subjects are injected with placebo and 6 subjects are injected with 500 jog of peptide<br>
composition.<br>
[0564] After 4 weeks following the first injection, all subjects receive a booster inocutation at the same dosage.<br>
[0565] The endpoints measured in this study relate to the safety and tolerability of the peptide composition as well as<br>
its immunogenicity. Cellular immune responses to the peptide composition are an index of the intrinsic activity of this the<br>
peptide composition, and can therefore be viewed as a measure of biological efficacy. The following summarize the clinical<br>
and laboratory data that relate to safety and efficacy endpoints.<br>
[0566] Safety: The incidence of adverse events is monitored in the placebo and drug treatment group and assessed<br>
in terms of degree and reversibility.<br>
[0567] Evaluation of Vaccine Efficacy: For evaluation of vaccine efficacy, subjects are bled before and after<br>
injection. Peripheral blood mononuclear cells are isolated from fresh heparinized blood by Ficoll-Hypaque density gradient<br>
centrifugation, aliquoted in freezing media and stored frozen. Samples are assayed for CTL and HTL activity.<br>
[0568] The vaccine is found to be both safe and efficacious.<br>
Example 30: Phase II Trials In Patients Expressing PSCA<br>
[0569] Phase II trials are performed to study the effect of administering the CTL-HTL peptide compositions to<br>
patients having cancer that expresses PSCA. The main objectives of the trial are to determine an effective dose and<br>
regimen for inducing CTLs in cancer patients that express PSCA, to establish the safety of inducing a CTL and HTL<br>
response in these patients, and to see to what extent activation of CTLs improves the clinical picture of these patients, as<br>
manifested, e.g., by the reduction and/or shrinking of lesions. Such a study is designed, for example, as follows:<br>
[0570] The studies are performed in multiple centers. The trial design is an open-label, uncontrolled, dose<br>
escalation protocol wherein the peptide composition is administered as a single dose followed six weeks later by a single<br>
booster shot of the same dose. The dosages are 50,500 and 5,000 micrograms per injection. Drug-associated adverse<br>
effects (severity and reversibility) are recorded.<br>
[0571] There are three patient groupings. The first group is injected with 50 micrograms of the peptide composition<br>
the second and third groups with 500 and 5,000 micrograms of peptide composition, respectively. The patients within<br>
each group range in age from 21-65 and represent diverse ethnic backgrounds. Ail of them have a tumor that expresses<br>
PSCA.<br>
[0572] Clinical manifestations or antigen-specific T-cell responses are monitored to assess the effects of<br>
administering the peptide compositions. The vaccine composition is found to be both safe and efficacious in the treatment<br>
of PSCA-associated disease.<br>
Example 31: Induction of CTL Responses Using a Prime Boost Protocol<br>
[0573] A prime boost protocol similar in its underlying principle to that used to confirm the efficacy of a DNA vaccine<br>
in transgenic mice, such as described above in the Example entitled "The Plasmid Construct and the Degree to Which It<br>
Induces Immunogenicity," can also be used for the administration of the vaccine to humans. Such a vaccine regimen can<br>
include an initial administration of, for example, naked DNA followed by a boost using recombinant virus encoding the<br>
vaccine, or recombinant protein/polypeptide or a peptide mixture administered in an adjuvant.<br>
[0574] For example, the initial immunization may be performed using an expression vector, such as that constructed<br>
in the Example entitled "Construction of "Minigene" Multi-Epitope DNA Plasmids" in the form of naked nucleic acid<br>
administered IM (or SC or ID) in the amounts of 0.5-5 mg at multiple sites. The nucleic acid (0.1 to 1000 µg ) can also be<br>
administered using a gene gun. Following an incubation period of 3-4 weeks, a booster dose is then administered. The<br>
booster can be recombinant fowlpox virus administered at a dose of 5-107 to 5x109 pfu. An alternative recombinant virus,<br>
such as an MVA, canarypox, adenovirus, or adeno-associated virus, can also be used for the booster, or the polyepitopic<br>
protein or a mixture of the peptides can be administered. For evaluation of vaccine efficacy, patient blood samples are<br>
obtained before immunization as well as at intervals following administration of the initial vaccine and booster doses of the<br>
vaccine. Peripheral blood mononuclear cells are isolated from fresh heparinized blood by Ficoll-Hypaque density gradient<br>
centrifugation, aliquoted in freezing media and stored frozen. Samples are assayed for CTL and HTL activity.<br>
[0575] Analysis of the results indicates that a magnitude of response sufficient to achieve a therapeutic or protective<br>
immunity against PSCA is generated.<br>
Example 32: Administration of Vaccine Compositions Using Dendritic Cells (DC)<br>
[0576] Vaccines comprising peptide epitopes of the invention can be administered using APCs, or "professional"<br>
APCs such as DC. In this example, peptide-pulsed DC are administered to a patient to stimulate a CTL response in vivo.<br>
In this method, dendritic cells are isolated, expanded, and pulsed with a vaccine comprising peptide CTL and HTL epitopes<br>
of the invention. The dendritic cells are infused back into the patient to elicit CTL and HTL responses in vivo. The induced<br>
CTL and HTL then destroy or facilitate destruction, respectively, of the target cells that bear the PSCA protein from which<br>
the epitopes in the vaccine are derived.<br>
[0577] For example, a cocktail of epitope-comprising peptides is administered ex vivo to PBMC, or isolated DC<br>
therefrom. A pharmaceutical to facilitate harvesting of DC can be used, such as Progenipoietin™ (Monsanto, St. Louis,<br>
MO) or GM-CSF/IL-4. After pulsing the DC with peptides, and prior to reinfusion into patients, the DC are washed to<br>
remove unbound peptides.<br>
[0578] As appreciated clinically, and readily determined by one of skill based on clinical outcomes, the number of DC<br>
reniwsed into the patient can vary (see, e.g., Nature Med. 4:328,1998; Nature Med. 2:52,1996 and Prostate 32:272,<br>
1997). Although 2-50 x 106 DC per patient are typically administered, larger number of DC, such as 107 or 108 can also be<br>
provided. Such cell populations typically contain between 50-90% DC.<br>
[0579] In some embodiments, peptide-loaded PBMC are injected into patients without purification of the DC. For<br>
example, PBMC generated after treatment with an agent such as Progenipoietin™ are injected into patients without<br>
purification of the DC. The total number of PBMC that are administered often ranges from 108 to 1010. Generally, the cell<br>
doses injected into patients is based on the percentage of DC in the blood of each patient, as determined, for example, by<br>
immunofluorescence analysis with specific anti-DC antibodies. Thus, for example, if Progenipoietin™ mobilizes 2% DC in<br>
the peripheral blood of a given patient, and that patient is to receive 5 x 10s DC, then the patient will be injected with a total<br>
of 2.5 x 108 peptide-loaded PBMC. The percent DC mobilized by an agent such as Progenipoietin™ is typically estimated<br>
to be between 2-10%, but can vary as appreciated by one of skill in the art.<br>
Ex vivo activation of CTL/HTL responses<br>
[0580] Alternatively, ex vivo CTL or HTL responses to PSCA antigens can be induced by incubating, in tissue<br>
culture, the patient's, or genetically compatible, CTL or HTL precursor cells together with a source of APC, such as DC, and<br>
immunogenic peptides. After an appropriate incubation time (typically about 7-28 days), in which the precursor cells are<br>
activated and expanded into effector cells, the cells are infused into the patient, where they will destroy (CTL) or facilitate<br>
destruction (HTL) of their specific target cells, i.e., tumor cells.<br>
Example 33: An Alternative Method of Identifying and Confirming Motif-Bearing<br>
Peptides<br>
[0581] Another method of identifying and confirming motif-bearing peptides is to elute them from cells bearing<br>
defined MHC molecules. For example, EBV transformed B cell lines used for tissue typing have been extensively<br>
characterized to determine which HLA molecules they express. In certain cases these cells express only a single type of<br>
HLA molecule. These cells can be transfected with nucleic acids that express the antigen of interest, e.g. PSCA. Peptides<br>
produced by endogenous antigen processing of peptides produced as a result of transfection will then bind to HLA<br>
molecules within the cell and be transported and displayed on the cell's surface. Peptides are then eluted from the HLA<br>
molecules by exposure to mild acid conditions and their amino acid sequence determined, e.g., by mass spectral analysis<br>
(e.g., Kubo et al., J. Immunol. 152:3913,1994). Because the majority of peptides that bind a particular HLA molecule are<br>
motif-bearing, this is an alternative modality for obtaining the motif-bearing peptides correlated with the particular HLA<br>
molecule expressed on the ceil.<br>
[0582] Alternatively, cell lines that do not express endogenous HLA molecules can be transfected with an expression<br>
construct encoding a single HLA allele. These cells can then be used as described, i.e., they can then be transfected with<br>
nucleic acids that encode PSCA to isolate peptides corresponding to PSCA that have been presented on the cell surface.<br>
Peptides obtained from such an analysis will bear motif(s) that correspond to binding to the single HLA allele that is<br>
expressed in the cell.<br>
[0583] As appreciated by one in the art, one can perform a similar analysis on a cell bearing more than one HLA<br>
allele and subsequently determine peptides specific for each HLA allele expressed. Moreover, one of skill would also<br>
recognize that means other than transfection, such as loading with a protein antigen, can be used to provide a source of<br>
often to the cell.<br>
Example 34: Complementary Polynucleotides<br>
[0584] Sequences complementary to the PSCA-encoding sequences, or any parts thereof, are used to detect,<br>
decrease, or inhibit expression of naturally occurring PSCA. Although use of oligonucleotides comprising from about 15 to<br>
30 base pairs is described, essentially the same procedure is used with smaller or with larger sequence fragments.<br>
Appropriate oligonucleotides are designed using, e.g., OLIGO 4.06 software (National Biosciences) and the coding<br>
sequence of PSCA. To inhibit transcription, a complementary oligonucleotide is designed from the most unique 5'<br>
sequence and used to prevent promoter binding to the coding sequence. To inhibit translation, a complementary<br>
oligonucleotide is designed to prevent ribosomal binding to a PSCA-encoding transcript.<br>
Example 35: Purification of Naturally-occurring or Recombinant PSCA Using PSCA-<br>
Specific Antibodies<br>
[0585] Naturally occurring or recombinant PSCA is substantially purified by immunoaffinity chromatography using<br>
antibodies specific for PSCA. An immunoaffinity column is constructed by covalently coupling anti-PSCA antibody to an<br>
activated chromatographic resin, such as CNBr-activated SEPHAROSE (Amersham Pharmacia Biotech). After the<br>
coupling, the resin is blocked and washed according to the manufacturer's instructions.<br>
[0586] Media containing PSCA are passed over the immunoaffinity column, and the column is washed under<br>
conditions that allow the preferential absorbance of PSCA (e.g., high ionic strength buffers in the presence of detergent).<br>
The column is eluted under conditions that disrupt antibody/PSCA binding (e.g., a buffer of pH 2 to pH 3, or a high<br>
concentration of a chaotrope, such as urea or thiocyanate ion), and GCR.P is collected.<br>
Example 36: Identification of Molecules Which Interact with PSCA<br>
[0587] PSCA, or biologically active fragments thereof, are labeled with 1211 Bolton-Hunter reagent. (See, e.g.,<br>
Bolton et al. (1973) Biochem. J. 133:529.) Candidate molecules previously arrayed in the wells of a multi-well plate are<br>
incubated with the labeled PSCA, washed, and any wells with labeled PSCA complex are assayed. Data obtained using<br>
different concentrations of PSCA are used to calculate values for the number, affinity, and association of PSCA with the<br>
candidate molecules.<br>
Example 37: In Vivo Assay for PSCA v.4 Tumor Growth Promotion<br>
[0588] The effect of the PSCA v.4 protein on tumor cell growth is evaluated in vivo by evaluating tumor development<br>
and growth of cells expressing or lacking PSCA v.4. For example, SCID mice are injected subcutaneously on each flank<br>
with 1 x 106 of either 3T3, prostate (e.g. PC3 cells), bladder (e.g. UM-UC3 cells) or pancreas (e.g. PANC1 cells) cancer cell<br>
lines containing tkNeo empty vector or PSCA v.4. At least two strategies may be used: (1) Constitutive PSCA v.4<br>
expression under regulation of a promoter such as a constitutive promoter obtained from the genomes of viruses such as<br>
polyoma virus, fowlpox virus (UK 2,211,504 published 5 July 1989), adenovirus (such as Adenovirus 2), bovine papilloma<br>
virus, avian sarcoma virus, cytomegalovirus, a retrovirus, hepatitis-B virus and Simian Virus 40 (SV40), or from<br>
heterologous mammalian promoters, e.g., the actin promoter or an immunoglobulin promoter, provided such promoters are<br>
compatible with the host cell systems, and (2) Regulated expression under control of an inducible vector system, such as<br>
evisone, tetracycline, etc., provided such promoters are compatible with the host cell systems. Tumor volume is then<br>
monitored by caliper measurement at the appearance of palpable tumors and followed over time to determine if PSCA v.4-<br>
expressing cells grow at a faster rate and whether tumors produced by PSCA v.4-expressing cells demonstrate<br>
characteristics of altered aggressiveness (e.g. enhanced metastasis, vascularization, reduced responsiveness to<br>
chemotherapeutic drugs).<br>
[0589] Additionally, mice can be implanted with 1 x 105 of the same cells orthotopically to determine if PSCA v.4 has<br>
an effect on local growth in the pancreas, and whether PSCA v.4 affects the ability of the cells to metastasize, specifically to<br>
lymph nodes, and bone (Miki T et al, Oncol Res. 2001 ;12:209; Fu X et al, int. J Cancer. 1991,49:938). The effect of PSCA<br>
v.4 on bone tumor formation and growth may be assessed by injecting tumor cells intratibially.<br>
[0590] The assay is also useful to determine the PSCA v.4 inhibitory effect of candidate therapeutic compositions,<br>
such as for example, PSCA v.4 intrabodies, PSCA v.4 antisense molecules and ribozymes.<br>
Example 38:<br>
PSCA v.4 Monoclonal Antibody-mediated Inhibition of Tumors In Vivo<br>
[0591] The significant expression of PSCA v.4 in cancer tissues, together with its restrictive expression in normal<br>
tissues makes PSCA v.4 a good target for antibody therapy. Similarly, PSCA v.4 is a target for T cell-based<br>
immunotherapy. Thus, the therapeutic efficacy of anti-PSCA v.4 mAbs in human cancer xenograft mouse models,<br>
including prostate, bladder, and pancreas (e.g. PANC1 cells) and other -PSCA v.4 cancers listed in table 1, is evaluated by<br>
using recombinant ceil lines such as PC3-PSCA v.4, UM-UC3-PSCA v.4, PANC1-PSCA v.4, and 3T3-PSCA v.4 (see, e.g.,<br>
Kaighn, M.E., et al., Invest Urol, 1979.17(1): 16-23), as well as human xenograft models (Saffran et al PNAS1999,<br>
10:1073-1078).<br>
[0592] Antibody efficacy on tumor growth and metastasis formation is studied, e.g., in a mouse orthotopic ovary,<br>
pancreas, or blood cancer xenograft models. The antibodies can be unconjugated, as discussed in this Example, or can be<br>
conjugated to a therapeutic modality, as appreciated in the art. Anti-PSCA v.4 mAbs inhibit formation of tumors in mouse<br>
xenografts. Anti-PSCA v.4 mAbs also retard the growth of established orthotopic tumors and prolonged survival of tumor-<br>
bearing mice. These results indicate the utility of anti-PSCA v.4 mAbs in the treatment of local and advanced stages<br>
several solid tumors. (See, e.g., Saffran, D., et al., PNAS 10:1073-1078 or world wide web URL<br>
pnas.org/cgi/doi/10.1073/pnas.051624698).<br>
[0593] Administration of the anti-PSCA v.4 mAbs led to retardation of established orthotopic tumor growth and<br>
inhibition of metastasis to distant sites, resulting in a significant prolongation in the survival of tumor-bearing mice. These<br>
studies indicate that PSCA v.4 as an attractive target for immunotherapy and demonstrate the therapeutic potential of anti-<br>
PSCA v.4 mAbs for the treatment of local and metastatic cancer. This example indicates that unconjugated PSCA v.4<br>
monoclonal antibodies are effective to inhibit the growth of human pancreatic, ovarian, and lymphomas tumor xenografts<br>
grown in SCID mice; accordingly a combination of such efficacious monoclonal antibodies is also effective.<br>
Tumor inhibition using multiple unconjugated PSCA v.4 mAbs<br>
Materials and Methods<br>
PSCA v.4 Monoclonal Antibodies:<br>
[0594] Monoclonal antibodies are raised against PSCA v.4 as described in the Example entitled "Generation of<br>
PSCA v.4 Monoclonal Antibodies (mAbs)." The antibodies are characterized by ELISA, Western blot, FACS, and<br>
immunoprecipitatton for their capacity to bind PSCA v.4. Epitope mapping data for the anti-PSCA v.4 mAbs, as determined<br>
by ELISA and Western analysis, recognize epitopes on the PSCA v.4 protein. Immunohistochemical analysis of cancer<br>
tissues and cells with these antibodies is performed.<br>
[0595] The monoclonal antibodies are purified from ascites or hybridoma tissue culture supernatants by Protein-G<br>
Sepharose chromatography, dialyzed against PBS, filter sterilized, and stored at -20°C. Protein determinations are<br>
performed by a Bradford assay (Bio-Rad, Hercules, CA). A therapeutic monoclonal antibody or a cocktail comprising a<br>
mixture of individual monoclonal antibodies is prepared and used for the treatment of mice receiving subcutaneous or<br>
orthotopic injections of PC3, UM-UC3, CaKi, and A427 tumor xenografts.<br>
Cell Lines and Xenografts<br>
[0596] The LAPC-9 xenograft, which expresses a wild-type androgen receptor and produces prostate-specific<br>
antigen (PSA), is passaged in 6- to 8-week-old male ICR-severe combined immunodeficient (SCID) mice (Taconic Farms)<br>
by s.c. trocar implant (Craft, N., et al., 1999, Cancer Res. 59:5030-5036). The AGS-K3 and AGS-K6 kidney xenografts are<br>
also passaged by subcutaneous implants in 6- to 8- week old SCID mice. Single-cell suspensions of tumor cells are<br>
prepared as described in Craft, et al.<br>
[0597] The cancer cell lines PC3, UM-UC3 and PANC1 cell lines, as well as the fibroblast line NIH 3T3 (American<br>
Type Culture Collection). The prostate carcinoma cell line PC3 is maintained in RPMI supplemented with L-glutamine and<br>
10% FBS, and the bladder and pancreas carcinoma lines, UM-UC3 and PANC1 respectively, are maintained in DMEM<br>
supplemented with L-glutamine and 10% FBS. PC3-PSCA v.4, UM-UC3-PSCA v.4, PANC1-PSCA v.4 and 3T3-PSCA v.4<br>
cell populations are generated by retroviral gene transfer as described in Hubert, R.S., et al., Proc Natl. Acad. Sci USA,<br>
1999. 96(25): 14523.<br>
Xenograft Mouse Models.<br>
[0598] Subcutaneous (s.c.) tumors are generated by injection of 2 x 106 cancer cells mixed at a 1:1 dilution with<br>
Matrigel (Collaborative Research) in the right flank of male SCID mice. To test antibody efficacy on tumor formation, i.e.<br>
antibody injections are started on the same day as tumor-cell injections. As a control, mice are injected with either purified<br>
mouse IgG (ICN) or PBS; or a purified monoclonal antibody that recognizes an irrelevant antigen not expressed in human<br>
cells. In preliminary studies, no difference is found between mouse IgG or PBS on tumor growth. Tumor sizes are<br>
determined by caliper measurements, and the tumor volume is calculated as length x width x height. Mice with<br>
Subcutaneous tumors greater than 1.5 cm in diameter are sacrificed.<br>
[0599] Orthotopic injections are performed under anesthesia by using ketamine/xylazine.. For prostate orthotopic<br>
studies, an incision is made through the abdominal muscles to expose the bladder and seminal vesicles, which then are<br>
delivered through the incision to expose the dorsal prostate. LAPC-9 cells (5 x 105) mixed with Matrigel are injected into<br>
each dorsal lobe in a 10 µl volume. To monitor tumor growth, mice are bled on a weekly basis for determination of PSA<br>
level. For pancreas orthotopic model, an incision is made through the abdominal muscles to expose the mammary<br>
tissues and a single cell suspension of pancreas cancer cells is injected into the mammary pad. For the bladder orthotopic<br>
model, AGS-B1 bladder cancer tissue is adhered onto the bladder wall. Following tumor implantation, the mice are<br>
segregated into groups for the appropriate treatments, with anti-PSCA v.4 or control mAbs being injected i.p. To monitor<br>
tumor growth, mice are palpated and blood is collected on a weekly basis to measure hCG levels.<br>
Anti-PSCA v.4 mAbs Inhibit Growth of PSCA v.4-Expressinq Xenograft-Cancer Tumors<br>
[0600] The effect of anti-PSCA v.4 mAbs on tumor formation is tested by using cell line (e.g. PC3, UM-UC3, PANC1<br>
and 3T3) and patient-derived tumor orthotopic models. As compared with the s.c. tumor model, the orthotopic model,<br>
which requires injection of tumor cells directly in the mouse organ results in a local tumor growth, development of<br>
metastasis in distal sites, deterioration of mouse health, and subsequent death (Saffran, D., et al., PNAS supra). The<br>
features make the orthotopic model more representative of human disease progression and allowed us to follow the<br>
therapeutic effect of mAbs on clinically relevant end points.<br>
[0601] A major advantage of the orthotopic cancer models is the ability to study the development of metastases.<br>
Formation of metastasis in mice bearing established orthotopic tumors is studies by IHC analysis on lung sections using an<br>
antibody against a tumor-specific cell-surface protein such as anti-CK20 for prostate cancer (Lin S et al, Cancer Detect<br>
Prev.2001;25:202).<br>
[0602] Another advantage of xenograft cancer models is the ability to study neovascularization and angiogenesis.<br>
Tumor growth is partly dependent on new blood vessel development. Although the capillary system and developing blood<br>
network is of host origin, the initiation and architecture of the neoyasculature is regulated by the xenograft tumor (Davidoff<br>
AM et al, Clin Cancer Res. 2001;7:2870; Solesvik 0 et al„ Eur. J Cancer Clin Oncol. 1984, 20:1295). The effect of<br>
antibody and small molecule on neovascularization is studied in accordance with procedures known in the art, such as by<br>
IHC analysis of tumor tissues and their surrounding microenvironment.<br>
[0603] Mice bearing established orthotopic tumors are administered 1000µg injections of either anti-PSCA v.4 mAb<br>
or PBS over a 4-week period. Mice in both groups are allowed to establish a high tumor burden, to ensure a high<br>
frequency of metastasis formation in mouse lungs. Mice then are killed and their bladders, livers, bone and lungs are<br>
analyzed for the presence of tumor cells by IHC analysis. These studies demonstrate a broad anti-tumor efficacy of anti-<br>
PSCA v.4 antibodies on initiation and progression of prostate cancer in xenograft mouse models. Anti-PSCA v.4 antibodies<br>
inhibit tumor formation of tumors as well as retarding the growth of already established tumors and prolong the survival of<br>
treated mice. Moreover, anti-PSCA v.4 mAbs demonstrate a dramatic inhibitory effect on the spread of local prostate tumor<br>
to distal sites, even in the presence of a large tumor burden. Thus, anti-PSCA v.4 mAbs are efficacious on major clinically<br>
relevant end points (tumor growth), prolongation of survival, and health.<br>
Example 39:<br>
Therapeutic and Diagnostic use of Anti-PSCA Antibodies in Humans.<br>
[0604] Anti-PSCA monoclonal antibodies are safely and effectively used for diagnostic, prophylactic, prognostic<br>
and/or therapeutic purposes in humans. Western blot and immunohistochemical analysis of cancer tissues and cancer<br>
xenografts with anti-PSCA mAb show strong extensive staining in carcinoma but significantly lower or undetectable levels<br>
in normal tissues. Detection of PSCA in carcinoma and in metastatic disease demonstrates the usefulness of the mAb as a<br><br>
diagnostic and/or prognostic indicator. Anti-PSCA antibodies are therefore used in diagnostic applications such as<br>
immunohistochemistry of kidney biopsy specimens to detect cancer from suspect patients.<br>
[0605] As determined by flow cytometry, anti-PSCA mAb specifically binds to carcinoma cells. Thus, anti-PSCA<br>
antibodies are used in diagnostic whole body imaging applications, such as radioimmunoscintigraphy and<br>
radioimmunotherapy, (see, e.g., Potamianos S., et. al. Anticancer Res 20(2A):925-948 (2000)) for the detection of localized<br>
and metastatic cancers that exhibit expression of PSCA. Shedding or release of an extracellular domain of PSCA into the<br>
extracellular milieu, such as that seen for alkaline phosphodiesterase B10 (Meerson, N. R., Hepatology 27:563-568<br>
(1998) allows diagnostic detection of PSCA by anti-PSCA antibodies in serum and/or urine samples from suspect<br>
pa:.ents<br>
[0606] Anti-PSCA antibodies that specifically bind PSCA are used in therapeutic applications for the treatment of<br>
cancers that express PSCA. Anti-PSCA antibodies are used as an unconjugated modality and as conjugated form in which<br>
the antibodies are attached to one of various therapeutic or imaging modalities well known in the art, such as a prodrugs,<br>
enzynes or radioisotopes. In preclinical studies, unconjugated and conjugated anti-PSCA antibodies are tested for efficacy<br>
o! turror pretention and growth inhibition in the SCID mouse cancer xenograft models, e.g., kidney cancer models AGS-K3<br>
and AGS-K6, (see, e.g., the Example entitled "PSCA Monoclonal Antibody-mediated Inhibition of Bladder and Lung Tumors<br>
In Vivo) Either conjugated and unconjugated anti-PSCA antibodies are used as a therapeutic modality in human clinical<br>
trials eithet alone or in combination with other treatments as described in following Examples.<br>
Example 40: Human Clinical Trials for the Treatment and Diagnosis of Human<br>
Carcinomas through use of Human Anti-PSCA Antibodies In vivo<br>
[0607] Antibodies are used in accordance with the present invention which recognize an epitope on PSCA, and are<br>
used in the treatment of certain tumors such as those listed in Table I. Based upon a number of factors, including PSCA<br>
expression levels, tumors such as those listed in Table I are presently preferred indications. In connection with each of<br>
these indications, three clinical approaches are successfully pursued.<br>
I.) Adjunctive therapy: In adjunctive therapy, patients are treated with anti-PSCA antibodies in<br>
combination with a chemotherapeutic or antineoplastic agent and/or radiation therapy. Primary cancer targets, such as<br>
those listed in Table I, are treated under standard protocols by the addition anti-PSCA antibodies to standard first and<br>
second line therapy. Protocol designs address effectiveness as assessed by reduction in tumor mass as well as the ability<br>
to reduce usual doses of standard chemotherapy. These dosage reductions allow additional and/or prolonged therapy by<br>
reducing dose-related toxicity of the chemotherapeutic agent. Anti-PSCA antibodies are utilized in several adjunctive<br>
clinical trials in combination with the chemotherapeutic or antineoplastic agents adriamycin (advanced prostrate<br>
carcinoma), cisplatin (advanced head and neck and lung carcinomas), taxol (breast cancer), and doxorubicin (preclinical).<br>
II.) Monotherapy: In connection with the use of the anti-PSCA antibodies in monotherapy of tumors, the<br>
antibodies are administered to patients without a chemotherapeutic or antineoplastic agent. In one embodiment,<br>
monotherapy is conducted clinically in end stage cancer patients with extensive metastatic disease. Patients show some<br>
disease stabilization. Trials demonstrate an effect in refractory patients with cancerous tumors.<br>
III.) Imaging Agent: Through binding a radionuclide (e.g., iodine or yttrium (I131, Y90) to anti-PSCA<br>
antibodies, the radiolabeled antibodies are utilized as a diagnostic and/or imaging agent. In such a role, the labeled<br>
antibodies localize to both solid tumors, as well as, metastatic lesions of cells expressing PSCA. In connection with the use<br>
of the anti-PSCA antibodies as imaging agents, the antibodies are used as an adjunct to surgical treatment of solid tumors,<br><br>
as both a pre-surgical screen as well as a post-operative follow-up to determine what tumor remains and/or returns. In one<br>
embodiment, a (111 ln)-PSCA antibody is used as an imaging agent in a Phase I human clinical trial in patients having a<br>
carcinoma that expresses PSCA (by analogy see, e.g., Divgi et al. J. Natl. Cancer Inst. 83:97-104 (1991)). Patients are<br>
followed with standard anterior and posterior gamma camera. The results indicate that primary lesions and metastatic<br>
lesions are identified.<br>
Dose and Route of Administration<br>
[0608] As appreciated by those of ordinary skill in the art, dosing considerations can be determined through<br>
comparison with the analogous products that are in the clinic. Thus, anti-PSCA antibodies can be administered with doses<br>
in the range of 5 to 400 mg/m2, with the lower doses used, e.g., in connection with safety studies. The affinity of anti-<br>
PSCA antibodies relative to the affinity of a known antibody for its target is one parameter used by those of skill in the art<br>
for determin. analogous dose regimens. Further, anti-PSCA antibodies that are fully human antibodies, as compared to<br>
the chimeric antibody, have slower clearance; accordingly,,dosing in patients with such fully human anti-PSCA antibodies<br>
can be lower, perhaps in the range of 50 to 300 mg/m2, and still remain efficacious. Dosing in mg/m2, as opposed to the<br>
conventional measurement of dose in mg/kg, is a measurement based on surface area and is a convenient dosing<br>
measurement that is designed to include patients of all sizes from infants to adults.<br>
[0609] Three distinct delivery approaches are useful for delivery of anti-PSCA antibodies. Conventional intravenous<br>
delivery is one standard delivery technique for many tumors. However, in connection with tumors in the peritoneal cavity,<br>
such as tumors of the ovaries, biliary duct, other ducts, and the like, intraperitoneal administration may prove favorable for<br>
obtaining high dose of antibody at the tumor and to also minimize antibody clearance. In a similar manner, certain solid<br>
tumors possess vasculature that is appropriate for regional perfusion. Regional perfusion allows for a high dose of antibody<br>
at the site of a tumor and minimizes short term clearance of the antibody.<br>
Clinical Development Plan (CDP)<br>
[0610] Overview: The CDP follows and develops treatments of anti-PSCA antibodies in connection with adjunctive<br>
therapy, monotherapy, and as an imaging agent. Trials initially demonstrate safety and thereafter confirm efficacy in repeat<br>
doses. Trails are open label comparing standard chemotherapy with standard therapy plus anti-PSCA antibodies. As will<br>
be appreciated, one criteria that can be utilized in connection with enrollment of patients is PSCA expression levels in their<br>
tumors as determined by biopsy.<br>
[0611] As with any protein or antibody infusion-based therapeutic, safety concerns are related primarily to (i)<br>
cytokine release syndrome, i.e., hypotension, fever, shaking, chills; (ii) the development of an immunogenic response to the<br>
material (i.e., development of human antibodies by the patient to the antibody therapeutic, or HAHA response); and, (iii)<br>
toxicity to normal cells that express PSCA. Standard tests and follow-up are utilized to monitor each of these safety<br>
concerns. Anti-PSCA antibodies are found to be safe upon human administration.<br>
Example 41: Human Clinical Trial Adjunctive Therapy with Human Anti-PSCA Antibody<br>
and Chemotherapeutic Agent<br>
[0612] A phase I human clinical trial is initiated to assess the safety of six intravenous doses of a human anti-PSCA<br>
antibody in connection with the treatment of a solid tumor, e.g., a cancer of a tissue listed in Table I. In the study, the safety<br>
of single doses of anti-PSCA antibodies when utilized as an adjunctive therapy to an antineoplastic or chemotherapeutic<br><br>
agent as defined herein, such as, without limitation: cisplatin, topotecan, doxorubicin, adriamycin, taxol, or the like, is<br>
accessed. The trial design includes delivery of six single doses of an anti-PSCA antibody with dosage of antibody<br>
escalating from approximately about 25 mg/m 2to about 275 mg/m 2over the course of the treatment in accordance with the<br>
following schedule:<br><br>
[0613] Patients are closely followed for one-week following each administration of antibody and chemotherapy. In<br>
particular, patients are assessed for the safety concerns mentioned above: (i) cytokine release syndrome, i.e., hypotension,<br>
fever, shaking, chills; (ii) the development of an immunogenic response to the material (i.e., development of human<br>
antibodies by the patient to the human antibody therapeutic, or HAHA response); and, (iii) toxicity to normal cells that<br>
express PSCA. Standard tests and follow-up are utilized to monitor each of these safety concerns. Patients are also<br>
assessed for clinical outcome, and particularly reduction in tumor mass as evidenced by MRI or other imaging.<br>
[0614] The anti-PSCA antibodies are demonstrated to be safe and efficacious, Phase II trials confirm the efficacy<br>
and refine optimum dosing.<br>
Example 42: Human Clinical Trial: Monotherapy with Human Anti-PSCA Antibody<br>
[0615] Anti-PSCA antibodies are safe in connection with the above-discussed adjunctive trial, a Phase II human<br>
clinical trial confirms the efficacy and optimum dosing for monotherapy. Such trial is accomplished, and entails the same<br>
safety and outcome analyses, to the above-described adjunctive trial with the exception being that patients do not receive<br>
chemotherapy concurrently with the receipt of doses of anti-PSCA antibodies.<br>
Example 43: Human Clinical Trial: Diagnostic Imaging with Anti-PSCA Antibody<br>
[0616] Once again, as the adjunctive therapy discussed above is safe within the safety criteria discussed above, a<br>
human clinical trial is conducted concerning the use of anti-PSCA antibodies as a diagnostic imaging agent. The protocol is<br>
designed in a substantially similar manner to those described in the art, such as in Divgi et al. J. Natl. Cancer Inst. 83:97-<br>
104 (1991). The antibodies are found to be both safe and efficacious when used as a diagnostic modality.<br>
Example 44 Homology Comparison of PSCA v.4 to Known Sequences:<br>
[0617] The PSCA v.4 gene encodes a 189 aa protein. The human PSCA v.4 protein exhibit a high degree of<br>
homology to human prostate stem cell antigen (gi 27482160), exhibiting 98% identity to PSCA v.4 at the protein level<br>
(Figure 4). The mouse homolog of PSCA v.4 has not been identified.<br>
[0618] The PSCA v.4 protein has several variants (figure 11). These include 8 SNPs and a splice variant, referred to<br>
as PSCA v.3. The PSCA v.3 protein encompasses the C-termial portion of PSCA v.4, and corresponds to aa 94-189 of that<br>
variant. Bioinformatics analysis using topology prediction programs indicate that PSCA v.4 is a soluble protein with no<br>
transmembrane domains (Table L)..<br><br>
[0619] Motif analysis revealed the presence of two protein functional motifs in the PSCA v.4 protein (Table L),<br>
namely a cadherin motif and a granulin domain have been identified. Cadherins belong to a family of calcium-dependent<br>
cell adhesion molecules. They are single transmembrane proteins containing immunoglobulin like domains, and are<br>
involved in cell adhesion and sorting (Shan et al, Biophys Chem 1999,82:157). For examples, cadherins mediate tissue-<br>
specific cell adhesion of lymphocytes to the surface of epithelial cells. Cadherins have been shown to function in tissue<br>
morphogenesis, cell adhesion, cell differentiation, cell migration and tumour metastasis (Yap AS, Kovacs EM. J Biol Chem<br>
2003, 160.11; Vestweber D. Curr Opin Cell Biol 2002,14:587; Bloom et al, Mol Biol Cell. 1999,10:1521; Brodt P. Cancer<br>
Met Rev 1991.10:23). Granulins or epithelins are growth factors originally purified from cell-conditioned media, shown to<br>
enhance cell proliferation (Xu, S. Q. et al, J. Biol. Chem. 1998,273:20078). Granulins are expressed at elevated levels in<br>
several cancers, including gliomas and renal cancer (Liau L et al, Cancer Res. 60:1353, Donald, C. D et al, Anticancer<br>
Res 21.3739).<br>
[0620] The motifs found in PSCA v.4 indicate that PSCA v.4 can participate in tumor growth, and progression by<br>
regulating cell proliferation, cell adhesion, cell communication, invasion and metastasis.<br>
[0 621 ] Accordingly, when PSCA v.4 functions as a regulator of tumor establishment, tumor growth, tumor invasion,<br>
survival or cell signaling, PSCA v.4 is used for therapeutic, diagnostic, prognostic and/or preventative purposes. In<br>
addition. when a molecule, such as a splice variant or SNP of PSCA v.4 is expressed in cancerous tissues, such as those<br>
listed in Table I, they are used for therapeutic, diagnostic, prognostic and/or preventative purposes.<br>
Example 45: Regulation of Transcription<br>
[0622] The mitochondrial localization of PSCA v.4 coupled to the presence of cadherin domains within its sequence<br>
indicates that PSCA v.4 modulates the transcriptional regulation of eukaryotic genes. Regulation of gene expression is<br>
confirmed, e.g., by studying gene expression in cells expressing or lacking PSCA v.4. For this purpose, two types of<br>
experiments are performed.<br>
[0623] In the first set of experiments, RNA from parental and PSCA v.4-expressing cells are extracted and<br>
hybridized to commercially available gene arrays (Clontech) (Smid-Koopman E et al. Br J Cancer. 2000.83:246). Resting<br>
cells as well as cells treated with FBS, androgen or growth factors are compared. Differentially expressed genes are<br>
identified in accordance with procedures known in the art. The differentially expressed genes are then mapped to biological<br>
pathways (Chen Ket al. Thyroid. 2001. 11:41).<br>
[0624] In the second set of experiments, specific transcriptional pathway activation is evaluated using commercially<br>
available (Stratagene) luciferase reporter constructs including: NFkB-luc, SRE-luc, ELK1-luc, ARE-luc, p53-luc, and CRE-<br>
luc. These transcriptional reporters contain consensus binding sites for known transcription factors that lie downstream of<br>
well-characterized signal transduction pathways, and represent a good tool to ascertain pathway activation and screen for<br>
positive and negative modulators of pathway activation.<br>
[0625] Thus, PSCA v.4 plays a role in gene regulation, and it is used as a target for diagnostic, prognostic,<br>
preventative and/or therapeutic purposes.<br>
Example 46:<br>
Identification and Confirmation of Potential Signal Transduction Pathways<br>
[0626] Many mammalian proteins have been reported to interact with signaling molecules and to participate in<br>
regulating signaling pathways. (J Neurochem. 2001; 76:217-223). Cadherin molecules have been associated with Cdc42<br><br>
and Rho signaling (Kouklis J Biol Chem. 2003,278:16230). Using immunoprecipitation and Western blotting techniques,<br>
presins are identified that associate with PSCA v.4 and mediate signaling events. Several pathways known to play a role<br>
in cancer biology can be regulated by PSCA v.4, including phospholipid pathways such as PI3K, AKT, etc, adhesion and<br>
migration pathways, including FAK, Rho, Rac-1, catenin, etc, as well as mitogenic/survival cascades such as ERK, p38, etc<br>
(Cell Growth Differ. 2000,11:279; J Biol Chem. 1999,274:801; Oncogene. 2000,19:3003, J. Cell Biol. 1997,138:913.).<br>
). In order to determine whether expression of PSCA v.4 is sufficient to regulate specific signaling pathways not otherwise<br>
active in resting cancer cells, the effect of PSCA v.4 on the activation of the signaling cascade is investigated in the cancer<br>
cell lines PA-1, Panel and Daudi. Cancer cells stably expressing PSCA v.4 or neo are stimulated with growth factor, FBS<br>
or other activating molecules. Whole cell lysates are analyzed by western blotting.<br>
[0627] To confirm that PSCA v.4 directly or indirectly activates known signal transduction pathways in cells,<br>
luciferase (luc) based transcriptional reporter assays are carried out in cells expressing individual genes. These<br>
transcriptional reporters contain consensus-binding sites for known transcription factors that lie downstream of well-<br>
characterized signal transduction pathways. The reporters and examples of these associated transcription factors, signal<br>
transduction pathways, and activation stimuli are listed below.<br>
1. NFkB-luc, NFkB/Rel; lk-kinase/SAPK;growth/apoptosis/stress<br>
2. SRE-luc,SRF/TCF/ELK1; MAPK/SAPK; growth/differentiation<br>
3. AP-1-luc, FOS/JUN; MAPK/SAPK/PKC; growth/apoptosis/stress<br>
4. ARE-luc, androgen receptor; steroids/MAPK; growth/differentiation/apoptosis<br>
5. p53-luc, p53; SAPK; growth/differentiation/apoptosis<br>
6. CRE-luc, CREB/ATF2; PKA/p38; growth/apoptosis/stress<br>
7. TCF-luc, TCF/Lef; 0-catenin, Adhesion/invasion<br>
[0628] Gene-mediated effects can be assayed in cells showing mRNA expression. Luciferase reporter plasmids can<br>
be introduced by lipid-mediated transfection (TFX-50, Promega). Luciferase activity, an indicator of relative transcriptional<br>
activity, is measured by incubation of cell extracts with luciferin substrate and luminescence of the reaction is monitored in a<br>
luminometer.<br>
[0629] Signaling pathways activated by PSCA v.4 are mapped and used for the identification and validation of<br>
therapeutic targets. When PSCA v.4 is involved in cell signaling, it is used as target for diagnostic, prognostic, preventative<br>
and/or therapeutic purposes.<br>
Example 47: Involvement in Tumor Progression<br>
[0630] Based on the role of granulin and cadherin motifs in cell growth, adhesion and protein interactions, the PSCA<br>
v.4 gene can contribute to the growth, adhesion, invasion and transformation of cancer cells. The role of PSCA v.4 in<br>
tumor growth is confirmed in a variety of primary and transfected cell lines including prostate cell lines, as well as NIH 3T3<br>
ceils engineered to stably express PSCA v.4. Parental cells lacking PSCA v.4 and cells expressing PSCA v.4 are<br>
evaluated for cell growth using a well-documented proliferation assay (Fraser SP, Grimes JA, Djamgoz MB. Prostate.<br>
2000;44:61, Johnson DE, Ochieng J, Evans SL Anticancer Drugs. 1996,7:288).<br>
[0631] To confirm the role of PSCA v.4 in the transformation process, its effect in colony forming assays is<br>
investigated. Parental NIH-3T3 cells lacking PSCA v.4 are compared to NIH-3T3 cells expressing PSCA v.4, using a soft<br>
agar assay under stringent and more permissive conditions (Song Z. et al. Cancer Res. 2000;60:6730).<br>
[0632] To confirm the role of PSCA v.4 in invasion and metastasis of cancer cells, a well-established assay is used, a Transwell Insert System assay (Becton Dickinson) (Cancer Res. 1999; 59:6010). Control cells, including prostate,<br>
pancreas and kidney cell lines lacking PSCA v.4 are compared to cells expressing PSCA v.4. Cells are loaded with the<br>
fluorescent dye, calcein, and plated in the top well of the Transwell insert coated with a basement membrane analog.<br>
Invasion is determined by fluorescence of cells in the lower chamber relative to the fluorescence of the entire cell<br>
population.<br>
[0633] PSCA v.4 can also play a role in cell cycle and apoptosis. Parental cells and cells expressing PSCA v.4 are<br>
compared for differences in cell cycle regulation using a well-established BrdU assay (Abdel-Malek ZA. J Cell Physiol.<br>
1988,136:247). In short, cells are grown under both optimal (full serum) and limiting (low serum) conditions are labeled<br>
with BrdU and stained with anti-BrdU Ab and propidium iodide. Cells are analyzed for entry into the G1, S, and G2M<br>
phases of the cell cycle. Alternatively, the effect of stress on apoptosis is evaluated in control parental cells and cells<br>
expressing PSCA v.4, including normal and tumor prostate cells. Engineered and parental cells are treated with various<br>
chemotherapeutic agents, such as etoposide, taxol, etc, and protein synthesis inhibitors, such as cycloheximide. Cells are<br>
stained with annexin V-FITC and cell death is measured by FACS analysis. The modulation of cell death by PSCA v.4 can<br>
play a critical role in regulating tumor progression and tumor load.<br>
[0634] When PSCA v.4 plays a role in cell growth, transformation, invasion or apoptosis, it is used as a target for<br>
diagnostic, prognostic, preventative and/or therapeutic purposes.<br>
Example 48: Involvement in Angiogenesis<br>
[0635] Angiogenesis or new capillary blood vessel formation is necessary for tumor growth (Hanahan D, Folkman J.<br>
Cell. 1996,86:353; Folkman J. Endocrinology. 1998139:441). Several assays have been developed to measure<br>
angiogenesis in vitro and in vivo, such as the tissue culture assays endothelial cell tube formation and endothelial cell<br>
proliferation. Using these assays as well as in vitro neovascularization, the role of PSCA v.4 in angiogenesis,<br>
enhancement or inhibition, is confirmed.<br>
[0636] For example, endothelial cells engineered to express PSCA v.4 are evaluated using tube formation and<br>
proliferation assays. The effect of PSCA v.4 is also confirmed in animal models in vivo. For example, cells either<br>
expressing or lacking PSCA v.4 are implanted subcutaneously in immunocompromised mice. Endothelial cell migration<br>
and angiogenesis are evaluated 5-15 days later using immunohistochemistry techniques. PSCA v.4 affects angiogenesis,<br>
and it is used as a target for diagnostic, prognostic, preventative and/or therapeutic purposes.<br>
Example 49: Involvement in Protein-Protein Interactions<br>
[0637] Cadhesrin motifs have been shown to mediate interaction with other proteins. Using immunoprecipitation<br>
techniques as well as two yeast hybrid systems, proteins are identified that associate with PSCA v.4. Immunoprecipitates<br>
from cells expressing PSCA v.4 and cells lacking PSCA v.4 are compared for specific protein-protein associations.<br>
[0638] Studies are performed to confirm the extent of association of PSCA v.4 with effector molecules, such as<br>
nuclear proteins, transcription factors, kinases, phosphates etc. Studies comparing PSCA v.4 positive and PSCA v.4<br>
negative cells as well as studies comparing unstimulated/resting cells and cells treated with epithelial cell activators, such<br>
as cytokines, growth factors, androgen and anti-integrin Ab reveal unique interactions.<br>
[0639] In addition, protein-protein interactions are confirmed using two yeast hybrid methodology (Curr Opin Chem<br>
Biol. 1999,3:64). A vector carrying a library of proteins fused to the activation domain of a transcription factor is introduced<br><br>
into yeast expressing a PSCA v.4-DNA-binding domain fusion protein and a reporter construct. Protein-protein interaction<br>
protected by colorimetric reporter activity. Specific association with effector molecules and transcription factors directs<br>
one of skill to the mode of action of PSCA v.4, and thus identifies therapeutic, prognostic, preventative and/or diagnostic<br>
targets for cancer. This and similar assays are also used to identify and screen for small molecules that interact with PSCA<br>
v.4.<br>
[0640] Thus it is found that PSCA v.4 associates with proteins and small molecules. Accordingly, PSCA v.4 and<br>
these proteins and small molecules are used for diagnostic, prognostic, preventative and/or therapeutic purposes.<br>
Example 50: Involvement of PSCA v.4 in cell-cell communication<br>
[0641 ] Cell-cell communication is essential in maintaining organ integrity and homeostasis, both of which become<br>
deregulated during tumor formation and progression. Based on the presence of a cadhesrin motif in PSCA v.4, a motif<br>
known to be involved in cell interaction and cell-cell adhesion, PSCA v.4 can regulate cell communication. Intercellular<br>
communications can be measured using two types of assays (J. Biol. Chem. 2000,275:25207). In the first assay, cells<br>
loaded with a fluorescent dye are incubated in the presence of unlabeled recipient cells and the cell populations are<br>
examined under fluorescent microscopy. This qualitative assay measures the exchange of dye between adjacent cells, in<br>
the second assay system, donor and recipient cell populations are treated as above and quantitative measurements of the<br>
recipient cell population are performed by FACS analysis. Using these two assay systems, cells expressing PSCA v.4 are<br>
compared to controls that do not express PSCA v.4, and it is found that PSCA v.4 enhances cell communications. Small<br>
molecules and/or antibodies that modulate cell-cell communication mediated by PSCA v.4 are used as therapeutics for<br>
cancers that express PSCA v.4. When PSCA v.4 functions in cell-cell communication and small molecule transport, it is<br>
used as a target or marker for diagnostic, prognostic, preventative and/or therapeutic purposes.<br>
[0642] Throughout this application, various website data content, publications, patent applications, and patents are<br>
referenced. (Websites are referenced by their Uniform Resource Locator, or URL, addresses on the World Wide Web.)<br>
The disclosures of each of these references are hereby incorporated by reference herein in their entireties. In addition, this<br>
application relates to U.S. Serial No. 09/359,326, filed July 20,1999; U.S. Serial No. 09/308,503, filed May 25,1999; U.S.<br>
Serial No. 09/251,835, filed February 17,1999; U.S. Serial No. 09/203,939, filed December 2,1998; U.S. Serial No.<br>
09/038,261, filed March 10,1998; U.S. Serial No. 08/814,279, filed March 10,1997; U.S. Serial No. 60/071,141 filed<br>
January 12,1998; U.S. Serial No. 60/074,675, filed February 13,1998; U.S. Serial Nos. 60/124,658, filed March 16,1999;<br>
U.S. serial No. 60/120,536 filed February 17,1999; and 60/113,230 filed December 21,1998. The contents of all of the<br>
foregoing applications are fully incorporated by reference into the present application.<br>
[0643] The present invention is not to be limited in scope by the embodiments disclosed herein, which are intended<br>
as single illustrations of individual aspects of the invention, and any that are functionally equivalent are within the scope of<br>
the invention. Various modifications to the models and methods of the invention, in addition to those described herein, will<br>
become apparent to those skilled in the art from the foregoing description and teachings, and are similarly intended to fall<br>
within the scope of the invention. Such modifications or other embodiments can be practiced without departing from the<br>
true scope and spirit of the invention.<br>
TABLE III: Amino Acid Substitution Matrix<br>
Adapted from the GCG Software 9.0 BLOSUM62 amino acid substitution matrix (block substitution matrix). The<br>
higher the value, the more likely a substitution is found in related, natural proteins. (See world wide web URL<br>
ikp.unibe.ch/manual/blosum62.html)<br>
I / We Claim:<br>
1. An isolated polynucleotide that encodes a PSCA protein, wherein the<br>
polynucleotide is selected from the group consisting of:<br>
(a) a polynucleotide comprising the sequence of SEQ ID NO: 6537,<br>
from nucleotide residue numbers 424 through 993;<br>
(b) a polynucleotide comprising the sequence of SEQ ID NO: 6537,<br>
from nucleotide residue numbers 424 through 993, wherein the nucleotide<br>
residue at 521 is T;<br>
(c) a polynucleotide comprising the sequence of SEQ ID NO: 6537,<br>
from nucleotide residue numbers 424 through 993, wherein the nucleotide<br>
residue at 578 is C;<br>
(d) a polynucleotide comprising the sequence of SEQ ID NO: 6537,<br>
from nucleotide residue numbers 424 through 993, wherein the nucleotide<br>
residue at 649 is G;<br>
(e) a polynucleotide comprising the sequence of SEQ ID NO: 6537,<br>
from nucleotide residue numbers 424 through 993, wherein the nucleotide<br>
residue at 653 is T;<br>
(f) a polynucleotide comprising the sequence of SEQ ID NO: 6537,<br>
from nucleotide residue numbers 424 through 993, wherein the nucleotide<br>
residue at 721 is A;<br>
(g) a polynucleotide comprising the sequence of SEQ ID NO: 6537,<br>
from nucleotide residue numbers 424 through 993, wherein the nucleotide<br>
residue at 781 is A;<br>
(h) a polynucleotide comprising the sequence of SEQ ID NO: 6537,<br>
from nucleotide residue numbers 424 through 993, wherein the nucleotide<br>
residue at 788 is G;<br>
(i) a polynucleotide comprising the sequence of SEQ ID NO: 6537, from<br>
nucleotide residue numbers 424 through 993, wherein the nucleotide residue at<br>
989 is A;<br>
(j) a polynucleotide comprising the sequence of SEQ ID NO: 6533,<br>
from nucleotide residue numbers 423 through 707; and<br>
(k) a polynucleotide comprising the sequence of SEQ ID NO: 6545,<br>
from nucleotide residue numbers 83 through 427.<br>
2. A polynucleotide that is fully complementary to a polynucleotide of any<br>
one of (a)-(k) of claim 1.<br>
3. A polynucleotide of claim 1 that encodes the polypeptide sequence shown<br>
in SEQ ID NO: 6540, 6547, 6548, 6549, 6550, 6551, 6552, 6553, 6554,<br>
6536, or 6546.<br>
4. A polynucleotide of claim 1 that encodes the polypeptide sequence shown<br>
in SEQ ID NO: 6540 wherein P33L, Y52S, H76D, P77L, A100T, G120S,<br>
I122S, or R189Q.<br>
5. A recombinant expression vector comprising a polynucleotide of any one<br>
of claims 1-4.<br>
6. A process for producing a PSCA protein comprising culturing a host cell<br>
containing the recombinant expression vector of claim 5 under conditions<br>
sufficient for the production of the PSCA protein.<br>
7. The process of claim 6, optionally comprising recovering the PSCA<br>
protein so produced.<br>
8. The process of claim 7, wherein the protein is recovered using<br>
chromatography.<br>
9. A PSCA protein produced by the process as described in any one of<br>
claims 6 to 8.<br>
10. A composition comprising a pharmaceutically acceptable carrier and a<br>
protein of claim 9.<br>
11. An isolated PSCA protein, wherein the PSCA protein comprises SEQ ID<br>
NO: 6540, 6547, 6548, 6549, 6550, 6551, 6552, 6553, 6554, 6536, or 6546.<br>
12. An antibody or functional fragment thereof that immunospecifically binds<br>
to an epitope on a PSCA protein having amino acid sequence as set forth in<br>
SEQ ID NO: 6540, 6547, 6548, 6549, 6550, 6551, 6552, 6553, 6554, 6536,<br>
or 6546.<br>
13. The antibody or functional fragment thereof of claim 12, which is<br>
monoclonal.<br>
14. The antibody or functional fragment thereof of claim 13, wherein the<br>
monoclonal antibody is a recombinant protein.<br>
15. The antibody or functional fragment thereof of claim 14, which is a single<br>
chain monoclonal antibody.<br>
16. The antibody or functional fragment thereof as described in any one of<br>
claims 12, 13, 14, or 15, wherein the fragment is an Fab, F(ab')2, Fv or Sfv<br>
fragment.<br>
17. The antibody or functional fragment thereof of claim 12, wherein said<br>
antibody is a human antibody.<br>
18. The antibody or functional fragment thereof of any one of claims 12, 13,<br>
14, 15 or 16, wherein said antibody is labeled with a cytotoxic agent.<br>
19. The antibody or functional fragment thereof of claim 18, wherein the<br>
cytotoxic agent is selected from the group consisting of radioactive isotopes,<br>
chemotherapeutic agents and toxins.<br>
20. The antibody or functional fragment thereof of claim 19, wherein the<br>
radioactive isotope is selected from the group consisting of 211At, 131I, 125I,<br>
90Y, 186Re, 188Re, 153Sm, 212Bi, 32P and radioactive isotopes of Lu.<br>
21. The antibody or functional fragment thereof of claim 19, wherein the<br>
chemotherapeutic agent is selected from the group consisting of taxol,<br>
actinomycin, mitomycin, etoposide, tenoposide, vincristine, vinblastine,<br>
colchicine, gelonin, and calicheamicin.<br>
22. The antibody or functional fragment thereof of claim 19, wherein the toxin<br>
is selected from the group consisting of diphtheria toxin, enomycin,<br>
phenomycin, Pseudomonas exotoxin (PE) A, PE40, abrin, abrin A chain,<br>
mitogellin, modeccin A chain, and alpha-sarcin.<br>
23. The antibody or functional fragment thereof of any one of claims 12, 13,<br>
and 14 to 22, wherein the antibody or functional fragment thereof optionally<br>
comprises a pharmaceutically acceptable carrier.<br>
24. A vector comprising a polynucleotide encoding a monoclonal antibody<br>
according to claims 13 or 14.<br>
25. A composition comprising a PSCA protein epitope having amino acid<br>
sequence as set forth in SEQ ID NO: 6540, 6547, 6548, 6549, 6550, 6551,<br>
6552, 6553, 6554, 6536, or 6546 for inducing an immune response in a<br>
subject.<br>
26. The composition as claimed in claim 25, wherein the immune response<br>
comprises activation of a B cell, wherein the activated B cells generate<br>
antibodies that specifically bind to the PSCA protein (SEQ ID NO: 6540,<br>
6547, 6548, 6549, 6550, 6551, 6552, 6553, 6554, 6536, or 6546).<br>
27. The composition as claimed in claim 25, wherein the immune response<br>
comprises activation of a T cell, wherein the activated T cell is a cytotoxic T<br>
cell (CTL), which, when activated kills an autologous cell that expresses the<br>
PSCA protein (SEQ ID NO:6540, 6547, 6548, 6549, 6550, 6551, 6552,<br>
6553, 6554, 6536, or 6546).<br>
28. The composition as claimed in claim 25, wherein the immune response<br>
comprises activation of a T cell, wherein the activated T cell is a helper T<br>
cytotoxic activity of a CTL or antibody producing activity of a B cell.<br>
29. A composition comprising an antibody according to any one of claims 18<br>
to 23, wherein the composition delivers an agent to a cell expressing a<br>
PSCA protein having amino acid sequence as set forth in SEQ ID NO:<br>
6540, 6547, 6548, 6549, 6550, 6551, 6552, 6553, 6554, 6536, or 6546.<br>
30. A composition comprising an effective amount of an antibody according<br>
to any one of claims 12, 13, or 14 to 23, wherein the composition inhibits<br>
growth of a cell expressing a PSCA protein having amino acid sequence as<br>
set forth in SEQ ID NO: 6540, 6547, 6548, 6549, 6550, 6551, 6552, 6553,<br>
6554,6536, or 6546.<br><br>
PSCA and its encoded protein, and variants thereof, are described wherein PSCA exhibits tissue specific expression<br>
in normal adult tissue, and is aberrantly expressed in the cancers listed in Table 1. Consequently, PSCA provides a diagnostic,<br>
prognostic, propylactic and/or therapeutic target for cancer. The PSCA gene or fragment thereof, or its encoded protein, or variants<br>
thereof, or a fragment thereof, can be used to elicit a humoral or cellular immune response; antibodies or T cells reactive with PSCA<br>
can he used in active or passive immunization.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNy1LT0xOUC0yMDA1LUZPUk0tMjcucGRm" target="_blank" style="word-wrap:break-word;">2607-KOLNP-2005-FORM-27.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNy1rb2xucC0yMDA1LWdyYW50ZWQtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">2607-kolnp-2005-granted-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNy1rb2xucC0yMDA1LWdyYW50ZWQtYXNzaWdubWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">2607-kolnp-2005-granted-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNy1rb2xucC0yMDA1LWdyYW50ZWQtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">2607-kolnp-2005-granted-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNy1rb2xucC0yMDA1LWdyYW50ZWQtY29ycmVzcG9uZGVuY2UucGRm" target="_blank" style="word-wrap:break-word;">2607-kolnp-2005-granted-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNy1rb2xucC0yMDA1LWdyYW50ZWQtZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">2607-kolnp-2005-granted-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNy1rb2xucC0yMDA1LWdyYW50ZWQtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">2607-kolnp-2005-granted-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNy1rb2xucC0yMDA1LWdyYW50ZWQtZXhhbWluYXRpb24gcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">2607-kolnp-2005-granted-examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNy1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">2607-kolnp-2005-granted-form 13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNy1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSAxOC5wZGY=" target="_blank" style="word-wrap:break-word;">2607-kolnp-2005-granted-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNy1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSAyLnBkZg==" target="_blank" style="word-wrap:break-word;">2607-kolnp-2005-granted-form 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNy1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">2607-kolnp-2005-granted-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNy1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">2607-kolnp-2005-granted-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNy1rb2xucC0yMDA1LWdyYW50ZWQtZ3BhLnBkZg==" target="_blank" style="word-wrap:break-word;">2607-kolnp-2005-granted-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNy1rb2xucC0yMDA1LWdyYW50ZWQtcmVwbHkgdG8gZXhhbWluYXRpb24gcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">2607-kolnp-2005-granted-reply to examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwNy1rb2xucC0yMDA1LWdyYW50ZWQtc3BlY2lmaWNhdGlvbi5wZGY=" target="_blank" style="word-wrap:break-word;">2607-kolnp-2005-granted-specification.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="233640-communication-system.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="233642-a-peg-linked-polypeptide.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>233641</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2607/KOLNP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>14/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>03-Apr-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>01-Apr-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>15-Dec-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>AGENSYS, INC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1545, 17TH STREET, SANTA MONICA CA 90404</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>CHALLITA EID PIA M</td>
											<td>15745 MORRISON STREET ENCINO, CA 91436</td>
										</tr>
										<tr>
											<td>2</td>
											<td>JAKOBOVITS AYA</td>
											<td>3135 HUTTON DRIVE, BEVERLY HILLS, CA 90210</td>
										</tr>
										<tr>
											<td>3</td>
											<td>GE WANGMAO</td>
											<td>4838 HOLLOW CORNER ROAD APARTMENT 314, CULVER CITY CA 90230</td>
										</tr>
										<tr>
											<td>4</td>
											<td>RAITANO ARTHUR B</td>
											<td>12685 ROSE AVENUE, LOS ANGELES CA 90066</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2004/017231</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-05-28</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/475,064</td>
									<td>2003-05-30</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/233641-prostate-stem-cell-antigen-psca-variants by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:49:07 GMT -->
</html>
