<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/233755-pharmacogenetic-marker-of-human-transforming-growth-factor-beta1-tgfss1-for-predicting-predispodition-to-immunological-disorders by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:52:50 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 233755:PHARMACOGENETIC MARKER OF HUMAN TRANSFORMING GROWTH FACTOR BETA1 (TGFß1) FOR PREDICTING PREDISPODITION TO IMMUNOLOGICAL DISORDERS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">PHARMACOGENETIC MARKER OF HUMAN TRANSFORMING GROWTH FACTOR BETA1 (TGFß1) FOR PREDICTING PREDISPODITION TO IMMUNOLOGICAL DISORDERS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Asthma is a chronic lung disorder characterized by inflammation as a result of complex interactions between cells, their mediators and tissues in the airways. Various genetic and environmental factors are known to affect the disease process. Of the genetic factors, human Transforming Growth Factor Betal (TGFß1) is an important candidate gene for causation of susceptibility and/or pathogenesis. TGFß1 plays a dual role. On one hand, where it suppresses inflammation and decreases airway hyper reactivity in lungs, it also has a pro-inflammatory role where it is one of the mediators involved in subepithelial fibrosis, which is a fundamental feature of the remodeled airway in chronic asthmatics. The present invention relates to allelic variants of the human Transforming Growth Factor Betal (TGFß1)gene and provides primers and methods suitable for the detection of these allelic variants for applications such as molecular diagnosis, prediction of an individual&#x27;s disease susceptibility, and/or the genetic analysis of the (TGFß1) gene in a population. Specifically, the invention provides a method for detection of predisposition to atopic disorders /other immunological disorders such as, autoimmune disorders, inflammatory disorders (especially in concern with liver), cancer, multiple sclerosis, fibrosis, tuberculosis, sarcoidosis, hypertension and disorders developing due to hypertension, diabetes and disorders developing due to diabetes, alcohol abuse, anxiety, asthma, chronic obstructive pulmonary disease (COPD), cholecystectomy, degenerative joint disease (DJD), seizure disorder, arthritis, etc. where human Transforming Growth Factor Betal(TGFß1)plays an important role.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The effects of such polymorphisms can be at various levels of cellular organization. Polymorphic elements in the promoter and/or regulatory regions are known to modulate the levels of mRNA of the genes. Polymorphisms in the un-translated regions (UTR's) of the RNA have also been documented to regulate the transcriptional and translational rates of the genes. Their presence in the intron-exon boundaries can also lead to changes in splicing and or splice products that are formed from the native full length mRNA. Polymorphisms in the coding region may change the function of the protein if it is a non-synonymous change and if it occurs in a critical domain of the protein leading to functional changes of the protein.<br>
Thus polymorphisms are useful in defining genomic regions (for example as genetic markers) and they may also lead to disease (for example functional polymorphisms). Numerous examples are documented in the scientific literature and persons trained in this field are familiar with it (please see Abney M et al, Am J Hum Genet 70:920-34, 2002; Baron M, Mol Psychiatry 6:143-9, 2001; Bodmer WF, Ciba Found Symp 130:215-28, 1987; Breslow JL, Physiol Rev 68:85-132, 1988; Caraballo LR and Hernandez M, Tissue Antigens 35:182-6, 1990; Levitt RC, Am J Respir Crit Care Med 150:S94-9, 1994; Xu J et al, Clin Exp Allergy 28 Suppl 5:1-5; discussion 26-8, 1998).<br>
Atopic diseases are a clinically heterogeneous group of diseases characterized by elevated serum IgE levels and varying phenotypic expressions such as Asthma and Atopic Dermatitis (Barnes KC, Clin Exp Allergy 29 Suppl 4:47-511999; Barnes PJ , Respir Res 2:64-5, 2001; Blumenthal MN and Amos DB, Chest 9I:176S-184S, 1987; Thomas NS et al, Am J Respir Crit Care Med 156:S144-51, 1997). Specifically, Asthma is a chronic airway disease, affecting 15-18% of the world's population. It is mainly a childhood disorder though the age of onset can vary and is seen to be 35-45 yr. in the general population. Another case of extrinsic asthma is observed where the age of onset is above 45 yr. and is mainly due to the age induced changes in the lung function. The pathophysiology of atopic asthma is well documented. It is a T helper type 2 (Th2) mediated disorder with cytokines such as interleukin-4, interleukin-5, interleukin-13, implicated in the deviation of the immune system towards atopicity. Increased levels of these cytokines lead to elevated total serum IgE levels, eosinophil recruitment, and bronchial hyper-responsiveness that ultimately culminate in asthma pathogenesis. These interieukins are also known to interact and stimulate the alveolar cells and bronchial smooth muscle cells resulting in the clinical phenotypes of bronchial hyper-responsiveness (Barnes PJ, Respir Res 2:64-5, 1999). Gene-gene and gene-environment interactions have been implicated in the development of asthma (Tay et al, Asian Pac J Allergy Immunol 17:239-42, 1999; Bleecker ER, Am J Respir Crit Care Med 156:S113-6, 1997; Cookson W, Nature 402:B5-11, 1999).<br>
Although inflammation is undoubtedly a cornerstone of asthma, the structural changes associated with asthma with respect to the cells and tissues of the airways have escaped the importance deserved. These structural changes are collectively given the term "Airway Remodeling" and the complex picture of asthma would be incomplete without giving this aspect its due relevance.<br>
Various genetic studies have shown multiple loci to be associated with the disease. Asthma is therefore a multigenic disorder with a number of genes contributing minor effects leading to pathogenesis. Linkage studies, in various populations, have narrowed down the presence of susceptibility or disease genes to chromosomal locations such as lp31, 5q31-33, llpl3, 12ql3-24, 13ql4, 17ql2-21. However, all the causative genes and mutations have so far not been identified (Bleecker ER et al, Am J Respir Crit Care Med 156:S113-6, 1997; Blumenthal MN, Chest 91:176S-184S, 1987, Duffy DL, Epidemiol Rev 19:129-43, 1997). Recent advances in genome wide scans have identified several chromosomal regions such as 1 lq, 10p, 20p, 5q, 8p, 12p, 14q to be associated with asthma/atopy (Blumenthal MN et al, Hum Genet 114(2):157-164, 2004). Also a study has linked chromosomes 19, 20, 3, 12, 18, 11, 13 to mite sensitivity, which is a major risk factor for asthma (Blumenthal MN et al, Genes Immun 5(3): 226-231, 2004)<br>
Transforming growth factor beta 1 (TGFpi) plays an important role in airway wall remodeling, an established pathological feature in asthma (Elias et al, J Clin Invest 104:1001-1006, 1999; Redington et al, Am J Respir Crit Care Med 156:642-647, 1997). It is implicated in several aspects of fibrosis (Minshall et al, Am J Respir Cell Mol Biol 17:326-333, 1997) wherein subepithelial fibrosis is increased in severe asthmatics (Massague, Annu Rev Cell Biol 6:597-641, 1990). In addition, it decreases synthesis of enzymes that degrade the ECM (extracellular matrix), such as collagenase and stromelysin, and increases synthesis of inhibitors of these enzymes, including tissue inhibitor of metalloprotienase-1 (TIMP-1) and plasminogen activator inhibitor type-1 (PAI-1) (Massague, Annu Rev Cell Biol 6:597-641, 1990). TGFpl mRNA levels in eosinophils are increased in patients with severe asthma as compared to mild asthma (Minshall et al, Am J Respir Cell Mol Biol 17:326-333, 1997; Ohno et al, Am J Respir Cell Mol Biol 15:404-409, 1996). Alternatively, it prevents the development of allergic inflammation through the capacity to inhibit IgE synthesis and through inhibition of basophil and eosinophil proliferation (Taylor et al, Int Arch Allergy Immunol 135(l):73-82, 2004). Additionally, it abrogates the survival effects of hematopoietins on eosinophils and thereby induces their apoptosis (Alam et al, J Exp Med 179:1041-1045, 1994). This complex mix of pro- and anti-inflammatory activities make TGFpi a promising candidate gene for asthma.<br>
The TGFpl gene is located on chromosome 19ql3.1-13.2 (Fujii et al, Somat Cell Mol Geriet;lZ:281-288) and has recently been linked to mite sensitivity (Blumenthal et al, Genes Immun 5:226-231, 2004). Studies carried out in different populations have identified various polymorphisms, such as -988C/A, -800G/A, -509C/T, 869T/C and 915G/C. Earlier studies also reported a strong linkage disequilibrium (LD) between -509C/T, 869T/C and 915G/C (Dunning et al, Cancer Res 63(10):2610-2615, 2003; Pulleyn et al, Hum Genet 109:623-627, 2001). Out of these SNPs, the C to T transition at -509 position has been found to be associated with elevated IgE levels (Hobbs et al, Am J Respir Crit Care Med 158:1958-1962, 1998) and TGFpi levels (Grainger et al, Hum Mol Genet 8:93-97, 1999). In another study, this polymorphism was associated with asthma severity (Pulleyn et al, Hum Genet 109:623-627, 2001). In the same study, four other polymorphisms, -988C/A, -800G/A, 869T/C and 915G/C were assessed for association with asthma but no significant association was found, A similar study in the Czech population showed no association with asthma (Buckova et al, Allergy 56:1236-1237, 2001),<br>
Thus, both genetic and biochemical evidence indicate that TGFpi is a potential candidate gene for disease pathogenesis and/or susceptibility to disorders including asthma. To elucidate its genetic role in asthma, we have carried out a case-control study in two independent cohorts of asthma patients and controls (Nagpal et al, J Allergy Clin Immunol. 115(3):527-33, 2005). Here, we have genotyped a novel repeat (Accession number BV209662) and two SNPs, viz, -800G/A and the -509C/T, encompassing a region of 24.7 Kb and have analyzed the association of these polymorphisms independently and at the level of haplotype with asthma and also with serum TGFpi levels.<br>
Moreover, there is evidence to suggest that ethnic differences exist in the susceptibility genes associated with asthma (Xu J et al, Am J Hum Genet 68:1437-46, 2001). Chromosome 19ql3,1-13-2 harbors the Transforming Growth Factor Betal (TGFpi) gene (consisting of 7 exons spanning a region of 23kbp) (Fujii D et al, Somat. Cell Mol. Genet. 12:281-288, 1986). This gene mediates a complex mix of pro- and anti-inflammatory activities. Like IL-10, TGFpi indirectly inhibits T-cell activation by modulation of antigen presenting cell function and deactivating macrophages (Tsunawaki S et al, Nature 334:260-262, 1998). It also prevents the development of allergic inflammation through a capacity to inhibit IgE synthesis and through inhibition of mast cell proliferation. Additionally, it abrogates the survival [ effects of hematopoietic on eosinophils and thereby induces their apoptosis (Alam R et al, J Exp Med 179:1042-1045, 1994). It has also been shown that T cells engineered to secrete TGFpl, in contrast to IFN-y secreting Thl cells, could very effectively reduce airway inflammation and AHR (Hansen G et al, J Clin Invest 105:61-70), Furthermore, the blockade of TGFp signaling in mature T cells enhanced airway inflammation and AHR, suggesting that the regulation of T ceils via TGFp reduces inflammatory responses in the lungs (Nakao A et al, J Exp Med 192:151-158). In contrast to these observations supporting an anti-inflammatory role for TGFpi, the secretion of TGFP 1 after an allergic disorder developed contributes to fibrosis and the irreversible changes associated with airway remodeling in chronic asthma (Aubert J.D., Thorax 49:225-232, 1994). TGFpi is implicated in several aspects of fibrosis, including the deposition of extracellular matrix (ECM) components such as collagens type I and III, fibronectin, vitronectin, tenascin and proteoglycans (Massague J, Annu. Rev. Cell Biol. 6:597-641, 1990). In addition, it decreases synthesis of enzymes that degrade the ECM, such as collagenase and stromelysin, and increases synthesis of inhibitors of these enzymes, including tissue inhibitor of metalloprotienase-1 (TIMP-1) and plasminogen activator inhibitor type-1 (PAI-1) (Massague J, Annu. Rev. Cell Biol. 6:597-641, 1990). TGFpi mRNA levels in eosinophils are increased in patients with severe asthma as compared to mild asthma (Minshall et al, Am J Respir Cell Mol Biol 17:326-333, 1997; Ohno I et al, Am J Respir Cell Mol Biol 15: 404-409, 1996). Also in most, but not all studies, TGFß1 expression correlates with basement membrane thickness and fibroblast number and/or disease severity. TGFpi is therefore a promising candidate gene for asthma. A PCT patent application WO0208468 of Moscowitz (2001) titled "Diagnostic polymorphisms for the TGF-betal promoter" identified two single nucleotide polymorphisms (SNPs) at positions 216 and 563 on the TGF-pl Promoter and have shown association with various diseases including cancer, diabetes, COPD, coronary heart disease, hypertension, asthma, anxiety etc.<br>
Various studies indicate that repeats are involved in gene regulation. In addition to the above mentioned SNPs, putative repetitive sequences in and around the TGFpi gene were identified by us using the RepeatMasker™ Software (http://www.repeatmasker.org). A CT/CA repeat 24.9 kb upstream of the TGFßl gene was validated for distribution in our study population (Nagpal et al, J Allergy Clin Immunol. 115(3):527-33, 2005). It is biologically relevant to study repetitive sequences in and around the genes as it is a known fact that repetitive sequences found hundreds of bases away from the coding sequence may be involved in the regulation of gene expression through long-range interactions with the basic promoter and the trans-acting proteins. A change in repeat number can alter the nucleosomal positioning thereby altering transcriptional activity. Also, variable repeats can form secondary structures, which can augment or interfere with the gene expression (Hefferon et al, PNAS 101(10):3504-3509, 2004).<br>
Thus, modulation of the levels of TGFpi by transcriptional and translational mechanisms may be responsible for the wide array of actions mediated by this cytokine. In light of the above evidence, it appears that TGFpi could be an important genetic locus affecting the predisposition of an individual towards various immunological disorders, including atopic asthma, COPD, hypertension, anxiety, arthritis, etc.<br>
Studies carried out in different populations have identified various polymorphisms in the regulatory and coding regions of this gene. Particularly, the C to T transition at -509 position in the promoter region induces a YY1 consensus-binding site within a negative regulatory region of TGFpi transcription (Shrivastava et al, Nucleic Acids Research 24:5151-5155, 1994). This substitution has previously been associated with elevated IgE levels (Hobbs K et al, Am J Respir Crit Care Med 158:1958-1962,1998) and TGFß1 levels (Grainger et al, Hum Mol Genet 8:93-97,1999).<br>
In another study, this polymorphism was associated with asthma severity, with a greater proportion of individuals having the TT genotype in the severe asthmatics group as compared to the mild and control group (Pulleyn LJ et al, Hum Genet 109(6):623-627; 2001). In accordance with the study in the Caucasian population (Pulleyn et al, Hum Genet 109:623-627, 2001), we have found the association of the -509C/T polymorphism with asthma and serum IgE. In our study population, the CC genotype was over-represented in the control group and on being changed to CT, a significant increase in risk was observed. We also observed that the individuals with CC genotype had the lowest TGFß1 levels in their serum followed by CT while the TT homozygotes had the highest levels (Nagpal et al, J Allergy Clin Immunol. 115(3):527-33, 2005). Our observations support the study conducted by Silverman et al. (Am J Respir Crit Care Med 169(2):214-219, 2004). Also, the -509OT polymorphism plays a greater role in asthma as seen in a recent study by Silverman et al (Am J Respir Crit Care Med 169(2):214-219, 2004), which showed that the C to T transition at position -509 augments Yin-Yang 1 (YY1) transcription factor binding and enhances basal promoter function for TGFfil gene; the T allele thus being associated with higher levels of TGFpi transcript.<br>
A similar study in the Czech population however showed no association with asthma (Buckova I et al, Allergy Net 1236-1237). In the same study, four other polymorphisms, -988C/A, -800G/A, 869T/C and 915G/C, were assessed for association with asthma but no significant association was found with the asthmatic phenotype.<br>
These studies suggest that there is a component of ethnic variation that is involved which depends on the particular population under study. In the present study, the CC genotype at -509 locus is present at a higher frequency in the controls, thereby being protective in nature. The TT genotype however is present at similar frequencies in the case and control groups. Also, in initial genotyping of TGF/31 polymorphisms, two SNPs (-988 C/A and 788 GT), reported to be polymorphic in populations of other ethnicities were found to be non-polymorphic in the Indian population (N=200). The difference in results obtained in our population could be explained by ethnic differences that exist between our population and the other populations studied. Also, the sampling strategies used in the studies are different. The sampling strategy used by the applicants is a case control study although they have recruited individuals with a familial history of asthma and atopy.<br>
The two SNPs assessed in the study population lie within the promoter region of the gene. Polymorphisms in the promoter region of the gene can lead to abnormal transcriptional regulation. It is known that the C to T transition at the -509 position creates a Yin-Yang 1 (YY1) transcription factor binding site, which possibly augments the basal promoter function for the TGFß1 gene (Silverman et al, Am J Respir Crit Care Med 169(2):214-219, 2004). Also the polymorphism at -800 representing a G to A transition lies in a CREB transcription factor binding site (Grainger et al, Hum Mo! Genet 8:93-97, 1999).<br>
The present invention for the first time report a novel polymorphism wherein a CT/CA repeat 24.9 kb upstream of the TGFfil gene has been identified. This polymorphism along with two known polymorphisms viz, G/A at -800 and C/T at -509 position in the promoter of TGFß1 gene, has been shown to be associated with susceptibility for immunological disorders, asthma in particular. This is the first time demonstration of the association of a combination of three polymorphisms in the TGFß1 gene with immunological disorders, particularly asthma.<br>
Here the applicants have for the first time provided novel gene variants and haplotypes useful for prediction of immunological disorders, particularly asthma.<br>
The invention also provides novel  primers for detection of novel polymorphism, a CT/CA repeat 24,9 kb upstream of the TGFß1 gene.<br>
To summarize, this is the first study in any population, identifying novel protective and risk haplotypes of the TGFpi gene (especially in comparison to that undertaken by Silverman et al, Am J Respir Crit Care Med, 169, 214-9, 2004). These haplotypic combinations have further been functionally validated. In addition to supporting the already known association of-509C/T polymorphism with the TGFß1 levels, it is reported here the association of the (CT)n(CA)m repeat alleles with the TGFpi levels. In contrast to earlier observations, where TGFpi was primarily implicated to mediate anti-inflammatory actions; in the present study, low TGFpi levels associated with protective genotypes as well as haplotypes point towards the pro-inflammatory role of this cytokine. During the development of an allergic reaction, TGFpi is a negative regulator of inflammation, but in the case of chronic asthma (as in the patient population studied by us), where the inflammatory conditions have already established, TGFpi exacerbates lung deterioration, thus having a pro-inflammatory role (Nagpal et al, J Allergy Clin Immunol 115: 527-533, 2005). Thus, both genetic and biochemical evidence indicates that TGFß1 is a potential candidate gene for disease pathogenesis and/or susceptibility to asthma. To elucidate its role in asthma genetics, in the present study a case-control study in 2 independent cohorts namely Cohort A and Cohort B of patients with asthma and controls was performed. Here, we have genotyped a novel repeat (accession number BV209662) and 2 SNPs, 2800G/Aand 2509C/T, encompassing a region of 24.7 kb, and have analyzed the association of these polymorphisms independently and at the level of haplotype with asthma and also with serum TGF-bl levels.<br>
Objects of the invention<br>
The main object of the present invention is to provide novel protective and risk haplotypes of the Transforming Growth Factor Beta 1 (TGß1) for prediction of susceptibility to immunological disorders.<br>
Another object of the invention is to provide a method for studying association of TGFß1 haplotypic and allelic variants with immunological disorders.<br>
Still another object of the invention is to provide a method for screening disease prone individuals in a given population by screening for the TGFß1 gene variants<br>
Yet another object is to provide specific primers for detection of single nucleotide polymorphisms and specific repetitive sequences in the TGFß1 gene.<br>
Another object is to provide the haplotypes generated by the allelic variants of the TGFß1 gene in the general (control) population.<br>
Yet another object of the invention is to provide a method for predicting an individual's risk towards developing immunological disorders, specifically atopic asthma, by studying the haplotype pattern.<br>
Still another object of the invention is to provide a method for predicting an individual's tolerance to immunological disorders, specifically atopic asthma, by analyzing the haplotype pattern.<br>
Summary of the present invention:<br>
Asthma is a chronic lung disorder characterized by inflammation as a result of complex interactions between cells, their mediators and tissues in the airways. Various genetic and environmental factors are known to affect the disease process. Of the genetic factors, human Transforming Growth Factor Betal (TGFß1) is an important candidate gene for causation of susceptibility and/or pathogenesis. TGFß1 plays a dual role. On one hand, where it suppresses inflammation and decreases airway hyper reactivity in lungs, it also has a pro-inflammatory role where it is one of the mediators involved in subepithelial fibrosis, which is a fundamental feature of the remodeled airway in chronic asthmatics. The present invention is related to the haplotypes of human Transforming Growth Factor Betal (TGF(31) and prediction of susceptibility for immunological disorders. The present invention also relates to allelic variants of the human Transforming Growth Factor Betal (TGFpi) gene. The said invention also provides primers and methods suitable for the detection of these haplotypes and allelic variants for the prediction of an individual's disease susceptibility, and /or the genetic analysis of the TGFpl gene for immunological disorders. Specifically, the invention provides a method for detection of predisposition to atopic disorders such as asthma and various other immunological disorders such as, fibrosis, tuberculosis, sarcoidosis, cancer, COPD, hypertension, arthritis, heart diseases, atherosclerosis, etc.<br>
DETAILED DESCRIPTION:<br>
TGFß1 has been found to be a promising candidate gene for finding association with immunological disorders, particularly asthma as several reports have detailed occurrence of polymorphisms in this gene.<br>
A PCT patent application WO0208468 of Moscowitz (2001), titled "Diagnostic polymorphisms for the TGF-betal promoter" identified two single nucleotide polymorphisms (SNPs) at positions 216 and 563 on the TGF-pl Promoter and have shown association with various diseases including cancer, diabetes, COPD, cholecystectomy, degenerative joint disease, seizure disorders, arthritis, coronary heart disease, hypertension, asthma, anxiety etc.<br>
Studies carried out in different populations have identified various polymorphisms in the regulatory and coding regions of this gene. Particularly, the C to T transition at-509 positions in the promoter region induces a YY1 consensus-binding site within a negative regulatory region of TGFpi transcription (Shrivastava et al, Nucleic Acids Research 24:5151-5155, 1994; Silverman et al, Am J. Resp. Crtic Care Med., 169: 214-219, 2004). This substitution has previously been associated with elevated IgE<br>
levels (Hobbs K et al, Am J Respir Crit Care Med 158:1958-1962, 1998) and TGFß1 levels (Grainger et al, Hum Mol Genet 8:93-97, 1999). In another study, this polymorphism was associated with asthma severity, with a greater proportion of individuals having the TT genotype in the severe asthmatics group as compared to the mild and control group (Pulleyn LJ et al, Hum Genet 109(6):623-627; 2001) . In the same study, four other polymorphisms, -988C/A, -800G/A, 869T/C and 915G/C, were assessed for association with asthma but no significant association was found with the asthmatic phenotype. A similar study in the Czech population however showed no association with asthma (Buckova I et al, Allergy Net 1236-1237). These studies suggest that there is a component of ethnic variation that is involved which depends on the particular population under study.<br>
The present application provides a dinucleotide polymorphic repeat at nucleotide 52106 to nucleotide 52151 in the clone ACO11462, which is 24,927 bp upstream of the TGFfil gene. (GenBank contig accession no. ACO 11462). The first polymorphic site (CT/CA), as shown in Figure 1, is 25,769 bp upstream of the ATG site (Genbank Accession No. AC011462, nucleotide 52106 to nucleotide 52151; Genbank Accession No. BV209662). The second polymorphic site (S1) is situated 800 bp upstream of the human TGFfil gene and represents a G to A transition (Genbank Accession No. AC011462, nucleotide 76280). The third polymorphic site S2 is a C to T transition, situated 509 bases upstream of the TGFfil gene (Genbank Accession No. ACO 11462, nucleotide 76571).<br>
The present results of the present study provide very unique results as compared to the prior arts. In addition to identifying, genotyping and establishing a positive association of the novel CT/CA repeat with asthma, the inventors have found an association of the known -800G/A and -509C/T polymorphisms in the Indian population. In contrast to the study by Pulleyn et al, in the present study, the TT genotype at the -509 locus has a similar distribution in the patient and control group whereas the CC genotype is present at a higher frequency in the controls, thus having a protective effect in the Indian population. The association of TGFpl variants with asthma are not based only on the ethnic differences observed in the present population and the Caucasian populations, but found generally in any population of the world.<br>
The present invention has identified the novel genetic variants, which exist in any type of population in the world irrespective of its origin, community, colour, geographical location or ethnicity. The inventors have compared allele frequencies in CT/CA repeat polymorphism 24.9 kb upstream of promoter and -800 G/A, -509 C/T SNPs in the promoter and the haplotypes generated using the three loci, in a population. Also, the sampling strategies used in the present studies are different. The present study is a case control study although the inventors have recruited individuals with a familial history of asthma and atopy. Further, the invention clearly defines that the variants identified would be useful for any kind of population of any geographical origin.<br>
Earlier studies indicate that dinucleotide repeats located far upstream of the promoter are involved in gene regulation (Texereau et al, Thorax 59:156-158, 2004; Holbert et al, Proc Natl Acad Sci U S A.:98(4):181M8166, 2001; Chakraborty et al, Biochem Biophys Res Commun 252:716-722, 1998). For example, a far upstream (CA)n repeat (allele 25) was found to repress Ha-ras gene activity (Chakraborty et al, Biochem Biophys Res Commun 252:716-722, 1998). In our study, the alleles of the (CT)n(CA)m locus showed a significant difference in distribution pattern amongst the patient and the control groups Figure 2. Interestingly, the different alleles were found to be associated with different levels of TGFpi Figure 6. However, the functional role of these repeats is yet to be established.<br>
To further the understanding of the contributions of this gene towards asthma, we constructed three-locus haplotypes using PHASE (Stephens et al, Am J Hum Genet 68:978-989, 2001). The overall distribution of the different haplotypic combinations was significantly different in the patients and the controls. In addition, we identified two major haplotypes, namely 22GC and 24_G_C as protective haplotypes for asthma. In contrast, the haplotypes 21_G_C and 23_G_T increase the susceptibility of the patient to develop this disorder, marking them as the major risk haplotypes. As already validated for the (CT)n(CA)m and -509 locus individually, the protective haplotypes were associated with lower serum TGFpl levels and vice-versa. Particularly, the individuals homozygous for the 22_G_C haplotype have the lowest levels of TGFpi, followed by heterozygous individuals while the ones lacking this haplotype have the highest levels of TGFpi in their sera Figure 6. This dose dependency is also reflected in the individuals with 23_G_T, wherein individuals homozygous for this haplotype have the highest levels of TGFpi in their sera. Thus, the polymorphisms constituting the three-locus haplotypes may be controlling TGFpi levels directly or may be present in linkage disequilibrium with another true functional polymorphism (s) within and/or around the gene. Although, the number of individuals homozygous for either the 22_G_C or the 23_G_T haplotypes were few (n=5 and n=3 respectively), the difference in TGFpi levels with each haplotype was highly significant (p=0.0019 in case of 22_G_C and p=0.01 for the 23_G_T haplotype). Also, appreciable numbers of individuals, heterozygous for either haplotype, have corroborated this observation further. However, to gain an insight into tissue specific expression of TGFß1, these results have to be further validated using BAL fluid / lung biopsy samples.<br>
Since, asthma is a complex disorder, we conducted a case-control study here that could provide better directions on promising loci (Sharma et al, Clin Genet. 66(5):417-25, 2004). These leads can be further tested using a well-controlled family-based study. Indeed, we have established the genetic homogeneity between the two groups by genotyping multiple loci to eliminate the errors due to stratification or an inherent statistical bias (Sharma et al, Clin Genet. 66(5):417-25, 2004). Moreover, the additional unlinked regions used as controls (Zak et al, Drug Discov Today 6:1111-1115, 2001) and validation of our results in another independent cohort could only add more confidence to our results.<br>
Therefore the present study is of first kind wherein in any population, novel protective and risk haplotypes of the TGFpi gene have been identified. Further, the present study also shows the functionally validity of these haplotypes and the identified genetic variants. In addition to support the already known association of -509C/T polymorphism with the TGFpl levels, the present invention reports the association of the (CT)n(CA)m repeat alleles with the TGFß1 levels. In contrast to earlier observations, where TGFpl was primarily implicated to mediate anti-inflammatory actions; in present study, low TGFß1 levels associated with protective genotypes as well as haplotypes point towards the pro-inflammatory role of this cytokine. During the development of an allergic reaction, TGFß1 is a negative regulator of inflammation, but in the case of chronic asthma (as in the patient population studied in the present study), where the inflammatory conditions have already established, TGFpl exacerbates lung deterioration, thus having a pro-inflammatory role.<br>
Accordingly, the present invention provides novel haplotypes of human Transforming Growth Factor Betal (TGFß1) gene having SEQ ID Nos. 1,2 and 3 and useful for prediction of susceptibility (low or high risk) of an individual to immunological disorders, particularly asthma.<br>
In an embodiment the invention, novel haplotypes of human Transforming Growth Factor Betal (TGFß1) gene having SEQ ID Nos. 1, 2 and 3 useful for prediction of susceptibility (low or high risk) of an individual to immunological disorders, particularly asthma.<br>
Another embodiment the invention, the said haplotypes are useful for predicting and detecting humans susceptible to the immunological disorders selected from group comprising of asthma, and various other immunological disorders such as, fibrosis, tuberculosis, sarcoidosis, cancer, COPD, hypertension, arthritis, heart diseases, atherosclerosis, etc. wherein TGFß1 plays an important role.<br>
Yet another embodiment the invention, the said immunological disorder selected is asthma.<br>
Still another embodiment of the invention, the said haplotype is pharmacogenetic marker for predicting and detecting humans susceptible to the immunological disorders selected from group comprising of asthma, and various other immunological disorders such as, fibrosis, tuberculosis, sarcoidosis, cancer, COPD, hypertension, arthritis, heart diseases, atherosclerosis, etc. wherein TGFß1 plays an important role.<br>
An embodiment the invention, the haplotype is pharmacogenetic marker for predicting and detecting humans susceptible to asthma.<br>
Another embodiment of the invention relates to a method of detecting novel haplotypes of human Transforming Growth Factor Betal (TGFpi) gene having SEQ ID Nos. 1, 2 and 3 and useful for prediction of susceptibility (low or high risk) of an individual to immunological disorders. The said method comprising the steps of:<br>
(i) designing primers having sequence ID no. 4 &amp; 5.<br>
(ii) synthesizing primers having SEQ ID Nos. 4, 5 , 6, 7, 8 and 9.<br>
(iii) amplifying the genomic DNA using primers of SEQ ID Nos. 4, 5 , 6, 7, 8 and 9 wherein primers of SEQ ID Nos 4, 6 and 8 are forward primers and primers of SEQ ID Nos 5, 7 and 9 are reverse primers.<br>
(iv) isolating and identifying the DNA stretch of SEQ ID No.l using novel primer combinations of SEQ ID Nos. 4 and 5,<br>
(v) isolating and identifying the DNA stretch of SEQ ID No.2 using primer combinations of SEQ ID Nos. 6 and 7,<br>
(vi) isolating and identifying the DNA stretch of SEQ ID No.3 using primer combinations of SEQ ID Nos, 8 and 9,<br>
(vii) sequencing the isolated and identified SEQ ID Nos. 1, 2 and 3 obtained in steps (iv, v and vi)<br>
Another embodiment to the invention, SEQ ID #1 is associated with CT/CA, SEQ ID # 2 is associated with SI and SEQ ID # 3 is associated with S2 locus of the TGFfil gene.<br>
Still another aspect of the invention is to provide a   method of preparing novel pharmacogenetic    markers    for    detecting    and    predicting    predisposition    to immunological disorders of TGFß1 gene in a subject, said method comprises of: (i) designing primers having sequence ID no. 4 &amp; 5.<br>
(ii) synthesizing primers having SEQ ID Nos. 4, 5 , 6, 7, 8 and 9.<br>
(iii) amplifying the genomic DNA using primers of SEQ ID Nos, 4, 5 , 6, 7, 8 and 9 wherein primers of SEQ ID Nos 4, 6 and 8 are forward primers and primers of SEQ ID Nos 5, 7 and 9 are reverse primers.<br>
(iv) isolating and identifying the DNA stretch of SEQ ID No.l using novel primer combinations of SEQ ID Nos. 4 and 5,<br>
(v) isolating and identifying the DNA stretch of SEQ ID No.2 using primer combinations of SEQ ID Nos. 6 and 7,<br>
(vi) isolating and identifying the DNA stretch of SEQ ID No.3 using primer combinations of SEQ ID Nos. 8 and 9,<br>
(vii) sequencing the isolated and identified SEQ ID Nos. 1, 2 and 3 obtained in steps (iv, v and vi)<br>
(viii).validating and identifying the specific TGFß1 gene variants computationally by comparison with the known wild type TGFß1 gene sequences,<br>
Another embodiment of the present invention the subject is a human.<br>
Yet another aspect of the invention provides a diagnostic kit for detecting and predicting predisposition to immunological disorders by screening for TGFß1 gene haplotypes in a subject, said kit comprising of:<br>
(i) Primers corresponding to SEQ ID Nos. 4, 5 , 6, 7, 8 and 9.<br>
Field of invention<br>
The present invention is related to the haplotypes of human Transforming Growth Factor Beta I (TGFß1) and prediction of susceptibility for immunological disorders. More particularly the present study is the first study in any population, identifying novel protective and risk haplotypes of the TGFpi gene The present invention also relates to allelic variants of the human Transforming Growth Factor Betal (TGFpi) gene. The said invention also provides primers and methods suitable for the detection of these haplotypes and allelic variants for the prediction of an individual's disease susceptibility, and /or the genetic analysis of the TGFpi gene for immunological disorders, particularly asthma, A vital aspect of the present study provides a method for predicting susceptibility of individuals to immunological disorders, particularly asthma.<br>
Background Information<br>
The genomic DNA of all organisms undergoes spontaneous changes in the sequence (termed as mutation) in the course of their continuing evolution thereby generating variant forms of progenitor sequences, which may lead to various evolutionary advantages or disadvantages to the survival of the organism. If such effects of the mutations or variations are not seen then they are termed as neutral changes/mutations. If the mutation is lethal then it is not transmitted to the following generations and thus is lost from the gene pool of that organism. A variant form may also confer an evolutionary advantage to the species and is eventually incorporated into the DNA of many or most members of the species, and hence, effectively it becomes the progenitor form. In many instances, both progenitor and variant form(s) survive and co-exist in the gene pool of the species. This coexistence of multiple forms of a sequence gives rise to polymorphisms.<br>
Several different types of polymorphism have been reported, A restriction fragment length polymorphism (RFLP) means a variation in DNA sequence that alters the length of a restriction fragment., The restriction fragment length polymorphism may create or delete a restriction site, thus changing the length of the restriction fragment. RFLPs have been widely used in human and animal genetic analyses. Other polymorphisms take the form of shor tandem repeats (STRs) that include tandem di-, tri- and tetranucleotide repeated "motifs. These tandem repeats are also referred to as variable number tandem repeat (VNTR) polymorphisms. VNTRs have been used in identity and paternity analysis and in a large number of genetic mapping studies. Other polymorphisms take the form of single nucleotide variations. Such polymorphisms are far more frequent than RFLPS, STRs and VNTRs. Some single nucleotide polymorphisms (SNPs) occur in protein-coding sequences, in which case, one of the polymorphic forms may give rise to the expression of a defective or other variant protein and, potentially, a genetic disease. Examples of genes, in which polymorphisms within coding sequences give rise to genetic disease include beta.-globin (sickle cell anemia) and CFTR (cystic fibrosis). Other single nucleotide polymorphisms occur in noncoding regions. Some of these polymorphisms may also result in defective protein expression (e.g., as a result of defective splicing). Other single nucleotide polymorphisms have no phenotypic effects.<br>
(ii) Standard Human DNA for control experiment.<br>
(iii) Reagents and buffers for carrying out a polymerase chain reaction and sample extraction.<br>
In yet another embodiment of the invention, SEQ ID #1 is associated with CT/CA, SEQ ID # 2 is associated with S1 and SEQ ID # 3 is associated with S2 locus of the TGFß1 gene.<br>
In yet another embodiment of the invention, the said haplotypes of the diagnostic kit are useful for predicting and detecting humans susceptible to immunological disorders selected from group comprising of asthma, and various other immunological disorders such as, fibrosis, tuberculosis, sarcoidosis, cancer, COPD, hypertension, arthritis, heart diseases, atherosclerosis, etc. wherein TGFß1 plays an important role.<br>
In yet another embodiment of the invention, the said haplotypes of the diagnostic kit useful for predicting and detecting humans susceptible to asthma.<br>
One more embodiment of the present invention provides pharmacogenetic markers for detecting and predicting predisposition to immunological disorders by screening for TGFpi gene haplotypes in a subject, said kit comprising of:<br>
(i) Primers corresponding to SEQ ID Nos. 4, 5 , 6, 7, 8 and 9.<br>
(ii) Standard Human DNA for control experiment,<br>
(iii) Reagents and buffers for carrying out a polymerase chain reaction and sample extraction.<br>
Detailed Methodology:<br>
Isolation of genomic DNA from peripheral blood leukocytes of the atopic<br>
asthmatic patients and the normal control individuals:	 <br>
Genomic DNA was isolated from the peripheral blood of the patients and controI individuals using a modified salting out procedure (Nagarkatti R et al., 2002). Briefly, 10ml blood was obtained from patients and unrelated control individuals using ACD Vaccutainers (BD Biosciences, San Jose, CA, USA). Equal volume of ice cold C1 buffer (4X) was added and then 30ml of ice cold sterile water was added to cause cell membrane lysis (Promega Genomic DNA Isolation Handbook). Following this, the nuclei were pelleted at 1300xg for 15 min at 4oC. The pellet was washed again with 1X Cl buffer. 12 ml of nuclear lysis buffer was added with 0.8ml of 10% SDS. 50µl of a 20µg/µl solution of proteinase-K was added and the pellet resuspended by brief vortexing. After incubation at 65°C for 2-3 hrs, the proteinaceous material was precipitated with the addition of 4 ml of 6M NaCl After centrifugation for 15 min at 2500 rpm, the supernatant was transferred to another tube and two volumes of absolute ethanol (at room temperature) was used to precipitate the DNA (Miller et al., 1988). The precipitated DNA was then washed with 70% ethanol twice, air-dried, and dissolved in TE buffer. Appropriate dilutions (1:100, in T.E buffer) were used to determine the OD at 260nm and 280nm. DNA quality was assessed using the 260nm/280nm ratio. The stock solution of the DNA was diluted to 50ng/µl and used for PCR amplification and genotyping experiments. The stock DNA solution was stored at -20°C.<br>
Identification of putative repeats in and around the TGFfil gene using RepeatMasker™ Software.:<br>
A novel CT/CA repeat was identified to be present 24.9 kb upstream of the TGFfil gene in the study population using RepeatMasker™ Software {http://www.repeatmasker.org).<br>
Designing and synthesis of two new oligonucleotide primers (sequence ID # 4, 5) for PCR amplification of 363 bp region, 24.9 kb upstream of TGFfil gene containing CT/CA polymorphism:<br>
PCR amplification of genomic DNA samples isolated from peripheral blood leukocytes of the atopic asthmatic patients and normal control individuals using the primers of Figure 2. For the CT/CA locus, PCR was carried out in a total volume of 15µl containing 25ng of genomic DNA, 4 pmol each of a 6-FAM-labeled forward primer and a non-labeled reverse primer, l.5Mm MgCl2, 0.25mM of each dNTP, 0.03U/µl of Taq DNA polymerase (Bangalore Genie, India) and the buffer recommended by the supplier. After PCR, lµ1 of the PCR product was loaded with and internal size standard (ROX labeled) on ABI Prism 3100 Genetic Analyzer (Applied Biosystems). Fragment lengths were determined using the GeneScan Analysis Software version 3 .7 (Applied Biosystems). The number of repeat for each loci was determined by sequencing PCR fragments of individuals (N=5) being homozygous for one allele. Repeat sizes were calculated using the formula n= {(flanking region-allele size)/2, rounding off to 0 decimal values}.<br>
PCR amplification of 388 and 418 bp stretches of the promoter region of TGFßl gene containing -800 G/A and -509 C/T polymorphisms respectively:<br>
PCR amplification of genomic DNA samples isolated from peripheral blood leukocytes of the atopic asthmatic patients and normal control individuals using the primers of Figure 2. For the-800 G/A polymorphism, PCR was carried out in a total volume of 25µl containing 50ng of genomic DNA, 1.25 pmol each of forward and reverse primers (sequence ID # 6, 7), 1.5mM MgCl2, 0.25mM of each dNTP, 4% DMSO, 0.03U/µl of Taq DNA polymerase (Bangalore Genie, India) and the buffer recommended by the supplier.<br>
For the -509 C/T polymorphism, PCR was carried out in a total volume of 10µl containing 25ng of genomic DNA, 2.5 pmol each of forward and reverse primers (sequence ID # 8, 9), 1.5mM MgCl2, 0.25mM of each dNTP, and 0.03U/µl of Taq DNA polymerase (Bangalore Genie, India) and the buffer recommended by the supplier.<br>
Direct sequencing of the purified PCR products:<br>
Direct sequencing of the purified PCR products using dye terminator chemistry was carried out on an ABI Prism 3100 automated DNA sequencer. Sequencing was<br>
carried out using specific primers on an ABI 3100 capillary sequencer (Applied Biosystems, Foster City, CA, USA) for a minimum of 20 atopic asthmatic and 20 control individuals. Nested primers were used for sequencing the entire PCR amplicons  PCR product was gel purified for sequencing. Briefly, sequencing primers, diluted to lpmol per µl, and 75-150ng/|d PCR product were added to 5µl reaction mix, and volume made up to l0µl with autoclaved MilliQ water as per the Big Dye Terminator kit instructions (Applied Biosystems, Foster City, CA, USA). PCR was set up with the following conditions: 96°C for 5 seconds, 55°C for 30 seconds and 60°C for 4 minutes. Sequencing reactions were purified with 70% ethanol washes to remove unincorporated primers and fluorescent ddNTPs. Briefly, 26µl autoclaved MilliQ water was added to the sequencing reaction. Sixty-four microliters of chilled 100% ethanol was added to the tubes and vortexed. The tubes were centrifiiged at 16,000 g for 20 minutes at room temperature. Washes were performed with 70% ethanol by centrifugation at 16,000 g for 5 minutes. The pellets were air dried and resuspended in 10µl of 100% Hi-Dye formamide. The tubes were incubated at 94°C for 5 minutes and placed in the 3100 Automated Sequencer. Sequence analysis was carried out using Sequence Navigator (ver 2.1, Applied Biosystems, Foster City, CA, USA) and DNAStar (ver 1.1, DNASTAR) software. Homozygous and heterozygous alleles were scored manually.<br>
Validation of -800 G/A polymorphism:<br>
The -800 G/A polymorphisms were studied using SNaPshot ddNTP Primer Extension Kit (Applied Biosystems, Foster City, USA). SnaPshot PCR was carried out using 50ng purified PCR template, i pmol primer, ABI ready reaction mix and 1X dilution buffer (as supplied by the manufacturer). PCR was set up with the following conditions: 96°C for 10 seconds, 50°C for 5 seconds and 60°C for 30 seconds for a total of 35 cycles. To clean up the primer extension reaction, 1U of calf intestinal phosphatase (CIP) diluted in 10X NEB3 (New England Biolabs), was added to the reaction mixture and the mixture was incubated at 37°C for 1 hour, followed by an incubation for 15 minutes at 72°C for enzyme inactivation. These samples were subsequently electrophoresed using the ABI Prism 3100 Genetic Analyzer as per the manufacturer's instructions. The results were analyzed using the program ABI Prism GeneScan™ and Genotyper™ (Applied Biosystems, Foster City, USA).<br>
Calculating and estimating the frequency of single nucleotide polymorphisms, -800G/A and -509C/T and that of repeat polymorphism CT/CA:<br>
Kolmogorov-Smirnov test has been used to test for allelic association with disease for CT/CA locus (Figure 3). Odds ratios were calculated and Chi-square test for association with phenotype, was carried out. The repeats have been denoted with the allele size (19, 20 etc for CT/CA), the genotypes with (19/20 for CT/CA), whereas the SNPs are designated as SI and S2.<br>
Estimating the frequencies of haplotypes generated using the above mentioned three loci in the normal individuals and atopic asthmatic patients for finding association between these haplotypes and the disease:<br>
Novel Haplotypes have been generated using the PHASE program for the patient and control groups (Stephens M, Am J Hum Genet. Nov 73:1162-9, 2003). Default parameters were used to generate the haplotypes. No missing values were allowed (http://archimedes.well.ox.ac.uk/pise/PHASE-simple.html, PHASE Ver, 2.0.2). Odds ratios were calculated and Chi-square test for association with phenotype, was carried out. The haplotypes are designated as CT/CA_S1_S2 or 19_G_C, 22G C etc.<br>
So the matter in which the above mentioned features, advantages and the objects of the invention, as well as others which will become clear, are attained and can be understood in detail.. These drawings form a part of the specification. It is to be noted, however, that the appended drawings illustrate preferred embodiments of the invention and therefore not to be considered limiting in their scope.<br>
BRIEF DESCRIPTION OF THE ACCOMPANYING DRAWINGS:<br>
Figure 1: The gene structure and polymorphisms investigated in the TGFßl gene. The positions marked in bold were found polymorphic in the Indian population<br>
Figure 2: Primer details for the PCR.<br>
Figure 3: Alleles frequencies of the CT/CA polymorphism upstream of the TGFßl.<br>
Figure 4: Distribution of Allele and genotype frequencies for -800 G/A and -509 C/T polymorphism.<br>
Figure 5: Multiple logistics regression analysis of the TGFßl polymorphisms for risk of asthma.<br>
Figure 6: Serum TGFß1 levels in context of alleles, genotypes and haplotypes<br>
Figure 7: The frequency distribution of the haplotypes of the TGFßl gene in patients and unrelated controls in cohorts A and B, respectively. The haplotypes were plotted on the X-axis and their respective relative frequencies (%) on the Y-axis.<br>
Figure 8: Profile and characterstics of study populations<br>
Association of the CT/CA repeat locus with atopic disorders such as asthma:<br>
To demonstrate the association of the CT/CA repeat locus with atopic disorders such as asthma, Kolmogorov-Smirnov test was performed (Nagarkatti et al., 2000; Carriere and Kochar 2000, Mukawa et. al., 1989). The Kolmogorov-Smirnov Test is a non-parametric test, which gives the likelihood of two ordered categorizations coming from different orderings or the same ordering. The Kolmogorov-Smirnov test (KS-test) tries to determine if two datasets differ significantly. The KS-test has the advantage of being more powerful than the chi-square test in many situations. We identified a novel (CT)n(CA)m repeat polymorphism, 24.9 kb upstream of the TGFßl gene (Fig. 1) and used it for genotyping patients and controls in two independent cohorts. A total of seven alleles were obtained for cohort A (heterozygosity index=0.728) and six alleles for cohort B (heterozygosity index=0.730) (Fig. 3). A highly significant difference was obtained in the allele count in the patients and the<br>
controls (KS X2=3422, df=6, p=0.00001 in cohort A and KS x2= 11.29, df=5, p=0.003 in cohort B). The distribution was statistically significant after Bonferroni correction (aα=0.007, bα=0.008) in both the cohorts. One-way ANOVA showed an association between the alleles at this locus and log total serum IgE levels in the patients {F-ratio (F)=2.76, df=6, p-0.012}, Tukey-Kramer HSD (Honestly Significant Difference) showed significant difference between mean log IgE levels for the pair of alleles: 22 and 23. However, none of the genotypes at the CT/CA locus was associated with log total serum IgE levels (F=L34, df=16, p=0,17)<br>
Demonstration of association of-800 G/A polymorphism with asthma:<br>
To demonstrate the association of-800 G/A polymorphism, X2 test was performed. Of the three genotypes, AA, GA, GG; AA was the rare genotype with only one individual in the control group having AA at -800 locus. Excluding AA genotype from the analysis, a significant difference in the distribution of the genotypes was observed between the patient and the control groups (x2=4.89, df=l, p=0.026) (Figure 4).<br>
To demonstrate the association of-509 C/T polymorphism, x2 test was performed. The pattern of distribution of the three genotypes, CC, CT, TT was significantly different in the two groups studied (x2=10.03, df=l, p=0.G06). The CC genotype was predominant in the controls as compared to the patient group {OR=0.51, 99% C.I (0.30, 0.89)} (Fig. 4).<br>
Logistic regression analysis for the polymorphisms investigated<br>
Using multiple logistic regression analyses, we confirmed the association between the (CT)n(CA)m, -800G/A and -509C/T polymorphisms with asthma and it was found not to be influenced by sex and age (Figure 5).<br>
Generation of haplotypes:<br>
We then used the PHASE program to generate haplotypes for the patient and control groups. The program PHASE implements a new statistical method for reconstructing haplotypes from population genotype data. Experiments with the software on both real and simulated data indicate that it can provide an improvement on the EM algorithm for reconstructing haplotypes. It allows for missing genotype data and also can handle more than one locus irrespective of the polymorphism, for example SNPs and repeats can be analyzed simultaneously. Based on the output from the software the probability values of the haplotypes are also predicted and can be utilized to differentiate more confident haplotypes. The PHASE software is suitable for genetic distances of 100 cM or less. The probability values for the chromosomes with uncertain phase ranged form 0.51 to 0.65 for both the groups; these chromosomes accounted for only 2.07% of the control and 2.60% of the patient chromosomes. The haplotypes are shown in Figure 7. The haplotypes 21G _C and 23_G_T were the most frequent in the patient population. The odds of patients rather than controls having 21_G_C was 2.45 with 99% CI= (1.58, 3.94) [Likelihood x2=27.92, df=l, p=0.00001] and 1.38 with 95% CI=(1.01, 1.87) [Likelihood x2=4.21, df=l, p=0.04] for cohort A and B respectively. Similarly, the odds of patients rather than controls having 23 _G_T was 2.64 with 99% CI= (1.51, 4.59) [Likelihood x2=21.77, df=l, p=0,00001]   and  3.55   with   99%  CI=(2.00,   6.29)   [Likelihood  x2=37.22,  df=l,<br>
p=0.0000l] for cohort A and cohort B respectively. These results suggest that 21_G_C (in cohort A) and 23_G_T (in both cohorts) are risk/susceptibility haplotypes and are positively associated with asthma. On the other hand, the haplotypes 22GC and 24_G_C were the most frequent in the control population. The odds (OR) of patients rather than controls having 22_G_C was 0.18 with 99% CI=(0.10, 0.31) [Likelihood x2=35.32, df=l, p=0.00001] and 0.33 with 99% CI=(0.20, 0.54) [Likelihood x2=72.28, df=l, p=0.00001] for cohort A and B respectively. Similarly, the odds of patients rather than controls having 24_G_C was 0.48 with 95% CI=(0.24, 0.98) [Likelihood x2=4-28, d=1, p=0.038] and 0.28 with 99% CI=(0.12, 0.68) [Likelihood x2=16.01, df=l, p=0.00001] for cohort A and B respectively. These results suggest that 22_G_C (in both cohorts) and 24_G_C (in cohort B) are protective haplotypes and are negatively associated with asthma,<br>
II. Analysis of Polymorphisms:<br>
A.	Preparation of Samples:<br>
Polymorphisms are detected in a target nucleic acid from an individual being analyzed. For assay of genomic DNA, virtually any biological sample (other than pure red blood cells) is suitable. For example, convenient tissue samples include whole blood, semen, saliva, tears, urine, fecal material, sweat, buccal, skin and hair. For assay of cDNA or mRNA, the tissue sample must be obtained from an organ in which the target nucleic acid is expressed.<br>
Many of the methods described below require amplification of DNA from target samples. This can be accomplished by e.g., PCR. See generally PCR Technology: Principles and Applications for DNA Amplification (ed. H. A. Erlich, Freeman Press, N.Y., N.Y., 1992); PCR Protocols; A Guide to Methods and Applications (eds. Innis, et al., Academic Press, San Diego, Calif., 1990); Mattila et al., Nucleic Acids Res. 19, 4967 (1991) and U.S. Pat No. 4,683,202 (each of which is incorporated by reference for all purposes).<br>
Other suitable amplification methods include the Ligase Chain Reaction (LCR) (see Barringer KJ et al, Gene 89:117-22, 1990; Friedhoff P et al, Anal Biochem 215:9-16, 1993) and Nucleic Acid Based Sequence Amplification (NASBA). The latter two amplification methods involve isothermal reactions based on isothermal transcription, which produce both single stranded RNA (ssRNA) and double stranded DNA (dsDNA) as the amplification products in a ratio of about 30 or 100 to 1, respectively.<br>
B.	Detection of Polymorphisms in Target DNA:<br>
There are two distinct types of analysis depending on whether a polymorphism in question has already been characterized or not. The first type of analysis is sometimes referred to as de novo characterization. This analysis compares target sequences in different individuals to identify points of variation, i.e., polymorphic sites. By analyzing groups of individuals representing the greatest ethnic diversity among humans and greatest breed and species variety in plants and animals, patterns characteristic of the most common alleles/haplotypes of the locus can be identified, and the frequencies of such populations in the population determined. Additional<br>
allelic frequencies can be determined for subpopulations characterized by criteria such as geography, race, or gender. The de novo identification of the polymorphisms of the invention is described in the Examples section. The ftecond type of analysis is determining which form(s) of a characterized polymorphisniare present in individuals under test. There are a variety of suitable procedures, which are discussed in turn.<br>
1.	Repeat detection (size variation detection):<br>
The design and use of primers flanking the sequence contain the repeat sequence or other polymorphic elements, which lead to a size difference. PCR amplification of the sequence leads to the presence of a pool of amplified products that differ by the specific repeat or polymorphism size. These size differences can then be detected using gel based, charge based methods. Usually for gel based detection one of the primers is labeled with a fluorescent compound which can then be excited and detected using a CCD camera or other methods.<br>
2.	Allele-Specific Probes:<br>
The design and use of allele-specific probes for analyzing polymorphisms is described by e.g., Saiki et al. Nature 324, 163-166, 1986; Dattagupta, EP 235,726, Saiki, WO 89/11548. Allele-specific probes can be designed that hybridize to a segment of target DNA from one individual but do not hybridize to the corresponding segment from another individual due to the presence of different polymorphic forms in the respective segments from the two individuals.<br>
3.	Allele-Specific Primers:<br>
An allele-specific primer hybridizes to a site on target DNA overlapping a polymorphism and only primes amplification of an allelic form to which the primer exhibits perfect complementarity. This primer is used in conjunction with a second primer which hybridizes at a distal site. See, e.g., WO 93/22456.<br>
4.	Direct-Sequencing:<br>
The direct analysis of the sequence of polymorphisms of the present invention can be accomplished using either the dideoxy chain termination method or the Maxam Gilbert method (see Sambrook et al., Molecular Cloning, A Laboratory Manual (2nd Ed., CSHP, New York 1989); Zyskind et al., Recombinant DNA Laboratory Manual, (Acad. Press, 1988)).<br>
5.	Denaturing Gradient Gel Electrophoresis:<br>
Amplification products generated using the polymerase chain reaction can be analyzed by the use of denaturing gradient gel electrophoresis. Different alleles can be identified based on the different sequence-dependent melting properties and electrophoretic migration of DNA in solution. Erlich, ed., PCR Technology, Principles and Applications for DNA Amplification, (W.H. Freeman and Co, New York, 1992), Chapter 7.<br>
6.  Single-Strand Conformation Polymorphism Analysis:<br>
Alleles of target sequences can be differentiated using single-strand conformation polymorphism analysis, which identifies base differences by alteration in electrophoretic migration of single stranded PCR products, as described in Orita et al., Proc. Nat. Acad. Sci. 86, 2766-2770, 1989, Amplified PCR products can be generated as described above, and heated or otherwise denatured, to form single stranded amplification products. Single-stranded nucleic acids may refold or form secondary structures which are partially dependent on the base sequence. The different electrophoretic mobilities of single-stranded amplification products can be related to base-sequence difference between alleles of target sequences.<br>
III. Methods of Use:<br>
After determining polymorphic form(s) present in an individual at one or more polymorphic sites, this information can be used in a number of methods,<br>
A, Correlation of Polymorphisms with Phenotypic Traits:<br>
Atopic diseases are heterogeneous in nature and as such there are many sub-phenotypes and traits to which the association can be observed. The polymorphisms of the invention may contribute to the phenotype of an organism in different ways. As described above, the polymorphisms may act at various levels of cellular organization by which the disease phenotypes are observed as the end result. These polymorphisms may yield different selection advantages or disadvantages. For example, a heterozygous sickle cell mutation confers resistance to malaria, but a homozygous sickle cell mutation is usually lethal. A single polymorphism may affect more than one phenotypic trait.<br>
Likewise, a single phenotypic trait may be affected by polymorphisms in different genes. Further, some polymorphisms predispose an individual to a distinct mutation that is causally related to a certain phenotype. Phenotypic traits include diseases that have known but hitherto unmapped genetic components. Phenotypic traits also include symptoms of, or susceptibility to, multifactorial diseases of which a component is or may be genetic, such as atopy, autoimmune diseases, inflammation, cancer, diseases of the nervous system, and infection by pathogenic microorganisms. Some examples of autoimmune diseases include systemic lupus erythematosus, rheumatoid arthritis, diabetes, multiple sclerosis, (insulin-dependent and non-independent), and Graves disease. Some examples of cancers include cancers of the breast, bladder, colon, brain, etc. As such, phenotypic traits also include characteristics, for example, susceptibility or receptivity to particular drugs or therapeutic treatments.<br>
To perform association analysis of the disease phenotypes and genetic markers, the presence or absence of a set of polymorphisms (i.e. a polymorphic set) is determined for a set/ population of the individuals, some of whom exhibit a particular trait termed variously as case/ patients/ affected/ diseased individuals etc, and some of which exhibit lack of the trait termed variously as control individuals/ normal etc. The alleles of each polymorphism of the set are then counted to determine if the presence or absence of a particular allele or a set of alleles or a haplotype is associated with the<br>
trait of interest. Test for such associations can be performed by standard statistical methods such as a X2 test, Kolmogrov-Smirnov test, etc. Based on the values obtained for the hypothesis tested for example, if the allele X is present more in patients than in controls and if the allele X is not present more in patients than in controls, the significance value is obtained. If this value lies in a particular range then it determines the significance level of the correlations. For example, it might be found that the presence of allele Al at polymorphic site 1 correlates with cystic fibrosis disease. As a further example, it might be found that the combined presence of allele Al at polymorphic site 1 and allele Bl at polymorphic site 2 correlates with 10 fold-increased severity of cystic fibrosis.<br>
Such associations can be of immediate benefit if an extremely strong correlation exists. For example, detection of cystic fibrosis polymorphism Al and Bl in a patient may allow for rapid diagnosis and discrimination from other diseases which exhibit similar phenotypes; it can also allow for treatment if available; it can allow for screening of neonates for detection and/or for susceptibility and/or risk assessment; it can allow for selection of better and improved management methods for the disease from those which are available; it may allow for the treatment to be given if it is determined that the polymorphic site also correlates with particular therapeutic regimes and that such therapeutic drugs are more beneficial to the patient than other drugs.<br>
B. Genetic Mapping of Phenotypic Traits:<br>
The previous section concerns identifying correlations between phenotypic traits and polymorphisms that directly or indirectly contribute to those traits. The present section describes identification of a physical linkage between a genetic locus associated with a trait of interest and polymorphic markers that are not associated with the trait, but are in physical proximity with the genetic locus responsible for the trait and co-segregate with it. Such analysis is useful for mapping a genetic locus associated with a phenotypic trait to a chromosomal position, and thereby cloning gene(s) responsible for the trait. Please see (Altshuler D et al, 1998, N Engl J Med 338:1626; Cargill M et al, 1999, Nat Genet 22:231-8; Chang C, 1988, Proc Natl Acad Sci U S A 85:6856-60; Hacia JG et al, 1999, Nat Genet 22:164-7; Hirschhorn JN et al,<br>
2000,	Proc Natl Acad Sci U S A 97:12164-9; Lander ES and Botstein D, 1986, Proc Natl Acad Sci U S A 83:7353-7; Lander ES, 1993, Nat Genet 4:5-6; Reich DE et al,<br>
2001,	Nature 411:199-204; Sachidanandam R et al, 2001, Nature 409:928-33. Genes localized by linkage can be cloned by a process known as directional cloning.<br>
Computer programs are available for the calculation of lod scores for differing values of theta. Other references on linkage and disease mapping use above mentioned approaches include, Kreutz R et al, 1995, Proc Natl Acad Sci U S A 92:8778-82; de Gouyon B et al, 1993, Proc Natl Acad Sci U S A 90:1877-81; Julier C et al, 1990, Proc Natl Acad Sci U S A 87:4585-9; Oberle 1 et al, 1986, Proc Natl Acad Sci U S A 83:1016-20; Lathrop GM et al, 1984, Proc Natl Acad Sci U S A 81:3443-6; Cohen D et al, 1984, Proc Natl Acad Sci U S A 81; 1774-8.<br>
IV, Modified Polypeptides and Gene Sequences<br>
The invention further provides variant forms of nucleic acids. These variants can be used to identify the chromosomal backgrounds of individuals and depending on the particular haplotype, risk may be assessed. The promoter polymorphism may also be important in the production of variant gene constructs containing the gene of interest so as to allow heterologus expression of the gene in various human and non-human cell lines. 5'-UTR polymorphism may lead to variant expression level changes due to transcriptional or post translational modifications,<br>
V. Kits<br>
The invention further provides kits comprising at least one specific oligonucleotide pair. For example, the same substrate can be used as a template for allele-specific oligonucleotide probes for detecting all of the polymorphisms listed. For initial screening purposes, the CT/CA repeat polymorphism found 24.9 kb upstream of the human TGFßl gene could be useful as the allele 22 of this polymorphism is negatively associated, whereas the allele 23 is positively associated with asthma. For this locus, PCR was carried out in a total volume of 5µl containing 25ng of genomic DNA, 1.25 pmol each of a 6-FAM-labelled forward primer and a non-labeled reverse primer, 1.5Mm MgC12, 0.25mM of each dNTP, 0.03U/µl of Taq DNA polymerase (Bangalore Genie, India) and the buffer recommended by the supplier. After PCR, lµl of the PCR product was loaded with an internal size standard (ROX labeled) on ABI Prism 3100 Genetic Analyzer (Applied Biosystems), Fragment lengths were determined using the GeneScan Analysis Software version 3 .7 (Applied Biosystems). The number of repeat for each loci was determined by sequencing PCR fragments of individuals (N-5) being homozygous for one allele. Repeat sizes were calculated using the formula n= {(flanking region-alleie size)/2, rounding off to 0 decimal values}. If subsequently required, genotyping at the other two loci, namely, -800 G/A and -509 C/T could also be carried out<br>
Optional additional components of the kit include, for example, restriction enzymes, reverse-transcriptase or polymerase, the substrate nucleoside triphosphates, means used to label (for example, an avidin-enzyme conjugate and enzyme substrate and chromogen if the label is biotin), and the appropriate buffers for reverse transcription, PCR, or hybridization reactions. Usually, the kit also contains instructions for carrying out the methods.<br>
The invention of the instant Application is illustrated by the help of following examples and should not be construed to limit the scope of the invention.<br>
EXAMPLE 1<br>
Patient and Healthy Volunteers<br>
Unrelated patients were recruited from various hospitals in Northern India in cohort A (N=187) and Western India in cohort B (N=209), respectively (Figure 8). Ethical approval was obtained from the review board of each hospital. Written informed consent was obtained from all individuals participating in the study. Asthma, in the recruited study population was defined by clinical history and validated later by interview questions (details of environmental factors, family history of asthma/atopy, the geographical region of origin and migration status). Patients (mean age 28.42 ± 0.97 and 34.22 yrs ±1.01 yrs for cohort A and B, respectively) were diagnosed for asthma on the basis of National Asthma Education and Prevention Program (Expert Panel Report-2) guidelines (National Asthma Education and Prevention Program, J Allergy Clin Immunol 110:S141-S219, 2002) and were examined for a self-reported history of breathlessness and wheezing. The clinical parameters are summarized in figure 8, Each patient showed airway reversibility as documented by an inhalant bronchodialator-induced improvement of more than 15% (using albuterol/salbutamol). Fifteen common environmental allergens were used for the skin prick test. Atopy was defined as having wheal reaction equal to or greater than histamine (3 mm diameter). All asthmatic subjects were positive to at least one antigen used for skin prick test (SPT). Total serum IgE levels were estimated for all individuals, except few individuals (
Healthy Volunteers (referred to as normal controls) (N= 187 &amp; 190 in cohort A and B, respectively) were recruited on the basis of the criteria of having no symptoms or history of allergic diseases. Individuals having a history of smoking and parasitic/helminthic infestations in the past were excluded from the study. All control individuals recruited for the study were screened negative for all the allergens used to perform SPT.<br>
In this study, as the samples were collected from individuals on the basis of their family history, origin and migration status, the error due to stratification is presumably minimized (Figure 8). Indeed, we have established the genetic homogeneity between the two groups by genotyping multiple loci, as yet unlinked to asthma or related atopic disorders ((Nagarkatti et al, J Allergy Clin Immunol 110:410-412, 2002)). Also, all individuals were age and sex matched (Figure 8).<br>
EXAMPLE 2<br>
To demonstrate the association of the CT/CA repeat locus with atopic disorders such as asthma, Kolmogorov-Smirnov test was performed (Nagarkatti et al., 2000; Carriere and Kochar 2000, Mukawa et. al., 1989). The Kolmogorov-Smirnov Test is a non-parametric test, which gives the likelihood of two ordered categorizations coming from different orderings or the same ordering. The Kolmogrov-Smirnov test (KS-test) tries to determine if two datasets differ significantly. The KS-test has the advantage of being more powerful than the chi-square test in many situations. This analysis showed significant differences between the allele count distribution of patient and control<br>
groups [probability value (p) = 10-6]. We observed a significantly different pattern of distribution of the alleles between the two groups; alleles 21 and 23 were over-represented in the patient group whereas alleles 22 and 24 were the major alleles in the control group (Fig. 3). When the cases and controls were compared with respect to the log total serum IgE levels, a highly significant difference was obtained (for cohort A: t-test=3.58, df=255, p
EXAMPLE 2<br>
To demonstrate the association of-800 G/A polymorphism, X2 test was performed. Of the three genotypes, AA, GA, GG; AA was the rare genotype with only one individual in the control group having AA at -800 locus (Fig. 4). Excluding AA genotype from the analysis, a significant difference in the distribution of the genotypes was observed between the patient and the control groups (X2=4.89, df=l, p=0.G26).<br>
EXAMPLE 3<br>
To demonstrate the association of -509 C/T polymorphism, X2 test was performed. The pattern of distribution of the three genotypes, CC, CT, TT was significantly different in the two groups studied (x2=10.03, df=1, p=0.006) (Fig. 4). The CC genotype was predominant in the controls as compared to the patient group {OR=0.51,99%C.I. (0.30,0.89)}<br>
EXAMPLE 4<br>
PHASE was used to construct haplotypes. It allows for missing genotype data and also can handle more than one locus irrespective of the polymorphism, for example SNPs and repeats can be analyzed simultaneously. Based on the output from the software the probability values of the haplotypes are also predicted and can be utilized to differentiate more confident haplotypes. The PHASE software is suitable for genetic distances of 100 cM or less. The probability values for the chromosomes with uncertain phase ranged form 0.51 to 0.65 for both the groups; these chromosomes accounted for only 2.07% of the control and 2.60% of the patient chromosomes. The haplotypes are shown in Figure 7. The haplotypes 21_G_C and 23_G_T were the most frequent in the patient population. The odds of patients rather than controls having 21_G_C was 2.45 with 99% CI= (1.58, 3.94) [Likelihood X2=27.92, df=l, p=0.00001] and 1.38 with 95% CI=(1.01, L87) [Likelihood X2=42l, df=l, p=0.04] for cohort A and B respectively. Similarly, the odds of patients rather than controls having 23_G_T was 2.64 with 99% CI= (1.51, 4.59) [Likelihood X2=21.77, df=l, p=0.00001] and 3.55 with 99% CI=(2.00, 6.29) [Likelihood x2=37.22, df=l, p=0.00001] for cohort A and cohort B respectively. These results suggest that 21_G_C (in cohort A) and 23_G_T (in both cohorts) are risk/susceptibility haplotypes<br>
and are positively associated with asthma. On the other hand, the haplotypes 22_G_C and 24_G_C were the most frequent in the control population. The odds (OR) of patients rather than controls having 22_G_C was 0.18 with 99% CI=(0.10, 0.31) [Likelihood x2=35,32, df=1, p=0.Q0001] and 0,33 with 99% CI=(0.20, 0.54) [Likelihood x2=72.28, df=l, p=0.00001] for cohort A and B respectively. Similarly, the odds of patients rather than controls having 24_G_C was 0.48 with 95% CI=(0.24, 0.98) [Likelihood x2=4.28, df=l, p=0.038] and 0.28 with 99% CI=(0.12, 0.68) [Likelihood x2=16.0l, df=l, p=0.00001] for cohort A and B respectively. These results suggest that 22GC (in both cohorts) and 24_G_C (in cohort B) are protective haplotypes and are negatively associated with asthma.<br>
Alternative embodiments of the invention can be envisaged by those skilled in the art from the information contained herein. All such alternative embodiments are intended to lie within the scope of this application.<br>
The reader's attention is directed to all papers and documents which are filed concurrently with this specification, and the contents of all such papers and documents are incorporated herein by reference.<br>
All the features disclosed in this specification (including any accompanying claims, abstract and drawing), and/or all of the steps or any method or process so disclosed, may be combined in any combination, except combination where at least some of such features and/or steps are mutually exclusive.<br>
Each feature disclosed in this specification (including any accompanying claims, abstract and drawings), may be replaced by alternative features serving the same or equivalent or similar purpose, unless expressly stated otherwise. Thus, unless expressly slated otherwise, each feature disclosed is one example only of a generic series of equivalent or similar features.<br>
The invention is not restricted to the details of the embodiments. This invention extends to any novel one, or any novel combination, or the features disclosed in this specification (including any accompanying claims, abstract and drawings), or to any novel one, or any novel combination, of the steps of any method or process so<br>
disclosed.<br>
Although the present invention has been described in some detail by way of illustration and example for purposes of clarity and understanding, it will be apparent that certain changes and modifications may be practiced within the scope of the appended claims.<br>
Sequence Listing:<br>
General Information APPLICANT: CSIR<br>
TITLE  OF  INVESTIGATION:   Transforming Growth  Factor beta  1   (TGFßl) Haplotypes and prediction of susceptibility for immunological disorders.<br>
NUMBER OF SEQUENCES: 9<br>
CORRESPONDENCE ADDRESS: Institute of Genomics and Integrative Biology<br>
Tel 00-91-11-27666157 Fax.00-91-27667471<br>
i.        INFORMATION FOR SEQ ID No: 1<br>
ii. SEQUENCE CHARACTERISTICS: consists of CT/CA dinucleotides at nucleotide 77 to 122<br>
iii.      LENGTH: 363 bases<br>
iv.       TYPE: DNA<br>
v. 5' TTA AGC CCG GGA GGT CAA GGT TGC AGT AAA CTA TGA TCG CTC CAC TGC ACT CCA GCC TGG GCA ACA GAG CGA GAC CCT GTC TCT CTC TCT CTC TCT CTC TCT CTC TCT CAC ACA CAC ACA CAA AGA GGG AGA CAG CAG GGT GGA CAA AAG CCA GAG GTG CAT GCA CTC TGA AGA CAC CAA CCA GCG GAG TCT GAG ATC AGA GGG AAG ACA GAG CCA AAA GCC AGG CAA CAA AGA TGA GAG ACA TAA AAC AAG AAG AAC CGT AAC TGG AAA TCT CAG AAG CCC ACA CAC AGC TGG GAA GTG AGA CTC CAG GCC TAA GGC AGA TCG GCA CAG CTG TCA ACC TGC CAG ATG CTG GCC TTA GTG GGG 3'<br>
vi.       ORGANISM: Human (Natural sequence)<br>
vii.      IMMIDIATE: Natural sequence<br>
viii.     NAME/KEY: N.A<br>
ix.       SEQUENCE ID # 1<br>
i.	INFORMATION FOR SEQ ID No: 2<br>
ii. SEQUENCE CHARACTERISTICS: consists of G/A polymorphism at nucleotide 326<br>
iii.       LENGTH: 388 bases<br>
iv.       TYPE: DNA<br>
v. 5' CCC GGC TCC ATT TCC AGG TGT GGT CCC AGG ACA GCT TTG GCC GCT GCC AGC TTG CAG GCT ATG GAT TTT GCC ATG TGC CCA GTA GCC CGG GCA CCC ACC AGC TGG CCT GCC CCA CGT GGC GGC CCC TGG GCA GTT GGC GAG AAC AGT TGG CAC GGG CTT TCG TGG GTG GTG GGC CGC AGC TGC TGC ATG GGG ACA CCA TCT ACA GTG GGG CCG ACC GCT ATC GCC TGC ACA CAG CTG CTG GTG GCA CCG TGC ACC TGG AGA TCG GCC TGC TGC TCC GCA ACT TCG ACC GCT ACG GCG TGG AGT GCT GAG GGA CTC TGC CTC CAA CGT CAC CAC CAT CCA CAC CCC GGA CAC ' CCA GTG ATG GGG GAG GAT GGC ACA GTG GTC AAG AGC A 3'<br>
vi.       ORGANISM: Human (Natural sequence)<br>
vii.      IMMIDIATE: Natural sequence<br>
viii.     NAME/KEY: N.A<br>
ix.       SEQUENCE ID # 2<br>
i.	INFORMATION FOR SEQ ID No: 3<br>
ii. SEQUENCE CHARACTERISTICS: consists of C/T polymorphism at nucleotide 229<br>
iii.       LENGTH: 418 bases<br>
iv.       TYPE: DNA<br>
v. 5' CAG ACT CTA GAG ACT GTC AGA GCT GAC CCC AGC TAA GGC ATG GCA CCG CTT CTG TCC TTT CTA GGA CCT CGG GGT CCC TCT GGG CCC AGT TTC CCT ATC TGT AAA TTG GGG ACA GTA AAT GTA TGG GGT CGC AGG GTG TTG AGT GAC AGG AGG CTG CTT AGC CAC ATG GGA GGT GCT CAG TAA AGG AGA GCA ATT CTT ACA GGT GTC TGC CTC CTG ACC CTT CCA TCC TTC AGG TGT CCT GTT GCC CCC TCC TCC CAC TGA CAC CCT CCG GAG GCC CCC ATG TTG ACA GAC CCT CTT CTC CTA CCT TGT TTC CCA GCC TGA CTC TCC TTC CGT TCT GGG TCC CCC TCC TCT GGT CGG CTC CCC TGT GTC TCA TCC CCC GGA TTA AGC CTT CTC CGC CTG GTC CTC TTT CTC TGG TGA C 3'<br>
vi.       ORGANISM: Human (Natural sequence)<br>
vii.      IMMIDIATE: Natural sequence<br>
viii.     NAME/KEY: Synthetic oligonucleotide<br>
ix.       SEQUENCE ID # 3<br>
i.	INFORMATION FOR SEQ ID No: 4<br>
ii.        SEQUENCE CHARACTERISTICS: Forward primer for SEQ ID No. 1 iii.       LENGTH: 20 bases<br>
iv.	TYPE: DNA<br>
v.	5' CCC CAC TAA GGC CAG CAT CT 3'<br>
vi.	ORGANISM: Human (Natural sequence)<br>
vii.	IMMIDIATE: Natural sequence<br>
viii.	NAME/KEY: Synthetic oligonucleotide<br>
ix.	SEQUENCE ID # 4<br>
i.	INFORMATION FOR SEQ ID No: 5<br>
ii.	SEQUENCE CHARACTERISTICS: Reverse primer for SEQ ID No. 1<br>
iii.	LENGTH: 19 bases<br>
iv.	TYPE: DNA<br>
v.	5' TTA AGC CCG GGA GGT CAA G 3'<br>
vi.	ORGANISM: Human (Natural sequence)<br>
vii.	IMMEDIATE: Natural sequence<br>
viii.	NAME/KEY: Synthetic oligonucleotide<br>
ix.	SEQUENCE ID # 5<br>
i.	INFORMATION FOR SEQ ID No: 6<br>
ii.	SEQUENCE CHARACTERISTICS: Forward primer for SEQ ID No. 2<br>
iii.	LENGTH: 20 bases<br>
iv.	TYPE: DNA<br>
v.	5' CCC GGC TCC ATT TCC AGG TG 3'<br>
vi.	ORGANISM: Human (Natural sequence)<br>
vii.	IMMEDIATE: Natural sequence<br>
viii.	NAME/KEY: Synthetic oligonucleotide<br>
ix.	SEQUENCE ID #6<br>
i.	INFORMATION FOR SEQ ID No: 7<br>
ii.	SEQUENCE CHARACTERISTICS: Reverse primer for SEQ ID No. 2<br>
iii.	LENGTH: 20 bases<br>
iv.	TYPE: DNA<br>
v.	5' TGC TCT TGA CCA CTG TGC CA 3'<br>
vi.	ORGANISM: Human (Natural sequence)<br>
vii.	IMMIDIATE: Natural sequence<br>
viii.	NAME/KEY: Synthetic oligonucleotide<br>
ix.	SEQUENCE ID # 7<br>
i.	INFORMATION FOR SEQ ID No: 8<br>
ii	SEQUENCE CHARACTERISTICS: Forward primer for SEQ ID No. 3<br>
iii	LENGTH: 20 bases<br>
iv	TYPE: DNA<br>
v	5' CAG ACT CTA GAG ACT GTC AG 3'<br>
vi	ORGANISM: Human (Natural sequence)<br>
vii	IMMD3IATE: Natural sequence<br>
viii	NAME/KEY: Synthetic oligonucleotide<br>
ix	SEQUENCE ID # 8<br>
i.	INFORMATION FOR SEQ ID No: 9<br>
ii	SEQUENCE CHARACTERISTICS: Reverse primer for SEQ ID No. 3<br>
iii	LENGTH: 19 bases<br>
iv	TYPE: DNA<br>
v	5' GTC ACC AGA GAA AGA GGA C 3'<br>
vi	ORGANISM: Human (Natural sequence)<br>
vii	IMMEDIATE: Natural sequence<br>
viii	NAME/KEY: Synthetic oligonucleotide<br>
ix	SEQUENCE ID # 9<br>
References:<br>
1.	Abney M, Ober C, McPeek MS (2002) Quantitative-trait homozygosity and association mapping and empirical genomewide significance in large, complex pedigrees: fasting serum-insulin level in the Hutterites. Am J Hum Genet 70:920-34.<br>
2.	Alam R et al (1994) Transforming growth factor beta abrogates the effects of hematopoietins on eosinophils and induces their apoptosis. J Exp Med 179: 1041-1045.<br>
3.	Altshuler D, Kruglyak L, Lander E (1998) Genetic polymorphisms and disease. N Engl J Med 338; 1626.<br>
4.	Anonymous. National Asthma Education and Prevention Program, Expert Panel Report, Guidelines for the Diagnosis and Management of Asthma Update on Selected Topics-2002. J Allergy Clin Immunol 2002;110:S141-S219.<br>
5.	Aubert JD, Dalai BI, Bai TR et al (1994) Transforming growth factor pi gene expression in human airways. Thorax 49:225-232.<br>
6.	Babron MC, Selinger-Leneman H, Dizier MH, Clerget-Darpoux F (2001) Homogeneity of asthma genome scan results. Genet Epidemiol 21 Suppl l:S44-8.<br>
7.	Barringer KJ, Orgel L, Wahl G, Gingeras TR (1990) Blunt-end and single-strand ligations by Escherichia coli ligase: influence on an in vitro amplification scheme. Gene 89:117-22.<br>
8.	Baron M (2001) The search for complex disease genes: fault by linkage or fault by association? Mol Psychiatry 6:143-9.<br>
9.	Barnes PJ (2001) Th2 cytokines and asthma: an introduction. Respir Res 2:64-5.<br>
10.	Barnes KC (1999) Gene-environment and gene-gene interaction studies in the molecular genetic analysis of asthma and atopy. Clin Exp Allergy 29 Suppl 4:47-51.<br>
11.	Barnes KC, Neely JD, Duffy DL, Freidhoff LR, Breazeale DR, Schou C, Naidu RP, et al (1996) Linkage of asthma and total serum IgE concentration to markers on chromosome 12q: evidence from Afro-Caribbean and Caucasian populations. Genomics 37:41-50.<br>
12.Ben-Asouli Y, Banai Y, Pel-Or Y, Shir A, Kaempfer R (2002) Human interferon-gamma mRNA autoregulates its translation through a pseudoknot that activates the interferon-inducible protein kinase PKR, Cell 108:221-32.<br>
13.	Bleecker ER, Postma DS, Meyers DA (1997) Evidence for multiple genetic susceptibility loci for asthma. Am J Respir Crit Care Med 156:S113-6.<br>
14.	Blumenthal MN, Amos DB (1987) Genetic and immunologic basis of atopic responses. Chest 91:176S-184S.<br>
15.	Blumenthal MN, Langefeld CD (2004) A genome-wide search for allergic response (atopy) genes in three ethnic groups: Collaborative Study on the Genetics of asthma. Hum Genet 114(2):157-164,<br>
16.	Blumenthal MN, Ober C (2004) Genome scan for loci linked to mite sensitivity: the Collaborative Study on the Genetics of Asthma (CSGA). Genes Immun 5(3):226-231.<br>
17.	Bodmer WF (1987) The human genome sequence and the analysis of multifactorial traits. Ciba Found Symp 130:215-28.<br>
18.	Breslow JL (1988) Apolipoprotein genetic variation and human disease. Physiol Rev 68:85-132.<br>
19.	Buckova I, Izakovicova LH, Vacha J TGFß1gene polymorphisms. Allergy Net 1236-1237.<br>
20.	Caraballo LR, Hernandez M (1990) HLA haplotype segregation in families with allergic asthma. Tissue Antigens 35:182-6.<br>
21.	Cargill M, Altshuler D, Ireland J, Sklar P, Ardlie K, Patii N, Shaw N, et al (1999) Characterization of single-nucleotide polymorphisms in coding regions of human genes. Nat Genet 22:231-8.<br>
22.	Carriere KC, Kochar SC(2000): Comparing sub-survival functions in a competing  risks model.Lifetime Data Anal.: 6(l):85-97.<br>
23.	Chakraborty AK and Hodgson CP. Role of far upstream repressor elements controlling proto-Ha-ras gene transcription. Biochem Biophys Res Commun 1998;252:716-722.<br>
24.	Chang C, Bowman JL, DeJohn AW, Lander ES, Meyerowitz EM (1988) Restriction fragment length polymorphism linkage map for Arabidopsis thaliana. Proc Natl Acad Sci USA 85:6856-60.<br>
25.	Christie L, Hine RJ, Parker JG, Burks W (2002). Food allergies in children affect nutrient intake and growth. J Am Diet Assoc.102 (11): 1648-51.<br>
26.	Cohen D, Cohen O, Marcadet A, Massart C, Lathrop M, Deschamps I, Hors J, et al (1984) Class II HLA-DC beta-chain DNA restriction fragments differentiate among HLA-DR2 individuals in insulin-dependent diabetes and multiple sclerosis. Proc Natl Acad Sci U S A 81:1774-8.<br>
27.	Cookson WO, Young RP, Sandford AJ, Moffatt MF, Shirakawa T, Sharp PA, Faux JA, et al (1992) Maternal inheritance of atopic IgE responsiveness on chromosome llq. Lancet 340:381-4.<br>
28.	Cookson W (1999) The alliance of genes and environment in asthma and allergy. Nature 402:B5-11.<br>
29.	Dunning AM, Ellis PD, McBride S, Kirschenlohr HL, Healey CS, Kemp PR, et al, A transforming growth factorbetal signal peptide variant increases secretion in vitro and is associated with increased incidence of invasive breast cancer. Cancer Res, 2003;63(10):2610-2615.<br>
30.	Duffy DL (1997) Genetic epidemiology of asthma. Epidemiol Rev 19:129-43.<br>
31.	Elias JA, Zhu Z, Chupp G, Homer RJ. Airway remodeling in asthma, J Clin Invest 1999;104:1001-1006.<br>
32.	Erlich HA (eds), Freeman Press, N.Y., N.Y., 1992); PCR Protocols: A Guide to Methods and Applications (eds. Innis, et ah, Academic Press, San Diego, Calif, 1990.<br>
33.	Friedhoff P, Hahn M, Wolfes H, Pingoud A (1993) Quantitative polymerase chain reaction with oligodeoxynucleotide ligation assay/enzyme-linked immunosorbent assay detection. Anal Biochem 215:9-16.<br>
34.	Fujii D, Brissenden J.E, Derynck R, Francke U (1986) Transforming growth factor beta gene maps to human chromosome 19 long arm and to mouse chromosome 7. Somat Cell Mol Genet 12:281-288<br>
35.	de Gouyon B, Melanitou E, Richard MF, Requarth M, Hahn IH, Guenet JL, Demenais F, et al (1993) Genetic analysis of diabetes and insulitis in an interspecific cross of the nonobese diabetic mouse with Mus spretus. Proc Natl Acad Sci U S A 90:1877-81.<br>
36.	Ghosh B, Genetic studies of atopic asthma in Indian Population, In: International Symposium on Genes, Evolution and Complex disease, February, 2005, at NCBS, Banglore.<br>
37.	Ghosh B. Genetics of Asthma In: Workshop on Genetic Epidemiological Methods for dissection of complex human traits, Organized by TG-ISI, Kolkota and University of Pittsburgh , USA, December 2004.<br>
38.	Grainger DJ, Heathcote K, Chiano M, Sneider H, Kemp PR et al Genetic control of the circulating concentration of transforming growth factor (31. Hum Mol Gen 8:93-97<br>
39.	Hacia JG, Fan JB, Ryder O, Jin L, Edgemon K, Ghandour G, Mayer RA, et al (1999) Determination of ancestral alleles for human single-nucleotide polymorphisms using high-density oligonucleotide arrays. Nat Genet 22:164-7.<br>
40.	Hansen G, Mclntire JJ, Yeung VP, Berry G, Thorbecke GJ, Chen L, DeKruyff RH, Umetsu DT (2000) CD4+ Th cells engineered to produce latent TGFbl reverse allergen-induced airway hyperreactivity and inflammation. J Clin Invest 105:61-70<br>
41.	Hefferon WT, Groman DJ, Yurk EC, Garry RC (2004) A variable dinucleotide repeat in the CFTR gene contributes to phenotype diversity by forming RNA secondary structures that alter splicing, PNAS 101(10):3504-3509.<br>
42.	Heinzmann A, Grotherr P, Jerkic SP, Lichtenberg A, Braun S, Kruse S, Forster J, et al (2000) Studies on linkage and association of atopy with the chromosomal region 12ql3-24. Clin Exp Allergy 30:1555-61.<br>
43.	Hill S, Herlaar E, Le Cardinal A, van Heeke G, Nicklin P (1999) Homologous human and murine antisense oligonucleotides targeting stat6. Functional effects on germline cepsilon transcript. Am J Respir Cell Mol Biol 21:728-37.<br>
44.	Hirschhorn JN, Sklar P, Lindblad-Toh K, Lim YM, Ruiz-Gutierrez M, Bolk S, Langhorst B, et al (2000) SBE-TAGS: an array-based method for efficient single-nucleotide polymorphism genotyping. Proc Natl Acad Sci U S A 97:12164-9.<br>
45.	Hobbs K, Negri J, Klinnert M, Rosenwasser L, Borish L (1998) Interleukin-10 and transforming growth factor-P promoter polymorphisms in allergies and asthma. Am J Respir Crit Care Med 158: 1958-1962<br>
46.	Holbert S, Denghien I, Kiechle T, Rosenblatt A, Wellington C, Hayden MR, et al.  The  Gin-Ala  repeat transcriptional   activator  CA150   interacts  with<br>
huntingtinineuropathologic and genetic evidence for a role in Huntington's disease pathogenesis.Proc Natl Acad Sci USA. 2001 ;98(4): 1811-18166.<br>
47.	Julier C, de Gouyon B, Georges M, Guenet JL, Nakamura Y, Avner P, Lathrop GM (1990) Minisatellite linkage maps in the mouse by cross-hybridization with human probes containing tandem repeats. Proc Natl Acad Sci U S A 87:4585-9.<br>
48.	Kreutz R, Hubner N, James MR, Bihoreau MT, Gauguier D, Lathrop GM, Ganten D, et al (1995) Dissection of a quantitative trait locus for genetic hypertension on rat chromosome 10. Proc Natl Acad Sci U S A 92:8778-82.<br>
49.	Kwoh DY, Davis GR, Whitfield KM, Chappelle HL, DiMichele LJ, Gingeras TR (1989) Transcription-based amplification system and detection of amplified human immunodeficiency virus type 1 with a bead-based sandwich hybridization format. Proc Nati Acad Sci U S A 86:1173-7.<br>
50.	Lander ES, Botstein D (1986) Strategies for studying heterogeneous genetic traits in humans by using a linkage map of restriction fragment length polymorphisms. Proc Natl Acad Sci U S A 83:7353-7.<br>
51.	Lander ES (1993) Finding similarities and differences among genomes. Nat Genet 4:5-6.<br>
52.	Lathrop GM, Lalouel JM, Julier C, Ott J (1984) Strategies for multilocus linkage analysis in humans. Proc Natl Acad Sci U S A 81:3443-6.<br>
53.	Levitt RC (1994) Molecular genetic methods for mapping disease genes. Am J Respir Crit Care Med 150:S94-9.<br>
54.	Longo DR, Johnson JC, Kruse RL, Brownson RC, Hewett JE (2001): A prospective investigation of the impact of smoking bans on tobacco cessation and relapse. Tob Control, Sep;10(3):267-72.<br>
55.	Massague J (1990) The transforming growth factor (3 family. Annu Rev Cell Biol 6:597-641.<br>
56.Mattila P, Korpela J, Tenkanen T, Pitkanen K (1991) Fidelity of DNA synthesis by the Thermococcus litoralis DNA polymerase--an extremely heat stable enzyme with proofreading activity. Nucleic Acids Res 19:4967-73.<br>
57.	Minshall EM, Leung DYM, Song YL, Cameron 1, Ernst P, Hamid Q (1997) Eosinophil associated TGFpl mRNA expression and airway fibrosis in bronchial asthma. Am J Respir Cell Mol Biol 17:326-333<br>
58.	Mokdad-Gargouri R, Belhadj K, Gargouri A (2001) Translational control of human p53 expression in yeast mediated by 5'-UTR-ORF structural interaction. Nucleic Acids Res 29:1222-7.<br>
59.	Mukawa A, Kamitsuma Y, Tsunekawa S, Tanaka N (1989): Report on a long-term trial of CYBEST Model 2 for prescreening for squamous cell carcinoma of the uterine cervix Anal Cell Pathol. Aug;l(4):225-33.<br>
60.	Nakao A, Miike S, Hatano M, Okumura K, Tokuhisa T, Ra C, et al. Blockade of transforming growth factor beta/Smad signaling in T cells by overexpression of Smad7 enhances antigen-induced airway inflammation and airway reactivity. J Exp Med. 2000;192(2):151-158.<br>
61.	Nagarkatti R, Rao CB, Rishi JP, Chetiwal R, Shandilya V, Vijayan V, et al. (2002) Association of IFNG gene polymorphism with asthma in the Indian population. J Allergy Clin Immunol 2002;110:410-412.<br>
62.	Nagpal K, Sharma S, B-Rao C, Nahid S, Niphadkar PV, Sharma SK, Ghosh B (2005) TGFbetal haplotypes and asthma in Indian populations. J Allergy Clin Immunol 115:527-533.<br>
63.	Nagpal K, Sharma S, B-Rao C, Nahid S, Niphadkar PV, Sharma SK, Ghosh B (2005) TGFbetal haplotypes and asthma in Indian populations. In: "Biotech 2004" Oct, 18-21, 2004, New Delhi.<br>
64.	Nakao A, Miike S, Hatano M, Okumura K, Tokuhisa T, Ra C, Iwamoto I (2000): Blockade of transforming growth factor b/Smad signaling in T cells by overexpression of Smad7 enhances antigen-induced airway inflammation and airway reactivity. J Exp Med 192:151-158<br>
65.	Ober C, Tsalenko A, Parry R, Cox NJ (2000) A second-generation genomewide screen for asthma-susceptibility alleles in a founder population. Am J Hum Genet 67:1154-62.<br>
66.	Oberle 1, Heilig R, Moisan JP, Kloepfer C, Mattei GM, Mattei JF, Boue J, et al (1986) Genetic analysis of the fragile-X mental retardation syndrome with two flanking polymorphic DNA markers. Proc Natl Acad Sci U S A 83:1016-20.<br>
67.	Ohno I, Nitta Y, Yamouchi K, Hoshi H et al (1996) Transforming growth factor pi (TGFpi) gene expression by eosinophils in asthmatic airway inflammation. Am J Respir Cell Mol Biol 15:404-409.<br>
68.	Orita M, Iwahana H, Kanazawa H, Hayashi K, Sekiya T (1989) Detection of polymorphisms of human DNA by gel electrophoresis as single-strand conformation polymorphisms. Proc Natl Acad Sci U S A 86:2766-70.<br>
69.	Paul WE (1997) Interleukin 4: signalling mechanisms and control of T cell differentiation. Ciba Found Symp 204:208-16; discussion 216-9.<br>
70.	Piacquadio DJ, Chen DM, Farber HF, Fowler JF Jr, Glazer SD, Goodman JJ, Hruza LL, Jeffes EW, Ling MR, Phillips TJ, Rallis TM, Scher RK, Taylor CR, Weinstein GD (2004): Photodynamic therapy with aminolevulinic acid topical solution and visible blue light in the treatment of multiple actinic keratoses of the face and scalp: investigator-blinded, phase 3, multicenter trials. Arch Dermatol.; 140(1): 41-6.<br>
71.PulIeyn LJ, Newton R, Adcock IM, Barnes PJ. TGFbetal allele association with asthma severity. Hum Genet. 2001;109(6):623-627.<br>
72.	Redington AE, Madden J, Frew AJ, Djukanovic R, Roche WR, Holgate ST et al. Transforming growth factor-beta 1 in asthma. Measurement in bronchoalveolar lavage fluid. Am J Respir Crit Care Med 1997;156:642-647.<br>
73.	Reich DE, Cargill M, Bolk S, Ireland J, Sabeti PC, Richter DJ, Lavery T, et al (2001) Linkage disequilibrium in the human genome. Nature 411:199-204.<br>
74.	Rothenburg S, Koch-Nolte F, Rich A, Haag F (2001) A polymorphic dinucleotide repeat in the rat nucleolin gene forms Z-DNA and inhibits promoter activity. Proc Natl Acad Sci U S A 98:8985-90.<br>
75.	Sachidanandam R, Weissman D, Schmidt SC, Kakol JM, Stein LD, Marth G, Sherry S, et al (2001) A map of human genome sequence variation containing 1.42 million single nucleotide polymorphisms. Nature 409:928-33.<br>
76.	Sambrook et al., Molecular Cloning, A Laboratory Manual, 2nd Ed., CSHP, New York 1989.<br>
77.	Saiki RK, Bugawan TL, Horn GT, Muliis KB, Erlich HA (1986) Analysis of enzymatically amplified beta-globin and HLA-DQ alpha DNA with allele-specific oligonucleotide probes. Nature 324:163-6.<br>
78.	Sharma S, Nagarkatti R, B-Rao C, Niphadkar PV, Vijayan V, Sharma SK, et al. A_16_C haplotype in FcsRI(3 gene confers higher risk for atopic asthma in Indian population. Clin Genet 2004: 66: 417-425.<br>
79.	Sherman MA, Secor VH, Brown MA (1999) IL-4 preferentially activates a novel STAT6 isoform in mast cells, J Immunol 162:2703-8.<br>
80.	Shrivastava A, K Calame (1994) An analysis of genes regulated by the multifunctional transcriptional regulatory Yin-Yang 1. Nucleic Acids Res 24:5151-5155<br>
81.	Silverman ES, Palmer LJ, Subramaniam V, Hallock A, Mathew S, Vallone J, et al. Transforming growth factor-beta 1 promoter polymorphism C-509T is associated with asthma. Am J Respir Crit Care Med 2004; 169(2):214-219.<br>
82.	Sorensen G, Emmons K, Stoddard AM, Linnan L, Avrunin J (2002): Do social influences contribute to occupational differences in quitting smoking and attitudes toward quitting? Am J Health Promot. Jan-Feb; 16(3): 135-41.<br>
83.	Stephens M, Donnelly P (2003): A comparison of bayesian methods for haplotype reconstruction from population genotype data. Am J Hum Genet. Nov 73:1162-9.<br>
84.	Stephens M, Smith NJ, Donnelly P, A new statistical method for haplotype reconstruction from population data. Am J Hum Genet 2001;68:978-989.<br>
85.	Taylor A, Verhagen J, Akdis CA, Akdis M. T regulatory cells in allergy and health: a question of allergen specificity and balance. Int Arch Allergy Immunol 2004;135(l):73-82.<br>
86.Tay AH, Tan EC, Chew FT, Goh DL, Shek LP, Lee BW (1999) Ethnic differences in genetic susceptibility to atopy and asthma. Asian Pac J Allergy Immunol 17:239-42.<br>
87.	Texereau J, Marullo S, Hubert D, Coste J, Dusser DJ, DairAva-Santucci<br>
J,Dinh-Xuan AT. Nitric oxide synthase 1 as a potential modifier gene of<br>
decline in lung function in patients with cystic fibrosis.Thorax 2004;59:156-<br>
158.<br>
88,	Thomas NS, Wilkinson J, Holgate ST (1997) The candidate region approach<br>
to the genetics of asthma and allergy. Am J Respir Crit Care Med 156:S144-<br>
51.<br>
89.Tsunawaki S, Sporn MB, Ding A, Nathan C (1998) Deactivation of macrophages by transforming growth factor beta. Nature 334:260-262.<br>
90.	WO0208468 (2001). Diagnostic polymorphisms for the TGFbetal promoter,<br>
91.	Xu J, Wiesch DG, MeyeTs DA (1998) Genetics of complex human diseases: genome screening, association studies and fine mapping. Clin Exp Allergy 28 Suppl 5:1-5; discussion 26-8.<br>
92.	Xu J, Meyers DA, Ober C, Blumenthal MN, Mellen B, Barnes KC, King RA, et al (2001) Genomewide screen and identification of gene-gene interactions for asthma-susceptibility loci in three U.S. populations: collaborative study on the genetics of asthma. Am J Hum Genet 68:1437-46,<br>
93.	Zak NB, Shifman S, Shalom A, Darvasi A. Population-based gene discovery in the post-genomic era. Drug Discov Today 2001 ;6:1111-1115.<br>
94.	Zuany-Amorim C, Sawicka E, Manlius C, Le Moine A, Brunet LR, Kemeny DM, et al. Suppression of airway eosinophilia by killed Mycobacterium vaccae-induced allergen-specific regulatory T-cells. Nat Med 2002;8:625-9.<br><br><br><br><br><br><br><br><br>
WE CLAIM:<br>
1.	Pharmacogenetic marker of human Transforming Growth Factor Betal (TGFβ1) gene wherein having atleast one sequence selected from the group consisting of SEQ ID Nos. 1, 2 and 3 is useful for predicting predisposition of an individual to an immunological disorder more particularly asthma.<br>
2.	Pharmacogenetic marker of human Transforming Growth Factor Betal (TGFβ1) gene as claimed in claim 1, wherein SEQ ID No.l is associated with CT/CA, SEQ ID No.2 is associated with SI and SEQ ID No. 3 is associated with S2 locus of the TGFβ1 gene.<br>
3.	A method of preparing novel pharmacogenetic markers for detecting and predicting predisposition to immunological disorders of TGFβ1 gene in a subject said method comprises of :-<br>
i.   designing primers having SEQ ID no.4 and 5, ii.   synthesizing primers having SEQ ID nos.4,5,6,7,8 and 9, iii.   amplifying the genomic DNA using primers of SEQ ID nos.<br>
4,5,6,7,8,and 9 wherein primers of SEQ ID nos. 4,6 and 8 are<br>
forward primers and primers of SEQ ID nos. 5,7 and 9 are reverse<br>
primers; iv.   isolating and identifying the DNA stretch of SEQ ID no. 1, using<br>
novel primer combination of SEQ ID nos. 4 and 5; V.   isolating and identifying the DNA stretch of SEQ ID no.2 using<br>
primer combinations of SEQ ID nos. 6 and 7; vi.   isolating and identifying the DNA stretch of SEQ ID no.3 using<br>
primer combinations of SEQ ID nos. 8 and 9; vii.   sequencing the isolated and identified SEQ ID nos.  1,2 and 3<br>
obtained in steps (lv,v and vi) and.<br><br>
viii. validating and identifying the specific TGFβ1 gene variants computationally by comparison with the known wild type TGFβ1 gene sequences.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAzNy1ERUwtMjAwNC1BYnN0cmFjdC0oMDktMDMtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1037-DEL-2004-Abstract-(09-03-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAzNy1kZWwtMjAwNC1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1037-del-2004-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAzNy1ERUwtMjAwNC1DbGFpbXMtKDA5LTAzLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">1037-DEL-2004-Claims-(09-03-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAzNy1ERUwtMjAwNC1DbGFpbXMtKDMwLTAzLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">1037-DEL-2004-Claims-(30-03-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAzNy1kZWwtMjAwNC1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">1037-del-2004-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAzNy1kZWwtMjAwNC1jb21wbGV0ZSBzcGVjaWZpY2F0aW9uIChncmFudGVkKS5wZGY=" target="_blank" style="word-wrap:break-word;">1037-del-2004-complete specification (granted).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAzNy1ERUwtMjAwNC1Db3JyZXNwb25kZW5jZS1PdGhlcnMtKDA5LTAzLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">1037-DEL-2004-Correspondence-Others-(09-03-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAzNy1ERUwtMjAwNC1Db3JyZXNwb25kZW5jZS1PdGhlcnMtKDMwLTAzLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">1037-DEL-2004-Correspondence-Others-(30-03-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAzNy1kZWwtMjAwNC1jb3JyZXNwb25kZW5jZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">1037-del-2004-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAzNy1kZWwtMjAwNC1jb3JyZXNwb25kZW5jZS1wby5wZGY=" target="_blank" style="word-wrap:break-word;">1037-del-2004-correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAzNy1kZWwtMjAwNC1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">1037-del-2004-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAzNy1kZWwtMjAwNC1kZXNjcmlwdGlvbiAocHJvdmlzaW9uYWwpLnBkZg==" target="_blank" style="word-wrap:break-word;">1037-del-2004-description (provisional).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAzNy1kZWwtMjAwNC1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">1037-del-2004-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAzNy1ERUwtMjAwNC1Gb3JtLTEtKDA5LTAzLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">1037-DEL-2004-Form-1-(09-03-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAzNy1kZWwtMjAwNC1mb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">1037-del-2004-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAzNy1kZWwtMjAwNC1mb3JtLTE4LnBkZg==" target="_blank" style="word-wrap:break-word;">1037-del-2004-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAzNy1ERUwtMjAwNC1Gb3JtLTItKDA5LTAzLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">1037-DEL-2004-Form-2-(09-03-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAzNy1kZWwtMjAwNC1mb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">1037-del-2004-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAzNy1kZWwtMjAwNC1mb3JtLTI2LnBkZg==" target="_blank" style="word-wrap:break-word;">1037-del-2004-form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAzNy1kZWwtMjAwNC1mb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">1037-del-2004-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAzNy1kZWwtMjAwNC1mb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">1037-del-2004-form-5.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="233754-a-media-system-for-organizing-selecting-and-launching-media-items.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="233756-a-communications-network-for-allocating-guard-bands-and-a-method-thereof.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>233755</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1037/DEL/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>20/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>15-May-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>06-Apr-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>04-Jun-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>COUNCIL OF SCIENTIFIC &amp; INDUSTRIAL RESEARCH</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>RAFI MARG, NEW DELHI-110001, INDIA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>BALARAM GHOSH</td>
											<td>INSTITUTE OF GENOMICS AND INTEGRATIVE BIOLOGY, MALL ROAD, NEW DELHI-110007, INDIA.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>KAMALPREET NAGPAL</td>
											<td>INSTITUTE OF GENOMICS AND INTEGRATIVE BIOLOGY, MALL ROAD, NEW DELHI-110007, INDIA.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>SHILPY SHARMAQ</td>
											<td>INSTITUTE OF GENOMICS AND INTEGRATIVE BIOLOGY, MALL ROAD, NEW DELHI-110007, INDIA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/233755-pharmacogenetic-marker-of-human-transforming-growth-factor-beta1-tgfss1-for-predicting-predispodition-to-immunological-disorders by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:52:51 GMT -->
</html>
