<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/233827-a-process-for-recovering-polypeptide-an-immunoglobin-variable-domain-that-unfolds-reversibly-from-a-repertoire-of-polypeptides by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:55:14 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 233827:&quot;A PROCESS FOR RECOVERING POLYPEPTIDE, AN IMMUNOGLOBIN VARIABLE DOMAIN THAT UNFOLDS REVERSIBLY FROM A REPERTOIRE OF POLYPEPTIDES&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A PROCESS FOR RECOVERING POLYPEPTIDE, AN IMMUNOGLOBIN VARIABLE DOMAIN THAT UNFOLDS REVERSIBLY FROM A REPERTOIRE OF POLYPEPTIDES&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a process for recovering at least one polypeptide that unfolds reversibly from a repertoire of polypeptides, wherein the polypeptides in the repertoire that unfold reversibly have a common selectable characteristic that distinguishes folded polypeptides from unfolded or misfolded polypeptides, and wherein said common selectable characteristic is selected from the group consisting of binding to a generic ligand, binding to a target ligand, a catalytic activity, resistance to proteolysis and an epitope presented on the displayed polypeptides when folded, but absent from misfolded or unfolded polypeptides, the process comprising unfolding substantially all polypeptides in a polypeptide display system that comprises said repertoire of polypeptides by increasing the temperature of said polypeptide display system to an unfolding temperature of at least about 80°C, whereby a portion thereof aggregate, and refolding at least a portion of the remaining unfolded polypeptides by reducing the temperature of said polypeptide display system to a refolding temperature that is at least about 10°C lower than said unfolding temperature; and recovering at least one polypeptide that unfolds reversibly and has said selectable characteristic from the refolded portion. Further, the present invention relates to the isolated polypeptides produced by the process of present invention and the composition comprising the said polypeptides.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>A PROCESS FOR RECOVERING POLYPEPTIDES THAT UNFOLD REVERSIBLY FROM A POLYPEPTIDE  REPERTOIRE<br>
RELATED APPLICATIONS<br>
This application claims the benefit of U.S. Provisional Patent Application No. 60/554,021, filed on March 17,2004, and of U.S. Provisional Patent Application No. 60/470,340, filed on May 14,2003. The entire teachings of the above applications are incorporated herein by reference.<br>
BACKGROUND OF THE INVENTION<br>
Polypeptides have become increasingly important agents in a variety of applications, including use as medical therapeutic and diagnostic agents and in industrial applications. One factor that has hinder further application of polypeptides is their physical and chemical properties. For example, polypeptides generally must retain proper folding to be active. However, polypeptides tend to unfold or denature under storage conditions or conditions where they could find utility (e.g., when exposed to heat, organic solvents). In addition, many polypeptides are produced only with relatively low yield using biological production systems. Accordingly, they can be prohibitively costly to produce.<br>
A key factor that limits further application of polypeptides is the tendency of unfolded or denatured polypeptides to aggregate irreversibly. Aggregation is influenced by polypeptide concentration and is thought to arise in many cases from partially folded or unfolded intermediates. Factors and conditions that favor partially folded intermediates, such as elevated temperature and high polypeptide concentration, promote irreversible aggregation. (Fink, A.L., Folding &amp; Design 3:R1-R23 (1998).) For example, storing purified polypeptides in concentrated form,<br>
sucfa as a lyophilized preparation, frequently results in irreversible aggregation of at least a portion of the polypeptides. Also, production of a polypeptide by expression in biological systems, such as E. coli, often results in the formation of inclusion bodies which contain aggregated polypeptides. Recovering active polypeptides from inclusion bodies can be very difficult and require adding additional steps, such as a refolding step, to a biological production system.<br>
One approach that has been attempted for preparing polypeptides with improved properties is the selection of polypeptide variants that have improved stability or solubility. (See, e.g., Jung, S. et al., J, Mol. Biol 294:163-180 (1999); Davies, J and Riechmannn, L., Prot. Eng. 9:531-537 (1996); Waldo, G.S., Curr. Opin. Chem, Biol, 7:33-38 (2003).) However, selection for improved stability or solubility does not address the aggregation problem because stability (e,g., thermal stability, theonodynamic stability) and solubility are characteristics of the properly folded polypeptide while aggregation arises from the partially folded or partially denatured state. In addition, there is no recognized correlation between polypeptide stability and aggregation. (Fink, A.L., Folding &amp; Design 3:R1-R23 (1998).)<br>
A need exists for polypeptides with unproved properties that can be produced with high yields using biological production systems.<br>
SUMMARY OF THE INVENTION<br>
The invention relates to polypeptides that unfold reversibly, to repertoires containing polypeptides that unfold reversibly and to libraries that contain polypeptides that unfold reversibly or nucleic acids that encode polypeptides that unfold reversibly. The invention further relates to processes for producing a library enriched in polypeptides that unfold reversibly or nucleic acids encoding polypeptides that unfold reversibly, processes for selecting and/or isolating polypeptides that unfold leversibly, and to methods for producing a polypeptide that unfolds reversibly.<br>
In one aspect, the invention is a process for selecting, isolating and/or recovering a polypeptide that unfolds reversibly from a library or a repertoire of polypeptides (e.g., a polypeptide display system). In one embodiment, the method comprises unfolding a collection of polypeptides (e.g., the polypeptides in a library, a repertoire or a polypeptide display system), refolding at least a portion of the<br>
unfolded polypeptides, and selecting, isolating and/or recovering a refolded polypeptide. In another embodiment, the method comprises providing a collection of unfolded polypeptides (e.g., the polypeptides in a library, a repertoire or a polypeptide display system), refolding at least a portion of the unfolded polypeptides, and selecting, isolating and/or recovering a refolded polypeptide. In another embodiment, the method comprises providing a polypeptide display system comprising a repertoire, heating the repertoire to a temperature (Ts) at which at least a portion of the displayed polypeptides unfold and cooling the repertoire to a temperature" (To) that is lower than Ts to produce a cooled repertoire. The cooled repertoire comprises at least a portion of polypeptides that have unfolded and refolded and a portion of polypeptides that have aggregated. The method further comprises recovering at a temperature (Tr) at least one polypeptide that binds a ligand and unfolds reversibly. Preferably the ligand binds folded polypeptide and does not bind aggregated polypeptides, the recovered polypeptide has a melting temperature (Tm), and Ts&gt;Trn&gt;Tc, and Ts&gt;Tm&gt;Tr.<br>
In other aspects, the invention relates to repertoires of polypeptides that unfold reversibly, to libraries of nucleic acids that encode polypeptides mat unfold reversibly, and to methods for producing such libraries and repertoires.<br>
In one aspect, the invention is an isolated polypeptide that unfolds reversibly. hi some embodiments, the polypeptide that unfolds reversibly is a variant of a parental polypeptide that differs from the parental polypeptide in amino acid sequence (e.g., by one or more amino acid replacements, additions and/or deletions), but qualitatively retains function of the parental polypeptide.<br>
The invention also relates to a process for producing an antibody variable domain library enriched in variable domains that unfold reversibly. In one embodiment, the process comprises (1) providing a phage display system comprising a plurality of displayed antibody variable regions, wherein at least a portion of the displayed variable regions have been unfolded and refolded, (2) selecting phage displaying variable regions that have unfolded, refolded and regained binding function from said phage display system, (3) obtaining nucleic acids encoding CDR1 and /or CDR2 of the variable regions displayed on the recovered phage, and (4) assembling a library of nucleic acids encoding antibody<br>
variable domains, wherein said nucleic acids obtained in (3) are operably linked to one or more other nucleic acids to produce a library of constructs that encode antibody variable domains in which CDR1 and/or CDR2 are encoded by the nucleic acid obtained in (3). In particular embodiments, substantially all of the displayed variable regions in (1) have been unfolded by heating to about 80°C and refolded by cooling. In other particular embodiments, library of nucleic acids assembled in (4) encodes an antibody variable domain in which CDR3 is randomized or is not derived from antibody variable regions that has been selected for the ability to unfold reversibly.<br>
The polypeptides that unfold reversibly described herein can be produced as soluble proteins in the supernatant of E. coli or yeast cultures with high yield.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Fig. 1 is an illustration of a nucleic acid encoding human immunoglobulin heavy chain variable region DP47dummy (also referred to herein as DP47d), comprising germline VH gene segment DP47 and germline VJ gene segment JH4b (SEQ ID NO:1, coding strand; SEQ ID N0:2 non-coding strand). Fig. 1 also presents the amino acid sequence of the encoded VH domain (SEQ ID NO:3). The amino acids are numbered according to the system of Rabat. (Kabat, E.A. et at., Sequences of Proteins oflmmunological Merest, Fifth Edition, U.S. Department of Health and Human Services, U.S. Government Printing Office (1991).)<br>
Fig. 2 is an illustration of a nucleic acid encoding human immunoglobulin light chain (K) variable region DPK9 dummy (also referred to herein as DPK9d), comprising germline V K gene segment DPK9 and germline JK gene segment JKU. (SEQ ID NO;4, coding strand; SEQ ID NO:5 non-coding strand). Fig. 2 also presents the amino acid sequence of the encoded V K domain (SEQ ID N0:6). The amino acids are numbered according to the system of Kabat<br>
Figs. 3A-3F are histograms that illustrate the relative amount of protein A binding activity retained by phage clones displaying VH domains based on a DP47d scaffold after heat-induced unfolding and refolding. A sample of each phage clone was heated to cause the displayed VH domains to unfold and then cooled to cause refolding, and a sample -was left unheated. Binding of the heat-treated and the untreated phage samples to protein A was assessed by ELISA. The binding activity<br>
of the heat-treated clones expressed as a percentage of the binding activity of the untreated clones is illustrated. The columns marked "dp47" refer to DP47d.<br>
Figs. 4A-4C are panels of a table illustrating the amino acid sequences of the complementarity determining regions (CDR1, CDR2, CDR3) of several of the VH domains displayed on the phage that retained greater than &gt;60% relative binding to protein A in Figs 3A-3F. The sequences are identified in Figs 4A-4C by clone number. These clones are also referred to herein using a "pA-" prefix. For example, Clone 13 is also referred to herein as "pA-C13." The first group of sequences presented in Fig. 4A and 4B are from clones that had a high degree of refolding as assessed by the results of protein A binding (group 1). The next group of sequences in Fig. 4C are from clones with good refolding. These clones also contain mutations outside the CDRs (group 2). The final group of sequences in Fig. 4C are from clones with lower refolding.<br>
Fig. 5 is a graph illustrating the relationship between the capacity of a displayed VH domain to undergo reversible heat unfolding and the hydrophobicity of the amino acid sequence from position 22 to position 36 of the VH domain. The Sweet/Eisenberg hydrophobicity score (SE-15 value) for the sequence for position 22 to position 36 of several displayed VH was determined using a window of 15 amino acids. The SE-15 value was plotted against the relative protein A binding activity (ELISA) for each clone after undergoing heat-induced unfolding and refolding. The graph illustrates that the ability of a displayed VH to undergo heat-induced unfolding and refolding correlates with an SE-15 value of 0 or less for the amino acid sequence from position 22 to position 36. Amino acid positions are defined according to Kabat<br>
Fig. 6 is a histogram illustrating the Sweet/Eisenberg hydrophobicity score (Sweet/Eisenberg value) for the sequence for position 22 to position 36 of several displayed VH. VH domains DP47d and BSA1 have Sweet/Eisenberg values that are greater than 0, and do not undergo reversible heat-induced unfolding. VH domains KEIA, pA-C13, pA-C36, pA-C47, pA-C59, pA-C76 and pA-C85 have Sweet/Eisenberg values that are less than 0, and the VH domains of each of these clones undergo reversible heat-induced unfolding, Amino acid positions are defined according to Kabat<br>
Fig. 7 is an illustration of the characteristic shape of an unfolding curve and a refolding curve. The curves are plotted using a measure of the concentration of properly foldedpolypeptide (e.g,, ellipticity or fluorescence) as the abscissa, and the unfolding agent (e.g., heat (temperature)) as the ordinate. The unfolding and refolding curves include a region in which the polypeptides are folded, an unfolding/refolding transition in which polypeptides are unfolded to various degrees, and a portion in which the polypeptides are unfolded. The y-axis intercept of the refolding curve is the relative amount of refolded protein recovered. In the illustrated plot, TM is the melting temperature of the polypeptide, and TM-10 and TM -HO are the melting temperature of the polypeptide minus 10 degrees and plus 10 degrees, respectively. The illustrated refolding curve indicates that greater than 75% of the polypeptides refolded.<br>
Fig. 8 is a graph showing heat-induced unfolding of dAb HEL4. dAb HEL4 was unfolded by heating and ellipticity assessed during heating (filled circles). The unfolded dAb was then refolded by decreasing the temperature. The refolded dAb was then again unfolded by heating and ellipticity assessed during heating (open diamonds). The graph shows that the unfolding curves of both heat-induced unfoldings are superimposable, demonstrating that dAb HEL4 undergoes reversible heat-induced unfolding. The inset shows the far-UV CD spectra for folded dAb HEL4 at 25°C (Fold) and for unfolded dAb HEL4 at 80°C (Unfold).<br>
Fig. 9 is a graph showing that a dAb comprising DP47 variant in which Trp47 was replace with Arg (DP47-W47R) does not unfold reversibly upon heating. dAb DP47-W47R was unfolded by heating and ellipticity assessed during heating (filled circles). The unfolded dAb was then refolded by decreasing the temperature. The refolded dAb was then again unfolded by heating and ellipticity assessed during heating (open squares). The unfolding curves are not superimposable and have a shape characteristic of denatured polypeptides.<br>
Fig. 10 is a graph showing that a dAb comprising DP47 variant in which Ser35 was replaced with Gly (DP47-S47G) unfolds reversibly upon heating to a limited extent. dAb DP47- S47G was unfolded by heating and ellipticity assessed during heating (filled circles). The unfolded dAb was then refolded by decreasing the temperature. The refolded dAb was then again unfolded by heating and<br>
ellipticity assessed during heating (open diamonds). The unfolding curves are not superimposable and reveal that a proportion of the ellipticity (and hence a portion of the original secondary structure) was recovered upon refolding, and that a melting transition is observed upon re-heating the sample.<br>
Fig. 11 is a graph illustrating the relationship between thennodynamic stability of polypeptides (AG folded-&gt; unfolded) and reversible unfolding of polypeptides displayed onphage. The graph shows thatnon-refoldable polypeptides BSA1 and DP47d have high thennodynamic stability, and that refoldable polypeptide HEM and several refoldable mutants of DP47 have lower thennodynamic stability. Thermodynamic stability of refoldable polypeptides was determined from ellipticity data obtained during heat-induced unfolding. Thermodynamic stability of polypeptides that were not refoldable was determined by monitoring fluorescence during urea-induced unfolding.<br>
Fig. 12 is a graph illustrating the relationship between thermodynamic stability of polypeptides (AG folded-* unfolded) and protein expression level w.E. coli supernatant The graph shows that non-refoldable polypeptides BSA1 and DP47 have high thermodynamic stability but are expressed at relatively low levels, and that refoldable polypeptide HEL4 and several refoldable mutants of DP47dhave lower thermodynamic stability but ate expressed at relatively high levels. Thermodynamic stability of refoldable polypeptides was determined from ellipticity data obtained during heat-induced unfolding. Thermodynamic stability of polypeptides that were not refoldable was determined by monitoring fluorescence during urea-induced unfolding. Protein expression is the amount of polypeptide purified using protein A sepharose from a 1L culture of E. coli, normalized to cell density (OD600) of the culture.<br>
Fig. 13 is a graph illustrating the relationship between protein expression levels in JS, coli supernatant and reversible unfolding of polypeptides displayed on phage. The graph shows that non-refoldable polypeptides BSA1 and DP47d are expressed at relatively low levels, and that refoldable polypeptide HEM and several refoldable mutants of DP47d have are expressed at relatively high levels. The graph reveals a direct correlation between reversible unfolding of polypeptides displayed on phage and the quantity of polypeptide in the supernatant of cultures of E. coli that<br>
express the polypeptide. Protein expression is the amount of polypeptide purified using protein A sepharose from a 1L culture of E. coli, normalized to cell density (OD600) of the culture.<br>
Fig. 14 is a graph showing heat-induced unfolding of a single chain Fv (scFv) containing a reversibly unfoldable VK (DPK9-I75N) and a reversibly unfoldable VH (DP47-F27D). The scFv was unfolded by heating and ellipticity assessed during heating (filled circles). The unfolded scFv was then refolded by decreasing the temperature. The refolded scFv was then again unfolded by heating and ellipticity assessed during heating (open diamonds). The graph shows that the unfolding curves of both heat-induced unfoldings are superimposable, demonstrating that scFv undergoes reversible heat-induced unfolding. Other scFvs containing germline VH and germline VK, germline VH and unfoldable VK (DPK9-I75N), or reversibly unfoldable VH (DP47-F27D) and germline VK aggregated under the conditions used. The inset shows the far-UV CD spectra for folded scFv before heat induced unfolding (dark trace) and for folded scFV following heat induced unfolding and refolding (lighter trace). The spectra are superimposable indicating that the scFv regained all secondary structure following refolding.<br>
Figs. 15 A and 15B are histograms showing the effect of heat-treatment of phage (80°C for 10 min then cooled to 4°C for 10 min) on binding of displayed dAbs to protein A or antibody. InFIG. 15A, phage displaying (5xl011 TU/ml) either DP47d or HEL4 d Ab multivalentiy were assayed for binding to either anti-c-myc antibody (9E10) or protein A by ELISA. 9E10 recognises the c-myc tag pepto'de tag appended to the dAb as a linear determinant. Using a dilution series of phage, the retained binding was calculated from the phage tilers required for an ELISA signal of 0.5 OD650nm-OD450nm (titer untreated/titer heat treated). In FIG. 15B, phage displaying DP47d were assayed for binding to protein A by ELISA. Phage concentration was high (5x1011 TU/ml) or low (IxlO9 TU/ml) and DP47d was displayed in multivalent or monovalent states.<br>
Fig. 15C is a histogram showing the effect of heat-treatment of phage (80°C for 10 min then cooled to 4°C for 10 min) on infectivity. Phage displaying either DP47d or HEL4 multivalently were heated and then cooled, after 10 min a phage<br>
sample was treated with 0,9 mg/ml, trypsin at 22°C for 10 min, and then used to infect E coli TGI cells.<br>
Figs. 16A - 16C are copies of transmission electron micrographs of negatively stained phage tips before and after heat treatment (10 min at 80°C). FIGS 16A and 16B show that heated DP47d phages form aggregates. However, as shown in FIG. 16C, heated HEL4 phages which display the HEL4 VH domain, which unfolds reversibly, did not form aggregates.<br>
Pig. 16D is a copy of an image of a Western blot in which 1010 transducing units (TU) of phage per lane were separated and detected using an anti-pin mouse monoclonal antibody. The phage loaded in lanes 1 to 6 were; fd, HEL4 (multivalent), DP47d (multivalent), M13, HEL4 (monovalent), and DP47d (monovalent), respectively.<br>
Fig. 17A is a representative analytical gel-filtration cbromatagram of<br>
selected human VK! dAhs. The chromatograms for Cl 3 (	), C36 (	),<br>
C47 (	), C59 (	), C76 (	) and C85 (	) dAbs (10 fiM in PBS) were<br>
obtained using a SUPERDEX-75 column (Amersham Biosciences), apparent Mr for C13,C36,C47,C59,C76andC85were22kDa, 17kDa, 19kDa, 10kDa,20kDaJ and 15 kDa, respectively.<br>
Fig. 17B is a graph showing heat induced denaturation curves of the C36 dAb (5 µM in PBS) recorded by CD at 235 tun: Y, mean residual ellipticity upon first heating; v, mean residual ellipticity upon second heating. Inset: CD spectra of dAb HEL4 (5 µM in PBS) m tin fat-UV region at different temperature: A. . 25°C before unfolding;«, S5°C (unfolded polypeptide); 0,25°C after sample cooling.<br>
Fig. 18A is a graph showing (hat TAR2-10-27 and variants TAR2-10-27 F27D and TAR2-10-27 Y23D bind human Tumor Necrosis Factor Receptor 1 (TNFR1) and inhibit binding of THF to the receptor in a receptor binding ELISA. TNFR1 was immobilized on the plate and TNF was mixed with TAR240-27, TAR2-10-27 F27D or TAR2-10-27 Y23D and then added to the wells. The amount of TNF that bound the immobilized receptor was quantified using an anti-TNF antibody. TAR2-10-27, TAR2-10-27 F27D andTAR2-10-27 Y23D eachbound human TNFR.1 and inhibited the binding of TNF to the receptor.<br>
Fig. 18B is a graph showing that TAR2-10-27 and variants TAR2-10-27 F27D and TAR2-10-27 Y23D bound human Tumor Necrosis Factor Receptor 1 (TNFR1) expressed on HeLa cells and inhibited TNF-induced production of IL-8 in an in vitro assay.  HeLa cells were plated in microtitre plates and incubated overnight with TAR2-10-27, TAR2-10-27 F27D or TAR2-10-27 Y23D and 300 pg/ml TNF. Post incubation, the supernatant was aspirated off the cells and the amount of IL-8 in the supernatant was measured using a sandwich ELISA. TAR2-10-27, TAR2-10-27 F27D and TAR2-10-27 Y23D each bound human TNFR1 and inhibited TNF-induced IL-8 production.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The invention relates to polypeptides that unfold reversibly and to methods for selecting and/or designing such polypeptides. Polypeptides that unfold reversibly provide several advantages. Notably, such polypeptides are resistant to aggregation or do not aggregate. Due to this resistance to aggregation, polypeptides that unfold reversibly can readily be produced in high yield as soluble proteins by expression using a suitable biological production system, such as E. coli. In addition, polypeptides that unfold reversibly can be formulated and/or stored at higher concentrations than conventional polypeptides, and with less aggregation and loss of activity.<br>
As described herein, polypeptides that unfold reversibly can be selected from a polypeptide display system hi which the polypeptides have been unfolded (e.g., by heating) and refolding (e.g., by cooling). The selection and design processes described herein yields polypeptides that unfold reversibly and are resistant to aggregation. These selection and design processes are distinct from methods that select polypeptides based on enhanced stability, such as polypeptides that remain folded at elevated temperature. (See, e,g., Jung, S. et al, J. Mot Biol. 2P4:163-180 (1999).)<br>
As used herein, "polypeptide display system" refers to a system in which a collection of polypeptides are accessible for selection based upon a desired characteristic, such as a physical, chemical or functional characteristic. The polypeptide display system can be a suitable repertoire of polypeptides (e.g., in a solution, immobilized on a suitable support). The polypeptide display system can<br>
aiso be a biochemical system that employs a cellular expression system (e.g., expression of a library of nucleic acids in, e.g., transformed, infected, transfected or transduced cells and display of the encoded polypeptides on the surface of the cells) or an acellular expression system (e.g., emulsion compartrnentalization and display). Preferred polypeptide display systems link the coding function of a nucleic acid and physical, chemical and/or functional characteristics of a polypeptide encoded by the nucleic acid. When such a polypeptide display system is employed, polypeptides that have a desired physical, chemical and/or functional characteristic can be selected and a nucleic acid encoding the selected polypeptide can be readily isolated or recovered. A number of polypeptide display systems that link the coding function of a nucleic acid and physical, chemical and/or functional characteristics of a polypeptide are known in the art, for example, bacteriophage display (phage display), ribosome display, emulsion compartmentalization and display, yeast display, puromycin display, bacterial display, polypeptide display on plasmid and covalent display and the like. (See, e.g., EP 0436597 (Dyax), U.S. Patent No. 6,172,197 (McCafferty et al), U.S. Patent No. 6,489,103 (Griffiths et al.).) The polypeptide display system can comprise a plurality of replicable genetic display packages, as described by McCafferty etal (e.g., WO 92/01047; U.S. Patent No. 6,172,197).  A replicable genetic display package (RGDP) is a biological particle which has genetic information providing the particle wiui the ability to replicate. An RGDP can display on its surface at least part of a polypeptide. The polypeptide can be encoded by genetic information native to the RGDP and/or artificially placed into the RGDP or an ancestor of it. The RGDP can be a virus e.g., a bacteriophage, such as fd or Ml 3. For example, the RGDP.can be a bacteriophage which displays an antibody variable domain (e.g., VH, Vi.) at its surface. This type of RGDP can be referred to as a phage antibody (pAb),<br>
As used herein, the terms "reversibly unfoldable" and "unfolds reversibly" refer to polypeptides that can be unfolded (e.g., by heat) and refolded in the method of the invention. A reversibly unfoldable polypeptide (a polypeptide that unfolds reversibly) loses function when unfolded but regains function upon refolding. Such polypeptides are distinguished from polypeptides that aggregate when unfolded or that improperly refold (misfolded polypeptides), i.e., do not regain function.<br>
Preferably, a polypeptide that unfolds reversibly can unfold reversibly when displayed in a polypeptide display system, for example, when display on bacteriophage. Particularly preferred polypeptide that unfold reversibly can unfold reversibly when displayed in a polypeptide display system and as a soluble polypeptide (e.g., an autonomous soluble polypeptide).<br>
As used herein, "repertoire of polypeptides" refers to a collection of polypeptides that are characterized by amino acid sequence diversity. The individual members of a repertoire can have common features, such as common structural features (e.g., a common core structure) and/or common functional features (e.g., capacity to bind a common ligand (e.g., a generic ligand or a target ligand)).<br>
As used herein, "library" refers to a collection of heterogeneous nucleic acids that can be expressed and preferably are reph'cable. For example, a library can contain a collection of heterogeneous nucleic acids that are incorporated into a suitable vector, such as an expression plasmid, a phagemid and the like. Expression of such a library can produce a repertoire of polypeptides. "Library" also refers to a collection of heterogeneous polypeptides that are displayed in a polypeptide display system that links coding function of a nucleic acid and physical, chemical and/or functional characteristics of a polypeptide encoded by the nucleic acid, and can be selected or screened to provide an individual polypeptide (and nucleic acid encoding same) or a population of polypeptides (and nucleic acids encoding same) that have a desired physical, chemical and/or functional characteristic. A collection of phage that displays a collection of heterogeneous polypeptides is one example of such a library. A library that is a collection of heterogeneous polypeptides encompasses a repertoire of polypeptides.<br>
As used herein, "functional" describes a polypeptide that is properly folded so as to have a specific desired activity, such as ligand-binding activity (e.g., binding generic h'gand, binding target ligand), or the biological activity of native or naturally produced protein. For example, the term "functional polypeptide" includes an antibody or antigen-binding fragment thereof lhat binds a target antigen through its antigen-binding site, and an enzyme that binds its substrate(s).<br>
As used herein, "generic ligand" refers to a Hgand that binds a substantial portion (e.g., substantially all) of the functional members of a given repertoire. A generic ligand (e.g., &amp; common generic ligand) can bind many members of a given repertoire even though the members may not have binding specificity for a common target ligand. In general, the presence of a functional generic ligand-binding site on a polypeptide (as indicated by the ability to bind a generic ligand) indicates that the polypeptide is correctly folded and functional. Accordingly, polypeptides that are correctly folded can be selected or recovered from a repertoire of polypeptides by binding to a generic ligand. Suitable examples of generic ligands include superantigens, antibodies that bind an epitope expressed on a substantial portion of functional members of a repertoire, and fhe like.<br>
As used herein, "target ligand" refers to a ligand which is specifically or selectively bound by a polypeptide. For example, a target ligand can be a ligand for which a specific binding polypeptide or polypeptides in a repertoire are identified. For example, when a polypeptide is an antibody or antigen-binding fragment thereof, the target ligand can be any desired antigen or epitope, and when a polypeptide is an enzyme, the target ligand can be any desired substrate. Binding to the target antigen is dependent upon the polypeptide being functional, and upon the specificity of the target antigen-binding site of the polypeptide.<br>
As used herein, "antibody polypeptide" is a polypeptide that is an antibody, a portion of an antibody, or a fusion protein that contains a portion of an antibody (e.g., an antigen-binding portion). Thus, "antibody polypeptides" include, for example, an antibody (e.g., an IgG), an antibody heavy chain, an antibody light chain, horaodimers and heterodimers of heavy chains and/or light chains, and antigen-binding fragments or portions of an antibody, such as a Fv fragment (e.g., single chain Fv (scFv), a disulfide bonded Fv), a Fab fragment, a Fab' fragment, a F(ab')2 fragment, a single variable domain (VH, VL), a dAb and the like.<br>
As used herein, "antibody format" refers to any suitable polypeptide structure in which an antibody variable domain can be incorporated so as to confer binding specificity for antigen on the structure. A variety of suitable antibody formats are known in the art, such as, chimeric antibodies, humanized antibodies, human antibodies, single chain antibodies, bispecific antibodies, antibody heavy<br>
chains, antibody light chains, homodimers and heterodimers of antibody heavy chains and/or light chains, antigen-binding fragments of any of the foregoing (e.g., a Fv fragment (e.g., single chain Fv (scFv), a disulfide bonded Fv), a Fab fragment, a Fab' fragment, a F(ab')2 fragment), a single variable domain (e.g,, VH, VL), a dAb, and modified versions of any of the foregoing (e.g., modified by the covalent attachment of polyethylene glycol or other suitable polymer).<br>
"Superantigen" is a term of art that refers to generic ligands that interact with members of the immunoglobulin superfamily at a site that is distinct from the conventional ligand-binding sites of these proteins. Staphylococcal enterotoxins axe examples of superantigens which interact with T-cell receptors. Superantigens that bind antibodies include Protein Q, which binds the IgG constant region (Bjorck and Kronvall, /. Immwol., 133:969 (1984)); Protein A which binds the IgG constant region and VH domains (Forsgren and Sjoquist, J. Immunol, 97:822 (1966)); and Protein L which binds Vt domains (Bjorck, J. Immunol., 140:1194 (1988)).<br>
As used herein, "unfolding agent" refers to an agent (e.g., compound) or to energy that can cause polypeptide unfolding. When an unfolding agent is a compound, the compound can be added to a polypeptide display system in an amount sufficient to cause a desired degree of polypeptide unfolding. Examples of suitable compounds include, chaotropic agents (e.g., guanidine hydrochloride, urea), acids (e.g., hydrochloric acid, acetic acid), bases (e.g., sodium hydroxide, potassium hydroxide), and organic solvents (e.g., an alcohol (e.g., methanol, eihanol), a ketone (e.g., methyl ethyl ketone), an aldehyde (e.g., formaldehyde, dimethylformaldehyde), tetrahydrofuran, dioxane, toluene and the like). The unfolding agent can be energy, such as heat and/or pressure. When the unfolding agent is heat, the polypeptide system is exposed to a sufficient amount of energy (e.gt) thermal, electromagnetic) to impart enough heat to the polypeptide display system to raise the temperature of the system to a temperature that is sufficient to cause a desired degree of polypeptide unfolding. Preferred unfolding agents (or combinations of unfolding agents) do not substantially inhibit aggregation of unfolded polypeptides that do not unfold reversibly.<br>
As used herein, "folding gatekeeper" refers to an amino acid residue that, by virtue of its biophysical characteristics and by its position in the amino acid<br>
sequence of a protein, prevents the irreversible formation of aggregates upon protein unfolding. A folding gatekeeper residue blocks off-pathway aggregation, thereby ensuring that the protein can undergo reversible unfolding. A folding gate keeper generally reduces the SB score (hydrophobicity score) of the amino acid sequence of the region in which it is found.<br>
As used herein "secretable" or "secreted" means that when a polypeptide is produced by expression in E, coll, it is produced and exported to the periplasmic space or to the medium.<br>
ASSESSING UNFOLDING AND REFOLDING<br>
Polypeptide unfolding and refolding can be assessed, for example, by directly or indirectly detecting polypeptide structure using any suitable method. For example, polypeptide structure can be detected by circular dichroism (CD) (e.g., far-UV CD, near-UV CD), fluorescence (e.g., fluorescence of tryptophan side chains), susceptibility to proteolysis, nuclear magnetic resonance (NMR), or by detecting or    measuring a polypeptide function mat is dependent upon proper folding. In one    example, polypeptide unfolding is assessed using a functional assay in which loss of    binding function (e.gi, binding a generic and/or target ligand, binding a substrate)    indicates that the polypeptide is unfolded.<br>
The extent of unfolding and refolding of a soluble polypeptide can determined by using an unfolding or denaturation curve. An unfolding curve can be produced by plotting the unfolding agent (e.g., temperature, concentration of chaotropic agent, concentration of organic solvent) as the ordinate and the relative concentration of folded polypeptide as the abscissa. The relative concentration of folded polypeptide can be determined directly or indirectly using any suitable method (e.g., CD, fluorescence, binding assay). For example, a polypeptide solution can be prepared and ellipticity of the solution determined by CD. The ellipticity value obtained represents a relative concentration of folded polypeptide of 100%. The polypeptide in the solution is then unfolded by incrementally adding unfolding agent (e.g., heat, a chaotropic agent) and ellipticiry is determined at suitable increments (e.g., after each increase of one degree in temperature). The polypeptide in solution is then refolded by incrementally reducing the unfolding agent and ellipticiry is determined at suitable increments. The data can be plotted to produce<br>
an unfolding curve and a refolding curve. As shown in Fig. 7, the unfolding and refolding curves have a characteristic shape that includes a portion in which the polypeptide molecules are folded, an unfolding/refolding transition in which polypeptide molecules are unfolded to various degrees, and a portion in which the polypeptide molecules are unfolded. The y-axis intercept of the refolding curve is the relative amount of refolded protein recovered. A recovery of at least about 50%, or at least about 60%, or at least about 70%, or at least about 75%, or at least about 80%, or at least about 85%, or at least about 90%, or at least about 95% is indicative that the polypeptide unfolds reversibly.<br>
In a preferred embodiment, the soluble polypeptide unfolds reversibly when heated. Reversibility of unfolding of the soluble polypeptide is determined by preparing a polypeptide solution and plotting heat unfolding and refolding curves for the polypeptide. The peptide solution can be prepared in any suitable solvent, such as an aqueous buffer that has a pH suitable to allow the peptide to dissolve (e.g., pH that is about 3 units above or below the isoelectric point (pI)). The polypeptide solution is concentrated enough to allow unfolding/folding to be detected. For example, the polypeptide solution can be about 0.1 µM to about 100 µM, or preferably about 1 µM to about 10 µM.<br>
If the melting temperature (Tm) of the soluble polypeptide is known, the solution can be heated to about ten degrees below the Tm (Tm-10) and folding assessed by elliptictty or fluorescence (e.g., far-UV CD scan from 200 nm to 250 nm, fixed wavelength CD at 235 nm or 225 nm; tryptophan fluorescent emission spectra at 300 to 450 run with excitation at 298 nm) to provide 100% relative folded polypeptide. The solution is then heated to at least ten degrees above Tm (TrrH-10) in predetermined increments (e.g., increases of about 0.1 to about 1 degree), and ellipticity or fluorescence is determined at each increment. Then, the polypeptide is refolded by cooling to at least Tm-10 in predetermined increments and ellipticity or fluorescence determined at each increment If the melting temperature of the polypeptide is not known, the solution can be unfolded by incrementally heating from about 25°C to about 100°C and then refolded by incrementally cooling to at least about 25"C, and ellipticity or fluorescence at each heating and cooling increment is determined. The data obtained can be plotted to produce an unfolding<br>
curve and a refolding curve, in which the y-axis intercept of the refolding curve is the relative amount of refolded protein recovered.<br>
Some polypeptides unfold reversibly as soluble polypeptides, but not as displayed polypeptides (e.g., displayed as phage coat protein fusion proteins on the surface of a bacteriophage). However, polypeptides that undergo reversible unfolding as displayed polypeptides generally also undergo reversible unfolding when prepared as soluble polypeptides. Thus, reversible unfolding in the context of a displayed polypeptide is highly advantageous and affords the ability to select polypeptides that are reversibly unfoldable as soluble polypeptides from a repertoire or library of displayed polypeptides.<br>
Unfolding and refolding of polypeptides that are contained within a polypeptide display system, for example, polypeptides displayed on bacteriophage, can be assessed by detecting polypeptide function that is dependent upon proper folding. For example, a polypeptide display system comprising displayed polypeptides that have a common function, such as binding a common ligand (e.g., a generic ligand, a target ligand, a substrate), can be unfolded and then refolded, and refolding can be assessed using a functional assay. For example, whether a polypeptide that has a binding activity unfolds reversibly can be determined by displaying the polypeptide on a bacteriophage and measuring or determining binding activity of the displayed polypeptide. The displayed polypeptide can be unfolded by heating the phage displaying the polypeptide to about 80°C, and then refolded by cooling the phage to about 20°C or about room temperature, and binding activity of the refolded polypeptide can be measure or determined. A recovery of at least, about 50%, or at least about 60%, or at least about 70%, or at least about 75%, or at least about 80%, or at least about 85%, or at least about 90%, or at least about 95% of the binding activity is indicative that the polypeptide unfolds reversibly. In a preferred embodiment, the displayed polypeptide comprises an antibody variable domain, and binding to a generic ligand (e.g., protein A, protein L) or a target ligand is determined.<br>
The polypeptides disclosed herein unfold reversibly in solution and/or when displayed in a suitable polypeptide display system at a polypeptide concentration of at least about 1 µM to about 1 mM, at least about 1 µM to about 500 µM, or at least<br>
about 1 µM to about 100 µM. For example, certain single human antibody variable domains can unfold reversibly when displayed in a multimeric phage display system that produces a local concentration (on the phage tip) of displayed antibody variable domain polypeptide of about 0.5 mM. In particular embodiments, the polypeptides unfold reversibly in solution or when displayed on the phage tip at a polypeptide concentration of about 10 µM, about 20 µM, about 30 µM, about 40 µM, about 50 µM, about 60 µM, about 70 µM, about 80 µM, about 90 µM, about 100 µM, about 200 µM, about 300 µM, about 400 µM or about 500 µM.<br>
SELECTION METHODS<br>
In one aspect, the invention is a process for selecting, isolating and/or recovering a polypeptide that unfolds reversibly from a library or a repertoire of polypeptides (e.g., a polypeptide display system). la one embodiment, the method comprises unfolding a collection of polypeptides (e.g., the polypeptides in a library, a repertoire or a polypeptide display system), refolding at least a portion of the unfolded polypeptides, and selecting, isolating and/or recovering a refolded polypeptide. In another embodiment, the method comprises providing a collection of unfolded polypeptides (e.g., the polypeptides in a library, a repertoire or a polypeptide display system), refolding at least a portion of the unfolded polypeptides, and selecting, isolating and/or recovering a refolded polypeptide.<br>
Polypeptide Display Systems<br>
Preferably, the polypeptide that unfolds reversibly is selected, isolated and/or recovered from a repertoire of polypeptides in a suitable polypeptide display system. For example, a polypeptide that unfolds reversibly can be selected, isolated and/or recovered from a repertoire of polypeptides that is in solution, or is covalentiy or noncovalently attached to a suitable surface, such as plastic or glass (e.g., microtiter plate, polypeptide array such as a microarray). For example an array of peptides on a surface in a manner that places each distinct library member (e.g., unique peptide sequence) at a discrete, predefined location in the array can be used. The identity of each library member in such an array can be determined by its spatial location in the array. The locations in the array where binding interactions between a target tigand,<br>
for example, and reactive library members occur can determined, thereby identifying the sequences of the reactive members on the basis of spatial location. (See, e.g., U.S. Patent No. 5,143,854, WO 90/15070 and WO 92/10092.)<br>
Preferably, the method employs a polypeptide display system that links the coding function of a nucleic acid and physical, chemical and/or functional characteristics of ihe polypeptide encoded by the nucleic acid. Preferably, the polypeptide display system comprises a library, such as a bacteriophage display library, Bacteriophage display is a particularly preferred polypeptide display system.<br>
A number of suitable bacteriophage display systems (eg., monovalent display and multivalent display systems) have been described. (See, eg., Griffiths et al., U.S. Patent No. 6,555,313 Bl (incorporated herein by reference); Johnson et alt U.S. Patent No. 5,733,743 (incorporated herein by reference); McCafferty et al., U.S. Patent No. 5,969,108 (incorporated herein by reference); Mulligan-Kehoe, U.S. Patent No. 5,702,892 (incorporated herein by reference); Winter, G. et al,,Annu. Rev. Immunol. 72:433-455 (1994); Soumillion, P. et al,, Appl Biochem. Biotechnol 47(2-3):175-189 (1994); Castagnoli, L. etal, Comb. Chem. High Throughput Screen, 4(2): 121-133 (2001).) The polypeptides displayed in a bacteriophage display system can be displayed on any suitable bacteriophage, such as a filamentous phage (e.g., fd, M13, Fl), a lyric phage (e.g., T4, T7, lambda), or an RNA phage (e.g., MS2), for example.<br>
Generally, a library of phage that displays a repertoire of polypeptides, as fusion proteins with a suitable phage coat protein, is produced or provided. Such a library can be produced using any suitable methods, such as introducing a library of phage vectors or phagemid vectors encoding the displayed polypetides into suitable host bacteria, and culturing the resulting bacteria to produce phage (eg., using a suitable helper phage or complementing plasmid if desired). The library of phage can be recovered from such a culture using any suitable method, such as precipitation and centrifugation.<br>
The polypeptide display system can comprise a repertoire of polypeptides that contains any desked amount of diversity. For example, the repertoire can contain polypeptides that have amino acid sequences that correspond to naturally<br>
occurring polypeptides expressed by an organism, group of organisms, desired tissue or desired cell type, or can contain polypeptides that have random or randomized amino acid sequences. If desired, the polypeptides can share a common core or scaffold. For example, all ploypeptides in the repertoire or library can be based on a scaffold selected from protein A, protein L, protein G, a fibronectin domain, an anticalin, CTLA4, a desired enzyme (e.g., a polymerase, a cellulase), or a polypeptide from the immunoglobulin superfamily, such as an antibody or antibody fragment (e.g., a VH, a VL). The polypeptides in such a repertoire or library can comprise defined regions of random or randomized amino acid sequence and regions of common amino acid sequence. In certain embodiments, all or substantially all polypeptides in a repertoire are of a desired type, such as a desired enzyme (e.g., a polymerase) or a desired antigen-binding fragment of an antibody (e.g., human VH or human VL). In preferred embodiments, the polypeptide display system comprises a repertoire of polypeptides wherein each polypeptide comprises an antibody variable domain. For example, each polypeptide in the repertoire can contain a VH, a VL or an Fv (e.g., a single chain Fv).<br>
Amino acid sequence diversity can be introduced into any desired region of a desired polypeptide or scaffold using any suitable method. For example, amino acid sequence diversity can be introduced into a target region, such as a complementarity determining region of an antibody variable domain or a hydrophobic domain, by preparing a library of nucleic acids that encode the diversified polypeptides using any suitable mutagenesis methods (e.g., low fidelity PCR, oligonucleotide-mediated or site directed mutagenesis, diversification using NNK codons) or any other suitable method. If desired, a region of a polypeptide to be diversified can be randomized.<br>
The size of the polypeptides that make up the repertoire is largely a matter of choice and uniform polypeptide size is not required. Generally, the polypeptides in the repertoire contain at least about nine amino acid residues and have secondary structure. Preferably, the polypeptides in the repertoire have at least tertiary structure (form at least one domain).<br>
The repertoires of polypeptides comprise polypeptides that unfold reversibly and can be unfolded using a suitable unfolding agent as described herein. A<br>
repertoire of polypeptides can be enriched in polypeptides that unfold reversibly and, for example, are secretable when expressed in E. coli. Generally, at least about 10% of the polypeptides contained in such an enriched repertoire unfold reversibly. More preferably, at least about 20%, or at least about 30%, or at least about 40%, or at least about 50%, or at least about 60%, or at least about 70%, or at least about 80%, or at least about 90% of the polypeptides in the enriched repertoire unfold reversibly. Preferred repertoires contain polypeptides that unfold reversibly when heated.<br>
In certain embodiments, substantially all polypeptides ha the repertoire share a common selectable characteristic (e.g., physical characteristic, chemical characteristic, functional characteristic). Preferably, the common selectable characteristic is dependent upon proper folding and distinguishes properly folded polypeptides from unfolded and tnisfolded polypeptides. For example, the common selectable characteristic can be a characteristic such as binding affinity which allows properly folded polypeptides to be distinguished from and selected over misfolded and unfolded polypeptides. In certain embodiments, the common selectable characteristic can be used to select properly folded polypeptides but is absent from unfolded and misfolded polypeptides. For example, a repertoire of polypeptides in which substantially all polypeptides in the repertoire have a common functional characteristic that distinguishes properly folded polypeptides from unfolded and misfolded polypeptides, such as a common binding function (e.g., bind a common generic ligand, bind a common target ligand, bind (or are bound by) a common antibody), a common catalytic activity or resistant to proteotysis (e.g., proteolysis mediated by a particular protease) can be used in the method. In other embodiments, a repertoire of polypeptides in which substantially all polypeptides that unfold reversibly in the repertoire have a common selectable characteristic that distinguishes properly folded polypeptides from unfolded and misfolded polypeptides can be used in the method.<br>
In particular embodiments, the polypeptide display system comprises a library of polypeptides that comprise immunoglobulm variable domains (eg:, VH, VL). The variable domains can be based on a germline sequence (e,g., DP47dummy (SEQ ID NO:3, DPK9 dummy (SEQ ID NO:6)) and if desired can have one or more<br>
diveisified regions, such as the complementarity determining regions. Other suitable gennline sequence for VH include, for example, sequences encoded by the VH gene segments DP4, DP7, DPS, DP9, DP10, DP31, DP33, DP45, DP46, DP49, DP50, DP51, DP53, DP54, DP65, DP66, DP67, DP68 and DP69, and the JH segments JH1, JH2, JH3, JH4, JH4b, JH5 and JH6. Other suitable germline sequence for VL include, for example, sequences encoded by the VK gene segments DPKl, DPK2, DPK3, DPK4, DPK5, DPK6, DPK7, DPK8, DPK9, DPK10, DPK12, DPK13, DPK15, DPK16, DPK18, DPK19, DPK20, DPK21, DPK22, DPK23, DPK24, DPK25, DPK26 and DPK 28, and the JK segements JK 1, JK 2, JK 3, JK 4 and JK 5.<br>
One or more of the framework regions (FR) of the variable domain can comprise (a) the amino acid sequence of a human framework region, (b) at least 8 contiguous amino acids of the amino acid sequence of a human framework region, or (c) an ammo acid sequence encoded by a human germline antibody gene segment, wherein said framework regions are as defined by Rabat. In certain embodiments, the amino acid sequence of one or more of the framework regions is the same as the amino acid sequence of a corresponding framework region encoded by a human germline antibody gene segment, or the amino acid sequences of one or more of said framework regions collectively comprise up to 5 amino acid differences relative to the amino acid sequence of said corresponding framework region encoded by a human germline antibody gene segment.<br>
In other embodiments, the amino acid sequences of FR1, FR2, FR3 and FR4 are Hie same as the amino acid sequences of corresponding framework regions encoded by a human germline antibody gene segment, or the amino acid sequences of FR1, FR2, FR3 and FR4 collectively contain up to 10 amino acid differences relative to the amino acid sequences of corresponding framework regions encoded by said human germline antibody gene segments. In other embodiments, the amino acid sequence of said FR1, FR2 and FR3 are the same as the amino acid sequences of corresponding framework regions encoded by said human germline antibody gene segment.<br>
The polypeptides comprising a variable domain preferably comprise a target Hgand binding site and/or a generic ligand binding site. In certain embodiments, the<br>
generic ligand binding site is a binding site for a superantigen, such as protein A, protein L or protein G.<br>
The variable domain can be based on any desired variable domain, for example a human VH (e.g., VH la, VH lb, VH 2, VH 3, VH 4, VH 5, VH 6), a human V (e.g., VI, VII, VIII, VIV, VV or VVI) or a human VK (e.g,, Vkl, VK2, VK3, VK4, VK5, VK6, VK7, VK8, VK9 or VK10). Preferably, the variable domain is not a Catnelid immunoglobulin domain, such as a VH H, or contain one or more amino acids (e.g., frame work amino acids) that are unique to Cametid immunoglobulin variable domains encoded by germline sequences but not, for example, to human immunoglobulin variable domains. (See, e.g., Davies et al., Protein Engineering 9:531-537 (1996); Tanha et al., J. Biol Chem. 276:24774-24780 (2001); Riechmann etal, J. Immunol. Methods 23:25-38 (1999).) In one embodiment, the VH that unfolds reversibly does not comprise one or more amino acids that are unique to murine (e.g., mouse) germline framework regions. Preferably, the variable domain unfolds reversibly when heated.<br>
The isolated polypeptide comprising a variable domain can be an antibody format Thus, hi certain embodiments, the isolated polypeptide comprising a variable domain that unfolds reversibly can be a homodhner of variable domain, a heterodimer comprising a variable domain, an Fv, a scFv, a disulfide bonded Fv, a Fab, a single variable domain or a variable domain fused to an immunoglobulin Fc portion.<br>
In one embodiment the polypeptide display system comprises polypeptides that are nucleic acid polymeiases, such as variants of a thermostable DNA polymerase (e.g., Taq polymerase.)<br>
Unfolding and Refolding<br>
The polypeptides (e.g., displayed polypeptides) can be unfolded using any desired unfolding agent Suitable unfolding agents include, for example, heat and/or pressure, low or highpH, chaotropic agents (e.g., guanidine hydrochloride, urea and the like) and organic solvents (e.g., an alcohol (e.g., methanol, ethanol), a ketone (e.g.t methyl ethyl ketone), an aldehyde (e.g., formaldehyde, dimethytformaldehyde), tetrahydrofuran, dioxane, toluene and the like). In certain<br>
embodiments, the displayed polypeptides are unfolded using an unfolding agent, with the proviso that the unfolding agent is not a chaotropic agent Generally, unfolding is effectuated by exposing the collection of polypeptides to an amount of unfolding agent (e.g., heat) that is sufficient to cause at least about 10% of the polypeptides in the collection to unfold. In particular embodiments, unfolding is effectuated by exposing the collection of polypeptides to an amount of unfolding agent (e.g., heat) that is sufficient to cause at least about 20%, or at least about 30%, or at least about 40%, or at least about 50%, or at least about 60% or at least about 70%, or at least about 80%, or at least about 90%, or at least about 95%, or at least about 98%, or at least about 99%, or substantially all of the polypeptides in me collection to unfold.<br>
In practice, the polypeptides in the repertoire would have a range of melting temperatures (Tm). A Tm (e.g., for use in methods described herein) can be obtained for a repertoire by obtaining a random sample of about 10 to about 100 polypetides from the repertoire and determining the average Tm for the polypeptides in the sample.<br>
Preferably, the displayed polypeptides are unfolded by heating the polypeptide display system to a suitable unfolding temperature (Ts), such as a temperature mat is sufficient to cause at least about 10% of the displayed polypeptides to unfold. A temperature that is sufficient to cause a desired percentage of displayed polypeptides to unfold can readily determined using any suitable methods, for example by reference to an unfolding and/or refolding curve (as described herein). When it is desirable to unfold substantially all displayed polypeptides, the lowest temperature that falls within the unfolded portion of the unfolding and refolding curve for the polypeptide system will generally be sufficient. The temperature selected will be dependent upon the thermostability of the displayed polypeptides. For example, an unfolding temperature of 100°C or higher can be used if the displayed polypeptides are from, or are variants of polypeptides from, a thermophile or extreme thermophile (e.g., Thermus aquations). When the polypeptides are unfolded using heat, the polypeptides are generally unfolded by heating to temperature (Ts) that is at least about the melting temperature<br>
(Tm) of the polypeptide to be selected or greater that the Tin of the polypeptide to he selected.<br>
In certain embodiments, the displayed polypeptides are unfolded by raising the temperature of the polypeptide display system to an unfolding temperature that is between about 25°C and about 100°C. When the polypeptide display system is phage display, it is preferred mat the displayed polypeptides are unfolded by raising the temperature of the phage display system to about 80°C. Unfolding displayed or soluble polypeptides at high temperatures (e.g., at about 100°C) is also preferred and can be advantageous. For example, heat sterilizable polypeptides can be selected, isolated and/or sterilized by heating the polypeptide display or system or soluble polypeptide to about 100"C.<br>
Once the desired unfolding temperature has been attained, the polypeptide display system can be maintained at that temperature for a period of time (e.g., up to about 10 hours), if desired. For example, the polypeptide display system can be maintained at the unfolding temperature for a period of about 100 milliseconds to about 10 hours. In particular embodiments, the polypeptide display system is maintained at the unfolding temperature for about 1 second to about 20 minutes.<br>
The temperature of the polypeptide display system can be raised at any suitable rate, for example at a rate of about 1°C per millisecond to about 1°C per hour. In a particular embodiment, the temperature is preferably raised at a rate of about 1°C per second.<br>
The unfolded polypeptides in the polypeptide display system, or a portion of the unfolded polypeptides, can be refolded by decreasing the amount or concentration of unfolding agent in the system. The amount or concentration of unfolding agent in the system can be decreased using any suitable method, for example, by dilution, dialysis, buffer exchange, titration or other suitable method. Heat can be reduced by cooling at room temperature or under refrigeration (e.g., in a refrigerated cooling block or bath), for example. Pressure can. be reduced, for example, by venting.<br>
It may be desirable to refold only a portion of the unfolded displayed polypeptides before selection. For example, highly stable polypeptides that unfold reversibly can be selected, isolated and/recovered when the unfolding agent is<br>
decreased minimally, such that only the most stable portion (e,g,t about 0.00001% to about 1%) of the unfolded displayed polypeptides refold. Accordingly, refolding can be effectuated by decreasing the amount or concentration of unfolding agent in the polypeptide display system to an amount or concentration that results in the desired degree of refolding. The amount or concentration of unfolding agent that can remain in the polypeptide display system but permit the desired percentage of unfolded displayed polypeptides to refold can be readily determined using any suitable method, for example by reference to an unfolding and refolding curve (as described herein). When it is desirable to refold substantially all unfolded displayed polypeptides, the amount or concentration of unfolding agent can be reduced to the concentration or amount in the polypeptide display system before unfolding (e.g. &gt; the system is cooled to room temperature) or the unfolding agent can be substantially removed.<br>
Generally, refolding is effectuated by decreasing the amount or concentration of unfolding agent (e.g., heat) so that at least about 0.00001% of the unfolded polypeptides refold. In particular embodiments, refolding is effectuated by decreasing the amount or concentration of unfolding agent (e.g., heat) so that at least about 0.0001% or at least about 0.001%, or at least about 0.01%, or at least about 0.1%, or at least about 1%, or at least about 10%, or about 20%, or at least about 30%, or at least about 40%, or at least about 50%, or at least about 60% or at least about 70%, or at least about 80%, or at least about 90%, or at least about 95%, or at least about 98%, or at least about 99%, or substantially all of the unfolded polypeptides that can undergo refolding in the polypeptide display system refold.<br>
As described herein, heat is a preferred unfolding agent. In certain embodiments where the displayed polypeptides are heat unfolded, the displayed unfolded polypeptides are refolded by lowering the temperature of the polypeptide display system to a refolding temperature (Tc) that is below about 99°C but above the freezing temperature of the polypeptide display system. As with unfolding, the refolding temperature selected will be dependent upon the thermostability of the displayed polypeptides. For example, a refolding temperature of 100°C or higher can be used if the displayed polypeptides are from, or are variants of polypeptides from, a thermophile or extreme thennophile (e.g., Thermits aquatlcus). Preferably,<br>
the unfolding temperature and refolding temperature differ by at least about 10°C to at least about 120°C.<br>
In one embodiment, the polypeptide display system is a phage display system, the displayed polypeptides are unfolded by heating the system to about 80°C, and the displayed unfolded polypeptides can be refolded by reducing the temperature of the phage display system to a temperature between about 1°C to about 70°C. In more particular embodiments, the displayed unfolded polypeptides are refolded by reducing the temperature of the phage display system to a temperature between about 1°C to about 60°C, or about 1°C to about 50°C, or about 1"C to about 40°C, or about 1°C to about 30°C, or about 1°C to about 20°C, or about 1°C to about 10°C.<br>
Once the desired refolding temperature has been attained, the polypeptide display system can be maintained at that temperature for any desired period of time (e.g., up to about 10 hours), if desired. For example, the polypeptide display system can be maintained at the refolding temperature for a period of about 100 milliseconds to about 10 hours. In particular embodiments, the polypeptide display system is maintained at the refolding temperature for about 1 minute to about 20 minutes.<br>
The temperature of the polypeptide display system can be decreased at any suitable rate, for example at a rate of about 1°C per millisecond to about 1°C per hour. In a particular embodiment, the temperature is preferably decreased at a rate of about 1°C per 100 milliseconds or about I°C per second.<br>
The polypeptide display system can be maintained at atmospheric pressure (~1 atm) during unfolding and refolding of the displayed polypeptides. However, if desired, the polypeptide display system can be maintained at lower or higher pressure. In such situations, more or less unfolding agent may be required to obtain the desired degree of unfolding. For example, more or less heat may need to be applied to the polypeptide display system (relative to a system at 1 atm) hi order to achieve the desired unfolding temperature because lowering or raising the pressure of the system can change the temperature of the system.<br>
Unfolding and refolding can be carried out under suitable pH or buffer conditions. Generally, unfolding and refolding are carried out at apH of about 1 to<br>
about 13, or about 2 to about 12. Preferred pH conditions for unfolding and refolding are a pH of about 5 to about 9, or about 6 to about 8, or about 7. Folding and unfolding the polypeptides under acidic or alkaline conditions can allow for selection of polypeptides that function under extreme acidic or alkaline conditions, such as polypeptide drugs that can be administered orally and/or have therapeutic action in the stomach.<br>
Selection/Isolation/Recovery<br>
A polypeptide that unfolds reversibly (e.g., a population of polypeptides that unfold reversibly) can be selected, isolated and/or recovered from a repertoire or library (e.g., in a polypeptide display system) using any suitable method. A polypeptide can be recovered by selecting and/or isolating the polypeptide. Recovery can be carried out at a suitable recovery temperature (Tr). Generally, a suitable recovery temperature is any temperature that is less than the melting temperature (Tm) of the polypeptide but higher than the freezing temperature of the polypeptide display system. In certain embodiments, the recovery temperature (Tr) is substantially (he same as the refolding temperature (To), (e.g,, Tr=Tc).<br>
In certain embodiments, a polypeptide that unfold reversibly is selected or isolated based on a selectable characteristic (e.g., physical characteristic, chemical characteristic, functional characteristic) mat distinguishes properly folded polypeptides from unfolded and tnisfolded polypeptides. Examples of such selectable characteristics include fluorescence, susceptibility to fluorescence quenching and susceptibility to chemical modification (e.g., with lodoacetic acid, lodoacetamide or other suitable polypeptide modifying agent). Preferably, a polypeptide mat unfolds reversibly is selected, isolated and/or recovered based on a suitable selectable functional characteristic that distinguishes properly folded polypeptides from unfolded and misfolded polypeptides. Suitable functional characteristics for selecting or isolating a polypeptide that unfolds reversibly include any function that is dependent on proper folding of the polypeptide. Accordingly, a polypeptide that unfolds reversibly can have the function when properly folded, but the function can be lost or diminished upon unfolding and can be absent or diminished when the polypeptide is unfolded or misfolded. Suitable selectable<br>
functional characteristics include, for example, binding to a generic ligand (e.g., a superantigen), binding to a target ligand (e.g., an antigen, an epitope, a substrate), binding to an antibody (e.g., through an epitope expressed on the properly folded polypeptide), a catalytic activity and resistance to proteolysis. (See, e.g., Tomlinson et al., WO 99/20749; WO 01/57065; WO 99/58655.)<br>
In preferred embodiments, the polypeptide that unfolds reversibly is selected and/or isokted from a library or repertoire of polypeptides in which substantially all polypeptides that unfold reversibly share a common selectable feature. For example, the polypeptide that unfolds reversibly can be selected from a library or repertoire of polypeptides in which substantially all polypeptides that unfold reversibly bind a common generic ligand, bind a common target ligand, bind (or are bound by) a common antibody, possess a common catalytic activity or are each resistant to proteolysis (e.g., proteolysis mediated by a particular protease). Selection based on binding to a common generic ligand can yield a population of polypeptide that contains all or substantially all polypeptides that unfold reversibly that were components of the original library or repertoire.<br>
Any suitable method can be used to select, isolate and/or recover the polypeptides that unfold reversibly. For example, polypeptides that bind a target ligand or a generic ligand, such as protein A, protein L or an antibody, can be selected, isolated and/or recovered by panning or using a suitable affinity matrix. Fanning can be accomplished by adding a solution of ligand (e.g., generic ligand, target ligand) to a suitable vessel (e.g., tube, petri dish) and allowing the ligand to become deposited or coated onto the walls of the vessel. Excess ligand can be washed away and polypeptides (e.g., a phage display library) can be added to the vessel and the vessel maintained under conditions suitable for polypeptides to bind the immobilized ligand. Unbound polypeptide can be washed away and bound polypeptides can be recovered using any suitable method, such as scraping or lowering the pH, for example.<br>
Suitable ligand affinity matrices generally contain a solid support or bead (e.g.t agarose) to which a ligand is covalently or noncovalently attached. The affinity matrix can be combined with polypeptides (e.g,, a phage display library) using a batch process, a column process or any other suitable process under<br>
conditions suitable for binding of polypeptides to the ligand on the matix. Polypeptides that do not bind the affinity matrix can be washed away and bound polypeptides can be eluted and recovered using any suitable method, such as elation with a lower pH buffer, with a mild denaturing agent (eg., urea), or with a peptide that competes for binding to the ligand.<br>
In some embodiments, the generic or target ligand is an antibody or antigen binding fragment thereof. Antibodies or antigen binding fragments that bind structural features of polypeptides mat are substantially conserved in the polypeptides of a library or repertoire are particularly useful as generic ligands. Antibodies and antigen binding fragments suitable for use as ligand for isolating, selecting and/or polypeptides that unfold reversibly can be monoclonal or polyclonal and can be prepared using any suitable method.<br>
Polypeptides that unfold reversibly can also be selected, for example, by binding metal ions. For example, immobilized metal affinity cinematography (IMAC; Hubert and Porath, J, Chromotography, 98:247(1980)) takes advantage of the metal-binding properties of histidine and cysteine amino acid residues, as well as others that may bind metals, on the exposed surfaces of numerous proteins. It employs a resin, typically agarose, comprising a bidentate metal chelator (e.g. iminodiacetic acid, IDA, a dicarboxylic acid group) to which is complexed metallic ions. Such resins can be readily prepared, and several such resins are commercially available, such as CHELATING SEPHAROSE 6B (Pharmacia Fine Chemicals; Piscataway, NJ). Metallic ion that are of use include, but are not limited to, the divalent cations Mi2+, Cu2+, Zn2+, and Co2+. A repertoire of polypeptides or library can prepared in binding buffer which consists essentially of salt (typically, NaCl or KC1) at a 0.1- to 1 .0M concentration and a weak ligand (e.g., Tris, ammonia), the latter of which has affinity for the metallic ions of the resin, but to a lesser degree than does a polypeptide to be selected according to the invention. Useful concentrations of Hie weak ligand range from 0.01- to 0.1M in the binding buffer.<br>
The repertoire of polypeptides or library can contacted with the resin under conditions which permit polypeptides having metal-binding domains to bind. Non-binding polypeptides can be washed away, and the selected polypeptides are eluted with a buffer in which the weak ligand is present in a higher concentration than in<br>
the binding buffer, specifically, at a concentration sufficient for the weak ligand to displace the selected polypeptides, despite its lower binding affinity for the metallic ions. Useful concentrations of the weak ligand in the elution buffer are 10- to 50-fold higher than hi the binding buffer, typically from 0.1 to 0.3 M. Preferably the concentration of salt in the elution buffer equals that in the binding buffer. According to the methods of the present invention, the metallic ions of the resin typically serve as the generic ligand; however, it is contemplated that they can also be used as the target ligand.<br>
IMAC can be carried out using a standard chromatography apparatus (columns, through which buffer is drawn by gravity, pulled by a vacuum or driven by pressure), or by batch procedure, in which the metal-bearing resin is mixed, in slurry form, with the repertoire of polypeptides or library.<br>
Partial purification of a serum T4 protein by IMAC has been described (Staples et al, U.S. Patent No. 5,169,936); however, the broad spectrum of proteins comprising surface-exposed metal-binding domains also encompasses other soluble T4 proteins, human serum proteins (e.#., IgG, haptoglobin, hemopexin, Gc-globulin, Clq, C3, C4), human desmoplasmin, Dolichos biflorus lectin, zinc-inhibited Tyr(P) phosphatases, phenolase, carboxypeptidase isoenzymes, Cu, Zn superoxide dismutases (including those of humans and all other eukaryotes), nucleoside   . diphosphatase, leukocyte interferon, lactoferrin, human plasma α2-SH glycoprotein, 02-macroglobulin, ai-antitrypsin, plasminogen activator, gastrointestinal polypeptides, pepsin, human and bovine serum albumin, granule proteins from granulocytes, lysozymes, non-histone proteins, human, fibrinogen, human serum transferrin, human lymphotoxm, calmodulin, protein A, avidin, myoglobins, somatomedins, human growth hormone, transforming growth factors, platelet-derived growth factor, α-human atrial natriuretic polypeptide, cardiodilatin and others. In addition, extracellular domain sequences of membrane-bound proteins may be purified using IMAC. Polypeptides that unfold reversibly and comprise a scaffold from any of the above proteins or metal-binding variants thereof can be selected, isolated or recovered using the methods described herein.<br>
Polypeptides that unfold reversibly can also be selected from a suitable phage display system (or other suitable polypeptide display system), based on<br>
infectivity of phage following unfolding and refolding or based on aggregation of phage displaying polypeptides. A high looal concentration of displayed polypeptide can be produced by displaying polypeptides on multivalent phage proteins (e.g., pffl protein of filamentous bacteriophage). The displayed polypeptides are adjacently located at the phage tip and can interact with each other and form aggregates if they do not unfold reversibly. Such aggregates can be intraphage aggregates which form between the polypeptides displayed on a particular phage and/or interphage aggregates which form between polypeptides displayed on two or more phage when the phage display system comprises a plurality of phage particles at a sufficient concentration. Accordingly, polypeptides that unfold reversibly can be selected from a multivalent phage display system by recovering displayed polypeptides that do not aggregate using any suitable method, such as centrifugation (e.g., ulrracentrirugation), or by selecting based on function of the displayed polypeptide or infectivity of the phage.<br>
For example, when heated to a suitable temperature (e.g., about 80'C) the displayed polypeptides unfold. However, heating filamentous phage to 80'C reduces infectivity of the phage only slightly (Holliger et al., J, Mol. BioL 288:649-657 (1999)). The high local concentration of displayed polypeptides can lead to aggregation of unfolded polypeptides and thereby, substantially reduce the infectivity of phage that display polypeptides that do not unfold reversibly. For example, infectivity of phage that display polypeptides that do not unfold reversibly can be reduced by 70 fold or more. However, infectivity of phage displaying polypeptides that unfold reversibly is reduced to a lesser degree or substantially unchanged by heating, for example, at 80°C (relative to infectivity of phage that do not display polypeptides following heating at 80°C). Accordingly, in certain embodiments, polypeptides that unfold reversibly can be selected by unfolding and refolding the polypeptides in a suitable polypeptide display system (e.g., a filamentous bacteriophage display system), and selecting phage with infectivity that is not substantially reduced or that is substantially unchanged. The selected polypeptides that unfold reversibly can be further selected for any desired properties, such as binding a desired antigen, catalytic activity, and the like. Selection based on infectivity can also be employed to prepare a library or repertoire fhat is enriched in<br>
polypeptides that unfold reversibly or nucleic acids encoding polypeptides that unfold reversibly.<br>
Polypeptides that unfold reversibly can also be selected from a suitable phage display system (or other suitable polypeptide display system), based on aggregation of phage displaying polypeptides that do not unfold reversibly. For example, a suitable polypeptide display system can be unfolded (e.g., by heating to about 80°C) and refolded by cooling under conditions in which at least a portion of polypeptides that do not unfold reversibly aggregate. The presence or degree of aggregation can be determined using any suitable method, such as electron microscopy or an infectivity assay. Polypeptides mat unfold reversibly can be selected by recovering polypeptides (e.g., displayed on phage) that do not aggregate using any suitable method, such, as centrifugation (e.g., ultracentritugation) or by infeceting suitable host bacteria (when die polypeptides are displayed on phage).<br>
Other polypeptide display systems, including systems in which polypeptides are immobilized on a solid support, can be prepared in which displayed polypeptides that do not unfold reversibly can form aggregates. Generally, the displayed polypeptides in such a system are positioned in close proximity to each other. For example, the displayed polypeptides can separated by no more than about twice the length of linear amino acid sequence of the polypeptide. The displayed polypeptides are separated by no more that a distance that is determined by multiplying the number of amino acid residues in the polypeptide by the length of a peptide bond (3.8 A) times 2. For example, the polypeptides can be separated by no more than about 200 A to about 300 A. In a particular example, the polypeptide contains 100 amino acids, and they are separated by no more that 760A. The displayed polypeptides are spaced no closer than the distance between the centers of two adjacent identical globular polypetides. For example, two immunoglobulin variable domains that are tethered to a substrate by their C-termini should be no closer that 25A.<br>
Such polypeptide display systems can be prepared using any suitable method. For example, polypeptides can be concatenated (see, e.g,, WO 02/30945) or produced as fusion proteins which bring the polypeptides together (e.g., by dimerizing or oligomerizing, by assembly into a viral coat or capsid). Such<br>
olypeptide display systems can also be prepared by immobilizing polypeptides on to a suitable solid support (e.g., a bead, plastic, glass) using any suitable method. Polypeptides that unfold reversibly can be selected from such a polypeptide display system by recovering displayed polypeptides that do not aggregate using any suitable method. For example, when the polypeptides are displayed on a mobile sold support (e.g., beads) interbead aggregates can form and be removed by as centrifugation or other suitable method.<br>
Generally, the polypeptide display system comprises a plurality of polypeptide species and more than one copy (polypeptide molecule) of each species. Preferably, each polypeptide species is present in a concentration that is sufficient to permit aggregation of species that do not unfold reversibly. For example, in polypeptide display systems in which more than one copy of a displayed polypeptides species are located adjacent to each other or co-localized, such as phage display, each species of polypeptide is present in a concentration (e.g., local concentration such as on the phage tip) that is sufficiently high to permit aggregation of species that do not unfold reversibly.<br>
In one embodiment, the process for selecting a polypeptide that binds a ligand and unfolds reversibly from a repertoire of polypeptides comprises, providing a polypeptide display system comprising a repertoire of polypeptides; heating Hie repertoire to a temperature (Ts) at which at least a portion of the displayed polypeptides are unfolded; cooling the repertoire to a temperature (Tc) wherein Ts&gt;Tc, whereby at least a portion of the polypeptides have refolded and a portion of the polypeptides have aggregated; and recovering at a temperature (Tr) at least one polypeptide that unfolds reversibly and binds a ligand, Preferably, the ligand is bound by (binds) folded polypeptide but is not bound by (does not bind) aggregated polypeptides. The recovered polypeptide that unfolds reversibly has a melting temperature (Tm), and preferably, the repertoire was heated to Ts, cooled to Tc and the polypeptide that unfolds reversibly was isolated at Tr, such that Ts&gt;Tm&gt;Tc, and Ts&gt;Tm&gt;Tr. Preferably, the recovered polypeptide is not misfolded. Misfolded polypeptides can be identified based on certain selectable characteristic (e.g., physical characteristic, chemical characteristic, functional characteristic) that<br>
distinguishes properly folded polypeptides from unfolded and tnisfolded polypeptides as described herein.<br>
In additional embodiments, the process further comprises confirming that me recovered polypeptide binds a ligand (e.g., target ligand, generic ligand). The process can further comprise confirming that the Tm of the recovered polypeptide is less than the temperature Ts to which the repertoire was heated. The Tm of a recovered polypeptide can be determined using any suitable method, e.g., by circular dichroism, change in fluorescence of the polypeptide with increasing temperature, differential scanning calorimetry (DSC). Preferably, the recovered polypeptide has a Tm that is equal to or greater than about 37°C, and more preferably greater that about 37°C, when rounded up to the nearest whole number. In other embodiments, the Tm of the recovered polypeptide is less than about 60°C. In other embodiments, the Tm of the recovered polypeptide is greater than about 37°C and less than about 60°C.<br>
In one embodiment, the polypeptide is recovered at a temperature (Tr) that is substantially the same as Tc (e.g, Tc=Tr). In other embodiments, Ts is at least about 60°C, In certain embodiments, the Tm of the recovered polypeptide is less than about 60°C, and the repertoire was heated to a temperature (Ts) greater than about 60°C.<br>
In certain embodiments, the polypeptide display system comprises a plurality of replicable genetic display packages. Preferably, the polypeptide display system is a phage display system, such as a multivalent phage display system. In some embodiments, the polypeptide display system, pre-selection repertoire and/or preselection library comprises at least about 103 members (species), at least about 104 members, at least about 10s members, at least about 106 members, or at least about 107 members. In one embodiment, a phage display system is provided and at least a portion of the polypeptides that aggregate at Tc form interphage aggregates. In another embodiment, a phage display system is provided and at least a portion of the polypeptides that aggregate at Tc form interphage aggregates and intraphage aggregates.<br>
METHODS FOR DESIGNING POLYPEPTIDES<br>
In another aspect, the invention relates to a method for designing and/or preparing a variant polypeptide that unfolds reversibly. The method comprises providing the amino acid sequence of a parental polypeptide, identifying one or more regions of said amino acid sequence that are hydrophobic, selecting one or more of the hydrophobic regions, and replacing one or more amino acids in a selected region to produce a variant amino acid sequence in which the hydrophobicity of the selected region is reduced, A polypeptide that comprises or consists of the variant amino acid sequence can be produced and, if desired, its ability to unfold reversibly can be assessed or confirmed using any suitable method. Preferably, the variant unfolds reversibly when heated and cooled, as described herein.<br>
A variant polypeptide that unfolds reversibly can be prepared using any desired parental polypeptide. For example, the polypeptide that unfolds reversibly can be prepared using a parental polypeptide that is based on a scaffold selected from an enzyme, protein A, protein L, a fibronectin, an anticalin, a domain of CTLA4, or a polypeptide of the immunoglobulin superfamily, such as an antibody or an antibody fragment.<br>
When the parental polypeptide is an antibody variable domain (e.g., a human VH, a human VL), it can comprise V and/or D (where the polypeptide comprises a VH domain) and/or J segment sequences encoded by a germline V, D or J segment respectively, or a sequence that results from naturally occurring or artificial mutations (e.g., somatic mutation or other processes). For example, the parental polypeptide can be a parental immunoglobulin variable domain having an amino acid sequence that is encoded by germline gene segments, or that results from insertion and/or deletion of nucleotides during V(D)J recombination (e.g., N nucleotides, F nucleotides) and/or mutations that arise during affinity maturation.<br>
Similarly, the parental polypeptide can be a parental enzyme that has an amino acid sequence encoded by the germline or that results from naturally occurring or artificial mutation (e.g., somatic mutation).<br>
Hydrophobic regions of the parental polypeptide can be identified using any suitable scale or algorithm. Preferably, hydrophobic regions are identified using the method of Sweet and Eisenberg. (Sweet, R.M. and Eisenberg, D., /. Mot Biol.<br>
777:479-488 (1983).) The Sweet and Eisenberg method can produce a hydrophobicity score (S/E score) for an amino acid sequence (e.g.t an identified hydrophobic region) using a 9 to 18 amino acid window (e.g., a window of 9,10,11, 12,13,14,15,16,17 or 18 amino acids). A15 amino acid window is generally preferred. In some embodiments, a parental amino acid sequence is analyzed using the Sweet and Eisenberg method, and a hydrophobicity plot for a selected hydrophobio region is generated (ordinate is the S/E score using a suitable window, abscissa is the amino acid positions of the selected region). A variant amino acid sequence for the selected hydrophobic region is designed in which one or more of the amino acid residues in the parental sequence with a different amino acid residue, and analyzed using the Sweet and Eisenberg method. A hydrophobicity plot for the variant amino acid sequence is produces. A decrease in the area under the curve of the hydrophobicity plot of a selected hydrophobic region in the variant amino acid sequence relative to the area under the curve of the corresponding region in the parental amino acid sequence is indicative of a reduction in hydrophobicity of the selected region. The amino acid sequence of the variant VH or variant VL can comprise one or more of the amino acid replacements for VH or VL described herein. The amino acid sequence of the variant VH or variant VL can comprise one or more framework regions as described herein.<br>
In a preferred embodiments, a polypepu'de comprising a variant antibody variable domain that unfolds reversibly is designed and/or prepared. In one embodiment, the variant antibody variable domain is a variant VH domain in which the hydrophobicity of the amino acid sequence from position 22 to position 36 is reduced (Kabat amino acid numbering system) relative to the corresponding amino acid sequence of the parental VH. In another embodiment, the variant antibody variable domain is a variant VH and the hydrophobicity of the HI loop is reduced relative to the Hi loop of the parental VH (HI loop is as defined by the AbM amino acid numbering system). Preferably, hydrophobicity is determined using the method of Sweet and Eisenberg with a 15 amino acid window for VH. In certain embodiments, the S/E score of the amino acid sequence from position 22 to position 36 of the variant VH is 0.15 or less, or 0.1 or less for an S/E score rounded to one decimal place, or 0.09 or less, or 0.08 or less, or 0.07 or less, or 0.06 or less, or 0.05<br>
or less, or 0.04 or less, or 0 or less. Preferably, the S/E score of the HI loop of the variant VH is 0 or less.<br>
In another embodiment, the variant antibody variable domain is a variant VL and the hydrophobicity of the FR2-CDR2 region and/or FR3 is reduced relative to the FR2-CDR2 region and/or FR3 of the parental VL ( FR2-CDR2 region and FR3 region are as defined by the Kabat amino acid numbering system). In another embodiment, tiie variant antibody variable domain is a variant VL and the hydrophobicity of the amino acid sequence from position 44 to position 53 and/or position 73 to position 76 is reduced relative to the corresponding amino acid sequence of the parental VL (Kabat amino acid numbering system). Preferably, hydrophobicity is determined using the method of Sweet and Eiseriberg with an 11 amino acid window for VL. In certain embodiments, the S/E score of the amino acid sequence from position 44 to position 53 that has a S/E score of less than 0.23. For example, the S/E score for position 44 to position 53 of the variant VL can be than 0.2, less than 0.17, less than 0.15, less man 0.13, less than 0.10, or less than -0.1. la some embodiments, the S/E score for FR3 of the variant VL is less than 0.35. For example, die S/E score for FR3 of the variant VL can be less man 0.3, less than 0,25, less than 0.2, less than 0.17, less than 0.15, less than 0.13, less than 0.10, or less than -0.1.<br>
A variant polypeptide mat comprises a variant amino acid sequence and unfolds reversibly can be prepared using any suitable method. For example, the amino acid sequence of a parental polypeptide can be altered at particular positions (as described herein with respect to antibody variable domain polypeptides) to produce a variant polypeptide mat unfolds reversibly.<br>
LIBRARIES/REPERTOIRES<br>
In other aspects, the invention relates to repertoires of polypeptides that unfold reversibly, to libraries that encode polypeptides that unfold reversibly, and to methods for producing such libraries and repertoires.<br>
The libraries and repertoires of polypeptides comprise polypeptides that unfold reversibly (and/or nucleic acids encoding polypeptides that unfold reversibly) using a suitable unfolding agent as described herein. Generally at least about 1% of<br>
the polypeptides contained in the repertoire or library or encoded by the library unfold reversibly. More preferably, at least about 10%, or at least about 20%, or at least about 30%, or at least about 40%, or at least about 50%, or at least about 60%, or at least about 70%, or at least about 80%, or at least about 90% of the polypeptides in the repertoire or library or encoded by the library unfold reversibly. Such libraries are referred to herein as being enriched in polypeptides that unfold reversibly or in nucleic acids encoding polypeptides that unfold reversibly. Preferred repertoires and libraries contain polypeptides that unfold reversibly when heated. Preferably, the libraries of the invention comprise heterogeneous nucleic acids that are replicable in a suitable host, such as recombinant vectors that contain nucleic acids encoding polypeptides (e.g., plasmids, phage, phagmids) that are replicable inE. coli, for example.<br>
Libraries that encode and/or contain polypeptides that unfold reversibly can be prepared or obtained using any suitable method. The library of the invention can be designed to encode polypeptides that are based on a polypeptide of interest (e.g., a polypeptide selected from a library) or can be selected from another library using the methods described herein. For example, a library enriched hi polypeptides that unfold reversibly can be prepared using a suitable polypeptide display system.<br>
In one example, a phage display library comprising a repertoire of displayed polypeptides comprising an antibody variable domain (e.g.} VH, VK, Vk) is unfolded and refolded as described herein and a collection of refolded polypeptides is recovered thereby yielding a phage display library enriched in polypeptides that unfold reversibly. In another example, a phage display library comprising a repertoire of displayed polypeptides comprising an antibody variable domain (e.g., VH, VK, VX) is first screened to identify members of the repertoire that have binding specificity for a desired target antigen. A collection of polypeptides having the desired binding specificity are recovered, the collection is unfolded and refolded, and a collection of polypeptides that unfold reversibly and have the desired binding specificity is recovered, yielding a library enriched in polypeptides that unfold reversibly.<br>
In another example, a polypeptide of interest, such as an immunoglobulin variable domain, is selected (e.g., from a library) and the amino acid sequence of the<br>
polypeptide is analyzed to identify regions of hydrophobicity. Hydrophobicity can be determined using any suitable method, scale or algorithm. Preferably, hydrophobicity is determined using the method of Sweet and Eisenberg. (Sweet, R.M. and Eisenberg, D., J. Mol. Biol 777:479-488 (1983).) A region of hydrophobicity in the polypeptide is selected and a collection of nucleic acids is prepared that contains sequence diversity targeted to the region encoding the hydrophobic region of the polypeptide. The collection of nucleic acids can be randomized in the target region or sequence diversity that results in reduced hydrophobicity of the selected region of the polypeptide (through amino acid replacement) can be introduced. The collection of nucleic acids can then be insetted into a suitable vector (e.g., a phage, a phagemid) to yield a library. The library can be expressed in a suitable polypeptide display system and the polypeptides in the display system can be unfolded, refolded and a collection of polypeptides that unfold reversibly can be selected or recovered as described herein to yield a library enriched in polypeptides that unfold reversibly.<br>
When the library is based on a polypeptide of interest, it is preferred that the hydrophobic region targeted for sequence diversification is a hydrophobic region that is associated with aggregation of unfolded polypeptides. Such aggregation prone hydrophobic regions can be identified using any suitable method. For example, amino acids in the hydrophobic regions can be replaced with less hydrophobic residues (e.g., Tyr replaced with Asp or Glu). The resulting polypeptides can be unfolded and aggregation can be assessed using any suitable method. Decreased aggregation of polypeptide that contains an amino acid replacement indicates that the hydrophobic region that contains the amino acid replacement is an aggregation prone hydrophobic region.<br>
A particular repertoire of polypeptides or library comprises VH domains (e.g., based on a parental VH domain comprising a germline VH gene segment) containing at least one amino acid replacement that reduces the hydrophobicity of the amino acid sequence from position 22 to position 36, or position 26 to position 35, as defined by the Kabat amino acid numbering scheme. Another particular repertoire of polypeptides or library comprises VH domains wherein the hydrophobicity of the amino acid sequence from position 22 to position 36 (Kabat<br>
amino acid numbering scheme) has a Sweet/Eiseriberg hydrophobicity score of 0 or less. Preferably, at least about 1% of the polypeptides contained in the repertoire or library have a Sweet/Eisenberg hydrophobicity score of 0 or less for the arnino acid sequence from position 22 to position 36. More preferably, at least about 10%, or at least about 20%, or at least about 30%, or at least about 40%, or at least about 50%, or at least about 60%, or at least about 70%, or at least about 80%, or at least about 90% of the polypeptides have a SweefEiseuberg hydrophobicity score of 0 or less for the amino acid sequence from position 22 to position 36.<br>
Another particular repertoire of polypeptides or Ubrary comprises VL domains (e.g., based on a parental VL domain comprising a germline VL gene segment) containing at least one amino acid replacement that reduces the hydrophobicity of the amino acid sequence from position 44 to position 53, as defined by the Kabat amino acid numbering scheme. Another repertoire of polypeptides or Ubrary comprises VL domains (e.g., based on a parental VL domain comprising a germline VL gene segment) containing at least one amino acid replacement fhat reduces the hydrophobicity of framework 3 (FR3), as defined by the Kabat amino acid numbering scheme. Other repertoires and libraries can be produced that contain polypeptides comprising an antibody variable region having amino acid sequence diversity at particular residues or regions as described herein.<br>
The libraries and repertoires can be based on a polypeptide that unfolds reversibly (e.g., when heated), such as a polypeptide that unfolds reversibly that is selected or designed as described herein. Generally, one or more amino acid residues are identified in the amino acid sequence or the polypeptide that unfolds reversibly that prevent irreversible aggregation of the polypeptide upon unfolding. Such amino acid residues are referred to herein as folding gatekeepers. (See Examples, Sections 16-19) Folding gatekeepers can be identified, for example, by aligning the amino acid sequences of two or more polypeptides that unfold reversibly and that are based on a common ammo acid sequence (polypeptide scaffold amino acid sequence) but contain some degree of sequence diversity. Such an alignment can be used to identify amino acid substitutions in the scaffold sequence that are common to polypeptides that unfold reversibly. A polypeptide based on the scaffold sequence but containing one or more the identified amino acid<br>
substitutions can be prepared and assessed for reversible unfolding, to confirm that the identified amino acid residues are folding gatekeepers. Folding gatekeeper residues can also be designed into a desired polypepride, such as a human VH as described herein. (See Examples, Sections 16-19)<br>
The library can comprise a collection of nucleic acids that encode polypeptides that unfold reversibly and/or a collection of polypeptides that unfold reversibly in which eachpolypeptide contains a common folding gatekeeper residue (one or more common folding gatekeeper residues). For example, as described herein, a library comprising a collection of heterogeneous nucleic acids that each encode a human antibody variable domain containing a folding gatekeeper residue can be prepared. If desired, the members of the library can have sequence diversity in a particular region (e.g., hi the CDRs). In certain embodiments, the library comprises a collection of heterogeneous nucleic acids encoding antibody variable domains (e.g., VH (human VH), VL, (human VL)) that contain folding gatekeeper residues. In one embodiment, the library comprises a collection of heterogeneous nucleic acids encoding VMs (human VHS) that contain folding gatekeeper residues in CDR1, CDR1 and CDR2 or CDR2. Preferably, the nucleic acids in such a library encode VHs that contain diverse CDR3s (e.g., randomized CDR3s).<br>
The gatekeeper residues can be designed into CDR1 and/or CDR2 using the methods described herein or other suitable methods, and nucleic acids produced that encode VHS fhat contain those gatekeeper residues. VHS that unfold reversibly can be isolated or selected, using the methods described herein or other suitable method, and nucleic acid(s) encoding CDR1 and/or CDR2 from a VH that unfolds reversibly (the same or different VHS) can be obtained and ligatcd to one or more nucleic acids encoding suitable framework regions and CDRS.<br>
In one embodiment, the library of nucleic acids encodes a polypeptide that unfolds reversibly, wherein each member of the library encodes a polypeptide comprising the amino acid sequence of a parental polypeptide in which at least one amino acid residue is replaced with a folding gatekeeper residue and at least one other amino acid residue is replaced, added or deleted. The parental polypeptide can be an antibody variable domain that comprises framework regions as described herein. In particular embodiments, the parental polypeptide is a human VH and a<br>
folding gatekeeper is introduced into CDR1. In another particular embodiment, the parental polypeptide is a human VL and a folding gatekeeper is introduced into the FR2-CDR2 region.<br>
In another embodiment, the library of nucleic acids encodes an antibody variable domain that unfolds reversibly, wherein each member of the library comprising a first nucleic acid encoding CDR1 and optionally CDR2 of the variable domain or an antibody variable domain that unfolds reversibly, where said first nucleic acid is operably linked to one or more other nucleic acids to produce a construct that encodes antibody variable domains in which CDR1 and optionally CDR2 are encoded by said first nucleic acid. If desired, the members of the library encode an antibody variable domain in which CDR3 is diversified (e,g., at targeted positions) or randomized (e,g,, across the entire CDR3 sequence and/or has CDR3s of varying length).<br>
Libraries that encode a repertoire of a desired type of polypeptides can readily be produced using any suitable method. For example, a nucleic acid sequence that encodes a desired type of polypeptide (e.g., a polymerase, an immunoglobulin variable domain) can be obtained and a collection of nucleic acids that each contain one or more mutations can be prepared, for example by amplifying the nucleic acid using an error-prone polymerase chain reaction (PCR) system, by chemical mutagenesis (Deng etal. J. Biol. Chem., 269:9533 (1994)) or using bacterial mutator strains (Low et al. J. Mol. BioL, 260:359 (1996)).<br>
In other embodiments, particular regions of the nucleic acid can be targeted for diversification. Methods for mutating selected positions are also well known in the art and include, for example, the use of mismatched oligonucleotides or degenerate oligonucleotides, with or without the use of PCR. For example, synthetic antibody libraries have been created by targeting mutations to the antigen binding loops. Random or semi-random antibody H3 and L3 regions have been appended to germline immunoglobulin V gene segments to produce large libraries with unmutated framework regions (Hoogenboom and Winter (1992) supra; Nissim etal. (1994) supra; Griffiths etal. (1994) supra; DeKruif etal. (1995) supra). Such diversification has been extended to include some or all of the other antigen binding loops (Crameri et al (1996) Nature Me&amp;, 2:100; Riechmann et al. (1995)<br>
io/Technology&gt; 13:475; Morphosys, WO 97/08320, supra). In other embodiments, particular regions of the nucleic acid can be targeted for diversification by, for example, a two-step PCR strategy employing the product of the first PCR as a "mega-primer." (See, e.g., Landt, 0. etal, Gene P&amp;125-128 (1990).) Targeted diversification can also be accomplished, for example, by SOE PCR. (See, e.g., Horton, R.M. et al, Gene 77:61-68 (1989).)<br>
As described herein, sequence diversity at selected positions can be achieved by altering the coding sequence which specifies the sequence of the polypeptide such that a number of possible amino acids (e.g., all 20 or a subset thereof) can be incorporated at that position. Using the IUPAC nomenclature, the most versatile codon is NNK, which encodes all amino acids as well as the TAG stop codon. The NNK codon is preferably used in order to introduce the required diversity. Other codons which achieve the same ends are also of use, including the NNN codon, which leads to the production of the additional stop codons TGA and TAA. Such a targeted approach can allow me full sequence space in a target area to be explored.<br>
Preferred libraries of polypeptides that unfold reversibly comprise polypeptides that are members of the immunoglobulin superfkmily (e.g., antibodies or portions thereof). For example the libraries can comprise antibody polypeptides that unfold reversibly and have an known main-chain conformation. (See, e.g,, Tomlinson et al., WO 99/20749.)<br>
Libraries can be prepared hi a suitable plasmid or vector. As used herein, vector refers to a discrete element that is used to introduce heterologous DNA into cells for the expression and/or replication thereof. Any suitable vector can be used, including plasmids (e.g., bacterial plasmids), viral or bacteriophage vectors, artificial chromosomes and episomal vectors. Such vectors may be used for simple cloning and mutagenesis, or an expression vector can be used to drive expression of the library. Vectors and plasmids usually contain one or more cloning sites (e,g., a polylinker), an origin of replication and at least one selectable marker gene. Expression vectors can further contain elements to drive transcription and translation of a polypeptide, such as an enhancer element, promoter, transcription termination signal, signal sequences, and the like. These elements can be arranged in such a way as to be operably linked to a cloned insert encoding a polypeptide, such that the<br>
polypeptide is expressed and produced when such an expression vector is maintained under conditions suitable for expression (e.g., in a suitable host cell).<br>
Cloning and expression vectors generally contain nucleic acid sequences that enable the vector to replicate in one or more selected host cells. Typically in cloning vectors, this sequence is one that enables the vector to replicate independently of the host chromosomal DNA and includes origins of replication or autonomously replicating sequences. Such sequences are well known for a variety of bacteria, yeast and viruses. The origin of replication from the plasmid pBR322 is suitable for most Gram-negative bacteria, the 2 micron plasmid origin is suitable for yeast, and various viral origins (e.g. SV40, adenovirus) are useful for cloning vectors in mammalian cells. Generally, the origin of replication is not needed for mammalian expression vectors unless these are used hi mammalian cells able to replicate high levels of DNA, such as COS cells.<br>
Cloning or expression vectors can contain a selection gene also referred to as selectable marker. Such marker genes encodes a protein necessary for the survival or growth of transformed host cells gown in a selective culture medium. Host cells not transformed with the vector containing the selection gene will therefore not survive in the culture medium. Typical selection genes encode proteins that confer resistance to antibiotics and other toxins, e.g. ampicillin, neomycin, methotrexate or tetracycline, complement auxotrophic deficiencies, or supply critical nutrients not available in the growth media.<br>
Suitable expression vectors can contain a number of components, for example, an origin of replication, a selectable marker gene, one or more expression control elements, such as a transcription control element (e.g., promoter, enhancer, terminator) and/or one or more translation signals, a signal sequence or leader sequence, and the like. Expression control elements and a signal or leader sequence, if present, can be provided by the vector or other source. For example, the transcriptional and/or translational control sequences of a cloned nucleic acid encoding an antibody chain can be used to direct expression.<br>
A promoter can be provided for expression in a desired host cell. Promoters can be constitutive or inducible. For example, a promoter can be operably linked to a nucleic acid encoding an antibody, antibody chain or portion thereof, such that it<br>
directs transcription of the nucleic acid. A variety of suitable promoters for procaryotic (e.g., the ß-lactamase and lactose promoter systems, alkaline phosphatase, the tryptophan (trp) promoter system, lac, tac, T3, T7 promoters for E. coif) and eucaryotic (e.g., simian virus 40 early or late promoter, Rmis sarcoma virus long terminal repeat promoter, cytoniegalovirus promoter, adenovirus late promoter, EG-1 a promoter) hosts are available.<br>
In addition, expression vectors typically comprise a selectable marker for selection of host cells carrying the vector, and, in the case of a replicable expression vector, an origin or replication. Genes encoding products which confer antibiotic or drug resistance are common selectable markers and may be used in procaryotic (e.g., ß-lactamase gene (ampicillin resistance), Tet gene for tetracycline resistance) and eucaryotic cells (e.g., neomycin (G418 or geneticin), gpt (mycophenolic acid), ampicillin, or hygromycin resistance genes). Dihydrofolate reductase marker genes permit selection with methotrexate in a variety of hosts. Genes encoding the gene product of auxotrophic markers of the host (e.g., LEU2, URA3, HISS) are often used as selectable markers in yeast. Use of viral (e.g., baculovirus) or phage vectors, and vectors which are capable of integrating into the genome of the host cell, such as retroviral vectors, are also contemplated.<br>
Suitable expression vectors for expression in prokaryotic (e.g., bacterial cells such as E. coli) or mammalian cells include, for example, a pET vector (e.g., pET-12a, pET-36, pET-37, pET-39, pET-40, Novagen and others), a phage vector (e,g., pCANTAB 5 E, Pharmacia), pRTT2T (Protein A fusion vector, Pharmacia), pCDMS, pCDNAl.I/amp, pcDNAS.l, pRc/RSV, pEF-1 (ravitrogen, Carlsbad, CA), pCMV-SCRDPT, pFB, pSG5, pXTl (Stratagene, La Jolla, CA), pCDEF3 (Goldman, L.A., et al, Biotechniques, 27:1013-1015 (1996)), pSVSPORT (GibcoBRL, Rockville, MD), pEF-Bos (Mizushima, S., et al., Nucleic Acids Res., 18:5322 (1990)) and the like. Expression vectors which are suitable for use in various expression hosts, such as prokaryotic cells (E. coif), insect cells (Drosophila Schnieder S2 cells, Sf9) and yeast (P. methcmolica, P. pastoris, S. cerevisiae) and mammalian cells (eg, COS cells) are available.<br>
Preferred vectors are expression vectors that enable the expression of a nucleotide sequence corresponding to a polypeptide library member. Thus, selection<br>
with generic and/or target ligands can be performed by separate propagation and expression of a single clone expressing the polypeptide library member. As described above, the preferred selection display system is bacteriophage display. Thus, phage or phagemid vectors may be used. The preferred vectors are phagemid vectors which have an E, coll, origin of replication (for double stranded replication) and also a phage origin of replication (for production of single-stranded DNA). The manipulation and expression of such vectors is well known in the art (Hoogenboom and Whiter (1992) supra; Nissim et al (1994) supra). Briefly, the vector can contain a /3-lactamase gene to confer selectivity on the phagemid and a lac promoter upstream of an expression cassette mat can contain a suitable leader sequence (e.g., pelB leader sequence), a multiple cloning site, one or more peptide tags, one or more TAG stop codons and the phage protein pill. Thus, using various suppressor and non-suppressor strains of E. coli and with the addition of glucose, iso-propyl thio-/3-D-galactoside (DPTG) or a helper phage, such as VCS M13, the vector is able to replicate as a plasmid with no expression, produce large quantities of the polypeptide library member only or product phage, some of which contain at least one copy of the polypeptide-pIH fusion on their surface.<br>
The libraries and repertoires of the invention can contain antibody formats. For example, the polypeptide contained within the libraries and repertoires can be whole antibodies or fragments therefore, such as Fab, F(ab')z, Fv or scFv fragments, separate VH or VL domains, any of which is either modified or unmodified. scFv fragments, as well as other antibody polypeptides, can be readily produced using any suitable method. A number of suitable antibody engineering methods are well<br>
known in the art. For example, a scFv can be formed by linking nucleic acids<br>
encoding two variable domains with an suitable oligonucleotide that encodes an<br>
appropriately linker peptide, such as (Gly-Gly-Gly-Gly-Ser)3 or other suitable linker peptide(s). The linker bridges the C-terminal end of the first V region and N-terminal end of the second V region. Similar techniques for the construction of other antibody formats, such as Fv, Fab and F(ab')z fragments. To format Fab and F(ab )z fragments, VH and VL polypeptides can combined with constant region segments, which may be isolated from rearranged genes, germline C genes or<br>
synthesized from antibody sequence data. A library or repertoire according to the invention can be a VH or VL, library or repertoire.<br>
In particular embodiments, the libraries and repertoires comprise imrrmnoglobulin variable domains that unfold reversibly (e.g., VH, VL). The variable domains can be based on a gennline sequence (e,g., DP47durnmy (SEQ ID NO:3, DPK9 dummy (SEQ ID N0:6)) and if desired can have one or more diversified regions, such as the complementarity determining regions.<br>
One or more of the framework regions (FR) of the variable domains can comprise (a) the amino acid sequence of a human framework region, (b) at least 8 contiguous amino acids of the amino acid sequence of a human framework region, or (c) an amino acid sequence encoded by a human gerraline antibody gene segment, wherein said framework regions are as defined by Kabat. In certain embodiments, the amino acid sequence of one or more of the framework regions is the same as the amino acid sequence of a corresponding framework region encoded by a human gerraline antibody gene segment, or the amino acid sequences of one or more of said framework regions collectively comprise up to 5 amino acid differences relative to the amino acid sequence of said corresponding framework region encoded by a human gennline antibody gene segment<br>
In other embodiments, the amino acid sequences of FR1, FR2, FR3 and FR4 are the same as the ammo acid sequences of corresponding framework regions encoded by a human gennline antibody gene segment, or the amino acid sequences of FR1, FR2, FR3 and FR4 collectively contain up to 10 amino acid differences relative to the amino acid sequences of corresponding framework regions encoded by said human gennline antibody gene segments. In other embodiments, the ammo acid sequence of said FR1, FR2 and FR3 are the same as the amino acid sequences of corresponding framework regions encoded by said human gennline antibody gene segment.<br>
The polypeptides comprising a variable domain that unfolds reversibly preferably comprise a target ligand binding site and/or a generic ligand binding site. In certain embodiments, the generic ligand binding site is a binding site for a superantigen, such as protein A, protein L or protein G.<br>
The variable domains can be based on any desired variable domain, for example a human VH (e.g., VH 1 a, VH lb, VH 2, VH 3, VH 4, VH 5, VH 6), a human V (e.g., V , VII, VIII, VIV, VV or VVI) or a human VK (e.g., VK!, Vx2, VK3, VK4, VK5, VK6, VK7, VK8, VK9 or VK10). Preferably, the variable domains are not a Camelid immunoglobulin domain, such as a VH H, or contain one or more amino acids (e.g., framework amino acids) that are unique to Camelid immunoglobulin variable domains encoded by germline sequences but not, for example, to human immunoglobulin variable domains. (See, e.g., Davies et aL, Protein Engineering 9:531-537 (1996); Tariha et al, J. Biol. Chem. 276:24774-24780 (2001); Riechmann et al., J. Immunol Methods 23:25-38 (1999).) In one embodiment, the VH that unfolds reversibly does not comprise one or more amino acids mat are unique to murine (e.g., mouse) germline framework regions. Preferably, the variable domain is unfold reversibly when heated and cooled.<br>
The isolated polypeptide comprising a variable domain can be an antibody format. Thus, in certain embodiments, the isolated polypeptide comprising a variable domain that unfolds reversibly can be a homodimer of variable domain, a heterodimer comprising a variable domain, an Fv, a scFv, a disulfide bonded Fv, a Fab, a single variable domain or a variable domain fused to an immunoglobulin Fc portion.<br>
POLYPEPTIDES<br>
In one aspect, the invention is an isolated polypeptide that unfolds reversibly. As described herein, such polypeptides can be expressed in E. coll and recovered in high yield. As described herein, in preferred embodiments the polypeptides that unfold reversibly (e,g., when heated) are secreted when expressed in E. coli, and easily recovered as soluble polypeptides. Such polypeptides are also referred to as secretable when expressed hi E. coli. In preferred embodiments, the polypeptide that unfolds reversibly is secreted in a quantity of at least about 0.5 mg/L when expressed in E. coli. In other preferred embodiments, the polypeptide that unfolds reversibly is secreted in a quantity of at least about 0.75 mg/L, at least about 1 mg/L, at least about 4 mg/L, at least about 5 mg/L, at least about 10 mg/L, at least about 15 mg/L, at least about 20 mg/L, at least about 25 mg/L, at least about 30 mg/L, at least<br>
about 35 rag/L, at least about 40 mg/L, at least about 45 mg/L, or at least about 50 mg/L, or at least about 100 mg/L, or at least about 200 mg/L, or at least about 300 mg/L, or at least about 400 mg/L, or at least about 500 mg/L, or at least about 600 mg/L, or at least about 700 mg/L, or at least about 800 mg/L, at least about 900 mg/L, at least about 1 g/L when expressed in E. coli.  In other preferred embodiments, the polypeptide that unfolds reversibly is secreted in a quantity of at least about 1 mg/L to at least about Ig/L, at least about 1 mg/L to at least about 750 mg/L, at least about 100 mg/L to at least about 1 g/L, at least about 200 mg/L to at least about 1 g/L, at least about 300 mg/L to at least about 1 g/L, at least about 400 mg/L to at least about 1 g/L, at least about 500 mg/L to at least about Ig/L, at least about 600 mg/L to at least about 1 g/L, at least about 700 mg/L to at least about 1 g/L, at least about 800 mg/L to at least about Ig/L, or at least about 900 mg/L to at least about Ig/L when expressed in E. coli. Although, the polypeptides described herein can be secretable when expressed in E. coli, they can be produced using any suitable method, such as synthetic chemical methods or biological production methods mat do not employ E. coli. hi particularly preferred embodiments, the polypeptide that unfolds reversibly is a human antibody variable domain (VH, VL.) or comprises a human antibody variable domain mat unfolds reversibly.<br>
In some embodiments, the polypeptide that unfolds reversibly is a variant of a parental polypeptide that differs from the parental polypeptide in amino acid sequence (e.g., by one or more amino acid replacements, additions and/or deletions), but qualitatively retains function of the parental polypeptide. Preferably, the activity of the variant polypeptide that unfolds reversibly is at least about 25%, at least about 30%, at least about 40%, at least about 50%, at least about 60%, at least about 70%, at least about 80%, at least about 90%, at least about 95%, or essentially the same as of the activity of the parental polypeptide. For example, if the parental polypeptide is an enzyme, the variant polypeptide can contain an amino acid sequence that differs from the parental enzyme (e.g., by one to about ten amino acid substitutions, deletions and/or insertions) but will retain the catalytic activity of parental enzyme. Preferably, the variant enzyme that unfolds reversibly is characterized by a catalytic rate constant that is at least about 25% of the catalytic rate constant of the parental enzyme.<br>
A variant polypeptide that unfolds reversibly can be prepared using any desired parental polypeptide. For example, the polypeptide that unfolds reversibly can be prepared using a parental polypeptide that is based on a scaffold selected from an enzyme, protein A, protein L, a fibronectm, an anticalin, a domain of CTLA4, or a polypeptide of the immunoglobulin superfamily, such as an antibody or an antibody fragment.<br>
The parental polypeptide can comprise V and/or D (where the polypeptide comprises a VH domain) and/or J segment sequences encoded by a germline V, D or I segment respectively, or a sequence that results from naturally occurring or artificial mutations (e.g., somatic mutation or other processes). For example, the parental polypeptide can be a parental immunoglobulm variable domain having an amino acid sequence that is encoded by germline gene segments, or that results from insertion and/or deletion of nucleotides during V(D)J recombination (e.g., N nucleotides, F nucleotides) and/or mutations mat arise during affinity maturation. Similarly, (he parental polypeptide can be a parental enzyme that has an amino acid sequence encoded by the germline or that results from naturally occurring or artificial mutation (e.g, somatic mutation).<br>
A variant polypeptide that unfolds reversibly can be prepared using any suitable method. For example, the amino acid sequence of a parental polypeptide can be altered at particular positions (as described herein with respect to antibody variable domain polypeptides) to produce a variant polypeptide that unfolds reversibly. A variant polypeptide that unfolds reversibly can also be produced, for example, by providing a nucleic acid that encodes a parental polypeptide, preparing a library of nucleic acids that encode variants of the parental polypeptide (e.g., by error prone PCR or other suitable method) and expressing the library in a suitable polypeptide display system. A variant polypeptide that unfolds reversibly and retains a desired function of the parental polypeptide can be selected and isolated from such a polypeptide display system using the methods described herein or other suitable methods.<br>
Polypeptides Comprising V Domains That Unfold Reversibly<br>
In preferred embodiments, the isolated polypeptide comprises an immunoglobulin variable domain that unfolds reversibly (e.g., VH, a variant VH, VL and/or variant VL)- in certain embodiment, the isolated polypeptide comprises a variant VH and/or a variant VL that unfolds reversibly. Preferably the variable domain that unfolds reversibly (e.g., VH, variant VH, VL, variant VL) has binding specificity for a target ligand and binds to the target ligand with a suitable dissociation constant (Kj) and suitable off rate (Koff)- A suitable Kd can be about 50 nM to about 20 pM or less, for example about 50 nM, about 1 nM, about 900 pM, about 800 pM, about 700 pM, about 600 pM, about 500 pM, about 400 pM, about 300 pM, about 200 pM, about 100 pM or about 20 pM or less. A suitable Kotr can be about 1 x 10"1 s"1 to about 1 x 10"7 s"1 or less, for example about 1 x 10"1 s"1, about 1 x 10"2 s"1, about 1 x 10"3 s"1, about 1 x 104 s'1, about 1 x 10'5 s'1, about 1 x 10"6 s'1 or about 1 x 10"1 s"1 or less. Preferably, Kj and K
In certain embodiments, the isolated polypeptide comprises a VH (eg., variant VH) that unfolds reversibly. In some embodiments, the amino acid sequence of the variant VH that unfolds reversibly differs from the amino acid sequence of the parental VH by at least one amino acid from position 22 to position 36, or differs from the amino acid sequence of the parental VH by at least one amino acid in the HI loop. The amino acid positions and CDR (HI, H2 and H3) and framework regions (FR1, FR2, FR3 and FR4) of the VH can be defined using any suitable system, such as the systems of Kabat, Chothia or AbM. Preferably, positions 22 through 36 are defined according to the amino acid numbering system of Kabat, and the HI loop is defined according to the amino acid numbering system used in the AbM software package (antibody analysis and structural modeling software; Oxford Molecular). The amino acid sequence of the variant VH can contain at least one amino acid replacement from position 22 to position 36 relative to the parental VH.<br>
In certain embodiments, the amino acid sequence of the VH that unfolds reversibly contains at least one Pro or Gly residue from position 22 to position 36. In other embodiments, the amino acid sequence of the VH that unfolds reversibly contains at least one Pro or Gly residue in the HI loop. In certain embodiments, the amino acid sequence of the variant VH that unfolds reversibly contains at least one<br>
Pro or Gly replacement from position 22 to position 36 relative to the amino acid sequence of the parental VH. In other embodiments, the amino acid sequence of the variant VH that unfolds reversibly contains at least one Pro or Gly replacement in the HI loop relative to the amino acid sequence of the parental VH. In a particular embodiment, the variant VH that unfolds reversibly comprises an amino acid sequence wherein the parental amino acid residue at position 29 is replaced with Pro or Gly.<br>
In other embodiments, the variant VH that unfolds reversibly contains at least one amino acid replacement from position 22 to position 36 relative to the parental VH amino acid sequence, such that the hydrophobicity of the amino acid sequence from position 22 to position 36 of the variant VH is reduced relative to the parental Vu. Hydrophobicity can be determined using any suitable scale or algorithm. Preferably, hydrophobicity is determined using the method of Sweet and Eisenberg, and the Sweet and Eisenberg hydrophobicity score (S/E score) of the amino acid sequence from position 22 to position 36 of the variant VH is reduced relative to the parental VH- The S/E method can use a 9 to 18 amino acid window (e.g., a window of 9,10,11,12,13,14,15,16,17 or 18 amino acids). Preferably, a 15 amino acid window is used for VH.<br>
In certain embodiments the S/E score of the amino acid sequence from position 22 to position 36 of the variant VH is 0.15 or less, or 0.1 or less for an S/E score rounded to one decimal place, or 0.09 or less, or 0.08 or less, or 0.07 or less, or 0.06 or less, or 0.05 or less, or 0.04 or less. In preferred embodiments, the S/E score or the amino acid sequence from position 22 to position 36 of the variant VH is 0.03 or less, 0.02 or less or 0.01 or less. In more preferred embodiments, the S/E score or the amino acid sequence from position 22 to position 36 of the variant VH is 0 or less.<br>
Producing VH variants that have a reduced S/E score from position 22 to position 36 relative to a parental VH can yield polypeptides that have several advantages. For example, it has been determined that a lower S/E score from position 22 to position 36 correlates with resistance to aggregation and enhanced yields from expression systems (e.g., bacterial expression systems). The capacity of VH domains to resist aggregation at high protein concentrations is associated with a<br>
low S/E score. Thus, a variant VH domain with an S/E score from position 22 to position 36 that is lower than the S/E score of the parental VH domain can display enhanced resistance to aggregation, while lowering the S/E score of this region to 0 or less can produce variant VH domains with superior aggregation resistance at high protein concentrations (e,g., about 200 µM).<br>
In other embodiments, the VH that unfolds reversibly contains an amino acid sequence from position 22 to position 36 that has an S/E score of 0.15 or less, or 0.1 or less, or 0.09 or less, or 0.08 or less, or 0.07 or less, or 0.06 or less, or 0.05 or less, or 0.04 or less. In preferred embodiments, me VH that unfolds teversibly contains an amino acid sequence from position 22 to position 36 that has an S/E score of 0,03 or less, 0.02 or less or 0.01 or less. In more preferred embodiments, the VH that unfolds reversibly contains an amino acid sequence from position 22 to position 36 that has an S/E score of 0 or less.<br>
In another embodiment, the amino acid sequence of the variant VH differs from the amino acid sequence of the parental VH by at least one amino acid in the HI loop. Preferably, the variant VH contains at least one amino acid replacement in the HI loop, such that the hydrophobicity of HI loop is reduced relative to the HI loop of the parental VH. Preferably, the S/E score of the HI loop of the variant VH is 0 or less.<br>
In other embodiments, the VH that unfolds reversibly contains an HI loop mat has an S/E score of 0 or less.<br>
In particular embodiments, the VH that unfolds reversibly (e.g., variant VH) comprises an amino acid sequence in which one or more of the parental amino acid residues at position 27,29,30,31,32,33 and 35 (Kabat numbering) is replaced with another amino acid residue as set forth in Table 1.   <br>
(Table Removed)<br>
The amino acid sequence of the VH that unfolds reversibly (e.g., variant VH) can comprise any one or any combination of amino acid replacements set forth in Table 1, and if desired, a variant VH that unfolds reversibly can further differ from the parental VH by replacement of one or more of the amino acid residues at parental position 22 through parental position 26, parental position 28 and parental position 36 (Kabat numbering).<br>
In more particular embodiments, the variant VH that unfolds reversibly comprises an amino acid sequence in which the parental amino acid residue at position 27 is replaced with Asp or Glu; the parental amino acid residue at position 29 is replaced with Asp, Glu, Pro or Gly; and/or the parental amino acid residue at position 32 is replaced with Asp or Glu (Kabat numbering). In one such embodiment, the variant VH that unfolds reversibly comprises an amino acid sequence in which the parental amino acid residue at position 27 is replaced with Asp; the parental amino acid residue at position 29 is replaced with Asp, Pro or Gly; and/or the parental amino acid residue at position 32 is replaced with Glu (Kabat numbering).<br>
In a specific embodiment, the variant VH that unfolds reversibly comprises an amino acid sequence in which the parental amino acid residue at position 27 is replaced with Asp; the parental amino acid residue at position 29 is replaced with Val; and/or the parental amino acid residue at position 32 is replaced with Asp or Glu; and the parental residue at position 35 is replaced with Gly (Kabat numbering).<br>
In preferred embodiments, the variant VH that unfolds reversibly comprises an amino acid sequence in which the parental amino acid residue at position 27 is replaced with Asp and/or the parental amino acid residue at position 32 is replaced with Asp (Kabat numbering). If desired, the variant VH that unfolds reversibly can further comprise amino acid replacements relative to the parental sequence at one or more of positions 22-26,28-31 and 33-36 as described herein (Kabat numbering),e.g,, to bring the S/E score of one or more of these legions to zero or less.<br>
One or more of the framework regions (FR) of the VH or variant VH that unfold reversibly can comprise (a) the amino acid sequence of a human framework region, (b) at least 8 contiguous amino acids of the amino acid sequence of a human framework region, or (c) an amino acid sequence encoded by a human germline antibody gene segment, wherein said framework regions are as defined by Kabat. In certain embodiments, the amino acid sequence of one or more of the framework regions is the same as the amino acid sequence of a corresponding framework region encoded by a human germline antibody gene segment, or the amino acid sequences of one or more of said framework regions collectively comprise up to 5 amino acid differences relative to the amino acid sequence of said corresponding framework region encoded by a human germline antibody gene segment.<br>
In other embodiments, the amino acid sequences of FR1, FR2, FR3 andFR4 are the same as the amino acid sequences of corresponding framework regions encoded by a human germline antibody gene segment, or the amino acid sequences of FR1, FR2, FR3 and FR4 collectively contain up to 10 amino acid differences relative to the amino acid sequences of corresponding framework regions encoded by said human germline antibody gene segments. In other embodiments, the amino acid sequence of said FR1, FR2 and FR3 are the same as the amino acid sequences of corresponding framework regions encoded by said human germline antibody gene segment For example, the variant VL can be a variant of human DP47 dummy (SEQIDNO:3).<br>
The isolated polypeptide comprising a VH that unfolds reversibly (e.g., variant VH) comprises a target ligand binding site and/or a generic ligand binding<br>
site. In certain embodiments, the generic ligand binding site is a binding site for a superantigen, such as protein A, protein L or protein G.<br>
The VH that unfolds reversibly can be based on any desired parental VH, for example a human VH (e.g., VH la, VH lb, VH 2, VH 3, VH 4, VH 5, VH 6). Preferably, the VH that unfolds reversibly is not a Camelid immunoglobulin domain, such as a VwH, or does not contain one or more amino acids (e.g., framework amino acids) that are unique to Camelid immunoglobulin variable domains encoded by germline sequences but not, for example, to human immunoglobulin variable domains. (See,e.&amp;, Varies etal., Protein Engineering 9:531-537 (1996); Tanha et al, J. Biol. Chem. 276:24774-24780 (2001); Riechmann et al, J. Immunol. Methods 23:25-38 (1999).) In one embodiment, the VH mat unfolds reversibly does not comprise one or more amino acids that are unique to murine (e.g., mouse) germline framework regions. Preferably, the VH unfolds reversibly when heated and cooled.<br>
The isolated polypeptide comprising a VH that unfolds reversibly (e.g. variant VH) can be an antibody format. Thus, in certain embodiments, the isolated polypeptide comprising a VH that unfolds reversibly can he a homodimer of VH, a heterodimer comprising a VH, an Fv, a scFv, a disulfide bonded Fv, a Fab, a VH or a VH fused to an immunoglobulin Fc portion.<br>
The invention also relates to an isolated polypeptide comprising an immunoglobulin light chain variable domain (VL) (e.g., a variant VL) that unfolds reversibly. In one embodiment, the isolated polypeptide comprises a VL that unfolds reversibly and comprises an amino acid sequence from position 44 to position S3 that has a Sweet/Eisenberg hydrophobicity score (S/E score) of less than 0.23. In other embodiments, the S/E score for position 44 to position S3 can be less than 0.2, less than 0.17, less than 0.15, less than 0.13, less than 0.10, or less than -0.1 or less. The S/E method can use a 9 to 18 amino acid window (e,g., a window of 9,10,11, 12,13,14,15,16,17 or 18 amino acids). Preferably, an 11 amino acid window is used for VL. The amino acid positions and CDR, (LI, L2 and L3) and framework regions (FRl, FR2, FR3 and FR4) of the VL can be defined using any suitable system, such as the systems of Rabat, Chothia or AbM. Preferably, the amino acid positions and CDR and framework regions are according to the amino acid numbering system of Kabat. In other embodiments, VL that unfolds reversibly<br>
further comprises a FR3 that has a Sweet/Eiseriberg hydrophobicity score (S/E score) of less than 0.35, and FR3 is as defined by the Kabat amino acid numbering system. In other embodiments, the S/E score for FR3 can be less than 0.3, less than 0.25, less than 0.2, less than 0.17, less thanO.15, less than 0,13, less than 0.10, or less man -0.1 or less.<br>
In certain embodiments, the isolated polypeptide comprises a variant VL that unfolds reversibly. In some embodiments, the amino acid sequence of the variant VL that unfolds reversibly differs from the amino acid sequence of the parental VL by at least one amino acid from position 44 to position 53 of said variant VL, such that the variant VL comprises the amino acid sequence of a parental VL wherein at least one amino acid residue from position 44 to position 53 is replaced such that the Sweet/Eisenberg hydrophobicity score (S/E score) of the amino acid sequence from position 44 to position 53 of said variant VL is less than 0.23. The amino acid sequence of the variant VL that unfolds reversibly can contain at least one amino acid replacement from position 44 to position 53 relative to the parental VL. In other embodiments, the amino acid sequence of the variant VL that unfolds reversibly further differs from the amino acid sequence of the parental VL by at least one amino acid in FR3 of said variant VL- In these embodiments, the variant VL that unfolds reversibly comprises FR3 having the amino acid sequence of a parental VL FR3 wherein at least one amino acid residue is replaced such that the Sweet/Eisenberg hydrophobicity score (S/E score) of FR3 said variant VL is less than 0.35.<br>
In other embodiments, the amino acid sequence of the variant Vtmat unfolds reversibly differs from the amino acid sequence of the parental VL by at least one amino acid in FR3 of said variant VL, such that the variant VL comprises the amino acid sequence of a parental VL wherein at least one amino acid residue in FR3 is replaced such that the Sweet/Eisenberg hydrophobiciry score (S/E score) of the amino acid sequence of FRe of said variant VL is less than 0.35. For example, the amino acid sequence of the variant VL that unfolds reversibly can differ from the .ammo acid sequence of the parental VL by at least one amino acid from position 73 to position 76 of said variant VL, such that the variant VL comprises the amino acid sequence of a parental VL wherein at least one amino acid residue from position 73<br>
to position 76 is replaced such that the Sweet/Eiseriberg hydrophobicity score (S/E score) of the amino acid sequence of FR3 of said variant VL is less fhan 0.35.<br>
In particular embodiments, the VL that unfolds reversibly comprises an amino acid sequence in which one or more of the amino acid residues at position 10, 13,20,23,26,27,29,31,32,35, 36, 39,40,42,45,46,47,48,49, 50,57,59, 60, 68,75,79,80, 83, 89,90 and 92 (Kabat numbering) is replaced with another amino acid residue as set forth in Table 2. For example, when the isolated VL that unfolds reversibly is a variant Vt, it can comprise an amino acid sequence in which one or more of the parental amino acid residues at position 10,13,20,23,26,27,29,31, 32,35,36,39,40,42,45,46,47,48,49, 50,57,59,60,68,75, 79, 80, 83, 89, 90 and 92 (Kabat numbering) is replaced with another amino acid residue as set forth in <br>
Table 2.   (Table Removed)   <br>
The amino acid sequence of the VL or variant VL that unfolds reversibly can comprise any one or any combination of amino acid replacements set forth in Table 2, and if desired can further differ from the parental VL by replacement of the amino acid residue at parental position 26 with Asn and/or replacement of the ammo acid residue at parental position 89 with Arg (Kabat numbering).<br>
In more particular embodiments, the variant VL that unfolds reversibly comprises an amino acid sequence in which the parental amino acid residue at position 45 is replaced with Glu, Asp, Gin, Pro, Asn, His or Thr, the parental amino acid residue at position 48 is replaced with Asn, Pro, Asp, Thr, Gly or Val; the parental amino acid residue at position 49 is replaced with Asn, Asp, Ser, Cys, Glu, Gly, Lys. or Arg; the parental amino acid residue at position 50 is replaced with Pro, Asp, Asn, Glu or Arg; and/or the parental amino acid residue at position 75 is replaced with Asn or Met. (Kabat numbering).<br>
In other particular embodiments, the VL or variant VL that unfolds reversibly comprises an amino acid sequence which contain one of the amino acid replacements set forth in Table 2 and has an amino acid sequences that comprises:<br>
Gly at position 31 and Asn at position 49;<br>
Ser at position 32 and Asn at position 75;<br>
Ser at position 40 and Asp at position 49;<br>
Arg at position 39 and Asn at position 49;<br>
Glu at position 45 and Asn at position 75;<br>
Pro at position 46 and Asp at position 50;<br>
Asn at position 26, Thr at position 42 and Asp at position SO;<br>
Phe at position 32, Glu at position 45 and Glu at position 57;<br>
Asp at position 49, Ala at position 80 and Arg at position 89;<br>
Asn at position 49, Glu at position 68 and Arg at position 79;<br>
Ser at position 20, Tip at position 23, Phe at position 46 and Asn at position 49;<br>
Val at position 29, Asn at position 42, Glu at position 45, Leu at position 83 and His at position 92;<br>
Thr at position 3 5 and Pro at position 90;<br>
Asp at position 45 and Pro at position .60;<br>
Arg at position 49 and Phe at position 10;<br>
Ser at position 49 and Ala at position 20;<br>
Ser at position 49 and Arg at position 27;<br>
Pro at position 50 and Val at position 48; and<br>
Arg at position 50, Gly at position 13 and Glu at position 42, wherein amino acid positions are by Kabat numbering.<br>
In certain embodiments, the amino acid sequence of the variant VL that unfolds reversibly contains at least one Pro or Gly replacement from position 44 to position 53 and/or in FR3 (e.g., from position 73 to position 76) relative to the amino acid sequence of the parental VL. In a particular embodiment, the amino acid sequence of the variant VL comprises Pro at position 45, position 48 and/or position 50,<br>
One or more of the framework regions (FR) of the VL or variant VL that unfolds reversibly can comprise (a) the amino acid sequence of a human framework region, (b) at least 8 contiguous amino acids of the amino acid sequence of a human framework region, or (c) an amino acid sequence encoded by a human germline antibody gene segment, wherein said framework regions are as defined by Kabat In certain embodiments, the amino acid sequence of one or more of the framework regions is the same as the amino acid sequence of a corresponding framework region encoded by a human germline antibody gene segment, or the amino acid sequences of one or more of said framework regions collectively comprise up to 5 amino acid differences relative to the amino acid sequence of said corresponding framework region encoded by a human germline antibody gene segment.<br>
In other embodiments, the amino acid sequences of FRl, FR2, FR3 and FR4 are the same as the amino acid sequences of corresponding framework regions encoded by a human germline antibody gene segment, or the amino acid sequences<br>
of FR1, FR2, FR3 and FR4 collectively contain up to 10 amino acid differences relative to the amino acid sequences of corresponding framework regions encoded by said human germline antibody gene segments. In other embodiments, the amino acid sequence of said FR1, FR2 and FR3 are the same as the amino acid sequences of corresponding framework regions encoded by said human germline antibody gene segment. For example, the variant VL can a variant of human DPK9 dummy (SEQ IDNO:6).<br>
The isolated polypeptide comprising a VL that unfolds reversibly (e.g., variant VL) comprises a target ligand binding site and/or a generic ligand binding site. In certain embodiments, the generic ligand binding site is a binding site for a superantigen, such as protein A, protein L or protein G.<br>
The VL that unfolds reversibly can be based on any desired parental VL, for example a human VX (e.g., VU, VAJI, VXHI, WIV, VW or VXVI) or a human VK (e.g., VKl, VK2, Vic3, VK4, VK5, Vx6, VK?, VicS, Vic9 orVKlO). Preferably, me VL that unfolds reversibly is not a Ca/ne/iWimmunoglobulin domain, or contain one or more amino acids (e.g., framework amino acids) mat are unique to Camelid immunoglobulin variable domains encoded by germline sequences but not, for example, to human immunoglobulin variable domains. (See, e.g., Davies et al., Protein Engineering 9:531-537 (1996); Tanha et al, J. Biol. Ghent. 276:24774-24780 (2001); Riechmann et al, J. Immunol. Methods 23:25-38 (1999).) In one embodiment, the VH that unfolds reversibly does not comprise one or more amino acids that are unique to murine (e.g., mouse) germline framework regions. Preferably, the VL unfolds reversibly when heated.<br>
The isolated polypeptide comprising a VL that unfolds reversibly (e.g., variant VL) can be an antibody format. Thus, in certain embodiments, the isolated polypeptide comprising a VL that unfolds reversibly can be a homodimer of VL, a heterodimer comprising a VL, an Fv, a scFv, a disulfide bonded Fv, a Fab, a VL or a VL fused to an immunoglobulin Fc portion.<br>
Isolated Polypeptides Comprising Disulfide Bonded V Domains<br>
The invention also relates to an isolated polypeptide comprising an immunoglobulin variable domain (e.g, VH, VL), wherein said variable domain<br>
comprises a disulfide bond between a cysteine in CDR2 and a cysteine in CDR3, wherein CDR2 and CDR3 are defined using the Kabat ambo acid numbering system. In some embodiments, the disulfide bond is between cysteine residues at position 52a and position 98; position 51 and position 98; orposition 51 and position l00b, wherein said positions are defined using the Kabat amino acid numbering system. In other embodiments, the disulfide bond is between a cysteine at position 51 and a cysteine in CDR3; or a cysteine in CDR2 and a cysteine at position 98, wherein CDR2, CDR3 and said positions are defined using the Kabat amino acid numbering system.<br>
The isolated polypeptide comprising a disulfide bonded variable domain can be an antibody format. Thus, in certain embodiments, the isolated polypeptide comprising a disulfide bonded variable domain can be a homodiraer of variable domains, a heterodimer of variable domains, anFv, a scFv, a Fab, a single variable domain or a single variable domain fused to an immunoglobulin Fc portion.<br>
NUCLEIC ACIDS, HOST CELLS AND METHODS FOR PRODUCING REFOLDABLE POLYPEPTIDES<br>
The invention also relates to isolated and/or recombinant nucleic acids encoding polypeptides that unfold reversibly as described herein.<br>
Nucleic acids referred to herein as "isolated" are nucleic acids which have been separated away from other material (e.g., other nucleic acids such as genomic DNA, cDNA and/or RNA) in its original environment (e.g., in cells or in a mixture of nucleic acids such as a library). An isolated nucleic acid can be isolated as part of a vector (e.g., a plasmid). Nucleic acids can be naturally occurring, produced by chemical synthesis, by combinations of biological and chemical methods (e.g., semisynthetic), and be isolated using any suitable methods.<br>
Nucleic acids referred to herein as "recombinant" are nucleic acids which have been produced by tecombinant DNA methodology, including methods which rely upon artificial recombination, such as cloning into a vector or chromosome using, for example, restriction enzymes, homologous recombination, viruses and the like, and nucleic acids prepared using the polymerase chain reaction (PCR). "Recombinanf' nucleic acids are also those that result from recombination of<br>
endogenous or exogenous nucleic acids through the natural mechanisms of cells 01 cells modified to allow recombination (e.g., cells modified to express Cre or other suitable recombinase), but are selected for after the introduction to the cells of nucleic acids designed to allow and make recombination probable. For example, a functionally rearranged human-antibody transgene is a recombinant nucleic acid.<br>
Nucleic acid molecules of the present invention can be used in the production of antibodies (e.g., human antibodies, humanized antibodies, chimeric antibodies and antigen-binding fragments of the foregoing), e.g., antibodies that bind an oE integral or integrin aE chain (CD103). For example, a nucleic acid (e.g., DNA) encoding an antibody of the invention can be incorporated into a suitable construct (e.g., an expression vector) for further manipulation or for production of the encoded polypeptide in suitable host cells.<br>
Expression constructs or expression vectors suitable for the expression of a antibody or antigen-binding fragment are also provided. For example, a nucleic acid encoding all or part of a desired antibody can be inserted into a nucleic acid vector, such as a plasmid or virus, for expression. The vector can be capable of replication in a suitable biological system (e.g., a replicon). A variety of suitable vectors are known in the art, including vectors which are maintained in single copy or multiple copy, or which become integrated into the host cell chromosome. Suitable expression vectors can contain a number of components, for example, an origin of replication, a selectable marker gene, one or more expression control elements, such as a transcription control element (e.g., promoter, enhancer, terminator) and/or one or more translation signals, a signal sequence or leader sequence, and the like. Expression control elements and a signal or leader sequence, if present, can be provided by the vector or other source. For example, the transcriptional and/or translational control sequences of a cloned nucleic acid encoding an antibody chain can be used to direct expression.<br>
In another aspect, the invention relates to recombinant host cells and a method of preparing an polypeptide of the invention that unfolds reversibly. The polypeptide that unfolds reversibly can be obtained, for example, by the expression of one or more recombinant nucleic acids encoding a polypeptide that unfolds reversibly or using other suitable methods. For example, the expression constructs<br>
described herein can be introduced into a suitable host cell, and the resulting cell can be maintained (e.g., in culture, in an animal, in a plant) under conditions suitable for expression of the constructs. Suitable host cells can be prokaryotic, including bacterial cells such as E. coli, B. subtilis and/or other suitable bacteria; eucaryotic cells, such as fungal or yeast cells (e.g., Pichiapastoris, Aspergillus sp., Saccharomyces cerevisiae, Schizosaccharomyces pombe, Newospora crassa), or other lower eukaryotic cells, and cells of higher eucaryotes such as those from insects (e.g., Drosophtta Schnieder S2 cells, Sf9 insect cells (WO 94/26087 (O'Connor)), mammals (e.g., COS cells, such as COS-1 (ATCC Accession No. CRL-1650) and COS-7 (ATCC AccessionNo. CRL-1651), CHO (e.g.t ATCC Accession No. CRL-9096), 293 (ATCC Accession No. CRL-1573), HeLa (ATCC Accession No. CCL-2), CV1 (ATCC AccessionNo. CCL-70), WOP (Dailey, L., et al, J. Viroi, 54:739-149 (1985), 3T3,293T (Pear, W. S., et al., Proc. Natl. Acad. Sci. U.S.A., 50:8392-8396 (1993)) NSO cells, SP2/0, HuT 78 cells and the like, or plants (e.g.s tobacco). (See, for example, Ausubel, F.M. et al., eds. Current Protocols in Molecular Biology, Greene Publishing Associates and John Wiley &amp; Sons Inc. (1993).)<br>
The invention also relates to a recombinant host cell which comprises a (one or more) recombinant nucleic acid or expression construct comprising a nucleic acid encoding a polypeptide that unfolds reversibly. The invention also includes a method of preparing an polypeptide that unfolds reversibly, comprising maintaining a recombinant host cell of the invention under conditions appropriate for expression of a polypeptide that unfolds reversibly. The method can further comprise the step of isolating or recovering the polypeptide that unfolds reversibly, if desired.<br>
For example, a nucleic acid molecule (i.e., one or more nucleic acid molecules) encoding a polypeptide that unfolds reversibly, or an expression construct (i.e., one or more constructs) comprising such nucleic acid molecule(s), can be introduced into a suitable host cell to create a recombinant host cell using any method appropriate to the host cell selected (e.g., transformation, transfection, electroporation, infection), such that the nucleic acid molecule(s) are operably linked to one or more expression control elements (e.g., in a vector, in a construct created by processes in the cell, integrated into the host cell genome). The resulting<br>
recombinant host cell can be maintained under conditions suitable for expression (e.g., in the presence of an inducer, in a suitable animal, in suitable culture media supplemented with appropriate salts, growth factors, antibiotics, nutritional supplements, etc.), whereby the encoded polypeptide(s) are produced. If desired, the encoded protein can be isolated or recovered (e.g., from the animal, the host cell, medium, milk). This process encompasses expression in a host cell of a transgenic animal (see, e.g., WO 92/03918, GenPharm International).<br>
The polypeptides mat unfolds reversibly described herein can also be produced in a suitable in vitro expression system, by chemical synthesis or by any other suitable method.<br>
In certain embodiments, the invention does not include a polypeptide having a variable domain comprising a sequence encoded by a germline VH or germline VL gene segment, or consisting of or comprising SEQ ID NOS:7-60.<br>
ASP ILE GLN MET THR GLN SER PRO SER SER LEU SER ALA SER VAL GLY ASP ARG VAL THR ILE THR CYS GLN ALA SER GLN ASP ILE SER ASN TYR LEU ALA TRP TYR GLN GLN LYS PRO GLY LYS ALA PRO GLU LEU ARG ILE TYR ASP ALA SER ASN LEU GLU THR GLY VAL PRO SER ARG PHE SER GLY SER GLY SER GLY THR ASP PHE THR PHE THR ILE SER SER LEU GLN PRO GLU ASP ILE ALA THR TYR TYR CYS GLN GLN TYR GLN ASN LEU PRO LEU THR PHE GLY PRO GLY THR LYS VAL ASP ILE LYS ARG THR VAL ALA ALA PRO SER VAL SEQIDNO:?<br>
GLN ILE VAL LEU THR GLN SER PRO ALA ILE MET SER ALA SER PRO GLY GLU LYS VAL THR MET THR CYS SER ALA SER SER SER VAL TYR TYR MET TYR TRP TYR GLN GLN LYS PRO GLY SER SER PRO ARG LEU LEU ILE TYR ASP THR SER ASN<br>
LEU ALA SER GLY VAL PRO VAL ARG PHE SER GLY SER GLY<br>
SER GLY THR SER TYR SER LEU THR ILE SER ARG MET GLU<br>
ALA GLU ASP ALA ALA THR TYR TYR CYS GLN GLN TRP SER<br>
SER TYR PRO PRO ILE THR PHE GLY VAL GLY THR LYS LEU<br>
GLU LEU LYS ARG ALA ASP ALA ALA PRO THR VAL SER ILE<br>
PHE PRO PRO SER SER GLU GLN LEU THR SER GLY GLY ALA<br>
SER VAL VAL CYS PHE LEU ASN ASN PHE TYR PRO LYS ASP<br>
ILE ASN VAL LYS TRP LYS ILE ASP GLY SER GLU ARG GLN   <br>
ASN GLY VAL LEU ASN SER TRP THR ASP GLN ASP SER LYS<br>
ASP SER THR TYR SER MET SER SER THR LEU THR LEU THR<br>
LYS ASP GLU TYR GLU ARG HIS ASN SER TYR THR CYS GLU<br>
ALA THR HIS LYS THR SER THR SERPRO ILE VAL LYS SER<br>
PHE ASN ARG ASN GLU CYS	SEQDDNO:8<br>
ASP ILE VAL LEU THR GLN SER PRO ALA ILE MET SER ALA SER PRO GLY GLU LYS VAL THR MET THR CYS SER ALA SER SER SER VAL ASN TYR MET TYR TRP TYR GLN GLN LYS SER GLY THR SER PRO LYS ARG TRP ILE TYR ASP THR SER LYS LEU ALA SER GLY VAL PRO VAL ARG PHE SER GLY SER GLY SER GLY THR SER TYR SER LEU THR ILE SER SER MET GLU THR GLU ASP ALA ALA THR TYR TYR CYS GLN GLN TRP GLY ARG ASN PRO THR PHE GLY GLY GLY THR LYS LEU GLU ILE LYS ARG ALA ASP ALA ALA PRO THR VAL SER ILE PHE PRO PRO SER SER GLU GLN LEU THR SER GLY GLY ALA SER VAL VAL CYS PHE LEU ASN ASN PHE TYR PRO LYS ASP ILE ASN VAL LYS TRP LYS ILE ASP GLY SER GLU ARG GLN ASN GLY   -68-<br>
VAL LEU ASN SER TRP THR ASP GLN ASP SER LYS ASP SER<br>
THR TYR SER MET SER SER THR LEU THR LEU THR LYS ASP<br>
GLU TYR GLU ARG HIS ASN SER TYR THR CYS GLU ALA THR<br>
HIS LYS THR SER THR SER PRO ILE VAL LYS SER PHE ASN<br>
ARG ASN GLU CYS    SEQIDNO:9<br>
GLN ILE VAL LEU THR GLN SER PRO ALA ILE MET SER ALA<br>
SER PRO GLY GLU LYS VAL THR MET THR CYS SER ALA SER<br>
SER SER VAL SER TYR MET HIS TRP TYR GLN GLN LYS SER<br>
GLY THR SER PRO LYS ARG TRP ILE TYR ASP THR SER LYS<br>
LEU ALA SER GLY VAL PRO ALA ARG PHE SER GLY SER GLY<br>
SER GLY THR SER TYR SER LEU THR ILE SER SER MET GLU<br>
ALA GLU ASP ALA ALA THR TYR TYR CYS GLN GLN TRP SER<br>
SER ASN PRO TYR THR PHE GLY GLY GLY THR LYS LEU GLU<br>
ILE LYS SEQIDNO:10<br>
GLN ILE VAL LEU THR GLN SER PRO ALA ILE MET SER ALA<br>
SER PRO GLY GLU LYS VAL THR MET THR CYS SER ALA SER<br>
SER SER VAL SER TYR MET HIS TRP TYR GLN GLN LYS SER<br>
GLY THR SER PRO LYS ARG TRP ILE TYR ASP THR SER LYS<br>
LEU ALA SER GLY VAL PRO ALA ARG PHE SER GLY SER GLY<br>
SER GLY THR SER TYR SER LEU THR ILE SER SER MET GLU<br>
ALA GLU ASP ALA ALA THR TYR TYR CYS GLN GLN TRP SER<br>
SER ASN PRO TYR THR PHE GLY GLY GLY THR LYS LEU GLU<br>
ILE LYS SEQIDNO:!!    <br>
GLU ILE GLN LEU THR GLN SERPRO SER SER LEU SER ALA<br>
SER LEU GLY GLU ARG VAL SER LEU THR CYS ARG THR SER<br>
GLN GLU ILE SER GLY TYR LEU SER TRP LEU GLN GLN LYS<br>
PRO ASP GLY THR ILE LYS ARG LEU ILE TYR ASP ALA THR<br>
LYS LEU ASP SER GLY ALA PRO LYS ARG PHE SER GLY SER<br>
ARG SER GLY SER ASP TYR SER LEU THR ILE SER SER LEU<br>
GLU SER GLU ASP PHE ALA ASP TYR TYR CYS LEU GLN TYR<br>
ALA SER PHE PRO ARG THR PHE GLY GLY GLY THR LYS LEU<br>
GLU ILE LYS ARG THR VAL ALA ALA PRO SER VAL PHE ILE<br>
PHE PRO PRO SER ASP GLU GLN LEU LYS SER GLY THR ALA<br>
SER VAL VAL CYS LEU LEU ASN ASN PHE TYR PRO ARG GLU<br>
ALA LYS VAL GLN TRP LYS VAL ASP ASN ALA LEU GLN SER<br>
GLY ASN SER GLN GLU SER VAL THR GLU GLN ASP SER LYS<br>
ASP SER THR TYR SER LEU SER SER THR LEU THR LEU SER<br>
LYS ALA ASP TYR GLU LYS HIS LYS VAL TYR ALA CYS GLU<br>
VAL THR HIS GLN GLY LEU SER SER PRO VAL THR LYS SER<br>
PHE ASN ARG GLY GLU CYS	SEQIDNO:12<br>
ASP ILE GLN MET THR GLN SER PRO SER SER LEU SER ALA SER VAL GLY ASP ARG VAL THR ILE THR CYS GLN ALA SER GLN ASP ILE SER ASP TYR LEU ILE TRP TYR GLN GLN LYS LEU GLY LYS ALA PRO ASN LEU LEU ELE TYR ASP ALA SER THR LEU GLU THR GLY VAL PRO SER ARG PHE SER GLY SER GLY SER GLY THR GLU TYR THR PHE THR ILE SER SER LEU GLN PRO GLU ASP ILE ALA THR TYR TYR CYS GLN GLN TYR ASP ASP LEU PRO TYR THR PHE GLY GLN GLY THR LYS VAL  -70-GLU ILE LYS ARG SEQIDNO:13<br>
ASP ILE GLN MET THR GLN SER PRO SER SER LEU SER ALA<br>
SER VAL GLY ASP ARG VAL THR ILE THR CYS GLN ALA SER<br>
GLN ASP ILE SER ASP TYR LEU ILE TRP TYR GLN GLN LYS<br>
LEU GLY LYS ALA PRO ASN LEU LEU ILE TYR ASP ALA SER<br>
THR LEU GLU THR GLY VAL PRO SER ARG PHE SER GLY SER<br>
GLY SER GLY THR GLU TYR THR PHE THR ILE SER SER LEU<br>
GLN PRO GLU ASP ILE ALA THR TYR TYR CYS GLN GLN TYR<br>
ASP ASP LEU PRO TYR THR PHE GLY GLN GLY THR LYS VAL<br>
GLU ILE LYS ARG	SEQ ID NO:14<br>
ASP ILE GLN MET THR GLN SER PRO SER SER LEU SER ALA<br>
SER VAL GLY ASP ARG VAL THR ILE THR CYS GLN ALA SER<br>
GLN ASP ILE SER ASP TYR LEU ILE TRP TYR GLN GLN LYS<br>
LEU GLY LYS ALA PRO ASN LEU LEU ILE TYR ASP ALA SER<br>
THR LEU GLU THR GLY VAL PRO SER ARG PHE SER GLY SER<br>
GLY SER GLY THR GLU TYR THR PHE THR ILE SER SER LEU<br>
GLN PRO GLU ASP ILE ALA THR TYR TYR CYS GLN GLN TYR<br>
ASP ASP LEU PRO TYR THR PHE GLY GLN GLY THR LYS VAL<br>
GLU ILE LYS ARG	SEQ ID NO: 15<br>
ASP ILE GLN MET THR GLN SER PRO SER SER LEU SER ALA SER VAL GLY ASP ARG VAL THR ILE THR CYS GLN ALA SER GLN ASP ILE SER ASP TYR LEU ILE TRP TYR GLN GLN LYS LEU GLY LYS ALA PRO ASN LEU LEU ILE TYR ASP ALA SER<br>
THR LEU GLU THR GLY VAL PRO SER ARG PHE SER GLY SER GLY SER GLY THR GLU TYR THR PHE THR ILE SER SER LEU GLN PRO GLU ASP ILE ALA THR TYR TYR CYS GLN GLN TYR ASP ASP LEU PRO TYR THR PHE GLY GLN GLY THR LYS VAL SEQE&gt;NO:16<br>
DIQMTQSPSTLSASVGDRYTITCRASQSISSWLAWYQQKPGKAPKLLIYDAS SLESGVPSRFSGSGSGTEFTLTISSLQPDDFATYYCQQYNSYS SEQIDNO:17<br>
DIVLTQAPPSLDVSQGRATISCRTSKSVRTSSYSYMHWYQQKPGQPPKLLNL CASNQVSRVPARFSGSGSGTDFTLKIHPVEEEDAATYFCQQSNENP SEQ ID NO: 18<br>
VrWMTQSPSLLSASTGDRVTISCRMSQGISSYLAWYQQKPGKAPELLIYAAS<br>
TLQSGVPSRFSGSGSGTDFTLTISCLQSEDFATYYCQQYYSFP	SEQ ID<br>
NO: 19<br>
AIQLTQSPSSLSASVGDRVTITCRASQGISSALAYQQKPGKAPKLUYDASSLE SGVPSRFSGSGSGTDFTLTISSLQPEDFATYYCQQFNNYP      SEQ ID NO:20<br>
AIQLTQSPSSLSASVGDRVTITCRASQGISSALAWYQQKPGKAPKLLIYDASS LESGVPSRFSGSGSGTDFTLTISSLQPEDFATYYCQQFNNYP SEQ ID NO:21<br>
DIQMTQSPSSLSASVGDRVTITCQASQDISNYLNWYQQKPGKAPBXLIVDAS<br>
NLETGVPSRFSGSGSGTDFTFTISSLQPEDIATYYCQQYDNLP	SEQ ID<br>
NO:22<br>
IQMTQSPSSLSASVGDRVTTrCQASQDISNYLNWYQQKPGKAPKLLIYDAS<br>
NLETGVPSRFSGSGSGTDFTFTISSLQPEDIATYYCQQYDNLP	SEQ ID<br>
NO:23<br>
DIQMQSPSFLSASVGDRVSnCWASEGISSNLAWYLQKPGKSPKLFLYDAKD LHPGVSSRFSGRGSGTDFTLTnSLKPEDFAAYYCKQDFSYP SEQ ID NO:24<br>
ErVXTQSPATLSLSPGERATLSCRASQSVSSYLAWYQQKPGQAPRLLIYDASN RATGIPARFSGSGSGTDFTLTISSLEPEDFAVYYCQQRSNWP SEQ IDNO:25<br>
EIYLTQSPATLSLSPGERATLSCRASQGVSSYLAWYQQKPGQAPRLLIYDAS NRATGIPARFSGSGPGTDFTLTISSLEPEDFAVYYCQQRSNWH SEQ ID N0:26<br>
EIVLTQSPATLSI^PGERATLSCGASQSVSSSYLAWYQQIffGIJ^RLIJYDAS<br>
SRATGIPDRFSGSGSGTDFTLTISRLEPEDFAVYYCQQYGSSP	SEQ ID<br>
N0:27<br>
QIVLTQSPAIMSASPGEKVTMTCSASSSVSYMYWYQQKPGSSPRLLIYDTSN LASGVPVRFSGSGSGTSYSLTISRMEAEDAATYYCQQWSSYPP       SEQ ID NO:28<br>
QIVLTQSPAMSASPGEKVTMTCSASSSVSYMHWYQQKSGTSPKRWIYDTS KLASGVPARFSGSGSGTSYSLTISSMEAEDAATYYCQQWSSNP SEQ ID N0:29<br>
QIVLTQSPAIMSASPGEKVTMTCSASSSVSYMYWYQQKPGSSPRLWIYDTSN LVSGVPARFSGSRSGTSYSLTISSMEAEDAATYYCQQYSGYPS       SEQ ID N0:30<br>
NVLTQSPAIMSASPGEKVTMTCSASSSVSYMHWYQQKSSTSPKLWIYDTS KLASGVPGRFSGSGSGNSYSLTISSMEAEDVATYYCFQGSGYPL SEQ ID NO:31<br>
QILLTQSPAIMSASPGEKVTMTCSASSSVSYMHWYQQKPGSSPKPWIYT&gt;TSN LASGFPARFSGSGSGTSYSLHSSMEAEDAATYYCHQRSSYP SEQ ID N0:32<br>
QIVLTQSPAIMSASPGEKVIMrCSASSSISYMHWYQQKPGTSPKRWIYDTSK<br>
LASGVPARFSGSGSGTSYSLHSSMEAEDAATYYCHQRSSYP	SEQ ID<br>
NO:33<br>
DIQMTQSPASLSASVGETVTITCQASENIASDLAWYQKQGKSPQLLVYDAR NLADGVPSRFSGSGSGTHYSLNIHSLQSEDVARYYCQHYYGTP SEQ ID NO:34<br>
AQVLTQTESPVSAPVGGTVTINCQASQSVYDNNYLSWYQQKPGQPPECLLIY DASKLASGVPSRFSGSGSGTQFTLTISGVQCDDAATYYCQGSYYSSGWY SEQIDNO:35<br>
AQGPTQTPSSVSAAVGGTVTINCQTSESFYSNNILSWYQQKPGQPPKLLIYD ASTLASGVPSRFKGSGSGTQFTLTISDVQCDDAATYYCQGSYHSSGWY SEQIDNO:36<br>
AAVLTQTPSPVSAAVGGTVTKCQSSQSVYNNNLLSWYQQKPGQPPKLLIY DASNLASGVPDKFSGSGSGTQFTLTISGVQCDDAATYYCLGGYDDDAD SEQIDNO:37<br>
AWLTQTASPVSAAVGGTVHNCQASQSISTALAWYQQKPGQRPKLLIYDAS<br>
KLASGVPSRFKGSGSGTQFTLTISGVQCDDAATYYCQQGYSSSSAD SEQIDNO:38<br>
GLU VAL LYS VAL ILE GLU SER GLY GLY GLY LEU VAL GLN<br>
PRO GLY GLY SER LEU LYS LEU SER CYS ALA ALA SER GLY<br>
PHE ASP PHE SER ARG TYR TRP MET SER TRP VAL ARG GLN<br>
ALA PRO GLY LYS GLY LEU GLU TRP ILE GLY GLU ILE ASN<br>
PRO ASP SER SER THR ILE ASN TYR THR PRO SER LEU LYS<br>
ASP LYS PHE ILE ILE SER ARG ASP ASN ALA LYS ASN THR<br>
LEU TYR LEU GLN MET SER LYS VAL ARG SER GLU ASP THR<br>
ALA LEU TYR TYR CYS ALA ARG LEU GLY GLY ASP LEU HIS<br>
TYR TYR ALA MET ASP TYR TRP GLY GLN GLY THR SER VAL<br>
THR VAL SER SER	SEQ ID NO:39<br>
GLU VAL LYS VAL ELE GLU SER GLY GLY GLY LEU VAL GLN PRO GLY GLY SER LEU LYS LEU SER CYS ALA ALA SER GLY PHE ASP PHE SER ARG TYR TRP MET SER TRP VAL ARG GLN ALA PRO GLY LYS GLY LEU GLU TRP ILE GLY GLU ILE ASN PRO ASP SER SER THR ILE ASN TYR THR PRO SER LEU LYS ASP LYS PHE ILE ILE SER ARG ASP ASN ALA LYS ASN THR LEU TYR LEU GLN MET SER LYS VAL ARG SER GLU ASP THR ALA LEU TYR TYR CYS ALA ARG LEU GLY GLY ASP LEU HIS TYR TYR ALA MET ASP TYR TRP GLY GLN GLY THR SER VAL THR VAL SER SER SEQ ID N0:40<br>
GLU VAL LYS LEU LEU GLU SER GLY GLY GLY LEU VAL GLN<br>
  PRO GLY GLY SER LEU LYS LEU SER CYS ALA ALA SER GLY PHE ASP PHE SER LYS TYR TRP MET SER TRP VAL ARG GLN ALA PRO GLY LYS GLY LEU GLU TRP TTP. GLY GLU ILE HIS PRO ASP SER GLY THR ILE ASN TYR THR PRO SER LEU LYS ASP LYS PHE ILE ILE SER ARG ASP ASN ALA LYS ASN SER LEU TYR LEU GLN MET SER LYS VAL ARG SER GLU ASP THR ALA LEU TYR TYR CYS ALA ARG LEU HIS TYR TYR GLY TYR ASN ALA TYR TRP GLY GLN GLY THR LEU VAL THR VAL SER ALA GLU SER ALA ARG ASN PRO THR BLE TYR PRO LEU THR LEU PRO PRO ALA LEU SER SER ASP PRO VAL ILE ILE GLY CYS LEU ILE HIS ASP TYR PHE PRO SER GLY THR MET ASN VAL THR TRP GLY LYS SER GLY LYS ASP ILE THR THR VAL ASN PHE PRO PRO ALA LEU ALA SER GLY GLY ARG TYR THR MET SER ASN GLN LEU THR LEU PRO ALA VAL GLU CYS PRO GLU GLY GLU SER VAL LYS CYS SER VAL GLN HIS ASP SER ASN PRO VAL GLN GLU LEU ASP VAL ASN CYS SER GLY SEQIDNO:41<br>
METTYRARG SERALAPHE SER VAL GLY LEUGLUTHRARG VAL THR VAL PRO ASN VAL PRO ILE ARG PHE THR LYS ILE PHE TYR ASN GLN GLN ASN HIS TYR ASP GLY SER THR GLY LYS PHE TYR CYS ASN ILE PRO GLY LEU TYR TYR PHE SER TYR HIS ILE THR VAL TYR MET LYS ASP VAL LYS VAL SER LEU PHE LYS LYS ASP LYS ALA VAL LEU PHE THR TYR ASP GLN TYR GLN GLU LYS ASN VAL ASP GLN ALA SER GLY SER VAL LEU LEU HIS LEU GLU VAL GLY ASP GLN VAL TRP LEU<br>
GLN VAL TYR GLY ASP GLY ASP HIS ASN GLY LEU TYR ALA ASP ASN VAL ASN ASP SER THR PHE THR GLY PHE LEU LEU TYR HIS ASP THR ASN  SEQIDNO:42<br>
MET TYR ARG SER ALA PHE SER VAL GLY LEU GLU THR ARG VAL THR VAL PRO ASN VAL PRO ILE ARG PHE THR LYS ILE PHE TYR ASN GLN GLN ASN HIS TYR ASP GLY SER THR GLY LYS PHE TYR CYS ASN ILE PRO GLY LEU TYR TYR PHE SER TYR HIS ILE THR VAL TYR MET LYS ASP VAL LYS VAL SER LEU PHE LYS LYS ASP LYS ALA VAL LEU PHE THR TYR ASP GLN TYR GLN GLU LYS ASN VAL ASP GLN ALA SER GLY SER VAL LEU LEU HIS LEU GLU VAL GLY ASP GLN VAL TRP LEU GLN VAL TYR GLY ASP GLY ASP HIS ASN GLY LEU TYR ALA ASP ASN VAL ASN ASP SER THR PHE THR GLY PHE LEU LEU TYR HIS ASP THR ASN     SEQIDNO:43<br>
MET TYR ARG SER ALA PHE SER VAL GLY LEU GLU THR ARG VAL THR VAL PRO ASN VAL PRO ILE ARG PHE THR LYS ILE PHE TYR ASN GLN GLN ASN HIS TYR ASP GLY SER THR GLY LYS PHE TYR CYS ASN ELE PRO GLY LEU TYR TYR PHE SER TYR HIS ILE THR VAL TYR MET LYS ASP VAL LYS VAL SER LEU PHE LYS LYS ASP LYS ALA VAL LEU PHE THR TYR ASP GLN TYR GLN GLU LYS ASN VAL ASP GLN ALA SER GLY SER VAL LEU LEU HIS LEU GLU VAL GLY ASP GLN VAL TRP LEU GLN VAL TYR GLY ASP GLY ASP HIS ASN GLY LEU TYR ALA ASP ASN VAL ASN ASP SER THR PHE THR GLY PHE LEU LEU<br>
TYR HIS ASP THR ASN	SEQ ID NO:44<br>
MET TYR ARG SER ALA PHE SER VAL GLY LEU GLU THR ARG<br>
VAL THR VAL PRO ASN VAL PRO ILE ARG PHE THR LYS ILE<br>
PHE TYR ASN GLN GLN ASN HIS TYR ASP GLY SER THR GLY<br>
LYS PHE TYR CYS ASN ILE PRO GLY LEU TYR TYR PHE SER<br>
TYR HIS ILE THR VAL TYR MET LYS ASP VAL LYS VAL SER<br>
LEU PHE LYS LYS ASP LYS ALA VAL LEU PHE THR TYR ASP<br>
GLN TYR GLN GLU LYS ASN VAL ASP GLN ALA SER GLY SER<br>
VAL LEU LEU HIS LEU GLU VAL GLY ASP GLN VAL TRP LEU<br>
GLN VAL TYR GLY ASP GLY ASP HIS ASN GLY LEU TYR ALA<br>
ASP ASN VAL ASN ASP SER THR PHE THR GLY PHE LEU LEU<br>
TYR HIS ASP THR ASN	SEQ ID NO:45<br>
MET TYR ARG SER ALA PHE SER VAL GLY LEU GLU THR ARG<br>
VAL THR VAL PRO ASN VAL PRO ILE ARG PHE THR LYS ILE<br>
PHE TYR ASN GLN GLN ASN HIS TYR ASP GLY SER THR GLY<br>
LYS PHE TYR CYS ASN ILE PRO GLY LEU TYR TYR PHE SER<br>
TYR HIS ILE THR VAL TYR MET LYS ASP VAL LYS VAL SER<br>
LEU PHE LYS LYS ASP LYS ALA VAL LEU PHE THR TYR ASP<br>
GLN TYR GLN GLU LYS ASN VAL ASP GLN ALA SER GLY SER<br>
VAL LEU LEU HIS LEU GLU VAL GLY ASP GLN VAL TRP LEU<br>
GLN VAL TYR GLY ASP GLY ASP HIS ASN GLY LEU TYR ALA<br>
ASP ASN VAL ASN ASP SER THR PHE THR GLY PHE LEU LEU<br>
TYR HIS ASP THR ASN	SEQIDNO:46<br>
MET TYR ARG SER ALA PHE SER VAL GLY LEU GLU THR ARG<br>
VAL THR VAL PRO ASN VAL PRO ILE ARC PHE THR LYS ILE<br>
PHE TYR ASN GLN GLN ASN HIS TYR ASP GLY SER THR GLY<br>
LYS PHE TYR CYS ASN ILE PRO GLY LEU TYR TYR PHE SER<br>
TYR HIS ILE THR VAL TYR MET LYS ASP VAL LYS VAL SER<br>
LEU PHE LYS LYS ASP LYS ALA VAL LEU PHE THR TYR ASP<br>
GLN TYR GLN GLU LYS ASN VAL ASP GLN ALA SER GLY SER<br>
VAL LEU LEU HIS LEU GLU VAL GLY ASP GLN VAL TRP LEU<br>
GLN VAL TYR GLY ASP GLY ASP HIS ASN GLY LEU TYR ALA<br>
ASP ASN VAL ASN ASP SER THR PHE THR GLY PHE LEU LEU<br>
TYR HIS ASP THR ASN	SEQIDNO:47<br>
ALA VAL GLN LEU GLU GLN SER GLY PRO GLY LEU VAL ARG PRO SER GLN THR LEU SER LEU THR CYS THR VAL SER GLY THR SER PHE ASP ASP TYR TYR TRP THR TRP VAL ARG GLN PRO PRO GLY ARG GLY LEU GLU TRP ILE GLY TYR VAL PHE TYR THR GLY THR THR LEU LEU ASP PRO SER LEU ARG GLY ARG VAL THR MET LEU VAL ASN THR SER LYS ASN GLN PHE SER LEU ARG LEU SER SER VAL THR ALA ALA ASP THR ALA VAL TYR TYR CYS ALA ARG ASN LEU ILE ALA GLY GLY ILE ASP VAL TRP GLY GLN GLY SER LEU VAL THR VAL SER SER ALA SER THR LYS GLY PRO SER VAL PHE PRO LEU ALA PRO SER SER LYS SER THR SER GLY GLY THR ALA ALA LEU GLY CYS LEU VAL LYS ASP TYR PHE PRO GLU PRO VAL THR VAL SER TRP ASN SER GLY ALA LEU THR SER GLY VAL HIS THR<br>
PHE PRO ALA VAL LEU GLN SER SER GLY LEU TYR SER LEU SER SER VAL VAL THR VAL PRO SER SER SER LEU GLY THR GLN THR TYR ILE CYS ASN VAL ASN HIS LYS PRO SER ASN THR LYS VAL ASP LYS LYS VAL GLU PRO  SEQ ID NO:48<br>
GLU VAL GLN LEU VAL GLU SER GLY GLY GLY LEU VAL GLN<br>
PRO GLY ARG SER LEU ARG LEU SER CYS VAL THR SER GLY<br>
PHE THR PHE ASP ASP TYR ALA MET HIS TRP VAL ARG GLN<br>
SER PRO GLY LYS GLY LEU GLU TRP VAL SER GLY ILE SER<br>
TRP ASN THR GLY THR ILE ILE TYR ALA ASP SER VAL LYS<br>
GLY ARG PHE ILE ILE SER ARG ASP ASN ALA LYS ASN SER<br>
LEU TYR LEU GLN MET ASN SER LEU ARG VAL GLU ASP THR<br>
ALA LEU TYR TYR CYS ALA LYS THR ARG SER TYR VAL VAL<br>
ALA ALA GLU TYR TYR PHE HIS TYR TRP GLY GLN GLY ILE<br>
LEU VAL THR VAL SER SER GLY SER ALA SER ALA PRO THR<br>
LEU PHE PRO LEU VAL SER CYS GLU ASN SER ASN PRO SER<br>
SER THR VAL ALA VAL GLY CYS LEU ALA GLN ASP PHE LEU<br>
PRO ASP SER ILE THR PHE SER TRP LYS TYR LYS ASN ASN<br>
SER ASP ILE SER SER THR ARG GLY PHE PRO SER VAL LEU<br>
ARG GLY GLY LYS TYR ALA ALA THR SER GLN VAL LEU LEU<br>
PRO SER LYS ASP VAL MET GLN GLY THR ASN GLU HIS VAL<br>
VAL CYS LYS VAL GLN HIS PRO ASN GLY ASN LYS GLU LYS<br>
ASP VAL PRO LEU	SEQIDNO:49<br>
ASP VAL GLN LEU GLN ALA SER GLY GLY GLY SER VAL GLN ALA GLY GLY SER LEU ARG LEU SER CYS ALA ALA SER GLY<br>
TYR THR ILE GLY PRO TYR CYS MET GLY TRP PHE ARG GLN<br>
ALA PRO GLY LYS GLU ARG GLU GLY VAL ALA ALA ILE ASN<br>
MET GLY GLY GLY ILE THR TYR TYR ALA ASP SER VAL LYS<br>
GLY ARG PHE THR ILE SER GLN ASP ASN ALA LYS ASN THR<br>
VAL TYR LEU LEU MET ASN SER LEU GLU PRO GLU ASP THR<br>
ALA ILE TYR TYR CYS ALA ALA ASP SER THR ILE TYR ALA<br>
SER TYR TYR GLU CYS GLY HIS GLY LEU SER THR GLY GLY<br>
TYR GLY TYR ASP SER TRP GLY GLN GLY THR GLN VAL THR<br>
VAL SER SER GLY ARG TYR PRO TYR ASP VAL PRO ASP TYR<br>
GLY SER GLY ARG ALA	SEQIDNO:50<br>
ASP VAL GLN LEU GLN GLN SER GLY PRO GLU LEU GLU LYS PRO GLY ALA SER VAL LYS ILE SER CYS LYS ALA SER GLY PHE SER LEU PRO GLY HIS ASN ILE ASN TRP ILE VAL GLN ARG ASN GLY LYS SER LEU GLU TRP ILE GLY ASN ILE ASP PRO TYR TYR GLY GLY THR ASN PHE ASN PRO LYS PHE LYS GLY LYS ALA THR LEU THR VAL ASP LYS SER SER SER THR LEU TYR MET HIS LEU THR SER LEU GLN SER GLU ASP SER ALA VAL TYR TYR CYS ALA ARG ARG ARG ASP GLY ASN TYR GLY PHE THR TYR TRP GLY GLN GLY THR LEU VAL THR VAL SER ALA ALA LYS THR THR PRO PRO SER VAL TYR PRO LEU ALA PRO GLY SER ALA ALA GLN THR ASN SER MET VAL THR LEU GLY CYS LEU VAL LYS GLY TYR PHE PRO GLU PRO VAL THR VAL THR TRP ASN SER GLY SER LEU SER SER GLY VAL HIS THR PHE PRO ALA VAL LEU GLN SER ASP LEU TYR THR LEU SER SER SER VAL THR VAL PRO SER SER THR TRP PRO<br>
SER GLU THR VAL THR CYS ASN VAL ALA HIS PRO ALA SER SERTHRLYSVALASPLYSLYSILE    SEQIDNO:51<br>
QDTLKEYGPGKLPSQTFSLTCTFSGFSLSTYGMMVSWMCQPSGKGLVWLA LIWCNNDKGYNPFLRSQLTESKDTSNNQVFLKITSVDPADTATYYCA SEQEDNO:52<br>
QVQLQQSGPGLVKPSQTLSLTCAISGDSVSSNSAAWNWIRQSPSRGLEWLG RTYYRSKWYNDYAVSVKSRITINPDTSKNQFSLQLNSVTPEDTAVYYCAR SEQIDNO-.53<br>
ASP ILE GLN MET THR GLN SER PRO ALA ILE MET SER ALA SER PRO GLY GLU LYS VAL THR MET THR CYS SER ALA SER SER SER VAL SER TYR MET TYR TRP TYR GLN GLN LYS PRO GLY SER SER PRO ARG LEU LEU ILE TYR ASP SER THR ASN LEU ALA SER GLY VAL PRO VAL ARG PHE SER GLY SER GLY SER GLY THR SER TYR SER LEU THR ELE SER ARG MET GLU ALA GLU ASP ALA ALA THR TYR TYR CYS GLN GLN TRP SER THR TYR PRO LEU THR PHE GLY ALA GLY THR LYS LEU GLU LEU LYS ARG ALA ASP ALA ALA PRO THR VAL SER ILE PHE PRO PRO SER SER GLU GLN LEU THR SER GLY GLY ALA SER VAL VAL CYS PHE LEU ASN ASN PHE TYR PRO LYS ASP ILE ASN VAL LYS TRP LYS ILE ASP GLY SER GLU ARG GLN ASN GLY VAL LEU ASN SER TRP THR ASP GLN ASP SER LYS ASP SER THR TYR SER MET SER SER THR LEU THR LEU THR LYS ASP GLU TYR GLU ARG HIS ASN SER TYR THR CYS GLU ALA THR HIS LYS THR SER THR SER PRO ILE VAL LYS SER PHE ASN ARG  SEQIDNO:54<br>
GLN ILE VAL SER THR GLN SER PRO ALA ILE MET SER ALA SER PRO GLY GLU LYS VAL THR MET THR CYS SER ALA SER<br>
SER SER ARG SER TYR MET GLN TRP TYR GLN GLN LYS PRO GLY THR SER PRO LYS ARG TRP ILE TYR ASP THR SER LYS LEU ALA SER GLY VAL PRO ALA ARG PHE SER GLY SER GLY SER GLY THR SER TYR SER LEU THR ILE SER SER MET GLU ALA GLU ASP ALA ALA THR TYR TYR CYS HIS GLN ARG SER SER TYR THR PHE GLY GLY GLY THR LYS LEU GLU ILE LYS ARG THR VAL ALA ALA PRO SER VAL PHE ILE PHE PRO PRO SER ASP GLU GLN LEU LYS SER GLY THR ALA SER VAL VAL CYS LEU LEU ASN ASN PHE TYR PRO ARG GLU ALA LYS VAL GLN TRP LYS VAL ASP ASN ALA LEU GLN SER GLY ASN SER GLN GLU SER VAL THR GLU GLN ASP SER LYS ASP SER THR TYR SER LEU SER SBR THR LEU THR LEU SER LYS ALA ASP TYR GLU LYS HIS LYS VAL TYR ALA CYS GLU VAL THR HIS GLN GLY LEU SER SER PRO VAL THR LYS SER PHE ASN ARG GLY GLU       SEQIDNO:55<br>
GLU LEU VAL MET THR GLN SER PRO SER SER LEU SER ALA SER VAL GLY ASP ARG VAL ASN ILE ALA CYS ARG ALA SER GLN GLY ILE SER SER ALA LEU ALA TRP TYR GLN GLN LYS PRO GLY LYS ALA PRO ARG LEU LEU ILE TYR ASP ALA SER ASN LEU GLU SER GLY VAL PRO SER ARG PHE SER GLY SER GLY SER GLY THR ASP PHE THR LEU THR ILE SER SER LEU GLN PRO GLU ASP PHE ALA ILE TYR TYR CYS GLN GLN PHE ASN SER TYR PRO LEU THR PHE GLY GLY GLY THR LYS VAL GLU ILE LYS ARG THR VAL ALA ALA PRO SER VAL PHE ILE PHE PRO PRO SER ASP GLU GLN LEU LYS SER GLY THR ALA SER VAL VAL CYS LEU LEU ASN ASN PHE TYR PRO ARG GLU ALA LYS VAL GLN TRP LYS VAL ASP ASN ALA LEU GLN SER GLY ASN SER GLN GLU SER VAL THR GLU GLN ASP SER LYS ASP SER THR TYR SER LEU SER SER THR LEU THR LEU SER LYS ALA ASP TYR GLU LYS HIS LYS VAL TYR ALA CYS GLU<br>
VAL THR HIS GLN GLY LEU SER SER PRO VAL THR LYS SER PHE ASN ARG GLY GLU CYS SEQ ID NO:56<br>
GLU LEU VAL MET THR GLN SER PRO SER SER LEU SER ALA SER VAL GLY ASP ARG VAL ASN ILE ALA CYS ARG ALA SER GLN GLY ILE SER SER ALA LEU ALA TRP TYR GLN GLN LYS PRO GLY LYS ALA PRO ARG LEU LEU ILE TYR ASP ALA SER ASN LEU GLU SER GLY VAL PRO SER ARG PHE SER GLY SER GLY SER GLY THR ASP PHE THR LEU THR ILE SER SER LEU GLN PRO GLU ASP PHE ALA ILE TYR TYR CYS GLN GLN PHE ASN SER TYR PRO LEU THR PHE GLY GLY GLY THR LYS VAL GLU ILE LYS ARG THR VAL ALA ALA PRO SER VAL PHE ILE PHE PRO PRO SER ASP GLU GLN LEU LYS SER GLY THR ALA SER VAL VAL CYS LEU LEU ASN ASN PHE TYR PRO ARG GLU ALA LYS VAL GLN TRP LYS VAL ASP ASN ALA LEU GLN SER GLY ASN SER GLN GLU SER VAL THR GLU GLN ASP SER LYS ASP SER THR TYR SER LEU SER SER THR LEU THR LEU SER LYS ALA ASP TYR GLU LYS HIS LYS VAL TYR ALA CYS GLU VAL THR HIS GLN GLY LEU SER SER PRO VAL THR LYS SER PHE ASN ARG GLY GLU CYS SEQ ID N0:57<br>
GLU ILE VAL MET THR GLN SER PRO ALA SER LEU SER LEU<br>
SER PRO GLY GLU ARG ALA THR LEU SER CYS ARG ALA SER<br>
GLN SER VAL SER ASN TYR LEU ALA TRP TYR GLN GLN LYS<br>
PRO GLY GLN ALA PRO ARG LEU LEU ILE HIS ASP ALA SER<br>
GLY ARG ALA THR GLY ILE PRO ASP ARG PHE SER GLY SER<br>
THR ASP PHE THR LEU THR ILE SER ARG LEU<br>
GLU PRO GLU ASP PHE ALA VAL TYR TYR CYS GLN GLN ARG<br>
ALA ASN TRP GLY THR TRP THR PHE GLY GLN GLY THR LYS<br>
VAL GLU ILE LYS ARG THR	SEQIDNO:58<br>
GLU ILE VAL LEU THR GLN SER PRO ALA THR LEU SER LEU<br>
SER PRO GLY GLU ARG ALA THR LEU SER CYS GLY ALA SER<br>
GLN SER VAL SER SER ASN TYR LEU ALA TRP TYR GLN GLN<br>
LYS PRO GLY GLN ALA PRO ARG LEU LEU ILE TYR ASP ALA<br>
SER SER ARG ALA THR GLY ILE PRO ASP ARG PHE SER GLY<br>
SER GLY SER GLY THR ASP PHE THR LEU THR ILE SER ARG<br>
LEU GLU PRO GLU ASP PHE ALA VAL TYR TYR CYS GLN GLN<br>
TYR GLY SER SER PRO LEU THR PHE GLY GLY GLY THR LYS<br>
VAL GLU ILE LYS ARG THR VAL ALA ALA PRO SER VAL PHE<br>
ILE PHE PRO PRO SER ASP GLU GLN LEU LYS SER GLY THR<br>
ALA SER VAL VAL CYS LEU LEU ASN ASN PHE TYR PRO ARG<br>
GLU ALA LYS VAL GLN TRP LYS VAL ASP ASN ALA LEU GLN<br>
SER GLY ASN SER GLN GLU SER VAL THR GLU GLN ASP SER<br>
LYS ASP SBR THR TYR SER LEU SER SER THR LEU THR LEU<br>
SER LYS ALA ASP TYR GLU LYS HIS LYS VAL TYR ALA CYS<br>
GLU VAL THR HIS GLN GLY LEU SER SER PRO VAL THR LYS<br>
SER PHE ASN ARG GLY GLU CYS	SEQIDNO:59<br>
ASP ILE GLN MET THR GLN SER PRO ALA HJS MET SER ALA SER PRO GLY GLU LYS VAL THR MET THR CYS SER ALA SER SER SER VAL SER TYR MET TYR TRP TYR GLN GLN LYS PRO GLY SER SER PRO ARG LEU LEU ILE TYR ASP SER THR ASN LEU ALA SER GLY VAL PRO VAL ARG PHE SER GLY SER GLY SER GLY THR SER TYR SER LEU THR ILE SER ARG MET GLU ALA GLU ASP ALA ALA THR TYR TYR CYS GLN GLN TRP SER THR TYR PRO LEU THR PHE GLY ALA GLY THR LYS LEU GLU LEU LYS ARG ALA ASP ALA ALA PRO THR VAL SER ILE PHE PRO PRO SER SER GLU GLN LEU THR SER GLY GLY ALA SER VAL VAL CYS PHE LEU ASN ASN PHE TYR PRO LYS ASP ILE ASN VAL LYS TRP LYS ILE ASP GLY SER GLU ARG GLN ASN GLY VAL LEU ASN SER TRP THR ASP GLN ASP SER LYS ASP SER THR TYR SER MET SER SER THR LEU THR LEU THR LYS<br>
ASP GLU TYR GLU ARG HIS ASN SER TYR THE CYS GLU ALA<br>
THR HIS LYS THR SER THR SER PRO ILE VAL LYS SER PHE<br>
ASN ARG	SEQIDNO:60<br>
TARGET LIGANDS, COMPOSITIONS AND METHODS<br>
The polypeptides that unfold reversibly described herein can have binding specificity for a target ligand. For example, a polypeptide that comprises an antibody variable region that unfolds reversibly and has binding specificity for a particular target ligand can be selected, isolated and/or recovered using any suitable method, such as the binding methods described herein. Exemplary target ligands that polypeptides that unfold reversibly (e,g., polypeptides comprising a reversibly unfoldable VH or VK) can have binding specificity for include, human or animal proteins, cytokines, cytokine receptors, enzymes, co-factors for enzymes and DNA binding proteins. Suitable cytokines and growth factors, cytokine and growth factor receptor and other target ligands include but are not limited to: ApoE, Apo-SAA, BDNF, Cardiotrophin-1, CEA, CD40, CD40 Ligand, CD56, CD38, CD138, EGF, EGF receptor, ENA-78, Eotaxin, Eotaxin-2, Exodus-2, FAPa, FGF-acidic, FGF-basic, fibroblast growth factor-l0, FLT3 ligand, Fractalkine (CX3C), GDNF, G-CSF, GM-CSF, GF-pl, human serum albumin, hisulin, 1FN-7, IGF-I, IGF-II, IL-la, IL-1 p, IL-1 receptor, IL-2, IL-3, IL-4, IL-5, IL-6, IL-7, BL-8 (72 a.a.), IL-8 (77 a.a.), IL-9, IL-10, IL-11, IL-12, IL-13, IL-15, IL-16, IL-17, IL-18 (IGIF), Inhibin a, Inhibin P, IP-10, keratinocyte growth factor-2 (KGF-2), KGF, Leptin, LIF, Lymphotactin, Mullerian inhibitory substance, monocyte colony inhibitory factor, monocyte attractant protein, M-CSF, MDC (67 a.a.), MDC (69 a.a.), MCP-1 (MCAF), MCP-2, MCP-3, MCP-4, MDC (67 a.a.), MDC (69 a.a,), MIG, MlP-la, MIP-lß, MIP-3α, MIP-3ß, MIP-4, myeloid progenitor inhibitor factor-1 (MPIF-1), NAP-2, Neurturm, Nerve growth factor, P-NGF, NT-3, NT-4, Oncostatin M, PDGF-AA, PDGF-AB, PDGF-BB, PF-4, RANTES, SDFla, SDFlp, SCF, SCGF, stem cell factor (SCF), TARC, TGF-a, TGF-p, TGF-p2, TGF-p3, tumour necrosis factor (TNF), TNF-α, TNF-ß, TNF receptor I, TNF receptor II, TNIL-1, TPO, VEGF, VEGF A, VEGF B, VEGF C, VEGF D, VEGF receptor 1, VEGF receptor 2, VEGF<br>
receptor 3, GCP-2, GRO/MGSA, GRO-ß GRO-γ, HCC1,1-309, HER 1, HER 2, HER 3 and HER 4. It will be appreciated that this list is by no means exhaustive.<br>
In some embodiments, the invention is an isolated polypeptide that comprises an antibody variable domain that unfolds reversibly and binds a target ligand. Preferably, the antibody variable domain mat unfolds reversibly binds a target ligand that is a cytokine, growth factor, cytokine receptor or growth factor receptor (e.g&gt;, a human cytokine, human growth factor, human cytokine receptor or human growth factor receptor). More preferably, the antibody variable domain that unfolds reversibly neutralizes the activity of the a cytokine, growth factor, cytokine receptor or growth factor receptor with a neutralized dose 50 (ND50) of about 1 µM or less, or 500 nM or less, in a standard cellular assay, such as the assay that measures TNF-induced IL-8 secretion by HeLa cells described herein. In particular embodiments, the antibody variable domain that unfolds reversibly neutralizes the activity of the a cytokine, growth factor, cytokine receptor or growth factor receptor with a ND50 of about or about 400 nM or less, or about 300 nM or less, or about 200 nM or less, or about 100 nM or less, or about 1 nM or less, or about 100 pM or less, or about 10 pM or less.<br>
In other embodiments, the antibody variable domain that unfolds reversibly binds a cytokine or growth factor, and inhibits the interaction of the cytokine or growth factor with a cognate cytokine receptor or growth factor receptor with an inhibitory concentration 50 (IC50) of about 1 uM or less, or about 500 nM or less, in a standard receptor binding assay, such as the assay TNF Receptor 1 (p55) assay described herein. In particular embodiments, the antibody variable domain mat unfolds reversibly inhibits the interaction of the cytokine or growth factor with a cognate cytokine receptor or growth factor receptor with IC50 of about 400 nM or less, or about 300 nM or less, or about 200 nM or less, or about 100 nM or less, or about 1 nM or less, or about 100 pM or less, or about 10 pM or less. In other embodiments, the antibody variable domain that unfolds reversibly binds a cytokine receptor or growth factor receptor, and inhibits the interaction of the cytokine receptor or growth factor receptor with a cognate cytokine or growth factor with an inhibitory concentration 50 (IC50) of about 1 uM or less, or about 500 nM or less, in a standard receptor binding assay, such as the assay TNF Receptor 1 (p55) assay<br>
described herein. In particular embodiments, the antibody variable domain that unfolds reversibly inhibits the interaction of the cytokine receptor or growth factor receptor with a cognate cytokine or growth factor with IC50 of about or about 400 nM or less, or about 300 nM or less, or-about 200 nM or less, or about 100 nM or less, or about 1 nM or less, or about 100 pM or less, or about 10 pM or less.<br>
Compositions comprising a polypeptide that unfolds teversibly, including pharmaceutical or physiological compositions are provided. Pharmaceutical or physiological compositions comprise one or more polypeptide that unfolds reversibly and a pharmaceutically or physiologically acceptable carrier. Typically, these carriers include aqueous or alcoholic/aqueous solutions, emulsions or suspensions, including saline and/or buffered media. Parenteral vehicles include sodium chloride solution, Ringer's dextrose, dextrose and sodium chloride and lactated Ringer's. Suitable physiologically-acceptable adjuvants, if necessary to keep a polypeptide complex in suspension, may be chosen from thickeners such as carboxymethylcellulose, polyvinylpyrrolidone, gelatin and alginates. Intravenous vehicles include fluid and nutrient replenishes and electrolyte replenishes, such as those based on Ringer's dextrose. Preservatives and other additives, such as antimicrobials, antioxidants, chelating agents and inert gases, may also be present (Mack (1982) Remington's Pharmaceutical Sciences, 16th Edition).<br>
The compositions can comprise a desired amount of polypeptide that unfolds reversibly. For example the compositions can comprise about 5% to about 99% polypeptide mat unfolds reversibly by weight In particular embodiments, the composition can comprise about 10% to about 99%, or about 20% to about 99%, or about 30% to about 99% or about 40% to about 99%, or about 50% to about 99%, or about 60% to about 99%, or about 70% to about 99%, or about 80% to about 99%, or about 90% to about 99%, or about 95% to about 99% polypeptide that unfolds reversibly by weight.<br>
In one example, the composition is a biological washing powder comprising a polypeptide that unfolds reversibly (e.g., a polypeptide comprising an immunoglobulin variable domain that unfolds reversibly). In another embodiment, the composition is freeze dried (lyopbilized).<br>
The invention also provides a sealed package (e.g., a sealed sterile package) comprising a polypeptide that unfolds reverslbly (e.g., when heated (e.g., a polypeptide comprising an immunoglobulin variable domain that unfolds reversibly)). In some embodiments, the sealed package further comprises a sterile instrument. In particular embodiments, the sterile instrument is a medical instrument, such as a surgical instrument.<br>
The polypeptides that unfold reversibly described herein will typically find use in preventing, suppressing or treating inflammatory states, allergic hypersensitivity, cancer, bacterial or viral infection, and autoimmune disorders (which include, but are not limited to, Type I diabetes, multiple sclerosis, rheumatoid arthritis, systemic lupus erythematosus, Crohris disease and myasthenia gravis).<br>
In the instant application, the term "prevention" involves administration of the protective composition prior to the induction of the disease. "Suppression1* refers to administration of the composition after an inductive event, but prior to the clinical appearance of the disease. "Treatment" involves administration of the protective composition after disease symptoms become manifest<br>
Animal model systems which can be used to screen the effectiveness of polypeptides that unfold reversibly in protecting against or treating the disease are available. Methods for the testing of systemic lupus erythematosus (SLE) in susceptible mice are known in the art (Knight et al. (1978) /. Exp. Med., 147:1653; Reinersten et al. (1978) New Eng. J. Med., 299:515). Myasthenia Gravis (MG) is tested hi SJI/J female mice by inducing the disease with soluble AchR protein from another species (Lindstrom et al (1988) Adv. Immunol., 42:233). Arthritis is induced in a susceptible strain of mice by injection of Type II collagen (Stuart et al. (1984) Ann. Rev. Immunol, 42:233). A model by which adjuvant arthritis is induced in susceptible rats by injection of mycobacterial heat shock protein has been described (Van Eden et al (1988) Nature, 331:171). Thyroiditis is induced hi mice by administration of thyroglobulin as described (Maron et al (1980) J. Exp. Med., 152:1115). Insulin dependent diabetes mellirus (IDDM) occurs naturally or can be induced in certain strains of mice such as those described by Kanasawa et al. (1984) Diabetohgia, 27: 113. EAE in mouse and rat serves as a model for MS in human.<br>
In this model, the demyelinating disease is induced by administration of myelin basic protein (see Paterson (1986) Textbook oflmmunopathology, Mischer et al., cds., Grune and Stratton, New York, pp. 179-213; McFarlin et al (1973) Science, 179: 478: and Satoh et al. (1987) J. ImmunoL, 138:179).<br>
The selected polypeptides of the present invention may be used as separately administered compositions or in conjunction with other agents. These can include various immunotherapeutic drugs, such as cylcosporine, methotrexate, adriamycin or cisplatinum, and immunotoxins. Pharmaceutical compositions can include "cocktails" of various cytotoxic or other agents in conjunction with the selected antibodies, receptors or binding proteins thereof of the present invention, or even combinations of selected polypeptides according to the present invention having different specificities, such as polypeptides selected using different target ligands, whether or not they are pooled prior to administration. The route of administration of pharmaceutical compositions according to the invention may be any of those commonly known to those of ordinary skill in the art. For therapy, including without limitation immunotherapy, the selected antibodies, receptors or binding proteins thereof of the invention can be administered to any patient in accordance with standard techniques. The administration can be by any appropriate mode, including parenterally, intravenously, intramuscularly, intraperitoneally, transdennally, via the pulmonary route, or also, appropriately, by direct infusion with a catheter. The dosage and frequency of administration will depend on the age, sex and condition of the patient, concurrent administration of other drugs, counterindications and other parameters to be taken into account by the clinician. A therapeutically effective amount of a polypeptide that unfolds reversibly (e,g„ an antibody variable domain that unfolds reversibly) is administered. A therapeutically effective amount is an amount sufficient to achieve the desired therapeutic effect, under the conditions of administration.<br>
The invention also provides a kit use in administering a polypeptide that unfold reversibly to a subject (e.g., patient), comprising a polypeptide that unfolds reversibly, a drug delivery device and, optionally, instructions for use. The polypeptide that unfolds reversibly can be provided as a formulation, such as a freeze dried formulation. In certain embodiments, the drug delivery device is<br>
selected from the group consisting of a syringe, an inhaler, an intranasal or ocular administration device (e.g., a mister, eye or nose dropper) a needleless injection device.<br>
The selected polypeptides of this invention can be lyophilised for storage and reconstituted in a suitable carrier prior to use. Any suitable lyophilization method (e.g., spray drying, cake drying) and/or reconstitution techniques can be employed. It will be appreciated by those skilled in the art that lyophilisation and reconstitution can lead to varying degrees of antibody activity loss (e.g., with conventional immunoglobulins, IgM antibodies tend to have greater activity loss than IgG antibodies) and that use levels may have to be adjusted upward to compensate. In a particular embodiment, the invention provides a composition comprising a lyophilized (freeze dried) polypeptide that unfolds reversibly as described herein. Preferably, the lyophilized (freeze dried) poiypeptide loses no more man about 20%, or no more than about 25%, or no more than about 30%, or no more man about 35%, or no more man about 40%, or no more than about 45%, or no more than about 50% of its activity when rehydrated. Activity is the amount of polypeptide required to produce the effect of the polypeptide before it was lyophilized. For example, the amount of rehydrated enzyme needed to produce half maximal conversion of a substrate into a product in a give time period, or the amount of a binding polypeptide needed to achieve half saturation of binding sites on a target protein. The activity of the polypeptide can be determined using any suitable method before lyophilization, and the activity can be determined using the same method after rehydration to determine amount of lost activity.<br>
Compositions containing the present selected polypeptides or a cocktail thereof can be administered for prophylactic and/or therapeutic treatments. In certain therapeutic applications, an adequate amount to accomplish at least partial inhibition, suppression, modulation, killing, or some other measurable parameter, of a population of selected cells is defined as a "therapeutically-effective dose." Amounts needed to achieve this dosage will depend upon the severity of the disease and the general state of the patient's own immune system, but generally range from 0.005 to 5.0 mg of selected antibody, receptor (e.g., a T-cell receptor) or binding protein thereof per kilogram of body weight, with doses of 0.05 to 2.0 rag/kg/dose<br>
beingmore commonly used. For prophylactic applications, compositions containing the present selected polypeptides or cocktails thereof may also be administered in similar or slightly lower dosages.<br>
A composition containing a selected polypeptide according to the present invention may he utilised hi prophylactic and therapeutic settings to aid in the alteration, inactivation, killing or removal of a select target cell population in a mammal. In addition, the selected repertoires of polypeptides described herein may be used extracorporeally or in vitro selectively to kill, deplete or otherwise effectively remove a target cell population from a heterogeneous collection of cells. Blood from a mammal may be combined extracorporeally with the selected antibodies, cell-surface receptors or binding proteins thereof whereby the undesired cells are kilted or otherwise removed from me blood for return to the mammal in accordance with standard techniques.<br>
EXEMPLIFICATION<br>
Section 1: EP- VH library and EP- VL library<br>
Error-prone PCR is a random mutagenesis strategy that introduces mutations into a DNA segment. Hence, it is a useful tool for preparing nucleic acids encoding diversified proteins that contain random ammo acid substitutions. Error-prone PCR can be carried out in a number of ways, generally by altering buffer conditions (e.g., varying dNTP concentrations) to reduce the fidelity of nucleotide incorporation. In this study, an error-prone PCR library was constructed using the GENEMORPH PCR Mutagenesis Kit (Stratagene). The kit incorporates the MUTAZYME DNA polymerase that has a high intrinsic error rate of nucleotide incorporation compared to Taq polymerase. Mutation frequency using this system can be altered by manipulating the starting concentration of template (VH and VK coding sequence) in the reaction. The starting concentration was altered accordingly to give a varying rate of mutational frequency. Initially, two libraries were constructed for each VH and VK template and cloned into a phagemid vector pR2. The VH template was V3-23/DP47 and JH4b, and the VL template was 012/02/DPK9 and JK! . The pR2 vector is derived from pHENl. (Hoogenboom, HR et al, Nucleic Acids Res. /P:4133-4137 (1991).) pR2 contains a lac promoter, a leader sequence upstream of the cloning site<br>
which is followed by His6 and VSV tags, an amber stop condon and the gene encoding the pIII phage coat protein. These consisted of one having a low mutational frequency (~lbp change/template) and the other with a medium mutational frequency (~2bp changes/template). These libraries were combined to give a diversity of 1-2 x 105.<br>
Subcloning the Error-Prone PCR Library into Phage Fd-Myc<br>
The strategy involved the PCR amplification of the error-prone PCR libraries from the pR2 phagemid vector followed by subcloning the amplified product into the phage Fd-Myc. The error-prone PCR libraries, in the pR2 phagemid in the E. coli host HB2151, were plated out to give confluent growth on large plates (22x22 cm). The estimated total number of colonies was 107 -108, ensuring that the diversity of the error-prone libraries was well covered. The colonies were scraped from the plates and the DNA of the phagemid library subsequently isolated. The isolated phagemid DNA library was used as the template for PCR amplification of the error-prone library.<br>
The library was PCR amplified using synthetic oligonucleotide primers that contained the restriction sites ApaL 1 and Not 1. Thus facilitating subcloning of the amplified products into the corresponding sites in phage Fd-Myc.<br>
VH Fd-Myc PCR Primers:<br>
5' GAG CGC CGTGCACAG GTG CAG CTG TTG 3'    (SEQ ID NO:61) 5' GAG TCG ACT TGCGGCCGC GCT CGA GAC GGT GAG 3' (SEQ ID N0:62)<br>
VK Fd-Myc PCR Primers:<br>
5' GAG CGC CGTGCACAG ATC CAG ATG ACC CAG TCT CC 3* (SEQ ID<br>
NO:63)<br>
5' GAG TCG ACT TGCGGCCGC CCG TTT GAT TTC CAC CTT GG 3' (SEQ<br>
IDNO:64)<br>
Restriction sites ApaL 1 (GTGCAC) and Not 1 (GCGGCCGC) are underlined. The primers were biotinylated at the 5' terminus. Incorporating the ApaLl site into nucleic acids encoding VH or YK causes the first amino acid of both VH and Vtc to become a glutamine.<br>
The PCR products of the amplified library were purified from an agarose gel and then restriction enzyme digested with ApaL 1 and Not 1. The digested product was purified by phenol/chloroform extraction, treated with streptavidin DYNABEADS (superparamagnetic monodisperse polymer beads; Dynal Biotech) to remove cut 5* ends and undigested product, and men subjected to QIAQU1CK PCR purification kit (Qiagen). The product was ligated into the ApaL I/Not 1 sites in phage Fd-Myc and transformed into E. coli TGI giving a library size of 106-107.<br>
Section 2: Library 3.25G<br>
Construction of Fd-myc Vector<br>
The Fd-myc vector was assembled from Fd-tet-Dogl (McCafferty et al. (Nature) 1989) by cutting the vector at ApaLl and Notl, and ligating a synthetic double-stranded DNA cassette composed of 5'-end phosphorylated oligos LJ1012 and LJ1013 (SEQ ID NO:65 and SEQ ID NO:66, respectively) which encode a myc tag and a trypsin cleavage site. The resulting vector Fd-myc is very similar to Fd-Tet-Dogl in that ApaLl and Notl sits are present for the cloning of insert in between the leader sequence and gene III. The additional feature is the presence of a myc-tag in between the Notl site and gene HI which allows for immunological detection of encoded gene M fusion protein, and also allows bound phage (e.g., selected using anti-myc antibody) to be eluted by digestion with trypsin since there is a trypsin cleavage site in the myc-tag.<br>
LJ1012:P-TGCACAGGTCCACTGCAQGAGGCGGCCGCAGAACAAAA<br>
ACTCATCTCAGAAGAGGATCTGAATTC        (SEQIDNO:65)<br>
LJ1013: P- GGCCGAATTCAGATCCTCTTCTGAGATGAGTTTTTGTTCTGCG<br>
GCCGCGAGGACGTCACCTGCTG	(SEQ ID N0:66)<br>
Preparation of the 3.25G Insert and Ligation<br>
Eleven PCRs using the Hgated DNAs of libraries pR3-7781, pR3-7782, pR3-7783, pR3-7784, pR3-7785, pR3-7786, pR3-7787, pR3-7788, pR3-7789, pR3-77890 or pR3-7791 as template were performed. These libraries are based on VH -DP47 and contain diversified CDR1, CDR2, and CDR3 (CDR3 varying from 10 to 20 atnino acids in length). The PCRs were performed on each sub-library using primers LJ1011 and LJ1027, to append an ApaLl site at the 5'-end and a Notl site at the 3'-end. The resulting amplified fragments were purified, digested consecutively with ApaLl and with Notl, re-purified and then ligated into the corresponding sites of Fd-myc.<br>
LJ1011: GAGTCGACTTGCGGCCGCGCTCGAGACGGTGACCAG     (SEQ ID<br>
NO:67)<br>
LJ1027: GAGCGCCGTGCACAGGTGCAGCTGTTGGAGTCTGGG      (SEQ ID<br>
NO:68)<br>
Electroporation of Library 3.25G and Storage<br>
After purification, the 11 ligations were pooled and electro-transformed into E. coli TGI cells. After electroporation, the cells were resuspended in 2XTY and incubated for 1 hour at 37°C for phenotypic expression. The resulting library was then plated on TYE plates supplemented with 15 fig/ml tetracycline for overnight growth, and an aliquot was taken for titration on TYE plates supplemented with 15 µg/ml tetracycline. The size of the library was 1.6 x 109 clones. Aliquots of the library were prepared by resuspending the bacteria at an OD600 of 40, diluting with an equivalent volume of glycerol (final OD600 = 20), and aliquoting as 1 ml samples that were frozen and stored at-80°C until use.<br>
Phage Production and Purification<br>
A 1 ml sample of library 3.25G was thawed and used to inoculate 500 ml of 2XTY supplemented with 15 µg/ml of tetracycline, in a 2,5 L shaker flask. The culture was incubated for 20 hours at 30°C for phage production. The cells were pelleted by centrifugation at 5,500 g for 15 min to remove bacteria. To precipitate<br>
phage, 90 ml PEG/NaCl (20 % Polyethylene glycol 8000 [Sigma; formally sold as PEG 6000], 2.5 M NaCl) were added to 450 ml of culture supernatant, and after mixing, the solution was incubated for 1 hour on ice. Phage were collected from the resulting mixture by centrifugation at 5,500 g for 30 min at 4°C. The supernatant was discarded, the tubes were re-centrifuged briefly, and the remaining PEG/NaCl was carefully removed. The pellet was resuspended in 10 ml of PBS, centrifuged at 3,300 g for 15 min to remove remaining bacteria, and then filtrated through a 0.45 µm disposable filter. Phage titer was estimated by spectroscopy: a 100-dilution in PBS was prepared and the absorbance at 260 run is measured. The phage titer (in TU per ml) was calculated using the formula: OD260 x 1013 x 2.214. The phage were stored at-20°C after adding 10-15% glycerol (final concentration).<br>
Section 3: Phage Selection<br>
Coating/Blocking of Immunotube<br>
Immunotubes (Nunc) were coated overnight (about 18 hours) at room temperature with either 4 ml of PBS containing 10 µg/ml of protein A, or 4 ml of PBS containing 10 µg/ml of protein L. In the morning, the solutions were discarded and the tubes were blocked with PBS supplemented with 2% v/v TWEEN 20 (Polyoxyethylensorbitan monolaurate; for protein A-coated irnrmmotubes) or 2% w/v BS A. (for protein L-coated immunotubes). The tubes were incubated for 1 hour at 37°C, then washed three times with PBS, before use for pbage selection.<br>
Heat Unfolding and Refolding of Fd-phage Displayed Polypeptides<br>
Approximately 5xl010 TU of domain antibody phage library was diluted into 200 µl of PBS, and aliquoted in two thin-walled PCR tubes. The tubes were then placed in a PCR apparatus for heating at 80°C for 10 minutes (temperature of cover lid: 85°C). After heating, the solutions were rapidly cooled down to 4°C in the PCR apparatus to produce refolded phage solutions.<br>
Selection of Refolded Fd-phage Displayed Polypeptides<br>
The refolded phage solutions were pooled and added to 4 ml of PBS supplemented with either 2% v/v TWEEN 20 (Polyoxyethylensorbitanmonolaurate; for selection on protein A) or 2% w/v BSA (for selection on protein L). The resulting phage solutions were added to Immunotufaes coated with protein A or Iimnunotubes coated with protein L, after sealing the tubes were rotated end-over-end 30 min at room temperature, and then held on the bench at room temperature for 1.5 hours. Unbound phage were removed by washing the tubes ten times with PBS supplemented with 0.1% TWEEN 20 (Polyoxyefhylensorbitan monolaurate), and ten times with PBS, Bound phage were eluted by adding 1 ml of PBS supplemented with trypsin (1 mg/ml) and incubating for 10-15 min white gently rotating. The solution containing the eluted phage was then transferred to a fresh microcentrifuge tube and stored on ice.<br>
E. coli Infection of Selected fd-Phage<br>
From an overnight culture of E. coli TGI cells in 2xTY at 37°C, a 100-fold dilution was prepared in 25 ml of fresh 2XTY medium, and the culture was incubated at 37°C with shaking (250 rpm) until the optical density at 600 ran (ODsoo) was 0.5-0.7 (mid-log phase). Ten milliters of this culture was then incubated with 500 µl of the eluted phage sample (the remaining 500 µl were kept at 4C) at 37°C for 30 nun without shaking to allow for phage infection. After phage infection, a 100 µl aliquot was taken for titration of the phage: 10 µl of a 1:102 dilution and 10 µlo of a 1:104 dilution in 2xTY were spotted on TYE plates containing 15 µg/ml tetracycline and grown overnight at 37°C. The titre was determined by multiplyuig the number of colonies by the dilution factor (i.e., 100 or 10,000) then multiplying by 1000 (10 µl spotted from 10 ml infected culture), to gives the titre for 500 µl of eluted phage. The total number of eluted phage was determined by multiplyuig by 2 (1 ml total eluate). The remaining infected E. coli TGI culture (9.9 ml) was transferred to a disposable 14 ml tube, and centrifuged at 3,300 g in for 10 min. The cell pellet was resuspended in 2 ml of fresh 2xTY, plated on a large 22 cm2 dish containing TYE, 15 µg/ml tetracycline, and incubated overnight at 37°C<br>
Amplification of Selected fd-phage Vectors<br>
The next day, 10 ml of 2xTY supplemented with 15% glycerol were added to the 22 cm2 dish dish, the cells were loosened using a glass spreader, and the resulting mixure was transferred to a fresh 50 ml disposable tube. Fifty microliters of the cell suspension were then used to inoculate 100 ml of 2xTY containing 15 µg/ml tetracycline, whilst 1 ml of bacterial suspension was diluted with 1 ml sterile glycerol and stored at -70°C. The 100 ml culture was grown with shaking at 37°C overnight<br>
Purification of Selected fd-phage Vectors<br>
The next day, the 100 ml overnight culture was centrifuged at 3,300 g for 15 min to remove bacteria. The supernatant was filtered through a 0.45 um disposable filter. To precipitate phage, 20 ml PEG/NaCl (20 % Polyethylene glycol 8000; Sigma [formally sold as PEG 6000], 2.5 M Nad) was added to 80 ml of supernatant, and after mixing, the solution was incubated for 1 hour on ice. Phage were collected from the resulting mixture by centrifugation at 3,300 g for 30 min at 4°C. The supernatant was discarded, the tubes were re-centriftiged briefly, and the remaining PEG/NaCl was carefully removed. The pellet was resuspended in 1 ml of PBS, then transferred to a fresh micro-centrifuge tube. Remaining bacterial debris were removed by centriging the micro-centrifuge tube at 11,600 g for 10 min. The supernatant was transferred to a fresh micro-centrifuge tube, and stored at 4°C, until the next selection round. Phage titer was estimated by spectroscopy: a 100-dilution in PBS was prepared and the absorbance at 260 nm was measured. The phage titer (in TIT per ml) was calculated using the formula: OD260 x 1013 x 2.214.<br>
Results<br>
Using this protocol and 3.25G phage library (Section 2), three rounds of selection for binding to protein A-coated immunotubes were conducted. After round 1, the phage titer was below 107 TU, whilst after the third selection round, the phage titer had risen to be greater than 109 TU. A sample of the bacterial suspension obtained from phage amplification after round 3, was serially diluted (10-fold series)<br>
and plated on TYE plates supplemented with 15 µg/ml tetracycline. After overnight incubation at 37°C, individual colonies were picked for screening (see section 4).<br>
Using this protocol and EP- VH phage library (Section 1), three selection rounds (for protein A binding) were performed to produce a high titer preparation. A sample of the bacterial suspension obtained from phage amplification after round 3, was serially diluted (10-fold series) and plated on TYE plates supplemented with 15µg/ml tetracycline. After overnight incubation at 37eC, individual colonies were picked for screening (see section 4).<br>
Using this protocol and EP- VK phage library (Section 2), three selection rounds (for protein L binding) were performed to produce a high titer preparation. A sample of the bacterial suspension obtained from phage amplification after round 3, was serially diluted (10-fold series) and plated on TYE plates supplemented with 15 µg/ml tetracycline. After overnight incubation at 37°C, individual colonies were picked for screening (see section 4).<br>
Section 4: Phage Screening 1<br>
Growing Phage Clones and Preparation<br>
A 96-well culture plate (flat bottom, with evaporation lid, Corning) was used for individual phage growth: each well was filled with 175 µl of 2xTY containing 15 µg/ml tetracycline, and inoculated with a single colony from the selected bacterial clones obtained using the Phage Selection protocol. The plate was incubated overnight (about 18 hours) with shaking at 37°C. The next day, the cells were pelleted by plate centrifugation for 20 min at 2,000 rpm at room temperature. One hundred microliters (100 µl) of each culture supernatant (containing the phage) were transferred to a fresh 96-well culture plate.  .<br>
To facilitate detection of bound phage by ELISA, phage were chemically derivatized with a biotinylated reagent. This procedure allowed bound phage to be detected using a conjugate of streptavidin and horseradish peroxydase (Str-HRP, Sigma). This strategy was chosen to decrease the possibility of cross reactivity between antibodies used to detect bound phage and immobilized protein A or protein L in the wells. Thus, each 100 µl sample of culture supernatant was reacted<br>
with 100 µl of PBS containing a 500 µM concentration of EZ-link Sulfo-NHS biotin (Perbio), for 1 hour at room temperature (with medium agitation on a rotating plate).<br>
Phage Thermodenatuiation (Heat Induced Unfolding of Phage Displayed Polypeptide)<br>
After biotinylation of the phage, 80 µl of each 200 µl sample was transferred to a THERMOWELL 96-well plate (Costar). This step was repeated with another 96-well THERMOWELL 96-well plate (Costar). The first plate was covered with a lid and placed in a PCR apparatus for incubation at 80°C during 10 min (temperature of cover lid: 85°C)., After heating, the plate was rapidly cooled down to 4°C in the PCR apparatus. The second plate was kept on ice. Both plates were then treated in parallel: to each well (containing 80 fil of heated or non-heated phage supernatant), 20 µl of PBS supplemented with 10% v/v TWEEN 20 (Polyoxyethylensorbitan monolauratejfor protein A-based ELISA) or 10% w/v BSA (for protein L-based ELISA) was added and mixed. The samples were then assayed by ELISA.<br>
Coating and Blocking of ELISA Plates<br>
MAXISORB 96-well plates (Nunc) were coated overnight (about 18 hours) at room temperature with either 100 µl of PBS containing 10 µg/ml of protein A, or 100 µl of PBS containing 10 µg/ml of protein L, per individual well. The next day, the plates were emptied, and the wells were blocked with 200 µl of PBS supplemented with 2% v/v TWEEN 20 (Polyoxyethylensorbitan monolaurate; for protein A-coated wells) or 2% w/v BSA (for protein L-coated wells). The plates are incubated for Ih at 37QC, then washed three times with PBS, before use for screening.<br>
Phage ELISA<br>
Heat treated or control phage samples were transferred to the empty wells of coated/blocked ELISA plates, and incubated for 2 hours at room temperature. Unbound phage was then removed by washing the wells 6-times with PBS. For detection, Str-HRP (from a 1 mg/ml stock in PBS) is diluted 1/2000 hi PBS supplemented with 2% v/v TWEEN 20 (Polyoxyethylensoibitan monolaurate; for<br>
piotein A-based ELISA) or 2% w/v BSA (for protein L-based ELISA), and 100 pi of this solution was added to each ELISA well. After incubation for 1 hour at room temperature, the unbound Str-HRP was removed by washing the wells 6-times with PBS. For colorimetric reaction, a 10 mg/ml solution of 1MB (tetramethylbenzidine) was diluted 100-fold in a buffer of 0.1 M sodium acetate, pH 6.0. Next, hydrogen peroxyde was added (0.4 µl per ml of buffer/TMS), and 100 µl of the resulting solution was added to each ELISA well After color (blue) had developed, the reaction was stopped by adding SO µl of 1M sulfuric acid per well (color turns to yellow). The optical density or each well at 450 nm was measured.<br>
Results<br>
The described method was used for screening phage in five separate studies;<br>
Clones from the 3.25G phage library (Section 2) selected for refolding<br>
Clones from the EP-Vn phage library (Section 1) selected for refolding<br>
Clones from the EP- VK phage library (Section 1) selected for refolding<br>
Clones from the tenmini-phage libraries in Vn-DP47d (Section 10) Clones from the five mini-phage libraries in Vic-DPK9d (Section 10) For each screening, positive and negative controls were used in order to establish the % refolding range: thus for ELISA on protein A, negative control was phage displaying VH -DP47, and positive control was phage displaying HEL4. As shown herein VH -DP47 (also referred to as VH -DP47 dummy) does not unfold reversibly when heated and cooled; while HEL4 (a single VH that binds hen egg white lysozyme that was selected from a library based on a VH 3 scaffold (DP47 gennline -f JH4 segment) with randomised CDR1,2 and 3) does unfold reversibly under these conditions. Thus for ELISA on protein L, negative control was phage displaying Vtc-DPK9 which does not unfold reversibly when heated and cooled, and positive control was phage displaying VK-DPK9-A50P (which was obtained after screening the EP VK library for VK domains that unfold reversibly when heated).<br>
For each series of phage clones, the un-heated and the heat-treated phage were tested for binding in an ELISA. This approach provides a semi-quantitative measure of refoldability of the antibody polypeptide displayed on the phage. For example, if an un-heated phage clone produces an OD450 of say 1.5 by ELJSA (1.45 when the value of the background using a blank supernatant is substracted), and the same phage clone produces an OD450 of say 0.6 by ELISA (and 0.55 when the value of the background using a blank supernatant is substracted) after heat treatment, then the "percentage refolding" (which is assimilated to the percentage of protein A-binding activity that remains after heating/cooling) of this particulaT clone is (0.55/1.45)*100 - 29.0%.<br>
Phage clones which yielded significant % refolding were then submitted to DNA sequencing of the VH gene or VK gene insert (see section 6). In addition, selected clones were submitted to a second ELISA (see section 7) to further quantify refolding.<br>
Section 5: Phage Screening 2<br>
Growing Phage Clones and Preparation<br>
Fifty milliliter disposable tubes (Coming) were used for individual pbage growth: each tube was rilled with 11 ml of 2xTY containing 15 µg/ml tetracycline, and inoculated with a single colony from the selected bacterial clones obtained using the Phage Selection protocol (Section 3) or after Pbage Screening 1 (Section 4). The tubes were incubated overnight (~18 hours) with shaking at 37°C. In the morning, the cells were pelleted by centrifugation for 20 min at 3,300 g at 4°C to remove bacteria. The supernatant was filtered through a 0.45 urn disposable filter. To precipitate phage, 2 ml PEG/NaCl (20 % Polyethylene glycol 8000 [Sigma; formally sold as PEG 6000], 2.5 M NaCl) was added to 80 ml supernatant, arid after mixing, the solution was incubated for 1 hour on ice. Phage were collected by centrifoging the resulting mixture at 3,300 g for 30 nun at 4°C, The supernatant was discarded, the tubes were recentrifiiged briefly, and the remaining PEG/NaCI was carefully removed. The pellet was resuspended in 0.2 ml of PBS, then transferred to a fresh micro-centrifuge tube. Remaining bacterial debris was removed by centrifoging the<br>
micro-centrifuge tube at 11,600 g for 10 min. The supernatant was transferred to a fresh micro-centrifuge tube, and stored on ice. Phage titer was estimated by spectroscopy: a 100-dilution in PBS was prepared and the absorbance at 260 nm was measured. The phage titer (in TU per ml) was calculated with the formula: OD260 x I013x 2.214.<br>
To facilitate detection of bound phage by ELISA, phage were chemically derivatized with a biotinylated reagent. This procedure allowed bound phage to be detected using a conjugate of streptavidin and horseradish peroxydase (Str-HRP, Sigma). This strategy was chosen to decrease the possibility of cross reactivity between antibodies used to detect bound phage and immobilized protein A or protein L in the wells. Thus, 4 x 1010 TU phage in 100 µl of PBS was reacted with 100 µl of PBS containing a 50 µM concentration of EZ-link Sulfo-NHS biotin (Perbio), for 1 hour at room temperature (with medium agitation on a rotating plate) or overnight at 4°C.<br>
Phage Thennodenaturation (Heat Induced Unfolding of Phage Displayed Polypeptide)<br>
After biotinylation of the phage, for each clone, a 100 µl sample of phage was transferred to a thin-wall PCR tube (and the remaining biotinylated phage was kept on ice). The tube was placed in a PCR apparatus for incubation at 80°C during 10 min (temperature of cover lid: 85°C). After heating, the tube was rapidly cooled down to 4°C in the PCR apparatus. Both tubes (heated sample and sample kept on ice) were then treated in parallel: first eight 4-fold dilations were prepared for both heat-treated and nonheat-treated samples of the same clone, in the appropriate buffer (PBS supplemented with 2% v/v TWEEN 20 (Polyoxyethylensorbitan monolaurate) for VH -DP47 displaying phage; or PBS supplemented with 2% w/v BSA for VK-DPK9 displaying phage. Thus, in the tubes with the highest phage concentration, the titer was 1011 TU per ml. The samples were then ready for assay by ELISA.<br>
Phage ELISA<br>
ELISA plates were coated and blocked as described in Section 4. The samples were transferred to the empty wells of coated/blocked ELISA plates, and<br>
incubated for 2 hours at room temperature. Unbound phage was then removed by washing the wells 6-times with PBS. For detection, Str-HRP (from a 1 mg/ml stock in PBS) were diluted 1/2000 in PBS supplemented with 2% v/v TWEEN 20 (Polyoxyethylensorbitan monolaurate; for protein A-based ELISA) or 2% w/v BSA (for protein L-based ELISA), and 100 µl of the resulting solution was added to each ELISA well. After incubation for 1 hour at room temperature, the unbound conjugate was removed by washing the wells 6-times with PBS. For colorunetric reaction, a 10 mg/ml solution of TMB (tetramethylbenzidme) was diluted 100-fold in a buffer of 0.1 M sodium acetate, pH 6.0. Next, hydrogen peroxyde was added (0.4 ul per ml of buffer/IMS, and 100 ul of this solution was added to each ELISA well. After color (blue) developed, the reaction was stopped by adding 50 µl of 1 M sulfuric acid per well (color turns to yellow). The optical density was recorded at 450 tun.<br>
Results<br>
This method was used for screening clones in five experiments:<br>
Clones with individual or multiple amino acid mutations in VH -DP47;<br>
Clones with individual or multiple amino acid mutations in Vtt-DPK9;<br>
Clones such as DP47, BSA1, HEM, pA-C (13,36,47,59,76,85), VK-DPK9; and<br>
A subset of clones from the ten mmi-phage libraries in VH -DP47, that showed<br>
promising % refolding.<br>
BSA1 is a single VH that binds bovine serum albumin and does not unfold reversibly when heated and cooled, that was selected from a library based on a VH 3 scaffold (DP47 germline + JH4 segment) with randomised CDRs 1,2 and 3.<br>
For each screening, positive and negative controls were used in order to establish the % refolding range: thus for ELISA on protein A, negative control was phage displaying VH -DP47, and positive-control was phage displaying HEL4, Thus for ELISA on protein L, negative control was phage displaying VK-DPK9, and positive control was phage displaying VK-DPK9-A50P.<br>
The scoring was done as follows: for each clone, the phage were tested as un-heated sample and as heat-treated sample in an eight-point dilution series. This<br>
approach permitted quantitative deductions about the refoldability of the domain antibody displayed on phage to be made. Thus, the OD450 were plotted onto a semilog graph (on the X-axis, concentration of phage (in TU) per well according to a semi-log scale; on the Y-axis, OD450 observed at each phage concentration), and linked together by simple linear interpolation between each data points.<br>
The percent refolding was calculated as illustrated in the following example. For each clone, the phage concentration that produced a particular OD450 (e.g., 0.2) was calculated (one value for the phage concentration of the non-treated sample, and one value for the heat-treated sample). Assume the concentration that produced that OD450 was 2 x 10s for the non-treated phage, and 5 x 10s for the heat-treated phage. The percent refolding would then be calculated using the formula: (2 x 108 / 5 x 109) *100 - 4% refolding. Using this system, we repeatedly observed a percent refolding of approx 0.5% for VH -DP47 on phage, and about 18% refolding for phage displaying HEIA<br>
Section 6: DNA Sequencing<br>
DMAs encoding variable domains from selected clones that display reversible heat unfolding were sequenced as follows: PCR reaction mix:<br>
5 (all Ox Buffer<br>
1 ul Primer LJ212 (20pnx&gt;yul)<br>
1 ul Primer LJ006 (20pmol/ul)<br>
1 ul 20mM dNTPs<br>
0.5 uJ Taq DNA polymerase<br>
41.5ulH20<br>
50 ul of the PCR reaction mix was aliquoted into each well of a 96 well PCR microplate. A colony (Fd-Myc/TGl) was gently touched with a sterile tooflrpick and transferred into the PCR mix. The toothpick was twisted about 5 times in the mixture. The mix was overlayed with mineral oil. The PCR parameters were: 94°C for lOmin, followed by 30 cycles of: 94°C 30sec, 50°C 30sec, 72°C 45sec; and a final incubation at 72°C 5min. The amplified samples were purified using a<br>
QIAQUICK PCR product purification kit (Qiagen). Sequencing was carried out using either of the original PCR primers (LJ212 and/or LJ006).<br>
Mutations that were detected in the sequencing of clones that encode immunoglobulin variable domains that unfold reversibly are presented in Tables 3 and 4,<br>
Primer Sequences:<br>
LJ006 5' ATGGTTGTTGTCATTGTCGQCGCA 3' (SEQ ID NO:69) LJ212 51 ATGAGGTrrTGCTAAACAACTTTC 3' (SEQ IDNO:70)<br>
Section 7: List of Selected EP VH<br>
Table 3 Mutations Found in Clones From EP- VH Library Selected for Reversible Heat Unfolding     <br>
(Table Removed)<br>
*m indicates medium ELISA signal and H indicates high ELISA signal.<br>
Table 4 Frequency of Mutation in Clones From EP- VH Library Selected for Reversible Heat Unfolding<br><br>
(Table Removed)<br>
Section 8: List of Selected EP VK<br>
Table 5 Mutations Found in Clones From EP- VK Library Selected for Reversible Heat Unfolding    <br>
(Table Removed)<br>
All clones selected gave a high ELIS A signal.<br>
Table 6 Frequency of Mutation in Clones From EP- Vk Library Selected for Reversible Heat Unfolding   <br>
(Table Removed)<br>
Section 9: List of Selected 3.25G VH<br>
After selecting library 3.25G with the heat/cool phage selection (see section 3), a large scale screening (see section 4) was performed to identify VH clones that refolded when displayed on phage after thermodenaturan'on.<br>
Clones were analyzed by assessing binding of heat treated and control phage to protein A in an ELISA as described herein. (Figs. 3A-3F) Clones that had above 60% refolding were further analyzed by sequencing. Figs. 4A-4C present die sequence of may of the clones that had above 60% refolding.- The first set of sequences present in Figs. 4A-4C are from clone giving a high QD450 in the ELISA and a high % refolding. These sequences do not contain cysteines. This group of sequences forms the dataset for the analysis of arnino acid preferences at all positions of CDR1, CDR2 and CDR3.<br>
The next set of sequences in Figs. 4A-4C are from clones with excellent ELISA signals and good refolding. These clones contain mutations outside the CDRs.<br>
The final group of sequences in Figs. 4A-4C are from with relatively low ELISA signals and/or refolding.<br>
From the sequences presented in Figs. 4A-4C, six clones were selected and further analyzed in detail on phage (as displayed polypeptide) and as soluble proteins. The six clones selected are referred to as pA-Cl3, pA-C36, pA-C47, pA-C59, pA-C76 and pA-C85. (Clones are identified in Figs. 4A-4C using the suffix only,i.e.,"PA-C13"is"C13.")<br>
In addition,* individual mutants were designed into DP47 and analyzed for refolding on phage (as displayed polypeptide) as for some of these mutations, as soluble protein. The particular mutations were: F27D, F29V, F27D/F29V, Y32E, S35G, F27D/F29V/Y32D/S35G, S53P, G54D, S53P/G54D, W47R, FlOOnV, Y102S, F100nV/Y102S, W103R.<br>
These clones were characterized forrefoldability on phage (according to the protocol of Section 5, but with three data points being taken) and hi solution. For the refolding in solution, thermodenaturation was followed by CD (Section 12), The results of these studies is presented in Table 7.<br>
Table 7    <br>
(Table Removed)<br>
* Y indicates refoldablility in solution, N indicates that the variable domain was not refoldable in solution, nd - not determined<br>
Section 10: Mini-library Protocol<br>
The positions of amino acid substitutions found in clones from the temperature selected libraries (the Error-Prone and G3.25 libraries) were analysed so that the entire sequence space at that particular site could be investigated.<br>
Error-prone PCR samples a limited sequence space. For example, the A50P substitution selected from the error-prone VK library that has a frequency of Ibp change/ template would sample only 6 additional amino acids in total. Alanine at position 50 is encoded by the codon GCT. Changing this codon by 1 base gives the following codon permutations and amino acids:<br>
Alanine	= GCT, GCA, GCG, GCC<br>
Aspartate	= GAT<br>
Glycine	= GOT<br>
Proline	= CCT<br>
Serine	= TCT<br>
Threonine	- ACT<br>
Valine	=GTT<br>
Oligonucleotides randomized at a particular site by the codon NNK. (N = A, G, C or T; K = G or T; M * A or C) were used in a PCR strategy to sample the entire sequence space (see Table 8). For VK a two-step PCR strategy was used. (See, Landt, O. et al. Gene P6":125-128 (1990)). The first PCR product (also referred to as "mega-primer") was generated using the oligonucleotide carrying the NNK. changes (see Tables 8 and 9) together with one of the forward or reverse Fd-Myc primers<br>
no-carrying the ApaL 1 or Not 1 restriction site for subcloning into Fd-Myc as described previously:<br>
VK Fd-Myc PCR Primers:<br>
5' GAG CGC CGTGCACAG ATC CAG ATG ACC CAG TCT CC 3' (SEQ ID<br>
N0:71)<br>
5' GAG TCG ACT TGCGGCCGC CCG TTT GAT TTC CAC CTT GG 3'(SEQ<br>
IDNO:72)<br>
Restriction sites ApaL 1 (GTGCAC) and Not 1 (GCGGCCGC) are underlined. The primers were biotinylated at the 5' terminus. Incorporating the ApaLl site causes the first amino acid of both VH and VK to become a glutamine.<br>
DPK9 was used as a DNA template. The first PCR product or mega-primer was subsequently used in a second PCR reaction together with the second Fd-Myc primer not used in the first reaction. This PCR gave a product containing Apall/Notl restriction sites suitable for subcloning into the respective sites in Fd-Myc.<br>
For VH, SOE PCR was employed. (See, Horton, R.M. et al. Gene 77:61-68 (1989)). The primers used to generate the substitution by SOE PCR are shown in the Table 9. The additional primers necessary for amplification of the mutagenised product and subsequent subcloning into the ApaL 1 and Not 1 sites in Fd-Myc are:<br>
5' GAG CGC CGTGCACAG GTG CAG CTG TTG 3' (SEQ ID NO:73) 5' GAG TCG ACT TGCGGCCGC GCT CGA GAC GOT GAC 3' (SEQ ID N0:74)<br>
TABLE 8 VK Oligonucleotides containing the NNK codon    <br>
(Table Removed)<br>
"Primers for positions 45,48,49 and 50 are to the reverse/antisense strand, hence the reversal of NNK to KNN, and were used in the first PCR reaction with the VK Fd-Myc primer designed to the coding strand containing the ApaL 1 site. Conversely, position 75 was designed to the coding strand and used with the antisense FD-Myc primer containing the Not 1 site.<br>
TABLE 9 VH Oligonucleotides for SOE PCR  (Table Removed)       <br>
Once assembled and digested, the PCR fragments were ligated into the ApaL l and Notl sites of Fd-myc. The ligated DNA was used to transform E. coli TGI cells by eleotroporation. After one hour of phenotypic expression at 37°C» the cells were plated on TYE plates supplemented with 15 ug/ml of tetracycline.   -113-<br>
For the screening of the mini-libraries, cultures were made in 96-well cell culture plates as described in Section 4. For the appropriate controls, three wells (usually Al, D6 and HI 1) were inoculated with a positive control phage (eg. Fd-myc-HEL4) and three wells were inoculated with a negative control phage (eg. Fd-myc-DP47d). The remaining 90 wells were inoculated with 90 different clones from a mini-library. This plate preparation procedure was repeated with each mini-library.<br>
The use of 90 clones from each mini-library is sufficient to cover the encoded diversity. Indeed, since a NNK codon was used to diversify each position, there are 32 different codons possible. The screening of 90 clones ensures that there is over 90% chance that every possible codon (and hence amino acid) will be present at least once in the screened mini-library. This ensures that the screening covers all possible amino acid substitutions at the explored (randomized) positions. The procedures forbiotinylation of phage, and detection/selection by ELISA are described in Section 4.<br>
   Several phage were obtained from the VH -mini libraries that showed good refolding. These phage were then subjected to the Phage Screening 2 procedure (Section 5) to obtain more quantitative data on the refolding.<br>
Section 11: Subcloning, Expression and Purification of dAbs<br>
Subcloning<br>
Selected substitutions were subcloned into an expression plasmid for expression in-E. coli BL21(DE3)(pLysS). DNA from selected Fd-Myc phage clones was amplified by FCR using primers containing Sal 1 and BamH 1 restriction sites. The protocol was essentially as that described for DNA sequencing, isolating VH or VK DNA from Fd-Myc/TGl colonies by PCR. The primers used arc shown in Table 10. The forward primers introduced an additional two amino acids (Ser and Thr) to the N-terminus. This is a result of creating a Sal 1 restriction site. The reverse primers were designed to have two consecutive stop codons (TAA) at the end of the coding region.    <br>
TABLE 10  Oligonucleotides for Subcloning   <br>
(Table Removed)<br>
Expression and Purification<br>
A colony, from freshly transformedKcoli strain BL21(DE3)pLysS containing the expression plasmid clone, was grown overnight in 2xTY (ampicillin/chloramphenicol) at 37°C, 250rpm. A 1/100 aliquot of the overnight culture was then used to inoculate a larger volume of the same media and allowed to grow under the same conditions until the OD6oo=0.9. ImM IPTG (isopropyl IB.D-thiogalactisidase) was then added to the culture, and the culture allowed to grow overnight at 30°C, 250rpm. The culture was then centrifuged at 3300g for 20mm at 4°C. The supernatant was then filtered through a 0.45µm filter.<br>
Soluble VH or VK dAb were then captured on a protein A or protein L matrix (protein A agarose or protein L agarose). Depending on the culture volume, the supernatant was either loaded directly onto a prepacked protein A or L matrix column, or the matrix was added directly to the supernatant (batch binding). Elution from batch binding can be accomplished directly from collected matrix, or the matrix can be packed into a suitable column. Elution from protein A or protein L matrix was carried out at low pH. The dAbs can be further purified by gel filtration using Superdex75 (Amersham Pharmacia Biotech).<br>
Section 12: CD Analysis<br>
Purified dAbs were dialyzed overnight in PBS at 4°C, and concentrated (if needed) by centrifugation using Millipore 5K Molecular Weight Cut Off<br>
centrifugation concentrator tubes (at 20 °C). One and a half ml of dAb at 1-5 µM in PBS was transferred to a CD cuvette (1 cm patblength) and introduced in the Jasco J-720 spectropolarimeter. Spectra at room temperature (25°C) or at high temperature (85°C) were recorded in the far-UV from 200 nm to 250 nm (four accumulations followed by averaging) at a scan speed of 12 nm min"1, with a 2-nm bandwith and a 1 second integration time.<br>
Heat-induced unfolding curves were recorded at fixed wavelength (usually 235 nm, sometimes at 225 nm) using a 2 run bandwith. The temperature in the cuvette was raised at a rate of 50 °C per hour, from 25 °C to 85 °C. Data were acquired with a reading frequency of 1/20 sec'1, a 1 second integration time and a 2 nm bandwith. After unfolding, the sample was rapidly cooled down to 25 °C (15 °C min"'), a spectrum was recorded, and a new heat-induced unfolding curves was recorded.<br>
The ability of a dAb molecule to unfold reversibly following to heat-induced denaturation was evaluated by comparing the first and the second heat-induce unfolding curve. Super imposable first and second heat-induced unfolding curves indicate that the dAb underwent thermal-unfoldrng reversibly in PBS at 1-5 µM (e.g., pA-C36, pA-C47). The same holds true for the far-UV spectra: super imposable first and second spectra recorded at 25 °C indicated that the dAb unfolds reversibly (Fig. 8).<br>
If a dAb aggregates upon thermal unfolding, the first unfolding curve is characterized by a steep transition upon melting and a "noisy" post-transition line (due to the accumulation of aggregates). Moreover, the second unfolding curve is radically different from the first one: a melting transition is barely detectable because no, or very few, unfolded molecules properly refold upon cooling the sample. As a result, the first and second far-UV spectra differ considerably, the latter being more akin to mat observed using a denaturated molecule. A typical example of an aggregating dAb is DP47 dummy, or DP47-W47R (Fig. 9).<br>
Some dAbs do exhibit partial refolding at 1-5 uM (e.g.t DP47-S35G): i.e., upon cooling, a proportion of the ellipticity (and hence a portion of the original secondary structure) is recovered, and a melting transition is observed upon reheating the sample (Fig, 10). To calculate the percentage of refolding, the amount of<br>
ellipticiry recovered after the first thermal denaturation is divided by the amount of ellipticity of the sample before the first thermal denaturation, and multiplied by 100.<br>
Using this assay, the following isolated human VH dAbs were shown to undergo irreversible thermal unfolding: DP47 dummy, BSA1, DP47-F29V, DP47-W47R, DP47-G54D, DP47-W103R.<br>
Fully reversible unfolding was demonstrated with the following isolated human VH dAbs: HEL4, pA-C13, pA-C36, pA-C47, pA-C59, pA-C76s pA-C85, DP47-F27D, DP47-Y32D, DP47-F27D/F29V/Y32E/S35G. While DP47-S35G demonstrate reversible unfolding to a lesser degree.<br>
Section 13: Mini-library in VK results<br>
Phage ELISA and scoring of the clones were done according to protocol of Section 4. DNA sequencing of selected clones that showed reversible heat unfolding was done according to protocol of Section 6.<br>
Table 11 Mutations Found in Clones From VK Mini Library Selected for Reversible Heat Unfolding    <br>
(Table Removed)<br>
Table 12 Substitutions at Position 45 thatGave &gt;70% Retention of Protein L Binding Activity After Heat Treatment    (Table Removed)<br><br>
"The single substitutions at K45E, I48N, Y49D/N, A50P and I75N were previously temperature selected from an error-prone phage display library (Section 1). Error-prone PCR samples a limited sequence space. Hence, these sites were randomised by NNK oligonucleotide mutagenesis so that the global sequence space was investigated.<br>
'Phage from 94 clones from each mini-library were screened for the retention of protein L binding activity after heating to 80°C for l0min. Phage were biotinylated prior to heat treatment and subsequently caputured on protein L coated plates and detected with stteptavidin HRP. Approximately 10 clones, with an activity greater than 70%, from each library were subjected to sequencing. The total number of sequences is sometimes less than 10 due to second site substitutions as well as poor sequence signals. (E.g. Multiple substitutions found giving a high retention of activity were: K45T/Q90P; K45D/S60P; Y49R/S10F; Y49S/T20A; Y49S/Q27R; A50P/I48V; A50R/A13G/K42E).<br>
Note that five clones from the A50X mini-library with an activity in the 0-10% range gave the following sequences: 2A, IT, IV, 1Y. 'Substitutions at position 175 represent partial results.<br>
Section 14; Mini-library in VH results •<br>
Phage ELISA and scoring of the clones were done according to protocol of Section 4. DNA sequencing of selected clones mat unfold reversibly when heated and coold was done according to protocol of Section 6.<br>
Table 13    <br>
(Table Removed)<br>
In the selected amino acids columns, the number in () corresponds to the number of clones carrying this mutation, that were picked as positive by the phage ELISA screening. In the Best % refolding (i.e.. protein A ELISA), the number in () corresponds to the mean % of refolding observed for all clones carrying the particular mutation.<br>
A spurious mutation (Ser25 to Pro) was found in clones carrying the following mutations: A84E, or A33 V, or A33E, or A33Q. It is possible that the S25P mutation has a positive effect on refolding, that is most likely surpassing the effect of the mutations at the intended positions.<br>
A number of positive clones were then further analyzed for refolding on phage by following the protocol described in Section 5. The results are presented in Table 14.<br>
Table 14   <br>
(Table Removed)<br>
phage or heated HEL4 phage (FIG. 16C). These clusters of phage provide direct evidence of DP47d aggregation after heat denaturation. The appearance of clusters requires both high concentration of phage and high local concentration of the dAb at the phage tip (number of dAbs displayed per tip). Western blot analysis shows that for multivalent phage ~80% of the five pin coat proteins carry a fused dAb and for monovalent phage ~20% (FIG. 16D). Thus no phage clusters were observed on heating multivalent DP47d phage at titers of Ixl09 TU/ml or monovalent DP47d phage at titers of 5xlOn TU/ml, and in both cases the binding to protein A was only reduced 8-fold and 6-fold respectively (FIG. 15B). Without wishing to be bound by any particular theory, it appears that upon phage heating to 80°C, aggregation of DP47d dAb is nucleated by the formation of a micro-aggregate at the phage tip (intra-phage step) which then grows into oligomeric aggregates by phage clustering (inter-phage step), and hence that aggregation in our phage system follows a two-step process, as noted for other proteins (Dobson, C.M. Trends Biochem Sci 24,329-332 (1999)).<br>
Heating phage displaying dAbs that do not unfold reversibly reduces infectivity<br>
Heating at 80'C slightly reduced (3-fold down) the infectivity of the HEL4 phage, as previously observed with wild-type filamentous phage (Holliger, P., et al. JMolBiol 288,649-657 (1999)), but considerably reduced the infectivity of the DP47d phage ( 3?0-fold down) (FIG. 15C). The infectivity of heated DP47d phage could be partly restored by addition of trypsin which presumably cleaves wimin the dAb and/or the peptide linker connecting the dAb to the pin protein. These observations are consistent with a model wherein intra- and inter-phage aggregates of DP47d dAb prevent the N-terminal domains of the pill protein from binding to the bacterial pilus and/or to the TolA receptor (Holliger, P., et al JMol Biol 288, 649-657 (1999)).<br>
A selection in which phage displaying the HEL4 and DP47d dAbs were mixed in 1:106 ratio respectively, incubated at 80°C for 10 min, cooled and selected on immobilized protein A was performed. Bound phage were eluted with trypsin, and used to re-infect bacteria. After two rounds of such selection, supernatants from<br>
infected colonies (n=86) were tested by ELISA on immobilized hen egg lysozyme (HEL) to distinguish the HEL-specific HEL4 phage from the DP47d phage. Phage from twelve colonies bound to hen egg lysozyme (corresponding to a 360-fold enrichment of the HEL4 phage per selection round). None of the 86 tested colonies secreted the HBL4 phage when the selection on protein A was repeated without the heat step. Thus, by two rounds of heat denaturation and biopanning on protein A, phage dAbs that resisted thermal aggregation were selected from those that did not.<br>
A repertoire of human VH dAbs (1.6xl09 clones) was prepared by diversification of the loops comprising the CDRs in the DP47d dAb, and displayed multivalently on phage. After three rounds of heat denaturation followed by selection on protein A, 179 out of 200 colonies secreted dAb phage that retained more than 80% of protein A-binding activity after heating. Twenty clones were sequenced and revealed as many unique dAb sequences with a large variability hi the CDR sequences and lengths (Table 15). The diversity shows that, as with HEL4, mutations located entirely in the loops comprising the CDRs are sufficient to confer resistance to aggregation. Eighteen of these dAbs had an acidic isoelectric point (5.1 ±1.1, meaniSD), consistent with earlier proposals of a direct correlation between net protein charge and resistance to aggregation (Wilkinson, D.L. &amp; Harrison, R.G. Nature341, 544-546 (1991); Chiti, F. etal. ProcNatlAcadSci USA. 99,16419-16426(2002)).<br>
Six dAbs with CDR3s comprising 10 to 20 amino acids were chosen for further characterization (C13, C36, C47, C59, C76, and C85). These proteins were well secreted from bacteria, and the C36, C47 and C59 dAbs were recovered in yields greater than 20 mg/L in bacterial supernatants compared to only 2.9 mg/L for the D?47d dAb (Table 16). After purification on immobilized protein A, the dAbs were subjected to size-exclusion chromatography on a SUPERDEX-75 column (gel filtration column; Amersham Biosciences). The dAbs eluted as mono-disperse symmetric peaks and the recoveries were nearly quantitative, indicating that in contrast to other human dAbs (Ewert, S., et al Biochemistry 41,3628-3636 (2002); Ewert, S., et al. JMol Biol 325,531-553 (2003)) they did not stick to the column matrix. The dAbs eluted at a mean apparent molecular mass (Mr-pp) of 17 kDa (range 10 to 22 kDa), similar to the calculated molecular weight (Mr-calc) of 13-14<br>
kDa for a monomeric dAb species (FIG. 17A). Variation, in Mr.Rpp has been observed for other dAbs (Ewert, S., el al Biochemistry 41,3628-3636 (2002); Ewert, S., et al. J Mol Biol 325, 531-553 (2003)) and may result from weak transient interactions with the column matrix or monomer/dimer equilibria (Sepulveda, J., et aL J Mol Bioim, 355-365 (2003)). Importanfly, at 5 µM, each selected dAb unfolded reversibly and cooperatively upon repeated cycles of thermodenaturation (FIG. 17B). Thus the selected dAbs not only resisted aggregation, but as reported for camel and llama dAbs (Ewert, S., et al. Biochemistry 41,3628-3636 (2002); Arbabi Ghahtoudi, M., et al. FEBS Lett 414,521 -526 (1997)), were mainly monomeric, well expressed, and purified in good yield by gel filtration.<br>
Selection on Antigen<br>
In the work described above, protein A, a generic ligand that binds each member of the repertoire, was used for selection. However, any desired antigen can be used to select dAbs that combine the properties of reversible unfolding with a desired antigen specificity. This was demonstrated by selection of a synthetic human VH repertoire for binding to human serum albumin (SSA), with and without a heat denaturation step, and followed by screening of 44 clones for binding to HSA after two rounds of selection. Without the heat step, six unique dAb clones (Table 15) that bound HSA were selected. When the heating step was employed, a single dAb clone (Clone #10, Table 15) was recovered. Only Clone #10 exhibited the properties of reversible unfolding (100% of HSA binding signal was retained after heating Clone #10 phage, compared with less than 10% retained binding signal the others) despite the close similarity in sequence to three of the other clones (#2, #5, #6 and #10 share 92% identity).<br>
Discussion<br>
Protein aggregation is an off-pathway process that competes with the folding pathway, and usually involves association of the unfolded states, or partially unfolded states. Resistance to aggregation can be achieved by introducing mutations that stabilize the native state (increasing ΔGN-u the free energy of folding) and/or that reduce the propensity of the unfolded or partially unfolded states to aggregate<br>
(for example by increasing the solubility of these states). Several selection strategies have been devised to select for protein variants with improved stability: (i) by linking the infectivity of the phage to the proteolyttc resistance of the displayed protein (Kristensen, P. &amp; Winter, G. Fold Des 3, 321-328 (1998); Sieber, V., etal Nat Biotechnol 16, 955-960 (1998); Martin, A., etal JMol Biol 309, 717-726 (2001)) and/or (ii) by challenging the displayed protein with elevated temperatures or denaturants (Shusta, E.V., etal, Nat Biotechnol 18,754-759 (2000); Jung, S-, et al. J Mol Biol 294,163-180 (1999)). The focus until now has been to destabilize (and promote elimination of) all but the most stable protein variants. For example, by heating a phage antibody library to 60°C, Jung et al. ( JMol Biol 294,163-180 (1999)) selected a variant of the 4D5Flu antibody fragment, with an improved AG$.u (+3.7 kcal/mol) and which remained folded at 60°C. Beyond this temperature, unfolding of the antibody fragment resulted in aggregation and loss of infectivity when phage displaying the antibody fragment was heated.<br>
By contrast, in the studies described herein, selection included inducing unfolding of all the dAbs in the repertoire, stable and unstable alike. This selection process operates on the ability of the unfolded dAbs to avoid irreversible aggregation at the phage tip upon heating at 80°C and cooling. The folding properties of the selected dAbs cannot be attributed to stabilization of the native state, because biophysical analysis of thennodynamic stabilities of the selected dAbs indicates that the selected domains are less stable than typical aggregation-prone human dAbs. Thus the free energies of folding (ΔGN-u) at 25°C (from 14 to 23 kJ/mol) (Table 16) are lower than those of me DP47d dAb (35 kJ/mol) and other aggregation-prone dAbs based on the same human DP47/3-23 germ-line segment (from 39.7 to 52.7 kJ/mol) (Tomlinson, LM., et al, JMol Biol 227,776-798 (1992); Ewert, S.3 et al. Biochemistry 41,3628-3636 (2002)). It appears that the described selection process using heat denaturation selects favorable properties of the unfolded or partially unfolded states.<br>
Other beneficial properties of these selected dAbs appear to follow directly from their resistance to aggregation. For example, the high level of expression obtain for these aggregation-resistant dAbs is consistent with the identification of periplasmic aggregates as the major yield-limiting factor for the production of<br>
recombinant antibody fragments in E. coli (Worn, A, &amp; Pltickthun, A. JMol Biol 305,989-1010 (2001)). In addition, the dAbs selected here were uniformly "non-sticky" upon gel filtration.<br>
The methods described herein can be used to produce improved versions of other polypeptides (e.g., polypeptides expressed in the bacterial periplasm) that can be functionally displayed on the surface of filamentous bacteriophage (e.g., in a multivalent state) and bound by a ligand mat recognizes only the properly folded state of the polypeptide (e.g., an antibody that binds properly folded polypeptide, a receptor that binds properly folded polypeptide). Such polypeptides can be diversified (e.g.t by engineering random mutations) displayed on phage, denatured, and selected by bio-panning (or other suitable methods) after returning to conditions permissive to the native state (refolding). The methods described herein also provide an analytical tool to identify amino acid residues or polypeptide segments involved hi off-pathway aggregation of proteins upon folding, including those involved hi diseases of protein misfolding (Dobson, CM. Trends Biochem Sci 24, 329-332 (1999); Rochet, J.C. &amp; Lansbury, P.T., Jr. Curr Opin Struct Biol 10,60-68 • (2000)).<br>
Table 15. Sequences of loops comprising the CDRs of dAbs described in this study   <br>
 (Table Removed)<br><br>
Table 16. Biophysical and expression data of selected dAbs    <br>
(Table Removed)<br>
1	Temperature of mid-point transition upon reversible unfolding (or aggregation for<br>
DP47ddAb).<br>
2	Thermodynamic stability value obtained from thenno-denaturation curves.<br>
3	Thermodynamic stability value obtained from urea-induced denaturation curves at<br>
25°C (unpublished data, L.J., O.S., G.W. and L.C. James).<br>
4	Obtained by integrating (he areas of the peak(s) eluted from Superdex G75,<br>
5	Yield of purified protein obtained from a 1 L supernatant- of bacterial culture<br>
normalised to 5.0 ODeoonm-<br>
6	A peak corresponding 7% of the sample migrated as multimeric species.<br>
7	nd: not determined.   <br>
Methods<br>
Phage display of a human VH dAb library. The dAb repertoire was created in two-steps by oligonucleotide-mediated diversification of several codons in the sequence of the DP47d dAb as follows (Kabat numbering for the amino acid positions and IUPAC-IUB code for the nucleotides): 27, KWT; 28, ANS; 29, NTT; 30, ANC; 31, NMT; 32, NAS; 33, DHT; 35, RSC; 50, RSC; 52, NNK; 52a, RNS; 53, WW; 54, CGT; 94, RSW; 101, NVS; 102, THT; and NNK codons for all CDR3 positions from 95 to l00x (where x ranges alphabetically from a to k). The DNA inserts were flanked with ApaLl (at 5'-end) and Notl (at 3'-end) sites by PCR, digested and ligated into the corresponding sites of fd-myc, a multivalent phage vector derived from fdCATl (McCafferty, J., et al. Nature 348,552-554 (1990)) that contains a c-myc tag between the Notl site and gene Iff. The ligation products were transformed by electroporation into E. coli TGI cells, and plated on 2xTY plates supplemented with 15 µg/ml of tetracycline (2xTY-Tet), yielding a library of 1.6xl09 clones. For monovalent display, dAb genes (as Ncol-Notl DNA fragments) were ligated into the corresponding sites of pR2, a phagemid vector derived from pHENI (Hoogenboom, H.R. et al. Nucleic Acids Res 19,4133-4137 (1991)) mat contains the (His)8 and VSV tags between the Notl site and gene HI. Phage were prepared, purified and stored as described (McCafferty, J., et al. Nature 348,552-554 (1990); Hoogenboom, H.R. et al. Nucleic Acids Res 19,4133-4137 (1991)). For the analysis of phage proteins, Ixl010 transducing units (TU) was subjected to SDS PAGE (4-12% Bis-Tris gel, Inviirogen), and transferred to a PVDF Immobilon-P membrane (Millipore) for detection; the blocked membrane was incubated with murine anti-pffl antibody (MoBiTec), then anti-murine horseradish peroxidase conjugate (Sigma-Aldrich), and electro-chemiluminescence reagents (Amersham Biosciences).<br>
Phage ELISA assays. The ELISA wells were coated overnight at 4°C with one of the following Kgands: 10 ug/ml of protein A in PBS, 10 fig/ml of mAb 9E10 in PBS, or 3 mg/ml of HEL in 0.1 M NaHC03 buffer, pH 9.6. After blocking the wells with<br>
PBS containing 2% Tween-20 (PBST), a dilution series of phage in PBST was incubated for 2h. After washing with PBS, bound phage were detected as follows. In assays for binding to HEL, phage was detected directly using a conjugate of horseradish peroxidase with an anti-Mi 3 monoclonal antibody (Amersham) using 3,3',5,5'-tetramemylt&gt;enzidine as substrate. In assays for binding to protein A, the phage (4xl010 TU/ml in PBS) was first biotinylated at 4CC with biotin-NHS (Perbio) (SO µM final concentration) and detected by sequential addition of stieptavidin-horseradish peroxidase conjugate (1 µg/ml) (Sigma-Aldrich) in PBST, and substrate as above. In assays for binding to human serum albumin (HSA) (Sigma, coating at 10 µg/ml in PBS), the ELISA was performed in PBS supplemented with 2% skim-milk powder (PBSM) and bound phage were detected in two steps using a rabbit anti-fd bacteriophage monoclonal antibody (Sigma, 1/1000 dilution) and goat anti-rabbit IgG serum conjugated with horseradish peroxidase (Sigma, 1/1000 dilution) and substrate as above.<br>
Electron microscopy. Phage (Ixl012 TU/ml in PBS) were heated at 80°C for 10 rnin and then cooled at 4°C for 10 min or left untreated as control. After dilution to Ixl010 TU/ml in PBS, phage were adsorbed on glow discharged carbon coated copper grids (SI 60-3, Agar Scientific), washed with PBS and then negatively stained with 2% uranyl acetate (w/v). The samples were studied using a FBI Tecnai 12 transmission electron microscope operating at 120 kV and recorded on film with calibrated magnifications.<br>
Phage selection. Immunotubes (Nunc) were coated overnight with protein A, and blocked with PBST. Purified phage (Ixl01' TU/ml in PBS) were heated as described above, diluted with 3 ml of PBST, and incubated for 2h in the immunotubes. After 10 washes with PBS, protein A-bound phage were eluted in 1 ml of 100 ug/ml trypsin in PBS during 10 min, then used to infect 10 ml of log-phase E. coli TGI cells at 37°C during 30 min. Serial dilutions (for phage titer) and library plating were performed on 2xTY-Tet agar plates. For the next selection round, cells were scraped from the plates and used to inoculate 200 ml of 2xTY-Tet at 37°C for phage amplification (McCafferty, J., et al Nature 348,552-554 (1990)). For selection on<br>
HSA, a similar synthetic human VH repertoire (Domantis Ltd) was used, the antigen (10 µg/ml in PBS) was coated in immunotubes, the blocking agent was PBSM, and the phage library (Ixl012 TU in 1 ml PBS) was heat-treated or left untreated before each round of biopanning.<br>
Protein expression and purification. The genes encoding for the various dAbs were subcloned into pET-l2a (Novagen) followed by the addition of  Soft saiiBamHl sites by PCR. After transformation of E. coli BL21(DE3)pLysS cells (Novagen), dAb expression (1 L scale) was induced in the presence of 1 mM IPTG (final concentration) at 30°C during 16 hours. After centrifugation, fhe supernatants were filtered (0.22 µM) and each incubated overnight with 5 mL of STREAMLINE-protein A beads (Amersham Biosciences) at 4°C. The beads were packed into a column, washed with PBS, and bound dAbs were eluted in 0.1 M glycine-HCl, pH 3.0. After neutralisation to pH 7.4, the protein samples were dialyzed in PBS and concentrated before storage at 4°C. Protein purity was estimated by visual inspection after SDS-PAGE on 12% Bis-Tris gel (Invitrogen). To obtain expression yields from normalized cultures, five individual colonies from freshly transformed bacteria were grown overnight at 37°C, and induced for expression as described above (50 ml scale). Following to overnight expression, the cultures were combined and a culture volume corresponding to 600xOD
Circular dichroism measurements. CD cuvettes (1 cm path-length) were filled with a 5 µM solution of dAb in PBS and transferred to a J-720 polarimeter (Jasco). CD spectra at 25°C and 85°C were recorded in the far-UV (200 nm to 250 run) with a 2 run bandwidth and a 1 second integration time. Unfolding curves from 25°C to 85°C were monitored at 235 nm and repeated twice for each dAb. The unfolding curves<br>
were assumed to be two-state and fitted as described (Pace, C.N. &amp; Scholtz, J.M. in Protein Structure, A Practical Approach, Edn. 2. (ed. TJE. Creighton) 299-321 (Oxford University Press, New York; 1997)) using a ΔCP contribution of 12 cal per amino acid residue (Myers, J.K., et al. Protein Sci 4,2138-2148 (1995)). The values obtained for Tm (midpoint transition temperature in Kelvin), ΔHm (enthalpy change for unfolding at Tm) were then used to calculate the thermodynamic stability (ΔGN-u) of the protein at 25°C as described (Pace, C.N. &amp; Scholtz, J.M. in Protein Structure, A Practical Approach, Edn. 2. (Ed, T.E. Creighton) 299-321 (Oxford University Press, New York; 1997)).<br>
Section 16. Introducing "folding gatekeeper" residues into single variable domain with defined specificity.<br>
A "folding gatekeeper" is an amino acid that, by the virtue of its biophysical characteristics and by its position in the primary sequence of a protein, prevents (he irreversible formation of aggregates upon protein unfolding. A folding gatekeeper residue blocks off-pathway aggregation, thereby ensuring that the protein can undergo reversible unfolding. The effectiveness of folding gatekeepers (determined by the position(s) and the biophysical characteristics) influence the maximal concentration at which the unfolded protein can remain in solution without forming aggregates. As described herein, folding gatekeepers have been introduced into single variable domains (DP47d, or VKdummy) lacking antigen-specificity.<br>
TAR2-10-27 is a human VH3 domain which has binding specificity for human tumor necosis factor receptor 1 (TNFRI, also referred to as "TAR2"). The amino acid sequence of TAR2-10-27 is:<br>
EVQLLESGGGLVQPGGSLRLSCAASGFTFEWYWMGWVRQAPGKGLEWVS AISGSGGSr^YADSVKGRFnSRDNSKNTLYLQMNSIJElAEDAAVYYCAKVK LGGGPNFGYRGQGTLVTVSS (SEQIDNO:191)<br>
The results of this rational-design approach complements another approach in which folding gatekeepers are introduced ab initio in dAb libraries displayed on phage (see Section 18).<br>
Choosing the folding gatekeepers for TAR2-10-27<br>
In the human DP-47-Vn3 domain, aggregation is mediated by the hydrophobic segment encompassing the Hi-loop. Site-directed mutagenesis and biophysical analysis have shown that even a single point mutation that lower the SE-score over residue 22 to 36 in DP-47 to below zero, can confer proper folding in this variable domain at concentration up to 5 uM. The preferred positions for single point mutations are residues Phe27 and Tyr32 and, the preferred substitutions are the hydropMlic Asp, Glu, Gin, Asn residues, 01 the beta-strand breaking residue Pro. As shown in Table 17, four single mutants in the Hl-loop of TAR2-10-27 significantly lower the SE-15 score, and may therefore confer high folding yield. Combinations of mutations would further lower the SE-15 score below zero.<br>
Table 17   <br>
(Table Removed)<br>
The F27D, F27Q, Y32D or Y32Q point mutations were introduced into the DP-47-VHS domain using SOE-PCR, and the nucleic acids encoding the mutated VH3 domains were subcloned, expressed and the encoded proteins were purified The PCR-amplified genes encoding these proteins were subcloned into a derivative of the pET-12A vector in which the OmpT leader sequence was replaced with an eukaryotic leader sequence, using the Sail and Notf restriction sites. The ligated DNAs were used to transform E. coli BL21(DE3)pLysS, which were then<br>
plated on agar plates supplemented with TYE-agar+5%glucose+100µg/ml ampicillin and incubated at 37"C overnight<br>
A l0mL 2xTY culture was inoculated with a single bacterial colony from the plates and grown overnight at 37*C. The 2xYT culture contained ampicillin (100 µg/mL and glucose (5%). A 500 mL 2xTY culture in a 2.5L culture flask was inoculated with 5 mL of the over-night culture and grown at 37°C with shaking (250 rpm) until an OD6oonm of 0.5-0.6. The media also contains ampicilin (50 µg/mL) and glucose (0.1%). When the OD600nm reached 0.5-0.6, the culture flasks were further incubated at 30 'C with shaking. After about 20 minutes, protein production was induced by the addition of ImM IPTG (final concentration) and the cultures were grown overnight.<br>
Then the culture media was transferred to cylindrical 0.5 or 1 L bottles (glass or plastic). 0.5-1 mL of a 1:1 slurry of protein A-beads (Amersham Biosciences) was added, and the mixture was rolled on horizontal flask rollers at 4°C overnight. Then the bottles were left upright allowing the resin to settle. The resin (beads) was repeatedly washed with IxPBS, 500 mM NaCl (2x), and the washed resin was loaded into a drip column.  The resin wass washed again with 5 column volumes of IxPBS, 500 mM NaCl. After the resin had almost run dry, 0.1 M glycine pH 3.0 was applied to the column and 1m1 fractions are collected. The collection tubes were filled prior to the elution with 200 µL 1M Tris, pH 7.4 in order to neutralise the solution. The OD280nm was checked for each fraction and the protein elution monitored. The fractions that contained protein were combined and dialysed against an appropriate buffer (Ix PBS in most cases). The protein concentration was again determined (using a calculated exctinction coefficient) and purity was evaluated by SDS-PAGE analysis.<br>
The protein yields per litre of culture supernatant are shown in Table 18. As previously described herein, recombinant expression of the DP-47-Vn3 domain in the pET/BL21(DE3)pLysS system was increased when folding gatekeepers were inserted into the protein sequence.<br>
Table 18<br>
(Table Removed)<br>
Aggregation of heated and cooled proteins was assessed at different concentrations. The proteins were heated at different concentrations (1-lOOjiM) for 10 minutes at 80*C. Then, die protein was allowed to refold at either room temperature or 4°C for 10 minutes. Protein A beads were added to the refolded protein solution, and the mixture was rolled until all refolded protein has been captured by the beads. The beads were washed repeatedly and equal amounts of beads were loaded onto a SDS-PAGE gel for visualisation, Densitometry was used to quantify the amount/concentration of protein recovered, and to calculate the concentration at which half the protein properly refolded after exposure to high temperature ([protein]5o%).<br>
Samples of each protein at different concentrations ranging from 1-100^M were prepared. The samples were heated for 10 minutes at 80°C in a JPCR block using 0.5 mL thin-walled tubes. The ramping was ~10*C/min. Duplicate samples were prepared and processed in parallel but omitting the heating step.<br>
After heating, the samples were kept at room temperature or 4"C for a few minutes. The samples were transferred to 1.5 mL micro tubes and centri&amp;ged at 4°C for 10 minutes at high speed. The supernatant was recovered without disturbing any pellets that formed. 20 uL of a 1:1 slurry of protein A-beads was added to each tube, the tube was topped up to ImL of PBS and rolled for 1 -2 hours on an overhead roller at room temperature. When the protein A beads had settled, the supernatant was removed without disturbing the beads. The beads were washed three times with 1 mL PBS and resuspended in 40 uL of SDS-loading buffer, including DTT. The SDS-loading buffer contained a known concentration of BSA which served as a standard to normalise for possible error the loading volume. The<br>
samples were heated at 90°C for 10 minutes, centrifuged for 10 minutes at 14,000 rpm and then equal amounts of each sample were loaded and separated on a 12% Bis/Tris SDS-NuPage gel. Equal loading, background and intensity of bands were quantified using densitometry. The densitometry data was plotted against the concentration of initial protein to produce a sigmoidal curve that was used to determine the protein concentration at which 50% of the protein refolded after heat-unfolding (tprotein]so%). The [proteinjsoy. Me shown in Table 19.<br>
Table 19  <br>
(Table Removed)<br>
Introducing folding gatekeeper mutations had a beneficial effect on TAR2-10-27 by conferring the ability to resist aggregation at higher protein concentration: 2- to 4-fold higher for TAR210-27 F27D and TAR2-10-27 Y32Q, and 7 to 25-fold higher for TAR210-27 Y32D, relative to the parental TAR2-10-27.<br>
Functional Assays<br>
TNFR1 binding activity of the VH domains was assessed in a receptor binding assay and a cell-based assay.<br>
In the receptor bindng assay, anti-TNFRl VH domains were tested for the ability to inhibit the binding of TNF to recombinant TNF receptor 1 (p55). Briefly, MAXISORP plates were incubated overnight with 30jig/ml anti-human Fc mouse monoclonal antibody (Zymed, San Francisco, USA). The wells were washed with phosphate buffered saline (PBS) containing 0.05% TWEEN-20 and then blocked with 1% BSA in PBS before being incubated with l00ng/ml TNF receptor 1 Fc fusion protein (R&amp;D Systems, Minneapolis, USA), Anti-TNFRl VH domain was mixed with TNF which was added to the washed wells at a final concentration of<br>
l0ng/ml. TNF "binding was detected with 0.2µg/ml biotinylated anti-TNF antibody (HyCult biotechnology, Uben, Netherlands) followed by 1 in 500 dilution of horse radish peroxidase labelled streptavidin (Amersham Biosciences, UK), and then incubation with 1MB substrate (KPL, Gaithersburg, USA). The reaction was stopped by the addition of HCI and the absorbance was read at 450nm. Anti-TNFRl VH domain binding activity lead to a decrease in TNF binding to receptor and therefore a decrease in absorbance compared with the TNF only control. (FIG. 18A)<br>
In the cell-based assay, anti-TNFRl VH domains were tested for the ability to neutralise the induction of EL-8 secretion by TNF in HeLa cells (method adapted from that of Akeson, L et al Journal of Biological Chemistry 271, 30517-30523 (1996), describing the induction of IL-8 by EL-1 in HUVEC). Briefly, HeLa cells were plated in microtitre plates and incubated overnight with VH domain proteins and 300 pg/ml TNF. Post incubation, the supernatant was aspirated off the ceils and the amount of IL-8 in the supernatant was measured using a sandwich ELISA (R&amp;D Systems). Anti-TNFRl VH domain activity lead to a decrease in IL-8 secretion into the supernatant compared wife the TNF only control. (FIG. 18B)<br>
The results of both assays demonstrate that TAR2-10-27 F27D and TAR2-10-27 Y32D retained biological activity. The activity of TAR2-10-27 Y32D was about equivalent to that of the parental dAb TAR2-10-27, while the activity of TAR2-10-27 was slightly reduced relative to TAR2-10-27. (FIGS. ISA and 18B) For both the receptor-binding assay and the cell based assay, IC50 values can be calculated from the sigmoidal curves with an average precision within a factor of two.<br>
Conclusion<br>
This section shows that a folding gatekeeper residue can be introduced, into a VH domain of pre-defined specificity such that the resulting variant combines increased resistance to aggregation with antigen-binding activity. This was achieved without knowing the functional epitope that is bound by the VH domain. One variant in particular (TAR2-1027-Y32D) retained full biological activity and had a 10-fold increase in resistance to aggregation ([protein]so%). Another benefit of introducing folding gatekeepers, is the increased expression level of the proteins<br>
 (e.g., by &gt;10-fold in the pET/BL2l(DE3)pLysS system). The SE-score of TAE2-10-27-Y32D (0.068) is still above 0 and above the value obtained for HEM (-0,187). Thus, if desired, additional folding gatekeepers could be introduced into TAR2-10-27-Y32D for further improvement of its biophysical properties.<br>
Section 17. Selection of an antigen-specific VH domain that contains a folding gatekeepers from a synthetic library of VH domains.<br>
Using phage display technology, antibody variable domains can be isolated from a synthetic repertoire based on binding to an antigen (e.g., by binding antigen immobilized on in a Petri dish and recovering adherent phage, "biopanning'). This approach was used to select a human VH that binds human serum albumin (HSA) from a synthetic human VH repertoire following a heat denaturation step (or without the heat denaturation step for negative control).<br>
The phage library of synthetic VH domains was the Ifbray 4G, which is based on a human VH3 comprising the DP47 germline gene and the JH4 segment. The diversity at the following specific positions was introduced by mutagenesis (using NNK codons; numbering according to Rabat) in CDR1: 30,31,33,35; in CDR2: 50, 52,52a, 53,55,56; and in CDR3:4-12 diversified residues: e.g. H95, H96, H97, and H98 in 4G HI 1 and H95, H96, H97, H98, H99, HI 00, Hl00a, Hl00b, Hl00c, Hl00d, Hl00e and H.l00f  in 4G H19. The last three CDR3 residues are FDY so CDR3 lengths vary from 7-15 residues. The library comprises &gt;lx!010 individual clones.<br>
Protocol<br>
The 4G library was divided into two sub-libraries of equal number of transducing units (Ixl011). One was used as such for panning against HSA, and the other was heat-unfolded at 80'C for 10 minutes before each panning step. For the heating step, the phage sample was dispensed as 100 µL fractions in a PCR tube. After heating, the fractions were combined. Both samples (heat treated and unheated control) were centrifuges for 10 minutes at 14,000 rpm at 4°C. The supematants were taken further into the selections on immuno-tubes. The antigen<br>
(HSA, Sigma-Aldricli) was coated in immune-tubes (at 100 ug/mL in PBS), stored at 4°C. The tubes were blocked with 2% skim-milk in PBS (2MPBS).<br>
Results<br>
After the first round of selection (Rl), approx. 3000 phage were eluted from the tubes (both from the heated phage library and from the un-heated library). After a second round of selection (R2), a confluent lawn of colonies was obtained from the selection with un-heated phage, whereas the enrichment was less pronounced for the heated phage library (approx, 2 to 5-fold increase in titer). Individual colonies (n=44) were randomly picked from both libraries (heat treated and control) after R2 and separately grown overnight to secrete phage in the supernatant (using a 96-well microtiter cell culture plate).<br>
A pbage ELISA was performed according to the protocol of Section 4, with some modifications. The microtiter wells were coated overnight with HSA (l0µg/mL in PBS) and blocked with 2MPBS. Each of the samples was washed three times with 0.1% TWEEN-20 fa PBS. The detection reagents were tabbit-anti-bacteriophage fd full IgG (Sigma) at 1:1000 dilution followed by goat-anti-rabbit conjugated with horseradish peroxidase (full IgG) (Sigma) at 1:1000 dilution. All individual clones were tested against HSA with heat-step or without heat step. They were also tested against BSA (l0ug/mL) and plastic. Only 2/37 positive signals were unspecific.<br>
As predicted, all colonies obtained from the unheated control selection secreted phage VH domains that did not sustain heat-treatment In contrast, all colonies from the heated-selection secreted phage VH domains that refold upon cooling. Thirty-six positive clones in ELISA were sequenced, and 6 unique sequences were obtained for clones from the unheated panning and 1 unique sequence was obtained from the heat-unfolding panning (clone #10) (Table 15 b) #l-#4, #6, #7 and #10).<br>
Clone 10, which has two acidic residues at positions 30 and 31 in the CDR1, exhibits the lowest SE-score (0.06) (I5 amino acid window). In contrast, the mean SE score obtained from all sequences from the unheated phage selection (Clones #1-7) was above 0.15 (was 0.203±0.020; mean±standar deviation), thereby explaining their greater propensity to aggregate. (Interestingly, clone #10 shares 92% identity<br>
with clones #2, #5, and #6, suggesting a common binding epitope on HSA for these four VH domains. But only clone #10 exhibits the refolding property and has folding gatekeepers at positions 30 and 31.<br>
The behaviour of the phage-displayed VH domains was further analysed with purified phage. A phage ELISA was performed according to the protocol of Section 5 with the following modifications. IxlO10 phage were either heat-treated (10 minutes at 80'C in 100 µL aliquots) or left untreated. The samples were centrifuged for 10 minutes at 14,000 rpm at 4°C and the supernatants were added to ELISA wells coated with HSA (with a dilution series of 1/3). After developing the ELISA, the OD450mn values (y-axis) for each heat treated/unheated phage pair were plotted against the phage number (x-axis) and the %-refoldability was calculated. The results are shown in Table 20.<br>
Table 20   <br>
(Table Removed)<br>
Conclusion<br>
The results show that the VH domain clone selected on HSA (clone #10) following a heat unfolding step is fully resistant to aggregation when expressed on. phage, whereas none of the VH domain clones selected without the heat unfolding step exhibited greater than 10% refolding after heating. This study demonstrates that antigen-specific variable domains that unfold reversibly upon heating can be selected from a repertoire of aggregation-prone VH domains (&gt;98% of the clones from the primary library 4G have been shown to aggregate upon heating).<br>
Section 18. Creation of phage libraries of synthetic VH domains that include folding gatekeeper residues.<br>
Section 16, demonstrated that a VH domain with good folding properties can be isolated from a repertoire of aggregation-prone VH domains in which the CDR1 loop was diversified and hence, a small percentage of the VH domains in the primary repertoire had folding gatekeepers in the CDR1 loop. This section describers the preparation of a phage repertoire in which the majority of VH domains unfold reversibly.<br>
Methods and results<br>
Diversity was introduced by randomly combining DNA fragments encoding diversified CDR1, CDR2 and CDR3 using assembly PCR. The resulting fragments were cloned into a phage vector (Fd-myc in which the leader sequence was replaced with an eukaryotic leader sequence; see Section 2), to yield a primary library (VH-6G). The VH library is based on a single human framework (V3-23/DP-47 and JH4b). The canonical structures (V~n: 1-3) encoded by this frameworks are the most common in the human antibody repertoire. Side chain diversity was incorporated using NNK diversified codons at positions in the antigen binding site that make contacts with antigen in known structures, and are highly diverse in the mature repertoire. Diversity was incorporated at the following positions (Kabat numbering):<br>
VH CDR1: H30, H31, H32, H33, H35.<br>
VH CDR2: H50, H52, H52a, H53, H55 and H56.<br>
VH CDR3:4-12 diversified residues: e.g. H95, H96, H97, and H98 in 4G<br>
HI 1 and H95, H96, H97, H98, H99, H100, Hl00a, Hl00b, Hl00c, Hl00d,<br>
Hl00e and Hl00f in 4G H19. The last three CDR3 residues are FDY so<br>
CDR3 lengths vary from 7-15 residues.<br>
The primary library was then used to produce phage stock. The phage stock was split in 2: one stock was used as such for panning on immobilised protein A (which binds correctly folded VH domains). Bound phages were eluted, used to infect E. coti and library DNA was prepared ftom the pooled infected cells (DNA pool "E").<br>
The second stock was first heated at 80'C for 10 minutes, and subsequently panned on protein A. The library of phage propagated after panning was enriched for VH domains that refold after heat denataration. The DNA pool obtained from these phages is referred to as DNA pool "H".<br>
Individual colonies from the primary library and from the selected libraries were separately grown for phage production. The phages were tested for their ability to bind to protein A (to assess production as well as correct folding) when untreated or after the heat treatment at 80°C. The results are shown in Table 21.<br>
Table 21   <br>
(Table Removed)<br>
Next a DNA fragment comprising the pooled CDR1 and CDR2 regions was amplified from DNA pool H. From DNA pool E, only the pooled CDR3s were amplified. Finally, a pool of CDR3s was PCR amplified from a naive passage (nP) with the 6G primary library. The naive passage pool was produced by growing phage from the primary library, purifying the phage by precipitation, infecting E. coli with the purified phage and recovering DNA from the bacteria. Using SOE-PCR, the pools of CDR1 and CDR2s were randomly recombined with the pool of CDR3s, to recreate full VH domain genes. The following combinations were prepared (Table 22).<br>
Table 22   (Table Removed)<br><br>
The CDR1 and CDR2s were systematically taken from the DNA pool H, because the CDR1 segment contains the mam determinants of aggregation. In contrast, the CDR3s were from the 'E' and 'nP' pools, which are less important for aggregation.<br>
After SOE-PCR, the fragments were cloned into the phage vector (Fd-myc in which the leader sequence was replaced with an eukaryotic leader sequence; see Section 2), and used to transform E. coli by electroporation in order to obtain large repertoires (&gt;1010 clones). Individual colonies (n=44) from the 6G-H+E and 6G-H-f-nP VH libraries were separately grown for phage production. The phages were tested for their ability to bind to protein A (to assess production as well as correct folding) when untreated or after the heat treatment at 80°C. The results are shown in Table 23.<br>
Table 23.  <br>
(Table Removed)<br>
About 85% of the phage that displayed a functional VH domain containing heat selected CDR1 and CDR2 loops, retained protein A binding activity after the heating step. Thus most of the VH domains in library VH-6G include folding gatekeepers. In contrast, only 12% of the VH domain clones in the unoptimised library 4G (VH-4G-H20) resist aggregation upon heating and retain protein A binding activity. These results demonstrate that adding a single heating step at the library clean-up stage (on protein A), resulted in enrichment of the VH domain library (by 7-fold) with DNA segments encoding VH domains that contain folding gatekeepers and that unfold reversibly upon heating.<br>
Fourteen randomly picked clones from libraries VH-6G-H+E and VH-6G-H+nP were picked and sequenced. The sequences of the loop comprising the CDR1 are listed below as well as their SE-score (using a window of 15 amino acids). The diversified amino acids are underlined.<br>
Clone	Sequence	SE-score<br>
1.	CAASGFTFQYOTMSW(SEQIDNO:197)	0.152<br>
2.	CAASGFTFAADNMDW (SEQ ID NO: 198)	-0.074<br>
3.	CAASGFITFMMSW(SEQIDNO:199)	-0.015<br>
4.	CAASGFTFGNA1MDW(SEQIDNO:200)	-0.041<br>
5.	CAASGnFEDDLMNW (SEQ ID N0:201)	-0.021<br>
6.	CAASGFTFGGD£MTW(SEQIDNO:202)	-0.061<br>
7.	CAASGFTFDDSTMaW(SEQIDN0503)	-0.075<br>
8.	CAASGFTFDNSYMGW(SEQIDNO:204)	0.046<br>
9.	CAASGFTFDQIAMHW(SEQIDNO:205)	-0.021<br>
10.	CAASGFTFPELPMGW (SEQ ID NO:206)	0.105<br>
11.	CAASGFTFTPGKMNW (SEQ ID NO-.207)	0.013<br>
12.	CAASGFTFEB^TMGW(SEQIDNO:208)	-0.011<br>
13.	CAASGFnTNgDMVW (SEQ ID N0:209)	0.058<br>
14.	CAASGFTFGKSTMAW (SEQ ID N0:210)	0.044<br>
Overall, the mean (±SD) SE-score for this region of the 14 Clones is 0.007 (±0.018) which is well below the mean SE-score of VH dummy (DP47d) (0.190) and the mean SB score (0.220) calculated for the encoded diversity (NNK codons) for that region. The diversity at each position was relatively large: 7,11,8,10 and 9 different ammo acids were counted in the dataset, at position 30,31,32,33 and 35, respectively. Strongly hydrophilic and 0-sheet breakers such as Asp, Glu, Pro and Gly accounted for 43% of the amino acids (versus 19% based on the encoded genetic diversity).<br>
Conclusion<br>
A synthetic library of human VH3 domain was created by recombination of PCR fragments and displayed on the surface of filamentous phage. A majority of the VH3 dAbs (85%) unfolded reversibly upon beating at 80'C. This library was produced by (1) diversification of the CDR1 region (at position 30,31,32,33 and 35), (2) heat-treatment of the primary phage library before clean-up on protein A,<br>
and (3) PCR-mediated recombination of heat-selected CDRl-CDR2-encoding genes and naive passage- (or simply protein A-cleaned-up) CDR3-encoding genes. In the resulting library, the CDR1 sequences were diverse, but enriched for folding gatekeeper residues such as Asp, Glu, Pro and Gly.<br>
Section 19. Additional studies introducing folding gatekeepers into dAbs with predefined specificity<br>
This section describes the implementation of folding gatekeepers into a dAb with pre-defined specificity. TAR1-5-19 is a VK1 domain which is specific for TNF-α. Its sequence is:<br>
DIQMTQSPSSLSASVGDRVTITCRASQSIDSYLHWYQQKPGKAPKLLrYSASE LQSGVPSRFSGSGSGTDFTLTISSLQPEDFATYYCQQVVWRPFTFGQGTKVEI KR (SEQIDNO:211)<br>
The results of this rational-design work complements another approach wherein 'folding gatekeepers' are introduced ab initio in dAb libraries displayed onphage.<br>
Choosing the folding gatekeepers for TAR1-5-19:<br>
As described and demonstrated herein, folding gatekeepers can be introduced into the region comprising the HI loop of DP47d, or the region comprising FR2-CDR2 of VK dummy (DPK9). For TAR1-5-19, we selected position 49, a solvent-exposed residue at the boundary between FR2 and CDR2. It should be noted mat in the absence of structural and mutagenesis data on the dAb, mutations within (or close to) the paratope may be likely to affect binding activity.<br>
As shown in Table 28, two single mutants in PEP1 -5-19 lower the SE-15 score (using the Sweet-Eiseriberg (SE) scale over a window of 15 amino acids -Sweet R.M. &amp; Eiseriberg D., J. Mol BioL 171:479-488 (1983)), and therefore indicate that the mutant polypeptide would have high folding yield. It should be also noted mat within that 15 aa region, the SE-score of PEP1-5-19 is already fairly low.<br>
Table 28   <br>
(Table Removed)<br>
Aggregation assay of phage-displayed TAR1-5-19 and mutants thereof:<br>
Using the methods described in Section 5, Phage Screening 2, monoclonal phages displaying TAR1-5-19 and variants were heated for 10 min at 80°C or left un-heated, and then incubated in TNF-a coated wells for ELISA. The % refolding data were calculated as described in Section 5, Phage Screening 2, and the results are shown in Table 30.<br>
Table 30  <br>
(Table Removed)<br>
The folding gatekeepers have a beneficial effect on TAR1-5-19 refolding, and the variants that contained folding gatekeepers resisted aggregation when heated to 80°C even when displayed on phage. However, it should be noted that parental TAR1-5-19 also showed good refolding ability due to its low SB-score (0.011).<br>
Receptor binding assay of TAR1-5-19 and mutants thereof:<br>
The activity assay was a RBA-ELISA (receptor-binding assay where increasing amount of dAb is used to compete with soluble TNF-a binding to immobilised TNF- α-receptor. Bound TNF-α was then detected with a biotinylated non-competing antibody, and streptavidin-HRP conjugate. The IC50 (in nM) is the dAb dose require to produce a 50% redu<br>
+-aq1-+22ction of a non-saturating amount of TNF-α to the receptor. The results are shown in Table 31.<br>
(Table Removed)<br>
Here, the introduction of the folding gatekeeper into TAR1-5-19 affected the in-vitro activity of TAR1-5-19. This suggests that residue Y49 in TAR1-5-19 may be involved (directly or indirectly) in specific contact with TNF-o. For TAR1-5-19, Y49 does not appear to be the best choice for introducing a folding gatekeeper (although the mutants are still moderately active). Other positions within the IS amino acid segment centered around Y49 could be considered to achieve a reduction of the SE-score without reducing the activity of the dAb. For example, TAR1-5-19 variants in which 148 is replaced with P, D, T, G or N, Y49 is replaced with S, C, E, G, K, N or R and/or 175 is replaced with N or M can be produced and characterized as described herein. Such a TAR1-5-19 variant can readily be produce using the methods demonstrated herein. Conclusion<br>
In this study, folding gatekeepers were introduced into a dAb of pre-defined specificity, without prior knowledge of structure of the antigen-dAb complex. Binding of variants of TAR1-5-19 was reduced by 6- to 20-fold, but the variants still bound the target with moderate activity, demonstrating that the variants have biological activity. It should be noted that only a two single point mutants were created and studied. Further benefits of introducing folding gatekeepers would be realized by creating and screening additional variants that contain single point mutations or combinations of mutations, for example by positional diversification or by library generation.<br>
Table 24 Sweet-Eisenberg (S/E) values of DP47- VH (with window of 11 amino acids)<br>
(Table Removed)<br>
Table 25 Sweet-Eisenberg (S/E) values of DPK9- VK (with window of 11 amino      acids)      Table 26 Sweet-Eisenberg (S/E) values of DP47- VH (with window of 15 amino acids)<br>
  (Table Removed)<br>
Table 27 Sweet-Eisenberg (S/E) values of DPK9- VK (with window of 15 amino acids)   <br>
(Table Removed)<br>
While this invention has been particularly shown and described with references to preferred embodiments thereof, it will be understood by those skilled in the art that various changes in form and details may be made therein without departing from the scope of the invention encompassed by the appended claims.    <br><br><br><br><br><br><br><br><br>
WE CLAIM:<br>
1.	A process for recovering polypeptide, an immunoglobin variable domain<br>
characterized in that it unfolds reversibly from a repertoire of polypeptides,<br>
wherein the said polypeptide in the repertoire that unfold reversibly have a<br>
common selectable characteristic that distinguishes folded polypeptides from<br>
unfolded or misfolded polypeptides, and wherein said common selectable<br>
characteristic is selected from the group consisting of binding to a generic<br>
ligand, binding to a target ligand, a catalytic activity, resistance to proteolysis<br>
and an epitope presented on the displayed polypeptides when folded, but<br>
absent from misfolded or unfolded polypeptides, the process comprising<br>
unfolding substantially all polypeptides in a polypeptide display system maintained at atmospheric pressure that comprises said repertoire of polypeptides at a concentration 1 micro molar to 1 milli molar by increasing the temperature of said polypeptide display system to an unfolding temperature of at least 80°C (Ts) for 100 milli second to 10 hours, and cooling the polypeptide display system to a temperature (Tc) that is lower than Ts, whereby a portion thereof aggregate, and refolding at least a portion of the remaining unfolded pol5^eptides by reducing the temperature of said polypeptide display system to a refolding temperature that is at least 10°C lower than said unfolding temperature at pH 1 to 13; and<br>
recovering at a temperature (Tr) the polypeptide that unfolds reversibly and has said selectable characteristic from the refolded portion and wherein the recovered polypeptides, has a melting temperature (Tm), wherein Ts&gt;Tm&gt;Tc and Ts&gt;Tm&gt;Tr, wherein Tc is substantially the same as Tr, and Tm is greater than 37°C and less than 60°C.<br>
2.	The process as claimed in claim 1, wherein said polypeptide display<br>
system is a phage display system.<br><br>
3.	The process as claimed in any one of the preceding claims, wherein optionally each displayed polypeptide comprises an antibody variable domain.<br>
4.	The process as claimed in any one of the preceding claims, wherein said pol3^eptide display system is bacteriophage display and the polypeptide that unfolds reversibly is recovered by recovering infective bacteriophages or the progeny of infective bacteriophages.<br>
5.	The process as claimed in any one of the preceding claims, wherein the polypeptide display system is a multivalent bacteriophage display system in which each displayed polypeptide comprises an antibody variable domain.<br>
6.	The process as claimed in any one of the preceding claims, wherein said polypeptide display system comprises a plurality of replicable genetic display packages.<br>
7.	The process as claimed in claim 2, wherein said phage display system is a multivalent phage display system.<br>
8.	The process as claimed in any one of the preceding claims, wherein said repertoire comprises at least about 103 members.<br>
9.	A polypeptide produced by the process as claimed in any one of claims 1 to 8, comprising an immunoglobulin variable domain, wherein said immunoglobulin variable domain has binding specificity for a target ligand and unfolds reversibly when heated to a suitable unfolding temperature, and is secretable when expressed in E. coli, wherein said variable domain is a human immunoglobulin heavy chain variable domain (VH), a human immunoglobulin light chain variable domain (VL) and said polypeptide consists of a single VH, with the proviso that said variable domain does not contain one of more<br><br>
framework amino acids that are unique to Camelid immunoglobulin variable domains encoded by germline sequences.<br>
10.	The process as claimed in any one of claims  1  to 8, wherein each displayed polypeptide consists of a single antibody variable domain.<br>
11.	The process as claimed in claim 10, wherein said single antibody variable domain is a human single antibody variable domain.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1ERUxOUC0yMDA1LUFic3RyYWN0LSgxMS0wMi0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">5170-DELNP-2005-Abstract-(11-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1ERUxOUC0yMDA1LUFic3RyYWN0LSgzMC0wMy0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">5170-DELNP-2005-Abstract-(30-03-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1kZWxucC0yMDA1LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">5170-delnp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1ERUxOUC0yMDA1LUFzc2lnbm1lbnQtKDExLTAyLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">5170-DELNP-2005-Assignment-(11-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1ERUxOUC0yMDA1LUNsYWltcy0oMTEtMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">5170-DELNP-2005-Claims-(11-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1ERUxOUC0yMDA1LUNsYWltcy0oMzAtMDMtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">5170-DELNP-2005-Claims-(30-03-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1kZWxucC0yMDA1LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">5170-delnp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1kZWxucC0yMDA1LWNvbXBsZXRlIHNwZWNpZmljYXRpb24gKGdyYW50ZWQpLnBkZg==" target="_blank" style="word-wrap:break-word;">5170-delnp-2005-complete specification (granted).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1ERUxOUC0yMDA1LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTEtMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">5170-DELNP-2005-Correspondence-Others-(11-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1kZWxucC0yMDA1LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">5170-delnp-2005-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1kZWxucC0yMDA1LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">5170-delnp-2005-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1ERUxOUC0yMDA1LURyYXdpbmdzLSgxMS0wMi0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">5170-DELNP-2005-Drawings-(11-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1kZWxucC0yMDA1LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">5170-delnp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1ERUxOUC0yMDA1LUZvcm0tMS0oMzAtMDMtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">5170-DELNP-2005-Form-1-(30-03-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1kZWxucC0yMDA1LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">5170-delnp-2005-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1kZWxucC0yMDA1LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">5170-delnp-2005-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1ERUxOUC0yMDA1LUZvcm0tMi0oMzAtMDMtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">5170-DELNP-2005-Form-2-(30-03-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1kZWxucC0yMDA1LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">5170-delnp-2005-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1ERUxOUC0yMDA1LUZvcm0tMy0oMTEtMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">5170-DELNP-2005-Form-3-(11-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1kZWxucC0yMDA1LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">5170-delnp-2005-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1kZWxucC0yMDA1LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">5170-delnp-2005-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1ERUxOUC0yMDA1LUdQQS0oMTEtMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">5170-DELNP-2005-GPA-(11-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1kZWxucC0yMDA1LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">5170-delnp-2005-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1ERUxOUC0yMDA1LU90aGVycy1Eb2N1bWVudC0oMTEtMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">5170-DELNP-2005-Others-Document-(11-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1kZWxucC0yMDA1LXBjdC0xMDEucGRm" target="_blank" style="word-wrap:break-word;">5170-delnp-2005-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1kZWxucC0yMDA1LXBjdC0xMDUucGRm" target="_blank" style="word-wrap:break-word;">5170-delnp-2005-pct-105.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1kZWxucC0yMDA1LXBjdC0xMDYucGRm" target="_blank" style="word-wrap:break-word;">5170-delnp-2005-pct-106.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1kZWxucC0yMDA1LXBjdC0yMDIucGRm" target="_blank" style="word-wrap:break-word;">5170-delnp-2005-pct-202.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1kZWxucC0yMDA1LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">5170-delnp-2005-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1kZWxucC0yMDA1LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">5170-delnp-2005-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1kZWxucC0yMDA1LXBjdC0yMjQucGRm" target="_blank" style="word-wrap:break-word;">5170-delnp-2005-pct-224.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1kZWxucC0yMDA1LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">5170-delnp-2005-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1kZWxucC0yMDA1LXBjdC0zMDEucGRm" target="_blank" style="word-wrap:break-word;">5170-delnp-2005-pct-301.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1kZWxucC0yMDA1LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">5170-delnp-2005-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1kZWxucC0yMDA1LXBjdC00MDEucGRm" target="_blank" style="word-wrap:break-word;">5170-delnp-2005-pct-401.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1ERUxOUC0yMDA1LVBldGl0aW9uLTEzNy0oMTEtMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">5170-DELNP-2005-Petition-137-(11-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE3MC1ERUxOUC0yMDA1LVBldGl0aW9uLTEzOC0oMTEtMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">5170-DELNP-2005-Petition-138-(11-02-2009).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="233826-a-process-for-preparation-of-an-arylamine.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="233828-fitting-for-providing-a-fluid-tight-seal-and-method-of-forming-said-seal.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>233827</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>5170/DELNP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>18/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>01-May-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>15-Apr-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>10-Nov-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>DOMANTIS LIMITED</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>315 CAMBRIDGE SCIENCE PARK, CAMBRIDGE CB4 0WG, ENGLAND.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>GREGORY P. WINTER</td>
											<td>H3 GREAT COURT, CAMBRIDGE CB2 1TQ, ENGLAND.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>LAURENT S. JESPERS</td>
											<td>14 ROBERT MAY CLOSE, CAMBRIDGE CB1 3UH, ENGLAND.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>PHILIP C. JONES</td>
											<td>15 IMPINGTON LANE, IMPINGTON, CAMBRIDGE CB4 9LT, ENGLAND.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>KRISTOFFER H.J. FAMM</td>
											<td>TRINITY COLLEGE, CAMBRIDGE CB2 1TQ, ENGLAND.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/13</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/GB2004/002102</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-05-14</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/470,340</td>
									<td>2003-05-14</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/554,021</td>
									<td>2004-03-17</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/233827-a-process-for-recovering-polypeptide-an-immunoglobin-variable-domain-that-unfolds-reversibly-from-a-repertoire-of-polypeptides by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:55:15 GMT -->
</html>
