<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/233950-small-exported-protein-c-terminal-apf-domain by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:59:20 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 233950:&quot;SMALL EXPORTED PROTEIN C-TERMINAL (APF) DOMAIN&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;SMALL EXPORTED PROTEIN C-TERMINAL (APF) DOMAIN&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention relates to a peptide and a nucleic acid molecule and to uses thereof for producing heterologous proteins, pro-biotic organisms and functional food components and products..</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Field of the invention<br>
The invention relates to a peptide and a nucleic acid molecule and to uses thereof for producing heterologous proteins, pro-biotic organisms and functional food components and products.<br>
Background of the invention<br>
Lactic acid bacteria, such as Lactobacillus and Lactococcus, and other Gram-positive bacteria such as Bifidobacterium, Leuconostoc and Streptococcus, are widely used for manufacturing food products and for the fermentation of raw agricultural products.<br>
As these bacteria tend to be harmless and tend to remain viable in the intestinal environment, there is now interest in using these bacteria to produce heterogenous proteins (i.e. proteins that are not naturally produced by the bacteria), especially for use in manufacturing functional food products that provide beneficial health effects, and also in the manufacture of new bio-pharmaceutical products.<br>
In view of these potential applications of lactic acid bacteria, there is a need for molecules that can be expressed on the cell surface of Gram-positive bacteria such as Lactobacillus, Lactococcus, Bifidobacterium, Leuconostoc and Streptococcus, or for example, in a culture supernatant derived from these bacteria. Further, there is a need for molecules that can be expressed in lactic acid bacteria together with a heterogenous protein in the form of a fusion protein.<br><br>
  Statement of invention:<br>
Accordingly, the present invention relates to a peptide comprising an amino acid sequence represented by SEQ ID NO. 2 or a variant thereof, and optionally comprising one or more amino acid sequences selected from SEQ ID NO. 1, SEQ ID NO. 3 and SEQ ID NO. 4.<br><br>
Description of the invention<br>
The invention seeks to address the above described need and accordingly, in one aspect, the invention provides a peptide including a LysM domain at the N-tenninus of the peptide, an apf-like domain at the C-terminus of the peptide and a glulamine rich region that is arranged between the LysM and apf-like domains.<br>
As described herein, the inventors have isolated and characterised a novel bacterial peptide that is expressed on the cell surface of Lactobacillus fermentum. The<br><br><br>
peptide is also secreted from the cell surface and has a high relative abundance in culture supernatant, suggesting that it is relatively stable when secreted into solution. This protein has been named "Small Exported Protein" or "Sep".<br>
In view of the stable expression of Sep on the cell surface and in culture supernatant, the inventors recognised that Sep would be particularly useful for targeting expression of heterologous proteins to the cell surface of bacteria that are preferred for use in the preparation of functional food components, especially components produced from Gram-positive bacteria such as Lactobacillus, Lactococcus, Bifidobacterivm, Leuconostoc and Streptococcus.<br>
LysM domains, otherwise known as a "lysin motif domains" have been observed in enzymes capable of binding to proteoglycan such as transglycosylases derived from E. co/i. An example of a LsyM domain is that found in tytic murein transglycosylase D (MUD) of K coli (Bateman, A., and M. Bycroft 2000. The structure of a LysM domain from E. cott membrane-bound lytic murein transglycosylase D (MltD). J. Mol. Biol. 299:1113-1119).<br>
The LysM domain of the peptide of the invention is typically about 40 to 50 amino acids in length, although it may have fewer or more residues. Typically, the LysM domain has a sequence shown in SEQ ID No: 1.<br>
Apf-like domains, otherwise known as "aggregation-promoting factor domains" are understood to have a role in the attachment of proteins to the bacterial cell wall. Examples of apf domains include those found in the apfl and apf 2 proteins of L. joknsonni and L. gasseri (Ventura, M., I. Jankovic, D.C. Walker, R.D. Pridmore, and R. Zink. 2002. Identification and characterization of novel surface proteins in Lactobacillus johnsonii and Lactobacillus gasseri. Appl. Environ. Microbiol. 68:6172-6181).<br>
The apf domain of the peptide of the invention is typically about 80 amino acids in length, although it may have fewer or more residues. Typically, the apf domain has a sequence shown in SEQ ID No:2.<br><br>
The glutamine rich region of the peptide of the invention typically has about 13 glutamine residues in a sequence having about 44 residues. This region is typically hydrophilic. Typically the glutamine rich region has a sequence shown in SEQ ID No:3.<br>
The peptide of the invention may further include a secretion signal sequence, otherwise known as a "leader sequence". The secretion signal sequence has a role in the secretion of the peptide through the cell membranes, so that tile peptide may be attached to the cell surface and/or secreted from the cell surface, for example, into a liquid culture. Typically the secretion signal sequence has about 30 amino acids in length, although it may have fewer or more residues. Typically, the secretion signal sequence has a sequence shown in SEQ ID No: 4.<br>
Typically the peptide of the invention has the sequence shown in SEQ ID NO: 5. Where the peptide farther includes a secretion signal sequence, the peptide typically has the sequence shown in SEQ IDNo:6.<br>
The inventors recognise that the secretion signal sequence, LysM domain, glutamine rich region and apf domain each have utility as separate functional units, for example in the expression of heterologous proteins. Examples of these utilities are described further below.<br>
For example, the secretion signal sequence is particularly useful for targeting the expression of a heterologous protein to the cell surface of Gram-positive bacteria such as Lactobacillus, Lactococcus, Bifidobacterium, Leuconostoc and Streptococcus Accordingly, the secretion signal sequence is particularly useful for producing, for example, functional food components that contain a heterologous protein of interest, and in particular, a protein that is not naturally expressed by these bacteria.<br>
Thus in one aspect, the invention provides a peptide including the sequence shown hi SEQ ID No: 6.<br>
As LysM domains have been shown to bind to proteoglycan, it is recognised that the LysM domain may be particularly useful for binding bioactive compounds that include carbohydrate, for example, for the purpose of concentrating bioactive compounds at a site of interest, such as intestinal mucosal epithelium. Alternatively, the LysM<br><br>
domain may be particularly useful for removing bioactives that include carbohydrate, such as pathogenic bacteria, from a site of interest, such as intestinal mucosal epithelium.<br>
Thus in one aspect, the invention provides a peptide including the sequence shown in SEQ ID No: 1.<br>
As apf domains are understood to have a role in the attachment of some S-layer proteins to the bacterial cell wall, the apf domain may be particularly useful for attaching a heterologous protein to the cell surface of Gram-positive bacteria such as Lactobacillus, Lactococcus, Bifidobacterium, Leuconostoc and Streptococcus Accordingly, the apf domain may be useful for producing, for example, functional food components that contain a protein of interest.<br>
Thus in one aspect, the invention provides a peptide including the sequence shown in SEQ ID No: 2.<br>
As the inventors have found that the gnnamine rich region of the peptide is particularly hydrophilic, they recognise that mis region may be very useful in a chhneric protein or fusion protein (described farmer herein) for spacing hydrophobic domains of a fusion protein apart, thus improving the functionality of each hydrophobic domain. Thus the inventors envisage mat the glutamine rich region will have particular utility in the expression of heterologous proteins by Gram-positive bacteria such as Lactobacillus, Lactococcus, Bifidobacterium, Leuconostoc and Streptococcus.<br>
Thus in one aspect, the invention, provides a peptide including the sequence shown in SEQ ID No: 3.<br>
It will be understood that the peptide of the invention may include one or more of the sequences shown in SEQ ID No:l, 2, 3 and 4.<br>
The peptide of the invention is typically about 175 amino acid residues in length, although it may include more amino acid residues. When the peptide is attached to the secretion signal sequence, it is typically about 205 amino acid residues in length.<br>
The inventors recognise that a peptide that includes a sequence that, but for one or more amino acid residues, is essentially the same as the sequence shown in SEQ ID No: 5, would be expected to have a capacity to be expressed either on the surface of Gram-<br><br>
positive bacteria such as Lactobacillus, Lactococcus, Bifidobacterium, Leucanostoc and Streptococcus, or in a culture supernatant derived therefrom. These peptides could be made according to the processes described further herein. The capacity of these peptides to be expressed on the cell surface or secreted from the cell surface, for example, into a culture supernatant, could be determined by the assays described further herein.<br>
In view of the above, it will be understood that the invention includes peptides that have an amino acid sequence that is homologous to the sequence shown in one of SEQ ID Nos: 1, 2, 3, 4, 5 and 6. These peptides are referred to as 'Variants". Further to amino acid sequence homology wife one of the sequences of SEQ ID Nos: 1, 2, 3, 4, 5 and 6, the variants are characterised in terms of a capacity to be expressed either on the surface of Gram-positive bacteria such as Lactobacillus, Lactococcus, Bifidobacterium, Leuconostoc and Streptococcus, or in a culture supernatant derived therefrom, as determined by the assays described herein.<br>
"Homology" with respect to amino acid sequence is defined as the percentage of amino acid residues in a candidate sequence that are identical with the amino acid residues of one of the sequences of SEQ ID Nos 1, 2, 3, 4, 5 and 6, after aligning the sequences and introducing gaps if necessary to achieve the maximum identity. No N- or C-terminal extension or deletion in the candidate sequence shall be construed as reducing homology. An example of an algorithm for aligning sequences is CLUSTAL W.<br>
Typically a variant is a peptide that has for example, at least about 75% amino acid homology with one of the sequences of SEQ ID Nos 1, 2, 3, 4, 5 and 6. The variant may have at least 80%, more typically, greater than 85% sequence homology, for example, 90% amino acid homology, with one of the sequences of SEQ ID Nos 1,2,3,4, 5 and 6. However, a variant may exhibit less than 50% sequence homology with the sequence of SEQ ID Nos 1, 2,3, 4,5 and 6 and still retain the characteristics of a variant as described herein.<br>
As described herein, peptides of the invention, including variants, may be prepared by chemical synthesis methodologies or by recombinant DNA technology. For example, peptides of the invention can be prepared from monomers using a chemical synthesis methodology based on the sequential addition of amino acid residues, for<br><br>
example as described in Merrifield, J. Am. Chem. Soc., 85: 2149 (1963). These monomers may be naturally occurring residues, or non naturally occurring residues, examples of which are described below. Alternatively, the peptides of the invention, and in particular, a variant, can be prepared enzymatically or chemically by treating a peptide including the sequence shown in one of SEQ ID Nos: 1, 2, 3, 4, 5 or 6. Where the peptides are to be synthesised by recombinant DMA technology, they may be prepared by random or pre-determined mutation (eg site directed PCR mutagenesis) of a nucleic acid molecule that encodes an amino acid sequence shown in one of SEQ ID Nos: 1,2, 3,4, S or 6, or a sequence that has homology with one of the sequences of SEQ ID Nos: 1,2,3, 4, 5 and 6, and expression of the sequence in a host cell to obtain the peptide. This is a particularly useful process for preparing variants. An alternative process is de navo chemical synthesis of a nucleic acid molecule that encodes one of the sequences of SEQ ID Nos: 1, 2, 3, 4, 5 and 6 or a sequence that is homologous to one of the sequences of SEQ ID Nos: 1, 2, 3, 4, 5 and 6 and expression of the sequence in the host cell to obtain the peptide.<br>
The peptides of the invention mat are variants of one of the sequences of SEQ ID Nos: 1,2, 3, 4, 5 and 6, typically differ in terms of one or more conservative amino acid substitutions from mese sequences. Examples of conservative substitutions are shown in Table 1 below.<br>
Table Removed<br><br><br>
As noted above, thepeptides of the invention may mctade mm naturaUy occurring amino acid residues. Commonly encountered amino acids which are not encoded by the genetic code, include:<br>
2-amino adipic acid (Aad) for Ghi and Asp;<br>
2-aminopimelic acid (Apm) for Glu and Asp;<br>
2-aminobutyric (Abu) acid for Met, Leu, and other aliphatic amino acids;<br>
2-aminoheptanoic acid (Ahe) for Met, Leu and other aliphatic amino acids;<br>
2-aminoisobutyric acid (Aib) for Gly;<br>
cyclohexylalanine (Cha) for Val, and Leu and He;<br>
homoarginine (Har) for Arg and Lys;<br>
2,3-diaminopropionic acid (Dpr) for Lys, Arg and His;<br>
N-ethylglycine (EtGly) for Gly, Pro, and Ala;<br>
N-ethylasparigine (EtAsn) for Asn, and Ghi;<br>
Hydroxyllysine (Hyl) for Lys;<br><br>
aUohydroxyllysine (AHyf) for Lys;<br>
3-<and hydroxyprokne for pro ser and thr></and>
alloisoleucine (Alle) far lie, Leu, and Val;<br>
p-amidinophenylalanine for Ala;<br>
N-methylglycine (MeGly, sarcosine) for Gly, Pro, Ala.<br>
N-methylisoleucine (Melle) for Be;<br>
Norvaline (Nva) for Met and other aliphatic amino acids;<br>
Norleucine (Nle) for Met and other aliphatic amino acids;<br>
Ornithine (Om) for Lys, Arg and His;<br>
Citrulline (Cit) and methionine sulfoxide (MSO) for Thr, Asn and GLn;<br>
N-methyiphenylalanine (MePhe), trimetbylphenylalanine, halo (F, CL Rr and I) phenylalfinine, trifk&gt;arylpheiivlalanine, forPhe.<br>
A useful method for identification of a residue of one of the sequences shown in SEQ ID Nos: 1, 2, 3, 4 , 5 or 6 for amino acid substitution to generate a variant is called «lanin« scanning mutageneras as described by Cunningham and Wells (1989) Science, 244:1081-1085. Here a residue or group of target residues are identified (eg charged residues such as Glu, Asp, Asn, Gin and Lys) and replaced by a neutral or negatively charged amino acid to affect the interaction of the amino acids with the surrounding environment. Those domains demonstrating functional sensitivity to the substitution then are refined by introducing further or other variations at or for the sites of substitution. Thus while the site for introducing an amino acid sequence variation is predetermined the nature of the mutation per se need not be predetermined. For example, to optimize the performance of a mutation at a given site, Ala scanning or random mutagenesis may be conducted at the target codon or region and the expressed peptide screened for the optimal combination of desired activity.<br>
Phage display of protein or peptide libraries offers another methodology for the selection of peptide with improved or altered affinity, specificity, or stability (Smith, G,<br><br>
P, (1991) Curr Opin Biotechnol (2:668-673). High affinity proteins, displayed in a monovalent fashion as fusions with the M13 gene IH coat protein (Clackson, T, (1994) et al, Trends Biotechnol 12:173-183), can be identified by cloning and sequencing the corresponding DNA packaged in the phagemid particles after a number of rounds of binding selection.<br>
The peptides of the invention may be prepared as the free acid or base or converted to salts of various inorganic and organic acids and bases. Such salts are within the scope of this invention. Examples of such salts include ammonium, metal salts like sodium, potassium, calcium and magnesium; salts with organic bases like dicyclohexylamine, N-methyl-D-glucamine and the like; and salts with amino acids like arginine or lysine. Salts with inorganic and organic acids may be likewise prepared, for example, using hydrochloric, hydrobromic, sulfuric, phosphoric, trifluoroacetic, methanesulfonic, malic, maleic, ramaric and the like. Non-toxic and physiologically compatible salts are particularly useful, although other less desirable salts may have use in the processes of isolation and purification.<br>
The peptide may include at least one carbohydrate molecule and/or at least one lipid molecule.<br>
The peptide may include at least one alkyi group.<br>
One particular application of the peptides of the invention is their use to provide fusion proteins that permit expression of a heterologous peptide on Gram-positive bacteria such as Lactobacillus, Lactococcus, Bifidobacterium, Leuconostoc and Streptococcus on the surface of these bacteria, or in culture supernatant derived therefrom.<br>
Fusion proteins can be made by the chemical synthesis methods described below, or they can be made by recombinant DNA techniques, for example, wherein a nucleic acid molecule encoding the peptide having a sequence shown hi one of SEQ ID Nos: 1,2, 3, 4, 5 or 6 is arranged in a vector with a gene encoding a heterologous protein. Expression of the vector results in the peptide of the invention being produced as a fusion with the heterologous protein.<br><br>
Three broad classes of fusion proteins are contemplated. The first class is that wherein the fusion protein includes a peptide having a sequence shown in one of SEQ ID Nos: 1, 2, 3,4, 5 or 6 and a heterologous protein, wherein the heterologous protein is an antibody fragment or another high affinity molecule. These fusion proteins may have application as follows: in binding to and inactivating microbial toxins; binding to and blocking pathogenic microbe colonisation determinants such as fimbriae, non-fimbrial adhesins or other cell surface molecules involved in the virulence process; binding to and blocking host molecules that serve as receptors for pathogenic microorganisms; directly killing microorganisms; binding to cancer cells for the purpose of docking a chemotherapy compound; as in-vitro diagnostic reagents for use in e.g. ELISA assays; as in-vivo diagnostic reagents i.e. visualisation of a diagnostic target in a living body; or as irmnunohistochemistry reagents. Examples of heterologous proteins within this class include those described in: Kruger C, Hu Y, Pan Q, Marcotte H, Hultberg A, Dehvar D, van Dakn PJ, Pouwels PH, Leer RJ, Kelly CG, van Dollenweerd C, Ma JK, Hammarstrom L m situ delivery of passive immunity by lactobacilh' producing single-chain antibodies. Nat Biotechnol. 2002 JuU20(7):702-6; Oggkxri MR, Barinati C, Bocctnera M, Medaghni D, Spinosa MR, Maggj T, Conn' S, Magliani W, De Bcarnardis F, Ten' G, Cassone A, Pozzi 6, Polonclli L. Recombinant Strq&gt;tococcus gordonii for mucosal delivery of a scFv nricrotncidal antibody, fat Rev hnmunol. 2001;20(2):275-87; Souriau C, Hudson PJ. Recombinant antibodies for cancer diagnosis and therapy. 2003 Expert Opin Biol Ther Apr;3(2):305-18; Ross JS, Gray K, Gray GS, Worland PJ, Rolfe M. Anticancer antibodies. Am J Clin Pathol. 2003 Apr;119(4):472-85; Kreitman RJ. Recombinant toxins for the treatment of cancer. Curr Opin Mol Ther. 2003 Feb;5(l):44-51.<br>
The second class is that wherein the fusion protein includes a peptide having a sequence shown in one of SEQ ID Nos: 1, 2, 3, 4, 5 or 6 and a heterologous protein, wherein the heterologous protein is a lysin, such as a phage lysin. These are particularly useful for specifically killing bacterial cells by disrupting the cell wall. Examples of lysins within mis class include those described in:<br>
Fischetti VA. Novel method to control pathogenic bacteria on human mucous membranes Ann N Y Acad Sci. 2003 Apr;987:207-14; Schuch R, Nelson D, Fischetti VA.A bacteriolytic agent that detects and kills Bacillus anthracis. Nature. 2002 Aug<br><br>
22;418(6900):884-9; Loeffler JM, Fischetti VA Synergistic lethal effect of a combination of phage lytic enzymes wim different activities on penicillin-sensitive and -resistant Streptococcus pneumoniae strains. Antiniicrob Agents Chemother. 2003 Jan;47(l):375-7; Gaeng S, Scherer S, Neve H, Loessner MJ. 2000. Gene cloning and expression and secretion of Listeria manocytogenes bacteriophage-lytic enzymes in Lactococcus lactis. Appl Environ Microbiol 66:2951-8.<br>
The third class is that wherein the fusion protein includes a peptide having a sequence shown in one of SEQ ID Nos: 1, 2, 3, 4, 5 or 6 and a heterologous protein, wherein the heterologous protein is capable of eliciting a protective immune response against infection and disease. Examples of heterologous proteins within this class includes those described in: B.Smith DJ, King WF, Barnes LA, Peacock Z, Taubman MA Immunogenicity and protective immunity induced by synthetic peptides associated with putative immunodominant regions of Streptococcus m"*qii,«' ghican-binding protein. Infect Immun. 2003 Mar,71(3):l 179-84; Olive C, Clair T, Yarwood P, Good MF. Protection of mice fiom group A streptococcal infection by mtranasnl immunisation wim a peptide vaccine mat contains a conserved M protein B cell epitope and lacks a T cell autoepitope. Vaccine. 2002 Jun 21;20(21-22):2816-25; Souza Femandes RC, Sousa de Macedo Z, Medina-Acosta E Expression and purification of the recombinant enteropathogenic Escherichia coli vaccine candidates BfpA and EspB. Protein Expr Purif. 2002 Jun;25(l): 16-22; Pal S, Davis HL, Peterson EM, de hi Maza LM Immunization with the Chlamydia trachomatis mouse pneumonitis major outer membrane protein by use of CpG oligodeoxynucleotides as an adjuvant induces a protective immune response against an intranasal chlamydial challenge. Infect Immun. 2002 Sep;70(9):4812-7.<br>
It will be understood that the invention contemplates the use of a peptide having a sequence shown in one of SEQ ID Nos: 1,2, 3, 4, 5 or 6 with any heterologous protein, irrespective of the function of me heterologous protein. Particularly useful are heterologous proteins involved in the induction of immune tolerance and other modifications of immune system function, and the direct inhibition of pathogen binding using a non-immunoglobulin protein. Specific examples include a fusion protein having a sequence shown in one of SEQ ID Nos: 1, 2,3,4, 5, or 6 and (i) the der pi antigen from<br><br>
the house dust mite; or (ii) the irrterleukin 1 receptor antagonist; or (iii) cyanovirin N; and interleukin 2 or y interferon together with a heterologous protein to be used as a vaccine.<br>
Fusion proteins can be cleaved using chemicals, such as cyanogen bromide, which cleaves at a methionine, or hydroxylamine, which cleaves between an Asn and Gty residue. Using standard recombinant DNA methodology, the nucleotide base pairs encoding these arnino acids may be inserted just prior to the 5' end of the gene encoding the desired peptide.<br>
Alternatively, one can employ proteolytic cleavage of fusion protein, see for example Carter in Protein Purification: From Molecular mechanisms to Large-Scale Processes, Ladisch et al., eds. (American Chemical Society Symposium Series No. 427, 1990), Ch 13, pages 181-193.<br>
Proteases such as Factor Xa, thrombin, and subtilisin or its mutants, and a number of others have been successfully used to cleave fusion proteins. Typically, a peptide linker that is amenable to cleavage by the protease used is inserted between the former proteins (e.g., the Z domain of protein A) and the peptide of the invention. Using recombinant DNA methodology, the nucleotide base pairs encoding the tinker are inserted between the genes or gene fragments coding for the other proteins. Proteolytic cleavage of the partially purified fusion protein containing the correct linker can then be carried out on either the native fusion protein, or the reduced or denatured fusion protein.<br>
The peptide of the invention may not be properly folded when expressed as a fusion protein. Also, the specific peptide linker containing the cleavage site may or may not be accessible to the protease. These factors determine whether the fusion protein must be denatured and refolded, and if so, whether these procedures are employed before or after cleavage.<br>
When denaturing and refolding are needed, typically the peptide is treated with a chaotrope, such as guanidine HC1, and is then treated with a redox buffer, containing, for example, reduced and oxidized dithiothereitol or glutathione at the appropriate ratios, pH, and temperature, such mat the peptide is refolded to its native structure.<br><br>
Other fusion proteins of the invention include those wherein the peptide of the invention is fused to a protein having a long half-life such as immunoglobulin constant region or other immunoglobulin regions, albumin, or ferritin.<br>
Examples of ways of making fusion proteins are described further herein.<br>
The peptides of the invention described above can be made by chemical synthesis or by employing recombinant DNA technology. These methods are known in the art. Chemical synthesis, especially solid phase synthesis, is preferred for short (e.g., less than SO residues) peptides or those containing unnatural or unusual amino acids such as D-Tyr, Ornithine, amino adipic acid, and the like. Recombinant procedures are preferred for longer peptides. When recombinant procedures are selected, a synthetic gene may be constructed de novo or a natural gene may be mutated by, for example, cassette mutagenesis. These procedures are described further herein. Set form below are exemplary general procedures for chemical synthesis of peptides of the invention.<br>
Peptides are typically prepared using solid-phase synthesis, such as that generally described by Merrifield, J. Am. Chan. Soc., 85: 2149 (1963), although other equivalent chemical syntheses known in the art are employable. Solid-phase synthesis is initiated from the C-terminus of the peptide by coupling a protected a-amino acid to a suitable resin. Such a starting material can be prepared by attaching a a-amino-protected amino acid by an ester linkage to a chloromethylated resin or a hydroxymethyl resin, or by an amide bond to a BHA resin or MBHA resin. The preparation of the hydroxymethyl resin is described by Bodansky et al., Chem. Ind. (London), 38: 1597-1598 (1966). Chloromemylated resins are commercially available from BioRad Laboratories, Richmond, Calif. And from Lab. Systems, Inc. The preparation of such a resin is described by Stewart et al., "Solid Phase Peptide Synthesis" (Freeman &amp; Co., San Francisco 1969), Chapter 1, pp. 1-6. BHA and MBHA resin supports are commercially available and are generally used only when the desired polypeptide being synthesized has an unsubstituted amide at the C-terminus.<br>
The amino acids are coupled to the peptide chain using techniques well known in the art for the formation of peptide bonds. One method involves converting the amino acid to a derivative that will render the carboxyl group more susceptible to reaction with<br><br>
the free N-terminal amino group of the peptide fragment For example, the amino acid can be converted to a mixed anhydride by reaction of a protected amino acid with ethychloroformate, phenyl chloroformate, sec-butyl chloroformate, isobutyl chloroformate, pivaloyl chloride or like acid chlorides. Alternatively, the amino acid can be converted to an active ester such as a 2,4,5-trichlorophenyl ester, a pentachlorophenyl ester, a pentafluorophenyl ester, a p-nitrophenyl ester, a N-hydroxysuccinimide ester, or an ester formed from 1-hvdroxybenzotriazole.<br>
Another coupling method involves use of a suitable coupling agent such as N.N1-dicyclohexylcarbodiimide or NJSf'-diisopropylcarbodiimide. Other appropriate coupling agents, apparent in those skilled in the art, are disclosed in E Gross &amp; J Meienhofer, The Peptides: Analysis, Structure, Biology, Vol. I: Major Methods of Peptide Bond Formation (Academic Press, New York, 1979).<br>
It should be recognized that the ot-amino group of each amitm acid employed in the peptide synthesis must be protected during the coupling reaction to prevent side reactions involving their active a-amino function. It should also be recognized mat certain amino acids contain reactive side-chain functional groups (eg sulfhydryl, amino, carboxvl, and hydroxyi) and that such functional groups must also be protected with suitable protecting groups to prevent a chemical reaction from occurring at that site during bom the initial and subsequent coupling steps. Suitable protecting groups, known in the art, are described in Gross and Meienhofer, The Peptides: Analysis, Structure, Biology, Vol. 3: "Protection of Functional Groups in Peptide Synthesis" (Academic Press, New York 1981).<br>
In the selection of a particular side-chain protecting group to be used in synthesizing the peptides, the following general rules are followed. An a-amino protecting group must render the o-amino function inert under the conditions employed in the coupling reacting, must be readily removable after the coupling reaction under conditions that will not remove side-chain protecting groups and will not alter the structure of the peptide fragment, and must eliminate the possibility of racemization upon activation immediately prior to coupling. A side-chain protecting group must render the side chain functional group inert under the conditions employed in the coupling reaction,<br><br>
must be stable under the conditions employed in removing the ot-amino protecting group, and must be readily removable upon completion of the desired atnino acid peptide under reaction conditions mat will not alter the structure of the peptide chain.<br>
It will be apparent to those skilled in the art that the protecting groups known to be useful for peptide synthesis will vary in reactivity with the agents employed for their removal. For example, certain protecting groups such as triphenylmethyl and 2-(p-biphenylyl)isopropyloxycarbonyl are very labile and can be cleaved under mild acid conditions. Other protecting groups, such as t-butyloxycarbonyl (BOQ, t-amyioxycarbonyl, adamantyloxycarbonyl, and p-methoxybenzyloxycarbonyl are less labile and require moderately strong acid, such as trifluoroacetic, hydrochloric, or boron trifluoride in acetic acid, for their removal. Still other protecting groups, such as benzyioxy-carbonyl (CBZ or Z), halobenzyloxycarbonyl, p-nitrobenzyloxycarbonyl cycloalkyloxycarbonyl, and isopropytoxycarbonyl, are even less labile and require stronger acids, such as hydrogen fluoride, hydrogen bromide, or boron trifluoroacetate in trifluoroacetic acid, for their removaL Among the classes of useful ammo acid protecting groups are included:<br>
(1)	for an o-amino group, (a) aromatic urethane-type protecting groups, such as<br>
fluorenyhnethyloxycarbonyl (FMOQ CBZ, and substituted  CBZ,  such as, eg, p-<br>
chlorobenzyloxycarbonyl, p-6-nitrobenzyloxycarbonyl, p-bromobenzyloxycarbonyl, and<br>
p-methoxybenzyloxycarbonyl,	o-chlorobenzyloxycarbonyl,	2,4-<br>
dichlorobenzyloxycarbonyl, 2,6-dichlorobenzyloxycarbonyl, and the like; (b) aliphatic<br>
urethane-type protecting groups, such as BOC, t-amyloxycarbonyl, isopropyloxycarbonyl,<br>
2-(p-biphenylyl)-isopropyloxycarbonyl, allyloxycarbonyl and the like; (c) cycloalkyl<br>
urethane-type protecting groups, such as cyclopentyloxycarbonyl, adamantyloxycarbonyl,<br>
and cyclohexyloxycarbonyl; and (d) allyloxycarbonyl. The preferred o-amino protecting<br>
groups are BOX or FMOC.<br>
(2)	for the side chain amino group present in Lys, protection may be by any of the<br>
groups mentioned above hi (1) such as BOC, p-chlorobenzyloxycarbonyl, etc.<br><br>
(3)	for the guanidino group of Arg, protection may be by mitro, tosyi, CBZ,<br>
adamantyloxycarbonyl, 2^,5,7,8-pentamethylchroman-6-sulforiyl or 2,3,6-trimethyl-4-<br>
methoxvphenylsulfonyl, or BOC.<br>
(4)	for toe hydroxyl group of Ser, Thr, or Tyr, protection may be, for example, by<br>
C1-C4 alkyl, such as t-butyl; benzyl (BAL); substituted BZL, such as p-methoxybenzyl,<br>
p-nitrobenzyl, p-chlorobenzyl, o-chlorobenzyl, and 2,6-dichlorobenzyl.<br>
(5)	for the carboxyl group of Asp or Glu, protection may be, for example, by<br>
esterification using groups such as BZL, t-butyl, cyclohexyl, cyclopentyl, and the like.<br>
(6)	for the imidazole nitrogen of His, the tosyl moiety is suitable employed.<br><br>
(7)	for the phenolic hydroxyl group of Tyr, a protecting group	such as<br>
tetrahydropyranyl,   tert-buryl,   trityl,   BZL,   chlorobenzyl,   4-bromobenzyl,	or   2,6-<br>
dichlorobenzyl   is   suitably   employed.   The  preferred   protecting   group	is   2,6-<br>
dichlorobenzyl.<br>
(8)	for the side chain amino group of Asn or Gb, xanthyi (Xan) is preferably<br>
employed.<br>
(9)	for Met, the amino acid is preferably left unprotected.<br>
(10)	for die thio group of Cys, p-methoxybenzyl is typically employed.<br>
The C-terminal amino acid, eg, Lys, is protected at the N-amino position by an appropriately selected protecting group, in the case of Lys, BOC. The BOC-Lys-OH can be first coupled to the benzyhydrylamine or chloromethylated resin according to the procedure set forth in Horiki et al, Chemistry Letters, 165-168 1978) or using isopropylcarbodiimide at about 25°C for 2 hours with stirring. Following the coupling of the BOC-protected amino acid to the resin support, the o-arnino protecting group is removed, as by using trifluoroacetic acid (TFA) in methylene chloride or TFA alone. The deprotection is carried out at a temperature between about 0°C and room temperature. Other standard cleaving reagents, such as HCI in dioxane, and conditions for removal of specific oamino protecting groups are described in the literature.<br><br>
After removal of the o-amino protecting group, the remaining o-amino and side-chain protected amino acids are coupled stepwise within the desired order. As an alternative to adding each amino acid separately in the synmesis, some may be coupled to one another prior to addition to the solid-phase synthesizer. The selection of an appropriate coupling reagent is within the skill of the art Particularly suitable as a coupling reagent is NJ^-dicyclohexyl carbodiimide or diisopropylcarbodiimide.<br>
Each protected amino acid or amino acid sequence is introduced into the solid-phase reactor in excess, and the coupling is suitably carried out in a medium of dimemylformamide (DMF) or CHzCl2 or mixtures thereof. If incomplete coupling occurs, the coupling procedure is repeated before removal of the N-amino protecting group piror to the coupling of the next amino acid. The success of the coupling reaction at each stage of the synmesis may be monitored. A preferred memod of monitoring the synthesis is by the ninhydrin reaction, as described by Kaiser et aL, Anal Biochem, 34: 595 (1970). The coupling reactions can be performed automatically using well known methods, for example, a BIOSEARCH 9500™ peptide synthesizer.<br>
Upon completion of the desired peptide sequence, the protected peptide must be cleaved from the resin support, and all protecting groups must be removed. The cleavage reaction and removal of the protecting groups is suitably accomplished simultaneously or stepwise. When the resin support is a chloromethylated polystyrene resin, the bond anchoring the peptide to the resin is an ester linkage formed between the free carboxyl group of the C-terminal residue and one of the many chloromethyl groups present on the resin matrix. It will be appreciated that the anchoring bond can be cleaved by reagents that are known to be capable of breaking an ester linkage and of penetrating the resin matrix.<br>
One especially convenient method is by treatment with liquid anhydrous hydrogen fluoride. This reagent not only will cleave the peptide from the resin but also will remove all protecting groups. Hence, use of this reagent will directly afford the fully deprotected peptide. When the chloromethylated resin is used, hydrogen fluoride treatment results in the formation of the free peptide acids. When the benzhydrylamine resin is used, hydrogen fluoride treatment results directly in the free peptide amines. Reaction with<br><br>
hydrogen fluoride in the presence of anisole and dimethylsulfide at 0°C for one hour will simultaneously remove the side-chain protecting groups and release the peptide from the resin.<br>
When it is desired to cleave the peptide without removing protecting groups, the protected peptide-resin can undergo methanolysis to yield the protected peptide-resin can undergo methanolysis to yield the protected peptide in which the C-terminal carboxyl group is methylated. The methyl ester is then hydrolysed under mild alkaline conditions to give the free C-terminal carboxyl group. The protecting groups on the peptide chain men are removed by treatment with a strong acid, such as liquid hydrogen fluoride. A particularly useful technique for methanolysis is that of Moore et al, Peptides, Proc Fifth Amer Pept Symp, M Goodman and J Meienhofer, Eds, (John Wiley, N.Y., 1977), p.518-521, in which the protected peptide-resin is treated with methanol and potassium cyanide in the presence of crown ether.<br>
Another method of cleaving the protected peptide form the resin when the chloromefcylated resin is employed is by ammonorysis or by treatment with hydrazme. If desired, the resulting C-terminal amide or hydrazide can be hydrolysed to the free C-tenninal carboxyl moiety, and Hie protecting groups can be removed conventionally.<br>
It will also be recognized that the protecting group present on the N-tenninal o-amino group may be removed preferentially either before or after the protected peptide is cleaved from the support.<br>
If in the peptides being created carbon atoms bonded to four non identical substituents are asymmetric, then the compounds may exist as disastereoisomers, enantiomers or mixtures thereof. The syntheses described above may employ racemates, enantiomers or disastereoisomers as starting materials or intermediates. Disastereomeric products resulting from such syntheses may be separated by chromatographic or crystallization methods. Likewise, enantiomeric product mixtures may be separated using the same techniques or by other methods known in the art. Each of the asymmetric carbon atoms, when present, may be in one of two configurations (R. or S) and both are within the scope of the present invention.<br><br>
Purification of the peptide is typically achieved using conventional procedures such as preparative HPLC (including reversed phase HPLC) or other known chromatographic techniques such as gel permeation, ion exchange, partition chromatography, affinity chromatography (including monoclonal antibody columns) or counter-current distribution.<br>
As described above, the peptide of the invention may be prepared as salts of various inorganic and organic acids and bases. A number of methods are useful for the preparation of these salts and are known to those skilled in the art. Examples include reaction of the free acid or free base form of the peptide with one or more molar equivalents of the desired acid or base in a solvent or solvent mixture in which the salt is insoluble; or in a solvent like water after which the solvent is removed by evaporation, distillation or freeze drying. Alternatively, the free acid or base form of the produce may be passed over an ion-exchange resin to form the desired salt or one salt form of the product may be convened to another using the same general process.<br>
The starting materials required for use in die chemical synthesis of peptides described above are known in the literature or can be prepared using known methods and known starting materials.<br>
The invention also provides a nucleic acid molecule that encodes a peptide according to the invention.<br>
In one aspect, the nucleic acid molecule encodes a peptide having a sequence shown in one of SEQ ID Nos: 1,2, 3,4, 5 or 6.<br>
Typically, the nucleic acid molecule of the invention includes one of the sequences shown in SEQ ID Nos: 7, 8, 9,10,11,12,13,14,15, 16, 17, or 18,19 or 20 or a sequence that is complementary to one of the sequences shown in SEQ ID Nos: 7, 8, 9, 10,11,12,13,14,15,16,17,18,19 or 20.<br>
A nucleic acid molecule that can hybridise to a molecule having one of the above described nucleotide sequences in high stringency conditions is particularly useful as the complementary strand of this nucleic acid molecule may well encode a peptide of the invention that is a variant. As is well known in the art, hybridisation of nucleic acid<br><br>
molecules may be controlled by the type of buffer used for hybridisation and the temperature of the buffer. "High stringency conditions" are conditions in which the buffer includes about 0.1 x SSC, 0.1% SDS and the temperature is about 60°C.<br>
The above described nucleic acid molecules can be obtained from genomic DNA, for example by PCR amplification, from a genomic library, from cDNA derived from mRNA, from a cDNA library, or by synthetically constructing the DNA sequence using synthetically derived nucleotides; (Sambrook et al., Molecular Cloning: A Laboratory Manual (2d ed.), Cold Spring Harbour laboratory, N.Y., 1989).<br>
The nucleic acid molecule of the invention may be a deoxyribonucleotide, a ribonucleotide, a peptide nucleic acid or a combination thereof.<br>
The invention also provides a vector or construct including a nucleic acid molecule of the invention.<br>
A vector containing a sequence shown in SEQ ID NO 19 or 20 is particularly useful for expression of peptides of the invention and fusion proteins including a heterologous protein, as these sequences regulate expression in Gram-positive bacteria such as Lactobacillus and Lactococcus.<br>
The vector or construct is typically obtained by inserting a nucleic acid molecule of the invention into an appropriate plasmid or vector which can be used to transform a cell, for example, a host cell. In general, plasmid vectors containing replication and control sequences which are derived from species compatible with the host cell are used in connection with those hosts. The vector ordinarily carries a replication site, as well as sequences which encode proteins or peptides that are capable of providing phenotypic selection in transformed cells.<br>
Particularly preferred are vectors that permit the introduction of a nucleic acid molecule into Gram-positive bacteria such as Lactobacillus, Lactococcus, Bifidobacterium, Leuconostoc and Streptococcus. Examples of these vectors are described further herein.<br>
A vector mat may be useful for preparing, for example fusion protein constructs is pBR322 and derivatives thereof. pBR322 is a plasmid derived from an E. coli species,<br><br>
see for example Mandel et al., J. Mol. Biol. 53: 154 (1970). Plasmid pBR322 contains genes for ampicillin and tetracyctine resistance and thus provides for easy means for selection. Other vectors include different features such as different promoters, which are often important in expression. For example, plasmids pKK223-3, pDR72O, and pPL-lambda represent expression vectors with the tac, trp, or PL promoters that are currently available (Pharmacia Biotechnology).<br>
A useful vector is pB0475. This vector contains origins of replication for phage and E. coli mat allow it to be shuttled between such host, thereby facilitating both mutagenesis and expression, see for example, Cunningham et al., Science, 243: 1330-1336 (1989); U.S. Pat. No. 5,580,723. Other useful vectors are pRITS and pR!T2T (Pharmacia Biotechnology). These vectors contain appropriate promoters followed by the Z domain of protein A, allowing genes inserted into the vectors to be expressed as fusion proteins.<br>
Other useful vectors can be constructed using standard techniques by combining the relevant traits of the vectors described above. Relevant traits include die promoter, the ribosome binding site, the decorsin or omatin gene or gene fusion (the Z domain of protein A and decorsin or omatin and its linker), the antibiotic resistance marikers, and the appropriate origins of replication.<br>
The invention also provides a cell including a vector or construct as described above. The host cell is typically prokaryotic and typically is a Gram-positive bacteria such as Lactobocillus, Lactococcus, Bifidobacteriwn, Leuconostoc or Streptococcus. Examples are shown in Table 2.<br>
Table Removed<br><br><br>
Prokaryotes may be used for cloning and expressing a nucleic acid molecule of the invention to produce the peptide of the invention. For example, E. coli K12 strain 294 (ATCC No. 31446) may be used as well as E. coli B, E. coli X1776 (ATC No. 31537), and E. coli c600 and c600hfl, E. coli W3110 (F-,gama-,prototrophic/ATCC No. 27325), bacilli such as Bacillus subtilis, and other Enterobacteriaceae such as Salmonella_typhimuriwn or Serratia marcesans, and various Pseudomonas species. When expressed by prokaryotes the peptide of the invention may contain an N-terminal methionine or a formyl methionine and may not be glycosylated. In the case of fusion proteins, the N-terminal methionine or formyl methionine may reside on the amino terminus of the fusion protein or the signal sequence of the fusion protein.<br>
The invention also provides a process for producing a peptide of the invention. The process inchides maintaining a cell containmg a nucleic acid molecule as described above, or a vector or construct as described above, in conditions for permitting the cell to produce the peptide.<br>
The process may optionally include the step of recovering and or purifying the protein. Purification of the peptide is typically achieved using conventional procedures such as preparative HPLC (including reversed phase HPLC) or other known chromatographic techniques such as gel permeation, ion exchange, partition chromatography, affinity chromatography (including monoclonal antibody columns) or counter-current distribution.<br>
The expression of a peptide of the invention is described further herein. Other exemplary expression systems include those described in Table 3 below.<br>
TableRemoved<br><br><br>
Kleerebezem M, Beerthuyzen MM, Vanghan BE, de Vos WM, Ruipers OP. Controlled gene expression systems for lactic acid bacteria: transferable oisin-inducible expression cassettes for Lactococcus, Leuconostoc, and Lactobacillus spp. Appl Environ Microbiol. 1997 Nov,63(ll):4581-4; Kruger C, Hu Y, Pan Q, Maxcotte H, Hultberg A, Dehvar D, van Dalen PI, Pouwels PH, Leer RJ, Kelly CG, van Dollenweerd C, Ma JK, Hammarstrom L. In situ delivery of passive immunity by lactobacilli producing single-chain antibodies. Nat Biotechnol. 2002 Jul;20(7):702-6; Perez-Arellano I, Perez-MartLnez G. Optimization of the green fluorescent protein (GFP) expression from a lactose-inducible promoter in Lactobacillus casei. FEMS Microbiol Lett. 2003 May 16;222(1): 123-7; Savijoki K, Kahala M, Palva A. High level heterologous protein production in Lactococcus and Lactobacillus using a new secretion system based on the Lactobacillus brevis S-layer signals. Gene. 1997 Feb 28;186(2):255-62; Wells JM, Wilson PW, Norton PM, Gasson MJ, Le Page RW. Lactococcus lactis: high-level expression of tetanus toxin fragment C and protection against lethal challenge. Mol Microbiol. 1993 Jun;8(6):l 155-62.<br>
Brief description of the drawings<br>
Figure 1. Analysis of proteins found in the culture supernatant of L. fermentum BR11 grown in MRS broth. Growth of L. fermentum BR11 was monitored<br><br>
over 24-h by optical density measurements at 600nm. At various time points, indicated by a number in a circle, aliquots were taken, centrifuged and the supernatant filtered and precipitated with 5% TCA. The equivalent of 225-fil of culture supernatant was analysed by SDS-PAGE followed by staining with Coomassie brilliant blue G-250. The arrow indicates Sep.<br>
Figure 2. Expression and subcellular location of a Hise-Sep fusion protein in<br>
L. fermentum BR11, L, rhamnosus GG and L. lactis MG1363. Above shows the<br>
arrangement of the constructs which were either integrated into the L. fermentum BR11<br>
chromosome (Sep-6xHis-Sep and BspA-6xHis-Sep) or introduced into L. rhamnosus GG<br>
or L. lactis MG1363 on the pGh9:ISS7 plasmid (Sep-6xHis-Sep only). Below shows<br>
Western blot detection of fusion proteins in cell extracts and in the supernatant using an<br>
anti-Hiss antibody. For the diagrams the sep terminator (Tsep) and DNA encoding the<br>
Sep secretion signal (ssSep), BspA secretion signal (ssBspA) and Hise (grey box) are<br>
indicated. The DNA region winch is the site of single crossover homologous<br>
recombinatkm into either the 5^ or fa^ Ira	is spotted and below<br>
is marked with a cross. Sizes of molecular mass markers are indicated in kDa on the left. The lanes containing cell extracts prepared by boiling in 2x SDS-loading dye (SDS), by sonication (son) and with 5M LiCl (LiQ) and the precipitated supernatant fractions (SN) are indicated. The amount of cells or medium loaded in each lane are the equivalent to 500ul (SDS), 50ul (son), 160^1 (LiCl) and 675ul (SN) of culture.<br>
Figure 3. Expression and secretion of human E-cadherin fusion protein by L. fermentum BR11. Above shows the arrangement of the constructs which were introduced into L. fermentum BR11 (Sep-6xHis-Ecad and BspA-6xHis-Ecad). Below shows Western blot detection of fusion proteins in cell extracts and in the supernatant using an anti-Hiss antibody (A, B and C[left side]) and in the supernatant using an anti-E-cadherin antibody (C[right side]). For the diagrams the bspA terminator (T bspA) and DNA encoding the Sep secretion signal (ssSep), BspA secretion signal (ssBspA) and Hise (grey box) are indicated. The DNA region which is the site of single crossover homologous recombination into either the sep or bspA loci of L. fermentum BR11 is spotted and below is marked with a cross. The sizes of the molecular mass markers are indicated in kDa on<br><br>
the left The lanes containing cell extracts prepared by boiling in 2x SDS-loading dye (SDS), by sonication (son) and with 5M LiCl (LiCl) and the precipitated supernatant fractions (SN) are indicated. The amount of cells or medium loaded in each lane are the equivalent to SOOul (SDS), 50ul (son), 160fil (LiCl) and 675ul (SN) of culture. For the Western blot in part C, the equivalent of 1.2-ml of culture supernatant from L. fermenutm BR11 parent (BR11) or L. fermentum containing BspA-6xHis-Ecad (BspA-6xHis-Ecad) was loaded in each lane.<br>
Figure 4. Expression of human vitronectin using the expression and secretion signals of Sep. The lanes containing cell extracts (C) prepared by boiling cells in 2x SDS-PAGE loading buffer and the precipitated supernatant fractions (S) are indicated. The amount of cells or medium loaded in each lane is the equivalent to 1ml (C) and 900^1 (S) of culture.<br>
FigureS. Features of the pSepSllsec plasmid used for expression and secretion of PtySll in lactic acid bacteria. The origin of the temperature sensitive origin of replication (Ts) of pQh9:JSiS7 is indicated while the direction of the erymromycin resistance (EmR) marker gene is also huticatrd by an arrow. The Sep-6xHis-Ply511 expression construct cloned into pGh9::IS£/ is shown with an arrow head trnV-atrng the likely sep promoter, a hatched box indicating the Sep secretion signal (ssSep), a grey box indicating the Hise epitope encoding DNA and a lollipop indicating the bspA operon terminator (TbspA). At the bottom of the figure is the nucleotide and translated amino acid sequence of the junction between the Sep secretion signal and the Ply511 encoding DNA. The vertical arrow indicates the signal peptide cleavage site while the horizontal arrow indicates the start of PlyS 11.<br>
Figure 6. Analysis of expression, secretion and activity of PlyS 11 produced by Lactobacittus spp. and L. lactis. (A) Western blot detection of proteins in the cell extract (C) and supernatant (S) of lactic acid bacteria containing pSepSllsec using anti-Hiss-HRP conjugate. The amounts of cell extract or medium loaded in each lane are the equivalent to 500/il and 675/il of culture, respectively. (B) Detection of bacteriolytic activity of lactic acid bacterial supernatant fractions using renaturing SDS-PAGE with autoclaved L. monocytogenes as the substrate. For each strain the (—) lane indicates<br><br>
either pGh9::IS,SV containing (L. lactis, L. fermentum and L. rhamnosus) or wild-type (L. plantation) strains while the (4-) lane indicates pSepSl Isec containing strains.<br>
Figure 7. Cell wall lytic activity of strains of lactic acid bacteria grown on buffered agar medium containing autoclaved L. monocytogenes cells. L. lactis were grown on buffered GM17E while Lactobacillus spp. strains containing plasmids were grown on buffered MRS with erythromycin while L. plantarum wild-type was grown on buffered MRS without erythromycin.<br>
Figure 8	Sep LysM domain amino acid sequence<br>
Figure 9	Sep C-terminal (apf) domain amino acid sequence<br>
Figure 10	Sep glutamine-rich region<br>
Figure 11	Sep secretion signal amino acid sequence<br>
Figure 12	Sep amino acid sequence<br>
Figure 13	Sep amino acid sequence including secretion signal sequence<br>
Figure 14 Sep secretion signal nucleon'de sequence. TUB Mixed Base Codes: R=AG Y=CT M=AC K=GT S=GC W=AT H=ACT B=GCT V=AGC I&gt;=AGT N=AGCT<br>
Figure IS Sep LysM domain nucleotide sequence. IUB Mixed Base Codes: R=AG Y=CT M=AC K=GT S=GC W=AT H=ACT B=GCT V=AGC E&gt;=AGT N=AGCT<br>
Figure 16 Sep glutamine-rich region nucleotide sequence. IUB Mixed Base Codes: R=AG Y=CT M=AC K=GT S=GC W=AT H=ACT B=GCT V=AGC D=AGT N=AGCT<br>
Figure 17 Sep C-terminal nucleotide sequence. IUB Mixed Base Codes: R=AG Y=CT M=AC K=GT S=GC W=AT H=ACT B=GCT V=AGC D=AGT N=AGCT<br>
Figure 18 Sep nucleotide sequence. IUB Mixed Base Codes: R=AG Y=CT M=AC K=GT S=GC W=AT H=ACT B=GCT V=AGC D=AGT N=AGCT<br>
Figure 19 Sep nucleotide sequence including secretion signal coding sequence. IUB Mixed Base Codes: R=AG Y=CT M=AC K=GT S=GK: W=AT H=ACT B=GCT V=AGC D=AGT N=AGCT<br><br>
Figure 20 Sep secretion signal backtranslation sequence. IUB Mixed Base Codes: R=AG Y=CT M=AC K=GT S=GC W=AT ENACT B=GCT V=AGC D=AGT<br>
N=AGCT<br>
Figure 21 Sep LysM domain signal backtranslation sequence. IUB Mixed Base Codes: R=AG Y=CT M=AC K=GT S=GC W=AT H=ACT B=GCT V=AGC D=AGTN=AGCT<br>
Figure 22 Sep glutamine-rich region signal backtranslation sequence. IUB Mixed Base Codes: R=AG Y=CT M=AC K=GT S=GC W=AT H=ACT B=GCT V=AGC D=AGT N=AGCT<br>
Figure 23 Sep C-terminal signal backtranslation sequence. IUB Mixed Base Codes: R=AG Y=CT M-AC K=GT S=GC W=AT H=ACT B=GCT V=AGC D=AGT N=AGCT<br>
Figure 24 Sep backtranslation sequence. IUB Mixed Base Codes: R=AG Y=CT M=AC K=GT S=GC W=AT H=ACT B=GCT V=AGC D=AGT N=AGCT<br>
Figure 25 Sep including secretion signal backtranslation sequence. IUB Mixed Base Codes: R=AG Y=CT M=AC K=GT S=GC W=AT H=ACT BK3CT V=AGC D=AGT 1SNAGCT<br>
Figure 26 Nucleotide sequence of 310-bp immediately upstream of sep containing a possible sep promoter (putative -35 and -10 recognition hexamers are indicated as shaded letters and bold letters respectively [note one -35 consensus also may be a -10 consensus]; TG motifs upstream of the putative -10 consensus hexamers are italisised; the sep ribosome binding site is underlined. IUB Mixed Base Codes: R=AG Y=CT M=AC K=GT S=GC W=AT H=ACT B=GCT V=AGC D=AGT N=AGCT<br>
Figure 27 Nucleotide sequence of 150-bp immediately downstream of sep containing the sep transcription terminator (indicated as converging arrows above the sequence. IUB Mixed Base Codes: R=AG Y=CT M=AC K=GT S=GC W=AT H=ACT B=GCT V=AGC D=AGT N=AGCT<br><br>
The invention is described below with reference to certain non-limiting examples. It will be appreciated by persons skilled in the art that numerous variations and/or modifications may be made to die invention as described without departing from the spirit or scope of the invention as broadly described. The following examples are, thereof, to be considered in all respects as illustrative and not restrictive.<br>
Example 1   Production of Sep<br>
L. fermentum BR11 was grown in standing MRS broth at 37°C and fractions were taken at five timepoints (Figure 1). SDS-PAGE analysis revealed a number of proteins which accumulated in the supernatant during growth (Figure 1). The smallest visible protein (indicated by the arrow) was still abundant in late stationary phase when the level of a number of other proteins had reduced. This protein was called Sep for small exported protein. When its small size is taken into account, Sep is one of the most abundant proteins found in the supernatant of L. fermentum BR11. To further characterise Sep we identified the N-tenninal sequence which was found to be: DTrrrvQsarjn*SGi. Sep is a 205 ammo acid protein with a 30 ammo acid N-tenninal secretion signal giving rise to a predicted 19-kDa mature protein whn an isodectric point of S3.<br>
Example 2   Sep -E cadherin fusion protein under control of Sep promoter.<br>
Escherichia call JM109 was used in molecular cloning experiments. Ampicillin was used at a concentration of 100 or 200 fig per ml for E. coli while erythromycin was used at concentrations of 750 ug per ml for E. coli. Plasmids pUCl 8, pBluescriptn (KS) and pGEMSzf were used for routine cloning.<br>
The region encoding the amino-terminal 1 to 216 ammo acids of the mature E-cadherin protein was amplified by PCR from cDNA template prepared from cultured mammalian T47D and LNCap cells using oligonucleotides E-cad-PstI and E-cad-XhoI. This fragment was cloned in frame downstream of DNA encoding the Sep secretion signal to generate construct Sep-6xHis-Ecad. The sequence of the cloned E-cadherin DNA fragment which contained an introduced stop codon after codon 216 was checked by DNA sequencing. The putative bspA transcription terminator was amplified using oligonucleotides Term-Xho and Term-Hind and cloned downstream of the E-cadherin encoding DNA.<br><br>
The construct within pJRS233 was transformed into L. fermentum BR11 using penicillin as a cell wall weakening agent at concentrations of 1 or 10 jig per ml, respectively, as described previously (Rush, C.M., L.M. Hamer, and P. Timms. 1994. Genetic modification of a vaginal strain of Lactobacillus fermentum and its maintenance within the reproductive tract after intravaginal administration. J. Med. Microbiol. 41:272-278; McCracken, A., M.S. Turner, P. Giffard, LJM. Hamer, and P. Timms. 2000. Analysis of promoter sequences from Lactobacillus and Lactococcus and their activity in several Lactobacillus species. Arch. Microbiol. 173:383-389). The construct was integrated into the chromosome of L. fermentum downstream of the sep promoter by incubating transformants at 40°C in the presence of ervthromycin selection.<br>
L. fermentum BR11 were grown on solid MRS medium (Oxoid, Basingstoke, United Kingdom) anaerobically or in standing liquid culture tubes. Erythromycin was used at a concentration of 10 fig per ml for L. fermentum.<br>
Cell extracts were prepared from late log or early stationary phase cultures while supernatants were taken from late exponential phase cultures. Two different whole cell protein extraction methods which involved either boiling cells in 2x SDS-P AGE loading buffer (Sambrook, J., EJ. Fritsch, and T. Maniatis. 1989. Molecular cloning: a laboratory manual, 2nd ed. Cold Spring Harbor Laboratory press, Cold Spring Harbor, N.Y.) or sonication were used as described previously (Turner, M.S., L.M. Hafher, T. Walsh, and P.M. Giffard. 2003. Peptide surface display and secretion using two LPXTG-containing surface proteins from Lactobacillus fermentum BR11. Appl. Environ. Microbiol. 69:5855-5863). 5M LiCl extractions of cells and supernatant fractions were also obtained as described previously (Turner 2003). Prior to loading of SDS-PAGE all samples were boiled for 5 minutes. Proteins were transferred to nitrocellulose, blocked and then probed with an anti-His; monoclonal antibody (Qiagen, Hilden, Germany) at 1 in 1000 dilution. Following washes, the membrane was incubated with rabbit anti-mouse-horseradish peroxidase (HRP) conjugate (Dako, Glostrop, Denmark). The bound antibodies were detected using the HRP chemiluminescence kit (Roche, Mannheim, Germany). To estimate levels of Hise proteins in extracts varying amounts of His« labeled protein markers (Qiagen, Glostrop, Denmark) were included alongside the samples.<br><br>
These markers have known quantities of Hiss containing proteins in each band allowing densitometry to be done on films using the TotalLab vl. 11 package (Phoretix, Newcastle upon Tyne, United Kingdom). For detection of E-cadherin a mouse monoclonal anti-human E-cadherin antibody (from clone HBCD-1; Zymed Laboratories Inc.) was used at a concentration of 1 in 750.<br>
E-cadherin fusion protein was detected in the SDS cell extract and supernatant from cells grown at 30°C (Figure 2). Levels of E-cadherin fusion protein for this strain at 30 °C were ~30 pg per liter culture in the supernatant and -370 jig per liter culture in the SDS cell extract. The predicted size of the E-cadherin fusion protein is 25-kDa, however the protein recognized by the anti-Hiss antibody in the Western blot resolved -38-kDa. To confirm that mis protein is indeed E-cadherin, a mouse monoclonal anti-human E-cadherin antibody was used as the primary antibody in a Western blot The anti-E-cadherin antibody recognized a protein the same size as the protein recognized using the<br>
anti-Higg antthndy «md did nnt ^fff^m-Tf pitting fiqgfyl in Hi* qipf-matamt pf fl|* parent 1<br>
fermentum BRll strain. These resnhs suggest mat Sep expression and secretion, signals can be used to secrete a human ammo-terminal E-cadherin peptide in L. fermentum. As the ammo-terminus of E cadherin is a major intestinal cell receptor for the food-borne disease causing pathogen L. monocytogenes mis construct may have potential as an intestinal L. monocytogenes attachment -inhibiting therapeutic.<br>
Example 3   Sep- 6xHis fusion proteins under control of Sep promoter.<br>
E, coli JM109 was used in molecular cloning experiments. Ampicillin was used at a concentration of 100 or 200 \\% per ml for E. coli. Plasmids pUCIS, pBluescriptn (KS) and pGEMSzf were used for routine cloning.<br>
The construct (Sep-6xHis-Sep) consists of DMA upstream of sep and the sep 5' region encoding the secretion signal and a six-histidine (Hisfi)epitope (amplified and cloned using Nterm-US-Xba and Nterm-Pst-US) and DNA encoding the mature Sep protein and the putative sep transcription terminator (amplified and cloned using SepDS-PstXho and SepDS-ApaSal). The construct (BspA-6xHis-Sep) consists of DNA encoding the mature Sep protein and putative sep transcription terminator as above but instead contains upstream DNA encoding a full length BspA protein followed by DNA encoding<br><br>
the BspA secretion signal and a Hisc epitope as described previously (Turner et al., supra). The extra amino acids added onto the mature N-termini of Sep in the Sep-6xHis-Sep construct are: DTIYTDHHHHHHSAAGSR and in the BspA-6xHis-Sep construct are: ASDDVHHHHHHSAAGSR.<br>
These expression cassettes were constructed in pBluescriptn and then cloned into the Xbal and Sail digested pJRS233. The Sep-6xHis-Sep constuct in pBluescript H is also digested with Sail and cloned into Xhol digested pGh9:IS£/.<br>
The construct within pJRS233 was transformed into L. fermentum BR11 using penicillin as a cell wall weakening agent at concentrations of 1 or 10 /ig per ml, respectively, as described previously (Rush, McCracken, supra). Expression in L. fermentum was achieved by integrating the chimenc genes downstream of either the sep or bspA promoters (Figure 3A and 3B) by incubating transformants at 40°C in the presence of erytfaromycin selection.<br>
L. fermentum BR11 was grown on solid MRS medium (Oxoid, Basingstoke, United Kingdom) anaerobicalry or in standing liquid culture tubes. Erythromycin was used at a concentration of 1 Ofig per ml far L. fermentum.<br>
In the cases of L. rhamnosus and L. lactis, me Sep-6xHis-Sep construct was transformed into the cells cloned into the pGh9:ISS/ plasmid (Maguin, E., H. Prevost, S.D. Ehrlich, and A. Gruss. 1996. Efficient insertional mutagenesis in lactococci and other gram-positive bacteria. J. Bacteriol. 178:931-935).<br>
L. rhamnosus GG (ATCC 53103) was grown on solid MRS medium (Oxoid, Basingstoke, United Kingdom) anaerobically or in standing liquid culture tubes. L. lactis MG1363 was grown at 30°C in Ml 7 medium (Oxoid, Basingstoke, United Kingdom) supplemented with 0.5% (wt/vol) glucose (GM17). Erythromycin is used at concentrations of 10 ug per ml for L. rhamnosus and 5 |ig per ml for L. lactis.<br>
Transformation of L. rhamnosus is done using penicillin as a cell wall weakening agent at concentrations of 1 or 10 /tg per ml, respectively, as described previously (Rush, McCracken, 2000, supra). L lactis is transformed using 1% glycine as a cell wall weakening agent as described previously (Holo, H., and UP. Nes. 1989. High-frequency<br><br>
transformation, by electroporation, of Lactococcus lactis subsp. cremoris grown with glycine in osrnotically stabilized media. Appl. Environ. Microbiol. 55: 3119-3123), except transformants are selected directly on GM17 plates containing 5/tg per ml erythromycin. L. lactis transformants are grown at 30°C to allow replication of the temperature sensitive pGh9:IS&gt;S7 plasmid derivatives while L. rhamnosus transformants are grown on plates at 30°C and in liquid at 30°C or 37°C.<br>
Cell fractionation, protein extraction and Western blot analysis were performed as described in Example 2. The accessibility of the Hiss epitope on whole cells is done the same as that described previously (Turner 2003 supra).<br>
The predicted molecular mass of the mature Hise-Sep fusion proteins is 21-kDa,<br>
although the bands in Western blots correspond to proteins 28-kDa in size (Figure 3A<br>
and 3B). m L. fermentum containing the Sep-6xHis-Sep and the BspA-6xHis-Sep<br>
(xnistnicts the Sc? fusion protem was found pred	levels of<br>
~2 mg per fiter of culture in bom cases. Levels of me Sep fusion protein in the SDS ceU extracts for L fermentum containing the Sep-6xHis-Sep and BspA-6xHis-Sep constructs were ~9% and -13% of mat found in the supematants, respectively. No Sep fusion protein was detected in sonicate or 5 M LiCl extracts. When SDS cell and supernatant extracts were run in neighbouring lanes the Sep fusion protein bands migrated identically on SDS-PAGE (data not shown), which suggests mat the Sep fusion protein associated with cells is the mature form and therefore does not contain a signal sequence and is located outside the cytoplasmic membrane.<br>
In L. rhamnosus and L. lactis, levels of Sep fusion protein expressed from the Sep-6xHis-Sep construct were found to be ~200 and ~300 u.g per liter of culture in the supernatant, respectively (Figure 3C and 3D). Levels of the Sep fusion protein in the SDS cell extracts for L. rhamnosus and L. lactis were 
 <br>
M Lid extracts of either L rhamnosus or L lactis containing the Sep-oxHis-Sep construct.<br>
To examine if Sep is exposed on the cell surface of L. fermentum, a whole cell enzyme-linked immunosorbent assay for the His&amp; epitope was performed me same to that described previously (Turner et al., supra). It was found that the A^ostm signal per ODfioomn unit of cells obtained for L. fermentum cells containing the BspA-6xHis-Sep construct (0.0302 ± 0.0003) was significantly greater (1.8-fold) than for L. fermentum BR11 cells (0.0168 ± 0.0007). This result suggests that at least some cell associated Sep is located hi an exposed form in the cell envelope of L. fermentum.<br>
Example 4 Expression of human vitronectln In Lactobacillus fermentum BR11 and Lactobacillus rhamnosus GG using the expression and secretion signals of Sep.<br>
The expression construct used here utilises the Sep expression and secretion fljgpafo such that human vitronectin is exported into the culture supernatant from the host Lactobacillus. The mature vitronectm gene was amplified from cDNA and cloned into the Sep-6xHis-Ecad expression cassette using restriction enzymes PsA and XkoL Following its insertion into the pGh9: 'JSS1 plasmid, me hydrid was transformed into L. fermentum BR11 and L. rhamnosus GG. Following 2 days growth at 32°C, cells were fractionated and analysed using Western blot to detect Rise-tagged vitronectm.<br>
The lanes containing cell extracts (C) prepared by boiling cells in 2x SDS-PAGE loading buffer and the precipitated supernatant fractions (S) are indicated. The amount of cells or medium loaded in each lane is the equivalent to 1ml (C) and 900ul (S) of culture.<br>
The predicted size of mature non-glycosylated His6-vitronectin is 54-kDa. For clones of L. fermentum BR11 there is a clear dominant protein ~52-kDa in the supernatant which probably corresponds to full-length vitronectm Figure 4. The cell extracts of L. fermentum BR11 clones show a number of bands with the majority being smaller, probably degraded forms of vitronectin. For clones of I. rhamnosus GG there is one dominant band in the supernatant fraction which is ~47-kDa which probably corresponds to His6-vitronectin with a small amount of C-terminal degradation.<br><br>
Therefore in conclusion Sep has been shown to express and secrete a large, cysteine-rich human protein of commercial importance in the generally regarded as safe lactobacilli.<br>
Examples   Expression and secretion of active Usterta monocytogenes bacteriophage A511 endolysin Ply511 by Lactobaclllus<br>
Material and Methods<br>
Bacterial strains, plasmids and growth conditions<br>
L fermentum BR11, L rhomnosus GG (ATCC 53103) and L. plantarum ATCC 14917 were grown on solid MRS medium (Oxoid, Basingstoke, U.K.) anaerobically or in standing liquid culture tubes. L. lactis MG1363 was grown at 30°C in M17 medium (Oxoid, Basingstoke, UJL) supplemented with 0.5% (wt/vol) glucose (GM17) and when appropriate in the presence of erymromycin (GM17E). Escherichia coli JM109 was used m molecular ctonmg experiments. AmpicUlm was used at a concentration of 100 or 200 ug per ml for £ coli while erythromycin was used at concentrations of 500-750 ug per ml for £. orfi, 10 ug per ml for lactobacilli and 5 ugpermlfor.L. lactis. Expression cassettes were introduced into lactic acid bacteria using pGh9-JS57. Plasmid pGEM3Zf was used for routine cloning.<br>
Transformation of lactobacilli and L. lactis<br>
Transformation of L. fermentum, L rhamnosus and L. plantarum was done using penicillin as a cell wall weakening agent at concentrations of 1.25, 10 and 5/xg per ml, respectively. L lactis was transformed using 1% glycine as a cell wall weakening agent, except that transformants were selected directly on GM17 plates containing 5/ig per ml erythromycin. L. lactis transformants (containing pGh9:IS57 derivatives) were grown at 30°C while lactobacilli transformants (containing pGh9:IS.S7 derivatives) were grown on plates and in liquid media at 32-33°C.<br>
Construction of ply 511 into the Sep secretion expression cassette<br>
DNA encoding amino acids 1 to 341 of the endolysin PlySll of L. monocytogenes phage AS 11 was amplified using cloned pfySll as template. This<br><br>
fragment was cloned in frame downstream of DNA encoding the Sep secretion signal in plasmid pGEM-3Zf containing me Sep-6xHis-Ecad expression construct such mat the E-cadherin encoding gene fragment is replaced with the pfySll gene. The new plasmid was named pSep-6xHis-Ply511. The functional parts of the Sep-6xHis-Ply511 construct were amplified using oh'gonucleotides SepUS-Eco and Term-Hind-trunc and this 1.9-kb fragment was then digested with EcoRl and HindSL This fragment was h'gated to similarly digested pGh9::ISS7 (which removes the 1SS1 insertion sequence) and the ligation mix was transformed directly into L. lactis. L. lactis transformants were plated onto GM17 agar containing 5 jig per ml erythromycin and sufficient autoclaved L. monocytogenes 491 cells to obtain visible turbidity. After 2 days clones secreting active PlySll could be detected by the formation of clearing zone around the colonies. Plasmids from the positive clones were purified and were used to transform lactobacilli.<br>
Cell firaetkHMUM, protefa extraction and Wester* blot analysis<br>
Cell extracts and superaatants were prepared from late exponential or early stationary phase cultures. Cell extracts were prepared by bofling cells in 2x SDS-PAGE loading buffer and supematants were concentrated using 5% trichloroacetic acid as described previously. Proteins were transferred to nitrocellulose, blocked and then probed with an anti-Hiss-horseradish peroxidase (HRP) conjugate antibody (Qiagen, Hilden, Germany) at 1 in 4,000 dilutioa The bound antibodies were detected using the Lumi-Light chemiluminescence kit (Roche, Mannheim, Germany). To estimate levels of Hisg proteins in extracts, varying amounts of Hise labeled protein markers (Qiagen, Glostrop, Denmark) were included alongside the samples. These markers have known quantities of Hisg containing proteins in each band allowing densitometry to be done on films using the Toto/Lab vl. 11 package (Phoretix, Newcastle upon Tyne, U.K.).<br>
Renaturing SDS-PAGE<br>
Proteins were separated in SDS-PAGE using a 4% stacking gel and a 10-ml 12% separating gel containing 0.5ml of L monocytogenes 491 cells which had been autoclaved and concentrated 100-fold in spent Bffl broth. Following electrophoresis, the gel was washed in distilled water for 10 minutes and then gently shaken in three to four changes of renaturing buffer (50 mM Tris-HCl pH 8, 100 mM NaCl and 1% Triton-<br><br>
X100)at23°Cfor20bours. The gel was then briefly rinsed with distilled water and then stained with 0.1% (wt/vol) methyiene blue in 0.01% (wt/vol) KOH for 2 hours and then destained with several changes of distilled water.<br>
Photometric detection of L. monocytogenes cell wall tytic activity<br>
L. monocytogenes was grown overnight in BHI and the cells were either (i) concentrated 100-fold in SM buffer and stored frozen or (ii) autoclaved and concentrated 100-fold in spent BHI and stored frozen. To test for secreted endolysin activity, supernatants (900 pi) from lactic acid bacteria grown to mid-late exponential phase were buffered by the addition of 100 pi 1M Tris-HCl (pH 8) and placed in a cuvette. L. monocytogenes substrate cells were .added to the cuvette and any change in ODeoo was monitored over time at 23°.<br>
Detection otpfySU secretion by lactic acid bacterial colonies on agar plates<br>
To 15-ml (final volume), 03-ml of 100-fold concentrated autoclaved L. monocytogenes 491 was added to MRS or GM17 agar. Erythromycin was also added to the ag?r at concentrations mentioned earlier (except fix the agar used to grow L, plantanan wild-type). Agar was buffered using 0.2 M potassium phosphate buffer pH 7 (final concentration) and the final volume was kept at 15-ml.<br>
Killing of L. monocytogenes by lactic add bacteria secreting PlySll<br>
Lactic acid bacteria and L. monocytogenes QUT0085 were grown to mid-late exponential phase and the cells were harvested by centrifugation. Lactic acid bacteria were washed twice in fresh BHI broth and once in SM buffer (50 mM Tris-HCl, 100 mM NaCl, 10 mM MgSO4; pH 7.5) while L monocytogenes QUT0085 was washed once in BHI and once in SM buffer. Cells were resuspended at ~1010 cells/ml for lactic acid bacteria and ~109 cells/ml for L. monocytogenes QUT0085. Two hundred microliters of lactic acid bacteria was added to 800 pi of SM buffer and mis was then mixed with 100 pi of L. monocytogenes QUT0085. The mixes were incubated at 32°C for 2.5 hours, then diluted in SM buffer and plated onto Oxford agar (Oxoid, Basingstoke, U.K.) to determine viable L. monocytogenes QUT0085.<br>
Results<br><br>
Ckming of the endolysin gtatepfySll into the Sep secretion cassette<br>
DNA encoding the PrySll endolysin ofL, monocytogenes bacteriophage A511 was cloned in frame downstream of DNA encoding the Sep secretion signal and the sep promoter (Figure 5). To facilitate detection of expressed proteins, a His<s epitope is located between the sep secretion signal and start of heterologous protein sep-6xhis-ply511 construct was amplified by pcr digested ligated into similarly pgh9::is57. ligation reaction transformed l. lactis cells were plated onto gm17 including erythromycin autoclaved monocytogenes to detect cell wall lytic activity ply511. several transformants that had clearing zones around colonies harvested their plasmids purified. plasmid from one these clones named pseps isec.></s>
Expression and secretion of PlySll in L. fermentum BR11, L. rhamnosus GG, L. plantorum ATCC14917 and L. lactis MG1363<br>
The plasmid pSepSllsec was transformed by ekctroporation into L. fermentum &amp;mi,L. rhamnosus GGaadLplantarumATCC 14911. CeU extracts and supernatant fractions were collected from me lactobacilli and L. lactis transfbnnants and were analysed by Western btomng usmg an anti-Hiss-HRP conjugate. Bands were observed in all strains with the largest in each lane being ~40-kDa which is close to the calculated molecular weight of Hiss-PlySl 1 (38.2-kDa) (Figure 6A). All strains had His6-PlySl 1 in the supernatant fraction as well as in the cell-associated fraction. All strains except L. rhamnosus had lower molecular weight bands indicating possible proteolytic degradation. Analysis by Western blot, L. plantarum produced the greatest amount of Hs6-Ply511 and the level of full-length Hise-PlySl 1 was estimated to be -10 jug per liter of culture. To determine if the Hise-PlySll is active, supematants from strains containing either no plasmid, pGh9::IS*S7 or pSepSllsec were analysed by renaturing SDS-PAGE. Autoclaved L. monocytogenes 491 cells were incorporated into the separating gel and activity was identified as clear bands (Figure 6B). Bands at ~40-kDa were observed in all supernatant fractions from strains containing pSepSl Isec but not in wild-type or strains containing pGh::IS,S7. Extra endogenous higher molecular weight<br><br>
clearing bands were observed L. lactis (~45-lcDa) and L. fermentum (-52 and ~70-kDa) supernatant fractions.<br>
Activity of PlySll secreted by lactic acid bacteria is enhanced by buffering with 0.2M potassium phosphate, pH 7<br>
To determine if the lactic acid produced by lactobacilli and L. lactis effects PlySll activity, we buffered the agar growth media containing autoclaved L. monocytogenes 491 at near neutral by the addition of 0.2 M potassium phosphate buffer (pH 7). L. lactis containing pSepSllsec produced a small clearing zone on normal growth media and large clearing zones on buffered growth media (Table 5, Figure 7). On normal growth media, both L. fermentum containing pGh9::IS,S7 or pSepSllsec produced the same medium sized clearing zones, while on buffered growth media only/.. fermentum pSepSllsec produced a medium clearing zone (Table S, Figure 7). Both L. rhamnosus containing pGh9::ISS7 or pSepSllsec did not produce clearing zones on normal growth media, however large clearing zones were produced by L. rhamnosus containing pSepSllsec but not by L. rhamnosus containing pGh9::I£S7 grown on buffered growth media (Table 5, Figure 7). L. plantanm containing pSepSllsec produced a small sized clearing zone while wild-type ^.pfarttonwn did not On buffered growth media, L plantarum containing pSepSllsec produced a small clearing zone, but grew very poorly, while L. plantarum wild-type did not produce any clearing zone, but grew well (Table 5, Figure 7).<br>
A photometric activity assay was used to compare the L. monocytogenes cell wall lytic activity of supernatants of the lactobacilli and L. lactis strains expressing PlySll. Supematants were collected from mid- to late-logarithmic growth phase, buffered by the addition of 0.1 M Tris-HCl (pH 8), mixed with L. monocytogenes cell suspension and any change in absorbance at 600nm was followed. It was found using this assay that all of the supernatants of strains containing pSepPlySllsec had endolysin activity. L. plantarum had the greatest endolysin activity while L. lactis and L. rhamnosus having moderate endolysin activity with L fermentum having the least, but still detectable, endolysin activity (data not shown).<br>
Killing of L. monocytogenes by lactic add bacteria secreting PlySll<br><br>
Washed L. lactis (~109 cells) were mixed with washed live L. monocytogenes QUT0085 (~10* cells) in SM buffer. After 2.5 hours, serial dilutions were plated onto Oxford agar to enumerate viable Listeria and GM17E to enumerate viable L. lactis. L. lactis containing pSepSllsec reduced viability of Listeria by 1.88 log CFU/mL<br><br>
Table Removed<br>
Underline indicates restriction endonuclease recognition sites.<br>
Y=C or T; H=A, C orT;N=A, G, C or<br><br><br>
TableS.<br>
L. monocytogenes cell wall lytic activity of lactic acid bacteria grown on agar plates with and without 0.2 M potassium phosphate buffer*.<br><br>
Normal growth media<br><br>
Buffered growth media<br><br><br><br>
pGh9:IS5/ or wild-type<br><br>
pSepSllsec<br><br>
or wild-type<br><br>
pSepSllsec<br><br><br>
* clearing zones around the colonies were scored as follows: no clearing zone (-), small just visible clearing zone (+), medium sized clearing zone (++), and large clearing zone (+++).<br>
b this strain grew very poorly on buffered growth media but grew well on normal growth media.<br>
All publications and patent applications mentioned in this specification are indicative of the level of skill of those skilled in the art to which this invention pertains. All publications and patent applications are herein incorporated by reference to the same extent as if each individual publication or patent application was specifically and individually intended to be incorporation by reference.<br><br>
S80769691.ST25 SEQUENCE  LISTING<br>   Queensland university of Technology    Protein expression     S80769691<br>    2003905886     2003-10-24<br>    20<br> Patentln version 3.2<br>	50<br>	PRT<br>	Lactobacillus fermentum<br> 1<br>
Asp Thr lie Tyr Thr Val Gin  Ser Gly Asp Thr Leu Ser Gly lie Ser<br>
15	10	15<br>
Tyr Lys phe Ala Lys Asp Asn Ser Met lie Asn Asp Leu Ala Lys Lys<br>
20	25	30<br>
Asn Asn lie Gin Asp lie Asn  Lys lie Phe Val Gly Gin Lys Leu lie<br>
35	40	45<br>
lie Lys 50<br>	2<br>	81<br>	PRT<br>	Lactobacillus fermentum<br>    2<br>
Ser Tyr Thr ser Asn Ala Ser Gly Ser Glu Ala Ala Ala Lys Ala Trp<br>
15	10	15<br>
lie Ala Gly Arg Glu Ser Gly Gly Asn Tyr Asn Ala Thr Asn Gly Gin<br>
20	25	30<br>
Tyr lie Gly Lys Tyr Gin Leu Ala Ala Ser Tyr Leu Gly Gly Asp Tyr<br>
35	40	45<br>
Ser Pro Ala Asn Gin Glu Arg val Ala Asp Gin Tyr Val Ala ser Arg<br>
50	55	60<br>
Tyr Gly ser Trp Thr Ala Ala Gin Gin Phe Trp Gin Ala Asn Gly Trp<br>
65	70	75	80<br>
Tyr<br><br>
S80769691.ST25<br>	3<br>	44<br>	PRT<br>	Lactobaclllus ferment urn<br>   3<br>
Ser Asp Gly Glu He Gin Glu Tyr Asn Ala Gin Asn Ala Ala Asn Ala<br>
15	10	15<br>
Asn val Ala Asn Asn Asn Thr Gin Ala Thr Gin Gin Gin Thr Ala Gin<br>
20	25	30<br>
Ala Gin Pro Gin Gin Ala Gin Ser Gin Ala Asn Gin<br>
35	40<br>	4<br>	30<br>	PRT<br>	Lactobacillus fermentum<br>    4<br>
Met lie Ser Lys Lys Asn Phe Ala Lys val Ser Ala Thr Leu Gly Ala<br>
15	10	15<br>
val Ala Leu Gly val Ser Ala Thr Ala Thr Ala Ala Asn Ala<br>
20	25	30<br>	5<br>	175<br>	PRT<br>	Lactobaclllus fermentum<br>    5<br>
Asp Thr He Tyr Thr Val  Gin Ser Gly Asp Thr Leu Ser Gly lie Ser<br>
15	10	15<br>
Tyr Lys Phe Ala Lys Asp Asn Ser Met lie Asn Asp Leu Ala Lys Lys<br>
20	25	30<br>
Asn Asn lie Gin Asp lie Asn Lys lie Phe Val  Gly Gin Lys Leu lie<br>
35	40	45<br>
lie Lys Ser Asp Gly Glu lie Gin Glu Tyr Asn Ala Gin Asn Ala Ala<br>
50	55	60<br>
Asn Ala Asn Val Ala Asn Asn Asn Thr Gin Ala Thr Gin Gin Gin Thr<br>
65	70	75	80<br>
Ala Gin Ala Gin Pro Gin Gin Ala Gin Ser Gin Ala Asn Gin ser Tyr<br>
85	90	95<br>
Thr ser Asn Ala ser Gly ser Glu Ala Ala Ala Lys Ala Trp lie Ala<br>
100	105	110<br><br>
S80769691.ST25<br>
Gly Arg Glu Ser Gly Gly Asn Tyr Asn Ala Thr Asn Gly Gin Tyr lie<br>
115	120	125<br>
Gly Lys Tyr Gin Leu Ala Ala Ser Tyr Leu Gly Gly Asp Tyr Ser Pro<br>
130	135	140<br>
Ala Asn Gin Glu Arg Val Ala Asp Gin Tyr Val Ala Ser Arg Tyr Gly<br>
145	150	155	160<br>
ser Trp Thr Ala Ala Gin Gin Phe Trp Gin Ala Asn Gly Trp Tyr<br>
165	170	175<br>	6<br>	205<br>	PRT<br>	Lactobacillus fermentum<br>    6<br>
Met lie Ser Lys Lys Asn Phe Ala Lys Val Ser Ala Thr Leu Gly Ala<br>
15	10	15<br>
val Ala Leu Gly Val ser Ala Thr Ala Thr Ala Ala Asn Ala Asp Thr<br>
20	25	30<br>
lie Tyr Thr val Gin Ser Gly Asp Thr Leu ser Gly lie ser Tyr Lys<br>
35	40	45<br>
Phe Ala Lys Asp Asn ser Met lie Asn Asp Leu Ala Lys Lys Asn Asn<br>
50	55	60<br>
lie Gin Asp He Asn Lys lie Phe val Gly Gin Lys Leu He lie Lys<br>
65	70	75	80<br>
Ser Asp Gly Glu lie Gin Glu Tyr Asn Ala Gin Asn Ala Ala Asn Ala<br>
85	90	95<br>
Asn val Ala Asn Asn Asn Thr Gin Ala Thr Gin Gin Gin Thr Ala Gin<br>
100	105	110<br>
Ala Gin Pro Gin Gin Ala Gin Ser Gin Ala Asn Gin ser Tyr Thr Ser<br>
115	120	125<br>
Asn Ala Ser Gly Ser Glu Ala Ala Ala Lys Ala Trp lie Ala Gly Arg<br>
130	135	140<br>
Glu ser Gly Gly Asn Tyr Asn Ala Thr Asn Gly Gin Tyr lie Gly Lys<br>
145	150	155	160<br>
Tyr Gin Leu Ala Ala Ser Tyr Leu Gly Gly Asp Tyr Ser Pro Ala Asn<br>
165	170	175<br><br>
S80769691.ST25<br>
Gin Glu Arg val Ala Asp Gin Tyr val Ala ser Arg Tyr Gly ser Trp<br>
180	185	190<br>
Thr Ala Ala Gin Gin Phe Trp Gin Ala Asn Gly Trp Tyr<br>
195	200	205<br>	7<br>	90<br>	DNA<br>	Lactobaclllus fermentum<br> 7<br>
atgatttcta agaaaaactt tgctaaagta tctgctactc ttggtgcagt ggccttaggt   60<br>
gttagtgcaa cggctactgc tgctaatgct	90<br>	8<br>	150<br>	DMA<br>	Lactobacillus fermentum<br> 8<br>
gacactatct acaccgtaca aagtggtgac acactttcag gtatttctta caaatttgct   60<br>
aaagacaaca gtatgatcaa tgatcttgct aagaagaaca atattcaaga tattaacaag   120<br>
atttttgttg gtcaaaagtt aatcatcaag	150<br>	9<br>	132<br>	DNA<br>	Lactobacillus fementum<br>    9<br>
agcgatggtg aaattcaaga atacaatgct caaaatgcag ctaatgcaaa tgtagcaaac         60<br>
aacaatactc aagctacaca acaacaaact gctcaagcac aacctcaaca agcacaaagc       120<br>
caagctaacc aa	132<br>	10<br>	246<br>	DNA<br>	Lactobacillus fermentum<br> 10<br>
agctacactt caaatgcttc aggttcagaa gctgctgcta aagcttggat tgccggtcgt   60<br>
gaatcaggtg gtaactacaa cgccacaaac ggtcaataca ttggtaagta ccaattagct	120<br>
gcatcatacc ttggtggtga ctactcacca gctaaccaag aacgcgttgc tgaccaatac	180<br>
gttgcaagtc gttacggttc ttggactgct gcccaacaat tctggcaagc aaacggttgg	240<br>
tactaa	246<br>	11<br>	528<br>	DNA<br>	Lactobacillus fermentum<br><br>
S80769691.ST25     11 gacactatct acaccgtaca aagtggtgac acactttcag gtatttctta caaatttgct         60<br>
aaagacaaca gtatgatcaa tgatctrtgct aagaagaaca	atattcaaga tattaacaag	120<br>
atttttgttg gtcaaaagtt aatcatcaag agcgatggtg	aaattcaaga atacaatgct	180<br>
caaaatgcag ctaatgcaaa tgtagcaaac aacaatactc	aagctacaca acaacaaact	240<br>
gctcaagcac aacctcaaca agcacaaagc caagctaacc	aaagctacac ttcaaatgct	300<br>
tcaggttcag aagctgctgc taaagcttgg attgccggtc	gtgaatcagg tggtaactac	360<br>
aacgccacaa acggtcaata cattggtaag taccaattag	ctgcatcata ccttggtggt	420<br>
gactactcac cagctaacca agaacgcgtt gctgaccaat	acgttgcaag tcgttacggt	480<br>
tcttggactg ctgcccaaca attctggcaa gcaaacggtt	ggtactaa	528<br>	12<br>	618<br>	DNA<br>	Lactobacillus fermentum<br>   12<br>
atgatttcta agaaaaactt tgctaaagta tctgctactc ttggtgcagt ggccttaggt         60<br>
gttagtgcaa cggctactgc tgctaatgct gacactatct acaccgtaca aagtggtgac	120<br>
acactttcag gtatttctta caaatttgct aaagacaaca gtatgatcaa tgatcttgct	180<br>
aagaagaaca atattcaaga tattaacaag atttttgttg gtcaaaagtt aatcatcaag	240<br>
agcgatggtg aaattcaaga atacaatgct caaaatgcag ctaatgcaaa tgtagcaaac	300<br>
aacaatactc aagctacaca acaacaaact gctcaagcac aacctcaaca agcacaaagc	360<br>
caagctaacc aaagctacac ttcaaatgct tcaggttcag aagctgctgc taaagcttgg	420<br>
attgccggtc gtgaatcagg tggtaactac aacgccacaa acggtcaata cattggtaag	480<br>
taccaattag ctgcatcata ccttggtggt gactactcac cagctaacca agaacgcgtt	540<br>
gctgaccaat acgttgcaag tcgttacggt tcttggactg ctgcccaaca attctggcaa	600<br>
gcaaacggtt ggtactaa	618<br>	13<br>	90<br>	DNA<br>	Lactobacillus fermentum<br><br>	misc_feature<br>	(9) .. (9)<br>	any nucleotide<br><br>	misc_feature<br>	(24) .. (24)<br>	any nucleotide<br><br>   misc_feature<br>    (30) .. (30)<br><br>
S80769691.ST25<br>	any nucleotide<br><br>	mi sc_feature<br>	(33)..(33)<br>	any nucleotide<br><br>	mi sc_feature<br>	(36) .. (36)<br>	any nucleotide<br><br>	mi sc_feature<br>	(39) .. (39)<br>	any nucleotide<br><br>	misc_feature<br>	(42) .. (42)<br>	any nucleotide<br><br>	misc_feature<br>	(45).. (45)<br>	any nucleotide<br><br>	mi sc_feature<br>	(48).. (48)<br>	any nucleotide<br><br>	misc_feature<br>	(51).. (51)<br>	any nucleotide<br><br>	misc_feature<br>	(54).. (54)<br>	any nucleotide<br><br>	mi sc_feature<br>	(57).. (57)<br>	any nucleotide<br><br>	mi sc_feature<br>	(60).. (60)<br>	any nucleotide<br><br>	misc_feature<br>	(63)..(63)<br>	any nucleotide<br><br>	misc_feature<br>	(66) . . (66)<br>	any nucleotide<br><br>	misc_feature<br>	(69) .. (69)<br>	any nucleotide<br><br><br>
S80769691.ST25<br>	iri sc_featu re<br>	(72)..(72)<br>	any nucleotide<br><br>	mi sc_f eatu re<br>	(75)..(75)<br>	any nucleotide<br><br>	misc_feature<br>	(78)..(78)<br>	any nucleotide<br><br>	misc_feature<br>	(81)..(81)<br>	any nucleotide<br><br>	mi sc_featu re<br>	(84) .. (84)<br>	any nucleotide<br><br>	misc_feature<br>	(90).. (90)<br>	any nucleotide<br>	13<br>
atgathwsna araaraaytt ygcnaargtn wsngcnacny tnggngcngt ngcnytnggn         60<br>
gtnwsngcna cngcnacngc ngcnaaygcn	90<br>	14<br>	150<br>	DNA<br>	Lactobadllus fe men turn<br><br>	Hi sc_featu re<br>	(6).. (6)<br>	any nucleotide<br><br>	misc_feature<br>	(15)..(15)<br>	any nucleotide<br><br>	misc_feature<br>	(18)..(18)<br>	any nucleotide<br><br>	mi s c_f eatu re<br>	(24)..(24)<br>	any nucleotide<br><br>	misc_feature<br>	(27)..(27)<br>	any nucleotide<br><br>	misc_feature<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
(33)..(33) any nucleotlde<br>
wise-feature (36)..(36) any nucleotlde<br>
misc_feature (39)..(39) any nucleotlde<br>
nrisc_feature (42)..(42) any nucleotlde<br>
nrisc_feature (48)..(48) any nucleotlde<br>
misc_feature (60)..(60) any nucleotlde<br>
nrisc_feature (72)..(72) any nucleotlde<br>
vise-feature (87)..(87) any nucleotlde<br>
•ISC-feature (90)..(90) any nucleotlde<br>
misc_feature (129)..(129} any nucleotlde<br>
misc_feature (132)..(132) any nucleotlde<br>
misc_feature (141)..(141) any nucleotlde<br><br>
S80769691.ST25<br><br> 14<br>
gayacnatht ayacngtnca rwsnggngay acnytnwsng gnathwsnta yaarttygcn	60<br>
aargayaayw snatgathaa ygayytngcn aaraaraaya ayathcarga yathaayaar	120<br>
athttygtng gncaraaryt nathathaar	150<br> 15  132<br><br>
S80769691.ST25  DMA  Lactobacillus ferwentum<br><br>	mi sc_f eatu re<br>	(3)..(3)<br>	any nucleotide<br><br>	misc_feature<br>	(9).. C9)<br>	any nucleotide<br><br>	mi sc_f eatu re<br>	(30)..(30)<br>	any nucleotide<br><br>	misc_feature<br>	(39)..(39)<br>	any nucleotide<br><br>	misc_feature<br>	(42)..(42)<br>	any nucleotide<br><br>	misc_feature<br>	(48).. (48)<br>	any nucleotide<br><br>	mi sc_f eatu re<br>	(54)..(54)<br>	any nucleotide<br><br>	»isc_feature<br>	(57)..(57)<br>	any nucleotide<br><br>	misc_feature<br>	(69)..(69)<br>	any nucleotide<br><br>	misc_feature<br>	(75)..(75)<br>	any nucleotide<br><br>	mi sc_feature<br>	(78)..(78)<br>	any nucleotide<br><br>	mi sc_feature<br>	(90)..(90)<br>	any nucleotide<br><br>	mi sc_feature<br>	(93)..(93)<br>	any nucleotide<br><br>
S80769691.ST25<br><br>	mi sc__f eatu re<br>	(99).. (99)<br>	any nucleotide<br><br>	misc_feature<br>	(105)..(105)<br>	any nucleotide<br><br>	nisc_feature<br>	(114)..(114)<br>	any nucleotide<br><br>	Biisc_feature<br>	(120)..(120)<br>	any nucleotide<br><br>	misc_feature<br>	(126) .. (126)<br>	any nucleotide<br>	15<br>
wsngayggng arathcarga rtayaaygcn caraaygcng cnaaygcnaa ygtngcnaay         60<br>
aayaayacnc argcnacnca  rcarcaracn gcncargcnc arccncarca rgcncarwsn        120<br>
cargcnaayc ar	132<br>	16<br>	243<br>	DMA<br>	Lactobacillus ferment urn<br><br>	irisc_feature<br>	(3).. (3)<br>	any nucleotide<br><br>	misc_feature<br>	(9)..(9)<br>	any nucleotide<br><br>	mi s c_featu re<br>	(12)..(12)<br>	any nucleotide<br><br>	nrisc_feature<br>	(18)..(18)<br>	any nucleotide<br><br>	mi sc_feature<br>	(21)..(21)<br>	any nucleotide<br><br>	misc_feature<br>	(24)..(24)<br><br>
S80769691.ST25<br>	any nucleotide<br><br>	misc_feature<br>	(27)..(27)<br>	any nucleotide<br><br>	niscjfeature<br>	(33)..(33)<br>	any nucleotide<br><br>	mi sc_feature<br>	(36)..(36)<br>	any nucleotide<br><br>	misc_feature<br>	(39)..(39)<br>	any nucleotide<br><br>	misc_feature<br>	(45)..(45)<br>	any nucleotide<br><br>	misc_feature<br>	(54)..(54)<br>	any nucleotide<br><br>	nrisc_feature<br>	(57)..(57)<br>	any nucleotide<br><br>	»isc_feature<br>	(60)..(60)<br>	any nucleotide<br><br>	mi sc_feature<br>	(66).. (66)<br>	any nucleotide<br><br>	misc_feature<br>	(69)..(69)<br>	any nucleotide<br><br>	misc_feature<br>	(72)..(72)<br>	any nucleotide<br><br>	misc_feature<br>	(84)..(84)<br>	any nucleotide<br><br>	mi sc_feature<br>	(87)..(87)<br>	any nucleotide<br><br><br>
S80769691.ST25<br>	itrisc_f eature<br>	(93)..C93)<br>	any nucleotide<br><br>	misc_feature<br>	(105).. C105)<br>	any nucleotide<br><br>	mi sc_featu re<br>	(117).. C117)<br>	any nucleotide<br><br>	mi sc_feature<br>	(120).. (120)<br>	any nucleotide<br><br>	misc_feature<br>	(123).. C123)<br>	any nucleotide<br><br>	nrisc_feature<br>	(126).. C126)<br>	any nucleotide<br><br>	mi sc_feature<br>	(132)..C132)<br>	any nucleotide<br><br>	mi sc_feature<br>	(135)..C135)<br>	any nucleotide<br><br>	mi sc_feature<br>	C138)..C138)<br>	any nucleotide<br><br>	mi sc_f eature<br>	(147).. C147}<br>	any nucleotide<br><br>	mi sc_f eature<br>	(150).. C150)<br>	any nucleotide<br><br>	mi sc_f eature<br>	(153)..C153)<br>	any nucleotide<br><br>	misc_feature<br>	(165).. C165)<br>	any nucleotide<br><br>	mi sc_f eature<br>	(168).. C168)<br>	any nucleotide<br><br>
S80769691.ST25<br><br><br><br><br><br><br><br><br><br><br><br><br>
•risc_feat ure (171).. (171) any nucleotide<br>
misc_feature (183).. (183) any nucleotide<br>
•rise-feature (186).. (186) any nucleotide<br>
misc_feature (189).. (189) any nucleotide<br>
misc_feature (192).. (192) any nucleotide<br>
misc_feature (198).. (198) any nucleotide<br>
•rise-feature (201).. (201) any nucleotide<br>
•rise-feature (207).. (207) any nucleotide<br>
•rise—feature (210).. (210) any nucleotide<br>
misc_feature (213).. (213) any nucleotide<br><br><br><br><br>	mi sc_f eatu re<br>	(231)..(2 31)<br>	any nucleotide<br><br>	nrisc-feature<br>	(237).. (237)<br>	any nucleotide<br>    16<br>
wsntayacnw snaaygcnws nggnwsngar gcngcngcna argcntggat hgcnggnmgn<br>
garwsnggng gnaaytayaa ygcnacnaay ggncartaya thggnaarta ycarytngcn genwsntayy tnggnggnga ytaywsnccn gcnaaycarg armgngtngc ngaycartay<br><br>
60 120 180<br><br>
S80769691.ST25 gtngcnwsnm gntayggnws ntggacngcn gcncarcart tytggcargc naayggntgg   240<br>
tay	243<br>	17<br>	525<br>	DMA<br>	Lactobaci 11 us fermentum<br><br>	mi sc_f eatu re<br>	(6).. (6)<br>	any nucleotide<br><br>	misc_feature<br>	(15).. 
	any nucleotide<br><br>	nrisc_feature<br>	(18).. C18)<br>	any nucleotide<br><br>	misc_feature<br>	(24) .. C24)<br>	any nucleotide<br><br>	uri sc_f eatu re<br>	(27).. C27)<br>	any nucleotide<br><br>	wise-feature<br>	(33).. C33)<br>	any nucleotide<br><br>	mi sc_f eature<br>	(36).. C36)<br>	any nucleotide<br><br>	mi sc_f eature<br>	(39)..C39)<br>	any nucleotide<br><br>	misc_feature<br>	(42).. C42)<br>	any nucleotide<br><br>	misc_feature<br>	(48) .. C48)<br>	any nucleotide<br><br>	mi sc_f eature<br>	(60)..C60)<br>	any nucleotide<br><br>	misc_feature<br><br>    C72)..(72)	S80769691.ST25<br>	any nucleotide<br><br>	mi sc_f eatu re<br>	C87)..(87)<br>	any nucleotide<br><br>	mi sc_f eature<br>	C90).. (90)<br>	any nucleotide<br><br>	mi sc__f eature<br>	C129)..(129)<br>	any nucleotide<br><br>	mi sc_f eature<br>	C132) .. (132)<br>	any nucleotide<br><br>	misc_feature<br>	C141).. (141)<br>	any nucleotide<br><br>	misc_feature<br>	C153).. (153)<br>	any nucleotide<br><br>	nisc_feature<br>	C159) .. (159)<br>	any nucleotide<br><br>	mi sc_feature<br>	(180).. (180)<br>	any nucleotide<br><br>	mi sc_f eature<br>	(189) . . (189)<br>	any nucleotide<br><br>	mi sc_f eature<br>	(192) . . (192)<br>	any nucleotide<br><br>	misc_feature<br>	(198) . . (198)<br>	any nucleotide<br><br>	mi sc_feature<br>	(204) .. (204)<br>	any nucleotide<br><br>	mi sc_f eature<br>	(207) .. (207)<br>	any nucleotide<br><br>
S80769691.ST25 <br>	mi sc_f eature<br>	(219). . (219)<br>	any nucleotide<br><br>	misc_feature<br>	C225)..C225)<br>	any nucleotide<br><br>	mi sc_f eature<br>	(228). . (228)<br>	any nucleotide<br><br>	mi sc_featu re<br>	(240) . . (240)<br>	any nucleotide<br><br>	mi sc_f eatu re<br>	(243). . (243)<br>	any nucleotlde<br><br>	mi sc_f eature<br>	(249). . (249}<br>	any nucleotide<br><br>	mi sc_f eatu re<br>	(255).. (255)<br>	any nucleotide<br><br>	Brisc_feature<br>	(264)..C264)<br>	any nucleotide<br><br>	mi sc_fe atu re<br>	(270).. C270)<br>	any nucleotide<br>	mi sc_featu re<br>	(276).. C276)<br>	any nucleotide<br><br>	mi sc_feature<br>	(285) .. C285)<br>	any nucleotide<br><br>	mi sc_f eatu re<br>	(291).. C291)<br>	any nucleotide<br><br>	misc_feature<br>	(294) .. C294)<br>	any nucleotide<br><br>	misc_feature<br>	(300).. C300)<br>
_Eage-l€<br><br>
S80769691.ST25<br>	any nucleotide<br><br>	misc_feature<br>	(303)..(303)<br>	any nucleotide<br><br>	misc_feature<br>	(306)..(306)<br>	any nucleotide<br><br>	misc_feature<br>	(309)..(309)<br>	any nucleotide<br><br>	misc_feature<br>	(315)..(315)<br>	any nucleotide<br><br>	inisc_feature<br>	(318)..(318)<br>	any nucleotide<br><br>	misc_feature<br>	(321)..(321)<br>	any nucleotide<br><br>	nrisc_feature<br>	(327)..(327)<br>	any nucleotide<br><br>	«isc__feature<br>	(336)..(336)<br>	any nucleotide<br><br>	misc_feature<br>	(339)..(339)<br>	any nucleotide<br><br>	misc_feature<br>	(342)..(342)<br>	any nucleotide<br><br>	misc_feature<br>	(348)..(348)<br>	any nucleotide<br><br>	misc_feature<br>	(351)..(351)<br>	any nucleotide<br><br>	misc_feature<br>	(354)..(354)<br>	any nucleotide<br><br><br>
S80769691.ST25<br>	nrisc_feature<br>	(366)..(366)<br>	any nucleotide<br><br>	miscL-feature<br>	(369)..(369)<br>	any nucleotide<br><br>	mi sc_feature<br>	(375)..(375)<br>	any nucleotide<br><br>	misc_feature<br>	(387)..(387)<br>	any nucleotide<br><br>	mi sc_f eature<br>	(399)..(399)<br>	any nucleotide<br><br>	mi sc_f eature<br>	(402)..(402)<br>	any nucleotide<br><br>	misc_feature<br>	(405).. (405)<br>	any nucleotide<br><br>	wise-feature<br>	(408) .. (408)<br>	any nucleotide<br><br>	wise-feature<br>	(414) .. (414)<br>	any nucleotide<br><br>	mi sc_f eature<br>	(417) .. (417}<br>	any nucleotide<br><br>	mi s c_f eatu re<br>	(420)..(420)<br>	any nucleotide<br><br>	misc_feature<br>	(429)..(429)<br>	any nucleotide<br><br>	misc_feature<br>	(432)..(432)<br>	any nucleotide<br><br>	mi sc_feature<br>	(435)..(435)<br>	any nucleotide<br><br><br>	misc_feature<br>	(447).. (447)<br>	any nucleotide<br><br>	raisc_feature<br>	(450)..(450)<br>	any nucleotide<br><br>	mi sc_feature<br>	(453) .. (453)<br>	any nucleotide<br><br>	misc_feature<br>	(465)..(465)<br>	any nucleotide<br><br>	misc_feature<br>	(468) .. (468)<br>	any nucleotide<br><br>	mi sc_f eature<br>	(471)..(471)<br>	any nucleotide<br><br>	mi sc_feature<br>	(474).. (474)<br>	any nucleotide<br><br>	nrisc_feature<br>	(480).. (480)<br>	any nucleotide<br><br>	misc_feature<br>	(483) .. (483)<br>	any nucleotide<br><br>	misc_feature<br>	(489)..(489)<br>	any nucleotide<br><br>	misc_feature<br>	(492)..(492)<br>	any nucleotide<br><br>	misc_feature<br>	(495) . . (495)<br>	any nucleotide<br><br>	misc_feature<br>	(513)..(513)<br>	any nucleotide<br><br>	mi sc_f eature<br><br>
S80769691.ST25<br><br>
S80769691.ST25  (519)..(519)  any nucleotide<br> 17<br>
gayacnatht ayacngtnca rwsnggngay acnytnwsng gnathwsnta yaarttygcn   60<br>
aargayaayw snatgathaa ygayytngcn aaraaraaya ayathcarga yathaayaar	120<br>
athttygtng gticaraaryt nathathaar wsngayggng arathcarga rtayaaygcn	180<br>
caraaygcng cnaaygcnaa ygtngcnaay aayaayacnc argcnacnca rcarcaracn	240<br>
gcncargcnc arccncarca rgcncarwsn cargcnaayc arwsntayac nwsnaaygcn	300<br>
wsnggnwsng argcngcngc naargcntgg athgcnggnm gngarwsngg nggnaaytay	360<br>
aaygcnacna ayggncarta yathggnaar taycarytng cngcnwsnta yytnggnggn	420<br>
gaytaywsnc cngcnaayca rgarmgngtn gcngaycart aygtngcnws nmgntayggn	480<br>
wsntggacng cngcncarca rttytggcar gcnaayggnt ggtay	525<br>	18<br>	615<br>	DNA<br>	Lactobacillus fermentura<br><br>	nrisc_feature<br>	(9).. (9)<br>	any nucleotide<br><br>	»isc_feature<br>	(24)..(24)<br>	any nucleotide<br><br>	misc_feature<br>	(30)..(30)<br>	any nucleotide<br><br>	misc_feature<br>	(33)..(33)<br>	any nucleotide<br><br>	misc_feature<br>	(36)..(36)<br>	any nucleotide<br><br>	mi s c_f eat u re<br>	(39)..(39)<br>	any nucleotide<br><br>	nrisc_feature<br>	(42)..(42)<br>	any nucleotide<br><br>	misc_feature<br>	(45)..(45)<br>	any nucleotide<br><br><br>	«isc_feature<br>	(48).. (48)<br>	any nucleotide<br><br>	nrisc_f eature<br>	(51)..(51)<br>	any nucleotide<br><br>	nrisc_feature<br>	(54)..(54)<br>	any nucleotide<br><br>	misc_feature<br>	(57)..(57)<br>	any nucleotide<br><br>	misc_feature<br>	(60)..(60)<br>	any nucleotide<br><br>	misc_feature<br>	(63)..(63)<br>	any nucleotide<br><br>	wise-feature<br>	(66)..(66)<br>	any nucleotide<br><br>	sri sc_feature<br>	(69).. (69)<br>	any nucleotide<br><br>	misc_feature<br>	(72)..(72)<br>	any nucleotide<br><br>	misc_feature<br>	(75)..(75)<br>	any nucleotide<br><br>	misc_feature<br>	(78)..(78)<br>	any nucleotide<br><br>	misc_feature<br>	(81)..(81)<br>	any nucleotide<br><br>	mi sc_feature<br>	(84)..(84)<br>	any nucleotide<br><br>	misc_feature<br><br>
S80769691.ST25<br><br>
S80769691.ST25<br>	(90).. C90)<br>	any nucleotide<br><br>	mi sc_featu re<br>	(96)..(96)<br>	any nucleotide<br><br>	misc_feature<br>	(105)..(105)<br>	any nucleotide<br><br>	misc_feature<br>	(108)..(108)<br>	any nucleotide<br><br>	mi sc_featu re<br>	(114)..(114}<br>	any nucleotide<br><br>	misc_feature<br>	(117)..(117)<br>	any nucleotide<br><br>	misc_feature<br>	(123)..(123)<br>	any nucleotide<br><br>	wise-feature<br>	(126)..(126)<br>	any nucleotide<br><br>	urisc-feature<br>	(129).. (129)<br>	any nucleotide<br><br>	misc_feature<br>	(132)..(132)<br>	any nucleotide<br><br>	misc_feature<br>	(138)..(138)<br>	any nucleotide<br><br>	misc_feature<br>	(150) .. (150)<br>	any nucleotide<br><br>	misc_feature<br>	(162)..(162}<br>	any nucleotide<br><br>	misc_feature<br>	(177)..(177)<br>	any nucleotide<br><br>
S80769691.ST25 <br>	»isc_feature<br>	(180)..(180)<br>	any nucleotide<br><br>	miscL-feature<br>	(219)..(219}<br>	any nucleotide<br><br>	Brisc_feature<br>	(222)..(222)<br>	any nucleotide<br><br>	misc_feature<br>	(231)..(231)<br>	any nucleotide<br><br>	misc_feature<br>	(243)..(243)<br>	any nucleotide<br><br>	misc_feature<br>	(249)..(249)<br>	any nucleotide<br><br>	Brisc_feature<br>	(270).. (270)<br>	any nucleotide<br><br>	irisc_feature<br>	(279)..(279)<br>	any nucleotide<br><br>	misc-feature<br>	(282)..(282)<br>	any nucleotide<br><br>	misc_feature<br>	(288)..(288)<br>	any nucleotide<br><br>	misc_feature<br>	(294)..(294)<br>	any nucleotide<br><br>	misc_feature<br>	(297)..(297)<br>	any nucleotide<br><br>	misc_feature<br>	(309)..(309)<br>	any nucleotide<br><br>	misc_feature<br>	(315)..(315)<br><br>
S80769691.ST25<br>	any nucleotide<br><br>	mi sc_feature<br>	(318)..(318)<br>	any nucleotide<br><br>	misc_feature<br>	(330)..(330)<br>	any nucleotide<br><br>	misc_feature<br>	(333)..(333)<br>	any nucleotide<br><br>	misc_feature<br>	(339)..(339)<br>	any nucleotide<br><br>	misc_feature<br>	(345)..(345)<br>	any nucleotide<br><br>	misc_feature<br>	(354)..(354)<br>	any nucleotide<br><br>	«risc_feature<br>	(360)..(360)<br>	any nucleotide<br><br>	nrisc_feature<br>	(366)..(366)<br>	any nucleotide<br><br>	wise-feature<br>	(375)..(375)<br>	any nucleotide<br><br>	misc_featupe<br>	(381)..(381)<br>	any nucleotide<br><br>	mi sc_feature<br>	(384)..(384)<br>	any nucleotide<br><br>	misc_feature<br>	(390)..(390)<br>	any nucleotide<br><br>	misc_feature<br>	(393)..(393)<br>	any nucleotide<br><br><br>
S80769691.ST25<br>	mi sc_feature<br>	(396)..(396)<br>	any nucleotide<br><br>	mi sc_feature<br>	C399),.(399)<br>	any nucleotide<br><br>	mi sc_feature<br>	(405)..(405)<br>	any nucleotide<br><br>	misc_feature<br>	(408)..(408)<br>	any nucleotide<br><br>	misc_feature<br>	(411).. (411)<br>	any nucleotide<br><br>	misc_feature<br>	(417)..(417)<br>	any nucleotide<br><br>	nrisc_feature<br>	(426)..(426)<br>	any nucleotide<br><br>	nrisc_feature<br>	(429)..(429)<br>	any nucleotide<br><br>	mi sc_featu re<br>	(432)..(432)<br>	any nucleotide<br><br>	misc_feature<br>	(438)..(438)<br>	any nucleotide<br><br>	mis c_featu re<br>	(441)..(441)<br>	any nucleotide<br><br>	mi sc_feature<br>	(444)..(444)<br>	any nucleotide<br><br>	misc_feature<br>	(456)..(456)<br>	any nucleotide<br><br>	misc_feature<br>	(459)..(459)<br>	any nucleotide<br><br><br>	misc_feature<br>	(465)..(465)<br>	any nucleotide<br><br>	nrisc_feature<br>	(477)..(477)<br>	any nucleotide<br><br>	mi sc_feature<br>	(489)..(489)<br>	any nucleotide<br><br>	misc_feature<br>	(492)..(492)<br>	any nucleotide<br><br>	mi s c_featu re<br>	(495)..(495)<br>	any nucleotide<br><br>	misc_feature<br>	(498)..(498)<br>	any nucleotide<br><br>	wise-feature<br>	(504)..(504)<br>	any nucleotide<br><br>	misc_feature<br>	(507)..(507)<br>	any nucleotide<br><br>	misc_feature<br>	(510)..(510)<br>	any nucleotide<br><br>	misc_feature<br>	(519)..(519)<br>	any nucleotide<br><br>	misc_feature<br>	(522)..(522}<br>	any nucleotide<br><br>	mi sc_feature<br>	(525)..(525}<br>	any nucleotide<br><br>	misc_feature<br>	(537)..(537)<br>	any nucleotide<br><br>	mi s c_featu re<br><br>
S80769691.ST25<br><br>
580769691.ST25<br>	C540) .. (540)<br>	any nucleotide<br><br>	misc_feature<br>	(543)..(543)<br>	any nucleotide<br><br>	misc_feature<br>	(555)..(555)<br>	any nucleotide<br><br>	misc_feature<br>	(558)..(558)<br>	any nucleotide<br><br>	misc_feature<br>	(561) .. (561)<br>	any nucleotide<br><br>	misc_feature<br>	(564)..(564)<br>	any nucleotide<br><br>	mi sc_f eatu re<br>	(570)..(570)<br>	any nucleotide<br><br>	wise-feature<br>	(573)..(573}<br>	any nucleotide<br><br>	*isc_feature<br>	(579)..(579)<br>	any nucleotide<br><br>	mi sc_feature<br>	(582)..(582)<br>	any nucleotide<br><br>	misc_feature<br>	(585)..(585)<br>	any nucleotide<br><br>	mi s c_f eatu re<br>	(603)..(603)<br>	any nucleotide<br><br>	mi sc_f eature<br>	(609)..(609)<br>	any nucleotide<br>	18<br>
atgathwsna araaraaytt ygcnaargtn wsngcnacny tnggngcngt ngcnytnggn   60<br>
gtnwsngcna cngcnacngc ngcnaaygcn gayacnatht ayacngtnca rwsnggngay  120<br><br>
S80769691.ST25 acnytnwsng gnathwsnta yaarttygcn aargayaayw snatgathaa ygayytngcn       180<br>
aaraaraaya ayathcarga yathaayaar athttygtng gncaraaryt nathathaar	240<br>
wsngayggng arathcarga rtayaaygcn caraaygcng cnaaygcnaa ygtngcnaay	300<br>
aayaayacnc argcnacnca rcarcaracn gcncargcnc arccncarca rgcncarwsn	360<br>
cargcnaayc arwsntayac nwsnaaygcn wsnggnwsng argcngcngc naargcntgg	420<br>
athgcnggnm gngarwsngg nggnaaytay aaygcnacna ayggncarta yathggnaar	480<br>
taycarytng cngcnwsnta yytnggnggn gaytaywsnc cngcnaayca rgarmgngtn	540<br>
gcngaycart aygtngcnws nmgntayggn wsntggacng cngcncarca rttytggcar	600<br>
gcnaayggnt ggtay	615<br>	19<br>	310<br>	DMA<br>	Lactobacillus fermentura<br>   19<br>
taaagatagt tataaacgga aaataaaggg cggttttgga gcaaatatga aatttttgcg         60<br>
aagaaatcag ctttttttat ttattttttt ataaatcatc tgtaaaagtt atgcaaaccg	120<br>
aaaacgcaac ccgcacaagg aattagccga ttatgactat aatattttaa aagctatatt	180<br>
acaaaaagca aacggagagt agtaaataga aatggtgctg ttacagcttt gtaatattaa	240<br>
gagtgtagta tatagggtgt tgaaacggaa aagataattt gctaaataat aaaggatggt	300<br>
tatttaattt	310<br>	20<br>	150<br>	DMA<br>	Lactobacillus fermentum<br>    20<br>
aataaataag attaatcaat tttattgcga gactgatgga atattatttc cttctgtctc         60<br>
gcttttttgg gctaatatgt tataatggta gtacttctta tggggatgtt tatggattcg       120<br>
acaggtatag gtcgagtttc aactgcgttt	150<br><br><br>
We claim:<br>
1.	An apf-like domain represented by SEQ ID No. 2 capable of being formed as C-terminal of a Small Exported Protein or a variant thereof.<br>
2.	An apf-like domain as claimed in claim 1, wherein the variant bears at least 90% homology with SEQ ID NO. 2.<br>
3.	An apf-like domain as claimed in claim 2, wherein the amino acid sequence of the variant differs from SEQ ID NO. 2 only in the conservative substitution of one or more amino acids.<br>
4.	A nucleic acid sequence capable of encoding the apf-like domain as claimed in any one of the previous claims.<br>
5.	A nucleic acid as claimed in claim 4, wherein the nucleic acid comprises a sequence selected from SEQ ID NO. 10.<br>
6.	A vector comprising a nucleic acid sequence as claimed in any of the claims 4 or 5.<br>
7.	A prokaryotic host cell comprising the apf-like domain or nucleic acid or vector as claimed in any one of the previous claims.<br>
8.	A host cell as claimed in claim 7, wherein the prokaryotic cell is gram positive bacteria.<br>
9.	A host cell as claimed in claim 8, wherein the gram positive bacteria is selected from the group comprising Lactobacillus, Lactococcus, Bifidobacterium, Leuconostoc and Streptococcus.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1ERUxOUC0yMDA2LUFic3RyYWN0LSgwNC0wNy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">2940-DELNP-2006-Abstract-(04-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1ERUxOUC0yMDA2LUFic3RyYWN0LSgwOS0wNC0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">2940-DELNP-2006-Abstract-(09-04-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1kZWxucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2940-delnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1ERUxOUC0yMDA2LUNsYWltcy0oMDQtMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">2940-DELNP-2006-Claims-(04-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1ERUxOUC0yMDA2LUNsYWltcy0oMDktMDQtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">2940-DELNP-2006-Claims-(09-04-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1ERUxOUC0yMDA2LUNsYWltcy0oMTgtMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">2940-DELNP-2006-Claims-(18-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1kZWxucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2940-delnp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDQtMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">2940-DELNP-2006-Correspondence-Others-(04-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDktMDQtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">2940-DELNP-2006-Correspondence-Others-(09-04-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTgtMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">2940-DELNP-2006-Correspondence-Others-(18-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1kZWxucC0yMDA2LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2940-delnp-2006-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1ERUxOUC0yMDA2LURlc2NyaXB0aW9uIChDb21wbGV0ZSktKDE4LTAyLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">2940-DELNP-2006-Description (Complete)-(18-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1kZWxucC0yMDA2LWRlc2NyaXB0aW9uIChjb21wbGV0ZSktMDQtMDctMjAwOC5wZGY=" target="_blank" style="word-wrap:break-word;">2940-delnp-2006-description (complete)-04-07-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1kZWxucC0yMDA2LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">2940-delnp-2006-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1kZWxucC0yMDA2LWRyYXdpbmcucGRm" target="_blank" style="word-wrap:break-word;">2940-delnp-2006-drawing.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1ERUxOUC0yMDA2LURyYXdpbmdzLSgwNC0wNy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">2940-DELNP-2006-Drawings-(04-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1ERUxOUC0yMDA2LUZvcm0tMS0oMDQtMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">2940-DELNP-2006-Form-1-(04-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1ERUxOUC0yMDA2LUZvcm0tMS0oMDktMDQtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">2940-DELNP-2006-Form-1-(09-04-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1kZWxucC0yMDA2LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">2940-delnp-2006-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1kZWxucC0yMDA2LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">2940-delnp-2006-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1ERUxOUC0yMDA2LUZvcm0tMi0oMDQtMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">2940-DELNP-2006-Form-2-(04-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1ERUxOUC0yMDA2LUZvcm0tMi0oMDktMDQtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">2940-DELNP-2006-Form-2-(09-04-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1ERUxOUC0yMDA2LUZvcm0tMi0oMTgtMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">2940-DELNP-2006-Form-2-(18-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1kZWxucC0yMDA2LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">2940-delnp-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1ERUxOUC0yMDA2LUZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">2940-DELNP-2006-Form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1ERUxOUC0yMDA2LUZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">2940-DELNP-2006-Form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1kZWxucC0yMDA2LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">2940-delnp-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1ERUxOUC0yMDA2LUdQQS0oMDQtMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">2940-DELNP-2006-GPA-(04-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1kZWxucC0yMDA2LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">2940-delnp-2006-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1kZWxucC0yMDA2LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">2940-delnp-2006-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1kZWxucC0yMDA2LXBjdC0zMDEucGRm" target="_blank" style="word-wrap:break-word;">2940-delnp-2006-pct-301.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1kZWxucC0yMDA2LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">2940-delnp-2006-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1kZWxucC0yMDA2LXBjdC0zMDgucGRm" target="_blank" style="word-wrap:break-word;">2940-delnp-2006-pct-308.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1kZWxucC0yMDA2LXBjdC0zMTEucGRm" target="_blank" style="word-wrap:break-word;">2940-delnp-2006-pct-311.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1kZWxucC0yMDA2LXBjdC1yZXF1ZXN0IGZvcm0ucGRm" target="_blank" style="word-wrap:break-word;">2940-delnp-2006-pct-request form.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1kZWxucC0yMDA2LXBjdC1zZWFyY2ggcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">2940-delnp-2006-pct-search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1ERUxOUC0yMDA2LVBldGl0aW9uLTEzNy0oMDQtMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">2940-DELNP-2006-Petition-137-(04-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk0MC1ERUxOUC0yMDA2LVBldGl0aW9uLTEzNy0oMTgtMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">2940-DELNP-2006-Petition-137-(18-02-2009).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="233949-a-novel-n-2-4-r-2-hydroxy-2-phenylethylamino-phenyl-ethyl-r-2-hydroxy-2-3-formamido-4-hydroxyphenyl-ethylamine-monohydrochloride.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="233951-compositions.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>233950</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2940/DELNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>21/2005</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>22-May-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>22-Apr-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>22-May-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>QUEENSLAND UNIVERSITY OF TECHNOLOGY</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>2 GEORGE STREET, BRISBANE, QUEENSLAND 4001, AUSTRALIA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>TURNER, MARK, S.</td>
											<td>QUEENSLAND UNIVERSITY OF TECHNOLOGY, 2 GEORGE STREET, BRISBANE, QUEENSLAND 4001, AUSTRALIA.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>GIFFARD, PHILIP, M.</td>
											<td>QUEENSLAND UNIVERSITY OF TECHNOLOGY, 2 GEORGE STREET, BRISBANE, QUEENSLAND 4001,AUSTRALIA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A61L</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/AU2004/001461</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-10-22</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>2003905886</td>
									<td>2003-10-24</td>
								    <td>Australia</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/233950-small-exported-protein-c-terminal-apf-domain by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:59:21 GMT -->
</html>
